<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/264179-methods-for-detecting-a-target-nucleotide-sequence-in-a-sample-utilising-a-nuclease-aptamer-complex by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 00:06:09 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 264179:METHODS FOR DETECTING A TARGET NUCLEOTIDE SEQUENCE IN A SAMPLE UTILISING A NUCLEASE-APTAMER COMPLEX</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">METHODS FOR DETECTING A TARGET NUCLEOTIDE SEQUENCE IN A SAMPLE UTILISING A NUCLEASE-APTAMER COMPLEX</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>The present invention relates to methods for detecting a target nucleotide sequence in a sample. More particularly, the present invention relates to methods for detecting a target nucleotide sequence in a sample which utilise a nuclease-aptamer complex. The present invention also provides nuclease-binding aptamers, nuclease-aptamer complexes and linker molecules that may be used in accordance with the methods of the present invention.</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td>FORM 2<br>
THE PATENTS ACT 1970<br>
(39 of 1970)<br>
&amp;<br>
The Patents Rules, 2003<br>
COMPLETE SPECIFICATION<br>
(See section 10 and rule 13)<br><br><br><br>
1.  ' METHODS FOR DETECTING A TARGET NUCLEOTIDE SEQUENCE IN A SAMPLE UTILISING A NUCLEASE-APTAMER COMPLEX '<br><br>
2.<br><br>
1.  (A)  LIFEPRINT AUSTRALIA PTY LTD<br>
     (B)  A Company incorporated under the laws of Australia<br>
     (C)  Plant Genomics Centre, Hartley Grove, <br>
            Urrbrae, South Australia 5064,  Australia.<br><br><br>
The following specification particularly describes the invention and the manner in which it is to be performed.<br><br><br><br><br>
BIOSENSOR<br><br><br>
PRIORITY CLAIM<br><br>
This international patent application claims priority to Australian provisional patent application 2007901826 filed on 5 April 2007, the content of which is herein incorporated by reference.<br><br><br>
FIELD OF THE INVENTION<br><br>
The present invention relates to methods for detecting a target nucleotide sequence in a sample. More particularly, the present invention relates to methods for detecting a target nucleotide sequence in a sample which utilise a nuclease-aptamer complex.<br><br><br>
BACKGROUND OF THE INVENTION <br><br>
A range of molecular technologies for the detection of a nucleotide sequence of interest (also referred to herein as a ‘target nucleotide sequence’) have been developed. Such methods have application in, for example, public health, the detection of pathogens in food or water, epidemiological studies, genetically modified organism (GMO) detection, medicine, clinical diagnoses, disease susceptibility diagnoses, tissue typing, blood screening, forensic medicine, bioweapon detection, molecular toxicology, gene therapy, and DNA tagging, among many other applications.<br><br>
Current methods for detecting a target DNA sequence generally involve one, or a combination of, molecular techniques. These techniques generally fall into three groups loosely defined as sequence-specific detection, sequence-specific enrichment and signal amplification. <br><br>
Most detection techniques gain their sequence specificity through base paring of complementary probes or oligonucleotides to a sequence of interest within the target DNA sample. <br><br>
The two most commonly used DNA detection methods, polymerase chain reaction (PCR) and Southern blotting, differ in how they proceed from this point. The PCR method enriches a target DNA through a series of amplification cycles and signal detection can be, for example, though the use of stains, fluorescence or radiolabelling.  Southern blotting involves no DNA enrichment step, but uses high-energy 32P for signal amplification. These extensively used techniques, though highly developed, still retain significant drawbacks. For PCR, the equipment required is expensive, the process is time-consuming and the degree of expertise required is high. Southern blotting often uses hazardous radioactive labelling, takes up to a week to complete, and requires large amounts of substrate DNA. <br><br>
An effective biosensor for detecting a specific DNA sequence in a sample that addresses one or more of the deficiencies of the currently available techniques, without substantially losing target-specificity or sensitivity, would be desirable. Furthermore, such a technique would also ideally require a low capital input (particularly in the case of equipment requirement), minimal expertise or technique-specific training, and provide quick and accurate results. <br><br>
Reference to any prior art in this specification is not, and should not be taken as, an acknowledgment or any form of suggestion that this prior art forms part of the common general knowledge in any country.<br><br><br><br>
SUMMARY OF THE INVENTION<br><br>
The present invention is predicated in part on methods for the detection of a target nucleotide sequence in a sample, wherein the method utilises a nuclease-aptamer complex. <br><br>
Accordingly, in a first aspect, the present invention provides a method for detecting a target nucleotide sequence in a sample, the method comprising:<br>
providing a nuclease-aptamer complex, the complex comprising a nuclease bound to an aptamer, wherein binding of the aptamer to the nuclease inhibits the activity of the nuclease and wherein inhibition of the nuclease’s activity by the aptamer is directly or indirectly reduced or eliminated when the target nucleotide sequence is present in the sample; <br>
applying the nuclease-aptamer complex to the sample;<br>
detecting the activity of the nuclease, wherein increased nuclease activity is indicative of the presence of the target nucleotide sequence in the sample.<br><br>
Nucleases are particularly useful as indicators in aptamer complexes, as many nucleases are highly active and may achieve a higher degree of signal amplification than other enzymes. Furthermore, a wide array of nucleases are available, many of which act on different nucleic acid substrates or specific nucleic acid sequences. This target specificity also allows the present invention to provide ‘multiplex’ detection methods which allow the simultaneous detection of a plurality of target sequences in a sample. <br><br>
In some particular embodiments of the invention the nuclease may be a restriction endonuclease. <br><br>
In one embodiment the method of the present invention may exploit hybridisation between the aptamer and the target nucleotide sequence in order to modify, reduce or eliminate binding between the nuclease and the aptamer. Thus, in one embodiment the method of the present invention may comprise the steps of:<br>
providing a nuclease-aptamer complex, the complex comprising a nuclease bound to an aptamer, wherein binding of the aptamer to the nuclease inhibits the activity of the nuclease and wherein the aptamer is able to hybridise to a target nucleotide sequence to modify, reduce or eliminate binding between the nuclease and the aptamer when the target nucleotide sequence is present in the sample;<br>
applying the nuclease-aptamer complex to the sample;<br>
allowing the target nucleotide sequence, if present in the sample, to hybridise with the aptamer; and<br>
detecting the activity of the nuclease wherein increased nuclease activity is indicative of the presence of the target nucleotide sequence in the sample.<br><br>
In the above embodiment of the invention, the aptamer is designed or selected such that it comprises both a region which binds to the nuclease and a region which can hybridise to the target nucleotide sequence. However, in another embodiment of the invention, the necessity to design or select the aptamer to contain both a nuclease-binding region and a target nucleotide sequence binding region may be eliminated via the use of a linker nucleic acid. <br><br>
Accordingly, in another embodiment, the method of the present invention comprises the steps of:<br>
providing a nuclease-aptamer complex, the complex comprising a nuclease bound to an aptamer, wherein binding of the aptamer to the nuclease inhibits the activity of the nuclease;<br>
providing a linker nucleic acid comprising a first portion which can hybridise with the target nucleotide sequence if it is present in the sample, and a second portion which can hybridise with the aptamer when the first portion of the linker nucleic acid is hybridised to the target nucleotide sequence, wherein hybridisation between the aptamer and the second portion of the linker nucleic acid modifies, reduces or eliminates binding between the aptamer and the nuclease;<br>
applying the nuclease-aptamer complex to the sample;<br>
applying the linker nucleic acid to the sample;<br>
allowing the first portion of the linker nucleic acid to hybridise with the target nucleotide sequence if it is present in the sample;<br>
allowing the second portion of the linker nucleic acid to hybridise with the aptamer when the first portion of the linker nucleic acid hybridises with the target nucleotide sequence; and<br>
detecting the activity of the nuclease, wherein increased nuclease activity in the sample is indicative of the presence of the target nucleotide sequence in the sample.<br><br>
The use of a nuclease-aptamer complex in the methods of the present invention also allow the methods of the present invention to be multiplexed, that is, the present invention also provides methods to simultaneously detect a plurality of target nucleotide sequences in a sample. <br><br>
In a second aspect, the present invention provides a method of screening an organism for the presence of a target nucleotide sequence in the organism, the method comprising obtaining a nucleic acid-containing sample from the organism and determining the presence of the target nucleotide sequence in the sample according to the method of the first aspect of the invention.<br><br>
In a third aspect, the present invention provides a nuclease-binding aptamer wherein binding of the aptamer to a nuclease inhibits the activity of the nuclease.<br><br>
In a fourth aspect, the present invention also provides a nuclease-aptamer complex comprising a nuclease bound to an aptamer.<br><br>
In a fifth aspect, the present invention comprises a linker nucleic acid comprising a first portion which can hybridise with a target nucleotide sequence, and a second portion which can hybridise with an aptamer in a nuclease-aptamer complex when the first portion of the linker nucleic acid is hybridised to the target nucleotide sequence, wherein hybridisation between the aptamer and the second portion of the linker nucleic acid modifies, reduces or eliminates binding between the aptamer and the nuclease in the nuclease-aptamer complex.<br><br>
In a sixth aspect, the present invention provides a kit for performing any of the methods of the present invention, the kit comprising one or more of an aptamer according to the third aspect of the invention; a nuclease-aptamer complex according to the fourth aspect of the invention; and/or a linker nucleic acid according to the fifth aspect of the invention.<br><br>
Throughout this specification, unless the context requires otherwise, the word “comprise”, or variations such as “comprises” or “comprising”, will be understood to imply the inclusion of a stated element or integer or group of elements or integers but not the exclusion of any other element or integer or group of elements or integers.<br><br>
Nucleotide and amino acid sequences are referred to herein by a sequence identifier number (SEQ ID NO:). The SEQ ID NOs: correspond numerically to the sequence identifiers in the sequence listing, eg.  1 (SEQ ID NO :1),  2 (SEQ ID NO : 2), etc. A summary of the sequence identifiers is provided in Table 2. A sequence listing is provided at the end of the specification.<br><br>
TABLE 1 – Summary of Sequence Identifiers<br>
Sequence Identifier	Sequence<br>
SEQ ID NO: 1	EcoRI-binding aptamer nucleotide sequence motif 1<br>
SEQ ID NO: 2	EcoRI-binding aptamer nucleotide sequence motif 2<br>
SEQ ID NO: 3	Aptamer 3_ADJ_L nucleotide sequence<br>
SEQ ID NO: 4	Aptamer 3_ADJ_L-binding nucleotide sequence<br>
SEQ ID NO: 5	CaMV 35S promoter targeting nucleotide sequence<br>
SEQ ID NO: 6	Mycobacterium tuberculosis targeting nucleotide sequence<br><br><br>
DESCRIPTION OF EXEMPLARY EMBODIMENTS<br><br>
It is to be understood that the following description is for the purpose of describing particular embodiments only and is not intended to be limiting with respect to the above description.<br><br>
In a first aspect, the present invention provides a method for detecting a target nucleotide sequence in a sample, the method comprising:<br>
providing a nuclease-aptamer complex, the complex comprising a nuclease bound to an aptamer, wherein binding of the aptamer to the nuclease inhibits the activity of the nuclease and wherein inhibition of the nuclease’s activity by the aptamer is directly or indirectly reduced or eliminated when the target nucleotide sequence is present in the sample; <br>
applying the nuclease-aptamer complex to the sample;<br>
detecting the activity of the nuclease, wherein increased nuclease activity is indicative of the presence of the target nucleotide sequence in the sample.<br><br>
The “sample” in which a target nucleotide sequence may be detected may be any sample that putatively contains the target nucleotide sequence. For example, the sample may be a biological sample including samples derived from an organism, a sample containing one or more cells, a blood sample, a plasma sample, a CSF fluid sample, an amniotic fluid sample and the like; an environmental sample such as a water, air or soil sample; a food or beverage sample; and the like. The samples contemplated herein may be used in a crude form, or the samples may be processed for use in accordance with the present invention. For example, the sample may have one or more nucleic acid extraction or purification steps performed thereon in order to purify or semi-purify any nucleic acids present in the sample. Methods for the extraction and purification of nucleic acids from a range of samples would be readily ascertained by one of skill in the art. In this regard, reference is made to The Nucleic Acid Protocols Handbook (Rapley Ed. Humana Press, 2000). <br><br>
As referred to herein, a “nuclease” should be understood as any enzyme that can cleave the sugar-phosphate backbone of a nucleic acid. As such, the term “nuclease” should be understood to encompass both endonucleases and exonucleases. Furthermore, the nucleases contemplated for use in accordance with the present invention may be deoxyribonucleases (which cleave DNA) or ribonucleases (which cleave RNA). Generally, the nuclease used in accordance with the present invention is a nuclease that, once inhibition by the aptamer has been reduced or eliminated, still cannot cleave or digest the target nucleotide sequence, but can cleave or digest another nucleotide sequence (such as a reporter nucleic acid – see later) such that the activity of the nuclease may be detected. Nucleases that may be used in accordance with the present invention include, for example, restriction endonucleases, nucleases that cleave at sequence mis-matches, S1 nuclease, T7 endonuclease I, T4 endonuclease VII, CEL I (a plant-specific extracellular glycoprotein that belongs to the S1 nuclease family), and ribonucleases such as RNases A &amp; H.<br><br>
However, in some particular embodiments of the invention (as set out later) the nuclease may be a restriction endonuclease. <br><br>
The term “aptamer”, as referred to herein, should be understood as a nucleic acid molecule, at least a portion of which is able to bind to another molecule. <br><br>
Nucleic acid aptamers are generally single-stranded nucleic acid molecules with complex secondary or tertiary structures (which as discussed later may include double-stranded portions or regions) that can specifically bind a target molecule with high affinity. When bound to enzymes, certain aptamers are able to reduce or inhibit their enzymatic activity. Generally, the aptamers contemplated by the present invention can bind at least to a nuclease, and thus alter the activity of the nuclease. Furthermore, the nuclease-binding aptamers contemplated by the present invention generally reduce or eliminate the activity of the nuclease when bound thereto. The aptamers of the present invention may also comprise a region which does not bind to the nuclease. This region, or the nuclease binding region itself, may also exhibit binding affinity toward another molecule such as a target nucleotide sequence or linker nucleic acid (as described later).<br><br>
The aptamers contemplated for use in accordance with the present invention may be any suitable nucleic acid or equivalent thereof. In this regard, the aptamers may comprise, for example, DNA, RNA, a nucleic acid mimic such as Peptide Nucleic Acid (PNA) or Locked Nucleic Acid (LNA), DNA or RNA comprising one or more modified nucleotides, and the like. “Modified” nucleotides include, for example, nucleotides having chemical modifications to any of the phosphate backbone, sugar moiety or base moiety of the nucleotide, tritylated bases and unusual bases such as inosine. The use of modified nucleotides may also affect the binding characteristics of the aptamer to the nuclease, for example as described in Latham et al. (Nucl Acids Res 22(14): 2817-2822, 1994).<br><br>
In some specific embodiments RNA aptamers may be used, since RNA can form secondary structures that DNA generally does not, such as pseudoknots and base triples. <br><br>
Nucleic acid aptamers may also be modified, for example to increase stability, in a number of ways including, for example:<br><br>
(i)	Synthesis of aptamers using L-nucleotides (the mirror image of natural nucleotides) so that they cannot be degraded by naturally occurring nucleases;<br>
(ii)	Incorporation of locked nucleic acid (LNA) and/or peptide nucleic acid (PNA) residues into the aptamer. LNAs and PNAs also increase stability of nucleic acid duplexes;<br>
(iii)	Other chemical modifications of ribonucleotides, such as 2’-amino- and 2’-fluoro-pyrimidine nucleotides or 2’-O-methyl nucleotides; and/or<br>
(iv)	Capping at the 3’ end with a deoxythymidine to increase resistance to exonuclease degradation.<br><br>
Nucleic acid aptamers that bind to, and inhibit the activity of, a particular protein (such as a nuclease) may be produced using methods known in the art. For example, in-vitro selection methods (eg. see Ellington and Szostak, Nature 346(6287): 818-22, 1990) and SELEX methods (eg. see Tuerk and Gold, Science 249(4968): 505–510, 1990) may be used. Further details relating to the production and selection of aptamers may also be found in the review of Osborne and Ellington (Chem Rev 97(2): 349-370, 1997).<br><br>
In light of the above, a “nuclease-aptamer complex” should be understood as a nuclease to which an aptamer is bound, or a nuclease and aptamer which may become bound under the conditions under which the method of the invention is performed, such that binding of the aptamer to the nuclease inhibits the activity of the nuclease and inhibition of the nuclease’s activity by the aptamer is directly or indirectly reduced or eliminated when the target nucleotide sequence is present in the sample.<br><br>
In some embodiments of the invention the presence of the target nucleotide sequence in the sample modifies, reduces or eliminates binding between the nuclease and the aptamer, thereby reducing or eliminating inhibition of the nuclease’s activity by the aptamer.<br><br>
As referred to herein modification, reduction or elimination of binding between the nuclease and the aptamer refers to any qualitative or quantitative change in the nature of the binding between the aptamer and the nuclease which brings about a reduction or elimination of inhibition of the nuclease’s activity by the aptamer. Such changes may arise as a result of binding between the aptamer and another molecule such as the target nucleotide sequence or a linker nucleic acid or binding between the nuclease-aptamer complex and another molecule such as the target nucleotide sequence or a linker nucleic acid.<br><br>
“Qualitative or quantitative changes” in the nature of the binding between the aptamer and the nuclease may include, for example: a change in the binding strength between the nuclease and the aptamer; a change in the interaction or binding sites on the nuclease and/or aptamer; dissociation of the nuclease-aptamer complex, optionally with binding of the aptamer to another molecule; a change in the conformation, secondary structure or tertiary structure of the aptamer or nuclease which reduces the level of inhibition of the nuclease by the aptamer; a change in the relative positions of the binding sites on the aptamer and/or nuclease; and the like.<br><br>
In one embodiment the method of the present invention may exploit hybridisation between the aptamer and the target nucleotide sequence in order to modify, reduce or eliminate binding between the nuclease and the aptamer. <br><br>
Thus, in one embodiment the method of the present invention may comprise the steps of:<br>
providing a nuclease-aptamer complex, the complex comprising a nuclease bound to an aptamer, wherein binding of the aptamer to the nuclease inhibits the activity of the nuclease and wherein the aptamer is able to hybridise to a target nucleotide sequence to modify, reduce or eliminate binding between the nuclease and the aptamer when the target nucleotide sequence is present in the sample;<br>
applying the nuclease-aptamer complex to the sample;<br>
allowing the target nucleotide sequence, if present in the sample, to hybridise with the aptamer; and<br>
detecting the activity of the nuclease wherein increased nuclease activity is indicative of the presence of the target nucleotide sequence in the sample.<br><br>
In this embodiment of the invention, the aptamer is able to hybridise to the target nucleotide sequence to modify, reduce or eliminate binding between the nuclease and the aptamer. Hybridisation between the target nucleotide sequence and the aptamer, together with concomitant modification, reduction or elimination of binding between the nuclease and aptamer, may occur under the initial conditions of the sample or, alternatively, the conditions may be modified to allow or promote binding between the aptamer and the target nucleotide sequence (if it is present in the sample). <br><br>
For example, binding between the aptamer and the target nucleotide sequence and/or binding between the aptamer and nuclease may be modulated by altering the salt concentration and/or temperature. In addition, destabilising agents such as formamide may also be used to modulate the binding of the aptamer to any of the nuclease, the target nucleotide sequence and/or a linker nucleic acid. <br><br>
In this embodiment of the invention, the aptamer should be designed or selected such that it comprises both a region which binds to the nuclease and a region which can hybridise, at least under some hybridisation conditions, to the target nucleotide sequence. These regions may be separated by one or more nucleotide residues, partially overlapping, completely overlapping or one region may be contained within the other. <br><br>
However, in another embodiment of the invention, the necessity to design or select the aptamer to contain both a nuclease-binding region and a target nucleotide sequence binding region may be eliminated via the use of a linker nucleic acid. <br><br>
Accordingly, in another embodiment, the method of the present invention comprises the steps of:<br>
providing a nuclease-aptamer complex, the complex comprising a nuclease bound to an aptamer, wherein binding of the aptamer to the nuclease inhibits the activity of the nuclease;<br>
providing a linker nucleic acid comprising a first portion which can hybridise with the target nucleotide sequence if it is present in the sample, and a second portion which can hybridise with the aptamer when the first portion of the linker nucleic acid is hybridised to the target nucleotide sequence, wherein hybridisation between the aptamer and the second portion of the linker nucleic acid modifies, reduces or eliminates binding between the aptamer and the nuclease;<br>
applying the nuclease-aptamer complex to the sample;<br>
applying the linker nucleic acid to the sample;<br>
allowing the first portion of the linker nucleic acid to hybridise with the target nucleotide sequence if it is present in the sample;<br>
allowing the second portion of the linker nucleic acid to hybridise with the aptamer when the first portion of the linker nucleic acid hybridises with the target nucleotide sequence; and<br>
detecting the activity of the nuclease, wherein increased nuclease activity in the sample is indicative of the presence of the target nucleotide sequence in the sample.<br><br>
In one embodiment, the linker nucleic acid comprises a stem loop structure and at least a portion of the first portion of the linker is comprised within a loop of the stem-loop structure and at least a portion of the second portion of the linker is comprised within a stem of the stem-loop structure.<br><br>
In one embodiment, the second portion of the linker (ie. the aptamer-binding portion) is generally of a length sufficient to enable denaturation or dissociation of the stem portion when the linker binds to the target nucleotide sequence. That is, hybridisation of the first portion of the linker (at least partially comprised within the loop region of the stem loop) generally effects denaturation or dissociation of the stem portion of the linker, thus exposing the aptamer-binding region(s) of the linker nucleic acid. In light of the above, the stem-portion of the stem loop structure generally comprises at least 5 nucleotides and may extend to about 50 nucleotides in length. However, the present invention should not be considered limited to any specific length of sequence and other sequence lengths may be used that provide the functionality described above. <br><br>
In another embodiment, the linker is designed such that the melting temperature of the first portion of the linker, when hybridised to the target, is higher than the melting temperature of the stem of the linker. In some embodiments, the melting temperature of the first portion of the linker when hybridised to the target is about 1°C, about 2°C, about 3°C, about 4°C, about 5°C, about 6°C, about 7°C, about 8°C, about 9°C, about 10°C, about 11°C, about 12°C, about 13°C, about 14°C, about 15°C, about 16°C, about 17°C, about 18°C, about 19°C or about 20°C higher than the melting temperature of the stem of the linker. In one specific embodiment, the melting temperature of the first portion of the linker when hybridised to the target is about 10°C higher than the melting temperature of the stem of the linker.<br><br>
In this embodiment, when the target nucleotide sequence is present in the sample, a heat or chemically denatured linker binds to the target in preference to reverting to its native stem/loop structure following heat or chemical denaturation followed by the introduction of renaturation conditions. <br><br>
The stem-portion of the stem-loop structure may be completely homologous, ie. no mismatches in the hybridising sequences. Alternatively, the stem portion may include one or more mis-matched base pairs. For example, mis-matches may be useful in promoting denaturation of the stem portion of the stem loop structure during the method of the present invention and/or to allow hybridisation between the aptamer and the stem portion. Features such as mismatches and/or overhangs may also be incorporated into the stem to provide an appropriate melting temperature difference to be obtained without the requirement for an excessively long target-complementary sequence.<br><br>
In further embodiments, the first portion may not be wholly within the loop, and may also be part of the stem. Similarly, the second portion may not necessarily be wholly contained within the stem portion, and may extend into the loop portion<br><br>
In further embodiments, the first and second portions of the linker may be separated by one or more nucleotide residues, partially overlapping, completely overlapping or one region may be contained within the other.<br><br>
The use of a linker nucleic acid in accordance with this embodiment of the invention also provides a significant advantage in that it removes the requirement to design or select the aptamer to have a target nucleotide sequence binding region. Rather, the target nucleotide sequence binding region can be incorporated into the first portion of the linker and the second portion of the linker may be designed to bind to the aptamer. This provides an advantage in that incorporation of an aptamer-binding nucleotide sequence into a linker nucleic acid is simpler and/or less time-consuming than having to design or select an aptamer to include both a nuclease-binding region and a target binding region. That is, the linker nucleic acid may be designed such that it binds to an aptamer that binds to a nuclease rather than having to select an aptamer on the basis of the ability to bind to both a nuclease and a target nucleotide sequence. Another significant advantage is that a single nuclease-aptamer complex may be applied to the detection of many different targets, simply by modifying the first portion of the linker.<br><br>
As set out above, the methods of the present invention involve detecting the activity of the nuclease, wherein increased nuclease activity is indicative of the presence of the target nucleotide sequence in the sample.<br><br>
The method used for detecting the activity of the nuclease may be any suitable method for the subject nuclease. For example, the activity of a DNase or RNase may be ascertained by observing degradation or cleavage of a DNA, RNA or DNA/RNA hybrid reporter nucleic acid, respectively. The reporter nucleic acid may be single-stranded or double-stranded, as appropriate for the activity of the nuclease. Cleavage of the reporter nucleic acid may be detected by any known method. For example, cleavage of a reporter nucleic acid into a lower molecular weight product may be determined by electrophoretic methods, staining methods, the release of a labelled nucleotide, cleavage of a fluorophore/quencher labelled nucleic acid to release a fluorophore, and the like. <br><br>
As set out above, in some specific embodiments, the nuclease used in accordance with the present invention is a restriction endonuclease. As referred to herein, a “restriction endonuclease” refers to any endonuclease that binds to double-stranded DNA at a specific nucleotide sequence and then, if both strands of the DNA lack appropriate modification at that sequence, cleaves the DNA either at the recognition sequence or at another site in the DNA molecule. A wide array of restriction endonucleases with different recognition sites and different cleavage sites would be readily ascertained by one of skill in the art. For example, a range of restriction endonucleases may be sourced from New England Biolabs (Ipswich, MA, USA).<br><br>
In the embodiments of the invention utilising a restriction endonuclease, the activity of the restriction endonuclease may be determined by the rate or extent of cleavage of a reporter nucleic acid which comprises at least a region of double stranded DNA.<br><br>
In one specific embodiment, a fluorophore is bound to the reporter nucleic acid and a quencher, which quenches the fluorescence of the fluorophore, is also bound to the reporter nucleic acid, wherein cleavage of the reporter nucleic acid by the nuclease reduces or eliminates the quenching of the fluorophore by the quencher.<br><br>
Exemplary fluorophores and quenchers would be readily ascertained by one of skill in the art. In this regard, reference is made to the review of Marras (Methods Mol Biol 335, 3-16, 2006).<br><br>
In one specific embodiment, the reporter nucleic acid comprising a fluorophore and quencher may comprise a molecular break light (MBL) as described by Biggins et al. (Proc Natl Acad Sci U S A. 97(25): 13537–13542, 2000).<br><br>
In another exemplary embodiment, a polypeptide is bound to the reporter nucleic acid and an immobilisable agent is bound to the reporter nucleic acid, wherein cleavage of the reporter nucleic acid releases the polypeptide from the immobilisable agent; such that after cleavage of the reporter nucleic acid and immobilisation of the immobilisable agent, the amount of non-immobilised polypeptide is indicative of the activity of the nuclease. <br><br>
A wide array of “immobilisable agents” would be readily ascertained by one of skill in the art and may include, for example:<br><br>
(i)	an antigen, which may be immobilised by contacting with an immobilised antibody that can bind the antigen;<br>
(ii)	an antibody, which may be immobilised by contacting with an immobilised antigen or anti-idiotypic antibody that can bind the antibody;<br>
(iii)	a polypeptide comprising a histidine tag, which may be immobilised by contacting an affinity medium comprising nickel or cobalt ions; <br>
(iv)	biotin, which may be immobilised by contacting with immobilised avidin or streptavidin;<br>
(v)	avidin or streptavidin, which may be immobilised by contacting with immobilised biotin; and/or<br>
(vi)	a magnetic or paramagnetic particle, which may be immobilised via a magnetic field. <br><br>
As set out above, some immobilisable agents may be immobilised by contacting the immobilisable agent with a binding partner that is itself immobilised. The immobilisation of the binding partner may be achieved using any means known in the art. For example, the binding partner of the immobilisable agent may be immobilised onto a surface of a culture vessel, tube or plate (which may have been pre-treated with an agent such as a silane), immobilised onto the surface of a bead or other particle, immobilised onto a column or other chromatography medium, or immobilised onto a membrane. <br><br>
A range of other immobilisable agents would also be readily ascertained by one of skill in the art, and the present invention should not be considered in any way limited to the immobilisable agents exemplified above.<br><br>
In one specific embodiment, the immobilisable agent is a magnetic bead and immobilisation of the immobilisable agent is effected by the application of a magnetic field to the sample.<br><br>
After immobilisation of the immobilisable agent, any polypeptide remaining “free” in the sample may be detected using any standard methods of protein detection, as are known in the art such as electrophoresis, immunochromatographic tests, including lateral flow strips, western blotting, mass spectroscopy, detection using a biosensor (for a review of biosensor-based detection of proteins in solution see Leca-Bouvier and Blum, Analytical Letters 38(10): 1491-1517, 2005). A range of exemplary protein detection methods may be found in Proteins and Proteomics: A Laboratory Manual (Simpson, CSHL Press, 2003). <br><br>
The use of a nuclease-aptamer complex in the methods of the present invention also allow the methods of the present invention to multiplexed, that is, the present invention also provides methods to simultaneously detect a plurality of target nucleotide sequences in a sample. <br><br>
For example, in the embodiments of the invention wherein the aptamer binds to the target nucleotide sequence, a method for simultaneously detecting a plurality of target nucleotide sequences in a sample may comprise providing a plurality of nuclease-aptamer complexes, wherein at least two of the nuclease-aptamer complexes comprise: (i) aptamers which hybridise to different target nucleotide sequences; and (ii) different nucleases, the activity of which may be separately detected.<br><br>
In the embodiments of the invention wherein the aptamer binds to a linker nucleic acid, a method for simultaneously detecting a plurality of target nucleotide sequences in a sample may comprise:<br>
providing a plurality of linker nucleic acids at least two of which comprise different first portions which can hybridise to different target nucleotide sequences and also comprise different second portions which can hybridise with different aptamers; and <br>
providing a plurality of nuclease-aptamer complexes, wherein at least two of the nuclease-aptamer complexes comprise aptamers which preferentially hybridise to the different second portions of the linker nucleic acids and comprise different nucleases, the activity of which may be separately detected.<br><br>
In another embodiment, a single nuclease aptamer complex may be used to detect the presence of one or more of a plurality of different targets in a sample. That is, in an embodiment of the invention wherein the aptamer binds to a linker nucleic acid, a method for simultaneously detecting a plurality of target nucleotide sequences in a sample may comprise:<br>
providing a plurality of linker nucleic acids at least two of which comprise different first portions which can hybridise to different target nucleotide sequences but wherein the linker nucleic acids comprise second portions which can hybridise to the same aptamer; and <br>
providing a nuclease-aptamer complex comprising an aptamer which can hybridise with the second portions of the linker nucleic acids; <br><br>
In the above embodiment, an increase in nuclease activity is indicative of the presence of one or more of the target nucleotide sequences in the sample.<br><br>
As set out above, some embodiments of the methods of the present invention for simultaneously detecting a plurality of target nucleotide sequences are predicated, in part, on the use of a plurality of nuclease-aptamer complexes wherein at least two of the nuclease-aptamer complexes comprise different nucleases, the activity of which may be separately detected.<br><br>
As referred to herein “different nucleases, the activity of which may be separately detected” refers to, for example, nucleases which act to cleave or digest distinguishable reporter nucleic acids. For example, one nuclease may cleave or digest RNA, while another may cleave or digest DNA. <br><br>
In another embodiment, a plurality of reporter nucleic acids are provided, at least two of which comprise different nucleotide sequences that represent recognition and/or cleavage sites for different restriction endonucleases. <br><br>
The cleavage of different reporter nucleic acids may be detected by, for example, electrophoretic methods, the release of a labelled nucleotide, cleavage of a fluorophore/quencher labelled nucleic acid to release a fluorophore, and the like, with the proviso that it is possible to distinguish the cleavage of one reporter nucleic acid from another. <br><br>
For example, in one embodiment the reporter nucleic acids for the different nucleases comprise fluorophores that can be separately detected when released from a quencher. <br><br>
In another embodiment the reporter nucleic acids for the different nucleases comprise polypeptides that can be separately detected when released from an immobilisable agent.<br><br>
Polypeptides “that can be separately detected when released from an immobilisable agent” include, for example, polypeptides of different molecular mass which may be resolved by electrophoresis or mass spectroscopy, polypeptides having different charge which may be resolved by isoelectric focusing, polypeptides with different antigenicity that may be resolved by differences in antibody-binding activity, polypeptides with different enzymic activity that may be resolved by one or more enzymic activity assays, polypeptides that have one or more distinct physical characteristics (such as fluorescence at a particular wavelength) that allows resolution of polypeptides having different physical characteristics, polypeptides that incorporate one or more distinct detectable labels such as a radiolabel, chemical label or fluorescent label, and the like.<br><br>
In a second aspect, the present invention provides a method of screening an organism for the presence of a target nucleotide sequence in the organism, the method comprising obtaining a nucleic acid sample from the organism and determining the presence of the target nucleotide sequence in the sample according to the method of the first aspect of the invention.<br><br>
The method of the second aspect of the invention may be used to detect any target nucleotide sequence in a nucleic acid sample derived from an organism. For example, the target nucleotide sequence may be a nucleotide sequence that is endogenous or native to the organism or, alternatively, the target nucleotide sequence may be an introduced or exogenous nucleotide sequence. As such, target nucleotide sequences that may be detected using the method of the second aspect of the invention include, for example, genomic nucleotide sequences, transgenes, allele nucleotide sequences, single nucleotide polymorphisms, mutant nucleotide sequences, transposon nucleotide sequences or viral nucleotide sequences.<br><br>
The method of the second aspect of the invention may be applied to the detection of a target nucleotide sequence in any organism. However, in one specific embodiment, the method of the second aspect of the invention may be used to detect a target nucleotide sequence in a nucleic acid sample derived from a plant. In the context of this embodiment of the invention, the nucleic acid sample derived from a plant should be understood to include a sample from a live plant (or part thereof such as an organ, tissue or cell) or a sample taken from a plant or part thereof which has been processed, such as a plant part processed for use as a food (eg. flours and the like). <br><br>
The detection of specific nucleotide sequences in plants has particular application in, for example, the diagnosis of plant diseases, detection of food contamination, taxonomic identification of plants or plant varieties and the detection of genetically modified plants (either environmentally or in plants processed for food).<br><br>
In one particular embodiment, the present invention has application for the detection of genetically modified plants in foodstuffs. In this regard, it has been recognised that at least one of the Cauliflower Mosaic Virus 35S promoter, the Agrobacterium tumefaciens nopaline synthase terminator and the 5-enolpyruvylshikimate-3-phosphate synthase (EPSPS) encoding nucleotide sequence are found in more that 95% of the presently available genetically modified crop plants. Thus, these sequences represent particularly valuable targets for the detection of genetically modified plant material.<br><br>
Thus, in one specific embodiment of the second aspect of the invention, the target nucleotide sequence comprises one or more of: a Cauliflower Mosaic Virus 35S (CaMV 35S) promoter sequence, an Agrobacterium tumefaciens nopaline synthase (nos) terminator sequence or a 5-enolpyruvylshikimate-3-phosphate synthase (EPSPS) encoding nucleotide sequence. <br><br>
In a third aspect, the present invention also provides a nuclease-binding aptamer. Generally, the nuclease-binding aptamers of the third aspect of the invention effect inhibition of the activity of a nuclease when bound to the nuclease.<br><br>
The aptamers of the present invention can be generated by known aptamer generation or selection methods as previously described. <br><br>
In some embodiments, the aptamers of the present invention can bind to and/or inhibit the activity of a restriction endonuclease and comprise a modified restriction endonuclease recognition sequence. The term “modified restriction endonuclease recognition sequence” should be understood as a nucleotide sequence within the aptamer that allows a restriction endonuclease to bind to the aptamer, but which is sufficiently different to the restriction endonuclease’s native recognition sequence that the aptamer is not itself a substrate for cleavage by the restriction endonuclease..<br><br>
In some embodiments, the modified restriction endonuclease recognition sequence may comprise a double stranded region of the aptamer that comprises a native recognition sequence for a restriction endonuclease with one or more of the following modifications:<br><br>
(i)	one or more nucleotide substitutions in one or both strands relative to the nucleotide sequence of the native recognition sequence;<br>
(ii)	one or more nucleotide pair mismatches in the modified recognition sequence; <br>
(iii)	one or more double-stranded insertions or deletions relative to the native recognition sequence; and/or<br>
(iv)	one or more insertions or deletions in one or both strands of the modified recognition sequence thus forming a loop structure comprising one or more unpaired nucleotides in one or both strands of the double stranded region.<br><br>
The modified restriction endonuclease recognition sequence may comprise one or more modifications relative to a native recognition sequence. For example, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10 or more modifications may be made depending on the nature and length of the native recognition sequence to be modified. However, in practice, the number of modifications would be limited to the number of modifications tolerated while the aptamer retains the ability to bind to the restriction endonuclease.<br><br>
In one specific embodiment, the modified restriction endonuclease recognition sequence comprises a single nucleotide substitution relative to an endonuclease’s native recognition sequence. In another specific embodiment, the modified restriction endonuclease recognition sequence comprises a single stranded deletion relative to a native endonucleases recognition sequence thus forming a loop structure comprising a single unpaired nucleotide residue in the opposite strand of the modified endonuclease recognition sequence. In a further embodiment, the single unpaired nucleotide residue in the loop structure is substituted relative to the native recognition sequence. <br><br>
In one embodiment, the nuclease-binding aptamer binds to EcoRI and inhibits the activity of EcoRI when bound thereto.<br><br>
In one embodiment, the EcoRI-binding aptamer of the present invention includes the nucleotide sequence motif: <br><br>
5’ – GAGTTC – 3’ (SEQ ID NO: 1)<br><br>
In a further embodiment, the EcoRI-binding aptamer of the present invention may also include the nucleotide sequence motif: <br><br>
5’ – HSGAGTTCWR – 3’ (SEQ ID NO: 2)<br><br>
wherein H may be any of A, T or C; S may be C or G; W may be A or T; and R may be A or G.<br><br>
In a yet further embodiment, the EcoRI-binding aptamer of the present invention includes a modified EcoRI recognition sequence as set out below:<br><br>
5’ – GAGTTC – 3’<br>
3’ – CT-AAG – 5’<br><br>
wherein the G residue shown in bold is an unpaired nucleotide residue forming a loop structure.<br><br>
In a yet further embodiment, the EcoRI-binding aptamer may comprise the nucleotide sequence set forth in SEQ ID NO: 3 or a variant thereof which retains the ability to bind to and inhibit the activity of EcoRI. Typically, the contemplated variants comprise the nucleotide sequence motifs set forth in SEQ ID NO: 1 and/or SEQ ID NO: 2 and/or comprise the modified EcoRI recognition sequence as set out above.<br><br>
In a fourth aspect, the present invention also provides a nuclease-aptamer complex comprising a nuclease bound to an aptamer. Generally, binding of the aptamer to the nuclease inhibits the activity of the nuclease in the nuclease-aptamer complex.<br><br>
In some embodiments, the nuclease-aptamer complex may comprise a restriction endonuclease bound to an aptamer. In these embodiments, the aptamer in the nuclease-aptamer complex may comprise a modified restriction endonuclease recognition sequence as described above.<br><br>
In a specific embodiment, the nuclease-aptamer complex comprises an EcoRI-aptamer complex. In this embodiment, the aptamer in the EcoRI-aptamer complex may be a EcoRI-binding aptamer as described above.<br><br>
As will be appreciated, the present invention also extends to methods for the detection of a target nucleotide sequence as described earlier wherein the nuclease-aptamer complex is a nuclease-aptamer complex as described with respect to the fourth aspect of the invention.<br><br>
In a fifth aspect, the present invention also comprises a linker nucleic acid comprising a first portion which can hybridise with a target nucleotide sequence, and a second portion which can hybridise with an aptamer in a nuclease-aptamer complex when the first portion of the linker nucleic acid is hybridised to the target nucleotide sequence, wherein hybridisation between the aptamer and the second portion of the linker nucleic acid modifies, reduces or eliminates binding between the aptamer and the nuclease in the nuclease-aptamer complex.<br><br>
In one embodiment, the linker nucleic acid comprises a stem loop structure and at least a portion of the first portion of the linker is comprised within a loop of the stem-loop structure and at least a portion of the second portion of the linker is comprised within a stem of the stem-loop structure.<br><br>
In another embodiment, the linker is designed such that the melting temperature of the first portion of the linker, when hybridised to the target, is higher than the melting temperature of the stem of the linker as hereinbefore described.<br><br>
In one specific embodiment, the linker comprises a second portion which can hybridise with an EcoRI-binding aptamer as previously described. In a further specific embodiment, the second portion of the linker, which can hybridise with an EcoRI-binding aptamer, comprises the nucleotide sequence set forth in SEQ ID NO: 4.<br><br>
The first portion of the linker may be adapted to bind to any target nucleotide sequence of interest. In one exemplary embodiment, the first portion of the linker molecule is adapted to hybridise to a region of the Cauliflower Mosaic Virus 35S promoter nucleotide sequence. In a more specific embodiment, the first portion of the linker molecule that is adapted to hybridise to a region of the Cauliflower Mosaic Virus 35S promoter nucleotide sequence comprises the nucleotide sequence set forth in SEQ ID NO: 5. <br><br>
In another exemplary embodiment, the first portion of the linker molecule is adapted to hybridise to a Mycobacterium tuberculosis derived nucleotide sequence. In a more specific embodiment, the first portion of the linker molecule that is adapted to hybridise to a Mycobacterium tuberculosis derived nucleotide sequence comprises the nucleotide sequence set forth in SEQ ID NO: 6. <br><br>
As will be appreciated, the present invention also extends to methods for the detection of a target nucleotide sequence as described earlier wherein the linker nucleic acid is a linker nucleic acid as described with respect to the fifth aspect of the invention.<br><br>
In a sixth aspect, the present invention provides a kit for performing any of the methods of the present invention, the kit comprising one or more of an aptamer according to the third aspect of the invention; a nuclease-aptamer complex according to the fourth aspect of the invention; and/or a linker nucleic acid according to the fifth aspect of the invention.<br><br>
Finally, reference is made to standard textbooks of molecular biology that contain methods for carrying out basic techniques encompassed by the present invention, including DNA restriction and ligation for the generation of the various genetic constructs described herein. See, for example, Maniatis et al., Molecular Cloning: A Laboratory Manual (Cold Spring Harbor Laboratory Press, New York, 1982) and Sambrook et al. (2000, supra).<br><br>
The present invention is further described by the following non-limiting examples:<br><br><br>
BRIEF DESCRIPTION OF THE FIGURES <br><br>
Figure 1 is a schematic representation of a method for detecting a target nucleotide sequence in a sample according to one embodiment of the present invention.<br><br>
Figure 2 is a schematic representation of a method for simultaneously detecting a plurality of target nucleotide sequences in a sample according to another embodiment of the present invention.<br><br>
Figure 3 shows example linker molecules and their targets. A. Arbitrary target-linker and target. The target complementary sequence is shaded blue. B. 35S-linker and target. The target complementary sequence is shaded green. C. Mycobacterium tuberculosis-linker and target.  The target complementary sequence is shaded yellow.  The aptamer binding sequence (trigger moiety) is shaded red in all detectors. The predicted secondary structures were obtained using the MFOLD programme (Zucker, 2003 http://frontend.bioinfo.rpi.edu/applications/mfold/cgi-bin/dna-form1.cgi).<br><br>
Figure 4 shows A. Aptamer 3_ADJ_L. Apart from removal of a portion of the 5’ and 3’ primer-binding regions, the aptamer 3_ADJ_L nucleotide sequence is unchanged from that of aptamer 3 identified in the SELEX process. The conserved region prevalent among strong inhibitors of EcoRI is shaded red. The binding site for the trigger moiety of the linker molecule is adjacent to the green dotted line. B. Conserved regions of aptamers that strongly inhibit EcoRI. This portion of the aptamer nucleotide sequence is hypothesised to be an element of the primary site mediating inhibition of EcoRI activity.<br><br>
Figure 5 shows an example of a linker transformed from an inactive to an active state in the presence of a target nucleic acid. In the linker’s inactive state, the trigger moiety is bound to its complementary stem partner and thus unable to bind to aptamer 3_ADJ_L. In its active state, the target-complementary sequence is bound to a target nucleic acid, and the trigger sequence is free to bind to aptamer 3_ADJ_L.<br><br>
Figure 6 shows an aptamer 3 gel-shift assay. The asymmetric PCR product derived from aptamer 3 was used.  The aptamer 3/EcoRI complex is evident by a high molecular weight band associated with reduced electrophoretic mobility.  There is no evidence for similar bands indicating aptamer 3 binding to either the restriction endonuclease EcoRV or bovine serum albumin (BSA).<br><br>
Figure 7 shows the EcoRI molecular break light (MBL). The predicted stem/loop hairpin secondary structure is shown here.  The oligonucleotide comprises a 5’ flourophore and a 3’ quencher.  The recognition sequence for the restriction endonuclease EcoRI is present within the molecule’s stem (shaded red).  Upon cleavage by EcoRI, the fluorophore and quencher melt apart and a detectable fluorescent signal is generated. <br><br>
Figure 8 shows the results of a biosensor assay for a Mycobacterium tuberculosis target. Normalised fluorescent output for each treatment is the mean of at least three replicate reactions.  The presence of the 300 nM target is clearly discernable from the 0 nM target and 300 nM non-target DNA negative controls after a time period of five minutes. <br><br>
Figure 9 shows the results of a biosensor assay for a CaMV 35S promoter target. Normalised fluorescent output for each treatment is the mean of four replicate reactions. The presence of the 300 nM target is clearly discernable from the 0 nM target and 300 nM non-target DNA negative controls after a time period of fifteen minutes. <br><br>
Figure 10 shows the results of a biosensor assay for an arbitrary target. Normalised fluorescent output for each treatment is the mean of four replicate reactions. The presence of the arbitrary target (300 nM) is clearly discernable from the 0 nM arbitrary target and 300 nM non-target DNA negative controls after a time period of ten minutes. <br><br>
Figure 11 shows the results of a semi-quantitative biosensor assay. Increasing target (Mycobacterium tuberculosis-derived oligonucleotide) concentration from 0 nM to 500 nM results in an increase in signal intensity (fluorescence units) at a given time period. Data are the normalised means of at least two replicate reactions.<br><br>
Figure 12 shows the results of a multiplexed biosensor assay.  The presence of either or both of two targets (35S and Mycobacterium tuberculosis) was detected simultaneously in a single reaction.  <br><br><br>
EXAMPLE 1<br>
Method for detecting a target nucleotide sequence<br><br>
A method according to one embodiment of the invention will be described with respect to Figure 1.<br><br>
Panel 1 of Figure 1 shows a target nucleotide sequence 100 and a linker nucleic acid 110 comprising a stem-loop structure. The loop-portion 112 of the stem-loop structure comprises a first nucleotide sequence 113 that can hybridise with the target nucleotide sequence 100, and the stem-portion 114 comprises a second nucleotide sequence 115 to which an aptamer can bind when the first nucleotide sequence 113 has hybridised with the target nucleotide sequence 100.<br><br>
Panel 2 shows hybridisation between the first nucleotide sequence 113 in the linker nucleic acid 110, and the target nucleotide sequence 100. This hybridisation causes the stem portion 114 of the stem-loop structure to dissociate and thus expose second nucleotide sequence 115 and make it available for binding with the aptamer 124 of a nuclease-aptamer complex 120.<br><br>
Panel 3 shows hybridisation between the aptamer 124 and the second nucleotide sequence 115. This hybridisation reduces or eliminates inhibition of the nuclease 122 by the aptamer 124. In the figure, this reduction or elimination of inhibition is shown as separation of the aptamer 124 and nuclease 122, in part as a conceptual aid. However, as hereinbefore described, a reduction or elimination of inhibition of the nuclease 122 by the aptamer 124 may or may not involve actual dissociation between the nuclease 122 and the aptamer 124.<br><br>
In panel 4, the activity of the nuclease 122, freed from inhibition by the aptamer 124, can be detected by cleavage of a reporter nucleic acid 130 by the nuclease 122. The reporter nucleic acid 130 comprises a nucleotide sequence 132 that is cleavable by the nuclease 122. The reporter nucleic acid 130 also comprises a bound fluorophore 134 and a bound quencher 136 that serves to quench the fluorescence of the fluorophore 134. <br><br>
As shown in panel 5, however, cleavage of the reporter nucleic acid 130 into a fluorophore-containing fragment 138 and a quencher containing fragment 140, reduces or eliminates quenching of the fluorophore 134 by the quencher 136. Thus, cleavage of the reporter nucleic acid 130 by the nuclease 122 leads to the detection of a fluorescent signal from fluorophore 134.<br><br><br>
EXAMPLE 2<br>
Method for simultaneously detecting a plurality of target nucleotide sequences<br><br>
A method according to another embodiment of the invention will be described with respect to Figure 2.<br><br>
Panel 1 of Figure 2 shows three different target nucleotide sequences 200/300/400, and three linker nucleic acids 210/310/410, each comprising a stem-loop structure. The loop-portions of the stem-loop structures each comprise a first nucleotide sequence 213/313/413 that can hybridise with one of the specific target nucleotide sequences 200/300/400. Specifically, first nucleotide sequence 213 can specifically hybridise with target nucleotide sequence 200, first nucleotide sequence 313 can specifically hybridise with target nucleotide sequence 300 and first nucleotide sequence 413 can specifically hybridise with target nucleotide sequence 400. The stem-portions of the linker nucleic acids each comprise a second nucleotide sequence 215/315/415 to which specific aptamers can specifically bind when the first nucleotide sequences 213/313/413 have hybridised with the target nucleotide sequences 200/300/400.<br><br>
Panel 2 shows hybridisation between the first nucleotide sequences 213/313/413 and the target nucleotide sequences 200/300/400. This hybridisation causes the stem portions of the stem-loop structures to dissociate and thus expose second nucleotide sequences 215/315/415 and make them available for binding with the aptamers 224/324/424, respectively, of nuclease-aptamer complexes 220/320/420. Each of the aptamers preferentially hybridise to the different second nucleotide sequences, specifically, each of aptamers 224/324/424 preferentially hybridise to each of second nucleotide sequences 215/315/415, respectively. Furthermore, each of the nuclease-aptamer complexes 220/320/420 comprise a different nuclease 222/322/422 the activity of which may be separately detected. Specifically, in this embodiment each of nucleases 222/322/422 are restriction endonucleases recognising a different cleavage site.<br><br>
Panel 3 shows hybridisation between the aptamers 224/324/424 and the second nucleotide sequences 215/315/415. This hybridisation reduces or eliminates inhibition of the nucleases 222/322/422 by the aptamers 224/324/424. As in figure 1, this reduction or elimination of inhibition is shown as separation of the aptamers 224/324/424 and nucleases 222/322/422, in part as a conceptual aid. However, as hereinbefore described, a reduction or elimination of inhibition of the nucleases 222/322/422 by the aptamers 224/324/424 may or may not involve actual dissociation between the nucleases 222/322/422 and the aptamers 224/324/424.<br><br>
In panel 4, the activity of each of the nucleases 222/322/422, can be detected by the cleavage of different reporter nucleic acids. Each of the reporter nucleic acids comprises a different nucleotide sequence, each of which is specifically cleavable by one of the nucleases 222/322/422, respectively. <br><br>
Each of the reporter nucleic acids also each comprise a different bound fluorophore 234/334/434, and a bound quencher 236/336/436 that serves to quench the fluorescence of each fluorophore 234/334/434. However, cleavage of each of the reporter nucleic acids into fluorophore-containing fragments 238/338/438 and quencher containing fragments 240/340/440, reduces or eliminates quenching of the fluorophores 234/334/434 by the quenchers 236/336/436. Each of the different fluorophores 234/334/434 have distinct emission spectra, such that once released from the action of the quenchers 236/336/436, signals from each of the fluorophores 234/334/434 may be detected separately.<br><br>
Thus, cleavage of one or more of reporter nucleic acids by the nucleases 222/322/432, respectively, leads to the detection of an individually detectable fluorescent signal from one or more of fluorophores 234/334/434.<br><br><br><br><br>
EXAMPLE 3<br>
Biosensing using the EcoRI-Aptamer 3_ADJ_L complex<br><br>
In one embodiment, the method of the present invention utilizes a target sequence-specific detection mechanism comprising a linker molecule that switches from an ‘inactive’ to an ‘active’ conformation upon hybridisation to a target nucleic acid. In its ‘active’ conformation, the linker is able to bind to an aptamer in complex with a restriction endonuclease, in this case: EcoRI, and release the endonuclease from aptamer-mediated inhibition.  In its ‘inactive’ conformation, the linker is unable to, or has a significantly reduced ability to, interact with the endonuclease/aptamer complex.  When released from inhibition, the nuclease molecule is able to cleave a signaling molecule, thus generating a detectable signal. The system is semi-quantitative, with the signal intensity proportional over a discrete range to the quantity of target nucleic acid molecules in solution.<br><br>
To demonstrate that the system is applicable to real-world nucleic acid detection situations, synthetic targets were used that represented nucleotide sequences contained within the Cauliflower Mosaic Virus 35S promoter, often indicative in plants of genetic modification, and the genome of Mycobacterium tuberculosis, which causes serious infectious disease world-wide.<br><br>
This embodiment of the biosensor system is modular, with three primary components: a target-detecting linker, a restriction endonuclease/aptamer complex and a molecular break light signaling molecule.<br><br>
The target-detection module comprises an oligonucleotide linker that folds into a stable stem/loop hairpin secondary structure, with the loop portion complementary to the target sequence and one strand of the double-stranded stem portion, termed the ‘trigger’ moiety, complementary to the hypothesised EcoRI binding region of aptamer 3_ADJ_L (Figures 3 &amp; 4).  <br><br>
The linker is designed such that the melting temperature of the target-complementary portion of the molecule, when hybridised to the target, is approximately 10°C higher than the melting temperature of the stem structure containing the trigger moiety.  This ensures that when the target nucleic acid is present in solution, the linker binds to the target in preference to reverting to its native stem/loop structure following heat denaturation and cooling.  Accordingly, the stem contains a mismatch and the trigger is overhanging, thus allowing the appropriate melting temperature difference to be obtained without the requirement for an excessively long target-complementary sequence (figure 5).<br><br>
Numerous unique aptamers that bound strongly and specifically to EcoRI (e.g. figure 6) were identified using a SELEX protocol.  A majority of the aptamers analysed inhibited the activity of EcoRI to a significant degree.  The ideal aptamer for use in the biosensor aptamer/EcoRI complex should not only inhibit EcoRI strongly and specifically, but have secondary and tertiary structures that are amenable to disruption via the hybridisation of the short trigger moiety of the linker molecule.   <br><br>
Many of the EcoRI-inhibiting aptamers identified through SELEX contained a highly-conserved nucleotide sequence hypothesised to bind to the active site of the EcoRI enzyme, possibly due to the conserved region’s sequence similarity to the EcoRI recognition site (figure 4b).  As the predicted aptamer secondary structure in the area of the conserved region contained a mis-match, the site was particularly amenable to denaturation through strand displacement via the binding of a perfectly complementary oligonucleotide of sufficient length.  Aptamer 3 was selected as an example of such an aptamer.  Based upon the aptamer 3 nucleotide sequence, a modified version with a portion of the SELEX primer binding sites removed (aptamer 3_ADJ_L) was commercially synthesised and formed the basis of the aptamer/EcoRI module for the biosensor.<br><br>
Upon the linker molecule binding to a target nucleic acid and conforming to its active state, its exposed trigger moiety was able to bind to aptamer 3_ADJ_L.  Subsequently, aptamer 3_ADJ_L changed conformation in such a way as to no longer inhibit the activity of EcoRI.  <br><br>
To detect active EcoRI activity and thus the presence of a target nucleic acid, a modified version of a molecular break light (MBL) molecule (as described in Biggins et al., 2000, supra) served as a signaling module (figure 7). The MBL is an oligonucleotide, 5’-labelled with a fluorophore (e.g. FAM) and 3’-labelled with a quencher (e.g. fluorescein). It forms a hairpin stem/loop secondary structure at the temperatures at which EcoRI is active. Due to the close proximity of the fluorophore to the quencher, Förster resonance energy transfer (FRET) ensures that the excited fluorophore’s emission is strongly quenched. Within its stem structure is an EcoRI recognition sequence. Upon cleavage by EcoRI, the fluorophore immediately melts away from the quencher and its fluorescent signal is no longer quenched, and thus easily detected. The intensity of the fluorescent signal is proportional to the quantity of break light molecules digested and, therefore, the amount of active EcoRI in solution.<br><br>
The full biosensor assay is carried out as two sequential reactions. Following addition of a DNA sample to a solution containing the linker, the reaction is heat-denatured and cooled to at least 37°C. The prepared target/linker sample is added to a buffered solution containing the appropriate concentrations of the aptamer/EcoRI complex and MBL, and immediately analysed. Fluorescence measurements were carried out on a RotorGene RG-3000 Real-Time PCR machine operating at a constant temperature of 30°C or 37°C.<br><br>
Figures 8, 9, and 10 demonstrate biosensor reaction data where the target is a synthetic Mycobacterium tuberculosis target; a synthetic 35S target; and a non-secondary structure-producing arbitrary target, respectively. Controls to demonstrate that the resultant signal is a product of the presence of the target nucleic acid and not an artefact include: a 0-target control; a non-target control, where another non-target oligonucleotide is used to demonstrate specificity; a non-linker control to demonstrate that interaction between the target and linker is specifically releasing EcoRI from inhibition; and a non-target, non-linker control, used to demonstrate the baseline aptamer 3_ADJ_L-inhibited EcoRI activity. The biosensor in its current form is able to specifically detect a nucleic acid target in the low picomole range in a very low assay volume (20 µl).  <br><br>
The primary determinants of the sensitivity of the biosensor system are the detection limits of the instrument used to perform the assay; the time over which the assay is performed; and the degree of background signal present in the assay. Low levels of target-generated signal can easily be rendered undetectable by background fluorescence associated with incomplete aptamer-mediated inhibition of EcoRI and/or the release of EcoRI molecules from inhibition by linkers in their ‘inactive’ state. <br><br>
Background fluorescence associated with incompletely inactive linkers is mediated by the stability of the linker molecule’s secondary structure. The stem/loop structure of the linker designed against an arbitrary target is less stable at biosensor reaction temperatures (30°C-37°C) than those of the 35S and Mycobacterium tuberculosis linkers, and a portion of the linker stem melts and exposes the trigger to aptamer 3_ADJ_L hybridisation even in the absence of the target nucleic acid. However, a significant difference in fluorescence intensity between the presence and absence of target nucleic acid is produced after 15 to 20 minutes at the 30°C reaction temperature.  Due to the more stable secondary structures within the target nucleic acid binding sequences of the Mycobacterium tuberculosis and 35S linker molecules, these linkers remain inactive at 30°C and either insignificant or markedly reduced linker-mediated background is present. In each case the presence of 6 picomoles of target is clearly discernable after 15 minutes reaction time.<br><br>
The second source of background present in all of the biosensor examples is due to incomplete inhibition of EcoRI by aptamer 3_ADJ_L. This is primarily due to a system of simply mixing EcoRI and excess aptamer 3_ADJ_L and binding for 45 minutes at 37°C before use in a biosensor reaction.  Significant amounts of unbound aptamer 3_ADJ_L (which reduces biosensor sensitivity) and unbound EcoRI (which produces background signal) remain. Removal of unbound aptamer/EcoRI and potentially covalently bonding the aptamer to EcoRI is expected to significantly decrease or eliminate this form of background.  <br><br>
As the intensity of fluorescence is proportional to the amount of target in solution over a certain discrete range, the system is semi-quantitative (figure 11). A reduction in background signal will further increase the range in which the system is semi-quantitative.  <br><br>
The use of multiple restriction endonucleases allows for multiplexed target detection to take place concurrently. Under optimal reaction conditions, restriction endonucleases are highly sequence specific, even when multiple enzymes are present in the same digestion reaction. Similarly, multiple unique linkers, aptamer/enzyme complexes and signaling molecules in a single reaction would be expected to specifically and accurately detect the presence of multiple unique target molecules.<br><br><br>
Materials and Methods<br><br>
(i)	Selection of EcoRI-binding aptamers<br>
DNA aptamers are selected from a random pool using the SELEX (systematic evolution of ligands by exponential enrichment) procedure (see Tuerk and Gold, 1990, supra).<br><br>
To select aptamers that bind to, and inhibit, the restriction endonuclease EcoRI, a DNA oligonucleotide library (5’-CAAGGCCTCTCCTGATCCGA-40N-GTCGGGAGCTGAAGCTGCTT-3’) comprising a random 40-mer flanked by 20 base-pair primer binding sites was used. Sequences of oligonucleotide PCR primers for library amplification were 5’-CAAGGCCTCTCCTGATCCGA-3’ and 5’-biotin-AAGCAGCTTCAGCTCCCGAC-3’. For the 14 rounds of SELEX, a nitrocellulose filter-based binding method was employed (Pileur et al., Nucleic Acids Research 31: 5776-5788, 2003). The nitrocellulose filter was able to bind EcoRI and any DNA also bound to EcoRI. A majority of unbound DNA passed freely through the membrane, with a small amount of background retained. Bound DNA was eluted from the filter, amplified, and used in the subsequent round of SELEX.<br><br>
For each round of SELEX, 4 µg of oligonucleotide library buffered in a 500 µl solution of 50 mM NaCl, 10 mM Tris-HCl pH 8 and 10 mM MgCl2 was used. EcoRI is highly active under these buffering conditions. To allow individual sequences within the oligonucleotide library to conform to their natural secondary structure, the buffered pool was heated to 94°C for two minutes and immediately placed on ice for 10 minutes. Initially, 4 µl of concentrated EcoRI (New England Biolabs, 100 000 U/ml) was added to the buffered pool,  followed by a 20 minute incubation at room temperature. Reduced amounts of EcoRI were added in each subsequent SELEX step.  A negative-control solution with no added EcoRI was used in SELEX rounds seven and nine to relatively quantitate recovered EcoRI-bound DNA versus background filter-bound DNA. These mixtures were vacuum-filtered through pre-washed alkali-treated 25 mm 0.45 µM nitrocellulose HAWP filters (Millipore). Following a wash with 3 ml of buffering solution, the bound contents were eluted by crushing the filter in a 400 µl 7 M urea, 50 mM EDTA, 400 mM sodium acetate solution pre-heated to 80°C, and incubating at room temperature for 10 minutes. DNA was precipitated from the supernatant with 800 µl 100% ethanol and 2 µl GlycoBlue (Ambion). The DNA pellets were washed with 1.5 ml 70% ethanol and air-dried prior to re-suspension in 150 µl of 10 mM Tris-HCl pH 8 solution.<br><br>
To obtain sufficient DNA for the next round of SELEX, recovered DNA was amplified using PCR. Each 100 µl PCR comprised 1 mM of each primer (Sigma-Proligo), 200 µM dNTPs (Bioline), 1.5 mM MgCl2 (Bioline), 0.5 U Immolase DNA polymerase (Bioline), 1 × Immolase DNA polymerase buffer (Bioline) and 10 µl of DNA solution. Cycling conditions for the initial eight rounds were as follows; 7 minutes at 95°C, followed by 30 cycles of 10 seconds at 94°C, 30 seconds at 60°C and 20 seconds at 72°C. A five minute extension at 72°C completed the PCR.  Following the ninth round of SELEX, adjusted PCR cycling conditions of 7 minutes at 95°C, followed by 15 cycles of 35 seconds at 94°C, 30 seconds at 60°C and 20 seconds at 72°C, were used to prevent accumulation of non-specific PCR artefacts. For each SELEX round a total of 12 100 µl PCRs were performed. All reaction products were subsequently purified with Qiagen MinElute columns, resuspended in 5 µl 10 mM Tris-HCl pH 8 solution and pooled into a total volume of 60 µl. DNA concentrations in these solutions ranged from 500 to 800 ng/µl.<br><br>
To create single-stranded DNA oligonucleotides from amplified double-stranded DNA, 30 µg of biotinylated PCR product was bound to M-270 streptavidin-coated Dynabeads® (Invitrogen) in 1 M NaCl binding buffer. After washing in binding buffer, the Dynabead®-PCR product complexes were incubated in a 60 µl volume of a 150 mM NaOH solution to break hydrogen bonds binding complementary DNA strands together. The supernatant containing free single-stranded DNA was removed and neutralised with 40 µl of 200 mM HCl and 7.6 µl of 1 M Tris-HCl. The neutralised solution was diluted with an additional 400 µl of 10 mM Tris-HCl and used in the next round of SELEX. <br><br>
14 rounds of SELEX were performed in total before PCR products were cloned and sequenced.<br><br>
(ii)	Cloning of EcoRI binding aptamers<br>
The PCR products produced in SELEX round 14 were cloned into the pGEM-T Easy vector (Promega) according to the manufacturer’s instructions.  The plasmids were then transformed into E. coli heat shock-competent DH5a cells and grown under ampicillin selection overnight at 37°C. The pGEM-T Easy system allowed for white/blue selection of colonies containing the inserted PCR product.<br><br>
(iii)	Colony and asymmetric PCRs<br>
To rapidly assay individual aptamers’ binding and inhibition efficiency, a colony PCR followed by an asymmetric PCR strategy was used.  PCR templates were prepared from colonies by removing a portion of individual colonies with a sterile pipette tip followed by suspension in 100 µl dH2O.   These solutions were denatured at 95°C for 2 minutes and cooled on ice for 10 minutes.  20 µl PCRs were carried out with 1 mM non-biotinylated SELEX forward and reverse primers (Sigma-Proligo), 0.2 mM dNTPs (Bioline), 1x reaction buffer (Bioline), 5% dimethyl sulfoxide (Sigma), 5 U of Immolase DNA polymerase (Bioline) and 2 µl of the prepared PCR template.   PCR cycling conditions were the same as those used in the latter stages of the SELEX process.<br><br>
Upon completion of the colony PCR, reactions were purified using a QIAGEN MinElute PCR purification kit according to the manufacturer’s instructions in an effort to remove excess primer and reaction components.  PCR products were resuspended in 11 µl of elution buffer.<br><br>
An asymmetric PCR was performed with the purified PCR products from the colony PCR.  Only the sense SELEX forward primer was added which, when extended in the PCR reaction, produces a single-stranded DNA product identical in sequence to the original aptamer.  Other PCR reaction conditions were the same as the colony PCR, apart from the omission of dimethyl sulfoxide.  Total reaction volume was 100 µl, with 10µl of the purified PCR product from the colony PCR used.  Following completion of the PCR, products were heated to 95°C for 2 minutes and cooled on ice for 10 minutes to allow the single-stranded aptamers to form their correct secondary structure.  Due to incomplete removal of excess primer in the PCR purification reaction, some exponential as well as linear amplification occurred.  Accordingly, after 21 cycles of PCR approximately 50% of the product was single-stranded and 50% was double-stranded.    <br><br>
(iv)	Gel-shift assay<br>
Gel shift assays were performed with individual aptamers generated using the asymmetric PCR protocol.  10 µl of each PCR was added to 300 units of EcoRI (New-England Biolabs) in 1x NEBuffer 1.  For selectivity control reactions, the restriction endonuclease EcoRV (New-England Biolabs) or bovine serum albumin (New-England Biolabs) were added instead of EcoRI.  Aptamer/ligand binding was carried out for 20 minutes at room temperature.  Samples were loaded onto a 10% non-denaturing polyacrylamide gel and electrophoresed at 150V for 1 hour, followed by ethidium bromide staining and viewing on a UV transilluminator (Syngene). <br><br>
(v)	Aptamer inhibition test<br>
To determine the EcoRI-inhibition capacity of individual aptamers, 1µl of asymmetric PCR product was added to 4 units of EcoRI in 1x buffer.  Aptamer/EcoRI binding took place at 37°C for 45 minutes.  4 µl of EcoRI molecular break light (330 nM) was added to each reaction (to a final concentration of 66 nM).  Total individual reaction volume was 20 µl.  Reactions were then immediately analysed using a Corbett RotorGene RG-3000 operating at a constant temperature of 37°C, with readings taken on the FAM channel at intervals of one minute.  <br><br>
(vi)	Sequencing of plasmids<br>
Inserts from individual plasmids containing aptamer sequences that demonstrated strong EcoRI inhibition were sequenced in the forward and reverse directions using M13 primers and Big Dye Terminator V3.1 reaction mixture (Applied Biosystems) according to the manufacturer’s instructions.<br><br>
(vii)	Biosensor assay<br>
For the biosensor assays, a synthesised version of the strong inhibitor aptamer 3 was used (Sigma-Proligo).  The aptamer, aptamer 3_ADJ_L, was identical in sequence to aptamer 3 except for the removal of a portion of both primer binding sites.  The total length of aptamer 3_ADJ_L was 49 nucleotides (5’- CCGACGAGCAAGTAGCTCCAAGACGAGTTCAACCCCAGAATCAGGTCGG-3’; SEQ ID NO: 3).<br><br>
Before carrying out the assay, aptamer 3_ADJ_L was bound to EcoRI at a ratio of approximately 40 ng of aptamer to 4 units of EcoRI (New-England Biolabs) in a solution containing 1 µg/µl BSA (New-England Biolabs).  Binding took place at 37°C for 45 minutes.  <br><br>
The linker and target or control DNA at concentrations required for the particular experiment were heated to 95°C for 2 minutes and then slowly cooled to 4°C at the rate of 0.1°C/second.  The cooled linker/target sample was mixed with the EcoRI (4 U)/aptamer 3_ADJ_L (40 ng) complex and MBL (66 nM) in 1.5x EcoRI NEbuffer.  Reactions were analysed in a Corbett RotorGene RG-3000 running at a constant temperature of 30°C.  Fluorescence was measured on the FAM channel at intervals of one minute.<br><br>
(viii)	Semi-quantitative biosensor assay<br>
The semi-quantitative biosensor assay was carried out as per the biosensor assay except for the addition of higher concentrations of EcoRI (8 U)/aptamer 3_ADJ_L (80 ng) and MBL (122 nM) to the assay.  The Mycobacterium tuberculosis linker and target were used.  Target concentrations in the final reaction were 0 nM, 100 nM, 200 nM, 300 nM, 400 nM and 500 nM.<br><br>
(ix)	Multiplexed biosensor assay<br>
The multiplexed biosensor assay was carried out as per the biosensor assay except for the addition of higher concentrations of EcoRI (7 U)/aptamer 3_ADJ_L (53 ng) and MBL (100 nM) to the assay.  The Mycobacterium tuberculosis linker and the 35S linker were present in the target binding reaction.  Five picomoles of the 35S and/or Mycobacterium tuberculosis target nucleic acids were added to each reaction at a final concentration of 250 nM.   <br><br>
Those skilled in the art will appreciate that the invention described herein is susceptible to variations and modifications other than those specifically described. It is to be understood that the invention includes all such variations and modifications. The invention also includes all of the steps, features, compositions and compounds referred to, or indicated in this specification, individually or collectively, and any and all combinations of any two or more of the steps or features.<br><br><br>
SEQUENCE LISTING<br><br>  Lifeprint Australia Pty. Ltd.<br><br>  Biosensor<br><br>  Biosensor<br><br>  6     <br><br>  PatentIn version 3.3<br><br>  1<br>  6<br>  DNA<br>  Artificial<br><br><br>  Aptamer sequence motif<br><br>  1<br>
gagttc                                                               6<br><br><br>  2<br>  10<br>  DNA<br>  Artificial<br><br><br>  Aptamer sequence motif<br><br><br><br>  variation<br>  (1)..(1)<br>  A may be replaced with T or C<br><br><br>  variation<br>  (2)..(2)<br>  C may be replaced with G<br><br><br>  variation<br>  (9)..(9)<br>  A may be replaced with T<br><br><br>  variation<br>  (10)..(10)<br>  A may be replaced with G<br><br>  2<br>
acgagttcaa                                                          10<br><br><br>  3<br>  49<br>  DNA<br>  Artificial<br><br><br>  Aptamer nucleotide sequence<br><br>  3<br>
ccgacgagca agtagctcca agacgagttc aaccccagaa tcaggtcgg               49<br><br><br>  4<br>  15<br>  DNA<br>  Artificial<br><br><br>  Linker nucleic acid sequence motif<br><br>  4<br>
gttgaactgg tcttg                                                    15<br><br><br>  5<br>  27<br>  DNA<br>  Artificial<br><br><br>  Linker nucleic acid sequence motif<br><br>  5<br>
cgcacaatcc cactatcctt cgcaaga                                       27<br><br><br>  6<br>  27<br>  DNA<br>  Artificial<br><br><br>  Linker nucleic acid sequence motif<br><br>  6<br>
cagtttccca ggcttatccc gaagtgc                                       27<br><br><br><br>
 We claim:-<br><br>
1.	A method for detecting a target nucleotide sequence in a sample, the method comprising:<br>
providing a nuclease-aptamer complex, the complex comprising a nuclease bound to an aptamer, wherein binding of the aptamer to the nuclease inhibits the activity of the nuclease and wherein inhibition of the nuclease’s activity by the aptamer is directly or indirectly reduced or eliminated when the target nucleotide sequence is present in the sample; <br>
applying the nuclease-aptamer complex to the sample;<br>
detecting the activity of the nuclease, wherein increased nuclease activity is indicative of the presence of the target nucleotide sequence in the sample.<br><br>
2.	A method according to claim 1 wherein the presence of the target nucleotide sequence in the sample modifies, reduces or eliminates binding between the nuclease and the aptamer, thereby reducing or eliminating inhibition of the nuclease’s activity by the aptamer.<br><br>
3.	A method according to claim 1 or 2 comprising the steps of:<br>
providing a nuclease-aptamer complex, the complex comprising a nuclease bound to an aptamer, wherein binding of the aptamer to the nuclease inhibits the activity of the nuclease and wherein the aptamer is able to hybridise to the target nucleotide sequence to modify, reduce or eliminate binding between the nuclease and the aptamer when the target nucleotide sequence is present in the sample;<br>
applying the nuclease-aptamer complex to the sample;<br>
allowing the target nucleotide sequence, if present in the sample, to hybridise with the aptamer; and<br>
detecting the activity of the nuclease wherein increased nuclease activity is indicative of the presence of the target nucleotide sequence in the sample.<br><br>
4.	A method according to any one of claims 1 to 3 wherein the method comprises a reaction for simultaneously detecting a plurality of target nucleotide sequences in a sample, and the method comprises providing a plurality of nuclease-aptamer complexes, wherein at least two of the nuclease-aptamer complexes comprise: (i) aptamers which hybridise to different target nucleotide sequences; and (ii) different nucleases, the activity of which may be separately detected.<br><br>
5.	A method according to claim 1 or 2 comprising the steps of:<br>
providing a nuclease-aptamer complex, the complex comprising a nuclease bound to an aptamer, wherein binding of the aptamer to the nuclease inhibits the activity of the nuclease;<br>
providing a linker nucleic acid comprising a first portion which can hybridise with the target nucleotide sequence if it is present in the sample, and a second portion which can hybridise with the aptamer when the first portion of the linker nucleic acid is hybridised to the target nucleotide sequence, wherein hybridisation between the aptamer and the second portion of the linker nucleic acid modifies, reduces or eliminates binding between the aptamer and the nuclease;<br>
applying the nuclease-aptamer complex to the sample;<br>
applying the linker nucleic acid to the sample;<br>
allowing the first portion of the linker nucleic acid to hybridise with the target nucleotide sequence if it is present in the sample;<br>
allowing the second portion of the linker nucleic acid to hybridise with the aptamer when the first portion of the linker nucleic acid hybridises with the target nucleotide sequence; and<br>
detecting the activity of the nuclease, wherein increased nuclease activity in the sample is indicative of the presence of the target nucleotide sequence in the sample.<br><br>
6.	A method according to claim 5 wherein the linker nucleic acid comprises a stem loop structure and at least a portion of the first portion of the linker is comprised within a loop of the stem-loop structure and at least a portion of the second portion of the linker is comprised within a stem of the stem-loop structure.<br><br>
7.	A method according to claim 5 or 6 wherein the method comprises a reaction for simultaneously detecting a plurality of target nucleotide sequences in a sample, and the method comprises:<br>
providing a plurality of linker nucleic acids at least two of which comprise different first portions which can hybridise to different target nucleotide sequences and also comprise different second portions which can hybridise with different aptamers; and <br>
providing a plurality of nuclease-aptamer complexes, wherein at least two of the nuclease-aptamer complexes comprise aptamers which preferentially hybridise to the different second portions of the linker nucleic acids and comprise different nucleases, the activity of which may be separately detected.<br><br>
8.	A method according to claim 5 or 6 wherein the method comprises a method for simultaneously detecting a plurality of target nucleotide sequences in a sample, and the method comprises:<br>
providing a plurality of linker nucleic acids at least two of which comprise different first portions which can hybridise to different target nucleotide sequences but wherein the linker nucleic acids comprise second portions which can hybridise with the same aptamer; and <br>
providing a nuclease-aptamer complex comprising an aptamer which can hybridise with the second portions of the linker nucleic acids.<br><br>
9.	A method according to any one of claims 1 to 8 wherein the activity of the nuclease is determined by the rate or extent of cleavage of a reporter nucleic acid.<br><br>
10.	A method according to claim 9 wherein the nuclease is a restriction endonuclease.<br><br>
11.	A method according to claim 9 or 10 wherein a fluorophore is bound to the reporter nucleic acid and a quencher, which quenches the fluorescence of the fluorophore, is also bound to the reporter nucleic acid, wherein cleavage of the reporter nucleic acid by the nuclease reduces or eliminates the quenching of the fluorophore by the quencher.<br><br>
12.	A method according to claim 9 or 10 wherein a polypeptide is bound to the reporter nucleic acid and an immobilisable agent is bound to the reporter nucleic acid, wherein cleavage of the reporter nucleic acid releases the polypeptide from the immobilisable agent such that after immobilisation of the immobilisable agent, the amount of non-immobilised polypeptide is indicative of the activity of the nuclease. <br><br>
13.	A method according to claim 12 wherein the immobilisable agent is a magnetic bead and immobilisation of the immobilisable agent is effected by the application of a magnetic field.<br><br>
14.	A method according to any one of claims 9 to 13 as appended to claim 4 or claim 7, wherein a plurality of DNA reporter nucleic acids are provided, at least two of which comprise cleavage sites for different restriction endonucleases. <br><br>
15.	A method according to claim 14 as appended to claim 11 wherein the at least two reporter nucleic acids that comprise cleavage sites for different restriction endonucleases also comprise fluorophores that can be separately detected when released from the quencher. <br><br>
16.	A method according to claim 14 as appended to claim 12 wherein the at least two reporter nucleic acids that comprise cleavage sites for different restriction endonucleases also comprise polypeptides that can be separately detected when released from the immobilisable agent.<br><br>
17.	A method of screening an organism for the presence of a target nucleotide sequence, the method comprising obtaining a nucleic acid sample from the organism and determining the presence of the target nucleotide sequence in the sample according to the method of any one of claims 1 to 16.<br><br>
18.	A method according to claim 17 wherein the target nucleotide sequence is a non-native nucleotide sequence in the organism.<br><br>
19.	A method according to claim 18 wherein the target nucleotide sequence is an introduced nucleotide sequence.<br><br>
20.	A method according to claim 17 wherein the target nucleotide sequence is a genomic nucleotide sequence, an allele nucleotide sequence, a mutant nucleotide sequence, a single nucleotide polymorphism, a transposon nucleotide sequence or a viral nucleotide sequence.<br><br>
21.	A method according to any one of claims 17 to 20 wherein the organism is a plant.<br><br>
22.	A method according to claim 21 as appended to claim 19 wherein the target nucleotide sequence comprises one or more of: a Cauliflower Mosaic Virus 35S (CaMV 35S) promoter sequence, an Agrobacterium tumefaciens nopaline synthase (nos) terminator sequence or a 5-enolpyruvylshikimate-3-phosphate synthase (EPSPS) encoding nucleotide sequence.<br><br>
23.	A nuclease-binding aptamer wherein binding of the aptamer to a nuclease inhibits the activity of the nuclease.<br><br>
24.	An aptamer according to claim 23 wherein the nuclease is a restriction endonuclease.<br><br>
25.	An aptamer according to claim 24 wherein the aptamer comprises a modified restriction endonuclease recognition sequence. <br><br>
26.	An aptamer according to any one of claims 23 to 25 wherein the aptamer binds to EcoRI and inhibits the activity of EcoRI when bound thereto.<br><br>
27.	An aptamer according to claim 26 wherein the aptamer comprises the nucleotide sequence motif set forth in SEQ ID NO: 1 and/or the nucleotide sequence motif set forth in SEQ ID NO: 2.<br><br>
28.	An aptamer according to claim 26 or 27 comprising a modified EcoRI recognition sequence as set out below:<br><br>
5’ – GAGTTC – 3’<br>
3’ – CT-AAG – 5’<br><br>
wherein the G residue shown in bold is an unpaired nucleotide residue forming a loop structure.<br><br>
29.	An aptamer according to any one of claims 23 to 28 wherein the aptamer comprises the nucleotide sequence set forth in SEQ ID NO: 3; or a variant of said aptamer which retains the ability to bind to and inhibit the activity of EcoRI. <br><br>
30.	A nuclease-aptamer complex comprising a nuclease bound to an aptamer, wherein binding of the aptamer to the nuclease inhibits the activity of the nuclease in the nuclease-aptamer complex.<br><br>
31.	A nuclease-aptamer complex according to claim 30 wherein the nuclease is a restriction endonuclease. <br><br>
32.	A nuclease-aptamer complex according to claim 31 wherein the nuclease is EcoRI.<br><br>
33.	A nuclease-aptamer complex according to any one of claims 30 to 32 wherein the aptamer is an aptamer according to any one of claims 23 to 29.<br><br>
34.	A method according to any one of claims 1 to 22 wherein the nuclease-aptamer complex is a nuclease-aptamer complex according to any one of claims 30 to 33. <br><br>
35.	A linker nucleic acid comprising a first portion which can hybridise with a target nucleotide sequence, and a second portion which can hybridise with an aptamer in a nuclease-aptamer complex when the first portion of the linker nucleic acid is hybridised to the target nucleotide sequence, wherein hybridisation between the aptamer and the second portion of the linker nucleic acid modifies, reduces or eliminates binding between the aptamer and the nuclease in the nuclease-aptamer complex.<br><br>
36.	A linker nucleic acid according to claim 35 wherein the linker nucleic acid comprises a stem loop structure and at least a portion of the first portion of the linker is comprised within a loop of the stem-loop structure and at least a portion of the second portion of the linker is comprised within a stem of the stem-loop structure.<br><br>
37.	A linker nucleic acid according to claim 36 wherein the second portion can hybridise with an aptamer according to any one of claims 23 to 29 when the first portion of the linker is hybridised to a target nucleotide sequence. <br><br>
38.	A linker nucleic acid according to claim 36 wherein the second portion of the linker comprises the nucleotide sequence set forth in SEQ ID NO: 4.<br><br>
39.	A linker nucleic acid according to any one of claims 36 to 38 wherein the first portion of the linker molecule is adapted to hybridise to a region of the Cauliflower Mosaic Virus 35S promoter nucleotide sequence. <br><br>
40.	A linker nucleic acid according to claim 39 wherein the first portion of the linker nucleic acid comprises the nucleotide sequence set forth in SEQ ID NO: 5. <br><br>
41.	A linker nucleic acid according to any one of claims 36 to 38 wherein the first portion of the linker molecule is adapted to hybridise to a Mycobacterium tuberculosis derived nucleotide sequence. <br><br>
42.	A linker nucleic acid according to claim 41 wherein the first portion of the linker nucleic acid comprises the nucleotide sequence set forth in SEQ ID NO: 6. <br><br>
43.	A method according to any one of claims 5 to 22 wherein the linker nucleic acid comprises a linker nucleic acid according to any one of claims 35 to 42. <br><br>
44.	A kit for performing the method of any one of claims 1 to 22, 34 or 43, the kit comprising one or more of an aptamer according to any one of claims 23 to 29; a nuclease-aptamer complex according to any one of claims 30 to 33; and/or a linker nucleic acid according to any one of claims 35 to 42.<br>
45.	A Methods For Detecting A Target Nucleotide Sequence In A Sample Utilising A Nuclease-Aptamer Complex is claimed substantially as herein described with forgoing description &amp; drawings.<br><br><br>
Dated this 6th  day of October 2009.</td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTg2My1NVU1OUC0yMDA5LUFTU0lHTk1FTlQoMjItMi0yMDEyKS5wZGY=" target="_blank" style="word-wrap:break-word;">1863-MUMNP-2009-ASSIGNMENT(22-2-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTg2My1NVU1OUC0yMDA5LUNMQUlNUyhBTUVOREVEKS0oMjAtMTAtMjAxNCkucGRm" target="_blank" style="word-wrap:break-word;">1863-MUMNP-2009-CLAIMS(AMENDED)-(20-10-2014).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTg2My1NVU1OUC0yMDA5LUNMQUlNUyhBTUVOREVEKS0oMjctMTItMjAxMykucGRm" target="_blank" style="word-wrap:break-word;">1863-MUMNP-2009-CLAIMS(AMENDED)-(27-12-2013).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTg2My1NVU1OUC0yMDA5LUNMQUlNUyhNQVJLRUQgQ09QWSktKDIwLTEwLTIwMTQpLnBkZg==" target="_blank" style="word-wrap:break-word;">1863-MUMNP-2009-CLAIMS(MARKED COPY)-(20-10-2014).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTg2My1NVU1OUC0yMDA5LUNPUlJFU1BPTkRFTkNFKDEzLTEtMjAxMCkucGRm" target="_blank" style="word-wrap:break-word;">1863-MUMNP-2009-CORRESPONDENCE(13-1-2010).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTg2My1NVU1OUC0yMDA5LUNPUlJFU1BPTkRFTkNFKDIyLTItMjAxMikucGRm" target="_blank" style="word-wrap:break-word;">1863-MUMNP-2009-CORRESPONDENCE(22-2-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTg2My1NVU1OUC0yMDA5LUNPUlJFU1BPTkRFTkNFKDMxLTMtMjAxMSkucGRm" target="_blank" style="word-wrap:break-word;">1863-MUMNP-2009-CORRESPONDENCE(31-3-2011).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTg2My1NVU1OUC0yMDA5LUNPUlJFU1BPTkRFTkNFKDUtMy0yMDEyKS5wZGY=" target="_blank" style="word-wrap:break-word;">1863-MUMNP-2009-CORRESPONDENCE(5-3-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTg2My1NVU1OUC0yMDA5LUNPUlJFU1BPTkRFTkNFKDUtNC0yMDEwKS5wZGY=" target="_blank" style="word-wrap:break-word;">1863-MUMNP-2009-CORRESPONDENCE(5-4-2010).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTg2My1NVU1OUC0yMDA5LUNPUlJFU1BPTkRFTkNFKDUtNi0yMDE0KS5wZGY=" target="_blank" style="word-wrap:break-word;">1863-MUMNP-2009-CORRESPONDENCE(5-6-2014).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTg2My1NVU1OUC0yMDA5LURFQ0xBUkFUSU9OKDEzLTEtMjAxMCkucGRm" target="_blank" style="word-wrap:break-word;">1863-MUMNP-2009-DECLARATION(13-1-2010).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTg2My1NVU1OUC0yMDA5LURSQVdJTkcoMjctMTItMjAxMykucGRm" target="_blank" style="word-wrap:break-word;">1863-MUMNP-2009-DRAWING(27-12-2013).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTg2My1NVU1OUC0yMDA5LUZPUk0gMSgxMy0xLTIwMTApLnBkZg==" target="_blank" style="word-wrap:break-word;">1863-MUMNP-2009-FORM 1(13-1-2010).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTg2My1NVU1OUC0yMDA5LUZPUk0gMSgyMi0yLTIwMTIpLnBkZg==" target="_blank" style="word-wrap:break-word;">1863-MUMNP-2009-FORM 1(22-2-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTg2My1NVU1OUC0yMDA5LUZPUk0gMSgyNy0xMi0yMDEzKS5wZGY=" target="_blank" style="word-wrap:break-word;">1863-MUMNP-2009-FORM 1(27-12-2013).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTg2My1NVU1OUC0yMDA5LUZPUk0gMTMoMjItMi0yMDEyKS5wZGY=" target="_blank" style="word-wrap:break-word;">1863-MUMNP-2009-FORM 13(22-2-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTg2My1NVU1OUC0yMDA5LUZPUk0gMTgoMzEtMy0yMDExKS5wZGY=" target="_blank" style="word-wrap:break-word;">1863-MUMNP-2009-FORM 18(31-3-2011).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTg2My1NVU1OUC0yMDA5LUZPUk0gMihUSVRMRSBQQUdFKS0oMjItMi0yMDEyKS5wZGY=" target="_blank" style="word-wrap:break-word;">1863-MUMNP-2009-FORM 2(TITLE PAGE)-(22-2-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTg2My1NVU1OUC0yMDA5LUZPUk0gMjYoMTMtMS0yMDEwKS5wZGY=" target="_blank" style="word-wrap:break-word;">1863-MUMNP-2009-FORM 26(13-1-2010).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTg2My1NVU1OUC0yMDA5LUZPUk0gMjYoNS0zLTIwMTIpLnBkZg==" target="_blank" style="word-wrap:break-word;">1863-MUMNP-2009-FORM 26(5-3-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTg2My1NVU1OUC0yMDA5LUZPUk0gMygxMy0xLTIwMTApLnBkZg==" target="_blank" style="word-wrap:break-word;">1863-MUMNP-2009-FORM 3(13-1-2010).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTg2My1NVU1OUC0yMDA5LUZPUk0gMygyMC0xMC0yMDE0KS5wZGY=" target="_blank" style="word-wrap:break-word;">1863-MUMNP-2009-FORM 3(20-10-2014).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTg2My1NVU1OUC0yMDA5LUZPUk0gMyg1LTQtMjAxMCkucGRm" target="_blank" style="word-wrap:break-word;">1863-MUMNP-2009-FORM 3(5-4-2010).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTg2My1NVU1OUC0yMDA5LUZPUk0gMyg5LTEyLTIwMTMpLnBkZg==" target="_blank" style="word-wrap:break-word;">1863-MUMNP-2009-FORM 3(9-12-2013).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTg2My1NVU1OUC0yMDA5LUZPUk0gNSgxMy0xLTIwMTApLnBkZg==" target="_blank" style="word-wrap:break-word;">1863-MUMNP-2009-FORM 5(13-1-2010).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTg2My1NVU1OUC0yMDA5LUZPUk0gNigyMi0yLTIwMTIpLnBkZg==" target="_blank" style="word-wrap:break-word;">1863-MUMNP-2009-FORM 6(22-2-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTg2My1NVU1OUC0yMDA5LUZPUk0gUENULUlCLTM3Myg5LTEyLTIwMTMpLnBkZg==" target="_blank" style="word-wrap:break-word;">1863-MUMNP-2009-FORM PCT-IB-373(9-12-2013).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTg2My1NVU1OUC0yMDA5LUZPUk0gUENULUlTQS0yMzcoOS0xMi0yMDEzKS5wZGY=" target="_blank" style="word-wrap:break-word;">1863-MUMNP-2009-FORM PCT-ISA-237(9-12-2013).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTg2My1NVU1OUC0yMDA5LU1BUktFRCBDT1BZKDI3LTEyLTIwMTMpLnBkZg==" target="_blank" style="word-wrap:break-word;">1863-MUMNP-2009-MARKED COPY(27-12-2013).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTg2My1NVU1OUC0yMDA5LU9USEVSIERPQ1VNRU5UKDIwLTEwLTIwMTQpLnBkZg==" target="_blank" style="word-wrap:break-word;">1863-MUMNP-2009-OTHER DOCUMENT(20-10-2014).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTg2My1NVU1OUC0yMDA5LU9USEVSIERPQ1VNRU5UKDIyLTItMjAxMikucGRm" target="_blank" style="word-wrap:break-word;">1863-MUMNP-2009-OTHER DOCUMENT(22-2-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTg2My1NVU1OUC0yMDA5LU9USEVSIERPQ1VNRU5UKDktMTItMjAxMykucGRm" target="_blank" style="word-wrap:break-word;">1863-MUMNP-2009-OTHER DOCUMENT(9-12-2013).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTg2My1NVU1OUC0yMDA5LVJFUExZIFRPIEVYQU1JTkFUSU9OIFJFUE9SVCgyMC0xMC0yMDE0KS5wZGY=" target="_blank" style="word-wrap:break-word;">1863-MUMNP-2009-REPLY TO EXAMINATION REPORT(20-10-2014).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTg2My1NVU1OUC0yMDA5LVJFUExZIFRPIEVYQU1JTkFUSU9OIFJFUE9SVCgyNy0xMi0yMDEzKS5wZGY=" target="_blank" style="word-wrap:break-word;">1863-MUMNP-2009-REPLY TO EXAMINATION REPORT(27-12-2013).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTg2My1NVU1OUC0yMDA5LVJFUExZIFRPIEVYQU1JTkFUSU9OIFJFUE9SVCg5LTEyLTIwMTMpLnBkZg==" target="_blank" style="word-wrap:break-word;">1863-MUMNP-2009-REPLY TO EXAMINATION REPORT(9-12-2013).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTg2My1NVU1OUC0yMDA5LVNQRUNJRklDQVRJT04oQU1FTkRFRCktKDI3LTEyLTIwMTMpLnBkZg==" target="_blank" style="word-wrap:break-word;">1863-MUMNP-2009-SPECIFICATION(AMENDED)-(27-12-2013).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=RHJhd2luZ3MuZG9j" target="_blank" style="word-wrap:break-word;">Drawings.doc</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Rm9ybS0xLmRvYw==" target="_blank" style="word-wrap:break-word;">Form-1.doc</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Rm9ybS0zLmRvYw==" target="_blank" style="word-wrap:break-word;">Form-3.doc</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Rm9ybS01LmRvYw==" target="_blank" style="word-wrap:break-word;">Form-5.doc</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=UHJpb3JpdHkgRG9jdW1lbnQucGRm" target="_blank" style="word-wrap:break-word;">Priority Document.pdf</a></p>
		<br>
		<div class="pull-left">
			<a href="264178-a-composition-an-encapsulated-sweetener-composition-and-method-of-preparation-thereof.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="264180-a-method-and-apparatus-for-monitoring-and-controlling-the-stability-of-a-burner-of-a-fired-heater.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>264179</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>1863/MUMNP/2009</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>50/2014</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>12-Dec-2014</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>11-Dec-2014</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>07-Oct-2009</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>AUSTRALIAN CENTRE FOR PLANT FUNCTIONAL GENOMICS PTY LTD.</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td>Hartley Grove, Urrbrae, South Australia 5064, Australia</td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>MILLIGAN  Andrew  Simon</td>
											<td>35 Anzac Ridge Road  Bridgewater  South Australia 5155  Australia.</td>
										</tr>
										<tr>
											<td>2</td>
											<td>FLETCHER  Stephen  John</td>
											<td>Lot 73 Second Street  Mylor  South Australia 5153  Australia.</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>C12Q1/44,C07H21/04,C12N9/16,G01N33/53</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>PCT/AU2008/000496</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td>2008-04-04</td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td>2007901826</td>
									<td>2007-04-05</td>
								    <td>Australia</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/264179-methods-for-detecting-a-target-nucleotide-sequence-in-a-sample-utilising-a-nuclease-aptamer-complex by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 00:06:10 GMT -->
</html>
