<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/264264-a-method-of-identifying-a-stable-vh-and-or-a-vl-antibody-fragment by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 00:31:35 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 264264:&quot;A METHOD OF IDENTIFYING A STABLE VH AND/OR A VL ANTIBODY FRAGMENT&quot;</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">&quot;A METHOD OF IDENTIFYING A STABLE VH AND/OR A VL ANTIBODY FRAGMENT&quot;</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>Polypeptides with desirable biophysical properties such as solubility, stability, high expression, monomericity, binding specificity or non-aggregation, including monomeric human VHs and VLs, are identified using a high throughput method for screening polypeptides, comprising the steps of obtaining a phage display library, allowing infection of a bacterial lawn by the library phage, and identifying phage which form larger than average plaques on the bacterial lawn. Sequences of monomeric human VHs and VLs are identified, which may be useful for immunotherapy or as diagnostic agents. Multimer complexes of human VHs and VLs are also identified. The VHs and VLs identified may be used to create further libraries for identifying additional polypeptides. Further, The VHs and VLs may be subjected to DNA shuffling to select for improved biophysical properties.</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td>Method for Isolation of Soluble Polypeptldes Field of the Invention<br>
This invention relates to the isolation, identification and manipulation of polypeptides, especially monomeric human antibody fragments.<br>
Background of the Invention<br>
Antibodies in vertebrates are typically composed of paired heavy (H) and light (L) chains. The first domain of the combined H and L chains, the VH and VL, are more variable in sequence, and this is the portion of the antibody that recognizes and binds to the antigen. The VH and VL domains recognize the antigen as a pair.<br>
The immune repertoire of camelidae (camels, dromedaries and llamas) is unique in that it possesses unusual types of antibodies referred to as heavy-chain antibodies (Hamers, Casterman C. et al., 1993). These antibodies lack light chains and thus their combining sites consist of one domain, termed VHH.<br>
Recombinant VHH single-domain antibodies (sdAbs) provide several advantages over single-chain Fv (scFv) fragments derived from conventional four-chain antibodies. While sdAbs are comparable to their scFv counterparts in terms of affinity, they outperform scFvs in terms of solubility, stability, resistance to aggregation, refoldability, expression yield, and ease of DNA manipulation, library construction and 3-D structural determinations. Many of the aforementioned properties of VHH sdAbs are desired in applications involving antibodies.<br>
However, the non-human nature of VHHs limits their use in human immunotherapy due to immunogenicity. In this respect, human VH and VL sdAbs are ideal candidates for immunotherapy applications because they are expected to be least immunogenic.<br>
Human VHs and VLs, however, are by and large prone to aggregation, a characteristic common to VHS and VLs derived from conventional antibodies (Davies, J. et a/., 1994;Tanha, J. et al., 2001;Ward, E. S. ef a/., 1989). Thus, attempts have been made to obtain monomer human VHS and VLS suitable for antibody applications. Such VHS and VLS have also displayed other useful properties typical of VHHS such as high expression yield, high readability and resistance to aggregation. Synthetic libraries built on these VHS and VLS as library scaffolds might serve as a promising source of therapeutic proteins.<br>
Camelization as well as (lamination which involves incorporating key solubility residues from camel and llama VnHs, respectively, into human VHS or VLS have been employed to generate monomeric human VHS and VLs. Synthetic sdAb libraries constructed based on these VHS and VLS and generated by CDR randomization were shown to be functional in terms of yielding binders to various antigens (Davies, J. et a/., 1995;Tanha, J. et a/., 2001).<br>
In another approach, fully human monomeric VHs and Vt&amp; were isolated from human synthetic VH and VL libraries without resorting to engineering of the sort mentioned above. In one experiment a monomeric human VH, was discovered when a human VH library was panned against hen egg lysozyme (Jespers, L. et a/., 2004b). More recently, a selection method based on reversible unfolding and affinity criteria yielded many monomeric VHs from synthetic human VH libraries (Jespers, L. et al., 2004a). This finding underlined the fact that an appropriate selection method is key to efficient capturing of rare monomer human VHS with desirable biophysical properties.<br>
Objects of the Invention<br>
A first object of the invention is to provide a high throughput screening method for identifying polypeptides, especially antibody fragments, with<br>
improved biophysical properties, including solubility, high expression, and/or stability (such as high refolding after thermal denaturation, high resistance to chemical denaturant, and high resistance to proteases, in particular gastrointestinal proteases such as trypsin).<br>
A second object of the invention is to provide a high throughput screening method for identifying monomeric human VHS and Vus.<br>
A third object of the invention is to identify, isolate and characterize monomeric human VHS and VLs.<br>
A fourth object of the invention is to construct and characterize multimers of antibody fragments, especially monomeric human VHS and VLS<br>
A fifth object of the invention is to construct display libraries from polypeptides, especially antibody fragments, and most especially monomeric human VHs and VLS<br>
A sixth object of the invention is to provide a DNA shuffling method for producing polypeptides, especially antibody fragments, and most especially monomeric human VHS and VLS with improved biophysical properties.<br>
Summary of the Invention<br>
A method is provided for isolating polypeptides, preferably antibody fragments, and most preferably human VHS and VLS with desirable biophysical properties (solubility, stability, high expression, monomericity, non-aggregation, binding specificity). The method includes the steps of obtaining a phage display library capable of expressing a variety of polypeptide sequences, allowing infection of a bacterial lawn by the library phage, and identifying phage which form larger than average plaques on the bacterial lawn. The phage are then isolated, and steps are taken to sequence or otherwise characterize the polypeptide sequences.<br>
The invention also provides for polypeptides, especially monomeric human VHS and VLs, identified by the above method, which may be useful for immunotherapy, and/or as diagnostic or detection agents. The monomeric human VHs and VLs may also be combined to form dimers, trimers, pentamers or other multimers, which may be useful for immunotherapy and/or as diagnostic or detection agents.<br>
The polypeptides identified by the above method, including human VHS and VLS, can be manipulated by methods such as DNA shuffling to select for improved biophysical properties such as solubility, stability, monomericity, high expressibility, binding specificity and human origin.<br>
The polypeptides identified by the above method, including human VHS and VLS, may also be used to generate further display libraries, which can then in turn be used to isolate further polypeptides by the above method.<br>
In a first aspect, the present invention provides a method of identifying target polypeptides , comprising a) obtaining a phage display library capable of expressing a variety of polypeptide sequences, b) allowing infection of a bacterial lawn by the library phage and c) identifying phage which form larger than average plaques on the bacterial lawn.<br>
In a second aspect, the present invention provides polypeptide having an amino acid sequence selected from the group consisting of: SEQ ID NO:8-54<br>
In a third aspect, the present invention provides a VH antibody fragment comprising at least one amino acid sequence selected from the group consisting of: SEQ ID NO:8-22.<br>
In a fourth aspect, the present invention provides a VL antibody fragment comprising at least one amino acid sequence selected from the group consisting of: SEQ ID NO:23-54.<br>
Ih a fifth aspect, the present invention provides A method for producing polypeptides with desirable biophysical properties, comprising the steps of a) providing at least one first nucleic acid sequence that encodes an antibody fragment as claimed in claim 41,42,44,45,47,48, 59 or 70 or that encodes a polypeptide sequence as claimed in claim 24,27,37 or 39, and having a first desirable property; b) providing at least one second nucleic acid sequence that encodes an antibody fragment having a second desirable property; c) cutting the at least one first and at least one second nucleic acid sequences into random fragments; d) reassembling the random fragments; e) expressing the random fragments; and f) screening the expressed random fragments for the first and second desirable properties.<br>
Detailed Description of the Drawings<br>
Figure legends<br>
Figure 1. A pictorial representation of selected example results: The contrast in plaque size between phages displaying a soluble VH (HVHP428) and those displaying an insoluble one (BT32/A6). The photo depicts a part of the bacterial lawn agar plate which was magnified to enhance plaque visualization. Although the plate contained an equal number of each of the two plaque types, the photo essentially contains the large, HVHP428 plaques. The majority of the BT32/A6 plaques were too small to produce clear, well-defined images in the photo. The plaques marked by arrows, thus, represent a minor proportion of BT32/A6 phages which were large enough to be visible in this image. Asterisks marks representative plaque sizes for HVHP428 phages. The identities of plaques were determined by DNA sequencing.<br>
Figure 2. Amino acid sequence of the human VHs selected based on affinity for protein A and plaque size. The dots in the sequence entries indicate amino acid identity with HVHP2M10 or HVHP44. Dashes are included for sequence alignment. Residues at the key solubility positions and residue 57T which associates with VHs/VHHs with protein A binding property are in bold. The Kabat numbering system is used. The total "frequency" value is 114.<br>
CDR = complementarity determining region; FR = framework region; gin seq = germline sequence<br>
Figure 3. Aggregation tendencies of the human VMS. Gel filtration chromatograms comparing the oligomerization state of a human VH isolated in this study (HVHP428) to that of a llama VHH (H11C7) and a typical human VH (BT32/A6). The peak eluting last in each chromatogram corresponds to monomeric VH. The dimeric H11C7 peak Is marked by an arrow. B, One-dimensional 1H NMR spectra of HVHP414 at 800 MHz (i), HVHP423 at 500 MHz (ii) and HVHP428 at 800 MHz (iii). The spectra in the left panel are scaled up by a factor of two to enable better viewing of low-intensity signals.<br>
Figure 4. Stability of the human VMS in terms of their resistance to trypsin at 37°C and integrity following long incubation at 37°C. A, SDS-PAGE comparing the mobilities of the untreated and trypsin-treated HVHP414 VH at 15, 30 and 60 min relative to a 21 kDa marker. HVHP414-cMyc denotes HVHP414 VH lacking the c-Myc. B, Molecular mass profiles obtained by mass spectrometry of untreated and trypsin-treated (60 min) HVHP414 VH. The mass spectrometry profile of the treated VH is superimposed onto that for the untreated one to provide a better visual comparison. The experimental molecular mass of the untreated VH is 14,967.6 Da, which is essentially identical to the expected molecular mass, 14,967.7 Da. The observed molecular mass of the trypsin-treated VH (13,368.5 Da) indicates loss of 13 amino acids at the C-terminus by cleavage at K (Lys) in the c-Myc tag to give an expected molecular mass of 13,368.0 Da. The trypsin cleavage site is shown by a vertical arrow above the amino acids sequence of HVHP414. C, Gel filtration chromatograms comparing the oligomerization state of the 37°C-treated HVHP420 VH (upper profile) to that of untreated VH (lower profile). The chromatograms were shifted vertically because they were indistinguishable when superimposed. The major and minor peaks in each chromatogram correspond to monomeric and dimeric VHs, respectively. The dimeric VH constitutes 3% of the total protein. The inset shows the sensorgram overlays for the binding of 37°C-treated HVHP420 to protein A at<br>
various concentrations.  The VHs used for temperature stability studies were from stocks which had already been at 4°C for several months.<br>
Figure 5. Sensogram overlays showing the binding of native (thick lines) and refolded (thin lines) HVHP423 to immobilized protein A at 75,100,150 and 200 nM concentrations. KDn and  KD0rg/were calculated from respective sensograms and used to determine RE as described below.<br>
Figure 6. Amino acid sequences of the human VLs selected based on affinity<br>
for protein L and plaque size. The dots in the sequence entries indicate<br>
amino acid identity with HVLP333. Dashes are included for sequence<br>
alignment.	See	the	V	BASE	(http://vbase.mrc-<br>
cpe.cam.ac.uk/index.php?module=pagemaster&amp;PAGE_user_op=view_page&amp; PAGE_id=7&amp;MMN_position=5:5) for sequence numbering and CDR designation. L6, A27, 12, L16, O2/O12, A30 and 1b are V germline designation. J germline designations are in the brackets. NF, not found.<br>
Figure 7. Size exclusion chromatograms of human VL domains. In A, the were applied at a concentration of 0.6 mg/ml. In B, the Vus were applied at their highest concentration available: HVLP342, 1.0 mg/ml; HVLP3103, 5.9 mg/ml; HVLP335, 4.9 mg/ml; HVLP351, 0.89 mg/ml. "rand — represent aggregate and monomer peaks, respectively. The aggregates elute in the exclusion volume. The peak marked by an arrow in the HVLP342 panel (B) is the carry over from a previous run.<br>
Figure 8. Sensorgram overlays showing the binding of VLS to immobilized protein L at concentrations of 0.2, 0.5, 0.75, 1, 2, 3, 5 and 10µM (HVLP389, HVLP351 and HVLP364); 1, 2, 3. 5, 7.5 and 10 nM (HVLP342); 0.2, 0.5, 1, 2, 3, 5 and 10 nM (HVLP335); 0.2, 0.5, 1, 1.5, 2 and 5 µM (HVLP325). 0.2, 0.5, 0.75, 1, 1.5, 2, 3 and 5 µM (HVLP3103) and 1, 2, 3, 4, 5 and 6 nM (HVLP324). The sensorgrams for HVLP324 and HVLP342 bindings to the<br>
low affinity site of protein L are not included but the calculated KDs are recorded in Table 3.<br>
Figure 9. Bindings of HVHP328PTV2 to protein A and HVLP335PTV2 to protein L in surface plasmon resonance experiments. (A) Sensorgram overlays showing the binding of HVH28PTV2 to immobilized protein A at 1, 2, 3, 4, 6, 8 and 10 nM concentrations. (B) Sensorgram overlays showing the binding of HVLP335PTV2 to immobilized protein L at 1, 2, 2.5, 3, 3.5, 4 and 4.5 nM concentrations. The binding data are recorded in Table 4.<br>
Figure 10. Figure showing the results of the microagglulination experiments with S. aureus cells. The concentration of the pentamers decreases two-fold from well 1 to well 11 with well 12 having the pentamers replaced with PBS buffer. The top row wells contain HVHP328PTV2 pentamer and the bottom ones HVLP335PTV2 pentamer. The concentrations of the pentamers in wells 1 to 6 are 215, 108, 54, 27,13 and 7 ng/ml, respectively.<br>
Detailed Description of the Invention<br>
It is desirable to identify polypeptides, especially antibody fragments, that are of human origin, soluble, stable, resistant to aggregation, refoldable, highly expressed, easily manipulated at the DNA level, ideal for library construction and for 3-D structural determinations. Such antibody fragments are useful for a wide variety of immunotherapeutical applications, and also as diagnostic and detection agents. Human monomeric VH and V|_ antibodies are of particular interest, as they are likely to have many of the above-mentioned properties.<br>
Polypeptides with the above-mentioned properties may be identified by high throughput screening of libraries capable of expressing a variety of polypeptide sequences. For example, phage display libraries (preferably filamentous phage such as M13 or fd) may be screened by infecting a field of bacteria susceptible to the phage (a bacterial lawn) with the phage, then<br><br>
determining which phages have successfully lysed the bacteria by looking for clear, bacteria-free areas known as plaques. Phages displaying monomeric llaminated VHs and VLS form larger plaques on bacterial lawns than phages displaying fully human VHs with aggregation tendencies. Thus, plaque size may be used as a means of identifying rare, naturally-occurring monomer VHS and VLs from the human VH repertoire.<br>
The method disclosed herein Is also useful in identifying soluble, stable (stability covers a number of characteristics, including but not limited to high thermal refolding efficiency, high melting temperature, maintaining functionality after long (several days) incubation at 37°C, resistant to chemical denaturants, resistant to proteases, having a long shelf life at below 0°C, and 4°C, and at room temperature, maintaining functionality in intracellular environments, and maintaining functionality inside the human body, such as in the bloodstream) and high expressing proteins of differing origins, including:<br>
1.	VHs, VLS, Fabs, scFvs and whole antibodies such as IgGs, more specifically<br>
human ones<br>
2.	Protein variants based on non-antibody scaffolds single-chain T-cell<br>
receptors, T-cell receptor domains, transferin, lipocalins, kunitz domains,<br>
ankyrin repeats, and cytotoxic T-lymphocyte-associated antigen (CTLA-4),<br>
including human ones<br>
3.	Vaccines such as viral and bacterial protein vaccines<br>
4.	Therapeutic proteins, e.g., insulin, growth hormone, arythropoietin<br>
5.	Proteinacious diagnostic and biochemical reagents, e.g., protein A, protein<br>
G.<br>
Once polypeptides have been identified by this method, they can be used to construct additional libraries. This is done by selecting a nucleic acid sequence of, for example, a VH. Oligonucleotides with randomized codons are created and incorporated into the VH sequence. Thus, each unique oligonucleotide is incorporated into a VH gene, and the modified VH genes constitute a library of sequences with slight variations. Typically, the Oligonucleotides are designed such that the CDRs or loops of the VH are randomized. For example, one, two or all three of VH CDRs may be<br>
randomized. The VH library is then cloned into an appropriate vector, depending on the type of library to be used, and the nucleic acid sequences are expressed as polypeptides. The library is screened for molecules that bind to the library polypeptides, typically by panning. The libraries may be phage display libraries, or other display libraries such as ribosome display and yeast display.<br>
Polypeptides identified by the method discussed herein may be used for immunotherapy by, for example, the cross-linking of monomers to form dimers, trimers, pentamers and other multimers. This may result in better affinity for antigen molecules and slower dissociation rates for some antigens. Another possible approach is to link or fuse polypeptides to a variety of molecules with various functions. For example, antibody fragments may be linked to radionuclides, cytotoxic drugs, toxins, peptides, proteins, enzymes, liposomes, lipids, T-cell superantigens or viruses in order to target and destroy or modify specific cells or molecules.<br>
Once the VHS or VLS identified by the selection method described herein have been isolated, they can be further manipulated to select for improved biophysical properties such as solubility, stability, monomericity, binding specificity, human origin or high expressability. This can be achieved by In vitro recombination techniques such as DNA shuffling or a staggered extension process. DNA shuffling involves cutting the nucleic acid sequence of first (donor) and second (acceptor) polypeptides, such as antibody fragments, into random fragments, then reassembling the random fragments by a PCR-like reaction. The reassembled fragments are then screened to select for the desired properties.<br>
For example, one or more VHs with high stability (donors) can be mixed with one or more VHs lacking sufficient stability (acceptors) and subjected to DNA shuffling. This generates mutants of the acceptor VHs which have incorporated stability residues from the donor VHs. The newly stable mutants can be identified by the methods described herein, or through other evolutionary protein screening systems such as ribosome display, yeast<br>
display, bacterial cell display and phage display. Similarly, this technique can be used to transfer desirable traits such as solubility, monomericity, and high expression.<br>
This technique may be used where both donor and acceptor VHS have desirable properties, to produce a VH with both properties. For example, an unstable donor VH which binds to an important therapeutic or diagnostic; ligand can be shuffled with a stable acceptor VH. In order to ensure that new generated stable VHS also have the ability to bind to the ligand, the screening system may involve a ligand binding step.<br>
DNA shuffling may also be useful for humanizing non-human VHs such as camelid heavy chain antibody variable domains and nurse shark and wobbegong shark variable domains, or non-human VLs which bind to therapeutic targets. Human VHS and VLS with desirable properties such as solubility, stability, monomericity and high expressability may be used as donors. For example, one or more human VHS with good stability (donors) can be mixed with one or more non-human therapeutic VHS (acceptors) and subjected to DNA shuffling. This generates mutants of the acceptor VHs which are both stable and humanized. The newly generated humanized and stable mutants can be identified by the methods described herein, or through other evolutionary protein screening systems such as ribosome display, yeast display, bacterial cell display and phage display. In a further example, the acceptor VH could be a therapeutic VnH (camelid heavy chain antibody variable domain).<br>
Further, this technique is also useful for selecting desirable properties of polypeptides other than VHs and VLs. As discussed above, the donor polypeptide and the acceptor polypeptide may be both human, or the donor may be human and the acceptor non-human.<br>
A possible approach for imparting solubility, monomericity, high expressability or stability to VHs and VLS may be through grafting complementarity determining regions (CDRs) onto acceptor VHS and VLs. Since CDRs are<br>
known to be involved in the solubility and stability of single-domain antibodies, and accordingly the grafting of these regions, such as the CDRs from VMS and VLs isolated by the methods described herein, may impart solubility and/or stability to acceptor VHs and VL6.<br>
Human Monomeric VHs and VLs<br>
Several monomeric human VHs with different germline and overall sequences were identified (see Figure 1 and SEQ ID NO. 8 through 22) from a naive human VH phage display library by this selection method based on phage plaque size. The VHS remain functional and monomeric following trypsin treatment at 37°C, weeks of incubations at 37"C or months of storage at 4°C, have high thermal refolding efficiencies, are produced in good yields in E. coli and possess protein A binding activity.<br>
In addition, several monomeric human VLS were identified (see Figure 6 and SEQ ID NO. 23 through 54). The VLs are also produced in good yields in E. coli and possess protein L binding activity.<br>
Such properties will also be manifested by VHS from synthetic libraries that utilize the above VHS as scaffolds. Thus, such libraries may yield therapeutic or diagnostic VHS which would have good efficacy at physiological temperature, extended shelf life and a cost-effective production. High thermal refolding efficiency characteristic would further extend the biotechnological applications of these libraries to situations where VH binders are required to maintain their activity after exposure to transient high temperatures. The VHS should also be very suitable for intrabody applications because of their desirable biophysical properties. The protein A binding property will simplify VH purification and detection in diagnostic tests, immunoblotting and immunocytochemistry and can be exploited to enhance library performance by removing nonfunctional VHs from the libraries. Similarly, libraries that utilize VLs as scaffolds will yield therapeutic or diagnostic VtS which have similarly desirable properties. Since VLs bind with protein L, VL purification<br>
and detection is simplified by taking advantage of this protein L binding property.<br>
Display libraries built on the present VHs and VLS may also be a useful source of diagnostics and detection agents.<br>
Previously reported fully human VHS with favorable biophysical properties were based on a single V germline sequence: DP-47 ((Jespers, L. et a/., 2004b;Jespers, L. et al., 2004a). The observation that the monomeric human VHS in this study stem from six different germline sequences including DP-47, demonstrates that stable VHS are not restricted in terms of germline gene usage. In fact, it is very likely that we would have isolated monomeric VHs of family and germline origins different from the ones we describe here had we not restricted our selection to a subset of VHS family VHS with protein A binding activity. It is not possible to pinpoint amino acid mutations (Table 1) responsible for the observed biophysical behavior of the present VHS due to the occurrence of multiple mutations in VHS and the fact that CDR3 is also known to be involved in shaping the biophysical profiles of sdAbs. It may be, however, that mutations at positions known to be important for sdAbs stability and solubility, eg., V37F in HVHP423 and HVHP44B, or mutations occurring multiple times at the same position, e.g., L5V/Q and V5Q in nine VHS, have a role in determining VHs biophysical properties. In terms of library construction, it would be desirable that the monomericity of the present VHS not be dependent on CDRs, in particular CDR3, so that CDR randomization be performed without the worry of jeopardizing library stability. In this regard, the VHs with smaller CDR3, e.g, HVHB82, may be preferred scaffolds since there would be less dependence on CDR3 for stability.<br>
The diversity of the present VHS and VLs in terms of overall sequence and CDR3 length should allow the construction of better-performing libraries. Synthetic VH libraries have been constructed on single scaffolds. Such an approach to repertoire generation is in sharp contrast to the natural, in vivo "approach" which utilizes a multiplicity of scaffolds. Based on the sequences reported here one can take advantage of the availability of the diverse set of<br>
VHs and VLs and create libraries which are based on multiple VH and VL scaffolds. Such libraries would be a better emulation of in vivo repertoires and therefore, would have a more optimal complexity. Of the three CDRs in sdAbs, CDR3 generally contributes most significantly to repertoire diversity and for this reason CDR3 randomization on VH and VL scaffolds are typically accompanied by concomitant varying of CDR3 length. While this significantly improves library complexity, it may also compromise library stability by disrupting the length of the parental scaffold CDR3. The heterogeneity of the VHs and VLs disclosed herein in terms of CDR3 length permit the creation of libraries with both good complexity, good stability and good biophysical characteristics. Such libraries would preferably consist of sub-libraries, where each sub-library is created by CDR3 randomization (and CDR1 and/or CDR2 randomization, if desired) on a single VH or VL scaffold without disrupting the parental CDR3 length.<br>
The versatility of the present VHS and VLS is also beneficial in terms of choosing an optimal VH or VL framework for humanizing VHHs, VHs and VLs which are specific to therapeutic targets . High affinity camelid VHHs against therapeutic targets can be obtained from immune, non-immunized or synthetic VHH libraries with relative ease and be subsequently subjected to humanization (CDR grafting, resurfacing, deimmunization) to remove possible VnH immunogenicity, hence providing an alternative to human VH library approach for production of therapeutic VHS. Generating high affinity therapeutic VHS by the latter approach may often require additional tedious and time consuming in vitro affinity maturation of the lead binder(s) selected from the primary synthetic human VH libraries.<br>
Nonhuman VHs against therapeutic targets can be obtained from immune, non-immunized or synthetic VH libraries with relative ease and be subsequently subjected to humanization (CDR grafting, resurfacing, deimmunization) to eliminate nonhuman VH immunogenicity, hence providing an alternative to human VH library approach for production of therapeutic VHS.<br>
Nonhuman VLS against therapeutic targets can be obtained from immune, non-immunized or synthetic VHH libraries with relative ease and be subsequently subjected to humanization (CDR grafting, resurfacing, deimmunization) to eliminate VHH immunogenicity, hence providing an alternative to human VL library approach for production of therapeutic VLs.<br>
A number of evolutionary approaches for selection of proteins with improved biophysical properties have been described (Forrer, P. et al., 1999;Waldo, G. S., 2003); (Jespers, L. et al., 2004a;Jung, S. etal., 1999;Matsuura, T. et a/., 2003). Typically, stability pressure is required to ensure preferential selection of stable variants over unstable or less stable ones from a library population. For example, in a related work, heat treatment of VH phage display libraries was required to select aggregation resistant VHs (Jespers, L. et al., 2004a). Examples of evolutionary selection approaches involving phage display include conventional phage display, selectively infective phage and the proteolysis approaches. In the first two approaches affinity selection is used to select stable species from a library, based on the assumption that stable proteins possess better binding properties for their ligand than the unstable ones. However, even with the additional inclusion of a stability selection step, these approaches may primarily enrich for higher affinity rather than for higher stability (Jung, S. et al., 1999). A binding step requirement also limits the applicability of these approaches to proteins with known ligands. The third, proteolysis approach is based on the fact that stable proteins are generally compact and therefore are resistant to proteases whereas the unstable ones are not. The phage display format is engineered in such a way that the protease stability of the displayed protein translates to phage infectivity. Thus, when a variant phage display library is treated with a protease, only the phages displaying stable proteins retain their infectivity and can subsequently be selected by infecting an E. coli host. Since this approach is independent of ligand binding, it has general utility. However, even stable and well folded proteins have protease sensitive sites, e.g., loops and linkers, and this could sometimes hinder the selection of stable species in a proteolysis approach (Bai, Y. et al., 2004).<br><br>
By contrast, in the present evolutionary approach, proteins with superior biophysical properties are simply identified by the naked eye. The approach does not require ligand binding, proteolysis or destabilization steps, and thus, avoids complications which may be encountered in the reported selection approaches.    No requirement for a binding step also means that this approach has general utility. As an option, a binding step may be included to ensure that the selected proteins are functional. However, the dependency of the present approach on plating (for plaque visualization) introduces a possible logistical limitation in terms of the number of plates that can be handled and thus limits its application to smaller libraries. Nonetheless, the utility of the current approach can be extended to large libraries, if the library is first reduced to a manageable size. This can be done, for example, by incorporating into the selection system a step which would remove large populations of unstable species, e.g., library adsorption on a protein A surface, or on a hydrophobic interaction column to remove poorly folded proteins with exposed hydrophobic surfaces (Matsuura, T. et a/., 2003). Here, the approach was used to select VHS and VLS of good biophysical properties in a background of very unstable VHS and VLS. However, it may be more difficult to select the "best" species from a mutant library which is populated with proteins with reasonably good stabilities. In this case, the lead variants may be identified based on the rate of plaque formation by using shorter incubation times, or based on plaque size and frequency criteria.<br>
The present selection approach can be extended to identification of stable and well-folded antibody fragments such as scFvs and Fabs with the optional inclusion, in the selection system, of a binding step involving protein L, A or any ligand, as well as stable non-antibody scaffolds and variants thereof. Moreover, the observed correlation between phage plaque size and VH expression yield means that one can utilize the present approach for acquiring high-expressing versions of proteins with otherwise poor or unsatisfactory expression from mutant phage display libraries. This application would be particularly appealing in the case of therapeutic proteins or expensive poor-<br>
expressing protein reagents where boosting protein expression would significantly offset protein production cost.<br>
Binding analyses of pentamers<br>
Both VLS and VHS are amenable to pentamerization and the pentamerization can be used to quickly convert a low affinity VL or VH monomer to a high affinity VL or VH pentamer. Such pentamers are invaluable diagnostics and detection agents. In such applications, the binding of a VL or VH pentamer to its target can be detected by a reporter molecule such as an enzyme (for example, horse radish peroxidase or alkaline phosphatase), or a fluorescent molecule conjugated to the pentamer. Alternatively, the binding of the pentamer can be detected by a secondary molecule which is conjugated to a reporter molecule. The secondary molecule can be specific to the pentamer itself or to a tag thereof, such as a 6His tag or c-Myc tag. For example, a typical secondary molecule is an immunoglobulin.<br>
The interactions between the VHS and protein A and VLS with protein L are fundamentally different from those between VHS and VLs with their target antigens. The antigen binding of a VH or a VL involves three antigen binding loops which form the combining site of an antibody domain. The protein A binding of a VH with protein A binding activity and a VL with protein L binding activity involve binding sites and residues on the antibody domains that are totally distinct from the antibody combining site. Thus, a VH with protein A binding activity can simultaneously bind to protein A and its target antigen and a VL with protein L binding activity can simultaneously bind to protein L and its target antigen. Since the present VHS and VLs have affinity for protein A and L, respectively, protein A and L can be used as the secondary molecule for detection and diagnostic applications mentioned above. The human VH and VL pentamers can also be used for therapy.<br>
Pathogen detection by the pentamers<br>
The protein A and L binding activity of the VHs and VLs can be used to detect bacteria which have protein A and/or L on their surfaces. Protein A is present<br>
on the surface of the pathogenic bacteria, Staphylococcus aureus. Thus, the VHS with protein A binding activity such as the ones described here can be used to detect S. aureus. Similarly, the VL monomers and VL pentamers with protein L binding activity can be used for the detection of bacteria, in particular pathogenic bacteria such as Peptostreptococcus magnus, which have protein L on their cell surface.<br>
Protein L is implicated as a virulent factor in the pathogenesis of P. magnus (Ricci, S. et al, 2001) in humans. In vaginosis, protein L is thought to exert its effect by cross-linking surface associated IgE. VL monomers and/or pentamers with protein L binding activity have potential as therapeutics since they could interfere with the IgE cross-linking action of protein L.<br>
Protein A is implicated as a virulent factor in the pathogenesis of S. aureus in humans (Fournier, B. et al, 2004). Its virulence has been attributed to its ability to interact with host components including binding to antibodies. VH monomers and/or pentamers with protein A binding activity have potential as therapeutics since they could interfere with the interaction of of protein A with host components.<br>
EXAMPLES<br>
Identification and sequence analysis of monomeric human VHS<br>
During the course of the construction of fully human and (laminated human VH<br>
libraries, it was learned that the phages displaying monomeric llaminated VHS formed larger plaques on bacterial lawns than phages displaying fully human VHs with aggregation tendencies. Thus, plaque size was used as a means of identifying rare, naturally-occurring monomer VHS from the human VH repertoire (Figure 1). To this end, a phage library displaying human VHs with a size of 6 x 108 was constructed and propagated as plaques on agar plates. On the liter plates, the library consisted essentially of small plaques interspersed with some large ones. PCR on twenty clones revealed that the small plaques corresponded to the VH-displaying phages while the large ones represented the wild type phages, i.e., phages lacking VH sequence inserts.<br>
None of the VH-displaying phages were found with large plaque morphology. This was not unexpected due to the paucity of the monomeric VHS in the human repertoire and the large size of the library. To facilitate the identification of monomeric VHs, it was decided to reduce the library to a manageable size and remove the interfering wild type phages with large-plaque-size morphology by panning the library against protein A which binds to a subset of human VHS from VH3 family.<br>
Following a few rounds of panning, the library became enriched for phage producing large plaques, and PCR and sequencing of more than 110 such plaques showed that all had complete VH open reading frames. The size of the large plaques which were picked for analysis is represented in Figure 1. Sequencing revealed fifteen different VHS which belonged to the Vn3 family and utilized DP-38, DP-47, V3-49, V3-53, YAC-5 or 8-1B germline V segments (Table 1; Figure 2). DP-38 and DP-47 germline sequences have been previously implicated in protein A binding. In addition, all VHs had a Thr residue at position 57 (Figure 2), consistent with their protein A binding activity. The most frequently-utilized germline V segment was DP-47 which occurred in over 50% of the VHS, but the most frequent clone (i.e., HVHP428; relative frequency 46%) utilized the V3-49 germline V segment. HVHP429 with a DP-47 germline sequence was the second most abundant VH with a relative frequency of 21% (Figure 2). The VH CDR3 lengths ranged from 4 amino acids for HVHB82 to 16 amino acids for HVHP430 amino acids, with HVHP430 having a pair of Cys residues in CDR3. Amino acid mutations with respect to the parental germline V segment (residues 1-94) and FR4 (residues 103-113) sequences, were observed in all VHs and ranged from two mutations for HVHP44 (L5V and Q105R) and HVHB82 (E1Q and L5Q) to sixteen mutations for HVHP426 (Table 1). Mutations were concentrated in the V segments; only two mutations were detected in all the fifteen FR4s, at positions 105 and 108. HVHP44 and HVHB82 differed from other VHs in that they both had a positively-charged amino acid at position 105 instead of a Gin (Table 1; Figure 2). However, while the positively-charged amino acid in HVHP44 was acquired by mutation, the one in HVHB82 was germline-encoded. Except for HVHP423 and HVHP44B. the remaining VHs had the germline residues at the key solubility positions: 37V/44G/45L/47W or<br>
37F/44G/45L/47W (HVHP428); HVHP423 and HVHP44B had a V37F mutation. Mutations at other positions which are shown or hypothesized to be important in VH solubility included seven E6Q, three S35T/H, one R83G and one K83R, one A84P and one T84A and one M108L. Frequent mutations were also observed at positions 1 and 5 which included eleven E1Q, eight L5V/Q and one V5Q mutations.<br>
Biophysical characterization of the human VHS<br>
All  VHs  except  HVHP44B,  which  was  essentially the  same  as<br>
HVHP423, were expressed in one-litre-culture volumes in E. constrain TG1 in fusion with c-Myc-Hiss tag and purified to homogeneity from periplasmic extracts by immobilized metal affinity chromatography (IMAC). The expression yields ranged from 1.8 to 62.1 mg of purified protein per liter of bacterial culture in shaker flasks with majority of VHS having yields in several milligrams (Table 2). In the instance of HVHP423 and HVHP430, another trial under "apparently" the same expression conditions gave yields of 2.4 and 6.4 mg as opposed to 62.1 and 23.7 mg, respectively. This implies that for many of the VHS described here optimal expression conditions should be achieved, without much effort, resulting in expression yields significantly higher than the reported values in Table 2. As expected, all the VHS bound to protein A in surface plasmon resonance (SPR) analyses, with KD s of 0.2-3 µM, a range and magnitude comparable to the ones reported previously for llama VHH variants with protein A binding activity. None of the VHs bound to the Fab reference surface.<br>
The aggregation tendency of the human VHS was assessed in terms of their oligomerization states by gel filtration chromatography and NMR (Table 2). All VHS were subjected to Superdex 75 gel filtration chromatography. Similar to a llama VHH, i.e., H11C7, all VHS gave a symmetric single peak at the elution volume expected for a monomer, and were substantially free of any aggregates (see the example for HVHP428 in Figure 3A. In contrast, a typical human VH (i.e., BT32/A6) formed considerable amount of aggregates. For three of the VHS, a minor peak with a mobility expected for a VH dimer was also observed. SPR analyses of the minor peaks gave off-rate values which<br>
were significantly slower than those for the monomer VHs, consistent with them being dimers. The dimer peak was also observed in the case of the llama VHH, H11C7. The folding and ollgomerization states of the VHs at high concentrations were further studied by NMR spectroscopy. As shown in Table II, all the VH proteins studied appeared to be relatively soluble and assumed a well-folded three-dimensional structure. One-dimensonal NMR spectra of the VH fragments (Fig. 3B) showed structure folds characteristic of VH domains. The state of protein aggregation was also assessed by use of an PFG-NMR diffusion experiment for the HVHP414 fragment and two isoforms, VH14 and VH14-cMyc- with and without the c-Myc sequence, of the HVHP414. VH14 is a modified version of HVHP414 with a c-Myc N132E mutation and with an additional methionine residue at the N-terminus. In brief, the PFG-NMR data (not shown) indicated that all the protein samples had expected monomeric molecular weights even at the relatively high protein concentrations used for NMR experiments.<br>
The stability of the VHS was further investigated in terms of their resistance to trypsin at 37°C integrity following long incubations at 37°C. Trypsin cleaves polypeptide amide backbones at the C-terminus of an Arg or a Lys residue. There are 9-13 Arg and Lys residues in the human VHS (Figure 2). There is also an additional Lys residue in the C-terminal c-Myc tag which is susceptible to digestion by trypsin. Figure 4a is an SDS-PAGE analysis of HVHP414 during trypsin digestion. Within 1 h the original band was completely converted to a single product which had a mobility expected for the VH with no c-Myc-HiS5 tag. The same result was obtained for 12 other VHS following a one-hour incubation with trypsin. Mass spectrometry on a randomly selected sample of the trypsin-treated VHs (i.e., HVHP414, HVHP419, HVHP420. HVHP423, HVHP429, HVHP430 and HVHM81) confirmed that in every case the molecular mass of the digested product corresponded to a VH with the c-Myc Lys as the C-terminal residue. HVHM41 gave a significantly shorter fragment than the rest upon digestion, and in this case mass spectrometry experiments mapped the cleavage site to the Arg99 in CDR3 (data not shown).<br>
Eleven VHs ranging in concentration from 0.32 mg/ml (HVHP428) to 3.2 mg/ml (HVHP420) were incubated at 37°C for 17 days. Their stability was subsequently determined in terms of oligomerization state and protein A binding. As shown by gel filtration chromatography, treatment of VHS at 37°C did not induce any aggregate formation: all VHS gave chromatogram profiles which were virtually identical to those of untreated VHS and stayed essentially as monomers (see the example for HVHP420; Figure 4c). To ensure that the VHS maintained their native fold following 37°C treatment, two VHS, namely, HVHP414 (1.2 mg/ml) and HVHP420 (3.2 mg/ml), were selected at random and their KDS of binding to protein A were determined by SPR (Data shown for HVHP420; Figure Ac inset) and compared to the KDS obtained for untreated VHS (Table 2). The calculated KDS for the heat-treated VHS were 1.4 µM and 1.0 µM for HVHP414 and HVHP420, respectively. These values are essentially identical to the corresponding values for the untreated VHS (Table 2), demonstrating that 37°C treatment of VHS did not affect their native fold. The possibility that VHs may have been in a less compact, non-native fold during the 37°C-incubation periods and resumed their native fold upon returning to room temperature during gel filtration and SPR experiments is unlikely in light of the fact that the VHS were resistant to trypsin at 37°C (see above), a property typically associated for well folded native proteins.<br>
The refolding efficiency (RE) of the human VHS was investigated by comparing the KDs of the binding of the native (KDn) and heat-treated, refolded (KDref) VHS to protein A (Tanha, J. et al., 2002). When a fraction of the VH is inactivated by heat treatment the measured KD would be higher, since this parameter is based on the concentration of folded, i.e., active, antibody fragment. Thus, the ratio of KDn to KDref gives a measure of VH RE. Figure 5 compares sensorgrams for HVHP423 binding to immobilized protein A in native (thick lines) and refolded (thin lines) states at several selected VH concentrations. As can be seen, binding of the refolded VH to protein A is less in all instances, indicating that the unfolding is not fully reversible. For each of the 14 VHs, protein A binding in both native and refolded states was measured<br>
at several concentrations, and the KDs and subsequently' REs were determined (Table 2; KDref values are not shown). The KD s and REs of two anti-idiotypic llama VHHs, H11F9 and H11 B2, which were used as references, were also determined. Four VMS had REs in the range of 92%-95%, similar to the REs for H11F9 and H11B2, 95% and 100%, respectively. Another five had REs in the range of 84%-88% and three over 70%. Only two had significantly lower RE: HVHP413 (52%) and HVHP421 (14%). Several published VnHs examined previously had RE around 50% (van der Linden, R. H. etal., 1999).<br>
Human VH phage display library construction and panning. cDNA was synthesized from human spleen mRNA (Ambion Inc., Austin, TX) using random hexanucletide primers and First Strand cDNA™ kit (GE Healthcare, Bale d'Urfe, QC, Canada). Using the cDNAs as template, VH genes with flanking CH sequences were amplified by polymerase chain reaction (PCR) in nine separate reactions using VH framework region 1 (FRI)-specific primers and an immunoglobin M-specific primer (de Haard, H. J. et al., 1999). The products were gel-purified and used as the template in the second round of PCR to construct VH genes using the FR1- and FR4-specific primers (de Haard, H. J. et a/., 1999) that also introduced flanking Apal I and Not \ restriction sites for cloning purposes. The resultant VH repertoire DNAs were cloned into fd-tetGIIID phage vector and a VH phage display library was constructed (Tanha, J. et at., 2001). Panning against protein A (Amersham Biosciences Inc.) was performed as described (Tanha, J. et a/., 2001). Germline sequence assignment of the selected VHs was performed using DNAPLOT software Version 2.0.1 and V BASE version 1.0 (http://vbase.dnaplot.de/cgi-bin/vbase/vsearch.pl). Llama VHHs H11C7, H11F9 and H11B2 were isolated from a llama VHH phage display library by panning against H11 scFv as described (Tanha, J. et al, 2002).<br>
VH expression and purification. VHS were cloned into pSJF2 expression vectors by standard cloning techniques (Sambrook, J. Fritsch E. F. and Maniatis T, 1989). Periplasmic expression of sdAbs and subsequent<br>
purification by immobilized metal affinity chromatography (IMAC) were performed as described (Muruganandam, A. et al, 2002). Protein concentrations were determined by A280 measurements using molar absorption coefficients calculated for each protein (Pace, C. N. et a/., 1995). Gel filtration chromatography of the purified VMS was performed on a Superdex 75 column (GE Healthcare) as described (Deng, S. J. et a/., 1995).<br>
Binding and refolding efficiency experiments. Equilibrium dissociation constants (Kas) and refolding efficiencies (REs) of VHs/VHHs were derived from surface plasmon resonance (SPR) data collected with BIACORE 3000 biosensor system (Biacore Inc., Piscataway, NJ). To measure the binding of VHS to protein A, 2000 resonance units (RUs) of protein A or a reference antigen-binding fragment (Fab) were immobilized on research grade CM5 sensor chips (Biacore Inc.). Immobilizations were carried out at concentrations of 25 µg/ml (protein A) or 50 µg/ml (Fab) in 10 mM sodium acetate buffer pH 4.5, using the amine coupling kit provided by the manufacturer. To measure the binding of the anti-idiotypic llama VHHs to H11 scFv, 4100 RUs of 50 ug/ml H11 scFv or 3000 RUs of 10 ug/ml Se155-4 IgG reference were immobilized as described above. In all instances, analyses were carried out at 25°C in 10 mM HEPES, pH 7.4, containing 150 mM NaCI, 3 mM EDTA and 0.005% P20 at a flow rate of 40µl/min, and surfaces were regenerated by washing with the running buffer. To determine the binding activities of the refolded proteins, VHs or VHHs were denatured by incubation at 85eC for 20 min at 10 pg/ml concentrations. The protein samples were then cooled down to room temperature for 30 min to refold and were subsequently centrifuged in a microfuge at 14,000 rpm for 5 min at room temperature to remove any protein precipitates. The supernatants were recovered and analyzed for binding activity by SPR as described above. For both folded and refolded proteins data were fit to a 1:1 interaction model simultaneously using BIAevaluation 4.1 software (Biacore Inc.) and KDs were subsequently determined. REs were determined from<br>
(Formula Removed)<br>
Where KDn is the/:D of the native protein and KDref\s theKD of the refolded protein.<br>
Tryptic digest experiments. 3 µl of a freshly prepared 0.1 µg/µl sequencing grade trypsin (Hoffmann-La Roche Ltd., Mississauga, ON, Canada) in 1 mM HCI was added to 60 µg VH in 100 mM Tris-HCI buffer pH 7.8. Digestion reactions were carried out in a total volume of 60 \i\ for 1 h at 37°C and stopped by adding 5 µl of 0.1 µg/µl trypsin inhibitor (Sigma, Oakville, ON, Canada). Following completion of digestion, 5µl was removed and analyzed by SDS-PAGE; the remaining was desalted using ZipTipC4 (Millipore, Nepean, ON, Canada), eluted with 1% acetic acid in 50:50 methanolrwater and subjected to VH mass determination by MALDI mass spectrometry.<br>
Protein stability studies at 37°C. Single-domain antibodies (sdAbs) at 0.32-3.2 mg/ml concentrations were incubated at 37°C in PBS buffer for 17 days. Following incubation, the protein samples were spun down in a microfuge at maximum speed for 5 min even in the absence of any visible aggregate formation. The samples were then applied onto a Superdex 75 size exclusion column (GE Healthcare) and the monomeric peaks were collected for SPR analysis against protein A. SPR analyses were performed as described above except that 500 RUs of protein A or reference Fab was immobilized and that immobilizations were carried out at concentration of 50 µg/ml.<br>
NMR experiments - VH samples for NMR analysis were dissolved in 10 mM sodium phosphate, 150 mM NaCI, 0.5 mM EDTA, and 0.02% NaN3 at pH 7.0. The protein concentrations were 40 µM - 1.0 mM. All NMR experiments were carried out at 298 K on a Bruker Avance-800 or a Bruker Avance-500 NMR spectrometer. One-dimensional (1D) 1H NMR spectra were recorded with 16,384 data points and the spectral widths were 8,992.81 Hz at 500 MHz and 17,605.63 Hz at 800 MHz, respectively. Two-dimensional 1H-1H NOESY spectra of 2,048 x 400 data points were acquired on a Bruker Avance-800 NMR spectrometer with a spectral width of 11,990.04 Hz and a mixing time of 120 ms. In all NMR experiments, water suppression was achieved using the<br>
WATERGATE method implemented through the 3-9-19 pulse train (Piotto, M. era/., 1992;Sklenar, V. eral., 1993). NMR data were processed and analyzed using the Bruker XWINNMR software package. All PFG-NMR diffusion measurements were carried out with the water-suppressed LED sequence (Altieri, A. S. et al., 1995), on a Bruker Avance-500 NMR spectrometer equipped with a triple-resonance probe with three-axis gradients. One-dimensional proton spectra were processed and analyzed using Bruker Xwinnmr software package. NMR signal intensities were obtained by integrating NMR spectra in the methyl and methylene proton region (2.3ppm to -0.3ppm) where all NMR signals were attenuated uniformly at all given PFG strengths.<br>
Human VL phage display library construction and panning. cDNAs were synthesized from human spleen mRNA as described above for the human VHS. The cDNA was used as template in PCR to amplify Vu genes in 50 µl reaction volumes using six V* back primers, 11 Vx back primers (de Haard, H. J. et al., 1999), four VK For primers and two Vx For primers (Sblattero, D. et al., 1998). The back and forward primers were modified to have flanking Apa LI and Not I restriction sites, respectively, for subsequent cloning purposes. Forward primers were pooled together in ratios which reflected their degree of degeneracy. V* genes were PCRed in 11 separate reactions using the pooled Vx For primers and 11 individual V\ back primers. Similarly, Vx genes were amplified in 6 separate reactions using the pooled VK For primers and 6 individual Vx back primers. The PCR products were pooled, gel purified and digested with Apa LI and Not I restriction endonucleases. The library was constructed as described for human VHs. Plaque PCR was performed on individual library colonies and the amplified VL genes were sequenced as described (Tanha, J. et al., 2003). Panning against protein L (Biolynx Inc., Brockville, ON, Canada) and germline sequence assignment of the selected VLS were performed as described above for human VH library.<br>
VL expression and purification. VL expression, purification, concentration determination and gel filtration chromatography were carried out as described for VHS in "VH expression and purification.".<br>
Expression and purification of VL and VH pentamers. Specific primers were used in a standard PCR to amplify HVHP328 VH and HVLP335 VL genes. Standard cloning techniques were used to clone the HVHP328 and HVLP335 genes in fusion with VT1B pentamerization domain gene in an expression vector to yield HVHP328PVT2 and HVLP335PTV2 pentamers, (Zhang, J. et a/., 2004). Pentamers were expressed and purified as described (Zhang, J. et a/., 2004). Protein concentrations were determined as above.<br>
Surface plasmon resonance of VLs. The binding kinetics for the interaction of the VLs to protein L were determined by SPR using BIACORE 3000 biosensor system (Biacore, Inc., Piscataway, NJ). 680 RUs of protein L or 870 RUs of a Fab reference were immobilized on research grade CM5 sensor chips (Biacore). Immobilizations were carried out at a protein concentration of 50µg/ml in 10 mM acetate buffer pH 4.5 using the amine coupling kit supplied by the manufacturer. All measurements were carried out at 25°C in 10 mM HEPES buffer pH 7.4. containing 150 mM NaCI, 3 mM EDTA and 0.005% P20 at a flow rate of 50 µl/min or 100µl/min. Surfaces were regenerated by washing with the running buffer. Data were evaluated using the BIAevaluation 4.1 software (Biacore, Inc.).<br>
Surface plasmon resonance of the pentameric VL and VH. The binding kinetics for the interaction of HVHP328PVT2 with protein A and HVLP335PTV2 with protein L were also determined by SPR. 520 RUs of protein A or a Fab reference were immobilized as above. For the VL pentamer, the same surfaces prepared above were used. Measurements were carried out as above but at a flow rate of 20µl/min. Surfaces were regenerated by washing with 50 mM HCI for 3 s. Data were evaluated as described for the monomers.<br>
 Cell microagglutination<br>
A single S. aureus colony from a BHI plate was used to inoculate 15 mL of BHI media. The bacteria were grown overnight at 37°C at 200 rpm. In the morning, the culture was spun down in a swinging bucket, Sorvall RT6000B refrigerated centrifuge at 4000 rpm for 10 min, the supernatant was removed and the cell pellet was re-suspended in PBS buffer. The cells were re-spun, the supernatant was removed and the cell pellet was re-suspended again in PBS buffer. The cells were diluted to an A6oo of 1.0, and serial dilutions of the cells were spread on BHI plates at 37°C for overnight growth. The cell titer was determined in the morning. An A6oo of 1.0 corresponded to 1.5x 10* cells ml-1. Identical steps were taken to prepare E. coli starin TG1 cells for subsequent microagglutination assays, except that the growth media was 2xYT. The viable counts were similar, A6ool -0 = 2.1x109 cells m-1.<br>
To perform microagglutination assays, two fold dilutions of HVHP328PVT2 in PBS were performed from wells 1 to 11 in a microtiter plate. Well 12 (blank) had only PBS. The total volume in each well was 50 µl . Subsequently, 1 x 108 S. aureus cells in 50µl  PBS was added to all wells and the plate was incubated overnight at 4°C. To have a permanent record of the results, a picture was taken from the plate in the morning. For the pentamer control experiment, HVHP328PVT2 was replaced with the VL pentamer, HVLP335PTV2. In the cell control experiments, the same two sets of experiments were repeated with E. coliTG1 cells.<br>
Identification and sequence analysis of monomeric humanVLS<br>
Essentially the same selection method which was employed to isolate soluble VHs from a human VH phage display library was applied to a human VL library for isolating soluble, monomeric VLS. A human VL library with a size of 3 x 106 was constructed. Twenty four plaques from the library titer plates were picked and their VL genes were PCRed and sequenced. The sequences were diverse in terms of germ-line origin although 75% of the VLs were of Vλ . origin (data not shown). Three rounds of panning against protein L resulted in<br>
To perform microagglutination assays, two fold dilutions of HVHP328PVT2 in PBS were performed from wells 1 to 11 in a microtiter plate. Well 12 (blank) had only PBS. The total volume in each well was 50µl . Subsequently, 1 x 108 S. aureus cells in 50 µl PBS was added to all wells and the plate was incubated overnight at 4°C. To have a permanent record of the results, a picture was taken from the plate in the morning. For the pentamer control experiment, HVHP328PVT2 was replaced with the VL pentamer, HVLP335PTV2. In the cell control experiments, the same two sets of experiments wore repeated with E. coli TG1 cells.<br>
Identification and sequence analysis of monomerlc human VLs<br>
Essentially the same selection method which was employed to isolate soluble VHs from a human VH phage display library was applied to a human VL library for isolating soluble, monomeric VLS. A human VL library with a size of 3 x 106 was constructed. Twenty four plaques from the library titer plates were picked and their VL genes were PCRed and sequenced. The sequences were diverse in terms of germ-line origin although 75% of the VLS were of VX origin (data not shown). Three rounds of panning against protein L resulted in enrichment for large plaques. Thirty-nine of large plaques were sequenced and 32 unique sequences were identified (Figure 6). HVLP325, HVLP335 and HVLP351 occurred at frequency of 3, 4 and 2, respectively. Except for HVLP389 which is of lambda class (subgroup VX1, germline 1b), the remaining 31 VLs belonged to the VK class. Of the 31 kappa VLS, 24 fall within the Viclll subgroup and 7 within the VK! subgroup. Sixteen of the 24 VK!|| sequences utilize L6 germline sequence with the remaining utilizing A27, L2 and L6 germline sequences. The Vic1 subgroup VLs are originated from O2/O12 or A30 germline sequence. Noticeable mutations occurred at position 96. The germline amino acids at this position are aromatic and hydrophobic amino acids Trp, Phe, Tyr, Leu or lie for kappa VLs and Tyr, Val or Ala for lambda VLs. But in the selected pool of kappa VLs only 5 out of 31 have their germline amino acids at position 96: HVLP325. HVLP349, HVLP388, HVLP3109 and HVLP393. 21 amino acids at position 96 are<br>
charged of which 20 are positively-charged: Arg, Lys or His. Two amino acids are Pro, one Gin, one Ser and one Thr. Of seven kappa VLs analyzed by gel filtration chromatography for monomericity, six which had Arg or Lys at position 96 were also monomers, whereas HVLP325 with the germline amino acid Leu at position 96 formed aggregates (see below). Similarly, HVLP389 which was of the lambda class and had a germline mutation to Ser was also monomeric (see below). These data correlates the deviation from the germline amino acids at position 96 (27 out of 32) with improved biophysical properties of VLs such as monomericity.<br>
Eighteen VLS of the kappa class had their last three residues (105-107) replaced with amino acids Thr, Val and Leu which are only found in lambda VLS. These substitutions may have had a role in improving the biophysical properties of the kappa VLS, resulting in the selection of the aforementioned VLS over the parental clones with the original kappa residues at position 105-107.<br>
Characterization of the human<br>
Eight of the selected VLS with different V germline origins were expressed in E. coli in one-liter cultures and purified: HVLP324, HVLP325, HVLP335, HVLP342, HVLP351, HVLP364, HVLP389 and HVLP3103 (Table 6). All were expressed in good yields ranging from 6.2 mg for HVLP324 to around 75 mg for HVLP335 and HVLP364.<br>
The aggregation tendency of the human VLs was assessed in terms of their oligomerization state by gel filtration chromatography. VLS were subjected to Superdex 75 gel filtration chromatography at a concentration of 0.6 mg/ml. All except HVLP325 were essentially free of aggregates and gave symmetric single peaks with the mean apparent molecular mass of 12.7 kDa (range, 6.2-19.2 kDa) (Figure 7A and Table 3). This is in agreement with the expected molecular mass for monomeric VLS, 13.4-13.8 kDa. Variation in apparent molecular mass for single-domain antibodies has been reported previously (Jespers, L. et aL., 2004a; (Stevens, F. J. et a/., 1980). For HVLP325, the<br>
aggregates formed 11% of the total protein (aggregate plus monomer). HVLP351, HVLP342, HVLP335 and HVLP3103, were still monomer when tested at their highest concentration available, i.e., 0.89 mg/ml, 1.0 mg/ml, 4.9 mg/ml and 5.9 mg/ml, respectively (Figure 7B)<br>
Vis were subjected to Superdcx-75 chromatography prior to BIACORE analysis and purified monomer peaks collected even in the absence of any evidence of aggregated material. In SPR analysis, all selected VLs bound to protein L (Figure 8). This was not unexpected since the VLS were isolated by panning against protein L. For all, the KDS of binding to protein L were in 0.6-3 pM (Table 3). HVLP324 and HVLP342 had additional smaller K0s, 10 nM and 40 nM, respectively. Low affinity and high affinity bindings of VLS of VK! subgroup to protein L have been reported previously (Reference). Both, HVLP324 and HVLP342, belong to VK! subgroup (Table 3). As expected, the kinetic and equilibrium data were consistent with the monomeric peak being indeed monomeric.<br>
Binding analyses of pentamers<br>
Bindings of HVHP328PVT2 pentamer to protein A and HVLP335PTV2 pentamer to protein L were determined by surface plasmon resonance (Figure 9). The association rates were independently calculated from plots of kobs versus concentration. More than one dissociation rate (kd) could be calculated due to the heterogeneity in multivalent binding amongst the pentamer population. Therefore, more than one equilibrium dissociation constant, KD, could be obtained. HVHP328PTV2 and HVLP335PTV2 had minimum KDs of 2 nM and 200 pM, respectively (Table 4). With slower k,js, HVHP328PTV2 and HVLP335PTV2 had KDs as low as 900 and 90 pM, respectively.<br>
Pathogen detection by VLS and VHs<br>
The protein A and L binding activity of the VHS and VLS can be used to detect bacteria which have protein A and/or L on their surfaces. This is possible if the VHS and VLs are soluble and monomeric (lack of tendency to aggregate)<br>
such as the VHs and VLs here. Variable domains derived from antibodies which lack light chains such as camelid heavy chain antibodies or nurse shark and wobbegong shark IgNARs are naturally soluble and monomeric. From these, those with protein A and L binding activity.can also be used to detect bacteria which have protein A and/or L on their surfaces. Protein A is present on the surface of the pathogenic bacteria, Staphylococcus aureus. Thus, the VHS with protein A binding activity such as the ones described here can be used to detect S. aureus. We performed a microagglutination assay to detect the ability of HVHP328PVT2 VH pentamer to bind to S. aureus. A constant number of bacterial cells were incubated with two-fold dilutions of HVHP328PVT2 in microtiter wells (wells 1-11) (Figure 10). Well 12 had buffer instead of the pentamer. If the VHS bind to the bacterial cells, then the pentamer because of its multimeric nature should be able to cross-link the cells and results in cell agglutination. The agglutinated cells will appear as diffused cells in a microtiter well (Figure 10). In the absence of any binding, no agglutination should occur, hence no agglutination, and the cells will appear as a dot at the bottom of the well. As shown in Figure 10, the pentamer binds to the S. aureus, since there is agglutination of cells. The agglutination is observed up to well 7. Beyond well 7 the concentration of the pentamer is too low for binding, hence no agglutination. The control VL pentamer does not show any agglutination, demonstrating the specificity of the VH pentamer to S. aureus (Figure 10). The binding is also cell-specific since the VH pentamer as expected does not agglutinate E. coli (TG1 strain) or Salmonella cells (data not shown). Similarly, the VL monomers and VL pentamers with protein L binding activity can be used for the detection of bacteria, in particular pathogenic bacteria such as Peptostreptococcus magnus, which have protein L on their cells surface.<br>
It is understood that the examples described above in no way serve to limit the true scope of this invention, but rather are presented for illustrative purposes.<br>
Reference List<br>
Bai, Y, and Feng, H. (2004). Selection of stably folded proteins by phage-display with proteolysis. Eur.J.Biochem. 271: 1609-1614.<br>
Davies, J. and Riechmann, L. (2-21-1994). 'Camelising' human antibody fragments: NMR studies on VH domains. FEBS Lett 339: 285-290.<br>
Davies, J. and Riechmann, L. (1995). Antibody VH domains as small recognition units. Biotechnology N.Y. 13: 475-479.<br>
de Haard, H. J., van Neer, N., Reurs, A., Hufton, S. E., Roovers, R. C., Henderikx, P., de Bruine, A. P., Arends, J. W., and Hoogenboom, H. R. (6-25-1999). A large non-immunized human Fab fragment phage library that permits rapid isolation and kinetic analysis of high affinity antibodies. J.Biol.Chem. 274: 18218-18230.<br>
Deng, S. J., MacKenzie, C. R., Hirama, T., Brousseau, R., Lowary, T. L., Young, N. M., Bundle, D. R., and Narang, S. A. (5-23-1995). Basis for selection of improved carbohydrate-binding single-chain antibodies from synthetic gene libraries. Proc.Natl.Acad.Sci U.S.A 92: 4992-4996.<br>
Forrer, P., Jung, S., and Pluckthun, A. (1999). Beyond binding: using phage display to select for structure, folding and enzymatic activity in proteins. Curr.Opin.Struct.Biol. 9: 514-520.<br>
Foumier, B. and Klier, A. (2004). Protein A gene expression is regulated by DNA supercoiling which is modified by the ArlS-ArlR two-component system of Staphylococcus aureus. Microbiology 150: 3807-3819.<br>
Hamers, C. C., Atarhouch, T., Muyldermans, S., Robinson, G., Hamers, C., Songa, E. B., Bendahman, N., and Hamers, R. (6-3-1993). Naturally occurring antibodies devoid of light chains. Nature 363: 446-448.<br>
Jespers, L., Schon, O., Famm, K., and Winter, G. (2004a). Aggregation-resistant domain antibodies selected on phage by heat denaturation. Nat.Biotechnol. 22: 1161-1165.<br>
Jespers, L., Schon, O., James, L. C., Veprintsev, D., and Winter, G. (4-2-2004b). Crystal Structure of HEM, a Soluble, Refoldable Human V(H) Single Domain with a Germ-line Scaffold. J.Mol.Biol. 337: 893-903.<br>
Jung, S., Honegger, A., and Pluckthun, A. (11-19-1999). Selection for improved protein stability by phage display. J.Mol.Biol. 294: 163-180.<br>
Matsuura, T. and Pluckthun, A. (3-27-2003). Selection based on the folding properties of proteins with ribosome display. FEBS Lett. 539: 24-28.<br>
Muruganandam, A., Tanha, J., Narang, S., and Stanimirovic, D. (2002). Selection of phage-displayed llama single-domain antibodies that transmigrate across human blood-brain barrier endothelium. FASEB J. 16: 240-242.<br>
Pace, C. N., Vajdos, F., Fee, L., Grimsley, G., and Gray, T. (1995). How to measure and predict the molar absorption coefficient of a protein. Protein Sci. 4: 2411-2423.<br>
Ricci, S., Medaglini, D., Marcotte, H., Olsen, A., Pozzi, G., and Bjorck, L. (2001). Immunoglobulin-binding domains of peptostreptococcal protein L enhance vaginal colonization of mice by Streptococcus gordonii. Microb.Pathog. 30: 229-235.<br>
Sambrook, J. F. E. F. a. M. T. (1989). "Molecular Cloning: A laboratory Manual (2nd ed.)", Cold Spring Harbor Laboratory, Cold Spring Harbor, NY..<br>
Sblattero, D. and Bradbury, A. (1998). A definitive set of oligonucleotide primers for amplifying human V regions. Immunotechnology. 3: 271-278.<br>
Tanha, J., Dubuc, G., Hirama, T., Narang, S. A., and MacKenzie, C. R. (5-1-2002). Selection by phage display of llama conventional V(H) fragments with heavy chain antibody V(H)H properties. J.Immunol.Methods 263: 97-109.<br>
Tanha, J., Muruganandam, A., and Stanimirovic, D. (2003). Phage Display Technology for Identifying Specific Antigens on Brain Endothelial Cells. Methods Mol.Med. 89: 435-450.<br>
Tanha, J., Xu, P., Chen, Z. G., Ni, F., Kaplan, H., Narang, S. A., and MacKenzie, C. R. (7-6-2001). Optimal design features of camelized human single-domain antibody libraries. J.Biol.Chem 276: 24774-24780.<br>
van der Linden, R. H., Frenken, L. G., de Geus, B., Harmsen, M. M., Ruuls, R. C., Stok, W., de Ron, L., Wilson, S., Davis, P., and Verrips, C. T. (4-12-1999). Comparison of physical chemical properties of llama VHH antibody fragments and mouse monoclonal antibodies. Biochim.Biophys.Acta 1431: 37-46.<br>
Waldo, G. S. (2003). Genetic screens and directed evolution for protein solubility. Curr.Opin.Chem.Biol. 7: 33-38.<br>
Ward, E. S., Gussow, D., Griffiths, A. D., Jones, P. T., and Winter, G. (10-12-1989). Binding activities of a repertoire of single immunoglobulin variable domains secreted from Escherichia coli [see comments]. Nature 341: 544-546.<br>
Zhang, J., Li, Q., Nguyen, T. D., Tremblay, T. L., Stone, E., To, R., Kelly, J., and MacKenzie, C. R. (7-30-2004). A pentavalent single-domain antibody approach to tumor antigen discovery and the development of novel proteomics reagents. J.Mol.Biol. 341: 161-169.<br>
Table 1. VH sequence deviations from parental germline sequences<br><br>
(Table Removed)<br><br>
Table 2. Biophysical characteristics of the human VHS<br>
(Table Removed)<br>
Expression yield per liter of bacterial culture<br>
* KD s and REs were determined against H11 scFv,<br>
Table 3. Characteristics of the human VLs<br><br>
(Table Removed)<br>
a Expression yield per liter of bacterial culture.<br>
6 Oligomerization state was determined by ge! filtration chromatography.<br>
c The smaller KD values correspond to the binding of the of HVLP324 and<br>
HVLP342 to the high affinity sites on protein L.<br>
Table 4. Kinetic and equilibrium constants for the bindings of HVHP328PTV2 and HVLP335PTV2 to protein A and L, respectively<br><br>
(Table Removed)<br><br><br><br>
What is claimed is:<br>
1.	A method of identifying target polypeptides , comprising:<br>
(a)	obtaining a phage display library capable of expressing a variety<br>
of polypeptide sequences;<br>
(b)	allowing infection of a bacterial lawn by the library phage; and<br>
(c)	identifying phage which form larger than average plaques on the<br>
bacterial lawn.<br><br>
2.	A method as claimed in claim 1, where the target polypeptides are<br>
soluble.<br>
3.	A method as claimed in claim 1, where the target polypeptides are<br>
monomeric.<br>
4.	A method as claimed in claim 1, where the target polypeptides are<br>
non-aggregating.<br>
5.	A method as claimed in claim 1, where the target polypeptides are<br>
highly expressed.<br>
6.	A method as claimed in claim 1, where the target polypeptides are<br>
stable.<br>
7.	A method as claimed in claim 1, where the phage is a filamentous<br>
phage.<br>
8.	A method as claimed in claim 7, where the phage is M13 or fd.<br>
9.	A method as claimed in claim 6, wherein the stable polypeptides<br>
have relatively high thermal refolding efficiency.<br>
10.	A method as claimed in claim 6, wherein the stable polypeptides<br>
have a lelalively high melting temperature.<br><br>
11.	A method as claimed in claim 6, wherein the stable polypeptides<br>
have maintain their functionality following long incubation at 37°C.<br>
12.	A method as claimed in claim 6, wherein the stable polypeptides<br>
are relatively resistant to chemical denaturants.<br>
13.	A method as claimed in claim 6, wherein the stable polypeptides<br>
are relatively resistant to proteases.<br>
14.	A method as claimed in claim 6, wherein the stable polypeptides<br>
have a relatively long shelf life at or above room temperature, at<br>
4°C, or below 0°C.<br>
15.	A method as claimed in claim 6, wherein the stable polypeptides<br>
are functional in intracellular environments.<br>
16.	A method as claimed in claim 6, wherein the stable polypeptides<br>
are functional when administered internally to humans.<br>
17.	A method as claimed in any claim 1, further comprising the steps of:<br><br>
(d)	isolating the larger plaque phage in step (c); and<br>
(e)	determining the sequence or other characteristics of the<br>
polypeptides expressed by the larger plaque phage.<br><br>
18.	A method as claimed in claim 1, wherein the target polypeptides are<br>
antibodies or fragments of antibodies.<br>
19.	A method as claimed in claim 18, wherein the target polypeptides<br>
are human VH or VL antibody fragments.<br>
20.	A method as claimed in claim 1, wherein the target polypeptides are<br>
vaccines.<br><br>
21.	A method as claimed in claim 1, wherein the target polypeptides are<br>
therapeutic proteins.<br>
22.	A method as claimed in claim 1, wherein the target polypeptides are<br>
selected from the group consisting of: single-chain T-cell receptors,<br>
T-cell receptor domains, transferin, lipocalins, kunitz domains,<br>
ankyrin repeats, and cytotoxic T-lymphocyte-associated antigen.<br>
23.	A method as claimed in claim 1, wherein the target polypeptides are<br>
proteinacious diagnostic and biochemical reagents.<br>
24.	A polypeptide having an amino acid sequence selected from the<br>
group consisting of: SEQ ID NO:8-54.<br>
25.	A polypeptide having an amino acid sequence that is substantially<br>
identical to the amino acid sequence claimed in claim 24.<br>
26.	A nucleic acid sequence that encodes a polypeptide as claimed in<br>
claim 24 or 25.<br>
27.	A polypeptide comprising the CDR3 portion of an amino acid<br>
sequence as claimed in claim 24.<br>
28.	A nucleic acid sequence that encodes a polypeptide as claimed in<br>
claim 27.<br>
29.	A polypeptide comprising the FR1 portion of an amino acid<br>
sequence as claimed in claim 24.<br>
30.	A nucleic acid sequence that encodes a polypeptide as claimed in<br>
claim 29.<br>
31.	A polypeptide comprising the FR2 portion of an amino acid<br>
sequence as claimed in claim 24.<br><br>
32.	A nucleic acid sequence that encodes a polypeptide as claimed in<br>
claim 31,<br>
33.	A polypeptide comprising the FR3 portion of an amino acid<br>
sequence as claimed in claim 24.<br>
34.	A nucleic acid sequence that encodes a polypeptide as claimed in<br>
claim 33.<br>
35.	A polypeptide comprising the FR4 portion of an amino acid<br>
sequence as claimed in claim 24.<br>
36.	A nucleic acid sequence that encodes a polypeptide as claimed in<br>
claim 35.<br>
37.	A polypeptide comprising the CDR2 portion of an amino acid<br>
sequence as claimed in claim 24.<br>
38.	A nucleic acid sequence that encodes a polypeptide as claimed in<br>
claim 37.<br>
39.	A polypeptide comprising the CDR1 portion of an amino acid<br>
sequence as claimed in claim 24.<br>
40.	A nucleic acid sequence that encodes a polypeptide as claimed in<br>
claim 39.<br>
41.	A VH antibody fragment comprising at least one amino acid<br>
sequence selected from the group consisting of: SEQ ID NO:8-22.<br>
42.	A VL antibody fragment comprising at least one amino acid<br>
sequence selected from the group consisting of: SEQ ID NO:23-54.<br><br>
43.	A nucleic acid sequence that encodes a VH or VL antibody fragment<br>
as claimed in claim 41 or 42.<br>
44.	A VH antibody fragment comprising at least two amino acid<br>
sequences as claimed in claim 41.<br>
45.	A VL antibody fragment comprising at least two amino acid<br>
sequences as claimed in claim 42.<br>
46.	A nucleic acid sequence that encodes a VH or VL antibody fragment<br>
as claimed in claim 44 or 45.<br>
47.	A VH antibody fragment comprising at least three amino acid<br>
sequences as claimed in claim 41.<br>
48.	A VL antibody fragment comprising at least three amino acid<br>
sequences as claimed in claim 42.<br>
49.	A nucleic acid sequence that encodes a VH or VL antibody fragment<br>
as claimed in claim 47 or 48.<br>
50.	A multimer comprising at least two VH antibody fragments as<br>
claimed in claim 41, 44 or 47.<br>
51.	A multimer comprising at least two VL antibody fragments as<br>
claimed in claim 42, 45 or 48.<br>
52.	A multimer comprising at least one VH antibody fragment as claimed<br>
in claim 41, 44 or 47, and at least one VL antibody fragment as<br>
claimed in claim 42, 45 or 48.<br>
53.	A dimer comprising two VH antibody fragments as claimed in claim<br>
41, 44 or 47.<br><br>
54.	A dimer comprising two VL antibody fragments as claimed in claim<br>
42, 45 or 48.<br>
55.	A trimer comprising three VH antibody fragments as claimed in claim<br>
41.44	or 47.<br>
56.	A trimer comprising three VL antibody fragments as claimed in claim<br>
42.45	or 48.<br>
57.	A pentamer comprising five VH antibody fragments as claimed in<br>
claim 41, 44 or 47.<br>
58.	A pentamer comprising five VL antibody fragments as claimed in<br>
claim 42, 45 or 48.<br>
59.	A VH antibody fragment identified by the method of claim 1.<br>
60.	A VH antibody fragment as claimed in claim 59 that is of human<br>
origin.<br>
61.	A VH antibody fragment as claimed in claim 59 having an amino<br>
acid sequence that deviates from the corresponding parental<br>
germline sequence as shown in Table 1.<br>
62.	A VH antibody fragment as claimed in claim 59 belonging to the VH3<br>
family.<br>
63.	A VH antibody fragment as claimed in claim 59 belonging to a V<br>
germline selected from the group consisting of DP47, V3-49, DP-<br>
38, V3-53, YAC-5 and 8-1B.<br>
64.	A VH antibody fragment as claimed in claim 59 belonging to a J<br>
germline selected from the group consisting of JH4b, JH6c, JH3b,<br>
JH4, JH3a, and JH1.<br><br>
65.	A VH antibody fragment as claimed in claim 59 and having a residue<br>
other than valine at position 37 of its amino acid sequence.<br>
66.	A VH antibody fragment as claimed in claim 63 having a<br>
phenylalanine residue at position 37 of its amino acid sequence.<br>
67.	A VH antibody fragment as claimed in claim 63 having a tyrosine<br>
residue at position 37 of its amino acid sequence.<br>
68.	A VH antibody fragment as claimed in claim 59 having a mutation at<br>
a position selected from the group consisting of positions<br>
1,5,6,35,83.84,843 and 108 of its amino acid sequence.<br>
69.	A VH antibody fragment as claimed in claim 59 which binds to<br>
protein A.<br>
70.	A VL antibody fragment identified by the method of claim 1.<br>
71.	A VL antibody fragment as claimed in claim 70 that is of human<br>
origin.<br>
72.	A VL antibody fragment as claimed in claim 70 belonging to<br>
subgroup Vk1, VK3 or VA1.<br>
73.	A VL antibody fragment as claimed in claim 70 belonging to the<br>
kappa class.<br>
74.	A VL antibody fragment as claimed in claim 70 belonging to a V<br>
germline selected from the group consisting of L6, A27, L2, L16,<br>
O2/O12, A30 and 1b.<br><br>
75.     A VL antibody fragment as claimed in claim 70 belonging to a J germline selected from the group consisting of JK1, Jk4, Jk2, and<br>
76.	A VL antibody fragment as claimed in claim 70 having an arginine,<br>
proline, lysine, threonine, leucine, serine, tyrosine, glutamic acid,<br>
glutamine or histidine residue at position 96 of its amino acid<br>
sequence.<br>
77.	A VL antibody fragment as claimed in claim 70 having a charged<br>
amino acid residue at position 96 of its amino acid sequence.<br>
78.	A VL antibody fragment as claimed in claim 70 which binds to<br>
protein L<br>
79.	A VL antibody fragment as claimed in claim 70 having a threonine,<br>
valine or leucine amino acid residue at position 105, 106 and 107 of<br>
its amino acid sequence.<br>
80.	A polypeptide sequence identified by the method of claim 1 .<br>
81 .     A nucleic acid sequence that encodes the polypeptide sequence of claim 80.<br>
82.	A display library constructed using polypeptide sequences as<br>
claimed in claim 80.<br>
83.	A display library constructed using polypeptide sequences as<br>
claimed in claim 24, 59 or 70.<br>
84.	A display library constructed using VH or VL antibody fragments as<br>
claimed in claim 41 , 42, 44, 45, 47 or 48.<br><br>
85.	A display library constructed using polypeptide sequences as<br>
claimed in claims 27, 37 or 39.<br>
86.	A display library constructed using the loop (L1, L2, L3, H1, H2, or<br>
H3) regions of the polypeptides of claims 24, 59 or 70.<br>
87.	A display library as claimed in claim 82, 83, 84, 85 or 86, that is a<br>
phage display library.<br>
88.	A display library as claimed in claim 82, 83, 84, 85 or 86 that is a<br>
ribosome display, ARM ribosome display, yeast display, bacterial<br>
cell display or in vitro compartmentalization library.<br>
89.	A method for producing polypeptides with desirable biophysical<br>
properties, comprising the steps of<br><br>
a)	providing at least one first nucleic acid sequence that<br>
encodes an antibody fragment as claimed in claim<br>
41,42,44,45,47,48, 59 or 70 or that encodes a<br>
polypeptide sequence as claimed in claim 24,27,37 or<br>
39, and having a first desirable property;<br>
b)	providing at least one second nucleic acid sequence<br>
that encodes an antibody fragment having a second<br>
desirable property;<br>
c)	cutting the at least one first and at least one second<br>
nucleic acid sequences into random fragments;<br>
d)	reassembling the random fragments;<br>
e)	expressing the random fragments; and<br>
f)	screening the expressed random fragments for the first<br>
and second desirable properties.<br>
90.	A method as claimed in claim 89, wherein the first desirable<br>
property is selected from the group consisting of solubility, stability,<br>
monomertcity, high expression, human origin, and binding<br>
specificity.<br><br>
91.	A method as claimed in claim 89, wherein the second desirable<br>
property is selected from the group consisting of solubility, stability,<br>
monomericity, high expression, human origin and binding<br>
specificity.<br>
92.	A method as claimed in claim 89, where the at least one second<br>
antibody fragment is of non-human origin.<br>
93.	A method as claimed in claim 89, where the at least one second<br>
antibody fragment is of human origin.<br>
94.	A method as claimed in claim 89, where the at least one second<br>
antibody fragment is a human VH or VL.<br>
95.	A method as claimed in claim 89, where screening of the<br>
reassembled random fragments is accomplished by ribosome<br>
display, yeast display, phage display, bacterial cell display, ARM<br>
ribosome display or in vitro compartmentalization.<br>
96.	A pharmaceutical composition comprising the antibody fragment of<br>
claim 59 or 70, and a pharmaceutically suitable agent.<br>
97.	A pharmaceutical composition comprising the polypeptide<br>
sequence of claim 24 or 80, and a pharmaceutically suitable agent.<br>
98.	A recombinant vector comprising the nucleic acid sequence of claim<br>
26 or 81.<br>
99.	A host cell transformed with the recombinant vector of claim 98.<br></td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzM0NS1kZWxucC0yMDA3LUFic3RyYWN0LSgwMy0wMi0yMDE0KS5wZGY=" target="_blank" style="word-wrap:break-word;">7345-delnp-2007-Abstract-(03-02-2014).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzM0NS1kZWxucC0yMDA3LWFic3RyYWN0LnBkZg==" target="_blank" style="word-wrap:break-word;">7345-delnp-2007-abstract.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzM0NS1kZWxucC0yMDA3LUNsYWltcy0oMDMtMDItMjAxNCkucGRm" target="_blank" style="word-wrap:break-word;">7345-delnp-2007-Claims-(03-02-2014).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzM0NS1kZWxucC0yMDA3LWNsYWltcy5wZGY=" target="_blank" style="word-wrap:break-word;">7345-delnp-2007-claims.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzM0NS1kZWxucC0yMDA3LUNvcnJlc3BvbmRlbmNlIE90aGVycy0oMDItMDQtMjAxNCkucGRm" target="_blank" style="word-wrap:break-word;">7345-delnp-2007-Correspondence Others-(02-04-2014).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzM0NS1kZWxucC0yMDA3LUNvcnJlc3BvbmRlbmNlIE90aGVycy0oMDMtMDItMjAxNCkucGRm" target="_blank" style="word-wrap:break-word;">7345-delnp-2007-Correspondence Others-(03-02-2014).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzM0NS1ERUxOUC0yMDA3LUNvcnJlc3BvbmRlbmNlIE90aGVycy0oMDUtMDgtMjAxMSkucGRm" target="_blank" style="word-wrap:break-word;">7345-DELNP-2007-Correspondence Others-(05-08-2011).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzM0NS1kZWxucC0yMDA3LUNvcnJlc3BvbmRlbmNlIE90aGVycy0oMDctMTEtMjAxMikucGRm" target="_blank" style="word-wrap:break-word;">7345-delnp-2007-Correspondence Others-(07-11-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzM0NS1kZWxucC0yMDA3LUNvcnJlc3BvbmRlbmNlIE90aGVycy0oMTAtMDQtMjAxMykucGRm" target="_blank" style="word-wrap:break-word;">7345-delnp-2007-Correspondence Others-(10-04-2013).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzM0NS1kZWxucC0yMDA3LUNvcnJlc3BvbmRlbmNlIE90aGVycy0oMTctMDQtMjAxNCkucGRm" target="_blank" style="word-wrap:break-word;">7345-delnp-2007-Correspondence Others-(17-04-2014).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzM0NS1ERUxOUC0yMDA3LUNvcnJlc3BvbmRlbmNlIE90aGVycy0oMTktMDgtMjAxMSkucGRm" target="_blank" style="word-wrap:break-word;">7345-DELNP-2007-Correspondence Others-(19-08-2011).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzM0NS1kZWxucC0yMDA3LUNvcnJlc3BvbmRlbmNlIE90aGVycy0oMjAtMDEtMjAxNCkucGRm" target="_blank" style="word-wrap:break-word;">7345-delnp-2007-Correspondence Others-(20-01-2014).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzM0NS1ERUxOUC0yMDA3LUNvcnJlc3BvbmRlbmNlIE90aGVycy0oMjMtMDktMjAxMSkucGRm" target="_blank" style="word-wrap:break-word;">7345-DELNP-2007-Correspondence Others-(23-09-2011).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzM0NS1kZWxucC0yMDA3LUNvcnJlc3BvbmRlbmNlIE90aGVycy0oMjYtMDItMjAxMykucGRm" target="_blank" style="word-wrap:break-word;">7345-delnp-2007-Correspondence Others-(26-02-2013).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzM0NS1kZWxucC0yMDA3LUNvcnJlc3BvbmRlbmNlIE90aGVycy0oMjctMDEtMjAxNCkucGRm" target="_blank" style="word-wrap:break-word;">7345-delnp-2007-Correspondence Others-(27-01-2014).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzM0NS1ERUxOUC0yMDA3LUNvcnJlc3BvbmRlbmNlIE90aGVycy0oMzEtMDEtMjAxMikucGRm" target="_blank" style="word-wrap:break-word;">7345-DELNP-2007-Correspondence Others-(31-01-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzM0NS1kZWxucC0yMDA3LUNvcnJlc3BvbmRlbmNlLW90aGVycyAoMDgtMTEtMjAxMikucGRm" target="_blank" style="word-wrap:break-word;">7345-delnp-2007-Correspondence-others (08-11-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzM0NS1kZWxucC0yMDA3LUNvcnJlc3BvbmRlbmNlLU90aGVycy0oMDUtMDgtMjAxMykucGRm" target="_blank" style="word-wrap:break-word;">7345-delnp-2007-Correspondence-Others-(05-08-2013).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzM0NS1ERUxOUC0yMDA3LUNvcnJlc3BvbmRlbmNlLU90aGVycy0oMDctMTAtMjAwOSkucGRm" target="_blank" style="word-wrap:break-word;">7345-DELNP-2007-Correspondence-Others-(07-10-2009).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzM0NS1kZWxucC0yMDA3LUNvcnJlc3BvbmRlbmNlLU90aGVycy0oMDktMDItMjAwOSkucGRm" target="_blank" style="word-wrap:break-word;">7345-delnp-2007-Correspondence-Others-(09-02-2009).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzM0NS1kZWxucC0yMDA3LUNvcnJlc3BvbmRlbmNlLU90aGVycy0oMTktMDMtMjAwOCkucGRm" target="_blank" style="word-wrap:break-word;">7345-delnp-2007-Correspondence-Others-(19-03-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzM0NS1kZWxucC0yMDA3LUNvcnJlc3BvbmRlbmNlLU90aGVycy0oMjUtMDktMjAxNCkucGRm" target="_blank" style="word-wrap:break-word;">7345-delnp-2007-Correspondence-Others-(25-09-2014).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzM0NS1kZWxucC0yMDA3LWNvcnJlc3BvbmRlbmNlLW90aGVycy5wZGY=" target="_blank" style="word-wrap:break-word;">7345-delnp-2007-correspondence-others.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzM0NS1kZWxucC0yMDA3LWRlc2NyaXB0aW9uIChjb21wbGV0ZSkucGRm" target="_blank" style="word-wrap:break-word;">7345-delnp-2007-description (complete).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzM0NS1kZWxucC0yMDA3LWRyYXdpbmdzLnBkZg==" target="_blank" style="word-wrap:break-word;">7345-delnp-2007-drawings.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzM0NS1kZWxucC0yMDA3LUZvcm0tMS0oMDktMDItMjAwOSkucGRm" target="_blank" style="word-wrap:break-word;">7345-delnp-2007-Form-1-(09-02-2009).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzM0NS1kZWxucC0yMDA3LWZvcm0tMS5wZGY=" target="_blank" style="word-wrap:break-word;">7345-delnp-2007-form-1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzM0NS1ERUxOUC0yMDA3LUZvcm0tMTMtKDA1LTA4LTIwMTEpLnBkZg==" target="_blank" style="word-wrap:break-word;">7345-DELNP-2007-Form-13-(05-08-2011).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzM0NS1kZWxucC0yMDA3LUZvcm0tMTgtKDA5LTAyLTIwMDkpLnBkZg==" target="_blank" style="word-wrap:break-word;">7345-delnp-2007-Form-18-(09-02-2009).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzM0NS1kZWxucC0yMDA3LUZvcm0tMi0oMDMtMDItMjAxNCkucGRm" target="_blank" style="word-wrap:break-word;">7345-delnp-2007-Form-2-(03-02-2014).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzM0NS1kZWxucC0yMDA3LWZvcm0tMi5wZGY=" target="_blank" style="word-wrap:break-word;">7345-delnp-2007-form-2.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzM0NS1kZWxucC0yMDA3LUZvcm0tMy0oMDItMDQtMjAxNCkucGRm" target="_blank" style="word-wrap:break-word;">7345-delnp-2007-Form-3-(02-04-2014).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzM0NS1ERUxOUC0yMDA3LUZvcm0tMy0oMDctMTAtMjAwOSkucGRm" target="_blank" style="word-wrap:break-word;">7345-DELNP-2007-Form-3-(07-10-2009).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzM0NS1kZWxucC0yMDA3LUZvcm0tMy0oMDctMTEtMjAxMikucGRm" target="_blank" style="word-wrap:break-word;">7345-delnp-2007-Form-3-(07-11-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzM0NS1kZWxucC0yMDA3LUZvcm0tMy0oMTAtMDQtMjAxMykucGRm" target="_blank" style="word-wrap:break-word;">7345-delnp-2007-Form-3-(10-04-2013).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzM0NS1kZWxucC0yMDA3LUZvcm0tMy0oMTctMDQtMjAxNCkucGRm" target="_blank" style="word-wrap:break-word;">7345-delnp-2007-Form-3-(17-04-2014).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzM0NS1ERUxOUC0yMDA3LUZvcm0tMy0oMTktMDgtMjAxMSkucGRm" target="_blank" style="word-wrap:break-word;">7345-DELNP-2007-Form-3-(19-08-2011).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzM0NS1ERUxOUC0yMDA3LUZvcm0tMy0oMjMtMDktMjAxMSkucGRm" target="_blank" style="word-wrap:break-word;">7345-DELNP-2007-Form-3-(23-09-2011).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzM0NS1kZWxucC0yMDA3LUZvcm0tMy0oMjUtMDktMjAxNCkucGRm" target="_blank" style="word-wrap:break-word;">7345-delnp-2007-Form-3-(25-09-2014).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzM0NS1kZWxucC0yMDA3LUZvcm0tMy0oMjctMDEtMjAxNCkucGRm" target="_blank" style="word-wrap:break-word;">7345-delnp-2007-Form-3-(27-01-2014).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzM0NS1ERUxOUC0yMDA3LUZvcm0tMy0oMzEtMDEtMjAxMikucGRm" target="_blank" style="word-wrap:break-word;">7345-DELNP-2007-Form-3-(31-01-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzM0NS1kZWxucC0yMDA3LWZvcm0tMy5wZGY=" target="_blank" style="word-wrap:break-word;">7345-delnp-2007-form-3.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzM0NS1kZWxucC0yMDA3LWZvcm0tNS5wZGY=" target="_blank" style="word-wrap:break-word;">7345-delnp-2007-form-5.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzM0NS1kZWxucC0yMDA3LUdQQS0oMTktMTMtMjAwOCkucGRm" target="_blank" style="word-wrap:break-word;">7345-delnp-2007-GPA-(19-13-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzM0NS1kZWxucC0yMDA3LXBjdC0xMDEucGRm" target="_blank" style="word-wrap:break-word;">7345-delnp-2007-pct-101.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzM0NS1kZWxucC0yMDA3LXBjdC0yMTAucGRm" target="_blank" style="word-wrap:break-word;">7345-delnp-2007-pct-210.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzM0NS1kZWxucC0yMDA3LVBldGl0aW9uLTEzNy0oMjctMDEtMjAxNCkucGRm" target="_blank" style="word-wrap:break-word;">7345-delnp-2007-Petition-137-(27-01-2014).pdf</a></p>
		<br>
		<div class="pull-left">
			<a href="264263-method-and-network-controller-for-providing-point-to-multipoint-service-to-at-least-one-terminal-in-radio-communications-system-and-terminal-thereof.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="264265-a-method-of-synthesizing-a-substantially-monodispersed-mixture-of-oligomers.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>264264</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>7345/DELNP/2007</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>51/2014</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>19-Dec-2014</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>18-Dec-2014</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>24-Sep-2007</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>NATIONAL RESEARCH COUNCIL OF CANADA</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td>1200 MONTREAL ROAD, M-58, EG-06B, OTTAWA, ONTARIO, K1A 0R6,CANADA.</td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>TANHA, JAMSHID</td>
											<td>6292 SABLEWOOD PLACE, ORLEANS, ONTARIO K1C 7M6,CANADA</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>C12N 15/13</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>PCT/CA2006/000451</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td>2006-03-24</td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td>60/664,954</td>
									<td>2005-03-25</td>
								    <td>U.S.A.</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/264264-a-method-of-identifying-a-stable-vh-and-or-a-vl-antibody-fragment by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 00:31:36 GMT -->
</html>
