<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/264246-complexes-and-methods-for-determining-cell-culture-history by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 00:26:13 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 264246:&quot;COMPLEXES AND METHODS FOR DETERMINING CELL CULTURE HISTORY&quot;</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">&quot;COMPLEXES AND METHODS FOR DETERMINING CELL CULTURE HISTORY&quot;</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>The present invention relates in one aspect to a method for determining the cell culture history of a cell unit labelled with more than one type of tag comprising the steps of: (a) measuring one or more parameters of each tag that is used to label the cell unit; (b) identifying each tag in the cell unit; and (c) correlating the identity of each tag to the identity of the cell unit and/or the specific cell culture conditions to which the cell unit has been exposed.</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td>Method Field of the invention<br>
The invention broadly relates to cell culture - such as the culture of primary cells, cell lines, pluripotent cells, totipotent cells and stem cells.<br>
Background to the invention<br>
Over recent years cell culture "has become a core technology in the life sciences. Cell culture is described in 'Basic Cell Culture' Oxford University Press (2002) Ed. J.M. Davis; and 'Animal Cell Culture' Oxford University Press (2000) Ed. John R. W. Masters; both of which are incorporated herein in their entirety by reference. Cell culture provides the basis for studying cellular processes such as the viability, phenotype, genotype, proliferation and differentiation of cells, and the formation of biological molecules, intermediates and products. It has also provided the means to study the regulation of these -processes, from the genetic level - whether in isolation or in whole transgenic animals - down to the level of individual protein molecules. Notwithstanding its enormous contribution-to the current state of biology, in many respects cell culture remains a developing discipline, albeit an unusually exciting science ultimately offering the possibility of genetic therapy and tissue engineering.<br>
An important goal of cell culture is to be able to grow a wide variety of cells in vitro. The list of dlfferentcell-types that can be grown in culture is-extensive (see American-Type Culture collection, http://www.atcc.org: European Collection of Cell Cultures, -htto^/www.ecacc.org.uk: Deutsche Sammlung von Mikroorganismen- und Zellkulturerr GmbK", http://www.dsmz.de), Includes representatives of most celi types,, and is continually-increasing as-more and more culture conditions, are discovered.-Despite the •steady progress in the field, the method of determining suitable culture conditions for new ce» types remains totally empirical: growth conditions are almost always discovered by trial and error. The choice of starting point will often be based on what was previously used by others for similar cells, or even what is currently being used in the laboratory for different cells. Many times these will simply be completely inadequate, and a process of trial and error must begin anew. Even when new culture conditions are successful, it is worthwhile remembering that adaptations of previous protocols will have introduced a historical bias to the experiment. For instance, much of the early tissue culture<br>
experiments made extensive use of fibroblasts, and to this date most standard cell culture conditions favour growth of cells derived from the mesoderm (fibroblasts, endothelium, myoblasts). The development of selective growth media for epithelial and other cell types based on these conditions was a challenge. For some of these cell types it is now known that serum - a normal component of many culture media for mesodermal cells - actually inhibits growth. One aspect of the invention described herein is a method for developing suitable culture conditions which allow for the viability, proliferation or growth, and retention of the phenotype of particular cell types.<br>
Some common problems which are still encountered in cell culture are the limited lifespan. of primary cell lines, the- change of characteristics of cell lines with passage, and their transformation accompanied by loss of interesting cellular characteristics. These effects severely limit the utility of cultured cells for use in experiments or assays, for instance cell-based assays described below: Primary cells, i.e. cells freshly isolated from tissues, offer by far the most accurate cell culture -models, as they behave in a way that broadly resembles their tissue of origin. Remarkably, a reliable method of culturing primary cells has still not been developed and consequently-these cells, exhibit a limited lifespan 'm vitro. This presents a-serious technicanimitation, for instanc&amp; when attempting to- amplify the primary culture, or when attempting to perform a longer-term experiment. A further problem associate* with-the use of primary cultures is that since they require constant fresh isolation, it can 1be "hard to source primary material, particularly from humans and it is also difficulito obtain lines that behave consistently. A third aspect of the invention is therefore a method of culturing primary cells to obtain viable cultures with a prolonged lifespan.<br>
ff primary cultures are maintained In vitro for an extended period, they normally undergo a crisis in which the majority of cells perish-, however the surviving celts are longer-lived and can-be cultured indefinitely: Most ofthese-continuous cell lines-are almost invariably^ poor representations of the cell as it is found in intact animal tissues. One reason for this lies in the fact that the process that allows the cells to become immortal also has an impact on the characteristics of the cell. For example, most established cell cultures have stopped expressing tissue-specific genes and instead only-express housekeeping genes required for continuous growth in cell culture - as a result most such cell lines are more like each other than like the tissue from which they were originally sourced. For instance, most liver cell lines have stopped expressing the drug-metabolizing enzymes that would normally make them interesting tools for testing drug toxicity. A further aspect<br>
of the invention described herein is a method of culturing cells so that they provide more accurate models of tissues. This in turn would improve the reliability and predictive power of cell-based experiments and assays.<br>
Improved techniques for culturing cells and methods for discovering and implementing such techniques for regulation of cellular processes such as growth, differentiation, metabolic activity, and phenotypic expression are presented in our co-pending international application WO 20047031369. When handling large numbers of cell units, their identity and/or cell culture history (for example, the chronology and the exact nature of-a series of culture conditions-that any one-group .or unit- may have .been exposed to) car* become confused. W02004/031369 describes improved methods for determining the identity and/or cell culture history of cell units. In one aspect there is described the use of cell units which can be handled conveniently in cell biology experiments, enabling for example the splitting and pooling of said units.<br>
The present invention seeks to provide further improvements which overcome some of the limitations of the prior art.<br>
Summary of the Invention<br>
In a first aspect, there is provided a method for determining the cell culture history of a cell unit labelled .with morathan one tag (eg. type of tag)-comprising the steps of: (a) measuring one or more parameters of each tag that is used to label the cell unit; (b) identifying -each tag in -the^cell -uriit; and (c) correlating the identity erf each-tag io the identity of the cell unit and/or the specific cell culture conditions to which the cell unit has been exposed.<br>
In a further aspect, there is provided-a computer program product including-a computer program for controlling a computer to perform the method described herein.<br>
In a further aspect, there is provided an apparatus for determining the cell culture history of a cell unit comprising data processing logic operable to perform data processing operations in accordance with the method described herein.<br>
In a further aspect, there is provided a complex comprising a microcarrier and a charged (eg. negatively charged) tag.<br>
In a further aspect, there is provided a complex, comprising a microcarrier and a rod-shaped tag.<br>
The qualities of the tags that are described herein can advantageously be utilised in conjunction with the improvements that are disclosed herein. Such improvements comprise for example the improved labelling of specific types of tag and specific types of cell unit, the improved separation of tags from cell units and the improved analysis of tags.<br>
In- a-further aspect, there is provided -a method for separating a complex comprising a microcarrier and a tag, comprising the step of contacting said complex with a protease, wherein said microcarrier comprises, consists or consists essentially of protein.<br>
la a further aspect, there is provided a method for separating a complex comprising a microcarrier and a tag, comprising the step of contacting said complex with acid.<br>
Advantageously, these methods can be used to separate a cell unit and a tag in such a way that the. tag can be analysed using the methods described herein. Moreover, these methods do not resoJt in tags that are- spoiled by^such treatment. Stilj further, these methods address the problem of the tags being obtained in a manner which seriously complicates-their analysis - such as the tags-becoming dispersed over a large surface, area and/or floating on a denser aqueous solution.<br>
In a further aspect, there is provided a method for determining the effect of a plurality of culture conditions on a cell comprising the use of the complex described herein.<br>
In a further aspect, -there is provided a method for deteimining-tfie effect of a plurality of culture conditions on a cell, comprising the steps of: (a) providing a first set of groups of. ceH units each comprising one or more cells, and exposing said groups to desired culture conditions; (b) pooling two or more of said groups to form at least one second pool; (c) subdividing the second pool to create a further set of groups of cell units; (d) exposing said further groups to desired culture conditions; (e) optionally, repeating steps (b) - (d) iteratively as required; and (f) assessing the effect on a given cell unit of the culture conditions to which it has been exposed,<br>
wherein each cell unit comprises one or more cells adherent to or bounded by the complex described herein.<br>
In a further aspect, there is provided a method for exposing a cell to a variety of cell culture conditions, comprising the steps of: (a) providing a first set of groups of cell units each comprising one or more cells, and exposing said groups to desired culture conditions; (b) pooling two or more of said groups to form at least one second pool; (c) subdividing the second pool to create a further set of groups of cell units; (d) exposing said further groups to desired culture conditions; and (e) optionally, repeating steps (b) -(d) iteratively as-required, wherein each cell unit comprises one OF more cells adherent-to or bounded by the complex described herein.<br>
In a further aspect, there is provided a method for determining the effect of a plurality of culture coRditions on a cell, comprising the steps of: {a) providing a first set of groups of cell units each-comprising one or more cells, and exposing said groups to desired culture conditions; (b)' pooling two or more of said groups to form at least one second pool; (c) subdividing the second- pool to. create a further set of groups of cell units; (d) exposing said further- groups to desired culture conditions; (e) optionally, repeating steps {b) - (d) iteratively as required; and (f) assessing the effect on a given cell unit of- the culture conditions to which ittias been exposed,-wherein each cell unit comprises one or more cells adherent to or bounded-by the complex described herein.<br>
In a further aspect, there is provided a method for identifying a gene which influences a cellular-process, comprising the-steps of: a) determining the effect of one or more culture conditions on a cell unit, in accordance with the methods described herein; b) analysing gene expression in said cell units when exposed to -said culture conditions; and* c) identifying-genes which are differentially expressed under desired -culture-conditions.<br>
In a further aspect, there is provided a method for producing a nucleic acid which encodes a gene product which influences a cellular process, comprising identifying a gene in accordance with the method described herein, and producing at least the coding region of said gene by nucleic acid synthesis or biological replication.<br>
In a further aspect, there is provided a method for inducing a cellular process, comprising the steps of: (a) identifying one or more genes which are differentially expressed in<br>
association with the cellular process in accordance with the method described herein; and (b) modulating the expression of said one or more genes in the cell.<br>
In a further aspect, there is provided" a method for identifying the state of a cellular process of a cell, comprising the steps of: (a) identifying one or more genes which are differentially expressed in association with the cellular process in accordance with the method described herein; and (b) detecting the modulation of expression of said one or more genes in a cell, thereby determining the state of the cellular process of said cell.<br>
In a further aspect, therels provided a method for inducing a~ cellular-process, comprising the- steps of: (a) determining the effect of one-or more calture conditions on a cell unit, in accordance with the methods described herein; (b) exposing a cell to culture conditions which Induce the cellular process; and (c) isolating the desired cell.<br>
In a further aspect, there is provided a method for identifying an agent which is capable of inducing a cellular process, comprising -the steps of: (a) determining the effect of one or more agents on a cell unit, in accordance with the methods described herein; and (b) identifying those agent(s) which induce the cellular process in the cell'units.<br>
fn a further aspect, there is provided a method for-preparing an agent which is-capable of inducing a cellular process, comprising the steps of: (a) determining the effect of one or more agents on a cell unit, in accordance with the methods described herein; (b) identifying those agent(s) which induce the desired cellular process in the cell units; and c) synthesising or isolating the agents).<br>
In a further aspect, thece is provided a method for cutturing stem cells or cells ihat have been derived'frora stem cells In vitro comprising, the steps of: a) incubating a stem ceil culture; and-b) splitting said culture into two or more groups of stem, cells; and-culturing said group of stem cells under two or more different sets of culture conditions, wherein the-cells are cultured in cell units, each cell unit comprising one or more-cells adhered to or bounded by the complex described herein.<br>
In a further aspect, there is provided a method for culturing stem cells, comprising growing said stem cells adhered to the complex described herein.<br>
In a further aspect, there is provided a method for obtaining differentiated cells from stem cells in vitro, comprising the steps of: (a) growing stem cells adherent to the complex described herein in a culture medium; (b) transferring the complex from one culture medium to another; (c) optionally repeating step (b) as required; and (d) obtaining the differentiated cells attached to the complex.<br>
In a further aspect, there is provided a method of growing pluripotent stem cells in vitro comprising the steps of: (a) seeding said cells on the complex described herein; and (b) propagating the cells while attached to the complex.<br>
In a further aspect, there is provided a method ior culturing cells in vitro or in vitro, comprising growing said cells adhered to the complex described herein<br>
In a further aspect, there is provided a method for identifying a tag obtained or obtainable from a cell unit comprising the steps of: (a) separating the cell unit and the tag; (b) obtaining one or more images of the tag using a microscopic technique; and (c) analysing, the-images to determine one or more features of the tag.<br>
Irt a further aspect,, there is provided the use of the -complex described herein for determining the effect of a plurality of culture conditions <jrra cell.></jrra>
In a further aspect, there is provided thause of a rod-shaped tag - such-as a nanowife to label a microcarrier.<br>
In a further aspect, there is provided the use of a rod-shaped tag - such as a nanowire for determining the effect of-a plurality of culture conditions on a cell.<br>
In a further aspect, there-is provided- a -.method, a complex, a computer program, an apparatus or a use substantially as described herein with reference to the accompanying figures.<br>
Embodiments<br>
In some embodiments, the method is an automated method.<br>
In some embodiments, the cell unit is bound or adhered to a microcarrier.<br>
In some embodiments, the microcarrier is a porous or a solid microcarrier.<br>
In some embodiments, the porous microcarrier is selected from the group consisting of Cytopore microcarrier (eg. a Cytopore 1 microcarrier or a Cytopore 2 microcarrier), a Cultispher microcarrier, a Cultispher-G microcarrier, a Cultispher-GL microcarrier and a Cultispher-S microcarrier, an Irtformatrix microcarrier, a Microsphere microcarrier, a Siran microcarrier, and a Microporous MC microcarrier.<br>
In some embodiments, the solid microcarrier is selected from the group consisting of a Cytodex microcarrier (eg. a Cytodex 1, Cytodex 2 or Cytodex- 3 microcarrier) a Biosilon microcarrier, a Bioglass microcarrier, a FACT III microcarrier or a DE 52/53 microcarrier.<br>
In some embodiments, the parameters) are the size of the tags and/or the optical properties of the tags,<br>
In some embodiments, the optical properties are selected from the group consisting of: light reflectivity, colour, the fluorescence emission- wavelength(s) and the fluorescence emission intensity.<br>
In some embodiments, one or more image(s!) of each tag in the cell unit in the field of interest is measured.<br>
In some embodiments,-ene or-more image(s) of each tag In the cell, unit in the field of interest is measured using microscopy.<br>
In some embodiments, the microscopy method is selected-fro "the group consisting'of bright field mieroscopy.-phase-contrast-microscopy-, oblique illumination=microscopy, dark field microscopy, differential interference contrast microscopy, reflection contrast microscopy, varel contrast microscopy, polarizing microscopy, interference microscopy and fluorescence microscopy.<br>
In some embodiments, outlines are drawn for the one or more image(s) of each tag In the cell unit in the field of interest.<br>
In some embodiments, one or more fluorescence image(s) of each tag in the cell unit in the field of interest is measured.<br>
In some embodiments, outlines for the one or more image(s) are loaded onto the one or more fluorescence image(s).<br>
In some embodiments, one or more fluorescence image(s) of each tag in the cell unit in the field of interest is measured for each fluorophore that is used to label the tags.<br>
In some embodiments, «ach tag in the eeir-unit is identified-by reading the one or more par-ameters-of the tag within the one or more, outlines.<br>
In some embodiments, the one or more parameters of the tag are the area and/or optical density of the tag.<br>
In some embodiments, the one or more: parameters of the tag are inputted into a spreadsheet.<br>
-In some- embodiments, the fluorophore is selected from the group consisting of a fluorophore that-emits a blue, green,, near red-or far red fluorescence.<br>
In some embodiments, where two or more fluorophores are used, the fluorophores do not quench each other.<br>
In some embodiments, the sizes are selected from at least 3 different sizes - such as 4, 5,6 or 7 different sizes.<br>
In some embodiments, the sizes are selected from the group consisting of about 1-.9|im, about 4.4um, about 5.4um, about 5.8nm, about 7.4um, about 9.7um and about 9.8unr<br>
In some embodiments, the fluorophore is selected from the group consisting of UV2, Starfire Red and TRITC.<br>
In some embodiments, the quantity of fluorophore is selected from 5 different quantities.<br>
In some embodiments, these quantities are indicated by ranges (eg. ranges of fluorescence intensity).<br>
In some embodiments, each different quantity gives about a 5 to 10-fold difference in brightness.<br>
In some embodiments, there are at least 2 different parameters. In some embodiments, there are at least 5 different integers. In some embodiments, the microcarrier is a porous.microcarrier. In some embodiments, the microcarrier has a net charge.<br>
In some embodiments, the microcarrier comprises, consists or consists essentially of protein, cellulose, polyethylene, polystyrol, glass, collagen, collagen-gylcose-aminoglycan and/or gelatin.<br>
In some-embodiments, the tag has a charge.<br>
In some embodiments, the tag has a net charge.<br>
In some embodiments, the tag has a negative charge.<br>
In some embodiments, the charged tag is a sphere.<br>
In "some embodiments, ttie sphere is-a microsphere.<br>
In some embodiments, the microsphere is about 9 uM or less in diameter.<br>
In some embodiments, the microsphere is a carboxylate modified (CML) microsphere.<br>
In some embodiments, the tag comprises, consists or consists essentially of polystyrene.<br>
In some embodiments, the complex (eg. the microcarrier) is adhered or bound to a cell unit.<br>
In some embodiments, at least one antibody is bound to the cell unit.<br>
In-some embodiments-, the rod-shaped tag-is a nanowire.<br>
In some embodiments, the nanowire is an aluminium nanowire.<br>
In some embodiments, the nanowire is coated with silver and/or gold.<br>
In some embodiments, the nanowire-is about 1uM or less in diameter-..<br>
In some embodiments, the nanowire is about 10 uM or less in length.<br>
In some embodiments, the microcam'er is a porous microcarder.<br>
-In-some embodiments, the porous microcarrier is a charge neutral rrricrocarrier.<br>
In some embodiments, at least ona antibody is bound to the cell unit.<br>
In some embodiments, the microcarrier -comprises, consists or consists essentially of rgelatin.<br>
In some embodiments, the microcarrier is a Cultispher microcarrier.<br>
'In some embodiments, the microcarrier is selected from the group consisting.of a Cultispher-G microcarrier, a Cultispher-GL microcarrier and Cultispher-S microcarrier.<br>
In some embodiments, tha protease-is setected from the group consisting of proteinase K, typsin, thermoiysin and caspase.<br>
In some embodiments, the proteinase K is used in an amount of about 0.5U/ml or less.<br>
In some embodiments, the complex is contacted with proteinase K for at least 20-60 minutes.<br>
In some embodiments, the complex is contacted with the protease in a volume of about 5 or less.<br>
In some embodiments, the microcarrier comprises, consists or consists essentially of cellulose.<br>
In some embodiments, the microcarrier is a Cytopore microcamer. in some, embodiments, the microcarrier is a Cytopore 2 microcarrier. In some embodiments, the tag is a sphere.<br>
In some embodiments, the acid is selected from the group consisting of hydrochloric acid, sulphuric acid and sodium. hypocMortte.<br>
In some embodiments, the add is 37% hydrochloric acid (-12M). In some embodiments, the acid is concentrated sulphuric acid.<br>
In some embodiments, the complex is contacted'with acid in -a volume of about 5 ul or less.<br>
In some embodiments, the complex is heatedJnlhe presence of the acid.<br>
In some embodiments, the cells are cultured in cell units; each cell unit comprising one or more cells.<br>
In some embodiments, the cell-units are single ceils.<br>
In some embodiments, the culture conditions are media to which the cell is exposed.<br>
In some embodiments, the media contain one or more specific agents which influence a cellular process.<br>
In some embodiments, the cell culture conditions comprise culturing at one or more specific temperatures.<br>
In some embodiments, the cell culture conditions comprise culturing on one or more specific substrates.<br>
In some embodiments, the desired culture conditions influence a cellular process.<br>
In some embodiments, modulation of gene expression in the cell comprises transfection of said one or more genes into the cell.<br>
In some embodiments, modulation of gene expression comprises the exogenous administration of a gene product<br>
In some embodiments, said one or more genes encode a marker. In some embodiments, said marker may be detected by an immunoassay. In some embodiments, the cellular process is cell proliferation or-differentiation. In some embodiments, the cell units are single-cells.<br>
In some embodiments, said stem cells are subjected to at least one change of culture conditions.<br>
In- some embodiments, said change of- culture -conditions comprises a change of medium.<br>
Jn some embodiments, the differentiated cells are- isolated- by enzymatic or chemical detachment from the-complex.<br>
In some embodiments, the differentiated cells are isolated by digestion of the complex: Description of the Figures<br>
Figure 1 shows a sample of coloured Duke microbeads viewed under light microscopy with Black, Red and Blue tags in two different sizes (5.4um and 7.6pm).<br>
Figure 2 shows Cultispher-G microcarriers trapped on a 70pm filter during washes to remove unbound, microbeads (size range 4.4um - 9.8um), which pass through.<br>
Figure 3 shows the digestion of Cultispher-G microcarriers with 2U/ml proteinase k (figures show the same microcarrier over a 15 minute period, as indicated).<br>
Figure 4 shows the digestion of Cytopore 2 microcarriers with 37% HCl (-12M) (figures show the same microcarrier over a 1 hr 30m period, as indicated)<br>
Figure. 5 shows an Imagepro Tag Analysis flowchart for microscope analysis to identify one or more area and/or optical density values for a microbead. Images are taken using various exposures and filters: UV2 filter: Ex 340-380nm; Em 435-485nm; TRITC filter Ex 540/25nm; Em 605/55 nm; Starfire Red: Ex 620/60nm; Em 700/75nm.<br>
Figure 6 shows a Tag ID Flowchart in which the microbeads (tags) are of 2 different sizes, each with 4 levels of fluorophore A ± fluoroptiore B. Fluorophore A Is UV2 and FJurorophore B is-TRITC. F1 - mean optical density measurement for tag -using UV2 channel; F2 - mean optical density measurement for tag- using Starfire Red channel; F3 -mean optical- density- measurement for tag using TRITC channel; ft - lower limit for fluorescence of tag 1; f2 - upper limit for fluorescence-of tag 1; f3 - lower threshold for TRTTC fluorescence of all tags; f4 - lower limit for fluorescence of tag 2; f5 - upper limit Tor fluorescence of-tag 2 etc;. Area - area measurement; a1 - lower area limit for-tag of size 1; a2 - upper area limit for tag of size 1; a3 - lower area limit for tag of size 2; a4 -upper-area limit for tag of size 2 etc; Y~= yes; N = no.<br>
In further detail, box (a) shows that the lower limit for a first parameter is less than the mean measurement for a- first parameter and the upper limit for a first parameter is greater than, the mean measurement-for a first parameter; box (b) shows-that the lower-limit for a second parameter is less than the mean measurement for a second parameter and the upper-limit for a second parameter is greater than the mean measurement for a second parameter, boxes (c), (f) (i), (k), (o), (q), (t) and (v) each show that the mean measurement for a third parameter is greater than the lower limit for the third parameter; box (e) shows that if the result from box (b) is negative then the upper, lower and mean measurements for a further integer of a parameter are measured. In particular, box (e) shows that the lower area limit for an integer of a parameter is less than the mean measurement of the parameter and that the upper area limit for the integer of the<br>
parameter is greater than the mean measurement of the parameter. This is repeated for further integers of a parameter in boxes (h), (j), (n), (p), (s) and (u).<br>
Figure 7 shows an alternative flowchart in which one or more integers of a parameter are examined serially, such that the final 'No' decisions from Figure 6 will go on to interrogate those integers for a further parameter. Fluorophore A is Starfire Red and Flurorophore B is TRITC. For example, F1 and F2 may be examined serially, such that the final 'No' decisions from Figure 6 will go on to interrogate the Starfire Red values for the relevant size tag. F1 - mean density measurement for UV2 channel; F2 - mean density -measurement for Starfire Red channel; F3 - mean density measurement for TRITC channel; f18 - lower limit for fluorescence of tag 9; f19 - upper limit for fluorescence of tag 9; f3 - lower threshold for TRITC fluorescence of all tagsl ; Y = yes; N=no.<br>
Figure 8 shows a macroporeus CultiSpher-G gelatine microcam'ers (Percell Biolytica AB) seeded with mouse -ES cells and stained with a kit showing alkaline phosphatase activity, a marker of plur-ipotency,<br>
Figure 9 shows a -sample of the* 140,000 microcarriers processed by Combinatorial Cell culture through "the matrix shown in Table 1 and incubated-on DT3 with the macrophage assay reagent DQ-ovalbumin (Molecular Probes). Top left panel shows a phase contrast fmage of the miCFoearriers. The top right image shows the same field using a FITC filter set, whereby macrophages are easily distinguishable as large, round cells internally labelled with gr-een fluorescence. The .bottom left image shows: the same field. -using a FITC filter set, revealing UV2-loaded tags associated with the microcarriers.<br>
is as for Figure 9 buffer a different field.<br>
Figure 11 represents graphs showing the number and identity of all tags found on the microphage-bearing microcarriers a) C5, b) A22 and e) E6. The bars marked with an asterisk in b) and c) denote the conditions which were later determined to result in reproducible macrophage differentiation.<br>
Figure 12 is an example of a microcarrier bearing large number of macrophage stained with DQ-ovalbumin (Molecular Probes).<br>
Figure 13 is a graphical representation of the 50 microsphere tags commissioned from Bangs Laboratories (Fishers, IN). The microspheres were sized between 1.87pm and 9.77(im and were dyed with five distinguishable intensities of the fluorophore Starfire Red or alternatively UV2.<br>
Figure 14 is a calibration sample of Starfire Red dyed microsphere tags analysed according to tag size and fluorescence intensity at a given exposure setting. A similar calibration sample was used to determine parameters (denoted by ovals) for the classification of tags bound to microcarriers.<br>
Figure 15 shows quality control- images of rod-shaped particles used in the experiment, obtained using illumination at 430nm and a 63X objective. Left image 101010; Right image 100001 (where 1 = Ag and 0 = Au)<br>
Figure 16 is an image of a labelled cell unit stained with neutral red vital stain to reveal living cells and showing associated rod-shaped particles -(indicated with arrows), obtained using bright field illumination and a 20X objective.<br>
Figure 17 is an image-of rod-shaped particles liberated following.proteinase K digestion of a labelled cell unit, obtained, using bright field illumination and. a 20X objective.<br>
Figure 18 is-an image-showing discrimination of two different-particles liberated following-proteinaseK digestion of a labelled cell unit, obtained using a CFP filter set (Ex. 436/10; Em. 465/30) with a-100x oit immersion objective. The particle on-the-left-is 100001 and the particle on the right is 101010.<br>
Petal led" Description of-the^lnvention Definitions<br>
Culture Conditions As used herein, the term "culture conditions" refers to the environment which cells are placed in or are exposed to in order to promote growth or differentiation of said cells. Thus, the term refers to the medium, temperature, atmospheric conditions, substrate, stirring conditions and the like which may affect the growth and/ or differentiation of cells. More particularly, the term refers to specific agents<br>
which may be incorporated into culture media and which may influence the growth and/or differentiation of cells.<br>
-Cell A cell, as referred to herein, is defined as the smallest structural unit of an organism that is capable of independent functioning, or a single-celled organism, consisting of one or more nuclei, cytoplasm, and various organelles, all surrounded by a semipermeable cell membrane or cell wall. The cell may be prokaryotic, eukaryotic or archaebacterial. For example, the cell may be a eukaryotic cell. Mammalian cells are preferred, especially human cells. Cells may be natural or modified, such as by genetic manipulation or passaging in culture, to achieve desired properties. A stem cell- is defified in more detail-below, and is a totipotent, pluripotent or multipotentcell capable of giving rise to more than one differentiated cell type. Stem cells may be differentiated in vitro to give rise to differentiated cells, which may themselves be multipotent, or may be terminally differentiated. Cells differentiated in vitro are cells which have been created artificially by exposing stem cells to one or more agents which promote cell differentiation.<br>
Cellular process A cellular process is -any characteristic, function, process-, event, cause or effect, intracellular or extracellular, which-occurs or is observed or which can be attributed to a. -cell. -Examples of cellular processes include, but are not limited- tor, viability, senescence, death, pluripotency, morphology, signalling, binding, recognition,, molecule production or destruction (degradation), mutation, protein folding, transcription, translation, catalysis, synaptic transmission, vesicular transport, organelle function, cell cycle, metabolism, proliferation, division, differentiation, phenotype, genotype, gene expression, or the control of these processes.<br>
Cell-unit A group- of cells, which may be a group of. ORB. Pools of cell units may be sorted, subdivided and-- handled without substantially -dissociating the cell units themselves, such that the cell unit behaves as a colony of cells and each cell in the cell unit is exposed to the same culture conditions. For some embodiments, a cell unit may comprise a microcarrier or bead to which is adhered a group of cells.<br>
Totipotent A totipotent cell is a cell with the potential to differentiate into any type of somatic or germ cell found in the organism. Thus, any desired cell may be derived, by some means, from a totipotent cell.<br>
PIuripotent A pluripotent cell is a cell which may differentiate into more than one, but not all, cell types.<br>
Tag In one aspect, the terra "tag", as used herein, refers to any tag that is used to identify a cell unit and/or determine a culture condition, or a sequence of culture conditions, to which the cell unit has been exposed. In another aspect, the term "tag" refers to any tag that is added to a cell unit as a means of specifically labelling that cell unit, thus facilitating the identification of a cell unit and/or the determination of a culture condition and/or a sequence of culture conditions, to which the cell unit has been exposed'. Suitably, the tag-exists in a number of related but distinct variants which are easily distinguishable, as described in further detail-herein. The tag typically-forms part of a complex with a microcarrier - such as a Cultispher-G microcarrier or a Cytopore 2 microcarrier. For some embodiments, the tag is a sphere or a bead - such as a microsphere or a microbead. For some embodiments the tag is a rod--shaped particle -such, as a nanowire. As referred to herein, the term "tag" is synonymous with the term "label".<br>
Exposure to culture conditions A cell is exposed to culture conditions when it is placed in -contact with a medium, or. grown under conditions which affect one- or mor&amp; cellular process(es) such as "the growth, -differentiation, or metabolic state--df the cell. Thus, ifthe .culture conditions comprise culturing the cell in a medium, the cell is .placed in the-*nedium for a sufficient period of time for it to. have an effect. Likewise, if the conditions are temperature conditions, the cells are cultured at the desired temperature.<br>
Pooling The pooling of one or more groups of cell units involves the admixture of the groups~to create a-single group or pool which comprises cell units of more than-one backgroundrthat is, that have been exposed to more than-one different sets-of culture conditions. A pool may be subdivided further into groups, either randomly or non'-randomly; such groups are not themselves "pools" for the present purposes, but may themselves be pooled by combination, for example after exposure to different sets of culture conditions.<br>
Proliferation Cell growth and cell proliferation are used interchangeably herein to denote multiplication of cell numbers without differentiation into different cell types or lineages. In other words, the terms denote increase of viable cell numbers. In some<br>
embodiments, proliferation is not accompanied by appreciable changes in phenotype or genotype.<br>
Differentiation Cell differentiation is the development, from a cell type, of a different cell type. For example, a bipotent, pluripotent or totipotent cell may differentiate into a neural cell. Differentiation may be accompanied by proliferation, or may be independent thereof. The term 'differentiation' generally refers to the acquisition of a phenotype of a mature cell type from a less developmental^ defined cell type, e.g. a neuron, or a lymphocyte, but does not preclude transdifferentiation, whereby one mature cell type may convert to another mature cell type-e-.g. a neuron to-a- lymphocyte.<br>
Differentiation state The differentiation state of a cell is the level to which a cell has differentiated along a particular pathway or lineage.<br>
State of a cellular process The state of a cellular process refers to whether a cellular process is occurring or not and in complex cellular processes can denote a- particular step or stage in that cellular process. For example, a cellular differentiation pathway in a cell may be inactive or may have been induced and may comprise a- number of discrete steps or components such as signalling events -characterised .by the presence of a characteristic set of enzymes or intermediates.<br>
Gene A gene is a nucleic acid which encodes a gene product, be-it a polypeptide or an RNA gene product. As used herein, a gene includes at least the coding sequence which -encodes the gene product; it mayr optionally, include-one or more recjulatory -regions necessary for the transcription and/or translation of the coding sequence.<br>
"TGene Frottact A Tjene product is typically a protein encoded by "a gene -in the conventional manner. -However, the term also encompasses non-polypeptide -gene products, such as ribonucleic adds, which are encoded by the gene.<br>
Nucleic acid synthesis Nucleic acids may be synthesised according to any available technique. In some embodiments, nucleic acid synthesis is automated. Moreover, nucleic acids may be produced by biological replication, such as by cloning and replication in bacterial or eukaryotic cells, according to procedures known in the art.<br>
Differential Expression Genes which are expressed at different levels in response to cell culture conditions can be identified by gene expression analysis, such as on a gene array, by methods known in the art Genes which are differentially expressed display a greater or lesser quantity of mRNA or gene product in the cell under the test conditions than under alternative conditions, relative to overall gene expression levels.<br>
Transfection Genes may be transfected into cells by any appropriate means. The term is used herein to signify conventional transfection, for example using calcium phosphate, but also to include other techniques for transferring nucleic acids into a cell, including transformation, viral transduction,-electroporation and'the like.<br>
Modulation The term modulation is used to signify an increase and/or decrease in the parameter being modulated. Thus, modulation of gene expression includes both increasing gene expression and decreasing gene expression,<br>
Integer This term refers to an individual entity of a parameter. By way -of example, if the parameter is the size of a -tag, then the integer will be one or more of-specific sizes within the parameter.<br>
Identifving^aqs associated with one or more cell-units<br>
In-one aspect, .there Js provided a .method for determining the cell-culture historyof a cell unit labelled with more than one tag (eg. more than one type of tag) comprising the steps of: (a) measuring-one or more parameters of each tag that 4s used to label the cell unit; (b) identifying each tag in the cell unit; and (c) correlating the identity of each tag to the identity of the cell unit and/ the specific cell culture conditions to-which the cell unit has been exposed.<br>
In a further aspect, there is also provided a method for determining the cell culture -history of a cell unit labelled with one or more- types of tag comprising the steps of: (a) obtaining the tags from the cell units in a form suitable for analysis; (b) measuring one or more parameters of each tag that is used to label a cell unit; (c) identifying each tag in the cell unit; and (d) correlating the identity of each tag to the chronology and identity of the cell culture conditions to which the cell unit has been exposed.<br>
The parameters) may be the size of the tags and/or the optical properties of the tags. Examples of the optical properties of the tags include, but are not limited to, light reflectivity, colour, the fluorescence emission wavelength(s) and/or the fluorescence emission intensity.<br>
The cell unit(s) to be analysed may be separated from a sample that may contain many different cell units.<br>
The tags in a cell unit(s) of interest may be isolated from the cell unit. In some embodiments, this -is achieved by enzymatic digestion and/or acid hydrolysfs of the cell unit. In .some- embodiments, this is achieved using the methods described herein. Advantageously, in certain embodiments this results in destruction of the cell unit but the tags are obtained intact.<br>
In one embodiment of th&amp;present invention, one or more different tags are measured for  cell unit-- such-as a cell unit contained -in a sample comprising a plurality of cell units.<br>
In some embodiments,-there will be from about 2 to about 50 or more different (types of), tags per oell unit In some-embodiments there will be from about 2 to about 40 different (types of) tags per cell unit. In some embodiments there will be from about 2 to about 30 different (types of) tags pen cell unit. In some embodiments there will be from about 2 to about 20 different (types of) tags per cell unit. In some embodiments there will'be from about 2 to about 10 different (types of) tags per cell unit.<br>
In some embodiments, there will be from about 5 to about 50 or more different (types of) tags per ceil unit. In some embodiments, there will be from about 10 to about 50 or more different (types of) tags per cell unit. In- some embodiments, there •will be from about 1-5 to about-50. or more different (types of) tags per cell unit. In some embodiments, there will be from about 20 to about 50 or more different (types of) tags-per cell unit. In some embodiments, there will be-from about 25 to about §0 OF more different (types of) tags per cell unit. In some embodiments, there will be from about 30 to about 50 or more different (types of) tags per cell unit In some embodiments, there will be from about 35 to about 50 or more different (types of) tags per cell unit. In some embodiments, there will be from about 40 to about 50 or more different (types of) tags per cell unit. In some embodiments, there will be from about 45 to about 50 or more different (types of) tags<br>
per cell unit. In some embodiments, it is expected that there will be about 50 or more different (types of) tags per cell unit<br>
In some embodiments, there will be about 55,60, 65,70, 75 or even 80 or more different (types of) tags per cell unit. In some embodiments, there will be about 55,60,65, 70,75 or even 80,90,100,250, 500, 750, or 1000 or more different (types of) tags per cell unit.<br>
Each of the parameters described above may have a number of different integers. In some embodiments, there are at least 2, 3, 4 or 5 different integers for each parameter. Jn some embodiments, there-are at least 10 different integers for each parameter. In some-embodiments, there .are at least 15 different integers for each parameter. In some embodiments, there are at least 20 different integers for each parameter. In some embodiments, there are at least 25 different integers for each parameter. In some embodiments, there are at least 30 different integers'for each parameter. In some embodiments, there are at least 35 different integers for each parameter. I IT some embodiments, there are at least 40 -different integers for each parameter. In some embodiments, there are at least 45 different- integers -for each parameter. In some embodiments, there are at least 50" different integers-for each parameter.<br>
In some -embodiments, thera are -at least 2 different parameters and .more than 5 different integers for each parameter.<br>
In some embodiments, there are at least 2 different parameters and more than 5 and less-than 5tt different integers for -each parameter- In some embodiments^here are.at-least 2 different parameters and 50 or less different integers for each parameter. In some embodiments, there are at least 2 different parameters and 50 or more different integers for each parameter. In-some embodiments; there are at least 2 different parameters and'40 orless-different integers for eaeh parameter. In somer embodiments, there are at least 2 different parameters and 30- or less different-integers for each parameter. In some embodiments, there are at least 2 different parameters and 20 or less different integers for each parameter. In some embodiments, there are at least 2 different parameters and 10 or less different integers for each parameter.<br>
Each cell unit may contain more than one (eg. multiple) tags of each species.<br>
Advantageously, this method can be automated such that many cell units and/or many samples can be processed. This is particularly advantageous, since each sample may have many (eg. hundreds) of different tags in it, thereby generating a large amount of data.<br>
For some embodiments of the present invention, there is a maximum of 50 different tags. By way of example, using the CML hydrophilic microbeads as described herein will typically result in this number of microbeads. The CML hydrophilic microbeads may be used together with one or more microcarriers. In some embodiments, the CML hydrophiWc microbeads are used together with a Cytopore 2- microcafrier.<br>
The tags in the sample may all be different or they may all be the same. The tags may be of different sizes - such as between about 1-10 uxn, preferably about 1.9um, about 4'.4um, about 5.4um, about 5.8um, about 7.4um, about 9.7ura and/or about 9.8um. The tags may be microspheres - -such as CML microspheres - and the diameters of such microspheres may be between 1-10 urn, preferably about 1.9um, about 4.4um, about 5.4um, about S.Sum, about 7.4nm, about 9.7pm and about 9 .Sum.<br>
For some embodiments, there is a maximum of 1000 different tags. By-way of example, using the rod-shaped nanowices striped- with silver and gold -as described herein can result in this number of tags. The- said nanowires may be used, together with one or more microcanriers. In some embodiments, the nanowires- are used together -with a Cultispber-G rnicrocarrier.<br>
In one embodiment, a particular tag is correlated with a known cell culture condition. Accordingly, once the cell unit has -been labelled with this tag and has been cultured in the corresponding conditions, it is possible to to Identify or selectihose culture conditions-that have given rise to-a particular cell unit of interest.<br>
Whilst various methods of image analysis are available in the art, a combination of Image Pro Plus imaging software by Media Cybernetics (www.medlacy.com), a Nikon TE2000-S fluorescence microscope and an Evolution VF cooled monochrome camera, supplied by Media Cybernetics may be used.<br>
In one embodiment one or more image(s) of each tag in the cell unit in the field of interest is measured. Typically, this may be achieved using microscopy - such as bright<br>
field microscopy, phase-contrast microscopy, oblique illumination microscopy, dark field microscopy, differential interference contrast microscopy, reflection contrast microscopy, varel contrast microscopy, polarizing microscopy, interference microscopy and fluorescence microscopy. Suitably, outlines may be drawn for the one or more image(s) of each tag in the cell unit in the field of interest and/or one or more fluorescence image(s) of each tag in the cell unit in the field of interest is measured. The fluorophore may emit, for example, a blue, green, near red or far red fluorescence. Suitably, the fluorophore is selected from the group consisting of UV2, Starfire Red and TRITC. The quantity of fluorophore may be selected from 5 different ranges (eg. ranges of fluorescence intensity) and each different range is discrete: In -some embodiments, the -ranges may-differ by about 2 to 5-foid or more in brightness. 1n some embodiments, the ranges may differ by about 5 to 10-fold or more in brightness. In some embodiments, the ranges may differ by about 2 to 100-fold or more in brightness. In some embodiments, the ranges may differ by about 2 to 1000-fold-or more in brightness. In some embodiments, the ranges may differ by about 2, 5,10,100 or 1000-fold or more in brightness.<br>
Suitably, the outlines for the one or more images) are loaded onto the one or more "fluorescence image(s) and one or more fluorescence image(s) of each tag in the cell unit in .the field of interest may-be measured for each fluorophore that is used to label the tags. .Each tag. in the ceirunit may be identified by reading the one or more parameters -suclras the area and/or optical density - of the tag within the one or more outlines.<br>
Thus, by way of example, tegs may be analysed by microscopy-using-a:Nikon TE20QO-S inverted epifluorescent microscope equipped with filter sets for visualization of the fluorophores - such as TRLTC, DAPI (UV2), GFP-B (all ffom"Nifcon) and Cy5 (Chroma Technology). Images may be captured -using an Evolution- VF coaled monochrome camera and jmage- analysis perfoFmed using Image Pro Plus (beth from Media Cybernetics).<br>
The outlines of single microspheres are typically captured using microscopy methods and the areas within these calculated to size the tags. The fluorescence intensities in the different channels may be used to further specify the identity of each tag by comparison to reference samples containing known tags.<br>
Suitably, the microscopy methods is selected fro the group consisting of bright field microscopy, phase-contrast microscopy, oblique illumination microscopy, dark field microscopy, differential interference contrast microscopy, reflection contrast microscopy, varel contrast microscopy, polarizing microscopy, interference microscopy and fluorescence microscopy.<br>
In one embodiment, the microscopy method is bright field microscopy.<br>
Optionally, the data that is obtained may be processed using further methods described herein, that provide for the chronology-and identity of cell culture conditions to which a cell unit in a sample has been exposed to be determined. Advantageously; this process may be automated thereby allowing a large amount of data to be rapidly processed.<br>
Advantageously,, the process utilises software that counts the number of each type of tag and then outputs this data in the form of a list, a graph or a chart.<br>
Advantageously, the -data may be inputted Into a spreadsheet for high throughput analysis.<br>
In one- aspect the method- of analysing this- data-comprises a first step- of comparing, the-upper and Tower limits of at least one parameter of a tag with the mean measurement"of the at least one parameter of the* tag.. In one embodiment of the present invention, the upper and lower limits of any parameter are obtained or obtainable by analysing a plurality of reference tags of the same-species. If the tower limit is less than the mean measurement and the upper limit is higher than the mean measurement then the method compares the upper and lower limits of a second parameter of a tag with the mean measurement of-- the second -parameter of -the tag. If the- tower limit- of- the second parameter is less than the mean measurement for that parameter and the upper limit of the second parameter is higher than the mean measurement of that parameter then the -method compares one or more further parameters relative to a lower limit of the corresponding parameter.<br>
If at any point the mean measurement of any parameter does not fall between the upper and lower limits determined for that parameter then the method compares that mean measurement relative to a second range comprising different limits for that same parameter.<br>
In one aspect, there is provided a method for determining the chronology and identity of the cell culture conditions to which a cell unit has been exposed comprising the step of: comparing the upper and lower limits of at least one parameter of a tag with the mean measurement of the at least one parameter of the tag.<br>
In a further aspect, there is provided a method for determining the chronology and identity of the cell culture conditions to which a cell unit has been exposed comprising the steps of: (a) comparing the upper and lower limits of at least one parameter of a tag with the mean measurement ofUie at least one parameter of the tag; (b) comparing, the mean measurement for the at least-one parameter of-the-tag with the lowest limit for the at least one parameter of the tag; and (c) correlating the measurement of the at least one parameter of the tag with the identity of the tag associated with the cell unit.<br>
In one embodiment; if 1he lower limit is Jess than the mean measurement and the upper limit is higher than the mean measurement then the^-method compares the upper and lower limits of a second parameter of a tag with the mean measurement of the second parameter of the tag. For example, the first -parameter may be size and" the second-parameter may be the intensity or level -of fluorescence.<br>
In one embodiment, if the lower limit oT the second parameter is less than the mean measurement for that parameter and the upper-limit of the-second parameter- is higher than the mean measurement of that parameter then the method compares one or more .further parameters-relative to a lower limit of the-corresponding parameter.<br>
In one embodiment, if the lower limit is not less--than the mean measurement and the mean-measurement-is net less then than the-upper limit them the upper and lower limits of at least one further-integer of the-same parameter of the tag is measured. By way of example, the further integer may include, but is not limited to a different size, a different fluorophore or a different range of fluorescence intensity. Accordingly, the parameter for each of these integers will be size, type of fluorophore and intensity of fluorescence, respectively.<br>
In one embodiment, if the lower limit is less than the mean measurement and the mean measurement is less then than the upper limit then the at least one further parameter for<br>
the tag is measured - such as the size, type of fluorophore and/or level of fluorophore of the tag.<br>
In one embodiment, if the mean measurement for the at least one parameter of the tag is greater than the lowest limit for at least one parameter of the tag, then the result is correlated with one or more tags that possess the one or more parameters.<br>
In one embodiment, if the mean measurement for the at least one parameter of the tag is not greater than the lowest limit for the one or more parameters of the tag, then the result is correlated with a tag that possess the parameter.<br>
In one embodiment, if the lower limit is not less than the mean measurement and the mean measurement is not less then than the upper limit for the at least one further parameter then the upper and lower limits t&gt;f at least one further integer of the same parameter of the tag is measured.<br>
In one embodiments-trie method-may-be repeated for one or more further- integers of the same parameter of the tag. In another embodiment,, the method may even be repeated forall integers of-the same parameter of the tag.<br>
In one embodiment, if the lower limit is less than the mean measurement and the mean measurement is less then than the upper limit for the at least one further parameter and the mean measurement for the at least one parameter of the tag is greater than the lowest limit for at least one parameter ef the tag, then the result is. correlated-with one or more tags that possess the one or more parameters.<br>
In one embodiment, if-the lower limit is less thairfhe mean measurement and the meaa measurement is less then than^ the upper limit for the at least one^orther_parameter-and the mean measurement for the at least one parameter of the tag is not greater than the lowest limit for the one or more parameters of the tag, then-the result is correlated with a tag that possess the parameter.<br>
In one embodiment, if the measurement of the at least one parameter of the tag cannot be correlated with the identity of the tag associated with the cell unit, then the method may comprises the additional steps of, for example, examining fluorophores serially,<br>
such that the final 'No' decisions from will go on to interrogate the fluorophores values for the relevant size tag,<br>
Suitably, the methods described herein are performed using one or more of the complexes described herein.<br>
In a further aspect, there is provided a method for identifying one or more area and/or density values for a tag comprising the steps of: (a) obtaining one or more images of a tag that is used to label a cell unit; (b) drawing outlines for the one or more images of the tag; (c) loading the outlines for the one or more images of the tag; and (d-) reading the area and/of density within the one or more outlines.<br>
In one embodiment, the image may be a phase image. Determination of the Identity or cell culture history of a cell unit<br>
As already described above, when handling- large numbers-of cell units, their identity and/or cell culture history (for example the chronology, and the exact nature of a series of culture conditions that any one group- or unit may .have been exposed to) can become confused Forinstance, the split-pool protocol of cell culture necessarily involves mixing cell units in each round.-making it difficult io follow individual units. Determining the cell culture history of a- cell unit in a mixture of cell units, which, have been subjected to multiple culture conditions, is sometimes referred to as 'deconvolution' of the cell culture history.<br>
One method of determining the cell culture history of a cell unit in a mixture of cell units, is to-label cell units- and it is therefore advantageous-to label the celt-units. As described herein, tags - such BS rod-shaped particles, microbeads and/or microspheres may be used as labels or tags which are conjugated to a microcarrier - such as a cell-associated microcarrier. Subsequent detection and identification provides for a record of the chronology and identity of the cell culture conditions to which the cell unit has been exposed.<br>
Microcarriers<br>
A variety of microcarriers are available, ranging in shape and size and made of different materials.<br>
By way of example, the microcarrier may be a porous microcarrier selected from the group consisting of Cytopore microcarrier (eg. a Cytopore 1 microcarrier or a Cytopore 2 microcarrier), a Cultispher microcarrier, a Cultispher-G microcarrier, a Cultispher-GL microcarrier and a Cultispher-S microcarrier, an Informatrix microcarrier, a Microsphere microcarrier, a Siran microcarrier, and a Microporous MC microcarrier.<br>
By way of further example, the microcarrJer may-be a solid microcarrier - such as a Cytodex microcamer-(eg. a Cytodex 1r Cytodex 2-or Cytodex 3 microcarrier) a Biosilon microcamer, a Bioglass microcarrier, a FACT III microcarrier or a DE 52/53 microcarrier.<br>
Microcarrier culture has significant advantages, including the -scale-up of cultures, and also allows units of cells to be exposed-to selected culture conditions as required in order to obtain the desired growth and/or differentiation conditions.<br>
The surfaces of the microcarriers may. be further modified by physical or chemical treatments, such-as adsorption or covaient cross-linking of molecular entities with a desired charge or other desired characteristic.<br>
In a "broad embodiment, therefore, the invention-provides a method Tor culturing cells in vitro, comprising growing said cells adhered to the microcarrier complexes described herein.<br>
Jn one aspect, there-is provided a complex comprising a microcarrier and a .charged (eg. negatively-charged), tag. -1n other, words, there-is provided a microcarrier conjugated or labelled with a charged (eg ^negatively charged) mterospfiere.<br>
In a further aspect, there is provided a microcarrier and a rod-shaped tag. In other words, there is provided a microcarrier conjugated or labelled with a rod-shaped tag.<br>
Cultispher microcarriers<br>
CultiSpher is manufactured from pharmaceutical grade porcine gelatin via a process which yields a highly cross-linked gelatin matrix with high mechanical and thermal<br>
stability. When used in cell cultures, cells can attach to both the external and the internal surfaces of the matrix. The increased surface area of the matrix together with the protection from stress afforded to the cells in the interior of the matrix results in enhanced cell production capabilities. An additional advantage of the product is that the matrix can be dissolved with proteolytic enzymes resulting in the harvesting of cells with almost 100% viability.<br>
In one embodiment, the microcarrier is a Cultispher-G microcarrier. Cultispher-G has a particle dfameter of 130-380 urn, a volume of 12-18 ml/g dry, a density of 1.04 g/ml with an average pore diameter of 20 urn.<br>
In order to prepare and use Cultispher-G microcarriers, reference can be made to inter alia Biotech. Bioeng. (2000) 68, 1 p59-70; Brit. J. Cancer. Suppl. XXVII, S-78-S82 (1996); and the manufacturer's website atwww.percell.se.<br>
Cytopore 2 microcarrier<br>
Cytopore microcarriers are available from GE Healthcare (previously Amersham) fwww.microcarFier-s.com). Cytopore is made of 100% cellulose, which is non-toxic tocthe cets and -biodegradable. It is positively charged, due to the N,N;-diethylaminoethyl .groups. It has a very precise particle size distribution and a network structure, the ratio of surface area to particle material is. more than 95 to 1. The network structure enables stained cells to be closely observed while they grow inside the microcarriers. The typical particle diameter is 200-280--um and effective surface-area is 1-.1 m2/g dry. The relative density is 1.03 g/ml, the average diameter of pore openings is 30 urn and the volume, is 40-ml/g dry. In order to prepare-and use Cytopore microcarriers reference is made to inter alia Applied Microbiology and Biotechnology. (1-997) 47, 4 p352-7; Cytotechnolbgy (1999) 30 p143-147,- Chinese Journal of Biotechnology £1999)' 15, 4 p239-44 and/tefe Oto-Laryngologica (2002) 122, 5 p541-5.<br>
Cytopore 2 has been optimised for anchorage-dependent cells requiring a charge density around 1.8 meq/g.<br>
In some embodiments, the microcarrier is a porcine gelatin microcarrier. In some embodiments, the microcarrier is made of 100% cellulose.<br>
Advantageously, microcarriers with a strong positive charge are optimal with tags carrying a strong negative charge.<br>
As described above, tags may be used as labels or tags which are conjugated to a microcam'er — such as a cell-assoeiated rnicrocarrier. Subsequent detection- and identification provides for an unambiguous record of the chronology and identity of the cell-culture conditions to which the cell unit has been exposed.<br>
Various molecular or macromolecurar tags may be used in combination with the microcarriers so long as they can be detected. The tags typically comprise uniquely shaped or objects modified with markings and/or coloured- and/or fluorescent compounds.<br>
In one embodiment the tags which are used to label cell units have one or more (preferably all) of theibllowing.qualities:<br>
i.  They are small in size relative to the rnicrocarrier they are labeliing^-and/or smaller<br>
•than the mean pore size--0f-a porous microcarrien it.  They are capable of forming a complex with the microcam'er such that binding<br>
persists'throughout the Bxperfment and so unbound tags can-be separated from<br>
the complex without affecting the labelled cell units; iii.  They are separable from cell units with which they have formed a complex under<br>
conditions which do not perturb the unique qualities of the tags iv.  They are made of one or more Inert substances which-da not substantially-affect<br>
-the biology of the cell unit and which in.1unris.not affected t&gt;y-tbe-celkunits--or their<br>
biology; v.  They are obtainable in large numbers and moreover in many related but distinct<br>
variants which are easily distinguishable using an appropriate technique; vl. They are distinguished by a method which is convenient, highly reliable and which<br>
can be automated.<br>
In one embodiment, the tag is a microsphere - such as a fluorescent and/or coloured microsphere.<br>
More than 2000 different microspheres made by emulsion or suspension polymerization, precipitation etc. and comprised of polystyrene, other polymers, copolymers, terpolymers and/or silica etc. are available in a variety of sizes, densities, colours etc, for example from Duke Scientific Corporation {Palo Alto CA, USA) or Bangs Laboratories Inc. (Fishers IN, USA).<br>
A common type of microsphere is the Polystyrene (PS) and styrene/divinylbenzene copolymer (S/DVB) microsphere. Other polymers include polymethylmethacrylate (PMMA), polyvinyltoluene (PVT), styrene/butadiene (S/B) copolymer, styrene/vinyltoluene (S/VT) copolymer. Many -of these microspheres can be functionalised, for instance 6y carbsxyljgr-oupsras in the CML microspheres, or by amino functionalized or nitrogen-containing compounds, like primary, secondary, tertiary, and quaternary aliphatic amines, aromatic amines, and pyridines, which offer alternative coupling reactions to the COOH beads.<br>
Suitably, the microsphere is a hydraphilic microsphere. More suitably the microsphere is a polystyrene microsphere. Most suitably, the. microsphere is a surface-modified microsphere such as a carboxylate -modified (CML) microsphere from Duke Scientific Corporation (Palo-Alto CA, USA).<br>
In one embodiment, one or more CML microspberes are complexed together with one or .more cytopore 2 microcarriers.<br>
-CML microspheres have a highly charged surface -layer of-carboxyl-groups derived from a copolymerisation process. The surface is somewhat porous and relatively hydrophilic, but retains overall hydrophobkrproperties. The charge density of these particles ranges. from about 10-125 A2~per carboxyl group, and they are stable to high concentrations-of electrolytes (up to 4lvr -unlvalent salt). The CML latex will adsorb proteins and other biomolecules, but much less strongly than hydrophobic microspheres.<br>
In some embodiments, conjugates of microspheres and proteins, e.g. streptavidin are prepared.<br>
For example, conjugates with CML microspheres may be prepared as follows. CML microspheres may be activated using a water soluble carbodiimide reagent that makes the carboxyl groups reactive with primary amines on the proteins to be coupled. A 50<br>
mM reaction buffer at pH 6.0 is prepared. Sodium acetate or 2-[N-morpholino)ethanesulfonic acid (MES) are suitable buffers. The protein is dissolved in the reaction buffer at a concentration of 10 rng/mL A 1% (w/v) suspension of microspheres is prepared in the reaction buffer. One volume protein solution to ten volumes microsphere suspension is prepared and the mixture allowed to incubate, at room temperature for 20 minutes. A solution of 10 mg/mL (52 u.Mol/mL) of 1-ethyl-3-(3-dimethylaminopropyl)carbodiimide (EDAC) in deionized water is prepared and used immediately. A calculated amount of EDAC solution to the microsphere suspension is added and the pH of the reaction mixture adjusted to 6.5 ± 0.2 with 0. IN NaOH. The mixture is incubated on a rocker or mixing wheel for 2 hours -at room temperature. Unbound protein is removed and stored in storage buffer.<br>
Advantageously, CML and other microspheres can be obtained in various formats - such as various colors (eg. blue, red, green, yellow, black), various fluorophores (eg. Fluorescein (green), Fluorescein (red) or Fluorescein and Rhodamine (red green) and various sizes (eg. 5.4 um (1.14 x 1010 beads/gram), and 7.6- urn- (4.tO x 109 beads/gram-)).<br>
CML-and other microspheres may be prepared such that they are-loaded with one or more visible dyes and/or fluorophores.<br>
In one embodiment, tags - such as microspheres -are-not coated with proteins.<br>
Advantageously, CML microspheres not coated with proteins-(eg. streptavidin)-are highly negatively charged and they adhere extremely tightly to Cytopore microcarriers, which •carry a high positive charge. The ratio of CML microsphere(s): Cytopore mierocarrier(s) required to give bulk complexes may be as low as about 1:1. In previous-systems that the present inventors have used a ratio of about 1:250-was required in order to obtain even a few bound tags in each complex after multiple washes.<br>
By varying various parameters in the fabrication process, commercial microsphere providers - such as Bangs Laboratories - can manufacture bead sets which can be distinguished based on differing sizes (e.g. bead sets of 4.4um and S.Sum diameter). Beads within each size group can be furthermore distinguished from each other based on differing fluorescence intensity owing to differential loading with a single fluorescent dye. It is possible to use many different dyes with different absorption or emission<br>
characteristics, which can be attached to the microcarriers described herein. Accordingly, Tag diversity may result from varying tag size and/or fluorophore loading (fe fluor intensity) and/or fluorophore identity/combination. In particular, Tag diversity may result from the type of fluorophore they carry (eg. beads can be loaded with either UV2 or Starfire Red); size (eg. for each fluorophore there are 5 different bead sizes: 1.87, 4.41, 5.78, 5.37 and 9.77 microns) and/or the quantity of fluorophore they carry (5 different intensities of each dye are available). Other fluorophores - such as TRITC may be used.<br>
Filters can then be used to detect the at least 4 different dyes on any given bead - such as the TRITC filter (ex 540/25; dm 565; ba 605/55) for TRITC visualization-from Nikon; the DAP! filter (ex 340-380; dm 400; ba 435-485) for UV2 visualization from Nikon; the GFP-B filter (ex 460-500; dm 505; ba 510-560) for FITC visualization from Nikon and the Gy5 filter set (cat- no 4t008 from Chroma Technology) for Strarfire Red visualization.<br>
Micraspheres can be dyed internally or externally, with visible or fluorescent dyes. Internal dyeing occurs when, the.dye is integrated into the microsphere mass, typically by-soaking the microsphere in a -solution containing -a dye or fluorophore. External modification, occurs -when a dye is conjugated to the surface of the microsphere, for instance modification of a CML microsphere with an isothiocyanate derivative as described herein.<br>
Accordingly, in some embodiments, the microsphere may be dyed internally or externally, with visible or fluorescent dyes.<br>
It-is furthermore possible to use 'quantum dots' to obtain a very high number of different fluorescent labels which can be read, conveniently. Thusr in arfurther embodiment-of the present invention, Quantum dots instead- of fluorophopes are used, -in certain embodiments, quantum dots are preferable due to the fact they do not fade (photo-bleach) when exposed- to light. For instance the fluorophore FITC is known to photo-bleach and cell units treated with tags containing FITC are ideally handled in the dark and are difficult to analyse reliably. Quantum dots may be incorporated into microspheres at the time of polymerizing the polystyrene resulting in even loading of tags,. Quantum dots are available in many colours and they can be excited at the same wavelength so allowing visualization of multiple colours without filters, by using a colour CCD camera. Further background information on Quantum dots is available from<br>
US6.322.901, US6.576.291. US2003/0017264, US6.423.551, US6.251.303, US6.319,426 US6.426.513, US6.444.143, US2002/0045045, US5,990,479, US6.207.392, US6.251.303, US6,319,426, US6.426.513 and US6.444.143.<br>
Advantageously, the tags are protected against degradation by the components of the cell culture, for example by chemical or other modification or by encapsulation. Encapsulation of tags can take place in many different media, for example in beads as already described herein - such as those from Bangs Laboratories Inc. (Fishers IN, USA), and encapsulation may be used to standardise tag dosage in addition to providing --components for tag amplification and/or detection (for example "by providing PCR primers for use with a DMA tag).<br>
Detection of tags can be accomplished by a variety of methods familiar to those skilled in the art. Methods include mass spectrometry,-nuclear magnetic, resonance, sequencing, hybridisation, antigen detection, electrophoresis, spectroscopy, microscopy, image analysis, fluorescence detection;--etc. In some embodiments, since the tags typically contain a colour or a fluorophore then microscopy, spectroscopy, image analysis and/or fluorescence detection are used.<br>
The tags do not necessarily bavelolbe distinguished by their chemical-or molecular structure in the first instance. Multiple variations of the non-chemical tagging strategy can be devised to determine the identity of a given cell unit- in a mixture or of deducing the. identity of the different cell units that comprise a mixture. For instance optical or visual -methods of tagging have been described where-different shaped objects, graphically, encoded objects or different colours denote the identity of a sample (for example see "1-998, Guiles et ai, Angew. Chem. IntI EdJEngl, vol. 37, p926; Luminex Corp, Austin-TX, USA;-BOT-Bioseiences; Memobead Technologies, Ghent, Belgium)^<br>
Suitably, the tag may be a charged tag (eg. a negatively charged tag). Accordingly, in a further aspect, there is provided a complex comprising- a microcarrier - such as a porous microcarrier- and a charged tag.<br>
Typically, the microcarrier has a net charge. It may comprise, consist or consist essentially of protein, cellulose, polyethylene, polystyrol, glass, collagen, collagen-gylcose-aminoglycan and/or gelatin. Accordingly, the microcarrier may be selected from the group consisting of a Cytopore microcarrier, a Cytopore 1 microcarrier, a Cytopore 2<br>
microcarrier, a Cultispher microcarrier, a Cultispher-G microcarrier, a Cultispher-GL microcarrier and a Cultispher-S microcarrier, an Informatrix microcarrier, a Microsphere microcarrier, a Siran microcarrier, and a Microporous MC microcarrier.<br>
Suitably , the charged tag is a sphere - such as a microsphere that is about 9 uM or less in diameter. The microsphere may be a carboxylate modified (CML) microsphere.<br>
In a further embodiment, the tag is a rod-shaped particle. Suitably, the rod-shaped tag is a nanowire. The nanowire may comprise, consist or consist essentially of various metals - soch as aluminium. The nanowire may be coated with various metals - such as silver and/or gold. Suitably, the nanowire is-about 1u.M or less in diameter and/or is about 10 u.M or less in length.<br>
The nanowire may be a nanowire as described in Science voP. 294, p. 137-141 (2001). Accordingly, in a further aspect, there is provided a complex comprising- a microcarrier and a nanowire. Briefly, nanowires are multimetal microrods intrinsically encoded with submicrometer stripes. Complex patterns can be generated' by sequential electrochemical deposition of metaf ions onto templates with uniformly sized pores. Advantageously, the nanowires are small enough to be used as tags thaLmay be added after each split. This is more convenient as it is necessary to r-ead tags only in the positive microcarriers.<br>
Parameters for the rod-shaped particle - such as the nanowire - include but are not limited to see, optical properties-and/or metal composition. In-one embodiment, -the optical properties are selected from the group consisting of: light reflectivity - such as light reflectivity of a particular wavelength, colour, the fluorescence emission-wavelength(s) and the fluorescence emission intensity.<br>
In some embodiments, the rod-shaped particle - such as-the nanowire is externally dyed.<br>
The microcarrier that is used together with the rod-shaped tag may be a porous microcarrier - such as a charge neutral microcarrier.<br>
The microcarrier may comprise, consist or consist essentially of protein, cellulose, polyethylene, polystyrol, glass, collagen, collagen-gylcose-aminoglycan and/or gelatin.<br>
The microcarrier may be selected from the group consisting of a Cytopore microcamer, a Cytopore 1 microcarrier, a Cytopore 2 microcarrier, a Cultispher microcarrier, a Cultispher-G microcarrier, a Cultispher-GL microcarrier and a Cultispher-S microcarrier, an Informatrix microcarrier, a Microsphere microcarrier, a Siran microcarrier, and a Microporous MC microcarrier.<br>
In one embodiment, the microcarrier that is used together with the rod-shaped microcarrier is a Cultispher microcarrier - such as a Cultispher-G microcarrier, a Cultispher-GL microcamer or a Cultispher-S microcarrier. In one embodiment, the •microcarrier that is used -together with the rod-shaped raicrocarrier is a- Cultispher-G -mferocarrier.<br>
Advantageously, it has been found that using rod shaped tags and charge-neutral porous microcarriers is better than using spherical tags on the same microcarriers. Without wishing to be bound by an particular theory it is believed that smaller tags penetrate the pores of the microcarriers better and become jammed (presumably due to siza asymmetry). Accordingly, the binding of nanowires is better than, for example, the binding of microsphere tags and results in a tiigh level permanent tagging.<br>
In another embodiment; one or more polystyrene micrdbeads are complexed together with one or more Cultispher-G microcarriers.<br>
For some embodiments, the tag is not a DNAtag.<br>
In some embodiments, the tag Is an externally dyed tag.<br>
Stem-Celts -<br>
Stem cells are described in detail in Stem Cells: Scientific.Progress and Future Research Directions. Department of Health and Human Services. June 2001. http://www.nih.gov/news/stemcell/scireport.htm. The contents of the report are herein incorporated by reference.<br>
There is still considerable debate about what constitutes a stem cell, however for the purposes of this discussion a key characteristic is the ability to differentiate into a different cell type. Examples of stem cells are given below.<br>
Various factors that have been used to induce directed differentiation of stem cells include: retinoic acid, epidermal growth factor (EGF), bone morphogenic proteins (BMPs), basic fibroblast growth factor (bFGF), activin-A, transforming growth factor-beta-1 (TFG p-1), hepatocyte growth factor, nerve growth factor, sonic hedgehog (SHH), interleukin-3 (IL-3), interleukin-6 (IL-6), granulocyte macrophage colony stimulating factor (GM-CSF), erythropoietin, vitamin D3, dexamethasone, (5-mercaptoethanol, butylated hydroxyanisole, 5-azacytidine, DMSO, insulin, thyroid hormone (T3), LIF, foetal calf, serum, vascular endothelial growth factor (VEGF), steel factor, variations in oxygen concentration, ascorbic acid, fj-glycerophosphate, nteotinamide, platelet derived .growth-faclor (PDGF), cAMP, various cell, adhesion molecules and substrates, and others. In addition to these defined factors, it is likely that undefined extracts, such as conditioned media, human and animal tissue homogenates, or plant extracts can be used to direct stem cell differentiation. Progressive fractionation of these undefined extracts may yield active fractions or even pure .components with high potency. These factors can be added to the growth medium-used, in a particular experiment, either alone, or in combination, or in a defined-order which is craciaHo the experimental-result.<br>
Formation otcelfunite<br>
Groups of ceHs (cell: colonies) can be grown in ceil culture under various conditions and the colony=-can largely maintain-its integrity under various conditions, when.-disturbed,, -and when mixed with other colonies. Such groups.or colonies are referred to herein, as cell units. Formation of cell units may be achieved, by way of illustration, by growing-cells as adherent cultures on solid substrates such, as carriers. If cell proliferation occurs after seeding on the carriers, the daughter ceils will attach on the same carrier, and form part of thasame colony. In general, live aSherent-cells, do not readily dissociate from their growth substrate, and so the integrity of the cell colony persists despite any mechanical manipulation of the carrier, agitation of the culture medium, or transfer into another tissue culture system. Similarly, if at any time multiple carriers are placed in the . same vessel (e.g. beads are pooled) then there will be no substantial transfer of cells from one bead to another.<br>
An important advantage of forming cell units on solid substrates is that the substrate -and therefore the attached cells by reason of association - can be labelled as described herein.<br>
/Vhen cells are grown on smaller carriers they can be treated as a suspension culture. A common method of growing cells on small carriers is referred to as microcarrier cell culture (see 'Microcarrier cell culture-, Principles and Methods', Edition AA, available from 'tfnersham Biosciences (18-1140-62); herein incorporated in its entirety by reference). Microcarrier cultures are used commercially for antibody and interferon production in Fermenters of up to 4000 litres.<br>
As the physical properties of carriers are well known it is easy to calculate the number of carriers used in an experiment. The carriers may be available as dried products, whicrr can be accurately weighed,, and subsequently-prepared by swelling in liquid medium. In-addition the number of cells used to inoculate a microcarrier culture can be worked out and varied.<br>
Harvesting of cells grown on the microcarriers described* herein, or liberation of labels from microcarriers, can be achieved by enzymatic detachment of cells, and/or by digestion of the carrier where applicable as described-herein.<br>
Separation of tags from labelled cell units<br>
In a further aspect, the present invention provides improved methods for the separation of tags from cell units (eg. cell units completed with microcarriers):<br>
Advantageously, when-the tags are separated in accordance with the methods described herein, they are obtained in an intact state such that the unique parameters of each tag can be measured faithfully.<br>
Advantageously, when the tags- are separated'-in order to be analysed by an -optical-method - such-as by microscopy - they are furthermore obtained" in close proximity to each -other such that the they can be imaged conveniently (eg. using just one or two image fields).<br>
In one aspect, there is provided a method for separating a complex comprising a microcarrier and a tag, comprising the step of contacting said complex with a protease, wherein said microcarrier comprises, consists or consists essentially of protein.<br>
Suitably, the microcarrier comprises, consists or consists essentially of collagen and/or gelatin - such as a Cultispher microcarrier (eg. a Cultispher-G microcarrier, a Cultispher-GL microcarrier and Cultispher-S microcarrier).<br>
The protease may be proteinase K, typsin, thermolysin and/or caspase.<br>
If the protease is proteinase K then in some embodiments, it is used in an amount of about 0.5U/ml or more. Suitably, the complex is contacted with proteinase K for at least about 20-60 minutes. Suitably, the complex is contacted with the protease in a volume of about 5 u.1 or less.<br>
In a further aspect, there is provided a method for separating a complex comprising a microcarrier - such as a porous microcarrier - and a tag, comprising the step of contacting said complex with acid.<br>
Suitably, the microcarrjer has a net charge and may comprise., consist .or consist essentially .of cellulose. Thus, by way of example, the microcarrier may be a Cytepore microcarrier - such.as is a Cytopore 2 microcarrier.<br>
In some embodiments, the tag.is a sphere - such as a microspheFe - that comprises, consists or consists essentially of polystyrene.<br>
In some embodiments, the tag is a rod-shaped particle - such as a nanowire.<br>
Various reagents may be used for the liberation of tags from microcarrier - such as acid (eg. Hydrocholoric acid and/or sulphuric acid) or sodium hypochlorite -and/or sodium hydroxide in addition to other reagents - such as trypsin-~EDTA, «e1lulase, proteinase -K, and sod urn-chloride-(e.g. ~5M sodium chioridef.<br>
In some embodiments, the reagent is hydrochloric acid, preferably 37%-hydrochloric acid (about 12 M).<br>
in some embodiments, the reagent is sodium hypochlorite. In some embodiments, the reagent is a protease.<br>
In some embodiments, the reagent is a proteinase K.<br>
Suitably, the digestion of the one or more microcarriers is carried out on an optically clear surface - such as a glass microscope slide. This allows convenient imaging in situ which is advantageous as no material is lost.<br>
Suitably, the surface is treated to prevent the spreading of a liquid containing the microcarrier, for instance using a siliconising (silanising) agent.<br>
Suitably, the microcarriers to be xligested -are washed in distilled water to remove any traces of medium-and/or salts and applied to the surface in a minimal volume of liquid<br>
Suitably, the surface is heated to achieve complete dehydration of the microcarriers and to adhere the tags to the surface, thus preventing dispersal.<br>
In order to-cany out microcarrier djgestion in a minimal volume, preferably less than 5u,l and more preferably less than -2ul, the digestion is performed irra humidified box.<br>
In some embodiments, -the digestion is achieved using, one or more proteases. The protease solution is. typically-dispended directly onto the_dried microearriers- and placed in the humidified box. Once the microcarriers have dissolved (typically- 30-60rnins) the slides are removed from- the humidified, box to check "for complete digestion. The protease solution is evaporated and complete drying of the released tags onto the slide-is provided for to allow the tags-to be analysed.<br>
In some embediments, the digestion of the one or more microcarriers is-carried out in- a microcentrifuge or PCR tube such as those manufactured by Eppendorf. The interior of the tubes may be treated to prevent the tags fromBdhering to it once they are -liberated from the microcarriers.<br>
In some embodiments, the tubes are placed in a PCR machine allowing for precise temperature control.<br>
In some embodiments, the PCR machine operates a heated lid, allowing for use of minimal volumes of liquid.<br>
Suitably, once the one or more tags have been separated from the cell unit(s) and the tag(s), one or more images of the tag(s) may be obtained (using, for example, a microscopic technique). The images may then be analysed to determine one or more features of the tag.<br>
In a further aspect, there is also provided a method for identifying one or more tags that are obtained or obtainable from one or more cell units comprising the steps of: (a) separating the cell unit(s) and the tag(s); (b) obtaining one or more images of the tag(s) (eg. by using a microscopic technique); and (c) analysing the images to determine one or more features of the tag. Suitably, the one-or more features of the tag may be used to determine the-ceH culture history of the cell unit(s) from which the tag(s) was derived or obtained.<br>
Combinatorial serial culture of cells Split-pool cell culture<br>
-Forming cett units (particularly microscopic cell units) "is furthermore usefuKor sampling multiple tissue culture conditions as each celTunit-constitutes an easily handled unit that can be -exposed te a variety of cell" culture-conditions. In accordance with the present invention, cell groupings are typically produced by growing cells in microcarrier culture, and the terms-cell unit, cell group, colony and-bead are used interchangeably. A particularly efficient method for sampling a large number of cell culture conditions is referred to as .Combinatorial Cell Culture or splitrpool'cell culture and in one embodiment involves the serial subdividing and combining of groups of cell units in order to sample •multiple combinations of cell culture conditions. In one aspect of the invention the method operates-by-taking an initial starter culture for different starter cultures) of cell units divided Into- Xi number of aliqtiots -each containing multiple beads (groups/colonies/carriers) which are grown separately under different culture conditions. Following cell culture for a given time, the cell units can be peeled by combining and mixing the beads from the different aliquots. This pool can be split again into X2 number of aliquots, each of which is cultured under different conditions for a period of time, and subsequently also pooled. This iterative procedure of splitting, culturing and pooling (or pooling, splitting and culturing; depending on where one enters the cycle) cell units allows systematic sampling of many different combinations of cell culture conditions. The complexity of the experiment, or in other words the number of different combinations of<br>
cell culture conditions tested, is equal to the product of the number of different conditions (X! x X2x ...Xn) sampled at each round. Note that the step of pooling all the cell units prior to a subsequent split can be optional - a step in which a limited number of cell units are pooled can have the same effect. The invention therefore embodies a number of related methods of systematically sampling multiple combinations of cell culture conditions where groups of cell units are handled in bulk.<br>
Regardless of the precise manner in which a diversity of cell culture conditions is sampled by this means the procedure is efficient because multiple cell units can share a single vessel, where they are cultured under identical conditions, and-it-can be carried out using, only a few culture vessels at any ona time (the number of culture vessels in use is equal to the number of split samples). In many respects the principle of this procedure resembles that of split synthesis of large chemical libraries (known as combinatorial chemistry), which samples all possible combinations of linkage between chemical building block groups (see for example: Combinatorial Chemistry, Oxford University Press (2000), Hicham Fenrriri (Editor)). Split-pool cell culture can-be repeated over any number of rounds, and any number of conditions can be sampled ^at each round. So long as the number of cell unite (or colonised beads in this example) is greater than or equal to the number of" different conditions, sampled over all" rounds, and assuming that the splitting of cell units-occurs-totally randomly, itJa expectecHhat there will* be at least one cell unit thatTias been cultured according to each of the various combinations of culture conditions sampled- by the experiment. This procedure can be used to sample growth or differentiation conditions for any cell type, or the efficiency of biomoiecule-production <e.g. production of erythropoeitin or inteiferon by any celrtype-. because the procedure is iterative it ideally suited to testing multistep tissue culture protocols for instance those described above in connection with stem cell differentiation. variables which can be. sampled using this technique include type.-celt .grouping microcarrier cell-encapsulation whole organism growth substrate fibronectin on duration round temperature different media concentrations constituents factors conditioned co-culture various types feeder cells animal plant extracts drugs other synthetic chemicals infection viruses transgenic addition transgenes antisense anti-gene molecules rnai triple helix sensory inputs case organisms electrical light red-ox stimuli and others.></e.g.>
Split-split cell culture<br>
The purpose of performing split-pool processes on cell units is to systematically expose these to a pre-defined combination of conditions. The person skilled in the art will conceive of many different means of achieving this outcome. In addition to split-pool processes and variations thereof, it is worthwhile briefly discussing split-split processes. A split-split process involves subdividing a group of cell units at least twice, without intervening pooling of cell units. If split-split processes are used over a large number of rounds, the number of separate samples that are generated increases exponentially. In this case-it is important to employ-some level of automation, for-example the use of a robotic-platform and sophisticated sample tracking systems. The advantage of-split-split steps is that (since cell units are not combined) it is possible to segregate lineages of the various cell units based on their cell culture history. Consequently split-split steps can be used to deduce, if a particular cell culture condition is responsible for any given cellular •process and therefore used to deduce the culture history of cell units.<br>
Predetermined-protocols<br>
The-splitting-and/or pooling of cell" units- may be accomplished totally randomly or may follow a predetermined protocol. Where-cell units are split and/or pooled randomly, the -segregation of a given cell unit into any group is not predetermined or prejudiced in any way. In orderto-result in a high .probability that at least one cell unit-has-been exposed to each of the possible combinations of cell culture conditions, it is advantageous to employ a larger number-of ceil units, than the total number of combinatioas. of cell culture conditions that are being tested. Under certain circumstances it is therefore advantageous to split and/or pool cell units according to a predetermined protocol, the overall -effect being that adventitious duplications or omissions of combinations -are prevented. Predetermined-handling of cell units can. be optionally planned'in-advance and logged on a spreadsheet or computer programme, and splitting and/or pooling operations executed using automated protocols, for instance robotics, Labelling of cell units (see below) can be by any of a number of means, for instance labelling by RFID, optical tagging or spatial encoding. Robotic devices capable of determining the identity of a sample, and therefore partitioning the samples according to a predetermined protocol, have been described (see 'Combinatorial Chemistry, A practical Approach', Oxford University Press (2000), Ed H. Fenniri). Alternatively, standard laboratory liquid handling and/or tissue culture robotics (for example such as manufactured by: Beckman Coulter<br>
Inc, Fullerton, CA; The Automation Partnership, Royston, UK) is capable of spatially encoding the identity of multiple samples and of adding, removing or translocating these according to pre-programmed protocols.<br>
Analysis and/or separation of cell units<br>
Following each round of cell culture, or after a defined number of rounds, the cell units can be studied to observe any given cellular process that may have been affected by the tissue culture conditions.<br>
The invention will now be further-described by. way of Examples, which are meant to serve to assist one of ordinary skill' in the art in carrying out the invention and are not intended in any way to limit the scope of the invention.<br>
Examples Example 1<br>
Biotin-conjugation of Cultispher-G microcarriers<br>
Fluorescent microspheres (tags), from Bangs labs are streptavid'm-coated and so the Cultispher-G microcarriers tCSG) are bletinylated in order tolacilitate binding of .the two during the course of a split-pool experiment. CSG are supplied dry and are hydrated in excess sterile RBS overnight (according-to manufacturer's instructions) and autoclaved. They are then rinsed several times in sterile PBS.<br>
Example 2<br>
Biotinylation reagent<br>
The biotinylation reagent, Biotinamidohexanoyl-6-amino-hexanoic acid N-hydroxysuccinimide ester (Sigma B3295 10mg) is used. 10mg of the reagent is dissolved in 400ul dimethylformamide (DMF) and added to 5ml settled volume of hydrated CSG (5ml settled volume in 50ml tube) and mixed well by gentle pipetting ((J.1% v/v BSA/PBS is pipetted up/down into the pipette tips beforehand to prevent sticking of CSG to the inside surface of the tips).<br>
The biotinylation reagent is allowed to incubate with the CSG overnight at room temperature before several rinses in sterile PBS to remove excess biotinylation reagent (rinsing is performed in a 50ml tube with the CSG being allowed to settle between rinses and careful aspiration with a vacuum).<br>
The biotinylated CSG are now ready for use in an experiment. They are stored under sterile conditions in a known volume of PBS and the concentration "in mixed suspension is calculated based on this volume and the dry mass of CSG added to the tube initially (typically we have 2000 GSG in 100ul)<br>
Example 3<br>
Tagging methods in wells and columns<br>
AHer the seeding of CSG with ES cells, the fluorescent tags are added to tag each stage in the split-pool matrix. The following method for tagging biotinylated CSG with -streptavidin-coated tags is used.<br>
The experiments are.conducted Tn a-5x5 well (10cm-x 1Qcrri).cell culture dish (each well therefore measures 2cm x 2cm). Typically, one well will contain 7DQO CSG/ES cell complexes.<br>
The fluorescent-tags- are-added to the well at a known^Goncentration of tags/CSG. The mixture is gently pipetted to mix (using 0.1% BSA/PBS-treated tips to prevent sticking) and then the whole 5x5 dish is tilted to facilitate dose contact of the CSG and tags.<br>
Thedish-is-placed in a 37°G incubator for approximately 1hr and then placed flaLuntil the next stage of the split-pool experiment.<br>
Example 4<br>
Washing methods and sieving<br>
The contents of the wells which have received the same treatment are gently pooled together and washed through a 70um sieve. The CSG are larger than 70um and are<br>
caught in the sieve (see Figure 2). A minimal volume of PBS (typically 5ml) is washed through the sieve to remove unbound, fluorescent tags and the sieve is then inverted and 
The contents of all wells containing different culture conditions are washed separately before being pooled together prior to splitting for the next round of culture conditions.<br>
Example 5<br>
•Digestion .of CSG and Cytopore 2<br>
The following methods are used to digest the beads prior to analysis of the attached fluorescent Bangs' tags or non-fluorescent Duke tags (as shown in Figure 1) on Cytopore 2 er CSG.<br>
(a)	CSG<br>
Incubation of individual CSG -beads in 2U/ml proteinase K (Sigma P4850) in wells of glass~-bottomed, 384~-well mlcrotitre plates results in complete digestion of the CSG bead whilst leaving the polystyrene, fluorescent tags intact. Complete digestion is typically complete within 20 mlns and the fluorescent tags then remain in the proteinase k solution indefinitely, without inactivation .of proteinase k, with no detrimental effect to the tags-or their fluorescence (see Figure 3).<br>
(b)	Cytopore 2<br>
Incubation-of cytopore 2 micr-ocarriers (made from cellulose)-in 37% HCI ("-12M) results -in complete digestion of the microeam'er within 90 minutes whilst leaving.-the attached coloured Duke tags intact As with the CSG, the tags remain in the digestion medium indefinitely, without inactivation and with no detrimental effect to the tags (see Figure 4).<br>
Example 6<br>
Flowchart<br>
A spreadsheet is prepared which examines tags in parallel for UV2 (and rhodamine) and Starfire Red (and fluorescein) fluorescence, /e. the flowchart shown in Figure 5 is duplicated in parallel with all references to Fluorescence 1 replaced with Fluorescence 4, and Fluorescence 3 with Fluorescence 2. For this system the final 'No' decisions from pages 2 and 4 will go to a "No Call".<br>
This system can result in a tag remaining unidentified, or given two identities. The latter can occur if the thresholds for the lowest intensity tags are set incorrectly, such that a low fluorescing tag is identified as being both UV2 and Starfire positive. Another example of a double-call is when two tightly associated tags are identified as one by an 'Outlines' macro due to insufficient tightness of the Radius Ratio setting. Additional IF/ AND and COUNT functions have been used to flag up such Double and No Calls.<br>
An alternative flowchart could examine the Fluors 1 and 4 serially, such that the final 'No1 decisions from pages 2 and 4 will go on to -interrogate the Starfire Red values for the relevant size tag as. set fortrr-ln Figure 6. This system can also result in a tag remaining unidentified if its size,and/or fluorescence does-not fall onto the defined ranges. There will not be double calls with-this flowchart, but this is at-the risk-of mis-identifying tow intensity tags or doublets as described above.<br>
Example 7<br>
ES Cell Differentiation usfng Combinatorial Cell Culture and deconvolution of results using labeling methods<br>
In Combinatorial Cell Culture,. ES cell colonies grown on microscopic beads are shuffled through muttiple combinations of growth conditions using random spTrt-pool methodology accompanied by concomitant labeling: At the end of the- process, any "beads bearing differentiated progeny are Identified and isolated; and the associated tags are analysed to deduce the cell culture history.<br>
An example of this process is described using haematopoietic differentiation of mouse ES cells as a model system. The developmental roadmap to haematopoiesis is relatively well charted, and many of the recombinant growth factors that influence cell fate are known and readily available, nevertheless directed differentiation reported in the literature has used materials and methods (embryoid bodies; semisolid culture media;<br>
supplementation with animal serum) which are cumbersome and undefined, and which we have sought to eliminate using the experimental power of the new technique.<br>
In particular we set out to differentiate mouse ES cells to the monocyte - macrophage (mononuclear phagocyte) lineage (Gordon S. &amp; Taylor P.R., Nature Rev. Immunol. 2005 5: 953-964), because this has been reported to occur in vitro and because it is possible to carry out a simple functional screen for these cells based on phagocytosis of fluorescent antigens.<br>
Materials- and methods Reagents<br>
murine stem cell factor- (SCF) (R&amp;D Systems, 455-MC)<br>
murine thrombopoietin (TPO) (R&amp;D Systems, 488-TO)<br>
human erythropoietin (EPO) (R&amp;D Systems, 287-TCjr<br>
human interleakin 6 (I1.-6) (R&amp;D Systems, 206-1L)<br>
human transforming growth factor D1 (TGFD1) (R&amp;D Systems, 240^)<br>
murine macrophage colony stimulating factor (M-CSF) (R&amp;D-Systems, 416-ML)<br>
murine interleukin 3 $li-3) (R&amp;D Systems, 403-ML)<br>
human bone morphogenetic protein 2 (BMP2) (R&amp;D Systems, 355^BM)<br>
human fibroblast growth-factor (bFGF) (R&amp;D Systems, 233-FB)<br>
retinoic acid (Sigma, R2625)<br>
bovine Insulin (Ins) (SigmaJ0516)<br>
insulin/transferrin/selenium supplement (ITS) (Sigma, 13146)<br>
Micrecutture<br>
CultiSpher-G microcarriers (Percell Biolytica AB) were hydrated and sterilized according to manufacturer's recommendations. Biotinylation was performed by addition of 10mg Biotinamidohexanoyl-6-amino-hexanoic acid N-hydroxysuccinimide ester (Sigma), dissolved in 0.4ml N.N-dimethyfformamide (Sigma), to 5ml of Gaz*/Mg2*-free PBS (CMF-PBS) containing 3x105 microcarriers and incubation overnight at room temperature, followed by 5 washes using an excess of PBS. The biotinylated microcarriers were stored in CMF-PBS.<br>
D3 ES cells (ATTC no. CRL-1934) were grown on gelatine-coated plastic in KO-DMEM containing 15% knock-out serum replacement (KOSR), 1% non-essential amino acids (NEM)±1% Glutamax, 0.5% penicillin/streptomycin, 0.1 mM D-mercaptoethanol (D-ME; Sigma) and 1000U/ml Leukemia Inhibitory Factor (LIF; Chemicon); all from Invitrogen unless indicated otherwise.<br>
On the day preceding day 1 of the experiment, approximately 1.4 x 10s biotinylated micocarriers equilibrated in medium A (IMDM (Gibco), 15% KOSR, 1% NEAA, 0.5% pen/strep, 0.1mM D-ME, 1000U/ml LIF and 1.5x1 O^M 1-thioglycerol (MTG; Sigma)) were added to 10Orrrl of medium A-containing 3 x 107 ES cePs, aliquoted equally into wells-of a-100mm square petri dish-(25 wells; Bibby Sterilin) and incubated overnight<br>
An aliquot of seeded microcarriers (-100 beads) were fixed in 4% paraformaldehyde {Sigma) for 10min at- RT, washed and re-suspended Jn PBS, and. stained with Vector -Blue Alkaline Phospnatase Substrate Kit HI (Vector Laboratories) according to the manufacturer's instructions.<br>
Combinatorial Cell Culture.<br>
Seeded microcaniiers were transferred into a 700m nylon -cell strainer ^Falcon) and washed with T5ml PBS, then transferred into an excess of medium B (Stemline™ Haematopoietic Expansion .Medium (Sigma).containing 1.5xK)-4-M MTG). On days 1,_4, 6, 8 and 10 microcarriers were split equally as required by the experimental plan and -each sample incubated in-wells of a- 100mmr 25 well, square petri dish such that each well was loaded with approx. 5000 beads in 4ml medium B containing the relevant chemicals-and/or growth-factors, and 1xt06 streptavTdtn-coated mfcrosphere tags. This, -procedure was followed for each-split-pool cycle except following the split-on D10, wnen micrjacarriersr were not tagged. Following D10, trie nffcrocaiTiers were processed separately (i.e. not pooled) so tagging was not required.<br>
On day 13 of the experiment, 1mg of the macrophage assay reagent DQ-ovalbumin (Molecular Probes) was made up in 0.4ml PBS and added to each sample at a dilution of 1:100. Following incubation for at least 4h, the medium was aspirated and replaced with PBS. The samples were examined on a Nikon TE2000-S inverted epifluorescent microscope using a FITC filter set to identify microcarriers bearing large, round cells internally labelled with green fluorescence.<br>
Positive microcarriers were transferred by pipette into the well of a 384 well glass-bottom assay plate (Bibby Sterilin) containing a solution of proteinase K (2U/ml; Sigma) in PBS and incubated at 37°C for SOmin, following which time fluorescent tags could be seen arrayed on the glass surface.<br>
Tags<br>
A set of 50 tags (Fig 13), each set comprising a population of fluorescent, streptavidin-coated polystyrene microspheres, was commissioned from Bangs Laboratories. (Fishers, IN). The difference between the 50 sets was a function, of the microsphere diameter-1 (sized at 1.87um, 4.41 urn, 5.78um, 7.37um or 9.77uin) and fluorescence emission/brightness (dyed with five intensities of either Starfire Red or UV2). Aliquots of these 50 sets were further conjugated with tetramethylrhodamine-5-(and-6)-isothiocyanate (TRITC; InvTtrogen) according to the manufacturer's instructions, to produce a-maximum of 100 discrete tags. From these, a set of 28 highly distinctive tags were used in-the experifnent-(Table 1).<br>
Tag analysis<br>
Tags were analysed by -microscopy using a Nikon TE2000-S inverted eptfluorescent microscope equipped-w'rth filter sets for visualization of the fluorophorea TRITC, _DAPI (UV2X GFP-B (all from Nikon) and Cy5 (Chroma Technology). Images were captured using an-Evolution VF cooled-monochrome camera and image analysis was using Image Pro Plus (both from Media Cybernetics). The outlines of single microspheres were captured using bright-field illumination and the areas within these calculated to size the tags. The area^ fluorescence, intensities in trie- UV2, Cy5 and- TRITC channels were used to further specify the identity of-each tag-by comparison to reference samples containing known tags (e.g. Fig T4).<br>
Results<br>
An experimental matrix of cell culture conditions is devised which we hypothesised contained one or more pathways capable of directing ES cell differentiation to macrophage (Table 1). The matrix was comprised of six alternative culture conditions on the first day (D1) of the experiment, followed by six alternative conditions on D4, a further<br>
eight on D6, eight more on D8 and finally six alternatives on D10. The total number of possible pathways through this matrix, i.e. the number of different combinations of cell culture conditions tested, was 13,824 (= 6x6x8x8x6 = the 'experimental complexity'). All conditions were based on a commercially available medium which permits growth of human haematopoietic progenitors, but varied in supplementation with different growth factors and morphogens known to influence mesoderm formation, haematopoietic development and commitment to the macrophage lineage (Kaushansky K., N. Engl. J. Med. 2006 354: 2034-2045; Godin I. &amp; Cumano A. 2002 2: 593-603).<br>
Pluripotent mouse ES cells were seeded on bibtinylated tnacroporous -gelatine microcarrier beads in the presence of LIF,-where they formed cell colonies that stained positive for alkaline phosphatase activity (Fig 8). Following LIF withdrawal, the beads were systematically passaged through all possible combinations of conditions specified in the experimental matrix using an iterative split-pool methodology as described in WO 04013969.<br>
At the beginning of the experiment approximately-1A X..105 seeded microcarrier beads were split randomly into six sets, each of which was cultured separately in one of the. six different media specified for D1 in the matrix. F.our days later the-beads-were washed, pooled, and randomly split again into six sets each-of which was cultured in-one-of the media specified on D4; and this split-pool procedure repeated on D6, D8 and D10. Each time an aliquot of beads was euttured-fri a different medium, those beads were labelled with a unique tag; except on D10, following which the split aliquots were handled separately and therefore did not require labelling-Thus, the-movement of any given bead through the various culture media could be inferred by analysing the collection of tags associated with it. The labelling strategy we employed involved an- array of uniquely distihstive, streptavidin-coatetf-fluorescent microsphere tags which bound to -the biotinylated microcarrier substrate.<br>
At the end of the experiment, D13, the microcarriers were exposed to a hyper-fluoresceinated (self-quenched) ovalbumin antigen. This reagent is known to be specifically internalised and digested by phagocytic macrophages, thus dispersing the fluorophore and giving rise to a bright, intracellular signal (Fig 9 &amp; 10). On screening the beads we found that approximately 100 (i.e. 0.07% of the 1.4 x 10s input beads) bore large, round, internally fluorescent cells. It was assumed that these beads had been passaged through conditions that were either permissive or instructive for ES cell<br>
differentiation to macrophage. Beads carrying large numbers of macrophages were analysed to deconvolute their cell culture history: these beads were isolated and the gelatine substrate proteolysed to release bound tags, which were categorised using fluorescence microscopy followed by image analysis.<br>
Some microcarriers were labelled with large numbers of tags, all of which could be unambiguously categorised into four different species, revealing the pathway of these beads through the experimental matrix. For instance, bead C5 was found to have tags corresponding to the pathway 1.2-+ 4.2-&gt; 6.1-&gt; 8.2 and was isolated from the group of beads finally partitioned into condition 10.5 (Fig 1 Ta).<br>
Some other microcarriers (e.g. bead A22) were labelled with large numbers of tags, of which the majority fell into the four different species acquired through labelling, and a small minority appeared to. be acquired through adventitious transfer between different microcarriers (Fig 11b). In most of these latter cases, there was a large excess of one-class of tag, which allowed a measure of confidence in deconvoluting (he cell culture history. In.a few cases (normally where overall labelling was-inefficient, e.g. bead E6) there were equal numbers of tags from two different species added on- the -same day, giving -rise to an ambiguity (Fig 11c). Both these types" of ambiguities were easily resolved on further experimentation to validate the results, as described below.<br>
A batch of plurjpotent mES" cells- seeded on microcarriers was divided into a series of aliquots, each of which was used to test one putative differentiation pathway as defined by Combinatorial Cell Culture: Ambiguous pathways <e.g. a22 and-e6 were tested by-assaying each alternative in parallel. using this method bead was determined to have passed through conditions>- 6.4-* 8.2-* 10.3. Similarly, the-route of bead E6 through the matrix was T.2—&gt; 4.2-&gt;- 6".3-» 8.4-» tD.5&gt;<br>
It is interesting that these pathways which produced macrophage had certain conditions in-common. In particular, treatments on D1 and D4 were consistent in all three, possibly revealing a requirement for those conditions in the generation of haematopoietic progenitors using this system. When reproduced, these pathways consistently resulted in 10-25% of all microcarriers being decorated with substantial numbers of macrophages (Fig 12). Deviations from the optimal pathways were found to be generally detrimental to macrophage differentiation. Similarly, when random pathways through the matrix were<br>
tested for the ability to produce macrophage, no effective protocols were found suggesting that these are relatively rare.<br>
Conclusions<br>
An example of mES cell differentiation is described using Combinatorial Cell Culture featuring the use of labels or tags. The process was used to screen an experimental matrix comprising almost 14,000 different cell culture protocols for conditions leading to differentiation to the monocyte-macrophage lineage. By screening such a large number of potential pathways and using a suitable labeling strategy it was possible to Identify multiple differentiation protocols, and to overcome the use of embryoid bodies and animal serum, one or both of which are normally required for monocyte-macrophage development in vitro.<br>
Example 8<br>
Digestion of Cultispher-G microcaniers on-glass slides for tag deconvolution Reagents and Equipment<br>
•	Proteinase K (Sigma P4850 1 ml)<br>
•	Sigmacote (Sigma SL-2)<br>
•	Glass microscope slides<br>
•	68°C oven<br>
•	Home-made humidified box<br>
•	GJycerol (or Citifluor)<br>
•	Forceps<br>
•	13mm-diameter glass coverslips<br>
Methods<br>
Microcaniers are digested individually to reveal the complement of tags that are attached. Digestion is carried out on a glass microscope slide (can digest up to 6 microcarriers on one slide)<br>
Coat glass microscope slides with Sigmacote in fume hood: holding slide horizontally, carefully pipette 1ml of Sigmacote onto the top of the slide, ensuring that it covers the entire glass surface. Leave in place for approx 30s and then carefully tip the slide and remove the Sigmacote from the edge with a pipette. The Sigmacote may be re-used so return to the bottle. Most of the Sigmacote is removed in this way but do allow any remaining Sigmacote to evaporate completely. If there are any traces of Sigmacote left on the slide or if there are any smears, wipe the surfaces of the slide with gloved fingers to produce clean surfaces on the slide<br>
Ensure that -microcarriers to be digested have, been carefully washed in dH2O to remove -any traces of medium and/or salts. Using a 20ul pipette tip, carefully remove individual microcarriers from the dHzO in a minimal volume (1-2uf max) and deposit on the-glass slide. It is not always necessary to pipette the whole 1-2ul volume onto the slide; just pipette enough "to deposit the microcarrier on the slide (ensure the •microcarriers are sufficiently spaced to -allow later addition of a coverslip without overlap - 5-6 microcarriers per slide is-achievabie with care)<br>
Carefully place the slides in a. 68°C" oven to achieve, complete -dehydration of the micr-ocarriers and sticking, to the slide (10 rains should-suffice):<br>
Make a fresh 5U/ml solution of proteinase' k in dH2O (note that the concentration of proteinase k stock varies between -batches so the dilution-should be .calculated with each new batch (eg. Stock solution 1230U/ml to 5U/ml = 5/1230 = 1/246. ie. 1 in 246 djlutiorrof- stock solution i<br>
Prepare a humidified box in which the digestion reaction will be performed (a 10cm x tGcm .cell-culture Petri -works well with disposable plastic pipettes- cut to size to fit along 2 sides-. .Place water-wetted tissue in&amp;etween the pipettes)<br>
Carefully but quickly pipette 0.5ul of 5U/ml proteinase-k solution directly onto dried microcarriers (this can be quite fiddly as O.Sul is a very small volume and will evaporate quickly outside the humidified box). Once all the microcarriers have had proteinase k added to them, place the slide in the humidified box and cover with a lid. Leave at room temperature, protected from light as much as possible, until the microcarriers have dissolved (typically 3Q-60mins). It is necessary to remove the slides from the humidified box to check for complete digestion and this needs to be<br>
performed as quickly as possible before immediate return of the slides to the box to avoid evaporation of proteinase k soln)<br>
Once satisfied that the microcarriers have completely dissolved, remove the slides from the box and allow to sit at room temp to allow evaporation of the proteinase k soln (this will proceed quite quickly). Place the slides in a 68°C oven for approx 10 mins to ensure complete drying of the released tags onto the slide).<br>
Carefully add a 6ul drop of glycerol directly to the dried tags, taking care to avoid air bubbles (if air bubbles are present in the-drop it is often easy to remove them by pricking with a small needle). Using- forceps, carefully lower a dust-free, 13mm-diameter coversfip onto the glycerol droplet, taking extreme care to avoid air bubbles (it is almost impossible to remove air bubbles from under the coverslip once in place). Allow the coverslip to settle (the glycerol will spread throughout the entire coverslip. Protect slides from light as much as possible bat be careful not to disturb the coverslips too much and spread glycerol over the surface.<br>
Analyse tags using a microscope: Example 9<br>
Digestion ofCytapoFe 2 micFocarriers<br>
Reagents and Equipment<br>
•	'Lab coat safety-glasses, gloves<br>
•	^Hydrochloric-acid, 37%<br>
•	Petri dish<br>
•	Thin-walled PCR tubes<br>
•	Sigmacote (Sigma cat # SL-2)<br>
•	Citifluor mounting medium (Agar scientific)<br>
Method<br>
Following a split-pool experiment, microcarriers are digested individually to reveal the complement of tags that are attached. Digestion of individual microcarriers is carried out in a thin-walled PCR tube in a thermal cycler or heater block at 65°C.<br>
1.	Draw individual cytopore-2 microcarrier + tags (+cells) into pipette tip in smallest<br>
volume possible, typically 
5ul drop of 37% HCI in petri dish. Move quickly to next step...<br>
2.	Quickly draw up the microcarrier into a pipette tip set to 1 ul and deposit into the<br>
very bottom of a thin-walled PCR tube and close the lid<br>
3.	Place the tube in a thermal cycler or heater block set at 65°C for 7 mins (check<br>
that the carrier 'has dissolved by inspection through the tube wall using a<br>
microscope and increase-time by another 2 mins if necessary.   Carrier should<br>
have-dissolved after 7-9 mins).<br>
•4. Treat glass microscope slide with Sigmacote.<br>
5-. Treat 2ul pipette tip with--sigmacote, ensuring -that it dries completely, before carefully removing the:HGLTn the PCR tube containing4he tags.<br>
&amp;. Deposit 1ul droplet onto microscope slide. Invert so-drop is hanging drop and incubate in 68oC dry over until completely dry (15-30 mins). Add drop of Citifluor and eoverslip. Analyse tags using a microscope.<br>
Example 10<br>
Labelling cell units with-rod-shapedtags^<br>
3500 CultispherG microcarriers were seeded with D3 mouse ES cells in a single cell suspension at a density of 50 cells per carrier in 2ml growth medium comprising KO-DMEM + 15% KO-SR, 100U/ml penicillin, 50ug/ml streptomycin, 2mM GlutaMAX, 1X NEAA, 1000U/ml LIF, 100uM pME, and the cell unite cultured overnight at 37°C.<br>
Neutral red vital staining was performed on the cell units prior to addition of nanobarcode particles to confirm viability of cells before particle addition.<br>
Suspensions of rod-shaped particles comprising 6 micron-long Ag/Au coated aluminium nanowires (Nicewamer-Pena, S.R. et al., Science 294: 137-141, 2001) were obtained from Oxonica Healthcare (Kidlington. UK) at a concentration of 1x109 particles per ml. Two barcodes designated 101010 and 100001 (where 1 = Ag and 0 = Au) were used to label the cell units.<br>
A sample of the rod-shaped particles was removed from the stock and sterilised in 95% ethanol for 1 hr before washing and resuspension in sterile PBS. A mixture of the two nanobarcode particles was added to -875-cell units in 2ml medium in a well of-a 251weH cell culture dish at a ratio -of 1-000 particles per cell unit. The cell culture dish was incubated at 37°C overnight on an orbital shaker.<br>
After 24 hours in culture the labelled cell units were placed on a 70um filter to remove unbound rod-shaped particles Bnd washed 5 times with 5ml PBS. Individual cell units were removed and the- continued viability of the D3 cells in the presence of the rod-shaped particles assessed -by neutral red-vital stain.<br>
Individual cell units were deposited on Slgmacote-treated glass microscope slides and" placed-at-68°CTfor 10-15 rains to ensure complete dehydration. Fresh 5U/ml protelnase k in H2O was prepared and O.Sul deposited directly on the dehydrated material. Slides were immediately^placed in a hurtridiffed chamber uatil the gelatine material had digested (typically 30-60 mins) and then removed to a 68°C incubator to allow complete evaporation of the pceteinasek solution (typically 10 mins).<br>
A 5uJ drop of glycerol was applied-directly to the dried spot-and a~!3mm round-coverslip applied". Microscopic images-of "the released rod-shaped particles were captured" using brightfleld-illumination with a~Nikon TE2000S inverted epifluerescent microscope:<br>
Discrimination of the rod-shaped particles in the sample was achieved using a Deltavision RT microscope equipped with a Hg lamp and a Chroma CFP filter set (Ex. 436/10; Em. 465/30) with a 100x oil immersion objective.<br>
Further Aspects<br>
Further aspects and embodiments are presented in the following numbered paragraphs.<br>
1.	A complex comprising a microcarrier and a microsphere or microbead.<br>
2.	The complex according to paragraph 1, wherein the microcarrier is a Cultispher-G<br>
microcarrier or a Cytopore 2 microcarrier.<br>
3.	The complex according to paragraph 1 or paragraph 2, wherein the microsphere or<br>
microbead is a fluorescent and/or coloured microsphere or microbead.<br>
4.	The complex according to any of the preceding paragraphs, wferein-the microcarrier<br>
is biotinylated.<br>
5.	The complex according to paragraph 4, wherein the fluorescent microsphere is a<br>
hydrophilic microsphere.<br>
6.	The complex according to paragraph -5, wherein the hydrophilic microsphere is a<br>
carboxylate-modified (CML) microsphere.<br>
7.	The complex according.to paragraph 6, wherein the complex comprises a Cytopore 2<br>
microcarrier and a CMt microsphere.<br><br>
8.	The   complex   according   to   paragraph  -7,    wherein   the   ratio   of   CML<br>
microsphererCytopore 2 microcarrier is about 1:1.<br>
9.	The complex according to any of paragraphs 1-4, wherein the microbead is a<br>
polystyrene microbead.<br>
10.	The complex-aceording to any of-paragraphs 1-4 and 9, wherein the-microbead is<br>
streptavidin-coated.<br><br>
11.	The complex according to paragraph 10, wherein the complex comprises a<br>
Cultispher-G microcarrier and a polystyrene microbead.<br>
12.	The complex according to any of paragraphs 3-11, wherein the fluorophore is<br>
TRITC.<br>
13.	A method for separating a complex comprising a Cultispher-G microcarrier and a<br>
microsphere, comprising the step of contacting said complex with proteinase K.<br>
14.	The method according to paragraph 13, wherein the proteinase K is used in an<br>
amount of 2U/ml.<br>
15.	The method according to paragraph 13 or paragraph 14, wherein the complex is<br>
contacted with proteinase K for about 20 minutes.<br>
16.	A method for separating a complex comprising a Cytopore "2 and a microbead,<br>
comprising the step of contacting said complex with HCI.<br>
17.	The method according, to paragraph 16, wherein 37% HCI (-12M) is used.<br>
18.	The method according to paragraph 16 or paragraph 17, wherein the complex is<br>
contacted with HCI for about 90 minutes<br>
19.	A method for determining the effect of a plurality of culture conditions on a cell<br>
comprising the use ofa complex according to any of paragraphs 1-12.<br>
"20. A method for determining the effect of a plurality of culture conditions on a cell, comprising, the-steps of:<br>
(a)	providing- a first set of groups of cell -units each comprising one or more eells, and<br>
exposing said groups to desired culture conditions;<br>
(b)	subdividing one or more of said groups to create a further set of groups of cellunits;-<br>
(c)	exposing said further groups to further desired culture conditions;<br>
(d)	optionally, repeating steps (b) - (c) iteratively as required; and<br>
(e)	assessing the effect on a given cell unit of the culture conditions to which it has been<br>
exposed,<br>
wherein each cell unit comprises one or more cells adherent to or bounded by a complex according to any of paragraphs 1-12.<br>
21.   A method for determining the effect of a plurality of culture conditions on a cell, comprising the steps of:<br>
a) providing a first set of groups of cell units each comprising one or more cells, and exposing said groups to desired culture conditions;<br>
(b)	pooling two-or more of said groups to form at least'one second pool;<br>
(c)	subdividing the second pool to create a further set of groups of cell units;<br>
(d)	exposing said further groups to desired-culture conditions;<br>
(e). optionally, repeating.steps (b) - (d) rteratively as required; and<br>
(f) assessing 1he effect on a given cell unit of the culture conditions to which it has been exposed,<br>
wherein each cell unit comprises one or more cells adherent to or bounded by a complex according to any of paragraphs-1-12.<br>
22, A method" for exposing a cell to-a variety of cell culture-conditions, comprising the steps of:<br>
a) providing a first set of groups of cell units each comprising one or more cells, and exposing said groups to-desired culture-conditions;<br>
(b)	pooling two or more of said groups to form at least- one second pool;<br>
(c)	subdividing the second pool to create a further set of groups of cell units;<br>
(d)	exposing said further groups to desired culture conditions; and<br>
(e)	optionally, repeating steps (b) - (d) iteratively as required,<br>
wherein each cell unit comprises one or more cells adherent to or bounded by a complex according to any of paragraphs 1-12.<br>
23.  A method for determining the effect of a plurality of culture conditions on a cell, comprising the steps of:<br>
(a)	providing a first set of groups of cell units each comprising one or more cells, and<br>
exposing said groups to desired culture conditions;<br>
(b)	pooling two or more 0f said groups to form at least-one second pool;<br>
(c)	subdividing the second pool to create a further set of groups of cell units;<br>
(d)	exposing said further groups to desired culture conditions;<br>
(e)	optionally, repeating steps (b) - (d) iteratively as required;-and<br>
(f)	assessing the-effect on a given cell unit of the cCifture conditions to-whictvit has been<br>
exposed,<br>
wherein each cell unit-comprises one or more-cells-adherent to or bounded by a complex according to any of paragraphs 1-12.<br>
24.	A method according to any of paragraphs 20-23, wherein the cells are cultured in<br>
cell units, each cell unit comprising one or more cells.<br>
25.	A method according to -paragraph 24, wherein the cell units jare single cells.<br>
26.	A method according fo any of paragraphs 20-25, wherein the culture conditions are<br>
media to which the cell is exposed.<br>
27.	A method according to paragraph 26, wherein the media contain one or more<br>
specific agents which influence a cellular process.<br>
28.	A method according to any of paragraphs 20-27, wherein the cell culture conditions<br>
comprise culturing at one or more specific temperatures.<br>
29.	A method according to any of paragraphs 20-28, wherein the cell culture conditions<br>
comprise culturing on one or more specific substrates.<br>
30.	A method for identifying- a gene which influences a cellular process, comprising<br>
the steps of:<br>
a&gt; determining the effect of one or more culture -conditions on a cell unit, in accordance •with any one of paragraphs 20-29;<br>
b)	analysing gene expression in said cell units when exposed to said culture conditions;<br>
and<br>
c)	identifying genes which are differentially expressed under desired culture conditions.<br>
31.	A method- according to paragraph 30, wherein the desired culture conditions<br>
influence a cellular process.<br>
32- A method for producing a nucleic acid- which encodes a gene product which influences a cellular process, comprising identifying- a gene in accordance witft paragraph 30 or paragraph 31,- and producing at least .the coding region of said gene by nucleic acid synthesis or biological replication.<br>
33.	A~ method for inducing a cellular process, comprising the steps of:<br>
(a)	identifying one or more genes which are differentially expressed in association with<br>
the cellular process in accordance with paragraph 30 or paragraph 31; and<br>
(b)	modulating the expression of said one or more genes in the cell.<br>
34.	A method according to paragraph 33, wherein modulation of gene expression in<br>
the cell comprises transfection of said one or more genes into the cell.<br>
35.	A method according to paragraph 33, wherein modulation of gene expression<br>
comprises the exogenous administration of a gene product.<br>
36.	A method for identifying the state of a cellular process of a cell, comprising the<br>
steps of:<br><br>
(a)	identifying one or more genes which are differentially expressed in association with<br>
the cellular process in accordance with paragraph 30 or paragraph 31; and<br>
(b)	detecting the modulation of- expression of said ofie or more genes in .a cell, thereby<br>
determining the state-of-the cellular process of said cell.<br>
37.	A method according to paragraph 36, wherein said one or more genes encode a<br>
marker.<br>
38.	A method according to paragraph 37, wherein said marker may be detected by<br>
-an immunoassay.<br>
39.	A method for inducing a cellular processi comprising the steps of:<br>
(a) determining the effect of one or more culture conditions on a cell unit, in accordance withiany one of paragraphs 20 to 29;<br>
(b)=exposing -a celLto culture conditions which-induce the cellalar process; and .(c) isolating the desired celt<br>
40.	A method for identifying-an- agent which is capable of inducing a cellular process,<br>
comprising the steps of:<br>
(a)	determining the effect of one or more agents on a cell unit, in accordance with any<br>
one of paragraphs 20 to 29; and<br>
(b)	identifying those agent(s) which induce the cellular process in the cell units.<br>
41.	A method for preparing an agent which is capable of inducing a cellular process,<br>
comprising the steps of:<br>
(a)	determining the effect of one or more- agents on a cell unit, in accordance with any<br>
one of paragraphs 20 to 29;<br>
(b)	identifying those agent(s) which induce the desired cellular process in the cell units;<br>
and<br>
c) synthesising or isolating the agent(s).<br>
42.	A method according to any of the preceding claims, wherein the cellular process<br>
is cell proliferation or differentiation.<br>
43.	A method for culturing stem cells or cells that-tiave been derived from stem cells<br>
jn vitro comprising the steps of:<br><br>
a)	incubating a stem cell-culture; and-<br>
b)	splitting said culture into two or more groups of stem cells, and cultur-ing said<br>
group of stem- cells under two or more different sets of culture conditions<br>
wherein each cell unit comprises one-or more cells-adherent .to or bounded by a complex according to any of paragraphs 1 -12.<br>
43.	A method according to paragraph 43, whereinthe cells are-cuftured in cell units,<br>
each cell unit comprising one or more cells.<br>
44.	A method according to paragraph 42 or paragraph 43, wherein the cell units are<br>
single cells.<br>
45.	A method for culturing stem cells, comprising growing said stem cells adhered to<br>
a complex according to any of paragraphs 1-12.<br>
46.	A method according to paragraph 45, wherein said stem cells are subjected to at<br>
least one change of culture conditions.<br>
47.	A method according to paragraph 46, wherein said change of culture conditions<br>
comprises a change of medium.<br>
48.	A method for obtaining differentiated cells from stem cells in vifro, comprising the<br>
steps of:<br><br>
(a)	growing stem cells adherent to the complex according to any of paragraphs 1-12 in a<br>
culture medium;<br>
(b)	transferring the complex from one culture medium to another;<br>
(c)	optionally repeating step (b)'as required; and<br>
{d&gt; obtaining the differentiated cells attached to the complex.<br>
49.	A method according to paragraph 48, wherein the differentiated cells are isolated by<br>
enzymatic or chemical detachment from the complex.<br>
50.	A method-according to paragraph 49, wherein the enzyme is proteinase K.<br>
51.	A method according to paragraph 49, wherein the chemical is HCI.<br>
52.	A method according -to any of paragraphs 48Tto 54, wherein the differentiated ceHs-<br>
are isolated by digestion of the complex according to any of paragraphs 1-12-.<br>
53.	A method of growing pluripotent stem cells in vitro comprising the steps of:<br><br>
(a)	seeding said cells on the complex according to any of paragraphs 1-12; and<br>
(b)	propagating the cells while attached-to the complex.<br>
54.	A method for identifying one or more area and/or density values for a tag obtained<br>
from a cell unit comprising the steps of:<br>
(B) obtaining one or more images of the tag;<br>
(b)	drawing outlines for the one or more images;<br>
(c)	loading the outlines for the one or more images; and<br>
(d)	reading the area and/or brightness within the one or more outlines.<br>
55.	A method for identifying one or more area and/or density values for a tag obtained<br>
from a cell unit comprising the steps of:<br>
(a)	separating the cell unit and the tag;<br>
(b)	obtaining one or more images of the tag using a microscopic technique;<br>
(c)	drawing outlines for the one or more images;<br>
(d)	loading the outlines for the one or more images onto further corresponding images<br>
obtained using a different microscopic technique; and<br>
(e)	reading the area and/or brightness within the one or more outlines.<br><br>
56.	A-method" according to paragraph 55, wherein step (e) comprises counting and/or<br>
classifying one or more different tags in one or more cell units.<br>
57.	A method according-to paragraph 54 or 55, wherein the-method comprises counting<br>
and/or classifying at least different types of tags in one or more cell units.<br>
58.	A method^according to-paragraph 54 or 55, wherein the method comprises counting<br>
and Jor- classifying up to about 10~orup to about 50 different types of tags in one or more<br>
cell units.<br>
59.	A method according to any of paragraphs 54:58, wherein the area and/or density<br>
values are inputted into a spreadsheet<br>
60.	A method according to any of paragraphs 54-59, wherein-the image is selected from<br>
the group consisting of a phase image, a blue image, a green image, a near red image<br>
and a far red image.<br>
61.	A computer program product including -a computer- program for controlling a<br>
computer to perform a method as claimed in any one of paragraphs 54-60.<br>
62.	An apparatus for identifying one or more area and/or density values for a tag, said<br>
apparatus comprising data processing- logic operable to perform data processing<br>
operations in accordance with a method as claimed in any one of paragraphs 54-60.<br>
63.	A method for determining the chronology and identity of the cell culture conditions to<br>
which a cell unit has been exposed comprising the steps of:<br><br>
68<br>
(a)	comparing the upper and lower limits of at least one parameter of a tag with the mean<br>
measurement of the at least one parameter of the tag;<br>
(b)	comparing the mean measurement for the at least one parameter of the tag with the<br>
lowest limit for the at least one parameter of the tag; and<br>
(c)	correlating the measurement of the at least one parameter of the tag with the identity<br>
of the tag associated with the cell unit.'<br>
"64. A method accordingrto paragraph 63,-wherein if the lower limit is less than the mean measurement and the mean measurement is less then than the upper limit then the upper and lower limits of a further parameter of the tag are compared with the mean measurement for at least one "further parameter of the tag,<br>
65.	A method according to paragraph 63,-wherein if the lower limit isTiot less than the<br>
mean measurement and the mean measurement is not less then than the upper limit<br>
then the upper and lower limits of at least one further integer of the same parameter of<br>
•the tag is measured.<br>
66.	A method according to paragraph 65, wherein if the lower limit Is less than the mean<br>
measurement and the mean-measurement-is less thenJhan the upper limit then the at<br>
least one further parameter for the tag is measured.<br>
67.	A method according to any of paragraphs 63-66, wherein if the mean measurement<br>
for-the at least one parameter of-the tag is greater than the .lowest limit for at least one<br>
parameter efthe-tag, then the result is correlated with one or more tags that possess trie-<br>
one ormoFeparameters.<br>
68.	A method according to any of paragraphs 63-66, wherein if the mean measurement<br>
for the at least one parameter of the tag is not greater than the lowest -limit for the one or<br>
more parameters of the tag, then the result is correlated with a tag that possess the<br>
parameter.<br>
69.	A method according to paragraph 63, wherein if the lower limit is not less than the<br>
mean measurement and the mean measurement is not less then than the upper limit for<br>
the at least one further parameter then the upper and lower limits of at least one further integer of the same parameter of the tag is measured.<br>
70.	A method according to paragraph 68, wherein the method is repeated for one or<br>
more further integers of the same parameter of the tag.<br>
71.	A method according to paragraph 69, wherein the method is repeated for all integers<br>
of the same parameter of the tag.<br>
72.	A-method according 1o .paragraph 68, wherein if the "lower-limit- is less than the-mean<br>
measurement and the mean measurement' is less then than the-upper limit for the at<br>
least one further parameter and the mean measurement for the at least one parameter of<br>
the tag is greater than the lowest limit for at least one parameter of the tag, then the<br>
result is correlated with one or more tags that possess the one or more parameters,<br>
73.	A method according to paragraph 71, wherein if the lower limit- is less than the mean<br>
measurement and the mean measurement is less then than-the upper limit for the-at<br>
least one further parameter and^the mean measurement for the at least one parameter of<br>
Ihetag is not greater than the-lowest limit for the one .or more parameters of the tag, then<br>
the result is correlated with a-tag that-possess the parameter.<br>
74.	A.rnethod according to-any of-paragraphs 62-72, wherein the parameters) of the tag.<br>
are selected from the group consisting of size of the tags, the type of fluorophore that<br>
they carry; and the quantity of fluorophore that they carry.<br>
75: A method .according to paragraph 73, wherein the sizes are-selected from 5 different sizes.<br>
76.	A method according to paragraph 73 or paragraph 74, wherein the sizes are<br>
selected from the group consisting of about 1.87um, about 4.41um, about 5.37um, about<br>
5.78um and about 9.66um.<br>
77.	A method according to any of paragraphs 73-75, wherein the fluorophore is either<br>
UV2 or Starfire Red.<br>
78.	A method according to any of claims 73-75, wherein the quantity of fluorophore is<br>
selected from 5 different quantities.<br>
79.	A method according to paragraph 77, wherein each different quantity gives a 5 to 10-<br>
fold difference in brightness.<br>
80.	A method according to any of paragraphs 62-79, wherein there are up to 3 different<br>
parameters and 50 different integers or populations.<br>
83. 'A-method complex according to any of paragraphs 62-79, -wherein each cell unit comprises one or more cells adherent to or bounded by a -complex comprising a microcarrierand a polystyrene microbead.<br>
82.	A method according to paragraph 80, wherein-the microbead is-streptavidin-coated".<br>
83.	A method according to paragraph 80 or paragraph 81, wherein the complex<br>
comprises a Cultispher-G microcarrierand a polystyrene microbead.<br>
Bxlr A- computer program  product including a computer program for controlling -a--computer-to perform a method as claimed in -any oner of paragraphs 62-82.<br>
-85. An apparatus for determining, the chronology and identity of the cell culture conditions to which a cell unit has been exposed, said apparatus comprising data processing logic operable to perform data -processing--operatrons-in. accordance with a method as claimed in any one of paragraphs 62,82.<br>
TABLE 1<br><br>
(Table Removed)<br>
Legend to Table 1<br>
List of the conditions comprising the experimental matrix, and the corresponding tags used to label microcarriers exposed to those conditions. Tag nomenclature denotes the day of the experiment and the experimental condition (day.condition). Columns 2-5 list the characteristics of the corresponding tag, i.e. size (diameter); fluorophore identity (UV2 or Starfire Red); fluorophore intensity (1 lowest - 5 highest) and whether the tag was TRITC modified. For each condition of the matrix, the final concentrations of the different growth factors and/or morphogens present in the basal medium are indicated in the final column.<br>
All publications mentioned in the above specification are herein incorporated by reference. Various modifications and variations of the described methods and system of the invention will be apparent to those skilled in the art without departing from the scope and spirit of the invention. Although the invention has been described in connection with specific preferred embodiments, it should be understood that the invention as claimed should not be unduly limited to such specific embodiments. Indeed, various modifications of the described modes for carrying out the invention which are obvious to those skilled in molecular biology or related fields are intended to be within the scope of the following claims.<br><br><br><br><br><br>
CLAIMS<br>
1.	A method for determining the cell culture history of a cell unit labelled with more than<br>
one tag comprising the steps of:<br>
(a)	measuring one or more parameters of each tag that is used to label the cell unit;<br>
(b)	identifying each tag in the cell unit; and<br>
(c) correlating the identity of each tag to the identity of the cell unit and/or-the specific cell-culture conditions to which "the cell unit has been exposed.<br>
2.	The method according to claim 1, wherein the method is an automated method.<br>
3.	The method according to claim t or claim 2, wherein the cell unit is bound or adhered<br>
to a microcarrier.<br>
4.	The method according to claim 3 wherein the mierocarrier is a porous or a solid<br>
microcarrier.<br>
5.	The method according to claim 4, wherein the porous microcarrier is selected from<br>
the group consisting of Cytopore microcarrier (eg. a Cytopore 1 microcarrier or a<br>
Cytopore 2 microcarrier),    a Cultispher microcarrier, a Cultispher-G microcarrier, a<br>
Cultispher GL microcarrier and a  Cultispher-S microcaroer, an Informatrix-microcarrier; a<br>
Microsphere microcarrier, a Siran microcarrier, and a Microporous MC microcarrier,<br>
6.	The methodl according to- claim 4, wherein -the soTid microcarrier -is a- Cytodex<br>
microcarrier (eg. -a Cylodex 1, .Cytodex 2 or -Cytodex 3 microcairier=) a Biosilon<br>
microcarrier, a Bioglass microcarrier, a FACT III microcarrier or a DE 52/53 microcarrier.<br>
7.	The method according to any of the preceding claims, wherein the parameters) are<br>
the size of the tags and/or the optical properties of the tags.<br>
8.	The method according to claim 7, wherein the optical properties are selected from the<br>
group consisting of: light reflectivity, colour, the fluorescence emission wavelength(s) and<br>
the fluorescence emission intensity.<br>
9.	The method according to any of the preceding claims, wherein one or more image(s)<br>
of each tag in the cell unit in the field of interest is measured.<br>
10.	The method according to claim 9, wherein one or more image(s) of each tag in the<br>
cell unit in the field of interest is measured using microscopy.<br>
11.	The method according to claim 10, wherein the microscopy is selected from the<br>
group  consisting  of bright field  microscopy,   phase-contrast  microscopy,   oblique<br>
illumination   microscopy,   darlc  field   microscopy,   differential   interference   contrast<br>
microscopy,  reflection  contrast microscopy,  varel  contrast  microscopy,  polarizing<br>
microscopy, interference microscopy and fluorescence microscopy.<br>
12.	The method according to any of claims 9-11, wherein outlines are drawn for the one<br>
or more irnage(s-) of each tag in the cell unit In the field of interest<br><br>
13.	The method according ta any of toe preceding claims, wherein one or more<br>
fluorescence image(s) of each tag in Hie cell uriit-fn the field of interest is measured.<br>
14.	The method -according to claim 1-3, wherein the outlines for the one -or more<br>
image(s) (eg. phase image(s)) are loaded onto the one or more fluorescence image(s).<br>
15.	The method according to claim 13 or claim 14, wherein one or more fluorescence<br>
Tmage(s) of each- tag; in the cell- unit in -the field of interest is measured for each<br>
fluorophore that is used to label the tags.<br>
16. The method-according to any-of claims 12-15, wherein each tag in the cell unit is identified by reading the. one or more parameters of the tag within the one-or more outlines.<br>
17. The method according to claim 16, wherein the one or more parameters of the tag are the area and/or optical density of the tag.<br>
18.	The method according to any of the preceding claims, wherein the one or more<br>
parameters of the tag are inputted into a spreadsheet.<br>
19.	The method according to any of claims 8-18, wherein the fluorophore is selected<br>
from the group consisting of a fluorophore that emits a blue, green, near red or far red<br>
fluorescence.<br>
20.	The method according to any of claims 7-19, wherein the sizes are selected from at<br>
least 3 different sizes (eg. 5, 6 or 7 different sizes).<br>
21.	The method according to claim-,20, wherein the sizes are selected from-the group<br>
consisting of about 1.9µm, about 4.4µm, about 5.4µm, about 5.8µm, about 7.4µm, about<br>
9.7µm and about 9.8µm.<br>
22.	The method according to any of claims 8-21, wherein the fluorophore is selected<br>
from the group consisting of UV2, Starfire Red and TRITC.<br>
23-.   Tha method to any of claims.8-22, wherein the quantity of fluorophore is selected "from 5 different quantities.<br>
24.	The method according, to claim 23, wherein each different quantity gives about a 5 to<br>
-10-fold difference inr brightness.<br>
25.	The method according to any of the preceding claims, wherein there are-at least 2<br>
different parameters.<br>
26.	The-method according to any of the preceding claims, wherein each-parameter has<br>
at least-5 different integers.<br>
27.	The method according to any of claims 1-25, wherein each parameter has less than<br>
or equal to 20 different integers for each parameter.<br>
28.	The method according to any of claims 1-25, wherein each parameter has less than<br>
or equal to 50 different integers for each parameter.<br>
29. A computer program product including a computer program for controlling a computer to perform the method as claimed in any of claims 1-28.<br>
30: An- apparatus for determining the cell culture history of a cell unit comprising data processing logic operable to perform data processing operations in accordance with the method as claimed in any of claims 1-28.<br>
31. A complex comprising a microcarrier and a charged (eg. negatively charged) tag.<br>
32: The complex according to .claim 31, -wherein the microcarrier is -a porous mierocarrier.<br>
33.	The complex according to claim 31 or claim 32, wherein the microcarrier has a net<br>
charge.<br>
34.	The. complex according to any of claims 31-33, wherein flie microcarrier comprises,<br>
consists or consists essentially of protein, cellulose, polyethylene, polystyrol, glass,<br>
collagen, collagen-gylcose-aminoglycan and/or gelatin.<br>
35.	The complex according to any-oGclaims 31-34, wherein the microcarrier is-selected"<br>
from the group, consisting, of a  Cytopore microcarrier, a Cytopore t microcarrier, a<br>
-Cytopore 2- microcanier,    a Cultispher micnocarrier, a Cultispher-G microcarrier, a<br>
Cultispher-GL microcarrier and a Cultispher-S microcarrier, an Informatrix microcarrier, a Microsphere microearrier, a Siran microcarrier, and a Micraporous MC microcarrieT,<br>
-36. The complex according to-any of claims 31-35, wherein the charged tag-is a sphere. 37.~The complex according to-cteirn 36, wherein the sphere is a microsphere.<br>
38.	The complex according to claim 37, wherein the microsp'here is about 9 µM or less in<br>
diameter.<br>
39.	The complex according to claim 37 or claim 38, wherein the microsphere is a<br>
carboxylate modified (CML) microsphere.<br>
40.	The complex according to any of claims 31-39, wherein the tag comprises, consists<br>
or consists essentially of polystyrene and/or latex.<br>
41.	The complex according to any of claims 31-40, wherein- the complex (eg. the<br>
microcamer) is adhered or bound to a cell unit.<br>
42.	The complex according to any of claims 31-41, wherein at least one antibody is<br>
bound to the cell unit.<br><br>
43.	A complex comprising a miorocarrier and a rod-shaped tag,<br>
44.	The complex according to claim 43, wherein the rod-shaped tag is a nanowire.<br>
45.	The complex according to claim 44 wherein the nanowire Is an aluminium nanowire.<br>
46.	The complex according to-claim 44 or claim 45, wherein the nanowire is coated with<br>
silver and/or gold.<br>
47.	The complex according fo any .of. claims 44-46, wherein the nanowire is about 1µM<br>
or less in diameter.<br><br>
48.	The .complex according to any of claims 44-4-7 whereirrthe nanawire is about 10 µM<br>
or less in length.<br>
49.	The complex according to any of claims 43-48, wherein the microcamer is a porous<br>
microcarrier.<br>
50: The complex-according to anyjof claims 43-49, wherein Ihe porous microcarrier is a •charge neutral microcarrier.<br>
51.	The complex according to any of claims 43-50, wherein the microcarrier comprises,<br>
consists or consists essentially of protein, cellulose, polyethylene, polystyrol, glass,<br>
collagen, collagen-gylcose-aminoglycan and/or gelatin .<br>
52.	The complex according to any of claims 43-51, wherein the microcarrier is selected<br>
from the group consisting of a Cytopore microcarrier, a Cytopore 1 microcamer, a<br>
Cytopore 2 microcarrier, a Cultispher microcarrier, a Cultispher-G microcarrier, a Cultispher-GL microcarrier and a Cultispher-S microcanrier, an Informatrix microcarrier, a Microsphere microcarrier, a Siran microcarrier, and a Microporous MC microcarrier,<br>
53.	The complex according to any of claims 43-52, wherein the complex (eg. the<br>
microcarrier) is adhered or bound to a cell unit<br>
54.	The complex according to any of claims 43-53, wherein at least one antibody is<br>
bound to the cell unit.<br>
55.	A method for separating a complex comprising a microcafrier and a tag, oomprising<br>
the step of contacting said complex with a  protease, wherein said microcarrier<br>
comprises, consists or consists essentially of protein.<br>
56.	The method according to claim 55, wherein the microcarrier comprises,.consists-or<br>
consists essentially of gelatin.<br>
57.	The method according, to claim 55 or claim. 56, wherein the microcarrier is a<br>
Cultispher microcarrier.<br>
58.	The method according to claim 57, wherein the microcarrier is selected from the<br>
group consisting, of a Cultispher-G microcarrier, a Cultispher-GL microcarrier and-<br>
Cultispher-S microcarrier.<br>
59.	The method according to any of claims 55-58, wherein the protease is selected from<br>
the group consisting of proteinase K, typsin, thermolysin and caspase.<br>
60. The method according to. claim 59, wherein the proteinase K is used in an amount of about 0.5U/ml or more.<br>
61.	The method according to claim 59 or claim 60, wherein the complex is contacted<br>
with proteinase K for at least 20-60 minutes.<br>
62.	The method according to any of claims 55-61, wherein the complex is contacted with<br>
the protease in a volume of about 5 µl or less.<br>
63.	A method for separating a complex comprising a microcarrier and a tag, comprising<br>
the step of contacting said complex with acid.<br>
64.	The method according to claim 63, wherein the microcarrier is a porous microcarrier.<br>
65.	The method according to claim 63 or claim 64, wherein the microcarrier has a net<br>
charge.<br>
66.	The method according to any of claims 63-65, wherein the microcarrier comprises,<br>
consists or consists essentially of cellulose.<br>
67.	The method according to any of claims 63-66, wherein the microcarrier is a Cytopore<br>
microcarrier.<br>
68.	The method according to any of claims 63-67, wherein the microcarrier is a Cytopore-<br>
2 microcarrier.<br>
69.	The method according to any of claims 63-68, wherein the tag is a sphere.<br>
70.	The method according to claim 69, wherein the sphere Is a micresphere<br>
71.	The method  according to claim 70, wherein the mlcrosphere comprises, consists or-<br>
or consists essentiaUy of polystyrene.<br><br>
72.	The method according to any of claims 63-71, wherein the acid is selected from the<br>
-group consisting of hydrochloric acid, sulphuric acid and sodium hypochlorite.<br>
73.	The method according to claim 72, wherein the acid is 37% hydrochloric, acid (~12M).<br>
74.	The method according to claim 72, wherein the acid is concentrated sulphuric acid.<br>
75.	The method according to any of claims 63-74, wherein the complex is contacted with<br>
acid in a volume of about 5 µl or Jess.<br>
76.	The method according to any of claims 63-75, wherein the complex is heated in the<br>
presence of the acid.<br>
77.	A method for determining the effect of a plurality of culture conditions on a cell<br>
comprising the use of a complex according to any of claims 31-54.<br>
78.	A method for determining the effect of a plurality of culture conditions on a cell,<br>
comprising the steps of:<br><br>
(a)	providing a first set of groups of cell units each comprising one or more cells, and<br>
exposing said groups to desired culture conditions;<br>
(b)	pooling two or more of said groups to form at least one second pool;<br>
(c)	subdividing the second pool to create a further set of groups of cell units;<br>
(d)	exposing said further groups io-desired cultare conditions;<br>
(e)	optionally, repeating steps (b) -(d)-ileratively as-required; and<br>
(f)	assessing, the efiect on a given -cell unit of the culture conditions to which it has been<br>
exposed,<br>
wherein each cell unit composes-one or more cells adherent to  or bounded by a complex according to any of claims 31-54.<br>
79. A method for exposing a cell to a variety of cell culture conditions, comprising the steps oT:<br>
ay-providing a first set of groups, of cell units- each -comprising one or more cells, and exposing said groups to desired culture conditions;<br>
(b)	pooling two or more of said groups to form at least one second pool;<br>
(c)	subdividing the second pool to create a further set of groups of cell units;<br>
(d)	exposing said further groups to desired culture conditions; and<br>
(e) optionally, repeating steps (b) - (d) iteratively as required,<br>
wherein each cell unit comprises one or more cells adherent to or bounded by a complex according to any of claims 31-54.<br>
80.	A method for determining the effect of a plurality of culture conditions on a cell,<br>
comprising the steps of:<br>
(a)	providing a first set of groups of cell units each comprising one or more cells, and<br>
exposing said groups io desired culture conditions;<br>
(b)	pooling two or more of said groups to form at least one second pool;<br>
(c)	subdividing the second pool to create a further set of groups of cell"units;<br>
(d)	exposing said further groups to desired culture conditions;<br>
(e)	optionally, repeating steps (b) - (d) iteratively as required;  and<br>
(f)	assessing the effect on a given cell unit of the culture conditions to which it has been<br>
exposed,<br>
wherein each cell unit comprises one or more cells adherent to or bounded by a complex according to any of claims-31 =54.<br>
81.	The method according t0 any of claims 77-80, wherein the cells are cultured in cell<br>
units, each cell unit comprising one-or more cells.<br>
82.	The method according to claim 81, wherein the cell units are single cells.<br>
83.	The method according to any of claims 77-82, wherein the culture conditions are<br>
media to which the cell is exposed.<br>
84.	The method according to claim 83, wherein the media contain one or more specific<br>
agents which influence a cellular process.<br>
85.	The method according to any of claims 77-84, wherein the cell culture conditions<br>
comprise culturing at one or more specific temperatures.<br>
86.	The method according to any of clafms 77-85, wherein the cell culture conditions<br>
comprise culturing on one or more specific substrates.<br>
87.	A method for identifying a gene which influences a cellular process, comprising the<br>
steps of:<br>
a)-determining the effect of one or more culture conditions on a cell unit, in accordance with-any one of claims 77-86;<br>
b)	analysing gene expression in said cell units when exposed to said culture conditions;<br>
and<br>
c)	identffying genes which are differentially expressed under desired culture conditions.<br>
88.	A method according to. claim 87, wherein the desired culture conditions influence a<br>
cellular-process.<br>
89.	A method for producing a nucleic acid which encodes a gene product which<br>
influences a-cellular process, comprising identifying a gene in accordance with claim 82<br>
or claim 88, and producing at least the coding region of said gene by nucleic acid<br>
synthesis-or biological replication.<br>
90.	A method for inducing a cellular process, comprising the steps of:<br>
(a)	identifying one or more genes which are-differentially expressed in association with<br>
the cellular process in accordance with claim 87 or claim 88; and<br>
(b)	modulating the expression of said one or more genes in the cell.<br>
91.	The method according to claim 90, wherein modulation of gene expression in the<br>
cell comprises transfection of said one or more genes into the cell.<br>
92.	The method according to claim 90, wherein modulation of gene expression<br>
comprises the exogenous administration of a gene product.<br>
93.	A method-for identifying the state of a cellular process of a cell, comprising the steps<br>
of:<br><br>
(a)	identifying one or more genes which are differentially expressed in association with<br>
the cellular process in accordance with claim 87 or claim 88; and<br>
(b)	detecting the modulation of-expression of said one or more genes in a cell, -ther-eby<br>
determining the state-of the cellular process of said cell.<br><br>
94.	The method according to claim 93, wherein said one or more genes encode a<br>
marker.<br>
95.	The method according to claim 94, wherein said marker may be detected' by an<br>
immunoassay.<br>
96.	A method for inducing a cellular process, comprising-the steps of:<br><br>
(a)	determining ffie effect of. one or more culture conditions on a cell unit, in accordance<br>
with any one of claims 77 to 86;<br>
(b)	exposing a cell to culture conditions which induce the cellular process; and<br>
(c)	isolating the desired-cell.<br>
97: A method Tor identifying an agent which is capable of inducing ar cellular process, comprising the steps of:<br>
(a)	determining the effect of one or more agents on a cell unit, in accordance with any<br>
one of claims 77 to 86; and<br>
(b)	identifying those agent(s) which induce the cellular process in the cell units.<br>
98.	A method for preparing an agent which is capable of inducing a cellular process,<br>
comprising the steps oft<br>
(a)	determining the effect of one or more agents on a cell unit, in accordance with any<br>
one of claims 77 to 86;<br>
(b)	identifying those agent(s) which induce the desired cellular process in the cell units;<br>
and<br>
c) synthesising or isolating the agent(s).<br>
99.	The method according to any of claims 77 to 98, wherein the cellular process is cell<br>
proliferation or differentiation.<br>
100.	A method for culturing stem cells or cells that have been derived from stem cells In<br>
vitro comprising the steps of:<br>
a) incubating a stem cell culture; and<br>
b) splitting said culture into two-or more groups of stem cells, and culturing said group of stem cells under two ormore different sets of culture-conditions<br>
wherein the cells are cultured in cell units, each cell-unit comprising one or more cells  adhered to or bounded by the complex according to any of claims 31 -54-.<br>
101.	The-method according to claim 100, wherein .the cell units are single cells.<br>
102.	A method for culturing stem cells.-comprising-growing said stem-cells adhered to<br>
the complex accerding to any of'elaims 31-54.<br>
103.	The method according to claim 102, wherein said stem cells are subjected to af<br>
least one change of culture conditions.<br>
104.	The method according to claim 103, wherein said change of culture conditions<br>
comprises a change of medium.<br>
105.	A method for obtaining differentiated cells from stem cells in vitro, comprising the<br>
steps of:<br>
(a)	growing stem cells adherent to the complex according to any of claims 31-54 in a<br>
culture medium;<br>
(b)	transferring the complex from one culture medium to another;<br>
(c)	optionally repeating step (b) as required; and<br>
(d)	obtaining the differentiated cells attached to the complex.<br><br>
106.	The method according to claim 105, wherein the differentiated cells are isolated by<br>
enzymatic or chemical detachment from the complex.<br>
107.	The method according to elaim 105 to or claim 106-, wherein the differentiated cells<br>
are isolated by digestion of the complex.<br>
108.	A method of growing pluripotent-stem cells in vitro comprising the steps of:<br>
(a) seeding said cells on the complex according to any of claims 31-54;-and<br>
(b) propagating the cells while attached to the complex.<br>
109.	A method for cultunng cells .in vitro, comprising growing said cells adhered to the<br>
complex according to any of claims 31-54.<br>
110.	A method for identifying a tag obtained or obtainable from a cell unit comprising the<br>
steps of:<br>
(a) separating the cell unit and the fag;<br>
(b).obtaining.one or more-images of the tag:using a microscopic technique; and<br>
(c-)-analysing the images to determine one or more-features of the tag.<br>
111.	Use of a complex according to any of claims 31-54 for determining the effect of a<br>
plurality of culture conditions on a cell.<br>
112.	Use of a rod-shaped tag (eg. a nanowire) to label a microcarrier.<br>
113.	Use of a rod-shaped tag (eg. a nanowire) for determining the effect of a plurality of<br>
culture conditions on a cell.<br>
114.	Use according to claim 113 wherein said tag is adhered to or bound to a<br>
microcarrier.<br>
115.	A method, a complex, a computer program, an apparatus or a use substantially as<br>
described herein with reference to the accompanying figures.<br></e.g.></td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzE0LWRlbG5wLTIwMDgtQWJzdHJhY3QtKDIzLTEyLTIwMTMpLnBkZg==" target="_blank" style="word-wrap:break-word;">714-delnp-2008-Abstract-(23-12-2013).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzE0LWRlbG5wLTIwMDgtQWJzdHJhY3QtKDMwLTA2LTIwMTQpLnBkZg==" target="_blank" style="word-wrap:break-word;">714-delnp-2008-Abstract-(30-06-2014).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzE0LWRlbG5wLTIwMDgtYWJzdHJhY3QucGRm" target="_blank" style="word-wrap:break-word;">714-delnp-2008-abstract.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzE0LWRlbG5wLTIwMDgtQ2xhaW1zLSgyMy0xMi0yMDEzKS5wZGY=" target="_blank" style="word-wrap:break-word;">714-delnp-2008-Claims-(23-12-2013).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzE0LWRlbG5wLTIwMDgtQ2xhaW1zLSgzMC0wNi0yMDE0KS5wZGY=" target="_blank" style="word-wrap:break-word;">714-delnp-2008-Claims-(30-06-2014).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzE0LWRlbG5wLTIwMDgtY2xhaW1zLnBkZg==" target="_blank" style="word-wrap:break-word;">714-delnp-2008-claims.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzE0LWRlbG5wLTIwMDgtQ29ycmVzcG9uZGVuY2UgT3RoZXJzLSgyMC0xMS0yMDEzKS5wZGY=" target="_blank" style="word-wrap:break-word;">714-delnp-2008-Correspondence Others-(20-11-2013).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzE0LWRlbG5wLTIwMDgtQ29ycmVzcG9uZGVuY2UgT3RoZXJzLSgyMy0xMi0yMDEzKS5wZGY=" target="_blank" style="word-wrap:break-word;">714-delnp-2008-Correspondence Others-(23-12-2013).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzE0LWRlbG5wLTIwMDgtQ29ycmVzcG9uZGVuY2UgT3RoZXJzLSgzMC0wNi0yMDE0KS5wZGY=" target="_blank" style="word-wrap:break-word;">714-delnp-2008-Correspondence Others-(30-06-2014).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzE0LURFTE5QLTIwMDgtQ29ycmVzcG9uZGVuY2UtMDcxMTE0LnBkZg==" target="_blank" style="word-wrap:break-word;">714-DELNP-2008-Correspondence-071114.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzE0LWRlbG5wLTIwMDgtY29ycmVzcG9uZGVuY2Utb3RoZXJzLnBkZg==" target="_blank" style="word-wrap:break-word;">714-delnp-2008-correspondence-others.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzE0LWRlbG5wLTIwMDgtZGVzY3JpcHRpb24gKGNvbXBsZXRlKS5wZGY=" target="_blank" style="word-wrap:break-word;">714-delnp-2008-description (complete).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzE0LWRlbG5wLTIwMDgtRHJhd2luZ3MtKDIzLTEyLTIwMTMpLnBkZg==" target="_blank" style="word-wrap:break-word;">714-delnp-2008-Drawings-(23-12-2013).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzE0LWRlbG5wLTIwMDgtZHJhd2luZ3MucGRm" target="_blank" style="word-wrap:break-word;">714-delnp-2008-drawings.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzE0LWRlbG5wLTIwMDgtZm9ybS0xLnBkZg==" target="_blank" style="word-wrap:break-word;">714-delnp-2008-form-1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzE0LWRlbG5wLTIwMDgtRm9ybS0yLSgyMy0xMi0yMDEzKS5wZGY=" target="_blank" style="word-wrap:break-word;">714-delnp-2008-Form-2-(23-12-2013).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzE0LWRlbG5wLTIwMDgtRm9ybS0yLSgzMC0wNi0yMDE0KS5wZGY=" target="_blank" style="word-wrap:break-word;">714-delnp-2008-Form-2-(30-06-2014).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzE0LWRlbG5wLTIwMDgtZm9ybS0yLnBkZg==" target="_blank" style="word-wrap:break-word;">714-delnp-2008-form-2.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzE0LWRlbG5wLTIwMDgtRm9ybS0zLSgyMC0xMS0yMDEzKS5wZGY=" target="_blank" style="word-wrap:break-word;">714-delnp-2008-Form-3-(20-11-2013).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzE0LWRlbG5wLTIwMDgtRm9ybS0zLSgzMC0wNi0yMDE0KS5wZGY=" target="_blank" style="word-wrap:break-word;">714-delnp-2008-Form-3-(30-06-2014).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzE0LWRlbG5wLTIwMDgtZm9ybS0zLnBkZg==" target="_blank" style="word-wrap:break-word;">714-delnp-2008-form-3.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzE0LWRlbG5wLTIwMDgtZm9ybS01LnBkZg==" target="_blank" style="word-wrap:break-word;">714-delnp-2008-form-5.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzE0LWRlbG5wLTIwMDgtR1BBLSgyMy0xMi0yMDEzKS5wZGY=" target="_blank" style="word-wrap:break-word;">714-delnp-2008-GPA-(23-12-2013).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzE0LWRlbG5wLTIwMDgtcGN0LTEwMS5wZGY=" target="_blank" style="word-wrap:break-word;">714-delnp-2008-pct-101.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzE0LWRlbG5wLTIwMDgtcGN0LTIxMC5wZGY=" target="_blank" style="word-wrap:break-word;">714-delnp-2008-pct-210.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzE0LWRlbG5wLTIwMDgtcGN0LTIyMC5wZGY=" target="_blank" style="word-wrap:break-word;">714-delnp-2008-pct-220.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzE0LWRlbG5wLTIwMDgtcGN0LTIzNy5wZGY=" target="_blank" style="word-wrap:break-word;">714-delnp-2008-pct-237.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzE0LWRlbG5wLTIwMDgtUGV0aXRpb24tMTM3LSgyMC0xMS0yMDEzKS5wZGY=" target="_blank" style="word-wrap:break-word;">714-delnp-2008-Petition-137-(20-11-2013).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=QW1lbmRlZCBGb3JtIDEsIEZvcm0gMiBhbmQgRm9ybSA1LnBkZg==" target="_blank" style="word-wrap:break-word;">Amended Form 1, Form 2 and Form 5.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Q292ZXJpbmcgbGV0dGVyLnBkZg==" target="_blank" style="word-wrap:break-word;">Covering letter.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Rm9ybSAxMy5wZGY=" target="_blank" style="word-wrap:break-word;">Form 13.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=UGV0aXRpb24gdW5kZXIgcnVsZSAxMzcucGRm" target="_blank" style="word-wrap:break-word;">Petition under rule 137.pdf</a></p>
		<br>
		<div class="pull-left">
			<a href="264245-a-process-for-production-of-alkyl-esters.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="264247-a-pharmaceutical-composition-for-altering-the-affinity-of-a-nucleic-acid-ligand-for-a-coagulation-factor-in-a-mammalian-host.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>264246</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>714/DELNP/2008</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>51/2014</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>19-Dec-2014</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>17-Dec-2014</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>25-Jan-2008</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>PLASTICELL LIMITED</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td>STEVENAGE BIOSCIENCE CATALYST, GUNNELS WOOD ROAD, STEVENAGE, SG1 2FX, GREAT BRITAIN</td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>JOHN GIRDLESTONE</td>
											<td>5 MAPLE ROAD, BOREHAMWOOD WD6 1BN, GREATE BRITAIN.</td>
										</tr>
										<tr>
											<td>2</td>
											<td>YEN CHOO</td>
											<td>20 NORFOLK MANSIONS, PRINCE OF WALES DRIVE, LONDON SW11 4HL, GREAT BRITAIN.</td>
										</tr>
										<tr>
											<td>3</td>
											<td>FRASER HORNBY</td>
											<td>67 TOPAZ DRIVE, ANDOVER, HAMPSHIRE SP10 3EE, GREAT BRITAIN.</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>C12N 5/00</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>PCT/GB2006/003186</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td>2006-08-25</td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td>0517382.8</td>
									<td>2005-08-26</td>
								    <td>U.K.</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/264246-complexes-and-methods-for-determining-cell-culture-history by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 00:26:14 GMT -->
</html>
