<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/264276-a-nucleic-acid-construct by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 00:35:09 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 264276:&quot;A NUCLEIC ACID CONSTRUCT&quot;</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">&quot;A NUCLEIC ACID CONSTRUCT&quot;</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>A nucleic acid construct comprising a first nucleic acid molecule having the nucleotide sequence ofSEQIDNO. 10, wherein said first nucleic acid molecule is a promoter and is operably linked to a second nucleic acid molecule encoding a protein of interest and a 3&quot;non-translated region.</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td>Transcription factor gene induced by water deficit<br>
conditions and abscisic acid from Helianthus  anmzus,<br>
promoter and transgenic plants<br>
BACKGROUND OF THE INVENTION<br>
1. Field of the Invention.<br>
The present invention relates to a new gene that encodes a transcription factor induced by water deficit conditions and abscisic acid from Helianthus annuus, having a homeodomain associated to a leucine zipper. The transcription factor is useful to be cloned in DNA constructions for transforming host cells and plants. The transgenic plants comprising the transcription factor are resistant to harmful environment conditions such as the water deficit stress and high salinity. A nucleic acid promoting sequence is also provided wherein the sequence is inducible by water deficit or abscisic acid, constructions, host cells and transgenic plants that comprise the sequence.<br>
2. Description of the Prior Art.<br>
The homeodomain is a 60 amino acid motif present in a number of eukaryotic transcription factors involved in<br>
developmental processes (Gehring, Science 236, 1245-1252, 1987). Genes containing homeoboxes have been isolated from many eukaryotic organisms  including fungi,  mammals and plants (Gehring, W. J., et al., Annu. Rev. Biochem. 63, 487-526,  1994).  Plant  homeoboxes  can be  divided into several families according to sequence conservation and structure in and outside the homeodomain (Chan, R.L., et al. Biochim. Biophys. Acta 1442 (1), 1-19, 1998). Members of one of these families have a distinct feature: they code for  proteins  termed  Hd-Zip,  because  they  contain  an homeodomain associated with a leucine zipper, a coiled-coil structure involved in dimerization. Hd-Zip proteins bind DNA efficiently only as dimers (Sessa, G., et al., EMBO J. 12, 3507-3517, 1993; Palena C.M., et al., Biochem J. 341, 81-87, 1999). It has been suggested that these proteins may be   involved  in  regulating  developmental  processes associated with the response of plants to environmental conditions (Chan, R.L., et al. Biochim. Biophys. Acta 1442 (1), 1-19, 1998, Carabelli, M., et al., Plant J. 4, 469-479, 1993; Schena, M., et al., Genes Devel. 7, 367-379, 1993). One of the most common environmental stresses to which plants are exposed is dehydration. Although many seeds tolerate extreme dehydration, tolerance is rare in vegetative parts of the plant. Plants respond to water stress with the expression of a specific set of genes, which allows them to adapt to the altered environmental<br>
conditions (Bray, E.A. Trends Plant Sci. 2, 48-54, 1997 y Shinozaki, K. and Yamaguchi-Shinozaki, K. Plant Physiol. 115, 327-334, 1997). The hormone abscisic acid (ABA) plays an important role in a sub-set of these responses (Shinozaki, K. and Yamaguchi-Shinozaki, K. Plant Physiol. 115, 327-334, 1997 y Leung, J. and Giraudat, J. Ann. Rev. Plant Physiol. Plant Mol. Biol. 49, 199-222, 1998).<br>
The characterization of the promoting regions of the genes involved in the tolerance or resistance to water stress shown the existence of an element that responds to ABA, namely ABRE, and an element responding to desiccation, namely DRE.<br>
Soderman et al. have disclosed [described] the [two]<br>
genes, ATHB-7 and -6 of Arabidopsis that is [are] induced by abscisic acid and water deficit (Soderman E. et al., The Plant Journal 10: 375-381, 1996 and Soderman E. et al. Plant Molecular Biology 40: 1073-1083, 1999). The authors have not shown that the over-expression of these genes provide tolerance to water deficit.<br>
US No. 5.981.729 discloses a new gene that is induced by water deficit and abscisic acid and that encodes a transcription factor of A. Thaliana. This patent does not discloses any reference to transgenic plants carrying the gene of the present invention and resisting to water stress conditions.<br>
SUMMARY OF THE INVENTION<br>
It is therefore an object of the invention to provide an isolated nucleic acid molecule encoding the transcription factor Hahb-4, a functionally active fragment or variant thereof, having the nucleic acid sequence of SEQ ID N° 1 or a fragment thereof, wherein the nucleic acid molecule is derived from Helianthus annuus, and it may be an mRNA or the cDNA of SEQ ID N° 2, wherein the molecule is capable of binding to a 5'-CAAT(A/T)ATTG-3' DNA sequence or to a dehydration transcription .regulating region of plant species.<br>
It is still another object of the present invention to provide a vector, comprising a promoter operably linked to the nucleic acid sequence selected from the group comprising of SEQ ID N° 1, SEQ ID N0 2 and fragments thereof, wherein the vector drives the expression of the transcription  factor Hahb-4     or  a  functionally  active fragment or variant thereof, and wherein said transcription factor Hahb-4 or a functionally active fragment or variant thereof  is  capable  of  binding  to  a  dehydration transcription regulating region of plant  species,  and wherein the expression of the vector in a host cell increases  the  tolerance  of the  cell  to  environmental stress, such as water stress, as compared to a wild type variety of such host cell.<br>
It is a further object of the present invention to provide  a  transgenic plant  stably transformed with  a nucleic acid molecule having a sequence selected from the group comprising SEQ ID N° 1, SEQ ID N° 2 or fragments thereof,  wherein the nucleic acid molecule encodes the transcription  factor  Hahb-4  or  a  functionally  active fragment or variant thereof,  and wherein the plant is provided with an increased tolerance to environmental stress,  such as drought, salinity, osmotic, and others, preferably water stress, as compared to a wild type variety of such plant, and wherein the plant that may be a monocot, a dicot or any other agronomic one,  is water stress tolerant by binding the transcription factor Hahb-4 or a functionally active  fragment  or variant  thereof to  a dehydration transcription regulating region of plant.<br>
It is a further object of the present invention to provide a plant seed stably transformed with a nucleic acid molecule having a sequence selected from the group comprising SEQ ID N° 1, SEQ ID N° 2 and fragments thereof, wherein the nucleic acid molecule sequence encodes the transcription factor Hahb-4 or a functionally active fragment or variant thereof.<br>
It is a further object of the present invention to provide a host cell that has been stably transformed with a nucleic acid molecule having a sequence selected from the group comprising SEQ ID N° 1, SEQ ID N° 2 and fragments<br>
thereof, wherein the nucleic acid molecule encoding the transcription factor Hahb-4 or a functionally active fragment or variant thereof, wherein the host cell is selected from the group comprising bacterial, fungal, insect, plant and animal cell and preferably it is a plant cell.<br>
It is a further object of the present invention to provide a method of producing a water stress tolerant transgenic plant, the method comprising stably transforming a plant cell or cell culture with the nucleic acid sequence selected from the group comprising SEQ ID N° 1, SEQ ID N° 2 and fragments thereof, and regenerating the cells or cell cultures into plants.<br>
It is another object of the present invention to provide an isolated nucleic acid molecule selected from the group comprising:<br>
(a)	a nucleic acid molecule having the nucleotide sequence SEQ ID N° 3;<br>
(b)	a nucleic acid molecule having the nucleotide sequence SEQ ID N° 10;<br>
(c)	a nucleic acid molecule having the nucleotide sequence of nucleotides 805 to 1221 of SEQ ID N° 3;<br>
(d)	a nucleic acid molecule having the nucleotide sequence of nucleotides 904 to 1221 of SEQ ID N° 3;<br>
(e)	a nucleic acid molecule having the nucleotide sequence of nucleotides 1011 to 1221 of SEQ ID N° 3;<br>
(f)	a nucleic acid molecule having the nucleotide sequence of nucleotides 15 to 622 of SEQ ID N° 3;<br>
(g)	a nucleic acid molecule having the nucleotide sequence of nucleotides 15 to 409 of SEQ ID N° 10;<br>
(h) a nucleic acid molecule having a nucleotide sequence complementary to the nucleic acid molecule of (a), (b), (c), (d), (e), (f) or (g); and<br>
(i) a nucleic acid molecule having a length of at least 150 nucleotides and having at least 80% sequence identity to the nucleic acid molecule of (a) , (b) , (c) , (d), (e), (f), (g) or (h), wherein said. nucleic acid molecules are capable of promoting expression of a heterologous nucleic acid molecule in a transformed cell or tissue selected from the group comprising bacteria, fungal, insect, plant [or] animal cell, embryogenic tissue, plant callus and plant seed.<br>
It is a further object of the present invention to provide a nucleic acid construct comprising a first nucleic acid molecule selected from the group comprising:<br>
(a)	a nucleic acid molecule having the nucleotide sequence SEQ ID N° 3;<br>
(b)	a nucleic acid molecule having the nucleotide sequence SEQ ID N° 10;<br>
(c)	a nucleic acid molecule having the nucleotide sequence of nucleotides 805 to 1221 of SEQ ID N° 3;<br><br>
(d)	a nucleic acid molecule having the nucleotide sequence of nucleotides 904 to 1221 of SEQ ID N° 3;<br>
(e)	a nucleic acid molecule having the nucleotide sequence of nucleotides 1011 to 1221 of SEQ ID N° 3;<br>
(f)	a nucleic acid molecule having the nucleotide sequence of nucleotides 15 to 622 of SEQ ID N° 3;<br>
(g)	a nucleic acid molecule having the nucleotide sequence of nucleotides 15 to 409 of SEQ ID N° 10,<br>
(h) a nucleic acid molecule having a nucleotide sequence complementary to the nucleic acid molecule of (a), (b), (c), (d), (e), (f) or (g); and<br>
(i) a nucleic acid molecule with at least 80% of homology or at least 150 nucleotides in length to the nucleic acid molecule of (a), (b) , (c) , (d) , (e), (f), (g) or (h), wherein said first nucleic acid molecule is operably linked to a second nucleic acid molecules encoding a protein of interest and a 3' non-translated region. Preferably the nucleic acid molecule is the promoter having the SEQ ID N°3 or SEQ ID No. 10. Also host cells and transgenic plants stably transformed with at least one of the mentioned constructs are provided.<br>
It is a further object of the present invention to provide a method for expressing at least one protein of interest in a host cell, the method comprises the introduction of one of the above mentioned constructs into a host cell and allowing the host cell to produce a protein<br>
of interest, wherein the host cell is selected from the group comprising bacterial, fungal, insect, plant and animal cell.<br>
It is a further object of the present invention to provide a method for obtaining a transgenic plant expressing at least one protein of interest, the method comprises the stable transformation of a plant cell or cell culture with one of the above mentioned nucleic acid constructs, and the regeneration of the cells or cell cultures into a whole plant that expresses at least one protein, wherein the transgenic plant is selected from the group comprising a monocot and dicot plant.<br>
It is a further object of the present invention tq provide a transgenic plant stably transformed with at least one of the above mentioned constructs, wherein the protein of interest is the transcription factor Hahb-4, having the nucleic acid sequence selected from the group comprising SEQ ID N° 1, SEQ ID N° 2 and fragments thereof, and wherein the plant is selected from the group comprising monocot and dicot plants and said plant is environmental stress tolerant to situations like drought, high salinity, high osmotic pressure and others, and preferably the plant is resistant and tolerant to water deficit. Most preferably, the plant is water stress tolerant by binding the transcription factor Hahb-4 or a functionally active fragment or variant thereof to a dehydration transcription<br>
regulating region of the plant and the dehydration transcription regulating region of the plant is a 5'-CAAT(A/T)ATTG-3' DNA sequence.<br>
The above and other objects, features and advantages of this invention will be better understood when taken in connection with the accompanying drawings and description.<br>
BRIEF DESCRIPTION OF THE DRAWINGS<br>
The present invention is illustrated by way of example in the following drawings wherein:<br>
Figure 1 shows the genomic sequence encoding sunflower Hahb-4 of the invention. The deduced protein sequence of the open reading frame is indicated below the nucleotide sequence. The homeodomain is shown in bold; leucines from the leucine zipper are shown in bold and underlined. The lower part of the Figure shows an alignment of the Hd-Zip domain of Hahb-4 with those of Athb-1, -6, 7 and 12. Shaded boxes indicate identical amino acids.<br>
Figure 2a shows a scheme of the Hahb-4  cDNA showing the length and polarity of the riboprobe (arrow) , and the region protected by the Hahb-4   mRNA (+81 to +429; shaded box) .<br>
Figure 2b shows the expression of Hahb-4 induced by water stress subjected to different treatments. Four-day-old seedlings were treated as follows: 100 µM ABA during 24<br>
h; water stress during 2 h; 4°C during 24 h; 42°C during 2 h; 0.5 M mannitol during 4 h.<br>
Figure 2c shows the expression of Hahb-4 induced by water stress in different organs during 2 h, embryos, and dry seeds were analyzed.<br>
Figure 3 shows time dependence of Hahb-4 induction by water stress in roots, stems and leaves. Seedlings were subjected to water stress during 0.5 or 1 h, or re-hydrated at different times after a 1 h drought treatment. The same filter was hybridized with an rRNA probe as a control for RNA loading and transfer (lower panel).<br>
Figure 4 shows time dependence of the response of Hahb-4 to ABA. Total RNA (20 µg) was isolated from seedlings that were either untreated, or treated with 100 µM ABA for different times as indicated in each lane. In the lower panel, the same filter was hybridized with an rRNA probe as a control for RNA loading and transfer.<br>
Figure 5 shows time dependence of Hahb-4 induction by water stress in roots, stems and leaves. RNA was prepared from different organs of control, 30 min (1), 60 min- (2), or 90 min-water-stressed plants (3). The same filters were hybridized with an rRNA probe as a control for RNA loading and transfer (right panels).<br>
Figure 6 show that ABA and water stress induce the expression of nuclear proteins that specifically bind to the Hahb-4 DNA target sequence. Lane 0, DNA alone; Lane 1,<br>
control plants; lane 2, ABA-treated plants; lane 3, water-stressed plants; A and B indicate two different shifted bands observed with nuclear extracts.<br>
Figure 7 shows a graph illustrating the strategy employed for cloning the cDNA encoding Hahb-4 under the control of CaMV35S promoter. Plasmid p35SHB4 was constructed in E. coli and then it was used for transforming strain CV2260 of A. tumefaciens. Strain A. tumefaciens having the plasmid containing Hahb-4 cDNA under the control of CaMV35S promoter was named ATH4. BD y BI: right and left edges; Pnos: nopaline synthetase gene promoter; Tnos: termination sequence of nopaline synthetase; Kmr: kanamycin resistantce gene; P35S: 35ScaMV promoter, gus:   ß-glucuronidase gene.<br>
Figure 8 is a graph showing the germination time of the inventive transformed plant (dark bars) and the untransformed control plants (white bars) in percentage of germinated plants after a 48 treatment at 4°C (n=22).<br>
Figure 9 is a graph showing the length of the stem measured in mm of the inventive transformed plants (dark bars) and the non transformed control plants (white bars) from the 28 days to 33 days since germination. The two end bars show the stem length of transgenic plants subject to water stress tests wherein there were no survivors in the group of control plants.<br><br>
Figure 10 is a bar graph showing the amount of siliques formed during the development of the plants, non transformed control (white bars) and transgenic plants (dark bars). The two end bars show the quantity of siliques of transgenic plants subject to water stress tests wherein there were no survivors in the control plants group. This plants were irrigated at the end of the cycle and once recuperated the parameters of interest were recorded.<br>
Figure 11 is a bar graph showing the percentage of non transformed control (white bars) and transgenic plants (dark bars) germinated at different times after breaking the dormancy in the presence of 50 mM mannitol.<br>
Figure 12 is a bar graph showing the percentage of non transformed control (white bars) and transgenic plants (dark bars) germinated at different times after breaking the dormancy in the presence of 200 mM mannitol.<br>
Figure 13 is a bar graph showing the percentage of non transformed control (white bars) and transgenic plants (dark bars) germinated at different times after breaking the dormancy in the presence of 300 mM mannitol.<br>
Figure 14 is a bar graph showing the percentage of non transformed control (white bars) and transgenic plants (dark bars) germinated at different times after breaking the dormancy in the presence of 50 mM NaCl.<br>
Figure 15 is a bar graph showing the percentage of non transformed control (white bars) and transgenic plants<br>
 (dark bars) germinated at different times after breaking the dormancy in presence of 150 mM NaCl.<br>
Figure 16 is a bar graph showing the percentage of non transformed control surviving plants (white bars) and transgenic plants (dark bars) in three independent tests. In the first test (1) the stress occurred when the plants were adult (reproductive stage). In the second test (2) the plants were in advanced vegetative stage (complete rosette). In the third test (3) the stress occurred at the germination.<br>
Figure 17 shows the phenotypes of control and transgenic plants, (a) the phenotype of transgenic plants (1st and 2nd rows) and of non transformed control plants (3rd y 4th rows) is observable. These plants were subjected to water stress when adult and re-irrigated at the end of the life cycle. In b the phenotype of transgenic (left side) and non transformed control plants (right side) is observable. This group of plants was subjected to water stress when they were in vegetative stage. In c the phenotype of transgenic (left side) and non transformed control plants (right side) subjected to water stress since the germination and then re-irrigated is observable, (d) In d is shown the phenotype of transgenic plants subjected to extreme drought at the vegetative stage.<br>
Figure 18 shows the sequence of nucleotides of the promoter region of Hahb-4   gene, remarking the sequences<br>
corresponding to the TATA box, the element responding to water stress/low temperatures, ABRE regions and the sequences indicating the recognizing sites of Myb and Myc.<br>
Figure 19 shows an scheme of pBI 101.3 vector used for cloning several segments of the Hahb-4 promoter, ß-glucuronidase gene is indicated, as well as the genes providing resistance to kanamycin in E. coli and A. tumefaciens. The replication origins of E. coli and A. tumefaciens  are also marked.<br>
Figure 20 shows a photograph comparing the expression levels measured by hystochemistry of gene gus in 10-day old seedlings carrying the constructs of promoter of 0 to -400 sequence (nucleotides 805 to 1221 of SEQ ID N° 3) (left side) and 0 to -1015 sequence (small allele) (right side) .<br>
Figure 21 shows the expression of gus in the roots<br>
of A. thaliana. (a) shows the expression of gene gus in 20-day old roots of plants transformed with construct of 0 to -400. (b) shows the expression of gene gus in 20-day old roots of plants transformed with construct of 0 to -1015. (c) shows the expression of gene gus in 10-day old roots of plants transformed with construct of 0 to -1015.<br>
Figure 22 shows the effect of ABA treatment in 20-day old seedlings transformed with the construct 0 to -400. At the left side the control plant is shown and at the right side the seedling treated with ABA is shown.<br>
Figure 23 shows a scheme of Hahb-4 gene structure. At the top: large alleloe, at the bottom: small alleloe. The oligonucleotides employed for the isolation of the promoting region and for the construction of recombinant plasmids and used in plant transformation are indicated.<br>
DESCRIPTION OF THE PREFERRED EMBODIMENTS<br>
Now referring in detail to the invention, the same refers to the characterization of a new gene Hd-Zip, the sunflower Hahb-4 gene responsive to water stress conditions. Within the homeodomain, Hahb-4 shows a partial homology with Athb-7 and Athb-12, two related proteins from A. thaliana. Protein Hahb-4 has the homeodomain located closely to the N-terminal.<br>
The invention discloses a clone from cDNA library. This clone represents a member of the Hd-Zip family that was named Hahb-4. Sequences corresponding to the 5' and 3' ends were obtained by PCR. The complete cDNA sequence obtained in this way is 674 bp long (SEQ ID N° 2) and contains an open reading frame of 177 amino acids (Fig. 1).<br>
Comparison of the encoded protein with other Hd-Zip protein sequences indicates that it can be included in the subfamily I of Hd-Zip proteins, sharing about 50% identical, amino acids within the homeodomain with other members of this sub-family, with the exception of Athb-7 and Athb-12, that have 60 and 53% identity, respectively, within this<br>
region (Fig. 1) . It is also noteworthy that the Hahb-4 homeodomain is located almost at the amino terminal end of the protein (amino acids 17-7 6) . As a consequence, Hahb-4 lacks the acidic domain present in other members of the Hd-Zip protein family adjacent to the N-terminal end of the homeodomain. This characteristic is also shared with Athb-7 and Athb-12. In order to investigate the genomic structure of the Hahb-4 gene, we have amplified genomic DNA (genomic DNA is SEQ ID N° 1) with several oligonucleotides comprising the entire cDNA. A single intron of 101 bp was detected between nucleotides 381 and 382 (aminoacids 108 and 109) of the cDNA.<br>
Northern blot analysis indicated that the Hahb-4 gene of the invention is expressed at very low levels in sunflower plants grown under controlled and normal environmental conditions. Only faint signals were obtained with total RNA extracted from several tissues and developmental stages.<br>
We have analyzed the expression of Hahb-4 under different environmental conditions (i.e. water, osmotic, salt, cold, heat, and oxidative stress) by RNAse protection, which is more sensitive than the Northern technique (Fig. 2) . Figure 2b shows that Hahb-4 was not detected in 4-day-old seedlings grown under normal conditions. In water-stressed seedlings, however, a strong signal  was  observed.  Mannitol  also  induced Hahb-4<br><br>
expression, although at lower levels, probably reflecting the decrease in water activity caused by this compound (Fig. 2b). A similar result was obtained with NaCl.<br>
As many responses to water stress are mediated by ABA, we have also analyzed the effect of this hormone on the expression. As shown in Fig. 2b, induction was observed after 24 h of treatment of irrigated seedlings with 100 µM ABA. A smaller, but significant increase in transcript levels was also observed with 10 uM ABA. No detectable effect was observed when the seedlings were subjected to either cold (4°C) or heat (42°C) stress conditions (Fig. 2b). These results indicate that the effect of water stress is specific.<br>
Figure 2c shows that the response to water stress conditions is also observed in roots, stems, and leaves of older (21-day-old) plants. The level of induction in aerial parts of the plant is similar to that observed in seedlings. Roots, on the contrary, show considerably lower transcript levels under water stress conditions.<br>
Since ABA also participates in seed development during late embryogenesis, which involves a desiccation process, we analyzed Hahb-4 transcript levels in sunflower embryos (20 d after pollination) and in dry seeds. No signal was obtained in RNAse protection experiments (Fig. 2c), indicating that the response of Hahb-4  to water stress<br>
and  ABA  is  characteristic  of  vegetative  phases  of development.<br>
The high level of induction observed under water stress conditions allowed us to analyze the time-dependent increase in transcript levels by Northern experiments. Figure 3 shows that seedlings exposed to drought significantly increased their Hahb-4 transcript levels after only 30 min; the response reached a maximum after 1 h of withholding water. After this time, no further increments in Hahb-4 transcript levels were observed. Rehydration of the seedlings lowered Hahb-4 transcript levels more slowly, with only about a 50% decrease after 2 h (Fig. 3, lanes 4 and 5) . The effect of ABA treatment was also time dependent. As shown in Fig. 4, the response to ABA was detectable within 1 h and reached a maximum after 3-6 h. After that, transcript levels decreased slowly, but were still significantly high after 24 h of treatment. The high Hahb-4 transcript levels of the invention are reverted after the re-hydration of the sunflower plants, thus indicating again that said gene participates in the response to the water stress.<br>
The response of Hahb-4 to water stress in different organs of hydroponically grown 3-week-old plants was also rapid. In roots, transcript levels reached a maximum after 60 min. of stress treatment (Fig. 5).<br>
In leaves and stems, a maximum response was observed after more prolonged treatments. This may be related to the fact that the initial perception of water stress is accomplished by the root, which synthesizes ABA that is then translocated to aerial parts of the plant. This, together with the results previously shown in Figs 2-4, most likely indicates that Hahb-4 expression is related to endogenous ABA levels in water-stressed plants.<br>
The presence of functional DNA binding proteins in sunflower nuclei was analyzed by electrophoretic mobility shift   assays  using   a   synthetic   double-stranded oligonucleotide comprising the sequence 5'-CAAT(A/T)ATTG-3', bound in  vitro  by Hahb-4 expressed in bacteria. Nuclear extracts prepared from 4-day-old seedlings showed distinct shifted bands, indicating the existence of at least two different protein complexes that bind to this sequence (Fig. 6) . The amount of both complexes was significantly increased in extracts obtained from ABA-treated plants. On the contrary, only the slower-migrating complex increased when plants were subjected to water stress. Formation of both complexes was almost completely abolished by an excess of the same unlabelled DNA, but not by an equivalent amount of a similar DNA containing the sequence 5'-CACT(A/T)AGTG-3' (changed positions underlined) not bound by Hahb-4 (Fig. 6).  This  result  strongly  suggests  that  at  least  one functional protein with the same DNA-binding specificity as<br>
Hahb-4 is synthesized and translocated to the nucleus in the presence of ABA or under water stress conditions.<br>
The intense Hahb-4 expression was detected only in plants subjected to a water stress or to a treatment with ABA. Heat, cold and oxidative stress do not induce such expression. The salts and the osmotic treatments only produce small increments in Hahb-4 transcription levels. The water stress effect was entirely reversed when the plants were re-hydrated. These characteristics suggest that the inventive Hahb-4 expression directly responds to the hydric status of the vegetal cells and that the induction is not an effect from damage or general stress.<br>
For obtaining transgenic plants of Arabidopsis thaliana that over-express sunflower Hahb-4 of the invention, the vector pBI121 and the "floral dip" method were used, as disclosed in Example 3 and Figure 7. As a result, many independent lines of transgenic plants have been obtained wherein the plants were tolerant or resistant to growth in a medium with kanamycin as PCR positive reaction with specific oligonucleotides. From the obtained independent lines, those having a transgene stable expression (the Hahb-4 gene of the present invention) were selected. Subsequently, the homozigotes lines F3 were selected to analyize the phenotype.<br>
As  it  is shown in Figure  8,  the A.     thaliana transgenic plants carrying the inventive gene germinates<br>
more rapidly as compared to the non transformed control plants. After 14 hours, 85% of the transformed plants according to the invention germinated vs. the 58% of non transformed ones.<br>
In addition, the stems of the inventive transformed plants enlarges more slowly and reach a maximum height that is equivalent to the 85% of the height of a control plant that was grown in the same conditions (with normal water availability - Figure 9) . In addition, the transgenic plants subjected to water stress reached a stem height similar to the one of the same transgenic plants grown with normal water availability, thus indicating that the water stress does not affect the stem growth in the transgenic plants of the invention (Figure 9) . The inventive transgenic plants are not only tolerant to the water stress but also they normally growth in lack-of-water conditions, without important alterations in the observed phenotype.<br>
The number of formed siliques did not significantly vary between wild type and transgenic plants and this number is slightly higher in the transgenic plants in spite of the shortening of the floral pedicel (Figure 10) . When the transgenic plants grown under water stress conditions are compared with the same plants grown in normal conditions, the number of formed siliques is higher in those subjected to stress (Figure 10) . In addition, as it is shown in Table 1, the total weight of the seeds produced<br>
by plants transformed according to the invention was about 15% higher compared with that of the seeds from control non transformed plants.<br>
TABLE 1<br><br>
(Table Removed)<br>
Since the Hahb-4 product acts as a transcription. factor and its expression at a the transcription level seems to be regulated by the availability or presence/absence of water, studies have been performed for to determine the tolerance to water stress in transformed plants over-expressing the sunflower Hahb-4 gene of the invention.<br>
First, the germination process, under conditions imitating the lack of water, such as presence of mannitol, and other conditions generating salt stress, was analyzed. Figures 11, 12 and 13 show the germination times of plants transformed according to the invention as compared to control ones. The presence of mannitol delays the germination. This effect is more remarked at higher concentrations   of   the   carbohydrate.   However,   the<br>
transformed plants keep a good germination efficiency even at high concentrations of mannitol.<br>
When the germination tests were carried out in the presence of different concentrations of NaCl the obtained results were similar as in the case of mannitol. As can be seen in Figures 14 and 15, the over-expression Hahb-4 provides the transformed plants with a higher germination capacity in a saline medium.<br>
It is to be remarked that the roots of the transgenic plants grown in the different stress conditions tested or assayed are larger than the roots of the control plants, thus indicating a phenotype that provides tolerance to several stress conditions.<br>
Subsequently, drought tests have been performed at several plant developmental stages cultured in soil. Figure 16 shows three tests that show survival to water stress. Clearly, a higher tolerance to water stress in the inventive transformed plants is observed when compared to control non transformed plants.<br>
When the plants were subjected to water stress at different developmental stages, the transformed ones shown a higher tolerance to stress conditions. Figure 17 shows the plant status after they have been subjected to lack-of-water conditions at different stages of growth. Whatever the stage is wherein the plant is subjected to water stress, the transgenic ones carrying the inventive Hahb-4<br>
gene are more tolerant and resistant to the above mentioned conditions.<br>
Briefly, the present invention discloses the obtaining of Arabidopsis thaliana transformed plants that over-express the sunflower gene Hahb-4 of the invention under the control of 35S cauliflower mosaic virus promoter, the construction shown in Figure'7. The inventive gene, initially isolated from sunflower, encodes a Hd-Zip protein having a proteic domain of the type homeodomain associated to a leucine-zipper. The experts in. the art know, that any promoter or construction of nucleic acid driving the expression of the inventive gene may be employed without altering the spirit and scope of the invention. Even more, nucleic acid constructions permitting the expression of the inventive gene in any host cell, such as bacteria, yeast, fungus, animal and vegetal cells, may be prepared. In a preferred embodiment of the invention, the gene expresses in plants cells and tissues.<br>
The transformed plants expressing Hahb-4 have, in normal growing conditions, in average, shorter stems. It seems that this characteristic is mainly due to the inhibition of cellular expansion and not to the inhibition of cellular division, as indicate hystological studies.<br>
In addition to the foregoing, the adult leaves of the transgenic plants are more rounded and less elongated<br>
as compared to those of non transformed plants. Both characteristics altogether seem to indicate that the gene product is acting as an inhibitor of the cellular expansion and elongation. These phenotypic . characteristics (short stem and round leaf) would be directly related to the capacity of the transgenic plants to tolerate and resist the scarcity or lack of water, as a mechanism for "saving water". On the contrary, the roots of the inventive transgenic plants are longer as compared to the roots of non transformed plants. This would indicate the presence of another advantageous mechanism for obtaining water.<br>
The transgenic plants of the present invention are markedly tolerant to water stress in different growing stages, either in the germination as well as in the early and end vegetative periods, and also in the reproductive period. The survival percentage of transgenic plants is higher than the one observed for the non transformed control plants when both are subjected to drought conditions.<br>
With all these results we may conclude that the Hahb-4 gene is involved in the plant response to water stress and that its specific function would be to generate phenotypic changes enhancing the tolerance of the plant to the lack of water.<br>
Remarkably, none of these phenotypic changes negatively affects the production and germination of the<br><br>
plants. On the contrary, the seeds production (measured in weight) of the transgenic plants is higher than the production of non transformed plants.<br>
The Hahb-4 of the invention may be employed for producing transgenic plants having a commercial interest wherein said transgenic plants have characteristic tolerances to water stress. It may be expected that the productivity of said agronomic valued plants subjected to water stress is similar to the productivity of the variety that is non transformed and not subjected to such stress conditions. As an example, such agronomic valued plants may include, but are not restricted to, sunflower, wheat, barley, soybean, potato, corn, sugar cane or rice.<br>
It is important to remark that there is not a large variety of genes that over-expresses when the plant is subjected to water stress and there is no concrete evidence that said genes could provide a plant with tolerance to drought like the inventive Hahb-4  does.<br>
The inventors have isolated and characterized the sequence of the promoting region of sunflower Hahb-4 according to the invention. The isolation of Hahb-4 promoting region was carried out in three stages by inverse PCR technique, using the information obtained in a one stage for designing the new pair of oligonucleotides to be used in the next. The obtained fragments from the PCR reaction were cloned into pGEM. -T easy   vector (Promega) ,<br>
and the sequence of the promoting region was manually completed by overlapping the repeated regions of each construction.<br>
The sequence of Hahb-4 (SEQ ID N° 3) promoter corresponds to a 1221 pb sequence, comprising a TATA box located 24 pb upstream of the transcription initiation site. The comparison of this sequence with the ones existing in the data bases, reveals the existence of regions with homology to sequences thought to be involved in the response to several environmental factors, such as light, abscisic-acid and hormones. As an example, Figure 18 shows putative elements responding to ABA of the type ABRE as well as an element that responds to the drought stress or low temperatures of the type DRE.<br>
The consensus sequences of the transcription factor joint involved in the response to transcription factors, such as the proteins of the families Myb y Myc (Abe et al., Plant cell 9: 1859-1868, 1997; Shinozaki and Yamaguchi-Shinozaki, Plant Physol. 115: 327-334, 1997, herein incorporated as references) were also identified.<br>
After the isolation and sequencing of Hahb-4 promoter the sequences involved in the driving of the gene expression were sought for. Particularly, providing organ specificity and the sequences responding to water stress and the presence of ABA. For this purpose plants of A. thaliana  with constructions comprising a reporter gene (gus<br>
gene)  were transformed under the control of the total sequence of said promoter or parts of said sequence.<br>
The complete sequence of the promoter was cloned in one construct. For this purpose two specific oligonucleotides that hybridized in the ends of the promoting region were designed, and they were employed in a PCR reaction by using sunflower genomic DNA as template. As a product of the amplification reactions two bands of about 1000 and 1200 bp were obtained. Considering that the vegetal material used for these tests was isolated from a hybrid (contiflor 15), the presence of two bands seems to indicate the existence of two different alleles present in the genome.<br>
Both PCR products were cloned into pGEM-Teasy vector (Promega). Then they were restricted with BamRI and HindIII and cloned into pBI101.3 vector (Figure 19) as it is indicated in Example 4. Two constructs, each comprising one of the alleles of the promoter, were obtained, wherein said promoter alleles drive the expression of gus  gene.<br>
Subsequently, different fragments of the promoter including the transcription initiation site were cloned into pBI101.3 vector. For cloning the promoter segments that are more distant from the transcription initiation site a modified pBI vector was utilized, the vector carries a minimal promoter (-90 CaMV35S) including the TATA box. The obtained clones are described as follows:<br>
Clone 416: fragment containing the region from 0 to -400 (from the transcription initiation site up to IPCR4 (SEQ ID N° 4)) of the promoter cloned into pBI101.3 Hindlll/BamRl. Clone 416 comprises the nucleotides 805 to 1221 of SEQ ID N° 3.<br>
Clone 1015: fragment containing the promoting region from 0 to -1015 (small allele) obtained with the nucleotides IPCR10 and IPCR8. (SEQ ID N° 5 and SEQ ID N° 6, respectively) cloned in SalI/BamHI  of pBI101.3.<br>
Clone 1221: fragment containing the promoting region, from 0 a -1221 (large allele) obtained with the oligonucleotides IPCR10 e IPCR8 (SEQ ID N° 5 and SEQ ID N° 6, respectively) cloned in Sall/BamRl  of pBI101.3.<br>
Clone 318: fragment containing the region from 0 to -300 (from the transcription initiation site up to IPCR6 (SEQ ID N° 7) amplified with the oligonucleotides IPCR6/IPCR8 (SEQ ID N° 7/SEQ ID N° 6)) of the promoter cloned in Hindlll/BamRl of pBI101.3. Clone 318 comprises the nucleotides 904 to 1221 of SEQ ID N° 3.<br>
Clone 211: fragment containing the region from 0 to -211 (from the transcription initiation site up to IPCR7 (SEQ ID N° 8) amplified with the oligonucleotides IPCR7/IPCR8(SEQ ID N° 8/SEQ ID N° 6) of the promoter cloned in HindIII/BamHI of pBI101.3. Clone 211 comprises the nucleotides 1011 to 1221 of SEQ ID N° 3.<br>
Clone 608: fragment of 608 pb (from the large allele) corresponding to 5' promoter region (amplified with IPCR5/IPCR10 (SEQ ID N° 9/SEQ ID N° 5)) cloned in SalI of pBI -90 with a minimal promoter. Clone 608 comprises the nucleotides 15 to 622 of SEQ ID N° 3.<br>
Clone 4 07: fragment of 4 07 pb (from the small allele) corresponding to the 5' promoter region (amplified with IPCR5/IPCR10(SEQ ID N° 9/SEQ ID N° 5)) cloned in SalI of pBI -90 with a minimal promoter. Clone 407 comprises the nucleotides 15 to 409 of SEQ ID N° 10.<br>
Subsequently, " plants of A. thaliana were transformed with the above mentioned constructs, the transformed plants were selected and several independent lines were chosen for isolating, in the third generation, homozigote sub-lines. The selected homozigote plants were used for characterizing the alleles and the different regions of the Hahb-4 promoter by hystochemistry and fluorimetry tests.<br>
The region 0 to -400 of the promoter was enough to drive the expression of gene gus in cotyledons of 2-day germination seedlings. Such expression was not detectable in the young seedlings at the 10 germination days.<br>
When the construct including the complete segment of the small allele was employed, the expression of the gus gene in the cotyledons of 2-day germination seedlings was also viewed. However, and differing from the region 0 to -<br>
400, the expression of gus gene was very intense in the leaves and roots of the young seedlings (Figure 20). Finally, it was not detected any expression of gus gene in the plants during the reproductive stage.<br>
These results indicate that while the expression occurs in the same organ or organs, the intensity of such expression is different. It is possible that in the region -400 to -1015 important sequences for transcription activation may be present, while the sequences necessary for driving the specific expression in cotyledons and leaves are in the region 0 to.-400.<br>
When the expression of gus gene in the roots of the transgenic plants was analyzed, it has been found that the expression driven by the construct 0 to -400 in the roots of 20-day old plants is consistently more strong in the lateral roots primordium and in the intermediate region of the lateral roots already developed. In addition, it was viewed that in the roots of the transformed transgenic plants with the construct of 0 to -400, the expression of the gus gene is also detectable in cells of the vascular cylinder (Figure 21) . 10 days after germination a strong expression in all the main and lateral root, and even stronger in the region of the roots base (Figure 21c) is observed.<br>
The results obtained by fluorimetry tests shown that the construct including the complete segment has a<br>
promoting activity 10 times higher than the promoting activity of the construct 0 to -400. Also, the construct 0 to -400 is capable of being induced by ABA, as it is shown in Figure 22.<br>
The analysis of several independent lines of plants transformed with the constructs comprising the region 0 to -300 or the region 0 to -200 shown that the gus gene is not expressed in the different development stages, in none of the studied organs, and said promoting regions are not induced by ABA.<br>
The inventors also disclose the presence of only one intron that interrupts the region of the leucine-zipper encoding sequence. This intron has 101 pb and is located specifically between the sequences encoding the sixth and seventh heptade of the leucine-zipper domain (aminoacids 108 and 109) and responds to rule 5'-GT...AG-3' present in the introns of other organisms. A scheme of location and sequence of the intron is shown in Figure 1.<br>
Therefore, the present invention comprises the Hahb-4 promoter (SEQ ID N° 1) and the cDNA of said gene (SEQ ID N° 2), wherein said gene encodes a protein of the sunflower Hd-Zip family. The promoter has two alleles that were cloned and sequenced, having a divergent or not conserved region around the nucleotide -900. The large allele comprises a sequence of 1221 bp and the small allele<br>
comprises a sequence of 1015 pb (SEQ ID N° 3 and SEQ ID N° 10, respectively).<br>
The analysis of the promoter nucleotide sequences indicated that there are regions homologous to different sequences involved in the response to several environmental factors such light, abscisic acid and hormones. The promoter of the invention comprises putative elements responsive to ABA commonly named ABRE as well as an element responsive to stress produced by lack of water or low temperatures called DRE. The consensus sequences joining the transcription factors involved in the response to environmental factors were also identified.<br>
When the activity of different regions of the inventive promoter was analyzed, the obtained results indicated that while there are two putative elements ABRE between the transcription initiation site and 300 nucleotides upstream, this segment is not capable to drive the reporter gus activity, showing that this segment, while necessary, is not enough to activate the transcription. On the contrary, by taking the first 400 nucleotides adjacent<br>
the position +1, the expression of gus gene in cotyledons<br>
after 2-days germination and in roots has been observed. The expression is not very strong but it is specific and capable to be induced by ABA.<br>
On the other hand,  the complete segment of the promoting region produces expression levels that are at<br>
least 10 times higher than the levels generated by the segment 0 to -400. In turn, this expression is evident in older stages of the development (plants of up to 20 days of germination) and in the central region of the root.<br>
The inventive promoter (both alleles) and the segments or parts thereof may be employed for driving the expression of any gene of interest. Preferably, the promoter of the invention (both alleles or segments thereof) may be employed in nucleic acid constructs useful to transform host cells, wherein said promoter drives the expression of any protein of interest. The host cells may be bacteria, yeast, animal cells or vegetal cells.<br>
Also a vector may be constructed wherein the inventive promoter drives the expression of the Hahb-4 gene of the invention in vegetal cells. Said constructions are useful for obtaining transgenic plants that are tolerant to water stress. The transgenic plants carrying the Hahb-4 gene of the invention and the promoter of the invention may be commercial plants including, while not restricted to, sunflower, wheat, barley, soybean, potato, corn, sugar cane or rice.<br>
The invention may be better understood with reference to the following examples which are not limitative or restrictive of the scope of protection. On the contrary, it must be clearly understood that many other embodiments, modifications and alterations equivalent to<br>
the elements of the invention may be suggested by persons skilled in the art after reading the present description, without departing from the spirit of the present invention and/or the scope of the appended claims.<br>
EXAMPLES<br>
EXAMPLE 1: isolation and characterization of the new sunflower gene Hahb-4, that is a gene encoding a protein belonging to the Hd-Zip I family.<br>
A. Hahb-4 gene cDNA isolation:<br>
For the isolation of partial cDNA clones containing homeobox sequences, a polymerase chain reaction (PCR)-based strategy was carried out as previously described (Gonzalez and Chan, Trends in Genetics 9:231-232, 1993, herein incorporated as reference) on total DNA from a sunflower stem cDNA library constructed in lambda gtlO. Sequences representing the 3'end of the Hahb-4 transcript were obtained by PCR using lambda gtlO sequencing primers and the specific primer H41 (5'-GGCGGATCCAACAGAAACAACCACCAGG-3' (SEQ ID N° 11)), that matches nucleotides 81-100 of the cDNA sequence (SEQ ID N° 2 and Fig. 1). The 5'-end of the transcript was obtained applying rapid amplification of cDNA ends (RACE) to polyA + RNA obtained from water-stressed sunflower stems using the specific oligonucleotide IPCR0 5'-GGCGGATCCCCTGGTGGTTGTTTCTGTT-3' (SEQ ID N° 12) and<br>
primers Qt and Qo  (SEQ ID N°  13 and SEQ ID N°  14, respectively)  according to Frohman (Frohman Cloning PCR products.  In The  Polymerase  Chain Reaction,  eds.  K.B. Mullis,  F.  Fre,  &amp; R.A. Gibas, pages 14-37. Birkhauser, Boston, MA, USA, 1994, herein incorporated as reference).<br>
B. Isolation of Hahb-4 genomic sequence: The 5'non-transcribed region of Hahb-4 was characterized using inverse PCR according to Ochman, Ayala &amp; Hartl (In Methods of Enzymology (ed R. Wu) Vol 218, pages 309-321. Academic Press, San Diego, CA. USA, 1993). Sunflower genomic DNA was partially digested under controlled conditions with Sau3A or HindIII. Circularization of DNA after digestion and purification was carried out overnight in the presence of 5 U of T4 DNA ligase (PromegaCorp., Madison, WI, USA) according to the manufacturer's instructions. Primer pairs used for amplification were 5'-GGCGGATCCCCTGGTGGTTGTTTCTGTT-3'(SEQ ID N° 12) and 5'-GCCGAATTCAGATTGAGCAAGAGTATAAC-3' (SEQ ID N° 15), or 5'-ACCTTTATAAAGACCACTC-3' (SEQ ID N° 16) and 5'ACGCAATGGTGAGTTGTAC-3' (SEQ ID N° 17).<br>
C. DNA sequence analysis:<br>
The PCR products were cloned into either pUC119 or pGEM-Teasy  (Promega Corp.).  The nucleotide sequence of<br>
inserts was obtained by the chain termination method using the fmol sequencing system (Promega Corp.).<br>
Example 2: Tests showing that Hahb-4   is induced by lack of water.<br>
A.	Plant material,  growth conditions and water<br>
stress treatments<br>
Helianthus annuus L. (sunflower cv. contiflor 15, from Zeneca, Balcarce, Argentina, or cv. Sunweed, from Rhone-Poulenc, Lyon, France) seeds were surface sterilized and grown on filter paper inside Petri dishes for a period of 4 d. Seedlings were then transferred to plastic supports containing Hoagland's medium and grown until they had six leaves (approximately 3 weeks). Water stress was imposed either by transferring 4-day-old- seedlings to Petri dishes with dry filter paper or by removing plants from the hydroponic culture. The treatment times were as indicated in the figures.<br>
B.	RNAse protection analysis:<br>
For RNAse protection analysis, total RNA (15 µg) prepared as described (Almoguera C, et. al.; Plant molecular Biology 19: 781-792, 1992) was hybridized to a specific Hahb-4 riboprobe synthesized by in vitro transcription using T3 RNA polymerase and [32P]CTP following the instructions of the manufacturer (Boehringer<br>
Mannheim, Mannheim, Germany). The template consisted of an insert corresponding to the coding region between +81 and + 429 (Fig. 1 and SEQ ID N° 1) cloned into the Spel/BamHI sites of pBlue-script SK-. The BamRI   site, not present in the cDNA, derives from amplification with oligonucleotide H41 as described above.  Restriction digestion of this template DNA with EcoRI    allows transcription of a 411-nucleotide RNA probe that contains 63 nucleotides from the vector  (from the T3 promoter to the Spel    site in the polylinker,  and  from the BamRI     to  the EcoRI     sites). Conditions  for  riboprobe  preparation,  hybridization, digestion with RNAse A,  and subsequent electrophoretic analysis of protected RNA fragments were as previously described (Coca M: A: et. al.; Plant Molecular Biology 31: 863-876, 1996, herein incorporated as reference).<br>
C. Northern analysis:<br>
Total RNA (20 µg) was denatured with formamide and formaldehyde,, separated in a 1.5% (w/v) agarose/6% formaldehyde gel, and blotted onto nylon membranes (Hybond N; Amersham-Pharmacia, Buckinghamshire, UK) essentially as described by Sambrook, Fritsch &amp; Maniatis (1989).<br>
Hybridization was performed overnight at 65 °C in 6 x SSC (1 x SSC is 0.15 M NaCl, 0.015 M Na3-citrate, pH 7.0), 0.1% (w/v) polyvinylpirrolidone, 0.1% (w/v) bovine serum albumin, 0.1% (w/v) Ficoll, 0.5% (w/v) sodium dodecyl<br>
sulphate (SDS). An SpeI/EcoRI fragment (from +424 to +674), corresponding to the 3' non coding region of the Hahb-4 cDNA plus the last 177 nucleotides of the coding region, which does not include the Hd-Zip domain, was labeled with [32P]dATP (1 x l08 dpm µg-1) by random priming (Sambrook et al. 1989) and used as probe. The filters were autoradiographed using Bio-Max films and transcreen (Eastman Kodak, Rochester, NY, USA) overnight. To check the amount of total RNA loaded in each lane, filters were then re-probed with a 25S rRNA from Vicia faba under similar conditions as those described above, except that hybridization was performed at 62 °C<br>
D. Preparation of nuclei:<br>
Sunflower nuclei and nuclear extracts were prepared<br>
from control, water-stressed, or ABA-treated seedlings (4-<br>
days-old) according to the technique described in Maliga et<br>
al.      (Methods in Plant Molecular Biology.  A Laboratory<br>
Course Manual, pages 233-260. Cold Spring Harbor Laboratory<br>
Press,  Cold  Spring  Harbor,  NY,  USA,  1995,  herein<br>
incorporated by reference). Protein patterns were analyzed<br>
by SDS-polyacryla-mide gel electrophoresis (PAGE) and total<br>
protein concentration was measured as described Sedmak J.<br>
et. al., Analytical Biochemistry 79: 544-552, 1977.<br>
E. DNA-binding assays:<br>
For electrophoretic mobility shift assays (EMSA),<br>
aliquots of purified nuclear proteins (30ug) were incubated<br>
with double-stranded DNA (0.3-0.6 ng, 30 000 c.p.m.,<br>
labeled with [32P]dATP by filling-in the 3'ends using the<br>
Klenow fragment of DNA polymerase) generated by either<br>
hybridization of the complementary oligonucleotides 5' -<br>
AATTCAGATCTCAATAATTGAGAG-3'	and	5'-<br>
GATCCTCTCAATTATTGAGATCTG-3' (SEQ ID N° 18 and SEQ ID N° 19) (binding site for Hahb-4 is underlined). Binding reactions (20uL) containing 20 mM HEPES-NaOH (pH 7-6), 40 mM NaCl, 0-2 mM ethylenedi-aminetetraacetic acid (EDTA), 1.0 mM dithiothreitol (DTT), 0.5% Triton X-100, 20% glycerol, and 1.5 µg poly(dl-dC), were incubated for 20 min at 25°C, supplemented with 2.5% (w/v) Ficoll and immediately loaded onto a running gel (5% acrylamide, 0.08% bis-acrylamide in 0.5 x TBE plus 2-5% glycerol; 1 x TBE is 90 mM Tris-borate, pH 8.3, 2 mM EDTA). The gel was run in 0.5 x TBE at 30 mA for 1.5 h and dried prior to autoradiography.<br>
Example 3: Test for providing tolerance to the water stress in Arabidopsis thaliana employing the sunflower Hahb-4 of the invention.<br>
A. Biologic Material:<br>
E. coli DH5α strain and Agrobacterium tumefaciens GV2260 strain were employed. For the transforming plants<br>
tests seeds of Arabidopsis  thaliana  ecotype Columbia-0 were used.<br>
B. Molecular cloning:<br>
For cloning Hahb-4    under the control of CaMV35S promoter a PCR reaction was carried out by using [as template] the clone corresponding to cDNA (Gago et al., Plant  Cell  &amp;  Environment  25:  633-640,  2002,  herein incorporated   as   a   reference)   and   two   specific oligonucleotides Tl:  5'-GCGGGATCCACCATGTCTCTTCAACAAGTA-3'; (SEQ.ID N° 20) and T2: 5'-GCCGAGCTCTTAGAACTCCCAACCACCTTTTG-3'  (SEQ ID N° 21) that hybridized in both ends of the coding region. In this manner regions 3' and 5' that do not encode the messenger ARN are eliminated thus decreasing the possible effects of post-transcriptional regulation. These oligonucleotides were designed in a manner that they can introduced in the amplified fragment in sites BamHI   and SacI   of the plasmid. In addition, the sequence established as consensus for an optimal translation was added to the oligonucleotide in the 5' cDNA end (oligonucleotide Tl) . The product of the PCR reaction was purified and digested with the above mentioned enzymes and cloned into pBI 121 vector using E.     coli    for the transformation. Once the desired clone was obtained the plasmidic DNA was introduced in A.     tumefaciens    according to the method disclosed by Hofgen and Willmitzer (Hofgen and Willmitzer, Nucleic Acid<br>
Research 16: 9977, 1998). The Agrobacterium strain having the pBI 121 plasmid where the gus gene has been replaced by Hahb-4  was called ATH4.<br>
The cloning and checking techniques were taken from Sambrook,  J.,  Fritsch,  E.F.  and  Maniatis,  T.  (1989) Molecular    Cloning:    A    Laboratory   Manual.     Second edition. Cold Spring Harbor Laboratory Press, Cold Spring Harbor, N.Y.<br>
C. Arabidopsis thaliana transformation: The method employed for transforming Arabidopsis thaliana was the one using immersion (floral dip) described by Clough and Bent (Clough and Bent, Plant J. 16: 735-743, 1998). The seeds recovered from the transforming test were sterilized and cultured in Petri dishes containing medium MS supplemented with kanamycin 40 mg/1. The resistant plants (Fl) were passed into soil and maintained up to produce seeds. The resulting lines (F2) were analyzed for the presence of the transgene {Hahb-4 gene) using PCR and the expression of the corresponding transcript was also analyzed by Northern. The lines expressing the transgene were reproduced up to obtain homozigote sub-lines.<br>
The preparations of total RNA from Arabidopsis thaliana  were carried out according to the method disclosed by Carpenter and Simon (Carpenter, C. and Simon, A. (1998) Preparation of RNA. Enln: Methods  in Molecular Biology,   vol<br>
82: Arabidopsis Protocols. J.M. Martinez-Zapater and J. Salinas (Eds.)/ Humana Press Inc., Totowa, New Jersey). For the Northern analysis, disclosed processes have been carried out (Ausubel, F.M., Brent, R., Kingston, R.E., Moore, D.D., Seidman, J.G., Smith, J. A. and Struhl, K. (1983) Current Protocols in Molecular Biology. John Wiley &amp; Sons, N.Y.<br>
For analyzing the Arabidopsis thaliana transformed plants by PCR technique preparations of total DNA were made by employing a rapid method disclosed by Li and Chory (Li, J. and Chory, J. (1998) Preparation of DNA from Arabidopsis. In: Methods in Molecular Biology, vol. 82: Arabidopsis Protocols. J.M. Martinez-Zapater and J. Salinas (Eds.), Humana Press Inc., Totowa, New Jersey)<br>
D. Phenotypic Analysis:<br>
Dl. Analysis in Petri plates:<br>
The seeds from Arabidopsis thaliana were sterilized by washing with ethanol 70% (v/v) (1 min), chlorine 5%-SDS 1% (15 min) and sterile distilled water (3 times). Then the seeds were suspended in 8ml of agar 0,1% and sowed in 150 mm Petri plates containing MS medium, with the addition of Gamborg's vitamins. The plates were maintained at 4°C during two days and then placed into a culture chamber with controlled light and temperature (16 hours light at 24°C and 8 hours darkness at 21°C). The required illumination<br>
conditions (150-200 µE/m 2) were artificially obtained by keeping the plants at a distance of 25 cm from six fluorescent tubes, intercalating white light and Grolux tubes (from Sylvania) adjacently located.<br>
The handling of plant material was done in sterile conditions. The MS culture medium was sterilized in autoclave and subsequently vitamins were added by filtration.<br>
D2: Tests in soil:<br>
The tests in soil were carried out in 12 cm diameter and 10 cm high pots. Depending on the test, each pot was sowed with about one to three seeds from Arabidopsis equidistantly distributed. The tray was covered with transparent plastic material until the appearance of the seedling, then the plastic was removed. Each 16 pots were placed in a plastic tray and the growing of the plants was followed in a culture chamber under the same above disclosed illumination conditions. Irrigation water was supplemented in the plastic trays.<br>
For carrying out water stress tests the tray was supplemented from the begining with 1, 1.5 or 2 litres of water. In all the cases no additional water was added until the end of the reproductive cycle. These irrigation conditions, by keeping the culture chamber under very low humidity conditions, generates in the plants a water stress<br>
when they have two pairs of leaves, complete rosette or developed flowers respectively. Water stress can be visualized from the dryness and cracking of the soil, the loss of turgence in the leaves and finally the death of the plants.<br>
Example 4: Isolation and characterization of Hahb-4 promoter, constructs containing different fragments of the promoter and transgenic plants from A. Thaliana containing said constructs.<br>
A. Vegetal material, culturing and treating conditions:<br>
Sunflower  seeds   from  Contiflor   15   culture {Helianthus  annuus  L.) (Zeneca) have been employed. For the plant transformation tests seeds from Arabidopsis   thaliana ecotype Columbia-0 were used.<br>
The seeds from Arabidopsis thaliana were sterilized by washing in ethanol 70% (v/v) (1 min), chlorine 5%-SDS 1% (15 min) and sterile distilled water (3 times). Subsequently the seeds were re-suspended in agar 8 ml at 0,1% and sowed in 150 mm Petri plates containing MS medium, with the addition of Gamborg's vitamins. The plates were maintained at 4°C for two days and then were placed into a culture chamber with controlled light and temperature (16 hours light at 24°C and 8 hours darkness at 21°C).<br>
Culturing of plants from Arabidopsic thaliana was carried out in a chamber with controlled light and temperature (16 hours with light at 24°C and 8 hours in darkness at 21°C). The required illumination conditions (150-200 µE/m 2) were artificially obtained by keeping the plants at a distance of 60 cm from six fluorescent tubes, intercalating white light tubes and Grolux tubes (from Sylvania) adjacently located.<br>
B.	Purification of plant DNA:<br>
For extracting total DNA from plants the method disclosed by Doyle y Doyle (Doyle, J.J. and Doyle, J.L. (1987) was used. A rapid DNA isolation procedure for small quantities from fresh leaf tissue. Phytochemical Bulletin 19, 11-15) has been carried out.<br>
C.	Plasmids, strains and molecular cloning methods<br>
employed:<br>
The pGEM.-T easy plasmid (Promega) , was used for cloning products from amplification reactions with Taq DNA polymerase (Promega).<br>
pBI10l plasmid (Jefferson et al., EMBO J. 6: 3901-3907, 1987) was also used, the plasmid being a derivative of pBIN19 binary vector. It contains the gene encoding E. coli ß-glucuronidase (gus) with the polyadenilation signal from nopaline synthetase [nos).   It also contains nptII  gene<br>
conferring resistance to kanamycin in plants. Other sequences that are relevant in the vector include the gene providing resistance to kanamycin in bacteria and a bacteria replication origin RK2. This plasmid was used for cloning different fragments of sunflower Hahb-4 promoter in Arabidopsis thaliana, in unique restriction sites located upstream of gus  gene.<br>
Cloning constructs in T-easy or pBI vector was firstly made by using competent host cells of E. coli DH5a following classic transformation protocols or electroporation as described in Sambrook (Sambrook, J., Fritsch, E.F. and Maniatis, T. (1989) Molecular Cloning: A Laboratory Manual. Second edition. Cold Spring Harbor Laboratory Press, Cold Spring Harbor, N.Y). For preparing competent cells of A. tumefaciens and further transformation thereof the method described by Hofgen and Willmitzer has been followed (Hofgen, R. and Willmitzer, L. (1988) Storage of competent Agrobacterium cells for transformation was carried out according to. Nucleic Acids Res.   16, 1977) .<br>
D. Molecular cloning of DNA fragments:<br>
For cloning the promoting region of the inventive<br>
gene the strategy of inverse PCR described by Ochman et al.<br>
has been employed (Ochman, H., Ayala, F.J. and Hartl, D.L.<br>
(1993) Use of polymerase chain reaction to amplify segments<br>
outside boundaries of known sequences. In: Methods in Enzimology vol 218. R. Wu (Ed.), Academic Press, San Diego, CA), was carried out as follows.<br>
The extraction of sunflower genomic DNA was made as described above, providing 5 ug of DNA and 1-3 U of SauIIIA or HindIII (Promega) per microgram of DNA were used. Once the digestion was verified (by sowing an aliquot in agarose gel 0,7% (p/v) ) , the fragments were precipitated by the addition of 1/10 vol. of 3 M NaAc (pH 5.2) and 2 volumes of absolute ethanol..<br>
For enhancing the re-ligation of the restriction fragments, the reactions were carried out in a final volume 100 µl. 2, 10 and 20 ng of the fragments obtained in each reaction and 5U T4 DNA ligase (Promega) were employed. The reactions were carried out for 16 hours at 14°C. The fragments were precipitated and purified for use as template in a PCR reaction.<br>
The PCR reactions in the first step were made by using as a template the DNA from the recycling of the fragments digested with SauIIIA, with oligonucleotides IPCR0/IPCR1 (SEQ ID N° 12/SEQ ID N°15) and in the case of the DNA digested with HindIII, the employed oligonucleotides were IPCR2/IPCR3 (SEQ ID N° 16/SEQ ID N° 17). The obtained fragments were cloned in pGEM®-T easy-vector (Promega), according to the protocol suggested by the manufacturer.  Once  the  cloning was  verified,  the<br>
corresponding   sequence	was   determined   and   the<br>
oligonucleotides necessary	for the next step were designed.<br>
The sequence and location	of the oligonucleotides used in<br>
the next cloning steps are	shown in Figure 23:<br>
IPCR0	[5'-GGCGGATCCCCTGGTGGTTGTTTCTGTTG-3']<br>
IPCR1	[5'-GCCGAATTCAGATTGAGCAAGAGTATAAC-3']<br>
IPCR2	[5'-ACCTTTATAAAGACCACTC-3']<br>
IPCR3	[5'-ACGCAATGGTGAGTTGTAC-3']<br>
IPCR4	[5'-GCGAAGCTTGATGCGAACGAGTGGTTTA]<br>
IPCR5	[5'-ATTTCGCAAGTAGTCCATT-3']<br>
IPCR6	[5'-CCCAAGCTTAACCTAAGTCCGCCTTTG-3']<br>
IPCR7	[5'-GGCAAGCTTATCTCAACCGAAAGTGAC-3]<br>
Finally, since this technique is apt for segmented cloning, the knowledge of the sequence for amplifying the complete segment using sunflower genomic DNA as mold[template], the polymerase chain reaction, and two oligonucleotides designed in such a manner to hybridize the opposite ends (IPCRIO: GCGGTCGACACCTGGCACATCGTATCT (SEQ ID N° 5) and IPCR8: CGCGGATCCGAGGGTTTGATAAGTGAT (SEQ ID N° 6)) were used. The amplification product was cloned in pGEM-Teasy vector and subsequently its sequence was determined.,.<br>
The strategy used for the inverse PCR as well as for the construction of recombinant plasmids used in the transformation of plants are shown in Figure 23.<br>
The different fragments of the promoter including the transcription initiation site were afterwards cloned into pBI101.3 vector. Alternatively, for studying the promoter segments that are distant from the transcription<br>
initiation site, a modified pBI vector carrying a minimum promoter (-90 CaMV35S) including TATA box was employed. This vector also contains the gus gene as a reporter thus permitting to measure the promoting activity of each cloned segment.<br>
E.	Cloning of Hahb-4  intron:<br>
For cloning the hahb-4 intron a PCR reaction was<br>
carried out by using as template sunflower genomic DNA and<br>
the	oligonucleotides	IPCR1	(5'-<br>
GCCGAATTCAGATTGAGCAAGAGTATAAC-3 (SEQ ID N° 15) y Nl (5'-GCGGGATCCGTCTGGCAGTTGTTCTTC-3'SEQ ID N° 22)). The obtained product was digested with EcoRI and BamHI (sites provided by the nucleotides) and subsequently cloned into pUC119 plasmid previously digested with the same enzymes. Once verifying the presence of the insert having the expected size in some of the white obtained colonies, its sequence was determined as described below.<br>
F.	Transformation of plants from Arabidopsis<br>
thaliana  with the constructs obtained in the prior stage.<br>
The method employed for transforming plants from Arabidopsis   thaliana   was the immersion method (floral   dip) described by Clough and Bent (Clough, S.J. and Bent, A.F. (1998) Floral dip: a simplified method for Agrobacterium-<br>
mediated transformation of Arabidopsis   thaliana.    Plant   J. 16, 735-743).<br>
For each construction, 10-12 pots having a diameter of 10 cm were prepared with soil and covered with a fibber fabric. The cloth was fixed in a manner to be well adhered to the soil surface. Then, the seeds were well spread in the soil and the pots were placed in a tray covered with transparent nylon paper. The plants were cultivated under the above mentioned conditions; after a week the nylon paper was removed and the strongest plants were selected.<br>
The plants were cultured up to the flowering stage (about 4 weeks). When the flower pedicels were breaking out (1-2 cm of the rosette), the inflorescences were cut by taking care of not damaging the caulinar leaves. 4-6 days after the above mentioned cut new inflorescences broke out. It was waited until all the inflorescences had at least not-yet open 4 flowers and then the transformation was carried out.<br>
For preparing the transformation, A. tumefaciens suspension cells were cultured in 3 flasks containing 10 ml LB medium supplemented with rifampicin 50 mg/1 and kanamycin 50 mg/1 for 24 hours at 28°C with agitation. These cultures were used for inoculating 3 Erlenmeyers containing 200 ml of the same medium and they were grown up to the stationary stage (12-16 hours at 28°C, with agitation). The cells were harvested by centrifugation at<br>
5500 x g for 20 minutes. The pellets were resuspended in a liter of 5% sucrose solution containing 300 ul Silwet L-77 (OSI Specialties, Inc.) and the suspension was placed in a precipitating glass with a magnetic stirrer. The plants were submerged for 10 to 60 seconds, trying to prevent the liquid to enter in contact with the soil. Then the pots were placed in a horizontal position in a tray, they were covered by a nylon paper and brought to the culture chamber. Next day the pots were placed in a normal position, water was added into the tray and the plants were left to grow up to the seeds were mature (4-5 weeks).<br>
Finally, the seeds were collected from each pot separately, and the siliques and soil were cleaned manually. Seeds were kept into a refrigerator for further analysis.<br>
For selecting the transformed plants, the seeds collected in the transformation tests were sterilized and sowed in Petri plates containing MS medium supplemented with 40 mg/1 kanamycin as it is above disclosed. During the first days in the culture chamber most of the seeds germinated (95%-99%).<br>
At about 10 days the cotyledons of the sensitive plants became yellow while the ones of the transformed plants remained green. The plates were maintained in the chamber for 7 additional days and during this period very green and real leaves were visible only in the transformed<br>
plants. The non transformed plants died. The resistant plants were transferred into flowerpots with soil. To prevent humidity suddenly decrease the pots were placed in a tray with water and covered with transparent nylon paper for a week. After this period of time the paper was removed to uncovered the tray and the plants were cultured up to the maturing of the seeds, the seeds were collected and placed into a refrigerator for further analysis. In addition to selecting the plants by their resistance to kanamycin the presence of the transgene was verified by specific PCR.<br>
Finally the homozygote lines of the third plant generation were selected by sowing in plates with kanamycin and observing 100% tolerance in several sublines.<br>
G. Histochemical analysis of p-glucuronidase activity in the inventive transformed plants.<br>
The inventive plants from A. thaliana that resulted resistant to kanamycin in the selection and PCR reaction of were subjected to histochemical ß-glucuronidase activity test. The tested embryos and organs were washed with buffer 50 mM Na2HPO4 pH 7. They were then transferred to a solution 50 mM Na2HPO4 pH 7, 0,1 % Triton X-100, 2 mM X-gluc (5-bromine-4-chlorine-3-indolil ß-D-glucuronide) , subjected to vacuum for 5 minutes and incubated at 37°C in darkness between 2 and 12 hours. After incubation they were<br>
fixed in 10 % formaldehide solution, 50% ethanol and 5% ascetic acid for 10 minutes at room temperature. Formaldehide was removed, ethanol 70% was added and conserved at 4°C.<br>
The histochemical analysis was made in 2, 10 and 20 day-old seedlings grown in Petri dishes containing MS medium and 0.8 % agar and adult plants grown in flowerpots containing soil.<br>
H. Fluorometric Analysis of ß-glucuronidase activity in the inventive transformed plants:<br>
The transformed plants that resulted resistant to the antibiotic, confirmed as positive lines by PCR reaction, and that expressed histochemically the enzyme ß-glucuronidase were used for studying the regulation of the promoting region by fluorometric tests. Between 30 and 50 seeds of each line were cultured in 30 mm Petri dishes with MS-agar medium. After an appropriate growth, the plants were transferred to tubes with MS liquid medium supplemented or not with ABA and incubated for different times as detailed below. After incubation the plants were maintained in liquid N2 up to their processing.<br>
The protein extract was obtained by homogenization of vegetal material (about 2-5 mg) up to obtain a fine powder. Then 500 µl extraction buffer (50 mM Na2HPO4 pH 7, 10 mM EDTA, 0,1%SDS, 10 mM ß-mercaptoethanol, 1% Triton X-<br>
100) was added. The suspension was transferred into an Eppendorf tube and centrifuged at 13000 xg for 10 minutes at 4°C. The supernatant was removed and the pellet was kept in ice.<br>
The fluorometric reaction was made according to the Jefferson method (Jefferson, R.A., Kavanagh, T.A. and Bevan, M.W. (1987) Gus fusions: ß-glucuronidase as a sensitive and versatile gene fusion marker in higher plants. EMBO J. 6, 3901-3907). 100 µl protein extract were added to l00µl methanol and 300 µl substrate MUG (4- methyl umbeliferil ß-D-glucuronide) . A 100 µl aliquot was extracted and a flurometric measure was immediately made aft] time 0. The remaining 400 µl were incubated at 37°C in boiling water and 100 µl aliquots were removed at the times of 30, 60 and 120 minutes. For stopping the reaction 0,9 ml of 0,2 M Na2CO3 were used. The values of the fluorometric reactions were expressed in pmols of product/mg of total proteins per minute according to a pattern curve of RFU as a function of concentration of product 4-MU (7-hydroxi-4 methyl umbeliferone).<br>
The fluorometric measures were made in an equipment VersaFluorTM Fluorometer System, from Bio-Rad (filters EM 460/10 y EX 360/40) in 1 ml dishes.<br>
I. Quantification of total proteins:<br>
The concentration of the soluble proteins of the protein extracts was determined by using the method disclosed by Sedmak y Grossberg (Sedmak, J. and Grossberg, S. (1977) A rapid, sensitive, and versatile assay for protein using Coomassie brilliant blue G-250. Anal. Biochem. 79, 544-552). Bovine serum albumin (BSA) was used as pattern.<br>
J. Determination and analysis of sequences: For determining the DNA sequence of the obtained constructs  a  commercial  equipment  T7  Sequencing  kit (Amersham Biosciences) was used, the method being based on the Sanger method (Sanger, F., Nicklen, S. and Coulson, A.R.   (1977)   DNA   sequencing   with   chain-terminating inhibitors.  Proc. Natl. Acad.    Sci.     USA    74,  5463-5467), combining  DNA extension/termination  in  only  one  step. Furthermore, these sequences were confirmed in an automatic sequencer (Service given High Complexity Laboratory, INTA,,. Castelar, Bs. As. Argentina).<br>
For identifying the regulating sequences within the promoting regions the data bases PLACE (Higo, K., Ugawa, Y., Iwamoto, M. and Korenaga, T. (1999) Plant cis-acting regulatory DNA elements (PLACE) database: 1999. Nucleic Acids Res. 27, 297-300) and PlantCARE (Rombauts, S., Dehais, P., Van Montagu, M. and Rouze, P. (1999) PlantCARE,<br><br>
a plant cis-acting regulatory element database. Nucleic Acids Res.   27, 295-296) were used.<br>
When the employed technique is not specified the classical protocols disclosed in Sambrook, J., Fritsch, E.F. and Maniatis, T. (1989) Molecular Cloning: A Laboratory Manual. Second edition. Cold Spring Harbor Laboratory Press, Cold Spring Harbor, N.Y. and Ausubel, F.M., Brent, R., Kingston, R.E., Moore, D.D., Seidman, J.G., Smith, J.A. and Struhl, K. (1983) Current Protocols in Molecular Biology. John Wiley &amp; Sons, N.Y., have been employed<br>
While preferred embodiments of the present invention have been illustrated and described, it will be obvious to those skilled in the art that various changes and modifications may be made therein without departing from the scope of the invention as defined in the appended claims.<br><br><br><br><br><br><br><br>
WE CLAIM:<br>
1.	A nucleic acid construct comprising a first nucleic acid molecule having the nucleotide<br>
sequence of SEQ ID NO. 10,<br>
wherein said first nucleic acid molecule is a promoter and is operably linked to a second nucleic acid molecule encoding a protein of interest and a 3'non-translated region.<br>
2.	The nucleic acid construct as claimed in claim 1, wherein said second nucleic molecule is<br>
SEQ ID NO: 2.<br></td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDgzNC1kZWxucC0yMDA5LUFic3RyYWN0LSgyOC0wNC0yMDE0KS5wZGY=" target="_blank" style="word-wrap:break-word;">4834-delnp-2009-Abstract-(28-04-2014).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDgzNC1kZWxucC0yMDA5LWFic3RyYWN0LnBkZg==" target="_blank" style="word-wrap:break-word;">4834-delnp-2009-abstract.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDgzNC1ERUxOUC0yMDA5LUFzc2lnbm1lbnQtKDI4LTA4LTIwMDkpLnBkZg==" target="_blank" style="word-wrap:break-word;">4834-DELNP-2009-Assignment-(28-08-2009).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDgzNC1kZWxucC0yMDA5LUNsYWltcy0oMjgtMDQtMjAxNCkucGRm" target="_blank" style="word-wrap:break-word;">4834-delnp-2009-Claims-(28-04-2014).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDgzNC1kZWxucC0yMDA5LWNsYWltcy5wZGY=" target="_blank" style="word-wrap:break-word;">4834-delnp-2009-claims.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDgzNC1kZWxucC0yMDA5LUNvcnJlc3BvbmRlbmNlIE90aGVycy0oMTktMDEtMjAxMikucGRm" target="_blank" style="word-wrap:break-word;">4834-delnp-2009-Correspondence Others-(19-01-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDgzNC1kZWxucC0yMDA5LUNvcnJlc3BvbmRlbmNlIE90aGVycy0oMjgtMDQtMjAxNCkucGRm" target="_blank" style="word-wrap:break-word;">4834-delnp-2009-Correspondence Others-(28-04-2014).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDgzNC1ERUxOUC0yMDA5LUNvcnJlc3BvbmRlbmNlLU90aGVycy0oMDMtMTItMjAwOSkucGRm" target="_blank" style="word-wrap:break-word;">4834-DELNP-2009-Correspondence-Others-(03-12-2009).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDgzNC1ERUxOUC0yMDA5LUNvcnJlc3BvbmRlbmNlLU90aGVycy0oMjgtMDgtMjAwOSkucGRm" target="_blank" style="word-wrap:break-word;">4834-DELNP-2009-Correspondence-Others-(28-08-2009).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDgzNC1kZWxucC0yMDA5LUNvcnJlc3BvbmRlbmNlLU90aGVycy0oMjgtMTAtMjAxNCkucGRm" target="_blank" style="word-wrap:break-word;">4834-delnp-2009-Correspondence-Others-(28-10-2014).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDgzNC1kZWxucC0yMDA5LWNvcnJlc3BvbmRlbmNlLW90aGVycy5wZGY=" target="_blank" style="word-wrap:break-word;">4834-delnp-2009-correspondence-others.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDgzNC1kZWxucC0yMDA5LWRlc2NyaXB0aW9uIChjb21wbGV0ZSkucGRm" target="_blank" style="word-wrap:break-word;">4834-delnp-2009-description (complete).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDgzNC1kZWxucC0yMDA5LURyYXdpbmdzLSgyOC0wNC0yMDE0KS5wZGY=" target="_blank" style="word-wrap:break-word;">4834-delnp-2009-Drawings-(28-04-2014).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDgzNC1kZWxucC0yMDA5LWRyYXdpbmdzLnBkZg==" target="_blank" style="word-wrap:break-word;">4834-delnp-2009-drawings.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDgzNC1ERUxOUC0yMDA5LUZvcm0tMS0oMjgtMDgtMjAwOSkucGRm" target="_blank" style="word-wrap:break-word;">4834-DELNP-2009-Form-1-(28-08-2009).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDgzNC1kZWxucC0yMDA5LWZvcm0tMS5wZGY=" target="_blank" style="word-wrap:break-word;">4834-delnp-2009-form-1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDgzNC1ERUxOUC0yMDA5LUZvcm0tMTgtKDAzLTEyLTIwMDkpLnBkZg==" target="_blank" style="word-wrap:break-word;">4834-DELNP-2009-Form-18-(03-12-2009).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDgzNC1ERUxOUC0yMDA5LUZvcm0tMi0oMjgtMDgtMjAwOSkucGRm" target="_blank" style="word-wrap:break-word;">4834-DELNP-2009-Form-2-(28-08-2009).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDgzNC1kZWxucC0yMDA5LWZvcm0tMi5wZGY=" target="_blank" style="word-wrap:break-word;">4834-delnp-2009-form-2.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDgzNC1kZWxucC0yMDA5LUZvcm0tMy0oMTktMDEtMjAxMikucGRm" target="_blank" style="word-wrap:break-word;">4834-delnp-2009-Form-3-(19-01-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDgzNC1kZWxucC0yMDA5LUZvcm0tMy0oMjgtMDQtMjAxNCkucGRm" target="_blank" style="word-wrap:break-word;">4834-delnp-2009-Form-3-(28-04-2014).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDgzNC1kZWxucC0yMDA5LWZvcm0tMy5wZGY=" target="_blank" style="word-wrap:break-word;">4834-delnp-2009-form-3.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDgzNC1kZWxucC0yMDA5LWZvcm0tNS5wZGY=" target="_blank" style="word-wrap:break-word;">4834-delnp-2009-form-5.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDgzNC1kZWxucC0yMDA5LWZvcm0tNi0oMjgtMDgtMjAwOSkucGRm" target="_blank" style="word-wrap:break-word;">4834-delnp-2009-form-6-(28-08-2009).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDgzNC1kZWxucC0yMDA5LUdQQS0oMjgtMDQtMjAxNCkucGRm" target="_blank" style="word-wrap:break-word;">4834-delnp-2009-GPA-(28-04-2014).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDgzNC1ERUxOUC0yMDA5LUdQQS0oMjgtMDgtMjAwOSkucGRm" target="_blank" style="word-wrap:break-word;">4834-DELNP-2009-GPA-(28-08-2009).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDgzNC1kZWxucC0yMDA5LVBldGl0aW9uLTEzNy0oMjgtMDQtMjAxNCkucGRm" target="_blank" style="word-wrap:break-word;">4834-delnp-2009-Petition-137-(28-04-2014).pdf</a></p>
		<br>
		<div class="pull-left">
			<a href="264275-use-of-polyetheresters-for-rotomolding.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="264277-gamma-undecenolactones-and-process-of-making-the-same.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>264276</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>4834/DELNP/2009</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>51/2014</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>19-Dec-2014</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>18-Dec-2014</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>24-Jul-2009</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>BIOCERES S.A.,</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td>878 MORENO STREET-3 FLOOR, (S 2000 DKP) ROSARIO, SANTA FE, ARGENTINA,</td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>LIA RAQUEL CHAN</td>
											<td>CATEDRA DE BIOLOGIA CELULAR Y MOLECULAR, FACULTAD DE BIOQUMICAY CIENCIAS BIOLOGICS UNL-CC 242-&quot;E1 POZO&quot; S/N-3000 SANTA FE-ARGENTINA.</td>
										</tr>
										<tr>
											<td>2</td>
											<td>HECTOR DANIEL GONZALEZ</td>
											<td>CATEDRA DE BIOLOGIA CELULAR Y MOLECULAR, FACULTAD DE BIOQUMICAY CIENCIAS BIOLOGICS UNL-CC 242-&quot;E1 POZO&quot; S/N-3000 SANTA FE-ARGENTINA.</td>
										</tr>
										<tr>
											<td>3</td>
											<td>CARLOS ALBERTO DEZAR</td>
											<td>CATEDRA DE BIOLOGIA CELULAR Y MOLECULAR, FACULTAD DE BIOQUMICAY CIENCIAS BIOLOGICS UNL-CC 242-&quot;E1 POZO&quot; S/N-3000 SANTA FE-ARGENTINA.</td>
										</tr>
										<tr>
											<td>4</td>
											<td>GABRIELA MARISA GAGO</td>
											<td>CATEDRA DE BIOLOGIA CELULAR Y MOLECULAR, FACULTAD DE BIOQUMICAY CIENCIAS BIOLOGICS UNL-CC 242-&quot;E1 POZO&quot; S/N-3000 SANTA FE-ARGENTINA.</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>C12N 5/04</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>PCT/US2003/013770</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td>2003-05-02</td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td>na</td>
									<td>1900-01-01</td>
								    <td>IB</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/264276-a-nucleic-acid-construct by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 00:35:10 GMT -->
</html>
