<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/264464-method-for-preparing-an-antigen-loaded-immunogenic-carrier-complex by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 01:31:10 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 264464:METHOD FOR PREPARING AN ANTIGEN-LOADED IMMUNOGENIC CARRIER COMPLEX</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">METHOD FOR PREPARING AN ANTIGEN-LOADED IMMUNOGENIC CARRIER COMPLEX</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>The invention relates to the areas of immunology and vaccine delivery. More specifically, it relates to a bacterial vaccine delivery technology with built-in immunostimuiatory properties which allows the immobilization of any antigen of interest, without prior antigen modification. Provided is an antigen-loaded immunogenic carrier complex comprising at least one bifunctional polypeptide attached to an immunogenic carrier, said bifunctional polypeptide comprising a peptidoglycan binding domain (PBD) through which the polypeptide is attached to said carrier, fused to an antigen binding domain (ABD) to which at least one antigen of interest is bound. Also provided is a pharmaceutical (e.g. vaccine) composition comprismg an antigen-loaded immunogenic carrier complex of the invention.</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td> <br><br>
Title:	Bifunctional protein anchors<br>
The invention relates to the areas of immunology and vaccine delivery. More specifically, it relates to a bacterial vaccine delivery technology with built-in immunostimuLatory properties, which allows the immobilization of any antigen of interest, without prior antigen modification.<br>
Vaccine delivery or immunisation via attenuated bacterial vector strains expressing distinct antigenic determinants against a wide variety of diseases is now commonly being developed. Mucosal (for example nasal or oral) vaccination using such vectors has received a great deal of attention. For example, both systemic and mucosal antibody responses against an antigenic determinant of the hornet venom were detected in mice oraUy colonised with a genetically engineered human oral commensal Streptococcus gordonii expressing said antigenic determinant on its surface (Medaglini et al., PNAS 1995, 2; 6868-6872).<br>
Also, a protective immune response could be elicited by oral delivery of a recombinant bacterial vaccine wherein tetanus toxin fragment C was expressed constitutively in Lactococcus lactis (Robinson et al., Nature Biotechnology 1997, 15; 653-657), Especially mucosal immunisation as a means of inducing IgG and secretory IgA antibodies directed against specific pathogens of mucosal surfaces is considered an effective route of vaccination. Immunogens expressed by bacterial vectors are presented in particulate form to the antigen-presenting cells (for example M-cells) of the immune system and should therefore be less likely to induce tolerance than soluble antigens. In addition, the existence of a common mucosal immune system permits immunisation on one specific mucosal surface to induce secretion of antigen-specific IgA, and other specific immune responses at distant mucosal sites. A drawback to this approach is the potential of the bacterial strain to cause inflammation and disease in itself, potentially leading to fever and bacteraemia. An alternative approach avoids the use of attenuated bacterial strains that may become<br><br>
pathogenic themselves by choosing recombinant commensal bacteria as vaccine carriers, such as Lactobacillus ssp. and Lactococcus ssp.<br>
However, a drawback of the use of such recombinant organisms is that they may colonise the mucosal surfaces, thereby generating a long term exposure to the target antigens expressed and released by these recombinant micro-organisms. Such long term exposure can cause immune tolerance. In addition, the mere fact alone that such organisms are genetically modified and contain recombinant nucleic acid(s) is meeting considerable opposition from the(lay) public as a whole, stemming from a low level of general acceptance for products containing recombinant DNA or RNA. Similar objections exist against the use of (even attenuated) strains of a pathogenic nature or against proteins or parts of proteins derived from pathogenic strains. As explained above, commonly used techniques of heterologous surface display of proteins in general entail the use of anchoring or targeting proteins that are specific and selective for a limited set of micro-organisms which in general are of recombinant or pathogenic nature, thereby greatly restricting their potential applications. We previously addressed this issue in patent applications WO 99/25836 and WO 02/101026, which describe the use of a chimeric fusion protein containing an AcmA(-like) binding domain fused to an antigen to attach antigens to non-viable spherical peptidoglycan particles derived from non-recombinant Gram-positive bacteria. The Gram-positive bacteria receive a non-enzymatic pretreatment (see WO 02/101026) before they are formulated with the antigens. The peptidoglycan particles, previously referred to as 'ghosts*, still contain bacterial components, like peptidoglycan, which have immunostimulatory properties. Accordingly, these particles are now referred to as Gram-positive Enhancer Matrix C'GEM") or "GEM particles".<br>
Thus, the methods disclosed in WO 99/25836 and WO 02/101026 avoid the use of live bacteria and / or of micro-organisms which in general are of recombinant or pathogenic nature. However, these previously disclosed methods are limited to the attachment of proteinaceous antigens that can be produced (recombinantly) as a chimeric proteinaceous product. For some protein antigens this may not be a feasible approach. There may for instance be specific requirements for the production of the antigen in which the presence of an AcmA(-like) binding domain, can interfere. In<br><br>
addition, for non-proteinaceous antigens a genetic fusion can of course not be made. Also, the method does not allow the attachment of particulate antigens. It can be envisaged to couple an antigen of interest covalently by chemical means to a peptidoglycan particle, for instance using a chemical cross-linker reactive with both the antigen and the bacterial particle. The peptidoglycan layer of the cell wall of lactic acid bacteria is covered by a variety of substances, for example (lipo)teichoic acids, neutral and acidic polysaccharides, and (surface) proteins. However, this chemical approach may not be suitable for every type of antigen since chemical modification can interfere with antigen efficacy to induce the immune system. Furthermore, most chemical crosslinkers require a specific reactive group (e.g. SH) to mediate a covalent interaction, which group may not always be present or which may be located at an undesirable (e.g. antigen binding) site within the molecule(s) cross-linked.<br>
It is a goal of the present invention to overcome these limitations. To that end, bifunctional polypeptides were developed that contain a functionality to bind (non-covalently) an antigen of interest as well as a functionality to bind (non-covalently) an immimogenic carrier, such as a GEM particle. This system allows the immobilization of any antigen of interest, without prior modification, on the surface of GEM particles. The antigens can be (poly)peptides, carbohydrates, lipids, DNA, RNA or any other bio-organic compound and can even have a particulate nature by themselves, e.g. viral particles.<br>
Therefore, the invention relates to an antigen - loaded immunogenic carrier complex comprising at least one polypeptide attached to an immunogenic carrier, said polypeptide comprising a peptidoglycan binding domain (PBD) through which the polypeptide is attached to said carrier, fused to an antigen binding domain (ABD) capable of binding an antigen of interest. In an antigen-loaded complex of the invention, at least one antigen of interest is bound (non-covalently) to said ABD. The PBD comprises an amino acid sequence capable of binding to peptidoglycan, which sequence is selected from the group consisting of (i) a LysM domain, (ii) an amino acid sequence retrieved from a homology search in an amino acid sequence database with one of the three LysM domains (repeated regions) in the C-terminus oi Lactococcus lactis cell wall hydrolase AcmA (said domains herein also referred to as AcmA LysM<br><br>
domains) and (iii) a sequence showing at least 70% identity to any one of the three AcmA LysM domains.<br>
The PBD is capable of attaching to the cell wall of a Gram-positive microorganism. The term "antigen binding" is meant to indicate the capacity to bind an antigen of interest. Said capacity is conferred by at least one bifunctional polypeptide. The term "bifunctional" indicates that the polypeptide has at least two different functionalities: a peptidoglycan binding functionality and an antigen binding functionaliiy. The functionahties can be multivalent, e.g. a bifunctional polypeptide may comprise multiple antigen binding sites.<br>
The term "immunogenic carrier' refers to a moiety which, upon administration to a subject, has the capacity to enhance or modify the immune-stimulating properties of an antigen attached to it. An immunogenic carrier thus has adjuvant properties. Furthermore, it comprises peptidoglycans to allow attachment of one or more bifunctional linker polypeptide(s) via its peptidoglycan binding domain (PBD). Non-recombinant immunogenic carriers are preferred for reasons given above. In a preferred embodiment, the immunogenic carrier complex is a non-viable spherical peptidoglycan particle obtained from a Gram-positive bacterium (GEM particle, or "ghost"). Methods for the preparation of GEM particles have been described before, for instance in patent applications WO 02/101026 and WO 2004/102199. The process preserves most of the bacteria's native spherical structure. Briefly, the method comprises treating Gram-positive bacteria with a solution capable of removing a cell-waU component, such as a protein, lipoteichoic acid or carbohydrate, from the cell-wall material. The resulting GEM particles may be subsequently stored until it is contacted with a desired bifunctional polypeptide. GEM particles bind substantially higher amounts of a PBD fusion than untreated Gram-positive bacteria. Therefore, a high loading capacity can be achieved for antigens on GEM particles (WO 02/101026). GEM particles are also better able to bind to and/or are more easily taken up by specific cells or tissues than mechanically disrupted cell-wall material. The ability of GEM particles to target macrophages or dendritic cells enhances their functional efficacy. The non-recombinant, non-living immunogenic carrier complex of the present invention is therefore weU suited as a vaccine delivery vehicle. See also WO 02/101026 and WO2004/102199.<br><br>
In one embodiment the invention provides a vaccine delivery technology, which is based on GEM particles with one or more antigens attached to the particles through the use of bifunctional polypeptides, wherein the GEM particles serve as immunogenic backbone to surface attach compounds of pathogenic origin, thereby mimicking a pathogenic particle (Fig. 1). This delivery technology can mimic a pathogen by delivering subunit vaccines as a particle to the immunoreactive sites.<br>
The GEM particles can in principle be prepared &amp;om any Gram-positive bacterium. The cell walls of Gram-positive bacteria include complex networks of peptidoglycan layers, proteins, lipoteichoic acids and other modified carbohydrates. Chemical treatment of the bacterial cell-wall material may be used to remove cell-wall components such as proteins and lipoteichoic adds to result in GEM particles with improved binding characteristics. Preferably, an antigen binding immunogenic carrier complex of the invention comprises GEM particles obtained using an acid solution (see e.g. WO 02/101026).<br>
In a preferred embodiment, the immunogenic carrier complex is prepared faom a nonpathogenic bacterium, preferably a food-grade bacterium or a bacterium with the G.R-A.S. (generally-recognized-as-safe) status. In one embodiment, the cell-wall material is derived from a Lactococcus, a Lactobacillus, a Bacillus or a Mycobacterium ssp. Use of a Gram-positive, food-grade bacterium, such as Lactococcus lactis, offers significant advantages over use of other bacteria, such as Salmonella or Mycobacteriumy as a vaccine delivery vehicle. L. lactis does not replicate in or invade human tissues and reportedly possesses low intrinsic immunity (Norton et al. 1994). L. lactis expressing tetanus toxin fragment C has been shown to induce antibodies after mucosal delivery that protect mice against a lethal challenge with tetanus toxin even if the carrier bacteria were killed prior to administration (Robinson et al. 1997). In contrast to the non-recombinant GEM particles in an immunogenic carrier complex disclosed herein, these bacteria still contain recombinant DNA that will be spread into the environment, especially when used in wide-scale oral-immunization programmes. This uncontrollable shedding of recombinant DNA into the environment may have the risk of uptake of genes by other bacteria or other (micro) organisms.<br><br>
A polypeptide of the invention comprises a peptidoglycan binding domain (PBD) which allows for the attachment of any antigen of interest to an immunogenic carrier, such as a GEM. In one embodiment, the PBD comprises an amino acid sequence capable of binding to peptidoglycan, which sequence is a LysM domain. Preferably, a polypeptide comprises at least two, more preferably at least three LysM domains. The LysM (lysin motif) domain is about 45 residues long. It is found in a variety of enzymes involved in bacterial cell wall degradation (Joris et al., FEMS Microbiol Lett 1992;70:257-264). The LysM domain is assxmied to have a general peptidoglycan binding function. The structure of this domain is known ("The structure of a LysM domain from JE, coli membrane-bound lytic murein transglycosylase D (MltD)". Bateman A, Bycroft M; J Mol Biol 2000;299:1113-11192). The presence of the LysM domains is not limited to bacterial proteins. They are also present in a niunber of eukaryotic proteins, whereas they are lacking in archaeal proteins. A cell wall binding function has been postulated for a number of proteins containing LysM domains. Partially purified muramidase-2 of Enterococcus hirae, a protein similar to AcmA and containing six LysM domains, binds to peptidoglycan fragments of the same strain. The p60 protein oi Listeria monocytogenes contains two LysM domains and was shown to be associated with the cell surface. The y-D-glutamate-meso-diaminopimelate muropeptidases LytE and LytF oi Bacillus subtilis have three and five repeats, respectively, in their N-termini and are both cell wall-bound.<br>
A skilled person will be able to identify a LysM domain amino acid sequence by conducting a homology-based search in publicly available protein sequence databases using methods known in the art. A variety of known algorithms are disclosed publicly and a variety of publicly and commercially available software can be used. Examples include, but are not limited to MacPattern (EMBL), BLASTP (NCBI), BLASTX (NCBI) and FASTA (University of Virginia). In one embodiment, PFAM accession number PF01476 for the LysM domain (see httD://www.sanger.ac.uk/cgi-bin/Pfam/getacc?PFO 1476) is used to search for an amino acid sequence which fulfils the criteria of a LysM domain. The PFAM website provides two profile hidden Markov models (profile HMMs) which can be used to do sensitive database searching using statistical descriptions of a sequence family's consensus. HMMER is a freely distributable implementation of profile HMM software for protein sequence analysis.<br><br>
The C-terminal region of the major autolysin AcmA of L. lactis contains three homologous LysM domains, which are separated by nonhomologous sequences. For the amino acid sequences of the three AcmA LysM domains see for example Figure 10 of W099/25836 wherein the three LysM domains are indicated by Rl, R2 and R3. The C-terminal region of AcmA was shown to mediate peptidoglycan binding of the autolysin (Buist et al. [1995] J. Bacteriol. 177:1554-1563). In one embodiment, an antigen binding immunogenic carrier complex of the invention comprises a bifunctional polypeptide bound via its PBD to a peptidoglycan at the surface of the immunogenic carrier, preferably a GEM particle, wherein said PBD comprises at least one LysM domain as present in AcmA Variations within the exact amino acid sequence of an AcmA LysM domain are also comprised, under the provision that the peptidoglycan binding functionality is maintained. Thus, amino acid substitutions, deletions and/or insertions may be performed without losing the peptidoglycan binding capacity. Some parts of the AcmA LysM domains are less suitably varied, for instance the conserved GDTL and GQ motifs found in all three domains. Others may however be altered without affecting the efficacy of the PBD to bind the immunogenic carrier. For example, amino acid residues at positions which are of very different nature (polar, apolar, hydrophilic, hydrophobic) amongst the three LysM domains of AcmA can be modified. Preferably, the PBD comprises a sequence that is at least 70%, preferably 80%, more preferably 90%, like 92%, 95%, 97% or 99%, identical to one of the three LysM domains of L. lactis AcmA The PBD of a polypeptide for use in the present invention may contain one or more of such (homologous) AcmA LysM domains, either distinct or the same. Typically, the LysM domains are located adjacent to each other, possibly separated by one or more amino acid residues. The LysM domains can be separated by a short distance, for example 1-15 amino acids apart, or by a medium distance of 15-100 amino acids, or by a large distance, like 150 or even 200 amino adds apart.<br>
In a certain aspect, the invention concerns a PBD, comprising an amino acid sequence having at least about 80% amino acid sequence identity, alternatively at least about 81% amino acid sequence identity, alternatively at least about 82% amino acid sequence identy, alternatively at least about 83% amino acid sequence identity, alternatively at least about 84% amino acid sequence identity, alternatively at least<br><br>
about 85% amino acid sequence identity, alternatively at least about 86% amino add sequence identity, alternatively at least about 87% amino acid sequence identity, alternatively at least about 88% amino acid sequence identity, alternatively at least about 89% amino acid sequence identity, alternatively at least about 90% amino acid sequence identity, alternatively at least about 91% amino acid sequence identity, alternatively at least about 92% amino acid sequence identity, alternatively at least about 93% amino add sequence identity, alternatively at least about 94% amino acid sequence identity, alternatively at least about 95% amino add sequence identity, alternatively at least about 96% amino add sequence identity, alternatively at least about 97% amino add sequence identity, alternatively at least about 98% amino acid sequence identity and alternatively at least about 99% amino add sequence identity to an AcmA LysM domain.<br>
The "percentage of amino acid sequence identity" for a polypeptide, such as 70, 80, 90, 95, 98, 99 or 100 percent sequence identity may be determined by comparing two optimally aligned sequences over a comparison window, wherein the portion of the polypeptide sequence in the comparison window may include additions or deletions (i.e. gaps) as compared to the reference sequence (which does not comprise additions or deletions) for optimal alignment of the two amino acid sequences. The percentage is calculated by: (a) determining the number of positions at which the identical amino acid occurs in both sequences to yield the number of matched positions; (b) dividing the number of matched positions by the total number of positions in the window of comparison; and (c) multiplying the result by 100 to yield the percentage of sequence identity. Optimal ahgnment of sequences for comparison may be conducted by computerized implementations of known algorithms, or by inspection. Readily available sequence comparison and multiple sequence alignment algorithms are, respectively, the Basic Local Alignment Search Tool (BLAST) (Altschul, S.F. et al. 1990. J. MoL Biol, 215:403; Altschul, S.F. et al, 1997, Nucleic Acid Res. 25:3389-3402) and ClustalW programs both available on the internet.<br>
In another embodiment, a PBD comprises a LysM domain which is present in an amino acid sequence retrieved from a homology search in an amino acid sequence database with an AcmA LysM domain, wherein the LysM domain is capable of attaching the substance to the cell wall of a Gram-positive microorganism. Preferably,<br><br>
the amino acid sequence retrieved is an amino acid sequence originating from a Gram-positive bacterium. It is for instance an amino acid sequence of a bacterial cell wall hydrolase. Preferably, the retrieved amino acid sequence shows at least 70%, more preferably 80%, most preferably at least 90% sequence identity with an AcmA LysM domain. Examples of sequences that may be retrieved can be found in Figure 11 of patent application W099/25836.<br>
As will be clear from the above, a PBD can be structurally defined in various manners. However, in all cases a PBD can be defined as a means for binding to the cell wall of a microorganism, wherein said means for binding is of peptidic nature. In one embodiment, the PBD is capable of binding to a Gram-positive bacterium or cell wall material derived thereof (e.g. a GEM particle). The binding capacity of a PBD can be readily determined in a binding assay comprising the steps of labeling the PBD with a reporter molecule, contacting the labeled PBD with a Gram-positive micro-organism to allow for binding of said means to said micro-organism; and determining the binding capacity of said PBD by detecting the absence or presence of reporter molecule associated with the micro-organism.<br>
The reporter molecule, also referred to as detectable molecule, for use in the binding assay can be of various nature. Many types of reporter molecules are known in the art. It is for example a fluorescent molecule (e.g. FITC), an antigen, an affinity tag (e.g. biotin) an antibody or an enzyme. A reporter molecule can be conjugated to the PBD by methods known in the art. In case the reporter molecule is of peptidic nature, the step of labeling the PBD with a reporter molecule preferably comprises the generation of a genetic fusion between the PBD and reporter molecule. Such fusions have been described in the art. For example, W099/25836 describes the generation effusion constructs between a polypeptide comprising zero, one, two or three AcmA LysM domains and a reporter enzyme (in that case either a-amylase or 6-lactamase). To determine whether a given polypeptide is a PBD of the present invention, a person skilled in the art will be able to apply standard recombinant DNA techniques to provide a fusion with a reporter polypeptide<br><br>
(enzyme, antigen or the like) which can subsequently be tested for cell binding activity.<br>
Preferred enzyme reporter molecules are those that allow for colorimetric or fluorescent detection of their activity. Many reporter enzyme systems are described in the art which make use of colorimetric or fluorimetric substrates, like horseradish peroxidase (HEP)/ 2,2'-a2dno-bis(3-ethylbenzthiazoline-6-sulphonic acid) (ABTS) ; alkaline phosphatase/4-nitro phenylphosphate or beta-galactosidase / 2-nitrophenyl-beta-D-galactopyranoside (2-NPG).<br>
In case the PBD is labelled with an antigen, determining the binding capacity of said PBD by detecting the absence or presence of reporter molecule associated with the micro-organism typically comprises the use of an antibody (e.g. a monoclonal murine antibody) specifically reactive with the antigen. The antigen-antibody complex can be detected using a secondary antibody (e.g. rabbit anti-mouse IgG antibody) carrying a detectable label in a so-called sandwich format. The secondary antibody is for instance provided with a reporter enzyme whose activity can be measured using a colorimetric substance mentioned above. It is also possible to label the PBD with a primary antibody as reporter molecule and detect the absence or presence of reporter molecule associated with the micro-organism using a secondary antibody carrjdng a detectable I     label (enzyme, fluorochrome).<br>
A Gram-positive microorganism for use in a binding assay can be viable or non-viable. Included are Gram-positive bacteria, such as a Bacillus ssp,. Streptococcus ssp., Mycobacterium ssp,. Listeria ssp. or a Clostridium ssp. The step of contacting the labeled PBD with a Gram-positive micro-organism to allow for binding of said means to said micro-organism can involve the resuspension of a pelleted culture of exponentially growing Gram-positive bacteria, like L. lactis, in a solution comprising the labelled PBD or a crude cell extract containing the labeled PBD. Said solution can also be a culture supernatant of a host cell expressing and secreting the labeled PBD. Following a certain period of incubation, for instance 1-120 min at 4-40 °C, like 1-30 min. at 10-40°C, the Gram-positive bacteria are pelleted and washed to remove any non-specifically bound reporter molecule. Thereafter, the amount of reporter molecule associated with the pelleted bacteria is determined.<br><br>
In a specific embodiment, a cell binding assay comprises the use of a-amylase from Bacillus licheniformis or E.coli TBM B-lactamase as reporter molecule as a fusion to PBD. Fusion proteins are recombinantly produced in a bacterial host cell which secretes the fusion protein in the culture supernatant. GEM particles are be used as Gram-positive microorganism in the binding assay. They can be prepared as described in WO 02/101026 and herein below. GEM particles loaded with both fusion proteins were spun down and washed twice with PBS. Enzyme activity of bound a-amylase -and B-lactamase PBD fusions are measured colorimetrically. a-Amylase activity is determined by incubating the loaded GEM particles in 1 ml amylose azure (Sigma) substrate solution (0.6 mg/ml amylose azure in 20 mM K2HP04/KH2P04-buffer, 50 mM NaCl, pH 7.5), at 37** C and 200 rpm. After 60 min, GEM particles and insoluble amylose azure were spun down, and the absorbance at 595 nm was measured. P-Lactamase activity was measured by adding 40 µ1 nitrocefin (CalBiochem) to GEM particles loaded with |3-lactamase PBD fusion in a final volume of 1 ml PBS. After 30 min the absorbance at 486 nm was measured.<br>
A polypeptide comprising a PBD and an ABD wherein the PBD comprises the three LysM domains of L, lactis cell wall hydrolase AcmA, also referred to as cA or protein anchor in WO 99/25836 and WO 02/101026, will herein be termed "Protan linker". As will be understood by the skilled person, the Protan linker may contain one or more amino add substitutions as compared to the naturally occurring AcmA LysM sequences, provided that the peptidoglycan binding capacity is maintained.<br>
The relative positions of the ABD and the PBD within the polypeptide can vary. However, it will be understood that allowing attachment of the polypeptide to the immunogenic carrier via the PBD on the one hand and binding of an antigen via the ABD on the other hand requires a certain degree of spacing between the domains to avoid or minimize mutual interference. In a preferred embodiment, a polypeptide of the invention comprises a PBD fused via a linker or spacer sequence to an ABD. Said linker or spacer can be a relatively short stretch of amino acids, e.g. 1-200, a medium size linker, e.g. 200-600 amino acids, or a larger linker, of more than 600 residues. For example, in one embodiment the N-terminal part of the polypeptide comprises a PBD<br><br>
which is fused via a linker to an ABD located in the C-terminal part. In another embodiment, the PBD constitutes the C-terminal part of the polypeptide and the ABD the N-terminal part. The ABD and/or PBD do not have to reside at the extreme ends of the polypeptides; one or more amino acid residues can be present at either end of the polypeptide which are neither part of the ABD nor of the PBD.<br>
A polypeptide of the invention comprises one or more antigen binding domains (ABDs). A multiplicity of ABDs within a single polypeptide allows the presentation of an antigen on an immunogenic carrier complex at a high density. In one embodiment, a polypeptide comprises two ABDs, capable of binding either the same or distinct antigens of interest. If multiple ABDs are present, it may be advantageous to place them adjacent to each other, e.g. with one or more amino acids in between to allow for an optimal binding of the multiple antigens to the polypeptide.<br>
An ABD present in a polypeptide of the invention is a proteinaceous moiety capable of binding to an antigen of interest. Any type of antigen can be bound to an antigen binding immunogenic carrier complex of the invention, provided that there is a suitable ABD available. The antigen of interest can be selected from the group consisting of polypeptides, carbohydrates, lipids, polynucleotides and pathogenic antigens, including inactivated viral particles and purified antigenic determinants. In one embodiment, an antigen of interest is an antigen which cannot be produced as a fusion to a PBD, like an antigen comprising at least one non-proteinaceous moiety.<br>
In one embodiment, the antigen of interest is a polynucleotide. Immunization with polynucleotides is a recent development in vaccine development. This technology has been referred to as genetic immunization or DNA immunization. The basis for this approach to immunization is that cells can take-up plasmid DNA and express the genes within the transfected cells. Thus, the vaccinated animal itself acts as a bioreactor to produce the vaccine. This makes the vaccine relatively inexpensive to produce. Some of the advantages of polynucleotide immunization is that it is extremely safe, induces a broad range of immune responses (cellular and humoral responses), long-lived immunity, and, most importantly, can induce immune responses in the presence of maternal antibodies. Although this is one of the most<br><br>
attractive developments in vaccine technology, there is a great need to develop better delivery systems to improve the transfection efficiency in vivo. In a specific aspect of the invention, an immunogenic carrier complex of the invention is used to deliver dsRNA, for example in an RNA interference (RNAi) - based therapy. Such therapy is particularly suitable to combat viral infections.<br>
It will be understood that the structural characteristics of an ABD will primarily depend on the antigen of interest. Known binding partners of an antigen of interest, or a part of such known binding partner, may be used as ABD. For example, the capacity of the polypeptide to bind a pathogen, e.g. virus or bacterium, may be conferred by using a normal host receptor for the pathogen. Pathogen host receptors are known in the art and their sequences have been determined and stored in publicly available databases. For example, ICAM-1 is a host receptor for human rhinovirus (HRV) and CD4 for HIV.<br>
In another aspect, the ABD comprises an antibody or functional fragment thereof, e.g. a Fab fragment, containing the antigen binding site, or other polypeptide, that binds to an antigen of interest. Many specific antibody (fragments) known in the art can be used in the present invention. For instance, an antibody (fragment) that binds to a conserved determinant on the viral surface, such as VP4 on pohovirus, or gpl20 on HIV, or HA on influenza virus. Industrial molecular affinity bodies (iMab®) are also suitably used as ABD according to the invention (see e.g. WO2004108749 from CatchMabs BV, The Netherlands). NanobodiesTM developed by Ablynx, Gent, Belgium may also be used.<br>
WO 02/101026 in the name of the applicant discloses the use of GEM particles as delivery vehicles for a polypeptide fusion between an AcmA-type protein anchor and a reactive group, like proteins, peptides and antibodies. Therein, the antibodies do not serve as carrier for an antigen but they are therapeutic substances themselves i.e. through specific interaction with endogenous antigens. Of course, for that purpose the antibody must not be "pre-loaded" with antigen, as is the case in the present invention. WO 02/101026 therefore does not disclose or suggest the loading of a antibodies attached to GEM-particles with antigen of interest.<br><br>
Antibody fragments and peptides specific for essentially any antigen, be it a peptide, sugar, lipid, nucleic acid or whole organism etc., can be selected by methods known in the art. Peptide libraries containing large amounts of randomly synthesized peptides which can be used in selecting a suitable binding partner for an antigen of interest are commercially available. For instance, New England Biolabs offers pre-made random peptide libraries, as well as the cloning vector M13KE for construction of custom libraries. The pre-made libraries consist of linear heptapeptide and dodecapeptide libraries, as well as a disulfide-constrained heptapeptide library. The randomized segment of the disulfide-constrained heptapeptide is flanked by a pair of cysteine residues, which are oxidized during phage assembly to a disulfide linkage, resulting in the displayed peptides being presented to the target as loops. All of the libraries have complexities in excess of 2 billion independent clones. The randomized peptide sequences in all three libraries are expressed at the N-terminus of the minor coat protein pIII, resulting in a valency of 5 copies of the displayed peptide per virion. All of the libraries contain a short linker sequence (Gly-Gly-Gly-Ser) between the displayed peptide and pIIL<br>
Of particular interest is the use of phage display technology. Many reviews on phage display are available, see for example Smith and Petrenko [1997] Chem. Rev. 97:391-410. Briefly, phage display technology is a selection technique in which a library of variants of a peptide or human single-chain Fv antibody is expressed on the outside of a phage virion, while the genetic material encoding each variant resides on the inside. This creates a physical linkage between each variant protein sequence and the DNA encoding it, which allows rapid partitioning based on binding affinity to a given target molecule (antibodies, enzymes, cell-surface receptors, etc.) by an in vitro selection process called panning. In its simplest form, panning is carried out by incubating a library of phage-displayed peptides with a plate (or bead) coated with the target (i.e. antigen of interest), washing away the unbound phage, and eluting the specifically bound phage. The eluted phage is then amplified and taken through additional binding/amplification cycles to enrich the pool in favour of binding sequences. After 3-4 rounds, individual clones are typically characterized by DNA sequencing and ELISA. The DNA contained within the desired phage encoding the particular peptide<br><br>
sequence can then be used as nucleic acid encoding an ABD in a nucleic acid construct encoding a polypeptide of the invention.<br>
There are several examples in the art of successful applications of phage display technology to identify peptides that bind selectively to micro-organisms. These teachings can be used to identify a peptide which can be used as antigen binding domain according to the present invention. For example, Knurr et al. (Appl. Environ Microbiol. 2003 Nov;69(ll):6841) describe the screening of phage display peptide libraries for 7-and 12-mer peptides that bind tightly to spores of B. subtilis and closely related species.<br>
Lindquist et al. (Microbiology. 2002 Feb;148(Pt 2):443-51) used a phage-displayed human single-chain Fv antibody library to select binding partners specific to components associated with the surface of Chlamydia trachomatis elementary bodies (EBs). While phage display has been used in the art primarily to select specific antibodies for purified components, these data show that this technology is suitable for selection of specific probes from complex antigens such as the surface of a microbial pathogen.<br>
As another useful example, JP2002284798 discloses peptides, obtained by phage display technology, that bind specifically to influenza virus/ hemaglutinin (HA). Also of particular interest for the present invention is a recent study by Kim et al. (J. Biochem Biophys Res Commun. 2005 Apr 1;329(1):312) which describes the screening of LPS-specific peptides from a phage display library using epoxy beads. LPS (lipopolysaccharide; endotoxin) is the major surface-exposed structural component of the outer membrane of Gram-negative bacteria. Its structure can be divided into three regions: (1) a phospholipids (lipid A) that is responsible for most of its biological activities, (2) a core oligosaccharide, and (3) an 0-specific chain, which is an antigenic polysaccharide composed of a chain of highly variable repeating oligosaccharide subunits.<br>
Kim et al., using biopanning on LPS-conjugated epoxy beads, repeatedly enriched clones encoding AWLPWAK and NLQEFLF. These peptides were found to interact with the polysaccharide moiety of LPS, which is highly variable among Gramnegative bacterial species. In addition, it was found that phages encoding these peptides<br><br>
preferentially bound to the LPS of Salmonella family, AWUPWAK-conjugated beads covdd be used to absorb Salmonella enteritidis from solution.<br>
Whereas the present invention allows the immobilization of unmodified antigens to an immunogenic carrier, it is not restricted to immodified antigens. In one embodiment of the invention, the ABD is capable of binding to an antigen of interest through a (chemical) modified or tagged version of the antigen of interest. For instance, an antigen can be provided with an affinity tag, which tag can be bound to the ABD. Example 2 herein below shows the binding of a biotin-tagged enzyme to GEM particles by virtue of a bifunctional Streptavidin-Pxrotan biEunctional linker.<br>
Also provided herein is a method for providing an antigen binding immunogenic carrier complex according to the invention. As is exemplified below, such a method comprises the steps of providing an immunogenic carrier, providing a polypeptide comprising a peptidoglycan binding domain (PBD) fused to an antigen binding domain (ABD), and allowing the attachment of said polypeptide to said immunogenic carrier to yield an antigen binding immunogenic carrier complex. As already indicated above, the use of phage display technology is particularly useful to obtain an ABD for a particular antigen of interest. Use can be made of commercial peptide or antibody fragment libraries.<br>
The bifunctional polypeptide comprising an ABD and an PBD can be readily made by constructing a genetic fusion of the respective domains, typically spaced by a linker sequence, and expressing the gene in a suitable (bacterial) host cell employing methods well known in the art. As is exemplified in the Examples below, the recombinantly obtained polypeptide can be simply contacted with the immunogenic carrier to allow binding of the bifunctional polypeptide to peptidoglycans at the surface of the particles resulting in the antigen binding immunogenic carrier complex. In a specific aspect, the step of providing an immunogenic carrier comprises the preparation of non-viable spherical peptidoglycan particles from a Gram-positive bacterium (GEM particles).<br>
The resulting carrier complex is contacted with one or more (modified) antigen(s) of interest to provide an antigen-loaded immunogenic carrier complex according to the invention wherein at least one antigen of interest is bound to an ABD. It is however<br><br>
also possible to reverse the order of binding, i.e. bind an antigen of interest to a polypeptide via its ABD prior to attaching the antigen-loaded polypeptide(s) via the PBD to the immunogenic carrier.<br>
In one embodiment, the invention provides a pharmacelltical composition comprising an antigen-loaded immunogenic carrier complex according to the invention. For example, it provides an immunogenic composition comprising an antigen-loaded immunogenic carrier complex. An immunogenic composition is capable of inducing an immune response in an organism. In one embodiment, the immunogenic composition is a vaccine composition capable of inducing a protective immune response in an animal. The immunogenic composition, e.g. the vaccine, may be delivered to mucosal surfaces instead of being injected since mucosal surface vaccines are easier and safer to administer. A L. lactis derived immunogenic carrier complex may be used for mucosal vaccination since this bacterium is of intestinal origin and no adverse immune reactions are generally expected from L. lactis. Also provided is the use of an antigen binding immunogenic carrier complex according to the invention for the deUvery of an (protective) antigen of interest to the immune system of a subject. The antigen binding immunogenic carrier complex comprises at least one bifunctional polypeptide attached to an immunogenic carrier, said pol5rpeptide comprising a peptidoglycan binding domain (PBD) through which the polypeptide is attached to said carrier, fused to an antigen binding domain (ABD) capable of binding said antigen of interest, wherein said PBD comprises an amino acid sequence selected from the group consisting of (i) a LysM domain, (ii) an amino acid sequence retrieved from a homology search in an amino acid sequence database with a LysM domain in the C-terminus of AcmA LysM domain and (iii) a sequence showing at least 70% sequence identity to an AcmA LysM domain, provided that the PBD is capable of attaching the substance to the cell wall of a Gram-positive microorganism. Also provided is the use of an antigen-loaded immunogenic carrier complex for the delivery of an (protective) antigen of interest to the immune system of a subject, preferably a human subject. delivery to the immune system preferably comprises antigen delivery to a mucosal site, such as intranasal delivery, e.g. by means of a spray, or oral, vaginal or rectal delivery.<br><br>
In a preferred embodiment, the invention provides a subunit vaccine based on an immunogenic carrier complex disclosed herein. Subunit vaccines are vaccines developed against individual viral or bacterial components, also referred to as "immunogenic determinants* that play a key role in eliciting protective immunity. In order to develop subunit vaccines, it is important to identify those components (often (glyco)proteins) of the pathogen that are important for inducing protection and eliminate the others. Some proteins, if included in the vaccine, may be immunosuppressive, whereas in other cases immune responses to some proteins may actually enhance disease. Combining genomics with our understanding of pathogenesis, it is possible to identify specific proteins from most pathogens that are critical in inducing the immune responses (see WO2004/102199), In addition to using a whole protein as a vaccine, it is possible to identify individual epitopes within these protective proteins and develop peptide vaccines. The potential advantages of using subunits as vaccines are the increased safety and less antigenic competition since only a few components are included in the vaccine, ability to target the vaccines to the site where immunity is required, and the ability to differentiate vaccinated animals from infected animals (marker vaccines). One of the disadvantages of subunit vaccines known in the art is that they generally require strong adjuvants and these adjuvants often induce tissue reactions. An immunogenic carzier complex as disclosed herein has built-in immunostimulatory properties and can efficiently deliver antigenic determinants as a particle to immunoreactive sites. Especially GEM particles are readily bound by and/or taken up by specific cells or tissues. The ability of GEMs to target macrophages or dendritic cells enhances their functional efficacy. In fact, it is now possible to mimic a pathogen with respect to its antigenic components while avoiding the imdesired effects of other components while maintaining the adjuvant properties. Of course, an immunogenic carrier can be provided with multiple polypeptides. Some of these polypeptides being hybrid antigen-Protan fusions (e.g. as described in WO 99/25836 and WO 02/101026) and some being bifunctional Protan fusions as disclosed herein, each comprising at least one ABD (see Fig. 1). The use of polypeptides with distinct ABDs allows the binding of distinct antigens of interest to a single immunogenic carrier. The use of multiple ABDs, being part of a single polypeptide or of distinct polypeptides, allows for the preparation of multiple epitope vaccines.<br><br>
In a further aspect, the invention relates to a diagnostic method comprising the use of an immunogenic carrier complex according to the invention. Also provided is a diagnostic kit comprising the use of an immunogenic carrier complex according to the invention. The ABD can be used to capture and immobilize an antigen of interest in a sample, e.g. a biological sample, onto the carrier complex. This loaded' carrier complex is suitably used to separate the antigen of interest from the remainder of the sample, for example by centrifugation. Subsequently, the amount of carrier-associated antigen of interest can be detected or quantitated. Thus, the immunogenic carrier complex, for instance a GEM particle, can be used as 'biological affinity bead' to isolate an antigen of interest, optionally followed by analysis of the antigen of interest.<br>
The invention is illustrated by the examples below.<br>
LEGENDS TO THE FIGURES<br>
Fig. 1. Schematic nresentafcLon of the vaccine deliverv technolosrv of the invention.<br>
Shown on the right is an immunogenic GEM particle loaded with several different antigens bound to the particle through the use of antigen-Protan fusion proteins and/or bifunctional polypeptides comprising an antigen binding domain (ABD) and a peptidoglycan binding domain (PBD), in this case Protan.<br>
Fig. 2. GEM-binding analysis of the ProtA-Protan bifunctional polvpentide.<br>
M        = Molecular weight marker prestained Precision Plus All Blue (BioRad) BG     = TCA precipitation of ProtA-Protan production medium before binding to<br>
GEM particles (200µl supernatant) AG      = TCA precipitation of production medium after binding to and removal of<br>
GEM particles (200 µl supernatant) G        = GEM particles loaded with ProtA-Protan (0,4 unit GEM with 800 µl<br>
supernatant)<br><br>
The arrow indicates the expected migration position of the ProtA-Protan fusion protein (50.7 kilodalton [kDa]).<br>
Fig. 3. Binding of mouse leG to GEM particles with attached ProtA-Protan.<br>
Upper part: Colorimetric values obtained due to the activity of alkaline phosphatase (AP), AP is conjugated to a secondary antibody that recognizes mouse IgG. Thus, if mouse IgG is boimd, AP activity can be detected.<br>
Lower part: description of the sample composition of the coinresponding samples in the upper part.<br>
Fig. 4, GEM-binding analysis of Strentavidin-Protan<br>
M	= Molecular weight marker prestained Precision Plus All Blue (BioRad)<br>
BG       = TCA precipitation of Streptavidin-Protan containing medium before binding<br>
to GEM particles (200 pi supernatant) AG       = TCA precipitation of medium after binding to and removing of GEM<br>
particles (200 µl supernatant)<br>
G	= GEM particles loaded with Streptavidin-Protan (0.4 unit GEM with 800 pi<br>
supernatant) The arrow indicates the expected migration position of the Streptavidin-Protan fusion protein (35.8 kDa).<br>
Fig. 5. Binding of Biotin-HRP to GEM particles with attached Streptavidin-Protan.<br>
Upper part: the graphic shows the colorimetric values that are obtained due to the<br>
activity of horse radish peroxidase (HRP). Biotin is conjugated to HRP. Thus, if biotin<br>
is bound, HRP activity will be measured.<br>
Lower part: describes the sample composition of the corresponding samples in the<br>
upper part of the figure.<br>
If no Streptavidin-Protan fusion is present (sample 1 and 3), only Protan is present on<br>
the GEM particles (sample 3), no Biotin-HRP (sample 2) or GEM particles (sample 5)<br>
are added: no activity is measured as expected. Activity is only measured in sample 4,<br><br>
which means that Streptavidin-Protan on the GEM partides binds the Biotin-HRP conjugate.In conclusion, the Streptavidin-Protan bifunctional linker can be attached to GEM particles and this complex can bind biotinylated compounds.<br>
EXPERIMENTAL SECTION<br>
EXAMPLE 1: Loading of antibodies on GEM particles<br>
This example describes the preparation of an antigen binding immunogenic carrier complex using GEM particles as immunogenic carrier and Protein A as antigen binding domain to attach antibodies as antigen of interest to the carrier complex.<br>
Protein A (ProtA) of Staphylococcus aureus is a 42 kDa protein that binds to the Fc region of IgG antibodies. It can be used to capture antibodies from a solution and immobilize them on a surface. Here we made a genetic fusion of ProtA with the peptidoglycan binding domain (cA) of the L. lactis cell wall hydrolase AcmA comprising three AcmA LysM domains, also herein referred to as protein anchor or 'Protan linker'. The resulting ProtA-Protan bifunctional linker was expressed and secreted by recombinant i. lactis. After removal of the recombinant producer cells, the bifunctional linker was attached to lactococcal GEM particles by the Protan moiety in the hybrid Unker. The ProtA moiety in the same hybrid linker was still able to bind IgG antibodies, thereby immobilizing these on the GEM particles.<br>
Bacterial strains and growth conditions<br>
The bacterial strains used in this study are listed in Table 1. L, lactis strains were grown in 30°C in M17 broth (DIFCO) as standing cultures or on M17 plates containing 1.5% agar. All media were supplemented with 0.5% glucose (w/v) (GM17) and, when necessary, supplemented with 5 |ig/ml chloramphenicol (SIGMA) for plasmid selection. Induction for PniM-driven gene expression was done with the culture supernatant of the nisin producing L. lactis strain NZ9700 as described previously (Kuipers et al.[1997] Trends Biotechnol. 15:135-140).<br><br><br><br>
General molecular biology<br>
Enzymes and buffers were purchased from New England Biolabs or Fermentas. Electro-transformation of L. lazctis was carried out as described previously (Holo and Nes [1995] Methods MoL Biol. 47:195-199) using a Bio-Rad Gene Pulser (Bio-Rad). Nucleotide sequence analyses were performed by BaseClear (Leiden, The Netherlands).<br>
Production of the fdsion construct containing ProtA-Protan<br>
ProtA (NCBI accession number BAB93949&gt;1: US 5,151,350; Uhlen et al. [1984] J, Biol. Chem. 259:1695-1702) from S. aureus contains five homologous IgG-Fc binding regions consisting of approximately 58 amino acids each. For the fusion of ProtA to Protan, only the Fc binding domains were amplified by PCR using primers SpA.fiv and SpA.rev (see Table 2).<br><br>
After amplification of the Fc-binding part from the S. aureus genome, the 710 bp PCR firagment was isolated from gel and digested with EspSl, restdting in NcoI and EcoRl sticky ends. The digested product was ligated into pPA3 which was digested with .BcoRI and iVcoL The ligation mixture was transferred by<br><br>
electroporation to L. lactis PAlOOl and resxilted in plasmid pPA217. Strain L. lactis (pPA217) produces secreted ProtA-Protan polypeptide.<br>
TCA precipitation of produced fusion proteins<br>
For detection of the amount of produced polypeptide in the cell free cultture medium, a TCA precipitation was performed. This was done by addition of 200 id 50% trichloroacetic acid (TCA) to 1 ml of cell free culture medium containing the Protan fusion protein. The mixture was placed on ice for 1 hour sifter vortexing. The precipitated protein was spun down in a centrifuge for 20 minutes at 14,000 rpm (4°C), was washed with acetone, dried in a vacuum exicator and resuspended in SDS sample buffer.<br>
GEM production and binding conditions<br>
Chemical pre-treatment of L. lactis NZ9000 for the production of GEM, was routinely done with hydrogen chloride (HCl, pH 1.0) as follows: cells of stationary phase cultures were collected by centrifugation and washed once with 0.5 volume of phosphate-buffered saline (PBS: 58 mM Na2HPO4, 17 mM Na2H2PO4, 68 mM NaCl, pH 7,2). Cells were resuspended in 1/5th volume of HCl, pHl.O solution and boiled for 30 min. Subsequently, the GEM particles formed in this way were washed three times with PBS, and resuspended in PBS until an average of 2.5x1010 GEM particles/ml as was determined with a Burker-Turk hemocyrtometer. GEM particles were either immediately used for binding experiments or stored in 1,0 ml aliquots at -80° C until use.<br>
In a typical binding experiment 2.5x109 GEM particles (1 unit) were incubated for 30 min at room temperature in an over-end rotator with 2 ml of cell-free culture medium containing a bifunctional polypeptide (Protan fusion protein). After binding, GEM particles were collected by centrifugation, washed twice with PBS and analyzed by SDS-PAGE or enzymatic activity.<br>
AP enzyme assay<br>
Enzyme activity of bound rabbit anti-mouse IgG Alkaline Phosphatase (AP) (Sigma) was measured colorimetrically. 0.5 unit GEM particles loaded with the fusion protein ProtA-Protan, mouse IgGl (kappa hght chain) (Sigma) (1ml 1:100 dilution in<br><br>
PBS) and rabbit anti-mouse IgG AP (Sigma) (1 ml 1:10,000 dilution in PBS) were spun down and washed twice with PBS. Alkaline phosphatase activity was determined by incubating the loaded GEM particles in 1 ml 4-nitro phenylphosphate (Sigma) (Img/ml in 50imM sodium carbonate buffer, pH 9.6,1 mM MgCh) at room temperature. After 5 min, the reaction was stopped by addition of 0.5 ml 2M NaOH. GEM particles were spxm down, and the absorbance of the supernatant at 405 nm was measured by a spectrophotometer (BioRad Smartspec 300).<br>
Attaching of the ProtA-Protan polypeptide to GEM particles<br>
Production of the polypeptide was induced as described above. After overnight induction, the expression of the protein was tested by performing a GEM-binding assay with 1 ml supernatant of the producing strain to 0.5 U of GEM. The results are given in Fig. 2. It is clear that most of the produced ProtA-Protan fusion peptide (lane BG) is specifically removed from the production medium (lane AG) and binds efficiently to the GEM particles (lane G). The smear in lane G is caused by the degraded L. lactis proteins present in the GEM particles.<br>
Mouse IgG binding to ProtA-Protan-GEM particles<br>
The antibody-binding activity of the ProtA-Protan polypeptide attached to GEM particles was tested using the reported enzyme alkaline phosphatase (AP) as described above. For this experiment different control groups were taken into account, as described in Fig. 3. The results clearly demonstrate that mouse-IgG binds to GEM particles that are activated with attached ProtA-Protan fusion protein (sample 1). No activity was detected when no ProtA-Protan was added to the GEM particles (sample 2) or when no secondary antibody with conjugated AP was added (sample 5), as expected. The anti-mouse secondary antibody that contains the conjugated AP is also an IgG antibody and binds as well to the ProtA-Protan-GEM complex even in the absence of mouse IgG (sample 4). In the absence of GEM particles some activity is measured (sample 3), most likely due to some aggregation of the ProtA-Protan fusion that is spun down during the procedure or due to some aspecific binding of the protein to the plastic reaction tube. In conclusion, the ProtA-Protan bifunctional linker can be attached to GEM particles and this complex can bind IgG antibodies.<br><br>
In conclusion, the ProtA-Protan bifunctional polypeptide can be attached to GEM particles to yield an immunogenic carrier complex and this complex can be loaded with IgG antibodies.<br>
EXAMPLE 2 : Immobilization of biotinylated compounds on GEM particles<br>
Streptavidin of Streptomyces avidinii is a 15 kDa protein that is functional as a tetramer and binds biotin. It can be used as antigen binding domain (ABD) to capture biotinylated substances from a solution and immobilize them on a surface. Here we made a genetic fusion of Streptavidin with the Protan linker described in Example 1. The resulting Streptavidin-Protan bifunctional polypeptide was expressed and secreted by recombinant L. lactis. After removal of the recombinant producer cells, the bifunctional linker was attached to lactococcal GEM particles by the peptidoglycan binding domain (PBD) of the Protan moiety of the polypeptide. The ABD in the same pol3rpeptide was still functional and was used to bind to and immobilize biotinylated horse radish peroxidase as antigen of interest on the GEM particles.<br>
Bacterial strain, plasmids and procedures for growth conditions, general molecular biology techniques, GEM production and binding conditions and TCA precipitation of produced fusion proteins were the same as in Example 1.<br>
HRP enzyme assay<br>
Enzyme activity of bound Biotin-Horseradish Peroxidase (HRP) (Molecular Probes) was measured colorimetrically. 0,5 U GEM particles loaded with the fusion protein Streptavidin-Protan and Biotin-HRP (1 ml 1:2000 Biotin-HRP (1 mg/ml) in PBS) were spun down and washed twice with PBS, Horse radish peroxidase activity was determined by incubating the loaded GEM particles in 1 ml ABTS (Fluka) (10 mg in 100 ml 0.05 M phosphate-citrate buffer pH 5.0) and 1 µ1 H2O2 (100%) at room temperature. After 5 min, the reaction was stopped by addition of 100 µl 10% SDS. GEM particles were spun down, and the absorbance of the supernatant at 405 nm was measured by a spectrophotometer (BioRad Smartspec 300).<br><br>
Production of the fusion construct containing Streptavidin-Protan<br>
Only the core of Streptavidin was used as ABD for the production of the bifunctional polypeptide (Streptavidin-Protan). This core is the biotin binding unit of Streptavidin (NCBI accession number CAA00084'). containing amino acids A37-S163-(Argarana et al [1986] Nucleic Acid Research 14:1871-1882, Pahler et al [1987] J. Biol. Chem. 262: 13933-13937). For the fusion of Streptavidin core to the Protan moiety comprising the PBD, 8 primers were designed. These primers could be amplified to each other, first Strep L. fw until Strep4.rev and Strep5.fw until Strep8.rev in two different PCR reactions. The two PCR products were mixed and amplified with the two exterior primers Strep l.fw and Strep8.rev in which a streptavidin-core gene-fragment of 397 bp was produced that was optimized for L. lactis codon usage. The primers used for the production of this gene fragment are described in Table 3.<br>
To be able to screen the PCR fi:agment for containing the correct DNA sequence, the Zero Blunt® TOPO® Cloning Kit (Invitrogen) was used. The Zero Blunts TOPO® plasmid containing the correct streptavidin-core gene fragment was digested with EcdR1 and iVcoL This digestion product was ligated into pPA3 which was also digested with EcoR1 and JVcoI. The ligation mixture was transferred by electroporation to L. lactis PAlOOl and resulted in plasmid pPA218. Strain L. lactis (pPA218) produced and secreted Streptavidin-Protan polypeptide in the culture medium.<br>
Attaching the polypeptide to immunogenic carrier<br>
Production of the fusion protein was induced as described above. After overnight induction, the expression of the protein was tested by performing a GEM-binding assay with 1 ml supernatant of the producing strain to 0.5 unit of GEM particles (Fig. 4). It is clear that most of the produced Streptavidin-Protan fusion Gane BG) is specifically removed from the production medium (lane AG) and binds efficiently to the GEM particles (lane G). The smear in lane G is caused by the degraded L. lactis proteins present in the GEM particles.<br><br>
'able 3: Primers used for production of streptavidin core gene. The nucleotide tretches which are either in italics, underlined, double underlined, in lower case atters, in italics and underlined, in lower case letters and underline or in lower case 3tters and in italics and underlined can anneal to each other.<br><br><br>
Biotin-HRP binding to Streptavidin-Protan-GEM particles<br>
The biotin binding activity of the fusion protein bound to GEM was tested as described herein below using HRP as reported enzyme. For this experiment different control groups were taken into account, as described in Fig. 5. If no Streptavidin-Protan fusion is present (sample 1 and 3), only Protan is present on the GEM particles (sample 3), no Biotin-HRP (sample 2) or GEM particles (sample 5) are added, no activity is measured as expected. Activity is only measured in sample 4 which means that Streptavidin-Protan on the GEM particles binds the Biotin-HRP conjugate.<br>
In conclusion, the Streptavidin-Protan bifunctional polypeptide can be attached to immunogenic GEM particles and this complex can bind a biotin-modified antigen of interest.<br>
EXAMPLE 3 : Immobilization of inactivated whole poliovirus on GEM particles using bifunctional Protan linkers<br>
Phage display (Smith and Petrenko [1997] Chem. Rev. 97:391-410) has emerged as a powerful technique for the selection of specific binding peptides (Sidhu et aL [2000] Meth. Enzymol. 328:333-344; Gwrirla et al [1990] Proc. Natl. Acad. Sci. USA. 87:6378-6382). A DNA sequence encoding the peptide is translationaUy fused to DNA encoding the gene 3 minor coat protein, yielding display of the peptide on the surface of the phage. In this way a physical linkage was established between the displayed peptide and the DNA encoding this peptide. Phage peptide libraries can be used to efficiently search for specific binders out of a pool of variants by selection on a specific target, a process called panning. Selected phage can subsequently be amplified in Escherichia coli and subjected to additional rounds of panning to enrich peptides that specifically bind to the target.<br>
Random peptide libraries has been used in various applications such as binding to proteins (Sidhu et aL [2000] Meth. enzymol. 328:333-344), polysaccharides Kim et aL [2005] Biochem. Biophys, Res. Commun. 329:312-317), bacterial spores Knurr et aL [2003] Appl. Environ. Microbiol. 69:6841-6847), whole cells (Brown [2000]<br><br>
Curr, Opinion. Chem. Biol. 4:16-21), and inorganic materials (Whaley et al [2000] Nature 405:665-668).<br>
In the current application phage display can be used for the selection of specific binding peptides that are subsequently used for the construction of bifunctional Protan linkers. Application of the bifunctional polypeptides like Protan linkers allows the non-covalent attachment of a compound of interest, ie. proteins, polysaccharides, bacteria, viruses, or fungi to GEM particles. Peptides are advantageous over binding proteins in that they are less immunogenic, and easy to produce. In this Example we describe the construction of a phagemid-based peptide library which was used for the selection of specific binding peptides targeted to inactivated whole poliovirus. Selected peptides that specifically bind to whole poliovirus were genetically fused to Protan and the bifunctional Protan Linkers were attached to lactococcal GEM particles. This allows the non-covalent coupling of inactivated whole poliovirus. The resulting antigen -loaded immunogenic carrier complex of GEM particle with inactivated whole poliovirus is directly applicable in vaccines.<br>
Construction of peptide phage display vector<br>
The phagemid pPEP is constructed from pCANTAB 5EST (Amersham Pharmacia) for the display of short peptide sequences. The display of peptides requires the in frame fusion of peptides to the minor coat protein 3 (g3p) of phage M13. Therefore, superfluous nucleotide sequences in pCANTAB 5EST between the Hindlll and BaniHI recognition sequence are removed and replaced by a PCR-assembled fragment encoding only the relevant sequence elements. In addition, Kpnl and Bpil recognition sequences are introduced to allow cloning of peptide sequences at the 5'-endof gene 3. Furthermore, in order to improve the target accessibilily of a displayed peptide a small spacer sequence of three glycine residues is included between the cloned peptide and the minor coat protein.<br>
The construction of the peptide phage display vector involves a number of polymerase chain reaction (PCR) steps. First a DNA fragment from the Hindlll recognition sequence until the start of gene 3 is synthesized by two successive overlap PCRs. A temporary assembly PCR product is produced from oligonucleotides Cb lF2.fw, 5'-ggagccttttttttggagattttcaacgtgaaaaaattattattc gcaattcctttagtggta; CblF.3.fw, 5'-gcaattcctttagtggtacctttctatgcggcccagccggccat ggcccagggcgctgggaga and<br><br>
CblF4.rev, 5'-ttcaacagtaccgccaccccgtcttctcccagcgccctgggc. This temporary amplicon is purified and used in a second overlap PCR as template together with the outside primers CblFl.fw, 5'-atgattacgccaagcttt-ggagccttttttttggag and CblF5,rev 5'-aggttttgctaaacaactttcaacagtaccgccacc, shielding the final assembled PCR fragment. A second DNA fragment containing the first 617 nucleotides of gene 3 is produced by PCR using the oligonucleotides Cb2.fw, 5'-actgttgaaagttgtttagcaaaacct and Cb2.rev 5'-agacgattgccttgatattcacaaac. In the final overlap PCR the latter amplicon is combined with the first assembly PCR product and the outside primers CblFl.fw and Cb2.rev yielding a 711 bp PCR firagment. This amplification product is digested with Hindlll and BamHI and ligated into the same sites of pCANTAB 5EST resulting in phagemid pPEP.<br>
Construction of random peptide libraries<br>
Phagemid pPEP contains Kpnl and BpiI recognition sites at the 5' end of gene 3 for display of random peptides as N-terminal g3p fusions. Since pPEP is a phagemid, peptides are displayed in a monovalent format i.e. only one or two copies of g3p on the surface of each phage particle will be fused to the cloned peptide.<br>
A library of oligonucleotides encoding 12-amino acid linear random peptides is constructed according to Noren and Noren [2001] (Methods 23:169-178). Briefly, a 92 nucleotide library oligonucleotide, PEP12Lib.rev, is designed with the sequence 5'-accgaagaccccacc(BNN)i2Ctgggccatggccggctgggccgcatagaaaggtacccggg (B= C or 6 or T). The universal extension primer PEPext.fw 5'-catgcccgggtacctttctatgcgg is annealed and extended in a Klenow reaction. The resulting double stranded library oligonucleotide is purified, digested with Kpnl and Bpil, and ligated into pPEP that had been digested with the same enzymes yielding pPEP12. The Ligation mixture is transferred to E. coli XLl-blue or TGI cells (Stratagene) by electroporation until wlO® independent clones are obtained. To produce phagemid particles E. coli TGI or XLl blue cells containing pPEP12 are infected with a 30-field excess M13K07 helper phage. From the infected culture phagemid particles are purified by PEG precipitation.<br>
Biopanning<br>
Inactivated whole poliovirus particles are used for affinity selection of specific binding peptides. Poliovirus is captured on ELISA plates coated with rabbit anti-<br><br>
poliovirus IgG (0.5-1 µg/ml). Alternatively, poliovirus is displayed on GEM particles loaded with anti-poliovirus IgG bound to ProtA-Protan fusions (Example 1), Approximately 1011 phagemid particles in phosphate buffered saline (PBS) + 0.1% Tween 20 (PBS-T) from the dodecapeptide library are allowed to react in wells, or with «109 GEMs with the inactivated poliovirus for 1 h at room temperature. After incubation, unbound phages are removed. The GEMs or wells are washed ten times with PBS-T and bound phages are eluted with 0.2 M glycine/HCl (pH 2.2). The eluted phage suspension is neutralized with 2 M Tris base. The eluted phages are used to infect E, coli TGI or XLl blue cells. A total of 6 cycles of selection are performed, after which individual phage clones are isolated for further analysis.<br>
Binding analysis<br>
To evaluate binding of peptides to the poliovirus multi-well plates are coated with anti-poliovirus IgG (0.5-1 µl/ml). After washing with PBS-T, the wells are blocked with 1% BSA in PBS-T for 1 h at room temperature. Inactivated whole poliovirus is added in PBS-T + 1% BSA, and incubated for 1 h at room temperature. Selected peptide-phages (1010 cfu/ml) in PBS-T are added to the wells. After 1 h room temperature the plates are washed three times with PBS-T. Peptide-phages bound to pohovirus are detected with HRP-conjugated anti-M13 antibody (Pharmacia) using ABTS [2,2'-azinobi8-(3-ethylbenzothiazoline-6-sulfonic acid)] as a substrate. The absorbance is measured after a suitable time period at 410 nm. Two peptide-phages that show the best binding to the poliovirus are selected for further characterization by sequence analysis. The two phagemids are designated pPEP-PVl and pPEP-PV2.<br>
Construction of PEP-Protan fusion proteins<br>
Based on the nucleotide sequence of the binding peptides in pPEP-PVl and pPEP-PV2, two complementary oligonucleotides corresponding to binding peptides PVl and PV2 are designed and produced with Bsal and Bpil overhanging 5'-ends. Equal molar concentrations of both oligonucleotides are annealed in a total volume of 100 µl 10 mM Tris-HCl (pH 8.0). The mixture is heated to 94° C and slowly cooled to room temperature in a thermal cycler. The annealed oligonucleotides are ligated into pPA224 digested with Bsal and Bpil. Plasmid pPA224 is a derivative of pPA3 (Steen et al [2003] J. Biol. Chem. 278:23874-23881), which lacks the c-myc sequence and has<br><br>
a modified multiple cloning site between the usp45 signal sequence and the Protan sequence. After ligation the mixtures are transferred to L. lactis PAlOOl by electroporation. The result are plasmids pPA224-PVl and pPA224-PV2 in which PVl and PV2, respectively, are transcriptionally fused to the 5'-end of the Protan sequence.<br>
L lactis PA1001(pPA224-PVl) and L. lactis PA1001(pPA224.PV2) secrete PVl- Protan and PV2- Protan bifunctional linkers, respectively, into the growth medium. The producer cells are removed from the production medium by microfiltration and/or centrifugation.<br>
Binding of inactivated poliovirus to antigen-binding GEM particles<br>
The bifunctional polypeptides PVl- Protan and PV2- Protan attaches efficiently to lactococcal GEM particles, either each alone or in combination. The antigen-binding immunogenic carrier complexes PVl-Protan-GEM, PV2-Protan-GEM and PVl+PV2-Protan-GEM thus obtained are mixed with a suspension containing inactivated poliovirus particles. In all cases the poliovirus particles efficiently bind to the GEM-bifunctional Protan complexes.<br>
This Example illustrates that specific antigen binding domains can be selected using phage display, even for an entire viral particle and that this binding domain can be used in a bifunctional polypeptide to immobilize the entire virus on an immunogenic carrier.<br>
Furthermore, the examples demonstrate that either known antigen binding domains or newly selected binding domains from a random peptide library can be used in Protan fusions as bifunctional Protan linkers to immobilize a desired compound (e.g. antigen) on GEM particles, without the need to modify the compound of interest before binding to an immunogenic carrier.<br><br><br><br><br><br><br><br><br><br><br><br><br>
1.	An antigen-loaded immunogenic carrier complex, comprising at least one<br>
bifunctional polypeptide attached to an immunogenic carrier, said polypeptide<br>
comprising a peptidoglycan binding domain (PBD) through which the polypeptide<br>
is attached to said carrier, fused to an antigen binding domain (ABD) capable of<br>
binding non-covalently an antigen of interest,<br>
wherein said PBD comprises an amino acid sequence selected from the group consisting of (i) a LysM domain, (ii) an amino acid sequence retrieved from a homology search in an amino acid sequence databane with a LysM domain in the C-tarminus of Loctococcus loctis cell wall hydrolase AcmA (said domain herein also referred to as AcmA LysM domain) and (iii) a sequence showing at least 70% sequence identity to an AcmA LysM domain,<br>
provided that the PBD is capable of attaching to the cell wall of a Gram-positive microorganism;<br>
and wherein at least one antigen of interest is bound to said ABD.<br>
2.	A complex according to claim 1, wherein at least two biEunctional polypeptides are attached to said carrier, each having a distinct ABD and distinct antigen of interest.<br>
3.	A complex according to claim 1 or 2, wherein said immunogenic carrier comprises a non-viable spherical peptidoglycan particle obtained from a Gram-positive bacterium (GEM particle).<br>
4.	A complex according to any of claim 1 to 3, wherein said bacterium is a non-pathogenic bacterium, preferably a food-grade bacterium.<br>
5.	A complex according to any of claim 1 to 4, wherein said bacterium is selected from the group consisting of a Lactococcus, a Lactobcuiillus, a Bacillus and a Mycobacterium asp.<br>
6.	A complex according to any one of claims 1 to 5, wherein said<br>
peptidoglycan binding domain (PBD) comprises the C-terminal peptidoglycan<br>
binding domain of the Lactococcus lactis cell wall hydrolase AcmA.<br><br>
7.	A complex according to any of one of the above claims, wherein said ABD is capable of binding an antigen selected from the group consisting of polypeptides, carbohydrates, lipids, polynucleotides and pathogenic antigens, including inactivated viral partides and purified antigenic determinants.<br>
8.	A method for providing an antigen-loaded immunogenic carrier complex according to any of the above claims, comprising the steps of:<br><br>
-	providing an immunogenic carrier;<br>
-	providing a bifunctional polypeptide comprising a peptidoglycan binding domain (PBD) fused to an antigen binding domain (ABD) allowing attachment of said polypeptide to said immunogenic carrier; and<br>
-	contacting said immimogenic carrier and said polypeptide;<br>
-	contacting said polypeptide with an antigen of interest.<br><br>
9.	A method according to claim 8, wherein providing an immunogenic carrier comprises preparing non-viable spherical peptidoglycan particles faom a Gram-positive bacterium.<br>
10.	A method according to claim 8 or 9, wherein providing said bifunctional polypeptide comprises selecting an antigen binding domain from a random peptide or antibody library, preferably using phage display technology.<br>
11.	A method according to any one of claims 8 to 10, comprising producing the bifunctional polypeptide in a host cell by recombinant expression of a nucleic acid construct encoding said polypeptide.<br>
12.	A method according to claim 11, wherein said host cell secretes the polypeptide in the culture medium.<br>
13.	A pharmaceutical composition comprising an antigen-loaded immunogenic<br>
carrier complex according to any one of claims 1 to 7.<br>
14      An immunogenic composition comprising an antigen-loaded immunogenic carrier complex according to any one of claims 1 to 7.<br><br>
15.	Use of an antigen binding immunogenic carrier complex for the delivery of<br>
an antigen of interest to the immune system of a subject, said complex<br>
comprising:<br>
at least one bifunctional polypeptide attached to an immunogenic carrier, said polypeptide comprising a peptidoglycan binding domain (PBD) through which the polypeptide is attached to said carrier, fused to an antigen binding domain (ABD) capable of binding non-covalently said antigen of interest,<br>
wherein said PBD comprises an amino acid sequence selected from the group consisting of (i) aLysM domain, (u) an amino acid sequence retrieved from a homology search in an amino) acid sequence database with a LysM domain in the C-terminus of Lactocoocus lactis cell wall hydrolase AcmA (said domain herein also referred to as AcmA LysM domain) and (iii) a sequence showing at least 70% sequence identity to an AcmA LysM domain,<br>
provided that the PBD is capable of attaching the substance to the cell wall of a Gram-positive microorganism.<br>
16.	Use according to claim 15, wherein delivery to the immune system<br>
comprises delivery to mucosal sites, such as intranasal, oral, vaginal or rectal<br>
delivery.<br>
17.   Use according to claim 15 or 1,6, wherein the complex is loaded with said antigen of interest.<br><br></td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
		<br>
		<div class="pull-left">
			<a href="264463-double-fed-synchronous-generator-motor.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="264465-crosslaminate-of-oriented-films-and-methods-and-apparatus-for-manufacturing-same.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>264464</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>854/CHENP/2008</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>01/2015</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>02-Jan-2015</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>30-Dec-2014</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>20-Feb-2008</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>APPLIED NANOSYSTEMS B.V</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td>POSTBUS 100, 9700 AC GRONINGEN, THE NETHERLANDS.</td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>LEENHOUTS, CORNELIS, JOHANNES</td>
											<td>TUSSENKOELEN 6, NL-9753 KX HAREN, THE NETHERLANDS.</td>
										</tr>
										<tr>
											<td>2</td>
											<td>VAN ROSMALEN, MAARTEN, LEONARDUS</td>
											<td>NIEK ENGELSCHMANSTRAAT 25, NL-9745 DG GRONINGEN, THE NETHERLANDS.</td>
										</tr>
										<tr>
											<td>3</td>
											<td>BOSMA, TJIBBE</td>
											<td>DE BUORREN 100, NL-8408 HP LIPPENHUIZEN, THE NETHERLANDS.</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>C07K14/335</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>PCT/NL2006/000382</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td>2006-07-20</td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td>05076680.7</td>
									<td>2005-07-20</td>
								    <td>EUROPEAN UNION</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/264464-method-for-preparing-an-antigen-loaded-immunogenic-carrier-complex by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 01:31:11 GMT -->
</html>
