<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/264466-polypeptides-having-lipase-activity-and-polynucleotides-encoding-same by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 01:31:42 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 264466:POLYPEPTIDES HAVING LIPASE ACTIVITY AND POLYNUCLEOTIDES ENCODING SAME</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">POLYPEPTIDES HAVING LIPASE ACTIVITY AND POLYNUCLEOTIDES ENCODING SAME</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>The present invention relates to polypeptide having lipase activity and which further has a RP of at least 0 8 and a BR of at least 11 at the test conditions given in the specification</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td> <br><br>
POLYPEPTIDES HAVING LIPASE ACTIVITY AND POLYNUCLEOTIDES ENCODING SAME<br>
FIELD OF THE INVENTION<br>
The present invention relates to polypeptides having lipase activity and polynucleotides encoding same.<br>
BACKGROUND OF THE INVENTION<br>
Lipases are useful, e.g., as detergent enzymes to remove lipid or fatty stains from clothes and other textiles, as additives to dough for bread and other baked products. Thus, a lipase derived from Thermomyces lanuginosus (synonym Humicola lanuginosa, EP 258 068 and EP 305 216) is sold for detergent use under the tradename Lipolase ® (product of Novo Nordisk A/S). WO 0060063 describes variants of the T. lanuginosus lipase with a particularly good first-wash performance in a detergent solution. WO 9704079, WO 9707202 and WO 0032758 also disclose variants of the T. lanuginosus lipase. WO 02062973 discloses T. lanuginosus lipase with a C-terminal extension with reduced tendency to form odor.<br>
SUMMARY OF THE INVENTION<br>
The present invention relates to isolated polypeptides having lipase activity selected from the group consisting of lipases having an Average Relative Performance (RP) of at least 0.8 and a Benefit-Risk (BR) of at least 1.1 at the test conditions given in the specification.<br>
The present invention also relates to lipase variants with reduced potential for odor generation and to a method of preparing them. It particularly relates to variants of the Thermomyces lanuginosus lipase having a preference for long fatty acid chains while at the same time having a good relative performance.<br>
In a further aspect the invention relates to an isolated polynucleotide comprising a nucleotide sequence which encodes the polypeptide, a nucleic acid construct comprising the polynucleotide, a recombinant expression vector comprising the nucleic acid construct and a recombinant host cell comprising the nucleic acid construct.<br>
The present invention also relates to a method for producing the lipases of the Invention.<br>
SEQUENCE LISTING<br>
SEQ ID NO: 1 shows the DNA sequence encoding lipase from Thermomyces lanoginosus.<br><br>
SEQ ID NO: 2 shows the amino acid sequence of a lipase from Thermomyces<br>
lanoginosus.<br>
SEQ ID NO: 3 - SEQ ID NO: 16 show sequences for alignment in figure 1<br>
SEQ ID NO: 17 and SEQ ID NO: 18 show sequences used for alignment example.<br>
DEFINITIONS<br>
Lipase activity: The term "lipase activity" is defined herein as a carboxylic ester hydrolase activity which catalyzes the hydrolysis of triacylglycerol under the formation of diacylglycerol and a carboxylate. For purposes of the present invention, lipase activity Is determined according to the procedure described in "Lipase activity" in "Materials and Methods". One unit of lipase activity is defined as the amount of enzyme capable of releasing 1.0 micro mole of butyric acid per minute at 30°C, pH 7.<br>
The polypeptides of the present invention have at least 70%, such at least 75% or 80% or 85% or 90%, more preferably at least 95%, even more preferably 96% or 97%, most preferably 98% or 99%, and even most preferably at least 100% of the lipase activity measured as Relative Performance of the polypeptide consisting of the amino acid sequence shown as the mature polypeptide of SEQ ID N0:2, with the substitutions T231R + N233R.<br>
Isolated polypeptide: The term "isolated polypeptide" as used herein refers to a polypeptide which is at least 20% pure, preferably at least 40% pure, more preferably at least 60% pure, even more preferably at least 80% pure, most preferably at least 90% pure, and even most preferably at least 95% pure, as determined by SDS-PAGE.<br>
Substantially pure polypeptide: The term "substantially pure polypeptide" denotes herein a polypeptide preparation which contains at most 10%, preferably at most 8%, more preferably at most 6%, more preferably at most 5%, more preferably at most 4%, at most 3%, even more preferably at most 2%, most preferably at most 1%, and even most preferably at most 0.5% by weight of other polypeptide material with which it is natively associated. It is, therefore, preferred that the substantially pure polypeptide is at least 92% pure, preferably at least 94% pure, more preferably at least 95% pure, more preferably at least 96% pure, more preferably at least 96% pure, more preferably at least 97% pure, more preferably at least 98% pure, even more preferably at least 99%, most preferably at least 99.5% pure, and even most preferably 100% pure by weight of the total polypeptide material present in the preparation.<br>
The polypeptides of the present invention are preferably in a substantially pure form. In particular, it is preferred that the polypeptides are in "essentially pure form", i.e., that the<br><br>
polypeptide preparation is essentially free of other polypeptide material with which it is natively associated. This can be accomplished, for example, by preparing the polypeptide by means of well-known recombinant methods or by classical purification methods.<br>
Herein, the term "substantially pure polypeptide" is synonymous with the terms "isolated polypeptide" and "polypeptide in isolated form."<br>
identity: The relatedness between two amino acid sequences or between two nucleotide sequences is described by the parameter "identity".<br>
For purposes of the present invention, the alignment of two amino acid sequences is determined by using the Needle program from the EMBOSS   (http://emboss.org) version 2.8.0. The Needle program implements the global alignment algorithm described in Needleman, S. B. and Wunsch, C. D. (1970) J. Mol. Biol. 48, 443-453. The substitution matrix used is BLOSUM62, gap opening penalty is 10, and gap extension penalty is 0.5.<br>
The degree of identity between an amino acid sequence' of the present invention ("invention sequence"; e.g. amino acids 1 to 269 of SEQ ID N0:2) and a different amino acid sequence ("foreign sequence") is calculated as the number of exact matches in an alignment of the two sequences, divided by the length of the "invention sequence" or the length of the "foreign sequence", whichever is the shortest. The result is expressed in percent identity.<br>
An exact match occurs when the "invention sequence" and the "foreign sequence" have identical amino acid residues in the same positions of the overlap (in the alignment example below this is represented by "|"). The length of a sequence is the number of amino acid residues in the sequence (e.g. the length of SEQ ID N0:2 is 269).<br>
In the alignment example below, the overlap is the amino acid sequence "HTWGER-NL" of Sequence A; or the amino acid sequence "HGWGEDANL" of Sequence B. In the example a gap is indicated by a "-". Alignment example<br>
Sequence A:  ACMSHTWGER-NL<br>
  Sequence B:     HGWGEDANLAMNPS<br>
Polypeptide Fragment: The term "polypeptide fragment" is defined herein as a polypeptide having one or more amino acids deleted from the amino and/or carboxyl terminus of SEQ ID NO: 2 or a homologous sequence thereof, wherein the fragment has lipase activity.<br>
Subsequence: The term "subsequence" is defined herein as a nucleotide sequence having one or more nucleotides deleted from the 5' and/or 3" end of SEQ ID NO: 1 or a<br><br>
homologous sequence thereof, wherein the subsequence encodes a polypeptide fragment having lipase activity.<br>
Allelic variant: The term "allelic variant" denotes herein any of two or more alternative forms of a gene occupying the same chromosomal locus. Allelic variation arises naturally through mutation, and may result in polymorphism within populations. Gene mutations can be silent (no change in the encoded polypeptide) or may encode polypeptides having altered amino acid sequences. An allelic variant of a polypeptide is a polypeptide encoded by an allelic variant of a gene.<br>
Substantially pure polynucleotide: The term "substantially pure polynucleotide" as used herein refers to a polynucleotide preparation free of other extraneous or unwanted nucleotides and in a form suitable for use within genetically engineered protein production systems. Thus, a substantially pure polynucleotide contains at most 10%, preferably at most 8%, more preferably at most 6%, more preferably at most 5%, more preferably at most 4%, more preferably at most 3%, even more preferably at most 2%, most preferably at most 1%, and even most preferably at most 0.5% by weight of other polynucleotide material with which it is natively associated. A substantially pure polynucleotide may, however, include naturally occurring 5' and 3' untranslated regions, such as promoters and terminators. It is preferred that the substantially pure polynucleotide is at least 90% pure, preferably at least 92% pure, more preferably at least 94% pure, more preferably at least 95% pure, more preferably at least 96% pure, more preferably at least 97% pure, even more preferably at least 98% pure, most preferably at least 99%, and even most preferably at least 99.5% pure by weight. The polynucleotides of the present invention are preferably in a substantially pure form. In particular, it is preferred that the polynucleotides disclosed herein are in "essentially pure form", i.e., that the polynucleotide preparation is essentially free of other polynucleotide material with which it is natively associated. Herein, the term "substantially pure polynucleotide" is synonymous with the terms "isolated polynucleotide" and "polynucleotide in isolated form." The polynucleotides may be of genomic, cDNA, RNA, semisynthetic, synthetic origin, or any combinations thereof.<br>
cDNA: The term "cDNA" is defined herein as a DNA molecule which can be prepared by reverse transcription from a mature, spliced, mRNA molecule obtained from a eukaryotic cell. cDNA lacks intron sequences that are usually present in the corresponding genomic DNA. The initial, primary RNA transcript is a precursor to mRNA which is processed through a series of steps before appearing as mature spliced mRNA. These steps include the removal of intron sequences by a process called splicing. cDNA derived from mRNA lacks, therefore, any intron<br><br>
sequences.<br>
Nucleic acid construct: The term "nucleic acid construct" as used herein refers to a nucleic acid molecule, either single- or double-stranded, which is isolated from a naturally occurring gene or which is modified to contain segments of nucleic acids in a manner that would not otherwise exist in nature. The term nucleic acid construct is synonymous with the term "expression cassette" when the nucleic acid construct contains the control sequences required for expression of a coding sequence of the present invention.<br>
Control sequence: The term "control sequences" is defined herein to include all components, which are necessary or advantageous for the expression of a polynucleotide encoding a polypeptide of the present invention. Each control sequence may be native or foreign to the nucleotide sequence encoding the polypeptide. Such control sequences include, but are not limited to, a leader, polyadenylation sequence, propeptide sequence, promoter, signal peptide sequence, and transcription terminator. At a minimum, the control sequences include a promoter, and transcriptional and translations! stop signals. The control sequences may be provided with linkers for the purpose of introducing specific restriction sites facilitating ligation of the control sequences with the coding region of the nucleotide sequence encoding a polypeptide.<br>
Operably linked: The term "operably linked" denotes herein a configuration in which a control sequence is placed at an appropriate position relative to the coding sequence of the polynucleotide sequence such that the control sequence directs the expression of the coding sequence of a polypeptide.<br>
Coding sequence: When used herein the term "coding sequence" means a nucleotide sequence, which directly specifies the amino acid sequence of its protein product. The boundaries of the coding sequence are generally determined by an open reading frame, which usually begins with the ATG start codon or alternative start codons such as GTG and TTG. The coding sequence may a DNA, cDNA, or recombinant nucleotide sequence.<br>
Expression: The term "expression" includes any step involved in the production of the polypeptide including, but not limited to, transcription, post-transcriptional modification, translation, post-translational modification, and secretion.<br>
Expression vector: The term "expression vector" is defined herein as a linear or circular DNA molecule that comprises a polynucleotide encoding a polypeptide of the invention, and which is operably linked to additional nucleotides that provide for its expression.<br>
Host cell: The term "host cell", as used herein, includes any cell type which is susceptible to transformation,  transfect ion,  transduction,  and the like with  a nucleic acid  construct<br><br>
comprising a polynucleotide of the present invention.<br>
Modification: The term "modification" means herein any chemical modification of the polypeptide consisting of the mature polypeptide of SEQ ID NO: 2 as well as genetic manipulation of the DNA encoding that polypeptide. The modificatlon(s) can be substltution(s), deletion(s) and/or insertlons(s) of the amino acid(s) as well as replacement(s) of amino acid side chain(s).<br>
Artificial variant: When used herein, the term "artificial variant" means a polypeptide having lipase activity produced by an organism expressing a modified nucleotide sequence of SEQ ID NO: 1. The modified nucleotide sequence is obtained through human intervention by modification of the nucleotide sequence disclosed in SEQ ID NO: 1.<br>
Relative performance (RP): The term relative performance reflects performance of the enzyme variant compared to a reference enzyme when measured as the brightness of the color of the textile samples washed with that specific enzyme variant.<br>
Benefit-Risk factor (BR): The Benefit-Risk factor describes the wash performance compared to risk for odor when the substrate is removed.<br>
Conventions for Designation of Variants:<br>
In describing lipase variants according to the invention, the following nomenclature is used for ease of reference:<br>
Original amino acld(s):position(s):substituted amino acid(s)<br>
According to this nomenclature, for instance the substitution of glutamic acid for glycine in position 195 is shown as G195E. A deletion of glycine in the same position is shown as G195*, and insertion of an additional amino acid residue such as lysine is shown as G195GK.<br>
Where a specific lipase contains a "deletion" in comparison with other lipases and an insertion is made in such a position this is indicated as *36D for insertion of an aspartic acid in position 36.<br>
Multiple mutations are separated by pluses, i.e.: R170Y+G195E, representing mutations in positions 170 and 195 substituting tyrosine and glutamic acid for arginine and glycine, respectively.<br>
X231 indicates the amino acid in a parent polypeptide corresponding to position 231, when applying the described alignment procedure. X231R indicates that the amino acid is replaced with R. For SEQ ID N0:2 X is T, and T231R thus indicates a substitution of T in position 231 with R. Where the amino acid in a position (e.g. 231) may be substituted by<br><br>
another amino acid selected from a group of amino acids, e.g. the group consisting of R and P and Y, this will be indicated by X231R/PA'.<br>
In all cases, the accepted lUPAC single letter or triple letter amino acid abbreviation is employed.<br>
DETAILED DESCRIPTION OF THE INVENTION Polypeptides having Lipase Activity<br>
The present invention relates to isolated polypeptides having lipase activity selected from the group consisting of lipases having a RP of at least 0.8 and a BR of at least 1.1 at the test conditions given in the specification.<br>
In a preferred embodiment the lipase has a RP of at least 0.9, such as 1.0 or 1.1. In an even more preferred embodiment the lipase has a RP of at least 1.2, such as 1.3 or even 1.4.<br>
In another preferred embodiment the lipase has a BR of at least 1.2, such as 1.3 or even 1.4. In an even more preferred embodiment the lipase has a BR of at least 1.5, such as 1.6 or even 1.7.<br>
In a further aspect the lipase of the present invention further has a relative LU/A280 less than1, such as less than 0.95 at the test conditions given in the specification. In a preferred embodiment the relative LU/A280 is less than 0.90, such as less than 0.85 or even less than 0.80.<br>
In a further aspect, the present invention relates to isolated polypeptides having an amino acid sequence which is comprised by or comprises SEQ ID N0:2, or an allelic variant thereof, and which further has BR of at least 1.1 and RP of at least 0.8. In another aspect, the present invention relates to isolated polypeptides having an amino acid sequence which is comprised by or comprises the mature part of SEQ ID N0:2, or an allelic variant thereof, and which further has BR of at least 1.1 and RP of at least 0.8<br>
In a still further aspect, the present invention relates to isolated polypeptides having an amino acid sequence which has a degree of identity to the mature polypeptide of SEQ ID NO: 2 (i.e., the mature polypeptide) of at least 80%, such as at least 85% or 90%, or at least 95%, preferably at least 97%, most preferably at least 98%, and even most preferably at least 99%, which have lipase activity (hereinafter "homologous polypeptides"). In a preferred aspect, the homologous polypeptides have an amino acid sequence which differs by ten amino acids, by nine amino acids, by eight amino acids, by seven amino acids, by six amino acids, preferably by five amino acids, more preferably by four amino acids, even more preferably by three amino acids, most preferably by two amino acids, and even most preferably by one amino acid from<br>
8<br><br>
the mature polypeptide of SEQ ID NO: 2.<br>
In a further aspect, the present invention relates to isolated polypeptides having lipase activity which are encoded by polynucleotides which hybridize under very low stringency conditions, preferably low stringency conditions, more preferably medium stringency conditions, more preferably medium-high stringency conditions, even more preferably high stringency conditions, and most preferably very high stringency conditions with (i) nucleotides 644 to 732 of SEQ ID NO: 1, (ii) the cDNA sequence contained in nucleotides 644 to 732of SEQ ID NO: 1, (iii) a subsequence of (i) or (ii), or (iv) a complementary strand of (i), (ii), or (iii) (J. Sambrook, E.F. Fritsch, and T. Maniatus, 1989, Molecular Cloning, A Laboratory Manual, 2d edition, Cold Spring Harbor, New York). A subsequence of SEQ ID NO: 1 contains at least 100 contiguous nucleotides or preferably at least 200 contiguous nucleotides. Moreover, the subsequence may encode a polypeptide fragment which has lipase activity.<br>
The nucleotide sequence of SEQ ID NO: 1 or a subsequence thereof, as well as the amino acid sequence of SEQ ID NO: 2 or a fragment thereof, may be used to design a nucleic acid probe to identify and clone DNA encoding polypeptides having lipase activity from strains of different genera or species according to methods well known in the art. In particular, such probes can be used for hybridization with the genomic or cDNA of the genus or species of interest, following standard Southern blotting procedures, in order to identify and isolate the corresponding gene therein. Such probes can be considerably shorter than the entire sequence, but should be at least 14, preferably at least 25, more preferably at least 35, and most preferably at least 70 nucleotides in length. It is, however, preferred that the nucleic acid probe is at least 100 nucleotides in length. For example, the nucleic acid probe may be at least 200 nucleotides, preferably at least 300 nucleotides, more preferably at least 400 nucleotides, or most preferably at least 500 nucleotides in length. Even longer probes may be used, e.g., nucleic acid probes which are at least 600 nucleotides, at least preferably at least 700 nucleotides, or more preferably at least 800 nucleotides in length. Both DNA and RNA probes can be used. The probes are typically labeled for detecting the corresponding gene (for example, with ‘‘P, ‘H, "S, biotin, or avidin). Such probes are encompassed by the present invention.<br>
A genomic DNA or cDNA library prepared from such other organisms may, therefore, be screened for DNA which hybridizes with the probes described above and which encodes a polypeptide having lipase activity. Genomic or other DNA from such other organisms may be separated by agarose or polyacrylamide gel electrophoresis, or other separation techniques. DNA from the libraries or the separated DNA may be transferred to and immobilized on<br>
9<br><br>
nitrocellulose or other suitable carrier material. In order to identify a clone or DNA which is homologous with SEQ ID NO: 1 or a subsequence thereof, the carrier material is used in a Southern blot.<br>
For purposes of the present invention, hybridization indicates that the nucleotide sequence hybridizes to a labeled nucleic acid probe corresponding to the nucleotide sequence shown in SEQ ID NO: 1, its complementary strand, or a subsequence thereof, under very low to very high stringency conditions. Molecules to which the nucleic acid probe hybridizes under these conditions can be detected using X-ray film.<br>
In a still further aspect, the present invention relates to artificial variants comprising a conservative substitution, deletion, and/or insertion of one or more amino acids of SEQ ID NO: 2 or the mature polypeptide thereof, said variants having a BR of at least 1.1 and a RP of at least 0.8. Preferably, amino acid changes are of a minor nature, that is conservative amino acid substitutions or insertions that do not significantly affect the folding and/or activity of the protein; small deletions, typically of one to about 30 amino acids; small amino- or carboxyl-terminal extensions, such as an amino-terminal methionine residue; a small linker peptide of up to about 20-25 residues; or a small extension that facilitates purification by changing net charge or another function, such as a poly-histidine tract, an antigenic epitope or a binding domain.<br>
Examples of conservative substitutions are within the group of basic amino acids (arginine, lysine and histidine), acidic amino acids (glutamic acid and aspartic acid), polar amino acids (glutamine and asparagine), hydrophobic amino acids (leucine, isoleucine and valine), aromatic amino acids (phenylalanine, tryptophan and tyrosine), and small amino acids (glycine, alanine, serine, threonine and methionine). Amino acid substitutions which do not generally alter specific activity are known in the art and are described, for example, by H. Neurath and R.L. Hill, 1979, In, The Proteins, Academic Press, New York. The most commonly occurring exchanges are Ala/Ser, Val/lle, Asp/Glu, Thr/Ser, Ala/Giy, Ala/Thr, Ser/Asn, AlaA/al, Ser/Gly, Tyr/Phe, Ala/Pro, Lys/Arg, Asp/Asn, Leu/He, LeuA/al, Ala/Glu, and Asp/Gly.<br>
In addition to the 20 standard amino acids, non-standard amino acids (such as 4-hydroxyproline, 6-A/-methyl lysine, 2-aminoisobutyric acid, Isovaline, and alpha-methyl serine) may be substituted for amino acid residues of a wild-type polypeptide. A limited number of non-conservative amino acids, amino acids that are not encoded by the genetic code, and unnatural amino acids may be substituted for amino acid residues. "Unnatural amino acids" have been modified after protein synthesis, and/or have a chemical structure in their side chain(s) different from that of the standard amino acids. Unnatural amino acids can be chemically synthesized, and preferably, are commercially available, and include pipecolic acid, thiazolidine carboxylic<br>
10<br><br>
acid, dehydroproline, 3- and 4-methylproline, and 3,3-dimethylproline.<br>
Alternatively, the amino acid changes are of such a nature that the physico-chemical properties of the polypeptides are altered. For example, amino acid changes may improve the thermal stability of the polypeptide, alter the substrate specificity, change the pH optimum, and the like.<br>
Essential amino acids in the parent polypeptide can be identified according to procedures known in the art, such as site-directed mutagenesis or alanine-scanning mutagenesis (Cunningham and Wells, 1989, Science 244: 1081-1085). In the latter technique, single alanine mutations are introduced at every residue in the molecule, and the resultant mutant molecules are tested for biological activity (i.e., lipase activity) to identify amino acid residues that are critical to the activity of the molecule. See also, Hilton et al., 1996, J. Biol. Chem. 271: 4699-4708. The active site of the enzyme or other biological interaction can also be determined by physical analysis of structure, as determined by such techniques as nuclear magnetic resonance, crystallography, electron diffraction, or photoaffinity labeling, in conjunction with mutation of putative contact site amino acids. See, for example, de Vos et al., 1992, Science 255: 306-312; Smith et al., 1992, J. Mol. Biol. 224: 899-904; Wlodaver et al., 1992, FEBS Lett. 309:59-64. The identities of essential amino acids can also be inferred from analysis of identities with polypeptides which are related to a polypeptide according to the invention.<br>
Single or multiple amino acid substitutions can be made and tested using known methods of mutagenesis, recombination, and/or shuffling, followed by a relevant screening procedure, such as those disclosed by Reidhaar-Olson and Sauer, 1988, Science 241: 53-57; Bowie and Sauer, 1989, Proc. Natl. Acad. Sci. USA 86: 2152-2156; WO 95/17413; or WO 95/22625. Other methods that can be used include error-prone PCR, phage display (e.g., Lowman et al., 1991, Biochem. 30:10832-10837; U.S. Patent No. 5,223,409; WO 92/06204), and region-directed mutagenesis (Derbyshire et al., 1986, Gene 46:145; Ner ef al., 1988, DNA 7:127).<br>
Mutagenesis/shuffling methods can be combined with high-throughput, automated screening methods to detect activity of cloned, mutagenized polypeptides expressed by host cells. (Mutagenized DNA molecules that encode active polypeptides can be recovered from the host cells and rapidly sequenced using standard methods in the art. These methods allow the rapid determination of the importance of individual amino acid residues in a polypeptide of interest, and can be applied to polypeptides of unknown structure.<br>
The total number of amino acid substitutions, deletions and/or insertions of amino acids 1 to 291 of SEQ ID NO: 2 is 10, preferably 9, more preferably 8, more preferably 7, more preferably at most 6, more preferably at most 5, more preferably 4, even more preferably 3,<br>
11<br><br>
most preferably 2, and even most preferably 1.<br>
identification of regions and substitutions<br>
Substitutions covered by the present application may be identified as disclosed in this section.<br>
The positions referred to in Region I through Region IV below are the positions of the amino acid residues in SEQ ID N0:2. To find the corresponding (or homologous) positions in a different lipase, the procedure described in "Homology and alignment" is used. Substitutions in Region I<br>
Region I consists of amino acid residues surrounding the N-terminal residue E1. In this region it is preferred to substitute an amino acid of the parent lipase with a more positive amino acid.<br>
Amino acid residues corresponding to the following positions are comprised by Region I: 2 to 11 and 223-239. The following positions are of particular interest: 4, 8, 11, 223, 229, 231, 233, 234, 236.<br>
In particular the following substitutions have been identified: X4V, X231R and X233R.<br>
In a preferred embodiment the variant lipase has at least 80%, such as 85% or 90%, such as at least 95% or 98% or 99% identity to SEQ ID N0:2   ■<br>
Substitutions in Region II<br>
Region II consists of arfiino acid residues in contact with substrate on one side of the acyl chain and one side of the alcohol part. In this region it is preferred to substitute an amino acid of the parent lipase with a more positive amino acid or with a less hydrophobic amino acid.<br>
Amino acid residues corresponding to the following positions are comprised by Region II: 202 to 211 and 249 to 269. The following positions are of particular interest: 202, 210, 211, 253, 254, 255, 256.<br>
In particular the following substitutions have been identified: X202G, X255YA/ and X256K/R.<br>
In a preferred embodiment the variant lipase has at least 80%, such as 85% or 90%, such as at least 95% or 98% or 99% identity to SEQ ID N0:2 Substitutions in Region III<br>
Region III consists of amino acid residues that forms a flexible structure and thus allowing the substrate to get into the active site. In this region it is preferred to substitute an<br>
12<br><br>
amino acid of the parent lipase with a more positive amino acid or a less hydrophobic amino acid.<br>
Amino acid residues corresponding to the following positions are comprised by Region ill: 82 to 102. The following positions are of particular interest: 86, 87, 90, 91, 95, 96, 99.<br>
In particular the following substitutions have been identified: X86V and X90A/R.<br>
In a preferred embodiment the variant lipase has at least 80%, such as 85% or 90%, such as at least 95% or 98% or 99% identity to SEQ ID N0:2 Substitutions in Region iV<br>
Region IV consists of amino acid residues that binds electrostatically to a surface. In this region it is preferred to substitute an amino acid of the parent lipase with a more positive amino acid.<br>
Amino acid residues corresponding to the following positions are comprised by Region IV: 27 and 54 to 62. The following positions are of particular interest: 27, 56, 57, 58, 60.<br>
In particular the following substitutions have been identified: X27R, X58N/AG/T/P and X60V/S/G/N/R/K/A/L.<br>
In a preferred embodiment the variant lipase has at least 80%, such as 85% or 90%, such as at least 95% or 98% or 99% identity to SEQ ID N0:2 Amino acids at other positions<br>
The parent lipase may optionally comprise substitution of other amino acids, particularty less than 10 or less than 5 such substitutions. Examples are substitutions corresponding to one or more of the positions 24, 46, 74, 81, 83, 127, 131, 137, 147, 150, 203, 206, 211, 263, 264, 265, 267 and 269 of SEQ ID NO: 2. In a particular embodiment there is a substitution in at least one of the positions corresponding to position 81,147,150, 227 and 249. In a preferred embodiment the at least one substitution is selected from the group consisting of X81Q/E, X147MA', X150G, X227G and X249R/I/L<br>
Further substitutions may, e.g., be made according to principles known in the art, e.g. substitutions described in WO 92/05249, WO 94/25577, WO 95/22615, WO 97/04079 and WO 97/07202.<br>
Homology and alignment<br>
For purposes of the present invention, the degree of homology may be suitably determined by means of computer programs known in the art, such as GAP provided in the GCG program package (Program Manual for the Wisconsin Package, Version 8, August 1994, Genetics Computer Group, 575 Science Drive, Madison, Wisconsin, USA 53711) (Needleman,<br>
13<br><br>
S.B. and Wunsch, CD., (1970), Journal of Molecular Biology, 48, 443-45), using GAP with the following settings for polypeptide sequence comparison: GAP creation penalty of 3.0 and GAP extension penalty of 0.1.<br>
In the present invention, corresponding (or homologous) positions in the lipase sequences of Absidia reflexa, Absidia corymbefera, Rhizmucor miehei, Rhizopus delemar, Aspergillus niger, Aspergillus tubigensis, Fusarium oxysporum, Fusarium heterosporum, Aspergillus oryzea, Penicilium camembertii, Aspergillus foetidus, Aspergillus niger, Thermomyces lanoginosus (synonym: Humicola lanuginose) and Landerina penisapora are defined by the alignment shown in Figure 1.<br>
To find the homologous positions in lipase sequences not shown in the alignment, the sequence of interest is aligned to the sequences shown in Figure 1. The new sequence is aligned to the present alignment in Figure 1 by using the GAP alignment to the most homologous sequence found by the GAP program. GAP is provided in the GCG program package (Program Manual for the Wisconsin Package, Version 8, August 1994, Genetics Computer Group, 575 Science Drive, Madison, Wisconsin, USA 53711) (Needleman, S.B. and Wunsch, CD., (1970), Journal of Molecular Biology, 48, 443-45). The following settings are used for polypeptide se-quence comparison: GAP creation penalty of 3.0 and GAP extension penalty of 0.1.<br>
Any suitable parent lipase may be used. In a preferred embodiment, the parent lipase has a homology of at least 50% with the T. lanuginosus lipase (SEQ ID NO: 2), particularly at least 55%, at least 60%, at least 75%, at least 85% , at least 90%, more than 95%, 96%, 97% or more than 98 % or 99%. In a particular embodiment the parent lipase is identical to the T. lanuginosus lipase (SEQ ID N0:2).<br>
Sources of Polypeptides Having Lipase Activity<br>
A polypeptide of the present invention may be obtained from microorganisms of any genus. For purposes of the present invention, the term "obtained from" as used herein in connection with a given source shall mean that the polypeptide encoded by a nucleotide sequence is produced by the source or by a strain in which the nucleotide sequence from the source has been inserted. In a preferred aspect, the polypeptide obtained from a given source is secreted extracellularly.<br>
A polypeptide of the present invention may be a bacterial polypeptide. For example, the polypeptide may be a gram positive bacterial polypeptide such as a Bacillus polypeptide, e.g., a Bacillus alkalophilus, Bacillus amyloliquefaciens. Bacillus brevis, Bacillus circulans, Bacillus<br>
14<br><br>
coagulans, Bacillus lautus, Bacillus lentus, Bacillus Hcheniformis, Bacillus megaterium, Bacillus stearothermophilus, Bacillus subtilis, or Bacillus thuringiensis polypeptide; or a Streptomyces polypeptide, e.g., a Streptomyces lividans or Streptomyces murinus polypeptide; or a gram negative bacterial polypeptide, e.g., an E. coli or a Pseudomonas sp. polypeptide.<br>
A polypeptide of the present invention may also be a fungal polypeptide, and more preferably a yeast polypeptide such as a Candida, Kluyveromyces, Pictiia, Saccharomyces, Schizosaccharomyces, or Yarrowia polypeptide; or more preferably a filamentous fungal polypeptide such as an Acremonium, Aspergillus, Aureobasidium, Cryptococcus, Filobasidium, Fusarium, Humicola, Magnaporthe, Mucor, Myceliophthora, Neocallimastix, Neurospora, Paecilomyces, Penicillium, Piromyces, Schizophyllum, Talaromyces, Thermoascus, Thielavia, Tolypocladium, or Trichoderma polypeptide.<br>
In a preferred aspect, the polypeptide is a Saccharomyces carlsbergensis, Saccharomyces cerevisiae, Saccharomyces diastaticus, Saccharomyces douglasii, Saccharomyces kluyveri, Saccharomyces norbensis, or Saccharomyces oviformis polypeptide having lipase activity.<br>
In another preferred aspect, the polypeptide is an Aspergillus aculeatus, Aspergillus awamori, Aspergillus fumigatus, Aspergillus foetidus, Aspergillus japonicus, Aspergillus nidulans, Aspergillus niger, Aspergillus oryzae, Aspergillus turbigensis, Fusarium bactridioides, Fusarium cerealis, Fusarium crool<wellense fusahum culmorum fusarium graminearum graminum heterosporum negundi oxysporum reticulatum roseum sambucinum fusanum sarcochroum sporothchioides sulphureum torulosum trichothecioides venenatum humicola insolens thermomyces lanoginosus fsynonym: lanuginose mucor miehei myceliophthora thermophila neurospora crassa penicillium purpurogenum trichoderma harzianum thchoderma koningii longibrachiatum reesei or viride polypeptide.></wellense>
In another preferred aspect, the polypeptide is a Thermomyces polypeptide.<br>
In a more preferred aspect, the polypeptide is a Thermomyces lanuginosus polypeptide, e.g., the polypeptide of SEQ ID NO: 2 with mutations as disclosed in the present application.<br>
It will be understood that for the aforementioned species, the invention encompasses both the perfect and imperfect states, and other taxonomic equivalents, e.g., anamorphs, regardless of the species name by which they are known. Those skilled in the art will readily recognize the identity of appropriate equivalents.<br>
Strains of these species are readily accessible to the public in a number of culture<br>
15<br><br>
collections, such as the American Type Culture Collection (ATCC), Deutsche Sammlung von Mikroorganismen und Zellkulturen GmbH (DSMZ), Centraalbureau Voor Schimmelcultures (CBS), and Agricultural Research Service Patent Culture Collection, Northern Regional Research Center (NRRL).<br>
Furthermore, such polypeptides may be identified and obtained from other sources including microorganisms isolated from nature (e.g., soil, composts, water, etc.) using the above-mentioned probes. Techniques for isolating microorganisms from natural habitats are well known in the art. The polynucleotide may then be obtained by similarly screening a genomic or cDNA library of another microorganism. Once a polynucleotide sequence encoding a polypeptide has been detected with the probe(s), the polynucleotide can be isolated or cloned by utilizing techniques which are well known to those of ordinary skill in the art (see, e.g., Sambrook etal., 1989, supra).<br>
Polypeptides of the present invention also include fused polypeptides or cleavable fusion polypeptides in which another polypeptide is fused at the N-terminus or the C-terminus of the polypeptide or fragment thereof. A fused polypeptide Is produced by fusing a nucleotide sequence (or a portion thereof) encoding another polypeptide to a nucleotide sequence (or a portion thereof) of the present invention. Techniques for producing fusion polypeptides are known in the art, and include ligating the coding sequences encoding the polypeptides so that they are in frame and that expression of the fused polypeptide is under control of the same promoter(s) and terminator.<br>
Polynucleotides<br>
The present invention also relates to isolated polynucleotides having a nucleotide sequence which encode a polypeptide of the present invention. The present invention also encompasses nucleotide sequences which encode a polypeptide being a variant of the amino acid sequence of SEQ ID NO: 2 or the mature polypeptide thereof, which differ from the encoding polynucleotide by virtue of the degeneracy of the genetic code. The present invention also relates to subsequences of SEQ ID NO: 1 which encode fragments of SEQ ID NO: 2 that have lipase activity, said fragments having a BR of at least 1.1 and a RP of at least 0.8.<br>
The techniques used to isolate or clone a polynucleotide encoding a polypeptide are known in the art and include isolation from genomic DNA, preparation from cDNA, or a combination thereof. The cloning of the polynucleotides of the present invention from such genomic DNA can be effected, e.g., by using the well known polymerase chain reaction (PCR) or antibody screening of expression libraries to detect cloned DNA fragments with shared<br>
16<br><br>
structural features. See, e.g., Innis ef a/., 1990, PCR: A Guide to Methods and Application, Academic Press, New York. Other nucleic acid amplification procedures such as ligase chain reaction (LCR), ligated activated transcription (LAT) and nucleotide sequence-based amplification (NASBA) may be used. The polynucleotides may be cloned from a strain of Ttiermomyces, or another or related organism and thus, for example, may be an allelic or species variant of the polypeptide encoding region of the nucleotide sequence.<br>
The present invention also relates to polynucleotides having nucleotide sequences which have a degree of identity to the mature polypeptide coding sequence of SEQ ID NO: 1 of at least 60%, preferably at least 65%, more preferably at least 70%, more preferably at least 75%, more preferably at least 80%, more preferably at least 85%, more preferably at least 90%, even more preferably at least 95%, and most preferably at least 97%, 98%, or 99% identity, which encode an active polypeptide having lipase activity and BR of at least 1.1 and RP of at least 0.8.<br>
Modification of a nucleotide sequence encoding a polypeptide of the present invention may be necessary for the synthesis of polypeptides substantially similar to the polypeptide. The term "substantially similar" to the polypeptide refers to non-naturally occurring forms of the polypeptide. These polypeptides may differ in some engineered way from the polypeptide isolated from its native source, e.g., artificial variants that differ in specific activity, thermo stability, pH optimum, or the like. The variant sequence may be constructed on the basis of the nucleotide sequence presented as the polypeptide encoding region of SEQ ID NO: 1, e.g., a subsequence thereof, and/or by introduction of nucleotide substitutions which do not give rise to another amino acid sequence of the polypeptide encoded by the nucleotide sequence, but which correspond to the codon usage of the host organism intended for production of the enzyme, or by introduction of nucleotide substitutions which may give rise to a different amino acid sequence. For a general description of nucleotide substitution, see, e.g.. Ford at a!., 1991, Protein Expression and Purification 2: 95-107.<br>
It will be apparent to those skilled in the art that such substitutions can be made outside the regions critical to the function of the molecule and still result in an active polypeptide. Amino acid residues essential to the activity of the polypeptide encoded by an isolated polynucleotide of the invention, and therefore preferably not subject to substitution, may be identified according to procedures known in the art, such as site-directed mutagenesis or alanine-scanning mutagenesis (see, e.g., Cunningham and Wells, 1989, Science 244: 1081-1085). In the latter technique, mutations are introduced at every positively charged residue in the molecule, and the resultant mutant molecules are tested for lipase activity to identify amino<br>
17<br><br>
acid residues that are critical to the activity of the molecule. Sites of substrate-enzyme interaction can also be determined by analysis of the three-dimensional structure as determined by such techniques as nuclear magnetic resonance analysis, crystallography or photoaffinity labelling (see, e.g., de Vos et al., 1992, Science 255: 306-312; Smith e’ a/., 1992, Journal of Molecular Biology 224: 899-904; Wlodaver et al., 1992, FEBS Letters 309: 59-64).<br>
The present invention also relates to isolated polynucleotides encoding a polypeptide of the present invention, which hybridize under very low stringency conditions, preferably low stringency conditions, more preferably medium stringency conditions, more preferably medium-high stringency conditions, even more preferably high stringency conditions, and most preferably very high stringency conditions with (i) of SEQ ID NO: 1, (ii) the cDNA sequence contained in SEQ ID NO: 1, or (ill) a complementary strand of (i) or (ii); or allelic variants and subsequences thereof (Sambrook etal., 1989, supra), as defined herein.<br>
The present invention also relates to isolated polynucleotides obtained by (a) hybridizing a population of DNA under very low, low, medium, medium-high, high, or very high stringency conditions with (i) nucleotides SEQ ID NO: 1, (ii) the cDNA sequence contained in nucleotides of SEQ ID NO: 1, or (ill) a complementary strand of (i) or (ii); and (b) isolating the hybridizing polynucleotide, which encodes a polypeptide having lipase activity.<br>
Nucleic Acid Constructs<br>
The present invention also relates to nucleic acid constructs comprising an isolated polynucleotide of the present invention operably linked to one or more control sequences which direct the expression of the coding sequence in a suitable host cell under conditions compatible with the control sequences.<br>
An isolated polynucleotide encoding a polypeptide of the present invention may be manipulated in a variety of ways to provide for expression of the polypeptide. Manipulation of the polynucleotide's sequence prior to its insertion into a vector may be desirable or necessary depending on the expression vector. The techniques for modifying polynucleotide sequences utilizing recombinant DNA methods are well known in the art.<br>
The control sequence may be an appropriate promoter sequence, a nucleotide sequence which is recognized by a host cell for expression of a polynucleotide encoding a polypeptide of the present invention. The promoter sequence contains transcriptional control sequences which mediate the expression of the polypeptide. The promoter may be any nucleotide sequence which shows transcriptional activity in the host cell of choice including mutant, truncated, and hybrid promoters, and may be obtained from genes encoding extracellular or<br>
18<br><br>
Intracellular polypeptides either homologous or heterologous to the host cell.<br>
Examples of suitable promoters for directing the transcription of the nucleic acid constructs of the present invention, especially in a bacterial host cell, are the promoters obtained from the £. coli lac operon, Streptomyces coelicolor agarase gene (dagA), Bacillus subtilis levansucrase gene (sacB), Bacillus licheniformis alpha-amylase gene (amyL), Bacillus stearothermophilus maltogenic amylase gene (amyM), Bacillus amyloliquefaciens alpha-amylase gene (amyQ), Bacillus licheniformis penicillinase gene (penP), Bacillus subtilis xylA and xylB genes, and prokaryotic beta-lactamase gene (Villa-Kamaroff et al., 1978, Proceedings of the National Academy of Sciences USA 75: 3727-3731), as well as the tac promoter (DeBoer et al., 1983, Proceedings of the National Academy of Sciences USA 80: 21-25). Further promoters are described in "Useful proteins from recombinant bacteria" in Scientific American, 1980, 242: 74-94; and in Sambrook et al., 1989, supra.<br>
Examples of suitable promoters for directing the transcription of the nucleic acid constructs of the present invention in a filamentous fungal host cell are promoters obtained from the genes for Aspergillus oryzae TAKA amylase, Rhizomucor miehei aspartic proteinase, Aspergillus niger neutral alpha-amylase, Aspergillus niger acid stable alpha-amylase, Aspergillus niger or Aspergillus awamoh glucoamylase (glaA), Rhizomucor miehei lipase, Aspergillus oryzae alkaline protease, Aspergillus oryzae those phosphate isomerase, Aspergillus nidulans acetamidase, Fusarium venenatum amyloglucosidase (WO 00/56900), Fusarium venenatum Dana (WO 00/56900), Fusarium venenatum Quinn (WO 00/56900), Fusarium oxysporum trypsin-like protease (WO 96/00787), Thchodenna reesei beta-glucosidase, Trichoderma reesei cellobiohydrolase I, Trichodenva reesei endoglucanase 1, Trichoderma reesei endoglucanase II, Trichoderma reesei endoglucanase III, Thchoderma reese/endoglucanase IV, Tr/c/voderma reese/endoglucanase V, Trichoderma reesei x’\anase I, Trichoderma reesei xylanase II, Trichoderma reesei beta-xylosidase, as well as the NA2-tpi promoter (a hybrid of the promoters from the genes for Aspergillus niger neutral alpha-amylase and Aspergillus oryzae those phosphate isomerase); and mutant, truncated, and hybrid promoters thereof.<br>
In a yeast host, useful promoters are obtained from the genes for Saccharomyces<br>
cerevisiae enolase (ENO-1), Saccharomyces cerevisiae galactokinase (GAL1), Saccharomyces<br>
cerevisiae	alcohol	dehydrogenase/glyceraldehyde-3-phosphate	dehydrogenase<br>
(ADH1,ADH2/GAP), Saccharomyces cerevisiae those phosphate isomerase (TPI), Saccharomyces cerevisiae metallothionine (CUP1), and Saccharomyces cerevisiae 3-phosphoglycerate kinase.    Other useful promoters for yeast host cells are described by<br>
19<br><br>
Romanes etal., 1992, Yeast 8: 423-488.<br>
The control sequence may also be a suitable transcription terminator sequence, a sequence recognized by a host cell to terminate transcription. The terminator sequence is operably linked to the 3' terminus of the nucleotide sequence encoding the polypeptide. Any terminator which is functional in the host cell of choice may be used in the present invention.<br>
Preferred terminators for filamentous fungal host cells are obtained from the genes for Aspergillus oryzae TAKA amylase, Aspergillus niger glucoamylase, Aspergillus nidulans anthranilate synthase, Aspergillus niger alpha-glucosidase, and Fusarium oxysporum trypsin-iike protease.<br>
Preferred terminators for yeast host cells are obtained from the genes for Saccharomyces cerevisiae enolase, Saccharomyces cerevisiae cytochrome C (CYC1), and Saccharomyces cerevisiae glyceraldehyde-3-phosphate dehydrogenase. Other useful terminators for yeast host cells are described by Romanes et al., 1992, supra.<br>
The control sequence may also be a suitable leader sequence, a nontranslated region of an mRNA which is important for translation by the host cell. The leader sequence is operably linked to the 5' terminus of the nucleotide sequence encoding the polypeptide. Any leader sequence that is functional in the host cell of choice may be used in the present invention.<br>
Preferred leaders for filamentous fungal host cells are obtained from the genes for Aspergillus oryzae TAKA amylase and Aspergillus nidulans triose phosphate isomerase.<br>
Suitable leaders for yeast host cells are obtained from the genes for Saccharomyces cerevisiae enolase (ENO-1), Saccharomyces cerevisiae 3-phosphoglycerate kinase, Saccharomyces cerevisiae alpha-factor, and Saccharomyces cerevisiae alcohol dehydrogenase/glyceraldehyde-3-phosphate dehydrogenase (ADH2/GAP).<br>
The control sequence may also be a polyadenylation sequence, a sequence operably linked to the 3' terminus of the nucleotide sequence and which, when transcribed, is recognized by the host cell as a signal to add polyadenosine residues to transcribed mRNA. Any polyadenylation sequence which is functional in the host cell of choice may be used in the present invention.<br>
Preferred polyadenylation sequences for filamentous fungal host cells are obtained from the genes for Aspergillus oryzae TAKA amylase, Aspergillus niger glucoamylase, Aspergillus nidulans anthranilate synthase, Fusarium oxysporum trypsin-iike protease, and Aspergillus niger alpha-giucosidase.<br>
Useful polyadenylation sequences for yeast host cells are described by Guo and Sherman, 1995, Molecular Cellular Biology 15: 5983-5990.<br>
20<br><br>
The control sequence may also be a signal peptide coding region that codes for an amino acid sequence linked to the amino terminus of a polypeptide and directs the encoded polypeptide into the cell's secretory pathway. The 5' end of the coding sequence of the nucleotide sequence may inherently contain a signal peptide coding region naturally linked in translation reading frame with the segment of the coding region which encodes the secreted polypeptide. Alternatively, the 5' end of the coding sequence may contain a signal peptide coding region which is foreign to the coding sequence. The foreign signal peptide coding region may be required where the coding sequence does not naturally contain a signal peptide coding region. Alternatively, the foreign signal peptide coding region may simply replace the natural signal peptide coding region in order to enhance secretion of the polypeptide. However, any signal peptide coding region which directs the expressed polypeptide into the secretory pathway of a host cell of choice may be used in the present invention.<br>
Effective signal peptide coding regions for bacterial host cells are the signal peptide coding regions obtained from the genes for Bacillus NCIB 11837 maltogenic amylase. Bacillus stearothermophilus alpha-amylase. Bacillus licheniformis subtilisin, Bacillus licheniformis beta-lactamase. Bacillus stearothermophilus neutral proteases {nprT, nprS, nprM), and Bacillus subtilis prsA. Further signal peptides are described by Simonen and Palva, 1993, Microbiological Reviews 57: 109-137.<br>
Effective signal peptide coding regions for filamentous fungal host cells are the signal peptide coding regions obtained from the genes for Aspergillus oryzae TAKA amylase, Aspergillus niger neutral amylase, Aspergillus niger glucoamylase, Rhizomucor miehei aspartic proteinase, IHumicola insolens cellulase, and Humicola lanuginosa lipase.<br>
Useful signal peptides for yeast host cells are obtained from the genes for Saccharomyces cerevisiae alpha-factor and Saccharomyces cerevisiae invertase. Other useful signal peptide coding regions are described by Romanes etal., 1992, supra.<br>
The control sequence may also be a propeptide coding region that codes for an amino acid sequence positioned at the amino terminus of a polypeptide. The resultant polypeptide is known as a proenzyme or propolypeptide (or a zymogen in some cases). A propolypeptide is generally inactive and can be converted to a mature active polypeptide by catalytic or autocatalytic cleavage of the propeptide from the propolypeptide. The propeptide coding region may be obtained from the genes for Bacillus subtilis alkaline protease (aprE), Bacillus subtilis neutral protease (nprT), Saccharomyces cerevisiae alpha-factor, Rhizomucor miehei aspartic proteinase, and Myceliophthora thermophila laccase (WO 95/33836).<br>
Where both signal peptide and propeptide regions are present at the amino terminus of a<br>
21<br><br>
polypeptide, the propeptide region is positioned next to the amino terminus of a polypeptide and the signal peptide region is positioned next to the amino terminus of the propeptide region.<br>
It may also be desirable to add regulatory sequences which allow the regulation of the expression of the polypeptide relative to the growth of the host cell. Examples of regulatory systems are those which cause the expression of the gene to be turned on or off in response to a chemical or physical stimulus, including the presence of a regulatory compound. Regulatory systems in prokaryotic systems include the lac, tac, and trp operator systems. In yeast, the ADH2 system or GAL1 system may be used. In filamentous fungi, the TAKA alpha-amylase promoter, Aspergillus niger glucoamylase promoter, and Aspergillus oryzae glucoamylase promoter may be used as regulatory sequences. Other examples of regulatory sequences are those which allow for gene amplification. In eukaryotic systems, these include the dihydrofolate reductase gene which is amplified in the presence of methotrexate, and the metallothionein genes which are amplified with heavy metals. In these cases, the nucleotide sequence encoding the polypeptide would be operably linked with the regulatory sequence.<br>
Expression Vectors<br>
The present invention also relates to recombinant expression vectors comprising a polynucleotide of the present invention, a promoter, and transcriptional and translational stop signals. The various nucleic acids and control sequences described above may be joined together to produce a recombinant expression vector which may include one or more convenient restriction sites to allow for insertion or substitution of the nucleotide sequence encoding the polypeptide at such sites. Alternatively, a nucleotide sequence of the present invention may be expressed by inserting the nucleotide sequence or a nucleic acid construct comprising the sequence into an appropriate vector for expression. In creating the expression vector, the coding sequence is located in the vector so that the coding sequence is operably linked with the appropriate control sequences for expression.<br>
The recombinant expression vector may be any vector (e.g., a plasmid or virus) which can be conveniently subjected to recombinant DNA procedures and can bring about expression of the nucleotide sequence. The choice of the vector will typically depend on the compatibility of the vector with the host cell into which the vector is to be introduced. The vectors may be linear or closed circular plasmids.<br>
The vector may be an autonomously replicating vector, i.e., a vector which exists as an extrachromosomal entity, the replication of which is independent of chromosomal replication, e.g.,   a   plasmid,   an   extrachromosomal   element,   a   minichromosome,   or   an   artificial<br>
22<br><br>
chromosome. The vector may contain any means for assuring self-replication. Alternatively, the vector may be one which, when introduced into the host cell, is integrated into the genome and replicated together with the chromosome(s) into which it has been integrated. Furthermore, a single vector or plasmid or two or more vectors or plasmids which together contain the total DNA to be Introduced into the genome of the host cell, or a transposon may be used.<br>
The vectors of the present invention preferably contain one or more selectable markers which permit easy selection of transformed cells. A selectable marker is a gene the product of which provides for biocide or viral resistance, resistance to heavy metals, prototrophy to auxotrophs, and the like.<br>
A conditionally essential gene may function as a non-antibiotic selectable marker. Non-limiting examples of bacterial conditionally essential non-antibiotic selectable markers are the dal genes from Bacillus subtilis, Bacillus licheniformis, or other Bacilli, that are only essential when the bacterium is cultivated in the absence of D’alanine. Also the genes encoding enzymes involved in the turnover of UDP-galactose can function as conditionally essential markers in a cell when the cell is grown in the presence of galactose or grown in a medium which gives rise to the presence of galactose. Non-limiting examples of such genes are those from B. subtilis or B. licheniformis encoding UTP-dependent phosphorylase (EC 2.7.7.10), UDP-glucose-dependent uridylyltransferase (EC 2.7.7.12), or UDP-galactose epimerase (EC 5.1.3.2). Also a xylose Isomerase gene such as xylA, of Bacilli can be used as selectable markers in cells grown in minimal medium with xylose as sole carbon source. The genes necessary for utilizing gluconate, gntK, and gntP can also be used as selectable markers in cells grown in minimal medium with gluconate as sole carbon source. Other examples of conditionally essential genes are known in the art. Antibiotic selectable markers confer antibiotic resistance to such antibiotics as ampicillin, kanamycin, chloramphenicol, erythromycin, tetracycline, neomycin, hygromycin or methotrexate.<br>
Suitable markers for yeast host cells are ADE2, HISS, LEU2, LYS2, MET3, TRP1, and LIRA3. Selectable markers for use in a filamentous fungal host cell include, but are not limited to, amofS (acetamidase), argB (ornithine carbamoyltransferase), bar (phosphinothricin acetyltransferase), hph (hygromycin phosphotransferase), niaD (nitrate reductase), pyrG (orotidine-5'-phosphate decarboxylase), sC (sulfate adenyltransferase), and trpC (anthranilate synthase), as well as equivalents thereof. Preferred for use in an Aspergillus cell are the amdS and pyrG genes of Aspergillus nidulans or Aspergillus oryzae and the bar gene of Streptomyces hygroscopicus.<br>
23<br><br>
The vectors of the present invention preferably contain an element(s) that permits integration of the vector into the host cell's genome or autonomous replication of the vector in the cell independent of the genome.<br>
For integration into the host cell genome, the vector may rely on the polynucleotide's sequence encoding the polypeptide or any other element of the vector for integration into the genome by homologous or nonhomologous recombination. Alternatively, the vector may contain additional nucleotide sequences for directing integration by homologous recombination into the genome of the host cell at a precise location(s) in the chromosome(s). To increase the likelihood of integration at a precise location, the integrational elements should preferably contain a sufficient number of nucleic acids, such as 100 to 10,000 base pairs, preferably 400 to 10,000 base pairs, and most preferably 800 to 10,000 base pairs, which have a high degree of identity with the corresponding target sequence to enhance the probability of homologous recombination. The integrational elements may be any sequence that is homologous with the target sequence in the genome of the host cell. Furthermore, the integrational elements may be non-encoding or encoding nucleotide sequences. On the other hand, the vector may be integrated into the genome of the host cell by non-homologous recombination.<br>
For autonomous replication, the vector may further comprise an origin of replication enabling the vector to replicate autonomously in the host cell in question. The origin of replication may be any plasmid replicator mediating autonomous replication which functions in a cell. The term "origin of replication" or "plasmid replicator" is defined herein as a nucleotide sequence that enables a plasmid or vector to replicate in vivo.<br>
Examples of bacterial origins of replication are the origins of replication of plasmids pBR322, pUC19, pACYC177, and pACYC184 permitting replication in E. coli, and pUB110, pE194, pTA1060, and pAMBI permitting replication in Bacillus.<br>
Examples of origins of replication for use in a yeast host cell are the 2 micron origin of replication, ARS1, ARS4, the combination of ARS1 and CEN3, and the combination of ARS4 and CEN6.<br>
Examples of origins of replication useful in a filamentous fungal cell are AMA1 and ANSI (Gems et al., 1991, Gene 98:61-67; Cullen at al., 1987, Nucleic Acids Research 15: 9163-9175; WO 00/24883). Isolation of the AMA1 gene and construction of plasmids or vectors comprising the gene can be accomplished according to the methods disclosed in WO 00/24883.<br>
More than one copy of a polynucleotide of the present invention may be inserted into the host cell to increase production of the gene product. An increase in the copy number of the polynucleotide can be obtained by integrating at least one additional copy of the sequence into<br>
24<br><br>
the host cell genome or by including an amplifiable selectable marker gene with the polynucleotide where cells containing amplified copies of the selectable marker gene, and thereby additional copies of the polynucleotide, can be selected for by cultivating the cells in the presence of the appropriate selectable agent.<br>
The procedures used to ligate the elements described above to construct the recombinant expression vectors of the present invention are well known to one skilled in the art (see, e.g., Sambrook et al., 1989, supra).<br>
Host Ceils<br>
The present invention also relates to recombinant host cells, comprising a polynucleotide of the present invention, which are advantageously used in the recombinant production of the polypeptides. A vector comprising a polynucleotide of the present invention is introduced into a host cell so that the vector is maintained as a chromosomal integrant or as a self-replicating extra-chromosomal vector as described earlier. The term "host cell" encompasses any progeny of a parent cell that is not identical to the parent cell due to mutations that occur during replication. The choice of a host cell will to a large extent depend upon the gene encoding the polypeptide and its source.<br>
The host cell may be a unicellular microorganism, e.g., a prokaryote, or a non-unicellular microorganism, e.g., a eukaryote.<br>
Useful unicellular microorganisms are bacterial cells such as gram positive bacteria including, but not limited to, a Bacillus cell, e.g., Bacillus alkalophilus, Bacillus amyloliquefaciens, Bacillus brevis, Bacillus circulans, Bacillus clausii, Bacillus coagulans, Bacillus lautus, Bacillus lentus, Bacillus licheniformis. Bacillus megaterium, Bacillus stearothermophilus, Bacillus subtilis, and Bacillus thuringiensis; or a Streptomyces cell, e.g., Streptomyces lividans and Streptomyces murinus, or gram negative bacteria such as E. coli and Pseudomonas sp. In a preferred aspect, the bacterial host cell is a Bacillus lentus, Bacillus licheniformis. Bacillus stearothermophilus, or Bacillus subtilis cell. In another preferred aspect, the Bacillus cell is an alkalophilic Bacillus.<br>
The introduction of a vector into a bacterial host cell may, for instance, be effected by protoplast transformation (see, e.g., Chang and Cohen, 1979, Molecular General Genetics 168: 111-115), using competent cells (see, e.g.. Young and Spizizin, 1961, Journal of Bacteriology 81: 823-829, or Dubnau and Davidoff-Abelson, 1971, Journal of Molecular Biology 56: 209-221), electroporation (see, e.g., Shigekawa and Dower, 1988, Biotechniques 6: 742-751), or conjugation (see, e.g., Koehlerand Thome, 1987, Journal of Bacteriology ‘69: 5771-5278).<br>
25<br><br>
The host cell may also be a eukaryote, such as a mammalian, insect, plant, or fungal cell.<br>
In a preferred aspect, the host cell is a fungal cell. "Fungi" as used herein includes the phyla Ascomycota, Basidiomycota, Chytridiomycota, and Zygomycota (as defined by Hawksworth et al., In, Ainsworth and Bisby's Dictionary of The Fungi, 8th edition, 1995, CAB International, University Press, Cambridge, UK) as well as the Oomycota (as cited in Hawksworth et al., 1995, supra, page 171) and all mitosporic fungi (Hawksworth et al., 1995, supra).<br>
In a more preferred aspect, the fungal host cell is a yeast cell. "Yeast" as used herein includes ascosporogenous yeast (Endomycetales), basidiosporogenous yeast, and yeast belonging to the Fungi Imperfect! (Blastomycetes). Since the classification of yeast may change in the future, for the purposes of this invention, yeast shall be defined as described in Biology and Activities of Yeast (Skinner, F.A., Passmore, S.M., and Davenport, R.R., eds, Soc. App. Bacteriol. Symposium Series No. 9, 1980).<br>
In an even more preferred aspect, the yeast host cell is a Candida, Hansenula, Kluyveromyces, Pichia, Saccharomyces, Schizosaccharomyces, or Yarrowia cell.<br>
In a most preferred aspect, the yeast host cell is a Saccharomyces carlsbergensis, Saccharomyces cerevisiae, Saccharomyces diastaticus, Saccharomyces douglasii, Saccharomyces kluyveri, Saccharomyces norbensis or Saccharomyces oviformis cell. In another most preferred aspect, the yeast host cell is a Kluyveromyces lactis cell. In another most preferred aspect, the yeast host cell is a Yarrowia Hpolytica cell.<br>
In another more preferred aspect, the fungal host cell is a filamentous fungal cell. "Filamentous fungi" include all filamentous forms of the subdivision Eumycota and Oomycota (as defined by Hawksworth et al., 1995, supra). The filamentous fungi are generally characterized by a mycelial wall composed of chltin, cellulose, glucan, chitosan, mannan, and other complex polysaccharides. Vegetative growth is by hyphal elongation and carbon catabolism is obligately aerobic. In contrast, vegetative growth by yeasts such as Saccharomyces cerevisiae is by budding of a unicellular thallus and carbon catabolism may be fermentative.<br>
In an even more preferred aspect, the filamentous fungal host cell is an Acremonium, Aspergillus, Aureobasidium, Bjerkandera, Ceriporiopsis, Coprinus, Coriolus, Cryptococcus, Filobasidium, Fusarium, Humicola, Magnaporthe, Mucor, Myceliophthora, Neocallimastix, Neurospora, Paecilomyces, Penicillium, Phanerochaete, Phlebia, Piromyces, Pleurotus, Schizophyllum, Talaromyces, Thermoascus, Thielavia, Tolypocladium, Trametes, or Trichoderma cell.<br>
26<br><br>
In a most preferred aspect, the filamentous fungal host cell is an Aspergillus awamori, Aspergillus fumigatus, Aspergillus foetidus, Aspergillus japonicus, Aspergillus nidulans, Aspergillus niger or Aspergillus oryzae cell, in another most preferred aspect, the filamentous fungal host cell is a Fusarium bactridioides, Fusarium cerealis, Fusarium crookwellense, Fusarium culmorum, Fusarium graminearum, Fusarium graminum, Fusarium heterosporum, Fusarium negundi, Fusarium oxysporum, Fusarium reticulatum, Fusarium roseum, Fusarium sambucinum, Fusarium sarcochroum, Fusarium sporotrichioides, Fusarium sulphureum, Fusarium torulosum, Fusarium trichothecioides, or Fusarium venenatum cell. In another most preferred aspect, the filamentous fungal host cell is a Bjerkandera adusta, Ceriporiopsis aneirina, Ceriporiopsis aneirina, Ceriporiopsis caregiea, Ceriporiopsis gilvescens, Ceriporiopsis pannocinta, Ceriporiopsis rivulosa, Ceriporiopsis subrufa, or Ceriporiopsis subvermispora, Coprinus cinereus, Coriolus hirsutus, l-lumicola insolens, Humicola lanuginosa, Mucor miehei, Myceliophthora thermophila, Neurospora crassa, Penicillium purpurogenum, Phanerochaete chrysosporium, Phlebia radiata, Pleurotus eryngii, Thielavia terrestris, Trametes villosa, Trametes versicolor, Trichoderma harzianum, Trichoderma koningii, Trichoderma longibrachiatum, Trichoderma reesei, or Trichoderma viride strain cell.<br>
Fungal cells may be transformed by a process involving protoplast formation, transformation of the protoplasts, and regeneration of the cell wall in a manner known per se. Suitable procedures for transformation of Aspergillus and Trichoderma host cells are described in EP 238 023 and Yelton et al., 1984, Proceedings of the National Academy of Sciences USA 81: 1470-1474. Suitable methods for transforming Fusarium species are described by Malardier et al., 1989, Gene 78: 147-156, and WO 96/00787. Yeast may be transformed using the procedures described by Becker and Guarente, In Abelson, J.N. and Simon, M.I., editors. Guide to Yeast Genetics and Molecular Biology, Methods in Enzymology, Volume 194, pp 182-187, Academic Press, Inc., New York; Ito et al., 1983, Journal of Bacteriology 153: 163; and Hinnen etal., 1978, Proceedings of the National Academy of Sciences USA 75: 1920.<br>
Methods of Production<br>
The present invention also relates to methods for producing a polypeptide of the present invention, comprising (a) cultivating a cell, which in its wild-type form is capable of producing the polypeptide, under conditions conducive for production of the polypeptide; and (b) recovering the polypeptide. Preferably, the cell is of the genus Aspergillus, and more preferably Aspergillus Oryzae.<br>
The present invention also relates to methods for producing a polypeptide of the present<br>
27<br><br>
invention, comprising (a) cultivating a host cell under conditions conducive for production of the polypeptide; and (b) recovering the polypeptide.<br>
The present invention also relates to methods for producing a polypeptide of the present invention, comprising (a) cultivating a host cell under conditions conducive for production of the polypeptide, wherein the host cell comprises a mutant nucleotide sequence having at least one mutation in the mature polypeptide coding region of SEQ ID NO: 1, wherein the mutant nucleotide sequence encodes a polypeptide which is a lipase comprised by or comprising the polypeptide of SEQ ID NO: 2, and (b) recovering the polypeptide. In a preferred embodiment the nucleotide sequence encodes a polypeptide which is a lipase comprised by or comprising the mature part of the polypeptide of SEQ ID NO: 2, and (b) recovering the polypeptide.<br>
In the production methods of the present invention, the cells are cultivated in a nutrient medium suitable for production of the polypeptide using methods well known in the art. For example, the cell may be cultivated by shake flask cultivation, and small-scale or large-scale fermentation (including continuous, batch, fed-batch, or solid state fermentations) in laboratory or industrial fermentors performed in a suitable medium and under conditions allowing the polypeptide to be expressed and/or isolated. The cultivation takes place in a suitable nutrient medium comprising carbon and nitrogen sources and inorganic salts, using procedures known in the art. Suitable media are available from commercial suppliers or may be prepared according to published compositions (e.g., in catalogues of the American Type Culture Collection). If the polypeptide is secreted into the nutrient medium, the polypeptide can be recovered directly from the medium. If the polypeptide is not secreted, it can be recovered from cell lysates.<br>
The polypeptides may be detected using methods known in the art that are specific for the polypeptides. These detection methods may include use of specific antibodies, formation of an enzyme product, or disappearaince of an enzyme substrate. For example, an enzyme assay may be used to determine the activity of the polypeptide as described herein.<br>
The resulting polypeptide may be recovered using methods known in the art. For example, the polypeptide may be recovered from the nutrient medium by conventional procedures including, but not limited to, centrifugation, filtration, extraction, spray-drying, evaporation, or precipitation.<br>
The polypeptides of the present invention may be purified by a variety of procedures known in the art including, but not limited to, chromatography (e.g., ion exchange, affinity, hydrophobic, chromatofocusing, and size exclusion), electrophoretic procedures {e.g., preparative isoelectric focusing), differential solubility (e.g., ammonium sulfate precipitation),<br>
28<br><br>
SDS-PAGE, or extraction (see, e.g., Protein Purification, J.-C. Janson and Lars Ryden, editors, VCH Publishers, New York, 1989).<br>
Compositions<br>
The present invention also relates to compositions comprising a polypeptide of the present invention. Preferably, the compositions are enriched in such a polypeptide. The term "enriched" indicates that the lipase activity of the composition has been increased, e.g., with an enrichment factor of 1.1.<br>
The composition may comprise a polypeptide of the present invention as the major enzymatic component, e.g., a mono-component composition. Alternatively, the composition may comprise multiple enzymatic activities, such as an aminopeptidase, amylase, carbohydrase, carboxypeptidase, catalase, cellulase, chitinase, cutinase, cyclodextrin glycosyltransferase, deoxyribonuclease, esterase, alpha-galactosidase, beta-galactosidase, glucoamylase, alpha-glucosidase, beta-glucosidase, haloperoxidase, invertase, laccase, lipase, mannosidase, oxidase, pectinolytic enzyme, peptidoglutaminase, peroxidase, phytase, polyphenoloxidase, proteolytic enzyme, ribonuclease, transglutaminase, or xylanase. The additional enzyme(s) may be produced, for example, by a microorganism belonging to the genus Aspergillus, preferably Aspergillus aculeatus, Aspergillus awamoh, Aspergillus fumigatus, Aspergillus foetidus, Aspergillus japonicus, Aspergillus nidulans, Aspergillus niger, or Aspergillus oryzae; Fusarium, preferably Fusarium bactridioides, Fusarium cerealis, Fusarium crookwellense, Fusarium culmorum, Fusarium graminearum, Fusanum graminum, Fusarium heterosporum, Fusarium negundi, Fusarium oxysporum, Fusarium reticulatum, Fusarium roseum, Fusarium sambucinum, Fusarium sarcochroum, Fusarium sulphureum, Fusarium toruloseum, Fusarium trichothecioides, or Fusarium venenatum; Humicola, preferably Humicola insolens or Humicola lanuginosa; or Trichoderma, preferably Trichoderma harzianum, Trichoderma koningii, Trichoderma longibrachiatum, Trichoderma reesei, or Trichoderma Wide.<br>
The polypeptide compositions may be prepared in accordance with methods known in the art and may be in the form of a liquid or a dry composition. For instance, the polypeptide composition may be in the form of a granulate or a microgranulate. The polypeptide to be included in the composition may be stabilized in accordance with methods known in the art.<br>
Examples are given below of preferred uses of the polypeptide compositions of the invention. The dosage of the polypeptide composition of the invention and other conditions under which the composition is used may be determined on the basis of methods known in the art.<br>
29<br><br>
DETERGENT APPLICATIONS<br>
The enzyme of the invention may be added to and thus become a component of a detergent composition.<br>
The detergent composition of the invention may for example be formulated as a hand or machine laundry detergent composition including a laundry additive composition suitable for pre-treatment of stained fabrics and a rinse added fabric softener composition, or be formulated as a detergent composition for use in general household hard surface cleaning operations, or be formulated for hand or machine dishwashing operations. Enzymes<br>
In a specific aspect, the invention provides a detergent additive comprising the enzyme of the invention. The detergent additive as well as the detergent composition may comprise one or more other enzymes such as a protease, a lipase, a cutinase, an amylase, a carbohydrase, a cellulase, a pectinase, a mannanase, an arabinase, a galactanase, a xylanase, an oxidase, e.g., a laccase, and/or a peroxidase.<br>
In general the properties of the chosen enzyme(s) should be compatible with the selected detergent, (i.e. pH-optimum, compatibility with other enzymatic and non-enzymatic ingredients, etc.), and the enzyme(s) should be present in effective amounts. Proteases:<br>
Suitable proteases include those of animal, vegetable or microbial origin. Microbial origin is preferred. Chemically modified or protein engineered mutants are included. The protease may be a serine protease or a metallo protease, preferably an alkaline microbial protease or a trypsin-like protease. Examples of alkaline proteases are subtilisins, especially those derived from Bacillus, e.g., subtilisin Novo, subtilisin Carlsberg, subtilisin 309, subtilisin 147 and subtilisin 168 (described in WO 89/06279). Examples of trypsin-like proteases are trypsin (e.g. of porcine or bovine origin) and the Fusarium protease described in WO 89/06270 and WO 94/25583.<br>
Examples of useful proteases are the variants described in WO 92/19729, WO 98/20115, WO 98/20116, and WO 98/34946, especially the variants with substitutions in one or more of the following positions: 27, 36, 57, 68, 76, 87, 97, 101, 104, 106, 120, 123, 167, 170, 194, 206, 218, 222, 224, 235, 245, 252 and 274, and amongst other variants with the following mutations: (K27R, V104Y, N123S, T124A), (N76D, S103A, VI041), or (S101G, S103A, VI041, G159D, A232V, Q236H, Q245R, N248D, N252K).<br>
30<br><br>
Preferred commercially available protease enzymes include Alcalase™, Savinase™, Primase™, Duralase™, Esperase™, Coronase™, Polarzyme™ and Kannase™ (Novozymes A/S),   Maxatase™,   Maxacal™,   Maxapem™,   Properase™,   Purafect™,   Purafect   Prime™, Purafect OxP™, FN2, FN3 and FN4 (Genencor international Inc.). Lipases;<br>
Lipases include those of bacterial or fungal origin. Chemically modified or protein engineered mutants are included. Examples of useful lipases include lipases from Humicola (synonym Thermomyces), e.g. from H. lanuginosa (synonymous T. lanuginosus) as described in EP 258 068 and EP 305 216 or from H. insolens as described in WO 96/13580, a Pseudomonas lipase, e.g. from P. alcaligenes or P. pseudoalcaligenes (EP 218 272), P. cepacia (EP 331 376), P. stutzeri (GB 1,372,034), P. fluorescens, Pseudomonas sp. strain SD 705 (WO 95/06720 and WO 96/27002), P. wisconsinensis (WO 96/12012), a Bacillus lipase, e.g. from B. subtilis (Dartois et al. (1993), Biochemica et Biophysica Acta, 1131, 253-360), B. stearothermophilus (JP 64/744992) or B. pumilus (WO 91/16422).<br>
Other examples are lipase variants such as those described in WO 92/05249, WO 94/01541, EP 407 225, EP 260 105, WO 95/35381, WO 96/00292, WO 95/30744, WO 94/25578, WO 95/14783, WO 95/22615, WO 97/04079 and WO 97/07202.<br>
Other commecially available lipase enzymes include Lipolase™, Lipolase Ultra™ and Lipex™ (Novozymes A/S).<br>
Preferred lipases are lipases of the present invention. Amylases:<br>
Suitable amylases (a and/or p) include those of bacterial or fungal origin. Chemically modified or protein engineered mutants are included. Amylases Include, for example, a-amylases obtained from Bacillus, e.g. a special strain of B. licheniformis, described in more detail in GB 1,296,839.<br>
Examples of useful amylases are the variants described in WO 94/02597, WO 94/18314, WO 96/23873, and WO 97/43424, especially the variants with substitutions in one or more of the following positions: 15, 23, 105, 106, 124,128, 133, 154,156, 181, 188, 190, 197, 202, 208, 209, 243, 264, 304, 305, 391, 408, and 444.<br>
Commercially available amylases are Duramyl™, Termamyl™, Stainzyme™ , Stainzyme Ultra™, Fungamyl™ and BAN™ (Novozymes A/S), Rapidase™ and Purastar™ (from Genencor International Inc.). Cellulases:<br>
31<br><br>
Suitable cellulases include those of bacterial or fungal origin. Chemically modified or protein engineered mutants are included. Suitable cellulases include cellulases from the genera Bacillus, Pseudomonas, Humicola, Fusarium, Thielavia, Acremonlum, e.g. the fungal cellulases produced from Humicola insolens, Myceliophthora thermophila and Fusarium oxysporum disclosed in US 4,435,307, US 5,648,263, US 5,691,178, US 5,776,757 and WO 89/09259.<br>
Especially suitable cellulases are the alkaline or neutral cellulases having colour care benefits. Examples of such cellulases are cellulases described in EP 0 495 257, EP 0 531 372, WO 96/11262, WO 96/29397, WO 98/08940. Other examples are cellulase variants such as those described in WO 94/07998, EP 0 531 315, US 5,457,046, US 5,686,593, US 5,763,254, WO 95/24471, WO 98/12307 and PCT/DK98/00299.<br>
.TM<br>
Commercially available cellulases include Renozyme   , Celluzyme™, and Carezyme (Novozymes A/S), Clazinase™, and Puradax HA™ (Genencor International Inc.), and KAC-500(B)™ (Kao Corporation). Peroxidases/Oxidases:<br>
Suitable peroxidases/oxidases include those of plant, bacterial or fungal origin. Chemically modified or protein engineered mutants are included. Examples of useful peroxidases include peroxidases from Coprinus, e.g. from C. cinereus, and variants thereof as those described in WO 93/24618, WO 95/10602, and WO 98/15257.<br>
Commercially available peroxidases include Guardzyme™ (Novozymes A/S). Detergents<br>
The detergent enzyme(s) may be included in a detergent composition by adding separate additives containing one or more enzymes, or by adding a combined additive comprising all of these enzymes. A detergent additive of the invention, i.e. a separate additive or a combined additive, can be formulated e.g. as a granulate, a liquid, a slurry, etc. Preferred detergent additive formulations are granulates, in particular non-dusting granulates, liquids, in particular stabilized liquids, or slurries.<br>
Non-dusting granulates may be produced, e.g., as disclosed in US 4,106,991 and 4,661,452 and may optionally be coated by methods known in the art. Examples of waxy coating materials are poly(6thylene oxide) products (polyethylene glycol, PEG) with mean molar weights of 1000 to 20000; ethoxylated nonylphenols having from 16 to 50 ethylene oxide units; ethoxylated fatty alcohols in which the alcohol contains from 12 to 20 carbon atoms and in which there are 15 to 80 ethylene oxide units; fatty alcohols; fatty acids; and mono- and di- and triglycerides of fatty acids. Examples of film-forming coating materials suitable for application by fluid bed techniques are given in GB 1483591. Liquid enzyme preparations may, for instance,<br>
32<br><br>
be stabilized by adding a polyol such as propylene glycol, a sugar or sugar alcohol, lactic acid or boric acid according to established methods. Protected enzymes may be prepared according to the method disclosed in EP 238,216.<br>
The detergent composition of the invention may be in any convenient form, e.g., a bar, a tablet, a powder, a granule, a paste or a liquid. A liquid detergent may be aqueous, typically containing up to 70 % water and 0-30 % organic solvent, or non-aqueous.<br>
The detergent composition comprises one or more surfactants, which may be non-ionic including semi-polar and/or anionic and/or cationic and/or zwitterionic. The surfactants are typically present at a level of from 0% to 60% by weight.<br>
When included therein the detergent will usually contain from about 0% to about 40% of an anionic surfactant such as linear alkylbenzenesulfonate, alpha-olefinsulfonate, alkyl sulfate (fatty alcohol sulfate), alcohol ethoxysulfate, secondary alkanesulfonate, alpha-sulfo fatty acid methyl ester, alkyl- or alkenylsuccinic acid or soap.<br>
When included therein the detergent will usually contain from about 0% to about 40% of a non-ionic surfactant such as alcohol ethoxylate, nonylphenol ethoxylate, alkylpolyglycoside, alkyldimethylamineoxide, ethoxylated fatty acid monoethanolamide, fatty acid monoethanolamide, polyhydroxy alkyl fatty acid amide, or N-acyl N-alkyI derivatives of glucosamine ("glucamides").<br>
The detergent may contain 0-65 % of a detergent builder or complexing agent such as zeolite, diphosphate, triphosphate, phosphonate, carbonate, citrate, nitrilotriacetic acid, ethylenediaminetetraacetic acid, diethylenetriaminepentaacetic acid, alkyl- or alkenylsuccinic acid, soluble silicates or layered silicates (e.g. SKS-6 from Hoechst).<br>
The detergent may comprise one or more polymers. Examples are carboxymethylcellulose, poly(vinylpyrrolidone), poly (ethylene glycol), poly(vinyl alcohol), poly(vinylpyridine-N-oxide), poly(vlnyiimidazole), polycarboxylates such as polyacrylates, maleic/acrylic acid copolymers and lauryl methacrylate/acrylic acid copolymers.<br>
The detergent may contain a bleaching system which may comprise a H202 source such as perborate or percarbonate which may be combined with a peracid-forming bleach activator such as tetraacetylethylenediamine or nonanoyloxybenzenesulfonate. Alternatively, the bleaching system may comprise peroxyacids of e.g. the amide, imide, or sulfone type.<br>
The enzyme(s) of the detergent composition of the invention may be stabilized using conventional stabilizing agents, e.g., a polyol such as propylene glycol or glycerol, a sugar or sugar alcohol, lactic acid, boric acid, or a boric acid derivative, e.g., an aromatic borate ester, or<br>
33<br><br>
a phenyl boronic acid derivative such as 4-formylphenyl boronic acid, and the composition may be formulated as described in e.g. WO 92/19709 and WO 92/19708.<br>
The detergent may also contain other conventional detergent Ingredients such as e.g. fabric conditioners including clays, foam boosters, suds suppressors, anti-corrosion agents, soil-suspending agents, anti-soil redeposltion agents, dyes, bactericides, optical brighteners, hydrotropes, tarnish inhibitors, or perfumes.<br>
It is at present contemplated that in the detergent compositions any enzyme, in particular the enzyme of the invention, may be added in an amount corresponding to 0.001-100 mg of enzyme protein per litre of wash liquor, such as 0.01-50 mg or 0.03-30 mg, preferably 0.05-5 mg of enzyme protein per litre of wash liquor, in particular 0.1-1 mg of enzyme protein per litre of wash liquor.<br>
The enzyme of the invention may additionally be incorporated in the detergent formulations disclosed in WO 97/07202, WO 04/041979, WO 04/074419, which is hereby incorporated as reference.<br>
Uses<br>
The present invention is also directed to methods for using the polypeptides having lipase activity.<br>
The variants of the invention can be used In known applications of lipolytic enzymes by analogy with the prior art, e.g.:<br>
A variant with lipase activity can be used in the pulp and paper industry, to remove pitch or to remove ink from used paper. WO 9213130, WO 9207138, JP 2160984 A, EP 374700.<br>
A variant with phospholipase and/or galactolipase activity can be used in the preparation of dough, bread and cakes, e.g. to increase dough stability and dough handling properties, or to improve the elasticity of the bread or cake. WO 94/04035, WO 00/32758.<br>
A variant with phospholipase activity can be used in a process for reducing the content of phospholipid in an edible oil. US 5,264,367 (Metallgesellschaft, R6hm); K. Dahike &amp; H. Buchold, INFORM, 6 (12), 1284-91 (1995); H. Buchold, Fat Sci. lechnol., 95 (8), 300-304 (1993); JP-A 2-153997 (Showa Sangyo); or EP 654,527 (Metallgesellschaft, Rohm).<br>
A variant with lysophosphollpase activity can be used to improve the filterabil-ity of an aqueous solution or slurry of carbohydrate origin, e.g. starch hydrolysate, especially a wheat starch hydrolysate. EP 219,269.<br>
A variant with phospholipase activity can be used for the preparation of lyso-phospholipid, e.g. lyso-lecithin (EP 870840, JP-A 10-42884, JP-A 4-135456 or JP-A 2-49593) of for the<br>
34<br><br>
production of mayonnaise (EP 628256, EP 398666 or EP 319064).<br>
A variant with phospholipase activity may also be used in the processing of dairy and other food products, e.g. as described in EP 567,662 (Nestle), EP 426,211 (Unilever), EP 166,284 (Nestle), JP-A 57-189638 (Yakult) or US 4,119,564 (Unile-ver).<br>
A variant with phospholipase activity can be used in the leather industry. GB 2233665, EP 505920.<br>
A variant with lipase activity may be used for removing fatty matter containing hydrophobic esters (e.g. triglycerides) during the finishing of textiles. WO 93/13256.<br>
The present invention is further described by the following examples which should not be construed as limiting the scope of the invention.<br>
EXAMPLES<br>
Chemicals used as buffers and substrates were commercial products of at least reagent grade.<br>
Media and Solutions<br>
Product     Tradename "<br>
LAS :	Surfac PS<br>
Zeolite A   Wessalith P<br>
Materials<br>
Product     Supplier<br>
EMPA221  EMPA St. Gallen, Lerchfeldstrasse 5, CH-9014 St. Gallen, Switzerland<br>
Example 1 Production of enzyme<br>
A plasmid containing the gene encoding the lipase is constructed and transformed into a suitable host cell using standard methods of the art.<br>
Fermentation is carried out as a fed-batch fermentation using a constant medium temperature of 34°C and a start volume of 1.2 liter. The initial pH of the medium is set to 6.5. Once the pH has increased to 7.0 this value is maintained through addition of 10% H3P04. The level of dissolved oxygen in the medium is controlled by varying the agitation rate and using a<br>
35<br><br>
fixed aeration rate of 1.0 liter air per liter medium per minute. The feed addition rate is maintained at a constant level during the entire fed-batch phase.<br>
The batch medium contains maltose syrup as carbon source, urea and yeast extract as nitrogen source and a mixture of trace metals and salts. The feed added continuously during the fed-batch phase contains maltose syrup as carbon source whereas yeast extract and urea is added in order to assure a sufficient supply of nitrogen.<br>
Purification of the lipase may be done by use of standard methods known in the art, e.g. by filtering the fermentation supernatant and subsequent hydrophobic chromatography and ion exchange chromatography, e.g. as described in EP 0 851 913 EP, Example 3.<br>
Example 2<br>
AMSA - Automated Mechanical Stress Assay - for calculation of RP<br>
The enzyme variants of the present application are tested using the Automatic Mechanical Stress Assay (AMSA). With the AMSA test the wash performance of a large quantity of small volume enzyme-detergent solutions can be examined. The AMSA plate has a number of slots for test solutions and a lid firmly squeezing the textile swatch to be washed against all the slot openings. During the washing time, the plate, test solutions, textile and lid are vigorously shaken to bring the test solution in contact with the textile and apply mechanical stress. For further description see WO 02/42740 especially the paragraph "Special method embodiments" at page 23-24. The containers, which contain the detergent test solution, consist of cylindrical holes (6 mm diam, 10 mm depth) in a metal plate. The stained fabric (test material) lies on the top of the metal plates and is used as a lid and seal on the containers. Another metal plate lies on the top of the stained fabric to avoid any spillage from each container. The two metal plate together with the stained fabric are vibrated up and down at a frequency of 30 Hz with an amplitude of 2 mm.<br>
The assay is conducted under the experimental conditions specified below:<br><br>
Test solution	0.5 g/l LAS 0.52 g/l Na2C03 1.07 g/l Zeolite A 0.52 g/l NaSCitrate<br>
Test solution volume	160 micro 1<br>
PH	As is (»9.9)<br>
36<br><br>
Wash time	20 minutes<br>
Temperature	30°C<br>
Water hardness	15°dH<br>
Ratio of Ca’VMg’VNaHCOs: 4:1:7.5<br>
Enzyme concentration in test solution	0.125, 0.25, 0.50,1.0 mgep/I<br>
Drying	Wash performance: After washing the textile pieces is immediately flushed in tap water and air-dried at 85C in 5 min<br>
Odour: After washing the textile pieces is immediately flushed in tap water and dried at room temperature (20°C) for 2 hours<br>
Test material	Cream turmeric swatch as described below (EMPA221 used as cotton textile)<br>
Table 1<br>
Cream-turmeric swatches were prepared by mixing 5 g of turmeric (Santa Maria, Denmark) with 100 g cream (38% fat, Aria, Denmark) at 50'C, the mixture was left at this temperature for about 20 minutes and filtered (50'C) to remove any un-dissolved particles. The mixture is cooled to 20°C and woven cotton swatches, EMPA221, were immersed in the cream-turmeric mixture and aftenwards allowed to dry at room temperature over night and frozen until use. The preparation of cream-turmeric swatches is disclosed in the patent WO 2006/125437.<br>
The performance of the enzyme variant is measured as the brightness of the colour of the textile samples washed with that specific enzyme variant. Brightness can also be expressed as the intensity of the light reflected from the textile sample when laminated with white light. When the textile is stained the intensity of the reflected light is lower, than that of a clean textile. Therefore the intensity of the reflected light can be used to measure wash performance of an enzyme variant.<br>
Color measurements are made with a professional flatbed scanner (PFU DL2400pro), which is used to capture an image of the washed textile samples. The scans are made with a resolution of 200 dpi and with an output color depth of 24 bits. In order to get accurate results, the scanner is frequently calibrated with a Kodak reflective ITS target.<br><br>
To extract a value for the light intensity from the scanned images, a special designed software application is used (Novozymes Color Vector Analyzer). The program retrieves the 24 bit pixel values from the image and converts them into values for red, green and blue (RGB). The intensity value (Int) is calculated by adding the RGB values together as vectors and then taking the length of the resulting vector:<br>
Int=‘Jr' +g’ +b’<br>
The wash performance (P) of the variants is calculated in accordance with the below formula:<br>
P      =   Int(v) - Int(r) where<br>
Int(v) is the light intensity value of textile surface washed with enzyme, and Int(r) is the light intensity value of textile surface washed without enzyme.<br>
A relative performance score is given as the result of the AMSA wash in accordance with the definition:<br>
Relative Performance scores (RP) are summing up the performances (P) of the tested enzyme variants against the reference enzyme:<br>
RP = P(test enzyme) / P(reference enzyme).<br>
RPavg indicates the average relative performance compared to the reference enzyme at all three enzyme concentrations (0.125, 0.25, 0.5,1.0 mg ep/l)<br>
RPavg = avg(RP(0.125), RP(0.25) RP(0.5), RP(I.O))<br>
A variant is considered to exhibit improved wash performance, if it performs better than the reference.<br>
In the context of the present invention the reference enzyme is the mature part of SEQ ID N0:2 with the substitutions T231R + N233R.<br>
Example 3<br>
GC - Gas Chromatograph - for calculation of risk factor<br>
The butyric acid release from the lipase washed swatches were measured by Solid Phase Micro Extraction Gas Chromatography (SPME-GC) using the following method. Four textile pieces (5 mm in diameter), washed in the specified solution in Table 1 containing 1 mg/ L lipase, were transferred to a Gas Chromatograph (GC) vial. The samples were  <br><br>
  followed by 20 min sampling with the SPME fibre in the head-space over the textile pieces. The sample was subsequently injected onto the column (injector temperature = 250°C). Column flow = 2 ml Helium/min. Column oven temperature gradient: 0 min = 40°C, 2 min = 40°C, 22 min = 240°C, 32 min = 240'C. The butyric acid was detected by FID detection and the amount of butyric acid was calculated based on a butyric acid standard curve.<br>
The Risk Performance Odour, R, of a lipase variant is the ratio between the amount of released butyric acid from the lipase variant washed swatch and the amount of released butyric acid from a swatch washed with the mature part of the lipase of SEQ ID NO: 2, after both values have been corrected for the amount of released butyric acid from a non-lipase washed swatch. The risk (R) of the variants is calculated in accordance with the below formula:<br>
Odour =measured in micro g buturic acid developed at 1 mg enzyme protein / 1<br>
corrected for blank<br>
Alhpa test enzyme =  fest enzyme " Blank<br>
Alpha reference enzyme ~ '-'"0LIr reference enzyme " DianK<br>
K	=      test enzyme   ' "'Pna reference enzyme<br>
A variant is considered to exhibit reduced odor compared to the reference, if the R factor is lower than 1.<br>
Example 4<br>
Activity (LU) relative to absorbance at 280nm<br>
The activity of a lipase relative to the absorbance at 280 nm is determined by the following assay:<br>
LU/A280:<br>
A substrate for lipase is prepared by emulsifying tributyrin (glycerin tributyrate) using gum Arabic as emulsifier. The hydrolysis of tributyrin at 30 °C at pH 7 or 9 is followed in a pH-stat titration experiment. One unit of lipase activity (1 LU) equals the amount of enzyme capable of releasing 1 micro mol butyric acid/min at pH 7.<br>
The absorbance of the purified lipase at 280 nm is measured (A280) and the ratio LU/A280 is calculated. The relative LU/A280 is calculated as the LU/A280 of the variant divided by the LU/A280 of a reference enzyme. In the context of the present invention the reference enzyme is the mature part of SEQ ID N0:2 with the mutations T231R and N233R.<br>
39<br><br><br><br>
The invention described and claimed herein is not to be limited in scope by the specific aspects herein disclosed, since these aspects are intended as illustrations of several aspects of the invention. Any equivalent aspects are intended to be within the scope of this invention. Indeed, various modifications of the invention in addition to those shown and described herein will become apparent to those skilled in the art from the foregoing description. Such modifications are also intended to fall within the scope of the appended claims. In the case of conflict, the present disclosure including definitions will control.<br>
Various references are cited herein, the disclosures of which are incorporated by reference in their entireties.<br><br><br><br><br>
CLAIMS<br>
1.	A polypeptide having lipase activity and which further has a Average Relative Performance (RPavg) of at least 0.8 and a Benefit Risk (BR) of at least 1.1 at the test conditions given In the specification.<br>
2.	The polypeptide of claim 1, which further has a relative LU/A280 less than 1.00 at the test conditions given in the specification.<br>
3.	The polypeptide of claim 1 which is a bacterial polypeptide <br>
4.	The polypeptide of claim 1 which Is a fungal polypeptide.<br>
5.	The polypeptide of claim 4 which is a Thermomyces polypeptide.<br>
6.	The polypeptide of claim 5 which is a Thermomyces lanuginosus polypeptide.<br>
7.	The polypeptide of claim 1 which is a variant of a lipase comprised by the polypeptide of SEQ ID NO: 2<br>
8.	The polypeptide of claim 1 which is a variant of a lipase comprised by the mature part of the polypeptide of SEQ ID NO: 2.<br>
9.	The polypeptide of claim 1 which is a variant of a lipase comprising the polypeptide of SEQ ID NO: 2<br>
10.	The polypeptide of claim 1 which is a variant of a lipase comprising the mature part of the polypeptide of SEQ ID NO: 2.<br>
11.	The polypeptide of claim 1, which is encoded by a polynucleotide which hybridizes under at least high stringency conditions with nucleotides 644 to 732 of SEQ ID NO: 1  or a<br>
complementary strand hereto.<br>
12.	An  Isolated  polynucleotide  comprising  a  nucleotide  sequence which  encodes the<br><br>
polypeptide of any of the preceding claims.<br>
13.	A nucleic acid construct comprising the polynucleotide of claim 12 operably linked to one<br>
or more control sequences that direct the production of the polypeptide in an expression host.<br>
14.	A recombinant expression vector comprising the nucleic acid construct of claim 13.<br>
15.	A recombinant host cell comprising the nucleic acid construct of claim 14.<br>
16.	A method for producing the polypeptide of any of claims 1 -12 comprising<br>
(a)	cultivating a cell, which in its wild-type form is capable of producing the polypeptide,<br>
under conditions conducive for production of the polypeptide; and<br>
(b)	recovering the polypeptide.<br>
17.	A method for producing the polypeptide of any of claims 1 -12 comprising<br>
(a)	cultivating a host cell comprising a nucleic acid construct comprising a nucleotide<br>
sequence encoding the polypeptide under conditions conducive for production of the<br>
polypeptide; and<br>
(b)	recovering the polypeptide.<br><br></td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
		<br>
		<div class="pull-left">
			<a href="264465-crosslaminate-of-oriented-films-and-methods-and-apparatus-for-manufacturing-same.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="264467-spatially-correlated-audio-in-multipoint-videoconferencing.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>264466</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>3863/CHENP/2008</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>01/2015</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>02-Jan-2015</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>30-Dec-2014</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>23-Jul-2008</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>NOVOZYMES, INC.</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td>1445 DREW AVENUE, DAVIS, CA 95616</td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>VIND, JESPER,</td>
											<td>HEJREBAKKEN 20, DK-3500 VAERLOSE</td>
										</tr>
										<tr>
											<td>2</td>
											<td>BORCH, KIM,</td>
											<td>VANDTAARNSVEJ 18, DK-3460 BIRKEROED</td>
										</tr>
										<tr>
											<td>3</td>
											<td>MIKKELSEN, MIKAEL,</td>
											<td>VALMUEHAVEN 44, DK-2765 SMORUM</td>
										</tr>
										<tr>
											<td>4</td>
											<td>KNOTZEL, JURGEN, CARSTEN, FRANZ,</td>
											<td>OSTERBROGADE 226, 2, TH., DK-2100 COPENHAGEN</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>C12N9/20</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>PCT/US07/60825</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td>2007-01-22</td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td>60/761,106</td>
									<td>2006-01-23</td>
								    <td>U.S.A.</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/264466-polypeptides-having-lipase-activity-and-polynucleotides-encoding-same by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 01:31:43 GMT -->
</html>
