<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/264493-method-of-determining-the-haplotype-of-multiple-allelic-genes by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 01:38:54 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 264493:METHOD OF DETERMINING THE HAPLOTYPE OF MULTIPLE ALLELIC GENES</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">METHOD OF DETERMINING THE HAPLOTYPE OF MULTIPLE ALLELIC GENES</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>ABSTRACT In amplifying a region containing two allelic genes, the polymorphism of one of the allelic genes is positioned at or near the 3&#x27; end of an amplification primer, so that the chain would extend only from one of the two primers, one that matches the mutant type nucleotide(s) and one that matches the wild-type nucleotide {s) . The other primer is unaffected by the allelic genes but is positioned to include the other allelic gene in the amplicon. The PCR amplifies only the target genomic DNA having one specific allele of the first allelic gene. The technique exploits the fact that the haplotype of the first and the second allelic genes can be determined if the allele of the second allelic gene in the amplicon can be identified. Designing probes with complementary sequences for identifying the two alleles (major and minor) of the second allelic gene, and fixing the probe set on a solid support and hybridizing them with the amplicon to identify the probe that forms a hybrid, are characteristic features of the present invention- 30</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td> <br><br>
DESCRIPTION<br>
METHOD OF DETERMINING THE HAPLOTYPE OF MULTIPLE<br>
ALLELIC GENES<br>
TECHNICAL FIELD<br>
The present invention relates to a method of determining the haplotype of a target nucleic acid ha.ving  at   least two   heterozygous   polymorphisms,    and the primer set used for such determination.<br>
BACKGROUND ART<br>
A haplotype is the cis arrangement of two oi: more polymorphisms located on a single chromosome. Haplotype information is considered to be very useful because the haplotype is related to certain diseases or abnormalities, and specific drug sensitivities. Until now, haplotype analysis required tracking studies of genetic information about polymorph!sms covering several generations of a family, and computing machine-based estimation algorithms.  On the other hand, the advances in polymerization chain reaction (PCR) technology have enabled direct, molecular level analysis of DNA, and the use of the results of such analysis for determining the haplotype has been suggested.  However, for determining the haplotype using PCR, it is necessary<br><br>
to use multiple combinations of primers specific to the allelic genes, and to carry out PCR a number of times.  If we consider only 2 allelic genes, there have been attempts to determine the haplotype by positioning 2 primer pairs at the polymorphic sites and examining whether any am.pli f ication occurred. This method has been suggested for determination of the haplotype with a single test tube assay (see Japanese Patent Application Laid-Open No. 2002-272482).  To be more sped fie, a forward primer containing the first polymorphism is modified with a label that yields different signals depending on the allele type on the 5' end.  Moreover, for the reverse primer containing the second polymorphism, a flap sequence that is not present in the target sequence is attached to the 5' end, so that the primer is designed to yield an ampiicon having a length that depends on the allele.  Using these primers, allele-specific PCR assay is carried out from both the directions.  Which primer pair caused amplification, can be known by detecting the fluorescent label in the PCR ampiicon and from the length of the ampiicon, and thus the haplotype can be determined.<br>
In the above described known method, however, it is essential that the primers contain sites with the polymorphic sequences.  This restricts the freedom of selecting the base sequence  of   the primers.  Here, to<br><br>
carry out allele-speci f ic PCR, the Tn,s of the primers used for the PCR have to be similar.  However, with the above described restrictions in selecting the primer base sequences, it may  be difficult in some cases to make the T^s of the primers similar.  In other words, with the method of the Japanese Patent Application Laid-Open Ho. 2002-272482, highly accurate detection of the haplotype is likely to be restricted to cases where the polymorphisms are at locations for which primers of similar T^ can be used. Besides this, in this method of detection, it is difficult to shift the positions of the sequences of both the forward and reverse primers.  Thus, if the forward and reverse primers form a primer dimer, it becomes impossible to detect the haplotype.<br>
DISCLOSURE OF THE INVENTION<br>
The present invention provides a method of haplotype determination with better accuracy irrespective of the location of the polymorphisms. More specifically, the invention provides a method of determining the haplotype of the target nucleic acid having at least two heterozygous polymorphisms, and the primer set used for this purpose.<br>
The method of the present invention determines the haplotype of a target nucleic acid having a first heterozygous polymorphi c site on the 3' side and a<br><br><br>
second heterozygous polymorphic site on the 5' side, and the method is characterized by including (i) carrying out PCR of the target nucleic acid using one of the forward primers (a-l), i.e.,<br>
(a-1-1) a forward primer that contains a base sequence complementary to the first polymorphic site and can extend the target nucleic acid if the first polymorphism is the mutant type, and does not extend the target nucleic acid if the first polymorphism is the wild type, and<br>
(a-1-2) a forward primer that contains a base sequence complementary to the first polymorphic site and can extend the target nucleic acid if the first polymorphism is the wild type, and does not extend the target nucleic acid if this first polymorphism is the mutant type, and<br>
(b-1) a reverse primer that contains a base sequence complimentary to a certain base sequence that does not contain the second polymorphism, and is located on the 3' side of the second polymorphism, on the complimentary strand of the target nucleic acid; and<br>
(ii) hybridizing the product obtained as a result of (i) with a first probe that has a base sequence identical with a certain segment of the base sequence of the target nucleic acid, which contains the wild-type nucleotide of the second polymorphism, and a<br><br>
second probe that has a base sequence identical with a certain segment of the base sequence of the target nucleic acid, which contains the mutant nucleotide of the second polymorphism, and detecting the signals of hybrid formation with the first and second probes.<br>
The primer set used for determining the haplotype according to the present invention is a primer set for determining the haplotype of a target nucleic acid having the first polymorphic site on the 3' side and the second polymorphic site on   the 5' side, and the primer set is characterized by having one of a forward primers i.e.,<br>
a forward primer that contains a base sequence complementary to the first polymorphic site and can extend the target nucleic acid if the first polymorphism of the target nucleic acid is the mutant type, and does not extend the target nucleic acid if the first polymorphism of the target nucleic acid is the wild type, and I        a forward primer that contains a base sequence complementary to the first polymorphic site and can extend the   target   nucleic acid if the first polymorphism of the target nucleic acid is the wild type and does not extend if the first polymorphism is I  the mutant type, and<br>
a reverse primer that contains a base sequence complimentary to a certain base sequence that does<br><br><br>
not contain The second polymorphism, and is located on the 3' side of the second polymorphism, on the complimentary strand of the target nucleic acid.<br>
As described above, the present invent ion can simultaneously and exhaustively determine the haplotype when multiple heterozygous polymorph!sms are present.<br>
BRIEF DESCRIPTION OF THE DRAWING<br>
FIG. 1 is a flowchart of the determination of haplotypes of two allelic genes in the first embodiment of the present invention;<br>
FIG, 2 is a diagram that illustrates the arrangement of the two allelic genes in the first embodiment of the present invention;<br>
FIG. 3 is a diagram that illustrates the protocol used for the PCR amplification for extracting specific segments of the genome in the first embodiment of the present invention;<br>
FIG. 4 is a diagram that illustrates the protocol used for the PCR ampli floation in the first embodiment of the present invention; and<br>
FIG. 5 is a diagram that illustrates the arrangement of two or more multiple allelic genes in the second embodiment of the invention.<br>
BEST MODES FOR CARRYING OUT THE INVENTION<br><br><br>
In the present invention, an allelic gene-specific PCR amplification protocol is used for the polymorphisms of interest in the genomic DNA targeted for determining the haplotype with a microarray. Firstly, an amplicon that is specific to one of the alleles of the first allelic gene, and also contains the second allelic gene, is produced.  This amplicon is then hybridized with oligoneucleotide probes, each specific to one of the alleles of the second allelic gene, to identify the allele of the second allelic gene.  This technique can be a very powerful method of detection at the molecular level when the first and second allelic genes form a haplotype.  A complementary strand probe that forms a DNA hybrid when the second allelic gene is the major allele, and another complimentary strand probe that forms a DNA hybrid if the second allelic yene is the minor allele, are prepared and fixed on a solid support, such as a microarray.  The haplotype can be determined by verifying, with a label, the probe with which hybrid!zation occurs.  When there are multiple pairs of allelic gene candidates, the N haplotypes can be analyzed exhaustively by hybridizing with Nx2 probes loaded on a solid support.  If the first allelic gene is a pre-selected heterozygous one, exclusive solid supports can be prepared for identifying the haplotypes.  One of the amp11fication primers can be<br><br><br>
designed in such a way that it becomes specific to one of the alleles of one allelic gene, and another amplification primer must be designed to include the other allelic genes 'in the amplicon.  The label can, for instance, be a fluorophore carried on the primer and structured in such a way that the fluorescence can be detected in the probe hybridized with the other allelic gene only when allele-specific amplification has occurred successfully.<br>
One of the forward primers described below is used in the present invention to obtain an amplicon having the region specific to one of the alleles of the first allelic gene.<br>
(a-l-l] A forward primer that contains a base sequence complementary to the above described first polymorphic site and can extend the target nucleic acid if the first polymorphism of the target nucleic acid is the mutant type, and does not extend if the first polymorphism is the wild type.<br>
(a-1-2) A forward primer that contains a base sequence complementary to the above described first polymorphic site and can extend the target nucleic acid if the first polymorphism of the target nucleic acid is the wild type, and does not extend if the fiist polymorphism is the mutant type.<br>
In these forward primers, the base sequence of the primer [and/or the length of the sequence) can be<br><br>
set by a standard method on the basis of the sequence of a segment containing the known first polymorphism, based on typing of the target nucleic acid, in such a manner that this segment would be obtained in the PCR amplicon.<br>
Either the mutant type or the wild-type polymorphism can be selected.<br>
The reverse primer for obtaining, in the amplicon, the segment containing the second polymorphism is designed to have a base sequence complementary to a certain base sequence that does not contain the second polymorphism, from the 3' side of the second polymorphism on the complementary strand of the target nucleic acid.  The base sequence (sequence length) of the reverse primer can be set by a standard method, using as basis, the sequence of the segment containing tne known second polymorphism, based on typing of the target nucleic acid, and in such a way that this segment can be obtained in the PCR amplicon.<br>
With the probes used for detecting the segment containing the second polymorphism also, a sequence specific to this segment can be selected and the base sequence (and/or the sequence length) of the probe that can detect the selected sequence can be set by the standard method, as the segment containing the second polymorphism is already known from typing of<br><br><br>
the target nucleic acid.<br>
The embodiments of the present invention are described below. First embodiment<br>
The first embodiment of this invention is described below referring to FIG. 1.  This embodiment concerns   the   technique   of   determining the allele types of mult iple allelic genes on the genomic DNA, More specifically, the procedure of determining the haplotype of 2 or more than 2 allelic genes is described below sequentially.<br>
In this embodiment, the haplotypes of two heterozygous allelic genes on the genomic DNA are determined.  The distance between the two allelic )  genes may range from about a few base pairs to a few tens of base pairs to as much as a few hundred to a few thousand base pairs.  Therefore, it is necessary to prepare an amp1icon having both the allelic genes. The two allelic genes are defined here as the first J  allelic gene and the second allelic gene, counting from the 5' side.  The major allele of the first allelic gene is G and the minor allele is C, and the major allele of the second allelic gene is G and the minor  allele   is   T.        For determining the haplotype in this embodiment, a primer's 3' end is matched to the locus of the allele to selectively amplify either the major or the<br><br><br>
minor allele of the first allelic gene.  For example, the first primer (the forward primer) is designed to be complementary to the minor allele of the first allelic gene at its 3' end so that extension would occur if the first allelic gene is the minor allele and extension would not proceed if it were the maj or allele.  On the other hand, the second primer (reverse primer) of the opposite strand is designed to interpose the second allelic gene using the binding position of the first primer as the reference point. The second primer does not depend on the allele type. With this arrangement, if the first allelic gene is the minor allele, a large amount of amplicon is produced, which confirms that the first allelic gene is the minor allele, and the amplicon containing the second allelic gene has been preferentially produced. Therefore, by determining the allele type of the second allelic gene in the amplicon, we can determine the haplotype of the first and the second allelic genes.<br>
The method of determining the haplotype according to the present invention includes the steps 1) and 2) described below:<br>
1)	selecting one of the alleles of the first allelic gene, and producing the amplicon containing the second allelic gene only;<br>
2)	hybridizing the amplicon with complementary<br><br><br>
probes specific to the alleles for the second allelic gene, and thereby identifying the allele of the second allelic gene.<br>
FIG. 1 is a flowchart illustrating the above described steps.  S-1 is a step to confirm that both the first and second allelic genes are heterozygous. There is no restriction on the method of typing used for each allelic gene.  If at least one out of the first and second allelic genes is homozygous, the haplotype combination of the two allelic genes becomes known, and there is no need to apply the present invention.<br>
Only the combinations where the heterozygosity has been confirmed after each genotyping are targeted in this embodiment.  S-2 is the step in which the amplicon is obtained from the target genome through the combined use of a first primer specific to, the minor allele of the first allelic gene and a second primer of the opposite strand for including the second allelic gene.  The first primer specific to the minor allele of the first allelic gene is a primer in which the mutant site is positioned at its 3' end so that it is complementary, and the primer does not get extended if the target genomic sequence has the major allele.  Here, as the first allelic gene is heterozygous, the extension reaction advances only when the genome of the minor allele and the first<br><br><br>
primer get annealed.  In other words, the amplicon formed would be limited to one having the minor allele of the first allelic gene.  The second primer is not influenced by the target genome, and will produce an amplicon containing the second allelic gene matching its configuration in the target genome. In this case, where the first allelic gene is the minor allele, an amplicon containing the matching allele of the second allelic gene is formed.  Whether the second allelic gene is the major or minor allele does not have any effect, but the haplotype with the first allelic gene can be determined by hybridizing the amplicon with probes specific to the alleles of the second allelic gene.  The essence of the present invention remains the same even when the first primer is designed to be specific to the maj or allele, unlike in the above described case.<br>
A label also needs to be used for determining the second allelic gene.  Either the first or the second primer can be labeled.  Alternatively, a labeled nucleotide is incorporated into the amplicon in the course of the extension reaction.  The label can be a fluorophore, a chemiluminescent substance, or a radioisotope.  A Cy3 fluorescent dye label is used in this embodiment.  S-3 is a step where the amplicon is allowed to hybridize with the solid support on which probes specific to the second<br><br><br>
allelic gene have been fixed.  Probes, each specific to the major or the minor allele, are prepared.  These probes can be fixed on a solid support or used in an arrangement that would cause FRET in a liquid phase reaction.  In this embodiment, the second allelic gene is determined using the solid support.  In other words, if fluorescence is seen only at the particular location on the solid support where the probe specific for the minor allele of the second allelic gene is positioned, this means that the first and second allelic genes form a minor/minor haplotype.  At the same t ime, this also confirms the existence of the major/major haplotype.  On the other hand, if fluorescence is seen only at the particular location on the solid support where the probe specific for the major allele of the second allelic gene is positioned, this means that the first and second allelic genes form a minor/major haplotype, which simultaneously confirms the existence of the major/minor haplotype. Moreover, if fluorescence is seen at both the sites on the solid support, where the two probes, one specific to the major allele and the other to the minor allele of the second allelic gene, are positioned, it should be interpreted that the first and second allelic genes have weak haplotype association.  Thus, S-4 is the step where the alleles of the second allelic gene and the haplotype of the<br><br><br>
first and second allelic genes are determined.<br>
FIG. 2 is a diagram illustrating the positional relationship of the above described first allelic gene 201 and the second allelic gene 202 on the target genome.  The first primer 203 and the second primer 204 and their amplicon 205 are also illustrated in the diagram.  As described earlier, the 3' end of the first primer 203 corresponds to the mutant site of the first allelic gene 201, and the second primer 204 is positioned downstream of the region that contains the second allelic gene 202. Because of the first primer 203, the production of the extension product, which contains one of the alleles of the first allelic gene 201 (upper part of the FIG.), is greater than the production of the extension product containing the other allele (lower part of the FIG.).  In other words, the PCR amplifies only that side of the target genome that contains one of the two alleles of the first allelic gene 201. Therefore, the second allelic gene 202 will also have the allele matching only that side of the target genome.  Besides this, the probes are designed to contain the alleles of the second allelic gene 202 shown in the diagram.<br>
Table 1 shows the base sequences of a pair of amplification primers, their amplicon, and the probes for the second allelic gene, all pertaining to this<br>
i&amp;<br><br>
embodiment.  Starting from the 5' side, the bold characters correspond to the allelic locus (mutant nucleotide C) of the first allelic gene, and the allelic locus (mutant nucleotide T) of the second allelic gene, in that order.  The mutant nucleotide C has been selected for the forward primer (FP).<br><br>
Second embodiment<br>
The second embodiinent of the invention is described below, referring to FIG. 5.  The polymorphic locus of each allelic gene is marked with "x".  Only the polymorphisms at the loci N, N+1, and N+2 are illustrated here.  Others are omitted, which does not imply any limits on the number of polymorphisms.  In this case, the forward primer is Cy3-labeled, but the essential features of the invention would of course not change even when it is the reverse primer or the extending nucleotide chain itself that is labeled.<br>
i1<br><br>
For this embodiment, we shall describe a generalized technique for determining the haplotype of multiple allelic genes.  N number of heterozygous polymorphisms are tested here.  As can be seen in FIG. 5, FP(H)  (the forward primer for the Nth polymorphism) and RP(N) (the reverse primer for the opposite direction) are defined.  For N+1 onwards also, we can similarly define the forward primers FP (N+l), FP(N + 2), -.., and the reverse primers RP(N + 1), RP(N + 2),<br>
The forward primer FP(N) and the reverse primer RP(N+1) are designed to produce the PCR amplicon containing the Mth polymorphism and the N+lth polymorphism.  FP(N) is designed to have, at its 3' end, the Nth polymorphism, with one of the two alleles selected.  RP(N+1) is designed on the opposite side of the Nth polymorphism in such a way that the PCR amplicoii would contain the N+lth polymorphism. The primer pair of FP(N) and RP(N(-1) is the pair that would yield a PCR amplicon containing the Nth polymorphism and the N+lth polymorphism when the Nth polymorphism matches with only one of the two alleles. Reaction products are then obtained from m number of PCR amplifications, for N=l, 2, 3, ..., m, and hybridized with probes that can determine the N+lth allelic gene for the Nth allelic gene selected by FP{N).  In the case described here, the probes designed to match the 2xm alleles are fixed at the<br>
IS<br><br>
desired posit ions on the microarray.<br>
The amplicons obtained in the m PCR amplifications are mixed and hybridized with the probes on the microarray according the protocol described for the following Example 1.  By doing this, the haplotype combinations of multiple allelic genes can be determined exhaustively with a single labeling Moreover, the technique of this invention, i.e., identifying the allele by finding out which of the two probes, with sequences corresponding the two alleles of one allelic gene, undergoes hybridization, improves the accuracy of analyzing the haplotype.<br>
Here we have limited the discussion to heterozygous allelic genes.  Two sets of primers individually specific to two alleles of the first allelic gene can be prepared and hybridization carried out separately, for the determination. Examples Example 1<br>
An example according to the first embodiment is described below.<br>
I. Specimen preparation and extraction of a specific region of the genome<br>
The template DNA used was prepared by amplifying only a 5 kbp region of the metabolic enzyme gene CyP2D6 of the genomic DNA extracted from a B celJ line derived from a Japanese individual, and diluting<br>
/^<br><br>
the amplicon to 8 nq/\iL   with ultra pure water.  This processing was done to eliminate duplicated sequence regions, such as CYP2D7 and CYP2D3, which are pseudogenes, and to obtain the pure CYP2D6 region.  To be more specific, the generally, widely used primers listed in Table 2 were used.<br><br>
The PCR solution was prepared as shown in Table 3 and the PCR cycle was done as described in FIG. 3, and as a result, the amplicon (5079 bp) of the CYP2D6 gene region was obtained.<br>
In the reaction process, the 3-step cycle of denaturation, annealing, and extension was repeated 35 times, and the amplicon was purified by a purification process, after cooling.<br>
The PCR amplicon was purified on a purification column (Qiagen QIAquick PCR Purification Kit), and the volume of the PCR amplicon solution was adjusted to 50 ^iL.  A part of the purified PCR amplicon solution thus obtained was sampled and subjected to electrophoresis by a standard method, and it was confirmed from the size of the product that the desired PCR product had been synthesized.  The product was further diluted to 8 ng/)iL with pure water.<br><p0></p0><br><br>
II, PCR amplification<br>
Designing the primers and purification of the PCR<br>
amplicon<br>
Table 4 lists the reagents used in the amplification reaction and their mixing ratio.  The concentrations and the amounts used, of the PCR buffer, DNft polymerase, nucleotides, primers and template DNA are also listed here.  A Cy3-labelled primer was used here.<br><br>
PIG. 4 illustrates the protocol applied for the<br><br>
ampiification reaction.  In the reaction process, the 3-step cycle of denaturation, annealing, and extension was repeated 35 times, and the aruplicon was then purified, by a purification process, after cooling.<br>
The PCR amplicon was purified on a purification column (Qiagen QIAquick PCR Purification Kit).  After the purification, the volume of the PCR amplicon solution was adjusted to 50 f4.L.  A part of the purified PCR amplicon solution thus obtained was samipled and subjected to electrophoresis by a standard method, and it was confirmed from the size of the product that the desired PCR product had been synthesized.  The sequences of the primers used and the amplicon were as shown earlier in Table 1. III. Preparation of the microarray<br>
(1) Designing the probes<br>
Two probes were designed for the above described PCR product. As in the designing of the primers, the probes were designed with careful consideration so that each probe can specifically recognize the base sequence of the alleles of the second allelic gene. In this probe design, the DNA strand that extends from the forward primer FP is the one that forms the hybrid with a probe.  The base sequence of each probe was designed carefully, by adjusting the base pair length, etc taking the stability of the hybrid formed<br>
5^<br><br>
into account.  The sequences of the two designed probes are shown in Table 1.<br>
(2)    Synthesis of the probes and preparation of the microarray<br>
Synthesis of the probes and preparation of the microarray were carried out using the methods of preparing DNA microarrays disclosed by Canon Inc. (Japanese Patent Application laid-open No. 11-187900! In short, for processing the substrate, quartz glass was treated with a silane coupling agent, and EMC3 bound to it, to introduce maleimide groups on the surface.  As for probe synthesis, probes in which a thiol group was introduced at the 5' end were synthesized and purified by HPLC.  For preparing the DNA microarray, each probe was spotted on the glass substrate using a modified version of an ink jet printer (proprietary name BJF~850, manufactured by Canon Inc.).  The size of the glass substrate was 25 mmx 75 mmx 1 mm [Wx L x T). IV. Hybridization<br>
Hybridization on microarray was done using the DNA microarray prepared in III and the PCR amplicon prepared in II as the nucleic acid specimen sample.<br>
(1) Blocking the DNA microarray<br>
BSA (bovine serum albumin Fraction V, manufactured by Sigma Co.) was dissolved in 100 mM NaCi/10 mM phosphate buffer to the concentration 1<br><br><br>
vit.l.     The DNA microarray prepared in III was immersed in this solution for 2 hours at room temperature to get the glass substrate surface blocked.  After the blocking, it was washed with 2x SSC solution (NaCl 300 mM and sodium citrate (trisodium citrate dihydrate, i.e., C6H5Ha3 ■ 2H2O) 30 mM, pH 7.0) containing 0.1 wt.% SDS (sodium dodecyl sulfate) .  It was then rinsed with pure water.  After that the DHA microarray was dried in a spin-drier.<br>
(2)	Preparation of hybridization solution<br>
The hybridization solution was prepared using 2 \xL   of the PCR amp1icon solution to achieve the final concentration given below.  The hybridization solution had the following composition.<br>
6 X SSPE/10% formamide/PCR amplicon solution (6 X SSPE: NaCl 900 mM, NaH2PO4.H20 60 rtiM, and EDTA 6 iTiM, pH 7.4)<br>
(3)	Hybridization<br>
The dried DNA microarray was set in a hybridization apparatus (Hybridization Station, Genomic Solutions Inc.) and the hybridization reaction carried out using the hybridization solution with the above described composition, following the procedures and conditions given below.<br>
The hybridization conditions and procedures used were as follows:<br>
The hybridization solution was heated to 65°C,<br><br><br>
maintained at that temperature for 3 minutes, and held at 92°C for 2 mrnutes and then at 50°C for 4 hours.  After that, it was washed at 40°C, with 2x SSC containing 0.1% SDS.  It was further washed at 25°C, with 2x SSC and rinsed with pure water as needed, following the ordinary manual, and finally dried in a spin-drier.<br>
(4) Fluorescence measurement<br>
After completing the hybridization reaction, the fluorescence originating from the hybrid was measured on the spin-dried DNA microarray using a DNA microarray fluorescence scanner [Genepix 4000B, manufactured by Axon).  The results of measurement obtained for each probe are given below in Table 5. Table 5<br><br>
Probe type	Brightness of fluorescence (relative to the value at 535 nm)<br>
Majnr allele	530<br>
Minor allele	1800<br>
In calculating the brightness, the fluorescence intensity on parts of the DNA microarray with no probe DNA spot was taken as the background value, and the apparent fluorescence intensity of each spot minus the background value was taken as the measured fluorescence intensity.  The measurements were made twice, and tho mean values are given here.<br>
From the results, the haplotype of the first<br><br><br>
allelic gene and the second allelic gene was determined to be C/T (minor/minor!.  Also, because both the allelic genes were heterozygous, the diplotype was determined to be G/G (major/major). Here, we have limited ourselves to heterozygous allelic genes.  Two sets of primers individually specific to the two alleles of the first allelic gene may also be prepared and hybridization carried out separately,<br>
The present invent ion is not limited to the above embodiments and various changes and modifications can be made within the spirit and scope of the present invention.  Therefore to apprise the public of the scope of the present invention, the following claims are made.<br>
This application claims the benefit of Japanese Patent Application No. 2005-325951, filed December 1, 2006, which is hereby incorporated by reference herein in its entirety.<br><br><br><br>
CLAIMS<br>
1. A method of determining the haplotype of a target nucleic acid having a first heterozygous polymorphic site on the 3' side and a second heterozygous polymorphic site on the 5' side, which is characterized by comprising;<br>
(i) carrying out PCR of the target nucleic acid using one of the forward primers (a-l&gt; listed below,<br>
(a-1-11 a forward primer that contains a base sequence complementary to the first polymorphic site and can extend the target nucleic acid if the first polymorphism is the mutant type, and does not extend the target nucleic acid if the first polymorphism is the wild type, and<br>
[a-1-2) a forward primer that contains a base sequence complementary to the first polymorphic site and can extend the target nucleic acid if the first polymorphism is the wild type, and does not extend the target nucleic acid if the first polymorphism is the mutant type, and<br>
(b-1) a reverse primer that contains a base sequence complimentary to a certain base sequence that does not contain the second polymorphism, and is located on the 3' side of the second polymorphism, on the complimentary strand of the target nucleic acid; and<br><br>
(ii) hybridizing the product obtained as a result of (i) with a first probe that has a base sequence identical with a certain segment of the base sequence of the target nucleic acid, which contains the wild-type nucleotide (s) of the second polymorphism, and a second probe that has a base sequence identical with a certain segment of the base sequence of the target nucleic acid, which contains the mutant nucleotide(s) of the second polymorphism, and detecting the signals of hybrid formation with the first and second probes.<br>
2.	The method of determining the haplotype according to claim 1, wherein the hybridization (ii) further comprises hybridizing a probe carrier on which the first probe and the second probe are fixed with the product obtained as a result of (i).<br>
3.	The method of determining the haplotype according to claim 1 or 2, wherein the forward primer<br>
(a-1-1) has, at or near its 3' end, a nucleotide (s) complementary to the first polymorphism having a mutant type nucleotide(s1, and<br>
the forward primer (a-1-2) has, at or near its 3' end, a nucleotide (s) complementary to the first polymorphism having a wild type nucleotide(s) .<br>
4.	A primer set for determining the haplotype of<br>
a target nucleic acid having a first polymorphic site<br>
on the 3' side and a second polymorphic site on the<br><br><br>
5' side, wherein the primer set is characterized by comprising one of the forward primers, i.e., a forward primer that contains a base sequence complementary to the first polymorphic site, and can extend the target nucleic acid if the first polymorphism of the target nucleic acid is the mutant type, and does not extend the target nucleic acid if the first polymorphism of the target nucleic acid is the wild type, and<br>
a forward primer that contains a base sequence complementary to the first polymorphic site and can extend the target nucleic acid if the first polymorphism of the target nucleic acid is the wild type and does not extend if the first polymorphism of the target nucleic acid is the mutant type; and a reverse primer that contains a base sequence complimentary to a certain base sequence that does not contain the second polymorphism, and is located on the 3' side of the second polymorphism, on the complimentary strand of the target nucleic acid.<br><br></td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mzg0MyBDSEVOUCAyMDA5IFBldGl0aW9uIGZvciBQT1IucGRm" target="_blank" style="word-wrap:break-word;">3843 CHENP 2009 Petition for POR.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mzg0My1DSEVOUC0yMDA5ICAgQU1FTkRFRCBDTEFJTVMgICAwMS0xMS0yMDEzLnBkZg==" target="_blank" style="word-wrap:break-word;">3843-CHENP-2009   AMENDED CLAIMS   01-11-2013.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mzg0My1DSEVOUC0yMDA5ICAgQU1FTkRFRCBQQUdFUyBPRiBTUEVDSUZJQ0FUSU9OICAgMDEtMTEtMjAxMy5wZGY=" target="_blank" style="word-wrap:break-word;">3843-CHENP-2009   AMENDED PAGES OF SPECIFICATION   01-11-2013.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mzg0My1DSEVOUC0yMDA5ICAgQ09SUkVTUE9OREVOQ0UgT1RIRVJTICAyNS0wNC0yMDEzLnBkZg==" target="_blank" style="word-wrap:break-word;">3843-CHENP-2009   CORRESPONDENCE OTHERS  25-04-2013.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mzg0My1DSEVOUC0yMDA5ICAgRVhBTUlOQVRJT04gUkVQT1JUIFJFUExZIFJFQ0VJVkVEICAwMS0xMS0yMDEzLnBkZg==" target="_blank" style="word-wrap:break-word;">3843-CHENP-2009   EXAMINATION REPORT REPLY RECEIVED  01-11-2013.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mzg0My1DSEVOUC0yMDA5ICAgRVhBTUlOQVRJT04gUkVQT1JUIFJFUExZIFJFQ0lFVkVEIDAzLTEyLTIwMTQucGRm" target="_blank" style="word-wrap:break-word;">3843-CHENP-2009   EXAMINATION REPORT REPLY RECIEVED 03-12-2014.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mzg0My1DSEVOUC0yMDA5ICAgRk9STS0zICAwMS0xMS0yMDEzLnBkZg==" target="_blank" style="word-wrap:break-word;">3843-CHENP-2009   FORM-3  01-11-2013.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mzg0My1DSEVOUC0yMDA5ICAgT1RIRVJTICAwMS0xMS0yMDEzLnBkZg==" target="_blank" style="word-wrap:break-word;">3843-CHENP-2009   OTHERS  01-11-2013.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mzg0My1DSEVOUC0yMDA5ICBBTUVOREVEIENMQUlNUyAgMjEtMDItMjAxNC5wZGY=" target="_blank" style="word-wrap:break-word;">3843-CHENP-2009  AMENDED CLAIMS  21-02-2014.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mzg0My1DSEVOUC0yMDA5ICBBTUVOREVEIFBBR0VTIE9GIFNQRUNJRklDQVRJT04gIDIxLTAyLTIwMTQucGRm" target="_blank" style="word-wrap:break-word;">3843-CHENP-2009  AMENDED PAGES OF SPECIFICATION  21-02-2014.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mzg0My1DSEVOUC0yMDA5ICBDT1JSRVNQT05ERU5DRSBPVEhFUlMgIDA3LTA4LTIwMTQucGRm" target="_blank" style="word-wrap:break-word;">3843-CHENP-2009  CORRESPONDENCE OTHERS  07-08-2014.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mzg0My1DSEVOUC0yMDA5ICBDT1JSRVNQT05ERU5DRSBPVEhFUlMgIDExLTExLTIwMTQucGRm" target="_blank" style="word-wrap:break-word;">3843-CHENP-2009  CORRESPONDENCE OTHERS  11-11-2014.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mzg0My1DSEVOUC0yMDA5ICBDT1JSRVNQT05ERU5DRSBPVEhFUlMgIDEzLTEwLTIwMTQucGRm" target="_blank" style="word-wrap:break-word;">3843-CHENP-2009  CORRESPONDENCE OTHERS  13-10-2014.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mzg0My1DSEVOUC0yMDA5ICBDT1JSRVNQT05ERU5DRSBPVEhFUlMgIDE1LTEwLTIwMTQucGRm" target="_blank" style="word-wrap:break-word;">3843-CHENP-2009  CORRESPONDENCE OTHERS  15-10-2014.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mzg0My1DSEVOUC0yMDA5ICBFWEFNSU5BVElPTiBSRVBPUlQgUkVQTFkgUkVDRUlWRUQgIDIxLTAyLTIwMTQucGRm" target="_blank" style="word-wrap:break-word;">3843-CHENP-2009  EXAMINATION REPORT REPLY RECEIVED  21-02-2014.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mzg0My1DSEVOUC0yMDA5ICBGT1JNLTMgIDIxLTAyLTIwMTQucGRm" target="_blank" style="word-wrap:break-word;">3843-CHENP-2009  FORM-3  21-02-2014.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mzg0My1DSEVOUC0yMDA5ICBPVEhFUiBQQVRFTlQgRE9DVU1FTlQgIDA3LTA4LTIwMTQucGRm" target="_blank" style="word-wrap:break-word;">3843-CHENP-2009  OTHER PATENT DOCUMENT  07-08-2014.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mzg0My1DSEVOUC0yMDA5ICBPVEhFUlMgIDIxLTAyLTIwMTQucGRm" target="_blank" style="word-wrap:break-word;">3843-CHENP-2009  OTHERS  21-02-2014.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mzg0My1jaGVucC0yMDA5IGFic3RyYWN0LnBkZg==" target="_blank" style="word-wrap:break-word;">3843-chenp-2009 abstract.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mzg0My1DSEVOUC0yMDA5IEFTU0lHTk1FTlQgMDYtMDgtMjAxNC5wZGY=" target="_blank" style="word-wrap:break-word;">3843-CHENP-2009 ASSIGNMENT 06-08-2014.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mzg0My1jaGVucC0yMDA5IGNsYWltcy5wZGY=" target="_blank" style="word-wrap:break-word;">3843-chenp-2009 claims.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mzg0My1DSEVOUC0yMDA5IENPUlJFU1BPTkRFTkNFIE9USEVSUyAwNi0wOC0yMDE0LnBkZg==" target="_blank" style="word-wrap:break-word;">3843-CHENP-2009 CORRESPONDENCE OTHERS 06-08-2014.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mzg0My1jaGVucC0yMDA5IGNvcnJlc3BvbmRlbmNlIG90aGVycy5wZGY=" target="_blank" style="word-wrap:break-word;">3843-chenp-2009 correspondence others.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mzg0My1jaGVucC0yMDA5IGRlc2NyaXB0aW9uIChjb21wbGV0ZSkucGRm" target="_blank" style="word-wrap:break-word;">3843-chenp-2009 description (complete).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mzg0My1jaGVucC0yMDA5IGRyYXdpbmdzLnBkZg==" target="_blank" style="word-wrap:break-word;">3843-chenp-2009 drawings.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mzg0My1jaGVucC0yMDA5IGZvcm0tMS5wZGY=" target="_blank" style="word-wrap:break-word;">3843-chenp-2009 form-1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mzg0My1jaGVucC0yMDA5IGZvcm0tMTgucGRm" target="_blank" style="word-wrap:break-word;">3843-chenp-2009 form-18.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mzg0My1jaGVucC0yMDA5IGZvcm0tMjYucGRm" target="_blank" style="word-wrap:break-word;">3843-chenp-2009 form-26.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mzg0My1DSEVOUC0yMDA5IEZPUk0tMyAzMS0xMi0yMDA5LnBkZg==" target="_blank" style="word-wrap:break-word;">3843-CHENP-2009 FORM-3 31-12-2009.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mzg0My1jaGVucC0yMDA5IGZvcm0tMy5wZGY=" target="_blank" style="word-wrap:break-word;">3843-chenp-2009 form-3.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mzg0My1jaGVucC0yMDA5IGZvcm0tNS5wZGY=" target="_blank" style="word-wrap:break-word;">3843-chenp-2009 form-5.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mzg0My1jaGVucC0yMDA5IHBjdC5wZGY=" target="_blank" style="word-wrap:break-word;">3843-chenp-2009 pct.pdf</a></p>
		<br>
		<div class="pull-left">
			<a href="264492-vessels-with-personnel-access-provisions.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="264494-vibrating-wet-shaver.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>264493</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>3843/CHENP/2009</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>01/2015</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>02-Jan-2015</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>31-Dec-2014</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>01-Jul-2009</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>CANON KABUSHIKI KAISHA</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td>30-2, SHIMOMARUKO 3-CHOME, OHTA-KU, TOKYO 146-8501</td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>MIZUTANI, YURI</td>
											<td>C/O CANON KABUSHIKI KAISHA 30-2, SHIMOMARUKO 3-CHOME, OHTA-KU, TOKYO 146-8501</td>
										</tr>
										<tr>
											<td>2</td>
											<td>TAKAHAMA, SHIN-ICHIRO</td>
											<td>C/O CANON KABUSHIKI KAISHA 30-2, SHIMOMARUKO 3-CHOME, OHTA-KU, TOKYO 146-8501</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>C12N15/09</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>PCT/JP07/73621</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td>2007-11-30</td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td>2006-325951</td>
									<td>2006-12-01</td>
								    <td>Japan</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/264493-method-of-determining-the-haplotype-of-multiple-allelic-genes by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 01:38:55 GMT -->
</html>
