<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/264529-improved-enzymes by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 01:48:27 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 264529:IMPROVED ENZYMES.</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">IMPROVED ENZYMES.</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>The present invention relates to modified GTP cyclohydrolase II enzymes that display increased specific activity, and to polynucleotides encoding them. The invention further pertains to vectors comprising these polynucleotides and host cells containing such vectors. The invention provides a method for producing the modified enzyme and a method for producing riboflavin, a riboflavin precursor, FMN, FAD, or a derivative thereof.</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td>Improved enzymes<br>
The present invention provides modified enzymes with higher GTP cyclohydrolase n activity than the respective -wild-type enzymes. The modified enzymes and polynucleotides encoding the same can be used for the production of riboflavin, riboflavin precursors, flavin mononucleotide (FMN), flavin adenine dinucleotide (FAD), and derivatives thereof.<br>
Riboflavin (vitamin B2) is synthesized by all plants and many microorganisms but is not produced by higher animals. Because it is a precursor to coenzymes such as flavin adenine dinucleotide and flavin mononucleotide that are required in the enzymatic oxidation of carbohydrates, riboflavin is essential to basic metabolism. In higher tmirrwis insufficient riboflavin can cause loss of hair, inflammation of the skin, vision deterioration, and growth failure.<br>
Engineering of riboflavin production strains with increased rates and yields of riboflavin has been achieved in the past in a number of different ways. For instance, (1) classical mutagenesis was used to create variants with random mutations in the genome of the organism of choice, followed by selection for higher resistance to purine analogs and/or by screening for increased production of riboflavin. (2) Alternatively, the terminal enzymes of riboflavin biosynthesis, i.e., the enzymes catalyzing the conversion of guanosine triphosphate (GTP) and ribulose-5-phosphate to riboflavin, were overexpressed, resulting also in a higher flux towards the target product However, in this latter approach, strong overexpression' of the riboflavin biosynthesis proteins imposes an additional metabolic burden on the host cells which may, in turn, induce stress response reactions and other undesirable negative effects on the cells' physiology.<br><br>
The enzymes required catalyzing the biosynthesis of riboflavin from guanosine triphosphate (GTP) and ribulose-5-phosphate are encoded by four genes (ribG, ribBy ribA, and ribH) in B. subtilis. These genes are located in an operon, the gene order of which differs from the order of the enzymatic reactions catalyzed by the enzymes. For example, GTP cyclohydrolase n, which catalyzes the first step inriboflavin biosynthesis, is encoded by the third gene in the operon, ribA. The ribA gene also encodes a second enzymatic activity, i.e.t 3,4-dihydroxy-2-butanone 4-phosphate synthase (DHBPS), which catalyzes the conversion of ribulose-5-phosphate to the four-carbon unit 3,4-dihydroxy-2-butanone 4-phosphate (DHBP). Deaminase and reductase are encoded by the first gene of the operon, ribG. The pennltimate step in riboflavin biosynthesis is catalyzed by lumazine synthase, the product of the last rib gene, ribH. Riboflavin synthase, which controls the last step of the pathway, is encoded by the second gene of the operon, HbB. The function of the gene located at the 3' end of the rib operon is, at present, unclear; however, its gene product is not required for riboflavin synthesis.<br>
Transcription of the riboflavin operon from the ribPl promoter is controlled by an attenuation mechanism involving a regulatory leader region located between ribPl and ribG. The ribO mutations within this leader region result in deregulated expression of the riboflavin operon. Deregulated expression is also observed in strains containing nrissense mutations in the ribC gene. The ribC gene has been shown to encode the flavin kinase/FAD syntiiase of B. subtilis (Mack, M., et al., J. Bacteriol., 180:950-955,1998). Deregulating mutations reduce the flavokinase activity of the ribC gene product resulting unreduced intracellular concentrations of fJavrnmononucleotide (FMN), the effector molecule of me riboflavin regulatory system.<br>
'Recently, £adlhts subtilis was'genetically engineered'to-produce high yieWfiCofriboflavin . during a short fennentation cycle (U.S. Patent No. 5,837,528). This approach combined classical genetic mutant selection and fennentation improvement with genetic engineering of the riboflavin biosynthetic genes by deregulating and increasing the level of gene expression. In this system, the expression of the rib genes was increased "by mutating the flavokinase encoding ribC gene, by linking the rib genes to strong, constitutive promoters, and by increasing the copy number of the rib genes.<br>
As already discussed above, overexpression of the rib genes poses an additional burden on the production strains which, may, potentially, have a negative impact on the production of riboflavin precursors, riboflavin, FMN, FAD, or their derivatives. In order to circumvent this shortcoming, it is a subject of thepresent invention to describe GTP cyclohydrolase E mutants with increased specific activity. Use of such mutant enzymes in production strains, either alone or combined with improved mutants of the other Rib proteins, will allow higher flux rates with less or no additional burden on the cells' metabolism.<br><br>
As used herein, the term "GTP cyclohydrolase E" may include any enzyme that is capable of catalyzing the conversion of GTP to 2,5-diamino-6-ribosylamino-4 (3H)-pyrimidanone-5'-phosphate (DRAPP). It is irrelevant whether this enzyme is capable of catalyzing further reactions, as for example the conversion of ribulose-5-phosphate to DHBP. A "GTP cyclohydrolase H" maybe homologous to one or more of the enzymes the amino acid sequences of which are shown in Figure 1 or in Table 4. "Homologous" refers to a GTP cyclohydrolase IE that is at least about 50% identical, preferably at least about 60% identical, more preferably at least about 70% identical, even more preferably at least about 80% identical, even more preferably .at least about 85% identical, even more preferably at least about 90% or 95% identical, and most preferably at least about 98% identical to one or more of the amino acid sequences as shown in Figure 1 or in Table 4.<br>
The term "% identity", as known in the art, means the degree of relatedness between polypeptide or polynucleotide sequences, as the case may be, as determined by the match between strings of such sequences. "Identity" can. be readily determined by known methods, e.g., "with the program BESTFTT (GCG Wisconsin Package, version 10.2, Accehys Inc., 9685 ScrantonRoad, San Diego, CA 92121-3752, USA) using the following parameters: gap creation penalty 8, gap extension penalty 2 (default parameters).<br>
"Wild-type enzyme" or "wild-type GTP cyclohydrolase IF may include any GTP cyclohydrolase H homologous to any one of fhe enzymes shown in Figure 1 or in Table 4 that is used as starting point for designing mutants with increased activity according to the present invention. "Wild-type" hi the context of the present invention may include both GTP cyclohydrolase H sequences derivable from nature as well as variants of synthetic GTP cyclohydrolase IE enzymes (as long as they are homologous to any one of the • sequences shown, in Figure 1 or in-Table 4), if they can be made more active the teachings of the present invention., The terms "wild-type GTP cyclohydrolase IT1 and "non-modified GTP cyclohydrolase II" are used interchangeably herein.<br>
A "mutant", "mutant enzyme", or "mutant GTP cyclohydrolase IT may include any variant derivable from a given wild-type enzyme/GTP cyclohydrolase II (according to the above definition) according to the teachings of the present invention and being more active than the respective wild-type enzyme. For the scope of the present invention, it is not relevant how the mutant(s) are obtained; such mutants maybe obtained, e,g., by site-directed mutagenesis, saturation mutagenesis, random mutagenesis/directed evolution, chemical or UV mutagenesis of entire cells/organisms, and other methods which are known in the art. These mutants may also be generated, e.g., by designing synthetic genes, and/or produced by in vitro (cell-free) translation. For testing of specific activity, mutants may be (over-) expressed by methods known to those skilled in the art. The terms "mutant GTP<br><br>
cyclohydrolase n" and "modified GTP cyclohydrolase II" are used interchangeably herein. This also applies to the terms "mutant enzyme" and "modified enzyme".<br>
"Riboflavin precursor" and "derivatives of riboflavin, FMN or FAD" in the context of this patent application may include any and all metabolite(s) requiring GTP cyclohydrolase n as an intermediate enzyme hi their (bio-) synthesis. In the context of this patent application, it is irrelevant whether such (bio-) synthesis pathways are natural or non-natural (z.e., pathways not occurring in nature, but engineered biotechnologically). Preferably, the synthesis pathways are biochemical in nature. Riboflavin precursors and derivatives of riboflavin, FMN or FAD include but are not limited to: DRAPP; 5~amino-6-ribosylamino-2,4 (IH.SH pvrm Tidinedione-S'-phosphate; 2,5-dianxmo-6-ribitylarnino-4 (3H)-pyrimidmone-5'-phosphate; 5-amino-6-ribitylammo-2&gt;4 (lH,3H)-pyrimidmedione-5'-phosphate; 5-ammo-6-ribitylamitw2,4 (lH,3H)-pyrhnidinedione; 6,7-dimethyl-8-ribityEumazine (DMRL); and flavoproteins. The term "riboflavin" also includes derivatives of riboflavin, such as e.g. riboflavin-5-phosphate and salts thereof, such as e.g. sodium riboflavin-5-phosphate.<br>
It is in general an object of the present invention to provide an enzyme having GTP cyclohydrolase n activity, said enzyme being modified in a way that its catalytic properties are more favorable (i.e., showing higher specific activity) than those of the non-modified GTP cyclohydrolase IE enzymes.<br>
The invention relates to a modified GTP cyclohydrolase n which exhibits higher (specific) activity in comparison to the corresponding non-modified GTP cyclohydrolase H •wherein (i) the BTTri-rxn acid sequence of me modified GTP cyclohydrolase H contains at least one - mutation,wieri compared within amino acid sequence of the corresponding non-modified<br>
•	'	<br>
GTP cyclohydrolase n, and (ii) the at least one mutation is at one or more amino acid positions selected from the group<br>
consisting of amino acid positions corresponding to positions 261,270,276,279,308 and<br>
347 of the amino acid sequence of Bacillus subtilis GTP cyclohydrolase II as shown in<br>
SEQIDNO:2.	;<br>
Thus, it is an object of the present invention to provide amodified GTP cyclohydrolase H,<br>
wherein<br>
(i) the specific activity of the modified enzyme is increased in comparison to the<br>
corresponding non-modified enzyme, and<br>
(ii) the amino acid sequence of the modified enzyme comprises one or more mutation(s)<br>
including 1,2,3,4,5, or 6 mutation(s) on amino acid position(s) corresponding to<br>
positions 261,270,276,279,308 and/or 347 of SEQ ID NO:2.<br><br>
The term "at least one mutation" means one or more mutation on a position as defined above leading to a modified GTP cyclohydrolase H having an increased specific activity compared to the non-modified enzyme. A modified enzyme as described above may consists of only 1,2,3,4,5 or 6 mutation(s) on a position as defined above leading to an increased specific activity compared to the non-modified enzyme, but may also include further amino acid mutations on other positions, as long as the resulting modified enzyme has an increased specific activity. Thus, the modified enzyme comprises one or more mutation(s) including 1,2,3,4,5,or6 mutation(s) on amino acid position(s) corresponding to positions 261,270,276,279,308 and/or 347 of the amino acid sequence of Bacillus subtilis GTP cyclohydrolase n as shown in SEQ ID NO:2. Examples of such mutations on positions other than the ones defined above are amino acidmutation(s) on a position corresponding to amino acid position 196,282, and/or 325 of SEQ ID NO:2.<br>
As used herein, the term "specific activity" denotes the reaction rate of the wild-type and mutant GTP cyclohydrolase n enzymes under properly defined reaction conditions as described e.g. in Ritz et al. (J. Biol. Chem. 276,22273-22277,2001), Koh et al (Mol. Gen. Genet. 251, 591-598,1996), or Schramek et al (J. Biol. Chem. 276,44157-44162, 2001) or as described in detail in Example 2. The "specific activity" defines the amount of substrate consumed and/or product produced in a given time period and per defined amount of protein at a defined temperature. Typically, "specific activity" is expressed in ftmol substrate consumed or product formed per min per mg of protein. Typically, pnrd/min is abbreviated by U (= unit). Therefore, the unit definitions for specific activity (mg of protein) or U/(mg of protein) are used interchangeably throughout this document It is understood that in the context of the present invention, specific activity must be compared on the basis of a similar, or preferably identical, length of the p' chain. The invention not be. given wild-type enzyme through, e.g., formation of a fusion protein, thereby reducing apparent specific activity of the overall enzyme.<br>
According to the present invention the modified GTP cyclohydrolase U exhibits a specific activity that is higher than that of the corresponding non-modified enzyme. Preferably, the specific activity of the modified GTP cyclohydrolase U of the invention is increased by at least about 5,10,25,40, 60,70, 80, 85,90%, more preferably at least about 70% in comparison to the corresponding non-modified GTP cyclohydrolase n (for measurement of specific activity, see below). Preferably, increases in specific activity refer to the experimental conditions described in Example 1 of this application. Approx. 0.004-0.02 U/ml (corresponding to approx. 40 Jig/ml of Bacillus subtilis GTP cyclohydrolase n or 20 jig/ml for the best mutants described here), preferably approx. 0.004 U/ml of GTP<br><br>
cyclohydrolase H activity, were present in the assay mixture, and the reaction was carried out at 37°C.<br>
The amino acid sequence of the modified GTP cyclohydrolase II of the invention contains at least one mutation as denned above when compared with the amino acid sequence of the corresponding non-modified GTP cyclohydrolase n. Said mutation may be one or more addition, deletion and/or substitution, preferably one or more amino acid substitution wherein a given amino acid present in the amino acid sequence of the non-modified GTP cyclohydrolase H is replaced with a different amino acid in the amine acid sequence of the modified GTP cyclohydrolase n of the invention. The amino acid sequence of the modified GTP cyclohydrolase n may contain at least one amino acid substitution when compared with the amino acid sequence of the corresponding non-modified GTP cyclohydrolase 1, i.e. may comprise one or more mutation(s) including 1, 2, 3, 4, 5, or 6 amino acid substitution(s) on amino acid position(s) corresponding to positions 261, 270, 276, 279, 308 and/or 347 of SEQ ID NO:2, preferably 2, 3, 4 or 5 amino acid substitutions. Thus, the modified enzyme preferably contains at least 2, at least 3, at least 4 or at least 5 substitutions when compared with the amino acid sequence of the corresponding non-modified GTP cyclohydrolase H.<br>
In one embodiment, a modified GTP cyclohydrolase n obtainable from Bacillus, preferably Bacillus subtilis, is provided, wherein<br>
(i) the specific activity of the modified enzyme is increased in comparison to the corresponding non-modified enzyme, and<br>
(ii) the amino acid sequence of the modified enzyme comprises one or more mutation(s) including 1, 2, 3, 4, 5, or 6 mutation(s) on amino acid positions) corresponding to<br>
, 279,&gt;p8:and/or 347 of.SEQJP.lp2,<br>
In one embodiment the non-modified enzyme corresponds to Bacillus subtilis GTP cyclohydrolase H as shown in SEQ ID NO:2. Thus, the modified enzyme having an increased specific activity in comparison to the wild type enzyme comprises one or more mutation(s) including 1, 2, 3, 4, 5, or 6 mutation(s) on amino acidposition(s) corresponding to positions 261, 270, 276, 279, 308 and/or 347 of SEQ ID NO:2. Ma further embodiment the modified enzyme having increased specific activity as defined above contains amino acid mutation(s) beside the amino acid positions as above, said further mutation(s) being on a position selected from the group consisting of position 196, 282, 235, and any combination thereof, preferably amino acid substitutions, more preferably the substitutions are Y196C (replacement of tyrosine by cysteine), A282T (replacement of alanine by threonine) or F325Y (replacement of phenylalanine by tyrosine).<br><br>
A non-modified GTP cyclohydrolase n may be any GTP cyclohydrolase II for which increasing the specific activity is desirable. Non-modified GTP cyclohydrolase II enzymes include but are not limited to GTP cyclohydrolase IE enzymes derivable from nature, such as enzymes of eukaryotic or prokaryotic origin, preferably fungal or bacterial origin. More preferably the non-modified enzyme is selected from the ones shown in Figure 1 or in Table 4 or which is homologous to any of the amino acid sequences as shown in Figure 1 or in Table 4, in particular selected from the group consisting ofAshbya, Saccharomyces, Eremotliecium, Candida, Neurospora, 5chizosaccharomyces,Archeoglobus, Streptomyces, Helicobacter, Escherichia, Corynebacterium, Thermotoga,Arabidopsis,Lycopersicum, Oryza, Alcaligenes, Pseudomonas, Dinococcus, Lactobacillus, Photobacterium and Bacillus and preferably selected from the group consisting of Candida guittiermondii, Ashbya gossypii (Eremoihechm ashbyii) (SEQ ID NO:33), Saccharomyces cerevisiae, Neurospora crassa, Schizosaccharomycespombe, Archeoglobusfulgidus, Streptomyces coelicolor, Helicobacter pylori J99, Eseherichia coli (SEQ ID NO:35), Corynebacterium glutamicum (SEQ ID N0:37), Bacillus amyloliguefaciens (SEQ ID NO.39), Bacillus cereus (SEQ ID NO:41), Bacillus halodurans (SEQ ID N0:43), Thermotoga marithna, Arabidopsis thaliana, Lycopersicum exculentum, Oryza sativum, Alcaligenes eutropkus, Pseudomonas putida strain KT2440, Corynebacterium efficiens,Demococcus radiodurans, LactobacUlus plantarum, Photobacterium phosphoreum, Pseudomonas putida strain KT2440 (second gene) and Bacillus subtilis (SEQ ID NO:2). Most preferably the non-modified enzyme is obtainable from Bacillus subtilis,<br>
The modified GTP cyclohydrolase H of the invention may be obtained by mutating the<br>
corresponding non-modified GTP cyclohydrolase IL In one embodiment, the non-<br>
modified enzyme corresponds to the B. subtilis GTP cyclohydrolase n shown in SEQ ID<br>
N0:2 and memtfdified enzyme c£mpriseslDne~ormofe anim6vacld'mtu^tlt^	1,<br>
2,3,4, 5, or 6 mutation(s) on amino acid positions) 261,270,276,279,308 and/or 347 of SEQ ID NO:2, wherein the specific activity of said modified enzyme is increased compared to the non-modified enzyme.<br>
Preferably, the at least one mutation is at one or more amino acid positions selected from the group consisting of amino acid positions corresponding to positions 261,279,308 and 347 of the amino acid sequence of Bacillus subtilis GTP cyclohydrolase n as shown in SEQ ID NO: 2. Thus,- in one embodiment the modified GTP cyclohydrolase n comprises one or more mutation(s) including 1,2,3 or 4 mutation(s) on amino acidposition(s) corresponding to positions 261,279, 308, and/or 347 of SEQ ID NO:2. In apreferred embodiment, the modified enzyme is obtainable from B. subtilis and comprises mutated amino acid positions 261,279,308, and/or 347 as shown in SEQ ID N0:2, corresponding to amino acids V261, Q279, K308, and M374, respectively.<br><br>
In another preferred embodiment the at least one mutation is at one or more amino acid positions selected from the group consisting of amino acid positions corresponding to positions 270,279,308 and 347 of the amino acid sequence of Bacillus subtilis GTP cyclohydrolase E as shown in SEQ ID NO: 2. Thus, in one embodiment the modified GTP cyclohydrolase H comprises one or more mutatipn(s) including 1,2,3 or 4 mutation(s) on amino acid positions) corresponding to positions 270,279,308, and/or 347 of SEQ ID NO:2. Preferably, the modified enzyme is obtainable from 5. subtilis and comprises mutated amino acid positions 270,279,308, and/or 347 as shown in SEQ ID NO:2, corresponding to amino acids G270, Q279, K308, and M374, respectively.<br>
In a further preferred embodiment the at least one mutation is at one ore more amino acid positions selected from the group consisting of amino acid positions corresponding to positions 276,279,308 and 347 of the amino acid sequence of Bacillus subtilis GTP cyclohydrolase n as shown in SEQ ID NO: 2. Thus, in a further embodiment the modified GTP cyclohydrolase n comprises one or more mutation(s) including 1,2,3 or 4 mutation(s) on amino acid position(s) corresponding to positions 276,279,308, and/or 347 of the amino acid sequence of Bacillus subtilis GTP cyolohydrolase H as shown in SEQ ID NO:2. Preferably, the modified enzyme is obtainable from B. subtilis and comprises mutated amino acid positions 276,279,308, and/or 347 as shown in SEQ ID NO:2, corresponding to amino acids A276, Q279, K308, and M374, respectively.<br>
Preferably, the one or more amino acid mutation(s) of the modified GTP cyclohydrolase n is one or more amino acid substitution^).<br>
A modified GTP cyclohydrolase H may comprise one or more mutation(s) including only one mutation on an amino acid position as defined above, such mutation, particularly.an amino acid substitution, may include one mutation on an amino acid position corresponding to position 261,270,276,279,308, or 347 of the amino acid sequence of Bacillus subtilis GTP cyclohydrolase H as shown in SEQ ID NO:2. The amino acid present in the non-modified GTP cyclohydrolase E corresponding to position 261 may be valrne, the amino acid present in the non-modified GTP cyclohydrolase H corresponding to position 270 may be glycine, the aminp acid present in the non-modified GTP cyclohydrolase E corresponding to position 276 may be alanine, the amino acid present in the non-modified GTP cyclohydrolase II corresponding to position 279 may be glutamine, the amino acid present in the non-modified GTP cyclohydrolase E corresponding to position 308 may be lysine, and the amino acid present in the non-modified GTP cyclohydrolase E corresponding to position 347 may be methionine.<br>
The amino acid in the sequence of the non-modified GTP cyclohydrolase n may be changed such that the amino acid corresp$ndjjig to position 261 may be changed to alanine<br><br>
 (e.g. V261A), the amino acid corresponding to position 270 may be changed to alanine or arginine (e.g. G270A and G270R), the amino acid corresponding to position 276 mjy be changed to threonine (e.g, A276T), the amino acid corresponding to position 279 maybe changed to arginine (e.g. Q279A), the amino acid corresponding to position 308 may be changed to arginine (e.g. K308R), and the amino acid corresponding to position 347 may be changed to isoleucine (e.g. M374I). In one embodiment, the modified enzyme is obtainable from B. subtilis comprising an amino acid substitution in a position of SEQ ID NO:2 which is selected from the group consisting of position 261,270,276,279,308, and 347. Preferably, the substitution is V261A, G270A, G270R, A276T, Q279R, K308R or M3471<br>
A modified GTP cyclohydrolase H may comprise one or more mutation(s) including two mutations on ammo acid positions as defined above, such mutations, particularly armnn acid substitutions, may include mutations on amino acid positions corresponding to two of the positions as defined above, e.g. combinations of positions corresponding to positions 261/270,261/276,261/279,261/308,261/347,270/276,270/279,270/308,270/347, 276/279,276/308,276/347,279/308,279/347, or 308/347 as shown in SEQ ID NO:2. Preferred are amino acid substitutions such as V261A/A276T, V261A/Q279R, V261A/K308R, V261AM347I, G270A/Q279R, G270A/K308R, G270A/M347I, A276T/Q279R, A276T/E308R, or A276TM347I, wherein the positions correspond to the amino acid positions of SEQ ID NO:2. In one embodiment, such preferred substitutions are comprised in a modified GTP cyclohydrolase n obtainable from B. subtilis wherein the non-modified enzyme corresponds to SEQ ID NO:2. Preferably, the modified B. subtilis GTP cyclohydrolase n as of SEQ ID NO:2 comprises substitutions V261A/A276T or A276T/M347L<br>
A modified GTP cyclohydrolase n may comprise one or more mutation(s) including three mutations on ammo acid positions as defined above, such mutations, particularly amino acid substitutions, may include mutations on amino acid positions corresponding to three of the positions as defined above, in particular combinations of positions corresponding to positions 261/279/308,261/279/347,261/308/347,270/279/308,270/279/347, 270/308/347,276/279/308,276/308/347, or 276/279/347 as shown in SEQ ID NO:2. Preferred are amino acid substitutions such as V261A/Q279R/K308R, V261A/K308RM3471, V261A/Q279RM347I, G270A/Q279R/K308R, G270A/K308R/M347I, G270A/Q279RM347I, A276T/Q279R/K308R, A276T/K308R/M347I, or A276T/Q279RM347I, wherein the positions correspond to the amino acid positions of SEQ ID N0:2. In one embodiment, such preferred substitutions are comprised in a modified GTP cyclohydrolase n obtainable from P. subtilis wherein the<br><br>
non-modified enzyme corresponds to SEQ ID NO:2. Preferably, the modified £. subtilis GTP cyclohydrolase n as of SEQ ID NO:2 comprises substitutions A276T/Q279R/M347L<br>
A modified GTP cyclohydrolase H may comprise one or more mutation(s) including four mutations on amino acid positions as defined above, such mutations, particularly ammo acid substitutions, may include mutations in ammo acid positions corresponding to four of the positions as defined above, in particular combinations of positions corresponding to positions 261/279/308/347,270/279/308/347, or 276/279/308/347 as shown in SEQ ID N0:2. Preferred are amino acid substitutions such as V261A/Q279R/K308R/M347I, G270A/Q279R/K308R/M347I or A276T/Q279R/K308R/M347I, wherein the positions correspond to the amino acid positions of SEQ ID NO:2. In one embodiment, such preferred substitutions are comprised in a modified GTP cyclohydrolase H obtainable from B. subtilis wherein the non-modified enzyme corresponds to SEQ ID NO:2. Preferably, the modified B. subtilis GTP cyclohydrolase n as of SEQ ID NO:2 comprises substitutions A276T/Q279R/K308RM347L<br>
Most preferred are the combinations of mutations disclosed in Table 1 or 2 (see infra). The amino acid positions identified in these examples may be transferred to GTP cyclohydrolase It enzymes of different origin, as e.g, shown in Figure 1 or in Table 4.<br>
The modified GTP cyclohydrolase H of the invention may comprise foreign amino adds, preferably at its N- or C-terminus, "Foreign amino acids" mean amino acids which are not present hi a native (occurring in nature) GTP cyclohydrolase H, preferably a stretch of at least about 3, at least about 5 or at least about 7 contiguous amine acids which are not present in a native GTP cyclohydrolase U Preferred stretches of foreign amino acids •include to "tags" .that, facilitate purification of the <br>
produced modified GTP cyclohydrolase IL Examples of such tags include but are not	<br>
limited to a "Hisg" tag, a FLAG tag, tag, and the like. For calculation of specific activity, the values need to be corrected for these additional amino acids (see also above).<br>
la another embodiment the modified GTP cyclohydrolase II may contain one or more, e.g. two, deletions when compared with the amino acid sequence of the corresponding non-modified GTP cyclohydrolase IL Preferably, the deletions affect N- or C-terminal amino acids of the corresponding non-modified GTP cyclohydrolase n and do not significantly reduce the functional properties, e.g., the specific activity, of the enzyme.<br>
The polypeptides and polynucleotides of the present invention, including modified GTP cyclohydrolase II enzymes, may be provided in an isolated form, and preferably are purified to homogeneity. As used herein, the term "isolated" means that the material is removed from its original environment (e.g., the natural environment if it is naturally<br><br>
occurring), For example, a naturally-occurring polynucleotide or polypeptide present in a living microorganism is not isolated, but the same polynucleotide or polypeptide, separated from some or all of the coexisting materials in the natural system, is isolated. Such polynucleotides could be part of a vector and/or such polynucleotides or polypeptides may be part of a composition and still be isolated in that such vector or composition is not part of its natural environment An isolated polypeptide is preferably greater than 80% pure, more preferably greater than 90% pure, even more preferably greater than 95% pure, most preferably greater than 99% pure. Purity may be determined according to methods known in the art, e.g., by SDS-PAGE and subsequent protein staining. Protein bands can then be quantified by densitometry. Further methods for determining the purity are within the level of ordinary skill.<br>
The invention further relates to a polynucleotide comprising a nucleotide sequence which codes for a modified GXP cyclohydrolase n according to the invention. "Polynucleotide" as used herein refers to a polyribomicleotide or polydeoxyribonucleoti.de that may be unmodified RNA or DNA or modified RNA or DNA. Polynucleotides include but are not limited to single- and double-stranded DNA, DNA that is a mixture of single- and double-stranded regions, single- and doi&amp;le-strandedKNA, and KNA that is a mixture of single-and double-stranded regions, hybrid molecules comprising DNA and RNA that may be single-stranded or, more typically, double-stranded or a mixture of single- and double-stranded regions. The term "polynucleotide" includes DNA or RNA that comprises one or more unusual bases, e.g., inosine, or one or more modified bases, e.g., tritylated bases.<br>
The polynucleotide of the invention can easily be obtained by modifying a polynucleotide sequence which codes for a non-modified GTP cyclohydrolase IL Examples of such polynucleotide sequences encodrngnon but are not limited to the amino acid sequences of Figure 1 or in Table 4, in particular to SEQ ID NOs:2,33,35,37,39,41, and 43. Non-limiting examples of polynucleotides encoding modified GTP cyclohydrolase n enzymes according to the invention are shown in SEQ ID NOs:6, 8,10,12,14,16,18,20,22,24 and 26.<br>
Methods for introducing mutations, e.g., additions, deletions and/or substitutions into the nucleotide sequence coding for the non-modified GTP cyclohydrolase H include but are not limited to site-directed mutagenesis and PCR-based methods.<br>
DNA sequences of the present invention can be constructed starting from genornic or cDNA sequences coding for GTP cyclohydrolase H enzymes known in the state of the art, as are available from, e.g., Genbank (Ihtelligenetics, California, USA), European Bioinformatics Institute (EQnstonHall, Cambridge, GB), NBRF (Georgetown University, Medical Centre, Washington DC, USA) and Vecbase (University of Wisconsin,<br><br>
Biotechnology Centre, Madison, Wisconsin, USA) or from the sequence information disclosed in Figure 1 or in Table 4 by methods of in vitro mutagenesis (see e.g. Sambrook et al., Molecular Cloning, Cold Spring Harbor Laboratory Press, New York). Another possibility of mutating a given DNA sequence which is also preferred for the practice of the present invention is mutagenesis by using the polymerase chain reaction (PCR). DNA as starting material can be isolated by methods known in the art and described, e.g., in Sambrook et al. (Molecular Cloning) from the respective strains/organisms. It is, however, understood that DNA encoding a GTP cyclohydrolase H to be constructed/mutated in accordance with the present invention can also be prepared on the basis of a known DNA sequence, e.g. by construction of a synthetic gene by methods known in the art (as described, e.g., in EP 747483).<br>
The polynucleotide of the invention may be an isolated polynucleotide, i,e. a polynucleotide that is substantially tree from other nucleic acid sequences such as but not limited to other chromosomal and extrachromosomal DNA and RNA. Conventional nucleic acid purification methods known to people skilled hi the art maybe used to obtain isolated polynucleotides. The term also embraces recombinant polynucleotides and chemically synthesized polynucleotides.<br>
In yet another embodiment the invention pertains to a functional polynucleotide in which a promoter, aribosome-binding site, if necessary as hi the case of bacterial cells, and a terminator are operably linked with a polynucleotide according to the invention. In yet a further embodiment the invention pertains to a vector or plasmid comprising such a polynucleotide. The vector or plasmid preferably comprises at least one marker gene. The term "operably linked" as used herein refers to the association of nucleic acid sequences on "asmgle"nut5Mc~Mtf<br>
example, a promoter is operably linked with a coding sequence when it is capable of affecting the expression of that coding sequence, i.e., the coding sequence is under the transcriptional control of the promoter. Coding sequences may be operably linked to regulatory sequences ha sense or anti-sense orientation. The team "expression" denotes the transcription of a DNA sequence into mRNA and/or the translation of mKNA into an amino acid sequence. The term "over-expression" means the production of a gene product in a modified organism (e.g., modified by transformation or transfection) that exceeds levels of production in the corresponding non-modified organism by deregulating the expression of the gene and/or by multiplying the gene itself inside of the organism.<br>
Once complete DNA sequences of the present invention have been obtained, they may be integrated into vectors or directly introduced into the genome of a host organism by methods known in the art and described in, e.g., Sambrook et al. (s.a.) to (over-) express the encoded polypeptide in appropriate host systems. However, a man skilled in the art<br><br>
knows that also the DNA sequences themselves can be used to transform the suitable host systems of the invention to get (over-) expression of the encoded polypeptide.<br>
Suitable host cells may be eukaryotic or prokaryotic cells. Examples of suitable host cells include but are not limited to bacterial cells such as cells of cyanobacteria, streptococci, staphylococci, enterococci, e.g., Bacilli as, e.g., Bacillus subtilis, or Streptomyces, as, e.g. Streptomyces lividans or Streptococcus pneumoniae, E. coli as, e.g., E. coli K12 strains, e.g. Ml 5 or HB 101. The host cells maybe a fungal cell including yeast cells, such as cells of Aspergilli, e.g. Aspergillus niger or Aspergillus oryzae, Trichoderma, e.g. Trichoderma reesei, Ashbya, e.g. Askbya gossypii, Eremothecium, e.g. Eremothecium ashbyii, Saccharomyces, e.g. Saccharamyces cerevisiae, Candida, e.g. Candida flareri, Pichia, e,g. Pichia pastoris, Hansenula polymorpha, e.g. H. polymorpha (DSM 5215), and Kluyveromyces. A suitable host cell may further be selected from animal cells, including mammalian cells, such as for instance CHO, COS, HeLa, 3T3, BHK, 293, CV-1 and insect cells tik&amp;Drosophila S2 so&amp;Spodoptera Sf9 cells; and plant cells such as cells of a gynmnsperm or angiospenn.<br>
Vectors which may be used for expression in fungi are known in the art and described e.g. in EP 420358, or by Cullen et al. (Bio/Technology 5,369-376,1987), Ward (in Molecular Industrial Mycology, Systems and Applications for Filamentous Fungi, Marcel Dekker, New York, 1991), Upshall etal (Bio/Technology 5,1301-1304,1987), Gwyrme et al. (Bio/Technology 5, 71-79,1987), or Punte al. (J. Biotechnol. 17,19-34,1991), and for yeast by Sreekrishna et al. (J. Basic Microbiol. 28,265-278,1988; Biochemistry 28,4117-4125,1989), Hitzemann et al. (Nature 293,717-722,1981) or in EP 183070, EP 183071, EP 248227, or EP 263311. Suitable vectors which maybe used for expression in E. coli art as described by Saaa.bFeok.0Z aL (s.&amp;). .Vectors which may be used for expression in Bacilli are known in the art and described, e.g. in EP 207459 or EP 4053707 byYansuraandHennerinProc.Nati. Acad. Sci. US A 81,43 9-443 (1984), orbyHenner, Le Grice and Nagarajan in Metb, Enzymol. 185,199-228,1990. Vectors which may be used for expression in H. polymorpha are known in the art as described, e.g. in Gellissen et a/., Biotechnology 9,291-295,199L<br>
Either such vectors already carry regulatory elements, e.g. promoters, or the polynucleotides of the present invention may be engineered to contain such elements. Suitable promoter elements which may be used are known in the art and are, e.g., for Trichoderma reesei the cbhl- or the pMl-promoter, fox Aspergillus oryzae the amy-promoter, and for Aspergillus niger the glaA-, alcA-, aphA-, tpiA-, gpdA- and the pkiA-promoter. Suitable promoter elements which may be used for expression in yeast are known in the art and are, e.g., the pho5- or the gap-promoter for expression in<br><br>
Saccharomyces cerevisiae, and e.g. the aoxl-promoter for Pichia pastoris or the FMD- or MOX promoter for H. polymorpha.<br>
Suitable promoters and vectors for bacterial expression include, e.g., a synthetic promoter described by Giacomini et al (Gene 144,17-24,1994), the vegl promoter from Bacillus subtilis or the strong bacteriophage T5 promoter. Appropriate teachings for expression of the claimed (mutant) GTP cyclohydrolase n enzymes ia bacteria, either by appropriate plasmids or through integration of GTP cyclohydrolase n-encodingDNA sequences into the chromosomal DNA, maybe found in many places, e.g., US Patent No. 6,322,995.<br>
Accordingly, vectors comprising a polynucleotide of the present invention, preferably for the expression of saidpolynucleotides in bacterial, fungal, animal or plant hosts, and such, transformed bacteria or fungal, animal or plant hosts are also an object of the present invention.<br>
The invention further relates to a method for producing riboflavin, a riboflavin precursor, FMN, FAD, or one or more derivatives thereof, comprising:<br>
(a)	culturing the host cell of the invention in a suitable medium under conditions that allow<br>
expression of the modified GTP cyclohydrolase n in said host cell; and<br>
(b)	optionally separating the product (riboflavin, a riboflavin precursor, FMN, FAD, or one<br>
or more derivatives thereof) from the medium,<br>
Such a method can be used for the biotechnological production of either one or more of the following products: riboflavin, a riboflavin precursor, FMN, FAD, or one or more derivatives thereof. Such derivatives may include flavoproteins.<br>
Methods of genetic and metabolic engineering of suita.blfr.host cell&amp;.accordingio the present invention are known to the man skilled in the art Similarly, (potentially) sultabler purification methods for riboflavin, a riboflavin precursor, FMN, FAD, or one or more derivatives thereof are well known in the area of fine chemical biosynthesis and production.<br>
It is understood that methods for biotechnological production of riboflavin, a riboflavin precursor, FMN, FAD, or one or more derivatives thereof according to the present invention are not limited to whole-cellular fermentation processes as described above, but may also use, e.g., permeabilized host cells, crude cell extracts, cell extracts clarified from cell remnants by, e.g., centrifiigation or filtration, or even reconstituted reaction pathways with isolated enzymes. Also combinations of such processes are in the scope of the present invention. In the case of cell-free biosynthesis (such as with reconstituted reaction pathways), it is irrelevant whether the isolated enzymes have been prepared by and isolated from a host cell, by in vitro transcription/translation, or by still othe*1 means.<br><br>
The invention further relates to a method for producing a modified GTP cyclohydrolase n of the invention comprising:<br>
(a)	culturing a host cell of the invention under conditions that allow expression of 1be<br>
modified GTP cyclohydrolase II of the invention; and<br>
(b)	recovering the modified GTP cyclohydrolase n from the cells or from the media.<br>
The modified GTP cyclohydrolase II enzymes of the invention maybe prepared from genetically engineered host cells comprising appropriate expression systems.<br>
For recombinant production of the polypeptides of the invention, host cells may be genetically engineered to incorporate polynucleotides or vectors orplasmids of the invention. Introduction of a polynucleotide or vector into the host cell may be effected by standard methods known hi the art such as calcium phosphate transfection, DEAE-dextran mediated transfection, microinjection, cationic lipid-mediated transfection, electroporation, transduction, ballistic introduction and infection.<br>
A great variety of expression systems may be used to produce the modified GTP cyclohydrolase n enzymes of the invention. Such vectors include, among others, those described supra. Generally, any system or vector suitable to maintain, propagate or express polynucleotides and/or to express a polypeptide in a host may be used for expression in this regard.<br>
In recombinant expression systems in eukaryotes, for secretion of a translated protein into the lumen of the endqplasmic reticulum, into the periplasmic space or into the extracellular<br>
1<br>
environment, appropriate secretion signals may be incorporated into the expressed polypeptide. These signals may be endogenous to the polypeptide or they may be<br>
Polypeptides of the invention may be recovered and purified from recombinant cell cultures by well-known methods including ammonium sulfate or ethanol precipitation, acid extraction, anion or cation exchange chromatography, phosphocsllulose chromatography, hydrophobic interaction chromatography, afSnity chromatography, hydroxyapatite chromatography, and high performance liquid chromatography. Well-known techniques for protein refolding may be employed to regenerate active conformation when the polypeptide is denatured during isolation and/or purification.<br>
GTP cyclohydrolase II enzymes of the present invention may also be expressed in plants according to methods as described, e.g., by Pen et at in Bio/Technology 11,811-814,1994 or in EP 449375, preferably in seeds as described, e.g., in EP 449376. Some suitable examples of promoters and terminators, include'those from nopaline synthase (nos), octopine synthase (ocs) and cauliflower mosaic virus (CaMV) genes. One type of efficient<br><br>
plant promoter that may be used is a high-level plant promoter. Such promoters, in operable hnkage with the genetic sequences of the present mvention should be capable of promoting expression of a gene product of the present invention. High-level plant promoters that may be used in mis invention include the promoter of the small subunit (ss) of the ribulose-l,5-bisphosphate carboxylase, for example from soybean, and the promoter of the chlorophyll a/b binding protein.<br>
Where commercial production of the instant proteins is desired, a variety of culture methodologies may be applied. For example, large-scale production of a specific gene product, overexpressed from arecombinantmicrobialhostmaybe achieved by both batch or continuous culture methodologies. .Batch and fed-batch culturing methods are common and well known in the art, and examples have been described by Thomas D. Brock in Biotechnology: A Textbook of Industrial Microbiology, Second Edition (1989), Sinauer Associates, Inc., Sunderland, Mass., or Deshpande, Appl. Biochem. Biotechnol. 36, 227-234, 1992. Methods of modulating nutrients and growth factors for continuous culture processes as well as techniques for mayirniTang the rate of product formation are well known in the art of industrial microbiology, and a variety of methods are detailed by Brock, supra.<br>
Fermentation media may further contain suitable carbon substrates. Suitable substrates may include but are not limited to monosaccharides such as glucose and fructose, oligosaccharides such as lactose or sucrose, polysaccharides such as starch or cellulose or mixtures thereof and unpurified mixtures from renewable feedstocks. It is contemplated that the source of carbon utilized in the present invention may encompass a wide variety of carbon containing substrates and will only be limited by the choice of organism.<br>
.<br>
The mvention further relates to a method for the preparation of a GTP cyclohydrolaseJL_ having increased specific activity, comprising the following steps:<br>
(a)	providing apolynucleotide encoding a first GTP cyclohydrolase n with a specific<br>
activity that, desirably, should be increased;<br>
(b)	introducing one or more mutation(s) into the polynucleotide sequence such that the<br>
mutated polynucleotide sequence encodes a modified GTP cyclohydrolase H comprising<br>
one or more mutation(s) when compared to the first GTP cyclohydrolase n wherein the on<br>
or more mutation(s) include 1, 2, 3, 4, 5, or 6 mutation(s) on amino acid posrtion(s)<br>
corresponding to positions 261, 270, 276, 279, 308 and/or 347 of SEQ ID NO:2;<br>
(c)	optionally inserting the mutated polynucleotide in a vector or plasmid;<br>
(d)	introducing the polynucleotide or the vector or plasmid into a suitable host cell; and<br>
(e)	culturing the host cell under conditions that allow expression of the modified GTP<br>
cyclohydrolase H<br><br>
The present invention, includes further the provision of a method for the preparation of a GTP cyclohydrolase E having increased specific activity, comprising the following steps:<br>
(a)	providing a polynucleotide encoding a first GTP cyclohydrolase n with a specific<br>
activity that, desirably, should be increased;<br>
(b)	providing the positions that have an effect on the specific activity,<br>
(c)	defining the optimal ammo acid for replacement of a given amino acid of the wild-type<br>
GTP cyclohydrolase II as defined in (b) and introducing one or more mutations into the<br>
polynucleotide sequence of (a) at the positions defined in (b) such that the mutated<br>
polynucleotide sequence encodes anew GTP cyclohydrolase H;<br>
(d)	optionally inserting the mutated polynucleotide in a vector or plasmid;<br><br>
(d)	introducing the polynucleotide or the vector or plasmid into a suitable host cell; and<br>
(e)	culturing the host cell under conditions that allow expression of the modified GTP<br>
cyclohydrolase n.<br>
In one embodiment, step (c) or the method described above is performed via saturated mutagenesis. However, it is understood that this may be not the only way to define the amino acid which should replace an amino acid at a given position of the wild-type GTP cyclohydrolase H in order to obtain a modified GTP cyclohydrolase n with increased specific activity.<br>
The preparation of a modified GTP cyclohydrolase n having increased specific activity from a non-modified GTP cyclohydrolase Has described above, e.g., via saturated mutagenesis, includes, but is not limited to, the preparation of mutated GTP cyclohydrolase II proteins from non-modified proteins as of Figure 1 or in Table 4, in particular those identified by SEQ ID NOs:2,33,35,37,39,41, and 43, such as for example non-modified GTP cyclohydrqlase n proteins of Bacillus subtilis arAshbya gossypii. The primers used for thePCR reaction are such that oneprirner, e.gv, the sense   -primer, may contain a mutated nucleotide sequence and the other primer, e.g., the anti-sense primer, may contain the wild-type nucleotide sequence. PCR with these primer pairs and genomic DNA of the wild-type ribA may result in a PCR product carrying the particular mutation at a given position, depending on the mutated nucleotide sequence of the primer used. After purification of the resulting PCR products using standard methods like, e.g., the QIAquick PCR purification kit (Qiagen), the DNA may be cut with restriction enzymes such as BamHI and^coRI, ligated into a suitable vector, e.g.} pQE60, and transformed into a strain which is negative for GTP cyclohydrolase IL An example for such a strain is me£. coli strain Rib? (Richter et al, J. BacterioL 175,4045-4051,1993) containing the plasmid pREP4. After confirmation of the correct sequence by DNA sequencing, the mutated RibA maybe purified and characterized as described above. If Ashbya gossypii is used for the generation of a GTP cyclohydrolase H having increased<br><br>
specific activity, saturated mutagenesis has to be performed at amino acid residues/positions T126, G135, A141, LI44, N182 and 1221 corresponding to the respective residues V261, G270, A276, Q279,.K308 and M347 of Bacillus subtilis GTP cyclohydrolase n as of SEQ ID NO:2 that were shown to have an impact on the specific activity of the latter enzyme (see Table 4).<br>
The preferred embodiments of this method correspond to the preferred embodiments of the modified GTP cyclohydrolase H, the polynucleotides encoding them, the vectors and plasmids, the host cells, and the methods described herein. The first and second GTP cyclohydrolase It correspond to the non-modified and modified GTP cyclohydrolase H, respectively (see supra).<br>
It is an object of the present invention to provide a polynucleotide comprising a nucleic acid sequence coding for a modified GTP cyclohydrolase II as described above, a vector, preferably an expression vector, comprising such a polynucleotide, a host cell which has been transformed by such a polynucleotide or vector, a process for the preparation of a GTP cyclohydrolase H of the present invention wherein the host ceE as described before is cultured under suitable culture conditions 'and tiie GTP cyclohydrolase n is isolated from such host cell or the culture medium by methods known in the art, and a process for the biotechnological production of riboflavin, a riboflavin precursor, FMN, FAD, or one or more derivatives thereof based on a host cell which has been transformed by such a polynucleotide or vector, and/or which may have stably integrated such a polynucleotide into its chromosome(s).<br>
It is also an object of the present invention to provide (i) a DNA sequence which codes for a GTP cyclohydrolase H carrying at least one of the specific mutations of they present invention and which hybridizes under standard conditions with any of the DNA sequences of the specific modified GTP cyclohydrolase n enzymes of the present invention, or (ii) a DNA sequence which codes for a GTP cyclohydrolase IE carrying at least one of the specific mutations of the present invention but, because of the degeneracy of the genetic code, does not hybridize but which codes for a polypeptide with exactly the same ammo acid sequence as a DNA sequence which hybridizes under standard conditions with any of the DNA sequences of the specific modified GTP cyclohydrolase H enzymes of the present invention, or (iii) a DNA sequence which is a fragment of such DNA sequences which maintains the activity properties of the polypeptide of which it is a fragment.<br>
"Standard conditions" for hybridization mean in the context of the present invention the conditions which are generally used by a man skilled hi the art to detect specific hybridization signals and which are described, e,g. by Sambrook et al, "Molecular Cloning", second edition, Cold Spring Harbor Laboratory Press 1989, New York, or<br><br>
preferably so-called stringent hybridization and non-stringent washing conditions or more preferably so-called stringent hybridization and stringent washing conditions a man skilled in the art is familiar with and which are described, e.g., in Sarnbrook et al. (s.a,). A specific example of stringent hybridization conditions is overnight incubation (e.g., 15 hours) at 42°C in a solution comprising: 50% form amide, 5 x SSC (150 mM NaCl, 15 mM trisodium citrate), 50 mM sodium phosphate (pH 7.6), 5 x Denhardf s solution, 10% dextran sulfate, and 20 fig/mi of denatured, sheared salmon sperm DNA, followed by washing the hybridization support in 0.1 x SSC at about 65°C.<br>
It is furthermore an object of the present invention to provide a DNA sequence which can be obtained by the so-called polymerase chain reaction method ("PCR") by PCR primers designed on the basis of the specifically described DNA sequences of the present invention. It is understood that the so obtained DNA sequences code for GTP   • cyclohydrolase H enzymes with at least the same mutation as the ones from which they are designed and show comparable activity properties,<br>
The various embodiments of the invention described herein may be cross-combined.<br>
Figure 1: Multiple sequence alignment calculated by the program PHEUP ofthe GCG program package of 92 GTP cyclohydrolase n sequences found by the program BLASTN using standard databases as SWISS-PHOT and TrEMBL (Candida guittiermondii, Ashbya gossypii, Saccharomyces cerevisiae, Neurospora crassa, Schizosaccharomycespom.be, Archaeoglobusfidgidus, Streptomyces coelicolor, Helicobacter pylori J99, Helicobacter pylori, Pyrococcusfuriosus, Thermatoga maritima, Chlamydia muridarum, Chlamydia trachomatis, Chlamydia caviae GPIC, Arabulopsis thaliana, Lycopersicum esculentum, _0ryza sativa,Alcaligenes. eutrophus,Neisseria meningitidis (serogroup.iA),.M2WJen"a meningitidis (serogroup B, two GTP cyclohydrolase n enzymes), Pseudomonas pufida (two GTP cyclohydrolase n enzymes), Pseudomonas syringae (two GTP cyclohydrolase H enzymes), Actinobacillus actinomycetemcomitans (Haemophilus actinomycetemcomitans), Haemophilus influenzae, Pasteuretta multocida, Escherichia coli, Escherichia coli O6, Salmonella typhimurium, Yersiniapestis, Bucknera aphidicola (subsp. Acyrthosiphon pisum) (Acyrthosiphon pisum symbiotic bacterium), Buchnera aphidicola (subsp. Schizaphis gramtmmi), Wigglesworthia glossinidia brevipalpis, Buchnera aphidicola (subsp. Baizongia pistaciae), Mycobacterium leprae, Mycobacterium tuberculosis, Coiynebacterium efficiens, Corynebacterium glutamicum, Corynebacterium ammoniagenes (Brevibacterium amrnoniagenes), Staphylococcus aureus, Staphylococcus epidermidis, Actinobacillus pleuropneumoniae, Lactococcus lactis (Streptococcus lactis), Streptococcus agalactiae, Streptococcuspneumoniae, Clostridium acetobutylicum, Fusobacterium nucleatum, Anabaena spec., Synechocystis spec., Synechococcus elongatus (Thermosynechococcus elongatus'), Bactfhis amyloliquefaciens, Bacillus subtilis, Bacillus<br><br>
cereus, Bacillus halodurans, Clostridiumperfringens., Clostridium tetani, Chlorobium tepidum, Aquifex aeolicus, Leptospira interrogans, Deinococcus radiodurans, Bacteroides thetaiotaomicron, Caulobacter crescentus, Coxiella burnetii, Khizobium etli, Lactobacillvs plantarum, Pseudornonas glumae, Streptomyces avermitilis, Photobacterium phosphoreum, Azospirillum brasilense, Agrobacterium tumefaciens, Rhizobivm meliloti (Sinarhizobium meliloti), Brucella melitensis, Brucella sins, Rhizobium loti (Mesorhizobium loti), Nitrosamonas europdea,Ralstonia solanacearum (Pseudomonas solanaceanari), Xanthamonas axonopodis, Xanthomonas campestris, Vibrio parahaemolyticus, Vibrio vuhuficus, Vibrio cholerae, Vibrio fischeri, Shewanella oneidensis, Photobacterium phosphoreum, Photobacterium leiognathi, Pseudomonas aeruginosa, Dehalospirillum multivorans, Xylellafastidiosa). Numbering relates to the alignment made. Some of the amino acid sequences code for an enzyme that has just GTP cyclohydrolase II activity like the enzymes tiomAshbya gossypii, Streptomyces coelicolor, Helioobacter pylori J99, Heliobacter pylori, Arabidopsis tkaliana, Alcaligenes eutrophus, Neisseria meningitidis (serogroup A), Neisserta meningitidis (serogroup B), Pseudomonas putida, Pseudomonas syringae, Actindbadllus actinomycetemcomitans (Haemophilias actinomycetemcomitans), Haemophilus influenzae, Pasteurella multocida, Escherichia coli, Escherichia coli O6, Salmonella typhimtrrium, Tersiniapestis, Buchnera aphidicola (subsp. Acyrthosiphon pisum) (Acyrthosiphon pisum aymbiotic bacterium)., Buchnera aphidicola (subsp. Schizaphis graminum), Wiggleswarthia glossinidia brevipalpis, Buchnera aphidicola (subsp. Baizongiapistaciae), Pseudomonas glumae, Streptomyces avermitilis, or Photobacterium phosphoreum. Other enzymes like the RibA enzyme fix»m B. subtilis contain, in addition, a domain having DEEP synthase activity. The amino acid sequence of RibA from B. subtilis is underlined. Positions that are homologous/equivalent  acid-residues found to have a j&gt;ositive,.effeptj,on .specific, actavityXanTdno acid residues 261,270,276,279,308,347) and on protease sensitivity (196) of RibA from B.-subtilis and that are discussed in one of the following examples are in bold letters. The numbering used for those positions is done according to the B. subtilis wild-type amino acid sequence. The figure starts with the name of the sequences used, the database accession number, and in parenthesis the source organism of the sequence.<br>
The following non-limiting examples further illustrate the invention.<br>
Example 1:   Measurement of GTP cyclohydrolase n activity and determination of specific activity<br>
The enzymatic assay used for measuring GTP cyclohydrolase E activity was adapted from Rite et al (J. Biol. Chem. 276,22273-22277,2001). The final assay buffer contained 50 mM Tris/HCl, pH 8.5,10 mM MgCl2,7.5 mM mercaptoethanol, 2.5 mM GTP and 0.1 nag/ml bovine serum albumin. After purification (see Example 5), the enzyme was kept in<br><br>
a buffer containing 50 mM Tris/HCl, pH 8.5,10 mM MgCb, 7.5 mM mercaptoethanol, and 10% glycerol. Substrate was added to the enzyme and the absorption at 310 nm, at which GTP shows no absorption, was followed over 20-30 rain. The final reaction mixture contained between 0.02 and 0.04 nag/ml GTP cyclohydrolase n from J. subtilis or one of the mutants as shown in Table 1 or 2, An absorption coefficient of 6.28 [mM"1 cm" l] for DRAPP was used for tiie calculation of the activity. Protein determination was done with the Protein Assay from Bio-Rad (Cat No. 500-0002, Bio-Rad Laboratories AG, Nenzlingerweg 2, CH-4153 Reinach, Switzerland).<br>
According to the definition of "specific activity" given above, one unit is the amount of RibA that catalyzes the formation of 1 jumol DRAPP per minute under the conditions as described above. The specific activity is the amount of DRAPP that is formed by 1 mg of RibA per minute under the conditions as described above. Using the aforementioned definitions, the specific GTP cyclohydrolase H activity of the Hiss-tagged RibA protein of JB. subtilis was 0.115 U/mg.<br>
Example 2:    Testing of the quality of the enzymatic assay<br>
An optimal assay should fulfill a number of requirements, such as linearity with enzyme concentration and linearity with time. Using the conditions described in Example 1 and 22 /ig enzyme, the increase in the absorption at 310 nm was followed over 25 min. To test in which range the assay is linear with the enzyme concentration, the dependence of the assay on increasing enzyme concentration (0-40 /ig Hise-tagged RibA) was tested. The assay proved to be linear over 25 min and between 0 and 40 jtg Base-tagged RibA from B. subtilis.<br>
After this the dependence of therGTP'cyclohydrolase II activity o£SiS6»tagged-RibA-from -B. subtilis on GTP concentration was tested. The conditions as described in Example"1" were used. However, the GTP concentration was varied from 0.05 to 2.5 mM final concentration. The data indicate a-Kin value for GTP of 0.07 mM and a specific activity of around 115 mU/mg protein at 37°C for the GTP cyclohydrolase n activity of the Hasg-tagged RibA enzyme from B. subtilis. The experiments of this example showed that the GTP cyclohydrolase H activity assay, in fact, is linear with time and enzyme (GTP cyclohydrolase H) concentration, and that under the given conditions for Bacillus subtilis GTP cyclohydrolase H, a GTP concentration of 2.5 mM maybe optimal to allow reliable measurements of the specific activity of the enzyme.<br>
Example 3:   Isolation of genomic DNA from Bacillus subtilis<br>
B, subtilis was grown at 30°C in Veal Infusion Broth (Becton Dickinson, Sparks, MD 21152, USA) overnight. 1.5 ml culture was transferred into a 1.5 ml tube and •centrifuged.<br><br>
The cell pellet was resuspended in 0,5 ml suspension buffer (50 mM Tris/HCl, pH 7.5, 50 mM NaEDTA, 15% sucrose and 1 nag/ml freshly added lysozyme). After 10 mm incubation at room temperature 1 /xl diethyloxydiformate was added. Then 10 /il of 10% SDS solution was added and the tube was inverted several times. The tube was incubated for 5 min at 70°C to release the bacterial DNA. 50 (il 5 M potassium acetate was added, the tube was cooled on ice and left there for 45 min. After this, the sample was centrifuged for 30 min at 4°C. The supernatant was transferred into a new 1.5-ml tube, which was filled (up to 1.5 ml) with ethanol at room temperature. After 5 mm centrifugation, the supernatant was discarded and the DNA pellet was dried. Then the DNA was washed with 70% and 96% ethanol and dissolved in 10 mM Tris/HCl, pH 7.5,1 mM EDTA, and 10 jug/ml RNase A,<br>
Example 4:    Construction of the expression plasmids for expressing ribA coding for GTF cyclohydrolase n and DHBP synthase from B, subtilis and its mutants<br>
Cloning of the ribA gene (SEQ ID NO:1) of B. subtilis that codes for the GTP cyclohydrolase II and the DHBP synthase was done by PCEL Genomic DNA of B. subtilis was isolated according to Example 3. 100 ng of this DNA or of a template coding for a mutated form of the ribA gene were used for.aPCR using primers RibA IS (SEQ ID NO:27) and RibA IAS (SEQ ID NO:28). The following PCR conditions were used: 2 fiM of each primer, 0.2 mM of each nucleotide, 2.5 U of a proof-reading DNA polymerase (Stratagene, Gebouw California, 1101 CB Amsterdam Zuidoost, The Netherlands), and 100 ng genomic DNA in the appropriate buffer as supplied together with the DNA polymerase.<br>
Temperature regulation was as follows:<br>
Stepl:	3minat95°C<br>
Step 2:	30secat95°C<br>
Step 3:	30secat52°C<br>
Step 4:	60secat72°C<br>
Steps 2 to 4 were repeated 3 0-times.<br>
The PCR product of 1.3 kb was used as template for PCR 2, in which primer RibA 1S was replaced by primer RibA 2S (SEQ ID N0:29). The PCR product of this reaction (SEQ 3D NO:3), encoding an N-terminally Hise-tagged version of B. subtilis RibA (SEQ ID NO:4), was separated by agarose gel electrophoresis, eluted from the gel, digested with EcoKL and BamlSi, and ligated into the .EcoRI and BaniHl digested vector pQE60 (Qiagen AG, Hilden, Germany). The plasmid was called pQE60rlbANlhis.<br><br>
Example 5:    Characterization of the wild-type and the mutant enzymes<br>
The generation of mutated enzymes was performed using methods described above and which are known to the skilled person. Mutants of RibA from B. subtilis that were further investigated are depicted in Table 1. All mutant genes were cloned into a pQE60 vector as described in Example 4. All final constructs contained an N-terminal Hisg-tag.<br>
Table 1: Rib A mutants as defined by the amino acid exchanges compared to the wild-type RibA protein of B. subtilis (the numbers define the respective amino acid positions in SEQ E&gt;NO:2)(Table Remove)  The RibA mutant enzymes were expressed from the plasmids of Example 4 and purified as described in "The QiaExpressionist", Qiagen, HUden, Germany, March 2001, edition 5. The enrymatic properties of the purified enzymes (RibA mutants) were analyzed as described in Examples 1 and 2. Table 2 compares the specific GTP cyclohydrolase H activities of the RibA mutants (see Table 1) to that of the GTP cyclohydrolase H of the wild-type RibA of B. subtilis. The activity was measured using the N-terminally Jffis6-<br>
tagged enzyme versions of Rib A as described in Example 4. The numbers define the respective arnino acid positions in SBQ ID NO:2.<br>
ale 2: Comparison of the specific GTP cyclohydrolase n activities of mutated and wild-type (WT) B. subtilis RibAs (all N-terminally Hise-tagged)<br>
The amino add replacements in parentheses have most probably no effect-on GTP cyclohydrolase n activity of me mutants. Amino acid exchange Y196C reduces the ~" protease sensitivity of RibA.<br>
Example 6:    Construction of recombinant B. subtilis strains over-expressing RibA mutants that show a higher specific GTP cyclohydrolase EL activity<br>
In the following example, the mutated ribA polynucleotide sequences of RibA Y196C^276TA282T (PCRID), RfcA Y196C,A276T,Q279R^282T,K308RM347I (construct C), and RibA Y196CrA276T,Q279R^282T^308RrF325Y,M347I (construct E) were first introduced into a vector containing the strong constitutive promoter T?vegi, and then further manipulated in E. coli. Transformation of a natural competent JB. subtilis microorganism with the polynucleotide sequence and flanking vector sequences resulted in a B. subtilis strain over-expressing the mutated ribA. Standard recombinant DNA techniques were used for the construction of the polynucleotide sequjjjce sad the B.<br><br>
subtilis strains. See, for example, Sambrook et at., Molecular Cloning. A Laboratory Manual (2nd Ed.), Cold Spring Harbor Laboratory Press (1989), andHarwood and Cutting, Molecular Biology Methods for Bacillus, John Wiley and Sons (1990).<br>
To amplify the mutated ribA, a 1.2-kb DNA fragment containing the entire ribA coding sequence was amplified by PCR using DNA from a plasmid containing mutants PCR TIT, construct C or construct E, and RibANde+1 (SEQ ID NO:30) and RibA4AS (SEQ ID NO:31) as primers.<br>
The reaction conditions for the PCR reaction consisted of 30 cycles of denaturation at 95°C for 1 min, annealing at 52°C for 1 min, and extension at 72°C for 2 mm  The Pfii Turbo DNA polymerase (Stratagene, Gebouw California, 1101 CB Amsterdam Zuidoost, The Netherlands) was used to minimize PCR-generated errors. The PCR products were purified using the QIAquick PCR purification kit (Qiagen), and doubly digested rising NdeL and BamHL The digested PCR products were cloned into the pXI16 vector (Huembelin et al, J. hid. Microbiol. Biotechnol. 22,1-7,1999), which consists of suitable restriction sites for the cloning of polynucleotide sequences immediately downstream of the strong constitative Py^ promoter from B. subtilis. The pXI16 vector also contains the cryT transcriptional terminator from B. thuringiensis, the sacB flankhig sequences for homologous recombination into the B. subtilis genome by a double-crossover event, and an erythromycin-xesistance marker. That each plasmid contained the mutated ribA was confirmed by DNA sequencing.<br>
Each plasmid was digested with Apal to remove the spacer region from the Pvegr promoter, re-ligated and digested again withFjpI, and transformed into natural competent B. subtilis 1012 cells. Transformants were selected on TBAB plates (Tryptose Blood Agar Base,<br>
Becton Dickinson, Sparks, MD 21152, USA) containing erythromyein to a final concentration of 2 jKg/ml. DNA sequencing verified that the mutated ribA polynucleotide sequences were correct in these strains. Overproduction of riboflavin was tested according to Example 7.<br>
The mutated ribA polynucleotide sequences driven by the PV^ promoter were introduced into riboflavin over-producing .B. subtilis straiaRB50::(pRP69)n::(pRF93)m, which has been described in Perkins et al, J. Ind. Microbiol. Biotechnol. 22:8-18 (1999), by generalized transduction. Standard techniques using bacteriophage PBS1 were employed according to Harwood and Cutting, Molecular Biology Methods for Bacillus, John Wiley and Sons (1990). Transductants were selected for on TBAB plates containing erythromyein to a final concentration of 2 jig/ml. Transformants were checked by PCR analysis and DNA sequencing to verify correct insertion of the mutated ribA polynucleotide sequence.<br><br>
Example 7:    Improved production of riboflavin using a GTP cyclohydrolase n with increased specific activity<br>
To test the in  effect of mutations affecting the specific activity of GTP cyclohydrolase n, the Bacillus subtilis GTP cyclohydrolase H (RibA) mutants PCR m, construct C, or construct E were introduced into riboflavin over-producing B. subtilis strains, such, as strain RB50::(pKF69y,:j-pRF93)m (Perkins et al, J. Ind. Microbiol. Biotechnol. 22, 8-18, 1999), e.g. in the sacB locus. The production of riboflavin was compared directly in two recombinant strains of B. subtilis that differ only by the presence or absence of the mutations in the ribA gene. Culturing of the Bacillus strains was done as described in Example 8.<br>
Example 8:    Culture conditions for evaluating riboflavin production<br>
Riboflavin production was tested in fed-batch cultivations of ribofLavin-overproducing B. subtilis strain RB50::(pRF69)n::(pRF93)inin which the GTP cyclohydrolase II mutants PCR HL, construct C, or construct E driven by the ?vegi promoter were integrated in the sacB locus (see Example 6). Fermentation of the strains was done as described in EP 405370.<br>
Example 9:   Analytical methods for determination of riboflavin<br>
For determination of riboflavin, the following analytical method can be used (Bretzel et al, J. Ind. McrobioL Biotechnol, 22,19-26,1999).<br>
The chromatographic system was a Hewlett-Packard 1100 System equipped with a binary pump; a column thermostat and a poolei.atttQsampleri,;B.pth a^ode^arrayjdetector and a fluorescence detector were used in line. Two signals wereTecorded, TJV at 280 nm and-a -fluorescence trace at excitation 446 nm, emission 520 nm,<br>
A stainless-steel Supelcosil LC-8-DB column (150 x 4.6 mm, 3 (an. particle size) was used, together with a guard cartridge. The mobile phases were 100 mM acetic acid (A) and methanol (B). A gradient elation according to the following scheme was used: <br>
(Table Remove) The column temperature was set to 20°C, and the flow rate was 1.0 ml/min. The run time was 25 min.<br>
Fermentation samples were diluted, filtered and analyzed without further treatment. RLboflavin was quantitated by comparison with an external standard. The calculations were based on the UV signal at 280 nm. Riboflavin purchased from Fluka (9471 Buchs, Switzerland) was used as standard material (purity £ 99.0%).<br>
Example 10: Identification of corresponding residues in GTP cyclohydrolase n<br>
enzymes that are homologous to Bacillus subtilis GTP cyclohydrolase n<br>
A multiple amino acid sequence alignment of 92 different GTP cyclohydrolase n enzymes found by the program BLASTN using standard databases such as SWISS-PROT and TrEMBL (see Figure 1) was calculated with the program "PILETJP" (GCG Wisconsin Package, version 10.2, Accelrys Inc., 9685 ScrantonRoad, San Diego, CA 92121-3752, USA) using the following parameters: gap creation penalty 8, gap extension penalty 2, and blosum62.cmp matrix (default parameters).<br>
A homologous GTP cyclohydrolase H in the context of the present invention may show sequence similarity with any of the GTP cyclohydrolase IE amino acid sequences shown in Figure 1. Figure 1 provides an example of a multiple sequence alignment of 92 GTP cyclohydrolase II amino acid sequences, with the sequence of the GTP cyclohydrolase n    • from Bacillus subtilis being underlined. Figure 1 just serves as an example and is not meant to be a complete collection of all known GTP cyclohydrolase n enzymes. Homologous residues, Le. residues of the different GTP cyclohydrolase It enzymes that are located at the same position in the amino acid sequence alignment (i e,, are located in the same column in, e.g., Figure 1), are expected to be similarly positioned in the 3D structure of each protein and to fulfill in each protein a comparable function structure-wise and function-wise. Amino -acid resio^s.horaoJ£&gt;gctus.ipJtherjiTnJinpJ<br>
cyclohydrolase H fromS. subtilis that are discussed in the Examples are highlighted in bold in Figure 1, and the respective position in the B. subtilis amino acid sequence is added above the respective column of the alignment.<br>
Amino acid residues of 92 different organisms corresponding to specific amino acid positions, i.e. positions that are homologous/equivalent to the amino acid residues found to have a positive effect on specific activity (amino acid residues 261, 270, 276, 279, 308, 347) of the amino acid sequence of Bacillus subtilis GTP cyclohydrolase E (SEQ ID NO:2) are summarized in Table 4, wherein the number in the left column defines the organism (in accordance to Fig. 1), starting with the name of the sequence used, the database accession number, and in parenthesis the source organism of the sequence:<br>
(1)	gch2_bacsu: SWISS-PROT: gch2_bacsu (Bacillus subtilis)<br>
(2)	gch2_cangu: geneseqp:aay69776 (Candida guiBiennondii)<br>
 (3)	gch2_ashgo: TrEMBL: CAA02912: (Aahbya gossypii (Eremothecium gosypii))<br>
(4)	gch2_yeast: SWISS-PROT: gdh2_yca^t (SaccharcHnyces cerevisiae)<br>
(5)	gch2_neucr: TrEMBL: Q871B3 (Neurospora crassa)<br><br>
(6)	gch2_schpo: TrEMBL: Q9P7M9 (Sciiizosaccharomyces pombe)<br>
(7)	gch2_arcfu: SWISS-PROT: gch2_arcfti (Arcnaeoglobus fulgidus)<br><br>
(8)	gcfc2_strco: SWISS-PROT: gch2_strco (Streptomyces coelicolor)<br>
(9)	gcb2 Jielpj: SWISS-PROT: gch2_b.elpj (Helicobacter pylori J99)<br>
(10)	gch2_helpy; SWISS-PROT: gch2_helpy (Heliobacter pylori)<br>
(11)	gch2_pyrfu: TrEMBL: Q8U4L7 (Pyrococcus furiosus)<br>
(12)	gah2_thema: SWISS-PROT: gch2_1:hema (Thermotoga maritime)<br>
(13)	gch2_chlmu: SWISS-PROT: gc^_chlmu (Chlamydia muridarum)<br>
(14)	goh2_chlte SWISS-PROT: gch2_chltr (Chlamydiatracliomatis)<br>
(15)	gch2_ohlca: TrEMBL: AAP05635 (Chlamydia caviae GPIC)<br>
(16)	gch2_chlpn: SWISS-PROT: gdh2_chlpn (Chlamydia pneumoniae)<br>
(17)	gdi2_arath: SWISS-PROT: gch2_arath (Arabidopsis thaliana)<br>
(18)	gch2_lyces: TrEMBL: CAC09119 (Lyoopersicum esculentum)<br>
(19)	gch2_orysa: TrEMBL: AAO72560 (Oryza sativum)<br>
(20)	gch2_alceu: TrEMBL: Q9F1 84 (Alcaligenes eutroplius)<br>
(21)	gch2_neima: SWISS-PROT: gdh2jneama (Neisseriameningitidis (serogroup A))<br>
(22)	gch2_neimb': SWISS-PROT: gch2_neimb (TSfasseriameniiigitidis (serogroup B))<br>
(23)	gch2_pseplc SWISS-PROT: gch2_psepk (Pseoidomonas putida (strain KT2440))<br>
(24)	gch2_psesm: SWISS-PROT: gch2_psesm (Pseudomonas syringae (pv. tomato))<br>
(25)	goh2_actac: TrEMBL: Q9JRRO (Actinobaoillus actinomycctemcomitans<br>
(Haeonophilus actinomycetemcomitans))<br>
(26)	gch2_haem: SWISSrPROT: gch2^amm.gcb2Jiaeaii ^ (Haemophilus Mluenzae)<br>
(27)	gch2_pasmu: SWISS-PROT: (Pasteurella multocida)	<br>
(28)	gch2__ecO6: TrEMBL: Q8FETLJ5 (Esoherichia coli O6)<br>
(29)	gch2_ecoli: SWISS-PROT: gch2_ecoli (Escherichia coli)<br>
(30)	gcb2_salry: TrEMBL: Q8XFY7 (Salmonella typhimiiriiim)<br>
(31)	gch2_^erpe: TrEMBL: Q8ZEFO (Yersiniapestis)<br>
(32)	gdh2J)ucai: SWISS-PROT: gch2_bucai (Buclmera apbidioola (subsp, Acyrthosiphoi<br>
pisum) (Acyrfhosiplionpisum symbiotic bacterium.))<br>
(33)	gch2j3ucap: SWISS-PROT: gch2_bucap (Bucbnera aphidicola (subsp. Scbizapbis<br>
(34)	gch2_wigbr: SWISS-PROT: gch2_wigbr (Wigglesworfbia glossinidiabrevipalpis)<br>
(35)	gch2_bucbp: SWISS-PROT: gch2_wigbr (Buchnera aphidicola (subsp. Baizongia<br>
pistaciae))<br>
(36)	gch2_mycle: TrEMBL: Q9CCP4 (Mycobacterium leprae)<br>
(37)	gch2_myctu: SWISS-PROT: gch2_myctu (Mycobacteri-om tuberculosis)<br>
 (38)	gch2_coref: TrEMBL: Q8FT57 (Corynebacterium efficiens)<br>
(39)	gch2_corgl: GENESEQP: AAB79913 (Corynebacterium glutanricum)<br>
(40)	gch2_coram: SWISS-PROT: gch2_coram (Corynebacterium ammoniagenes<br>
(Brevibacterium arnmoniagenes))<br>
(41)	gch2_staau: TrEMBL: Q8NW14 (Staphylococcus aureus (strain MW2))<br>
(42)	gcb2_staep: GENESEQP: ABP40248 (Staphylococcus epidermidis)<br>
(43)	gcb2_actpl:SWlSS-PROT:gcb2_actpl(A(rtinobacilliispleuropneumoniae)<br>
(44)	gcb2_lacla: TrEMBL: Q9CGU7 (Lactococcus lactis (subsp. lactis) (Streptococcus<br>
lactis))<br>
(45)	gcb2_stcag: TrEMBL: .Q8E658 (Streptococcus agalactiae (serotype ffl))<br>
(46)	gch2_stepn: TrEMBL: Q8DKF1 (Streptococcus pneumoniae (strain ATCC BAA-<br>
255/R6))<br>
(47)	gch2_cloac: TrEMBL: Q97LG9 (aostridium acetoburylicum)<br>
(48)	gch2_fusnu: TrEMBL: Q8RIR1 (Fusobacterium nucleatum (subsp. nucleatum))<br>
(49)	gch2_anasp: TrEMBL: Q8RIR1 (Anabaena sp. (strainPCC 7120))<br>
(50)	gcli2_syny3: SWISS-PROT: gcfa2_syny3 (Synecb-ocystis sp. (strainPCC 6803))<br>
(51)	gch2_synel: TrEMBL: Q8DI64 Synechococcus elongatus (Thermosynechococcyos<br>
elongatos)<br>
(52)	gch2_bacam: SWISS-PROT: gcb2_bacam (Bacillus amylottqnefaciens)<br>
(53)	gch2_bacce: TrEMBL: AAP11030 (Bacillus cereus ATCC 14579)<br>
(54)	gch2__bacha: TrEMBL: Q9KCL5 (Bacillus halodurans)<br>
(55)	gcli2_clope: TrEMBL: Q8XMXO (Clostridium perfiragens)<br>
(56)	gci2_clote: .TrEMBL: Q897Q8 (Clostridium tetani)<br>
(57)	gch2_chlte: TrEMBL: Q8KC35 (Chlorobium tepidom)<br>
(58)	gcb2^aquae; SWISS-PROT: gch2_aqaae (Aquifex aeolicus)<br>
(59)	gck2_lepin:TrEMBL: Q8F701 (LeptospiTawmterrogans)'J' "'<br>
(60)	gch2_deira: TrEMBL: Q9RXZ9 (Deinococcus radiodurans)<br>
(61)	gch2_bactb.: TrEMBL: Q8A528 (Bacteroides thstaiotaomicron)<br>
(62)	gch2 caucr: TrEMBL: Q9A9S5 (Caulobacter orescentus)<br>
(63)	gch2_coxbu: TrEMBL: AA090191 (CoxieUa burnetii RSA 493)<br>
(64)	gch2_rhiet: TrEMBL: Q8KL38 (Rhizobium etH)<br>
(65)	gcb2_lacpl: TrEMBL: Q88X17 (Lactobacillus plantamm)<br>
(66)	gch2_psegl: TrEMBL: Q8RS38 (Pseudomonas glumae)<br>
(67)	gcb2_strav: TrEMBL: BAC71833 (Streptomyces avermitilis)<br>
(68)	gch2_pliopo: S"WISS-PROT: gch2_phopo (Photobacterium pbosphoreum)<br>
(69)	gcb2_azobr: SWISS-PROT: gck2_azobr (Azospirillumbrasilense)<br>
(70)	gcb2_agrtu: TrEMBL: Q8UHC9 (Agrobacterium tumefaciens (strain C58 / ATCC<br>
33970))<br>
(71)	gcb2_rhime: TrEMBL: Q92RH2 (Rbizobium msliloti (Sinorbizobium meliloti))<br><br>
(72)	gch2_brume: TrEMBL: Q8YFL5 (BruceUa melitensis)<br>
(73)	gch2_brusu: TrEMBL: Q8G298 (Brucellasuis)<br>
(74)	gch2_rhilo: TrEMBL: Q985Z3 (Rbizobium loti (Mesorbizobium loti))<br>
(75)	goh2_braja: TrEMBL: Q89RZ7 (Bradyrhizobium japonicum)<br>
(76)	gch2__niteu: TrEMBL: CAD86468 (Nitrosomonas europaea ATCC 19718)<br>
(77)	gch2_ralso: TrEMBL: Q8Y1H7 (Ralstonia solanacearum (Pseudomonas<br>
solanacearum))<br>
(78)	gch2_neime: TrEMBL: Q9JZ77 (Neisseriameningitidis (serogroup B, second<br>
enzyme found))<br>
(79)	gch2_xanax: TrEMBL: Q8PPD7 (Xanthomonas axonopodis (pv. citri))<br>
(80)	gch2_xanca: TrEMBL: Q8PCM8 (Xanthomonas campestris (pv. campestris))<br>
(81)	gch2_vibpa: TrEMBL: Q87RU5 (Vibrio parahaemolyticus)<br>
(82)	goh2_vibvu: TrEMBL: Q8DF98 (Vibrio vulnificus)<br>
(83)	gch2_vibcli: TrEMBL: Q9KPU3 (Vibrio cholerae)<br>
(84)	gch2_vibfi: TrEMBL: Q8G9G5 (Vibrio fischeri)<br>
(85)	gcb2_sheon: TrEMBL: Q8EBP2 (Shewanella oneideosis)<br>
(86)	gch2_plioph: TrEMBL: Q8G9H7 (Ehotobacteriunipb.osplioreura)<br>
(87)	ribb_pb.olB: TrEMBL: Q93E93 (PhotobactOTum Idognafei)<br>
(88)	gch2_psepu: TrEMBL: Q88GB1 (Pseudomonas putida(straiaK.T2440) second<br>
enzyme found))<br>
(89)	gch2_psesy: TrEMBL: Q882GO (Pseudomonas syringae (pv. Tomato, second<br>
enzyme found))<br>
(90)	gch2jaseae: TrEMBL: Q9HWX4 (Pseudomonas aeragtnosa)<br>
(91)	ribb_dehmu: SWBS-PROT: ribb_dehmu (T&gt;ehalospirillum multivorans)<br>
(92)	gch2_xylfa: TrEMBL: Q87D69,(Xylella fastidiosa (strain TemecuJai / ATCC<br>
700964))<br>
Table 4: Positions/ammo acid residues corresponding to positions V261, G270, A276, Q279, K308, and M347 of RibA of B. subtilis as of SEQ ID NO:2. The numbers in the left column refer to the different organisms (see above). (Table Remove) The examples shown in Table 4 serve as iUustration of the principle. Corresponding residues canbe determined for all other GTP cyclohydrolase H amino acid sequences that areliomologoTis to any one of the sequences shown in Figure 1 or Table 4.<br><br>
1.	A modified GTP cyclohydrolase II, wherein<br>
(i)     the specific activity of the modified enzyme is increased in comparison to the corresponding non-modified enzyme, and<br>
(ii)    the amino acid sequence of the modified enzyme comprises one or more mutation(s) including 1, 2, 3,4, 5, or 6 mutation(s) on amino acid position(s) corresponding to positions 261, 270, 276, 279, 308 and/or 347 of SEQ ID NO:2,<br>
2.	The modified GTP cyclohydrolase II according to claim 1 wherein the modified GTP<br>
cyclohydrolase II exhibits a specific activity that is at least about 10% higher in comparison to<br>
the corresponding non-modified GTP cyclohydrolase II.<br>
3.	The modified GTP cyclohydrolase II according to claim 1 or 2 wherein the one or more<br>
mutation(s) is one or more substitution(s).<br>
4.	The modified GTP cyclohydrolase II according to any one of claims 1 to 3 wherein the<br>
sequence of the non-modified GTP cyclohydrolase II is selected from the group consisting of<br>
sequence ID NOs:2,33, 35, 37, 39, 41, and 43.<br>
5.	The modified GTP cyclohydrolase II according to any one of claims 1 to 4 comprising a<br>
mutation on an amino acid position corresponding to a position of SEQ ID NO:2 which is<br>
selected from the group consisting of:<br><br>
(a)	position 261 of SEQ ID NO:2, preferably a replacement of valine by alanine.<br>
(b)	position 270 of SEQ ID N0:2, preferably a replacement of glycine by alanine or<br>
arginine;<br>
(c)	position 276 of SEQ ID N0:2, preferably a replacement of alanine by threonine;<br>
(d)	position 279 of SEQ ID N0:2, preferably a replacement of glutamine by arginine;<br>
(e)	position 308 of SEQ ID N0:2, preferably a replacement of lysine by arginine; and<br>
(f)	position 347 of SEQ ID N0:2, preferably a replacement of methionine by isoleucine.<br><br>
6.	A polynucleotide comprising a nucleotide sequence which codes for a modified GTP<br>
cyclohydrolase II according to any one of claims 1 to 5.<br>
7.	The polynucleotide according to claim 6 wherein the nucleotide sequence which codes<br>
for a modified GTP cyclohydrolase II is selected from the group consisting of sequence ID<br>
N0s:6, 8,10,12,14,16,18, 20,22, 24 and 26.<br><br>
8.	A host cell comprising a polynucleotide according to any one of claims 6 to 7.<br>
9.	The host cell according to claim 8 which is selected from the group consisting of<br>
Bacillus subtilis, Candida flareri, Eremothecium ashbyii, Ashbya gossypii, and<br>
Saccharomyces cerevisiae.<br>
10.	A method for producing riboflavin, a riboflavin precursor, FMN, FAD, or a derivative<br>
thereof comprising:<br><br>
(a)	culturing the host cell according to any one of claims 8 to 9 in a suitable medium; and<br>
(b)	optionally separating riboflavin, a riboflavin precursor, FMN, FAD, or a derivative<br>
thereof from the medium.<br>
11.	A process for producing a modified GTP cyclohydrolase II according to any one of<br>
claims 1 to 5 comprising:<br>
(a)	culturing a population of host cells according to any one of claims 8 to 9 in a suitable<br>
medium; and<br>
(b)	optionally recovering the modified GTP cyclohydrolase II from the cells or from the<br>
medium.<br>
*<br>
12.	A process for producing a GTP cyclohydrolase II having increased specific activity,<br>
comprising the following steps:<br>
(a)	providing a polynucleotide encoding a GTP cyclohydrolase II;<br>
(b)	introducing one or more mutatidri(s) Into the pdlynucleofide sequence such That the<br>
mutated polynucleotide sequence encodes a modified GTP cyclohydrolase II which has a<br>
higher specific activity than the non-modified GTP cyclohydrolase II and which comprises<br>
one or more mutation(s) including 1,2, 3,4, 5, or 6 mutation(s) on amino acid position(s)<br>
corresponding to positions 261, 270,276,279, 308 and/or 347 of SEQ ED NO:2;<br>
(c)	optionally functionally linking the mutated polynucleotide with a promoter, a ribosome-<br>
binding site and a terminator or inserting the mutated polynueleotide-in-a-vector or plasmid;<br>
(d)	introducing the polynucleotide, the transcriptionally functional polynucleotide or the<br>
vector or plasmid into a suitable host cell; and<br>
(e)	culturing the host cell under conditions that allow expression of the modified GTP<br>
cyclohydrolase II.<br><br>
13.	The use of a modified GTP cyclohydrolase II according to any one of claims 1 to 5 or a<br>
polynucleotide according to any one of claims 6 to 7 for increasing the production of<br>
riboflavin, a ribofiavin precursor, FMN, FAD, or a derivative thereof.<br>
14.	A process for the preparation of a GTP cyclohydrolase II having increased specific<br>
activity comprising:<br><br>
(a)	providing a polynucleotide encoding a first GTP cyclohydrolase II with a specific<br>
activity that, desirably, should be increased;<br>
(b)	providing the positions that have an effect on the specific activity;<br>
(c)	defining the optimal amino acid for replacement of a given amino acid of the wild-type<br>
GTP cyclohydrolase II as defined in (b) and introducing one or more mutation(s) into the<br>
polynucleotide sequence of (a) at the positions defined in (b) such that the mutated<br>
polynucleotide sequence encodes a new GTP cyclohydrolase II;<br>
(d)	optionally inserting the mutated polynucleotide in a vector or plasmid;<br><br>
(d)	introducing the polynucleotide or the vector or plasmid into a suitable host cell; and<br>
(e)	culturing the host cell under conditions that allow expression of the modified GTP<br>
cyclohydrolase II.<br></td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTIyLWRlbG5wLTIwMDctQWJzdHJhY3QtKDE2LTA3LTIwMTMpLnBkZg==" target="_blank" style="word-wrap:break-word;">922-delnp-2007-Abstract-(16-07-2013).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTIyLWRlbG5wLTIwMDctYWJzdHJhY3QucGRm" target="_blank" style="word-wrap:break-word;">922-delnp-2007-abstract.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTIyLWRlbG5wLTIwMDctQ2xhaW1zLSgxNi0wNy0yMDEzKS5wZGY=" target="_blank" style="word-wrap:break-word;">922-delnp-2007-Claims-(16-07-2013).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTIyLWRlbG5wLTIwMDctQ2xhaW1zLSgyNS0xMS0yMDE0KS5wZGY=" target="_blank" style="word-wrap:break-word;">922-delnp-2007-Claims-(25-11-2014).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTIyLWRlbG5wLTIwMDctY2xhaW1zLnBkZg==" target="_blank" style="word-wrap:break-word;">922-delnp-2007-claims.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTIyLWRlbG5wLTIwMDctQ29ycmVzcG9uZGVuY2UgT3RoZXJzLSgxNi0wNy0yMDEzKS5wZGY=" target="_blank" style="word-wrap:break-word;">922-delnp-2007-Correspondence Others-(16-07-2013).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTIyLWRlbG5wLTIwMDctQ29ycmVzcG9uZGVuY2UgT3RoZXJzLSgyNS0xMS0yMDE0KS5wZGY=" target="_blank" style="word-wrap:break-word;">922-delnp-2007-Correspondence Others-(25-11-2014).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTIyLURFTE5QLTIwMDctQ29ycmVzcG9uZGVuY2Utb3RoZXJzICgyMy0wNi0yMDA4KS5wZGY=" target="_blank" style="word-wrap:break-word;">922-DELNP-2007-Correspondence-others (23-06-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTIyLWRlbG5wLTIwMDctQ29ycmVzcG9uZGVuY2Utb3RoZXJzLSgxNS0wMS0yMDEzKS5wZGY=" target="_blank" style="word-wrap:break-word;">922-delnp-2007-Correspondence-others-(15-01-2013).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTIyLURFTE5QLTIwMDctQ29ycmVzcG9uZGVuY2UtT3RoZXJzLnBkZg==" target="_blank" style="word-wrap:break-word;">922-DELNP-2007-Correspondence-Others.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTIyLWRlbG5wLTIwMDctZGVzY3JpcHRpb24gKGNvbXBsZXRlKS5wZGY=" target="_blank" style="word-wrap:break-word;">922-delnp-2007-description (complete).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTIyLWRlbG5wLTIwMDctZHJhd2luZ3MucGRm" target="_blank" style="word-wrap:break-word;">922-delnp-2007-drawings.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTIyLWRlbG5wLTIwMDctZm9ybS0xLnBkZg==" target="_blank" style="word-wrap:break-word;">922-delnp-2007-form-1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTIyLURFTE5QLTIwMDctRm9ybS0xMy0oMDItMDItMjAwNykucGRm" target="_blank" style="word-wrap:break-word;">922-DELNP-2007-Form-13-(02-02-2007).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTIyLWRlbG5wLTIwMDctZm9ybS0xMy5wZGY=" target="_blank" style="word-wrap:break-word;">922-delnp-2007-form-13.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTIyLURFTE5QLTIwMDctRm9ybS0xOCAoMjMtMDYtMjAwOCkucGRm" target="_blank" style="word-wrap:break-word;">922-DELNP-2007-Form-18 (23-06-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTIyLWRlbG5wLTIwMDctZm9ybS0yLnBkZg==" target="_blank" style="word-wrap:break-word;">922-delnp-2007-form-2.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTIyLWRlbG5wLTIwMDctRm9ybS0zLSgxNi0wNy0yMDEzKS5wZGY=" target="_blank" style="word-wrap:break-word;">922-delnp-2007-Form-3-(16-07-2013).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTIyLURFTE5QLTIwMDctRm9ybS0zLnBkZg==" target="_blank" style="word-wrap:break-word;">922-DELNP-2007-Form-3.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTIyLWRlbG5wLTIwMDctZm9ybS01LnBkZg==" target="_blank" style="word-wrap:break-word;">922-delnp-2007-form-5.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTIyLWRlbG5wLTIwMDctZ3BhLnBkZg==" target="_blank" style="word-wrap:break-word;">922-delnp-2007-gpa.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTIyLWRlbG5wLTIwMDctcGN0LTIxMC5wZGY=" target="_blank" style="word-wrap:break-word;">922-delnp-2007-pct-210.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTIyLWRlbG5wLTIwMDctcGN0LTMwNC5wZGY=" target="_blank" style="word-wrap:break-word;">922-delnp-2007-pct-304.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTIyLWRlbG5wLTIwMDctUGV0aXRpb24tMTM3LSgxNi0wNy0yMDEzKS5wZGY=" target="_blank" style="word-wrap:break-word;">922-delnp-2007-Petition-137-(16-07-2013).pdf</a></p>
		<br>
		<div class="pull-left">
			<a href="264528-synergistic-herbicidal-composition-of-acifluorfen-sodium-and-clodinafop-propargyl.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="264530-an-isolated-bone-morphogenetic-protein-3-bmp-3.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>264529</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>922/DELNP/2007</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>02/2015</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>09-Jan-2015</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>02-Jan-2015</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>02-Feb-2007</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>DSM IP ASSETS B.V.</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td>HET OVERLOON 1, NL-6411 TE HEERLEN, THE NETHERLANDS.</td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>EBERT, SYBILLE</td>
											<td>DOFFINGER WEG 2, 70569 STUTTGART, GERMANY.</td>
										</tr>
										<tr>
											<td>2</td>
											<td>MOUNCEY, NIGEL JOHN</td>
											<td>CURT-GOETZ STRASSE 18, CH-4102 BINNINGEN, SWITZERLAND.</td>
										</tr>
										<tr>
											<td>3</td>
											<td>HOHMANN, HANS-PETER</td>
											<td>JOSEF-PFEFFER-WEG 16, 79540 LORRACH, GERMANY.</td>
										</tr>
										<tr>
											<td>4</td>
											<td>LEHMANN, MARTIN</td>
											<td>TALSTRASSE 21, 79639 GRENZACH-WYHLEN, GERMANY.</td>
										</tr>
										<tr>
											<td>5</td>
											<td>WYSS, MARKUS</td>
											<td>WIDMANNSTRASSE 4, CH-4410 LIESTAL, SWITZERLAND.</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>C12N 9/00</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>PCT/EP2005/007320</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td>2005-07-07</td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td>04015584.8</td>
									<td>2004-07-07</td>
								    <td>EUROPEAN UNION</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/264529-improved-enzymes by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 01:48:28 GMT -->
</html>
