<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/264665-use-of-fungal-mutants-for-expression-of-antibodies by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 02:20:38 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 264665:USE OF FUNGAL MUTANTS FOR EXPRESSION OF ANTIBODIES</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">USE OF FUNGAL MUTANTS FOR EXPRESSION OF ANTIBODIES</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>The present invention relates to a filamentous fungal cell in which an endogenous alkaline protease activity, an endogenous neutral metalloprotease activity and an endogenous serine protease activity of the subtilisin type have been completely or partially inactivated. Particularly the endogenous alkaline protease activity, the endogenous neutral metalloprotease activity and the endogenous serine protease activity of the subtilisin type are encoded by the alp, npl and pepC genes respectively. The filamentous fungal cell is particularly suitable for production of heterologous proteins such as antibodies.</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td> <br><br>
TITLE: USE OF FUNGAL MUTANTS FOR EXPRESSION OF ANTIBODIES<br>
FIELD OF THE INVENTION<br>
The present invention relates to a filamentous fungal cell in which an endogenous alkaline Protease, an endogenous metalloprotease and an endogenous sehne protease activity have been completely or partially inactivated. Furthermore the present invention relates to a method for producing a heterologous Polypeptide in a filamentous fungal host cell.<br>
BACKGROUND OF THE INVENTION<br>
The use of recombinant host cells in the expression of heterologous proteins has in recent years greatly simplified the production of large quantities of commercially valuable pro-teins which otherwise are obtainable only by purification from their native sources. Currently, there is a varied selection of expression Systems from which to choose for the production of any given protein, including bacterial and eukaryotic hosts. The selection of an appropriate expression System offen depends not only on the ability of the host cell to produce adequate yields of the protein in an active State, but, to a large extent, may also be governed by the in-tended end use of the protein.<br>
One Problem frequently encountered is the high level of proteolytic enzymes produced by a given host cell or present in the culture medium. Proteases are a major problem for the production of heterologous proteins in filamentous fungi, since they are present at every stage of the secretion process as well as several extracellular proteases.<br>
It has been suggested that one could provide host organisms deprived of the ability to produce specific proteolytic Compounds. For example, International Patent Application WO 90/00192 (Genencor) describes filamentous fungal hosts incapable of secreting enzymatically active aspartic Proteinase, and EP 574 347 (Ciba Geigy AG) describes Aspergillus hosts de-fective in a sehne protease of the subtilisin-type.<br>
Other examples of proteases which have been reported to reduce the stability of proÂ¬tein products include metalloproteases and alkaline proteases.<br>
WO 1998/012300 describes fungal hosts having improved stability of heterologous protein products in which the host cell have been genetically modified to express significantly reduced levels of both a metalloprotease and an alkaline protease.<br>
Another intracellular protease, a sehne protease of the subtilisin type produced e.g. by A. niger and designated PepC has been described, the gene expressing it cloned, and a dele-tion mutant deschbed in EP 574 347 and in Frederick et al., Gene, 125 57-64 (1993).<br>
Another group of processing proteases is the kexin family of proteases, which recog-nizes dibasic amino acid motifs and are involved in removing N-terminal propeptides. Kexin proteases have a narrow Substrate specificity. Fungal kexin like maturases have been cloned<br><br>
and characterized in Aspergillus niger (Jalving et al., 2000, Applied and Environmental Micro-biology 66:363-368), in Aspergillus nidulans, designated KpcA (Kwon et al., 2001, Molecular Cell 12:142-147). and in Aspergillus oryzae (Mizutani et al., 2004, Eukaryotic Cell 3:1036-1048).<br>
For heterologous production of recombinant proteins in a fungal host the choice of a suitable host resulting in the optimal expression and stability of the recombinant protein is to a large extent dependent on the protein and thus the presence of specific protease recognitlon sites in the protein. The presence of such recognition sites can be very difficult to predict. It is there-fore desirable to identify new mutant fungal strains specifically tailored to the production of a desired protein. The present invention relates to the Identification of desired protease muta-tions In fungal host strains usefui for the production of heterologous proteins and particularly to the comblnation of such mutations.<br>
SUMMARY OF THE INVENTION<br>
The present invention provides such usefui protease deficient strains in which the expression of at least three proteases has been completely or partially abolished. The affected genes according to the invention encode an endogenous alkaline protease activity, an endogenous neutral metalloprotease activity and an endogenous serine protease activity of the subtilisin type. Optionally an endogenous calcium dependent, neutral, serine protease of the kexin subfamily has been completely or partially inactivated.<br>
In a first aspect the present invention relates to a filamentous fungal cell in which an endogenous alkaline protease activity, an endogenous neutral metalloprotease activity and an endogenous serine protease activity of the subtilisin type have been completely or partially inactivated, wherein the alkaline protease, the neutral metalloprotease, and the serine Protease of the subtilisin type are encoded from nucleotide sequences selected from the group consisting of:<br>
(a) a nucleotide sequence encoding an alkaline protease having an amino acid sequence which has at least 65% identity with the amino acids of SEQ ID NO: 30, a nucleotide sequence encoding a neutral metalloprotease which has at least 65% identity with the amino acids of SEQ ID NO: 31, and a nucleotide sequence encoding a serine protease of the subtilisin type having an amino acid sequence which has at least 65% identity with the amino acids of SEQ ID NO: 32; or<br><br>
(b)	a nucleotide sequence having at least 65%  identity with  SEQ  ID  NO:  26,  a<br>
'  nucleotide sequence having at least 65% identity with SEQ  ID NO :  27.  and a<br>
nucleotide sequence having at least 65% identity with SEQ ID NO : 28; or<br>
(c)	nucleotide sequences which hybridizes under at least medium stringency conditions<br>
with either of the nucleic acid sequences SEQ ID NO: 26, 27, or 28.<br>
In a second aspect the present invention relates to a method for producing a heterologous Polypeptide in a filamentous fungal host cell according to the invention, comprising:<br>
(a)	introducing into the host cell a nucleic acid sequence encoding the heterologous Polypeptide;<br>
(b)	cultivating the cell from (a) in a suitable growth medium under conditions conducive for expression of the heterologous Polypeptide; and<br>
(c)	isolating the protein product.<br>
DETAILED DESCRIPTION OF THE INVENTION<br>
The present invention relates to the expression of heterologous proteins in a filamenÂ¬tous fungal host, which has been modified compared to the parent fungal host in order to re-duce the expression of endogenous proteases that would otherwise affect the production yield and stability of the heterologous protein.<br>
In particular the filamentous fungal host cell has been modified in order to reduce or preferably abolish the expression of an endogenous alkaline protease, an endogenous metalÂ¬loprotease, and an endogenous serine protease of the subtilisin type.<br>
Metalloproteases<br>
In the context of this invention, a metalloprotease is a proteolytic enzyme con-taining a catalytic zinc metal center which participates in the hydrolysis of the peptide back-bone of the Substrate. The active zinc centre differentiates these proteases from the calpains, whose activities are dependent upon the presence of calcium. Confirmation of a protease as a metalloprotease is by the reversible loss of proteolytic activity upon removal of the zinc centre with 1 mM 1,10-phenanthroline, and its restoration after titration with Zn2+ (0.1-100 mM).<br>
In particular, the metalloprotease contemplated in the context of this invention is a neutral metalloprotease, which is a metalloprotease possessing optimal proteolytic activity in the neutral pH region, i.e. in the rÃ¤nge of about pH 6-8, preferably in the rÃ¤nge of about pH 6.5-7.5, more specifically, around pH 7. More particularly, the metalloprotease contemplated in the<br><br>
context of this invention is a neutral Aspergillus metalloprotease of the group Npl er Npll (Ta-tsumi, 'et al., 1.991, supra).<br>
In a more particular embodiment, the metalloprotease is an Aspergillus oryzae neutral metalloprotease I (Npl) preferably encoded by a cDNA sequence comprising the nucleotide sequence presented as SEQ ID NO: 27 or a sequence homologous thereto. Particularly the homologous sequence has a degree of identity to SEQ ID NO: 27 of at least 60%. preferably at least 65%, more preferably at least 70%, more preferably at least 75%, more preferably at least 80%, more preferably at least 85%, even more preferably at least 90%. most preferably at least 95%, and even most preferably at least 97%.<br>
Preferably the homologous sequence encodes a protease having an amino acid seÂ¬quence which has a degree of identity to the amino acids of SEQ ID NO: 31 {i.e., the complete Polypeptide including Signal peptide and propeptide) of at least 60%, preferably at least 65%, more preferably at least 70%, more preferably at least 75%, more preferably at least 80%, more preferably at least 85%, even more preferably at least 90%, most preferably at least 95%, and even most preferably at least 97%, which have metalloprotease activity (hereinafter "homologous Polypeptides").<br>
Alkaline proteases<br>
In the context of this invention an alkaline protease is a serine protease with activity which peaks in the neutral to alkaline pH rÃ¤nge. Analyses of the amino acid sequence of alkaÂ¬line proteases indicate homology to the subtilase subgroup of subtilisin-like sehne proteases. As summarised by Siezen, et al. (1991 Protein Eng. 4:719-737) more than 50 subtilases have been identified from a wide variety of organisms, ranging from various species of bacteria. inÂ¬cluding gram positive and gram negative species, to fungi and yeast to higher eukaryotes, inÂ¬cluding Worms, insects, plants and mammals. The amino acid sequences have been deter-mined in more than 40 of these subtilases, and reveal that the mature region of the enzyme ranges from 268 to 1775 amino acids in length and a pre-pro-region of 27 to 280 amino acids in the N-terminal vicinity. In fungi and yeast, the Variation is apparently smaller, with corre-sponding ranges of 279 to 397 and 105 to 121 in fungi, and 297 to 677 and 126 to 280 in yeast. A cDNA fragment of the entire coding region of the alkaline protease from Aspergillus oryzae was cloned and expressed in Saccharomyces cerevisiae (Tatsumi. H, et al. 1989. Mol. Gen. Genet. 219:33-38). The primary structure was shown to share 29% to 44%) homotogy with other sequenced subtilisins, and the three residues in the active site, Asp32, His 64 and Ser221 in subtilisin BPN', were conserved.<br>
In a particular embodiment, the alkaline protease is an Aspergillus oryzae alkaline Protease (a/p), preferably encoded by a cDNA sequence comprising the nucleotide sequence<br><br>
presented as SEQ ID NO: 26 or a sequence homologous thereto. Particularly the homologous sequence has a degree of identity to SEQ ID NO: 26 of at least 60%, preferably at least 65%, more preferably at least 70%, more preferably at least 75%, more preferably at least 80%, more preferably at least 85%, even more preferably at least 90%, most preferably at least 95%, and even most preferably at least 97%.<br>
Preferably the homologous sequence encodes a protease having an amino acid seÂ¬quence which has a degree of identity to the amino acids of SEQ ID NO: 30 {i.e., the complete Polypeptide including signal peptide and propeptide) of at least 60%, preferably at least 65%, more preferably at least 70%, more preferably at least 75%, more preferably at least 80%, more preferably at least 85%, even more preferably at least 90%, most preferably at least 95%, and even most preferably at least 97%, which have alkaline protease activity (hereinafter "homologous Polypeptides").<br>
Serine protease<br>
In the context of this inventlon the protease is a serine protease with a broad rÃ¤nge of activity between pH 4.5 and 11 which are released from a cell wall fraction. Analyses of the amino acid sequence of the serine proteases indicate homology to the subtilase subgroup of subtilisin-like serine proteases. As summarised by Siezen, et al. (1991. Protein Eng. 4:719-737) more than 50 subtilases have been identified from a wide variety of organisms, ranging from varjous species of bacteria, including gram positive and gram negative species, to fungi and yeast to higher eukaryotes, including worms, insects, plants and mammals. The amino acid sequences have been determined in more than 40 of these subtilases, and reveal that the mature region of the enzyme ranges from 268 to 1775 amino acids in length and a pre-pro-region of 27 to 280 amino acids in the N-terminal vicinity. In fungi and yeast, the Variation is apparently smaller, with corresponding ranges of 279 to 397 and 105 to 121 in fungi, and 297 to 677 and 126 to 280 in yeast. Genomic clones of the entire coding region of the serine ProÂ¬tease from Aspergillus oryzae, Aspergillus fumigatus and Aspergillus niger has been cloned (WO97/22705, Reichard et al. 2000. Int. J. Med. Microbiol. 290, 549-558, and Frederick et al 1993. Gene 125. 57-64.). The primary structure was shown to share 29% to 78% homology with other sequenced subtilisins, and the three residues in the active site, Asp32, His 64 and Ser221 in subtilisin BPN', were conserved.<br>
In a particular embodiment, the serine protease of the subtilisin type is an Aspergillus oryzae serine protease (pepC), preferably encoded by a cDNA sequence comprising the nu-cleotide sequence presented as SEQ. ID. NO: 28 or a sequence homologous thereto. ParticuÂ¬larly the homologous sequence has a degree of identity to SEQ ID NO: 28 of at least 60%.<br><br>
preferably at least 65%. more preferably at least 70%, more preferably at least 75%, more preferably at least 80%, more preferably at least 85%, even more preferably at least 90%, most preferably at least 95%, and even most preferably at least 97%.<br>
Preferably the homologous sequence encodes a protease having an amino acid se-quence which has a degree of identity to the amino acids of SEQ ID NO: 32 {i.e., the complete Polypeptide including Signal peptide and propeptide) of at least 60%, preferably at least 65%, more preferably at least 70%, more preferably at least 75%, more preferably at least 80%, more preferably at least 85%, even more preferably at least 90%, most preferably at least 95%, and even most preferably at least 97%, which have serine protease activity (hereinafter "homologous Polypeptides").<br>
In a particular embodiment the filamentous fungal host cell in addition to the above endogenous proteases also has reduced or abolished expression of a serine protease of the kexin subfamily.<br>
Serine protease of the kexin subfamily<br>
Kexin is a Ca2+-dependent transmembrane serine protease that cleaves the secretory proproteins on the carboxyl side of Lys-Arg and Arg-Arg in a late Golgi compartment (FÃ¼ller and Thorner, 1989, PNAS 86:1434-1438; Mizuno et al., 1988, Biochem. Biophys. Res. Commun. 156:246-254). All members of the kexin subfamily are caicium-dependent, neutral serine proteases that are activated by the removal of the amino-terminal propeptide at a kexin-specific (auto) processing site. The active proteases all contain two additional domains, a sub-tilisin-like domain containing the catalytic triad and a conserved P or Homo B domain of ap-proximately 150 residues. The P domain, which is absent in other subtilases, is essential for the catalytic activity and the stability of the protein. Aspergillus kexins are found in Aspergillus nidulans (Kwon et al., 2001, Mol. Cell 12:142-147), A. n/ger (Jalving et al., 2000, Appl. Environ. Microbiol. 66:363-368), and A. oryzae (Mizutani et al., 2004, Eukaryotic Cell 3:1036-1048).<br>
In a particular embodiment, the serine protease of the kexin subfamily is an AspergilÂ¬lus oryzae serine protease (KexÃ), preferably encoded by a cDNA sequence compnsing the nucleotide sequence presented as SEQ ID NO: 29 or a sequence homologous thereto. Rar-ticularly the homologous sequence has a degree of identity to SEQ ID NO: 29 of at least 60%. preferably at least 65%, more preferably at least 70%, more preferably at least 75%, more preferably at least 80%, more preferably at least 85%, even more preferably at least 90%, most preferably at least 95%, and even most preferably at least 97%.<br><br>
Preferably the homologous sequence encodes a protease having an amino acid se-quehce which has a degree of identity to the amino acids of SEQ ID NO: 33 (i, e., the complete Polypeptide including signal peptide and propeptide) of at least 60%, preferably at least 65%, more preferably at least 70%, more preferably at least 75%, more preferably at least 80%, more preferably at least 85%, even more preferably at least 90%, most preferably at least 95%, and even most preferably at least 97%. which have serine protease activity (hereinafter "homologous Polypeptides").<br>
In a particular embodiment the filamentous fungal cell according to the invention has the phenotype alp, np1', pepC, and kexB, wherein the alp gene is encoded by a nucleotide sequence which has at least 70% identity to SEQ ID NO: 26, the np1 gene is encoded by a nucleotide sequence which has at least 70% identity to SEQ ID NO: 27, the pepC gene is enÂ¬coded by a nucleotide sequence which has at least 70% identity to SEQ ID NO: 28, and the kexB gene is encoded by a nucleotide sequence which has at least 70% identity to SEQ ID NO: 29.<br>
In the above the nucleotide sequences referred to are the cDNA sequences (CDS without introns) corresponding to the mature mRNA after splicing.<br>
Hybridization<br>
For purposes of the present invention, hybridization indicates that the nucleic acid sequence hybridizes to at least one of the nucleic acid sequences shown in SEQ ID NO: 26, 27, 28, or 29 under very low to very high stringency conditions. Molecules to which the nucleotide sequence hybridizes under these conditions may be detected using X-ray film or by any other method known in the art. Whenever the term "polynucleotide probe" is used in the present context, it is to be understood that such a probe contains at least 15 nucleotides.<br>
In one embodiment, the polynucleotide probe is the nucleotide sequence shown in SEQ ID NO: 26, 27, 28, or 29 or the complementary Strand of SEQ ID NO: 26, 27, 28, or 29.<br>
In one embodiment hybridization is performed under at least medium stringency conditions, more particularly under at least medium high stringency conditions, and even more particularly under at least high stringency conditions.<br>
For long probes of at least 100 nucleotides in length, very low to very high stringency conditions are defined as pre-hybridization and hybridization at 42Â°C  in 5X SSPE, 1.0% SDS, 5X Denhardt's Solution, 100 |.ig/ml sheared and denatured salmon sperm DNA, following Standard Southern blotting procedures. Preferably, the long probes of at least 100 nucleotides<br><br>
do not contain more than 1000 nucleotides. For long probes of at least 100 nucleotides in length, the carrier material is finally washed three times each for 15 minutes using 2 x SSC, 0.1% SDS at 42Â°C  (very low stringency), preferably washed three times each for 15 minutes using 0.5 x SSC, 0.1% SDS at 42Â°C (low stringency), more preferably washed three times each for 15 minutes using 0.2 x SSC, 0.1% SDS at 42Â°C (medium stringency), even more preferably washed three times each for 15 minutes using 0.2 x SSC, 0.1% SDS at 55"C (medium-high stringency), most preferably washed three times each for 15 minutes using 0.1 x SSC, 0.1% SDS at 60Â°C (high stringency), in particular washed three times each for 15 minutes using 0.1 x SSC, 0.1% SDS at 68Â°C  (very high stringency).<br>
Although not particularly preferred, it is contemplated that shorter probes, e.g. probes which are from about 15 to 99 nucleotides in length, such as from about 15 to about 70 nucleotides in length, may be also be used. For such short probes, stringency conditions are defined as prehybridization, hybridization, and washing post-hybhdization at 5Â°C to 10Â°C  below the caiculated Tm using the caiculation according to Bolton and McCarthy (1962, Proceedings ofthe National Academy of Sciences USA 48:1390) in 0.9 M NaCI, 0.09 M Tris-HCI pH 7.6, 6 mM EDTA, 0.5% NP-40, IX Denhardt's Solution, 1 mM sodium pyrophosphate, 1 mM sodium monobasic phosphate, 0.1 mM ATP, and 0.2 mg of yeast RNA per ml following Standard Southern blotting procedures.<br>
For Short probes which are about 15 nucleotides to 99 nucleotides in length, the carner material is washed once in 6X SCC plus 0.1% SDS for 15 minutes and twice each for 15 minutes using 6X SSC at 5'C to 10Â°C below the caiculated Tm.<br>
Genetic Modifications of the Host Cell<br>
The host cell of the invention, in Order to express significantly reduced levels of metal-loprotease, alkaline protease, and serine protease activity, and optionally serine protease of the kexin subfamily is genetically modified which may be achieved by using Standard recombiÂ¬nant DNA technology known to the person skilled in the art. The gene sequences respectively responsible for production of the protease activity may be inactivated or partially or entirely eliminated. Thus, a host cell of the invention expresses reduced or undetectable levels of met-alloprotease, alkaline protease, and serine protease or expresses functionally inactive prote-<br>
ases.<br>
In a particular embodiment, the said inactivation is obtained by modification of the re-spective structural or regulatory regions encoded within the protease genes of interest. Known and usefui techniques include, but are not limited to, specific or random mutagenesis, PCR<br><br>
generated mutagenesis, site specific DNA deletion, insertion and/or Substitution, gene disrup-tioh or gene replacement, anti-sense techniques, or a combination thereof.<br>
Mutagenesis may be performed using a suitable physical or chemical mutagenising agent, Examples of a physical or chemical mutagenising agent suitable for the present pur-pose include, but are not limited to, ultravlolet (UV) Irradiation, hydroxylamine, N-methyl-N'-nitro-N-nitrosoguanidine (MNNG), 0-methyl hydroxylamine, nitrous acid, ethyl methane sul-phonate (EMS), sodium bisulfite, formic acid, and nucleotide analogues, When such agents are used, the mutagenesis is typically performed by incubating the cell to be mutagenised in the presence of the mutagenising agent of choice under suitable conditions, and selecting for cells showing a significantly reduced production of the protease of choice.<br>
Measurements of the extracellular proteases activity of the alkaline and neutral metal-loprotease I can be done as described by Markaryan et al (1996) J Bacteriology 178, no 8, 2211-2215. In Short the strain is grown in a media which induce the production of extracellular proteases. The broth is then separate from the mycelium and are assay for alkaline and metal-loproteinase activity with the Substrate Suc-Ala-Ala-Pro-Leu-pNa and Abz-Ala-Ala-Phe-Phe-pNa, respectively.<br>
Measurement of the intracellular Kexin is done as described by Jalving et al (2000) Applied and Environmental Microbiology 66, no 1, 363-368. In short strains are grown minimal media (COVE (1966) Biochim. Biophys. Acta 113, 51-56), mycelium is harvested, grounded and the membrane protein fraction were suspended in HEPES buffer and stored at -20Â°C  until used. The isolated membrane protein fraction were analysed for kexin activity by using the Substrate Boc-Leu-Lys-Arg-MCA.<br>
Measurement of pepC which are cell wall bound was done according to Reihard et al. (2000) Int. J. Med. Microbiol. 290, 85-96. In short strains are grown in minimal media (COVE (1966) Biochim. Biophys. Acta 113. 51-56), mycelium is harvested, grounded, and cell wall fragments were suspended in Na-citrate buffer and stored at -20Â°C  until used for Proteinase assays. The isolated cell wall were analyse for pepC activity by an azocasein assay (ScharÂ¬mann and Balke (1974) Physiol. Chem. 355, 443-450.<br>
Modification may also be accomplished by the introduction, Substitution or removal of one or more nucleotides in the structural sequence or a regulatory element required for the transcription or translation of the structural sequence. For example, nucleotides may be in-serted or removed so as to result in the introduction of a stop codon, the removal of the start codon or a change of the open reading frame of the structural sequence. The modification or inactivation of the structural sequence or a regulatory element thereof may be accomplished by site-directed mutagenesis or PCR generated mutagenesis in accordance with methods known in the art.  Although, in principle, the modification may be performed //) vivo, i.e. directly<br><br>
on the cell expressing the metalloprotease, alkaline protease, and serine protease genes, it is presently preferred that the modification be performed in vitro as exemplified below.<br>
A convenient way to inactivate or reduce the said protease production in a host cell of choice is based on techniques of gene Interruption. In this method a DNA sequence corre-sponding to the endogenous gene or gene fragment of interest is mutagenised in vitro. Said DNA sequence thus encodes a defective gene which is then transformed into the host cell. By homologous recombination, the defective gene replaces the endogenous gene or gene fragÂ¬ment- It may be desirable that the defective gene or gene fragment also encodes a marker which may be used to select for transformants in which the respective genes encoding metalÂ¬loprotease and/or alkaline protease have been modified or destroyed.<br>
Methods for deleting or disrupting a targeted gene are specifically described by Miller, et al (1985. Mol. Cell. BioL 5:1714-1721); WO 90/00192; May, G. (1992. Applied Molecular Genetics of Filamentous Fungi. J.R. Kinghorn and G. Turner, eds., Blackie Academic and ProÂ¬fessional, pp. 1-25); and Turner, G. (1994. Vectors for Genetic Manipulation. S.D. Martinelli and J.R. Kinghorn, eds., Elsevier, pp.641-665).<br>
Alternatively, the modification or inactivatlon of the DNA sequence may be performed by established anti-sense techniques using a nucleotide sequence complementary to a coding sequence for a metalloprotease, e.g. the nucleotide sequences presented as SEQ ID NO: 27, an alkaline protease encoding sequence, e.g. the nucleotide sequence shown in SEQ. ID. NO: 26, or a serine protease of the subtilisin type and optionally also of the kexin subfamily, e.g. the nucleotide sequences shown in SEQ ID NO: 28 and SEQ ID NO: 29. The anti-sense technol-ogy and its application are described in detail in US Patent No. 5,190,931 (University of New York).<br>
Therefore, due to genetic modification, the host cell of the invention expresses signifi-cantly reduced levels of metalloprotease, alkaline protease, and serine protease of the subtilÂ¬isin type activity. In a particular embodiment the host cell in addition expresses significantly reduced levels of a serine protease of the kexin subfamiliy. In a particular embodiment, the level of these proteolytic activities expressed by the host cell is individually reduced more than about 50%, preferably more than about 85%, more preferably more than about 90%, and most preferably more than about 95%. In another particular embodiment, these proteolytic activities in the host cell of the invention may be reduced in any combination. In a most particular emÂ¬bodiment, the product expressed by the host cell is essentially free from proteolytic activity due to any of the above proteases.<br>
Methods of Producing Proteins<br>
By the method of the invention, the proteolytic activities of metalloprotease, alkaline Protease, serine protease of the subtilisin type and optionally serine protease of the kexin sub-<br><br>
family are significantly reduced, thereby improving the stability and increasing the yield of sus-ceptible protein products synthesised by the host cell of the invention. More specifically, by the method of the invention, the host cell is genetically modified within structural and/or regulatory reglons encoding or Controlling the metalloprotease, alkaline. serine protease of the subtilisin type and optionally serine protease of the kexin subfamily protease genes.<br>
Therefore, another aspect of the invention provides a method of producing proteins in a host cell of the invention, including heterologous Polypeptides, in which the method com-prises introducing into said host cell a nucleic acid sequence encoding the protein product of interest, cultivating the host cell in a suitable growth medium, followed by recovery of the proÂ¬tein product.<br>
Thus, the host cell of the invention must contaln structural and regulatory genetic re-gions necessary for the expression of the desired product. The nature of such structural and regulatory regions greatly depends on the product and the host cell in question. The genetic design of the host cell of the invention may be accomplished by the person skilled in the art using Standard recombinant DNA technology for the transformation or transfection of a host cell (vide, e.g., Sambrook et al., interalla).<br>
Preferably, the host cell is modified by methods known in the art for the introduction of an appropriate cloning vehicie, i.e- a plasmid or a vector, comprising a DNA fragment encoding the desired protein product. The cloning vehicie may be introduced into the host cell either as an autonomously replicating plasmid or integrated into the chromosome. Preferably, the clonÂ¬ing vehicie comprises one or more structural regions operably linked to one or more appropriÂ¬ate regulatory regions.<br>
The structural regions are regions of nucleotide sequences encoding the desired proÂ¬tein product. The regulatory regions include promoter regions comprising transcription and translation control sequences, terminator regions comprising stop signals, and polyadenylation regions. The promoter, i.e. a nucleotide sequence exhibiting a transcnptional activity in the host cell of choice, may be one derived from a gene encoding an extracellular or an intracellu-lar protein, preferably an enzyme, such as an amylase, a glucoamylase, a protease, a lipase, a cellulase, a xylanase, an oxidoreductase, a pectinase, a cutinase, or a glycolytic enzyme.<br>
Examples of suitable Promoters for heterologous protein expression in a filamentous fungal host cell are Promoters obtained from the genes for Aspergillus oryzae TAKA amylase. Rhizomucor miehei aspartic Proteinase, Aspergillus niger neutral alpha-amylase, Aspergillus iiiger acid stable alpha-amylase, Aspergillus niger or Aspergillus awamori glucoamylase (glaA), Rhizomucor miehei lipase, Aspergillus oryzae alkaline protease, Aspergillus oryzae those Phosphate isomerase, Aspergillus nidulans acetamidase, Fusarium venenatum amy-loglucosidase (WO 00/56900). Fusarium venenatum Daria (WO 00/56900). Fusarium veneÂ¬natum Quinn (WO 00/56900), Fusarium oxysporum trypsin-like protease (WO 96/00787),<br><br>
Trichoderma reesel beta-glucosidase, Trichoderma reesei cellobiohydrolase I, Trichoderma reesai endoglucanase I, Trichoderma reesei endoglucanase II, Trichoderma reesei endoglu-canase III, Trichoderma reesei endoglucanase IV, Trichoderma reesei endoglucanase V, Trichoderma reesei xylanase I, Trichoderma reesei xylanase II, Trichoderma reesei beta-xylosidase, as well as the NA2-tpi promoter (a hybrid of the Promoters from the genes for AsÂ¬pergillus niger neutral alpha-amylase and Aspergillus oiyzae triose phosphate isomerase); and mutant, truncated, and hybrid Promoters thereof.<br>
The cloning vehicie may also inctude a selectable marker, such as a gene product which complements a defect in the host cell, or one which confers antibiotic resistance. Ex-amples of antibiotics usefui as Aspergillus selection markers include hygromycin, phleomycin and basta. Other examples of Aspergillus sefection markers include amdS, which encodes an enzyme involved in acetamide utilisation; pyrG, which encodes an enzyme involved in uridine biosynthesis; argB, which encodes an enzyme involved in arginine biosynthesis; niaD, which encodes an enzyme involved in Ihe nitrate assimilation pathway; and sC, which encodes an enzyme involved in the sulfate assimilation pathway. Preferred for use in an Aspergillus host cell are the amdS and pyrG markers of Aspergillus nidulans or Aspergillus oryzae. Further-more, selection may be accomplished by co-transformation, wherein the transformation is car-ried out with a mixture of two vectors and the selection is made for one vector only.<br>
The procedures used to ligate the DNA construct of the invention, the promoter. ter-minator and other elements, respectively, and to insert them into suitable cloning vehicies con-taining the Information necessary for replication, are well known to persons skitied in the art {vide e.g., Sambrook et al., 1989; interalia),<br>
The culture broth or medium used may be any conventional medium suitable for cul-turing the host cell of the invention, and formulated according to the principles of the phor art. The medium preferably contains carbon and nitrogen sources as well as other inorganic salts. Suitable media, e.g. minimal or complex media, are available from commercial suppliers, or may be prepared according to published recipes, as in The Cataloque of Strains. pubtished by The American Type Culture Collection.  Rockville MD, USA.  1970.<br>
The appropriate pH for fermentation will often be dependent on such factors as the nature of the host cell to be used, the composition of the growth medium, the stability of the Polypeptide of interest, and the like. Consequently, although the host cell of the invention may be cultured using any fermentation process performed at any pH, it is advantageous that the pH of the fermentation process is such that acidic and/or neutral protease activities of the host cell are essentially eliminated or at least significantly reduced. Thus, removal of aspartic proÂ¬tease activity as described in WO 90/00192 may also be accomplished by raising the fermenÂ¬tation pH, and does not present any additional advantageous effect on the yield of a desired protein from host cells cultivated in the alkaline pH rÃ¤nge.<br><br>
If the pH of the fermentation process is within the rÃ¤nge from 5 to 11, such as from 6 to 10.5, 7 to 10, or 8 to 9.5, the activity of acidic proteases, such as aspartic and sehne prote-ases, and neutral proteases in the pH ranges above 7, will be reduced or biocked, Examples of enzymes produced under alkaline fermentation conditions include endoglucanases, phyta-ses and protein disulfide isomerases.<br>
However, the alkaline pH rÃ¤nge will support alkaline protease activity in an unmodified host cell, which, in turn, may potentlally result in degradation of the Polypeptide product of in-terest. Consequently, in such cases the inactivation of the gene encoding alkaline protease is especially advantageous.<br>
Inactivation of the alkaline protease gene of the invention is also especially advantaÂ¬geous for certain host cells, as the levels of acidic, neutral and alkaline protease activities vary from species to species. For example, the level of alkaline protease activity in the Aspergillus oryzae is higher than in Aspergillus niger<br>
After cultivation, the desired protein is recovered by conventional methods of protein isolation and purification from a culture broth. Well established purification procedures include separating the cells from the medium by centrifugation orfiltration, precipitating proteinaceous components of the medium by means of a salt such as ammonium sulphate, and ChroÂ¬matographie methods such as ion exchange chromatography, gel filtration chromatography, affinity chromatography, and the like.<br>
Products<br>
The desired end product, i.e. the heterologous protein, expressed by the host cell of the invention, may be any eubacterial or eukaryotic protein or peptide.<br>
As defined herein, a "heterologous protein" is a protein or Polypeptide gene product which is not native to the host cell, or is a native protein in which modifications have been made to alter the native sequence, or is a native protein whose expression is quantitatively alÂ¬tered as a result of a manipulation of a native regulatory sequence required for the expression of the native protein, such as a promoter, a ribosome binding Site, etc., or other manipulation of the host cell by recombinant DNA techniques.<br>
Sequence identity and alignment<br>
In the present context, the homology between two amino acid sequences or between two nuÂ¬dele acid sequences is described by the parameter "identity".<br>
For purposes of the present invention, alignments of sequences and caiculation of homology scores may be done using a fÃ¼ll Smith-Waterman alignment, usefui for both protein and DNA alignments. The default scoring matrices BLOSUM50 and the identity matrix are used for<br><br>
protein and DNA alignments respectively. The penalty for the first residue in a gap is -12 for proteins and -16 for DNA, while the penalty for additional residues in a gap is -2 for proteins and -4 for DNA. Alignment may be made with the FASTA package verston v20u6 (W R. Pearson and D. J. Lipman (1988), "Improved Tools for Biological Sequence Analysis". PNAS 85:2444-2448, and W. R.^Pearson (1990) "Rapid and Sensitive Sequence Comparison with FASTP and FASTA", Methods in Enzymology, 183:63-98),<br>
Multiple alignments of protein sequences may be made using "ClustalW" (Thompson, J.D., Higgins, D.G. and Gibson, T.J. (1994) CLUSTAL W: improving the sensitivity of progressive multiple sequence alignment through sequence weighting, positions-specific gap penalties and weight matrtx choice. Nucleic Acids Research, 22:4673-4680). Multiple alignments of DNA sequences may be done using the protein alignment as a template, replacing the amino acids with the corresponding codon from the DNA sequence.<br>
Alternatively different Software can be used for aligning amino acid sequences and DNA seÂ¬quences. The alignment of two amino acid sequences is e.g. determined by using the Needle program from the EMBOSS package (http://emboss.org) version 2.8.0. The Needle program implements the global alignment algorithm described in Needleman, S. B. and Wunsch, C. D. (1970) J. Mol. Biol. 48, 443-453. The Substitution matrix used is BLOSUM62, gap opening penalty is 10, and gap extension penalty is 0.5.<br>
The degree of identity between an amino acid sequence of the present invention ("in-vention sequence"); e.g. SEQ ID NO: 32 and a different amino acid sequence ("foreign seÂ¬quence") is caiculated as the number of exact matches in an alignment of the two sequences, divided by the length of the "invention sequence" er the length of the "foreign sequence", whichever is the shortest. The result is expressed in percent identity.<br>
An exact match occurs when the "invention sequence" and the "foreign sequence" have identical amino acid residues in the same positions of the overlap (in the alignment example below this is represented by "|"). The length of a sequence is the number of amino acid resiÂ¬dues in the sequence (e.g. the length of SEQ ID NO: 32 is 495).<br>
In the purely hypothetical alignment example below, the overlap is the amino acid seÂ¬quence "HTWGER-NL" of Sequence 1; or the amino acid sequence "HGWGEDANL" of SeÂ¬quence 2. In the example a gap is indicated by a "-".<br>
Hypothetical alignment example:<br><br><br>
For purposes of the present invention, the degree of identity between two nucleotide sequences is preferably determined by the Wilbur-Lipman method (Wilbur and Lipman, 1983, Proceedings of the National Academy of Science USA 80: 726-730) using the LASERGENEâ¢ MEGALIGNTM Software (DNASTAR, Inc., Madison, Wl) with an identity table and the following multiple alignment parameters: Gap penalty of 10 and gap length penalty of 10. Pairwise alignment parameters are Ktuple=3, gap penalty=3, and windows=20.<br>
In a particular embodiment. the percentage of identity of an amino acid sequence of a Polypeptide with, or to, amino acids 1 to 495 of SEQ ID NO: 32 is determined by i) aligning the two amino acid sequences using the Needle program, with the BLOSUM62 Substitution matrix. a gap opening penalty of 10, and a gap extension penalty of 0.5; ii) counting the number of ex-act matches in the alignment; iii) dividing the number of exact matches by the length of the shortest of the two amino acid sequences, and iv) Converting the result of the division of iii) into percentage. The percentage of identity to. or with, other sequences of the invention is calcu-lated in an analogous way.<br>
Host cells<br>
The host cell of the invention is a filamentous fungus. It is advantageous to use a host cell of the invention in recombinant production of a Polypeptide of interest. The cell may be transformed with the DNA construct encoding the Polypeptide of interest, conveniently by inte-grating the DNA construct in one or more copies into the host chromosome. This Integration is generally considered to be an advantage as the DNA sequence is more likely to be stably maintained in the cell. Integration of the DNA construct into the host chromosome may be per-formed according to conventional methods, e.g., by homologous or heterologous recombina-tion. Alternatively, the cell may be transformed with an expression vector as described in the examples below in connection with the different types of host cells.<br>
Filamentous fungal Host Cells<br>
The host cell of the invention is a filamentous fungus represented by one of the followÂ¬ing groups of Ascomycota, include, e.g., Neurospora, Eupenicillium {=Penicillium), Emericella {-Aspergillus), Eurotium {=Aspergillus).<br><br>
In a preferred embodiment, the filamentous fungus belongs to one of the filamentous forms of the subdivision Eumycota and Oomycota as defined by Hawksworth et aL, In, Ains-worth and Ãisby's Dictionary of The Fungi, 8th edition, 1995, GAB International, University Press, Cambridge, UK. The filamentous fungi are characterized by a vegetative mycellum composed of chitin, cellulose, glucan, chitosan, mannan, and other complex Polysaccharides. Vegetative growth is by hyphal elongation and carbon catabolism is obligatety aerobic.<br>
In a more particular embodiment, the filamentous fungal host cell is a cell of a species of, but not limited to, Acremonium, Aspergillus, Fusarium, Humicola, Mucor, Myceliophthora, Neurospora, Penicillium, Thielavia, Tolypocladium, and Trichoderma or a teleomorph or synoÂ¬nym thereof. In an even more particular embodiment, the filamentous fungal host cell is an Aspergillus cell. In another even more particular embodiment, the filamentous fungal host cell is an Acremonium cell. In another even more particular embodiment, the filamentous fungal host cell is a Fusarium cell. In another even more particular embodiment, the filamentous funÂ¬gal host cell is a IHumicola cell. In another even more particular embodiment, the filamentous fungal host cell is a MucorceW, In another even more particular embodiment, the filamentous fungal host cell is a Myceliophthora cell. In another even more particular embodiment, the filÂ¬amentous fungal host cell is a Neurospora cell. In another even more particular embodiment, the filamentous fungal host cell is a Peniclllium cell. In another even more particular embodiÂ¬ment, the filamentous fungal host cell is a Thielavia cell. In another even more particular emÂ¬bodiment, the filamentous fungal host cell is a Tolypocladium cell. In another even more parÂ¬ticular embodiment, the filamentous fungal host cell is a Trichoderma cell. In a most particular embodiment, the filamentous fungal host cell is an Aspergillus awamori, Aspergillus foetidus, Aspergillus japonicus, Aspergillus aculeatus, Aspergillus niger, Aspergillus nidulans or AsperÂ¬gillus oryzae cell. In another particular embodiment, the filamentous fungal host cell is a FusaÂ¬rium cell of the section Discolor (also known as the section Fusarium). For example, the filaÂ¬mentous fungal host cell may be a Fusarium bactridioides, Fusarium cerealis, Fusarium crookwellense, Fusarium culmorum, Fusarium graminearum (in the perfect State named Gib-berella zeae, previously Sphaeria, synonym with Gibberella roseum and Gibberella roseum f.sp. ceralis), Fusarium graminum, Fusarium heterosporum, Fusarium negundi, Fusarium re-ticulatum, Fusarium roseum, Fusarium sambucinum, Fusarium sarcochroum, Fusarium sul-phureum, Fusarium trichothecioides or Fusarium venenatum cell. In another preffered emÂ¬bodiment, the filamentous fungal host cell is a Fusarium strain of the section Elegans, e.g., Fusarium oxysporum. In another most particular embodiment, the filamentous fungal host cell IS a Humicola insolens or Humicola lanuginosa cell. In another most particular embodiment, the filamentous fungal host cell is a Mucor miehei cell. In another most particular embodiment, the filamentous fungal host cell is a Myceliophthora thermophilum cell. In another most particuÂ¬lar embodiment, the filamentous fungal host cell is a Neurospora crassa cell.  In another most<br><br>
particular embodiment, the filamentous fungal host cell is a Penicillium purpurogenum, Penicil-lium chrysogenum or Penicillium funiculosum (WO 00/68401) cell. In another most particular embodiment, the filamentous fungal host cell is a Thielavia terrestris cell. In another most parÂ¬ticular embodiment, the Trichoderma cell is a Trichoderma harzianum, Trichoderma koningii, Trichoderma longibrachiatum, Trichoderma reesei on Trichoderma viride cell.<br>
In all embodiments of the invention the filamentous fungal host cell is a protease defi-cient or protease minus strain in which the expression of at least three endogenous proteases has been reduced or completely abolished.<br>
In a particular embodiment the parent strain is the protease deficient Aspergillus oryzae strain BECh2 described in WO 00/39322, example 1. which is further referring to JaL228 described in WO 98/12300, example 1. This strain, which is alp and npl' (deficient in the atkaline protease Alp and the neutral metalloprotease Npl) can be further modified to a par-ticularly usefui strain according to the invention, in which strain additional mutations have been introduced, as described in the following examples, to produce a filamentous fungal strain acÂ¬cording to the invention, wherein additionally the serine protease of the subtilisin type desigÂ¬nated PepC and optionally the calcium dependent, neutral, serine protease, KexB, are defiÂ¬cient.<br>
Transformation of Filamentous Fungal Host Cells<br>
Filamentous fungal host cells may be transformed by a process involving protoptast formation, transformation of the protoplasts, and regeneration of the cell wall in a manner known per se. Suitable procedures for transformation of Aspergillus host cells are described in EP 238 023, EP 184,438, and Yelton et al, 1984, Proceedings of the National Academy of Sciences USA 81:1470-1474. A suitable method of transforming Fusarium species is deÂ¬scribed by Malardier et aL, 1989, Gene 78:147-156 or in co-pending US Serial No. 08/269,449,<br>
The filamentous fungal cells according to the invention are particularly usefui for the expression of heterologous Polypeptides.<br>
In one aspect of the invention the filamentous fungal cells according to the invention are used for the expression of heterologous Polypeptides. Thus one further aspect of the inÂ¬vention relates to a method for producing a heterologous Polypeptide in a filamentous fungal host cell of the invention, comprising the steps:<br>
(a)	introducing into the host cell a nucleic acid sequence encoding the hete-<br>
rologous Polypeptide;<br><br>
(b)	cultivating the cell from (a) in a suitable growth medium under conditions conducive for expression of the heterologous Polypeptide; and<br>
(c)	isolating the protein product.<br>
The heterologous Polypeptide is in one embodiment a Polypeptide of mammalian origin. Par-ticularly the Polypeptide is an antibody or an antibody fragment. In a further particular embodiÂ¬ment the antibody or antibody fragment is selected from the group consisting of IgGI, lgG2, lgG3, lgG4, IgM, IgA. IgD. IgE. F(ab')2 and Fab.<br>
MATERIALS AND METHODS Materials<br>
Strains<br>
Aspergillus oryzae IF04177: available from Institute for fermentation, Osaka;  17-25 Juso<br>
Hammachi 2-Chome Yodogawa-Ku. Osaka, Japan. This strain is also equivalent to strain<br>
NBRC4177. The strain is therefore also available from NITE Biological Resource Center under<br>
the accession number NBRC4177,<br>
JaLSIO is described in example 3.<br>
JaL893 is described in example 7.<br>
JaL895 is described in example 7.<br>
BECh2 is described in WO 00/39322, example 1, which is further referring JaL228 described in<br>
WO 98/12300, example 1.<br>
NZ-2 is described in example 7.<br>
HowB101 is described in WO 97/3556, example 1 is pyrG mutant of IF04177.<br>
JaL352 is described in example 2.<br>
JaL355 is described in example 2.<br>
JaL741 is described in example 7.<br>
MT2874 is described in example 2.<br>
ToC1418 is described in example 2.<br>
JaL799 is described in example 10.<br>
JaL827 is described in example 11.<br>
Genes<br>
pyrG: This gene codes for orotidine-S'-phosphate decarboxylase, an enzyme involved in the<br>
biosynthesis of uridine.<br><br>
Plasmids<br>
pBLSK-2 : The construction is described in Short et al, 1988, Nucleic acids Res. 16:7583-<br>
7600.<br>
pCRÂ®4Blunt-TOPO is frotn Invitrogen A/S, Denmark.<br>
pDV8 is described in patent WO 0168864, example 8<br>
pNZ-3 is described in example 5.<br>
pNZ"4 is described in example 6.<br>
plC7 is described In Marsh et al, 1984, Gene 32:481-485.<br>
pJaL173 is described in patent WO 98/12300. example 1.<br>
pJaL235 is described in WO 97/22705, example 1<br>
pJaL270 is described in example 1.<br>
pJaL335 is described in patent WO 98/12300, example 1.<br>
pJaL504 is described in example 1.<br>
pJaL554 is described in example 9.<br>
pJaL574 is described in example 9.<br>
pJaL676 is described in WO 03/008575, example 5.<br>
pJaL720 is described in example 4.<br>
pJaL721 is described in WO 03/008575, example 17.<br>
pJaL723 is described in example 4.<br>
pJaL728 is described in example 4.<br>
pJaL784 is described in example 4.<br>
pJaL790 is described in example 4.<br>
pJaL800 is described in example 9.<br>
pJaL818 is described in example 9.<br>
pJaL819 is described in example 9.<br>
pJaL835 is described in example 9,<br>
pJaL836 is described in example 9.<br>
pMT2830 is described in example 1.<br>
pl\/IT2833 is described in example 1.<br>
pMT2835 is described in example 1.<br>
pS02 is described in patent WO 9735956, example 1.<br>
pUC19: The construction is deschbed in Vieira et al, 1982, Gene 19:259-268.<br>
Primers. DNA sequences. and amino acid sequences Pnmer419(SEQ ID NO: 1) Primer 424 (SEQ ID NO: 2) Primer 423 SEQ ID NO:3)<br><br>
Primer 420 SEQ ID NO:4)<br>
Primer 104025 (SEQ ID NO: 5)<br>
Primer 104027 (SEQ ID NO; 6)<br>
Primer 104026 (SEQ ID NO: 7)<br>
Primer 104028 (SEQ ID NO: 8)<br>
Primer 108089 (SEQ ID NO: 9)<br>
Primer 108091 (SEQ ID NO: 10)<br>
Primer 114839 (SEQ ID NO: 11)<br>
Primer 135944 (SEQ ID NO: 12)<br>
Primer B6577F12 (SEQ ID NO: 13)<br>
Primer B6575F12 (SEQ ID NO: 14)<br>
Heavy chain (SEQ ID NO: 15)<br>
Primer H-N (SEQ ID NO: 16)<br>
Primer H-C (SEQ ID NO: 17)<br>
Light Chain (SEQ ID NO: 18)<br>
Primer L-N (SEQ ID NO: 19)<br>
Primer L-C (SEQ ID NO: 20)<br>
Primer 172450 (SEQ ID NO: 21)<br>
Primer 172449 (SEQ ID NO: 22)<br>
Primer T5483H12 (SEQ ID NO 23)<br>
Primer T5425G10 (SEQ ID NO 24)<br>
Sal I fragment comprising kexB from A. oryzae IF04177 (SEQ ID NO: 25)<br>
alp cDNA (SEQ ID NO: 26)<br>
npl cDNA (SEQ ID NO: 27)<br>
pepC cDNA (SEQ ID NO: 28)<br>
kexB cDNA (SEQ ID NO: 29)<br>
Alp amino acid sequence (SEQ ID NO: 30)<br>
Npl amino acid sequence (SEQ ID NO: 31)<br>
PepC amino acid sequence (SEQ ID NO: 32)<br>
KexB amino acid sequence (SEQ ID NO; 33)<br>
Methods<br>
General methods of PCR, cloning, ligation nucleotides etc. are well-known to a person skilled in the art and may for example be found in "Molecular cloning; A laboratory manual", Sambrook et al. (1989), Cold Spring Harbor lab., Cold Spring Harbor, NY; Ausubel, F. M. et al. (eds.); "Current protocols in Molecular Biology", John Wiley and Sons, (1995), Harwood, C. R., and Cutting, S. M. (eds.); "DNA Cloning: A Practical Approach, Volumes I and 11", D.N. Glover<br><br>
ed. (1985); "Oligonucleotide Synthesis", M.J. Galt ed. (1984); "Nucleic Acid Hybndization", B.D. Harnes &amp; S.J. Higgins eds (1985); "A Practica! Guide To Molecular Cloning", B. Perbai, (1984).<br>
DNA hvbridizatlon<br>
In Short all DNA hybridisation was carried out for 16 hours at 65Â°C in a Standard hybridisation buffer of 10 x Denhart's Solution, 5x SSC, 0.02 M EDTA, 1% SDS, 0,15 mg/mt polyA RNA and 0.05 mg/ml yeast tRNA. After hybridisation the filters were washed in 2x SSC, 0,1% SDS at 65'C twice and exposed to X-ray films.<br>
PCR amplification<br>
All PCR amplifications was performed in a volunne of 100 microL containing 2.5 units Taq PoÂ¬lymerase, 100 ng of pS02, 250 nM of each dNTP, and 10 pmol of each of the two primers de-scribed above in a reaction buffer of 50 mM KCl, 10 mM Tris-HCI pH 8.0, 1.5 mM MgCl2. Amplification was carried out in a Perkin-Elmer Cetus DNA Termal 480, and consisted of one cycle of 3 nninutes at 94Â°C, followed by 25 cycles of 1 minute at 94Â°C, 30 seconds at 55'C, and 1 minute at 72Â°C.<br>
Example 1<br>
Construction of the Aspergillus oryzae pepC deletion Plasmid pMT2835<br>
The Single restriction endonuclease site BamHi and Bglll in pDV8 was removed by two succeeding rounds of cutting with each of the restriction endonucleases and the free over-hang-ends was filled out by treatment with Klenow Polymerase and the four deoxyribonucleo-tides and ligated resulting in plasmid pJaL504.<br>
Plasmid pJaL235 was digested with Sali and the 5912 bp fragment was re-ligated, reÂ¬sulting in Plasmid pJaL270.<br>
Based on the sequence previously determined for the pepC gene as cloned in pJaL270, the sequences flanking the pepC gene of A.oryzae were cloned from A.oryzae BECh2 by PCR. For amplification of the 5' flank, primers 419 and 424 were used (SEQ ID NO: 1 and 2) and for amplification of the 3' flank the primers used were 423 and 420 (SEQ ID NO: 3 and 4). The amplified flanks (0.8 kb and 1.1 kb, respectively) do not overlap with any of the pepC coding sequence. PrImers 423 and 424 were designed to allow fusion of the two flanks by SOE and at the same time to introduce Nhel and Xhol sites in the linker fusing the two flanks. The SOE fusion product was cloned into pCRÂ®4-TOPO blunt to give pMT2830. The Nhel and Xhol sites were used to insert the repeat flanked pyrG selection marker from pJaL335 which had previously been cloned in the Hindlll site of pBLSK 2 such that it could be moved as a Spe1-Xho1 fragment. The plasmid containing the pyrG insert was calied pMT2833. Finally, the deletion cassette of pMT2833 containing the two pepC flanks on either<br><br>
side of the pyrG selection marker was transferred as a Notl-Spel fragment into the Notl and Xbalsites of the tk counter selectable plasmid pJaL504 to give the deletion plasmid pMT2835. Note that pMT2835 contains a unique Notl site immediately downstream of the pepC 3' flank, which can be used to linearize the plasmid prior to transformation into A.oryzae.<br>
Example 2<br>
Construction of pepC deleted A.oryzae strain, MT2874.<br>
For removing the defect pyrG gene resident in the alkaline protease gene in the A. oryzae strain BECh2 the following was done;<br>
A.	Isolation of a pyrG minus A. oryzae strain, ToC1418<br>
The A. oryzae strain BECh2 (alp minus and npl minus) was screened for resistance to 5-flouro-orotic acid (FOA) to identify spontaneous pyrG mutants on minimal plates (Cove DJ. 1966. Biochem. Biophys. Acta. 113:51-56) supplemented with 1.0 M sucrose as carbon source, 10 mM sodiumnitrate as nitrogen source, and 0.5 mg/ml FOA. One strain, ToC1418, was identified as being pyrG minus. ToC1418 is uridine dependent, therefore it can be trans-formed with the wild type pyrG gene and transformants selected by the ability to grow in the absence of uridine.<br>
B.	Construction of a pyrG plus A. oryzae strain, JaL352.<br>
The mutation in the defect pyrG gene resident in the alkaline protease gene was de-termined by sequencing. Chromosomal DNA from A. oryzae strain BECh2 was prepared and a 933 bp fragment was amplified containing the coding region of the defect pyrG gene by PCR with primers 104025 (SEQ ID NO: 5) and 104026 (SEQ ID NO: 7). The 933 bp fragment was purified and sequenced with the following primers: 104025, 104026, 104027 (SEQ ID NO: 6), 104028 (SEQ ID NO: 8), 108089 (SEQ ID NO: 9), and 108091 (SEQ ID NO: 10). Sequencing Shows that an extra base, a G, was inserted at position 514 in the pyrG-coding region (count-ing from the A in the Start codon of the pyrG gene), thereby creating a frame-shift mutation.<br>
In Order to recreate a wild type pyrG gene from the defect pyrG gene resident in the alkaline protease gene, alp, the A. oryzae pyrG minus strain ToC1418 was transformed with 150 pmol of oligo-nucleotide 114839 (SEQ ID NO: 11) phosphorylerated in the 5' end, using Standard procedures. The oligo-nucleotide restores the pyrG reading frame, but at the same time a silence mutation is introduce thereby creating a StuI restriction endonuclease site. Transformants were then selected by their ability to grow in the absence of uridine on minimal plates (Cove D.J. 1966. Biochem. Biophys. Acta. 113:51-56) supplemented with 1.0 M sucrose as carbon source, and 10 mM sodiumnitrate as nitrogen source. After re-isolation chromoso-<br><br>
mal DNA was prepared from 8 transformants. To confirm the changes a 785 bp fragment was amplified by PCR with the primers 135944 (SEQ ID NO: 12) and 108089 (SEQ ID NO: 9), which is covering the reglon of interest. The 785 bp fragment was purified and sequenced with the primers 108089 (SEQ ID NO: 9) and 135944 (SEQ ID NO; 12). One strain having the ex-pected changes was named JaL352.<br>
C.	Isolation of a pyrG minus A. oryzae strain, JaL355<br>
For removing the pyrG gene resident in the alkaline protease gene JaL352 was trans-formed by Standard procedure with the 5.6 kb BamHI fragment of pJaL173 harboring the 5' and 3' flanking sequence of the A, oryzae alkaline protease gene. Protoplasts were regener-ated on non-selective plates and spores were collected. About 109 spores were screened for resistance to FOA to identify pyrG mutants. After re-isolation chromosomal DNA was prepared from 14 FOA resistance transformants. The chromosomal DNA was digested with Bai I and analysed by Southern blotting, using the 1 kb 32P-labelled DNA Bai I fragment from pJaL173 containing part of the 5' and 3' flanks of the A. oryzae alkaline protease gene as the probe. Strains of interest were identified by the disappearance of a 4.8 kb Bai I band and the appear-ance of a 1 kb Bai I band, Probing the same filter with the 3.5 kb 32P-labelled DNA Hind III fragment from pJaL335 containing the A. oryzae pyrG gene shows that the 4.8 kb Bai I band has disappeared in the strains of interest. One strain resulting from these transformants was named JaL355.<br>
D.	Construction of pepC deleted A. oryzae strain, MT2874<br>
20 microgram of pMT2835 was cut with Notl and subsequently the enzyme was heat Inactivated as recommended by the manufacturer (New England Biolabs). The plasmid was then ethanol precipitated and re-dissolved in Tris buffer (10mM pH 8.0) at a concentration suit-able for transformation into Aspergillus oryzae.<br>
The linearized plasmid DNA was transformed into Aspergillus oryzae JaL355 with se-lection for pyrG and counter selection of the tk gene on FDU plates as previously described in WO 0168864. 24 transformant colonies were twice re-isolated and finally grown up in liquid medium (YPD). Chromosomal DNA was prepared as previously described in WO 0168864, and used for Southern analysis of the pepC locus with the aim to identify transformants in which a clean double cross-over between the chromosomal pepC and the deletion cassette had occurred. The chromosomal DNA was digested with EcoRI and Pvul. The Southern blot was first probed with the 5' flank excised as a 0.8 kb EcoRI-Xhol fragment from pMT2B35 (Probel). For an intact wt pepC locus, a 3.1 kb EcoRI-Pvul fragment is expected to hybridize to this probe, while for the pepC deleted derivative originating from the desired double cross-<br><br>
over a 5.5 kb fragment will hybridize, Only one out of the 24 transformants, JaL355/pMT2835#5, showed both the disappearance of the 3.1 kb hybridizing fragment and the appearance of the 5.5 kb hybridizing fragment in a Southern blot analysis. The Southern was stripped of the first 5' flank probe and re-probed with a 3' flank probe excised as a 1.1 kb EcoRI fragment from pMT2835 (Probe 2). For the wt pepC locus a 3.1 kb is again expected to hybridize to this fragment while for the pepC deletion strain originating from the desired cross over it is expected that a 1.1 kb fragment will hybridize. Again only one of the 24 transforÂ¬mants, JaL355/pMT2835#5, showed both the disappearance of the 3.1 kb hybridizing fragment and the appearance of the 1.1 kb hybridizing fragment in a Southern blot anatysis. The filter was again stripped and re-probed with a third probe. The third probe was excised as a 0.7 kb Asp718-Pstl fragment from pJaL270 (Probe3). This fragment is part of the pepC coding se-quence itself. This third probe is again expected to hybridize to a 3.1 kb EcoRI-Pvul fragment of the wt pepC locus while no hybridization signal is expected from the desired pepC deleted derivative. A few transformant strains including JaL355/pMT2835#5 faiied to hybridize to the third probe.<br>
In conclusion, of the 24 transformant only JaL355/pMT2835#5 contains the desired clean double cross over leading to a deletion of the pepC coding sequence. JaL355/pMT2835#5 was preserved as A.oryzae strain MT2874. The pepC deleted strain MT2874 does appear to have slightly changed colony morphology and sporulation proficiency relative to the parent strain. In particular sporulation at elevated temperature e.g. 37 degrees C seems poor.<br>
Example 3<br>
Construction of a pepC deleted A.oryzae strain, JaL810 (A1560-ApepC)<br>
Clean pepC disruptions were made in A.oryzae strain HowBIOI. The procedures for making and verifying the deletion is identical to those described in example 2, sections D.<br>
Example 4<br>
Construction of Aspergillus expression Plasmid pJaL790<br>
The Aspergillus expression plasmid pJaL790, suitable for expressing a gene of inter-est under the control of the A. niger neutral amylase promoter (NA2) was constructed in the following way:<br>
The Single restriction endonuclease site Hindill in the vector pUC19 was removed by cutting with Hindill and the free overhang-ends was filled out by treatment with Klenow PolyÂ¬merase and the four deoxyribonucleotides and ligated, resulting in plasmid pJaL720. The 1140 bp EcoRI-BamHI fragment from pJaL721, containing the double A. niger NA2 promoter, was<br><br>
5<br><br>
cloned into the corresponding sites in pJaL720, resulting in pJaL723. A 537 bp fragment was amplified by PCR with pJaL676 as template and the primers B6577F12 (SEQ ID NO; 13) and B6575F12 (SEQ ID NO; 14). This was digested with EcoRI, the free overhang-ends were filled in by treatment with Klenow Polymerase and the four deoxyhbonucleotides and the resulting 524 bp fragment was cloned into the Hindlll site, which was blunt ended in pJaL723, giving Plasmid pJaL728. The Single restriction endonuclease site Hindlll in the vector pJaL728 was removed by cutting with Hindlll and the free overhang-ends was filled in by treatment with Klenow Polymerase and the four deoxyhbonucleotides and ligated, resulting in plasmid pJaL784. A 1671 bp EcoRI-BamHI fragment from pJaL784 was ligated to the 5735 bp EcoRI-BamHI fragment from pJaL721, resulting in pJaL790.<br>
Example 5<br>
Construction of a native IqGI heavv-chain Aspergillus expression Plasmid A human IgGI heavy chain encoding sequence was amplified by PCR using SEQ ID NO; 15 as template and the forward primer H-N (SEQ ID NO; 16) and the reverse primer H-C (SEQ ID NO;17). Primers H-N and H-C introduce a BamHI and Xhol restriction site upstream of the translational start codon and after the translation termination Signal, respectively. for cloning purposes. The PCR product on 1431 bp was purified and cut with the resthction endonucle-ases BamHI and Xhol. The resulting 1419 bp fragment was cloned into the corresponding site in pJaL790 to create pNZ-3. DNA from ctone pNZ-3 was sequenced to check that it was the right sequence.<br>
Example 6<br>
Construction of a native IgG kappa light-chain Aspergillus expression Plasmid A human IgG kappa light-chain encoding sequence was amplified by PCR using SEQ ID NO; 18 as template and with the forward phmer L-N (SEQ ID NO; 19) and the reverse primer L-C (SEQ ID NO; 20). Primers L-N and L-C introduce a BamHI and Xhol restriction site upstream the translational start codon and after the translation termination Signal, respectively, for clonÂ¬ing purposes. The PCR product of 732 bp was purified and cut with the restriction endonucle-ases BamHI and Xhol. The resulting 720 bp fragment was cloned into the corresponding site in pJaL790 to create pNZ-4. DNA from clone pNZ-4 was sequenced to check that it was the right sequence.<br>
Example 7<br>
Expression of IgGI antibody in Aspergillus oryzae<br>
The strains MT2874. BECh2, IF04177, and JaL810 was transformed with a one to one ratio of the expression plasmid pNZ-3, and pNZ-4 as deschbed by Christensen et al.; Bio-<br><br>
technology 1988, 6:1419-1422. In Short, A.oryzae mycelia were grown in a rieh nutrient brolh. ' The mycelia were separated from the broth by filtration. The enzyme preparation Novozyme (Novo Nordisk) was added to the mycelia in osmotically stabilizing buffer such as 1.2 M MgSO4 buffered to pH 5.0 with sodium phosphate (alternatively other enzyme mixes such as e,g, Glu-canexÂ® 200G can be used). The Suspension was incubated for 60 minutes at 37degrees C with agitation. The protoplast was filtered through mira-cloth to remove mycelial debris. The protoplast was harvested and washed twice with STC (12 M sorbitol, 10 mM CaCl2, 10 mM Tris-HCI pH 7.5). The protoplasts were finally re-suspended in 200-1000 microt STC.<br>
For transformation, 5 microg DNA was added to 100 microl protoplast Suspension and then 200 microl PEG Solution (60% PEG 4000, 10 mM CaCI2, 10 mM Tris-HCI pH 7.5) was added and the mixture is incubated for 20 minutes at room temperature, The protoplast were harvested and washed twice with 1.2 M sorbitol. The protoplast was finally re-suspended 200 microl 1.2 M sorbitol. Transformants containing the amdS gene, present on pNZ-3 or pNZ-4, were selected on minimal plates (Cove D.J. 1966. Biochem. Biophys. Acta. 113:51-56) conÂ¬taining 1.0 M sucrose as carbon source, 10 mM acetamide as nitrogen source, 15 mM CsCI to inhibit background growth, and 250 mM 5-ALA. After 4-5 days of growth at 37 degrees C, stÃ¤Â¬hle transformants appeared as vigorously growing and sporulating colonies. Transformants were purified twice through conidiospores.<br>
Shake flask containing 10 ml YPM medium (2 g/l yeast extract. 2 g/l peptone, and 2% maitose) was inoculated with spores from the transformants from the 3 strains and incubated at 30 degrees C, 200 rpm for 4 days.<br>
The supernatants were screened for intact IgGI by ELISA (Harlow, E. and Lane, D.. Antibodies: A Laboratory Manual. Cold Spring Harbor Laboratory Press, (New York, 1988), p. 579). Intact IgG was determined using an ELISA which uses goat anti-human IgG (Fe specific) as the capture antibody and goat anti-human kappa chain conjugated with alkaline PhosÂ¬phatase as the detection antibody. As Standard was used a human myeloma IgGI. kappa puriÂ¬fied from human plasma. The ELISA procedure was a Standard protocol.<br>
One transformant (containing both expression Plasmids pN2-3 and pNZ-4) from each parent strain MT2874, BECh2, IF04177 and JaL810, producing intact IgGI, was selected and preserved as A.oryzae strain JaL741, NZ-2, JaL893 and JaL895, respectively.<br>
Example 8<br>
Production and characterization of IgGI from Aspergillus oryzae<br>
The transformed A.oryzae strains JaL741, NZ-2, JaL893, and JaL895 were grown in a fermentor using Standard Substrate and after 5 days of incubation the fermentation broth was harvested. The whole broth was centrifuged and the supernatant passed through a 0,22 Âµm PES filter membrane (Corning).<br><br>
MEP HyperCel column<br>
The filter flow through was loaded on a MEP HyperCel column (BioSepra) on an ÃKTA explorer System (Amersham Biosciences). The column was washed with 50mM HEPES pH 7.5 followed by 25mM Caprylate in 50mM Tris-HCI pH 8. lgG1 was eluted with 50mM MES pH 5.5.<br>
Protein A column<br>
Eluted IgGI from the MEP HyperCel column was adjusted to pH 7 with 1M Tris base and loaded on a nProtein A Sepharose 4 Fast Flow column (Amersham Biosciences) on an ÃKTA explorer system (Amersham Biosciences). The column was washed with 20mM Na-Phosphate pH 7. lgG1 was eluted with lOOmM Citric acid pH 4.<br>
IgG characterisation<br>
Purified lgG1 samples were analysed by SDS-PAGE on 4-20% Tris-glycine gels un-der reducing and non-reducing conditions. Protein bands were identified by N-terminal seÂ¬quencing. For N-terminal sequencing gels were blotted to a PVDF membrane that was stained with coomassie blue. Protein bands were cut out from the blot and applied directly to a Procise protein sequencer (Applied Biosystems).<br>
Intact IgG in the starting material as well as in the purified fractions was quantified as described in example 7.<br>
SDS-gel analysis<br>
The produced IgG was analysed on SDS-gels in order to characterize the products in terms of degree of degradation.<br>
In NZ-2 (Bech2 with IgG produced from pNZ-3 and pNZ-4) substantial antibody heavy chain degradation was observed with several fragments appearing on SDS-PAGE as further identified by N-terminal sequencing. Cleavage was not sequence specific but was found in less structured linker regions of the heavy chain.<br>
In JaL741 (MT2874 with IgG produced from pNZ-3 and pNZ-4) significantly reduced levels of heavy chain degradation was observed compared to what was found in the NZ-2 product. Cleavage was found to be sequence specific with cleavage after the one Lys-Arg Site of the heavy chain sequence. Lys-Arg sites are known to be targets for the site specific KexB Protease of Aspergillus oryzae cleaving only the carboxyl side of Lys-Arg and Arg-Arg.<br><br>
Before PepC inactivation the site specific degradation, possibly due to kexB, was ob-scured.<br>
In JaL893 (IF04177 with IgG produced from pNZ-3 and pNZ-4) a very substantial anÂ¬tibody heavy chain degradation was observed with several fragments appearing on SDS-PAGE as further identified by N-terminal sequencing. Cleavage was not sequence specific but was found in less structured linker regions of the heavy chain.<br>
In JaL895 (JaL810 with IgG produced from pNZ-3 and pNZ-4) a very substantial antiÂ¬body heavy chain degradation was observed with several fragments appearing on SDS-PAGE as further identified by N-terminal sequencing. Cleavage was not sequence specific but was found in less structured linker regions of the heavy chain. This shows that deletion of the pepC Protease alone is not enough for reducing the heavy chain degradation.<br>
Example 9<br>
Construction of the Aspergillus orvzae kexB deletion Plasmid pJaL836<br>
From Plasmid pS02, which encode the A, oryzae IS04177 pyrG gene, a 5336 bp Spel-SspBI fragment and a 316 bp Asp718-Nhel fragment (part of the pyrG promoter) were purified and ligated resulting in plasmid pJaL554. The 316 bp fragment was cloned down-stream of the encoded pyrG gene, thereby creating a pyrG gene which is flanked by a re-peated sequence of 316 bp.<br>
From pJaL504 a 2514 bp fragment were amplified by PCR with primer 172450 and 172449 (SEQ ID NO: 21 and 22) and cloned into the vector pCRÂ®4Blunt-TOPO resulting in Plasmid pJaL574.<br>
From pJaL574 a 2587 bp fragment were amplified by PCR with primer T5483H12 and T5425G10 (SEQ ID NO: 23 and 24). This fragment was restriction digested with Hindlll and Ndel resulting in a 2582 bp fragment, which was cloned into the corresponding site in the vecÂ¬tor pUC19 resulting in plasmid pJaL835.<br>
Plasmid pJaL800 contains a 6968 bp Sali fragment from A.oryzae IF04177 encoding the kexB gene (SEQ ID NO: 25) in pUC19. A 4658 Bglll fragment from pJaL800 were sub-cloned into the Bglll site of the vector plC7 resulting in pJaL818. The repeat flanked pyrG se-lection marker from pJaL554 were moved as a 2313 bp Smal fragment and cloned into the Ball Site of pJaL818 resulting in plasmid pJaL819. The pyrG gene replaces thereby a 911 bp Ball encoding part of the kexB gene and the pyrG gene is then flanked by a 1292 bp fragment of the 5' end of kexB and a 2455 bp fragment of the 3' end of kexB. Finally, the deletion cas-<br><br>
sette of pMT2833 containing the two pepC flanks on either side of the pyrG selection marker was transferred as a 4063 bp EcoRI fragment into the EcoRI Sites of the tk counter selectable Plasmid pJaL835 to give the deletion plasmid pJaL836. Note that pJaL836 contains a unique Notl Site immediately downstream of the kexB 5' flank, which can be used to linearize the plasmid prior to transformation into A.oryzae.<br>
Example 10<br>
Construction of pepC and kexB deleted A.oryzae strain, JaL799 (JaL763-AkexB).<br>
A.	Isolation of a MT2874 pyrG minus strain, JaL763 (alp minus, npl minus, pepC miÂ¬<br>
nus)<br>
The A. oryzae strain MT2874 was screened for resistance to 5-flouro-orotic acid (FOA) to identify spontaneous pyrG mutants on minimal plates (Cove D.J. 1966. Biochem. Biophys. Acta. 113:51-56) supplemented with 1.0 M sucrose as carbon source, 10 mM sodi-umnitrate as nitrogen source, and 0.5 mg/ml FOA. One strain, ToC1418, was identifying as being pyrGminus. JaL763 is uridine dependent, therefore it can be transformed with the wild type pyrG gene and transformants selected by the ability to grow in the absence of uridine.<br>
B.	Construction of pepC and KexB deleted A.oryzae strain, JaL799-<br>
20 microgram of pJaL836 was cut with Notl and subsequently the enzyme was heat inactivated as recommended by the manufacturer (New England Biolabs). The plasmid was then ethanol precipitated and re-dlssolved in Tris buffer (lOmM pH 8.0) at a concentration suit-able for transformation into Aspergillus oryzae,<br>
The linearized plasmid DNA was transformed into Aspergillus oryzae JaL763 with seÂ¬lection for pyrG and counter selection of the tk gene on FDU plates as previously described in WO 0168864. Transformant colonies were twice re-isolated and finally grown up in liquid meÂ¬dium (YPD). Chromosomal DNA was prepared as previously described in WO 0168864, and used for Southern analysis of the kexB locus with the aim to identify transformants in which a clean double cross-over between the chromosomal kexB and the deletion cassette had oc-curred. The chromosomal DNA was digested with Bglll and Bglll-Hindlll. The Southern blot was first probed with the 5' flank excised as a 1292 kb Bglll-MluNI fragment from pJaL818 (Probel). For the wt intact kexB locus, a 4.6 kb Bglll fragment is expected to hybridize to this probe for both the Bglll and the Bglll-Hindlll digestion, white for the kexB deleted derivative originating from the desired double cross-over a 6.0 kb fragment and a 1.3 kb fragment will hybridize, respectively. The Southern was stripped of the first 5' flank probe and re-probed with a probe excised as a 0,8 kb MluNI fragment of pJaL818 (Probe 2). For the wt kexB locus a 4.6 kb fragment was again shown to hybridize to this fragment for both digestions, while for the kexB deletion strain originating from the desired cross over no hybridization was obtained. A<br><br>
strain with the above characteristics was preserved as A.oryzae strain JaL799 (JaL763-AkexB).<br>
Example 11<br>
Expression of lgG1 antibody in A. oryzae JaL799<br>
The strain JaL799 was transformed with expression plasmids pNZ-3, and pNZ-4, and a transformant was selected, as described in example 7, and named JaL827.<br>
The resulting IgG produced from JaL827 was characterized as described in example 8. In JaL827 (JaL799 with IgG produced from pNZ-3 and pNZ-4) significantly reduced levels of heavy chain degradation was observed compared to what was found in JaL741 (example 8). In JaL741 the major protein bands observed were found to be the result of sequence specific deÂ¬gradation with cleavage after the one Lys-Arg site of the heavy chain sequence. In contrast to this, in JaL827 none of this sequence specific degradation was seen and the minor bands were found to be the result of unspecific degradation. The result was that less degradation products were seen from JaL827.<br><br><br><br><br><br><br><br><br>
CLAIMS<br>
1.	A filamentous fungal cell in which an endogenous alkaline protease activity, an<br>
endogenous neutral metalloprotease activity and an endogenous serine protease activity of the<br>
subtilisin type have been completely or partially inactivated, wherein the alkaline protease, the<br>
neutral metalloprotease, and the serine protease of the subtilisin type are encoded from<br>
nucleotide sequences selected from the group consisting of:<br>
(a)	a nucleotide sequence encoding an alkaline protease having an amino acid sequence which has at least 65% identity with the amino acids of SEQ ID NO: 30, a nucleotide sequence encoding a neutral metalloprotease which has at least 65% identity with the amino acids of SEQ ID NO: 31, and a nucleotide sequence encoding a serine protease of the subtilisin type having an amino acid sequence which has at least 65% identity with the amino acids of SEQ ID NO: 32; or<br>
(b)	a nucleotide sequence having at least 65% identity with SEQ ID NO : 26, a nucleotide sequence having at least 65% identity with SEQ ID NO : 27, and a nucleotide sequence having at least 65% identity with SEQ ID NO : 28; or<br>
(c)	nucleotide sequences which hybridizes under at least medium stringency conditions<br>
with either of the nucleic acid sequences SEQ ID NO: 26, 27, or 28.<br>
2,	The filamentous fungal cell according to claim 1, wherein the alkaline protease activity<br>
is encoded by the alp gene, the neutral metalloprotease activity is encoded by the npl gene,<br>
and the serine protease activity of the subtilisin type is encoded by the pepC gene.<br>
3	The filamentous fungal cell according to claims 1  or 2, wherein an endogenous<br>
calcium dependent, neutral, serine protease activity of the kexin subfamily has been completely or partially inactivated, and wherein the said calcium dependent, neutral, serine protease is encoded from a nucleotide sequence selected from the group consisting of:<br>
(a)	a nucleotide sequence encoding a calcium dependent, neutral, serine protease activity of the kexin subfamily which has at least 65% identity with the amino acids of SEQ ID NO: 33; or<br>
(b)	a nucleotide sequence having at least 65% identity with SEQ ID NO : 29; or<br><br>
(c) nucleotide sequences which hybridizes under at least medium stringency conditions with the nucleic acid sequences SEQ ID NO: 29.<br>
4.	The filamentous fungal cell according to claim 3, which has the phenotype alp, np1, pepC, and kexB, wherein the alp gene is encoded by a nucleotide sequence which has at least 70% identity to SEQ ID NO: 26, the np1 gene is encoded by a nucleotide sequence which has at least 70% identity to SEQ ID NO: 27, the pepC gene is encoded by a nucleotide sequence which has at least 70% identity to SEQ ID NO: 28, and the kexB gene is encoded by a nucleotide sequence which has at least 70% identity to SEQ ID NO: 29.<br>
5.	The filamentous fungal cell according to any of the preceding claims, wherein the fungal cell is chosen from the group consisting of the genera Aspergillus, Acremonium, Fusarium, Humlcola, Mucor, Myceliophthora, Neurospora, Penicillium, Thielavia, Tolypocladium, and Trichoderma.<br>
6.	The filamentous fungal cell according to claim 5, wherein the Aspergillus comprises Aspergillus awamori, Aspergillus foetidus, Aspergillus japonicus, Aspergillus aculealus, Aspergillus niger, Aspergillus nidulans or Aspergillus oryzae.<br>
7.	A method for producing a heterologous polypeptide in a filamentous fungal host cell according to any of the claims 1-6, comprising<br>
(a)	introducing into the host cell a nucleic acid sequence encoding the heterologous<br>
polypeptide;<br>
(b)	cultivating the cell from (a) in a suitable growth medium under conditions conducive for expression of the heterologous polypeptide; and<br>
(c)	Isolating the protein product.<br><br>
8.	The method according to claim 7, wherein the heterologous polypeptide is a mammalian polypeptide.<br>
9.	The method according to claim 8, wherein the polypeptide is an antibody or an antibody fragment.<br><br>
10.	The method according to claim 9, wherein the antibody or antibody fragment is<br>
*<br>
selected from the group consisting of lgG1, lgG2. lgG3, lgG4, IgM, IgA, IgD, IgE, F(ab')2, and Fab.<br>
11.	The method according to claim 10, wherein the antibody is IgGI,<br><br></td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTkwNyBDSEVOUCAyMDA4IFBFVElUSU9OIFBPUi5wZGY=" target="_blank" style="word-wrap:break-word;">1907 CHENP 2008 PETITION POR.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTkwNy1DSEVOUC0yMDA4ICAgQU1FTkRFRCAgQ0xBSU1TICAgIDMwLTAxLTIwMTQucGRm" target="_blank" style="word-wrap:break-word;">1907-CHENP-2008   AMENDED  CLAIMS    30-01-2014.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTkwNy1DSEVOUC0yMDA4ICAgQU1FTkRFRCAgUEFHRVMgIE9GIFNQRUNJRklDQVRJT04gICAzMC0wMS0yMDE0LnBkZg==" target="_blank" style="word-wrap:break-word;">1907-CHENP-2008   AMENDED  PAGES  OF SPECIFICATION   30-01-2014.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTkwNy1DSEVOUC0yMDA4ICAgQU1FTkRFRCAgUEFHRVMgT0YgU1BFQ0lGSUNBVElPTiAgIDIyLTEwLTIwMTMucGRm" target="_blank" style="word-wrap:break-word;">1907-CHENP-2008   AMENDED  PAGES OF SPECIFICATION   22-10-2013.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTkwNy1DSEVOUC0yMDA4ICAgQU1FTkRFRCBDTEFJTVMgICAyMi0xMC0yMDEzLnBkZg==" target="_blank" style="word-wrap:break-word;">1907-CHENP-2008   AMENDED CLAIMS   22-10-2013.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTkwNy1DSEVOUC0yMDA4ICAgQU1FTkRFRCBQQUdFUyBPRiBTUEVDSUZJQ0FUSU9OICAwMy0xMi0yMDE0LnBkZg==" target="_blank" style="word-wrap:break-word;">1907-CHENP-2008   AMENDED PAGES OF SPECIFICATION  03-12-2014.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTkwNy1DSEVOUC0yMDA4ICAgQ09SUkVTUE9OREVOQ0UgT1RIRVJTIDA3LTA1LTIwMTMucGRm" target="_blank" style="word-wrap:break-word;">1907-CHENP-2008   CORRESPONDENCE OTHERS 07-05-2013.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTkwNy1DSEVOUC0yMDA4ICAgRVhBTUlOQVRJT04gUkVQT1JUIFJFUExZIFJFQ0VJVkVEICAyMi0xMC0yMDEzLnBkZg==" target="_blank" style="word-wrap:break-word;">1907-CHENP-2008   EXAMINATION REPORT REPLY RECEIVED  22-10-2013.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTkwNy1DSEVOUC0yMDA4ICAgRVhBTUlOQVRJT04gUkVQT1JUIFJFUExZIFJFQ0VJVkVEICAzMC0wMS0yMDE0LnBkZg==" target="_blank" style="word-wrap:break-word;">1907-CHENP-2008   EXAMINATION REPORT REPLY RECEIVED  30-01-2014.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTkwNy1DSEVOUC0yMDA4ICAgRVhBTUlOQVRJT04gUkVQT1JUIFJFUExZIFJFQ0lFVkVEIDAzLTEyLTIwMTQucGRm" target="_blank" style="word-wrap:break-word;">1907-CHENP-2008   EXAMINATION REPORT REPLY RECIEVED 03-12-2014.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTkwNy1DSEVOUC0yMDA4ICAgRk9STS0xICAyMi0xMC0yMDEzLnBkZg==" target="_blank" style="word-wrap:break-word;">1907-CHENP-2008   FORM-1  22-10-2013.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTkwNy1DSEVOUC0yMDA4ICAgRk9STS0xMyAgMzAtMDEtMjAxNC5wZGY=" target="_blank" style="word-wrap:break-word;">1907-CHENP-2008   FORM-13  30-01-2014.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTkwNy1DSEVOUC0yMDA4ICAgRk9STS0zICAyMi0xMC0yMDEzLnBkZg==" target="_blank" style="word-wrap:break-word;">1907-CHENP-2008   FORM-3  22-10-2013.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTkwNy1DSEVOUC0yMDA4ICAgRk9STS0zICAzMC0wMS0yMDE0LnBkZg==" target="_blank" style="word-wrap:break-word;">1907-CHENP-2008   FORM-3  30-01-2014.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTkwNy1DSEVOUC0yMDA4ICAgT1RIRVIgUEFURU5UIERPQ1VNRU5UICAgMjItMTAtMjAxMy5wZGY=" target="_blank" style="word-wrap:break-word;">1907-CHENP-2008   OTHER PATENT DOCUMENT   22-10-2013.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTkwNy1DSEVOUC0yMDA4ICAgUE9XRVIgT0YgQVRUT1JORVkgICAyMi0xMC0yMDEzLnBkZg==" target="_blank" style="word-wrap:break-word;">1907-CHENP-2008   POWER OF ATTORNEY   22-10-2013.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTkwNy1DSEVOUC0yMDA4ICAgUFJJT1JJVFkgRE9DVU1FTlQgICAyMi0xMC0yMDEzLnBkZg==" target="_blank" style="word-wrap:break-word;">1907-CHENP-2008   PRIORITY DOCUMENT   22-10-2013.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTkwNy1DSEVOUC0yMDA4ICBBTUVOREVEIENMQUlNUyAxNi0xMi0yMDE0LnBkZg==" target="_blank" style="word-wrap:break-word;">1907-CHENP-2008  AMENDED CLAIMS 16-12-2014.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTkwNy1DSEVOUC0yMDA4ICBBTUVOREVEIFBBR0VTIE9GIFNQRUNJRklDQVRJT04gIDE2LTEyLTIwMTQucGRm" target="_blank" style="word-wrap:break-word;">1907-CHENP-2008  AMENDED PAGES OF SPECIFICATION  16-12-2014.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTkwNy1DSEVOUC0yMDA4ICBDT1JSRVNQT05ERU5DRSBPVEhFUlMgICAyOC0xMS0yMDE0LnBkZg==" target="_blank" style="word-wrap:break-word;">1907-CHENP-2008  CORRESPONDENCE OTHERS   28-11-2014.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTkwNy1DSEVOUC0yMDA4ICBDT1JSRVNQT05ERU5DRSBPVEhFUlMgIDEwLTA3LTIwMTQucGRm" target="_blank" style="word-wrap:break-word;">1907-CHENP-2008  CORRESPONDENCE OTHERS  10-07-2014.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTkwNy1DSEVOUC0yMDA4ICBFWEFNSU5BVElPTiBSRVBPUlQgUkVQTFkgUkVDSUVWRUQgMTYtMTItMjAxNC5wZGY=" target="_blank" style="word-wrap:break-word;">1907-CHENP-2008  EXAMINATION REPORT REPLY RECIEVED 16-12-2014.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTkwNy1DSEVOUC0yMDA4ICBPVEhFUiBQQVRFTlQgRE9DVU1FTlQgMTYtMTItMjAxNC5wZGY=" target="_blank" style="word-wrap:break-word;">1907-CHENP-2008  OTHER PATENT DOCUMENT 16-12-2014.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTkwNy1DSEVOUC0yMDA4IEZPUk0tMTggMDYtMTAtMjAwOS5wZGY=" target="_blank" style="word-wrap:break-word;">1907-CHENP-2008 FORM-18 06-10-2009.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTkwNy1jaGVucC0yMDA4LWFic3RyYWN0LnBkZg==" target="_blank" style="word-wrap:break-word;">1907-chenp-2008-abstract.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTkwNy1jaGVucC0yMDA4LWNsYWltcy5wZGY=" target="_blank" style="word-wrap:break-word;">1907-chenp-2008-claims.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTkwNy1jaGVucC0yMDA4LWNvcnJlc3BvbmRuZWNlLW90aGVycy5wZGY=" target="_blank" style="word-wrap:break-word;">1907-chenp-2008-correspondnece-others.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTkwNy1jaGVucC0yMDA4LWRlc2NyaXB0aW9uKGNvbXBsZXRlKS5wZGY=" target="_blank" style="word-wrap:break-word;">1907-chenp-2008-description(complete).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTkwNy1jaGVucC0yMDA4LWZvcm0gMS5wZGY=" target="_blank" style="word-wrap:break-word;">1907-chenp-2008-form 1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTkwNy1jaGVucC0yMDA4LWZvcm0gMjYucGRm" target="_blank" style="word-wrap:break-word;">1907-chenp-2008-form 26.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTkwNy1jaGVucC0yMDA4LWZvcm0gMy5wZGY=" target="_blank" style="word-wrap:break-word;">1907-chenp-2008-form 3.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTkwNy1jaGVucC0yMDA4LWZvcm0gNS5wZGY=" target="_blank" style="word-wrap:break-word;">1907-chenp-2008-form 5.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTkwNy1jaGVucC0yMDA4LXBjdC5wZGY=" target="_blank" style="word-wrap:break-word;">1907-chenp-2008-pct.pdf</a></p>
		<br>
		<div class="pull-left">
			<a href="264664-a-releasable-drive-coupling-for-an-injection-device-used-for-administering-a-drug-to-a-patient-and-an-injection-device.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="264666-catalyst-composition.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>264665</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>1907/CHENP/2008</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>03/2015</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>16-Jan-2015</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>14-Jan-2015</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>17-Apr-2008</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>NOVOZYMES A/S</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td>KROGSHOEJVEJ 36 DK-2880 BAGSVAERD</td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>HANSEN, MOGENS, TRIER</td>
											<td>MOSEVANG 9 DK-3540 LYNGE</td>
										</tr>
										<tr>
											<td>2</td>
											<td>JORGENSEN, CHRISTIAN, ISAK</td>
											<td>BONDEHAVEVEJ 92 DK-2880 BAGSVAERD</td>
										</tr>
										<tr>
											<td>3</td>
											<td>LEHMBECK, JAN</td>
											<td>KORSHOJ 126 DK-3670 VEKSO</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>C12N 1/15</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>PCT/DK06/00586</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td>2006-10-17</td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td>PA 2005 01452</td>
									<td>2005-10-17</td>
								    <td>Denmark</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/264665-use-of-fungal-mutants-for-expression-of-antibodies by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 02:20:39 GMT -->
</html>
