<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/265066-methods-for-drug-discovery-using-a-single-molecule-detection-apparatus by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 03:17:49 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 265066:&quot;METHODS FOR DRUG DISCOVERY USING A SINGLE-MOLECULE DETECTION APPARATUS&quot;</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">&quot;METHODS FOR DRUG DISCOVERY USING A SINGLE-MOLECULE DETECTION APPARATUS&quot;</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>The present application discloses methods and apparatuses for single molecule drag screening, discovery and validation. These methods and apparatuses allow a user to detect rapidly, using observation of single molecules, whether and how a drag candidate interferes with a target enzyme involved in a particular disease pathway. The methods and apparatuses described herein utilize single molecule manipulation and detection technologies (e.g., optical or magnetic tweezers) to directly detect whether the characteristic dynamics, or &amp;quot;mechanical signature,&amp;quot; of the target enzyme-substrate interaction are substantially altered or modulated by a drag candidate. Furthermore, the methods and apparatuses are useful for analyzing the modulation of the mechanical signature in order to identify potential interference mechanisms of a drag candidate. In one aspect of the invention, the methods and apparatuses disclosed herein relate to monitoring the real-time dynamic mechanical signatures of individual polymerase molecules (e.g. DNA polymerases, RNA polymerases, and reverse transcriptases) along a polynucleotide substrate in the presence of drag candidates that either inhibit or otherwise modulate the polymerization process. Identification and analysis of such drag candidates is critical for anti-viral, anti-cancer, and antibiotic drug development</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td>Single-Molecule Platform for Drug Discovery: Methods and Apparatuses for Drug Discovery, including Discovery of Anticancer and Antiviral Agents<br>
CONTINUING APPLICATION DATA<br>
This application claims benefit of U.S. Provisional Application No. 60/793,720, filed Apri3 21, 2006, which is incorporated by reference herein in its entirety.<br>
FIELD OF THE INVENTION<br>
The invention relates to the screening and validation of drug candidates that target enzymes, including DNA polymerases, RNA polymerases, and reverse-transcriptases.<br>
BACKGROUND OF THE INVENTION<br>
Approximately thirty percent of drugs in clinical use inhibit a disease-related enzymatic process (Copeland, R.A. Evaluation of enzyme inhibitors in drug discovery: a guide for medicinal chemists and phannacologists. (Wiley-Interscience, 2005)). Thus, the discovery of new enzyme inhibitors is an important area of research in biochemistry and pharmacology.<br>
Polymerase inhibitors are valuable in both clinical and research settings. These inhibitors help in elucidating the mechanistic aspects of transcription and DNA replication, in mapping structure-function relationships, and in characterizing protein activity. Polymerase inhibitors are also among the most attractive drug targets. Knowledge about these inhibitors, their structures, and their mechanisms enable the design of new drugs such as anti-cancer agents, antiviral agents, and antibiotics that will be effective against new pathogens and antibiotic-resistant mutants of known pathogens. Because some of these inhibitors have been reported to induce and/or inhibit apoptosis, they also provide valuable tools for investigating apoptosis. Likewise, because some of these agents block specific steps of DNA transcription, polymerase inhibitors can help to elucidate the role of transcriptional control in regulating the expression of target genes in various healthy and disease states.<br>
Drugs that target polymerase proteins involved in particular disease pathways are well known in the art. Reverse transcriptase inhibitors (RTIs), for example, are a class of antiretroviral drugs that target construction of viral DNA by inhibiting the activity of reverse transcriptase.<br>
There are two subtypes of RTIs with different mechanisms of action: nucleoside and nucleotide analogue RTIs are incorporated into the viral DNA leading to chain termination, while non-nucleoside-analogue RTIs act as competitive inhibitors of the reverse transcriptase enzyme. Current AIDS therapeutics that function by inhibiting HIV reverse transcriptase are described in the art (see, e.g., Bean et al., Appl Environ Microbiol 72:5670-5672 (2005)), and include Efavirenz (brand names SUSTTVA® and STOCRIN®) and Nevirapine (also marketed under the trade name VIRAMUNE®). Antibiotics that target polymerase proteins (e.g. rifampin) and cancer drugs that target polymerase proteins (e.g. cisplatin) are also known in the art.<br>
Many drugs have been found to be efficacious in the treatment of cancer. These include diverse chemical compounds such as antimetabolites (e.g., methotrexate and fluorouracil), DNA-damaging agents (e.g., cyclophosphamide, cisplatin, and doxorubicin), mitotic inhibitors (e.g., vincristine), nucleotide analogues (e.g., 6-mercaptopurine), inhibitors of topoisomerases involved in DNA repair (e.g., etoposide), inhibitors of DNA polymerase (e.g., bleomycin), and intercalating agents like mitoxantrone.<br>
Several drugs targeting enzymes of mammalian DNA replication are currently being investigated as promising candidates for cancer chemotherapy or as probes for understanding the roles of specific enzymes in DNA replication and repair. These potential drug candidates include corylifolin, bakuchiol, resveratrol, Neobavaisoflavone,and daidzein (see Sun et al., J. Nat. Prod. 61, 362-366 (1998)).<br>
Other examples of DNA and RNA polymerase inhibitors include Actinomycin D, Streptomyces sp.; a-Axnanitin, Amanita sp.; Aphidicolin, HSV replication inhibitor, BP5; Methyl a-Amanitin Oleate; Novobiocin, Sodium Salt; Rifarnpicin; RNA Polymerase ITf Inhibitor; and Actinomycin D, 7-Amino. Three polymerase inhibitors currently in Phase II trials for use against Hepatitis C Virus are Idenix/Novartis' valopicitabine (NM283); ViroPharma's HCV-796; and Roche's R1626. Roche/Idenix are also investigating valtorcitabine (val-LdC), a first strand viral DNA synthesis inhibitor in Phase IIHCV trials after initial success as an HBV treatment.<br>
DNA damaging agents provide some of the most successful treatments for cancer. The enzyme Poly(ADP-ribose)polymerale (i.e. PARP) can help repair DNA damage caused by the DNA damaging agents used to treat cancer. As PARP activity is often increased in cancer cells, it provides these cells with a survival mechanism. ABT-888 (Abort Oncology), for example, is an oral PARP-inhibitor developed by Abbott to prevent DNA repair in cancer cells and increase the effectiveness of common cancer therapies such as radiation and alkylating agents. Moreover, preclinical data indicates ABT-888 has improved the effectiveness of radiation and many types of chemotherapy in animal models of cancer.<br>
These selected publications from the last 5 years illustrate the current state of the art with<br>
regard to the activity, mechanisms, and biochemistry of polymerase inhibitors:<br>
Brown JA, Duym WW, Fowler JD, Suo, Z.. (2007) "Single-turnover Kinetic Analysis of the<br>
Mutagenic Potential of 8-Oxo-7,8-dihydro--2'-deoxyguanosine during Gap-filling Synthesis<br>
Catalyzed by Human DNA Polymerases lambda and beta." J Mol Biol. [EpUb ahead of print]<br>
Suo, Z., Abdullah MA. (2007) "Unique Composite Active Site of the Hepatitis C Virus NS2-<br>
3 Protease: a New Opportunity for Antiviral Drug Design." ChemMedChem. 2(3), 283-284.<br>
Roettger MP, FialaKA, Sompalli S, Dong Y, Suo Z. (2004) "Pre-steady-state kinetic studies<br>
of the fidelity of human DNA polymerase mu", Biochemistry 43 (43), 13 827-38.<br>
FialaKA, Abdel-Gawad W, Suo Z. (2004) "Pre-steady-state kinetic studies of the fidelity<br>
and mechanism of polymerization catalyzed by truncated human DNA polymerase lambda.",<br>
Biochemistry 43(21), 6751-62.<br>
Fiala, K. A &amp; Suo Z* (2004) Pre-Steady State Kinetic Studies of the Fidelity of Sulfolobus<br>
solfataricus P2 DNA Polymerase IVJBiochemistry 43,2106-2115<br>
Fiala, K. A &amp; Suo Z.* (2004) Mechanism of DNA Polymerization Catalyzed by Sulfolobus<br>
solfataricus P2 DNA Polymerase rV. Biochemistry 43, 2116-2125<br>
Fiala, K. A, Abdel-Gawad, W. &amp; Suo Z.* (2004) Pre-Steady-State Kinetic Studies of the<br>
Fidelity and Mechanism of Polymerization Catalyzed by Truncated Human DNA Polymerase<br>
Lambda.Biochemistry, accepted and in press.<br>
Allison, A. J., Ray, A., Suo Z.., Colacino, J. M., Andeson, K. S., Johnson, K.A. (2001)<br>
"Toxicity of Antiviral Nucleoside Analogs and the Human Mitochondrial DNA Polymerase",<br>
J. Biol. Chem. 276, 40847-40857.<br>
New drugs are the products of a long and involved drug development process, the first step of which is the discovery of compounds with promising activity. New enzyme inhibitors can be discovered by screening libraries of drug candidate compounds against a target enzyme. Conventional drug screening and validation approaches utilize micro- to milli-scale biochemical or cellular assays to detect downstream biochemical or cellular signatures of enzymatic interference. In view of the limitations of conventional drug screening methods, there remains a need in the art for improved methods and apparatuses for the detection of promising drug candidates.<br>
SUMMARY OF THE INVENTION<br>
The present application discloses methods and apparatuses for single molecule drug screening, discovery and validation. These methods and apparatuses allow a user to detect rapidly, using observation of single molecules, whether and how a drug candidate interferes with a target enzyme involved in a particular disease pathway. The methods and apparatuses described herein utilize single molecule manipulation and detection technologies (e.g., optical or magnetic tweezers) to directly detect whether the characteristic dynamics, or "mechanical signature," of the target enzyme-substrate interaction are substantially altered or modulated by a drug candidate. Furthermore, the methods and apparatuses are useful for analyzing the modulation of the mechanical signature in order to identify potential interference mechanisms of a drug candidate.<br>
In one aspect of the invention, the methods and apparatuses disclosed herein relate to monitoring the real-time dynamic mechanical signatures of individual polymerase molecules (e.g. DNA polymerases, RNA polymerases, and reverse transcriptases) along a polynucleotide substrate in the presence of drug candidates that either inhibit or otherwise modulate the polymerization process. Identification and analysis of such drug candidates is critical foT anti-viral, anti-cancer, and antibiotic drug development.<br>
BRIEF DESCRIPTION OF THE DRAWINGS<br>
FIG. 1 illustrates exemplary flow charts of a method aspect of this invention for screening multiple drug candidates that target a particular enzyme.<br>
FIG. 2 illustrates exemplary mechanical si gnatures from DNA polymerase in the presence of various drug candidates (such as polymerase inhibitors). Panel A of FIG. 2 shows a representative signal expected from a control sample where no polymerase inhibitor is present. In this scenario, the polymerase binds to single stranded (ss) DNA and converts this template into double stranded (ds) DNA, steadily shortening the overall template length with time. Panel B shows an exemplary mechanical signature expected from an optical tweezers apparatus that would indicate that a drug candidate has slowed the replication process. Note that the overall slope of the template length plot, which corresponds to the polymerization velocity, has lessened. Panel C shows an exemplary mechanical signature from an optical tweezers apparatus that indicates that a drug candidate has induced abortive transcription/premature termination, wherein the drug candidate has derailed or stalled the replication process before its natural completion point. Panel D shows an exemplary mechanical signature from an optical tweezers apparatus that indicates that initiation of DNA replication is inhibited by the drug candidate. Panel E shows an exemplary mechanical signature from an optical tweezers apparatus that indicates the drug candidate induces the polymerase enzyme to operate in an exonucleolysis mode, wherein it excises bases rather than polymerizes base-pairs. This signature would likely only occur in polymerases with an active exonucleolysis, or "proof-reading" site.<br>
FIG. 3 illustrates exemplary flow charts of a method aspect of this invention for screening multiple drug candidates that target a particular polymerase enzyme, including DNA and RNA polymerase inhibitors and reverse transcriptase inhibitors.<br>
FIG. 4a illustrates exemplary experimental geometries for obtaining, via an optical tweezers-based single-molecule measurement system, the measurement of a polymerase's position along a nucleic acid template template. Here a DNA molecule is held immobilized and stretched between two plastic latex beads, as the polymerase moves along the DNA template. In the case of DNA replication, the distance between the two beads decreases at a given force as the template DNA is converted from single to double stranded DNA. The methods of attachment of the nucleic acid to the beads can be via streptavidin-biotin, or dig-anti-dig, or other covalent linkages.<br>
FIG. 4b illustrates additional exemplary experimental geometries for obtaining, via an optical tweezers-based single-molecule measurement system, the length of a nucleic acid as it is processed by a polymerase enzyme. Here a DNA molecule is held immobilized and stretched between a plastic latex bead and a streptavidin-coated (triangles) glass surface. As the polymerase moves along the DNA template in DNA replication, the length of the the DNA tether decreases at a given force as the immobilized template DNA is converted from single to double stranded DNA.<br>
FIG. 4c illustrates an embodiment of the present invention where the nucleic acid template is attached to a dielectric bead that is held by a second, immoblized optical trap, shown at left, that exerts a strong trapping force that is,much greater than the force exerted by the polymerase on the nucleic acid template. Here the polymerase is attached by the relevant affinity chemistry to the other bead. Figure 4-d demonstrates how the nucleic acid can be immobilized to a rigid surface, such as a cover slip or microwell plate, via complementary functionalization of the nucleic acid template and the rigid surface, while again the polymerase is attached to the second bead.<br>
FIG. 5a illustrates schematically the single molecule detection system which includes a representative optical tweezers apparatus that comprises trapping, force detection, beam steering and isotension capabilities and force-feedback optical trapping subsystem that maintains a constant force on a trapped bead, even as the bead experiences other forces due to the enzymatic activity of a polymerase.<br>
FIG. 5b illustrates the detection system in the single molecule tracking system by which a quadrant photodiode is utilized to detect the displacement of a dielectric sphere from an optical trap's center. The position of the bead with respect to the optical trap is recorded onto this quadrant photodetector. Deviations from this center of the optical well are used to quantify the picoNewton sized forces acting on the bead.<br>
FIGS. 6a and b illustrate data demonstrating the capability to observe the dynamics of the DNA polymerase motor with an optical tweezers apparatus as it moves forwards (polymerization, FIG. 6a) and backwards (exonuclease activity, FIG. 6b) along a DNA<br>
template. A velocity of about 0.5 rnicrons/minute corresponds to about 25 bases-pairs polymerized/second. In the figure, position of the polymerase along DNA is shown as a function of time. The slope gives the rate of change in length of the DNA template at a given force while it is being replicated. This is just the single molecule velocity of the polymerase motor.<br>
FIG. 7 illustrates the ability to probe even fine-structured dynamics of DNA polymerization. To achieve this level of resolution, an acousto-optic deflector system is preferably utilized to achieve isotension conditions along the DNA strand during polymerization. In the figure, the extension (um) of the DNA is shown versus time (min). In the left plot, the black line shows full data set, including relaxation due to flow (left boxed region) before the contraction (right hand boxed region) begins. At right, extension versus time is shown for the contraction region. The raw data line shows original length-vs-time. data, while the smoothed line shows 100-pt adjacent average of this data. The straight line indicates the average slope through this contraction region, about 300 bp/second.<br>
FIG. 8 illustrates key features of the conventional approach to drug screening and discovery.<br>
FIG. 9 illustrates key features of the novel drug screening and discovery techniques described herein.<br>
FIG. 10 illustrates some of the advantages of the novel drug screening and discovery techniques described herein as compared to conventional approaches to drug screening and discovery.<br>
DETAILED DESCRIPTION OF EMBODIMENTS OF THE INVENTION<br>
DNA and RNA polymerase and reverse transcriptase (RT) inhibitors are valuable in both clinical and research settings. These inhibitors help in elucidating the mechanistic aspects of transcription and DNA replication, in mapping structure-function relationships, and in characterizing protein activity. These polymerase and RT inhibitors are also among the most attractive drug targets. Knowledge about these inhibitors, their structures, and their mechanisms enable the design of new drugs such as anti-cancer agents and antibiotics that<br>
will be effective against new pathogens and antibiotic-resistant mutants of known pathogens. Because some of these inhibitors have been reported to induce and/or inhibit apoptosis, they also provide valuable tools for investigating apoptosis. Likewise, because some of these agents block specific steps of DNA transcription, polymerase inhibitors can help to elucidate the role of transcriptional control in regulating the expression of target genes in various healthy and disease states.<br>
Drugs that target polymerase proteins involved in a particular disease pathway are well known in the art. Reverse transcriptase inhibitors (RTIs), for example, are a class of antiretroviral drugs that target construction of viral DNA by inhibiting the activity of reverse transcriptase. There are two subtypes of RTIs with different mechanisms of action: nucleoside and nucleotide analogue RTIs are incorporated into the viral DNA leading to chain termination, while non-nucleoside-analogue RTIs act as competitive inhibitors of the reverse transcriptase enzyme. Current AIDS therapeutics that function by inhibiting HTV reverse transcriptase are described in the art (see, e.g., Bean et al., Appl Environ Microbiol 72:5670-5672 (2005)), and include Efavirenz (brand names SUSTIVA® and STOCRTN®) and Nevirapine (also marketed under the trade name VIRAMUNE®).<br>
Many drugs have been found to be efficacious in the treatment of cancer. These include diverse chemical compounds such as antimetabolites (e.g., methotrexate and fluorouracil), DNA-damaging agents (e.g., cyclophosphamide, cisplatin, and doxorubicin), mitotic inhibitors (e.g., vincristine), nucleotide analogues (e.g., 6-mercaptopurme), inhibitors of topoisomerases involved in DNA repair (e.g., etoposide), inhibitors of DNA polymerase (e.g., bleomycin), or intercalating agents like mitoxantrone.<br>
Several drugs targeting enzymes of mammalian DNA replication are currently being investigated as promising candidates for cancer chemotherapy or as probes for understanding the roles of specific enzymes in DNA replication and repair. These potential drug candidates.include, but are not limited to corylifolin, bakuchiol,resveratrol, Neobavaisoflavone, and daidzein. (cite Sun et al)<br>
Other examples of DNA and RNA Polymerase Inhibitors include: Actinomycin D, Streptomyces sp.; a-Amanitin, Amanita sp»; Aphidicolin; HSV Replication Inhibitor, BP5; Methyl a-Amanitin Oleate; Novobiocin, Sodium Salt; Rifampicin; RNA Polymerase III Inhibitor; Actinomycin D, 7-Amino. Three polymerase inhibitors currently in Phase II trials for use against Hepatitis C Virus are Idenix/Novartis' valopicitabine (NM283); ViroPharma's HCV-796 and Roche's R1626.<br>
Roche/ldenix are also investigating valtorcitabine (val-LdC) - a first strand viral DNA synthesis inhibitor in Phase IIHCV trials after initial success as an HBV treatment.<br>
DNA damaging agents provide some of the most successful treatments for cancer. The enzyme Poly (ADP-ribose)polymerase (i.e. PARP) can help repair DNA damage caused by the DNA damaging agents used to treat cancer a. As PARP activity is often increased in cancer cells, it provides these cells with a survival mechanism. ABT-888, for example, is an oral PARP-inhibitor developed by Abbott Oncology to prevent DNA repair in cancer cells and increase the effectiveness of common cancer therapies such as radiation and alkylating agents.. Moreover, preclinical data indicates ABT-888 has improved the effectiveness of radiation and many types of chemotherapy in animal models of cancer.<br>
New drugs are the products of a long and involved drug development process, the first step of which is the discovery of compounds with promising activity. New enzyme inhibitors "can be discovered by screening libraries of drug candidate compounds against a target enzyme. Drug candidates include compounds found in nature, compounds synthesized by combinatorial chemistry approaches, and compounds created via rational drug design. Conventional drug screening and validation approaches utilize micro- to milli-scale biochemical or celluiar assays to detect downstream biochemical or cellular signatures of enzymatic interference. For example, an assay might, via radioactive labeling, measure any changes in the quantity of a reaction product whose synthesis is catalyzed by the target enzyme.<br>
Single-molecule techniques offer several key benefits over conventional in vitro assay methods for drug screening, as they use less reagents and offer much more detail into the mechanism of drug action on the target. For example, single molecule techniques enable<br>
transient states to be observed, thereby making it possible to selectively screen for chemical compounds that isolate these steps. Single molecule approaches thus enable the identificiation, testing, and validation of polymerase or enzyme inhibitors that target key phases in biochemical processes, e.g., transcription or replication initiation. Many biochemical processes consist of multiple Ixansient steps, such as promoter binding, initiation, elongation, and termination in transcription. Because the total number of potential drug targets can be extremely high, single molecule approaches provide a critical advantage in speeding up the process of drug screening and discovery by focusing the efforts early on to only those steps of the process that are most affected by the drug candidate.<br>
By elucidating the kinetic mechanisms of enzymes involved in DNA/RNA replication, repair, antiviral and anti-cancer drug candidates can be identified based on rational drug design. Kinetic studies use a variety of pre-steady state kinetic methods including rapid chemical quench-flow and stopped-fiow techniques. These methods allow reactions to be quenched in milliseconds, and provide more kinetic information than the traditional steady-state kinetic methods. Single molecule techniques elucidate the elementary steps of reactions occurring at the active sites of enzymes and can significantly enhance rational drug design.<br>
The DNA in every cell of the human body is spontaneously damaged more than 10,000 times every day.DNA repair plays a major role to maintain genome integrity in cells.<br>
Fears about the possible release of smallpox bybioterrorists have led to intensive efforts to find an effective molecule to inhibit viral infection which does not yet exist. Since smallpox virus (variola virus) and the smallpox vaccine (vaccinia virus) are highly homologous, the latter has been used as a very good surrogate model. Vaccinia virus DNA polymerase, for instance, is about 99% identical to in the polymerase in the smallpox virus.<br>
The Hepatitis C virus has infected at least 2-3% of human population. Viral genome replication has been intensively studied. The RNA-dependent RNA polymerase, NS5B, is central to viral replication, and is a major antiviral drug target. Although there are extensive biochemical and steady-state kinetic studies on this polymerase, the elementary steps of .<br>
nucleotide incorporation catalyzed by NS5B are still undefined. These studies investigations enable the rational design of nucleoside inhibitors.<br>
In the last decade, new tools (for example optical tweezers, atomic-force-microscopy, and small glass fibers) have been developed to manipulate small objects and also to investigate the forces involved in the systems studied (see, e.g., Smith et al., Science 271:795 (1996) and Cluzel, et al., Science 271, 795 (1996)). In particular, optical or magnetic "tweezers" or "traps" trap particles with forces generated by optical intensity gradients, and can be used to manipulate and study microscopic molecules at the single-molecule level. Optically generated forces strong enough to form a three-dimensional trap can be obtained by bringing a laser beam with an appropriately shaped wavefront to a tight focus with a high numerical aperture lens. The principles of optical trapping are well known in the art and are summarized in, for example, Neuman and Block, Rev. Sci. Instr. 75:2787-2809 (2004).<br>
In the biological sciences, optical tweezers have been used to measure displacements in the nm range of molecules ranging in size from lOnm to over 100 mm. Common to most optical tweezers biophysical experiments is the attachment of dielectric beads to biological molecules (e.g. substrates and/or enzymes), so that the biological molecules can be manipulated by the optical trap and mechanical measurements can be taken. Various biochemical and molecular biology methods are known in the art for attaching nucleic acids, other substrates, enzymes and other biomacromolecules to functionalized surfaces and beads. For example, DNA can be labeled with biotin moieties that will bind to commercially available, streptavidin-coated, micron-coated dielectric spheres (e.g., from Bangs' Laboratories).<br>
Two of the main uses for optical traps in biology have been the study of the physical properties of DNA, and the study of molecular motors such as DNA and R3STA polymerases ((see, e.g., Davenport et al., Science 287:2497-2500 (2000); Maier et al., PNAS 97:12002-12007 (2000); Wang et al., Science 283:902-907 (1998); Wuite et al., Nature 404:103-106 (2000); and Yin et al., Science 270:1653-1656(1995)). For example, researchers have been able to measure the sequence-dependence of the forces necessary to "unzip" double-stranded DNA (Voulgarakis, et al.., Nano Letters 6, 1483-1486 (2006)). In addition, optical tweezers<br>
were used to elucidate the mechanism whereby kinesin walks along a microtubule (Kuo. &amp; Sheetz, Science 260, 232 (1993) and Block, et al., Nature 348, 348-352 (1990)). Quite recently, researchers detected with single base-pair resolution the stepping action of the RNA polymerase along a molecule of DNA (Abbondanzieri, et al., Nature 438, 460-465 (2005)). At Nanobiosym, high-resolution optical tweezers have been utilized to experimentally demonstrate the role of various environmental factors on the dynamics of polymerases (Goel et al, Nature Nanotechnology review article in press).<br>
In all such studies, optical tweezers were utilized to directly measure the mechanical dynamics of a substrate-enzyme interaction. In these studies, the details of experimental setup and measurements taken are dependent on the biological function of the enzyme and/or the substrate involved. For example, the mechanical measurements of interest may include: the elasticity of substrate polymers, including stretching and relaxation dynamics; the time-dependent velocity of an enzyme that is "processing" a linear substrate, such as polymerase bound to a nucleic acid; the deformation of a substrate caused by enzymatic binding; and/or the efficiency or accuracy of substrate binding and processing. By integrating position-and/or force-sensing subsystems into an optical tweezers apparatus, all such measurements are possible.<br>
Novel methods and apparatuses for single molecule drug screening discovery and validation are disclosed herein. These methods and apparatuses allow a user to detect rapidly, at the single-molecule level, whether and how a drug candidate interferes with an enzyme-substrate interaction involved in a particular disease pathway. In particular, interactions between candidate drugs and a single target enzyme molecule can be observed. The methods and apparatuses described herein utilize single molecule manipulation technologies (e.g. optical or magnetic tweezers or traps) to directly detect, at the single-molecule level, whether a drug candidate can mechanically or chemically alter the enzyme-substrate interaction.<br>
In a preferred embodiment, the present methods and apparatuses can be utilized to quickly screen, test, and validate new drug candidates that modify, inhibit or otherwise interfere with polymerase enzymes such as DNA polymerase, RNA polymerase, and RNA reverse<br>
transcriptase, etc., and to better elucidate the mechanism whereby the polymerase/substrate interaction is inhibited.<br>
Normal enzymatic activity on a substrate produces a dynamic "mechanical signature." The term "mechanical signature" as used herein refers to the biomechanical trace of a single molecule of an enzyme as it interacts with it's substrate. The biomechanical trace can be measured using instruments, such as optical tweezers, that can detect displacements in the nm range. As described above, this dynamic mechanical signature can be determined by making a "mechanical measurement," for example by measuring changes in the elasticity of substrate polymers, including stretching and relaxation dynamics; the time-dependent velocity of an enzyme that is "processing" a linear substrate, such as polymerase bound to a nucleic acid; the deformation of a substrate caused by enzymatic binding; and/or the efficiency or accuracy of substrate binding and processing.<br>
The term "mechanical measurement" as used herein means a measurement of the mechanical dynamics of a substrate-enzyme interaction, wherein the mechanical measurement detects the mechanical signature of a single molecule of a target enzyme and/or a single molecule of a substrate of the target enzyme. "Making a mechanical measurement" includes, for example, measuring changes in the elasticity of substrate polymers, including stretching and relaxation dynamics; the time-dependent velocity of an enzyme that is "processing" a linear substrate, such as polymerase bound to a nucleic acid; the deformation of a substrate caused by enzymatic binding; and/or the efficiency or accuracy of substrate binding and processing.<br>
A preferred "mechanical measurement" is a measurement of the movement of a reverse transcriptase, DNA polymerase, or RNA polymerase enzyme along a polynucleotide (e.g. DNA or RNA) substrate. Thus, in particular embodiments of the method, discussed in more detail below, the real-time single molecule dynamics of a polymerase along a nucleic acid sequence is monitored in the presence and absence of drug candidates via optical trapping techniques. Other mechanical measurements of a polymerase/substrate interaction which are not explicitly described herein but are also measurable via single molecule detection techniques (e.g. optical tweezers) are also possible.<br>
A mechanical measurement can be made in the presence or absence of a drug candidate. A "baseline mechanical signature" is a mechanical signature that was determined via a mechanical measurement that was made in the absence of a drug candidate.<br>
The term "target" or "drug target" as used herein refers to a biomolecule that is involved in a disease pathway. Inhibiting or otherwise interfering with the activity of the target could be beneficial in treating and/or preventing the disease. The term "target enzyme" as used herein refers to an enzyme that is involved in a disease pathway. Typically a target enzyme is a key enzyme involved in a particular metabolic or signaling pathway that is specific to a disease condition or pathology, or to the infectivity or survival of a microbial pathogen. The "activity of a target enzyme" means the interaction of the target enzyme with a substrate of the target enzyme.<br>
Target enzymes suitable for the present invention include enzymes that bind to and interact with DNA and/ox RNA. Examples include polymerases, such as DNA polymerases, RNA polymerases, and reverse transcriptases; topoisomerases; gyrases; exoncucleases; and helicases. The target enzymes can be human enzymes, for example human enzymes involved in a disease pathway such as cancer. In another embodiment, the target enzymes can be viral or bacterial enzymes, such as viral or bacterial enzymes involved in viral- and/or bacterial-mediated diseases. Other microbial enzymes are also contemplated as target enzymes suitable for the present invention.<br>
A preferred target enzyme is a polymerase enzyme such as a DNA polymerase, an RNA polymerase, or a reverse transcriptase. Polymerases involved in cancer pathways, especially human DNA polymerases involved in human cancer pathways, are particularly preferred. Polymerases involved in viral-mediated disease pathways, especially viral reverse transcriptases involved in viral-mediated disease pathways in humans (e.g. hepadnaviral reverse transcriptases such as Hepatitis B reverse transcriptase, and retroviral reverse transcriptases such as HIV-1 reverse transcriptase) are also particularly preferred.<br>
The present invention is also suitable for screening drug candidates that may interact with non-enzyme targets involved in disease pathways. Representative examples of non-enzyme<br>
targets are microtubules and ribozymes (such as ribosomes). Ribosomes, in particular, use RNA as a template to build polypeptide chains, and thus can be thought of as a giant enzymes.<br>
The term "substrate of the target enzyme" (or "enzyme substrate" or "substrate") as used herein refers to a molecule upon which a target enzyme acts. Enzymes catalyze chemical reactions involving one or more substrates. Enzyme substrates are well known in the art.<br>
Preferred substrates include polymerase substrates, such as polynucleotides (e.g. DNA and RNA). Polynucleotide substrates are also referred to herein as polynucleotide or nucleic acid "templates." Polynucleotide substrates can be double-stranded or single-stranded DNA or RNA sequences.<br>
The term "drug candidate" or "candidate" as used herein refers to a compound that may inhibit or otherwise interfere with the activity of a target, particularly a target enzyme. Drug candidates include compounds found in nature, compounds synthesized by combinatorial chemistry approaches, and compounds created via rational drug design. Examples of drug candidates include compounds that interact with or may interact with polynucleotides (e.g. DNA and/or RNA), and/or compounds that interfere with or may interfere with the activity of enzymes that interact with polynucleotides. Such conipounds can be known or potential DNA modifying agents, including DNA damaging agents (e.g. intercalating agents that interfere with the structure of nucleic acids); DNA bending agents; mismatch binding proteins; and/or alkylating agents.<br>
In another embodiment, a drug candidate can be a compound that interacts with or may interact with a non-enzyme target involved in disease pathway. Examples include compounds that interact with or may interact with microtubules and/or ribosomes.<br>
Some exemplary classes of drug candidates that can be probed via the present methods are described next. First, several classes of antibiotic drugs are suitable for interrogation by the present methods, including drugs that inhibit or otherwise interfere with the activity of bacterial polymerases such as bacterial DNA polymerase, bacterial RNA polymerase (e.g.,<br>
rifampin), and/or bacterial reverse-transcriptase. Second, the present methods could also be utilized to quickly screen, test, and validate new antiviral drug candidates that inhibit or somehow interfere with viral polymerases, especially viral reverse transcriptases (e.g. efavirenz and nevirapine), and to better elucidate the mechanism whereby the RNA-reverse transcriptase interaction is inhibited. Third, several classes of anti-cancer drugs are also suitable for interrogation by our methods. These include drugs that inhibit or otherwise interfere with the activity of DNA polymerases, RNA polymerases, topoisomerases, ribosomes, and/or microtubules (e.g. microtubule antagonists such as vincristine and taxol). Additionally, entirely new drug mechanisms, heretofore unknown, could be discovered and elucidated by our approach described herein.<br>
The term "single molecule detection apparatus'* (or "single molecule detection device") as used herein refers to an apparatus that can be used to make a mechanical measurement of an enzyme-substrate interaction at the single-molecule level. Single molecule detection apparatuses suitable for the present invention include apparatuses used for magnetic or optical trapping (e.g. optical tweezers), high-resolution fluorescent imaging coupled with quantum-dot labeling, and atomic force microscopy. Other apparatuses and variants of the apparatuses disclosed herein could be readily envisioned.<br>
A preferred single molecule detection apparatus is an apparatus comprising an optical trap or tweezers.<br>
Instead of screening for a downstream effect of enzymatic interference, the methods described herein, dubbed NANO-VALID™, determine via direct, single-molecule observation if a drug candidate alters the normal "mechanical signature" of the target enzyme. Further, the methods described herein enable the analysis and determination of the mechanism(s) by which the enzymatic dynamics are affected. The methods utilize single-molecule manipulation, detection and analysis apparatuses, to determine if and how the drug candidate modulates this "mechanical signature" in a way indicative of enzymatic inhibition.<br>
The first step in the NANO-VALID™ screening process is to choose a mechanical signature that captures the functionality of the target enzyme and is reliably measurable at the single<br>
molecule level. We discuss below in greater detail exemplary signatures appropriate for specific classes of enzymes that maybe extracted with single-molecule detection and manipulation technologies. The second step in the NANO-VALID™ method is to experimentally determine the normal, baseline mechanical signature of the target enzyme in the absence of any inhibitor, including a drug candidate. As discussed above, the baseline mechanical signature is determined by making a mechanical measurement using a single-molecule detection apparatus. Due to the single-molecule nature of the approach, this may involve taking ensemble averages of several experiments. We discuss below exemplary technologies and apparatuses that enable single-molecule mechanical measurement of the dynamics of several classes of enzymes.<br>
To screen each drug candidate, the chosen mechanical measurement of the target enzyme is made with the same experimental techniques and apparatus and under the same conditions as those used in determining the baseline mechanical signature, except the drug candidate is present in the single-molecule assay. Depending on the nature of the target enzyme, it may be desirable to first incubate the target, the target's substrate(s), or both, with the drug candidate for some controlled period of time prior to conducting this measurement.<br>
Next, extensive signal processing is conducted to compare the candidate-specific mechanical signature with the baseline mechanical signature. We discuss in detail below exemplary variants of this analysis appropriate for various classes of enzymes. If no significant deviation from the signature is detected, the candidate is rejected and not subjected to further screening. This feature drastically reduces the time and cost associated with drug candidate screening, testing, and validation, because unsuccessful drug candidates can be eliminated from testing much earlier in the process, much before the onset of expensive clinical trials, This leads to much more specific drug candidates being chosen earlier on, such that only those candidates that are more likely to be successful make it to clinical trials. This increased selectivity criterion early on in the drug discovery process significantly reduce the cost and time of single molecule drug discovery processes as compared to conventional drug validation and discovery approaches.<br>
However, if a significant deviation from the baseline mechanical signature is detected, further analysis and processing is conducted to identify potential mechanism(s) of interference. We discuss in detail below the. particular embodiments of this secondary analysis appropriate for various classes of enzymes. If the identified mechanism(s) are riot desirable for the disease pathway, the candidate is rejected and not subjected to further screening. Otherwise, the candidate is considered for further screening and validation.<br>
This NANO-VALID™ process is summarized in FIGURE 1.<br>
NANO-VALID™ for Polymerase Targets<br>
We describe here a preferred embodiment of the NANO-V ALID™ method that is appropriate for screening drug candidates that target polymerase enzymes. The embodiment is summarized in Figure 3. In this method embodiment, the time-dependent position of the polymerase enzyme along the nucleic acid template is always chosen at step 1 to be subsequently measured in steps 2 and 3a.<br>
Polymerases process a nucleic acid template in a primarily linear fashion. For this reason, the mechanical signature common to all polymerases is the linear progression of the polymerase along this substrate. When the nucleic acid template is aligned with a fixed line and held at constant tension, then the progression of the polymerase correlates simply to the position of the polymerase along this fixed line as a function of time.<br>
In the absence of an inhibitor, a polymerase normally binds at an initiation point, proceeds with a relatively constant velocity and then terminates polymerization. Some minor stochastic behavior typically occurs during the course of normal polymerization. These anomalies may include short pauses on the order of milli-seconds or less, short reversals of direction on the order of less than 100 bases (20-55 nanometers, depending on the experimental conditions), and variations in the polymerization velocity on the order of a few hundred base pairs-per-second.<br>
Shown in Figure 6 are data demonstrating the capability to observe the dynamics of the DNA polymerase motor with an optical tweezers apparatus as it moves forwards (polymerization)<br>
and backwards (exonuclease activity) along a DNA template. A velocity of about 0.5 microns/min corresponds to about 25 bases-pairs polymerized/second. Without wishing to be bound by theory, it is surmised that polymerase inhibitors impact DNA polymerases (DNAp) through one of several mechanisms, each with a distinct "mechanical signature" in the plot of template length versus time, and/or the velocity of polymerization versus time. The plots can be analyzed to identify the mechanisms. Additionally, there may be other entirely new mechanisms, hereto unforeseen that could also be discovered via our methods described herein.<br>
Figure 2 illustrates what polymerization might look like in the presence or absence of a drug candidate. Note that for the figure shown, the initiation point is a y = 200 um and the polymerase is presumed to be moving from right (y = 200 urn) to left (y = 0 um) in about 200 minutes. When a drug candidate interferes with a polymerase target, the dynamics of the polymerase along a nucleic acid template are clearly affected in one of several ways, which may include: modulating the rate of polymerization, inactivating the enzyme from binding or polymerizing, altering the processivity of the enzyme, altering the binding affinity of the enzyme to the template, or altering the sequence-dependent fidelity of the enzyme. Each of these scenarios produces a mechanical signature distinct from normal polymerization. Thus this preferred embodiment of the NANO- VALID™ method for polymerase targets uses the time-dependent position of the enzyme as the mechanical signature in steps 2 and 3a (see Figure 3, NANO-VALID™ Drug Screening and Validation process map). Figure 2b shows an exemplary mechanical signature that would indicate that a drug candidate has slowed the polymerization process. Note that the average slope'of the plot, which corresponds to the average polymerization velocity, has lessened. Some DNA polymerase inhibitors may work, for example, by just slowing the overall speed of DNA replication. This trace would indicate a polymerase inhibitor drug candidate that worked by this mechanism. Figure 2c shows an exemplary mechanical signature that indicates that a drug candidate has induced premature termination, wherein the drug candidate has derailed or stalled the polymerization process before its natural completion point. Some DNA polymerase inhibitor candidates may work by such a mechanism. This trace would indicate a polymerase inhibitor drug candidate that worked by this mechanism. Figure 2d shows an exemplary mechanical signature that indicates that initiation of the polymerase is inhibited by the drug candidate. This trace<br>
would indicate a polymerase inhibitor drug candidate that worked by this mechanism. Figure 2e shows an exemplary mechanical signature that indicates the drug candidate induces the polymerase enzyme to operate in an exonucleolysis mode, in which it excises bases rather than polymerizing base-pairs. This signature would likely only occur in polymerases with an active exonucleolysis, or "proof-reading" site. This trace would indicate a polymerase inhibitor drug candidate that worked by this mechanism.<br>
To avoid false screening results, the mechanical signature used in this embodiment of the NANO-VALID™ method should be on a length- and time- scale that far exceeds the scale of the stochastic events of normal polymerization. Typically, for most polymerases, the mechanical signature should be taken over a time window that will allow the target, under the chosen experimental conditions, to traverse a nucleic acid template of at least 5000 bases or base-pairs.<br>
In a preferred embodiment, we utilize a software algorithm to implement step 3b so that we may determine with high accuracy in step 3c if a drug candidate should be eliminated.<br>
The stochastic nature of normal polymerization requires that we extract the salient features of the polymerase dynamics in steps 2 and 3 a, rather than using or comparing raw data traces. The nature of the target enzyme, as well as the desired interference mechanisms will dictate which salient features are extracted and analyzed. These features may include: total time the motor is paused, terminal polymerization velocity, efficiency of termination, average velocity. Typically, for all polymerases, a low-pass filter with a cutoff frequency of 100-1000 Hz will also be applied to the raw signal to filter out the effects of the stochastic fluctuations in the dynamics of the enzyme.<br>
NANO-VAJLID™ for DNA Polymerase Targets using Template Length Measurements Here we describe a variant embodiment of the previously described NANO-VALID™ method that is appropriate for DNA. polymerase targets that act on a single-stranded DNA template. Each base in the single-stranded DNA template has a length of approximately 0.7nm under standard environmental conditions and a constant template tension of approximately 0-1 pN. Each unpaired base on the template is converted by the DNA<br>
polymerase into a base-pair with a length of approximately 0.34nm under the same environmental and constant-tension conditions. Hence, under isotension conditions, as the DNA polymerase replicates the linear template, converting single-stranded DNA into double-stranded DNA, the difference in elasticity of the two states causes a shortening of the DNA strand by approximately 0.36nm per base-pair that is polymerized at a given force.   Thus, tracking the length of the DNA strand during polymerization under constant DNA template tension is analogous to tracking the position of a polymerase along a fixed template. Therefore, in steps 2 and 3 a of this method variant, the mechanical signature directly obtained by the single-molecule measurement apparatus is the length of the DNA template over time. The signal processing methods described previously for analyzing a polymerase's position and velocity in steps 3b and 3d can again be utilized in this variant method.<br>
Apparatus for Performing NANO-VALID™ Screening of Drug Candidates We describe here an apparatus aspect of this invention for performing the NANO-VALID™ method that comprises a single-molecule measurement system interfaced to a personal computer via a rapid port (e.g., a USB port) that allows near real-time data-acquisition and control of the single-molecule apparatus system. See Figure 9a and 9b. Via software drivers, this apparatus is controlled and interrogated by a custom software program that has a graphical user interface (GUI). In this embodiment, the single-molecule measurement system would have a receptacle for loading and addressing several individual samples on a single plate, for example a 96-weIl microplate. The measurement system would integrate temperature controls to ensure reliable and reproducible environmental conditions. The measurement system could comprise one or more of the following: atomic force microscope, scanning electronic microscope, etc, etc.<br>
To utilize the apparatus, the user would load the plate into the single-molecule measurement. Via the GUI, they would initialize the single-molecule measurement system, including any necessary calibrations. The user would then select the mechanical signature, and any control parameters regarding the signature (for example the length of time that the signature is measured). Next, via the GUI, they would direct the single-molecule measurement system to interrogate the control samples to obtain the target enzyme's baseline mechanical signature. The precise execution of this measurement might require some manual control and input<br>
from the human user via keyboard, joystick or other computer input device. The custom software program would then acquire this signature, and via signal-processing routines, extract and store salient features of the signal.   The GUI would display the salient features, including perhaps the raw mechanical signature traces.<br>
Next, the GUI would allow the user to traverse the remainder of the plate, to acquire a mechanical signal from each candidate; this may entail several individual measurements. As before, this signature acquisition step might require direct user input. At each step, the raw mechanical signature would be acquired, salient features of the mechanical signature would be extracted via signal processing, stored in the computer memory and displayed to the user, and compared to the reference baseline signal.<br>
Apparatuses for Performing NANO-VALID™ Screening of Drug Candidates Targeting a Polymerase<br>
To perform the NANO-VALID™ method for polymerase targets, it is necessary that the single-molecule measurement apparatus accurately capture the real-time dynamics of the polymerase along a nucleic acid template. Several technologies, could be utilized to accomplish this functionality. For example, high-resolution fluorescent imaging coupled with quantum-dot labeling of the polymerase could be utilized. Alternatively, atomic force microscopy could be utilized to measure these polymerase dynamics (is that true). Magnetic or optical "tweezers" or "traps" could also be utilized for pico-Newton control of the template tension and measurements of nanoscale displacements of the polymerase along the DNA template. Other variants could be readily envisioned.<br>
Optical Tweezers-Based Apparatuses for Performing NANO-VALID™ Screening of Drug Candidates Targeting a Polymerase<br>
In a preferred embodiment, the NANO-VALID™ apparatus for polymerase targets would utilize an optical tweezers-based single-molecule measurement subsystem. An optical tweezers traps particles with forces generated by optical intensity gradients. Optically generated forces strong enough to form a three-dimensional trap can be obtained by bringing a laser beam with an appropriately shaped wavefront to a tight focus with a high numerical aperture lens.   Optical tweezers techniques have been used extensively for single-molecule<br>
studies of the polymer properties of DNA and the force-dependent kinetics of biomolecular motors, including polymerase enzymes. As a result of these extensive biophysical studies, the experimental protocols for using optical tweezers to track polymerase enzymes along an isotension nucleic acid template are quite mature and well-known in the art (Block, et al., Nature 348, 348-352 (1990)).<br>
To track the polymerase along the template, one end of the nucleic acid template is effectively immobilized, while the polymerase molecule is attached to a bead trapped in an electronically-steerable optical tweezers apparatus. As the polymerization proceeds, the optical trap is designed to automatically move in order to maintain constant force on the polymerase's bead.   The dynamic position of the trap under these isotension conditions then correlates then to the dynamics of the polymerase along the template during polymerization. A preferred embodiment of this "Force-Feedback Optical Trapping Subsystem" is described in extensive detail below. Several other variants could be readily imagined.<br>
Figure 4 shows experimental variants of this embodiment; they vary in their method of nucleic acid immobilization. In figure 4-C the nucleic acid template is attached to a dielectric bead that is held by a second, immoblized optical trap, shown at left, that exerts a strong trapping force that is much greater than the force exerted by the polymerase on the nucleic acid template. Figure 4-D demonstrates how the nucleic acid can be immobilized to a rigid surface, such as a cover slip or microwell plate, via complementary functionalization of the nucleic acid template and the rigid surface.<br>
Methods for attaching nucleic acids and polymerases to beads or other surfaces are well known in the art; some examples given below illustrate exemplary methods for accomplishing this task. One such method biotinylates the nucleic acid or polymerase via standard methods so that it maybe attached to streptavidin-coated microbeads (e.g., Bang's Laboratories) or streptavidin-coated coverslips (e.g., Xenopore Corporation).<br>
Optical Tweezers-Based Apparatuses for Performing NANO-VALID™ Screening via Single-Molecule Measurement of Nucleic Acid Template Length<br>
The previously described preferred apparatus embodiment can be used to perform the variant NANO-VALID™ method used for DNA polymerases along a single-stranded DNA template. As previously discussed, this method variant measures the length of the DNA template, instead of directly tracking the polymerase motion. To conduct this method variant, one end of the DNA must be immobilized as hefore; however, the other end of the DNA template is now attached to a dielectric bead that is interrogated by the steerable, constant-force optical trap. The enzyme remains free in solution.<br>
In this variant method, instead of the trap moving in response to the force of the polymerase enzyme moving along the template, it will move to maintain constant force on the DNA template, even as the template molecule shortens, as a result of polymerization. As discussed previously, this is a nearly analogous measurement. While the method variant requires slight differences in the sample preparation, this method variant requires no difference in the apparatus nor controlling software. Figures 4 a-b summarize the experimental variants for performing this method variant.<br>
Force-Feedback Optical Trapping Subsystem<br>
Figure 5A illustrates an exemplary design of an optical trapping system that maintains a constant force on a trapped bead, even as the bead experiences other forces due to the enzymatic activity of the polymerase. An IR laser source (A) is focused via a series of lenses and mirrors (B) to a position (xo, yo) in the sample plane. This focus position, (xo, yo), is established by the deflection angle of a two-axis acousto-optic deflector (AOD) (L). The Gaussian beam profile at the sample slide (C) traps the bead that is attached to either a polymerase (not shown) or nucleic acid template (shown).<br>
A quadrant photodiode or QPD, (D) detects the interference pattern of the TR signal scattering off the bead, and outputs the x and y perturbations of this signal. These signals are amplified (E), obtained by a data acquistion card (DAQ), interfaced to a personal computer (G). This data is subsequently processed in a data acquistion and analysis program (programmed, for example, in Labview® from National Instruments), to determine the position of the bead relative to the center of the optical trap, with near-nanometer resolution and to measure the trapping force on the bead in picoNewtons.   (See also Figure 5B)<br>
Direct images of the bead are captured by a sensitive CCD camera (e.g., Cooke's PixelFly) that is coupled to the light microscope (F). Custom data analysis computer programs provide nm scale position detection of the bead; an image analysis program provides an image of the sample plane, including the trapped bead.<br>
With the force feedback system operating (L, K), the force exerted by the laser on the bead remains constant even in the presence of the enzymatic forces that act on the bead, shifting its position within the trap. This is accomplished via a software program that determines the instantaneous position of the bead with respect to the center of the laser trap and compares this to a reference position (corresponding to a given force on the trapped bead). The difference between this current position and the desired bead position is converted into a two-channel frequency signal (Afx, Afy ) that the radio frequency (RF) driver (K) inputs into the two-axis AOD crystal. This radio frequency creates a standing wave in the crystal which then diffracts the incident laser beam. The position of the first order diffracted beam changes with the radio frequency.<br>
Thus the laser beam can be precisely and rapidly steered by controlling the RF input into the AOD crystal. The data acquisition program calculates the new feedback frequency that should be generated by the RF driver (K) to deflect the beam enough to compensate for ezymatically-driven shift in the bead's position. A record of this output frequency can be post-processed to output the time evolution of the trap's position over the polymerization time.<br>
The unique features of NANO-VALID™ drug validation process as compared to conventional approaches are that the NANO-VALID™ system enables high resolution realtime tracking of how the drug candidate modifies or interferes the polymerase movement along the nucleic acid template.' This system is furthermore, highly integrated and automated making it easily scalable for high throughput drug screening applications.<br>
The methods and apparatuses described herein also provide numerous cost-related advantages over conventional drug screening and validation techniques. As illustrated in Figures 8-10,<br>
the present methods and apparatus enable rapid screening, providing for an approximately 115-fold reduction in labor time. In addition, the single-molecule approach of the present invention allows for an approximately 130-fold reduction in reagent volumes, and the reduced instrumentation cycle time leads lo an approximately 10- to 50-fold reduction in machine time. Thus, overall, the present invention provides an approximately 20- to 100-fold improvement in overall cost.<br>
EXAMPLES<br>
Method and Apparatus for Detecting Inhibition of DNA Replication using Dynamics of Template Length<br>
In this example, we illustrate a method and apparatus for detecting the inhibition of DNA replication via single-molecule measurement of the DNA template length. In this example, we seek to screen for drugs that inhibit DNA polymerization through one of two mechanisms: interference of polymerase binding, or ultra-low processivity. Figure 2-A shows an expected normal signature, while 2-B shows a result expected from an effective drug candidate. The apparatus used for single-molecule measurements comprises a force-feedback optical trapping subsystem, as previously described. The experimental setup for this mechanical signature measurement is shown in figures 5a and 5b.<br>
The interior surface of an optically transparent multi-well microplate is coated with streptavidin according to standard methods. A sample of biotinylated ssDNA template with a length exceeding 5kb is prepared (Rothenberg- add full cite) and suspended in a buffered solution. A DNA primer is also designed that can initiate polymerization. A suspension of streptavidin coated 0.5-1 micron beads (available from Bang's Laboratories) is prepared and distributed among the wells; subsequently a solution of the ssDNA template is also distributed among the wells. The plate is incubated for approximately 30 minutes for the streptavidin-biotin bonding to occur such that a sufficient population of DNA tethers are formed, which have one end of the nucleic acid attached to the microplate, the other to a streptavidin-coated bead. The drug candidates are distributed among the wells as follows: each well contains at most one candidate; each candidate is added to N wells to provide redundancy. A set of C wells do not have any candidate added so that they may be used as control samples.<br>
The optical tweezers apparatus is initiated and calibrated from the graphical user interface. The microweli plate is loaded into the optical tweezers apparatus, such that its bottom is flush with the focal plane of the optical trap. The GUI is used to set the interrogation time and the trapping force to a desired value from 0-35pN. The GUI also is used to identify which candidate (if any) is present in each microweli.<br>
Input is provided to the GUI to drive the microweli plate so that the optical tweezers beam is interrogating the first control microweli. Using a joystick and the CCD image of the sample plane, a suitable bead is trapped manually. The force-feedback system is engaged and immediately thereafter, a buffered suspension of DNA primer, dNTP mixture and the target DNA polymerase is added to this single microweli. This could likewise be applied to RNA polymerase and Reverse transcriptase. The force-feedback system is allowed to run to track the position of the trap during the chosen time interval; these results are stored in memory so that they are addressable and identifiable both by the well address and as a control result. This process is repeated for the remainder of the control wells. From the GUI, a program is run to analyze the multiple control samples in order to extract several salient features common to the polymerase target. First, a selection of different low-pass filters, with cutoff frequencies in the range of 100-1000 Hz, are applied to these controls, and the results displayed so that the user may select the filter with a cutoff frequency coc that gives the highest level of signal smoothing while maintaining the integrity of the overall dynamics of the target polymerase.<br>
Once filtered, a numerical differentiation scheme determines the instantaneous velocity at each point. Next, nearest-neighbor averaging of the velocity is determined for different window-sizes, ranging from the entire acquisition window (i.e., t == [0:T]), down to a window corresponding roughly to 100 base-pairs (windows of approximately 1 second length, depending on the velocity of the polymerase).<br>
Input is provided to the GUI to drive the microweli plate so that the optical tweezers beam is interrogating the first non-control microweli. Using a joystick and the CCD image of the sample plane, a suitable bead is trapped manually. The force-feedback system is engaged and<br>
immediately thereafter, a buffered suspension of DNA primer, dNTP mixture and the target DNA polymerase is added to this single microwell. The force-feedback system continues to track the position of the trap during the chosen time interval; these results are stored in memory so that they are addressable and identifiable both by the well address, as well as the drug candidate present. This process is repeated for the remainder of the control wells.<br>
All of the obtainedjresults are filtered with a low-pass filter with cutoff frequency oc. With the GUI, the results are numerically differentiated, and nearest-neighbor velocity plots are calculated for the same time-scales as for the controls. Each drug candidate is considered in turn.  Substantially non-zero velocity over any time scale, for any one of the N samples would indicate that the drug candidate had not interfered reliably or efficiently with the DNA replication process. For that reason, those candidates showing a substantially non-zero velocity (e.g., 1% of the average velocity of all of the controls) over any time scale, for any of the N samples, are rejected.<br>
Method and Device for Detecting Early Termination of RNA Polymerase Targets using Dynamics of Polymerase Along DNA Strand<br>
In this example, we illustrate a method and apparatus for detecting the early termination of RNA transcription by an RNA polymerase. In this example we are screening for drug candidates that induce early termination of RNA polymerization after initiation at a specific site. Figure 2-A shows an expected normal signature, while 2-B shows a result expected from an effective drug candidate. The apparatus used for single-molecule measurements is a dual-beam optical tweezers apparatus that comprises both a force-feedback optical trapping subsystem and a higher-power steerable optical trap.<br>
A sample of double stranded (ds) DNA is prepared via standard methods to comprise both a stalled transcription complex comprising a biotin tag, as well as biotin tag on the downstream end of the DNA (Neuman, K.C., et. Al, Cell, 115: 437-447, 2003). The transcription factor tag is subsequently attached to streptavidin-coated 1 micron diameter dielectric bead (e.g., Bang's Laboratories), and the DNA tag is attached to a streptavidin-coated .5 micron diameter dielectric bead (e.g., Bang's Laboratories) . This forms a sample of DNA "dumbbells," that have a bead "handle" on each end: one attached to the end of the DNA, the<br>
other to the RNA polymerase. The drug candidates are distributed among the wells of an optically transparent microwell plate as follows: each well contains at most one candidate; each candidate is added to N wells to provide redundancy. A set of C wells do not have any candidate added so that they may be used as control samples.<br>
The optical tweezers apparatus is initiated and calibrated from the graphical user interface. The microwell plate is loaded into the optical tweezers apparatus, such that its bottom is flush with the focal plane of the optical trap. The GUI is used to set the interrogation time, T, and the trapping force of the force-feedback beam to a desired value from 0-3 5pN. The GUI also is used to identify which candidate (if any) is present in each microwell, so that the microwells may be subsequently addressed by both their position and contents.<br>
Input is provided to the GUI to drive the microwell plate so that the optical tweezers beam is interrogating the first control microwell. Using a joystick and the CCD image of the sample plane, a dumbbell is trapped manually by trapping the larger bead in the force-feedback optical trap, and the smaller bead in the strong secondary trap. The force-feedback system is engaged and immediately thereafter, a buffered suspension of RNA dNTP mixture is added to this single microwell. The force-feedback system continues to track the position of the trap during the chosen time interval; these results are stored in memory so that they are addressable and identifiable both by the well address and as a control result. This process is repeated for the remainder of the control wells. From the GUI, a program is run that analyzes the multiple control samples in order to extract several salient features common to the polymerase target. First, a selection of different low-pass filters, with cutoff frequencies in the range of 100-1000 Hz, are applied to these controls, and the results displayed so that the user may select the filter with a cutoff frequency coc that gives die highest level of signal smoothing while maintaining the integrity of the overall dynamics of the target polymerase.<br>
Once filtered, a numerical differentiation scheme determines the instantaneous velocity at each point. Next, nearest-neighbor averaging of the velocity is determined for different window-sizes, ranging from the entire acquisition window (i.e., t = [0:T]), down to a window corresponding roughly to 100 base-pairs (windows of approximately 1 second length, depending on the average velocity of the polymerase). The user then selects a timepoint, To<br>
Input is provided to the GUI to drive the microwell plate so that the optical tweezers beam is interrogating the first non-control microwell. Using a joystick and the CCD image of the sample plane, a suitable bead is trapped manually. The force-feedback system is engaged and immediately thereafter, a buffered suspension of RNA dNTP mixture is added to this single microwell. The force-feedback system is allowed to run to track the position of the trap during the chosen time interval; these results are stored in memory so that they are addressable and identifiable both by the well address, as well as the drug candidate present. This process is repeated for the remainder of the control wells.<br>
All of the obtained results are filtered with a low-pass filter with cutoff frequency coc. With the GUI, the results are numerically differentiated, and nearest-neighbor velocity plots are calculated for the same time-scales as the controls. Each drug candidate is considered in turn. If a drug candidate is reliably interfering with transcription in the manner desired, all of the related N samples should show substantially non-zero velocity after the time To. For that reason, those candidates showing a substantially non-zero velocity for any of the N samples (e.g., 1 % of the average velocity of all of the controls) for any time window after To are rejected.<br>
While the invention has been described in detail with reference to particular embodiments thereof, it will be apparent to one skilled in the art that various changes can be made, and equivalents employed, without departing from the scope of the invention.<br>
The publications discussed herein are provided solely for their disclosure prior to the filing date of the present application. Nothing herein is to be construed as an admission that the present invention is not entitled to antedate such publication by virtue of prior invention. Further, the dates of publication provided may be different from the actual publication dates, which may need to be independently confirmed.<br>
All publications cited herein are hereby incorporated by reference in their entirety.<br><br><br><br><br><br><br><br><br>
CLAIMS<br>
1.	A method for screening drug candidates, wherein the method comprises:<br>
(a)	contacting a target enzyme with a substrate of the target enzyme;<br>
(b)	determining a baseline mechanical signature of the target enzyme in the presence of the substrate of the target enzyme by using a single-molecule detection apparatus to make a mechanical measurement;<br>
(c)	contacting the target enzyme and the substrate of the target enzyme withNone or more drug candidates;<br>
(d)	determining a mechanical signature of the target enzyme in "the presence of the substrate of the target enzyme and one or more drug candidates by using a single-molecule detection apparatus to make a mechanical measurement; and<br>
(e)	comparing the baseline mechanical signature of step (b) with the mechanical signature of step (d); wherein:<br>
the baseline mechanical signature of step (b) and the mechanical signature of step (d) are determined using the same single-molecule detection apparatus and the same mechanical measurement.<br>
2.	The method of claim 1, wherein the mechanical measurement is selected from the group consisting of: a measurement of the time-dependent velocity of the target enzyme along the substrate of the target enzyme, a measurement of the time-dependent change in the length of the substrate of the target enzyme, a measurement of a change in the elasticity of the substrate of the target enzyme, and a measurement of the efficiency or the accuracy of substrate binding and processing by the target enzyme.<br>
3.	The method of claim 1, wherein the single-molecule detection apparatus is selected from the group consisting of: an apparatus that utilizes magnetic or optical trapping, an apparatus that utilizes high-resolution fluorescent imaging coupled with quantum-dot labeling, and an apparatus that utilizes atomic force microscopy.<br>
4.	The method of claim 1, wherein the target enzyme is a polymerase.<br>
5.	The method of claim 4, wherein the polymerase is selected from the group consisting of: a DNA polymerase, an RNA polymerase, and a reverse transcriptase.<br>
6.	The method of claim 4, wherein the polymerase is a human polymerase.<br>
7.	The method of claim 4, wherein the polymerase is a viral polymerase.<br>
8.	The method of claim 7, wherein the viral polymerase is a viral reverse transcriptase.<br>
9.	The method of claim8, wherein the viral reverse transcriptase is an HTV-1 reverse transcriptase.<br>
10.	The method of claim4, wherein the polymerase is a bacterial polymerase.<br>
11.	The method of claim 1, wherein the target enzyme is a polymerase, the substrate of the target enzyme is a polynucleotide, and the molecular measurement is a measurement of the movement of the polymerase along the polynucleotide substrate or a measurement is a measurement of the time dependent change in the length of the polynucleotide substrate during polymerization.<br><br>
12.	The method of claim 11, wherein the molecular measurement is made using a single-molecule detection apparatus that utilizes magnetic or optical trapping.<br>
13.	The method of claim 1, wherein the method provides detailed enzymatic dynamics data which precisely characterizes the inhibition and/or interference mechanism of the drug candidate tested.<br>
14.	A method for screening drug substantially as herein described with reference<br>
to the   accompanying specification, example and drawings.<br></td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTQ2Ni1kZWxucC0yMDA4LTEtQ29ycmVzcG9uZGVuY2UgT3RoZXJzLSgxNS0wMS0yMDE0KS5wZGY=" target="_blank" style="word-wrap:break-word;">9466-delnp-2008-1-Correspondence Others-(15-01-2014).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTQ2Ni1kZWxucC0yMDA4LTEtRm9ybS0zLSgxNS0wMS0yMDE0KS5wZGY=" target="_blank" style="word-wrap:break-word;">9466-delnp-2008-1-Form-3-(15-01-2014).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTQ2Ni1kZWxucC0yMDA4LTEtUGV0aXRpb24tMTM3LSgxNS0wMS0yMDE0KS5wZGY=" target="_blank" style="word-wrap:break-word;">9466-delnp-2008-1-Petition-137-(15-01-2014).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTQ2Ni1kZWxucC0yMDA4LUFic3RyYWN0LSgxNS0wMS0yMDE0KS5wZGY=" target="_blank" style="word-wrap:break-word;">9466-delnp-2008-Abstract-(15-01-2014).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTQ2Ni1kZWxucC0yMDA4LWFic3RyYWN0LnBkZg==" target="_blank" style="word-wrap:break-word;">9466-delnp-2008-abstract.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTQ2Ni1kZWxucC0yMDA4LUFzc2lnbm1lbnQtKDE1LTAxLTIwMTQpLnBkZg==" target="_blank" style="word-wrap:break-word;">9466-delnp-2008-Assignment-(15-01-2014).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTQ2Ni1kZWxucC0yMDA4LWFzc2lnbm1lbnQucGRm" target="_blank" style="word-wrap:break-word;">9466-delnp-2008-assignment.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTQ2Ni1kZWxucC0yMDA4LUNsYWltcy0oMTUtMDEtMjAxNCkucGRm" target="_blank" style="word-wrap:break-word;">9466-delnp-2008-Claims-(15-01-2014).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTQ2Ni1kZWxucC0yMDA4LWNsYWltcy5wZGY=" target="_blank" style="word-wrap:break-word;">9466-delnp-2008-claims.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTQ2Ni1kZWxucC0yMDA4LUNvcnJlc3BvbmRlbmNlIE90aGVycy0oMTUtMDEtMjAxNCkucGRm" target="_blank" style="word-wrap:break-word;">9466-delnp-2008-Correspondence Others-(15-01-2014).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTQ2Ni1kZWxucC0yMDA4LUNvcnJlc3BvbmRlbmNlIE90aGVycy0oMjQtMTItMjAxMykucGRm" target="_blank" style="word-wrap:break-word;">9466-delnp-2008-Correspondence Others-(24-12-2013).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTQ2Ni1kZWxucC0yMDA4LUNvcnJlc3BvbmRlbmNlLU90aGVycy0oMTgtMDYtMjAxMykucGRm" target="_blank" style="word-wrap:break-word;">9466-delnp-2008-Correspondence-Others-(18-06-2013).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTQ2Ni1kZWxucC0yMDA4LWNvcnJlc3BvbmRlbmNlLW90aGVycy5wZGY=" target="_blank" style="word-wrap:break-word;">9466-delnp-2008-correspondence-others.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTQ2Ni1kZWxucC0yMDA4LWRlc2NyaXB0aW9uIChjb21wbGV0ZSkucGRm" target="_blank" style="word-wrap:break-word;">9466-delnp-2008-description (complete).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTQ2Ni1kZWxucC0yMDA4LWRyYXdpbmdzLnBkZg==" target="_blank" style="word-wrap:break-word;">9466-delnp-2008-drawings.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTQ2Ni1kZWxucC0yMDA4LWZvcm0tMS5wZGY=" target="_blank" style="word-wrap:break-word;">9466-delnp-2008-form-1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTQ2Ni1kZWxucC0yMDA4LWZvcm0tMi5wZGY=" target="_blank" style="word-wrap:break-word;">9466-delnp-2008-form-2.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTQ2Ni1kZWxucC0yMDA4LUZvcm0tMy0oMTgtMDYtMjAxMykucGRm" target="_blank" style="word-wrap:break-word;">9466-delnp-2008-Form-3-(18-06-2013).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTQ2Ni1kZWxucC0yMDA4LUZvcm0tMy0oMjQtMTItMjAxMykucGRm" target="_blank" style="word-wrap:break-word;">9466-delnp-2008-Form-3-(24-12-2013).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTQ2Ni1kZWxucC0yMDA4LWZvcm0tMy5wZGY=" target="_blank" style="word-wrap:break-word;">9466-delnp-2008-form-3.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTQ2Ni1kZWxucC0yMDA4LWZvcm0tNS5wZGY=" target="_blank" style="word-wrap:break-word;">9466-delnp-2008-form-5.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTQ2Ni1kZWxucC0yMDA4LXBjdC0yMTAucGRm" target="_blank" style="word-wrap:break-word;">9466-delnp-2008-pct-210.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTQ2Ni1kZWxucC0yMDA4LXBjdC0zMDQucGRm" target="_blank" style="word-wrap:break-word;">9466-delnp-2008-pct-304.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTQ2Ni1kZWxucC0yMDA4LXBjdC0zMDYucGRm" target="_blank" style="word-wrap:break-word;">9466-delnp-2008-pct-306.pdf</a></p>
		<br>
		<div class="pull-left">
			<a href="265065-materials-handling-vehicle-having-substantially-all-hydraulic-components-mounted-on-a-main-frame-assembly.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="265067-method-for-predicting-a-speed-output-of-a-hybrid-powertrain-system.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>265066</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>9466/DELNP/2008</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>06/2015</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>06-Feb-2015</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>04-Feb-2015</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>12-Nov-2008</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>NANOBIOSYM, INC.</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td>200 BOSTON AVENUE, SUITE 4700, MEDFORD, MA 02155-4248 (US)</td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>GOEL, ANITA</td>
											<td>6 CANAL PARK, SUITE 209, CAMBRIDGE, MA 02141 (US)</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>C12Q 1/70</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>PCT/US2007/009747</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td>2007-04-23</td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td>60/793,720</td>
									<td>2006-04-21</td>
								    <td>U.S.A.</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/265066-methods-for-drug-discovery-using-a-single-molecule-detection-apparatus by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 03:17:50 GMT -->
</html>
