<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/265415-phage-display-using-cotranslational-translocation-of-fusion-polypeptides by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 04:32:54 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 265415:&quot;PHAGE DISPLAY USING COTRANSLATIONAL TRANSLOCATION OF FUSION POLYPEPTIDES&quot;</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">&quot;PHAGE DISPLAY USING COTRANSLATIONAL TRANSLOCATION OF FUSION POLYPEPTIDES&quot;</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>The present invention relates to a filamentous phage display method wherein the polypeptides of interest displayed on the phage particle are cotranslationally translocated across the cytoplasmic membrane of Gram-negative bacteria based on the signal recognition particle pathway. This method is particularly suitable for polypeptides, which are known to be difficult to display on phages, and for proteins of cDNA libraries and other combinatorial libraries, in particular when derived from very fast folding, stable protein scaffolds. The invention further relates to phage or phagemid vectors useful in the method comprising a gene construct coding for a fusion polypeptide comprising the polypeptide to be displayed on the phage particle and an N-terminaJ signal sequence promoting cotranslational translocation.</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td>Phage display using cotranslattonal translocation of fusion polypeptides<br>
Field of the invention<br>
The present invention relates to a novel phage display method, phage or phagemid vectors used therein and the phage particles so obtained.<br>
Background of the invention<br>
Display of polypeptides on bacteriophage (phage display) is a selection technique that allows to extract polypeptides with desired properties from a large collection of variants (Russel, M., Lowman, H.B., and Clackson, T., Introduction to phage biology and phage display, in "Phage Display", Clackson, T. and Lowman, H.B., eds., Oxford University Press, 2004, pp. 1-26). Phage display has been intensively investigated for the selection from combinatorial antibody or peptide libraries.<br>
By far the most widely used bacteriophages used in phage display are filamentous phages. Filamentous phages constitute a large family of bacterial viruses that infect many Gram-negative bacteria. The best-known filamentous phages are those that infect Escherichia coli; these are f1/M13/fd and IKe. Phages f1, M13, and fd are those that have so far been used for filamentous phage display. Their genomes are more than 98% identical and their gene products are interchangeable.<br>
A unique aspect of filamentous phage assembly, in contrast to the assembly of many other bacteriophages, is that it is a secretory process. Incorporation of coat polypeptides into the growing phage occurs in the cytoplasmic membrane, and nascent phages are extruded from the cell as they assemble (Russel et a!., loc. cit.). The E. co//cel! does not lyse in this process. The five viral coat proteins (pill, pVI, pVII, pVIII and pIX) are inserted in the cytoplasmic membrane prior to their incorporation into phage particles (Figure 1). For example, the major part of pill is translocated across the membrane into the periplasm, while its C-terminal hydrophobic tail anchors the protein in the membrane.<br>
One prerequisite for filamentous phage display is the translocation of the polypeptide of nterest (POI) across the cytoplasmic membrane. This is normally achieved by genetically :using the POI to a phage coat protein and translocation of the corresponding fusion jolypeptide. Alternatively, the POI and phage coat protein are translocated independently.<br><br>
In this situation the POI is stably linked to the phage particle in the periplasm by, for example, formation of a disulfide bond (Cys-Display) or formation of a leucine-zipper (pJuFo system) with a corresponding phage coat protein. In conventional filamentous phage display using fusions to pHI, the Sec pathway is used for translocation of the fusion polypeptide comprising the POI. In this pathway, the polypeptide is first synthesized at the ribosome and then posttranslationally translocated, in its unfolded state, by the Sec translocon (Figure 2, (3)-(4)-(5)). That is, the translocation across the cytoplasmic membrane begins only after a substantial amount of the polypeptide chain has been synthesized. However, the contribution of the mechanism of translocation for the success of phage display has not been fully elucidated, and the possibility to use a cotranslational translocation pathway was not explored in the prior art.<br>
Intracellular and extracellular proteins of a wide range of sizes and structures have been functionally displayed on filamentous phage (Russel et al., loc. cit.). Nevertheless, some polypeptides are recalcitrant to display due to individual properties, mostly because of unknown reasons. This makes the success of the display of a certain protein unpredictable. Thus, it has usually been recommended to first test the efficiency of display on filamentous phage for.each protein to be used. In addition, when a combinatorial library is created for phage display, not afl clones will display with similar efficiency; this is especially true for libraries generated from cDNAs. The display problems of polypeptides may be a result of their interference with the phage production, their periplasmic aggregation, their proteolysis, their toxicity to E. colior their incompatibility with the used translocation pathway. Especially, the step preceding translocation is an important factor influencing the incorporation of the fusion polypeptides into the phage particles. If the polypeptides fold prematurely, they can be refractory to translocation or even exhibit cytoplasmic toxicity. Thus, it is important whether the protein is translocated posttranslationally (potentially allowing premature folding) or cotranslationally (not permitting cytoplasmic folding). Current filamentous phage display methods use posttranslational pathways for translocation of the fusion polypeptide across the cytoplasmic membrane (Russel et al., loc. cit.; Paschke, M. and Hohne, W., Gene 350, 79-88, 2005). Thus, polypeptides incompatible with these pathways will be refractory to display, making phage display selections very inefficient or even impossible. For example, the posttranslational Sec pathway, which is almost exclusively used in phage display, is nherently incapable of translocating proteins that cannot remain in an unfolded state in he cytoplasm, since the Sec translocon itself can only transport unfolded polypeptides<br><br>
(Huber, D., Boyd, D., Xia, Y., Olma, M.H., Gerstein, M., and Beckwith, J., J. Bacteriol. 187, 2983-2991, 2005; Paschke et al., loc. cit.).<br>
Thus, the technical problem underlying the present invention is to identify novel translocation approaches for the efficient display of those polypeptides on filamentous phages that are displayed inefficiently by using posttranslational translocation. The solution to this technical problem is achieved by providing the embodiments characterized in the claims.<br>
Summary of the invention<br>
The present invention relates to a filamentous phage display method wherein the polypeptides of interest (POI) displayed on the phage particles are cotranslationally translocated across the cytoplasmic membrane of Gram-negative bacteria, in particular based on the signal recognition particle pathway.<br>
Accordingly, the present invention allows phage display by cotranslational translocation of the fusion polypeptides comprising the POI. This method is particularly suitable for polypeptides, which are known to be difficult to display on phages, and for proteins of cDNA libraries and other combinatorial libraries, in particular when derived from very fast folding, stable protein scaffolds.<br>
The invention further relates to phage or phagemid vectors comprising a gene construct coding for a fusion polypeptide comprising the POI to be displayed on the phage particle and an N-terminal signal sequence promoting cotranslational translocation based on the signal recognition particle pathway, and to phages obtained by the method of the invention.<br>
Brief Description of the Figures<br>
Figure 1. Membrane Insertion and Display of the Polypeptide of Interest<br>
N-termini (N), C-termini (C), polypeptide of interest (POI); N-terminal domains of pill (N1,<br>
N2), C-terminal domain of pill (CT).<br>
A) The display of the polypeptide of interest (POI) on a filamentous phage particle always<br>
includes translocation of the POI across the cytoplasmic membrane (cm) into the<br>
periplasm (pp). Most often, the POI is translocated as a fusion polypeptide including the<br><br>
coat protein III (pill) or a fragment'thereof. pill comprises two N-terminal domains (N1, N2) and the C-terminal domain (CT). In this specific example, the fusion polypeptide consists of the N-terminal POI and the C-terminal CT. The fusion polypeptide and pill are anchored to the cytoplasmic membrane through a C-terminal hydrophobic stretch at the CT moiety after translocation and prior to their incorporation into the phage particle. Cytoplasm (cp), outer membrane (om), extracellular space (ex).<br>
B) A simplified view of a filamentous phage particle displaying pill and the fusion polypeptide of A). The N-terminal domains of pill (N1, N2) and the POI are incorporated into the phage particle via the CT moiety.<br>
Figure 2. Translocation of Polypeptides Across the Cytoplasmic Membrane of Gram-negative Bacteria.<br>
A simplified view of the three major pathways known for the translocation of polypeptides across the cytoplasmic membrane (cm) into the periplasm (pp) of Gram-negative bacteria. These pathways are the SRP pathway, the Sec pathway and the Tat pathway. Both the Sec and the SRP pathway rely on the Sec translocon (SecTR), whereas the Tat pathway relies on the Tat translocon (TatTR). Both the Sec and the Tat pathway translocate polypeptides posttranslationally whereas the SRP pathway translocates polypeptides cotranslationally. The Sec and SRP pathways converge at the Sec translocase that transports the proteins in an unfolded (uf) state through the membrane. In contrast, the Tat translocon only translocates folded (f) proteins. The amino acid composition of the signal sequence (ss) strongly favors the targeting of the preprotein to one of these pathways. After translocation, the signal sequence is cleaved off from the preprotein by a peptidase. The SRP pathway is mediated by the signal recognition particle (SRP), a ribonucleoprotein consisting of the 54-kDa protein homolog and a 4.5S RNA. In this pathway, it is the SRP that targets the preprotein to the Sec translocon. The SRP recognizes and binds corresponding signal sequences emerging from the ribosome (R), delivers the ribosome-nascent chain complex to the SRP-receptor (SR) and subsequently, the preprotein is cotranslationally translocated through the Sec translocon. Polypeptides that are incompatible with the Sec pathway because of their premature cytoplasmic folding can thus be efficiently translocated by the SRP pathway while being translated. (1) The SRP binds to particularly hydrophobic signal sequences of nascent proteins emerging from the ribosome. (2) The SRP directs the ribosome nascent chain complex via the SRP-receptor to the Sec translocon, were the cotranslational translocation takes place. Most preproteins have less hydrophobic signal sequences and undergo SecB dependent export. (3) Trigger factor (TF), a cytosolic chaperone that has a general affinity for nascent<br><br>
polypeptides, binds to the mature region of nascent preproteins and remains effectively bound until the translation is almost finished. (4) Following TF dissociation, cytpsolic factors such as SecB help to maintain preproteins in an extended unfolded conformation. (5) Preproteins that retain an extended conformation are efficiently transported trough the Sec translocon. (6) However, if folding of the preprotein occurs in the cytoplasm, the protein is usually degraded (10) or may even plug up the Sec translocon. Preproteins with signal sequences containing the twin-arginine motif are destined to the Tat translocon. (7) Association with a chaperone (TC), such as DnaK or another Tat-specific factor, probably shields the signal sequence until folding is completed (8). This same factor or an additional factor may also promote correct folding. Tat translocation proceeds only if the preprotein is correctly folded; otherwise, the preprotein is degraded by the proteolytic machinery (9, 10) of the cell.<br>
Figure 3. Schematic Representation of the pDST Phagemid Vector Series. A) Enlarged view of the expression cassette of the pDST phagemid vector series. The expression cassette comprises a promoter/operator element of the /acZgene of E. coll (/acZp/o), a ribosome binding site (not depicted), the coding sequences for the signal sequence (ss) and a polypeptide of interest (POI) to be displayed, a suppressor stop codon (TAG), the coding sequences for a flexible glycine/serine linker (G/S) and for the C-terminal domain (amino acids 250-406) of protein III of filamentous phage (CTplll) mediating incorporation of the fusion polypeptide into the phage particle, two stop codons (TGATAA, not depicted) and a transcription terminator element (not depicted). The coding sequence of the POI is flanked by DNA sequences encoding a Flag-tag (Flag) and a c-myc-tag (c-myc). The single letter ibmino acid sequences for the DsbA signal sequence (DsbAss) as a representative of signal sequences targeting the SRP pathway and for the PhoA signal sequence (PhoAss) as a representative of a signal sequences targeting the Sec pathway are shown. These signal sequences contain a positively charged N-terminal region (n-region), an apolar hydrophobic core (h-region) and a more polar C-terminal region (c-region).<br>
B) Schematic representation of the phagemid pDST23. !n addition to the elements shown in A) the filamentous phage replication origin (Ff ori), the lac represser gene from E. coll (lacf), which produces lac represser needed for the tight control of the lacZo/o, the ColE1 origin of replication (ColE1 ori) for bacterial replication of the vector, the antibiotic resistance gene (cat) encoding a chloramphenicol-acetyl-transferase mediating chloramphenicol resistance and the restriction sites Xbal, Bam\-\\, Psti, and EcoRI are<br><br>
depicted. The in pDST23 encoded POI is the designed ankyrin repeat protein (DARPin) 3a.<br>
Figure 4. Display Yield Comparison by Western Blot Analysis<br>
A) and B) CsCI-purified phage particles produced by the use of the respective phagemids were separated by SDS-PAGE, blotted onto PVDF membranes and detected with antibodies specific for the C-terminal domain of protein III (anti-pill) or the Flag-tag (anti-FlagMI). Aliquots applied per lane have been normalized and correspond to 5 x 1011 phage particles. The display yields on phage particles for various polypeptides are analyzed. The abbreviated names of the polypeptides are indicated on top of the lanes and refer to the polypeptides listed in Table 1. In addition, Table 1 indicates the corresponding phagemids used to produce the phage particles, and an outline of the expression cassette is given in Figure 3A. The display yields are compared for each polypeptide using either the PhoA signal sequences (lanes labeled "p") or the DsbA signal sequence (lanes labeled "d") to translocate the corresponding fusion polypeptide by the Sec pathway or the SRP pathway, respectively. Stronger bands indicate higher display yields. The molecular weights of marker proteins in kDa are indicated at both sides of the blot. The band at 62 kDa in the anti-pill blot corresponds to the pill wild-type protein.<br>
Figure 5. Display Yield Comparison by ELISA Analysis<br>
Phage particles displaying either the cJun N-terminal kinase 2 (JNK2)-binding DARPin called 2_3 (open symbols) or the aminoglycoside kinase APH(3')-llla (APH)-binding DARPin called 3a (filled symbols) were incubated in neutravidin coated and BSA-blocked wells containing immobilized biotinylated JNK2 or APH proteins, respectively. After washing, the bound phage particles were detected with anti-M13 antibody coupled to horseradish peroxidase and visualized with soluble BM Blue POD substrate. The data plotted show the absorbance (Abs) on the y-coordinate measured at 360 nm after subtracting the background measured at 392 nm versus the number of phage particles (pp) applied per well on the x-axis. Phage particles produced using the DsbA signal sequence encoding phagemid variants (triangle symbols) showed half-maximum signal already at about 8x108 phages per well, whereas phage particles produced from PhoA signal sequence encoding variants (square symbols) showed half-maximum signal only at about 2x1011 phage particles per well, indicating more than 100-fold lower display yields. Thus, the use of DsbA signal sequence-mediated SRP pathway for the translocation of the fusion polypeptides strongly increased the display yields in comparison to using the PhoA signal sequence-mediated Sec pathway.<br><br>
Figure 6. Quantification of Display Yield by ELISA Analysis<br>
Phage particles displaying DARPin 2__3 or 3a were analyzed as described for Figure 5. The results are given in a column diagram (logarithmic scale) with PhoAss display yields set to 1. The display level (fold increase) for each signal sequence used corresponds to the number of phage particles giving a signal of OD45o = 0.5 relative to the number of PhoAss-containing phage particles giving a signal of OD4so = 0.5. PelBss: Signal sequence of Erwinia carotovora PelB (a putative Sec-dependent signal sequence). SRP-dependent signal sequences of the E. co//proteins TorT (TorTss), TolB (TolBss) and SfmC (SfmCss) were tested in addition to the DsbAss. An increased display yield of up to 700-fold was observed with the SRP-dependent TorTss compared to the Sec-dependent PhoAss. The SRP-dependent TolBss and SfmCss gave an increased display yield up to 300-fold. The putative Sec-dependent PelBss showed only a two- to sixfold increased display yield.<br>
Detailed description of the invention<br>
In the context of the present invention, the term "filamentous phage display" refers to phage display based on filamentous phages. Filamentous phages constitute a large family of bacterial viruses that infect many Gram-negative bacteria. Preferred filamentous phages are those that infect E. co//; in particular f1/M13/fd and IKe. Methods to practice filamentous phage display are well known to the person skilled in the art (e.g. Russel et al., loc. cit).<br>
In the context of the present invention, the term "signal sequence" refers to an N-terminal stretch of amino acids of a polypeptide resulting in targeting of the polypeptide to a translocase. In E. coll, N-terminal signal sequences generally comprise 15 to 52 amino acids. Most signal sequences contain a positively charged N-terminal region (n-region), an apolar hydrophobic core (h-region) and a more polar C-terminal region (c-region). The c-region contains the cleavage site for signal peptidase. Signal peptidase is a membrane-bound protease that removes the signal sequence from the polypeptide during the translocation reaction. Such signal sequences comprising 18 to 30 amino acids are preferred. The determination of signal sequences is well known to the person skilled in the art. For example, they can be obtained from databases such as Swiss-Prot or GenBank or using annotated genome-wide data sets.<br><br>
In the context of the present invention, the term "preprotein" refers to a polypeptide comprising an N-terminal signal sequence. The signal sequence is cleaved.from the preprotein during the translocation reaction thus yielding the mature protein.<br>
In the context of the present invention, the term "fusion polypeptide" refers to a polypeptide comprising an N-terminal signal sequence, the polypeptide of interest (POI) and an additional amino acid sequence allowing display on filamentous phage. Preferably, this additional sequence comprises a filamentous phage coat polypeptide or a fragment thereof. Alternatively, this sequence connects the POI to the phage particle in the periplasm by formation of a stable linkage, for example, by formation of a disulfide bond (Cys-Display) or by formation of a leucine-zipper (pJuFo system) with a corresponding phage coat protein. In this alternative strategy, the POI and the corresponding phage coat protein are translocated independently across the cytoplasmic membrane.<br>
In the context of the present invention, the term "translocation" refers to the translocation of a polypeptide across a biological membrane mediated by a translocon (Holland, I.B. et al., Biochim. Biophys. Acta 1694, 5-16, 2004). The translocation occurs posttranslationally or cotranslationally. A translocase is hence an enzyme or enzyme complex that specifically transports a polypeptide through the translocon (Holland et al., loc. cit).<br>
In the context of the present invention, the term "Sec pathway" refers to a protein transport mechanism for posttranslational translocation of preproteins across the cytoplasmic membrane of Gram-negative bacteria through the Sec translocon (Holland et al., loc. cit.). The Sec pathway is mediated by molecular chaperones, most often SecB, that keep preproteins in an unfolded state before translocation. The Sec pathway is the major route of protein translocation in Gram-negative bacteria.<br>
In the context of the present invention, the term "SRP pathway" refers to a protein transport mechanism for cotranslational translocation of preproteins across the cytoplasmic membrane of Gram-negative bacteria through the Sec translocon (Schierle, C.F., Berkmen, M., Huber, D., Kumamoto, C., Boyd, D., and Beckwith, J., J. Bacteriol. 185, 5706-5713, 2003; Huber et al., loc. cit.). The SRP pathway is mediated by the signal recognition particle (SRP), a ribonucleoprotein consisting of the 54-kDa protein homolog (Fifty-four homolog; Ffh) and a 4.5S RNA. In the SRP pathway, the signal sequence interacts with SRP as soon as it appears from the ribosome. The complex consisting of SRP, nascent polypeptide and ribosome is then transferred via the SRP-receptor to the<br><br>
Sec translocon where the polypeptide is cotranslationally translocated through the Sec translocon.<br>
The Sec and SRP pathways converge at the Sec translocase that transports the proteins in an unfolded state through the membrane. It is the amino acid composition of the signal sequence that will strongly favor the targeting of the preprotein to the SRP pathway over the Sec pathway for its translocation (Huber et al., loc. cit.).<br>
In the context of the present invention, the term "DsbA" refers to the periplasmic £ coli thiol:disulfide interchange protein DsbA (Swiss-Prot accession number P24991). DsbA is a substrate of the SRP pathway (Huber et al., loc. cit.). DsbA is exported cotranslationally to avoid its folding in the cytoplasm, which would inhibit its export.<br>
In the context of the present invention, the term "TrxA" refers to the E. coli protein thioredoxin 1 (Swiss-Prot accession number P00274). TrxA can be used as a reporter protein to distinguish signal sequences that target a preprotein to the SRP pathway or the Sec pathway (Schierle et al., loc. cit.; Huber et al., loc. cit.).<br>
In the context of the present invention, the term "Tat pathway" refers to the twin-arginine protein translocation (Tat) pathway (Paschke et al., loc. cit.). The Tat pathway differs fundamentally from the Sec pathway and SRP pathway. In contrast to the Sec translocon, the Tat translocon exports only fully folded proteins. In contrast to the SRP pathway, the Tat translocon passes proteins posttranslationally through the membrane.<br>
In one particular embodiment the signal sequence of the fusion polypeptide comprising the POI to be displayed on the phage particle is a signal sequence promoting cotranslational translocation.<br>
Methods to test if a signal sequence of interest promotes cotranslational translocation across the cytoplasmic membrane of Gram-negative bacteria are well known to the person skilled in the art. For example, the signal sequence of interest is genetically fused to the mature MalE (Swiss-Prot accession number P02928, residues 27 to 396). This artificial preprotein is then expressed in £ coli and the yield of cotranslational proteolytic processing of its nascent chain is analyzed by two-dimensional gel electrophoresis as described (Josefsson, L.-G. and Randall, L.L., Methods Enzymol. 97, 77-85, 1983; Schierle et al., loc.cit.). Removal of the N-terminal signal sequence of interest while the<br><br>
preprotein chains are still nascent indicates that translocation is initiated before the synthesis of the polypeptide is complete and thus that translocation is cotranslational. Preferred signal sequences are those that promote yields of cotranslational translocation of over 80%, more preferably over 90%, when fused to mature MalE. Most preferred signal sequences are those that do only promote cotranslational translocation and no posttranslational translocation when fused to mature MalE. Alternatively, signal<br>
»<br>
sequences already known to promote cotranslational translocation, such as the signal sequence from DsbA, may be used.<br>
In another particular embodiment, the signal sequence of the fusion polypeptide comprising the POI to be displayed on the phage particle is a signal sequence targeting the signal recognition pathway.<br>
The signal recognition pathway of Gram-negative bacteria and methods to test if a signal sequence of interest targets a preprotein to the SRP pathway are well known to the person skilled in the art. For example, the translocation of TrxA fused to a signal sequence targeting the SRP pathway is strongly inhibited in E. coll bearing a mutation in the gene ffh (e.g. a ffh77 or ffh&amp;7 mutant strain), which encodes a component of the SRP (Schierle et al., loc. cit.; Huber et al., loc. cit.). Thus, those signal sequences that target the SRP pathway promote translocation of TrxA across the cytoplasmic membrane in wild-type E. coli, but very inefficiently in an ffh mutant strain. Signal sequences can thus be grouped into two distinct classes: Those that target the SRP pathway and those that do not target the SRP pathway. Signal sequences that target the Sec pathway can be redirected to the SRP pathway by increasing overall hydrophobicity of the signal sequence, in particular by increasing the hydrophobicity of its h-region. For example, modest alterations of the MalE signal sequence that simply increase its hydrophobicity by replacing polar or small (Gly or Ala) amino acids in the h-region by large hydrophobic residues reroute the protein from the Sec to the SRP pathway. Alternatively, signal sequences already known to target the SRP pathway, such as the signal sequence from DsbA, may be used. Other examples of signal sequences using the SRP pathway are those of a subset of autotransporters, such as that of the hemoglobin protease (Hbp, UniProtKB accession number O88093). Unusually, the Hbp signal sequence is relatively long (52 amino acids) and contains a N-terminal extension that precedes a classical signal sequence. In addition, the h-region of the Hbp signal sequence is not particularly hydrophobic. The signal sequence of SecM (Swiss-Prot accession number P62395) is<br><br>
another example of a long signal sequence that comprises an N-terminal extension and a moderately hydrophobic h-region that is known to target the SRP pathway.<br>
In still another particular embodiment, the signal sequence of the fusion polypeptide comprising the POI to be displayed on the phage particle is a signal sequence promoting translocation of TrxA across the cytoplasmic membrane of Gram-negative bacteria.<br>
Many commonly used signal sequences, e.g. those of PhoA (Swiss-Prot accession number P00634) and MalE (Swiss-Prot accession number P02928), do only inefficiently promote the translocation of TrxA across the cytoplasmic membrane (Schierle et al., loc. cit.). In contrast, the signal sequence from DsbA promotes efficient translocation of TrxA. Subcellular fractionation of host cells expressing TrxA fused to the signal sequence of interest allows to discriminate those signal sequences that are able to promote translocation of TrxA across the cytoplasmic membrane into the periplasm from those that are not (Huber et al., loc. cit.). The quantity of TrxA in the periplasmic fraction describes the efficiency of the signal sequence to promote translocation of TrxA. Alternatively, signal sequences already known to promote translocation of TrxA, such as the signal sequence from DsbA, may be used.<br>
In a preferred embodiment, the signal sequence of the fusion polypeptide comprising the POI to be displayed on the phage particle is a signal sequence selected from the group consisting of TorT, SfmC, FocC, CcmH, Yral, TolB, NikA, Flgl and DsbA, and homologs thereof.<br>
In a particularly preferred embodiment, the signal sequence of the fusion polypeptide comprising the POI to be displayed on the phage particle is a signal sequence selected from the group consisting of TorT, SfmC, TolB and DsbA.<br>
In the context of the present invention, the term "homolog" of a signal sequence means an amino acid sequence with 70%, preferably 80%, and in particular 90% or more amino acid identity with any of the signal sequences mentioned hereinbefore, while conserving the overall charge, hydrophobicity and cleavage properties of the n-region, h-region and c-region of the signal sequence, respectively. Examples of such homologs are amino acid sequences wherein one, two, three or four, in particular one or two, amino acids are replaced by other amino acids, wherein one, two, three or four amino acids are deleted, or one or two amino acids are added, or combinations of replacements, deletions and<br><br>
additions as mentioned hereinbefore. In replacements of amino acids, apolar amino acids are preferably replaced by other apolar amino acids, e.g. lie by Leu, Val, Ala, Trp, Phe or Met or vice versa, polar amino acids by other polar amino acids, e.g. Thr by Ser, Asn or Gin or vice versa, negative charged amino acids by other negative charged amino acids, e.g. Asp by Glu or vice versa, or positive charged amino acids by other positive charged amino acids, e.g. Lys by Arg or His or vice versa.<br>
For example, for the preferred signal sequence of DsbA with the amino sequence MKKIWLALAG LVLAFSASA (SEQ ID NO:1), the replacement of amino acid Lys2 by Arg, Ala9 by Leu, Ala14 by Val and Ser16 by Thr is possible.<br>
The signal sequences of.TorT, SfmC, FocC, CcmH, Yral, TolB, NikA, Flgl and DsbA are known to promote cotranslational translocation of TrxA by targeting the SRP pathway, and possess, in most cases, a higher overall hydrophobicity compared to signal sequences targeting the Sec pathway (Huber et al., loc. cit). The proteins have the following Swiss-Prot accession numbers: TorT P38683, SfmC P77249, FocC P62609, CcmH P33925, Yral P42914, TolB POA855, NikA P33590, Flgl POA6S3, and DsbA P24991.<br>
Hydrophobicity calculations alone do not allow to discriminate SRP dependent and non-SRP dependent signal sequences in the high hydrophobicity range (Huber et al., loc. cit). Thus, features other than hydrophobicity (e.g. the structure of the signal peptide) influence the preference for one translocation pathway.<br>
In a particularly preferred embodiment, the signal sequence is the DsbA signal sequence or any amino acid sequence possessing 90% identity with the DsbA signal sequence.<br>
The method is applicable to any of the filamentous phage display methods, for example with phages f1, M13, fd and Ike, in particular f1, M13 and fd.<br>
In particular, the method of the invention comprises the steps of (a) constructing a filamentous phage or phagemid vector containing an expression cassette for a fusion polypeptide that possesses an N-terminal signal sequence promoting cotranslational translocation of the fusion polypeptide across the cytoplasmic membrane of Gram-negative bacteria;<br><br>
(b)	constructing a combinatorial library of phage or phagemid vectors by cloning of a DNA<br>
library encoding the polypeptides of interests into the expression cassette-of the vector of<br>
step (a);<br>
(c)	transforming suitable Gram-negative bacteria with the library of vectors of step (b); and<br>
(d)	performing phage display selection cycles to separate phage particles based on the<br>
properties of the displayed proteins of interest.<br>
In step (a), the filamentous phage or phagemid vector is constructed using standard methods of gene technology. For example, the signal sequence encoding part of the expression cassette for the fusion polypeptide of an established phage or phagemid vector is replaced by the coding sequence of said signal sequence using standard DNA techniques. Alternatively, a novel phage or phagemid vector containing an expression cassette for the fusion polypeptide containing said signal sequence is constructed de novo using general knowledge on the composition of such vectors (e.g. Russel et al., loc. cit) and standard DNA synthesis and assembly methods. Phage or phagemid vectors useful for that purpose are, for example, pAK100, pCombS, pEXmide3, pHEN1, pJuFo or pSEX. An example of such a phagemid vector (pDST23) is described in Figure 3 and in the accompanying Example, as an illustration of the invention without limiting the invention to this particular embodiment.<br>
Preferably, the signal sequence of the fusion polypeptide in step (a) promotes translocation of the fusion polypeptide through the SRP pathway.<br>
More preferably, the signal sequence of the fusion polypeptide in step (a) promotes the cotranslational translocation of TrxA.<br>
In step (b), standard methods for the preparation of combinatorial libraries of vectors are used. For example, combinatorial DNA libraries encoding the proteins of interest are generated by random or site-directed mutagenesis, by DNA shuffling, by preparation of cDNA through amplification of cellular mRNA or by consensus design and then ligated into the expression cassette of said vector by standard DNA techniques.<br>
In step (c), standard methods for the transformation of Gram-negative bacteria are used. For example, the bacteria are transformed by the combinatorial library of vectors of step (b) by electroporation or chemical means. Such methods are well known to the person skilled in the art.<br><br>
In step (d), standard phage display selection cycles are performed. Such phage display selection cycles are well known to the person skilled in the art (e.g. Russel et al., loc. cit.).<br>
Preferably, the property of the displayed polypeptide of interest of step (d) is specific binding to a target molecule of interest. In this case, phage particles displaying a POI binding to the target molecule are separated from phage particles displaying irrelevant polypeptides by applying the amplified phage particles in each selection cycle to the target molecule functionally immobilized on a surface, washing unbound phage particles away, eluting the bound phage particles and using the eluted phage particles as input for the amplification of phage particles of the next selection cycle.<br>
It is understood that, whenever in the context of this invention, "a signal sequence" or "the signal sequence" is mentioned, such an expression also means "one or more", e.g. one, two, three or four, signal sequences of different composition. Using more than one signal sequences may be advantageous for particular applications, and is also within the ambit of this invention.<br>
The invention further relates to a phage or phagemid vector comprising a gene construct coding for a fusion polypeptide comprising the POI fused to an N-terminal signal sequence promoting cotranslational translocation of TrxA, in particular a signal sequence that is selected from the group consisting of signal sequences from TorT, SfmC, FocC, CcmH, Yral, TolB, NikA Flgl, and DsbA, and homologs thereof, preferably selected from TorT, SfmC, TolB and DsbA. Most preferred is a phage or phagemid vector comprising the signal sequence DsbA or a homolog thereof.<br>
Preferably, the fusion polypeptide comprises the POI fused to the phage coat protein pill or pVIII, or to a fragment of. the coat protein pill. Such a fragment is, for example, a fragment comprising amino acids 250 to 406 of pill.<br>
The signal sequence of the periplasmic enzyme DsbA directs fused reporter proteins to the SRP pathway and thus enhances their cotranslational export. This indicates that the relatively hydrophobia DsbA signal peptide interacts with SRP and promotes cotranslational translocation of DsbA. Thus, the signal sequence of DsbA can be used as a generic signal sequence for other proteins, as will be shown below in the Example.<br><br>
Likewise, homologs of the signal sequence of DsbA, and signal sequences of TorT, SfmC, FocC, CcmH, Yral, TolB, NikA, and Flgl and homologs thereof may be used.<br>
The method of the invention is particularly suitable for the application with libraries of compounds, for example DMA libraries, in particular cDNA libraries. In contrast to methods used hitherto in phage display, the method of the invention allows reliable presentation of polypeptides obtained by expression of such libraries.<br>
A particular embodiment of the invention is the described filamentous phage display method wherein POIs encoded by a DNA library are displayed on the phage particles. Preferably, cotranslational translocation for the POIs encoded by a DNA library is accomplished based on the signal recognition particle pathway, such as a signal sequence promoting cotranslational translocation of TrxA. Most preferred is the method as described herein for the phage display of repeat proteins.<br>
As with any selection technology, the success of phage display selections strongly depends on the diversity of displayed library members. A large combinatorial DNA library does not by itself guarantee that a large diversity of library members can be displayed. In phage display, the polypeptides to be displayed have to be translocated across the cytoplasmic membrane before their incorporation into phage particles. Current filamentous phage display methods all use a posttranslational pathway for translocation of the fusion polypeptide across the cytoplasmic membrane (Russel et al., loc. cit.; Paschke et al., loc. cit.). Thus, library members incompatible with these pathways will be refractory to display thus clearly reducing the displayed library diversity.<br>
DNA libraries considered are all possible combinatorial DNA libraries including those produced by random or site-directed mutagenesis, by DNA shuffling, or by consensus design. Such methods will generate library members with novel properties, such as binding properties; some of them will be incompatible with the translocation using the Sec pathway. Such libraries include DNA libraries that encode peptide libraries, antibody libraries or libraries based on alternative scaffolds (Russel et al., loc. cit.; Nygren, P.A. and Skerra, A., J. Immunol. Methods, 290, 3-28, 2004).<br>
Further DNA libraries considered are cDNA libraries, especially those of eukaryotic origin. cDNA libraries encode a great variety of naturally occurring cellular proteins including cytoplasmic proteins, membrane proteins and extracellular proteins. Some of these<br><br>
naturally occurring library members will be incompatible with translocation using the Sec pathway.<br>
Further DNA libraries considered are those encoding single-chain Fv (scFv) antibody libraries that are used to select intracellularly active scFv fragments (intrabodies). A prerequisite for intrabodies is that they fold well and are stable in the cytoplasm of E. coli. Thus, cytoplasmic-stable and well-folded intrabodies are inefficiently translocated by the posttranslational mechanism of the Sec pathway.<br>
Further DNA libraries considered are those encoding alternative scaffold libraries based on repeat proteins, including ankyrin repeat proteins, leucine-rich repeat proteins, tetratricopeptide repeat protein, pentatricopeptide repeat proteins or armadillo/HEAT repeat proteins.<br>
A preferred DNA library is a library encoding designed ankyrin repeat proteins (DARPins) (Binz, H.K., Amstutz, P., Kohl, A,, Stumpp, M.T., Briand, C., Forrer, P., Grutter, M.G., and Pluckthun, A., Nat. Biotechnol., 22, 575-582, 2004). Examples of DARPins displayed on phage particles are shown in the Example.<br>
The invention further relates to the phages produced by the method of the invention.<br>
The invention further relates to the periplasmic expression of very fast folding and cytoplasmically stable proteins, in particular to the periplasmic expression of DARPins. The DsbAss and DARPins encoding phagemids of the Examples can directly be used for the efficient periplasmic expression of the DARPins in a non-suppressing E. coli. Alternatively, standard periplasmic expression vectors can be adapted by replacing the DNA encoding the signal sequence used for periplasmic expression by DNA encoding a signal sequence of the current invention, in particular by the DNA encoding the DsbAss. For example, the efficient periplasmic expression of DARPins is instrumental to the expression of DARPins fused to effector proteins, in particular toxins or cytokines, which are very difficult to express in the cytoplasm.<br>
The new phage display method exemplified herein below allows to efficiently incorporate a very broad range of POIs to be displayed into phage particles and thus enables efficient phage display. The key difference of this new method compared to traditional phage display methods is the use of signal sequences directing the fusion polypeptide<br><br>
comprising the POI to be displayed to the cotranslational SRP pathway (Figure 2, (1 )-(2)). In this way, the POI to be displayed is efficiently translocated across the cytoplasmic membrane into the periplasm, thus enabling efficient incorporation of the POI into the phage particles. Preferred fusion proteins also comprise one of the filamentous phage coat proteins or truncated versions of the coat proteins. In this case, the POI is anchored to the cytoplasmic membrane by the hydrophobic extension of the phage coat protein after translocation (Figure 1) and before incorporation into the phage particle.<br>
One particular example of a signal sequence targeting the SRP pathway is the signal sequence of the E. coll protein DsbA (DsbAss). All other elements of the exemplified pDST phagemids are derived from classical phagemids such as the pAK1 00 series, which carry the signal sequences of PelB (PelBss) or the PhoA (PhoAss) directing the polypeptides of interest to be displayed via the posttranslational Sec pathway (Figure 2,<br>
In one series of experiments, the display yields were compared between phage particles produced from pDST phagemids encoding PhoAss and pDST phagemids encoding DsbAss. Phage particles were produced with standard protocols as described beiow and purified by CsCI gradient centrifugation. Western Blotting showed that for the display of a single-chain Fv antibody the pDST phagemid produced phage particles that have about the same display yields of the POI independent of the signal sequence used (Figure 4A, scFv). In stark contrast however, all four tested DARPins could only be efficiently displayed when using the DsbAss containing pDST phagemids, and almost no protein displayed could be detected when using the PhoAss containing pDST phagemids (Figure 4A, DARPins). Similarly, the DsbAss containing pDST phagemids resulted in considerably higher display yields in case of the polypeptides GCN4 (Figure 4A), lambda head protein D, TrxA and APH (Figure 4B). Only slightly higher display yields were observed for proteins Taq polymerase, phage Lambda protein phosphatase (A.PP) and no displayed protein could be detected in case of c-jun N-terminal kinase 2 (JNK2, Figure 4B). This demonstrates that the display yields on phage particles produced by using DsbAss containing pDST phagemids are at least comparable to those produced by classical phagemids using PhoAss, but show strongly increased display yields when displaying DARPins and other rapid folded and thermodynamically stable proteins.<br>
To quantify this difference, phage particles were used in phage ELISA experiments. Phage particles displaying DARPins specifically binding the proteins APH and JNK2 were<br><br>
compared after production from either DsbAss containing pDST phagemids or PhoAss containing pDST phagemids. Based on the detection of bound phage particles as quantified with an anti-M13 antibody, an increased display yield of more than 100-fold was observed (Figure 5).<br>
t<br>
To demonstrate that the higher display yields obtained by using DsbAss also benefit selection experiments, two test mixtures containing three different types of phage particles produced from phagemids encoding either DsbAss or PhoAss were mixed in various dilutions. For both test mixtures phage particles displaying DARPins specifically binding the proteins APH and JNK2 were spiked at a 1:107 dilution into phage particles displaying unselected DARPins E3_5 and E3_J9. These two test mixtures were used as input libraries for standard phage display selections on the target proteins APH and JNK2 (Table 2). Whereas the APH and JNK2 specific phage particles could be enriched from the test mixtures produced from DsbAss-encoding phagemids around 1000-fold per selection cycle (already more than 10% of the tested clones were specific for their target after only two cycles of selection), no enrichment from the test library produced from PhoAss containing phagemids could be observed even after five selection cycles (no specific clones observed).<br>
In another series of experiments, the display yields were compared by phage ELISA between phage particles produced from pDST phagemids encoding PhoAss, PelBss, DsbAss, TorTss, TolBss or SfmCss. Phage particles displaying DARPins specifically binding the proteins APH and JNK2 were compared after production from individual pDST phagemids. Based on the detection of bound phage particles as quantified with an anti-Mi 3 antibody, an increased display yield of up to 700-fold was observed with the SRP-dependent signal sequences (DsbAss, TorTss, TolBss or SfmCss) compared to the Sec-dependent signal sequence PhoAss. (Figure 6).<br>
Table 1. Description ofpr-pteins displayed on the surface of filamentous bacteriophage<br><br>
(Table Removed)BscFv, single chain Fv antibody fragment; DARPin, designed ankyrin repeat protein; PhoAss, PhoA signal sequence; DsbAss, DsbA signal sequence<br>
'The first and last amino acids used are indicated in single letter amino acid code, point<br>
mutations are mentioned .<br>
C(SEQ ID NO:2)<br>
d(SEQ ID NO:3)<br>
°Binz, H.K. etal., loc. cit.<br>
'GenBank accession number AAO25689<br>
9GenBank accession number AAO25690<br>
h(SEQ ID N0:4)<br>
'Swiss-Prot accession number P03712<br>
jSwiss-Prot accession number P45984<br>
kSwiss-Prot accession number P00274<br>
'Swiss-Prot accession number P19821<br>
mSwiss-Prot accession number P03772 "Swiss-Prot accession number POA3Y5<br>
Table 2. Enrichment of DARPins 3a and 2_3 presenting phage8<br>
(Table Removed)alnput mixtures produced from phagemids encoding either PhoAss or DsbAss were produced as described in the Example. To a 1 :1 mixture of phage particles displaying the unselected DARPins E3_5 and E3_19, phage particles displaying the target specific DARPins 3a or 23 were added in a 1 :107 dilution.<br>
"Colonies were screened by DMA sequencing<br>
Table 3. Signal sequences<br><br>
(Table Removed)Example<br>
Materials<br>
Chemicals were purchased from Fluka (Switzerland). Oligonucleotides were from Microsynth (Switzerland). Vent DMA polymerase, restriction enzymes and buffers were from New England Biolabs (USA) or Fermentas (Lithuania). Helper phage VCS M13 was from Stratagene (USA). All cloning and phage amplification was performed in E. co//XL1-Blue from Stratagene (USA).<br>
Molecular Biology<br>
Unless stated otherwise, all molecular biology methods were performed according to described protocols (Ausubel, P.M., Brent, R., Kingston, R.E., Moore, D.D., Sedman, J.G., Smith, J.A. and Stuhl, K. eds., Current Protocols in Molecular Biology, New York: John Wiley and Sons, 1999). Brief protocols are given below.<br>
Phage Display Related Methods<br>
Unless stated otherwise, all phage display related methods were performed according to described protocols (Clackson, T. and Lowman, H.B. eds., Phage Display A Practical Approach, New York: Oxford University Press, 2004; Barbas III, C.F., Burton, D.R., Scott, J.K. eds., Phage Display: A Laboratory Manual, Cold Spring Harbor Laboratory Press, 2001). Brief protocols are given below.<br>
Cloning<br>
A derivative of phagemid pAK100 (Krebber, A., Bornhauser, S., Burmester, J., Honegger, A., Willuda, J., Bosshard, H.R., and Pluckthun, A., J. Immunol. Methods 201, 35-55, 1997) encoding the DARPin 3a was the starting point for the cloning of the first phagemid of this study, called pDST23.<br>
To replace the signal sequence of this pAK100 derivative, the Oligonucleotides oDST4 (SEQ ID NO:5), oDST5 (SEQ ID NO:6), oDST6 (SEQ ID N0:7), and oDST8 (SEQ ID NO:8) were designed. These four Oligonucleotides encode the E. co//DsbA signal sequence. The E. coli DsbA protein can be found in the Swiss-Prot database (accession number P24991). Its signal sequence is MKKIWLALAG LVLAFSASA (SEQ ID NO:1). The Oligonucleotides oDST4, oDST5, oDSTG, and oDST8 were annealed and amplified with Oligonucleotides oDST6 and oDST8 by PCR. The resulting DNA fragment encodes the DsbA signal sequence and is flanked by the restriction endonuclease sites Xba\ and BamHl. This DNA fragment was digested with Xba\ and BamH\ and ligated into the<br>
similarly treated and dephosphorylated pAK100 derivative. The resulting phagemid pDST23 (Figure 3) was isolated and the correct sequence was verified by DNA sequencing.<br>
To allow direct experimental comparison to phagemids encoding the signal sequence of PhoA (SEQ ID NO:9), a second phagemid called pDST22 was generated. Again, four oligonucleotides - called oDST4p (SEQ ID NO:10), oDSTSp (SEQ ID NO:11), oDSTG (SEQ ID NO:7), and oDSTSp (SEQ ID NO:12) - were annealed and amplified with oDST6 and oDSTSp by PCR. The resulting DNA fragment encodes the PhoA signal sequence and is flanked by the restriction endonuclease sites Xba\ and BamH\. This DNA fragment was digested with Xba\ and BamH\ and ligated into the similarly treated and dephosphorylated pDST23. The resulting phagemid pDST22 was isolated and the correct sequence was verified by DNA sequencing.<br>
The other phagemids used in this study are listed in Table 1 and were obtained as follows: The coding sequences of the proteins of interest were PCR amplified using appropriate designed PCR primers and template DNA, such as prepared cDNA or public available plasmid DNA. Thereby, either a BamHl or a Bgl\\ restriction sites was introduced 5-prime to each of the coding sequences and two restriction sites (EcoRl and Pstl) were introduced 3-prime to each of the coding sequences. These PCR fragments were digested either with BamH\ or Bgl\\ and either EcoR\ or Psfi, and then ligated into the similarly treated and dephosphorylated phagemids pDST23 or pDST22. The open reading frame of the expression cassette for the fusion polypeptide comprising the cloned PCR product was maintained for all constructs, especially the correct reading frame for the C-terminal fusion to the C-terminal domain of phage protein III (CTp3) was maintained. The first and the last amino acids of the cloned proteins of interest are given in Table 1 as well as the reference or accession number for either the GenBank or the Swiss-Prot databases. The correct sequence of all phagemids was verified by DNA sequencing.<br>
For the designed ankyrin proteins (DARPins) 3a and 2_3, phagemids encoding the PelBss (pDSTSO and pDST81, respectively), SfmCss (pDST86 and pDST87, respectively), TolBss (pDST84 and pDST85, respectively) and TorTss (pDST88 and pDST89, respectively) were generated, using the same cloning strategy as described above for DsbAss and PhoAss.<br>
Phage production and Purification<br>
5 ml 2xYT medium containing 1% glucose, 34 ng/ml chloramphenicol (cam) and 15 ng/ml<br>
tetracycline (tet) were inoculated with a single colony of E. co//XL-1 Blue harboring the<br>
phagemid of interest and the cells were grown overnight at 30°C with shaking. Fresh 5 ml 2xYT medium containing 1% glucose, 34 ug/ml cam and 15 jog/ml tet were inoculated with the overnight cultures at a ratio of 1:100 (OD600 ~ 0.04) and grown at 37^0 to an OD600 of 0.5 with shaking. The cultures were infected with VCS M13 helper phage at 4 x 1010 pfu (plaque forming units) per ml (multiplicity of infection ~ 20) and the cells were incubated for 30 min at 37°C without agitation and then for 30 min at 37° C with shaking. The medium was changed by harvesting the cells by centrifugation (3500 g, 24°C, 10 min) and resuspending the pellet in 50 ml of 2xYT medium containing 34 jag/ml cam, 50 ug/ml kanamycin (kan) and 0.1 mM isopropyl-p-D-thiogalactoside (IPTG). Aftergrowth for 14 to 16 h at 30°C with shaking, the cells were removed by centrifugation (5600 g, 4°C, 10 min). The culture supernatant was incubated on ice for 1 h with one-fourth volume of ice-cold PEG/NaCI solution (20 % polyethyleneglycol (PEG) 6000, 2.5 M NaCI). The precipitated phage particles were then collected by centrifugation at (5600 g, 4°C, 15 min) and redissolved in 1 ml of TBS150 (25 mM Tris/HCI, 150 mM NaCI, pH 7.5). Further purification of the phage particles was done by CsCI gradient centrifugation as follows. After addition of 1.6 g of CsCI, the volume was adjusted to 4 ml with TBSi50. The CsCI solution was transferred into a 1/2 x 11/2 inch polyallomer tube (Beckmann, USA, No 358980) and centrifuged at 100000 r.p.m. for 4 h in aTLN-100 rotor (Beckman Instruments) at 4°C. After centrifugation the phage band was recovered. The phages were transferred to 1/z x 2 inch polycarbonate tubes .(Beckmann, USA, No 349622), which were filled with TBS150 to 3 ml. After centrifugation at 50000 r.p.m. for 1 h in a TLA-100.3 rotor at 4°C, the pelleted phages were redissolved in 3 ml TBS. After an additional centrifugation at 50,000 r.p.m. for 1 h in a TLA-100.3 rotor at 4°C, the phages were dissolved in 1 ml TBS. The total concentration of phage particles was quantified spectrophotometrically. The infective titer of the phage samples was determined by titration on E. co//XL-1 Blue cells using 2xYT agar plates containing 1% glucose and 34 jjg/ml cam. The colonies were counted after overnight incubation at 37 <c.></c.>
Phage blots<br>
5 x 1011 phage particles, purified by CsCI gradient, were applied to 15% sodium dodecyl sulfate polyacrylamide gel electrophoresis (SDS-PAGE) under reducing conditions and transferred to a polyvinylidene fluoride (PVDF) Immobilon-P Transfer Membrane (Millipore, USA) by electroblotting. The membrane was blocked with MTTBS150 (TBS150l 0.1 % Tween 20, 5 % skimmed milk) for 1 h at room temperature (RT) and incubated with a murine anti-pill antibody (MoBiTec, Germany, No. PSKAN3) (1:1000 in MTTBS150l 20 min at RT) as primary antibody, which recognizes the C-terminal domain of pill. A F(ab')2<br>
fragment goat anti-mouse IgG horseradish peroxidase conjugate (Pierce, USA, No. 31438) (1:10000 in MTTBSi5o, 1. *1 at RT) was used as secondary antibody. The proteins were detected with ChemjG low West substrate (Alpha Innotech, USA). In a second experiment, the blocked membrane was incubated with murine anti-FLAG M1 antibody (Sigma, USA, No. F3040) (1:5000 in MTTBSi50, 1 h at RT) as primary antibody. A goat anti-mouse IgG alkaline phosphatase conjugate (Sigma, USA, No. A3562) (1:10000 in MTTBS150, 1 h at RT) was used as secondary antibody. The proteins were detected with the substrates 5-bromo-4-chloro-3-indolyl phosphate (BCIP) and nitro blue tetrazolium (NBT) (Fluka, Switzerland).<br>
Phage ELISA<br>
Phage ELISAs were carried out to assay the amount of functionally displayed DARPins on M13 phage particles. Biotinylated APH and JNK2 proteins (Binz et al., loc. cit.) were immobilized as follows: Neutravidin (66 nM, 100 ^I/well; Socochim, Switzerland) in TBSi50 was immobilized on MaxiSorp plates (Nunc, Denmark, No. 442404) by overnight incubation at 4°C. The wells were blocked with 300 jil BTTBS15o (TBS150, 0.1 % Tween 20,1 % BSA) for 1 h at room temperature. Binding of the biotinylated APH and JNK2 proteins (100 uJ, 1 jj.M) in BTTBS150 was done for 1 h at 4°C.<br>
Dilution series of phage particles in BTTBS150 were added to the wells and incubated at RT for 2 h. After washing the wells five times with 300 jil TTBSiso (TBSi50, 0.1 % Tween 20) for 5 min, bound phage particles were detected with anti-M13 horseradish peroxidase conjugate (Amersham Pharmacia Biotech, UK, No. 27-9421-01) and soluble BM Blue POD substrate (Roche Diagnostics, Germany, No. 1484281).<br>
Phage panning<br>
E3_5, E3_19, 3a and 2_3 displaying phage particles were produced from either phagemids encoding the PhoA signal sequence (pDSTSO, pDST65, pDST22, pDST34) or from phagemids encoding the DsbA signal sequence (pDST32, pDST66, pDST23, pDST37), respectively. These phage particles were used to prepare mixtures of phage particles produced from phagemids encoding either PhoAss or DsbAss. To a 1:1 mixture of phage particles displaying the non-binding DARPins E3_5 and E3__19, phage particles displaying the target-specific DARPins 3a or 2J3 were added in a 1:107 dilution. Biotinylated APH and JNK2 proteins were coated as described for the phage ELISA. To each well 0.1 ml of phage particle mixtures (1013 cfu/ml) were added to 0.1 ml BTTBS150 and incubated for 2 h. After washing (3 times for the first selection cycle, 4 times for the second cycle and 5 times for additional cycles) with TTBS150 and (3 times for the first<br>
selection cycle, 4 times for the second cycle and 5 times for additional cycles) with TBS150, the phage particles were eluted by incubating for 15 min with 0.2 ml elution buffer (0.2 M glycine/HCI, pH 2.2) at about 22°C followed by an elution for 30 min with 0.2 ml trypsin (10 mg/ml in TBS150) at 37°C. The combined eluates (neutralized with 10 pJ of 2 M Tris-base) were used for the infection of 4 ml of exponentially growing E. co//XL1-Blue. After 30 min at 37°C without agitation and 30 min at 37°C with shaking, the cells were spread on 2xYT agar plates containing 1%'glucose and 34 jjg/ml cam and 15 jag/ml tet and grown overnight at 37 °C. The cells were washed from the plates with 2xYT containing 1% glucose, 15 % glycerol, 34 jig/ml cam and 15 ug/ml tet and used for the phage production for the next cycle of panning. After each panning cycle, the identity of 9 to 16 eluted phage particles was determined. This was done by infection of E. coll with these phage particles and screening of the colonies by PCR with clone-specific primers.<br><br><br><br><br>
Claims<br>
1.	A filamentous phage display method wherein polypeptides of interest encoded by a<br>
DMA library are displayed'on the phage particles and wherein the polypeptides of interest<br>
are cotranslationally translocated across the cytoplasmic membrane of Gram-negative<br>
bacteria.<br>
2.	The method of claim 1 wherein the cotranslational translocation is accomplished based<br>
on the signal recognition particle pathway.<br>
3.	The method of claim 1 or 2 wherein the signal sequence used for the translocation of<br>
the polypeptide of interest to be displayed on the phage particle is a signal sequence<br>
promoting cotranslational translocation of TrxA.<br>
4.	The method of claim 1, 2 or 3 wherein the signal sequence used for the translocation<br>
of the polypeptides of interest to be displayed on the phage particle is selected from the<br>
group consisting of signal sequences of TorT, SfmC, FocC, CcmH, Yral, TolB, NikA, Flgl,<br>
and DsbA, and homologs thereof.<br>
5.	The method of claim 4 wherein the signal sequence used for the translocation of the<br>
polypeptides of interest to be displayed on the phage particles is selected from the group<br>
consisting of signal sequences of TorT, SfmC, TolB and DsbA.<br>
6.	The method of anyone of claims 1 to 5 wherein the polypeptides of interest to be<br>
displayed on the phage particles are repeat proteins.<br>
7.	The method of anyone of claims 1 to 6 comprising the steps of<br><br>
(a)	constructing a filamentous phage or phagemid vector containing an expression<br>
cassette for a fusion polypeptide that possesses an N-terminal signal sequence promoting<br>
cotranslational translocation of the fusion polypeptide across the cytoplasmic membrane<br>
of Gram-negative bacteria;<br>
(b)	constructing a combinatorial library of phage or phagemid vectors by cloning of a DNA<br>
library encoding the polypeptides of interests into the expression cassette of the vector of<br>
step (a);<br>
(c)	transforming suitable Gram-negative bacteria with the library of vectors of step (b); and<br><br>
 (d) performing phage display selection cycles to separate phage particles based on the properties of the displayed polypeptides of interest.<br>
8.	A filamentous phage display method wherein a polypeptide of interest displayed on the<br>
phage particle is cotranslationally translocated across the cytoplasmic membrane of Gram-negative bacteria using a signal sequence selected from the group consisting of signal sequences of TorT, SfmC.-pocC, CcmH, Yral, TolB, NikA, FIgl, and DsbA, and homologs thereof.<br>
9.	The method of claim 8 wherein the signal sequence is selected from the group<br>
consisting of signal sequences Of TorT, SfmC, TolB and DsbA.<br>
10.	A phage or phagemid vector comprising a gene construct coding for a fusion protein<br>
comprising the polypeptide of interest to be displayed on the phage particle and a signal<br>
sequence selected from-the group consisting of signal sequences of TorT, SfmC, FocC,<br>
CcmH, Yral, TolB, NikA, FIgl, and DsbA, and homologs thereof.<br>
11.	The vector according to claim 10 wherein the signal sequence is selected from the<br>
group consisting of signal sequences of TorT, SfmC, TolB, and DsbA.<br>
12.	A library of phage or phagemid vectors comprising gene constructs coding for fusion<br>
proteins wherein each of the fusion proteins comprises a signal sequence promoting<br>
cotranslational translocatiqn of TrxA and a polypeptide of interest to be displayed on a<br>
phage particle wherein each polypeptide of interest is encoded by a member of a DNA<br>
library.<br>
13.	The library of vectors of claim 12 wherein the signal sequence is selected from the<br>
group consisting of signal sequences of TorT, SfmC, FocC, CcmH, Yral, TolB, NikA, FIgl,<br>
and DsbA, and homologs thereof.<br>
14.	The library of vectors of claim 12 wherein the signal sequence is selected from the<br>
iroup consisting of signal sequences of TorT, SfmC, TolB, and DsbA.<br>
15. The library of vectors of any one of claims 12 to 14 wherein the DNA library encoding ie polypeptides of interest is a DNA library encoding repeat proteins.</td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTA0LWRlbG5wLTIwMDgtQWJzdHJhY3QtKDExLTAyLTIwMTUpLnBkZg==" target="_blank" style="word-wrap:break-word;">904-delnp-2008-Abstract-(11-02-2015).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTA0LWRlbG5wLTIwMDgtQWJzdHJhY3QtKDI5LTA1LTIwMTQpLnBkZg==" target="_blank" style="word-wrap:break-word;">904-delnp-2008-Abstract-(29-05-2014).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTA0LWRlbG5wLTIwMDgtYWJzdHJhY3QucGRm" target="_blank" style="word-wrap:break-word;">904-delnp-2008-abstract.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTA0LURFTE5QLTIwMDgtQ2xhaW1zLSgwOS0wNi0yMDA5KS5wZGY=" target="_blank" style="word-wrap:break-word;">904-DELNP-2008-Claims-(09-06-2009).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTA0LWRlbG5wLTIwMDgtQ2xhaW1zLSgxMS0wMi0yMDE1KS5wZGY=" target="_blank" style="word-wrap:break-word;">904-delnp-2008-Claims-(11-02-2015).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTA0LWRlbG5wLTIwMDgtQ2xhaW1zLSgyOS0wNS0yMDE0KS5wZGY=" target="_blank" style="word-wrap:break-word;">904-delnp-2008-Claims-(29-05-2014).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTA0LWRlbG5wLTIwMDgtY2xhaW1zLnBkZg==" target="_blank" style="word-wrap:break-word;">904-delnp-2008-claims.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTA0LWRlbG5wLTIwMDgtQ29weSBBYnN0cmFjdC0oMTEtMDItMjAxNSkucGRm" target="_blank" style="word-wrap:break-word;">904-delnp-2008-Copy Abstract-(11-02-2015).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTA0LWRlbG5wLTIwMDgtQ29ycmVzcG9uZGVuY2UgT3RoZXJzLSgxMC0xMi0yMDEzKS5wZGY=" target="_blank" style="word-wrap:break-word;">904-delnp-2008-Correspondence Others-(10-12-2013).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTA0LWRlbG5wLTIwMDgtQ29ycmVzcG9uZGVuY2UgT3RoZXJzLSgxMS0wMi0yMDE1KS5wZGY=" target="_blank" style="word-wrap:break-word;">904-delnp-2008-Correspondence Others-(11-02-2015).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTA0LWRlbG5wLTIwMDgtQ29ycmVzcG9uZGVuY2UgT3RoZXJzLSgyOS0wNS0yMDE0KS5wZGY=" target="_blank" style="word-wrap:break-word;">904-delnp-2008-Correspondence Others-(29-05-2014).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTA0LURFTE5QLTIwMDgtQ29ycmVzcG9uZGVuY2UtT3RoZXJzLSgwOS0wNi0yMDA5KS5wZGY=" target="_blank" style="word-wrap:break-word;">904-DELNP-2008-Correspondence-Others-(09-06-2009).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTA0LWRlbG5wLTIwMDgtQ29ycmVzcG9uZGVuY2UtT3RoZXJzLSgxMC0wNi0yMDE0KS5wZGY=" target="_blank" style="word-wrap:break-word;">904-delnp-2008-Correspondence-Others-(10-06-2014).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTA0LWRlbG5wLTIwMDgtY29ycmVzcG9uZGVuY2Utb3RoZXJzLnBkZg==" target="_blank" style="word-wrap:break-word;">904-delnp-2008-correspondence-others.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTA0LWRlbG5wLTIwMDgtZGVzcmlwdGlvbiAoY29tcGxldGUpLnBkZg==" target="_blank" style="word-wrap:break-word;">904-delnp-2008-desription (complete).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTA0LWRlbG5wLTIwMDgtZHJhd2lnbnMucGRm" target="_blank" style="word-wrap:break-word;">904-delnp-2008-drawigns.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTA0LWRlbG5wLTIwMDgtRHJhd2luZ3MtKDI5LTA1LTIwMTQpLnBkZg==" target="_blank" style="word-wrap:break-word;">904-delnp-2008-Drawings-(29-05-2014).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTA0LWRlbG5wLTIwMDgtZm9ybS0xLnBkZg==" target="_blank" style="word-wrap:break-word;">904-delnp-2008-form-1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTA0LWRlbG5wLTIwMDgtZm9ybS0xMy0oMDktMDYtMjAwOSkucGRm" target="_blank" style="word-wrap:break-word;">904-delnp-2008-form-13-(09-06-2009).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTA0LWRlbG5wLTIwMDgtZm9ybS0yLnBkZg==" target="_blank" style="word-wrap:break-word;">904-delnp-2008-form-2.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTA0LWRlbG5wLTIwMDgtRm9ybS0zLSgxMC0xMi0yMDEzKS5wZGY=" target="_blank" style="word-wrap:break-word;">904-delnp-2008-Form-3-(10-12-2013).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTA0LWRlbG5wLTIwMDgtRm9ybS0zLSgxMS0wMi0yMDE1KS5wZGY=" target="_blank" style="word-wrap:break-word;">904-delnp-2008-Form-3-(11-02-2015).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTA0LWRlbG5wLTIwMDgtRm9ybS0zLSgyOS0wNS0yMDE0KS5wZGY=" target="_blank" style="word-wrap:break-word;">904-delnp-2008-Form-3-(29-05-2014).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTA0LWRlbG5wLTIwMDgtZm9ybS0zLnBkZg==" target="_blank" style="word-wrap:break-word;">904-delnp-2008-form-3.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTA0LWRlbG5wLTIwMDgtZm9ybS01LnBkZg==" target="_blank" style="word-wrap:break-word;">904-delnp-2008-form-5.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTA0LWRlbG5wLTIwMDgtR1BBLSgyOS0wNS0yMDE0KS5wZGY=" target="_blank" style="word-wrap:break-word;">904-delnp-2008-GPA-(29-05-2014).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTA0LWRlbG5wLTIwMDgtTWFya2VkIENsYWltcy0oMTEtMDItMjAxNSkucGRm" target="_blank" style="word-wrap:break-word;">904-delnp-2008-Marked Claims-(11-02-2015).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTA0LWRlbG5wLTIwMDgtT3RoZXJzLSgxMS0wMi0yMDE1KS5wZGY=" target="_blank" style="word-wrap:break-word;">904-delnp-2008-Others-(11-02-2015).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTA0LWRlbG5wLTIwMDgtcGN0LTEwMS5wZGY=" target="_blank" style="word-wrap:break-word;">904-delnp-2008-pct-101.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTA0LWRlbG5wLTIwMDgtcGN0LTIxMC5wZGY=" target="_blank" style="word-wrap:break-word;">904-delnp-2008-pct-210.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTA0LWRlbG5wLTIwMDgtcGN0LTIzNy5wZGY=" target="_blank" style="word-wrap:break-word;">904-delnp-2008-pct-237.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTA0LWRlbG5wLTIwMDgtcGN0LTMwMS5wZGY=" target="_blank" style="word-wrap:break-word;">904-delnp-2008-pct-301.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTA0LWRlbG5wLTIwMDgtcGN0LTMwNC5wZGY=" target="_blank" style="word-wrap:break-word;">904-delnp-2008-pct-304.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTA0LWRlbG5wLTIwMDgtU2VxdWVuY2UgTGlzdGluZy0oMTEtMDItMjAxNSkucGRm" target="_blank" style="word-wrap:break-word;">904-delnp-2008-Sequence Listing-(11-02-2015).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Q292ZXJpbmcgbGV0dGVyLnBkZg==" target="_blank" style="word-wrap:break-word;">Covering letter.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=UGV0aXRpb24gdW5kZXIgcnVsZSAxMzcucGRm" target="_blank" style="word-wrap:break-word;">Petition under rule 137.pdf</a></p>
		<br>
		<div class="pull-left">
			<a href="265414-process-for-preparation-of-clopidogrel-its-salts-and-pharmaceutical-composition.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="265416-a-integrated-bed-plate-structure-for-two-cylinder-direct-injection-high-pressure-common-rail-four-stroke-reciprocating-piston-ic-diesel-engine.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>265415</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>904/DELNP/2008</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>09/2015</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>27-Feb-2015</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>24-Feb-2015</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>01-Feb-2008</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>UNIVERSITY OF ZURICH</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td>PROREKTORAT FORSCHUNG, RAMISTRASSE 71, CH-8006 ZURICH, SWITZERLAND.</td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>DANIEL STEINER</td>
											<td>SCHURBUNGERT 41, CH-8057 ZURICH, SWITZERLAND.</td>
										</tr>
										<tr>
											<td>2</td>
											<td>PATRIK FORRER</td>
											<td>FRIEDACKERSTRASSE 2, CH-8050 ZURICH, SWITZERLAND.</td>
										</tr>
										<tr>
											<td>3</td>
											<td>MICHAEL T. STUMPP</td>
											<td>OBERWIESENSTRASSE 70, CH-8050 ZURICH, SWITZERLAND</td>
										</tr>
										<tr>
											<td>4</td>
											<td>ANDREAS PLUCKTHUN</td>
											<td>MOHRLISTRASSE 97, CH-8006 ZURICH, SWITZERLAND.</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>C12N 15/10</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>PCT/EP2006/063729</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td>2006-06-30</td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td>05106236.2</td>
									<td>2005-07-08</td>
								    <td>EPO</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/265415-phage-display-using-cotranslational-translocation-of-fusion-polypeptides by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 04:32:55 GMT -->
</html>
