<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/265496-fermentive-vitamin-c-production by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 04:48:51 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 265496:FERMENTIVE VITAMIN C PRODUCTION</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">FERMENTIVE VITAMIN C PRODUCTION</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>The present invention relates to newly identified microorganisms capable of direct production of L-ascorbic acid (hereinafter also referred to as Vitamin C). The invention also relates to polynucleotide sequences comprising ,genes that encode proteins which are involved in me synthesis of Vitamin C. The invention also features polynucleotides comprising the full length polynucleotide sequences of the novel genes and fragments thereof, the novel polypeptides encoded by the polynucleotides and fragments thereof, as well as their functional equivalents. The present invention also relates to the use of said polynucleotides and polypeptides as hiotechnological tools in the production of Vitamin C from microorganisms, whereby a modification of said polynucleotides and/or encoded polypeptides has a direct or indirect impact on yield, production, and/or efficiency of production of the : fermentation product in said microorganism. Also included are methods/processes of using the polynucleotides and modified polynucleotide sequences to transform host &quot; microorganisms. The invention also relates to genetically engineered microorganisms and their use for the direct production of Vitamin C.</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td>Fermentative Vitamin C Production<br>
The present invention relates to the use of polynucleotides and polypeptides as biotechnological tools in the production of Vitamin C from microorganisms, whereby said polynucleotides and/or encoded polypeptides have a direct or indirect impact on yield, production, and/or efficiency of production of the fermentation product. The invention also relates to genetically engineered microorganisms and their use for the direct production of Vitamin C.<br>
Vitamin C is one of very important and indispensable nutrient factors for human beings. Vitamin C is also used in animal feed even though some farm animals can synthesize it in their own body.<br>
For the past 70 years, Vitamin C has been produced industrially from D-glucose by the well-known Reichstein method. All steps in this process are chemical except for one (the conversion of D-sorbitol to L-sorbose), which is carried out by microbial conversion. Since its initial implementation for industrial production of Vitamin C, several chemical and technical modifications have been used to improve the efficiency of the Reichstein method. Recent developments of Vitamin C production are summarized in Ullmann's Encyclopedia of Industrial Chemistry, 5th Edition, Vol. A27(1996),pp. 547ff.<br>
Different intermediate steps of Vitamin C production have been performed with the help of microorganisms or enzymes isolated therefrom. Thus, 2-keto-L-gulonic acid (2-KGA), an intermediate compound that can be chemically converted into Vitamin C by means of an alkaline rearrangement reaction, may be produced by a fermentation process starting from L-sorbose, by means of strains belonging e.g. to the Ketogulonicigenium or Gluconobacter genera, or by an alternative fermentation process starting from D-glucose, by means of recombinant strains belonging to the Gluconobacter or Pantoea genera.<br>
Current chemical production methods for Vitamin C have some undesirable characteristics such as high-energy consumption and use of large quantities of organic and inorganic solvents. Therefore, over the past decades, other approaches to manufacture Vitamin C using microbial conversions, which would be more economical as well as ecological, have been investigated.<br>
Fermentative Vitamin C production from a number of substrates including D-sorbitol, L-sorbose and L-sorbosone has been reported in several microorganisms, such as algae and yeast, using different cultivation methods. The disadvantage of using these<br>
microorganisms, however, is the low yield of Vitamin C produced since thes organisms are known to be capable of the production of both 2-keto-L-guionic acid and Vitamin C, the yield of microbiologically produced Vitamin C is then limited by the relatively high production of 2-KGA which is more readily synthesized by said microorganism, leading, for instance, to ratios between the concentration of Vitamin C and 2-KGA which are less than 0.1.<br>
Therefore it is desirable to develop production systems which have better industrial applicability, e.g. can be manipulated for increased titers and/or which have reduced fermentation times. One particularly useful system employs genes encoding membrane-bound L-sorbosone dehydrogenases or membrane-bound PQQ bound D-sorbitol dehydrogenases. An example of such a system uses a gene from Gluconobacter oxydans N44-1 encoding L-sorbosone dehydrogenase (hereafter called SNDHai) which converts L-sorbosone to L-ascorbic acid. This gene and homologous thereof have already been described in WO 2005/017159 which are incorporated herein.<br>
There is a continuing need in even more optimized fermentation systems for the microbial production of Vitamin C to get higher yields as with the systems described above.<br>
Surprisingly, it has now been found that under suitable culture conditions host cells expressing SNDHai can be used for further optimizing the direct production of Vitamin C.<br>
This may be achieved by concurrent manipulation of a specific set of genes as further described herein. Such genes may be selected from the group consisting of RCS or SMS genes. This group of genes/proteins and the manipulation of each set is further described and exemplified herein.<br>
The term "direct fermentation", "direct production", "direct conversion" and the like is intended to mean that a microorganism is capable of the conversion of a certain substrate into the specified product by means of one 01 more biological conversion steps, without the need of any additional chemical conversion step. For instance, the term "direct conversion of D-sorbitol into Vitamin C" is intended to describe a process wherein a microorganism is producing Vitamin C and wherein D-sorbitol is offered as a carbon source without the need of an intermediate chemical conversion step. A single microorganism capable of directly fermenting Vitamin C is preferred.<br>
As used herein, "improved" or "improved yield of Vitamin C" caused by a genetic alteration means an increase of at least 5%, 10%, 25%, 30%, 40%, 50%, 75%, 100%, 200% or even more than 500%, compared to a cell which is not genetically altered. Such unaltered cells are also often referred to as wild type cells<br>
Therefore, it is in the first instance an object of the present invention to provide a process for the direct fermentative production of Vitamin C by cultering under suitable culture conditions a host cell which genome is genetically engineered by DNA sequences comprising the following polynucleotides<br>
a)	a polynucleotide encoding L-sorbosone dehydrogenase according to SEQ ID NO:<br>
2 or an active fragment or derivative thereof, and<br>
b)	at least one polynucleotide encoding a protein selected from the group consisting<br>
of<br>
1)	proteins which are involved in the Sorbitol/Sorbose Metabolization System<br>
(SMS); and<br>
2)	proteins which are involved in the Respiratory Chain System (RCS);     ,;<br>
and by isolation of Vitamin C from such cells or the culture medium.<br>
The term "genetically engineered" or "genetically altered" means the scientific alteration of the structure of genetic material in a living organism. It involves the production and use of recombinant DNA. More in particular it is used to delineate the genetically engineered or modified organism from the naturally occurring organism. Genetic engineering may be done by a number of techniques known in the art, such as e.g. gene replacement, gene amplification, gene disruption, transfection, transformation using plasmids, viruses, or other vectors. A genetically modified organism, e.g. genetically modified microorganism, is also often referred to as a recombinant organism, e.g. recombinant microorganism.<br>
SMS proteins are proteins involved in the Sorbitol/Sorbose Metabolization System. Polynucleotides and proteins encoded by these polynucleotides are herein abbreviated by SMS. SMS proteins functipn in the direct metabolization of D-sorbitol or L-sorbose.<br>
RCS proteins are involved in the Respiratory Chain System. Polynucleotides and " proteins encoded by these polynucleotides are herein abbreviated by RCS. RCS proteins function in the well-known respiratory chain of an organism, also known as the electron transport system.In a preferred embodiment, the activity of the protein selected from RCS or SMS proteins is manipulated in such a way that it leads to an improved yield and/or efficiency of production of Vitamin C produced by said host cell compared to the wild type counterpart of said protein. The term "manipulated" as used herein is intended to encompass the genetic modification or alteration of a gene including the modification of its expression level, preferably by molecular biological techniques. In particular the term is intended to include the upregulation and downregulation of the activity of a protein, such a regulation can be achieved by the upregulation or downregulation of a gene encoding the protein. Further methods for the upregulation or downregulation of the activity of a certain protein as detailed above may also be used in this embodiment of the invention.<br>
It is also an object of the present invention to provide vectors comprising such polynucleotides, preferebly in the form of an expression vector.<br>
Furthermore, it is also an object of the present invention to provide a process for producing a host cell which is genetically engineered, for example transformed by such DNA sequences or vectors. This may be accomplished, for example, by transferring polynucleotides es exemplified herein into a recombinant or non-recombinant host cell that may or may not contain an endogenous equivalent of the corresponding gene. Such a transformed cell is also an object of the invention.<br>
Advantageous embodiments of the invention become evident from the dependent • claims. These and other aspects and embodiments of the present invention should be apparent to those skilled in the art from the teachings herein.<br>
Any cell that serves as recipient of the foreign nucleotide acid molecules may be used as a host cell, such as for instance a cell carrying a replicable expression vector or cloning vector or a cell being genetically engineered or genetically altered by well known techniques to contain desired gene(s) on its chromosome(s) or genome. The host cell may be of prokaryotic or eukaryotic origin, such as, for instance bacterial cells, animal cells, including human cells, fungal cells, including yeast cells, and plant cells. Preferably the host cell is a microorganism. More preferably the microorganism belongs to bacteria that can express the L-sorbosone dehydrogenase as an active form in vivo.<br>
Examples of known bacteria able to directly produce Vitamin C in good quantities when altered according to the present invention include strains from the genera of<br>
Ketogulonicigenium, Pantoea, Pseudomonas or Escherichia or Corynebacterium and acetic acid bacteria.<br>
Microorganisms which can be used in the present invention in order to improve the direct production of Vitamin C may be publicly available from different sources, e.g., Deutsche Sammlung von Mikroorganismen und Zellkulturen (DSMZ), Mascheroder Weg IB, D-38124 Braunschweig, Germany, American Type Culture Collection (ATCC), P.O. Box 1549, Manassas, VA 20108 USA or Culture Collection Division, NITE Biological Resource Center, 2-5-8, Kazusakamatari, Kisarazu-shi, Chiba, 292-0818, Japan (formerly: Institute for Fermentation, Osaka (IFO), 17-85, Juso-honmachi 2-chome,Yodogawa-ku, Osaka 532-8686, Japan). Examples of preferred bacteria deposited with IFO are for instance Gluconobacter oxydans (formerly known as G. melanogenus) IFO 3293, Gluconobacter oxydans (formerly known as G. melanogenus) IFO 3292, Gluconobacter oxydans (formerly known as G. rubiginosus) IFO 3244, Gluconobacter frateurii (formerly known as G. industrius) IFO 3260, Gluconobacter cerinus IFO 3266, Gluconobacter oxydans IFO 3287, and Acetobacter ,-„ aceti subsp. orleanus IFO 3259, which were all deposited on April 05, 1954; Acetobacter aceti subsp. xylinum IFO 13693 deposited on October 22, 1975, and Acetobacter aceti subsp. xylinum IFO 13773 deposited on December 08,1977. Strain Acetobacter sp. ATCC 15164, which is also an example of a preferred bacterium, was deposited with ATCC. Strain Gluconobacter oxydans (formerly known as G. melariogenus) N 44-1 as another example of a preferred bacterium is a derivative of the strain IFO 3293 and is described in Sugisawa et al., Agric. Biol. Chem. 54: 1201-1209,1990.<br>
Acetic acid bacteria are preferred in the present invention to directly produce Vitamin C in high yields from a number of substrates including D-sorbitol, L-sorbose and L-sorbosone. Strains from the genera of Gluconobacter, Gluconacetobacter and Acetobacter are further preferred, they were found to be able to directly produce Vitamin C from L-sorbosone, whereas at least Gluconobacter oxydans DSM 17078 was found to be able to produce Vitamin C directly from D-sorbitol, L-sorbose or L-sorbosone. Gluconobacter oxydans DSM 17078 (formerly known as Gluconobacter oxydans N44-1) has been deposited at Deutsche Sammlung von Mikroorganismen und Zellkulturen (DSMZ), Mascheroder Weg IB, D-38124 Braunschweig, Germany according to the Budapest Treaty on 26. January 2005.<br>
In particular, the present invention is related to a process for the direct production of Vitamin C wherein a combination of polynucleotides as disclosed herein or a combination of modified polynucleotide sequences as described hereinafter are<br>
introduced into a suitable microorganism, the recombinant microorganism is cultured under conditions that allow the production of Vitamin C in high productivity, yield, and/or efficiency, the produced fermentation product is isolated from the culture medium and optionally further purified.<br>
Several substrates may be used as a carbon source in the above-mentioned process. Particularly suited carbon sources are those that are obtainable from the D-glucose or D-sorbitol metabolization pathway such as, for example, D-glucose, D-sorbitol, L-sorbose, L-sorbosone, 2-keto-L-gulonate, D-gluconate, 2-keto-D-gluconate or 2,5-diketo-gluconate. Preferably, the substrate is selected from for instance D-glucose, D-sorbitol, L-sorbose or L-sorbosone, more preferably from D-glucose, D-sorbitol or L-sorbose, and most preferably from D-sorbitol or L-sorbose. The term "substrate" and "production substrate" in connection with the above process using a microorganism is used interchangeably herein.<br>
Conversion of the substrate into Vitamin C in connection with the above process . using a microorganism means that the conversion of the substrate resulting in Vitamin C is performed by the microorganism, i.e. the substrate may be directly converted into Vitamin C. Said microorganism is cultured under conditions which allow such conversion from the substrate as defined above.<br>
A medium as used herein for the above process using a microorganism may be any suitable medium for the production of Vitamin C. Typically, the medium is an aqueous medium comprising for instance salts, substrate(s), and a certain pH. The medium in which the substrate is converted into Vitamin C is also referred to as the production medium.<br>
"Fermentation" or "production" or "fermentation process" as used herein may be the use of growing cells using media, conditions and procedures known to the skilled person, or the use of non-growing so-called resting cells, after they have been cultivated by using media, conditions and procedures known to the skilled person, under appropriate conditions for the conversion of suitable substrates into desired" products such as Vitamin C.<br>
In connection with the above process it is understood that the above-mentioned microorganisms also include synonyms or basonyms of such species having the same physiological properties, as defined by the International Code of Nomenclature of Prokaryotes. The nomenclature of the microorganisms as used herein is the one officially accepted (at the filing date of the priority application) by the International<br>
Committee on Systematics of Prokaryotes and the Bacteriology and Applied Microbiology Division of the International Union of Microbiological Societies, and published by its official publication vehicle International Journal of Systematic arid Evolutionary Microbiology (IJSEM). A particular reference is made to Urbance et al,, IJSEM (2001) vol 51:1059-1070, with a corrective notification on IJSEM (2001) vol 51:1231-1233, describing the taxonomically reclassification of G, oxydans DSM 4025 as Ketogulonicigenium vulgare.<br>
As used herein, resting cells refer to cells of a microorganism which are for instance viable but not actively growing, or which are growing at low specific growth rates [u], for instance, growth rates that are lower than 0.02 h"1, preferably lower than 0.01 h"1. Cells which show the above growth rates are said to be in a "resting cell mode".<br>
The process of the present invention may be performed in different steps or phases: preferably, the microorganism is cultured in a first step (also referred to as step;(a) or growth phase) under conditions which enable growth. This phase is terminated by changing of the conditions such that the growth rate of the microorganism is reduced leading to resting cells, also referred to as step (b), followed by the production of Vitamin C from the substrate using the (b), also referred to as production phase.<br>
Growth and production phase as performed in the above process using a microorganism may be performed hi the same vessel, i. e., only one vessel, or in two or more different vessels, with an optional cell separation step between-the two phases. The produced Vitamin C can be recovered from the cells by any suitable means. Recovering means for instance that the produced Vitamin C may be separated from the production medium. Optionally, the thus produced Vitamin C may be further processed.<br>
For the purpose of the present invention relating to the above process, the terms "growth phase", "growing step", "growth step" and "growth period" are used interchangeably herein. The same applies for the terms "production phase", "production step", "production period".<br>
One way of performing the above process may be a process wherein the microorganism is grown in a first vessel, the so-called growth vessel, as a source for the resting cells, and at least part of the cells are transferred to a second vessel, the so-called production vessel. The conditions in the production vessel may be such that the cells transferred from the growth vessel become resting cells as defined above. Vitamin C is produced in the second vessel and recovered therefrom.<br>
In connection with the above process, the growing step can be performed in an aqueous medium, i.e. the growth medium, supplemented with appropriate nutrients for growth under aerobic conditions. The cultivation may be conducted, for instance, in batch, fed-batch, semi-continuous or continuous mode. The cultivation period may vary depending on the kind of cells, pH, temperature and nutrient medium to be used, and may be for instance about 10 h to about 10 days, preferably about 1 to about 10 days, more preferably about 1 to about 5 days when run in batch or fed-batch mode, depending on the microorganism. If the cells are grown in continuous mode, the residence time may be for instance from about 2 to about 100 h, preferably from about 2 to about 50 h, depending on the microorganism. If the microorganism is selected from bacteria, the cultivation may be conducted for instance at a pH of about 3.0 to about 9.0, preferably about 4.0 to about 9.0, more preferably about 4.0 to about 8.0, even more preferably about 5.0 to about 8.0. If algae or yeast are used, the cultivation may be conducted, for instance, at a pH below about 7.0, preferably below about 6.0, more preferably below about 5.5, and most preferably below about 5.0. A suitable temperature range for carrying out the cultivation using bacteria may be for instance from about 13°C to about 40°C, preferably from about 18°C to about 37°C, more preferably from about 13°C to about 36°C, and most preferably from about 18°C to about 33°C. If algae or yeast are used, a suitable temperature range for carrying out the cultivation may be for instance from about 15°C to about 40°C, preferably from about 20°C to about 45°C, more preferably from about 25°C to about 40°C, even more preferably from about 25°C to about 38°C, and most preferably from about 30°C to about38°C. The culture medium for growth usually may contain such nutrients as assimilable carbon sources, e.g., glycerol, D-mannitol, D-sorbitol, L-sorbose, erythritol, ribitol, xylitol, arabitol, inositol, dulcitol, D-ribose, D-fructose, D-' glucose, and sucrose, preferably L-sorbose, D-glucose, D-sorbitol, D-mannitol, and glycerol; and digestible nitrogen sources such as organic substances, e.g., peptone, yeast extract and amino acids. The media may be with or without urea and/or corn steep liquor and/or baker's yeast. Various inorganic substances may also be used as nitrogen sources, e.g., nitrates and ammonium salts. Furthermore, the growth medium usually may contain inorganic salts, e.g., magnesium sulfate, manganese sulfate, potassium phosphate, and calcium carbonate.<br>
In connection with the above process, the specific growth rates are for instance at least 0.02 h:1. For cells growing in batch, fed-batch or semi-continuous mode, the growth rate depends on for instance the composition of the growth medium, pH, temperature, and the like. In general, the growth rates may be for instance in a range<br>
from about 0.05 to about 0.2 h1, preferably from about 0.06 to about 0.15 h"1, and most preferably from about 0.07 to about 0.13 h"'.<br>
In another aspect of the above process, resting cells may be provided by cultivation of the respective microorganism on agar plates thus serving as growth vessel, using essentially the same conditions, e.g., cultivation period, pH, temperature, nutrient medium as described above, with the addition of agar agar.<br>
If the growth and production phase are performed in two separate vessels, then the cells from the growth phase may be harvested or concentrated and transferred to a second vessel, the so-called production vessel. This vessel may contain an aqueous medium supplemented with any applicable production substrate that can be converted to Vitamin C by the cells. Cells from the growth vessel can be harvested or concentrated by any suitable operation, such as for instance centrifugation, membrane crossflow ultrafiltration or microfiltration, filtration, decantation, flocculation. The cells thus obtained may also be transferred to the production vessel in the form of the original broth from the growth vessel, without being harvested, concentrated or washed, i. e. in the form of a cell suspension. In a preferred embodiment, the cells are transferred from the growth vessel to the production vessel in the form of a cell suspension without any washing or isolating step in-between.<br>
If the growth and production phase are performed in the same vessel, cells may be grown under appropriate conditions to the desired cell density followed by a replacement of the growth medium with the production medium containing the production substrate. Such replacement may be, for instance, the feeding of production medium to the vessel at the same time and rate as the withdrawal or harvesting of supernatant from the vessel. To keep the resting cells in the vessel, operations for cell recycling or retention may be used, such as for instance cell recycling steps. Such recycling steps, for instance, include but are not limited to methods using centrifuges, filters, membrane crossflow microfiltration of ultrafiltration steps, membrane reactors, flocculation, or cell immobilization in appropriate porous, non-porous or polymeric matrixes. After a transition phase, the vessel is brought to process conditions under which the cells are in a resting cell mode as defined above, and the production substrate is efficiently converted into Vitamin C.<br>
The aqueous medium in the production vessel as used for the production step in connection with the above process using a microorganism, hereinafter called production medium, may contain only the production substrate(s) to be converted into Vitamin C, or may contain for instance additional inorganic salts, e.g., sodium<br>
chloride, calcium chloride, magnesium sulfate, manganese sulfate, potassium phosphate, calcium phosphate, and calcium carbonate. The production medium may also contain digestible nitrogen sources such as for instance organic substances, e.g., peptone, yeast extract, urea, amino acids, and corn steep liquor, and inorganic substances, e.g. ammonia, ammonium sulfate, and sodium nitrate, at such concentrations that the cells are kept in a resting cell mode as defined above. The medium may be with or without urea and/or corn steep liquor and/or baker's yeast. The production step may be conducted for instance in batch, fed-batch, semi-continuous or continuous mode. In case of fed-batch, semi-continuous or continuous mode, both cells from the growth vessel and production medium can be fed continuously or intermittently to the production vessel at appropriate feed rates. Alternatively, only production medium may be fed continuously or intermittently to the production vessel, while the cells coming from the growth vessel are transferred at once to the production vessel. The cells coming from the growth vessel may be used as a cell suspension within the production vessel or may be used as for instance flocculated or immobilized cells in any solid phase such as porous or polymeric   -matrixes. The production period, defined as the period elapsed between the entrance of the substrate into the production vessel and the 'harvest of the supernatant containing Vitamin C, the so-called harvest stream, can vary depending for instance on the kind and concentration of cells, pH, temperature and nutrient medium to be used, and is preferably about 2 to about 100 h. The pH and temperature can be different from the pH and temperature of the growth step, but is essentially the same as for the growth step.<br>
In a preferred embodiment, the production step is conducted in continuous mode, meaning that a first feed stream containing the cells from the growth vessel and a second feed stream containing the substrate is fed continuously or intermittently to the production vessel. The first stream may either contain only the cells isolated/separated from the growth medium or a cell suspension, coming directly from the growth step, i.e. cells suspended in growth medium, without any intermediate step of cell separation, washing and/or isolating. The second feed stream as herein defined may include all other feed streams necessary for the operation of the production step, e.g. the production medium comprising the substrate in the form of one or several different streams, water for dilution, and base for pH control.<br>
In connection with the above process, when both streams are fed continuously, the ratio of the feed rate of the first stream to feed rate of the second stream may vary between about 0.01 and about 10, preferably between about 0.01 and about 5, most<br>
preferably between about 0.02 and about 2. This ratio is dependent on the concentration of cells and substrate in the first and second stream, respectively.<br>
Another way of performing the process as above using a microorganism of the present invention may be a process using a certain cell density of resting cells in the production vessel. The cell density is measured as absorbance units (optical density) at 600 nm by methods known to the skilled person. In a preferred embodiment, trie cell density in the production step is at least about 10, more preferably between about 10 and about 200, even more preferably between about 15 and about 200, even more preferably between about 15 to about 120, and most preferably between about 20 and about 120.<br>
hi order to keep the cells in the production vessel at the desired cell density during the production phase as performed, for instance, in continuous or semi-continuous mode, any means known in the art may be used, such as for instance cell recycling byv centrifugation, filtration, membrane crossflow ultrafiltration of microfiltratioru decantation, flocculation, cell retention in the vessel by membrane devices or cell immobilization. Further, in case the production step is performed in continuous or semi-continuous mode and cells are continuously or intermittently fed from the growth vessel, the cell density hi the production vessel may be kept at a constant level by, for instance, harvesting an amount of cells from the production vessel corresponding to the amount of cells being fed from the growth vessel.<br>
hi connection with the above process, the produced Vitamin C contained in the so-called harvest stream is recovered/harvested from the production vessel. The harvest stream may include, for instance, cell-free or cell-containing aqueous solution coming from the production vessel, which contains Vitamin C as a result of the conversion of production substrate by the resting cells in the production vessel. Cells still present in the harvest stream may be separated from the Vitamin C by any operations known in the art, such as for instance filtration, centrifugation, decantation, membrane crossflow ultrafiltration or microfiltration, tangential flow ultrafiltration or microfiltration or dead end filtration. After this cell separation operation, the harvest stream is essentially free of cells.<br>
In a further aspect, the process of the present invention may be combined with further steps of separation and/or purification of the produced Vitamin C from other components contained in the harvest stream, i.e., so-called downstream processing steps. These steps may include any means known to a skilled person, such as, for' instance, concentration, crystallization, precipitation, adsorption, ion exchange,<br><br>
electrodialysis, bipolar membrane electrodialysis and/or reverse osmosis. Vitamin C may be further purified as the free acid form or any of its known salt forms by means of operations such as for instance treatment with activated carbon, ion exchange, adsorption and elution, concentration, crystallization, filtration and drying. Specifically, a first separation of Vitamin C from other components in the harvest stream might be performed by any suitable combination or repetition of, for instance, the following methods: two- or three-compartment electrodialysis, bipolar membrane electrodialysis, reverse osmosis or adsorption on, for instance, ion exchange resins or non-ionic resins. If the resulting form of Vitamin C is a salt of Vitamin C, conversion of the salt form into the free acid form may be performed by for instance bipolar membrane electrodialysis, ion exchange, simulated moving bed chromatographic techniques, and the like. Combination of the mentioned steps, e.g., electrodialysis and bipolar membrane electrodialysis into one step might be also used as well as combination of the mentioned steps e.g. several steps of ion exchange by using simulated moving bed chromatographic methods. Any of these procedures alone or in combination constitute a convenient means for isolating and purifying the product, i.e. Vitamin C. The product thus obtained may further be isolated in a manner such as, e.g. by concentration, crystallization, precipitation, washing and drying of the crystals and/or further purified by, for instance, treatment with activated carbon, ion exchange and/or re-crystallization.<br>
hi a preferred embodiment of the process, Vitamin C is purified from the harvest stream by a series of downstream processing steps as described above without having to be transferred to a non-aqueous solution at any time of this processing, i.e. all steps are performed in an aqueous environment. Such preferred downstream processing procedure may include for instance the concentration of the harvest stream coming from the production vessel by means of two- or three-compartment electrodialysis, conversion of Vitamin C hi its salt form present hi the concentrated solution into its acid form by means of bipolar membrane electrodialysis and/or ion exchange, purification by methods such as for instance treatment with activated carbon, ion exchange or non-ionic resins, followed by a further concentration step and crystallization. These crystals can be separated, washed and dried. If necessary, the crystals may be again re-solubilized in water, treated with activated carbon and/or ion exchange resins and recrystallized. These crystals can then be separated, washed and dried.<br>
According to the invention, host cells (in particular recombinant microorganisms from the genera ofGluconobacter, Gluconacetobacter and Acetobacter) carrying a<br><br>
SNDHai gene according to the invention and at least one genetically engineered gene selected from SMS or RCS as exemplified herein are able to directly produce Vitamin C from a suitable carbon source in significantly higher yield, productivity, and/or efficiency then other known organsisms, for example in quantities of 300 mg/1 or more or 800 mg/1 or more from D-sorbitol or L-sorbose, respectively when measured in a resting cell method after an incubation period of 20 hours. The yield of Vitamin C produced from D-sorbitol when measured in a resting cell method after an incubation period of 20 hours may even be as high as 400, 600,1000 mg/1 or even exceed 1.5, 2, 4,10,20, 50,100 g/1. The yield of Vitamin C produced from L-sorbose when measured in a resting cell method after an incubation period of 20 hours may even be as high as 1000 mgA or even exceed 1.5, 2, 4, 10, 20, 50, 100 g/1.<br>
The SNDHai protein shown in SEQ ID N0:2 and described herein performs an important function in the direct Vitamin C production in microorganisms, in particular in bacteria, such as acetic acid bacteria, such as Gluconobacter, Acetobacier and Gluconacetobacter, i.e. whose function compared to the wild type counterpart is enhanced or improved. This means that the direct production of Vitamin C is enhanced and/or increased and/or improved when a protein with SNDHai activity is expressed or preferably overexpressed in a particularly suitable host organism.<br>
Production of Vitamin C in such a host organism is even further improved when at least one polynucleotide encoding a protein selected from SMS or RCS systems is altered concurrently.<br>
The SNDHai protein may be encoded by a nucleotide sequence as shown in SEQ ID NO:1, which was isolated from G. oxydans DSM 17078 or by a polynucleotide that is substantially identical thereto<br>
In this context it should be mentioned that the expression of "a polynucleotide which is substantially identical" refers with respect to the SNDHai encoding sequence to a polynucleotide sequence selected from the group consisting of:<br>
a)	polynucleotides comprising the nucleotide sequence according to SEQ ID NO: 1<br>
b)	polynucleotides comprising a nucleotide sequence obtainable by nucleic acid<br>
amplification such as polymerase chain reaction, using genomic DNA from a<br>
microorganism as a template and a primer set according to SEQ ID NO:3 and<br>
SEQ ID NO:4;<br>
c)	polynucleotides comprising a nucleotide sequence encoding a fragment or<br>
derivative of a polypeptide comprising the amino acid sequence according to SEQ::<br>
ID NO:2 or encoding a fragment or derivative of a polypeptide encoded by a polynucleotide of any of (a) or (b) wherein in said derivative or fragment one or more amino acid residues are conservatively substituted compared to said polypeptide. and said fragment or derivative has the activity of a SNDHai polypeptide;<br>
d)	polynucleotides the complementary strand of which hybridizes under stringent<br>
conditions to a polynucleotide encoding a polypeptide comprising the amino acid<br>
sequence according to SEQ ID NO:2 or to a polynucleotide as defined in any one<br>
of (a) to (c) and which encode a SNDHai polypeptide; and<br>
e)	polynucleotides which are at least 70%, such as 85, 90 or 95% identical to a<br>
polynucleotide encoding a polypeptide comprising the amino acid sequence<br>
according to SEQ ID NO:2 or to a polynucleotide as defined in any one of (a) to<br>
(c) and which encode a SNDHai polypeptide<br>
or	_	-<br>
the complementary strand of such a polynucleotide.<br>
The polypeptide according to SEQ ID N0:2 was isolated from several different microorganisms according to the method described in example 22 and its annotated function was confirmed hi activity assays as described in examples 23 and 24.<br>
SNDHai activity is defined herein as the enzymatic activity that is able to convert L-Sorbosone directly to Ascorbic acid.<br>
A nucleic acid as defined above may be obtained by nucleic acid amplification using cDNA, mRNA or alternatively, genomic DNA, as a template and appropriate oligonucleotide primers such as the nucleotide primers according to SEQ ID NO:3 and SEQ ID N0:4 according to standard PCR amplification techniques. The nucleic acid thus amplified may be cloned into an appropriate vector and characterized by DNA sequence analysis.<br>
Further included are polynucleotide sequences coding for partial polypeptide sequences of a polypeptide which retains L-sorbosone dehydrogenase activity to produce Vitamin C from L-sorbosone such as, for example, polypeptides represented by SEQ ID NOs: 12, 14, 16, 18, 20, 22, and 27. The polypeptides comprise preferably partial amino acid sequences of at least 25 consecutive amino acids selected from'the amino acids sequences of the polypeptides disclosed in the present application. The person skilled in the art is aware of the fact that certain stretches in polypeptides are essential for the biological activity. There are, however, other areas wherein amino<br>
acids can be inserted, deleted or substituted by other amino acids preferably such ammo acids which are similar to the amino acids to be replaced.<br>
As used herein, "active fragment or derivative" means a polypeptide which retains essentially the same biological function or activity as the polypeptide shown in SEQ ID NO:2. Examples of biological activity may for instance be enzymatic activity, signaling activity or antibody reactivity. The term "same biological function" or "functional equivalent" as used herein means that the protein has essentially the same biological activity, e.g. enzymatic, signaling or antibody reactivity, as a polypeptide shown in SEQ ID N0:2.<br>
Metabolization of D-sorbitol or L-sorbose includes on one side the assimilation of ' these compounds into the cytosol and further conversion into metabolites useful for assimilation pathways such as the Embden-Meyerhof-Parnas pathway, the pentpse phosphate pathway, the Entner-Doudoroff pathway, and the tricarboxyclic acid, cycle, all of them involved in all vital energy-forming and anabolic reactions necessary for growth and maintenance of living cells. On the other side, metabolization ofD-sorbitol or L-sorbose also includes the conversion of these compounds into further oxidized products such as L-sorbosone, 2-KGA and Vitamin C by so-called incomplete oxidation processes.<br>
SMS proteins are herein defined as proteins involved in the Sorbitol/Sorbose Metabolisation System. Preferably, SMS proteins are selected from the group consisting of membrane-bound PQQ-dependent D-sorbitol dehydrogenase, membrane-bound L-sorbose dehydrogenase, membrane-bound L-sorbosone dehydrogenase, membrane-bound FAD-dependent D-sorbitol dehydrogenase, cytosolic NAD-dependent D-sorbitol dehydrogenase, NAD(P)-dependent D-sorbitol dehydrogenase (also called as NADPH-dependent sorbose reductase), NAD-dependent xylitol dehydrogenase, NAD-dependent alcohol dehydrogenase, membrane-bound L-sorbose dehydrogenase, NAD(P)H-dependent L-sorbose reductase, cytosolic NADP-dependent sorbosone dehydrogenase, cytosolic NAD(P)H-dependent L-sorbosone reductase, membrane-bound aldehyde dehydrogenase, cytosolic aldehyde dehydrogenase, glyceiol-3-phoph.ate dehydrogenase, glyceraldehyde-3-phosphate dehydrogenase.Even more preferred SMS proteins are selected from the family of oxidoreductases, more preferably oxidoreductases [EC 1] as recommended by the Nomenclature Committee of the International Union of Biochemistry and Molecular Biology (NC-IUBMB). Particularly preferred are oxidoreductases acting on the CH-OH group of donors [EC 1.1], in particular oxidoreductases withNAD+ orNADP*as acceptor [EC 1.1.1]and<br>
oxidoreductases with other acceptors [EC 1.1.99], or oxidoreductases acting on the aldehyde or oxo group of donors [EC 1.2], in particular oxidoreductases with NAD* or NADP+ as acceptor [EC 1.2.1]. Even more preferably, they are selected from dehydrogenases belonging to enzyme classes [EC 1.1.1.1], [EC 1.1.1.15] or [EC 1.2.1.-].<br>
The biological, enzymatic or other activity of SMS proteins can be measured by methods well known to a skilled person, such as, for example, by incubating a cell fraction containing the SMS protein in the presence of its substrate, electron acceptors) or donor(s) including phenazine methosulfate (PMS), dichlorophenol-indophenol (DCIP), NAD, NADH, NADP, NADPH, which consumption can be directly or indirectly measured by photometric, colorimetric or fluorimetric methods, and other inorganic components which might be relevant for the development of the activity. Thus, for example, the activity of membrane-bound D-sorbitol dehydrogenase can be measured in an assay where membrane fractions containing this enzyme are incubated in the.presence of phosphate buffer at pH 6, D-sorbitol and the artificial electron acceptors DCIP and PMS. The rate of consumption of DCIP can be measured at 600 nm, and is directly proportional to the D-sorbitol dehydrogenase activity present in the membrane fraction.<br>
Thus, for example, by increasing the activity of SMS proteins involved in incomplete oxidations of the D-sorbitol metabolization pathway, one can achieve increased conversion yields of D-sorbitol into products such as Vitamin C. In another example, by decreasing the activity of SMS proteins involved in the assimilation of D-sorbitol into the central metabolism, one can achieve increased conversion yields of D-sorbitol into products such as Vitamin C as well.<br>
RCS proteins are known to be important in the mechanism through which electrons generated by any oxidoreduction reaction in the cell are further transported, in general by means of a series of oxidoreduction reactions involving co-factors and oxidases, and a final electron acceptor.<br>
The main mechanism that living organisms use for producing energy necessary for vital activities is respiration. In higher organisms, carbohydrates, proteins, aliphatic acids are metabolised into acetyl-CoA by means of the glycolysis catabolic pathway and oxidation in cytoplasm. Acetyl-CoA is further metabolised through a series of reactions known as the citric acid cycle, which happens at the mitochondria. Energy resulting from these reactions is used for the production of reducing power, saved in the form of compounds such as FADH2 and NADH. These compounds are then used <br>
in the so-called electron transport chain, a series of oxido-reduction chain reactions involving different components localized in the mitochondria! inner membranes. The final electron acceptor is oxygen, which then reacts with the protons resulting from the reaction chain and forms water. The proton concentration gradient resulting from this process is the driving force of the ATP synthesis.<br>
In bacteria, this basic respiration process follows the same physiologic principle, but can occur in different ways, involving different components, intermediates, enzymatic complexes and final products. The efficiency of bacterial respiration processes can greatly vary, depending on the functional biological components expressed by each species, which in its turn depends on the genetic machinery available and on given growing conditions.<br>
As an example, acetic acid bacteria, which are obligate aerobe, gram-negative microorganisms belonging to the genus Acetobacter, Gluconobacter, and Gluconacetobacter, present peculiar characteristics in terms of energy generating processes. These bacteria are well known for their ability to incompletely oxidize different substrates such as alcohols, sugars, sugar alcohols and aldehydes. These processes are generally known as oxidative fermentations, and they have been well established for a long time in the food and chemical industry, especially in vinegar and in L-sorbose production. Useful products known to be obtained from incomplete oxidations using strains belonging to the Gluconobacter genus are 2-keto-L-gulonic acid (2-KGA) starting from D-sorbitol and L-sorbose, and 5-keto-D-gluconic acid, a precursor for the biosynthesis of D-tartaric acid, starting from D-glucose. Incomplete oxidations are the main mechanism of generation of energy for acetic acid bacteria. They accomplish these reactions by means of different dehydrogenases located either in the periplasmic space, on the periplasmic membrane as well as in the cytoplasm. Different co-factors are employed by the different dehydrogenases, the most common being PQQ and FAD for membrane-bound or periplasmic enzymes, and NAD/NADP for cytoplasmic enzymes. The electron transport chain of Gluconobacter/Gluconacetobacter and Acetobacter strains is known to include co-enzyme Q10 (CoQIO) and CoQ9 , respectively, as universal electron transport compound for all processes, as well as in some cases several kinds of cytochrome c elements. Gluconobacter strains are reported not to contain cytochrome c oxidase, but have other kinds of terminal oxidases, such as the bo type.<br>
In one preferred embodiment, the RCS proteins or subunits of proteins which are involved in the transport of electrons are selected from respiratory chain proteins, more preferably, they are selected from those functioning in the biosynthesis of<br>
cofactors, prosthetic groups or which function as carrier proteins, in particular proteins involved in the biosynthesis or maturation of cofactors and/or their precursors such as FAD, NAD, NADP, PQQ, ubiquinone including CoQ10, cytochromes a, b, c, d, and heme. Most preferably, they are selected from PQQ biosynthetic proteins such as'PQQ biosynthetic proteins A, B, C, D, E or from heme exporters such as CcmA or CcmB.<br>
The biological, enzymatic or other activity of RCS proteins can be measured by methods well known to a skilled person, such as, for example, by incubating a membrane fraction or cell-free extract containing the RCS protein in the presence of coenzyme Q2 (CoQ2), an artificial electron acceptor, and by measuring the consumption of oxygen by methods such as the Clark-type oxygen electrode (Rank Brothers, Cambridge, United Kingdom). Thus, for example, the activity of ubiquinol oxidase bd, a cyanide-resistant terminal oxidase, can be measured in an assay where membrane fractions or cell-free extracts containing this enzyme are incubated in the presence of 50 mM phosphate buffer at pH 6.5, 0.02% of the detergent Tween20 and 100 p.M cyanide in order to inactivate other cyanide-sensitive oxidases. The enzyme reaction can then be started by addition of 30 mM of the reduced artificial electron acceptor, CoQ2red, and followed by measuring the increase in absorbance at 275 nm. The rate of consumption of oxygen can be measured with help of the Clark-type electrode, and is directly proportional to the ubiquinol oxidase bd activity present in the membrane fraction or in the cell-free extract.<br>
Thus, for example, by modifying the RCS polynucleotide/proteins involved in the biosynthesis of terminal oxidases in such a form that they have enhanced activity, the overall efficiency of production of fermentation products depending on a series of dehydrogenation reactions, such as Vitamin C or 2-KGA, might be enhanced. In another example, by modifying the RCS polynucleotide/proteins involved in the biosynthesis of cofactors such as, for example, CoQ10 or cytochrome c, so that these cofactors are synthesized at a higher level, the overall capacity of the electron transfer system in bacteria which depend on those compounds as important elements of the respiratory chain can be enhanced, and have a positive impact on growth and production of fermentation compounds depending on oxidoreduction reactions, such as Vitamin C or 2-KGA. In still another example, the modification of the RCS polynucleotides/proteins involved in the biosynthesis of the cyanide-insensitive<br>
bypass oxidase (non-energy forming type) in such a way that its activity is enhanced,<br><br>
might have a positive impact on the overall capacity of production of fermentation<br>
/<br>
products by bacteria even at non-growing or at low overall metabolic activity stateThe Vitamin C production by direct fermentation can greatly be improved when a protein selected from RCS or SMS as disclosed herein is expressed or modified as described hereinafter in a microorganism that also expresses SNDHai. Such is herein also referred to as concurrent expression. If the concurrent expression is the result of a genetic manipulation event, this is also referred to as concurrent manipulation. This may be accomplished for instance in a microorganism that expresses SNDHai, such as recombinant SNDHai, the latter microorganisms are then referred to as recombinant microorganisms.<br>
Exemplified herein are also 7 novel SMS genes that may have an impact on the improvement of the production of Vitamin C in a microorganism that expresses SNDHai: Each of these genetically engineered genes may be used alone or in combination with at least one additional gene selected from the same group or from RCS.<br>
The 7 different genes encoding a SMS polypeptide comprising the amino acid, sequence according to SEQ ID NO: 125, SEQ ID NO: 129, SEQ ID NO: 133, SEQ ID NO: 45, SEQ ID NO: 137, SEQ ED NO: 141, SEQ ID NO: 145 respectively.<br>
Corresponding nucleotide sequences are shown in SEQ ID NO: 124, SEQ ED NO: 128, SEQ ID NO: 132, SEQ ED NO: 44, SEQ ED NO: 136, SEQ ID NO: 140, SEQ ID NO: 144 respectively, which were isolated from G. oxydans DSM 17078. The invention also encompasses polynucleotides which are substantially homologous to one of these sequences.<br>
In this context it should be mentioned that the expression of "a polynucleotide which is substantially homologous" with respect to the SMS encoding sequence refers to a polynucleotide sequence selected from the group consisting of:<br>
a)	polynucleotides comprising the nucleotide sequence according to SEQ ED NO:<br>
124, SEQ ID NO: 128, SEQ ID NO: 132, SEQ ED NO: 44, SEQ ID NO: 136, SEQ<br>
ED NO: 140, SEQ ID NO: 144 respectively<br>
b)	polynucleotides comprising a nucleotide sequence obtainable by nucleic acid<br>
amplification such as polymerase chain reaction, using genomic DNA from a<br>
microorganism as a template and a primer set according to SEQ ED NO: 126 and<br>
SEQ ED NO: 127 or SEQ ID NO: 130 and SEQ ED NO: 131 or SEQ ED NO: 134<br>
and SEQ ID NO: 135 or SEQ ID NO: 46 and SEQ ED NO: 47 or SEQ ED NO: 138<br>
and SEQ ID NO: 139 or SEQ ED NO: 142 and SEQ ED NO: 143 or SEQ ED NO:<br>
146 and SEQ ED NO: 147 respectively<br>
2)  polynucleotides comprising a nucleotide sequence encoding a fragment or<br>
derivative of apolypeptide comprising the amino acid sequence according to SEQ ID NO: 125, SEQ ID NO: 129, SEQ ID NO: 133, SEQ ID NO: 45, SEQ ID NO: 137, SEQ ED NO: 141, SEQ ID NO: 145 respectively or encoding a fragment or derivative of apolypeptide encoded by a polynucleotide of any of (a) or (b) wherein in said derivative one or more amino acid residues are conservatively substituted compared to said polypeptide, and said fragment or derivative has the activity of a SMS Protein;<br>
d)	polynucleotides the complementary strand of which hybridizes under stringent<br>
conditions to a polynucleotide encoding a polypeptide comprising the amino acid<br>
sequence according to SEQ ID NO: 125, SEQ ID NO: 129, SEQ ID NO: 133,<br>
SEQ ID NO: 45, SEQ ID NO: 137, SEQ ID NO: 141, SEQ ID NO: 145<br>
respectively or to a polynucleotide as defined in any one of (a) to (c) and which<br>
encode a SMS Protein; and<br>
e)	polynucleotides which are at least 70%, such as 85, 90 or 95% homologous to a<br>
polynucleotide encoding a polypeptide comprising the amino acid sequence<br>
according to SEQ ID NO: 125, SEQ ID NO: 129, SEQ ID NO: 133, SEQ ID NO:<br>
45, SEQ ID NO: 137, SEQ ID NO: 141, SEQ ID NO: 145 respectively or to a<br>
polynucleotide as defined in any one of (a) to (c) and which encode a SMS<br>
Protein;<br>
or<br>
the complementary strand of such a polynucleotide.<br>
The nucleotide and amino acid sequences determined above were used as a "query sequence" to perform a search with Blast2 program (version 2 or BLAST from National Center for Biotechnology [NCBFj against the database PRO SW-SwissProt (full release plus incremental updates). Gene SMS 02 (SEQ ID NO: 124) was annotated as encoding a protein showing similarity to NAD(P)-dependent D-sorbitol dehydrogenase of Bacillus subtilis (SEQ ID NO: 125). Gene SMS 03 (SEQ ID NO:. 128) was annotated as encoding a protein showing similarity to NAD(P)-dependent sorbitol dehydrogenase of Bacillus subtilis (SEQ ID NO: 129). Gene SMS 04 (SEQ ED NO: 132) was annotated as encoding NAD(P)H-dependent L-sorbose reductase (SEQ ID NO: 133). Gene SMS 05 (SEQ ID NO: 44) was annotated as encoding NAD(P)-dependent sorbosone dehydrogenase (SEQ ID NO: 45). Gene SMS 12 (SEQ ID NO: 136) was annotated as encoding membrane-bound L-sorbose dehydrogenase (SDH) (SEQ ID NO: 137). Gene SMS 13 (SEQ ID NO: 140) was annotated as encoding subunit A of membrane-bound PQQ-dependent D-sorbitol dehydrogenase (SEQ ID NO: 141). Gene SMS 14 (SEQ TD NO: 144) was annotated as encoding.....subunit B of membrane-bound PQQ-dependent D-sorbitol dehydrogenase (SEQ ID NO: 145).<br>
A nucleic acid encoding an SMS protein according to the invention may be obtained from any suitable organism by nucleic acid amplification using cDNA, mlRNA or alternatively, genomic DNA, as a template and appropriate oligonucleotide primers such as the nucleotide primer pairs according to SEQ ID NO: 126 and SEQ ID NO: 127 or SEQ ID NO: 130 and SEQ ID NO: 131 or SEQ ID NO: 134 and SEQ ID NO: 135 or SEQ ID NO: 46 and SEQ ID NO: 47 or SEQ ID NO: 138 and SEQ ID NO: 139 or SEQ ID NO: 142 and SEQ ID NO: 143 or SEQ ID NO: 146 and SEQ ID NO: 147 respectively according to standard PCR amplification techniques. The nucleic acid thus amplified may be cloned into an appropriate vector and characterized by DNA sequence analysis.<br>
The different SEQ ID NO's related to the SMS proteins/genes are summarized,below:<br>
Protein/Gene           SEQ ID NO's          SEQ ID NO's of          SEQ ID NO's of<br>
Polynucleotide Seq.	Amino Acid Seq.	Primers<br>
(Table Removed)<br>
Exemplified herein are also 5 novel RCS genes that may have an impact on the improvement of the production of Vitamin C in a microorganism that expresses SNDHai: Each of these genetically engineered genes may be used alone or in combination with at least one additional gene selected from the same group or from SMS.<br>
The 5 different genes encoding a RCS polypeptide comprising the amino acid sequence according to SEQ ID NO: 181, SEQ ED NO: 185, SEQ ID NO: 189, SEQ ID NO: 193 , SEQ ID NO: 197, respectively.<br>
Corresponding nucleotide sequences are shown in SEQ ID NO: 180, SEQ ID NO: 184, SEQ ID NO: 188, SEQ ID NO: 192 , SEQ ID NO: 196, respectively, which were<br>
isolated from G. oxydans DSM 17078. The invention also encompasses polynucleotides which are substantially homologous to one of these sequences.<br>
In this context it should be mentioned that the expression of "a polynucleotide which is substantially homologous" with respect to the RCS encoding sequence refers to a polynucleotide sequence selected from the group consisting of:<br>
a)	polynucleotides comprising the nucleotide sequence according to SEQ ID NO:<br>
180, SEQ ID NO: 184, SEQ ID NO: 188, SEQ ID NO: 192, SEQ ID NO: 196,<br>
respectively<br>
b)	polynucleotides comprising a nucleotide sequence obtainable by nucleic acid<br>
amplification such as polymerase chain reaction, using genomic DNA from a<br>
microorganism as a template and a primer set according to SEQ ID NO: 182 and<br>
SEQ ID NO: 183 or SEQ ID NO: 186 and SEQ ID NO: 187 or SEQ ID NO: 190<br>
and SEQ ID NO: 191 or SEQ ID NO: 194 and SEQ ID NO: 195 or SEQ ID;NO:<br>
198 and SEQ ID NO: 199 respectively<br>
c)	polynucleotides comprising a nucleotide sequence encoding a fragment of<br>
derivative of a polypeptide comprising the amino acid sequence according to SEQ<br>
ID NO: 181,SEQ ID NO: 185, SEQ ID NO: 189, SEQ ID NO: 193 ,SEQ ID NO:<br>
197, respectively or encoding a fragment or derivative of a polypeptide encoded<br>
by a polynucleotide of any of (a) or (b) wherein in said derivative one or more<br>
amino acid residues are conservatively substituted compared to said polypeptide,<br>
and said fragment or derivative has the activity of a RCS Protein;<br>
d)	polynucleotides the complementary strand of which hybridizes under stringent<br>
conditions to a polynucleotide encoding a polypeptide comprising the amino acid<br>
sequence according to SEQ ID NO: 181, SEQ ID NO: 185, SEQ ID NO: 189,<br>
SEQ ID NO: 193 , SEQ ID NO: 197, respectively or to a polynucleotide as<br>
defined in any one of (a) to (c) and which encode a RCS Protein; and<br>
e)	polynucleotides which are at least 70%, such as 85, 90 or 95% homologous to a<br>
polynucleotide encoding a polypeptide comprising the amino acid sequence<br>
according to SEQ ID NO: 181, SEQ ID NO: 185, SEQ ID NO: 189, SEQ ID NO:<br>
193 , SEQ ID NO; 197, respectively or to a polynucleotide as defined in any one<br>
of (a) to (c) and which encode a RCS Protein;<br>
or<br>
the complementary strand of such a polynucleotide.<br>
The nucleotide and amino acid sequences determined above were used as a "query sequence" to perform a search with Blast2 program (version 2 or BLAST from National Center for Biotechnology [NCBI] against the database PRO SW-SwissProt<br>
 (full release plus incremental updates). Gene RCS 21 (SEQ ID NO: 180) was annotated as encoding coenzyme PQQ biosynthesis protein A (SEQ ID NO: 181). Gene RCS 22 (SEQ ID NO: 184) was annotated as encoding coenzyme PQQ biosynthesis protein B (SEQ ID NO: 185). Gene RCS 23 (SEQ ID NO: 188) was annotated as encoding coenzyme PQQ biosynthesis prqtein C (SEQ ID NO: 189). Gene RCS 24 (SEQ ID NO: 192) was annotated as encoding coenzyme PQQ biosynthesis protein D (SEQ ID NO: 193). Gene RCS 25 (SEQ ID NO: 196) was annotated as encoding coenzyme PQQ biosynthesis protein E (SEQ ID NO: 197).<br>
A nucleic acid encoding an RCS protein according to the invention may be obtained from any suitable organism by nucleic acid amplification using cDNA, mRNA or alternatively, genomic DNA, as a template and appropriate oligonucleotide primers such as the nucleotide primer pairs according to SEQ ID NO: 182 and SEQ ID NO; 183 or SEQ ID NO: 186 and SEQ ID NO: 187 or SEQ ID NO: 190 and SEQ ID NO 191 or SEQ ID NO: Wand SEQ ID NO: 195 or SEQ ID NO: 198 and SEQ ID NO: 199 respectively according to standard PCR amplification techniques. The nucleic acid thus amplified may be cloned into an appropriate vector and characterized by DNA sequence analysis.<br>
The different SEQ ID NO's related to the RCS proteins/genes are summarized below:<br>
Protein/Gene            SEQ ID NO's          SEQ ID NO's of         SEQ ID NO's of<br>
Polynucleotide Seq.	Amino Acid Seq.	Primers<br>
(Table Removed)<br>
Before the concurrent expression will be illustrated further the construction and expression of a single gene/polynucleotide and the alteration in the genome of the host cell as exemplified above will be described in more detail on the basis of SNDHai. Unless otherwise indicated, the description is also applicable for the construction and expression of RCS and SMS genes disclosed herein.<br>
A wide variety of host/cloning vector combinations may be employed in cloning the double stranded DNA. Preferred vectors for the expression of the genes of the present invention, i.e. the SNDHai gene, in E. coll may be selected from any vectors usually used in E. coli, such as for instance p' QE vectors which can express His-<br>
tagged recombinant proteins (QIAGEN AG Switzerland), pBR322 or its derivatives including for instance pUCIS and pBluescript II (Stratagene Cloning Systems, Calif., USA), pACYCl 77 and pACYC184 and their derivatives, and a vector derived from a broad host range plasmid such as RK2 and RSF1010. A preferred vector for the expression of the nucleotide sequence of the present invention in bacteria including Gluconobacter, Gluconacetobacter, Acetobacter, and Pseudomonas is selected from any vectors which can replicate in Gluconobacter, Acetobacter, or Pseudomonas as well as in a preferred cloning organism such as E. coli. The preferred vector is a broad-host-range vector such as for instance a cosrnid vector like pVKl00 and its derivatives and RSF1010. Copy number and stability of the vector should be carefully considered for stable and efficient expression of the cloned gene and also for efficient cultivation of the host cell carrying the cloned gene. Nucleic acid molecules containing for instance transposable elements such as Tn5 may also be used as a -vector to introduce the desired gene into the preferred host, especially on a chromosome. Nucleic acid molecules containing any DNAs isolated from the preferred host together with a the SNDHai gene of the present invention maygbe also useful to introduce this gene into the preferred host cell, especially on a chromosome. Such nucleic acid molecules may be transferred to the preferred host by applying any of conventional methods, e.g., transformation, transduction, conjugal mating or electroporation, which are well known in the art, considering the nature of the host cell and the nucleic acid molecule.<br>
The L-sorbosone dehydrogenase gene/nucleotide sequences may be ligated into a suitable vector containing a regulatory region such as for instance a promoter, a ribosomal binding site, and a transcriptional terminator operable in the host cell described above with a well-known method in the art to produce an expression vector.<br>
The polypeptides and polynucleotides as exemplified herein are preferably provided in an isolated form, and preferably are purified to homogeneity.<br>
The term "isolated" means that the material is removed from its original environment (e.g., the natural environment if it is naturally occurring). For example, a naturally-occurring polynucleotide or polypeptide present in a living microorganism is not isolated, but the same polynucleotide or polypeptide, separated from some or all of the coexisting materials in the natural system, is isolated. Such polynucleotides could be part of a vector and/or such polynucleotides or polypeptides could be part of a composition and still be isolated in that such vector or composition is not part of its natural environment.<br>
An isolated polynucleotide or nucleic acid as used herein may be a DNA or RNA that is not immediately contiguous with both of the coding sequences with which it is immediately contiguous (one on the 5'-end and one on the 3'-end) in the naturally' occurring genome of the organism from which it is derived. Thus, in one embodiment, a nucleic acid includes some or all of the 5'-non-coding (e.g., promoter) sequences that are immediately contiguous to the coding sequence. The term "isolated polynucleotide" therefore includes, for example, a recombinant DNA that is incorporated into a vector, into an autonomously replicating plasmid or virus, or into the genomic DNA of a prokaryote or eukaryote, or which exists as a separate molecule (e.g., a cDNA or a genomic DNA fragment produced by PCR or restriction endonuclease treatment) independent of other sequences. It also includes a recombinant DNA that is part of a hybrid gene encoding an additional polypeptide that is substantially free of cellular material, viral material, or culture medium (when produced by recombinant DNA techniques), or chemical precursors or other chemicals (when chemically synthesized). Moreover, an "isolated nucleic acid fragment" is a nucleic acid fragment that is not naturally occurring as a fragment and would not be found hi the natural state.<br>
As used herein, the terms "polynucleotide", "gene" and "recombinant gene" refer to nucleic acid molecules which may be isolated from chromosomal DNA, which include an open reading frame encoding a protein, e.g. G. oxydans DSM 17078 SNDHai proteins. A polynucleotide may include a polynucleotide sequence as shown in SEQ ID N0:l or fragments thereof and regions upstream and downstream of the gene sequences which may include, for example, promoter regions, regulator regions and terminator regions important for the appropriate expression and stabilization of the polypeptide derived thereof.<br>
A gene may include coding sequences, non-coding sequences such as for instance untranslated sequences located at the 3'- and 5'-ends of the coding region of a gene, and regulatory sequences. Moreover, a gene refers to an isolated nucleic acid molecule as defined herein. It is furthermore appreciated by the skilled person that DNA sequence polymorphisms that lead to changes in the amino acid sequences of SNDHai proteins may exist within a population, e.g., the Gluconobacter oxydans population. Such genetic polymorphism in the SNDHai gene may exist among individuals within a population due to natural variation or in cells from different populations. Such natural variations can typically resultiin 1-5% variance in the          nucleotide sequence of the SNDHai gene. Any and all such nucleotide variations and the resulting ammo acid polymorphism in SNDHai are the result of natural variation<br>
and that do not alter the functional activity of SNDHai proteins are intended to be within the scope of the invention.<br>
As used herein, the terms "polynucleotide" or "nucleic acid molecule" are intended to include DNA molecules (e.g., cDNA or genomic DNA) and RNA molecules (e.g., mRNA) and analogs of the DNA or RNA generated using nucleotide analogs. The nucleic acid molecule may be single-stranded or double-stranded, but preferably is double-stranded DNA. The nucleic acid may be synthesized using oligonucleotide analogs or derivatives (e.g., inosine or phosphorothioate nucleotides). Such oligonucleotides may be used, for example, to prepare nucleic acids that have altered base-pairing abilities or increased resistance to nucleases.<br>
The sequence information as provided herein should not be so narrowly construed as to require inclusion of erroneously identified bases. The specific sequences disclosed herein may be readily used to isolate the complete gene from a recombinant or non-recombinant microorganism capable of converting a given carbon source directly into Vitamin C, in particular Gluconobacter oxydans, preferably Gluconobacter dxydans DSM 17078 which in turn may easily be subjected to further sequence analyses thereby identifying sequencing errors.<br>
Unless otherwise indicated, all nucleotide sequences determined by sequencing a DNA molecule herein were determined using an automated DNA sequencer and all amino acid sequences of polypeptides encoded by DNA molecules determined herein were predicted by translation of a DNA sequence determined as above. Therefore, as is known in the art for any DNA sequence determined by this automated approach, any nucleotide sequence determined herein may contain some errors. Nucleotide sequences determined by automation are typically at least about 90% identical, more typically at least about 95% to at least about 99.9% identical to the actual nucleotide sequence of the sequenced DNA molecule. The actual sequence may be more precisely determined by other approaches including manual DNA sequencing methods well known in the art. As is also known in the art, a single insertion or deletion in a determined nucleotide sequence compared to the actual sequence will cause a frame shift in translation of the nucleotide sequence such that the predicted amino acid sequence encoded by a determined nucleotide sequence will be completely different from the amino acid sequence actually encoded by the sequenced DNA molecule, beginning at the point of such an insertion or deletion.<br>
The person skilled in the art is capable of identifying such erroneously identified bases and knows how to correct for such errors.<br>
A nucleic acid molecule according to the invention may comprise only a portion or a fragment of the nucleic acid sequence provided by the present invention, such as for instance the sequence shown in SEQ ID NO: 1, for example a fragment which may be used as a probe or primer such as for instance SEQ ID N0:3 or SEQ ID NO:4 or a fragment encoding a portion of a protein according to the invention. The nucleotide sequence determined from the cloning of the SNDHai gene allows for the generation of probes and primers designed for use in identifying and/or cloning other SNDHai family members, as well as SNDHai homologues from other species. The probe/primer typically comprises substantially purified oligonucleotides which typically comprises a region of nucleotide sequence that hybridizes preferably under highly stringent conditions to at least about 12 or 15, preferably about 18 or 20, more preferably about 22 or 25, even more preferably about 30, 35,40,45, 50, 55, 60, 65, or 75 or more consecutive nucleotides of a nucleotide sequence shown in SEQ ID NO: 1 or a fragment or derivative thereof.<br>
A nucleic acid molecule encompassing all or a portion of the nucleic acid sequence of SEQ ED NO:1 may be isolated by the polymerase chain reaction (PCR) using synthetic oligonucleotide primers designed based upon the sequence information contained herein.<br>
A nucleic acid of the invention may be amplified using cDNA, mRNA or alternatively, genomic DNA, as a template and appropriate oligonucleotide primers according to standard PCR amplification techniques. The nucleic acid thus amplified may be cloned into an appropriate vector and characterized by DNA sequence analysis.<br>
Fragments of a polynucleotide may also comprise polynucleotides not encoding functional polypeptides. Such polynucleotides may function as probes or primers for a PCR reaction.<br>
Nucleic acids irrespective of whether they encode functional or non-functional polypeptides, may be used as hybridization probes or polymerase chain reaction (PCR) primers. Uses of the nucleic acid molecules of the present invention that do not encode a polypeptide having a SNDHai activity include, inter alia, (1) isolating the gene encoding the protein of the present invention, or allelic variants thereof from a cDNA library, e.g., from other organisms than Gluconobacter oxydans and (2) Northern blot analysis for detecting expression of mRNA of said protein in specific cells or (3) use in enhancing and/or improving the function or activity of homologous SNDHai genes in said other organisms.<br><br>
Probes based on the nucleotide sequences provided herein may be used to detect transcripts or genomic sequences encoding the same or homologous proteins for instance in other organisms. Nucleic acid molecules corresponding to natural variants and non-G. oxydans homologues of the G. oxydans SNDHai DNA of the invention which are also embraced by the present invention may be isolated based on their homblogy to the G. oxydans SNDHai nucleic acid disclosed herein using the G. oxydans DNA, or a portion thereof, as a hybridization probe according to standard hybridization techniques, preferably under highly stringent hybridization conditions.<br>
In preferred embodiments, the probe further comprises a label group attached thereto, e.g., the label group can be a radioisotope, a fluorescent compound, an enzyme, or an enzyme cofactor.<br>
Homologous or substantially identical gene sequences may be isolated, for example, by performing PCR using two degenerate oligonucleotide primer pools designedson the basis of nucleotide sequences as taught herein.<br>
The template for the reaction may be cDNA obtained by reverse transcription of mRNA prepared from strains known or suspected to express a polynucleotide according to the invention. The PCR product may be subcloned and sequenced to ensure that the amplified sequences represent the sequences of a new nucleic acid sequence as described herein, or a functional equivalent thereof.<br>
The PCR fragment may then be used to isolate a full length cDNA clone by a variety of known methods. For example, the amplified fragment may be labeled and used to screen a bacteriophage or cosmid cDNA library. Alternatively, the labeled fragment may be used to screen a genomic library.<br>
PCR technology can also be used to isolate full-length cDNA sequences from other organisms. For example, RNA may be isolated, following standard procedures, from an appropriate cellular or tissue source. A reverse transcription reaction may be   -performed on the RNA using an oligonucleotide primer specific for the most 5'-end of the amplified fragment for the priming of first strand synthesis.<br>
The resulting RNA/DNA hybrid may then be "tailed" (e.g., with guanines) using a standard terminal transferase reaction, the hybrid may be digested with RNaseH, and second strand synthesis may then be primed (e.g., with a poly-C primer). Thus, cDNA sequences upstream of the amplified fragment may easily be isolated. For a review of useful cloning strategies, see e.g., Sambrook et al., supra; and Ausubel et al., supra.<br><br>
Homologues, substantially identical sequences, functional equivalents, and orthologs of the genes exemplified herein, such as the SNDHai gene according to SEQ ID NO:1 may be obtained from a number of different microorganisms. The procedures for the isolation of specific genes and/or fragments thereof are exemplified herein. Following these procedures, SNDHai genes have successfully been isolated from Gluconobacter oxydans IFO 3292, Gluconobacter oxydans IFO 3287, Acetobacter sp. ATCC 15164 and Gluconobacter oxydans IFO 3244. Accordingly, nucleic acids encoding other SNDHai family members, which thus have a nucleotide sequence that differs from a nucleotide sequence according to SEQ ID NO: 1, are within the scope of the invention. Moreover, nucleic acids encoding SNDHai proteins from different species which thus have a nucleotide sequence which differs from a nucleotide sequence shown in SEQ ID NO: 1 are within the scope of the invention.<br>
The invention also discloses an isolated polynucleotide hybridisable under stringent conditions, preferably under highly stringent conditions, to a polynucleotide according to the present invention, such as for instance a polynucleotide shown in SEQID NO: 1. Advantageously, such polynucleotide may be obtained from a microorganism capable of converting a given carbon source directly into Vitamin C, in particular Gluconobacter oxydans, preferably Gluconobacter oxydans DSM 17078.<br>
As used herein, the term "hybridizing" is intended to describe conditions for hybridization and washing under which nucleotide sequences at least about 50%, at least about 60%, at least about 70%, more preferably at least about 80%, even more preferably at least about 85% to 90%, most preferably at least 95% homologous to each other typically remain hybridized to each other.<br>
In one embodiment, a nucleic acid of the invention is at least 40%, 45%, 50%, 55%, 60%, 65%, 70%, 75%, 80%, 85%, 90%, 91%, 92%, 93%, 94%, 95%, 96%, 97%, 98%, 99%, or more homologous to a nucleic acid sequence shown in SEQ ID NO:1 or the complement thereof.<br>
A preferred, non-limiting example of such hybridization conditions are hybridization in 6x sodium chloride/sodium citrate (SSC) at about 45 °C, followed by one or more washes in Ix SSC, 0.1% SDS at 50°C, preferably at 55°C, more preferably at 60°C and even more preferably at 65 °C .<br>
Highly stringent conditions include, for example, 2 h to 4 days incubation at 42°C using a digoxigenin (DIG)-labeled DNA probe (prepared by using a DIG labeling system; Roche Diagnostics GmbH, 68298 Mannheim, Germany) in a solution such as.<br><br>
DigEasyHyb solution (Roche Diagnostics GmbH) with or without 100 (ig/ml salmon sperm DNA, or a solution comprising 50% formamide, 5x SSC (150 mM Nad, 15 mM trisodium citrate), 0.02% sodium dodecyl sulfate, 0.1% N-lauroylsarcosine, and 2% blocking reagent (Roche Diagnostics GmbH), followed by washing the filters twice for 5 to 15 minutes in 2x SSC and 0.1% SDS at room temperature and then washing twice for 15-30 minutes in 0.5x SSC and 0.1% SDS or O.lx SSC and 0.1% SDS at 65-68°C.<br>
Preferably, an isolated nucleic acid molecule of the invention that hybridizes under preferably highly stringent conditions to a nucleotide sequence of the invention corresponds to a naturally-occurring nucleic acid molecule. As used herein, a "naturally-occurring" nucleic acid molecule refers to an RNA or DNA molecule having a nucleotide sequence that occurs in nature (e.g., encodes a natural protein). In one embodiment, the nucleic acid encodes a natural G. oxydans SNDHai protein.<br>
The skilled artisan will know which conditions to apply for stringent and highly stringent hybridization conditions. Additional guidance regarding such conditions is readily available in the art, for example, in Sambrook et al., 1989, Molecular Cloning, A Laboratory Manual, Cold Spring Harbor Press, N. Y.; and Ausubel et al. (eds.), 1995, Current Protocols in Molecular Biology, (John Wiley &amp; Sons, N. Y.). Of course, a polynucleotide which hybridizes only to a poly (A) sequence (such as the 3'-terminal poly (A) tract of mRNAs), or to a complementary stretch of T (or U) residues, would not be included in a polynucleotide of the invention used to specifically hybridize to a portion of a nucleic acid of the invention, since such a ' polynucleotide would hybridize to any nucleic acid molecule containing a poly (A) stretch or the complement thereof (e.g., practically any double-stranded cDNA clone).<br>
In a typical approach, genomic DNA or cDNA libraries constructed from other organisms, e.g. microorganisms capable of converting a given carbon source such as D-Sorbitol, L-Sorbose or L-Sorbosone, directly into Vitamin C, in particular other Gluconobacter species may be screened.<br>
For example, Gluconobacter strains may be screened for homologous polynucleotides by Northern blot analysis. Upon detection of transcripts homologous to polynucleotides according to the invention, DNA libraries may be constructed from RNA isolated from the appropriate strain, utilizing standard techniques well known to those of skill in the art. Alternatively, a total genomic DNA library may be screened using a probe hybridisable to a polynucleotide according to the invention.<br><br>
A nucleic acid molecule of the present invention, such as for instance a nucleic acid molecule shown in SEQ ID NO:1 or a fragment or derivative thereof, may be isolated using standard molecular biology techniques and the sequence information provided herein. For example, using all or portion of the nucleic acid sequence shown in SEQ ID N0:l as a hybridization probe, nucleic acid molecules according to the invention may be isolated using standard hybridization and cloning techniques (e.g., as described in Sambrook, J., Fritsh, E. F., and Maniatis, T. Molecular Cloning : A Laboratory Manual. 2nd, ed., Cold Spring Harbor Laboratory, Cold Spring Harbor Laboratory Press, Cold Spring Harbor, NY, 1989).<br>
Furthermore, oligonucleotides corresponding to or hybridisable to nucleotide sequences according to the invention may be prepared by standard synthetic techniques, e.g., using an automated DNA synthesizer.<br>
The terms "homology", "identically" or "percent identity" are used interchangeably herein. For the purpose of this invention, it is defined here that in order to determine the percent identity of two amino acid sequences or of two nucleic acid sequences, the sequences are aligned for optimal comparison purposes (e.g., gaps may be introduced in the sequence of a first amino acid or nucleic acid sequence for optimal alignment with a second amino or nucleic acid sequence). The amino acid residues or nucleotides at corresponding amino acid positions or nucleotide positions are then compared. When a position in the first sequence is occupied by the same amino add residue or nucleotide as the corresponding position in the second sequence, then the molecules are identical at that position. The percent identity between the two sequences is a function of the number of identical positions shared by the sequences (i.e., % identity = number of identical positions/total number of positions (i.e., overlapping positions) x 100). Preferably, the two sequences are the same length.<br>
The skilled person will be aware of the fact that several different computer programs are available to determine the homology between two sequences. For instance, a comparison of sequences and determination of percent identity between two sequences may be accomplished using a mathematical algorithm. In a preferred embodiment, the percent identity between two amino acid sequences is determined using the Needleman and Wunsch (J. Mol. Biol. (48): 444-453 (1970) ) algorithm which has been incorporated into the GAP program in the GCG software package (available at http://www.accelrys.com), using either a Blossom 62 matrix or a PAM250 matrix, and a gap weight of 16, 14, 12, 10, 8, 6 or 4 and a length weight of 1, 2, 3,4, 5 or 6. The skilled person will appreciate that all these different parameters<br><br>
will yield slightly different results but that the overall percentage identity of two sequences is not significantly altered when using different algorithms.<br>
In yet another embodiment, the percent identity between two nucleotide sequences is determined using the GAP program in the GCG software package (available at http://www.accelrys.com), using a NWSgapdna.CMP matrix and a gap weight of 40, 50, 60, 70 or 80 and a length weight of 1, 2, 3,4, 5 or 6. In another embodiment, the percent identity between two amino acid or nucleotide sequences is determined using the algorithm of E. Meyers and W. Miller (CABIOS, 4: 11-17 (1989) which has been incorporated into the ALIGN program (version 2.0) (available at http://vega.igh.cnrs.fr/bm/align-guess.cgi) using a PAM120 weight residue table, a gap length penalty of 12 and a gap penialty of 4.<br>
The nucleic acid and protein sequences of the present invention may further be used as a "query sequence" to perform a search against public databases to, for example, identify other family members or related sequences. Such searches may be performed using the BLASTN and BLASTX programs (version 2.0) of Altsch.ul, et al. (1990) J. Mol. Biol. 215:403-10. BLAST nucleotide searches may be performed with the BLASTN program, score = 100, word length = 12 to obtain nucleotide sequences homologous to the nucleic acid molecules of the present invention. BLAST protein searches may be performed with the BLASTX program, score = 50, word length - 3 to obtain amino acid sequences homologous to the protein molecules of the present invention. To obtain gapped alignments for comparison purposes, Gapped BLAST may be utilized as described in Altschul et al., (1997) Nucleic Acids Res. 25 (17): 3389-3402. When utilizing BLAST and Gapped BLAST programs, the default parameters of the respective programs (e.g., BLASTX and BLASTN) may be used. See http://www.ncbi.nim.nih.gov.<br>
In another embodiment, an isolated nucleic acid molecule of the invention comprises a nucleic acid molecule which is the complement of a nucleotide sequence as of the present invention, such as for instance the sequence shown in SEQ ID NO: 1. A nucleic acid molecule, which is complementary to a nucleotide sequence disclosed herein, is one that is sufficiently complementary to a nucleotide sequence shown in SEQ ID NO:1 such that it may hybridize to said nucleotide sequence thereby forming a stable duplex.<br>
In a further embodiment, a nucleic acid of the invention as shown in SEQ ID NO:1 or the complement thereof contains at least one mutation leading to a gene product with modified function/activity. The at least one mutation may be introduced by methods!<br><br>
known in the art or described herein. In regard to SNDHai, the at least one mutation leads to a SNDHai protein whose function compared to the wild type counterpart is enhanced or improved. The activity of the SNDHai protein is thereby increased. Methods for introducing such mutations are well known in the art.<br>
Cells with an increased SNDHai activity are preferred.since such cells will produce more Vitamin C particularly when any of the other genes selected from SMS or RCS genes is genetically altered as described herein.<br>
Another aspect pertains to vectors, containing a nucleic acid encoding a protein according to the invention or a functional equivalent or portion thereof. As used herein, the term "vector" refers to a nucleic acid molecule capable of transporting another nucleic acid to which it has been linked. One type of vector is a "plasmid", which refers to a circular double stranded DNA loop into which additional DNA segments may be ligated. Another type of vector is a viral vector, wherein additional DNA segments may be ligated into the viral genome. Certain vectors are capable of autonomous replication in a host cell into which they are introduced (e.g., bacterial vectors having a bacterial origin of replication). Other vectors are integrated into the genome of a host cell upon introduction into the host cell, and thereby are replicated -along with the host genome.<br>
Moreover, certain vectors are capable of directing the expression of genes to which they are operatively linked. Such vectors are referred to herein as "expression vectors". In general, expression vectors of utility in recombinant DNA techniques are often in the form of plasmids. The terms "plasmid" and "vector" can be used interchangeably herein as the plasmid is the most commonly used form of vector. However, the invention is intended to include such other forms of expression vectors, such as viral vectors (e.g., replication defective retroviruses, adenoviruses and adeno-associated viruses), which serve equivalent functions<br>
The recombinant vectors of the invention comprise a nucleic acid of the invention in a form suitable for expression of the nucleic acid in a host cell, which means that the recombinant expression vector includes one or more regulatory sequences, selected on the basis of the host cells to be used for expression, which is operatively linked to the nucleic acid sequence to be expressed. Within a recombinant expression vector, "operatively linked11 is intended to mean that the nucleotide sequence of interest is linked to the regulatory sequence(s) in a manner which allows for expression of the nucleotide sequence (e.g., in an in vitro transcription/translation system or in a host cell when the vector is introduced into the host cell). The term "regulatory sequence"::<br><br>
ers and other expression control elements (e.g., attenuator). Such regulatory sequences are described, for example, in Goeddel; Gene Expression Technology: Methods hi Enzymology 185, Academic Press, San Diego, CA (1990). Regulatory sequences include those which direct constitutive or inducible expression of a nucleotide sequence in many types of host cells and those which direct expression of the nucleotide sequence only in a certain host cell (e.g. tissue-specific regulatory sequences). It will be appreciated by those skilled in the art that the design of the expression vector can depend on such factors as the choice of the host cell to be transformed, the level of expression of protein desired, etc. The expression vectors of the invention may be introduced into host cells to thereby produce proteins or peptides, encoded by nucleic acids as described herein, including, but not limited to, mutant proteins, fragments thereof, variants or functional equivalents thereof, and fusion proteins, encoded by a nucleic acid as described herein, e.g., SNDHai proteins, mutant forms of SNDHai proteins, fusion proteins and the like.<br>
Functional equivalents of polypeptides as exemplified herein may be also part of the present invention and are defined on the basis of the amino acid sequences of the present invention by addition, insertion, deletion and/or substitution of one or more amino acid residues of such sequences wherein such derivatives preferably still have the L-sorbosone dehydrogenase activity measured by an assay known in the art or specifically described herein. Such functional derivatives may be made either by chemical peptide synthesis known in the art or by recombinant techniques on the basis of the DNA sequences as disclosed herein by methods known in the state of the art. Amino acid exchanges in proteins and peptides which do not generally alter the   -activity of such molecules are known.<br>
The recombinant expression vectors of the invention may be designed for expression of SNDHai proteins in a suitable microorganism. For example, a protein according to the invention may be expressed in bacterial cells such as strains belonging to the genera Gluconobacter, Gluconacetobacter or Acetobacter. Expression vectors useful in the present invention include chromosomal-, episomal- and virus-derived vectors e.g., vectors derived from bacterial plasmids, bacteriophage, and vectors derived from combinations thereof, such as those derived from plasmid and bacteriophage genetic elements, such as cosmids and phagemids.<br>
The DNA insert may be operatively linked to an appropriate promoter, which may be either a constitutive or inducible promoter. The skilled person will know how to select suitable promoters. The expression constructs may contain sites for<br><br>
transcription initiation, termination, and, in the transcribed region, a ribosome binding site for translation. The coding portion of the mature transcripts expressed by the constructs may preferably include an initiation codon at the beginning and a termination codon appropriately positioned at the end of the polypeptide to be translated.<br>
Vector DNA may be introduced into suitable host cells via conventional transformation or transfection techniques. As used herein, the terms "transformation", "transconjugation" and "transfection" are intended to refer to a variety of art-recognized techniques for introducing foreign nucleic acid (e.g., DNA) into a host cell, including calcium phosphate or calcium chloride co-precipitation, DEAE-dextran-mediated transfection, transduction, infection, lipofection, cationic lipidmediated transfection or elect-operation. Suitable methods for transforming or transfecting host cells may be found in Sambrook, et al. (supra), Davis et al., Basic Methods in Molecular Biology (1986) and other laboratory manuals.<br>
In order to identify and select cells which have integrated the foreign DNA into their genome, a gene that encodes a selectable marker (e.g., resistance to antibiotics) is generally introduced into the host cells along with the gene of interest. Preferred selectable markers include those that confer resistance to drugs, such as kanamycin, tetracycline, ampicillin and streptomycin. A nucleic acid encoding a selectable marker is preferably introduced into a host cell on the same vector as that encoding a protein according to the invention or can be introduced on a separate vector such as, for example, a suicide vector, which cannot replicate in the host cells. Cells stably transfected with the introduced nucleic acid can be identified by drug selection (e.g., cells that have incorporated the selectable marker gene will survive, while the other cells die).<br>
The invention provides also an isolated polypeptide having the amino acid sequence shown in SEQ ID NO:2 or an amino acid sequence obtainable by expressing a polyriucleotide of the present invention, such as for instance a polynucleotide sequence shown in SEQ ID N0:l in an appropriate host.<br>
Polypeptides according to the invention may contain only conservative substitutions of one or more amino acids in the amino acid sequence represented by SEQ ID NO:2 or substitutions, insertions or deletions of non-essential amino acids. Accordingly, a non-essential amino acid is a residue that may be altered in the amino acid sequences shown in SEQ ID NO:2 without substantially altering the biological function. For example, amino acid residues that are conserved among the proteins of the present<br><br>
invention, are predicted to be particularly unamenable to alteration. Furthermore, amino acids conserved among the proteins according to the present invention and other SNDHai proteins are not likely to be amenable to alteration.<br>
The term "conservative substitution" is intended to mean that a substitution in which the amino acid residue is replaced with an amino acid residue having a similar side chain. These families are known in the art and include amino acids with basic side chains (e.g., lysine, arginine and histidine), acidic side chains (e.g., aspartic acid, glutamic acid), uncharged polar side chains (e.g., glycine, asparagine, glutamine, serine, threonine, tyrosine, cysteine), non-polar side chains (e.g., alanine, valine, leucine, isoleucine, proline, phenylalanine, methionine, tryptophan), beta-branched side chains (e.g., threonine, valine, isoleucine) and aromatic side chains (e.g., tyrosine, phenylalanine, tryptophan, histidine).<br>
As mentioned above, the polynucleotides of the present invention may be utilized in the genetic engineering of a suitable host cell to make it better and more efficient hi the fermentation, for example in a direct fermentation process for Vitamin C*<br>
Therefore, the invention also relates to the concurrent use of a gene encoding a SNDHai polypeptide or an active fragment or derivative thereof and a gene encoding a polypeptide having SMS or RCS activity or an active fragment or derivative thereof for the preparation of a recombinant host cell. Such a host cell will then show an improved capability to directly produce Vitamin C.<br>
The alteration in the genome of the microorganism may be obtained e.g. by replacing through a single or double crossover recombination a wild type DNA sequence by a DNA sequence containing the alteration. For convenient selection of transformants of the microorganism with the alteration in its genome the alteration may, e.g. be a DNA sequence encoding an antibiotic resistance marker or a gene complementing a possible auxotrophy of the microorganism. Mutations include, but are not limited to, deletion-insertion mutations.<br>
An alteration in the genome of the microorganism leading to a more functional polypeptide may also be obtained by randomly mutagenizing the genome of the microorganism using e.g. chemical mutagens, radiation or transposons and selecting or screening for mutants which are better or more efficient producers of one or more fermentation products. Standard methods for screening and selection are known to the skilled person.<br><br>
In a specific embodiment, it is desired to knockout or suppress a represser of the SNDHai gene of the present invention, i.e., wherein its represser gene expression is artificially suppressed in order to improve the yield, production, and/or efficiency of production of the fermentation product when introduced into a suitable host cell. Methods of providing knockouts as well as microorganisms carrying such suppressed genes are well known in the art. As used herein, "suppression of the gene expression" includes complete and partial suppression, as well as suppression under specific conditions and also suppression of the expression of either one of the two alleles.<br>
The aforementioned mutagenesis strategies for SNDHai proteins may result in increased yields of a desired compound in particular Vitamin C. This list is not meant to be limiting; variations on these mutagenesis strategies .will be readily apparent to one of ordinary skill in the art. By these mechanisms, the nucleic acid and protein molecules of the invention may be utilized to generate microorganisms such as Gluconobacter oxydans or related strains of bacteria expressing mutated SNDHai nucleic acid and protein molecules such that the yield, production, and/or efficiency of production of a desired compound such as Vitamin C is improved.<br>
The nucleic acid molecules, polypeptides, vectors, primers, and recombinant microorganisms described herein may be used in one or more of the following methods: identification of Gluconobacter oxydans and related organisms; mapping of genomes of organisms related to Gluconobacter oxydans', identification and localization of Gluconobacter oxydans sequences of interest; evolutionary studies; determination of SNDHai protein regions required for function; modulation of a SNDHai protein activity or function; modulation of the activity of a SNDHai pathway; and modulation of cellular production of a desired compound, such as Vitamin C.<br>
The activity of an SNDHai peptide may be determined by methods known in the art and further exemplified herein.<br>
hi order to illustrate the invention in more detail, several examples are provided herein, wherein the manipulation of genes selected from RCS or SMS genes is exemplified in great detail. It will be evident for the skilled person now to select other genes encoding proteins selected from RCS or SMS proteins and manipulate them according to the teachings provided herein in order to obtain a microorganism that produces improved yields of Vitamin C.<br><br>
In order to provide additional guidance in the above process, the following table is provided wherein it is further detailed how to manipulate specific genes selected from RCS or SMS genes. Herein the expressions "Up" and"Down" refer to the preferably overexpression and underexpression of the proteins, respectively, such as may be achieved by the up-regulation and down-regulation of the corresponding genes respectively. Equivalent results may be obtained when the activity of the corresponding proteins is enhanced or reduced by any other means than genetic manipulation.<br><br>
(Table Removed)<br>
The skilled person will know how to enhance and/or improve the activity of a protein, as for example SNDHai or alcohol dehydrogenase. Such may be accomplished by either genetically modifying the host organism in such a way that it produces more or more stable copies of the said protein than the wild type organism. It may also be accomplished by increasing the specific activity of the protein:,<br>
In the following paragraphs procedures are described how to achieve this goal, i.e. the increase in the yield and/or production of Vitamin C which is directly produced from D-sorbitol or L-sorbose by increasing (up-regulation) the activity of a specific protein, as for example SNDHai. These procedures apply mutatis mutandis for the similar RCS and SMS proteins whose functions compared to the wild type counterpart have to be enhanced or improved.<br>
Modifications in order to have the organism produce more copies of the SNDHai gene, i. e. overexpressing the gene, and/or protein may include the use of a strong promoter, or the mutation (e.g. insertion, deletion or point mutation) of (parts of) the SNDHai gene or its regulatory elements. It may also involve the insertion of multiple copies of the gene into a suitable microorganism. An increase in the specific activity of an SNDHai protein may also be accomplished by methods known in the art. Such methods may include the mutation (e.g. insertion, deletion or point mutation) of .(parts of) the SNDHai gene.<br>
A mutation as used herein may be any mutation leading to a more runctional*or more<br>
stable polypeptide, e.g. more functional or more stable SNDHai gene products. This<br>
may include for instance an alteration in the genome of a microorganism, which<br>
improves the synthesis of SNDHai or leads to the expression of a SNDHai protein<br>
with an altered amino acid sequence whose function compared with the wild type<br>
counterpart having a non-altered amino acid sequence is improved and/or enhanced.<br>
The interference may occur at the transcriptional, translational or post-translational<br>
level.	-<br>
Also known in the art are methods of increasing the activity of a given protein by contacting the SNDHai protein with a specific enhancer or other substances that specifically interact with the SNDHai protein. In order to identify such specific enhancers, the SNDHai protein may be expressed and tested for activity in the presence of compounds suspected to enhance the activity of the SNDHai protein. The activity of the SNDHai protein may also be increased by stabilizing the messenger RNA encoding SNDHai. Such methods are also known in the art, see for example, in Sambrook et al.,1989, Molecular Cloning, A Laboratory Manual, Cold Spring Harbor Press, N. Y.; and Ausubel et al. (eds.), 1995, Current Protocols in Molecular Biology, (John Wiley &amp; Sons, N. Y.).<br>
The term "increase" of activity as used herein encompasses increasing activity of one or more polypeptides in the producing organism, which in turn are encoded by the corresponding polynucleotides described herein. There are a number of methods:<br>
available in the art to accomplish the increase of activity of a given protein, in this case the SNDHai protein. In general, the specific activity of a protein may be increased or the copy number of the protein may be increased. The term "increase of activity" or equivalent expressions also encompasses the situation wherein SNDHai protein activity is introduced in a cell that did not contain this activity before, e.g. by introducing a gene encoding SNDHai in a cell that did not contain an equivalent of this gene before, or that could not express an active form of the corresponding protein before.<br>
To facilitate such an increase, the copy number of the genes corresponding to the polynucleotides described herein may be increased. Alternatively, a strong promoter may be used to direct the expression of the polynucleotide. In another embodiment, the promoter, regulatory region and/or the ribosome binding site upstream of the gene can be altered to increase the expression. The expression may also be enhanced or increased by increasing the relative half-life of the messenger RNA. In another embodiment, the activity of the polypeptide itself may be increased by employing one or more mutations in the polypeptide amino acid sequence, which increase the activity. For example, altering the relative Km of the polypeptide with its corresponding substrate will result in improved activity. Likewise, the relative half-life of the polypeptide may be increased. In either scenario, that being enhanced gene expression or increased specific activity, the improvement may be achieved by altering the composition of the cell culture media and/or methods used for culturing. "Enhanced expression" or "improved activity" as used herein means an increase of at least 5%, 10%, 25%, 50%, 75%, 100%, 200% or even more than 500%, compared to a wild-type protein, polynucleotide, gene; or the activity and/or the concentration of the protein present before the polynucleotides or polypeptides are enhanced and/or improved. The activity of the SNDHai protein may also be enhanced by contacting the protein with a specific or general enhancer of its activity.<br>
In other cases disclosed herein the activity of an SMS or RCS polypeptide is to be reduced or abolished so that the yield of Vitamin C which is directly produced from D-sorbitol or L-sorbose is additionaly increased.<br>
The following procedures relating to the decrease (down-regulation) of SMS 05 protein activity apply mutatis mutandis for the RCS and SMS proteins, in particular those exemplified herein whose functions compared to the wild type counterpart have to be decreased.<br>
To facilitate such a decrease, the copy number of the genes corresponding to the polynucleotides described herein may be decreased, such as for instance by underexpression or disruption of a gene. A gene is said to be "underexpressed" if the level of transcription of said gene is reduced in comparison to the wild type gene. This may be measured by for instance Northern blot analysis quantifying the amount of mRNA as an indication for gene expression. As used herein, a gene is underexpressed if the amount of generated mRNA is decreased by at least 1%, 2%, 5% 10%, 25%, 50%, 75% or 100% compared to the amount of mRNA generated from a wild-type gene. Alternatively, a weak promoter may be used to direct the expression of the polynucleotide. In another embodiment, the promoter, regulatory region and/or the ribosome binding site upstream of the gene can be altered to achieve the down-expression. The expression may also be reduced by decreasing the relative half-life of the messenger RNA.<br>
In another embodiment, the activity of the polypeptide itself may be decreased by employing one or more mutations in the gene encoding the polypeptide, resujting inat least one mutation in the polypeptide arnino acid sequence, which decreases the     . activity. For example, altering the affinity of the polypeptide for its corresponding substrate may result in reduced activity. Likewise, the relative half-life of the polypeptide may be decreased.<br>
In either scenario, that being reduced gene expression or reduced activity, the reduction may be achieved by altering the composition of the cell culture media and/or methods used for culturing. "Reduced expression" or "reduced activity" as used herein means a decrease of at least 5%, 10% 25%, 50%, 75% or 100% compared to a wild-type protein, polynueleotide or gene. The activity of a given " protein may also be reduced by contacting the protein with a specific or general inhibitor of its activity. The terms "reduced activity", "decreased or abolished activity" are used interchangeably herein.<br>
To improve the Vitamin C production of a certain recombinant host cell, SMS 05 gene expression may be inhibited in that organism for instance by targeting nucleotide sequences complementary to the regulatory region of a SMS 05 nucleotide sequence (e.g., a SMS 05 promoter and/or enhancers) to form triple helical structures that prevent transcription of a SMS 05 gene in target cells. See generally, Helene, C. (1991) AnticancerDrugDes. 6 (6): 569-84; Helene, C. et al. (1992) Ann. N. Y Acad Sci. 660: 27-36; and Maher, L. J. (1992) Bioassays 14 (12): 807-15.<br>
Inhibition or prevention of gene expression may also be achieved by modifying the SMS 05 gene, e.g., by introducing one or more mutations into the SMS O5gene wherein said modification leads to a SMS 05 protein with a function which is significantly decreased in comparison to the wild-type protein.<br>
A mutation as used herein may be any mutation leading to a less functional or unstable polypeptide, e.g. less functional or unstable SMS 05 gene products. This may include for instance an alteration in the genome of a microorganism, which interferes with the synthesis of SMS 05 or leads to the expression of a SMS 05 protein with an altered amino acid sequence whose function compared with the wild type counterpart having a non-altered amino acid sequence is completely or partially destroyed. The interference may occur at the transcriptional, translational or post-translational level.<br>
The alteration in the genome of the microorganism may be obtained e.g. by replacing through a single or double crossover recombination a wild type DNA sequence by a DNA sequence containing the alteration. For convenient selection of transformants of the microorganism with the alteration hi its genome the alteration may, e.g. be a DNA sequence encoding an antibiotic resistance marker or a gene complementing a possible auxotrophy of the microorganism. Mutations include, but are not limited to, deletion-insertion mutations. An example of such an alteration includes a gene disruption, i.e. a perturbation of a gene such that the product that is normally produced from this gene is not produced in a functional form. This could be due to a complete deletion, a deletion and insertion of a selective marker, an insertion of a selective marker, a frameshift mutation, an in-frame deletion, or a point mutation that leads to premature termination. In some of these cases the entire mRNA for the gene is absent, in others the amount of mRNA produced varies. In all cases the polypeptide encoded by said gene is not produced in a functional form, either absent or in a mutated form, such as e.g. a protein having reduced activity as defined herein.<br>
An alteration in the genome of the microorganism leading to a less functional or nonfunctional polypeptide may also be obtained by randomly mutagenizing the genome of the microorganism using e.g. chemical mutagens, radiation or transposons and selecting or screening for mutants which are better or more efficient producers of one or more fermentation products. Standard methods for screening and selection are known to the skilled person.<br>
In a specific embodiment, it is desired to knockout the SMS 05 gene of the present invention, i.e., wherein its gene expression is artificially suppressed in order to<br>
improve the yield, productivity, and/or efficiency of production of the fermentation product Vitamin C when introduced into a suitable host cell. Methods of providing knockouts as well as microorganisms carrying such suppressed genes are well known in the art. The suppression of the endogenous SMS 05 gene may be induced by deleting at least a part of the gene or the regulatory region thereof. As used herein, "suppression of the gene expression" includes complete and partial suppression, as well as suppression under specific conditions and also suppression of the expression of either one of the two alleles.<br>
In order to create a knockout microorganism in which the expression of the SMS 05 gene is artificially suppressed, first the SMS 05 gene may be cloned and then a vector for homologous recombination may be constructed by using the gene to inactivate the endogenous SMS 05 gene in the target microorganism. The vector for homologous recombination then contains a nucleic acid sequence designed to inactivate the, endogenous SMS 05 gene in the target microorganism. Such a nucleic acid may be for instance a nucleic acid sequence of the SMS 05 gene or the regulatory regjon thereof, such as the existing flanking region of the gene to be inactivated (in cis), or existing separately (in trans), containing at least a partial deletion, or alternatively it may be a nucleic acid sequence of the SMS 05 gene or the regulatory region thereof containing other genes. A gene which can also function as a marker is preferably selected as the gene to be inserted into the SMS 05 gene or the regulatory region thereof. The insert genes to be used include for instance drug-resistance genes as defined above. There is no particular limitation on the position where the genes may be inserted in the SMS 05 gene, as long as the insertion at that position results in the_ suppression of the expression of the endogenous SMS 05 gene in the target. To avoid polar effects of the insertion, in-frame silent deletions can be introduced by using, for example, the sacB system or long-flanking homology PCR. These techniques are well known to the person skilled in the art.<br>
The aforementioned mutagenesis strategies for SMS 05 proteins may result in increased yields of a desired compound in particular Vitamin C. This list is not meant to be limiting; variations on these mutagenesis strategies will be readily apparent to one of ordinary skill in the art. By these mechanisms, the nucleic acid and protein molecules of the invention may be utilized to generate microorganisms such as Gluconobacter oxydans or related strains of bacteria expressing mutated SMS 05 nucleic acid and protein molecules such that the yield, productivity, and/or efficiency of production of a desired compound such as Vitamin C is improved.<br>
In other cases disclosed herein the activity of an SMS or RCS polypeptide is to be increased so that the yield of Vitamin C which is directly produced from D-sorbitol or L-sorbose is additionaly increased.<br>
Also, particularly good results were obtained when a recombinant microorganism overexpressing SNDHai was transformed in such a way that an endogenous gene SMS 05 (SEQ ID NO: 44) encoding a NAD(P)-dependent sorbosone dehydrogenase (SEQ ID NO: 45) was knocked out according to examples 15-19.<br>
Also, particularly good results were obtained when a recombinant microganism overexpressing SNDHai was transformed in such a way that a gene RCS;21 (SEQ ID NO: 180) encoding a protein (pqqA) involved in the PQQ biosynthesis (SEQ ID NO: 1 81) is overexpressed according to examples 20 and 21,<br>
This invention is further illustrated by the following examples which should not be construed as limiting. The contents of all references, patent applications, patents and published patent applications, cited throughout this application are hereby incorporated by reference.<br>
Examples<br>
Example 1:    Preparation of chromosomal DNA and amplification of DNA fragment by PCR<br>
Chromosomal DNA of Gluconobacter oxydans DSM 1 7078 were prepared from the „ \ cells cultivated at 30°C for 1 day in mannitol broth (MB) liquid medium. consisting of 25 g/1 mannitol, 5 g/1 of yeast extract (Difco), and 3 g/1 of Bactopeptone (Difco) by the method described by Sambrook et al (1989) "Molecular Cloning: A Laboratory Manual/Second Edition", Cold Spring Harbor Laboratory Press).<br>
A DNA fragment was prepared by PCR with the chromosomal DNA prepared above and a set of primers, Pf (SEQ ID N0:3) and Pr (SEQ ID NO:4). For the reaction, the Expand High Fidelity PCR kit (Roche Diagnostics) and 10 ng of the chromosomal DNA was used in total volume of 1 00 |ul according to the supplier's instruction to have the PCR product containing SNDHai DNA sequence (SEQ ID NO:1). The PCR product was recovered from the reaction and its correct sequence confirmed.<br>
Example 2:    Production of Vitamin C from L-sorbosone using resting cells grown on mannitol broth agar medium<br>
IFO strains 3293, 3292, 3244, 3260, 3266, 3287, 3259, 13693, and 13773 as well as Acetobacter sp. ATCC 15164 and Gluconobacter oxydans DSM 17078, a derivative of the strain IFO 3293, were used for the production of Vitamin C from L-sorbosone.<br>
Strains IFO 13693 and IFO 13773 were grown at 27°C for 3 days on No. 350 medium containing 5 g/1 Bactopeptone (Difco), 5 g/1 yeast extract (Difco), 5 g/1 glucose, 5 g/1 mannitol, 1 g/1 MgSO4.7H2O 5 ml/1 ethanol, and 15 g/I agar. All other Acetobacter strains and all Gluconobacter strains were grown at 27°C for 3 days on mannitol broth (MB) agar medium containing 25 g/1 mannitol, 5 g/1 yeast extract (Difco Laboratories, Detroit, Mich., USA), 3 g/I Bactopeptone (Difco), and 18 g/1 of agar (Difco).<br>
Cells were scraped from the agar plates, suspended in distilled water and used for resting cell reactions conducted at 30°C for 20 h in 5 ml tubes with shaking at 230 rpm. The reaction mixtures (0.5 ml) contained 1% L-sorbosone, 0.3% NaCl, 1% and cells at a final concentration of 10 absorbance units at 600 nanometers ). At the conclusion of the incubation period, the reaction mixtures were analyzed by high performance liquid chromatography (HPLC) using an Agilent 1100 HPLC system (Agilent Technologies, Wilmington, USA) with a LiChrospher-100-RP18 (125 x 4.6 mm) column (Merck, Darmstadt, Germany) attached to an Aminex-HPX-78H (300 x 7,8 mm) column (Bioiad, Reinaca, Switzerland). The mobile-phase was 0.004 M sulfuric acid, and the flow rate was 0.6 ml/mm. Two signals were recorded using an UV detector (wavelength 254 run) in combination with a refractive index detector. In addition, the identification of the Vitamin C was done using an amino-column (YMC-Pack Polyamine-II, YMC, Inc., Kyoto, Japan) with UV detection at 254 nm. The mobile phase was 50 mM NH4H2PO4 and acetonitrile (40:60).<br>
An Agilent Series 1100 HPLC-mass spectrometry (MS) system was used to identify Vitamin C. The MS was operated in positive ion mode using the electrospray interface. The separation was carried out using a LUNA-C8(2) column (100 x 4.6 mm) (Phenomenex, Torrance, USA). The mobile phase was a mixture of 0.1% formic acid and methanol (96:4). Vitamin C eluted with a retention time of 3.1 minutes. The identity of the Vitamin C was confirmed by retention time and the molecular mass of the compound.<br>
To exclude the presence of D-isoascorbic acid, the identification of Vitamin C was additionally done by retention time using an amino-column (YMC-Pack Polyamine-<br>
2, YMC, Inc., Kyoto, Japan) with UV detection at 254 nm. The mobile phase was 50 mM NH4H2P04 and acetonitrile (40:60).<br>
The Gluconobacter and Acetobacter strains produced Vitamin C from L-sorbosone as shown in Table 1.<br>
Table 1. Production of Vitamin C from L-sorbosone<br><br>
(Table Removed)<br>
Blank; reaction was done in the reaction mixture without cells.<br>
Example 3:    Vitamin C production from L-sorbose and D-sorbitol in tube and flask fermentations<br>
Cells of G. oxydans DSM 17078 were used to inoculate 4 ml of No. 3BD liquid medium and cultivated in a tube (18 mm diameter) at 30°C for 3 days with shaking at 220 rpm. 20 mg/1 of Vitamin C had accumulated at the end of the incubation period.<br>
Cells of strain DSM 17078 were cultivated (in triplicate) in 50 ml of No. 5 medium containing 100 g/1 D-sorbitol, 0.5 g/1 glycerol, 15 g/1 yeast extract (Difco), 2.5 g/1 of MgS04-7H20, and 15 g/L of CaC03 in a 500 ml baffled shake flask at 30°C with. shaking at 200 rpm. After 72 h of cultivation, the amounts of Vitamin C measured by HPLC in the three flasks were 400, 500 and 680 mg/1,<br>
Example 4:    Vitamin C production from D-sorbitol in fed-batch fermentation<br>
Cells of G. oxydans DSM 17078 were grown in 200 ml No. 5 medium containing 100 g/1 D-sorbitol, 0.5 g/1 glycerol, 15 g/1 yeast extract (Fluka BioChemika, Buchs, Switzerland), 2,5 g/1 MgS04'7H2O and 15 g/1 CaCO3 in a 2-1 baffled shake flask at 30°C with shaking at 180 rpm. After 48 h, 150 ml of this culture was used to inoculate a 10-1 bioreactor (B. Braun ED 10, Melsungen, Germany) previously prepared with 5.31 medium containing 20 g/1 D-sorbitol, 0.5 g/1 glycerol, 15 g/1 yeast extract (Fluka BioChemika, Buchs, Switzerland) and 2.5 g/1 MgSO4.7H2O and equipped with temperature, pH and dissolved oxygen sensors and control loops. Temperature was controlled at 30°C, pH was controlled at 6.0 by adding a 28% ammonia solution, airflow was 4.5 1/min and dissolved oxygen was controlled at 30% by a cascade with stirring speed (minimum 300 rpm). After 6 h process time, a 500 g/1 sorbitol solution was fed at a rate of 25 g/h for a period of 44 h. After 96 h process time, about 1% substrate was left in the supernatant, and 950 mg/1 Vitamin C had been produced.<br>
Example 5:    Vitamin C production from L-sorbosone or L-sorbose with a cell membrane fraction<br>
Cells of G. oxydans DSM 17078 were cultivated in 100 ml of No, 3BD liquid medium in a 500 ml baffled shake flask at 30°C with shaking at 220 rpm for 3 da3rs. The resulting culture was centrifuged at 500 rpm to remove CaCOa. The supernatant from this step was then centrifuged at 5,000 rpm to pellet the cells. The collected cells were suspended in 3 ml of 50 mM potassium phosphate buffer (pH 7.0) and the cells were disrupted by two passages through a French Pressure cell (SIM-AMTNCO Spetronic Instruments, USA) at 900 psi. The resulting homogenate was first centrifuged at 5,000 rpm to remove cell debris. Then the supernatant was diluted to a final protein concentration of 3 mg of protein/ml. This diluted sample is designated as cell-free extract (CFE). The CFE was centrifuged at 100,000 x g for 60 min. The supernatant was discarded and the pellet was collected as the membrane fraction.<br>
The reaction (200 ul) with the membrane fraction (100 ul) was carried out in 50 mM potassium phosphate buffer (pH7.0), 30°C with shaking at 220 rpm for 15 h. The substrates tested were L-sorbosone (1% final concentration) and L-sorbose (2% final concentration). The final protein concentration used in the reaction was 1.5 mg/ml. At the end of the incubation period, 680 mg/1 and 10 mg/1 of Vitamin C had been produced from 1% L-sorbosone and 2% L-sorbose, respectively.<br>
Example 6:    Production of Vitamin C from D-sorbitoI, L-sorbose or L-sorbosone using resting cells grown on 3BD agar medium<br>
Cells of G. oxydans DSM 17078 were grown at 27°C for 3 days on No. 3BD agar medium containing 70 g/1 L-sorbose, 0.5 g/1 glycerol, 7.5 g/1 yeast extract (Difco), 2.5 g/1 MgS04-7H20,10 g/1 CaCO3 and 18 g/1 agar (Difco). The resting cell reactions (1 ml reaction mixture in 10 ml tube) were carried out with 2% D-sorbitol, 2% L-sorbose, or 1% L-sorbosone at 30°C for 24 h as described in Example 2. Strain DSM 17078 produced280, 400 and 1780 mg/1 of Vitamin C from D-sorbitol, L-sorbose, and L-sorbosone, respectively.<br>
Other reactions (0.5 ml reaction mixture in 10 ml tube) were carried out with DSM 17078 cells grown on No. 3BD agar medium in reaction mixtures containing 2% D-sorbitol, 2% L-sorbose or 2% L-sorbosone for 2 days as described in Example 2. Strain DSM 17078 produced 1.8,2.0 and 5.1 g/1 of Vitamin C from D-sorbitol, L-sorbose, and L-sorbosone, respectively.<br>
A reaction using cells of G. oxydans EFO 3293 was carried out with 2% L-sorbosone as described above. Strain IFO 3293 produced 5.7 g/1 of Vitamin C in 2 days.<br>
Example 7:    Production of Vitamin C from D-sorbitol using resting cells grown in liquid medium<br>
Cells of G. oxydans DSM 17078 were grown in 200 ml of No. 5 medium containing 100 g/1 D-sorbitol, 0.5 g/1 glycerol, 15 g/1 yeast extract (Fluka BioChemika, Buchs, Switzerland), 2.5 g/1 MgSO4-7H2O and 15 g/1 CaCO3 in a 2-1 baffled shake flask at 30°C with shaking at 180 rpm. After 24 h, the culture was centrifuged at 3220 g (Eppendorf 5810R, Hamburg, Germany), and the cells were resuspended in 0.9% NaCl solution, centrifuged again at 3220 g and the cell pellet was used to inoculate one baffled 500 ml shake flask containing 50 ml of full growth medium (100 g/1 D-sorbitol, 0.5 g/1 glycerol, 15 g/1 yeast extract, 2.5 g/1 MgS04.7H20, 15 g/1 CaCO3) and another baffled 500 ml shake flask containing 50 ml production medium (100 g/1 D-sorbitol, 3 g/1 NaCl, 10 g/1 CaCO3). The initial cell density, measured as optical<br>
density at 600 nm (OD600), in both flasks was 10. Both flasks were incubated at 30°C with shaking at 180 rpm. After 48 h, the cell suspension in growth medium and production medium had accumulated 1.06 and 1.18 g/1 Vitamin C, respectively. No additional growth was observed in full medium during the incubation period time.<br>
Example 8:    Southern biot analysis of the bacteria producing Vitamin C from L-sorbosone<br>
Chromosomal DNA was prepared from cells of Gluconobacter oxydans IFO 3293, IFO 3292, IFO 3244, IFO 3287, Gluconobacter frateurii IFO 3260 and IFO 3265, Gluconobacter cerinus IFO 3266 and IFO 3269, Acetobacter aceti subsp. orleanus IFO 3259, Acetobacter aceti subsp. xylinum IFO 13693 and IFO 13773, Acetobacter sp. ATCC 15164, and Escherichia coli K-12. Strains IFO 13693 and IFO 13773 were grown at 27°C for 3 days on No. 350 medium containing 5 g/1 Bactopeptone (Difco), 5 g/L yeast extract (Difco), 5 g/L glucose, 5 g/L mannitol, 1 g/L MgSO4-7H2O,; 5 ml/L ethanol, and 15 g/L agar. All other Acetobacter strains and atl Gluconobacter, strains were grown at 27°C for 3 days on mannitol broth (MB) agar medium containing 25 g/1 mannitol, 5 g/1 yeast extract (Difco Laboratories, Detroit, Mich., USA), 3 g/1 Bactopeptone (Difco), and 18 g/1 of agar (Difco). E. coli K-12 was grown onLuria Broth agar medium. The chromosomal DNA preparations were used for Southern blot hybridization under stringent conditions. The chromosomal DNA preparations were digested with Clal (when analyzing the N-domain region) or EcoRI (when analyzing the C-domain region), and 1 ug of the DNA fragments were separated by agarose gel electrophoresis (1% agarose). The gel was treated with 0.25 N HC1 for 15 min and then 0.5 N NaOH for 30 min, and then was blotted onto a nylon membrane with Vacuum Blotter Model 785 (BIO-RAD Laboratories AG, Switzerland) according to the instruction of the supplier. The probes were prepared with PCR-DIG labeling kit (Roche Diagnostics) by using the primer sets as described in Table 2. The PCR product PI corresponds to the region of SNDHai designated the N-domain (possible transmembrane region) while PCR product P2 corresponds to the region of SNDHai designated as the C-domain (possible primary dehydrogenase region).<br>
Table 2. Primers used for PCR to generate labeled probes for Southern<br>
hybridizations<br><br>
(Table Removed)<br>
Table 2 shows the results of the Southern blot hybridization experiments. In the hybridization with the PI (N-domain) probe, clear positive bands were observed for G. oxydans IFO 3293, IFO 3292, IFO 3244, IFO 3287 and ,4. sp. ATCC 15164. In the hybridization with the P2 (C-domain) probe, clear positive bands were observed for strains IFO 3293, IFO 3292, IFO 3244, IFO 3287 and A. sp. ATCC 15164, while a faint band was observed for stains IFO 3260, IFO 3265, IFO 3266, IFO 3269 and IFO 13773. The control strain, E. coliK-12, showed no detectable signals for either domain.<br>
Table 3. Detection of hybridization signals in different strains obtained by Southern blot hybridization with probes for N- and C-domains of SNDHai (probes PI .and P2)<br><br>
(Table Removed)<br>
Tr, trace; nd, not detected. Probes P1and P2 were synthesized (as DIG-labeled PCR products) with the primer sets specified in Table 2.<br>
Example 9:    PCR amplification and sequencing of orthologs of the Gluconobacter oxydans DSM 17078 SNDHai gene<br>
Chromosomal DNA preparations (prepared as described in Example 8) were used as templates for PCR with the four primer sets shown in Table 2. Five to 100 ng of , chromosomal DNA was used per reaction (total volume, 50 ul). Unless specified otherwise, the Expand High Fidelity PCR system was used (Roche Diagnostics). The PCR conditions were as follows:<br>
Incubation at 94°C for 2 min; 30 cycles of (i) denaturation step at 94°C for 15 sec, (ii) annealing step at 60°C for 30 sec, (iii) synthesis step at 72QC for 45 to 120 sec (time for the synthesis step for primer sets PI, P2, P3 and P4 were 45 sec, 120 sec, 90 sec, and 90 sec, respectively); extension at 72°C for 7 min.<br>
Samples of the PCR reactions were separated by agarose gel electrophoresis and the<br>
bands were visualized with a transilluminator after staining with ethidium bromide.<br>
The results of the PCR reactions are summarized hi Table 4.	:<br>
Table 4. Detection of PCR products PI, P2, P3 and P4 in different strains obtained with the primer sets of Table 2 (products visualized via agarose gel electrophoresis)<br><br>
(Table Removed)<br>
+, detected; nd, not detected; nt, not tested. "This PCR was done with GC-rich PCR system (Roche Diagnostics) with the same reaction cycle as was used for Expand High Fidelity PCR system.<br>
When clear PCR bands were observed on the agarose gel (Table 4), the PCR products were used directly for nucleotide sequencing using standard methods. The nucleotide sequences obtained for the different PCR products, and the corresponding amino acid sequences of the encoded peptides, were compared with the full length sequence of the SNDHai gene and protein from G. oxydans DSM 17078.<br>
Gluconobacter oxydans IFQ 3292 SNDHai ortholog<br>
The PCR product (about 1 kb) obtained upon amplification with primers SNDH1391F (SEQ ID NO:10) and SNDH2364R (SEQ ID N0:8) and chromosomal DNA from G. oxydans BFO 3292 as the template, was used for sequencing with primer SNDH 1391F (SEQ ID NO: 10). The determined nucleotide sequence of 771 bp (SEQ ID NO:11) showed 98.7% (761/771) homology with nucleotides 1431-2201 of the sequence of SNDHai from G. oxydans DSM 17078 (SEQ ID NO: 1). The deduced amino acid sequence of 256 amino acids (SEQ ED NO: 12) showed 100% identity to amino acids 478-733 of the amino acid sequence of SNDH from G. oxydans DSM 17078 (SEQ ID NO:2).<br>
Gluconobacter oxvdans IFQ 3287 SNDHai ortholog<br>
The PCR product (about 0.4 kb) obtained upon amplification with primers SNDHIF (SEQ ID NO:5) and SNDH420R (SEQ ID NO:6) and chromosomal DNA from G. oxydans IFO 3287 as the template, was used for sequencing with primer SNDH420R (SEQ ID NO:6). The determined nucleotide sequence of 350 bp (SEQ ED NO: 13) showed 97.4% (341/350) homology with nucleotides 31-380 of SEQ ID NO:1. The deduced amino acid sequence of 116 residues (SEQ ID NO: 14) showed 100% identity with amino acids 11-126 of SEQ ID N0:2.<br>
The PCR product (about 1.9 kb) obtained upon amplification with primers SNDH501F (SEQ ID NO:7) and SNDH2364R (SEQ ED NO:8) was used for sequencing with primer SNDH501F (SEQ ID N0:7). The determined nucleotide sequence of 808 bp (SEQ ID NO: 15) showed 98.0% (745/808) homology with nucleotides 578-1385 of SEQ ID N0:l). The deduced amino acid sequence of 268 .residues (SEQ ID NO: 16) showed 100% identity to amino acids 194-461 of SEQ ID NO:2.<br>
The PCR product (about 1 kb) obtained upon amplification with primers SNDH1391F (SEQ ID NO: 10) and SNDH2364R (SEQ ID NO:8) was used for sequencing with primer SNDH1391F (SEQ ED N0:10). The determined nucleotide sequence of 800 bp (SEQ ID NO: 17) showed 98.8% (790/800) homology with nucleotides 1469-2268 of SEQ ID NO: 1. The deduced amino acid sequence of 266 residues (SEQ ID NO:18) showed 100% identity with amino acids 491-756 of SEQ IDNO:2.<br>
Acetobacter sv. ATCC 15164 SNDHai ortholog<br>
The PCR product (about 0.4 kb) obtained upon amplification with primers SNDH1F<br>
(SEQ IDNO:5) and SNDH420R (SEQ ED N0:6) and chromosomal DNA from A. sp.<br>
ATCC 15164 as the template, was used for sequencing with primer SNDH420R (SEQ<br>
ID N0:6). The determined nucleotide sequence of 360 bp (SEQ ID NO: 19) showed<br>
97.8% (352/360) homology with nucleotides 31-390 of SEQ IDNO:1. The deduced<br>
amino acid sequence of 120 residues (SEQ ID N0:20) showed 100% identity with<br>
amino acids 11-130 of SEQ ED N0:2.	-<br>
The PCR product (about 1.9 kb) obtained upon amplification with primers<br>
SNDH501F (SEQ ID NO:7) and SNDH2364R (SEQ ID N0:8) was used for<br>
sequencing with primer SNDH501F (SEQ ID NO:7). The determined nucleotide<br>
sequence of 760 bp (SEQ ID NO:21) showed 98.0% (745/760) homology with<br>
nucleotides 563-1322 of SEQ ED NO:1. The deduced amino acid sequence of 252<br>
residues (SEQ ID NO:22) showed 100% identity with amino acids 189-440 of SEQ<br>
IDNO:2.	<br>
Gluconobacter oxydans IFO 3244 SNDHai ortholog<br>
Complete nucleotide sequence-of the SNDHai ortholog gene of G. oxydans IFO 3244 was determined by using the PCR products obtained with the chromosomal DNA of G. oxydans IFO 3244 as the template and the following primer sets: SNDH1F (SEQ ID N0:5) and SNDH420R (SEQ ID N0:6); SNDH501F (SEQ ID NO:7) and SNDH1530R (SEQ IDNO:9); SNDH1391F (SEQ IDNO:10) and SNDH2364R (SEQ ID NO:8); SNDH382 (SEQ ID N0:23) and SNDH1530R (SEQ ED NO:9); SNDH1F (SEQ ED NO:5) and SNDH689R (SEQ ID NO:24). Chromosomal DNA digested with Bgltt and BamHl and ligated was used for two more PCRs with following primer sets: SNDH420R (SEQ ID NO:6) and SNDH501F (SEQ ID N0:7) and SNDH1530R (SEQ ID NO:9) and IS-50.3 (SEQ ID NO:25). The complete nucleotide sequence (SEQ ID NO:26) showed 98.4% homology to the nucleotide.<br>
sequence of SNDHai from G. oxydans DSM 17078 (SEQ ID NO: 1). The deduced amino acid sequence (SEQ ID NO:27) showed 100% identity to the amino acid sequence of SEQ ID N0:2.<br>
Example 10: Increased Vitamin C production from L-sorbosone by increasing the SNDHai gene dosage<br>
The SNDHai gene with upstream and downstream flanking regions was amplified by PCR with chromosomal DNA of strain DSM 17078 as template and the primer set Nl (SEQ ID N0:28) and N2 (SEQ ID NO:29).<br>
The PCR was done with the GC-rich PCR system (Roche Diagnostics) according to the instructions of the supplier. The amplified DNA fragment was inserted into vector pCR2.1-TOPO (Invitrogen, Carlsbad, CA, USA). The resulting plasmid was then digested with HindSS. and Xhol. The HindSL-Xhol fragment including the SNDHai gene was ligated to vector pVKl00 (available from the American Type Culture Collection, catalog no. ATCC 37156) previously treated with MinfIIIand Xhol. The ligation mixture was used to transform E. coli TGI. The desired plasmid, designated pVK-P-SNDHai-T, was isolated from E. coli, and introduced into G. oxydans strain DSM 17078 by electroporation using standard methods (Electrocell manipulator ECM600, BTX Inc., San Diego, CA, USA).<br>
Cells of G. oxydans strains DSM 17078 and DSM 17078 carrying the plasmid pVK-P-SNDHai-T were cultivated in 50 ml of No. 5 medium containing 100 g/1 D-sorbitol, 0.5 g/1 glycerol, 15 g/1 yeast extract (Difco), 2.5 g/1 of MgSO4-7H2O, and 15 g/L of CaC03 in a 500 ml baffled shake flask at 30°C with shaking at 200 rpm. After 48 h of cultivation, the amounts of Vitamin C measured in the supernatant by HPLC in the two flasks were 110 mg/1 and 200 mg/1, respectively.<br>
Example 11: Production of Vitamin C from L-sorbosone or D-sorbitol by resting cells of recombinant microorganisms with increased SNDHai gene dosage<br>
The SNDHai gene of G. oxydans DSM 17078 (SEQ ID NO:1) with upstream and downstream flanking regions was amplified by PCR with chromosomal DNA of strain DSM 17078 as template and the primer set Nl (SEQ ID NO:28) and N2 (SEQ IDNO.-29).<br>
The PCR was done with the GC-rich PCR system (Roche Diagnostics GmbH) according to the instructions of the supplier. The amplified DNA fragment was     inserted into vector pCR2.1-TOPO (Invitrogen, Carlsbad, CA, USA). The resulting plasmid was then digested with HindII and Xhol. The HindIII-XhoI fragment including the SNDHai gene was ligated to vector p VK100 (available from the American Type Culture Collection, catalog no. ATCC 37156) previously treated with HindIII and Xhol. The ligation mixture was used to transform E. coli TG1. The desired plasmid, designated pVK-P-SNDHai-T, was isolated from E. coli, and introduced into G. oxydans strain DSM 17078 by electroporation using standard . methods (Electrocell manipulator ECM600, BTX Inc., San Diego, CA, USA).<br>
Three independent transformants, designated DSM 17078(pVK-P-SNDHai-T) clone number 1,2, and 3, together with the parental strain G. oxydans DSM 17078, were each grown on No. 3BD agar and MB agar media. The cells were scraped from the   -plates and used for resting cell reactions (1 % L-sorbosone as the substrate) as described in Example 9. After growth on No. 3BD agar, hi the resting cell assay strain DSM 17078 produced 2.5 g/1 Vitamin C, while strains DSM 17078(pVK-P-SNDHai-T) clones 1, 2 and 3 produced 4.2,4.1 and 4.2 g/1 Vitamin C, respectively. After growth on MB agar, hi the resting cell assay strain DSM 17078 produced 0.12 g/1 Vitamin C, while strains DSM 17078(pVK-P-SNDHai-T) clones 1, 2 and 3     v produced 1.8, 2.5 and 0.94 g/1 Vitamin C, respectively.<br>
Another reaction was carried out using cells of G. oxydans DSM 17078 and clone 2 (see above) cultivated in 50 ml of No. 5 medium (100 g/1 D-sorbitol, 0.5 g/1 glycerol, 15 g/1 yeast extract, 2.5 g/1 MgSO4-7H20,15 g/1 CaC03) in duplicate 500ml baffted shake flasks at 30QC with shaking at 220 rpm for 3 days. From one flask for each strain, the resulting broth was centrifuged at 500 rpm to remove CaCOs. The supernatant from this step was then centrifuged at 5,000 rpm to pellet the cells. The collected cells were re-suspended in 10 ml of 0.9% NaCl solution, and again centrifuged at 5,000 rpm to pellet the cells. The collected cells were re-suspended in water and used to inoculate 1 ml of production medium (20 g/1 D-sorbitol, 3 g/1 Nad 10 g/1 CaCOs) in 10 ml reaction tube at a final resting cell density corresponding to 5 OD units at 600 nm. After 20 h reaction time at 30°C and 220 rpm, the supernatant harvested from the production flask contained 360 and 760 mg/1 Vitamin C, respectively for strains DSM 17078 and DSM 17078 overexpressing SNDHai. In contrast, after 72 h the supernatant harvested from the remaining growth medium contained 0 and 440 mg/1 Vitamin C, respectively.<br>
Example 12: Production of Vitamin C from L-sorbosone in resting cells of E. coli<br><br>
The SNDHai gene without stop codon named SNDHai-1, corresponding to nucleotides 1-2364 of SEQ ID NO: 1, was amplified from strain DSM 17078 chromosomal DNA by PCR (Roche High Fidelity kit) using the primer pair SNDHai-Nde (SEQ ID N0:30) and SNDHaiHis-X (SEQ ID NO:31).<br>
The amplified DNA was cloned into pCR2.1-TOPO (Invitrogen, Carlsbad, CA, USA) to obtain pCR2.1-TOPO-SNDHai-l, whose SNDHai sequence was confirmed to be correct by nucleotide sequencing. Then the SNDHai-1 gene was cut out with Ndel and Xhol and ligated between Ndel and Xhol sites of pET-21b(+) (Novagen, Madison, WI, USA) to produce pET21b-SNDHaiHis; 6xHis was added at the C-terminus of SNDHai, The pET21b-SNDHaiHis was introduced into E. coli BL21 (DE3).<br>
Five ml of one overnight culture of £ coli BL21 (DE3)/pET21b- SNDHaiHis in LB with carbenicillin 50 ug/ml was inoculated into 200 ml of the same medium. The cells were cultivated at 230 rpm at 37°C for 2 h, then induced with 1 mM IPTG and continued to be cultivated at 230 rpm at 25°C for 3 h. The resulting culture was centrifuged and washed twice with saline and the cell pellet was resuspendedin 2 ml of water. The cells were used for resting cell reaction with the reaction mixture (500 ul in 5 ml tube) containing cells at OD600=10,1% sorbosone monohydrate, 5 u.M PQQ, 5mM MgCl2,0.3% NaCl, and 1% CaCO3 conducted at 30°C for 15 h. 0.14 g/L of Vitamin C was produced after incubation for 15 h. When the resting cell reaction was done with 1 uM PQQ (the other conditions were the same as those described above), 0.05 g/L of Vitamin C was produced after incubation for 3 h.<br>
Example 13: Production of Vitamin C from D-sorbitol by resting cells of<br>
recombinant microorganisms with increased SNDHai gene dosage<br>
Cells of G. oxydans DSM 17078 overexpressing SNDHai are grown in 50 ml of No. 5<br>
medium containing 100 g/1 D-sorbitol, 0.5 g/1 glycerol, 15 g/1 yeast extract (Fluka<br>
BioChemika, Buchs, Switzerland), 2.5 g/1 MgS04-7H2O and 15 g/1 CaCO3 in a 500-<br>
ml baffled shake flask at 30°C with shaking at 180 rpm for 48 h. The resulting cell<br>
suspension is used to inoculate a 2-L bioreactor, called growth vessel (Biostat-MD, B.<br>
Braun Melsungen, Melsungen, Germany) containing 1.251 of medium composed of<br>
100 g/1 D-sorbitol, 15 g/1 yeast extract (Fluka BioChemika, Buchs, Switzerland), 2.5<br>
g/1 MgSO4-7H20,0.3 g/1 KH2PO4 and 0.12 g/1 CaSO4. Cells are cultivated at 30°C, 1<br>
1/min aeration rate, the pH is controlled to 5.7 with a 25% solution of NaaCOa,<br>
dissolved oxygen is controlled to 10% saturation by varying the stirring speed. After<br>
24 h, the cell density measured as absorption units at 600 nm is 20. At this time<br>
point, a feed solution containing 100 g/1 D-sorbitol, 15 g/1 yeast extract (Fluka:;<br><br>
BioChemika, Buchs, Switzerland), 2.5 g/1 MgSO4-7H2O, 0.3 g/1 KH2PO4 and 0.12 g/1 CaS04 is fed into the growth vessel at a feed rate of 125 ml/h, and broth is continuously harvested at a harvest rate of 125 ml/h. By this means, the volume in the growth vessel is kept constant at 1.25 1. Other process parameters continue to be controlled as mentioned above.<br>
This broth is continuously fed at a rate of 125 ml/h into a second reactor, called production vessel, filled with 5 1 production medium containing 100 g/1 D-sorbitol, 0.3 g/1 NaCl and 0.12 g/1 CaSO4, and the temperature is kept at 30°C, pH at 7.0 by controlling with a 20% solution of NaOH. The aeration rate is kept constant at 10 1/min, and dissolved oxygen is controlled at 20% by varying the stirrer speed. Production medium with the same composition is also continuously fed to the production vessel at a feed rate of 375 ml/h. The vessel volume is kept constant at 5 "1 by continuously harvesting supernatant at 500 ml/h rate, resulting as a filtrate stream from a crossflow ultrafiltration module with 500 kDa pore size (UFP-500-E-9A, Amersham Biosciences), through which the cell suspension harvested from the production vessel is pumped at 501/h using a Masterflex pump. The retentate flow is pumped back into the vessel. Once the cell density in the production vessel reaches 100 absorption units at 600 nm, cells start to be harvested from the concentrated cell stream flowing back into the production vessel at a rate of.25 ml/h, in order to keep : the cell density in the production vessel constant.<br>
The harvest stream of cell-free supernatant contains 4 g/1 Vitamin C and is continuously fed at a rate of 500 ml/h into a collecting vessel with a double jacket at 30°C (Ecoline Rel 12, Lauda, Lauda-Koenigshofen, Germany). This vessel feeds continuously supernatant to the diluate compartment of a two-compartment electrodialysis unit (stack containing 10 cell pairs with cation exchange membranes CMX-S and anion exchange membranes ASM, total membrane area 0.2 m2, from Eurodia Industries, Wissous, France) at a rate of 1801/h, and a constant stream is . pumped out of the vessel to keep its volume constant at 2 1. Another vessel with double jacket containing initially deionized water at 30°C is continuously fed with fresh deionized water at a rate of 62.5 ml/h, pumps constantly aqueous solution into the concentrate compartment of the electrodialysis unit at a rate of 200 1/h, and a constant harvest stream is pumped out of the vessel. Feed solutions are pumped to the electrodialysis stack using peristaltic pumps (7518-00, Masterflex, USA), and recirculation of solutions through each electrodialysis compartment is done with help of rotary pumps (MD-20, IWAK, Tokyo, Japan). During the whole process, 14 V areapplied to the electrodialysis stack (power source FuMATech TS001/5, St. Ingbert, Germany). The concentration of Vitamin C in the harvest stream is 16 g/1.<br>
Example 14: Purification of Vitamin C produced by a resting eel) reaction via downstream processing steps<br>
The harvest stream of Example 14 containing 16 g/1 Vitamin C is fed to a chelating resin (Amberlite IRC 748, Rohm and Haas, Philadelphia, PA, USA) to eliminate divalent cations from the stream. It is then collected in a cooled vessel (feed vessel), and when 101 have been collected, they are processed in batch mode through a bipolar membrane electrodialysis unit (stack containing 7 Neosepta BP1 / CMB membranes, total membrane area 0.14 m2, from Eurodia Industries, Wissous, France). This solution is pumped at 2001/h through the feed compartment of the electrodialysis unit, and recycled into the feed vessel. Another cooled vessel (concentrate vessel) containing initially 51 of a 2 g/1 NaOH solution is pumped at 100 1/h through the concentrate compartment of the bipolar membrane electrodialysis unit. By applying a maximal voltage of 25 V and maximal electric current of 20 A,'sodium cations from the feed compartment are transferred to the concentrate compartment, and thus the sodium form of Vitamin C present in the feed stream is converted into     • the corresponding free acid form. After reaching 90% conversion yield, the process is stopped. In the concentrate vessel, 61 of solution containing 7.5 g/1 NaOH are collected in the diiuate vessel, 91 solution containing about 16 g/1 Vitamin C in its free acid form and 1.6 g/1 Vitamin C in its sodium salt form are further processed through a cation exchange resin (Amfaerlite FPC 21 H, Rohm and Haas, Philadelphia, PA, USA), in order to increase conversion yield of the sodium salt into the free acid! form to about 99%. Alternatively, the 101 solution containing 16 g/1 Vitamin C in its sodium salt form coming from the electrodialysis step is directly treated by cation exchange resin, being converted to the free acid form at 99% yield. The stream of Vitamin C in the form of the free acid, obtained by either of the methods described above, is then further processed by a sequence of the following steps: anion exchange, activated carbon treatment, concentration, crystallization, filtration of the crystals, and drying. The final purity of the obtained crystals is 98%, and the yield obtained with the combined downstream processing steps is 80%.<br>
Example 15: Preparation of chromosomal DNA and amplification of SMS 05 DNA fragment by PCR<br>
Chromosomal DNA of Gluconobacter oxydans DSM 17078 were prepared from the cells cultivated at 30°C for 1 day in mannitol broth (MB) liquid medium consisting of<br>
25 g/1 mannitol, 5 g/1 of yeast extract (Difco), and 3 g/1 of Bactopeptone (Difco) by the method described by Sambrook et al (1989) "Molecular Cloning: A Laboratory Manual/Second Edition", Cold Spring Harbor Laboratory Press).<br>
A DNA fragment was prepared by PCR with the chromosomal DNA prepared above and a set of primers, Pf (SEQ ID NO:46) and Pr (SEQ ID N0:47). For the reaction, the Expand High Fidelity PCR kit (Roche Diagnostics) and 10 ng of the chromosomal DNA was used in total volume of 100u1 according to the supplier's instruction to have the PCR product containing SMS 05 DNA sequence (SEQ ID NO:44). The PCR product was recovered from the reaction and its correct sequence confirmed.<br>
Example 16: Disruption of SMS 05 gene in G. oxydans DSM 17078<br>
The PCR product obtained in Example 15 was cloned in an E. coli vector pCR2.1-TOPO and transform E. coli TGI to have a Apr transformant carrying pCR2.1-SMS 05. Then, thepCR2.1-SMS 05 was subjected to two PCR reactions: (i) PCR1 with a. set of primers Psms5No (SEQ ID NO:78) and Psms5Ni (SEQ ID NO:79) to feve a fragment SMS5No-Ni. and (ii) PCR2 with a set of primers PsmsSCi (SEQ ID N0: 76) and Psms5Co (SEQ ID N0:77). The resulting two PCR products were used for the 3rd PCR reaction, PCR3, with a set of primers PsmsSNo and PsmsSCo to yield SMS5dNo-Co. Then, the fragment was digested with Pstl and Hindm and ligated with pKI9mobsacB (A. Punier et al. Gene 145, 69-73,1994) digested with Psfi. and HindII and used to transform E. coli TG to have Kmr colonies carrying pK19mobsacB-dSMS 05. The plasmid was used for transformation ofG. oxydans DSM 17078 by elect-operation.<br>
One of the Kmf colony was streaked on MB+Km 50 ug/ml (MK) plates and then checked for Km resistance and sucrose sensitivity on MK and MB+sucrose 10% (MSuc) plates, respectively to confirm Kmr and sucrose5.   Then, one Kmr and sucrose8 colony was grown on MB agar plate and resulting grown cells were scraped, diluted appropriately and spread on MB agar. The resulting 100 colonies were streaked on MB, MK, and Msuc agar plates to isolate Kms sucroser colonies. One of the resulting strains was designated as G. oxydans DSM 17078-dSMS 05.<br>
Example 17: Production of Vitamin C from D-sorbitol, L-sorbose or L-sorbosone using resting cells grown on 3BD agar medium containing 7% L-sorbose<br>
Cells of G. oxydans DSM 17078 and G. oxydans DSM 17078- dSMS 05 were grown<br>
at 27°C for 3 days on No. 3BD agar medium containing 70 g/1 L-sorbose, 0.5 g/1	<br>
glycerol, 7.5 g/1 yeast extract (Difco), 2.5 g/i MgSO4-7H2O! 10 g/1 CaCO3 and 18 g/1 agar (Difco).<br>
Cells were scraped from the agar plates, suspended in distilled water and used for resting cell reactions conducted at 30°C with shaking at 220 rpm. At the conclusion of the incubation period, the reaction mixtures were analyzed by high performance liquid chromatography (HPLC) using the method described in Example 2.<br>
A series of resting cell reactions (0.5 ml reaction mixture in 5 ml reaction tube) was carried out with 2% D-sorbitol or with 2% L-sorbose, and all reaction mixtures further contained 0.3% NaCl, 1% CaCO3 and cells at a final concentration of 5 absorbance units at 600 nanometers (OD600). After 20 h incubation time, G, oxydans DSM 17078 produced 270 mg/I or 670 mg/1 of Vitamin C, respectively from 2% D-sorbitol or 2% L-sorbose, respectively. In comparison, strain G. oxydans DSM 17078-dSMS 05 produced 1540 mg/1 or 1990 mg/I of Vitamin C, respectively.<br>
Example 18:   Concurrent mutagenesis of genes encoding SNDHai and SMS 05 results in improved production of Vitamin C from L-sorbosone<br>
The pVK-P-SNDHai-T plasmid was introduced into G oxydans DSM 17078- dSMS 05 by electroporation.   Cells of G. oxydans DSM 17078, G. oxydans DSM 17078-. dSMS 05 and G. oxydans DSM 17078-dSMS 05/pVK-P-SNDHai-T were grown at 27°C for 3 days on No. 3BD agar medium containing 70 g/1 L-sorbose, 0.5 g/1 glycerol, 7.5 g/1 yeast extract (Difco), 2.5 g/1 MgSO4-7H20,10 g/1 CaCO3 and 18 g/1 agar (Difco).<br>
Cells were scraped from the agar plates, suspended hi distilled water and used for resting cell reactions conducted at 30°C with shaking at 220 rpm. A series of reactions (0.5 ml reaction mixture in 5 ml reaction tubes) was carried out hi 2% L-sorbosone hi reaction mixtures further containing 0.3% NaCl, 1% CaCOs was incubated with cells at a final concentration of OD600 - 5. After incubation periods of 5h, 20h, and 3 Oh, samples of the reaction mixtures were analyzed by high performance liquid chromatography according to the method described in Example 2. The following concentrations of Vitamin C were measured in the supernatants:<br><br>
(Table Removed)<br>
Example 19:   Concurrent mutagenesis of genes encoding SNDHai and SMS 05 results in improved production of Vitamin C from D-sorbitol in liquid cultures<br>
Cells of G. oxydans DSM 17078, G. oxydans DSM 17078/pVK-P-SNDHai-T, G. oxydans DSM 17078-dSMS 05, and G. oxydans DSM 17078-dSMS 05/pVK-P-SNDHai-T were grown in 50 ml of No, 5 medium containing 100 g/1 D-sorbitol, 0.5 g/1 glycerol, 15 g/1 yeast extract (Fluka BioChemika, Buchs, Switzerland), 2.5 g/1 MgSO4-7H20 and 15 g/1 CaCO3 in a 2-1 baffled shake flask at 30°C with shaking at 180 rpm. After 48 h, the optical density ODeoo of the two cultures was measured, and the value obtained was used to calculate the volume of inoculum into two other shake flasks for each strain (duplicate experiments) containing 50 ml of medium No. 5, in order to obtain standardized inoculum density in the second cultures corresponding to OD600 = 0.12. The flasks were incubated at 30°C with shaking at 180 rpm. After 48 h and 96 h, samples were taken for analysis using the high performance liquid chromatography method described above. The following concentrations of Vitamin C were measured in supernatants:<br><br>
(Table Removed)<br>
Example 20: Overexpression of RCS 21 in G. oxydans DSM 17078 using an integrative system<br>
For the overexpression of RCS 21, the promoter of RCS 21 may be replaced by the strong constitutive modified Psndh promoter [SEQ ID NO: 204]. In order to achieve this, a DNA fragment is built up by Long Flanking Homology (LFH)-PCR consisting of 500-bp of the upstream region of RCS 21, a kanamycin-resistance cassette, the Psndh-promoter fused to a modified ribosome binding site and the first 500-bp of RCS 21. In order to construct the DNA fragment, firstly the single parts are amplified by PCR using the GC-rich PCR kit (Roche Molecular Biochemicals). The upstream region of RCS 21 is amplified using primer pair RCS 21 US+1 [SEQ ID NO: 213] and Km RCS 21 US-1 [SEQ ID NO: 214 containing complementary kanamycin-resistance cassette overhang sequence at 5'-end]. The Psndh promoter fragment is amplified using primer pair Km Psndh+1 [SEQ ID NO: 207 containing complementary kanamycin-resistance cassette overhang sequence at 5'-end] and RCS<br>
21 Psndh-1 [SEQ ID NO: 217 containing complementary RCS 01 overhang sequence at 5'-end]. The first 500-bp of RCS 21 is amplified using primer pair Psndh RCS 21+1 [SEQ ID NO: 215 containing complementary Psndh promoter overhang sequence at 5'-end] and RCS 21-1 [SEQ ID NO: 216]. In these cases G. oxydans DSM17078 genomic DNA may be used as a template. The kanamycin-resistance' cassette is amplified using plasmid pUC4K as a template and primer pair Km-rl [SEQ ID NO: 211] andKm-1 [SEQ ID NO: 212]. The PCR reaction conditions consist of  . 35 cycles of denaturation at 94°C for 30 sec., annealing at 50°C for 30 sec. and extension at 72°C for 1 min. The individual PCR fragments are gel-purified, mixed and re-amplified using the primer pair RCS 21 US+1 / RCS 21 -1 to amplify a full length product whereby the Psndh promoter is inserted upstream of RCS 21. The PCR reaction conditions for the second round reaction consist of 94°C, 2 rnin., then 10 cycles of [94°C, 30 sec., 63°C, 30 sec., 68CC, 6 min.], followed by 20 cycles of [94°C, 30 sec., 63°C, 30 sec., 68°C, 6 min. with an additional 20 sec. per cycle] and a final extension at 68°C for 10 min.<br>
The PCR product is transformed directly into competent G. oxydans DSM17078 cells<br>
and transformants are selected on mannitol broth agar medium containing kanamycin<br>
to a final concentration of 50 u.g/ml. Several putative transformants may be observed<br>
of which several then can be analysed by PCR using primer pair RCS 21 US+1 / RCS<br>
21 -1 to verify that the DNA fragment has inserted into the genome via a double<br>
crossover. Strains showing the correct size PCR product have the PCR product<br>
sequenced. Strains with the correct sequence are named DSM17078-RCS 21 upl and<br>
DSM17078-RCS21up2.	<br>
Example 21: Concurrent mutagenesis of genes encoding SNDHai and RCS 21 results in improved production of Vitamin C from D-sorbitoI.<br>
The pVK-P-SNDHai-T plasmid is introduced into G. oxydans DSM 17078 and G. oxydans DSM 17078-RCS 21 upl by electroporation. Cells of G. oxydans DSM 17078/pVK-P-SNDHai-Tand G. oxydans DSM 17078-RCS 21 upl/pVK-P-SNDHai-T are grown at 27°C for 3 days on No. 3BD agar medium containing 70 g/1 L-sorbose, 0.5 g/1 glycerol, 7.5 g/1 yeast extract (Difco), 2.5 g/1 MgS047h203 10 g/1 CaCO3 and 18g/Iagar(Difco).<br>
Cells are scraped from the agar plates, suspended in distilled water and used for   -resting cell reactions conducted at 30°C with shaking at 220 rpm. A series of reactions (0.5 ml reaction mixture in 5 ml reaction tubes) is carried out in 2% L-sorbitol in reaction mixtures further containing 0.3% NaCl, 1% CaCOa is incubated ..;;..:<br>
with cells at a final concentration of OD6oo= 5. After an incubation period of 24h, samples of the reaction mixtures are analyzed by high performance liquid chromatography according to the method described in Example 2.<br>
The supernatant of the reaction mixture incubated with cells of G. oxydans DSM 17078-RCS 21 upl/pVK-P-SNDHai-T contains at least 20% more Vitamin C than the supernatant of the reaction mixture incubated with cells of G. oxydans DSM 17078/pVK-P-SNDHai-T.<br>
Example 22: Purification of SNDHai<br>
Cells of a microorganism capable of producing SNDHai cultivated by fed-batch fermentation (for cultivation see Example 3) were suspended in 25 ml of phosphate buffer (20 mM, pH 7.0) containing MgCI2, 2 mM, dithiothreitol (DTT), 1 mM, and 2-3 EDTA-free protease inhibitor tablets (Roche Diagnostics GmbH). The cell suspension was treated three tunes with a French Pressure cell. Subsequently, 25 ml of phosphate buffer (20 mM, pH 7.0) containing 2mM MgCb and 1 M NaCl were added and the suspension was ultracentrifuged (30.000 rpm, 60 rain, 4°C). The pellet containing the membrane fraction was washed with phosphate buffer (20 mM, pH 7.0) containing 2 mM MgCk and 500 mM NaCl and then suspended in an appropriate amount of phosphate buffer (20 mM, pH 7.0) containing 2mM MgCl2. N-Octylglucoside (Fluka) was then added at a final concentration of 2 % (w/v) and the suspension was incubated for 90 min with gentle stirring on ice. After centrifugation (20.000 rpm, 60 min, 4°C) the clear reddish supernatant was collected and polyethylene glycol 6000 (Fluka) at a final concentration of 15 % (w/v) was added. After incubation for 60 min at 4°C with gentle shaking followed by centrifugation (10.000 rpm, 30 min, 4CC), the pellet was dissolved in Tris-HCl buffer (20 mM, pH 7.6) containing 2 mM MgCl* and 0.5 % (w/v) lauryl sulfobetaine (Fluka). After gentle shaking at 4°C overnight the solution was centrifuged (20.000 rpm, 30 min, 4°C). The supernatant was collected and further purified as follows.<br>
The following purification steps were done at4°C on an AKTA Explorer 10 S-system (Amersham Biosciences) with software UNICORN 3.1. Typical flow rates for ion exchange chromatography were in the range of 1 -2 ml/min. Protein elution was monitored at 280 nm and SNDHai-active fractions were determined using the standard photometric assay at all stages of the purification (s. below) or the product assay with purified fractions.<br>
The clear supernatant IV was desalted in 2.5 mi-portions on a Sephadex G 25-gel filtration column (void volume: 2.5 ml) using 20 mM Tris-HCl buffer (pH 7.6) containing 2 mM MgCh and 0.5 % (w/v) lauryl sulfobetaine.<br>
SNDHai-positive fractions were pooled and an aliquot (approximately 10 ml) was put on a strong anion exchange column (e.g. Mono-Q HR, Amersham Biosciences, column volume: 8 ml) which had been equilibrated prior to use with buffer Al (10 mM Tris, 10 mM BisTris, 10 mM MES, 2 mM MgCl2, 0.5% lauryl sulfobetaine, pH 7.6). Non-binding proteins were eluted with 100% buffer Al and after 4 column volumes a linear pH-gradient in 6 column volumes to 100% buffer Bl (Tris, 10 mM; BisTris, 10 mM; MES, 10 mM; MgCl2,2 mM, and lauryl sulfobetaine, 0.5%, pH 4.7) was applied followed by 8 column volumes of 100% buffer Bl. SNDHai eluted at a pH-value of approximately 6.5, which is very close to the pi-value of 6.52 calculated from the amino acid sequence. Active fractions were pooled, diluted with the same amount of HEPES-buffer (50 mM; pH 8.0) containing 2 mM MgCl2 and 0.5% lauryl sulfobetaine (final volume: 15-20 ml), and applied to another strong anion exchange column (e.g. Mono-Q HR, Amersham Biosciences, column volume: 1 ml) which had been equilibrated with buffer A2 (15 mM HEPES, 2 mM MgCl2,0.5% lauryl sulfobetaine, pH 7.6). Non-binding proteins were eluted with 100% buffer A2 and after 4 column volumes a linear salt-gradient in 20 column volumes to 40% buffer B2 (HEPES, 15 mM; MgCl2,2 mM, LiCl, 1 M, and lauryl sulfobetaine, 0.5 %, pH 7.6) was applied followed by a step gradient to 100% buffer B2. SNDHai eluted around 150 mM LiCl. Active fractions showed one single band at approximately 85 kDa in SDS gel electrophoresis.<br>
Example 23: Photometric assay for SNDHai.<br>
The reaction mixture for the photometric SNDHai-activity measurement consisted of 0.196 mM nitroblue tetrazolium chloride (NBT), 0.137 mM phenazine methosulfate (PMS), 20.4 mM L-sorbosone, and enzyme solution in a final volume of 1.0 ml of 0.1 M sodium phosphate buffer, pH 7.5. The reaction was started with the addition of enzyme, and the enzyme activity was measured in a cuvette with 1-cm light path as the initial reduction rate of NBT at 570 nm(T = 25°C). One unit of the enzyme activity'was defined as the amount of enzyme catalyzing the reduction of 1 uM NBT per minute. The extinction coefficient of NBT at pH 7.5 was taken as 100 mM"1 cm"1. Two kinds of reference cuvettes were used for the activity determination: one contained the above-mentioned components except for L-sorbosone and another one contained all components except for the enzyme solution.<br>
Example 24: Product assay for SNDHai.<br>
Pure SNDHai-containing fractions (see above) were analyzed directly for L-ascorbic acid production from L-sorbosone with an assay of the following composition (0.5 ml total volume): 0.14 mg/ml of purified and desalted SNDHai, 50 mM phosphate buffer (pH 6.5), 8 mg/ml bovine serum albumin (BSA), 100 mM L-sorbosone, 1 mM PMS, 5 mM CaCl2,50 uM PQQ-K2. The assay was conducted in appropriate reaction tubes at 25°C with sufficient shaking (900 rpm on a benchtop shaker) under exclusion of light.<br>
Samples were analyzed by high performance liquid chromatography (HPLC) using an Agilent 1100 HPLC system (Agilent Technologies, Wilmington, USA) with a LiChrospher-100-RP18 (125 x 4.6 mm) column (Merck, Darmstadt, Germany) attached to an Aminex-HPX-78H (300 x 7.8 mm) column (Biorad, Reinach, Switzerland). The mobile phase was 0.004 M sulfuric acid, and the flow rate was 0.6 ml/min. Two signals were recorded using a UV detector (wavelength 254 nm) in combination with a refractive index detector. In addition, the identification of the,L-ascorbic acid was done using an amino-column (YMC-Pack Polyamine-II, YMCi Inc., Kyoto, Japan) with UV detection at 254 nm. The mobile phase was 50 mM " NH4H2PO4 and acetonitrile (40:60).<br>
Example 25: Presence of the SMS and RCS genes and equivalents thereof in other organisms<br>
The presence of the SMS and RCS polynucleotide sequences and/or equivalents showing similarity/identity to these sequences as described in other organisms than the ones disclosed herein before may be determined by a simple DNA hybridization experiment. Genomic DNA is extracted from an organism belonging to e.g. Gluconobacter, Acetobacter, Pseudomonas, Paracoccus, Rhodopseudomonas, Pantoea, Esckerichia, Saccharomyces, Aspergillus or mouse, in particular the organisms mentioned in the tables B and C.<br>
Strains of Acetobacter aceti subsp. xylinum IFO 13693 and IFO 13773 are grown at 27°C for 3 days on No. 350 medium containing 5 g/1 Bactopeptone (Difco), 5 g/1 yeast extract (Difco), 5 g/1 glucose, 5 g/1 mannitol, 1 g/1 MgS04-7H20, 5 ml/1 ethaCnol, and 15 g/1 agar. All other Acetobacter, Gluconacetobacter and all Gluconobacter strains are grown at 27°C for 3 days on mannitol broth (MB) agar medium containing 25 g/1 mannitol, 5 g/1 yeast extract (Difco), 3 g/1 Bactopeptone (Difco), and 18 g/1 agar (Difco). E. coli K-12 is grown on Luria Broth agar medium. The other strains are<br>
grown on medium recommended by the suppliers or according to methods known in the art. Genomic DNA is extracted as described by e.g. Sambrook et a/,, 1989, "Molecular Cloning: A Laboratory Manual/Second Edition", Cold Spring Harbor Laboratory Press) from a suitable organism as, e.g. mentioned in Table B and C.<br>
Genomic DNA preparations are digested with restriction enzymes such as EcoRl or HindII, and 1 jig of the DNA fragments are separated by agarose gel electrophoresis (1% agarose). The gel is treated with 0.25 N HC1 for 15 min and then 0.5 N NaOH for 30 min, and then blotted onto nitrocellulose or a nylon membrane with Vacuum Blotter Model 785 (BIO-RAD Laboratories AG, Switzerland) according to the instruction of the supplier. The resulting blot is then brought into contact/hybridized with a solution containing DNA probes of SMS, STS, RCS or VCS, which may be prepared by using the PCR-DIG labeling kit (Roche Diagnostics) and for each gene a specific set of primers as disclosed in table A. A result of such a blot is depicted in the first column of Table B and C respectively.<br>
Table A: SEQ ID NO's of Primer pairs<br><br>
(Table Removed)<br>
The hybridization may be performed under stringent or highly stringent conditions. A preferred, non-limiting example of such conditions are hybridization in 6x sodium chloride/sodium citrate (SSC) at about 45°C, followed by one or more washes in 1x SSC, 0.1% SDS at 50°C, preferably at 55°C, more preferably at 60°C and even more preferably at 65°C. Highly stringent conditions include, for example, 2 h to 4 days incubation at 42°C in a solution such as DigEasyHyb solution (Roche Diagnostics GmbH) with or without 100 u.g/ml salmon sperm DNA, or a solution comprising 50% formamide, 5x SSC (150 mM NaCl, 15 mM trisodium citrate), 0.02% sodium dodecyl sulfate, 0.1% N-lauroylsarcosine, and 2% blocking reagent (Roche Diagnostics GmbH), followed by washing the filters twice for 5 to 15 min hi 2x SSC and 0.1 % SDS at room temperature and then washing twice for 15-30 min in 0.5x SSC and 0.1% SDS or O.lx SSC and 0.1% SDS at 65-68°C. To detect DNA fragments with lower identity to the probe DNA, final washing steps can be done at lower temperatures such as 50-65°C and for shorter washing time such as 1-15 min.<br>
The genes corresponding to the positive signals within the respective organisms shown in Table B or C can be cloned by a PCR method well known in the art using genomic DNA of such an organism together with a suitable primer set under the following conditions: 5 to 100 ng of genomic DNA is used per reaction (total volume 50 u,I), Expand High Fidelity PCR system (Roche Diagnostics) can be used with reaction conditions consisting of 94°C for 2 min; 30 cycles of (i) denaturation step at 94°C for 15 sec, (ii) annealing step at 60°C for 30 sec, (iii) synthesis step at 72°Cf0r 0.5 to 5 min depending to the target DNA length (1 min /I kb); extension at 72°C for 7 min. Alternatively, one can perform a PCR with degenerate primers, which can be synthesized based on the ammo acid sequence of the corresponding SMS or RCS protein disclosed herein or amino acid sequences as consensus sequences selected by aligning several amino acid sequences obtained by a sequence search program such as BLASTP (or BLASTX when nucleotide sequence is used as a "query sequence") to find proteins having a similarity to the specific protein sequence.. For PCR using degenerate primers, temperature of the second annealing step (see above) can be lowered to 55°C, or even to 50-45°C. Results of these experiments are shown, in column 2 and 3 of the Table B and C respectively.<br>
Samples of the PCR reactions are separated by agarose gel electrophoresis and the bands are visualized with a transilluminator after staining with e.g. ethidium bromide, isolated from the gel and the correct sequence is confirmed.<br>
Consensus sequences mentioned above might be amino acid sequences belonging to certain categories of several protein domain/family databases such as PROSITE (database of protein families and domains), COGs (Cluster of Ortholog Groups), , CDD (Conserved Domain Databases), pfam (large collection of multiple sequence . alignments and hidden Markov models covering many common protein domains and families). Once one can select certain protein with identical/similar function to the protein of this invention from proteins containing domain 01 family of such databases, corresponding DNA encoding the protein can be amplified by PCR using the protein sequence or its nucleotide sequence when it is available in public databases.<br>
Disruption ofgenes and equivalents in other organisms for production of Vitamin C<br>
In order to improve Vitamin C production in a suitable microorganism which is capable to directly produce Vitamin C from a given substrate, the selected gene and equivalents as e.g. a PCR product obtained as described in the above paragraphs can be disrupted in accordance to examples 15 - 19 to generate a knockout mutant<br>
carrying equivalent gene::Km. Suitable host strains for generation of such knockout mutants may be selected from e.g. Gluconobacter strains listed in Table B, in particular e.g. G. oxydans FO 3293, G. oxydans IFO 3292, G. oxydans ATCC 621H, G. oxydans WO 12528, G. oxydans WO 3291, G. axydansIFO 3255, G. oxydans IFO 3244, G, cerinusIFO 3266, G.frateurii IFO 3260, G. oxydans IFO 3287, Acetobacter aceti subsp. orleanus IFO 3259, Acetobacter aceti subsp. xylinum IFO 13693, Acetobacter aceti subsp. xylinum IFO 13773 and Acetobacter sp. ATCC 15164.<br>
The knockout mutant can be generated as follows: The PCR product obtained as described above is cloned in an E. coli vector pCR2.1-TOPO and used to transform E. coli TGI to have a Apr transformant carrying pCR2.1-gene X. Then, Kmr cassette isolated from pUC-4K (Amersham Bioscience, accession No. X06404) is inserted into one of the restriction site of the target gene with ligase and the resulting ligation product is used to transform E. coli TGI to have Apr Kmr transformant carrying pCR2.1 -gene X: :Km. The pCR2.1 -gene X: :Km plasmid prepared from the transformant is digested by two restriction enzymes selected from the multi-cloning site of the vector part to isolate a DNA fragment containing gene X::Km. The1 resulting DNA fragment is used to transform the host strain carrying the equivalent gene by electroporation to have the gene disruptant.<br>
Further modifications including SNDHai and other genes involved in the conversion of D-sorbitol, L-sorbose and/or L-sorbosone into Vitamin C within said strains may be generated to improve Vitamin C production by such strains according to the invention.<br>
In the resting cell reaction with 1% L-sorbosone as the substrate, the mutant strain can produce at least more than 20% Vitamin C compared to the wild-type strain.<br>
Production of Vitamin C using equivalents to genes which may be up regulated<br>
A PCR product obtained as described above can be used in an SNDHai<br>
overexpression system in accordance to examples 20-21 or in accordance with the<br>
foil wing procedure exemplified with RCS 21. This procedure apply mutates mutandis<br>
for the similar RCS and SMS genes.	"	:<br>
In order to improve Vitamin C production in a suitable microorganism which is capable to directly produce Vitamin C from a given substrate, the RCS 21 gene and equivalents as, e.g. a PCR product obtained as described in the above paragraphs, referred to herein as gene X, can be used in an overexpression system as described herein or can be cloned into pCR2.1-TOPO (Invitrogen, Carlsbad, CA, USA) and<br>
used to transform E. coli TGI to have a Apr transformant carrying pCR2.1-TOPO-gene X, i.e. carrying a PCR product. The insert is amplified with a set of primers, PfNdel [SEQ ID NO: 182 with CCCAT at the 5'-end] and PrHindII [SEQ ID NO: 183 with CCAAGCTT at the 5'-end], by PCR. Resulting PCR product is digested with Ndel and HindII and the fragment is inserted together with PcrtE-SD (Shine-Dalgarno) fragment (WO 02/099095) digested with Xhol and Ndel into pVKl 00 (ATCC 37156) between the sites of Xhol and HindIII E. coli TGI is transformed with the ligation product to have Tcr transformant carrying plasmid pVK-PcrtE-SD-gene X, which is then used to transform a suitable host, e.g. G. oxydans DSM 17078 by electroporation to have e.g. Tcr G. oxydans DSM 17078/pVK-PcrtE-SD-gene X.<br>
Production of Vitamin C using the recombinant cells of e.g. G. oxydans strains DSM<br>
17078 and the corresponding wild-type strain may be additionally transformed with<br>
SNDHai as described herein above.	.<br>
Further modifications including SNDHai and other genes involved in the conversion of D-sorbitol, L-sorbose and/or L-sorbosone into Vitamin C within said strains may be generated to improve Vitamin C production within such strains according to the invention.<br>
hi the resting cell reaction with 1% L-sorbosone as the substrate, the recombinant cells can produce at least more than 20% Vitamin C compared to the wild-type strain.<br>
To the following tables B and C:<br>
Signal 1: Detection of DNA on a blot with genomic DNA of different strains. Signal 2: Detection of DNA of different strains in a PCR reaction using primer pairs. Signal 3: Detection of DNA of different strains in a PCR reaction using degenerate primers.<br>
Table B cont: Equivalents of the SMS 02 gene in other organisms.<br><br>
(Table Removed)<br>
Table B cont.: Equivalents of the SMS 03 gene in other organisms. <br>
(Table Removed)<br><br><br>
Table B cont.: Equivalents of the SMS 04 gene in other organisms.<br><br>
(Table Removed)<br>
Table B cont.: Equivalents of the SMS 05 gene in other organisms.<br><br>
(Table Removed)<br>
Table C cont.: Equivalents of the RCS 21 gene in other organisms.<br><br>
(Table Removed)<br>
Table C cont.: Equivalents of the RCS 22 gene in other organisms.<br><br>
(Table Removed)<br>
Table C coat.: Equivalents of the RCS 23 gene in other organisms.<br><br>
(Table Removed)<br>
Table C cont.: Equivalents of the RCS 24 gene in other organisms.<br><br>
(Table Removed)<br>
Table C cont.: Equivalents of the RCS 25 gene in other organisms.<br><br>
(Table Removed)<br>
Table C cont: Equivalents of the SMS 12 gene in other organisms.<br><br>
(Table Removed)<br>
fable C cont.: Equivalents of the SMS 13 gene in other organisms.<br><br>
(Table Removed)<br>
Table C cont.: Equivalents of the SMS 14 gene in other organisms.<br><br>
(Table Removed)<br><br><br><br><br><br><br><br><br>
We claim:<br>
1.	A process for the fermentative production of Vitamin C, wherein a host cell is<br>
cultivated under suitable culture conditions that allow the direct production of<br>
Vitamin C from a carbon source obtainable from the D-glucose or D-sorbitol<br>
metabolization pathway and wherein the genome of said host cell is genetically<br>
engineered with DNA sequences comprising the following polynucleotides<br>
a)	a polynucleotide encoding L-sorbosone dehydrogenase according to SEQ ID NO:<br>
2 an active fragment or derivative thereof and<br>
b)	at least one polynucleotide encoding a protein selected from proteins which are<br>
involved in the Sorbitol/Sorbose Metabolization System (SMS) or proteins which<br>
are involved in the Respiratory Chain System (RCS);<br>
and optionally by isolation of Vitamin C from such cells or the culture medium.<br>
2.	A process according to claim 1, wherein the expression of a polynucleotide<br>
according to 1 a) refers to a polynucleotide sequence which is substantially identical to<br>
SEQ ID NO: 1 and/or which is selected from the group consisting of:<br>
a)	polynucleotides comprising the nucleotide sequence according to SEQ ID NO: 1<br>
b)	polynucleotides comprising a nucleotide sequence obtainable by nucleic acid<br>
amplification such as polymerase chain reaction, using genomic DNA from a<br>
microorganism as a template and a primer set according to SEQ ID NO:3 and<br>
SEQ ID NO:4;<br>
c)	polynucleotides comprising a nucleotide sequence encoding a fragment or<br>
derivative of a polypeptide comprising the amino acid sequence according to SBQ<br>
ID NO:2 or encoding a fragment or derivative of a polypeptide encoded by a<br>
polynucleotide of any of (a) or (b) wherein in said derivative or fragment one or<br>
more amino acid residues are conservatively substituted compared to said<br>
polypeptide, and said fragment or derivative has the activity of its counterpart<br>
polypeptide;<br>
d)	polynucleotides the complementary strand of which hybridizes under stringent<br>
conditions to a polynucleotide encoding a polypeptide comprising the amino acid<br>
sequence according to SEQ ID NO:2 or to a polynucleotide as defined in any one<br>
of (a) to (c) and which encode a polypeptide having L-sorbosone dehydrogenase<br>
activity; and<br>
e)	polynucleotides which are at least 70%, such as 85, 90 or 95% identical to a    .<br>
polynucleotide encoding a polypeptide comprising the amino acid sequence<br>
according to SEQ ID NO:2 or to a polynucleotide as defined in any one of (a) to (c) and which encode a polypeptide having L-sorbosone dehydrogenase activity.<br>
or<br>
the complementary strand of such a polynucleotide.<br>
3.	A process according to claim 1 or 2, wherein said host cell contains a<br>
polynucleotide encoding a RCS protein selected from the group consisting of proteins<br>
functioning in the biosynthesis of cofactors and prosthetic groups and proteins which<br>
function as carriers; in particular proteins involved in the biosynthesis or maturation<br>
of cofactors and/or their precursors such as FAD, NAD, NADP, PQQ, CoQ10<br>
cytochromes a, b, c, d, and heme.<br>
4.	A process according to any one of the claims 1 to 3, wherein said host cell<br>
contains a polynucleotide encoding a SMS protein selected from the group consisting<br>
of oxidoreductases [EC 1]; in particular oxidoreductases acting on the CH-OH group<br>
of donors [EC 1.1].<br>
5.	A process according to any one of claims 1 to 4 wherein said host cell contains<br>
at least one polynucleotide selected from<br><br>
-	the SMS group consisting of SEQ ID NO: 136, SEQ ID NO: 140, SEQ ID NO:<br>
144 and functional equivalents or homologues thereof; and<br>
-	the RCS group consisting of SEQ ID NO: 180, SEQ ID NO: 184, SEQ ID NO:<br>
188, SEQ ID NO: 192 , SEQ ID NO: 196 and functional equivalents or<br>
homologues thereof.<br><br>
6.	A process according to any one of claims 1 to 5 wherein said host cell is<br>
genetically engineered with at least one underexpressed or disrupted polynucleotide<br>
selected from the SMS group consisting of SEQ ID NO: 124, SEQ ID NO: 128, SEQ<br>
ID NO: 132, SEQ ID NO: 44 and functional equivalents or homologues thereof.<br>
7.	A process according to any one of the claims 1 to 6 wherein said host cell is<br>
cultivated in a aqueous nutrient medium under conditions that allow the direct<br>
production of Vitamin C from D-sorbitol or L-sorbose.<br>
8.	A process according to any of the claims 1 to 7, wherein the genetically<br>
engineered host cell is a procaryotic cell selected from the group consisting of<br>
Pseudomonas, Pantoea, Escherichia, Corynebacterium, Ketogulonicigenium and<br>
acetic acid bacteria like e.g., Gluconobacter, Acetobacter or Gluconacetobacter,<br>
preferably Acetobacter sp., Acetobacter aceti, Gluconobacter frateurii,<br>
Gluconobacter cerinus, Gluconobacter thailandicus, Gluconobacter oxydans,<br>
preferably Gluconobacter oxydans, more preferably Gluconobacter oxydans DSM 17078.<br>
9.	A host ce.ll expressing a polypeptide having L-sorbosone dehydrogenase activity<br>
with the amino acid sequence according to SEQ ID NO: 2 or a functional equivalent<br>
thereof, genetically engineered with at least one polynucleotide encoding a protein<br>
selected from proteins which are involved in the Sorbitol/SorboseMetabolization<br>
System (SMS) or proteins which are involved in the Respiratory Chain System<br>
(RCS).<br>
10.	A host cell according to claim 9, wherein the host cell is genetically engineered<br>
with at least one polynucleotide as defined hi any of the claims 2 to 6,<br>
11.	A host cell according to claim 9 or 10, wherein a protein is expressed selected<br>
from the group consisting of<br><br>
-	SEQ ID NO: 137, SEQ ID NO: 141, SEQ ID NO: 145<br>
-	SEQ ID NO: 181, SEQ ID NO: 185, SEQ ID NO: 189, SEQ ID NO: 193 , SEQ ID-<br>
NO: 197<br>
and functional equivalents or homologues thereof.<br>
12.	A host ceil according to any one of claim 9 to 11, wherein a protein is<br>
underexpressed or disrupted selected from the group consisting of SEQ ID NO: 125,<br>
SEQ ID NO: 129, SEQ ID NO: 133, SEQ ID NO: 45 and functional equivalents or<br>
homologues thereof.<br>
13.	A host cell according to any one of plaims 9 to 12 capable of directly producing"<br>
Vitamin C from D-sorbitol in quantities of 300 mg/1 or more when measured in a<br>
resting cell method after an incubation period of 20 hours.<br>
14.	A host cell according to any one of claims 9 to 12 capable of directly producing<br>
Vitamin C from L-sorbose in quantities of 800 mg/1 or more when measured in a<br>
resting cell method after an incubation period of 20 hours.<br>
15.	A host cell according to any one of claims 9 to 14, which is a procaryotic cell<br>
selected from the group consisting of Pseudomonas, Pantoea, Escherichia,<br>
Corynebacterium, Ketogulonicigenium and acetic acid bacteria like e.g.,<br>
Gluconobacter, Acetobacter tor Gluconacetobacter, preferably Acetobacter sp.,<br>
Acetobacter aceti, Gluconobacter frateurii, Gluconobacter cerinus, Gluconobacter<br>
thailandicus, Gluconobacter oxydans, preferably Gluconobacter oxydans, more<br>
preferably Gluconobacter oxydans DSM 17078.<br>
16.	A process for producing cells according to claim 9 comprising the step of<br>
genetically engineering cells with polynucleotides as defmed in any one :of claims 2 to<br>
6.<br>
17.	A process according to claim 16, wherein at least one-of these polynucleotide<br>
sequences is altereddn such# way that it leads to .an:improved yield flnd/or efficiency<br>
of production 0f Vitamin C prodwced by saidrmicrroorganism</td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzAwMS1kZWxucC0yMDA3LUFic3RyYWN0LSgwNC0wMy0yMDE0KS5wZGY=" target="_blank" style="word-wrap:break-word;">7001-delnp-2007-Abstract-(04-03-2014).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzAwMS1kZWxucC0yMDA3LWFic3RyYWN0LnBkZg==" target="_blank" style="word-wrap:break-word;">7001-delnp-2007-abstract.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzAwMS1kZWxucC0yMDA3LUNsYWltcy0oMDQtMDMtMjAxNCkucGRm" target="_blank" style="word-wrap:break-word;">7001-delnp-2007-Claims-(04-03-2014).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzAwMS1kZWxucC0yMDA3LUNsYWltcy0oMDktMDItMjAxNSkucGRm" target="_blank" style="word-wrap:break-word;">7001-delnp-2007-Claims-(09-02-2015).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzAwMS1kZWxucC0yMDA3LWNsYWltcy5wZGY=" target="_blank" style="word-wrap:break-word;">7001-delnp-2007-claims.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzAwMS1kZWxucC0yMDA3LUNvcnJlc3BvbmRhbmNlIE90aGVycy0oMDktMDItMjAxNSkucGRm" target="_blank" style="word-wrap:break-word;">7001-delnp-2007-Correspondance Others-(09-02-2015).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzAwMS1kZWxucC0yMDA3LUNvcnJlc3BvbmRlbmNlICBPdGhlcnMtKDIyLTA4LTIwMTMpLnBkZg==" target="_blank" style="word-wrap:break-word;">7001-delnp-2007-Correspondence  Others-(22-08-2013).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzAwMS1ERUxOUC0yMDA3LUNvcnJlc3BvbmRlbmNlLTE3MTIxNC5wZGY=" target="_blank" style="word-wrap:break-word;">7001-DELNP-2007-Correspondence-171214.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzAwMS1ERUxOUC0yMDA3LUNvcnJlc3BvbmRlbmNlLU90aGVycyAoMTQtMTAtMjAwOSkucGRm" target="_blank" style="word-wrap:break-word;">7001-DELNP-2007-Correspondence-Others (14-10-2009).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzAwMS1kZWxucC0yMDA3LUNvcnJlc3BvbmRlbmNlLU90aGVycy0oMDQtMDMtMjAxNCkucGRm" target="_blank" style="word-wrap:break-word;">7001-delnp-2007-Correspondence-Others-(04-03-2014).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzAwMS1kZWxucC0yMDA3LWNvcnJlc3BvbmRlbmNlLW90aGVycy5wZGY=" target="_blank" style="word-wrap:break-word;">7001-delnp-2007-correspondence-others.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzAwMS1kZWxucC0yMDA3LWRlc2NyaXB0aW9uIChjb21wbGV0ZSkucGRm" target="_blank" style="word-wrap:break-word;">7001-delnp-2007-description (complete).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzAwMS1kZWxucC0yMDA3LWZvcm0tMS5wZGY=" target="_blank" style="word-wrap:break-word;">7001-delnp-2007-form-1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzAwMS1kZWxucC0yMDA3LUZvcm0tMTMtKDEwLTA5LTIwMDcpLnBkZg==" target="_blank" style="word-wrap:break-word;">7001-delnp-2007-Form-13-(10-09-2007).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzAwMS1kZWxucC0yMDA3LWZvcm0tMTMucGRm" target="_blank" style="word-wrap:break-word;">7001-delnp-2007-form-13.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzAwMS1kZWxucC0yMDA3LWZvcm0tMi5wZGY=" target="_blank" style="word-wrap:break-word;">7001-delnp-2007-form-2.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzAwMS1ERUxOUC0yMDA3LUZvcm0tMyAoMTQtMTAtMjAwOSkucGRm" target="_blank" style="word-wrap:break-word;">7001-DELNP-2007-Form-3 (14-10-2009).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzAwMS1kZWxucC0yMDA3LUZvcm0tMy0oMDQtMDMtMjAxNCkucGRm" target="_blank" style="word-wrap:break-word;">7001-delnp-2007-Form-3-(04-03-2014).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzAwMS1kZWxucC0yMDA3LWZvcm0tMy5wZGY=" target="_blank" style="word-wrap:break-word;">7001-delnp-2007-form-3.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzAwMS1kZWxucC0yMDA3LWZvcm0tNS5wZGY=" target="_blank" style="word-wrap:break-word;">7001-delnp-2007-form-5.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzAwMS1kZWxucC0yMDA3LWdwYS5wZGY=" target="_blank" style="word-wrap:break-word;">7001-delnp-2007-gpa.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzAwMS1kZWxucC0yMDA3LU1hcmtlZC1DbGFpbXMtKDA5LTAyLTIwMTUpLnBkZg==" target="_blank" style="word-wrap:break-word;">7001-delnp-2007-Marked-Claims-(09-02-2015).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzAwMS1kZWxucC0yMDA3LXBjdC0yMTAucGRm" target="_blank" style="word-wrap:break-word;">7001-delnp-2007-pct-210.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzAwMS1kZWxucC0yMDA3LXBjdC0zMDQucGRm" target="_blank" style="word-wrap:break-word;">7001-delnp-2007-pct-304.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=UEVUSVRJT04gMTM3LnBkZg==" target="_blank" style="word-wrap:break-word;">PETITION 137.pdf</a></p>
		<br>
		<div class="pull-left">
			<a href="265495-benzoylpyrazole-compounds-and-herbicides-containing-them.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="265497-condensed-imidazolo-derivatives-for-the-inhibition-of-aldosterone-synthase-and-aromatase.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>265496</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>7001/DELNP/2007</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>09/2015</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>27-Feb-2015</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>25-Feb-2015</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>10-Sep-2007</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>DSM IP ASSETS B.V.</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td>HET OVERLOON 1, NL-6411 TE HEERLEN, THE NETHERLANDS.</td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>KOEKMAN, BERTUS PIETER</td>
											<td>PRUNUSSTRAAT 8, NL-2636 BG SCHIPLUIDEN, THE NETHERLANDS</td>
										</tr>
										<tr>
											<td>2</td>
											<td>LEE, CONNIE</td>
											<td>TOMASARHAGI 24, 15-107 REYKJAVIK, ICELAND</td>
										</tr>
										<tr>
											<td>3</td>
											<td>MAYER, ANNE FRANCOISE</td>
											<td>RENNWEG 100, CH-4052 BASEL, SWITZERLAND</td>
										</tr>
										<tr>
											<td>4</td>
											<td>MEURY, ANJA</td>
											<td>BREISACHERSTRASSE 12, CH-4057 BASEL, SWITZERLAND</td>
										</tr>
										<tr>
											<td>5</td>
											<td>SCHIPPER, DICK</td>
											<td>OOSTSINGEL 205, NL-2612 HL DELFT, THE NETHERLANDS</td>
										</tr>
										<tr>
											<td>6</td>
											<td>SHINJOH, MASAKO</td>
											<td>5-5-30 DAI, KAMAKURA, 247-0061 KANAGAWA, JAPAN.</td>
										</tr>
										<tr>
											<td>7</td>
											<td>TOEPFER, CHRISTINE</td>
											<td>HANFBUNDTEN 2, 79730 MURG, GERMANY.</td>
										</tr>
										<tr>
											<td>8</td>
											<td>VOLLEBREGT, ADRIANUS WILHELMUS HERMANUS</td>
											<td>DE DINTEL 62, NL-2673 BB NAALDWIJK, THE NETHERLANDS.</td>
										</tr>
										<tr>
											<td>9</td>
											<td>BEUZELIN OLLIVIER, MARIE-GABRIELLE</td>
											<td>RUE JEAN-DE-LA-GRANGE, 8, CH-2000 NEUCHATEL, SWITZERLAND</td>
										</tr>
										<tr>
											<td>10</td>
											<td>CHEVREUX, BASTIEN</td>
											<td>KATZENBUCKELWEG 14, 79618 RHEINFELDEN, GERMANY.</td>
										</tr>
										<tr>
											<td>11</td>
											<td>KOEKMAN, BERTUS PIETER</td>
											<td>PRUNUSSTRAAT 8, NL-2636 BG SCHIPLUIDEN, THE NETHERLANDS</td>
										</tr>
										<tr>
											<td>12</td>
											<td>LEE, CONNIE</td>
											<td>TOMASARHAGI 24, 15-107 REYKJAVIK, ICELAND</td>
										</tr>
										<tr>
											<td>13</td>
											<td>MAYER, ANNE FRANCOISE</td>
											<td>RENNWEG 100, CH-4052 BASEL, SWITZERLAND</td>
										</tr>
										<tr>
											<td>14</td>
											<td>MEURY, ANJA</td>
											<td>BREISACHERSTRASSE 12, CH-4057 BASEL, SWITZERLAND</td>
										</tr>
										<tr>
											<td>15</td>
											<td>SCHIPPER, DICK</td>
											<td>OOSTSINGEL 205, NL-2612 HL DELFT, THE NETHERLANDS</td>
										</tr>
										<tr>
											<td>16</td>
											<td>SHINJOH, MASAKO</td>
											<td>5-5-30 DAI, KAMAKURA, 247-0061 KANAGAWA, JAPAN.</td>
										</tr>
										<tr>
											<td>17</td>
											<td>TOEPFER, CHRISTINE</td>
											<td>HANFBUNDTEN 2, 79730 MURG, GERMANY.</td>
										</tr>
										<tr>
											<td>18</td>
											<td>VOLLEBREGT, ADRIANUS WILHELMUS HERMANUS</td>
											<td>DE DINTEL 62, NL-2673 BB NAALDWIJK, THE NETHERLANDS.</td>
										</tr>
										<tr>
											<td>19</td>
											<td>BEUZELIN OLLIVIER, MARIE-GABRIELLE</td>
											<td>RUE JEAN-DE-LA-GRANGE, 8, CH-2000 NEUCHATEL, SWITZERLAND</td>
										</tr>
										<tr>
											<td>20</td>
											<td>CHEVREUX, BASTIEN</td>
											<td>KATZENBUCKELWEG 14, 79618 RHEINFELDEN, GERMANY.</td>
										</tr>
										<tr>
											<td>21</td>
											<td>KOEKMAN, BERTUS PIETER</td>
											<td>PRUNUSSTRAAT 8, NL-2636 BG SCHIPLUIDEN, THE NETHERLANDS</td>
										</tr>
										<tr>
											<td>22</td>
											<td>LEE, CONNIE</td>
											<td>TOMASARHAGI 24, 15-107 REYKJAVIK, ICELAND</td>
										</tr>
										<tr>
											<td>23</td>
											<td>MAYER, ANNE FRANCOISE</td>
											<td>RENNWEG 100, CH-4052 BASEL, SWITZERLAND</td>
										</tr>
										<tr>
											<td>24</td>
											<td>MEURY, ANJA</td>
											<td>BREISACHERSTRASSE 12, CH-4057 BASEL, SWITZERLAND</td>
										</tr>
										<tr>
											<td>25</td>
											<td>SCHIPPER, DICK</td>
											<td>OOSTSINGEL 205, NL-2612 HL DELFT, THE NETHERLANDS</td>
										</tr>
										<tr>
											<td>26</td>
											<td>SHINJOH, MASAKO</td>
											<td>5-5-30 DAI, KAMAKURA, 247-0061 KANAGAWA, JAPAN.</td>
										</tr>
										<tr>
											<td>27</td>
											<td>TOEPFER, CHRISTINE</td>
											<td>HANFBUNDTEN 2, 79730 MURG, GERMANY.</td>
										</tr>
										<tr>
											<td>28</td>
											<td>VOLLEBREGT, ADRIANUS WILHELMUS HERMANUS</td>
											<td>DE DINTEL 62, NL-2673 BB NAALDWIJK, THE NETHERLANDS.</td>
										</tr>
										<tr>
											<td>29</td>
											<td>BEUZELIN OLLIVIER, MARIE-GABRIELLE</td>
											<td>RUE JEAN-DE-LA-GRANGE, 8, CH-2000 NEUCHATEL, SWITZERLAND</td>
										</tr>
										<tr>
											<td>30</td>
											<td>CHEVREUX, BASTIEN</td>
											<td>KATZENBUCKELWEG 14, 79618 RHEINFELDEN, GERMANY.</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>C12N 9/04</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>PCT/EP2006/001214</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td>2006-02-10</td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td>05405151.1</td>
									<td>2005-02-11</td>
								    <td>EUROPEAN UNION</td>
								</tr>
								<tr>
									<td>2</td>
									<td>05405110.7</td>
									<td>2005-02-11</td>
								    <td>EUROPEAN UNION</td>
								</tr>
								<tr>
									<td>3</td>
									<td>05405140.4</td>
									<td>2005-02-11</td>
								    <td>EUROPEAN UNION</td>
								</tr>
								<tr>
									<td>4</td>
									<td>05405139.6</td>
									<td>2005-02-11</td>
								    <td>EUROPEAN UNION</td>
								</tr>
								<tr>
									<td>5</td>
									<td>05405169.3</td>
									<td>2005-02-11</td>
								    <td>EUROPEAN UNION</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/265496-fermentive-vitamin-c-production by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 04:48:52 GMT -->
</html>
