<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/266052-nanofabricated-polypeptide-multilayer-films-coatings-and-microcapsules by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 06:33:22 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 266052:NANOFABRICATED POLYPEPTIDE MULTILAYER FILMS, COATINGS AND MICROCAPSULES</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">NANOFABRICATED POLYPEPTIDE MULTILAYER FILMS, COATINGS AND MICROCAPSULES</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>A method for designing polypeptides for the nanofabrication of thin films, coatings, and microcapsules by ELBL for applications in biomedicine and other fields</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td>I.        Title of Invention<br>
Method for Designing Polypeptides for the Nanofabrication of Thin Films, Coatings, and Microcapsules by Electrostatic Layer-by-Layer Self Assembly, II.       Cross-reference to Related Application<br>
None. III.      Statement Regarding Federally Sponsored Research or Development<br>
None. IV.      Background of the Invention<br>
A.	Field of the Invention<br>
The present invention relates to the fabrication of ultrathin multilayered films on suitable surfaces by electrostatic layer-by-layer self assembly ("ELBL"). More specifically, the present invention relates to a method for designing polypeprides for the nanofabrication of thin films, coatings, and microcapsules by ELBL for applications in biomedicine and other fields.<br>
B.	Description of Related Art<br>
ELBL is an established technique in which ultrathin films are assembled by alternating the adsorption of oppositely-charged polyelectrolytes. The process is based on the reversal of the surface charge of the film after the deposition of each layer. Figure 1 shows a schematic diagram of the general ELBL process: films of oppositely charged polyions (cationic polyions 10 and anionic polyions 11) are assembled in successive layers on a negatively-charged planar surface 12; the surface charge is reversed after the deposition of each layer. This process is repeated until a film of desired thickness is formed. The physical basis of association is electrostatics—gravitation and nuclear forces play effectively no role. Because of the generality and relative simplicity of the process, ELBL allows for the deposition of many different types of materials onto many different types of surface. There is, therefore, a vast number of possible useful combinations of materials and surfaces. For a general discussion of ELBL, including its history, see Yuri Lvov, "Electrostatic Layer-by-Layer Assembly of Proteins and Polyions" in Protein Architecture: Interfacial Molecular Assembly and Immobilization Biotechnology, Y. Lvov &amp; H. M6hwald eds. (New York: Marcel Dekker, 1999), pp.125-167, which is incorporated herein by reference hi its entirety.<br>
ELBL has recently become a focus area in the field of nanotechnology because it can be used to fabricate films substantially less than 1 micron in thickness. Moreover, ELBL permits exceptional control over the film fabrication process, enabling the use of nanoscale materials and permitting nanoscale structural modifications. Because each layer has a<br>
thickness on the order of a few nanometers or less, depending on the type of material used and the specific adsorption process, multilayer assemblies of precisely repeatable thickness can be formed.<br>
A number of synthetic polyelectrolytes have been employed in ELBL applications, including sodium poly(styrene sulfonate) ("PSS"), poly(aUylamine hydrochloride) ("PAH"), poly(diallyldimethylammonium chloride) ("PDDA"), poly(acrylamide-co-diallyldimethylammonium chloride), poly(ethyleneimine) ("PEI"), poly(acrylic acid) " ("PAA"), poly(anetholesulfonic acid,poly (vinyl sulfate) ('TVS"), and poly(vinylsulfonic acid). Such materials, however, are not generally useful for biomedical applications because they are antigenic or toxic.<br>
Proteins, being polymers with side chains having ionizable groups, can be used in ELBL for various applications, including biomedical ones. Examples of proteins that have been used in ELBL include cytochrome c, hen egg white lysozyrne, immunoglobulin G, myoglobin, hemoglobin, and serum albumin (ibid.). There are, however, difficulties with using proteins for this purpose. These include limited control over multilayer structure (because the surface of the protein is highly irregular and proteins will not ordinarily adsorb on a surface in a regular pattern), restrictions on pH due to the pH-dependence of protein solubility and structural stability, lack of biocompatibility when using exogenous proteins, and the cost of scaling up production if the gene has not been cloned; unless the protein were identical in a readily available source, e.g. a cow, the protein would have to be obtained from the organism in which it was intended for use, making the cost of large-scale production of the protein prohibitive.<br>
By contrast polypeptides, which are generally smaller and less complex than proteins, constitute an excellent class of material for ELBL assembly, and polypeptide film structures formed by ELBL will be useful in a broad range of applications. The present invention provides a method for designing polypeptides for the nanofabrication of thin films, coatings, and microcapsules by ELBL. Polypeptides designed using the method of the present invention should exhibit several useful properties, including, without limitation, completely determined primary structure, minimal secondary structure in aqueous solution, monodispersity, completely controlled net charge per unit length, ability to form cross-links on demand, ability to reverse cross-link formation, ability to form more organized thin films than is possible with proteins, and relatively inexpensive large-scale production cost (assuming gene design, synthesis, cloning, and host expression in E. coli or yeast, or peptide synthesis).<br><br>
Polypeptides designed using the method of the present invention have been shown useful for ELBL of thin film structures with targeted or possible applications in biomedical technology, food technology, and environmental technology. Such polypeptides could be used, for example, to fabricate artificial red blood cells, drug delivery devices, and antimicrobial films. V.       Brief Summary of the Invention<br>
The present invention provides a novel method for identifying "sequence motifs" of a defined length and net charge at neutral pH in amino acid sequence information for use in ELBL, and recording a desired number of the motifs. The method comprises the steps of: (a) Obtaining an amino acid sequence for a peptide or a protein from a particular organism; (b) Locating a starter amino acid in the amino acid sequence; (c) Examining the starter amino acid and the following n amino acids to determine the number of charged amino acids having a polarity opposite the certain polarity; (d) If the number of the charged amino acids having a polarity opposite the certain polarity is one or more, continuing the method at step g; (e) Examining the starter amino acid and the following n amino acids to determine the number of charged amino acids having the certain polarity; (f) If the number of charged amino acids having the certain polarity is equal to or greater than x, recording the amino acid sequence motif consisting of the starter amino acid and the following n amino acids; (g) Locating another starter amino acid in the amino acid sequence; and (h) Repeating the method beginning at step c until the desired number of amino acid sequence motifs have been identified or all of the amino acids in the amino acid sequence have been used as the starter amino acid in step c; wherein x is greater than or equal to approximately one-half of n.<br>
The present invention also provides a novel method for designing a polypeptide for use in ELBL, comprising the steps of: (a) Identifying and recording one or more amino acid sequence motifs having a net charge of a certain polarity using the steps mentioned in the preceding paragraph and (b) Joining a plurality of said recorded amino acid sequence motifs to form a polypeptide.<br>
The present invention also provides a novel method for designing a polypeptide for use in ELBL comprising the following steps: (a) Designing de novo a plurality of amino acid sequence motifs, wherein said amino acid sequence motifs consist of n amino acids, at least* of which are positively charged and none is negatively charged, or at least x of which are negatively charged and none is positively charged, wherein x is greater than or equal to approximately one-half of n; and (b) Joining said plurality of said amino acid sequence motifs. The amino acid sequence motifs can comprise the 20 usual amino acids or non-<br>
natural amino acids, and the amino acids can be either left-handed (L-amino acids) or right handed (D-amino acids).<br>
The present invention also provides a thin film, the film comprising a plurality of layers of polypeptides, the layers of polypeptides having alternating charges, wherein the polypeptides comprise at least one amino acid sequence motif consisting of n amino acids, at least x of which are positively charged and none is negatively charged, or at least* of which are negatively charged and none is positively charged, wherein x is greater than or equal to approximately one-half of n. The motifs in these polypeptides may be selected using either of me methods described above.<br>
The present invention also provides a novel process for using cysteine or other sulfhydryl-containing amino acid types to "lock" and "unlock" the layers of polypeptide ELBL films. This process enables the films to remain stable at extremes of pH ,giving greater control over the mechanical stability and diffusive properties of films nanofabricated from designed polypeptides and increasing their utility in a broad range of applications. VI.      Brief Description of the Several Views of the Drawings<br>
Figure 1 is a schematic diagram of the general ELBL process.<br>
Figure 2 is a graph of the cumulative secondary structure propensities of the amino acid sequence motifs identified in human amino acid sequence information using the method of the present invention, compared with the distribution of structure propensities of 105 random amino acid sequences.<br>
Figure 3 (a) shows adsorption data as monitored by the quartz crystal microbalance technique ("QCM") for a combination of amino acid sequences designed according to the present invention.<br>
Figure 3(b) shows a comparison of adsorption data as monitored by QCM for different combinations of amino acid sequences designed according to the present invention.<br>
Figure 3(c) shows a graph of adsorbed mass in nanograms versus layer number for amino acid sequences designed and fabricated according to the present invention.<br>
Figure 4(a) illustrates intra-layer disulfide bonds according to the cysteine locking method of the present invention.<br>
Figure 4(b) illustrates inter-layer disulfide bonds according to the cysteine locking method of the present invention.<br>
Figure 4(c) illustrates the oxidation and reduction of disulfide bonds in microcapsules fabricated from polypeptides designed according to the method of the present invention.<br>
Figure 5 is a schematic of the selection process of the present invention used to identify in existing amino acid sequence information amino acid sequence motifs having suitable electrostatic properties for ELBL.<br>
Figure 6 shows the number of non-redundant sequence motifs identified in available human amino acid sequence data.<br>
Figure 7 shows the ELBL adsorption of poly-L-glutamate and poly-L-lysine from an aqueous medium as a function of ionic strength.<br>
Figure 8 shows the adsorption of polypeptides designed according to the method of the present invention for experiments to probe the effect of disulfide bond formation.<br>
Figure 9 shows the percentage of material remaining during thin film disassembly at acidic pH as discussed with reference to Figure 8.<br>
Figure 10 shows the percentage of material lost during the acidic pH disassembly step of an experiment involving de now-designed polypeptides containing cysteine.<br>
Figure 11 (a) illustrates the role of solution structure of peptides on film assembly, showing how the assembly behavior of poly-L-glutamate and poty-L-lysine depends on pH. QCM resonant frequency is plotted against adsorption layer. The average molecular mass of poly-L-glutamate was 84,600 Da, while that of poly-L-lysine was 84,000 Da. The numbers refer to pH values. E = Glu, K = Lys. The peptide concentration used for assembly was 2 rng/rnL.<br>
Figure 1 l(b) illustrates the role of solution structure of peptides on film assembly, showing how the solution structure of poly-L-glutamate and poly-L-lysine depends on pH. Mean molar residue ellipticity is plotted as a function of pH. The peptide concentration was 0.05 mg/mL.<br>
Figure 12 shows adsorption data for polyelectrolytes of different lengths, illustrating that long polyelectrolytes adsorb better than short ones. Vfl.    Detailed Description of the Invention<br>
A.       Explanations of Terms<br>
For convenience in the ensuing description, the following explanations of terms are adopted. However, these explanations are intended to be exemplary only. They are not intended to limit the terms as they are described or referred to throughout the specification. Rather, these explanations are meant to include any additional aspects and/or examples of the terms as described and claimed herein.<br>
As used herein, "biocompatibility" means causing no adverse health effect upon ingestion, contact with the skin, or introduction to the bloodstream.<br>
As used herein, "immune response" means the response of the human immune system to the presence of a substance in the bloodstream. An immune response can he characterized in a number of ways, for example, by an increase in the bloodstream of the number of antibodies that recognize a certain antigen. (Antibodies are proteins made by the immune system, and an antigen is an entity that generates an immune response.) The human body fights infection and inhibits reinfection by increasing the number of antibodies in the bloodstream. The specific immune response depends somewhat on the individual, though general patterns of response are the norm.<br>
As used herein, "epitope" means the structure of a protein that is recognized by an antibody. Ordinarily an epitope will be on the surface of a protein. A "continuous epitope" is one that involves several amino acids in a row, not one that involves amino acid residues that happen to be in contact in a folded protein.<br>
As used herein, "sequence motif and "motif mean an amino acid sequence of a given number of residues identified using the method of the current invention. In a preferred embodiment, the number of residues is 7.<br>
As used herein, "amino acid sequence" and "sequence" mean any length of polypeptide chain that is at least two amino residues long.<br>
As used herein, "residue" means an amino acid in a polymer; it is the residue of the amino acid monomer from which (lie polymer was formed. Polypeptide .synthesis involves dehydration—a single water molecule is "lost" on addition of the amino acid to a polypeptide chain.<br>
As used herein, "designed polypeptide" means a polypeptide designed using the method of the present invention, and the terms "peptide" and "polypeptide" are used interchangeably.<br>
As used herein, "primary structure" means the linear sequence of amino acids in a polypeptide chain, and "secondary structure" means the more or less regular types of structure stabilized by non-covalent interactions, usually hydrogen bonds—examples include α-helix, ß-sheet, and ß-turn.<br>
As used herein, "amino acid" is not limited to the 20 naturally occurring amino acids;<br><br>
the term also refers to D-amino acids, L-amino acids, and non-natural amino acids, as the context permits.<br>
As used herein, "non-natural amino acids" means amino acids other than the 20 naturally occurring ones.<br><br>
The following three-letter abbreviations are used herein for the 20 usual umino acids:<br>
Ala = alanine	Cys = cysteine	Asp = aspartic acid<br>
Glu = glutamic acid	Phe = phenylalanine	Gly = glycine<br>
His = histidine	Ile = isoleucine	Lys = lysine<br>
Leu = leucine	Met = methionine	Asn = asparagine<br>
Pro = proline	Gln = glutamine	Arg = arginine<br>
Ser = serine	Thr = threonine	Val = valine<br>
Trp = tryptophan	Tyr = tyrosine ,<br>
B.        Description of the Invention<br>
The present invention provides a method for designing polypeptides for the nanofabrication by ELBL of thin films, coatings, and microcapsules for applications in biomedicine and other fields. The method involves 5 primary design concerns: (l)the electrostatic properties of the polypeptides; (2) the physical structure of the polypeptides; (3) the physical stability of the films formed from the polypeptides; (4) the biocompatibility of the polypeptides and films; and (5) the bioactivity of the polypeptides and films. The first design concern, electrostatics, is perhaps the most important because it is the basis of ELBL. Without suitable charge properties, a polypeptide will not be soluble in aqueous solution and cannot be used for the ELBL nanofabrication of films. We have devised a novel process for identifying in amino acid sequence information amino acid sequence motifs having electrostatic properties suitable for ELBL.<br>
The secondary structure of the polypeptides used for ELBL is also important, because the physical properties of the film, including its stability, will depend on how the solution structure of the peptide translates into its structure in the film. Figure 11 illustrates how the solution structure of certain polypeptides correlates with film assembly. Panel (a) shows how the assembly behavior of poly-L-glutamate and poly-L-lysine depends on pH. It is clear that the α-helix conformation correlates with a greater extent of deposited material than the $-sheet conformation. The precise molecular interpretation of this behavior remains to be elucidated. Panel (b) shows how the solution structure of these peptides depends on pH. At pH 4.2 poly-L-glutamate is largely α-helical, as is poly-L-lysine at pH 10.5. Both polypeptides are in a largely unstructured coil-like conformation at pH 7.3.<br>
The remaining concerns relate to the applications of the polypeptide films. In practicing the invention, more or less weight will be placed on these other concerns depending on the design requirements of a particular application.<br><br>
By using the selection process of the present invention to identify in amino acid sequence information amino acid sequence motifs having suitable charge characteristics, and using the other design concerns to select particular motifs, one can design polypeptides suitable for the ELBL fabrication of nano-organized films for applications in biomedicine arid other fields. Alternatively, one can use the method of the present invention to design polypeptides de novo for use in ELBL. The approach to de novo design is essentially the same as identifying motifs in existing amino acid sequence information, except that each residue in an amino acid sequence motif is selected by the practitioner rather than an entire motif being identified in the genomic or proteomic information of a specific organism. It must be emphasized that the fundamental polypeptide design principles adduced in the present invention are independent of whether the amino acids involved are the 20 naturally-occuring ones, non-natural amino acids, or some novel combination of these, in the case of de novo polypeptide design. Further, both D-amino acids and L-amino acids could be used.<br>
The design concerns of the present invention are discussed in more detail below. 1.        Electrostatics<br>
We have devised a novel process for identifying in amino acid sequence information amino acid sequence motifs having electrostatic properties suitable for ELBL. Using this process, we have identified 88,315 non-redundant amino acid sequence motifs in human proteome data—the translation of the portion of the genome that encodes all known proteins in the human body. This information is publicly available at the National Center for Biotechnology Information's ("NCBI") Web site: <http:>, among other places. Such information is constantly being updated as the human genome is further analyzed. As the amount of such information increases, the number of amino acid sequence motifs that could be identified in human sequence information by the selection process of the present invention as having suitable electrostatic properties for ELBL will also increase. The same is true for any organism. Accepted biochemical and physics principles, as well as the experimental results described below, indicate that the identified sequence motifs will be useful for the design of polypeptides for the nanofabrication of ELBL structures.<br>
The key selection criterion is the average charge per unit length at neutral pH (pH 7, close to the pH of human blood). In addition, there are several structural preferences. First, it is preferred that each amino acid sequence motif consist of only 7 residues.<br>
a.       Total Number of Residues in the Motif<br>
The motif length of 7 was chosen in an effort to optimize biocompatibility, physical structure, and the number of non-redundant sequence motifs in available amino acid sequence data.<br>
As discussed below, it is preferred that at least half of the amino acid residues in each sequence motif be charged. Moreover, it is preferred that all of the charged residues in each motif be of the same charge. These requirements ensure that each motif will be sufficiently soluble in aqueous solvent and have sufficient charge at neutral pH to be useful for ELBL. Because only a relatively small percentage of amino acid types are charged, as the length of a given amino acid sequence increases, the odds decrease that the sequence will have a sufficient percentage of appropriately charged amino acids for ELBL. 4 charged amino acids is the preferred minimum for a motif size of 7, because fewer than 4 charges yields substantially decreased peptide solubility and decreased control over ELBL.<br>
Regarding biocompatibility (discussed further below), each identified sequence motif is long enough at 7 residues to constitute a continuous epitope (relevant to the possible immune response of an organism into which a designed peptide might be introduced), but not so long as to correspond substantially to residues both on the surface of a protein and in its interior, the charge requirements help to ensure that the sequence motif occurs on the surface of the folded protein; a charged residue cannot be formed in the core of a folded protein. By contrast, a very short motif could appear to the body to be a random sequence, or one not specifically "self," and therefore elicit an immune response. Although the ideal length of a peptide for generating antibodies is a point of some dispute, most peptide antigens range in length from 12 to 16 residues, Peptides that are 9 residues or shorter can be effective antigens; peptides longer than 12 to 16 amino acids may contain multiple epitopes (Angeletti, R.H. (1999) Design of Useful Peptide Antigens, J. Biomol. Tech. 10:2-10, which is hereby incorporated by reference in its entirety). Thus, to minimize antigenicity one would prefer a peptide shorter than 12 and, better yet, shorter than 9 residues.<br>
The preferred motifs should not be too long for another reason: to minimize secondary structure formation. Secondary structure decreases control of the physical structure of the polypeptides (see below) and the films made from them.<br>
Furthermore, the maximum number of non-redundant motifs is found when the number of residues in each motif is 7. Figure 6 sbows the number of non-redundant sequence motifs in available human amino acid sequence information. The greatest number of positive motifs is for a 5-residue length, while the greatest number of negative motifs is for a 7-<br><br>
residue length. The greatest number of positive and negative motifs is about the same for 5 and 7. Thus, a motif length of 7 residues would appear to maximize the number of non-redundant motifs.<br>
For all of the above reasons, 7 residues is the preferred length of motif to optimize polypeptide design for ELBL. Nevertheless, it is possible that in some cases either slightly shorter or slightly longer motifs will work equally as well. For example, motifs 5 or 6 residues long may be employed, and motifs on the order of 8 to 15 residues in length could also be useful.<br>
b.         Number of Charged Residues<br>
Second, it is preferred that at least 4 positively-charged (basic) arnino acids (Arg, His, or Lys) or at least 4 negatively-charged (acidic) amino acids (Glu or Asp) are present in each 7-residue motif at neutral pH. Combinations of positive and negative charges are disfavored in an effort to ensure a sufficiently high charge density at neutral pH. It is possible, however, that a motif containing both positive and negative amino acids could be useful for ELBL. For example, a slightly longer motif, say of 9 residues, could have 6 positively charged amino acids and 1 negatively charged amino acid. It is the balance of charge that is important—the overall peptide must be either sufficiently positively charged or sufficiently negatively charged at neutral pH. Preferred embodiments of me motifs, however, will contain only Glu or Asp or only Arg, His, or Lys as (he charged amino acids (although other non-charged amino acids could, and ordinarily do, form part of the motifs), unless non-natural amino acids are admitted as acidic or basic amino acids.<br>
Figure 5 is a flow chart showing the steps involved in the selection process for identifying amino acid sequences having suitable electrostatic properties. It is assumed that only the 20 usual arnino acids are involved. If searching for negatively-charged motifs, the process begins by locating an amino acid in the sequence data. This amino acid will be called the "starter amino acid" because it is the starting point for the analysis of the surrounding amino acids (i.e., it will begin the motif). Next, the starter amino acid and the following 6 residues are examined for occurrences of Arg, His, or Lys. If one or more Arg, His, or Lys is located in these 7 amino acids, the process is begun anew at another starter amino acid. If no Arg, His, or Lys is found, the 7 arnino acids are examined to determine the number of occurrences of Glu and/or Asp. If there are at least 4 occurrences of Glu and/or Asp in the 7 residues, the sequence motif is cataloged. The selection process is essentially the same for positively charged amino acids, except that Glu and Asp are replaced by Arg, His, and Lys, and Arg, His, and Lys are replaced by Glu and Asp, respectively. Obviously, one could also<br>
begin the method at the beginning of the amino acid sequence (amino terminus) and proceed to the end (carboxyl terminus), or, alternatively, one could begin at a random point and work through all of the amino acids in the sequence, randomly or systematically in either direction. Moreover, one could use the method to identify motifs in sequence information containing non-natural amino acids, for example, if codes were used for each non-natural amino acid type. In such a case, one would search for non-natural acidic or basic amino acids instead of Glu and Asp, and Arg, Lys, and His, respectively.<br>
The remaining design concerns, namely, physical structure, physical stability, biocompntibility, and biofimctionality, deal primarily with the particular application for which the designed polypeptides will be used. As noted above, more or less weight will be placed on these concerns during the design process, depending on the desired peptide properties for a particular application.<br>
2.        'Physical Structure<br>
A design concern regarding the amino acid sequence motifs is their propensity to form secondary structures, notably α-helix or /3-sheet We have sought in several ways to control, notably minimize, secondary structure formation of designed polypeptides in an aqueous medium in order to maximize control over thin film layer formation. First, it is preferred that the sequence motifs be relatively short, because long motifs are more likely to adopt a stable toe-dimensional structure in solution. Second, we place a glycine residue between each motif in preferred embodiments of the polypeptide designs. Glycine has a very low a-helix propensity and a very low /3-sheet propensity, making it energetically very unfavorable for a glycine and its neighboring amino acids to form regular secondary structure in aqueous solution. Proline has similar properties in some respects and could be used as an altemative to glycine to join motifs. Third, we have sought to minimize the a-helix and ß-sheet propensity of the designed polypeptides themselves by focusing on motifs for which the summed α-helix propensity is less than 7.5 and the summed ß-sheet propensity is less than 8. ("Summed" propensity means the sum of the o^elix or /J-sheet propensities of all amino acids in a motif.) It is possible, however, that amino acid sequences having a somewhat higher summed o-helix propensity and/or summed ß-sheet propensity would be suitable for ELBL under some circumstances, as the Gly (or Pro) residues between motifs will play a key role in inhibiting stable secondary structure formation in the designed polypeptide. In fact, it may be desirable in certain applications for the propensity of a polypeptide to form secondary structure to be relatively high, as a specific design feature of thin film fabrication; the necessary electrostatic charge requirements for ELBL must still be met, as discussed above.<br><br>
In order to be able to select amino acid sequences with desired secondary structure propensities, we first calculated the secondary structure propensities for all 20 amino acids using the method of Chou and Fasman (see P. Chou and G. Fasman Biochemistry 13:211 (1974), which is incorporated by reference herein in its entirety) using structural information from more than 1,800 high-resolution X-ray crystallographic structures (1,334 containing a.-helices and 1,221 containing ß-strands). Structures were selected from the Protein Data Bank (a publicly-accessible repository of protein structures) based on:  (a) method of structure determination (X-ray diffraction); (b) resolution (better than 2.0 A)—"resolution" in this context refers to the minimurn size of a structure one can resolve, as in the Rayleigh criterion; and (c) structural diversity (less than 50 % sequence identity between the protein crystallographic structures used to compute the helix and sheet propensities of the various amino acids). The rationale was to choose high resolution structures determined by the most reliable methodology and not to bias the propensity calculation by having similar structures. Next, for comparison 100,000 non-redundant random sequences were produced using a random number generator in a personal computer. We then calculated the secondary structure propensities for the 88,315 amino acid sequences identified using the selection process described in part VII(B)(1) above (59,385 non-redundant basic sequence motifs and 28,930 non-redundant acidic sequence motifs). The propensities for the random sequences were then compared to the propensities of the selected sequences. Figure 2 shows the distribution of secondary structure formation propensities in these sequence motifs. The rectangle in Figure 2 highlights the sequence motifs we have identified as least likely to form secondary structure on the basis of secondary structure propensities. 3.        Physical Stability<br>
Another design concern is control of the stability of the polypeptide ELBL films. Ionic bonds, hydrogen bonds, van der Waals interactions, and hydrophobic interactions provide some, albeit relatively limited, stability to ELBL films. By contrast, covalent disulfide bonds could provide exceptional structural strength. We have devised a novel process for using cystcine (or some other type of sulfhydryl-containing amino acid) to "lock" and "unlock" adjacent layers of polypeptide ELBL film. This process enables a polypeptide nanofabricated film to remain stable at extremes of pH, giving greater control over its mechanical stability and diffusive properties (for discussions of porosity of multilayer films made of non-polypeptide polyelectrolytes, see Caruso, F., Niikura, K, Furlong, N. and Okahata (1997) Langmuir 13:3427 and Caruso, F., Furlong, N., Ariga, K., Ichinose, I., and Kunitake, T. (1998) Langmuir 14:4559, both of which are incorporated herein by reference in<br><br>
their entireties). Also, the incorporation of cysteine (or some other type of sulfhydryl-containing amino acid) in a sequence motif of a designed polypeptide enables the use of relatively short peptides in thin film fabrication, by virtue of intermolecular disulfide bond formation. Without cysteine, such peptides would not generally yield sufficiently stable films (see figure 12, discussed below). Thus, our novel use of cysteine will obviate the need to produce expensive long versions of the designed polypeptides in a substantial percentage of possible applications. This will be particularly advantageous in situations where the thin film is to be fabricated over material to be encapsulated, for example a small crystal of a drug, a small spherical hemoglobin crystal, or a solution containing hemoglobin.<br>
For applications in which the physical stability of the films is important, amino acid sequence motifs containing cysteine (or some other type of sulfhydryl-containing amino acid) may be selected from the library of motifs identified using the methods discussed above, or designed de novo using the principles described above. Polypeptides can then be designed and fabricated based on the selected or designed amino acid sequence motifs. Once the polypeptides have been synthesized chemically or produced in a host organism, ELBL assembly of cysteine-containing peptides is done in the presence of a reducing agent, to prevent premature disulfide bond formation. Following assembly, the reducing agent is removed and an oxidizing agent is added, In the presence of the oxidizing agent disulfide bonds form between cysteine residues, thereby "locking" together the polypeptide layers that contain them.<br>
This "locking" method may be further illustrated using the following specific example of microcapsule fabrication. First, designed polypeptides containing cysteine are used to form multilayers by ELBL on a suitably charged spherical surface, normally in aqueous solution at neutral pH and in the presence of dithiothreitol ("DTT"), a reducing agent. Next, DTT is removed by filtration, diffusion, or some other similar method known in the art, causing cystine to form from pairs of cysteine side chains and thereby stabilizing the film. If the peptide multilayers are constructed on a core particle containing the materials one wishes to encapsulate, for instance a crystalline material, the fabrication process is complete and the core particle can thereafter be made to dissolve in the encapsulated environment, for example by a change of pH. If, however, the multilayers are constructed on a "dummy" core particle, the core must be removed. In the case of melamine formaldehyde particles ("MF"), for example, the core is ordinarily dissolved by decreasing the pH—dissolution is acid-catalyzed. Following dissolution of the core, the pH of solution is adjusted to 4, where partial charge on the peptide polyanions makes the microcapsules semi-permeable (compare Lvov et al. (2001)<br><br>
Nano Letters 1:125, which is hereby incorporated herein in its entirety). Next, 10 mM DTT is added to the microcapsule solution to reduce cystine to cysteine. The microcapsules may then be "loaded" by transferring them to a concentrated solution of the material to be encapsulated, for example a protein (ibid.). The protein enters the microcapsules by moving down its concentration gradient. The encapsulated protein is "locked in" by removal of reductant and addition of oxidant, thereby promoting the reformation of disulfide bonds.<br>
A schematic of the cysteine "locking" and "unlocking" method of the present invention is shown in Figure 4. Cysteine can form both intra- and inter-molecular disulfide bonds. Further, disulfide bonds can be formed between molecules in the same layer or adjacent layers, depending on the location of cyslcinc-conlaining popt ides in the film. Referring to Figure 4(a), basic polypeptides 2 are linked by disulfide bonds 3 in all layers in which the basic peptides contain cysteine. The acidic peptides of the intervening layer (represented in the figure by a translucent layer 4) do not contain cysteine. However, alternating layers continue to attract each other electrostatically, if the acidic and basic side chains are charged at the pH of the surrounding environment. Referring to Figure 4(b), disulfide bonds are shown between layers. Such structures will form when both the acidic and basic polypeptides (i.e., alternating polypeptide layers) used for ELBL contain cysteine and the procedure used has been suitable for disulfide bond formation. Referring to Figure 4(c), reduction and oxidation reactions are used to regulate the release of encapsulated compounds 5 by breaking and forming disulfide bonds 3, respectively, and thereby regulating the diffusion of particles through the capsule wall.<br>
The cysteine "locking" and "unlocking" is a novel way of regulating the structural integrity and permeability of ELBL films. It is known in the art that glutaraldehyde can be used to cross-link proteins, and this chemical could therefore be used to stabilize polypeptide films. Glutaraldehyde cross-linking, however, is irreversible. In contrast, the cysteine "locking" and "unlocking" method of the present invention is reversible and, therefore, offers better control over structure formation and, importantly, use of the films and capsules that can be fabricated using the present invention. Blood is an oxidizing environment. Thus, in certain biomedical applications, for example artificial red blood cells or drug delivery systems fabricated from designed polypeptides, exposing Cys-crosslnked polypeptide film to the blood or some other oxidizing environment after the formation of disulfide bonds is not expected to cause those bonds to be broken. Finally, it should also be noted that applications involving non-natural amino acids would replace Cys with some other sulfhydryl-containing amino acid type. For example, a sulfhydryl could be added toß-amino acids such as D,L-/3-<br><br>
amino-ß-cylohexyl propionic acid; D,L-3-aminobutanoic acid; or 5-(methylthio)-3-aminopentanoic acid (see http://www.synthatex.com). 4.        Biocompatibility<br>
Biocompatibility is a major design concern in biomedical applications. In such applications, the practitioner of the present invention will aim to identify genomic or proteomic information that will yield "immune inert" polypeptides., particularly if the fabricated or coated object will make contact with circulating blood. For purposes of the present invention, it is preferred that the selection process discussed in Part VII(B)(1) above be used to analyze the amino acid sequences of blood proteins. This will maximize the odds of minimizing the immune response of an organism.<br>
Computer algorithms exist for predicting the antigenicity of an amino acid sequence. Such methods, however, are known in the art to be semi-reliable at best. In the present invention, the sequence motifs identified using the selection method discussed above in Part VII(B)(1) are highly polar. The motifs must, therefore, occur on the surface of the native state of the proteins of which they are part of the sequence. The "surface" is that part of a folded protein that is in contact with the solvent or inaccessible to the solvent solely because of the granular nature of water. The "interior" is that part of a folded protein mat is inaccessible to solvent for any other reason. A folded globular soluble protein is like an organic crystal, the interior being as densely packed as in a crystal lattice and the exterior being in contact with the solvent, water. Because of their charge properties, the polypeptide sequence motifs identified using the method of the present invention must occur mostly, if not exclusively, on the surface of a protein. Thus, all of the sequence motifs identified in human blood proteins using the selection process of the current invention are effectively always in contact with the immune system while uie protein is in the blood. This holds for all conformations of the protein that might become populated in the bloodstream, including denatured states, because it is highly energetically unfavorable to transfer a charge from an aqueous medium to one of low dielectric (as occurs in a protein interior). Accepted biochemical principles indicate, therefore, that the polypeptides designed from blood proteins using the method of me present invention will either not illicit an immune response or will elicit a minimal immune response. For the same reasons, polypeptides designed using the method of the present invention should be biocompatible. All sequence motifs identified from genomic data using the selection process of the current invention, not only those in blood proteins, should be biocompatible, though the extent of immune response or any other type of biological response may well depend on specific details of a sequence motif.<br><br>
(Because the polypeptide sequences on which the motifs are based actually occur in the organism for which the film as been fabricated, this approach will, at least in principle, work equally well for any type of organism. For instance, the approach may be of significant value to veterinary science.) Both immune response andbiocompatibility are important regarding the use of the designed peptides in biomedical applications, including, without limitation, the manufacture of artificial red blood cells, drug delivery systems, or polypeptides for fabrication of biocompatible films to coat implants for short-term or long-term introduction into an organism.<br>
5.       Bioactivity<br>
In some applications of polypeptide thin films, coatings, or microcapsules, it may be desirable to modify the design of the polypeptides to include a functional domain for use in some layer of the structure, often the outermost A functional domain in this context is an independently thermostable region of a protein that has specific biofunctionality (e.g. binding phosphotyrosine). It is well known in the art that such biofunctionality may be integrated with other functionalities in a multi-domain protein, as for example in the protein tensin, which encompasses a phosphotyrosine binding domain and a protein tyrosine phosphatase domain. The inclusion of such a domain in a designed polypeptide could function in a number of ways, including without limitation specific ligand binding, targeting in vivo, biosensing, or biocatalysis.<br>
C.        Uses for Polypeptides Designed Using the Method of the Present Invention<br>
As noted above, polypeptides of suitable design are excellent materials for ELBL, and polypeptide film structures formed using ELBL will be useful in a large number of different types of applications. Polypeptides designed using the method of the present, invention have been shown to be useful for ELBL of film structures for possible applications in biomedical technology, food technology, and environmental technology. For example, such polypeptides could be used to fabricate artificial red blood cells, 1.        Artificial Red Blood Cells<br>
A number of different approaches have been taken to red blood cell substitute development. One approach involves the use of perfluorocarbons. Perfluorocarbon emulsions contain synthetic fluorinated hydrocarbons capable of binding oxygen and delivering it to tissues. This approach however, increases reticulo-endothelial cell blockage. The perfluorocarbons can become trapped in the reticulo-endothelial system, which may result in adverse consequences.<br><br>
Another approach focuses on antigen camouflaging, which involves coating red blood cells with a biocompatible polymer called polyethylene glycol (PEG). The PEG molecules form permanent covalent bonds on the surface of the cell. The coating effectively hides the antigenic molecules on the surface of the red blood cells, so that the blood recipient's antibodies do not recognize the cells as foreign. For example, the immune system of a normal person who has type A blood will naturally have antibodies that recognize antigens on the surface of type B red blood cells, leading to cell destruction. The attachment of PEG to the surface of a type B red blood cell"camouflages" the surface of the cell, so that its surface antigens can no longer be recognized by the immune system and the antigenically-foreign red blood cells will not be destroyed as quickly (see Pargaonkar, N.A., G. Sharma, and ELK. Vistakula. (2001) "Artificial Blood: Current Research Report," which is hereby incorporated by reference in its entirety).<br>
A number of diseases, including thalassemia, that require repeated blood transfusions are often complicated by the development of antibodies to "minor" red cell antigens. This "allosensitization" can render these patients almost impossible to transfuse, rendering the situation life-threatening. In in vitro testing, the PEG-modified red cells appear not to trigger allosensitization and may also be useful in clinical situations where allosensitization has already occurred (see Scott, M.D. et al. (1997) "Chemical camouflage of antigenic dcterminants:: Stealth erythrocytes," Proc.. Natl Acad. Sci. USA. 94 (14): 7566-7571, which is hereby incorporated by reference in its entirety).<br>
Other approaches involve purified hemoglobin. Unmodified cell-free hemoglobin has known limitations. These include oxygen affinity that is too high for effective tissue oxygenation, a half-life within the intravascular space that is too short to be clinically useful, and a tendency to undergo dissociation into dimers with resultant renal tubular damage and toxicity. Because of these limitations, hemoglobin used to make a cell-free red blood cell substitute must be modified. A number of modification techniques have been developed. Hemoglobin can be cross-linked (a covalent bond between two molecules is made by chemical modification) and polymerized using reagents such as glutaraldehyde. Such modifications result in a product that has a higher P<br>
 (partial pressure of oxygen at which 50 % of all oxygen-binding sites are occupied) than that of normal hemoglobin, and an increase in the plasma half-life of up to 30 hours. The source of the hemoglobin for this purpose can be human (outdated donated blood), bovine, or human recombinant. The solution of modified hemoglobin is prepared from highly purified hemoglobin and taken through various biochemical processes, to eliminate phospholipids, endotoxins, and viral contaminants (see<br><br>
Nester, T. and Simpson, M (2000) "Transfusion medicine update," Blood Substitutes, which is hereby incorporated by reference in its entirety). Biopure Corporation (Cambridge, MA) has been using modified hemoglobin for their product, Hemopure.<br>
The main potential adverse effect of modified hemoglobin solutions is an increase in systemic and pulmonary vascular resistance that may lead to a decrease in cardiac index. Decreases in the cardiac index may impair optimum oxygen delivery and outweigh the advantage of an oxygen-carrying solution (see Kasper S.M. et al. (1998) "The effects of increased doses of bovine hemoglobin on hemodynamics and oxygen transport in patients undergoing preoperative hemodilution for elective abdominal aortic surgery," Anesth. Analg. 87: 284-91, which is hereby incorporated by reference in its entirety). One study has examined the utility of these solutions in the acute resuscitation phase of unstable trauma patients. Design of the study, however, was poor, and any role of the solutions in influencing ultimate patient outcome was unclear (see Koenigsberg D. et al. (1999) "The efficacy trial of diaspirin cross-linked hemoglobin in the treatment of severe traumatic hemorrhagic shock," Acad. Emerg. Med. 6: 379-80, which is hereby incorporated by reference in its entirety).<br>
Many of the problems of cell-free hemoglobin can be overcome by encapsulating it ' with an artificial membrane. Liposomes are being used to encapsulate hemoglobin for use as a blood substitute. The approach is technically challenging because not only must the hemoglobin be prepared, it must be encapsulated in relatively high concentration and yield. The final products must be sterile and the liposomes must be relatively uniform in size.<br>
Encapsulated hemoglobin has several advantages over cell-free hemoglobin. Firstly, the artificial cell membrane protects hemoglobin from degradative and oxidative forces in the plasma. Secondly, the membrane protects the vascular endothelium from toxic effects of hemoglobin. These relate to heme loss, the production Oj free radicals and vasoconstrictor effects of NO binding. Thirdly, encapsulation greatly increases the circulating persistence of the hemoglobin. Moreover, encapsulated hemoglobin can be freeze-dried for convenient storage.<br>
Liposomal encapsulation involves phospholipids, as in cell membranes. One major problem associated with liposomal encapsulation, however, is that it is very difficult to regulate the average size and distribution of liposomes. Another is that unlike red blood cells, liposomes are often not very stable, as they ordinarily lack an organized cytoskeleton. Yet another problem is that liposomes often consist of multiple layers of phospholipid. (A recent review of blood substitute development is presented in S to we 11 et al. (2001) Progress in the development of RBC substitutes, Transfusion 41:287-299, which is hereby<br><br>
incorporated by reference in its entirety. See also Chang, T. 1998 "Modified hemoglobin-based blood substitutes: cross linked, recombinant and encapsulated hemoglobin," Artificial Cell 74 Suppl 2:233-41, which is hereby incorporated by reference in its entirety.)<br>
Red blood cell substitutes employing polypeptides designed using the method of the present invention should offer several advantages over approaches to the development of red blood cell substitutes known in the art, including, without limitation, superior oxygen and carbon dioxide binding functionality, lower production cost (large-scale and therefore low-cost production is possible because bacteria can be used to mass-produce the peptides and because peptide ELBL can be automated), the possibility of using suitable preparations of hemoglobin as a template for ELBL, polypeptide biodegradability, the immune "inertness" of designed polypeptides based on blood protein structure, and the structural stability exhibited by designed polypeptide films, which exceeds that of liposomes. Polypeptide ELBL assembly yields semi-porous films, minimizing the amount of material required for fabricating a means of encapsulation and enabling glucose, oxygen, carbon dioxide, and various metabolites to diffuse as freely through the films as a lipid bilayer. In contrast, other polymers potentially suitable for this purpose have undesirable side effects—for example, polylactide degrades into lactic acid, the substance that causes muscle cramps, and poly (styrene sulfonate) is not biocompatible.<br>
Microcapsules could be formed of designed polypeptides to encapsulate hemoglobin to serve as a red blood cell substitute. Hemoglobin polypeptide microcapsules could also be engineered to incorporate enzymes, including superoxide dismutase, catalase, and methemoglobin reductase, which are ordinarily important for red blood cell function. Moreover, the nanofabricated microcapsules can predictably be dehydrated, suggesting that artificial red blood cells made as described herein could be dehydrated, without loss of function, particularly because hemoglobin can be lyophilized (i.e., freeze-dried) and reconstituted without loss of function, and polyion films are stable to dehydration. This will be important for long-term storage, transport of blood substitutes, battlefield applications (particularly in remote locations), and space exploration.<br>
Polypeptides designed using the method of the present invention could also be used for drug delivery.<br>
2.       Drug Delivery<br>
Mioron-sized "cores" of a suitable therapeutic material in "crystalline" form can be encapsulated by designed polypeptides, and the resulting microcapsules could be used for drug delivery. The core must be insoluble under some conditions, for instance high pH or<br><br>
low temperature, and soluble under the conditions where controlled release will occur. The surface charge on the crystals can be determined by f-potential measurements (used to determine the charge in electrostatic units on colloidal particles in a liquid medium). The rate at which microcapsule contents are released from the interior of the microcapsule to the surrounding environment will depend on a number of factors, including the thickness of the encapsulating shell, the polypeptides used in the shell, the presence of disulfi.de bonds, the extent of cross-Hnking of peptides, temperature, ionic strength, and the method used to assemble the peptides. Generally, the thicker the capsule, the longer the release time—the principle resembles that of gel filtration chromatography.<br>
Some work has been done on sustained release from ELBL microcapsules (see Antipov, A., Sukhorukov, G.B., Donath, E., andMohwald, H. (2001) J. Phys.Chem. B, 105:2281-2284 and Freemantle, M. (2002) Polyelectrolyte multilayers, Chem. Eng. News, 80: 44-48, both of which are incorporated herein by reference in their entireties). Polyelectrolytes that have been used are PSS, PAH, PAA, PVS, PEI, and PDDA.<br>
Polypeptides designed using the method of the present invention should offer a number of advantages in the context of drug delivery, including without limitation control over the physical, chemical, and biological characteristics of the microcapsule; the ability to make capsules with a diameter of less than 1 mm, making the capsules suitable for injection; low likelihood of eliciting an immune response; generally high bioconipatibility of capsules; control over the diffusive properties of the microcapsules by varying the thickness of the layers and using cysteine, as discussed below; the ability to target specific locations by modification of the microcapsule surface using the highly reactive sulfhydryl groups in cysteine (as is well known in the art, free sulfhydryl groups, free amino groups, and free carboxyl groups are sites to which molecules for specific targeting could be attached), or by incorporation of a specific functional domain in the design of the polypeptide; and the ability of microstructures to be taken up by cells using either endocytosis or pinocytosis.<br>
Polypeptides designed using the method of the present invention could also be used for antimicrobial coatings.<br>
3.        Antimicrobial Coatings<br>
The method of the present invention could be used to manufacture films encompassing antimicrobial peptides. For example, one suitable sequence might be Histatin 5, which occurs in humans:<br>
Asp Ser His Ala Lys Arg His His Gly Tyr Lys Arg Lys His Glu<br>
Lys His His Ser His Arg Gly Tyr	(SEQ ID NO: 8)<br>
The preponderance of positive charge at slightly basic pH makes this sequence quite suitable for ELBL. It could be appended to a peptide designed using the method of the present invention, resulting in an antimicrobial peptide suitable for use in ELBL. This peptide could be used as an anti-biofouling coating. For instance, the peptide could be used to form a coating on devices used for implantation.<br>
There are also a number of other areas in which polypeptides designed using the method of the present invention could be useful. 4.        Other Uses<br>
Other possible uses for peptides designed using the method of the present invention include without limitation food covers, wraps, and separation layers; food casings, pouches, bags, and labels; food coatings; food ingredient microcapsules; drug coatings, capsules, and microcapsules; disposable food service items (plates, cups, cutlery); trash bags; water-soluble bags for fertilizer and pesticides; microcapsules for fertilizer and pesticides; agricultural mulches; paper coatings; loose-fill packaging; disposable medical products (e.g. gloves and gowns); and disposable diapers.<br>
D.       Fabrication<br>
Once amino acid sequence motifs have been selected from those identified using the method discussed in Part VII(B)(1) above or designed de novo, the designed polypeptide is synthesized using methods well known in the art, such as solid phase synthesis and F-moc chemistry orheterologous expression following gene cloning and transformation. Designed polypeptides may be synthesized by a peptide synthesis company, for example SynPep Corp. (Dublin, California), produced in the laboratory using a peptide synthesizer, or produced by recombinant methods.<br>
In one embodiment, a designed polypeptide consists of individual amino acid sequence motifs joined in tandem, The same motif may be repeated, or different motifs may be joined in designing a polypeptide for ELBL. Moreover, functional domains may be included, as discussed above. Other amino acids than glycine could be used to link the sequence motifs, and amino acids other than the 20 usual ones could be included in the motifs themselves, depending on the properties desired of the polypeptide. Other properties could likewise be specified by design requirements, using methods known in the art. For example, proline could be included for turn formation, glycine for chain flexibility, and histidine for pH-sensitive charge properties near neutral pH. "Hydrophobic" amino acids could also be<br><br>
included—hydrophobic residue content could play a role in assembly behavior and contribute to layer stability in a way resembling the hydrophobic stabilization of globular proteins.<br>
It is preferred that fabricated polypeptides be at least 15 amino acids long, although it is more preferred that the fabricated polypeptides be at least 32 amino acids long. The reason for this is that the entropy loss per molecule is so thermodynamically unfavorable for short polymers that adsorption to an oppositely-charged surface is inhibited, even if the polypeptide has a charge per unit length of 1; long polyelectrolytes adsorb better than short ones. This is illustrated in Figure 12. The average molecule masses of the peptides utilized for the length-dependence studies were 1,500-3,000 Da (poly-L-glutamate, "small"), 3,800 Da (poly-L.-lysinc, "small"), 17,000 Da (poly-L-glutamate, "medium"), 48,100 Da (poly-L-lysine, "medium"), 50,300 Da (poly-L-glutamate, "large"), and 222,400 Da (poly-L-lysine, "large"). The data shown in Figure 12 clearly indicate that ELBL depends on length of peptide. Inclusion of Cys enables the use of relatively small peptides for ELBL, because the sulfhydryl group can be used to form disulfide crosslinks between polypeptides. E.        Experiments<br>
1.        Example 1—Design of Polypeptides Based on Human Blood Protein<br>
Sequences and their Use in Polypeptide Film Fabrication<br>
For this work, amino acid sequences were selected using the process described in Part V11(B)(1) above to identify sequence motifs in the primary structure of human blood proteins: Complement C3 (gi|68766) was the source of the anionic sequence motifs, and lactotransferrin (gi|4505043) the source of the cationic sequence motifs. As discussed above, blood protein sequences were used to minimize the immune response of patients into whom devices involving the polypeptides might be introduced (including, e.g. artificial red blood cells). In principle, this approach should be applicable for any organism having an immune system; it is not limited to humans. Polypeptides were synthesized by SynPep Corp. (Dublin, California). The polypeptide sequences were:<br>
Tyr Glu Glu Asp Glu Cys Gin Asp Gly Glu Glu Asp Glu Cys<br>
Gln Asp Gly Glu Glu Asp Glu Cys Gin Asp Gly Glu Glu Asp<br>
Glu Cys0 Gln Asp         .	(SEQIDNO: 2)<br>
Tyr Arg Arg Arg Arg Ser Val Gin Gly Arg Arg Arg Arg Ser<br>
Val Gln Gly Arg Arg Arg Arg Ser Val Gln Gly Arg Arg Arg<br>
Arg Ser Val Gln	(SEQIDNO: 1)<br><br>
Tyr Glu Glu Asp Glu Cys Gin Asp Gly Glu Glu Asp Glu Cys<br>
Gln Asp Gly Glu Glu Asp Glu Cys Gln Asp Gly Glu Glu Asp<br>
Glu Cys Gin Asp Gly Glu Glu Asp Glu Cys Gln Asp Gly Glu<br>
Glu Asp Glu Cys Gln Asp	(SEQ ID NO: 4)<br>
Tyr Arg Arg Arg Arg Ser Val Gln Gly Arg Arg Arg Arg Ser<br>
Val Gin Gly Arg Arg Arg Arg Ser Val Gin Gly Arg Arg Arg<br>
Arg Ser Val Gln Gly Arg Arg Arg Arg Ser Val Gln Gly Arg<br>
Arg Arg Arg Ser Val Gln	(SEQIDNO: 3)<br>
The amino acid residues are represented by the three-letter code given above. One glycine was introduced between each 7-residue motif to inhibit secondary structure formation. Glycine was selected for this purpose because it allows the greatest variability in combination of dihedral angles (see Ramachandran, G.N. and Saisekharan, V. (1968), Adv. Protein Chemistry, 23:283, which is incorporated by reference herein in its entirety) and has a low helix propensity (0.677) and low sheet propensity (0.766). Alternatively, proline could be substituted for glycine between motifs on the basis of calculated structure propensities. Additionally, a single tyrosine was included at the N-terminus of each peptide for concentration determination by UV absorption at 280 run.<br>
The polypeptides were named SN1 (SEQ ED NO: 2), SP2 (SEQ ED NO: 1), LN3 (SEQ ID NO: 4), and LP4 (SEQ ID NO 3), respectively, meaning short negative^short positive, long negative, and long positive. These sequences are quite different from polylysine (commonly used in the art as a polycation) and polyglutamate (commonly used in the art as a polyanion) which, though available commercially and inexpensive, have a high a-helix propensity under conditions of mild pH and, crucially, are immunoreactive. The calculated charge per unit length on the designed peptides at neutral pH is 0.5 electrostatic units for SP and LP and 0.6 electrostatic units for SN and LN. The positive peptides are somewhat more hydrophobic than the negative ones, owing to the presence of valine and the long hydrocarbon side chain of arginine. (As mentioned above, hydrophobic interactions between polypeptide layers could stabilize films to some extent.) The lengths are consistent with published studies showing that polyions must have greater than 20 charged groups (i.e. aspartic acid and glutanaic acid; lysine, arginine, and histidine) to be suitable for ELBL (see Kabanov, V. and Zezin, A. (1984) Pure Appl, Chem. 56:343 and Kabanov, V. (1994) Polym. Sci. 36:143, both of which are incorporated by reference herein in their entireties).<br><br>
a.        Experimental demonstration<br>
i.        Materials<br>
QCM electrodes (USI-System, Japan) coated with evaporated silver had a surface area of 0.16 ±0.01 cm2 on each side, a resonant frequency of 9 MHz (AT-cut), and a long-term stability of ± 2 Hz. The polypeptide molecular weight was verified by electrospray mass spectrometry. Peptide purity was greater than 70 %. The polypeptide buffer was 10 mM sodium phosphate or 10 mM Tris-HCI, 1 mM DTT, 0.1 mM sodium azide, pH 7.4. All chemicals other than polypeptides were purchased from Sigma-Aldrich (USA).<br>
ii.        Procedures<br>
Experiments were done using pairs of designed polypeptides, one negative and one positive. Multilayer films consisting of at least 5 bi-layers of the above-identified SP2, SN1, LP4, and LN3 were deposited onto the QCM resonators using standard ELBL techniques (a bi-layer consists of one layer of polycation and one layer of polyanion). The polypeptide concentration used for layer adsorption was 2 mg-mU-1. It is known that dependence of polyion layer thickness on polyelectrolyte concentration is not strong (see Lvov, Y. and Decher, G. (1994) Crystallog. Rep. 39:628, which is incorporated herein by reference in its entirety); in the concentration range 0.1 to 5 mg mL-1, bilayer thickness was approximately independent of concentration for PSS/PAH. By contrast, polypeptide thin films appear substantially less thick than those fabricated using high molecular weight PSS/PAH (mass calculated using A/data using the well-known Sauerbrey equation); see Lvov, Y. and Decher,<br>
G. (1994) Crystallog. Rep. 39:628. This follows from calculating film thickness on the basis<br>
/ of mass deposited, as is ordinarily done in the art for proteins. The calculated thickness for<br>
the designed polypeptide assembly shown in Figure 3(c) is greater man the end-to-end length of the peptides used to make the film. DTT was included at 1 mM to inhibit disulfide bond formation. The adsorption time was 20 minutes.<br>
Resonators were rinsed for 1 min. in pure water between subsequent adsorption cycles (removing perhaps 10-15 % of weakly adsorbed material) and dried in a stream of gaseous N2. Then the mass of the deposited peptide was measured indirectly by QCM. The mass measurement includes some water, despite drying, and low mass ions like K+, Na+, and Cl~. Partial interpenetration of neighboring layers of peptide is probable (see Decher, G. (1997) Science 227:1232; Schmitt et al. (1993) Macromolecules 26:7058; and Korneev et al. (1995) Physica B 214:954); this could be important for the efficiency of disulfide "locking."<br><br>
iii.       Results<br>
After adsorption of the polypeptide and rinsing and drying the QCM resonator, the resonant frequency of the resonator was measured. This enabled calculation of the frequency shift on adsorption and change in adsorbed mass. A decrease in frequency indicates an increase in adsorbed mass. The results are provided in Figures 3(a) and 3(b). Figure 3(a) shows a comparison of adsorption data for LP4 and LN3 in different buffers (10 mM sodium phosphate, pH 7.4, 1 mM DTT and 10 mM Tris-HCl, pH 7.4,1 mM DTT). It is clear from these data that adsorption depends more on the properties of the peptides than the specific properties of the buffer used. Figure 3(b) shows resonator frequency versus adsorbed layer for different combinations of SP2, SN1, LP4, and LN3 (namely, SP2/SN1, SP2/LN3, LP4/SN1, and LP4/LN3) in 10 mM sodium phosphate, pH 7.4 and 1 mM DTT (the lines merely connect experimental data points). Each of these combinations involved one negative polypeptide and one positive polypeptide, as required by ELBL. Figure 3(c) shows a graph of calculated adsorbed mass versus layer number for SN1 and LP4 in 10 mM Tris-HCl, pH 7.4 and 1 mM DTT (calculated from experimental data using the Sauerbrey equation). The total adsorbed mass, approximately 5 /ig, corresponds approximately to 1 nmol of peptide. The equation used for mis calculation was Am = — 0.87-10~ A/, where m is mass in grams and/is frequency in Hz (see Lvov, Y., Ariga, KL, Ichinose, I., and Kunitake, T. (1995) J. Am. Chem. Soc. 117:6117 and Sauerbrey, G. (1959) Z. Physik 155:206, both of which are incorporated herein by reference in their entireties). Film thickness, d, is estimated as d = -0.016A/, where d is in nm (see Yuri Lvov, "Electrostatic Layer-by-Layer Assembly of Proteins and Polyions" in Protein Architecture: Interfacial Molecular Assembly and Immobilization Biotechnology, (Y. Lvov &amp; H. Mohwald eds., 2000) (New York: Dekker, 2000) pp. 125-167, which is incorporated herein by reference). The line in Figure 3(c) is a linear fit to experimental data points. The linearity of the data is a likely indicator of precise, regular assembly during adsorption and an approximately uniform density of the polypeptides in each adsorbed layer. Adsorption occurred with a frequency shift of- 610 db 60 Hz (cations) or - 380 ± 40 Hz (anions). Linear growth of deposited polypeptide mass indicates repeatability of adsorption steps early in the assembly process and the general success of the multilayer fabrication process.<br>
iv.       Conclusions<br>
The above results show that polypeptides designed using the method of the present invention are suitable for ELBL, despite significant qualitative differences from PSS and PAH, flexible homopolymers having 1 charge per unit length at pH 7.4. The charge per unit<br><br>
length on poly-L-lysine and poly-L-glutamic acid is 1 at pH 7.4, as with PSS and PAH, but both of these polypeptides have a marked propensity to form Of-helical structure under various conditions, making them substantially less suitable for multilayer assembly when control over thin film structure is desired. The monodisperse polypeptides of the present invention, however, enable the practitioner to know, quite precisely, the structure of the material being used for ELBL. Moreover, usual commercial preparations of poly-L-lysine and poly-L-glutamic acid are polydisperse, and poly-L-lysine, poly-L-glutamic acid, PSS, and PAH evoke an immune response (i.e.are immunogenic)in humans.<br>
Because the designed polypeptides are readily adsorbed on an oppositely charged surface, as demonstrated by experiment, there is no need for a "precursor" layer. As is known in the art, "precursor" layers are deposited on a substrate to enhance adsorption of less adsorptive substances. The lack of a precursor layer enhances the biocompatibility of the polyion films because polymers ordinarily used as precursors are immunogenic or allow less precise control over polymer structure or thin film structure than designed polypeptides.<br>
Multilayers of the designed polypeptides were stable at the pH of human blood, 7.4. Thus, the multilayers should be useful for a broad range of biological applications. Adsorption of the designed polypeptides, each of less than 1 charge per residue, was essentially complete in less than 10 min. at 2 mg/mL and low ionic strength. This implies that these polypeptides can be used to form multilayer films quickly and with relative ease. Drying the peptide film with N2(g) after deposition of each layer did not impair assembly. Drying is done to get an accurate QCM frequency measurement, but is not required for assembly.<br>
The film assembly experiments were done at a lower ionic strength than that of blood, but the process gives a qualitatively similar result at higher ionic strength. The chief difference is the amount of peptide deposited per adsorption layer—the higher the ionic strength, the greater the amount of peptide deposited. This is illustrated by the graph in Figure 7, which shows the amount of material deposited as a function of ionic strength—the peptides used were poly-L-glutamic acid and poly-L-lysine. QCM resonant frequency is plotted against adsorption layer. The average molecular mass of poly-L-glutamate was 84,600 Da, while that of poly-lys was 84,000 Da. The peptide concentration used for assembly was 2 mg/mL. The data indicate salt concentration (ionic strength of solution) influences thin film assembly. In general, the amount of material deposited per layer increases with ionic strength in the range 0-100 mM NaCl. As the essential character of ELBL with designed polypeptides appears not to depend on the choice of buffer under<br><br>
conditions of relatively high net charge per unit length and low ionic strength, qualitatively similar results are expected at the ionic strength of human blood. Thus, the choice of buffer should not fundamentally alter the stability of the multilayers in their target environment. However, even if the choice of buffer did affect the stability of the multilayers, the "locking" mechanism would be available as a design feature to stabilize the capsule.<br>
The greater apparent deposition of positive polypeptides than negative ones may result from the higher charge per unit length on the positive polypeptides at pH 7.4. The material deposited in each layer probably corresponds to that required for neutralization of the charge of the underlying surface. Hydrophobic interactions could also help to explain this feature of adsorption behavior.<br>
The usual thin film thickness calculation for proteins and other polymers is probably invalid for short polypeprides (calculated thickness is 60-90 nm, but summed length of 10 polypeptides is approximately 120 nm). This probably results from a high density of packing of the relatively short polypeptides onto the adsorption surface; the result is also consistent with finding that film thickness varies with ionic strength, as changes in structural properties of a polymer will occur and screening of charges by ions will reduce intra-layer charge repulsion between adsorbed peptides. The thickness of the designed polypeptide thin film discussed here is estimated at 20-50 nm.<br>
Many aspects of the design and fabrication cycles could be automated. For example, a computer algorithm could be used to optimize the primary structure of peptides for ELBL by comparing predicted peptide properties with observed physical properties, including structure in solution, adsorption behavior, and film stability at extremes of pH. Moreover, the polypeptide film assembly process can be mechanized, once the details of the various steps have been sufficiently determined.<br>
2.        Example 2—Experiments Involving De Novo-Designed Polypeptides Containing Cysteine a.        Polypeptides The polypeptides used were:<br>
Tyr Lys Cys Lys Gly Lys Val Lys Val Lys Cys Lys Gly Lys<br>
Val Lys Val Lys Cys Lys Gly Lys Val Lys Val Lys Cys Lys<br>
Gly Lys Val Lys	(SEQEDNO: 5)<br><br>
Tyr Glu Cys Glu Gly Glu Val Glu Val Ghi Cys Glu Gly Glu<br>
Val Glu Val Glu Cys Glu Gly Glu Val Glu Val Glu Cys Glu<br>
Gly Glu Val Glu	(SEQIDNO: 6)<br>
Unlike the other polypeptides used hi the experiments described herein, these two were not<br>
/<br>
designed using human genome information; they were designed for the sole purpose of assessing the role of disulfide bond formation in polypeptide film stabilization.<br>
b.	Procedures<br>
All experiments were conducted at ambient temperature.<br>
All assembly experiments using QCM were conducted in the same conditions, except that the samples to undergo oxidation were dried using air instead of nitrogen gas. The assembly conditions were 10 mM Tris-HCl, 10 mM DTT, pH 7.4. The nominal peptide concentration was 2mg/ml. The number of layers formed was 14.<br>
Disulfide locking conditions for the oxidizing samples were 10 mM Tris-HCl, 1 % DMSO, saturation of water with air, pH 7.5. The duration of the "locking" step was 6 hours. Conditions for the reducing samples were 10 mM Tris-HCl, 1 mM DTT, saturation of water with nitrogen, pH 7.5. The duration of this step was 6 hours.<br>
All disassembly experiments using QCM were conducted in the same conditions, except that the oxidizing samples were dried using air instead of nitrogen. Disassembly conditions were 10 mM KC1, pH 2.0 Samples were rinsed with D.I. water for 30 seconds prior to drying.<br>
Three different types of experiments were conducted: (1) Reducing—no treatment: disassembly was conducted immediately after assembly; (2) Reducing—6 hours, as described above for reducing samples; and (3) Oxidizing—6 hours, as described above for oxidizing samples.<br>
c.	Results<br>
The results are illustrated in Figure 10. In the first two experiments (both reducing), all of the deposited material (100 %) disassembled within 50 minutes. By contrast, in the oxidizing experiment, a substantial amount of material remained after substantial incubation of the peptide film-coated QCM resonator at pH 2 for over 5 hours. The stability of the polypeptide films at acidic pH is determined by the conditions of assembly; in this way, film or capsule stability is a design feature that becomes possible by using polypeptides as the polyelectrolytes for ELBL.<br><br>
d.        Conclusions<br>
Electrostatic play a key role in holding together oppositely-charged layers of designed polypeptides. At acidic pH, the net charge on one of the peptides is neutralized and the polypeptide film disassembles due to electrostatic repulsion. Reducing conditions prevent disulflde bond formation. Therefore, the electrostatic attraction between the layers is the only binding force for stabilizing the layers under these conditions. By contrast, under oxidizing conditions disulflde bonds are formed. At acidic pH, disulfide bonds inhibit film disassembly. The results indicate that layer stability at acidic pH is directly affected by the formation of intra- and/or inter-layer disulfide bonds—i.e. between molecules in the same layer, between molecules in adjacent layers, or both. This is illustrated by the results shown in Figure 10—due to disulfide locking, more than 30 % of the film remained stable at acidic pH, despite electrostatic repulsion at relatively low ionic strength. Peptides with more cysteine residues are anticipated to further improve disulfide locking efficiency. Moreover, adjustment of the conditions of peptide assembly will be an important aspect of engineering films to have the desired physical as well as chemical and biological properties.<br>
3.         Example 3—Experiments Involving Designed Polypeptides Containing Cysleine a.        Materials<br>
The essential elements of this experiment were a quartz crystal microbalance instrument; silver-coated resonators (9 MHz resonant frequency); the negative 48-residue peptide (LN3) (SEQ ID NO: 4); and a positive 48-residue peptide named "SP5" of the following sequence:<br>
Tyr Lys Gly Lys Lys Ser Cys His Gly Lys Gly Lys Lys Ser Cys<br>
His Gly Lys Gly Lys Lys Ser Cys His Gly Lys Gly Lys Lys Ser<br>
Cys His	(SEQ ID NO: 7)<br>
Like the other designed peptides discussed above in Part VII(E)(1), SP5 was designed using Ihe process described above in Part VII(B)(1) to analyze the amino acid sequence of the human blood protein lactotransferrin (gi|4505043). The ELBL buffer was 10 mM Tris, pH 7.4,10 mM NaCl, and 1 mM DTT. The disassembly buffer was 10 mM KC1, pH 2. 2 mL peptide solutions were prepared for SP5 and LN3 by adding 4 mg of each peptide to 2 mL of the above buffer solution and adjusting the pH of each solution to 7.4; the peptide concentration was 2 mg-mU"1.<br><br>
b.	Procedure for Monitoring Assembly of Polypeptide Layers on<br>
QCM Resonators<br>
Reducing procedures were as follows: (1) The frequency of the resonator was measured and recorded prior to peptide adsorption; (2) The resonator was dipped into the SP5 peptide solution for 20 min.; (3) The resonator was dipped into the SP5 rinse solution for 30 sec.; (4) The resonator was removed from the rinse solution and dried using nitrogen gas; (5) The QCM resonant frequency of the resonator was recorded; (6) The resonator was dipped into the LN3 peptide solution for 20 min.; (7) The resonator was dipped into the LN3 rinse solution for 30 sec.; (8) The resonator 1 was removed from the rinse solution and dried using nitrogen gas; (9) The QCM resonant frequency of the resonator was recorded; (10) Steps 2 through 9 were repeated until 16 layers were adsorbed onto the resonator.<br>
Oxidizing procedures were the same as the reducing procedures, except that the resonator was rinsed in D.I. water instead of the SP5 buffer or the LN3 buffer and dried with air instead of nitrogen before each measurement.<br>
c.	Locking Procedures<br>
Reducing procedures were as follows: The resonator was placed in an aqueous solution containing 1 mM DTT for 6 hours. DTT, a reducing agent, inhibited disulfide bond formation.<br>
Oxidizing procedures were as follows: The resonator was placed in an air-saturated aqueous solution containing 1 % DMSO for 6 hours. DMSO, an oxidizing agent, promoted disulfide bond formation.<br>
d.	Disassembly on Resonator<br>
i.         Solutions<br>
Reducing conditions were as follows: 10 mM K.C1, 1 mM DTT, pH 2.<br>
Oxidizing conditions were as follows: 10 mM KC1, 20 % DMSO, pH 2.<br>
ii.        Procedure for Disassembly<br>
Reducing procedures were as follows: (1) The initial resonant frequency of the resonator was measured by QCM and recorded; (2) The resonator was dipped into the reducing disassembly solution for 5 min.; (3) The resonator was rinsed in reducing buffer solution for 30 sec.; (4) The resonator was dried with gaseous N2; (5) The resonant frequency of the resonator was measured by QCM and recorded; (6) Steps 2 through 5 were repeated for reading times of 5,10,15,20, 30, 60, and 90 min.<br>
Oxidizing procedures were the same as for reducing procedures, except that rinsing of the resonator was done in D.I. water saturated with air instead of reducing buffer.<br><br>
e.	Results<br>
Figure 8 shows approximately linear increase in mass deposited during thin film assembly of SP5 and LN3. Both resonators show almost identical deposition of mass throughout the process of assembly, despite differences in assembly conditions.<br>
Figure 9 shows the percentage of material remaining during film disassembly. The layers subjected to oxidizing conditions showed a minimal loss of material at acidic pH with almost 90 to 95 % of mass retention. By contrast, layers subjected to reducing conditions lost almost all the film material with the first, 5 minutes of exposure to acidic pH.<br>
f.	Conclusions<br>
The results demonstrate that at acidic pH, disulfide bonds prevent layer degeneration and hold the layers firmly together. Layer stability at acidic pH is directly affected by the formation of intra- and/or inter-layer disulfide bonds. Disulfide bond formation is dependent on the concentration and proximity of cysteine residues to each other. Increasing the concentration per unit chain length of the polypeptide would therefore directly influence disulfide bond formation and thin film stability. Increasing the ionic strength of the buffer solutions used for film assembly influences the concentration of cysteine in the film by increasing the amount of material deposited per adsorption cycle and the thickness of each layer. The increased number of cysteine aniino acids in a single layer would in this way increase the number of disulfide bonds formed, and, on oxidation, increase film stability.<br>
Other embodiments of the invention are possible and modifications may be made without departing from the spirit and scope of the invention. Therefore, the detailed description above is not meant to limit the invention. Rather, the scope of the invention is defined by the appended claims.<br><br><br><br><br><br>
1.	A method for identifying, in amino acid sequence information, and recording a<br>
desired number of amino acid sequence motifs having a defined length and a net charge of a<br>
certain polarity for use in electrostatic layer-by-layer self assembly, comprising the steps of:<br>
a.	Obtaining an amino acid sequence for a peptide or a protein from a particular<br>
organism;<br>
b.	Locating a starter amino acid in said amino acid sequence;<br>
c.	Examining said starter amino acid and the following n amino acids to<br>
determine the number of charged ainino acids having a polarity opposite said certain<br>
polarity;<br>
d.	If the number of said charged amino acids having a polarity opposite said<br>
certain polarity is one or more, continuing the method at step g;<br>
e.	Examining said starter amino acid and the following n ainino acids to<br>
determine the number of charged amino acids having said certain polarity;<br>
f.	If the number of charged amino acids having said certain polarity is equal to or<br>
greater than x,  recording the amino acid sequence motif consisting of said starter<br>
amino acid and the following n ainino acids;<br>
g.	Locating another starter amino acid in said amino acid sequence; and<br>
h.        Repeating the method beginning at step c until the desired number of ainino acid sequence motifs have been identified or all of the amino acids in said amino acid sequence have been used as said starter amino acid in step c; wherein x is greater than or equal to approximately one-half of n.<br>
2.	The method of claim 1, wherein n is less than or equal to 11, but greater than or equal<br>
to 3.<br>
3.	The method of claim 1, wherein n = 8 and x = 5.<br>
4.	The method of claim 1, wherein n = 6 and x = 4.<br>
5.	The method of claim 1, wherein at least one of said recorded amino acid sequence<br>
motifs comprises cysteine.<br>
6.	A method for designing a polypeptide for use in electrostatic layer-by-layer self<br>
assembly, comprising the steps of:<br>
a.        Identifying, in amino acid sequence information, and recording one or more amino acid sequence motifs having a defined length and a net charge of a certain polarity using the following steps:<br>
i.         Obtaining an amino acid sequence for a peptide or a protein from a<br>
particular organism;<br>
ii.        Locating a starter amino acid in said amino acid sequence;<br>
iii.       Examining said starter amino acid and the following n amino acids to<br>
determine the number of charged amino acids having a polarity opposite said<br>
certain polarity;<br>
iv.        If the number of said charged amino acids having a polarity opposite<br>
said certain polarity is one or more, continuing the method at step vii;<br>
v.        Examining said starter amino acid and the following n amino acids to<br>
determine the number of charged amino acids having said certain polarity;<br>
vi.        If the number of charged amino acids having said certain polarity is<br>
equal to or greater than x, recording the amino acid sequence motif consisting<br>
of said starter amino acid and the following n amino acids;<br>
vii.       Locating another starter amino acid in said amino acid sequence; and<br>
viii.     Repeating the method beginning at step iii until the desired number of<br>
amino acid sequence motifs have been identified or all of the amino acids in<br>
said amino acid sequence have been used as said starter amino acid in step iii;<br>
wherein x is greater than or equal to approximately one-half of n; and<br>
b.        Joining a plurality of said recorded amino acid sequence motifs to form a<br>
polypeptide.<br>
7.	The method of claim 6, wherem n is less than or equal to 11, but greater than or equal<br>
to 3.<br>
8.	The method of claim 6, wherein n = 8 and x — 5.<br>
9.	The method of claim 6, wherein n - 6 and x = 4.<br>
10.	The method of claim 6, wherein at least one of said plurality of joined amino acid<br>
sequence motifs comprises cysteine.<br>
11.	The method of claim 6, wherein at least one of said plurality of joined amino acid<br>
sequence motifs has a summed α-helix propensity of less than 7.5 and a summed ß-shect<br>
propensity is less than 8.<br>
I?..       The method of claim 6, further comprising the step of including in the polypeptide a functional domain.<br>
13.      The method of claim 6, wherein said amino acid sequence comes from the proteome of said organism.<br><br>
14.	The method of claim 13, wherein said amino acid sequence comes from a blood<br>
protein from said organism.    <br>
15.	The method of claim 14, wherein said organism is a human.<br>
16.	The method of claim 14, wherein at least one of said plurality of joined amino acid<br>
sequence motifs comprises the sequence described in SEQ ID NO: 1.<br>
17.	The method of claim 14, wherein at least one of said plurality of joined amino acid<br>
sequence motifs comprises the sequence described in SEQ ID NO: 2.<br>
18.	The method of claim 14, wherein at least one of said plurality of joined amino acid<br>
sequence motifs comprises the sequence described in SEQ ID NO: 3:<br>
19.	The method of claim 14, wherein at least one of said plurality of joined amino acid<br>
sequence motifs comprises the sequence described in SEQ ED NO: 4.<br>
20.	The method of claim 14, wherein at least one of said plurality of joined amino acid<br>
sequence motifs comprises the sequence described in SEQ ID NO: 7.<br>
21.	A thin film, said film comprising a plurality of layers of polypeptides, said layers of<br>
polypeptides having alternating charges, wherein at least one of said polypeptides comprises<br>
one or more amino acid sequence motifs consisting of n ammo acids, at least x of which are<br>
positively charged and none is negatively charged, or at least x of which are negatively<br>
charged and none is positively charged, wherein x is greater than or equal to approximately<br>
one-half of n.<br>
22.	The thin film of claim 21, wherein said at least one of said polypeptides comprises<br>
non-natural amino acids.<br>
23.	The thin film of claim 21, wherein said thin film further comprises at least one<br>
antimicrobial polypeptide.<br>
24.	The thin film of claim 23, wherein said antimicrobial polypeptide comprises the<br>
sequence described in SEQ ED NO: 8.<br>
25.	The method of claim 21, wherein said at least one of said polypeptides comprises a<br>
functional domain.<br>
26.	The thin film of claim 21 wherein said at least one amino acid sequence motif is<br>
  identified by the method of:<br>
a.	Obtaining an amino acid sequence for a peptide or a protein from a particular<br>
organism;<br>
b.	Locating a starter amino acid in said amino acid sequence;<br><br>
c.	Examining said starter amino acid and the following n -1 amino acids to<br>
determine the number of charged amrno acids having a polarity opposite said certain<br>
polarity;<br>
d.	If the number of said charged amino acids having a polarity opposite said<br>
certain polarity is one or more, continuing the method at step g;<br>
e.	Examining said starter amino acid and the following n - 1 amino acids to<br>
determine the number of charged amino acids having said certain polarity;<br>
f.	If the number of charged amino acids having said certain polarity is equal to or<br>
greater than x, recording the amino acid sequence motif consisting of said starter<br>
amino acid and the following n - 1 amino acids;<br>
g;,	Locating another starter amino acid in said amino acid sequence;  and<br>
h.         Repeating the method beginning at step c until the desired number of amino<br>
acid sequence motifs have been identified or all of the amino acids in said amino acid<br>
sequence have been used as said starter amino acid in step iii;<br>
wherein x is greater than or equal to approximately one-half of n.<br>
27.	The thin film of claim 26, wherein n is less man or equal to 12, but greater than or<br>
equal to 4.<br>
28.	The thin film of claim 26, wherein n=9 and x = 5.<br>
29.	The thin film of claim 26, wherein n = 7 and x = 4.<br>
30.	The thin film of claim 26, wherein said at least one amino acid sequence motif<br>
comprises cysteine.<br>
31.	The thin film of claim 26, wherein said at least one amino acid sequence motif has a<br>
summed α-helix propensity of less than 7.5 and a summed ß-shect propensity of less than 8.<br>
32.	The thin film of claim 26, wherein said at least one amino acid sequence motif is<br>
comprised of human blood proteins.<br>
33.	The thin film of claim 26, wherein said at least one amino acid sequence motif<br>
comprises the sequence described in SEQ ED NO: 1.<br>
34.	The thin film of claim 26, wherein said at least one amino acid sequence motif<br>
comprises the sequence described in SEQ ID NO: 2.<br>
35.	The thin film of claim 26, wherein said at least one amino acid sequence motif<br>
comprises the sequence described in SEQ ID NO: 3.<br>
36.	The thin film of claim 26, wherein said at least one amino acid sequence motif<br>
comprises the sequence described in SEQ ID NO: 4.<br><br>
37.	The thin film of claim 26, wherein said at least one amino acid sequence motif<br>
comprises the sequence described in SEQ ID NO: 7.<br>
38.	The thin film of claim 26, wherein said thin film forms a microcapsule.<br>
39.	The thin film of claim 38, wherein said microcapsule comprises a core, and said core<br>
comprises hemoglobin.<br>
40.	The thin film of claim 39, wherein said hemoglobin is in crystallized form.<br>
41.	The thin film of claim 39, wherein said hemoglobin is in liquid form.<br>
42.	The thin film of claim 38, wherein, said microcapsule comprises a core, and said core<br>
comprises a therapeutic drug.<br>
43.	The thin film of claim 42, wherein said therapeutic drug is in crystallized form.<br>
44.	The thin film of claim 42, wherein said therapeutic drug is in liquid form.<br>
45.	The thin film of claim 26, wherein said thin film is formed on a solid substrate.<br>
46.	The thin film of claim 45, wherein said solid substrate is a microparticle.<br>
47.	The thin film of claim 45, wherein said substrate is a medical implant.<br>
48.	A method for designing polypeptides for use in electrostatic layer-by-layer self<br>
assembly comprising the following steps:<br>
a.	Designing de novo a plurality of amino acid sequence motifs, wherein at least<br>
one of said amino acid sequence motifs consists of n amino acids, at least x of which<br>
are positively charged and none is negatively charged, or at least x of which are<br>
negatively charged and none is positively charged, x is greater than or equal to<br>
approximately one-half of n; and<br>
b.	Joining said plurality of said amino acid sequence motifs to form a<br>
polypeptide.<br>
49.	The method of claim 48, wherein n is less than or equal to 12, but greater than or<br>
equal to 4.<br>
50.	The method of claim 48, wherein n=9 and x = 5.<br>
51.	The method of claim 48, wherein n = 6 and x = 4.<br>
52.	The method of claim 48, wherein said at least one amino acid sequence motif has a<br>
summed α-helix propensity of less than 7.5 and a summed ß-sheet propensity of less than 8.<br>
53.	The method of claim 48, wherein said at least one amino acid sequence motif<br>
comprises one or more non-natural amino acids.<br>
54.	The method of claim 48, wherein said at least one amino acid sequence motif<br>
comprises right-handed amino acids.<br><br>
55.	The method of claim 48, further comprising the step of joining at least one<br>
antimicrobial peptide along with said plurality of amino acid sequence motifs.<br>
56.	The method of claim 55, wherein said at least one antimicrobial peptide comprises the<br>
sequence described in SEQ ID NO: 8.<br>
57.	The method of claim 48, further comprising the step of including a functional domain<br>
in forming said polypeptide.<br>
58.	A method for controlling the mechanical stability and porosity of a multi-layer thin<br>
film, comprising the step of providing a multi-layer thin film, said film comprising a plurality<br>
of layers of polypeptides, said layers of polypeptides having alternating charges, wherein at<br>
least one of said layers of polypeptides comprises one or more free sulfhydryl-containmg<br>
amino acids.<br>
59.	The method of claim 58, wherein said one or more free sulfhydryl-containing amino<br>
acid is cysteine.<br>
60.	The method of claim 59, wherein at least one set of adjacent layers of polypeptides<br>
comprises cysteine.<br>
61.	The method of claim 58, wherein at least one of said layers of polypeptides comprises<br>
non-natural amino acids.<br>
62.	The method of claim 58, wherein at least one of said layers of polypeptides comprises<br>
an antimicrobial peptide.<br>
63.	The method of claim 62, wherein said antimicrobial peptide comprises the sequence<br>
described in SEQ ID NO: 8.<br>
64.	The method of claim 58, wherein at least one of said layers of polypeptides comprises<br>
a functional domain.<br>
65.	The method of claim 58, wherein said polypeptides comprise at least one amino acid<br>
sequence motif consisting of n amino acids, at least x of which are positively charged and<br>
none is negatively charged, or at least x of which are negatively charged and none is<br>
positively charged, wherein x is greater than or equal to approximately one-half of n.<br><br><br><br></http:></td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mzc0NC1kZWxucC0yMDA3LWFic3RyYWN0LnBkZg==" target="_blank" style="word-wrap:break-word;">3744-delnp-2007-abstract.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mzc0NC1kZWxucC0yMDA3LWNsYWltcy5wZGY=" target="_blank" style="word-wrap:break-word;">3744-delnp-2007-claims.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mzc0NC1kZWxucC0yMDA3LUNvcnJlc3BvbmRlbmNlIE90aGVycy0oMDItMDItMjAwOSkucGRm" target="_blank" style="word-wrap:break-word;">3744-delnp-2007-Correspondence Others-(02-02-2009).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mzc0NC1ERUxOUC0yMDA3LUNvcnJlc3BvbmRlbmNlIE90aGVycy0oMDMtMDYtMjAxMSkucGRm" target="_blank" style="word-wrap:break-word;">3744-DELNP-2007-Correspondence Others-(03-06-2011).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mzc0NC1kZWxucC0yMDA3LUNvcnJlc3BvbmRlbmNlIE90aGVycy0oMDktMDUtMjAxMykucGRm" target="_blank" style="word-wrap:break-word;">3744-delnp-2007-Correspondence Others-(09-05-2013).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mzc0NC1kZWxucC0yMDA3LUNvcnJlc3BvbmRlbmNlIE90aGVycy0oMDktMDctMjAxMikucGRm" target="_blank" style="word-wrap:break-word;">3744-delnp-2007-Correspondence Others-(09-07-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mzc0NC1kZWxucC0yMDA3LUNvcnJlc3BvbmRlbmNlIE90aGVycy0oMTMtMTEtMjAxMykucGRm" target="_blank" style="word-wrap:break-word;">3744-delnp-2007-Correspondence Others-(13-11-2013).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mzc0NC1kZWxucC0yMDA3LUNvcnJlc3BvbmRlbmNlIE90aGVycy0oMTctMDItMjAxNSkucGRm" target="_blank" style="word-wrap:break-word;">3744-delnp-2007-Correspondence Others-(17-02-2015).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mzc0NC1kZWxucC0yMDA3LUNvcnJlc3BvbmRlbmNlIE90aGVycy0oMTctMTItMjAxMykucGRm" target="_blank" style="word-wrap:break-word;">3744-delnp-2007-Correspondence Others-(17-12-2013).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mzc0NC1kZWxucC0yMDA3LUNvcnJlc3BvbmRlbmNlIE90aGVycy0oMjgtMTAtMjAxMykucGRm" target="_blank" style="word-wrap:break-word;">3744-delnp-2007-Correspondence Others-(28-10-2013).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mzc0NC1kZWxucC0yMDA3LUNvcnJlc3BvbmRlbmNlIE90aGVycy0oMzAtMDktMjAxMykucGRm" target="_blank" style="word-wrap:break-word;">3744-delnp-2007-Correspondence Others-(30-09-2013).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mzc0NC1ERUxOUC0yMDA3LUNvcnJlc3BvbmRlbmNlIE90aGVycy0oMzAtMTEtMjAxMSkucGRm" target="_blank" style="word-wrap:break-word;">3744-DELNP-2007-Correspondence Others-(30-11-2011).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mzc0NC1ERUxOUC0yMDA3LUNvcnJlc3BvbmRlbmNlLU90aGVycy0oMDItMDItMjAwOSkucGRm" target="_blank" style="word-wrap:break-word;">3744-DELNP-2007-Correspondence-Others-(02-02-2009).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mzc0NC1ERUxOUC0yMDA3LUNvcnJlc3BvbmRlbmNlLU90aGVycy0oMTktMDUtMjAxMCkucGRm" target="_blank" style="word-wrap:break-word;">3744-DELNP-2007-Correspondence-Others-(19-05-2010).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mzc0NC1kZWxucC0yMDA3LUNvcnJlc3BvbmRlbmNlLU90aGVycy0oMjctMDgtMjAxMykucGRm" target="_blank" style="word-wrap:break-word;">3744-delnp-2007-Correspondence-Others-(27-08-2013).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mzc0NC1kZWxucC0yMDA3LWNvcnJlc3BvbmRlbmNlLW90aGVycy5wZGY=" target="_blank" style="word-wrap:break-word;">3744-delnp-2007-correspondence-others.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mzc0NC1kZWxucC0yMDA3LWRlc2NyaXB0aW9uIChjb21wbGV0ZSkucGRm" target="_blank" style="word-wrap:break-word;">3744-delnp-2007-description (complete).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mzc0NC1kZWxucC0yMDA3LWRyYXdpbmdzLnBkZg==" target="_blank" style="word-wrap:break-word;">3744-delnp-2007-drawings.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mzc0NC1kZWxucC0yMDA3LUZvcm0tMS0oMTctMDItMjAxNSkucGRm" target="_blank" style="word-wrap:break-word;">3744-delnp-2007-Form-1-(17-02-2015).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mzc0NC1kZWxucC0yMDA3LWZvcm0tMS5wZGY=" target="_blank" style="word-wrap:break-word;">3744-delnp-2007-form-1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mzc0NC1kZWxucC0yMDA3LWZvcm0tMTMtKDAyLTAyLTIwMDkpLnBkZg==" target="_blank" style="word-wrap:break-word;">3744-delnp-2007-form-13-(02-02-2009).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mzc0NC1kZWxucC0yMDA3LUZvcm0tMi0oMTctMDItMjAxNSkucGRm" target="_blank" style="word-wrap:break-word;">3744-delnp-2007-Form-2-(17-02-2015).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mzc0NC1kZWxucC0yMDA3LWZvcm0tMi5wZGY=" target="_blank" style="word-wrap:break-word;">3744-delnp-2007-form-2.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mzc0NC1ERUxOUC0yMDA3LUZvcm0tMy0oMDMtMDYtMjAxMSkucGRm" target="_blank" style="word-wrap:break-word;">3744-DELNP-2007-Form-3-(03-06-2011).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mzc0NC1kZWxucC0yMDA3LUZvcm0tMy0oMDktMDUtMjAxMykucGRm" target="_blank" style="word-wrap:break-word;">3744-delnp-2007-Form-3-(09-05-2013).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mzc0NC1kZWxucC0yMDA3LUZvcm0tMy0oMDktMDctMjAxMikucGRm" target="_blank" style="word-wrap:break-word;">3744-delnp-2007-Form-3-(09-07-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mzc0NC1ERUxOUC0yMDA3LUZvcm0tMy0oMTktMDUtMjAxMCkucGRm" target="_blank" style="word-wrap:break-word;">3744-DELNP-2007-Form-3-(19-05-2010).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mzc0NC1ERUxOUC0yMDA3LUZvcm0tMy0oMzAtMTEtMjAxMSkucGRm" target="_blank" style="word-wrap:break-word;">3744-DELNP-2007-Form-3-(30-11-2011).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mzc0NC1kZWxucC0yMDA3LWZvcm0tNS5wZGY=" target="_blank" style="word-wrap:break-word;">3744-delnp-2007-form-5.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mzc0NC1kZWxucC0yMDA3LUdQQS0oMDktMDUtMjAxMykucGRm" target="_blank" style="word-wrap:break-word;">3744-delnp-2007-GPA-(09-05-2013).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mzc0NC1kZWxucC0yMDA3LUdQQS0oMTctMDItMjAxNSkucGRm" target="_blank" style="word-wrap:break-word;">3744-delnp-2007-GPA-(17-02-2015).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mzc0NC1kZWxucC0yMDA3LU90aGVycy0oMTctMDItMjAxNSkucGRm" target="_blank" style="word-wrap:break-word;">3744-delnp-2007-Others-(17-02-2015).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mzc0NC1kZWxucC0yMDA3LXBjdC0xMDEucGRm" target="_blank" style="word-wrap:break-word;">3744-delnp-2007-pct-101.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mzc0NC1kZWxucC0yMDA3LXBjdC0yMjAucGRm" target="_blank" style="word-wrap:break-word;">3744-delnp-2007-pct-220.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mzc0NC1kZWxucC0yMDA3LXBjdC0yMzcucGRm" target="_blank" style="word-wrap:break-word;">3744-delnp-2007-pct-237.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mzc0NC1kZWxucC0yMDA3LVBldGl0aW9uLTEzNy0oMDktMDUtMjAxMykucGRm" target="_blank" style="word-wrap:break-word;">3744-delnp-2007-Petition-137-(09-05-2013).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=YWJzdHJhY3QuanBn" target="_blank" style="word-wrap:break-word;">abstract.jpg</a></p>
		<br>
		<div class="pull-left">
			<a href="266051-method-and-apparatus-for-processing-multimedia-data.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="266053-a-device-for-cooling-a-metal-strip-between-two-rolling-stands.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>266052</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>3744/DELNP/2007</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>14/2015</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>03-Apr-2015</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>30-Mar-2015</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>18-May-2007</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>LOUISIANA TECH UNIVERSITY FOUNDATION</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td>900 TECH DRIVE, RUSTON, LOUISIANA 71270 (US)</td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>HAYNIE, DONALD,T.</td>
											<td>601 E. MARYLAND AVENUE, APT. 205, RUSTON, LOUISIANA 71270 (US)</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>C12Q 1/68</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>PCT/US2004/039209</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td>2004-11-22</td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td></td>
									<td></td>
								    <td>NA</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/266052-nanofabricated-polypeptide-multilayer-films-coatings-and-microcapsules by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 06:33:23 GMT -->
</html>
