<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/266062-pgro-expression-units by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 06:35:01 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 266062:PGRO EXPRESSION UNITS</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">PGRO EXPRESSION UNITS</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>The invention relates to the use of nucleic acid sequences for regulating gene transcription and expression, said novel promoters and expression units, methods for modifying or inducing the gene transcription rate and/or expression rate, expression cassettes containing said expression units, genetically modified microorganisms having a modified or induced transcription rate and/or expression rate, and methods for producing biosynthetic products by cultivating said genetically modified microorganisms.</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td> <br><br>
Pgro expression units<br>
The present invention relates to the use of nucleic acid sequences for regulating the transcription and expression of genes, the novel promoters and expression units themselves, methods for altering or causing the transcription rate and/or expression rate of genes, expression cassettes comprising the expression units, genetically modified microorganisms with altered or caused transcription rate and/or expression rate, and methods for preparing biosynthetic products by cultivating the genetically modified microorganisms.<br>
Various biosynthetic products such as, for example, fine chemicals, such as, inter alia, amino acids, vitamins, but also proteins, are produced in cells by natural metabolic processes and are used in many branches of industry, including the cosmetics, feed, food and pharmaceutical industries. These substances, which are referred to collectively as fine chemicals/proteins, comprise inter alia organic acids, both proteinogenic and non-proteinogenic amino acids, nucleotides and nucleosides, lipids and fatty acids, diols, carbohydrates, aromatic compounds, vitamins and cofactors, and proteins and enzymes. Their production takes place most expediently on the industrial scale by culturing bacteria which have been developed in order to produce and secrete large quantities of the particular desired substance. Organisms particularly suitable for this purpose are coryneform bacteria, gram-positive non-pathogenic bacteria.<br>
It is known that amino acids are prepared by fermentation of strains of coryneform bacteria, especially Corynebacterium glutamicum. Because of the great importance, continuous work is done on improving the production processes. Process improvements may relate to fermentation technique measures such as, for example, stirring and oxygen supply, or the composition of the nutrient media, such as, for example, the sugar concentration during the fermentation, or the working up to give the product, for example by ion exchange chromatography or else spray drying, or the intrinsic performance properties of the microorganism itself<br>
Methods of recombinant DNA technology have likewise been employed for some years for strain improvement of Corynebacterium strains producing fine chemical/proteins, by amplifying individual genes and investigating the effect on the production of fine chemicals/proteins.<br>
Other ways for developing a process for producing fine chemicals, amino acids or proteins, or for increasing or improving the productivity of a pre-existing process for producing fine chemicals, amino acids or proteins, are to increase or to alter the expression of one or more genes, and/or to influence the translation of an mRNA by suitable polynucleotide sequences. In this connection, influencing may include<br><br>
increasing, reducing, or else other parameters of the expression of genes, such as chronological expression patterns.<br>
Various constituents of bacterial regulatory sequences are known to the skilled worker. A distinction is made between the binding sites for regulators, also called operators, the binding sites for RNA polymerase holoenzymes, also called -35 and -10 regions, and the binding site for ribosomal 16S RNA, also called ribosome binding site or else Shine-Dalgarno sequence.<br>
The sequence of a ribosome binding site, also called Shine-Dalgarno sequence, means for the purposes of this invention polynucleotide sequences which are located up to 20 bases upstream of the translation initiation codon.<br>
In the literature (E. coli and S. typhimurium, Neidhardt F.C. 1995 ASM Press) it is reported that both the composition of the polynucleotide sequence of the Shine-Dalgarno sequence, the sequence string of the bases, but also the distance of a polynucleotide sequence present in the Shine-Dalgarno sequence from has a considerable influence on the translation initiation rate.<br>
Nucleic acid sequences having promoter activity can influence the formation of mRNA in various ways. Promoters whose activities are independent of the physiological growth phase of the organism are called constitutive. Other promoters in turn respond to external chemical and physical stimuli such as oxygen, metabolites, heat, pH, etc. Others in turn show a strong dependence of their activity in different grov^h phases. For example, promoters showing a particularly pronounced activity during the exponential growth phase of microorganisms, or else precisely in the stationary phase of microbial growth, are described in the literature. Both characteristics of promoters may have a beneficial effect on productivity for a production of fine chemicals and proteins, depending on the metabolic pathway.<br>
For example, promoters which switch off the expression of a gene during growth, but switch it on after an optimal growth, can be used to regulate a gene which controls the production of a metabolite. The modified strain then displays the same growth parameters as the starting strain but produces more product per cell. This type of modification may increase both the titer (g of product/liter) and the C yield (g of product/g of C source).<br>
It has already been possible to isolate in Corynebacterium species those nucleotide sequences which can be used to increase or diminish gene expression. These regulated promoters may increase or reduce the rate at which a gene is transcribed,<br><br>
depending on the internal and/or external conditions of the cell. In some cases, the presence of a particular factor, known as inducer, can stimulate the rate of transcription from the promoter. Inducers may influence transcription from the promoter either directly or indirectly. Another class of factors, known as suppressors, is able to reduce or else inhibit the transcription from the promoter. Like the inducers, the suppressors can also act directly or indirectly. However, temperature-regulated promoters are also known. Thus, the level of transcription of such promoters can be increased or else diminished for example by increasing the growth temperature above the normal growth temperature of the cell.<br>
A small number of promoters from C. glutamicum have been described to date. The promoter of the malate synthase gene from C. glutamicum was described in DE 4440118. This promoter was inserted upstream of a structural gene coding for a protein. After transformation of such a construct into a coryneform bacterium there is regulation of the expression of the structural gene downstream of the promoter. Expression of the structural gene is induced as soon as an appropriate inducer is added to the medium.<br>
Reinscheid et al., Microbiology 145:503 (1999) described a transcriptional fusion between the pta-ack promoter from C. glutamicum and a reporter gene (chloramphenicol acetyltransferase). Cells of C. glutamicum comprising such a transcriptional fusion exhibited increased expression of the reporter gene on growth on acetate-containing medium. By comparison with this, transformed cells which grew on glucose showed no increased expression of this reporter gene.<br>
Pa'tek et al., Microbiology 142:1297 (1996) describe some DNA sequences from C. glutamicum which are able to enhance the expression of a reporter gene in C. glutamicum cells. These sequences were compared together in order to define consensus sequences for C. glutamicum promoters.<br>
Further DNA sequences from C. glutamicum which can be used to regulate gene expression have been described in the patent WO 02/40679. These isolated polynucleotides represent expression units from Corynebacterium glutamicum which can be used either to increase or else to reduce gene expression. This patent additionally describes recombinant plasmids on which the expression units from Corynebacterium glutamicum are associated with heterologous genes. The method described herein, effusing a promoter from Corynebacterium glutamicum with a heterologous gene, can be employed inter alia for regulating the genes of amino acid biosynthesis.<br><br>
It is an object of the present invention to provide further promoters and/or expression units with advantageous properties.<br>
We have found that this object is achieved by the use of a nucleic acid having promoter activity, comprising<br>
A)	the nucleic acid sequence SEQ. ID. NO. 1 or<br>
B)	a sequence derived from this sequence by substitution, insertion or deletion of nucleotides and having an identity of at least 90% at the nucleic acid level with the sequence SEQ. ID. NO. 1,<br>
or<br>
C)	a nucleic acid sequence which hybridizes with the nucleic acid sequence SEQ. ID. NO. 1 under stringent conditions, or<br>
D)	functionally equivalent fragments of the sequences of A), B) or G)<br>
for the transcription of genes.<br>
"Transcription" means according to the invention the process by which a complementary RNA molecule is produced starting from a DNA template. Proteins such as RNA polymerase, so-called sigma factors and transcriptional regulator proteins are involved in this process. The synthesized RNA is then used as template in the translation process, which then leads to the biosynthetically active protein.<br>
The formation rate with which a biosynthetically active protein is produced is a product of the rate of transcription and of translation. Both rates can be influenced according to the invention, and thus influence the rate of formation of products in a microorganism.<br>
A "promoter*' or a "nucleic acid having promoter activity" means according to the invention a nucleic acid which, in a functional linkage to a nucleic acid to be transcribed, regulates the transcription of this nucleic acid.<br>
A "functional linkage" means in this connection for example the sequential arrangement of one of the nucleic acids of the invention having promoter activity and a nucleic acid sequence to be transcribed and, where appropriate, further regulatory elements such as, for example, nucleic acid sequences which ensure the transcription of nucleic acids, and for example a terminator, in such a way that each of the regulatory elements is able to fulfill its function in the transcription of the nucleic acid sequence. A direct linkage in the chemical sense is not absolutely necessary therefor. Genetic control sequences, such as, for example, enhancer sequences, are able to exercise their function on the target sequence even from more remote positions or even from other<br><br>
DNA molecules. Arrangements in which the nucleic acid sequence to be transcribed is positioned behind (i.e. at the 3' end) of the promoter sequence of the invention, so that the two sequences are covalently connected together, are preferred. In this connection, the distance between the promoter sequence and the nucleic acid sequence to be expressed transgenically is preferably fewer than 200 base pairs, particularly preferably less than 100 base pairs, very particularly preferably less than 50 base pairs.<br>
"Promoter activity" means according to the invention the quantity of RNA formed by the promoter in a particular time, that is to say the transcription rate.<br>
"Specific promoter activity" means according to the invention the quantity of RNA formed by the promoter in a particular time for each promoter.<br>
The term "wild type" means according to the invention the appropriate starting microorganism.<br>
Depending on the context, the term "microorganism" means the starting microorganism (wild type) or a genetically modified microorganism of the invention, or both.<br>
Preferably, and especially in cases where the microorganism or the wild type cannot be unambiguously assigned, "wild type" means for the alteration or causing of the promoter activity or transcription rate, for the alteration of causing of the expression activity or expression rate and for increasing the content of biosynthetic products in each case a reference organism.<br>
In a preferred embodiment, this reference organism is Corynebacterium glutamicum ATCC 13032.<br>
In a preferred embodiment, the starting microorganisms used are already able to produce the desired fine chemical. Particular preference is given in this connection among the particularly preferred microorganisms of bacteria of the genus Corynebacterium and the particularly preferred fine chemicals L-lysine, L-methionine and L-threonine to those starting microorganisms already able to produce L-lysine, L-methionine and/or L-threonine, These are particularly preferably corynebacteria in which, for example, the gene coding for an aspartokinase (ask gene) is deregulated or the feedback inhibition is abolished or reduced. Such bacteria have, for example, a mutation leading to a reduction or abolition of the feedback inhibition, such as, for example, the mutation T3111, in the ask gene.<br>
In the case of a "caused promoter activity" or transcription rate in relation to a gene<br><br>
compared with the wild type, therefore, compared with the wild type the fonnation of an RNA which was not present in this way in the wild type is caused.<br>
In the case of an altered promoter activity or transcription rate in relation to a gene compared with the wild type, therefore, compared with the wild type the quantity of RNA produced in a particular time is altered.<br>
"Altered" means in this connection preferably increased or reduced.<br>
This can take place for example by increasing or reducing the specific promoter activity of the endogenous promoter of the invention, for example by mutating the promoter or by stimulating or inhibiting the promoter.<br>
A further possibility is to achieve the increased promoter activity or transcription rate for example by regulating the transcription of genes in the microorganism by nucleic acids of the invention having promoter activity or by nucleic acids with increased specific promoter activity, where the genes are heterologous in relation to the nucleic acids having promoter activity.<br>
The regulation of the transcription of genes in the microorganism by nucleic acids of the invention having promoter activity or by nucleic acids with increased specific promoter activity is preferably achieved by<br>
introducing one or more nucleic acids of the invention having promoter activity, appropriate with altered specific promoter activity, into the genome of the microorganism so that transcription of one or more endogenous genes takes place under the control of the introduced nucleic acid of the invention having promoter activity, appropriate with altered specific promoter activity, or<br>
introducing one or more genes into the genome of the microorganism so that transcription of one or more of the introduced genes takes place under the control of the endogenous nucleic acids of the invention having promoter activity, where appropriate with altered specific promoter activity, or<br>
introducing one or more nucleic acid constructs comprising a nucleic acid of the invention having promoter activity, where appropriate with altered specific promoter activity, and functionally linked one or more nucleic acids to be transcribed, into the microorganism.<br>
The nucleic acids of the invention having promoter activity comprise<br><br>
A)	the nucleic acid sequence SEQ. ID. NO. 1 or<br>
B)	a sequence derived from this sequence by substitution, insertion or deletion of nucleotides and having an identity of at least 90% at the nucleic acid level with the sequence SEQ. ID. NO. 1,<br>
or<br>
C)	a nucleic acid sequence which hybridizes with the nucleic acid sequence SEQ. ID. NO. 1 under stringent conditions, or<br>
D)	functionally equivalent fragments of the sequences of A), B) or C).<br>
The nucleic acid sequence SEQ. ID. NO. 1 represents the promoter sequence of GroES chaperonin (Pgro) from Corynebacterium glutamicum. SEQ. ID. NO. 1 corresponds to the promoter sequence of the wild type.<br>
The invention additionally relates to nucleic acids having promoter activity comprising a sequence derived from this sequence by substitution, insertion or deletion of nucleotides and having an identity of at least 90% at the nucleic acid level with the sequence SEQ. ID. NO. 1.<br>
Further natural examples of the invention for promoters of the invention can easily be found for example from various organisms whose genomic sequence is known, by identity comparisons of the nucleic acid sequences from databases with the sequence SEQ ID NO: 1 described above.<br>
Artificial promoter sequences of the invention can easily be found starting from the sequence SEQ ID NO: 1 by artificial variation and mutation, for example by substitution, insertion or deletion of nucleotides.<br>
The term "substitution" means in the description the replacement of one or more nucleotides by one or more nucleotides. "Deletion" is the replacement of a nucleotide by a direct linkage. Insertions are insertions of nucleotides into the nucleic acid sequence, with formal replacement of a direct linkage by one or more nucleotides.<br>
Identity between two nucleic acids means the identity of the nucleotides over the complete length of the nucleic acid in each case, in particular the identity calculated by comparison with the aid of the vector NTI Suite 7.1 software from Informax (USA) using the Clustal method (Higgins DG, Sharp PM. Fast and sensitive multiple sequence alignments on a microcomputer. Comput Appl. Biosci. 1989 Apr;5(2):151-1), setting the following parameters:<br><br>
Multiple alignment parameter: Gap opening penalty    10 Gap extension penalty   10 Gap separation penalty range 8 Gap separation penalty    off % identity for alignment delay 40 Residue specific gaps    off Hydrophilic residue gap   off Transition weighing    0<br>
Pairwise alignment parameter: FAST algorithm on K-tuplesize     1 Gap penalty     3 Window size     5 Number of best diagonals 5<br>
A nucleic acid sequence having an identity of at least 90% with the sequence SEQ ID NO: 1 accordingly means a nucleic acid sequence which, on comparison of its sequence with the sequence SEQ ID NO: 1, in particular in accordance with the above programming algorithm with the above parameter set, shows an identity of at least<br>
90%.<br>
Particularly preferred promoters show an identity of 91%, more preferably 92%, 93%, 94%, 95%, 96%, 97%, 98%, particularly preferably 99%, with the nucleic acid sequence SEQ. ID. NO. 1.<br>
Further natural examples of promoters can moreover easily be found starting from the nucleic acid sequences described above, in particular starting from the sequence SEQ ID NO: 1 from various organisms whose genomic sequence is unknown, by hybridization techniques in a manner known per se.<br>
A further aspect of the invention therefore relates to nucleic acids having promoter activity comprising a nucleic acid sequence which hybridizes with the nucleic acid sequence SEQ. ID. No. 1 under stringent conditions. This nucleic acid sequence comprises at least 10, more preferably more than 12,15, 30, 50 or particularly preferably more than 150, nucleotides.<br>
The hybridization takes place according to the invention under stringent conditions. Such hybridization conditions are described for example in Sambrook, J., Fritsch, E.F.,<br><br>
Maniatis, T.. in: Molecular Cloning (A Laboratory Manual), 2nd edition, Cold Spring Harbor Laboratory Press, 1989, pages 9.31-9.57 or in Current Protocols in Molecular Biology, John Wiley &amp; Sons, N.Y. (1989), 6.3.1-6.3.6:<br>
Stringent hybridization conditions mean in particular:<br>
incubation at 42째C  overnight in a solution consisting of 50% formamide, 5 x SSC<br>
(750 mM NaCI, 75 mM trisodium citrate), 50 mM sodium phosphate (pH 7.6),<br>
5 X Denhardt's solution, 10% dextran sulfate and 20 g/ml denatured, sheared salmon<br>
sperm DNA, followed by washing the filters with 0.1 x SSC at 65째C .<br>
A "functionally equivalent fragment" means for nucleic acid sequences having promoter activity fragments which have substantially the same or a higher specific promoter activity than the starting sequence.<br>
"Essentially identical" means a specific promoter activity which displays at least 50%, preferably 60%, more preferably 70%, more preferably 80%, more preferably 90%, particularly preferably 95% of the specific promoter activity of the starting sequence.<br>
"Fragments" mean partial sequences of the nucleic acids having promoter activity which are described by embodiment<br>
A), B) or C). These fragments preferably have more than 10, but more preferably more than 12, 15, 30, 50 or particularly preferably more than 150, connected nucleotides of the nucleic acid sequence SEQ. ID. NO. 1.<br>
It is particularly preferred to use the nucleic acid sequence SEQ. ID. NO. 1 as promoter, i.e. for transcription of genes.<br>
SEQ. ID. NO. 1 has been described without assignment of function in the Genbank entry AP005283. The invention therefore further relates to the novel nucleic acid sequences of the invention having promoter activity.<br>
The invention relates in particular to a nucleic acid having promoter activity, comprising<br>
A)	the nucleic acid sequence SEQ. ID. NO. 1 or<br>
B)	a sequence derived from this sequence by substitution, insertion or deletion of nucleotides and having an identity of at least 90% at the nucleic acid level with the sequence SEQ. ID. NO. 1,<br>
or<br>
C)	a nucleic acid sequence which hybridizes with the nucleic acid sequence<br>
SEQ. ID. NO. 1 under stringent conditions, or<br><br>
D) functionally equivalent fragments of the sequences of A). B) or C).<br>
with the proviso that the nucleic acid having the sequence SEQ. ID. NO. 1 is excluded.<br>
All the nucleic acids having promoter activity which are mentioned above can additionally be prepared in a manner known per se by chemical synthesis from the nucleotide building blocks such as, for example, by fragment condensation of individual overlapping complementary nucleic acid building blocks of the double helix. The chemical synthesis of oligonucleotides can take place for example in known manner by the phosphoramidite method (Voet, Voet, 2nd edition, Wiley Press New York, pp. 896-897). Addition of synthetic oligonucleotides and filling in of gaps using the Klenow fragment of DNA polymerase and ligation reactions, and general cloning methods, are described in Sambrook et al. (1989), Molecular cloning: A laboratory manual, Cold Spring Harbor Laboratory Press.<br>
The invention further relates to the use of an expression unit comprising one of the nucleic acids of the invention having promoter activity and additionally functionally linked a nucleic acid sequence which ensures the translation of ribonucleic acids for the expression of genes.<br>
An expression unit means according to the invention a nucleic acid having expression activity, i.e a nucleic acid which, in functional linkage to a nucleic acid to be expressed, or gene, regulates the expression, i.e. the transcription and the translation of this nucleic acid or of this gene.<br>
A "functional linkage" means in this connection for example the sequential arrangement of one of the expression units of the invention and of a nucleic acid sequence which is to be expressed transgenically and, where appropriate, further regulatory elements such as, for example, a terminator in such a way that each of the regulatory elements can fulfill its function in the transgenic expression of the nucleic acid sequence. A direct linkage in the chemical sense is not absolutely necessary for this. Genetic control sequences, such as, for example, enhancer sequences, can exercise their function on the target sequence also from more remote positions or even from different DNA molecules. Arrangements in which the nucleic acid sequence to be expressed transgenically is positioned behind (i.e. at the 3' end) the expression unit sequence of the invention, so that the two sequences are covalently connected together, are preferred. It is preferred in this case for the distance between the expression unit sequence and the nucleic acid sequence to be expressed transgenically to be less than 200 base pairs, particularly preferably fewer than 100 base pairs, very particularly preferably fewer than 50 base pairs.<br><br>
"Expression activity" means according to the invention the quantity of protein produced in a particular time by the expression unit, i.e. the expression rate.<br>
"Specific expression activity" means according to the invention the quantity of protein produced by the expression unit in a particular time for each expression unit.<br>
In the case of a "caused expression activity" or expression rate in relation to a gene compared with the wild type, therefore, compared with the wild type the production of a protein which was not present in this way in the wild type is caused.<br>
In the case of an "altered expression activity" or expression rate in relation to a gene compared with the wild type, therefore, compared with the wild type the quantity of protein produced in a particular time is altered.<br>
"Altered" preferably means in this connection increased or decreased.<br>
This can take place for example by increasing or reducing the specific activity of the endogenous expression unit, for example by mutating the expression unit or by stimulating or inhibiting the expression unit.<br>
The increased expression activity or expression rate can moreover be achieved for example by regulating the expression of genes in the microorganism by expression units of the invention or by expression units with increased specific expression activity, where the genes are heterologous in relation to the expression units.<br>
The regulation of the expression of genes in the microorganism by expression units of the invention or by expression units of the invention with increased specific expression activity is preferably achieved by<br>
introducing one or more expression units of the invention, where appropriate with altered specific expression activity, into the genome of the microorganism so that expression of one or more endogenous genes takes place under the control of the introduced expression units of the invention, where appropriate with altered specific expression activity, or<br>
introducing one or more genes into the genome of the microorganism so that expression of one or more of the introduced genes takes place under the control of the endogenous expression units of the invention, where appropriate with altered specific expression activity, or<br><br>
introducing one or more nucleic acid constructs comprising an expression unit of the invention, where appropriate with altered specific expression activity, and functionally linked one or more nucleic acids to be expressed, into the microorganism.<br>
The expression units of the invention comprise a nucleic acid of the invention, described above, having promoter activity and additionally functionally linked a nucleic acid sequence which ensures the translation of ribonucleic acids.<br>
This nucleic acid sequence which ensures the translation of ribonucleic acids preferably comprises the nucleic acid sequence SEQ. ID. NO. 42 as ribosome binding<br>
site.<br>
In a prefenred embodiment, the expression unit of the invention comprises:<br>
E)	the nucleic acid sequence SEQ. ID. NO. 2 or<br>
F)	a sequence derived from this sequence by substitution, insertion or deletion of<br>
nucleotides and having an identity of at least 90% at the nucleic acid level with<br>
the sequence SEQ. ID. NO. 2, or<br>
G)	a nucleic acid sequence which hybridizes with the nucleic acid sequence SEQ.<br>
ID. NO. 2 under stringent conditions, or<br>
H)  functionally equivalent fragments of the sequences of E), F) or G).<br>
The nucleic acid sequence SEQ. ID. NO. 2 represents the nucleic acid sequence of the expression unit of GroES chaperonin (Pgro) from Corynebacterium glutamicum. SEQ. ID. NO. 2 corresponds to the sequence of the expression unit of the wild type.<br>
The invention further relates to expression units comprising a sequence which is derived from this sequence by substitution, insertion or deletion of nucleotides and which have an identity of at least 90% at the nucleic acid level with the sequence SEQ. ID. NO. 2.<br>
Further natural examples of the invention for expression units of the invention can easily be found for example from various organisms whose genomic sequence is known, by identity comparisons of the nucleic acid sequences from databases with the sequence SEQ ID NO: 2 described above.<br>
Artificial sequences of the invention of the expression units can easily be found starting from the sequence SEQ ID NO: 2 by artificial variation and mutation, for example by substitution, insertion or deletion of nucleotides.<br><br>
A nucleic acid sequence having an identity of at least 90% with the sequence SEQ ID NO: 2 accordingly means a nucleic acid sequence which, on comparison of its sequence with the sequence SEQ ID NO: 2, in particular in accordance with the above programming algorithm with the above parameter set, shows an identity of at least 90%.<br>
Particularly preferred expression units show an identity of 91%, more preferably 92%, 93%, 94%, 95%, 96%, 97%, 98%, particularly preferably 99%, with the nucleic acid sequence SEQ. ID, NO. 2.<br>
Further natural examples of expression units can moreover easily be found starting from the nucleic acid sequences described above, in particular starting from the sequence SEQ ID NO: 2 from various organisms whose genomic sequence is unknown, by hybridization techniques in a manner known per se.<br>
A further aspect of the invention therefore relates to expression units comprising a nucleic acid sequence which hybridizes with the nucleic acid sequence SEQ. ID. No. 2 under stringent conditions. This nucleic acid sequence comprises at least 10, more preferably more than 12, 15, 30, 50 or particulariy preferably more than 150, nucleotides.<br>
"Hybridization" means the ability of a poly- or oligonucleotide to bind under stringent conditions to a virtually complementary sequence, while nonspecific bindings between non-complementary partners do not occur under these conditions. For this, the sequences ought preferably to be 90-100% complementary. The property of complementary sequences being able to bind specifically to one another is made use of for example in the<br>
Northern or Southern blotting technique or in primer binding in PGR or RT-PCR.<br>
The hybridization takes place according to the invention under stringent conditions. Such hybridization conditions are described for example in Sambrook, J., Fritsch, E.F., Maniatis, T., in: Molecular Cloning (A Laboratory Manual), 2nd edition, Cold Spring Harbor Laboratory Press, 1989, pages 9.31-9.57 or in Current Protocols in Molecular Biology, John Wiley &amp; Sons, N.Y. (1989), 6.3.1-6.3.6:<br>
Stringent hybridization conditions mean in particular:<br>
incubation at 42째C overnight in a solution consisting of 50% formamide, 5 x SSC<br>
(750 mM NaCI, 75 mM trisodium citrate), 50 mM sodium phosphate (pH 7.6),<br><br>
5 X Denhardt's solution, 10% dextran sulfate and 20 g/ml denatured, sheared salmon sperm DNA, followed by washing the filters with 0.1 x SSC at 65째C .<br>
The nucleotide sequences of the invention further make it possible to produce probes and primers which can be used for identifying and/or cloning homologous sequences in other cell types and microorganisms. Such probes and primers normally comprise a nucleotide sequence region which hybridizes under stringent conditions onto a least approximately 12, preferably at least approximately 25, such as, for example, approximately 40, 50 or 75 consecutive nucleotides of a sense strand of a nucleic acid sequence of the invention or of a corresponding antisense strand.<br>
Also comprised according to the invention are nucleic acid sequences which comprise so-called silent mutations or are modified in accordance with the codon usage of a specific original or host organism compared with a specifically mentioned sequence, as well as naturally occurring variants such as, for example, splice variants or allelic variants, thereof.<br>
A "functionally equivalent fragment" means for expression units fragments which have substantially the same or a higher specific expression activity than the starting sequence.<br>
"Essentially identical" means a specific expression activity which displays at least 50%, preferably 60%, more preferably 70%, more preferably 80%, more preferably 90%, particulariy preferably 95% of the specific expression activity of the starting sequence.<br>
"Fragments" mean partial sequences of the expression units which are described by embodiment E), F) or G). These fragments preferably have more than 10, but more preferably more than 12, 15, 30, 50 or particulariy preferably more than 150, connected nucleotides of the nucleic acid sequence SEQ. ID. NO. 1.<br>
It is particulariy preferred to use the nucleic acid sequence SEQ. ID. NO. 2 as<br>
expression unit, i.e. for expression of genes.<br>
SEQ. ID. NO. 2 has been described without assignment of function in the Genbank entry AP005283. The invention therefore further relates to the novel expression units of the invention.<br>
The invention relates in particular to an expression unit comprising a nucleic acid of the invention having promoter activity and additionally functionally linked a nucleic acid sequence which ensures the translation of ribonucleic acids.<br><br>
The invention particularly preferably relates to an expression unit comprising<br>
E)	the nucleic acid sequence SEQ. ID. NO. 2 or<br>
F)	a sequence derived from this sequence by substitution, insertion or deletion of<br>
nucleotides and having an identity of at least 90% at the nucleic acid level with<br>
the sequence SEQ. ID. NO. 2, or<br>
G)	a nucleic acid sequence which hybridizes with the nucleic acid sequence SEQ.<br>
ID. NO. 2 under stringent conditions, or<br>
H)  functionally equivalent fragments of the sequences of E), F) or G),<br>
with the proviso that the nucleic acid having the sequence SEQ. ID. NO. 2 is excluded.<br>
The expression units of the invention comprise one or more of the following genetic elements: a minus 10<br>
("-10") sequence; a minus 35 ("-35") sequence; a transcription sequence start, an enhancer region; and an operator region.<br>
These genetic elements are preferably specific for species of corynebacteria, especially for Corynbacterium glutamicum.<br>
All the expression units which are mentioned above can additionally be prepared in a manner known per se by chemical synthesis from the nucleotide building blocks such as, for example, by fragment condensation of individual overlapping complementary nucleic acid building blocks of the double helix. The chemical synthesis of oligonucleotides can take place for example in known manner by the phosphoramidite method (Voet, Voet, 2nd edition, Wiley Press New York, pp. 896-897). Addition of synthetic oligonucleotides and filling in of gaps using the Klenow fragment of DNA polymerase and ligation reactions, and general cloning methods, are described in Sambrook et al. (1989), Molecular cloning: A laboratory manual. Cold Spring Harbor Laboratory Press.<br>
The methods and techniques used for the inventions in this patent are known to the skilled worker trained in microbiological and recombinant DNA techniques. Methods and techniques for growing bacterial cells, inserting isolated DNA molecules into the host cell, and isolating, cloning and sequencing isolated nucleic acid molecules etc. are examples of such techniques and methods. These methods are described in many standard literature sources: Davis et al., Basic Methods In Molecular Biology (1986); J. H. Miller, Experiments in<br><br>
Molecular Genetics, Cold Spring Harbor Laboratory Press, Cold Spring Harbor, New York (1972); J.H. Miller, A Short Course in Bacterial Genetics. Cold Spring Harbor Laboratory Press. Cold Spring Harbor, New York (1992); M. Singer and P. Berg. Genes &amp; Genomes. University Science Books, Mill Valley, California (1991); J. Sambrook, E.F. Fritsch and T. Maniatis, Molecular Cloning: A Laboratory Manual, 2nd ed., Cold Spring Harbor Laboratory Press. Cold Spring Harbor, New York (1989); P.B. Kaufmann et ah. Handbook of Molecular and Cellular Methods in Biology and Medicine. CRC Press, Boca Raton, Florida (1995); Methods in Plant Molecular Biology and Biotechnology. B.R. Glickand J.E. Thompson, eds., CRC Press, Boca Raton, Florida (1993); and P.F. Smith-Keary, Molecular Genetics of Escherichia coli, The Guilford Press, New York, NY (1989).<br>
All nucleic acid molecules of the present invention are preferably in the form of an isolated nucleic acid molecule. An "isolated" nucleic acid molecule is separated from other nucleic acid molecules which are present in the natural source of the nucleic acid, and may additionally be substantially free of other cellular material or culture medium if it is prepared by recombinant techniques, or free of chemical precursors or other chemicals if it is chemically synthesized.<br>
The invention additionally includes the nucleic acid molecules complementary to the specifically described nucleotide sequences, or a section thereof.<br>
The promoters and/or expression units of the invention can for example be used particularly advantageously in improved methods for the preparation of biosynthetic products by fermentation as described hereinafter.<br>
The promoters and/or expression units of the invention have in particular the advantage that they are induced in microorganisms by stress. It is possible by suitable control of the fermentation process to control this stress induction specifically for an increase in the transcription/expression rate of desired genes. In the production of L-lysine in particular, this stress phase is reached very early, so that in this case an increased transcription/expression rate of desired genes can be achieved very eariy.<br>
The nucleic acids of the invention having promoter activity can be used to alter, i.e. to increase or reduce, or to cause the transcription rate of genes in microorganisms compared with the wild type.<br>
The expression units of the invention can be used to alter, i.e. to increase or reduce, or to cause the expression rate of genes in microorganisms compared with the wild type.<br><br>
The nucleic acids of the invention having promoter activity and the expression units of the invention can also serve to regulate and enhance the production of various biosynthetic products such as, for example, fine chemicals, proteins, in particular amino acids, microorganisms, in particular in Corynebacterium species.<br>
The invention therefore relates to a method for altering or causing the transcription rate of genes in microorganisms compared with the wild type by<br>
a)	altering the specific promoter activity in the microorganism of endogenous nucleic acids of the invention having promoter activity, which regulate the transcription of endogenous genes, compared with the wild type or<br>
b)	regulating transcription of genes in the microorganism by nucleic acids of the invention having promoter activity or by nucleic acids with altered specific promoter activity according to embodiment a), where the genes are heterologous in relation to the nucleic acids having promoter activity.<br>
According to embodiment a), the alteration or causing of the transcription rate of genes in the microorganism compared with the wild type can take place by altering, i.e. increasing or reducing, the specific promoter activity in the microorganism. This can take place for example by targeted mutation of the nucleic acid sequence of the invention having promoter activity, i.e. by targeted substitution, deletion or insertion of nucleotides. An increased or reduced promoter activity can be achieved by replacing nucleotides in the RNA polymerase holoenzyme binding sites (known to the skilled worker also as -10 region and -35 region). Additionally by reducing or enlarging the distance of the described RNA polymerase holoenzyme binding sites from one another by deleting nucleotides or inserting nucleotides. Additionally by putting binding sites (also known to the skilled worker as operators) for regulatory proteins (known to the skilled worker as repressors and activators) in the spatial vicinity of the binding sites of the RNA polymerase holoenzyme so that, after binding to a promoter sequence, these regulators diminish or enhance the binding and transcription activity of the RNA polymerase holoenzyme, or else place it under a new regulatory influence.<br>
The nucleic acid sequence SEQ. ID. NO. 53 preferably represents the ribosome binding site of the expression units of the invention, and the sequence SEQ. ID. NO. 52 represents the -10 region of the expression units of the invention. Alterations in the nucleic acid sequence in these regions lead to an alteration in the specific expression<br>
activity.<br>
The invention therefore relates to the use of the nucleic acid sequence SEQ. ID. NO.<br><br>
53 as ribosome binding site in expression units which enable genes to be expressed in bacteria of the genus Corynebactenum or Brevibactenum.<br>
The invention further relates to the use of the nucleic acid sequence SEQ. ID. NO. 52 as -10 region in expression units which enable genes to be expressed in bacteria of the genus Corynebacterium or Brevibactenum.<br>
The invention relates in particular to an expression unit which enables genes to be expressed in bacteria of the genus Corynebacterium or Brevibactenum, comprising the nucleic acid sequence SEQ. ID. NO. 53. In this case, the nucleic acid sequence SEQ. ID. NO. 53 is preferably used as ribosome binding site.<br>
The invention further relates to an expression unit which enables genes to be expressed in bacteria of the genus Corynebacterium or Brevibacterium, comprising the nucleic acid sequence SEQ. ID. NO. 52. In this case, the nucleic acid sequence SEQ. ID. NO. 52 is preferably used as -10 region.<br>
In relation to the "specific promoter activity", an increase or reduction compared with the wild type means an increase or reduction in the specific activity compared with the nucleic acid of the invention having promoter activity of the wild type, i.e. for example compared with SEQ. ID. NO. 1.<br>
According to embodiment b), the alteration or causing of the transcription rate of genes in microorganisms compared with the wild type can take place by regulating the transcription of genes in the microorganism by nucleic acids of the invention having promoter activity or by nucleic acids with altered specific promoter activity according to embodiment a), where the genes are heterologous in relation to the nucleic acids having promoter activity.<br>
This is preferably achieved by<br>
b1) introducing one or more nucleic acids of the invention having promoter activity, where appropriate with altered specific promoter activity, into the genome of the microorganism so that transcription of one or more endogenous genes takes place under the control of the introduced nucleic acid having promoter activity, where appropriate with altered specific promoter activity, or<br>
b2) introducing one or more genes into the genome of the microorganism so that transcription of one or more of the introduced genes takes place under the control of the endogenous nucleic acids of the invention having promoter<br><br>
activity, where appropriate with altered specific promoter activity, or<br>
b3) introducing one or more nucleic acid constructs comprising a nucleic acid of the invention having promoter activity, where appropriate with altered specific promoter activity, and functionally linked one or more nucleic acids to be transcribed, into the microorganism.<br>
It is thus possible to alter, i.e. to increase or to reduce, the transcription rate of an endogenous gene of the wild type by<br>
according to embodiment b1), introducing one or more nucleic acids of the invention having promoter activity, where appropriate with altered specific promoter activity, into the genome of the microorganism so that transcription of one or more endogenous genes takes place under the control of the introduced nucleic acid having promoter activity, where appropriate with altered specific promoter activity, or<br>
according to embodiment b2), introducing one or more endogenous genes into the genome of the microorganism so that transcription of one or more of the introduced endogenous genes takes place under the control of the endogenous nucleic acids of the invention having promoter activity, where appropriate with altered specific promoter activity, or<br>
according to embodiment b3), introducing one or more nucleic acid constructs comprising a nucleic acid of the invention having promoter activity, where appropriate with altered specific promoter activity, and functionally linked one or more endogenous nucleic acids to be transcribed, into the microorganism.<br>
It is thus further possible to cause the transcription rate of an exogenous gene compared with the wild type by<br>
according to embodiment b2), introducing one or more endogenous genes into the genome of the microorganism so that transcription of one or more of the introduced exogenous genes takes place under the control of the endogenous nucleic acids of the invention having promoter activity, where appropriate with altered specific promoter activity, or<br>
according to embodiment b3), introducing one or more nucleic acid constructs comprising a nucleic acid of the invention having promoter activity, where appropriate with altered specific promoter activity, and functionally linked one or more exogenous nucleic acids to be transcribed, into the microorganism.<br><br>
The insertion of genes according to embodiment b2) can moreover take place by integrating a gene into coding regions or noncoding regions. Insertion preferably takes place into noncoding regions.<br>
Insertion of nucleic acid constructs according to embodiment b3) may moreover take place chromosomally or extrachromosomally. There is preferably chromosomal insertion of the nucleic acid constructs. A "chromosomal" integration is the insertion of an exogenous DNA fragment into the chromosome of a host cell. This term is also used for homologous recombination between an exogenous DNA fragment and the appropriate region on the chromosome of the host cell.<br>
In embodiment b) there is preferably also use of nucleic s of the invention with altered specific promoter activity in accordance with embodiment a). In embodiment b), as described in embodiment a), these may be present or be prepared in the microorganism, or be introduced in isolated form into the microorganism.<br>
"Endogenous" means genetic information, such as, for example, genes, which is already present in the wild-type genome.<br>
"Exogenous" means genetic information, such as, for example, genes, which is not present in the wild-type genome.<br>
The term "genes" in relation to regulation of transcription by the nucleic acids of the invention having promoter activity preferably means nucleic acids which comprise a region to be transcribed, i.e. for example a region which regulates the translation, and a coding region and, where appropriate, further regulatory elements such as, for example, a terminator.<br>
The term "genes" in relation to the regulation, described hereinafter, of expression by the expression units of the invention preferably means nucleic acids which comprise a coding region and, where appropriate, further regulatory elements such as, for example, a terminator.<br>
A "coding region" means a nucleic acid sequence which encodes a protein.<br>
"Heterologous" in relation to nucleic acids having promoter activity and genes means that the genes used are not in the wild type transcribed under the regulation of the nucleic acids of the invention having promoter activity, but that a new functional linkage which does not occur in the wild type is produced, and the functional combination of<br><br>
nucleic acid of the invention having promoter activity and specific gene does not occur in the wild type.<br>
"Heterologous" in relation to expression units and genes means that the genes used are not in the wild type expressed under the regulation of the expression units of the invention having promoter activity, but that a new functional linkage which does not occur in the wild type is produced, and the functional combination of expression unit of the invention and specific gene does not occur in the wild type.<br>
The invention further relates in a preferred embodiment to a method for increasing or causing the transcription rate of genes in microorganisms compared with the wild type by<br>
ah) increasing the specific promoter activity in the microorganism of endogenous nucleic acids of the invention having promoter activity, which regulate the transcription of endogenous genes, compared with the wild type, or<br>
bh) regulating the transcription of genes in the microorganism by nucleic acids of the invention having promoter activity or by nucleic acids with increased specific promoter activity according to embodiment a), where the genes are heterologous in relation to the nucleic acids having promoter activity.<br>
The regulation of the transcription of genes in the microorganism by nucleic acids of the invention having promoter activity or by nucleic acids of the invention with increased specific promoter activity according to embodiment ah) is preferably achieved by<br>
bhi) introducing one or more nucleic acids of the invention having promoter activity, where appropriate with increased specific promoter activity, into the genome of the microorganism so that transcription of one or more endogenous genes takes place under the control of the introduced nucleic acid of the invention having promoter activity, where appropriate with increased specific promoter activity, or<br>
bh2) introducing one or more genes into the genome of the microorganism so that transcription of one or more of the introduced genes takes place under the control of the endogenous nucleic acids of the invention having promoter activity, where appropriate with increased specific promoter activity, or<br>
bh3) introducing one or more nucleic acid constructs comprising a nucleic acid of<br><br>
the invention having promoter activity, where appropriate with increased specific promoter activity, and functionally linked one or more nucleic acids to be transcribed, into the microorganism.<br>
The invention further relates in a preferred embodiment to a method for reducing the transcription rate of genes in microorganisms compared with the wild type by<br>
ar) reducing the specific promoter activity in the microorganism of endogenous nucleic acids of the invention having promoter activity, which regulate the transcription of the endogenous genes, compared with the wild type, or<br>
br) introducing nucleic acids with reduced specific promoter activity according to embodiment a) into the genome of the microorganism so that transcription of endogenous genes takes place under the control of the introduced nucleic acid with reduced promoter activity.<br>
The invention further relates to a method for altering or causing the expression rate of a gene in microorganisms compared with the wild type by<br>
c)	altering the specific expression activity in the microorganism of endogenous expression units of the invention, which regulate the expression of the endogenous genes, compared with the wild type, or<br>
d)	regulating the expression of genes in the microorganism by expression units of the invention or by expression units of the invention with altered specific expression activity according to embodiment c), where the genes are heterologous in relation to the expression units.<br>
According to embodiment c), the alteration or causing of the expression rate of genes in microorganisms compared with the wild type can take place by altering, i.e. increasing or reducing, the specific expression activity in the microorganism. This can take place for example by targeted mutation of the nucleic acid sequence of the invention having promoter activity, i.e. by targeted substitution, deletion or insertion of nucleotides. For example, extending the distance between Shine-Dalgarno sequence and the translation start codon usually leads to a change, a diminution or else an enhancement of the specific expression activity. An alteration of the specific expression activity can also be achieved by either shortening or extending the distance of the sequence of the Shine-Dalgarno region (ribosome binding site) from the translation start codon through deletions or insertions of nucleotides. But also by altering the sequence of the Shine-Dalgarno region in such a way that the homology to<br><br>
complementary 3' side 16S rRNA is either enhanced or else diminished.<br>
In relation to the "specific expression activity", an increase or reduction compared with the wild type means an increase or reduction of the specific activity compared with the expression unit of the invention of the wild type, i.e. for example compared with SEQ. ID. NO. 2.<br>
According to embodiment d), the alteration or causing of the expression rate of genes in microorganisms compared with the wild type can take place by regulating the expression of genes in the microorganism by expression units of the invention or by expression units of the invention with altered specific expression activity according to embodiment c), where the genes are heterologous in relation to the expression units.<br>
This is preferably achieved by<br>
d1) introducing one or more expression units of the invention, where appropriate with altered specific expression activity, into the genome of the microorganism so that expression of one or more endogenous genes takes place under the control of the introduced expression units, or<br>
d2) introducing one or more genes into the genome of the microorganism so that expression of one or more of the introduced genes takes place under the control of the endogenous expression units of the invention, where appropriate with altered specific expression activity, or<br>
d3) introducing one or more nucleic acid constructs comprising an expression unit of the invention, where appropriate with altered specific expression activity, and functionally linked one or more nucleic acids to be expressed, into the microorganism.<br>
It is thus possible to alter, i.e. to increase or to reduce, the expression rate of an endogenous gene of the wild type by<br>
according to embodiment d1) introducing one or more expression units of the invention, where appropriate with altered specific expression activity, into the genome of the microorganism so that expression of one or more endogenous genes takes place under the control of the introduced expression units, or<br>
according to embodiment d2) introducing one or more genes into the genome of the microorganism so that expression of one or more of the introduced genes takes place<br><br>
under the control of the endogenous expression units of the invention, where appropriate with altered specific expression activity, or<br>
according to embodiment d3) introducing one or more nucleic acid constructs comprising an expression unit of the invention, where appropriate with altered specific expression activity, and functionally linked one or more nucleic acids to be expressed, into the microorganism.<br>
It is thus further possible to cause the expression rate of an endogenous gene compared with the wild type by<br>
according to embodiment d2) introducing one or more exogenous genes into the genome of the microorganism so that expression of one or more of the introduced genes takes place under the control of the endogenous expression units of the invention, where appropriate with altered specific expression activity, or<br>
according to embodiment d3) introducing one or more nucleic acid constructs comprising an expression unit of the invention, where appropriate with altered specific expression activity, and functionally linked one or more exogenous nucleic acids to be expressed, into the microorganism.<br>
The insertion of genes according to embodiment 62) can moreover take place by integrating a gene into coding regions or noncoding regions. Insertion preferably takes place into noncoding regions.<br>
Insertion of nucleic acid constructs according to embodiment d3) may moreover take place chromosomally or extrachromosomaliy. There is preferably chromosomal insertion of the nucleic acid constructs.<br>
The nucleic acid constructs are also referred to hereinafter as expression cassettes.<br>
In embodiment d) there is preferably also use of expression units of the invention with altered specific expression activity in accordance with embodiment c). In embodiment d), as described in embodiment c), these may be present or be prepared in the microorganism, or be introduced in isolated form into the microorganism.<br>
The invention further relates in a prefered embodiment to a method for increasing or causing the expression rate of a gene in microorganisms compared with the wild type by<br><br>
ch) increasing the specific expression activity in the microorganism of endogenous expression units of the invention, which regulate the expression of the endogenous genes, compared with the wild type, or<br>
dh) regulating the expression of genes in the microorganism by expression units of the invention or by expression units with increased specific expression activity according to embodiment a), where the genes are heterologous in relation to the expression units.<br>
The regulation of the expression of genes in the microorganism by expression units of the invention or by expression units with increased specific expression activity according to embodiment c) is preferably achieved by<br>
dhl)  introducing one or more expression units of the Invention, where appropriate with increased specific expression activity, into the genome of the microorganism so that expression of one or more endogenous genes takes place under the control of the introduced expression units, where appropriate with increased specific expression activity, or<br>
dh2)  introducing one or more genes into the genome of the microorganism so that expression of one or more of the introduced genes takes place under the control of the endogenous expression units of the invention, where appropriate with increased specific expression activity, or<br>
dh3) introducing one or more nucleic acid constructs comprising an expression unit of the invention, where appropriate with increased specific expression activity, and functionally linked one or more nucleic acids to be expressed, into the microorganism.<br>
The invention further relates to a method for reducing the expression rate of genes in microorganisms compared with the wild type by<br>
cr)   reducing the specific expression activity in the microorganism of endogenous expression units of the invention, which regulate the expression of the endogenous genes, compared with the wild type, or<br>
dr) introducing expression units with reduced specific expression activity according to embodiment cr) into the genome of the microorganism so that expression of endogenous genes takes place under the control of the introduced expression units with reduced expression activity.<br>
I<br><br>
In a preferred embodiment of the methods of the invention described above for altering or causing the transcription rate and/or expression rate of genes in microorganisms, the genes are selected from the group of nucleic acids encoding a protein from the biosynthetic pathway of fine chemicals, where the genes may optionally comprise further regulatory elements.<br>
In a particulariy prefen^ed embodiment of the methods of the invention described above for altering or causing the transcription rate and/or expression rate of genes in microorganisms, the genes are selected from the group of nucleic acids encoding a protein from the biosynthetic pathway of proteinogenic and non-proteinogenic amino acids, nucleic acids encoding a protein from the biosynthetic pathway of nucleotides and nucleosides, nucleic acids encoding a protein from the biosynthetic pathway of organic acids, nucleic acids encoding a protein from the biosynthetic pathway of lipids and fatty acids, nucleic acids encoding a protein from the biosynthetic pathway of diols, nucleic acids encoding a protein from the biosynthetic pathway of carbohydrates, nucleic acids encoding a protein from the biosynthetic pathway of aromatic compounds, nucleic acids encoding a protein from the biosynthetic pathway of vitamins, nucleic acids encoding a protein from the biosynthetic pathway of cofactors and nucleic acids encoding a protein from the biosynthetic pathway of enzymes, where the genes may optionally comprise further regulatory elements.<br>
In a particulariy preferred embodiment, the proteins from the biosynthetic pathway of amino acids are selected from the group of<br>
aspartate kinase, aspartate-semialdehyde dehydrogenase, diaminopimelate dehydrogenase, diaminopimelate decarboxylase, dihydrodipicolinate synthetase, dihydrodipicolinate reductase, glyceraldehyde-3-phosphate dehydrogenase, 3-phosphoglycerate kinase, pyruvate carboxylase, triosephosphate isomerase, transcriptional regulator LuxR, transcriptional regulator LysR1, transcriptional regulator LysR2, malate-quinone oxidoreductase, glucose-6-phosphate deydrogenase, 6-phosphogluconate dehydrogenase, transketolase, transaldolase, homoserine 0-acetyltransferase, cystathionine gamma-synthase, cystathionine beta-lyase, serine hydroxymethyltransferase, 0-acetylhomoserine sulfhydrylase, methylenetetrahydro-folate reductase, phosphoserine aminotransferase, phosphoserine phosphatase, serine acetyltransferase, homoserine dehydrogenase, homoserine kinase, threonine synthase, threonine exporter carrier, threonine dehydratase, pyruvate oxidase, lysine exporter, biotin ligase, cysteine synthase I, cysteine synthase II, coenzyme B12-dependent methionine synthase, coenzyme B12-independent methionine synthase activity, sulfate adenylyltransferase subunit 1 and 2, phosphoadenosine-phosphosulfate reductase, ferredoxin-sulfite reductase, ferredoxin NADP reductase, 3-phosphoglycerate dehydrogenase, RXA00655 regulator, RXN2910 regulator, arginyl-<br><br>
tRNA synthetase, phosphoenolpyruvate carboxylase, threonine efflux protein, serine hydroxymethyltransferase, fructose-1,6-bisphosphatase. protein of sulfate reduction RXA077, protein of sulfate reduction RXA248, protein of sulfate reduction RXA247, protein OpcA, 1-phosphofructokinase and 6-phosphofructokinase.<br>
Preferred proteins and nucleic acids encoding these proteins of the proteins described above from the biosynthetic pathway of amino acids are respectively protein sequences and nucleic acid sequences of microbial origin, preferably from bacteria of the genus Corynebacterium or Brevibacterium, preferably from coryneform bacteria, particularly preferably from Corynebacterium glutamicum.<br>
Examples of particularly preferred protein sequences and the conresponding nucleic acid sequences encoding these proteins from the biosynthetic pathway of amino acids, the document referring thereto, and the designation thereof in the referring document are listed in Table 1:<br><br><br><br><br><br><br><br>
A further example of a particularly preferred protein sequence and the corresponding nucleic acid sequence encoding this protein from the biosynthetic pathway of amino acids is the sequence of fructose-1,6-bisphosphatase 2, also called fbr2. (SEQ. ID. NO. 51) and the con-esponding nucleic acid sequence encoding a fructose-1,6-bisphosphatase 2 (SEQ. ID. NO. 50).<br>
A further example of a particularly preferred protein sequence and the corresponding nucleic acid sequence encoding this protein from the biosynthetic pathway of amino<br><br>
acids is the sequence of the protein in sulfate reduction, also called RXA077, (SEQ. ID. NO. 4) and the corresponding nucleic acid sequence encoding a protein in sulfate reduction (SEQ. ID. NO. 3).<br>
Further particularly preferred protein sequences from the biosynthetic pathway of amino acids have in each case the amino acid sequence indicated in Table 1 for this protein, where the respective protein has, in at least one of the amino acid positions indicated in Table 2/column 2 for this amino acid sequence, a different proteinogenic amino acid than the respective amino acid indicated in Table 2/column 3 in the same line. In a further preferred embodiment, the proteins have, in at least one of the amino acid positions indicated in Table 2/column 2 for the amino acid sequence, the amino acid indicated In Table 2/column 4 in the same line. The proteins indicated in Table 2 are mutated proteins of the biosynthetic pathway of amino acids, which have particularly advantageous properties and are therefore particularly suitable for expressing the corresponding nucleic acids through the promoter of the invention and for producing amino acids. For example, the mutation T3111 leads to the feedback inhibition of ask being switched off.<br>
The corresponding nucleic acids which encode a mutated protein described above from Table 2 can be prepared by conventional methods.<br>
A suitable starting point for preparing the nucleic acid sequences encoding a mutated protein is, for example, the genome of a Corynebacterium glutamicum strain which is obtainable from the American Type Culture Collection under the designation ATCC 13032, or the nucleic acid sequences referred to in Table 1. For the back-translation of the amino acid sequence of the mutated proteins into the nucleic acid sequences encoding these proteins, it is advantageous to use the codon usage of the organism into which the nucleic acid sequence is to be introduced or in which the nucleic acid sequence is present. For example, it is advantageous to use the codon usage of Corynebacterium glutamicum for Corynebacterium glutamicum. The codon usage of the particular organism can be ascertained in a manner known per se from databases or patent applications which describe at least one protein and one gene which encodes this protein from the desired organism.<br>
The infomnation in Table 2 is to be understood in the following way:<br>
In column 1 "identification", an unambiguous designation for each sequence in relation to Table 1 is indicated.<br>
in column 2 "AA-POS", the respective number refers to the amino acid position of the<br><br>
corresponding polypeptide sequence from Table 1. A "26" in the column "AA-POS" accordingly means amino acid position 26 of the correspondingly indicated polypeptide sequence. The numbering of the position starts at +1 at the N terminus.<br>
In column 3 "AA wild type", the respective letter designates the amino acid -represented in one-letter code - at the position indicated in column 2 in the corresponding wild-type strain of the sequence from Table 1.<br>
In column 4 "AA mutant", the respective letter designates the amino acid - represented in one-letter code - at the position indicated in column 2 in the corresponding mutant strain.<br>
In column 5 "function", the physiological function of the corresponding polypeptide sequence is indicated.<br>
For mutated protein with a particular function (column 5) and a particular initial amino acid sequence (Table 1), columns 2. 3 and 4 describe at least one mutation, and a plurality of mutations for some sequences. This plurality of mutations always refers to the closest initial amino acid sequence above in each case (Table 1). The term "at least one of the amino acid positions" of a particular amino acid sequence preferably means at least one of the mutations described for this amino acid sequence in columns 2, 3 and 4.<br>
One-letter code for proteinogenic amino acids:<br>
A alanine C cysteine D aspartate E glutamate F phenylalanine G glycine H histidine 1   isoleucine K lysine L  leucine M methionine N asparagine P proline Q glutamine R arginine<br><br>
S serine T threonine<br>
V	valine<br>
W tryptophan<br>
Y	tyrosine<br><br><br><br><br><br>
In the methods of the invention described above for altering or causing the transcription rate and/or expression rate of genes in microorganisms, and the methods described hereinafter for producing genetically modified microorganisms, the genetically modified microorganisms described hereinafter and the methods described hereinafter for producing biosynthetic products, the introduction of the nucleic acids of the invention having promoter activity, of the expression units of the invention, of the genes described above and of the nucleic acid constructs or expression cassettes described above into the microorganism, in particular into coryneform bacteria, preferably takes place by the SacB method.<br>
The SacB method is known to the skilled worker and described for example in Schafer A, Tauch A, JagerW, Kalinowski J, Thierbach G, Puhler A.; Small mobilizable multi-purpose cloning vectors derived from the Escherichia coli plasmids pK18 and pK19: selection of defined deletions in the chromosome of Corynebacterium glutamicum, Gene. 1994 Jul 22;145(1):69-73 and Blomfield IC, Vaughn V, Rest RF. Eisenstein BI.; Allelic exchange in Escherichia coli using the Bacillus subtilis sacB gene and a temperature-sensitive pSC101 replicon; Mol Microbiol. 1991 Jun;5(6):1447-57.<br>
In a preferred embodiment of the methods of the invention described above, the alteration or causing of the transcription rate and/or expression rate of genes in microorganisms takes place by introducing nucleic acids of the invention having promoter activity or expression units of the invention into the microorganism.<br>
In a further preferred embodiment of the methods of the invention described above, the alteration or causing of the transcription rate and/or expression rate of genes in microorganisms takes place by introducing the nucleic acid constructs or expression cassettes described above into the microorganism.<br><br>
The invention therefore also relates to an expression cassette comprising at least one expression unit of the invention<br>
at least one further nucleic acid sequence to be expressed, i.e. a gene to be expressed and<br>
where appropriate further genetic control elements such as, for example, a terminator,<br>
where at least one expression unit and a further nucleic acid sequence to be expressed are functionally linked together, and the further nucleic acid sequence to be expressed is heterologous in relation to the expression unit.<br>
The nucleic acid sequence to be expressed is preferably at least one nucleic acid encoding a protein from the biosynthesis pathway of fine chemicals.<br>
The nucleic acid sequence to be expressed is particularly preferably selected from the group of nucleic acids encoding a protein from the biosynthetic pathway of proteinogenic and non-proteinogenic amino acids, nucleic acids encoding a protein from the biosynthetic pathway of nucleotides and nucleosides, nucleic acids encoding a protein from the biosynthetic pathway of organic acids, nucleic acids encoding a protein from the biosynthetic pathway of lipids and fatty acids, nucleic acids encoding a protein from the biosynthetic pathway of diols, nucleic acids encoding a protein from the biosynthetic pathway of carbohydrates, nucleic acids encoding a protein from the biosynthetic pathway of aromatic compounds, nucleic acids encoding a protein from the biosynthetic pathway of vitamins, nucleic acids encoding a protein from the biosynthetic pathway of cofactors and nucleic acids encoding a protein from the biosynthetic pathway of enzymes.<br>
Prefen-ed proteins from the biosynthetic pathway of amino acids are described above and examples thereof are described in Tables 1 and 2.<br>
The physical location of the expression unit relative to the gene to be expressed in the expression cassettes of the invention is chosen so that the expression unit regulates the transcription and preferably also the translation of the gene to be expressed, and thus enables one or more proteins to be produced. "Enabling production" includes in this connection a constitutive increase in the production, diminution or blocking of production under specific conditions and/or increasing the production under specific conditions. The "conditions " comprise in this connection: (1) addition of a component to the culture medium, (2) removal of a component from the culture medium, (3)<br><br>
replacement of one component in the culture medium by a second component, (4) increasing the temperature of the culture medium, (5) reducing the temperature of the culture medium, and (6) regulating the atmospheric conditions such as, for example, the oxygen or nitrogen concentration in which the culture medium is kept<br>
The invention further relates to an expression vector comprising an expression cassette of the invention described above.<br>
Vectors are well known to the skilled worker and can be found in "Cloning Vectors" (Pouwels P.H. etal., editors, Elsevier, Amsterdam-New York-Oxford, 1985). Apart from plasmids, vectors also mean all other vectors known to the skilled worker, such as, for example, phages, transposons, IS elements, phasmids, cosmids, and linear or circular DNA. These vectors may undergo autonomous replication in the host organism or chromosomal replication.<br>
Suitable and particularly preferred plasmids are those which are replicated in corynefonn bacteria. Numerous known plasmid vectors such as, for example, pZI (Menkel et al., Applied and Environmental Microbiology (1989) 64: 549-554), pEKExl (Eikmanns et al.. Gene 102: 93-98 (1991)) or pHS2-1 (Sonnen et al.. Gene 107: 69-74 (1991)) are based on the cryptic plasmids pHM1519, pBLI or pGAI. Other plasmid vectors such as, for example, pCLiKSMCS, or those based on pCG4 (US-A 4,489,160) or pNG2 (Senvold-Davis et al., FEMS Microbiology Letters 66, 119-124 (1990)) or pAGI (US-A 5,158,891), can be used in the same way.<br>
Also suitable are those plasmid vectors with the aid of which the method of gene amplification by integration into the chromosome can be used, as described for example by Reinscheid et al. (Applied and Environmental Microbiology 60,126-132 (1994)) for the duplication and amplification of the hom-thrB operon. In this method the complete gene is cloned into a plasmid vector which is able to replicate in a host (typically E. coli) but not in C. glutamicum. Examples of suitable vectors are pSUP301 (Simon et al.. Bio/Technology 1,784-791 (1983)). pK18mob or pK19mob (Schafer et al., Gene 145,69-73 (1994)). Bernard et al., Journal of Molecular Biology, 234: 534-541 (1993)), pEM1 (Schrumpf et al. 1991, Journal of Bacteriology 173: 4510-4516) or pBGS8 (Spratt et al., 1986, Gene 41: 337-342). The plasmid vector which comprises the gene to be amplified is subsequently transferred by transformation into the desired strain of C. glutamicum. Methods for transformation are described for example in Thierbach et al. (Applied Microbiology and Biotechnology 29, 356-362 (1988)), Dunican and Shivnan (Biotechnology 7, 1067-1070 (1989)) and Tauch et al. (FEMS Microbiological Letters 123,343-347 (1994)).<br><br>
The invention further relates to a genetically modified microorganism where the genetic modification leads to an alteration or causing of the transcription rate of at least one gene compared with the wild type, and is dependent on<br>
a)	altering the specific promoter activity in the microorganism of at least one endogenous nucleic acid having promoter activity according to claim 1, which regulates the transcription of at least one endogenous gene, or<br>
b)	regulating the transcription of genes in the microorganism by nucleic acids having promoter activity according to claim 1 or by nucleic acids having promoter activity according to claim 1 with altered specific promoter activity according to embodiment a), where the genes are heterologous in relation to the nucleic acids having promoter activity.<br>
As described above for the methods, the regulation of the transcription of genes in the microorganism by nucleic acids having promoter activity according to claim 1 or by nucleic acids having promoter activity according to claim 1 with altered specific promoter activity according to embodiment a), is achieved by<br>
b1) introducing one or more nucleic acids having promoter activity according to claim 1, where appropriate with altered specific promoter activity, into the genome of the microorganism so that transcription of one or more endogenous genes takes place under the control of the introduced nucleic acid having promoter activity according to claim 1, where appropriate with altered specific promoter activity, or<br>
b2) introducing one or more genes into the genome of the microorganism so that transcription of one or more of the introduced genes takes place under the control of the endogenous nucleic acids having promoter activity according to claim 1, where appropriate with altered specific promoter activity, or<br>
b3) introducing one or more nucleic acid constructs comprising a nucleic acid having promoter activity according to claim 1, where appropriate with altered specific promoter activity, and functionally linked one or more nucleic acids to be transcribed, into the microorganism.<br>
The invention further relates to a genetically modified microorganism having elevated or caused transcription rate of at least one gene compared with the wild type, where<br>
ah) the specific promoter activity in the microorganism of endogenous nucleic acids having promoter activity according to claim 1, which regulate the transcription of<br><br>
endogenous genes, is increased compared with the wild type, or<br>
bh) the transcription of genes in the microorganism is regulated by nucleic acids having promoter activity according to claim 1 or by nucleic acids having increased specific promoter activity according to embodiment ah), where the genes are heterologous in relation to the nucleic acids having promoter activity.<br>
As described above for the methods, the regulation of the transcription of genes in the microorganism by nucleic acids having promoter activity according to claim 1 or by nucleic acids having promoter activity according to claim 1 with increased specific promoter activity according to embodiment a), is achieved by<br>
bh1) introducing one or more nucleic acids having promoter activity according to claim 1, where appropriate with increased specific promoter activity, into the genome of the microorganism so that transcription of one or more endogenous genes takes place under the control of the introduced nucleic acid having promoter activity, where appropriate with increased specific promoter activity, or<br>
bh2) introducing one or more genes into the genome of the microorganism so that transcription of one or more of the introduced genes takes place under the control of the endogenous nucleic acids having promoter activity according to claim 1, where appropriate with increased specific promoter activity, or<br>
bh3) introducing one or more nucleic acid constructs comprising a nucleic acid having promoter activity according to claim 1, where appropriate with increased specific promoter activity, and functionally linked one or more nucleic acids to be transcribed, into the microorganism.<br>
The invention further relates to a genetically modified microorganism with reduced transcription rate of at least one gene compared with the wild type, where<br>
ar) the specific promoter activity in the microorganism of at least one endogenous nucleic acid having promoter activity according to claim 1, which regulates the transcription of at least one endogenous gene, is reduced compared with the wild type, or<br>
br) one or more nucleic acids having reduced promoter activity according to embodiment a) are introduced into the genome of the microorganism so that the transcription of at least one endogenous gene takes place under the control of the introduced nucleic acid having reduced promoter activity.<br><br>
The invention further relates to a genetically modified microorganism, where the genetic modification leads to an alteration or causing of the expression rate of at least one gene compared with the wild type, and is dependent on<br>
c)	altering the specific expression activity in the microorganism of at least one endogenous expression unit according to claim 2 or 3, which regulates the expression of at least one endogenous gene, compared with the wild type or<br>
d)	regulating the expression of genes in the microorganism by expression units according to claim 2 or 3 or by expression units according to claim 2 or 3 with altered specific expression activity according to embodiment a), where the genes are heterologous in relation to the expression units.<br>
As described above for the methods, the regulation of the expression of genes in the microorganism by expression units according to claim 2 or 3 or by expression units according to claim 2 or 3 with altered specific expression activity according to embodiment a) is achieved by<br>
d1) introducing one or more expression units according to claim 2 or 3, where appropriate with altered specific expression activity, into the genome of the microorganism so that expression of one or more endogenous genes takes place under the control of the introduced expression units according to claim 2 or 3, where appropriate with altered specific expression activity, or<br>
d2) introducing one or more genes into the genome of the microorganism so that expression of one or more of the introduced genes takes place under the control of the endogenous expression units according to claim 2 or 3, where appropriate with altered specific expression activity, or<br>
d3) introducing one or more nucleic acid constructs comprising an expression unit according to claim 2 or 3, where appropriate with altered specific expression activity, and functionally linked one or more nucleic acids to be expressed, into the microorganism.<br>
The invention further relates to a genetically modified microorganism with increased or caused expression rate of at least one gene compared with the wild type, where<br>
ch) the specific expression activity in the microorganism of at least one endogenous expression unit according to claim 2 or 3, which regulates the expression of the<br><br>
endogenous genes, is increased compared with the wild type, or<br>
dh) the expression of genes in the microorganism is regulated by expression units according to claim 2 or 3 or by expression units according to claim 2 or 3 with increased specific expression activity according to embodiment a), where the genes are heterologous in relation to the expression units.<br>
As described above for the methods, the regulation of the expression of genes in the microorganism by expression units according to claim 2 or 3 or by expression units according to claim 2 or 3 with increased specific expression activity according to embodiment a) is achieved by<br>
dhl) introducing one or more expression units according to claim 2 or 3, where appropriate with increased specific expression activity, into the genome of the microorganism so that expression of one or more endogenous genes takes place under the control of the introduced expression units according to claim 2 or 3, where appropriate with increased specific expression activity, or<br>
dh2) introducing one or more genes into the genome of the microorganism so that expression of one or more of the introduced genes takes place under the control of the endogenous expression units according to claim 2 or 3, where appropriate with increased specific expression activity, or<br>
dh3) introducing one or more nucleic acid constructs comprising an expression unit according to claim 2 or 3, where appropriate with increased specific expression activity, and functionally linked one or more nucleic acids to be expressed, into the microorganism.<br>
The invention further relates to a genetically modified microorganism with reduced expression rate of at least one gene compared with the wild type, where<br>
cr)the specific expression activity in the microorganism of at least one endogenous expression unit according to claim 2 or 3, which regulates the expression of at least one endogenous gene, is reduced compared with the wild type, or<br>
dr) one or more expression units according to claim 2 or 3 with reduced expression activity are introduced into the genome of the microorganism so that expression of at least one endogenous gene takes place under the control of the introduced expression unit according to claim 2 or 3 with reduced expression activity.<br><br>
The invention further relates to a genetically modified microorganism comprising an expression unit according to claim 2 or 3 and functionally linked a gene to be expressed, where the gene is heterologous in relation to the expression unit.<br>
This genetically modified microorganism particularly preferably comprises an expression cassette of the invention.<br>
The present invention particularly preferably relates to genetically modified microorganisms, in particular coryneform bacteria, which comprise a vector, in particular shuttle vector or plasmid vector, which harbors at least one recombinant nucleic acid construct as defined according to the invention.<br>
In a preferred embodiment of the genetically modified microorganisms, the genes described above are at least one nucleic acid encoding a protein from the biosynthetic pathway of fine chemicals.<br>
In a particularly prefenred embodiment of the genetically modified microorganisms, the genes described above are selected from the group of nucleic acids encoding a protein from the biosynthetic pathway of proteinogenic and non-proteinogenic amino acids, nucleic acids encoding a protein from the biosynthetic pathway of nucleotides and nucleosides, nucleic acids encoding a protein from the biosynthetic pathway of organic acids, nucleic acids encoding a protein from the biosynthetic pathway of lipids and fatty acids, nucleic acids encoding a protein from the biosynthetic pathway of diols, nucleic acids encoding a protein from the biosynthetic pathway of carbohydrates, nucleic acids encoding a protein from the biosynthetic pathway of aromatic compounds, nucleic acids encoding a protein from the biosynthetic pathway of vitamins, nucleic acids encoding a protein from the biosynthetic pathway of cofactors and nucleic acids encoding a protein from the biosynthetic pathway of enzymes, where the genes may optionally comprise further regulatory elements.<br>
Preferred proteins from the biosynthetic pathway of amino acids are selected from the group of aspartate kinase, aspartate-semialdehyde dehydrogenase, diaminopimelate dehydrogenase, diaminopimelate decarboxylase, dihydrodipicolinate synthetase, dihydrodipicolinate reductase, glyceraldehyde-3-phosphate dehydrogenase, 3-phosphoglycerate kinase, pyruvate carboxylase, triosephosphate isomerase, transcriptional regulator LuxR, transcriptional regulator LysR1, transcriptional regulator LysR2, malate-quinone oxidoreductase, glucose-6-phosphate deydrogenase, 6-phosphogluconate dehydrogenase, transketolase, transaldolase, homoserine 0-acetyltransferase, cystathionine gamma-synthase, cystathionine beta-lyase, serine hydroxymethyltransferase, O-acetylhomoserine sulfhydrylase, methylenetetrahydro-<br><br>
folate reductase, phosphoserine aminotransferase, phosphoserine phosphatase, senne acetyltransferase, homoserine dehydrogenase, homoserine kinase, threonine synthase, threonine exporter carrier, threonine dehydratase, pyruvate oxidase, lysine exporter, biotin ligase, cysteine synthase 1, cysteine synthase II, coenzyme B12-dependent methionine synthase, coenzyme B12-independent methionine synthase activity, sulfate adenylyltransferase subunit 1 and 2, phosphoadenosine-phosphosulfate reductase, ferredoxin-sulfite reductase, ferredoxin NADP reductase, 3-phosphoglycerate dehydrogenase, RXA00655 regulator, RXN2910 regulator, arginyl-tRNA synthetase, phosphoenolpyruvate carboxylase, threonine efflux protein, serine hydroxymethyltransferase, fructose-1,6-bisphosphatase, protein of sulfate reduction RXA077, protein of sulfate reduction RXA248, protein of sulfate reduction RXA247, protein OpcA, 1-phosphofructokinase and 6-phosphofructokinase.<br>
Particularly preferred examples of the proteins and genes from the biosynthetic pathway of amino acids are described above in Table 1 and Table 2.<br>
Preferred microorganisms or genetically modified microorganisms are bacteria, algae, fungi or yeasts.<br>
Particularly preferred microorganisms are, in particular, coryneform bacteria.<br>
Preferred coryneform bacteria are bacteria of the genus Corynebacterium, in particular of the species Corynebacterium glutamicum, Corynebacterium acetoglutamicum, Corynebacterium acetoacidophilum, Corynebacterium themaoaminogenes, Corynebacterium melassecola and Corynebacterium efficiens or of the genus Brevibacterium, in particular of the species Brevibacterium flavum, Brevibacterium lactofermentum and Brevibacterium divaricatum.<br>
Particularly preferred bacteria of the genera Corynebacterium and Brevibacterium are selected from the group of Corynebacterium glutamicum ATCC 13032, Corynebacterium acetoglutamicum ATCC 15806, Corynebacterium acetoacidophilum ATCC 13870, Corynebacterium themioaminogenes FERM BP-1539, Corynebacterium melassecola ATCC 17965, Corynebacterium efficiens DSM 44547, Corynebacterium efficiens DSM 44548, Corynebacterium efficiens DSM 44549, Brevibacterium flavum ATCC 14067, Brevibacterium lactofermentum ATCC 13869, Brevibacterium divaricatum ATCC 14020, Corynebacterium glutamicum KFCC10065 and Corynebacterium glutamicum ATCC21608.<br>
The abbreviation KFCC means the Korean Federation of Culture Collection, the abbreviation ATCC the American type strain culture collection and the abbreviation<br><br>
DSM the Deutsche Sammlung von Mikroorganismen.<br>
Further particularly preferred bacteria of the genera Corynebacterium and Brevibacterium are listed in Table 3:<br><br><br><br><br><br><br><br>
The abbreviations have the following meaning:<br>
ATCC: American Type Culture Collection. Rockville, MD, USA PERM: Fermentation Research Institute, Chiba, Japan<br><br>
NRRL: ARS Culture Collection, Northern Regional Research Laboratory, Peoria, IL,<br>
USA<br>
CECT: Coleccion Espanoia de Cultivos Tipo, Valencia, Spain<br>
NCIMB: National Collection of Industrial and Marine Bacteria Ltd., Aberdeen, UK<br>
CBS: Centraalbureau voor Schimmelcultures, Baarn, NL<br>
NCTC: National Collection of Type Cultures, London, UK<br>
DSMZ: Deutsche Sammlung von Mikroorganismen und Zellkulturen, Braunschweig,<br>
Gemiany<br>
Through the nucleic acids of the invention having promoter activity and the expression units of the invention it is possible with the aid of the methods of the invention described above to regulate the metabolic pathways in the genetically modified microorganisms of the invention described above to specific biosynthetic products.<br>
For this purpose, for example, metabolic pathways which lead to a specific biosynthetic product are enhanced by causing or increasing the transcription rate or expression rate of genes of this biosynthetic pathway in which the increased quantity of protein leads to an increased total activity of these proteins of the desired biosynthetic pathway and thus to an enhanced metabolic flux toward the desired biosynthetic product.<br>
In addition, metabolic pathways which diverge from a specific biosynthetic product can be diminished by reducing the transcription rate or expression rate of genes of this divergent biosynthetic pathway in which the reduced quantity of protein leads to a reduced total activity of these proteins of the unwanted biosynthetic pathway and thus additionally to an enhanced metabolic flux toward the desired biosynthetic product.<br>
The genetically modified microorganisms of the invention are able for example to produce biosynthetic products from glucose, sucrose, lactose, fructose, maltose, molasses, starch, cellulose or from glycerol and ethanol.<br>
The invention therefore relates to a method for producing biosynthetic products by cultivating genetically modified microorganisms of the invention.<br>
Depending on the desired biosynthetic product, the transcription rate or expression rate of various genes must be increased or reduced. Ordinarily, it is advantageous to alter the transcription rate or expression rate of a plurality of genes, i.e. to increase the transcription rate or expression rate of a combination of genes and/or to reduce the transcription rate or expression rate of a combination of genes.<br>
In the genetically modified microorganisms of the invention, at least one altered, i.e.<br><br>
increased or reduced, transcription rate or expression rate of a gene is attributable to a nucleic acid of the invention having promoter activity or expression unit of the invention.<br>
Further, additionally altered, i.e. additionally increased or additionally reduced, transcription rates or expression rates of further genes in the genetically modified microorganism may, but need not, derive from the nucleic acids of the invention having promoter activity or the expression units of the invention.<br>
The invention therefore further relates to a method for producing biosynthetic products by cultivating genetically modified microorganisms of the invention.<br>
Preferred biosynthetic products are fine chemicals.<br>
The term "fine chemical" is known in the art and includes compounds which are produced by an organism and are used in various branches of industry such as, for example but not restricted to, the pharmaceutical industry, the agriculture, cosmetics, food and feed industries. These compounds include organic acids such as. for example, tartaric acid, itaconic acid and diaminopimelic acid, and proteinogenic and non-proteinogenic amino acids, purine bases and pyrimidine bases, nucleosides and nucleotides (as described for example in Kuninaka, A. (1996) Nucleotides and related compounds, pp. 561-612, in Biotechnology vol. 6, Rehm et aL, editors. VCH: Weinheim and the references present therein), lipids, saturated and unsaturated fatty acids (e.g. arachidonic acid), diols (e.g. propanediol and butanediol), carbohydrates (e.g. hyaluronic acid and trehalose), aromatic compounds (e.g. aromatic amines, vanillin and indigo), vitamins and cofactors (as described in Ullmann*s Encyclopedia of Industrial Chemistry, vol. A27, "Vitamins", pp. 443-613 (1996) VCH: Weinheim and the references present therein; and Ong, A.S,. Niki, E. and Packer, L. (1995) "Nutrition, Lipids, Health and Disease" Proceedings of the UNESCO/Confederation of Scientific and Technological Associations in Malaysia and the Society for Free Radical Research - Asia, held on Sept. 1 -3, 1994 in Penang, Malaysia, AOCS Press (1995)), enzymes and all other chemicals described by Gutcho (1983) in Chemicals by Fermentation, Noyes Data Corporation, ISBN: 0818805086 and the references indicated therein. The metabolism and the uses of certain fine chemicals are explained further below.<br>
I. Amino acid metabolism and uses<br>
The amino acids comprise the fundamental structural units of all proteins and are thus essentiaf for normal cell functions. The term "amino acid" is known in the art. The proteinogenic amino acids, of which there are 20 types, serve as structural units for proteins, in which they are linked together by peptide bonds, whereas the non-<br><br>
proteinogenic amino acids (of which hundreds are known) usually do not occur in proteins (see Ullmann's Encyclopedia of Industrial Chemistry, vol. A2, pp. 57-97 VCH: Weinheim (1985)). The amino acids may be in the D or L configuration, although L-amino acids are usually the only type found in naturally occurring proteins. Biosynthetic and degradation pathways of each of the 20 proteinogenic amino acids are well characterized both in prokaryotic and in eukaryotic celts (see, for example, Stryer, L Biochemistry, 3rd edition, pp. 578-590 (1988)). The "essential" amino acids (histidine, isoleucine, leucine, lysine, methionine, phenylalanine, threonine, tryptophan and valine), so-called because they must, owing to the complexity of their biosynthesis, be taken in with the diet, are converted by simple biosynthetic pathways into the other 11 "nonessential" amino acids (alanine, arginine, asparagine, aspartate, cysteine, glutamate, glutamine, glycine, proline, serine and tyrosine). Higher animals have the ability to synthesize some of these amino acids, but the essential amino acids must be taken in with the food in order for normal protein synthesis to take place.<br>
Apart from their function in protein biosynthesis, these amino acids are chemicals of interest per se, and it has been found that many have uses in various applications in the food, feed, chemicals, cosmetics, agriculture and pharmaceutical industries. Lysine is an important amino acid not only for human nutrition but also for monogastric species such as poultry and pigs. Glutamate is used most frequently as flavor additive (monosodium glutamate, MSG) and widely in the food industry, as well as aspartate, phenylalanine, glycine and cysteine. Glycine, L-methionine and tryptophan are all used in the pharmaceutical industry. Glutamine, valine, leucine, isoleucine, histidine, arginine, proline, serine and alanine are used in the pharmaceutical industry and the cosmetics industry. Threonine, tryptophan and D-/L-methionine are widely used feed additives (Leuchtenberger, W. (1996) Amino acids - technical production and use, pp. 466-502 in Rehm et al (editors) Biotechnology vol. 6, chapter 14a, VCH: Weinheim). It has been found that these amino acids are additionally suitable as precursors for synthesizing synthetic amino acids and proteins such as N-acetylcysteine, S-carboxymethyl-L-cysteine, (S)-5-hydroxytryptophan and other substances described in Ullmann's Encyclopedia of Industrial Chemistry, vol. A2, pp, 57-97, VCH, Weinheim, 1985.<br>
The biosynthesis of these natural amino acids in organisms able to produce them, for example bacteria, has been well characterized (for a review of bacterial amino acid biosynthesis and its regulation, see Umbarger, H.E. (1978) Ann. Rev. Biochem. 47: 533 - 606). Glutamate is synthesized by reductive amination of a-ketoglutarate, an intermediate in the citric acid cycle. Glutamine, proline and arginine are each generated successively from glutamate. Biosynthesis of serine takes place in a three-step method and starts with 3-phosphoglycerate (an intemnediate of glycolysis) and yields this amino<br><br>
acid after oxidation, transamination and hydrolysis steps. Cysteine and glycine are each produced from serine, the former by condensation of homocysteine with serine, and the latter by transfer of the side-chain 棺-carbon atom to tetrahydrofolate in a reaction catalyzed by serine transhydroxymethylase. Phenylalanine and tyrosine are synthesized from the precursors of the glycolysis and pentose phosphate pathways, erythrose 4-phosphate and phosphenolpyruvate in a 9-step biosynthetic pathway which differs only in the last two steps after the synthesis of prephenate. Tryptophan is likewise produced from these two starting molecules, but its synthesis takes place in an 11-step pathway. Tyrosine can also be produced from phenylalanine in a reaction catalyzed by phenylalanine hydroxylase. Alanine, valine and leucine are each biosynthetic products of pyruvate, the final product of glycolysis. Aspartate is formed from oxalacetate, an intermediate of the citrate cycle. Asparagine, methionine, threonine and lysine are each produced by conversion of aspartate. Isoleucine is formed from threonine. Histidine is formed in a complex 9-step pathway from 5-phosphoribosyl 1-pyrophosphate, an activated sugar.<br>
Amino acids whose quantity exceeds the protein biosynthesis requirement of the cell cannot be stored and are instead degraded, so that intermediates are provided for the main metabolic pathways of the cell (for a review, see Stryer, L., Biochemistry, 3rd edition, chapter 21 "Amino Acid Degradation and the Urea Cycle"; pp, 495-516 (1988)). Although the cell is able to convert unwanted amino acids into useful metabolic intermediates, amino acid production is costly in terms of the energy, the precursor molecules and the enzymes required for their synthesis. It is therefore not surprising that amino acid biosynthesis is regulated by feedback inhibition, where the presence of a particular amino acid slows down or entirely terminates its own production (for a review of the feedback mechanism in amino acid biosynthetic pathways, see Stryer, L., Biochemistry, 3rd edition, chapter 24, "Biosynthesis of Amino Acids and Heme", pp. 575-600 (1988)). The output of a particular amino acid is therefore restricted by the quantity of this amino acid in the cell.<br>
II. Vitamins, oof actors and nutraceutical metabolism, and uses<br>
Vitamins, cofactors and nutraceuticals comprise a further group of molecules. Higher animals have lost the ability to synthesize these and therefore need to take them in, although they are easily synthesized by other organisms such as bacteria. These molecules are either biologically active molecules per se or precursors of biologically active substances which serve as electron carriers or intermediates in a number of metabolic pathways. These compounds have, besides their nutritional value, also a significant industrial value as coloring agents, antioxidants and catalysts or other processing aids. (For a review of the structure, activity and industrial applications of<br><br>
these compounds, see. for example, Ullmann's Encyclopedia of Industrial Chemistry, "Vitamins", vol. A27, pp. 443-613, VCH: Weinheim, 1996). The term "vitamin" is known in the art and includes nutrients which are required by an organism for normal functioning, but cannot be synthesized by this organism itself. The group of vitamins may include cofactors and nutraceutical compounds. The term "cofactor" includes nonprotein compounds which are necessary for the occunrence of normal enzymic activity. These compounds may be organic or inorganic; the cofactor molecules of the invention are preferably organic. The term "nutraceutical" includes food additives which promote health in plants and animals, especially in humans. Examples of such molecules are vitamins, antioxidants and likewise certain lipids (e.g. polyunsaturated fatty acids).<br>
Biosynthesis of these molecules in organisms able to produce them, such as bacteria, has been characterized in detail (Ullmann's Encyclopedia of Industrial Chemistry, 'Vitamins", vol. A27, pp. 443-613, VCH: Weinheim, 1996, Michal. G. (1999) Biochemical Pathways: An Atlas of Biochemistry and Molecular Biology, John Wiley &amp; Sons; Ong, A.S., Niki, E, and Packer, L. (1995) "Nutrition, Lipids. Health and Disease" Proceedings of the UNESCO/Confederation of Scientific and Technological Associations in Malaysia and the Society for free Radical Research - Asia, held on Sept 1-3, 1994, in Penang, Malaysia, AOCS Press, Champaign, ILX, 374 S).<br>
Thiamine (vitamin Bi) is formed by chemical coupling of pyrimidine and thiazole units. Riboflavin (vitamin B2) is synthesized from guanosine 5'-triphosphate (GTP) and ribose 5-phosphate. Riboflavin in turn is employed for the synthesis of flavin mononucleotide (FMN) and flavin-adenine dinucleotide (FAD). The family of compounds referred to jointly as "vitamin B6" (e.g. pyridoxine, pyridoxamine, pyridoxal 5-phosphate and the commercially used pyridoxine hydrochloride) are all derivatives of the common structural unit 5-hydroxy-6-methylpyridine. Pantothenate (pantothenic acid. R-(+)-N-(2,4-dihydroxy-3,3-dimethyl-1-oxobutyl)-p-alanine) can be produced either by chemical synthesis or by fermentation. The last steps in pantothenate biosynthesis consist of ATP-driven condensation of 棺-alanine and pantoic acid. The enzymes responsible for the biosynthetic steps for conversion into pantoic acid, into p-alanine and for condensation to pantothenic acid are known. The metabolically active form of pantothenate is coenzyme A, whose biosynthesis proceeds through 5 enzymatic steps. Pantothenate, pyridoxal 5-phosphate. cysteine and ATP are the precursors of coenzyme A. These enzymes catalyze not only the formation of pantothenate but also the production of (R)-pantoic acid, (R)-pantolactone. (R)-panthenol (provitamin B5), pantethein (and its derivatives) and coenzyme A.<br>
The biosynthesis of biotin from the precursor molecule pimeioyl-CoA in microorganisms has been investigated in detail, and several of the genes involved have been identified.<br><br>
It has emerged that many of the corresponding proteins are involved in Fe cluster synthesis and belong to the class of nifS proteins. Lipoic acid is derived from octanoic acid and sen/es as coenzyme in energy metabolism, where it becomes a constituent of the pyruvate dehydrogenase complex and of the a-ketoglutarate dehydrogenase complex. The folates are a group of substances which are all derived from folic acid, which in turn is derived from L-glutamic acid, 棺-aminobenzoic acid and 6-methylpterin. The biosynthesis of folic acid and its derivatives starting from the metabolic intermediates guanosine 5'4riphosphate (GTP), L-glutamic acid and p-aminobenzoic acid has been investigated in detail in certain microorganisms.<br>
Corrinoids (such as the cobalamins and in particular vitamin B12) and the porphyrins belong to a group of chemicals which are distinguished by a tetrapyrrole ring system. The biosynthesis of vitamin B12 is so complex that it has not yet been completely characterized, but most of the enzymes and substrates involved are now known. Nicotinic acid (nicotinate) and nicotinamide are pyridine derivatives, which are also referred to as "niacin". Niacin is the precursor of the important coenzymes NAD (nicotinamide-adenine dinudeotide) and NADP (nicotinamide-adenine dinucleotide phosphate) and their reduced forms.<br>
The production of these compounds on the industrial scale is based for the most part on cell-free chemical syntheses, although some of these chemicals have likewise been produced by large-scale culturing of microorganisms, such as riboflavin, vitamin Be, pantothenate and biotin. Only vitamin B12 is produced solely by fermentation, because of the complexity of its synthesis. In vitro methods require a considerable expenditure of materials and time and frequently of high costs.<br>
III. Purine, pyhmidine, nucleoside and nucleotide metabolism and uses<br>
Genes for purine and pyrimidine metabolism and their corresponding proteins are important targets for the therapy of neoplastic diseases and viral infections. The tenrt "purine" or "pyrimidine" comprises nitrogenous bases which are a constituent of nucleic acids, coenzymes and nucleotides. The term "nucleotide" comprises the fundamental structural units of nucleic acid molecules, which include a nitrogenous base, a pentose sugar (the sugar in RNA is ribose, and the sugar in DNA is D-deoxyribose) and phosphoric acid. The term "nucleoside" comprises molecules which serve as precursors of nucleotides but which, in contrast to nucleotides, have no phosphoric acid unit. It is possible by inhibiting the biosynthesis of these molecules or their mobilization for formation of nucleic acid molecules to inhibit RNA and DNA synthesis; targeted inhibition of this activity in carcinogenic cells allows the ability of tumor cells to divide and replicate to be inhibited.<br><br>
There are also nucleotides which do not form nucleic acid molecules but serve as energy stores (i.e. AMP) or as coenzymes (i.e. FAD and NAD).<br>
Several publications have described the use of these chemicals for these medical indications, where purine and/or pyrimidine metabolism is influenced (e.g. Christopherson, R.L and Lyons, S.D. (1990) "Potent inhibitors of de novo pyrimidine and purine biosynthesis as chemotherapeutic agents". Med. Res. Reviews 10: 505-548). Investigations on enzymes involved in purine and pyrimidine metabolism have concentrated on the development of novel medicaments which can be used for example as immunosuppressants or antiproliferatives (Smith, J.L. "Enzymes in Nucleotide Synthesis" Curr. Opin. Struct. Biol. 5 (1995) 752-757; Biochem. Soc. Transact. 23 (1995) 877-902). Purine and pyrimidine bases, nucleosides and nucleotides have, however, also other possible uses: as intermediates in the biosynthesis of various fine chemicals (e.g. thiamine. S-adenosylmethionine, folates or riboflavin), as energy carriers for the cell (e.g. ATP or GTP) and for chemicals themselves, are commonly used as flavor enhancers (e.g. IMP or GMP) or for many medical applicatons (see. for example, Kuninaka, A., (1996) "Nucleotides and Related Compounds" in Biotechnology, vol. 6, Rehm et al., editors VCH: Weinheim, pp. 561-612). Enzymes involved in purine, pyridine, nucleoside or nucleotide metabolism are also increasingly serving as targets for the development of chemicals for crop protection, including fungicides, herbicides and insecticides.<br>
The metabolism of these compounds in bacteria has been characterized (for reviews, see. for example, Zaikin. H. and Dixon, J.E. (1992) "De novo purine nucleotide biosynthesis" in Progress in Nucleic Acids Research and Molecular biology, vol. 42, Academic Press, pp. 259-287; and Michal. G. (1999) "Nucleotides and Nucleosides"; chapter 8 in: Biochemical Pathways: An Atlas of Biochemistry and Molecular Biology, Wiley, New York). Purine metabolism, which is the subject of intensive research, is essential for normal functioning of the cell. Impaired purine metabolism in higher animals may cause severe disorders, e.g. gout. The purine nucleotides are synthesized over a number of steps via the intermediate compound inosine 5'-phosphate (IMP) from ribose 5-phosphate, leading to production of guanosine 5'-monophosphate (GMP) or adenosine 5'-monophosphate (AMP), from which the triphosphate forms, which are used as nucleotides, can easily be prepared. These compounds are also used as energy stores, so that their degradation provides energy for many different biochemical processes in the cell. Pyrimidine biosynthesis takes place via the formation of uridine 5'-monophosphate (UMP) from ribose 5-phosphate. UMP in turn is converted into cytidine 5'-triphosphate (CTP). The deoxy forms of all nucleotides are prepared in a one-step reduction reaction from the diphosphate ribose form of the nucleotide to give<br><br>
the diphosphate deoxyribose form of the nucleotide. After phosphorylation, these molecules are able to take part in DNA synthesis.<br>
IV. Trehalose metabolism and uses<br>
Trehalose consists of two glucose molecules which are linked together via an a,a-1,1 linkage. It is commonly used in the food industry as sweetener, as additive to dried or frozen foods and in beverages. However, it is also used in the pharmaceutical industry, the cosmetics and biotechnology industry (see, for example, Nishimoto et al., (1998) US patent No. 5 759 610; Singer, M.A. and Lindquist, S. Trends Biotech. 16 (1998) 460-467; Paiva. C.LA. and Panek, A.D. Biotech Ann. Rev. 2 (1996) 293-314; and Shiosaka, M. FPU. Japan 172 (1997) 97-102). Trehalose is produced by enzymes of many microorganisms and is released in a natural way into the surrounding medium, from which it can be isolated by methods known in the art.<br>
Particularly preferred biosynthetic products are selected from the group of organic acids, proteins, nucleotides and nucleosides, both proteinogenic and non-proteinogenic amino acids, lipids and fatty acids, diols, carbohydrates, aromatic compounds, vitamins and cofactors, enzymes and proteins.<br>
Preferred organic acids are tartaric acid, itaconic acid and diaminopimelic acid.<br>
Preferred nucleosides and nucleotides are described for example in Kuninaka, A. (1996) Nucleotides and related compounds, pp. 561-612, in Biotechnology, vol. 6. Rehm et al., editors VCH: Weinheim and references present therein.<br>
Preferred biosynthetic products are additionally lipids, saturated and unsaturated fatty acids such as. for example, arachidonic acid, diols such as, for example, propanediol and butanediol, carbohydrates such as, for example, hyaluronic acid and trehalose, aromatic compounds such as, for example, aromatic amines, vanillin and indigo, vitamins and cofactors as described for example in Ullmann's Encyclopedia of Industrial Chemistry, vol. A27, "Vitamins", pp. 443-613 (1996) VCH: Weinheim and the references present therein; and Ong, A.S., Niki, E. and Packer, L. (1995) "Nutrition, Lipids, Health and Disease" Proceedings of the UNESCO/Confederation of Scientific and Technological Associations in Malaysia and the Society for Free Radical Research - Asia, held on Sept. 1-3,1994 in Penang, Malaysia, AOCS Press (1995)), enzymes, polyketides (Cane et al. (1998) Science 282: 63-68) and all other chemicals described by Gutcho (1983) in Chemicals by Fermentation. Noyes Data Corporation, ISBN: 0818805086 and the references indicated therein.<br><br>
Particularly preferred biosynthetic products are amino acids, particularly preferably essential amino acids, in particular L-glycine, L-alanine, L-leucine, L-methionine, L-phenylalanine, L-tryptophan, L-iysine, L-glutamine, L-glutamic acid. L-serine. L-proline, L-valine, L-isoleucine, L-cysteine. L-tyrosine, L-histidine, L-arginine, L-asparagine, L-aspartic acid and L-threonine, L-homoserine, especially L-lysine, L-methionine and L-threonine. An amino acid such as, for example, lysine, methionine and threonine means hereinafter both in each case the L and the D fomi of the amino acid, preferably the L form, i.e. for example L-lysine, L-methionine and L-threonine.<br>
The invention relates in particular to a method for producing lysine by cultivating genetically modified microorganisms with increased or caused expression rate of at least one gene compared with the wild type, where<br>
ch) the specific expression activity in the microorganism of at least one endogenous expression unit of the invention, which regulates the expression of the endogenous genes, is increased compared with the wild type, or<br>
dh) the expression of genes in the microorganism is regulated by expression units of the invention or by expression units with increased specific expression activity according to embodiment a), where the genes are heterologous in relation to the expression units,<br>
and where the genes are selected from the group of nucleic acids encoding an aspartate<br>
kinase, nucleic acids encoding an aspartate-semialdehyde dehydrogenase, nucleic acids encoding a diaminopimelate dehydrogenase, nucleic acids encoding a diaminopimelate decarboxylase, nucleic acids encoding a dihydrodipicolinate synthetase, nucleic acids encoding a dihydrodipicolinate reductase, nucleic acids encoding a glyceraldehyde-3-phosphate dehydrogenase, nucleic acids encoding a 3-phosphoglycerate kinase, nucleic acids encoding a pyruvate carboxylase, nucleic acids encoding a triosephosphate isomerase, nucleic acids encoding a transcriptional regulator LuxR, nucleic acids encoding a transcriptional regulator LysRI, nucleic acids encoding a transcriptional regulator LysR2, nucleic acids encoding a malate-quinone oxidoreductase, nucleic acids encoding a glucose-6-phosphate dehydrogenase, nucleic acids encoding a 6-phosphogluconate dehydrogenase, nucleic acids encoding a transketolase, nucleic acids encoding a transaldolase, nucleic acids encoding a lysine exporter, nucleic acids encoding a biotin ligase, nucleic acids encoding an arginyl-tRNA synthetase, nucleic acids encoding a phosphoenolpyruvate carboxylase, nucleic acids encoding a fructose-1,6-bisphosphatase, nucleic acids encoding a protein<br><br>
OpcA, nucleic acids encoding a 1-phosphofructokinase and nucleic acids encoding a 6-phosphofructokinase.<br>
As described above for the methods, the regulation of the expression of these genes in the microorganism by expression units of the invention or by expression units of the invention with increased specific expression activity in accordance with embodiment ch) is achieved by<br>
dhl) introducing one or more expression units of the invention, where appropriate with increased specific expression activity, into the genome of the microorganism so that expression of one or more endogenous genes takes place under the control of the introduced expression units of the invention, where appropriate with increased specific expression activity, or<br>
dh2) introducing one or more of these genes into the genome of the microorganism so that expression of one or more of the introduced genes takes place under the control of the endogenous expression units of the invention, where appropriate with increased specific expression activity, or<br>
dh3) introducing one or more nucleic acid constructs comprising an expression unit of the invention, where appropriate with increased specific expression activity, and functionally linked one or more nucleic acids to be expressed, into the microorganism.<br>
A further preferred embodiment of the method described above for preparing lysine comprises the genetically modified microorganisms, compared with the wild type, having additionally an increased activity, of at least one of the activities selected from the group of aspartate kinase activity, aspartate-semialdehyde dehydrogenase activity, diaminopimelate dehydrogenase activity, diaminopimelate decarboxylase activity, dihydrodipicolinate synthetase activity, dihydrodipicolinate reductase activity, glyceraldehyde-3-phosphate dehydrogenase activity, 3-phosphoglycerate kinase activity, pyruvate carboxylase activity, triosephosphate isomerase activity, activity of the transcriptional regulator LuxR, activity of the transcriptional regulator LysRI, activity of the transcriptional regulator LysR2, malate-quinone oxidoreductase activity, glucose-6-phosphate dehydrogenase activity, 6-phosphogluconate dehydrogenase activity, transketolase activity, transaldolase activity, lysine exporter activity, arginyl-tRNA synthetase activity, phosphoenolpyruvate carboxylase activity, fructose-1,6-bisphosphatase activity, protein OpcA activity, 1-phosphofructokinase activity, 6-phosphofructoklnase activity and biotin ligase activity.<br>
A further particularly preferred embodiment of the method described above for<br><br>
preparing lysine comprises the genetically modified microorganisms having, compared with the wild type, additionally a reduced activity, of at least one of the activities selected from the group of threonine dehydratase activity, homoserine O-acetyl-transferase activity, O-acetylhomoserine sulfhydrylase activity, phosphoenolpyruvate carboxykinase activity, pyruvate oxidase activity, homoserine kinase activity, homoserine dehydrogenase activity, threonine exporter activity, threonine efflux protein activity, asparaginase activity, aspartate decarboxylase activity and threonine synthase activity.<br>
These additional increased or reduced activities of at least one of the activities described above may, but need not, be caused by a nucleic acid of the invention having promoter activity and/or an expression unit of the invention.<br>
The invention further relates to a method for producing methionine by cultivating genetically modified microorganisms with increased or caused expression rate of at least one gene compared with the wild type, where<br>
ch) the specific expression activity in the microorganism of at least one endogenous expression unit of the invention, which regulates the expression of the endogenous genes, is increased compared with the wild type, or<br>
dh) the expression of genes in the microorganism is regulated by expression units of the invention or by expression units of the invention with increased specific expression activity according to embodiment a), where the genes are heterologous in relation to the expression units,<br>
and where the genes are selected from the group of nucleic acids encodng an aspartate kinase, nucleic acids encoding an aspartate-semialdehyde dehydrogenase, nucleic acids encoding a homoserine dehydrogenase, nucleic acids encoding a glyceraldehyde-3-phosphate dehydrogenase, nucleic acids encoding a 3-phosphoglycerate kinase, nucleic acids encoding a pyruvate carboxylase, nucleic acids encoding a triosephosphate isomerase, nucleic acids encoding a homoserine 0-acetyltransferase, nucleic acids encoding a cystathionine gamma-synthase, nucleic adds encoding a cystathionine beta-tyase, nucleic acids encoding a serine hydroxymethyltransferase, nucleic acids encoding an 0-acetylhomoserine sulfhydrylase, nucleic acids encoding a methylenetetrahydrofolate reductase, nucleic acids encoding a phosphoserine aminotransferase, nucleic acids encoding a phosphoserine phosphatase, nucleic acids encoding a serine acetyltransferase, nucleic acids encoding a cysteine synthase I, nucleic acids encoding a cysteine synthase II activity, nucleic acids encoding a coenzyme B12-dependent methionine synthase<br><br>
activity, nucleic acids encoding a coenzyme B12-independent methionine synthase activity, nucleic acids encoding a sulfate adenylyltransferase activity, nucleic acids encoding a phosphoadenosine phosphosulfate reductase activity, nucleic acids encoding a ferredoxin-sulfite reductase activity, nucleic acids encoding a ferredoxin NADPH-reductase activity, nucleic acids encoding a ferredoxin activity, nucleic acids encoding a protein of sulfate reduction RXA077, nucleic acids encoding a protein of sulfate reduction RXA248, nucleic acids encoding a protein of sulfate reduction RXA247, nucleic acids encoding an RXA0655 regulator and nucleic acids encoding an RXN2910 regulator.<br>
As described above for the methods, the regulation of the expression of these genes in the microorganism by expression units of the invention or by expression units of the invention with increased specific expression activity according to embodiment ch) is achieved by<br>
dh1) introducing one or more expression units of the invention, where appropriate with increased specific expression activity, into the genome of the microorganism so that expression of one or more of these endogenous genes tai<es place under the control of introduced expression units invention where appropriate with increased specific activity or></es>
dh2) introducing one or more genes into the genome of the microorganism so that expression of one or more of the introduced genes takes place under the control of the endogenous expression units of the invention, where appropriate with increased specific expression activity, or<br>
dh3) introducing one or more nucleic acid constructs comprising an expression unit of the invention, where appropriate with increased specific expression activity, and functionally linked one or more nucleic acids to be expressed, into the microorganism.<br>
A further preferred embodiment of the method described above for preparing methionine comprises the genetically modified microorganisms having, compared with the wild type, additionally an increased activity, of at least one of the activities selected from the group of aspartate kinase activity, aspartate-semialdehyde dehydrogenase activity, homoserine dehydrogenase activity, glyceraldehyde-3-phosphate dehydrogenase activity, 3-phosphoglycerate kinase activity, pyruvate carboxylase activity, triosephosphate isomerase activity, homoserine O-acetyltransferase activity, cystathionine gamma-synthase activity, cystathionine beta-lyase activity, serine hydroxymethyltransferase activity, 0-acetylhomoserine sulfhydrylase activity, methylenetetrahydrofolate reductase activity, phosphoserine aminotransferase activity,<br><br>
phosphoserine phosphatase activity, serine acetyltransferase activity, cysteine synthase I activity, cysteine synthase II activity, coenzyme B12-dependent methionine synthase activity, coenzyme B12-independent methionine synthase activity, sulfate adenylyltransferase activity, phosphoadenosine-phosphosulfate reductase activity, ferredoxin-sulfite reductase activity, ferredoxin NADPH-reductase activity, ferredoxin activity, activity of a protein of sulfate reduction RXA077, activity of a protein of sulfate reduction RXA248, activity of a protein of sulfate reduction RXA247, activity of an RXA655 regulator and activity of an RXN2910 regulator.<br>
A further particularly preferred embodiment of the method described above for preparing methionine comprises the genetically modified microorganisms having, compared with the wild type, additionally a reduced activity, of at least one of the activities selected from the group of homoserine kinase activity, threonine dehydratase activity, threonine synthase activity, meso-diaminopimelate D-dehydrogenase activity, phosphoenolpyruvate carboxykinase activity, pyruvate oxidase activity, dihydrodipicolinate synthase activity, dihydrodipicolinate reductase activity and diaminopicolinate decarboxylase activity.<br>
These additional increased or reduced activities of at least one of the activities described above may, but need not, be caused by a nucleic acid of the invention having promoter activity and/or an expression unit of the invention.<br>
The invention further relates to a method for preparing threonine by cultivating genetically modified microorganisms with increased or caused expression rate of at least one gene compared with the wild type, where<br>
ch) the specific expression activity in the microorganism of at least one endogenous expression unit of the invention, which regulates the expression of the endogenous genes, is increased compared with the wild type, or<br>
dh) the expression of genes in the microorganism is regulated by expression units of the invention or by expression units of the invention with increased specific expression activity according to embodiment a), where the genes are heterologous in relation to the expression units,<br>
and where the genes are selected from the group of nucleic acids encoding an aspartate kinase, nucleic acids encoding an aspartate-semialdehyde dehydrogenase, nucleic acids encoding a glyceraldehyde-3-phosphate dehydrogenase, nucleic acids encoding a 3-phosphoglycerate kinase, nucleic acids encoding a pyruvate carboxylase, nucleic acids encoding a triosephosphate isomerase, nucleic acids encoding a<br><br>
homoserine kinase, nucleic acids encoding a threonine synthase, nucleic acids encoding a threonine exporter carrier, nucleic acids encoding a glucose-6-phosphate dehydrogenase, nucleic acids encoding a transaldolase, nucleic acids encoding a transketolase, nucleic acids encoding a maiate-quinone oxidoreductase, nucleic acids encoding a 6-phosphogluconate dehydrogenase, nucleic acids encoding a lysine exporter, nucleic acids encoding a biotin ligase, nucleic acids encoding a phosphoenolpyruvate carboxylase, nucleic acids encoding a threonine efflux protein, nucleic acids encoding a fructose-I.e-bisphosphatase. nucleic acids encoding an OpcA protein, nucleic acids encoding a 1-phosphofructokinase, nucleic acids encoding a 6-phosphofructokinase, and nucleic acids encoding a homoserine dehydrogenase.<br>
As described above for the methods, the regulation of the expression of these genes in the microorganism by expression units of the invention or by expression units of the invention with increased specific expression activity according to embodiment ch) is achieved by<br>
dhl) introducing one or more expression units of the invention, v^here appropriate with increased specific expression activity, into the genome of the microorganism so that expression of one or more of these endogenous genes takes place under the control of the introduced expression units of the invention, where appropriate with increased specific expression activity, or<br>
dh2) introducing one or more of these genes into the genome of the microorganism so that expression of one or more of the introduced genes takes place under the control of the endogenous expression units of the invention, where appropriate with increased specific expression activity, or<br>
dh3) introducing one or more nucleic acid constructs comprising an expression unit of the invention, where appropriate with increased specific expression activity, and functionally linked one or more nucleic acids to be expressed, into the microorganism.<br>
A further preferred embodiment of the method described above for preparing threonine comprises the genetically modified microorganisms having, compared with the wild type, additionally an increased activity, of at least one of the activities selected from the group of aspartate kinase activity, aspartate-semialdehyde dehydrogenase activity, glyceraldehyde-3-phosphate dehydrogenase activity, 3-phosphoglycerate kinase activity, pyruvate carboxylase activity, triosephosphate isomerase activity, threonine synthase activity, activity of a threonine export carrier, transaldolase activity, transketolase activity, glucose-6-phosphate dehydrogenase activity, maiate-quinone oxidoreductase activity, homoserine kinase activity, biotin ligase activity.<br><br>
phosphoenolpyruvate carboxylase activity, threonine efflux protein activity, protein OpcA activity. 1-phosphofructokinase activity, 6-phosphofructokinase activity, fructose-1,6-bisphosphatase activity, 6-phosphogiuconate dehydrogenase and homoserine dehydrogenase activity.<br>
A further particularly preferred embodiment of the method described above for preparing threonine comprises the genetically modified microorganisms<br>
having, compared with the wild type, additionally a reduced activity, of at least one of the activities selected from the group of threonine dehydratase activity, homoserine O-acetyltransferase activity, serine hydroxymethyltransferase activity. O-acetyl-homoserine sulfhydrylase activity, meso-diaminopimelate D-dehydrogenase activity, phosphoenolpyruvate carboxykinase activity, pyruvate oxidase activity, dihydrodipicolinate synthetase activity, dihydrodipicolinate reductase activity, asparaginase activity, aspartate decarboxylase activity, lysine exporter activity, acetolactate synthase activity, ketol-acid reductoisomerase activity, branched chain aminotransferase activity, coenzyme B12-dependent methionine synthase activity, coenzyme B12-independent methionine synthase activity, dihydroxy-acid dehydratase activity and diaminopicolinate decarboxylase activity.<br>
These additional increased or reduced activities of at least one of the activities described above may, but need not, be caused by a nucleic acid of the invention having prompter activity and/or an expression unit of the invention.<br>
The term "activity" of a protein means in the case of enzymes the enzymic activity of the corresponding protein, and in the case of other proteins, for example structural or transport proteins, the physiological activity of the proteins<br>
The enzymes are ordinarily able to convert a substrate into a product or catalyze this conversion step.<br>
Accordingly, the "activity" of an enzyme means the quantity of substrate converted by the enzyme, or the quantity of product formed, in a particular time.<br>
Thus, where an activity is increased compared with the wild type, the quantity of the substrate converted by the enzyme, or the quantity of product formed. in a particular time is increased-compared with the wild type.<br>
This increase in the "activity" preferably amounts, for all activities described hereinbefore and hereinafter, to at least 5%, further preferably at least 20%, further<br><br>
preferably at least 50%, further preferably at least 100%, more preferably at least 300%, even more preferably at least 500%, especially at least 600% of the "activity of the wild type".<br>
Thus, where an activity is reduced compared with the wild type, the quantity of substrate converted by the enzyme, or the quantity of product formed, in a particular time is reduced compared with the wild type.<br>
A reduced activity preferably means the partial or substantially complete suppression or blocking, based on various cell biological mechanisms, of the functionality of this enzyme in a microorganism.<br>
A reduction in the activity comprises a quantitative decrease in an enzyme as far as substantially complete absence of the enzyme (i.e. lack of detectability of the corresponding activity or lack of immunological detectability of the enzyme). The activity in the microorganism is preferably reduced, compared with the wild type, by at least 5%, further preferably by at least 20%, further preferably by at least 50%, further preferably by 100%. "Reduction" also means in particular the complete absence of the corresponding activity.<br>
The activity of particular enzymes in genetically modified microorganisms and in the wild type, and thus the increase or reduction in the enzymic activity, can be measured by known methods such as, for example, enzyme assays.<br>
For example, a pyruvate carboxylase means a protein which exhibits the enzymatic activity of converting pyruvate into oxaloacetate.<br>
Correspondingly, a pyruvate carboxylase activity means the quantity of pyruvate converted by the pyruvate carboxlyase protein, or quantity of oxaloacetate formed, in a particular time.<br>
Thus, where a pyruvate carboxylase activity is increased compared with the wild type, the quantity of pyruvate converted by the pyruvate carboxylase protein, or the quantity of oxaloacetate formed, in a particular time is increased compared with the wild type.<br>
This increase in the pyruvate carboxylase activity is preferably at least 5%, further preferably at least 20%, further preferably at least 50%, further preferably at least 100%, more preferably at least 300%, even more preferably at least 500%, in particular at least 600%, of the pyruvate carboxylase activity of the wild type.<br>
In addition, for example a phosphoenolpyruvate carboxykinase activity means the<br><br>
enzymic activity of a phosphoenolpyruvate carboxykinase.<br>
A phosphoenolpyruvate carboxykinase means a protein which exhibits the enzymatic activity of converting oxaloacetate into phosphoenolpyruvate.<br>
Correspondingly, phosphoenolpyruvate carboxykinase activity means the quantity of oxaloacetate converted by the phosphoenolpyruvate carboxykinase protein, or quantity of phosphoenolpyruvate formed, in a particular time.<br>
When the phosphoenolpyruvate carboxykinase activity is reduced compared with the wild type, therefore, the quantity of oxaloacetate converted by the phosphoenolpyruvate carboxykinase protein, or the quantity of phosphoenolpyruvate formed, in a particular time, is reduced compared with the wild type.<br>
A reduction in phosphoenolpyruvate carboxykinase activity comprises a quantitative decrease in a phosphoenolpyruvate carboxykinase as far as a substantially complete absence of phosphoenolpyruvate carboxykinase (i.e. lack of detectability of phosphoenolpyruvate carboxykinase activity or lack of immunological detectability of phosphoenolpyruvate carboxykinase). The phosphoenolpyruvate carboxykinase activity is preferably reduced, compared with the wild type, by at least 5%, further preferably by at least 20%, further preferably by at least 50%, further preferably by 100%. In particular, "reduction" also means the complete absence of phosphoenolpyruvate carboxykinase activity.<br>
The additional increasing of activities can take place in various ways, for example by switching off inhibitory regulatory mechanisms at the expression and protein level or by increasing gene expression of nucleic acids encoding the proteins described above compared with the wild type.<br>
Increasing the gene expression of the nucleic acids encoding the proteins described above compared with the wild type can likewise take place in various ways, for example by inducing the gene by activators or, as described above, by increasing the promoter activity or increasing the expression activity or by introducing one or more gene copies into the microorganism.<br>
Increasing the gene expression of a nucleic acid encoding a protein also means according to the invention manipulation of the expression of endogenous proteins intrinsic to the microorganism.<br>
This can be achieved for example, as described above, by altering the promoter and/or<br><br>
expression unit sequences of the genes. Such an alteration, which results in an increased expression rate of the gene, can take place for example by deletion or insertion of DNA sequences.<br>
It is possible, as described above, to alter the expression of endogenous proteins by applying exogenous stimuli. This can take place through particular physiological conditions, i.e. through the application of foreign substances.<br>
The skilled worker may have recourse to further different procedures, singly or in combination, to achieve an increase in gene expression. Thus, for example, the copy number of the appropriate genes can be increased, or the promoter and regulatory region or the ribosome binding site located upstream of the structural gene can be mutated. It is additionally possible to increase the expression during fermentative production through inducible promoters. Procedures to prolong the lifespan of the mRNA likewise improve expression. Enzymic activity is likewise enhanced also by preventing degradation of enzyme protein. The genes or gene constructs may be either present in plasmids with varying copy number or integrated and amplified in the chromosome. It is also possible as an alternative to achieve overexpression of the relevant genes by altering the composition of the media and management of the culture.<br>
The skilled worker can find guidance on this inter alia in Martin et al. (Biotechnology 5, 137-146 (1987)), in Guerrero et al. (Gene 138, 35-41 (1994)), Tsuchiya and Morinaga (Bio/Technology 6, 428-430 (1988)), in Eikmanns et al. (Gene 102, 93-98 (1991)), in European patent 0472869, in US patent 4,601,893, in Schwarzer and Puhler (Biotechnology 9, 84-87 (1991), in Reinscheid et al. (Applied and Environmental Microbiology 60,126-132 (1994), in LaBarre et al. (Journal of Bacteriology 175, 1001-1007 (1993)), in the patent application WO 96/15246, in Malumbres et al. (Gene 134, 15-24 (1993)), in the Japanese published specification JP-A-10-229891, in Jensen and Hammer (Biotechnology and Bioengineering 58,191-195 (1998)), in Makrides (Microbiological Reviews 60 : 512-538 (1996) and in well-known textbooks of genetics and molecular biology.<br>
It may additionally be advantageous for the production of biosynthetic products, especially L-lysine, L-methionine and L-threonine, besides the expression or enhancement of a gene, to eliminate unwanted side reactions (Nakayama: "Breeding of Amino Acid Producing Microorganisms", in: Overproduction of Microbial Products, Krumphanzl, Sikyta, Vanek (eds.). Academic Press, London, UK, 1982).<br>
In a preferred embodiment, gene expression of a nucleic acid encoding one of the<br><br>
proteins described above is increased by introducing at least one nucleic acid encoding a corresponding protein into the microorganism. The introduction of the nucleic acid can take place chromosomally or extrachromosomally, i.e. through increasing the copy number on the chromosome and/or a copy of the gene on a plasmid which replicates in the host microorganism.<br>
The introduction of the nucleic acid, for example in the form of an expression cassette comprising the nucleic add, preferably takes place chromosomally, in particular by the SacB method described above.<br>
It is possible in principle to use for this purpose any gene which encodes one of the proteins described above.<br>
In the case of genomic nucleic acid sequences from eukaryotic sources which comprise introns, if the host microorganism is unable or cannot be made able to express the con-esponding proteins it is preferred to use nucleic acid sequences whch have already been processed, such as the corresponding cDNAs.<br>
Examples of the corresponding genes are listed in Table 1 and 2.<br>
The activities described above in microorganisms are preferably reduced by at least one of the following methods:<br>
	introduction of at least one sense ribonucleic acid sequence for inducing cosuppression or of an expression cassette ensuring expression thereof<br>
	introduction of at least one DNA- or protein-binding factor against the corresponding gene, RNA or protein or of an expression cassette ensuring expression thereof<br>
	introduction of at least one viral nucleic acid sequence which causes RNA degradation, or of an expression cassette ensuring expression thereof<br>
	introduction of at least one construct to produce a loss of function, such as, for example, generation of stop codons or a shift in the reading frame, of a gene, for example by producing an insertion, deletion, inversion or mutation in a gene. It is possible and preferred to generate knockout mutants by targeted insertion into the desired target gene through homologous recombination or introduction of sequence-specific nucleases against the target gene.<br><br>
    introduction of a promoter with reduced promoter activity or of an expression unit with reduced expression activity.<br>
The skilled worker is aware that further methods can also be employed within the scope of the present invention for reducing its activity or function. For example, the introduction of a dominant negative variant of a protein or of an expression cassette ensuring expression thereof may also be advantageous.<br>
It is moreover possible for each single one of these methods to bring about a reduction in the quantity of protein, quantity of mRNA and/or activity of a protein. A combined use is also conceivable. Further methods are known to the skilled worker and may comprise impeding or suppressing the processing of the protein, of the transport of the protein or its mRNA, inhibition of ribosome attachment, inhibition of RNA splicing, induction of an RNA-degrading enzyme and/or inhibition of translation elongation or tennination.<br>
In the method of the invention for producing biosynthetic products, the step of cultivation of the genetically modified microorganisms is preferably followed by an isolation of biosynthetic products from the microorganisms or/or from the fermentation broth. These steps may take place at the same time and/or preferably after the cultivation step.<br>
The genetically modified microorganisms of the invention can be cultivated to produce biosynthetic products, in particular L-lysine, L-methionine and L-threonine, continuously or discontinuously in a batch method (batch cultivation) or in the fed batch or repeated fed batch method. A summary of known cultivation methods is to be found in the textbook by Chmiel (Bioprozelitechnik 1. Einfuhrung in die Bioverfahrenstechnik (Gustav Fischer Verlag, Stuttgart, 1991)) or in the textbook by Storhas (Bioreaktoren und periphere Einrichtungen (Vieweg Verlag, Braunschweig/wiesbaden, 1994)).<br>
The culture medium to be used must satisfy in a suitable manner the demands of the respective strains. There are descriptions of culture media for various microorganisms in the handbook "Manual of Methods for General Bacteriology" of the American Society for Bacteriology (Washington D.C., USA, 1981).<br>
These media which can be employed according to the invention usually comprise one or more carbon sources, nitrogen sources, inorganic salts, vitamins and/or trace elements.<br>
Preferred carbon sources are sugars such as mono-, di- or polysaccharides. Examples<br><br>
of very good carbon sources are glucose, fructose, mannose, galactose, ribose, ribose, sorbose, ribulose, lactose, maltose, sucrose, raffinose, starch or cellulose. Sugars can be put in the media also via complex compounds such as molasses, or other byproducts of sugar refining. It may also be advantageous to add mixtures of various carbon sources. Other possible carbon sources are oils and fats such as, for example, soybean oil, sunflower oil, peanut oil and coconut fat, fatty acids such as, for example, palmitic acid, stearic acid or linoleic acid, alcohols such as, for example, glycerol, methanol or ethanol and organic acids such as, for example, acetic acid or lactic acid.<br>
Nitrogen sources are usually organic or inorganic nitrogen compounds or materials containing these compounds. Examples of nitrogen sources include ammonia gas or ammonium salts such as ammonium sulfate, ammonium chloride, ammonium phosphate, ammonium carbonate or ammonium nitrate, nitrates, urea, amino acids or complex nitrogen sources such as corn steep liquor, soybean flour, soybean protein, yeast extract, meat extract and others. The nitrogen sources may be used singly or as mixtures.<br>
Inorganic salt compounds which may be present in the media comprise the chloride, phosphoric or sulfate salts of calcium, magnesium, sodium, cobalt, molybdenum, potassium, manganese, zinc, copper and iron.<br>
For producing fine chemicals, especially methionine, it is possible to use as sulfur source inorganic compounds such as, for example, sulfates, sulfites, dithionites, tetrathionates, thiosulfates. sulfides, but also organic sulfur compounds such as mercaptans and thiols.<br>
It is possible to use as phosphorus source phosphoric acid, potassium dihydrogenphosphate or dipotassium hydrogenphosphate or the corresponding sodium-containing salts.<br>
Chelating agents can be added to the medium in order to keep the metal ions in solution. Particularly suitable chelating agents comprise dihydroxyphenols such as catechol or protocatechuate, or organic acids such as citric acid.<br>
The fermentation media employed according to the invention normally also comprise other growth factors such as vitamins or growth promoters, which include for example biotin, riboflavin, thiamine, folic acid, nicotinic acid, pantothenate and pyridoxine. Growth factors and salts are frequently derived from complex components of the media, such as yeast extract, molasses, corn steep liquor and the like. Suitable precursors may also be added to the culture medium. The exact composition of the<br><br>
compounds in the media depends greatly on the particular experiment and will be decided individually for each specific case. Information on optimization of media is obtainable from the textbook "Applied Microbiol. Physiology, A Practical Approach" (editors P.M. Rhodes, P.P. Stanbury, IRL Press (1997) pp. 53-73, ISBN 0 19 963577 3). Growth media can also be purchased from commercial suppliers, such as Standard 1 (Merck) or BHI (Brain heart infusion, DIFCO) and the like.<br>
All the components of the media are sterilized either by heat (20 min at 1.5 bar and 121 '째C) or by sterilizing filtration. The components can be sterilized either together or, if necessary, separately. All the components of the media may be present at the start of culturing or optionally be added continuously or batchwise.<br>
The temperature of the culture is normally between 15째C and 45째C, preferably at 25째C to 40'C and can be kept constant or changed during the experiment. The pH of the medium should be in the range from 5 to 8.5, preferably around 7.0. The pH for the culturing can be controlled during the culturing by adding basic compounds such as sodium hydroxide, potassium hydroxide, ammonia or aqueous ammonia or acidic compounds such as phosphoric acid or sulfuric acid. The development of foam can be controlled by employing antifoams such as, for example, fatty acid polyglycol esters. The stability of plasmids can be maintained by adding to the medium suitable substances with a selective action, such as, for example, antibiotics. Aerobic conditions are maintained by introducing oxygen or oxygen-containing gas mixtures such as, for example, ambient air into the culture. The temperature of the culture is normally 20째C to 45째C. The culture is continued until formation of the desired product is at a maximum. This aim is normally reached within 10 hours to 160 hours.<br>
The dry matter content of the fermentation broths obtained in this way is normally from 7.5 to 25% by weight.<br>
It is additionally advantageous also to run the fermentation with sugar limitation at least at the end. but in particular over at least 30% of the fermentation time. This means that the concentration of utilizabie sugar in the fermentation medium is kept at 0 to 3 g/l, or is reduced, during this time.<br>
Biosynthetic products are isolated from the fermentation broth and/or the microorganisms in a manner known per se in accordance with the physical/chemical properties of the required biosynthetic product and the biosynthetic by-products.<br>
The fermentation broth can then be processed further for example. Depending on the requirement, the biomass can be removed wholly or partly from the fermentation broth<br><br>
by separation methods such as, for example, centrifugation, filtration, decantation or a combination of these methods, or left completely in it.<br>
The fermentation broth can then be concentrated by known methods such as, for example, with the aid of a rotary evaporator, thin-film evaporator, falling-film evaporator, by reverse osmosis or by nanofiltration. This concentrated fermentation broth can then be worked up by freeze drying, spray drying, spray granulation or by other methods.<br>
However, it is also possible to purify the biosynthetic products, especially L-lysine, L-methionine and L-threonine, further. For this purpose, the product-containing broth is subjected, after removal of the biomass, to a chromatography using a suitable resin, with the desired product or the impurities being retained wholly or partly on the chromatography resin. These chromatographic steps can be repeated if required, using the same or different chromatography resins. The skilled worker is proficient in the selection of suitable chromatography resins and their most effective use. The purified product can be concentrated by filtration or ultrafiltration and be stored at a temperature at which the stability of the product is a maximum.<br>
The biosynthetic products may result in various forms, for example in the form of their salts or esters.<br>
The identity and purity of the isolated compound(s) can be determined by prior art techniques. These comprise high performance liquid chromatography (HPLC), spectroscopic methods, staining methods, thin-layer chromatography, NIRS, enzyme assay or microbiological assays. These analytical methods are summarized in: Patek et al. (1994) Appl. Environ. Microbiol. 60:133-140; Malakhova et al. (1996) Biotekhnologiya 11 27-32; and Schmidt et al. (1998) Bioprocess Engineer. 19:67-70. Ulmann's Encyclopedia of Industrial Chemistry (1996) vol. A27, VCH: Weinheim, pp. 89-90, pp. 521-540, pp. 540-547, pp. 559-566, 575-581 and pp. 581-587; Michal. G (1999) Biochemical Pathways: An Atlas of Biochemistry and Molecular Biology, John Wiley and Sons; Fallon, A. et al. (1987) Applications of HPLC in Biochemistry in: Laboratory Techniques in Biochemistry and Molecular Biology, vol. 17.<br>
The invention is now described in more detail by means of the following nonlimiting examples:<br><br>
Example 1<br>
Preparation of an integrated plasmid for overexpression of the pycA gene with the aid<br>
of the heterologous expression unit Pgro (SEQ. ID. 2)<br>
The following oligonucleotides were defined for amplification of the promoter of the gene which codes for chaperonin Gro ES.<br>
SEQ. ID. NO 5:<br>
gro3: 5'-gccgcagcaaacccagtag -3'<br>
SEQ. ID. NO. 6:<br>
gro11: 5'- agtcgacacgatgaatccctccatgagaaaa-3'<br>
The primers were employed in a PCR reaction with chromosomal DMA from C. glutamicum ATCC13032. It was possible with this approach to amplify a DNA fragment which corresponded to the expected size of 427 bp.<br>
The following oligonucleotides were defined for amplification of a part of the gene which codes for pyruvate carboxylase.<br>
SEQ. ID. NO. 7:<br>
pyc6:   5'-tttttctcatggagggattcatcgtgtcgactcacacatcttcaacgcttccag-3'<br>
SEQ. ID. NO. 8:<br>
pyc3:   5'- cccgcagcaacgcacgcaagaaa-3'<br>
The primers were employed in a PCR reaction with chromosomal DNA from C. glutamicum ATCC13032. It was possible with this approach to amplify a DNA fragment which corresponded to the expected size of 1344 bp.<br>
The primers gro11 and pyc6 contain an overlapping sequence and are homologous to one another at their 5' ends.<br>
The PCR products obtained above were employed as template for a further PCR in which the following primers were used.<br>
SEQ. ID. NO. 9:<br>
gro12: 5'-gcattcgcgccgctcgtaacta-3'<br>
SEQ. ID. NO. 10:<br>
pyc11: 5'- ggttcccgcgccctggtaa-3'<br><br>
It was possible with this approach to amplify a DNA fragment which corresponded to the expected size of 1107 bp. This Pgro/pycA fusion was then cloned into the vector pCR2.1 (from Invitrogen GmbH, Karlsruhe. Germany). In a further step, the Pgro/pycA fusion was cloned from the plasmid pCR2.1 (from Invitrogen GmbH, Karlsruhe, Germany) as 1125 bp EcoRI fragment into the integration vector pK19 mob sacB SEQ ID NO 11, which had previously been cut with the restriction endonuclease EcoRI. The resulting plasmid was referred to as pk19 mob sacB Pgro/pycA. The following oligonucleotides were defined for amplification of a 5' region of the pycA gene:<br>
SEQ. ID, NO. 12:<br>
pyc14: 5*- ccggcgaagtgtctgctcgcgtga-3'<br>
SEQ. ID. NO. 13:<br>
pyc15: 5'-accccgccccagtttttc-3*<br>
The primers were employed in a PCR reaction with chromosomal DNA from C. glutamicum ATCC13032. It was possible with this approach to amplify a DNA fragment which corresponded to the expected size of 487 bp. This DNA fragment was cloned into the vector pCR2.1 (from Invitrogen GmbH, Karlsruhe, Germany). A 593 bp Spel/Xbal fragment was then subsequently cloned into the vector pK19 mob sacB Psod ask, which had previously been digested with the restriction enzyme Nhel. The resulting plasmid was referred to as pK19 mob sacB Pgro pycA + US (SEQ. ID. NO. 14). Up to this step, all clonings were carried out in Escherichia coli XL-1 Blue (from Stratagene, Amsterdam, Netheriands).<br>
The transformation plasmid pK19 mob sacB Pgro pycA + US was then used to transform E. coli Mn522 (from Stratagene, Amsterdam, Netheriands) together with the plasmid pTclSAcglM as described by Liebl et al. (1989) FEMS Microbiology Letters 53:299-303. The plasmid pTclSAcglM enables DNA to be methylated according to the methylation pattern of Corynebacterium glutamicum (DE 10046870). This step enables Corynebacterium glutamicum subsequently to undergo electroporation with the integration plasmid pK19 mob sacB Pgro pycA + US. This electroporation and the subsequent selection on CM plates (10 g/1 glucose; 2.5 g/l NaCI; 2 g/l urea, 10 g/l Bacto Peptone (Difco); 10 g/l yeast extract, 22.0 g/l agar (DIfco)) with kanamycin (25 |jg/nnl) resulted in a plurality of transconjugants.<br>
To select for the second recombination event, which should lead to excision of the vector together with the pycA promoter and the pycA gene, these transconjugants were<br><br>
cultured in CM medium without kanamycin overnight and then plated out on CM plates with 10% sucrose for selection. The sacB gene present on the vector pK19 mob sacB codes for the enzyme levansucrase and leads to the synthesis of levan on growth on sucrose. Since levan is toxic for C. glutamicum, the only C. glutamicum cells able to grow on sucrose-containing medium are those which have lost the integration plasmid through the second recombination step (Jager et al,, Joumal of Bacteriology 174 (1992) 5462-5466). 100 sucrose-resistant clones were examined for their kanamycin sensitivity. It was possible to demonstrate for 15 of the tested clones not only resistance to sucrose but also sensitivity to kanamycin. A polymerase chain reaction (PCR) was used to check whether the desired replacement of the natural expression unit by the Pgro expression unit had also taken place. Chromosomal DNA was isolated from the initial strain and the 15 clones for this analysis. For this purpose, the respective clones were removed from the agar plate with a toothpick and suspended in 100 쨉l of H2O and boiled at 95째C for 10 min. 10 쨉l portions of the resulting solution were employed as template in the PCR. The primers used were oligonucleotides which are homologous to the Pgro expression unit and the pycA gene.<br>
The PCR conditions were chosen as follows: predenaturation: 5 min at 95oC denaturation 30 sec at 95째C; hybridization 30 sec at 56oC; amplification 1 min at 72oC 30 cycles; final extension 5 min at 72oC. In the mixture with the DNA of the initial strain it was not possible for a PCR product to result owing to the choice of the oligonucleotides. Only with clones in which the second recombination effected replacement of the natural expression unit (PpycA) by Pgro was a band with a size of 310 bp expected. In total, 7 of the tested 15 clones were positive.<br>
The 7 positive clones and the initial strain were then cultured in 10 ml of CM medium (10 glucose; 2.5 g/1 NaCI; 2 g/1 urea, 10 g/l Bacto Peptone (Difco); 10 g/l yeast extract) overnight The cells were then pelleted and taken up in 0.5 ml of buffer (50 mM Tris, 10 mM MgCI2, 50 mM KCI; pH 7.7). The cells were disrupted with the aid of a Ribolyzer (3 x 30 sec at level 6, from Hybaid). After a protein determination by the Bradford method, 15 쨉g portions of protein were loaded onto a 10% SDS gel, and the proteins were fractionated. An increased amount of PycA protein was detectable compared with the initial strain (figure 1). Figure 1 shows a 10% SDS gel of the Pgro pycA clones.<br>
Example 2<br>
Preparation of the vector pCLiK5MCS<br>
Firstly, the ampicillin resistance and origin of replication of the vector pBR322 were amplified by the polymerase chain reaction (PCR) using the oligonucleotide primers SEQ ID NO: 15 and SEQ ID NO: 16.<br><br>
SEQ ID NO:15<br>
5'-CCCGGGATCCGCTAGCGGCGCGCCGGCCGGCCCGGTGTGAAATACCGCACA<br>
G-3'<br>
SEQ ID NO: 16<br>
5*-TCTAGACTCGAGCGGCCGCGGCCGGCCTTTAAATTGAAGACGAAAGGGCCTC<br>
G-3'<br>
Besides the sequences complementary to pBR322, the oligonucleotide primer SEQ ID NO: 15 contains in the 5'-3' direction the cleavage sites for the restriction endonucleases Smal, BamHI, Nhel and AscI and the oligonucleotide primer SEQ ID NO: 16 contains in the 5-3' direction the cleavage sites for the restriction endonucleases Xbal, Xhol, NotI and Dral. The PCR reaction was carried out with PfuTurbo polymerase (Stratagene, La Jolla. USA) by a standard method such as Innis et al. (PCR Protocols. A Guide to Methods and Applications, Academic Press (1990)). The resulting DNA fragment with a size of approximately 2.1 kb as purified using the GFXPCR, DNA and Gel Band purification kit (Amersham Pharmacia, Freiburg) in accordance with the manufacturer's instructions. The blunt ends of the DNA fragment were iigated together using the rapid DNA ligation kit (Roche Diagnostics, Mannheim) in accordance with the manufacturer's instructions, and the ligation mixture was transformed into competent E. coli XL-1 Blue (Stratagene, La Jolla, USA) by standard methods as described in Sambrook et al. (Molecular Cloning. A Laboratory Manual, Cold Spring Harbor (1989)). Plasmid-harboring cells were selected by plating out on LB agar (Lennox, 1955, Virology, 1:190) containing ampicillin (50 쨉g/ml).<br>
The plasmid DNA of an individual clone was isolated using the Qiaprep spin miniprep kit (Qiagen, Hilden) in accordance with the manufacturer's instmctions and checked by restriction digestions. The plasmid obtained in this way is called pCLiKI.<br>
Starting from the plasmid pWLTI (Liebl et al., 1992) as template for a PCR reaction, a kanamycin resistance cassette was amplified using the oligonucleotide primers SEQ ID NO: 17 and SEQ ID NO: 18.<br>
SEQ ID NO: 17: 5'-GAGATCTAGACCCGGGGATCCGCTAGCGGGCTGCTAAAGGAAGCGGA-3'<br>
SEQ ID NO: 18 5*-GAGAGGCGCGCCGCTAGCGTGGGCGAAGAACTCCAGCA-3'<br><br>
Besides the sequences complementary to pWLT1, the oligonucleotide primer SEQ ID NO: 17 contains in the 5'-3' direction the cleavage sites for the restriction endonucleases Xbal, Smal, BamHI, Nhel and the oligonucleotide primer SEQ ID NO: 18 contains in the 5'-3' direction the cleavage sites for the restriction endonucleases AscI and Nhel. The PCR reaction was carried out with PfuTurbo polymerase (Stratagene, La Jolla, USA) by standard methods such as Innis et al. (PCR Protocols. A Guide to Methods and Applications, Academic Press (1990)). The resulting DNA fragment with a size of approximately 1.3 kb was purified using the GFXPCR, DNA and Gel Band purification kit (Amersham Pharmacia, Freiburg) in accordance with the manufacturer's instructions. The DNA fragment was cut with the restriction endonucleases Xbal and Ascl (New England Biolabs, Beverly, USA) and subsequently again purified using the GFXPCR, DNA and Gel Band purification kit (Amersham Phamnacia, Freiburg) in accordance with the manufacturer's instructions. The vector pCLiKI was likewise cut with the restriction endonucleases Xbal and Ascl and dephosphorylated with alkaline phosphatase (I (Roche Diagnostics, Mannheim)) in accordance with the manufacturer's instructions. After electrophoresis in a 0.8% agarose gel, the linearized vector (approx. 2.1 kb) was isolated using the GFXPCR, DNA and Gel Band purification kit (Amersham Pharmacia. Freiburg) in accordance with the manufacturer's instructions. This vector fragment was ligated with the cut PCR fragment using the rapid DNA ligation kit (Roche Diagnostics, Mannheim) in accordance with the manufacturer's instructions, and the ligation mixture was transformed into competent E. coli XL-IBlue (Stratagene, La Jolla, USA) by standard methods as described in Sambrook et al. (Molecular Cloning. A Laboratory Manual, Cold Spring Harbor, (1989)). Plasmid-harboring cells were selected by plating out on LB agar (Lennox, 1955, Virology, 1:190) containing ampicillin (50 pg/ml) and kanamycin (20 쨉glm\).<br>
The plasmid DNA of an individual clone was isolated using the Qiaprep spin miniprep kit (Qiagen, Hilden) in accordance with the manufacturer's instructions and checked by restriction digestions. The plasmid obtained in this way is called pCLiK2.<br>
The vector pCLiK2 was cut with the restriction endonudease Dral (New England Biolabs, Beverly, USA). After electrophoresis in a 0.8% agarose gel, a vector fragment<br>
approx. 2.3 kb in size was isolated using the GFXPCR, DNA and Gel Band purification kit (Amersham Pharmacia, Freiburg) in accordance with the manufacturer's instructions. This vector fragment was religated using the rapid DNA ligation kit (Roche Diagnostics, Mannheim) in accordance with the manufacturer's instructions, and the ligation mixture was transformed into competent E. coll XL-IBlue (Stratagene. La Jolla, USA) by standard methods as described in Sambrook et al. (Molecular Cloning. A Laboratory Manual, Cold Spring Harbor, (1989)). Plasmid-harboring cells were selected<br><br>
by plating out on LB agar (Lennox, 1955, Virology, 1:190) containing kanamycin<br>
The plasmid DNA of an individual done was isolated using the Qiaprep spin miniprep Qiagen, Hilden) in accordance with the manufacturer's instructions and checked by restriction digestions. The plasmid obtained in this way is called pCLiK3.<br>
Starting from the plasmid pWLQ2 (Liebl et aL, 1992) as template for a PCR reaction, the origin of replication pHM1519 was amplified using the oligonucleotide primers<br>
SEQ ID NO: 19 and SEQ ID NO: 20.<br>
3EQ ID NO: 19: SGAGAGGGCGGCCGCGCAAAGTCCCGCTTCGTGAA-3'<br>
S:EQ ID NO. 20: r-GAGAGGGCGGCCGCTCAAGTCGGTCAAGCCACGC-3'<br>
Besides the sequences complementary to pWLQ2, the oligonucleotide primers SEQ ID NO: 19 and SEQ ID NO: 20 contain cleavage sites for the restnction endonuclease Notl. The PCR reaction was carried out with PfuTurbo polymerase Stralagene. La Jolla, USA) by a standard method such as Innis et al. (PCR Protocols, - Gulage to Methods and Applications, Academic Press (1990)). The resulting DNA fragment with a size of approximately 2.7 kb was purified using the GFXPCR, DNA and Gel Band purification kit (Amersham Pharmacia, Freiburg) in accordance with the manufactur's instructions. The DNA fragment was cut with the restnction endonuclease Notl (New England Biolabs, Beverly, USA) and then again purified with the GF'XTM PCR, DNA and Gel Band purification kit (Amersham Pharmacia, Freiburg) in iccordance with the manufacturer's instructions. The vector pCLiK3 was likewise cut witn the restriction endonuciease Notl and dephosphorylated with alkaline phosphatase<br>
IRoche Diagnostics, Mannheim)) in accordance with the manufacturer's instructions. Alter electrophoresis in a 0.8% agarose gel, the linearized vector (approx. 2.3 kb) was isolated with the GFXPCR, DNA and Gel Band purification kit (Amersham Pharmacia. Freiburg) in accordance with the manufacturer's instructions. This vector<br>
fragment was ligated with the cut PCR fragment using the rapid DNA ligation kit (Roche Diagnostics, Mannheim) in accordance with the manufacturer's instructions, and the ligated mixture was transformed into competent E. coli XL-IBlue (Stratagene, La Jolla,USA)<br>
 standard methods as described in Sambrook et at. (Molecular Cloning. A<br><br><br>
The plasmid DNA of an individual clone was isolated using the Qiaprep spin miniprep kit (Qiagen, Hilden) in accordance with the manufacturer's instructions and checked by restriction digestions. The plasmid obtained in this way is called pCLiK5.<br>
To extend pCLiK5 by a multiple cloning site (MCS), the two synthetic, very substantially complementary oligonucleotides SEQ ID NO: 21 and SEQ ID NO: 22, which contain cleavage sites for the restriction endonucleases Swal, Xhol, AatI, Apal, Asp718, Mlul, Ndel, Spel, EcoRV, Sail, Clal, BamHI, Xbal and Smal, were combined by heating together at 95oC and slow cooling to give a double-stranded DNA fragment.<br>
SEQ ID NO: 21:<br>
5'-TCGAATTTAAATCTCGAGAGGCCTGACGTCGGGCCCGGTACCACGCGTCATAT GACTAGTTCGGACCTAGGGATATCGTCGACATCGATGCTCTTCTGCGTTAATTAAC AATTGGGATCCTCTAGACCCGGGATTTAAAT-3'<br>
SEQ ID NO: 22:<br>
5'-GATCATTTAAATCCCGGGTCTAGAGGATCCCAATTGTTAATTAACGCAGAAGAG CATCGATGTCGACGATATCCCTAGGTCCGAACTAGTCATATGACGCGTGGTACCG GGCCCGACGTCAGGCCTCTCGAGATTTAAAT-3'<br>
The vector pCLiK5 was cut with the restriction endonucleases Xhol and BamHI (New England Biolabs, Beverly, USA) and dephosphorylated with alkaline phosphatase (I (Roche Diagnostics, Mannheim)) in accordance with the manufacturer's instructions. After electrophoresis in a 0.8% agarose gel, the linearized vector (approx. 5.0 kb) was isolated with the GFXPCR, DNA and Gel Band purification kit (Amersham Pharmacia, Freiburg) in accordance with the manufacturer's instructions. This vector fragment was ligated to the synthetic double-stranded DNA fragment using the rapid DNA ligation kit (Roche Diagnostics, Mannheim) in accordance with the manufacturer's instructions, and the ligation mixture was transformed into competent E. coli XL-1 Blue (Stratagene, La Jolla, USA) by standard methods as described in Sambrook et al. (Molecular Cloning. A Laboratory Manual, Cold Spring Harbor, (1989)). Plasmid-harboring cells were selected by plating out on LB agar (Lennox, 1955, Virology, 1:190) containing kanamycin (20 쨉g/ml).<br>
The plasmid DNA of an individual clone was isolated using the Qiaprep spin miniprep kit (Qiagen, Hilden) in accordance with the manufacturer's instructions and checked by<br>
restriction digestions. The plasmid obtained in this way is called pCL.iKSMCS.<br>
Sequencing reactions were carried out as described by Sanger et al. (1977)<br><br>
Proceedings of the National Academy of Sciences USA 74:5463-5467. The sequencing reactions were fractionated and evaluated using an ABI prism 377 (PE Applied Biosystems, Weiterstadt).<br>
The resulting plasmid pCLiKSMCS is listed as SEQ ID NO: 23.<br>
Example 3<br>
Preparation of the plasmid PmetA metA<br>
Chromosomal DNA was prepared from C. glutamicum ATCC 13032 as described by Tauch et al. (1995) Plasmid 33:168-179 or Eikmanns et al. (1994) Microbiology 140:1817-1828. The metA gene including the noncoding 5' region was amplified by the polymerase chain reaction (PCR) by standard methods as described in Innis et al. (1990) PCR Protocols. A Guide to Methods and Applications, Academic Press, using the oligonucleotide primers SEQ ID NO: 24 and SEQ ID NO: 25, the chromosomal DNA as template and Pfu Turbo polymerase (from Stratagene).<br>
SEQ ID NO: 24<br>
5*-GCGCGGTACCTAGACTCACCCCAGTGCT-3'<br>
and<br>
SEQ ID NO: 25<br>
5'-CTCTACTAGTTTAGATGTAGAACTCGATGT -3'<br>
The resulting DNA fragment with a size of approx. 1.3 kb was purified using the QFXTwpcR, DNA and Gel Band purification kit (Amersham Pharmacia. Freiburg) in accordance with the manufacturer's instructions. It was then cleaved with the restriction enzymes Asp718 and Spel (Roche Diagnostics, Mannheim) and the DNA fragment was purified with the GFXPCR, DNA and Gel Band purification kit.<br>
The vector pClikSMCS SEQ ID NO: 23 was cut with the restriction enzymes Asp718 and Spel and, after fractionation by electrophoresis, a fragment 5 kb in size was isolated using the GFXPCR, DNA and Gel Band purification kit.<br>
The vector fragment was ligated together with the PCR fragment using the rapid DNA ligation kit (Roche Diagnostics, Mannheim) in accordance with the manufacturer's instructions, and the ligation mixture was transfomned into competent E. coli XL-IBlue (Stratagene, La Jolla, USA) by standard methods as described in Sambrook et al. (Molecular Cloning. A Laboratory Manual, Cold Spring Harbor, (1989)). Plasmid-harboring cells were selected by plating out on LB agar (Lennox, 1955, Virology, 1:190) containing kanamycin (20 쨉g/mi).<br><br>
The plasmid DNA was prepared by methods and using materials from Qiagen. Sequencing reactions were carried out as described by Sanger et al. (1977) Proceedings of the National Academy of Sciences USA 74:5463-5467. The sequencing reactions were fractionated and evaluated using an ABI prism 377 (PE Applied Biosystems, Weiterstadt).<br>
The resulting plasmid pCLiK5MCS PmetA metA is listed as SEQ ID NO: 26.<br>
Example 9<br>
Preparation of the plasmid pCLiK5MCS Pgro metA<br>
Chromosomal DNA was prepared from C. glutamicum ATCC 13032 as described by Tauch et al. (1995) Plasmid 33:168-179 or Eikmanns et al. (1994) Microbiology 140:1817-1828. A DNA fragment of approx. 200 base pairs from the noncoding 5' region (region of the expression unit) of the gene GroES (Pgro) was amplified by the polymerase chain reaction (PCR) by standard methods such as Innis et al. (1990) PCR Protocols. A Guide to Methods and Applications, Academic Press, using the oligonucleotide primers SEQ ID NO: 27 and SEQ ID NO: 28, the chromosomal DNA as template and Pfu Turbo polymerase (from Stratagene).<br>
SEQ ID NO: 27<br>
5'-GAGACTCGAGCGGCTTAAAGTTTGGCTGCC-3'<br>
and<br>
SEQ ID NO: 28<br>
5'-CCTGAAGGCGCGAGGGTGGGCATGATGAATCCCTCCATGAG-3'<br>
The resulting DNA fragment was purified with the GFXPCR, DNA and Gel Band purification kit (Amersham Pharmacia, Freiburg) in accordance with the manufacturer's instructions.<br>
Starting from plasmid PmetA metA as template for a PCR reaction, a part of metA was amplified using the oligonucleotide primers SEQ ID NO: 29 and SEQ ID NO: 30.<br>
SEQ ID NO: 29 5'-CCCACCCTCGCGCCTTCAG -3'<br>
and<br>
SEQ ID NO: 30<br>
5'.CTGGGTACATTGCGGCCC -3'<br><br>
The resulting DNA fragment of approximately 470 base pairs was purified with the GFXPCR, DNA and Gel Band purification kit in accordance with the manufacturer's instructions.<br>
In a further PCR reaction, the two fragments obtained above were employed together as template. Owing to the sequences which have been introduced with the oligonucleotide primer SEQ ID NO: 28 and are homologous to metA, during the PCR reaction the two fragments are attached to one another and extended to give a continuous DNA strand by the polymerase employed. The standard method was modified by adding the oligonucleotide primers used SEQ ID NO: 27 and SEQ ID NO: 30, to the reaction mixture only at the start of the second cycle.<br>
The amplified DNA fragment of approximately 675 base pairs was purified using the GFXPCR, DNA and Gel Band purification kit in accordance with the manufacturer's instructions. It was then cleaved with the restriction enzymes Xhol and Ncol (Roche Diagnostics, Mannheim) and fractionated by gel electrophoresis. Subsequently, the DNA fragment approximately 620 base pairs in size was purified from the agarose using the GFXPCR, DNA and Gel Band purification kit (Amersham Pharmacia, Freiburg). The plasmid PmetA metA SEQ ID NO: 26 was cleaved with the restriction enzymes Ncol and Spel (Roche Diagnostics, Mannheim). After fractionation by gel electrophoresis, a metA fragment approximately 0.7 kb in size was purified from the agarose using the GFXPCR, DNA and Gel Band purification kit.<br>
The vector pClikSMCS SEQ ID NO: 23 was cut with the restriction enzymes Xhol and Spel (Roche Diagnostics, Mannheim) and, after fractionation by electrophoresis, a fragment 5 kb in size was isolated using the GFXPCR. DNA and Gel Band purification kit.<br>
The vector fragment was ligated together with the PCR fragment and the metA fragment using the rapid DNA ligation kit (Roche Diagnostics, Mannheim) in accordance with the manufacturer's instructions, and the ligation mixture was transformed into competent E.coli XL-1Blue (Stratagene, La Jolla, USA) by standard methods as described in Sambrook et al. (Molecular Cloning. A Laboratory Manual, Cold Spring Harbor, (1989)). Plasmid-harboring cells were selected by plating out on LB agar (Lennox, 1955. Virology, 1:190) containing kanamycin (20 쨉g/ml).<br>
The plasmid DNA was prepared by methods and using materials from Qiagen. Sequencing reactions were carried out as described by Sanger et al. (1977) Proceedings of the National Academy of Sciences USA 74:5463-5467. The sequencing reactions were fractionated and evaluated using an ABI prism 377 (PE Applied<br><br>
Biosystems, Weiterstadt).<br>
The resulting plasmid pCLiKSMCS PGroESmetA is listed as SEQ ID NO: 31.<br>
Example 10 MetA activities<br>
The strain Corynebacterium glutamicum ATCC13032 was transformed with each of the plasmids pClikS MCS, pClik MCS PmetA metA, pCLiKSMCS PGroESmetA by the method described (Liebl, et al. (1989) FEMS Microbiology Letters 53:299-303). The transformation mixture was plated on CM plates which additionally contained 20 mg/l kanamycin in order to select for plasmid-containing cells. Resulting Kan-resistant clones were picked and isolated.<br>
C. glutamicum strains which contained one of these plasmid constructs were cultured in MMA medium (40 g/1 sucrose, 20 g/l (NH4)2SO4, 1 g/l KH2PO4.1 g/l K2HPO4, 0.25 g/l MgSO4 X 7H2O. 54 g Aces, 1 mi CaCb (10 g/l), 1 ml protocatechuate (300 mg/10 ml), 1 ml trace element solution (10 g/l FeSO4 x 7H20,10 g/l MnSO4 x H2O, 2 g/l ZnSO4 x 7H2O, 0.2 g/l CUSO4, 0.02 g/l NiCL2 x 6H2O),100쨉g/l vitamin B12, 0.3 mg/l thiamine, ImM leucine, 1 mg/l pyridoxal HCI, 1 mi biotin (100 mg/l), pH 7.0) at 30'C overnight. The cells were spun down at 4oC and then washed twice with cold Tris-HCI buffer (0.1%, pH 8.0). After renewed centrifugation, the cells were taken up in cold Tris-HCI buffer (0.1%,.pH 8.0) and adjusted to an OD600 of 160. For cell disruption, 1 ml of this cell suspension was transferred into 2 ml Ribolyser tubes from Hybaid and lysed in a Ribolyser from Hybaid with a rotation setting of 6.0 three times for 30 sec each time. The lysate was clarified by centrifugation at 15 000 rpm and 4**C in an Eppendorf centrifuge for 30 minutes, and the supernatant was transferred into a new Eppendororf cup. The protein content was determined as described by Bradford, M.M. (1976) Anal. Biochem. 72:248-254.<br>
The enzymatic activity of metA was determined as follows. The 1 ml reaction mixtures contained 100 mM potassium phosphate buffer (pH 7.5), 5 mM MgCl2, 100 쨉M acetyl-CoA, 5 mM L-homoserine, 500 쨉M DTNB (Ellman's reagent) and cell extract. The assay was started by adding the respective protein lysate and incubated at room temperature. Kinetics were then recorded at 412 nm for 10 min.<br>
The results are shown in Table la.<br>
Table la<br><br><br>
It was possible to increase MetA activity considerably by using the heterologous expression unit.<br>
Example 11<br>
Preparation of the plasmid pClikSMCS metA without start codon<br>
Chromosomal DNA was prepared from C. glutamicum ATCC 13032 as described by Tauch et al. (1995) Plasmid 33:168-179 or Eikmanns et al, (1994) Microbiology 140:1817-1828. The oligonucleotide primers SEQ ID NO 32 to SEQ ID NO 33. the chromosomal DNA as template and Pfu Turbo Polymerase (from Stratagene) were used in a polymerase chain reaction (PCR) by standard methods, as described in Innis et al. (1990) PCR Protocols. A Guide to Methods and Applications, Academic Press, to amplify the termination region of the groEL gene.<br>
SEQ ID NO 32<br>
5'- GGATCTAGAGTTCTGTGAAAAACACCGTG-3'<br>
SEQ ID NO 33<br>
5'- GCGACTAGTGCCCCACAAATAAAAAACAC-3'<br>
The resulting DNA fragments about 60 bp in size were purified using the GFXPCR, DNA and Gel Band Purification Kit (Amersham Pharmacia, Freiburg) in accordance with the manufacturer's instructions. After this, it was cleaved with the restriction enzymes Xbal and Bcnl (Roche Diagnostics, Mannheim), and the DNA fragment was purified using GFXPCR, DNA and Gel Band Purification Kit.<br>
The vector pClikSMCS SEQ ID NO: 23 was cut with the restriction enzyme Xbal, and a fragment 5 kb in size was isolated after electrophoretic fractionation with GFXPCR, DNA and Gel Band Purification Kit.<br>
The vector fragment was ligated together with the fragment 60 bp in size using the rapid DNA ligation kit (Roche Diagnostics, Mannheim) in accordance with the manufacturer's instructions, and the ligation mixture was transformed into competent E. coli XL-IBlue (Stratagene, La Jolla, USA) by standard methods as described in<br><br>
Sambrook et al. (Molecular Cloning. A Laboratory Manual, Cold Spring Harbor, (1989)). Plasmid-harboring cells were selected by plating out on LB agar (Lennox, 1955, Virology, 1:190) containing kanamycin (20 쨉g/ml).<br>
The plasmid DNA was prepared by methods and using materials from Qiagen. Sequencing reactions were carried out as described by Sanger et al. (1977) Proceedings of the National Academy of Sciences USA 74:5463-5467. The sequencing reactions were fractionated and evaluated using an ABI prism 377 (PE Applied Biosystems, Weiterstadt).<br>
The resulting plasmid pCLiKSMCS PgroES term is listed as SEQ ID NO: 34.<br>
Chromosomal DNA was prepared from C. glutamicum ATCC 13032 as described by Tauch et al. (1995) Plasmid 33:168-179 or Eikmanns et al. (1994) Microbiology 140:1817-1828. The oligonucleotide primers SEQ ID NO 35 and SEQ ID NO 36. the chromosomal DNA as template and Pfu Turbo Polymerase (from Stratagene) were used in a polymerase chain reaction (PCR) by standard methods, as described in Innis et al. (1990) PCR Protocols. A Guide to Methods and Applications, Academic Press, to amplify the metA gene without start codon.<br>
SEQ ID NO 35<br>
5'-GAGACATATGCCCACCCTCGCGCCTTCAGG-3'<br>
and<br>
SEQ ID NO 36<br>
5'-CTCTACTAGTTTAGATGTAGAACTCGATGT -3'<br>
The resulting DNA fragment about 1.2 kb in size was purified using the GFXTMPCR, DNA and Gel Band Purification Kit (Amersham Pharmacia, Freiburg) in accordance with the manufacturer's instructions. After this, it was cleaved with the restriction enzymes Xbal and Bcnl (Roche Diagnostics, Mannheim), and the DNA fragment was purified using GFXPCR, DNA and Gel Band Purification Kit.<br>
The vector pClik5MCS groEL term SEQ ID NO: 34 was cut with the restriction enzymes Ndel and Bcnl, and a fragment 5 kb in size was isolated after electrophoretic fractionation with GFXPCR, DNA and Gel Band Purification Kit.<br>
The vector fragment was ligated together with the PCR fragment using the rapid DNA ligation kit (Roche Diagnostics, Mannheim) in accordance with the manufacturer's instructions, and the ligation mixture was transformed into competent E. coli XL-IBlue (Stratagene, La Jolla, USA) by standard methods as described in Sambrook et al.<br><br>
(Molecular Cloning. A Laboratory Manual, Cold Spring Harbor, (1989)). Plasmid-harboring cells were selected by plating out on LB agar (Lennox, 1955, Virology, 1:190) containing kanamycin (20쨉g/nil).<br>
The plasmid DNA was prepared by methods and using materials from Qiagen. Sequencing reactions were carried out as described by Sanger et al. (1977) Proceedings of the National Academy of Sciences USA 74:5463-5467. The sequencing reactions were fractionated and evaluated using an ABi prism 377 (PE Applied Biosystems, Weiterstadt).<br>
The resulting plasmid pCLiKSMCS metA without start codon ist listed as SEQ ID NO: 37.<br>
Example 12<br>
Construction of Pgro expression units with different specific expression activities due to<br>
different RBS sequences and distances of metA from the start codon<br>
Chromosomal DNA was prepared from C. glutamicum ATCC 13032 as described by Tauch et al. (1995) Plasmid 33:168-179 or Eikmanns et al. (1994) Microbiology 140:1817-1828. The oligonucleotide primers SEQ ID NO 38 to SEQ ID NO 43, the chromosomal DNA as template and Pfu Turbo Polymerase (from Stratagene) were used in a polymerase chain reaction (PCR) by standard methods, as described in Innis et al. (1990) PCR Protocols. A Guide to Methods and Applications, Academic Press, to amplify the various expression units. In this case, the oligonucleotide primer 1701 (SEQ ID NO 38) was used as sense primer and was combined with the other oligonucleotide primers.<br>
SEQ ID NO 38<br>
Oligonucleotide primer 1701<br>
5'- GAGACTCGAGCGGCTTAAAGTTTGGCTGCC-3'<br>
SEQ ID NO 39<br>
Oligonucleotide primer 1828<br>
5'- ctctcatatgcAATCCCTCCATGAGAAAAATT-3'<br>
SEQ ID NO 40<br>
Oligonucleotide primer 1831 5'-ctatcatatgcgcggccgcAATCCCTCCATGAGAAAAATT-3'<br>
SEQ ID NO 41<br><br>
Oligonucleotide primer 1832<br>
5'- ctctcatatgcAAtctctccATGAGAAAAATTTTGTGTG"3'<br>
SEQ ID NO 42<br>
Oligonucleotide primer 1833<br>
5'- ctctcatatgcAAtctcctcATGAGAAAAATTTTGTGTG-3'<br>
SEQ ID NO 43<br>
Oligonucleotide primer 1834<br>
5'- ctctcatatgcAAtcccttcATGAGAAAAATTTTGTGTG-3'<br>
The resulting DNA fragments with a size of approx. 200 bp were purified using the GFXPCR, DNA and Gel Band Purification Kit (Amersham Pharmacia, Freiburg) in accordance with the manufacturer's instructions.<br>
The vector pBS KS+ (SEQ ID NO: 44) was cut with the restriction enzyme EcoRV, and a fragment 2.9 kb in size was isolated after electrophoretic fractionation with GFXPCR, DNA and Gel Band Purification Kit.<br>
The vector fragment was ligated together with the PCR fragments using the rapid DNA ligation kit (Roche Diagnostics, Mannheim) in accordance with the manufacturer's instructions, and the ligation mixture was transformed into competent E. coli XL-1Blue (Stratagene, La Jolla, USA) by standard methods as described in Sambrook et al. (Molecular Cloning. A Laboratory Manual, Cold Spring Harbor, (1989)). Plasmld-harboring cells were selected by plating out on LB agar (Lennox, 1955, Virology, 1:190) containing kanamycin (20 쨉g/ml).<br>
The plasmid DNA was prepared by methods and using materials from Qiagen. Sequencing reactions were carried out as described by Sanger et al. (1977) Proceedings of the National Academy of Sciences USA 74:5463-5467. The sequencing reactions were fractionated and evaluated using an ABI prism 377 (PE Applied Biosystems, Weiterstadt).<br>
The resulting plasmids were called pKS Pgro 1701/1828, pKS Pgro 1701/1831. pKS Pgro 1701/1832, pKS Pgro 1701/1833 and pKS Pgro 1701/1834.<br>
These plasmids were then cut with the restriction enzymes Ndel and Xhol. The resulting DNA fragments approx. 200 bp in size were isolated and purified using the GFXPCR, DNA and Gel Band Purification Kit (Amersham Pharmacia, Freiburg) in accordance with the manufacturer's instructions.<br><br>
The vector pCUK5MCS metA without start codon SEQ ID NO: 37 was cut with the restriction enzymes Ndei and Xhol. and a fragment 5 kb in size was isolated after electrophoretic fractionation with GFXPCR, DNA and Gel Band Purification Kit.<br>
The vector fragment was ligated together with the fragment 200 bp in size using the rapid DNA ligation kit (Roche Diagnostics, Mannheim) in accordance with the manufacturer's instructions, and the ligation mixture was transformed into competent E. coli XL-1Blue (Stratagene, La Jolla, USA) by standard methods as described in Sambrook et al. (Molecular Cloning. A Laboratory Manual, Cold Spring Harbor, (1989)). Piasmid-harboring cells were selected by plating out on LB agar (Lennox, 1955, Virology, 1:190) containing kanamycin (20 쨉g/ml).<br>
The plasmid DNA was prepared by methods and using materials from Qiagen. Sequencing reactions were carried out as described by Sanger et al. (1977) Proceedings of the National Academy of Sciences USA 74:5463-5467. The sequencing reactions were fractionated and evaluated using an ABI prism 377 (PE Applied Biosystems, Weiterstadt).<br>
The resulting plasmids pCLiK5MCS Pgro 1701/1828 metA, pCLiK5MCS Pgro 1701/1831 metA, pCLiKSMCS Pgro 1701/1832 metA, pCLiK5MCS Pgro 1701/1833 metA und pCLiK5MCS Pgro 1701/1834 metA are listed as SEQ ID NO: 45 to 49.<br>
The Pgro expression unit was altered through the choice of the oligonucleotides as described in figure 2.<br>
The strain Corynebacterium glutamicum ATCC13032 was transfonned with each of the plasmids pClik5 MCS, pClik MCS Pgro metA, pCUKSMCS Pgro 1701/1828 metA, pCLiK5MCS Pgro 1701/1831 metA, pCUKSMCS Pgro 1701/1832 metA , pCLiK5MCS Pgro 1701/1833 metA und pCLiKSMCS Pgro 1701/1854 by the described method (Liebl, et al. (1989) FEMS Microbiology Letters 53:299-303). The transformation mixture was plated on CM plates which additionally contained 20 mg/l kanamycin in order to select for plasmid-containing cells. Resulting Kan-resistant clones were picked and isolated.<br>
C. glutamicum strains which contained one of these plasmid constructs were cultured in MMA medium (40 g/l sucrose, 20 g/l (NH4)2SO4, 1 g/l KH2PO4, 1 g/l K2HPO4, 0.25 g/l MgSO4 X 7H2O, 54 g Aces, 1 ml CaCI2 (10 g/l), 1 ml protocatechuate (300 mg/10 ml), 1 ml trace element solution (10 g/l FeSO4 x 7H2O, 10 g/l MnSO4 x H2O, 2 g/l ZnSO4 x 7H2O, 0.2 g/l CUSO4, 0.02 g/l NiCb x 6H2O),100 쨉g/l vitamin B12, 0.3 mg/l thiamine,<br><br>
ImM leucine, 1 mg/l pyridoxal HCI, 1 ml biotin (100 mg/l). pH 7.0) at 30oC for 5 h. The cells were spun down at 4oC and then washed twice with cold Tris-HCI buffer (0.1%, pH 8.0). After renewed centrifugation, the cells were taken up in cold Tris-HCI buffer (0.1%, pH 8.0) and adjusted to an OD600 of 160. For cell disruption, 1 ml of this cell suspension was transfered into 2 ml Ribolyser tubes from Hybaid and lysed in a Ribolyser from Hybaid with a rotation setting of 6.0 three times for 30 sec each time. The lysate was clarified by centrifugation at 15 000 rpm and 4oC in an Eppendorf centrifuge for 30 minutes, and the supernatant was transferred into a new Eppendorf cup. The protein content was determined as described by Bradford, M.M. (1976) Anal. Biochem. 72:248-254.<br>
The enzymatic activity of metA was determined as follows. The 1 ml reaction mixtures contained 100 mM potassium phosphate buffer (pH 7,5), 5 mM MgCl2. 100 쨉M acetyl-CoA, 5 mM L-homoserine, 500 쨉M DTNB (Ellman's reagent) and cell extract. The assay was started by adding the respective protein lysate and incubated at room temperature. Kinetics were then recorded at 412 nm for 10 min.<br>
The results are shown in table 2a.<br><br>
Table 2a<br><br><br><br><br><br><br><br><br><br>
We claim<br>
1.	The use of a nucleic acid having promoter activity, comprising<br>
A)	the nucleic acid sequence SEQ. ID. NO. 1 or<br>
B)	a sequence derived from this sequence by substitution, insertion or deletion of nucleotides and having an identity of at least 90% at the nucleic add level with the sequence SEQ. ID. NO. 1,<br>
or<br>
C)	a nucleic acid sequence which hybridizes with the nucleic acid sequence SEQ. ID. NO. 1 under stringent conditions, or<br>
D)	functionally equivalent fragments of the sequences of A), B) or C)<br>
for the transcription of genes.<br>
2.	The use of an expression unit comprising a nucleic acid having promoter activity according to claim 1, and additionally functionally linked a nucleic acid sequence which ensures the translation of ribonucleic acids, for the expression of genes.<br>
3.	The use according to claim 2, wherein the expression unit comprises:<br>
E)	the nucleic acid sequence SEQ. ID. NO. 2 or<br>
F)	a sequence derived from this sequence by substitution, insertion or<br>
deletion of nucleotides and having an identity of at least 90% at the nucleic<br>
acid level with the sequence SEQ. ID. NO. 2, or<br>
G)	a nucleic acid sequence which hybridizes with the nucleic acid sequence<br>
SEQ. ID. NO. 2 under stringent conditions, or<br>
H)  functionally equivalent fragments of the sequences of E), F) or G).<br>
4.	The use according to claim 3, wherein the expression unit consists of a nucleic acid of sequence SEQ. ID. NO. 2.<br>
5.	A nucleic acid having promoter activity, comprising A)   the nucleic acid sequence SEQ. ID. NO. 1 or<br><br>
B)	a sequence derived from this sequence by substitution, insertion or<br>
deletion of nucleotides and having an identity of at least 90% at the nucleic<br>
acid level with the sequence SEQ. ID. NO. 1,<br>
or<br>
C)	a nucleic acid sequence which hybridizes with the nucleic acid sequence SEQ. ID. NO. 1 under stringent conditions, or<br>
D)	functionally equivalent fragments of the sequences of A), B) or C),<br>
with the proviso that the nucleic acid having the sequence SEQ. ID. NO. 1 is excluded.<br>
6.	An expression unit comprising a nucleic acid having promoter activity according to claim 5 and additionally functionally linked nucleic acid sequence which ensures the translation of ribonucleic acids.<br>
7.	An expression unit according to claim 6, comprising<br>
E)	the nucleic acid sequence SEQ. ID. NO. 2 or<br>
F)	a sequence derived from this sequence by substitution, insertion or<br>
deletion of nucleotides and having an identity of at least 90% at the nucleic<br>
acid level with the sequence SEQ. ID. NO. 2, or<br>
G)	a nucleic acid sequence which hybridizes with the nucleic acid sequence<br>
SEQ. ID. NO. 2 under stringent conditions, or<br>
H)  functionally equivalent fragments of the sequences of E), F) or G),<br>
with the proviso that the nucleic acid having the sequence SEQ. ID. NO. 2 is excluded.<br>
8.	A method for altering or causing the transcription rate of genes in<br>
microorganisms compared with the wild type by<br>
a)	altering the specific promoter activity in the microorganism of endogenous nucleic acids having promoter activity according to claim 1, which regulate the transcription of endogenous genes, compared with the wild type or<br>
b)	regulating the transcription of genes in the microorganism by nucleic acids having promoter activity according to claim 1 or by nucleic acids having promoter activity according to claim 1 with altered specific promoter activity according to embodiment a), where the genes are heterologous in relation to the nucleic acids having promoter activity.<br><br>
9.	The method according to claim 8, wherein the regulation of the transcription of<br>
genes in the microorganism by nucleic acids having promoter activity<br>
according to claim 1 or by nucleic acids having promoter activity according to<br>
claim 1 with altered specific promoter activity according to embodiment a) is<br>
achieved by<br>
b1) introducing one or more nucleic acids having promoter activity according to claim 1, where appropriate with altered specific promoter activity, into the genome of the microorganism so that transcription of one or more endogenous genes takes place under the control of the introduced nucleic acid having promoter activity according to claim 1, where appropriate with altered specific promoter activity, or<br>
b2) introducing one or more genes into the genome of the microorganism so that transcription of one or more of the introduced genes takes place under the control of the endogenous nucleic acids having promoter activity according to claim 1, where appropriate with altered specific promoter activity, or<br>
b3) introducing one or more nucleic acid constructs comprising a nucleic acid having promoter activity according to claim 1, where appropriate with altered specific promoter activity, and functionally linked one or more nucleic acids to be transcribed, into the microorganism.<br>
10.	The method according to claim 8 or 9, wherein to increase or cause the<br>
transcription rate of genes in microorganisms compared with the wild type<br>
ah) the specific promoter activity in the microorganism of endogenous nucleic acids having promoter activity according to claim 1, or which regulate the transcription of endogenous genes, is increased compared with the wild type, or<br>
bh) the transcription of genes in the microorganism is regulated by nucleic acids having promoter activity according to claim 1 or by nucleic acids having increased specific promoter activity according to embodiment a), where the genes are heterologous in relation to the nucleic acids having promoter activity.<br><br>
11.	The method according to claim 10, wherein the regulation of the transcnption<br>
of genes in the microorganism by nucleic acids having promoter activity<br>
according to claim 1 or by nucleic acids having promoter activity according to<br>
claim 1 with increased specific promoter activity according to embodiment a) is<br>
achieved by<br>
bh1) introducing one or more nucleic acids having promoter activity according to claim 1, where appropriate with increased specific promoter activity, into the genome of the microorganism so that transcription of one or more endogenous genes takes place under the control of the introduced nucleic acid having promoter activity according to claim 1, where appropriate with increased specific promoter activity, or<br>
bh2) introducing one or more genes into the genome of the microorganism so that transcription of one or more of the introduced genes takes place under the control of the endogenous nucleic acids having promoter activity according to claim 1, where appropriate with increased specific promoter activity, or<br>
bh3) introducing one or more nucleic acid constructs comprising a nucleic acid having promoter activity according to claim 1, where appropriate with increased specific promoter activity, and functionally linked one or more nucleic acids to be transcribed, into the microorganism.<br>
12.	The method according to claim 8 or 9, wherein to reduce the transcription rate<br>
of genes in microorganisms compared with the wild type<br>
ar)   the specific promoter activity in the microorganism of endogenous nucleic acids having promoter activity according to claim 1, which regulate the transcription of endogenous genes, is reduced compared with the wild type, or<br>
br)   nucleic acids having reduced specific promoter activity according to embodiment a) are introduced into the genome of the microorganism so that the transcription of endogenous genes takes place under the control of the introduced nucleic acid having reduced promoter activity.<br>
13.	A method for altering or causing the expression rate of a gene in<br>
microorganisms compared with the wild type by<br><br>
c)	altering the specific expression activity in the microorganism of endogenous expression units according to claim 2 or 3, which regulate the expression of the endogenous genes, compared with the wild type or<br>
d)	regulating the expression of genes in the microorganism by expression units according to claim 2 or 3 or by expression units according to claim 2 or 3 with altered specific expression activity according to embodiment c), where the genes are heterologous in relation to the expression units.<br>
14.	The method according to claim 13, wherein the regulation of the expression of<br>
genes in the microorganism by expression units according to claim 2 or 3 or by<br>
expression units according to claim 2 or 3 with altered specific expression<br>
activity according to embodiment a) is achieved by<br>
d1) introducing one or more expression units according to claim 2 or 3, where appropriate with altered specific expression activity, into the genome of the microorganism so that expression of one or more endogenous genes takes place under the control of the introduced expression units, or<br>
d2) introducing one or more genes into the genome of the microorganism so that expression of one or more of the introduced genes takes place under the control of the endogenous expression units according to claim 2 or 3, where appropriate with altered specific expression activity, or<br>
d3) introducing one or more nucleic acid constructs comprising an expression unit according to claim 2 or 3, where appropriate with altered specific expression activity, and functionally linked one or more nucleic acids to be expressed, into the microorganism.<br>
15.	The method according to claim 13 or 14, wherein to increase or cause the<br>
expression rate of a gene in microorganisms compared with the wild type<br>
ch)   the specific expression activity in the microorganism of endogenous<br>
expression units according to claim 2 or 3, which regulate the expression of the endogenous genes, is increased compared with the wild type, or<br>
dh)   the expression of genes in the microorganism is regulated by expression units according to claim 2 or 3 or by expression units according to claim 2 or 3 with increased specific expression activity according to embodiment a), where the genes are heterologous in relation to the expression units.<br><br>
16.	The method according to claim 15, wherein the regulation of the expression of<br>
genes in the microorganism by expression units according to claim 2 or 3 or by<br>
expression units according to claim 2 or 3 with increased specific expression<br>
activity according to embodiment a) is achieved by<br>
dhl) introducing one or more expression units according to claim 2 or 3, where appropriate with increased specific expression activity, into the genome of the microorganism so that expression of one or more endogenous genes takes place under the control of the introduced expression units, where appropriate with increased specific expression activity, or<br>
dh2) introducing one or more genes into the genome of the microorganism so that expression of one or more of the introduced genes takes place under the control of the endogenous expression units according to claim 2 or 3, where appropriate with increased specific expression activity, or<br>
dh3) introducing one or more nucleic acid constructs comprising an expression unit according to claim 2 or 3, where appropriate with increased specific expression activity, and functionally linked one or more nucleic acids to be expressed, into the microorganism.<br>
17.	The method according to claim 13 or 14, wherein to reduce the expression rate<br>
of genes in microorganisms compared with the wild type<br>
cr)    the specific expression activity in the microorganism of endogenous<br>
expression units according to claim 2 or 3, which regulate the expression of the endogenous genes, is reduced compared with the wild type, or<br>
dr)   expression units with reduced specific expression activity according to embodiment cr) are introduced into the genome of the microorganism so that expression of endogenous genes takes place under the control of the introduced expression units with reduced expression activity.<br>
18.	The method according to any of claims 8 to 17, wherein the genes are selected<br>
from the group of nucleic acids encoding a protein from the biosynthetic<br>
pathway of proteinogenic and non-proteinogenic amino acids, nucleic acids<br>
encoding a protein from the biosynthetic pathway of nucleotides and<br>
nucleosides, nucleic acids encoding a protein from the biosynthetic pathway of<br>
organic acids, nucleic acids encoding a protein from the biosynthetic pathway<br><br>
of lipids and fatty acids, nucleic acids encoding a protein from the biosynthetic pathway of diols, nucleic acids encoding a protein from the biosynthetic pathway of carbohydrates, nucleic acids encoding a protein from the biosynthetic pathway of aromatic compounds, nucleic acids encoding a protein from the biosynthetic pathway of vitamins, nucleic acids encoding a protein from the biosynthetic pathway of cofactors and nucleic acids encoding a protein from the biosynthetic pathway of enzymes, where the genes may optionally comprise further regulatory elements.<br>
19.	The method according to claim 18, wherein the proteins from the biosynthetic pathway of amino acids are selected from the group of aspartate kinase, aspartate-semialdehyde dehydrogenase, diaminopimelate dehydrogenase, diaminopimelate decarboxylase, dihydrodipicolinate synthetase, dihydrodipicolinate reductase, glyceraldehyde-3-phosphate dehydrogenase, 3-phosphoglycerate kinase, pyruvate carboxylase, triosephosphate isomerase, transcriptional regulator LuxR, transcriptional regulator LysRI, transcriptional regulator LysR2, malate-quinone oxidoreductase, glucose-6-phosphate deydrogenase, 6-phosphogluconate dehydrogenase, transketolase, transaldolase, homoserine 0-acetyltransferase, cystathionine gamma-synthase, cystathionine beta-lyase, serine hydroxymethyltransferase, 0-acetylhomoserine sulfhydrylase, methylenetetrahydrofolate reductase, phosphoserine aminotransferase, phosphoserine phosphatase, serine acetyl-transferase, homoserine dehydrogenase, homoserine kinase, threonine synthase, threonine exporter carrier, threonine dehydratase, pyruvate oxidase, lysine exporter, biotin ligase, cysteine synthase I, cysteine synthase II, coenzyme B12-dependent methionine synthase, coenzyme B12-independent methionine synthase, sulfate adenylyltransferase subunit 1 and 2, phosphoadenosine-phosphosulfate reductase, ferredoxin-sulfite reductase, ferredoxin NADP reductase, 3-phosphoglycerate dehydrogenase, RXA00655 regulator, RXN2910 regulator, arginyl-tRNA synthetase, phosphoenolpyruvate carboxylase, threonine efflux protein, serine hydroxymethyltransferase, fructose-1,6-bisphosphatase, protein of sulfate reduction RXA077, protein of sulfate reduction RXA248, protein of sulfate reduction RXA247, protein OpcA, 1-phosphofructokinase and 6-phosphofructokinase.<br>
20.	An expression cassette comprising<br><br>
a)	at least one expression unit according to claim 2 or 3 and<br>
b)	at least one further nucleic acid to be expressed, and<br><br>
c) where appropriate further genetic control elements,<br>
where at least one expression unit and a further nucleic acid sequence to be expressed are functionally linked together, and the further nucleic acid sequence to be expressed is heterologous in relation to the expression unit.<br>
21.	The expression cassette according to claim 20, wherein the further nucleic acid sequence to be expressed is selected from the group of nucleic acids encoding a protein from the biosynthetic pathway of proteinogenic and non-proteinogenic amino acids, nucleic acids encoding a protein from the biosynthetic pathway of nucleotides and nucleosides, nucleic acids encoding a protein from the biosynthetic pathway of organic acids, nucleic acids encoding a protein from the biosynthetic pathway of lipids and fatty acids, nucleic acids encoding a protein from the biosynthetic pathway of diols, nucleic acids encoding a protein from the biosynthetic pathway of carbohydrates, nucleic acids encoding a protein from the biosynthetic pathway of aromatic compounds, nucleic acids encoding a protein from the biosynthetic pathway of vitamins, nucleic acids encoding a protein from the biosynthetic pathway of cofactors and nucleic acids encoding a protein from the biosynthetic pathway of enzymes.<br>
22.	The expression cassette according to claim 21, wherein the proteins from the biosynthetic pathway of amino acids are selected from the group of aspartate kinase, aspartate-semialdehyde dehydrogenase, diaminopimelate dehydrogenase, diaminopimelate decarboxylase, dihydrodipicolinate synthetase, dihydrodipicolinate reductase, glyceraldehyde-3-phosphate dehydrogenase, 3-phosphoglycerate kinase, pyruvate carboxylase, triosephosphate isomerase, transcriptional regulator LuxR, transcriptional regulator LysRI, transcriptional regulator LysR2, malate-quinone oxidoreductase, glucose-6-phosphate deydrogenase, 6-phosphogluconate dehydrogenase, transketolase, transaldolase, homosehne O-acetyltransferase, cystathionine gamma-synthase, cystathionine beta-lyase, serine hydroxymethyltransferase, O-acetylhomoserine sulfhydrylase, methylenetetrahydrofolate reductase, phosphoserine aminotransferase, phosphoserine phosphatase, serine acetyltransferase, homosehne dehydrogenase, homosehne kinase, threonine synthase, threonine exporter carrier, threonine dehydratase, pyruvate oxidase, lysine exporter, biotin ligase, cysteine synthase I, cysteine synthase II, coenzyme B12-dependent methionine synthase, coenzyme B12-independent methionine synthase<br><br>
activity, sulfate adenylyltransferase subunit 1 and 2, phosphoadenosine-phosphosulfate reductase, ferredoxin-sulfite reductase, ferredoxin NADP reductase, 3-phosphoglycerate dehydrogenase, RXA00655 regulator, RXN2910 regulator, arginyl-tRNA synthetase, phosphoenolpyruvate carboxylase, threonine efflux protein, serine hydroxymethyltransferase, fructose-1,6-bisphosphatase, protein of sulfate reduction RXA077, protein of sulfate reduction RXA248, protein of sulfate reduction RXA247, protein OpcA, 1-phosphofructokinase and 6-phosphofructokinase.<br>
23.	An expression vector comprising an expression cassette according to any of claims 20 to 22.<br>
24.	A genetically modified microorganism, where the genetic modification leads to an alteration or causing of the transcription rate of at least one gene compared with the wild type, and is dependent on<br><br>
a)	altering the specific promoter activity in the microorganism of at least one endogenous nucleic acid having promoter activity according to claim 1, which regulates the transcription of at least one endogenous gene, or<br>
b)	regulating the transcription of genes in the microorganism by nucleic acids having promoter activity according to claim 1 or by nucleic acids having promoter activity according to claim 1 with altered specific promoter activity according to embodiment a), where the genes are heterologous in relation to the nucleic acids having promoter activity.<br>
25.	The genetically modified microorganism according to claim 24, wherein the<br>
regulation of the transcription of genes in the microorganism by nucleic acids<br>
having promoter activity according to claim 1 or by nucleic acids having<br>
promoter activity according to claim 1 with altered specific promoter activity<br>
according to embodiment a), is achieved by<br>
b1) introducing one or more nucleic acids having promoter activity according to claim 1, where appropriate with altered specific promoter activity, into the genome of the microorganism so that transcription of one or more endogenous genes takes place under the control of the introduced nucleic acid having promoter activity according to claim 1, where appropriate with altered specific promoter activity, or<br><br>
b2) introducing one or more genes into the genome of the microorganism so that transcription of one or more of the introduced genes takes place under the control of the endogenous nucleic acids having promoter activity according to claim 1, where appropriate with altered specific promoter activity, or<br>
b3) introducing one or more nucleic acid constructs comprising a nucleic acid having promoter activity according to claim 1, where appropriate with altered specific promoter activity, and functionally linked one or more nucleic acids to be transcribed, into the microorganism.<br>
26.	The genetically modified microorganism according to claim 24 or 25 having<br>
increased or caused transcription rate of at least one gene compared with the<br>
wild type, wherein<br>
ah) the specific promoter activity in the microorganism of endogenous nucleic acids having promoter activity according to claim 1, which regulate the transcription of endogenous genes, is increased compared with the wild type, or<br>
bh) the transcription of genes in the microorganism is regulated by nucleic acids having promoter activity according to claim 1 or by nucleic acids having increased specific promoter activity according to embodiment ah), where the genes are heterologous in relation to the nucleic acids having promoter activity.<br>
27.	The genetically modified microorganism according to claim 26, wherein the<br>
regulation of the transcription of genes in the microorganism by nucleic acids<br>
having promoter activity according to claim 1 or by nucleic acids having<br>
promoter activity according to claim 1 with increased specific promoter activity<br>
according to embodiment a), is achieved by<br>
bh1) introducing one or more nucleic acids having promoter activity according to claim 1, where appropriate with increased specific promoter activity, into the genome of the microorganism so that transcription of one or more endogenous genes takes place under the control of the introduced nucleic acid having promoter activity, where appropriate with increased specific promoter activity, or<br><br>
bh2) introducing one or more genes into the genome of the microorganism so that transcription of one or more of the Introduced genes takes place under the control of the endogenous nucleic acids having promoter activity according to claim 1, where appropriate with increased specific promoter activity, or<br>
bh3) introducing one or more nucleic acid constructs comprising a nucleic acid having promoter activity according to claim 1, where appropriate with increased specific promoter activity, and functionally linked one or more nucleic acids to be transcribed, into the microorganism.<br>
28.	The genetically modified microorganism according to claim 24 or 25 having<br>
reduced transcription rate of at least one gene compared with the wild type,<br>
wherein<br>
ar)   the specific promoter activity in the microorganism of at least one<br>
endogenous nucleic acid having promoter activity according to claim 1, which regulates the transcription of at least one endogenous gene, is reduced compared with the wild type, or<br>
br) one or more nucleic acids having reduced promoter activity according to embodiment a) are introduced into the genome of the microorganism so that the transcription of at least one endogenous gene takes place under the control of the introduced nucleic acid having reduced promoter activity.<br>
29.	A genetically modified microorganism, where the genetic modification leads to<br>
an alteration or causing of the expression rate of at least one gene compared<br>
with the wild type, and is dependent on<br>
c)	altering the specific expression activity in the microorganism of at least one endogenous expression unit according to claim 2 or 3, which regulates the expression of at least one endogenous gene, compared with the wild type or<br>
d)	regulating the expression of genes in the microorganism by expression units according to claim 2 or 3 or by expression units according to claim 2 or 3 with altered specific expression activity according to embodiment a), where the genes are heterologous in relation to the expression units.<br><br>
30.	The genetically modified microorganism according to claim 29, wherein the<br>
regulation of the expression of genes in the microorganism by expression units<br>
according to claim 2 or 3 or by expression units according to claim 2 or 3 with<br>
altered specific expression activity according to embodiment a) is achieved by<br>
d1) introducing one or more expression units according to claim 2 or 3, where appropriate with altered specific expression activity, into the genome of the microorganism so that expression of one or more endogenous genes takes place under the control of the introduced expression units according to claim 2 or 3, where appropriate with altered specific expression activity, or<br>
d2) introducing one or more genes into the genome of the microorganism so that expression of one or more of the introduced genes takes place under the control of the endogenous expression units according to claim 2 or 3, where appropriate with altered specific expression activity, or<br>
d3) introducing one or more nucleic acid constructs comprising an expression unit according to claim 2 or 3, where appropriate with altered specific expression activity, and functionally linked one or more nucleic acids to be expressed, into the microorganism.<br>
31.	The genetically modified microorganism according to claim 29 or 30 with<br>
increased or caused expression rate of at least one gene compared with the<br>
wild type, wherein<br>
ch)   the specific expression activity in the microorganism of at least one endogenous expression unit according to claim 2 or 3, which regulates the expression of the endogenous genes, is increased compared with the wild type, or<br>
dh) the expression of genes in the microorganism is regulated by expression units according to claim 2 or 3 or by expression units according to claim 2 or 3 with increased specific expression activity according to embodiment a), where the genes are heterologous in relation to the expression units.<br>
32.	The genetically modified microorganism according to claim 31, wherein the<br>
regulation of the expression of genes in the microorganism by expression units<br>
according to claim 2 or 3 or by expression units according to claim 2 or 3 with<br>
increased specific expression activity according to embodiment a) is achieved<br>
by<br><br>
dh1) introducing one or more expression units according to claim 2 or 3, where appropriate with increased specific expression activity, into the genome of the microorganism so that expression of one or more endogenous genes takes place under the control of the introduced expression unit according to claim 2 or 3, where appropriate with increased specific expression activity, or<br>
dh2) introducing one or more genes into the genome of the microorganism so that expression of one or more of the introduced genes takes place under the control of the endogenous expression units according to claim 2 or 3, where appropriate with increased specific expression activity, or<br>
dh3) introducing one or more nucleic acid constructs comprising an expression unit according to claim 2 or 3, where appropriate with increased specific expression activity, and functionally linked one or more nucleic acids to be expressed, into the microorganism.<br>
33.	The genetically modified microorganism according to claim 29 or 30 with<br>
reduced expression rate of at least one gene compared with the wild type,<br>
wherein<br>
cr)   the specific expression activity in the microorganism of at least one endogenous expression unit according to claim 2 or 3, which regulates the expression of at least one endogenous gene, is reduced compared with the wild type, or<br>
dr)   one or more expression units according to claim 2 or 3 with reduced expression activity are introduced into the genome of the microorganism so that expression of at least one gene takes place under the control of the introduced expression unit according to claim 2 or 3 with reduced expression activity.<br>
34.	A genetically modified microorganism comprising an expression unit according to claim 2 or 3 and functionally linked a gene to be expressed, where the gene is heterologous in relation to the expression unit.<br>
35.	The genetically modified microorganism according to claim 34, comprising an<br>
expression cassette according to any of claims 20 to 22.<br>
expression cassette according to any of claims 20 to 22<br><br>
36.	The genetically modified microorganism according to any of claims 24 to 35, wherein the genes are selected from the group of nucleic acids encoding a protein from the biosynthetic pathway of proteinogenic and non-proteinogenic amino acids, nucleic acids encoding a protein from the biosynthetic pathway of nucleotides and nucleosides, nucleic acid encoding a protein from the biosynthetic pathway of organic acids, nucleic acids encoding a protein from the biosynthetic pathway of lipids and fatty acids, nucleic acids encoding a protein from the biosynthetic pathway of diols, nucleic acids encoding a protein from the biosynthetic pathway of carbohydrates, nucleic acids encoding a protein from the biosynthetic pathway of aromatic compounds, nucleic acids encoding a protein from the biosynthetic pathway of vitamins, nucleic acids encoding a protein from the biosynthetic pathway of cofactors and nucleic acids encoding a protein from the biosynthetic pathway of enzymes, where the genes may optionally comprise further regulatory elements.<br>
37.	The genetically modified microorganism according to claim 36, wherein the proteins from the biosynthetic pathway of amino acids are selected from the group of aspartate kinase, aspartate-semialdehyde dehydrogenase, diaminopimelate dehydrogenase, diaminopimelate decarboxylase, dihydrodipicolinate synthetase, dihydrodipicolinate reductase, glyceraldehyde-3-phosphate dehydrogenase, 3-phosphoglycerate kinase, pyruvate carboxylase, triosephosphate isomerase, transcriptional regulator LuxR, transcriptional regulator LysRI, transcriptional regulator LysR2, malate-quinone oxidoreductase, glucose-6-phosphate deydrogenase, 6-phosphogluconate dehydrogenase, transketolase, transaldolase, homoserine 0-acetyltransf erase, cystathionine gamma-synthase, cystathionine beta-lyase, serine hydroxymethyltransferase, O-acetylhomoserine sulfhydrylase, methylenetetrahydrofolate reductase, phosphoserine aminotransferase, phosphoserine phosphatase, serine acetyltransferase, homoserine dehydrogenase, homoserine kinase, threonine synthase, threonine exporter carrier, threonine dehydratase, pyruvate oxidase, lysine exporter, biotin ligase. cysteine synthase I, cysteine synthase II, coenzyme B12-dependent methionine synthase, coenzyme B12-independent methionine synthase, sulfate adenylyltransferase subunit 1 and 2, phosphoadenosine-phosphosulfate reductase, ferredoxin-sulfite reductase, ferredoxin NADP reductase, 3-phosphoglycerate dehydrogenase, RXA00655 regulator, RXN2910 regulator, arginyl-tRNA synthetase, phosphoenolpyruvate carboxylase, threonine efflux protein, serine hydroxymethyltransferase, fructose-1,6-bisphosphatase, protein of sulfate reduction RXA077, protein of<br><br>
sulfate reduction RXA248, protein of sulfate reduction RXA247, protein OpcA, 1-phosphofructokinase and 6-phosphofructokinase.<br>
38.	A method for preparing biosynthetic products by cultivating genetically modified microorganisms according to any of claims 24 to 37.<br>
39.	A method for preparing lysine by cultivating genetically modified microorganisms according to any of claims 24, 25, 31 or 32, wherein the genes are selected from the group of nucleic acids encoding an aspartate kinase, nucleic acids encoding an aspartate-semialdehyde dehydrogenase, nucleic acids encoding a diaminopimelate dehydrogenase, nucleic acids encoding a diaminopimelate decarboxylase, nucleic acids encoding a dihydrodipicolinate synthetase, nucleic acids encoding a dihydrodipicolinate reductase, nucleic acids encoding a glyceraldehyde-3-phosphate dehydrogenase, nucleic acids encoding a 3-phosphoglycerate kinase, nucleic acids encoding a pyruvate carboxylase, nucleic acids encoding a triosephosphate isomerase, nucleic acids encoding a transcriptional regulator LuxR, nucleic acids encoding a transcriptional regulator LysRI, nucleic acids encoding a transcriptional regulator LysR2, nucleic acids encoding a malate-quinone oxidoreductase, nucleic acids encoding a glucose-6-phosphate dehydrogenase, nucleic acids encoding a 6-phosphogluconate dehydrogenase, nucleic acids encoding a transketolase, nucleic acids encoding a transaldolase, nucleic acids encoding a lysine exporter, nucleic acids encoding a biotin ligase, nucleic acids encoding an arginyl-tRNA synthetase, nucleic acids encoding a phosphoenolpyruvate carboxylase, nucleic acids encoding a fructose-1,6-bisphosphatase, nucleic acids encoding a protein OpcA, nucleic acids encoding a 1-phosphofructokinase and nucleic acids encoding a 6-phosphofructokinase.<br>
40.	The method according to claim 39, wherein the genetically modified microorganisms have, compared with the wild type, additionally an increased activity, of at least one of the activities selected from the group of aspartate kinase activity, aspartate-semialdehyde dehydrogenase activity, diaminopimelate dehydrogenase activity, diaminopimelate decarboxylase activity, dihydrodipicolinate synthetase activity, dihydrodipicolinate reductase activity, glyceraldehyde-3-phosphate dehydrogenase activity, 3-phosphoglycerate kinase activity, pyruvate carboxylase activity, triosephosphate isomerase activity, activity of the transcriptional regulator LuxR, activity of the transcriptional regulator LysR1, activity of the transcriptional regulator LysR2, malate-quinone oxidoreductase activity, glucose-6-phosphate deydrogenase activity, 6-phosphogluconate<br><br>
dehydrogenase activity, transketolase activity, transaldolase activity, lysine exporter activity, arginyl-tRNA synthetase activity, phosphoenolpyruvate carboxylase activity, fructose-1,6-bisphosphatase activity, protein OpcA activity, 1-phosphofructokinase activity, 6-phosphofructokinase activity and biotin ligase activity.<br>
41.	The method according to claim 39 or 40, wherein the genetically modified microorganisms have, compared with the wild type, additionally a reduced activity, of at least one of the activities selected from the group of threonine dehydratase activity, homoserine 0-acetyltransferase activity, O-acetyl-homoserine sulfhydrylase activity, phosphoenolpyruvate carboxykinase activity, pyruvate oxidase activity, homoserine kinase activity, homoserine dehydrogenase activity, threonine exporter activity, threonine efflux protein activity, asparaginase activity, aspartate decarboxylase activity and threonine synthase activity.<br>
42.	A method for preparing methionine by cultivating genetically modified microorganisms according to any of claims 24, 25, 31 or 32, wherein the genes are selected from the group of nucleic acids encoding an aspartate kinase, nucleic acids encoding an aspartate-semialdehyde dehydrogenase, nucleic acids encoding a homoserine dehydrogenase, nucleic acids encoding a glyceraldehyde-3-phosphate dehydrogenase, nucleic acids encoding a 3-phosphoglycerate kinase, nucleic acids encoding a pyruvate carboxylase, nucleic acids encoding a triosephosphate isomerase, nucleic acids encoding a homoserine 0-acetyltransferase. nucleic acids encoding a cystathionine gamma-synthase, nucleic acids encoding a cystathionine beta-lyase, nucleic acids encoding a serine hydroxymethyltransferase, nucleic acids encoding an 0-acetylhomoserine sulfhydrylase, nucleic acids encoding a methylenetetrahydrofolate reductase, nucleic acids encoding a phosphoserine aminotransferase, nucleic acids encoding a phosphoserine phosphatase, nucleic acids encoding a serine acetyltransferase, nucleic acids encoding a cysteine synthase I, nucleic acids encoding a cysteine synthase II, nucleic acids encoding a coenzyme B12-dependent methionine synthase, nucleic acids encoding a coenzyme B12-independent methionine synthase, nucleic acids encoding a sulfate adenylyltransferase, nucleic acids encoding a phosphoadenosine phosphosulfate reductase, nucleic acids encoding a ferredoxin-sulfite reductase, nucleic acids encoding a ferredoxin NADPH-reductase, nucleic acids encoding a ferredoxin activity, nucleic acids encoding a protein of sulfate reduction RXA077, nucleic acids encoding a protein of sulfate reduction RXA248, nucleic acids encoding a protein of sulfate reduction<br><br>
RXA247, nucleic acids encoding an RXA0655 regulator and nucleic acids encoding an RXN2910 regulator.<br>
43.	The method according to claim 42, wherein the genetically modified microorganisms have, compared with the wild type, additionally an increased activity, of at least one of the activities selected from the group of aspartate kinase activity, aspartate-semialdehyde dehydrogenase activity, homoserine dehydrogenase activity, glyceraldehyde-3-phosphate dehydrogenase activity, 3-phosphoglycerate kinase activity, pyruvate carboxylase activity, triosephosphate isomerase activity, homoserine O-acetyltransferase activity, cystathionine gamma-synthase activity, cystathionine beta-lyase activity, serine hydroxymethyltransferase activity, O-acetylhomoserine sulfhydrylase activity, methylenetetrahydrofolate reductase activity, phosphoserine aminotransferase activity, phosphoserine phosphatase activity, serine acetyltransferase activity, cysteine synthase I activity, cysteine synthase II activity, coenzyme B12-dependent methionine synthase activity, coenzyme B12-independent methionine synthase activity, sulfate adenylyltransferase activity, phosphoadenosine-phosphosulfate reductase activity, ferredoxin-sulfite reductase activity, ferredoxin NADPH-reductase activity, ferredoxin activity, activity of a protein of sulfate reduction RXA077, activity of a protein of sulfate reduction RXA248, activity of a protein of sulfate reduction RXA247, activity of an RXA655 regulator and activity of an RXN2910 regulator.<br>
44.	The method according to claim 42 or 43, wherein the genetically modified microorganisms have, compared with the wild type, additionally a reduced activity, of at least one of the activities selected from the group of homoserine kinase activity, threonine dehydratase activity, threonine synthase activity, meso-diaminopimelate D-dehydrogenase activity, phosphoenolpyruvate carboxykinase activity, pyruvate oxidase activity, dihydrodipicolinate synthase activity, dihydrodipicolinate reductase activity and diaminopicolinate decarboxylase activity.<br>
45.	A method for preparing threonine by cultivating genetically modified microorganisms according to any of claims 24, 25, 31 or 32, wherein the genes are selected from the group of nucleic acids encoding an aspartate kinase, nucleic acids encoding an aspartate-semialdehyde dehydrogenase, nucleic acids encoding a glyceraldehyde-3-phosphate dehydrogenase, nucleic acids encoding a 3-phosphoglycerate kinase, nucleic acids encoding a pyruvate carboxylase, nucleic acids encoding a triosephosphate isomerase, nucleic acids encoding a homoserine kinase, nucleic acids encoding a threonine<br><br>
synthase, nucleic acids encoding a threonine exporter carrier, nucleic acids encoding a glucose-6-phosphate dehydrogenase, nucleic acids encoding a transaldolase, nucleic acids encoding a transketolase, nucleic acids encoding a malate-quinone oxidoreductase, nucleic acids encoding a 6-phosphogluconate dehydrogenase, nucleic acids encoding a lysine exporter, nucleic acids encoding a biotin ligase, nucleic acids encoding a phosphoenolpyruvate carboxylase, nucleic acids encoding a threonine efflux protein, nucleic acids encoding a fructose-1,6-bisphosphatase, nucleic acids encoding an OpcA protein, nucleic acids encoding a 1-phosphofructokinase, nucleic acids encoding a 6-phosphofructokinase, and nucleic acids encoding a homoserine dehydrogenase.<br>
46.	The method according to claim 45, wherein the genetically modified microorganisms have, compared with the wild type, additionally an increased activity, of at least one of the activities selected from the group of aspartate kinase activity, aspartate-semialdehyde dehydrogenase activity, glyceraldehyde-3-phosphate dehydrogenase activity, 3-phosphoglycerate kinase activity, pyruvate carboxylase activity, triosephosphate isomerase activity, threonine synthase activity, activity of a threonine export carrier, transaldolase activity, transketolase activity, glucose-6-phosphate dehydrogenase activity, malate-quinone oxidoreductase activity, homoserine kinase activity, biotin ligase activity, phosphoenolpyruvate carboxylase activity, threonine efflux protein activity, protein OpcA activity, 1-phosphofructokinase activity, 6-phosphofructokinase activity, fructose-1-6-bisphosphatase activity, 6-phosphogluconate dehydrogenase and homoserine dehydrogenase activity.<br>
47.	The method according to claim 45 or 46, wherein the genetically modified microorganisms have, compared with the wild type, additionally a reduced activity, of at least one of the activities selected from the group of threonine dehydratase activity, homoserine O -acetyltransferase activity, serine hydroxymethyltransferase activity, O-acetylhomoserine sulfhydrylase activity, meso-diaminopimelate D-dehydrogenase activity, phosphoenolpyruvate carboxykinase activity, pyruvate oxidase activity, dihydrodipicolinate synthetase activity, dihydrodipicolinate reductase activity, asparaginase activity, aspartate decarboxylase activity, lysine exporter activity, acetolactate synthase activity, ketol-acid reductoisomerase activity, branched chain aminotransferase activity, coenzyme B12-dependent methionine synthase activity, coenzyme B12-lndependent methionine synthase activity, dihydroxy-acid dehydratase activity and diaminopicolinate decarboxylase activity.<br><br>
48.	The method according to any of claims 38 to 47, wherein the biosynthetic<br>
products are isolated and, where appropriate, purified from the cultivation<br>
medium after and/or during the cultivation step.<br>
49.	The use of the nucleic acid sequence SEQ. ID. NO. 53 as ribosome binding site in expression units which enable genes to be expressed in bacteria of the genus Corynebacterium or Brevibacterium.<br>
50.	The use of the nucleic acid sequence SEQ. ID. NO. 52 as<br>
-10 region in expression units which enable genes to be expressed in bacteria of the genus Corynebacterium or Brevibacterium.<br>
51.	An expression unit which enables genes to be expressed in bacteria of the<br>
genus Corynebacterium or Brevibacterium, comprising the nucleic acid<br>
sequence SEQ. ID. NO. 53.<br>
52.	The expression unit according to claim 51, wherein the nucleic acid sequence<br>
SEQ. ID. NO. 53 is used as ribosome binding site.<br>
53.	An expression unit which enables genes to be expressed in bacteria of the genus Corynebacterium or Brevibacterium, comprising the nucleic acid sequence SEQ. ID. NO. 52.<br>
54.	The expression unit according to claim 53, wherein the nucleic acid sequence SEQ. ID. NO. 52 is used as -10 region.<br><br></td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDM5LUNIRU5QLTIwMDggICAgQ09SUkVTUE9OREVOQ0UgT1RIRVJTICAwMy0xMi0yMDEyLnBkZg==" target="_blank" style="word-wrap:break-word;">439-CHENP-2008    CORRESPONDENCE OTHERS  03-12-2012.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDM5LUNIRU5QLTIwMDggICBDT1JSRVNQT05ERU5DRSBPVEhFUlMgICAgIDIyLTAxLTIwMTUucGRm" target="_blank" style="word-wrap:break-word;">439-CHENP-2008   CORRESPONDENCE OTHERS     22-01-2015.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDM5LUNIRU5QLTIwMDggICBDT1JSRVNQT05ERU5DRSBPVEhFUlMgIDA1LTA5LTIwMTMucGRm" target="_blank" style="word-wrap:break-word;">439-CHENP-2008   CORRESPONDENCE OTHERS  05-09-2013.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDM5LUNIRU5QLTIwMDggICBDT1JSRVNQT05ERU5DRSBPVEhFUlMgIDE4LTA0LTIwMTMucGRm" target="_blank" style="word-wrap:break-word;">439-CHENP-2008   CORRESPONDENCE OTHERS  18-04-2013.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDM5LUNIRU5QLTIwMDggICBDT1JSRVNQT05ERU5DRSBPVEhFUlMgMDEtMDQtMjAxMy5wZGY=" target="_blank" style="word-wrap:break-word;">439-CHENP-2008   CORRESPONDENCE OTHERS 01-04-2013.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDM5LUNIRU5QLTIwMDggICBDT1JSRVNQT05ERU5DRSBPVEhFUlMgMjUtMDYtMjAxMy5wZGY=" target="_blank" style="word-wrap:break-word;">439-CHENP-2008   CORRESPONDENCE OTHERS 25-06-2013.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDM5LUNIRU5QLTIwMDggICBDT1JSRVNQT05ERU5DRSBPVEhFUlMuICAgMjktMTAtMjAxMy5wZGY=" target="_blank" style="word-wrap:break-word;">439-CHENP-2008   CORRESPONDENCE OTHERS.   29-10-2013.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDM5LUNIRU5QLTIwMDggICBFWEFNSU5BVElPTiBSRVBPUlQgUkVQTFkgUkVDRUlWRUQgMTMtMDMtMjAxMy5wZGY=" target="_blank" style="word-wrap:break-word;">439-CHENP-2008   EXAMINATION REPORT REPLY RECEIVED 13-03-2013.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDM5LUNIRU5QLTIwMDggICBGT1JNLTMgIDEzLTAzLTIwMTMucGRm" target="_blank" style="word-wrap:break-word;">439-CHENP-2008   FORM-3  13-03-2013.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDM5LUNIRU5QLTIwMDggICBGT1JNLTMgIDE4LTA0LTIwMTMucGRm" target="_blank" style="word-wrap:break-word;">439-CHENP-2008   FORM-3  18-04-2013.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDM5LUNIRU5QLTIwMDggICBGT1JNLTMgIDI5LTEwLTIwMTMucGRm" target="_blank" style="word-wrap:break-word;">439-CHENP-2008   FORM-3  29-10-2013.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDM5LUNIRU5QLTIwMDggIE9USEVSIFBBVEVOVCBET0NVTUVOVCAgMTMtMDMtMjAxMy5wZGY=" target="_blank" style="word-wrap:break-word;">439-CHENP-2008  OTHER PATENT DOCUMENT  13-03-2013.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDM5LUNIRU5QLTIwMDggIE9USEVSUyAgMTMtMDMtMjAxMy5wZGY=" target="_blank" style="word-wrap:break-word;">439-CHENP-2008  OTHERS  13-03-2013.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDM5LUNIRU5QLTIwMDggQ0xBSU1TIDIwLTAzLTIwMTUucGRm" target="_blank" style="word-wrap:break-word;">439-CHENP-2008 CLAIMS 20-03-2015.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDM5LUNIRU5QLTIwMDggQ09SUkVTUE9OREVOQ0UgT1RIRVJTIDIwLTAzLTIwMTUucGRm" target="_blank" style="word-wrap:break-word;">439-CHENP-2008 CORRESPONDENCE OTHERS 20-03-2015.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDM5LWNoZW5wLTIwMDgtYWJzdHJhY3QucGRm" target="_blank" style="word-wrap:break-word;">439-chenp-2008-abstract.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDM5LWNoZW5wLTIwMDgtY2xhaW1zLnBkZg==" target="_blank" style="word-wrap:break-word;">439-chenp-2008-claims.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDM5LWNoZW5wLTIwMDgtY29ycmVzcG9uZG5lY2Utb3RoZXJzLnBkZg==" target="_blank" style="word-wrap:break-word;">439-chenp-2008-correspondnece-others.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDM5LWNoZW5wLTIwMDgtZGVzY3JpcHRpb24oY29tcGxldGUpLnBkZg==" target="_blank" style="word-wrap:break-word;">439-chenp-2008-description(complete).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDM5LWNoZW5wLTIwMDgtZHJhd2luZ3MucGRm" target="_blank" style="word-wrap:break-word;">439-chenp-2008-drawings.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDM5LWNoZW5wLTIwMDgtZm9ybSAxLnBkZg==" target="_blank" style="word-wrap:break-word;">439-chenp-2008-form 1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDM5LWNoZW5wLTIwMDgtZm9ybSAyNi5wZGY=" target="_blank" style="word-wrap:break-word;">439-chenp-2008-form 26.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDM5LWNoZW5wLTIwMDgtZm9ybSAzLnBkZg==" target="_blank" style="word-wrap:break-word;">439-chenp-2008-form 3.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDM5LWNoZW5wLTIwMDgtZm9ybSA1LnBkZg==" target="_blank" style="word-wrap:break-word;">439-chenp-2008-form 5.pdf</a></p>
		<br>
		<div class="pull-left">
			<a href="266061-personal-care-composition-containing-a-non-guar-galactomannan-polymer-derivative-and-an-anionic-surfactant-system.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="266063-double-ended-inverter-system-for-a-vehicle-having-two-energy-sources-that-exhibit-different-operating-characteristics.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>266062</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>439/CHENP/2008</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>14/2015</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>03-Apr-2015</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>30-Mar-2015</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>25-Jan-2008</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>BASF AKTIENGESELLSCHAFT</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td>D-67056 LUDWIGSHAFEN</td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>ZELDER, OSKAR</td>
											<td>FRANZ-STUTZEL-STRASSE 8 67346 SPEYER</td>
										</tr>
										<tr>
											<td>2</td>
											<td>KLOPPROGGE, CORINNA</td>
											<td>RASTATTER STRASSE 10 68239 MANNHEIM</td>
										</tr>
										<tr>
											<td>3</td>
											<td>KROGER, BURKHARD</td>
											<td>IM WALDHOF 1 67117 LIMBURGERHOF</td>
										</tr>
										<tr>
											<td>4</td>
											<td>SCHRODER, HARTWIG</td>
											<td>BENZSTRASSE 4 69226 NUSSLOCH</td>
										</tr>
										<tr>
											<td>5</td>
											<td>HAEFNER, STEFAN</td>
											<td>LUITPOLDSTRASSE 11 67063 LUDWIGSHAFEN</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>C07K 14/34</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>PCT/EP04/14263</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td>2004-12-15</td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td>103 59 595.3</td>
									<td>2003-12-18</td>
								    <td>Germany</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/266062-pgro-expression-units by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 06:35:02 GMT -->
</html>
