<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/266170-multi-gene-expression-vehicle by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 06:44:29 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 266170:MULTI-GENE EXPRESSION VEHICLE</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">MULTI-GENE EXPRESSION VEHICLE</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>A multigene expression vehicle (MGEV) consisting essentially of a polynucleotide comprising 2 to 8 domain segments, D, each domain encoding a functional protein, each domain being joined to the next in a linear sequence by a Linker (L) segment encoding a Linker peptide, the D and L segments all being in the same reading frame, and at least one of the domains is not a type two protease inhibitor.</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td>MULTI-GENE EXPRESSION VEHICLE<br>
BACKGROUND OF THE INVENTION<br>
[0001] The potato type two inhibitors are a family of serine proteins inhibitors that are found in many Solanaceous plants. The inhibitors are so named because the first members described were isolated from potato and tomato plants [Bryant, J. et al. (1976) Biochemistry 15:3418-3424; Plump<eat g. et al. arch. brioche. biopsy the inhibitors often consist of two repeated domains each domain about and with a reactive site to either chemotropism or try sin. these two-domain are encoded by genes temped pin2 gene family which expressed in tomato fruit potato tubers as well leaves both plants after mechanical wounding insect damage js j. biol. chem. keil m. nun. acids res. thornberg rw proc. nati. acad. sci. usa several members this have been cloned from most same structure original described mol. gen. genet. thornburg supra></eat>
[0002] The potato type two inhibitors are referred to simply as "type two" inhibitors herein. Type two inhibitors are structurally related proteins that are encoded by a family of genes knows as Pin2. At least 11 homologous Pin2 genes have been found in both mono- and dicotyledonous plants. Pin 2 genes can encode either a single 6kDa proteins inhibitor (PI) domain, two 6kDa PI domains like those that are common in potato and tomato or several highly homologous repeated 6kDa domains that inhibit try sin or chemotropism, often circularly permuted. For a catalog of sequences and discussion of structural relationships, see Bart et al., (2002) Trends in Genetics 18:600-603. Sequences have been compiled in a database accessible at http.7/www.ba.itb.cnr.it/Plant-Pls (see also Deleon, F. et al., (2002) Null. Acid Res. 30:347-348.)<br><br>
[0003] In addition to the two-domain 12kDa inhibitors, potatoes also contain lower levels of a series of single-domain inhibitors of approximately 6kDa [Hass, GM, et al. (1982) Biochemistry 21:752-756] which are identical in sequence to the central portion of the two-domain proteins and are likely to be proteolytic products [Sanchez-Serrano supra]. Similar single-domain proteins inhibitors (Pi's) have been isolated from eggplant [Richardson, M. (1979) FEBS. Let. 104:322-326] and tobacco [Pearce, G. et al. (1993) Plant Pfiyslol. 102:639-644], although It is not known if they are derived from a larger precursor molecule. Both tomato and tobacco contain a gene encoding a three-domain inhibitor [Taylor, BH, et al. (1993) Plant Mol. Biol. 23:10 05-1014; Balding, R. et al. (1995) Plant Mol. Biol. 27:1197-1204], and a gene encoding a six-domain inhibitor (NaPI-ii) has been isolated from the reproductive tissues of the ornamental tobacco, Nicotine alata [Atkinson, AH, et al. (1993) Plant Cell 5:203-213].<br>
[0004] Napa-ii (SEQ ID N0:1) encodes a 40.3kDa precursor protein that contains six inhibitory domains, two reactive against chymotrypsin and four reactive against try sin [Atkinson supra]. Proteolytic processing of the precursor protein occurs in a linker region between domains resulting in the release of six mature, active inhibitors [Heath, RL, et al. (1995) Eur. J. Biochip. 230:250-257; Lee, MCS, et al. (1999) Nature Struct. Biol. 6:526-530]. In addition to the proteinase inhibitory domains, the precursor also has an N-terminal putative ER signal peptide and a C-terminal non-repeated domain which probably functions as a vacuolar sorting signal [Miller, EA, et al, (1999) Plant Cell 11:1499-1508; Nielsen, KJ, et al. (1996) Biochemistry 35:369-378]. Previously we have shown that immature stigmas express two mRNAs that hybridise to the NaPI-ii cDNA [Atkinson supra]. One message of 1.4kb corresponds to the six-domain inhibitor, while a second message of approximately I.Okb encodes a smaller isoform.<br>
[0005] A second type two Pi proteinase precursor having four repeated proteinase inhibitor domains has been isolated from N. alata stigmas, designated NaPI-iv, [Miller, EA, et al. (2000) Plant Mol. Biol. 42:329-333] (SEQ ID N0:2). The<br><br>
amino acid sequences of NaPI-ii and NaPI-iv align to reveal a high level of identity between the two proteins. (See Fig. 1.) A single amino acid change is present within the predicted signal peptide. A second conservative amino acid change is present within the second repeat, which has been designated T1 in NaPI-ii (SEQ ID N0:3). Therefore the second repeat in NaPI-iv has been designated T5 (SEQ ID N0:4). The relationship between the functional domains of NaPI-ii and NaPI-iv is diagrammed in Fig. 2. A C-teminal non-repeated domain (CTPP) identical in amino acid sequence to that of NaPI-ii is found with NaPI-iv (SEQ ID N0:1, amino acids 374-397, SEQ ID N0:2, amino acids 268-281).<br>
[0006] A nucleotide sequence of cDNA encoding NaPI-ii has been disclosed in PCT Publication No. WO 94/138810, SEQ ID N0:1 thereof, the entire publication incorporated herein by reference, to the extent not inconsistent herewith. The NaPI-iv cDNA sequence SEQ ID N0:2, GenBank Accession No. AF105340, is essentially that of NaPI-ii except for two alterations that result in the two conservative amino acid changes shown in Fig. 1 and several silent changes having no effect on the translated amino acid sequence.<br>
[0007] Expression of both NaPI-ii and NaPI-iv results in a protein which is post-translationally processed to yield individual mature 6 kDa proteinase inhibitor (PI) proteins having the designated trypsin (T) or chymotrypsin (C) inhibitory activities. Post-translational glycosylation has not been observed following expression in plant cells. Unprocessed precursor Pi's retain the CTPP and are located outside the vacuole of the cell. Once the precursor protein is deposited in the vacuole, the C-terminal domain is rapidly removed and processing that yields individual 6 kDa Pi's occurs [Miller (1999) supra].<br>
[0008] The NaPI-ii precursor PI has been shown to adopt a circular structure by formation of disulfide bonds between the cys residues in the C2N (SEQ ID N0:1 or 2, amino acids 31-53) and C2c (SEQ ID N0:1, amino acids 344-373, SEQ ID N0:2, amino acids 228-2587) domains, [Lee (1999) supra].    The resulting product of<br><br>
cyclization of the precursor followed by post-translational proteolysis is a unique heterodimeric PI having chymotrypsin-inhibitor activity (C2).<br>
[00091 Like other members of the type two family, the N.alata Pi's inhibit the digestive proteases of several insect species [Heath, RL, et al. (1997) J. Insect Physiol. 43:833-842] and probably function to limit damage to floral tissues and leaves by insect pests. The Pi's significantly retard the growth and development of Helicoverpa punctlgera larvae when incorporated into artificial diets or expressed in the leaves of transgenic tobacco [Heath (1997) supra].<br>
[0010] Various strategies have been adopted for expressing more than one transgene in a single transgenic plant. One technique has been to transform individual parent plants each with a single transgene and then to combine the transgenes in a single plant by crossing the parents, [Zhu, Q. et al. (1994) Bio/Technology 12:807-812; Bizily, SP, et al. (2000) Nat. Biotechnol. 18:213-217]. The breeding can be complicated where individual transgenes are recombined at different loci. The method is not applicable for vegetatively propagated plants.<br>
[0011] Sequential single gene transformations have been carried out but have limited practical value because of limited availability of selectable markers for each transformation step.<br>
[0012] The use of multiple transgenes linked on the same vector each separately controlled by its own copy of the same promoter has resulted in unexpected transcriptional silencing. [Matzke, AJM, et al. (1998) Curr Opin. Plant Biol. 1:142-148] or non-uniform expression [Van der Elzen, PJM, et al. (1993) Phil. Trans. R. Soc. Land. B 342:271-278]. The use of different individual promoters to drive multiple linked transgenes appears feasible but expression is presumably subject to individual characteristics of each promoter.<br>
[0013] Several investigators have reported adaptation of virus systems for expressing a polyprotein followed by specific protease cleavage in cis to release<br><br>
individual proteins. (See, e.g. IVIarcos, JF, et al. (1994) Plant MoL Biol. 24:495-503; Beck von Bodman, S. et al. (1995) Blo/technology 13:587-591). The systems require introducing a viral protease to cleave the polyprotein with the possibility of undesired side effects of the introduced protease.<br>
[0014] Urwin, PE, et al. (1998) Planta 204:472-479 described a dual proteinase inhibitor construct joined by a protease-sensitive propeptide from Pisum sativum, expressed in Arabidopsis. Only partial cleavage of the expressed polyprotein was reported. Using a 20 amino acid long linkage sequence, termed 2A, from foot-and-mouth disease virus, Halpin, C, et al. (1999) Plant J. 17:453-459 described constructing a polyprotein having two reporter coding regions joined by 2A in a single open reading frame. The 2A linker was reported to mediate co-translational cleavage at Its own carboxy terminus by an enzyme-Independent reaction. Although expression of the polyprotein and cleavage did occur, one of the resulting protein products retained 19 amino acids of the 2A linker and the 20th was attached to the other protein.<br>
[0015] A similar result was described by Francois, lEJA, et al. (2002) Plant Physiol. 28:1346-1358, who joined coding regions of two proteins, DmAMPI, a plant defensin from seeds of Dahlia nrterckHand RsAFP2, a defensin from Raphanus sativus, using a propeptide of 16 amino acids from seeds of Impatiens balsamina. The propeptide of /, balsamina was obtained from a polyprotein precursor, IbAMP, described by Tailor, RA, et al. (1997) J. Biol. Chem. 272:24480-24487. The described polyprotein construct of DmAIVIPI and RsAFP2 was expressed and post-translationally cleaved in Arabidopsis; however, portions of the linking propeptide were found attached to the C- and N- temnini of the linked proteins, regardless of their orientation in the polyprotein construct relative to the linker.<br>
[0016] Using a composite linker of 29 amino-acids in length, Francois, I.F.I.A. et al. (2004) Plant Science 166:113-121 reported expression in Arabidopsis of DmAMPI and RsAFP2 as a polyprotein precursor. The precursor was processed to<br><br>
yield DmAMPI cross-reactive protein primarily in intracellular extracts and RsAFP2 cross-reactive protein primarily in extracellular fluid. The linker sequence was a composite of part of the /. balsamina linker and part of the foot-and-mouth disease 2A linker sequence. A recombination-based system for introducing a plurality of genes into a plant cell has been described by Chen, Q.-J., et al. (2006) Plant Mol. Biol. 62:927-936. Each gene has Its own promoter and terminator.<br>
SUMMARY<br>
[0017] Described herein is a multi-gene expression vehicle (MGEV) for concurrently expressing a plurality of genes in a plant cell, tissue or whole plant, under control of a single promoter. A MGEV can be constructed to express a linear polyprotein that lacks features necessary to cause the C-terminal and N-terminal ends to join together. The MGEV includes a single isolated polynucleotide whose sequence includes the following segments described by the function encoded by each segment: from 2 to 8 open reading frames (D2-B), each of which encodes a functional protein, and a plurality of linker segments (L1-7), each one situated between two D segments. The MGEV preferably Includes, in addition, a 5' terminal segment encoding an endoplasmic reticulum signal sequence (S) and a 3'-terminal segment encoding a C-terminal vacuole targeting peptide (V). Translation of a linear MGEV yields a linear polyprotein which is further processed by cleavage at the linker (L) segments, to separate the protein domains from one another. Optionally, in its circular form, the MGEV additionally includes segments encoding a first "Clasp" peptide (C2N) on the C-terminal side of S and a second "Clasp" peptide (C2c) on the N-terminal side of V. Preferably, the C2N and C2c proteins have secondary and tertiary structures that allow them to Interact to form a hetero-dimer that can be covalently linked together by post-translational formation of disulfide bonds, thereby forming a "circular" polyprotein (having a cyclic topology). In one embodiment, the cross-linked C2N-C2C dimer has activity as a chymotrypsin inhibitor (C2). The circular MGEV can have from 3-8 reading frames (D3.8) with linkers between each domain and each "clasp" peptide {L4-8).   Ultimately, the circular polyprotein is also<br><br>
cleaved at each L segment. In both linear and circular forms, the signal polypeptide (S) and the vacuole targeting peptide (V) function to control intracellular transport of the entire poiyprotein, prior to cleavage at L sites.<br>
BRIEF DESCRIPTION OF THE FIGURES<br>
[0018] Fig. 1 shows an amino acid alignment of NaPI-ii (SEQ ID N0:1) and NaPI-iv (SEQ ID N0:2).<br>
[0019] Fig. 2 is a diagram showing how expression of both NaPI-ii and NaPI-iv results in a precursor protein which is post-translationally processed to yield individual mature 6 kDa proteinase inhibitor proteins (arrowed). The proteins either have trypsin (T) or chymotrypsin (C) inhibitory activity. Amino acid sequences of T1 (SEQ ID N0:3) and T5 (SEQ ID N0:4) are shown. SP, signal peptide; CTPP, C-terminal propeptide; N-ter (C2N) and C-ter (C2c) are the clasp peptides that interact via disulphide bonds to fomn a two chain proteinase inhibitor (C2) of 6kDa.<br>
[0020]     Fig. 3 is a plasmid map of pHEX29 used in Example 1.<br>
The following abbreviations are used in all plasmid maps herein (Figs. 3, 5, 7, 9, 11,13,15, 17, 19, 21, 23, 25, 27, 29, 31 and 33):<br>
oriV - origin of replication<br>
ColEI ori - origin of DNA replication from Colicin El<br>
TDNA RB - right hand border of TDNA from Agrobacterium tumefaciens.<br>
Nos promoter - Nopaline synthase promoter from TDNA of A. tumefaciens.<br>
NPTII - neomycin phosphotransferase coding segment.<br>
Nos terminator- Nopaline synthase terminator from TDNA of A. tumefaciens.<br><br>
Disrupted lacZ - partial segment of (3-galactosidase gene of Eschericia coli.<br>
CaMV 35S promoter - promoter segment of the Cauliflower Mosaic Virus (CaMV) gene encoding CaMV 35S protein.<br>
Pot1A in MGEV - described herein.<br>
CaMV 35S terminator - terminator segment of the CaMV gene encoding CaMV 35S protein.<br>
M13 ori - origin of replication from Ml 3 bacteriophage coat protein.<br>
TDNA LB - left hand border of TDNA from A. tumefaciens.<br>
Arrows indicate direction of transcription.<br>
Unless described in detail herein, the abbreviated features are standard components well-known to those of skill in the art and described in standard textbooks. See, e.g., Molecular Cloning (2001) Sambrook J., and Russell, D.W., Cold Spring Harbor Press, Cold Spring Harbor, New York.<br>
[0021] Figs. 4A-4E provide data from a 4-domain MGEV for expression of two NaPIs and PotIA in cotton, as described in Example 1. Figure 4A is a diagram of the circular protein encoded by MGEV-5 and expressed in pHEX29 which has an endoplasmic reticulum signal sequence (stick), two 6kDa proteinase inhibitor domains (spheres), one PotIA domain (diamond) and a vacuolar targeting sequence (helix). A third proteinase inhibitor domain is represented by a sphere with 3 horizontal lines to illustrate the 3 disulphlde bonds that link the two peptides [N-ter (C2N) and C-ter (C2c)], that form the clasp. A linker peptide is indicated by a solid line connecting each protein domain. The predicted size of the unprocessed MGEV-5 product is 31.4 KDa minus the signal sequence. Figure 4B is a bar graph of data from ELISA detection of NaPIs in extracts from leaves of primary transgenic cotton<br><br>
lines from experiment CT89. Samples were diluted 1:5,000 and 1:20,000. Coker is a non-transgenic control. Figure 4C is a bar graph of data from ELISA detection of NaPIs in extracts from leaves of T2 plants of line 89.5,1. Samples were diluted 1:5,000. Coker is a non-transgenic control. NaPI standard is 2, 4 or 6 |jg of pure 6kDa NaPIs isolated from Nicotiana alata flowers. Figure 4D is a protein blot of leaf extracts prepared from primary transgenic cotton lines (T1) from experiments CT89 and CT90. Leaf proteins were extracted directly into NuPAGE LDS sample buffer (4x) (NOVEX), separated on a 4-12% Novex Bis-Tris SDS gel and transferred onto a 0.22 micron nitrocellulose membrane. The blot was probed with NaPI antibody. The precursor protein and 6kDa NaPI peptides are arrowed. Lane 1: 80 ng of purified NaPI, lane 2: 89.5, lane 3: 89.20, lane 4: 89,60, lane 5: 89,111, lane 6: 89.120, lane 7: 89.122, lane 8: 90.131, lane 9: untransformed Coker. Figure 4E is an immunoblot blot of extracts prepared from cotton leaves of T1 and T2 plants from selected lines from experiments CT89 and CT90. Proteins were precipitated with acetone prior to solubilisation in sample buffer, separated on a 4-12% Novex Bis-Tris SDS gel and transferred onto a 0.22 micron nitrocellulose membrane. The blot was probed with NaPI antibody. The precursor protein and 6kDa NaPI peptides (arrowed) were observed in both the primary lines (T1) and their progeny (T2). Processing intermediates can be observed in lanes 3 and 7. Lane 1: 150 ng purified NaPI, lane 2: 89.177 (T1), lane 3: 89.177 (T2), lane 4: 90.73 (T1), lane 5: 90.73 (T2), lane 6: 89.5 (T1), lane 7: 89.5 (T2), lane 8: untransformed Coker.<br>
[0022]     Fig. 5 is a plasmid map of pHEX56 used in Example 2.<br>
[0023] Figs. 6A-6D provide data based on use of a 3-domain linear MGEV for expression of NaPI and PotIA in cotton cotyledons, as described in Example 2. Fig. 6A is a diagram of the linear protein encoded by MGEV-8 and expressed in pHEX56 which has an endoplasmic reticulum signal sequence (stick), two 6kDa proteinase inhibitor domains (spheres), one PotIA domain (diamond) and a vacuolar targeting sequence (helix), A linker peptide is indicated by a solid line connecting each protein domain. The predicted size of the unprocessed MGEV-8 product is 25.4 kDa<br><br>
minus the signal sequence. Fig. 6B Is a bar graph of data from ELISA detection of NaPIs in extracts from cotton cotyledons after transient expression with pHEX56 or PBIN19 empty vector. Samples were diluted 1:1,000 and compared to various amounts of purified 6k Da NaPIs. Fig. 6C is a bar graph of data from ELISA detection of PotIA in extracts from cotton cotyledons after transient expression with pHEX56. Samples were diluted 1:20 and compared to purified Pot1A standards. Fig 6D is a protein blot of extracts prepared from cotton cotyledons after transient expression with pHEX56. Proteins were precipitated with acetone prior to solubllisation In sample buffer, separated on a 4-12% Novex Bis-Tris SDS gel and transferred onto a 0.22 micron nitrocellulose membrane. The blot was probed with NaPI antibody. Lane 1: 150 ng of purified NaPI, lane 2: seedling 1, lane 3: seedling 2, lane 4: seedling 3, lane 5: cotyledon sample transfected with pBIN19 empty vector. The precursor protein and 6kDa NaPI peptides (arrowed) were detected in all three seedlings infiltrated with Agrobacterium containing the pHEX56 construct.<br>
[0024]     Fig. 7 is a plasmid map of pHEX31  used in Example 3.<br>
[0025] Figs. 8A-8G provide data based on use of a 4-domain MGEV for expression of NaPI and mature NaDI in cotton, as described in Example 3. Fig. 8A is a diagram of the circular protein encoded by MGEV-6 and expressed in pHEX31 which has an endoplasmic reticulum signal sequence (stick), two 6kDa proteinase inhibitor domains (spheres), one NaD1 domain (triangle) and a vacuolar targeting sequence (helix). A third proteinase inhibitor domain is represented by a sphere with 3 horizontal lines to illustrate the 3 disulphide bonds that link the two peptides [N-ter (C2N) and C-ter (C2c)] that form the clasp. A linker peptide is indicated by a solid line connecting each protein domain. The predicted size of the unprocessed MGEV-6 product is 28.2 kDa minus the signal sequence. Fig. 8B Is a bar graph of data from ELISA detection of NaPIs in extracts from leaves of T2 plants of line 93.4. Samples were diluted 1:5,000. Coker Is a non-transgenic control. PBS-T is a negative control. NaPI standard is the positive control of purified 6kDa NaPI. Fig. 8C is a bar graph of data from ELISA detection of NADI in extracts from leaves of<br><br>
T2 plants of line 93.4 Samples were diluted 1:50. Fig. 8D is a bar graph of data from ELISA detection of NaPIs in extracts from leaves of T2 plants of line 93.279 Samples were diluted 1:1,000. Fig. 8E is a bar graph of data from ELISA detection of NAD1 in extracts from leaves of T2 plants of line 93.279. Samples were diluted 1:50. Fig. 8F is a protein blot of extracts prepared from cotton leaves of transgenic cottoa lines (T1 and T2) from experiment CT93. Proteins were separated on a 4-12% Novex Bis-Tris SDS gel and transferred onto a 0.22 micron nitrocellulose membrane. The blot was probed with NaPI antibody. Lane 1:150 ng purified NaPI, lane 2: 93.4.1 T2, lane 3: 93.36.2 T1, lane 4: 93.36.2 T2. Both the precursor protein and 6kDa NaPI peptides (arrowed) were present. Fig. 8G is a protein blot of extracts prepared from cotton leaves of transgenic cotton lines (T1 and T2) from experfment CT93. Proteins were separated on a 4-12% Novex Bis-Tris SDS gel and transferred onto a 0.22 micron nitrocellulose membrane. The blot was probed with NaDI antibody. Lanes 1 and 2: 93.4.1, lane 3: 50 ng mature NaD1, lane 4: 150 ng mature NaDI. NaDI is arrowed. A faint band of about 6 kDa was observed in lanes 1 and 2 confirming that the mature NaDI was present in transgenic line 93.4.1 and had been processed correctly.<br>
[0026]      Fig. 9 is a plasmid map of pHEX46 used in Example 4.<br>
[0027] Figs. 10A-10F provide data based on use of the MGEV for expression and targeting of GFP to the vacuole in cotton cotyledons and Nicotiana tabacum leaves, as described in Example 4. Fig. 10A is a diagram of the circular protein encoded by MGEV-7 and expressed in pHEX46 which has an endoplasmic reticulum signal sequence (stick), three 6kDa proteinase inhibitor domains (spheres), GFP (cylinder) and a vacuolar targeting sequence (helix). A linker peptide is indicated by a solid line connecting each protein domain. The predicted size of the unprocessed MGEV-7 product is 49.6 kDa minus the signal sequence. Fig. 10B is a bar graph of data from ELISA detection of NaPls in extracts from cotton cotyledons after transient expression with pHEX46 or BIN19 empty vector. Samples were diluted 1:1,000 and compared to purified 6kDa standards. Fig. IOC is a protein<br><br>
blot of extracts prepared from cotton cotyledons after transient expression with pHEX46. Proteins were precipitated with acetone prior to solubilisation in sample buffer, separated on a 4-12% Novex Bls-Tris SDS gel and transferred onto a 0.22 micron nitrocellulose membrane. The blot was probed with NaPI antibody. Lane 1: cotyledon sample transfected with pHEX46, lane 2: cotyledon sample transfected with pBiN19 empty vector. The 6kDa NaPI peptides (arrowed) were present in the cotyledon sample transfected with pHEX46. Fig. 10D shows protein blot of extracts prepared from Nicotiana benthamiana leaves after transient expression with pHEX46 (MGEV-7). Fig. 10D-1 and Fig. 10D-2 are the same protein blot containing 6kD NaPIs purified from N. alata flowers in lane 1 (NaPI) and an extract from N. benthamiana leaves after transient expression of pHEX46 (MGEV-7) in the second lane. Fig. 10D-1. NaPI antibodies bound to the 6l<da pis in lane and to a protein of the expected size for mgev-7 arrowed leaf extracts. fig. is blot from after stripping reprobing with gfp antibody. antibody did not bind but thus has both pi domains domain. are second that was probed antibodies before it stripped reprobed napi bacterially expressed one an extract n. benthamiana leaves transient expression phex46 lane. bound same extracts expressing mgev-7. also unprocessed as well potential processing intennediate about confirming this highlighted by consistent release free mgev benthamiana. micrograph showing></da><br>
GFP from pHEX46 in the epidermal cells of cotton leaves. The GFP fluorescence is located in the vacuoles (arrowed). GFP fluorescence examined with an Olympus BX50 fluorescence microscope. Fig. 10F is a micrograph showing transient expression of GFP from pHEX45 in the epidermal cells of cotton leaves. The GFP fluorescence is extracellular (arrowed). GFP fluorescence examined with an Olympus BX50 fluorescence microscope.<br>
[0028]     Fig. 11 Is a plasmid map of pHEX55 used in Example 5.<br>
[0029] Figs. 12A-12E provide data based on use of a 6-domain MGEV for the expression of NaPI, NaD1 and Pot 1A in cotton cotyledons, as described in Example 5. Fig. 12A is a diagram of the circular protein encoded by MGEV-9 and expressed in pHEX55 which has an endoplasmic reticulum signal sequence (sticl
 <br>
lane 2: cotyledon sample transfected with pHEX55, lane 3: untransformed Coker. The 6kDa NaP! peptides (arrowed) were present in the cotyledon sample transfected with pHEX55. Several processing intermediates ranging from about 17 kDa to 38 kDa were also detected.<br>
[0030]     Fig. 13 is a plasmid map of pHEX45 used in Example 6.<br>
[0031] Figs. 14A-14F provide data based on use of the MGEV for expression and targeting of GFP to the extracellular space in Nicotiana benthamiana leaves, as described in Example 6. Fig. 14A is a diagram of each of the proteins encoded by the four constructs. The endoplasmic reticulum signal sequences are represented by a stick, the 6kDa proteinase inhibitor domains including the clasp domain are spheres, GFP is a cylinder and the vacuolar targeting sequence (V) is represented as a helix. A linker peptide is indicated by a solid line connecting each protein domain. The predicted size of the unprocessed encoded proteins minus the signal sequence is given next to the cartoons. Figs. 14B,-E are micrographs showing transient expression of GFP from pHEX45 (IV1GEV 10) and CI (Fig 14A). In the absence of V the GFP from both constructs is directed outside the cell. GFP fluorescence was examined using a Leica TCS SP2 confocal laser-microscope.<br>
Fig. 14B - Transient expression of pHEX45 in epidermal cells.<br>
Fig. 14C - Transient expression of pHEX45 in mesophyll ceils.<br>
Fig. 14D - Transient expression of control gene construct CI in epidermal cells.<br>
Fig. 14E - Transient expression of control gene construct C1 in mesophyll cells.<br>
Fig. 14F - Protein blots of extracts prepared from Nicotiana bentliamiana leaves after transient expression with pHEX45 (MGEV-10), pHEX46 (MGEV-7), CI(S-GFP) and C2 (S-GFP-V). Blots A and B are probed with<br><br>
GFP antibody. Lane 1. Positive control, Bacterially expressed GFP. Lanes "2-5 are extracts from leaves after transient expression of C1, C2, pHEX 46 (!\/IGEV.7) and pHEX45 (MGEV-10) respectiveiy. All constructs produced a protein of 28i<da that bound the gfp antibody. phex and phex45 also produced a protein of about was expected size ivigev- mgev-10 reacted with gfp-antibody. corresponding to napi antibody blot c showing this has both domains.></da>
[0032]      Fig. 15 is a plasmid map of pHEX42 used in Example 7.<br>
[0033] Figs. 16A-16E provide data based on use of a 4-domain MGEV for expression of NaPI and NaDI (with CTPP) in cotton cotyledons, as described in Example 7. Fig. 16A is a diagram of the circular protein encoded by MGEV-11 and expressed in pHEX42 which has an endoplasmic reticulum signal sequence (stick), three 6kDa proteinase inhibitor domains (spheres), one NaD1 domain (triangle) + CTPP tail (helix) and a vacuolar targeting sequence (helix). A linker peptide is indicated by a solid line connecting each protein domain. The predicted size of the unprocessed MGEV-11 product is 31.8 kDa minus the signal sequence. Fig. 16B is a bar graph of data from ELISA detection of NaPI In extracts from cotton cotyledons after transient expression with pHEX42 or empty vector. Samples were diluted 1:100. Fig. 16C is a bar graph of data from ELISA detection of NaD1 in extracts from cotton cotyledons after transient expression with pHEX42. Samples were diluted 1:5,000. Fig. 16D is a protein blot of extracts prepared from cotton cotyledons after transient expression with pHEX42. Proteins were precipitated with acetone prior to solubilisation in sample buffer, separated on a 4-12% Novex Bis-Tris SDS gel and transferred onto a 0.22 micron nitrocellulose membrane. The blot was probed with NaPI antibody. Lane 1: cotyledon sample transfected with pHEX42, lane 2: cotyledon sample transfected with pBIN19 empty vector, lane 3: blank, lane 4: 200 ng purified NaPI.   The precursor and 6kDa NaPI peptides (arrowed) were<br><br>
present in the cotyledon sample transfected with pHEX42. Fig. 16E is a protein blot of extracts prepared from cotton cotyledons after transient expression with pHEX42. Proteins were precipitated with acetone prior to solubilisation in sample buffer, separated on a 10 - 20% Novex Tricine SDS gel and transferred onto a 0.22 micron nitrocellulose membrane. The blot was probed with NaDI antibody. Lane 1: cotyledon sample transfected with pHEX42, lane 2: cotyledon sample transfected with pBIN19 empty vector, lane 3: blank, lane 4: 150 ng purified NaD1. The precursor and 6kDa NaD1 (arrowed) were present in the cotyledon sample transfected with pHEX42.<br>
[0034]     Fig. 17 is a plasmid map of pHEX33 used in Example 8.<br>
[0035] Figs. 18A-18C provide data based on use of a 6-domain MGEV for expression of NaPI and PotIA in cotton cotyledons. Fig. 18A has a diagram of the circular protein MGEV-12 encoded by pHEX33 which has an endoplasmic reticulum signal sequence (stick), three 6kDa proteinase inhibitor domains (spheres), two PotIA domains (diamond) and a vacuolar targeting sequence (helix). A linker peptide is indicated by a solid line connecting each protein domain. The predicted size of the unprocessed MGEV-12 product is 40.4 kDa minus the signal sequence. Fig. 18B is a bar graph of data from ELISA detection of NaPIs in extracts from cotton cotyledons after transient expression with pHEX33. Samples were diluted 1:1,000. Fig. 18C is a bar graph of data from ELISA detection of Pot 1A in extracts from cotton cotyledons after transient expression with pHEX33, Samples were diluted 1:20.<br>
[0036]      Fig. 19 is a plasmid map of pHEX39 used in Example 9.<br>
[0037] Figs. 20A-20C provide data based on use of a 5-domain MGEV for expression of NaPI, mature NaD1 and NaD2 in cotton cotyledons. Fig. 20A is a diagram of the circular protein MGEV-13 encoded by pHEX39 which has an endoplasmic reticulum signal sequence (stick), three 6kDa proteinase inhibitor<br><br>
domains (spheres), one NaD2 domain (triangle), one NaD1 domain (triangle) and a vacuolar targeting sequence (helix). A linker peptide is indicated by a solid line connecting each protein domain. The predicted size of the unprocessed MGEV-13 product is 34 KDa minus the signal sequence. Fig. 20B is a bar graph of data from ELISA detection of NaPIs in extracts from cotton cotyledons after transient expression with pHEX39. Samples were diluted 1:1,000. Fig. 20C is a bar graph of data from ELISA detection of NaD1 in extracts from cotton cotyledons after transient expression with pHEX39. Samples were diluted 1:100,<br>
[0038]      Fig. 21 is a plasmid map of pHEX48 used in Example 10.<br>
[0039] Figs. 22A-22D provide data based on use of a 4-domain linear MGEV for expression of NaPI and PotIA in cotton cotyledons. Fig, 22A Is a diagram of the linear protein encoded by MGE\/-14 and expressed in pHEX48 which has an endoplasmic reticulum signal sequence (stick), two 6kDa proteinase Inhibitor domains (spheres), two PotIA domains (diamond) and a vacuoiar targeting sequence (helix). A linker peptide is indicated by a solid line connecting each protein domain. The predicted size of the unprocessed MGEV-14 product is 34.5 kDa minus the signal sequence. Fig. 22B is a bar graph of data from ELISA detection of NaPIs in extracts from cotton cotyledons after transient expression with pHEX48. Samples were diluted 1:1,000. Fig. 22C is a bar graph of data from ELISA detection of Pot 1A In extracts from cotton cotyledons after transient expression with pHEX48. Samples were diluted 1:20, Fig. 22D is a protein blot of extracts prepared from cotton cotyledons after transient expression with pHEX48. Proteins were precipitated with acetone prior to solubilisatlon In sample buffer, separated on a 4-12% Novex Bis-Tris SDS gel and transferred onto a 0,22 micron nitrocellulose membrane. The blot was probed with NaPI antibody. Lane 1: 150 ng of purified NaPI, lane 2: cotyledon sample transfected with pHEX48, lane 3: cotyledon sample transfected with pBIN19 empty vector. The 6kDa NaPI peptides are arrowed. The NaPI peptides and several processing intermediates were detected in the cotyledon tissue transfected with pHEX48.<br><br>
[0040]      Fig. 23 is a plasmid map of pHEX47 used in Example 11.<br>
[0041] Figs. 24A-24D provide data based on use of a 3-domain linear MGEV for expression of NaPI and mature NaD1 in cotton cotyledons. Fig. 24A is a diagram of tine linear protein encoded by MGEV-15 and expressed in pHEX47 wliich tias an endoplasmic reticulum signal sequence (stick), two 6i<da proteinase inliibitor domains one nad1 domain and a vacuolar targeting sequence rml peptide is indicated by solid line connecting each protein domain. the predicted size of unprocessed mgev-15 product kda minus signal sequence. fig. bar graph data from elisa detection napis in extracts cotton cotyledons after transient expression with phex47. samples were diluted nadi blot prepared proteins precipitated acetone prior to solubilisation sample buffer separated on novex bis-tris sds gel transferred onto micron nitrocellulose membrane. was probed napi antibody. lane ng purified cotyledon transfected phex47 untransformed coker. peptides present></da>
[0042]      Fig. 25 is a plasmid map of pHEX35 used in Example 12.<br>
[0043] Figs. 26A-26C provide data based on use of a 2-domain linear MGEV for expression of PotIA in cotton cotyledons. Fig. 26A is a diagram of the linear protein encoded by MGEV-16 and expressed in pHEX35 which has an endoplasmic reticulum signal sequence (stick), a PotIA prodomain (rectangle) and hvo PotIA domains (diamond). A linker peptide is indicated by a solid line connecting each protein domain. The predicted size of the unprocessed MGEV-16 product is 19.4 kDa minus the signal sequence. Fig. 268 is a bar graph of data from ELISA detection of PotIA in extracts from cotton cotyledons after transient expression with<br><br>
pHEX35 and pHEX6. Samples were diluted 1:50. pHEX6 is the same as construct pHEX35 except that there is only one copy of the Pot1A gene. In the 3 seedlings assessed, expression of Pot1A was higher when the Pot1A dimer was used (pHEX35) compared to a single Pot1A domain (pHEX6). pHEX6 is disclosed in published patent application (WO2004/094630). Fig. 26C is a protein blot of extracts prepared from cotton cotyledons after transient expression with pHEX35. Proteins were precipitated with acetone prior to solubilisation in sample buffer, separated on a 4-12% Novex Bis-Tris SDS gel and transferred onto a 0.22 micron nitrocellulose membrane.The blot was probed with PotiA antibody. Lane 1: cotyledon sample (seedling 2) transfected with pHEX35, lane 2: cotyledon sample transfected with pBIN19 empty vector, lane 3: 100 ng purified Pot 1A. The mature Pot 1A (arrowed) was produced in the cotyledon seedling transfected with pHEX35.<br>
[0044]      Fig. 27 is a plasmid map of pHEX41 used in Example 13.<br>
[0045] Figs. 28A-28F provide data based on use of a 2-domain linear IVIGEV for expression of NaDI in cotton cotyledons. Fig. 28A is a diagram of the linear protein encoded by MGEV-17 and expressed in pHEX41 which has an endoplasmic reticulum signal sequence (stick), one 6kDa proteinase inhibitor domain (sphere), one NaD1 domain (triangle) and the CTPP tail that enables targeting to the vacuole (helix). A linker peptide is indicated by a solid line connecting each protein domain. The predicted size of the unprocessed IVIGEV-I? product is 15.8 kDa minus the signal sequence. Fig. 288 is an ELISA detection of NaDI in extracts from cotton cotyledons after transient expression with pHEX41 and pHEX3. Samples were diluted 1:500. pHEX3 is the same as pHEX41 except that it does not contain the NaPI domain. In the 2 seedlings assessed, expression of NaDI was higher when expressed with the NaPI domain (pHEX35) compared to expression of NaDI alone (pHEX6). pHEX3 is disclosed in U.S. Patent 6,031,087. Fig. 28C is a bar graph of data from ELISA detection of NaPI in extracts from cotton cotyledons after transient expression with pHEX41. Samples were diluted 1:1,000. Fig. 28D is a bar graph of data from ELISA detection of NaD1 In extracts from cotton cotyledons after transient<br><br>
expression with pHEX41. Samples were diluted 1:500. Fig. 28E is a protein blot of extracts prepared from cotton cotyledons after transient expression with pHEX41, Proteins were precipitated with acetone prior to solubilisation in sample buffer, separated on a 4-12% Novex Bis-Tris SDS gel and transferred onto a 0.22 micron nitrocellulose membrane. The blot was probed with NaPI antibody. Lane 1; cotyledon sample (seedling 2) transfected with pHEX41, lane 2: cotyledon sample transfected with pBIN19 empty vector, lane 3: blank, lane 4: 200 ng purified NaPI. The 6kDa NaPI peptides (arrowed) were present in the cotyledon sample transfected with pHEX41. Fig. 28F is a protein blot of extracts prepared from cotton cotyledons after transient expression with pHEX41. Proteins were precipitated with acetone prior to solubilisation in sample buffer, separated on a 4-12% Novex Bis-Tris SDS gel and transferred onto a 0.22 micron nitrocellulose membrane. The blot was probed with NaDI antibody. Lane 1: cotyledon sample (seedling 2) transfected with pHEX41, lane 2: cotyledon sample transfected with pBIN19 empty vector, lane 3: blank, lane 4: 150 ng purified NaDI. The precursor and 6kDa NaDI (arrowed) were present in the cotyledon sample transfected with pHEX41.<br>
[0046]      Fig. 29 is a plasmid map of pHEX52 used in Example 14.<br>
[0047] Fig. 30 provides data based on use of a 2-domain linear MGEV for expression of NaD2 and NaDI in cotton cotyledons. Fig. 30A Is a diagram of the linear protein encoded by MGEV-18 and expressed in pHEX52 which has an endoplasmic reticulum signal sequence (stick), one NaD2 domain (triangle), one NaDI domain (triangle) and the CTPP tall that enables targeting to the vacuole (helix). A linker peptide is indicated by a solid line connecting each protein domain. The predicted size of the unprocessed MGEV-18 product Is 14.7 kDa minus the signal sequence. Fig. 30B is a bar graph of data from ELISA detection of NaDI in extracts from cotton cotyledons after transient expression with pHEX52.<br>
[0048]      Fig. 31 is a plasmid map of pHEXSI used in Example 15.<br><br>
[0049] Figs. 32A-32B provide data based on use of a 2-domaln linear MGEV for expression and targeting of NaD2 and NaD1 to the extracellular space in cotton cotyledons. Fig. 32A is a diagram of the linear protein encoded by MGEV-19 and expressed in pHEX51 which has an endoplasmic reticulum signal sequence (stick), one NaD2 domain (triangle) and one NaDI domain (triangle). A linker peptide is indicated by a solid line connecting each protein domain. The predicted size of the unprocessed MGEV-19 product is 11.1 kDa minus the signal sequence. Fig. 32B is a bar graph of data from ELISA detection of NaD1 in extracts from cotton cotyledons after transient expression with pHEX51. Samples were diluted 1:100.<br>
[0050]     Fig. 33 is a plasmid map of pHEX58 used in Example 16.<br>
[0051] Figs. 34A-34C provide data based on use of a 2-domaln linear MGEV for expression and targeting of GUS to the vacuole In cotton cotyledons. Fig. 34A is a diagram of the linear protein encoded by MGEV-20 and expressed In pHEX58 which has an endoplasmic reticulum signal sequence (stick), two 6kDa proteinase inhibitor domains (spheres), one GUS (square) and a vacuolar targeting sequence (helix). A linker peptide is Indicated by a solid line connecting each protein domain. The predicted size of the unprocessed MGEV-20 product is 84.8 kDa minus the signal sequence. Fig. 34B is a bar graph of data from ELISA detection of NaPI in extracts from cotton cotyledons after transient expression with pHEX58. Samples were diluted 1:1,000. Fig. 34C is a protein blot of extracts prepared from cotton cotyledons after transient expression with pHEX58. Proteins were precipitated with acetone prior to solubilisation in sample buffer, separated on a 4-12% Novex Bis-Tris SDS gel and transferred onto a 0.22 micron nitrocellulose membrane. The blot was probed with NaPI antibody. Lane 1: cotyledon sample (seedling 2) transfected with pHEX58, lane 2: cotyledon sample transfected with pBIN19 empty vector, lane 3: 150 ng purified NaPI peptides. The NaPI peptides (arrowed) were produced in the cotyledon seedling transfected with pHEX58.<br><br>
DETAILED DESCRIPTION OF THE INVENTION<br>
[0052] Various MGEV structures are detailed herein and in tlie following examples. A general IVIGEV structure encoding a circular polyprotein (MGEV-P) is diagrammed as follows:<br>
S-C2N-(LjDk)m-LjC2c-V<br>
where each capital letter symbolizes a polynucleotide encoding a segment of amino acids designated according to its function, thus: S is a polynucleotide segment with an open reading frame encoding a signal peptide; Dk is a polynucleotide segment with an open reading frame encoding a functional protein (hereinafter a "Domain") wherein k represents an ordinal number to identify any single functional Domain selected from a group of domains having from 3 to m members and at least one of D does not encode a type two protease inhibitor; Lj is a polynucleotide segment with an open reading frame encoding a linker polypeptide where Lj is a ordinal number to identify each single linker (L) selected from a group having from 3 to m+1 members; C2N is a polynucleotide segment with an open reading frame encoding a N-terminal clasp peptide; C2c is a polynucleotide with an open reading frame encoding a C-termina) clasp peptide; V is a vacuolar targeting peptide; m is a cardinal number from 3-8; and S,C2N,L,D,C2C and V are all in the same reading frame same as each other. As an example, a MGEV encoding 3 functional domains (D) can be diagrammed as shown above, where m Is 3, k is 1, 2 or 3, j is 1, 2, 3, or 4, In another linear embodiment, described below, clasp proteins are omitted or truncated. In the absence of a clasp peptide, there is no requirement for any of D to encode a type two proteinase inhibitor.<br>
[0053] Lj encodes a linker amino acid sequence as described herein. Each Lj can have the same or a different sequence. A generic linker amino acid sequence is givenatSEQIDN0:17.<br><br>
[0054] In a plant cell, the MGEV encoded protein (MGEV-P) undergoes several steps of post-translational processing. Tliese include intracellular transport to the endoplasmic reticulum, provided the leader (S) is present, followed by removal of S and subsequent transport to an intracellular storage vacuole provided the vacuolar targeting sequence (V) is present. V is removed in the vacuole. If C2N and C2c are present, the ends of the MGEV-P become joined together to form a closed loop, diagrammed as follows:<br><br>
where C2N,LI-4, D1-3, C2c, and V are as described supra.<br>
[0055] Post-translational proteolysis cleavage at each linker and between C2c and V results in release of Di, D2, D3 and, in one embodiment, C2, as separate proteins. Expression of the MGEV thereby results In concurrent expression of at least three separate proteins at least one of which is not a type two proteinase Inhibitor, from a single promoter.<br>
[0056] A circular MGEV can encode from 3 to 8 functional domains (D), concurrently expressed. Concurrent expression is defined herein to mean the intracellular synthesis of a plurality of functional proteins from a single transcript. Concurrent expression is especially useful when it is desired or necessary to produce and accumulate large amounts of proteins in a plant cell, for example, plant protectant proteins, or economically significant proteins, or when it is advantageous to control the relative amounts of expressed proteins, or for expression of certain proteins, such as cysteine-rich peptides, that are normally expressed poorly in plant cells. When the MGEV includes a vacuole targeting peptide (V), the concurrently expressed proteins are accumulated in a storage vacuole in the cell, which can serve two purposes: (1) to provide the proteins in concentrated form to maintain an effective dose of plant protectant in the event of pathogen attack, or to ease<br><br>
purification of an economically valuable protein; and (2) to sequester othe	rwise toxic proteins whicii can confer added pest resistance and economic value to a plant expressing such proteins. V can be combined with any domain to be expressed, most conveniently at the 3'-end of MGEV. More than one V can be included if desired. In the absence of V, proteins released from MGEV-P by proteolysis can be exported from the cell.<br>
[0057] The expressed components of an MGEV are described herein in greater detail.<br>
[0058] The protein domains (D) encoded by open reading frames of the MGEV nucleotide sequence can, in principle, be any protein. No upper size limit is known for a protein expressible as a component of a MGEV. Exemplified herein are data demonstrating concurrent expression of individual domains encoding proteins ranging from about 5kDa to greater than 65kDa. Practical considerations known to those skilled in the art can be considered when choosing proteins appropriate for expression using an MGEV. For example, very large proteins may be expressed individually more efficiently, rather than as part of a MGEV. Certain proteins may sterically interfere with cyclization under certain circumstances. Each protein domain (D) is connected to a linker peptide (L) by peptide bonds at the N-terminal and C-terminal amino acids of the domain.<br>
[0059] It is presently believed that efficiency of post-translational peptide cleavage that liberates individual protein domains from the MGEV-P is maximized when the N-and C-termini of each domain and connecting linkers are exposed by the protein conformation to the aqueous environment on the surface of the protein, rather than sequestered internally within the protein. Therefore, candidate proteins for expression as part of the MGEV-P preferably have exposed N- and C-terminal amino acids.<br>
[0060]      Examples of proteins which can be expressed using an MGEV include<br><br>
(without limitation) potato type one Pi's, potato type two Pi's, plant defensins, animal defensins, proteinaceous toxins, chimeric and fusion proteins, as well as indicator proteins such as Green Fluorescent Protein (GFP), 28kDa, and beta-glucuronidase (GUS), 68lkDa. Examples of protein-coding domains that can be expressed in the MGEV include plant protection proteins such as potato proteinase inhibitors of type one (Pot 1A), plant seed defensins, plant floral defensins, insect-toxic peptides such as scorpion toxin. Bacillus thuringiensis toxins, heat shock proteins, Bowman-Birk trypsin inhibitors, and cystatins and indicators such as green fluorescent protein (GFP) and beta-glucuronidase (GUS). Proteins of economic value for purposes other than plant protection can be expressed using the MGEV, taking advantage of high expression levels, including anti-microbial peptides, antibody fragments and the like suitable for medical use. Also large hetero-dimeric or hetero-multimeric proteins are especially suitable for IVIGEV expression where concurrent and correctly proportional expression is desired. At least one protein encoded by a MGEV is not a type two PI. The MGEV is particularly useful for expression of proteins that may be toxic to the cell in which they are expressed, by providing for transport to, and sequestration in, a storage vacuole within the plant cell.<br>
[0061] A linker (L) is a short peptide positioned between each domain that separates each adjacent domain and exposes a peptidase-sensitive site for post-translational cleavage between individual domains. The amino acid sequence -EEKKN (SEQ ID N0:5) - is an example of a linker peptide. Other amino acid sequences can serve as linkers, for example, sequences where E and K are substituted by similar amino acids, such as D (asp) or R (arg) or N (asn) is substituted by a Q (gin). A consensus linker sequence can be expressed as X1X2X3X4X5 where Xi is E (glu) or D (asp), X ais E (glu) or D (asp), X3 is K (lys) or R (arg), X4 is K (lys) or R (arg) and X5 is N (asn) or Q (gin) (SEQ ID N0:17). The linker provides a highly hydrophilic segment that exposes a proteolytic cleavage site (N -X) to the outer surface of MGEV-P. Any short highly hydrophilic peptide can serve as a linker in the MGEV-P. The linker peptides described herein are advantageous<br><br>
because post-translational processing of domains joined by a linker can result in removal of the entire linker in transgenic plants. (See Heath, R.L. et al., (1995) Eur. J. Biochem. 230:250-257).<br>
[0062] The leader peptide, also refen-ed to as a signal peptide (S), is a sequence of about 10 to about 30 mostly hydrophobic amino acids which serves a transport function for intracellular transport. Many signal peptides are known in the art. Any known signal peptide can be used in the MGEV-P, as well as modifications thereof wherein homologous amino acids are substituted.<br>
[0063] The vacuole targeting peptide (V) is located at the C-terminus of the MGEV-P. A variety of vacuolar targeting determinants are known to exist in plant cells, see, e.g. Maruyama et al. Plant Cell (2006) 18:1253-1273. Suitable vacuolar targeting peptides can be chosen from a wide variety of known candidates. Also, a suitable V segment need not be placed at the C-terminus of the MGEV, but could, in principle be located elsewhere in the sequence; for example attached to the N-terminus of C2N, between S and C2N. In one embodiment, a suitable sequence can be one which binds to the known BP-80 vacuolar sorting receptor. Any such vacuole targeting sequence that binds BP-80 or a homolog thereof can be used as a component of the MGEV-P. Another example of a suitable vacuole targeting sequence is shown in Miller, et al. supra, Fig. 1, amino acids 258-281 of the NaPI-iv sequence (SEQ ID N0:2), Other examples include the C-termina! propeptide of NaD1 (SEQ ID N0:14, amino acids 27-105 and the PotIA prodomain, SEQ ID N0:20),<br>
[0064] The clasp segments, C2N and C2c are represented herein by amino acids 30-48 (C2N) and 228-267 (C2c) SEQ ID N0:6. The folded configuration of peptides C2N and C2c is such that they readily bind to one another, and the heterodimer formed by the binding is then stabilized covaiently by formation of inter-peptide disulfide cross-links. The cross-linked [C2N : C2c] protein has chymotrypsin activity and is designated simply as C2 herein. In the MGEV-P structure, fomriation of C2<br><br>
results in cyclization of MGEV-P with a C-terminal extension, the vacuole targeting peptide, V. A clasp structure can be formed using any of the type 2 inhibitors regardless of protease specificity, because of the high degree of homology among them. Deletion of the four amino acid sequence PRNP (or PKNP in the case of T5) which is common to these inhibitors will create the appropriate N-termlnal and C-terminal segments of a clasp peptide. Formation of a cyclic structure is not necessary for activity of MGEV-P. A cyclic structure of MGEV-P is considered advantageous for efficient intracellular transport. A further advantage of the cyclic configuration is that the additional inhibitor thereby formed is a useful plant protectant against insect damage.<br>
[0066] The total or partial deletion of C2N and C2c can prevent formation of a cyclic structure and result in a linear configuration. The Invention includes both linear and cyclic configurations of MGEV-P. A linear MGEV is advantageous whenever a large protein, a mix of large and small proteins, or a protein lacking a compact tertiary structure is to be expressed. In certain circumstances expression levels can be increased by use of a linear MGEV-P instead of the cyclic form. Targeting to the endoplasmic reticulum by S and vacuolar targeting by V can occur as previously described. A linear MGEV can have as few as two domains. Post-translational processing of linear MGEV-P can occur as described, with release of individual active domains (Dk). A diagram of a linear MGEV-P having 3 protein domains lacking C2N and C2c is shown, wherein non-specific peptides PN and Pc are provided in place of C2N and C2c, respectively.<br>
S-PN-(LjDk)mLjPc-V<br>
where j is 1,2 or 4, kis 1, 2 or 3, mis 3.<br>
[0066] PN and PC can be modified or partially deleted versions of C2N and C2c, respectively. Preferably, C2N and C2c are entirely deleted, such that a linear 3-domain MGEV has the diagram structure:<br><br>
where j and k are 1 or 2 and m is 2.<br>
[0067] As noted previously, V need not be at the C -terminus, but could be located elsewhere in the sequence, for example between S and D.<br>
[0068] The linear MGEV-P can have up to eight functional protein domains, at least one of which is not a type two proteinase inhibitor. As with cyclic MGEV-P, the linear form can be exported from the cell by deletion of the vacuole targeting sequence, V.<br>
[0069] Constructing a MGEV can be carried out by known methods of combining the nucleic acid segments in the designated order, by DNA synthesis, or a combination of both methods. A convenient method is to employ components of naturally-occurring iype two PI multimers, such as NaPI-lv from N. alata, SEQ ID N0:2 [Miller, (2000) supra, GenBank accession number AF105340]. One or more open reading frames encoding a functional protein domain of interest that is not a type two PI can be inserted together with appropriate linkers into the naturally-occurring multimer, thereby increasing the number of expressed domains, or pre¬existing domains can be deleted, followed by insertion of desired domain-coding segments to keep the total number of domains unchanged as long as all coding segments remain in the same reading frame from one to the next. Examples of protein-coding domains that can be expressed in the MGEV include plant protection proteins such as potato proteinase inhibitors of type one, for example as disclosed in International Publication No. WO 2004/094630, including PotIA exemplified herein, plant seed defensins, plant floral defensins, insect-toxic peptides such as scorpion toxin. Bacillus thuringiensis toxins, heat shock proteins, Bowman-Birk trypsin inhibitors, and cystatins and indicators such as green fluorescent protein (GFP) and beta-glucuronldase (GUS). Proteins of economic value for purposes other than plant protection can be expressed using the MGEV, taking advantage of high<br><br>
expression levels, including anti-microblal peptides, antibody fragments and the like suitable for medical use. Also large hetero-dimeric or hetero-multimeric proteins are especially suitable for MGEV expression where concurrent and correctly proportional expression is desired.<br>
[0070] The following Examples demonstrate construction of MGEV's encoding plant-protective proteins, plant transformation with MGEV, transgenic plants containing and expressing the MGEV and protection from plant pests due to expression of non- Potato Type Two proteins encoded within a MGEV, and MGEVs encoding a mix of large and small proteins. These Examples are presented to illustrate, but not limit, the invention as claimed.<br>
[0071] A MGEV can be expressed in plants or plant cells after being incorporated into a plant transformation vector. Many plant transformation vectors are well known and available to those skilled in the art, e.g., BIN19 (Bevan, (1984) NucI, Acid Res. 12:8711-8721), pBI 121 (Chen, P-Y, et al., (2003) Molecular Breeding 11:287-293), pHEX 22 (U.S. Patent 7,041,877), and vectors exemplified herein. Such vectors are well-known in the art, often termed "binary" vectors from their ability to replicate in a bacteria such as Agrobacterim tumefaciens and in a plant cell. A typical plant transformation vector, such as exemplified herein, includes genetic elements for expressing a selectable marker such as NPTII under control of a suitable promoter and terminator sequences, active In the plant cells to be transformed (hereinafter "plant-active" promoter or terminator) a site for inserting a gene of interest, including a MGEV under expression control of suitable plant-active promoter and plant-active terminator sequences and T-DNA borders flanking the MGEV and selectable marker to provide integration of the genes into the plant genome.<br>
[0072] Plants are transformed using a strain of A. tumefaciens, typically strain LBA4404 which is widely available. After constructing a plant transformation vector that carries a MGEV encoding the desired proteins, the vector is used to transform an A. tumefaciens strain such as LBA4404. The transformed LBA4404 is then used<br><br>
to transform the desired plant cells using an art-known protocol appropriate for the plant species to be transformed. Standard and art-recognized protocols for selecting transformed plant cells, multiplication and regeneration of selected cells are employed to obtain transgenic plants. The examples herein further disclose methods and materials used for transfomriation and regeneration of cotton plants, as well as transgenic cotton plants transformed by and expressing a variety of MGEVs, A MGEV can be transferred into plant cells by any of several known methods besides those exemplified herein. Examples of well-known methods include microprojectile bombardment, electroporation, and other biological vectors including other bacteria or viruses.<br>
[0073] The MGEV can be used for multigene expression in any monocotylodenous or dicotyledonous plant. Particularly, useful plants are food crops such as corn (maize) wheat, rice, bariey, soybean and sugarcane and oilseed crops such as sunflower and rape. Particularly useful non-food common crops include cotton, flax and other fiber crops. Flower and ornamental crops include rose, carnation, petunia, lisianthus, lily, iris, tulip, freesia, delphinium, limonium and pelargonium.<br>
[0074] Techniques for introducing vectors, chimeric genetic constructs and the like into cells include, but are not limited to, transformation using CaCIa and variations thereof, direct DNA uptake into protoplasts, PEG-mediated uptake to protoplasts, microparticle bombardment, electroporation, microinjection of DNA, microparticle bombardment of tissue explants or cells, vacuum-infiltration of tissue with nucleic acid, and T-DNA-mediated transfer from Agrobacten'um to the plant tissue.<br>
[0075] For microparticle bombardment of cells, a microparticle is propelled into a cell to produce a transformed cell. Any suitable ballistic cell transformation methodology and apparatus can be used in performing the present invention. Exemplary procedures are disclosed in Sanford and Wolf (U.S.  Patent Nos.<br><br>
4,945,050, 5,036,006, 5,100,792, 5,371,015). When using ballistic transformation procedures, tlie genetic construct can incorporate a plasnnici capable of replicating in the cell to be transformed.<br>
[0076] Examples of microparticles suitable for use in such systems include 0.1 to 10 Mm and more particularly 10.5 to 5 pm tungsten or gold spheres. The DNA construct can be deposited on the microparticle by any suitable technique, such as by precipitation.<br>
[0077] Plant tissue capable of subsequent clonal propagation, whether by organogenesis or embryogenesis, can be transformed with a MGEV of the present invention and a whole plant generated therefrom, as exemplified herein. The particular tissue chosen will vary depending on the clonal propagation systems available for, and best suited to, the particular species being transformed. Examples of tissue targets Include leaf disks, pollen, embryos, cotyledons, hypocotyls, megagametophytes, callus tissue, existing meristematic tissue (e.g. apical meristem, axillary buds, and root merlstems), and induced meristem tissue (e.g. cotyledon meristem and hypocotyl meristem).<br>
[0078] The regenerated transformed plants can be propagated by a variety of means, such as by clonal propagation or classical breeding techniques. For example, a first generation (or T1) transformed plant may be selfed to give a homozygous second generation (or T2) transformant and the T2 plants further propagated through classical breeding techniques.<br>
[0079] Accordingly, this aspect of the present invention, insofar as it relates to plants, further extends to progeny of the plants engineered to express the nucleic acid of the MGEV as well as vegetative, propagative and reproductive parts of the plants, such as flowers (Including cut or severed flowers), parts of plants, fibrous material from plants (for example, cotton) and reproductive portions including cuttings, pollen, seeds and callus.<br><br>
[0080] Another aspect of the present invention provides a genetically modified plant cell or multicellular plant or progeny thereof or parts of a genetically modified plant capable of producing a protein or peptide encoded by the MGEV as herein described wherein said transgenic plant has acquired a new phenotypic trait associated with expression of the protein or peptide.<br>
[0081] MGEV structures and MGEV expression vectors exemplified herein are listed in Table 2, together with the number of the Example where they are described. Sequence ID listings are listed in Table 3.<br>
EXAMPLE 1<br>
Construction and expression of an MGEV having one type one PI and 3 potato<br>
type twoPI's.<br>
[0082] The MGEV described in this example (MGEV-5) SEQ ID N0:6 has the structure diagrammed as:<br><br>
wherein Li-4 encodes the linker amino acid sequence -EEKKN- SEQ ID N0:5, Di encodes a potato type two trypsin inhibitor, T1 SEQ ID N0:3; SEQ ID N0:1 amino acids 112-164; Da encodes a potato type one chymotrypsin inhibitor, potato Pot 1A SEQ ID N0:11, (also SEQ ID N0:5, bases 352-376); Da encodes a Type Two chymotrypsin inhibitor, C1 SEQ ID N0:2 amino acids 54-106; C2N SEQ ID N0:1 amino acids 31-48 and C2cSEQ ID N0:1 amino acids 344-373 encode peptides that interact with each other to form a heterodimer C2 stabilized by disulfide crosslinks, the cross-linked protein having potato type two chymotrypsin inhibitor activity. S encodes a signal peptide and V encodes a vacuole translocation peptide.<br>
[0083] Amino acid sequences encoded by the above-identified segments are described in the following sources:<br><br>
For S-C2N-LIDI, amino acids 1-29(S); 30-48 (C2N); 112-164 (Di) of SEQ ID N0:2<br>
For L2, amino acids EEKKN, SEQ iD N0:5<br>
For Dz, (SEQ ID N0:11) [see also Intemationai Publication No. WO2004/094630 (Nov. 4, 2004) SEQ ID N0:81, incorporated herein by reference to the extent not inconsistent herewith]<br>
For L3D3, amino acids 49-106 of SEQ ID NO:2<br>
For L4C2cV, SEQ ID N0:2 amino acids 223-257 (L4C2c); and 258-281 (V)<br>
[0084] A multipurpose vector, pRR19 was constructed. The vector contained sequences obtained from NaPI-iv SEQ ID N0:2 and NaPI-ii SEQ ID NO: 1 [Miller (2000) supra] plus restriction sites for insertion of new genes. The entire MGEV-1 sequence was assembled in consecutive order into pRR19.<br>
[0085] The vector pRR19 was designed to allow convenient modular assembly of linkers (L) and open reading frames (D) into a MGEV having the desired combination of components. As step 1, polymerase chain reaction (PCR) was used to amplify the respective N- and C- terminal end segments of NaPI-iv, specifically S-C2N (SEQ ID N0:2 amino acids 1-48) and C2c-V (SEQ ID N0:2 amino acids 228-281), and to provide Xhol restriction sites. The Xhol restriction sites were provided to permit joining of desired segments between the terminal segments, such that the amplified segments had the diagram structure S-C2NLI- Xhol and Xhol -C2c-V, respectively. After cutting and ligation, the segment S-C2N-Li-Xho1-C2c-V was cloned into the pGEM T-Easy (Promega, Madison, Wl) vector.<br>
[0086] Any desired DMA segment having Xhol sites at its N and C termini could then be inserted into the Xhol site of the resulting vector.<br>
[0087]     As the step 2, in parallel preparations, DNA encoding the T1 of NaPI-ii<br><br>
(SEQ ID N0:1, amino acids 112-164) (to be in position Di in IVIGEV-S) and the DNA encoding the CI domain of NaPI-iv (SEQ ID N0:2, amino acids 54-106) (to be position Da in MGEV-5) were PCR-amplified with restriction sites added as diagrammed:<br>
Xhol - T1 - Li - Xbal, and Xbal- CI- Li - Xho1.<br>
[0088] Each of the constructs was separately cloned into pGEIVI T-easy vectors, digested with Xbal and Xho1 and purified.<br>
[0089] The modified T1 and CI domains from the preceding step were combined in a DNA ligation reaction mixture with Xho1-digested product of the first step. The ligation mixture was transformed into E. co//XL1-Blue cells (Stratagene, LaJolla, CA) and restriction digests and sequencing were carried out to confinn the desired orientation of and order of the proteinase inhibitor domains. The predicted ligation reactions were DNA segments encoding the following components:<br>
S - C2N- L1 - Xhol... Xho1- T1 - L1 - Xbal.... Xbal- C1 - LiXhol... Xhol- C2c- V<br>
(Step 1 product)      (Step 2 product)	(Step 2 product)     (Step 1 product)<br>
[0090] The ligation product, as verified by electrophoresis of restriction digests and sequence analysis, was<br>
S - C2N- Lr Xho1- T1- L1 Xbal- CI - L1 - Xhol- C2c- V<br>
[0091] The ligation product contained a unique Xba1 site (underlined) into which could be inserted any desired coding sequence provided with Xbal restriction sites at both ends. The vector having the described construct was designated pRR19.<br>
[0092] For the Da domain, the DNA coding for Pot 1A, previously described, was provided with a linker (L) at the C-terminal-coding end, followed by Xbal restriction sites at the 3' and 5' ends.    Insertion at the Xba1 site of pRR19 resulted in a<br><br>
construct that was then inserted into pAI\/l9 {pMAQ was modified from pDHA, Tabe et al„ Journal of Animal Science, 73: 2752-2759, 1995) to produce l\/IGEV-5. insertion in pAIVIQ resulted in the attachment of the 35S CaMV promoter at the 3' end and the 35S CaMV terminator at the 5' end. MGEV-5 was then inserted into pBINIS at the EcoR1 site resulting in vector pHEX 29, diagrammed in Fig. 3. See also Fig. 4A.<br>
[0093] The use of restriction sites in IVIGEV-5 could be avoided, if desired, by using DNA synthesis to make the disclosed MGEV-5 sequence of Table 1. See also SEQ ID N0:6 (DNA sequence) and SEQ ID NO: 12 (deduced amino acid sequence).<br>
Table 1. Signal peptide (bases 7-93), N-terminal clasp peptide domain (bases 94-150 (C2N) and C-terminal clasp peptide 778-864), (C2c), T1 domain (bases 172-330), Pot 1A (bases 352-576), CI domain (bases 598- 756) and vacuole targeting sequence (bases 865-939).<br><br><br><br>
[0094] Seeds of Gossyipium hirsutum cultivar Coker 315 were surface sterilized in sodium fiypochlorite (2% available chlorine) for 60 min followed by several washes in sterile water. The sterilized seed were sown onto Cotton Seed Medium (CSM) [0.22% w/v MS (Murashige and Skoog salt mixture Austratec M524), 0.05% w/v B5 vitamins (Sigma G1019), 1.5% w/v glucose (Austratec G386), 0.2% w/v gellan gum Gelrite, trademark of Merck &amp; Co., (Phyto Technology Laboratories), pH 5.8] and incubated at 30°C in the dark for 10 days. A. tumefaciens (LBA4404) transformed with the pHEX29 construct was grown overnight in 25 ml LB medium supplemented with the antibiotic kanamycin (50 jjg/mL) at 28'C. The absorbance at 550 nm was measured and the cells were diluted to 2 X 10° cells per ml in MS liquid media (0.43% w/v Murashige and Skoog basal salts, pH 5.8). Cotton hypocotyls were cut into 1.5-2 cm pieces and mixed briefly (0.5-3 min) in the diluted Agrobacterium culture. The explants were drained and transferred to medium 1 (0.43% w/v Murashige and Skoog salt mixture, 0.1% v/v Gamborg's B5 vitamin solution (Sigma), 0.1 g/L myo-inositol, 0.9 g/L MgCl2, (hexahydrate), 1.9 g/L potassium nitrate, 0.2% w/v Gelrite, 3% w/v glucose, pH 5,8) overlayed with sterile filter paper and incubated for 3 days at 26**C under lights.<br><br>
[0095] Following co-cultivation, explants were transferred to medium 2 (medium 1 plus 0.1 mg/L Icinetin, 0.1 mg/L 2,4-D, 500 mg/L carbenicillin, 35 mg/L kanamycin) and maintained at 30°C under low light. After 4 weeks explants were transferred to medium 3 (medium 1 plus 500 mg/L carbenicillin, 25 mg/L kanamycin) and maintained at 30°C under low light. Explants and callus were sub-cultured every 4 weeks on medium 3 and maintained at 30°C under low light. Embryos were excised from the tissue and gemninated in medium 4 (1.2 mM CaCl22H20, 5.0 ml\/l KNOs, 2.0 mM l\/IgS047H20, 3.0 mM NH4NO3, 0.2 mM KH2PO4, 4 pM nicotinic acid, 4 pM pyridoxine HCI, 4 pM thiamine HCI, 30 pM H3BO3, 30 pM MnS04H20, 9 pM ZnS047H20, 1.5 pM Kl, 0.9 pM Na2Mo042H20, 0.03 pM CuS045H20, 0.03 pM C0CI26H2O, 15 pM FeNaEDTA, 0.5% w/v glucose, 0.3% w/v gellan gum Gelrite, pH 5.5) and maintained at SO'C under high light.<br>
[0096] Germinated embryos were then transferred to Magenta boxes containing medium 4 and maintained at 30'C under high light. Once a plant has formed a good root system and produced several new leaves it was transferred to soil in pots and acclimatised in a growth cabinet at 28°C and then grown in a glasshouse at (27-29°C day, 20-24'C night).<br>
PCR analysis<br>
[0097] DNA isolation: Cotton leaf discs (0.5 - 0.7 cm ) were sampled from the 2nci fully expanded leaf, avoiding vein tissue. Extraction solution (100 pi) from the REDExtract-N-Amp Plant PCR kit (Sigma) was added to each leaf disc ensuring the tissue was fully submerged. Samples were heated at 95'C on a heat block for 10 minutes before vortexing. Dilution solution (100 pi , Sigma) was added and the sample was vortexed thoroughly and placed on ice.<br>
[0098] The PCR reaction mix consisted of the following components: 10 pi PCR ready mix (REDExtract-N-Amp, Sigma) 0.8 pi fonward primer, 0.8 pi reverse primer, 2.8 pi H2O, 4 pi DNA extract (from above).   PCR conditions were 94°C, 4 min,<br><br>
followed by 33 cycles of 94°C 30 sec, 62C 30 sec, 72°C 1 min followed by 72°C for 10 min. Samples were stored at 4'C.<br>
Primers:<br>
nptll forward: GTGGAGAGGCTATTCGGCTATGAC - SEQ ID N0:7<br>
nptll reverse: CGGGTAGCCAACGCTATGTCC - SEQ ID N0:8<br>
StPot 1A forward: GCTCTAGAAAGGAATCGGAATGTGAATC - SEQ ID N0:9<br>
StPot 1A reverse: GCTCTAGAATTCTTGTTTTCTTCAGCCACCCTAGGAATTTG - SEQ ID NO:10<br>
Detection of NaPI and StPotIA in transgenic cotton<br>
ELISA<br>
[0099] Protein extract: leaves were excised from plants grown either in the growth cabinet or in the glasshouse. The tissue (100 mg) was frozen in liquid nitrogen and ground in a mixer mill (Retsch MM300) for 2 x 15 sec at frequency 30. 1 mL of 2% insoluble PVP (Polyclar)/PBS/0.05% Tween 20 was added prior to vortexing for 20 sec. The samples were centrifuged for 10 min and the supernatant was collected.<br>
[00100] Coat ELISA plate (Nunc Maxisorp #442404) with 100 μL/well of primary antibody in PBS.<br>
[00101] lOOng/well of anti-NaPI (polyclonal antibody was made by a standard method to purified NaPI peptides isolated from stigmas) or anti-Pot 1A (antibody made to Pot 1A that was expressed as a dimer with CI in E. coli and then cleaved and separately purified). Incubate overnight at 4"'C in a humid box. Wash plates 2 min X 4 with PBS/0.05 % Tween 20. Block plate with 200 μL/well 3 % BSA (Sigma A- 7030: 98% ELISA grade) in PBS. Incubate for 2 hr at 25'C. Wash plates 2 min x 4 with PBS/0.05 % Tween 20. The anti-NaPI antibody binds to the T and C protease<br><br>
inhibitors of N. alata.<br>
[00102] Apply 100 μL/well of cotton protein extracts (diluted in ^BS/0.05 % Tween 20). Incubate 2hr at 25°C. Wash plates 2 min x 4 with PBS/0.05 % Tween 20. Apply 100 pL/well of secondary antibody in PBS (50ng/well biotin-labelled NaPl antibody, 200ng/well biotin-labelled Pot 1A antibody). Incubate for 1 hr at 25'C. The biotin labelled antibody is prepared using the EZ-link Sulfo-NHS-LC-biotinylation kit (Pierce). Use 2 ml of protein A purified antibody and 2 mg of the biotin reagent.<br>
[00103] Wash plates 2 min x 4 with PBS/0,05 % Tween 20. Apply 100 pL/well NeutriAvldin HRP-conjugate (Pierce #31001; 1:1000 dilution; 0.1μL/well) in PBS. Incubate for 1 hr at 25°C.<br>
[00104] Wash plates 2 min x 4 with PBS/0.05 % Tween 20, followed by 2 min x 2 with H2O. Just before use, prepare substrate by dissolving 1 ImmunoPure OPD tablet (Pierce #34006) in 9 mL H2O, then add 1 mL stable peroxide buffer (10x, Pierce #34062). Add 100 pL/well substrate. Incubate at 25'C until colour develops. Stop reaction with 50 pL 2.5 M sulfuric acid. Measure absorbance at 490 nm in plate reader (Molecular Devices, Milenla Kinetic Analyzer).<br>
Immunoblot analysis<br>
[00105] Leaves were excised from plants grown either in the growth cabinet or in the glasshouse. Leaf tissue (100 mg) was frozen in liquid nitrogen and ground to a fine powder in a mixer mill (Retsch MM300), for 2 x 15 sec at frequency 30. The powder was added to 2 x sample buffer (300 pi, Novex NuPAGE LDS sample buffer, 10% v/v p- mercaptoethanol), vortexed for 30 sec, boiled for 5 min and then centrifuged at 14,000 rpm for 10 min and the supernatant retained for SDS-PAGE. Alternatively, the powder was added to 1 ml acetone, vortexed thoroughly and centrifuged at 14,000 rpm (18,000g) for 2 min and the supernatent discarded. The pellet was resuspended in 300 pi of IP lysis buffer (50mM Tris pH 8, 5mM EDTA, 150mM NaCI, 0.1% Triton X-100) with 2% Polyclar AT (water-soluble polyvinyl<br><br>
polypyrrolidine) by vortexing thoroughly and supernatant was collected after centrifugation at 14,000 rpm for 10 mIn. For analysis by SDS-PAGE, 30 ΜL of sample in 1 X sample buffer (Novex NuPAGE LDS sample buffer) and 5% v/v p-mercaptoethanol was used.<br>
[00106] Extracted leaf proteins were separated by SDS-PAGE on preformed 4-12% w/v polyacrylamide gradient gels (Novex, NuPAGE bis-tris, MES buffer) for 35<br>
min at 200V in a Novex X Cell II mini-cell electrophoresis apparatus. Prestained<br>
molecular weight markers (Novex SeeBlue Plus 2) were included as a standard. Proteins were transferred to nitrocellulose membrane (Osmonics 0.22 micron NitroBInd) for 60 min at 30V using the Novex X Cell mini-cell electrophoresis apparatus in NuPAGE transfer buffer with 10% v/v methanol. After transfer, membranes were incubated for 1 min in isopropanol, followed by a 5 min wash in TBS.<br>
[00107] The membrane was blocked for 1 h in 3% w/v BSA at RT followed by incubation with primary antibody overnight at RT (NaPI antibody: 1:2000 dilution in TBS/1 %BSA of Img/ml stock. Pot 1A antibody: 1:1000 in TBS/1 %BSA of Img/ml stock). The membrane was washed 5x10 min in TBST before incubation with goat anti-rabbit IgG conjugated to horseradish peroxidase for 60 min at RT (Pierce, 1:100,000 dilution in TBS). Five further 10 min TBST washes were performed before the membrane was incubated with the SuperSignal West Pico Chemiluminescent substrate (Pierce) according to the Manufacturer's instructions. Membranes were exposed to ECL Hyperfilm (Amersham).<br>
Results<br>
[00108] From 2 experiments (CT 89 and CT 90) we produced 86 potential transgenic plants.  All plants were screened by PCR using the npt primers and the<br>
StPotIA primers. Plants positive for npt II were assessed for NaPI protein expression<br><br>
by ELISA. 38 plants were expressing detectable levels of NaPI (Figure 4).<br>
[00109] Line 89.5.1 was selfed and the T2 progeny seed grown and the plants assessed for NaPI expression by ELISA. 20 of the 27 plants (74%) were expressing NaPI and 7 plants (26%) were null segregants (Fig. 4C) demonstrating that the genes had been transferred to the next generation in a heritable manner.<br>
[00110] Immunoblot analysis of selected lines using the NaPI antibody confirmed that the precursor protein and the processed peptides were present (Fig. 4D and 4E). However, detection of PotI was unsuccessful suggesting that detection sensitivity in the assay was not sufficient.<br>
[00111] The results demonstrate that a MGEV encoding four peptides, at least one of which is not a type 2 protease inhibitor, can be constructed using conventional methods and used to successfully transform a plant (cotton) of a different species than that from which any of the component DNA segments were derived. The encoded protein is expressed and post-translationally processed to yield component peptides of the expected size.<br>
EXAMPLE 2<br>
Construction and expression of a linear MGEV having one type one PI and 2<br>
potato type two Pis<br>
[00112] The MGEV described in this example (MGEV-8) has the structure diagrammed as:<br>
S-D1L1D2L2D3L3-V<br>
where D, is T1 of NaPI-ii - SEQ ID N0:2. aa 112-164<br>
D2 is potato Pot 1A - SEQ ID N0:11<br>
D3 is CI or amino acids 200 to 252 of SEQ ID N0:2<br><br>
Li and L2 and L3 are each EEKKN (SEQ ID N0:5)<br>
S is the signal peptide of NaPI-iv- SEQ ID N0:2, aa 1-29<br>
and V is the vacuole targeting peptide of NaPI-iv - SEQ ID N0:2, aa 258-281<br>
A linear MGEV (MGEV-8) (Fig. 6A) was constructed as follows. The signal sequence of NaPI-iv SEQ ID N0:2, aa 1-29 was PCR-amplified with a Bam H1 site at the 5' end and a Xho 1 site at the 3' end. The vacuole targeting peptide of NaPI-iv was PCR-amplified with a Xho 1 site at the 5' end and a Sal 1 site at the 3' end. These DNA fragments were ligated together into pAM9 cut with Bam H1 and Sal 1 (see Example 1).<br>
[00113] The Xho 1-flanked Tl-Xba 1-C1 fragment was cut from the multipurpose vector pRR20 (see Example 3) and ligated into the S-Xho 1-V construct described above, resulting in a S-Xho 1-T1-Xba 1-C1-Xho 1-V construct. This linear multipurpose vector was designated pSPI.<br>
[00114] The mature domain of potato Pot 1A (see Example 1) was PCR-amplified with an EEKKN linker sequence (SEQ ID NO: 5) at the 3' end and with Xba 1 sites at both ends. This was then ligated into the Xba 1 site of pSPI to produce MGEV-8 (Fig. 6A). MGEV-8 was inserted into pBIN19 to produce the vector pHEX 56, diagrammed in Fig. 5.<br>
Transient expression in cotton cotyledons<br>
[00116] pHEX 56 was Introduced into A. tumefaciens and the expression of T1, CI and Pot 1A was determined by a transient assay with cotton cotyledons.<br>
Bacterial "lawns" of the Agrobacterium were spread on selective plates and grown in the dark at 30 °C for 3 days. Bacteria were then resuspended to an OD600 of 1,0 in infiltration buffer (10 mM magnesium chloride and 10 pM acetosyringone (0.1 M stock in DMSO)) and incubated at room temperature for 2-4 h.  Cotton plants were<br><br>
grown for 8 days in a controlled temperature growth cabinet (25 °C, 16 h/8 h light/darl
Results<br>
[00116] NaPI (Fig. 6B) and Pot 1A (Fig. 6C) were detected by ELISA in cotton cotyledons. Immunoblot analysis using the NaPI antibody confirmed that the precursor protein and the processed peptides were present (Fig. 6D).<br>
[00117] The results confinn previous conclusions from Example 1 and demonstrate, in addition, expression of Pot1A. The results also demonstrate that cyclization of a primary MGEV expression product is not required for processing to yield predicted component peptides.<br>
EXAMPLE 3<br>
Construction and expression of an MGEV having one defensin and 3 potato<br>
type two Pis<br>
Note: In Examples 3-16, linker peptides (L) are omitted from the IVIGEV diagram in order to simplify the diagram.<br>
[00118] The MGEV described in this example (MGEV-6) has the structure diagrammed as: <br><br>
(See also Fig. 8A).<br>
[00119] MGEV-6, expressing a defensin and 3 potato type two Pi's, was constructed essentially as described for MGEV-5 (Example 1) except that a modified multipurpose vector (pRR20) was used and a defensin coding sequence was inserted instead of Pot 1A. The defensin was NaDI as described in U.S. Patent 7,041,877, and herein SEQ ID N0:14, amino acids 26-72, having a mature defensin domain but lacking the C-termlnal acidic peptide tail, and without the N-terminal signal peptide.<br>
[00120] The modified multipurpose vector (pRR20) is the same as the multipurpose vector (pRR19) described in Example 1, except that the codon encoding N in the EEKKN linker (SEQ ID N0:5) (Li) of the Xho1-T1-LrXbal DNA fragment was changed from AAT to AAC SEQ ID N0:12. This deleted an undersired Eco R1 restriction site that was present in pRR19.<br>
[00121] NaD1 DIMA was ligated into the Xba 1 site of pRR20, then excised with Bam HI and Sal 1 and the complete fragment inserted into pAIV19 to produce MGEV-6. MGEV-6 was then inserted into pBINIS to produce the vector pHEX31, diagrammed in Fig. 7.<br>
Transformation of cotton<br>
[00122] Cotton transformation with pHEX31 was carried out as described in Example 1,<br>
Protein Detection<br>
[00123] Protein expression was determined by ELISA as described in Example 1. The primary NaDI antibody and the secondary NaD1-biotln antibody were used at 50 ng/well.<br>
[00124]    Immunoblot analysis was carried out as described In Example 1 with the<br><br>
modification described in Example 2. The primary NaD1 antibody was diluted 1:1,000 dilution from a Img/ml stock and the secondary antibody (goat anti-rabbit IgG conjugated to horseradish peroxidase) was used at a 1:50,000 dilution.<br>
Results<br>
[00125] From one experiment (CT 93) 88 potential transgenic plants were produced. All plants were screened by PCR using the nptll primers and primers specific for NaDI. 57 plants were positive for the presence of the nptll gene, with 33 of these plants also carrying the NaD1 gene. PCR positive plants were assessed for NaPI and NaD1 protein expression by ELISA. 13 primary transgenic plants were expressing detectable levels of NaPI and NaDI.<br>
[00126] Three transgenic lines (93.4, 93.36 and 93.279) were selected for further study. The primary transgenic lines were selfed and the T2 seed collected. T2 plants from two of these lines (93.4 and 93.279) were assessed for NaPI expression (Figs. 8B, 8D) and NaDI expression (Figs. 8C, 8E) by ELISA. Both lines produced a segregating population consistent with genes being transferred in a Mendelian manner.<br>
[00127] Immunoblot analysis of lines 93.4 and 93.36 using the NaPI antibody confirmed that the precursor protein and the processed peptides were present (Fig. 8F). Further analysis of line 93.4 with the NaDI antibody confirmed that the mature NaD1 protein was present (Fig. 8G), although at low levels.<br>
[00128] The results demonstrate utility of MGEV for simultaneously expressing a protein other than a protease inhibitor (NaD1, a defensin).<br>
EXAMPLE 4<br>
Construction and expression of an MGEV having one GFP and 3 potato type<br>
two Pi's<br><br>
[00129]   The  MGEV described in this example (MGEV-7) has the structure diagrammed as:<br>
r<br>
S-C2N-T1-GFP-C1-C2C-V<br>
(See also Fig. 10A)<br>
[00130] MGEV-7 has a similar structure to MGEV-5 (Example 1) except that a DNA sequence encoding a Green Fluorescent Protein (GFP) was inserted in place of Pot 1A. The GFP is a soluble, highly fluorescent variant of green fluorescent protein (GFP) for use in higher plants (Davies, SJ and Vierstra, RD: Plant Mol. Biol. 36(4): 521-528 (1998). The DNA was obtained from TAIR (the Arabidopsis information resource) (SEQ ID N0:13). Sequence information is available from Genbank at accession number U70495, and herein at SEQ ID NO: 13.<br>
[00131] For construction of MGEV-7, a third multipurpose vector (pRR21) was used. This was made in the same way as pRR20 except that the DNA encoding the CI domain of NaPI-iv was PCR-amplified with an extra EEKKN linker sequence (SEQ ID N0:5) at the 3' end resulting in an Xbal-L-CI-L-Xhol DNA fragment. pRR21 has the following structure: S-C2N-L-Xho1-T1-L-Xba1-L-C1-L-Xho1-C2c-V. In addition this construct was inserted into pAM9 before additional insertions were made. The DNA sequence encoding GFP was PCR-amplified with Xbal ends (no 3' linker sequence) and inserted into the Xba1 site between T1 and C1 of pRR21 to produce MGEV-7. MGEV-7 was inserted into pBIN19 to produce the vector pHEX 46, diagrammed in Fig. 9.<br>
Transient expression in tobacco leaves<br>
[00132] pHEX 46 was introduced into A. tumefaciens and the expression of T1, C1 and GFP was determined by a transient assay with tobacco leaves. The method was that essentially described in Example 2 for cotton cotyledons except that<br><br>
Nicotlana benthamiana plants were grown for 5 weeks in a controlled temperature growth cabinet (25 °C, 16 h/8 h light/dark cycle). The underside of leaves (4-6 nodes from the top, 6-10 cm in maximum width) was.infiltrated by gently pressing a 1 mL syringe and filling the leaf cavity with the Agrobacterium suspension. Four to six infiltrations were made on each leaf. Plants were grown for a further 4 days. The infiltrated areas were then cut out, weighed and frozen in liquid nitrogen. Protein expression was determined by immunoblots as described in Example 1.<br>
Transient expression in cotton cotyledons<br>
[00133] Expression of pHEX 46 was also determined in a transient assay with cotton cotyledons as previously described in Example 2.<br>
Protein detection<br>
[00134]    Expression of NaPI was determined by ELISA as described in Example 1.<br>
[00135] Immunoblot analysis was carried out as described in Example 1 with the modification described in Example 2.<br>
IVIicroscopy<br>
[00136] Three days after infiltration with A. tumefaciens the N. benthamiana and cotton plants were placed in the dark for 24h. The infiltrated leaf areas were then removed and epidermal peels (~5mm^) were prepared. Small pieces (1-2mm^ of the epidermal or mesodermal tissue were placed on a glass slide with water as a mounting medium. A cover slip was placed over the top and sealed with hot wax. The sections were examined for GFP fluorescence using an Olympus BX50 fluorescence microscope. A W1B filter (excitation range 460-490nm) was used for fluorescence excitation and a long pass filter which detects signals at 515nm plus was used for emission. GFP fluorescence was also examined using a Leica TCS SP2 confocal laser-microscope. The Argon laser excitation wavelength was 488nm;GFP emission was detected with the filter set for FITC (505-530nm).<br><br>
[00137] Several transient assays with both tobacco leaves and cotton cotyledons were conducted. NaPI was detected by ELISA in cotton cotyledons (Fig. 10B). Immunoblot analysis using the NaPI antibody confirmed that the processed peptides were present in cotton cotyledons (Fig. 10C).<br>
[00138] Immunoblot analysis of tobacco leaf extracts after transient expression confimied that the GFP protein was present (Fig. 10D). The GFP and the NaD1 antibodies both bound to a protein of about 50kDa which is consistent with the expected size of the precursor protein encoded by pHEX 46. The GFP antibody also highlighted a protein of ~28kDa which is the same size as bacterially expressed GFP and thus represents GFP that has been proteolytically excised from the precursor encoded by pHEX 46.<br>
[00139] GFP produced from transient expression of MGEV-7 in the epidermal cells of cotton leaves was located in the vacuole (Fig. 10E). This contrasted to GFP fluorescence produced from a construct (MGEV-7A) that was identical to MGEV-7 except the vacuole targeting peptide (V) was deleted (see example 7). Transient expression of MGEV-7A resulted in an extracellular location for the GFP fluorescence (Fig. 10F).<br>
[00140] The results demonstrate that proteins of disparate sizes can be expressed as a polyprotein using a MGEV, and correctly processed after translation to yield individual protein components. In this example, 4 proteins ranging in size from ~6kDa to ~28kDa were effectively expressed together and correctly processed. A single vacuole targeting sequence resulted In transfer of each expressed protein to the cell vacuole prior to processing. The use of GFP in a MGEV is therefore a convenient means to indicate intracellular location of proteins co-expressed in a MGEV.<br><br>
EXAMPLE 5<br>
Construction and expression of an MGEV having one defensin, one type one<br>
Pi and 3 potato type two Pis<br>
[00141]   The  MGEV described in this example (MGEV-9) has the structure diagrammed as:<br>
S - C2N - T1 - NaDI - Pot 1A - Pot 1A - CI - C2c - V (See Fig. 12A).<br>
[00142] MGEV-9 expressing six proteins, a defensin, two potato type one Pi's and 3 type two Pi's was constructed using the following method, NaD1 was prepared as per Example 3. The Pot 1A dimer was constructed by splice overlap PCR. The first Pot 1A was PCR-amplified with a 5' Xbal site and a 3' linker sequence. The second Pot 1A was PCR-amplified with linker sequences at both ends and a 3' Xbal site. The two PCR fragments were annealed to each other and extended for 8 cycles; outer primers were then added to PCR-amplify the dimer sequence. The NaDI and Pot 1A dimer fragments were inserted into the Xba 1 site of pSPI (Example 2) in a 3 way ligation. The new larger fragment (T1-NaD1-Pot lA-Pot 1A-C1) was cut at the Xho 1 sites to produce MGEV-9. MGEV-9 was inserted into pBIN19 to produce the vector pHEX55, diagrammed in Fig. 11.<br>
Transient expression In cotton cotyledons<br>
[00143] Expression of pHEX55 was detemriined in a transient assay with cotton cotyledons as previously described in Example 2.<br>
Protein detection<br>
[00144]    Expression of NaPI was determined by ELISA as described in Examples<br><br>
1,2 and 3,<br>
[00145] Immunoblot analysis was carried out as described in Example 1 with the modification described in Example 2.<br>
Results<br>
[00146] NaPI (Fig. 12B), NaD1 (Fig. 12C) and Pot 1A (12D) were detected by ELISA in cotton cotyledons. Immunoblot analysis using the NaPI antibody confimried that the precursor protein and the processed NaPI 6 kDa peptides were present (Fig. 12E).<br>
[00147] The results demonstrate simultaneous expression and correct processing of several different proteins in a 6-domain circular MGEV.<br>
EXAMPLE 6<br>
Construction and expression of an MGEV that targets proteins to the extracellular space in plant tissues.<br>
[00148]    The MGEV described in this example has the structure diagrammed as:<br>
S-C2N-T1 -GFP-C1-C2c<br>
(SeeFig. 14A, MGEV10).<br>
[00149] This MGEV (MGEV-10) was essentially the same as MGEV-7 (Example 4) except that it did not have the NaPI vacuole targeting peptide (V) and the multipurpose vector pRR20 was used (Example 3). pRR20 was PCR-amplified using a reverse primer which excluded the vacuole targeting peptide (V). Xbal-flanked GFP was then ligated into the Xbal site. Details of the GFP are given in Example 4. The fragment (S - C2N - T1 - GFP - C1 - C2c) was then inserted into pAM9 to<br><br>
produce MGEV-10. MGEV-10 was then inserted into pBIN19 to produce the vector pHEX45, diagrammed in Fig. 13.<br>
Transient expression assays<br>
[00150] Expression of pHEX45 was determined in transient assays with tobacco leaves and cotton cotyledons as described In Example 4. Protein expression was determined by immunoblots as described in Example 4. Two non-MGEV constructs C1 and C2 (Fig. 14A) were used as controls. These constructs employed the same promoters and temiinators as the MGEV constructs and were cloned into the same vectors for expression in plant cells. The coding sequence of C1 contained GFP with the signal sequence (S) from the MGEV. The second control construct (C2) encoded GFP with the endoplasmic reticulum signal sequence (S) and the vacuolar targeting sequence (V) from the MGEV. The location of the GFP in the plant tissue was confirmed by microscopy as described in Example 4.<br>
Results<br>
[00151] Several transient assays with tobacco leaves were conducted. Immunoblot analysis of tobacco leaf extracts after transient expression confirmed that the GFP protein was produced from both the control constructs (CI and C2) and was the same size as the 28kDa bacterlally expressed GFP (Fig 14F). Additionally, the C2 construct produced another slightly larger protein that corresponds to GFP plus the vacuolar targeting sequence (V), The GFP-antibody detected proteins of ~50kDa and 28kDa in extracts from leaves that were expressing MGEV-7 and MGEV-10 . The SOkDa protein also bound to the NaPI antibody as expected for the unprocessed product encoded by the MGEV. The presence of the 28kDa protein which corresponds to free GFP is consistent with processing of the linker in the MGEV to release the individual PI and GFP domains.<br>
[00162] GFP produced from transient expression of MGEV-7 in the epidemnal cells of cotton leaves was located in the vacuole (Example 4, Fig. 10E). This<br><br>
contrasted to GFP fluorescence produced from the construct that was identical to MGEV-7 except the vacuole targeting peptide (V) was deleted (MGEV-10). Transient expression from IVIGEV-IO resulted in an extracellular location for the GFP fluorescence (Example 4, Fig. 10F).<br>
[00153] GFP was also directed extracellularly when MGEV-10 was expressed transiently in the leaves of N. benthamiana. Figs 14 B, C, D and E show the confocal images obtained when MGEV-10 and a control construct that encodes only GFP and a signal peptide (CI) were expressed in N. benthamiana. Both constructs lack the vacuolar targeting sequence (V) and hence the GFP was secreted outside both epidermal and mesophyll cells and was not directed to the vacuole.<br>
[00154] The results confirm and amplify those obtained in Example 4. Vacuolar targeting of GFP was observed regardless of whether the targeting sequence was directly attached to GFP protein or to the unprocessed MGEV protein.<br>
EXAMPLE 7<br>
Construction and expression of an IVIGEV having one defensin with CTPP and<br>
3 potato type two Pis<br>
[00155] The MGEV described in this example (MGEV-11) has the structure diagrammed as:<br>
I	1<br>
S - C2N - T1 - NaDICTPP - C1 - C2c - V<br>
(See also Fig. 16A).<br>
[00156] A MGEV expressing a defensin and 3 potato type two Pi's was constructed, essentially as described for MGEV-7 (Example 4) except that NaD1 defensin included the C-termlnal acidic peptide tail. NaDICTPP, SEQ ID N0:14 amino acids 26-105 was inserted into the Xbal site of the multipurpose vector<br><br>
pRR21 (Example 4) to produce MGEV-11. MGEV-11 was then inserted into pBIN19 to produce the vector pHEX 42, diagrammed in Fig. 15.<br>
Transient expression in cotton cotyledons<br>
[00157] Expression of pHEX 42 was determined in a transient assay with cotton cotyledons as described in Example 2.<br>
Protein detection<br>
[00158] Protein expression was determined by ELISA and immunoblots as described in Example 4.<br>
Results<br>
[00159] NaPI (Fig. 16B) and NaDI (Fig. 16C) were both detected by ELISA in cotton cotyledons transfected with pHEX42. Immunoblot analysis using the NaPI antibody confirmed that the precursor protein and the processed NaPI 6 kDa peptides were present (Fig. 16D). The precursor protein and the NaD1 protein could also be detected by the NaD1 antibody (Fig 16E). The processed protein was the correct size for the mature NaDI protein (~ 6 l<da indicating that the ctpp tail had been correctly processed></da>
[00160] The results demonstrate expression and correct processing of NaDI having its own vacuolar targeting sequence (CTPP), in addition to the vacuole targeting sequence of the MGEV.<br>
EXAIWPLE 8<br>
Construction and expression of an IVIGEV having two type one Pis and 3<br>
potato type two Pis<br>
[00161]    The MGEV described in this example has the structure diagrammed as:<br><br>
S-C2N-T1 -Pof1A-Pot1A-C1 -C2c-V<br>
(See Fig. 18A).<br>
[00162] A MGEV expressing two potato type 1 Pis and 3 potato type two Pi's was constructed, using pSP2 (Example 6). The Pot 1A dimer was produced as described in Example 5 and Inserted into pSP2 to produce MGEV-12. MGEV-12 was then inserted into pBIN19 to produce the vector pHEX 33, diagrammed in Fig. 17.<br>
Transient expression in cotton cotyledons<br>
[00163] Expression of pHEX33 was determined in a transient assay with cotton cotyledons as described in Example 2.<br>
Protein detection<br>
[00164]    Protein expression was determined by ELISA.<br>
Results<br>
[00165] NaPI (Fig. 18B) and Pot 1A (Fig. 18C) were both detected by ELISA in cotton cotyledons transfected with pHEX 33, The expression of Pot 1A was significant as expression of Pot 1A using pHEX29 (which only has one copy of the gene) could not be detected In the transient assay (data not shown).<br>
[00166] The results indicate that PotIA is expressed in a MGEV and correctly processed in concert with other proteins.<br><br>
Protein detection<br>
[00170]    Protein expression was determined by ELISA as described in Example 3.<br>
Results<br>
[00171] NaPI (Fig. 20B) and NaD1 (Fig. 20C) were both detected by ELISA in cotton cotyledons transfected with pHEX39.<br>
[00172] The results demonstrate the value of using MGEV to express a plurality of plant protective proteins simultaneously.<br>
EXAMPLE 10<br>
Construction and expression of a linear MGEV having two type one PI s and 2<br>
potato type two Pis<br>
[00173]   The MGEV described in this example has the structure diagrammed as:<br>
S - T1 - Pot 1A - PotIA - CI - V<br>
(See Fig. 22A).<br>
[00174] A linear MGEV expressing two potato type 1 Pis and 2 potato type two Pi's was constructed, essentially as described for MGEV-8 (Example 2) except that two Pot lAs were inserted. The Pot 1A - Pot 1A dimer was produced by PCR overlap as described in Example 5 and inserted into the linear multipurpose vector pSPI (Example 2) to produce MGEV-14. MGEV-14 was then inserted into pBIN19 to produce the vector pHEX 48, diagrammed in Fig. 21.<br>
Transient expression in cotton cotyiedons<br>
[00175] Expression of pHEX48 was determined in a transient assay with cotton cotyledons as described in Example 2.<br><br>
Protein detection<br>
[00176] Protein expression was determined by ELISA and immunoblots as described in Example 2.<br>
Results<br>
[00177] NaPI (Fig. 22B) and Pot 1A (Fig. 22C) were both detected by ELISA in cotton cotyledons transfected with pHEX 48. Expression levels of Pot 1A were similar to those produced in cotton cotyledons transfected with pHEX 33 which also contains 2 copies of the Pot 1A gene (Example 8). Immunoblot analysis using the NaPI antibody confirmed that the processed NaPl 6 l<da peptides were present></da>
[00178] The results demonstrate that expression of a linear MGEV protein is at least as effective for expressing multiple proteins as the circular form. MGEV efficacy does not depend on the presence of a "clasp" protein.<br>
EXAMPLE 11<br>
Construction and expression of a linear IVIGEV having one defensin and 2<br>
potato type two Pis<br>
[00179]    The MGEV described in this example has the structure diagrammed as:<br>
S-T1-NaD1-C1-V<br>
(See Fig. 24A).<br>
[00180] A linear MGEV expressing one defensin (NaDI) SEQ ID NO: 14 amino acids 26-72 and 2 potato type two Pi's (T1 and CI) was constructed, essentially as described for MGEV-8 (Example 2) except that a defensin (NaDI) was inserted instead of Pot 1A. NaDI (described in Example 3) was inserted into the linear multipurpose vector pSP1 (Example 2) to produce MGEV-15. MGEV-15 was then<br><br>
inserted into pBIN19 to produce the vector pHEX 47, diagrammed in Fig. 23.<br>
Transient expression in cotton cotyledons<br>
[00181] Expression of pHEX47 was determined in a transient assay with cotton cotyledons as described in Example 2.<br>
Protein detection<br>
[00182] Protein expression was determined by ELISA and immunoblots as described in Example 3.<br>
Results<br>
[00183] NaPI (Fig. 24B) and NaD1 (Fig. 24C) were both detected by ELISA in cotton cotyledons transfected with pHEX 47. Immunoblot analysis using the NaPI antibody confirmed that the processed NaPI 6 kDa peptides were present (Fig. 24D).<br>
[00184] The results further demonstrate efficacy of simultaneously expressing multiple proteins having disparate functions using a linear MGEV lacking coding sequences for cyclization of the expressed poly-protein.<br>
EXAMPLE 12<br>
Construction and expression of a linear iVIGEV having two potato type one Pis<br>
[00185]   The MGEV described in this example has the structure diagrammed as:<br>
S-ProPot1A-Pot1A<br>
(See Fig. 26A).<br>
[00186] A linear MGEV expressing 2 potato type one Pis was constructed by splice overlap PCR. The first fragment consisting of the Pot 1A signal sequence, prodomain (SEQ ID NO:20) (Pro) and mature domain Pot1A (SEQ ID N0:11, herein)<br><br>
was PCR amplified with a 5' Bam H1 site and a 3' linker sequence. The second fragment consisting of the mature Pot 1A was PCR amplified with a 5' linker sequence and a stop codon (TAA) followed by a Sal 1 site at the 3' end. The two PCR fragments were annealed to each other and extended for 8 cycles; outer primers were then added to PCR-amplify the complete sequence. The S - ProPot 1A - Pot 1A fragment was then inserted Into pAM9 to produce MGEV-16. MGEV-16 then inserted into pBIN19 to produce the vector pHEX35, diagrammed in Fig. 25.<br>
Transient expression in cotton cotyledons<br>
[00187] Expression of pHEX35 was determined in a transient assay with cotton cotyledons as described in Example 2.<br>
Protein detection<br>
[00188] Expression of Pot 1A was determined by ELISA as described in Example 1 except that a different Pot 1A antibody was used. The antibody was produced using a bacterially expressed C1-PotlA dimer (the C1 domain is from NaPlii SEQ ID N0:1 aa 54 to 106) and can detect both the C1 and Pot1A proteins. This antibody is better at detecting Pot 1A than the Pot 1A specific antibodies described In Examples 1 and 2, however the C1-Pot 1A antibody can only be used when Pot 1A protein is expressed without the presence of the NaPI peptides. The primary C1-Pot 1A antibody and the secondary CI-Pot lA-biotin antibody were used at 100 ng/well.<br>
[00189] An immunoblot to detect Pot 1A was carried out as described in Example 1 with the modification described in Example 2. The primary C1-Pot 1A antibody was diluted 1:2,000 dilution from a 1mg/ml stock and the secondary antibody (goat anti-rabbit IgG conjugated to horseradish peroxidase) was used at a 1:50,000 dilution.<br>
Results<br>
[00190]    Pot 1A was detected by ELISA In cotton cotyledons transfected with<br><br>
pHEX 35 (Fig. 26B). Pot 1A expression was higher with this linear construct containing two copies of the Pot 1A gene compared to Pot 1A expression produced by a single copy of the Pot 1A gene (Fig. 26B). For comparison, expression of Pot1A as a single gene (not MGEV) was measured using the vector pHEX6 (see published application WO 2004/094630, Example 6). CaMV 35S promoter was used to drive expression in both pHEX6 and pHEX35.<br>
[00191] Immunoblot analysis using the Cl-Pot 1A antibody confirmed that the Pot 1A protein was present (Fig. 26C). Two other Pot 1A specific bands were detected in this sample. The band at approximately 20 kDa is probably the precursor protein. The band at around 49 kDa may be an aggregation of the PotIA mature protein as it has been reported that the native PotI protein from potato tubers forms a oligomer.<br>
[00192] The results further corroborate expression of PotIA in a MGEV-like structure and show that the propeptide on Pot 1 which is a vacuolar targeting sequence is proteolytically removed.<br>
EXAMPLE 13<br>
Construction and expression of a linear MGEV having one potato type two PI<br>
and 1 defensin<br>
[00193]   The MGEV described in this example has the structure diagrammed as:<br>
S-T1-NaD1CTPP<br>
(See Fig. 28A).<br>
[00194] A linear MGEV expressing one potato type two PI (T1) and one defensin (NaDI) with C-terminal tail (CTPP) was constructed. NaDICTPP (See example 7) was PCR amplified with a 5' Xba 1 site and a 3' Sal 1 site. This fragment was inserted into the Xba 1 - Sal 1 cut site of pSP1 (with C1-V removed) to produce MGEV-17. MGEV-17 was then Inserted into pBIN19 to produce the vector pHEX41,<br><br>
diagrammed in Fig. 27.<br>
Transient expression in cotton cotyledons<br>
[00195] Expression of pHEX41 was determined in a transient assay with cotton cotyledons as described in Example 2.<br>
Protein detection<br>
[00196] Expression of NaD1 was determined by ELISA and immunoblots as described in Example 3.<br>
Resuits<br>
[00197] NaPl and NaDI were detected by ELISA in cotton cotyledons transfected with pHEX41 (Figs. 28 B, C and D). Expression of NaD1 from this linear construct in which the NaD1 CTPP is linked to T1 is significantly higher than the expression of NaDI CTPP alone (pHEX3) (see U.S. 7,041,877) in a transient cotton assay when both are driven by the 358 promoter (Fig. 28 B). CTPP targets NaD1 to the vacuole where it is proteolytically removed to release the mature ~6kDa NaDI.<br>
[00198] Immunoblot analysis using the NaPI antibody confirmed that the processed NaPI 6 kDa peptides were present (Fig. 28E). The NaD1 antibody detected both the 16 kDa precursor and the mature NaD1 ~6 kDa protein confirming correct processing of the linker between T1 and NaD1 and correct processing of the CTPP tail (Fig. 28D).<br>
EXAIWPLE 14<br>
Construction and expression of a linear MGEV having one class 1 defensin<br>
and one class two defensin<br>
[00199]    The MGEV described in this example has the structure diagrammed as:<br><br>
[00200] A linear MGEV expressing one class one defensin (NaD2) and one class two defensin (NaD1 with C-temninal tail) was constructed by splice overlap PCR essentially as described in Example 13 except that two defensins were used. NaD2 is described in Example 10 and NaD1-CTPP is described in Example 7. The first fragment consisted of the signal sequence and the coding sequence for NaD2, the second fragment consisted of the mature NaDI and the CTPP tail from NaDI. Following PCR, the full fragment (S - NaD2 - NaDI CTPP) was inserted into pAM9 to produce MGEV-18. MGEV-18 was then inserted Into pBIN19 to produce the vector pHEX52, diagrammed in Fig, 29. A diagram of MGEV-18 is shown in Fig. 30A.<br>
Transient expression in cotton cotyledons<br>
[00201] Expression of pHEX52 was determined In a transient assay with cotton cotyledons as described in Example 2.<br>
Protein detection<br>
[00202] Expression of NaD1 was determined by ELISA as described in Example 3.<br>
Results<br>
[00203] NaDI was detected by ELISA in cotton cotyledons transfected with pHEX52 (Fig. 308). The results demonstrate that a plurality of different proteins can be expressed in a linear MGEV in the absence of any type two PI.<br>
EXAMPLE 15<br>
Construction and expression of a linear MGEV having one class 1 defensin and one class two defensin (CTPP deleted)<br><br>
[00204]   The MGEV described in this example has the structure diagrammed as:<br>
S - NaD2 - NaD1<br>
[00205] A linear MGEV expressing one class one defensin (NaD2) and one class two defensin (NaD1) but lacking the CTPP tail was constructed as described in Example 15 except that the CTPP tail was not amplified. The S - NaD2 - NaDI fragment was inserted into pAM9 to produce MGEV-19. MGEV-19 was then inserted into pBIN19 to produce the vector pHEX51, diagrammed in Fig, 31. A diagram of MGEV-19 is shown in Fig. 32A.<br>
Transient expression in cotton cotyledons<br>
[00206] Expression of pHEX51 was determined in a transient assay with cotton cotyledons as described in Example 2.<br>
Protein detection<br>
[00207] Expression of NaD1 was determined by ELISA as described in Example 3.<br>
Results<br>
[00208] NaDI was detected by ELISA in cotton cotyledons transfected with pHEX51 (Fig. 32B).<br>
EXAIVIPLE16<br>
Construction and expression of a linear MGEV having one beta-Glucuronidase<br>
GUS and 2 potato type two Pis<br>
[00209]    The MGEV described in this example has the structure diagrammed as:<br>
S-T1 -GUSC1 -V<br><br>
[00210] A linear MGEV expressing one GUS and 2 potato type two Pi's (T1 and C1) was constructed, essentially as described for MGEV-8 (Example 2) except that a DNA sequence encoding beta-Glucuronldase (GUS) was inserted in place of Pot 1A. GUS is an E. coli enzyme with a molecular mass of approximately 68,000 Da and is encoded by the gusA gene, SEQ ID N0:18 and SEQ ID N0:19 for GUS DNA and amino acid sequences, respectively. GUS was PCR amplified from the binary vector pBI121 (Invitrogen) with Xba 1 sites at each end, and inserted into the linear multipurpose vector pSP1 (Example 2) to produce MGEV-20. MGEV-20 was then inserted into pBlN19 to produce the vector pHEX58, diagrammed in Fig. 33. In this construct, there was no linker between GUS and C1. Expression and processing was not adversely affected.<br>
Transient expression in cotton cotyledons<br>
[00211] Expression of pHEX58 was determined in a transient assay with cotton cotyledons as described in Example 2.<br>
Protein detection<br>
[00212]    Expression of NaPI was determined by ELISA as described in Example 1<br>
[00213] Immunoblot analysis to detect the NaPIs was carried out as described in Example 1 with the modification described in Example 2.<br>
Results<br>
[00214] NaPI was detected by ELISA in cotton cotyledons transfected with pHEX 58 (Fig. 348).<br>
[00215] Immunoblot analysis using the NaPI antibody confirmed that the mature NaPI peptides were present (Fig. 34C).<br><br>
The results demonstrate that proteins of at least 68 kDa can be expressed in the MGEV and processed correctly.<br>
[00216] As used herein, "comprising" is synonymous with "including," "containing," or "characterized by," and is Inclusive or open-ended and does not exclude additional, unrecited elements or method steps. As used herein, "consisting of excludes any element, step, or ingredient not specified in the claim element. As used herein, "consisting essentially of does not exclude materials or steps that do not materially affect the basic and novel characteristics of the claim. Any recitation herein of the term "comprising", particularly in a description of components of a composition or in a description of elements of a device, is understood to encompass those compositions and methods consisting essentially of and consisting of the recited components or elements. The invention illustratively described herein suitably may be practiced in the absence of any element or elements, limitation or limitations which is not specifically disclosed herein.<br>
[00217] The terms and expressions which have been employed are used as terms of description and not of limitation, and there is no intention in the use of such terms and expressions of excluding any equivalents of the features shown and described or portions thereof, but it is recognized that various modifications are possible within the scope of the invention claimed. Thus, it should be understood that although the present invention has been specifically disclosed by preferred embodiments and optional features, modification and variation of the concepts herein disclosed may be resorted to by those skilled in the art, and that such modifications and variations are considered to be within the scope of this invention as defined by the appended claims.<br>
[00218] In general the terms and phrases used herein have their art-recognized meaning, which can be found by reference to standard texts, journal references and contexts known to those skilled in the art. The following definitions are provided to clarify their specific use in the context of the invention.<br><br>
[00219] All patents and publications mentioned In the specification are incorporated by reference to the extent there is no Inconsistency with the present disclosure, and those references reflect the level of skill of those skilled in the art to which the invention pertains.<br>
[00220] One skilled in the art readily appreciates that the present invention is well adapted to carry out the objects and obtain the ends and advantages mentioned, as well as those inherent in the present invention. The methods, components, materials and dimensions described herein as currently representative of preferred embodiments are provided as examples and are not intended as limitations on the scope of the invention. Changes therein and other uses which are encompassed within the spirit of the invention will occur to those skilled in the art, are included within the scope of the claims.<br>
[00221] Although the description herein contains certain specific information and examples, these should not be construed as limiting the scope of the invention, but as merely providing illustrations of some of the embodiments of the invention. Thus, additional embodiments are within the scope of the invention and within the following claims.<br><br><br><br><br><br>
Claims:<br>
1.	A multigene expression vehicle (IVIGEV) consisting essentially of a polynucleotide comprising 2 to 8 domain segments, D, each domain encoding a functional protein, each domain being joined to the next in a linear sequence by a Linker (L) segment encoding a Linker peptide, the D and L segments all being in the same reading frame, and at least one of the domains is not a type two protease inhibitor.<br>
2.	The MGEV of claim 1 wherein each Linker has the sequence of Sequence I.D. No. 17.<br>
3.	The MGEV of claim 1 further comprising a segment encoding a signal peptide (S) at the N-terminus of a functional protein.<br>
4.	The IVIGEV of claim 3 further comprising a segment encoding a vacuole targeting signal peptide V.<br>
5.	The MGEV' of claim 2 further comprising a segment encoding an N-terminal clasp peptide, CN and a segment encoding a C-tenninal clasp peptide, Cc , the N-terminal and C-terminal clasp peptides being joined by disulfide bonds to one another after translation to form a single protein having protease inhibitor activity.<br>
6.	A MGEV according to claim 2, wherein the D and L coding segments are joined in translational order, designated as (DkLj), where k is an ordinal number for each Domain numbered from 1 to k and k Is in the range from 2 to 8, and j is an ordinal number for each Linker numbered from 1 to k-1.<br>
7.	A MGEV according to claim 5, wherein the CN, D, L and Cc segments are joined In translational order, designated as CN-Li-Dk-Lk+1- Cc, where k is an ordinal<br><br>
number for each domain numbered from 1 to k and k is in the range from 2 to 7, and I is an ordinal number for each Linker, numbered from 1 to k+1.<br>
8.	The IVIGEV of claim 6 further comprising a coding segment, S, encoding a<br>
signal peptide being combined in the order S-DK-LJ .<br>
9.	The IVIGEV of claim 7 further comprising a coding segment, S, encoding a<br>
signal peptide, being combined in the order S-CN-LJ-DK-LK+1<br>
10.	The IVIGEV of claim 6, further comprising a coding segment V, wherein V is<br>
combined with the coding segment of any of Di-Dk, at either end of the segment<br>
encoding any of Di-Dk.<br>
11.	The MGEV of claim 7, further comprising a coding segment, V, wherein V is<br>
combined with the coding segment of any of Di-Dk+i,CN or Cc, at either end of the<br>
segment encoding any of DrDK+i,CNor Cc.<br>
12.	The MGEV of claim 7 further comprising a coding segment, V, encoding a vacuole transport peptide, wherein V is combined with any of Dk, at either end of the segment encoding any of Dk.CN or Cc-<br>
13.	A MGEV expression vector comprising a plant transformation vector carrying and replicating a MGEV according to claim 6 or 7, the MGEV being inserted at a locus in the vector that is under expression control of a plant-active promoter and a    plant-active terminator.<br>
14.	A plant cell containing and expressing proteins encoded by a MGEV according to claim 6 or 7.<br><br>
15.	A transgenic plant, transformed by, and concurrently expressing proteins encoded by a IVIGEV according to claim 6 or 7.<br>
16.	A multigene expression vehicle (MGEV) according to claim  11,  having segments in the translation order:<br><br>
Where S encodes a signal peptide,<br>
CN encodes a N-termlnal clasp peptide,<br>
Cc is a C-terminal clasp peptide,<br>
L1, L2, L3, L4 each encodes a Linker peptide,<br>
D1 encodes a type two trypsin inhibitor,<br>
D2 encodes Pot 1A<br>
D3 encodes a type two chymotrypsin inhibitor, and;<br>
V encodes a vacuole targeting peptide.<br>
17.	A MGEV expression vector comprising a MGEV according to claim 16 under<br>
expression control of a plant-active promoter.<br>
18.	A MGEV according to claim 10 having coding segments in the translation<br>
order:<br><br>
Where S encodes a signal peptide,<br>
D1 encodes a type two trypsin inhibitor,<br>
D2 encodes Pot 1A,<br>
D3 encodes a type two chymotrypsin inhibitor.<br><br>
L1 and L2 encode Linker peptides,<br>
V	encodes a vacuole targeting peptide.<br>
19.	A MGEV expression vector comprising a IVIGEV according to claim 18 under<br>
expression control of a plant-active promoter.<br>
20.	A MGEV according to claim 11 having coding segments in tlie translational<br>
order:<br><br>
Where S encodes a signal peptide,<br>
CN encodes a N-terminal clasp peptide,<br>
L1, L2, L3 and l_4each encode a Linker peptide,<br>
D1encodes a type two trypsin inhibitor,<br>
D2 encodes Pot 1A,<br>
D3 encodes a type two chymotrypsin inhibitor,<br>
V	encodes a vacuole targeting peptide.<br>
21.	A MGEV expression vector comprising a MGEV according to claim 20 under expression control of a plant-active promoter.<br>
22.	A MGEV according to claim 11 having coding segments in the translational order:<br><br>
Where S encodes a signal peptide, CN encodes a N-terminal clasp peptide,<br><br>
L1, L2, Laand Ueach encode a Linker peptide, D1 encodes a type two trypsin inhibitor, D2 encodes a green fluorescent protein, D3 encodes a type two chymotrypsin inliibitor, Cc encodes a C-terminal clasp peptide, and<br>
V	encodes a vacuole targeting peptide.<br>
23.	A MGEV expression vector comprising a MGEV according to claim 22 under expression control of a plant-active promoter.<br>
24.	A MGEV according to claim 11 having coding segments in the translation order:<br><br>
Where S encodes a signal peptide,<br>
CN encodes a N-terminal clasp peptide,<br>
L1,L2,L3,L4,L5 and L5 each encode a Linker peptide,<br>
D1 encodes a type-two trypsin inhibitor.   D2 encodes a plant defensin, and;<br>
D3 and D4 each encode Pot 1A,<br>
D5 encodes a type-two chymotrypsin inhibitor,<br>
Cc encodes a C-terminal clasp peptide, and;<br>
V	encodes a vacuole targeting peptide.<br>
25.	A MGEV expression vector comprising a MGEV according to claim 24 under expression control of a plant-active promoter.<br>
26.	A MGEV according to claim 9 having coding segments in the translational order:<br><br><br>
Where S encodes a signal peptide, and;<br>
CN encodes a clasp peptide,.<br>
Di encodes a type two trypsin inliibltor,<br>
D2 encodes a green fluorescent protein,<br>
D3 encodes a type two chymotrypsin inhibitor,<br>
Cc encodes a clasp peptide, and;<br>
Li,L2,L3 and L4each encode a Linker peptide.<br>
27.	A plant transformation vector comprising a MGEV according to claim 26 under expression control of a plant-active promoter.<br>
28.	A MGEV according to claim 11 having coding segments in the translation order:<br>
S-CN-LrDi-La-Ds-La-Da-U- Cc-V<br>
Where S encodes a signal peptide,<br>
CN encodes a clasp peptide,<br>
D1 encodes a type two trypsin inhibitor,<br>
D2 encodes a defensin having a C-terminal propeptide,<br>
D3 encodes a type two chymotrypsin inhibitor,<br>
Cc encodes a clasp peptide,<br>
L1, L2, La and L4 each encode a Linker peptide, and;<br>
V encodes a vacuole targeting peptide.<br>
29.	A MGEV expression vector comprising a MGEV according to claim 28 under<br>
expression control of a plant-active promoter.<br><br>
30.	A MGEV according to claim 11 having coding segments in the translation<br>
order:<br><br>
Where S encodes a signal peptide, and;<br>
CN encodes a clasp peptide, and;<br>
Di encodes a type two trypsin inhibitor, and;<br>
D2 and Daeach encode Pot 1A, and;<br>
D4 encodes a chymotrypsin inhibitor, and;<br>
Cc encodes a chymotrypsin inhibitor, and;<br>
L1, L2, L3, U and L5 each encode a Linker peptide, and;<br>
V	encodes a vacuole targeting peptide.<br>
31.	A plant transformation vector comprising a MGEV according to claim 30 under expression control of a plant-active promoter.<br>
32.	A MGEV according to claim 11 having coding segments in the translation order:<br><br>
Where S encodes a signal peptide,<br>
CN encodes a N-terminal clasp peptide,<br>
Li, L2, L3, L4, and Lg each encode a Linker peptide,<br>
Cc encodes a C-temninal clasp peptide,<br>
V	encodes a vacuole targeting peptide,<br>
Di encodes a type-two trypsin inhibitor,<br>
D2 encodes a first plant defensin,<br><br>
Da encodes a second plant defensin, and; D4 encodes a type-two chymotrypsin inhibitor.<br>
33.	A MGEV expression vector comprising a MGEV according to claim 32 under<br>
expression control of a plant-active promoter.<br>
34.	A MGEV according to claim 10 having coding segments in the translation<br>
order:<br><br>
Where S encodes a signal peptide,<br>
Di encodes a type-two trypsin inhibitor,<br>
D2 and D3 each encode Pot 1A,<br>
D4 encodes a type-two chymotrypsin inhibitor,<br>
V encodes a vacuole targeting peptide, and;<br>
Li, L2, and L3 each encodes a Linker peptide.<br>
35.	A IVIGEV expression vector comprising a MGEV according to claim 34 under expression control of a plant-active promoter.<br>
36.	A MGEV according to claim 10 having coding segments in the translation order:<br><br>
Where S encodes a signal peptide, Di encodes a type-two trypsin inhibitor, D2 encodes a first plant defensin,<br><br>
Da encodes a type-two chymotrypsin inhibitor,<br>
V	encodes a vacuole targeting peptide, and;<br>
Li and La eacli encodes a Linl<er peptide.></er>
37.	A IVIGEV expression vector comprising a IVIGEV according to claim 36 under expression control of a plant-active promoter.<br>
38.	A MGEV according to claim 8 liaving coding segments in ttie translation order:<br><br>
Where S encodes a signal peptide,<br>
Li encodes a Linlcer peptide,<br>
Di and Da each encode a potato type one proteinase inhibitor.<br>
39.	A MGEV expression vector comprising a MGEV according to claim 38 under expression control of a plant-active promoter.<br>
40.	A MGEV according to claim 10 having coding segments in the translation order:<br><br>
Where S encodes a signal peptide, Li encodes a Linker peptide Di encodes a type-two trypsin inhibiter, Da encodes a plant defensin, and;<br>
V	encodes a vacuole targeting peptide.<br><br>
41.	A MGEV expression vector comprising a IVIGEV according to claim 40 under expression control of a plant-active promoter.<br>
42.	A MGEV according to claim 10 having coding segments in the translation order:<br><br>
Wliere S encodes a signal peptide, Li encodes a Linker peptide, V encodes a vacuole targeting peptide, Di encodes a first plant defensin, and; D2 encodes a second plant defensin.<br>
43.	A MGEV expression vector comprising a MGEV according to claim 42 under expression control of a plant-active promoter.<br>
44.	A MGEV according to claim 10 having coding segments in the translation order:<br><br>
Where S encodes a signal peptide, Li encodes a Linker peptide, Di encodes a first plant defensin, and; D2 encodes a second plant defensin.<br>
45.	A MGEV expression vector comprising a MGEV according to claim 44 under<br>
expression control of a plant-active promoter.<br><br>
46.	A MGEV according to claim 10 having coding segments in the translation<br>
order:<br><br>
Where S encodes a signal peptide, V encodes a vacuole targeting peptide, L1 and L2 each encode a Linker peptide, D1 encodes a type-two trypsin inhibitor, D2 encodes a beta-glucuronidase, and; D3 encodes a type-two chymotrypsin inhibitor.<br>
47.	A plant transformation vector comprising a MGEV according to claim 46 under expression control of a plant-active promoter.<br>
48.	A MGEV according to claim 4 selected from the group of MGEV's consisting of MGEV 5, MGEV 8, MGEV 6, MGEV 7, MGEV 9, MGEV 10, MGEV 11, MGEV 12, MGEV 13, MGEV 14, MGEV 15, MGEV 16, MGEV 17, MGEV 18, MGEV 19, MGEV 20.<br>
49.	A MGEV according to claim 5 selected from the group of MGEV's consisting of MGEV 6, MGEV 6, MGEV 7, MGEV 9, MGEV 10, MGEV 11, MGEV 12, MGEV 13.<br>
50.	MGEV expression vector selected from the group of MGEV expression vectors consisting of pHEX 29, pHEX 56, pHEX 31, pHEX 46, pHEX 55, pHEX 45, pHEX 42, pHEX 33, pHEX 39. pHEX 48, pHEX 47, pHEX 35, pHEX 41, pHEX 52, pHEX51,pHEX58.<br><br>
51.	A method of concurrently expressing fronn two to eight desired proteins in a<br>
plant cell comprising the steps of:<br>
a.	assembling a multi-gene expression vehicle (IVIGEV) consisting<br>
essentially of a polynucleotide segment comprising from 2 to 8 Domain<br>
segments, Dk, each Domain encoding a functional protein wherein at<br>
least one such protein is not a type-two protease Inhibitor and each<br>
Domain is joined to the next in a linear sequence by a Linker segment,<br>
L, encoding a Linker peptide having a sequence of SEQ ID N0:17, all<br>
the D and L coding segments being joined In the same reading frame<br>
in translatlonal order designated as DkLj, and where k is an ordinal<br>
number for each Domain numbered from 1 to k and k is in the range<br>
from 2 to 8, and j is an ordinal number for each Linker numbered from<br>
1 tok-1.<br>
b.	combining the MGEV with a plant transformation vector, at a locus in<br>
the vector that is under expressive control of a plant-active promoter<br>
and a plant-active terminator, thereby providing a IVIGEV expression<br>
vector, and;<br>
c.	transforming a plant cell with the MGEV expression vector, thereby<br>
providing a MGEV-transformed cell, and;<br>
d.	maintaining the MGEV-transformed cell and progeny thereof under<br>
conditions suitable for gene expression within the cell, whereby genes<br>
encoded within MGEV-transformed cells are concurrently expressed.<br>
52.	The method of claim 51 wherein proteins D1 to k are individually selected from<br>
the  group of proteins,  consisting  of a type-two trypsin  inhibitor,  a type-two<br><br>
chymotrypsin inhibitor, a Pot i protease inhibitor, a defensin, a defensin having a C-termina! propeptide, a green fluorescent protein, and an indicator enzyme.<br>
53.	The method of claim 52 wherein the IVIGEV expression vector is selected from the group of l\/IGEV expression vectors consisting of pHEX 56, pHEX 48, pHEX 47, pHEX 17, pHEX 18, pHEX 19, pHEX 20.<br>
54.	The method of claim 52 further comprising the step of regenerating an adult transgenic plant from the MGEV-transformed cell.<br>
55.	The method of claim 54 wherein the wherein the adult transformed plant is selected from the group of plants consisting of cotton, soybean, corn and rice.<br>
56.	A method of concurrently expressing from 3 to 8 proteins in a plant cell comprising the steps of:<br>
a. assembling a multi-gene expression vehicle (MGEV) consisting essentially of a polynucleotide segment comprising from 3 to 8 Domain segments, Dk, each Domain encoding a functional protein wherein at least one such protein is not a type-two protease inhibitor and each Domain is joined to the next in a linear sequence by a Linker segment, L, encoding a Linker peptide having a sequence of SEQ ID N0:17, ail the D and L coding segments being joined in the same reading frame in translational order, designated as CN-Lj-DK-Lk+i-Cc, where k is an ordinal number for each domain numbered from 1 to k and k is in the range from 3 to 7, and j is an ordinal number for each Linker, numbered from 1 to k+1.<br><br>
b.	combining the MGEV with a plant transformation vector, at a locus in<br>
the vector that is under expression control of a plant-active promoter<br>
and a plant-active terminator, thereby providing a MGEV expression<br>
vector, and;<br>
c.	transforming a plant cell with the MGEV expression vector, thereby<br>
providing a MGEV-transformed cell, and;<br>
d.	maintaining the MGEV-transfonned cell and progeny thereof under<br>
conditions suitable for gene expression within the cell, whereby genes<br>
encoded within MGEV transferred cells are concurrently expressed.<br>
57.	Plant transformation vector selected from the group of plant transfomriation vectors consisting of pHEX 10, pHEX 29, pHEX 31, pHEX 46, pHEX 55, pHEX 45, pHEX 42. pHEX 33 and pHEX 39.<br>
58.	The method of claim 56 further comprising the step of regenerating an adult transgenic plant from the MGEV-transfonned cell,<br>
59.	The method of claim 56 wherein the wherein the adult transformed plant is selected from the group of plants consisting of cotton, soybean, corn and rice.<br>
60.	A method for concurrently expressing from two to eight proteins in a plant cell comprising transfonning a plant cell with a MGEV according to ciaim 2, wherein the MGEV is under expression control of a single promoter.<br>
61.	The method of claim 60 wherein the linker has the sequence of SEQ ID N0;5.</td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjU0OC1DSEVOUC0yMDA4ICAgIEVYQU1JTkFUSU9OIFJFUE9SVCBSRVBMWSAgUkVDRUlWRUQgMTMtMDEtMjAxNC5wZGY=" target="_blank" style="word-wrap:break-word;">6548-CHENP-2008    EXAMINATION REPORT REPLY  RECEIVED 13-01-2014.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjU0OC1DSEVOUC0yMDA4ICAgIE9USEVSIFBBVEVOVCBET0NVTUVOVCAxMy0wMS0yMDE0LnBkZg==" target="_blank" style="word-wrap:break-word;">6548-CHENP-2008    OTHER PATENT DOCUMENT 13-01-2014.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjU0OC1DSEVOUC0yMDA4ICAgQU1FTkRFRCAgIFBBR0VTICBPRiBTUEVDSUZJQ0FUSU9OICAxMy0wMS0yMDE0LnBkZg==" target="_blank" style="word-wrap:break-word;">6548-CHENP-2008   AMENDED   PAGES  OF SPECIFICATION  13-01-2014.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjU0OC1DSEVOUC0yMDA4ICAgQU1FTkRFRCAgQ0xBSU1TICAgMTMtMDEtMjAxNC5wZGY=" target="_blank" style="word-wrap:break-word;">6548-CHENP-2008   AMENDED  CLAIMS   13-01-2014.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjU0OC1DSEVOUC0yMDA4ICAgQU1FTkRFRCBDTEFJTVMgIDE3LTEyLTIwMTQucGRm" target="_blank" style="word-wrap:break-word;">6548-CHENP-2008   AMENDED CLAIMS  17-12-2014.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjU0OC1DSEVOUC0yMDA4ICAgQU1FTkRFRCBQQUdFUyBPRiBTUEVDSUZJQ0FUSU9OICAxNy0xMi0yMDE0LnBkZg==" target="_blank" style="word-wrap:break-word;">6548-CHENP-2008   AMENDED PAGES OF SPECIFICATION  17-12-2014.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjU0OC1DSEVOUC0yMDA4ICAgQ09SUkVTUE9OREVOQ0UgT1RIRVJTICAyNC0wMi0yMDE0LnBkZg==" target="_blank" style="word-wrap:break-word;">6548-CHENP-2008   CORRESPONDENCE OTHERS  24-02-2014.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjU0OC1DSEVOUC0yMDA4ICAgQ09SUkVTUE9OREVOQ0UgT1RIRVJTIDAzLTEyLTIwMTQucGRm" target="_blank" style="word-wrap:break-word;">6548-CHENP-2008   CORRESPONDENCE OTHERS 03-12-2014.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjU0OC1DSEVOUC0yMDA4ICAgRVhBTUlOQVRJT04gUkVQT1JUIFJFUExZIFJFQ0VJVkVEICAxNy0xMi0yMDE0LnBkZg==" target="_blank" style="word-wrap:break-word;">6548-CHENP-2008   EXAMINATION REPORT REPLY RECEIVED  17-12-2014.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjU0OC1DSEVOUC0yMDA4ICAgT1RIRVJTICAxMy0wMS0yMDE0LnBkZg==" target="_blank" style="word-wrap:break-word;">6548-CHENP-2008   OTHERS  13-01-2014.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjU0OC1DSEVOUC0yMDA4ICAgUE9XRVIgT0YgQVRUT1JORVkgICAyNC0wMi0yMDE0LnBkZg==" target="_blank" style="word-wrap:break-word;">6548-CHENP-2008   POWER OF ATTORNEY   24-02-2014.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjU0OC1DSEVOUC0yMDA4ICBDT1JSRVNQT05ERU5DRSAgT1RIRVJTICAzMS0xMC0yMDE0LnBkZg==" target="_blank" style="word-wrap:break-word;">6548-CHENP-2008  CORRESPONDENCE  OTHERS  31-10-2014.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjU0OC1DSEVOUC0yMDA4ICBDT1JSRVNQT05ERU5DRSBPVEhFUlMgIDI0LTA5LTIwMTQucGRm" target="_blank" style="word-wrap:break-word;">6548-CHENP-2008  CORRESPONDENCE OTHERS  24-09-2014.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjU0OC1DSEVOUC0yMDA4ICBDT1JSRVNQT05ERU5DRSBPVEhFUlMgMTAtMDktMjAxMy5wZGY=" target="_blank" style="word-wrap:break-word;">6548-CHENP-2008  CORRESPONDENCE OTHERS 10-09-2013.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjU0OC1DSEVOUC0yMDA4ICBDT1JSRVNQT05ERU5DRSBPVEhFUlMgMTYtMDQtMjAxMi5wZGY=" target="_blank" style="word-wrap:break-word;">6548-CHENP-2008  CORRESPONDENCE OTHERS 16-04-2012.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjU0OC1DSEVOUC0yMDA4ICBGT1JNLTEgICAxMy0wMS0yMDE0LnBkZg==" target="_blank" style="word-wrap:break-word;">6548-CHENP-2008  FORM-1   13-01-2014.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjU0OC1DSEVOUC0yMDA4ICBGT1JNLTEzICAgMTMtMDEtMjAxNC5wZGY=" target="_blank" style="word-wrap:break-word;">6548-CHENP-2008  FORM-13   13-01-2014.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjU0OC1DSEVOUC0yMDA4ICBGT1JNLTEzLiAgIDEzLTAxLTIwMTQucGRm" target="_blank" style="word-wrap:break-word;">6548-CHENP-2008  FORM-13.   13-01-2014.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjU0OC1DSEVOUC0yMDA4ICBGT1JNLTMgMTAtMDktMjAxMy5wZGY=" target="_blank" style="word-wrap:break-word;">6548-CHENP-2008  FORM-3 10-09-2013.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjU0OC1DSEVOUC0yMDA4ICBGT1JNLTMgMTMtMDEtMjAxNC5wZGY=" target="_blank" style="word-wrap:break-word;">6548-CHENP-2008  FORM-3 13-01-2014.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjU0OC1DSEVOUC0yMDA4ICBGT1JNLTMgMTYtMDQtMjAxMi5wZGY=" target="_blank" style="word-wrap:break-word;">6548-CHENP-2008  FORM-3 16-04-2012.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjU0OC1jaGVucC0yMDA4IGFic3RyYWN0LnBkZg==" target="_blank" style="word-wrap:break-word;">6548-chenp-2008 abstract.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjU0OC1jaGVucC0yMDA4IGNsYWltcy5wZGY=" target="_blank" style="word-wrap:break-word;">6548-chenp-2008 claims.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjU0OC1jaGVucC0yMDA4IGNvcnJlc3BvbmRlbmNlIG90aGVycy5wZGY=" target="_blank" style="word-wrap:break-word;">6548-chenp-2008 correspondence others.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjU0OC1jaGVucC0yMDA4IGNvcnJlc3BvbmRuZWNlLW90aGVycy5wZGY=" target="_blank" style="word-wrap:break-word;">6548-chenp-2008 correspondnece-others.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjU0OC1jaGVucC0yMDA4IGRlc2NyaXB0aW9uKGNvbXBsZXRlKS5wZGY=" target="_blank" style="word-wrap:break-word;">6548-chenp-2008 description(complete).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjU0OC1jaGVucC0yMDA4IGRyYXdpbmdzLnBkZg==" target="_blank" style="word-wrap:break-word;">6548-chenp-2008 drawings.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjU0OC1jaGVucC0yMDA4IGZvcm0tMS5wZGY=" target="_blank" style="word-wrap:break-word;">6548-chenp-2008 form-1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjU0OC1jaGVucC0yMDA4IGZvcm0tMTMucGRm" target="_blank" style="word-wrap:break-word;">6548-chenp-2008 form-13.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjU0OC1DSEVOUC0yMDA4IEZPUk0tMTggMTctMDUtMjAxMC5wZGY=" target="_blank" style="word-wrap:break-word;">6548-CHENP-2008 FORM-18 17-05-2010.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjU0OC1jaGVucC0yMDA4IGZvcm0tMjYucGRm" target="_blank" style="word-wrap:break-word;">6548-chenp-2008 form-26.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjU0OC1jaGVucC0yMDA4IGZvcm0tMy5wZGY=" target="_blank" style="word-wrap:break-word;">6548-chenp-2008 form-3.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjU0OC1jaGVucC0yMDA4IGZvcm0tNS5wZGY=" target="_blank" style="word-wrap:break-word;">6548-chenp-2008 form-5.pdf</a></p>
		<br>
		<div class="pull-left">
			<a href="266169-dosing-pump-unit.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="266171-methods-for-plant-transformation-using-spectinomycin-selection.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>266170</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>6548/CHENP/2008</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>15/2015</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>10-Apr-2015</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>09-Apr-2015</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>01-Dec-2008</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>HEXIMA LIMITED</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td>LEVEL 5 114 WILLIAM STREET MELBOURNE VICTORIA 3000</td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>ANDERSON, MARILYN, ANNE</td>
											<td>24 GARDEN AVENUE KEILOR VICTORIA 3073.</td>
										</tr>
										<tr>
											<td>2</td>
											<td>HEATH, ROBYN, LOUISE</td>
											<td>3 BERRY STREET, CLIFTON HILL, VICTORIA 3068</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>A01H5/00</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>PCT/AU2007/000712</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td>2007-05-24</td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td>60/803,206</td>
									<td>2006-05-25</td>
								    <td>U.S.A.</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/266170-multi-gene-expression-vehicle by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 06:44:30 GMT -->
</html>
