<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/266167-a-method-for-leaf-senescence-in-a-transgenic-plant by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 06:43:59 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 266167:A METHOD FOR LEAF SENESCENCE IN A TRANSGENIC PLANT</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">A METHOD FOR LEAF SENESCENCE IN A TRANSGENIC PLANT</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>USE OF NAP GENE TO MANIPULATE LEAF SENESCIENCE IN PLANTS The present invention discloses transgenic plants having an altered level of NAP protein compared to that of a non-transgenic plant, where the transgenic plants display an altered leaf senescence phenotype relative to a non-transgenic plant, as well as mutant plants comprising an inactivated NAP gene, where mutant plants display a delayed leaf senescence phenotype compared to that of a non-mutant plant. The present invention also discloses methods for delating leaf senescence in a plant, as well as methods of making a mutant plant having a decreased level of NAP protein compared to that of a non-mutant plant, where the mutant plant displays a delayed leaf senescence phenotype relative to a non-mutant plant. Methods for causing precocious leaf sensescence or promoting leaf senescence in a plant are also disclosed. Also disclosed are methods of identifying a candidate plant suitable for breeding that displayed a delayed leaf senescence and/or enhanced yield phenotype.</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td> <br><br>
USE OF NAP GENE TO MANIPULATE LEAF SENESCENCE<br>
IN PLANTS<br>
[0001]	This application claims the benefit of U.S. Provisional Patent<br>
AppUcation Serial No. 60/786,602, filed March 28,2006, which is hereby incorporated by reference in its entirety.<br>
[0002]	This invention arose out of research sponsored by the U.S. Department<br>
of Energy Basic Energy Sciences (Grant No. DE-FG02-02ER15341) and the U.S.¬Israel Binational Agricultural Research and Development (BARD) Fund (Grant No. IS-3645-04). The U.S. Government may have certain rights in this invention.<br>
FIELD OF THE INVENTION<br>
[0003]	The present invention relates to transgenic plants having an altered<br>
level of NAP protein compared to that of a non-transgenic plant, where the transgenic plants display an altered leaf senescence phenotype relative to a non-transgenic plant. The present invention also relates to mutant plants comprising an inactivated NAP gene, where mutant plants display a delayed leaf senescence phenotype compared to that of a non-mutant plant. In addition, the present invention relates to methods for dela}dng leaf senescence in a plant, as well as methods for causing precocious leaf senescence or promoting leaf senescence in a plant. The present invention also relates to methods of identifying a candidate plant suitable for breeding that displays a delayed leaf senescence and/or enhanced yield phenotype.<br>
BACKGROUND OF THE INVENTION<br>
[0004]	The transition from a functional photosynthetic organ to an actively<br>
degenerating and nutrient-recycling tissue in a leafs life history represents the onset of leaf senescence. This onset is a developmental switch that involves dramatic differential gene expression. Differential gene expression is believed to play an important role in leaf senescence. In a senescing leaf, many genes that are expressed in green leaves, including those genes involved in photosynthesis, are down-regulated, while a subset of genes, generally referred to as senescence-associated<br><br>
genes (SAGs), are up-regulated. Leaf senescence is under direct nuclear control, and SAG expression is required for senescence to proceed. Inhibitors of transcription or translation prevent leaves from senescing (Buchanan-WoUaston et al., "The Molecular Analysis of Leaf Senescence - A Genomics Approach," P/an? Biotechnology Journal 1:3-22 (2003); Guo et al., "Leaf Senescence: Signals, Execution, and Regulation," Current Topics in Developmental Biology 71:82-l 12 (2005); Hadfield et al, "Programmed Senescence of Plant Organs," Cell Death Differ. 4:662-670 (1997); Lim et al., "The Molecular and Genetic Control of Leaf Senescence and Longevity in Arabidopsis," Current Topics in Developmental Biology 67:49-83 (2005); Smart, "Gene Expression During Leaf Senescence," A/^ew Phytologist 126:419-448 (1994)). For the past decade, much effort has been made to isolate SAG&amp;, and hundreds of SAGs, have been cloned from various plant species including Arabidopsis, barley, Brassica, maize, cucumber, rice, tobacco, radish, asparagus and soybean (Buchanan-Wollaston et al., "The Molecular Analysis of Leaf Senescence - A Genomics Approach." Plant Biotechnology Journal 1:3-22 (2003); Gepstein et al., "Large-Scale Identification of Leaf Senescence-Associated Genes," Plant Journal 36:629-642 (2003); He et al., "Molecular Characteristics of Leaf Senescence," In Recent Research Developments in Plant Molecular Biology, Kerala, India: Research Signpost, pp. 1-17(2003)). Recent application of genomics approaches has led to the identification of thousands of potential SAGs (Andersson et al., "A Transcriptional Timetable of Autumn Senescence," Genome Biology 5:R24 (2004); Bhalerao et al., "Gene Expression in Autumn Leaves," Plant Physiology 131:430-442 (2003); Buchanan-Wollaston et al, "The Molecular Analysis of Leaf Senescence - A Genomics Approach," Plant Biotechnology Journal 1:3-22 (2003); Buchanan-Wollaston et al, "Comparative Transcriptome Analysis Reveals Significant Differences in Gene Expression and Signalling Pathways Between Developmental and Dark/Starvation-Induced Senescence in Arabidopsis," The Plant Journal 42:567-585 (2005); Guo et al., "Transcriptome of Arabidopsis Leaf Senescence," Plant Cell and Environment 27:521-549 (2004); Lin et al, "Molecular      Events in Senescing Arabidopsis Leaves," Plant Journal 39:612-628 (2004); Zentgraf et al, "Senescence-Related Gene Expression Profiles of Rosette Leaves of Arabidopsis Thaliana: Leaf Age Versus Plant Age," Plant Biology 6:178-183 (2004)). Analysis of a leaf senescence EST database (dbEST) indicated that approximately<br><br>
10% (approximately 2500) of the Arabidopsis genes are expressed in senescent leaves (Guo et al., "Transcriptome of Arabidopsis Leaf Senescence," P/anr Cell and Environment 27:521-549 (2004)). Microarray analysis of the global gene expression changes during developmental leaf senescence in Arabidopsis has led to the identification of more than 800 genes that show a reproducible increase in transcript abundance (Buchanan-WoUaston et al, "Comparative Transcriptome Analysis Reveals Significant Differences in Gene Expression and Signalling Pathways Between Developmental and Dark/Starvation-Induced Senescence in Arabidopsis," The Plant Journal 42:567-585 (2005)).<br>
[0005]	Changes of gene expression are often regulated by transcription factors<br>
that bind to specific cis elements of target gene promoters, resulting in the activation and/or suppression of the target genes. There are approximately 1500 transcription factor genes in the Arabidopsis genome that belong to more than 30 gene families based on their DNA-binding domains (Riechmann et al., "Arabidopsis Transcription Factors: Genome-Wide Comparative Analysis Among Eukaryotes," Science 290:2105-2110 (2000)). Microarray analysis has identified 96 transcription factor genes with at least a threefold upregulation during leaf senescence (Buchanan-Wollaston et al., "Comparative Transcriptome Analysis Reveals Significant Differences in Gene Expression and Signalling Pathways Between Developmental and Dark/Starvation-Induced Senescence in Arabidopsis," The Plant Journal 42:567-585 (2005)), and analysis of the leaf senescence dbEST revealed 134 unique genes that encode transcription factors representing 20 different gene families (Guo et al., "Transcriptome of Arabidopsis Leaf Senescence," Plant Cell and Environment 27:521-549 (2004)). Among the largest transcription factor groups are NAC, WRKY, C2H2 type zinc finger, AP2/EREBP, and MYB proteins (Buchanan-WoUaston et al, "Comparative Transcriptome Analysis Reveals Significant Differences in Gene Expression and Signalling Pathways Between Developmental and Dark/Starvation-Induced Senescence in Arabidopsis," The Plant Journal 42:567-585 (2005); Chen et al, "Expression Profile Matrix of Arabidopsis Transcription Factor Genes Suggests Their Putative Functions in Response to Environmental Stresses, " Plant Cell 14:559-574 (2002); Guo et al, "Transcriptome of Arabidopsis Leaf Senescence," Plant Cell and Environment 27:521-549 (2004); Lin et al, "Molecular Events in Senescing Arabidopsis Leayss," Plant Journal 39:612-628 (2004)). Two WRKY transcription<br><br>
factor genes have been studied: WRKY53 plays an important role in controlling leaf senescence (Hinderhofer et al, "Identification of a Transcription Factor Specifically Expressed at the Onset of Leaf Senescence," Planta 213:469-473 (2001); Miao et al, "Targets of the WRKY53 Transcription Factor and Its Role During Leaf Senescence in Arabidopsis," Plant Mol Biol 55:853-867(2004); Robatzek et al., "Targets of AtWRKY6 Regulation During Plant Senescence and Pathogen Defense," Genes Dev 16:1139-1149 (2002)), while suppression of WRKY6 expression has little effect on either the onset or the progression of leaf senescence (Hinderhofer et al., "Identification of a Transcription Factor Specifically Expressed at the Onset of Leaf Senescence," Planta 213:469-473 (2001); Miao et al, "Targets of the WRKY53 Transcription Factor and Its Role During Leaf Senescence in Arabidopsis," Plant Mol Biol 55:853-867(2004); Robatzek et al., "Targets of AtWRKY6 Regulation During Plant Senescence and Pathogen Defense," Genes Dev 16:1139-1149 (2002)). The potential fiinctions of the majority of the leaf senescence-associated transcription factors remain to be elucidated.<br>
[0006]	A total of 20 genes encoding NAC transcription factors are in the leaf<br>
senescence dbEST (Guo et al., "Transcriptome of Arabidopsis Leaf Senescence," Plant Cell and Environment 27:521-549 (2004)), representing almost one-fifth of all the predicted 109 members of the NAC superfamily in Arabidopsis (Riechmann et al., "Arabidopsis Transcription Factors: Genome-Wide Comparative Analysis Among Eukaryotes," Science 290:2105-2110 (2000)). The NAC domain was originally defined by the highly conserved N-termini of the petunia NAM (NO APICAL MERISTEM) and Arabidopsis ATAFl and CUC2 (CUP-SHAPED C0TYLED0N2) genes. It exists widely in plants but not in other eukaryotes. Roles of the NAC family genes include embryo and shoot meristem development, lateral root formation, auxin signaling, defense, and abiotic stress response (Olsen et al., "NAC Transcription Factors: Structurally Distinct, Functionally Diverse," Trends Plant Sci 10:79-87 (2005)). Expression of the NAC family genes in senescing leaves has been reported by several groups (Andersson et al., "A Transcriptional Timetable of Autumn Senescence," Genome Biology 5 (2004); Buchanan-Wollaston et al, "Comparative Transcriptome Analysis Reveals Significant Differences in Gene Expression and Signalling Pathways Between Developmental and Dark/Starvation-Induced Senescence in Arabidopsis," The Plant Journal 42:567-585 (2005); Guo et al..<br><br>
"Transcriptome of Arabidopsis Leaf Senescence," Plant Cell and Environment<br>
27:521-549 (2004); John et al, "Cloning and Characterization of Tomato Leaf<br>
Senescence-Related cTMA&amp;r Plant Molecular Biology 2&gt;3-M\-65\ (1997); Lin et al.,<br>
"Molecular Events in Senescing Arabidopsis Leaves," Plant Journal 39:612-628<br>
(2004)), but whether these genes play a part in leaf senescence is unknown.<br>
[0007]	The present invention is directed to overcoming these deficiencies in<br>
the art.<br>
SUMMARY OF THE INVENTION<br>
[0008]	The present invention relates to a transgenic plant having an altered<br>
level of NAP protein capable of causing leaf senescence in a plant, compared to that<br>
of a non-transgenic plant, where the transgenic plant displays an altered leaf<br>
senescence phenotype, relative to a non-transgenic plant.<br>
[0009]	Another aspect of the present invention relates to a mutant plant<br>
comprising an inactivated NAP gene, where the mutant plant displays a delayed leaf senescence phenotj/pe, relative to a non-mutant plant.<br>
[0010]	The present invention also relates to a method for delaying leaf<br>
senescence in a plant. The method involves providing a transgenic plant or plant seed transformed with a nucleic acid construct effective in silencing expression of a NAP protein capable of causing leaf senescence in a plant. Then, the transgenic plant or the plant grown from the transgenic plant seed is grown imder conditions effective to delay leaf senescence in the transgenic plant or the plant grown from the transgenic plant seed.<br>
[0011]	Another aspect of the present invention relates to a method for<br>
delaying leaf senescence in a plant. The method involves transforming a plant cell with a nucleic acid molecule encoding a NAP protein capable of causing leaf senescence in a plant operably associated with a promoter to obtain a transformed plant cell, where expression of the nucleic acid molecule in the plant cell causes delayed leaf senescence by a form of post-transcriptional gene silencing. Then, a plant is regenerated from the transformed plant cell under conditions effective to delay leaf senescence in the plant.<br><br>
[0012]	Another aspect of the present invention relates to a method of making a<br>
mutant plant having a decreased level of NAP protein compared to that of a non-mutant plant, where the mutant plant displays a delayed leaf senescence phenotype relative to a non-mutant plant. The method involves providing at least one cell of a non-mutant plant containing a gene encoding a functional NAP protein. Next, the at least one cell of a non-mutant plant is treated under conditions effective to inactivate the gene, thereby yielding at least one mutant plant cell containing an inactivated NAP gene. Then, the at least one mutant plant cell is propagated into a mutant plant, where the mutant plant has a decreased level of NAP protein compared to that of the non-mutant plant and displays a delayed leaf senescence phenotype relative to a non-mutant plant.<br>
[0013]	Another aspect of the present invention relates to a method for causing<br>
precocious leaf senescence or promoting leaf senescence in a plant. The method<br>
involves transforming a plant cell with a nucleic acid molecule encoding a NAP<br>
protein capable of causing leaf senescence in a plant operably associated with a<br>
promoter to obtain a transformed plant cell. Next, a plant is regenerated from the<br>
transformed plant cell. Then, the promoter is induced under conditions effective to<br>
cause premature or precocious leaf senescence in the plant.<br>
[0014]	The present invention also relates to a method of identifying a<br>
candidate plant suitable for breeding that displays a delayed leaf senescence and/or<br>
enhanced yield phenotype. The method involves analyzing the candidate plant for the<br>
presence, in its genome, of an inactivated NAP gene.<br>
[0015]	Leaf senescence is a unique developmental process that is<br>
characterized by massive programmed cell death and nutrient recycling. During leaf<br>
I      senescence, chlorophyll and other macromolecules such as proteins, lipids, and nucleic acids are degraded, resulting in a sharp decrease in leaf photosynthetic activity. Leaf senescence may therefore substantially limit crop yield and forest biomass accumulation. Occurrence of senescence after harvest devaluates vegetable crops and ornamental plants during postharvest storage, transportation, and on<br>
)     shelves. Techniques that dday leaf senescence are of agricultural significance. The underlying molecular regulatory mechanisms of leaf senescence are not well understood.<br><br>
[0016]	The present application describes the functional analysis of AtNAP, a<br>
gene encoding a NAC family transcription factor. Expressionof this gene is closely associated with the senescence process of Arabidopsis rosette leaves. Leaf senescence in two T-DNA insertion lines of this gene is significantly delayed. The T-DNA knockout plants are otherwise normal. The mutant phenotype can be restored to wild type by the intact AtNAP, as well as by its homologs in rice, kidney bean, soybean, and maize that are also upregulated during leaf senescence. Furthermore, inducible overexpression of AtNAP causes precocious senescence. These data strongly suggest that^/NAP and its homologs play an important role in leaf senescence in Arabidopsis and in other plant species.<br>
BRIEF DESCRIPTION OF THE DRAWINGS<br>
[0017]	Figure 1 shows the CLUSTAL W (1.82) multiple sequence alignment<br>
of NAP proteins from various plants, i.e., kidney bean (SEQ ID NO: 1), Medicago (SEQ ID NO: 2), soybean (SEQ ID NO: 3), Populus (SEQ ID NO: 4), nightshade (SEQ ID NO: 5), Arabidopsis (SEQ ID NO: 6), wheat (SEQ ID NO: 7), rice (SEQ ID NO: 8), maize (SEQ ID NO: 9), peach (SEQ ID NO: 10), tomato (SEQ ID NO: 11), petunia (SEQ ID NO: 12), and potato (SEQ ID NO: 13).<br>
[0018]	Figures 2(a)-(c) show RNA gel blot analysis of AtNAP expression<br>
during leaf senescence in Arabidopsis. Figure 2(a) depicts AtNAP expression in<br>
leaves at different developmental stages. YL, a young leaf with half the size of fully<br>
expanded leaf; NS, a fiiUy expanded, non-senescent leaf; ES, an early senescent leaf,<br>
with less than 25% leaf area yellowing; LS, a late senescent leaf, with more than 50%<br>
leaf area yellowing. Figure 2(b) depicts AtNAP expression in leaves 1-12 of a 30-<br>
day-old plant. Leaves are counted firom the bottom of the rosette. Figure 2(c) depicts<br>
AtNAP expression in different part of a senescing leaf. B, base; M, middle; T, tip.<br>
The 18S rRNA autoradiographs (Figures 2(a) and 2(c)) and ethidium bromide-stained<br>
gel (Figure 2(b)) indicate the relative amount of total RNA loaded in respective lanes.<br>
[0019]	Figures 3(a)-(d) illustrate the nuclear localization of GFP-AtNAP<br>
fusion proteins. Figures 3(a) and 3(b) are fluorescent images of the GFP-AtNAP fusion proteins expressed in living onion epidermal cells. Figures 3(c) and 3(d) depict<br><br>
the DAPI (4',6'-diamidino 2-phenylindole) staining of the same images to show the positions of the nuclei (indicated by anows).<br>
[0020]	Figures 4(a)-(b) illustrate the expression ofAtNAP in two T-DNA<br>
insertion lines. Figure 4(a) depicts the gene structure ofAtNAP and locations of T-DNA inserts. Figure 4(b) shows the RNA gel blot analysis of AtNAP expression in senescing leaves (approximately 50% yellowing) of wild type, line 1, and line 2 plants.<br>
[0021]	Figures 5(a)-(d) illustrate the delayed leaf senescence phenotype of the<br>
T-DNA insertion lines compared with that of age-matched wild-type (WT) plants.<br>
Figure 5(a) shows the early stages of plant development in the null mutant (line 1) and<br>
WT plants. Figures 5(b) and 5(c) show senescence in the mutant lines and WT plants<br>
(note: the null plants are otherwise developmentally normal). Figure 5(d) shows<br>
leaves excised from age-matched plants in Figure 5(c). Leaves were numbered from<br>
bottom to top. Under the disclosed growth conditions, an adult Arabidopsis<br>
(accession Columbia) plant typically produces 12 rosette leaves.<br>
[0022]	Figures 6(a)-(e) show the physiological and molecular analyses of<br>
atnap null mutant plants (line 1). Figure 6(a) depicts leaf survival curves<br>
(combination of leaves 9 and 10) of wild-type (WT, n=27) and line 1 (n=22).<br>
Figures 6(b)-6(d) show the chlorophyll content (Figure 6(b)), Fy/Fm ratio<br>
(Figure 6(c)), and ion leakage (Figure 6(d)) in individual rosette leaves of age-<br>
matched WT and line 1 plants. Figure 6(e) shows the RNA gel blot analysis of<br>
SAG 12 and RBCS in the 12 rosette leaves of age-matched WT and line 1 plants.<br>
[0023]	Figures 7(a)-(d) illustrate the complementation of Arabidopsis atnap<br>
null plants with AtNAP, OsNAP (rice), and PvNAP (kidney bean). Figure 7(a) shows the RT-PCR analysis of expression of AtNAP (left lanes), O&amp;NAP (middle lanes), and P-vNAP (right lanes) in wild-type (WT), atnap null mutant, and atnap null mutant transformed with AtNAP, OsNAP, or PvNAP.  18s rRNA serves as an internal standard of equal loading. Figure 7(b) shows the phenotype of detached leaves of WT, null mutant, and various complementation lines. The leaves were kept in darkness for 4 days. Figure 7(c) depicts the Fv/Fm ratios of leaves shown in Figure 7(b). Figure 7(d) shows leaf senescence in intact plants of WT, null mutant, and various complementation lines. The plants were grown side by side in an Arabidopsis growth chamber.<br><br>
[0024]	Figures 8(a)-(d) illustrate that inducible overexpression of AtNAP<br>
causes precocious senescence. Figure 8(a) depicts the modified glucocorticoid-inducible gene expression system consisting of pTA7001 and pGL1167. pTA7001 provides the recombinant transcription factor GVG (GAL4 binding domain + VP16 activation domain + GR or glucocorticoid receptor), and pGL1167 contains the GAL4 cis elements and the AtNAP coding region. Figure 8(b) shows the phenotypes of WT and transgenic plants harboring different constructs. The picture was taken 4 days after treatment with 30 ]xM DEX inducer. Figure 8(c) depicts the Fy/Fm ratios of leaves from different plants that were freated with or without DEX. Figure 8(d) shows the RNA gel blot analysis of the expression of AtNAP, SAG] 2, SA G13, and RBCS in leaves of plants that were treated with or without DEX. C, no treatment control; D, DEX treatment.<br>
[0025]	Figures 9(a)-(d) show homologs of AtNAP in kidney bean (Phaseolus<br>
vulgaris) and rice (Ofyza sativa japonica cultivar group) and their senescence-specific<br>
expression patterns. Figure 9(a) shows a phylogenetic tree of NAP proteins from<br>
different plant species. Figure 9(b) is an alignment of amino acid sequences of NAP<br>
proteins fi'om Arabidopsis, kidney bean, and rice. Figure 9(c) shows the expression of<br>
FvNAP in senescing leaves of kidney bean. Figure 9(d) shows the expression of<br>
O&amp;NAP in senescing leaves of rice. Y, young leaf; S, senescing leaf.<br>
[0026]	Figures 10(a)-(b) illustrate the RT-PCR analysis of the expression of<br>
so&gt;'bean GmNAP (Figure 10(a)) and maize ZmNAP (Figure 10(b)) during leaf senescence. 18S rRNA was used as an internal standard for relative amount of total RNA that was used for each lane.<br>
[0027]	Figures ll(a)-(b) depict the RNAi constructs used for suppression of<br>
GmNAP (Figure 11 (a)) and ZmNAP (Figure 11 (b)).<br>
[0028]	Figures 12(a)-(b) illustrate the complementation of Arabidopsis atnap<br>
null plants with ZmNAP (maize) and GmNAP (soybean). Figure 12(a) shows leaf<br>
senescence in intact plants of WT, atnap null mutant, and atnap null mutant<br>
transformed with ZmNAP, while Figure 12(b) shows leaf senescence in intact plants<br>
of WT, atnap null mutant, and atnap null mutant transformed with GmNAP.<br>
[0029]	Figures 13(a)-(b) show the microarray analysis of expression profile of<br>
AtNAP. Figure 13(a) shows the expression levels of AtNAP in different plant tissues. The highest value shown in "44 senescent leaf is 21790±391. Figure 13(b) illustrates<br><br>
the effect of various treatments on the AtNAP expression. Ratios of expression change (numbers in shaded squares) and expression levels after different treatments are presented. The highest value shown in "PCD: senescence" is 26597±1957. The data were extracted from the Genevestigator microarray database (Zimmerrnann et al, "GENEVESTIGATOR. Arabidopsis Microarray Database and Analysis Toolbox," Plant Physiol 136:2621-2632 (2004), which is hereby incorporated by reference in its entirety).<br>
DETAILED DESCRIPTION OF THE INVENTION<br>
[0030]	The present invention relates to a transgenic plant having an altered<br>
level of NAP protein capable of causing leaf senescence in a plant, compared to that<br>
of a non-transgenic plant, where the transgenic plant displays an altered leaf<br>
senescence phenotype, relative to a non-transgenic plant.<br>
[0031]	In one embodiment of the present invention, the transgenic plant has a<br>
reduced level of NAP protein and displays a delayed leaf senescence phenotype. The<br>
plant can be transformed with a nucleic acid construct including a nucleic acid<br>
molecule configured to silence NAP protein expression.<br>
[0032]	In another embodiment (as described in more detail infra), the<br>
transgenic plant is transformed with a nucleic acid construct including a nucleic acid<br>
molecule that includes a dominant negative mutation and encodes a non-functional<br>
NAP protein. This construct is suitable in suppression or interference of endogenous<br>
mRNA encoding the NAP protein.<br>
[0033]	In another embodiment (as described in more detail infra), the<br>
transgenic plant is transformed with a nucleic acid construct including a nucleic acid<br>
;     molecule that is positioned in the nucleic acid construct to result in suppression or<br>
interference of endogenous mRNA encoding the NAP protein.<br>
[0034]	In another embodiment (as described in more detail infra), the<br>
transgenic plant is transformed with a nucleic acid construct including a nucleic acid molecule that encodes the NAP protein and is in sense orientation.<br>
)      [0035]	In still another embodiment (as described in more detail infra), the<br>
transgenic plant is transformed with a nucleic acid construct including a nucleic acid molecule that is an antisense form of a NAP protein encoding nucleic acid molecule.<br><br>
[0036]	In still another embodiment (as described in more detail infra), the<br>
transgenic plant is transformed with first and second of the nucleic acid constructs<br>
with the first nucleic acid construct encoding the NAP protein in sense orientation and<br>
the second nucleic acid construct encoding the NAP protein in antisense form.<br>
[0037]	In yet another embodiment (as described in more detail infra), the<br>
transgenic plant is transformed with a nucleic acid construct including a nucleic acid molecule including a first segment encoding the NAP protein, a second segment in an antisense form of a NAP protein encoding nucleic acid molecule, and a third segment linking the first and second segments.<br>
[0038]	In another embodiment of the present invention, the transgenic plant<br>
has an increased level of NAP protein and displays a premature or precocious leaf<br>
senescence phenotype. The plant can be transformed with a nucleic acid construct<br>
configured to overexpress NAP protein. In another embodiment (as described in<br>
more detail infra), the nucleic acid construct can include a plant specific promoter,<br>
such as an inducible plant promoter. The present invention further relates to seeds<br>
produced fi-om the transgenic plant of the present invention.<br>
[0039]	Another aspect of the present invention relates to a mutant plant<br>
comprising an inactivated NAP gene, where the mutant plant displays a delayed leaf<br>
senescence phenotype, relative to a non-mutant plant. The present invention fiirther<br>
relates to mutant plant seeds produced by growing the mutant plant of the present<br>
invention imder conditions effective to cause the mutant plant to produce seed.<br>
[0040]	The transgenic plants and mutant plants of the present invention can be<br>
any plant with a NAP gene, including crop plants and ornamental plants. Suitable crop plants include, but are not limited to, alfalfa, rice, wheat, barley, rye, cotton, sunflower, peanut, com, potato, sweet potato, kidney bean, pea, chicory, lettuce, endive, cabbage, bok choy, brussel sprout, beet, parsnip, turnip, cauliflower, broccoli, radish, spinach, onion, garlic, eggplant, pepper, celery, carrot, squash, pumpkin, zucchini, cucumber, apple, pear, melon, citrus, peach, strawberry, grape, raspberry, pineapple, soybean, Medicago, tobacco, tomato, sorghum, and sugarcane. Suitable ornamental plants include, but are not limited to, Arabidopsis thaliana, Saintpaulia, Populus, petunia, pelargonium, poinsettia, chrysanthemum, carnation, zinnia, turfgrass, lily, and nightshade.<br><br>
[0041]	The present invention also relates to a method for delaying leaf<br>
senescence in a plant. The method involves providing a transgenic plant or plant seed<br>
transformed with a nucleic acid construct effective in silencing expression of a NAP<br>
protein capable of causing leaf senescence in a plant. Then, the transgenic plant or<br>
the plant grown from the transgenic plant seed is grown under conditions effective to<br>
delay leaf senescence in the transgenic plant or the plant grown fi-om the transgenic<br>
plant seed. Leaf senescence can be delayed in the plant either before or after harvest.<br>
[0042]	In one embodiment, the above step of providing includes providing a<br>
nucleic acid construct having a nucleic acid molecule configured to silence NAP protein expression. The construct also includes a 5' DNA promoter sequence and a 3' terminator sequence. The nucleic acid molecule, the promoter, and the terminator are operatively coupled to permit expression of the nucleic acid molecule. A plant cell is then transformed with the nucleic acid construct. The method can further involve propagating plants from the transformed plant cell. Suitable methods for transforming the plant can include, for example, Agrobacterium-mediated transformation, vacutom infiltration, biolistic transformation, electroporation, micro¬injection, chemical-mediated transformation (e.g., polyethylene-mediated transformation), and/or laser-beam transformation. The various aspects of this method are described in more detail infra.<br>
[0043]	In one aspect of the present invention, the nucleic acid construct results<br>
in suppression or interference of NAP protein expression by the nucleic acid molecule of the construct containing a dominant negative mutation and encoding a non¬functional NAP protein.<br>
[0044]	In another aspect of the present invention, the nucleic acid construct<br>
resuhs in interference of NAP protein expression by sense or co-suppression in which the nucleic acid molecule of the construct is in a sense (5'-&gt;3') orientation. Co-suppression has been observed and reported in many plant species and may be subject to a transgene dosage effect or, in another model, an interaction of endogenous and transgene transcripts that results in aberrant mRNAs (Senior, "Uses of Plant Gene Silencing," Biotechnology and Genetic Engineering Reviews 15:79-119 (1998); Waterhouse et al., "Exploring Plant Genomes by RNA-Induced Gene Silencing," Nature Review: Genetics 4: 29-38 (2003), which are hereby incorporated by reference in their entirety). A construct with the nucleic acid molecule in the sense orientation<br><br>
may also give sequence specificity to PJSIA silencing when inserted into a vector along with a construct of both sense and antisense nucleic acid orientations as described infra (Wesley et al, "Constnict Design for Efficient, Effective and High-Throughput Gene Silencing in Plants," Plant Journal 27(6) 581-590 (2001), which is hereby incorporated by reference in its entirety).<br>
[0045]	In another embodiment of the present invention, the nucleic acid<br>
construct results in interference of NAP protein expression by the use of antisense suppression in which the nucleic acid molecule of the construct is an antisense (3' —&gt;5') orientation. The use of antisense RNA to down-regulate the expression of specific plant genes is well known (van der Krol et al, Nature, 333:866-869 (1988) and Smith et al. Nature, 334:724-726 (1988), which are hereby incorporated by reference in their entirety). Antisense nucleic acids are DNA or RNA molecules that are complementary to at least a portion of a specific mRNA molecule (Weintraub, "Antisense RNA and DNA," Scientific American 262:40 (1990), which is hereby incorporated by reference in its entirety). In the target cell, the antisense nucleic acids hybridize to a target nucleic acid and interfere with transcription, and/or RNA processing, transport, translation, and/or stability. The overall effect of such interference with the target nucleic acid function is the disruption of protein expression (Baulcombe, "Mechanisms of Pathogen-Derived Resistance to Viruses in Transgenic Plants," Plant Cell 8:1833-44 (1996); Dougherty, et al, "Transgenes and Gene Suppression: Telling us Something New?," Current Opinion in Cell Biology 7:399-05 (1995); LomonossofF, "Pathogen-Derived Resistance to Plant Viruses," Ann. Rev. Phytopathol. 33:323-43 (1995), which are hereby incorporated by reference in their entirety). Accordingly, one aspect of the present invention involves a nucleic acid construct which contains the NAP protein encoding nucleic acid molecule being inserted into the construct in antisense orientation.<br>
[0046]	Interference of NAP protein expression is also achieved in the present<br>
invention by the generation of double-stranded RNA ("dsRNA") through the use of inverted-repeats, segments of gene-specific sequences oriented in both sense and antisense orientations. In one embodiment of this aspect of the present invention, sequences in the sense and antisense orientations are linked by a third segment, and inserted into a suitable expression vector having the appropriate 5' and 3' regulatory nucleotide sequences operably linked for transcription. The expression vector having<br><br>
the modified nucleic acid molecule is then inserted into a suitable host cell or subject.<br>
In the present invention, the third segment linking the two segments of sense and<br>
antisense orientation may be any nucleotide sequence such as a fragment of the |3-<br>
glucuronidase ("GUS") gene. In another embodiment of this aspect of the present<br>
invention, a functional (splicing) intron of the NAP gene may be used for the third<br>
(linking) segment, or, in yet another aspect of the present invention, other nucleotide<br>
sequences without complementary components in the NAP gene may be used to link<br>
the two segments of sense and antisense orientation (Chuang et al., "Specific and<br>
Heritable Genetic Interference by Double-Stranded RNA in Arabidopsis thaliana,'"<br>
Proc. Nat'I Academy of Sciences USA 97(9):4985-4990 (2000); Smith et al., "Total<br>
Silencing by Intron-Spliced Hairpin R^^As," Nature 407:319-320 (2000); Waterhouse<br>
et al., "Exploring Plant Genomes by RNA-Induced Gene Silencing," Nature Review:<br>
Genetics 4:29-38 (2003); Wesley et al, "Construct Design for Efficient, Effective and<br>
High-Throughput Gene Silencing in Plants," Plant Journal 27(6):581-590 (2001),<br>
which are hereby incorporated by reference in their entirety). In any of the<br>
embodiments with inverted repeats of NAP protein, the sense and antisense segments<br>
may be oriented either head-to-head or tail-to-tail in the construct.<br>
[0047]	Another aspect of the present invention involves using hairpin RNA<br>
("hpKNA") which may also be characterized as dsRNA. This involves RNA hybridizing vidth itself to form a hairpin structure that comprises a single-stranded loop region and a base-paired stem. Though a linker may be used between the inverted repeat segments of sense and antisense sequences to generate hairpin or double-stranded RNA, the use of intron-free hpRNA can also be used to achieve silencing of NAP protein expression.<br>
[0048]	Alternatively, in another aspect of the present invention, a plant may<br>
be transformed with constructs encoding both sense and antisense orientation molecules having separate promoters and no third segment linking the sense and antisense sequences (Chuang et al, "Specific and Heritable Genetic Interference by Double-Stranded RNA in Arabidopsis thaliana" Proc. Nat 7 Academy of Sciences USA 97(9):4985-4990 (2000); Waterhouse et al., "Exploring Plant Genomes by RNA-Induced Gene Silencing," Nature Review: Genetics 4-.29-38 (2003); Wesley et al., "Construct Design for Efficient, Effective and High-Throughput Gene Silencing in<br><br>
Plants," Plant Journal 27(6):581-590 (2001), which are hereby incorporated by reference in their entirety).<br>
[0049]	The NAP nucleotide sequences used in the present invention may be<br>
inserted into any of the many available expression vectors and cell systems using reagents that are well known in the art. Suitable vectors include, but are not limited to, the following viral vectors such as lambda vector system gtl 1, gt WES.tB, Charon 4, and plasmid vectors such as pG-Cha, p35S-Cha, pBR322, pBR325, pACYC177, pACYC1084, pUC8, pUC9, pUC18, pUC19, pLG339, pR290, pKC37, pKClOl, SV 40, pBluescript II SK +/- or KS +/- (see "Stratagene Cloning Systems" Catalog (1993) from Stratagene, La Jolla, CA, which is hereby incorporated by reference in its entirety), pQE, pIH821, pGEX, pET series (see Studier et. al, "Use of T7 RNA Polymerase to Direct Expression of Cloned Genes," Gene Expression Technology vol. 185 (1990), which is hereby incorporated by reference in its entirety), and any derivatives thereof. Recombinant molecules can be introduced into cells via transformation, particularly transduction, conjugation, mobilization, or electroporation. The DNA sequences are cloned into the vector using standard cloning procedures in the art, as described by Sambrook et al.. Molecular Cloning: A Laboratory Manual, Second Edition, Cold Spring Harbor, NY:CoId Spring Harbor Press (1989), and Ausubel et al.. Current Protocols in Molecular Biology, New York, N.Y John Wiley &amp; Sons (1989), which are hereby incorporated by reference in their aitiret&gt;[0050]	In preparing a nucleic acid construct for expression, the various nucleic<br>
acid sequences may normally be inserted or substituted into a bacterial plasmid. Any convenient plasmid may be employed, which will be characterized by having a bacterial replication system, a marker which allows for selection in a bacterium, and generally one or more unique, conveniently located restriction sites. Numerous plasmids, referred to as transformation vectors, are available for plant transformation. The selection of a vector will depend on the preferred transformation technique and target species for transformation. A variety of vectors are available for stable transformation using Agrobacterium tumefaciens, a soilbome bacterium that causes crown gall. Crown gall are characterized by tumors or galls that develop on the lower stem and main roots of the infected plant. These tumors are due to the transfer and incorporation of part of the bacterium plasmid DNA into the plant chromosomal<br><br>
DNA. This transfer DNA (T-DNA) is expressed along with the normal genes of the plant cell. The plasmid DNA, pTi, or Ti-DNA, for "tumor inducing plasmid," contains the vir genes necessary for movement of the T-DNA into the plant. The T-DNA carries genes that encode proteins involved in the biosynthesis of plant regulatory factors, and bacterial nutrients (opines). The T-DNA is delimited by two 25 bp imperfect direct repeat sequences called the "border sequences." By removing the oncogene and opine genes, and replacing them with a gene of interest, it is possible to transfer foreign DNA into the plant without the formation of tumors or the multipUcation of Agrobacterium tumefaciens (Fraley et al., "Expression of Bacterial Genes in Plant Cells," Proc. Nat'IAcad. Sci. 80:4803-4807 (1983), which is hereby incorporated by reference in its entirety).<br>
[0051]	Further improvement of this technique led to the development of the<br>
binary vector system (Bevan, "Binary Agrobacterium Vectors for Plant<br>
Transformation," Nucleic Acids Res. 12:8711-8721 (1984), which is hereby<br>
incorporated by reference in its entirety). In this system, all the T-DNA sequences<br>
(including the borders) are removed from the pTi, and a second vector containing T-<br>
DNA is introduced into Agrobacterium tumefaciens. This second vector has the<br>
advantage of being replicable in E. coli as well as A. tumefaciens, and contains a<br>
multiclonal site that facilitates the cloning of a transgene. An example of a<br>
commonly-used vector is pBinl9 (Frisch et al., "Complete Sequence of the Binary<br>
Vector Binl9," PlantMolec. Biol. 27:405-409 (1995), which is hereby incorporated<br>
by reference in its entirety). Any appropriate vectors now known or later described<br>
for genetic transformation are suitable for use with the present invention.<br>
[0052]	U.S. Patent No. 4,237,224 to Cohen and Boyer, which is hereby<br>
incorporated by reference in its entirety, describes the production of expression<br>
systems in the form of recombinant plasmids using restriction enzyme cleavage and<br>
ligation with DNA ligase. These recombinant plasmids are then introduced by means<br>
of transformation and repUcated in unicellular cultures including prokaryotic<br>
organisms and exxkaryotic cells grown in tissue culture.<br>
[0053]	Certain "control elements" or "regulatory sequences" are also<br>
incorporated into the vector-construct. These include non-translated regions of the vector, promoters, and 5' and 3' untranslated regions which interact with host cellular proteins to carry out transcription and translation. Such elements may vary in their<br><br>
strength and specificity. Depending on the vector system and host utilized, any number of suitable transcription and translation elements, including constitutive and inducible promoters, may be used. I'issue-specific and organ-specific promoters can also be used.<br>
[0054]	A constitutive promoter is a promoter that directs expression of a gene<br>
throughout the development and life of an organism. Examples of some constitutive promoters that are widely used for inducing expression of transgenes include the nopaline synthase (NOS) gene promote-T, from Agrobacterium tumefaciens (U.S. Patent No. 5,034,322 to Rogers et al, vi^hich is hereby incorporated by reference in its entirety), the cauliflower mosaic virus (CaMV) 35S and 19S promoters (U.S. Patent No. 5,352,605 to Fraley et al., which is hereby incorporated by reference in its entirety), those derived fi-om any of the several actin genes, which are known to be expressed in most cells types (U.S. Patent No. 6,002,068 to Privalle et al, which is hereby incorporated by reference in its entirety), and the ubiquitin promoter, which is a gene product known to accumulate in many cell types.<br>
[0055]	An inducible promoter is a promoter that is capable of directly or<br>
indirectly activating transcription of one or more DNA sequences or genes in response to an inducer. In the absence of an inducer, the DNA sequences or genes will not be transcribed. The inducer can be a chemical agent, such as a metabolite, growth regulator, herbicide, or phenolic compound, or a physiological stress directly imposed upon the plant such as cold, heat, salt, toxins, or through the action of a pathogen or disease agent such as a virus or fungus. A plant cell containing an inducible promoter may be exposed to an inducer by externally applying the inducer to the cell or plant such as by spraying, watering, heating, or by exposure to the operative pathogen. An example of an appropriate inducible promoter is a glucocorticoid-inducible promoter (Schena et al., "A Steroid-Inducible Gene Expression System for Plant Cells," Proc. Natl. Acad. Sci. 88:10421-5 (1991), which is hereby incorporated by reference in its entirety). Expression of the transgene-encoded protein is induced in the transformed plants when the transgenic plants are brought into contact with nanomolar concentrations of a glucocorticoid, or by contact with dexamethasone, a glucocorticoid analog (Schena et al., "A Steroid-Inducible Gene Expression System for Plant Cells," Proc. Natl. Acad. Sci. USA 88:10421-5 (1991); Aoyama et al, "A Glucocorticoid-Mediated Transcriptional Induction System in Transgenic Plants,"<br><br>
Plant J. 11:605-612 (1997); McNellis et al., "Glucocorticoid-Inducible Expression of a Bacterial Avirulence Gene in Transgenic Arabidopsis Induces Hypersensitive Cell Death, Plant J. 14(2):247-57 (1998), which are hereby incorporated by reference in their entirety). In addition, inducible promoters include promoters that function in a tissue specific manner to regulate the gene of interest within selected tissues of the plant. Examples of such tissue specific or developmentally regulated promoters include seed, flower, fruit, or root specific promoters as are well known in the field (U.S. Patent No. 5,750,385 to Shewmaker et al., which is hereby incorporated by reference in its entirety).<br>
[0056]	A number of tissue- and organ-specific promoters have been developed<br>
for use in genetic engineering of plants (Potenza et al., "Targeting Transgene Expression in Research, Agricultural, and Environmental Applications: Promoters used in Plant Transformation," In Vitro Cell. Dev. Biol. Plant 40:1-22 (2004), which is hereby incorporated by reference in its entirety). Examples of such promoters include those that are floral-specific (Annadana et al, "Cloning of the Chrysanthemum UEPl Promoter and Comparative Expression in Florets and Leaves of Dendranthema grandiflora" Transgenic Res. 11:437-445(2002), which is hereby incorporated by reference in its entiret}'), seed-specific (Kluth et al., "5' Deletion of a gbssJ Promoter Region Leads to Changes in Tissue and Developmental Specificities," Plant Mol. Biol. 49:669-682 (2002), which is hereby incorporated by reference in its CTiirety), root-specific (Yamamoto et al., "Characterization of czi'-acting Sequences Regulating Root-Specific Gene Expression in Tobacco," Plant Cell 3:371-382 (1991), which is hereby incorporated by reference in its entirety), fhiit-specific (Eraser et ai., "Evaluation of Transgenic Tomato Plants Expressing an Additional Phytoene Synthase in a Fruit-Specific Manner," Proc. Natl. Acad. Sci. USA 99:1092-1097 (2002), which is hereby incorporated by reference in its entirety), and tuber/storage organ-specific (Visser et al., "Expression of a Chimaeric Granule-Bound Starch Synthase-GUS gene in transgenic Potato Plants," Plant Mol. Biol. 17:691-699 (1991), which is hereby incorporated by reference in its entirety). Targeted expression of an introduced gene (transgene) is necessaiy when expression of the transgene could have detrimental effects if expressed throughout the plant. On the other hand, silencing a gene throughout a plant could also have negative effects. However, this problem could be avoided by localizing the silencing to a region by a tissue-specific promoter.<br><br>
[0057]	The nucleic acid constmct used in the present invention also includes<br>
an operable 3' regulatory region, selected from among those which are capable of providing correct transcription termination and polyadenylation of mRNA for expression in the host cell of choice, operably linked to a modified trait nucleic acid molecule of the present invention. A number of 3' regulatory regions are known to be operable in plants. Exemplary 3' regulatory regions include, without limitation, the nopaline synthase ("nos") 3' regulatory region (Fraley et al, "Expression of Bacterial Genes in Plant Cells," Proc. Nat'I Acad. Sci. USA 80:4803-4807 (1983), which is hereby incorporated by reference in its entirety) and the cauliflower mosaic virus ("CaMV") 3' regulatory region (Odell et al, "Identification of DNA Sequences Required for Activity of the Cauliflower Mosaic Virus 35S Promoter," Nature 313(6005):810-812 (1985), which is hereby incorporated by reference in its entirety). Virtually any 3' regulatory region known to be operable in plants would be suitable for use in conjunction with the present invention.<br>
[0058]	The different components described above can be ligated together to<br>
produce the expression systems which contain the nucleic acid constructs used in the present invention, using well known molecular cloning techniques as described in Sambrook et al., Molecular Cloning: A Laboratory Manual, Second Edition Cold Spring Harbor, NY:Cold Spring Harbor Press (1989), and Ausubel et al. Current Protocols in Molecular Biology, New York, N.YiJohn Wiley &amp; Sons (1989), which are hereby incorporated by reference in their entirety.<br>
[0059]	Once the nucleic acid construct has been prepared, it is ready to be<br>
incorporated into a host cell. Basically, this method is carried out by transforming a host cell with the nucleic acid construct under conditions effective to achieve transcription of the nucleic acid molecule in the host cell. This is achieved with standard cloning procedures known in the art, such as described by Sambrook et al., Molecular Cloning: A Laboratory Manual, Second Edition, Cold Springs Laboratory, Cold Springs Harbor, New York (1989), which is hereby incorporated by reference in its entirety. Suitable host cells are plant cells. Methods of transformation may result in transient or stable expression of the nucleic acid under control of the promoter. Preferably, the nucleic acid construct of the present invention is stably inserted into the genome of the recombinant plant cell as a result of the transformation, although<br><br>
transient expression can serve an important purpose, particularly when the plant under investigation is slow-growing.<br>
■^0060]	Plant tissue suitable for transformation includes leaf tissue, root tissue,<br>
neristems, zygotic and somatic embryos, callus, protoplasts, tassels, pollen, embryos, mthers, and the like. The means of transformation chosen is that most suited to the issue to be transformed.<br>
0061]	Transient expression in plant tissue can be achieved by particle<br>
)ombardment (Klein et al., "High-Velocity Microprojectiles for Delivering Nucleic<br>
\cids Into Living Cells," Nature 327:70-73 (1987), which is hereby incorporated by<br>
■eference in its entirety), also known as biolistic transformation of the host cell, as<br>
iisclosed in U.S. Patent Nos. 4,945,050, 5,036,006, and 5,100,792, all to Sanford et<br>
il., and in Emerschad et al., "Somatic Embryogenesis and Plant Development from<br>
immature Zygotic Embryos of Seedless Grapes {Vitis vinifera)" Plant Cell Reports<br>
14:6-12 (1995), which are hereby incorporated by reference in their entirety.<br>
0062]	In particle bombardment, tungsten or gold microparticles (1 to 2 \ivci in<br>
liameter) are coated with the DNA of interest and then bombarded at the tissue using<br>
ligh pressure gas. In this way, it is possible to deliver foreign DNA into the nucleus<br>
md obtain a temporal expression of the gene under the current conditions of the<br>
issue. Biologically active particles (e.g., dried bacterial cells containing the vector<br>
md heterologous DNA) can also be propelled into plant cells. Other variations of<br>
^article bombardment, now known or hereafter developed, can also be used.<br>
0063]	An appropriate method of stably introducing the nucleic acid construct<br>
mo plant cells is to infect a plant cell vvdth Agrobacterium tumefaciens or igrobacterium rhizogenes previously transformed with the nucleic acid construct. As lescribed above, the Ti (or RI) plasmid of Agrobacterium enables the highly mccessful transfer of a foreign nucleic acid molecule into plant cells. A variation of igrobacterium transformation uses vacuum infiltration in which whole plants are jsed (Senior, "Uses of Plant Gene Silencing," .Szofec/zno/ogy and Genetic Engineering Reviews 15:79-119 (1998), which is hereby incorporated by reference in its entirety).<br>
[0064]	Yet another method of introduction is fusion of protoplasts with other<br>
sntities, either minicells, cells, lysosomes, or other fusible lipid-surfaced bodies [Fraley et al., Proc. Natl. Acad. Sci. USA 79:1859-63 (1982), which is hereby<br><br>
incorporated by reference in its entiret}'). The nucleic acid molecule may also be introduced into the plant cells by electroporation (Fromm et al, Proc. Natl. Acad. Sci. USA 82:5824 (1985), which is hereby incorporated by reference in its entirety). In this technique, plant protoplasts are electroporated in the presence of plasmids containing the expression cassette. Electrical impulses of high field strength reversibly permeabiHze biomembranes allowing the introduction of the plasmids. Electroporated plant protoplasts reform the cell wall, divide, and regenerate. Other methods of transformation include chemical-mediated plant transformation, micro¬injection, physical abrasives, and laser beams (Senior, "Uses of Plant Gene Silencing," Biotechnology and Genetic Engineering Reviews 15:79-119 (1998), which is hereby incorporated by reference in its entirety). The precise method of transformation is not critical to the practice of the present invention. Any method that results in efficient transformation of the host cell of choice is appropriate for practicing the present invention.<br>
[0065]	After transformation, the transformed plant cells must be regenerated.<br>
Plant regeneration from cultured protoplasts is described in Evans et al.. Handbook of Plant Cell Cultures, Vol. 1, New York, New York:MacMillan Publishing Co. (1983); Vasil, ed., Cell Culture and Somatic Cell Genetics of Plants, Vol. I (1984) and Vol. Ill (1986), Orlando:Acad. Press; and Fitch et al., "Somatic Embryogenesis and Plant Regeneration from Immature Zygotic Embryos of Papaya (Carica papaya L.),," Pmm Cell Rep. 9:320 (1990), which are hereby incorporated by reference in their entirety.<br>
[0066]	Means for regeneration vary from species to species of plants, but<br>
generally a suspension of fransformed protoplasts or a petri plate containing explants is first provided. Callus tissue is formed and shoots may be induced from callus and subsequently rooted. Alternatively, embryo formation can be induced in the callus tissue. These embryos germinate as natural embryos to form plants. The culture media will generally contain various amino acids and hormones, such as auxin and cytokinins. Efficient regeneration will depend on the medium, on the genotype, and on the history of the culture. If these three variables are controlled, then regeneration is usually reproducible and repeatable.<br>
[0067]	Preferably, transformed cells are first identified using a selection<br>
marker simultaneously introduced into the host cells along with the nucleic acid<br><br>
construct of the present invention. Suitable selection markers include, without limitation, markers encoding for antibiotic resistance, such as the neomycin phosphotransferae II ("nptll") gene which confers kanamycin resistance (Fraley et al., Proc. Natl Acad. Sci. USA 80:4803-4807 (1983), which is hereby incorporated by reference in its entirety), and the genes which confer resistance to gentamycin, G418, hygromycin, streptomycin, spectinomycin, tetracycline, chloramphenicol, and the like. Cells or tissues are grown on a selection medium containing the appropriate antibiotic, whereby generally only those transformants expressing the antibiotic resistance marker continue to grow. Other types of markers are also suitable for inclusion in the expression cassette of the present invention. For example, a gene encoding for herbicide tolerance, such as tolerance to sulfonylurea is useful, or the dhfr gene, which confers resistance to methotrexate (Bourouis et al, EMBOJ. 2:1099-1104 (1983), which is hereby incorporated by reference in its entirety). Similarly, "reporter genes," which encode for enzymes providing for production of an identifiable compound are suitable. The most widely used reporter gene for gene fusion experiments has been uidA, a gene from Escherichia coli that encodes the p-glucuronidase protein, also known as GUS (Jefferson et al, "GUS Fusions: P Glucuronidase as a Sensitive and Versatile Gene Fusion Marker in Higher Plants," EMBO J. 6:3901 -3907 (1987), which is hereby incoiporated by reference in its entirety). Similarly, enzymes providing for production of a compound identifiable by Ixnuinescence, such as luciferase, are useful. The selection marker employed will depend on the target species; for certain target species, different antibiotics, herbicide, or biosynthesis selection markers are preferred.<br>
[0068]	Plant cells and tissues selected by means of an inhibitory agent or other<br>
selection marker are then tested for the acquisition of the transgene (Sambrook et al..<br>
Molecular Cloning: A Laboratory Manual, Cold Spring Harbor, New York:Cold<br>
Spring Harbor Press (1989), which is hereby incorporated by reference in its entirety).<br>
[0069]	After the fusion gene containing a nucleic acid construct is stably<br>
incorporated in transgenic plants, the transgene can be transferred to other plants by sexual crossing. Any of a number of standard breeding techniques can be used, depending upon the species to be crossed. Once transgenic plants of this type are produced, the plants themselves can be cultivated in accordance with conventional procedure so that the nucleic acid construct is present in the resulting plants.<br><br>
Alternatively, transgenic seeds are recovered from the transgenic plants. These seeds can then be planted in the soil and cultivated using conventional procedures to produce transgenic plants.<br>
[0070]	An example of NAP protein that is encoded by the nucleic acid<br>
molecule used in the present invention is a NAP protein from kidney bean having an amino acid sequence of SEQ ID NO: 1, as shown in Figure 1. In addition, other examples of NAP proteins include NAP proteins from Medicago, soybean, Populus, nightshade, Arabidopsis, wheat, rice, maize, peach, tomato, petunia, and potato having amino acid sequences of SEQ ID NO: 2, SEQ ID NO: 3, SEQ ID NO: 4, SEQ ID NO: 5, SEQ ID NO: 6, SEQ ID NO: 7, SEQ ID NO: 8, SEQ ID NO: 9, SEQ ID NO: 10, SEQ ID NO: 11, SEQ ID NO: 12, and SEQ ID NO: 13, respectively, as shown in Figure 1.<br>
[0071]	The method ofthe present invention can be utilized in conjunction<br>
with plant cells from a wide variety of plants, as described supra. The present invention also relates to plants produced by the method ofthe present invention, described supra.<br>
[0072]	Another aspect ofthe present invention relates to a method for<br>
delaying leaf senescence in a plant. The method involves transforming a plant cell with a nucleic acid molecule encoding a NAP protein capable of causing leaf senescence in a plant operably associated with a promoter to obtain a transformed plant celL where expression ofthe nucleic acid molecule in the plant cell causes delayed leaf senescence by a form of post-transcriptional gene silencing. Then, a plant is regenerated from the transformed plant cell under conditions effective to delay leaf senescence in the plant.<br>
[0073]	In the aspect ofthe present invention in which delay of leaf senescence<br>
is desired, the method of interfering with endogenous NAP protein expression may involve an RNA-based form of gene-silencing known as RNA interference (RNAi) (also known more recently as siRNA for short, interfering RNAs). RNAi is a form of post-transcriptional gene silencing (PTGS). PTGS is the silencing of an endogenous gene caused by the introduction of a homologous double-stranded RNA (dsRNA), transgene, or virus. In PTGS, the transcript ofthe silenced gene is synthesized, but does not accumulate because it is degraded. RNAi is a specific from of PTGS, in which the gene silencing is induced by the direct introduction of dsRNA. Numerous<br><br>
reports have been published on critical advances in the understanding of the biochemistry and genetics of both gene silencing and RNAi (Matzke et al., "RNA-Based Silencing Strategies in Plants," Curr Opin GenetDev ll(2):221-227 (2001), Hammond et al, "Post-Transcriptional Gene Silencing by Double-Stranded RNA," Nature Rev Genl:UQ-\\9 (Abstract) (2001); Hamilton et al., "A Species of Small Antisense RNA in Posttranscriptional Gene Silencing in Plants," Science 286:950-952 (Abstract) (1999); Hammond et al, "An RNA-Directed Nuclease Mediates Post-Transcriptional Gene Silencing in Drosophila Cells," Nature 404:293-298 (2000); Hutvagner et al., "RNAi: Nature Abhors a Double-Strand," Curr Opin Genetics &amp; Development 12:225-232 (2002), which are hereby incorporated by reference in their entirety). In iRNA, the introduction of double stranded RNA (dsRNA) into animal or plant cells leads to the destruction of the endogenous, homologous mRNA, phenocopying a null mutant for that specific gene. In siRNA, the dsRNA is processed to short interfering molecules of 21-, 22- or 23-nucleotide RNAs (siRNA), which are also called " guide RAs," (Hammond et al, "Post-Transcriptional Gene Silencing by Double-Stranded RNA," Nature Rev Gen 2:110-119 (Abstract) (2001); Sharp, P.A., "RNA Interference-200]," Genes Dev 15:485-490 (2001); Hutvagner et al, "RNAi: Nature Abhors a Double-Strand," Curr Opin Genetics &amp; Development 12:225-232 (2002), which are hereby incorporated by reference in their entirety) in vivo by the Dicer enzyme, a member of the RNAse Ill-family of dsRNA-specific ribonucleases (Hurv^agner et al, "RNAi: Nature Abhors a Double-Strand," Curr Opin Genetics &amp; Development \1:11'S-11&gt;1 (2002); Bernstein et al, "Role for a Bidentate Ribonuclease in the Initiation Step of RNA Interference," Nature 409:363-366 (2001); Tuschl, T., "RNA Interference and Small Interfering RNAs," Chembiochem 2: 239-245 (2001); Zamore et al, "RNAi; Double Stranded RNA Directs the ATP-Dependent Cleavage of mRNA at 21 to 23 Nucleotide Intervals," Cell 101:25-3 (2000); United States Patent No. 6,737,512 to Wu et al, which are hereby incorporated by reference in their entirety). Successive cleavage events degrade the RNA to 19-21 bp duplexes, each with 2-nucleotide 3' overhangs (Hutvagner et al, "RNAi: Nature Abhors a Double-Strand," Curr Opin Genetics &amp; Development 12:225-232 (2002); Bernstein et al, "Role for a Bidentate Ribonuclease in the Initiation Step of RNA Interference," Nature 409:363-366 (2001), which are hereby incorporated by reference in their entirety). The siRNAs are incorporated into an effector known as the RNA-induced<br><br>
silencing complex (RISC), which targets the homologous endogenous transcript by base pairing interactions and cleaves the mRNA approximately 12 nucleotides form the 3' terminus of the siRNA (Hammond et al., "Post-Transcriptional Gene Silencing by Double-Stranded RNA," Nature Rev Gen 2:110-119 (Abstract) (2001); Sharp, P.A., "RNA Interference-2001," Genes Dev 15:485-490 (2001); Hutvagner et al, "RNAi: Nature Abhors a Double-Strand," Curr Opin Genetics &amp; Development n-.llS-lZl (2002); Nykanen et al, "ATP Requirements and Small Interfering RNA Structure in the RNA Interference Pathway," Cell 107:309-321 (2001), which are hereby incorporated by reference in their entirety).<br>
[0074]	There are several methods for preparing siRNA, including chemical<br>
synthesis, in vitro transcription, siRNA expression vectors, and PCR expression cassettes. In one aspect of the present invention, dsRNA for the nucleic acid molecule used in the present invention can be generated by transcription in vivo. This involves modifying the nucleic acid molecule for the production of dsRNA, inserting the modified nucleic acid molecule into a suitable expression vector having the appropriate 5' and 3' regulatory nucleotide sequences operably linked for transcription and translation, as described supra, and introducing the expression vector having the modified nucleic acid molecule into a suitable host or subject. Using siRNA for gene silencing is a rapidly evolving tool in molecular biology, and guidelines are available in the literature for designing highly effective siRNA targets and making antisense nucleic acid constructs for inhibiting endogenous protein (United States Patent No. 6,737,512 to Wu et al; Brown et al, "RNA Interference in Mammalian Cell Culture: Design, Execution, and Analysis of the siRNA Effect," Ambion TechNotes 9(l):3-5(2002); Sui et al, "A DNA Vector-Based RNAi Technology to Suppress Gene Expression in Mammalian Cells," Proc Natl Acad Sci USA 99(8):5515-5520 (2002); Yu et al, "RNA Interference by Expression of Short-Interfering RNAs and Hairpin RNAs in Mammalian Cells," Proc Natl Acad Sci USA 99(9).-6047-6052 (2002); Paul et al, "Effective Expression of Small Interfering RNA in Human Cells," Nature Biotechnology 20:505-508 (2002); Brummelkamp et al, "A System for Stable Expression of Short Interfering RNAs in Mammalian Cells," Science 296:550-553 (2002), which are hereby incorporated by reference in their entirety). There are also commercially available sources for custom-made siRNAs.<br><br>
[0075]	The present invention also relates to a method of making a mutant<br>
plant having a decreased level of NAP protein compared to that of a non-mutant plant,<br>
where the mutant plant displays a delayed leaf senescence phenotype relative to a<br>
non-mutant plant. The method involves providing at least one cell of a non-mutant<br>
plant containing a gene encoding a functional NAP protein. Next, the at least one cell<br>
of a non-mutant plant is treated under conditions effective to inactivate the gene,<br>
thereby yielding at least one mutant plant cell containing an inactivated NAP gene.<br>
The at least one mutant plant cell is then propagated into a mutant plant, v/here the<br>
mutant plant has a decreased level of NAP protein compared to that of the non-mutant<br>
plant and displays a delayed leaf senescence phenotype relative to a non-mutant plant.<br>
[0076]	In other embodiments of this method of making a mutant plant, the<br>
functional NAP protein can be any NAP protein from a wide variety of plants as described herein ^wpra.<br>
[0077]	In another embodiment of this method of making a mutant plant, the<br>
treating step involves subjecting the at least one cell of the non-mutant plant to a chemical mutagenizing agent under conditions effective to yield at least one mutant plant cell containing an inactive NAP gene. Suitable chemical mutagenizing agents can include, for example, ethylmethanesulfonate.<br>
[0078]	In another embodiment ofthis method ofmaking a mutant plant, the<br>
treating step involves subjecting the at least one cell of the non-mutant plant to a radiation source under conditions effective to yield at least one mutant plant cell containing an inactive NAP gene. Suitable radiation sources can include, for example, sources that are effective in producing ultraviolet rays, gamma rays, or fast neutrons.<br>
[0079]	In another embodiment of this method of making a mutant plant, the<br>
treating step involves inserting an inactivating nucleic acid molecule into the gene encoding the functional NAP protein or its promoter under conditions effective to inactivate the gene. Suitable inactivating nucleic acid molecules can include, for example, a transposable element. Examples of such transposable elements include, but are not limited to, an Activator (Ac) transposon, a Dissociator (Ds) transposon, or a Mutator (Mu) transposon.<br>
[0080]	In yet another embodiment of this method of making a mutant plant,<br>
the treating step involves subjecting the at least one cell of the non-mutant plant to<br><br>
Agrobacterium transformation under conditions effective to insert an Agrobacterium<br>
T-DNA sequence into the gene, thereby inactivating the gene. Suitable<br>
Agrobacterium T-DNA sequences can include, for example, those sequences that are<br>
carried on a binary transformation vector of pAC106, pAC161, pGABIl, pADISl,<br>
pCSAllO, pDAPlOl, derivatives of pBIN19, or pCAMBIAplasmid series.<br>
[0081 ]	In still another aspect of this method of making a mutant plant, the<br>
treating step involves subjecting the at least one cell of the non-mutant plant to site-<br>
directed mutagenesis of the NAP gene or its promoter under conditions effective to<br>
yield at least one mutant plant cell containing an inactive NAP gene. The treating<br>
step can also involve mutagenesis by homologous recombination of the NAP gene or<br>
its promoter, targeted deletion of a portion of the NAP gene sequence or its promoter,<br>
and/or targeted insertion of a nucleic acid sequence into the NAP gene or its<br>
promoter. The various plants that can be used in this method are the same as those<br>
described supra with respect to the transgenic plants and mutant plants. Other<br>
embodiments of the present invention relate to mutant plants produced by this<br>
method, as well as mutant plant seeds produced by growing the mutant plant under<br>
conditions effective to cause the mutant plant to produce seed.<br>
[0082]	The present invention also relates to a method for causing precocious<br>
leaf senescence or promoting leaf senescence in a plant. The method involves transforming a plant cell with a nucleic acid molecule encoding a NAP protein capable of caxising leaf senescence in a plant operably associated with a promoter to obtain a transformed plant cell. Next, a plant is regenerated from the transformed piam cell. Then, the promoter is induced under conditions effective to cause premature or precocious leaf senescence in the plant. The method of the present invention can be utilized in conjunction with plant cells from a wide variety of plants, as described supra. Preferably, this method is used to cause premature or precocious leaf senescence in cotton. The present invention also relates to plants produced by this method of the present invention.<br>
[0083]	Another aspect of the present invention relates to a method of<br>
identifying a candidate plant suitable for breeding that displays a delayed leaf senescence and/or enhanced yield phenotype. The method involves analyzing the candidate plant for the presence, in its genome, of an inactivated NAP gene. In one embodiment of the present invention, the method identifies a candidate plant suitable<br><br>
for breeding that displays a delayed leaf senescence phenotype. In another embodiment of the present invention, the method identifies a candidate plant suitable for breeding that displays an enhanced yield phenotype. In yet another embodiment of the present invention, the method identifies a candidate plant suitable for breeding that displays a delayed leaf senescence and enhanced yield phenotype. Because NAP gene promotes leaf senescence, if any breeding line contains a mutated NAP gene, this line will display significantly delayed leaf senescence and/or an enhanced yield phenotype. If this line is used as a parental line for breeding purposes, the NAP gene can be used as a molecular marker for selecting progenies that contain the non¬functional NAP gene. Accordingly, the NAP gene can be used as a molecular marker for breeding agronomic crops with a delayed senescence and enhanced yield. Alternatively, the NAP gene can be used as a molecular marker for breeding vegetables and flowers with delayed senescence (that do not necessarily have any concerns regarding yield).<br>
EXAMPLES<br>
[0084]	The following examples are provided to illustrate embodiments of the<br>
present invention but are by no means intended to limit its scope.<br>
Example 1 - Plant Materials and Growth Conditions<br>
[0085]	Arabidopsis plants were grown at 23 °C with 60% relative humidity<br>
under constant light (150 iimol m~~ s~' light from a mixture of fluorescent and<br>
incandescent bulbs). Seeds were sown on Petri dishes containing one-half strength of<br>
Murashige and Skoog salts, 0.8% (w/v) phytoagar (Sigma, St. Louis, MO), and<br>
appropriate antibiotics. After imbibition, the seeds were kept at 4°C overnight. Two-<br>
week-old seedlings were transplanted to Cornell mix soils (3 parts peat moss: 2 parts<br>
vermiculite: 1 part perlite; Tower Road Green house, Cornell University, Ithaca, NY).<br>
[0086]	Arabidopsis thaliana ecotype Columbia-0 was used. The T-DNA<br>
insertion lines, the inducible overexpression lines, and the complementation lines were grown side by side with wild-t}'pe and other control lines unless indicated otherwise.<br><br>
Example 2 - Isolation of T-DNA Insertions Within AtNAP<br>
[0087]	Two Arabidopsis lines for T-DNA insertions m AtNAP, SALK_0050] 0<br>
(line 1) and SALK_004077 (line 2) were obtained from the Salk T-DNA collection (Alonso et al., "Genome-Wide Insertional Mutagenesis of Arabidopsis Thaliana," Science 301:653-657 (2003), which is hereby incorporated by reference in its entirety). A PCR-based method was used to identify homozygous mutant plants. Genomic DNA was prepared from a small piece of leaf using a modified CTAB method (Murray et al, "Rapid Isolation of High Molecular Weight Plant DNA," Nucleic Acids Res 8:4321-4325 (1980), which is hereby incorporated by reference in its entirety). Briefly, 50-100 mg fresh leaf tissue was ground in a 1.5-ml microcentrifuge tube with a Craftsman 9-inch drill press (Sears, Roebuck and Co., Hoffnian Estates, IL). The powdered samples were incubated at 55°C for 30 min after 500 ^1 2x extraction buffer (0.7 M NaCl, 1% w/v CTAB, 50 mM Tris (pH 8.0), 10 mM EDTA, 1% beta-ME added fresh) was added. After incubation, 500 ^1 chloroform:isoamyl alcohol 24:1 was added and mixed, and the samples were centrifuged for 10 min at 13,000 g. The aqueous phase (approximately 500 )u,l) was transferred to a new microfuge tube and 500 )xl isopropanol was added to precipitate genomic DNA. PCR was used to amplify the genomic DNAs. The PCR conditions were as follows: 35 cycles with each cycle consisting of 94°C for 30 sec, 55°C for 30 sec, and 72°C for 1 min. T-DNA left border primer G1099 (5'-GCGTGGACCGC TTGCTGCAACT-3'; SEQ ID NO: 14) and gene-specific primers G1027 (5'-ATCAT GGAAG TAACTTCCCAATC-3'; SEQ ID NO: 15) and G1028 (5'-TTCAGTTCTTC TCTCTGCTTC-3'; SEQ ID NO: 16) for line 1, G1273 (5'-GGCCATTTTCTACGCT ACCT-3'; SEQ ID NO: 17) and Gl 123 (5'-CTTCCATGGTTTTCAGACAATTTAG -3'; SEQ ID NO: 18) for line 2 were used in the PCR reactions.<br>
Example 3 - Plasmid Construction<br>
[0088]	The GFP-AtNAP expression plasmid pGLl 185 was generated by<br>
cloning AtNAP coding region into pRTL2-S65TGFP (Lin et al., "Arabidopsis FHY3/FAR1 Gene Family and Distinct Roles of Its Members in Light Control of Arabidopsis Development," P/anr P/z^^/o/. 136:4010-4022 (2004), which is hereby incorporated by reference in its entirety). The coding region without the stop codon<br><br>
was amplified via PCR reaction using primers G1526 (5'-TAGTCGACAGTTCCTG TTCTATTAGATTG-3'; SEQ ID NO: 19; the underlined section is an engineered Sail site) and G1527 (5'-TATCATGAACTTAAACATCGCTTGACG-3': SEQ ID NO: 20; the underlined section is an engineered BspHI site). Pfli polymerase (Stratagene, La Jolla, CA) was used and the PCR product was sequenced. The PCR product cut with Sail and BspHI was cloned into pRTL2-S65TGFP at Xhol and Ncol sites.<br>
[0089]	For inducible overexpression of AtNAP, the 320 bp fragment of 6x<br>
GAL4 UAS and 358 TATA region from pTA7001 (Aoyama et al, "A Glucocorticoid-Mediated Transcriptional Induction System in Transgenic Plants," Plant J11:605-612 (1997), which is hereby incorporated by reference in its entirety) was cloned into a binary vector called pPZP211 (Hajdukiewicz et al, "The Small, Versatile pPZP Family of Agrobacterium Binary Vectors For Plant Transformation," Plant Mol Biol 25:989-994 (1994), which is hereby incorporated by reference in its entirety) to form pGLl 152. The full length cDNA ofAtNAP (907 bp, including 43 bp 5' UTR region and 57 bp 3' UTR region) was amphfied with primers Gl 100 (5'-CACTAGTTCCTGTTCTATTAGATTG-3'; SEQ ID NO: 21; the underlined section is an engineered Spel site) and Gl 101 (5'-GGTGCAGTAACTTTTCAAGCACATC-3'; SEQ ID NO: 22; the underlined section is an engineered PstI site) using Pfii polymerase. The PCR product, after an A-tailing procedure described by the maiEiiacmrer (Promega, Madison, \\T), was cloned into the pGEM-T vector (Promega) to formpGL1165. The plasmid was then sequenced. pGL1165was digested with Spel and PstI and the released AtNAP coding region was subcloned into pGLl 152, resulting in pGLl 167.<br>
[0090]	For the complementation test involving the Arabidopsis wild-type<br>
AtNAP, a 3166-bp genomic DNA containing the promoter (1961 bp) and coding region (1205 bp) of AtNAP was PCR amplified with primers G1628 (5'-GCGTCATC TCATCCTAATCCTCAT-3'; SEQ ID NO: 23) and G1629 (5'-CGTGACTTCGTCT TATCATGCTG-3'; SEQ ID NO: 24) using Pfu polymerase, and cloned into pGEM-T after A-tailing, to form pGLl 186 that was subsequently sequenced. pGLl 186 was digested with SacII, followed by treatment with T4 DNA polymerase (NEB, Beverly, MA) to remove the 3' overhangs to form blunt ends. The plasmid was fiirther digested with Sad and the released AtNAP was cloned into the binary vector<br><br>
pPZP221 (Hajdukiewicz et al., "The Small, Versatile pPZP Family of Agrobacterium Binary Vectors For Plant Transformation," Plant Mol Biol 25:989-994 (1994), which is hereby incorporated by reference in its entirety) at the Sad and Smal sites. The construct was named pGL 1199.<br>
[0091]	When using the rice {Oryza sativa, japonica cultivar group) homolog<br>
of AtNAP (Os NAP) for complementation, primers G1807 (5'-TTCTGCAGCGTCAT CTCATCCTAATCCTCAT -3'; SEQ ID NO: 25; the underlined section is an engineered PstI site) and G1808 (5'-GTTACTTCCAIGGTTTTCAGACAATTTAG¬S'; SEQ ID NO: 26; the underlined section is an engineered Ncol site) were used to PCR amplify the AtNAP promoter region. After an A-tailing procedure, the 2 kb PCR product was cloned into pGEM-T to form pGL1193. Genomic fragment containing the coding region ofO&amp;NAP (NP_912423) was PCR amplified using primers G1805 (5'-TTCjCATGGTTCTGTCGAACCCG-3'; SEQ ID NO: 27; the underlined section is an engineered Ncol site) and G1666 (5' -GATCTAGACGAAGAACGAGCTATCA G-3'; SEQ ID NO: 28). The 1.8 kb PCR product was cloned into pGEM-T to form pGL1191. The plasmids were sequenced. OsA/^P released from pGLl 191 upon Ncol digestion was then cloned into pGLl 193 to form pGL1195. The 3.8 kb chimeric gene was then released from pGL1195 after digestion with Sad and Apal (3' overhangs removed by T4 DN A-polymerase treatment) and cloned into pZP221 at Sad and Smal to form pGLl 197. A nos terminator was added to the end of the chimeric gene in pGLl 197 at the Xbal site to form pGL1800.<br>
[0092]	For complementation test involving the kidney bean (Phaseolus<br>
\'ulgaris) NAP homolog (?\N^AP\ primers G1807 (see above) and G1809 (S'-AAGICGACGATTTTCAGACAATTTAGAAAACAATC-S'; SEQ ID NO: 29; the underlined section is an engineered Sail site) were used to PCR amplify the AtNAP promoter region. The 2 kb PCR product was cloned into pGEM-T to form pGLl 194. The genomic fragment containing the coding region of PvNAP (AAK84884) was PCR amplified using primers G1806 (5 '-AAGTCGACATGGATGCTACCACACCC TC-3'; SEQ ID NO: 30; the underlined section is an engineered Sail site) and G1668 (5'-GATCTAGATGGACGAAGCTTATCGTC-3'; SEQ ID NO: 31). The 1.3 kb PCR product was cloned into pGEM-T to form pGLl 190. The plasmids were sequenced for sequence confirmation. The PvNAP coding region released from pGLl 190 by Sail was then cloned into pGLl 194 to form pGLl 196. The 3.1 kb<br><br>
chimeric gene was released from pGLl 196 by PstI and cloned into pPZP221, forming pGLl 198. A nos terminator was added to the end of the chimeric gene in pGLl 198 at the Xbal site to form pGLl 801.<br>
Example 4 - Agrobacterium and Plant Transformation<br>
[0093]	The above constructs in binary vectors (pGL 1167, pGL 1199, pGL 1800<br>
and pGL1801) were transferred into Agrobacterium tumefaciens strain ABI as previously described in He et al., "A Gene Encoding an Acyl Hydrolase is Involved in Leaf Senescence in Arabidopsis," Plant Cell 14:805-815 (2002), which is hereby incorporated by reference in its entirety. Similarly, pTA7001 was transferred into A. tumefaciens strain LBA4404. The ^^Lgrobacterium cells containing the respective constructs were then used to transform Arabidopsis ecotype Columbia-0 or the atnap null mutant plants via vacuum infiltration (Bechtold et al., "In Planta Agrobacterium-Mediated Gene Transfer by Infiltration of Adult Arabidopsis Plants," C. R. Acad. Sci. Paris 316:1194-1199 (1993), which is hereby incorporated by reference in its entirety). Transgenic plants were selected on plates containing 50 mg/1 kanamycin (for pGLl 167 transformants), 80mg/l gentamycin (pGLl 199, pGLl800 and pGLl801 transformants) or 25 mg/1 hygromycin (pTA7001 transformants). Arabidopsis plants harboring pGLl 167 were crossed with plants harboring pTA7001 and the hybrids were selected on plates containing both kanamycin (50 mg/1) and hygromycin i-iD nig,i).<br>
Example 5 - RNA Gel Blot and RT-PCR Analyses<br>
[0094]	Total RNA extraction from Arabidopsis leaves and RNA gel blot<br>
analysis were performed as described in He et al, "A Gene Encoding an Acyl Hydrolase is Involved in Leaf Senescence in Arabidopsis," Plant Cell 14:805-815 (2002), which is hereby incorporated by reference in its entirety. The hybridization was performed at 65°C. The Ambion RetroScript Kit (Ambion, Austin, TX) was used to perform RT-PCR analysis according the manufacturer's instruction. The QuantumRNATM Universal 18S Internal Standard Kit (Ambion) was used for equal loading control. DNA fragments for making related hybridization probes were PCR amplified using the following primers: G1027 and G1028 (see above) for AtNAP, GIO<br><br>
(5'-CAGCTGC GGATGTTGTTG-3'; SEQ ID NO: 32) and G246 (5'-CCACTTTCT CCCCATTTTG-3'; SEQ ID NO: 33) for SAG12, G9 (5'-GCAACCAAAGGAGCCA TG-3'; SEQ ID NO: 34) and G16 (5'-GTTTGGCCAACTAGTCTGC-3'; SEQ ID NO: 35) for SAG13, G1148 (5'-AG TAATGGCTTCCTCTATGC-3'; SEQ ID NO: 36) and G1149 (5'-GGCTTGTAGGCAATGAAACT-3'; SEQ ID NO: 37) for RuBISCO small subunit gene RBCS, G1665 (5' -ATCCCTTCCATTTCCG AC-3'; SEQ ID NO: 38) and G1666 (see above) for O&amp;NAP, G1667 (5'- CTGGGTCTTGTG CAGAAT-3'; SEQ ID NO: 39) and G1668 (see above) for ?wNAP. Some of the primers were also used for related RT-PCR analysis.<br>
Example 6 - Transient Gene Expression in Onion Epidermal Cells<br>
[0095]	Onion {Allium cepa) epidermal cells were transfected with pGLl 185<br>
using heUum biolistic gun transformation system (Bio-Rad, Hercules, CA) as described in Lin et al, "Arabidopsis FHY3/FAR1 Gene Family and Distinct Roles of Its Members in Light Control of Arabidopsis Development," Plant Physiol. 136:4010-4022 (2004), which is hereby incorporated by reference in its entirety, and incubated in light or darkness for 24-48 h at 22°C. The subcellular localization of GFP fusion proteins was visualized with a fluorescence microscope.<br>
Example 7 - Glucocorticoid Treatments<br>
[0096]	The glucocorticoid treatments were performed as described by<br>
Aoyama st al.. "A Glucocorticoid-Mediated Transcriptional Induction System in Transgenic Plants," Plant J11:605-612 (1997), which is hereby incorporated by reference in its entirety. Two-week-old plants grown in pots were sprayed with 30 jxM dexamethasone (DEX). The plants were sprayed once a day for 2 days and incubated for two additional days. Fy/Fm ratios of leaves of these plants were measured, and leaves were harvested for molecular analysis.<br>
Example 8 - Dark-Induced Leaf Senescence<br>
[0097]	Leaf number 6 from a 3-week-old Arabidopsis plant was excised and<br>
placed on moisturized filter papers in Petri dishes with adaxial side facing up. The plates were kept in darkness at 23°C for 4 days.<br><br>
Example 9 - Measurements of Chlorophyll Content, Fluorescence, and Ion Leakage<br>
[0098]	Chlorophyll was extracted and quantitated as described previously in<br>
He et al, "A Gene Encoding an Acyl Hydrolase is Involved in Leaf Senescence in Arabidopsis," Plant Cell 14:805-815 (2002), which is hereby incorporated by reference in its entirety. Fluorescence in leaves was measured using a portable modulated chlorophyll fluorometer (model: OSl-FL) according to the manufacturer's instructions (Opti-Sciences, Tyngsboro, MA). The ratio of variable fluorescence to maximal fluorescence (Fv/Fm) of each leaf was quantified directly using the fluorometer's test mode 1. For ion leakage, leaves were immersed into deionized distilled water, shaken in a 25°C water bath for 30 min, and the conductivity was measured using a digital conductivity meter (Fisher Scientific Traceable, Hampton, NH). Samples were boiled for 10 min and then monitored for conductivity. The percentage of the first measurement over the second measurement was used as the membrane leakage indicator.<br>
Example 10 - Data Mining from the Genevestigator Microarray Database<br>
[0099]	The "Gene Atlas" program of the microarray database Genevestigator<br>
(v&gt;^\^^genevestigator.ethz.ch) was used to search expression levels ofAtNAP ('Atlg69490) in different plant tissues. The program "Response Viewer" was used to search expression change and expression levels ofAtNAP under different treatments. To run both programs, chip type "ATHl :22k array" for "wild type only" were used. When running "Response Viewer", chips from all sources were selected in "ATHl :22k array" for "wild type only" chip type.<br>
Example 11 - Molecular Phylogenetic Analyses<br>
[0100]	The amino acid sequence of AtNAP was used to search different<br>
genomic databases including GenBank nitt,p://wvt'w.ncbi.nlm.nih.gov/BLAST/ for soybean, kidney bean, rice, nightshade, wheat, peach, tomato, petunia, and potato), the TIGR plant genome databases Chttp://wvvav.tii2r.org/plantProiects.shtml for maize and Medicago), and PopulusDB (http://www;popu]us.db.umu.se/ for Populus). The<br><br>
NAC family genes with highest sequence similarity with AtNAP from different plant species including kidney bean {Phaseolus vulgaris) (AAK84884), rice {Oiyza sativd) (NP_912423), soybean {Glycine max) (AAY46121), nightshade (Solanum demissum) (AAU90314), Medicago truncatula (AC140030_19.1), Populus trichocarpa (gene model gwl.X. 1066.1), wheat {Triticum aestivum) (AAU08785), maize (Zea mays) (AZM5_18141), peach {Prunus persica) (CAG28971), tomato (Lycopersicon esculentum) (AAU43923), potato {Solanum tuberosum) (AAU12055), and petunia {Petunia x hybrida) (AAM34773), were used for molecular phylogenetic analyses. The kidney bean and rice NAP homologs, which were further studied for their expression patterns and used to transfer atnap mutant in heterogeneous complementation tests, are referred to, in the present application, as VvNAP, and OsNAP, respectively. Predicted amino acid sequences of AtNAP homologs from different plant species were first aligned using the alignment program CLUSTALW (Chenna et al., "Multiple Sequence Alignment With the Clustal Series of Programs," Nucleic Acids Res 31:3497-3500 (2003), which is hereby incorporated by reference in its entirety) with the default parameter values (aUgnment algorithm: full; CPU mode: single; Kimura correction: off; output: alnl; output order: aligned; score type: percent; ignore gaps in alignment: off; Nixmber of sequences: 13; Figure 1). The alignments were then used to produce the phylogeny using the phylogenetic analysis program MEGA3-1 (Kumar et al, "MEGA3: Integrated Software for Molecular Evolutionary Genetics Ajial\^is and Sequence Alignment," Brief Bioinform 5:150-163 (2004), w^hich is hereby incorporated by reference in its entirety). Parameters used in this analysis w^ere: data type: amino acid; analysis: phylogeny reconstruction; method: neighbor-jointing method; gaps/missing data: complete deletion; model: amino:Poisson correction; substitutions to include: all; pattern among lineages: same (homogeneous). The bootstrap values for nodes in the phylogenetic tree are from 1000 replications.<br>
Example 12 - AtNAP is Upregulated During Leaf Senescence in Arabidopsis<br>
[0101]	Digital expression profile analysis of the Arabidopsis leaf senescence<br>
dbEST and microarray analysis (Buchanan-Wollaston et al, "Comparative Transcriptome Analysis Reveals Significant Differences in Gene Expression and<br><br>
Signalling Pathways Between Developmental and Dark/Stan^ation-Induced Senescence in Arabidopsis," The Plant Journal 42:567-585 (2005), which is hereby incorporated by reference in its entirety) revealed ihaXAtNAP is one of the most abundantly transcribed transcription factor genes in senescing leaves (Guo et al., "Transcriptome of Arabidopsis Leaf Senescence," Plant Cell and Environment 27:521-549 (2004), which is hereby incorporated by reference in its entirety). RNA gel blot analysis showed that the expression of AtNAP in rosette leaves of Arabidopsis was closely associated with the progression of leaf senescence (Figures 2(a)-(c)). When leaves from the same phyllotactical position (leaf number 6 from the bottom of the plant) were studied, RNA messenger (mKNA) of AtNAP was detected only when the leaves started senescing (2-3 weeks after emergence; Figure 2(a)). Leaf senescence in Arabidopsis grown under non-stressful conditions was age-dependent and progressed sequentially from the oldest leaf (the first leaf at the bottom of a plant) to the top young leaves. As shown in Figure 2(b), AtNAP transcript was detected in the old, senescing leaves, but not in the young, green leaves. In a given leaf, senescence started from the leaf tip and progressed toward leaf base (petiole). The yellow tip showed stronger AtNAP expression than the proximal part of a leaf (Figure 2(c)).<br>
Example 13 - AtNAP is Targeted to Nuclei<br>
[0102]	Although AtNAP is predicted to be a nuclear protein by PredictNLS<br>
(Cokol et aL, ^Tinding Nuclear Localization Signals," EMBO i?ep 1:411-415 (2000), which is hereby incorporated by reference in its entirety) and PSORT (Nakai et al., "A ICnowledge Base for Predicting Protein Localization Sites in Eukaryotic Cells," Genomics 14:897-911 (1992), which is hereby incorporated by reference in its entirety), it does not have any obvious nuclear localization signal. To determine the subcellular localization of AtNAP, a chimeric gene containing a GFP-AtNAP construct driven by the 3 5S promoter was transiently expressed in onion (Allium cepa) epidermal cells using particle bombardment. Subcellular localization of the GFP fusion protein was visualized with a fluorescence microscope. DAPI (4',6'-diamidino 2-phenylindole) staining of DNA revealed GFP-AtNAP protein<br><br>
localization in the nuclei, suggesting that AtNAP is a nuclear protein (Figures 3(a)-(d)).<br>
Example 14 - The AtNAP Expression is Knocked Out in One T-DNA Line and Knoclced Down in Another Line<br>
[0103]	The AtNAP gene consists of three axons and encodes a protein with<br>
268 amino acids (Figure 4(a)). Two Salk T-DNA lines (Columbia background) were obtained from the Arabidopsis Biological Resource Center (ABRC) at Ohio State University (Alonso et al, "Genome-Wide Insertional Mutagenesis of Arabidopsis Thaliana," Science 301:653-657 (2003), which is hereby incorporated by reference in its entirety). Line 1 (SALK_005010) has a T-DNA insertion in the second exon, and line 2 (SALK_004077) has a T-DNA insertion in the promoter region (at -227 from the translation start site; Figure 4(a)). RJSJA gel blot analysis showed that the AtNAP transcript in senescent leaves of the homozygous line 1 was not detectable, while AtNAP transcript levels in senescent leaves of line 2 plants was reduced to 5% of that in age-matched wild-type leaves (Figui'e 4(b)). This suggested that line 1 was an atnap null mutant, while line 2 was a knockdown line.<br>
Example 15 - Leaf Senescence is Significantly Delayed in the AtNAP Null Mutant Plants<br>
[61 ©4]	To compare any phenotypic changes in growth and development<br>
among line 1 (the atnap null mutant), line 2 (the atnap knockdown mutant) and wild lype (Coltnnbia accession), these plants w-ere gro^wTi side by side in an Arabidopsis growth chamber. There were no visible differences in growth and development, except for the significantly delayed leaf senescence phenotype in the atnap null plants, and the less significantly retarded leaf senescence phenotype in the atnap knockdown plants (Figures 5(a)-(d)).<br>
[0105]	The «?««/? null mutant plants were further characterized. As shown in<br>
the mortality curves in Figure 6(a), leaves of the atnap null mutant plants senesced later than those of wild-type plants. Consistent with a delayed visible yellowing phenotype (Figures 5(a)-(d)), chlorophyll levels in individual rosette leaves of the null line were generally higher than in counterpart leaves of the age-matched wild-type plants (Figure 6(b)). The Fy/Fm ratios in individual leaves of the null line were also<br><br>
higher than in counterpart leaves of the age-matched wild-type plants (Figure 6(c)). The Fv/Fm ratio reflects the photochemical quantum efficiency of photosystem II, as well as the photoreduction efficiency of the primary electron-accepting plastoquinone of photosystem II. In contrast, ion leakage in individual leaves of the null plants was less than that in wild-type plants (Figure 6(d)). Ion leakage is an indicator of intactness of plasma membrane. The plasma membrane of a senescing cell becomes fragile and leaky.<br>
[0106]	The expression of SAGl2 and the Rubisco small subunit gene (RBCS)<br>
was also monitored. SAG J 2 is a highl}' senescence-specific gene in Arabidopsis and<br>
has been widely used as a molecular marker for leaf senescence, while RBCS is a<br>
typical senescence downregulated gene. As shown in Figure 6(e), the expression of<br>
SAGl 2 was readily detectable in leaf number 7 of a 30-day-old wild-type plant, but it<br>
was barely detectable in leaf number 4 of an age-matched null plant.<br>
[0107]	All the data described above indicated that the leaf senescence process<br>
was dramatically delayed in the ama/7 mutant plants (approximately 10 days).<br>
Example 16 -AtNAP Restores the atnap Null Mutant Plants to Wild Type<br>
[0108]	To confirm that the T-DNA insertional null mutation in AtNAP was<br>
responsible for the delayed senescence phenotype, a complementation test experiment was performed. The wild-type copy of AtNAP, including the 2 kb promoter region, was iimodaced into the atnap niill mutant plants. The introduced AtNAP was expressed in senescing leaves (see "nap + AtNAP" panel in Figure 7(a)). The senescence phenotype in leaves that were either detached or in planta was characterized. The leaves detached from the AtNAP complemental lines senesced in the same manner as those leaves from wild type did, both phenotypically (Figure 7(b)), and in terms of the Fv/Fm ratio (Figure 7(c)). In planta leaves of the complemental plants also senesced in the same manner as wild-type leaves (Figure 7(d)). These data confirmed that loss of AtNAP expression in the atnap null mutant was the only cause of the delayed senescence phenotype.<br><br>
Example 17 - Inducible Overexpression of AtNAP Causes Precocious Senescence<br>
[0109]	The role of AtNAP in leaf senescence was further investigated by<br>
performing gain-of-function analysis. Considering the fact that constitutive expression of this gene might be lethal, the chemical inducible gene expression system (Aoyama et al., "A Glucocorticoid-Mediated Transcriptional Induction System in Transgenic Plants," Plant J11:605-612 (1997), which is hereby incorporated by reference in its entirety) was used. First, transgenic lines that harbored either the pTA7001 orpGL1167 constructs (Figure 8(a)) were generated. pTA7001 contained the chimeric transcription factor GVG consisting of a DNA-binding domain of the yeast transcription factor GAL4, a transactivation domain of the herpes simplex virus transcriptional regulatory protein VP16 and a glucocorticoid receptor domain (Figure 8(a)), while pGLl 167 was a construct in which AtNAP is driven by a promoter containing six tandem copies of the GAL4 upstream activation sequence (Figure 8(a)). Treatment with dexamethasone (DEX, a synthetic glucocorticoid) caused precocious leaf yellowing (Figure 8(b)) and a significant reduction of the Fv/Fm ratio (Figure 8(c)) in Fl plants (pGLI 167 homozygous plants x pTA7001 homozygous plants) but not in controls (wild type, plants containing pGLl 167 or pTA7001 only). RNA blot analysis showed that AtNAP expression was strongly induced in the Fi plants but not in the controls (Figure 8(d)). The precocious leaf yeIlo"v\Tiig was a senescence process because SAG 12 and SAG 13 were both expressed i.Tigiire 8(ifVi. S4G12 and SAG13 are leaf senescence-specific marker genes. These data suggested that AtNAP was sufficient to promote leaf senescence.<br>
Example 18 - AtNAP Homologs in Rice and Kidney Bean are Specifically Expressed in Senescing Leaves<br>
[0110]	The amino acid sequence of AtNAP was analyzed with the use of<br>
different genomic databases such as the GenBank<br>
rhttp://www.ncbi.nlm.nih.gov/BLAST/). the TIGR plant genome databases (http://www.tigr.org/plantProiects.shtmn. and PopulusDB rhttp://www.populus.db.umu.se/\ and genes with high sequence similarity were identified from many different plant species (Figure 9(a)). Among them are the NAG family transcription factor PvNAP (256 amino acids; AAK84884) from the dicot<br><br>
kidney bean (Phaseolus vulgaris) with 66% identity, and the NAC family transcription factor OsNAP (392 amino acids; NP_912423) from the monocot rice (Oryza sativa) with 70% identity (Figure 9(b)). It was hypothesized that those homologs were functional orthologs of AtNAP. To test this hypothesis, it was first examined whether VvNAP and OsNAP shared the same leaf senescence-specific expression pattern as AtNAP. Kidney bean leaves at five distinct developmental stages, ranging from young leaves to entirely yellow leaves (Figure 9(c)), were used for RNA gel blot analysis ofFvNAP expression. As shown in Figure 9(c), 'PvNAP transcript was detected in senescing leaves only. The expression of O&amp;NAP was also shown to be senescence specific in rice leaves (Figure 9(d)).<br>
Example 19 -AtNAP Homologs in Rice and Kidney Bean are Able to Restore the Arabidopsis atnap Null Mutant to Wild Type<br>
[0111]	To further test the hypothesis that the homologs are functional<br>
orthologs of the Arabidopsis AtNAP, heterogeneous complementation tests were performed. The 2 kb AtNAP promoter was used to direct the expression of the coding region of OsNAP or PvNAP. These genes were expressed in senescent leaves of respective complementation lines (Figure 7(a)). Phenotypically, fully expanded non-senescing leaves detached from wild-tj'pe plants became senescent after being incubated in darkness for 4 days. In contrast, age-matched leaves from atnap null muran*: 'Tine 1) remained green (Figure 7(b)) and photosynthetically active fjigure ^fc)). However, the leaves of the null plants complemented with OsIVAP or P\"A'.4P senesced like those of wild tj-pe (Figure 7(b)). Similar observations were obtained when natural leaf senescence was examined in intact plants (Figure 7(d)). Similarly to AtNAP, OsNAP and PvNAP were able to restore the Arabidopsis null mutant to wild type, which suggests that OsNAP and PvNAP are functional orthologs of AtNAP.<br>
)     Example 20 - RT-PCR Analysis of Soybean GmNAP and Maize ZmNAP During Leaf Senescence<br>
[0112]	RT-PCR analysis of the expression of the AtNAP homologs in soybean<br>
{Glycine max) (AAY46121) {GmNAP) and maize {Zea mays) (AZM5_18141)<br><br>
(ZmNAP) was performed. As shown in Figures 10(a)-(b), both GmNAP and ZmNAP were upregulated during leaf senescence of soybean and maize, respectively.<br>
Example 21 - Use of RNA Interference to Suppress thtAtNAP Homologs in Soybean {GmNAP) and Maize {ZmNAP)<br>
[0113]	The NAP gene is a master regulator that promotes leaf senescence.<br>
RNA interference (RNAi) approach was used to silence this gene in soybean and<br>
maize so that leaf senescence would be dramatically delayed. RNAi uivolves the use<br>
of double strand RNA (dsRNA) that can efficiently cause silencing of target gene and<br>
has, thus, been widely used in genetics. The dsRNA construct contains inverted<br>
repeats of a small fragment of the gene of interest, i.e., GmNAP or ZmNAP, so that,<br>
upon transcription, the RNA transcript will pair each other to form double strand<br>
between the inverted repeat sequences. The dsRNA triggers the cellular machinery to<br>
destroy any mRNA whose sequence matches with the repeat sequence. Therefore,<br>
RNAi constructs were prepared to suppress the expression of GmNAP and ZmNAP in<br>
soybean and maize, respectively, where the suppression of the homolog was expected<br>
to lead to the significant delay of leaf senescence in the respective crops.<br>
[0114]	For GmNAP RNAi, primers G2132 f5'-TCTAGAGGCAAAAGAGGA<br>
CTAC-3'; SEQ ID NO: 40; the underUned section is an engineered Xba I site) and G2133 (5'-GGATCCTGGTACTTCCCTGAATCT-3'; SEQ ID NO: 41; the undaiinsd section is an engineered BamH I site) were used to PCR amplify a 271 -bp ftagmeni of the 3' end of GmNAP cDNA. The PCR product was cloned into pGEM-T (Tromega, Madison, WT) to form plasmid pGL 1818. pGL 1818 was cut by Xba I after BamH I digestion and Klenow fill-in. The released GmNAP cDNA fragment was cloned into a binary vector pGLl 100 at Xba I and Spe I (filled-in) sites, to form pGLl820. pGLllOO contains a 690-bp intron sequence of Arabidopsis AtWRKY75 between the 35S promoter and the RBS (Rubisco small subunit gene) terminator. Another copy of the GmNAP cDNA fragment released by Sac II and Xba I (filled-in) from pGL1818 was cloned into pGLI820 at Sac II and Hind III (filied-in) sites to form pGL1822 such that pGL1822 contains two copies of the 271-bp GmNAP cDNA in opposite orientation separated by the 690-bp intron. The 35S promoter-RNAi with intron-RBS terminator in pGL1822 was PCR amplified with primers G205 (5'-GGAATTCGCCCGGGGATCTCCTTTG-3'; SEQ ID NO: 45; the underiined section<br><br>
is an engineered EcoR I site) and G206 (5'-TAGGCCTTGATGCATGTTGTCAATC<br>
AATTG-3'; SEQ ID NO: 42; the underlined section is an engineered Stu I site). The<br>
PCR product was cut by EcoR I and Stu I and cloned into a binary vector at the EcoR<br>
I and Hind III (fiUed-in) sites, to form pGL1823 (Figure 11(a)).<br>
[0115]	The above GmNAP RNAi silencing construct was used to transform<br>
soybean genotype Thome (Ohio State University) using the soybean cotyledonary-node Agrobacterium-mQdia.tQd transformation system as described at the website (http://www.biotech.unl.edu/transgenic/protocols.btnil'). Eighteen transgenic lines were generated.<br>
[0116]	For ZmNAPRNAi, primers G2136 (5'- ICTAGACGGAGCTGTTCA<br>
AC-3'; SEQ ID NO: 43; the underlined section is an engineered Xba I site) and<br>
G2137 (5'-AAGCTTAGAGTGAAGCGGCAT-3': SEQ ID NO: 44; the underlined<br>
section is an engineered Hind III site) were used to PCR amplify a 410-bp fragment of<br>
the 3' end of ZmNAP cDNA. The PCR product was cloned into pGEM-T to form<br>
pGLl 817. The plasmid of pGLl 817 was cut by Xba I after Hind III digestion and<br>
Klenow fill-in. The released ZmNAP cDNA fragment was cloned into pGLl 100 at<br>
Xba I and Spe I (filled-in) sites, to form pGL1819. Another copy of the ZmNAP<br>
cDNA fragment released by Sac II and Xba I (filled-in) from pGL1817 was cloned<br>
into pGL1819 at Sac II and Hmd III (filled-in) sites to form pGL1821 such that<br>
pGL1821 contains two copies of the 410 bp ZmNAP cDNA in opposite orientation<br>
sepaxHSsd "rj'iiie 690-bp intron sequence of Arabidopsis AtWRKY75 (Figure 11 (b)).<br>
[0117]	The above ZmNAP RNAi silencing construct was transferred into<br>
Agrobacrerium mmefaciens strain .ABI to prepare them for plant transformation.<br>
Example 22 -AtNAP Homologs in Maize and Soybean are Able to Restore the Arabidopsis atnap Null Mutant to Wild Type<br>
[0118]	Using a similar strategy as described in Example 19 above, it was<br>
examined whether maize and soybean NAPs, i.e., ZmNAP and GmNAP, respectively, can complement the Arabidopsis atnap null mutant. The 2-kb AtNAP promoter was used to direct the expression of the coding region oi ZmNAP or GmNAP. These genes were expressed in senescent leaves of respective complementation lines. Natural leaf senescence was examined in intact plants, where phenotypically leaves of wild-type plants became senescent (Figures 12(a)-(b)). In contrast, leaves of atnap null mutant<br><br>
remained green (Figures 12(a)-(b)). However, the leaves of the null plants complemented with ZmNAP or GmNAP senesced like those of wild type (Figures 12(a)-(b)). Therefore, ZmNAP and GmNAP,w&amp;ve able to restore the Arabidopsis null mutant to wild type.<br>
[0119]	Leaf senescence limits crop yield and biomass accumulation. In<br>
soybean, insufficient phosphate (Pi) nutrient during soybean seed development promotes leaf senescence. When leaf senescence was significantly delayed (for eight days) by supplemental Pi via stem infusions, soybean grain yield was increased as much as 3-fold (Grabau et al, "P Nutrition During Seed Development: Leaf Senescence, Pod Retention, and Seed Weight of Soybean," Plant Physiol. 82:1008-1012 (1986), which is hereby incorporated by reference in its entirety). Both field and greenhouse experiments also proved that leaf senescence limits soybean yield by restricting the seed filling period (Hayati et al., "Carbon and Nitrogen Supply During Seed Filling and Leaf Senescence in Soybean," Crop Sci. 35:1063-1069 (1995), which is hereby incorporated by reference in its entirety).<br>
[0120]	The effect of delayed leaf senescence on yields has also been observed<br>
in maize and many other crops. For example, a retrospective analysis over 50 years (1930-1980) of hybrid maize data in US revealed that late onset of leaf senescence contributed to significant increases in maize yields (Duvick, "Genetic Contribution to Yield Gains of U.S. Hybrid Maize 1930-1980," in Fehr, ed., Genetic Contributions to Yield Gains of five Major Crop Plants, vol. 7, Crop Science Society of America, Madison, WL pp. 15-47 (1984), which is hereby incorporated by reference in its entiret}'). Similar analysis of nearly 30 years (1959-1988) of maize hybrid yields in Ontario, Canada, also reached the same conclusion (ToUenaar, "Physiological-Basis of Genetic-Improvement of Maize Hybrids in Ontario from 1959 to 1988," Crop Sci. 31:119-124 (1991), which is hereby incorporated by reference in its entirety). Further physiological and genetic studies on the relationship between leaf senescence and maize yields have confirmed the correlation of delajdng leaf senescence and yield enhancement (Valentinuz and ToUenaar, "Vertical Profile of Leaf Senescence During the Grain-Filling Period in Older and Newer Maize Hybrids," Crop. Sci. 44:827-835 (2004); Ougham et al, "The Genetic Control of Senescence Revealed by Mapping Quantitative Trait Loci," in Gan, ed., Senescence Processes in Plants, Blackwell<br><br>
Publishing, pp. 171-201 (2007), which are hereby incorporated by reference in their entirety).<br>
Example 23 - Use of NAP Gene to Manipulate Leaf Senescence in Crops<br>
[0121]	Various molecular, genetic and genomic strategies have been used to<br>
isolate genes that are differentially expressed during senescence, and as a result, thousands ofSAGs have been identified. The structure and function of most SAGs have been predicted bioinformatically. There are only a few of those genes whose enzymatic activities have been shown biochemically, including several RNases (Lers et al., "Senescence-Induced RNases in Tomato," Plant Molecular Biology 36:439-449 (1998), which is hereby incorporated by reference in its entirety), a phospholipase D (Fan et al., "Antisense Suppression of Phospholipase D Alpha Retards Abscisic Acid-and Ethylene-Promoted Senescence of Postharvest Arabidopsis Leaves," Plant Cell 9:2183-2196 (1997), which is hereby incorporated by reference in its entirety), and an acyl hydrolase (He et al., "A Gene Encoding an Acyl Hydrolase is Involved in Leaf Senescence in Arabidopsis," Plant Cell 14:805-815 (2002), which is hereby incorporated by reference in its entirety). Similarly, there are only a few genes whose role in leaf senescence has been investigated genetically. For example, the ABA-promoted senescence in detached leaves of PLDa-antisense Arabidopsis plants was delayed (Fan et al., "Antisense Suppression of Phospholipase D Alpha Retards Abscisic Acid- snd Ethylene-Promoted Senescence of Postiarvest Arabidopsis Leaves." Plant Cell 9:2183-2196 (1997), which is hereby incorporated by reference in its entirety'). The Arabidopsis F-box gene 0RE9 has also been shown to play a role in leaf senescence because the ore9 mutant plants displayed increased leaf longevity (Woo et al., "0RE9, An F-Box Protein That Regulates Leaf Senescence in Arabidopsis," Plant Cell 13:1779-1790 (2001), which is hereby incorporated by reference in its entirety). It has previously shown that SAGlOl, a gene encoding an acyl hydrolase, plays a significant role in leaf senescence in Arabidopsis; leaf senescence is delayed for 4-5 days in the SAGJOI antisense plants (He et al, "A Gene Encoding an Acyl Hydrolase is Involved in Leaf Senescence in Arabidopsis," Plant Cell 14:805-815 (2002), which is hereby incorporated by reference in its entirety). In the present application, it was shown that leaf senescence in the atnap null mutant line<br><br>
was delayed for up to 10 days (Figures 5(a)-(d) and Figures 6(a)-(e)). The null phenotype was rescued by the wild type AtNAP, confirming that the lack ofAtNAP was responsible for the retardation in leaf senescence in the null mutant plants. The role of AtNAP in controlling leaf senescence was further confirmed by gain-of-function analysis. Young leaves began yellowing as early as 2 days after the initial induction of the AtNAP overexpression, and became completely senescent 4 days after the initial induction (Figures 8(a)-(d)), suggesting that AtNAP was sufficient to cause senescence.<br>
[0122]	The RNA gel blot analysis revealed that AtNAP was expressed in<br>
senescing leaf cells (Figures 2(a)-(c)). Microarray data currently available at<br>
Genevestigator (Zimmermann et al, "GENEVESTIGATOR. Arabidopsis Microarray<br>
Database and Analysis Toolbox," Plant Physiol 136:2621-2632 (2004), which is<br>
hereby incorporated by reference in its entirety) also showed that AtNAP is mainly<br>
expressed in senescent rosette leaves, cauline leaves, sepals and petals (Figure 13(a))<br>
although very low levels of the AtNAP expression in young seedlings and other parts<br>
of adult plants have been detected (Figure 13(b)). The AtNAP expression can be<br>
strongly induced by inducer of programmed cell death (PCD) in cell suspension, a<br>
process similar to leaf senescence. In contrast, other senescence-promoting factors<br>
such as ethylene and ABA, osmotic and salt stress only moderately induced the<br>
expression of AtNAP while darkness, drought, oxidative stress, jasmonic acid and<br>
sancyiic arid did not have significant effects on AtNAP expression levels<br>
(Pigxire ISfo)). In the absence of external stressors, initiation of leaf senescence is<br>
depsidsni on age and developmental stage (Hensel et al., "Developmental and Age-<br>
Related Processes That Influence the Longevity and Senescence of Photosynthetic<br>
Tissues in Arabidoposis," Plant Cell 5:553-564 (1993); Nooden et al, "Correlative<br>
Controls of Senescence and Plant Death in Arabidopsis Thaliana (Brassicaceae),"<br>
Journal of Experimental Botany 52:2151-2159 (2001), which are hereby incorporated<br>
by reference in their entirety). The data disclosed herein and the microarray data<br>
suggest that AtNAP may be primarily up-regulated by age.<br>
[0123]	Although AtNAP is not readily induced by darkness (Buchanan-<br>
WoUaston et al., "Comparative Transciiptome Analysis Reveals Significant Differences in Gene Expression and Signalling Pathways Between Developmental and Dark/Starvation-Induced Senescence in Arabidopsis," The Plant Journal 42:567-<br><br>
585 (2005); Lin et al., "Molecular Events in Senescing Arabidopsis Leaves," Plant<br>
Journal 39:612-628 (2004), which are hereby incorporated by reference in their<br>
entirety), dark-induced senescence of detached leaves was delayed in the atnap null<br>
mutant (Figure 7(b)-(c)), suggesting that AtNAP may function in dark-induced<br>
senescence downstream of the dark-responsive signaling pathway. During natural<br>
senescence of leaves on intact plants, AtNAP was only expressed in leaf tissues that<br>
are already senescent (Figures 2(a)-(c)). These data suggested that AtNAP is likely to<br>
play a key role in regulating the common execution process of leaf senescence<br>
downstream of various senescence-inducing pathways. Even though expression of<br>
AtNAP may not necessarily be responsive to a particular senescence-inducing factor,<br>
some of these factors may need AtNAP to trigger the senescence syndrome. As a<br>
transcription factor, AtNAP might control the leaf senescence process by<br>
transcriptionally activating/repressing genes involved in the execution of senescence.<br>
[0124]	Plant transcription factors of the same family often have similar<br>
functions. At some developmental stages or cellular processes, certain families of transcription factors may play predominant roles (Liu et al., "Transcription Factors and Their Genes in Higher Plants Functional Domains, Evolution and Regulation," EurJBiochem 262:247-257 (1999); Riechmann et al, "A Genomic Perspective on Plant Transcription Factors," Curr Opin Plant Biol 3:423-434 (2000), which are hereby incorporated by reference in their entirety), such as the MADS box genes in<br><br>
(Ulker et al., "WRKY Transcription Factors: From DNA Binding Towards Biological Function," Current Opinion in Plant Biology 7:491-498 (2004), which is hereby incorporated by reference in its entirety). The senescence-associated expression pattern of more than 20 other NAC family members (Buchanan-WoUaston et al, "Comparative Transcriptome Analysis Reveals Significant Differences in Gene Expression and Signalling Pathways Between Developmental and Dark/Starvation-Induced Senescence in Arabidopsis," The Plant Journal 42:567-585 (2005); Guo et al., "Transcriptome of Arabidopsis Leaf Senescence," Plant Cell and Environment 27:521-549 (2004), which are hereby incorporated by reference in their entirety) suggests a general role of the NAC family genes in leaf senescence. The evidence of<br><br>
transcriptional self-regulation (Xie et al., "Arabidopsis NACl Transduces Auxin<br>
Signal Downstream of TIRl to Promote Lateral Root Development," Genes Dev<br>
14:3024-3036 (2000), which is hereby incorporated by reference in its entirety) and<br>
inter-regulation (Vroemen et al, "The CUP-SHAPED COTYLEDONS Gene is<br>
Required for Boundary and Shoot Meri^stem Formation in Arabidopsis," Plant Cell<br>
15:1563-1577 (2003), which is hereby incorporated by reference in its entirety)<br>
between NAC members as well as homodimerization (Ernst et al., "Structure of the<br>
Conserved Domain of ANAC, a Member of the NAC Family of Transcription<br>
Factors," EMBO Rep 5:297-303 (2004); Xie et al, "Arabidopsis NACl Transduces<br>
Auxin Signal Downstream of TIRl to Promote Lateral Root Development," Genes<br>
Dev 14:3024-3036 (2000), which are hereby incorporated by reference in their<br>
entirety) and heterodimerization (Hegedus et al., "Molecular Characterization of<br>
Brassica Napus NAC Domain Transcriptional Activators Induced in Response to<br>
Biotic and Abiotic Stress," Plant Mol Biol 53:383-397 (2003), which is hereby<br>
incorporated by reference in its entiretj^ among NAC proteins suggest possible<br>
regulatory networks of leaf senescence involving many NACs.<br>
[0125]	Sequence homologs of AtNAP in kidney bean (a dicot) and rice (a<br>
monocot) also displayed a leaf senescence-specific expression pattern (Figure 9(c)-(d)). Pv NAP and Os NAP were able to restore the Arabidopsis atnap null mutant to "wild type (Figures 7(a)-(d)). In addition to rice and kidney bean, sequence homologs<br><br>
(Figure 9a). These suggest that NAP may be a universal regulator in plant leaf senescence. It is likely that knocking NAP out in other plant species will cause a significant delay of leaf senescence, which may be a new strategy for manipulating leaf senescence in agriculturally important crops. [0126] <br>
homeotic genes APETALA3/PISTILLATAL that are essential for petal and stamen formation (Sablowski et al, "A Homolog of NO APICAL MERISTEM is an Immediate Target of the Floral Homeotic Genes APETALA3/PISTILLATA," Cell 92:93-103 (1998), which is hereby incorporated by reference in its entirety). In<br><br>
AtNAP antisense lines, the first 2-4 flowers of the main and lateral inflorescences had short stamens and their anthers often did not dehisce. No leaf senescence phenotype was described (Sablowski et al, "A Homolog of NO APICAL MERISTEM is an Immediate Target of the Floral Homeotic Genes APETALA3/PISTILLATA," Cell 92:93-103 (1998), which is hereby incorporated by reference in its entirety). In the experiments disclosed in the present application, no developmental abnormities other than the delayed leaf senescence in the two T-DNA mutant lines were observed. This discrepancy may be due to different .Arabidopsis ecotypes that were used: Landsberg erecta (Sablowski et al., "A Homolog of NO APICAL MERISTEM is an Immediate Target of the Floral Homeotic Genes APETALA3/PISTILLATA," Cell 92:93-103 (1998), which is hereby incorporated by reference in its entirety) vs. Columbia in the present application. Leaf senescence is a trait with great variations within Arabidopsis ecotypes and the molecular regulation of leaf senescence may differ in different genetic backgrounds (Levey et al., "Natural Variation in the Regulation of Leaf Senescence and Relation to Other Traits in Arabidopsis," Plant Cell and Environment 28:223-231 (2005), which is hereby incorporated by reference in its entirety). This discrepancy may also result from different research approaches employed; T-DNA insertion mutation and antisense approach have shown completely different roles of the phytochrome interacting factor 3 (Kim et al., "Fimctional<br><br>
which are hereby incorporated by reference in their entirety).<br>
[0127]	Although the invention has been described in detail, for the purpose of<br>
illustration, it is understood that such detail is for that purpose and variations can be made therein by those skilled in the art without departing from the spirit and scope of the invention which is defined by the following claims.<br><br><br>
WHAT IS CLAIMED:<br>
1.	A transgenic plant having an altered level of NAP protein<br>
capable of causing leaf senescence in a plant, compared to that of a non-transgenic<br>
plant, wherein the transgenic plant displays an altered leaf senescence phenotype,<br>
relative to a non-transgenic plant.<br>
2.	The transgenic plant according to claim 1, wherein said transgenic plant has a reduced level of NAP protein and displays a delayed leaf senescence phenotype.<br>
3.	The transgenic plant according to claim 2, wherein the plant is transformed with a nucleic acid construct comprising a nucleic acid molecule configured to silence NAP protein expression.<br>
4.	The transgenic plant according to claim 3, wherein the nucleic<br>
acid molecule comprises a dominant negative mutation and encodes a non-functional<br>
NAP protein, resulting in suppression or interference of endogenous mRNA<br>
encoding the NAP protein.<br>
5.	The transgenic plant according to claim 3, w^herein the nucleic<br>
  positioned ia the nucleic acid construct to result in suppression or<br>
interference of endogenous mRNA encoding the NAP protein.<br>
6.	The transgenic plant according to claim 5, wherein the nucleic<br>
acid molecule encodes the NAP protein and is in sense orientation.<br>
7.	The transgenic plant according to claim 5, wherein the nucleic<br>
acid molecule is an antisense form of a NAP protein encoding nucleic acid molecule.<br>
8.	The transgenic plant according to claim 5, wherein the plant is<br>
transformed with first and second of the nucleic acid constructs with the first nucleic<br>
R912390,l<br><br>
acid construct encoding the NAP protein in sense orientation and the second nucleic acid construct encoding the NAP protein in antisense form.<br>
9.	The transgenic plant according to claim 5, wherein the nucleic acid molecule comprises a first segment encoding the NAP protein, a second segment in an antisense form of a NAP protein encoding nucleic acid molecule, and a third segment linking the first and second segments.<br>
10.	The transgenic plant according to claim 1, wherein said transgenic plant has an increased level of NAP protein and displays a premature or precocious leaf senescence phenotype.<br>
11.	The transgenic plant according to claim 10, wherein the plant is transformed with a nucleic acid construct configured to overexpress NAP protein.<br>
12.	The transgenic plant according to claim M, wherein the nucleic acid construct comprises a plant specific promoter.<br>
13.	The transgenic plant according to claim 12, wherein the promoter is an inducible plant promoter.<br>
14.	The transgenic plant according to claim 1, wherein the plant is<br>
a crop pisni.<br>
15.	The transgenic plant according to claim 14, wherein the crop<br>
plant is selected from the group consisting of alfalfa, rice, wheat, barley, rye, cotton,<br>
sunflower, peanut, com, potato, sweet potato, kidney bean, pea, chicory, lettuce,<br>
endive, cabbage, bok choy, brussel sprout, beet, parsnip, turnip, cauliflower,<br>
broccoli, radish, spinach, onion, garhc, eggplant, pepper, celery, carrot, squash,<br>
pumpkin, zucchini, cucumber, apple, pear, melon, citrus, peach, strawberry, grape,<br>
raspberry, pineapple, soybean, Medicago, tobacco, tomato, sorghum, and sugarcane.<br><br>
16.	The transgenic plant according to claim 1, wherein the plant is<br>
an ornamental plant.<br>
17.	The transgenic plant according to claim 16, wherein the<br>
ornamental plant is selected from the group consisting oiArabidopsis thaliana,<br>
Saintpaulia, Populus, petunia, pelargonium, poinsettia, chrysanthemum, carnation,<br>
zinnia, turfgrass, lily, and nightshade.<br>
18.	Seed produced from the plant according to claim 1.<br>
19.	A mutant plant comprising an inactivated NAP gene, wherein<br>
the mutant plant displays a delayed leaf senescence phenotype, relative to a non-<br>
mutant plant.<br>
20.	The mutant plant according to claim 19, wherein the plant is a<br>
crop plant.<br>
21.	The mutant plant according to claim 20, wherein the crop plant<br>
is selected from the group consisting of alfalfa, rice, wheat, barley, rye, cotton,<br>
sunflower, peanut, com, potato, sweet potato, Mdney bean, pea, chicory, lettuce,<br><br>
raspberry, pineapple, soybean, Medicago, tobacco, tomato, sorghum, and sugarcane.<br>
22.	The mutant plant according to claim 19, wherein the plant is an<br>
ornamental plant.<br>
23.	The mutant plant according to claim 22, wherein the<br>
ornamental plant is selected from the group consisting of Arabidopsis thaliana,<br>
Saintpaulia, Populus, petunia, pelargonium, poinsettia, chrysanthemum, carnation,<br>
ziimia, turfgrass, lily, and nightshade.<br><br>
24.	A mutant plant seed produced by growing the mutant plant according to claim 19 under conditions effective to cause the mutant plant to produce seed.<br>
25.	A method for delaying leaf senescence in a plant, said method comprising:<br>
providing a transgenic plant or plant seed transformed with a nucleic acid construct effective in silencing expression of a NAP protein capable of causing leaf senescence in a plant; and<br>
growing the transgenic plant or the plant grown from the transgenic plant seed under conditions effective to delay leaf senescence in the transgenic plant or the plant grown from the transgenic plant seed.<br>
26.	The method according to claim 25, wherein a transgenic plant is provided.<br>
27.	The method according to claim 25, wherein a transgenic plant seed is provided.<br>
28.	The method according to claim 25, wherein said providing<br>
providing a nucleic acid construct comprising:<br>
a nucleic acid molecule configured to silence NAP protein expression:<br>
a 5' DNA promoter sequence; and<br>
a 3' terminator sequence, wherein the nucleic acid molecule, the promoter, and the terminator are operatively coupled to permit expression of the nucleic acid molecule; and<br>
transforming a plant cell with the nucleic acid construct.<br>
29.	The method according to claim 25, wherein the nucleic acid<br>
molecule is effective in silencing expression of a NAP protein having an amino acid<br>
sequence selected from the group consisting of SEQ ID NO: 1, SEQ ID NO: 2, SEQ<br>
ID NO: 3, SEQ ID NO: 4, SEQ ID NO: 5, SEQ ID NO: 6, SEQ ID NO: 7, SEQ ID<br><br>
NO: 8, SEQ ID NO: 9, SEQ ID NO: 10, SEQ ID NO: 11, SEQ ID NO: 12, and SEQ ID NO: 13.<br>
30.	The method according to claim 28, wherein the nucleic acid molecule comprises a dominant negative mutation and encodes a non-functional NAP protein, resulting in suppression or interference of endogenous mRNA encoding the NAP protein.<br>
31.	The method according to claim 28, wherein the nucleic acid molecule is positioned in the nucleic acid construct to result in suppression or interference of endogenous mRNA encoding the NAP protein.<br>
32.	The method according to claim 28, wherein the nucleic acid molecule encodes the NAP protein and is in sense orientation.<br>
33.	The method according to claim 28, wherein the nucleic acid molecule is an antisense form of a NAP protein encoding nucleic acid molecule.<br>
34.	The method according to claim 28, wherein the plant is transformed ■with first and second of the nucleic acid constructs with the first nucleic acid constrac: sicodng the NAP protein in sense orientation and the second nucleic acid construct encoding the NAP protein in antisense form.<br>
35.	The method according to claim 28, wherein the nucleic acid molecule comprises a first segment encoding the NAP protein, a second segment in an antisense form of a NAP protein encoding nucleic acid molecule, and a third segment linking the first and second segments.<br>
36.	The method according to claim 28, wherein the plant cell is from a crop plant.<br>
37.	The method according to claim 36, wherein the crop plant is selected fi-om the group consisting of alfalfa, rice, wheat, barley, rye, cotton.<br><br>
sunflower, peanut, com, potato, sweet potato, kidney bean, pea, chicory, lettuce, endive, cabbage, bok choy, brussel sprout, beet, parsnip, turnip, cauliflower, broccoli, radish, spinach, onion, garlic, eggplant, pepper, celery, carrot, squash, pumpkin, zucchini, cucumber, apple, pear, melon, citrus, peach, strawberry, grape, raspberry, pineapple, soybean, Medicago, tobacco, tomato, sorghum, and sugarcane.<br>
38.	The method according to claim 28, wherein the plant cell is from an ornamental plant.<br>
39.	The method according to claim 38, wherein the ornamental plant is selected from the group consisting of Arabidopsis thaliana, Saintpaulia, Populus, petunia, pelargonium, poinsettia, chrysanthemum, carnation, zinnia, turfgrass, lily, and nightshade.<br>
40.	The method according to claim 28 further comprising:<br>
propagating plants from the transformed plant cell.<br>
41.	The method according to claim 28, wherein said transforming<br>
is carried out by a method selected from the group consisting of Agwbacterium<br>
mediated transformation, vacuum infiltration, biolistic gene transformation,<br>
elsciroiiorHtioiL nricioinjection, chemical-mediated transformation, and laser-beam<br>
transibrmanoxL<br>
42.	The method according to claim 25, wherein leaf senescence is delayed in a plant before harvest.<br>
43.	The method according to claim 25, wherein leaf senescence is delayed in a plant after harvest.<br>
44.	A plant produced by the method of claim 25.<br>
45.	A method for delaying leaf senescence in a plant, said method comprising:<br><br>
transforming a plant cell with a nucleic acid molecule encoding a NAP protein capable of causing leaf senescence in a plant operably associated with a promoter to obtain a transformed plant cell, wherein expression of the nucleic acid molecule in the plant cell causes delayed leaf senescence by a form of post-transcriptional gene silencing; and<br>
regenerating a plant from the transformed plant cell under conditions effective to delay leaf senescence in the plant.<br>
46.	The method according to claim 45, wherein the form of post-<br>
transcriptional gene silencing is RNA interference.<br>
47.	A method of making a mutant plant having a decreased level of<br>
NAP protein compared to that of a non-mutant plant, wherein the mutant plant<br>
displays a delayed leaf senescence phenotype relative to a non-mutant plant, said<br>
method comprising:<br>
providing at least one cell of a non-mutant plant containing a gene encoding a functional NAP protein;<br>
treating said at least one cell of a non-mutant plant imder conditions effective to inactivate said gene, thereby yielding at least one mutant plant cell containing an inactivated NAP gene; and<br>
propagating said at least one mutant plant ceil into a mutant plant, wherein said muiaiit plant has a decreased level of NAP protein compared to that of ih.e aon-nruiHnr plani and displays a delayed leaf senescence phenot\'pe relative to a non-mutant plant.<br>
48.	The method according to claim 47, wherein the functional NAP protein has an amino acid sequence selected from the group consisting of SEQ ID NO: 1, SEQ ID NO: 2, SEQ ID NO: 3, SEQ ID NO: 4, SEQ ID NO: 5, SEQ ID NO: 6, SEQ ID NO: 7, SEQ ID NO: 8, SEQ ID NO: 9, SEQ ID NO: 10, SEQ ID NO: 11, SEQ ID NO: 12, and SEQ ID NO: 13.<br>
49.	The method according to claim 47, wherein said treating comprises subjecting the at least one cell of the non-mutant plant to a chemical<br><br>
mutagenizing agent under conditions effective to yield at least one mutant plant cell containing an inactive NAP gene.<br>
50.	The method according to claim 49, wherein the chemical mutagenizing agent is ethylmethanesulfonate.<br>
51.	The method according to claim 47, wherein said treating comprises subjecting the at least one cell of the non-mutant plant to a radiation source under conditions effective to yield at least one mutant plant cell containing an inactive<br>
'NAP gene.<br>
52.	The method according to claim 51, wherein the radiation source is effective in producing ultraviolet rays, gamma rays, or fast neutrons.<br>
53.	The method according to claim 47, wherein said treating comprises inserting an inactivating nucleic acid molecule into the gene encoding the functional NAP protein or its promoter under conditions effective to inactivate the gene.<br>
54.	The method according to claim 53, wherein said inactivating<br>
niideic add rDolecule is a transposable element<br>
55.	The method according to claim 54, wherein the transposable element is selected from the group consisting of an Activator (Ac) transposon, a Dissociator (Ds) transposon, and a Mutator (Mu) transposon.<br>
56.	The method according to claim 47, wherein said treating comprises subjecting the at least one cell of the non-mutant plant to Agrobacterium transformation under conditions effective to insert an Agrobacterium T-DNA sequence into the gene, thereby inactivating the gene.<br>
57.	The method according to claim 56, wherein said treating comprises subjecting the at least one cell of the non-mutant plant to site-directed<br><br>
mutagenesis of the NAP gene or its promoter under conditions effective to yield at least one mutant plant cell containing an inactive NAP gene.<br>
58.	The method according to claim 56, wherein said treating comprises mutagenesis by homologous recombination of the NAP gene or its promoter.<br>
59.	The method according to claim 56, wherein said treating comprises targeted deletion of a portion of the NAP gene sequence or its promoter.<br>
60.	The method according to claim 56, wherein said treating comprises targeted insertion of a nucleic acid sequence into the NAP gene or its promoter.<br><br>
61.	The method according to claim 47, wherein the non-mutant plant is a crop plant.<br>
62.	The method according to claim 61, wherein the crop plant is selected from the group consisting of alfalfa, rice, wheat, barley, rye, cotton, sunflower, peanut com, potato, sweet potato, kidney bean, pea, chicory, lettuce, en.'5n-e- cabbage, bok choy, brussel sprout, beet parsnip, turnip, cauliflower, nroccoii radish, ^inach, onion, garhc, eggplant, pepper, celery, carrot squash, pimipkiii. zuccnini, cucumber, apple, pear, melon, citrus, peach, strawberry, grape, raspberry, pineapple, soybean, Medicago, tobacco, tomato, sorghum, and sugarcane.<br>
63.	The method according to claun 47, wherein the non-mutant plant is an ornamental plant.<br>
64.	The method according to claim 63, wherein the ornamental plant is selected from the group consisting of Arabidopsis thaliana, Saintpaulia, Populus, petunia, pelargonium, poinsettia, chrysanthemum, carnation, zinnia, turf grass, lily, and nightshade.<br><br>
65.	A mutant plant produced according to the method of claim 47.<br>
66.	A mutant plant seed produced by growing the mutant plant according to claim 65 under conditions effective to cause the mutant plant to produce seed.<br>
67.	A method for causing precocious leaf senescence or promoting<br>
leaf senescence in a plant, said method comprising:<br>
transforming a plant cell with a nucleic acid molecule encoding a NAP protein capable of causing leaf senescence in a plant operably associated with a promoter to obtain a transformed plant cell;<br>
regenerating a plant from the transformed plant cell; and inducing the promoter under conditions effective to cause premature or precocious leaf senescence in the plant.<br>
68.	The method according to claim 67, wherein the nucleic acid molecule encodes a NAP protein having an amino acid sequence selected from the group consisting of SEQ ID NO: 1, SEQ ID NO: 2, SEQ ID NO: 3, SEQ ID NO: 4, SEQ ID NO: 5, SEQ ID NO: 6, SEQ ID NO: 7, SEQ ID NO: 8, SEQ ID NO: 9, SEQ ID NO: 10, SEQ ID NO: 11, SEQ ID NO: 12, and SEQ ID NO: 13.<br>
69.	The method according to claim 67, wherein the plant cell is<br>
70.	The method according to claim 69, wherein the crop plant is selected from the group consisting of alfalfa, rice, wheat, barley, rye, cotton, sunflower, peanut, com, potato, sweet potato, kidney bean, pea, chicory, lettuce, endive, cabbage, bok choy, bmssel sprout, beet, parsnip, turnip, cauliflower, broccoli, radish, spinach, onion, garlic, eggplant, pepper, celery, carrot, squash, pumpkin, zucchini, cucumber, apple, pear, melon, citrus, peach, strawberry, grape, raspberry, pineapple, soybean, Medicago, tobacco, tomato, sorghum, and sugarcane.<br><br>
71.	The method according to claim 70, wherein the crop plant is cotton.<br>
72.	The method according to claim 67, wherein the plant cell is from an ornamental plant.<br>
73.	The method according to claim 72, wherein the ornamental plant is selected from the group consisting of Arabidopsis thaliana, Saintpaulia, Populus. petunia, pelargonium, poinsettia, chr^'santhemum, carnation, zinnia, turfgrass, lily, and nightshade.<br>
74.	The method according to claim 67, wherein said transforming is carried out by a method selected from the group consisting of Agrobacterium mediated transformation, vacuum infiltration, biohstic gene transformation, electroporation, microinjection, chemical-mediated transformation, and laser-beam transformation.<br>
75.	A plant produced by the method of claim 67.<br>
76.	A method of identifying a candidate plant suitable for breeding ±21 displays a delayed leaf senescence and'or enhanced yield phenotype, said siemod comprising:<br>
analyzing the candidate plant for the presence, in its genome, of an inactivated NAP gene.<br>
77.	The method according to claim 76, wherein the method identifies a candidate plant suitable for breeding that displays a delayed leaf senescence phenotype.<br>
78.	The method according to claim 76, wherein the method identifies a candidate plant suitable for breeding that displays an enhanced yield phenotype.<br><br>
79.	The method according to claim 76, wherein the method<br>
identifies a candidate plant suitable for breeding that displays a delayed leaf<br>
senescence and enhanced yield phenotype.<br>
80.	The method according to claim 76, wherein the plant is a crop<br>
plant.<br>
81.	The method according to claim 80, wherein the crop plant is<br>
selected from the group consisting of alfalfa, rice, wheat, barley, rye, cotton,<br>
sunflower, peanut, com, potato, sweet potato, kidney bean, pea, chicory, lettuce,<br>
endive, cabbage, bok choy, brussel sprout, beet, parsnip, turnip, cauliflower,<br>
broccoli, radish, spinach, onion, garlic, eggplant, pepper, celery, carrot, squash,<br>
pumpkin, zucchini, cucumber, apple, pear, melon, citrus, peach, strawberry, grape,<br>
raspberry, pineapple, soybean, Medicago, tobacco, tomato, sorghum, and sugarcane.<br>
82.	The method according to claim 76, wherein the plant is an<br>
ornamental plant.<br>
83.	The method according to claim 82, wherein the ornamental<br>
plant is selected from liie group consisting of Arabidopsis tkaliana, Saintpaulia,<br><br><br></td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTIwOC0yMDA4LUZPUk0gMy5wZGY=" target="_blank" style="word-wrap:break-word;">5208-2008-FORM 3.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTIwOC0yMDA4LVBldGl0aW9uIDEzNyAtIEZvcm0gMy5wZGY=" target="_blank" style="word-wrap:break-word;">5208-2008-Petition 137 - Form 3.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTIxMy1DSEVOUC0yMDA4ICAgIEFGRklGREFWSVQgICAgIDIyLTAxLTIwMTUucGRm" target="_blank" style="word-wrap:break-word;">5213-CHENP-2008    AFFIFDAVIT     22-01-2015.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTIxMy1DSEVOUC0yMDA4ICAgIE9USEVSUyAwNS0wOC0yMDE0LnBkZg==" target="_blank" style="word-wrap:break-word;">5213-CHENP-2008    OTHERS 05-08-2014.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTIxMy1DSEVOUC0yMDA4ICAgQU1FTkRFRCBDTEFJTVMgIDA2LTAxLTIwMTUucGRm" target="_blank" style="word-wrap:break-word;">5213-CHENP-2008   AMENDED CLAIMS  06-01-2015.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTIxMy1DSEVOUC0yMDA4ICAgQ09SUkVTUE9OREVOQ0UgT1RIRVJTICAgICAyMi0wMS0yMDE1LnBkZg==" target="_blank" style="word-wrap:break-word;">5213-CHENP-2008   CORRESPONDENCE OTHERS     22-01-2015.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTIxMy1DSEVOUC0yMDA4ICAgQ09SUkVTUE9OREVOQ0UgT1RIRVJTICAwOS0xMi0yMDE0LnBkZg==" target="_blank" style="word-wrap:break-word;">5213-CHENP-2008   CORRESPONDENCE OTHERS  09-12-2014.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTIxMy1DSEVOUC0yMDA4ICAgQ09SUkVTUE9OREVOQ0UgT1RIRVJTICAxNS0wOS0yMDE0LnBkZg==" target="_blank" style="word-wrap:break-word;">5213-CHENP-2008   CORRESPONDENCE OTHERS  15-09-2014.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTIxMy1DSEVOUC0yMDA4ICAgQ09SUkVTUE9OREVOQ0UgT1RIRVJTICAxNy0xMi0yMDE0LnBkZg==" target="_blank" style="word-wrap:break-word;">5213-CHENP-2008   CORRESPONDENCE OTHERS  17-12-2014.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTIxMy1DSEVOUC0yMDA4ICAgQ09SUkVTUE9OREVOQ0UgT1RIRVJTIDMwLTEyLTIwMTMucGRm" target="_blank" style="word-wrap:break-word;">5213-CHENP-2008   CORRESPONDENCE OTHERS 30-12-2013.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTIxMy1DSEVOUC0yMDA4ICAgQ09SUkVTUE9OREVOQ0UgT1RIRVJTLiAwNS0wOC0yMDE0LnBkZg==" target="_blank" style="word-wrap:break-word;">5213-CHENP-2008   CORRESPONDENCE OTHERS. 05-08-2014.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTIxMy1DSEVOUC0yMDA4ICBBTUVOREVEIENMQUlNUyAgMDQtMDYtMjAxNC5wZGY=" target="_blank" style="word-wrap:break-word;">5213-CHENP-2008  AMENDED CLAIMS  04-06-2014.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTIxMy1DSEVOUC0yMDA4ICBBTUVOREVEIFBBR0VTIE9GIFNQRUNJRklDQVRJT04gIDA0LTA2LTIwMTQucGRm" target="_blank" style="word-wrap:break-word;">5213-CHENP-2008  AMENDED PAGES OF SPECIFICATION  04-06-2014.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTIxMy1DSEVOUC0yMDA4ICBBTUVOREVEIFBBR0VTIE9GIFNQRUNJRklDQVRJT04gIDA1LTA4LTIwMTQucGRm" target="_blank" style="word-wrap:break-word;">5213-CHENP-2008  AMENDED PAGES OF SPECIFICATION  05-08-2014.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTIxMy1DSEVOUC0yMDA4ICBDT1JSRVNQT05ERU5DRSBPVEhFUlMgICAyOS0wOS0yMDE0LnBkZg==" target="_blank" style="word-wrap:break-word;">5213-CHENP-2008  CORRESPONDENCE OTHERS   29-09-2014.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTIxMy1DSEVOUC0yMDA4ICBDT1JSRVNQT05ERU5DRSBPVEhFUlMgIDE1LTEwLTIwMTQucGRm" target="_blank" style="word-wrap:break-word;">5213-CHENP-2008  CORRESPONDENCE OTHERS  15-10-2014.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTIxMy1DSEVOUC0yMDA4ICBFTkdMSVNIIFRSQU5TTEFUSU9OICAwNS0wMS0yMDE1LnBkZg==" target="_blank" style="word-wrap:break-word;">5213-CHENP-2008  ENGLISH TRANSLATION  05-01-2015.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTIxMy1DSEVOUC0yMDA4ICBFWEFNSU5BVElPTiBSRVBPUlQgUkVQTFkgUkVDRUlWRUQgIDA1LTAxLTIwMTUucGRm" target="_blank" style="word-wrap:break-word;">5213-CHENP-2008  EXAMINATION REPORT REPLY RECEIVED  05-01-2015.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTIxMy1DSEVOUC0yMDA4ICBFWEFNSU5BVElPTiBSRVBPUlQgUkVQTFkgUkVDRUlWRUQgIDA2LTAxLTIwMTUucGRm" target="_blank" style="word-wrap:break-word;">5213-CHENP-2008  EXAMINATION REPORT REPLY RECEIVED  06-01-2015.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTIxMy1DSEVOUC0yMDA4ICBFWEFNSU5BVElPTiBSRVBPUlQgUkVQTFkgUkVDSUVWRUQgIDA0LTA2LTIwMTQucGRm" target="_blank" style="word-wrap:break-word;">5213-CHENP-2008  EXAMINATION REPORT REPLY RECIEVED  04-06-2014.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTIxMy1DSEVOUC0yMDA4ICBGT1JNLTEgIDA0LTA2LTIwMTQucGRm" target="_blank" style="word-wrap:break-word;">5213-CHENP-2008  FORM-1  04-06-2014.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTIxMy1DSEVOUC0yMDA4ICBGT1JNLTEgIDA1LTAxLTIwMTUucGRm" target="_blank" style="word-wrap:break-word;">5213-CHENP-2008  FORM-1  05-01-2015.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTIxMy1DSEVOUC0yMDA4ICBGT1JNLTE4ICAxOC0wMi0yMDEwLnBkZg==" target="_blank" style="word-wrap:break-word;">5213-CHENP-2008  FORM-18  18-02-2010.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTIxMy1DSEVOUC0yMDA4ICBGT1JNLTMgIDA1LTAxLTIwMTUucGRm" target="_blank" style="word-wrap:break-word;">5213-CHENP-2008  FORM-3  05-01-2015.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTIxMy1DSEVOUC0yMDA4ICBPVEhFUiBQQVRFTlQgRE9DVU1FTlQgICAgMDQtMDktMjAxNC5wZGY=" target="_blank" style="word-wrap:break-word;">5213-CHENP-2008  OTHER PATENT DOCUMENT    04-09-2014.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTIxMy1DSEVOUC0yMDA4ICBPVEhFUiBQQVRFTlQgRE9DVU1FTlQgICAgMjktMDktMjAxNC5wZGY=" target="_blank" style="word-wrap:break-word;">5213-CHENP-2008  OTHER PATENT DOCUMENT    29-09-2014.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTIxMy1DSEVOUC0yMDA4ICBPVEhFUiBQQVRFTlQgRE9DVU1FTlQgIDA0LTA2LTIwMTQucGRm" target="_blank" style="word-wrap:break-word;">5213-CHENP-2008  OTHER PATENT DOCUMENT  04-06-2014.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTIxMy1DSEVOUC0yMDA4ICBPVEhFUlMgIDA0LTA2LTIwMTQucGRm" target="_blank" style="word-wrap:break-word;">5213-CHENP-2008  OTHERS  04-06-2014.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTIxMy1DSEVOUC0yMDA4ICBQT1dFUiBPRiBBVFRPUk5FWSAgMDUtMDEtMjAxNS5wZGY=" target="_blank" style="word-wrap:break-word;">5213-CHENP-2008  POWER OF ATTORNEY  05-01-2015.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTIxMy1jaGVucC0yMDA4IGFic3RyYWN0LnBkZg==" target="_blank" style="word-wrap:break-word;">5213-chenp-2008 abstract.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTIxMy1DSEVOUC0yMDA4IEFNRU5ERUQgIFBBR0VTIE9GIFNQRUNJRklDQVRJT04gIDA1LTAxLTIwMTUucGRm" target="_blank" style="word-wrap:break-word;">5213-CHENP-2008 AMENDED  PAGES OF SPECIFICATION  05-01-2015.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTIxMy1DSEVOUC0yMDA4IEFNRU5ERUQgQ0xBSU1TICAwNS0wMS0yMDE1LnBkZg==" target="_blank" style="word-wrap:break-word;">5213-CHENP-2008 AMENDED CLAIMS  05-01-2015.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTIxMy1jaGVucC0yMDA4IGNsYWltcy5wZGY=" target="_blank" style="word-wrap:break-word;">5213-chenp-2008 claims.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTIxMy1jaGVucC0yMDA4IGNvcnJlc3BvbmRlbmNlIG90aGVycy0wMy0wNy0yMDA5LnBkZg==" target="_blank" style="word-wrap:break-word;">5213-chenp-2008 correspondence others-03-07-2009.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTIxMy1jaGVucC0yMDA4IGNvcnJlc3BvbmRlbmNlLW90aGVycy5wZGY=" target="_blank" style="word-wrap:break-word;">5213-chenp-2008 correspondence-others.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTIxMy1jaGVucC0yMDA4IGRlc2NyaXB0aW9uKGNvbXBsZXRlKS5wZGY=" target="_blank" style="word-wrap:break-word;">5213-chenp-2008 description(complete).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTIxMy1jaGVucC0yMDA4IGRyYXdpbmdzLnBkZg==" target="_blank" style="word-wrap:break-word;">5213-chenp-2008 drawings.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTIxMy1jaGVucC0yMDA4IGZvcm0tMS5wZGY=" target="_blank" style="word-wrap:break-word;">5213-chenp-2008 form-1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTIxMy1jaGVucC0yMDA4IGZvcm0tMjYtMDMtMDctMjAwOS5wZGY=" target="_blank" style="word-wrap:break-word;">5213-chenp-2008 form-26-03-07-2009.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTIxMy1jaGVucC0yMDA4IGZvcm0tMy5wZGY=" target="_blank" style="word-wrap:break-word;">5213-chenp-2008 form-3.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTIxMy1jaGVucC0yMDA4IGZvcm0tNS5wZGY=" target="_blank" style="word-wrap:break-word;">5213-chenp-2008 form-5.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTIxMy1jaGVucC0yMDA4IHBjdC5wZGY=" target="_blank" style="word-wrap:break-word;">5213-chenp-2008 pct.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=UGV0aXRpb24gZm9yIHNlcXVlbmNlIGxpc3RpbmcucGRm" target="_blank" style="word-wrap:break-word;">Petition for sequence listing.pdf</a></p>
		<br>
		<div class="pull-left">
			<a href="266166-method-and-apparatus-for-configuring-software-resources-for-playing-network-programs.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="266168-a-process-for-solvent-less-synthesis-of-benzoxazine-bz.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>266167</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>5213/CHENP/2008</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>15/2015</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>10-Apr-2015</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>09-Apr-2015</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>29-Sep-2008</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>CORNELL RESEARCH FOUNDATION, INC.</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td>20 THORNWOOD DRIVE, SUITE 105, ITHACA, NEW YORK 14850</td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>GUO, YONGFENG</td>
											<td>1929 PLYMOUTH RD. #3033, ANN ARBOR, MI 48105</td>
										</tr>
										<tr>
											<td>2</td>
											<td>GAN, SUSHENG</td>
											<td>202 WINTHROP DRIVE, ITHACA, NY 14850</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>A01H5/00</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>PCT/US07/65321</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td>2007-03-28</td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td>60/786,602</td>
									<td>2006-03-28</td>
								    <td>U.S.A.</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/266167-a-method-for-leaf-senescence-in-a-transgenic-plant by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 06:44:00 GMT -->
</html>
