<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/268229-a-method-and-system-for-identifying-proteins-in-mixtures by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 06:46:39 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 268229:&quot;A METHOD AND SYSTEM FOR IDENTIFYING PROTEINS IN MIXTURES&quot;</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">&quot;A METHOD AND SYSTEM FOR IDENTIFYING PROTEINS IN MIXTURES&quot;</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>Protein identification in a complex sample begins by selecting a database having proteins likely to be in the sample. In-silico digestion is performed on the proteins in the database to produce peptides. A target peptide is selected from the produced peptides. The masses of the Y- and B-ion fragments of the target peptide are determined. Thus, a list of masses is assembled. This list contains the Y-ions, B-ions as well as the unique mass associated with the unfragmented precursor itself. These masses are used to search previously obtained low- and high-energy AMRTs obtained from LC/MS analysis of the complex sample for any and all of the masses on the list.  Any mass observed in the data within a detection threshold are considered a hit.  If enough hits accumulate in a given retention time, the target peptide is identified as being in the sample.  Peptides in the data base can detect peptides in the sample provided there is sufficient sequence overlap.  Because sequence identity is not required, modified peptides can be found in the sample. The list of peptides identified in the complex sample can be used to identify the proteins present in the sample, track the chromatographic retention times of peptides between samples, and quantitate the peptides and proteins present in complex samples.</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td>METHOD AND APPARATUS FOR IDENTIFYING PROTEINS IN MIXTURES<br>
[0001]	This application claims the benefit of U.S. Provisional Application No.<br>
60/572,532, filed May 20, 2004, which is herein incorporated by reference in its<br>
entirety.<br>
RELATED APPLICATIONS<br>
f0002]	The present application is related to co-filed and co-pending PCT<br>
application no. PCT/US05/	having attorney docket number, WAA-<br>
393, entitled "System and Method for Grouping Precursor and Fragment Ions<br>
Using Selected Ion Chromatograms."<br>
BACKGROUND<br>
Field of the Invention<br>
[0003]	The present invention relates generally to proteomics. 'More specifically,<br>
the present invention relates to using liquid chromatography in combination with mass spectrometry to identify and to quantify proteins, and peptides in a complex mixture, as well as to identify and quantify molecules in a complex mixture that produce precursor and fragment ions in a mass spectrometer. Further, the present invention also relates to using liquid chromatography in combination with mass spectrometry to retention-time track peptides in a complex mixture. More importantly the present invention provides a method of peptide identification without requiring the presence of a precursor ion mass thereby enabling the<br>
•«££.<br>
method to identify both chemically and post-translationally modified versions of peptides, allelic differences, peptides containing point mutations, as well as any other modifications of sequences deposited in the database being queried. Background of the Invention<br>
[0004]	Proteomics generally refers to studies involving complex mixtures of<br>
proteins. The field of proteomics includes studying and cataloging proteins in biological systems. Proteomic studies typically focus on identification of proteins, determination of changes in relative abundance among different conditions, or both. Identification and quantification of proteins in complex biological samples is a fundamental problem in proteomics.<br>
[0005]	Liquid chromatography coupled with mass spectrometry (LC/MS) has<br>
become a fundamental tool in proteomic studies. Separation of intact proteins or of their proteolyzed peptide products by liquid chromatography (LC) and subsequent analysis by mass spectrometry (MS) forms the basis of many common proteomic methodologies. Methods that measure changes in the expression level of proteins are of great interest as they can form the basis of biomarker discovery and clinical diagnostics.<br>
[0006]	In conventional proteomic studies, proteins of interest typically are first<br>
digested to produce a specific set of proteolytic pep tides rather than studying the<br>
intact proteins directly. The resulting peptides are then characterized during the<br>
proteomic analysis. A common enzyme used for such digestion is trypsin. In<br>
tryptic digestion, the proteins present in the complex mixture are cleaved to<br>
produce peptides as determined by the cleavage specificity of the proteolytic<br>
enzyme. From the identity and concentration of the observed peptides, algorithms<br>
known in the art can infer the identity and the concentration of the parent proteins,<br>
[0007]	In LC/MS analysis, the peptide digest is separated and analyzed by on-<br>
line, liquid chromatographic (LC) separation followed by on-line mass spectrometric (MS) analysis. Ideally, the mass of a single peptide, measured with sufficient accuracy, is sufficient to uniquely identity the peptide. In practice, however, achieved mass accuracies typically are on the order of 10 ppm or larger. In general, such mass accuracy is not sufficient to uniquely identify a peptide based upon mass measurement alone. For example, in the case of a mass accuracy of 10 ppm, on the order of 10 peptide sequences are identified in typical database searches. This number of sequences would increase significantly if search restraints on mass accuracy were lowered to allow for chemical or post-translational modifications, losses of H2O orNH3, point mutations, etc. Sequence repositories typically contain translated DNA sequences that have been annotated by homology to a known substrate. Thus, if a peptide's sequence is modified by either a deletions or substitutions, then tentative identification to that peptide by precursor mass only must be false.<br>
[0008]	Furthermore, two peptides can have the same amino acid composition but<br>
have different sequences. Mass accuracy alone is not sufficient to distinguish<br>
between peptides that differ in sequence but not in composition. Fragmentation<br>
techniques are known that cause peptides to break into fragments ions. These<br>
fragments can correspond to a subsequence of the original peptide, but other types<br>
of fragment ions may be observed. Fragment masses seen in the data can be used<br>
to confirm or deduce the precursor's sequence.<br>
[0009]	In the case of peptide precursors, subsequences can arise by the<br>
fragmentation at a single peptide bond in the precursor. Such fragmentation results in two sub-sequences. The fragment containing the peptide's C-terminal, if ionized, is termed a Y-ion, and the fragment containing the peptide's N-terminal, if ionized is termed a B-ion.<br>
[0010]	Known protein identification techniques search databases using accurate<br>
mass retention time (AMRT) data of precursors and fragments obtained from LC/MS experiments. For example, one way of obtaining such data is described in U.S. Patent No. 6,717,130 to Bateman ("Bateman"), which is hereby incorporated by reference in its entirety. In Bateman, such data can be obtained using a high-and low-energy switching protocol applied as part of an LC/MS analysis of a single injection of a peptide mixture. In such data the low-energy spectra contains ions primarily from unfragmented precursors, while the high-energy spectra contain ions primarily from fragmented precursors.<br>
[0011]	To identify the presence of a protein in such data, an AMRT (empirically<br>
describing those ions from a peptide or from a fragment) is selected from the low-<br>
energy data. If trypsin is used in the digest, this AMRT is presumed to be a<br>
tryptic precursor. Using the AMRT data, known methods search a database of<br>
peptide masses for tryptic peptides whose masses lay within a mass search<br>
window or threshold.<br>
[0012]	If a theoretical peptide mass from a database lies within a mass search<br>
window of the mass of a precursor measured in the data, it is deemed a hit. That<br>
is, the precursor in the data is hit by the peptide in the database; or alternatively<br>
the peptide in the database is hit by the precursor in the data.<br>
[0013]	The search results in a hit-list of possible matching peptides from the<br>
database. These possible matching database peptides may or may not be weighted by statistical factors. The possible outcomes of such a search are that no possible<br>
matching database peptides are identified, one possible matching database peptide<br>
is identified, or more than one possible matching database peptide are identified.<br>
The higher the resolution of the MS, assuming proper instrument calibration, the<br>
smaller the ppm threshold, and consequently, the fewer the false identifications.<br>
[0014]	If there is one or more hit to the theoretical peptides in the database,<br>
conventional searches then use data from high-energy AMRTs to validate a<br>
possible matching database peptide. High-energy AMRTs are first searched to<br>
isolate those high-energy AMRTs that occur at the same retention time as the low-<br>
energy AMRT being validated. Typically, the high-energy AMRTs that are<br>
isolated are those whose retention times are substantially the same as the retention<br>
time of the low-energy AMRT being validated.<br>
[0015]	For each database peptide on the hit list, the algorithm determines the<br>
masses of all possible Y-ions and B-ions that can be obtained through collisionally<br>
induced disassociation of the precursor. The isolated high-energy AMRT data is<br>
then searched for each of these Y- and B-ions. The peptide sequence having the<br>
greatest number of hits, or satisfying other criteria, is returned as the correct hit,<br>
i.e., the identity of the target precursor. This result can be stored and displayed.<br>
[0016]	This process can be repeated for each low-energy AMRT in the digestion<br>
mixture. Further analysis can be performed on the results including storing the<br>
results, displaying the results, quantitation and combining the results with those of<br>
other injections.<br>
[0017]	During the search, multiple charge states and multiple isotopes can be<br>
searched. In addition, the ions, or the charged reduced AMRTs could be searched.<br>
Further, empirically produced confidence rules can be applied to help identify<br>
valid hits, and better confidence is obtained with a higher number of high-energy<br>
hits.<br>
[0018]	In summary, given a set of data acquired by an LC/MS system, known<br>
protein identification techniques search a database of theoretical protein sequences to identify the proteins in the data. That is, known protein identification techniques start with the data and search a database. The invention described below, in contrast, starts with the database and searches the data.<br>
RIEF SUMMARY OF THE INVENTION<br>
[0019]	In contrast to conventional protein identification techniques, embodiments<br>
of the present invention, start with a theoretical peptide sequence (generally, but not always, obtained from a protein database), and search the data for evidence of precursor and fragment ions that correspond to the theoretical peptide sequence. If a sufficient number of such masses are found in the data, and at a common retention time, then the peptide sequence is identified in the data. If this method finds in the data one or more peptide sequences associated with a given protein, then the protein is taken to be identified in the sample.<br>
[0020]	Embodiments of the present invention search the data obtained using an<br>
LC/MS system using a preselected database. For example, in one embodiment of<br>
the present invention, an eluent output by the liquid chromatograph (LC) is<br>
introduced into a mass spectrometer (MS) through an ESI interface. The first<br>
quadrupole (Ql) of the MS functions simply as an ion guide. An alternating<br>
voltage is applied to a collision cell. Spectra are collected of all precursors and all<br>
of their fragments in an alternating fashion as described in Bateman.<br>
[0021]	More specifically, embodiments of the present invention collect spectra<br>
that alternate uniformly in time between a low-energy mode and a high -energy<br>
mode.   There is no MS spectral selection applied prior to high-energy<br>
fragmentation. The high-energy mode spectra contain fragment ions of all<br>
precursor ions. Because of the high-duty cycle of the alternating mode of data<br>
acquisition, the chromatographic profiles of all detected precursors and fragments<br>
are preserved. This mode of data acquisition enables determination or<br>
measurement of the retention times, as well as m/z and intensities of all ions seen<br>
in the low and in the high-energy modes..<br>
[0022]	The low-energy mode corresponds to conventional LC/MS acquisition.<br>
The high-energy mode is alternatively referred to herein as elevated-energy mode. The high-energy or elevated-energy mode corresponds to LC/MSE acquisition. The low-energy mode contains spectra of primarily precursor ions. The high-energy mode contains spectra of primarily fragment ions.<br>
BRIEF DESCRIPTION OF THE DRAWINGS<br>
[0023]	Figure 1 is a schematic diagram of a system for identifying and<br>
quantifying proteins in a complex biological mixture according to an embodiment of the present invention.<br>
[0024]	Figure 2 illustrates times at which exemplary spectra are acquired. These<br>
spectra result from application of the alternating low- and high-energy modes<br>
according to an embodiment of the present invention.<br>
[0025]	Figures 3A-3C are flow charts for a method for peptide identification<br>
according to an embodiment of the present invention.<br>
[0026]	Figure 4A is an exemplary plot showing all AMRTs found at high energy.<br>
[0027]	Figure 4B is an exemplary plot showing all AMRTs found at low energy.<br>
[0028]	Figure 4C is an exemplary detection chromatogram derived from the data<br>
shown in figures 4A and 4B showing the number of hits per retention time for a given theoretical peptide from a database. Figure 4C shows an example of a precursor peptide sequence in the database that appears to have a significant number of hits at approximately 78 minutes.<br>
[0029]	Figure 5A is an exemplary plot showing only hits of AMRTs in the high-<br>
energy plot of Figure 4A.<br>
[0030]	Figure 5B is an exemplary plot showing only hits of AMRTs in the low-<br>
energy plot of Figure 4B.<br>
[0031]	Figure 5C is a histogram plot of the hits in the high and low energy plots<br>
of figure 5A and 5B. Figure 5C shows an example of a precursor peptide sequence<br>
in the database that appears to have a significant number of hits at approximately<br>
78 minutes.<br>
[0032]	Figures 6A-B illustrate how the zero crossing of the 2nd derivative of a<br>
chromatographic peak can be obtained.<br>
[0033]	Figure 7 is a flowchart for a method for comparing peak shape and peak<br>
width according to an embodiment of the present invention.<br>
[0034]	Figure 8A il lustrates an exemplary spectrum of a peptide.<br>
[0035]	Figure 8B illustrates a series of mass chromatograms corresponding to<br>
each of the six ions illustrated in figure 8 A.<br>
[0036]	Figure 9A is a plot of an exemplary high-energy spectrum showing many<br>
spectral peaks.<br>
[0037]	Figure 9B plots only those spectral peaks that have identical retention time<br>
to that of one of the peaks illustrated in figure 9A.<br>
[0038]	Figure 9C is a series of plots illustrating the chromatographic profile<br>
corresponding to most of the peaks that appear in figure 9 A.<br>
[0039]	Figure 10 is a flowchart for a method for identifying peptides in a complex<br>
mixture using ions according to an embodiment of the present invention.<br>
[0040]	Figure 11 shows an example of a peptide at approximately 49 minutes that<br>
appears to have a significant number of hits, but for which there was no precursor<br>
in the database.<br>
[0041]	Figure 12 is a flow chart for a method for performing the electronic<br>
depletion algorithm (EDA) according to an embodiment of the present invention.<br>
[0042]	Figure 13 is a method for determining retention time error and mwHPlus<br>
error according to an embodiment of the present invention.<br>
[0043]	Figure 14 is a flow chart for a method of generating the detection<br>
chromatogram using addition of detection Gaussian peaks.<br>
[0044]	Figure 15 is a flowchart for a method for identifying sequence<br>
identifications and their retention times according to an embodiment of the present<br>
invention.<br>
[0045]	Figure 16 is a flowchart for collecting AMRTs and ions associated with<br>
each sequence identification according to an embodiment of the present invention.<br>
[0046]	Figures 17A-17C contain flow charts for a method for the determination of<br>
AMRTs from an ion list according to an embodiment of the present invention. DETAILED DESCRIPTION OF THE INVENTION Glossary<br>
[0047]	As used herein, the following terms have the specified meanings:<br>
[0048]	Protein - a specific primary sequence of amino acids assembled as a single<br>
polypeptide.<br>
[0049]	Peptide - a specific sequence of amino acids assembled as a single<br>
polypeptide contained within the primary sequence of a protein.<br>
[0050]	Tryptic peptides - peptides generated from a protein sequence that result<br>
from enzymatic cleavage of the protein by trypsin. In the ensuing description, digest peptides are referred to as tryptic peptides for convenience. However, it should be understood that embodiments of the present invention apply to other techniques for peptide digestion.<br>
[0051]	Precursor peptides- tryptic peptides (or other protein cleavage products)<br>
that are directly generated using the protein cleavage protocol. The precursor peptides from the sample are separated chromatographically and passed to mass spectrometer. In the mass spectrometer, the ion source ionizes these precursor peptides to produce a positively charged, protenated form of the precursor. The mass of such positively charged protenated form can be referred as the mwHPlus or MH+ of the precursor. In the following, we use the term "precursor mass" refers generally to the protenated, mwHPlus or MH+ mass of the ionized, peptide precursor.<br>
[0052]	Fragments—Multiple types of fragments can occur in MSE spectra. In the<br>
case of tryptic peptide precursors, fragments can include polypetide ions that are<br>
produced from collisional fragmentation of the intact peptide precursors and<br>
whose primary amino acid sequence is contained within the originating precursor<br>
peptide. Y-ions and B-ions are examples of such peptide fragments. Fragments<br>
of tryptic peptides can also include immonium ions, functional groups such as a<br>
phosphate ion (PC^), mass tags cleaved from a specific molecule or class of<br>
molecules, or "neutral loss" of water (HzO) or ammonia (NHa) molecules from the<br>
precursor.	x<br>
[0053]	Y-ions and B-ions - If a peptide fragments at the peptide bond, then if a<br>
charge is retained on the N terminal fragment, that fragment ion is termed a B-ion. If the charge is retained on the C terminal fragment, the fragment ion is termed a Y-ion. A more comprehensive list of possible fragments and their nomenclature is provided in See Roepstorff and Fohlman, Biomed Mass Spectrom, 1984; 11(11):601, and Johnson et al, Anal. Chem 1987, 59(21): 2621:2625, both of which are hereby incorporated by reference.<br>
[0054]	Chromatographic profile - the intensity versus time of a chromatographic<br>
peak at a single mass corresponding to a single precursor or fragment ion in an<br>
LC/MS analysis. Mass chromatograms can contain chromatographic profiles of<br>
one or more such ions.<br>
[0055]	Apex retention time or chromatographic retention time- the point in a<br>
chromatographic profile at which an entity reaches its maximum intensity during<br>
an LC/MS analysis.<br>
[0056]	Ions - each peptide appears as an ensemble of ions due to the natural<br>
abundance of the isotopes of the constituent elements. An ion has a retention time<br>
and an m/z value. The mass spectrometer (MS) detects only ions. The LC/MS<br>
technique produces a variety of observed measurements for every detected ion.<br>
This includes: the charge-to-mass ratio (m/z) m, the retention time, and the signal<br>
intensity of the ion.<br>
[0057]	MwHPlus -The neutral, monoisotopic mass of the peptide plus the weight<br>
of one proton, 1.007825 amu.<br>
[0058]	AMRT - accurate mass retention time. The AMRT is an empirical<br>
description of a peptide in terms of its mass, retention time and total intensity. When a peptide elutes from the chromatographic column, it elutes over a specific retention time period and reaches its maximum signal at a single retention time (apex retention time). After ionization and (possible) fragmentation, the peptide appears as a related set of ions. The different ions in the set correspond to different isotopic compositions and charges of the common peptide. Each ion within the related set of ions produces a single apex retention time and peak shape. Since these ions originate from a common peptide, the apex retention time and peak shape of each ion is identical, within some measurement tolerance. The MS acquisition of each peptide produces multiple ion detections for all isotopes and charge states, all sharing the same identical apex retention-time and peak shape within some measurement tolerance.<br>
[0059]	In an LC/MS separation, a single peptide (precursor or fragment) produces<br>
many ion detections, which appears as a cluster of ions, at multiple charge states. Deconvolution of these ion detections from such a cluster, indicates the presence of a single entity of a unique monoisotopic mass, at a specific retention time, of a measured signal intensity, in a charge state, giving rise to the AMRT.<br>
[0060]	It is not possible to infer directly from an AMRT whether it is a precursor,<br>
or a fragment, or a chemically modified peptide, let alone what its sequence is. Molecules other than peptides can be described using AMRTs.<br>
[0061]	Protein Database - In embodiments of the present invention, the user<br>
selects or otherwise supplies a database of proteins. Alternatively, a default<br>
database or other predetermined database can be used. Each protein is described<br>
by its primary sequence of amino acids. It is up to the user to choose which<br>
database (or database subset) is to be compared to the data. A user might choose a<br>
database that is intended to closely match the proteins under study. For example,<br>
an E. Coli database would be compared to data obtained from a cell lycate of E.<br>
Coli. Similarly, a human serum database would be compared to data obtained<br>
from human serum. A user could choose a subset database. A user could choose a<br>
superset database, such as all proteins listed in SwissProt. A user could choose a<br>
data a base that contains simulated proteins, described by random sequences of<br>
amino acids. Such random databases are used in control studies to evaluate or<br>
calibrate protein identification systems and search algorithms. A user could<br>
choose a database that combines both naturally occurring and artificial sequences.<br>
[0062]	From the protein database, software can infer from each sequence, the<br>
sequence and masses of tryptic precursor ions, Y- and B-ions, and other possible<br>
fragment ions that would result from those precursors.<br>
[0063]	Detailed Description<br>
[0064]	Figure 1 is a schematic diagram of a system for identifying and<br>
quantifying proteins in a complex biological mixture according to an embodiment of the present invention. A sample 102 is injected into a liquid chromatograph 104 through an injector 106. A pump 108 pumps the sample through a column 110 to separate the mixture into component parts according to retention time through the column.<br>
[0065]	The output from the column is input to a mass spectrometer 112 for<br>
analysis. Initially, the sample is desolvated and ionized by a desolvation/ionization device 114. Desolvation can be any technique for desolvation, including, for example, a heater, a gas, a heater in combination with a gas or other desolvation technique. lonization can be by any ionization<br>
techniques, including for example, electrospray ionization (ESI), atmospheric pressure chemical ionization (APCI), matrix assisted laser desorption (MALDI) or other ionization technique. Ions resulting from the ionization are fed to a collision cell 118 by a voltage gradient being applied to an ion guide 116. Collision cell 118 can be used to pass the ions (low-energy) or to fragment the ions (high-energy). For example, as described in Bateman, an alternating voltage can be applied across the collision cell 118 to cause fragmentation. Spectra are collected for the precursors at low-energy (no collisions) and fragments at high-energy (results of collisions).<br>
[0066]	The output of collision cell 118 is input to a mass analyzer 120. Mass<br>
analyzer 120 can be any mass analyzer, including quadrupole, time-of-flight (TOP), ion trap, magnetic sector mass analyzers as well as combinations thereof. A detector 122 detects ions emanating from mass analyzer 122. Detector 122 can be integral with mass analyzer 120. For example, in the case of a TOP mass analyzer, detector 122 can be a microchannel plate detector that counts intensity of ions, i.e., counts numbers of ions impinging it.<br>
[0067]	A storage medium 124 provides permanent storage for storing the ion<br>
counts for analysis, For example, storage medium 124 can be an internal or<br>
external computer disk. An analysis computer 126 analyzes the stored data. Data<br>
can also be analyzed in real time without requiring storage in a storage medium<br>
124. In real time analysis, detector 122 passes data to be analyzed directly to<br>
computer 126 without first storing it to permanent storage.<br>
[0068]	Collision cell 118 performs fragmentation of the precursor ions.<br>
Fragmentation can be used to determine the primary sequence of a peptide and<br>
subsequently lead to the identity of the originating protein.<br>
[0069]	Collision cell 118 includes a gas such as helium, argon, nitrogen, air, or<br>
methane. When a charged peptide interacts with gas atoms, the resulting collisions can fragment the peptide by breaking it up at one or more characteristic bonds. The most common resulting fragments are described as Y- or B- ions. Such fragmentation can be accomplished as on-line fragmentation by switching the voltage in a collision cell between a low voltage state (low energy, 
energy, &gt;15V) which obtains MS spectra of the collisionally induced fragments<br>
of the precursors. High and low voltage are referred to as high and low energy,<br>
since a high or low voltage respectively is used to impart kinetic energy to an ion.<br>
[0070]	Various protocols can be used to determine when and how to switch the<br>
voltage for such an MS/MS acquisition. For example, conventional methods<br>
trigger the voltage in either a targeted or data dependent mode (data-dependent<br>
analysis, DDA). These methods also include a coupled, gas-phase isolation (or<br>
pre-selection) of the targeted precursor. The low-energy spectra are obtained and<br>
examined by the software in real-time. When a desired mass reaches a specified<br>
intensity value in the low-energy spectrum, the voltage in the collision cell is<br>
switched to the high-energy state. The high-energy spectra are then obtained for<br>
the pre-selected precursor ion. These spectra contain fragments of the precursor<br>
peptide seen at low energy. After sufficient high-energy spectra are collected, the<br>
data acquisition reverts to low-energy in a continued search for precursor masses<br>
of suitable intensities for high-energy collisional analysis.<br>
[0071]	Although conventional switching techniques can be employed,<br>
embodiments of the present invention preferably use a novel fragmentation<br>
protocol in which the voltage is switched in a simple alternating cycle. This<br>
switching is done at a high enough frequency so that multiple high- and multiple<br>
low-energy spectra are contained within a single chromatographic peak. Unlike<br>
conventional switching protocols, the cycle is independent of the content of the<br>
data.<br>
[0072]	In summary, each sample 102 is injected into the LC/MS system. The<br>
LC/MS system produces two sets of spectra, a set of low-energy spectra and a set<br>
of high-energy spectra. The set of low-energy spectra contain primarily ions<br>
associated with precursors. The set of high-energy spectra contain primarily ions<br>
associated with fragments. These spectra are stored in a storage medium 124.<br>
After data acquisition, these spectra can be extracted from the storage medium and<br>
displayed and processed by post-acquisition algorithms in the analysis computer<br>
126.<br>
[0073]	The data acquired by the high-low protocol allows for the accurate<br>
determination of the retention times, mass-to-charge ratios, and intensities of all<br>
ions and AMRTs collected in both low- and high-energy modes. In general, different ions are seen in the two different modes, and the spectra acquired in each mode are then analyzed separately to determine the retention times, mass-to-charge ratios and intensities of the ions seen in the respective mode.<br>
[0074J	The ions from a common precursor as seen in one or both modes will share<br>
the same retention times and peak shapes. The high-low protocol allows the meaningful comparison of retention time and peak shape of ions within a mode and between modes. This comparison can then be used to group ions seen in both low-energy and high-energy spectra by their common retention time and peak shape. Figures 2, 8, and 9 and the following discussion show how the low-energy spectra and the high-energy spectra can be used to find ions that have common retention times and peak shapes.<br>
[0075]	Figure 2 illustrates the times at which spectra are obtained during the<br>
elution of a peak resulting from application of the alternating low- and high-<br>
energy modes according to an embodiment of the present invention. Figure 2<br>
shows that the chromatographic profiles and retention time of ions associated with<br>
the precursors can be reconstructed for both the high- and low-energy spectral<br>
data.<br>
[0076]	Peak 202 represents the chromatographic elution profile of a single<br>
precursor. The horizontal axis is elution time. The vertical axis is arbitrary, and<br>
represents the time-varying concentration, the chromatographic profile, of the<br>
precursor as it elutes from the chromatographic column.<br>
[0077]	Plots 204a (low-energy) and 204b (high-energy) in figure 2 depict the<br>
same chromatographic peak 202, wherein the horizontal axis represents time and<br>
the vertical axis represents intensity of an ion.<br>
[0078]	An eluting molecule, passed to the mass spectrometer, produces ions in<br>
both low- and high-energy modes. The ions produced in the low-energy mode are primarily those of the precursor ions in possibly different isotopic and charge states. In proteomic studies, the precursor ions are peptides generated from enzymatic digestion (typically a tryptic digest) of the intact protein(s). In high-energy mode, the ions are primarily different isotopes and charge states of the<br>
fragment ions of those precursors. High-energy mode can also be referred to as elevated-energy mode.<br>
[0079]	In the plot of peak 202, the alternating bars of different density represent<br>
the times at which spectra were collected with low and high-energy voltages during the elution of the depicted chromatographic peak. The bars alternate uniformly in time. Plot 204a illustrates exemplary the times at which low-energy voltage was applied in the collision cell, resulting in low-energy spectra. Plot 204b illustrates times at which high-energy voltage was applied in the collision cell, resulting in high-energy spectra. As shown in figures 204a and 204b, the chromatographic peak is sampled multiple times, by the high- and low-energy modes. From these multiple samples, accurate retention times of all the ions associated with the peak and seen in the high- and low-energy spectra can be inferred. These accurate retention times are obtained by interpolation of the intensities sampled by the respective spectra.<br>
[0080]	For a molecule that elutes at a retention time, tr, all associated ions will<br>
also be seen to elute at exactly the same retention time to within some measurement precision. This phenomenon is demonstrated in figures 8A-B and 9A-C. Ions associated with a common precursor have identical chromatographic peak shapes. Figures 8A-B illustrate an exemplary spectrum extracted at 72.5 minutes that contains isotopes of a Z=+2 ion. The six ions 802a-f (appearing in mass 740 - 742.5 amu) shown in figures 8A-B are isotopes from a tryptic peptide of the serotransferrin precursor protein in human serum. The peptide's sequence is MYLGYEYVTAIR. The spacing between the mass-to-charge (m/z) of the ions is 0.5 amu, indicating that the charge of the ion is Z=2. At Z=2, the m/z value for the 12C monoisotopic ion is 739.96 amu.<br>
[0081]	Figure 8B illustrates a series of mass chromatograms corresponding to<br>
each of the six ions illustrated in figure 8 A. As indicated by a vertical line 804, the retention time of each ion has the same value of 72.53 min, indicating that the chromatographic apex of each ion falls within the same, single scan. Vertical line 804, with arrow heads," indicates that the retention times of each of the six chromatographic peaks, 812a-f, is the same, confirming that the ions seen in the top plot are related to a common precursor.<br>
[0082]	Figure 9A is a plot of an exemplary high-energy spectrum showing many<br>
spectral peaks 910-916. Figure 9C is a series of plots illustrating the<br>
chromatographic profile corresponding to several of the peaks that appear in<br>
figure 9A. Spectral peaks 910,911,912, and 913 have corresponding<br>
chromatographic peaks plotted in figure 9C as peaks 920, 921, 922, and 923. The<br>
chromatographic peaks of these four spectral peaks have substantially the same<br>
peak shape and retention time, consistent with the hypothesis that they are from<br>
the same peptide. Spectral peaks 914, 915, and 916, correspond to<br>
chromatographic peaks 924, 925, and 926. The retention times of peaks 924, 925,<br>
and 926 are substantially the same, as are their peak shapes, consistent with the<br>
hypothesis that that spectral peaks 914, 915, and 916 are from the same peptide.<br>
[0083]	Figure 9B plots only those spectral peaks that have identical retention time<br>
to that of peak 910. Thus only peaks 910, 911, 912, and 913, are re-plotted as<br>
vertical lines 930, 931, 932, 933 in figure 9B. All other peaks in figure 9A are<br>
excluded and not plotted in figure 9B because of retention time match<br>
misalignment. This example demonstrates that chromatographic information can<br>
be used to select related spectral peaks, or exclude unrelated peaks. As described<br>
below, the PDS algorithm of embodiments of the present invention relies on<br>
correspondence in retention time between a precursor and its fragments, as seen in<br>
both low- and high-energy spectra.<br>
[0084]	Figures 3A-C are flowcharts for a method for peptide identification<br>
according to an embodiment of the present invention. For ease of reference, the method of the present invention is called the peptide data search (PDS) algorithm. Given a peptide sequence corresponding to a protein in an exact mass database, embodiments of the present invention search the data to determine if the peptide sequence is present. Prior knowledge about the peptide's retention time is not required. However, knowing a range of times at which the peptide elutes can help reduce the number of data AMRTs searched. This reduction speeds up the computation and reduces the rate of false positive detections. It also allows a lower detection threshold (described below) to be applied to the detection chromatogram.<br>
[0085]	Prior to searching the data, ions or AMRTs must be obtained from both the<br>
low- and high-energy data. Figure 3A is a flowchart for a method for obtaining the ions or AMRTs according to an embodiment of the present invention. The method of figure 3A is applied to both the low- and high-energy data to obtain the required ions or AMRTs.<br>
[0086]	In step 302, collected spectra are read from the hard drive. In step 304,<br>
ions are detected from the spectra. For example, ions can be detected by peak<br>
finding algorithms applied to spectra and mass chromatograms as described by<br>
Bateman (6,717,130), or by two-dimensional convolution techniques described in<br>
co-pending PCT Application No. PCT/US05/04180, filed February 11, 2005,<br>
entitled "Apparatus and Method for Identifying Peaks in Liquid<br>
Chromatography/Mass Spectrometery Data and for Forming Spectra and<br>
Chromatograms," (the "'4180 Application"), hereby incorporated by reference. In<br>
an embodiment of the present invention, the measured properties of an ion are its<br>
retention time, mass-to-charge ratio (m/z), and intensity. Step 304 stores a list of<br>
these ion properties in a table. From this table, the list of these ions and their<br>
properties is input to method 306. Method 306 determines accurate mass<br>
retention times (AMRTs) and writes AMRT parameters into a table and stores<br>
these parameters.<br>
[0087]	In an LC/MS experiment, a peptide appears as a set of ions, each ion<br>
corresponds to a peptide in a different isotopic and charge state. An AMRT is the set of ions generated by a peptide. The properties of an ARMT are determined from the set of ions that comprise an AMRT.<br>
[0088]	An AMRT corresponds to a set of ions from the ion list obtained in step<br>
304. Thus method 306 parses the ion list into sets of ions, where each set is an<br>
AMRT. The properties of an ARMT are determined from such sets. An AMRT is<br>
described by four parameters: a retention time, mwHPlus, intensity, and fractional<br>
charge state. An AMRT consists of a set of at least 2 or more ions. Two or more<br>
ions are needed to establish the charge and, hence, the mwHPlus of an ARMT.<br>
[0089]	The retention time and mwHPlus of an AMRT is the retention time and<br>
mwHPlus of the lowest mass, monoisotopic ion in the set. The intensity of the AMRT is the sum of the intensities of the ions in the set. It is also possible to<br>
derive for each AMRT a figure of merit known as a fractional charge state. The fractional charge state is the sum of the charges of each ion, weighted by fractional intensity of that ion with respect to the AMRT intensity. Ions not assembled into a set are treated as single ions, described by their retention time, m/z and intensity. A single ion can be effectively considered to be an AMRT if a charge for that ion is assumed or assigned by a rule.<br>
[0090]	The mwHPlus of a peptide is the neutral, monoisotopic mass of the peptide<br>
plus the mass of one proton, referred to as [M+H], or mwHPlus, or MH+. The monoisotopic mass M is the mass of the peptide when all its atoms are in their lowest mass, most abundant isotopic state.<br>
[0091]	The PDS and EDA algorithms (described below) as applied to peptides<br>
take as input an AMRT's retention time, mwHPlus, and intensity. The list of single ions is stored with the AMRT list. A single ion can be effectively considered to be an AMRT if a charge for that ion is assumed or assigned by a rule. Thus single ions can be optionally included with the ARMT list as input to the PDS and EDA algoirthms. Alternatively, the PDS and EDA algorithms (as described below) can be applied only to the ions, as obtained in step 304, bypassing step 306.<br>
[0092]	The algorithm implemented in step 306 makes use of known properties of<br>
the mass spectrometric properties of peptides. It is known that a peptide appears in<br>
a mass spectrum as a set of ions at different values of m/z. Examples of such<br>
spectra are described in Mass Spectrometry of Biological Materials, by Barbara<br>
Seliger Larsen (Editor), Charles N. McEwen (Editor), Marcel Dekker; 2nd Rev<br>
edition (March 1, 1998), pages 34-46, hereby incorporated by reference.<br>
[0093]	A peptide ion can have possible mass-to-charge ratios of<br>
where M is the monoisotopic mass of the neutral peptide, H = 1.00728 amu, is the mass of the proton responsible for the peptide's charge, Z is the peptide's charge, N is isotope number of the peptide (an integer), and 1.00335 amu is the mass difference between 13C and 12C isotopes. This mass difference is an approximation to the actual mass differences that occur between isotopes of the same peptide.<br><br>
[0094]	The value for N = 0 corresponds to this monoisotopic state. The<br>
monoisotopic mass of the peptide for N=0 and for Z = 1 is then [M + H\.<br>
[0095]	Figure 8 A shows a portion of a mass spectrum that contains a set of 6 ions<br>
802a-f associated with a single peptide. Such a set is commonly referred to as a cluster, or ion cluster. The spacing in m/z between the ions in this cluster is 0.5 amu, evidence that the ions are different isotopic states of a peptide having charge Z=2. The lowest mass ion is the monoisotope and appears at m/z = 739.96. We infer the mwHPlus of the peptide to be 739.96 * 2 - 1.00739 = 1478.91 amu. If this peptide appears at charge Z=l, then the monoisotope will appear at m/z of 1478.91 amu, and if the peptide appears at Z=3, the monoisotope will appear at (1478.91-K2)/3 = 493.64 amu. The mass spectrum of a peptide can be described as consisting of one or more clusters of ions; each cluster corresponds to ions of the same charge; and the different clusters correspond to the ions in different charge states.<br>
[0096]	In the LC/MS data generated here, an AMRT appears as a set of ions<br>
where each ion is described by a retention time, m/z, and intensity. It is straightforward to infer, from the ion list obtained in step 304, sets of ions, where each set corresponds to an AMRT, and each such AMRT is inferred to correspond to a peptide. Because we assume each AMRT derives from a peptide, we can make use of rules to arrive at such sets. For example, ions from a peptide must occur at a common retention time (as described by Bateman), and the mass-to-charge relationships between ions in such a set must conform to the rules described above. The method 306 then collects sets of ions whose properties satisfy such rules. Each such set corresponds to an AMRT, and each such AMRT is inferred to correspond to a peptide.<br>
[0097]	The method in 306 is then applied to the ion list obtained from the low-<br>
energy-spectra to obtain the low-energy AMRTs. It is also applied independently to the high-energy ion list obtained from the high-energy spectra to obtain the high-energy AMRTs.<br>
[0098]	In summary, step 306 determines an AMRT to be a set of ions where each<br>
ion in the set is inferred to arise from a single, common peptide. The peptide can be a precursor or fragment. Step 306 makes use of the known properties of peptide<br>
spectra as described and illustrated above, and the common retention time of such ions, to infer the sets of ions that correspond to AMRTs. Step 306 then computes the AMRT parameters (retention time, mwHPlus, intensity, and fractional charge state) from each ion set and stores those parameters and notes the ions comprising each set.<br>
[0099J	Known algorithms deconvolve ions seen peptide spectra in order to infer<br>
their charge state Z and mwHPlus. An example of such algorithms is given in<br>
Role of Accurate Mass Measurement C+/-10 ppm) in Protein Identification<br>
Strategies Employing MS or MS/MS and Database Searching Karl R. Clauser,<br>
Peter Baker, and Alma L. Burlingame, Anal. Chem. 1999, 71, 2871-2882.<br>
Another such algorithm is A Universal Algorithm for Fast and Automated Charge<br>
State Deconvolution of Electrospray Mass-to-Charge Ratio Spectra. Zhongqi<br>
Zhang and Alan G. Marshall, J. Am Soc. Mass Spectrom. 1998, 9, 224-233, each<br>
of which is hereby incorporated by reference.<br>
[00100]	But these known algorithms operate only on single spectra. Thus such<br>
algorithms can determine the charge state and mwHPlus of each peptide seen in a<br>
spectrum, but cannot determine an accurate retention time for such peptides.<br>
[00101]	The method employed in step 306 to obtain AMRTs from ion lists is novel<br>
and described in figure 17A-C. Figure 17A summarizes the method. Step 1704 takes the ion list 1702 and determines sets of ion clusters. Each ion cluster contains ions that have the same charge Z and retention time. The ion in the cluster having lowest mass is the monoisotopic ion of the peptide. This list of clusters is stored in list in 1706, and this list is input to step 1708.<br>
[00102]	Step 1708 examines the list and determines which clusters have the same<br>
retention time and mwHPlus, but different charge states. If several clusters do appear with the same retention time and mwHPlus and different charge states, then 1708 infers that these clusters must be from the same peptide. The clusters are assembled into a single set and that combined set is the AMRT for a single peptide. If a cluster appears with a unique retention time and mwHPlus, then we infer that the corresponding peptide produced only one cluster, and 1708 determines that the one cluster is an AMRT. Step 1710 stores the AMRTs and the<br>
ions not assembled into cluster. It is this combined list that is input to the PDS and<br>
EDA algorithms.<br>
[00103]	Figure 17B shows how step 1704 identifies ion clusters from the ion list<br>
1702. A nested iterative loop decrements two search parameters, Zm and Nm. The<br>
initial values for these search parameters are Zmax and Nmax. In each pass, Zm is<br>
the cluster charge and Nm is the minimum number of ions required to be in a<br>
cluster. For low-energy ions the initial parameter are Zmax = 6, and Nmax = 8.<br>
For high-energy ions these initial parameter are Zmax = 3, and Nmax = 8.<br>
[00104]	Step 1736 traverses all ions in the list and finds all pairs of ions that have<br>
the same retention time and are separated in m/z by 1,003 3 5/Zm. The value 1.00335 amu is the mass difference between 13C and 1ZC isotopes. This mass difference is an approximation to the actual mass differences that occur between isotopes of the same peptide Using a mass threshold of 20 ppm (as described below), this single, approximate, value is sufficient to determine whether a pair of ions are isotopes of a common peptide.<br>
[00105]	Step 1738 assembles ion pairs into clusters. Thus if ion 7 is paired with ion<br>
10, and ion 10 is paired with ion 15, then ion 7,10, and 15 forma cluster. If ion 15<br>
is paired with another ion, then the cluster is expanded by that one additional ion.<br>
An ion is considered for pairing only if it is not tagged. Initially, no ion is tagged<br>
and all ions are considered. Ions are tagged in subsequent steps, as described<br>
below. Step 1738 determines all possible clusters that satisfy the pairing<br>
requirements.<br>
[00106]	The retention time requirement applied in step 1736 is determined by a<br>
retention time window and the m/z requirement applied in step 1736 is determined by a ppm window. The retention time window is 20% of the chromatographic peak width (FWHM) and is +/- 0.1 minutes for a chromatographic peak width of 0.5 min (FWHM).. The ppm window for at TOP with resolution of 15,000 is +/-20 ppm. That is, ions are paired only if the difference in their retention time falls within the window, and their mass difference from the rn/z model described above lies within the ppm window.<br>
[00107]	In step 1738, a set of ions is recorded as a cluster and tagged provided two<br>
additional conditions are satisfied. The number of ions in the cluster must be<br>
greater than or equal to Nm, and the intensity ratios of the N=l and N=0 ions must<br>
be within range of values expected for such ions. We define r to be the intensity<br>
ratio of the N=l ion to the N=0, monoisotopic ion. The intensity distribution of<br>
ions in a cluster is well known and described in references cited above. In the<br>
method described here, the nominal intensity ratio r is approximated by rO =<br>
(mwHPlus/20) * 0.0107. Where (mwHPlus/20) is an approximation to the number<br>
of carbon atoms in the peptide, and 0.0107 is the approximate abundance of 13C<br>
atoms to 12C atoms. The allowed range of intensity ratios of N=l to N=0 ions is<br>
40%, or rO * 1.4 and rO/1.4. Thus we require that r&gt;rO/l .4 and r<ro></ro>
[00108]	If these two rules are not satisfied, then steps 1740 and 1742 will not tag<br>
the ions, allowing them to be considered in future iterations. These rules are applied to detect and remove from consideration accidental pairings of ions from unrelated peptides.<br>
[00109]	Step 1744 obtains cluster parameters for those clusters that were obtained<br>
in step 1738 and accepted by steps 1740 and 1742. The cluster parameters are retention time, mwHPlus, intensity and charge. The retention time and mwHPlus of a cluster is that of the lowest mass ion in the cluster. The intensity of the cluster is the sum of the intensities of ions in the cluster. The charge of the cluster is the Zm parameter. If the ion cluster is accepted, 1744 tags the ions so these ions are no longer considered in subsequent iterations. Step 1746 stores the cluster parameter including the ions that form the cluster for those clusters that have been accepted.<br>
[00110]	The next iteration decrements the charge parameter Zm. Thus, in the<br>
second iteration, clusters are found that have charge Zm=Zmax-l and have Nm=Nmax or greater ions. Continued iterations decrement Zm until Zm=l is reached. When Zm=l is reached, clusters of all charge states containing Nm=Nmax or more ions have been identified. After Zm=l is reached, the next iteration resets Zm= Zmax and decrements Nmax by one, so Nm=Nmax-l. This nested iteration proceeds until Nm = 2. Thus the iterations proceed from highest to lowest values for Nm in the outer loop, and highest to lowest value for Zm in the inner loop.<br>
[00111]	Step 1750 stores all clusters (cluster parameters and associated ions) found<br>
in the original ion list, and all ions not found to be in clusters.<br>
[00112]	The operation of Step 1708 is described in Figure 17C. The list of ion<br>
clusters 1750 is looped over. Increment variable nc refers to cluster number, and<br>
nc is initialized to 1. Step 1776 finds all clusters that have the same retention time<br>
and mwHPlus as cluster nc. If step 1778 determines there are no such other<br>
clusters, then step 1780 notes that cluster nc is an AMRT and stores it parameters<br>
in step 1786. Thus, at a given retention time, if there is only one cluster with a<br>
given mwHPlus value, than cluster is considered to be an AMRT of a peptide.<br>
That is there is a peptide that appears as a single cluster. That AMRT is added to<br>
AMRT list. The AMRT parameters are the same as the cluster parameters.<br>
[00113]	If step 1778 determines that one or more clusters have the same retention<br>
time and mwHPlus as cluster nc, then step 1782 notes that this set of clusters are<br>
an AMRT. Step 1784 assembles these clusters into a single AMRT and obtains<br>
their parameters, and 1786 accumulates the result. That is, if there are multiple<br>
clusters having the same mwHPlus, we infer that a peptide is in the data that has<br>
ions that appear in these different charge and isotope states. The AMRT<br>
parameters are retention time of the most intense cluster, mwHPlus of the most<br>
intense cluster, intensity is the sum of intensities of all clusters, and the fractional<br>
charge state is the sum of charges of clusters weighted by fractional intensity of<br>
each cluster.<br>
[00114]	The retention time requirement applied in step 1778 is determined by a<br>
retention time window and the m/z requirement applied in step 1778 is determined by a ppm window. These parameters are obtained and applied in the same fashion as in the case of the ion pairing determination, described above.<br>
[00115]	When all clusters are looped over, the loop terminates, and all results are<br>
stored. The final result is the AMRT list containing all AMRT parameters and the<br>
ions associated with those AMTS, as well as those ions that were not part of<br>
cluster. It is this final list that is input to the PDS and EDA algorithms.<br>
[00116]	The other input to the PDS and EDA algorithms are sequences of target<br>
peptide precursors and their fragment sequences. Figure 3B is a flowchart for a method of selecting target precursor peptides using a selected database of protein<br>
sequences. In step 310, an appropriate database is selected. Preferably, the database contains protein sequences corresponding to all possible proteins that are present or likely to be present in the sample. In the database, each protein is described by its primary sequence of amino acids. From such a sequence it is possible to predict the peptides that will result from a digestion protocol as well as other properties such as hydrophobicity and charge state. For example, tryptic digestion cleaves sequences at the known amino acids K and R. Based on these cleavage products, the Y- and B-ion fragments and corresponding masses that result from collisional fragmentation can be predicted. Thus, the database provides a model of the masses and other physical attributes that can occur in the low- and high-energy spectra.<br>
[00117]	During protein identification the AMRTs or ions seen in the data are<br>
compared with the masses contained in the database to effect a reliable identification of the peptides present in the acquired LC/MS data. Ideally, all database peptideslhat are in the data are identified without error.<br>
[00118]	In step 312, in-silico digestion is performed on one or more of the protein<br>
sequences in the database to generate precursor peptides in the database. In-silico digestion is a synthetic digestion based on known digestion properties, such as described above. In step 314, the exact mass of the precursor peptides is determined by looking at the amino acid sequences making up the precursor peptides. The exact masses and sequences corresponding to the precursor peptides are stored for subsequent use.<br>
[00119]	Figure 3C is a flowchart for a method for identifying peptides in a mixture<br>
according to an embodiment of the present invention. The method begins with selection of a precursor peptide (target precursor) from the database. Using the selected database peptide, the masses of the Y- and B-ion fragments corresponding to the selected peptide are determined, or obtained from the database. In this manner, a list of masses is assembled. This list of masses includes masses corresponding to each of the Y-ions, B-ions (possibly excluding the lowest mass Y- and B-ions), as well as the unique mass associated with the unfragmented precursor itself.<br>
[00120]	Other precursor masses and fragment masses might be considered, such as<br>
those corresponding to chemical modifications to the precursor peptide. Examples of such modifications are those due to glycocylation or phosphorylation. Other fragment mass might be considered such as fragmentation at peptide bonds other than the Y or B bond.<br>
[00121]	Both the low and the high energy AMRTs from the LC/MS and LC/MSE<br>
data are then searched for each mass on this list. Matching masses, or hits, occurs when a mass from the database (precursor or fragment) lies within a mass search window of a mass measured in the data (at low-energy or at high-energy). All AMRTs that are hit are recorded together with their mass, retention time, and intensity. Matching masses, or hits, are accumulated for each of a plurality of retention time bins. A bin having an accumulation greater than a detection threshold is deemed to be associated with the target precursor.<br>
[00122]	As described below, the method makes critical use of retention time<br>
alignment of the AMRTs seen at low and at high-energy. Also, as described<br>
below, the method can identify AMRTs in the data that are related to, but not<br>
identical to the peptides in the database. Such identification can be made when<br>
there is a significant overlap of masses associated with the peptide fragments from<br>
the database with masses of AMRTs found in the data at substantially identical<br>
retention times.<br>
[00123]	Referring to figure 3C, in step 350 a precursor peptide (e.g., a tryptic<br>
peptide) is selected from the in-silico digestion peptides. This peptide, alternately<br>
referred to as a target sequence or a target precursor, is described by its mass<br>
(mwHPlus) and the masses (mwHPlus) of its Y- and B-ions in step 352. Any<br>
peptide in the database can be selected as the target precursor in step 350.<br>
[00124]	A list of exact masses of the target sequence and its Y- and B-ions is<br>
determined in step 352, In step 354, the masses of the target, precursor sequence and its Y- and B-ions are used to search for all AMRTs in the data in the high- and low-energy lists that have mwHPlus's within a search tolerance (e.g., 20 ppm). In step 356, AMRTs that match a mass in the mass list from the database to within a search tolerance are noted, tagged, or otherwise identified. The search tolerance can be user-specified or automatically determined from the data by known<br>
statistical means. An automatic method for determining the mwHPlus tolerance is<br>
described below.<br>
[00125)	Ideally, low-energy spectra contain only precursor ions. In practice,<br>
precursor ions can fragment in the ion source, so as a result, low-energy spectra can contain fragments ions of precursors. Such ions are referred to as in-source fragments, and generally appear at attenuated intensity.<br>
[00126]	Ideally, high-energy spectra contain only fragment ions. But in practice,<br>
collisional fragmentation of precursor ions may not be complete, so as a result, high-energy spectra can contain precursor ions. Generally such precursor ions appear at intensities in the high-energy mode that are attenuated relative to their intensity in low-energy mode.<br>
[00127]	Thus, a precursor or fragment mass may appear in either the low-energy or<br>
the high-energy data or both. If a mass in the list appears in the low-energy<br>
AMRT data the PDS algorithm notes, tags, or otherwise identifies as appearing in<br>
that data in that mode. If a mass in the list appears in the high-energy AMRT data<br>
the PDS algorithm notes, tags, or otherwise identifies as appearing in that data in<br>
that mode. Thus, the present invention makes use of all ions derived from a<br>
common precursor molecule, regardless of the mode in which such ions are<br>
produced or detected.<br>
[00128]	In step 358 a detection chromatogram is formed. Assuming that a<br>
sequence is present in the data with ions at detectable levels, all such ions will be<br>
tagged in the search carried out in step 356. However, many other ions not<br>
corresponding to the sequence will also be tagged. The detection chromatogram<br>
shows the number of ions (both at low- and high-energy) that are tagged within a<br>
retention time interval, for each retention time, where the vertical signal is the<br>
number of tags observed in the retention time interval. The effect of false<br>
positives tags is to produce baseline noise.<br>
[00129J	According to one embodiment of the present invention, the detection<br>
chromatogram is a simple histogram. The histogram is a series of bins, the center of each bin corresponding to a retention time and the width of the bin corresponding to a retention time interval. The histogram is formed by simple one<br>
up counting for each hit the bin corresponding to the particular retention time interval that contained the retention time of the mass that was hit.<br>
[00130]	According to a second embodiment of the present invention, the detection<br>
chromatogram is derived using accumulated Gaussian-shaped peaks. In the second embodiment of the present invention, each AMRT hit is represented by a Gaussian-shaped peak in the detection chromatogram. Figure 14 is a flow chart for a method of generating the detection chromatogram according to the second embodiment.<br>
[00131]	In step 1402, a detection peak width is established. The detection peak<br>
width is the width of a Gaussian-shaped peak that is added to the detection chromatogram for each hit. The width of the added Gaussian peak (henceforth referred to as the detection Gaussian, or detection Gaussian peak) is set to a specified fraction of the FWHM of a chromatographic peak in the data. According to an embodiment of the present invention, the fraction is 10%. Thus if the FWHM peak width of a typical chromatographic peak is 0.5 minutes, the FWHM of the detection Gaussian is 0.05 minutes.<br>
[00132]	The time range of the detection chromatogram corresponds to the time<br>
range of the separation. If the FWHM peak width of a typical chromatographic<br>
peak is 0.5 minutes, for example, the sample period of the detection<br>
chromatogram is chosen to be about 1% of that width, or 0.005 minutes. In step<br>
1404, the detection chromatogram is initialized, The initial values of all points in<br>
the detection chromatogram are set to zero. The list of AMRTs that are hit (those<br>
found in step 350) are traversed (looped over).<br>
[00133]	In step 1406, detection Gaussian peaks corresponding to hits are added.<br>
This is done by analyzing all low and high-energy AMRTs that were hit. For each<br>
low- and high-energy AMRT that was hit, a single detection Gaussian of unit<br>
height (having width of the detection peak width) is added to the detection<br>
chromatogram at the AMRT's or ion's respective retention time.<br>
[00134]	If two AMRTs having different masses elute at the same time, their<br>
detection Gaussians will sum to a peak having a peak height of 2. If N AMRTs having different masses elute at the same time, their detection Gaussians will sum to a peak having a peak height of N.<br>
[00135]	The width of the detection Gaussian corresponds to the standard error with<br>
which the retention time of a peak is measured. A method for determining the<br>
standard error in the measurement of retention time is described below.<br>
[00136]	Referring back to figure 3, in step 362, the local maxima of the detection<br>
chromatogram are identified. A peptide detection threshold is determined. The peptide detection threshold determines if a peptide has been identified. Methods by which a detection threshold can be determined are specified below. For example, the peptide detection threshold might be chosen to be 4 AMRTs. Thus, a peptide is deemed identified if at least 4 AMRTs are present in the same retention time window. AMRTs detected in both the low-energy or high-energy spectra can contribute to this count.<br>
[00137]	That is, in step 362 a target peptide in the database is determined to be<br>
present in the data if (A) more than the threshold number of AMRTs are found, (B) the relative retention times of the AMRTs are within +/-0.05 min, and (C) the mwHPlus values of the AMRTs all lie with in 20 ppm of fragment molecular weights and precursor molecular weights in the selected peptide database. In an embodiment of the present invention, the detection chromatogram is constructed so that if (A) is true, then those masses that contribute to the local maximum must also satisfy (B) and (C). The PDS algorithm to this point identifies AMRTs that if present, satisfy this condition and thereby indicate that the selected precursor (target precursor) is present.<br>
[00138]	Any and all local maxima above the detection threshold indicate either that<br>
the selected peptide is in the data, or that a peptide is in the data that is closely<br>
related to the selected peptide. As used in this context, the term "closely related<br>
to" means that there is significant sequence correspondence between the database<br>
peptide and the peptide found in the data at the retention time tr. Note that such<br>
detection can be made whether or not an AMRT having the precursor molecular<br>
weight (mwHPlus) is found. Thus, one or more retention times can be found.<br>
[00139]	Figure 15 is a flowchart for a method for identifying sequence<br>
identifications and their retention times that can be employed in step 362 according to an embodiment of the present invention. Upon .completing the method illustrated in Figure 15, the target precursors found in the low-energy,<br>
LC/MS data will have been identified, as well as those precursors found in the low-energy, LC/MS data whose sequence is related to (but not identical with) that of the target precursor.<br>
[00140]	In step 1502 a detection threshold is established. The detection threshold<br>
can be determined from all the local maxima found in the detection chromatogram. Each local maxima in the detection chromatogram has a value. From these values, a median value is obtained. The detection threshold is typically set to about 4 times the median value. This detection threshold corresponds to a maximum number of fragments that would be likely to fall within a detection peak width by chance alone. Typical values for the detection threshold vary between 5 and 10 fragment ions per detection peak width of 0.05 minute.<br>
[00141]	In step 1504, all peaks in the detection chromatogram above the threshold<br>
are recorded. A peak will not be detected in the detection chromatogram if the target peptide (or a peptide with a sequence related to that of the target) is not present in the data. If, on the other hand, the target peptide, or a peptide with a sequence related to that of the target, or both, are present at sufficient concentration, then there can be one or more local maximum values above the detection threshold.<br>
[00142]	In step 1506, the retention time of the detection peak above the threshold is<br>
taken as the retention time of that peptide. The value for that retention time at<br>
which the target peptide (or sequence-related target peptide) was detected is tj.<br>
The height of the detection chromatogram gives the approximate number of ions<br>
detected for the target peptide (or sequence-related target peptide) and the location<br>
of the local maximum in time is the retention time at which the target peptide (or<br>
sequence-related target peptide) eluted from the chromatographic column.<br>
[00143]	Referring back to figure 3, in step 364, the low- and high-energy AMRTs<br>
for each identification are collected. These AMRTs are collected by the following rule: given the value td from the detection chromatogram for the elution time of the peptide, all AMRTs that are on the hit list and are within the detection width, +/-0.05 minutes of td in our example are noted, tagged, or otherwise collected. Thus these collected AMRTs then satisfy two conditions (A) the relative retention<br>
times of the AMRTs are within +/-0.05 min of/,/, and (B) the mwHPlus values of the AMRTs all lie with in 20 ppm of fragment molecular weights and precursor molecular weights in the selected peptide database.<br>
[00144]	The number of AMRTs collected by this rule will be close to, but not<br>
necessarily exactly the same as the height of the detection chromatogram at td.<br>
AMRTs associated with the peptide may have slightly different retention times,<br>
due to measurement error, so the detection Gaussian peaks may not exactly align.<br>
The height of detection peaks in the detection chromatogram that are above the<br>
detection threshold can be other than an integer if the retention times of the<br>
AMRTs do not have the same values. However, the number of AMRTs collected<br>
by the above rule must obviously be an integral value.<br>
[00145]	In step 366, the collected AMRTs are stored. The spectrum of the<br>
collected AMRT can be displayed if desired. In step 368, the search is repeated for the next precursor peptide if desired, by returning to step 350. If the search is not to be repeated, further analysis can be performed in step 370. Such further analysis can be displaying the result, combining the results with results from other injections or quantitating the identified peptide.<br>
[00146]	Combining results from other injections can consist of comparing the<br>
retention times at which the same peptide appears in two or more injections.<br>
Combining results from other injections can consist of comparing the intensities<br>
of the corresponding AMRTs found in two or more injections. These injections<br>
could be replicate injections of the same mixture, or injections of two samples<br>
taken under different conditions.<br>
[00147]	The retention times and intensities from replicate injections can be<br>
compared for consistency to further confirm the correct identity of the peptides. If the injections are from different samples (or conditions), the retention times can be compared for consistency to further confirm the correct identity of the peptides, and the intensities can be compared, or ratioed, to reveal changes in amount of the peptide in the sample between the two conditions.<br>
[00148]	Rules can be applied to the list of peptide identification to infer which<br>
proteins were present in the original sample.<br>
[00149]	Figure 16 is a flowchart for collecting AMRTs and ions associated with<br>
each identified sequence. In step 1602, a retention time window is established.<br>
Generally, the retention time window is set equal to the detection peak width,<br>
which is +/-0.05 min in the example above. In step 1604, all tagged low- and<br>
high-energy AMRTs whose retention times are within the retention time window<br>
threshold centered on the detection retention time td are collected. The detection<br>
peak gives the retention time for the elution of the peptide. The ions whose<br>
retention times are within the retention time window threshold centered on the<br>
detection retention time tj are the ions detected for that peptide. That collection of<br>
ions contains all ions or AMRTs that were hit at low-energy and high-energy.<br>
These ions may or may not contain the mass corresponding to the target precursor.<br>
[00150]	The results are the ions found in the retention time window centered on the<br>
apex of the detection peak. These ions have masses that correspond to the peptide fragment masses and will generally include, but not always, the mass of the target precursor. In step 1606 the results are stored on a storage device. In addition, in step 1606 the results can be displayed to a user.<br>
[00151]	Figure 4A is an exemplary plot showing all AMRTs found at high energy.<br>
Figure 4B is an exemplary plot showing all AMRTs found at low energy. In figures 4A and 4B, the vertical axis is the mwHPlus of the AMRT and the horizontal axis is retention time. Figure 4C is an exemplary detection chromatogram derived from the data shown in figures 4A and 4B showing the number of hits per retention time corresponding to precursor and fragment masses for a given peptide sequence. Figure 4C clearly illustrates a peak in hit occurrence near approximately 78 minutes. This peak contains a precursor peptide found in the data. The distribution observable in Figure 4C indicates the noise background against which the significance of the peak in hit occurrence can be judged.<br>
[00152]	Figure 5A is an exemplary plot showing only hits of AMRTs in the high-<br>
energy plot of Figure 4A corresponding to precursor and fragment masses for a given peptide sequence. Figure 5B is an exemplary plot showing only hits of AMRTs in the low-energy plot of Figure 4B corresponding to precursor and fragment masses for a given peptide sequence. Figure 5C is a histogram plot of<br>
the hits in the high- and low-energy plots of figure 5A and 5B corresponding to precursor and fragment masses for a given peptide sequence. Figure 5C is an exemplary detection chromatogram similar to figure 4C, but adding a threshold 502. Threshold 502 indicates the number of hits required to indicate the presence of a peptide. A clear sequence-related precursor peptide 506 is identified at a retention time of approximately 78 minutes, where more than 40 hits are counted. A possible sequence-related peptide 504 is identified at a retention time of 43 minutes. The hits of low-energy AMRTs to fragment masses are evidence of in-source fragmentation. In-source fragmentation of precursor ions results in the fragment ions seen in low-energy spectra.<br>
[00153]	Peak characteristics can be used to further assist in peptide identification.<br>
One such characteristic is peak shape. All chromatographic peaks related to the<br>
same precursor peptide must have the same peak shape and peak width. However,<br>
chromatographic peaks related to different peptides may not have the same peak<br>
shape and width. Consequently, it is possible for two peptides to elute at the same<br>
chromatographic retention time, but have different peak shapes and/or widths.<br>
Thus, peak shape can be used to reject coincidences that might otherwise lead to<br>
false identifications. This property of peak shape can also be used to reduce the<br>
threshold that is, the number of ions required to align at the same retention time to<br>
indicate the presence of a target peptide. Similarly, peak shape can confirm the<br>
relationship between the AMRTs.<br>
[00154]	Even though all chromatographic peaks related to the same precursor<br>
peptide must intrinsically have the same peak shape and peak width, variations in<br>
such peak shapes or width may be observed to occur due to measurement error.<br>
Another source of variance is interference by other peaks unrelated to the<br>
precursor.<br>
{00155]	As shown in figure 6A-B, there are several times on peaks that can be<br>
compared to determine peak width and peak shape. These include the apex time (retention time), the times of the up slope inflection point and the time of the down slope inflection point. Inflection points can be obtained from the times of the zero crossings of the 2nd derivative of the peak shape. The 2nd derivative can be obtained via a Savitzky-Golay or related polynomial filter. Figure 6A<br>
illustrates an exemplary chromatographic peak 602. Figure 6B illustrates plot the second derivative of chromatographic peak 602. The times of the apex 604 and inflection points 606a and 606b of the 2nd derivative trace are indicated. These times that can be compared to compare peak shape and width. For reference, the points on the peak in the top plot corresponding to the times found in the bottom plot are indicated by the dashed lines.<br>
[00156]	The time difference between the down slope and up slope inflection points<br>
measures the peak width. For a Gaussian chromatographic peak, this width is 2 times the Gaussian standard deviation. The ratios of heights of the peaks at the up and down slope inflection points are an additional measure of peak asymmetry or peak shape. The magnitude of the time difference between the apex time and the up and down slope inflection times are other measures of peak width. The ratios of these times are a measure of peak shape or asymmetry.<br>
[00157]	Taking peak shape into account, additional processing of the detection<br>
chromatogram can be performed. As described above, the local maxima of the detection chromatogram are found. The shapes and widths of the peaks that align during that local maximum are compared. A peak is rejected if its width or shape is an outlier.<br>
[00158]	Figure 7 is a flowchart for a method for comparing peak shape and peak<br>
width according to an embodiment of the present invention. In step 702, the retention times of the peaks are compared. In step 704, the inflection widths of the peaks are compared. The inflection width of a peak is the time between its inflection points. For example, the time between inflection points 606a and 606b in Figure 6A-B is the inflection width of peak 602. In step 706, the magnitude of difference between apex time and up slope inflection time is compared. In step 708, magnitude of difference between apex time and down slope inflection time is compared.<br>
[00159]	In step 710, the times are analyzed to determine if they fall within a time<br>
threshold. In an embodiment of the present invention, the time threshold for each of the comparisons is the detection width of 0.05 minutes. Thus, all of the time comparisons must fall within 0.05 minutes for the peaks to be considered as corresponding to the same peptide. This threshold can be user-specified or<br>
statistically determined. The user-specified or statistically-determined threshold can be an absolute time or a fraction of the peak width.<br>
[00160]	The preferred peak used for comparing peak shapes and sizes is the 12C<br>
monoisotopic peak of a cluster of ions associated with a peptide. The 12C monoisotope is the lowest mass peak where all isotopes are in their most abundant state. Other peaks in the peptide cluster of ions can be used as well. Further, averages of retention times and up and down slope inflection times can be used for the peak shape and width comparisons.<br>
[00161]	In a second embodiment of the present invention, ions, corresponding to<br>
precursors, fragments and their isotopes, are searched instead of, or in addition to,<br>
searching AMRTs. An advantage of using ions is that for low-intensity peptides,<br>
the peptide might appear as a single ion. For example, when using AMRTs, at<br>
least two ions are needed to detect an AMRT and establish its charge state.)<br>
[00162]	The ion-based search is similar in many respects to the AMRT search<br>
described above. Figure 10 is a flowchart for a method for identifying peptides in<br>
a complex mixture using ions according to an embodiment of the present<br>
invention. A summary of the steps in the PDS algorithm using ions follows.<br>
[00163]	In step 1002, low-energy and in high-energy ions are obtained from a<br>
single injection of a peptide mixture. The peptide mixture is generally obtained from a digest of a protein sample. The low- and high-energy data are obtained using a high-voltage/low-voltage switching technique as described above. In step 1004, a database of proteins is selected. In step 1006, a list of peptides corresponding to the proteins in the database is obtained using rules for peptide digestion.<br>
[00164]	In step 1008, a target precursor is selected from the database (e.g., a tryptic<br>
peptide). This peptide is described by its mass (mwHPlus) and the masses (mwHPlus) of its Y- and B-ions. In step 1009, data is searched for masses corresponding to the mass of the selected precursor. Given these masses, in step 1010 all ions in the data in the high- and low-energy lists that have masses within a search tolerance, (e.g., within 20 ppm) are noted. When using ions as the search criteria, the search must be over multiple charge states and isotope numbers. The<br>
charge states are generally restricted from 1 to 3 for high energy fragments. The charge states are generally restricted to 1-6 for low energy.<br>
[00165]	In a preferred embodiment of this ion-based search, the charge states of all<br>
ions obtained from the low-energy spectra are assumed to be Z=2, and the charge states of all ions obtained from the high-energy spectra are assumed to be Z=l. These assignments are made because Z=2 is the most commonly observed peptide charge in low-energy, and Z=l is the most commonly observed peptide charge in high-energy. In addition, the isotope number of all ions are assumed to be N=0, that is all ions are assumed to be in their monoistopic state. These charge state and isotope assignments are needed to determine mwHPlus value for the respective ions. It is these mwHPlus values that are then compared against the masses obtained from the database in step 1009, as described below.<br>
[00166]	The retention time of each ion hit is recorded in step 1012. The output at<br>
this stage is a list of ions from the high and low energy data that are hit by a mass from database peptide.<br>
[00167]	A synthetic, detection chromatogram is generated in step 1016 in as<br>
described above. In an embodiment of the present invention, each ion hit is represented by a detection Gaussian peak, as follows. The time range of the chromatogram corresponds to the time range of the separation. If the FWHM peak width of the chromatogram is 0.5 minutes, for example, in an embodiment of the present invention, the sample period of the detection chromatogram is chosen to be about 1% of that width, or 0.005 minutes. The initial values of all points in<br>
*<br>
the detection chromatogram are set to zero. The list of ions that are hit are<br>
traversed (looped over). For each entry in the list, a Gaussian-shaped peak<br>
(detection Gaussian peak) is added to the detection chromatogram. The width of<br>
the detection Gaussian peak is set to a specified fraction of the FWHM of a<br>
chromatographic peak in the data. In an embodiment of the present invention, the<br>
fraction is 10%. Thus, the FWHM of the detection Gaussian is 0.05 minutes. The<br>
width of the detection Gaussian peak corresponds to the standard error with which<br>
the retention time of a peak is measured.<br>
[00168]	If two ions that have different masses elute at the same time, their<br>
detection Gaussians will sum to a new peak that has a peak height of 2. If N ions<br>
that have different masses elute at the same time, their detection Gaussians will<br>
sum to a new peak that has a peak height of N.<br>
[00169]	Steps 1018-1026 are similar to steps 362-370 of figure 3C described<br>
above. The local maxima of the detection chromatogram are found in step 1018. The threshold is determined. Methods by which a detection threshold can be determined are specified below. A possible value for the threshold is &gt;=4 ions present (obtained by summing AMRTs at low with those at high energy) within the same retention time window.<br>
[00170]	All local maxima above the detection threshold indicate the presence of a<br>
peptide in the data that either is identical to the database peptide or closely related to it. The local maximum determines the retention time of the detection. The retention time that contains a significant number of ions is an indication of that the target peptide is present in the data. Given this retention time, all ions at both low and high energy that lie with in a threshold value of this retention time, and within a threshold value of the mass are selected in step 1020. An additional threshold can be applied to determine the number of ions that meet these requirements. Groups of ions that are above threshold are recorded in step 1022. These groups indicate peptide identifications. In step 1024 if there are additional precursors, the search is then repeated for the next precursor peptide, by returning to step 1008. Further analysis of the results can be performed in step 1026.<br>
[00171]	When generating the detection chromatogram, an additional requirement<br>
may be imposed that the 12C ion be seen for each charge and isotope cluster, for<br>
any related ion to be included as a hit. That is, if the I3C is seen it is not counted<br>
unless the UC in the same charge state is seen.<br>
[00172]	The above described PDS algorithm has a number of advantages over<br>
conventional peptide identification methods. One of the problems with the prior art is that it assumes that AMRTs (or ions) in low-energy are only precursor peptides. However, fragmentation can occur in low-energy (in-source fragmentation) as a part of the ionization and focusing process. Thus, in conventional system, a search can be initiated by an AMRT that is not in fact a precursor. Such a search either will not result is a hit to a target, or a hit to a target<br>
will occur resulting in a spurious, false identifications. Such false identifications<br>
are referred to as false-positives.<br>
[00173]	Using embodiments of the present invention however, as can be seen in<br>
figure 5B for example, in-source fragments that appear at low-energy are detected.<br>
Because embodiments of the present invention detect AMRTs in low- and high-<br>
energy data, all the AMRTs in low-energy that are in fact fragments (not tryptic<br>
precursors) are identified.<br>
[00174]	Another advantage of embodiments of the present invention is that<br>
searches can be performed and identifications of peptide sequences can be made without requiring the detection of the precursor mass. That is, the peptide from the database may have a precursor molecular weight (mwHPlus) of M. In conventional methods, the search is initiated by looking for peptides in the database with molecular weight (mwHPlus) of M. If the database does not contain a peptide with this molecular weight, then conventional systems make no identification.<br>
[00175]	However, the peptide mixture may contain peptides that are related to, but<br>
not identical to, the peptides obtained from the database. For example, a peptide may be present in the sample that is chemically related to a peptide in the database. In this case, while the Y and/or B ions may be present, the precursor mwHPlus may not be present. Using embodiments of the present invention, an over abundance of ions at a common retention time (exceeding the detection threshold) provides evidence of a molecule in the sample that is closely related to the sequence of a target precursor. Examples of processes that may give rise to such a situation are modification to the primary protein sequence, or post-translational modification of the protein, or after digestion it may be that one or the other end of the peptide is modified or clipped.<br>
[00176]	An example of a modified primary protein sequence is a single-nucleotide<br>
polymorphism (SNP), which is a difference in a single base in a DNA sequence. SNPs may occur at a frequency of 1 change per 100 bases. In an organism, a SNP can give rise to a tryptic peptide that differs by a single amino acid from a theoretical sequence derived from a protein database. The substitution of a single amino acid is sufficient to change the mass of a precursor relative to the<br>
theoretical mass of the unmodified sequence. This substitution leaves the rest of the peptide sequence intact. In particular, up to the point of the amino acid substitution, the Y- and B-ion series of the modified peptide is identical to that of the unmodified sequence.<br>
[00177]	Thus, a peptide in the sample mixture might occur with substantially the<br>
same sequence as the peptide in the database with mwHPlus mass M. But<br>
alteration of the sequence or chemical composition of the peptide in the sample<br>
will generally change the precursor of mass. Thus a precursor of mass M will not<br>
be present in the data, but a significant number of Y or B ions of the sequence<br>
corresponding to masses derived from the database are present in the sample data.<br>
The ions corresponding to these sub-sequence ions will appear in the data at<br>
substantially the same retention time. Thus, hit accumulation does not require that<br>
the theoretical mass of the precursor in the database be present in the data.<br>
[00178]	The retention time of the modified sequence will generally be different<br>
than the retention time of the target sequence (if present). The modified and target sequences derive from two distinct peptide molecules, each of which will be retained differently in the chromatographic separation. Thus if the modified and (unmodified) target peptides are both present in the sample, then in the detection chrornatogram for the target sequence, two detection peaks, one for each peptide, will appear. The retention times of these detection peaks will reflect the retention times of the respective molecules.<br>
[00179]	It is worth noting that for the present invention, the mass tolerance of a hit<br>
can reflect the inherent mass accuracy of the data. That is, mass tolerance does not need to be widened to take into account possible sequence modifications that affect precursor mass. In the case of a modified sequence, the present invention, which uses a narrow mass tolerance that reflects the inherent mass accuracy of the data, will reject the theoretical precursor mass. But such a narrow mass tolerance will still allow hits to those Y- and B-ions present in the data that correspond to theoretical Y and B masses obtained from the database.<br>
[00180]	Thus, if sufficient hits accumulate at the retention time of the elution of the<br>
modified peptide, then the modified peptide will be detected because matches to the Y- and B-ions alone can produce sufficient hits to detect the presence of the<br>
modified peptide in the data. The present invention can detect a modified peptide<br>
sequence by using the theoretical fragment masses of the unmodified peptide.<br>
[00181J	Thus, a search can be carried out and a peptide can be found in the sample<br>
data that is related to a peptide in the database. Further more the sample data can provide partial sequence information from the ions that are hit. Thus, a monoisotope of a database peptide may not be present in the LC/MS or LC/MS-E data, but embodiment of the present invention may still identify that the database peptide is present, in possibly modified form, in the sample.<br>
[00182]	Given the detection of a modified peptide, subsequent investigations of the<br>
masses in the data that elute at that detection retention time can reveal the correct sequence and mass of the modified peptide. For example, ab initio sequencing algorithms can be applied to the masses in the data to determine the peptide sequence.<br>
[00183]	Figure 11 shows, at about 87 minutes, a peptide from the database that is<br>
detected to be in the data. The highest value of mwHPlus seen at this retention time is coincident with the mwHPlus of the precursor. Figure 11 also shows an example of a peptide at approximately 49 minutes that appears to have a significant number of hits (greater than would be expected for noise as it exceeds the threshold), but for which there was no precursor in the database. Specifically, at about 49 minutes, about 5 ions are seen at a common retention tune of 49 minutes. No ion is seen with an mwHPlus associated with the peptide. This may be an example of a peptide that is present in the sample and is chemically related to the peptide in the database. Note that the two peptides elute at a different retention times, which indicates a different chemical composition.<br>
[00184]	Thus, as shown by the plots in figure 11, a peptide mixture may contain<br>
peptides that are related to, but not identical with, the peptides obtained from the<br>
database.<br>
[00185]	In summary, embodiments of the present invention provide numerous<br>
advantages over conventional systems. These include searching the data using a database of known peptide masses. Embodiments of the present invention are able to identify peptide fragments in low energy data (the prior art will assume<br>
falsely that each low-energy AMRT is a precursor), without misidentifying low<br>
energy fragment as low energy (tryptic) precursor.<br>
[00186]	Embodiments of the present invention can be configured to perform<br>
archival searches of data. As long as the mass resolution of the MS is sufficiently high, and the retention time resolution is sufficiently high, and data is obtained with the high/low switching protocol, peptides should be able to be identified. The intensity and retention time of these peptides can then be measured from the ions in the data that are hit.<br>
[00187]	Further, embodiments of the present invention enable global searches of<br>
data. For example, high energy data can be archived all over the world and search retrospectively with this algorithm.<br>
[00188]	Embodiments of the present invention are able to detect peptides in data<br>
that share sequences with to peptides in database, even when the shared sequence is not identical or the sharing is not complete. For example, the presence of partial Y-ion and B-ion sequences typically are sufficient to identify an AMRT as being related to a peptide from the database.<br>
[00189]	Further, embodiments of the present invention detect peptides in data that<br>
has the same sequence as peptides in database, but are chemically modified. The presence of partial Y-ion and B-ion sequences are sufficient to identify an AMRT as being related to a peptide from the database.<br>
[00190)	-     Embodiments of the present invention use retention time alignment in<br>
high/low switching MS analysis. Improvements in chromatographic resolution translate directly to improvements in the ability to identify peptides. For example, the detection threshold may be reduced as chromatographic peak width is reduced (resolution increased.) Embodiments of the present invention further use of peak shape and width agreement to tune high/low MS analysis.<br>
[00191]	Embodiments of the present invention can use retention time specificity to<br>
further identify ions that are chemical modifications of precursors, for example, peptides that have lost a neutral water molecule, such as an ammonia molecule. After a peptide is identified as being present in the data, all ions related to that peptide can be identified. As a result, these lower level peptides will not be erroneously identified as precursors or Y or B fragments.<br>
[00192]	Embodiments of the present invention provide measurement of<br>
background noise. As a result, the significance or confidence of identification can<br>
be calculated based upon statistical behavior of data. Rather than histogram the<br>
number of hits in a retention time interval, the detection chromatogram can count<br>
only the longest contiguous sequence of hits within a retention time interval. That<br>
is, if Y2, Y4, Y5, Y6, Y7, Y10 are hit, then only four these ions (Y4, Y5, Y6, Y7)<br>
form a contiguous sequence of Y ions. The detection histogram would contain 4 at<br>
the retention time of the ions. The significance of a detection threshold can be<br>
evaluated by monte-carlo means, or by obtaining the statistical properties of<br>
background hits from the data. Further, embodiments of the present invention can<br>
be configured to use distributions of peptide in database to assess significance of<br>
hits<br>
[00193]	Embodiments of the present invention can also be configured to use<br>
intensity and sequence rules in the database search. For example, if a candidate precursor AMRT is seen in both high and low-energy spectra, the intensity of the precursor at high-energy spectra can be compared with the intensity of the precursor seen in the low-energy spectra. For the AMRT to be a precursor, its intensity at high-energy must be less than that at low-energy. If the intensity at high-energy is measured to be greater than that at low-energy, it is likely the AMRT seen in low- and high-energy are in fact fragments of yet another precursor. In this case, the PDS algorithm can be configured to reject the low-energy AMRT as a possible precursor, thereby removing possible false-positive identification.<br>
(00194]	As an example of another rule, hits to a given precursor may be eliminated<br>
if the intensity of a fragment AMRT is judged to be an outlier with respect to the<br>
intensities of other fragment AJRMTs possibly associated with that precursor.<br>
[00195]	As an example of another rule, it may be that for certain precursor<br>
sequences, the amino acid composition indicates that certain Y or B ions of that precursor should ionize efficiently. If the relative or absolute intensities of fragment AMRTs are inconsistent with such a model of ionization efficiency, than that inconsistency may provide grounds to reject the identification of the sequence. On the other hand, if the relative or absolute intensities of fragment<br>
AMRTs are consistent with such models of ionization efficiency, than that may<br>
confirm the identification of the sequence.<br>
[00196]	A problem with analyzing AMRTs is their dynamic range of intensities.<br>
AMRTs in a set of data can occur with intensities that span a large dynamic range.<br>
The dynamic range of intensities can be more than 1:1000. Advances in MS<br>
technology may extend this dynamic range to 1:10,000 or more. The large<br>
dynamic range arises from two effects. First, a given protein in the sample may<br>
produce AMRTs whose dynamic range may be 1:100 or more, as variations in the<br>
composition of peptides give rise to variations in ionization efficiency. As a<br>
result, some AMRTs may ionize more efficiently than others. Second, different<br>
proteins can occur in a sample with large dynamic range of concentration.<br>
[00197]	Thus, low-intensity AMRTs may be generated from multiple sources. For<br>
example, low intensity AMRTs can be generated from high concentration proteins<br>
whose peptides have ionize poorly or from low concentration proteins whose<br>
peptides ionize efficiently.<br>
[00198]	Thus, the large dynamic range in intensity is a potential source of false<br>
positive identifications. AMRTs or ions from a high concentration protein that are<br>
poorly ionizing might be falsely interpreted as arising from a highly ionizing<br>
peptide from a low concentration protein.<br>
[00199]	A method is disclosed for handling the complexity caused by dynamic<br>
range. The technique, which has been termed the Electronic Depletion Algorithm (EDA) identifies and removes all ions associated with high concentration proteins, before the low-concentration proteins are analyzed. Removing AMRTs associated with high concentration proteins removes all the high intensities AMRTs in the sample as well as all the low intensity ions in the sample that come from the high concentration proteins. Consequently, low-intensity AMRTs from high concentration proteins will not be confused with AMRTs from low-concentration proteins that ionize efficiently.<br>
[00200]	Removing the low intensity AMRTs and ions associated with high<br>
concentration proteins reduces an important source of false positives. For example variations in a low-intensity AMRT may erroneously be interpreted as<br>
evidence of a biomarker of a low-concentration protein; where, in fact, it is a<br>
misidentified, poorly ionizing fragment of a high-concentration protein.<br>
[00201]	Figure 12 is a flow chart for a method for performing EDA according to an<br>
embodiment of the present invention. In step 1202, all AMRTs in the data are identified using the PDS described above. The peptides corresponding to the AMRTs are identified in step 1203. The identified AMRTs are sorted in intensity from most intense to least intense in step 1204 and stored in a list.<br>
[00202]	In step 1206, the most intense peptide remaining in the list (the top item in<br>
the list) is selected. Using the PDS, each precursor, Y- and B-ion associated with that peptide is labeled with the peptide in step 1208. In step 1210 each neutral-loss AMRT associated with the precursors, Y- and B-ions labeled in step 1208 is labeled with the peptide. In step 1212, all labeled AMRTs are removed from the data. If there are no AMRTs having intensity higher than a pre-determined intensity threshold as determined in step 1214, the method ends in step 1216. If, on the other hand, there exists another AMRT having intensity higher than the intensity threshold, processing continues in step 1206 using the remaining peptide with the highest intensity.<br>
[00203]	Embodiments of the present invention can be configured to measure<br>
standard error in retention time and in mwHPlus. The measurement error in<br>
retention time that is relevant to this method is the error of the AMRTs and ions<br>
that are common to a single peptide. Thus the error is not an elution error. The<br>
error is due only to the limitations with which we can measure retention time.<br>
[00204]	The measurement error in mwHPlus that is relevant to embodiments of the<br>
present invention is the error relative to a database of accurately mass-measured proteins and peptides. The error is due to the limitations with which m/z can be measured in a mass spectrometer. This error in m/z has two sources, which are statistical noise and calibration error.<br>
[00205]	Retention time and mwHPlus error must be estimated because these errors<br>
are used to set the thresholds used in determining which AMRTs constitute a hit and possibly in configuring the detection chromatogram in the PDS method described above.<br>
[00206]	Retention time error and mwHPlus error are measured as a standard<br>
deviation of an error distribution. Given the standard deviations, the thresholds<br>
used in determined which AMRTs constitute a hit and which are have a<br>
significant number of hits in the detection chromatogram are then some multiple<br>
of the standard deviation. Typical values might be 3 for a false-positive rate of 3<br>
sigma or 1 in 1000, or 6 sigma, specifying a formal false positive rate of 1 in 106.<br>
[00207]	According to one embodiment of the present invention, to determine<br>
retention time error and mwHPlus error, ions that are common in low and in high energy are identified. For example, a tryptic precursor that appears in low energy will also appear in high energy, albeit at reduced intensity. The nominal difference in retention time between the ions is perforce zero. The nominal difference in mwHPlus is zero.<br>
[00208]	The observed retention time difference between these ions is a measure of<br>
the error in retention time. The standard deviation of this error, as determined by combining errors from many such pairs, is the basis of the measurement of the standard error in retention time.<br>
[00209]	Based on the foregoing, figure 13 is a method for determining retention<br>
time error and mwHPlus error according to an embodiment of the present invention. In step 1302, loop through all AMRTS (or ions) in low energy. Using a large threshold in mass and retention time (e.g., 50 ppm and 0.5 minutes, respectively) find all AMRTs (or ions) in high energy that match in step 1304. In step 1306 the errors between matches in retention time and mwHPlus are analyzed. In step 1308, outliers are removed using standard techniques such as median filtering. In step 1310, the standard deviations for the resulting distributions are computed. In step 1312, the standard deviation in mwHPlus is multiplied by a statistical factor (between 3 and 6) to establish a molecular weight threshold that can be used in determining which AMRTs hit in the PDS algorithm. In step 1314, the standard deviation in retention time is multiplied by a statistical factor (between 3 and 6) to establish a retention time threshold that can be used in step generating the detection chromatogram of the PDS algorithm<br>
[00210]	The detection threshold used for determining whether a peptide is in the<br>
data or not can be determined in the following manner. A peptide in the database<br>
is considered detected in the data if there are more than N AMRTs (or ions) found. The number N depends on the complexity of the sample. The more complex the sample the higher must be N. N can be empirically determined from the data by examining the background of hits. A standard histogram technique or other statistical technique can be employed to establish N. In ractice, a value of N between 4 and 6, inclusive has been found to be an acceptable detection threshold for AMRTs found in high/low data.<br>
[00211]	The results obtained from the invention are lists of peptides identified in a<br>
sample. Each such peptide in the list contains the peptide sequence from the<br>
database, and the measured retention time, measured and theoretical mass of the<br>
precursor, the measured intensity of the precursor, as well as the measured<br>
retention times, intensities and masses of the fragment ions found in the data and<br>
associated with the precursor. It is anticipated that these results will have great<br>
utility in the field of proteomics. For example, four such proteomic applications<br>
are: identification of proteins in a sample, retention time tracking of peptides<br>
between samples, and quantitation of peptides and proteins between samples.<br>
[00212]	A peptide that is identified by this method gives evidence that the<br>
originating protein sequence occurred in the sample. For example, the list of<br>
proteins corresponding to the tryptic peptides identified by the present invention is<br>
one method for identifying proteins present in the sample. From the identity and<br>
concentration of the observed peptides, algorithms known in the art can also infer<br>
the identity and the concentration of the parent proteins.<br>
[00213]	The entrants on this list may contain false positives. That is, false<br>
identifications of peptides may give rise to false identification of proteins. The reduction or elimination of false positives can be accomplished by one of several means known in the prior art. One can require that for a protein to be identified more than one peptide to that protein be identified by the method of the current invention. The user may specify that N&gt;1 such peptides be detected. Alternatively, the user may specify that a minimum percentage coverage (by amino acid) of protein sequence be achieved by the detection of 1 or more peptides to that protein. In addition, a user may require that the protein be identified in each of several replicate LC/MS analyses of the sample. Other<br>
possible rules, known in the art, may be applied to reduce false positive identification of proteins.<br>
[00214]	The present invention can be applied to the data obtained multiple samples<br>
analyzed on the same or different LC/MS systems. If a peptide sequence from a<br>
database is seen in more than one such sample or analysis, then the retention times<br>
of that peptide from each sample or analysis can be compared. Thus the invention<br>
provides a means of tracking the retention time of a peptide from injection to<br>
injections. In addition, the peptide could be detected in data obtained by different<br>
instruments employing different LC separation methodologies. Thus the retention time of that peptide could be compared between different such instruments and<br>
LC separations.<br>
[00215]	The consistency of retention times of the same sequence as seen in<br>
multiple injections on the same or different instruments can be checked. Such consistency checking can be used to detect and to eliminate false positive identifications.<br>
[00216]	Given the intensities of the same peptide as seen in replicate injections of a<br>
given mixture, or as seen in injections of a sample under different conditions, methods of the prior art can be applied to calibrate intensities and to determine expression changes in peptide and proteins. For example, if the same peptide sequence is detected between injections of two samples, the ratios of intensities of the corresponding precursors could be computed. If these peptides are from a calibration standard protein, then the ratio allows for the relative or absolute concentration calibration of the respective injections. If the peptides are from proteins endogenous to the sample, then the ratio can be used to determine the expression level change of the peptide or to the originating protein.<br>
[00217]	The method of the current invention can be applied to mixtures other than<br>
that of peptides. Given (1) any mixture of molecules and (2) a database containing the masses of those molecules and their fragments, the method can be used to identify the molecules in the sample.<br>
[00218]	For the method to be applied, the sample is be analyzed by the LC/MS<br>
systems described about. The precursor molecules fragment by the methods described above, and the theoretical masses of the precursor and fragments are be<br>
known. If these conditions are met, then the precursors can be identified by the<br>
method of the present invention. In the above discussion of the method, the<br>
precursor then refers to mass of the ion any seen at low-energy. The fragment<br>
then refers to the mass of any fragment of the precursor seen in high-energy, or<br>
possibly in low-energy. The method then identifies the originating molecule that<br>
is separated in the column.<br>
[00219]	Thus, for example, metabolic studies can benefit from the present<br>
invention. The step of digestion is not needed for molecules of metabolism. All<br>
that is needed for the present invention are a list of exact, theoretical masses<br>
corresponding to the precursor and its associated fragments. With such a list, the<br>
method can detect the presence of that set or subset of masses and the retention<br>
time at which the originating, precursor molecule eluted from the<br>
chromatographic column.<br>
[00220]	In a preferred embodiment, the PDS algorithm and EDA algorithm is<br>
applied to data acquired by spectra collected in alternating high-energy and low-energy odes. owever both of these algorithms can be applied to spectra<br>
collected in only one mode, i.e., a fixed-energy mode. Thus, for example, these<br>
algorithms can be applied either to the low-energy spectra or to the high-energy<br>
spectra alone. That is, in principle, both modes could be collected, but peptides<br>
can be identified via the use of the PDS or EDA algorithms applied only to a<br>
single mode. As long as some fragmentation of a precursor ion occurs within a<br>
single-energy mode, then the application of these algorithms will detect the<br>
presence of the precursor and/or its fragments in the data. The in-source<br>
fragmentation evident in Figures 5C and 11C shows that the PDS or EDA<br>
algorithms could be applied to low-energy data alone.<br>
[00221]	Thus, the requirement that data be collected in two modes is preferred,<br>
though not necessary for the application of the PDS or EDA algorithms.<br>
[00222]	It should also be noted that the PDS or EDA algorithms can be applied to<br>
spectral data intentionally acquired only at a single, fixed-energy. In fact it may be advantageous to adjust the voltage (or voltage steps) in a fixed-energy mode to correspond to a voltage that is intermediate in value between that typically used for low-energy or high-energy acquisition. The intent would be to collect spectra<br>
that contain an optimum mix of precursors and fragments. Such acquisition<br>
would also lend itself to the identification of peptides via use of the PDS or EDA<br>
algorithms.<br>
[00223]	The foregoing disclosure of the preferred embodiments of the present<br>
invention has been presented for purposes of illustration and description. It is not intended to be exhaustive or to limit the invention to the precise forms disclosed. Many variations and modifications of the embodiments described herein will be apparent to one of ordinary skill in the art in light of the above disclosure. The scope of the invention is to be defined only by the claims appended hereto, and by their equivalents.<br>
[00224]	Further, in describing representative embodiments of the present invention,<br>
the specification may have presented the method and/or process of the present invention as a particular sequence of steps. However, to the extent that the method or process does not rely on the particular order of steps set forth herein, the method or process should not be limited to the particular sequence of steps described. As one of ordinary skill in the art would appreciate, other sequences of steps may be possible. Therefore, the particular order of the steps set forth in the specification should not be construed as limitations on the claims. In addition, the claims directed to the method and/or process of the present invention should not be limited to the performance of their steps in the order written, and one skilled in the art can readily appreciate that the sequences may be varied and still remain within he spirit and scope of the present invention.<br><br>
WHAT IS CLAIMED IS:<br>
1. A method for identifying proteins in a mixture, comprising: digesting the mixture to obtain a peptide mixture; applying the digested mixture to an LC/MS system;<br>
applying a low-energy mode in the mass spectrometer portion of the LC/MS system;<br>
applying a high-energy mode in the mass spectrometer portion of the LC/MS system to fragment the data;<br>
obtaining mass information and retention time information from the peptide precursor data acquired during the low-energy mode;<br>
obtaining mass information and retention time information from the fragment data acquired during the high-energy mode;<br>
selecting a target precursor from a database of proteins; determining a mass associated with the target precursor; determining a mass associated with each Y- and B-ion corresponding to the target precursor;<br>
comparing the mass associated with the target precursor, and the masses associated with the Y- and B-ions to the mass information related to the peptide precursor data and fragment data;<br>
detecting mass matches resulting from the comparisons; determining retention times of mass matches; and<br>
identifying whether the target peptide is present in the mixture based on the detected mass matches and retention times of detected mass matches.<br>
2.	The method recited in claim 1, further comprising switching between the high- and<br>
low-energy modes according to a protocol that alternates application of the high- and low-<br>
energy modes with sufficient frequency such that each of the high- and low-energy modes is<br>
applied a plurality of times during a chromatographic peak width.<br>
3.	The method recited in claim 1, further comprising creating a detection<br>
chromatogram.<br>
4.	The method recited in claim 3, further comprising adding detection Gaussian peaks<br>
corresponding to mass matches to respective retention time bins to create the detection<br>
chromatogram.<br>
5.	The method recited in claim 1, further comprising:<br>
determining a detection threshold; and<br>
identifying whether a peptide is present based on whether a count of mass matches in a particular retention interval exceeds the detection threshold.<br>
6.	The method recited in claim 5, further comprising:<br>
estimating a noise floor associated with the detection threshold; and analyzing a significance of the peptide identification based on the count and the estimate of the noise floor.<br>
7.	The method recited in claim 1, wherein the low energy mode is applied by<br>
applying a low voltage in a collision cell of the mass spectrometer portion of the LC/MS<br>
system, and the high energy mode is applied by applying a high voltage in the collision cell<br>
of the mass spectrometer portion of the LC/MS system.<br>
8.	The method recited in claim 1, further comprising selecting as the database a<br>
specialized database having proteins likely to be found in the mixture.<br>
9.	The method recited in claim 1, further comprising using peak characteristics to<br>
determine whether a particular chromatographic peak is associated with a target precursor.<br><br>
10.	The method recited in claim 9, wherein the peak characteristics is peak shape.<br>
11.	The method recited in claim 10, where in the peak shape includes analysis of a<br>
peak's apex, up-slope inflection point and down-slope inflection point.<br>
12.	A system for identifying proteins in a mixture, comprising:<br>
means for digesting the mixture to obtain a peptide mixture; a liquid chromatograph to which the mixture is applied to separate the mixture;<br>
a mass spectrometer to measure masses in the separated mixture output by the liquid chromatograph, the mass spectrometer, comprising: a collision cell to fragment the peptides;<br>
a mass analyzer to determine mass information related to fragmented and unfragmented peptides;<br>
a database of proteins from which a target precursor is selected; and an analysis computer, coupled to the database and an output of the mass spectrometer, having software executing thereon, for causing the computer to: determine a mass associated with the target precursor peptide;<br>
determine a mass associated with each Y- and B-ion corresponding to the target precursor peptide;<br>
compare the mass associated with the target precursor peptide, and Y-and B-ions to the mass information related to the peptide data and fragment data;<br>
detect mass matches resulting from the comparisons;<br>
determining retention tunes of mass matches; and<br>
identify whether the target peptide is present in the mixture based on the detected mass matches and retention times of detected mass matches.<br>
13.	The system recited in claim 12, wherein a high voltage is applied in the collision<br>
cell to fragment the data, and a low voltage is applied to provide unfragmented data.<br>
14.	The system recited in claim 12, in which the collision cell is configured to switch<br>
between the high- and low-energy mode according to a protocol that alternates application of<br>
the high- and low-energy modes with sufficient frequency such that each of the high- and<br>
low-energy modes is applied a plurality of times during a chromatographic peak width.<br>
15.	The system recited in claim 12, wherein the computer software further causes the<br>
computer to generate a detection chromatogram.<br>
16.	The system recited in claim 15, wherein the computer software further causes the<br>
computer to add detection Gaussian peaks corresponding to mass matches to respective<br>
retention time bins to create the detection chromatogram.<br>
17.	The system recited in claim 12, wherein the computer software further causes the<br>
computer to:<br>
determine a detection threshold; and<br>
identify whether a peptide is present based on whether a count of mass matches in a particular retention interval exceeds the detection threshold.<br>
18.	The system recited in claim 17, wherein the computer software further causes the<br>
computer to:<br>
estimate a noise floor associated with the detection threshold; and analyze a significance of the peptide identification based on the count and the estimate of the noise floor.<br>
19.	The method recited in claim 12, wherein the low energy mode is applied by<br>
applying a low voltage in the collision cell of the mass spectrometer portion of the LC/MS<br>
system, and the high energy mode is applied by applying a high voltage in the collision cell of the mass spectrometer portion of the LC/MS system.<br>
20.	The system recited in claim 12, wherein the database is a specialized database<br>
having proteins likely to be found in the mixture.<br>
21.	The system recited in claim 12, wherein the computer software causes the<br>
computer to use peak characteristics to determine whether a particular chromatographic peak<br>
is associated with a target precursor.<br>
22.	The system recited in claim 21, wherein the computer software causes the<br>
computer to use the peak shape in determining whether a particular chromatographic peak is<br>
associated with a target precursor.<br>
23.	The system recited in claim 22, wherein the computer software causes the<br>
computer to analyze a peak's apex, up-slope inflection point and down-slope inflection point.<br>
24.	A method for identifying precursors in a mixture, comprising:<br>
applying the mixture to an LC/MS system having a liquid chromatographic portion and a mass spectrometer portion;<br>
applying a low-energy mode in the mass spectrometer portion of the LC/MS system;<br>
applying a high-energy mode in the mass spectrometer portion of the LC/MS system to fragment the data;<br>
obtaining mass information and retention time information from the precursor data acquired during the low-energy mode;<br>
obtaining mass information and retention time information from the fragment data acquired during the high-energy mode;<br>
selecting a target precursor from a database;<br>
determining a mass associated with the target precursor;<br>
determining masses associated with fragment ions corresponding to the target precursor;<br>
comparing the mass associated with the target precursor and the masses associated with the fragment ions to the mass information related to the precursor data and fragment data;<br>
detecting mass matches resulting from the comparisons;<br>
determining retention times of mass matches; and<br>
identifying whether the target precursor is present in the mixture based on the detected mass matches and retention times of detected mass matches.<br>
25. A method for identifying precursors in a mixture, comprising:<br>
applying the mixture to an LC/MS system having a liquid chromatograph portion and a mass spectrometer portion;<br>
applying a fixed-energy mode in the mass spectrometer portion of the LC/MS system;<br>
obtaining mass information and retention time information from the precursor and fragment data acquired during the fixed-energy mode; selecting a target precursor from a database; determining a mass associated with the target precursor; determining a mass associated with fragment ions corresponding to the target precursor;<br>
comparing the mass associated with the target precursor and the masses associated with the fragment ions to the mass information related to the precursor data and fragment data;<br>
detecting mass matches resulting from the comparisons; determining retention times of mass matches; and<br>
identifying whether the target precursor is present in the mixture based on the detected mass matches and retention times of detected mass matches.<br></td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/viewdoc.aspx?id=C/yNl4WaRnE6NW4kDv1MtQ==&amp;amp;loc=+mN2fYxnTC4l0fUd8W4CAA==" target="_blank" style="word-wrap:break-word;">http://ipindiaonline.gov.in/patentsearch/GrantedSearch/viewdoc.aspx?id=C/yNl4WaRnE6NW4kDv1MtQ==&amp;amp;loc=+mN2fYxnTC4l0fUd8W4CAA==</a></p>
		<br>
		<div class="pull-left">
			<a href="268228-an-electrical-induction-device-for-high-voltage-applications.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="268230-high-speed-frequency-and-phase-estimation-for-flow-meters.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>268229</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>6903/DELNP/2006</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>35/2015</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>28-Aug-2015</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>21-Aug-2015</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>20-Nov-2006</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>WATERS TECHNOLOGIES CORPORATION</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td>34, MAPLE STREET, MILFORD, MA, 01757 USA,</td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>SCOTT J. GEROMANOS</td>
											<td>2 CREST ROAD, MIDDLETOWN, NJ 07748, USA</td>
										</tr>
										<tr>
											<td>2</td>
											<td>MARC VICTOR GORENSTEIN</td>
											<td>313 BROOKLINE STREET, NEEDHAM, MA 02492, USA</td>
										</tr>
										<tr>
											<td>3</td>
											<td>JEFFREY CRUZ SILVA</td>
											<td>95 HALE STREET, BEVERLY, MA 01915, USA,</td>
										</tr>
										<tr>
											<td>4</td>
											<td>GUO-ZHONG LI</td>
											<td>14 MORSE STREET, WESTBOROUGH, MA 01581, USA</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>H04L13/18</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>PCT/US2005/017742</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td>2005-05-20</td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td>60/572,532</td>
									<td>2004-05-20</td>
								    <td>U.S.A.</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/268229-a-method-and-system-for-identifying-proteins-in-mixtures by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 06:46:40 GMT -->
</html>
