<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/268439-a-method-of-the-biocatalytical-decarboxylation-of-a-compound by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 07:18:41 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 268439:A METHOD OF THE BIOCATALYTICAL DECARBOXYLATION OF A COMPOUND</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">A METHOD OF THE BIOCATALYTICAL DECARBOXYLATION OF A COMPOUND</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>The present invention relates to a method for the enzymatic decarboxylation of malonic acid (propanedioic acid) derivatives catalyzed by enzymes structurally and/or functionally related to arylmalonate decarboxylase (AMDase) as isolated from microorganisms of the genus Bordetella. The present invention also relates to novel enzymes with a decarboxylase activity, useful for performing the claimed method, mutants thereof, corresponding coding sequences and expression systems, methods of preparing said novel enzymes, and screening methods for obtaining further suitable enzymes also having said decarboxylase activity.</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td>New Malonate Decarboxylases for Industrial Applications<br><br>
The present invention relates to a method for the enzymatic decarboxylation of alpha-substituted malonic acid (propanedioic acid) derivatives catalyzed by enzymes structurally and/or functionally related to arylmalonate decarboxylase (AMDase) as iso¬lated from microorganisms of the genus Bordetella, in particular Bordetella bronchisep-tica. The present invention also relates to novel enzymes with decarboxylase activity, useful for performing the claimed method, mutants thereof, corresponding coding se¬quences and expression systems, methods of preparing said novel enzymes, and screening methods for obtaining further suitable enzymes also having said decarboxylase activity.<br>
Background of the invention<br>
Decarboxylation of malonyl substrates is a common enzymatic transformation that occurs in a number of important biological pathways. For example, numerous aerobic and anaerobic bacteria can grow on malonate as the sole carbon source, be¬cause they possess malonate decarboxylase systems that transform malonate or malonyl-CoA to acetate.1 Most of these archetypical malonate decarboxylases are mul-tienzyme systems involving acyl carrier proteins (ACPs) with unusual prosthetic groups as well as biotin dependent decarboxylases.2 Decarboxylation of malonyl units also oc¬curs during the Claisen-type condensation reaction catalysed by (3-ketoacyl ACP syn¬thase (KAS) enzyme and domains within fatty acid and polyketide synthases.<br>
An arylmalonate decarboxylase (AMDase) was isolated from the gram-negative bacterium, Alcaligenes bronchisepticus (Bordetella bronchiseptica),3,4 which catalyzes the enantioselective decarboxylation of a-aryl-a-methylmalonates to give optically pure a-arylpropionates. Whilst the biological role of the enzyme has not been established, it has been shown to be a robust catalyst decarboxylating a range of substrates includ¬ing: phenylmalonic acid 1; 2-methyl-2-phenylmalonic acid 2; 2-methyl-2-naphthyl malo¬nic acid 3 and 2-thienylmalonic acid 4 (see Fig.1 (a)).5 The AMDase is unusual amongst malonate decarboxylases6 as it does not require biotin or any other co-factors for activity. In addition, decarboxylation does not involve formation of a malonyl thio-ester-enzyme intermediate7 which is a common driving force for several malonate de¬carboxylases.1,2 The mechanism of the enzyme is thus of considerable interest. The chiral carboxylic acid products of the AMDase are also potentially attractive for indus¬trial applications, particularly as precursors for pharmaceutical synthesis.<br><br>
2<br>
AMDase from B. bronchiseptica (SEQ ID NO:2) shows sequence similarity with a number of racemases and isomerases.7,8 Moreover the putative catalytic cysteine is preserved in all enzymes as shown by sequences alignment. However no AMDase ac-tivty was reported for those enzyme and AMDase from B. bronchiseptica (SEQ ID NO:2) remained the only enzyme capable to catalyze the decarboxylation of a-arylmalonates to a-arylpropionates.<br>
In the case of the glutamate and aspartate racemases two cysteine residues are located on either side of the amino acid substrate in the active site. Indeed, it was dis¬cussed that the two cysteine residues function in generai acid-base cataiysis, abstract¬ing the a-proton from the substrate to generate a planar enolate which is re-protonated from the opposite face leading to the racemate.9,10 In the case of the AMDase, a single active site Cys residue (Cys188) has been shown to play a crucial role in enzyme ca¬talysis. This led to the proposal that the mechanism of the AMDase is similar to the ra¬cemases. For example, decarboxylation of the substrate 2-methyl-2-phenylmalonate is suggested to result in an enolate anion, which is protonated on the s/'-face by Cys188 to form a R-configured a-phenylpropionate product.8 To support the mechanistic simi¬larity with the racemase family, the introduction of second cysteine, guided by se¬quence alignment,8 resulted in a AMDase single point mutant (G74C), which was ca¬pable of racemising homochiral a-arylpropionates, albeit with very low catalytic effi¬ciencies.7 In addition the double mutation G74C/C188S, which was predicted to re¬position the key active site Cys on the opposite face of the enolate intermediate, led to an enzyme with opposite enantioselectivity. Once again the catalytic efficiency of this mutant was several orders of magnitude lower than the wild type AMDase.11<br>
WO 2005/07811 provides specific examples of mutants of and AMDase enzyme isolated from B. bronchiseptica KU1201 carrying up to two mutated amino acid resi¬dues. The following mutants with altered enzyme activity are suggested: A84G, F85A, A87G, R94A, T103A, I127A, (F85A, R173A), (F85A, E176A) and (F85A, A178G); The following mutants with improved thermostability are suggested: P15A, P32A, G74A, T75A, D128A, D163A, A165G und C171A. Generically, mutations in at least one of the positions 17, 19, 22, 24, 25, 32, 41, 42, 46, 47, 53, 60, 61, 63, 68, 74, 83, 84, 85, 87, 94, 103, 105, 112, 116, 119, 121, 139, 142, 155, 168, 171, 173, 178, 199, 201, 202, 203, 204, 205, 210, 221, 222, 224, 225, 226, 227, 228, 229, 230, 235, 238, 239 and 240 are mentioned. However, said document neither discloses the entire crystal struc¬ture of said mutants or of the parent AMDase or the structural organization of the cata-<br><br>
3<br>
lytic site of such enzymes, nor does said document teach or suggest that AMDase ac¬tivity might be associated with proteins of different origin. The document speculates about further "product structures" which might be accessible via enzyme mutants dis¬closed therein. However, enzyme activities of a limited number of mutants were merely reported for phenyl malonic acid and alpha-hydroxy-(4-methylphenyl) malonic acid. No hydrocarbyl-substituted, in particular no alkenyl-substituted malonic acid derivatives have been suggested or even tested.<br>
Further enzymes with AMDase activity, so far, have not been described in the art.<br>
There is therefore a need for alternative, optionally improved enzymes with AM¬Dase activity, which potentially are applicable in industrial processes for preparing preferably optically pure decarboxylation products of malonic acid derivatives, as for example a-arylpropionates, and which do not require biotin or any other co-factors for decarboxylase activity.<br>
There is also a need for identifying novel malonate decarboxylase enzyme with extended or altered/modified substrate specificity, i.e. enzymes decarboxylating malo¬nate derivatives different from alpha-aryl malonates, which potentially are applicable in industrial processes for preparing preferably optically pure alpha-substituted mono-carboxylic acids from corresponding malonate substrates.<br>
There is also a need for functional mutants derived from such enzymes with AM¬Dase activity or enzymes with extended or altered/modified substrate specificity, which potentially are applicable in industrial processes for preparing preferably optically pure mono-carboxylic acids from malonate substrates. Said functional mutants may be adapted to the preferred or exclusive decarboxylation of malonate substrates different from aryl malonates, as for example non-aromatic alpha-substituted malonate deriva¬tives, like alkenyl-substituted malonates.<br>
It is therefore an object of the present invention to provide such alternative or op¬tionally improved enzymes with malonate decarboxylase activity (MDase activity) as mentioned above.<br>
Summary of the invention:<br>
The above-mentioned problems could, surprisingly, be solved by identifying fur¬ther enzymes for which an AMDase activity or MDase activity so far has not been de¬scribed.<br><br>
4<br>
In particular, the present inventors identified a crystal structure of the AMDase from B. bronchiseptica (SEQ ID NO:2), and performed 180 labelling studies which allow the detailed molecular mechanism of this interesting decarboxylase to be demon¬strated. Based on this the present inventors were able to identify further industrially suitable AMDase enzymes. For example, an enzyme from Mesorhizobium sp. BNC1 (SEQ ID NO:4) was identified, which is shown to have a structure, stereochemistry, mechanism and catalytic efficiencies similar to AMDase from B. bronchiseptica. The present inventors also observed extended or modified substrate specificity and/or en¬zyme acitivity (if compared to the parent enzyme) for such enzymes, allowing the bio-catalytic conversion of further alpha-substituted malonate derivatives. Based on said studies the present inventors were also able to provide functional engineered mutants of said enzymes.<br>
Description of the figures:<br>
Figure 1: (a) Decarboxylation reaction catalysed by AMDase; (b) proposed mechanism of decarboxylation reaction catalysed by AMDase from B. bronchiseptica.<br>
Figure 2: Sequences of purified enzymes with C-terminal His6-tag as determined after ESI-MS analysis<br>
Figure 3. (a) Sequence alignment of AMDase from 6. bronchiseptica (upper line) and examples of structurally and functionally related enzymes from different microor¬ganisms: Mesorhizobium sp. BNCI, Bordetella bronchiseptica RB50, Pyrcoccus horiko-shii OT3 and Alkaliphilus metalliredigenes QYMF<br>
(b) alignment of functionally related sequence regions of AMDase from B. bron¬chiseptica (upper line) and examples of structurally and functionally related enzymes from different microorganisms.<br>
Figure 4: (a) Cartoon representation of AMDase crystal structure with highlighted: phosphate and Cys188. (b) AMDase active site with bound phosphate and hydrogen bonding pattern observed to CM, 02and 03 of phosphate.<br>
Figure 5: Active site of AMDase with residues Lys40, Val43, Tyr48, Val156, Pro14, Met159 associated in building the smaller binding pocket (lighter colour) sur¬rounding methyl group of intermediate enolate bound to the oxy-anion hole residues Ser76, Thre75, Gly189 (darker colour) and in proximity to the catalytic site Cys188.<br>
Figure 6: (a) Cross sections of the B. bronchiseptica AMDase active site with sol¬vent accessible surface showing a model of 2-methyl-2-phenylmalonate situated in the<br><br>
5<br>
binding pockets. Residues building oxy-anion hole are darker and residues building hydrophobic pockets lighter, (b) Active site of Mesorhizobium sp. BNCI AMDase.<br>
Figure 7: Orientation of 2-methyl-2-phenylmalonate (left) and the enolate (right) in the active site of AMDase.<br>
Figure 8: Determination of AMDase stereochemistry using 180 labelled water.<br>
Figure 9: In vivo assay for AMDase activity using 2-methyl-2-phenylmalonate as a substrate., left picture - colonies of E. coli containing pET30b vector without gene of AMDase, right picture - containing pET30b vector with the gene.<br>
Figure 10: In vivo assay for AMDase activity using 2-methyl-2-phenylmalonate as a substrate. Top plate - BL21(DE3) containing pET30b, bottom plate - BL21(DE3) containing pET30b with AMDase gene.<br>
Detailed description of the invention:<br>
1. Preferred Embodiments<br>
A first aspect of the invention relates to a biocatalytic method of decarboxylating an alpha-substituted malonate compound of formula I,<br>
HOOC-C(R1)(R2)-COOH     (I)<br>
or a salt form thereof,<br>
wherein<br>
residue R1 represents a mono- or bicyclic, optionally 1-, 2- or 3-fold substituted, ring,<br>
which is for example an optionally substituted, in particular monocyclic, aromatic or<br>
heteroaromatic ring; or a non-aromatic, in particular mono-or bicyclic ring optionally<br>
carrying one or two carbon-carbon double bonds within its ring; and<br>
residue R2 represents H, OH, SH, NH2, lower alkoxy, lower alkylthio, lower alkyl, lower<br>
alkenyl, lower alkynyl or hydroxyl-lower alkyl, in particular OH, SH or lower alkyl which<br>
method comprises<br>
a)     contacting at least on compound of formula I, in the form of any stereoisomer<br>
mixture or in optically pure form, with<br>
(i) a protein having malonate decarboxylase activity, in particular arylmalo-nate decarboxylase activity, and comprising an amino acid sequence having a sequence identity of 100 % or less or, in particular, of less than<br><br>
6<br>
99%, as for example less than 98, 97, 96 or 95%, but of at least 25%, as for example at least 30, 35, 40, 45, 50, 60, 70, 80 or 90 % sequence iden¬tity, for example 25 to 98, 30 to 97, 30 to 95, 30 to 90 or 30 to 80 or 30 to 70 or 30 to 70 % sequence identity to the Bordetella bronchiseptica pro¬tein  of SEQ  ID  NO:  2  (residues  1   to  240)  (Blast database  entry BAA02419.1)or (ii)       a protein having malonate decarboxylase activity, in particular arylmalo-nate decarboxylase activity and comprising an amino acid sequence hav¬ing a sequence identity of 100% or less as for example less than 99%, as for example less than 98, 97, 96 or 95%, but of at least 25%, as for ex¬ample at least 30, 35, 40, 45, 50, 60, 70, 80 or 90 % sequence identity, for example 25 to 98, 30 to 97, 30 to 95, 30 to 90 or 30 to 80 or 30 to 70 or 30 to 70 % sequence identity to the Aquifex pyrophilus protein of SEQ ID NO: 12 (residues 1 to 254) (Blast database entry AAF25672) b)     and optionally isolating at least one decarboxylation product of a compound of formula I, preferably one particular stereoisomer of the reaction product. In another alternative embodiment the present invention provides a method of the above type for decarboxylating a compound of formula I, or a salt form thereof, wherein residue R1 represents a linear or branched mono- or poly-unsaturated lower al-kenyl group optionally carrying one or more cyclic substituents; as for example being substituted by one or more, preferably one, mono- or bicyclic, optionally substituted aromatic or non-aromatic rings; or wherein two, for example terminal, substituents of the lower alkenyl group form together a carbo- or heterocyclic, saturated or unsatu¬rated, aromatic or in particular non-aromatic ring, as for example cycloalkylidene resi¬dues as defined herein; and<br>
residue R2 represents H, OH, SH, NH2, lower alkoxy, lower alkylthio, lower alkyl, lower alkenyl, lower alkynyl or hydroxyl lower alkyl, in particular, OH, SH or lower alkyl. According to said alternative embodiment the enzyme activity applied is an MDase activity as defined herein.<br>
In a particular group of compounds of formula I, or a salt form thereof, residue R1 represents a linear or branched mono-unsaturated lower alkenyl group optionally sub¬stituted by a mono- or bicyclic, optionally substituted ring, and residue R2 is a lower al¬kyl group.<br><br>
7<br>
In particular, compounds are to be mentioned, wherein R1 represents a linear d-C6-alken-1-yl group optionally substituted by a mono- or bicyclic, optionally substituted ring, and residue R2 is a CrC6-alkyl group.<br>
For example, residue R1 represents a linear Ci-C6-alken-1-yl group, like ethenyl, propen-1-yl or buten-1-yl, and residue R2 is a Ci-C6-alkyl group. Optionally said residue R1 may be further substituted by a mono- or bicyclic, in particular, monocyclic, aromatic or non-aromatic ring. Said ring group, which may preferably be aromatic, and may be a group as defined below, may be further substituted by 1, 2 or 3 optional substituents also as defined below. Optionally said residue R1 may also be terminally substituted by<br>
In another embodiment compounds have to be mentioned, wherein R1 represents a linear CrC6-alken-1-yl group terminally substituted by a cycloalkylidene ring, option¬ally in turn substituted, and residue R2 is a Ci-C6-alkyl group.<br>
As non-limiting examples of compounds of formula I there may be mentioned: 2-phenyl-malonic acid, 2-ethenyl-malonic acid, 2-propen-1yl-malonic acid, 2-buten-1yl-malonic acid, 2-(cyclohexylidenemethyl)malonic acid, 2-cyclohexen-1-yl-malonic acid; and the corresponding methylmalonic acid compounds.<br>
Preferably said methods comprise a decarboxylation reaction, which is a stereo¬selective mono-decarboxylation.<br>
Preferably, an optically pure substituted product of the formula II<br>
HC(R1)(R2)-COOH     (II)<br>
or a salt form thereof is obtained, wherein residue R1 and R2 are as defined above.<br>
A non-limiting example of a method of the invention may be illustrated by the fol¬lowing scheme:<br>
	R3<br>
*      Me<br>
J^COOH<br>
R4      ^^ ^COOH (III)<br>
R3   = H,    CH3<br>
4 R     = H,   CH3CH2<br>
Me	=  CH,<br><br>
R4/    ^ *^COOH (IV)<br>
R       Me<br><br>
8<br>
wherein an alpha-disubstituted malonate III is biocatalytically mono-decarboxylated to form, in high enantioselectivity, the corresponding mono-carboxylic acid IV.<br>
A further non-limiting example of a suitable method refers to the decarboxylation of compounds of formula III to form compounds of formula IV, wherein the Me (i.e. methyl) substituent in alpha position is replaced by SH, or, in particular, OH.<br>
A skilled reader will recognize that the present invention is not limited to the pre¬ferred decarboxylation reactions as described herein. As enzymatic reactions, in princi¬ple, are reversible reactions, the present invention also covers the corresponding re¬verse reaction, i.e. the carboxylation reactions, with compounds of formula II as sub¬strate and malonates of formula I as products.<br>
In a preferred embodiment, the protein having AMDase or MDase activity and showing an amino acid sequence identity of 100 % or less or, in particular, of less than 99% to the Bordetella bronchiseptica protein of SEQ ID NO: 2 forms a substrate pocket having the ability to carry a substrate molecule of formula I, and comprising in func¬tional arrangement the following structural elements<br>
a)	a Cystein containing catalytic site (CS), the Cystein residue of which is able to interact with the substrate, in particular by transferring a proton to the substrate forming an enolate intermediate,<br>
b)	an oxy-anion hole (OAH), comprising a plurality of polar amino acid side chains interacting with the carboxylic group of the substrate not to be cleaved off.<br>
In addition said substrate pocked may further comprise<br>
c)	a large binding pocket (LBP) interacting with the cyclic, preferably aromatic or heteroaromatic, residue R1 and<br>
d)	a small binding pocket (SBP) interacting with residue R2.<br>
As further illustrated by the attached figures, the analysis of the crystal structure of the AMDase enzyme of B. bronchiseptica (SEQ ID NO:2) allowed the identification of individual amino acid residues or amino acid sequence portions involved in the forma¬tion of the reactivity centre of the enzyme, so that a model system or reference enzyme for suitable further enzymes with AMDase or MDase activity and functional mutants derived there from could be established.<br>
In particular, for said specific reference enzyme certain "key amino acid residues" could be identified, which are predicted to be involved in the formation of functionally<br><br>
9<br>
distinct portions of the substrate pocket. Said functionally distinct portions are desig¬nated catalytic site (CS), oxy-anion hole (OAH) small binding pocket (SBP) and large binding pocket (LBP).<br>
A first functional portion is CS and a key amino acid residue is Cys188. Moreover, it was found that sequence portions not adjacent to each other in the pri¬mary amino acid sequence, are nevertheless functionally related, by contributing to the same functional portion of the binding pocket. Thus, is was observed that the functional portion OAH is formed by at least two distinct sequence portions, which were desig¬nated OAH1 and 2 comprising key amino acid residues as described below.<br>
OAH1	Thr75, Gly189<br>
OAH2	Ser76, Tyr126<br>
It was also observed that said reference enzyme forms the binding pocket re¬gions SBP and LBP the amino acid residues associated therewith may be further sub¬divided in accordance with their preferential orientation with respect to the substrate of formula I attached to the enzyme.<br>
LBP may be subdivided into LBP 1 and LBP2 characterized by the following key residues:<br>
LBP1	Pro14, Pro15, Gly189, Gly190, Met104<br>
LBP2	Val13, Met73, Pro21,Ser212<br>
LBP1 residues are located around cyclic group (R1) of the substrate, while LBP2 residues are building rest of big binding pocket).<br>
SBP may be subdivided into SBP 1 and SBP2 characterized by the following key residues:<br>
SBP1	Lys40, Val43, Tyr48, Leu77, Val156, Pro14, Gly74<br>
SBP2	VaM 56, Met159, Val43<br>
SBP1 residues form the bigger part of small binding pocket around the leaving carboxyl group of the substrate, while SBP2 residues are building the smaller part of said binding pocket around the less bulky residue (R2) of the substrate.<br><br>
10<br>
Based on this analysis a highly characteristic sequence pattern could be devel¬oped, by means of which further candidates of proteins with the desired enzymatic ac¬tivity may be searched, and actually could be found by the inventors. Examples of such newly recognized AMDase or MDase enzymes and corresponding portions of their 5 amino acid sequences, which also show said characteristic sequence pattern, are shown in the alignment of Figure 3(b). A characteristic sequence pattern of key amino acid residues (i.e. a subgroup of said key amino acids) may be derived from said alignment. Said subgroup comprises in particular:<br>
10	Pro14, Thr75, Ser76, Tyr126, Cys188<br>
or combinations of 3 or 4 of these residues, as for example:<br>
Thr75, Ser76, Tyr126, Cys188<br>
15	Pro14, Thr75, Ser76, Cys188<br>
Searching for further candidate enzymes and/or mutants by applying said se¬quence pattern would also be encompassed by the present invention. It will be under¬stood by a skilled reader that the above sequence pattern is not limited by the exact 20 distances between two adjacent amino acid residues of said pattern. Each of the dis¬tances between two neighbours in the above patterns may, for example, vary inde¬pendently of each other by up to ±10, ± 5, ±3, ±2 or ±1 amino acid positions without substantially affecting the desired enzymatic activity.<br>
In view of the close spatial proximity individual key amino acid residues may be 25     allocated to more than one of said above-mentioned sequence portions (see for exam¬ple amino acids in position 14, 43, 74, 75, 189).<br>
In line with said above functional and spatial analysis of individual amino acid residues based on the crystallographic data as obtained according to the present in¬vention, unique partial amino acid sequences characteristic of potentially useful en-30     zymes with aryl malonate or malonate decarboxylase activity of the invention may be identified.<br>
According to a further preferred embodiment the enzymes of the invention as pre¬ferred partial amino acid sequences the may be mentioned<br><br>
11<br>
a)	the Cystein containing catalytic site sequence "CSS", which is formed by a se¬<br>
quence region corresponding to residues 186 to 191 of SEQ ID NO: 1 which<br>
shows the following sequence pattern:<br>
"CSS"	(S/L)C (G/T)(G/N)	(SEQ ID NO: 13)<br>
and<br>
b)	the oxy-anion hole forming sequences "OAH1S" and "OAH2S" corresponding to<br>
residues 73 to 77 and 124 to 128, respectively of SEQ ID NO: 1, which show the<br>
following two sequence pattern:<br>
"OAH1S"	(M/G)(G/C)TS(L/G)	(SEQ ID NO: 14)<br>
"OAH2S"	T(A/P)Y(I/R)(D/Q)	(SEQ ID NO: 15)<br>
wherein in OAH2S the first and/or last amino acid residue may be missing.<br>
Specific examples of suitable partial sequences CSS, OAH1S and OAH2S can be derived from Figure 3 (b).<br>
According to a further embodiment said structural elements CSS, OAH1S and OAH2S are arranged in the amino acid chain in the following sequential (N-&gt;C) order:<br>
OAH1S-OAH2S-CSS and wherein said structural elements are linked together via amino acid sequences in¬dependently of each other comprising one or more amino acid residues.<br>
In particular, said structural elements CSS, OAH1S and OAH2S may be ar¬ranged in the amino acid chain in the following sequential (N-&gt;C) order:<br>
OAHS1 -sp1- OAHS2 -sp2 - CS wherein sp1 and sp2 are linking amino acid sequences with the following approximate sequence length<br>
sp1   43-47 amino acids sp2   55-57 amino acids.<br>
As further preferred partial amino acid sequences the may be mentioned<br>
c)	the large binding pocket sequence "LBPS" which is formed by a sequence region<br>
corresponding to residues 12 to 17 of SEQ ID NO: 1 and which shows the follow¬<br>
ing sequence pattern:<br>
"LBPS"	IVP(P/S/A)(A/T/S/P)(A/N/G)	(SEQ ID NO: 16)<br><br>
12<br>
Specific examples of suitable partial sequences LBPS can be derived from Figure 3<br>
(b).<br>
In particular, said structural elements CSS, OAH1S, OAH2S and LBPS may be arranged in the amino acid chain in the following sequential (N-&gt;C) order:<br>
LBPS -sp3- OAHS1 -sp1- OAHS2 -sp2- CS wherein sp1, sp2 and sp3 are linking amino acid sequences with the following ap¬proximate sequence length<br>
sp1   43-47 amino acids<br>
sp2   55-57 amino acids<br>
sp3   57-62 amino acids.<br>
A skilled reader will recognize that mutants of the specific decarboxylase en¬zymes as disclosed herein may be engineered in order to alter their substrate spectrum (substrate specificity) or improve their enzyme activity vis-a-vis a certain substrate (if compared to the parent enzyme). Said mutations (amino acid substitutions, additions, deletions, insertion, inversions) may also be performed within the structural elements as defined herein above. Particular mutants may comprise 1 to 10, in particular 1, 2, 3, 4 or 5 simultaneous mutations in one or more of said structural elements. Non-limiting examples of suitable mutations will be given below.<br>
A skilled reader will also recognize that additional single or multiple mutations may be performed in regions of the amino acid sequence of a decarboxylase enzyme as disclosed herein, which are not related to said structural elements, and, therefore, do not contribute to the substrate pocket or catalytic site of the enzyme, and, therefore are less critical to enzyme activity or substrate specificity. Guided by the technical teaching of the present invention (as for example the sequence alignments presented herein) a skilled reader will easily identify such regions open to more extensive modifi¬cations. For example, a skilled reader will be enabled to perform 1 to 30, 2 to 20 or 5 to 10 simultaneous mutations in such regions without running an unforseeable risk of negatively affecting enzyme activity and/or substrate specificity.<br>
A skilled reader will also recognize that additional single or multiple mutations may be performed in regions of the amino acid sequence of a decarboxylase enzyme as disclosed herein, which, although not building the substrate pocket or active site , may influence the shape of said regions, and, therefore, may influence enzyme activity and/or substrate specificity. Guided by the technical teaching of the present invention (as for example the sequence alignments presented herein) a skilled reader will easily<br><br>
13<br>
identify such regions as well. For example, a skilled reader will be enabled to perform 1 to 30, 2 to 20 or 5 to 10 simultaneous mutations in such regions. As a non-limiting ex¬ample of such an amino acid residue of the Bordetella bronchiseptica protein of SEQ ID NO: 2 is Leu191, which may be altered (replaced by any other amino acid residue) or deleted in order to influence the shape of active site and/or binding pocket of the en¬zyme.<br>
According to a further embodiment of the method of the invention, the protein having AMDase or MDase activity is selected from the group consisting of<br>
a)	a protein comprising amino acid residues 2 to 241 of SEQ JD NO: 4 and origi¬nating from Mesorhizobium sp. BNCI (Blast database entry EAN08019.1);<br>
b)	a protein comprising amino acid residues 2 to 240 of SEQ ID NO: 6 and origi¬nating from Pyrcoccus horikoshii OT3 (Blast database entry BAA30082.1);<br>
c)	a protein comprising amino acid residues 2 to 267 of SEQ ID NO: 8 and origi¬nating from Bordetella bronchiseptica RB50 (Blast database entry NP_888076.1);<br>
d)	a protein comprising amino acid residues 2 to 246 of SEQ ID NO: 10 and origi¬nating from Alkaliphilus metalliredigenes QYMF (Blast database entry ZP_00801202.1); and<br>
e)	proteins having arylmalonate or malonate decarboxylase activity and having a sequence identity less than 100% to one of the sequences of a), b), c) or d).<br>
Identifying novel proteins having arylmalonate decarboxylase activity from Aquifex and Pyrococus is very promising, since those organisms are thermophilic, thus the en¬zymes are expected to be also thermo stabile. Consequently, the present invention also provides novel thermo stabile proteins having arylmalonate or malonate decar¬boxylase activity and corresponding stabile proteins mutants and derivatives.<br>
According to a further preferred embodiment the decarboxylation reaction is per¬formed with isolated or purified, optionally immobilized decarboxylase as defined herein, or by culturing a microorganism expressing said decarboxylase enzyme activity in the presence of a compound of formula I.<br>
The present invention also relates to a protein having a decarboxylase activity as defined herein, and showing a sequence identity of less than 99%, as for example less than 98, 97, 96 or 95%, but of at least 25%, as for example at least 30, 35, 40, 45, 50, 60, 70, 80 or 90 %  sequence identity, for example 25 to 98, 30 to 97, 30 to 95, 30 to<br><br>
14<br>
90 or 30 to 80 or 30 to 70 or 30 to 70 % sequence identity to the Bordetella bronchisep-<br>
tica protein comprising amino acid residues 1 to 240 of SEQ ID NO: 2.<br>
The present invention also relates to a protein having a decarboxylase activity as defined herein, selected from<br>
a)	a protein comprising amino acid residues 1 to 241 of SEQ ID NO: 4 and origi¬nating from Mesorhizobium sp. BNCI;<br>
b)	a protein comprising amino acid residues 1 to 240 of SEQ ID NO: 6 and origi¬nating from Pyrcoccus horikoshii OT3;<br>
c)	a protein comprising amino acid residues 1 to 267 of SEQ ID NO: 8 and origi¬nating from Bordetella Bronchiseptica RB50;<br>
d)	a protein comprising amino acid residues 1 to 246 of SEQ ID NO: 10 and origi¬nating from Aklaliphilus metalliredigenes QYMF; and<br>
e)	a protein comprising amino acid residues 1 to 256 of SEQ ID NO: 12 and origi¬nating from Aquifex pyrophilus.<br>
as well as mutants thereof having a decarboxylase activity as defined herein, which mutant having a sequence identity less than 100% or less as for example less than 99%, as for example less than 98, 97, 96 or 95%, but of at least 25%, as for example at least 30, 35, 40, 45, 50, 60, 70, 80 or 90 % sequence identity, for example 25 to 98, 30 to 97, 30 to 95, 30 to 90 or 30 to 80 or 30 to 70 or 30 to 70 % sequence identity to the corresponding parent protein.<br>
For example, a protein having the desired decarboxylase activity selected from the group consisting of a protein originating from Mesorhizobium sp. BNCI and a pro¬tein originating from Bordetella bronchiseptica as defined hereinabove, may be used to decarboxylate compounds of the above-identified type wherein residue R1 represents a linear or branched mono- or poly-unsaturated lower alkenyl group optionally substituted by a mono- or bicyclic, optionally substituted ring as defined above, and residue R2 represents H, OH, SH, NH2, lower alkoxy, lower alkylthio, lower alkyl, lower alkenyl, lower alkynyl or hydroxyl lower alkyl, in particular lower alkyl.<br>
In particular said mutants of proteins of SEQ ID NO: 2, 4, 6, 8 and 10 still com¬prise part or all of the key amino acid residues and/or structural elements CSS, OAH1S, OAH2S and LBPS as defined above.<br>
Non-limiting examples of further potentially applicable mutations are for example disclosed in WO2005/078111, Figure 2 for proteins of SEQ ID NO:2, the disclosure of which document is herewith incorporated by reference. By aligning said sequence with<br><br>
15<br>
structurally related, newly screened proteins with AMDase activity of the invention (see for example Fig. 3 (a)) a skilled reader will be enabled to deduce suitable mutants form newly obtained enzymes with AMDase activity as well. Such deduced mutants may either have substantially the same enzyme characteristics, as for example enzymatic activity, substrate specificity, stereo specificity, stability, solubility, as the corresponding non-mutated parent enzyme, or have modified enzyme characteristics, as for example modified, i.e. increased or diminished enzymatic activity, catalytic efficiency, stability, solubility, stereo specificity, and/or modified, i.e. broader or narrower substrate specific¬ity.<br>
With respect to the Bordetella bronchiseptica protein of SEQ ID NO: 2 non-limiting examples of key amino acid substitution are listed below.<br><br>
Structural portion	Key amino acid	Mutation<br>
LBP1	Pro14, Pro 15, Gly189, Gly190, Met104	Gly, Ala, Val Gly, Ala, His Ala, Val, Del1) Ala, Val, Ser, Del1)<br>
LBP2	Val13, Met73, Pro21, Ser212	Gly, Ala<br>
SBP1	Leu40,<br>
Val43,<br>
Tyr48,<br>
Leu77,<br>
Val156,<br>
Pro14,<br>
Gly74	Ala, Gly Ala, Gly<br>
Ala<br>
Gly, Ala, Val<br>
Cys<br>
SBP2	Val156, Met159, Val43	Ala<br>
Gly, Ala, Val, Ser, Cys<br>
Ala, Gly<br>
OAH1	Thr75 Gly189,	Ala, Val, Del1)<br>
OAH2	Ser76, Tyr126,	Lys<br>
CS	Cys188	Ser<br><br>
16<br><br>
residues infuencing shape of active site	Leu191	Del<br>
' Dei = deletion of amino acid residue<br>
Non-limiting examples of mutants are those comprising at least on mutation in positions selected from 13, 14, 15, 189, 190 and 191; or multiple mutations comprising a combination of mutations obtained by any permutation of these positions, as for ex¬ample<br>
13 and 14,<br>
13	and 15,<br>
14	and 15,<br><br>
13,	14 and 15,<br>
14,	15 and 189, 14, 15 and 190,<br>
14, 15, 189 and 190, 13,14, 15 and 189, 13, 14, 15 and 190, 13, 14, 15, 189 and 190 13, 14 and 191,<br>
13,	15 and 191;<br>
14,	15qnd 191,<br><br>
13,	14, 15 and 191,<br>
14,	15, 189 and 191 14, 15. 190 and 191,<br>
14, 15, 189, 190 and 191,<br>
13, 14, 15, 189 and 191,<br>
13, 14, 15, 190 and 191,<br>
13, 14, 15, 189, 190 and 191 etc.<br>
Performing mutations in these positions allows improving the binding affinity of the enzyme to non-aromatic, non-cyclic malonates of formula (I), as for example those carrying an alkenyl substituent. Specific examples of amino acid mutations in this con¬text are<br>
Val13AlyorGly;<br>
Pro14Val, Ala or Gly;<br>
Pro15His, or Gly;<br>
Gly189AlaorDel;<br><br>
17<br>
Gly190Ala, SerorDel;<br>
Leu191Del.<br>
The mutations Pro14Val and Pro15His may provide for better binding of small unsaturated aliphatic substrates. Adding Gly190Ala to such a double mutant might pro¬vide even better results. After adding a fourth mutation such as Gly189Ala the enzyme might be capable of accepting only unsaturated aliphatic substrates and no more aro¬matic ones. Mutations of Val13 and Pro14 to smaller amino acids such as Gly/Ala and deletion of one of the residues Gly189 or 190 should make the enzyme to better accept branched substrates. The above explanation shall not be considered as limiting.<br>
In analogy a skilled reader will be able to define suitable mutants for the structur¬ally related enzymes, i.e. from<br>
a)	a protein comprising amino acid residues 2 to 241 of SEQ ID NO: 4 and origi from Mesorhizobium sp. BNCI;<br>
b)	a protein comprising amino acid residues 2 to 240 of SEQ ID NO: 6 and origi¬nating from Pyrcoccus horikoshii OT3;<br>
c)	a protein comprising amino acid residues 2 to 267 of SEQ ID NO: 8 and origi¬nating from Bordetella Bronchiseptica RB50; and<br>
d)	a protein comprising amino acid residues 2 to 246 of SEQ ID NO: 10 and origi¬nating from Aklaliphilus metalliredigenes QYMF;<br>
The present invention also relates to a nucleic acid encoding a protein having decarboxylase activity as defined herein.<br>
The present invention also relates to an expression cassette, comprising a nu¬cleic acid as defined above, operably linked with at least one regulatory nucleic acid sequence.<br>
The present invention also relates to a recombinant expression vector comprising at least one expression cassette or nucleic acid as defined above.<br>
The present invention also relates to a recombinant microorganism, carrying at least one expression vector as defined above.<br>
The present invention also relates to a bioreactor comprising at least one protein having decarboxylase activity as defined herein or a recombinant microorganism as defined above, optionally in immobilized form.<br><br>
18<br>
The present invention also relates to a method of preparing an enzyme with decarboxylase activity as defined herein, which method comprises cultivating a recom¬binant microorganism as defined above and optionally isolating the said decarboxylase from the culture.<br>
The present invention also relates to a crystalline form of a protein having decarboxylase activity as defined herein, in particular those forms, wherein the protein having arylmalonate decarboxylase activity is as defined above.<br>
A preferable crystalline form is obtained from the Bordetella bronchiseptica pro¬tein of SEQ JD NO: 2 (Blast database entry BAA02419.1), the crystals of which belong to the P2! space group with a unit cell having the constants a = 38.7, o = 65.6, c = 42.2 A (Angstrom), and fi = 110.8°.<br>
The present invention also relates to a method of preparing a crystalline form of a protein having decarboxylase activity as defined herein, which method comprises add¬ing to a solution containing said protein (in a concentration of about 1 to 50 or 5 to 20 mg/ml) a crystallization agent having a pH in the range of 8,3 to 8,7, as for example 8,5, preferably buffered with Tris buffer, and a polyalkylene glycol, like polyethylene glycol, in particular PEG 4000 to 8000, like PEG 6000.<br>
The present invention also relates to a screening method for candidate enzymes having decarboxylase activity as defined herein, which method comprises identifying a sequence motif (pattern) characteristic for an enzyme with decarboxylase activity as defined herein and screening known amino acid sequences, as for example amino acid sequence data bases, for sequences partially or fully matching with said sequence mo¬tif. Preferably said sequence motif is derived from a the crystalline structure of the crys¬talline form of a protein having decarboxylase activity as defined by SEQ ID NO: 2, 4, 6,8,9, 10 or 12.<br>
The present invention also relates to a method of preparing a candidate enzymes having decarboxylase activity as defined herein, screened by a method of as defined above, which method comprised obtaining a coding nucleotide sequence of the candi¬date enzyme, expressing said sequence in a recombinant microorganism and isolating said expression product. Finally, the present invention also relates to the use of an ex¬pression product as prepared by said method in a decarboxylation reaction decarboxy-lating a compound of formula I as defined above.<br>
2.   Explanation of particular terms<br><br>
19<br>
"AMDase" (EC 4.1.1.76) refers to enzymes having aryl malonate dehydrogenase activity and catalyze the enantioselective mono-decarboxylation of alpha-aryl-malonates, like alpha-aryl-alpha-methylmalonate to give optically pure arylpropionate. The enzyme does not require biotin or any other co-factors for activity. In addition, de¬carboxylation does not involve formation of a malonyl thioester-enzyme intermediate.<br>
An "enzymes having aryl malonate dehydrogenase (AMDase) activity" catalyzes the mono-decarboxylation, i.e. replacement of one carboxyl group by hydrogen, of at least one substrate of the general formula I as defined herein, in particular a substrate of formula I carrying at least one aromatic substituent in alpha position of malonate.<br>
As decarboxylase enzymes or mutants disclosed herein, also may either show an extended substrate specificity (if compared to a non-mutated reference enzyme as herin described), according to which not only cyclic or aryl group-containing malonates are decarboxylated, but also non-aromatic, non-cyclic malonates are decarboxylated, or may prefer as a substrate such non-cyclic, non-aromatic malonates, the term "MDase" or "enzyme having MDase activity" as used herein more generally refers to those decarboxylase enzyme with extended substrate specificity and/or to those de¬carboxylase enzymes with altered or modified substrate preference or substrate speci¬ficity. Said MDases catalyze the mono-decarboxylation, i.e. replacement of one car¬boxyl group by hydrogen, of at least one substrate of the general formula I as defined herein, in particular also or exclusively those substrates of formula I carrying no cyclic or no aromatic substituent in alpha position of malonate. The MDase enzyme does not require biotin or any other co-factors for activity. In addition, decarboxylation does not involve formation of a malonyl thioester-enzyme intermediate.<br>
An "extended substrate specificity" refers to enzymes which, if compared to a reference enzyme, convert additional structurally different substrates.<br>
An "altered / modified substrate specificity" refers to enzymes which, if compared to a reference enzyme, convert a partially or completely different set of substrate mole¬cules. Thus, the term "altered substrate specificity" may describes a situation where a decarboxylase enzyme, for example effected by mutation, is better adapted for the de¬carboxylation of a specific substrate molecule than a reference enzyme (as for example the non-mutated enzyme). For example, a higher preference or specificity may be caused by a higher substrate affinity of the binding pocked formed by the enzyme vari¬ants.<br><br>
20<br>
An "altered / modified enzyme activity" refers to enzymes which, if compared to a reference enzyme, show for at least one common substrate molecule a faster or slower conversion kinetics.<br>
The term "decarboxylase" or "enzyme having decarboxylase activity" as used herein, unless otherwise stated, generically refers to AMDase and/or MDase enzymes as defined herein.<br>
Due to the reversibility of enzymatic reaction the present invention also relates to the corresponding reverse (i.e. monocarboxylation) reaction of the biocatalytic decar-boxyiation reactions described herein.<br>
The term "biocatalytic method" refers to any method performed in the presence of catalytic activity of a decarboxylase enzyme (AMDase or MDase) as defined herein, i.e. in the presence of isolated pure or crude enzyme or entire microbial cells containing or expression such enzyme activity.<br>
The term „stereospecific" means that one of several enatiomers is formed by the enzyme in high enantiomeric excess or purity, of at least 90%ee, preferably at least 95 %ee, in particular at least 98 %ee, or at least 99 %ee. the ee% value is calculated according to the following formula<br>
ee% = [XA-XB]/[XA+XBri00, wherein XA and XB refer to the molar fraction of enantiomer A or B, respectively.<br>
The "substrate pocket" of an AMDase, MDase or decarboxylase enzyme as de¬fined herein, or its "reactivity centre" harbours during the reaction to be catalyzed a substrate molecule of formula I and converts it to a product of formula II. Said substrate pocket is composed of certain "structural elements" i.e. portions of said substrate pocket with different function. Said structural elements are in "functional arrangement" to each other, i.e. they cooperate during the reaction to be catalyzed.<br>
A "sequence motif or "pattern" represents a characteristic arrangement or "fin¬gerprint" of a plurality of amino acid residues which are either adjacent to each other within a specific amino acid sequence or are separated from each other in a defined manner, i.e. by intermediate spacer sequences of characteristic length.<br>
As regards substrates of formula I and products of formula II the following mean¬ings are preferred:<br>
A "salt form" refers to an ammonium salt or an alkali or alkaline earth metal salt, as for example Na+, K+ or Ca2+ salts.<br><br>
21<br>
"Lower alkyl" refers to d-C8-alkyl radicals which are linear or branched radicals having from 1 to 8 , preferably 1, 2, 3 or 4 carbon atoms. Examples thereof are the Cr C4-alkyl radicals methyl, ethyl, n-propyl, isopropyl, n-butyl, 2-butyl, isobutyl or terf-butyl; and additionally residues with more than 4 carbon atoms like pentyl, 1-methylbutyl, 2-methylbutyl, 3-methylbutyl, 2,2-dimethylpropyl, 1-ethylpropyl, hexyl, 1,1-dimethylpropyl, 1,2-dimethylpropyl, 1-methylpentyl, 2-methylpentyl, 3-methylpentyl, 4-methy I pentyl, 1,1-dimethylbutyl, 1,2-dimethylbutyl, 1,3-dimethylbutyl, 2,2-dimethylbutyl, 2,3-dimethylbutyl, 3,3-dimethylbutyl, 1-ethylbutyl, 2-ethylbutyl, 1,1,2-trimethylpropyl, 1,2,2-trimethylpropyl, 1-ethyl-1-methylpropyl, 1-ethyl-2-methylpropyl, heptyl, octyl and their constitutional isomers such as 2-ethylhexyl<br>
"Lower alkoxy" preferably refers to the Ci-C8-alkoxy analogues of the above-mentioned lower alkyl radicals.<br>
"Lower alkylthio" preferably refers to the Ci-C8-alkthio analogues of the above-mentioned lower alkyl radicals. Examples are methylthio, ethylthio, propylthio, isopro-pylthio, butylthio, sec-butylthio, isobutylthio and tert-butylthio.<br>
"Lower alkenyl" comprises C2-C8-alkenyl radicals which are monounsaturated lin¬ear or branched hydrocarbon radicals having from 2 to 8, preferably 2 to 4, carbon at¬oms, as for example ethenyl, 1- or 2-propenyl, 1-, 2- and 3-butenyl, 2-methylpropen-3-yl, 2-methylpropen-1-yl, 1-, 2-, 3- and 4-pentenyl, 1-, 2-, 3-, 4- and 5-hexenyl, 1-, 2-, 3-, 4-, 5- and 6-heptenyl 1-, 2-, 3-, 4-, 5-, 6- and 7-octenyl and also their constitutional isomers.<br>
"Lower alkynyl" comprises the alkynyl homologues of the above "lower alkeny" radicals.<br>
The term "hydroxy lower-alkyl" refers to CrC8-hydroxyalkyl which is a linear or branched alkyl radical having from 1 to 8, in particular from 1 to 4 carbon atoms, in which at least one hydrogen atom, for example 1 or 2 of the hydrogen atoms, is/are re¬placed by a hydroxyl group. Examples thereof are, hydroxymethyl, 2-hydroxy-1 -ethyl, 2- and 3-hydroxy-1-propyl, 2-, 3- and 4-hydroxy-1-butyl, 2-, 3-, 4- and 5-hydroxy-1-pentyl, 2-, 3-, 4-, 5- and 6-hydroxy-1-hexyl, 2-, 3-, 4-, 5-, 6- and 7-hydroxy-1-heptyl, 2-, 3-, 4-, 5-, 6-, 7- and 8-hydroxy-1-octyl, 2,3-dihydroxy-1-propyl and their constitutional isomers.<br>
The term "cyloalkylidene" represents ring structures comprising 4 to 7 ring carbon atoms, linked via a double bond to the residual structure of a molecule. As examples there may be mentioned cyclopentylidene, cyclohexylidene and cycloheptylidene.<br><br>
22<br>
As examples for "mono- or bicyclic ring" residues there may be mentioned carbo-or heterocyclic, optionally condensed, non-aromatic or preferably aromatic or het-eroaromatic rings, having 3 to 12 ring carbon atoms and optionally 1 to 4 heteroatoms, like N, S and O. As examples there may be mentioned cyclopropyl, cyclobutyl, cyclopenty, cyclohexyl, cycloheptyl, as well as the mono- or polyunsaturated analogues thereof, as for example cyclobutenyl, cyclopentenyl, cyclohexenyl, cycloheptenyl, cyclohexadienyl, cycloheptadienyl, as well as naphthyl and phenyl; as well as 5- to 7-membered saturated on mono-or polyunsaturated heterocyclic radicals having 1 to 4 heteroatoms, selected from O, N and S, and wherein the heterocyclic ring may be fur¬ther condensed with another heterocyclic or carbocyclic ring. Example of heterocyclic residues are those derived from pyrrolidine, tetrahydrofurane, piperidine, morpholine, pyrrole, furane, thiophene, pyrazole, imidazole, oxazole, thiazole, pyridine, pyrane, pyrimidine, pyridazine, pyrazine, coumarone, indole and quinoline. A "monocyclic" ring also comprises the above-mentioned cycloalkylidene structures.<br>
As "optional substituents" there may be mentioned NH2, OH, SH, halogen, like F, CI, Br, J; lower alkoxy, lower alkylthio, lower alkyl, lower alkenyl, lower alkynyl or hy-droxyl-lower alkyl, as defined above. "Optional substituents" also comprises the above-mentioned cycloalkylidene residues, which may be considered as being formed by two of such stubstituents.<br>
3.      Other embodiments of the invention<br>
3.1    Proteins according to the invention<br>
The present invention is not limited to the specifically disclosed "proteins with AMDase activity", or "proteins with MDase activity" but also extends to functional equivalents thereof.<br>
"Functional equivalents" or analogs of the concretely disclosed enzymes are, within the scope of the present invention, various polypeptides thereof, which moreover possess the desired biological function or activity, e.g. enzyme activity.<br>
For example, "functional equivalents" means enzymes, which, in a test used for enzymatic activity, display at least a 1 to 10%, or at least 20%, or at least 50%, or at least 75%, or at least 90% higher or lower activity of an enzyme, as defined herein.<br>
"Functional equivalents", according to the invention, also means in particular mu¬tants, which, in at least one sequence position of the amino acid sequences stated<br><br>
23<br>
above, have an amino acid that is different from that concretely stated, but neverthe¬less possess one of the aforementioned biological activities. "Functional equivalents" thus comprise the mutants obtainable by one or more amino acid additions, substitu¬tions, insertions, deletions and/or inversions, where the stated changes can occur in any sequence position, provided they lead to a mutant with the profile of properties ac¬cording to the invention. Functional equivalence is in particular also provided if the re¬activity patterns coincide qualitatively between the mutant and the unchanged polypep¬tide, i.e. if for example the same substrates are converted at a different rate. Examples of suitable amino add substitutions are shown in the following table:<br>
Original residue	Examples of substitution<br><br>
Ala	Ser<br>
Arg	Lys<br>
Asn	Gin; His<br>
Asp	Glu<br>
Cys	Ser<br>
Gin	Asn<br>
Glu	Asp<br>
Gly	Pro<br>
His	Asn ; Gin<br>
lie	Leu; Val<br>
Leu	lie; Val<br>
Lys	Arg ; Gin ; Glu<br>
Met	Leu ; lie<br>
Phe	Met; Leu ; Tyr<br>
Ser	Thr<br>
Thr	Ser<br>
Trp	Tyr<br>
Tyr	Trp ; Phe<br>
Val	lie; Leu<br>
"Functional equivalents" in the above sense are also "precursors" of the polypep¬tides described, as well as "functional derivatives" and "salts" of the polypeptides.<br>
"Precursors" are in that case natural or synthetic precursors of the polypeptides with or without the desired biological activity.<br>
The expression "salts" means salts of carboxyl groups as well as salts of acid addition of amino groups of the protein molecules according to the invention. Salts of carboxyl groups can be produced in a known way and comprise inorganic salts, for ex¬ample sodium, calcium, ammonium, iron and zinc salts, and salts with organic bases, for example amines, such as triethanolamine, arginine, lysine, piperidine and the like.<br><br>
24<br>
Salts of acid addition, for example salts with inorganic acids, such as hydrochloric acid or sulfuric acid and salts with organic acids, such as acetic acid and oxalic acid, are also covered by the invention.<br>
"Functional derivatives" of polypeptides according to the invention can also be produced on functional amino acid side groups or at their N-terminal or C-terminal end using known techniques. Such derivatives comprise for example aliphatic esters of carboxylic acid groups, amides of carboxylic acid groups, obtainable by reaction with ammonia or with a primary or secondary amine; N-acyl derivatives of free amino groups, produced by reaction with acyl groups; or O-acyl derivatives of free hydroxy groups, produced by reaction with acyl groups.<br>
"Functional equivalents" naturally also comprise polypeptides that can be ob¬tained from other organisms, as well as naturally occurring variants. For example, ar¬eas of homologous sequence regions can be established by sequence comparison, and equivalent enzymes can be determined on the basis of the concrete parameters of the invention.<br>
"Functional equivalents" also comprise fragments, preferably individual domains or sequence motifs, of the polypeptides according to the invention, which for example display the desired biological function.<br>
"Functional equivalents" are, moreover, fusion proteins, which have one of the polypeptide sequences stated above or functional equivalents derived there from and at least one further, functionally different, heterologous sequence in functional N-terminal or C-terminal association (i.e. without substantial mutual functional impairment of the fusion protein parts). Non-limiting examples of these heterologous sequences are e.g. signal peptides, histidine anchors or enzymes.<br>
"Functional equivalents" that are also included according to the invention are homologues of the concretely disclosed proteins. These possess percent identity val¬ues as stated above. Said values refer to the identity with the concretely disclosed amino acid sequences, and may be calculated according to the algorithm of Pearson and Lipman, Proc. Natl. Acad, Sci. (USA) 85(8), 1988, 2444-2448.<br>
The % identity values may also be calculated from BLAST alignments, algorithm blastp (protein-protein BLAST) or by applying the Clustal setting as given below..<br>
A percentage identity of a homologous polypeptide according to the invention means in particular the percentage identity of the amino acid residues relative to the total length of one of the amino acid sequences concretely described herein.<br><br>
25<br>
In the case of a possible protein glycosylation, "functional equivalents" according to the invention comprise proteins of the type designated above in deglycosylated or glycosylated form as well as modified forms that can be obtained by altering the glyco¬sylation pattern.<br>
Such functional equivalents or homologues of the proteins or polypeptides ac¬cording to the invention can be produced by mutagenesis, e.g. by point mutation, lengthening or shortening of the protein.<br>
Such functional equivalents or homologues of the proteins according to the invention can be identified by screening combinatorial databases of mutants, for example shortening mutants. For example, a variegated database of protein variants can be produced by combinatorial mutagenesis at the nucleic acid level, e.g. by enzymatic ligation of a mixture of synthetic oligonucleotides. There are a great many methods that can be used for the production of databases of potential homologues from a degenerated oligonucleotide sequence. Chemical synthesis of a degenerated gene sequence can be carried out in an automatic DNA synthesizer, and the synthetic gene can then be ligated in a suitable expression vector. The use of a degenerated genome makes it possible to supply all sequences in a mixture, which code for the desired set of potential protein sequences. Methods of synthesis of degenerated oligonucleotides are known to a person skilled in the art (e.g. Narang, S.A. (1983) Tetrahedron 39:3; Itakura et al. (1984) Annu. Rev. Biochem. 53:323; Itakura et al., (1984) Science 198:1056; Ike etal. (1983) Nucleic Acids Res. 11:477).<br>
In the prior art, several techniques are known for the screening of gene products of combinatorial databases, which were produced by point mutations or shortening, and for the screening of cDNA libraries for gene products with a selected property. These techniques can be adapted for the rapid screening of the gene banks that were produced by combinatorial mutagenesis of homologues according to the invention. The techniques most frequently used for the screening of large gene banks, which are based on a high-throughput analysis, comprise cloning of the gene bank in expression vectors that can be replicated, transformation of the suitable cells with the resultant vector database and expression of the combinatorial genes in conditions in which detection of the desired activity facilitates isolation of the vector that codes for the gene whose product was detected. Recursive Ensemble Mutagenesis (REM), a technique that increases the frequency of functional mutants in the databases, can be used in combination with the screening tests, in order to identify homologues    (Arkin and<br><br>
26<br>
Yourvan (1992) PNAS 89:7811-7815; Delgrave et al. (1993) Protein Engineering 6(3):327-331).<br>
3.2 Coding nucleic acid sequences<br>
The invention also relates to nucleic acid sequences that code for enzymes as defined herein.<br>
The present invention also relates to nucleic acids with a certain degree of "iden¬tity" to the sequences specifically disclosed herein. "Identity" between two nucleic acids means identity of the nucleotides, in each case over the entire length of the nucleic acid.<br>
For example the identity may be calculated by means of the Vector NTI Suite 7.1 program of the company Informax (USA) employing the Clustal Method (Higgins DG, Sharp PM. Fast and sensitive multiple sequence alignments on a microcomputer. Comput Appl. Biosci. 1989 Apr; 5(2): 151-1) with the following settings:<br>
Multiple alignment parameter:<br>
Gap opening penalty	10<br>
Gap extension penalty	10<br>
Gap separation penalty range	8<br>
Gap separation penalty	off<br>
% identity for alignment delay	40<br>
Residue specific gaps	off<br>
Hydrophilic residue gap	off<br>
Transition weighing	0<br>
Pairwise alignment parameter:<br>
FAST algorithm	on<br>
K-tuple       size	1<br>
Gap penalty	3<br>
Window size	5<br>
Number of best diagonals	5<br>
Alternatively the identity may be determined according to Chenna, Ramu, Suga-wara, Hideaki, Koike.Tadashi, Lopez, Rodrigo, Gibson, Toby J, Higgins, Desmond G, Thompson, Julie D. Multiple sequence alignment with the Clustal series of programs.<br><br>
27<br>
(2003)      Nucleic      Acids      Res      31       (13):3497-500,       the      web      page: http://www.ebi.ac.Uk/Tools/clustalw/index.html# and the following settings<br><br>
DNA Gap Open Penalty	15.0<br>
DNA Gap Extension Penalty	6.66<br>
DNA Matrix	Identity<br>
Protein Gap Open Penalty	10.0<br>
Protein Gap Extension Penalty	0.2<br>
Protein matrix	Gonnet<br>
Protein/DNA ENDGAP	-1<br>
Protein/DNA GAPDIST	4<br>
All the nucleic acid sequences mentioned herein (single-stranded and double-stranded DNA and RNA sequences, for example cDNA and mRNA) can be produced in a known way by chemical synthesis from the nucleotide building blocks, e.g. by fragment condensation of individual overlapping, complementary nucleic acid building blocks of the double helix. Chemical synthesis of oligonucleotides can, for example, be performed in a known way, by the phosphoamidite method (Voet, Voet, 2nd edition, Wiley Press, New York, pages 896-897). The accumulation of synthetic oligonucleotides and filling of gaps by means of the Klenow fragment of DNA polymerase and ligation reactions as well as general cloning techniques are described in Sambrook et al. (1989), see below.<br>
The invention also relates to nucleic acid sequences (single-stranded and double-stranded DNA and RNA sequences, e.g. cDNA and mRNA), coding for one of the above polypeptides and their functional equivalents, which can be obtained for ex¬ample using artificial nucleotide analogs.<br>
The invention relates both to isolated nucleic acid molecules, which code for polypeptides or proteins according to the invention or biologically active segments thereof, and to nucleic acid fragments, which can be used for example as hybridization probes or primers for identifying or amplifying coding nucleic acids according to the in¬vention.<br>
The nucleic acid molecules according to the invention can in addition contain non-translated sequences from the 3' and/or 5' end of the coding genetic region.<br><br>
28<br>
The invention further relates to the nucleic acid molecules that are complementary to the concretely described nucleotide sequences or a segment thereof.<br>
The nucleotide sequences according to the invention make possible the production of probes and primers that can be used for the identification and/or cloning of homologous sequences in other cellular types and organisms. Such probes or primers generally comprise a nucleotide sequence region which hybridizes under "stringent" conditions (see below) on at least about 12, preferably at least about 25, for example about 40, 50 or 75 successive nucleotides of a sense strand of a nucleic acid sequence according to the invention or of a corresponding antisense strand.<br>
An "isolated" nucleic acid molecule is separated from other nucleic acid molecules that are present in the natural source of the nucleic acid and can moreover be substantially free from other cellular material or culture medium, if it is being produced by recombinant techniques, or can be free from chemical precursors or other chemicals, if it is being synthesized chemically.<br>
A nucleic acid molecule according to the invention can be isolated by means of standard techniques of molecular biology and the sequence information supplied according to the invention. For example, cDNA can be isolated from a suitable cDNA library, using one of the concretely disclosed complete sequences or a segment thereof as hybridization probe and standard hybridization techniques (as described for example in Sambrook, J., Fritsch, E.F. and Maniatis, T. Molecular Cloning: A Laboratory Manual. 2nd edition, Cold Spring Harbor Laboratory, Cold Spring Harbor Laboratory Press, Cold Spring Harbor, NY, 1989). In addition, a nucleic acid molecule comprising one of the disclosed sequences or a segment thereof, can be isolated by the polymerase chain reaction, using the oligonucleotide primers that were constructed on the basis of this sequence. The nucleic acid amplified in this way can be cloned in a suitable vector and can be characterized by DNA sequencing. The oligonucleotides according to the invention can also be produced by standard methods of synthesis, e.g. using an automatic DNA synthesizer.<br>
Nucleic acid sequences according to the invention or derivatives thereof, homo-logues or parts of these sequences, can for example be isolated by usual hybridization techniques or the PCR technique from other bacteria, e.g. via genomic or cDNA librar¬ies. These DNA sequences hybridize in standard conditions with the sequences ac¬cording to the invention.<br><br>
29<br>
"Hybridize" means the ability of a polynucleotide or oligonucleotide to bind to an<br>
almost complementary sequence in standard conditions, whereas nonspecific binding does not occur between non-complementary partners in these conditions. For this, the sequences can be 90-100% complementary. The property of complementary sequences of being able to bind specifically to one another is utilized for example in Northern Blotting or Southern Blotting or in primer binding in PCR or RT-PCR.<br>
Short oligonucleotides of the conserved regions are used advantageously for hybridization. However, it is also possible to use longer fragments of the nucleic acids according to the invention or the compiete sequences for the hybridization. These standard conditions vary depending on the nucleic acid used (oligonucleotide, longer fragment or complete sequence) or depending on which type of nucleic acid - DNA or RNA - is used for hybridization. For example, the melting temperatures for DNA:DNA hybrids are approx. 10°C lower than those of DNA:RNA hybrids of the same length.<br>
For example, depending on the particular nucleic acid, standard conditions mean temperatures between 42 and 58°C in an aqueous buffer solution with a concentration between 0.1 to 5 x SSC (1 X SSC = 0.15 M NaCI, 15 mM sodium citrate, pH 7.2) or additionally in the presence of 50% formamide, for example 42°C in 5 x SSC, 50% formamide. Advantageously, the hybridization conditions for DNA:DNA hybrids are 0.1 x SSC and temperatures between about 20°C to 45°C, preferably between about 30°C to 45°C. For DNArRNA hybrids the hybridization conditions are advantageously 0.1 x SSC and temperatures between about 30°C to 55°C, preferably between about 45°C to 55°C. These stated temperatures for hybridization are examples of calculated melting temperature values for a nucleic acid with a length of approx. 100 nucleotides and a G + C content of 50% in the absence of formamide. The experimental conditions for DNA hybridization are described in relevant genetics textbooks, for example Sambrook et al., 1989, and can be calculated using formulae that are known by a person skilled in the art, for example depending on the length of the nucleic acids, the type of hybrids or the G + C content. A person skilled in the art can obtain further information on hybridization from the following textbooks: Ausubel et al. (eds), 1985, Current Protocols in Molecular Biology, John Wiley &amp; Sons, New York; Hames and Higgins (eds), 1985, Nucleic Acids Hybridization: A Practical Approach, IRL Press at Oxford University Press, Oxford; Brown (ed), 1991, Essential Molecular Biology: A Practical Approach, IRL Press at Oxford University Press, Oxford.<br><br>
30<br>
"Hybridization" can in particular be carried out under stringent conditions. Such hybridization conditions are for example described in Sambrook, J., Fritsch, E.F., Ma-niatis, T., in: Molecular Cloning (A Laboratory Manual), 2nd edition, Cold Spring Harbor Laboratory Press, 1989, pages 9.31-9.57 or in Current Protocols in Molecular Biology, John Wiley &amp; Sons, N.Y. (1989), 6.3.1-6.3.6.<br>
"Stringent" hybridization conditions mean in particular: Incubation at 42°C over¬night in a solution consisting of 50% formamide, 5 x SSC (750 mM NaCI, 75 mM tri-sodium citrate), 50 mM sodium phosphate (pH 7.6), 5x Denhardt Solution, 10% dextran sulfate and 20 g/ml denatured, sheared salmon sperm DNA, followed by washing of the filters with 0.1x SSC at 65°C.<br>
The invention also relates to derivatives of the concretely disclosed or derivable nucleic acid sequences.<br>
Thus, further nucleic acid sequences according to the invention can be derived from the sequences specifically disclosed herein and can differ from it by addition, sub¬stitution, insertion or deletion of individual or several nucleotides, and furthermore code for polypeptides with the desired profile of properties.<br>
The invention also encompasses nucleic acid sequences that comprise so-called silent mutations or have been altered, in comparison with a concretely stated se¬quence, according to the codon usage of a special original or host organism, as well as naturally occurring variants, e.g. splicing variants or allelic variants, thereof.<br>
It also relates to sequences that can be obtained by conservative nucleotide substitutions (i.e. the amino acid in question is replaced by an amino acid of the same charge, size, polarity and/or solubility).<br>
The invention also relates to the molecules derived from the concretely disclosed nucleic acids by sequence polymorphisms. These genetic polymorphisms can exist between individuals within a population owing to natural variation. These natural varia¬tions usually produce a variance of 1 to 5% in the nucleotide sequence of a gene.<br>
Derivatives of nucleic acid sequences according to the invention mean for ex¬ample allelic variants, having at least 60% homology at the level of the derived amino acid, preferably at least 80% homology, quite especially preferably at least 90% ho¬mology over the entire sequence range (regarding homology at the amino acid level, reference should be made to the details given above for the polypeptides). Advanta¬geously, the homologies can be higher over partial regions of the sequences.<br><br>
31<br>
Furthermore, derivatives are also to be understood to be homologues of the nucleic acid sequences according to the invention, for example animal, plant, fungal or bacterial homologues , shortened sequences, single-stranded DNA or RNA of the coding and noncoding DNA sequence. For example, homologues have, at the DNA level, a homology of at least 40%, preferably of at least 60%, especially preferably of at least 70%, quite especially preferably of at least 80% over the entire DNA region given in a sequence specifically disclosed herein.<br>
Moreover, derivatives are to be understood to be, for example, fusions with promoters. The promoters that are added to the stated nucleotide sequences can be modified by at least one nucleotide exchange, at least one insertion, inversion and/or deletion, though without impairing the functionality or efficacy of the promoters. Moreover, the efficacy of the promoters can be increased by altering their sequence or can be exchanged completely with more effective promoters even of organisms of a different genus.<br>
3.3    Constructs according to the invention<br>
The invention also relates to expression constructs, containing, under the genetic control of regulatory nucleic acid sequences, a nucleic acid sequence coding for a polypeptide or fusion protein according to the invention; as well as vectors comprising at least one of these expression constructs.<br>
"Expression unit" means, according to the invention, a nucleic acid with expression activity, which comprises a promoter as defined herein and, after functional association with a nucleic acid that is to be expressed or a gene, regulates the expression, i.e. the transcription and the translation of this nucleic acid or of this gene. In this context, therefore, it is also called a "regulatory nucleic acid sequence". In addition to the promoter, other regulatory elements may be present, e.g. enhancers.<br>
"Expression cassette" or "expression construct" means, according to the inven¬tion, an expression unit, which is functionally associated with the nucleic acid that is to be expressed or the gene that is to be expressed. In contrast to an expression unit, an expression cassette thus comprises not only nucleic acid sequences which regulate transcription and translation, but also the nucleic acid sequences which should be ex¬pressed as protein as a result of the transcription and translation.<br>
The terms "expression" or "overexpression" describe, in the context of the inven¬tion, the production or increase of intracellular activity of one or more enzymes in a mi-<br><br>
32<br>
croorganism, which are encoded by the corresponding DNA. For this, it is possible for example to insert a gene in an organism, replace an existing gene by another gene, increase the number of copies of the gene or genes, use a strong promoter or use a gene that codes for a corresponding enzyme with a high activity, and optionally these measures can be combined.<br>
Preferably such constructs according to the invention comprise a promoter 5'-upstream from the respective coding sequence, and a terminator sequence 3'-downstream, and optionally further usual regulatory elements, in each case functionally associated with the coding sequence.<br>
A "promoter", a "nucleic acid with promoter activity" or a "promoter sequence" mean, according to the invention, a nucleic acid which, functionally associated with a nucleic acid that is to be transcribed, regulates the transcription of this nucleic acid.<br>
"Functional" or "operative" association means, in this context, for example the sequential arrangement of one of the nucleic acids with promoter activity and of a nu¬cleic acid sequence that is to be transcribed and optionally further regulatory elements, for example nucleic acid sequences that enable the transcription of nucleic acids, and for example a terminator, in such a way that each of the regulatory elements can fulfill its function in the transcription of the nucleic acid sequence. This does not necessarily require a direct association in the chemical sense. Genetic control sequences, such as enhancer sequences, can also exert their function on the target sequence from more remote positions or even from other DNA molecules. Arrangements are preferred in which the nucleic acid sequence that is to be transcribed is positioned behind (i.e. at the 3' end) the promoter sequence, so that the two sequences are bound covalently to one another. The distance between the promoter sequence and the nucleic acid se¬quence that is to be expressed transgenically can be less than 200 bp (base pairs), or less than 100 bp or less than 50 bp.<br>
Apart from promoters and terminators, examples of other regulatory elements that may be mentioned are targeting sequences, enhancers, polyadenylation signals, selectable markers, amplification signals, replication origins and the like. Suitable regu¬latory sequences are described for example in Goeddel, Gene Expression Technology: Methods in Enzymology 185, Academic Press, San Diego, CA (1990).<br>
Nucleic acid constructs according to the invention comprise in particular sequences selected from those, specifically mentioned herein or derivatives and homologues thereof, as well as the nucleic acid sequences that can be derived from<br><br>
33<br>
amino acid sequences specifically mentioned herein which are advantageously associated operatively or functionally with one or more regulating signal for controlling, e.g. increasing, gene expression.<br>
In addition to these regulatory sequences, the natural regulation of these sequences can still be present in front of the actual structural genes and optionally can have been altered genetically, so that natural regulation is switched off and the expression of the genes has been increased. The nucleic acid construct can also be of a simpler design, i.e. without any additional regulatory signals being inserted in front of the coding sequence and without removing the natural promoter with its regulation. Instead, the natural regulatory sequence is silenced so that regulation no longer takes place and gene expression is increased.<br>
A preferred nucleic acid construct advantageously also contains one or more of the aforementioned enhancer sequences, functionally associated with the promoter, which permit increased expression of the nucleic acid sequence. Additional advantageous sequences, such as other regulatory elements or terminators, can also be inserted at the 3' end of the DNA sequences. One or more copies of the nucleic acids according to the invention can be contained in the construct. The construct can also contain other markers, such as antibiotic resistances or auxotrophy-complementing genes, optionally for selection on the construct.<br>
Examples of suitable regulatory sequences are contained in promoters such as cos-, tac-, trp-, tet-, trp-tet-, Ipp-, lac-, Ipp-lac-, laclq"' T7-, T5-, T3-, gal-, trc-, ara-, rhaP (rhaPBAD)SP6-, lambda-PR- or in the lambda-PL promoter, which find application advantageously in Gram-negative bacteria. Other advantageous regulatory sequences are contained for example in the Gram-positive promoters ace, amy and SP02, in the yeast or fungal promoters ADC1, MFalpha, AC, P-60, CYC1, GAPDH, TEF, rp28, ADH. Artificial promoters can also be used for regulation.<br>
For expression, the nucleic acid construct is inserted in a host organism advantageously in a vector, for example a plasmid or a phage, which permits optimum expression of the genes in the host. In addition to plasmids and phages, vectors are also to be understood as meaning all other vectors known to a person skilled in the art, e.g. viruses, such as SV40, CMV, baculovirus and adenovirus, transposons, IS elements, phasmids, cosmids, and linear or circular DNA. These vectors can be replicated autonomously in the host organism or can be replicated chromosomally. These vectors represent a further embodiment of the invention.<br><br>
34<br>
Suitable plasmids are, for example in E. coli, pLG338, pACYC184, pBR322,<br>
pUC18, pUC19, pKC30, pRep4, pHS1, pKK223-3, pDHE19.2, pHS2, pPLc236, pMBL24, pLG200, pUR290, plN-lll113-B1, Agt11 or pBdCI; in nocardioform actinomycetes pJAM2; in Streptomyces plJ101, plJ364, plJ702 or plJ361; in bacillus pUB110, pC194 or pBD214; in Corynebacterium pSA77 or pAJ667; in fungi pALS1, plL2 or pBB116; in yeasts 2alphaM, pAG-1, YEp6, YEp13 or pEMBLYe23 or in plants pLGV23, pGHIac+, pBIN19, pAK2004 or pDH51. The aforementioned plasmids represent a small selection of the possible plasmids. Other plasmids are well known to a person skilled in the art and will be found for example in the book Cloning Vectors (Eds. Pouwels P.H. et al. Elsevier, Amsterdam-New York-Oxford, 1985, ISBN 0 444 904018).<br>
In a further embodiment of the vector, the vector containing the nucleic acid construct according to the invention or the nucleic acid according to the invention can be inserted advantageously in the form of a linear DNA in the microorganisms and integrated into the genome of the host organism through heterologous or homologous recombination. This linear DNA can comprise a linearized vector such as plasmid or just the nucleic acid construct or the nucleic acid according to the invention.<br>
For optimum expression of heterologous genes in organisms, it is advantageous to alter the nucleic acid sequences in accordance with the specific codon usage employed in the organism. The codon usage can easily be determined on the basis of computer evaluations of other, known genes of the organism in question.<br>
The production of an expression cassette according to the invention is based on fusion of a suitable promoter with a suitable coding nucleotide sequence and a terminator signal or polyadenylation signal. Common recombination and cloning techniques are used for this, as described for example in T. Maniatis, E.F. Fritsch and J. Sambrook, Molecular Cloning: A Laboratory Manual, Cold Spring Harbor Laboratory, Cold Spring Harbor, NY (1989) as well as in T.J. Silhavy, M.L. Berman and L.W. Enquist, Experiments with Gene Fusions, Cold Spring Harbor Laboratory, Cold Spring Harbor, NY (1984) and in Ausubel, F.M. et al., Current Protocols in Molecular Biology, Greene Publishing Assoc, and Wiley Interscience (1987).<br>
The recombinant nucleic acid construct or gene construct is inserted advanta¬geously in a host-specific vector for expression in a suitable host organism, to permit optimum expression of the genes in the host. Vectors are well known to a person<br><br>
35<br>
skilled in the art and will be found for example in "Cloning Vectors" (Pouwels P.H. et al., Publ. Elsevier, Amsterdam-New York-Oxford, 1985).<br>
3.4    Hosts that can be used according to the invention<br>
Depending on the context, the term "microorganism" means the starting micro¬organism (wild-type) or a genetically modified microorganism according to the inven¬tion, or both.<br>
The term "wild-type" means, according to the invention, the corresponding start¬ing microorganism, and need not necessarily correspond to a naturally occurring or¬ganism.<br>
By means of the vectors according to the invention, recombinant microorgan¬isms can be produced, which have been transformed for example with at least one vec¬tor according to the invention and can be used for production of the polypeptides ac¬cording to the invention. Advantageously, the recombinant constructs according to the invention, described above, are inserted in a suitable host system and expressed. Preferably, common cloning and transfection methods that are familiar to a person skilled in the art are used, for example co-precipitation, protoplast fusion, electropora-tion, retroviral transfection and the like, in order to secure expression of the stated nu¬cleic acids in the respective expression system. Suitable systems are described for ex-, ample in Current Protocols in Molecular Biology, F. Ausubel et al., Publ. Wiley Inter-science, New York 1997, or Sambrook et al. Molecular Cloning: A Laboratory Manual. 2nd edition, Cold Spring Harbor Laboratory, Cold Spring Harbor Laboratory Press, Cold Spring Harbor, NY, 1989.<br>
In principle, all prokaryotic organisms can be considered as recombinant host organisms for the nucleic acid according to the invention or the nucleic acid construct. Bacteria are used advantageously as host organisms. Preferably they are selected from native or recombinant bacteria having the ability to produce inclusion bodies of the PHA-, TAG- or WE-type, as in particular the TAG-producing nocardioform actinomy-cetes, in particular of the genus Rhodococcus, Mycobacterium, Nocardia, Gordonia, Skermania and Tsukamurella; as well as TAG-producing Streptomycetes; WE-producing genera Acinetobacter and Alcanivorax; as well as recombinant strains of the genus Escherichia, especially E. coli, Corynebacterium, especially C. glutamicum und Bacillus, especially B. subtilis.<br><br>
36<br>
The host organism or host organisms according to the invention then preferably contain at least one of the nucleic acid sequences, nucleic acid constructs or vectors described in this invention, which code for an enzyme activity according to the above definition.<br>
The organisms used in the method according to the invention are grown or bred in a manner familiar to a person skilled in the art, depending on the host organism. As a rule, microorganisms are grown in a liquid medium, which contains a source of carbon, generally in the form of sugars, a source of nitrogen generally in the form of organic sources of nitrogen such as yeast extract or salts such as ammonium sulfate, trace elements such as iron, manganese and magnesium salts and optionally vitamins, at temperatures between 0°C and 100°C, preferably between 10°C to 60°C with oxygen aeration. The pH of the liquid nutrient medium can be maintained at a fixed value, i.e. regulated or not regulated during growing. Growing can be carried out batchwise, semi-batchwise or continuously. Nutrients can be supplied at the start of fermentation or can be supplied subsequently, either semi-continuously or continuously.<br>
3.5    Recombinant production proteins with decarboxylase activity<br>
The invention also relates to methods for production of proteins according to the invention by cultivating a microorganism which expresses said protein, and isolating the desired product from the culture.<br>
The microorganisms as used according to the invention can be cultivated con¬tinuously or discontinuously in the batch process or in the fed batch or repeated fed batch process. A review of known methods of cultivation will be found in the textbook by Chmiel (Bioprocesstechnik 1. Einfuhrung in die Bioverfahrenstechnik (Gustav Fi¬scher Verlag, Stuttgart, 1991)) or in the textbook by Storhas (Bioreaktoren und periphe-re Einrichtungen (Vieweg Verlag, Braunschweig/Wiesbaden, 1994)).<br>
The culture medium that is to be used must satisfy the requirements of the par¬ticular strains in an appropriate manner. Descriptions of culture media for various mi¬croorganisms are given in the handbook "Manual of Methods for General Bacteriology" of the American Society for Bacteriology (Washington D. C, USA, 1981).<br>
These media that can be used according to the invention generally comprise one or more sources of carbon, sources of nitrogen, inorganic salts, vitamins and/or trace elements.<br><br>
37<br>
Preferred sources of carbon are sugars, such as mono-, di- or polysaccharides. Very good sources of carbon are for example glucose, fructose, mannose, galactose, ribose, sorbose, ribulose, lactose, maltose, sucrose, raffinose, starch or cellulose. Sug¬ars can also be added to the media via complex compounds, such as molasses, or other by-products from sugar refining. It may also be advantageous to add mixtures of various sources of carbon. Other possible sources of carbon are oils and fats such as soybean oil, sunflower oil, peanut oil and coconut oil, fatty acids such as palmitic acid, stearic acid or linoleic acid, alcohols such as glycerol, methanol or ethanol and organic acids such as acetic acid or lactic acid.<br>
Sources of nitrogen are usually organic or inorganic nitrogen compounds or ma¬terials containing these compounds. Examples of sources of nitrogen include ammonia gas or ammonium salts, such as ammonium sulfate, ammonium chloride, ammonium phosphate, ammonium carbonate or ammonium nitrate, nitrates, urea, amino acids or complex sources of nitrogen, such as corn-steep liquor, soybean flour, soybean pro¬tein, yeast extract, meat extract and others. The sources of nitrogen can be used sepa¬rately or as a mixture.<br>
Inorganic salt compounds that may be present in the media comprise the chlo¬ride, phosphate or sulfate salts of calcium, magnesium, sodium, cobalt, molybdenum, potassium, manganese, zinc, copper and iron.<br>
Inorganic sulfur-containing compounds, for example sulfates, sulfites, dithionites, tetrathionates, thiosulfates, sulfides, but also organic sulfur compounds, such as mer-captans and thiols, can be used as sources of sulfur.<br>
Phosphoric acid, potassium dihydrogenphosphate or dipotassium hydrogen-phosphate or the corresponding sodium-containing salts can be used as sources of phosphorus.<br>
Chelating agents can be added to the medium, in order to keep the metal ions in solution. Especially suitable chelating agents comprise dihydroxyphenols, such as catechol or protocatechuate, or organic acids, such as citric acid.<br>
The fermentation media used according to the invention may also contain other growth factors, such as vitamins or growth promoters, which include for example biotin, riboflavin, thiamine, folic acid, nicotinic acid, pantothenate and pyridoxine. Growth fac¬tors and salts often come from complex components of the media, such as yeast ex¬tract, molasses, corn-steep liquor and the like. In addition, suitable precursors can be added to the culture medium. The precise composition of the compounds in the me-<br><br>
38<br>
dium is strongly dependent on the particular experiment and must be decided individu¬ally for each specific case. Information on media optimization can be found in the text¬book "Applied Microbiol. Physiology, A Practical Approach" (Publ. P.M. Rhodes, P.F. Stanbury, IRL Press (1997) p. 53-73, ISBN 0 19 963577 3). Growing media can also be obtained from commercial suppliers, such as Standard 1 (Merck) or BHI (Brain heart infusion, DIFCO) etc.<br>
All components of the medium are sterilized, either by heating (20 min at 1.5 bar and 121°C) or by sterile filtration. The components can be sterilized either together, or if necessary separately. All the components of the medium can be present at the start of growing, or optionally can be added continuously or by batch feed.<br>
The temperature of the culture is normally between 15°C and 45°C, preferably 25°C to 40°C and can be kept constant or can be varied during the experiment. The pH value of the medium should be in the range from 5 to 8.5, preferably around 7.0. The pH value for growing can be controlled during growing by adding basic compounds such as sodium hydroxide, potassium hydroxide, ammonia or ammonia water or acid compounds such as phosphoric acid or sulfuric acid. Antifoaming agents, e.g. fatty acid polyglycol esters, can be used for controlling foaming. To maintain the stability of plas-mids, suitable substances with selective action, e.g. antibiotics, can be added to the medium. Oxygen or oxygen-containing gas mixtures, e.g. the ambient air, are fed into the culture in order to maintain aerobic conditions. The temperature of the culture is normally from 20°C to 45°C. Culture is continued until a maximum of the desired prod¬uct has formed. This is normally achieved within 10 hours to 160 hours.<br>
The cells can be disrupted optionally by high-frequency ultrasound, by high pressure, e.g. in a French pressure cell, by osmolysis, by the action of detergents, lytic enzymes or organic solvents, by means of homogenizers or by a combination of sev¬eral of the methods listed.<br>
The following examples only serve to illustrate the invention. The numerous possible variations that are obvious to a person skilled in the art also fall within the scope of the invention.<br>
Experimental Part<br>
1. General information<br><br>
39<br>
a)	Instruments and material<br>
All chemicals (if not indicated otherwise) were purchased from Sigma-Aldrich. Xho\ and A/del restriction enzymes were purchased from Roche and T4 ligase from New England Biolabs. 1H NMR analysis was performed on Bruker 200 or 400 MHz spectrometers. ESI-MS analysis was carried out on a Bruker Esquire 3000+ ion trap MS, equipped with an electrospray ionisation source run in positive mode. Samples of protein for ESI-MS analysis were desalted by dialysis for 4h in ddH20.<br>
b)	Sequence similarity searches and identification of new AMDases.<br>
In order to find new malonate decarboxylases the protein sequence of AMDase from Bordetella bronchiseptica was aligned with sequences of known proteins from public databases, using the Basic Local Alignment Search Tool (BLAST). During the searches the following parameters were used: database - non-redundant protein se¬quences, algorithm - blastp (protein-protein BLAST).<br>
From the list of hits of the most similar proteins to AMDase from B. bronchisep¬tica the following bacterial enzymes were selected:<br>
protein EAN08019.1 from Mesorhizobium sp. BNC1,<br>
protein BAA30082.1 from Pyrococcus horikoshii OT3,<br>
protein NP_888076.1 from Bordetella bronchiseptica RB50 and<br>
protein ZP_00801202.1 from Alkaliphilus metalliredigenes QYMF.<br>
c)	Sub-cloning of AMDases and other enzymes.<br>
Genes coding for the following enzymes:<br>
AMDase from Bordetella bronchiseptica,<br>
protein EAN08019.1 from Mesorhizobium sp. BNC1,<br>
protein BAA30082.1 from Pyrococcus horikoshii OT3,<br>
protein NP_888076.1 from Bordetella bronchiseptica RB50 and<br>
protein ZP_00801202.1 from Alkaliphilus metalliredigenes QYMF and<br>
the glutamate racemase (AAF25672) from Aquifex pyrophilus<br>
were optimized for expression in E. coli, synthesised and cloned into the pGA4 vector, containing the ampicillin resistance gene (GENEART AG., Germany). The genes were<br><br>
40<br>
then sub-cloned into pET30b vector (Novagen, Cat.No. 69910-3, EMD Chemicals, Inc., San Diogo, USA); containing the kanamycin resistance gene, using Xho\ and Nde\ re¬striction sites. The ligation product was transformed into BL21(DE3) electrocompetent E. coli cells, which were plated out on Kan50-LB-Agar plates. Plasmids were then iso¬lated from the cells and successful sub-cloning was confirmed by sequencing. This sub-cloning allowed easy expression and purification (proteins were 6xHis-Tagged) of targeted enzymes in E. coli. After sub-cloning into pET30b vector, the sequence of all enzymes at the C-terminus was extended by the following amino acids: Leu-Glu-His-His-His-His-His-His.<br>
Nucleotide sequences according to SEQ ID NO: 1, 3, 5, 7, 9, and 11 represent those optimized sequences coding for his-tagged enzymes.<br>
d) Expression and purification of His6-fusion proteins.<br>
Plasmids encoding AMDases, containing a T7 RNA polymerase promoter and a kanamycin resistance gene were transformed into electrocompetent E. coli strain BL21 (DE3). Bacteria were initially incubated in growing medium (1% tryptone, 0.5% yeast extract, 1% sodium chloride, 0.4% glucose, 0.20 x M9 buffer (for 1 liter: KH2P04 3 g, Na2HP04 6 g, NaCI 5 g, MgS04 (1M) 1 ml, dH20 up to 1 liter), 50 mg/mL kanamycin) for approximately 4h at a temperature of 37 °C to achieve OD6oo = 1. Addition of iso-propyl-P-D-thiogalactopyranoside (1mM, final concentration) induced protein expres¬sion. After overnight incubation at 30 °C the cells were pelleted (4500 rpm for 15 min¬utes, 4 °C) and lysed at room temperature after the addition of hen egg white lysozyme (0.1 mg/1ml_ of buffer) in 50 mM TRIS buffer (pH 8) containing 25 mM NaCI, 2 mM EDTA, 0.2 mM DTT and the protease inhibitor - phenylmethanesulfonyl fluoride (10 ug/mL). After lysis, nucleic acids were precipitated by the addition of 2% streptomycin-sulfate followed by centrifugation (4500 rpm for 20 minutes, 4 °C). The supernatant, containing crude AMDase, was dialyzed (Spectra/Por 2, MWCO: 12-14 000) overnight against 25 mM TRIS buffer (pH 8) containing 25 mM NaCI, resulting in His6 fusion pro¬teins that were easily purified by FPLC on a GE Healthcare Bio-Science HiTrap Chelat¬ing 5ml_ column.. Elution of proteins was achieved by the use of a step gradient of so¬dium phosphate buffer (0.02 M, pH 7.4), NaCI (0.5 M) and imidazol (25, 60 &amp; 250 mM).The target protein was eluted at 250 mM concentration of imidazol. This proce¬dure yielded up to 15 mg of protein per 250 mL of culture and was typically 95% pure<br><br>
41<br>
by SDS-PAGE. The final protein concentration was calculated by using either Bradford assay or spectrophotometrical reading at OD2so-<br>
e)	Site-directed mutagenesis of AMDase from Bordetella bronchiseptica.<br>
Site-directed mutagenesis was carried out using the Pfu Turbo polymerase (Stratagene) following the manufacturer instructions for the Stratagene QuikChange® Site-directed mutagenesis kit. PCR machine conditions were as follows: 1 segment cy¬cle: 95 °C for 2 min; 2 segment cycle (repeated 30 times): 95 °C for 30 s, 55 °C for 30 s, and 72 °C for 5.5 min; 3 segment cycle: 4°C for 10 min. After PCR, Dpn I restriction enzyme was added to digest the non-mutated plasmid. Mutated DNA was transferred into E. coli BL21 (DE3) and proteins were expressed following the procedure presented above for the wild type. Mutagenesis of other decarboxylase enzymes of the invention can be performed in anology.<br>
f)	In vivo plate assay to determine AMDase activity.<br>
Test Petri dishes were prepared using following medium: 1% tryptone, 0.5% yeast extract, 1% sodium chloride, 1 % agar, 0.025% bromothymol blue (BTB). Then pH was adjusted to 6.5 and the medium autoclaved. When medium temperature reached 50-60 °C isopropyl-P-D-thiogalactopyranoside (1mM, final concentration), kanamycin (40 mg/mL, final concentration) and phenylmalonic acid (0.36%, final con¬centration) were added. 100 pi of liquid bacterial culture containing AMDase was plated on test Petri dishes. After 2-3 days of incubation at 37 °C the colonies and surrounding area changed colour to green-blue indicating AMDase activity.<br>
g)	Determination of kinetic constants.<br>
This method involves measuring spectrophotometrically at 620 nm the change of yellow colour to green/blue of pH indicator - BTB with increasing concentration of [OH"] formed during the decarboxylation reaction. To ensure that the increase of colour is linear with the increase of pH the measurement should be done around the pKa of the pH indicator (pKa of BTB = 7.3).17 We found that 5 mM (and/or 10mM) MOPS buffer pH 7.2 was suitable. More concentrated buffer (10 mM) gives better results with higher concentration of enzyme or quicker reactions since changes of colour are slower due to higher buffer capacity.<br><br>
42<br>
All kinetics assays were performed on an Anthos Zenyth 3100 96-well microplate reader. 10 pi of 20-500 mM phenylmalonate or methyl-phenylmalonate solution (pH adjusted to 7.2) was added to 180 pi of 5-1 OmM MOPS buffer pH 7.2 containing 0.0186mM bromothymol blue. The reaction was initiated by addition of 10 pi of decar¬boxylase solution (0.002-0.02 mM, final concentration). The sensitivity of the reaction depends on the concentration of buffer, lower concentrations give higher sensitivity. The values of KM and kcat were calculated from spectophotometrical measurements at 620 nm of the resulting blue solution using Lineweaver-Burk plot and the following equations:<br>
dA rate of reaction = — * Q * reaction volume where dt<br>
*- MOPS buffer	1<br>
Q = ^	   TZ	Vj and Ae616nm = 15703.79M"1 cm"1.17'18<br>
^ BTB indicator       nt,620nm      l<br>
h) In vitro decarboxylation of 2-methyl-2-phenylmalonate to 2-phenylpropanoic acid by AMDase from Bordetella bronchiseptica and Asp/Glu racemase from Mesorhizobium sp. BNC1.<br>
To 400 pi of 0.1 M TRIS buffer pH 8, 100 pi of 0.5 M water solution (after adjust¬ing pH to 8) of 2-methyl-2-phenylmalonic acid was added. Then 20 pi of an AMDase solution (10 mg/mL) was added and the reaction was stirred at 30 °C for 8h. Products were extracted with diethyl ether after acidifying with 1 M HCI and methylated with TMS-diazomethane. The reaction was analyzed by GC (conversion up to 95%) and 2-phenylpropanoic acid formation confirmed by GC-MS, 1H NMR. Both enzyme products (after methylation) had the same configuration (the same peak on GC with chiral col¬umn, ee &gt;99%). Thus according to the literature7 absolute configuration of the product was R.<br>
i) Substrates for enzymatic reactions.<br>
Phenylmalonate (1) was purchased from Sigma-Aldrich<br>
2-Ethyl-2-phenylmalonate was a kind gift from BASF SE. Diethyl 2-methyl-2-phenylmalonate (2a) was synthesized from diethyl phenylmalo¬nate in reaction with iodomethane. 10 g (42.3 mmol) of diethyl phenylmalonate was stirred for an hour under reflux in sodium ethanolate solution prepared by dissolving 1.6<br><br>
43<br>
g (69.5 mmol) metallic sodium in 60 mL of absolute ethanol. Then 6 mL (96.4 mmol) of Mel was added and the resulting mixture was stirred under reflux for another hour. Af¬ter reaction the solvent was removed under vacuum and the remaining residue was dissolved in diethyl ether and dried over anhydrous magnesium sulphate. Diethyl ether was evaporated and the final product (pale oil) was obtained with a yield of 98% and purity of 99% according to NMR analysis.<br>
2-Methyl-2-phenylmalonic acid (2) was synthesized from diethyl 2-methyl-2-phenylmalonate (2a) by hydrolysis in sodium hydroxide. 4g (16 mmol) of diethyl 2-methyl-2-phenylmalonate was dissolved in 100 mL of ethanol and 50 mL of 3.25 M aqueous solution of sodium hydroxide was added. The reaction was stirred overnight at room temperature. Ethanol was then evaporated under vacuum and the resulting white powder was dissolved in 10 mL of water, cooled on ice and acidified with 5 M HCI to pH 1. The product was extracted with 2x100 mL of diethyl ether and dried over anhy¬drous magnesium sulphate. Ether was evaporated to give the final product (white pow¬der) with a yield of 97% and purity of 99% as assayed by 1H NMR.<br>
Diethyl 2-propylidenemalonate (4a), diethyl 2-butylidenemalonate (5a) and diethyl 2-(2-methylpropylidene)malonate (6a) were prepared by reaction of diethyl malonate with propionaldehyde, butyraldehyde and isobutyraldehyde respectively. 16g (0.1 mol) of diethylmalonate was stirred together with 0.4ml of piperidine and 1.2ml of glacial acetic acid in 50 ml of toluene for 20 minutes. Then 0.12 mol of appropriate al¬dehyde was added and reaction was refluxed under Dean-Stark conditions for 4h. 100 ml of water acidified with hydrochloric acid was added to the reaction mixture and product was extracted with diethyl ether. Organic layer was washed few times with brine solution and dried over magnesium sulphate. All product were purified by flash column chromatography. Products were obtained with yield up to 80 % as colourless oils.<br>
Diethyl 2-ethylidenemalonate (3a) was purchased from Sigma-Aldrich.<br>
Diethyl 2-methyl-2 -vinyl malonate (3b), diethyl 2-methyl-2-((£)-prop-1-enyl)malonate (4b), diethyl 2-methyl-2-((E)-but-1-enyl)malonate (5b) and diethyl 2-methyl-2-(2-methylprop-1-enyl)malonate (6b) were obtained the same manner as shown below in diethyl 2-methyl-2-vinylmalonate (3b) synthesis. 2.4 ml of diisopropylamine in 15ml anhydrous THF was cooled to -78°C and 10.5 ml of BuLi (2.5M) in hexane was added dropwise. After 20 minutes 7.5 ml of DMPU and 2.8 g (0.014 mol) of diethyl 2-ethylidenemalonate (3a) was added. Reaction was stirred at<br><br>
44<br>
-78°C for 45 minutes and then 3.8g of iodomethane was added. Reaction was left stirring to reach RT within 1h and then was refluxed for 30 minutes. 70 ml of water acidified with hydrochloric acid was added to the reaction mixture and product was ex¬tracted with diethyl ether. Product was purified by flash column chromatography. Final ester was obtained as a colourless oil with 30% of yield.2-methyl-2-vinylmalonic acid (3), diethyl 2-methyl-2-((£)-prop-1-enyl)malonic acid (4), diethyl 2-methyl-2-((£)-but-1-enyl) malonic acid (5) and diethyl 2-methyl-2-(2-methylprop-1-enyl)malonic acid (6) were synthesised from esters the same manner as described for 2-methyl-2-phenyl malonic acid (2) above.<br>
k) 180 labelling experiments.<br>
Step I: 10 pi (40 pmol) of diethyl 2-methyl-2-phenylmalonate (5) was mixed with 10 pi of DMSO and suspended in 500 pi of phosphate buffer pH 7 (for H2180 experi¬ment: 0.5 mL buffer was evaporated to dryness and the remaining powder re-dissolved in 0.5ml_ of H2180 and "labelled" buffer used for reaction) containing 5 mg of porcine liver esterase (15 units/mg lyophilized powder, Sigma-Aldrich). The reaction was stirred overnight at 30 °C. Then the reaction was cooled down to 4 °C, acidified with 1M HCI to pH 1 and the product extracted with diethyl ether. Monoethyl 2-methyl-2-phenylmalonate was identified by 1H NMR and after methylation with TMS-diazomethane, by GC-MS (with isotopes analysis). The enantioselectivity of monoethyl 2-methyl-2-phenylmalonate was confirmed by HPLC-analysis on a chiral column Chiracel-OD. The separation of enantiomer peaks was achieved with a flow of 0.25 mL/min at 260 nm, by using a mixture of hexane-/so-propanol-trifluoroacetic acid (93:7:1.5) The formed product had configuration R according to literature data.14<br>
Step II: Product from step I was dissolved in 150 pi of dry ethanol and 55 pi of 8.75 M NaOH was added (for H2180 experiment: 100 mg of metallic sodium was dis¬solved in 55 pi of H2180, obtained Na18OH was used for reaction). The reaction was stirred for 2.5h at 30 °C. The reaction was evaporated to dryness and dissolved in cold (4 °C) 0.1 M TRIS buffer pH 8 (400 and pH pi) and pH was adjusted with 1M HCI to 8. 1/3 of the mixture was acidified with 1M HCI to pH 1 and the product extracted with di¬ethyl ether. 2-methyl-2-phenylmalonic was identified by 1H NMR and after methylation with TMS-diazomethane, by GC-MS (with isotopes analysis).<br>
Step III: To the remaining solution from step II, 20 pi of an AMDase solution (10 mg/mL) was added and the reaction mixture was stirred for 4.5h (overnight for AMDase<br><br>
45<br>
from Mesorhisobium sp.) at 30 °C. After this period the mixture was cooled down to 4<br>
°C acidified with 1M HCI to pH 1 and the final product extracted with diethyl ether. 2-phenylpropanoic acid was identified by 1H NMR and, after methylation with TMS-diazomethane, by GC-MS (with isotopes analysis). The enantioselectivity (ee &gt;99%) of the product (after methylation) was confirmed by GC with a chiral column.<br>
2. Examples<br>
Example 1: Overproduction and activity of the B. bronchiseptica AMDase.<br>
A synthetic gene encoding the B. bronchiseptica AMDase with codon optimiza¬tion for expression in E.coli, was sub-cloned into the pET30b vector. Expression in E. coii BL21 (DE3) strain following induction with IPTG resulted in the AMDase as a C-terminal His6-fusion protein. The enzyme was then purified by Ni-affinity chromatogra¬phy to greater than 95% purity as evidenced by SDS-PAGE and the molecular weight was confirmed by ESI-MS (m/z found 25 799, calc'd 25 800 Da)<br>
Arylmalonate decarboxylase activity was first tested with intact E. coii cells pos¬sessing the AMDase expression vector using a modification of a procedure reported in the literature,4 which uses bromothymol blue (BTB) as a pH indicator and phenylmalo-nate 1 as a substrate. In the presence of IPTG decarboxylation of the substrate by the AMDase was evident by a clear change in colour from yellow to blue. A similar colour change was noted with 2-methyl-2-phenylmalonate 2 as the substrates. The in vitro kinetic parameters of the decarboxylases were determined using a modification of a known UV spectrophotometric assay which relies on the change in absorption at 620 nm of BTB with increasing concentration of hyroxide ion4 following decarboxylation. From this, apparent kinetic parameters were determined with phenylmalonate and 2-methyl-2-phenylmalonate 2 as substrates (Table 1). Overall the kinetic constants agree with those determined for the native enzyme and suggest that the His6-tag has little ef¬fect on the activity of the AMDase. However, kcat for 2-methyl-2-phenylmalonate was higher than reported previously.6<br><br>
46<br>
Table 1. Kinetic constants for tested enzymes using phenylmalonate and 2-methyl-2-phenylmalonate as substrates (determined with BTB spectrophotometrical method).<br><br>
		2-phenylmalonic acid (1)	2-methyl-2-phenylmalonic acid (2)<br>
Protein	Km<br>
[M]	kcat<br>
Is"1]	kcat/Km<br>
[s"f*M"*J	Km<br>
[M]	kcat	kcat/Km<br>
ts'f*M"1]<br>
AMDase from<br>
Bordetella bronchi-<br>
septica	10.7*10° (13.9*10"3)a	316.1 (353)a	29630 (25390)a	26.9*10"° (25.5*10"3)a	278.9 (29.8)a	10370 (1160)a<br>
protein EAN08019.1<br>
from Mesorhizobium<br>
sp. BNC1	9.6*10"3	118	17100	22.2*10"3	55.8	2510<br>
protein NP_888076.1<br>
from Bordetella bronchiseptica RB50]	10*10"3	12.4	1240	No reaction<br>
glutamate racemase<br>
from Aquifex pyrophi-<br>
lus	3.1*10° (0.21*10"3)b	0.24 (3.3)b	77 (15710)b	No reaction<br>
protein BAA30082.1<br>
from Pyrococcus<br>
horikoshii OT3	41.7*10"3	0.09	2.2	No reaction<br>
protein<br>
ZP_00801202.1 from<br>
Alkaliphilus metallire-<br>
digenes QYMF	123*10°	0.26	2.1	No reaction<br>
literature data for AMDase from Bordetella bronchiseptica,<br>
glutamate racemase (kinetic constants for racemase activity) from Aquifex pyrophi-<br>
Example 2: Structure of the B. bronchiseptica AMDase<br>
a) AMDase crystallization by the Sitting Drop technique<br>
6. bronchisepticus AMDase as obtained according to example 1 was concen¬trated to 10 mg/mL and sitting drops were prepared using 2 uL mixed with 2 uL of mother liquor consisting of Tris (0.1 M, pH 8.5) and PEG 6000 (30 %). Trays were incu¬bated at 21 °C and small crystals appeared after 1 week. Crystals belong to the P2T space group with unit cell a = 38.7, 6 = 65.6, c = 42.2 A, IS = 110.8° and solvent content of ~ 34%. A complete data set to 1,5 A could be collected on a single crystal flash-<br><br>
47<br>
cooled to 100K after addition of 10% PEG 200 as a cryo-protectant (see Table 2). Dif¬fraction images were analyzed and data scaled using Crystal Clear.<br>
The phosphate anion, which binds in the active site of AMDase, as revealed by crystal structure analysis, derives from the FPLC purification process, where phosphate buffer was used as an eluting buffer.<br>
The position of heavy atom sites for two isomorphous lead chloride derivatives was found by manual interpretation of the difference Patterson functions. The sites lo¬cated were used to run resolve resolve to generate experimental MIRAS phases, fol¬lowed by autobuilding of a partial AMDase model using Arp/Warp. The resultant model was completed and refined using Coot and Refmac 5.<br>
Table 2: Crystallography data of AMDase from B. bronchisepticus<br><br>
Crystallogra	phy data	<br>
Wavelength (A)	1.0	<br>
Resolution (A)	20-1.50(1.55-1	.50)<br>
Total reflections	102017	<br>
Unique reflections	30804	<br>
Data completeness (%)	97.2(91,1)	<br>
Average redundancy	3.3(3.1)	<br>
1/0/1)	8.5 (2.7)	<br>
"merge	0.093 (0.292)	<br>
Rfac/Rfree	15.3/19.2	<br>
Overall B-factors (A')		<br>
Protein	11.9	<br>
Water molecules	24.6	<br>
P04	12.1	<br>
Rmsd		<br>
Bond Length (A)	0.011	<br>
Bond Angle (°)	1.421	<br>
b) Determination of AMDase structure<br>
The AMDase structure was solved to a nominal resolution of 1.45 A using multi¬ple isomorphous replacement with anomalous scattering (MIRAS) with two lead chlo¬ride derivatives. The structure reveals overall similarity with the crystal structures of Asp/Glu racemases,10,12 despite the apparent differences in catalytic functionality. The crystals were relatively small, but due to the unusually low solvent content (approx<br><br>
48<br>
22%) diffraction was very strong. The structure is clearly a tightly folded monomer, which surprisingly possesses a single phosphate molecule bound near the active site Cys188, at the centre of a solvent accessible channel between the ends of B-sheets (Figure 4(a)). The phosphate presumably mimics the carboxylate group of the pro¬posed enolate intermediate, with two out of four oxygens bound by a classic oxy-anion hole arrangement (Figure 4(b). Single isolated oxyanion holes have been described in many protein structures,(Robertus, J. D., Kraut, J., Alden, R. A. &amp; Birktoft, J. J. Subtil-isin; a stereochemical mechanism involving transition-state stabilization. Biochemistry 11, 4293-4303 (1972)). However the presence of two of these in the form of the di-oxyanion hole predicted to stabilize high-energy enediolate type intermediates has not been described before.<br>
Oi is bound by the side chain of Tyr126, and by both the amide backbone and hydroxyl side chain of Ser76. 02 on the other hand is bound by the amide backbone of Gly189 in addition to both the hydroxyl and amide backbone of Thr75. It cannot be es¬tablished from hydrogen bonding patterns alone whether either or both O atoms are deprotonated. In addition, the phosphate 03 is hydrogen bonded to the Cys188 side chain. However, very little additional direct contacts are made between the phosphate molecule and the remaining residues of the AMDase.<br>
Whilst the true biological substrate for this enzyme is not known, the prochiral 2-methyl-2-phenylmalonate 2 has been widely used as the model substrate. Initially therefore the proposed enolate intermediate derived from the decarboxylation of this substrate was modelled in to the active site of the structure (Figure 5). The model uses the coordinates of the O1-P-O2 atoms of the original phosphate group, in order to posi¬tion the carboxylate group of the enolate into the oxy-anion hole. In the original struc¬ture, the active site cavity volume is predominantly hydrophobic in nature and extends above the OrP-02 atoms. There is relatively little space available directly above the d group, while in comparison a large cavity extending into the solvent area is available directly above 02. The large cavity is also ideally shaped for binding of aromatic groups, with the Pro14 side chain positioned approximately 8A away from the Gly189-Gly190 amide bond, allowing for ideal VDW stacking of a flat aromatic system in be¬tween. From the available active site volume, it seems the enzyme will be able to ac¬commodate relatively bulky ortho-, meta- and para- groups including a 2-naphthyl sub-stituent, which is in line with the previously determined substrate specificity of this en¬zyme.6,13 The fact that there is little space available above d clearly indicates that this<br><br>
49<br>
is most likely to accommodate the smaller methyl substituent. The model of the pro¬posed intermediate, bound in this way (Figure 5), positions the Cys188 thiol in close contact to s/-face of the enolate, which is consistent with a subsequent protonation step leading to the R-configured 2-phenyl propionate product.<br>
Example 3: Active site modifications<br>
Further analysis of the crystal structure, with the proposed enolate intermediate docked in the active site, revealed the amino acid residues Leu40, Val43, Val156, Tyr48 and Met159 (situated on the opposite side to the carboxylate that is lost) make up the smaller hydrophobic binding pocket that might accommodate the methyl sub¬stituent (Figure 5).<br>
Accordingly three individual single mutants Leu40Ala, Val43Ala and Val156Ala were generated by site-directed mutagenesis.<br>
The following primers (forward and reverse) were used for each site directed mutagenesis (obtained from SigmaAldrich)<br>
L40A_F      GAGCGGCCTGGGCGCGGGTAGCGTTAC    (SEQIDNO:17) L40A_R     GTAACGCTACCCGCGCCCAGGCCGCTC     (SEQIDNO:18)<br>
V43A_F     CCTGGGTAGCGCAACCCCGGAAGGC	(SEQ ID NO: 19)<br>
V43A_R     GCCTTCCGGGGTTGCGCTACCCAGG	(SEQ ID NO: 20)<br>
V156A_F   GCATTACCGGCGCGGAAGCGATGGC	(SEQ ID NO: 21)<br>
V156A_R   GCCATCGCTTCCGCGCCGGTAATGC	(SEQ ID NO: 22)<br>
While the preparation of mutants of the present invention is exemplified for said three specific single mutants of one specific AMDase (BAA02419.1), a skilled reader will be enabled to introduce by means of substantially identical processes mutations at any position of the amino acid sequence in order to modify the activity profile of en¬zyme of the invention.<br>
Selected site-directed mutagenesis primers:<br>
Pro14Ala_F	CATTGGCATGATTGTGGCGCCGGCAGCGGGTC (SEQ ID NO: 23)<br>
Pro14Ala_R        GACCCGCTGCCGGCGCCACAATCATGCCAATG (SEQ ID NO: 24)<br><br>
50<br>
P14G_F     CATTGGCATGATTGTGGGTCCGGCAGCGGGTCTG      (SEQ ID NO: 25) P14G_R     CAGACCCGCTGCCGGACCCACAATCATGCCAATG       (SEQ ID NO: 26)<br>
G189A_F   CTGCTGTCTTGCGCGGGTCTGCTGACC	(SEQ ID NO: 27)<br>
G189A_R   GGTCAGCAGACCCGCGCAAGACAGCAG	(SEQ ID NO: 28)<br>
G190A_F   CTGCTGTCTTGCGGCGCGCTGCTGACCCTGGATG      (SEQ ID NO: 29) G190A_R   CATCCAGGGTCAGCAGCGCGCCGCAAGACAGCAG     (SEQ ID NO: 30)<br>
G189A+G190A_F<br>
CATTCTGCTGTCTTGCGCGGCGCTGCTGACCCTGGATG	(SEQ ID NO: 31)<br>
G189A+G190A_R<br>
CATCCAGGGTCAGCAGCGCCGCGCAAGACAGCAGAATG	(SEQ ID NO: 32)<br>
Selected site-directed saturated mutagenesis primers:<br>
VAL13+PR014(SAT)_F<br>
CCCCGACCATTGGCATGATTNDTNDTCCGGCAGCGGGTCTGGTTC<br>
(SEQ ID NO: 33)<br>
VAL13+PR014(SAT)_R<br>
GAACCAGACCCGCTGCCGGAHNAHNAATCATGCCAATGGTCGGGG<br>
(SEQ ID NO: 34)<br>
PR014+PR015(SAT)_F<br>
CGACCATTGGCATGATTGTGNDTNDTGCAGCGGGTCTGGTTCCGG<br>
(SEQ ID NO: 35)<br>
PR014+PR015(SAT)_R<br>
CCGGAACCAGACCCGCTGCAHNAHNCACAATCATGCCAATGGTCG<br>
(SEQ ID NO: 36)<br>
Preparation of the mutants is performed as described above.<br>
Example 4: Substrate orientation and mechanism of decarboxylation.<br>
In order to further explore the possibility that hydrophobic interactions are impor¬tant in decarboxylation, it was necessary to establish the likely orientation of the sub-<br><br>
51<br>
strate in the active site. To do this, 180 labelling experiments were carried out to estab¬lish the stereochemical course of the reaction and thus identify which of the two pro-chiral carboxylate groups of the 2-methyl-2-phenylmalonate 2 substrate is lost during decarboxylation, and which one is bound to the oxy-anion hole (Figure 8). Accordingly, pig liver esterase (PLE) was used to catalyse the enatioselective hydrolysis of 2-methyl-2-phenylmalonate diethyl ester to give the (R)-monoester 6 in 98 % e.e. in agreement with the literature.14 This was then hydrolysed with Na18OH in H2180 to give 2-methyl-2-phenylmalonic acids with an 180 label in the pro-R carboxylate 7. Similarly the diester 5 was hydrolysed first with PLE in 98 % H2180 to give 8 and then subjected to saponification with unlabelled NaOH leading to 2-methyl-2-phenylmalonic acid with an 180 label in the pro-S carboxylate 9. The chiral 180 labelled malonates 7 and 9 were separately incubated with the AMDase. The products of the decarboxylation were then methylated with TMS-diazomethane and analysed by GC-MS. From this it was clear that the product (2-phenyl propanoate methyl ester) resulting from the (/?)-malonate 7 (path A) had lost the 180 label (m/z 164), whilst the product from (S)-malonate 9 (path B) had retained the 180 label {m/z 166). This confirms that decarboxylation occurs, through the loss of the pro-R carboxylate with overall inversion of configuration.15<br>
With the stereochemical course of the reaction established it was possible to position the substrate 2-methyl-2-phenylmalonate in the active site of the AMDase structure (Figure 6). With the position of the pro-S carboxylate determined by the oxy-anion hole and phenyl substituent located in the solvent exposed pocket it is clear that the pro-S carboxylate occupies the opposite side of the active site to the Cys188 resi¬due and is indeed tightly bound by the Leu40, Val43, Val156, Tyr48 hydrophobic pocket (Figure 7). Indeed, from the model it is possible to conclude that mutation of Leu40Ala would serve to create sufficient space for at least one water molecule to bind and stabilise the pro-R carboxylate, thus retarding decarboxylation. The fact that Leu40 is essential for decarboxylase activity, whilst Val43, Val156 are apparently required for correct folding of the protein indicates the importance of these residues for the enzyme to adopt the correct functional conformation. Furthermore, these findings suggest that hydrophobic destabilisation of the pro-R carboxylate of the malonate substrate is a driv¬ing force for decarboxylation, resulting in delocalisation of electron density into the pro-S carboxylate of the enolate intermediate stabilised by the oxy-anion hole (Figure 1(b)). Thus the enzyme is able to effect efficient decarboxylation without co-factors or prior activation as an electron-withdrawing thioester, simply by orientating the malonate sub-<br><br>
52<br>
strate such that one carboxyl group is stabilised by an extensive network of H-bonding, whilst the other carboxylate is deprived of any such stabilising interactions.<br>
Example 5: Identification and catalytic properties of other AMDase related en¬zymes.<br>
Despite the significant promise of the AMDases for applications in biocataly-sis, only one such enzyme has been isolated and characterised to date. However, searches in the protein databases reveal that there are many proteins, most of un¬known function, that share sequence similarity to the S. bronchiseptica AMDase. Four protein sequences which show moderate to high similarity to the AMDase (Figure 3 (a),(b)) and which contain a cysteine at the same relative position as the Cys188 in the B. bronchiseptica enzyme, were selected for investigation (Table 3).<br>
Table 3. Enzymes selected for the search of arylmalonate decarboxylase activity and their mo¬lecular weights determined by ESI-MS.<br><br>
No. from BLAST data¬base	Microorganism	Proposed function	No of<br>
amino<br>
acid in<br>
6xHis-<br>
Tagged<br>
proteins	Identity to AM¬Dase [%] (no.<br>
of over-lapped amino acids)	Mw[D]<br><br><br><br><br><br>
	found	calculated<br>
BAA02419.1	Bordetella bronchiseptica	Arylmalonate decarboxylase	248	100(240)	25 799	25800<br>
EAN08019.1	Mesorhizobium sp. BNC1	Asp/Glu ra-cemase	248	52(219)	25770	25771<br>
NP_888076.1	Bordetella<br>
bronchiseptica<br>
RB50	Putative decarboxylase	273	36 (237)	29953	29950<br>
ZP_00801202.1	Alkaliphilus<br>
metalliredigenes<br>
QYMF	Asp/Glu ra-cemase	251	34 (183)	27413	27410<br>
BAA30082.1	Pyrococcus hori-koshii OT3	Hypothetical<br>
arylmalonate<br>
decarboxylase	246	30 (214)	27816	27818<br>
AAF25672	Aquifex pyrophi-lus	Glutamate racemase	262	-	29060	29058<br><br>
53<br>
Accordingly genes encoding these proteins were synthesised with codons optimised for E. coli. The synthetic genes were cloned, over-expressed in E. coli. and purified as His6-fusion proteins as described above. SDS-PAGE confirmed that all the proteins were of greater than 95% purity and ESI mass spectroscopy of all the proteins agreed with the calculated values (Table 3). In addition to this, a known glutamate ra-cemase from Aquifex pyrophilus was overproduced and purified. The sequence of this glutamate racemase has little similarity, to the sequence of the AMDase from B. bron-chiseptica. However it's structure10 and mechanism which is based on the formation of an intermediate enolate with two cysteines serving in general acid-base catalysis is well established.10,16 These facts made this enzyme an interesting candidate to study in order to establish if a native racemase could also demonstrate promiscuous decar¬boxylase activity.<br>
The activity of the new enzymes was initially measured in vivo on plates (see Figure 9 and 10) with BTB and phenylmalonate as a substrate. Surprisingly all of the transformed colonies became blue after extended periods of incubation, whilst trans-formants with no gene inserted did not give rise to a colour change. This suggests that all of these enzymes can catalyse the decarboxylation reaction to some extent. Subse¬quently, apparent kinetic constants for the purified enzymes were established (Table 1). Notably the enzyme from Mesorhizobium sp. BNC1, which shows highest similarity to the B. bronchiseptica AMDase, showed very efficient decarboxylase activity with both phenylmalonate 1 and 2-methyl-2-phenylmalonate 2 as substrates. The Bordetella bronchiseptica RB50 enzyme shows moderate activity with phenylmalonate, whilst the enzymes from Pyrococcus horikoshii OT3 and Alkaliphilus metalliredigenes QYMF show very low levels of activity with phenylmalonate, which is in line with in vivo assay which indicated only slow development of colour for transformants possessing these enzymes. Interestingly the known glutamate racemase from Aquifex pyrophilus is able to catalyse the decarboxylation of phenylmalonate albeit with 196 fold lower activity than the native glutamate racemase activity. It is possible to rationalise these results through protein sequence alignments (Figure 3(a)(b)). All enzymes possess oxy-anion hole, however only the B. bronchiseptica and the enzyme from Mesorhizobium sp. have the putative hydrophobic carboxylate binding pocket which is required to destabi¬lise the C02 group that is lost on decarboxylation. The conclusion from this is that tight binding of one carboxylate into the oxy-anion hole is sufficient for decarboxylation, however for efficient decarboxylation the carboxylate group that is lost must be destabi-<br><br>
54<br>
lised through hydrophobic contacts with the enzyme active site. These results indicate that decarboxylation of aryl malonates can be accomplished by a number of different enzymes, and this offers a number of possible starting points for engineering new de¬carboxylases with improved substrate specificities and biocatalytic activities.<br>
Example 6: Structure and mechanism of the AMDase from Mesorhizobium sp. BNC1<br>
From the pool of new enzymes only the decarboxylase, from Mesorhizobium sp. BNC1, was proficient in decarboxylating the 2-methyl-2-phenylmalonate 2. In this case the 2-phenylpropionate product obtained after 2-methyl- decarboxylation was shown to possess the same R-configuration (ee &gt;99%) as the decarboxylation product resulting from the AMDase from Bordetella bronchiseptica. Moreover 180 labelling studies with (/?)- and (S)-malonate 7 and 9 also indicate that the Mesorhizobium sp. BNC1 AMDase opperates with the same stereochemistry as the B. bronchiseptica enzyme, loosing the pro-R carboxylate with overall inversion of configuration. Given the apparent mechanis¬tic and sequence similarity to the B. bronchiseptica AMDase it was possible to gener¬ate a model of the Mesorhizobium sp. BNC1 AMDase active site (Figure 6(b with the substrate 2-methyl-2-phenylmalonate 2 bound. This shows a very similar active site structure with oxy-anion hole accommodating the pro-S carboxylate group and a con¬served hydrophobic pocket serving to destabilise the pro-R carboxylate. In addition a solvent accessible pocket exists which would ideally accommodate the phenyl group of the substrate. Taken together this confirms that this enzyme possesses a similar mechanism and structure as the B. bronchiseptica AMDase. In addition to proving valuable information for the design of new decarboxylases, these observations raise some interesting questions as to why these quite different organisms possess similar enzymes with no obvious function in primary or secondary metabolism. Clearly it re¬mains to be seen what physiological role this class of enzymes might fulfil in nature.<br>
Example 7: Activity of the B. bronchiseptica KU 1201 and Mesorhizobium sp BNC1 AMDases for different alkenyl-substituted substrates of formula I.<br>
A BTB activity assay was performed as described above. Compounds of formula I, wherein R2 is methyl and R1 is phenyl, ethenyl, propen-1-yl, buten-1-yl or 2-methyl-<br><br>
55<br>
propen-1-yl were tested as substrates. The results are summarized in following Table 4:<br>
Table 4: Kinetic constants for different compound of formula I at 30 °C determined by BTB assay.<br><br>
Substrate	AMDase from		AMDase from<br>
Compound of<br>
formula I<br>
Ri = methyl	Bordetella<br>
bronchiseptica<br>
KU1201	Mesorhizobium sp. BNC1<br>
R2 =	Km<br>
(mM )	Kcat<br>
(s-1)	kcat/Km<br>
mM"1)	Km<br>
(mM)	Kcat<br>
(s-1)	Kcat'vn<br>
(s"r mM"1)<br>
ethenyl (3)	7.8	42.8	5.5	19.4	8.9	0.46<br>
propen-1 -yl (4)	12.8	46.2	3.6	29.0	10.2	0,35<br>
n-buten-1-yl (5)	14.4	23.9	1.6	32.5	7.7	0.24<br>
2-methyl-propen-1-yl (6)	3.5	5.2	1.5	6.3	0.7	0.11<br>
The data show that each of the tested alkenyl-substituted, non-aromatic compounds is decarboxylated by each of said two different AMDase enzymes. For non-aromatic resi¬dues R1 the fastest reaction was observed for R1 = entenyl<br>
Example 8: Determination of optical purity and absolute configuration of mono-acids formed by of the B. bronchiseptica KU 1201 and Mesorhizobium sp BNC1 AMDases for different alkenyl-substituted substrates of formula I<br>
Different non-aromatic substrates were decarboxylated by the AMDases from Bordetella bronchiseptica KU1201 and Mesorhizobium sp. BNC1 as shown I the sub¬sequent scheme:<br><br>
56<br><br><br><br>
COOH COOH<br>
AMDase<br><br>
Me<br><br>
COOH<br><br><br><br>
R3   = H,    CH3<br><br>
Me=  CH,<br><br>
R     = H,   CH3CH2<br>
The conditions of said enzymatic reactions are the same as described for aryl¬malonates (see above) just the reaction time is longer. Usually after 24h all the sub¬strate has been converted.<br>
The reaction of said unsaturated aliphatic malonates goes to the completion (&gt;99% of conversion). The product can be recovered in pure form (as determined by NMR) by extraction with diethyl ether from an acidified aqueous layer after termination of the enzymatic reaction. Usually, no additional step of purification is required.<br>
Reactions were performed on a 200 mg scale, and the observed yields of pure recovered product were around 90-95%.<br>
In order to analyze the reaction product the obtained monoacids were derivatised with diazomethane to the corresponding methyl esters. The ee values of said methyl esters were then determined by means of gas chromatography applying a chiral col¬umn Varian Chirasil-Dex CB (25m x 250 urn x 0.26um) under the following temperature conditions: hold at 50°C for 30 min., then gradient 10°C/min until temperature reaches 200°C, then hold for 5 min at 200°C.<br>
Retention time values:<br>
methyl 2-methylbut-3-enoate:	12.286 min (enantiomer (R)) and 12.350 min (S),<br>
methyl (E)-2-methylpent-3-enoate: 24.984 min (S) and 27.823 min (R); methyl (E)-2-methylhex-3-enoate: 34.324 min (S) and 34.885 min (R); methyl 2,4-dimethylpent-3-enoate: 33.679 min (R) and 34.254 min (S).<br>
Monoacids were obtained with ee &gt;99% for each of the two AMDase enzymes from Bordetella bronchiseptica KU1201 and Mesorhizobium sp. BNC1.<br>
The absolute configuration of the obtained monoacids is (R)„ the same as for arylmalonates. Absolute configuration was determined by the comparison of measured optical rotatory power with corresponding literature data.<br><br>
57<br>
Corresponding non-aromatic HO-substituted substrates (wherein Me is replaced by<br>
hydroxy) are decarboxylated as well as assayed by colorimetric screening methods (da¬ta not shown).<br>
Concluding general remarks on the present invention<br>
The first X-ray crystal structure of a co-factor independent aryl malonate decar¬boxylase (AMDase), from Bordetella bronchiseptica, has been presented with phos¬phate bound to the active site. This reveals the presence of oxy-anion hole, which are required to bind one of the substrate carboxylate and stabilise the formation of a puta¬tive enolate intermediate. In addition a solvent accessible cavity is evident which can accommodate the large aryl substituent of typical substrates, whilst a tight hydrophobic pocket restricts the size of the additional aliphatic malonate substituents. Using 180 la¬belling studies it was possible to establish that the decarboxylation of the model sub¬strate 2-methyl-2-phenylmalonate occurs with inversion of stereochemistry and loss of the pro-R carboxylate, this allows a model of the substrate bound to the active site to be generated. This model shows that the pro-R carboxylate of the substrate lost during decarboxylation is tightly bound by the hydrophobic pocket, which is postulated to de¬stabilise the negative charge on the carboxylate, providing the necessary driving force for decarboxylation in the absence of co-factors or acyl enzyme intermediates. The structural and mechanistic importance of this hydrophobic pocket is exemplified by site-directed mutagenesis of the hydrophobic residues to Ala, which either results in mis-folding or complete loss of activity. Following decarboxylation the resulting enolate, which is stabilised by the oxy-anion hole, is then protonated from the si-face by the ac¬tive site Cys188 residue leading to (2/?)-phenyl propionic acid.<br>
In addition to this, four new enzymes that exhibit similarity to the B. bronchisep¬tica AMDase, are shown to catalyse decarboxylation of aryl malonate substrates. One of these enzymes from Mesorhizobium sp. BNC1, was found to possess similar sub¬strate specificity, catalytic efficiency and stereochemistry as the original B. bronchisep¬tica AMDase. Moreover the sequence similarity between the two enzymes allowed a structural model of the AMDase from Mesorhizobium to be generated, which supports the hypothesis that the two AMDases share a common mechanism. Furthermore the known glutamate racemase, from Aquifex pyrophilus, which is predicted to possess a similar catalytic mechanism, was also shown to possess promiscuous aryl malonate decarboxylase activity despite exhibiting little sequence similarity to the AMDase.<br><br>
58<br>
Overall these combined results provide the key structural and mechanistic insight that is necessary for engineering new decarboxylase enzymes, either by rational mutagenesis or directed evolution. Said techniques are for example disclosed in<br>
1)	Roberto A Chica, Nicolas Doucet and Joelle N Pelletier, Semi-rational approaches to engineering enzyme activity: combining the benefits of directed evolution and ra¬tional design, Current Opinion in Biotechnology, 16, 4, 2005, 378-384,<br>
2)	Enzyme functionality: Design, engineering, and screening edited by Allan Svendsen, Marcel Dekker, Inc., New York, Basel 2003<br>
3)	Paul A. Dalby Engineering Enzymes for Biocatalysis Recent Patents on Biotechnol¬ogy 2QQ7, 1,1-9<br>
This is an important improvement as the high stereoselectivity afforded by the AMDases enables the preparation of valuable homochiral carboxylic acids from cheap accessible malonate precursors.<br>
Documents cited herein are incorporated by reference.<br><br>
59<br>
References<br>
1.	Dimroth, P., Hilbi, H., Enzymic and genetic basis for bacterial growth on malo-nate. Mol. Microbiol. 1997, 25, (1), 3-10.<br>
2.	Hoenke, S., Wild, M. R., Dimroth, P., Biosynthesis of triphosphoribosyl-dephospho-coenzyme A, the precursor of the prosthetic group of malonate de¬carboxylase. . Biochemistry 2000, 39, (43), 13223-32.<br>
3.	Miyamoto, K., Ohta, H., Asymmetric decarboxylation of disubstituted malonic acid by Alcaligenes bronchisepticus KU1201. Biocatalysis 1991, 5, 49-60.<br>
4.	Miyamoto, K., Ohta, H., Cloning and heterologous expression of a novel aryl-malonate decarboxylase gene from Alcaligenes bronchisepticus KU 1201. Appl Microbiol Biotechnol 1992, 38, 234-238.<br>
5.	Miyamoto, K., Ohta, H., Enzyme-mediated asymmetric decarboxylationof disub¬stituted malonic acids. J Am Chem Soc 1990,112, 4077-4078<br>
6.	Miyamoto, K., Ohta, H., Purification and properties of a novel arylmalonate de¬carboxylase from Alcaligenes bronchisepticus KU 1201. European Journal of Bio¬chemistry 1992, 210, (2), 475-81.<br>
7.	Terao, Y., Miyamoto, K., Ohta, H., Introduction of single mutation changes aryl¬malonate decarboxylase to racemase. Chemical Communications 2006, 34, 3600-3602.<br>
8.	Matoishi, K., Ueda, M., Miyamoto, K., Ohta, H., Mechanism of asymmetric decar¬boxylation of a-aryl-a-methylmalonate catalyzed by arylmalonate decarboxylase originated from Alcaligenes bronchisepticus. Journal of Molecular Catalysis B: Enzymatic 2004, 27, (4-6), 161-168.<br>
9.	Glavas, S., Tanner, M. E., Active site residues of glutamate racemase. Biochem¬istry 2001, 40, (21), 6199-6204<br>
10.	Hwang, K. Y., Cho, C. S., Kim, S. S., Sung, H. C, Yu, Y. G., Cho, Y., Structure and mechanism of glutamate racemase from Aquifex pyrophilus. Nature structural biology 1999, 6, (5), 422-426.<br>
11.	Terao, Y., Ijima, Y., Miyamoto, K., Ohta, H., Inversion of enantioselectivity of arylmalonate decarboxylase via site-directed mutation based on the proposed re¬action mechanism. Journal of Molecular Catalysis B: Enzymatic 2007,45, (1-2), 15-20<br>
12.	Liu, L, Iwata, K., Kita, A., Kawarabayasi, Y., Yohda, M., Miki, K. , Crystal struc¬ture of aspartate racemase from Pyrococcus horikoshii OT3 and its implications for molecular mechanism of PLP-independent racemization. J. Mol. Biol. 2002, 319,479-489.<br><br>
60<br>
13.	Miyamoto, K., Ohta, H., Osamura, Y., Effect of Conformation of the Substrate on Enzymatic Decarboxylation of a-Arylmalonic Acid. Bioorganic and Medicinal Chemisby 1994, 2, (6), 469-475.<br>
14.	Dominguez de Maria, P., Kossmann, B., Potgrave, N., Buchholz, S., Trauthwein, H., May, O., Groeger, H., Improved process for the enantioselective hydrolysis of prochiral diethyl malonates catalyzed by pig liver esterase. Synlett 2005, 11, 1746-1748.<br>
15.	Miyamoto, K., Tsutsumi, T., Terao, Y., Ohta, H., Stereochemistry of Decarboxyla¬tion of Arylmalonate Catalyzed by Mutant Enzymes. Chemistry Letters 2007, 36, (5), 656-657.<br>
16.	Handa, S., Koo, J. H., Kim, Y.S., Floss, H. G., Stereochemical Course of Biotin-IndependentMalonate Decarboxylase Catalysis. Archives of Biochemistry and Biophysics 1999, 370, (1), 93-96.<br>
17.	Janes, L. E., Lowendahl, A. Ch., Kazlauskas, R. J., Quantitative screening of hy¬drolase libraries using pH indicators: identifying active and enantioselective hy¬drolases. Chemistry-A European Journal 1998,4, (11), 2324-2331.<br>
18.	Banerjee, A., Kaul, P., Sharma, R., Banerjee, U. C, A high-throughput amenable colorimetric assay for enantioselective screening of nitrilase-producing microor¬ganisms using pH sensitive indicators. Journal of Biomolecular Screening 2003, 8, (5), 559-565.<br><br>
61<br>
Claims:<br>
1.	A method of the biocatalytical decarboxylation of a compound of formula I,<br>
HOOC-C(R1)(R2)-COOH     (I)<br>
or a salt form thereof,<br>
wherein<br>
residue R1 represents a linear or branched, mono-unsaturated C2-C8-alkenyl<br>
group; and<br>
residue R2 represents H, OH, SH, NH2, CrC8-alkoxy, CrC8-alkylthio,Ci-C8-alkyl,<br>
C2-C8-alkenyl, C2-C8-alkynyl or hydroxyl CrC8-alkyl,<br>
which method comprises<br>
a)	contacting at least on compound of formula I with<br>
(i) a protein having decarboxylase activity and comprising an amino acid sequence having a sequence identity of 100 % to 25% to the Bordetella bronchiseptica protein of SEQ ID NO: 2 (residues 1 to 240) or<br>
(ii) a protein having decarboxylase activity and comprising an amino acid sequence having a sequence identity of 100% to 60% to the Aquifex pyrophilus protein of SEQ ID NO: 12 (residues 1 to 254);<br>
b)	and optionally isolating at least one decarboxylation product of a compound<br>
of formula I.<br>
2.	A method of the biocatalytical decarboxylation of a compound of formula I,<br>
HOOC-C(R1)(R2)-COOH     (I)<br>
or a salt form thereof,<br>
wherein<br>
residue R1 represents a mono- or bicyclic, optionally substituted, saturated or<br>
mono- or poly-unsaturated ring, and<br>
residue R2 represents H, OH, SH, NH2, CrC8-alkoxy, CrC8-alkylthio, CrC8-alkyl,<br>
C2-C8-alkenyl, C2-C8- alkynyl or hydroxyl CrC8-alkyl,<br><br>
62<br>
which method comprises<br>
a)	contacting at least on compound of formula I with<br>
(i) a protein having decarboxylase activity selected from the group consisting of<br>
(a)        a protein comprising amino acid residues 2 to 241 of SEQ ID NO: 4 and originating from Mesorhizobium sp. BNCI;<br>
(c)	a protein comprising amino acid residues 2 to 267 of SEQ ID NO: 8 and originating from Bordetella bronchiseptica;<br>
(d)	a protein comprising amino acid residues 2 to 246 of SEQ ID NO: 10 and originating from Alkaliphilus metalliredi-genes QYMF; and<br>
(e)	proteins having arylmalonate or malonate decarboxylase activity and having a sequence identity less than 100% to 60% to one of the sequences of a), b), c) ord);<br>
(ii) a protein having decarboxylase activity and comprising an amino acid sequence having a sequence identity of 100% to 60% to the Aquifex pyrophilus protein of SEQ ID NO: 12 (residues 1 to 254);<br>
b)	and optionally isolating at least one decarboxylation product of a compound<br>
of formula I.<br>
3.	The method of claim 1, wherein the protein having decarboxylase activity and<br>
showing an amino acid sequence identity of 100% to 60% to the Bordetella bron¬<br>
chiseptica protein of SEQ ID NO: 2 has a substrate pocket, said pocket having<br>
the ability to carry a substrate molecule of formula I, and comprising in functional<br>
arrangement the following structural elements<br>
a)	a Cystein containing catalytic site, the Cystein residue of which is able to interact with the substrate to, in particular by transferring a proton to the enolate intermediate of the decarboxylated substrate,<br>
b)	an oxy-anion hole, comprising a plurality of polar amino acid side chains interacting with the carboxylic group of the substrate not to be cleaved off.<br>
4.	The method of claim 3, wherein the enzyme has a substrate pocket comprises<br>
the following structural elements<br><br>
63<br>
a)	the Cystein containing catalytic site "CSS", which is formed by a se¬<br>
quence region which shows the following sequence pattern:<br>
"CSS"	(S/L) C (G/T)(G/N)	(SEQ ID NO: 13)<br>
and<br>
b)	the oxy-anion hole, which is formed amino acid residues of two se¬<br>
quence regions which show the following two sequence pattern:<br>
"OAH1S"	(M/G)(G/C)TS(L/G)	(SEQ ID NO: 14)<br>
"OAH2S"              T(A/P)Y(I/R)(D/Q)	(SEQIDNO:15)<br>
or has a mutated substrate pocket, still having the ability to carry at least one substrate of formula (I).<br>
The method of claim 4 wherein said structural elements are arranged in the amino acid chain in the following sequential (N-&gt;C) order:<br>
OAH1S-OAH2S-CSS and wherein said structural elements are linked together via amino acid se¬quences independently of each other comprising one or more amino acid resi¬dues.<br>
The method of one of the claims 1 and 3 to 5, wherein the protein having decar¬boxylase activity is selected from the group consisting of<br>
a)	a protein comprising amino acid residues 2 to 241 of SEQ ID NO: 4 and originating from Mesorhizobium sp. BNCI;<br>
b)	a protein comprising amino acid residues 2 to 240 of SEQ ID NO: 6 and originating from Pyrcoccus horikoshii;<br>
c)	a protein comprising amino acid residues 2 to 267 of SEQ ID NO: 8 and originating from Bordetella bronchiseptica;<br>
d)	a protein comprising amino acid residues 2 to 246 of SEQ ID NO: 10 and originating from Alkaliphilus metalliredigenes QYMF; and<br>
e)	proteins having arylmalonate or malonate decarboxylase activity and having a sequence identity less than 100% to 60% to one of the se¬quences of a), b), c) or d).<br>
The method of one of the preceding claims, wherein the decarboxylation reaction is a stereo-selective mono-decarboxylation.<br><br>
64<br>
8.	The method of claim 7, wherein an optically pure compound of the formula II<br>
HC(R1)(R2)-COOH     (II)<br>
or a salt form thereof is obtained,<br>
wherein residue R1 and R2 are as defined in claim 1.<br>
9.	The method of one of the preceding claims, wherein the decarboxylation reaction is performed with said decarboxylase in isolated or purified (optionally immobi¬lized) form, or by culturing a microorganism expressing said decarboxylase en¬zyme activity in the presence of a compound of formula I.<br>
10.	The method of one of the preceding claims, wherein a compound of formula I or a salt form thereof is decarboxylated, wherein residue R1 represents a linear or branched mono-unsaturated C2-C8-alkenyl group and residue R2 is a CrC8-alkyl group.<br>
11.	The method of claim 10, wherein R1 represents a linear CrC6-alken-1-yl group and residue R2 is a C-i-C6-alkyl group.<br>
12.	The method of claim 11, wherein R1 represents, ethenyl, propen-1-yl, buten-1-yl.<br>
13.	The method of one of the claims 10 to 12, wherein said protein having decar¬boxylase activity is selected from the group consisting of a protein originating from Mesorhizobium sp. BNCI and a protein originating from Bordetella bron-chiseptica as defined hereinabove.<br>
14.	A protein having decarboxylase activity, having a sequence identity of less than 99% to the Bordetella bronchiseptica protein of SEQ ID NO: 2 and comprising at least one functional mutation in one of the amino acid sequence positions 14, 15, 159 and 190 said positions being numbered according to the alignment of Fig. 3 (b).<br>
15.	A mutant of a protein having decarboxylase activity, selected from<br><br>
65<br>
a)	a protein comprising amino acid residues 2 to 241 of SEQ ID NO: 4<br>
and originating from Mesorhizobium sp. BNCI;<br>
b)	a protein comprising amino acid residues 2 to 240 of SEQ ID NO: 6 and originating from Pyrcoccus horikoshii OT3;<br>
c)	a protein comprising amino acid residues 2 to 267 of SEQ ID NO: 8 and originating from Bordetella Bronchiseptica RB50;<br>
d)	a protein comprising amino acid residues 2 to 246 of SEQ ID NO: 10 and originating from Aklaliphilus metalliredigenes QYMF; and<br>
which mutant having a sequence identity less than 100% to 60% to the corre¬sponding parent protein and comprising at least one functional mutation in one of the amino acid sequence positions 14, 15, 159 and 190 said positions being numbered according to the alignment of Fig. 3 (b).<br>
16.	A mutant of a protein having decarboxylase activity, which protein comprises amino acid residues 2 to 256 of SEQ ID NO: 12 and originating from Aquifex py-rophilus , which mutant having a sequence identity less than 100% to 60% to the corresponding parent protein.<br>
17.	The mutant of claim 16 still comprising the structural elements as defined in one of the claims 3 to 6.<br>
18.	A nucleic acid encoding a protein having decarboxylase activity of anyone of the claims 14 to 17.<br>
19.	An expression cassette, comprising a nucleic acid of claim 18 operably linked with at least one regulatory nucleic acid sequence.<br>
20.	A recombinant expression vector comprising at least one expression cassette of claim 19 or nucleic acid of claim 18.<br>
21.	A recombinant microorganism, carrying at least one expression vector of claim 20.<br><br>
66<br>
22.	A bioreactor comprising at least one protein having decarboxylase activity as de¬fined in one of the claims 14 to 17 or a recombinant microorganism of claim 20, optionally in immobilized form.<br>
23.	A method of preparing an enzyme having decarboxylase activity, which method comprises cultivating a recombinant microorganism as defined in claim 21 and optionally isolating the said enzyme having decarboxylase activity from the cul¬ture.<br>
24.	A screening method for candidate enzymes having a decarboxylase activity as defined in claim 1, which method comprises identifying a sequence motif (pat¬tern) characteristic for an enzyme as defined in one of the claims 14 to 17 and screening amino acid sequence data bases for data base entries partially or fully matching with said sequence motif.<br>
25.	The screening method of claim 24, wherein said sequence motif is defined in claims 3 to 6.<br>
26.	The use of an expression product as prepared according to claim 23 in a decar¬boxylation reaction decarboxylating a compound of formula I as defined in claim 1.</td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/viewdoc.aspx?id=PDQLRXL4HJVBpqDyap7UZg==&amp;amp;loc=egcICQiyoj82NGgGrC5ChA==" target="_blank" style="word-wrap:break-word;">http://ipindiaonline.gov.in/patentsearch/GrantedSearch/viewdoc.aspx?id=PDQLRXL4HJVBpqDyap7UZg==&amp;amp;loc=egcICQiyoj82NGgGrC5ChA==</a></p>
		<br>
		<div class="pull-left">
			<a href="268438-method-for-the-vinylation-of-amides.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="268440-novel-polyolefin-compositions-and-drawn-tapes-fibres-and-filaments-produced-therefrom.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>268439</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>3174/CHENP/2010</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>36/2015</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>04-Sep-2015</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>31-Aug-2015</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>28-May-2010</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>BASF SE</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td>D-67056 Ludwigshafen  Germany</td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>HAUER  Bernhard</td>
											<td>Merowinger Str. 1  67136 Fussgönheim  Germany</td>
										</tr>
										<tr>
											<td>2</td>
											<td>BAUDENDISTEL  Nina</td>
											<td>Carl-Blos-Str. 3  77654 Offenburg  Germany</td>
										</tr>
										<tr>
											<td>3</td>
											<td>OKRASA  Krzysztof</td>
											<td>Oxford Road  Manchester M13 9PL  United Kingdom</td>
										</tr>
										<tr>
											<td>4</td>
											<td>MICKLEFIELD  Jason</td>
											<td>Oxford Road  Manchester M13 9PL  United Kingdom</td>
										</tr>
										<tr>
											<td>5</td>
											<td>LEYS  David</td>
											<td>Oxford Road  Manchester M13 9PL  United Kingdom</td>
										</tr>
										<tr>
											<td>6</td>
											<td>LEVY  Colin</td>
											<td>Oxford Road  Manchester M13 9PL  United Kingdom</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>C12P 7/40</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>PCT/EP2008/010131</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td>2008-11-28</td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td>08156393.4</td>
									<td>2008-05-16</td>
								    <td>EPO</td>
								</tr>
								<tr>
									<td>2</td>
									<td>07023075.0</td>
									<td>2007-11-28</td>
								    <td>EPO</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/268439-a-method-of-the-biocatalytical-decarboxylation-of-a-compound by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 07:18:42 GMT -->
</html>
