<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/269063-biosensor-utilizing-dna-as-element by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 08:24:23 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 269063:BIOSENSOR UTILIZING DNA AS ELEMENT</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">BIOSENSOR UTILIZING DNA AS ELEMENT</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>An object of the present invention is to provide a method excellent in cost, operability, and rapidity, which can detect and/or quantify an analyte in a test sample by utilizing a sensor protein specifically binding to the analyte and a nucleic acid that is specifically recognized by the sensor protein. The present inventors have prepared ArsR-GFP comprising an ArsR protein (sensor protein capable of binding to arsenic) fused with a green fluorescent protein (GFP) and confirmed that this fusion protein is capable of binding to a specific recognition sequence (Pars-DNA) and that their binding is inhibited by arsenious acid. Next, the present inventors have prepared a Pars-DNA-immobilized plate and completed the present invention by finding that the amount of ArsR-GFP bound to the Pars-DNA-immobilized plate is decreased in an arsenious acid concentration-dependent manner.</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td>DESCRIPTION<br><br>
TITLE OF THE INVENTION <br><br>
BIOSENSOR UTILIZING DNA AS ELEMENT<br><br>
Technical Field<br><br>
[0001]<br>
The present invention relates to a method for detecting and/or quantifying an analyte in a test sample by utilizing a sensor protein specifically binding to the analyte and a nucleic acid that is specifically recognized by the sensor protein, etc.<br><br>
Background Art<br><br>
[0002]<br>
Currently, the pollution of soil, groundwater, epipelagic water, or the like with toxic metal compounds such as lead, cadmium, and arsenic has been reported and has become a major issue in many parts of the world. Flame atomic absorption spectrometry (AAS), flameless atomic absorption spectrometry (FLAA), inductively coupled plasma atomic emission spectrometry (ICP-AES), inductively coupled plasma mass spectrometry (ICP-MS), and the like are known as methods used in the analysis of toxic metal pollution in the environment. However, these instrumental analysis methods require sample pretreatment for conducting analysis and have the disadvantages that they cannot manage rapid on-the-spot analysis and have high running cost. <br><br>
[0003]<br>
The development of a simple toxic metal detection method is in urgent need to make up for such disadvantages of the instrumental analysis methods. A biosensor utilizing, as a sensor element, test molecules such as biologically derived enzymes, antibodies, or receptors recognizing substances to be detected has also received attention as one of such simple detection methods. The biosensor has advantages such as highly sensitive assay, rapid and convenient assay/detection, low running cost, and the small size and excellent transportability of the sensor itself, because the detection reaction is based on the specificity of biochemical reaction or binding. Moreover, the biosensor characteristically achieves analysis highly specific for particular substances by utilizing enzymatic reaction, antigen-antibody reaction, or biochemical reaction such as ligand-receptor binding. Therefore, only a small amount of samples is required, and a micro-sensing system can also be achieved. <br><br>
[0004]<br>
While various biosensors have been studied, a biosensor utilizing microbial cells can be expected to be widely applied, because a wide range of substances to be detected or reporter genes can be selected owing to easy culture and easy genetic operation. A recombinant microbial biosensor consists of three factors: host cells; an inducible promoter that promotes the transcription of downstream genes in response to particular substances; and a reporter gene downstream of the promoter. In some cases, it utilizes a system combined with an apparatus for detecting a signal emitted by the reporter. In this system, bacterium- or firefly-derived luciferase (see Non Patent Literatures 1 to 3), Aequorea victoria-derived GFP (see Non Patent Literatures 2 and 4), E. coil-derived LacZ (see Non Patent Literatures 2 and 5), or the like is often used as a reporter. <br><br>
[0005]<br>
The present inventors have already developed an arsenic-responsive biosensor utilizing CrtA, an enzyme of the carotenoid synthesis system, as a reporter and a marine photosynthesis bacterium Rhodovulu/n sulfidophilum as a host (Non Patent Literature 6). The flhodovulum sulfidophilum has a spheroidene pathway, one pathway of the carotenoid synthesis system. This bacterium originally accumulates therein the final product spheroidenone to exhibit a red color. This sensor accumulates therein spheroidene, a precursor of spheroidenone, to exhibit a yellow color, because the sensor utilizes a crtA-deficlent strain obtained by destroying crtA encoding spheroidene monooxygenase (CrtA) that catalyzes the final stage of the spheroidene pathway. Therefore, as a result of introducing plasmids comprising crtA as a reporter gene and an arsenic-inducible promoter incorporated upstream thereof into Rhodovulum sulfidophilum cells, the CrtA activity is restored in the presence of arsenic to change color from yellow to red. The presence of arsenic can be determined with such change in color as an index. <br><br>
[0006]<br>
The present inventors have further focused on a freshwater purple bacterium J?hodopseudo/nonas palustris that can be cultured under anaerobic conditions and on phytoene desaturase (CrtI)-catalyzed desaturation reaction from phytoene to lycopene in a spirilloxanthin pathway possessed by the Rhodopseudomonas   palustris, and have developed an arsenic-responsive biosensor that eliminates the need of a shake incubator, wherein CrtI activity is restored in the presence of arsenic by introducing plasmids comprising a reporter gene crtI incorporated downstream of an E. coli -derived arsenic-inducible operator/promoter region into Rhodopseudomonas palustrls  cells (Patent Literature 1).<br><br>
Citation List <br><br>
Patent Literature<br><br>
[0007]<br><br>
Patent  Literature  1:  Japanese  Patent  Application No. 2007-340804<br><br>
Non Patent Literature<br><br>
[0008]<br><br>
Non Patent Literature 1: Wilson T, Hastings JW (1998)<br>
Bioluminescence Annu Rev Cell Dev Biol 14:197-230<br><br>
Non Patent Literature 2: Stocker J, Balluch D, Gsell M, Harms H, Feliciano J, Daunert S, Malik KA, van der Meer JR (2003) Environ Sci Technol 37: 4743-4750<br><br>
Non Patent Literature 3: Trang PT, Berg M, Viet PH, van Mui N, van der Meer JR (2005) Environ Sci Technol 39: 7625-7630<br><br>
Non Patent Literature 4: Tsien RY (1998) Ann Rev Biochem 67: 509-544<br><br>
Non Patent Literature 5: Silhavy TJ, Beckwith JR (1985) Microbiol Rev 49: 398-418<br><br>
Non Patent Literature 6: Fujimoto et al. , (2006) Appl Microbiol Biotechnol 73 (2): 332-338<br><br>
Disclosure of the Invention<br><br>
Problems to be Solved by the Invention<br><br>
[0009]<br>
A biosensor utilizing microbes, as described above, requires instruments and time for culture and has problems such as unevenness attributed to cells. For solving these problems , the development of a cell-free biosensor is required. An object of the present invention is to provide a method excellent in cost, operability, and rapidity, which can detect and/or quantify an analyte in a test sample by utilizing a sensor protein specifically binding to the analyte and a nucleic acid that is specifically recognized by the sensor protein.<br><br>
Means for Solving the Problems<br><br>
[0010]<br>
To attain the object, the present inventors have first attempted to develop a method utilizing fluorescence resonance energy transfer, i.e., a system utilizing fluorescence resonance energy transfer (FRET), which detects change in the higher order structure of a complex molecule via the interaction between DNA and protein. FRET is a phenomenon in which excitation energy is transferred from an energy donor (also simply referred to as a donor) to an energy acceptor (also simply referred to as an acceptor) . In this regard, important is to select two different fluorescent molecules such that the emission energy level of a fluorescent molecule serving as a donor overlaps with the absorption energy level of a fluorescent molecule serving as an acceptor (Figure.24). When the acceptor is present near the donor in an excited state, its excitation energy excites the acceptor before the donor emits light. As a result, the light emission from the acceptor can be detected. The nearer the distance there between is, the more easily FRET occurs. Therefore, FRET is used as means for detecting the interaction between molecules or the structural change of a molecule. <br><br>
[0011]<br>
Thus, the present inventors have studied a biosensor utilizing a lead/cadmium sensor protein ZntR. E. coli has the gene zntA, which encodes an enzyme that works to discharge heavy metal from the cells. The nucleotide sequence of promoter region DNA (hereinafter, referred to as promoter DNA) locates in the region from -10-base to -35-base upstream from the transcription initiation point of this zntA. The lead/cadmium sensor protein ZntR binds to this promoter DNA to form a ZntR protein-promoter DNA complex. In the absence of heavy metal such as lead or cadmium compounds, the promoter DNA is in a folded state as a higher order structure. On the other hand, in the presence of heavy metal, the higher order structure is changed through the binding of the heavy metal to the ZntR protein, and along with this, the promoter DNA becomes linear. The present inventors have hypothesized that a heavy metal biosensor can be developed by utilizing this change in the higher order structure of the ZntR protein-promoter DNA complex depending on the presence or absence of heavy metal. <br><br>
[0012]<br>
Figure 25 shows the principle devised for externally displaying this change in the higher order structure depending on the presence or absence of heavy metal by utilizing FRET. First, two different fluorescent materials, i.e., fluorescein isothiocyanate (FITC) and tetramethylrhodamine (TAMRA) are respectively bound to both the ends of promoter DNA. At the occurrence of FRET, TAMRA is excited by the absorption of fluorescence emitted by FITC to emit fluorescence. In the absence of heavy metal, the promoter DNA is in a folded state as a higher order structure such that FITC and TAMRA are in proximity. Upon irradiation with the excitation light of FITC, the emitted FITC fluorescence excites TAMRA, and mainly TAMRA fluorescence should be detected. However, in the presence of heavy metal, the higher order structure of the promoter DNA is changed to a linear state, making the distance between FITC and TAMRA long . Thus , TAMRA hardly absorbs FITC fluorescence . Therefore, TAMRA is hardly excited, and mainly FITC fluorescence should be detected. The present inventors have thought that lead or cadmium compounds can be detected or quantified by determining, using a fluorophotometer, this change in fluorescence wavelength depending on the presence or absence of heavy metal. <br><br>
[0013]<br>
Likewise, the present inventors have studied a biosensor utilizing an arsenic sensor protein ArsR. E. coli contains arsenic resistance operon that imparts arsenic resistance to the cells. The expression of the arsenic resistance operon is regulated by the arsenic sensor protein ArsR encoded by the gene arsR. In the absence of arsenic compounds, the ArsR protein binds to promoter DNA to form an ArsR protein-promoter DNA complex. Therefore, RNA polymerase fails to bind to the promoter DNA, and the transcription of the arsenic resistance operon is repressed. However, in the presence of arsenic compounds, the ArsR protein binds to the arsenic compounds and dissociates from the promoter DNA due to the changed higher order structure.  Therefore, RNA polymerase binds to the promoter DNA to initiate the transcription of the arsenic resistance operon. The present inventors have hypothesized that a biosensor for detecting arsenic compounds can be developed by externally displaying such change between the binding and dissociation of the ArsR protein and the promoter DNA depending on the presence or absence of arsenic compounds. <br><br>
[0014]<br>
Moreover, a system for externally displaying the change between protein-DNA binding and dissociation by FRET seemed to be possible, as in use of the ZntR protein (Figure 26). First, TAMRA is bound to either end of promoter DNA. Moreover, for fluorescently modifying the ArsR protein, an ArsR-GFP fusion protein comprising an ArsR protein fused with a green fluorescent protein (GFP) was used. At the occurrence of FRET, TAMRA was supposed to be excited by the absorption of fluorescence emitted by GFP to emit fluorescence. In the absence of arsenic compounds, the promoter DNA is in a state bound with the ArsR-GFP fusion protein such that TAMRA and GFP are in proximity. Therefore, upon irradiation with the excitation light of GFP, the emitted GFP fluorescence excites TAMRA, and mainly TAMRA fluorescence is detected. However, in the presence of arsenic compounds, the promoter DNA strand is in a state dissociated from the ArsR-GFP fusion protein, making the average distance between TAMRA and GFP long. Thus, TAMRA hardly absorbs GFP fluorescence. Therefore, TAMRA is hardly excited, and mainly GFP fluorescence is detected. The present inventors have thought that arsenic compounds can be detected or quantified by determining, using a fluorophotometer, this change in fluorescence wavelength depending on the presence or absence of arsenic compounds.<br><br>
[0015]<br>
However, as is evident from Comparative Example described later, it was not possible to attain the object of the present invention by the FRET method. Thus, the present inventors must have developed a different approach. Focusing on the ArsR protein as a sensor protein capable of binding to arsenic and the CadC protein as a sensor protein capable of binding to cadmium and lead, the present inventors have prepared ArsR-GFP and CadC-GFP comprising each protein fused with a green fluorescent protein (GFP), and confirmed that these fusion proteins are capable of binding to their respective specific recognition sequence DNAs and that their bindings are inhibited by metal ions. Next, the present inventors have prepared a Parg-DNA (recognition sequence DNA of the ArsR protein)-immobilized plate and completed the present invention by finding that the amount of ArsR-GFP bound to the Pars-DNA-immobilized plate of ArsR-GFP is decreased in an arsenious acid concentration-dependent manner. <br><br>
[0016]<br>
Specifically, the present invention relates to: (1) a method for detecting or quantifying an analyte in a test sample in an aqueous system, comprising the following steps (A) and (B):<br><br>
(A)	reacting, in the presence of the test sample, a sensor protein specifically binding to the analyte, with a nucleic acid comprising a sequence that is specifically recognized by the sensor protein, wherein the nucleic acid is immobilized on a support; and<br><br>
(B)	detecting or assaying the sensor protein bound to the immobilized nucleic acid;<br><br><br>
(2)	the method according to (1), wherein the sensor protein is labeled with a detectable marker<br><br>
(3)	the method according to (1), wherein the sensor protein is a fusion protein with a detectable marker protein;<br><br>
(4)	a method for detecting or quantifying an analyte in a test sample in an aqueous system, comprising the following steps (A) and (B):<br><br>
(A)	reacting, in the presence of the test sample, a sensor protein specifically binding to the analyte, with a nucleic acid comprising a sequence that is specifically recognized by the sensor protein, wherein the sensor protein is immobilized on a support; and<br><br>
(B)	detecting or assaying the nucleic acid bound to the immobilized sensor protein; and<br><br>
(5)	the method according to (4), wherein the nucleic acid is labeled with a detectable marker.<br><br>
[0017]<br><br>
The present invention also relates to:<br><br>
(6)	the method according to any one of (1) to (5), wherein the binding between the sensor protein and the nucleic acid is inhibited by the analyte;<br><br>
(7)	the method according to any one of (1) to (6) , wherein the sensor protein is a protein encoded by a metal-responsive operon;<br><br>
(8)	the method according to (7) , wherein the protein encoded by a metal-responsive operon is an ArsR or CadC protein;<br><br>
(9)	the method according to any one of (1) to (8), wherein the analyte is a heavy metal compound;<br><br>
(10)	the method according to (9), wherein the heavy metal compound is selected from an arsenic (As) compound, a cadmium (Cd) compound, and a lead (Pb) compound;<br><br>
(11)	the method according to (10), wherein a pentavalent arsenic (As(V) ) compound is converted in advance to a trivalent arsenic (As(III)) compound by reduction treatment;<br><br>
(12)	the method according to any one of (9) to (11), wherein the reaction of the sensor protein with the nucleic acid comprising a sequence that is specifically recognized by the sensor protein occurs in a 50 mM or higher phosphate buffer (pH 7.4); and<br><br>
(13)	the method according to any one of (9) to (11), wherein the reaction of the sensor protein with the nucleic acid comprising a sequence that is specifically recognized by the sensor protein occurs in a 40 mM or higher sodium chloride solution.<br><br>
[0018]<br>
The present invention further relates to:<br><br>
(14)	a kit for detecting or quantifying an analyte in a test sample, comprising: a sensor protein that specifically binds to the analyte; and a nucleic acid comprising a sequence that is specifically recognized by the sensor protein, wherein the nucleic acid is immobilized on a support; and<br><br>
(15)	a kit for detecting or quantifying an analyte in a test sample, comprising: a sensor protein that specifically binds to the analyte, wherein the sensor protein is immobilized on a support; and a nucleic acid comprising a sequence that is specifically recognized by the sensor protein.<br><br>
Moreover, the embodiments of the present invention encompass: the kit, wherein the sensor protein is labeled with a detectable marker; the kit, wherein the sensor protein is a fusion protein with a detectable marker protein; and the kit wherein the nucleic acid is labeled with a detectable marker. <br><br>
[0019]<br>
Moreover, the embodiments of the present invention encompass: any one of the kits, wherein the binding between the sensor protein and the nucleic acid is inhibited by the analyte; any one of the kits, wherein the sensor protein is a protein encoded by metal-responsive operon; the kit, wherein the protein encoded by metal-responsive operon is an ArsR or CadC protein; and any one of the kits, wherein the analyte is a heavy metal compound such as arsenic (As), cadmium (Cd), or lead (Pb).<br><br>
Brief Description of Drawings<br><br>
[0020]<br><br>
[Figure 1] Figure 1 is a diagram showing the transcriptional regulation mechanism of toxic metal resistance operon in bacteria.  In the absence of toxic metal ions in the environment, the expression of a sensor protein gene and its downstream genes is repressed by the binding of a sensor protein to promoter DNA.<br><br>
[Figure 2] Figure 2 is a diagram showing the principle of a toxic metal biosensor utilizing DNA as an element.<br><br>
[Figure 3] Figure 3 is a diagram showing components of an arsenic compound biosensor utilizing DNA as an element.<br><br>
[Figure 4 ] Figure 4 is a diagram showing procedures of preparing a plasmid (pETarsRgfp) harbored in E.  coli  BL21 (DE3) (pLysS) for recombinant ArsR-GFP production.<br><br>
[Figure 5] Figure 5 is a diagram showing the complex formation between Pars-DNA and an ArsR-GFP protein by native-PAGE as well as the dissociation of the complex by the addition of arsenious acid.<br><br>
[Figure 6] Figure 6 is a diagram showing a result of analyzing, by   electrophoresis,   protein   after   addition   to Pars-DNA-immobilized wells.  Lane 1: a fraction obtained by adding a crude protein extracts to wells, removing the extracts after reaction, washing the wells, and then collecting the bound protein; lane 2: a fraction obtained by adding a crude protein extracts to wells, removing the extracts after reaction, washing the wells, and then collecting the bound protein; lane 3: a crude protein extracts; and lane 4: a molecular weight marker.<br><br>
[Figure 7] Figure 7 is a diagram showing change in the amount of an ArsR-GFP protein bound to a Pars-DNA-immobilized plate along with change in the amount of crude protein extracts added thereto.<br><br>
[Figure 8] Figure 8 is a diagram showing results of detecting arsenious acid by utilizing a biosensor utilizing Pars-DNA as an element.  * and ** represent significant differences of P
<br>
[Figure 9] Figure 9 is a diagram showing change in the amount of a Pars-ArsR-GFP complex depending on washing conditions (buffer).  Each measurement value represents fluorescence intensity measured after the completion of each wash.<br><br>
[Figure 10] Figure 10 is a diagram showing the flow of reduction treatment of pentavalent arsenic As(V) to trivalent arsenic As(III).<br><br>
[Figure 11] Figure 11 is a diagram showing difference in response of ArsR-GFP to different arsenic forms.  The graph depicts mean ± standard deviation of n=3.<br><br>
[Figure 12] Figure 12 is a diagram showing results of detecting As(V) by sodium thiosulfate reduction treatment. The arsenic concentration was set to 100 µg/L in both samples. The graph depicts mean ± standard deviation of n=3.<br><br>
[Figure 13] Figure 13 is a diagram showing the influence of difference in buffer and pH on arsenic detection. Buffers and pHs at the time of preparation are shown. The graph depicts mean ± standard deviation of n=3.<br><br>
[Figure 14] Figure 14 is a diagram showing a standard curve of ArsR-GFP in As(III) detection.  The plot depicts mean ± standard deviation of n=3. R2=0.95.<br><br>
[Figure 15] Figure 15 is a diagram showing results of studying the cross-response of an arsenic sensor. The graph depicts mean ± standard deviation of n=3, wherein evaluation was conducted with the fluorescence intensity of a sample without the addition of metal (-) as 1.<br><br>
[Figure 16] Figure 16 is a diagram showing procedures of preparing a plasmid (pETcadCgfp) harbored in E. coli BL21 (DE3) (pLysS) for recombinant CadC-GFP protein production.<br><br>
[Figure 17] Figure 17 is a diagram showing the complex formation between Pcad-DNA and a CadC-GFP protein by native-PAGE as well as the dissociation of the complex by the addition of divalent lead.<br><br>
[Figure 18] Figure 18 is a schematic diagram showing the confirmation of complex formation of Pcad and CadC-GFP in a microplate.  El and E2 each represent a Pcad34-immobilized well, and E3 represents a non-immobilized well.<br><br>
[Figure 19] Figure 19 is a diagram showing a result of analyzing protein remaining in Pcad34-immobilized wells. For samples in the lanes 1, 3, and 4, see Figure 18.  The lane 2 shows CadC-GFP-containing crude protein extracts, and the lane 5 shows a molecular weight marker (Prestained Protein Marker, Broad Range (6 to 175 kDa), New England Biolabs Inc.). <br><br>
[Figure 20] Figure 20 is a diagram showing change in the amount of CadC-GFP bound according to the concentration of Pcad34 added to wells.<br><br>
[Figure 21] Figure 21 is a diagram showing results of studying the amount of CadC-GFP-containing crude protein extracts added. The upper diagram shows measurement results after addition to Pcad34-immobilized wells, and the lower diagram shows measurement results after two washes of the wells. The plot depicts mean ± standard deviation of n=3.<br><br>
[Figure 22] Figure 22 is a diagram showing the influence of NaC1 addition before incubation on a lead detection test. The graph depicts mean ± standard deviation of n=3.<br><br>
[Figure 23] Figure 23 is a diagram showing a standard curve of CadC-GFP in Pb(II) and Cd(II) detection. The plot depicts mean ± standard deviation of n=3.   * and ** represent significant differences of 5% and 1% levels, respectively, relative to 0 µg/L.<br><br>
[Figure 24] Figure 24 is a diagram showing the spectral overlap between donor and acceptor fluorescent molecules at the occurrence of FRET.<br><br>
[Figure 25] Figure 25 is a diagram showing the principle of lead and cadmium compound detection via FRET in a biosensor.<br><br>
[Figure 26] Figure 26 is a diagram showing the principle of arsenic compound detection via FRET in a biosensor.<br><br>
[Figure 27] Figure 27 is a diagram showing the summary of generating of an E.   coli  strain for His tag-unmodified ZntR protein expression.<br><br>
[Figure 28] Figure 28 is a diagram showing results of confirming recombinant ZntR protein expression.<br><br>
[Figure 29] Figure 29 is a diagram showing chromatogram results of ZntR protein purification using a heparin column.<br><br>
[Figure 30] Figure 30 is a diagram showing results of confirming a ZntR protein in fractions purified using a heparin column.<br><br>
[Figure 31] Figure 31 is a diagram showing results of confirming a ZntR protein contained in partially purified fractions.<br><br>
[Figure 32] Figure 32 is a diagram showing results of EMSA using probe DNA labeled with FITC (left) or TAMRA (right).<br><br>
[Figure 33] Figure 33 is a diagram showing results of EMSA using probe DNA double-labeled with FITC and TAMRA.<br><br>
[Figure 34] Figure 34 is a diagram showing each fluorescence spectrum obtained with or without Pb addition from a reaction solution containing double-labeled probe DNA mixed with a partially purified ZntR protein solution.<br><br>
[Figure 35] Figure 35 is a diagram showing the summary of generating of an E.  coli  strain for recombinant ArsR protein expression.<br><br>
[Figure 36] Figure 36 is a diagram showing the summary ofgenerating of an E. coli strain for recombinant ArsR-GFP fusion protein expression.<br><br>
[Figure 37] Figure 37 is a diagram showing results of confirming recombinant ArsR-GFP fusion protein expression.<br><br>
[Figure 38] Figure 38 is a diagram showing chromatogram results with A280 and GFP fluorescence intensity as indexes in purification using a heparin column.<br><br>
[Figure 39] Figure 39 is a diagram showing results of confirming an ArsR-GFP fusion protein in fractions purified using a heparin column.<br><br>
[Figure 40] Figure 40 is a diagram showing chromatogram results with A2BO and GFP fluorescence intensity as indexes in purification using an ion-exchange column.<br><br>
[Figure 41] Figure 41 is a diagram showing results of confirming an ArsR-GFP fusion protein in fractions purified using an ion-exchange column.<br><br>
[Figure 42] Figure 42 is a diagram showing results of an As detection test using ParsR-350 probe DNA and an ArsR-GFP fusion protein.<br><br>
[Figure 43] Figure 43 is a diagram showing results of an As detection test using ParsR-50 probe DNA and an ArsR-GFP fusion protein.<br><br>
[Figure 44] Figure 44 is a diagram showing results of an As detection test using R773-50 probe DNA and an ArsR-GFP fusion protein.<br><br>
Best Mode of Carrying Out the Invention<br><br>
[0021]<br>
A method for detecting or quantifying an analyte in a test sample according to the present invention is not particularly limited as long as the method is [1] a method comprising: reacting, in the presence of the test sample, a sensor protein specifically binding to the analyte in the test sample, with a nucleic acid comprising a sequence that is specifically recognized by the sensor protein, wherein the nucleic acid is immobilized on a support; and detecting and/or assaying the sensor protein bound to the immobilized nucleic acid in an aqueous system (in the presence of water), or [2] a method comprising: reacting, in the presence of the test sample, a sensor protein specifically binding to the analyte, with a nucleic acid comprising a sequence that is specifically recognized by the sensor protein, wherein the sensor protein is immobilized on a support; and detecting and/or assaying the nucleic acid bound to the immobilized sensor protein in an aqueous system. In the method [1], the sensor protein bound to the immobilized nucleic acid can be detected and/or assayed in an aqueous system by utilizing the sensor protein that is labeled with a detectable marker or fused with a detectable marker protein and removing a sensor protein unbound to the immobilized nucleic acid from the system. Moreover, in the method [2], the nucleic acid bound to the immobilized sensor protein can be detected and/or assayed in an aqueous system by utilizing the nucleic acid that is labeled with a detectable marker and removing a nucleic acid unbound to the immobilized sensor protein from the system. Moreover, examples of the test sample can preferably include, but not particularly limited to, river water, sewage, well water, industrial waste water, and polluted soil containing heavy metal. Solid matter such as polluted soil can be suspended in water or extracted with water, for use. <br><br>
[0022]<br>
Moreover, a kit for detecting or quantifying an analyte in a test sample according to the present invention is not particularly limited as long as the kit is [3] a kit for detecting or quantifying an analyte in a test sample, comprising: a sensor protein that specifically binds to the analyte; and a nucleic acid comprising a sequence that is specifically recognized by the sensor protein, wherein the nucleic acid is immobilized on a support, or [4] a kit for detecting or quantifying an analyte in a test sample, comprising: a sensor protein that specifically binds to the analyte, wherein the sensor protein is immobilized on a support; and a nucleic acid comprising a sequence that is specifically recognized by the sensor protein. In the kit [ 3 ] , the sensor protein bound to the immobilized nucleic acid can be detected and/or assayed by utilizing the sensor protein that is labeled with a detectable marker or fused with a detectable marker protein. Moreover, in the kit [4], the nucleic acid bound to the immobilized sensor protein can be detected and/or assayed by utilizing the nucleic acid that is labeled with a detectable marker. <br><br>
[0023]<br>
The sensor protein used in the method for detecting and/or quantifying an analyte in a test sample according to the present invention is not particularly limited as long as the sensor protein is a protein that is capable of specifically binding to a nucleic acid comprising the predetermined sequence and whose binding to the nucleic acid is inhibited by the predetermined analyte. Preferable examples thereof can include sensor proteins encoded by metal-responsive operon. Examples of the metal-responsive operon can specifically include, but not particularly limited to, ars encoding an ArsR protein, czc encoding CzcR, cad encoding CadC, chr encoding ChrB, mer encoding MerR1 and MerR2, and pbr encoding PbrR. Among them, ars and cad can be shown preferably. Sensor proteins encoded by these metal-responsive operons recognize a DNA sequence in their respective particular metal-responsive operon regions. However, in the presence of a particular metal ion, these sensor proteins bind to the metal ion and thus fail to bind to the DNA due to their altered higher order structures. For example, the ArsR protein binds to ars region DNA, and this binding is inhibited by arsenic. Moreover, the CadC protein binds to cad region DNA, and this binding is inhibited by cadmium and/or lead. <br><br>
[0024]<br>
In the method [1] for detecting and/or quantifying an analyte in a test sample according to the present invention, it is preferred that the sensor protein should be labeled with a detectable marker. The detectable marker is not particularly limited as long as it is a previously known marker for peptide labeling. Examples thereof can specifically include: radioisotopes such as 32P, 3H, 14C, and125I; fluorescent materials such as FITC, TAMRA, SYBR Green, dansyl chloride, and tetramethylrhodamine isothiocyanate; biologically related binding structures such as biotin and digoxigenin; and bioluminescent or chemiluminescent compounds. Moreover, the sensor protein may be fused with a detectable marker protein to form a fusion protein. Examples of the marker protein can specifically include: marker proteins including enzymes such as alkaline phosphatase and HRP, antibody Fc regions, and fluorescent materials such as GFP; binding proteins having binding ability, such as MBP, lectin, and avidin; and epitope tag proteins such as HA, FLAG, andMyc. Particularly, ArsR-GFP, a fusion protein of an ArsR protein and GFP, has binding ability to Pars-DNA, which is a recognition sequence of the ArsR protein, and their binding is inhibited by arsenious acid in a concentration-dependent manner. Thus, this fusion protein can be utilized as an excellent protein for an arsenious acid biosensor in the method for detecting and/or quantifying arsenious acid according to the present invention. <br><br>
[0025]<br>
Moreover, in the method [2] for detecting and/or quantifying an analyte in a test sample according to the present invention, it is preferred that the nucleic acid comprising a sequence that is specifically recognized by the sensor protein should be labeled with a detectable marker. The detectable marker is not particularly limited as long as it is a previously known marker for nucleic acid labeling. Examples thereof can specifically include: radioisotopes such as 32P, 3H, 14C, and 125I; fluorescent materials such as FITC, TAMRA, SYBR Green, dansyl chloride, and tetramethylrhodamine isothiocyanate; biologically related binding structures such as biotin and digoxigenin; and bioluminescent or chemiluminescent compounds. <br><br>
[0026]<br>
In the method for detecting and/or quantifying an analyte in a test sample according to the present invention, the support on which the sensor protein or the nucleic acid is immobilized is not particularly limited as long as it is a support known in the art. Examples thereof can specifically include plastics (polystyrene, polyamide, polyethylene, polypropylene, etc.), agarose, cellulose, hydrophilic polyvinyl alcohol, acrylic polymers, polyacrylamide glass, and metal. Moreover, a method for immobilizing the sensor protein or the nucleic acid onto the support is not particularly limited as long as it is a method known in the art. For example, a physical adsorption method, a diazo method, a peptide method (acid amide derivative method, carboxy chloride resin method, carbodiimide resin method, maleic anhydride derivative method, isocyanate derivative method, cyanogen bromide-activated polysaccharide method, cellulose carbonate derivative method or method using a condensing reagent) , or a covalent bond method (carrier binding method using a cross-linking reagent, such as alkylation method) can be adopted. Among them, the physical adsorption method is preferable. Moreover, when the sensor protein or the nucleic acid is immobilized on the support, the immobilization may be performed indirectly via another substance. Specifically, the nucleic acid can be labeled with biotin, which is then bound to a support (plastic plate) through physical adsorption to immobilize the nucleic acid onto the support, as shown in Examples below. <br><br>
[0027]<br>
Thus, in the present invention, it has been shown in principle that heavy metal can be detected with a biosensor for arsenic or lead/cadmium detection utilizing bacterial promoter region DNA and a sensor protein as elements. This sensor is a cell-free sensor that directly captures change occurring in a bacterial transcriptional switch. Its feature is that, change between two states, i.e., binding and dissociation occurring between the sensor protein and the promoter region DNA constituting the transcriptional switch, is captured by utilizing the fluorescence of a green fluorescent protein (GFP) or the like. Hereinafter, the method of the present invention will be described more specifically. <br><br>
[0028]<br>
In the detection/quantification method of the present invention, a sensor element is immobilized in wells of a 9 6 -well microplate to prepare the sensor portion as a replaceable/detachable module. The element to be immobilized is any of two possible elements, i.e. , the promoter region DNA and the sensor protein. When the promoter region DNA is immobilized, promoter region DNA 3'-terminally labeled with biotin can be reacted with a streptavidin-immobilized 96-well microplate to immobilize the DNA thereon. In this system, a sensor protein-GFP solution is reacted in the promoter region DNA-immobilized wells of the microplate, followed by washing operation, and the fluorescence intensity of the sensor protein-GFP keeping the binding state with the promoter region DNA in the wells is measured with a fluorescence microplate reader. <br><br>
[0029]<br>
In the method of the present invention, for example, each E. coli strain producing sensor protein ArsR-GFP or CadC-GFP was generated. These recombinant proteins were confirmed to bind to promoter region DNA and heavy metal. In biosensor development using ArsR-GFP, it was further revealed that fluorescence intensity is decreased by mixing an As(III)-containing sample with ArsR-GFP. Next, study was conducted on conditions for stabilizing the binding between promoter region DNA and sensor protein-GFP, or on conditions for emphasizing change in state between the binding and dissociation of a complex. Furthermore, in biosensor development using CadC-GFP, it was revealed that fluorescence intensity is decreased by mixing a Pb(II)- and/or Cd(II)-containing sample with CadC-GFP. <br><br>
[0030]<br>
As a result, it was first shown that the mixing of sensor protein-GFP-containing crude protein extracts with a sample, salmon sperm DNA, and a buffer, followed by 30-minute incubation at room temperature or on ice is important for the procedures of a detection test. A method also seemed to be possible, wherein sensor protein-GFP was added to wells to form a DNA-sensor protein-GFP complex beforehand, and then, a sample was added thereto, followed by measurement of the degree of protein dissociation. However, this procedure did not work effectively. Moreover, it was shown that the presence of 40 mM or higher NaC1at the time of 30-minute incubation is important for emphasizing variables of fluorescence intensity in the subsequent measurement. <br><br>
[0031]<br>
In arsenic detection, it was possible to detect As(V) at the same level as As(III) by subjecting As(V) to reduction treatment with sodium thiosulfate. In this case, after the addition of hydrochloric acid, to produce an acidic condition, sodium thiosulfate is then added and finally neutralization was performed with sodium hydroxide. Through this process, a constant concentration of ions is present in the sample, as in the addition of NaCl. Moreover, study on the optimal pH of the buffer demonstrated that the sensor element most highly exerts its functions around pH 7.4. It was further revealed that not only is pH important, but also the effect differs depending on the type of the buffer used. Phosphate and tris(hydroxymethyl)aminomethane buffers were confirmed to be suitable for ArsR-GFP and CadC-GFP, respectively. The lower detection limit was 5 µg/L for arsenic, 10 µg/L for lead, and 1 µg/L for cadmium, demonstrating that detection is achieved at values equal to or lower than drinking water quality standards specified by the World Health Organization. Depending on future study, higher sensitivity may be achieved. The detection is expected to be completed within a required time of 40 minutes including mixing of a sample with protein extracts and the first 15-minute incubation. Neither ArsR-GFP nor CadC-GFP exhibited cross-response to other 100 [AM metals, demonstrating that they selectively respond to the metal of interest. <br><br>
[0032]<br>
Hereinafter, the present invention will be described more specifically with reference to Examples. However, the technical scope of the present invention is not limited to these examples.<br><br>
Example 1<br><br>
[0033]<br><br>
[Principle of biosensor]<br>
An attempt was made to develop a toxic metal biosensor utilizing the interaction between a fluorescently labeled sensor protein and DNA. As shown in Figure 1, sensor proteins such as ArsR and CadC proteins are responsible for regulating the transcriptional activity of toxic metal resistance operon in E. coli or S. aureus. In the absence of toxic metal ions, these proteins bind to promoter DNA to form a sensor protein-promoter DNA complex. A biosensor for arsenic or lead/cadmium compound detection was developed by utilizing such properties of the sensor proteins. Figure 2 shows the principle of the biosensor. <br><br>
[0034]<br>
The immobilization of DNA onto substrate surface was focused on two substances: a low-molecular-weight vitamin biotin; and a basic glycoprotein streptavidin. Binding formed between these substances has an association constant as very high as 10 5 M" or larger and is almost irreversible binding. The reason for focusing on this binding is that it could be advantageous in that: the DNA strand can be modified with biotin; streptavidin can be immobilized to a 96-well microplate substrate; the binding is routinely used in enzyme-linked immunosorbent assay; and the binding does not directly influence the DNA strand. Furthermore, use of a 96-well microplate enables simultaneous multi-sample treatment. Therefore, the construction of a high-throughput screening/detection system can be expected. A streptavidin-modified 96-well microplate (Reacti-Bind Streptavidin High Binding Capacity Coated 96-Well Plates: PIERCE) was adopted as a substrate. Corona fluorescence microplate reader MTP-601Lab (Hitachi High-Technologies Corp.) was used in fluorescence intensity measurement. The measurement conditions involved 490 nm for an excitation filter and 530 nm for a fluoresence measurement filter, and detection sensitivity set to "AUTO". <br><br>
[0035] <br><br>
[Components of biosensor]<br>
Figure 3 shows components of an arsenic compound biosensor. In E. colt, arsB and arsC genes encoding an arsenic efflux pump and an arsR gene encoding an ArsR sensor protein form operon in the order of arsR-arsB-arsC, and its transcriptional activity is regulated by the ArsR protein. An ArsR-GFP fusion protein and 50-bp promoter region DNA (Pars-DNA) centered on an ArsR protein-binding site (operator sequence) upstream of the arsR gene were selected for the arsenic compound biosensor. For the promoter region DNA, ParsR-50-S-3-B (the sequence of ParsR-50-S is shown in SEQ ID NO: 1) and ParsR-50-A (SEQ  ID NO:   2)   shown  in Table  1  were mixed in equal  amounts to form double-stranded DNA 3' -terminally (downstream site in the   promoter   sequence)    labeled   with   biotin    (hereinafter, referred to as Pars-biotin).    Next,  Pars-biotin was adjusted to a concentration of 25 pmol/100 µL and used in immobilization.<br><br>
[0036]<br><br>
[Table 1]<br><br>
The underline represents an ArsR-binding sequence.<br><br>
[0037]<br>
On the other hand, in S. aureus, the cadA gene encoding a lead/cadmium efflux pump and the cadC gene encoding a CadC sensor protein form operon in the order of cadC-cadA, and its transcription is regulated by the CadC protein. A CadC-GFP fusion protein and 34-bp promoter region DNA (Pca<i-dna centered on a cadc protein-binding site sequence upstream of the gene were selected for lead compound biosensor. promoter region dna pcadc34-s-3-b pcadc34-s is shown in seq id no: and pcadc34-a table mixed equal amounts to form double-stranded oligonucleotide labeled with biotin referred as pcad34-biotin next was adjusted concentration pmol used immobilization.></i-dna><br>
[0038] <br><br>
[Table 2]<br><br>
The underline represents a CadC-binding sequence.<br><br>
Example 2<br><br>
[0039]<br><br>
[Experimental materials and method]<br><br>
(1)	Strain (E.   coli  strain)<br>
JM109 and DH5a (manufactured by TAKARA BIO INC.) were used in gene cloning. Moreover, BL21 (DE3) pLysS (Novagen) in which the arsR gene of E. coli K12 was inserted upstream of the Acgfp1 gene of pAcGFP1 plasmid DNA, in the same reading frame as the Acgfp1 gene was used in recombinant protein production. Moreover, an LB medium was used in transformant culture and supplemented, if necessary, with ampicillin (Amp) and chloramphenicol (Cm) at final concentrations of 50 µg/mL and 34 µg/mL, respectively. <br><br>
[0040] <br>
(2)	Amplification of arsR, cadC, and Acgfp1 nucleotide sequences by PCR<br><br>
Pfx50 DNA Polymerase (manufactured by Invitrogen Corp.) was used in polymerase chain reaction (PCR). E. coli K12 arsR, cadC encoded by endogenous plasmid DNA pI258 of S. aureus NCTC50581, and Acgfp1 encoded by pAcGFP1 plasmid DNA (manufactured by Clontech Laboratories, Inc. ) were separately amplified. <br><br>
[0041]<br><br>
(3)	Reagents<br><br>
Sodium metaarsenite 98% (As(III)) (manufactured by<br>
Sigma-Aldrich Corp.) and lead(II) acetate trihydrate special grade (manufactured by Wako Pure Chemical Industries, Ltd.) were used as arsenious acid and divalent lead, respectively. <br><br>
[0042]<br><br>
(4)	Plasmid DNA<br>
pGEM-T vector plasmid DNA (manufactured by Promega Corp.) was used in the cloning of DNA fragments amplified by PCR, and pET-3a plasmid DNA (manufactured by Novagen) was used in the preparation of an expression unit for recombinant protein production. <br><br>
[0043]<br><br>
(5)	Recombinant protein production conditions and preparation method<br>
BL21 (DE3) pLysS transformants for recombinant protein expression were cultured overnight at 37°C in 5 mL of a medium supplemented with Amp and Cm (Amp+Cm+LB), and the obtained overnight culture was inoculated to 0.5 mL of Amp+Cm+LB. This was cultured at 37°C for approximately 12 hours without the addition of isopropyl-|3-D-thiogalactopyranoside (IPTG). After culturing, the bacterial cells were collected and washed twice with a 50 mM Tris-HCl buffer (pH 7.4). Then, the cells were resuspended in 4 mL of a buffer for cell disruption (Tris-HC1  containing  15%  glycerol)  and  subjected  to freeze-thaw procedure once at -80°C. After sonication, a supernatant obtained by centrifugation at 15000 rpm for 15 minutes was used as crude recombinant protein extracts. The crude protein extracts Were cryopreserved at -80°C and used after thawing at the time of a test. For arsenious acid (As(III)) detection using a microplate, a 10 mM phosphate buffer (PBS, pH 6 . 0) was used instead of the Tris-HC1. Moreover, cells were suspended in 2 mL of a buffer for cell disruption (PBS containing 15%) before sonication.<br><br>
[0044]<br><br>
(6) Gel shift assay (Electrophoresis Mobility Shift Assay: EMS A)<br><br>
Promoter DNA, a recombinant protein, and metal ions were mixed according to the composition shown in Table 3, and incubated for 3 hours (test using ArsR-GFP) or 1 hour (test using CadC-GFP) on ice. To the reaction solution, 5 µL of a sample buffer for native-PAGE of Table 4 was added to adjust the volume to 25 µL in total. Then, a gel in which DNA and protein in the reaction solution were subjected to the separation by native-PAGE was irradiated with blue light using an LED light source VISIRAYS (AE-6935B, manufactured by ATTO Corp.) under dark conditions to visualize the GFP fusion protein. <br><br>
[0045] <br><br>
[Table 3]<br><br>
[0046] <br><br>
[Table 4]<br><br>
[0047]<br><br>
 (7) Arsenious acid (As(III)) detection test using microplate <br><br>
A solution containing 30 pmol of Parg-DNA dissolved in 110 µL of a buffer for DNA immobilization (Tris buffered saline (TBS) containing 0.05% Tween 20) was added to a microplate (Reacti-Bind Streptavidin Coated High Binding Capacity (HBC) Black 96-Well Plates, manufactured by Pierce) and incubated at room temperature for 2 hours to immobilize Pars-DNA onto well surface. The wells were washed twice with 200 µL of a buffer for DNA immobilization and then once with 200 [XL of TBS. Crude protein extracts and an arsenious acid solution were mixed according to the composition shown in Table 5 and incubated for 2 hours on ice. Then, the mixed solution was added to the plate and incubated at room temperature for 1 hour. After removal of the supernatant, the wells were washed three times with 200 µL of a buffer for washing (10 mM PBS containing 0.05% Tween 20, pH 6.0), and 150 µL of 50 mM Tris-HCL (pH 7.9) was finally added thereto. Ten minutes later, fluorescence intensity was measured with Fluorescent Microplate Reader MTP-601Lab (manufactured by Corona-Hitachi High-Technologies Corp.). In this measurement, 492- and 530-nm band-pass filters were used on excitation light and detector sides, respectively. <br><br>
[0048] <br><br>
[Table 5]<br><br>
Example 3<br><br>
[0049]<br><br>
[Results and discussion: arsenious acid biosensor]<br><br>
(1)	Preparation of ArsR-GFP fusion protein<br>
First, pAcGFParsR in which the arsR gene of E. coli K12 was inserted upstream of the Acgfpl gene of pAcGFP1 plasmid DNA, in the same reading frame as the Acgfp1 gene was used as a template in PCR to amplify a DNA fragment containing ars R and Acgfp1 (this DNA fragment is referred to as arsRgfp) . Each primer used in this PCR consisted of 20 bases complementary to the 5' end of the arsR gene or the 3' end of the Acgfp1 gene and further comprised an Nde1 site added to the 5' end thereof. arsRgfp was inserted in pGEM-T vector plasmid DNA and cloned to obtain pGEMarsRgfp (Figure 4). arsRgfp excised from the plasmid DNA by the Nde1 digestion of pGEMarsRgfp was separated by agarose gel electrophoresis and collected. pET-3a was digested with Nde1 and ligated with the collected arsRgfp to obtain pETarsRgfp. E. coli BL21 (DE3) pLysS was transformed with pETarsRgfp to generate an ArsR-GFP fusion protein-producing strain. <br><br>
[0050]<br><br>
(2)	Functional evaluation of ArsR-GFP fusion protein<br><br>
Whether or not the recombinant protein thus obtained had functionality as an ArsR protein was confirmed by gel shift assay. In the gel shift assay, when protein and nucleic acid form a complex via specific binding, such molecules differ in migration distance from nucleic acid alone or protein alone during electrophoresis, leading to difference in the position of a band on a gel. As a result, the presence or absence of the specific binding between the protein and the nucleic acid can be determined. As a result of mixing ArsR-GFP protein-containing crude protein extracts with Pars-DNA, the presumed band of the Pars-ArsR-GFP complex was confirmed as shown in Figure 5. Moreover, this band was not confirmed in the electrophoresis of a reaction solution of crude protein extracts without the addition of Pars-DNA, supporting that this band was derived from the Pars-ArsR-GFP complex. Moreover, when arsenious acid was added to the reaction solution, the band of the Pars-ArsR-GFP complex was confirmed to become lighter with increases in the concentration of the added arsenious acid. These results demonstrated that the ArsR-GFP protein is capable of specifically binding to Pars-DNA to form a complex and that the binding with Parg-DNA is inhibited by the presence of arsenious acid. These results further mean that fluorescent modification with GFP does not impair the function of the ArsR protein. <br><br>
[0051]<br>
(3) Development of arsenious acid detection system utilizing microplate<br>
ArsR-GFP protein-containing crude protein extracts were added to Pars-DNA-immobilized wells to form a Pars-ArsR-GFP complex on well surface. Then, after removal of the crude protein extracts, protein on the wells (without washing) was denatured with a sample buffer for SDS-PAGE and collected. As a result of analysis by electrophoresis, a dark band was detected at the putative molecular weight position (41.8 kDa) of the ArsR-GFP protein, though a plurality of other bands were also confirmed (Figure 6, lane 2). Similar analysis was conducted on samples collected after Pars-ArsR-GFP complex formation on well surface and washing. As a result, almost completely purified protein was confirmed at the position of ArsR-GFP (Figure 6, lane 1) . In the sample of any of the lanes, the purification of the protein of interest was shown to drastically advance, compared with the lane 3 of Figure 6 in which crude protein extracts were directly developed. These results demonstrated that Parg-DNA and ArsR-GFP can bind to each other even on well surface, with their high specificity maintained, only by immobilizing Pars-DNA on well surface and adding crude protein extracts, followed by incubation. This result further shows that complicated column operation is unnecessary for protein purification. Thus, cost required for biosensor preparation can be reduced. <br><br>
[0052]<br><br>
(4) Amount of ArsR-GFP-containing crude protein extracts added Next, how fluorescence intensity remaining after well washing was changed depending on change in the amount of crude protein extracts added after Pars-DNA immobilization on well surface was examined. The abscissa of Figure 7 represents the volume of crude protein extracts per 100 mL volume in a well, and the ordinate represents the fluorescence intensity of ArsR-GFP forming a complex with Pars-DNA on the well. The fluorescence intensity was increased with increases in the volume of the crude extracts and reached a plateau at 40 mL. This indicates that 40 mL volume of crude protein extracts per 100 mL volume suffices for forming a complex of ArsR-GFP with almost all Pars-DNA immobilized on well surface. When the extracts are added in an amount over this volume, an excessive amount of free ArsR-GFP proteins is present in the reaction system. In arsenious acid detection, this free ArsR-GFP present in excess binds to Pars-DNA, presumably causing reduction in detection sensitivity.  Thus, the amount of ArsR-GFP corresponding to 40 mL of crude protein extracts was temporarily applied to arsenious acid detection. <br><br>
[0053]<br>
For arsenious acid detection, the amount corresponding to 40 mL of crude protein extracts was concentrated to 5 mL to prepare crude extracts. Then, the crude protein extracts and arsenious acid were mixed at a ratio of 5:95. Three hours later, fluorescence intensity was measured according to the procedures described in Example 2 [Experimental materials and method]. As a result, significant decrease in fluorescence intensity was observed by the addition of arsenious acid, and this decline was observed to tend to be larger at 100 µg/L than at 50 µg/L. These results demonstrated that a biosensor for arsenious acid detection utilizing DNA as an element in a microplate is capable of detecting 50 µg/L arsenious acid in principle (Figure 8). <br><br>
[0054] <br><br>
(5) Study on washing conditions<br><br>
As described above, it was determined that a complex consisting of Pars and ArsR-GFP can be constructed as a sensor element on a 96-well microplate. Moreover, as a result of removing a redundant ArsR-GFP protein solution from Pars-immobilized wells , which were then washed with 10 mM PB-T shown in Table 6, fluorescence intensity derived from GFP in the wells was observed to be decreased with increases in the number of washes. This decrease may be due to the dissociation of ArsR-GFP from Pars by the washing operation. Therefore, it was concluded that washing conditions capable of maintaining the sensor element complex on a substrate and removing redundant ArsR-GFP or contaminating proteins from the system is required. Thus, washing operation was performed with two buffers differing in pH shown below in Table 6 by utilizing ArsR-GFP, and the degree of decrease in fluorescence intensity was examined. <br><br>
[0055] <br><br>
[Table 6]<br><br>
Buffer used in study on washing conditions<br><br>
[0056]<br>
In one washing operation, 200 µL of a buffer was added to the wells respectively. The 96-well microplate was held by hand and lightly shaken for 5 seconds to shake off the buffer. Again, 200 µL of a buffer was added to the wells respectively, followed by fluorescence measurement.  This operation was performed 4 times. Finally, 100 µL of TBS-T (Table 6) was added to all the wells respectively to adjust pH to 7.4, followed by fluorescence intensity measurement (Figure 9). <br><br>
[0057]<br>
Fluorescence proteins such as GFP are known to have properties in which fluorescence intensity varies depending on pH. For GFP, it is known that fluorescence intensity gets higher with increases in pH around neutral. Fluorescence intensity after the 1st to 3rd washes with PB-T was measured under weakly acidic conditions of pH 6.0 and may therefore be lower than that obtained in measurement under weakly basic conditions of pH 7.4 after washing with TBS-T. For both of these two buffers, fluorescence intensity was observed to be decreased with increases in the number of washes. After the completion of the 4th washing operation, pH was adjusted to 7.4 by the addition of TBS-T of pH 7.4, followed by fluorescence intensity measurement. As a result, washing with PB-T produced almost the same value as fluorescence intensity at the completion of the 1st washing with TBS-T. The rate of decrease in complex attributed to washing operation was lower with PB-T than with TBS-T. Therefore, it was determined that PB-T is more suitable as a buffer for washing. The subsequent tests adopted PB-T (pH 6.0) as a buffer for washing, which was used by appropriately changing the number of washes and washing time. <br><br>
[0058] <br><br>
(6) Study on arsenic reduction treatment method<br><br>
The sensor protein ArsR is known to have different affinity for inorganic arsenics trivalent arsenic As (III) and pentavalent arsenic As(V) , and have higher affinity for As (III) . However, arsenic shown in the drinking water quality standards does not differentiate between arsenious acid and arsenic acid. For a sensor utilizing ArsR-GFP as an element, it is preferred that both the arsenics As(III) and As(V) should be detected with equal sensitivity. Thus, whether or not As(V) can be detected at a level equal to As(III) by reducing As(V) to As(III) was studied. In the test, sodium arsenate and sodium arsenite were used as As(V) and As(III), respectively, and sodium thiosulfate was used as a reducing agent.<br><br><br>
[0059]<br>
The reduction treatment of samples was performed according to the procedures shown in Figure 10. 20.0 µL of a 2 N hydrochloric acid solution was added to 450 µL of the sample to adjust pH to around 1. Next, 1. 0 (J.L of a 250 mM sodium thiosulfate aqueous solution prepared before use was added thereto and then incubated at room temperature for 10 minutes. Ten minutes later, the solution was neutralized by the addition of 16.0 µL of a 2.5 N sodium hydroxide solution. The confirmation of pH was performed with a pH test paper. <br><br>
[0060]<br>
92 (XL of a sample containing As that was adjusted to the predetermined concentration and subjected or not subjected to reduction treatment was mixed with 7.5 µL of ArsR-GFP-containing crude protein extracts and 0.5 µL of 10 mg/mL salmon sperm DNA and incubated for 30 minutes on ice. Then, the mixed solution was added to Pars-immobilized wells and shaken at room temperature for 2 hours. Two hours later, a mixed solution in the wells was removed, and the wells were washed once with 200 [AL of PB-T (Table 6) respectively for 5 seconds. After the washing, 150 µL of a buffer for fluorescence measurement (50 mM Tris-HC1, pH7.9, IMNaC1, 0.1% (w/v) Tween 20) was added to the wells respectively, followed by fluorescence measurement. <br><br>
[0061]<br>
In the sample containing the predetermined concentration of As without reduction treatment, decline in fluorescence intensity was increased in correlation to the concentration of As(III), whereas almost no decrease in fluorescence intensity was observed at 100 µg/L   of As(V) (Figure 11). However, decrease in fluorescence intensity was also observed for As(V) at a level equal to As(III) by performing reduction treatment (Figure 12) . Moreover, when fluorescence intensity from ultrapure water used as a sample was compared between the presence or absence of reduction treatment, no significant difference was observed, demonstrating that the reduction treatment itself does not influence detection. <br><br>
[0062]<br>
The developed arsenic sensor ArsR-GFP did not exhibit the change of fluorescence intensity even in the presence of 100 µg/L As(V) , demonstrating that ArsR-GFP is exceedingly low responsive to As(V). However, change in fluorescence intensity at a level equal to As(III) was shown by performing reduction treatment with sodium thiosulfate. This indicated that the sensor is capable of detecting even As(V) at a level equal to As (III) via reduction from As(V) to As (III). Though data is not shown, it was revealed that As(V) cannot be detected unless reduction treatment with sodium thiosulfate is performed after pH adjustment to a strongly acidic range. Therefore, the addition of an acid such as hydrochloric acid is probably essential for reduction treatment. <br><br>
[0063]<br><br>
(7) Buffer to be added to sample after reduction treatment and optimization of pH thereof<br>
Since the developed arsenic sensor element is protein, the element itself might be denatured under extreme pH conditions and fail to function as a sensor. Therefore, any pH of a sample will need to be brought close to the optimal pH at which functions as a sensor are exerted. Thus, the influence of pH on an arsenic detection test was confirmed. For the detection test, reduction treatment was performed by sequentially adding 20.0 µL of 2 N hydrochloric acid, 5.0 µL of 500 mM ethylenediaminetetraacetic acid (pH 8.0), 1.0 µl of 250 mM sodium thiosulfate, and 16 .0 µL of 2.5 N sodium hydroxide to 445 \iL each of an arsenious acid-free sample and a sample containing 100 µg/L As(III). Then, 25.0 µL of a buffer shown in Table 7 was added thereto, and the influence of pH conditions of the sample was studied. The concentration of the buffer was set to 1 M (final concentration: 50 mM) to enhance its buffering ability. <br><br>
[0064] <br><br>
[Table 7]<br><br>
Buffer added after reduction treatment<br><br>
[0065]<br>
The sample after addition of each buffer, ArsR-GFP-containing crude protein extracts, and salmon sperm DNA were mixed and incubated for 30 minutes on ice. Then, the mixed solution was added to Pars-immobilized wells and shaken at room temperature for 2 hours. After removal of a redundant ArsR-GFP protein solution in the wells, the wells were washed once with 200 ^L of PB-T (Table 6) respectively for 5 seconds. After the washing, 150 µL of a buffer for fluorescence measurement was added to the wells respectively, followed by fluorescence intensity measurement (Figure 13). <br><br>
[0066]<br>
In the comparison among 3 conditions of phosphate buffers, the amount of a Pars-ArsR-GFP complex formed was increased with increases in pH under conditions without the addition of arsenious acid, and decline in the fluorescence intensity of the sample containing 100 µg/L As (III) was also increased. However, in the comparison of two conditions of pH 7.4, such decline was decreased for the Tris buffer, demonstrating that the signal of the sensor is attenuated. This result suggests that different results may be obtained even at the same arsenic concentrations unless the arsenic detection test was conducted at constant pH. The pH adjustment of samples by utilizing a buffer or the like is probably essential. From the results of Figures 13 and 9, it was determined that the arsenic detection test is appropriately conducted with buffering sample pH at a constant value while adjusting pH to around 7.4 and a final phosphate buffer concentration to 50 mM or higher by utilizing approximately 1 M phosphate buffer having high buffering ability. <br><br>
[0067] <br><br>
(8) Standard curve for arsenic detection<br><br>
An arsenious acid detection test was conducted under the washing and fluorescence intensity measurement conditions obtained in the preceding tests. After reduction treatment of samples by the method performed in the paragraph " (7) Buffer to be added to sample after reduction treatment and optimization of pH thereof", 25 µL of a 1 M phosphate buffer of pH 7.4 was added thereto. 95µL of the treated sample thus obtained, 4 . 5 µL of an ArsR-GFP solution, and 0 . 5 µL of 10 mg/mL salmon sperm DNA were mixed and incubated for 30 minutes on ice. The mixed solution was added 30 minutes later to Pars-immobilized wells and shaken at room temperature for 10 minutes. Ten minutes later, a redundant mixed solution was removed, and the wells were washed once with 200 µL of PB-T (Table 6) respectively for 5 seconds. After the washing, 150 µL of 50 mM tris(hydroxymethyl)aminomethane-HC1, pH 7.9 was added to the wells respectively, followed by fluorescence measurement. Figure 14 shows a standard curve of ArsR-GFP in As (III) detection. <br><br>
[0068]<br>
In the test, negative correlation was observed between the concentration of As(III) and fluorescence intensity. Moreover, fluorescence intensity was shown to be significantly decreased at an As (III) concentration of 5 µg/L compared with at an As (III) concentration of 0 µg/L, and decreased by 50% at 50 µg/L. The results of this test demonstrated that the developed arsenic sensor is capable of detecting As(III) at 10 µg/L, which is the drinking water quality standards specified by the World Health Organization, by studying the assay conditions. <br><br>
[0069] <br><br>
(9) Cross-response of arsenic sensor<br><br>
The arsenic sensor was confirmed to be capable of detecting trivalent arsenic on µg/L order. However, its selective response, an important factor of the sensor, remained to be shown. Thus, the arsenic sensor was examined for its response to metal other than arsenic. 1.0 µM, 10 µM, or 100 JAM calcium chloride, cadmium chloride, copper(II) sulfate, iron(II) sulfate, iron(III) chloride, magnesium sulfate, manganese sulfate, lead(II) acetate, and zinc sulfate were tested as samples. The simultaneous detection of As(III) and As(V) require reduction treatment of samples with sodium thiosulfate. Thus, in this test as well, after reduction treatment of samples by the method performed in the paragraph "(7) Buffer to be added to sample after reduction treatment and optimization of pH thereof", 25 µL of a 1 M phosphate buffer of pH 7.4 was added thereto. Then, in the same way as in the preceding paragraph, an ArsR-GFP solution, and salmon sperm DNA were mixed and incubated for 30 minutes on ice. The mixed solution was added to Pars-immobilized wells and shaken at room temperature for 2 hours. Then, after removal of a redundant mixed solution in the wells, the wells were washed once with 200 µL of PB-T (Table 6) respectively for 5 seconds. After the washing, 150 µL of a buffer for fluorescence measurement was added to the wells respectively, followed by fluorescence measurement (Figure 15). <br><br>
[0070]<br>
From the standard curve of arsenic detection of Figure 14, the arsenic sensor is expected to exhibit, for 1 µM (=75 µg/L) As(III), 50% or more decline in fluorescence intensity from that obtained in the presence of 0 µg/L As(III). On the other hand, in the results of this test, the sensor did not exhibit cross-response to each metal even at concentrations 100 times larger there than. Thus, the arsenic sensor was confirmed to respond to arsenic with high selectivity. Presumably, this result is effectively affected by the metal chelating effect of ethylenediaminetetraacetic acid added in the reduction treatment of samples, in addition to the original high specificity of the ArsR protein for arsenic.<br><br>
Example 4<br><br>
[0071]<br><br>
[Results and discussion: lead/cadmium biosensor]<br><br>
(1)	Generating of CadC-GFP fusion protein-producing E. coli strain<br>
A CadC-GFP fusion protein was prepared in the same way as in the ArsR-GFP fusion protein preparation described in Example 3. pGEMarsRgfp was digested with SphI and BamHI, and the excised fragment was removed to obtain pGEMgfp (Figure 16) . pI258 plasmid DNA was used, as a template in PCR to amplify a cadC-containing DNA fragment. Each primer used in this PCR comprised 20 bases complementary to the 5' end of the cadC gene plus SphI and Ndel sites added thereto or 20 bases complementary to the 3' end thereof plus a BamHI site added thereto. cadC was digested with SphI and BamHI and ligated with pGEMgfp to obtain pGEMcadCgfp. cadCgfp excised from the plasmid DNA by the Nde1 digestion of pGEMcadCgfp was separated by agarose gel electrophoresis and collected. pET-3a was digested with Nde1 and ligated with the collected cadCgfp to obtain pETcadCgfp. E. coli BL21 (DE3) pLysS was transformed with pETcadCgfp to generate a CadC-GFP fusion protein-producing strain. <br><br>
[0072]<br><br>
(2)	Functional evaluation of CadC-GFP fusion protein<br>
In the same way as in Example 3, whether or not the CadC-GFP fusion protein had functions as a CadC protein was confirmed by gel shift assay. As a result of mixing CadC-GFP protein-containing crude protein extracts with Pcad-DNA, the presumed band of the Pcad-CadC-GFP complex was confirmed (Figure 17, upper diagram). This band was not confirmed in the electrophoresis of a reaction solution of crude protein extracts without the addition of Pcad-DNA, supporting that this band was derived from the Pcad-CadC-GFP complex. Moreover, when divalent cations lead, calcium, and magnesium ions were separately added to the reaction system, the band of the Pcad-CadC-GFP complex was confirmed to become lighter in all the cases (Figure 17, lower diagram) . However, the band became more significantly lighter by the addition of divalent lead than by the addition of calcium or magnesium. This presumably shows that the CadC-GFP fusion protein binds to divalent lead with given selectivity to change an association or dissociation constant to Pcad-DNA. <br><br>
[0073]<br><br>
(3) Development of lead/cadmium detection system utilizing microplate<br>
A biosensor for detecting lead and cadmium compounds based on the same principle as that of the arsenious acid detection system was constructed by utilizing CadC-GFP fusion protein-containing crude protein extracts and a Pcad-DNA-immobilized microplate. <br><br>
[0074]<br>
Pcad34-biotin having a CadC-binding sequence was immobilized on wells El and E2 shown in Figure 18 . No DNA strand was immobilized on a well E3. To the wells El to E3, 100 (AL of CadC-GFP-containing crude protein extracts was added and shaken at room temperature for 2 hours.  Two hours later, redundant crude protein extracts were removed. 100 ^.L of a protein solubilizing buffer (Table 8) was added to the well El and shaken at room temperature for 30 minutes. Then, the buffer was collected and loaded to lane 3 of electrophoresis (SDS-PAGE). Aside from this, 200 µL each of PB-T (Table 6) was added to the wells E2 and E3 and then removed, and this operation was repeated to perform two washes. Then, 100 µL each of a protein solubilizing buffer was added thereto and shaken at room temperature for 30 minutes to solubilize protein remaining in the wells. The solution was loaded to lane 4 or 1 of electrophoresis.  The collected protein solubilizing buffer was treated by incubation at 98°C for 5 minutes before being loaded. Electrophoresis was performed under conditions involving room temperature at 250 V and a constant current of 20 mA. <br><br>
[0075]<br><br>
[Table 8]<br><br>
[0076]<br>
The results of electrophoresis demonstrated that, compared with the CadC-GFP-containing crude protein extracts in the lane 2 of Figure 19, CadC-GFP was concentrated/purified in the remaining protein by contacting the crude protein extracts with the Pcad34-immobilized wells (lane 3) . This was determined from the main band of protein observed at the molecular weight position of CadC-GFP determined by calculation. Moreover, it was revealed that the washing of the wells further decreased the bands of contaminating proteins and allowed CadC-GFP to be concentrated/purified (lane 4) . On the other hand, it was revealed that the washing washed off CadC-GFP in the Pcad34-non-immobilized well (lane 1). This result demonstrated that CadC-GFP can be bound specifically from crude protein extracts onto a Pcad34-immobilized 96-well microplate. Accordingly, a system was studied in which a complex of Pcad DNA and a CadC-GFP sensor protein was formed as a lead/cadmium sensor element on a 96-well microplate. <br><br>
[0077]<br><br>
(4) Study on amount of Pcad immobilized in lead/cadmium sensor<br>
The amount of Pcad34-biotin (Pcad 3'-terminally labeled with biotin) bound to wells was confirmed and referred to for constructing a sensor.   Pcad34-biotin was adjusted to concentrations of 0 to 50 pmol/100 µL and used in immobilization onto wells. CadC GFP-containing crude protein extracts were prepared by suspending cells in 50 mL of a culture solution of the E. coli BL21 (DE3) pLysS (pETcadCgfp) generated as shown in Figure 16, in 4 mL of TG (50 mM tris(hydroxymethyl)aminomethane-HC1, pH 7.4, 15% (v/v) glycerol) and performing sonication (oscillation output: 3, interval: 50%, 5 minutes, 4 runs) on ice by utilizing UD-201 (TOMY SEIKO CO., LTD.). 99 µl of the crude protein extracts and 1. 0 µL of 10 mg/mL salmon sperm DNA were mixed. The mixture was added to Pcad34-immobilized wells and shaken at room temperature for 2 hours. Two hours later, redundant crude protein extracts were removed, and the wells were washed twice with 200 JAL of PB-T (Table 6) respectively for 5 seconds . After the washing, 150 JAL of 50 mM tris(hydroxymethyl)aminomethane-HC1 (pH 7.9) was added to the wells respectively, followed by fluorescence measurement. As a result, the amount of CadC-GFP bound was observed to be increased with increases in the amount of Pcad34 immobilized up to 20 pmol/100 µL (Figure 20) . As with the results of Pars immobilization, fluorescence intensity reached a plateau at an immobilization amount of 20 pmol/100 µL. Since the amount of CadC-GFP in the crude protein extracts used in this test seemed to be sufficient, the plateau was reached when the amount of Pcad34 immobilized on the well probably was 10-20 pmol/100 JAL. In conclusion, the concentration of Pcad34 used in immobilization for constructing a lead/cadmium sensor was determined to be 20 pmol/100 µL as the upper limit. <br><br>
[0078]<br><br>
(5) Study on amount of CadC-GFP added in lead/cadmium sensor Study was conducted on the optimal amount of CadC-GFP-containing crude protein extracts added with respect to the amount of Pcad34 used (20 pmol/well). Pcad34-biotin was adjusted to 20 pmol/100 (xL and immobilized on well by addition. A mixed solution containing 99 µL of CadC-GFP-containing crude protein extracts and 1.0 µL of 10 mg/mL salmon sperm DNA was prepared. Next, this mixed solution was diluted with TG (50 mM tris(hydroxymethyl)aminomethane-HC1, pH 7.4, 15% (v/v) glycerol) to 0-100 µL/100 µL as a volume ratio to the whole amount of the mixed solution. Then, this diluted sample was added to Pcad34-immobilized wells, followed by fluorescence measurement (Figure 21, upper diagram). Then, after shaking at room temperature for 1 hour, a redundant diluted sample was removed, and the wells were washed twice with 200 µL of PB-T (Table 6) respectively for 5 seconds. After the washing, 150 µL of 50 mM Tris-HC1, pH 7 . 9 was added to the wells respectively, followed by fluorescence measurement (Figure 21,  lower diagram). <br><br>
[0079]<br>
As a result, the amount of CadC-GFP bound to Pcad34 reached a plateau at approximately 80 to 90 [xL/100 JAL volume ratio of the solution added to the well. At the 80-90 (xL/100 µ,L or higher volume ratio of the crude protein extracts used, CadC-GFP was probably be in excess with respect to 20 pmol of the immobilized Pcad34 . This means that in the detection test, detection sensitivity for lead or cadmium is presumably reduced when the concentration of crude protein extracts is higher than this volume ratio of 80 to 90 jiL/100 µL. A lead/cadmium sensor was constructed by utilizing, as a guideline, the concentration of CadC-GFP in the crude protein extracts used in this test. However, as with ArsR-GFP, CadC-GFP-containing crude protein extracts are also an E. coli cell homogenate containing contaminating proteins. Therefore, it is difficult to correctly calculate the concentration of CadC-GFP. Thus, the concentration of CadC-GFP was .determined in an abbreviated manner based on the fluorescence intensity of crude protein extracts at pH 7.4. <br><br>
[0080]<br>
A large difference between CadC-GFP and ArsR-GFP compared is in that, assuming that the fluorescence intensity of each protein solution corresponds to the concentration of sensor protein-GFP, they differ in amount required for allowing binding to the constant amount of a promoter region DNA strand to reach a plateau. CadC-GFP requires adding crude protein extracts such that the product of the fluorescence intensity and the volume is larger. This may be due to the different association constant of each sensor protein-GFP to the promoter region DNA strand, an insufficient number of bases of Pcad34 used, etc. Thus, the influences of the number of Pcad bases and a salt concentration on complex formation were studied. <br><br>
[0081]<br><br>
(6) Study on the number of Pcad bases and salt concentration in construction of lead/cadmium sensor<br><br>
To examine the influence of varying numbers of Pcad bases, double-stranded DNA was newly designed. This was a Pcad34 nucleotide sequence that contained a CadC-binding sequence around a -10-base region at the promoter consensus sequence upstream from the transcription initiation point of cadC on S. aureus pI258 and further contained a CadC-binding sequence around a -35-base region added thereto. To prepare the newly designed double-stranded DNA, Pcad50-S-3-B (the sequence of Pcad50-S is shown in SEQ ID NO: 5) and Pcad50-A (SEQ ID NO: 6) shown in Table 9 were mixed in equal amounts to form a double-stranded oligonucleotide containing a promoter sequence 3'-terminally labeled with biotin (hereinafter, referred to as Pcad50-biotin). Next, Pcad34-biotin and Pcad50-biotiri were adjusted to 20 pmol/100 piL and used in immobilization. <br><br>
[0082]<br><br>
[Table 9]<br><br>
The underline represents a CadC-binding sequence.<br><br>
[0083]<br>
CadC-GFP-containing crude protein extracts were prepared by suspending cells in 300 mL of a culture solution, in 4 mL of TG2 (200 mM tris(hydroxymethyl)aminomethane-HC1, pH 7.4, 15% (v/v) glycerol) and performing sonication (oscillation output: 3, interval: 50%, 5 minutes, 4 runs) on ice by utilizing UD-201 (TOMY SEIKO CO. , LTD. ) . 7. 5 µl of the crude protein extracts, 0.5 µL of 10 mg/mL salmon sperm DNA, and 92 µL of ultrapure water were mixed to adjust the volume to 100 µL. The fluorescence intensity of this solution was 65.3 ± 0.6. This mixed solution was added to Pcad34- or Pcad50-immobilized wells and shaken at room temperature for 2 hours. Two hours later, a mixed solution in the wells was removed, and the wells were washed once with 200 µL of PB-T (Table 6) respectively for 5 seconds. After the washing, 150 µL of a buffer for fluorescence measurement was added to the wells respectively, followed by fluorescence measurement. As a result, the values of fluorescence intensity were 2.10 in the Pcad34-immobilized wells and 2.06 in the Pcad50-immobilized wells. Even the immobilization of Pcad5Q, which further comprised another CadC-binding sequence, did not cause any detectable change in the amount of CadC-GFP bound. <br><br>
[0084]<br>
Next, the influence of a salt concentration on complex formation was studied. Sodium chloride was added during complex formation reaction to adjust the salt concentration. Pcad50-biotin was adjusted to a concentration of 20 pmol/100 µL and immobilized on wells of a microplate. In a lead detection test, to examine the influence of a salt concentration on Pcad-CadC-GFP complex formation, a mixed solution of a sample and crude protein extracts was first prepared according to the composition shown in Table 10. In this case, a lead(II) acetate solution:4 M sodium chloride ratio is 99:1 (v/v) under sodium chloride addition conditions. Then, a series of operations were performed in the same way as in the preceding detection tests, followed by fluorescence intensity measurement. <br><br>
[0085]<br><br>
[Table 10]<br><br>
[0086]<br>
Under both conditions without the addition of sodium chloride and with the addition of sodium chloride, fluorescence intensity was observed to be decreased with increases in Pb(II) concentration (Figure 22) . However, under conditions with the addition of sodium chloride, fluorescence intensity was drastically increased at a Pb(II) concentration of 0 µg/L. Furthermore, the difference in fluorescence intensity between with and without the addition of lead was also remarkably increased, wherein the fluorescence intensity at a Pb(II) concentration of 100 µg/L was decreased to approximately 30% of that at 0 µg/L. This result demonstrated that the salt concentration largely influences the binding between CadC-GFP and Pcad. It was therefore determined that the lead/cadmium detection test is effectively conducted by adjusting the salt concentration to 40 mM. <br><br>
[0087] <br><br>
(7) Standard curve for lead/cadmium detection<br><br>
A lead/cadmium detection test was conducted under the incubation conditions obtained in the preceding tests. Pcad50-biotin was adjusted to 20 pmol/100|µL and immobilized on wells of a microplate. 376 µL of a Pb(II) or Cd(II) solution with known concentrations of 0 to 100 [Ag/L was mixed as samples with 4 . 0 µL of 4 M NaC1. To make the pHs of the samples constant, 20.0 µL of 1 M tris(hydroxymethyl)aminomethane-HC1 (pH 7.4) was further added thereto as a buffer to adjust the whole amount of the sample to 400 µL. 92 µL of each sample after this pretreatment, 7.5 JAL of CadC-GFP-containing crude protein extracts, and 0.5 [AL of 10 mg/mL salmon sperm DNA were mixed. A series of operations were performed in the same way as in the preceding detection tests, followed by fluorescence intensity measurement. <br><br>
[0088]<br>
As a result, fluorescence intensity was observed to be decreased in correlation to increases in the concentration of Pb(II) or Cd(II) (Figure 23). Significant difference in fluorescence intensity was observed at a Pb(II) concentration of 10 µg/L or a Cd(II) concentration of 5 µg/L, compared with 0 µg/L. As a result of conducting a further detailed test on the lower limit of Cd(II) detection, it was determined to be 1xg/L.  On the other hand, it was revealed that no more reduction in fluorescence intensity is seen at 75 µg/L or higher in both the heavy metals. <br><br>
[0089]<br>
The drinking water quality standards recommended by the World Health Organization were 10 µg/L for Pb(II) and 3 µg/L for Cd(II) . Thus, the developed lead/cadmium sensor was shown to be capable of detecting the standard value of lead and a value of cadmium equal to or lower than the standard. Moreover, though data is not shown, it was revealed that the sensor exhibits little cross-response to 1, 10, or 100 µM Ca(II), Mg(II), Fe(II), Mn(II), Fe(III), and As(III).<br><br>
[0090]<br><br>
[Comparative Example: FRET method]<br>
To detect change in the higher order structure of a complex molecule via the interaction between DNA and protein, an attempt was made to develop a system utilizing fluorescence resonance energy transfer (FRET). FRET is a phenomenon in which excitation energy is transferred from an energy donor (also simply referred to as a donor) to an energy acceptor (also simply referred to as an acceptor). When the acceptor is present near the donor in an excited state, its excitation energy excites the acceptor before the donor emits light. As a result, the light emission of the acceptor can be detected. The nearer the distance there between is, the more easily FRET occurs. Therefore, FRET is used as means for detecting the interaction between molecules or the structural change of a molecule. <br><br>
[0091]<br><br>
(1) Expression and extraction of recombinant protein and confirmation of expression<br>
E. coli transformants for ZntR protein expression were cultured overnight at 37°C in 5 mL of a medium supplemented with Amp and Cm, then inoculated to 250 mL of a medium supplemented with Amp and Cm, and shake-cultured at 37°C. When OD6oo reached 0.6-0.8, isopropyl-|3-D-thiogalactopyranoside (IPTG) was aseptically added thereto at a final concentration of 0.5 mM, followed by culture at 37°C for 3 hours. Bacterial cells were collected by the centrifugation of this culture solution at 8000 rpm for 5 minutes.  After removal of the supernatant, 3 freeze-thaw (freezing at -80°C for 30 minutes and thawing at room temperature for 30 minutes) cycles were performed, and the cells were resuspended in an appropriate amount of Tris buffer A (50 mM Tris-C1, pH 8.0, 2 mM EDTA, 5 mM dithiothreitol) and incubated at room temperature for 10 minutes. Crude ZntR protein extracts were obtained by centrifugation at 12000 rpm at 4°C for 10 minutes and then subjected to the separation by sodium dodecyl sulfate-polyacrylamide gel electrophoresis (SDS-PAGE) to confirm ZntR protein expression. <br><br>
[0092]<br>
E.  coli  transformants for ArsR protein expression were cultured overnight at 37°C in 5 mL of a medium supplemented with Amp and Cm and then inoculated to 250 mL of an LB medium supplemented with Amp. This was shake-cultured at 37°C until OD60o reached 0.8. Then, IPTG was aseptically added thereto at a final concentration of 1 mM, followed by culture at 37°C for 3 hours. Bacterial cells were collected by the centrifugation of this culture solution at 8000 rpm for 15 minutes. After removal of the supernatant, the cells were resuspended in 30 mL of Tris buffer A and sonicated to obtain crude ArsR protein extracts. The crude ArsR protein extracts were subjected to the separation by SDS-PAGE to confirm ArsR protein expression. DNase I treatment was performed for separation between protein and DNA. The DNase I treatment was performed by utilizing RNase-free DNase I (TAKARA BIO INC.) and performing a method according to the protocol of the product. <br><br>
[0093]<br>
The expression of an ArsR-GFP fusion protein was performed in the same way as in the ArsR protein expression except that only Amp was added to a medium. <br><br>
[0094]<br>
The compositions of the SDS-PAGE gel, the electrophoresis buffer, and the sample buffer are shown in Tables 11, 12, and 13, respectively. The protein solution and<br>
the sample buffer (both 20 µL) were mixed, and this 40 µL of the solution was boiled for 5 minutes in boiling water and added to the gel for electrophoresis. Prestained Protein Marker, Broad Range (New England Biolabs Inc. ) was used as a molecular weight marker. After the electrophoresis, the gel was dipped in a staining solution containing 0.2% (w/v) Coomassie Brilliant Blue R-250 dissolved in 40% (v/v) methanol-10% (v/v) aqueous acetic acid solution, and stained by gradual shaking for 30 minutes . For destaining, the gel was dipped in 20% (v/v) methahol-5% (v/v) aqueous acetic acid solution (destaining solution) and gradually shaken. Destaining was repeated with the destaining solution newly replaced until the gel was destained to clearly visualize protein bands.<br><br>
[0095]<br><br>
[Table 11]<br><br>
[0096] <br><br>
[Table 12]<br><br>
[0097] <br><br>
[Table 13]<br><br>
[0098]<br><br>
(2) Purification of ZntR and ArsR-GFP fusion protein<br><br>
All purification steps were performed in a low-temperature room (6 to 8°C) . Crude ZntR or ArsR-GFP fusion protein extracts were subjected to ammonium sulfate precipitation and then gel filtration for desalting. Ammonium sulfate was finely crushed with a pestle and a mortar. Then, the powder was added to the crude extracts in the order of 0%-»10%-»20%-&gt;30%-»40%-^45% with stirring with a stirrer and completely dissolved in each step. After the concentration reached 45%, the solution was stirred overnight with a stirrer. Then, after centrifugation at 12000 rpm at 4°C for 30 minutes, the supernatant was discarded, and the precipitate was completely dissolved by the addition of Tris buffer A in 1/10 of the amount of the crude extracts. Gel filtration was performed by utilizing PD-10 Desalting column (GE Healthcare) and performing a method according to the protocol included therein. In this gel filtration, Tris buffer A was used as a buffer for column equilibration and an eluting buffer. <br><br>
[0099]<br>
Then, affinity chromatography was performed in ZntR protein purification, and affinity chromatography and ion-exchange chromatography were performed in ArsR-GFP fusion protein purification. Hi Prep 16/10 Heparin FF (GE Healthcare) and Hi Trap SP HP (GE Healthcare) were used in the affinity chromatography and the ion-exchange chromatography, respectively. The methods were performed according to each protocol. Their respective purification conditions are shown in Tables 14 and 15. <br><br>
[0100] <br><br>
[Table 14]<br><br><br>
[0101] <br><br>
[Table 15]<br><br><br>
[0102]<br>
The ZntR protein thus purified was quantified by the Bradford method (after Coomassie Brilliant Blue staining, measurement of absorbance at 595 nm (A595)). In the absorbance measurement. Smart Spec™ Plus Spectrophotometer (manufactured by Bio-Rad Laboratories, Inc. ) was used. The ArsR-GFP fusion protein thus purified was quantified by A2eo measurement. At the same time, GFP fluorescence measurement (518 nm) was also performed. In the fluorescence measurement, model F-2500 fluorescence spectrophotometer (manufactured by Hitachi High-Technologies Corp.) was used. <br><br>
[0103]<br><br>
(3) Gel shift assay (Electrophoresis Mobility Shift Assay: EMSA)<br><br>
Fluorescently labeled oligonucleotide strands used in EMSA are shown in Table 16 (the sequences of zntA-Pr-S and zntA-Pr-A are shown in SEQ ID NOs: 7 and 8, respectively) .The same nucleotide sequences as the fluorescently labeled oligonucleotides were prepared as oligonucleotides without fluorescent labeling. Moreover, the composition of binding reaction between the promoter DNA and the ZntR protein is shown in Table 17. To prepare double-stranded DNA, two complementary single-stranded DNAs (100 µM) were mixed in equal amounts in a PCR tube and heat-treated at 95°C for 1 minute. After the heat treatment, the mixture was left at room temperature for 1 hour or longer to promote double strand (50 (J.M) formation. To promote the binding between the promoter DNA and the ZntR protein, a reaction solution was prepared and then incubated at 37°C for 30 minutes. The reaction solution is adjusted to 40 µL by the addition of a sample buffer for native-PAGE shown in Table 18. Then, native-PAGE was performed by utilizing an acrylamide gel prepared according to the composition shown in Table 19 and a TBE buffer shown in Table 20. <br><br>
[0104] <br><br>
[Table 16]<br><br>
[0105]<br><br>
[Table   17]<br><br>
[0106] <br><br>
[Table 18]<br><br>
[0107] <br><br>
[Table 19]<br><br>
[0108]<br><br>
[Table 20]<br><br>
[0109]<br>
The detection of fluorescently labeled DNA in EMSA was performed by photographing, with a digital camera, the gel irradiated with an LED light source VISIRAYS (ATTO Corp., AE-6935B (blue light source), AE-6935GL (green light source)) under dark conditions. <br><br>
[0110] <br><br>
(4) Lead compound detection test<br><br>
Promoter DNA used was double-stranded DNA labeled at both the ends with FITC and TAMRA in the same way as that used in EMSA. The promoter DNA was prepared by mixing in advance single-stranded DNAs labeled with FITC and TAMRA and forming double-stranded DNA by the same treatment as in the method used in EMSA. Lead(II) acetate (Pb(CH3COOH)2; hereinafter, referred to as Pb) and zinc sulfate (ZnS04; hereinafter, referred to as Zn) were prepared as heavy metal at each concentration of 10 nM. Sodium chloride (NaC1; hereinafter, referred to as Na) and a calcium ion (CaCl2-2H20; hereinafter, referred to as Ca) were prepared as controls at each concentration of 10 nM. Samples were studied under 5 conditions: 1 nM Pb; 1 nM Zn; 1 nM Na; 1 nM Ca; and EDTA-free Tris buffer A (50 mM Tris-Cl, pH 8.0, 5 mM dithiothreitol). A reaction solution was prepared according to the composition shown in Table 21 and incubated at 37°C for 30 minutes under dark conditions. Then, the reaction solution was mixed by the addition of 1 mL of EDTA-free Tris buffer A, followed by fluorescence measurement under the conditions shown in Table 22.<br><br>
[0111] <br><br>
[Table 21]<br><br>
[0112] <br><br>
[Table 22]<br><br>
[0113]<br><br>
(5) Arsenic compound detection test<br><br>
pAcGFP1 plasmid-derived GFP fused with an ArsR protein is a fluorescent protein that emits green fluorescence at an excitation wavelength of 475 nm and a fluorescence wavelength of 505 nm. Three different fluorescently labeled DNA probes were separately mixed with the ArsR-GFP fusion protein and sodium arsenite (NaAs02; hereinafter, referred to as As) to prepare reaction solutions. The 3 fluorescently labeled DNA probes used are shown in Table 23. As a result of studying excitation light at 350 nm, 360 nm, 370 nm, 380 nm, 390 nm, 400 nm, 410 nm, and 420 nm, the wavelengths of 350 nm and 360 nm were used at which the peaks of TAMRA and GFP fluorescence intensities were changed between with and without the addition of As. Fluorescence measurement conditions except for an excitation wavelength, a wavelength of fluorescence initiation, and a wavelength of fluorescence completion followed the conditions shown in Table 22. All incubation procedures were performed under shading conditions. A 350-bp DNA fragment centered on an ArsR protein-binding promoter region on genomic DNA was amplified by PCR using a combination of the primers "ParsR-S3-5-TAMRA" and "Ec arsR prm ext prm" shown in Table 23 (hereinafter, referred to as ParsR-350 probe DNA) (the sequences of ParsR-S3 and Ec arsR prm ext prm are shown in SEQ ID NOs: 9 and 10, respectively, and the sequences of R773-50-S and R773-50-A are shown in SEQ ID NOs: 11 and 12, respectively). <br><br>
[0114] <br><br>
[Table 23]<br><br>
[0115]<br>
A 350-bp DNA fragment centered on an ArsR protein-binding promoter region on genomic DNA was amplified by PCR using a combination of the primers ParsR-S3-5-TAMRA and Ec arsR prm ext prm shown in Table 23 (hereinafter, referred to as ParsR-350 probe DNA). Reaction composition in a detection test using the ParsR-350 probe DNA is shown in Table 24. The detection was performed according to the procedures shown in Table 24, and then, the fluorescence spectrum of 1 mL of the reaction solution was measured. The fluorescence detection was performed twice at 400-650 nm. <br><br>
[0116] <br><br>
[Table 24]<br><br>
[0117]<br>
The double-stranded DNA formed with ParsR-50-S-5-TAMRA and ParsR-50-A shown in Table 23 consists of a 50-bp nucleotide sequence of ArsR protein-binding promoter DNA on genomic DNA (hereinafter, referred to as ParsR-50 probe DNA).<br><br>
Moreover, the double-stranded DNA formed with R773-50-S-5-TAMRA and R773-50-A shown in Table 23 consists of a 50-bp nucleotide sequence of ArsR protein-binding promoter DNA on R-773 plasmid DNA (hereinafter, referred to as R773-50 probe DNA). These two fluorescently labeled DNA probes were separately incubated at 80°C and then at room temperature to form double-stranded DNA. <br><br>
[0118]<br><br>
Reaction composition in a detection test using the ParsR-50 probe DNA or the R773-50 probe DNA and detection procedures are shown in Table 25. The fluorescence detection was performed three times at 480 to 650 nm. <br><br>
[0119] <br><br>
[Table 25]<br><br>
[0120]<br><br>
(6) Results of generating E.   coli   strain for ZntR protein expression<br><br>
pET3a vector DNA was used in the generating of an E. coli strain for ZntR protein expression. Since the previously reported recombinant ZntR protein has a His tag at the N terminus, it was not possible to deny the possibility of its binding to heavy metal via the His tag. Therefore, an E. coli strain expressing a His tag-free ZntR protein was generated again. First, a JM109 strain carrying pET16zntR shown in Figure 27 was inoculated to 100 mL of an LB medium supplemented with Amp, and shake-cultured overnight at 37°C. On the next day, bacterial cells were collected, followed by plasmid extraction. The obtained plasmid solution was reacted overnight with restriction enzymes Ndel1 and BamHI. The restriction enzyme reaction solution was subjected to the separation by agarose gel electrophoresis, and the band of the zntR gene fragment of interest was excised and purified. This was used as insert DNA and ligated with pET3a by overnight incubation at 16°C. An E. coli JM109 strain was transformed with the ligation reaction solution. After the transformation, 5 selected colonies (designated as a, b, c, d, and e) were separately inoculated to 5 mL of an LB medium supplemented with Amp, and shake-cultured overnight at 3 7°C. Next, after plasmid extraction using each culture solution, restriction enzyme reaction was performed at 37°C for 1 hour with Nde1 and BamHI. As a result of confirming the presence or absence of the insert by agarose gel electrophoresis, the insert was observed in the plasmids extracted from the colonies b and c. As a result of sequencing, the sample c was confirmed to be the pET3zntR of interest. Therefore, BL21 (DE3) pLysS was transformed with this plasmid. Through these processes, the E. coli  strain for ZntR protein expression was generated (Figure 27). <br><br>
[0121] <br><br>
(7) Results of ZntR protein expression and purification<br><br>
E.  coli  for ZntR protein expression was shake-cultured overnight at 37°C in 5 mL of an LB medium supplemented with Amp and Cm, and used as an inoculum. Two LB media (5 mL each) supplemented with Amp and Cm were newly prepared, and the inoculum was separately inoculated to the media, one of which was supplemented with IPTG during the culture. From the obtained culture, bacterial cells were collected by centrifugation for 5 minutes under conditions involving 10,000 rpm and 4°C. The obtained bacterial cells were suspended in Tris buffer A and disrupted with a sonicator. Then, after centrifugation for 10 minutes under conditions involving 12,000 rpm and 4°C, the supernatant was collected. This was used as crude ZntR protein extracts to confirm ZntR protein expression by SDS-PAGE. As a result, a band believed to be a ZntR protein was confirmed only in the sample with the addition of IPTG (position indicated in the arrow in Figure 28) .  To purify the ZntR protein, a medium was adjusted to 250 mL for culture. Bacterial cells collected in the same way as in the culture scale of 5 mL were lysed by the addition of Tris buffer A and sonicated. Then, the homogenate was transferred to a centrifuge tube and centrifuged for 10 minutes under conditions involving 12,000 rpm and 4CC, and the supernatant was collected by decantation. This was used as crude ZntR protein extracts. <br><br>
[0122]<br>
Ammonium sulfate precipitation was performed as an initial purification process. Then, gel filtration was continued for desalting, followed by affinity chromatography using a heparin column. In this chromatography, approximately 5 mL each of 40 fractions was collected. As a result of measuring the absorbance of each fraction at 280 nm, some peaks of absorbance were obtained. However, as a result of performing SDS-PAGE on the fractions constituting these peaks, no band was confirmed at the position indicated in the arrow in Figure 28. The ZntR protein is free from aromatic amino acids and was therefore presumed to exhibit no light absorption at 280 nm. Therefore, the 40 fractions were subjected to protein staining by the Bradford method, and absorbance was measured at 595 nm (Figure 29). A peak observed until the fraction 5 was likely to be a non-adsorbed fraction. Thus, the fractions 9 to 15 constituting peaks observed after the fraction 5 were subjected to the separation by SDS-PAGE. <br><br>
[0123]<br>
In the fractions 10-13, a band believed to be a ZntR protein was confirmed at the position indicated in the arrow (Figure 30, position indicated in the arrow). Thus, the fractions 10-13 were concentrated together by ultrafiltration. and protein was quantified by the Bradford method, followed by gel filtration for desalting. The ZntR protein was further confirmed by SDS-PAGE (Figure 31) . As a result, a band believed to be a ZntR protein was confirmed at the position indicated in the arrow.  Thus, this partially purified ZntR protein solution was used in subsequent experiments.<br><br>
[0124]<br><br>
(8) Results of EMSA using partially purified ZntR protein solution<br><br>
As a result of quantifying the partially purified ZntR protein, its amount was 0.53 mg/mL. EMSA was conducted by utilizing the ZntR protein and probe DNA consisting of DNA labeled with each fluorescent dye. A list of samples loaded to a gel for electrophoresis is shown in Table 26. When FITC-labeled probe DNA was used, samples were irradiated with blue excitation light for FITC labeling and photographed through SCF515 filter (manufactured by ATTO Corp. ) . For TAMRA labeling, samples were irradiated with green excitation light and photographed through R-60 filter (manufactured by ATTO Corp. ) (Figure 32) . A band observed in the lane 1 is probably derived from single-stranded DNA, while bands observed in the lanes 2 and 6 are probably derived from double-stranded DNA. An upwardly shifted band believed to be derived from a ZntR protein-probe DNA complex was further observed in samples supplemented with the partially purified ZntR protein solution, indicating quantitative increases in the shifted band with increases in the volume of the added protein. Moreover, the FITC- or TAMRA-labeled probe DNA was detectable as green or red fluorescence. These results demonstrated that the prepared recombinant ZntR protein specifically binds to the probe DNA used to form a complex.<br><br>
[0125]<br><br>
[Table 26]<br><br>
[0126]<br>
A list of samples loaded to a gel for electrophoresis in EMSA using probe DNA labeled with FITC and TAMRA is shown in Table 27, and the experimental results are shown in Figure 33. The gel was irradiated with blue excitation light for FITC excitation and photographed via an orange film. An upwardly shifted band believed to be derived from a ZntR protein-probe DNA complex was observed in samples supplemented with the partially purified ZntR protein solution, as with the results of EMSA using the probe DNA labeled with one fluorescence. Green fluorescence was observed in the band of the FITC-labeled single-stranded DNA in the lane 1, whereas red fluorescence was hardly confirmed in the TAMRA-labeled single-stranded DNA in the lane 2. However, pink fluorescence, a color between green FITC fluorescence and red TAMRA fluorescence, was observed in a band derived from the probe DNA double-labeled with FITC and TAMRA. This result demonstrated that TAMRA is excited by green fluorescence emitted by FITC to emit red fluorescence, i.e. , FRET occurs from FITC to TAMRA on the probe DNA. <br><br>
[0127] <br><br>
[Table 27]<br><br>
[0128]<br><br>
(9) Results of lead compound detection using partially purified ZntR protein solution and fluorescently labeled probe DNA<br><br>
As a result of measuring the fluorescence spectrum of a reaction solution containing a mixture of probe DNA double-labeled with FITC and TAMRA and a partially purified ZntR protein solution, the addition of Pb or Zn merely shifted all fluorescence spectra upwardly, compared with a control supplemented with a buffer. The same results as in Pb and Zn were obtained for metal ions Na and Ca (Figure 34). <br><br>
[0129]<br>
The expected change in fluorescence spectrum was the increase of an FITC peak at 518 nm and the decrease of a TAMRA peak at 580 nm in the presence of Pb or Zn. Moreover, it was expected that no change was seen in the ratio between the FITC peak at 518 nm and the TAMRA peak at 580 nm for Na, Ca, and a control. However, the ratio between the FITC fluorescence peak (518 nm) and the TAMRA fluorescence peak (580 nm) was not changed between the buffer-supplemented control and the Pb-or Zn-supplemented sample. <br><br>
[0130]<br>
The reason why the ratio between the FITC fluorescence peak and the TAMRA fluorescence peak was not changed in this test is presumably that the distance between the fluorescent materials was less changed along with change in the higher order structure of the ZntR-promoter DNA complex at the occurrence of FRET. Moreover, the amount of the fluorescently labeled probe DNA or the ZntR protein added to samples might be inadequate for occurrence of FRET in the test. <br><br>
[0131]<br><br>
(10) Results of generating E.   coli   strain for ArsR protein expression<br><br>
To construct an As detection system via FRET, an attempt was made to prepare an ArsR protein ( sensor protein for arsenic) , as a recombinant protein.  If the recombinant ArsR protein can be prepared, it can be labeled directly with a fluorescent material based on the principle shown in Figure 26 and without assuming the form of an ArsR-GFP fusion protein. The primers E. coli ArsR-S and E. coli ArsR-A for arsR gene fragment amplification shown in Table 28 were designed based on the DNA sequencing data of an E. coli K12 strain (the sequences of E. coli ArsR-S and E. coli ArsR-A are shown in SEQ ID NOs: 13 and 14, respectively). The genomic DNA of the E. coli K12 strain was used as a template in polymerase chain reaction (PCR) using the primers E. coli ArsR-S and E. coli ArsR-A and Pfx50™ DNA Polymerase to amplify an arsR gene.  The amplification was TM confirmed by electrophoresis, and then, TAKARA LA Taq with GC Buffer was added to the PCR reaction solution to perform dATP addition reaction to the 3' end. This reaction solution was purified and ligated with TA cloning pGEM-T plasmid DNA, with which an E. coli JM109 strain was then transformed (Figure 35). pGEMarsR was extracted from the transformants and digested with Xho1 and Nde1, and the inserted fragment was confirmed by electrophoresis. pGEMarsR and pET16b were separately treated with Xhol and Ndel. For pGEMarsR, the band of the arsR gene fragment (approximately 360 bp) of interest was excised after electrophoresis and purified. Moreover, the reaction solution of pET16b was directly purified and ligated with the arsR gene fragment. An E. coli JM109 strain was transformed with the ligation reaction solution. The insertion of the arsR gene to pET16b was confirmed by extracting a plasmid from the transformants and treating the plasmid with Xho1 and Nde1, followed by confirmation of the band of the arsR gene fragment by electrophoresis. The arsR gene on pET16arsR was sequenced and compared with the DNA sequencing data of the E.   coli   K12 strain in the Gen Bank database to confirm no mutation in the cloned arsR gene. However, an ArsR protein expressed from pET16arsR has a His tag, via which the protein might bind to heavy metal. Therefore, the arsR gene fragment was newly ligated with His tag-free pET3a vector. pET16arsR and pET3a vectors were separately treated with BamHI and Ndel. For pET16arsR, the band of the arsR gene fragment of interest was excised after electrophoresis and purified. The arsR gene fragment was ligated with the pET3a vector to prepare pET3arsR. The gene fragment was sequenced in the same way as above and compared with the DNA sequencing data of the E. coli K12 strain in Gen Bank to confirm no mutation in the arsR gene on pET3arsR. Next, a plasmid solution of pET3arsR was used in the transformation of an£. coli BL21 (DE3) pLysS strain for protein expression to generate an E. coli strain for ArsR protein expression (Figure 35). <br><br>
[0132]<br><br>
[Table 28]<br><br>
[0133]<br><br>
(11) Expression of recombinant ArsR protein and confirmation An E. coli strain for ArsR protein expression was cultured and then sonicated .to obtain crude ArsR protein extracts. The crude ArsR protein extracts were subjected to the separation by SDS-PAGE to confirm ArsR protein expression. However, no band could be confirmed at the predicted position of the ArsR protein band (approximately 13 kDa), regardless of the presence or absence of IPTG addition. Bands were accumulated in the upper region of the gel for the crude protein extracts at the time of IPTG addition, indicating the possibility of the binding between the ArsR protein and DNA. Thus, the crude protein extracts were treated with DNase I for the purpose of removing protein-bound DNA. Then, the band of the ArsR protein was confirmed again by SDS-PAGE. However, no difference was seen between the band positions predicted from the presence or absence of IPTG addition. Though this reason was not identified, the recombinant ArsR protein was unsuccessfully prepared. Thus, it was prepared in the form of a fusion protein of ArsR and GFP and thus fluorescently labeled. <br><br>
[0134]<br><br>
(12) Results of generating E.  coli  strain for ArsR-GFP fusion protein expression<br>
To clone an arsR gene fragment, the primers Ec-arsR-pAcGFP-S (SEQ ID NO: 15) and Ec-arsR-pAcGFP-A (SEQ ID NO: 16) for arsR gene amplification shown in Table 29 were designed based on the DNA sequencing data of an E. coli K12 strain. Moreover, a restriction enzyme site required for cloning was added to each primer. First, the genomic DNA of the E. coli K12 strain was used as a template in PCR using the primers Ec-arsR-pAcGFP-S and Ec-arsR-pAcGFP-A to amplify the arsR gene fragment (Figure 36). The amplification was confirmed by electrophoresis, and then, the arsR gene fragment and a DNA vector pAcGFP1 were separately treated with Hindi 11 and Pstl. For pAcGFP1, the band of the vector site (approximately 3.3 kb) of interest was excised after electrophoresis  and purified.   Moreover,  the reaction solution of the arsR gene fragment was directly purified. Both the DNA fragments were ligated, and an E. coli JM109 strain was transformed with the obtained pAcGFParsR. The insertion of the arsR gene was confirmed by extracting pAcGFParsR from the transformants and treating the plasmid with Hindlll and PstI, followed by confirmation of the band of the arsR gene fragment (approximately 360 bp) by agarose gel electrophoresis. The gene fragment on pAcGFParsR was sequenced and compared with the arsR DNA sequencing data of the E. coli K12 strain in Gen Bank to confirm no mutation in the arsR gene on pAcGFParsR.<br><br>
[0135] <br><br>
[Table 29]<br><br>
 [0136]<br>
(13) Expression of ArsR-GFP fusion protein and confirmation An E. coli strain for ArsR-GFP fusion protein expression was cultured, and bacterial cells were disrupted to obtain crude ArsR-GFP fusion protein extracts. The crude ArsR-GFP fusion protein extracts were subjected to the separation by SDS-PAGE to confirm ArsR-GFP fusion protein expression. As a result, a band was confirmed at the predicted position of the ArsR-GFP fusion protein (approximately 47 kDa) (position indicated in the arrow in Figure 37) . However, due to the low expression level, the protein of interest was partially purified by affinity chromatography to enhance its ratio to contaminating proteins.<br>
80<br><br>
[0137]<br><br>
(14) Results of purifying ArsR-GFP fusion protein<br><br>
Crude ArsR-GFP fusion protein extracts were subjected to ammonium sulfate precipitation and redissolved in a buffer, followed by gel filtration for desalting. Then, the A2flo and GFP fluorescence (520 nm) of 40 fractions collected by affinity chromatography purification were measured. As a result, peaks of A28o and values of fluorescence was observed with the fractions 19 and 20 centered (Figure 38), demonstrating that the ArsR-GFP fusion protein is mainly present in these fractions. As a result of confirming protein by SDS-PAGE for the fractions 19 and 20, a band believed to be an ArsR-GFP fusion protein could be confirmed (Figure 39, position indicated in the arrow). <br><br>
[0138]<br>
However, contaminating proteins other than the protein of interest were still observed. Therefore, ion exchange chromatography using these fractions was performed to further increase purity. Then, the A2so and fluorescence intensity (420 nm) of the collected 40 fractions were measured. As a result, peaks of A28o were observed with the fractions 19 to 21 centered, and the fractions 20 and 21 mainly exhibited a high value of fluorescence (Figure 40). This demonstrated that the ArsR-GFP fusion protein is mainly present in the fractions 20 and 21. As a result of separating protein by SDS-PAGE for the fractions 20 and 21, a band believed to be an ArsR-GFP fusion protein could be confirmed (Figure 41, position indicated in the arrow) . The contaminating proteins other than the ArsR-GFP fusion protein could be removed drastically  by  performing  ion-exchange  chromatography purification.<br><br>
[0139]<br><br>
(15) As detection test using ArsR-GFP fusion protein and fluorescently labeled probe DNA<br><br>
A study was conducted on whether or not the presence of arsenic compounds (As) changed a fluorescence spectrum via FRET based on the principle shown in Figure 26 . A partially purified ArsR-GFP fusion protein solution and TAMRA-labeled ParsR-350 were mixed. Results of measuring a fluorescence spectrum at 350-nm and 360-nm excitation lights in the presence or absence of As are shown in Figure 42. For both the excitation lights, fluorescence intensity was observed to be slightly lower with the addition of As than without the addition of As at a TAMRA fluorescence peak around 580 nm. Moreover, fluorescent intensity was observed to be slightly increased with the addition of As than without the addition of As at a fluorescent peak around 510 nm. This tendency is consistent with the change in fluorescence spectrum predicted from the detection principle shown in Figure 26. Specifically, this was probably because: without the addition of As, the complex formation between protein and DNA decreased the distance of TAMRA from GFP whose fluorescence was thereby easily absorbed as excitation light of TAMRA, resulting in the increase of TAMRA fluorescence, whereas with the addition of As, the dissociation between protein and DNA increased the distance of TAMRA from GFP whose fluorescence was thereby absorbed at a decreased rate by TAMRA, resulting in the increase of GFP fluorescence. However, no significant change in the ratio of fluorescence intensity between TAMRA and GFP was observed by the addition of As.  Therefore, an As detection test was conducted by utilizing ParsR-50 or R773-50 probe DNA whose DNA strand length was reduced from 350 bp to 50 bp. As a result, for both the excitation lights and the probe DNAs, the same change in spectrum as in use of ParsR-350 probe DNA was observed by the addition of As (Figures 43 and 44). Moreover, TAMRA fluorescence around 580 nm was observed to be lower by the addition of As than that obtained using ParsR-350 probe DNA. However, for both the probe DNAs, no increasing tendency attributed to the addition of As was observed in GFP fluorescence intensity. In the comparison between ParsR-50 and R773-50 probe DNAs, decline in TAMRA fluorescence attributed to the addition of As was larger when ParsR-50 probe DNA was used. <br><br>
[0140]<br>
The results of an arsenic detection test using 3 different probe DNAs demonstrated that TAMRA fluorescence intensity is weakly changed by the addition of As by setting a probe DNA strand length to 50 bp and selecting the nucleotide sequence of an arsenic-responsive promoter on the genome. The expected change in fluorescence spectrum via FRET could not be brought about even when the probe DNA strand length or the nucleotide sequence was changed. <br><br>
[0141] <br><br>
(16) Conclusion<br><br>
The results of Pb or As detection test using the prepared two recombinant proteins ZntR and ArsR-GFP are summarized below.<br><br>
In the test using probe DNA comprising promoter DNA double-labeled with FITC and TAMRA, and a ZntR protein, no change in the ratio between FITC and TAMRA fluorescence intensities was observed by the addition of Pb. From the results of analysis by gel shift assay, the prepared ZntR protein and the probe DNA were confirmed to form a ZntR-promoter DNA complex. Thus, change in the higher order structure of the formed complex caused by the addition of Pb might be too small to cause change in energy transfer in FRET. It was difficult to externally display the change in the higher order structure of the ZntR-promoter DNA complex via FRET. <br><br>
[0142]<br>
On the other hand, it was difficult to allow an E. coli strain to express a wild-type ArsR protein for As detection. Therefore, the ArsR protein was expressed as an ArsR-GFP fusion protein and could thereby be modified fluorescently with GFP. Whether or not the dissociation of the DNA molecule from the protein-DNA complex caused by the addition of As can be externally displayed as change in fluorescence spectrum by FRET was confirmed by the fluorescent modification of promoter DNA with TAMRA. As a result, change was observed, albeit slightly, in the ratio between GFP and TAMRA fluorescence intensities by the addition of As. Moreover, the results of the test using 3 different probe DNAs demonstrated that the rate of change in fluorescence intensity caused by the addition of As is changed, albeit slightly, depending on difference in probe DNA strand length or nucleotide sequence. The probe DNA used in the test in which the fluorescence intensity of TAMRA was most largely changed was ParsR-50, which was TAMRA-labeled 50-bp arsenic-responsive promoter DNA on the genomic DNA of the strain. <br><br>
[0143]<br>
The reason why fluorescence intensity was weakly changed in the As detection test is that both molecules of protein and DNA interacting with each other were in a solution state . Since the molecules freely move in the solution, not a small amount of background FRET was likely to occur, regardless of the bound or dissociated state of the molecules. Thus, a method for emphasizing the change in fluorescence intensity comprises immobilizing probe DNA to substrate surface such that the substrate surface is modified with the probe DNA. If the immobilization of probe DNA significantly decreases FRET attributed to the addition of arsenic, this system can also be applied to, for example, an assay system using a microplate and can also be expected to be developed into a high-throughput As detection system in the future. However, the present approach without probe DNA immobilization did not cause change in FRET, i.e., change in fluorescence spectrum, at a practical level, after all.<br><br>
Industrial Applicability<br><br>
[0144]<br>
The present invention enables convenient and rapid detection and assay of an analyte in a test sample by utilizing a microbial-derived sensor protein.<br><br>
CLAIMS<br><br>
1.	A method for detecting or quantifying an analyte in a test sample in an aqueous system, comprising the following steps (A) and (B):<br><br>
(A)	reacting, in the presence of the test sample, a sensor protein specifically binding to the analyte, with a nucleic acid comprising a sequence that is specifically recognized by the sensor protein, wherein the nucleic acid is immobilized on a support; and<br><br>
(B)	detecting or assaying the sensor protein bound to the immobilized nucleic acid.<br><br>
2.	The method according to claim 1, wherein the sensor protein is labeled with a detectable marker.<br><br>
3.	The method according to claim 1, wherein the sensor protein is a fusion protein with a detectable marker protein.<br><br>
4.	A method for detecting or quantifying an analyte in a test sample in an aqueous system, comprising the following steps (A) and (B):<br><br>
(A)	reacting, in the presence of the test sample, a sensor protein specifically binding to the analyte, with a nucleic acid comprising a sequence that is specifically recognized by the sensor protein, wherein the sensor protein is immobilized on a support; and<br><br>
(B)	detecting or assaying the nucleic acid bound to the immobilized sensor protein.<br><br>
5.	The method according to claim 4, wherein the nucleic acid is labeled with a detectable marker.<br><br>
6.	The method according to any one of claims 1 to 5, wherein the binding between the sensor protein and the nucleic acid is inhibited by the analyte.<br><br>
7.	The method according to any one of claims 1 to 6, wherein the sensor protein is a protein encoded by a metal-responsive operon.<br><br>
8.	The method according to claim 7, wherein the protein encoded by a metal-responsive operon is an ArsR or CadC protein.<br><br>
9.	The method according to any one of claims 1 to 8, wherein the analyte is a heavy metal compound.<br><br>
10.	The method according to claim 9, wherein the heavy metal compound is selected from an arsenic (As) compound, a cadmium (Cd) compound, and a lead (Pb) compound.<br><br>
11.	The method according to claim 10, wherein a pentavalent arsenic (As(V)) compound is converted in advance to a trivalent arsenic (As(III)) compound by reduction treatment.<br><br>
12.	The method according to any one of claims 9 to 11, wherein the reaction of the sensor protein with the nucleic acid comprising a sequence that is specifically recognized by the sensor protein occurs in a 50 mM or higher phosphate buffer (pH 7.4).<br><br>
13.	The method according to any one of claims 9 to 11, wherein the reaction of the sensor protein with the nucleic acid comprising a sequence that is specifically recognized by the sensor protein occurs in a 40 mM or higher sodium chloride solution.<br><br>
14.	A kit for detecting or quantifying an analyte in a test sample, comprising: a sensor protein specifically binding to the analyte; and a nucleic acid comprising a sequence that is specifically recognized by the sensor protein, wherein the nucleic acid is immobilized on a support.<br><br>
15.	A kit for detecting or quantifying an analyte in a test sample, comprising: a sensor protein specifically binding to the analyte, wherein the sensor protein is immobilized on a support; and a nucleic acid comprising a sequence that is specifically recognized by the sensor protein.<br></td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/viewdoc.aspx?id=SkUmFYTh0h6ZGXNYZY7rtw==&amp;amp;loc=egcICQiyoj82NGgGrC5ChA==" target="_blank" style="word-wrap:break-word;">http://ipindiaonline.gov.in/patentsearch/GrantedSearch/viewdoc.aspx?id=SkUmFYTh0h6ZGXNYZY7rtw==&amp;amp;loc=egcICQiyoj82NGgGrC5ChA==</a></p>
		<br>
		<div class="pull-left">
			<a href="269062-antimony-free-photocurable-resin-composition-and-three-dimensional-article.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="269064-a-set-of-circuit-board-connectors-and-a-method-of-mounting-a-connector-to-a-circuit-board.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>269063</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>818/CHENP/2011</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>40/2015</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>02-Oct-2015</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>29-Sep-2015</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>04-Feb-2011</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>UTSUNOMIYA UNIVERSITY</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td>350, MINEMACHI, UTSUNOMIYA-SHI, TOCHIGI 321-8505</td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>MAEDA, ISAMU</td>
											<td>C/O UTSUNOMIYA UNIVERSITY, 350, MINEMACHI, UTSUNOMIYA-SHI, TOCHIGI 321-8505</td>
										</tr>
										<tr>
											<td>2</td>
											<td>INOUE, KOICHI</td>
											<td>C/O UTSUNOMIYA UNIVERSITY, 350, MINEMACHI, UTSUNOMIYA-SHI, TOCHIGI 321-8505</td>
										</tr>
										<tr>
											<td>3</td>
											<td>KAWAKAMI, YASUNARI</td>
											<td>C/O UTSUNOMIYA UNIVERSITY, 350, MINEMACHI, UTSUNOMIYA-SHI, TOCHIGI 321-8505</td>
										</tr>
										<tr>
											<td>4</td>
											<td>MIYASAKA, HITOSHI</td>
											<td>C/O POWER ENGINEERING R&amp;D CENTER, RESEARCH AND DEVELOPMENT DEPARTMENT, THE KANSAI ELECTRIC POWER CO., INC., 11-20, NAKOJI 3-CHOME, AMAGASAKI-SHI, HYOGO 661-0974</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>C12N15/09</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>PCT/JP09/003168</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td>2009-07-07</td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td>2008-180801</td>
									<td>2008-07-10</td>
								    <td>Japan</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/269063-biosensor-utilizing-dna-as-element by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 08:24:24 GMT -->
</html>
