<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/269205-enzymatic-synthesis-of-sulfated-polysaccharides by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 08:04:10 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 269205:&quot;ENZYMATIC SYNTHESIS OF SULFATED POLYSACCHARIDES&quot;</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">&quot;ENZYMATIC SYNTHESIS OF SULFATED POLYSACCHARIDES&quot;</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>A method of sulfating a polysaccharide is provided. The method can include providing a reaction mixture comprising at least one O-sulfotransferase (OST) enzyme and 3&#x27;-phosphoadenosine 5&#x27;-phosphosulfate (PAPS); incubating a polysaccharide substrate with the reaction mixture, wherein production of the sulfated polysaccharide from the polysaccharide substrate is catalyzed by the OST enzyme with a conversion of the PAPS to adenosine 3&#x27;,5&#x27;-diphosphate (PAP); and providing a reaction condition which modifies PAP to reduce an inhibitory effect of PAP on the polysaccharide sulfation.</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td>DESCRIPTION ENZYMATIC SYNTHESIS OF SULFATED POLYSACCHAR1DES<br>
RELATED APPLICATIONS<br>
This application claims the benefit of U.S. Provisional Patent Application Serial No. 60/680,392, filed May 12, 2005; the disclosure of which is incorporated herein by reference in its entirety.<br>
GOVERNMENT INTEREST<br>
The presently disclosed subject matter was made with U.S. Government support under Grant No. AI50050 awarded by the National Institutes of Health. Thus, the U.S. Government has certain rights in the presently disclosed subject matter.<br>
TECHNICAL FIELD<br>
The presently disclosed subject matter relates generally to methods of sulfating polysaccharides. In particular, the presently disclosed subject matter relates tq methods of sulfating polysaccharides using O-sulfotransferases, which includes a reaction condition that reduces potential inhibitory effects from sulfur donor byproducts.<br>
BACKGROUND<br>
Heparan sulfate (HS) is a ubiquitous component of the cell surface and extracellular matrix. It regulates a wide range of physiologic and pathophysiologic functions, including embryonic development and blood coagulation, and can facilitate viral infection (Esko and Selleck (2002) Annu. Rev. Biochem. 71, 435-471; Liu and Thorp (2002) Med. Res. Rev. 22,1-25). HS exerts its biological effects by interacting with the specific proteins involved in a given process (Capila and Lindhardt (2002) Angew. Chem. Int. Ed. 41, 390-412). HS is a highly charged polysaccharide consisting of 1 -* 4-linked glucosamine and glucuronic/iduronic acid units that contain both N- and Osulfo groups. Unique saccharide sequences within HS determine the specificity of the binding of HS to its target proteins (Linhardt (2003) J. Med. Chem.46,<br>
2551-2564). Heparin, a specialized form of HS, is a commonly used anticoagulant drug. Thus, new methods for the synthesis of heparin and HS attract considerable interest for those developing anticoagulant and other HS-related drugs having improved pharmacological effects.<br>
Chemical synthesis has been the major route to obtain structurally defined heparin and HS oligosaccharides (Petitou and van Boeckel (2004) Angew. Chem, Int. Ed. 43, 3118-3133). One example of a chemically synthesized HS oligosaccharide is a synthetic pentasaccharide having antithrombin-binding properties marketed in the United States under the trade name ARIXTRA® (GlaxoSmithKline, Middlesex, United Kingdom). ARIXTRA® is a specific factor Xa inhibitor that is used clinically to prevent venous thromboembolic incidents during surgery.<br>
Unfortunately, the total synthesis of heparin and HS oligosaccharides, larger than pentasaccharides, is difficult. HS analogues with 14 saccharide units inhibit the activity of thrombin, but these synthetic analogues are simplified hybrid molecules of HS oligosaccharides and highly sulfated glucose units (Petitou et al. (1999) Nature 398,417-422; Dementievetal. (2004) Nat. Struct. Biol. 11,867-863) and are not the naturally occurring structures. Although the pursuit for the chemical synthesis of heparin and HS oligosaccharides (Avci et al. (2003) Curr. Pharm. Des. 9,2323-2335) continues, it has become clear that chemical synthesis alone is currently incapable of generating most larger oligosaccharide structures. Thus, the application of HS biosynthetic enzymes for generating large heparin and HS oligosaccharides with desired biological activities offers a promising alternative approach.<br>
Six classes of enzymes are involved in HS biosynthesis. HS is initially synthesized as a copolymer of D-glucuronic acid and N-acetyiglucosamine (GlcNAc) through the action of D-glucuronyl and /V-acetyl-D-glucosaminyltransferase (Lindahl et al. (1998) J. Biol. Chem. 273, 24979-24982). Next, a series of modifications take place, including A/-deacetylationand /V-sulfation (carried out by /V-deacetylase/W-sulfotransferase) of the glucosamine residue to form A/-sulfoglucosamine (GlcNS), C6 epimerization of glucuronic acid (carried out by epimerase) to form L-iduronic acid (IdoUA), 2-O-sulfation of IdoUA (carried out by 2-Osulfotransferase (2-<br>
OST)), 6-O-sulfation of glucosamine (carried out by 6-O-sulfotransferase (6-OST)), and 3-Osuifation of glucosamine (carried out by 3-Osulfotransferase (3-OST)) (Sasisekharan et al. (2002) Nat. Rev. Cancer 2, 521-528). The reactions catalyzed by 2-OST, 6-OST, and 3-OST are shown in Figure 1A.<br>
Enzymes "in the pathway" for HS biosynthesis have been cloned and expressed, and have been employed in the synthesis of HS polysaccharides. Kuberan and Rosenberg (Balagurunathan et al. (2003) Nat. Biotechnol. 21, 1343-1346; Kuberan et al. (2003) J. Am. Chem. Soc. 125, 12424-12425; Balagurunathan et al. (2003) J. Biol. Chem. 278,52613-52621) utilized these enzymes to synthesize an HS containing antithrombin binding sites with anticoagulant activity. Although this approach demonstrated for the first time the feasibility of enzymatic synthesis of HS, only about 1 ug of product was generated, making extensive structural characterization and biological studies impossible. Recently, Lindahl and colleagues reported an alternative chemoenzymatic approach for the synthesis of anticoagulant heparin from heparosan, the E. collK5 capsular polysaccharide (Lindahl et al. (2005) J. Med. Chem. 48, 349-352). This method utilized the Cg epimerase to convert D-glucuronic acid to IdoUA, followed by the chemical persulfation and finally selective desulfation. Although this approach afforded approximately 5 g of a heparin-like polysaccharide with anticoagulant activity, unnatural saccharide units, such as 3-O-sulfo-D-glucuronic acid, were present in their product. This suggested a limitation in the selectivity of chemical sulfation/desulfation in HS synthesis. Further, the OST catalyzed sulfation reaction utilizes 3'-phosphoadenosine 5'-phosphosulfate (PAPS) as the sulfur donor, producing adenosine 3',5'-diphosphate (PAP). PAP can compete with PAPS for OST binding, which can result in inhibition of the sulfation reaction over time as PAP concentration increases in the reaction mixture.<br>
Thus, developing an effective and highly selective approach for O-sulfation of polysaccharides remains an unmet need in the art for the large scale synthesis of HS.<br>
SUMMARY<br>
This Summary lists several embodiments of the presently disclosed subject matter, and in many cases lists variations and permutations of these embodiments. This Summary is merely exemplary of the numerous and varied embodiments. Mention of one or more representative features of a given embodiment is likewise exemplary. Such an embodiment can typically exist' with or without the feature(s) mentioned; likewise, those features can be applied to other embodiments of the presently disclosed subject matter, whether listed in this Summary or not. To avoid excessive repetition, this Summary does not list or suggest all possible combinations of such features.<br>
In some embodiments of the presently disclosed subject matter, a method of sulfating a polysaccharide is provided, comprising (a) providing a reaction mixture comprising at least one O-sulfotransferase (OST) enzyme and 3'-phosphoadenosine 5'-phosphosulfate (PAPS); (b) incubating a polysaccharide substrate with the reaction mixture, wherein production of the sulfated polysaccharide from the polysaccharide substrate is catalyzed by the OST enzyme with a conversion of the PAPS to adenosine 3',5'-diphosphate (PAP); and (c) providing a reaction condition which modifies PAP to reduce an inhibitory effect of PAP on the polysaccharide sulfation. In some embodiments, providing the reaction condition comprises providing a PAPS regeneration system comprising a PAPS regenerating enzyme and a sulfur donor compound, wherein the PAPS regenerating enzyme catalyzes regeneration of the PAPS from the PAP utilizing the sulfur donor compound as a substrate. In some embodiments, the PAPS regenerating enzyme is an arylsulfotransferase, such as for example AST-IV. In some embodiments, the sulfur donor compound is an aryl sulfate compound, such as for example p-nitrophenol sulfate (PNPS). In some embodiments, providing the reaction condition comprises providing a phosphatase enzyme, wherein the phosphatase enzyme modifies the PAP.<br>
In some embodiments of the presently disclosed subject matter, a method of sulfating a polysaccharide is provided, comprising (a) providing a reaction mixture comprising PAP, a PAPS regenerating enzyme and a sulfur donor compound; (b) incubating the reaction mixture for a time period sufficient<br>
to catalyze tne production of PAPS from the PAP by the PAPS regenerating enzyme utilizing the sulfur donor compound as a substrate, such as for example a time period from about 1 minute to about 30 minutes; and (c) incubating a polysaccharide substrate and at least one OST enzyme with the reaction mixture, wherein production of a sulfated polysaccharide from the polysaccharide substrate is catalyzed by the OST enzyme with a conversion of the PAPS to PAP and wherein the PAPS regenerating enzyme catalyzes regeneration of the PAPS from the PAP utilizing the sulfur donor compound as a substrate. In some embodiments, the PAPS regenerating enzyme is an arylsulfotransferase, such as for example AST-IV. In some embodiments, the donor compound is an aryl sulfate compound, such as for example PNPS.<br>
In some embodiments of the methods for sulfating polysaccharides, the at least one OST enzyme is selected from the group consisting of 2-OST, 3-OST-1, 3-OST-3, 6-OST, and combinations thereof. In some embodiments, the at least one OST enzyme is a recombinant OST enzyme, which is, in some embodiments, produced in a bacterial expression system. Further, In some embodiments, the OST enzyme is a fusion protein, such as for example a maltose-binding protein (MBP)-2-OST fusion protein or a MBP-6-OSTfusion protein. In some embodiments, the OST enzyme is immobilized on a substrate, such as for example an agarose bead.<br>
In some embodiments of the methods for sulfating polysaccharides, the polysaccharide substrate is a chemically desulfated A/-sulfated (CDSNS) heparin. In some embodiments, the polysaccharide substrate is partially sulfated prior to reaction mixture incubation. In some embodiments, the sulfated polysaccharide is a glycosaminoglycan (GAG), such as for example a heparan sulfate (HS). In some embodiments, the sulfated polysaccharide is an HS that is an anticoagulant-active HS, an antithrombin-binding HS, afibroblast growth factor (FGF)-binding HS, a herpes simplex virus envelope glycoprotein D-binding HS, or has a combination of these properties.<br>
In some embodiments of the presently disclosed subject matter, a kit for sulfating a polysaccharide is provided. In some embodiments, the kit comprises at least one OST enzyme and a reagent which modifies PAP to reduce an inhibitory effect of PAP on polysaccharide sulfation. In some<br>
emDodiments, the kit further comprising instructions for sulfating a polysaccharide. In some embodiments, the at least one OST enzyme is contained within a first container and the reagent is contained within a second container. In some embodiments, the at least one OST enzyme is selected from the group consisting of 2-OST, 3-OST-1, 3-OST-3, 6-OST, and combinations thereof. Further, in some embodiments, the OST enzyme is a recombinant OST enzyme, such as for example a recombinant OST enzyme produced in a bacterial expression system. In some embodiments, the OST enzyme is a fusion protein, such as for example a maltose-binding protein (MBP)-2-OST fusion protein or a MBP-6-OST fusion protein. In some embodiments, the OST enzyme is immobilized on a substrate, such as for example an agarose bead.<br>
In some embodiments of the kit, the reagent comprises a PAPS regeneration system comprising a PAPS regenerating enzyme (e.g., an arylsulfotransferase, such as AST-IV) and a sulfur donor compound (e.g., an aryl sulfate compound, such as PNPS. In other embodiments of the kit, the reagent comprises a phosphatase enzyme.<br>
Accordingly, it is an object of the presently disclosed subject matter to provide for the enzymatic synthesis of sulfated polysaccharides. This object is achieved in whole or in part by the presently disclosed subject matter.<br>
An object of the presently disclosed subject matter having been stated above, other objects and advantages will become apparent to those of ordinary skill in the art after a study of the following description of the presently disclosed subject matter and non-limiting examples.<br>
BRIEF DESCRIPTION OF THE DRAWINGS<br>
Figures 1A and 1B are schematic drawings showing synthesis of<br>
sulfated polysaccharides coupled with a PAPS enzymatic regeneration system.<br>
Figure 1A shows the stepwise enzymatic synthesis of sulfated polysaccharides<br>
using HS O-sulfotransferases. The description of intermediate polysaccharides<br>
is disclosed in the Examples herein below.   Compounds 4a and 4b were<br>
prepared by inverting the order of sulfation steps.   4a was prepared by<br>
incubating compound 1 with 2-OST followed by 6-OST, whereas 4b was<br>
prepared by incubating compound 1 with 6-OST followed by 2-OST. Figure 1B shows the reaction catalyzed by arylsulfotransferase IV (AST-IV) to generate PAPS. R represents -H or -SO3.<br>
Figure 2 is a graph showing activities of recycled immobilized sulfotransferases. The immobilized enzymes were utilized in multiple cycles. The activities of the immobilized enzymes after each cycle were determined as described in the Examples herein below. A total of 10 cycles was conducted. 2-OST (•); 6-OST (o); 3-OST-1 (T); and 3-OST-3 (A).<br>
Figures 3A-3E are RPIP-HPLC chromatograms of the disaccharide analysis of synthesized polysaccharides. The synthesized polysaccharides were digested with a mixture of heparinases, including heparin lyase I, II, III, and heparinase IV. The resultant disaccharides were purified by BioGel P-2 and resolved on RPIP-HPLC. Figures 3A-3E show the chromatograms of the disaccharide analysis of compounds 1, 2, 3, 4a, and 4b, from Figure 1A respectively. The numbers above peaks indicate the eluted positions of authentic disaccharide standards, where 1 represents AUA-GlcNAc, 2 represents AUA-GlcNS, 3 represents AUA-GlcNS6S, 4 represents AUA2S-GlcNS, and 5 represents AUA2S-GlcNS6S. * marks the eluted position of A UA2S-GlcNAc. The quantity of AUA2S-GlcNAc not determined.<br>
Figures 4A and 4B are graphs showing the effect of the synthesized polysaccharides on FGF-2-dependent BaF3 FGFRIc cell proliferation. Figure 4A, BaF3 FGFRIc cells were seeded in 96-well plates as described with 2 nM FGF2 for control and 2 nM FGF2 plus a 1 ug/ml concentration of the following compounds: heparin, 1, 2,3,4a, and 4b (Figure 1A). Figure 4B shows dose-response curves of heparin, 4a, and 4b for their activities in stimulating cell proliferation. Cells were cultured for 40 hours (h), followed by incubating in the media containing [3H]thymidine for 4 h. The cellular proliferation was determined by [3H]thymidine incorporation into the DMA. heparin (•); 4a (o); and 4b (T). Data are mean ± range of duplicates.<br>
DETAILED DESCRIPTION<br>
The details of one or more embodiments of the presently disclosed subject matter are set forth in the accompanying description below. Other<br>
features, objects, and advantages of the presently disclosed subject matter will be apparent from the detailed description, and from the claims. All publications, patent applications, patents, and other references mentioned herein are incorporated by reference in their entirety. Some of the polynucleotide and polypeptide sequences disclosed herein are cross-referenced to GENBANK® accession numbers. The sequences cross-referenced in the GENBANK® database are expressly incorporated by reference as are equivalent and related sequences present in GENBANK® or other public databases. Also expressly incorporated herein by reference are all annotations present in the GENBANK® database associated with the sequences disclosed herein. In case of conflict, the present specification, including definitions, will control.<br>
1.        General Considerations<br>
Heparan sulfates (HSs) are highly sulfated polysaccharides, present on the surface of mammalian cells and in the extracellular matrix in large quantities. HS is a highly charged polysaccharide consisting of 1 -* 4-linked glucosamine and glucuronic/iduronic acid units that contain both N- and O-sulfo groups. Heparin, a specialized form of HS, is a commonly used anticoagulant drug. Thus, "heparan sulfate", as used herein, includes heparin.<br>
HSs play critical roles in a variety of important biological processes, including assisting viral infection, regulating blood coagulation and embryonic development, suppressing tumor growth, and controlling the eating behavior of test subjects by interacting with specific regulatory proteins (Liu, J., and Thorp, S. C. (2002) Med. Res. Rev. 22:1-25; Rosenberg, R. D., etal., (1997) J. Clin. Invest. 99:2062-2070; Bernfield, M.,etal., (1999) Annu. Rev. Biochem.68:729-777; Alexander, C. M., etal., (2000) Nat. Genet.25:329-332; Reizes, O., etal., (2001) Cell 106:105-116). The unique sequences determine to which specific proteins HSs bind, thereby regulating biological processes.<br>
The biosynthesis of HS occurs in the Golgi apparatus. It is initially synthesized as a copolymer of glucuronic acid and /V-acetylated glucosamine by D-glucuronyl and A/-acetyl-D-glucosaminyltransferase, followed by various modifications (Lindahl, U., et al., (1998) J. Biol. Chem. 273:24979-24982).<br>
These modifications include Akleacetylation and A/-sulfation of glucosamine, C$ epimerization of glucuronic acid to form iduronic acid residues, 2-O-sulfation of iduronic and glucuronic acid, as well as 6-Osulfation and 3-Osulfation of glucosamine. Several enzymes that are responsible for the biosynthesis of HS have been cloned and characterized (Esko, J. D., and Lindahl, U, (2001) J. Clin. Invest. 108:169-173).<br>
The expression levels of various HS biosynthetic enzyme isoforms contribute to the synthesis of specific saccharide sequences in specific tissues. HS A/-deacetylase/A/-sulfotransferase, 3-Q-sulfotransferase, and 6-O sulfotransferase are present in multiple isoforms. Each isoform is believed to recognize a saccharide sequence around the modification site in order to generate a specificsulfated saccharide sequence (Liu, J., et al., (1999) J. Biol. Chem. 274:5185-5192 ; Aikawa, J.-l., et al., (2001) J. Biol. Chem. 276:5876-5882; Habuchi, H., et al., (2000) J. Biol. Chem. 275:2859-2868). For instance, HS D-glucosaminyl 3-Osulfotransferase (3-OST) isoforms generate 3-O-sulfated glucosamine residues that are linked to different sulfated uronic acid residues. 3-OST isoform 1 (3-OST-1) transfers sulfate to the 3-OH position of an A/-su!fated glucosamine residue that is linked to a glucuronic acid residue at the nonreducing end (GlcUA-GlcNS±6S). However, 3-OST isoform 3 (3-OST-3) transfers sulfate to the 3-OH position of an /V-unsubstituted glucosamine residue that is linked to a 2-O-sulfated iduronic acid at the nonreducing end (ldoUA2S-GlcNH2±6S) (Liu, J., et al., (1999) J. Biol. Chem. 274:38155-38162). The difference in the substrate specificity of 3-OSTs results in distinct biological functions. For example, the HS modified by 3-OST-1 binds to antithrombin (AT) and possesses anticoagulant activity (Liu, J., et al., (1996) J. Biol. Chem. 271:27072-27082). However, the HS modified by 3-OST-3 (3-OST-3A and 3-OST-3B) binds to glycoprotein D (gD) of herpes simplex virus, type 1, (HSV-1) thus mediating viral entry (Shukla, D., et al., (1999) Cell 99:13-22).<br>
The HS- and heparin-regulated anticoagulation mechanisms have been studied extensively. It is now known that HS, including heparin, interact with AT, a serine protease inhibitor, to inhibit the activities of thrombin and factor Xa in the blood coagulation cascade (Rosenberg, R. D., et al., (1997) J. Clin. Invest. 99:2062-2070). Anticoagulant-active HS (HSaot) and heparin contain one or<br>
multiple AT-binding sites per polysaccharide chain. This binding site contains a specific pentasaccharide sequence with a structure of -GlcNS(or Ac)6S-GlcUA-GlcNS3S(±6S)-ldoUA2S-GlcNS6S-. The 3-O-sulfation of glucosamine for generating GlcNS3S(±6S) residue, which is carried out by 3-OST-1, is an important modification for the synthesis of HSact (Liu, J., et al., (1996) J. Biol. Chem. 271:27072-27082; Shworak, N. W. , et al., (1997) J. Biol. Chem. 272:28008-28019).<br>
Cell surface HS also assists HSV-1 infection (WuDunn, D., and Spear, P. G. (1989) J. Virol. 63:52-58). One report (Shukla, D., et al., (1999) Cell 99:13-22) suggests that a specific 3-OsulfatedHS is involved in assisting HSV-1 entry. The 3-O-sulfated HS is generated by 3-OST-3 but not by 3-OST-1. In addition, the 3-O-sulfated HS provides binding sites for HSV-1 envelope glycoprotein D, which is a key viral protein involved in the entry of HSV-1 (Shukla, D., et al., (1999) Cell 99:13-22). Because 3-OST-3-modified HS is rarely found in HS from natural sources, the study suggests that HSV-1 recognizes a unique saccharide structure. Indeed, the result from the structural characterization of a gD-binding octasaccharide revealed that the octasaccharide possesses a specific saccharide sequence (Liu, J., et al., (2002) J. Biol. Chem. 277:33456-33467). In addition, the binding affinity of the 3-O-sulfated HS for gD is about 2 pM (Shukla, D., et al., Cell99:13-22). This affinity is similar to that reported for the binding of gD to the protein receptors, suggesting that HSV-1 utilizes both protein and HS cell surface receptors to infect target cells (Willis, S. H., et al., (1998) J. Virol. 72:5938-5947; Krummenacher, C., etal., (1999) J. Virol. 73:8127-8137). It is believed that the interaction between gD and the 3-O-sulfated HS or the protein entry receptors somehow triggers the fusion between the virus and the cell in the presence of otrierviral envelope proteins, including gB, gH, and gL (Shukla, D., and Spear, P. G. (2001) J. Clin. Invest. 108:503-510). A study of the co-crystal structure of gD and herpes entry receptor HveA suggests that the binding of HveA to gD induces conformational changes in jjp (Carfi, A., et al., (2001) Mo/. Ce//8:169-179).<br>
1L      Methods of Sulfatlna Polvsaccharides<br>
The presently disclosed subject matter provides enzymatic methods for the sulfation of multimilligram amounts of heparan sulfate having particular functions using sulfotransferases coupled with a system for reducing inhibitory effects from sulfur donor byproducts. In some embodiments, the system for reducing inhibitory byproducts comprises a 3'-phosphoadenosine 5'-phosphosulfate regeneration system. In other embodiments, the system comprises a phosphatase enzyme. By utilizing the presently disclosed sulfation system and selecting appropriate enzymatic modification steps, an inactive precursor polysaccharide can been converted to a heparan sulfate having desired biological properties.<br>
In some embodiments, the presently disclosed subject matter employs recombinant sulfotransferases. Because the recombinant sulfotransferases can be recombinantly expressed in bacteria, and the disclosed methods can use low cost sulfo donors, the presently disclosed subject matter can be readily utilized to synthesize large quantities of biologically active heparan sulfates while reducing the production of reaction inhibitory byproducts.<br>
Two advantages provided by the presently disclosed subject matter facilitate the large scale synthesis of HS. First, large amounts of all the required HS sulfotransferases can be successfully recombinantly expressed in Escherichia coli. Second, the enzymatic sulfation reactions are coupled with a system for reducing inhibitory effects from sulfur donor byproducts (e.g., PAP) and reducing costs related to continuously providing a supply of th&amp; sulfur donor PAPS. PAPS, a universal sulfate donor and source of sulfate for all sulfotransferases, is a highly expensive and unstable molecule that has been an obstacle to the large-scale production of enzymatically sulfated products. The half-life of PAPS in aqueous solution at pH 8.0 is approximately 20 hours. Product inhibition by adenosine 3',5'-diphosphate (PAP) has also been a limiting factor to large-scale applications. For example, PAP inhibition of hydroxysteroid sulfotransferase was determined to be K/= 14 uM (Marcus et al. (1980) Aial. Blochem. 107, 296). PAP has also been shown to inhibit the sulfotransferase NodST with a K/=0.1 uM (Linetal., (1995) J.Am. Chem. Soc. 117,8031). In some embodiments of the presently disclosed subject matter, a<br>
PAPS regeneration system, such as the system developed by Burkhart and colleagues (Burkart et al. (2000) J. Org. Chem. 65, 5565-5574, incorporated herein by reference), has been modified and adapted to be coupled to the enzymatic synthesis reactions. The PAPS regeneration system converts PAP into PAPS, thereby reducing accumulation of inhibitory PAP in the reaction mixture and reducing production costs related to providing PAPS to drive the sulfation reaction. In other embodiments, phosphatase enzymes can be utilized to modify PAP so that it no longer has binding affinity for sulfotransferases.<br>
The presently disclosed sulfation system can be adapted to produce a multitude of HS molecules having varied biological activities by selecting appropriate sulfotransferases to include and by sequentially controlling the addition of those sulfotransferases to the reaction system to facilitate appropriate timing of sulfations of the polysaccharide template. For example, as disclosed herein, HS having specific biological activities can be synthesized utilizing the presently disclosed methods, including anticoagulant HS, fibroblast growth factor-2-binding activity, herpes simplex virus glycoprotein D (gD)-binding HS, and fibroblast growth factor 2 (FGF2) receptor -binding HS. Only two or three enzymatic steps are required for the synthesis of each of these biologically-active HS molecules (Figure 1A). Thus, the presently disclosed subject matter provides an efficient and effective method for the large scale synthesis of a wide range of HS with specific activity required. In addition, this method provides a model system to better understand the biosynthesis of HS.<br>
In some embodiments of the presently disclosed subject matter, a method of sulfating a polysaccharide is provided. In some embodiments, the method comprises incubating a polysaccharide substrate to be sulfated with a reaction mixture that comprises at least one sulfotransferase enzyme, such as for example an Osulfotransferase (OST) enzyme, and a sulfur donor, such as for example PAPS. Production of the sulfated polysaccharide from the polysaccharide substrate is catalyzed by the OST enzyme with a conversion of the PAPS to adenosine 3',5'-diphosphate (PAP). A reaction condition is further provided that modifies generated PAP to reduce an inhibitory effect of PAP on the polysaccharide sulfation. For example, a PAPS regeneration system can<br>
oe coupiea witn the sulfation reaction to convert PAP into PAPS or phosphatases can be added to the reaction mixture to modify PAP such that it does not compete with PAPS for binding with OSTs.<br>
In some embodiments, the polysaccharide substrate is a previously A/,O-desulfated and re-A/-sulfated polysaccharide, such as for example a chemically desulfated /V-sulfated (CDSNS) heparin. In other embodiments, the polysaccharide substrate is partially sulfated prior to reaction mixture incubation. For example, a CDSNS can be reacted with a particular OST to produce a sulfated polysaccharide intermediate product that can then be reacted with a different OST to further sulfate the polysaccharide at different locations. This sequential process of reacting the polysaccharide substrate with different OSTs can be continued until a final polysaccharide is produced exhibiting desired biological activities (based, at least in part, on the sulfation pattern of the polysaccharide). Figure 1A schematically illustrates several exemplary polysaccharide substrates sequentially reacted with different OSTs to produce different intermediate and end products. For example, compound 1 can be reacted with 2-OST and then 6-OST to produce compound 4a or compound 1 can be reacted with 6-OST and then 2-OST to produce compound 4b, each having fibroblast growth factor (FGF)-binding activity. Further, compounds 4a or 4b can be reacted with 3-OST-1 to produce compound 5 having antithrombin-binding and anticoagulant activities. Alternatively, compounds 4a or 4b can be reacted with 3-OST-3 to produce compound 6 having herpes simplex virus (HSV) envelope glycoprotein D (gD) binding activity. As such, the presently disclosed subject matter provides for the production of HS compounds having different biological properties based on the selection and sequential reaction of different OSTs with polysaccharide substrates. The polysaccharide substrate can be reacted with different OST enzymes by addition of each enzyme sequentially to the same reaction mixture, or intermediate polysaccharide products can be purified from the reaction mixture after reaction of a particular OST and then reacted with a different OST. In some embodiments, depending on the desired final product, different OST enzymes can be added to the reaction mixture simultaneously.<br>
In some embodiments the sulfated polysaccharide product can be a glycosaminoglycan (GAG). GAGs are the most abundant heteropolysaccharides in the body. These molecules are long unbranched polysaccharides containing a repeating disaccharide unit. The disaccharide units can contain either of two modified sugars: N-acetylgalactosamine (GalNAc) or N-acetylglucosamine (GlcNAc) and a uronic acid such as glucuronate or iduronate. GAGs are highly negatively charged molecules, with extended conformation that imparts high viscosity to the solution. Along with the high viscosity of GAGs comes low compressibility, which makes these molecules ideal for a lubricating fluid in the joints. At the same time, their rigidity provides structural integrity to cells and provides passageways between cells, allowing for cell migration. The specific GAGs of physiological significance are hyaluronic acid, dermatan sulfate, chondroitin sulfate, heparin, heparan sulfate (including heparin), and keratan sulfate. Thus, in some embodiments, the sulfated polysaccharide product is a HS. In some embodiments, the sulfated polysaccharide product is an anticoagulant-active HS, an antithrombin-binding HS, an FGF-binding HS, and an HSV gD-binding HS.<br>
II.A.    Sulfotransferases<br>
As previously noted, the presently disclosed subject matter utilizes sulfotransferases, particularly O-sulfotransferases (OSTs), to sulfate polysaccharides. Sulfotransferases comprise a family of enzymes that catalyze the transfer of a sulfonate or sulfuryl group (SO3) from the cofactor 3'-phosphoadenosine-5'-phosphosulfate (PAPS) to an acceptor molecule. Even though it is more accurate to call these sulfonation reactions, the term sulfation is still widely used. Therefore, the term "sulfation" as used herein refers to a transfer of a sulfonate or sulfuryl group from one molecule to another.<br>
Sulfotransferases mediate sulfation of different classes of substrates such as carbohydrates, oligosaccharides, peptides, proteins, flavonoids, and steroids for a variety of biological functions including signaling and modulation of receptor binding (Bowman et al., (1999) Chem. Blol. 6, R9-R22; and Falany (1997) FASEB J. 11, 1-2). Within the past few years, many new sulfotransferases have been identified and cloned (Aikawa et al., (1999) J. Blol.<br>
Chem. 274,2690; Dooley (1998) Chemico-Blol. Interact 109,29; Fukuta et al. (1998) Biochim. Biophys. Act. 1399, 57; Habuchi et al., (1998) J. Biol. Chem. 273,9208; Mazany et al., (1998) Biochim. Biophys. Act 1407,92; Nastuk et al. (1998) J. NeuroscienceW, 7167; Ong et al., (1998) J. Biol. Chem. 273,5190; Ouyang et al., (1998) J. Biol. Chem. 273, 24770; Saeki et al. (1998) J. Biochem. 124, 55; Uchimura et al. (1998) J. Biol. Chem. 273, 22577; and Yoshinari et al., (1998) J. Biochem. 123, 740).<br>
As used herein, the term "O-sulfotransferase (OST)" includes polypeptides and nucleic acids encoding HS O-sulfotransferases, such as for example "2-OST" (e.g., mouse 2-OST, GENBANK® Accession No. AAC40135 (SEQ ID NO:1); "3-OST-1" (e.g., human 3-OST-1, GENBANK® Accession No. NP_005105 (SEQ ID N0:2); "3-OST-3" (e.g., human 3-OST-3A, GENBANK® Accession No. NP_006033 (SEQ ID NO:3) and human 3-OST-3B, GENBANK® Accession No. NP_006032 (SEQ ID NO:4); and "6-OST" (e.g., mouse 6-OST-1, GENBANK® Accession No. NP_056633 (SEQ ID NO:5), mouse 6-OST-2, GENBANK® Accession No. BAA89247 (SEQ ID NO:6), and mouse 6-OST-3, GENBANK® Accession No. NP_056635 (SEQ ID NO:7)), which are HS 2-O-sulfotransferase, HS 3-Osulfotransferase isoform 1, HS 3-O-sulfotransferase isoform 3, and HS 6-O-sulfotransferase, respectively.<br>
The term "OST" includes invertebrate and vertebrate homologs of the O-sulfotransferases (e.g., mammalian (such as human and mouse), insect, and<br>
t<br>
avian homologs). As such, although exemplary embodiments of particular OSTs have been disclosed herein, the presently disclosed subject matter is not intended to be limited to the disclosed examples, but rather "OST", including particular OSTs (e.g., 2-OST, 3-OST-1, 3-OST-3, and 6-OST), includes all comparable OSTs known to the skilled artisan.<br>
The terms "OST gene product", "OST protein", and "OST polypeptide" refer to peptides having amino acid sequences which are substantially identical to native amino acid sequences from the organism of interest and which are biologically active in that they comprise all or a part of the amino acid sequence of a HS O-sulfotransferase isoform, or cross-react with antibodies raised against a HS O-sulfotransferase isoform polypeptide, or retain all or some of<br><br>
the biological activity of the native amino acid sequence or protein.  Such biological activity can include immunogenlcity.<br>
The terms "OST gene product", "OST protein", and "OST polypeptide" also include analogs of HS O-sulfotransferase molecules. By "analog" is intended that a DNA or peptide sequence can contain alterations relative to the sequences disclosed herein, yet retain all or some of the biological activity of those sequences. Analogs can be derived from genomic nucleotide sequences as are disclosed herein or from other organisms, or can be created synthetically. Those skilled in the art will appreciate that other analogs, as yet undisclosed or undiscovered, can be used to design and/or construct OST analogs. There is no need for a "OST gene product", "OST protein", and "OST polypeptide" to comprise all or substantially all of the amino acid sequence of a native OST gene product. Shorter or longer sequences are anticipated to be of use in the presently disclosed subject matter, shorter sequences are herein referred to as "segments." Thus, the terms "OST gene product", "OST protein", and "OST polypeptide" also include fusion or recombinant HS O-suIfotransferase polypeptides and proteins comprising sequences of the OST protein. Methods of preparing such proteins are known in the art. For example, in some embodiments, the OST is a 2-OST or a 6-OST fusion protein, such as a maltose-binding protein (MBP)-2-OST fusion protein or a MBP-6-OST fusion protein, as disclosed herein.<br>
The terms "OST gene", "OST gene sequence", and "OST gene segment" refer to any DNA sequence that is substantially identical to a polynucleotide sequence encoding a HS O-sulfotransferase isoform gene product, protein or polypeptide as defined above, and can also comprise any combination of associated control sequences. The terms also refer to RNA, or antisense sequences, complementary to such DNA sequences. As used herein, the term "DNA segment" refers to a DNA molecule that has been isolated free of total genomic DNA of a particular species. Furthermore, a DNA segment encoding a HS O-sulfotransferase polypeptide refers to a DNA segment that contains OST coding sequences, yet is isolated away from, or purified free from, total genomic DNA of a source species, such as for example Homo sapiens. Included within the term "DNA segment" are DNA segments<br>
and smaller fragments of such segments, and also recombinant vectors, including, for example, plasmids, cosmids, phages, viruses, and the like.<br>
The term "substantially identical", when used to define either a OST gene product or amino acid sequence, or a OST gene or nucleic acid sequence, means that a particular sequence varies from the sequence of a natural OST by one or more deletions, substitutions, or additions, the net effect of which is to retain at least some of biological activity of the natural gene, gene product, or sequence. Such sequences include "mutant" sequences, or sequences in which the biological activity is altered to some degree but retains at least some of the original biological activity.<br>
Alternatively, DMA analog sequences are "substantially identical" to specific DNA sequences disclosed herein if: (a) the DMA analog sequence is derived from coding regions of the natural OST gene; or (b) the DNA analog sequence is capable of hybridization of DNA sequences of (a) under stringent conditions and which encode biologically active OST gene products; or (c) the DNA sequences are degenerate as a result of alternative genetic code to the DNA analog sequences defined in (a) and/or (b). Substantially identical analog proteins will be greater than about 60%, 70%, 80%, 90%, 95%, 96%, 97%, 98%, or 99% identical to the corresponding sequence of the native protein. Sequences having lesser degrees of identity but comparable biological activity are considered to be equivalents. In determining nucleic acid sequences, all subject nucleic acid sequences capable of encoding substantially similar amino acid sequences are considered to be substantially similar to a reference nucleic acid sequence, regardless of differences in codon sequences or substitution of equivalent amino acids to create biologically functional equivalents.<br>
Sequence identity or percent similarity of a DNA or peptide sequence can be determined, for example, by comparing sequence information using the GAP computer program, available from the University of Wisconsin Geneticist Computer Group. The GAP program utilizes the alignment method of Needleman et al. (1970) J Mol Biol48:443, as revised by Smith et al. (1981) Adv Appl Math 2:482. Briefly, the GAP program defines similarity as the number of aligned symbols (I.e., nucleotides or amino acids) that are similar, divided by the total number of symbols in the shorter of the two sequences.<br>
The preferred parameters for the GAP program are the default parameters, which do not impose a penalty for end gaps. See Schwartz et al. (1979) Nuc Acids fles6(2):745-755; Gribskov et al. (1986) Nuc Acids Res 14(1):327-334.<br>
In certain embodiments, the present subject matter concerns the use of OST genes and gene products that include within their respective sequences a sequence that is essentially that of an OST gene, or the corresponding protein. The term "a sequence essentially as that of an OST gene", means that the sequence is substantially identical or substantially similar to a portion of an OST gene and contains a minority of bases or amino acids (whether DNA or protein) which are not identical to those of an OST protein or an OST gene, or which are not a biologically functional equivalent. The term "biologically functional equivalent" is well understood in the art and is further defined in detail herein. Nucleotide sequences are "essentially the same" where they have between about 75% and about 85% or more preferably, between about 86% and about 90%, or more preferably greater than 90%, or more preferably between about 91% and about 95%, or even more preferably, between about 96% and about 99%; of nucleic acid residues which are identical to the nucleotide sequence of a OST gene. Similarly, peptide sequences which have about 60%, 70%, 80%, or 90%, or preferably from 90-95%, or more preferably greater than 96%, or more preferably 95-98%, or most preferably 96%, 97%, 98%, or 99% amino acids which are identical or functionally equivalent or biologically functionally equivalent to the amino acids of an OST polypeptide will be sequences which are "essentially the same".<br>
OST gene products and OST encoding nucleic acid sequences, which have functionally equivalent codons, are also covered by the presently disclosed subject matter. The term "functionally equivalent codon" is used herein to refer to codons that encode the same amino acid, such as the ACG and AGU codons for serine. Applicants contemplate substitution of functionally equivalent codons of Table 1 into sequences of OSTs disclosed herein as equivalents.<br>
TABLE 1	-  Functionally Equivalent Codons<br>
(Tabble Removed)<br>
It will also be understood by those of skill in the art that amino acid and nucleic acid sequences can include additional residues, such as additional N-or C-terminal amino acids or 5' or 3' nucleic acid sequences, and yet still be encompassed by the OSTs disclosed herein, so long as the sequence retains biological protein activity where protein expression is concerned. The addition of terminal sequences particularly applies to nucleic acid sequences which can, for example, include various non-coding sequences flanking either of the 5' or 3' portions of the coding region or can include various internal sequences, i.e., introns, which are known to occur within genes.<br>
The present subject matter also encompasses the use of nucleotide segments that are complementary to the sequences of the present subject matter, in one embodiment, segments that are fully complementary, i.e. complementary for their entire length. Nucleic acid sequences that are "complementary" are those, which are base-paired according to the standard Watson-Crick complementarity rules. As used herein, the term "complementary sequences" means nucleic acid sequences which are substantially complementary, as can be assessed by the same nucleotide comparison set forth above, or is defined as being capable of hybridizing to the nucleic acid segment in question under relatively stringent conditions such as those described herein. A particular example of a complementary nucleic acid segment is an antisense oligonucleotide.<br>
One technique in the art for assessing complementary sequences and/or isolating complementary nucleotide sequences is hybridization. Nucleic acid hybridization will be affected by such conditions as salt concentration, temperature, or organic solvents, in addition to the base composition, length of the complementary strands, and the number of nucleotide base mismatches between the hybridizing nucleic acids, as will be readily appreciated by those skilled in the art. Stringent temperature conditions will generally include temperatures in excess of about 30°C, typically in excess of about 37°C, and preferably in excess of about 45°C. Stringent salt conditions will ordinarily be less than about 1,000 mM, typically less than about 500 mM, and preferably less than about 200 mM. However, the combination of parameters is much more important than the measure of any single parameter. See e.g.. Wethmur &amp; Davidson (1968) J Mol Biol 31:349-370. Determining appropriate hybridization conditions to identify and/or isolate sequences containing high levels of homology is well known in the art. See e.g., Sambrook et a). (2001) Molecular Cloning: A Laboratory Manual. Third Edition, Cold Spring Harbor Laboratory Press, Cold Spring Harbor, New York.<br>
For the purposes of specifying conditions of high stringency, preferred conditions are salt concentration of about 200 mM and temperature of about 45°C. One example of such stringent conditions is hybridization at 4XSSC, at 65°C, followed by a washing in 0.1XSSC at 65°C for one hour. Another<br>
exemplary stringent hybridization scheme uses 50% formamide, 4XSSC at 42°C. Another example of "stringent conditions" refers to conditions of high stringency, for example 6XSSC, 0.2% polyvinylpyrrolidone, 0.2% Ficoll, 0.2% bovine serum albumin, 0.1% sodium dodecyl sulfate, 100 µg/ml salmon sperm DNA and 15% formamide at 68°C. Nucleic acids having sequence similarity are detected by hybridization under low stringency conditions, for example, at 50°C and 10XSSC (0.9 M NaCI/0.09 M sodium citrate) and remain bound when subjected to washing at 55°C in 1XSSC. Sequence identity can be determined by hybridization under stringent conditions, for example, at 50°C or higher and 0.1 XSSC (9 mM NaCI/0.9 mM sodium citrate).<br>
Nucleic acids that are substantially identical to the provided OSTs, e.g., allelic variants, genetically altered versions of the gene, etc., bind to the disclosed OSTs under stringent hybridization conditions. By using probes, particularly labeled probes of DNA sequences, one can isolate homologous or related genes. The source of homologous genes can be any species, e.g., primate species, particularly human; rodents, such as rats and mice; canines; felines; bovines; ovines; equines; insects; yeasts; nematodes; etc.<br>
Between mammalian species, e.g., human, mouse and rat, homologs have substantial sequence similarity, i.e. at least 75% sequence identity between nucleotide sequences. Sequence similarity is calculated based on a reference sequence, which can be a subset of a larger sequence, such as a conserved motif, coding region, flanking region, etc. A reference sequence will usually be at least about 18 nucleotides long, more usually at least about 30 nucleotides long, and can extend to the complete sequence that is being compared. Algorithms for sequence analysis are known in the art, such as BLAST, described in Altschul et al. (1990) J Mol Blol 215:403-410. The sequences provided herein are essential for recognizing OST related and homologous proteins in database searches.<br>
At a biological level, identity is just that, i.e. the same amino acid at the same relative position in a given family member of a gene family. Homology and similarity are generally viewed as broader terms. For example,<br>
•<br>
biochemically similar amino acids, for example leuclne and isoleucine or glutamate/aspartate, can be present at the same position - these are not<br>
identical per se, but are biochemically "similar". As disclosed herein, these are referred to as conservative differences or conservative substitutions. This differs from a conservative mutation at the DNA level, which changes the nucleotide sequence without making a change in the encoded amino acid, e.g., TCC to TCA, both of which encode serine.<br>
When percentages are referred to herein, it is meant to refer to percent identity. The percent identities referenced herein can be generated by alignments with the program GENEWORKSTM (Oxford Molecular, Inc. of Campbell, California, U.S.A.) and/or the BLAST program at the NCBI website. Another commonly used alignment program is entitled CLUSTAL W and is described in Thompson et al. (1994) Nucleic Acids Res 22(22):4673-4680, among other places.<br>
The term "gene" is used for simplicity to refer to a functional protein, polypeptide or peptide encoding unit. As will be understood by those in the art, this functional term includes both genomic sequences and cDNA sequences.<br>
As noted above, modifications and changes can be made in the structure of the OST proteins and peptides described herein and still constitute a molecule having like or otherwise desirable characteristics. For example, certain amino acids can be substituted for other amino acids in a protein structure without appreciable loss of interactive capacity with, for example, structures in the nucleus of a cell. Since it is the interactive capacity and nature of a protein that defines that protein's biological functional activity, certain amino acid sequence substitutions can be made in a protein sequence (or the nucleic acid sequence encoding it) to obtain a protein with the same, enhanced, or antagonistic properties. Such properties can be achieved by interaction with the normal targets of the native protein, but this need not be the case, and the biological activity of the presently disclosed subject matter is not limited to a particular mechanism of action. It is thus contemplated in accordance with the present subject matter that various changes can be made in the sequence of the OST proteins and peptides or underlying nucleic acid sequence without appreciable loss of their biological utility or activity.<br>
Biologically functional equivalent peptides, as used herein, are peptides in which certain, but not most or all, of the amino acids can be substituted.<br>
Thus, applicants contemplate substitution of codons that encode biologically equivalent amino acids as described herein into the sequences of the disclosed OSTs, but which are not set forth herein in their entirety for convenience.<br>
Alternatively, functionally equivalent proteins or peptides can be created via the application of recombinant DNA technology, in which changes in the protein structure can be engineered, based on considerations of the properties of the amino acids being exchanged. Changes designed by man can be introduced through the application of site-directed mutagenesis techniques, e.g., to introduce improvements to the antigenicity of the protein or to test OST mutants in order to examine OST sulfotransferase activity, or other activity at the molecular level.<br>
Amino acid substitutions, such as those which might be employed in modifying the OST proteins and peptides described herein, are generally based on the relative similarity of the amino acid side-chain substituents, for example, their hydrophobicity, hydrophilicity, charge, size, and the like. An analysis of the size, shape and type of the amino acid side-chain substituents reveals that arginine, lysine and histidine are all positively charged residues; that alanine, glycine and serine are all of similar size; and that phenylalanine, tryptophan and tyrosine all have a generally similar shape. Therefore, based upon these considerations, arginine, lysine and histidine; alanine, glycine and serine; and phenylalanine, tryptophan and tyrosine; are defined herein as biologically functional equivalents. Other biologically functionally equivalent changes will be appreciated by those of skill in the art.<br>
In making biologically functional equivalent amino acid substitutions, the hydropathic index of amino acids can be considered. Each amino acid has been assigned a hydropathic index on the basis of their hydrophobicity and charge characteristics, these are: isoleucine (+ 4.5); valine (+ 4.2); leucine (+ 3.8); phenylalanine (+ 2.8); cysteine (+ 2.5); methionine (+1.9); alanine (+1.8); glycine (-0.4); threonine (-0.7); serine (-0.8); tryptophan (-0.9); tyrosine (-1.3); proline (-1.6); histidine (-3.2); glutamate (-3.5); glutamine (-3.5); aspartate (-3.5); asparagine (-3.5); lysine (-3.9); and arginine (-4.5).<br>
The importance of the hydropathic amino acid index in conferring interactive biological function on a protein is generally understood in the art<br>
(Kyte et al. (1982) J Mol Blol 157:105, incorporated herein by reference). It is known that certain amino acids can be substituted for other amino acids having a similar hydropathic index or score and still retain a similar biological activity. In making changes based upon the hydropathic index, the substitution of amino acids whose hydropathic indices are within ± 2 of the original value is preferred, those, which are within ± 1 of the original value, are particularly preferred, and those within ± 0.5 of the original value are even more particularly preferred.<br>
It is also understood in the art that the substitution of like amino acids can be made effectively on the basis of hydrophilicity. U.S. Patent No. 4,554,101, incorporated herein by reference, states that the greatest local average hydrophilicity of a protein, as governed by the hydrophilicity of its adjacent amino acids, correlates with its immunogenicity and antigenicity, i.e. with a biological property of the protein. It is understood that an amino acid can be substituted for another having a similar hydrophilicity value and still obtain a biologically equivalent protein.<br>
As detailed in U.S. Patent No. 4,554,101, the following hydrophilicity values have been assigned to amino acid residues: arginine (+ 3.0); lysine (+ 3.0); aspartate (+ 3.0 ± 1); glutamate (+ 3.0 ± 1); serine (+ 0.3); asparagine (+ 0.2); glutamine (+ 0.2); glycine (0); threonine (-0.4); proline (-0.5 ± 1); alanine (-0.5); histidine (-0.5); cysteine (-1.0); methionine (-1.3); valine (-1.5); leucine (-1.8); isoleucine (-1.8); tyrosine (-2.3); phenylalanine (-2.5); tryptophan (-3.4).<br>
In making changes based upon similar hydrophilicity values, the substitution of amino acids whose hydrophilicity values are within ± 2 of the original value is preferred, those, which are within ± 1 of the original value, are particularly preferred, and those within ± 0.5 of the original value are even more particularly preferred.<br>
While discussion has focused on functionally equivalent polypeptides arising from amino acid changes, it will be appreciated that these changes can be effected by alteration of the encoding DNA, taking into consideration also that the genetic code is degenerate and that two or more codons can code for the same amino acid.<br>
Thus, it will also be understood that the presently disclosed subject matter is not limited to the particular nucleic acid and amino acid sequences of<br>
the OSTs disclosed herein. Recombinant vectors and isolated DNA segments can therefore variously include the Osulfotransferase polypeptide-encoding region itself, include coding regions bearing selected alterations or modifications in the basic coding region, or include larger polypeptides which nevertheless comprise the Osulfotransferase polypeptide-encoding regions or can encode biologically functional equivalent proteins or peptides which have variant amino acid sequences. Biological activity of an Osulfotransferase can be determined using techniques generally known in the art, for example as disclosed herein in the Examples.<br>
The nucleic acid segments of the present subject matter, regardless of the length of the coding sequence itself, can be combined with other DNA sequences, such as promoters, enhancers, polyadenylation signals, additional restriction enzyme sites, multiple cloning sites, other coding segments, and the like, such that their overall length can vary considerably. It is therefore contemplated that a nucleic acid fragment of almost any length can be employed, with the total length preferably being limited by the ease of preparation and use in the intended recombinant DNA protocol. For example, nucleic acid fragments can be prepared which include a short stretch complementary to a nucleic acid sequence set forth in any of the OSTs disclosed herein, such as about 10 nucleotides, and which are up to 10,000 or 5,000 base pairs in length, with segments of 3,000 being preferred in certain cases. DNA segments with total lengths of about 4,000, 3,000, 2,000,1,000, 500,200,100, and about 50 base pairs in length are also contemplated to be useful.<br>
Recombinant vectors form further aspects of the present subject matter. Particularly useful vectors are those in which the coding portion of the DNA segment is positioned under the control of a promoter. The promoter can be that naturally associated with the OST gene, as can be obtained by isolating the 5' non-coding sequences located upstream of the coding segment or exon, for example, using recombinant cloning and/or polymerase chain reaction (PCR) technology and/or other methods known in the art, in conjunction with the compositions disclosed herein.<br>
In other embodiments, it is contemplated that certain advantages will be gained by positioning the coding DNA segment under the control of a recombinant, or heterologous, promoter. As used herein, a recombinant or heterologous promoter is a promoter that is not normally associated with a 3-O-sulfotransferase gene in its natural environment. Such promoters can include promoters isolated from bacterial, viral, eukaryotic, or mammalian cells. Naturally, it will be important to employ a promoter that effectively directs the expression of the DNA segment in the cell type chosen for expression. The use of promoter and cell type combinations for protein expression is generally known to those of skill in the art of molecular biology (See, e.g., Sambrook et al. (2001) Molecular Cloning; A Laboratory Manual. Third Edition, Cold Spring Harbor Laboratory Press, Cold Spring Harbor, New York). The promoters employed can be constitutive or inducible and can be used under the appropriate conditions to direct high level expression of the introduced DNA segment, such as is advantageous in the large-scale production of recombinant proteins or peptides.<br>
In some embodiments of the method disclosed herein for sulfating polysaccharides, the OST enzyme is immobilized on a substrate. This provides an advantage in that the substrate to which the OSTs are attached can be washed after a sulfation reaction to remove all components of the reaction except the bound OSTs. As such, the products of the reaction can be more easily separated from the enzymes catalyzing the reaction and the OSTs can be recycled and utilized again in multiple sulfation reactions. In some embodiments, the substrate is agarose. In particular embodiments, the agarose substrate is an agarose bead and the OSTs are linked to the beads.<br>
II.B.    Reduction of Inhibitory Effects of PAP<br>
The presently disclosed method for sulfating polysaccharides can comprise providing a "reaction condition" that modifies PAP to reduce inhibitory effects of PAP, such as competing with PAPS for binding with OSTs, on the polysaccharide sulfation. In some embodiments, the reaction condition comprises a phosphatase enzyme. The phosphatase enzyme can remove a phosphate from the PAP, which reduces its binding affinity for OSTs. In some embodiments, the phosphatase is 3'-ribonucleotide phosphohydrolase.<br>
In some embodiments, the reaction condition is a PAPS regeneration system, which comprises a PAPS regenerating enzyme and a sulfur donor compound. The PAPS regenerating enzyme catalyzes regeneration of the PAPS from the PAP utilizing the sulfur donor compound as a substrate. See, e.g., U.S. Patent No. 6,255,088; Burkart et al., (2000) J. Org. Chem. 65,5565-5574, which is herein incorporated by reference in its entirety. Thus, the PAPS regeneration system provides the dual advantages of reducing the inhibitory effects of PAP accumulation on sulfotransferase activity while also constantly "recharging" the reaction mixture with the primary sulfur donor molecule, PAPS.<br>
Thus, an aspect of the presently disclosed subject matter is directed to a sulfur donor compound (e.g., PAPS) regeneration process coupled with sulfation of a polysaccharide substrate. In particular, the process can be of a type wherein the sulfation of a polysaccharide substrate is catalyzed by a sulfotransferase, such as one or more OSTs, with a conversion of 3'-phosphoadenosine-5'-phosphosulfate (PAPS) to adenosine 3',5'-diphosphate (PAP). The sulfation process can be coupled with an enzymatic regeneration of the 3'-phosphoadenosine-5'-phosphosulfate from the adenosine 3',5'-diphosphate. The enzymatic regeneration can employ an arylsu Ifotransferase as the catalyst and an aryl sulfate as a substrate. As previously disclosed, preferred carbohydrate substrates can include GAGs, such as for example heparan sulfates, including heparin.<br>
In some embodiments, the arylsulfotransferase is a recombinant aryl sulfotransferase IV (AST-IV; e.g., rat AST-IV (SEQ ID NO:8)). This enzyme, when coupled to a sulfotransferase of choice, transfers sulfate from an aryl sulfate (e.g., p-nitrophenyl sulfate (PNPS) to PAP. This system averts product inhibition by PAP while regenerating PAPS In situ and can be monitored quantitatively by measurement of the absorbance of released p-nitrophenol at 400 nm.<br>
The enzyme AST-IV exists in two oxidative forms (Marshall et al., (1997) J. Biol. Chem. 272, 9153-9160; Marshall et al., (1998) Chem. -Biol. Interact. 109,107-116; Yang etal., (1998) Chem. -Biol. Interact. 109,129-135; Yang et al. (1996) Protein Expression Purif. 8,423-429; Quo et al. (1994) Chem.-Biol. Interact. 92,25-31; Chen et al. (1992) Protein Expression Purif. 3,421 -6; Lin et<br>
al. (1998) Anal, Biochem. 264,111-117; and Yang et al., (1997) Protein Eng. 10,70). These two oxidative forms can be easily resolved (Yang et al. (1996) Protein Expression Purif. 8,423-429), and the resolved physiologically relevant form has been utilized to assay picomole quantities of PAPS and PAP (Lin et al. (1998) Anal. Biochem. 264,111-117). As the bacterial expression of rat AST-IV has been demonstrated (Chen et al., (1992) Protein Expression Purif. 3,421 -6; and Ozawa et al., (1990) Nucleic Acids Res. 18,4001 z.), AST IV can be cloned from a rat liver cDNA library, overexpressed in a recombinant bacterial system (e.g., E. coli) and isolated (See, e.g., U.S. Patent No. 6,255,088, herein incorporated by reference in its entirety).<br>
Coupling the sulfotransferase catalyzed sulfation reaction with a PAPS regeneration system can provide a further advantage of generating PAPS utilized in the reaction directly from PAP. That is, the reaction mixture can be formulated to combine PAP with a PAPS regenerating enzyme prior to or simultaneously with addition of a sulfotransferase to the reaction mixture. The PAPS regenerating enzyme can then generate PAPS from the PAP for use by the sulfotransferase, thereby alleviating the need of supplying any of the more expensive and unstable PAPS to the reaction mixture. As such, in some embodiments of the presently disclosed subject matter a method of sulfating a polysaccharide is provided comprising providing a reaction mixture comprising therein adenosine 3',5'-diphosphate (PAP), a PAPS regenerating enzyme and a sulfur donor compound (other than PAPS) and incubating the reaction mixture for a time period sufficient to catalyze the production of 3'-phosphoadenosine 5'-phosphosulfate (PAPS) from the PAP by the PAPS regenerating enzyme utilizing the sulfur donor compound as a substrate. The method further comprises incubating a polysaccharide substrate and at least one Osulfotransferase (OST) enzyme with the reaction mixture, wherein production of a sulfated polysaccharide from the polysaccharide substrate is catalyzed by the OST enzyme with a conversion of the PAPS to PAP and wherein the PAPS regenerating enzyme then catalyzes regeneration of the PAPS from the PAP, again utilizing the sulfur donor compound as a substrate.<br>
11L      Kits<br>
The presently disclosed subject matter further provides kits for sulfating polysaccharides. In some embodiments, the kit comprises at least one sulfotransferase enzyme (e.g., at least one OST); and a reagent which modifies adenosine 3',5'-diphosphate (PAP) to reduce an inhibitory effect of PAP on the polysaccharide sulfation. In some embodiments of the kit the at least one sulfotransferase enzyme is contained within a first container and the reagent is contained within a second container. The kit can further comprise instructions for sulfating a polysaccharide.<br>
In some embodiments of the kit, the at least one sulfotransferase enzyme is an OST enzyme selected from the group consisting of 2-OST, 3-OST-1, 3-OST-3, 6-OST, and combinations thereof. In some embodiments, the OST enzyme is a recombinant OST enzyme, such as for example a recombinant OST enzyme produced in a bacterial expression system. In some embodiments, the OST enzyme is a fusion protein, such as for example a MBP-2-OST fusion protein or a MBP-6-OST fusion protein. Further, in some embodiments, the OST enzyme is immobilized to a substrate, such as for example an agarose bead.<br>
In some embodiments of the kit, the reagent comprises a PAPS regeneration system comprising a PAPS regenerating enzyme (e.g., AST-IV) and a sulfur donor compound (e.g., PNPS). In other embodiments of the kit, the reagent comprises a phosphatase enzyme (e.g., 3'-ribonucleotide phosphohydrolase).<br>
EXAMPLES<br>
The following Examples provide illustrative embodiments. In light of the present disclosure and the general level of skill in the art, those of skill will appreciate that the following Examples are intended to be exemplary only and that numerous changes, modifications, and alterations can be employed without departing from the scope of the presently claimed subject matter.<br>
MATERIALS AND METHODS FOR EXAMPLES 1-3 Preparation of Chemically Desulfated N-Sulfated Heparin (CDSNS Heparin). Treatment of heparin with a dimethyl sulfoxlde/methanol mixture (9:1, v/v) at 80 °C for 24 h resulted in nearly complete A/,O-desulfation with 
Expression and Purification of 2-OST and 6-OST. The catalytic domains of 2-OST of Chinese hamster ovary (Arg51-Asn356) and mouse 6-OST-1 (His53-Trp401) were cloned into apMAL-c2X vector (New England Biolabs, Beverly, Massachusetts, U.S.A.) using the BamHI and Hindlll sites to generate maltose-binding protein (MBP)-2-OST and MBP-6-OSTfusion proteins. The full-length cDNAs of 2-OST and 6-OST-1 were gifts from Dr. Rosenberg (Massachusetts Institute of Technology, Cambridge, Massachusetts, U.S.A.) and Dr. Kimata (Aichi University, Japan), respectively. Expression of 2-OST and 6-OST was achieved in Rosetta-gami B (DE3) cells (Novagen, a brand of EMD Biosciences, San Diego, California, U.S.A.) using a standard procedure. Briefly, cells containing the plasmid expressing 2-OST or 6-OST were grown in Luria broth (LB) medium supplemented with 2 mg/ml glucose, 15 ug/ml tetracycline, 15 ug/ml kenamycin, 35 ug/ml chloramphenicol, and 50 ug/ml carbenicillin at 37 °C. When the Aoo reached 0.6-0.8, Isopropyl-p-D-thiogalactopyranoside (1 mM) was added, and the cells were shaken overnight at 22 °C. The bacteria were harvested, and the proteins were purified by following a protocol from the manufacturer (New England Biolabs). The purified proteins migrated at 75 kDa on 12% SDS-PAGE with the purity greater than 80%.<br>
Preparation of 3-OST-1 bacterial expression plasmid (b3-OST-1-pET28). The cDNA fragment encoding the catalytic domain of 3-OST-1 (G48-H311) was amplified from m3-OST-1-pcDNA3 with a 5' overhang containing an Nde\ site and a 3' overhang containing an EcoR\ site. This construct was inserted into the pET28a vector (Novagen) using the Nde\ and EcoRl restriction sites to produce a (His)6-tagged protein. The resultant plasmid (b3-OST-1-pET28) was sequenced to confirm the reading frame and the lack of mutations within the<br>
coding region (University of North Carolina, DNA sequencing core facility). The plasmid, b3-OST-1-pET28, was transformed into BL21(DE3)RIL cells (Stratagene, La Jolla California, U.S.A) for the expression of 3-OST-1.<br>
Protein expression and purification of 3-OST-1. Cells containing the b3-OST-1 -pET28 were grown in twelve 2.8 L Fernbach flasks containing 1 L of LB media with 50 µg/mL of kanamycin at 37 °C. When the OD600 reached 0.6 to 0.8, the temperature was lowered to 22° C for 15 min. Isopropyl-p-D-thiogalactopyranoside (IPTG) was then added to a final concentration of 200 M-M, and the cells were allowed to shake overnight. Cells were pelleted and resuspended in 120 ml of sonication buffer, 25 mM Tris pH 7.5,500 mM NaCI, and 10 mM imidizole. Cells were disrupted by sonication then spun down. The supernatant was applied to NTA-agarose resin (Qiagen, Valencia, CA, U.S.A.) in batch and washed with sonication buffer. The resin was loaded onto a column and the protein was eluted with an imidizole gradient from 10 mM to 250 mM.<br>
Preparation of 3-OST-3 expression plasmid. The cDNA fragment encoding the catalytic domain of 3-OST-3 (G139-G406) was amplified from plasmid h3-OST-3A-pcDNA3 (Liu, J. et al (1999) J. Biol. Chem 274, 5185-5192) with a 5' overhang containing an A/del site and a 3' overhang containing an EcoR\ site. This construct was inserted into the pET28a vector (Novagen) using the Nde\ and Ecofll restriction sites to produce a (His)6-tagged protein. The resultant plasmid was sequenced to confirm the reading frame and the lack of mutations within the coding region (University of North Carolina, DNA sequencing core facility). The plasmid was transformed into BL21 (DE3)RIL cells (Stratagene) for the expression of 3-OST-3.<br>
Protein expression and purification of3-OST-3. Cells containing the 3-OST-3 were grown in twelve 2.8 L Fernbach flasks containing 1 L of LB media with 50 ng/mL of kanamycin at 37 °C. When the OD6oo reached 0.6 to 0.8, the temperature was lowered to 22° C for 15 min. isopropyl-(3-D-thiogalactopyranoside (IPTG) was then added to a final concentration of 200 µM, and the cells were allowed to shake overnight. Cells were pelleted and resuspended in 120 mL of sonication buffer, 25 mM Tris pH 7.5,500 mM NaCI, and 10 mM imidizole. Cells were disrupted by sonication then spun down. The<br>
supernatant was applied to NTA-agarose resin (Qiagen) in batch and washed with sonication buffer. The resin was loaded onto a column and the protein was eluted with an imidizole gradient from 10 mM to 250 mM.<br>
Preparation of Immobilized HS Sulfotransferases. Dialyzed sulfotransferases (3 ml, 4 mg/ml) in phosphate-buffered saline buffer (3 mM KCI, 1.5 mM KH2PO4,8 mM Na2HPO4, 138 mM NaCI, pH 7.2) were mixed with 1 ml of AMINOLINK® plus beads (Pierce, Rockford, Illinois, U.S.A.) following a protocol from the manufacturer. Immobilized enzyme was washed with 50 mM MES, 1% Triton X-100,1 mM MgCI2, and 1 mM MnCI2, pH 7.0, and stored at 4°C.<br>
Measurement of Enzymatic Activities of Immobilized Proteins. To determine the activity of 3-OST-1 and 3-OST-3, HS from bovine kidney (ICN Biomedicals, Aurora, Ohio, U.S.A.) was used as a substrate; to determine the activity of 6-OST and 2-OST, CDSNS heparin was used as a substrate. Immobilized proteins (100 µl, ~300 ug of immobilized enzyme) with 100 ug of substrate (HS for3-OST and CDSNS heparin for 2-OST or 6-OST) and 200 uM [35S]PAPS (1000 cpm/pmol) in 1 ml of 50 mM MES, pH 7.0,1 % Triton X-100,1 mM MgCI2, and 1 mM MnCI2. After rotating at room temperature for 1 h, the supernatant was collected, and the beads were washed with 3 x 200 ul of 1 M NaCI in 25 mM MOPS (pH 7.0). The supernatant and washes were combined, diluted with 2 ml of water, and subjected to DEAE chromatography to determine the amount of [35S]HS product used to determine the activities of the various HS O-STs.<br>
PAPS Regeneration System. The reactions involved in the PAPS regeneration system are shown in Figure 1B. N-terminal His6-tagged AST-IV was expressed in E. co//and purified as described by Burkat and colleagues (Burkartet al. (2000) J. Org. Chem. 65,5565-5574) at a yield of-50 mg/literof bacterial culture. The full-length cDNA of rat AST-IV was a generous gift of Dr. Michael Duffel (University of Iowa) (Sheng et al. (2004) Drug Metabol. Dispos. 32, 559-565).<br>
Modification of Polysaccharides. All enzymatic modifications, including those catalyzed by 2-OST, 6-OST, 3-OST-1, and 3-OST-3 followed the same protocol. Briefly, 20 mg of purified AST-IV was incubated with 40 uM PAP and<br>
1 mM PNPS in 20 ml of 50 mM MES, pH 7.0,1% Triton X-100,1 mM MgGI2, and 1 mM MnCI2 at 25 °C for 15 min. The reaction mixture was mixed with 4 ml of immobilized sulfotransferase, 2 mg of a polysaccharidesubstrate was added, and the mixture was rotated at 25 °Cfor 24 h. The supernatant was recovered, and the polysaccharide that bound to the beads was eluted by washing three times with 8 ml of 1 M NaCI in 25 mM MOPS. Both the supernatant and wash were combined and the product was precipitated by adding ethanol (3 volumes). After incubating overnight at 4°C, polysaccharide was recovered by centrifugation.<br>
Estimation of the Modification Level. The completion of 2-OST modification was monitored by incubating 10 ug of polysaccharide (Liu and Thorp (2002) Med. Res. Rev. 22, 1-25) with 20 Mg of soluble 2-OST in the presence of 100 µM of [35S]PAPS at 37 °C for 30 min. In a control experiment, 10 ug of compound 1 (Figure 1A) replaced the previously modified polysaccharide in the otherwise identical reaction mixture. By comparing the amount of 35S incorporation, the extent of the original 2-O-sulfation reaction could be estimated. For monitoring the completion of 6-OST, compound 1 or 2 (Figure 1A) was used as a substrate; for 3-OST-1, compound 4 was used (Figure 1A).<br>
Using f5S]PAPS as a Sulfate Donor. Antithrombin (AT)-binding and gD-binding experiments utilized 35S-labeled polysaccharides prepared using [35S]PAPS. In a typical reaction, 2 mg of HS substrate was incubated with 4 ml of beads with immobilized sulfotransferase (~12 mg of immobilized enzyme) at 25 °C in 20 ml of 50 mM MES, pH 7.0, 1% Triton X-100, 1 mM MgCI2, and 1 mM MnCI2, 200 MM [35S]PAPS (1000 cpm/pmol) for 1 h. The resultant polysaccharide was recovered using DEAE chromatography.<br>
Disaccharide Analysis. Synthesized polysaccharides (100 µg) were degraded by a mixture of heparin lyases as previously described (Moon et al. (2204) J. Btol. Chem. 279, 45185-45193) and desalted on BIOGEL® P-2 column (0.5 x 200cm; Bio-Rad Labs) in 0.1 M NH4HCO3. Disaccharides were analyzed by a C18 reversed phase column (0.45 x 25 cm; Vydac, Columbia, Maryland, U.S.A.) with UV 232 detection and identified by coelution with appropriate standards (Chen et al. (2003) Glycobiology 13, 785-794). The<br>
overall recovery yield of the disaccharide analysis was estimated by using 2-O [36S]heparin (100,000 cpm/70 ng; compound 2) as an internal control.<br>
NMR Analysis. Polysaccharide sample (1-2 mg) was dissolved in 0.5 ml of 2HaO (99.9%), freeze dried to remove exchangeable protons, redissolved in 75 ul of 2H2O (100.00%), and transferred to NMR microtubes (Shigemi, Inc., Allison Park, Pennsylvania, U.S.A.). NMR spectra were referenced relative to the HO2H at 4.80 ppm, and in COSY water was suppressed by presaturation of the HO2H resonance.<br>
Determination of the Binding to AT and FGF2 Using Surface Plasmon Resonance Spectroscopy (SPR). Heparin and the synthesized polysaccharides were biotinylated as described (Hernaiz et al. (2000) Biochem. Biophys. Res. Commun. 276, 292-297). A solution of the biotinylated polysaccharides (0.1 mg/ml) in HEPES (10 mM), 0.15 M NaCI, 3 mM EDTA, and 0.005% surfactant P20 (HBS-EP) was flowed over the cells of the streptavidin chip at 5 ul/min. To determine the Kb values, different concentrations (25-1600 nM in HBS-EP buffer) of FGF2 and AT were individually injected at 30 ul/min (running buffer: HBS-EP). Kinetic injection was used, leading the protein to flow for 180 seconds (s) and dissociated for the next 180 s, and the surface was regenerated by a 60 s injection of 30 ul of 2 M NaCI. Response units were monitored as a function of time to afford sensorgrams. The SPR curves for FGF2 fit well at individual concentrations, but the global fit suggested significant binding heterogeneity. Thus, the equilibrium response unit RU(eq) values from the sensorgrams of FGF2 binding to polysaccharide-containing surfaces were used to construct Scatchard plots, RU(eq)/C versus RU(eq), where Cis the free protein concentration, resulting in linear, first degree polynomial functions confirming the one-to-one binding of FGF-polysaccharide and to estimate binding affinity. A two-state reaction model was applied to the AT-polysaccharide interactions measured by SPR using curve fitting to estimate the association and dissociation rate constants and affinity constant.<br>
FGF2/FGFR1C-mediated Proliferation Assay. The BaF3 cells ectopically expressing FGFR1C have been previously described (Ornitz et al. (1996) J. Biol. Chem. 271,15292-15297). The BaF3-FGFR1c cells were maintained in<br>
RPMI 1640 medium (Sigma Chemical Co., St. Louis, Missouri, U.S.A.) supplemented with 10% fetal bovine serum, 0.5 ng/ml interleukin (IL)-3 (PeproTech Inc., Rocky Hill, New Jersey, U.S.A.), 2 mM L-glutamine, penicillin (50 lU/ml) and streptomycin (50 µg/ml), and 50 uM ß-mercaptoethanol. For mitogenic assays, BaF3 FGFRIc cells were washed three times with RPMI 1640 medium to remove IL-3 and resuspended in the growth medium lacking IL-3. About 30,000 cells were plated per well in a 96-well plate in medium containing 1 µg/m\ heparin, compound 1, 2, 3, 4a, or 4b, and 2 nM FGF-2 (PeproTech) in a total volume of 200 ul. The cells were then incubated at 37 °C for 40 h. To each well, an additional 50 ul of growth medium containing 1 µCi of [3H]thymidine was added. Cells were harvested after 4-5 h by filtration through glass fiber paper. The incorporation of [3H]thymidine into the DNA was determined by scintillation counting.<br>
Determination of the Binding of Compounds 5-8 to A T. Approximately 1 x 105 cpm of 35S-labeled compound was incubated with 5 µg of human AT (Cutter Biological, Clayton, North Carolina, U.S.A.) in 50 ul of binding buffer containing 10 mM Tris-HCI, pH 7.5,150 mM NaCI, 1 mM Mn2+, 1 mM Mg2*, 1 mM Ca2*, 10 uM dextran sulfate, 0.0004% Triton X-100, and 0.02% sodium azide for 30 min at room temperature. Concanavalin A-Sepharose (Sigma; 50 ul of 1:1 slurry) was then added, and the reaction was shaken at room temperature for 1 h. The beads were then washed by 3 x 1 ml of binding buffer, and the bound polysaccharide was eluted with 1 M NaCI.<br>
Inhibition Effect of the Polysaccharides on the Activities of Factor Xa and Thrombin. Assays were based on two previous methods (Zhang et al. (2001) J. Biol. Chem. 276, 42311-42321; Duncan et al. (2004) Biochim. Biophys. Acta 1671,34-43). Briefly, factor Xa (Enzyme Research Laboratories, South Bend, Indiana, U.S.A.) and thrombin (Sigma) were diluted to 20 and 8 units/ml with phosphate-buffered saline containing 1 mg/ml bovine serum albumin, respectively. AT was diluted with phosphate-buffered saline containing 1 mg/ml bovine serum albumin to give a stock solution at the concentration of 27 uM. The chromogenic substrates, S-2765 (for factor Xa assay) and S-2238 (for thrombin assay) were from Diapharma (West Chester, Ohio, U.S.A.) and made up at 1 mM with 1 mg/ml POLYBRENE® (Sigma) in water. The synthesized<br>
polysaccharide (compounds 5,7, and 8; Figure 1 A) or heparin was dissolved in a buffer containing 50 mM Tris-HCI, pH 8.4,7.5 mM Na2EDTA, 175 mM NaCI at various concentrations (1-10,000 ng/ml). The reaction mixture, which comprised 25 ul of AT stock solution and 25 ul of the solution containing polysaccharide, was incubated at37 °C for 2 min. Factor Xa (25 ul) orthrombin (25 ul) was added. After incubating 37 °C for 4 min, 25 ul of S-2765 or S-2238 was added. The absorbance of the reaction mixture was measured at 405 nm continuously for 10 min. The absorbance values were plotted against the reaction time. The initial reaction rates as a function of concentration were used to calculate the IC50 values. The concentrations of the synthesized polysaccharides were determined using Alcian blue as described by Bjornsson (Bjornsson (1993) Anal. Biochem. 210,282-291) and quantitative disaccharide analysis as described above.<br>
The Binding to Herpes Simplex Virus gD. The assay for determining binding of 35S-labeled polysaccharides (compounds 5 and 6; Figure 1A) togD was carried out by an immunoprecipitation procedure using anti-gD monoclonal antibody (DL6) (Shukla et al. (1999) C0//99, 13-22).<br>
EXAMPLE 1 DEVELOPMENT OF AN EFFICIENT ENZYMATIC SYNTHESIS OF<br>
SULFATED POLYSACCHARIDES<br>
Expression ofHS Sulfotransferases in E. coli. The presently disclosed subject matter provides methods for synthesizing biologically active HS, such as for example FGF-binding HS (e.g., compound 4, including 4a and 4b), AT-binding HS (e.g., compound 5), and herpes simplex virus gD-binding HS (e.g., compounds). See Figure 1A. Four enzymes, including 2-OST, 6-OST, 3-OST-1, and 3-OST-3, were utilized for the syntheses of these particular targets. Bacterial expressed 3-OST-1 and 3-OST-3 can exhibit substrate specificity and specific enzymatic activity comparable with those of their counterparts expressed in insect cells (Moon et al. (2004) J. Blol. Chem. 279,45185-45193; Edavettal et al. (2004) J. Biol. Chem. 279, 25789-25797). Expression of the catalytic domains of 2-OST and 6-OST was also achieved in relatively high yield by preparing a fusion protein with MBP and 2-OST or 6-OST in Rosetta-gami B<br><br>
cells. Because 2-OST and 6-OST fusion proteins were enzymatically active and highly soluble, the MBP domain was retained.<br>
Immobilized Enzymes Are Reusable. 2-OST, 6-OST, 3-OST-1, and 3-OST-3 were immobilized on agarose to be reusable and to enhance the thermal stability. Immobilized enzymes were incubated with polysaccharide substrate and PAPS for 1 h at room temperature. The sulfated polysaccharide product was separated from the immobilized enzyme by washing the beads with 1 M NaCI followed by centrifugation, making the immobilized enzymes ready for the next synthetic cycle. The cycle was repeated 10 times, after which each of the immobilized enzymes were assayed and showed &gt;80% of their catalytic activity (Figure 2). It was also determined that the immobilized enzymes also maintained &gt;65% of their catalytic activity after 2 months of storage at 8°C.<br>
Introduction of PAPS Regeneration System. PAP inhibits sulfotransferase-catalyzed reactions. A PAPS regeneration system can be used to convert PAP to PAPS by relying on AST-IV to catalyze the transfer of the sulfo group from PN PS to PAP, as illustrated in Figure 1B. The presently disclosed subject matter provides for the use of the PAPS regeneration system coupled with O-sulfotransferases. As disclosed herein, the PAPS regeneration system performed very well with 2-OST, 6-OST, and 3-OST-1. Complete modification of the substrate could be demonstrated by the low susceptibility of polysaccharide product to undergo additional sulfation using [35S]PAPS with soluble enzymes, as disclosed herein above. Under the standard conditions, 2-OST, 6-OST, and 3-OST-1 afforded 98, 97, and 98% complete modification, respectively, using the PAPS regeneration system. These results demonstrate that the PAPS regeneration system functioned properly, providing sufficient PAPS for the complete sulfotransferase-catalyzed modification of polysaccharide substrates. This conclusion was further supported by characterizing the structures of the newly synthesized polysaccharide products 2,3,4a, 4b, and 5 (described herein below; see also Figure 1 A). In a parallel experiment, [36S]PAPS was used as the sulfo donor to sulfate the polysaccharides in order to estimate the number of sulfo groups that were incorporated into the product. Six 2-O-sulfo and 6-O-suIfo groups and three3-<br>
O-sulfo groups were estimated to be transferred to one polysaccharide molecule, respectively, assuming that the length of the polysaccharide is 25 disaccharide units. Approximately 1.5 mg of anticoagulant HS (5) was synthesized from 10 mg of CDSNS heparin (1) using immobilized sulfotransferases and the PAPS regeneration system.<br>
EXAMPLE 2 STRUCTURAL CHARACTERIZATION OF SYNTHESIZED<br>
POLYSACCHARIDES<br>
Disaccharide Analysis of the Polysaccharides. Synthesized polysaccharide intermediates were digested with a mixture of heparin lyases, and the resulting disaccharides were analyzed using RPIP-HPLC (Figure 3). As expected, analysis of compound 1 showed a disaccharide component of the structure of AUA-GlcNS (disaccharide 2), residual unsulfated disaccharide, A UA-GlcNAc (disaccharide 1), and small amounts of sulfated disaccharides (disaccharides 3-5) due to incomplete chemical desulfation (Figure 3A and Table 2). The modification by 2-OST elevated the level of AUA2S-GlcNS (disaccharide 4) by about 5-fold (Figure 3B and Table 2), confirming the<br>
#<br>
structure of compound 2. Analysis of compound 3 afforded a 7-fold increase in the level of the disaccharide AUA-GlcNS6S (disaccharide 3) compared with compound 1, consistent with 6-OST-catalyzed modification (Figure 3C and Table 2). The level of trisulfodisaccharide, AUA2S-GlcNS6S (disaccharide 5), in compounds 4a and 4b was increased by about 10-fold compared with that of compound 1 (Figures 3D and 3E, and Table 2). The results from the disaccharide analysis clearly establish that the expected enzymatic modifications took place at each step. It is interesting to note that 6-O-sulfation occurs at A/-sulfoglucosamine, consistent with the substrate specificity of 6-OST In vitro (Smeds et al. (2003) Biochem. J. 372, 371-380). The 2-Osulfation predominantly occurs at the uronic acid with an A/-sulfoglucosamine residue at the reducing end. It is also noted that substantial amounts of AUA-GlcNS (disaccharide 2) remain in compounds 4a and 4b. This observation is not unexpected, since the HS from various tissues also afford a AUA-GlcNS<br>
aisaccnaride unit, suggesting that the structures of compounds 4a and 4b are similar to HS from natural sources (Ledin et al. (2004) J. Biol. Chem. 279, 42732-42741). <br>
Table 2 Summarv of the disaccharide compositions of the synthetic polvsaccharides<br>
(Tabble Removed)<br>
Each synthesized polysaccharide (100 ug) was digested with a mixture of heparin lyases. The resultant disaccharides were purified by a BioGel P-2 column and resolved by RPIP-HPLC.<br>
NMR Analysis of the Polysaccharides. The 1H NMR analysis was conducted on the synthesized polysaccharides to allow assessment of their structures at the polymer levels as well as to estimate the composition. The assignment of each non-exchangeable proton was made using COSY, and their chemical shifts are reported in Table 3.<br>
(Tabble Removed)Table 3 1H chemical shift data fin ppm) for the synthesized polvsaccharides<br>
The integration of the A/-acetyl signal in the 1H NMR of heparin was compared with the integration of the GlcNS-H1 signal, allowing estimation of the level of modifications. The ratio of these integrals showed that ~15% of the glucosamine residues contained A/-acetyl groups. Using this information, the incorporation of 2-O-, 3-O-, and 6-O-sulfo groups was examined. Comparison of the A/-acetyl peak with ldoUA2S-H2 in compound 2 demonstrates incorporation of a 2-Osulfo group into 35% of the IdoUA residues. Based on this incorporation into compound 2, the integral of GlcNS-H6aand ldoUA2S-H2 (overlapping signal) was compared with the integral of the A/-acetyl methyl group in compound 4a to estimate the level of incorporation of the 6-Osulfo group. Integration showed that the 6-O-sulfo group was incorporated into 25% of the GlcNS residues. The reduced level of incorporation is not surprising, since only the 6-OST-1, and not 6-OST-2 and -3, was used in this synthesis. In compound 5, the incorporation of the 3-O-sulfo group was calculated by comparing the integral of the A/-acetyl methyl group with GlcN-H3 and IdoUA-H4 (overlapping signal). Based on these calculations, the 3-O-sulfo group was incorporated into 32% of the GlcNS6S residues. According to 1H NMR and COSY experiments (Table 3), the structures of the compounds 4a, 4b, and 5 were all found to be similar to heparin and contained no unusual signals. Compound 5 showed a slightly greater heterogeneity, as evidenced by additional minor signals corresponding to additional 3-O-sulfo group-containing sequences in the 1H NMR, when compared with heparin. This is not unexpected, since heparin contains a lower level of 3-Osulfo groups/chain than the content of 3-O-sulfo groups/chain observed in compound 5 as determined by 35S incorporation.<br>
The composition estimated by NMR is consistent with the results of disaccharide analysis. NMR analysis suggests that about 15% of the glucosamine unit is /V-acetylated, which is similar to the results of disaccharide analysis for AUA-GlcNAc (14-22%; Table 2). NMR analysis demonstrates that compound 2 comprises 35% of ldoUA2S-GlcNS, and compound 4a comprises 25% of ldoUA2S-GlcNS6S, and disaccharide analysis demonstrates that compounds 2 and 4a comprise 40% of AUA2S-GlcNS and 30% of AUA2S-<br>
GlcNS6S, respectively.  Because of the signal overlap, the degree of 6-O-sulfation in compounds 3 and4b using 1H NMR was not calculated.<br>
EXAMPLE 3<br>
DETERMINATION OF THE BIOLOGICAL ACTIVITIES OF THE SYNTHESIZED POLYSACCHARIDES<br>
The Binding of the Polysaccharides to AT and FGF2. Characterization of the affinities of AT to heparin and enzymatically modified heparin derivatives were performed by SPR. A two-state reaction model was applied to the SPR study of AT and polysaccharide interactions using BIAevaluation™ Software (Biacore Life Sciences, Uppsala, Sweden) for curve fitting analysis. None of the derivatives with the exception of the 2,6,3-O-sulfopolysaccharide (5) and heparin had high affinity to AT. The binding constant (KD) for the binding of compound 5 to AT was determined to be 170 nM, which is very similar to that of heparin (75 nM). The binding affinity of FGF2 to the synthesized polysaccharides was also estimated. Compound 1 showed no interaction with FGF2, whereas compounds 4a and 4b showed the identical binding affinity to FGF2 at a Kd of 35 nM, which is similar to heparin (22 nM) and is consistent with the value reported in the literature (Ibrahimi et al. (2004) Biochemistry43, 4724-4730).<br>
Synthetic Polysaccharides Promote Cell Proliferation. The BaF3 FGFR1 c cells normally depend on IL-3 for growth. In the absence of IL-3, the cell proliferation depends on the addition of both FGF and heparin or HS (Ornitz et al. (1996) J. Biol. Chem. 271,15292-15297). The activity of compounds 1,2, 3, 4a, and 4b and heparin in promoting cell mitogenesis was measured using the FGF-2/FGFR1 c system in BaF3 cells as described herein above. The cells receiving compounds 4a and 4b (at 1 ug/ml) showed an increase in [3H]thymidine incorporation, which was about 60 and 40% of that of heparin, respectively, suggesting that the combinations of 2-O- and 6-Osulfations confer the activity in promoting cell proliferation (Figure 4A). The activity of 4a and 4b was also compared with that of heparin at different concentrations (Figure 4B). It was found that compound 3 had moderate activity in promoting cell proliferation, whereas compounds 1 and 2 did not exhibit activity. The<br><br>
results are consistent with previously reported data on the contributions of the sulfo groups of HS to the mitogenic activity (Guimond and Turnbull (1999) Curr. Biol. 9, 1343-1346; Pye et al. (1998) J. Biol. Chem. 273, 22936-22942). It is noted that 4a appears to have stronger activity in promoting cell proliferation than 4b, suggesting that different sequences are generated by the different order of 2-O- and .6-O-sulfation as suggested by Jacobsson and Lindahl (Jacobsson and Lindahl (1980) J. Biol. Chem. 255, 5094-5100). However, such sequence difference could not be detected by the disaccharide or NMR analysis.<br>
The Anticoagulant Activity of the Synthesized Polysaccharides. Heparin achieves its anticoagulant activity by forming a 1:1 complex with AT, which inhibits the activities of factor Xa and thrombin (Rosenberg et al. (1997) J. Clin. Invest 99, 2062-2070). Because it is known that the introduction of the 3-O-sulfo group by 3-OST-1 is essential for the synthesis of anticoagulant HS, different types of 3-O-sulfo group-containing polysaccharides, compounds 5,7, and 8, were prepared and their activities in inhibiting factor Xa and thrombin tested (Table 4). As expected, heparin is a potent activator for AT-mediated inhibition of factor Xa and thrombin, whereas ARIXTRA® (GlaxoSmithKline) specifically activates the AT-mediated inhibition of factor Xa (Petitou and van Boeckel (2004) Angew. Chem. Int. Ed. 43,3118-3133). Compound 5 has very similar potency to heparin, inhibiting the activities of both factor Xa and thrombin, suggesting that the presently disclosed enzyme-based approach is indeed capable of synthesizing the anticoagulant polysaccharide. It has been reported that the presence of 2-O-sulfo groups is not essential for HS binding to AT and its resulting anticoagulant activity (Zhang et al. (2001) J. Biol. Chem. 276,28806-28813). Indeed, polysaccharide intermediate compound 8 lacks 2-O-sulfo groups but still exhibits anticoagulant activity, consistent with this previous report (Zhang et al. (2001) J. Biol. Chem. 276, 28806-28813). In contrast, compound 7 lacks 6-Osulfo groups and, thus, has no anticoagulant activity, since 6-Osulfo groups are critical in AT binding (Atha et al. (1985) Biochemistry24,6723-6729). Another3-O-sulfated polysaccharide (compound 6) was also prepared to test for its anti-Xa and antithrombin activities. It is important to note that both compounds 5 and 6 carry a3-O-sulfoglucosamine<br><br>
unit, although it is located in different saccharide sequences (Figure 1A). It is known that 3-OST-3-modified HS does not bind to AT (Liu et al. (1999) J. Biol. Chem. 274,5185-5192). As expected, compound 6 does not exhibit any anti-Xa and antithrombin activities. The binding of AT to the synthesized compounds was also measured (Table 4). It is clear that the anticoagulant activities of the compounds correlated to their binding affinity to AT. Taken together, these results demonstrate that the anticoagulant activities of these enzymatically synthesized polysaccharides are consistent with the known structure activity relationship of HS.<br>
Table 4<br>
Anti-factor Xa and antithrombin activities of synthesized polvsaccharide<br>
intermediates<br><br>
(Tabble Removed)Preparation of the Polysaccharide That Binds to Herpes Simplex Virus gD. Herpes simplex virus utilizes HS as a receptor to infect the target cells (Shukla and Spear (2001) J. Clin. Invest. 108,503-510). A specific 3-O-sulfo group-containing HS, generated by 3-OST-3, -5, or -6, serves as an entry receptorfor herpes simplex virus type 1 through HS binding of gD (Shukla et al. (1999) Ce//99,13-22; Xia et al. (2002) J. Biol. Chem. 277,37912-37919; Xu et al. (2005) Biochem. J. 385, 451-459). Structural characterization of the gD-binding site revealed a unique octasaccharide sequence carrying a 3-O-sulfo glucosamine residue (Liu et al. (2002) J. Biol. Chem. 277, 33456-33467). To test if the presently disclosed enzymatic approach synthesizes gD-binding HS, compound 4 (Figure 1 A) was incubated with immobilized 3-OST-3 to generate compound 6 (Figure 1 A). It was found that 12% of the resultant polysaccharide<br>
(compound 6) bound to gD, whereas only 2.6% of compound 5, a 3-OST-1-modified polysaccharide, bound to gD as determined by immunoprecipitatlon (Table 5). Comparison with the appropriate controls showed that the percentage binding for 3-OST-3-modified HS closely resembled that of compound 6. In conclusion, the results suggest that the presently disclosed enzymatic approach is capable of effectively synthesizing gD-specific binding HS. It should be noted that commercial heparin does not bind to gD, and heparin is not a substrate for 3-OST-3 (Nicola et al. (1996) J. Virol. 70, 3815-3822; Liu et al. (1999) J. Biol. Chem. 274, 38155-38162). Therefore, these results also demonstrate that It is possible to redesign the sulfation patterns by first solvolytically removing all O-sulfo groups and then selectively enzymatically replacing sulfo groups required for specific interactions.<br>
Table 5 The binding of synthesized polvsaccharides to aD<br><br>
(Tabble Removed)<br>
DISCUSSION OF EXAMPLES 1-3<br>
HS, including heparin, has a wide range of biological activities, including anticoagulation, antiviral, and anticancer activities. Sulfo group-containing saccharide sequences dominate the specificity of the functions of heparin and HS. Thus, the synthesis of a polysaccharide with the appropriate positioning of these functional groups to carry out its unique biological activity is desirable. The presently disclosed subject matter provides an approach to synthesize<br>
sulfo group-containing HS polysaccharides that have desired biological activities, including for example HS that binds to FGF2, herpes simplex virus gD, or AT. These HS polysaccharides can also demonstrate appropriate biological activity, such as anticoagulant activity mediated through AT binding and the activity in promoting cell proliferation. More importantly, the presently disclosed subject matter permits the synthesis of greater than 1 mg amounts of specific sulfo group-containing polysaccharides, sufficient for testing their activities in biochemical and biological assays. The quantities synthesized are also sufficient for extensive structural analysis, including disaccharide analysis and one- and two-dimensional NMR analysis.<br>
In examples 1-3. two approaches were used to increase the scale of enzymatic synthesis. First, HS sulfotransferases were expressed in E. coli, readily affording 20-50 mg of purified enzymes. It has been shown that both 3-OST-1 and 3-OST-3 can be expressed in E. coli in relatively high yield. The presently disclosed subject matter also provides for the expression in E. co//of both 2-OST and 6-OST as the soluble 2-OST and 6-OST-MBP fusion proteins. Second, efficiency of enzymatic sulfation was improved by utilizing immobilized O-sulfotransferases and coupling their use to a PAPS regeneration system.<br>
When a PAPS regeneration system was previously applied for preparing /V-sulfoheparosanfrom heparosan using A/-deacetylase//V-sulfotransferase, the N-sulfation yield was lower than that obtained using added exogenous PAPS (Saribas etal. (2004) Glycobiology\4,1217-1228). Nevertheless, coupling of a PAPS regeneration system to enzymatic sulfation of polysaccharides, as disclosed herein, was considered a desirable goal for several reasons. First, PAP inhibits HS O-sulfotransferase activities with ICso values of ~100 uM under the reaction conditions used in the presently disclosed syntheses, making milligram-scale synthesis difficult without continuously removing PAP. Second, a PAPS regeneration system would permit use of PNPS as the sulfo donor and require only catalytic amounts of PAP, significantly reducing the cost of synthesis. The listed price of PNPS in the Aldrich catalog is about 300-fold less than PAPS. The immobilized enzyme format facilitates use of the enzymes repeatedly, making the method amendable to a large scale synthesis, since scale-up of immobilized enzyme columns is generally known in the art.<br>
Developing an effective approach to synthesize HS is also desirable for understanding the mechanism of its biosynthesis. Unlike protein biosynthesis, polysaccharide biosynthesis is not a template-driven process. Although the cDNAs encoding the HS biosynthetic enzymes have been cloned, the genetic regulation mechanism for the synthesis of the HS with defined biological functions is not fully understood. The unique and often remote sequence features in substrate can influence the action of HS O-sulfotransferases. Furthermore, whether additional factors or the formation of complexes of biosynthetic enzymes takes part in controlling the structure of HS remains to be elucidated. It is interesting to note that a complex of HS epimerase and 2-OST In vivo has been reported (Pinhal et al. (2001) Proc. Natl. Acad. Sci. U.S.A. 98, 12984-12989). The presently disclosed approach provides a tool to answer these questions, since it is capable of generating a sufficient amount of HS product required for extensive structural biochemical and biological analysis.<br>
HS is believed to be present in block structures, comprising highly sulfated and non-sulfated domains (Gallagher, J. T. (2001) J. Clin. Invest. 108, 357-361), and the HS with the biological activities largely contain the highly sulfated domains. Without wishing to be limited by theory, it is believed that the modifications in the present system indeed happen in a block fashion based on the following facts. First, the results of the disaccharide analysis concluded that both 6-O-sulfation and 2-O-sulfation are carried out predominantly in the sulfated region (Figure 3 and Table 2). Second, the enzymatically-modified products exhibit the anticipated biological functions, including the activation of FGF/FGF receptor signaling (compounds 4a and 4b), carrying anticoagulant activity (compound 5), and binding to herpes simplex virus glycoprotein D (compound 6). Because the HS carrying these functions must contain the domain structures with a size larger than pentasaccharide (Petitou and van Boeckel (2004) Angew. Chem. Int. Ed. 43, 3118-3133; Atha et al. (1985) Biochemistry 24,6723-6729; Liu etal. (2002) J. Biol. Chem. 277,33456-33467; Maccarana et al. (1993) J. Biol. Chem. 268, 23898-23905), the synthetic products with the desired functions suggest that the modifications indeed occur in a block fashion.<br>
In summary, the presently disclosed subject matter provides a method<br>
for enzymatic sulfation and preparation of HS with distinct biological activities. Unique sulfated saccharide sequences play a dominant role in the function and specificity of HS/heparin. The presently disclosed methods demonstrate the capability of using a collection of HS biosynthetic enzymes to synthesize HS/heparin with selected biological activities. The synthetic scale with this method can be easily increased for large scale synthesis, provided that both the enzymes and the sulfo donor are easily accessible. The current method clearly demonstrates that HS/heparin having specific biological activities can be synthesized by subjecting a backbone saccharide polymer to different enzymatic modifications. This enzymatic selectivity is currently not accessible by chemical sulfation approaches. The presently disclosed method can significantly aid the exploration of new potential therapeutic applications for HS. In addition, enzymatic synthesis of anticoagulant heparin can potentially lead to a better anticoagulant drug by reducing its side effects.<br>
REFERENCES<br>
The references listed below as well as all references cited in the specification are incorporated herein by reference to the extent that they supplement, explain, provide a background for or teach methodology, techniques and/or compositions employed herein. Aikawa et al., (1999) J. Biol. Chem. 274, 2690.     . Aikawa, J.-l., et al., (2001) J. Biol. Chem. 276, 5876-5882. Alexander, C. M., et al., (2000) Nat. Genet. 25, 329-332. Altschul et al. (1990) J Mol Biol 215, 403-410. Atha, D. H., Lormeau, J.-C., Petitou, M., Rosenberg, R. D., and Choay, J.<br>
(1985) Biochemistry 24, 6723-6729. Avci, F. Y., Karst, N. A., and Linhardt, R. J. (2003) Curr. Pharm. Des. 9,2323-<br>
2335. Balagurunathan, K., Beeler, D. L, Lech, M., Wu, Z. L, and Rosenberg, R. D.<br>
(2003) J. Biol. Chem. 278, 52613-52621. Balagurunathan, K., Lech, M. Z., Beeler, D. L., Wu, Z. L., and Rosenberg, R.<br>
D. (2003) Nat. Biotechnol. 21, 1343-1346. Bernfield, M., etal., (1999) Annu. Rev. Biochem. 68,729-777.<br>
Bjornsson, S. (1993) Anal. Biochem. 210, 282-291.<br>
Bowman et al., (1999) Chem. Biol. 6, R9-R22.<br>
Burkart, M. D., Izumi, M., Chapman, E., Lin, C., and Wong, C. (2000) J. Org.<br>
Chem. 65, 5565-5574.<br>
Capila, I., and Linhardt, R. J. (2002) Angew. Chem. Int. Ed. 41, 390-412. Carfi, A., et al., (2001) Mol. Cell8:169-179. Chen et al., (1992) Protein Expression Purif. 3,421-6. Chen, J., Duncan, M. B., Carrick, K., Pope, M., and Liu, J. (2003) Glycobiology<br>
13,785-794. Dementiev, A., Petitou, M., Herbert, J.-M., and Gettins, P. G. (2004) Nat.<br>
Struct. Biol. 11,867-863. Dooley (1998) Chemico-Biol. Interact. 109, 29. Duncan, M. B., Chen, J., Krise, J. P., and Liu, J. (2004) Biochim. Biophys. Acta<br>
1671,34-43. Edavettal, S. C., Lee, K. A., Negishi, M., Linhardt, R. J., Liu, J., and Pedersen,<br>
L. C. (2004) J. Biol. Chem. 279, 25789-25797. Esko, J. D., and Lindahl, U. (2001) J. Clin. Invest. 108:169-173. Esko, J. D., and Selleck, S. B. (2002) Annu. Rev. Biochem. 71,435-471. Falany (1997) EASES J. 11,1 -2. Fukuta et al. (1998) Biochim. Biophys. Act. 1399, 57. Gallagher, J. T. (2001) J. Clin. Invest. 108, 357-361. Gribskov et al. (1986) Nuc Acids Res 14(1), 327-334. Guimond, S. E., and Turnbull, J. E. (1999) Curr. Biol. 9,1343-1346. Quo et al. (1994) Chem.-Biol. Interact. 92, 25-31. Habuchi et al., (1998) J. Biol. Chem. 273, 9208. Habuchi, H., et al., (2000) J. Biol. Chem. 275, 2859-2868. Hernaiz, M., Liu, J., Rosenberg, R. D., and Linhardt, R. J. (2000) Biochem.<br>
Biophys. Res. Commun. 276,292-297. Holmborn, K., Ledin, J., Smeds, E., Eriksson, I., Kusche-Gullberg, M., and<br>
Kjellen, L. (2004) J. Biol. Chem. 279, 42355-42358. Ibrahim!, O. A., Zhang, F., Lang, S., Hrstka, C., Mohammad!, M., and Linhardt,<br>
R. J. (2004) Biochemistry 43, 4724-4730. Jacobsson, I., and Lindahl, U. (1980) J. Biol. Chem. 255,5094-5100.<br>
Krummenacher, C., et al., (1999) J. Virol. 73, 8127-8137.<br>
Kuberan, B., Beeler, D. L., Lawrence, R., Lech, M., and Rosenberg, R. (2003)<br>
J. Am. Chem. Soc. 125,12424-12425. Kyte et al. (1982) J Mol Biol 157, 105. Ledin, J., Staatz, W., Li, J.-P., Gotte, M., Selleck, S. B., Kjellen, L., and<br>
Spillmann, D. (2004) J. Biol. Chem. 279, 42732-42741. Lin et al., (1995) J. Am. Chem. Soc. 117, 8031. Lin et al. (1998) Anal. Biochem. 264,111-117. Lindahl, U., et al., (1998) J. Biol. Chem. 273, 24979-24982. Lindahl, U., Li, J., Kusche-Gullberg, M., Salmivirta, M., Alaranta, S., Veromaa,<br>
T., Ernies, J., Roberts, I., Taylor, C., Oreste, P., Zoppetti, G., Naggi, A.,<br>
Torri, G., and Casu, B. (2005) J. Med. Chem. 48, 349-352. Linhardt, R. J. (2003) J. Med. Chem. 46,2551-2564. Liu, J., and Thorp, S. C. (2002) Med. Res. Rev. 22,1-25. Liu, J., et al., (1996) J. Biol. Chem. 271, 27072-27082. Liu, J., et al., (1999) J. Biol. Chem. 274, 38155-38162. Liu, J., et al., (1999) J. Biol. Chem. 274, 5185-5192. Liu, J., Shworak, N. W., Sinay, P., Schwartz, J. J., Zhang, L., Fritze, L M. S.,<br>
and Rosenberg, R. D. (1999) J. Biol. Chem. 274, 5185-5192. Liu, J., Shriver, Z., Blaiklock, P., Yoshida, K., Sasisekharan, R., and<br>
Rosenberg, R. D. (1999) J. Biol. Chem. 274, 38155-38162. Liu, J., Shriver, Z., Pope, R. M., Thorp, S. C., Duncan, M. B., Copeland, R. J.,<br>
t<br>
Raska, C. S., Yoshida, K., Eisenberg, R. J., Cohen, G., Linhardt, R. J.,<br>
and Sasisekharan, R. (2002) J. Biol. Chem. 277, 33456-33467. Liu, J., et al., (2002) J. Biol. Chem. 277, 33456-33467. Maccarana, M., Casu, B., and Lindahl, U. (1993) J. Biol. Chem. 268, 23898-<br>
23905.<br>
Marcus et al. (1980) Aial. Biochem. 107, 296. Marshall et al., (1997) J. Biol. Chem. 272, 9153-9160. Marshall et al., (1998) Chem. -Biol. Interact. 109,107-116. Mazany et al., (1998) Biochim. Biophys. Act 1407,92. Moon, A., Edavettal, S. C., Krahn, J. X., Munoz, E. M., Negishi, M., Linhardt, R.<br>
J., Liu, J., and Pedersen, L. C. (2004) J. Biol. Chem. 279,45185-45193.<br>
Nastuk et al. (1998) J. Neuroscience 18,7167.<br>
Needleman et al. (1970) J Mol Biol 48,443.<br>
Nicola, A. V., Willis, S. H., Naidoo, N. N., Eisenberg, R. J., and Cohen, G. H.<br>
(1996) J. Virol. 70, 3815-3822. Ong et al., (1998) J. Biol. Chem. 273, 5190. Ornitz, D. M., Xu, J., Colvin, J. S., McEwen, D. G., MacArthur, C. A., Coulier,<br>
F., Gao, G., and Goldfarb, M. (1996) J. Biol. Chem. 271,15292-15297. Ouyang et al., (1998) J. Biol. Chem. 273, 24770. Ozawa et al., (1990) Nucleic Acids Res. 18, 4001 z. Petitou, M., Herault, L.-P., Bernat, A., Driguez, P.-A., Duchaussoy, P.,<br>
Lormeau, J.-C., and Herbert, J.-M. (1999) Nature 398,417-422. Petitou, M., and van Boeckel, C. A. A. (2004) Angew. Chem. Int. Ed. 43,3118-<br>
3133. Pinhal, M. A., Smith, B., Olson, S., Aikawa, J., Kimata, K., and Esko, J. D.<br>
(2001)	Proc. Natl. Acad. Sci. U. S. A. 98,12984-12989.<br>
Pye, D. A., Vives, R. R., Turnbull, J. E., Hyde, P., and Gallagher, J. T. (1998) J.<br>
Biol. Chem. 273, 22936-22942. Reizes, O., etal., (2001) Cell 106:105-116. Rosenberg, R. D., et al., (1997) J. Clin. Invest 99, 2062-2070. Saeki et al. (1998) J. Biochem. 124, 55. Sambrook et al. (2001) Molecular Cloning: A Laboratory Manual. Third Edition,<br>
Cold Spring Harbor Laboratory Press, Cold Spring Harbor, New York. Saribas, A. S., Mobasseri, A., Pristatsky, P., Chen, X., Barthelson, R., Hakes,<br>
D., and Wang, J. (2004) Glycobiology 14,1217-1228. Sasisekharan, R., Shriver, Z., Venkataraman, G., and Narayanasami, U.<br>
(2002)	Nat. Rev. Cancer 2, 521-528.<br>
Schwartz et al. (1979) Nuc Acids Res 6(2), 745-755.<br>
Sheng, J. J., Saxena, A., and Duffel, M. W. (2004) DrugMetabol. Dispos. 32,<br>
559-565.<br>
Shukla, D., etal., (1999) Ce//99,13-22. Shukla, D., and Spear, P. G. (2001) J. Clin. Invest. 108, 503-510. Shworak, N. W., et al., (1997) J. Biol. Chem. 272, 28008-28019. Smeds, E., Habuchi, H., Do, A.-T., Hjertson, E., Grundberg, H., Kimata, K.,<br><br>
Lindahl, U., and Kusche-Gullberg, M. (2003) Biochem. J. 372,371-380. Smith et al. (1981) Adv Appl Math 2,482. Thompson et al. (1994) Nucleic Acids Res 22(22), 4673-4680. U.S. Patent No. 6,255,088. Uchimura et al. (1998) J. Biol. Chem. 273, 22577. Yoshinari et al., (1998) J. Biochem. 123, 740. Wethmur &amp; Davidson (1968) J Mo/0/0/31, 349-370. Willis, S. H., et al., (1998) J. Virol. 72, 5938-5947. WuDunn, D., and Spear, P. G. (1989) J. Virol. 63, 52-58. Xia, G., Chen, J., Tiwari, V., Ju, W., Li, J.-P., MalmstrOm, A., Shukia, D., and<br>
Liu, J. (2002) J. Biol. Chem. 277, 37912-37919. Xu, D., Tiwari, V., Xia, G., Clement, C., Shukia, D., and Liu, J. (2005) Biochem.<br>
J. 385, 451-459.<br>
Yang et al. (1996) Protein Expression Purif. 8, 423-429. Yang et al., (1997) Prote/n Eng. 10, 70. Yang etal., (1998) Chem. -Biol. Interact. 109,129-135. Yoshinari et al., (1998) J. Biochem. 123, 740. Zhang, L., Beeler, D. L., Lawrence, R., Lech, M., Liu, J., Davis, J. C., Shriver,<br>
Z., Sasisekharan,, R., and Rosenberg, R. D. (2001) J. Biol. Chem. 276,<br>
42311-42321. Zhang, L., Lawrence, R., Schwartz, J. J., Bai, X., Wei, G., Esko, J. D., and<br>
Rosenberg, R. D. (2001) J. Biol. Chem. 276, 28806-28813.<br>
It will be understood that various details of the presently disclosed subject matter can be changed without departing from the scope of the presently disclosed subject matter. Furthermore, the foregoing description is for the purpose of illustration only, and not for the purpose of limitation.<br><br><br><br><br>
CLAIMS What is claimed is:<br>
1.	A method of sulfating a polysaccharide, comprising:<br>
(a)	providing a reaction mixture comprising:<br>
at least one O-sulfotransferase (OST) enzyme; and 3'-phosphoadenosine 5'-phosphosulfate (PAPS);<br>
(b)	incubating a polysaccharide substrate with the reaction mixture,<br>
wherein production of the sulfated polysaccharide from the<br>
polysaccharide substrate is catalyzed by the OST enzyme with a<br>
conversion of the PAPS to adenosine 3',5'-diphosphate (PAP);<br>
and<br>
(c)	providing a reaction condition which modifies PAP to reduce an<br>
inhibitory effect of PAP on the polysaccharide sulfation.<br><br>
2.	The method of claim 1, wherein the at least one OST enzyme is<br>
selected from the group consisting of 2-OST, 3-OST-1,3-OST-3,6-<br>
OST, and combinations thereof.<br>
3.	The method of claim 2, wherein the at least one OST enzyme is a<br>
recombinant OST enzyme.<br>
4.	The method of claim 3, wherein the recombinant OST enzyme is<br>
produced in a bacterial expression system.<br>
5.	The method of claim 3, wherein the at least one OST enzyme is a<br>
fusion protein.<br>
6.	The method of claim 5, wherein the fusion protein is a maltose-<br>
binding protein (MBP)-2-OST fusion protein or a MBP-6-OSTfusion<br>
protein.<br>
7.	The method of claim 1, wherein the OST enzyme is immobilized on a<br>
substrate.<br>
8.	The method of claim 7, wherein the substrate is an agarose bead.<br>
9.	The method of claim 1, wherein providing the reaction condition<br>
comprises providing a PAPS regeneration system comprising a<br>
PAPS regenerating enzyme and a sulfur donor compound, wherein<br>
the PAPS regenerating enzyme catalyzes regeneration of the PAPS<br>
from the PAP utilizing the sulfur donor compound as a substrate.<br>
10.	The method of claim 9, wherein the PAPS regenerating enzyme is<br>
an arylsulfotransferase.<br>
11.	The method of claim 10, wherein the arylsulfotransferase is AST-IV.<br>
12.	The method of claim 9, wherein the sulfur donor compound is an aryl<br>
sulfate compound.<br>
13.	The method of claim 12, wherein the aryl sulfate compound is p-<br>
nitrophenol sulfate (PNPS).<br>
14.	The method of claim 1, wherein providing the reaction condition<br>
comprises   providing   a   phosphatase   enzyme,   wherein   the<br>
phosphatase enzyme modifies the PAP.<br>
15.	The method of claim 1, wherein the polysaccharide substrate is a<br>
chemically desulfated /V-sulfated (CDSNS) heparin.<br>
16.	The method of claim 1, wherein the polysaccharide substrate is<br>
partially sulfated prior to reaction mixture incubation.<br>
17.	The method of claim 1, wherein the sulfated polysaccharide is a<br>
glycosaminoglycan (GAG).<br>
18.	The method of claim 17, wherein the GAG is a heparan sulfate (HS).<br>
19.	The method of claim 18, wherein the HS is an anticoagulant-active<br>
HS.<br>
20.	The method of claim 18, wherein the HS is selected from the group<br>
consisting of an antithrombin-binding HS, a fibroblast growth factor<br>
(FGF)-binding HS, and a herpes simplex virus envelope glycoprotein<br>
D-binding HS.<br>
21.	A method of sulfating a polysaccharide, comprising:<br><br>
(a)	providing  a reaction  mixture  comprising  adenosine 3',5'-<br>
diphosphate (PAP), a PAPS regenerating enzyme and a sulfur<br>
donor compound;<br>
(b)	incubating the reaction mixture for a time period sufficient to<br>
catalyze    the    production    of    3'-phosphoadenosine    5'-<br>
phosphosulfate (PAPS) from the PAP by the PAPS regenerating<br>
enzyme utilizing the sulfur donor compound as a substrate; and<br>
(c)	incubating a polysaccharide substrate and at least one O-<br>
sulfotransferase (OST) enzyme with the reaction mixture,<br>
wherein production of a sulfated polysaccharide from the polysaccharide substrate is catalyzed by the OST enzyme with a conversion of the PAPS to PAP and wherein the PAPS regenerating enzyme catalyzes regeneration of the PAPS from the PAP utilizing the sulfur donor compound as a substrate.<br>
22.	The method of claim 21, wherein the PAPS regenerating enzyme is<br>
an arylsulfotransferase.<br>
23.	The method of claim 22, wherein the arylsulfotransferase is AST-IV.<br>
24.	The method of claim 21, wherein the sulfur donor compound is an<br>
aryl sulfate compound.<br>
25.	The method of claim 24, wherein the aryl sulfate compound is p-<br>
nitrophenol sulfate (PNPS).<br>
26.	The method of claim 21, wherein the time period is from about 1<br>
minute to about 30 minutes.<br>
27.	The method of claim 21, wherein the polysaccharide substrate is a<br>
chemically desulfated A/-sulfated (CDSNS) heparin.<br>
28.	The method of claim 21, wherein the polysaccharide substrate is<br>
partially sulfated prior to reaction mixture incubation.<br>
29.	The method of claim 21, wherein the at least one OST enzyme is<br>
selected from the group consisting of 2-OST, 3-OST-1,3-OST-3,6-<br>
OST, and combinations thereof.<br>
30.	The method of claim 29, wherein the at least one OST enzyme is a<br>
recombinant OST enzyme.<br>
31.	The method of claim 30, wherein the recombinant OST enzyme is<br>
produced in a bacterial expression system.<br>
32.	The method of claim 30, wherein the at least one OST enzyme is a<br>
fusion protein.<br>
33.	The method of claim 32, wherein the fusion protein is a maltose-<br>
binding protein (MBP)-2-OST fusion protein or a MBP-6-OSTfusion<br>
protein.<br>
34.	The method of claim 21, wherein the OST enzyme is immobilized on<br>
a substrate.<br>
35.	The method of claim 34, wherein the substrate is an agarose bead.<br>
36.	The method of claim 21, wherein the sulfated polysaccharide is a<br>
glycosaminoglycan (GAG).	<br>
37.	The method of claim 36, wherein the GAG is a heparan sulfate (HS).<br>
38.	The method of claim 37, wherein the HS is an anticoagulant-active<br>
HS.<br>
39.	The method of claim 37, wherein the HS is selected from the group<br>
consisting of an antithrombin-binding HS, a fibroblast growth factor<br>
(FGF)-binding HS, and a herpes simplex virus envelope glycoprotein<br>
D-binding HS.<br>
40.	A kit for sulfating a polysaccharide, the kit comprising:<br><br>
(a)	at least one O-sulfotransferase (OST) enzyme; and<br>
(b)	a reagent which modifies adenosine 3',5'-diphosphate (PAP) to<br>
reduce an inhibitory effect of PAP on polysaccharide sulfation.<br><br>
41.	The kit of claim 40, further comprising instructions for sulfating a<br>
polysaccharide.<br>
42.	The kit of claim 40, wherein the at least one OST enzyme is<br>
contained within a first container and the reagent is contained within<br>
a second container.<br>
43.	The kit of claim 40, wherein the at least one OST enzyme is selected<br>
from the group consisting of 2-OST, 3-OST-1,3-OST-3,6-OST, and<br>
combinations thereof.<br>
44.	The kit of claim 43, wherein the at least one OST enzyme is a<br>
recombinant OST enzyme.<br>
45.	The kit of claim 44, wherein the recombinant OST enzyme is<br>
produced in a bacterial expression system.<br>
46.	The kit of claim 44, wherein the at least one OST enzyme is a fusion<br>
protein.<br>
47.	The kit of claim 46, wherein the fusion protein is a maltose-binding<br>
protein (MBP)-2-OST fusion protein or a MBP-6-OST fusion protein.<br>
48.	The kit of claim 38, wherein the OST enzyme is immobilized on a<br>
substrate.<br>
49.	The kit of claim 48, wherein the substrate is an agarose bead.<br>
50.	The kit of claim 40, wherein the reagent comprises a PAPS<br>
regeneration system comprising a PAPS regenerating enzyme and a<br>
sulfur donor compound.<br>
51.	The kit of claim 50, wherein the PAPS regenerating enzyme is an<br>
aryls u If otransf erase.<br>
52.	The kit of claim 51, wherein the arylsulfotransferase is AST-IV.<br>
53.	The kit of claim 50, wherein the sulfur donor compound is an aryl<br>
sulfate compound.<br>
54.	The kit of claim 53, wherein the aryl sulfate compound is /&gt;<br>
nitrophenol sulfate (PNPS).<br>
55.	The kit of claim 40, wherein the reagent comprises a phosphatase<br>
enzyme.</td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/viewdoc.aspx?id=h0NXvrzOIwHxCWwaJT0N/Q==&amp;amp;loc=+mN2fYxnTC4l0fUd8W4CAA==" target="_blank" style="word-wrap:break-word;">http://ipindiaonline.gov.in/patentsearch/GrantedSearch/viewdoc.aspx?id=h0NXvrzOIwHxCWwaJT0N/Q==&amp;amp;loc=+mN2fYxnTC4l0fUd8W4CAA==</a></p>
		<br>
		<div class="pull-left">
			<a href="269204-immobilization-of-proteins-onto-solid-support-and-its-process-thereof.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="269206-an-orally-administrable-composition-useful-for-treating-or-amelorating-the-risk-of-digestive-tract-infection.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>269205</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>9471/DELNP/2007</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>42/2015</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>16-Oct-2015</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>09-Oct-2015</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>07-Dec-2007</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>(1) THE UNIVERSITY OF NORTH CAROLINA AT CHAPEL HILL (2) RENSSELAER POLYTECHNIC INSTITUTE</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td>308 BYNUM HALL, CAMPUS BOX 4105, CHAPEL HILL, NC 27599-4105 USA.</td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>LIU JIAN</td>
											<td>106 IVY BROOK LANE, CHAPEL HILL, NC 27516 USA.</td>
										</tr>
										<tr>
											<td>2</td>
											<td>LINHARDT ROBERT J</td>
											<td>214 LANCASTER STREET, ALBANY, NY 12210 USA.</td>
										</tr>
										<tr>
											<td>3</td>
											<td>AVCI FIKRI Y</td>
											<td>181, LONGWOOD AVE, 6TH FLOOR, BOSTON, MA 02115, USA</td>
										</tr>
										<tr>
											<td>4</td>
											<td>MUNOZ EVA M</td>
											<td>RATZEBURGER ALLEE 67A, 23562 LUEBECK XX GERMANY</td>
										</tr>
										<tr>
											<td>5</td>
											<td>CHEN JINGHUA</td>
											<td>NAN HU YA YUAN H-3-302, SOUTH LUO SHI ROAD, HONGSHAN DISTRICT, WUHAN HUBEI XX 430070 CHINA</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>C12P 19/00</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>PCT/US2006/018778</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td>2006-05-12</td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td>60/680,392</td>
									<td>2005-05-12</td>
								    <td>U.S.A.</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/269205-enzymatic-synthesis-of-sulfated-polysaccharides by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 08:04:11 GMT -->
</html>
