<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/269277-trichoderma-reesei-glucoamylase-and-homologs-thereof by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 07:52:28 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 269277:&quot;TRICHODERMA REESEI GLUCOAMYLASE AND HOMOLOGS THEREOF&quot;</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">&quot;TRICHODERMA REESEI GLUCOAMYLASE AND HOMOLOGS THEREOF&quot;</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>The present invention is related to glucoamylascs having at least 80 % sequence identity to a Trichodenna glucoamylasc having the sequence of SEQ II) NO: 4 and biologically functional fragments thereof. The invention is also related to DNA sequences coding for the glucoamylascs, vectors and host cells incorporating the DNA sequences, enzyme compositions and methods of using the glucoamyluses in various applications.</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td>TRICHODERMA REESEI GLUCOAMYLASE<br>
AND HOMOLOGS THEREOF<br>
RELATED APPLICATIONS<br>
The present application claims priority to International Patent Application PCT/US05/18214, filed May 24,2005, which claims priority to provisional patent application serial number 60/647,925 filed January 28,2005, International Patent Application PCT/US04/041276 filed December 9, 2004; International Application PCT/US04/040040, filed November 30,2004; provisional application serial number 60/605,437, filed August 30,2004, now abandoned and provisional application serial number 60/575,175, filed May 27,2004, now abandoned, and claims priority to International Patent Application PCT/US05/0018212 filed May 24,2005; which claims priority to provisional applications serial number 60/647,925 filed January 28, 2005, serial number 60/605,437 filed Aug. 30, 2004, now abandoned and serial number 60/575,175 filed May 27,2004, now abandoned and International Application PCT/US04/040040, filed November 30,2004.<br>
FIELD OF THE INVENTION<br>
The present invention relates to new glucoamylases useful for the production of glucose and other end products from starch. The glucoamylases are suitable for use in various processes and are particularly suitable for use under conditions of conventional high temperature starch processing and under conditions of non-cook or low temperature starch processing.<br>
BACKGROUND OF THE INVENTION<br>
Glucoamylase enzymes (α-l,4-glucan glucohydrolases, E.C.3.2.1.3.) are starch hydrolyzing exo-acting carbohydrases. Glucoamylases catalyze the removal of successive<br><br>
glucose units from the non-reducing ends of starch or related oligo and polysaccharide molecules and can hydrolyze both linear and branched glucosidic linkages of starch (amylose and amylopectin).<br>
Glucoamylases are produced by numerous strains of bacteria, fungi, yeast and plants. Particularly interesting glucoamylases are fungal enzymes that are extracellularly produced, for example from strains of Aspergillus (Boel et.al., (1984) EMBO J 3:1097-1102; Hayashida et al (1989) Agric. Biol Chem. 53:923 - 929; USP 5,024,941; USP 4,794,175; and WO 88/09795), Talaromyces (USP 4,247,637; USP 6,255,084 and USP 6,620,924), Rhizopus (Ashikari et al. (1986) Agric. Biol. Chem. 50:957 - 964; Ashikari et al. (1989) App. Microbiol. and Biotech. 32:129 - 133 and USP 4,863,864), Humicola (WO05/052148 and USP 4,618,579) and Mucor (Houghton-Larsen et al., (2003) Appl Microbiol. Biotechnol., 62: 210- 217). Many of the genes, which code for these enzymes have been cloned and expressed in yeast and fungal cells.<br>
Commercially glucoamylases are very important enzymes that have been used in a wide variety of applications requiring the hydrolysis of starch. Glucoamylases are used for the hydrolysis of starch to produce high fructose corn sweeteners, and corn sweeteners comprise over 50% of the US sweetener market. In general, starch hydrolyzing processes involve the use of alpha amylases to hydrolyze the starch to dextrins and glucoamylases to hydrolyze the-dextrins to glucose. The glucose is then converted to fructose by other enzymes such as glucose isomerases. Glucose produced by glucoamylases can also be crystallized or used in fermentations to produce other end-products, such as citric acid, ascorbic acid, glutamic acid, 1, 3 propanediol and others. Glucoamylases are used in alcohol production, such as beer production and sake production. Glucoamylases also find use in the production of ethanol for fuel and for consumption. Recently, glucoamylases have been used in low-temperature processes for the hydrolysis of granular (non-cooked) starch. Glucoamylases are also used hi the preparation of animal feeds as feed additives or as liquid feed components for livestock animals.<br>
Although glucoamylases have been used successfully for many years, a need still exists for new useful glucoamylases. The present invention is based upon the finding of novel glucoamylases suitable for use in various applications and particularly starch conversion processes.<br><br>
SUMMARY OF THE INVENTION<br>
The invention is directed to an isolated DNA sequence encoding a glucoamylase having at least 80% identity to SEQ ID NO: 4.<br>
In another embodiment, the invention is directed to an enzyme having glucoamylase activity comprising the amino acid sequence of SEQ ID NO: 4 or substantially homologous sequences thereto and allelic variants and biologically functional fragments thereof.<br>
In another embodiment, the invention is related to an isolated DNA sequence encoding a Trichoderma reesei glucoamylase including the native gene sequence and biologically functional fragments thereof.<br>
In another embodiment, the invention is direct to vectors comprising a DNA sequence encoding the glucoamylases encompassed by the invention.<br>
In another embodiment, the invention is directed to stable transformed fungal host cells, particularly Trichodemia and Aspergillus host cells and methods for the expression of the glucoamylase therefrom.<br>
In another embodiment, the invention is directed to a culture medium including a glucoamylase encompassed by the invention and enzyme preparations obtained from the growth or culture of transformed hosts and the use of the enzyme preparations.<br>
In  another embodiment, the invention is directed to starch conversion processes using the enzyme preparations of the invention. In some embodiments, the glucoamylase will be used in a process of converting starch or partially hydrolyzed starch into a syrup containing dextrose, In other embodiments, the glucoamylase will be used in a process for producing specialty syrups. v further embodiments, the glucoamylase will be used in a fermentation to produce end products, such as alcohols and particularly ethanol.<br>
BRIEF DESCRIPTION OF THE DRAWINGS<br>
FIG.1A - B shows the genomic DNA sequence (SEQ ID NO: 1) coding for the Trichoderma reesei glucoamylase of Figure 3.<br>
FIG. 2A - B shows the intronless DNA sequence (SEQ ID NO: 2) coding for the Trichoderma reesei glucoamylase of Figure 3.<br>
FIG. 3 A shows the deduced amino acid sequence (SEQ ID NO: 3) of the Trichoderma reesei glucoamylase having 632 amino acids, wherein<br><br>
the signal sequence (SEQ ID NO: 38) is in bold and is represented by residue positions 1 - 20;<br>
the prosequence (SEQ ID NO: 39) is in bold and underlined and represented by '<br>
residue positions 21 - 33;<br>
the catalytic domain (SEQ ED NO: 40) is represented by residue positions 34 -<br>
486;<br>
the linker region (SEQ ID NO: 41) is in italics and represented by residue positions<br>
487 - 523;and<br>
the starch binding domain (SEQ ED NO: 42) is in italics and underlined and<br>
represented by residue positions 524 - 632.<br>
The N- terminal ammo acid residue of the mature protein represented by residue<br>
position 34 is serine.<br>
FIG. 3B shows the deduced mature protein sequence (SEQ ID NO: 4) of the Trichoderma reesei glucoamylase of Fig. 3A. The mature protein sequence includes the catalytic domain, which is underlined (SEQ ID NO: 40), the linker region (SEQ ID NO: 41) and starch binding domain (SEQ ID NO: 42).<br>
FIG. 4 shows the genomic DNA sequence having 2154 bp (SEQ ID NO: 5) coding for the Hypocrea citrina var. americana glucoamylase (GA102) (SEQ ID NO: 6).<br>
FIG. 5 shows the genomic DNA sequence having 2152 bp (SEQ ID NO: 7) coding for the Hypocrea vinosa glucoamylase (GA104) (SEQ ID NO: 8).<br>
FIG. 6 shows the genomic DNA sequence having 2158 bp (SEQ ID NO: 9) coding for a Trichoderma sp. glucoamylase (GA105) (SEQ ID NO: 10).<br>
FIG. 7 shows the genomic DNA sequence having 2144 bp (SEQ ID NO: 11) coding for a Hypocrea gelatinosa glucoamylase (GA107) (SEQ ID NO: 12).<br>
FIG. 8 shows the genomic DNA sequence having 2127 bp (SEQ ID NO: 13) coding for a Hypocrea orientalis glucoamylase (GA108) (SEQ ID NO: 14).<br>
FIG 9 shows the genomic DNA sequence having 2139 bp (SEQ ID NO: 15) coding for a Trichoderma konilangbra glucoamylase (GA109) (SEQ ID NO: 16).<br>
FIG. 10 shows the genomic DNA sequence having 2088 bp (SEQ ID NO: 28) coding for Trichoderma sp. glucoamylase (GA113) (SEQ ID NO: 29).<br>
FIG. 11 shows the genomic DNA sequence having 2141 bp (SEQ ID NO: 30) coding for a Trichodenna harzianum glucoamylase (GA103) (SEQ ID NO: 31).<br><br>
FIG. 12 shows the genomic DNA sequence having 2131 bp (SEQ ID NO: 32) coding for a Trichoderma longibrachiatum glucoamylase (GA124) (SEQ ED NO: 33).<br>
FIG. 13 shows the genomic DNA sequence having 2151bp (SEQ ID NO: 34) coding for Trichoderma asperellum glucoamylase (GA127) (SEQ ID NO: 35).<br>
FIG. 14 shows the genomic DNA sequence having 2142 bp (SEQ ID NO: 36) coding for Trichoderma strictipilis glucoamylase (GA128) (SEQ ID NO: 37).<br>
FIG. 15A -1 shows the putative amino acid sequences for glucoamylases encoded by the DNA sequences of SEQ ID NOs: 5,1,9,11,13,15,28,30, 32, 34 and 36, which correspond to the amino acid sequences of SEQ ID NOs: 6, 8,10,12,14,16, 29, 31, 33, 35 and 37 respectively, wherein the leader peptide is in bold and the prosequence is underlined and in bold for each protein. The mature protein sequence which excludes the leader and prosequence for each protein is also represented as SEQ ED NO: 17 for (1) GA102; SEQ ID NO: 18 for (2) GA104; SEQ ID NO: 19 for (3) GA105; SEQ ID NO: 20 for (4) GA107; SEQ ID NO: 21 for (5) GA108; SEQ ID NO: 22 for (6) GA109; SEQ ID NO: 43 for (7) GA113; SEQ ID NO: 44 for (8) GA103; SEQ ID NO: 45 for (9) GA124; SEQ ID NO: 46 for (10) GA127 and SEQ ID NO: 47 for (11) GA128.<br>
.  FIG. 16 illustrates the SDS-PAGE gel used for determining MW of the purified TrGA, wherein lane 1 exhibits the TrGA and lane 2 exhibits the molecular weight marker SeeBlue Plus 2 (Invitrogen).<br>
FIG. 17A is a plasmid map of T. reesei expression vector, pTrexSg.<br>
FIG 17B is a plasmid map that includes the T. reesei expression vector pNSP23, wherein the TrGA gene is cloned into pTrex3g.<br>
FIG. 18 shows (A) the % relative GA'activity of the TrGA at 37°C from pH 3 - 8 and (B) the % relative GA activity of the TrGA at pH 4.0 from 25°C to 78°C and reference is made to example 4.<br>
FIG. 19 illustrates the SDS-PAGE gel used for determining secretion of substantially homologous glucoamylases in the Trichoderma host strain (1A52), wherein the band at about 62kDa represents glucoamylase and lane 1 represents GA104, lane 2 represents GA105; lane 3 represents GC107; lane 4 represents GA109; lane 5 represents TrGA; "lane 6 represents a Trichoderma reesei control host strain (1A52); and lane 7 represents a standard molecular weight marker.<br><br>
FIG. 20 (A) illustrates the amino acid sequence (SEQ ED NO: 26) for an Aspergillus niger glucoamylase which includes the leader sequence. The N- terminal amino acid residue of the mature protein is represented by residue position 25, A (alanine); the linker region is underlined and the starch binding domain is in italics. (B) illustrates the amino acid sequence for an Aspergillus kawochi alpha amylase (SEQ ID NO: 27) which includes the leader sequence, wherein the leader sequence is in bold and underlined and is represented by amino acid residues 1 — 21; the linker region is underlined and the starch binding domain is in italics. The mature protein includes the catalytic domain, the linker and the starch binding domain.<br>
DETAILED DESCRIPTION OF THE INVENTION<br>
In some aspects, the present invention relies on routine techniques and methods used in the field of genetic engineering and molecular biology. The following resources include descriptions of general methodology useful in accordance with the invention: Sambrook et al. Eds., MOLECULE CLONING: A LABORATORY MANUAL (3rd Ed. 2000); Kriegler M. Ed., GENE TRANSFER AND EXPRESSION: A LABORATORY MANUAL (1990); and Ausubel et al. Eds., SHORT PROTOCOLS IN MOLECULAR BIOLOGY (5th Ed. 2002). Although any methods and materials similar or equivalent to those described herein can be used in the practice or testing of the invention, the preferred methods and materials are described below.<br>
Unless defined otherwise herein all technical and scientific terms used herein have the same meaning as commonly understood by one of ordinary skill in the art to which this invention belongs. Singleton et al., et al., DICTIONARY OF MICROBIOLOGY AND MOLECULAR BIOLOGY 2nd Ed, John Wiley and Sons, NY (1994) and Hale and Margham, THE HARPER COLLINS DICTIONARY OF BIOLOGY (1991) Addison Wesley Pub. Co. provides one of skill with dictionaries of many of the terms used in describing this invention.<br>
The invention will now be described in detail by way of reference only using the following definitions and examples. All patents and publications, including all sequences disclosed within such patents and publications referred to herein are expressly incorporated by reference.<br><br>
The singular forms "a", "an" and "the" include the plural references unless the content clearly dictates otherwise. Thus for example, reference to a composition containing "a compound" includes a mixture of two or more compounds. It should be noted that the term "or" is generally employed in the sense including "and/or" unless the content clearly dictates otherwise.<br>
Numeric ranges are inclusive of the numbers of the ranges.<br>
Unless otherwise indicated, nucleic acids are written left to right 5' to 3' orientation; amino acids sequences are written left to right in amino carboxy orientation, respectively.<br>
The headings provided herein are not limitations of the various aspects or embodiments of the invention, which can be had by reference to the specification as a whole.<br>
Definitions<br>
The term "glucoamylase" refers to the amyloglucosidase class of enzymes (E.G. 3.2.1.3, glucoamylase, 1,4-alpha-D-glucan glucohydrolase). These enzymes release glucosyl residues from the non-reducing ends of amylose and amylopectin molecules.<br>
The phrase "having granular starch hydrolyzing activity" means an enzyme that is capable of hydrolyzing starch in granular form.<br>
The phrase "Trichoderma/Hypocrea family cluster" means a member of the Family<br>
#<br>
Hypocreaceae including several anamorphs as Trichoderma and Gliocladium of the Order Hypocreales, Phylum Ascomycota and reference is made to Chapter 12, Alexopoulos, C.J., et al., in INTRODUCTORY MYCOLOGY 4th Edition, John Wiley &amp; Sons, NY 1996.<br>
The terms "nucleic acid sequence" and "polynucleotide" maybe used interchangeably herein. The term encompasses genomic DNA, intronless DNA, synthetic origins or combinations thereof.<br>
The term "intron" means an intervening DNA sequence that is transcribed but is removed from within the transcript by splicing together the coding sequences of the mature protein.<br>
The term "isolated nucleic acid sequence" means a nucleic acid sequence, which is essentially free of other nucleic acid sequences.<br><br>
The term "biologically functional fragments of a sequence" (e.g. biologically functional fragments of SEQ ID NO: 4) means a polypeptide having glucoamylase activity and one or more amino acid residues deleted from the amino and/or carboxyl terminus of the amino acid sequence.<br>
The term "vector" means a polynucleotide sequence designed to introduce nucleic acids into one or more cell types.<br>
The term "expression vector" means a DNA construct comprising a nucleic acid<br>
\ sequence, which is operably linked to a suitable control sequence capable of effecting<br>
expression of the nucleic acid sequence in a suitable host. Suitable control sequences include promoters to effect transcription, operator sequences, sequences encoding suitable ribosome binding sites on the mRNA, enhancers and/or termination sequences.<br>
The term "promoter" means a regulatory sequence involved in binding RNA polymerase to initiate transcription of a gene.<br>
The term "operably linked" refers to juxtaposition wherein the elopements are in an arrangement allowing them to be functionally related. For example, a promoter is operably linked to a coding sequence if it controls the transcription of the sequence.<br>
The term "an isolated polypeptide" means a polypeptide that is essentially free of other non-glucoamylase polypeptides. An isolated polypetide may be at least 20% pure, at least 40% pure, at least 60% pure, at least 70% pure, at least 80% pure, at least 90% pure, at least 95% pure as determined by SDS-PAGE.<br>
The term "signal sequence" means a sequence of amino acids bound to the N-terminal portion of a protein, which facilities the secretion of the mature form of a protein outside the cell. The definition of a signal sequence is a functional one. The mature form of the extracellular protein lacks the signal sequence, which is cleaved off during the secretion process. The terms "signal sequence", signal peptide" and "leader peptide" may be used interchangeability herein. In general the signal sequence refers to the nucleotide sequence and the term leader peptide refers to the amino acid sequence.<br>
The terms "protein" and "polypeptide" are used interchangeably herein. The conventional one-letter or three-letter code for amino acids residues is used herein.<br>
The term "catalytic domain" refers to a structural region of a polypetide, which contains the active site for substrate hydrolysis.<br><br>
The term "linker" refers to a short amino acid sequence generally having between 3 and 40 amino acids residues that covalently bind an amino acid sequence comprising a starch binding domain with an amino acid sequence comprising a catalytic domain.<br>
The term "starch binding domain" refers to an amino acid sequence that binds preferentially to a starch substrate.<br>
The term "allelic variants" means any of two or more alterative forms of a gene occupying the same chromosomal locus. Allelic variation arises naturally through mutation and may result in polymorphism between populations. An allelic variant of a polypeptide is a polypeptide encoded by an allelic variant of a gene.<br>
The term "host cell" or "host strain" means a suitable host for an expression vector or DNA construct comprising a polypeptide encoding a glucoamylase encompassed by the invention. Suitable host cells are used advantageously in the recombinant production of the glucoamylases encompassed by the invention.<br>
As used herein the term "derived from" used in connection with a polynucleotide or polypeptide means the polypeptide or polynucleotide is native to the microorganism.<br>
The term "heterologous" with reference to a polynucleotide or protein refers to a polynucleotide or protein that does not naturally occur in a host cell.<br>
The term " endogenous" with reference to a polynucleotide or protein refers to a polynucleotide or protein that occurs naturally in a host cell.<br>
The term "expression" means the process by which a polypeptide is produced based on the nucleic acid sequence of a gene.<br>
The term "over expression" means the process of expressing a polypeptide is a host cell wherein a polynucleotide has been introduced the host cell.<br>
The term "introduced" in the context of inserting a nucleic acid sequence into a cell means transfection, transformation or transduction and includes reference to the incorporation of the nucleic acid sequence into a host cell.<br>
The term "granular starch" refers to raw uncooked starch (e.g. granular starch that has not been subject to gelatinization).<br>
The term "starch" refers to any material comprised of the complex polysaccharide carbohydrates of plant, comprised of amylose and amylopectin with me formula (C6H10O5)x, wherein X can be any number.<br><br>
The term "gelatinization" means the solubilization of a starch molecule by cooking to form a viscous suspension. The phrase "below the temperature of gelatinization" refers to a temperature less than the temperature which starts gelatinization.<br>
The term "culturing" refers to growing a population of microbial cells under suitable conditions in a liquid or solid medium. In one embodiment, culturing refers to fermentative bioconversion of a starch substrate to an end-product (typically in a vessel or reactor). Fermentation is the enzymatic and anaerobic breakdown of organic substances by microorganisms to produce simpler organic compounds. While fermentation occurs under anaerobic conditions it is not intended that the term be solely limited to strict anaerobic conditions as fermentation also occurs in the presence of oxygen.<br>
The term "end-product" refers to any carbon source derived molecule product which is enzymatically converted from a starch substrate.<br>
The term" enzymatic conversion" refers to the modification of a substrate by enzyme action.<br>
The term "specific activity" means an enzyme unit defined as the number of moles of substrate converted to product by an enzyme preparation per unit time under specific conditions. Specific activity is expressed as units (U)/mg or protein.<br>
The term "monosaccharide" means a monomeric unit of a polymer such as starch wherein the degree of polymerization (DP) is 1 (e.g., glucose, mannose, fructose and galactose).<br>
The term "disaccharide" means a compound that comprises two covalently linked monosaccharide units (DP2). The term encompasses, but is not limited to such compounds as sucrose, lactose and maltose.<br>
The term "a DP &gt;3" means polymers with a degree of polymerization greater than 3.<br>
The term "oligosaccharide" means a compound having 2-10 monosaccharide units joined in glycosidic linkages.<br>
The term "polysaccharide" means a compound having multiple monosaccharide units joined in a linear or branched chain. In some embodiments the term refers to long chains with hundreds or thousands of monosaccharide units. Typical examples of polysaccharides are starch, cellulose and glycogen.<br><br>
As used herein the term "dry solids content (DS or ds)" refers to the total solids of a slurry in % on a dry weight basis.<br>
The term "milling" refers to the breakdown of cereal grams to smaller particles. In some embodiments the term is used interchangeably with grinding.<br>
The term "dry milling" refers to the milling of dry whole grain, wherein fractions of the grain such as the germ and bran have not been purposely removed.<br>
As used herein the terms "distillers dried gram (DDG)" and "distillers dried grain with solubles (DDGS)" refer to useful co-products of grain fermentation processes.<br>
The term "DE" or "dextrose equivalent" is an industry standard for measuring the concentration of total reducing sugars, calculated as D-glucose on a dry weight basis. Unhydrolyzed granular starch has a DE that is essentially 0 and D-glucose has a DE of 100.<br>
The term "sugar syrup" refers to an aqueous composition containing soluble carbohydrates. In one embodiment, the sugar syrup is a syrup containing glucose.<br>
Trichoderma reesei Glucoamvlase amino acid sequences<br>
A glucoamylase derived from Trichoderma reesei QM6a (ATCC, Accession No. 13631) has been cloned as further described in detail in Example 1. According to the invention the full length glucoamylase derived from Trichoderma reesei is illustrated in Figure 3 and has an amino acid sequence of SEQ ID NO: 3. The mature protein sequence of the Trichoderma reesei glucoamylase, (SEQ ID NO: 4) is represented by amino acid residues 34 - 632 of Figure 3.<br>
This invention relates to an isolated enzyme having glucoamylase activity comprising the sequence shown in SEQ ID NO: 4 or an enzyme with glucoamylase activity being substantially homologous thereto.<br>
In some embodiments, the invention is related to a glucoamylase comprising the sequence shown in SEQ ID NO: 3 or an enzyme with glucoamylase activity being substantially homologous thereto. The glucoamylase of SEQ ID NO: 3 includes the signal sequence of the glucoamylase obtained from Trichoderma reesei.<br>
In some embodiments the invention is related to a polypeptide having glucoamylase activity comprising the catalytic domain of the glucoamylase of SEQ ID NO: 4, which is also represented by SEQ ID NO: 40.<br><br>
In other embodiments, the invention is related to a starch binding domain having at least 90%, at least 95%, at least 97%, and at least 98% sequence identity to the starch binding domain of the glucoamylase illustrated in SEQ ID NO: 4. In some embodiments, the starch binding domain encompasses the sequence of residue position 524 to residue position 632 of SEQ ID NO: 4 and is represented by SEQ ID NO: 42.<br>
In other embodiments, the starch binding domain is a fragment of the starch binding domain of SEQ ID NO: 4. Preferably a fragment will encompass at least 90, at least 80 or at least 70 ammo acid residues of the starch binding domain of SEQ ID NO: 4.<br>
Homology of the Protein Sequence<br>
The homology between two glucoamylases may be determined by the degree of identity between the amino acid sequences of two protein sequences. A polypeptide or polynucleotide having a certain percent of identity with another sequence (i.e. 80%, 90%, and 95%) means that when aligned, that percent of bases or amino acid residues are the same in comparing the two sequences. This alignment and percent homology or identity can be determined by using any suitable software program known in the art. For example suitable programs are described in CURRENT PROTOCOLS IN MOLECULAR BIOLOGY (Ausubel et al., eds 1995, Chapter 19). Preferred programs include GCG Pileup program (Wisconsin Package, Version 8.1 and 10.0), FAST A, BLAST and TFASTA. Another preferred alignment program is ALIGN or ALIGN Plus (Dayhoff (1978) in ATLAS OF PROTEIN SEQUENCE AND STRUCTURE 5: Suppl. 3 (National Biomedical Research Foundation)) Further BLASTP, BLASTN and BLASTX algorithms can be used (Altschul et al., (1990)7. Mol Biol. 215:403-410). Other useful methods include ClustralW (Thompson et al., (1997) Nucleic Acid Research 25:4876 -4882) using software provide by DNASTAR (Madison WI). Also reference is made to Needleman et al., (1970) J. Mol. BioL 48:443, Smith et al., (1981) Adv. AppL Math. 2:482, Smith et al., (1997) Meth. Mol. Biol. 70:173-187 and Pearson et al., (1988) Proc. Natl Acad. Sci. 85:24444.<br>
According to the invention a "substantially homologous" amino acid sequence exhibits glucoamylase activity and at least 80% identity, at least 83%, at least 85%, at least 87%, at least 90%, at least 93%, at least 95%, at least 97%, at least 98% and at least 99% identity with the sequence illustrated in SEQ ID NO: 4 or the sequence illustrated in SEQ ID NO: 3. Particularly preferred substantially homologous glucoamylase sequences<br><br>
are the mature protein sequences as shown in Figure 15 and which correspond to SEQ ED NOs: 17,18,19, 20,21,22,43,44,45,46 and 47. Additionally, preferred substantially homologous glucoamylase sequences are the sequences shown in Figure 15, which correspond to SEQ ID NOs: 6,8,10,12,14,16,29, 31,33, 35 and 37 and include a leader sequence. Further substantially homologous polypeptides include allelic variations and natural mutants having glucoamylase activity.<br>
The glucoamylases of the present invention including substantially homologous polypeptides and biologically functional fragments, have at least 20%, at least 40%, at least 50%, at least 60%, at least 70%, at least 80%, at least 90%, at least 95% and at least 100% of the glucoamylase activity of the mature protein derived from Trichoderma reesei having the sequence illustrated in Figure 3 (SEQ ID NO: 4). In some preferred embodiments of the invention, the specific activity of the glucoamylases tested under essentially the same conditions will be at least 90%, at least 100%, at least 125%, at least 150%, at least 175% and also at least 200% of the specific activity of the mature protein derived from Trichoderma reesei having the sequence illustrated in Figure 3 (SEQ ID NO: 4). hi some embodiments, the specific activity may be measured on a soluble starch substrate and in other embodiments the specific activity may be measured on a granular starch substrate.    <br>
In some embodiments, an amino acid sequence having at least 80% sequence identity to the sequence of SEQ ID NO: 3 or SEQ ID NO: 4 will include conservative amino acid substitutions using L-amino acids, wherein one amino acid is replaced by another biologically similar amino acid. Conservative amino acid substitutions are those that preserve the general charge, hydrophobicity/hydrophilicity, and/or steric bulk of the amino acid being substituted. Non-limiting examples of conservative substitutions include those between the following groups: Gly/Ala, Val/Ile/Leu, Lys/Arg, Asn/Gln, Glu/Asp, Ser/Cys/Thr and Phe/Trp/Tyr. Other conservative substitutions can be taken from the table below.<br><br>
Table 1 - Conservative Amino Acid Replacements<br><br>
(Table Removed)<br>
In other embodiments, the amino acid substitutions will not be conservative substitutions.<br>
In some embodiments, it is-contemplated that a glucoamylase of the invention will be derived from a filamentous fungal strain and particularly substantially homologous sequences will be obtained from strains of the genus Aspergillus spp., Rhizopus spp.,<br>
Humicola spp., Fusarium spp., Mucor spp., Trichoderma spp., and the like. In a preferred embodiment, substantially homologous sequences having glucoamylase activity will be derived from strains of the Trichodenna/Hypocrea family cluster. Some of these species include T. stromaticum, H. citrina var. americana, H. citrina, H. lactea, H. hunua, T. fertile, T. tomentosum,H. vinosa, T. harzianum, T. inhamatum, T. oblongisporum, T. cf. aureoviride, T. cf. harzianum, T. fasciculatum, H. tawa, T. crassum, T.flavovirens, T. virens, T. longipilis, T. spirale, T. strictipilis, H. pilulifera, T. polysporum, T. croceum, T. minutisporum, T. hamatum, T. asperellum, T. atroviride, T. koningii, T. viride, H. gelatinosa, T. strigosum, T. pubescens, H. novazelandiae, T. saturnisporum, T. longibrachiattim, H. orientalis, T. citrinoviride, T. reesei, T. ghanense, T. pseudokonimgii, H. andinensis and H. aureoviride. Particularly preferred strains of the genus Trichoderma and allied Hypocrea spp. include H. citrina var. americana, H. citrina, H. lactea, H. vinosa, T. harzianum, T. atroviride, T. koningii, T. viride, H. gelatinosa, T. satumisporum, T. longibrachiatum, H. orientalis, T. citrinoviride, T. reesei, and T. konilangbra.<br>
Some strains of the species described above are accessible to the public from culture collections such as American Type Culture Collection (ATCC) P. O. Box 1549, Manassas, VA 20108; Deutsche Sammlung von Mikroorganismen und Zellkulturen GmbK (DSM); Agricultural Research Service Plant Culture Collection, Northern Regional Research Center (NRRL); the Centraalbureau voor Schirnmelcultures (CBS), P. O. Box 85167, 3508 AD Utrecht, The Netherlands; Plant Research Institute, Department of Agriculture, Mycology, Ottawa, (DAOM) Canada and International Mycological Institute (IMI), Genetic Resources Collection, Egham, United Kingdom..<br>
Biologically Functional Glucoamvlase Fragments<br>
In some embodiments, the invention is related to biologically functional fragments of the glucoamylase disclosed in SEQ ID NO: 3, SEQ ID NO: 4 or substantially homologous sequences thereto, in some embodiments, the biologically functional fragment will include the catalytic domain of a glucoamylase encompassed by the invention. Inother embodiments, the biologically functional fragments will include at least 400 amino acid residues, at least 425 amino acid residues, at least 450 amino acid residues, and also at least 460 amino acid residues.<br>
In some preferred embodiments, the fragment will encompass at least a part of the amino acid sequence represented by residue positions 1 to 453 of SEQ ID NO: 4, and in other embodiments, the fragment will encompass positions 1 to 453 of SEQ ID NO: 4. in further preferred embodiments, the fragment will encompass the amino acid sequence represented by residue positions 1 to 453 of SEQ ID NO: 17; residue positions 1 to 452 of SEQ ID NO: 18; residue positions 1 to 454 of SEQ ED NO: 19; residue positions 1 to 452 of SEQ ID NO: 20; residue positions 1 to 453 of SEQ ID NO: 21; residue positions 1 to 453 of SEQ ID NO: 22; residue positions 1 to 452 of SEQ ID NO: 43; residue positions 1 to 452 of SEQ ID NO: 44; residue positions 1 to 453 of SEQ ID NO: 45; residue positions 1 to 452 of SEQ ID NO: 46; or residue positions 1 to 453 of SEQ ID NO: 47.<br>
Biologically functional glucoamylase fragments encompassed by the invention can be generated by method known in the art.<br>
Glucoamylases having at least 85%, at least 90%, at least 93%, at least 95%, at least 97%, at least 98% and at least 99% sequence identity to the fragment which consists of amino acid residue 1 to 453 of SEQ ID NO: 4 are also contemplated by the invention.<br>
In other embodiments, the biologically functional fragments will include the<br>
1 .<br>
catalytic domain and the linker sequence of the glucoamylase disclosed in SEQ ID NO: 4.<br>
The biologically functional fragments may also comprise fused polypeptides or cleavable fused polypeptides in which another polypeptide is fused at the N- terminus and/or the C- terminus of the polypeptide. Techniques for producing fusion polypeptides are known in the art.<br>
Cloned Trichoderma reesei and Substantially Homologous DNA sequences The invention also relates to a cloned DNA sequence coding for a polypeptide exhibiting glucoamylase activity of the invention, said DNA sequence comprising<br>
a)	the DNA sequence illustrated in SEQ ID NO: 1;<br>
b)	the DNA sequence illustrated in SEQ ID NO: 2;<br>
c)	a DNA sequence encoding a glucoamylase having at least 80%, at least 83%, at<br>
least 85%, at least 87%, at least 90%, at least 93%, at least 95%, at least 97%, at<br>
least 98% and at least 99% identity with the sequence of SEQ ID NO: 3;<br><br>
d)	a DNA sequence encoding a glucoamylase having at least 80%, at least 83%, at<br>
least 85%, at least 87%, at least 90%, at least 93%, at least 95%, at least 97%, at<br>
least 98% and at least 99% identity with the sequence of SEQ ID NO: 4;<br>
e)	a DNA sequence encoding an enzyme having glucoamylase activity, wherein the<br>
enzyme has at least 95%, at least 96%, at least 97%, at least 98% and at least 99%<br>
sequence identity to any one of the sequences shown in SEQ ID NOs: 17,18,19,<br>
20,21,22,43,44,45,46 and 47;<br>
f)	a DNA sequence encoding a biologically functional fragment of a sequence having<br>
at least 85%, at least 90%, at least 95%, at least 96%, at least 97% and at least 98%<br>
identity to amino acid residue position 1 to 453 of the sequence shown in SEQ ID<br>
NO: 4;<br>
g)	a DNA sequence encoding an enzyme having glucoamylase activity comprising an<br>
amino acid sequence having at least 90%, at least 95%, at least 97% and at least<br>
98% sequence identity to any one of the following sequences<br>
a.	amino acid residue positions 1 to 453 of SEQ ID NO: 17;<br>
b.	amino acid residue positions 1 to 452 of SEQ ID NO: 18;<br>
c.	amino acid residue positions 1 to 454 of SEQ ID NO: 19;<br>
d.	amino acid residue positions 1 to 452 of SEQ ID NO: 20;<br>
e.	amino acid residue positions 1 to 453 of SEQ ID NO: 21;<br>
f.	amino acid residue positions 1 to 453 of SEQ ED NO: 22;<br>
g.	amino acid residue positions 1 to 452 of SEQ ID NO: 43;<br>
h.	amino acid residue positions 1 to 452 of SEQ ID NO: 44;<br>
i.	amino acid residue positions 1 to 453 of SEQ ID NO: 45;<br>
j.   amino acid residue positions 1 to 452 of SEQ ID NO: 46; and k.   amino acid residue positions 1 to 453 of SEQ ID NO: 47. h)   a DNA which is at least 80%, at least 85%, at least 90%, at least 93%, at least 95%, at least 97% and at least 99% identical to the sequence shown in SEQ ID NO: 1 or SEQ ID NO: 2, wherein said DNA sequence codes for an enzyme having glucoamylase activity; or<br>
i)   a DNA sequence, which hybridizes under high stringent conditions to a nucleic acid probe corresponding to the DNA sequence of SEQ ID NO: 2 or a fragment<br><br>
             thereof having at least 20, at least 30 at least 40, at least 50 at least 60, at least 70 at<br>
least 100, at least 150 consecutive nucleotides.<br>
The invention additionally encompasses a cloned DNA sequence encoding an enzyme having glucoamylase activity and at least 95%, at least 96%, at least 97% at least 98% and at least 99% sequence identity to the ammo acid sequences of any one of SEQ ID NOs: 6, 8,10,12,14,16, 29,31,33, 35, and 37.<br>
Because of the degeneracy of the genetic code, more than one codon may be used to code for a particular amino acid. Therefore, different DNA sequences may encode a polypeptide having exactly the same amino acid sequence as the polypeptide of, for example SEQ ID NO: 4. The present invention encompasses polynucleotides, which encode the same polypeptide. DNA sequences, which encode glucoamylases encompassed by the-invention may or may not include introns.<br>
Homology of DNA sequences is determined by the degree of identity between two DNA sequences. Homology may be determined using computer programs as described above for determining protein sequence homology.<br>
A nucleic acid is hybridizable to another nucleic acid when a single stranded form of the nucleic acid can anneal to the other nucleic acid under appropriate conditions of temperature and solution ionic strength. Hybridization and washing conditions are well known in the art for hybridization under low, medium, medium/high, high and very high stringency conditions (See., e.g. Sambrook et al., supra, particularly chapters 9 and 11). hi general, hybridization involves a nucleotide probe and a homologous DNA sequence that form stable double stranded hybrids by extensive base-pairing of complementary polynucleotides (See, Chapter 8, GeneCloning, An Introduction, T.A. Brown, (1995) Chapman and Hall, London).<br>
The filter with the probe and homologous sequence are washed in 2x sodium chloride /sodium citrate (SSC), 0.5%SDS at about 60°C (medium stringency); 65°C (medium/high stringency) 70°C (high stringency) and about 75°C (very high stringency).<br>
Vectors<br>
According to one embodiment of the invention, a DNA construct comprising a nucleic acid sequence encoding a glucoamylase encompassed by the invention and operably linked to a promoter sequence is assembled to transfer into a host cell. The DNA<br><br>
construct may be introduced into a host cell using a vector. The vector may be any vector which when introduced into a host cell is integrated into the host cell genome and is replicated. Vectors include cloning vectors, expression vectors, shuttle vectors, plasmids, phage particles, cassettes and the like. In some preferred embodiments, the vector is an expression vector that comprises regulatory sequences operably linked to the glucoamylase coding sequence.<br>
Examples of suitable expression and/or integration vectors are provided in Sambrook et al., (1989) supra, and Ausubel (1987) supra, and van den Hondel et al. (1991) in Bennett and Lasure (Eds.) MORE GENE MANIPULATIONS IN FUNGI, Academic Press pp. 396-428 and U.S. Patent No. 5,874,276. Reference is also made to the Fungal Genetics Stock Center Catalogue of Strains (FGSC, ) for a list of vectors. Particularly useful vectors include vectors obtained from for examples Invitrogen and Promega. Specific vectors suitable for use in fungal host cells include vectors such as pFB6, pBR322, pUCIS, pUClOO, pDON™201, pDONR™221, pENTR™, pGEM®3Z and pGEM®4Z.<br>
In some preferred embodiments, the promoter, which shows transcriptional activity in a fungal host cell may be derived from genes encoding proteins either homologous or heterologous to the host cell. The promoter may be a mutant, truncated and hybrid promoter. Preferably, the promoter is useful in a Trichoderma or Aspergillus host. Exemplary promoters include the T. reesei promoters cbh\, cbh2, eg/1, eg/2, eg5, xlnl and xln2. Other examples of useful promoters include promoters from A. awamori and A. niger glucoamylase genes (glaA) (See, Nunberg et al, (1984) Mol. Cell Biol 4:2306-2315 and Boel et al, (1984) EMBO J. 3:1581-1585), Aspergillus nidulans acetamidase genes and Rhizomucor miehei lipase genes.<br>
hi one embodiment, the promoter is one that is native to the host cell. For example, when T. reesei is the host, the promoter is a native T. reesei promoter.<br>
In another embodiment, the promoter is one that is heterologous to the fungal host cell.<br>
hi a preferred embodiment, the promoter is T. reesei cbhl, which is an inducible promoter and has been deposited in GenBank under Accession No. D86235.<br>
An "inducible promoter" is a promoter that is active under environmental or developmental regulation. In some embodiments, the DNA construct includes nucleic<br><br>
acids coding for a signal sequence that is an amino acid sequence linked to the amino terminus of the polypeptide which directs the encoded polypeptide into the cell's secretory pathway. The 5' end of the coding sequence of the nucleic acid sequence may naturally include a signal peptide coding region which is naturally linked in translation reading frame with the segment of the glucoamylase coding sequence which encodes the secreted glucoamylase or the 5' end of the coding sequence of the nucleic acid sequence may include a signal peptide which is foreign to the coding sequence. In some preferred embodiments, the DNA construct includes a signal sequence that is naturally associated with the glucoamylase gene to be expressed. Effective signal sequences may include the signal sequences obtained from glucoamylases of other filamentous fungal cells, such as from Humicola, Aspergillus, and Rhizopus.<br>
In preferred embodiments, the nucleic acid of the DNA construct codes for a signal sequence having at least 95%, at least 96%, at least 97%, at least 98% and at least 99% sequence identity to the signal sequence depicted in Figure 3.<br>
In additional embodiments, a DNA construct or vector comprising a signal sequence and a promoter sequence to be introduced into a fungal host cell are derived from the same source. For example, in some embodiments, the signal sequence is the cbhl signal sequence which is operably linked to a cbhl promoter. In other preferred embodiments the native glucoamylase signal sequence of a Trichoderma/Hypocrea family cluster member will be used.<br>
In some embodiments, the expression vector also includes a termination sequence. Any terminator sequence functional in the host cell may be used in the present invention. In one embodiment, the termination sequence and the promoter sequence are derived from the same source. In another embodiment, the termination sequence is homologous to the host cell. A particularly suitable terminator sequence is cbhl derived from a Trichoderma strain and particularly T. reesei. Other useful fungal terminators include the terminator from A. niger or A. awamori glucoamylase genes (Nunberg et al. (1984) supra, and Boel et al, (1984) supra), Aspergillus nidulans anthranilate synthase g,eaes,.Aspergillus oryzae TAKA amylase genes, or A nidulans trpC (Punt et al., (1987) Gene 56:117 - 124).<br>
In some embodiments, an expression vector includes a selectable marker. Examples of preferred selectable markers include ones which confer antimicrobial resistance (e.g., hygromycin and phleomycin). Nutritional selective markers also find use<br><br>
in the present invention including those markers known in the art as and S, argR and pyr4. Markers useful in vector systems for transformation of Trichodertna are known in the art (See, e.g., Finkelstein, chapter 6 in BIOTECHNOLOGY OF FILAMENTOUS FUNGI, Finkelstein et al. Eds. Butterworth-Heinemann, Boston, MA (1992), Chap. 6.; and Kinghorn et al. (1992) APPLIED MOLECULAR GENETICS OF FILAMENTOUS FUNGI, Blackie Academic and Professional, Chapman and Hall, London). In a preferred embodiment, the selective marker is the anidS gene, which encodes the enzyme acetamidase, allowing transformed cells to grow on acetamide as a nitrogen source. The use of A. nidulans amdS gene as a selective marker is described in Kelley et al, (1985) EMBO J. 4:475 - 479 and Penttila et al, (1987) Gene 61:155-164.<br>
Methods used to ligate the DNA construct comprising a nucleic acid sequence encoding a glucoamylase, a promoter, a terminator and other sequences and to insert them into a suitable vector are well known in the art. Linking is generally accomplished by ligation at convenient restriction sites. If such sites do not exist, synthetic oligonucleotide linkers are used in accordance with conventional practice. (See, Sambrook (1989) supra, and Bennett and Lasure, MORE GENE MANIPULATIONS IN FUNGI, Academic Press, San Diego (1991) pp 70 - 76.). Additionally, vectors can be constructed using known recombination techniques (e.g., Invitrogen Life Technologies, Gateway Technology).<br>
Host Cells<br>
The present invention also relates to host cells comprising a nucleic acid sequence encoding a glucoamylase of the invention, which are used in the production of the glucoamylases of the invention. Preferred host cells according to the invention are filamentous fungal cells, and the term host cell includes both the cells, progeny of the cells and protoplasts created from the cells of a filamentous fungal strains.<br>
The term "filamentous fungi" refers to all filamentous forms of the subdivision Eumycotina (See, Alexopoulos, C. J. (1962), INTRODUCTORY MYCOLOGY, Wiley, New York). These fungi are characterized by a vegetative mycelium with a cell wall composed of chitin, cellulose, and other complex polysaccharides. The filamentous fungi of the present invention are morphologically, physiologically, and genetically distinct from yeasts. Vegetative growth by filamentous fungi is by hyphal elongation and carbon catabolism is obligatory aerobic. In the present invention, the filamentous fungal parent<br><br>
cell may be a cell of a species of, but not limited to, Trichodenna, (e.g., Trichoderma reesei, the asexual morph of Hypocreajecorina, T. longibrachiatum, Trichoderma viride, Trichoderma koningii, Trichoderma harz.ia.nuni); Penicillium sp., Humicola sp. (e.g., H insolens, H. lanuginosa and H. grisea); Chrysosporium sp. (e.g., C. lucknowense), Gliocladium sp., Aspergillus sp. (e.g., A. oryzae, A. niger, A. nidulans, and A awamori),<br>
Fusarium sp.,(e.g. F. graminum and F. venenatum), Neurospora sp., Hypocrea sp.,<br>
i	<br>
Mucor, and Emericella sp. (See also, Innis et al., (1985) Sci. 228:21 -26). The term "Trichoderma" or "Trichoderma sp." refer to any fungal genus previously or currently classified as Trichoderma. In some embodiments, the host cell will be a genetically engineered host cell wherein native genes have been inactivated, for example by deletion. Where it is desired to obtain a fungal host cell having one or more inactivated genes known methods may be used (e.g. methods disclosed hi U.S. Patent No. 5,246,853, U.S. Patent No. 5,475,101 and WO92/06209). Gene inactivation may be accomplished by complete or partial deletion, by insertional inactivation or by any other means which renders a gene nonfunctional for its intended purpose (such that the gene is prevented from expression of a functional protein). Any gene from a Trichoderma sp. or other filamentous fungal host, which has been cloned can be deleted, hi some preferred embodiments, when the host cell is a Trichoderma cell and particularly a T. reesei host cells the cbhl, cbh2, egll and egl2 genes will be inactivated and preferably deleted. Particualrly preferred Trichoderma reesei host cells having quad-deleted proteins are set forth and described in USP 5,847,276 and WO 05/001036.<br>
Transformation of Host Cells<br>
Introduction of a DNA construct or vector into a host cell includes techniques such as transformation; electroporation; nuclear microinjection; transduction; transfection, (e.g., lipofection mediated and DEAE-Dextrin mediated transfection); incubation with calcium phosphate DNA precipitate; high velocity bombardment with DNA-coated microprojectiles; and protoplast fusion. General transformation techniques are known in the art (See, e.g., Ausubel et al., (1987), supra, chapter 9; and Sambrook (1989) supra, and Campbell etal, (1989) Curr. Genet. 16:53-56). The expression of heterologous protein in Trichoderma is described in USP 6,022,725; USP 6,268,328; Harkki et al. (1991); Enzyme Microb. Technol. 13:227-233; Harkki et al., (1989) Bio Technol. 7:596-<br><br>
603; EP 244,234; EP 215,594; and Nevalainen et al, "Tlie Molecular Biology of Trichvderma and its Application to the Expression of Both Homologous and Heterologous Genes", in MOLECULAR INDUSTRIAL MYCOLOGY, Eds. Leong and Berka, Marcel Dekker Inc., NY (1992) pp. 129 - 148). Reference is also made to Cao et al., (2000) Sci. 9:991 -1001 and EP 238 023 for transformation of Aspergillm strains and WO96/00787 for transformation of Fusarium strains.<br>
Preferably, genetically stable transformants are constructed with vector systems whereby the nucleic acid encoding the glucoamylase is stably integrated into a host strain chromosome. Transformants are then purified by known techniques, hi one nonlimiting example, stable transformants including an amdS marker are distinguished from unstable transformants by their faster growth rate and the formation of circular colonies with a smooth, rather than ragged outline on solid culture medium containing acetamide. Additionally, in some cases a further test of stability is conducted by growing the transformants on solid non-selective medium (i.e., NH4(SO2) (5 mg/mL) as a nitrogen source), harvesting spores from this culture medium and determining the percentage of these spores which subsequently germinate and grow on selective medium containing lOmM acetamide as a sole nitrogen source. Alternatively, other methods known in the art may be used to select transformants.<br>
In one specific embodiment, the preparation of Trichoderma sp. for transformation involves the preparation of protoplasts from fungal mycelia (See, Campbell et al., (1989) Curr, Genet. 16:53-56). Also agrobacterium tumefaciens-mediated transformation of filamentous fungi is known (See, de Groot et al., (1998) Nat. Biotechnol. 16:839 - 842).<br>
In some embodiments, the mycelia are obtained from germinated vegetative spores. The mycelia are treated with an enzyme that digests the cell wall resulting in protoplasts. The protoplasts are then protected by the presence of an osmotic stabilizer in the suspending medium. These stabilizers include sorbitol, mannitol, potassium chloride, magnesium sulfate and the like. Usually the concentration of these stabilizers varies between 0.8 M and 1.2 M. It is preferable to use about a 1.2 M solution of sorbitol in the suspension medium. Uptake of DNA into the host Trichoderma sp. strain is dependent upon the calcium ion concentration. Generally, between about 10 mM CaCla and 50 mM CaCl2 is used in an uptake solution. Reference is also made to USP 6,022,725 and USP 6,268,328 for transformation procedures used with filamentous fungal hosts.<br><br>
The present invention relates to methods of recombinantly producing the glucoamylase comprising expressing a polynucleotide encoding a glucoamylase of the invention in a filamentous fungal host cell and cultivating the host cell under conditions suitable for production of the glucoamylase and optionally recovering the glucoamylase.<br>
In the expression and production methods of the present invention the fungal cells are cultured under suitable conditions in shake flask cultivation, small scale or large scale fermentations (including continuous, batch and fed batch fermentations ) in laboratory or industrial fermentors, with suitable medium containing physiological-salts and nutrients (See, e.g., Pourquie, J. et al, BIOCHEMISTRY AND GENETICS OF CELLULOSE DEGRADATION, eds. Aubert, J. P. et al., Academic Press, pp. 71-86,1988 and Ilmen, M. et al, (1997) Appl. Environ, Microbiol. 63:1298-1306). Common commercially prepared media (e.g., Yeast Malt Extract (YM) broth, Luria Bertani (LB) broth and Sabouraud Dextrose (SD) broth) find use in the present invention: Preferred culture conditions for a given filamentous fungus are known hi the art and may be found in the scientific literature and/or from the source of the fungi such as the American Type Culture Collection and Fungal Genetics Stock Center. In cases where a glucoamylase coding sequence is under the control of an inducible promoter, the inducing agent (e.g., a sugar, metal salt or antimicrobial), is added to the medium at a concentration effective to induce glucoamylase expression.<br>
hi some embodiments, in order to evaluate the expression of a glucoamylase by a cell line that has been transformed with a polynucleotide encoding a glucoamylase encompassed by the invention, assays are carried out at the protein level, the RNA level and/or by use of functional bioassays particular to glucoamylase activity and/or production. Some of these assays include Northern blotting, dot blotting (DNA or RNA analysis), RT-PCR (reverse transcriptase polymerase chain reaction), or in situ hybridization, using an appropriately labeled probe (based on the nucleic acid coding sequence) and conventional Southern blotting and autoradiography.<br>
hi addition, the production and/or expression of a glucoamylase may be measured in a sample directly, for example, by assays directly measuring reducing sugars such as glucose in the culture medium and by assays for measuring glucoamylase activity, expression and/or production. In particular glucoamylase activity may be assayed by the 3,5-dinitrosalicylic acid (DNS) method (See, Goto et al., (1994) Biosci. Biotechnol.<br><br>
Biochem. 58:49 - 54). In additional embodiments, protein expression, is evaluated by immunological methods, such as immunohistochemical staining of cells, tissue sections or immunoassay of tissue culture medium, (e.g., by Western blot or ELISA). Such immunoassays can be used to qualitatively and quantitatively evaluate expression of a glucoamylase. The details of such methods are known to those of skill in the art and many reagents for practicing such methods are commercially available.<br>
The glucoamylases of the present invention may be recovered or purified from culture media by a variety of procedures known in the art including centrifugation, filtration extraction, precipitation and the like.<br>
Uses and Compositions<br>
The present invention is also directed to compositions comprising glucoamylases of the invention and methods of using the glucoamylases in industrial and commercial applications. Nonlimiting examples, which include the use of glucoamylases encompassed by the invention in industrial and commercial applications are briefly described below.<br>
The glucoamylases may be used in starch hydrolyzing and saccharifying compositions, cleaning and detergent compositions (e.g., laundry detergents, dish washing detergents, and hard surface cleaning compositions), and hi animal feed compositions. Further the glucoamylases may be used in baking applications, such as bread and cake production, brewing, healthcare, textile, environmental waste conversion processes, biopulp processing, and biomass conversion applications.<br>
In particular, the glucoamylases may be used for starch conversion processes, and particularly in the production of dextrose for fructose syrups, specialty sugars and in alcohol and other end-product (e.g. organic acid, ascorbic acid, and amino acids) production from fermentation of starch containing substrates (G.M.A van Beynum et al., Eds. (1985) STARCH CONVERSION TECHNOLOGY, Marcel Dekker Inc. NY). Dextrins produced using glucoamylase compositions of the invention may result in glucose yields of at least 80%, at least 85%, at least 90% and at least 95%. Production of alcohol from the fermentation of starch substrates using glucoamylases encompassed by the invention may include the production of fuel alcohol or portable alcohol.<br>
In one preferred embodiment, the glucoamylases of the invention will find use hi the hydrolysis of starch from various plant-based substrates, which are used for alcohol<br><br>
production. In some preferred embodiments, the plant-based substrates will include corn, wheat, barley, rye, milo, rice, sugar cane and combinations thereof. In some embodiments, the plant-based substrate will be fractionated plant material, for example a cereal grain such as corn, which is fractionated into components such as fiber, germ, protein and starch (endosperm) (USP 6,254,914 and USP 6,899,910). Methods of alcohol fermentations are described in THE ALCOHOL TEXTBOOK, A REFERENCE FOR THE BEVERAGE, FUEL AND INDUSTRIAL ALCOHOL INDUSTRIES, 3rd Ed., Eds K.A. Jacques et al., 1999, Nottingham University Press, UK. In certain preferred embodiments, the alcohol will be ethanol. In particular, alcohol fermentation production processes are characterized as wet milling or dry milling processes. In some embodiments, the glucoamylase will be used in a wet milling fermentation process and in other embodiments the glucoamylase will find use in a dry milling process.<br>
Dry grain milling involves a number of basic steps, which generally include: grinding, cooking, liquefaction, saccharification, fermentation and separation of liquid and solids to produce alcohol and other co-products. Plant material and particularly whole cereal grains, such as corn, wheat or rye are ground. In some cases the grain may be first fractionated into component parts. The ground plant material may be milled to obtain a coarse or fine particle. The ground plant material is mixed with liquid in a slurry tank. The slurry is subjected to high temperatures in a jet cooker along with liquefying enzymes (e.g. alpha amylases) to solubles and hydrolyze the starch in the cereal to dextrins. The mixture is cooled down and further treated with saccharifying enzymes, such as glucoamylases encompassed by the instant invention, to produce glucose. The mash containing glucose is then fermented for approximately 24 to 120 hours in the presence of fermentation microorganisms, such as ethanol producing microorganism and particularly yeast (Saccharomyces spp). The solids in the mash are separated from the liquid phase and alcohol such as ethanol and useful co-products such as distillers' grains are obtained.<br>
In some embodiments, the saccharification step and fermentation step are combined and the process is referred to as simultaneous saccharification and fermentation or simultaneous saccharification, yeast propagation and fermentation.<br>
In other embodiments, the cooking step or exposure of the starch containing substrate to temperatures above the gelatinization temperate of the starch in the substrate may be eliminated. These fermentation processes in some embodiments include milling of<br><br>
a cereal grain or fractionated grain and combining the ground cereal grain with liquid to form a slurry which is then mixed in a single vessel with a glucoamylase according to the invention and optionally other enzymes such as but not limited to alpha amylases, other glucoamylases and enzymes having granular starch hydrolyzing activity and yeast to produce ethanol and other co-products (USP 4,514,496, WO 04/081193 and WO 04/080923).<br>
In some embodiments, the invention pertains to a method of saccharifying a liquid starch solution, which comprises an enzymatic saccharification step using a glucoamylase of the invention.<br>
In some embodiments, an enzyme composition including a glucoamylase encompassed by the invention and obtained in culture media or recovered and purified from the culture medium will be optionally used in combination with any one or combination of the following enzymes - alpha amylases, proteases, pullulanases, isoamylases, cellulases, hernicellulases, xylanases, cyclodextrin glycotransferases, Upases, phytases, laccases, oxidases, esterases, cutinases, xylanases, granular starch hydrolyzing enzyme and other glucoamylases.<br>
In some particularly preferred compositions the glucoamylases of the invention will be combined with alpha amylases, such as fungal alpha amylases (e.g. Aspergillus sp.) or bacterial alpha amylases (e.g. Bacillus sp. such as B. stearothermophilus, B. amyloliquefaciens and B. licheniformis) and variants thereof. Li some embodiments the alpha amylase will be an alpha amylase having at least 90%, 93%, 95%, 96%, 97%, 98% and 99% sequence identity to the mature protein sequence of SEQ ID NO: 27. Commercially available alpha amylases contemplated for use in the compositions of the invention are known and include GZYME G997, SPEZYME FRED, SPEZYME EHTYL (Genencor International Inc.) and TERMAMYL 120-L and SUPRA (Novozymes, Biotech.).<br>
In other particularly preferred embodiments, the glucoamylases of the invention will be combined with other glucoamylases. In some embodiments, the glucoamylases of the invention will be combined with one or more glucoamylases derived from strains of Aspergillus or variants thereof, such as A. oryzae, A. niger (e.g., the mature protein sequence of Figure 20(A), A. kawachi, and A. awamori', glucoamylases derived from strains of Humicola or variants thereof, particualrly H. grisea, such as the glucoamylase<br><br>
having at least 90%, 93%, 95%, 96%, 97%, 98% and 99% sequence identity to SEQ ID NO: 3 disclosed in WO 05/052148; glucoamylases derived from strains of Talaromyces or variants thereof, particularly T, emersonir, and glucoamylases derived from strains of Athelia and particularly A. rolfsii.<br>
MATERIAL AND METHODS<br>
In the disclosure and experimental section which follows, the following abbreviations apply:<br>
TrGA (a Trichoderma reesei glucoamylase composition, the mature protein having the amino acid sequence of SEQ ID NO: 4); AlcAA (an Aspergillus kawochi alpha amylase composition having the mature protein of sequence SEQ ID NO: 27); AnGA (DISTILLASE comprising an Aspergillus niger GA (Genencor International Inc.,)); GA (glucoamylase); GAU (glucoamylase unit); AAU (alpha amylase unit); wt% (weight percent); °C (degrees Centigrade); rpm (revolutions per minute); H2O (water); dH2O (deionized water); dIH2O (deionized water, Milli-Q filtration); aa or AA (ammo acid); bp (base pair); kb (kilobase pair); kD or kDa (kilodaltons); g or gm (grams); µg (micrograms); mg (milligrams); µL (microliters); mland mL (milliliters); mm (millimeters); urn (micrometer); M (molar); mM (millimoiar); µM (micromolar); U (units); V (volts); MW (molecular weight); sec (seconds); min(s) (minute/minutes); hr(s) (hour/hours); DO (dissolved oxygen); and EtOH (ethanol).<br>
The following assays and methods are used in the examples provided below:<br>
1) GA Assay - Glucoamylase assay: Glucoamylase activity was measure using a well-known assay which is based on the ability of glucoamylase to catalyze the hydrolysis of p-nitrophenyl-alpha-D-glucopyranoside (pNPG) to glucose and p-nitrophenol. At an alkaline pH, the nitrophenol forms a yellow color that is proportional to glucoamylase activity and is monitored at 400nm and compared against an enzyme standard measured as a GAU (Elder, M. T. and Montgomery R. S., Glucoamylase activity in industrial enzyme preparations using colorimetric enzymatic method, Journal of AOAC International, vol. 78(2), 1995).<br><br>
One GAU is defined as the amount of enzyme that will produce 1 gm of reducing sugar calculated as glucose per hour from a soluble starch substrate (4% ds) at pH 4.2 and 60°C.<br>
2) Primers and PCR protocol for amplification of genes from Trichoderma/Hypocrea strains: <br>
(Table Removed)<br><br>
With respect to the PCR program, initial denaturation was 2 min, at 94°C for 1 cycle; denaturation 30 sec, at 94°C, annealing for 30 sec, at 55°C and extension for 2 min at 68°C for 30 cycles and a final extension step of 7 min at 68°C.<br>
3) Ethanol and carbohydrate determinations<br>
Ethanol and carbohydrate composition of the samples were determined using the HPLC method as described herein:<br><br>
a)	a 1.5 mL Eppendorf centrifuge tube was filled with fermentor beer and cooled on ice<br>
for 10 min;<br>
b)	the sample tube was centrifuged for 1 min in Eppendorf table top centrifuge;<br>
c)	a 0.5 mL sample of the supernatant was transferred to a test tube containing 0.05 mL of<br>
Kill solution (1.1N H2SO4) and allowed to stand for 5 min;<br>
d)	5.0 mL of water is added to the test tube sample and then filtered into a HPLC vial<br>
through 0.45 um Nylon Syringe Filter; and<br>
e)	run on HPLC.<br>
HPLC conditions:<br>
a)	Ethanol System: Column: Phenomenex Rezex Organic Acid Column (RHM-<br>
Monosaccharide) #OOH 0132-KO (Equivalent to Bio-Rad 87H); Column Temperature:<br>
60°C; Mobile Phase: 0.01 N H2SO4; Flow Rate: 0.6 mL/min; Detector: RI; and<br>
Injection Volume: 20µL.  	<br>
b)	Carbohydrate System: Column: Phenomenex Rezex Carbohydrate (RCM-<br>
Monosaccharide) #OOH-0130-KO (Equivalent to Bio-Rad 87H); Column Temperature:<br>
70°C ; Mobile Phase: Nanopure DIH2O; Flow Rate: 0.8 mL/min; Detector: RI;<br>
Injection Volume: 10 µL (3% DS material)<br>
The column separates based on the molecular weight of the saccharides, which are designated as DP-1 (monosaccharides); DP-2 (disaccharides); DP-3 (trisaccharides) and DP &gt; 3 (oligosaccharide sugars having a degree of polymerization greater than 3).<br>
4)	Residual starch iodine test: A sample of the beer (fermentation broth) was centrifuged<br>
in 2 ml plastic centrifuge tubes. The supernatant was decanted and the tube containing the<br>
pellet was placed in an ice bam. Several drops of 0.025N iodine solution (0.1N iodine<br>
from VWR Cat. No. VW3207-1 diluted 4X) was added to the pellet and mixed. A positive<br>
(+) starch shows a range of color from blue to purple and the intensity of color is directly<br>
proportional to the concentration of starch. A negative result (-) remains yellowish.<br>
5)	Determination of total starch content: The enzyme-enzyme starch liquefaction and<br>
saccharification process (dual enzyme method) was used to determine the total starch<br>
content. In a typical analysis, 2 g of the dry sample was taken in a 100 ml Kohlraucsh flask and 45 ml of MOPS buffer, pH 7.0 was added. The slurry was well stirred for 30 min. SPEZYME FRED (1:50 diluted in water), 1.0 ml was added and heated to boiling for 3-5 min. The flask was placed in an autoclave maintained at 121°C for 15 min. After autoclaving the flask was placed in a water bath at 95°C and 1 ml of 1:50 dilutes SPEZYME FRED was added and incubated for 45 min. The pH was adjusted to pH 4.2 and the temperature was reduced to 60°C. This was followed by addition of 20 ml acetate buffer, pH 4.2. Saccharification was carried out by adding 1.0 ml of 1:100 diluted OPTIDEX L-400 (Glucoamylase from Genencor International Inc.) and the incubation<br>
t<br>
was continued for 18 hr at 60°C. The enzyme reaction was terminated by heating at 95°C for 10 min. The total sugar composition was determined by HPLC analysis using glucose as a standard. The soluble starch hydrolysate from water extraction of a sample at room temperature without enzymatic treatment was subtracted from the total sugar.<br>
6) Total protein analysis: The total nitrogen (N) in the sample preparations was determined using the Kjeldhal method (American Assoc. Cereal Chemists (AACC), (1983), Methods 22B60 8th Ed. St Paul, MN). Protein content was calculated by 6.25 X total N.<br>
EXAMPLES<br>
The following examples are provided in order to demonstrate and further illustrate certain preferred embodiments and aspect of the present invention and are not to be construed as limiting the scope thereof.<br>
Example 1 - Isolation and Cloning of the TrGA<br>
Chromosomal DNA of Trichoderma reesei QM6a was isolated from mycelial mass of a liquid culture in Potato Dextrose Broth (Difco™ Cat. No. 254920) using the BIOIOI Fast Prep® System according to the method described by the supplier (Qbiogene). The DNA was purified using a Quick Spin column (Qiagen art No. 28106). The glucoamylase gene was isolated using primers with GA-specific sequences, NSP232R (SEQ ID NO: 24) and NSP233F (SEQ ID NO: 25) designed according to the predicted nucleotide sequence in the Trichoderma reesei genome database of Department of Energy<br>
 (DOE) Joint Genome Institute. The primers were flanked at the 5'-end by Gateway ® attB sequences (Invitrogen). T. reesei QM6a chromosomal DNA was used as template.<br>
The PCR mix contained the following components: Forward primer (lOµM) 4µL; Reverse primer (lOµM) 4µL; template DNA (500ng/µL) 1µL; dNTPmix (lOmM) µL; lOx Cx buffer 10µL and PfuTurbo® Cx Hotstart DNA polymerase 0.5µL (Stratagen Cat. No. 600410). Deionized water was added up to a total volume of lOOµL.<br>
The PCR protocol was as follows: Initial denaturation for 30 sec. at 98°C, denaturation /annealing and extension in 30 cycles of 10 sec at 98°C; 30 sec at 68°C; 45 sec at 72°C , respectively, and a final extension step of 10 min at 72°C.<br>
The PCR fragments were analyzed by electrophoresis hi 1% agarose. Fragments of the expected size were isolated using the Gel-Extraction Purification Kit (Qiagene Cat. no. 28706). The PCR fragments were cloned into the Gateway ® Entry vector pDONR201 and transformed into E.coli DH5alpha Max Efficiency cells (Invitrogen Cat.No. 18258012). The nucleotide sequence of the inserted DNA was determined, from which the genomic DNA sequence of the TrGA gene was deduced (Fig. 1 (SEQ ID NO: 1)).<br>
Example 2 - Transformation of T. reesei and Fermentation/ Expression of the TrGA<br>
Vector DNA containing the correct GA gene sequence was recombined into the T. reesei expression vector pTrex3g (Fig. 17).<br>
The vector pTrex3g is based on the E. coli vector pSLl 180 (Pharmacia Inc., Piscataway, NJ) which is a pUCl 18 phagemid based vector (Brosius, J. (1989), DNA 8:759) with an extended multiple cloning site containing 64 hexamer restriction enzyme recognition sequences. It was designed as a Gateway destination vector (Hartley et al., (2000) Genome Research 10:1788 -1795) to allow insertion using Gateway technology (Invitrogen) of any desired open reading frame between the promoter and terminator regions of the" T. reesei cbhl gene. It also contains the Aspergillus nidulans amdS gene for use as a selective marker hi transformation of T. reesei. The details of the pTrex3g vector are as follows (Figure 17A). The vector is 10.3 kb hi size. Inserted into the polylinker region of pSLl 180 are the following segments of DNA: a) A 2.2 bp segment of DNA from the promoter region of the T. reesei cbhl gene; b) the 1.7 kb Gateway reading frame A cassette acquired from Invitrogen that includes the attRl and attR2 recombination sites<br>
at either end flanking the chloramphenicol resistance gene (CmR) and the ccdB gene; c) a 336 bp segment of DNA from the terminator region of the T. reesei cbhl gene; and d) a 2.7kb fragment of DNA containing the Aspergillus nidulans cundS gene with its native promoter and terminator regions.<br>
The expression vector containing the T. reesei GA gene, pNSP23 (Fig. 17) was transformed into a T. reesei host strain derived from RL-P37 (IA52) and having various gene deletions (∆ cbhl, ∆cbh2, ∆egl, ∆eg2) using particle bombardment by the PDS-1000/Helium System (BioRad Cat. No. 165-02257). The protocol is outlined below, and reference is also made to examples 6 and 11 of WO 05/001036.<br>
A suspension of spores (approximately 5 xlO spores/ml) from a quad deleted strain of T. reesei was prepared. 100 ul — 200 ul of spore suspension was spread onto the center of plates of Minimal Medium (MM) acetamide medium. (MM acetamide medium had the following compositions: 0.6 g/L acetamide; 1.68 g/LCsCl; 20 g/L glucose; 20 g/L KH2PO4; 0.6 g/L CaCl2 2H2O; 1 ml/L 1000X trace elements solution; 20 g/L Noble agar; and pH 5.5. 1000X trace elements solution contained 5.0 g/L FeSO4 7H2O; 1.6 g/L MnSO4; 1.4 g/L ZnSO4 7H2O and 1.0 g/L CoCl2 6H2O. The spore suspension was allowed to dry on the surface of the MM acetamide medium.<br>
Transformation followed the manufacturers instruction. Briefly, 60 mg of M10 tungsten particles were placed in a microcentrifuge tube. 1 mL of ethanol was added and allowed to stand for 15 seconds. The particles were centrifuged at 15,000 rpm for 15 seconds. The ethanol was removed and die particles were washed three times with sterile dH2O before 250 uL of 50% (v/v) sterile glycerol was added. 25 ul of tungsten particle suspension was placed into a microtrifuge tube. While continuously vortexing, the following were added: 5 ul (100 - 200 ng/ul) of plasmid DNA, 25 ul of 2.5M CaCl2 and 10 ul of 0.1M spermidine. The particles were centrifuged for 3 seconds. The supernatant was removed and the particles were washed with 200 ul of 100% ethanol and centrifuged for 3 seconds. The supernatant was removed, 24 uL 100% ethanol was added and mixed by pipetting, then 8 ul aliquots of particles were removed and placed onto the center of macrocarrier disks that were held in a desiccator. Once the tungsten/DNA solution had dried the macrocarrier disk was placed in the bombardment chamber along with the plate of MM acetamide with spores and the bombardment process was performed according to the manufacturers instructions. After bombardment of the plated spores with the<br>
tungsten/DNA particles, the plates were incubated at 30°C. Transformed colonies were transferred to fresh plates of MM acetamide medium and incubated at 30°C.<br>
Example 3 - Demonstration of GA activity from the expressed TrGA in transformed cells<br>
After 5 days of growth on MM acetamide plates transformants displaying stable morphology were inoculated into 250 ml shake flasks containing 30 ml of Proflo medium. (Proflo medium contained: 30 g/L a-lactose; 6.5 g/L (NHLOaSO4; 2 g/L KH2PO4; 0.3 g/L MgSO4 7H2O; 0.2 g/L CaCl2; 1 ml/L 1000X trace element salt solution; 2 ml/L 10% Tween 80; 22.5 g/L ProFlo cottonseed flour (Traders protein, Memphis, TN); 0.72 g/L CaCO3.   After two days growth at 28 C and 140rpm, 10% of the Proflo culture was transferred to a 250 ml shake flask containing 30 ml of Lactose Defined Media. The composition of the Lactose defined Media was as follows 5 g/L (NILOaSO4; 33 g/L PIPPS buffers; 9 g/L casamino acids; 4.5 g/L KH2PO4; 1.0 g/L MgSO4 7H2O; 5 ml/L Mazu DF60-P antifoam (Mazur Chemicals, IL); 1000X trace element solution; pH 5.5; 40 ml/L of 40% (w/v) lactose solution was added to the medium after sterilization. The Lactose Defined medium shake flasks were incubated at 28°C, 140 rpm for 4 — 5 days.<br>
Mycelium was removed by centrifugation and the supernatant was analyzed for total protein (BCA Protein Assay Kit, Pierce Cat. No. 23225) and GA activity using pNPG as substrate (Sigma N-1377).<br>
Samples of the culture supernatant were mixed with appropriate volume of 2X sample laoding buffer with reducing agent and the protein profile was determined by sodium dodecyl sulfate-polyacrylamide gel electrophoresis (SDS-PAGE) using NuPAGE® Novex 10% Bis-Tris Gel with MES SDS Running Buffer. The gels were stained with SimplyBlue™ SafeStain (Invitrogen, Carlsbad, CA, USA).<br>
On SDS-PAGE analysis a protein band that was not observed in supernatant from a quad delete strain was observed in the supernatant of some transformants with the pTrexSg vector containing the glucoamylase open reading frame (Figure 16). This new protein band had an apparent molecular weight of approximately 64 kDa. This result confirms that TrGA is secreted into the medium.<br><br>
Example 4 - Biochemical characterization of the GA gene product GA producing transformants were grown at 4-L scale. The culture filtrate was concentrated using an ultra filtration unit with a nominal molecular weight limit of 10,000 Da (Pall Omega Centramate OSOlOclO). The crude enzyme preparation was purified by a 2-step procedure using an AKTA explorer 100 FPLC System (Amersham Biosciences). A HiPrep 16/10 FF Q-Sepharose column (Amersham BioSciences Cat. No. 17-5190-01) was equilibrated with 25 mM Tris pH 8.0 and the protein was eluted from the column with 100 mM NaCl in 25 mM Tris pH 8.0. A second affinity chromatography step was performed using Cbind 200 resin (Novagen Cat. No. 701212-3) and 50 mM Tris pH 7.0 containing 500 mM NaCl as elution buffer (Fig. 16). The N-terminus of the gene product (Ser-Val-Asp-Asp-Phe-De) (SEQ ID NO: 38) was determined by Edman degradation (Edman, P. (1956) Acta Chem Scand 10:761-768).<br>
The pH and temperature profiles of the glucoamylase activity of the gene product were determined using 4-nitrophenyl-a-D-glucopyranoside as substrate (Elder, M. T. and Montgomery R. S., Glucoamylase activity in industrial enzyme preparations using colorimetric enzymatic method; Collaborative study Journal of AOAC International, vol. 78(2), 1995) (Fig. 18).<br>
Example 5 - Isolation /cloning of glucoamvlase homologs from strains in the Trichoderma/Hypocrea family cluster<br>
Chromosomal DNA preparations of the strains (GA102) -Hypocrea citrina var. americana (CBS976.69); (GA104) - Hypocrea vinosa (CBS960.68); (GA105)-Trichoderma sp; (GA107) - Hypocrea gelatinosa (CBS254.62); GA108 - Hypocrea orientalis (ATCC 90550); (GA109) - Trichodenna konilangbra; (GA103) - Trichodenna harzianwn (CBS433.95); (GA113) - Trichoderma sp.; (GA124) - Trichoderma longibrachiatum; (GA127) - Trichoderma asperellum (ATCC 28020); and (GA128) -Trichodenna strictipilis (CBS 347.93) were isolated as described in example 1. Full-length GA genes were cloned as described hi example 1 using the TrGA-gene specific primers NSP231F (SEQ ID NO: 23) and NSP232R (SEQ ID NO: 24). The nucleotide sequences of the strains are disclosed in Fig. 4 for GA102 (SEQ ID NO: 5); Fig. 5 for GA104 (SEQ ID NO: 7); Fig. 6 for GA105 (SEQ ID NO: 9); Fig. 7 for GA107 (SEQ ID NO: 11); Fig. 8 for GA108 (SEQ ID NO: 13); Fig. 9 for GA109 (SEQ ID NO: 15); Fig. 10 for GA113 (SEQ ID NO: 28); Fig. 11 for GA103 (SEQ ID NO: 30); Fig. 12 for GA124 (SEQ ID NO: 32); Fig. 13 for GA127 (SEQ ID NO: 34) and Fig. 14 for GA128 (SEQ ID NO: 36). The corresponding amino acid sequences are illustrated in Fig. 15. Table 2 sets forth the percent identity of the amino acid sequences of the mature protein of T. reesei glucoamylase (Fig. 3B, SEQ ID NO: 4) with the glucoamylase homologs from the Hypocrea/Trichoderma cluster.<br>
Table 2 - % Identity of GA homologs from the Hypocrea/Trichoderma cluster<br><br>
(Table Removed)<br>
T. reesei strains over-expressing GA were obtained as described in example 2. Crude enzyme preparations were obtained as described in example 3 and Fig. 19 illustrates the gels obtained for some of the homologs. Table 3 sets forth the glucoamylase activity of some of the homologs.<br><br>
Table 3<br><br>
(Table Removed)<br>
Example 6 - Glucose Production Using TrGA<br>
A 32% DS slurry of Cargill bag starch was made up with reverse osmosis water. The pH of the slurry was adjusted to pH 5.8. The slurry was filtered through a 100-mesh screen and dosed at 4.0 AAU/g ds using SPEZYME® ETHYL, (Genencor International, Inc.). The slurry was then jetted at 107.3°C for 5 min (primary liquefaction). Enzyme activity is determined by the rate of starch hydrolysis, as reflected in the rate of decrease in iodine-staining capacity. One AAU unit of bacterial alpha amylase activity is the amount of enzyme required to hydrolyze 10 mg of starch per minute under specified conditions. After primary liquefaction, the liquefact was collected and placed in a 95°C water bath for 120 min (Secondary liquefaction). Samples were taken at 30,60,90 and 120 min and checked for DE by using the standard Schoorls reducing sugar method from the Corn Refiners Association. The liquefact was aliquoted in 100-g quantities into screw cap jars, the pH was adjusted to pH 4.5 and equilibrated to 60°C for 15 minutes prior to dosing. The TrGA enzyme was diluted so as to add 0.2 rnls of diluted enzyme to the jar at 0.22 GAU/g ds. After dosing, the liquefact was aliquoted into 7 screw cap tubes, each containing approximately 10 mis of material and returned to the designated temperature. Tubes were removed at selected time intervals (18,24,30,42, 50 and 55 hours) and analyzed by HPLC Carbohydrate System: Column: Phenomenex Rezex Carbohydrate (RCM- Monosaccharide) #OOH-0130-KO (Equivalent to Bio-Rad 87H); Column<br><br>
Temperature: 70°C; Mobile Phase: Nanopure DIH2O; Flow Rate: 0.8 mL/min; Detector: RI; Injection Vommn: lOuL (3% DS material) for sugar composition.<br>
Table 4- Production of Glucose from cornstarch using TrGA<br><br>
(Table Removed)<br>
Example 7 - Ethanol Production using TrGA in a Simultaneous Saccharification and<br>
Fermentation (SSF) Process<br>
A sample of corn mash liquefact from a local ethanol producer was obtained and diluted to 29% DS using thin stillage. The pH of the slurry was adjusted to pH 4.3 using 6 N sulphuric acid. A 300 g aliquot of the mash was placed into a 31°C water bath and allowed to equilibrate. TrGA was added to the sample (0.4 GAU/g ds, which is equal to 1.08 kg/MT ds). After enzyme addition, 1 ml of a 15 g in 45 ml DI water solution of Red Star Red yeast (Lesaffre yeast Corp. Milwaukee, WI) was added to each sample. Samples were taken at 18,26,41 and 53 hours and analyzed by HPLC Column: Phenomenex Rezex organic Acid Column (RHM- Monosaccharide) #OOH-0132-KO (Equivalent to Bio-Rad 87H); Column Temperature: 60°C; Mobile Phase: 0.01 NH2SO4; Flow Rate: 0.6 mL/min; Detector: RI; Injection Volumn: 20uL.<br>
Table 5 - Production of Ethanol by TrGA (0.4 GAU/g) in a SSF Process<br><br>
(Table Removed)<br><br>
Example 8 - A Non-cook Process for Ethanol Production using TrGA. In general a 33% slurry of corn flour (Azure Standard Farms) was prepared in DI to which 400 ppm urea was added. The pH was adjusted to 4.5. Fermentations were conducted in 125 ml flasks containing lOOg mash and various treatments of GAU/g TrGA. A 20% slurry of Fali dry yeast in water was prepared and mixed with a 32°C water bath one hour prior to inoculating the fermenters by adding 0.2 ml of the yeast slurry. The flasks were placed in a 32 °C water bath and the mash mix gently. During the fermentations samples were icmoved for HPLC analysis. The fermentations were terminated after 72 hours. Production of compounds including sugars, lactic acid, glycerol and ethanol at various sampling intervals is shown below in various tables. The mash was dried at 60°C to obtain the DDGS, and the starch content of the DDGS was determined by the dual enzyme method.<br>
A.       All conditions were as described above: the treatment included 1.2 GAU/g TrGA.<br>
Table 6 - Ethanol Production<br>
(Table Removed)<br>
B.        All conditions were as described above: the treatments included 0.75 GAU/g GA107 and 0.75GAU/g GA104<br>
Table 7 - Ethanol Production<br><br>
(Table Removed)<br>
C.        All conditions were as described above: the treatments included a) A. niger GA 0.75 GAU/g + 2.25 SSU AkAA and b) TrGA 0.75 GAU/g + 2.25 SSU AkAA. The residua] starch for AnGA + AkAA treatment was determined to be 5.26% and the residual starch for TrGA + AkAA treatment was determined to be 8.71%.<br>
The measurement of alpha amylase activity for AkAA is based on the degree of hydrolysis of soluble potato starch substrate (4% ds) by an aliquot of the enzyme sample at pH 4.5, 50°C. The reducing sugar content is measured using the DNS method as described in Miller, G. L. (1959) Anal. Chem. 31:426 - 428. One unit of the enzyme activity (SSU, soluble starch unit) is equivalent to the reducing power of 1mg of glucose released per minute at the specific incubation conditions.<br><br>
Table 8 - Ethanol Production<br>
(Table Removed)<br>
D.       All conditions were as described above: the treatments included a) TrGA 0.695 3AU/g + 2.25 SSU AkAA and b) TrGA 0.695 GAU/g + 2.25 SSU AKAA + 2 ASPU/g Pullulanase. One acid stable pullulanase unit (ASPU) is defined as the amount of enzyme which liberates one equivalent reducing potential as glucose per minute from pullulan at pH 4.5 and a temperature of 60°C.<br>
Table 9- Ethanol production<br>
(Table Removed)<br>
 E. All conditions were as described above: the treatments included TrGA 0.695 GAU/g and the following AkAA treatments: a) 3 SSU AkAA; b) 10 SSU AkAA and c) 30 SSU AkAA.<br><br>
Table 10<br>
(Table Removed)<br><br><br><br><br><br>
CLAIMS<br>
1.	An isolated DNA sequence encoding an enzyme having glucoamylase activity,<br>
wherein the enzyme has an amino acid sequence that has at least 80% identity with the<br>
glucoamylase of SEQ ID NO: 4.<br><br>
2.	The isolated DNA sequence of claim 1, wherein the enzyme has an amino acid<br>
sequence that has at least 85% identity with the glucoamylase of SEQ ID NO: 4.<br>
3.	The isolated DNA sequence of claim 2, wherein the enzyme has an amino acid<br>
sequence that has at least 90% identity with the glucoamylase of SEQ ID NO: 4.<br>
4.	The isolated DNA sequence of claim 3, wherein the enzyme has an amino acid<br>
sequence that has at least 95% identity with the glucoamylase of SEQ ID NO: 4.<br>
5.	The isolated DNA sequence of claim 4, wherein the enzyme has an amino acid<br>
sequence that has at least 99% identity with the glucoamylase of SEQ ID NO: 4.<br>
6.	The isolated DNA sequence of claim 1 characterized by SEQ ID NO: 1.<br>
7.	The isolated DNA sequence of claim 1 characterized by SEQ ID NO: 2.<br>
8.	The isolated DNA sequence of claim 1, wherein the enzyme is derived from a<br>
filamentous fungus.<br>
9.	The isolated DNA sequence of claim 8, wherein the filamentous fungus is a<br>
Trichoderma or Hypocrea strain.<br>
10.   The isolated DNA sequence of claim 9, wherein the filamentous fungus is a strain of Trichoderma reesei.<br><br>
11.	An isolated DNA sequence encoding an enzyme having glucoamylase<br>
activity, wherein the enzyme has at least 95% sequence identity to any one of the<br>
sequences shown in SEQ ID NOs: 17,18,19,20, 21,22,43,44,45,46 or 47.<br>
12.	The isolated DNA sequence of claim 11, wherein the enzyme has an amino<br>
acid sequence that has at least 95% sequence identity with the glucoamylase of SEQ ID<br>
NO: 20.<br>
13.	The isolated DNA sequence of claim 12, wherein the enzyme has an amino<br>
acid sequence that has at least 99% sequence identity with the glucoamylase of SEQ ID<br>
NO: 20.<br>
14.	An isolated DNA sequence encoding an enzyme having glucoamylase<br>
activity, wherein the enzyme has at least 90% sequence identity to any one of the<br>
following sequences<br><br>
a)	amino acid residue positions 1 to 453 of SEQ ID NO: 17,<br>
b)	amino acid residue positions 1 to 452 of SEQ ID NO: 18,<br>
c)	amino acid residue positions 1 to 454 of SEQ ID NO: 19,<br>
d)	amino acid residue positions 1 to 452 of SEQ ID NO: 20,<br>
e)	amino acid residue positions 1 to 453 of SEQ ID NO: 21,<br>
f)	amino acid residue positions 1 to 453 of SEQ ID NO: 22,<br>
g)	amino acid residue positions 1 to 452 of SEQ ED NO: 43,<br>
h)	amino acid residue positions 1 to 452 of SEQ ID NO: 44,<br>
i)	amino acid residue positions 1 to 453 of SEQ ID NO: 45,<br>
j)         amino acid residue positions 1 to 452 of SEQ ID NO: 46, and    <br>
k)    amino acid residue positions 1 to 452 SEQ ID NO: 47.<br>
15.	A vector comprising the DNA of claim 1,11 or 14.<br>
16.	A host cell transformed with the vector of claim 15.<br><br>
17.	The host cell of claim 16, wherein the host cell is obtained from a strain of a<br>
filamentous fungus.<br>
18.	The host cell of claim 17, wherein the filamentous fungal strain is<br>
Trichoderma.<br>
19.	The host cell of claim 18, wherein the Trichoderma is T. reesei.<br><br>
20.	A culture medium from the fermentation of a filamentous fungal host cell<br>
transformed with a nucleic acid sequence coding for a polypeptide having glucoamylase<br>
activity and at least 80% sequence identity to SEQ ED NO: 4.<br><br>
21.	The culture medium of claim 20, wherein the filamentous fungal host cell is a<br>
Trichoderma cell.<br>
22.	An isolated enzyme having glucoamylase activity and at least 80% sequence<br>
identity with SEQ ID NO: 4.<br>
23.	The isolated enzyme of claim 22, wherein the amino acid sequence is<br>
selected from the group of any one of SEQ ID NOS: 17,18,19,20,21,22,43,44,45,46<br>
and 47.<br>
24.	The isolated enzyme of claim 22, wherein the enzyme has an amino acid<br>
sequence with at least 85% identity to SEQ ID NO: 4.<br>
25.	The isolated enzyme of claim 24, wherein the enzyme has an amino acid<br>
sequence with at least 90% identity to SEQ ID NO: 4.<br>
26.	The isolated enzyme of claim 25, wherein the enzyme has an amino acid<br>
sequence with at least 95% identity to SEQ ID NO: 4.<br><br>
27.	The isolated enzyme of claim 26, wherein the enzyme' has an amino acid<br>
sequence with at least 97% identity to SEQ ID NO: 4.<br>
28.	The isolated enzyme of claim 22, wherein the enzyme has an amino acid<br>
sequence with at least 95% identity to SEQ ID NO: 20.<br>
29.	Ail isolated amino acid sequence characterized as a biologically functional<br>
fragment of a sequence having at least 90% identity to SEQ ID NO: 4.<br>
30.	The isolated amino acid sequence of claim 29 comprising amino acid residue<br>
positions 1 to 453 of SEQ ID NO: 4.<br>
31.	The isolated amino acid sequence of claim 29 comprising amino acid residue<br>
positions 34 to 486 of SEQ ID NO: 4.<br>
32.	A method of recombinantly producing a glucoamylase, comprising the step<br>
of expressing a polynucleotide encoding a polypeptide having glucoamylase activity in a<br>
filamentous fungal host cell, wherein the polypeptide comprises the amino acid sequence<br>
of claims 22 or 29.<br>
33.	The method according to claim 32, wherein the polypeptide comprises a<br>
signal peptide.<br>
34.	The method according to claim 33, wherein the signal peptide comprises an<br>
amino acid sequence which is at least 95% identical to the sequence shown in positions 1<br>
to 20 of SEQ ID NO: 3.<br>
35.	The method according to claim 32, wherein the filamentous fungal host cell<br>
is a Trichoderma cell.<br>
36.	The method according to claim 35, wherein the Trichoderma host is a 71<br>
reesei<br>
37.	The method according to claim 32, further comprising recovering the<br>
produced glucoamylase.<br>
38.	An enzyme composition comprising a glucoamylase having the amino acid<br>
sequence of claim 22 or 29.<br>
39.	The enzyme composition of claim 38, wherein the glucoamylase has an<br>
amino acid sequence that is at least 90% identical to SEQ ID NO: 4.<br>
40.	The enzyme composition of claim 38, wherein the composition is used in a<br>
baking application.<br><br>
41.	The enzyme composition of claim 38, wherein the composition is an animal<br>
feed composition.<br>
42.	The enzyme composition of claim 38, wherein the composition is used in a<br>
process to hydrolyze granular starch.<br>
43.	The enzyme composition of claim 38, wherein the composition is a detergent<br>
composition.<br>
44.	The enzyme composition of claim 38 further comprising one or more<br>
additional enzymes.<br>
45.	The enzyme composition of claim 44, wherein the one or more additional<br>
enzymes are selected from the group of alpha amylases, glucoamylases, granular starch<br>
hydrolyzing enzymes, proteases, cellulases, pullulanases and combinations thereof.<br>
46.	The enzyme composition of claim 45, wherein the additional enzyme is an<br>
alpha amylase.<br><br>
47.	The enzyme composition of claim 45, wherein the additional enzyme is a<br>
granular starch hydrolyzing enzyme.<br>
48.	The enzyme composition of claim 45, wherein the additional enzyme is a<br>
protease.<br>
49.	A method of hydrolyzing starchcomprising treating a starch containing<br>
substrate with an enzyme having at least 80% sequence identity to the sequence of SEQ<br>
ID NO: 4 or a biologically active fragment thereof.<br>
50.  A method for producing a fermentation product from a granular starch containing substrate comprising<br>
a)	contacting a granular starch containing substrate with a glucoamylase having<br>
at least 80% sequence identity to SEQ ID NO: 4 or a biologically functional fragment<br>
thereof at a temperature below the gelatinization temperature, at a pH of about 4 to<br>
7.0 for a period of time to produce a composition comprising glucose,<br>
b)	contacting the glucose with a fermentation organism under suitable<br>
fermentation conditions to produce a fermentation product.<br><br>
51.	The method of claim 50, wherein the fermentation product is ethanol.<br>
52.	A method for saccharifying liquefied starch comprising treating liquefied<br>
starch with a polypeptide having glucoamylase activity, wherein the polypeptide has at<br>
least 80% sequence identity to the sequence of SEQ ID NO: 4 or a biologically functional<br>
fragment thereof and obtaining a composition which includes at least 80% glucose.<br><br><br></td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/viewdoc.aspx?id=+nGYuK00ir1aJ4KtHcvuHg==&amp;amp;loc=+mN2fYxnTC4l0fUd8W4CAA==" target="_blank" style="word-wrap:break-word;">http://ipindiaonline.gov.in/patentsearch/GrantedSearch/viewdoc.aspx?id=+nGYuK00ir1aJ4KtHcvuHg==&amp;amp;loc=+mN2fYxnTC4l0fUd8W4CAA==</a></p>
		<br>
		<div class="pull-left">
			<a href="269275-electric-arc-furnace.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="269278-lightweight-thermoplastic-sheets-including-reinforcing-skins.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>269277</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>3631/DELNP/2007</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>42/2015</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>16-Oct-2015</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>13-Oct-2015</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>15-May-2007</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>GENENCOR INTERNATIONAL, INC.</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td>925 PAGE MILL ROAD, PALO ALTO, CALIFORNIA 94304 USA</td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>NEEFE-KRUITHOF PAULIEN</td>
											<td>GENENCOR INTERNATIONAL, INC.,925 PAGE MILL ROAD, PALO ALTO, CALIFORNIA 94304 USA</td>
										</tr>
										<tr>
											<td>2</td>
											<td>PILGRIM CRAIG E.</td>
											<td>GENENCOR INTERNATIONAL, INC.,925 PAGE MILL ROAD, PALO ALTO, CALIFORNIA 94304 USA</td>
										</tr>
										<tr>
											<td>3</td>
											<td>WARD DONALD E.</td>
											<td>GENENCOR INTERNATIONAL, INC.,925 PAGE MILL ROAD, PALO ALTO, CALIFORNIA 94304 USA</td>
										</tr>
										<tr>
											<td>4</td>
											<td>VAN SOLINGEN PIETER</td>
											<td>GENENCOR INTERNATIONAL, INC.,925 PAGE MILL ROAD, PALO ALTO, CALIFORNIA 94304 USA</td>
										</tr>
										<tr>
											<td>5</td>
											<td>DUNN-COLEMAN NIGEL</td>
											<td>GENENCOR INTERNATIONAL, INC.,925 PAGE MILL ROAD, PALO ALTO, CALIFORNIA 94304 USA</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>C12N 9/34</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>PCT/US2005/036307</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td>2005-10-07</td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td>PCT/US2004/040040</td>
									<td>2004-11-30</td>
								    <td>U.S.A.</td>
								</tr>
								<tr>
									<td>2</td>
									<td>60/647,925</td>
									<td>2005-01-28</td>
								    <td>U.S.A.</td>
								</tr>
								<tr>
									<td>3</td>
									<td>PCT/US2005/018214</td>
									<td>2005-05-24</td>
								    <td>U.S.A.</td>
								</tr>
								<tr>
									<td>4</td>
									<td>PCT/US2005/018212</td>
									<td>2005-05-24</td>
								    <td>U.S.A.</td>
								</tr>
								<tr>
									<td>5</td>
									<td>PCT/US2004/041276</td>
									<td>2004-12-09</td>
								    <td>U.S.A.</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/269277-trichoderma-reesei-glucoamylase-and-homologs-thereof by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 07:52:29 GMT -->
</html>
