<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/269400-method-for-amplification-of-a-specific-micro-rna-species by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 07:34:18 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 269400:METHOD FOR AMPLIFICATION OF A SPECIFIC MICRO-RNA SPECIES</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">METHOD FOR AMPLIFICATION OF A SPECIFIC MICRO-RNA SPECIES</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>The present invention relates to a method for amplification and quantification of microRNA molecules using quantitative reverse transcription polymerase chain reaction (qRT-PCR) technology. The method comprise the steps of (a) producing cDNA molecules complementary to microRNAs in a sample using polyadenylation and primer extension by reverse transcription; and (b) amplification and quantification of the cDNAs by qPCR using microRNA specific primer sets of forward and reverse primers containing LNA monomers.</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td>FIELD OF THE INVENTION<br><br>
The present invention relates to a method for amplification and quantification of small non-coding RNA molecules using quantitative reverse transcription polymerase chain reaction (qRT-PCR) technology.<br><br>
BACKGROUND OF THE INVENTION<br><br>
MicroRNAs are an abundant class of approximately 22-nucleotide-noncoding RNAs, which play important regulatory roles in animal, plant and virus development. The awareness of mlcroRNAs was initiated almost 15 years ago by the discovery of lin-4, which encode a small RNA involved in the timing and progression of the nematode in C. elegans life cycle and larval development (Lee et al. 1993 Cell 75:843-854, Wightman et al.1993 Cell 75:855-862), but it was only recently recognized that microRNAs form a major class of rlbo-regulaters that have broad regulatory functions In animals (Lagos-Qulntana et al. 2001 Science 294:853-858, Lau et al. 2001 Science 294(858-862, Lee and Ambroi. 2001 Science 294:862-864). Since then, a revolution In the study of microRNAs have taken place, and today the mIRBase database version 12,0 (littp://inlcmma.sanger.ac.uk/) Include 80(5 human microRNAs and the PubMed databaia (liU.p://www.ncbi.nlm.nlh.aov/pubiiiflti/) encompass 3900 mlcroRNA related articles, reflecting   the Interest and Importance of microRNAs.<br><br>
MicroRNAs are Involved In the regulation of gene expression at the posttranscrlptlonal level by degrading or blocking translation of messenger RNA targets, and It has been speculated that approximately 30 % of the human genome could be regulated by microRNAs. The importance of microRNAs is also obvious due to their Involvement In various cellular processes including development, growth and proliferation, apoptosis, differentiation, and various human diseases (http://www.mir2dlsease.org/) such as cancer and diabetes.<br><br>
The importance of microRNAs in cancer is highlighted in a recent article (Barbarotto et al 2008 Int. J. Cancer. 122:969-977), which summarizes the main paradigms for the miRNA involvement in human cancers: Thus, "(i) miRNAs are altered In every type of analyzed human cancer; (ii) miRNAs act as oncogenes and tumor suppressors; (iii) miRNAs alterations may cause cancer predisposition; (iv) miRNAs profiling is a new diagnostic tool for cancer patients and (v) mRNA profiling represents prognostic tools for cancer patients.". Accordingly, methods for expression profiling and quantification of microRNAs in cells and body-fluids from cancer patients are of great importance. To address this requirement, the present invention describes the development of a new robust and reliable qRT-PCR assay for microRNA measurements.<br><br>
Quantification of microRNAs by qRT-PCR procedures is very challenging due to the small size of microRNAs of only 21 to 25 nucleotides, which is the size of primers normally used for PCR.   Solutions  to  this   problem   have   been   published   in   Raymond   et  al.   RNA.   2005 Nov; 11(11): 1737-44, Gilad et al. PLoS ONE. 2008 Sep 5;3(9):e3148 and Sharbati-Tehrani et al. BMC Molecular Biology. 2008, 9:34.  Raymond et al. describes a qRT-PCR assay that Â® involves a gene-specific reverse transcription step followed by a SYBR     green qPCR step using a gene-specific forward primer containing locked nucleic acid (LNA) molecules and a universal reverse primer. Gilad et al. reports a qRT-PCR assay that Involves a polyadenylation step, an unspeclflc reverse transcription step, and a qPCR step Involving a gene-specific forward primer, a gene-speclflc TaqMan primer and a universal reverse primer. Sharbati-Tehrani et al. developed a qRT-PCR assay that involves a gene-speclflc reverse transcription <r step followed by a bybk green qpcr using gana-spaclflc forward primer and muniversal primers></r><br>
However, the existing techniques for quantification of microRNAs by qRT-PCR do not fulfil the present need for microRNA assays, which requires high specificity that allows discrimination between closely related microRNAs, high sensitivity, low background an a relatively simple procedure.<br><br>
The present Invention Is characterised by only one reverse transcription reaction for all microRNAs in a sample and furthermore provides an extremely sensitive PCR method with an unmatched specificity that can be used for accurate quantification of small RNA molecules such as microRNAs.<br><br>
SUMMARY OF THE INVENTION<br><br>
Establishing and understanding the microRNA dysregulation patterns associated with various human diseases such as cancers, call for novel, improved technologies for detection and quantification of microRNAs in human cells and body fluids. The present invention introduces a new highly sensitive and specific assay for this purpose.<br><br>
In one aspect, the present invention provides a procedure for amplifying and quantifying microRNA molecules in a sample: In the first step of the procedure complementary DNAs (cDNAs) of the microRNAs in a sample are produced by the concerted action of two enzymes in a single-tube reaction. Firstly, poly-A tails are added to the 3'-terminus of the microRNAs using a poly(A) polymerase, and secondly an extension primer is hybridized to the poly-A tail and the cDNAs are produced by a reverse transcriptase using the microRNA as template. The first step is unspeciflc and produces cDNAs of all microRNAs present In a sample. In the second step of the procedure specific cDNAs are amplified and quantified In a qPCR reaction using microRNA specific primer sets of forward and reverse primers containing LNA monomers.<br><br>
In another aspect, the Invention provides oligo nucleotide primers listed In Table 18 (SEQ ID NO 1 to SEQ ID NO J.2B),<br><br>
The primers of the Invention can be used for detecting mammalian microRNAs using the method of the Invention.<br><br>
In another aspect, the Invention provides kits for detection of mammalian microRNAs, the kits comprising a universal extension primer and microRNA-speclflc forward and reverse primers sets for quantification of at least one microRNA, a subset of microRNAs or all known microRNAs.<br><br>
The present invention is useful for reliable and specific quantitative microRNA assays, including assays for diagnosing and prognosing diseases such as cancer using single assays or high through-put applications on robot platforms. RNA containing samples extracted from various cell types from living organisms, such as mammals and plants and including virus infected cells may be analysed using the method of the invention.<br><br>
Although the present invention mainly aims at providing a method for quantification of microRNAs the method can be used for detection and/or quantification of all types of RNA in particular all types of small noncoding RNAs.<br><br>
FIGURES<br><br>
FIGURE 1 shows the steps involved in specific qRT-PCR of the present invention. To illustrate the principle the qRT-PCR of a microRNA serves as an example, the RNA to be analysed by the method may as well be any other small RNA molecule or even a mRNA. Step 1 is a one-tube-reaction for all microRNAs present in a sample. Step 2 is a microRNA specific qPCR using forward and reverse primer pairs for a specific microRNA. An oval indicate insertion of LNAs in forward and reverse primers. When the method is carried out in practice the miRNAs present In a sample are firstly poly-A-tailed using a poly(A) polymerase, which adds adenine residues to the 3'-end of RNA molecules. Secondly, an extension primer, which has a poly-T-core nucleotide sequence, a 3'-and VN-degenerate motif and a S'-and tall, Is annealed to the poly-A-talled mIRNA through hybridisation with the VN-poly-T-sequance of the extension primer, (N-C, Q, A and I; V- C, (3, and A). This primer may ba referred to as the Universal RT primer. Subsequently, the extension primer Is extended In a reverse transcription reaction using the mIRNA as template. All of these reaction are performed In a one-tube reaction. The resulting primary extension product Is composed of the extension primer and the newly synthesized DNA, which Is cDNA complementary to all the mRNAs In the sample. In the next step a mlRNA-specific PCR Is carried out. A mRNA-speclflc forward primer Is annealed to 3'-end of the newly synthesized cDNA and the upper-strand synthesis Is carried out by extending the forward primer In a DNA-polymerization reaction using the primary extension product as template. A mlRNA-speclflc reverse primer composed of a mlRNA-speclflc 3'-end sequence, a poly-T-stretch and a 5'-end tail is then hybridized to the upper-strand and the lower-strand is synthesized by extension of the reverse primer. Various demonstrations of the method of the invention are described in the EXAMPLES section.<br><br>
FIGURE 2 shows amplification of hsa-miR-197 (Example 1) with the forward primer 5' ttrrCaccaccttctcca (SEQ ID NO 1) and the reverse primer 5'-ctttttttttttttttGctgggt (SEQ ID NO 2). Synthetic template: 107 copies of synthetic hsa-miR-197 were used for polyA tailing/RT. An amount corresponding to 10s copies was used for PCR. Total human RNA: 100 ng of total human RNA was used for polyA-tailing/RT (step 1). An amount corresponding to 1 ng was used for PCR. Negative controls: water control and a total human RNA control without PolyA Polymerase in the RT reaction.The black line designates the result obtained with the synthetic template whereas the grey line show the result obtained with total human RNA as template. The melting curves of the result obtained with the synthetic template and the total human RNA are almost identical.<br><br>
FIGURE 3 shows the amplification curves and the first derivative of the melting curves of Example 2<br><br>
FIGURE 4 shows amplification plots of Example 4 wherein discrimination between ftsa-let-7a and the variants A7sa-let-7f, ftsa-let-7c and hsa-\et-7e are demonstrated. The amplification-signal from negative controls are also indicated.<br><br>
FIGURE 5A, 5B and 5C shows the method and processes that enables PCR amplification and discrimination between mature miRs, pre-miRs and pri-miRs. See example 6, 7 and 8. A) Polyadenylation of all RNA 3'-ends. B) Reverse transcription of polyadenylated RNA to cDNA using the Universal 111 pilmar (N-C, G, A and T; V- C, 0, and A), C) MIcroKNA specific PCR amplification U9lng gone specflc primers. Note that the site of polyadenylatlon In combination with the gene specific reverse primers ensures specific detection of each of the Individual molecules mature mIRs, pre-mlRs and pri-mlRs. Fig 5A Illustrate the assay used for assaying mature mIRs. Fig 5B Illustrate the assay used for assaying pre-mIRs and Fig 5C Illustrate the assay used for assaying prl-mlRs.<br><br>
FIGURE 6 shows amplification curves of (A) the pre-mlr-203 assay on pre-mlR-203, mlR-203 and non-template controls (NTC). (B) Amplification curves of (A) the mlR-203 assay on pre-miR-203, miR-203 and non-template controls (NTC).<br><br>
FIGURE 7 shows the different expression of a panel of selected miRNA genes in heart and liver. The data is presented as the difference in the crossing point (Cp) value (ACp) between heart and liver, see table 15.<br><br>
FIGURE 8 shows the detection of each of four listed mIRs assayed by the present method (black lines with diamonds) and compared with a commercial available method wherein the Reverse Transcription is performed as a one-tube-reaction for all microRNAs present in a sample (gray lines with spheres). The figure presents the crossing point (Cp) values as a function of template concentration. Each value is in triplicate.<br><br>
DEFINITIONS<br><br>
"small RNA molecules" refer to tiny RNA molecules in a living cell such as small "non-coding RNA molecules" i.e. molecules that are not translated into proteins. Non-coding RNA molecules include RNAs such as microRNAs (miRNAs), small interfering RNAs (siRNAs), small nuclear RNAs (snRNAs), small nucleolar RNAs (snoRNAs), small temporal RNAs (stRNAs), antigene RNAs (agRNAs) and piwi-interacting RNAs (piRNAs).<br><br>
The terms "microRNA", "miRNA" and "miR" are used synonymously and refer to 21-25 nucleotides non-coding RNAs derived from endogenous genes of living organisms such as animals and plants. These socalled mature miRNAs are biologically active and processed from longer hairpin-like precursors termed pre-miRNAs (pre-miRs) having a length of approximately 75 nucleotides. The precursers of pre-miRNAs are pri-miRNAs (pri-mlRs) with a length of 500 to 3000 nucleotides. MiRNAs assemble in complexes termed miRNPs and act as major regulators on important biological processes by binding to messenger RNA and interfering with translation efficiency. Target microRNAs of the present invention refer to all known microRNAs, such as microRNAs known from the scientific literature and public databases such as the miRBase database (http://microrna.sanger.ac.uk/) which is the home of microRNA data on the web administrated by the Sanger Institute, UK. The mIRBase release 12 Is hereby Incorporated by reference, including all the mature mIRNA and preÂ¬mature miRNA sequences disclosed therein. "microRNA profiling" describes a large-scale analysis in which the expression levels of all microRNAs in a sample such as a tumor sample are determined to build microRNA signatures for a particular disease such as a cancer disease.<br><br>
"Adding poly-A tails", "Poly-A tailing" and "polyadenylation" refers to the synthesis of a poly(A) tail, a stretch of RNA where all the bases are adenines, at the 3'-termini of an RNA molecule. Polyadenylation is a natural biological process in living organisms, but it can also be carried out in vitro using various polymerases such as commercial available E. coli Poly(A) Polymerase I (E-PAP).<br><br>
"Extension primer" and "RT-primer" refers to an oligonucleotide primer, comprising a recognition sequence, complementary to a sequence in the target deoxyribonucleic and/or ribonucleic acid sequence, e.g. to the 3'-end of the mature microRNA or small noncoding RNA in the target ribonucleic acid sequence, and an anchor sequence essential for subsequent amplification by PCR. The said extension primer is used as an anchored primer in a reverse transcription reaction to generate a primer extension product or cDNA.<br><br>
A "cDNA" refers to a complementary DNA produced by reverse transcription of an RNA template using a reverse transcriptase enzyme. Any reverse transcriptase can be used to synthesize the cDNA molecules, such as reverse transcriptases derived from moloney murine leukemia virus (M-MuLV) reverse transcriptase, avian myeloblastosis virus (AMV) reverse transcriptase, bovine leukemia virus (BLV) reverse transcriptase, Rous sarcoma virus (RSV) reverse transcriptase and human immunodeficiency virus (HIV) reverse transcriptase.<br><br>
"Primers" refers to short, chemically synthesized oligonucleotides, usually with a length of about twenty to thrity bases. They are hybridized to a target DNA, which is then copied by a DNA polymerase to produce a complementary DNA strand. A "forward primer" and a "reverse primer" constitute a "PCR primer set" used in PCR, where they hybridise to complementary DNA strands and direct replication towards each other producing the upper-strand and the lower-strand, respectively, leading to an exponential increase In the target DNA segment. The template derived extension of PCR primers can be carried out by any DNA polymerases, such as bacterial thermostable DNA polymerases, including Taq DNA polymerase from Thermus aquaticus, Pfu DNA, polymerase from Pyrococcus furiosus, Vent DNA polymerase from Thermococcus litoralis or recombinant DNA polymerases such as Phusion DNA polymerase.<br><br>
The terms "amplification", "PCR", "PCR reaction" and "PCR amplification", are Interchangeable terms used to signify use of a nucleic acid amplification system, which multiplies the target nucleic acids using the polymerase chain reaction (PCR).<br><br>
"qPCR" and "real-time quantitative PCR" refers to the use of PCR to amplify and simultaneously quantify a targeted DNA molecule. qPCR enables both detection and quantification of the number of copies or relative amount of a specific sequence in a DNA sample, when normalized to DNA input. Amplified DNA is quantified as it accumulates in the reaction in real time after each amplification cycle. Quantification is accomplished using various assay chemistries including fluorescent dyes, such as SYBRÂ® green, that intercalate with double-stranded DNA, and fluorescent reporter oligonucleotide probes such as Taqman probes, that release fluorescence signal during the amplification process.<br><br>
"qRT-PCR" refers to quantitative reverse transcription polymerase chain reaction in which a cDNA produced in a reverse transcription reaction serves as the initial DNA template for the amplification process, which is then combined with qPCR to quantify low abundance of RNA molecules such as microRNAs in a sample of a particular cell or tissue type. Methods for qPCR and qRT-PCR is described in "A-Z of Quantitative PCR." (Bustin, SA (ed.) International University Line (La Jolla, CA, USA), 2004) which is hereby incorporated by reference in its entirety.<br><br>
"Hybridisation" refers to the bonding of two complementary single stranded nucleic acid polymers (such as oligonucleotides), such as RNA, DNA or polymers comprising or consisting of nucleotide analogues (such as LNA oligonucleotides). Hybridisation is highly specific, and may be controlled by regulation of the concentration of salts and temperature. Hybridisation occurs between complementary sequences, but may also occur between sequences which comprise some mismatches. The oligonucleotides used in the methods of the present invention may, therefore be 100% complementary to the target molecule. Alternatively, in one embodiment the oligonucleotides may comprise at least one or two mismatches.<br><br>
The term "Tm" or "melting temperature" of an oligonucleotide is in the present context a measure of the stability of a duplex formed between the oligonucleotide and its perfect complement DNA strand determined at 115 mM Na+, no formamide. In general Tm is defined as the temperature at which 50 % of the duplexes formed between the oligonucleotide and a complement nucleotide strand are dissociated into single strands, The length and nucleotide composition, such aa the sequence of nucleotides and content of G and C nucleotides, of the oligonucleotide are Important factors affecting Tm. Subatltulton of the normal A, G, C and T nucleotides with the corresponding LNA molecules in an oligonucleotide increases Tm. Similary, hybridisation conditions defined by salt concentration, oligonucleotide concentration, and the presence of denaturants (such as formamide or DMSO) affects Tm. Those skilled in the art of molecular biology know that several useful formulas for calculation of theoretical Tm's have been developed to evaluate the Tm of an oligonucleotide for PCR,<br><br>
Southern and Northern blots, and in situ hybridization. Examples of Tm calculators are OligoCalc (W. A. Kibbe (2007) Nucleic Acids Res Volume 35, Web Server issue W43-W46) and LNA Probe Tm Predictor at http://www.exiaon.com/ollqo-tools .<br><br>
The term "base", as used herein covers the naturally occurring nucleobases adenine (A), guanine (G), cytosine (C), thymine (T) and uracil (U) as well as non-naturally occurring nucleobases such as xanthine, diaminopurine, 8-oxo-N6-methyladenine, 7-deazaxanthine, 7-deazaguanine, N4,N4-ethanocytosin, N6,N6-ethano-2,6-diaminopurine, 5-methylcytosine, 5-(C3-C6)-alkynyl-cytosine, 5-fluorouracil, 5-bromouracil, pseudolsocytosine, 2-hydroxy-5-methyl-4-triazolopyridin, isocytosine, isoguanine, inosine and the "non-naturally occurring" nucleobases described in Benner et al., U.S. Patent No. 5,432,272 and Susan M. Freler and Karl-Heinz Altmann, Nucleic Acid Research,25: 4429-4443, 1997. The term "nucleobase" thus includes not only the known purine and pyrimidine heterocycles, but also heterocyclic analogues and tautomers thereof. Further naturally and non naturally occurring nucleobases include those disclosed in U.S. Patent No. 3,687,808; in chapter 15 by Sanghvi, in Antisense Research and Application, Ed. S. T. Crooke and B. Lebleu, CRC Press, 1993; in Englisch, et al., Angewandte Chemie, International Edition, 30: 613-722, 1991 (see, especially pages 622 and 623, and in the Concise Encyclopedia of Polymer Science and Engineering, 3. I. Kroschwitz Ed., John Wiley &amp; Sons, pages 858-859, 1990, Cook, Anti-Cancer DrugDesign 6: 585-607, 1991, each of which are hereby incorporated by reference in their entirety).<br><br>
Nucleotides incorporated into oligonucleotides are referred to as nucleotide residues.<br><br>
The term "nucleoskllc base" or "nucleobase analogue" Is further Intended to Include heterocyclic compounds that can serve as like nucleoside bases Including certain "universal bases" that are not nucleoside bases in the most classical sense but serve as nucleoside bases. Especially mamlloned as a universal base is 3-nltropyrrola or a 5-nltrolndole. Other preferred compounds Include pyrene and pyrldyloxazola derivatives, pyrenyl, pyrenylmethylylycerul derivatives and the like. Other preferred universal bases Include, pyrrole, dlazole or triazble derivatives, including those universal bases known In the art.<br><br>
By "locked nucleic acid", "LIMA", "LIMA monomer" or "LNA molecule" (e.g., an LNA nucleoside or LNA nucleotide) or an LNA oligomer (e.g., an oligonucleotide or nucleic acid) is meant a nucleoside or nucleotide analogue that includes at least one LNA monomer.<br><br>
To distinguish between LNA and natural occurring nucleotide residues in LNA containing oligonucleotide sequences presented herein, the LNAs are indicated by capital letters, whereas natural occurring nucleotide residues are indicated by lowercase letters: mC denotes LNA methyl cytosine.<br><br>
LNA monomers as disclosed in PCT Publication WO 99/14226 are in general particularly desirable modified nucleic acids for incorporation into an oligonucleotide of the invention. Additionally, the nucleic acids may be modified at either the 3' and/or 5' end by any type of modification known in the art. For example, either or both ends may be capped with a protecting group, attached to a flexible linking group, attached to a reactive group to aid in attachment to the substrate surface, etc. Desirable LNA monomers and their method of synthesis also are disclosed in WO 98/39352.<br><br>
Preferred LNA monomers, also referred to as "oxy-LNA" are LNA monomers which include bicyclic compounds as disclosed in PCT Publication WO 03/020739 wherein the bridge between R4' and R2' as shown in formula (I) below together designate -CH2-0- or -CH2-CH2-0-.<br><br>
It Is understood thai. references herein to a nucleic acid unit, nucleic acid residue, LNA monomer, or similar term are Inclusive of both individual nucleoside units and nucleotide units and nucleoside units and nucleotide units within an oligonucleotide. <br>
A "modified base" or other similar terms refer to a composition (e.g., a non-naturally occurring nucleotide or nucleoside base), which can pair with a natural base (e.g., adenine, guanine, cytosine, uracil, and/or thymine) and/or can pair with a non-naturally occurring nucleotide or nucleoside base.  Desirably, the modified base provides a Tm differential of 15, 12, 10, 8, 6, 4, or 2Â°C or less as described herein. Exemplary modified bases are described in EP 1 072 679 and WO 97/12896.<br><br>
The term "chemical moiety" refers to a part of a molecule. "Modified by a chemical moiety" thus refer to a modification of the standard molecular structure by inclusion of an unusual chemical structure. The attachment of said structure can be covalent or non-covalent.<br><br>
The term "inclusion of a  chemical  moiety"  in  an  oligonucleotide  probe thus refers to attachment of a molecular structure. Such as chemical moiety include but are not limited to covalently and/or non-covalently bound minor groove binders (MGB) and/or intercalating nucleic acids (INA) selected from a group consisting of asymmetric cyanine dyes, DAPI,<br>
Â®	Â®	Â®	Â®<br>
SYBR     Green I, SYBR     Green II, SYBR     Gold, PicoGreen     , thiazole orange, Hoechst 33342, Ethidium Bromide, l-0-(l-pyrenylmethyl)glycerol and Hoechst 33258. Other chemical moieties include the modified nucleotides, nucleosidic bases or LNA modified oligonucleotides.<br><br>
"Oligonucleotide analogue" refers to a nucleic acid binding molecule capable of recognizing a particular target nucleotide sequence. A particular oligonucleotide analogue is peptide nucleic acid (PNA) in which the sugar phosphate backbone of an oligonucleotide Is replaced by a protein like backbone. In PNA, nucleotides are attached to the uncharged polyamide backbone yielding a chimeric pseudopeptide-nucleic acid structure, which Is homomorphous to nucleic acid forms.<br><br>
"High affinity nucleotide analogue" or "affinity-enhancing nucleotide analogue" refers to a non-naturally occurring nucleotide analogue that Increases the "binding affinity" of an oligonucleotide probe to Id complementary recognition sequence when substituted with at least one such hlgh-affinity   nucleotide analogue.<br><br>
As used herein, a probe with an Increased "binding affinity" for a recognition sequence compared to a probe which comprises the same sequence but does not comprise a stabilizing nucleotide, refers to a probe for which the association constant (K8) of the probe recognition segment is higher than the association constant of the complementary strands of a double-stranded molecule. In another preferred embodiment, the association constant of the probe recognition segment Is higher than the dissociation constant (Kd) of the complementary strand of the recognition sequence in the target sequence in a double stranded molecule.<br><br>
Monomers are referred to as being "complementary" if they contain nucleotides that can form hydrogen bonds according to Watson-Crick base-pairing rules (e.g. G with C, A with T or A with U) or other hydrogen bonding motifs such as for example diaminopurine with T, 5-methyl C with G, 2-thiothymldine with A, inosine with C, pseudoisocytosine with G, etc.<br><br>
By "oligonucleotide," "oligomer," or "oligo" is meant a successive chain of monomers (e.g., glycosides of heterocyclic bases) connected via internucleoside linkages. The linkage between two successive monomers in the oligo consist of 2 to 4, desirably 3, groups/atoms selected from -CH2-, -0-, -S-, -NRH-, &gt;C=0, &gt;C=NRH, &gt;C=S, -Si(R")2-, -SO-, -S(0)2-, -P(0)2-, -PO(BH3)-, -P(0,S)-, -P(S)2-, -PO(R")-, -PO(OCH3)-, and -PO(NHRH)-, where RH is selected from hydrogen and C^-alkyl, and R" is selected from C^-alky! and phenyl. Illustrative examples of such linkages are -CH2-CH2-CH2-, -CH2-CO-CH2-, -CH2-CHOH-CH2-, -0-CH2-0-, -0-CH2-CH2-, -0-CH2-CH= (including R5 (see formula I) when used as a linkage to a succeeding monomer), -CH2-CH2-0-, -NRH-CH2-CH2-, -CH2-CH2-NRH-, -CH2-NRH-CH2-, -0-CH2-CH2-NRH-, -NRH-CO-0-, -NRH-CO-NRH-, -NRH-CS-NRH-, -NRH-C(=NRH)-NRH-, -NRH-CO-CH2-NRH-, -0-CO-0-, -0-CO-CH2-0-, -OCH2-CO-0-, -CH2-CO-NRH-, -0-CO-NRH-, -NRH-CO-CH2-, -0-CH2-CO-NRH-, -0-CH2-CH2-NRH-, -CH=N-0-, -CH2-NRH-0-, -CH2-0-N= (including R5 when used as a linkage to a succeeding monomer), -CH2-0-NRH-, -CO-NRH-CH2-, -CH2-NRH-0-, -CH2-NRH-CO, -0-NRH-CH2-, -0-NRH-, -0-CH2-S-, -S-CH2-0-, -CH2-CH2-S-; -0-CH2-CH2-S-, -S-CH2-CH- (Including R5 when used as a linkage to a succeeding monomer), -S-CH2-CH2-, -S-CHJ-CHJ-O-, -S-CH2-CH2-S-, -CH2-S-CHr, -CHrSO-CH,-, -CH2-S02-CH2-, -0-SO-0-, -0-8(0)1-0-, -0-S(0),-CHj-, -0-S(0)2-NRM-, -NRH-S(0)|-CH,-Â» -0-S(0),-CHr, -O-P(0)2-0-, -0-P(0,S) 0-, -0-P(S)j-0-f -S-P(0)2-0-, -S-P(0,S)-0-, SP(S),-0-, -0-P(0)2-S-, -0-P(0,S)-S-, -0-P(B)rB , -S-P(0)rS-; -S-P(0,S)-6-, -S-P(S)rB-, 0-PO(R")-0-, -O-PO(OCH3)-0-, -O-POfOCMjCHjJ-O-, -O-PO(0CH2CH2S-R)-0-, -0-PO(l3H3)-0-, -0-PO(NHRN)-0-, -0-P(0)2-NRH-, -NRH-P(0)2-0-, -0-P(0,NRH)-0-, -CH2-P(0)2-0-; -0-P(0)2-CH2-, and -O-Si(R")2-0-; among which -CH,-CO-NRH-, -CH2-NRH-0-, -S-CHj-O-, -0-P(0)2-0-, -0-P(0,S)-0-, -0-P(S)2-0-, -NR"-P(0)2-0-, -0-P(0,NRH)-0-, -0-PO(R")-0-, -0-PO(CH3)-0-, and -0-P0(NHRN)-0-, where RM Is selected form hydrogen and Ci.4-alkyl, and R" Is selected from Ci.6-alkyl and phenyl, are especially desirable. Further Illustrative examples are given in Mesmaeker et. a/., Current Opinion In Structural Biology 1995, 5, 343-355 and Susan M. Freier and Karl-Heinz Altmann, Nucleic Acids Research, 1997, vol 25, pp 4429-4443. The left-hand side of the internucleoside linkage is bound to the 5-membered ring as substituent P* at the 3'-position, whereas the right-hand side is bound to the 5'-position of a preceding monomer.<br><br>
The term "succeeding monomer" relates to the neighbouring monomer In the 5'-terminal direction and the "preceding monomer" relates to the neighbouring monomer in the 3'-terminal direction. It should be noted that whereas the natural synthesis of DNA and RNA proceed in the direction 5' to 3', many chemical synthesis-schemes proceed in the direction of 3' to 5'.<br><br>
When performing quantitative PCR, the cycle at which the fluorescence from a sample crosses the threshold is called the "cycle threshold" or Ct. Ct is used for quantification of a template.<br><br>
The "crossing point" or Cp value is a slightly different, but related value that can be used for quantification of a template somewhat similar to the use of Ct. The LightCyclerÂ® 480 Software calculates the second derivatives of entire amplification curves and determines where this value is at its maximum. This value (crossing point, Cp) represents the cycle at which the increase of fluorescence Is highest and where the logarithmic phase of a PCR begins. Both terms Ct and Cp are further explained In Bustln, SA (ad.) "A-Z of Quantitative PCR." International University Una (La Jolla, California, USA), 2004, which Is Included herein by reference.<br><br>
Suitably, the "target" or "target nucleic acid" or "target rlbonucleic  acid" refers to any relevant nucleic add of a single specific sequence, e. g., a biological nucleic add, e. g., derived from a subject or human being. Within the context of the oligonucleotides and detection probes used In the Invention to detect mIRNAs, the "target" Is a human miRNA or precursor thereof, or In one embodiment, a molecule which retains the genetic sequence information contained therein - such as all or (a sufficient) part of the seqeunce of nucleotides or reverse complement thereof.<br><br>
"Target sequence" refers to a specific nucleic acid sequence (or corresponding nuceltobase seqeunce) within any target nucleic acid.<br><br>
The term "primer design" refers to methods such as those provided below. The primer design is an systematic approach used for designing the sequence of nucleotides in forward and reverse primers thus ensuring probe specificity and binding efficiency to target microRNAs. The following rules have been used for design of primers for miR-specific qPCR:<br><br>
Forward primer design:<br><br>
Preferably, the forward primers are designed to be identical of from 12 to 18 bases of the 5' end of the miR sequence. Preferably, the Tm of the forward primers should be in the range of between 55 Â°C and 65 Â°C, however, a Tm below 55 Â°C and above 65 Â°C may also be acceptable. To ensure that the Tm of the primers preferably is in the range of between 55 Â°C and 65 Â°C, one or more LNA monomers may be inserted into the sequence substituting the natural nucleotide. An artificial nucleotide sequence may also be added to the 5'-end of the forward primer to make certain that the Tm is in the range of between 55 Â°C and 65 CC.<br><br>
Reverse primer design:<br><br>
The reverse primer is according to formula II:<br>
R3-(T)X -R4 (II)<br><br>
wherein R   Is a 5'-e nd l nucleotide sequence, (T)x is a central part or x consecutive thymine 4 residues In the amount of x, wherein x is an Integer of from 1 to 100, and R   Is a 3'-end nucleotide sequence that specifically hybridizes to a part of a mlcroRNA molecule.<br><br>
4 R   is preferably a nucleotide sequence of from 1 to 10 nucleotides designed from the 3'-end 4 of a specific mIRNA. R Is able to hybridize specifically to the DNA strand (I.e. the upper-strand) produced by extension of the corresponding miRNA specific forward primer. To ensure miRNA specificity and make certain that the Tm of the primer preferably is in the 4 range of between 55 Â°C and 65 Â°C, one or more LNA monomer may be inserted into the R sequence substituting the corresponding natural nucleotide.<br><br>
The (T)x central  part is preferably a stretch   of approximately  15 consecutive thymine nucleotide residues that hybridizes to the poly-A-tail part of the DNA strand produced from the corresponding miRNA specific forward primer.<br><br>
The R   sequence of the reverse primer typically has a length of form 1 to 20 nucleotides. For example  R    can  be  17  nucleotides long,  8,   7 or 6 nucleotides long or even only one nucleotide long. In certain embodiments of the present invention, the R    sequence is 5'- 3 TGACACGGAGGTACTAG-3' (SEQ ID NO 3). The length of the R   sequence may be reduced from the 5'-end to adjust the Tm of the reverse primer to the preferable range of between 55 Â°C and 65 Â°C. The R    sequence is identical to or at least has overlap to part of the R sequence (formula (III)) of the extension primer.<br><br>
The procedure for designing the reverse primer is described below.<br><br>
A.	Design a number or pi linen that fulfil the following CONDITIONS:<br><br>
1)	Delete all adunlna residues from 3'-end of mIR. nucleotide   sequence. They will form part of the poly-A tall.<br><br>
2)	After deleting adenine residues start with reverse complement to 3' end base of the mIR and continue until one base overlap with forward primer.<br><br>
3)	Delete one base from the 3'end of the reverse primer If the sequence Is a cytosine residue and the 3' end of the forward primer is a guanine residue.<br><br>
4)	Delete one base from the 3'end of the primer if the sequence is a guanine residue and the 3' end of the forward primer is a cytosine residue.<br><br>
5)	Delete one base from the 3'end of the primer if the last two bases are overlapping with the last two bases of the forward primer (from the 3' end).<br><br>
6)	Repeat this process until the conditions are fulfilled.<br><br>
7)	If no primer fulfils the conditions disregard rules #2 and 3 and allow two bases overlap (but not more) for sequences ending with AA, AT, TA or TT.<br><br>
8)	If no primer fulfils the conditions try another forward primer design.<br><br>
B.	Choose the longest reverse primer that fulfils the following conditions:<br><br>
1)	At least four bases long.<br><br>
2)	Less than five cytosine residues and guanine residues in the last six bases at the 3' end of the primer.<br><br>
3)	Less than four cytosine residues and guanine residues in the last five bases at the 3' end of the primer except if the last or second last base is an adenine residue or thymine residue   .<br><br>
4)	If possible not more than two adenine residues or thymine residues in the 3'end.<br><br>
5)	If no primer fulfils the conditions try another forward primer design.<br><br>
C.	Optionally insert one LNA monomer according to the following rules:<br><br>
1)	Insert LNA at first cytoslne residue or guanine residue from 5' end of the R   part of the reverse primer.<br><br>
2)	No LNA in sequences with three or more consecutive cytosine residues or guanine residues.<br><br>
3)	LNA should be in position four or higher from the 3' end of the primer.<br><br>
4)	If the conditions cannot be fulfilled insert LNA at first adenine residue or thymine residue from 5' end of the R   part of the reverse primer.<br><br>
5)	LNA should be in position four or higher from the 3' end of the primer.<br><br>
6)	If the conditions cannot be fulfilled insert LNA-thymine (LNA-T) at 5' end of the R part of the primer.<br><br>
D.	Tailing:<br>
Add a sequence of the formula R -(T)x to the 5'end of the reverse primer. I certain embodiments of the present invention the nucleotide sequence:<br><br>
5'-TGACACGGAGGTACTAG I I I I I I I II I I I I I I -3' (SEQ ID NO 4) is added at the 5' end of the reverse primer.<br><br>
If an LNA-T was added to the miR-specific part of the reverse primer (step C6) delete one thymine residue from the tail as there should never be more than 15 consecutive thymine residues.<br><br>
Eventually, nucleotides are removed from the 5'-end of the reverse primer to fine-tune the Tm of the primer.<br><br>
"Sample" of RNA refers to RNA comprising compositions obtained from cells, tissues or fluids from an organism according to conventional procedures described e.g. in RNA Isolation and<br><br>
Characterization Protocols (Rapley, Ralph; Manning, David L (Eds.) 1998) known to the skilled artesian or by using commercial kits such as miRNeasy (QIAGEN GmbH, Hilden, Germany)) or miRVana (Ambion Inc., Austin, TX, US). Sources for isolation of RNA fractions are samples of cells, or tissue or fluid isolated from an organism or organisms, including but not limited to, for example, skin, plasma, serum, spinal fluid, lymph fluid, synovial fluid, urine, tears, blood cells, organs, tumours, and also to samples of in vitro cell culture constituents (including but not limited to conditioned medium resulting from the growth of cells in cell culture medium, recombinant cells and cell components). "Sample" may also refer to cells or fluids that can be used directly in a qRT-PCR proces without previous enrichment Of the RNA fraction or even RNA comprising compositions comprising artificially synthesised RNA.<br><br>
Cells or cell types also refer to any cells of Eukaryote, Prokaryote and Archaea origin.<br><br>
A " living organism" refers to a living entity, including but not limited to, for example, human, mouse, rat, Drosophila, C. elegans, yeast, Arabidopsis thaliana, maize, rice, zebra fish, primates, domestic animals, etc.<br><br>
DETAILED DESCRIPTION OF THE PREFERRED EMBODIMENT<br><br>
Detailed outline of the qRT-PCR reaction and the primers involved:<br><br>
The present invention provides methods for amplification of microRNA molecules in a sample, the method comprising the steps described schematically In Fig. 1, and comprise:<br><br>
(a)	adding poly-A tails to a population of RNA molecules in a sample;<br><br>
(b)	producing cDNA molecules of the poly-A-tailed RNA molecules using an extension primer in a reverse transcription reaction; and<br><br>
(c)	amplifying the cDNA molecules by PCR using a forward primer and a reverse primer both of which are specific for said RNA molecule.<br><br>
When performing the method in practice step (a) and (b) is typically performed as one concerted reaction which is universal for all RNAs to be analysed - hence its nickname: "Universal RT". The advantage of only one first-strand cDNA synthesis reaction (or RT reaction) to be used as template for multiple real-time PCR assays is that It saves precious sample, reduces technical variation and reduces time spent In the laboratory.<br><br>
During step (c) of the method individual (or groups of individual) RNAs are specifically PCR amplified using specific forward and reverse primers. Typically the primers are optimised by introducing one or more LIMA nucleotide analogues into the sequence of the primers, and typically the PCR is quantitative Real-Time PCR. As can be seen from the examples the method result in 1) uniquely specific assays that enable discrimination between highly related RNA sequences and 2) an extremely low background which enables accurate quantitation of very low RNA levels.<br><br>
The method has been extensively used to quantify small RNAs. In a preferred embodiment the small RNA, comprises small non-coding RNAs such as short inferring RNAs (siRNAs), mature microRNAs and pre-microRNAs. Also larger RNAs, e.g. precursers of pre-microRNAs the pri-miRNAs (pri-miRs) and mRNAs may be assessed by method.<br><br>
Most preferably the small RNA is a microRNA.<br><br>
The extension primer may be of a length in the range from 10 to 100 nucleotidbes, such as a length in the range from 15 to 45 nucleotides. Preferably, the extension primer has a length of 20, 21, 22, 23, 24, 25, 26, 27, 28, 29 or 30 nucleotides.<br><br>
In a preferred embodiment, the extension primer is of the formula III:<br>
f^-COy -R     (HI)<br>
wherein R    is a 5'-end nucleotide sequence, (T)y is a central part of consecutive thymine residues in the amount of y, wherein y is an integer of from 1 to 100, and R    is a 3'-end nucleotide sequence.<br><br>
Typically, the 5'-end  part R    is a  nucleotide  sequence with a  length of from  1 to 30 nucleotides. For example the R   sequence may be 6, 7, 8, 9, or 10 nucleotides long. The  sequence Includes at least a hybridisation sequence for the R    sequence In the reverse primer of the formula II (see below) to be used In the subsequent mIRNA-specific qPCR.<br><br>
Preferably, y In formula JII Is Hi tha interval of from 5 to DO, mora preferably y Is In the Interval of from 5 to 21, Por example y is 12, 13, 14, 13, 16, 17 or 18. Most preferably y is 15.<br><br>
In one especially preferred embodiment y of formula (HI) equals x of formula (II).<br><br>
In a preferred embodiment, R   is a degenerated anchor sequence motif VN, comprising the two 3'-end terminal nucleotide residues, wherein V designate a selection of residues comprising bases which can base pair to all but one of the natural bases (e.g. base pair to guanine, cytosine, uracil and thymine but not to adenine), typically V designate that the base in any primer molecule is a base randomly selected between adenine, guanine and cytosine and wherein N designate a base which can base pair to any of the natural bases (e.g. adenine, guanine, cytosine, uracil and thymine), typically N may be a base randomly selected between adenine, guanine, cytosine or thymine residue.<br><br>
In another preferred embodiment R   is a degenerated sequence motif VNN, comprising the three 3'-end terminal nucleotide residues, wherein V designate a selection of residues comprising bases which can base pair to all but one of the natural bases (e.g. base pair to guanine, cytosine, uracil and thymine but not to adenine), typically V designate that the base is a base randomly selected between adenine, guanine and cytosine and wherein N designate a base which can base pair to any of the natural bases (e.g. adenine, guanine, cytosine, uracil and thymine), typically N may be a base randomly selected between adenine, guanine, cytosine or thymine residue.<br><br>
In a preferable embodiment, the extension primer comprises at least one LNA.<br><br>
In a certain embodiment of the present invention the extension primer has the sequence 5'-GGTACTAGI I I I I I I I I I I I I I IVN-3' (SEQ ID NO 5).<br><br>
Preferably, the forward primer has a length in the range of from 10 to 100 nucleotides, such as 12 to 22 or 13 to 20 or 14,15, 16, 17, 18 or 19 nucleotides. Sea table 18 for specific embodiments.<br><br>
The forward primer rimy comprise one, two or even more I.NAs.<br><br>
In a preferred embodiment, the nucleotide sequence of the forward primer Is designed to specifically hybridize to the complement DNA molecule of a specific mlcroRNA molecule using the primer design rules described In the definition of "primer design" (see DEFINITIONS section)<br><br>
Preferably, the reverse primer Is of the formula II:<br><br>
 R -(T)x-R   (II)<br><br>
3 wherein R    is a 5'-end nucleotide sequence (T)x is a central part of consecutive thymine nucleotides in the amount of x, wherein x is an integer of from 1 to 100, and R   is a 3'-end nucleotide sequence that specifically hybridizes to a nucleotide sequence of a target RNA molecule.<br><br>
Preferably, the 5'-end nucleotide sequence R    of the reverse primer of formula II is a nucleotide sequence with a length of from 1 to 30 nucleotides.<br><br>
Preferably, x in formula II is in the interval of from 5 to 50, more preferably x is in the interval of from 5 to 21. For example x is 12, 13, 14, 15, 16, 17 or 18. Most preferably x is 15.<br><br>
In one especially preferred embodiment x of formula (II) equals y of formula (III).<br><br>
In a preferred embodiment, the 3'-end nucleotide sequence R of the reverse primer of formula II has a length In the range of from 1 to 10 nucleotides.<br><br>
As  illustrated   In  example  2  LIMA  has  a   profound  effect.   Accordingly,   In  a  preferred embodiment, the 3'-end part, II , of the reverse primer of formula II comprises at least one 4 LNA. More preferably, Ilia 3'-anU part, R , of the reverse primer of formula II contains only one LNA. In a particularly preferred embodiment the LNA Is situated In the 5' position or the 4 position adjacent to the 5' position of the R   part of the reverse primer.<br><br>
In a preferred embodiment the nucleotide sequence of the reverse primer is designed to specifically hybridize to a specific mlcroRNA molecule using the primer design rules described In the definition of "primer design" (see DEFINITIONS section)<br><br>
Design of primers<br><br>
The invention also provides methods for designing the nucleotide sequence of the forward primers and the reverse primers used in the invention. Systematic approaches and experimental evaluation of the primers are presented in the EXAMPLES 3 section.<br><br>
Examples of applications of the present invention<br><br>
The present invention is useful for amplification and quantification of small RNA molecules such as microRNAs or siRNA's in various cells or tissues originating from humans or other organisms as described in EXAMPLES 5 and 9. As illustrated in example 10 the present method is superior with respect to sensitivity and specificity when compaired with a somewhat similar single tube cDNA synthesis approach which is commercially available.<br><br>
Thus one aspect of the invention is a method for measuring the amount of a target microRNA in a sample from a living organism, the method comprising the steps of:<br><br>
a)	amplifying the target microRNA according to the method of any one of claims 1 to 28<br><br>
b)	measuring tha amount of the amplified DNA molaculei.<br><br>
The amount of the   amplitude d DNA molecules Is typically rneasured using  fluorescence-based Â® quantitative raal-tlma pert by e.g. monitoring SYBR    green fluorescence at a function of PCR cycle number.<br><br>
Accordingly, the present Invention can be used as a tool for microRNA profiling of different cell and tissue types within a living organism. Thus, the number of specific microRNAs and the amounts of specific microRNAs present in different cell and tissue types can be determined using the present Invention. Similarly, the Invention can be used to discriminate between the levels of mature microRNAs, pre-microRNAs and pri-microRNAs in different cell and tissue types by using appropiate primers targeting the mature microRNA or its precursors (see fig 5).<br><br>
In another aspect, the present invention can be used as a tool for microRNA profiling of patients suffering from various diseases such as cancer. By way of example, the microRNAomes of various cancer tissues can be established using the invention.<br><br>
In yet another aspect the present invention can be used for diagnosis of various diseases such as cancer by measuring microRNA expresssion in tissues and body fluids from normal individuals and from individuals having a disease, and subsequently analyse for differences in the microRNA profiles.<br><br>
In another embodiment the present invention may be used for measuring changes in the amount of specific microRNAs in response to treatment of a disease affected Individual by means of pharmaceuticals, such as chemotherapeutical agents, and by means of surgery.<br><br>
In another embodiment the present invention may be used for measuring specific predictive microRNA biomarkers that correlate with re-currence free survival of patients having a disease, such as a cancer disease.<br><br>
Due to Its slmpllcltyi one "universal-RT" step and one discriminatory PCR step, the present Invention Is also suitable for high-throughput methods on robot platforms directed to the quantification of a  single   microRNA or a collection of microRNAs. Thus, the method Is particularly suitable for multiple   subsequent PCR reactions of Individual microRNAs as described In EXAMPLE! 5, 9 and 10.<br><br>
In a certain embodiment the method of the invention may be used to quantitatively determine the amount of mature microRNA in a sample without Interference from the corresponding pre-mIR, and vice versa the amount of a pre-mIR can be determined without interference from the mature microRNA as described In EXAMPLES 6, 7 and 8.<br><br>
In another embodiment the method of the invention may be used for discrimination between targets with single nucleotide differences as described in EXAMPLE 4.<br><br>
In another aspect, the present invention provides kits for detecting at least one target microRNA, the kits comprising one or more primer sets specific for the detection of a target microRNA, each primer set comprising firstly, a universal extension primer for producing a cDNA molecule complementary to a target microRNA, and secondly, a PCR primer set comprising a forward primer specific for the 5'-end of the target miRNA and a reverse primer specific for the 3'-end of the target miRNA for amplifying the cDNA molecule.<br><br>
Accordingly in one aspect the invention a kit for detecting at least one target RNA is provided. The kit comprises at least one primer set specific for the detection of a target RNA, said primer set comprises:<br><br>
a) an extension primer of formula I: R -(T)y -R     (III)<br><br>
wherein R   is a 5'-end nucleotide sequence, (T)y is a central part of consecutive thymine residues in the amount of y, wherein y is an integer of from 1 to 100, and R   is a 3'-end nucleotide sequence.<br><br>
b)	a reverse primer of formula II: R -(T)x -R     (II)<br><br>
wherein R   Is a 5'-aiul nucleotide sequence, (T)x It a central part of consecutive thymine<br>
4 residues In the amount of x, wherein x is an Integer of from 1 to 100, and R   Is a 3'-end nucleotide sequence that specifically hybridizes to a nucleotide sequence of a target RNA molecule, and<br><br>
c)	a forward primer<br><br>
In one embodiment the kit is designed for detecting at least one mammalian target microRNA comprising at least one primer set specific for the detection of a target microRNA.<br><br>
Preferably, the extension, the reverse and the forward primers are designed following the design rules given in the "primer design"-part of the DEFINITIONS section and elsewhere and forward and reverse primers which includes at least one LNA molecule are especially preferred embodiments.<br><br>
A non-limiting example of an extension primer is shown as SEQ ID NO 5.<br><br>
Forward and reverse primers included in the kit may be designed to detect any mammalian target microRNA in accordance with the methods described herein. Non-limiting examples of forward and reverse primers are listed in TABLE 16.<br><br>
In certain embodiments, the kit includes multiple primer sets that may be used to detect various mammalian microRNA targets, such as two microRNA targets up to several hundred microRNA targets.<br><br>
A kit of the invention may also provide an array of primers delivered in microtiter qPCR plates such as 96, 768, 869, 1536 or even 3456-well microtiter qPCR plates suitable for robot handling as indicated in EXAMPLE 9.<br><br>
A kit of the Invention may also Include reagents required for the poly-A-talllng, primer extension and PCR reactions, such as buffers, salts, reducing agents, deoxy nucleoside triphosphates, nucleotide triphosphates, and enzymes, A detection reagent for the qPCR such as SYBRÂ® green may also he Included. Similarly a kit for UNA Isolation may also be Included.<br><br>
A further aspect of the Invention Is a high-throughput method for measuring the amount of specific target mlcroRNAs In a sample from a living organism by using the method of the present invention and Integrating an automated wherein the combined poly-A-talllng and reverse transcription reaction Is allquoted into individual wells of a mlcrotlter plate containing microRNA specific primer sets of forward and reverse primers, resulting In the steps of:<br><br>
a)	adding poly-A talis to a population of small RNA molecules In a sample and producing cDNA molecules of the poly-A-tailed small RNA molecules using an extension primer in a reverse transcription reaction; and<br><br>
b)	pipetting aliquots of the combined poly-A-tailing and reverse transcription reaction into individual wells of a microtiter plate<br><br>
c)	amplifying specific target microRNAs in individual wells of a microtiter plate containing microRNA specific primer sets<br><br>
d)	measuring the amount of specific microRNA molecules in individual wells.<br><br>
being performed in a fashion compatible with a high-throughput experimental set-up. Such a set-up will typically comprise one or more pipetting robots.<br><br>
EXAMPLES<br><br>
EXAMPLE 1: PRODUCTION OF SPECIFIC DNA MOLECULES USING MIR-SPECIFIC QPCR.<br><br>
In this example hsa-miR-197 was amplified from a human RNA sample using the mIR specific quantitative reverse transcription polymerase chain reaction (qRT-PCR) of this invention.<br><br>
Mixed on ice:<br><br>
â¢	1 Î¼l lOx Poly(A) polymerase buffer (New England Biolabs)<br><br>
â¢	1 Î¼l  1 mM ATP<br><br>
â¢	1 Î¼l  10 Î¼M RT-primer (=extension primer) (L2TA: 5'-ggtactagtttttttttttttttvn (SEQ ID NO 5), v designates cytosine, guanine and adenine residues, n designates cytosine, guanine, adenine and thymine residues)).<br><br>
â¢	1 pi mix of 1 mM dATP, 1 mM dCTP, 1 mM dGTP, 1 mM dTTP<br><br>
â¢	0,5 pi (200U/pl) MuLV reverse transcriptase (New England Biolabs Inc., Ipswich, MA, US)<br><br>
â¢	0,2 pi (5U/pl) Poly(A) polymerase (New England Biolabs)<br><br>
â¢	1 pi RNA sample (total human RNA sample =100 ng of a mixture of 25 ng human heart RNA, 25 ng human brain RNA, 25 ng human liver RNA and 25 ng human lung RNA all from Ambion, Austin, TX, US. Synthetic templates were prepared in lOng/pl phage MS2 RNA in TE, approximately 107 copies are added to the reaction. Synthetic template is obtained from Integrated DNA technologies Inc., Coralville, IA, US.)<br><br>
.    Water to 10 Î¼l <br><br>
Negative controls: water control and a total human RNA control without PolyA Polymerase in the RT reaction.<br><br>
The mixture was incubated at 42 Â°C, 1 hour followed by 95 Â°C, 5 minutes.<br><br>
For qPCR 1 pi or less of the polyA-tailing/RT reaction (step 1 on fig 1) was used for each PCR reaction with gene-specific forward and reverse primers for hsa-miR-197; forward primer 5' ttmCaccaccttctcca (SEQ ID NO 1) and the reverse primer 5'-ctttttttttttttttGctgggt (SEQ ID NO 2) (in the nucleotide sequences lower case designates a natural occurring nucleotide, upper case designates LNA, mC denotes LNA methyl cytosine).<br><br>
Â® Real-time PCR was done on a ABI 7500     thermocycler (Applied Biosystems Inc, Foster City,<br><br>
Â® CA, US) by monitoring SYBR     green fluorescence as a function of PCR cycle number. A typical PCR reaction mixture contained:<br><br>
â¢	10 pi  2x   PCR mastermix (Roche cat#04 673 484 001,  Roche Diagnostics A/S, Hvidovre, Denmark)<br><br>
â¢	1 ul or less of the RT reaction<br><br>
â¢	1  pi TE buffer with 2.5 Î¼M miR-specific forward primer and 2.5 pM miR-specific reverse primer (TE buffer: 10 mM Tris/HCI (pH 8.0), ImM EDTA).<br><br>
â¢	Water to 20 UL.<br><br>
The mixture was incubated at 95Â°C for 10 min followed 40 cycles of 95Â°C for 5 sec; 60Â°C for 60 sec with measurement of the fluorescence.<br><br>
Amplification of the correct product was measured by comparing the melting curve profile of the PCR product to the melting curve profile of a PCR product obtained by amplification of a synthetic template (Fig. 2). No signal was obtained with the negative controls. The experiment shows that the same and correct hsa-miR-197 PCR product was obtained both using the total human RNA sample and a c hsa-miR-197 as template in step 1.<br><br>
EXAMPLE 2: EFFECT OF LNA IN THE REVERSE PRIMER.<br><br>
The effect of different designs of miR-specific PCR primers of step 2 can be tested on artificial DNA templates with the same sequence as the product of the reverse transcription reaction. An important advantage of using an artificial DNA template for PCR is that experimental variations in the efficiency of the reverse transcription step are eliminated.<br><br>
Wsa-let-7a DNA template:<br><br>
5'-tgcggtgacacggaggtactagtttttttttttttttaactatacaacctactacctca-3' (SEQ ID NO 6)<br>
Salmon DNA: 2 ng/ul in TE buffer MiR-specific forward primer:<br><br>
F7a:   5'-tGaGgtagtaggttg   (SEQ   ID   NO   7)   (lower  case   designates   a   natural   occurring nucleotide, upper case designates LNA).<br><br>
MiR-specific reverse primers:<br><br>
r7a.2: 5'-cggtgacacggaggtactagtttttttttttttttaactata (SEQ ID NO 8)<br><br>
r7a.7: 5'-cggtgacacggaggtactagtttttttttttttttaamCtata (SEQ ID NO 9)<br><br>
A PCR mix was prepared:<br><br>
â¢	550 pi 2x PCR mastermix (Roche cat#04 673 484 001)<br><br>
â¢	440 ul water, and as described i tab 1. TABLE l:Four PCR reactions were prepared:<br><br>
 The mixture was incubated at 95Â°C for 10 min followed by 40 cycles of 95Â°C for 5 sec; 60Â°C for 60 sec with measurement of the fluorescence.<br><br>
Amplification of the correct product was measured according to the recommendations from the manufacturer of the thermocycler (Applied Biosystems, Foster City, CA, US). Briefly, the PCR reaction was incubated at 60Â°C for 1 minute and fluorescense was measured during a slow increase of the temperature to 95Â°C. The first derivative of the melting curves is shown in figure 3:<br><br>
The results show (Figure 3) that the reverse primer without LNA (r7a.2) gives a positive signal in the negative control whereas the reverse primer with LNA (r7a.7) does not. Thus the LNA base in r7a.7 is necessary in order to avoid an unspecific PCR product developed in the negative control reaction. Both primers amplify the positive control template.<br><br>
As can be seen from table 2, it was found that the inclusion of one LNA in the miR-speclfic sequence of the reverse primer reduced background signal in PCR.<br><br>
 EXAMPLE 3: MANUAL PRIMER DESIGN AND VALIDATION OF PRIMERS FOR MIRS.<br><br>
Primers were manually designed following the design-rules presented In the "primer design"-part of the DEFINITIONS section.<br><br>
Primer validation:<br><br>
By using the primer design rules it is possible to achieve PCR primers that have &gt;70 % success rate in real-time PCR experiments according to the following validation criteria: Prepare Rtmix for four reverse transcription (RT) samples:<br><br>
1.	Water<br><br>
2.	100 ng total human RNA mix* (as default we use a mix of 25 ng of each heart, brain, liver and lung total RNA from Ambion).<br><br>
*RNA mix: 1 pi 1ug/ul heart RNA, 1Î¼l  1Î¼g /ul brain RNA, 1ul 1Î¼g/ul liver RNA, 1Î¼l/ÎL  lung RNA, 36ul TE. Store in 1Î¼l aliquots at -80Â°C<br><br>
Reverse transcription reactions:<br><br>
3.	100 ng total human RNA mix without PolyA tailing.<br><br>
4.	107 copies of the synthetic miR in 10 ng phage MS2 RNA in TE (synthetic miRs were obtained from Integrated DNA technologies Inc., Coralville, IA, US.). The miR can be added as a pool comprising up to at least 20 mIRs.<br><br>
miR-specific qPCR:<br><br>
Run PCR with the forward and reverse primers on 1 ul of each reverse transcription (RT) reaction:<br><br>
Real-time PCR on the API 7500: 95Â°C, 10 min 95Â°C, 15 sec; 60Â°C, 60 sec; 40 cycles Run melting curve analysis<br><br>
Acceptance criteria:<br><br>
Sample 1: Cycle threshold (Ct) above 40 compared to exponential area for #2 and 4. Melting peak derivative below 5000.<br><br>
Sample 2: Ct below 35<br><br>
One peak on bell shaped melting curve, unless otherwise justified the peak should be between 69 Â°C and 80 Â°C. Peak at same temperature (+/- 0.5 Â°C) as in sample 4.<br><br>
In the exponential area the crossing point at 10 times delta Rn minus the crossing point at delta Rn should be between 3.2 and 4.4:<br><br>
3.2 
 <br>
Sample 3: Ct above 40 compared to exponential area for #2 and 4.<br><br>
Melting peak derivative below 5000.<br><br>
Sample 4: One peak on bell shaped melting curve, unless otherwise justified the peak should be between 69 Â°C and 80 Â°C. Peak at same temperature (+/- 0.5 Â°C) as in sample 2.<br><br><br>
Lower case designates natural occunng nucleotides, upper case aesignates LIMM, m&gt;~ uenuies LNA methyl cytosine.<br><br>
Result (see tab 3): Seventeen primers pairs were designed according to the design rules. Twelve of the 17 assays were successfully validated according to the validation protocol corresponding to a success rate of 71%.<br><br>
Lower case designates natural occuring nucleotides, upper case designates LNA, mC denotes LNA methyl cytosine.<br><br>
Result (see tab 4): 15 primers pairs were designed according to the design rules. 12 of the 15 assays were successfully validated according to the validation protocol corresponding to a success rate of 80%.<br><br>
EXAMPLE 4: DISCRIMINATION BETWEEN TARGETS WITH A SINGLE NUCLEOTIDE DIFFERENCE.<br><br>
There are three miRs that only differ by one nucleotide from the sequence of hsa-\et-7a (table below).<br><br>
Table 5: Nucleotide sequence of let-7 family miRNAs<br><br><br>
a;:  me airrerence Detween the rour closely reiatea memDers or trie let-/ ramny is inaicatea by bold letters.<br><br>
To test if the qPCR primers for hsa-\et-7a detect the miRs with a single nucleotide difference the following miR-specific qPCR experiment was performed:<br><br>
Mixed on ice:<br><br>
RT mix:<br><br>
â¢	1Î¼1  10x PAP buffer (Epicentre Biotechnologies, Madison, WI, US.)<br><br>
â¢	1 Î¼1  1 mM ATP<br><br>
â¢	1 Î¼1 10 MM RT-primer (L2TA: 5'-ggtactagtttttttttttttttvn (SEQ ID NO 5), v designates c, g and a, n designates c, g, a and t))<br><br>
â¢	1 Ml mix of 1 mM dATP, 1 mM dCTP, 1 mM dGTP, 1 mM dTTP<br>
   0,5 Ml 2 units/Ml Transcriptor (Roche Diagnostics A/S, Hvidovre, Denmark, cat* 03 531 295 001).<br><br>
â¢	0,2 Ml (5U/MI) Poly(A) polymerase (Epicentre)<br><br>
â¢	4.5 pi water<br><br>
The following samples were prepared and incubated at 42 Â°C, 1 hour followed by 95 Â°C, 5 minutes:<br><br>
 he synthetic templates were prepared in lOng/pl phage MS2 RNA in TE. The synthetic templates are obtained from Integrated DNA technologies Inc., Coralville, IA, US.<br><br>
For qPCR 1 pi of the polyA tailing/RT reaction was used for each PCR reaction with the hsa-let-7a forward primer 5'-tGaGgtagtaggttg (SEQ ID NO 7) and reverse primer 5'-cggaggtactagtttttttttttttttAactat (SEQ ID NO 94)<br><br>
Real-time PCR was done on a ABI 7500 thermocycler by monitoring SYBRÂ® green fluorescence as a function of PCR cycle number. The PCR reaction mixture contained:<br><br>
â¢	10 pi 2x PCR mastermix (Roche cat#04 673 484 001)<br><br>
â¢	1 pi or less of the RT reaction<br><br>
â¢	1 pi TE with 2.5 Î¼M miR-specific forward primer and 2.5 Î¼M miR-speclfic reverse primer<br><br>
â¢	water to 20 UL.<br><br>
The mixture was incubated at 95Â°C for 10 min followed 40 cycles of 95Â°C for 5 sec; 60Â°C for 60 sec with measurement of the fluorescence.<br><br>
Amplification of the correct product was measured by comparing the melting curve profile of the PCR product to the melting curve profile of a PCR product obtained by amplification of a synthetic template.<br><br>
The result of the real-time PCR experiment was analysed according to standard methods (Bustin, SA (ed.) "A-Z of Quantitative PCR." International University Line (La Jolla, California, USA), 2004).<br><br>
The result Is shown In labia 6 and as amplification plots In Figure 4.<br>
nd, not detected<br><br>
2The percent signal was calculated as ioo/2(CTtemplate)~CT(/'sa",et~7a)) assuming an amplification efficiency of 100.<br><br>
EXAMPLE 5: QUANTIFICATION OF MIRS IN HUMAN BRAIN TOTAL RNA.<br><br>
In this example the copy number of miRs hsa-let-7a, hsa-miR-21, hsa-miR-27b and hsa-miR-195 in total human brain RNA (Ambion) was determined.<br><br>
Mixed on Ice:<br><br>
RT mix;<br><br>
â¢      1 Î¼l  lOx PAP buffer (New England Blolabs)<br><br>
â¢      1 Î¼ l 1 mM ATP<br><br>
â¢      1 Î¼l 10 Î¼ M RT-prlmer (L2TA: 5'-ggtactagtttttttttttttttvn (SEQ ID NO 5), v designates cytoslne, guanine and adenine residues, n designates cytoslne, guanine, adenine and thymine residues)),<br><br>
â¢      1 Î¼ l mlx of I mM dATP, I. mM dCTP, 1 mM dGTP, 1 mM dTTP<br><br>
â¢     0,5 Î¼l 2 unlta/Î¼ l [ifliiscrlptor (Roche cat#03 531 295 001, llocha Diagnostics A/S, Hvldovre, Denmark).<br><br>
â¢     0,5Î¼ l (5U/Î¼ l) Poly(A) polymerase (New England Blolabs Inc., Ipswich, MA, US)<br><br>
â¢      3 Î¼ l water<br><br>
The following samples were prepared and incubated at 42 Â°C, 1 hour followed by 95 Â°C, 5 minutes:<br><br>
EXAMPLE 5: QUANTIFICATION OF MIRS IN HUMAN BRAIN TOTAL RNA.<br><br>
In this example the copy number of miRs hsa-let-7a, /jsa-miR-21, /jsa-mlR-27b and nsa-miR-195 in total human brain RNA (Ambion) was determined.<br><br>
Mixed on ice:<br><br>
RT mix:<br><br>
â¢	1 Î¼1  lOx PAP buffer (New England Biolabs)<br><br>
â¢	1 Î¼1 1 mM ATP<br><br>
â¢	1 Ml 10 MM RT-primer (L2TA: 5'-ggtactagtttttttttttttttvn (SEQ ID NO 5), v designates cytosine, guanine and adenine residues, n designates cytosine, guanine, adenine and thymine residues)).<br><br>
.    1 Ml mix of 1 mM UATP, 1 mM dCTP, 1 mM dGTP, 1 mM UTTP<br><br>
â¢	0,5 pi 2 units/) JI Iranscrlptor (Roche cat# 03 531 295 001, hotche  Diagnostics A/S, Hvidovre, Doimimk).<br><br>
â¢	0,5 M' (5U/MI) Poly(A) polymerase (New England Biolabs Inc., Ipswich, MA, US)<br><br>
â¢	3 Î¼1  water<br><br>
The following samples were prepared and incubated at 42 Â°C, 1 hour followed by 95 Â°C, 5 minutes:<br><br><br>
Synthetic template: Equal amounts (number of copies) of synthetic hsa -let-7a, hsa-m\R-21, hsa mR-27b and hsa-miR-195 in lung/ul phage MS2 RNA in TE (obtained from Integrated DNA technologies Inc).<br><br>
TE: lOmM Trls/HCI (pH 8.0), ImM EDTA.<br><br>
For qPCR 1 pi of the  polyA tailing/RT reaction was used for  each PCR reaction with the gene-speclflc primersi<br><br>
Table 7:<br><br>
Real-time PCR was done on a ABI 7500Â® thermocycler by monitoring SYBRÂ® green fluorescence as a function of PCR cycle number. The PCR reaction mixture contained:<br><br>
â¢	10 pi 2x PCR mastermix (Roche cat#04 673 484 001)<br><br>
â¢	1 ul or less of the RT reaction<br><br>
â¢	1 pi TE with 2.5 MM miR-specific forward primer and 2.5 Î¼M miR-speclfic reverse primer<br><br>
â¢	water to 20 Ml-<br><br>
The mixture was incubated at 95Â°C for 10 min followed 40 cycles of 95Â°C for 5 sec; 60Â°C for 60 sec with measurement of the fluorescence.<br><br>
Amplification of the correct product was measured by comparing the malting curve profile of the PCR product to  the   malting curve profile of a PCR product obtained by amplification of a synthetic template.<br><br>
For each mIR, thn rem.ilt of tha real-time PCR experiment wai analyes d according to standard methods (Bustln, SA (ed.) "A-Z of Quantitative PCR." International University Line (La Jolla, California, USA),, 2004) and the Ct values from the samples without human brain RNA was used to construct standard curves.<br><br>
The Ct of the sample with human brain RNA was compared to the standard curve to determine the number of mIRs In the sample (Bustln, SA (ed.) "A-Z of Quantitative PCR." International University Line (La Jolla, California, USA), 2004).<br><br>
Results:<br><br><br>
EXAMPLE 6: DESIGN OF PREMIR QPCR ASSAYS<br><br>
This example shows that the hsa-miR-10a assay does not detect the corresponding premiR, /jsa-premiR-10a. Similarly, it is possible to use the primer design to make an assay that detects nsa-premiR-lOa but not nsa-miR-lOa.<br><br>
Mixed on ice:<br><br>
RT mix:<br><br>
â¢	1 Î¼1  10x PAP buftai (New England Biolabs)<br><br>
â¢	i Î¼1 mM ATI'<br><br>
â¢	1 Ml 10 Î¼ M r-prlmer (L2TA: 5'-ggtactagtttttttttttltttvn (SEQ ID NO 5), v designates cytoslne, guanine and adenine residues, n designates cytoslne, guanine, adenine and thymine residues)).<br><br>
â¢	1 Ml mix of 1 mM dATP, 1 mM dCTP, 1 mM dGTP, 1 mM dTTP<br><br>
â¢	0,5 Ml 2 units/Ml Transcrlptor (Roche cat# 03 531 295 001).<br><br>
â¢	0,5 pi (5U/MI) Poly(A) polymerase (New England Biolabs)<br><br>
â¢	3 Ml water<br><br>
The following samples were prepared and incubated at 42 Â°C, 1 hour followed by 95 Â°C, 5 minutes:<br><br>
For qPCR 1 pi of the polyA tailing/RT reaction was used for each PCR reaction with the specific primers in Table 9.<br><br><br>
Real-time   PCR  was  done  on  a  ABI  7500  thermocycler by  monitoring  SYBRÂ®  green fluorescence as a function of PCR cycle number. The PCR reaction mixture contained:<br><br>
â¢	10 pi 2x PCR mastermix (Roche cat#04 673 484 001)<br><br>
â¢	1 Ml or less of the RT reaction<br><br>
â¢	1 ul TE with 2.5 |JM miR-specific forward primer and 2.5 uM miR-specific reverse primer<br><br>
â¢	water to 20 \j\.<br><br>
The mixture was incubated at 95Â°C for 10 min followed 40 cycles of 95Â°C for 5 sec; 60Â°C for 60 sec with measurement of the fluorescence.<br><br>
For each miR, the result of the real-time PCR experiment was analysed according to standard methods (Bustin, SA (ed.) WA-Z of Quantitative PCR." International University Line, 2004).<br><br>
Table 10:<br><br>
The result shows that both the hsa-miR-lOa   and the hsa -premiR-10a assays detect the correct target and that there is no cross reaction to premiR-lOa or miR -10a, respectively.<br><br>
EXAMPLE 7: DETECTION OF PRE-MIR BY QPCR<br><br>
In   this   example  hsa-m\R-10a  and   the   corresponding   pre-mIR,   a-pre-mlR-10a   were detected in total human RNA.<br><br>
Mixed on ice:<br><br>
RT mix:<br><br>
â¢	1 pi 10x PAP buffer (New England Biolabs Inc., Ipswich, MA, US)<br><br>
â¢	1 pi 1 mM ATP<br><br>
â¢	1 pi 10 pM RT-primer (L2TA: 5'-ggtactagtttttttttttttttvn (SEQ ID NO 5), v designates cytosine, guanine and adenine residues, n designates cytosine, guanine, adenine and thymine residues)).<br><br>
â¢	1 pi mix of 1 mM dATP, 1 mM dCTP, 1 mM dGTP, 1 mM dTTP<br><br>
â¢	0,5 pi 2 units/pl Transcriptor (Roche cat# 03 531 295 001).<br><br>
â¢	0,5 pi (5U/pl) Poly(A) polymerase (New England Biolabs)<br><br>
â¢	3 pi water<br><br>
The following samples were prepared and incubated at 42 Â°C, 1 hour followed by 95 Â°C, 5 minutes:<br><br>
aMixture of 1 pi of 1 pg/pl heart RNA, 1 pi of 1 pg/pl brain RNA, 1 pi of 1 pg/pl liver RNA, 1 pi of 1 pg/pl lung RNA, 1 pi of 1 pg/pl kidney RNA, 1 pi of 1 pg/pl lymph RNA, 1 pi of 1 pg/pl jejunum RNA, 1 pi of 1 pg/pl colon RNA, 1 pi of 1 pg/pl breast RNA and 1 pi of 1 pg/pl leukemia RNA in TE.<br><br>
TE buffer: lOmM Tris/HCI (pH 8.0), ImM EDTA.<br><br>
For qPCR 1 pi of the polyA tailing/RT reaction was used for each PCR reaction with the specific primers in Table 11.<br><br>
Real-time PCR was done on a ABI 7500 thermocycler by monitoring SYBRÂ® green fluorescence as a function of PCR cycle number. The PCR reaction mixture contained:<br><br>
â¢	10 pi 2x PCR mastermix (Roche cat#04 673 484 001)<br><br>
â¢	1 pi or less of the RT reaction<br><br>
â¢	1 ul TE with 2.5 pM miR-specific forward primer and 2.5 pM mIR-specific reverse primer<br><br>
â¢	water to 20 pi.<br><br>
The mixture was incubated at 95Â°C for 10 min followed 40 cycles of 95Â°C for 5 sec; 60Â°C for 60 sec with measurement of the fluorescence.<br><br>
For each miR, the result of the real-time PCR experiment was analysed according to standard methods (Bustin, SA (ed.) "A-Z of Quantitative PCR." International University Line, 2004). The result is shown in tab 12.<br><br>
The result shows that the total RNA sample contains more than 106 copies of hsa-m\R-10a because the CT is below the CT of the sample with 106 copies of hsa -mlR-lOa whereas the total RNA sample contains less than 106 copies of hsa  -premiR-lOa because the CT is below the CT of the sample with 106 copies of A?sa-premiR-10a.<br><br>
EXAMPLE 8: SPECIFIC DETECTION OF PRE-MIRNAS<br><br>
Aim of the experiment: To determine if the universal reverse transcriptase quantitative PCR methodology (UniRT qPCR) described in this application could be used to detect pre-miR specifically without co-detection of the corresponding mature mIR.<br><br>
Materials: Synthetic miR 203 RNA (5'- gugaaauguuuaggaccacuag) (SEQ ID NO 103) and pre-miR 203 RNA (5'- agugguucuuaacaguucaacaguucugu- agcgcaauugugaaauguuuaggaccacuag) (SEQ ID NO 104) were selected as test subjects. The synthetic RNAs were synthesized by Integrated DNA technologies Inc., Coralville, IA, US. The RNAs were diluted to 1*106 molecules/uL in TEMS2 (TE buffer (10 mM Tris HCI pH 8, and 0.1 mM EDTA mixed with 10 ng/uL MS2 viral RNA (Roche Applied Science Inc). Primers used were miR-203.Rev (5'-tgacacggaggtactagtttttttttttttttCtag) (SEQ ID NO 105), miR-203.Fwd (5'- gtGaaatGtttaggacca) (SEQ ID NO 106) and pre-miR- 203.Fwd (5'-cagttcaacagttctgtagc) (SEQ ID NO 107). The pre-mlR-203 Fwd primer was designed in the loop structure of the pre-miR-203 molecule. Mature miR-203 template and pre-miR-203 were subject to reverse transcription using Universal cDNA synthesis kit (Exiqon AS, Cat.no #203300). Mix:<br><br>
â¢	Synthetic RNA	1*106 molecules<br><br>
â¢	5 x UniRT reaction buffer	2 uL<br><br>
â¢	lOx Enzyme mixture	l uL<br><br>
â¢	Water	up to 10 uL<br><br>
Incubate at 42Â°C for 60 minutes and heat denature at 85Â°C for 5 minutes. Dilute 10 x in water.<br>
qPCR with SYBR Green Master Mix, UniRT (Exiqon AS, Cat.No. 203400) Mix:<br><br>
â¢	Primer mix (3uM each)	1 uL<br><br>
â¢	SYBR green master mix	5 uL<br><br>
â¢	cDNA template	1 uL<br><br>
â¢	Water	3 uL<br><br>
Two primer mixes were used: 1) miR-203 ((miR-203.Fwd   and mlR-203.Rev primer) and   2) pre-miR-203 (pre-miR- 203.Fwd and miR-203.Rev primer). Templates used were mir-203   \ and pre-mir-203. Non template control (NTC) qPCR was also run for each PCR assay. All qPCR were done in duplicates.<br><br>
q-RT-PCR reactions were performed in a 384 well plate in a LightCycler 480 (Roche Diagnostics) instrument using the following PCR protocol<br><br>
1.	95Â°C for 10 minutes<br><br>
2.	95Â°C for 10 seconds<br><br>
3.	60Â°C for 1 minute<br><br>
Signal detection with SYHR green (HRM dye) setup. Step* 2-3 repeated for 45 times followed by melting curve analysis.<br><br>
Results and discussion<br><br>
Standard miR-203 prlmeta detect the miR-203 template well with a Cp value of 28,86. Also the miR-203 assay, partially detects the pre-miR-203(Cp of 30,125) since the mIR is located on the 3' end of the pre-mlr-203. Pre-miR-203 assay design detects the pre-mIR with the Cp values of 25,7 however the mature miR-203 is not detected with the pre-mlR-203 specific assay (Cp=40). This data clearly shows that  pre-mir specific assays can be designed to specifically target the pre-miR molecules.<br><br><br>
EXAMPLE 9: MICRO-RNAS DIFFERENTIALLYLY EXPRESSED IN HEART AND LIVER TISSUES<br><br>
Aim of the experiment is to determine if well expressed mIRNAs that are differently expressed between heart and liver tissue can be discriminated using a qPCR array based on the SYBR green universal reverse transcriptase quantitative PCR (UniRT qPCR) method of the present invention.<br><br>
Materials and methods<br><br>
Total RNA from liver and whole-heart was obtained from Ambion Inc., and diluted in nuclease free water to a concentration of 10 ng/uL and stored at -80Â°C. We selected a few miRNA that are known to be differentially expressed in heart and liver tissue samples from the literature (see for example: Liang, Y., et al. (2007) BMC Genomics. 8: ppl66 and Landgraf P., et al.(2007) Nature Biotechnol. (9): pp 996-7). The miR selected were hsa-miR-1, hsa-miR-126 and hsa-miR-133b (heart) and hsa-miR-192, hsa-miR-122*, hsa-miR-194 and hsa-miR-122 (liver).<br><br>
RT reactions were performed in triplicate for each tissue and non enzyme control (NEC)) with Universal cDNA synthesis kit (Exiqon AS, Cat.no #203300)<br><br>
â¢	total RNA	20 ng<br><br>
â¢	5 x UniRT reaction buffer	4 uL<br><br>
â¢	Enzyme mixture      2 Ul<br><br>
â¢    Water	up to 20 ul_<br><br>
Incubate at 42Â°C for 60 minutes and heat denature at 85Â°C for 5 minutes. Dilute the cDNA 1/100 in nuclease free water.<br><br>
Quantitative polymerase chain reaction (q-PCR), step 2 of the protocol, was performed as follows:<br><br>
5 ul. diluted cDNA was mixed with 5 uL SYBR Green Master Mix, UniRT (Exiqon AS, Cat.No. 203400) into the 384 well plate with desiccated primer sets of the seven miR assays described below. The plate was sealed and directly put on to the LightCycler for amplification and anallysis.<br><br>
q-RT-PCR reactions ware performed In a LightCycler 400 (Roche Diagnostics) using the following PCR protocol <br><br>
1.95Â°C for 1.(1minute <br><br>
2.95Â°C for 10 seconds<br><br>
3.60Â°C for 1 minute<br><br>
Signal detection with SYBR green (HRM dye) setup. Steps 2-3 repeated for 45 times followed by melting curve analysis.<br><br>
Standard data analysis was performed on the LC480 raw data using the supplied data analysis software (Roche Diagnostics). Cp values were collected as Abs Quant/second derivatives max.<br><br>
For this experiment of looking at examples of well expressed miRNA genes from both liver and heart, we used mean of raw data Cp values without normalization or calibration. This is done since normalization between tissues is not very accurate way of estimating miRs from different sources. Then we compare the difference between the two tissues in terms of A Cp values. Note that a difference of 1 in Cp value represents approximately a two fold difference in expression.<br><br>
Results and discussion<br><br>
We selected total of 7 miRNAs that are known previously from the literature, to be differentially expressed between heart and liver tissue. The results are shown in table 15 and fig. 7. The data obtained showed that the three genes miR-1, miR-133b and 126 are all showing much higher expression in heart samples than liver samples. Similarly the miR-192, miR-194 and miR-122 and 122* are showing much higher expression in liver than heart. The difference is ranging from 2.6 to 12.9 Cp which corresponds to a range of from 5 fold to over 1000 fold difference In expression. In general the differentially expressed mIRs described from the literature are easily distinguished by the UnIRT expression platform using the assays described here.<br><br>
EXAMPLE 10: COMPARISON OF miR SPECIFIC ASSAYS DESIGNED IN PRESENT INVENTION TO COMPETING METHOD USING PURE DNA PRIMERS<br><br>
The LNA based design described in the present invention was compared to a commercial DNA based product, the miScript Reverse Transcription Kit (Qiagen, Cat. no. 218060, QIAGEN GmbH, Hilden, Germany). This DNA based product also relies on a miRNA 3'- polyadenylation step followed by reverse transcription with a DNA based poly dT primer where both reactions occur in a one-tube reaction. Because both these methods use the same enzymatic steps a comparison very well illustrate the surprising advantages of the present LNA based method since the miScript does not include LNA in the primers. Another difference is that the miScript uses a reverse primer specific to the universal tag added with the RT primer whereas the LNA based reverse primer of the present method is specific to the miRNA being detected.<br><br>
Reverse transcription was performed on a dilution series of synthetic mIRNA target (obtained from Integrated DNA technologies Inc., Coralville, IA, US.), with a background of 10 ng/ul MS2 bacteriophage RNA (Roche Applied Science, Catalog Number 10165948001), using either Universal cDNA Synthesis Kit (Exiqon A/S, Vedbaek, Denmark., Prod. No. 203300) or miScript Reverse Transcription Kit (Qiagen, Cat. no. 218060, QIAGEN GmbH, Hilden, Germany) according to the manufacturer's instructions.<br><br>
SECOND STEP<br><br>
On the cDNA obtained using the Universal cDNA Synthesis Kit (Exiqon, Prod. No. 203300), qPCR was performed using SYBR Green master mix, Universal RT (Exiqon, Prod. No. 203400) and the primer sets described in Table 17. On the cDNA obtained using miScript Reverse Transcription Kit (Qiagen, Cat. no. 218060), qPCR was performed using miScript SYBR Green PCR Kit (Qiagen, Cat. no. 218073). In both cases the amplification and detection was performed on a Roche LC480 LightCycler (Roche Diagnostics A/S, Hvidovre, Denmark), using the cycling conditions instructed by the manufacturer. Each cDNA/assay combination was run in triplicate.<br><br>
RESULTS<br><br>
Figure 8 shows the triplicate  Cp values versus template concentration obtained In the experiment for each of the four tested mIRNAs. In all four compared assays, the method of the present Invention was more sensitive (black lines connecting diamonds) than the mIScript assay (gray lines connecting sphaeres) as Indicated by the lower Cp values consistently obtained with the assay of the present Invention. In the case of /Â»sa-let-7a, there was a 10-fold improved sensitivity of the assay of the present Invention, measured as the lowest copy-number detected quantitatively. For  sa-miR-143 and a-mlR-155 the difference In sensitivity was 100-fold better with the assay of the present Invention. For /7sa-mlR-l with a very low gc-content (see table 16) and thus low melting temperature, the alternative assay was unable to detect the template even at highest concentration of RNA. The assay designed in the present invention quantitatively detected an equivalent of as little as 10 miRNA copies in the PCR reaction. This data clearly shows that the assay of the present invention is surprisingly more sensitive. We ascribe the improvement in sensitivity partly to the design of the extension primer which comprise both a 5' "tag-sequence" (R ) as well as a 3' "anchor sequence" (R ) and partly to the design of template-specific (e.g. miR-specific) forward and reverse primers of the qPCR reaction which includes LNA. Our results indicate that the gene specific design of the forward and reverse primers allow more sensitive detection than with pure DNA primers using a universal reverse primer.<br><br>
TABLE 18: Nucleotide sequences<br><br>
Lower case designates natural occuring nucleotides, upper case designates LNA, mC denotes LNA methyl cytosine. v is either an adenine residue, a guanine residue, or a cytosine residue and n is either an adenine residue, a guanine residue, a cytosine residue or a thymine residue.<br><br><br><br>
CLAIMS<br><br>
1.	A method for amplification of a specific RNA molecule In a sample, the method comprising the steps of:<br><br>
a) adding poly-A tails to a population of RNA molecules in a sample;<br>
 extenslon<br><br>
b) producing   CDNA molecules of the poly-A-tailed RNA molecule  lining  an primer In a reverse transcription   reaction and<br><br>
c) amplifying the C DNA molecules by PCR using n forward primar reverse  primer both of which  specific for  specific RNA molecule. ,<br><br>
wherein said extension primer comprise an anchor sequence.<br><br>
2.	The method of claim 1, wherein the specific RNA molecule Is a small, non-coding RNA.<br><br>
3. The method of claims 1 or 2, wherein the specific small RNA molecule Is a mlcroRNA.<br><br>
4.	The method according to of any of claims 1, 2 or 3, wherein the extension primer has a length in the range from 10 to 100 nucleotides.<br><br>
5.	The method according claim 4, wherein the extension primer has a length in the range from 15 to 45 nucleotides.<br><br>
6. The method according to any one of claims 1-5, wherein said extension primer is a nucleotide sequence according to formula III:<br>
R -(T)y -R     (III)<br>
wherein R    is a 5'-end nucleotide sequence, (T)y is a central part of y consecutive thymine residues, wherein y is an integer of from 1 to 100, and R   is a J-end nucleotide sequence.<br><br>
7. The method according to claim 6, wherein R   is a nucleotide sequence with a length of from 1 to 30 nucleotides.<br><br>
8.	The  method a to claim 7, wherein R   Is a  nucleotide  sequence  with a longlh of from 6 to 10 nucleotide <br><br>
9.	The method according   to claim B, wherein R   Is a nucleotide sequence  with n longlh or 0 nucleotides.<br><br>
10. The method according to any of claims 6 to 9, wherein y Is   Integer from H to 50,<br><br>
11.	The method according to claim 10, wherein y is an Integer from 5 to 21.<br><br>
12.	The method according to claim 11, wherein y Is 12, 13,14, 15, 16, 17 or 18.<br><br>
13.	The method according to claim 12, wherein y is 15.<br><br>
14.        A method according to any one of claims 6-13, wherein R    Is a sequence motif VN, consisting of two 3"-end terminal nucleotide residues, wherein V Is either an adenine residue, a guanine residue, or a cytoslne residue and N is either an adenine residue, a guanine residue, a cytoslne residue or a thymine residue.<br><br>
15.	The method according to any one of claims 6-13, wherein R   Is a sequence motif VNN, consisting of the three y-end terminal nucleotides, wherein V is either an adenine residue, a 15      guanine residue, or a cytosine residue and N is either an adenine residue, a guanine residue, a cytosine residue or a thymine residue.<br><br>
16.	The method according to any one of claims 1-15, wherein the extension primer comprises at least one LNA.<br><br>
17.	The method according to any one of claims 1-16, wherein the forward primer has a length in the range of from 10 to 100 nucleotides.<br><br>
18.	The method according to any one of claims 1-17, wherein the forward primer comprises at least one LNA.<br><br>
19.	The method according to any one of claims 1-18, wherein the forward primer is designed to specifically hybridize to the complement DNA molecule of a target RNA molecule.<br><br>
20.	The method according to any one of claims 1-19, wheraln the laverws primer Is a nucleotide sequenut according to formula II:<br>
R-{T)X-R   (II)<br>
3 wherein R   Is a 5'-end nucleotide sequence, (T)x is a central part of x consecutive thymine<br>
4 5      residues, wherein x is an Integer of from 1 to 100, and R   is a y-end nucleotide sequence that specifically hybridizes to a nucleotide sequence of a target RNA molecule.<br><br>
21.	The method according to claim 20, wherein R   is a nucleotide sequence with a length of from 1 to 30 nucleotides.<br><br>
22.	The method according to any of claims 20 or 21, wherein x is 5 to 50.<br><br>
23. The method according to daim 22, wherein x is 5 to 21.<br><br>
24.	The method according to claim 23, wherein x Is 15.<br><br>
25.	The method according to any of the preceding claims wherein x of formula (II) equals y of formula (III).<br><br>
26.	The method according to any one of claims 20-25, wherein R   has a length In the range of from 1 to 10 nucleotides.<br><br>
27.	The method according to any one of claims 20-26, wherein R    comprises at least one LNA.<br><br>
28.	The method according to claim 27, wherein R   contains only one LNA.<br><br>
29.	The method according to any one of claims 20-28, wherein R   Is designed to specifically hybridize to the 3"-end of a target RNA.<br><br>
30. The method according to any one of claims 27-29, wherein the LIMA Is situated In the 5' position or the position adjacent to the 5' position of the R4 part of the reverse primer.<br><br>
.31. A method for measuring the amount of a target microRNA In a sample from a living organism, the method comprising the steps of:<br><br>
a)	amplifying the target microRNA according to the method of any one of claims 1 to 30.<br><br>
b)   measuring the amount of the amplified DNA molecules.<br><br>
32.	The method according to claim 31, wherein the amount of the amplified DNA molecules is measured using a fluorescence-based quantitative real-time PCR method.<br><br>
33.	The method according to any of claims 31 or 32, wherein the amount of the amplified Â® 10      DNA molecules is measured using SYBR    green dye.<br><br>
34.	A kit for detecting at least one target RNA comprising at least one primer set specific for the detection of a the target RNA, the primer set comprising:<br><br>
a)	an extension primer of formula III: R -(T)y -R     (III)<br>
wherein R   is a 5'-end nucleotide sequence, (T)y is a central part of y consecutive thymine 15      residues, wherein y is an integer of from 1 to 100, and R ' Is a 3"-end nucleotide sequence.<br><br>
b)	a reverse primer of formula II: R -(T)x -R     (II)<br><br>
wherein R   is a 5'-end nucleotide sequence, (T)x is a central part of x consecutive thymine 4 residues, wherein x Is an integer of from 1 to 100, and R    is a 3"-end nucleotide sequence that specifica lly hybridizes to a nucleotide sequence of a target RNA molecule.<br><br>
c) a forward primer<br><br>
35.	The kit according to claim 34 for detecting at least one mammalian target microRNA comprising at least one primer set specific for the detection of a target microRNA.<br><br>
36.	The kit according to claim 34 or 35, wherein the forward primer has a length In the range from 10 to 100 nucleotides.<br><br>
37.	The kit according to any one of claim 34 to 36, wherein the forward primer comprises at least one LNA.<br><br>
38. The kit according to any one of claim 34 to 37, wherein the reverse primer comprises at least one LNA.<br><br>
39.	The kit according to any one of claims 34 to 38, wherein the forward primer is designed to specifically hybridize to the complement DNA molecule of a target mlcroRNA molecule.<br><br>
40.	The kit according to any of ciaims 34 to 39, wherein the reverse primer is a reverse primer according to any of claims 27 to 30.<br><br>
41.	The kit according any of claims 34 to 40, wherein the at least one mammalian target microRNA Is a human microRNA.<br><br>
42.	The kit according to any one of claims 34 to 41, wherein the at least one mammalian target microRNA Is selected from the group  consisting of all known human microRNAs available from the miRBase (http://mlcroma.sanger.ac.uk/).<br><br>
43.	The kit according to any one of claims 34 to 41, wherein the at least one mammalian target microRNA Is a murine microRNA.<br><br>
44.	A high-throughput method for detecting or measuring the amount of specific target microRNAs in a sample from a living organism, according to any one of claims 1 to 33, the method further comprising a distribution step in which the combined poly-A-tailing and reverse transcription reaction is aliquoted into Individual wells of a microtiter plate containing microRNA specific primer sets of forward and reverse primers, resulting in the steps of:<br><br>
c)	adding poly-A tails to a population of small RNA molecules In a sample and producing cDNA molecules of the poly-A-tailed small RNA molecules using an extension primer in a reverse transcription reaction; and<br><br>
d)	pipetting aliquots of the combined poly-A-talling and reverse transcription reaction into individual wells of a microtiter plate<br><br>
e) amplifying specific target microRNAs in individual wells of a microatlter plate containing microRNA specific primer sets<br><br>
f)   measuring  the amount of specific microRNA molecules in individual wells. being performed in a fashion compatible with a high-throughput experimental set-up.</td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/viewdoc.aspx?id=ZIpDhNWAWMh4ycY7KbXp1w==&amp;amp;loc=egcICQiyoj82NGgGrC5ChA==" target="_blank" style="word-wrap:break-word;">http://ipindiaonline.gov.in/patentsearch/GrantedSearch/viewdoc.aspx?id=ZIpDhNWAWMh4ycY7KbXp1w==&amp;amp;loc=egcICQiyoj82NGgGrC5ChA==</a></p>
		<br>
		<div class="pull-left">
			<a href="269399-a-process-for-preparing-5-r-2-5-6-diethyl-indan-2-ylamino-1-hydroxy-ethyl-8-hydroxy-1h-quinolinone-2-one-salts.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="269401-automotive-vehicle-having-engine-air-guide-passage.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>269400</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>6201/CHENP/2011</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>44/2015</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>30-Oct-2015</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>19-Oct-2015</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>29-Aug-2011</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>EXIQON A/S</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td>SKELSTEDET 16, DK-2950 VEDBAEK</td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>BUSK, PETER, KAMP</td>
											<td>MARS ALLE 4, DK-2860 SOBORG</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>C12Q1/68</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>PCT/DK2010/050029</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td>2010-02-02</td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td>PA 2009 01038</td>
									<td>2009-09-17</td>
								    <td>Denmark</td>
								</tr>
								<tr>
									<td>2</td>
									<td>PA 2009 00156</td>
									<td>2009-02-02</td>
								    <td>Denmark</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/269400-method-for-amplification-of-a-specific-micro-rna-species by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 07:34:19 GMT -->
</html>
