<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/269748-method-for-specific-amplification-of-polynucleotides by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 06:37:13 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 269748:METHOD FOR SPECIFIC AMPLIFICATION OF POLYNUCLEOTIDES</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">METHOD FOR SPECIFIC AMPLIFICATION OF POLYNUCLEOTIDES</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>The present invention provides a method for amplifying a pool of polynucleotide molecules in a sample, characterized by the steps of a) obtaining a sample or RNA and reverse transcription of entire RNA molecules thus creating full length cDNA or obtaining a sample of full length cDNA, b) tailing the 3&quot; end of the transcribed cDNA with a polynucleotide tail after the 3&quot; end, c) amplification of the cDNA using a pair of primers, wherein a first (3&quot;) primer is specific for the 5&quot; end of the cDNA and a second (5&quot;) primer is specific for the a upstream portion of the polynucleotide tail and the next 1 to 10 nucleotides upstream of the 3&quot; polynucleotide tail of the cDNA.</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td>The present invention relates to the field of specific polynucleotide amplification.<br>
To understand the causalities of the different states life presents itself, one needs to<br>
find the factors and processes that lead to it. One central element in this quest is the ge-<br>
nome where information is stored in such a fashion that it can be propagated through the<br>
generations in a highly conserved manner. In executing the information contained in the<br>
genome, DNA is transcribed into RNA and further translated into protein. Studying the<br>
genome (DNA), transcriptome (RNA) and proteome (protein) has in recent years greatly<br>
contributed to our understanding of the molecular basis of life.<br>
An array of possibilities exists to compare RNA samples in vitro. An early approach<br>
is the method of differential hybridization or differential screening (Rebagliati et<br>
al., 1985). It involves picking at random clones (or sequences) from cDNA libraries and<br>
checking for their presence in different concentration in samples of RNA. An enormous<br>
effort has to be put into identifying a single differentially expressed transcript as their<br>
population is thought to be below 1%. Subtractive hybridization has been proposed to<br>
overcome this problem by enriching for differentially expressed transcripts prior to screening.<br>
Another way of addressing the problem of abundance was made by introducing high-<br>
density micro arrays. In using micro array technology 100s to 1000s of cDNAs, corresponding<br>
to RNAs of genes of interest, are spotted and bound onto a surface. They are<br>
normally derived from sequenced EST libraries and - if non redundant - can cover as many<br>
transcripts as spots fit onto the membrane. In short, RNA extracted from cells or tissues of<br>
interest are labeled, e.g. during reverse transcription, and hybridized to these microarrays.<br>
The signal intensity of corresponding spots between two or more RNA preparations can<br>
then be compared. Even though this technology has produced massive amounts of data,<br>
there are two fundamental drawbacks of all these hybridization techniques. One is cross<br>
hybridization. This means that two DNA molecules do not need to be complementary over<br>
their entire sequence to hybridize. Therefore this introduces ambiguity towards the identity<br>
of the molecule that hybridizes (binds) to the spot. Secondly one can only compare molecules<br>
that have been spotted, limiting the search to sequences that are present in the library<br>
used to generate the array.<br>
An alternative approach was introduced in 1992 (Liang et al., 1992; Welsh et al.,<br>
1992) that today is generally known as "differential display" (DD). In DD the total<br>
mRNAs population of a sample is first divided into defined pools of cDNAs during reverse<br>
transcription (RT). This is accomplished by priming the RT with one, two or x base<br>
anchored oligo dT primers, subdividing the samples into 3, 12 or 3x4x-1 fractions. These<br><br>
anchor bases hybridize to the bases immediately upstream, of the poly A tail. Therefore selecting<br>
for mRNA molecules to be reverse transcribed whose bases immediately upstream<br>
of the poly A tail are complementary to the anchor bases of the primer. In a subsequent<br>
Polymerase chain reaction (PCR) arbitrary primers and anchored primers are used to amplify<br>
defined pools of cDNA. Corresponding pools are run side by side on a gel that separates<br>
them by size. Differences of expression levels are reflected in differences in band intensity.<br>
Bands of interest are cut out of the gel, reamplified and sequenced. One drawback<br>
of the original DD is that it uses arbitrary primers for amplification. Such a shotgun approach<br>
can reach full transcriptome coverage only statistically, by covering the transcrip;<br>
tome several times. Therefore some RNA molecules will necessarily be presented several<br>
times in the display and others might not be present at all.<br>
Recently methods have been introduced that amplify pools of RNA in a more systematic<br>
fashion (Prashar et al., 1996; Prashar et al., 1999). In principle, three additional<br>
steps are involved. After first strand cDNA synthesis second strand synthesis is carried out<br>
to get double stranded DNA that can be cut with a restriction enzyme. In a third step an<br>
adaptor is ligated to the 5' ends of the fragments. These fragments are then PCR amplified<br>
with one primer being complementary to the 5' adaptor sequence and an anchor primer<br>
that was used for RT. It was estimated by the authors that each 6-base cutting restriction<br>
enzyme cuts 8% of the cDNAs at a position between 50 and 400 bases from the 3' poly A<br>
tail of the rnRNA. Therefore more than 12 restriction enzymes will be necessary to approach<br>
a complete coverage of the transcriptome. A somewhat more elegant variety of this<br>
method was described by Matz et al. (1997)with the introduction of "ordered differential<br>
display" that takes advantage of the PCR suppression effect.<br>
However, besides covering the transcriptome only statistically at one level or another,<br>
all afore mentioned DD methods suffer from another drawback. They will give the<br>
investigator only part of the mRNA sequence that is differentially expressed. Downstream<br>
of these DD technologies one will still have to either screen cDNA libraries or where<br>
available online sequence libraries for full length clones or employ methodologies that can<br>
discover 5' and 3' sequences from a partially known mRNA sequence such as RACE<br>
(Frohman et al., 1988). Even identifying such full length transcripts will still not yield certainty<br>
at least in eukaryotes that the mRNA found is the mRNA that generated the signal as<br>
in eukaryotes a major contributor to genome complexity is splicing. Therefore the sequence<br>
found on the display can be common to several splice variants. Furthermore different<br>
splice variants of a gene may have different functions, making it highly desirable to<br>
know the correct splice variant that generated the signal.<br><br>
A goal of the present invention is to provide a method that can amplify (enrich for)<br>
defined subpopulations of transcripts in their entire length with full coverage of all mRNA<br>
molecules present in the sample.<br>
The present invention now provides a method for amplifying a pool of polynucleo-<br>
tide molecules in a sample, characterized by the steps of:<br>
a)	obtaining (or providing) a sample of RNA and reverse transcription of entire<br>
RNA molecules thus creating full length cDNA or obtaining (or providing) a<br>
sample of full length cDNA,<br>
b)	tailing the 3' end of the transcribed cDNA with a polynucleotide tail after the 3'<br>
end,<br>
c)	amplification of the cDNA using a pair of primers, wherein a first (3') primer is<br>
specific for the 5' end of the cDNA and a second (5') primer is specific for the upstream<br>
portion of the polynucleotide tail and the next 1 to 10 nucleotides (i.e. the<br>
next 1 or 2, 3, 4, 5, 6, 7, 8, 9 or 10 nucleotides), preferably 1 to 5 nucleotides,<br>
even more preferred 3 to 5 nucleotides, upstream of the 3' polynucleotide tail of<br>
the cDNA. Of course the second primer can also comprise more specific nucleotides<br>
further upstream of the 1 to 10 nucleotide region upstream of the tail specific<br>
region (complement).<br>
A full length cDNA can be created by a reverse transcription process described below,<br>
or by methods known in the state of the art. Methods to generate full length cDNA are<br>
abundantly known to the skilled man in the art, e.g. disclosed in patents US 5,962,271 and<br>
US 5,962,272. Such generated cDNA can also be used as starting material to pool full-<br>
length cDNA into different fractions. These methods use the template switching ability of<br>
reverse transcriptase to add a defined sequence to the 5' end of the cDNA in a cap dependent<br>
manner. The primer that is needed in the subsequent amplification will therefore have a<br>
portion that will be complementary to the so generated 3' end of the cDNA and will define<br>
what fraction of cDNA will be amplified through several, e.g. 1 to 10, additional bases defined<br>
by the 5' end of the mRNA molecules amplified.<br>
Therefore it is preferred that a tail is added to the cDNA by the template switching<br>
ability of the reverse transcriptase, preferably in a cap dependent manner and the second<br>
amplification primer (5' primer) comprises complementary nucleotides to the so added tail<br>
and oligo-dCs and 1 to 10 nucleotides upstream of the oligo-dC sequence.<br><br>
As used herein, the term "reverse transcriptase" relates to any polymerase that has<br>
reverse transcriptase activity and can be used to synthesize cDNA from RNA.<br>
As used herein, the term "full length cDNA" is defined as DNA that includes a sequence<br>
complementary to the RNA sequence from the first base to the last base of the<br>
RNA. In case of RNA molecules that have a cap and/or a polyA tail as is the case for e.g.<br>
most eukaryotic mRNA, "full length cDNA" is defined as DNA that includes a sequence<br>
complementary to the RNA sequence from the first base after the cap, e.g. the RNA 7-<br>
methylguanosine cap or RNA m7G cap, to the last base before the poly A tail of the RNA<br>
used as a template.<br>
As used herein, the term "full length amplification product" is defined as DNA that<br>
includes a sequence complementary to the RNA sequence from the first base to the last<br>
base of the RNA. In case of RNA molecules that have a cap and/or a polyA tail as is the<br>
case for e.g. most eukaryotic mRNA, "full length amplification product" is defined as amplification<br>
product that includes a sequence complementary to the RNA sequence from the<br>
first base after the cap to the last base before the poly (A) tail of the RNA used as a template.<br>
A central element of this invention is the fact that full length RNA molecules can<br>
be represented in subpopulations that are defined on their 5' and/or 3' end. In principle that<br>
means that after the generation of full length cDNA by any means known in the art, the 5'<br>
and/or 3' sequence can be used to formulate a set of primer combinations (primer matrix)<br>
that has the following quality.<br>
1)	Each cDNA will be amplified by only one primer pair in the absence of miss priming.<br>
Therefore full transcriptome coverage can be reached without redundancy.<br>
2)	Each cDNA sequence within such a defined pool will encode the full amino acid<br>
sequence of the protein translated from the corresponding RNA and also<br>
3)	Define its transcription start site on its gene (there could be different RNA molecules<br>
transcribed from the same gene that can have different transcription start<br>
sites).<br>
In comparing quality (1) to original DD one can state that therein the arbitrary primers<br>
have the property of either not being complementary to a specific cDNA, being comple-<br><br>
mentary once or being complementary more than once. Therefore each cDNA can be represented<br>
either not at all, or once or more than once. When using alternative methodologies<br>
that include a restriction enzyme, each restriction enzyme will cut a specific cDNA either<br>
not at all or once or more than once. Therefore truly complete transcriptome coverage can<br>
never be reached. And increasing the probability for mRNAs to be represented at least one<br>
time in the display will mean to increase redundancy.<br>
Qualities (2) and (3) can also not be reached by differential display techniques used<br>
to date without employing downstream technologies such as RACE.<br>
The present invention provides methods for amplifying and analyzing the transcriptome<br>
through qualitative and quantitative detection of differentially expressed RNA molecules.<br>
The technique according to the invention is an organized full length expression display.<br>
In a preferred embodiment the method according to the invention comprises a<br>
primer, which is specific for the (preferably 3' poly A) tail of the RNA or mRNA and the<br>
next 1 to 10 nucleotides, preferably 1 to 5 nucleotides, even more preferred 3 to 5 nucleo-<br>
tides, upstream of the (poly A) tail, that is used for the reverse transcription and/or one<br>
amplification primer is specific for a corresponding (5' poly-T) stretch of the cDNA, which<br>
is complementary to the (3' poly A) tail of the RNA or mRNA, and the next 1 to 10 nucleotides,<br>
preferably 1 to 5 nucleotides, even more preferred 3 to 5 nucleotides, downstream<br>
of the corresponding (5' poly-T) stretch of the cDNA. Accordingly in addition to<br>
the selective 3' primer described above a selective 5' primer can be used to increase the selective<br>
power of the primer pair and thus increase the number of cDNA pools defined by<br>
the specific nucleotides next to the end sequence anchors.<br>
Preferably the primer for reverse transcription consists of deoxynucleotides.<br>
Generally, the primers throughout this specification can be of any nucleotides, such<br>
as deoxyribonucleotides or ribonucleotides. Although deoxynucleotides are preferred in<br>
most cases, ribonucleotide primers are also feasible. Therefore a deoxyribonucleotide in a<br>
given primer, such as given in examples below, may be substituted by one of its ribonucleotide<br>
analogue (e.g. dG by G, dA by A, dC by C, dT by T or U) or other nucleotide analogues.<br>
Preferably the tailing of the 3' end is performed using terminal transferase. - Although<br>
other tailing methods are also disclosed, like ligation of a tail sequence, which can<br><br>
be e.g. a defined arbitrary sequence. The terminal transferase can add a certain number of<br>
nucleotides preferably uniformly selected from one nucleotide type. Any other means for<br>
tailing, adding a tail sequence can also be used, e.g. employing the template switching<br>
ability of the reverse transcriptase or by ligating the tail sequence which can be uniformly<br>
of one type of nucleotides or of varying nucleotides. Such a tail is preferably a sequence in<br>
the range between 5 and 500 nucleotides, more preferred less than 400, less than 300, less<br>
than 200, less than 100, less than 50 or less than 30 nucleotides.<br>
In a preferred embodiment the amplification of the cDNA sequences using a selective<br>
primer pair is performed by PCR. PCR or polymerase chain reaction is a well established<br>
technique known in the state of the art to amplify polynucleotides using a polymerase,<br>
nucleotidetriphosphates, primers, buffer substances and cofactors such as Mg ions<br>
and a temperature protocol for hybridization, polymerisation and melting or strand separation.<br>
Preferably the first primer and/or second primer for the amplification consists of<br>
deoxynucleotides.<br>
In a preferred embodiment the method according to the invention comprises an additional<br>
step of separation of the amplification products according to their length. The<br>
separation of the created polynucleotides renders a characterization, isolation and sequence<br>
determination possible for each pool after the selective polymerisation which allows a significant<br>
advantage for this preferred embodiment in practice.<br>
Preferably the separation is performed by gel electrophoresis, preferably agarose<br>
gel electrophoresis, polyacrylamide gel electrophoresis, or capillary electrophoresis.<br>
Even more preferred the method according to the invention comprises an additional<br>
step of determining the identity or sequence of the amplification products. Sequence determination<br>
can be done for example by gel sequencing, e.g. PCR with dideoxynucleotides,<br>
or in an automated sequencer.<br>
Preferably the step of determining the identity or sequence of the amplification<br>
products is performed by an automated process on a chip.<br>
The sample used for reverse transcription preferably contains total RNA or mRNA,<br>
preferably purified RNA or mRNA, from a specimen. Total RNA includes, but is not limited<br>
to, protein coding RNA also called coding RNA such as messenger RNA (mRNA)<br><br>
and non protein coding RNA (non coding RNA or ncRNA), such as ribosomal RNA<br>
(rRNA), transfer RNA (tRNA), micro RNA (miRNA), small interfering RNA (siRNA),<br>
piwi-interacting RNA (piRNA), small nuclear RNA (snRNA) and small nucleolar RNA<br>
(snoRNA). Every one of these classes, alone or together with one or more classes, can be<br>
analysed according to methods that are within the scope of the invention.<br>
These RNA molecules are preferably purified by affinity chromatography using e.g.<br>
an oligo-dT-cellulose column. Thus mRNA with a poly A tail can be preselected.<br>
It is preferred in the inventive method that reverse transcription is carried out at<br>
high stringency conditions to increase selectivity of the anchored primers, which can be<br>
used as a pool of anchored primers in one polymerisation reaction, as well as separate<br>
primers for different polymerisation reactions. In the present invention the phrases "stringent<br>
hybridization conditions" or "stringent conditions" refer to conditions under which a<br>
primer of the invention will hybridize to its target sequence, but to a minimal number of<br>
other sequences. High stringency can be achieved for example by using a high annealing<br>
temperature alone or in combination with a chemical additive such as dimethylsulfoxide<br>
(DMSO), formamide, tetramethylammonium chloride (TMAC) and Tetramethylammo-<br>
nium oxalate (TMAO) (Kovarova et al. 2000). Examples for high stringency hybridization<br>
conditions are given in Sambrook et al., Molecular Cloning: A Laboratory Manual (Cold<br>
Spring Harbor Laboratory, N.Y. 1989) and Ausubel et al., Current Protocols in Molecular<br>
Biology, Greene Publishing Assoc. and Wiley and Sons, N.Y. 1994. Under standard PCR<br>
conditions often an annealing temperature is chosen that is about 5 degree Celsius below<br>
the melting temperature of the primer. Keeping reaction conditions constant the melting<br>
temperature in turn is mainly dependent on the length and the GC-content of the oligonu-<br>
cleotides, which can be calculated by common means known in the art. Under more stringent<br>
 conditions the annealing temperature is raised above the temperature used in standard<br>
conditions to decrease the likelihood of mispriming. The annealing temperature can be<br>
raised to a temperature that will still allow enough priming events for the amplification to<br>
occur. For instance, if a primer has a melting temperature of 50°C, and an annealing temperature<br>
of 45°C would be chosen under standard conditions, an annealing temperature of<br>
50°C would thus be more stringent then 45°C. For example primers of the type dT16VN<br>
can lead to strong mispriming under normal RT conditions and therefore decrease the<br>
specificity of the cDNA pools synthesized (Mizuno et al., 1999). Analogous results have<br>
been reported for AMV and HIV-1 reverse transcriptase. Primer promiscuity can be decreased<br>
by increasing temperature during annealing and reverse transcription. For this purpose<br>
thermostable enzymes can be used that exhibit reverse transcriptase activity or reverse<br>
transcriptase enzymes can be thermo protected by adding thermo protectants such as<br><br>
trehalose to the reaction (Carninci et al, 1998). An added benefit of using trehalose is that<br>
the amount of full length cDNA increases (Carninci et al., 1998). This property was also<br>
described by Spiess et al. (2002). In addition Spiess showed that betaine can also increase<br>
the amount of full length cDNA and that a combination of betaine and trehalose gave the<br>
best result (Spiess et al., 2002). Such RT can be derived from moloney murine leukemia<br>
virus (M-MLV) (US 4,943,531 A) or in a more preferred embodiment M-MLV lacking<br>
RNAse H activity (US 5,405,776 A). Yet another possibility to raise primer specificity is<br>
to include competitive oligo nucleotide blockers as described by Mizuno et al. (1999).<br>
Therefore the present invention also comprises a method wherein high stringency conditions<br>
are used for the processes of reverse transcription and/or the amplification.<br>
Preferably, in the method according to the invention the selectivity of the reverse<br>
transcription and/or amplification reaction, in particular the polymerase chain reaction, is<br>
increased by the utilization of trehalose, betaine, tetramethylammonium chloride, tetrame-<br>
thylammonium oxalate, formamide and oligo-blockers, or by the utilization of dimethyl-<br>
sulfoxide during the polymerase chain reaction, to reduce the occurrence of mispriming<br>
and increase the amount of full length cDNA.<br>
As all reverse transcriptases have a high error rate it is preferred that a proofreading<br>
activity is included in the reaction. Enzymes that have such an activity through their 3'--&gt;5'<br>
exonuclease function are e.g. polymerases such as PFU, Ultma, Vent, Deep Vent, PWO<br>
and Tli, or E.coli. exonuclease III. In addition to reducing errors introduced into a cDNA<br>
during the RT the inclusion of a proofreading activity can also enhance full length cDNA<br>
synthesis (Hawkins, 2003).<br>
It is further preferred that such a proofreading activity is also included in the PCR<br>
step of the method. On the one hand this will reduce errors introduced into the sequence<br>
during PCR as well as extend the size of cDNA molecules that can be amplified (Barnes,<br>
1994). As such a 3'~&gt;5' exonuclease activity can degrade primers used in the reaction it is<br>
preferred to protect primers from degradation, e.g. by introducing a phosphorothioate bond<br>
at the 3' terminal of the primer (Skerra, 1992; Di Giusto, 2004)or by using LNAs (locked<br>
nucleic acids).<br>
It is preferred that during reverse transcription and/or PCR, enzymes or enzyme<br>
combinations are used that have a high processivity and fidelity.<br>
The present invention provides a method for full transcriptome coverage and no<br>
redundancy at least in principle. However, methodical properties such as mispriming and<br><br>
polymerase stop can influence the outcome nevertheless. Coverage of long mRNAs therefore<br>
depends on the used enzymes. Proofreading enzymes, used for long PCR methods,<br>
enlarge the scope of transcriptome coverage to at least 99% under ideal conditions for the<br>
given enzymes and method parameters (e.g. long extension times). Misanealing, which can<br>
be minimized by e.g. stringency and proofreading activities, may lead to a limited redundancy<br>
of the covered transcriptome.<br>
The maximal length of an RNA molecule that can be displayed depends in particular<br>
on the quality of the reverse transcription and the amplification reaction, e.g. PCR. In<br>
principle long PCR can amplify DNA up to at least 35kb (Barnes., 1994) and RT-PCR up<br>
to at least 20kb (Thiel et al., 1997), so at least 99.9% of all transcripts can be displayed.<br>
However there exist exceptional long RNA molecules such as titin (NCBI accession number<br>
X90568) that has about 80kb length and even though in theory displayable, for practical<br>
reasons one will probably not tune the display to such transcripts.<br>
Preferably, the reverse transcription is performed by a reverse transcriptase (RT)<br>
derived from the moloney murine leukemia virus, AMV, HIV-1, HIV-2, wherein the<br>
RNase H activity of these RTs is or is not present or is reduced. Reverse transcription can<br>
be performed by any enzyme that has a reverse transcriptase activity, e.g. Tth DNA polymerase<br>
that exhibits reverse transcriptase activity in the presence of Mn2+. In addition the<br>
RNA can be degraded after the reverse transcription step, for example by an RNA degrading<br>
enzyme such as RNase to facilitate long PCR.<br>
Preferably for reverse transcription as well as the amplification designed enzymes<br>
can be used, which are independent of a thermal cycling procedure, i.e. are capable of isothermal<br>
polymerisation.<br>
It is further preferred that primers used for reverse transcription have a wobble in<br>
one or more bases, preferably 1, 2, 3, 4 or 5 bases, on their 3' end preferable between position<br>
2 to 4 upstream of the last dT of the terminal dT stretch to further increase specificity.<br>
As used herein, the term "primer" refers to a mixture of oligonucleotides that have<br>
the same sequence.<br>
As used herein, the term "wobble primer" refers to a mixture of oligonucleotides<br>
that have the same sequence, except for the base or bases that are part of the "wobble".<br><br>
As used herein, the term "wobble" refers to a base within a primer pool that is present<br>
as a mixture of two or three or four different nucleotides.<br>
As an example a wobble primer can consist of oligonucleotides of the sequence<br>
ACACACN, where N is referred to as the wobble and N is A or C or T or G. This wobble<br>
primer would then consist of a mixture of nucleotides of the sequence ACACACA, ACA-<br>
CACC, ACACACT and ACACACG.<br>
When using N nucleotides as a mixture (N is a mixture of dA, dC, dG and dT) in<br>
wobble primers, nucleotides with universal bases can be used instead, such as deoxyi-<br>
nosine, 3-nitropyrrole 2'-deoxynucloside and 5-nitroindole 2'-deoxynucleoside. Universal<br>
bases will basepair with any nucleotide (dA, dC, dG, dT).<br>
Therefore, according to the invention the primers used for reverse transcription or<br>
amplification preferably have a wobble substitution, e.g. dT may be substituted by dG, dA,<br>
dC and vice versa, in one or two bases on their 3' end, preferably between position 2 and 4<br>
upstream of the last nucleotide, which is preferably dT for a 5' RT primer (for 5'--&gt;3'<br>
strand synthesis) and a 3' amplification primer and dA for a 5' amplification primer. In the<br>
PCR step of the method such "wobble" primers can be used to increase specificity for both<br>
the 5' and 3' end of the cDNA molecules to be amplified.<br>
Wobbles can also be used to anchor the primer that is complementary to the (e.g.<br>
poly A) tail of the original RNAs to the bases immediately upstream of the tail. In this case<br>
the selection into different pools can be achieved by primers selective to bases corresponding<br>
to bases after the cap of the original RNA. The same rational can be applied to the 3'<br>
end of the cDNA where wobbles can anchor a primer to the bases corresponding to bases<br>
downstream of the cap of the original RNA. Therefore the selection into different pools<br>
would be done through primers that select for bases that correspond to bases immediately<br>
upstream of the (e.g. poly A) tail of the original RNAs.<br>
Even further preferred is a method according to the invention, wherein the wobble<br>
base represents a substitution by universal bases, preferably deoxyinosine, 3-nitropyrrole<br>
2'-deoxynucloside or 5-nitroindole 2'-deoxynucleoside, which can basepair to any regular<br>
nucleobase such as dA, dT, dG or dC.<br>
In a preferred embodiment specificity of the cDNAs amplified can be enhanced by<br>
using a combination of a primer that is exonuclease resistant on its 3' end (e.g. by including<br>
a phosphorothioate modification or a locked nucleic acid) with a 3'--&gt;5' exonuclease<br><br>
function, such an activity being a property of e.g. Pfu DNA Polymerase. This will provide<br>
an "off switch" for amplification if a single mismatch, or multiple mismatches occur in the<br>
first 8 bases from the 3' end of the primer. (Yang et al., 2005). As the bases at the 3' prime<br>
end of the primers are the ones that discriminate the cDNA pools in methodologies presented<br>
in this invention, it is preferred that such an "off switch" is included to stop the extension<br>
of primers that mismatched to a cDNA, thus further enhancing specificity. Preferably<br>
in the inventive method a polymerase sensitive for a mismatch between nucleotides of<br>
the primer and the RNA or cDNA is used (either during reverse transcription or amplification),<br>
and the polymerase does not extend a mismatched exonuclease resistant primer,<br>
wherein it is preferred that the primers are resistant to exonucleases, especially preferred<br>
by a phosphorothioate modification or a locked nucleic acid.<br>
In a preferred embodiment the primer used for reverse transcription consists of an<br>
oligo dT stretch or tail complementary stretch without an anchor. Since the selection of<br>
specific RNAs occurs on the cDNA level via the amplification step described above, specificity<br>
(of the anchor) at the RT level is not a necessity. Of course mispriming should be<br>
avoided, especially where the (oligo T) primer aligns with the sequence upstream of the<br>
(poly A) tail stretch of the RNA. Mispriming can be avoided using stringent conditions<br>
and/or using an enzyme with proofreading properties. Preferably the annealing temperature<br>
during the RT process lies in the range of 30 to 50 °C, depending on the melting temperature<br>
of the primer used.<br>
Even though utilization of stringent conditions, such as the use of higher annealing<br>
temperatures, wobbled primers and oligoblockers, can decrease mispriming events during<br>
reverse transcription (RT), a preferred solution is to not separate into different pools during<br>
RT. As the reverse transcriptase reaction allows for more misprimed sequences to be extended<br>
than the amplification reaction (e.g. PCR reaction), it is most preferred that the<br>
separation into pools is done during the amplification reaction. This also decreases the RT<br>
reactions necessary to one per sample to be displayed.<br>
In a preferred embodiment the generation of full-length cDNA can be further advanced.<br>
Under non optimal conditions reverse transcriptase might stop polymerisation<br>
and/or fall off the RNA strand before reaching the 5' end of the RNA molecule. Generally<br>
this is attributed to secondary structure that has not melted and that the polymerase cannot<br>
traverse. Therefore generating also partial cDNA's. To lower the probability that such transcripts<br>
will interfere with subsequent (PCR) amplification one can exploit the intrinsic<br>
ability of reverse transcriptase to add a few (Mn2+ and Mg2+ dependent 1-6 nucleotides,<br>
Schmidt et al., 1999) dC once it reaches the 5' cap end of the mRNA molecule (Clark,<br><br>
1988). The cap (Furuichi et al., 1975) structure greatly increases the nontemplated nucleo-<br>
tide adding capability of RT compared to mRNAs that do not have the cap structure. This<br>
can be used to further enrich for full-length cDNAs. In a subsequent PCR reaction cDNAs<br>
can be selected that have these Cs present by including dG nucleotides in the 5' primer at<br>
the complementary position. In such a case the tailing can be done with any nucleotide except<br>
dC, preferably with dT or dA. The general structure of a 5' PCR primer that would<br>
select for cDNAs that have three Cs added to their 5' end, then will be dTxdGdGdGHNy or<br>
dAxdGdGdGHNy or dCxdGdGdGHNy; H can either be dA or dC or dT.<br>
In a preferred embodiment the method according to the invention, includes the<br>
generation of cDNA with a 3' sequence of oligo-dC, preferably with a length of 1 to 6 dC<br>
nucleotides, during reverse transcription, dependent on a 5' cap of the template RNA or<br>
mRNA.<br>
Preferably the ability of a reverse transcriptase to generate the 3' sequence of oligo-<br>
dC is increased by the addition of Mn2+ ions during the process of reverse transcription.<br>
The number of nucleotides added can be increased by including Mn ions in the RT reaction<br>
as described in Schmidt et al., 1999. Therefore in a preferred embodiment Mn ions are<br>
included in the RT reaction to increase quantitatively the nontemplated addition of dC on<br>
the 3' end of cDNA in a cap dependent manner. Subsequent tailing is carried out with any<br>
nucleotide but dC in order to allow the selection of the cap dependent dC. The selection of<br>
added nucleotides during tailing can be done by providing just the wanted nucleotide during<br>
this process. One primer that is used for (PCR) amplification can therefore have the<br>
general formula dPxdGy HNZ; Px representing nucleotides complementary to the tail; dG<br>
being deoxyguanylate and y an integer between 0 and 10, preferably between 1 and 5 more<br>
preferable between 3 and 5; H can either be dA or dC or dT; and N is a sequence of nucleotides,<br>
its members independently selected from either dA, dC, dG, dT; z is an integer<br>
between 0 and 10 preferably between 0 and 5.<br>
In a preferred embodiment the cap dependent 3' sequence of oligo-dC is used to<br>
isolate fully transcribed cDNA by using a second primer during the amplification which is<br>
specific for the oligo-dC sequence. This can be done by using a primer as described above.<br>
Preferably the tailing of the 3' end of the cDNA is performed with a polynucleotide<br>
sequence, which is characterized by the absence of dC.<br><br>
Even more preferred the DNA polymerase used for the amplification reaction is<br>
Taq DNA polymerase, Tfl DNA polymerase, aTaq DNA polymerase, Sequenase or Klen-<br>
taq.<br>
In special embodiments an enzyme with proofreading activity is used, preferably<br>
selected from PFU, Ultma, Vent, Deep Vent, PWO and Tli polymerases and E.coli. ex-<br>
onuclease III, is used during the amplification reaction. Such an enzyme can be used in addition<br>
to a polymerase, preferably in lower concentrations than the polymerase, or, if it is a<br>
polymerase itself, instead. Polymerases with proofreading activity prevent stops during<br>
DNA polymerisation and ensure long transcripts with significantly reduced errors. Preferably<br>
a combination of such an enzyme with proofreading capability is used with a polymerase<br>
above.<br>
In a more preferred embodiment a primer used for the amplification reaction,<br>
which is preferably a polymerase chain reaction, is characterized by the general formula<br>
dPxdGyHNz, wherein Px represents nucleotides complementary to the tail added to the 3'<br>
end of the cDNA, e.g. by the terminal transferase or by the template switching property of<br>
the reverse transcriptase or by any other means, dG being deoxyguanylate and y an integer,<br>
including 0, preferably between 2 and 5, H being either dT or dA or dC, N being a sequence<br>
of nucleotides with a length z wherein its member nucleotides are independently<br>
selected from dA, dC, dG and dT, and z being an integer between 0 and 10, preferably between<br>
0 and 5.<br>
The 5' end of primers used for RT and PCR can also be anchored to any sequence<br>
that will allow RT and PCR to be carried out. 5' and 3' PCR primers may have different<br>
melting temperatures and PCR conditions may be optimized by equalizing melting temperatures<br>
of primer pairs through the addition of an anchor to one or both of the primers.<br>
For certain downstream application it might also be of benefit to add a specific sequence<br>
such as a promoter that can start RNA Polymerase transcription (e.g. T7 or T3), or the inclusion<br>
of a restriction site.<br>
In a most preferred embodiment the amplification is performed by PCR and the<br>
PCR primers contain anchoring sequences, which allow proper placement at the end of the<br>
3' or 5' oligonucleotide end stretch of the cDNA.<br>
Preferably an amplification primer or a reverse transcription primer contains a<br>
promoter for RNA polymerases. This optional feature allows the transcription of the<br>
cDNA and its PCR products.<br><br>
Preferably the promoter enhances transcription by the T7, T3 or SP6 RNA poly-<br>
merase.<br>
Primers as well as PCR products can be labeled by any reporter group known in the<br>
art that enables their detection. Examples of reporter groups include fluorescent, chemilu-<br>
minescent or radioisotopes and others known in the art. Any reporter group can be used<br>
that can be detected by such techniques as fluorescent measurement, light emission measurement,<br>
scintillation counting and other means known in the art.<br>
Accordingly, the amplification primers and/or the amplification products are labeled<br>
by a reporter group, preferably fluorescence markers, chemiluminescence markers or<br>
radioisotopes in a preferred embodiment of the present invention. Preferably, these reporter<br>
groups are used to detect the DNA.<br>
Recently, it has been shown that during transcription at least some eukaryotes such<br>
as Arabidopsis (Jin et al., 2004) can add arbitrary nucleotides to RNA molecules on their 3'<br>
end immediately before the poly A tail, in most cases one or two. And even within a certain<br>
gene such nucleotide additions can be heterogenic. Therefore using a set of primers<br>
that discriminate for the first and/or second base upstream of the poly A during reverse<br>
transcription the intended pooling of a mRNA species into certain pools of cDNA will fail<br>
for certain transcripts. However a higher degree of pooling can be achieved using nucleotides<br>
upstream of these arbitrarily added nucleotides for the primers that define the fractions.<br>
A preferred primer for reverse transcription and PCR amplification that will be<br>
complementary to the poly A tail and select for bases above these additions be of following<br>
general structure dTxVNyMz, where dT depicts deoxythymidylate with x repetitions; V<br>
and N are wobble bases where V is either dA, dC or dG ; N is a sequence of dA, dC, dG,<br>
dT, independently selected for each member; y is an interger equal to or greater than 0,<br>
preferable between 1 and 5, and M is either dA, dC, dG, dT; z is an integer above 1, preferably<br>
between 1 and 10. In such a primer Mz will specify the cDNA pool to be amplified.<br>
As the number of nucleotides added to RNA molecules before polyadenylation might differ<br>
even for a specific transcript, a primer can be a mixture of oligonucleotides where the<br>
amount of y in N is different. An additional benefit is that in order for the Polymerase to be<br>
able to extend the chain the pressure for correct hybridization of bases after the wobble<br>
will strongly increase, therefore decreasing mispriming of the bases that define the fractions<br>
that will be amplified, hence specificity will be increased.<br><br>
In a preferred method according to the invention arbitrary nucleotides, preferably<br>
one to three nucleotides, are added to the 3' end of the RNA or mRNA posttranscription-<br>
ally but before the addition of a (e.g. poly A) tail, and the primers for reverse transcription<br>
and/or PCR contain a sequence with a wobble corresponding to the arbitrary sequence.<br>
A simplified version of the invention that will not specifically select for full length<br>
cDNA can be formulated by not using the 3' (dC) nontemplated adding function of the RT<br>
enzyme. There will still be a portion of full length cDNAs that have no dC added under<br>
normal RT condition. Tailing and PCR can be used as described to selectively amplify<br>
pools of cDNA. Each cDNA will still be present in exactly one pool. However the extent<br>
of PCR products present that will correspond to full length RNA molecules will be determined<br>
by the quality of the RT reaction alone. As one does not select in the PCR reaction<br>
that follows for full length cDNA copies, more than one cDNA molecule might give rise to<br>
PCR products that represent a single RNA molecule, therefore introducing redundancy.<br>
Still each RNA molecule will be represented at least once. Redundancy can be minimized<br>
by employing methods that favour full length reverse transcription, such as addition of tre-<br>
halose and/or betaine and/or increasing temperature as disclosed above. Even though redundancy<br>
might not be abolished in this embodiment, still each RNA molecule will be<br>
present at least once. This is an improvement compared to other DD techniques that can<br>
reach a representation for a certain mRNA molecule only statistically.<br>
Also there exist species of RNA molecules that do not have a poly A tail and a cap<br>
structure. However such RNA molecules will also have 5' and 3' prime end that can be<br>
used during subsequent amplification to enrich for defined fractions as is intended by this<br>
invention.<br>
Examples of RNA that may or may not have a cap and may have or may not have a<br>
poly A tail are non protein coding RNAs (ncRNA) such as ribosomal RNA (rRNA), transfer<br>
RNA (tRNA), micro RNA (miRNA), small interfering RNA (siRNA), piwi-interacting<br>
RNA (piRNA) and small nuclear RNA (snRNA).<br>
In RNA molecules that do not have a poly A tail, a tail such as a poly A tail, is<br>
added synthetically (e.g. enzymatically) prior to reverse transcription. Preferably a polynu-<br>
cleotide tail is added where the nucleotides are selected independently from A or C or G or<br>
U.<br>
There exist species of RNA that have a certain size range, such as miRNA that are<br>
about 19 to 23 bases long. Therefore size separation of pools of amplification products de-<br><br>
rived from miRNAs is limited. When displaying miRNAs with a size range of 19 to 23<br>
bases, a maximum of 5 molecules per pool can be resolved according to size. Therefore a<br>
quite large primer set (primer matrix) has to be used to resolve all miRNA molecules.<br>
In an alternative approach the nucleotides of one end of the miRNA can be used to<br>
artificially enlarge the amplification products for a specific number of nucleotides according<br>
to the terminal sequence of the miRNA. A general formula of such a primer is PxNy;<br>
with P representing nucleotides complementary to a tail; N represents nucleotides independently<br>
selected from either A, C, G, T; y is an integer between 0 and 10; x defines the<br>
number of nucleotides and is different for all Ny. If the 5' end of the miRNA is used for<br>
such a size enlargement, a primer mix containing certain or all possible 5' primers is used<br>
in combination with a 3' primer that is selective for nucleotides at the end of the miRNA<br>
molecule, to define a certain pool. One possible primer matrix according to the principles<br>
of this invention is shown in figure 6.<br>
Therefore in a preferred embodiment of the invention, amplification products - in<br>
particular full length amplification products - are generated that are differentially enlarged<br>
by certain numbers of nucleotides depending on the 5' or 3' sequences of the RNA or the<br>
full length cDNA.<br>
Preferably, amplification products are generated that are differentially enlarged by<br>
defined numbers of nucleotides at their 5' and/or 3' end as compared to the RNA, by using<br>
different primers which have different lengths and recognize different 5' or 3' endse-<br>
quences of the RNA or the full length cDNA. Also, pools of amplification products can be<br>
generated that are enlarged by defined numbers of nucleotides at their 5' and or 3' end as<br>
compared to the RNA, preferably by using as a 5' and/or as a 3' primer a mixture of primers,<br>
where each primer recognizes a different endsequence and has a different length. Preferably<br>
the primers have a defined sequence that is not complementary to the cDNA (during<br>
the amplification step) or RNA (during the reverse transcription step).<br>
In prokaryotes mRNA has no poly A tail. Methods have been devised (Amara et<br>
al., 1997) that can make full-length cDNA from such molecules, such as adding a synthetic<br>
poly A tail. It is also within the scope of this invention to apply said methods to such<br>
molecules.<br>
Preferably in a method according to the invention RNA or mRNA samples are<br>
used, which do not have a poly A tail or a tail sequence and, preferably a polynucleotide<br><br>
tail, especially preferred a poly A tail, is added synthetically (enzymatically) prior to reverse<br>
transcription.<br>
To decrease the necessary amount of RNA to perform the display a preamplifica-<br>
tion step can be introduced before the amplification reaction, e.g. the PCR. Whether or not<br>
separation into pools was done in the reverse transcription (RT) step, the preamplification<br>
step has to be designed in a way that will amplify all the cDNA that is intended to be separated<br>
into defined pools in the amplification (PCR) step. The preamplification step can be<br>
introduced before the tailing or after the tailing, before the main amplification with the<br>
specific primers.<br>
For example a promoter such as T7 RNA Polymerase can be added to the 5'- end of<br>
the primer that is used for priming the RT reaction. After the RT and before the amplification<br>
(PCR) RNA can be synthesized by an RNA polymerase specific to the promoter, for<br>
example T7 RNA polymerase. Such obtained amplified RNA has to be reverse transcribed<br>
again and depending whether or not a sequence is present downstream of the cDNA base<br>
that is complementary to the first base of the RNA and that can be used to prime the amplification<br>
reaction (PCR) on this end, such a sequence has to be added, for example by the<br>
template switching ability of the reverse transcriptase or by tailing with terminal deoxynu-<br>
cleotidyltransferase.<br>
Another example for a preamplification step is the use of a preamplification PCR.<br>
In case of a terminal deoxynucleotidyltransferase tailed cDNA one primer has to be specific<br>
to the so added tail. For example a poly T tailed cDNA will need a primer such as<br>
dAx and a primer complementary to the poly A tail of the original RNA for example dTx.<br>
Preferably one can preamplify only full length cDNA by including variable amounts of G,<br>
preferably between 1 and 5, to enable the primer to hybridize only to cDNA that was fully<br>
reverse transcribed and thus where the reverse transcriptase added Cs through its non tem-<br>
plated nucleotide adding capacity in a cap dependent manner. Additionally the primer that<br>
can hybridize to the sequence complementary to the original RNA poly A tail can be anchored<br>
to the 5'-end of the poly A tail by including one or more wobbles. This will help to<br>
increase the length of the amplification (PCR) products obtained, by not needlessly amplifying<br>
long stretches of poly A tail. Such a primer has the general formula of dTxVNy (x is<br>
the amount of dT, V either G, C or A, and N any nucleotide).<br>
The discrimination of the cDNA into different pools can also be accomplished by<br>
using more than one amplification (PCR) step. Each amplification (PCR) step would be<br>
more specific to the pool to be amplified. For instance in a first amplification (PCR) a<br><br>
primer such as dTx could be used for hybridizing to cDNA on the side that is complementary<br>
to the poly A tail side of the RNA. This primer would be not discriminating. In a next<br>
step, 3 reactions could be used with one being primed with dTxG, one with dTxA and one<br>
with dTxC. Thus creating 3 pools. In another PCR step the pools can be further divided.<br>
For example the dTxG pool can be divided in to 4 pools by priming with dTxGA, dTxGC,<br>
dTxGG and dTxGT. Thus creating a total of 12 pools. The same rational can be applied to<br>
the other end of the cDNA and the pooling of both sides can be combined if desired. Such<br>
a sequential pooling can on the one hand further decrease the amount of starting material<br>
(RNA) needed or help to achieve a pooling tuned to the complexity of the sample to be<br>
investigated or the depth the investigator wants to cover.<br>
It is within the scope of the invention that two or more samples of RNA that are<br>
combined by methods that yield normalization or subtraction between such RNA or their<br>
derived cDNA or PCR products can also be displayed according to disclosed methods. For<br>
example RNA can be (pre)treated by reducing RNA species in high concentration.<br>
Therefore the present invention also relates to a method wherein the RNA in the<br>
sample or the cDNA is preamplified, preferably by PCR, or wherein RNA samples are<br>
normalized or subtracted. These (pre)treatment steps are preferably introduced before the<br>
main (discriminating) amplification c).<br>
In a further embodiment of the method according to the invention the step of frac-<br>
tionation of the amplification product is performed by gel electrophoresis, pulsed-field<br>
electrophoresis, agarose gel electrophoresis, PAGE, capillary electrophoresis or pulsed-<br>
field capillary electrophoresis. Fractionation and comparison of samples generated during<br>
PCR can be done by any methods that allow distinguishing DNA molecules according to<br>
specific properties, such as sequence or size. Such methodologies include e.g. gel electrophoresis,<br>
such as polyacrylamide gel electrophoresis (PAGE). Especially suited is capillary<br>
gel electrophoresis because of its high resolving power and because many samples can easily<br>
be analyzed in parallel (Behr et al., 1999). Standard capillary electrophoresis machines<br>
today can process 384 samples in one run. And through the use of appropriate protocols<br>
one can achieve the necessary resolving power over a wide range of sizes. Especially long<br>
fragments can be resolved sufficiently with pulsed-field capillary electrophoresis (Heller et<br>
al., 2001; Morris et al., 2001). Also methods have been devised to collect fractions of the<br>
molecules separated in a capillary (Irie et al., 2000; Magnusdottir et al., 2001). Such fractions<br>
that represent transcripts of interest (e.g. differentially expressed genes) can then be<br>
subjected to further analysis, such as sequencing, to determine the identity of the molecule.<br><br>
In a preferred embodiment the detection of differences between two cDNA pools<br>
can be included into the PCR step of the method. Such methodologies, that can measure<br>
DNA levels and/or the change of such levels during PCR, are generally referred to as "real<br>
time PCR"(e.g. Wittwer et al. 1997a, Wittwer et al. 1997b). One procedure applicable<br>
measures fluorescence from SYBR Green I dye, that has been added to the PCR reaction.<br>
This dye will show a strong increase in fluorescence when intercalating with double<br>
stranded DNA. The more DNA present in the reaction the stronger the fluorescence.<br>
Therefore providing the means to calculate the amount of DNA present in the reaction.<br>
The measurement is usually done at the end of the polymerization step as the products will<br>
be double stranded. As the amount of DNA increases during subsequent cycles, so will the<br>
fluorescence. This data can be compared e.g by blotting on a curve and curves can be<br>
compared between different samples. If there is only one cDNA amplified the curve will<br>
have an exponential phase. By comparing the start of the exponential phase between samples<br>
one can calculate relative amounts of DNA. If two or more products will be present<br>
in a given sample, curves will add onto each other and will result in a change of steepness.<br>
Still such data can be compared between different samples, if the cDNA pool amplified is<br>
not too complex. It is preferred that the amount of cDNAs amplified in one PCR reaction<br>
is reduced to such a point that differences in amplification of each product can still be detected.<br>
Another way of analysing a sample, when using real time PCR, is the possibility of<br>
calculating a melting curve of the DNA present at the end of each cycle. Each PCR product<br>
 present in the reaction will melt at a specific temperature, that is defined by the sequence.<br>
As a product is melting into its single strands the dye is not intercalating with<br>
DNA anymore, whereby a drop of fluorescence will result. Products that will melt at different<br>
temperatures will be represented as an increase and then a decrease of the steepness<br>
at different areas of the melting curve. Comparing curves between cycles and different<br>
samples of the same cDNA pool to be amplified, one can identify samples with a different<br>
profile. A melting curve analysis can also be done only at the end of the PCR, however<br>
differences in products that have plateaued might not be detected.<br>
The use of such analysis methods preferably in an automated procedure can rapidly<br>
screen for samples that show differences during or at the end of PCR and eliminate the<br>
need to analyse all of them through a fractionation procedures such as gel electrophoresis.<br>
Therefore in a preferred embodiment the detection of differences between two<br>
cDNA pools, preferably corresponding pools between two or more samples, is done during<br>
or at the end of the amplification step, preferably being a PCR step, of the method, preferably<br>
by real time PCR.<br><br>
Clearly the methods described in this invention can be integrated into a partially or<br>
fully automated process. For example the size fractionation, comparison of different samples,<br>
collection of fractions, and sequencing can be all done on a chip. The technologies<br>
have already been demonstrated in principle (Xie et al., 2001).<br>
Such an integration of the method onto a chip can also be done for the PCR step of<br>
the method. It is preferred that the primers, or primer pairs are spotted and/or bound onto a<br>
solid support to enable solid phase DNA amplification (e.g. by PCR). Through microarray-<br>
ing or nanoarraying or single molecule arraying the whole set of primers onto a solid support,<br>
a single reaction vessel can be created that contains the whole primer matrix or parts<br>
thereof. Therefore instead of using an own reaction vessel for each 5'-3' primer pair, a single<br>
reaction vessel contains all 5'-3' primer pairs. It is preferred that each primer pair is<br>
located in a defined area.<br>
A primer matrix can also be created where each 5'-3' primer pair is bound onto its<br>
own solid support (e.g. a bead) and the solid support itself is then arranged in such a manner<br>
that each primer combination or primer pair can be addressed independently. For instance<br>
a matrix of beads can be arranged randomly in micro wells and the location of each<br>
bead decoded by methods known to the art (e.g. Gunderson et al., 2004). The method as<br>
described by Gunderson can thus be adapted and incorporated by the present invention.<br>
This method requires only a few labels and several sequential hybridizations to identify<br>
thousands of different DNA sequences with great accuracy. Regarding the primer matrix,<br>
the primers on the areas may comprise the information for decoding themselves (e.g. as<br>
sequence which can be decoded by other labeled oligo nucleotide by several steps of hybridisation,<br>
detection and washing) or together with the primers of the invention decoding<br>
oligonucleotides can be bound onto the carrier.<br>
Therefore in a preferred embodiment a 5'primer or a 3'primer or a 5'-3' primer pair<br>
is spotted and/or bound onto a solid support, preferably the spot or the solid support is located<br>
in a defined 2-dimensional or 3-dimensional area and it is preferred that such an area<br>
contains information to decode the identity of the primer or primer pair located in that<br>
area.<br>
In a preferred embodiment the primer matrix is generated by direct on chip synthesis<br>
of the 5' primers or 3' primers or 5'-3'primer pairs.<br>
In a preferred method according to the invention the steps of reverse transcription<br>
and amplification are performed in one step.<br><br>
It is obvious to the one skilled in the art that other possibilities exist to enrich for<br>
full-length cDNA. Such methods can easily be adapted to amplify subsequently pools of<br>
full length cDNA that are defined through nucleotides that are complementary to the 5' end<br>
and/or nucleotides on the 3' end upstream of the poly A tail.<br>
For example the CapSelect method (Schmidt et al., 1999) can be used to substitute<br>
the steps of reverse transcription and tailing the 3' end of the cDNA. This method is also<br>
drawn on the nontemplated addition of C nucleotides of reverse transcriptase, and addition<br>
of a poly N tail, preferably a poly A tail. However the poly A tail consists of ribonucleo-<br>
tides instead of deoxyribonucleotids as it is claimed that terminal transferase will preferably<br>
add only 3 to 4 ribonucleotides, compared to an unspecified amount of deoxyribonucleotids.<br>
This is important for the method as in a next step an adaptor will be specifically<br>
linked to the 3' end of the cDNA selecting for 3'-AAACCC-5'. Using a primer that is complementary<br>
to this adaptor and a primer complementary to the poly A tail it is intended to<br>
amplify full length cDNA. Instead of using universal primers to amplify all cDNAs one<br>
can use sets of primers that select for the 5' and/or 3' nucleotides and will amplify pools of<br>
cDNAs. The primer structure for hybridizing to the poly A tail and selecting for x amount<br>
of bases before the poly A tail will be the same as described. For amplifying the 5' end of<br>
mRNAs one can use the adaptor sequence followed by 3 to 4 Ts and 3 to 4 Gs and an x<br>
amount of bases that correspond to the 5' sequence of the RNA transcripts. Increasing x<br>
will decrease the number of transcripts amplified through such a primer.<br>
Preferably the polynucleotide tail added by the terminal transferase consists of ribonucleotides,<br>
preferably 2 to 6 ribonucleotides, preferably adenosine or thymidine, after a<br>
3' sequence of oligo-dC, preferably with a length of 1 to 6 dC nucleotides, which is generated<br>
during reverse transcription, dependent on a 5' cap of a template RNA or mRNA, and<br>
the second amplification primer (5' primer) comprises complementary nucleotides to the<br>
ribonucleotides and oligo-dCs and 1 to 10 nucleotides upstream of the olio-dC sequence.<br>
The method for amplification of certain cDNAs, derived from mRNA, can also be<br>
used to determine the expression level of individual mRNAs. Preferably for measuring the<br>
expression level of messenger RNA a method according to the invention is employed, including<br>
the separation and/or identification of amplification results. Like in common DD<br>
results of certain cell lines can be compared for better visualisation of different expressions.<br><br>
Furthermore the present invention provides a kit for a method according to the invention<br>
comprising a DNA polymerase, preferably Taq polymerase, a reverse transcriptase<br>
or mixtures of different polymerases, cofactors or salts of metal ions, preferably Mg2+ and<br>
Mn2+, requested by a polymerase, primers and optionally a terminal transferase and additional<br>
buffer substances. The primers are preferably selective for the 5' and 3' end of a<br>
RNA or cDNA as described above. Even more preferred the primers of the primerset are<br>
selective for the first, second, third, fourth and/or fifth terminal nucleotide of the RNA or<br>
cDNA (of either end, independently from each other). Preferably the kit comprises 1, 2, 3,<br>
4, 5, 10, more than 10, more than 20 or more than 30 primer pairs. With this kit a display<br>
method according to the invention can easily and quickly be performed. Preferably instructions<br>
for the execution of the method are also included. Preferably the full primerset<br>
(primer matrix) or parts thereof - preferably one of each primer pair - are bound to a solid<br>
support such as a chip, preferably a glass chip.<br>
In another aspect of the present invention there is provided a solid surface, preferably<br>
a chip, in particular preferred a glass chip, comprising in defined areas, preferably in<br>
spots, a pair of primers, wherein different primers are selective for at least two, preferably<br>
at least three, even more preferred at least four, most preferred at least 10, different 3' or 5'<br>
end sequences of target polynucleotides, preferably selected from RNA or cDNA, and each<br>
area comprises primers for at least two different end sequences (e.g. a pair of a 5' and a 3'<br>
primer), preferably one primer is selective for a (e.g. poly A) tail or a complement thereof<br>
and even more preferred selective for the next 1 to 10 nucleotides, preferably the next 1, 2,<br>
3 or 4 nucleotides (of either the 5' and/or the 3' primer. The primers can be specific as described<br>
above. These primers may also, e.g. in these combinations, be used in the kit or the<br>
method. A primer pair in one area selective for mRNA is specific for the poly A tail and<br>
the 5' (cap) end, as well as the next 1 nucleotide (different in each area). Preferably the<br>
solid surface comprises at least two, at least 4, at least 10, at least 20 or at least 40 different<br>
primer pairs.<br>
Provided is further a solid surface, preferably a chip, in particular preferred a glass<br>
chip, comprising in defined areas, preferably in spots, a 5' primer and a 3' primer or a pair<br>
of primers, wherein the primers are as defined above being specific for both ends of a<br>
cDNA or a RNA, especially comprising formula dPxdGy HNZ; in particular preferred the<br>
solid surface comprises at least two, more preferred at least 9, especially preferred at least<br>
20, most preferred at least 60, different areas with different primers or primer pairs.<br>
Also provided is a solid surface, preferably a chip, in particular preferred a glass<br>
chip, comprising in defined areas, preferably in spots, a 5' primer or a 3' primer compris-<br><br>
ing the general formula as defined above (dPxdGy Hnz) or a complement thereof, preferably<br>
selected from a primer comprising one of the following sequences: AGA GAT TTT TTT<br>
TTT TTT TT GA, AGA GAT TTT TTT TTT TTT T GG, AGA GAT TTT TTT TTT TTT<br>
T GC, AGA GAT TTT TTT TTT TTT TT GT, AGA GAT TTT TTT TTT TTT TT CA,<br>
AGA GAT TTT TTT TTT TTT TT AG, AGA GAT TTT TTT TTT TTT TTT AA, AAA<br>
AAA AAA AAA AAA GGG ATA, AAA AAA AAA AAA AAA GGG ACA, AAA AAA<br>
AAA AAA AAA GGG AGA, AAA AAA AAA AAA AAA GGG AAA, AAA AAA AAA<br>
AAA AAA GGG GTA, AAA AAA AAA AAA AAA GGG CTA, AAA AAA AAA AAA<br>
AAA GGG TTA, in particular preferred the solid surface comprises at least two, more preferred<br>
at least 9, especially preferred at least 20, most preferred at least 60, different areas<br>
with different primers or primer pairs.<br>
In a further aspect a method for amplifying a pool of polynucleotide molecules in a<br>
sample is provided, comprising the steps of:<br>
a)	obtaining (or providing) a sample of RNA and reverse transcription of entire<br>
RNA molecules thus creating full length cDNA or obtaining (or providing) a<br>
sample of full length cDNA,<br>
b)	tailing the 3' end of the transcribed cDNA with a polynucleotide tail after the 3'<br>
end,<br>
c)	amplification of the cDNA using a pair of primers, wherein a first (3') primer is<br>
specific for the 5' end of the cDNA and a second (5') primer is specific for the upstream<br>
portion of the polynucleotide tail and the next 1 to 100 nucleotides, preferably<br>
the next 1 to 50 nucleotides, especially preferred the next 1 to 20 nucleotides,<br>
in particular preferred the next 1 to 10 nucleotides (i.e. nucleotide 1, 2, 3, 4,<br><br>
5,	6, 7, 8, 9 or 10), upstream of the 3' polynucleotide tail of the cDNA, preferably<br>
also comprising unspecific wobble nucleotides, especially preferred in the upstream<br>
portion within the 1 to 100 nucleotide range - wherein at least 1, 2 ,3, 4, 5,<br>
6,	7, 8, 9 or 10 nucleotide (s) is/are specific for a nucleotide of the cDNA; the<br>
method further preferably defined as given above.<br>
The method preferably includes the use of a primer, which is specific for a 3' tail of<br>
the RNA (optionally an added 3' tail) or mRNA (preferably the endogenous 3' poly A tail)<br>
and the next 1 to 100 nucleotides, preferably the next 1 to 50 nucleotides, especially preferred<br>
the next 1 to 20 nucleotides, in particular preferred the next 1 to 10 nucleotides (i.e.<br>
nucleotide 1, 2, 3, 4, 5, 6, 7, 8, 9 or 10), upstream of the 3' tail is used for the reverse tran-<br><br>
scription and/or a first amplification primer is specific for a 5' tail of the cDNA, which is<br>
complementary to the 3' tail of the RNA or mRNA, and the next 1 to 100 nucleotides,<br>
preferably the next 1 to 50 nucleotides, especially preferred the next 1 to 20 nucleotides, in<br>
particular preferred the next 1 to 10 nucleotides (i.e. nucleotide 1, 2, 3, 4, 5, 6, 7, 8, 9 or<br>
10), downstream of the 5' tail. Of course, also both or independently from one another, in<br>
the 5' and the 3' primers, the nucleotide stretch (1 to 100 nucleotides) next to the tail (or<br>
tail complement) may comprise unspecific nucleotides (such as wobble nucleotides) in addition<br>
to at least 1, 2, 3, 4, 5, 6, 7, 8, 9, 10 or more specific nucleotides selecting for different<br>
pools (with or without pool separation) of the amplification product. Such primers can<br>
also of course be used for the reverse transcription as well. Products of such wobble primers<br>
(where any nucleotide may be present at a corresponding position to the wobble position)<br>
still fulfil the requirement of full-length cDNA or full-length amplification product<br>
although the sequence may not be preserved at positions that correspond to the unspecific<br>
nucleotide positions. Using longer unspecific nucleotide primer stretches primer-loops<br>
may occur thus resulting in a product with an altered length which is however still considered<br>
to be equivalents to full-length cDNA or amplification products for the present invention.<br>
The present invention is further illustrated by the following example and figure<br>
without being limited thereto.<br>
Fig. 1 shows a graphical representation of a procedure using the principles of the<br>
present invention to amplify full length cDNA into defined subpopulations. The oligo-T<br>
and oligo-G stretches indicated in the primers are terminal anchors of the primers and the<br>
N, NN and NNy indicate nucleotides selected from A, T, G, C for defining the subpopulations.<br>
Fig. 2: Gel elecrophoresis product of a display of a pool of artificial mRNA molecules<br>
according to example 2: In lane 1-7 primer nr. 8 was used together with primers nr.<br>
1-7. In lanes 8-13 primer nr. 1 was used together with primers 9-14.<br>
Fig. 3: Gel elecrophoresis product of an organised full length expression display of<br>
mouse liver RNA according to example 3.<br>
Fig. 4: Gel elecrophoresis product of an organised full length expression display of<br>
mouse liver RNA according to example 4 with PCR preamplification.<br><br>
Fig. 5: NCBI nucleotide database screening for non redundant full length mouse<br>
cDNA clones. 99% of the 16854 clones present in the database are between 200 and 6000<br>
bases in length and 99.9% below 7500 bases and no clone was above 9500 bases.<br>
Fig. 6 shows how RNA without a cap and a tail, such as miRNA, is divided into<br>
subpopulations. a) Steps of the procedure. Primer nomenclature as in fig.l b) One possible<br>
Primer matrix that increases the resolving power by increasing the size range of the amplification<br>
products. The B in the 5' primers indicate a mixed base (wobble) that is selected<br>
from T, G, C.<br>
Examples<br>
Expample 1: Principle of procedure<br>
An overview of an example of a polynucleotide amplification procedure according<br>
to the present invention is given in fig. 1. Anchored primers for reverse transcription can<br>
follow the formula dTxV or dTxVNy (with x defining a repetition of deoxythymidylate<br>
("dT") that has in combination with the V or VNy part a high enough specificity and melting<br>
temperature to successfully prime the RT reaction; V is either deoxyguanylate ("dG"),<br>
deoxyadenylate ("dA") or deoxycytidylate ("dC"); N is a sequence of y nucleotides selected<br>
independently from dA, dC, dG or dT; if more than, one base is used for anchored<br>
primers y defines their number; increasing y will increase the subpopulations of reverse<br>
transcripts. The tailing of the 3' end of such synthesized cDNA with a poly N tail can be<br>
done using terminal transferase - N stands for a sequence of uniformly nucleotides selected<br>
from dT, dA, dC or dG, independently form the "N" indicated in the reverse transcription<br>
primers. The amplification of the cDNA pools, optionally preselected by the VNy<br>
stretches defined by polymerase chain reaction (PCR) is performed using primers that hybridize<br>
to the 5' and 3' end of the cDNA by selecting to terminal sequences like the nucleotides<br>
appended by the terminal transferase (3' of the cDNA; corresponding to the 5'<br>
end of the original mRNA) and the poly T stretch (5' of the cDNA; complementary to the<br>
3' poly A tail of the original mRNA) and selecting for an oligo dNy sequence, wherein N is<br>
a stretch of several nucleotides, again independently selected from dT, dA, dC or dG and<br>
define different mRNA populations through y bases immediately adjacent to the tails.<br>
Example 2: Amplification of all the RNA molecules present in a sample.<br>
As all the mRNA molecules present in a sample are not known, even in well characterized<br>
tissues such as liver from human or mouse, and their number maybe in the thou-<br><br>
sands, an approach was chosen to use a pool of 14 artificially synthesized mRNA molecules<br>
to show that all RNA molecules present in that sample are detected. For this purpose<br>
stretches with different lengths of genomic DNA from the mouse GAPDH gene (GenBank<br>
Accession: NC000072) were amplified by standard PCR. 5' primers were used that had<br>
an anchored T7 polymerase promoter. Following the promoter a combination of 4 bases<br>
were included that would define each mRNA on its 5' end. Primers chosen for the 3' end<br>
included, after the GAPDH specific sequence, a combination of 4 bases, that would define<br>
these molecules upstream of the poly A tail. Following these 4 bases 21 dT or in one of the<br>
RNAs 22 dT were included for the poly A tail. The PCR products obtained were gel purified<br>
and subjected to standard T7 RNA transcription, that included 7-methyl-guanosine<br>
(the cap). Such T7 reaction products were again gel purified. A mixture of such obtained<br>
synthetic mRNA molecules was used as starting material for the reverse transcription<br>
(RT). To show that the methods disclosed are specific for the 5' as well as 3' end of the<br>
mRNA molecules, 7 of the 14 molecules had a fixed sequence on the 5' end and varied in<br>
the 3' end and 7 molecules had a fixed sequence on the 3' end and varied at the 5' end. For<br>
sequence of the resulting artificial mRNA molecules see table 1.<br>
Reverse transcription was carried out in a 25 ul reaction consisting of 50mM Tris-<br>
HC1 (ph 8.3), 75 mM KC1, 3mM MgCl2, lOmM DTT, 0.6M trehalose, 2M betaine, 0.5mM<br>
dATP, 0.5mM dTTP, 0.5mM dCTP, 0.5mM dGTP, 40nM primer (oligo dT(2i)). The artificial<br>
mRNA mix was included at a level of about 4 ng/ul reaction volume. The reaction<br>
(without the RT) was heated to 70 °C for 20 sec, 32°C for 2 min. Then the Molony Murine<br>
Leukemia Virus Reverse Trancriptase lacking RNase H activity (M-MLV RT (-H)) was<br>
added a concentration of 4-8 units/ul reaction volume and the reaction continued for 2 min<br>
at 32°C. At this point 0.5 U of Pfu polymerase was added. Four cycles at 50°C for 10min<br>
and 60°C for 10 sec were used to fully reverse transcribe the RNA, after that the reaction<br>
was cooled to 4°C.<br>
Table 1: Sequences of artificial RNAs to be displayed<br><br><br><br>
The cDNA was phenol/chloroform extracted, ethanol precipitated and dissolved in<br>
10 l of H2O. Poly T tailing was carried in a 20 l reaction containing 0.2mM dTTP,<br>
100M cacodylate buffer (pH 6.8), 1mM CoCl2, 0.1mM DTT and 15 units of terminal<br>
deoxynucleotidyl transferase for 15 min at 37 °C. The reaction was Phenol/Chloroform<br>
extracted, ethanol precipitated and dissolved in 100 l H2O.<br>
PCR was carried in a 10 ul reaction containing 2.5l tailed cDNA, 50mM Tris (pH<br>
8.8), 14mM (NH4)2SO4, 4mM MgCl2, 0.2mM of each dNTP, 0.2 M of each primer (table<br>
2, fig. 2), 0.5 units Taq polymerase and 0.03 units Pfu polymerase. Samples were denatured<br>
at 93°C for 30 sec, and cycled 21 times at 93°C for 10 sec, 55°C for 30 sec, 72°C for<br>
10 min. The initial annealing temperature of 55°C was decreased each cycle by 0.5°C min.<br>
Thereafter followed 19 cycles at 93°C for 10 sec, 49°C for 30 sec, 72°C for 10 min with a<br>
final extension step at 72°C for 2 min.<br><br><br>
2.5 l of the reaction product was added to 2.5 l 100% formamide loading buffer<br>
and denatured at 96 °C for 2 min and cooled on ice, loaded on a 3.5 % acrylamide, 7M<br>
urea gel and run at 180V for 2h and subsequently silver stained.<br>
The results can be seen in figure 2. In lane 1-7 primer nr. 8 was used together with<br>
primers nr. 1-7. In lanes 8-13 primer nr. 1 was used together with primers 9-14. In each<br>
lane the PCR amplified the specific product, as the predicted length of the PCR products<br>
shown in table 1 perfectly matches the lengths of the products on the gel.<br>
The primer combination in lane 1 amplifies the 2 specific products corresponding<br>
to the RNA to be displayed. The reaction carried out in lane 1 is the most demanding reaction<br>
of the set as two products have to be amplified, the ones representing RNA 1 and 8. In<br>
addition as can be seen in table 1 the RNA molecules were designed in a way to enhance<br>
miss priming of the PCR reaction represented in lane 1. All RNA molecules (except RNA<br>
Nr.6)have on position 3 and 4 before the T tail the same bases as on position 1 and 2 of<br>
RNA number 1 and 8. As primer nr. 1 uses its last two bases to select the cDNA to be amplified,<br>
it potentially can miss prime with any of the other cDNA present in the sample.<br>
And as cDNA moleclules 1-8 have the same bases on its other end it is only the two last<br><br>
bases of primer 1 that selects between these 8 molecules in reaction 1. However only the<br>
two RNAs to be amplified are displayed, showing the great specificity that can be reached<br>
in such a display.<br>
Lane 2 shows the limits of this reaction. In addition to the band to be amplified a<br>
faint band that corresponds to RNA nr. 4 was amplified. However the band to be amplified<br>
is several times more intense than the band that resulted through the miss priming<br>
event. Additionally even though a touch down protocol was used for the thermal cycling,<br>
the same profile was used for all the different primer combinations. To show that in principle<br>
the reaction can be optimized for a specific primer set another reaction was carried<br>
that included 1.302M betaine and 1.3% DMSO. The product is shown in lane 14 and only<br>
the correct band is displayed.<br>
Example 3: Amplification of total RNA from mouse liver.<br>
Total RNA (13ug) from mouse liver was reverse transcribed, tailed and PCR amplified<br>
under the same conditions as for the whole artificial RNA set in example 2.<br>
The same conditions for RT, tailing and PCR were used to show that it is possible<br>
to display RNA molecules from a complex mixture under the same stringent conditions as<br>
it is possible with artificial RNA. The PCR products were run on a 0.7% agarose gel to<br>
resolve longer molecules. As can be seen in figure 3 each primer combination amplifies a<br>
unique profile.<br>
The length distribution of the RNA molecules displayed in figure 3, matches very<br>
well the length distribution of cDNA in the NCBI nucleotide database screening for non<br>
redundant full length mouse cDNA clones (fig. 5).<br>
As on averaged about 10-15 bands can be seen on each lane the full set of 768<br>
primer combinations (3x44) displays about 7680-11520 PCR products representing full<br>
length RNA molecules expressed in liver. Being able to resolve such profiles on a simple<br>
0.7% agarose gel shows the power of the method and the ease, with what it can be performed<br>
in the laboratory.<br>
Example 4: Utilization of PCR Preamplification step.<br>
Preaimplification was carried out in a 10l reaction containing 2.5 ul of a purified<br>
tailed cDNA generated as in example 3, 50mM Tris (pH 8.8), 14mM (NH4)2 SO4, 4mM<br><br>
MgCl2, 0.2mM of each dNTP, 0.2M of Primer dT(21), 0.2M of Primer dA(21), 0.5 units<br>
Taq polymerase and 0.03 units Pfu polymerase. Samples were denatured at 93°C 30 sec,<br>
and cycled 11 times at 93°C 10 sec, 42°C 30 sec, 72°C 11 min with a final extension step at<br>
72°C for 2 min. This reaction was diluted to 1ml by adding 990l H2O. 2.5 l of this dilute<br>
was used in the following PCR using the same PCR conditions as in example 3 with the<br>
difference that the final 2 cycles where omitted. As can be seen in figure 4 a distinctive<br>
banding can be seen for each primer combination. The preamplification step as used in the<br>
example shows that 500ng of total RNA are enough to provide material for even a large<br>
display with 768 reactions per RNA sample, as might be needed to resolve highly complex<br>
RNA mixtures from tissues like brain. The amount of RNA used can easily be reduced by<br>
increasing the cycles during preamplification. Therefore the sensitivity of the display<br>
should meet even the most demanding applications, where only limited amounts of RNA<br>
are available. Furthermore the amount of bands per primer pair has increased compared to<br>
example 3. As the high annealing temperatures used in the discriminating PCR step, in<br>
both example 3 and 4, exert an extremely high stringency, some low level transcripts<br>
might be just below the detection limit when no preamplification step is used.<br>
Expample 5: miRNA Display<br>
An overview of an example of a polynucleotide amplification procedure according<br>
to the present invention, where the RNA has no cap and no poly A tail, in this case mi-<br>
croRNA (miRNA), is given in fig. 6.<br>
To be able to reverse transcribe full length cDNA from miRNA that has no Poly A<br>
tail, a synthetic tail is added by using poly A polymerase. In a second step the Poly A tailed<br>
RNA is reverse transcribed into cDNA using an oligo dT primer. After polynucleotide tailing<br>
of the cDNA by terminal transferase, the cDNA samples have sufficient long nucleotide<br>
terminals that allow for primers to anneal and amplify the cDNA sequences into different<br>
pools according to the 5' and 3' sequences of the original miRNA molecules (fig.<br>
6a).<br>
As microRNA (miRNA) molecules have a size of about 19-23 bases, a sufficiently<br>
large primer matrix (all possible combinations of 5' and 3' primers) has to be used to separate<br>
all miRNAs into different pools where differences in expression between samples can<br>
be measured.<br>
Alternatively one end of the miRNA sequence can be used to artificially enlarge the<br>
amplification products for a specific number of nucleotides according to the terminal se-<br><br>
quence of the miRNA. Figure 6b shows an example of such a matrix. In this case the 5'<br>
primer is used to split miRNA amplification products into different lengths according to<br>
the second and third nucleotide of the miRNA. In cases where the comparison of different<br>
pools is done by an additional size fractionation (such as capillary gel electrophoresis) all<br>
16 5' primers of fig. 6b can be used as a primer mix. This 5' primer mix together with one<br>
3' primer defines a pool. Therefore 192 pools have a similar resolution when compared to<br>
192 x 16 = 3072 pools, where each 5' primer would be used together with each 3' primer.<br>
References<br>
Amara et al., 1997, Nucleic Acids Res. 25, 3465-3470.<br>
Barnes, 1994, PNAS 91, 2216-2220.<br>
Behr et al., 1999, Electrophoresis 20, 1492-1507.<br>
Carninci et al., 1998, Proc. Natl. Acad. Sci. U. S. A 95, 520-524.<br>
Clark, 1988, Nucleic Acids Res. 16, 9677-9686.<br>
Di Giusto et al., 2004 Nucleic Acids Res. 32, e32 1-8.<br>
Frohman et al., 1988, Proc. Natl. Acad. Sci U. S. A 85, 8998-9002.<br>
Furuichi et al., 1975, Nature 253, 374-375.<br>
Gunderson et al., 2004 Genome Res. 14, 870-877.<br>
Hawkins et al., 2003, Biotechniques 34, 768-773.<br>
Heller et al., 2001, Methods Mol. Biol. 162, 293-305.<br>
Irie et al., 2000, Electrophoresis 21, 367-374.<br>
Jin et al., 2004, RNA. 10, 1695-1697.<br>
Kovarova et al., 2000, Nucleic Acids Res. 28, e70i-e70iii.<br>
Liang et al., 1992, Science 257, 967-971.<br>
Magnusdottir et al., 2001, Methods Mol. Biol. 162, 323-331.<br>
Matz et al., 1997, Nucleic Acids Res. 25, 2541-2542.<br>
Mizuno et al., 1999, Nucleic Acids Res. 27, 1345-1349.<br>
Morris et al., 2001, Methods Mol. Biol. 162, 307-321.<br>
Prashar et al., 1996, Proc. Natl. Acad. Sci. U. S. A 93, 659-663.<br>
Prashar et al., 1999, Methods Enzymol. 303, 258-272.<br>
Rebagliati et al., 1985, Cell 42, 769-777.<br>
Schmidt et al., 1999, Nucleic Acids Res. 27, e31.<br>
Spiess et al., 2002, Anal.Biochem. 301, 168-174.<br>
Skerra, 1992, Nucleic Acids Res. 20, 3551-54.<br>
Thiel et al., 1997, Anal.Biochem. 252, 62-70.<br>
Welsh et al., 1992, Nucleic Acids Res. 20, 4965-4970.<br>
Wittwer et al., 1997a, Biotechniques 22, 130-138.<br><br>
Wittwer et al., 1997b, Biotechniques 22, 176-181.<br>
Xie et al., 2001, Methods Mol. Biol. 162, 67-83.<br>
Yang et al., 2005, Biochem. Biophys. Res. Commun. 328, 265-272.<br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br>
We Claim :<br>
1.	Method for amplifying a pool of polynucleotide molecules in a sample, characterized<br>
by the steps of:<br>
a)	obtaining a sample of RNA and reverse transcription of entire RNA molecules<br>
thus creating full length cDNA or obtaining a sample of full length cDNA,<br>
b)	tailing the 3' end of the transcribed cDNA with a polynucleotide tail after the 3'<br>
end,<br>
c)	amplification of the cDNA using a pair of primers, wherein a first (3') primer is<br>
specific for the 5' end of the cDNA and a second (5') primer is specific for the upstream<br>
portion of the polynucleotide tail and the next 1 to 10 nucleotides upstream<br>
of the 3' polynucleotide tail of the cDNA.<br><br>
2.	Method according to claim 1 characterized in that a primer, which is specific for a<br>
3' poly A tail of the RNA or mRNA and the next 1 to 10 nucleotides upstream of the poly<br>
A tail is used for the reverse transcription and/or a first amplification primer is specific for<br>
a 5' poly-T stretch of the cDNA, which is complementary to the 3' poly A tail of the RNA<br>
or mRNA, and the next 1 to 10 nucleotides downstream of the 5' poly-T stretch.<br>
3.	Method according to any one of claims 1 or 2 characterized in that the tailing of the<br>
3' end is performed using terminal transferase.<br>
4.	Method according to any one of claims 1 to 3 characterized in that the amplification<br>
is performed by PCR.<br>
5.	Method according to any one of claims 1 to 4 comprising an additional step of<br>
separation of the amplification products according to their length, preferably by gel elec-<br>
trophoresis, preferably polyacrylamide gel electrophoresis, capillary electrophoresis,<br>
pulsed-field electrophoresis, agarose gel electrophoresis, PAGE or pulsed-field capillary<br>
electrophoresis.<br>
6.	Method according to any one of claims 1 to 5 comprising an additional step of determining<br>
the identity or sequence of the amplification products, preferably by an automated<br>
process on a chip.<br>
7.	Method according to any one of claims 1 to 6, wherein the sample contains total RNA<br>
or mRNA, preferably purified RNA or mRNA, from a specimen.<br><br>
8.	Method according to any one of claims 1 to 7, wherein high stringency conditions are<br>
used for the processes of reverse transcription and/or the amplification.<br>
9.	Method according to any one of claims 1 to 8, wherein the selectivity of the reverse<br>
transcription and/or amplification, preferably a polymerase chain reaction, is increased by<br>
the utilization of trehalose, betaine, tetramethylammonium chloride, tetramethylammo-<br>
nium oxalate, formamide and oligo-blockers, or by the utilization of dimethylsulfoxide<br>
during the polymerase chain reaction, to reduce the occurrence of mispriming.<br>
10.	Method according to any one of claims 1 to 9, wherein the reverse transcription is performed<br>
by a reverse transcriptase (RT) derived from the moloney murine leukemia virus,<br>
AMV, HIV-1, HIV-2, wherein the RNase H activity of these RTs is present, reduced or not<br>
present.<br>
11.	Method according to any one of claims 1 to 10, wherein primers used for reverse transcription<br>
or amplification have a wobble substitution, preferably dT is substituted by dG,<br>
dA or dC or vice versa, in one or two bases on its 3' end, preferably between position 2<br>
and 4 upstream of the last nucleotide, wherein the wobble base preferably represents a substitution<br>
by universal bases, preferably deoxyinosine, 3-nitropyrrole 2'-deoxynucloside or<br>
5-nitroindole 2'-deoxynucleoside, which can basepair to any regular nucleobase such as<br>
dA, dT, dG or dC.<br>
12.	Method according to any on of the claims 1 to 11, using a polymerase sensitive for a<br>
mismatch between nucleotides of the primer and the RNA or cDNA, and the polymerase<br>
does not extend a mismatched exonuclease resistant primer, wherein it is preferred that the<br>
primers are resistant to exonucleases, especially preferred by a phosphorothioate modification<br>
or a locked nucleic acid.<br>
13.	Method according to any one of claims 1 to 12, whereby cDNA with a 3' sequence of<br>
oligo-dC, preferably with a length of 1 to 6 dC nucleotides, is generated during reverse<br>
transcription, dependent on a 5' cap of the template RNA or mRNA, and preferably<br>
wherein the cap dependent 3' sequence of oligo-dC is used to isolate fully transcribed<br>
cDNA by using a second primer during the amplification which is specific for the oligo-dC<br>
sequence.<br>
14.	Method according to any one of claims 1 to 13, wherein the tailing of the 3' end of the<br>
cDNA is performed with a polynucleotide sequence, which is characterized by the absence<br>
ofdC.<br><br>
15.	Method according to any one of claims 1 to 14, wherein the DNA polymerase used for<br>
the amplification reaction is Taq DNA polymerase, Tfl DNA polymerase, aTaq DNA polymerase,<br>
Sequenase or Klentaq and/or an enzyme with proofreading activity, preferably<br>
selected from the PFU, Ultma, Vent, Deep Vent, PWO and Tli polymerases and E.coli.<br>
exonuclease III.<br>
16.	Method according to any one of claims 1 to 15, wherein the RNA in the sample or<br>
the cDNA is preamplified, preferably by PCR, or wherein RNA samples are normalized or<br>
subtracted.<br>
17.	Method according to any one of claims 1 to 16, wherein a primer used for the amplification<br>
reaction is characterized by the general formula dPxdGyHNz, wherein Px represents<br>
nucleotides complementary to the sequence added to the 3' end of the cDNA, dG being<br>
deoxyguanylate and y an integer, preferably between 2 and 5, H being either dT or dA or<br>
dC, N being a sequence of nucleotides with a length z wherein its member nucleotides are<br>
independently selected from dA, dC, dG and dT, and z being an integer between 0 and 10,<br>
preferably between 0 and 5.<br>
18.	Method according to any one of claims 1 to 17, wherein the amplification is performed<br>
by PCR and the PCR primers contain anchoring sequences, which allow proper<br>
placement at the end of the 3' or 5' oligonucleotide end stretch of the cDNA.<br>
19.	Method according to any one of claims 1 to 18, wherein an amplification primer or a<br>
reverse transcription primer contains a promoter for RNA polymerases, preferably wherein<br>
the promoter enhances transcription by the T7 or T3 or SP6 RNA polymerases.<br>
20.	Method according to any one of claims 1 to 19, wherein the amplification primers<br>
and/or the amplification products are labeled by a reporter group, preferably fluorescence<br>
markers, chemiluminescence markers or radioisotopes, preferably wherein these reporter<br>
groups are used to detect the DNA.<br>
21.	Method according to any one of claims 1 to 20, wherein arbitrary nucleotides, preferably<br>
one to three nucleotides, are added to the 3' end of the RNA or mRNA posttranscrip-<br>
tionally but before the addition of a (e.g. poly A) tail, and the primers for reverse transcription<br>
and/or PCR contain a sequence with a wobble corresponding to the arbitrary sequence.<br><br>
22.	Method according to any one of the claims 1 to 21, wherein amplification products are<br>
generated that are differentially enlarged by defined numbers of nucleotides at their 5'<br>
and/or 3' end as compared to the RNA, by using different primers which have different<br>
lengths and recognize different 5' or 3' endsequences of the RNA or the full length cDNA.<br>
23.	Method according to any one of claims 1 to 22, wherein RNA or mRNA samples are<br>
used, which do not have a poly A tail or a tail sequence, and preferably a polynucleotide<br>
tail, especially preferred a poly A tail, is added synthetically prior to reverse transcription.<br>
24.	Method according to any one of claims 1 to 23, wherein the detection of differences<br>
between two cDNA pools, preferably corresponding pools between two samples, is done<br>
during or at the end of the amplification step, preferably by real time PCR.<br>
25.	Method according to any one of claims 1 to 24, wherein a 5' primer or a 3' primer<br>
or a 5'-3' primer pair, is spotted and/or bound onto a solid support, preferably the spot or<br>
the solid support is located in a defined 2-dimensional or 3-dimensional area and preferably<br>
such an area contains information to decode the identity of the primer or primer pair<br>
located in that area.<br>
26.	Method according to any one of claims 1 to 25, wherein the steps of reverse transcription<br>
and amplification are performed in one step.<br>
27.	Method according to any one of claims 1 to 26, wherein the polynucleotide tail added<br>
by the terminal transferase consists of ribonucleotides, preferably 2 to 6 ribonucleotides,<br>
preferably adenosine or thymidine, after a 3' sequence of oligo-dC, preferably with a length<br>
of 1 to 6 dC nucleotides, which is generated during reverse transcription, dependent on a 5'<br>
cap of a template RNA or mRNA, and the second amplification primer (5' primer) comprises<br>
complementary nucleotides to the ribonucleotides and oligo-dCs and 1 to 10 nucleotides<br>
upstream of the olio-dC sequence.<br>
28.	Method according to any one of claims 1 to 27, wherein a tail is added to the cDNA by<br>
the template switching ability of the reverse transcriptase, preferably in a cap dependent<br>
manner, and the second amplification primer (5' primer) comprises complementary nucleotides<br>
to the so added tail and oligo-dCs and 1 to 10 nucleotides upstream of the oligo-<br>
dC sequence.<br><br>
29.	Method for measuring the expression level of messenger RNA characterized by<br>
employing a method according to any one of claims 1 to 28, including the separation<br>
and/or identification of amplification results.<br>
30.	Kit for a method according to any one of claims 1 to 29 comprising a DNA poly-<br>
merase, preferably Taq polymerase, a reverse transcriptase or mixtures of different poly-<br>
merases, cofactors or salts of metal ions, preferably Mg2+ and Mn2+, requested by a polymerase,<br>
at least three primers comprising the formulas dPxdGydTNz, dPxdGydANz and<br>
dPxdGydCNz wherein dPx represents nucleotides complementary to a 3' end tail of a<br>
cDNA, dG being deoxyguanylate and y an integer, preferably between 2 and 5, N being a<br>
sequence of nucleotides with a length z wherein its member nucleotides are independently<br>
selected from dA, dC, dG and dT, and z being an integer between 0 and 10, preferably between<br>
0 and 5, being selective for the 3' end of a cDNA or the corresponding 5' end of a<br>
RNA, preferably being selective for the first, second, third, fourth and/or fifth terminal nu-<br>
cleotide of the RNA or cDNA, and a primer being selective for the 5' end of a cDNA or the<br>
corresponding 3' end of a RNA, and optionally a terminal transferase and additional buffer<br>
substances.<br>
31.	A solid surface, preferably a chip, in particular preferred a glass chip, comprising in<br>
defined areas, preferably in spots, at least two, preferably at least 9, especially preferred at<br>
least 20, most preferred at least 60, different 5' primers and 3' primers or pairs of primers,<br>
wherein the 5' primers are specific for the upstream portion of a 3' polynucleotide tail and<br>
the next 1 to 10 nucleotides upstream of the 3' polynucleotide tail and comprise the formulas<br>
dPxdGydTN2, dPxdGydANz and dPxdGydCNz, wherein dPx represents nucleotides<br>
complementary to the 3' polynucleotide tail of a cDNA, dG being deoxyguanylate and y an<br>
integer, preferably between 2 and 5, N being a sequence of nucleotides with a length z<br>
wherein its member nucleotides are independently selected from dA, dC, dG and dT, and z<br>
being an integer between 0 and 10, preferably between 0 and 5.<br>
32.	A solid surface, preferably a chip, in particular preferred a glass chip, comprising in<br>
defined areas, preferably in spots, at least two, preferably at least 9, especially preferred at<br>
least 20, most preferred at least 60, different 5' primers or 3' primers comprising the general<br>
formulas dPxdGydTNz, dPxdGydANz and dPxdGydCNz, wherein dPx represents nucleotides<br>
complementary to a 3' end tail of a cDNA, dG being deoxyguanylate and y an integer,<br>
preferably between 2 and 5, N being a sequence of nucleotides with a length z wherein its<br>
member nucleotides are independently selected from dA, dC, dG and dT, and z being an<br>
integer between 0 and 10, preferably between 0 and 5, or a complement thereof, preferably<br>
selected from a primer comprising one of the following sequences: AGA GAT TTT TTT<br><br>
TTT TTT TT GA, AGA GAT TTT TTT TTT TTT T GG, AGA GAT TTT TTT TTT TTT<br>
T GC, AGA GAT TTT TTT TTT TTT TT GT, AGA GAT TTT TTT TTT TTT TT CA,<br>
AGA GAT TTT TTT TTT TTT TT AG, AGA GAT TTT TTT TTT TTT TTT AA, AAA<br>
AAA AAA AAA AAA GGG ATA, AAA AAA AAA AAA AAA GGG ACA, AAA AAA<br>
AAA AAA AAA GGG AGA, AAA AAA AAA AAA AAA GGG AAA, AAA AAA AAA<br>
AAA AAA GGG GTA, AAA AAA AAA AAA AAA GGG CTA, AAA AAA AAA AAA<br>
AAA GGG TTA.<br>
33.	Method for amplifying a pool of polynucleotide molecules in a sample, characterized<br>
by the steps of:<br>
a)	obtaining a sample of RNA and reverse transcription of entire RNA molecules<br>
thus creating full length cDNA or obtaining a sample of full length cDNA,<br>
b)	tailing the 3' end of the transcribed cDNA with a polynucleotide tail after the 3'<br>
end,<br>
c)	amplification of the cDNA using a pair of primers, wherein a first (3') primer is<br>
specific for the 5' end of the cDNA and a second (5') primer is specific for the upstream<br>
portion of the polynucleotide tail and the next 1 to 100 nucleotides, preferably<br>
the next 1 to 50 nucleotides, especially preferred the next 1 to 20 nucleotides,<br>
in particular preferred the next 1 to 10 nucleotides, upstream of the 3'<br>
polynucleotide tail of the cDNA, preferably also comprising unspecific wobble<br>
nucleotides, especially preferred in the upstream portion within the 1 to 100 nu-<br>
cleotide range - wherein at least 1 nucleotide is specific for a nucleotide of the<br>
cDNA; the method further preferably defined as in any one of claims 1 to 29.<br>
34.	Method according to claim 1 or 33, characterized in that a primer, which is specific<br>
for a 3' tail of the RNA (optionally an added 31 tail) or mRNA (preferably the endogenous<br>
3' poly A tail) and the next 1 to 100 nucleotides, preferably the next 1 to 50 nucleotides,<br>
especially preferred the next 1 to 20 nucleotides, in particular preferred the next 1 to 10<br>
nucleotides, upstream of the 3' tail is used for the reverse transcription and/or a first amplification<br>
primer is specific for a 5' tail of the cDNA, which is complementary to the 3' tail<br>
of the RNA or mRNA, and the next 1 to 100 nucleotides, preferably the next 1 to 50 nucleotides,<br>
especially preferred the next 1 to 20 nucleotides, in particular preferred the next<br>
1 to 10 nucleotides, downstream of the 5' tail.<br><br>
The present invention provides a method for amplifying a pool of polynucleotide molecules<br>
in a sample, characterized by the steps of a) obtaining a sample or RNA and reverse<br>
transcription of entire RNA molecules thus creating full length cDNA or obtaining a sample<br>
of full length cDNA, b) tailing the 3' end of the transcribed cDNA with a polynucleotide<br>
tail after the 3' end, c) amplification of the cDNA using a pair of primers, wherein a<br>
first (3') primer is specific for the 5' end of the cDNA and a second (5') primer is specific<br>
for the a upstream portion of the polynucleotide tail and the next 1 to 10 nucleotides upstream<br>
of the 3' polynucleotide tail of the cDNA.</td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/viewdoc.aspx?id=qleu7p5FVMEWrWLz9vZbnA==&amp;amp;loc=wDBSZCsAt7zoiVrqcFJsRw==" target="_blank" style="word-wrap:break-word;">http://ipindiaonline.gov.in/patentsearch/GrantedSearch/viewdoc.aspx?id=qleu7p5FVMEWrWLz9vZbnA==&amp;amp;loc=wDBSZCsAt7zoiVrqcFJsRw==</a></p>
		<br>
		<div class="pull-left">
			<a href="269747-device-with-pull-tab-activation.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="269749-a-tool-for-manipulating-a-sample-of-developmental-cells-in-process-of-cryopreservation.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>269748</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>2518/KOLNP/2008</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>45/2015</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>06-Nov-2015</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>04-Nov-2015</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>23-Jun-2008</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>LEXOGEN GMBH</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td>BRUNNERSTRASSE 69/OBJEKT 3, A-1230 VIENNA</td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>SEITZ, ALEXANDER</td>
											<td>PENZINGERSTRASSE 100/TOP 5, A-1140 VIENNA</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>C12Q 1/68</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>PCT/AT2006/000494</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td>2006-11-29</td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td>A 1923/2005</td>
									<td>2005-11-29</td>
								    <td>Austria</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/269748-method-for-specific-amplification-of-polynucleotides by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 06:37:14 GMT -->
</html>
