<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/269872-nucleic-acid-enzymes-and-complexes-and-methods-for-their-use by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 06:16:50 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 269872:&quot;NUCLEIC ACID ENZYMES AND COMPLEXES AND METHODS FOR THEIR USE&quot;</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">&quot;NUCLEIC ACID ENZYMES AND COMPLEXES AND METHODS FOR THEIR USE&quot;</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>The present invention relates to methods that utilise multi-component nucleic acid complex (MNA complex) cascades. The MNA complexes may have cleavage or ligase activity. Further, the invention provides cascades which may include one or more DNAzymes. The invention also provides methods which use these cascades for the identification, detection and quantification of targets.</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td>TECHNICAL FIELD<br>
[ 0002 ] The present invention relates to methods that utilise multi-component nucleic acid (MNA) complexes. The MNA complexes may be active enzymes (e.g. MNAzymes including apta-MNAzymes) with modifying activity such as ligase or cleavage activity. Further, the invention relates to cascades which use NfNAzymes and may also include one or more DNAzymes. The invention also relates to methods which use these cascades for the identification, detection and quantification of targets.<br>
Background of the Invention<br>
[0003] Various publications, which may include patents, published applications, technical articles and scholarly articles, are cited throughout the specification in parentheses, and full citations of each may be found at the end of the specification. Each of these cited publications is incorporated by reference herein, in its entirety<br>
[0004] hi addition to their evolutionary optimized functions, the extraordinary physical and functional properties of nucleic acids provide the opportunity for a plethora of new bio-molecular devices and methods. Designer nucleic acids have been contemplated for therapeutic entities, biosensors, nano-scale devices and tools for molecular computation. The methods exploit the characteristics of DNA self-assembly, electro-conductivity, information elements, amplification, switching, molecular detection and catalytic activity. Further, since DNA is robust, stable and thermostable it provides an ideal material for molecular engineering of mechanical or computational devices.<br>
[ 0005 ] Single stranded nucleic acids, such as DNA and RNA, have the ability to fold into complex three-dimensional structures that can function as highly specific receptors<br><br>
(e.g. aptamers) and catalysts (e.g. ribozymes, DNAzymes). Further, the requirement for complementarity between nucleic acid strands for hybridization forms the basis for a wide range of techniques, which allow target detection (e.g. microarray analysis. Northern blotting or Southern blotting), and/or target ampliflcation (e.g. the polymerase chain reaction). Further, hybridization provides the basis for nucleic acid nano-scale construction and for DNA based computational strategies.<br>
[ 0006 ] A wide variety of nucleic acid molecules, with enzymatic or catalytic activity, have been discovered in the last 20 years. RNA enzymes ("ribozymes") occur in nature but can be engineered to specifically recognize and modify a target RNA substrate (Haseloff and Gerlach, 1988). In vitro evolution techniques have facilitated the discovery and development of many more catalytic nucleic acids, including deoxyribonucleic acids often referred to as "DNA enzymes" or "DNAzymes" (reviewed Emillson and Breaker, 2002). In vitro evolved DNAzymes and/or ribozymes have been discovered which have the capacity to catalyse a broad range of reactions including, but not limited to, cleavage of nucleic acids, ligation of nucleic acids, phosphorylation of nucleic acids, nucleic acid capping, amino acid adenylation, cofactor synthesis, RNA polymerization, template-directed polymerization, RNA-protein conjugation, aldol reaction, alcohol oxidation, aldehyde reduction, purine and pyrimidine nucleotide synthesis, alkylation, amide synthesis, urea synthesis, formation of peptide bonds, peptidyl-RNA synthesis, acyl transfer, aminoacylation, carbonate hydrolysis, phosphorothioate alkylation, porphyrin metallation, formation of caii&gt;on-catbon bonds, Pd nanoparticle formation, biphenyl isomerization, formation of ester bonds, formation of amide bonds, DNA deglycosylation, thymine dimer photoreversion or phosphoramidate cleavage (reviewed Silverman, 2007)<br>
[0007] In particular, DNAzymes and ribozymes have been characterized which specifically cleave distinct nucleic acid sequences after hybridizing via Watson Crick base pairing. DNAzymes are capable of cleaving either RNA (Breaker and Joyce, 1994; Santoro and Joyce, 1997; Santoro and Joyce, 1998) or DNA (Carmi et at., 1996) molecules. Ribozymes are also able to cleave both RNA (Haseloff and Gerlach, 1988) and DNA (Raiiiard and Joyce, 1996) target sequences.<br><br>
[0008] The "10:23" and "8:17" DNAzymes arc capable of cleaving nucleic acid substrates at specific RNA phosphodiester bonds. These DNAzymes cleave native 3'-5' phosphodiester linkages to create reaction products which have 2', 3'-cyclic phosphate and 5'-hydroxyl groups (Santoro and Joyce, 1997; reviewed Emilsson and Breaker, 2002).<br>
[0009] DNAzymes which specifically ligate distinct nucleic acid sequences after hybridizing to two independent substrate nucleic acids have also been characterized. Specific deoxyribozymes (DNAzymes) can ligate 2', 3'-cyclic phosphate and 5'-hydroxyl products and create non-native 2'-5' linkages. Examples of such DNAzymes include the "7281" and "7Z48" ligases (Prior et al, 2004) as well as the "7Q10" DNAzyme ligase (Flynn-Charlebois et al, 2003). Other DNAzymes have been described (Coppins and Silverman, 2004) which can ligate 2*, 3'diol and 5'-triphosphate products and create native 3'-5' linkages.<br>
[0010] Several catalytic nucleic acids have similar basic structures with multiple domains including a conserved catalytic domain ("catalytic core") flanked by two non-conserved substrate-binding domains ("arms"), which specifically recognize and hybridise to the substrate. Examples of nucleic acids with this basic structure include, but are not limited to, the hammerhead ribozyme, the 10:23 and 8:17 DNAzymes, the "7Z81", "7Z48" and "7Q10" DNAzyme ligases, the "UVIC" thymine dimer photoreversion DNAzyme and the "DAB22" caibon-carbon bond forming DNAzyme. To date catalytic nucleic acids are typically uni-molecular although examples of catalytic nucleic acids with more than one component are known but these required extensive engineering (Tabor et al, 2006, Kurata et al, 2000).<br>
[ 0011 ] Catalytic nucleic acids have been shown to tolerate only certain modifications in the area that forms the catalytic core (Peireault et al., 1990; Perreault et al., 1991; Zaborowska et al., 2002; Cruz et al., 2004; Silverman, 2004). Depending on the stringency of the reaction conditions, some degree of mismatch may be tolerated within the substrate arms. However, the requirement for Watson Crick base pairing is sufficiently strict so as to have enabled the development of protocols that use catalytic<br><br>
nucleic acids to facilitate the discrimination of closely related sequences (Calms et al., 2000) (WO 99/50452).<br>
[ 0012 ] Target amplification and detection technologies, such as PCR, have been widely used in research and/or in clinical diagnostics. However, despite their power, each has inherent disadvantages. They all require the use of protein enzymes (e.g. DNA polymerase, RNA polymerase, reverse transcriptase, and or ligase). The inclusion of protein enzymes increases the complexity and cost of reagent manufacture and decreases the shelf life of kits containing reagents. Other associated technical challenges include contamination by replicons (target amplicons) fhsm previous reactions leading to false positive signal, and/or background signal caused by replication of primer sequences (primer-dimers) or background caused by target-independent ligation.<br>
[ 0013 ] Nucleic acid enzymes and nucleic acid enzyme cascades have been considered for a range of biotechnological applications, especially in diagnostics. They could allow detection of proteins and nucleic acids for disease diagnosis by facilitating signal amplification.<br>
[ 0014 ] Several groups have reported using catalytic nucleic acids for the detection of nucleic acid targets, and other analytes with colourimetric readouts (Elghanian et al., 1997, Mirkin et al, 1996, and Liu and Lu, 2004). Examples of signal amplification cascades, which use uni-molecular catalytic nucleic acids, are known in the art. For example, Paul and Joyce (2004) described a replication cascade mediated by a ribozyme with ligase activity. In another approach, a signal amplification cascade used two inactive, circularized 10:23 DNAzymes which were capable of activating each other by cross cleavage resulting in linearisation (Levy and Ellington, 2003).<br>
[ 0015 ] There is an ongoing need in the art for the detection of targets, in particular detection using amplification systems such as cascades, for example, detection systems involving cascades which comprise a plurality of nucleic acid enzymes and in particular at least one multicomponent nucleic acid enzyme (MNAzyme). The present invention provides detection methods involving nucleic acid enzyme cascades which incorporate at least one nucleic acid enzyme with ligase activity. Moreover, the use of nucleic acid<br><br>
enzymes with ligase activity allows formation of components for nucleic acid complexes, such as assembly facilitators, partzymes, substrates, DNAzymes or components thereof<br>
SUMMARY OF THE INVENTION<br>
[ 0016 ]    In one aspect of the invention there is provided a method for detecting the presence of at least one target comprising<br>
(a)	providing at least two or more oligonucleotide components and at least one substrate of at least a first nucleic acid complex wherein at least a first oligonucleotide component and a second oligonucleotide component are capable of self-assembly in the presence of a target to form at least a first nucleic acid enzyme which comprises a catalytically active multi-component nucleic acid enzyme with ligase activity (MNAzyrae ligase),<br>
(b)	contacting the two or more oligonucleotide components and at least one substrate with a sample putatively containing the at least one target under conditions permitting:<br><br>
(1)	the self-assembly of said at least one catalytically active MNAzyme ligase and<br>
(2)	the catalytic activity of said at least one MNAzyme ligase; and<br>
(c)	determining the presence of the catalytic activity of said at least one MNAzyme<br>
ligase, wherein the presence of the catalytic activity is indicative of the presence of said<br>
at least one target.<br>
[ 0017 ]   In another aspect of the invention there is provided a method for the detection of a target comprising<br>
a)	providing a first nucleic acid complex comprising components which require association with said target to facilitate formation of a first nucleic acid enzyme, wherein said first nucleic acid complex further comprises at least one substrate, and<br>
b)	providing at least one component of a second nucleic acid complex capable of associating with a product of a reaction catalysed by the first nucleic acid enzyme<br>
and forming a second nucleic acid complex capable of functioning as a second nucleic acid enzyme<br><br>
wherein at least one of said nucleic acid enzymes has ligase activity and at least one of said nucleic acid enzymes is an MNAzyme, and wherein said method comprises:<br>
i) contacting a sample putatively containing a target with the first nucleic acid complex, wherein said target associates with said first nucleic acid complex to facilitate formation of said first nucleic acid enzyme, and<br>
ii) contacting a product of the reaction catalysed by the first nucleic acid enzyme with said at least one component of the second nucleic acid complex thereby permitting formation of the second nucleic acid complex which ftmctions as said second nucleic acid enzyme, and<br>
wherein activity of at least one of the fu^t or second nucleic acid enzymes or both is indicative of the presence of said target.<br>
[ 0018 ]   In one embodiment the method may further comprise<br>
providing at least one component of a third nucleic acid complex capable of associating with a product of a reaction catalysed by the first or second nucleic acid enzyme or both<br>
and forming a third nucleic acid complex capable of functioning as a third nucleic acid enzyme wherein said method comprises<br>
contacting a product of the reaction catalysed by the first or second nucleic acid enzyme or both with said at least one component of the third nucleic acid complex thereby permitting formation of a third nucleic acid complex which functions as said third nucleic acid enzyme, and<br>
wherein activity of the first, second, or thitxl nucleic acid enzymes or any combination thereof is indicative of the presence of said target.<br>
[ 0019 ]   In another embodiment the method may further comprise<br>
providing at least one component of at least one additional nucleic acid complex<br>
capable of associating with a product of at least one of said first, second or third nucleic acid enzymes<br><br>
and forming at least one additional nucleic acid complex capable of functioning as at least one additional nucleic acid enzyme wherein said method comprises<br>
contacting a product of the reaction catalysed by at least one of said first, second or third nucleic acid enzymes with said at least one component of the at least one additional nucleic acid complex thereby permitting formation of at least one additional nucleic acid complex which functions as said at least one additional nucleic acid enzyme, and<br>
wherein activity of the first, second, third or at least one additional nucleic acid enzymes or any combination thereof is indicative of the presence of said target.<br>
[ 0020 ] In one embodiment at least one of said nucleic acid enzymes may have cleavage activity or ligase activity. Moreover the first nucleic acid enzyme may be an MNAzyme. Further at least one of the nucleic acid enzymes may be a DNAzyme.<br>
[ 0021 ] In another embodiment a product of a reaction catalysed by any one of the nucleic acid enzymes may be selected from the group comprising an assembly facilitator, partzyme, substrate, DNAzyme, stabiliser arm or component thereof.<br>
[ 0022 ]   In one embodiment the target may be a nucleic acid.<br>
[ 0023 ] In one embodiment at least one of said components may further comprise at least one aptamer and wherein said method may further comprise providing at least one assembly facilitator and at least one assembly inhibitor of said first nucleic acid complex wherein said target associates with said aptamer allowing said first nucleic acid complex to function as a first nucleic acid enzyme.<br>
[0024] In one embodiment the target is a non-nucleic acid target. The assembly facilitator may be an additional target to be detected.<br>
[ 0025 ] In a further embodiment a product of a reaction catalysed by at least one of said nucleic acid enzymes, or any combination thereof, associates with at least one component of at least one of the preceding nucleic acid complexes thereby permitting formation of further at least one of said preceding nucleic acid complexes which functions as further said preceding nucleic acid enzyme.<br>
[ 0026 ] In another aspect of the invention there is provided a method for the detection of a first assembly facilitator using a cascade comprising<br><br>
a)	providing components for a first MNAzyme, at least a first MNAzyme substrate, a first DNAzyme Hgase, a first DNAzyme ligase substrate, a second assembly facilitator, at least a second MNAzyme substrate, and a first partzyme for a second MNAzyme wherein said method comprises:<br>
b)	associating said first assembly ifacilitator with said components for said first MNAzyme under conditions permitting catalytic activity of said first MNAzyme thus facilitating the catalytic activity of said first MNAzyme thereby allowing modification of said first MNAzyme substrate wherein;<br>
the modification of the first MNAzyme substrate results in the production of a second DNAzyme ligase substrate; and<br>
wherein the first DNAzyme ligase assembles with the first and second DNAzyme ligase substrates; and<br>
wherein catalytic activity of the DNAzyme ligase produces a second partzyme for the second MNAzyme, and wherein;<br>
association of said second partzyme for the second MNAzyme with said second assembly facilitator, and the first partzyme for the second MNAzyme under conditions permitting catalytic activity of the second MNAzyme facilitates the catalytic activity of said second MNAzyme thereby allowing modification of at least the second MNAzyme substrate wherein modification of any of said substrates, or a combination thereof, is indicative of the presence of said first assembly facilitator.<br>
[ 0027 ]    In another aspect of the invention there is provided a method for the detection of a first assembly facilitator using a cascade comprising<br>
a)	providing components for a first MNAzyme, at least a first MNAzyme substrate, a DNAzyme ligase, a first DNAzyme ligase substrate, at least a second MNAzyme substrate, and first and second partzymes for a second MNAzyme and wherein said method comprises:<br>
b)	associating said first assembly facilitator with said components for the first MNAzyme under conditions permitting catalytic activity of said first MNAzyme thus<br><br>
facilitating the catalytic activity of said first MNAzyme thereby allowing modification of said first MNAzyme substrate wherein;<br>
the modification of the first MNAzyme substrate results in the production of a second DNAzyme ligase substrate; and<br>
wherein the first DNAzyme ligase assembles with the first and second DNAzyme ligase substrates; and<br>
wherein catalytic activity of the DNAzyme ligase produces a second assembly facilitator for the second MNAzyme, and<br>
wherein association of said second assembly facilitator with said first and second partzymes for the second MNAzyme under conditions permitting catalytic activity of the said second MNAzyme facilitates the catalytic activity of said second MNAzyme thereby allowing modification of at least the second MNAzyme substrate wherein modification of any of said substrates, or a combination thereof, is indicative of the presence of said fiirst assembly facilitator.<br>
[ 00281 In one embodiment a product of the reaction catalysed by the second MNAzyme may be a component required to permit function of a third nucleic acid enzyme. A product of the reaction catalysed by the third nucleic acid enzyme may be a component which participates in a reaction catalysed by the second MNAzyme or the DNAzyme ligase, or both.<br>
[ 0029 ] In another aspect of the invention there is provided a method for the detection of a first assembly facilitator using a cascade comprising<br>
a)	providing components for a first MNAzyme, at least a first MNAzyme substrate, components for a second MNAzyme wherein said second MNAzyme has ligase activity, a second assembly facilitator for the second MNAzyme, a second MNAzyme substrate, a third assembly facilitator for a third MNAzyme, at least a third MNAzyme substrate, and a first partzyme for the third MNAzyme and wherein said method comprises:<br>
b)	associating said first assembly facilitator with said components for said first MNAzyme under conditions permitting catalytic activity of said first MNAzyme thus<br><br>
facilitating the catalytic activity of said first MNAzyme thereby allowing modification of said first MNAzyme substrate wherein;<br>
the modification of the first MNAzyme substrate results in the production of a fourth MNAzyme substrate; and<br>
wherein the second MNAzyme assembly facilitator facilitates assembly of the components for a second MNAzyme with the second and fourth MNAzyme substrates; and<br>
wherein catalytic ligase activity of the second MNAzyme produces a second partzyme for the third MNAzyme, and wherein;<br>
association of said first and second partzymes for the third MNAzyme with said third assembly facilitator, under conditions permitting catalytic activity of the third MNAzyme facilitates the catalytic activity of said third MNAzyme thereby allowing modification of at least the third MNAzyme substrate wherein modification of any of said substrates, or a combination thereof, is indicative of the presence of said first assembly facilitator.<br>
[ 0030 ]    In another aspect of the invention there is provided a method for the detection of a first assembly facilitator using a cascade comprising<br>
a)	providing components for a first MNAzyme, at least a first MNAzyme substrate,<br>
components for a second MNAzyme wherein said second MNAzyme has ligase activity,<br>
a second assembly facilitator for the second MNAzyme, a second MNAzyme substrate, a<br>
first and a second partzyme for a third MNAzyme and at least a third MNAzyme<br>
substrate, and wherein said method comprises:<br>
b)	associating said first assembly facilitator with components for said first<br>
MNAzyme under conditions permitting catalytic activity of said first MNAzyme thus<br>
facilitating the catalytic activity of said first MNAzyme thereby allowing modification of<br>
said first MNA2yme substrate wherein;<br>
the modification of said first MNAzyme substrate results in the production of a fourth MNAzyme substrate; and<br><br>
wherein the second MNAzyme assembly facilitator facilitates assembly of the said components for second MNAzyme with the second and fourth MNAzyme substrates; and<br>
wherein catalytic ligase activity of the second MNAzyme produces a third assembly facilitator for the third MNAzyme, and wherein;<br>
association of said first and second partzymes for the third MNAzyme Avith said third assembly facilitator, imder conditions permitting catalytic activity of the third MNAzyme facilitates the catalytic activity of said third MNAzyme thereby allowing modification of at least the third MNAzyme substrate wherein modification of any one of said substrates, or a combination thereof, is indicative of the presence of said first assembly facilitator.<br>
[ 0031 ] In one embodiment a product of the reaction catalysed by the third MNAzyme may be a component required to permit function of a fourth nucleic acid enzyme. A product of the reaction catalysed by the fourth nucleic acid enzyme may be a component which participates in a reaction catalysed by the second or third MNAzyme or both.<br>
[ 0032 ] In one embodiment the modification of any one of said substrates, or a combination thereof, provides a detectable effect. The modification of at least one substrate may be selected fiom the group comprising cleavage of nucleic acids, ligation of nucleic acids, phosphorylation of nucleic acids, nucleic acid capping, amino acid adenylation, cofactor synthesis, RNA polymerization, template-directed polymerization, RNA-protein conjugation, aldol reaction, alcohol oxidation, aldehyde reduction, purine and pyrimidine nucleotide synthesis, alkylation, amide synthesis, urea synthesis, formation of peptide bonds, peptidyl-RNA synthesis, acyl transfer, amiiraacylation, carbonate hydrolysis, phosphorothioate alkylation, porphyrin metallation, formation of carbon-carbon bonds, Pd nanoparticle formation, biphenyl isomerization, formation of ester bonds, formation of amide bonds, DNA deglycosylation, thymine dimer photoreversion and phosphoramidate cleavage.<br>
[ 0033 ] The finst assembly facilitator may be a target to be identified, detected or quantitated.<br><br>
[ 0034 ] In one embodiment the DNAzyme ligase may be selected from the group comprising DNAzyme ligases 7Z81, 7Z48, 7Q10 and 9DB1.<br>
[ 0035 ] In another embodiment a product of the second MNAzyme may be the second DNAzyme ligase substrate, and wherein the DNAzyme ligase facilitates the ligation of the first and the second DNAzyme ligase substrates thereby producing further second partzyme for the second MNAzyme. A product of the second MNAzyme may be the second DNAzyme ligase substrate, and wherein the DNAzyme ligase facilitates the ligation of the first and the second DNAzyme ligase substrates thereby producing further second assembly facilitator for the second MNAzyme. Moreover a product of the third MNAzyme may be the fourth MNAzyme substrate, and wherein the second MNAzyme facilitates the ligation of the second and the fourth MNAzyme substrates thereby producing a further second partzyme for the third MNAzyme.<br>
[ 0036 ] In a fiirther embodiment a product of the third MNAzyme may be the fourth MNAzyme substrate, and wherein the second MNAzyme facilitates the ligation of the second and the fourth MNAzyme substrates thereby producing a further third assembly facilitator for the third MNAzyme.<br>
[ 0037 ] In another aspect of the invention there is provided use of a pair of isolated nucleic acids as catalytic core sequences incorporated into a partzyme pair for an MNAzyme ligase wherein said pairs of nucleic acids are selected from the group consisting of TGGAGGTGGGCTC (SEQ ID NO: 47) and ACG (SEQ ID NO: 48), GGAGGTGGGCTC (SEQ ID NO: 49) and ACGT (SEQ ID NO: 50), GGAGGTTAGCTC (SEQ ID NO : 71) and ACGGCGGAGTGATTG (SEQ ID NO: 72), TGGGAGGTTAGCTC (SEQ ID NO: 73) and ACGGCGGAGTGAT (SEQ ID NO : 74), ATTGGGAGGTTAGCTC (SEQ ID NO : 75) and ACGGCGGAGTG (SEQ ID NO : 76), TGATTGGGAGGTTAGCTC (SEQ ID NO : 77) and ACGGCGGAG (SEQ ID NO: 78), AGTGATTGGGAGGTTAGCTC (SEQ ID NO: 79) and ACGGCGG (SEQ ID NO : 80), GGAGTGATTGGGAGGTTAGCTC (SEQ ID NO : 81) and ACGGC (SEQ ID NO : 82).<br>
[ 0038 ] In another aspect of the invention there is provided an MNAzyme ligase comprising a pair of isolated nucleic acids as catalytic core sequences incorporated into a<br><br>
partzyme pair wherein said pairs of nucleic acids are selected fix)m the group consisting of TGGAGGTGGGCTC (SEQ ID NO: 47) and ACG (SEQ ID NO: 48), GGAGGTGGGCTC (SEQ ID NO: 49) and ACGT (SEQ ID NO: 50), GGAGGTTAGCTC (SEQ ID NO: 71) and ACGGCGGAGTGATTG (SEQ ID NO: 72), TGGGAGGTTAGCTC (SEQ ID NO: 73) and ACGGCGGAGTGAT (SEQ ID NO: 74), ATTGGGAGGTTAGCTC (SEQ ID NO : 75) and ACGGCGGAGTG (SEQ ID NO: 76), TGATTGGGAGGTTAGCTC (SEQ ID NO: 77) and ACGGCGGAG (SEQ ID NO: 78), AGTGATTGGGAGGTTAGCTC (SEQ ID NO: 79) and ACGGCGG (SEQ ID NO : 80), GGAGTGATTGGGAGGTTAGCTC (SEQ ID NO : 81) and ACGGC (SEQ ID NO : 82).<br>
[ 0039 ] In another aspect of the invention there is provided use of an MNAzyme ligase to iigate two substrates wherein said substrates are ligated by said MNAzyme ligase to form a product following the engagement of said substrates by the substrate arms of said MNAzyme ligase.<br>
[ 0040 ] In one embodiment the product may associate with the components of at least one nucleic acid complex.<br>
[ 0041 ] in another embodiment the product may be selected from the group comprising assembly facilitator, partzyme, substrate, DNAzyme, stabiliser arm, activity inhibitor, assembly inhibitor, or component thereof. The product may participate in a cascade.<br>
[ 0042 ] In another embodiment at least one substrate is the product of a reaction catalysed by a nucleic acid enzyme.<br>
[ 0043 ] In another aspect of the invention there is provided a method for the detection of a plurality of targets comprising:<br>
a)	providing a plurality of fu-st nucleic acid complexes each comprising components which require association with one of said plurality of targets to facilitate formation of a plurality of first nucleic acid enzymes, wherein each of said plurality of first nucleic acid complexes further comprises at least one substrate, and<br>
b)	providing at least one component of each of a plurality of second nucleic acid complexes capable of associating with a plurality of products of reactions catalysed by<br><br>
the plurality of first nucleic acid enzymes and forming a plurality of second nucleic acid complexes capable of functioning as a plurality of second nucleic acid enzymes, wherein at least one of said nucleic acid enzymes has ligase activity and at least one of said nucleic acid enzymes is an MNAzyme, and wherein said method comprises:<br>
i) contacting a sample putatively containing said plurality of targets with the plurality of first nucleic acid complexes, wherein said plurality of targets associate with said plurality of first nucleic acid complexes to facilitate formation of said plurality of first nucleic acid enzymes, and<br>
ii) contacting a plurality of products of the reactions catalysed by the plurality of first nucleic acid enzymes with at least one component of each of the plurality of second nucleic acid complexes thereby permitting formation of the plurality of said second nucleic acid complexes which function as said plurality of second nucleic acid enzymes, and<br>
wherein activity of the first or second or both pluralities of nucleic acid enzymes is indicative of the presence of said plurality of targets.<br>
[ 0044 ] In one embodiment at least one of the products of the reactions catalysed by any one of the plurality of first or second nucleic acid enzymes may be a component required to permit function of at least one additional nucleic acid enzyme.<br>
[ 0045 ] In another embodiment at least one of the products of the reactions catalysed by any one of the nucleic acid enzymes associates with the components of any one of the preceding nucleic acid complexes thereby permitting at least one of the nucleic acid complexes to function as further nucleic acid enzyme.<br>
[ 0046 ] In another aspect of the invention there is provided use of at least two oligonucleotides as multi-component nucleic acid enzyme ligase (MNAzyme ligase) substrates wherein said substrates are modified by an MNAzyme ligase following the engagement of said substrates by the substrate arms of said MNAzyme ligase.<br>
[ 0047 ] In another aspect of the invention there is provided a method for the detection of at least two targets wherein said method comprises:<br><br>
a)	providing at least two nucleic acid complexes each comprising components which require association with at least one of said targets to facilitate fonnation of at least two nucleic acid enzyme complexes, and<br>
b)	contacting a sample putatively containing said at least two targets with said at least two nucleic acid complexes, wherein said targets associate with said nucleic acid complexes to facilitate formation of said nucleic acid enzyme complexes, and wherein at least one product created by the activity of each nucleic acid enzyme complex is required for formation of an additional nucleic acid enzyme complex, and<br>
wherein activity of the additional nucleic acid enzyme complex is indicative of the presence of said at least two targets.<br>
[ 0048 ] In one embodiment at least one of said nucleic acid enzyme complexes may have iigase activity. Further, at least one of said nucleic acid oizyme complexes may be an MNAzyme complex.<br>
[ 0049 ] According to another aspect of the invention there is provided use of an NfNAzyme with Iigase activity (MNAzyme Iigase) to switch an active KfNAzyme to an MNAi complex; wherein<br>
said active MNAzyme is formed in the presence of at least a first assembly facilitator component and a second assembly facilitator component and wherein<br>
said MNAzyme Iigase forms in the presence of at least a third assembly facilitator and wherein<br>
said MNAzyme Iigase facilitates the ligation of said second assembly facilitator component with an activity inhibitor component to form an activity inhibitor; and wherein<br>
said activity inhibitor associates with one or more components of said MNAzyme to form an MNAi complex.<br>
[ 0050 ] In one embodiment at least one of the assembly facilitators, activity inhibitor component or activity inhibitor may be nucleic acid molecules.<br><br>
[ 0051 ] In one embodiment at least one of the assembly facilitators, activity inhibitor component or activity inhibitor may be comprised of DNA or an analogue thereof.<br>
[0052 J According to another aspect of the invention there is provided a multicomponent nucleic acid enzyme with ligase activity comprising at least two or more oligonucleotide components wherein at least a first oligonucleotide component and a second oligonucleotide component self-assemble in the presence of an MNAzyme ligase assembly facilitator to form a catalytically active multi-component nucleic acid enzyme with ligase activity (MNAzyme ligase),<br>
wherein each of said at least first and said second oligonucleotide components comprise a substrate arm portion, a catalytic core portion, and a sensor arm portion;<br>
wherein upon self-assembly, the sensor arm portion of said first and second oligonucleotide components act as sensor arms of the NfNAzyme ligase, the substrate arm portion of the first and second oligonucleotide components act as substrate arms of the MNAzyme ligase, and the catalytic core portion of the first and second oligonucleotide components act as a catalytic core of the MNAzyme ligase; and<br>
wherein the sensor arms of the MNAzyme ligase interact with said MNAzyme ligase assembly facilitator so as to maintain the first and second oligonucleotide components in proximity for association of their respective catalytic core portions to form the catalytic core of the MNAzyme ligase, said catalytic core capable of ligating at least two substrates, and<br>
wherein said substrate arms of said MNAzyme ligase engage first and second substrates so that said catalytic core of said MNAzyme can iigate said substrates. [ 0053 ]   In one embodiment at least one of said oligonucleotide components, assembly facilitator or substrates of the multicomponent nucleic acid enzyme with ligase activity may comprise DNA or an analogue thereof.<br>
[ 0054 ]   In one embodiment ligation of the substrates may provide a detectable effect.<br>
[ 0055 ] In another aspect of the invention there is provided a composition comprising at least two oligonucleotide partzymes each comprising at least a catalytic core portion, a sensor arm portion, and a reporter arm portion, and at least three fiuther oligonucleotide components comprising an assembly facilitator oligonucleotide and at least two substrate<br><br>
nucleic acids, wherein said composition is capable of forming a multicomponent nucleic acid complex capable of ligating the substrate nucleic acids.<br>
[ 0056 ] In one embodiment at least one of said oligonucleotide components, assembly facilitator or substrates of the composition may comprise DNA or an analogue thereof.<br>
[ 0057 ] In another aspect of the invention there is provided a composition comprising at least two or more oligonucleotide components wherein at least a first oligonucleotide component and a second oligonucleotide component self-assemble in the presence of an MNAzyme assembly facilitator to form a catalytically active multi-component nucleic acid enzyme with ligase activity (MNAzyme ligase)<br>
wherein each of said at least first and said second oligonucleotide components comprise a substrate arm portion, a catalytic core portion, and a sensor arm portion;<br>
wherein upon self-assembly, the sensor arm portion of said first and second oligonucleotide components act as sensor arms of the MNAzyme ligase, the substrate arm portion of the first and second oligonucleotide components act as substrate arms of the MNAzyme ligase, and the catalytic core portion of the first and second oligonucleotide components act as a catalytic core of the MNAzyme ligase;<br>
and wherein the sensor arms of the MNAzyme ligase interact with said MNAzyme assembly facilitator so as to maintain the first and second oligonucleotide components in proximity for association of their respective catalytic core portions to form the catalytic core of the MNAzyme ligase, said catalytic core capable of ligating at least one substrate to another, and<br>
wherein said substrate anns of said MNAzyme ligase engage first and second substrates so that said catalytic core of said MNAzyme ligase can ligate said substrates. [ 0058 ]   In one embodiment ligation of the substrates may provide a detectable effect.<br>
[ 0059 ] In one embodiment at least one of said oligonucleotide components, assembly facilitator or substrates of the composition may comprise DNA or an analogue thereof.<br>
[ 0060 ] In another aspect of the invention there is provided a multicomponent nucleic acid enzyme with ligase activity wherein at least one oligonucleotide component of said MNAzyme further comprises an aptamer.<br><br>
[ 0061 ]      In another aspect at least one of the assembly facilitator or substrates further comprises an aptamer.<br>
[ 0062 ] In one embodiment assembly of the MNAzyme ligase may be directed by the presence or absence of one or more analytes capable of binding the aptamer.<br>
[ 0063 ] In another aspect of the invention there is provided a method for detecting the presence of at least one assembly facilitator comprising<br>
(a)	providing at least two or more oligonucleotide components wherein at least a<br>
first oligonucleotide component and a second oligonucleotide component self-assemble<br>
in the presence of an assembly facilitator to form at least one catalytically active multi-<br>
component nucleic acid enzyme with ligase activity (MNAzyme ligase),<br>
(b)	contacting the two or more oligonucleotide components with a sample<br>
putatively containing the assembly facilitator under conditions permitting:<br>
(1)	the self-assembly of said at least one catalytically active MNAzyme ligase and<br>
(2)	the catalytic activity of said at least one MNAzyme ligase; and<br>
(c) determining the presence of the catalytic activity of said at least one MNAzyme ligase, wherein the presence of the catalytic activity is indicative of the presence of said at least one assembly facilitator.<br>
[0064] In one embodiment at least one of said oligonucleotide components or assembly facilitator may be comprised of DNA or an analogue thereof.<br>
[ 0065 ] In one embodiment said assembly facilitator may be a target to be identified, detected or quantitated.<br>
[ 0066 ]    In one embodiment said target may be a nucleic acid.<br>
[ 0067 ] In a preferred emobodiment said nucleic acid may be selected fix&gt;m the group comprising DNA, methylated DNA, alkylated DNA, RNA, methylated RNA, microRNA, siRNA, shRNA, mRNA, tRNA, snoRNA, stRNA, smRNA, pre- and pri-microRNA, other non-coding RNAs, ribosomal RNA, derivatives thereof, amplicons thereof or any combination thereof.<br><br>
[ 0068 J In one embodiment the source of the nucleic acid may be selected from the group comprising synthetic, mammalian, human, animal, plant, fungal, bacterial, viral, archael sources or any combination thereof.<br>
[ 0069 ] In one embodiment the method may further comprise a step of amplifying the assembly facilitator.<br>
[0070] In one embodiment the step of amplifying may comprise one or more of: polymerase chain reaction (PCR), strand displacement amplification, loop-mediated isothermal amplification, rolling circle amplification, transcription-mediated amplification, self-sustained sequence replication, ligase chain reaction, nucleic acid sequence based amplification, or reverse transcription polymerase chain reaction (RT-PCR).<br>
[0071] In one embodiment at least one of said assembly facilitator, said oligonucleotide components or substrates for the reaction or a combination thereof may be comprised of more than one molecule.<br>
[ 0072 ] In one embodiment the method may further comprise determination of the presence of said catalytic activity during or after said amplification.<br>
[ 0073 ] In one embodiment the self assembly of the \fNA2yme ligase may require contact of the assembly facilitator with one or both of said first and second oligonucleotide components.<br>
[ 0074 ] In one embodiment the method may further comprise providing at least a third oligonucleotide component that contacts at least a portion of either or both of the first and second oligonucleotide components to self-assemble the MNAzyme ligase.<br>
[ 007S ] In one embodiment the method said third oligonucleotide component may be comprised of more than molecule.<br>
[ 0076 ] In one aspect of the invention there is provided an isolated nucleic acid sequence capable of forming part of a catalytic core of an MNAzyme ligase comprising the sequence selected from the group comprising TGGAGGTGGGCTC (SEQ ID NO : 47). ACG (SEQ ID NO: 48), GGAGGTGGGCTC (SEQ ID NO: 49), ACGT (SEQ ID NO: 50),  GGAGGTTAGCTC  (SEQ ID NO : 71). ACGGCGGAGTGATTG (SEQ ID<br><br>
NO : 72). TGGGAGGTTAGCTC (SEQ ID NO : 73), ACGGCGGAGTGAT (SEQ ID NO : 74), ATTGGGAGGTTAGCTC (SEQ ID NO : 75), ACGGCGGAGTG (SEQ ID NO : 76), TGATTGGGAGGTTAGCTC (SEQ ID NO : 77), ACGGCGGAG (SEQ ID NO: 78), AGTGATTGGGAGGTTAGCTC (SEQ ID NO: 79), ACGGCGG (SEQ ID NO : 80), GGAGTGATTGGGAGGTTAGCTC (SEQ ID NO : 81), and ACGGC (SEQ ID NO: 82).<br>
[ 0077 ] According to another aspect of the present invention, there is provided a method for the detection of a first assembly facilitator using a cascade comprising components for a first MNAzyme, a first MNAzyme substrate, components for a second MNAzyme, a second assembly facilitator for the second MNAzyme, a second MNAzyme substrate, a third assembly facilitator for the third MNAzyme, at least a third MNAzyme substrate, and a first partzyme for the third MNAzyme wherein<br>
association of said first assembly facilitator with said components for a first MNAzyme under conditions permitting catalytic activity of said first MNAzyme facilitates the catalytic activity of said first MNAzyme thereby cleaving the said first MNAzyme substrate wherein;<br>
the cleavage of the first MNAzyme substrate results in the production of a fourth MNAzyme substrate; and<br>
wherein the second MNAzyme assembly facilitator facilitates assembly of the second MNAzyme with the second and fourth MNAzyme substrates; and<br>
wherein catalytic ligase activity of the second MNAzyme produces a second partzyme for the third MNAzyme, and wherein;<br>
association of said first and second partzymes for the third MNAzyme with said third assembly facilitator, under conditions permitting catalytic activity of the third MNAzyme facilitates the catalytic activity of said third MNAzyme thereby providing modification of the at least a third MNAzyme substrate.<br>
[ 0078 ] The modification of the first, second or at least a third MNAzyme substrates or any combination thereof may provide a detectable effect.<br><br>
[ 0079 ] The modification of the at least a third MNAzyme substrate may be selected from the group comprising cleavage of nucleic acids, ligation of nucleic acids, phosphorylation of nucleic acids, nucleic acid capping, amino acid adenylation, cofactor synthesis, RNA polymerization, template-directed polymerization, RNA-protein conjugation, aldol reaction, alcohol oxidation, aldehyde reduction, purine and pyrimidine nucleotide synthesis, alkylation, amide synthesis, urea synthesis, formation of peptide bonds, peptidyl-RNA synthesis, acyl transfer, aminoacylation, carbonate hydrolysis, phosphorothioate alkylation, porphyrin metallation, formation of carbon-carbon bonds, Pd nanoparticle formation, biphenyl isomerization, formation of ester bonds, formation of amide bonds, DNA deglycosylation, thymine dimer photoreversion and phosphoramidate cleavage.<br>
[ 0080 ] In one embodiment of the invention the method further comprises an amplification cascade wherein a product of the third MNAzyrae is the fourth MNAzyme substrate, and wherein the second MNAzyme facilitates the ligation of the second and the fourth MNAzjone substrates thereby producing a further second partzyme for the third<br>
MNAzyme.<br>
[ 0081 ] According to another aspect of the present invention, there is provided a method for the detection of a first assembly facilitator using a cascade comprising a components for a first MNAzyme, a first MNA2yme substrate, components for a second MNAzyme, a second MNAzyme assembly facilitator, a second MNAzyme substrate, components for a third MNAzyme and at least a third MNAzyme substrate, wherein<br>
association of said first assembly facilitator with said components for a first MNAzyme under conditions permitting catalytic activity of said first MNAzyme facilitates the catalytic activity of said first MNAzyme thereby cleaving the said first MNAzyme substrate wherein;<br>
the cleavage of the first MNAzyme substrate resuhs in the production of the fourth MNAzyme substrate; and<br>
wherein the second MNAzyme assembly facilitator facilitates assembly of the second MNAzyme with the second and fourth MNAzyme substrates; and<br><br>
wherein catalytic activity of the second MNAzyme produces a third MNAzyme assembly facilitator for the third MNAzyme, and wherein;<br>
association of said third MNAzyme assembly facilitator with said components for a third MNAzyme under conditions permitting catalytic activity of the third MNAzyme facilitates the catalytic activity of said third MNAzyme thereby providing modification of the at least a third MNAzyme substrate.<br>
[ 0082 ] The modification of the first, second or at least a third MNAzyme substrate or any combination thereof may provide a detectable effect.<br>
[ 0083 ] The modification of the at least a third MNAzyme substrate may be selected from the group comprising cleavage of nucleic acids, ligation of nucleic acids, phosphorylation of nucleic acids, niicleic acid capping, amino acid adenylation, cofactor synthesis, RNA polymerization, template-directed polymerization, RNA-protein conjugation, aldol reaction, alcohol oxidation, aldehyde reduction, purine and pyrimidine nucleotide synthesis, alkylation, amide synthesis, urea synthesis, formation of peptide bonds, peptidyl-RNA synthesis, acyl transfer, aminoacylation, carbonate hydrolysis, phosphorothioate alkylation, porphyrin metallation, formation of carbon-carbon bonds, Pd nanoparticle formation, biphenyl isomerization, formation of ester bonds, formation of amide bonds, DNA deglycosylation, thymine dimer photoreversion and phosphoramidate cleavage.<br>
[0084] In one embodiment of the invention the method further comprises an amplification cascade wherein a product of the third MNAzyme is the fourth MNAzyme substrate, and wherein the second MNAzyme facilitates the ligation of the second and the fourth MNAzyme substrates thereby producing further third assembly facilitator for the third MNAzyme.<br>
[ 0085 ] According to another aspect of the present invention, there is provided a method for the detection of a first assembly facilitator using a cascade comprising components for a first MNAzyme with cleavage activity, a first MNAzyme substrate, a first DNAzyme with ligase activity (DNAzyme ligase), a first DNAzyme ligase substrate, a second assembly facilitator, at least a second MNAzyme substrate, and a first partzyme for a second MNAzyme wherein<br><br>
association of said first assembly facilitator with said components for a first MNAzyme under conditions permitting catalytic activity of said first MNAzyme facilitates the catalytic activity of said first MNAzyme thereby providing modification of said first MNAzyme substrate wherein;<br>
the modification of the first MNAzyme substrate results in the production of the second DNAzyme ligase substrate; and<br>
wherein the first DNAzyme ligase assembles with the first and second DNAzyme ligase substrates; and<br>
wherein catalytic activity of the DNAzyme ligase produces a second partzyme for the second MNAzyme, and wherein;<br>
association of said second partzyme for the second MNAzyme with said second assembly facilitator, and the first partzyme for the second MNAzyme under conditions permitting catalytic activity of the second MNAzyme facilitates the catalytic activity of said second MNAzyme thereby providing modification of the at least second MNAzyme substrate.<br>
[0086] At least one of said oligonucleotide components, assembly facilitator or substrates may comprise DNA or an analogue thereof<br>
[ 0087 ] The first assembly facilitator may be a target to be identified, detected or quantitated.<br>
[ 0088 ] The modification of the first or second MNAzyme substrate or ligase substrates or any combination thereof may provide a detectable effect.<br>
[ 0089 ] The catalytic actiAaty of the second MNAzyme may be selected from the group comprising cleavage of nucleic acids, ligation of nucleic acids, phosphorylation of nucleic acids, nucleic acid capping, amino acid adenylation, cofactor synthesis, RNA polymerization, template-directed polymerization, RNA-protein conjugation, aldol reaction, alcohol oxidation, aldehyde reduction, purine and pyrimidine nucleotide synthesis, alkylation, amide synthesis, urea synthesis, formation of peptide bonds, peptidyl-RNA synthesis, acyl transfer, aminoacylation, carbonate hydrolysis, phosphorothioate alkylation,  porphyrin metallation, formation of carbon-carbon bonds.<br><br>
Pd nanoparticle formation, biphenyl isoraerization, formation of ester bonds, formation of amide bonds, DNA deglycosylation, thymine dimer photoreversion and phosphoramidate cleavage.<br>
[ 0090 ]   The modification of the second MNAzyme substrate may be cleavage.<br>
[ 0091 ]   The ligation of the substrates may provide a detectable effect.<br>
[ 0092 ] In one embodiment of the invention the method further comprises an amplification cascade vi'herein a product of the second MNAzyme is the second DNAzyme ligase substrate, and wherein the DNAzyme ligase facilitates the ligation of the first and second DNAzyme ligase substrates thereby producng a further second partzyme for the second MNAzyme.<br>
[ 0093 ] The DNAzyme ligase may be selected from the group comprising DNAzyme ligases 7Z81, 7Z48 and 7Q10.<br>
[ 0094 ] According to another aspect of the present invention, there is provided a method for the detection of a first assembly facilitator using a cascade comprising components for a first MNAzyme with cleavage activity, a first MNAzyme substrate, a first DNAzyme with ligase activity (DNAzyme ligase), a first DNAzyme ligase substrate, components for a second MNAzyme and at least a second MNAzyme substrate, and wherein<br>
association of said first assembly facilitator with said components for a first MNAzyme under conditions permitting catalytic activity of said first MNAzyme facilitates the catalytic activity of said first MNAzyme thereby providing modification of said first MNAzyme substrate wherein;<br>
the modification of the first MNAzyme substrate results in the production of a second DNAzyme ligase substrate; and<br>
wherein the first DNAzyme ligase assembles with the first and second DNAzyme ligase substrates; and<br>
wherein catalytic activity of the DNAzyme ligase produces a second assembly facilitator for the second MNAzyme, and wherein;<br><br>
association of said second assembly facilitator for the second MNAzyme with said components for a second NfNAzyme under conditions permitting catalytic activity of the second MNAzyme facilitates the catalytic activity of said second MNAzyme thereby providing modification of the at least second MNAzyme substrate.<br>
[0095] At least one of said oligonucleotide components, assembly facilitator or substrates may comprise DNA or an analogue thereof<br>
[ 0096 ] The modification of the first or second MNAzyme substrate or ligase substrates or any combination thereof may provide a detectable effect<br>
[ 0097 ] The first assembly facilitator may be a target to be identified, detected or quantitated.<br>
[ 0098 ] The catalytic activity of the second MNAzyme may be selected fix)m the group comprising cleavage of nucleic acids, ligation of nucleic acids, phosphorylation of nucleic acids, nucleic acid capping, amino acid adenylation, cofactor synthesis, RNA polymerization, template-directed polymerization, RNA-protein conjugation, aldol reaction, alcohol oxidation, aldehyde reduction, purine and pyrimidine nucleotide synthesis, alkylation, amide synthesis, urea synthesis, formation of peptide bonds, peptidyl-RNA synthesis, acyl transfer, aminoacylation, carbonate hydrolysis, phosphorothioate alkylation, porphyrin metallation, formation of carbon-carbon bonds, Pd nanoparticle formation, biphenyl isomerization, formation of ester bonds, formation of amide bonds, DNA deglycosylation, thymine dimer photoreversion and phosphoramidate cleavage.<br>
[ 0099 ] The DNAzyme ligase may be selected from the group comprising DNAzyme ligases 7Z8I, 7Z48 and 7Q10.<br>
[ 00100 ]   The modification of the second MNAzyme substrate may be cleavage.<br>
[ 00101 ]   The ligation of the substrates may provide a detectable effect.<br>
[ 00102 ] In one embodiment of the invention the method further comprises an amplification cascade wherein a product of the second MNAzyme is the second DNAzyme ligase substrate, and wherein the DNAzyme ligase facilitates the ligation of<br><br>
the first and second DNAzyme ligase substrates thereby producing further second assembly facilitator for the second MNAzyme.<br>
[ 00103 ] According to another aspect of the present invention, there is provided a method for the detection of a first assembly facilitator using a cascade comprising components for a first MNAzyme with cleavage activity (a first MNAzyme cleaver), a first cleavable MNAzyme substrate, components for a first MNAzyme with ligase activity (a first MNAzyme ligase), a first MNAzyme ligase assembly facilitator, a first MNAzyme ligase substrate, a second assembly facilitator for a second MNAzyme cleaver, a second cleavable MNAzyme substrate, and a first partzyme for a second MNAzyme cleaver wherein<br>
association of said first assembly facilitator with seiid components for a first MNAzyme cleaver under conditions permitting catalytic activity of said first MNAzyme cleaver facilitates the catalytic activity of said first MNAzyme cleaver thereby providing modification of said first cleavable MNAzyme substrate wherein;<br>
the modification of the first cleavable MNAzyme substrate results in the production of a second MNAzyme ligase substrate; and<br>
wherein the MNAzyme ligase assembly facilitator facilitates assembly of the first MNAzyme ligase with the fust and second MNAzyme ligase substrates; and<br>
wherein catalytic activity of the MNAzyme ligase produces a second partzyme for the second MNAzyme cleaver, and wherein;<br>
association of said first and second partzymes for the second MNAzyme cleaver with said second assembly facilitator, under conditions permitting catalytic activity of the second MNAzyme cleaver facilitates the catalytic activity of said second MNAzyme cleaver thereby providing modification of the second cleavable MNAzyme substrate.<br>
[00104] At least one of said oligonucleotide components, assembly facilitators or substrates may comprise DNA or an analogue thereof<br>
[ 00105 ] The first assembly facilitator may be a target to be identified, detected or quantitated.<br><br>
[ 00106 ] The modification of the first or second cleavable MNAzyme substrate or both may provide a detectable effect.<br>
[ 00107 ]   The ligation of the substrates may provide a detectable effect.<br>
[ 00108 ] In one embodiment the method further comprises an amplification cascade wherein the product of the second MNAzyme is the second MNAzyme ligase substrate, and wherein the MNAzyme ligase facilitates the ligation of the first and second MNAzyme ligase substrates thereby producing a further second partzyme for the second MNAzyme cleaver.<br>
[ 00109 ] According to another aspect of the present invention, there is provided a method for the detection of a first assembly facilitator using a cascade comprising components for a first MNAzyme with cleavage activity (a first MNAzyme cleaver), a first cleavable MNAzyme substrate, components for a first MNAzyme with ligase activity (a first MNAzyme ligase), a first MNAzyme ligase assembly facilitator, a first MNAzyme ligase substrate, components for a second MNAzyme cleaver and a second cleavable MNAzyme substrate, wherein<br>
association of said first assembly facilitator with said components for a first MNAzyme cleaver under conditions permitting catalytic activity of said first MNAzyme cleaver facilitates the catalytic activity of said first MNAzyme cleaver thereby providing modification of said first cleavable MNAzyme substrate wherein;<br>
the modification of the first cleavable MNAzyme substrate results in the production of the second MNAzyme ligase substrate; and<br>
wherein the MNAzyme ligase assembly facilitator facilitates assembly of the first MNAzyme ligase with the first and second MNAzyme ligase substrates; and<br>
wherein catalytic activity of the MNAzyme ligase produces a second MNAzyme assembly facilitator for the second MNAzyme cleaver, and wherein;<br>
association of said second MNAzyme assembly facilitator with said components for a second MNAzyme cleaver under conditions permitting catalytic activity of the second MNAzyme cleaver facilitates the catalytic activity of said second<br><br>
MNAzyme cleaver thereby providing modification of the second cleavable MNAzyme substrate.<br>
[00110]   At least one of said oligonucleotide components, assembly facilitators or substrates may comprise DNA or an analogue thereof<br>
[ 00111 ]   The modification of the first or second cleavable MNAzyme substrate or both may provide a detectable effect.<br>
[00112]   The ligation of the substrates may provide a detectable effect.<br>
[00113 ]   The first assembly facilitator may be a target to be identified, detected or quantitated.<br>
[ 00114 ]   In one embodiment the method further comprises an amplification cascade wherein the product of the second MNAayme is the second MNAzyme ligase substrate, and wherein the MNAzyme ligase facilitates the ligation of the first and second MNAzyme ligase substrates thereby producing a further second MNAzyme assembly facilitator for the second MNAzyme cleaver.<br>
[ 0011S ]   In another aspect of the invention there is provided a use of at least one oligonucleotide as a multi-component nucleic acid enzyme (MNAzyme) substrate wherein said substrate is modified by said MNAzyme following the engagement of said substrate by the substrate arms of said MNAzyme.<br>
[ 00116 ]    In another aspect of the invention there is provided a method for making MNAzymes which incorporate partial seqyences from a DNAzyme wherein said method comprises the following steps:<br>
a)	selecting positions within said DNAzyme sequence for splitting into partial catalytic core sequences<br>
b)	incorporating said partial catalytic core sequences into partzymes<br>
c)	detecting catalytic activity of the partzyme pairs to determine which combination of said partial catalytic core sequences in said partzymes is compatible with the formation of active MNAzymes.<br>
[00117 ]    In another aspect of the invention there is provided a method for testing partial catalytic core sequences derived from a DNAzyme catalytic core which, upon<br><br>
incorporation into partzymes, generate functionally active MNAzymes, wherein said method comprises<br>
a)	splitting the catalytic core sequence of said DNAisyme<br>
b)	incorporating the resulting partial catalytic core sequences into paitzymes:<br>
c)	detecting MNAzyrae activity to determine which combination of said partial catalytic core sequences in said partzymes is compatible with the formation of active MNAzymes.<br>
[ 00118 ]   In another aspect of the invention there is provided a method for identifying which positions within a DNAzyme catalytic core sequence are suitable for splitting into partial catalytic core sequences which, upon incorporation into partzymes, result in functionally active MNAzymes wherein said method comprises the following steps:<br>
a)	splitting the catalytic core sequence from said DNAzyme<br>
b)	incorporating the resulting partial catalytic core sequences into partzymes:<br>
c)	detecting MNAzyme activity to determine which combination of said partial catalytic core sequences in said partzymes is compatible with the formation of active MNAzymes.<br>
BRIEF DESCRIPTION OF THE DRAWINGS<br>
[ 00119 ] Preferred embodiments of the present invention will now be described, by way of example only, with reference to the accompanying drawings wherein:<br>
[00120] Figure 1: Depiction of one exemplary design of a Multi-component nucleic acid (MNAzyme): By way of exemplary disclosure, an MNAzyme is comprised of two oligonucleotide components (partzyme A and partzyme B), which self assemble in the presence of an assembly facilitator. When the two partzymes assemble in the presence of the assembly facilitator, a catalytically active MNAzyme forms which is capable of modifying, for example cleaving, a substrate. The two component paitzymes have (i) sensor arms, which bind to the assembly facilitator, (ii) substrate arms, which bind the substrate, and (iii) partial catalytic core sequences. The presence of an assembly facilitator molecule provides the "input" signal which directs the assembly of partzyme components in a highly specific fashion which is amenable to modulation. In some embodiments, the assembly facilitator may be, for example, a target nucleic acid<br><br>
sequence present in a test sample. In other embodiments, the assembly facilitator may be, for example a synthetic oligonucleotide included in the milieu to direct the self-assembly of the partzyme components in the presence of a detectable entity or event. Modification of the substrate by the assembled MNAzyme can provide an "output" signal which may be detected and/or quantified. For example, when the substrate is dual labelled with a fluorophore (F) and a quencher (Q), cleavage of the substrate by an active NfNAzyme separates the fluorophore and the quencher resulting in a concomitant increase in fluorescence.<br>
[00121 ] Figure 2: Additional exemplary designs for active MNAzymes. Panel (i): Depiction of an exemplary design for MNAzymes where multiple assembly facilitator components are required for MNAzyme formation. In this design, one assembly facilitator component (Fl) is complementary to regions of the sensor arms of both partzyme A and B, whereas a second assembly facilitator component (F2) is complementary to either partzyme B only (as per this illustration), or partzyme A only. The two assembly facilitator components together direct the assembly of an active MNAzyme which can modify (eg cleave) a substrate. Panel (ii): Depiction of an exemplary design where one partzyme A component and bi-partite partzyme B components assemble in the presence of assembly facilitator to produce an active MNAzyme capable of modifying (eg cleaving) a substrate. In this diagram, partzyme B has a truncated sensor arm (T), which is insufficient to allow stable MNAzyme assembly in the absence of a second component, referred to as a stabiliser arm component (S). Hybridization of the stabiliser arm to the assembly facilitator in a location adjacent to the truncated sensor arm of the partzyme, allows assembly of an active MNAzyme.<br>
[ 00122 ] It would be appreciated by one skilled in the art that the partzyme arm, which is truncated, could be any of the following; the partzyme A sensor arm, the partzyme B sensor arm (as illustrated), the partzyme A substrate arm or the partzyme B substrate arm, or a combination thereof<br>
[00123] Figure 3: Changes in output signal (fluorescence) over time in the presence of active and inactive multi-component nucleic acid complexes. In this example, the first partzyme (A) comprises a single component oligonucleotide.   The<br><br>
second partzyme (B) has one component containing a substrate arm, a partial catalytic core and a truncated sensor arm (T), and a second component which serves as a stabiliser arm (S).<br>
[ 00124 ] An increase in fluorescence was observed in reaction (i), which contained all the components of partzymes A and B and an assembly facilitator. This is consistent with the assembly of active MNAzymes and cleavage of the substrate in this reaction. The omission of the stabilizer arm portion of partzyme B (reaction (ii)) resulted in no increase in signal over time indicating that this component is essential for the assembly of active MNAzymes in this system. A control reaction lacking an assembly facilitator (reaction (iii)) also showed no increase in fluorescence over time.<br>
[00125} These reactions represent two alternate states for the oligonucleotide complexes. The active MNAzyme (reaction (i)) represents the active complex ("on" state). Those reactions where either a partzyme stabiliser arm component (reaction (ii)), or an assembly facilitator component (reaction (iii)) is omitted, are representative of inactive MNA complexes ("ofP' states). As such, the MNAzyme catalytic activity can be regulated by the presence or absence of various oligonucleotides and/or by the ability of such oligonucleotide components to be functionally active, for example to be capable of hybridizing with other oligonucleotide components to form stable MNAzymes or MNAzyme complexes. The truncated arm is designed to be insufficient to allow stable MNAzyme assembly under the reaction conditions, unless accompanied by a stabiliser arm component. The stabiliser arm, and the assembly facilitator, can function as "on" switches for MNAzyme activity.<br>
[00126] Figure 4: Depiction of an exemplary design for an inactive multi-component nucleic acid inhibitor (MNAI) complex O^ft band side) and an active MNAzyme (right hand side): An MNAi complex is formed when partzymes A and B complex with an assembly facilitator component and an activity inhibitor (left hand side). The inactive MNAi complex is capable of interacting with, but not catalytically modifying, the substrate. In some embodiments, the activity inhibitor may further include a labile or cleavable linker (indicated by the dotted arrow), which may separate two or more domains within the activity inhibitor. Such domains may include, for<br><br>
example, (i) an activity inhibitor portion which is substantially non-complementary to the partzyme components and which exerts an inhibitory effect by disrupting the secondary structure required for formation of a catalytically active MNAzyme and (ii) an activator assembly facilitator portion, which if separated from the activity inhibitor portion, may function as an assembly facilitator component and direct the assembly of an active MNAzyme.<br>
[00127] Figure 5: Demonstration of catalytic activity from various multi-component nucleic acid complexes. All reactions contained partzyme A, partzyme B, and a substrate labelled with a fluorophore quencher dye pair. In addition, reactions contained either (i) an assembly facilitator Fl/2, (ii) an assembly facilitator comprising portions Fl and F2 (the sequence of which together correspond to that of assembly facilitator FI/2), (iii) assembly facilitator portion Fl and an activity inhibitor containing two joined domains corresponding to an activity inhibitor portion and portion with the same sequence as F2 or (iv) no assembly facilitator or activity inhibitor.<br>
[ 00128 ] The change in fluorescence was monitored over time as a measure of catalytic cleavage of the substrate by an active MNAzyme (Figure 5). The fluorescence increased rapidly in reactions containing either assembly facilitator Fl/2 or assembly facilitator components Fl and F2, indicating the formation of active MNAzymes 1 and 2 respectively, both of which are ctqiable of cleaving the substrate. In contrast, the reaction containing Fl and the activity inhibitor showed no increase in fluorescence over time indicating the formation of inactive MNAi complexes. No increase in fluorescence was seen in the absence of assembly facilitator.<br>
[00129] Figure 6. Exemplary structures for variant MNAzyme designs. Some examples of active MNAzyme complexes are shown in Panels A to D. These structures are all capable of forming catalytically active enzymes, which can modify, for example cleave, a substrate (S). Illustrated are schemes for MNAzyme complexes, which include one assembly facilitator Fl (panel A), two assembly facilitator components Fl and F2 (panels B and D) or three assembly facilitator components Fl, F2 and F3 (panel C). Panel A includes sensor arms with self complementary regions within the partzyme sensor<br><br>
arms. MNAzyme complexes may also include one or more stabilising arms (sA) as shown in panel D.<br>
[ 00130 ] Figure 7: An exemplary strategy for the modulation of the activity of MNAzymes.The strategy in this system can be used cither as (i) a method to control MNAzyme activity using ligands as activator molecules, and/or (ii) a method of detection of nucleic and non-nucleic acid targets using apta-MNAzymes.<br>
[00131] The nucleic acid oligonucleotides included in this exemplary apta-MNAzyme detection strategy include;<br>
a)	a partzyme;<br>
b)	an apta-partzyme which is a partzyme with an aptamer incorporated into one of its ends;<br>
c)	an assembly facilitator which is an oligonucleotide which binds to both the apta-partzyme and the partzyme enabling assembly of an active MNAzyme;<br>
d)	a substrate e.g. a reporter substrate; and<br>
e)	an assembly inhibitor oligonucleotide which hybridises to the apta-partzyme in a region which spans at least part of the aptamer sequence and part of the substrate binding arm of the apta-partzyme sequence.<br>
[ 00132 ] In the absence of an activator ligand (left hand panel), the assembly inhibitor oligonucleotide binds to the apta-partzyme thus competing with and blocking binding of the reporter substrate. When an activator ligand is present (right hand panel), it binds to the aptamer sequence of the apta-partzyme, blocking the binding of the assembly inhibitor oligonucleotide, and thus allowing assembly of an active apta-MNAzyme complex and subsequent modification, for example cleavage of the substrate (as illustrated) or for example ligation of substrates. As such, an active apta-MNAzyme can only form and cause an increase or decrease in signal, for example, a fluorescent signal, in the presence of a ligand that can bind the aptamer portion of the apta-partzyme.<br>
[ 00133 ] It would be appreciated that an aptamer sequence could be attached to, for example, one or more of either a substrate arm and/or a sensor arm of either or both of the partzymes. If the aptamer were attached to a sensor arm then the binding of the<br><br>
assembly facilitator could be blocked by an assembly inhibitor oligonucleotide in the absence of a target iigand.<br>
[ 00134 ] When the apta-MNAzyme has cleavage activity as drawn in this figure then the modification can cause for example an increase in fluorescence by cleavage of a dual labelled reporter substrate thus causing separation of a fluorophore/quencherdye pair.<br>
[ 00135 ] It would be appreciated that the modification performed by an apta-MNAzyme could be a modification other than cleavage, for example, ligation of two substrates. In this case the signal change could be, for example, a decrease in fluorescence following ligation of two ligatable substrates each labelled with either the fluorophore or the quencher of a fluorophore/quencher dye pair.<br>
[00136] This apta-NfNAzyme approach could also be used to develop molecular switches that can turn on and off the catalytic activity of an MNA complex. Alternatively it can also be applied to detection of both nucleic acid and non-nucleic acid target ligands.<br>
[00137] Figure 8: An example of a cleavage/llgation cascade mediated by a DNAzyme with ligase activity and an MNAzyme with cleavage activity:<br>
[ 00138 ] A DNAzyme with ligase activity can ligate a first oligonucleotide (oligo 1) to a second oligonucleotide (oligo 2) to create a third ligation product (LP-Oligo 3) with the sequence of oligo 3 and a 2'-5' RNA linkage at the ligation junction, provided oligo 1 and oligo 2 have 2', 3'-cyclic phosphate and 5'-hydroxyl termini, for example, as shown. In turn, LP-oligo 3 could be cleaved by an MNAzyme capable of cleaving at a 2'-5' RNA linkage to create cleavage products (CP), CP-oligo 1 and CP-oligo 2, thus regenerating 2', 3'-cyclic phosphate and 5'-hydroxyl products, which can participate in fiirther rounds of ligation.<br>
[ 00139 ] Other DNAzymes with ligase activity may ligate a first oligonucleotide (oligo 1) to a second oligonucleotide (oligo 2) to create a third ligation product (LP-Oligo 3) with the sequence of oligo 3 and a 3'-S' RNA linkage at the ligation junction, provided oligo 1 and oligo 2 have 2',3'-diol and 5'-triphosphate termini, for example. In turn, LP-oligo 3 can be cleaved by an MNAzyme capable of cleaving at a 3*-5' RNA linkage to<br><br>
create cleavage products (CP), CP-oligo 1 and CP-oligo 2 producing products with a 2', 3'-cyclic phosphate and 5'-hydroxyI.<br>
[00140] Figure 9: An example of a cascade reaction which uses two types of nucleic acid enzymes namely MNAzymes, which can cleave substrates, and DNAzyrae ligases which can ligate substrates. In step (i) the MNAzyme A forms only in the presence of an assembly facilitator (e.g. a target nucleic acid) and then cleaves NfNAzyme substrate A releasing a 5' product A with a 2', 3' cyclic phosphate terminus. In step (ii) die DNAzyme B with ligase activity ligates the 5' product A from step (i) to a 3' ligation substrate B thus creating a ligation product which can serve as a partzyme component of another MNAzyme C. In step (iii) the partzyme/ligation product of step (ii) assembles with a second partzyme and an assembly facilitator C to form MNAzyme C which can cleave substrate C into two products, 5' product C and 3' product C.<br>
[ 00141 ] In step (i) an assembly facilitator, for example a target nucleic acid, can direct the formation of a first MNAzjmie A. MNAzyme A can cleave a substrate A thus generating a 5'cleavage product A which can in tiim be used as a substrate for a DNAzyme ligase B. In step (ii) a DNAzyrae ligase B can ligate the 5' cleavage product A (also referred to as S' substrate B) generated in step (i) to another oligonucleotide, 3' ligation substrate B, thus creating a new partzyme for an MNAzyme C. In step (iii) the new partzymeAigation product generated in step (ii), together with another partzyme, can form a new MNAzyme C in the presence of assembly facilitator C which can cleave a substrate C into two products, 5'product C and 3' product C. A detectable signal can be generated following cleavage of substrate C if this substrate were labelled, for example, with a fluorophore and quencher dye pair.<br>
[ 00142 ] Further, if the sequence of 5' product C were the same as 5' product A then this product can also serve as a 5' substrate B and a feedback amplification cascade reaction could be initiated. In this reaction MNAzyme C could constantly generate 5' substrate B which in turn could be ligated by DNAzyme ligase B to create more partzymes for formation of more MNAzyme C. This strategy provides a feedback: cascade or mechanism for signal amplification following initiation of a reaction by a target analyte which allows assembly of MNAzyme A.   The cascade strategy allows<br><br>
detection of target analytes followed by signal amplification using a DNAzyme which can ligate substrates and MNAzymes which can cleave a substrate.<br>
[ 00143 ] Figure 10: Example of a structure of aa MNAzyme with ligase activity (an MNAzyme ligase). The exemplary MNAzyme ligase is comprised of partzyme sequences which contain three domains including a (i) a sensor arm which hybridizes to an assembly facilitator (e.g. a target nucleic acid), (ii) a domain that constitutes part of a catalytic core and (iii) a substrate binding arm (Figure lOA). In the presence of assembly facilitator, for example a target nucleic acid, partzymes assemble and form active MNAzyme ligases. These MNAzymes can ligate a 5' substrate with a 2'3' cyclic phosphate and a 3* substrate with 5' hydroxyl (Figure lOB) to create ligation products that contain non-native 2'-5' linkages.<br>
[ 00144 ] Alternatively, MNAzymes could ligate two substrates with a 2'3' diol terminus on the 5' substrate with a 5'-triphosphate tenninus on the 3' substrate to create ligation products that contain native 3'-S' linkage.<br>
[ 0014S ] Figure 11: An exemplary strategy for reactions that use MNAzymes with ligase activity and MNAzymes with cleavage activity.<br>
[ 00146 ] An MNAzyme with ligase activity could ligate a first oligonucleotide (A) to a second oligonucleotide (B) to create a third ligation product (C) with a 2'-5' RNA linkage at the ligation junction, provided A and B have 2', 3'-cyclic phosphate and 5*-hydroxyl termini. In turn, product C could be cleaved by an MNAzyme capable of cleaving at a 2'-5' RNA linkage to create cleavage products A and B, thus regenerating 2', 3'cyclic phosphate and 5'-hydroxyl products, which can participate in further rounds of ligation.<br>
[ 00147 ] Other MNAzymes with ligase activity could ligate a first oligonucleotide (A) to a second oligonucleotide (B) to create a third ligation product (C) with a 3'-S' RNA linkage at the ligation junction, provided A and B have 2', 3*-diol and 5'-triphosphate termini. In turn, product C could be cleaved by an MNAzyme capable of cleaving at a 3'-5' RNA linkage to create cleavage products A and B, producing products with 2', 3'-cyclic phosphate and 5'-hydroxyl termini.<br><br>
[ 00148 ] Figure 12. A strategy for signal detection whicli uses MNAzymes wliicli cleave and MNAzymes wliich ligate. In step (i) an assembly facilitator, for example a target nucleic acid, can direct the formation of a first MNAzyme A. MNAzyme A can cleave a substrate A thus generating a 5' cleavage product A which can in turn be used as a substrate for an MNAzyme B which has ligase activity. In step (ii) an MNAzyme ligase B can ligate the 5 ■ cleavage product A (also called the 5' substrate B) generated in step (i) to another oligonucleotide, 3* substrate B, thus creating a new partzyme for an MNAzyme C. In step (iii) the new partzyme/ligation product generated in step (ii), together with another partzyme, can form a new MNAzyme C in the presence of assembly facilitator C which can cleave a substrate C into two products, S' product C and 3' product C. A detectable signal can be generated following cleavage of substrate C if this substrate were labelled, for example, with a fluorophore and quencher dye pair. The strategy allows detection of target analytes using two types of MNAzymes, namely MNAzymes which can ligate substrates (MNAzyme ligases) and MNAzymes which can cleave a substrate (MNAzyme cleavers).<br>
[ 00149 ] Further if the sequence of 5' product C were the same as 5' product A then this product could also serve as a S' substrate B and a feedback cascade reaction could be initiated. In this reaction, MNAzyme C could be constantly generating 5' substrate B which could in turn be ligated by MNAzyme ligase B to create more partzymes for formation of more MNAzyme C. This strategy could provide a mechanism for feedback signal amplification following initiation of a reaction by a target analyte which allows assembly of MNAzyme A.<br>
[001 SO] Figure 13: Depiction of an exemplary strategy for switching an MNA complex between the "on state" of an active MNAzyme to the **ofr state" of an MNAi complex using a second MNAzyme with ligase activity. An active MNAzyme A, which could be capable of modifying (e.g. cleaving or ligating) a substrate(s) A could be formed in the presence of assembly facilitator component 1 (AFC 1) and assembly facilitator component 2 (AFC 2). A second MNAzyme B, which has ligase activity, could form in the presence of assembly facilitator 3 (AF3) and then could ligate AFC 2 with an activity inhibitor comjjonent (AIC) resulting in the formation of an activity inhibitor (AI). This AI could bind to the partzyme components for the MNAzyme A resulting in the<br><br>
formation of an inactive MNAi complex. As such the MNAzyme ligase in this example can operate as an off switch to inactivate an MNAzyme A. The inactive MNAi complex and the catalytically active MNAzyme represent two alternate states for the assembled components, namely an "off" state and the "on" state respectively.<br>
[001 SI ] Figure 14: Depiction of a general method for engineering MNAzymes from uni-molecular DNAzymes.<br>
[ 00152 ] Many DNAzymes have similar basic structures with multiple domains. These DNAzymes have a conserved catalytic domain (catalytic core) flanked by two non-conserved substrate-binding domains ("substrate arms"), which specifically recognize and hybridise to the substrate (top left hand side structure). The substrate binding domains can be tailored to any substrate provided the substrate contains a site which can be modified by the DNAzyme.<br>
[ 00153 ] In the first step, positions are identified within the DNAzyme catalytic core at which it can be split, such that each partial portion of the catalytic core can be distributed between two partial sequences such that the two partial cores together constitute an entire catalytic core (top right hand side structure). Two oligonucleotides A and B (candidate partzymes) can then be synthesised (bottom left hand side structure). An oligonucleotide A can be synthesised to contain (i) one substrate binding arm portion capable of binding to a substrate, (ii) one partial catalytic core portion, and (iii) one sensor arm portion capable of binding to an assembly facilitator molecule. A second oligonucleotide B can be synthesised such that it contains (i) one substrate binding arm capable of binding to the same substrate as oligonucleotide A, whereby oligonucleotide B binds the substrate in a position adjacent to that of oligonucleotide A, (ii) one partial catalytic core portion which contains those bases from the entire DNAzyme catalytic core w^ch are not incorporated into oligonucleotide A and (iii) one sensor arm sequence capable of binding to the same assembly facilitator as oligonucleotide A, whereby oligonucleotide B binds the assembly facilitator in a position adjacent to that of oligonucleotide A (bottom left hand side structure).<br>
[ 00154 ] This process can be repeated thus making a series of pairs of oligonucleotides A and B which incorporate the structure and domains of partzymes, but may or may not<br><br>
have catalytic activity in the presence of a substrate and an assembly facilitator. It would be appreciated by one skilled in the art that a similar process can be undertaken for a nucleic acid enzyme which can act on at least two substrates. Candidate partzyme pairs (pairs of oligonucleotides A and B from the series) can be mixed with substrate(s) matching the substrate arms plus assembly facilitator(s) matching the sensor arms (bottom right hand side structure) and then assayed for the presence of the same type of modifying activity as exhibited by the DNAzyme (top left hand side). Pairs of oligonucleotides A and B which can catalyse the same type of modification of the DNAzyme are useful for the assembly of active MNAzymes.<br>
[ 00155 ] Figure 15: A cascade reaction using nucleic acid enzymes which cleave substrates, and enzymes which ligate substrates to form a new assembly facilitator.<br>
In this illustration the 5' end of the oligonucleotides are indicated by a circle. In step (1) the MNAzyme cleaver assembles in the presence of an assembly facilitator (e.g. a target nucleic acid) and then cleaves MNAzyme substrate 1 releasing a 5' cleavage product with a 2', 3' cyclic phosphate terminus. In step (2) the DNAzyme ligase (or MNAzyme ligase) ligates the 5' cleavage product from step (1) to a 3' ligation substrate thus creating a ligation product which can serve as an assembly facilitator for another MNAzyme cleaver 2. In step (3) the assembly facilitator formed by ligation in step (2) directs the assembly of partzymes which form MNAzyme cleaver 2 which cleaves substrate 2 into two products, a 5' cleavage product with a 2', 3' cyclic phosphate terminus and a 3' cleavage product. A detectable signal can be generated following cleavage of substrate 2 when this substrate is labelled, for example, with a fluorophore and quencher dye pair. The strategy allows detection of target analytes using multiple types of nucleic acid enzymes, namely enzymes which can ligate substrates and enzymes which can cleave a substrate.<br>
[ 00156 ] Further if the sequence of the 5' product of MNAzyme cleaver 2 were the same as the 5' product of MNAzyme cleaver 1 then this product could also serve as a substrate for the ligase and a feedback cascade reaction could be initiated. In this reaction, MNAzyme cleaver 2 could be constantly generating 5' substrate for the ligase which could in turn be ligated by the ligase to create more assembly facilitators for formation of more MNAzyme cleaver 2. This strategy could provide a mechanism for<br><br>
feedback signal amplification following initiation of a reaction by an assembly facilitator such as a target nucleic acid which allows assembly of MNAzyme cleaver 1.<br>
[ 00157 ] Figure 16: A possible schema for a feedbacic cascade reaction which uses two types of nucleic acid enzymes namely enzymes which can cleave substrates, and enzymes which can ligate substrates. In this illustration the S' end of oligonucleotides are indicated by a circle. In step (I) the MNAzyme cleaver 1 could assemble only in the presence of an assembly facilitator (e.g. a target nucleic acid) and could then cleave MNAzyme substrate 1 releasing a 5' cleavage product with a 2', 3' cyclic phosphate terminus and a 3' waste product. In step (2) the DNAzyme ligase (or MNAzyme ligase) could ligate the 5' cleavage product from step (1) to a 3' ligation substrate thus creating a ligation product which could serve as an assembly facilitator for an MNAzyme cleaver 2. In step (3) the assembly facilitator formed by ligation in step (2) could direct the assembly of partzymes to form MNAzyme cleaver 2 which could cleave substrate 2 into two products, a 5' cleavage product with a 2', 3' cyclic phosphate terminus and 3' cleavage product. A detectable signal could be generated following cleavage of substrate 2 if this substrate were labelled, for example, with a fluorophore and quencher dye pair.<br>
[ 00158 ] Further, if the sequence of either of the cleavage products of MNAzyme cleaver 2 was useful as an assembly facilitator component for an MNAzyme cleaver 3 then this could initiate a feedback cascade. In this case MNAzyme cleaver 3 could have different sensor arms than MNAzyme cleaver 1 but could have the same substrate arms as MNAzyme cleaver 1. Thus if MNAzyme cleaver 3 were assembled in the presence of an assembly facilitator component generated by cleavage by the MNAzyme cleaver 2, and if MNAzyme cleaver 3 cleaved substrate 1 then the 5* cleavage product of MNAzyme cleaver 3 could also serve as a 5' substrate for the DNAzyme ligase (or the assembled MNAzyme ligase) and a feedback amplification cascade reaction could be initiated.<br>
[00159] In this feedback cascade reaction MNAzyme cleaver 3 could constantly generate 5' cleavage product which in turn could serve as a substrate for ligation by the DNAzyme ligase (or the assembled MNAzyme ligase) to create more assembly facilitators that could direct the assembly of more MNAzyme cleaver 2.   This strategy<br><br>
could provide a mechanism for feedback signal amplification following initiation of a reaction by an assembly facilitator (eg a target analyte) which allowed assembly of MNAzyme cleaver 1. The strategy could allow detection of one or more assembly facilitators (eg target analytes) followed by signal amplification using a DNAzyme or an MNAzyme which can ligate substrates and MNAzymes which can cleave a substrate.<br>
DEFINITIONS<br>
[00160] Certain terms are used herein which shall have the meanings set forth as follows.<br>
[00161 ] The term "comprising" means "including principally, but not necessarily solely". Furthermore, variations of the word "comprising", such as "comprise" and "comprises", have correspondingly varied meanings.<br>
[ 00162 ] The terms "polynucleotide", "nucleic acid" and "oligonucleotide" may be used interchangeably and refer to a single- or double-stranded polymer of deoxyribonucleotide or ribonucleotide bases, or analogues, derivatives, variants, fragments or combinations thereof, including but not limited to DNA, methylated DNA, alkylated DNA, RNA. methylated RNA, microRNA, siRNA, shRNA, mRNA, tRNA, snoRNA, stRNA, smRNA, pre- and pri-microRNA, other non-coding RNAs, ribosomal RNA, derivatives thereof, amplicons thereof or any combination thereof. By way of non-limiting example, the source of a nucleic acid may be selected from the group comprising synthetic, mammalian, human, animal, plant, ftingal, bacterial, viral, archael sources or any combination thereof.<br>
[ 00163 ] The term "oligonucleotide" typically denotes a segment of DNA or a DNA-containing nucleic acid molecule, or RNA or RNA-containing molecule, or a combination thereof. Examples of oligonucleotides include nucleic acid targets; substrates, for example, those which can be modified by a DNAzyme or an NflNAzyme with cleavage, ligase or other enzymatic activity; primers such as those used for in vitro target amplification by methods such as PCR; and components of nucleic acid complexes including but not limited to assembly facilitators and assembly inhibitors, activators, activity inhibitors and/or substrates, which in certain embodiments, may comprise<br><br>
oligonucleotides as defined herein. Partzymes as used herein may also comprise oligonucleotides.<br>
[ 00164 ] A "displacer" or "displacer oligonucleotide" is an oligonucleotide which can bind to a single stranded region within a first oligonucleotide which comprises at least one region which is single stranded and at least one region which is double stranded by virtue of complementarities of at least one additional oligonucleotide; wherein the displacer oligonucleotide can cause dissociation of the additional oligonucleotide(s) from the first oligonucleotide within at least one double stranded region(s) by a process of branch migration. During branch migration the region(s) of complementarity between the additional oligonucleotide(s) and the first oligonucleotide are separated and replaced by at least one region of complementarity between the displacer oligonucleotide and the first oligonucleotide thus resulting in displacement of the additional oligonucleotide(s). The displacer oligonucleotide is required to comprise at least one region which has the same sequence as one or more regions of the additional oligonucleotide(s} within the double stranded region. The process of branch migration may result in a new double stranded region being formed between the displacer oligonucleotide and the first oligonucleotide and concomitant release of the additional oligonucleotide(s) which renders it single stranded.<br>
[00165] The terms "polynucleotide", "nucleic acid" and "oligonucleotide" include<br>
reference to any specified sequence as well as to the sequence complementary thereto,<br>
unless otherwise indicated. Oligonucleotides may comprise at least one addition or<br>
substitution, including but not limited to the group comprising LNA phosphoramidite, 4-<br>
acetylcytidine, 5-(carboxyhydroxylmethyl)uridine, 2'-0-methylcytidine, 5-<br>
carboxymethylaminomethyl thiouridine, dihydiouridine, 2'-0-methyIpseudouridine, beta<br>
D-galactosylqueosine, 2'-0-methylguanosine, inosine, N6-isopentenyladenosine, I-<br>
methyladenosine, 1-methylpseudouridine, 1-methyIguanosine, 1-methylinosine, 2,2-<br>
dimethylguanosine, 2-methyladenosine, 2-methylguanosine, 3-methylcytidine, S-<br>
methylcytidine, N6-methyladenosine, 7-methylguanosine, 5-methylaminomethyluridine,<br>
5-methoxyaminomethyl-2-thiouridine,	beta	D-mannosylmethyluridine,	5-<br>
methoxycarbonylmethyluridine,	5-methoxyuridine,	2-methylthio-N6-<br>
isopentenyladenosine,	N-((9-beta-ribofijranosyl-2-mcthylthiopurine-6-<br><br>
yl)carbamoyI)threonine,	N-((9-beta-ribofuranosylpurine-6-yl)N-methyl-<br>
carbamoyl)threonine, uridine-5-oxyacetic acid methylester, uridine-5-oxyacetic acid (v), wybutoxosine, pseudouridine, queosine, 2-thiocytidine, 5-methyl-2-thiouridine, 2-thiouridine, 4-thiouridine, 5-methyluridine, N-((9-beta-D-ribofiiranosyIpurine-6-yI)carbamoyl)threonine, 2'-0-methyl-5-methyluridine, 2'-0-methyluridine, wybutosine, 3-(3-amino-3-carboxypropyl)uridine, beta D-arabinosyl uridine, and beta D-arabinosyl thymidine.<br>
[ 00166 ] The term "derivative" when used in relation to a nucleic acid or nucleotide of the present invention includes any functionally equivalent nucleic acids or nucleotides, including any fusion molecules produced integrally (e.g., by recombinant means) or added post-synthesis (e.g., by chemical means). Such fusions may comprise oligonucleotides of the invention with RNA or DNA added thereto or conjugated to a polypeptide (e.g., puromycin or other polypeptide), a small molecule (e.g., psoralen) or an antibody.<br>
[ 00167 ] The term "analogue" when used in relation to a nucleic acid or nucleotide includes a compound having a physical structure that is related to a DNA or RNA molecule or residue, and may be capable of forming a hydrogen bond with a DNA or RNA residue or an analogue thereof (i.e., it is able to anneal with a DNA or RNA residue or an analogue thereof to form a base-pair), but such bonding is not so required for said compound to be encompassed within the term "analogue". Such analogues may possess different chemical and biological properties to the ribonucleotide or deoxyribonucleotide residue to which they are structurally related. Methylated, iodinated, brominated or biotinylated residues are examples of analogues. Active DNAzymes have been described which contain nucleotide analogues, including deoxyinosine, C-S-immidazole deoxyuridine, 3-(aminopropynyl)-7-deaza-dATP, 2'-0-methyI RNA, 2' O-methyl cap (Warashina et al., 1999; Cairns et al., 2003; Schubert et al., 2004; Sidorov et al., 2004). Other analogues could also be compatible with catalytic activity of DNAzymes and MNAzymes. Alteration of a nucleic acid with catalytic activity, for example by substitution of one base for another, by substitution of an analogue for a base, or alteration of the sugar component or phosphodiester backbone, can be straight forward for the skilled artisan.   For example, alterations can be made during synthesis, or by<br><br>
modification of specific bases after synthesis. Empirical testing of catalytic nucleic acids incorporating alterations such as base changes or base analogues allows for assessment of<br>
the impact of the	altered sequences, or specific analogues, on catalytic activity.<br>
Analogues of the bases A, C, G, T and U are known in the art, and a subset is listed in<br>
Table 1.<br>
Table 1: Examples of nucleotide analogues usejul herein<br>
Alsbreviatioa	Naaa<br>
ac4c	4-acetylcytidine<br>
chmSu	5-(carboxyhydroxylmethyl)uridine<br>
Cm	2'-0-methylcytidine<br>
Cmnin5s2u	S-carboxymethylaminoraethyl thiouridine<br>
D	Dihydrouridine<br>
Fm	2'-O-methylpseudouridine<br>
Galg	beta, D-galactosylqueosine<br>
Gm	2'-0-methylguanosine<br>
1	Inosine<br>
i6a	N6-isopentenyladenosine<br>
mla	1-methyladenosine<br>
mlf	1-methylpseudouridine<br>
mlg	1-methylguanosine<br>
Mil	1-methylinosine<br>
m22g	2,2-dimethylguanosine<br>
m2a	2-methyladenosine<br>
m2g	2-methylguanosine<br>
m3c	3-methylcytidine<br>
mSc	5-methylcytidine<br>
m6a	N6-methyladenosine<br>
m7g	7-methylguanosine<br>
mamSu	5-methylaminomethyluridine<br>
mam5s2u	5-methoxyaminomethyl-2-thiouridine<br>
Manq	beta, D-mannosylmethyluridine<br>
mcm5s2u	5-methoxycarbonylmethyluridine<br>
Mo5u	5-methoxyuridine<br>
Ms2i6a	2-methylthio-N6-isopentenyladenosine<br>
Ms2t6a	N-((9-beta-ribofuranosyl-2-<br><br>
Abbreviation	JStamm<br>
methylthi©purine-6-yl)carbamoyl)threonine<br>
Mt6a	N-({9-beta-ribofuranosylpurine-6-yl)N-<br>
methyl-carbamoyl)threonine<br>
Mv	Uridine-5-oxyacetic acid methylester<br>
o5u	Uridine-5-oxyacetic acid (v)<br>
Osyw	Wybutoxosine<br>
P	Pseudouridine<br>
Q	Queosine<br>
s2c	2-thiocytidine<br>
s2t	5-methyl-2-thiouridine<br>
s2u	2-thiouridine<br>
s4u	4-thiouridine<br>
T	5-methyluridine<br>
t6a	N-((9-beta-D-ribofuranosylpurine-6-<br>
yl)carbamoyl)threonine<br>
Tm	2'-0-methyl-5-methyluridine<br>
Urn	2•-0-methyluridine<br>
Yw	Wybutosine<br>
X	3-(3-eunino-3-carboxypropyl)uridine,   (acp3)u<br>
AraU	beta D-arabinosyluridine<br>
AraT	beta D-arzibinosylthymidine<br>
[ 00168 ] The terms "nucleic acid enzyme", "catalytic nucleic acid", "nucleic acid with catalytic activity", and "catalytic nucleic acid enzyme" are used herein interchangeably and shall mean a DNA or DNA-containing molecule or complex or an RNA or RNA-containing molecule or complex or a combination thereof, being a DNA-RNA hybrid molecule or complex, which may recognize at least one substrate and catalyse a modification (such as ligation or cleavage) of the at least one substrate. The nucleotide residues in the catalytic nucleic acids may include the bases A, C, G, T, and U, as well as derivatives and analogues thereof The terms above include uni-molecular nucleic acid enzymes which may comprise a single DNA or DNA-containing molecule (also known in the art as a "DNA enzyme", "deoxyribozyme" or "DNAzyme") or an RNA or RNA-containing molecule (also known in the art as a "RNA enzyme" or "ribozyme") or a<br><br>
combination thereof, being a DNA-RNA hybrid molecule which may recognize at least one substrate and catalyse a modification (such as ligation or cleavage) of the at least one substrate. The terms above include nucleic acid enzymes which comprise a DNA or DNA-containing complex or an RNA or RNA-containing complex or a combination thereof, being a DNA-RNA hybrid complex which may recognize at least one substrate and catalyse a modification (such as ligation or cleavage) of the at least one substrate. The terms "nucleic acid enzyme", "catalytic nucleic acid", "nucleic acid with catalytic activity", and "catalytic nucleic acid enzyme" include within their meaning MNAzymes.<br>
[ 00169 ] The term "MNAzyme" or "multi-component nucleic acid enzyme" as used herein, refers to two or more oligonucleotide sequences (e.g. partzymes) which, only in the presence of an MNAzyme assembly facilitator (for example, a target), form an active nucleic acid enzyme that is capable of catalytically modifying a substrate. MNAzymes can catalyse a range of reactions including cleavage of a substrate, ligation of substrates and other enzymatic modifications of a substrate or substrates. An exemplary MNAzyme comprising partzyme A and partzyme B which has cleavage activity is depicted in Figure 1. MNAzymes with endonuclease or cleavage activity are also known as "MNAzyme cleavers".. With reference to Figure 1, partzymes A and B each bind to an assembly facilitator (e.g. through Watson-Crick base pairing). The MNAzyme only forms when the sensor arms of partzymes A and B hybridize adjacent to each other on the assembly facilitator. The substrate arms of the MNAzyme engage the substrate, the modification (e.g. cleavage) of which is catalyzed by the catalytic core of the MNAzyme, formed by the interaction of the catalytic domains of partzymes A and B. Cleavage of a DNA/RNA chimeric reporter substrate is exemplified in the drawing. The MNAzyme cleaves the substrate between a fluorophore and a quencher dye pair, thus generating signal. The terms "multi-component nucleic acid enzyme" and "MNAzyme" are used herein interchangeably and comprise bipartite structures, comftosed of two molecules, or tripartite structures, composed of three nucleic acid molecules, or other multipartite structures, for example those formed by four or more nucleic acid molecules. The term "MNAzyme complex" refers to a complex formed by an assembled MNAzjmie with at least one of its substrates. The MNAzyme or MNAzyme complex may comprise an<br><br>
aptamer and be referred to as an apta-MNAzyme or apta-\fNAzyme complex, respectively.<br>
[ 00170 ] The term "MNAzyme ligase" as used herein, refer to MNAzymes with ligase activity that comprise two or more oligonucleotide sequences (e.g. partzymes) which, only in the presence of an MNAzyme assembly facilitator which may also be referred to herein as a ligase assembly facilitator or assembly facilitator (for example, a tai^et), form an active nucleic acid enzyme that is capable of ligating two substrates. An exemplary MNAzyme with ligase activity is illustrated in Figure 10. The MNAzyme ligase ligates the two substrates. The term "MNAzyme ligase" as used herein comprises bipartite structures, composed of two molecules, or tripartite structures, composed of three nucleic acid molecules, or other multipartite structures, for example those formed by four or more nucleic acid molecules.<br>
[ 00171 ] The term "ligate" as used herein means to join two or more molecules. For example, a "ligase" is an enzyme that catalyses the formation (ligation) of a bond between two or more substrate molecules to make a new molecule. The term "ligation" or "ligase activity" refer to the joining of two or more substrate molecules.<br>
[00172] As used herein, the terms "partzyme", "component partzyme" "partzyme component" and "component oligonucleotide" refer to a DNA-containing or RNA-containing or DNA-RNA-containing oligonucleotide, two or more of which, only in the presence of an MNAzyme assembly facilitator as herein defined, can together form an "MNAzyme." In certain preferred embodiments, one or more component partzymes, and preferably at least two, may comprise three regions or domains: a "catalytic" domain, which forms part of the catalytic core that catalyzes a modification; a "sensor arm" domain, which may associate with and/or bind to an assembly facilitator; and a "substrate arm" domain, which may associate with and/or bind to a substrate. Illustrations of these regions or domains are shown in Figure 1 and Figure 10. Partzymes may comprise at least one additional component including but not limited to an aptamer, referred to herein as an "apta-partzyme."<br>
[ 00173 ] An example of an MNAzyme which is composed of more than two molecules is illustrated in Figure 2 (ii). A partzyme may comprise multiple components, including<br><br>
but not limited to, a partzyme component with a truncated sensor arm and a stabilizing arm component which stabilises the MNAzyme structure by interacting with either an assembly facilitator (as depicted in Figure 2 (ii)) or with a substrate.<br>
[00174] The term "nucleic acid complex" refers to catalytically active or inactive complexes which comprise two or more components selected from the group comprising but not limited to, DNAzymes, partzymes, assembly facilitators, substrates, and other MNA components including stabiliser arms, activity inhibitors, assembly inhibitors, and components or parts thereof.<br>
[00175 ] As used herein the term "nucleic acid complex" includes multicomponent nucleic acid complexes (MNA complexes). Additionally, the term "nucleic acid complex" includes uni-molecular nucleic acid enzymes complexed with at least one substrate, for example, a DNAzyme with cleavage activity bound to a substrate or a DNAzyme ligase bound to at least one of its two required substrates. The term "nucleic acid enzyme complex" refers to a DNAzyme complex or an MNAzyme complex. The term "DNAzyme complex" includes DNAzymes with at least one substrate, for example, a DNAzyme with cleavage activity bound to a substrate or a DNAzyme ligase bound to at least one of its two required substrates.<br>
[ 001761 The terms "multi-component nucleic acid complex" or "MNA complex" refer to complexes which comprise two or more components selected from the group comprising but not limited to, partzymes, assembly facilitators, substrates, and othercomponents including stabiliser arms, activity inhibitors, assembly inhibitors, and components or parts thereof. In some embodiments the MNA complex comprises an active MNAzyme. In other embodiments the MNA complex is an inactive complex such as an MNAi complex which may also be referred to herein as a multi-component nucleic acid inhibitor (MNAi) complex or MNAi complex. In yet other embodiments the MNA complex is an inactive MNA complex which may lack one or more of the components required for assembly and/or catalysis by an MNAzyme including, but not limited to the assembly facilitator, and the partzymes, or components thereof. In yet another embodiment the MNA complex comprises components for an apta-MNAzyme in the presence or absence of an assembly inhibitor.<br><br>
[ 00177 ] References to "forming a nucleic acid complex capable of functioning as a nucleic acid enzyme" as used herein refer to providing a component necessary for formation of a nucleic acid enzyme complexed with its substrate(s) such that the enzyme can modify the substrate(s). "Permitting formation of a nucleic acid complex which functions as a nucleic acid enzyme" and "allowing a nucleic acid complex to function as a nucleic acid enzyme" as used herein refers to the process that occurs when a component allows formation of a nucleic acid enzyme complexed with its substrate(s) such that the enzyme can rtiodify the substrate(s).<br>
[00178] The term "modulator" as used herein is an entity which can increase or decrease the catalytic activity of an MNA complex. Modulators may be "activators", which activate or switch on the activity of an MNAzyme. In some embodiments modulators are "inhibitors" which can switch off or inhibit the activity of the MNAzyme or MNAzyme complex, including but not limited to, "assembly inhibitors" or "activity inhibitors".<br>
[00179] The terms "assembly facilitator molecule", "assembly facilitator", "MNAzyme assembly facilitator molecule", "facilitator" and "MNAzyme assembly facilitator" as used herein refer to entities that can facilitate the self-assembly of component partzymes to form a catalytically active MNAzyme by interaction with the sensor arms of the MNAzyme. As used herein, assembly facilitators may facilitate the assembly of MNAzymes which have cleavage, ligase or other enzymatic activities. The term "ligase assembly facilitator" refers to an assembly facilitator that facilitates the assembly of an MNAzyme with ligase activity. An assembly facilitator for MNAzymes with ligase activity may also be referred to by any of the other terms above. In preferred embodiments an assembly facilitator is required for the self-assembly of an MNAzyme. An assembly facilitator may be comprised of one molecule, or may be comprised of two or more "assembly facilitator components" that may pair with, or bind to, the sensor arms of one or more oligonucleotide "partzymes". Assembly facilitator portions may be contained within other components, for example within an activity inhibitor molecule, where they are present in a state whereby they can not contribute to active MNAzyme assembly until liberated from the component by, for example, cleavage.    Assembly<br><br>
facilitators may be comprised of one (e.g. Figure 1, Figure 2(ii)) or more molecules (e.g. Figure 2(i), Figure 4-6).<br>
[ 00180 ] The assembly facilitator may be a target. The target may be a nucleic acid selected from the group consisting of DNA, methylated DNA, alkylated DNA, RNA, methylated RNA, microRNA, siRNA, shRNA, tRNA, mRNA, snoRNA, stRNA, smRNA, pre- and pri-microRNA, other non-coding RNAs, ribosomal RNA, derivatives thereof, amplicons, or any combination thereof. The nucleic acid may be amplified. The ampiiflcation may comprise one or more of: polymerase chain reaction (PCR), strand displacement amplification, loop-mediated isothermal amplification, rolling circle amplification, transcription-mediated amplification, self-sustained sequence replication, ligase chain reaction, nucleic acid sequence based amplification, or reverse transcription polymerase chain reaction (RT-PCR).<br>
[00181] An "activator" as used herein is any MNA structural or modulator oligonucleotide component, any "ligand", "target", or "event" that results in activation of MNAzymes. Activator oligonucleotides include, but are not limited to, an assembly facilitator or assembly facilitator component; a displacer oligonucleotide; a paitzyme or component thereof, for example those with truncations of the sensor or substrate arm, and partzyme stabiliser arm components.<br>
[00182] In other embodiments activators may activate MNAzymes by removing oligonucleotides that exert an inhibitory effect. Examples of oligonucleotides which can activate through such a mechanism include modulator oligonucleotides which can displace (remove) inhibitory components, or parts thereof, including but not limited to, an "activity inhibitor" or an "assembly inhibitor".<br>
[00183] The term "target" as used herein includes any nahiral or synthetic entity, constituent or analyte which is sought to be detected, identified or quantitated by a method which uses a particular nucleic acid enzyme such as an MNAzyme(s), with or without an additional amplification step and/or cascade. Targets therefore encompass the broadest range of detectable entities, constituents or analytes for which methods of sensitive detection, identification and/or quantification are desirable. Some exemplary targets include, but are not limited to, nucleic acid, protein, polypeptide, peptide,<br><br>
glycoproteins, lipids, lipoproteins, entire organisms, cells, viruses, bacteria, archaea, yeast, fungi, antibodies, metabolites, pathogens, toxins, contaminants, poisons, small molecules, polymers, metal ions, metal salts, prions or any derivatives, portions or combinations thereof. Other targets are also contemplated for use herein. It will be understood that the target may also be an assembly facilitator or activator.<br>
[00184] As used herein the term "detectable event" includes a change in the microenvironment of an MNAzyme or MNAzyme complex and/or inactive MNA complex, for example, an MNAi complex or a complex comprising components for an apta-MNAzyme in the presence of an assembly inhibitor. The change may be, for example, physical manipulation, a change in temperature, a change in pH, salt concentration, electric charge, magnetic charge, a ch£mge in the concentration of one or more anaiytes, anions, cations, chelators, or a change in concentration of MNA or modulator components, or any combination thereof It will also be understood that reference to a "change in the concentration of includes an increase or a decrease in concentration and also includes the appearance of an entity, including any component(s) for one or more MNA complexes, for example an assembly facilitator or assembly facilitator component, that was previously absent or at undetectable concentration in the microenvironment of the MNA complex, including one or more component(s) for one or more MNA complexes including MNAzymes and apta-MNAzymes, MNAzyme complexes and apta-MNAzyme complexes and/or inactive MNA complexes.<br>
[ 00185 ] Entities which represent detectable events may also be used as "activators" or "inhibitors" of the catalytic activity of MNAzymes since changes in the microenvironment can be used to manipulate catalytic activity of MNA complexes. As such, these entities allow the catalytic activity of MNAzymes to be switched "on" or "off', for example by promoting transition from an inactive MNA complex to an active MNAzyme, or vice versa. In some embodiments the event or entity promotes assembly and activation of MNAzymes. In some embodiments the event or entity promotes disassembly and inactivation of MNAzymes. In other embodiments the event or entity may direct the assembly or disassembly of inactive MNA complexes. In preferred embodiments the process of activation and inactivation of MNAzyme catalytic activity is reversible.<br><br>
[ 00186 J Alternatively, multi-component nucleic acid complexes may be inactive due to the composition of the microenvironment, which may promote either disassembly of MNA complex components or assembly of only a subset of those molecules required for formation of catalytically active MNAzymes. In some embodiments the activator assembly facilitator component is the product of a reaction occurring in the microenvironment of the inactive MNA complex and/or MNAzyme.<br>
[ 00187 ] A "detectable effect" or "output" signal is an effect that can be detected or quantified as an indication that modification of a substrate(s) has occurred. The magnitude of the effect may be indicative of the quantity of an input such as an assembly facilitator (e.g. a target). The detectable effect may be detected by a variety of methods, including fluorescence spectroscopy, siuface plasmon resonance, mass spectroscopy, NMR, electron spin resonance, polarization fluorescence spectroscopy, circular dichroism, immunoassay, chromatography, radiometry, photometry, scintigraphy, electronic methods, UV, visible light or infra red spectroscopy, enzymatic methods or any combination thereof.<br>
[00188] An "activator" assembly facilitator or assembly facilitator portion is a molecule which can be used to control the assembly of active MNAzymes or facilitate the transition from inactive multi-component nucleic acid complexes to active MNAzymes. The said multi-component nucleic acid complexes may be catalytically inactive due to the presence of an activity inhibitor, such as in MNAi complexes.<br>
[ 00189 ] The term "activity inhibitor" refers to any entity that can bind to one or more components of an MNA complex and direct assembly of catalytically inactive "MNAi" complex (e.g. Figures 4, Figure 5(iii), Figure 13). The inhibition of catalytic activity by the activity inhibitor is mediated by an "activity inhibitor portion", also called an "activity inhibitor component" or an "activity inhibitor domain" which is substantially non-complementary to the partzymes. In preferred embodiments, an activity inhibitor may comprise several distinct functional domains, for example, including but not limited to, functional domains in any combination selected fh)m an activity inhibitor domain, an assembly facilitator domain, a substrate domain, and/or a reporter domain. Such distinct functional domains may or may not coincide with several distinct structural domains in<br><br>
an activity inhibitor. Accordingly, in some embodiments, an activity inhibitor may comprise an activity inhibitor domain which is substantially non-complementary to the partzyme components and which exerts an inhibitory effect by disrupting the secondary structure required for formation of a catalytically active MNAzyme. The presence of an activity inhibitor drives the assembly of inactive MNAi complexes that are capable of interacting with, but not catalytically modifying, a substrate. In some embodiments, an activity inhibitor may comprise an assembly facilitator domain.<br>
[ 00190 ] In some embodiments, the activity inhibitor may further include a labile or cleavable linker or substrate, which may be located between two or more domains within the activity inhibitor, for example an activity inhibitor domain and an activator assembly facilitator domain. Cleavage at the substrate or linker site may allow separation of an activity inhibitor portion from an activator assembly facilitator portion, which may then function as an assembly facilitator component and direct the assembly of an active MNAzyme,<br>
[ 00191 ] In some embodiments an MNA component such as an activity inhibitor may be conjugated to other entities. In some embodiments the component (e.g. activity inhibitor) could be conjugated to a gold nanoparticle coupled to a radio-frequency magnetic field to allow remote electronic control of hybridisation. In this approach radio-frequency magnetic fields function as antennas enabling reversible thermal denaturation of specific oligonucleotides, while leaving the surrounding molecules relatively unaffected. In some embodiments the component (e.g. the activity inhibitor) could be labelled with biotin to facilitate capture and physical isolation of the component.<br>
[ 00192 ] As used herein, an "assembly inhibitor' is a component which inhibits the assembly of an MNAzyme or MNAzyme complex by complonentaty binding to an essential component of an active MNAzyme or MNAzyme complex, for example by binding to a partzyme component or an assembly facilitator or a substrate. The binding of the assembly inhibitor sequence to at least a first MNAzyme or MNAzyme complex component leads to competition between the assembly inhibitor and the said first component for binding to a second MNAzyme or MNAzyme complex component. For example, the assembly inhibitor may bind to either a partzyme substrate arm (that binds<br><br>
the substrate) or a partzyme sensor arm (that binds the assembly facilitator). When the assembly inhibitor is complementary to (and bound to) the substrate arm, it competes (and blocks) binding of the substrate to the partzyme (Figure 7) and hence blocks formation of MNAzyme complexes. When the assembly inhibitor is complementary to (and bound to) the sensor arm, it competes (and blocks) binding of the assembly facilitator to the partzyme and hence blocks formation of active MNAzymes. In this manner an assembly inhibitor blocks the assembly of MNAzymes and/or MNAzyme complexes. The assembly inhibitor molecule can be used to control the assembly of MNAzymes and MNAzyme complexes, and further allow the development of strategies for the detection of both non-nucleic acid and nucleic acid analytes.<br>
{00193 ] The terms "substrate" and "substrate molecule" as used herein include any molecule which is capable of being recognized, acted upon or modified by an enzyme including a nucleic acid enzyme. The modification of the substrate(s) provides the "output" signal for monitoring the activity of the MNA system. In particular embodiments, one or more substrate(s) may be recognized and modified by an enzyme. In other embodiments, one or more substrate(s) may be recognized and modified by a nucleic acid with catalytic activity. In preferred embodiments, a substrate or substrates may be recognized and modified by an MNAzyme. The modification of a substrate or substrates can be measured by the appearance of, or increase in, a product(s) of the modification reaction, or by the disappearance of, or decrease in, a substrate(s) modified in the reaction(s). It will be understood that reference to a substrate may refer to at least one of two or more substrates when used in the context of a reaction requiring two or more substrates such as a ligation reaction.<br>
[ 00194 ] A substrate or substrates may be modified by various enzymatic activities including but not limited to cleavage or ligation. As used herein a substrate may include a "cleavable substrate" which is amenable to enzymatic cleavage, or a "ligatable substrate" or "ligase substrate" which is amenable to enzymatic ligation.<br>
[ 0019S ] A "reporter substrate" as used herein is a substrate (or substrates) that is particularly adapted to facilitate measurement of either the disappearance of a substrate or the appearance of a product in connection with a catalyzed reaction. Reporter substrates<br><br>
can be free in solution or bound (or "tethered"), for example, to a surface, or to another molecule. A reporter substrate can be labelled by any of a large variety of means including, for example, fluorophores (with or without one or more additional components, such as quenchers), radioactive labels, biotin (e.g. biotinylation) or chemiluminescent labels.<br>
[00196] As used herein, "generic substrates" are substrates, for example reporter j substrates, that are recognized by and acted on catalytically by a plurality of MNAzymes, each of which can recognize a different assembly facilitator. The use of such substrates facilitates development of separate assays for detection, identification or quantification of a wide variety of assembly facilitators using structurally related MNAzymes all of which recognize a universal generic substrate. These generic substrates can each be independently labelled with one or more labels. In preferred embodiments, independently detectable labels are used to label one or more generic substrates to allow the creation of a convenient system for independently or simultaneously detecting a variety of assembly facilitators using MNAzymes. In some embodiments, substrates cleaved by MNAzymes could be reconstituted, and hence recycled, using an MNAzyme or DNAzyme ligase. In some embodiments, substrate(s) cleaved or ligated by MNAzymes can be further used as components or modulators of additional MNAzyme(s) or DNAzyme(s).<br>
[ 00197 ] The term "product" refers to the new molecule or molecules that are produced as a result of enzymatic modification of a substrate. As used herein the term "cleavage product" refers to a new molecule produced as a result of cleavage or endonuclease activity by an enzyme. The term "ligation product" refers to a new molecule produced as a result of the ligation of substrates by an enzyme.<br>
[ 00198 ] The term "MNAi" complex as used herein refers to an MNA complex that is in a catalytically inactive state, wherein catalytic activity is inhibited by an "activity inhibitor" as herein defined. In preferred embodiments, the MNAi complex may be catalytically inactive due to binding of an activity inhibitor oligonucleotide, for example, as depicted in Figure 4, which shows an exemplary design for an MNAi complex. An MNAi complex can be formed when partzyme A, partzyme B, an assembly facilitator<br><br>
and an activity inhibitor associate to fonn an inactive complex which can bind a substrate but not modify it.<br>
[ 00199 ] As used herein an "aptamer" may comprise a structure that has the ability to recognize one or more ligands. For example, the recognition may have a high degree of specificity due to higher level structure of the aptamer, such as, a 3-dimensional binding domain or pocket. Aptamers may therefore bind protein, polypeptide, peptide or nucleic acid, glycoproteins, lipids, lipoproteins, cells, viruses, bacteria, archaea, fimgi, antibodies, metabolites, pathogens, toxins, contaminants, poisons, entire organisms, small molecules, polymers, metal ions, metal salts, prions or any derivative, portion or combination thereof, or any other entity. Preferred aptamers herein may comprise short single-stranded DNA or RNA oligomers that can be isolated from complex libraries of synthetic nucleic acid by an iterative process of adsorption, recovery, and reamplification. Other preferred iqstamers herein may comprise protein, polypeptide, peptide or phylomers or a combination thereof that has the ability to recognize one or more ligands. Aptamers may therefore be generated against almost any target, ranging from small molecules such as amino acids, or antibiotics to protein and nucleic acid structures. An aptamer may be used to build molecular switches into MNA complexs containing assembly inhibitors. The presence of an activator ligand can switch the MNAzyme activity on and the removal of an activator ligand can switch off the activity of an MNAzyme. Further, aptamers can be used to facilitate detection of nucleic acid and non-nucleic acid ligands. Detection of a ligand can further be used to trigger an amplification cascade.<br>
[ 00200 ] As used herein, the term "cascade" refers to any succession of processes or operations that occur in successive stages, wherein the occurrence of each stage is typically dependent on the occurtence of a preceding stage. As used herein a cascade may be a "linear cascade" wherein stages (or reaction steps) occur in one direction and the occurrence of each stage (or step) is dependent on the occurrence of a preceding stage or step. As used herein, the term "feedback cascade" or "feedback amplification cascade" refers to any succession of processes or operations that occur in successive stages, wherein the occurrence of each stage is typically dependent on the occurrence of a preceding stage and whereby the occurrence of a preceding stage typically depends on processes or operations that occur in subsequent stages. In feedback cascades, the product<br><br>
of any preceding stage could serve as a component or substrate for any subsequent stage and the product of any subsequent stage could serve as a component or substrate for any preceding stage. The cascade may involve activation of an MNAi complex via removal of the influence of an activity inhibitor or component thereof<br>
[ 00201 ] A cascade may therefore include, but is not limited to, an enzymatic cascade or any other signal transduction cascade. In some embodiments, a cascade may comprise amplification of a signal resulting from catalytic activity of an MNAzyme. In some embodiments, a cascade may comprise ampliflcation of a signal resulting from catalytic activity of an MNAzyme with cleavage activity. In some embodiments, a cascade may comprise amplification of a signal resulting from catalytic activity of an MNAzyme with ligase activity. In some embodiments, a cascade may comprise amplification of a signal resulting from catalytic activity of DNAzymes or MNAzymes with ligase activity and DNAzymes or MNAzymes with cleavage activity. In preferred embodiments, such an amplification cascade may involve repeated and therefore cyclic amplification of a signal, wherein catalytic activity of a first nucleic acid enzyme creates a molecule required for modification of a substrate(s) by a second nucleic acid enzyme, which can in turn create a molecule required for modification of a substrate(s) by one or more additional nucleic acid enzymes. In some embodiments, the required molecule which is created may include but is not limited to a partzyme, a nucleic acid enzyme, for example a DNAzyme, an assembly facilitator, an assembly facilitator component, a substrate, a stabiliser arm, a component or portion thereof or a combination thereof In some embodiments, a cascade may therefore involve production of a cumulative effect, and thus detect a target of low abundance by generating a signal to a level at which it may be detected. In other embodiments, more than two catalytic stages may be employed. The cascade may be linear. In a preferred embodiment, the cascade may be exponential. In preferred embodiments, the cascade may be a feedback ampliflcation cascade.<br>
[ 00202 ]   The following abbreviations are used herein and throughout the specification:<br>
MNA: multi-component nucleic acid; MNA  complex:     multi-component nucleic  acid Q,ora!p\ex\MNAzyme'.     multi-component nucleic acid enzyme;<br><br>
MNAi: multi-component nucleic acid inhibitor;<br>
DNAzyme: deoxyribonucleic acid enzyme;<br>
PCR: polymerase chain reaction;<br>
RT-PCR: reverse transcriptase polymerase chain reaction;<br>
LNA: locked nucleic acid;<br>
PNA: peptide nucleic acid;<br>
An: analyte or target;<br>
F: fluorophore;<br>
Q: quencher; FAM or 6-FAM: 6-Carboxyfluorescein. BHQl: Black Hole Quencher 1 BHQ2: Black Hole Quencher 2 shRNA: short hairpin RNA siRNA: short interfering RNA mRNA: messenger RNA tRNA: transfer RNA snoRNA: small nucleolar RNA stRNA: small temporal RNA smRNA: small modulatory RNA pre-microRNA: precursor microRNA<br>
pri-microRNA: primary microRNA GTP: guanosine 5'-triphosphate CTP: cytosine 5'-triphosphate dATP: deoxyadenosine 5'-triphosphate ATP: adenosine 5'-triphosphate LP: ligation product CP: cleavage product oligo: oligonucleotide<br>
DETAILED DESCRIPTION OF ILLUSTRATIVE EMBODIMENTS<br>
[ 00203 ]   It is to be understood at the outset, that the figures and examples provided herein are to exemplify, and not to limit the invention and its various embodiments.<br><br>
[ 00204 ] In accordance with the present invention, there is provided methods that utilise muhi-component nucleic acid (MNA) complexes. The MNA complexes may be active enzymes (MNAzymes) with modifying activity such as ligase or cleavage activity. Further, the invention relates to cascades which may also include one or more DNAzymes. The invention also relates to methods which use these cascades for the identification, detection and quantification of assembly facilitators such as targets.<br>
1.	MNA complexes including MNAzymes and inactive MNA complexes<br>
[ 00205 ] The Multi-component Nucleic Acid enzymes (also referred to herein as "MNAzymes") and inhibitors thereof are described in detail in co-pending US applications, serial numbers 11/544,926 filed October 6, 2006 and 11/697,021 filed April 5, 2007 and in co-pending international applications PCT/AU2006/001473 and PCT/AU2007/001517, the contents of which are ail incorporated herein by reference. MNAzymes are capable of self-assembling from two or more oligonucleotide components, also referred to herein as partzymes. The partzyme oligonucleotides self-assemble in the presence of an MNAzyme assembly facilitator to form an MNAzyme. MNAzymes are therefore catalytically active nucleic acid enzymes. In some embodiments, the presence of an MNAzyme can be detected, and is indicative of the presence of a target, because the MNAzyme forms only in the presence of the target, wherein the target comprises the assembly facilitator, A wide variety of assays based on the basic principles outlined above are provided herein. Compositions comprising oligonucleotides capable of forming MNAzymes, and MNAzymes of various sequences are also provided herein. In some embodiments at least one of the oligonucleotide components, assembly facilitator, activity inhibitor or substrate may comprise an aptamer which is capable of binding to an activator or target.<br>
[ 00206 ] The present invention describes new methods, and applications for nucleic acid enzymes and complexes. The invention provides oligonucleotide components that can be used to form an MNAzyme with ligase activity which can be useful in detection of analytes, in one step formats or in nucleic acid enzyme cascade reactions.<br>
[ 00207 ] Further MNAzymes with ligase activity or other catalytic modifying activities can be used to manipulate the activity of other MNAzymes by creating new components<br><br>
which could direct the formation of a new active MNAzyme or MNAzyme complex or promote assembly of an alternative structure which lacks catalytic activity.<br>
[ 00208 ] MNAzymes with ligase activity or other catalytic activity can create new components for nucleic acid enzymes or nucleic acid enzyme complexes such as new MNAzyme or MNAzyme complex components (e.g. assembly facilitators, partzymes, substrates, stabiliser arms) or DNAzyme complex components (e.g. active DNAzymes or DNAzyme substrates).<br>
[ 00209 ] Alternatively MNAzymes can be used to direct formation of inactive MNA complexes, including MNAi complexes. MNAi complexes comprise oligonucleotide components, which would be capable of being assembled into active MNAzymes under appropriate conditions, but which when assembled with an "activity inhibitor" result in the formation of complexes which are catalytically inactive. Such inactive MNA complexes are defined herein as "MNAi" complex, and such inactivity may result from the exertion of an inhibitory influence by an activity inhibitor. MNAi complexes can interact with the substrate via the substrate arms of the partzyme components, but caimot catalytically modify the substrate. The activity inhibitor may incorporate additional entities to facilitate removal by physical means which may include, but are not limited to, entities such as attached nucleic acids, nanoparticles, microparticles, proteins, antibodies, RNA, DNA, nucleic acid analogues, biotin groups, glycoproteins, lipoproteins, peptide nucleic acids, locked nucleic acids, peptide-nucleic acid chimeras, radio-frequency moiety, or any combination thereof. The MNAi complex represents an "off state, whereas the transition to an MNAzyme represents an "on" state.<br>
[00210] The present invention describes MNAzyme ligases and their use as a mechanism for switching from an active MNAzyme to an inactive MNAi complex (Figure 13). The MNAzyme ligase could be used to ligate an activity inhibitor component to an assembly facilitator component to create an activity inhibitor. When an activity inhibitor binds and replaces an assembly facilitator component in an active MNAzyme the complex is switched to an inactive MNAi complex.<br>
[ 00211 ] Other MNA complexes may contain some or all of the components of an MNAzyme but are not catalytically active either because some required component(s)<br><br>
have not associated to form an active MNAzyme or the microenvironment is incompatible with catalytic activity of an MNAzyme. The catalytic activity may be activated upon the occurrence of a particular event, for example, including but not limited to the presence of an activator component, for example an assembly facilitator, or a change in parameter including but not limited to a change in temperature, wavelength, pressure, concentration of salt, detergent, cations or any other parameter. In some embodiments the catalytic activity of one nucleic acid enzyme, such as an MNAzyme which is present in the microenvironment, can create a new activator for a new nucleic acid enzyme or nucleic acid enzyme complex which may include, but is not limited to, an assembly facilitator or a partzyme.<br>
[00212] In preferred embodiments, the MNAzymes are based on one or more DNAzymes and/or ribozymes. DNAzymes and/or ribozymes are known which can perform catalytic modification of at least one substrate wherein the modification may be selected from the group comprising cleavage of nucleic acids, ligation of nucleic acids, phosphorylation of nucleic acids, nucleic acid capping, amino acid adenylation, cofactor synthesis, RNA polymerization, template-directed polymerization, RNA-protcin conjugation, aldol reaction, alcohol oxidation, aldehyde reduction, purine and pyrimidine nucleotide synthesis, alkylation, amide synthesis, urea synthesis, formation of peptide bonds, peptidyl-RNA synthesis, acyl transfer, aminoacylation, carbonate hydrolysis, phosphorothioate alkylation, porphyrin metallation, formation of carbon-carbon bonds, Pd nanoparticle formation, biphenyl isomerization, formation of ester bonds, formation of amide bonds, DNA deglycosylation, thymine dimer photoreversion and phosphoramidate cleavage.<br>
[ 00213 ] More preferred partzyme components for MNAzymes are based on a particular DNAzyme structure. Presently preferred stractures for MNAzymes with cleavage activity are based on DNAzymes which cleave, including the 10:23 and 8:17 DNAzymes. Presently preferred structures for MNAzymes with ligase activity are based on DNAzyme ligases which ligate substrates and create T S' linkages, such as the "7281" and "7Z48" and 7QI0 DNAzymes, as well as DNAzyme ligases which ligate substrates and create 3'-S' linkages such as the 9DB1 ligase. In various embodiments the MNAzymes and inactive MNA complexes comprise either or both ribonucleotide bases<br><br>
and deoxyribonucleotide bases. In more preferred embodiments, an MNAzyme and inactive MNA complex is based at least in part on the structure of a DNAzyme. In other preferred embodiments, MNAzymes and inactive MNA complexes comprise at least some deoxyribonucleotide bases or analog;ues thereof. In more preferred embodiments, the catalytic core portions of partzymes assembled into an MNAzyme comprise one or more deoxyribonucleotide bases or analogues thereof. In still more preferred embodiments, one or more deoxyribonucleotide bases or analogues thereof are involved in the catalysis of a substrate by an MNAzyme. In other embodiments, at least one deoxyribonucleotide base, or its analogue, in the catalytic core improves catalytic activity of an MNAzyme. In yet other embodiments, there is a strict requirement for at least one deoxyribonucleotide base, or its analogue, in the catalytic core of the MNAzyme for catalysis to occur at a measurable rate, relative to that of a comparable MNAzyme without the deoxyribonucleotide base present.<br>
[ 00214 ] The present invention discloses MNAzyme ligases which are capable of self-assembling from two or more oligonucleotide components, also referred to herein as partzymes. The partzyme oligonucleotides self-assemble in the presence of at least one MNAzyme ligase assembly facilitator to form an MNAzyme ligase. MNAzyme ligases are therefore catalytically active nucleic acid ligases.<br>
[ 00215] An exemplary MNAzyme with ligase activity comprising a first partzyme and a second partzyme is depicted in Figure 10. With reference to Figure 10, the first partzyme and the second partzyme each bind to an assembly facilitator (e.g. through Watson-Crick base pairing). The MNAzyme ligase only forms when the sensor arms (i) of the first partzyme and the second partzyme hybridize adjacent to each other on the assembly facilitator. The substrate arms (iii) of the MNAzyme engage the substrates, the modification (ligation) of which is catalyzed by the catalytic core of the MNAzyme ligase, formed by the interaction of the catalytic domains (ii) of the first partzyme and the second partzyme. Ligation of two nucleic acid substrates is exemplified in the drawing.<br>
[ 00216 ] In some embodiments, the presence of an MNAzyme with ligase activity can be detected, and is indicative of the presence of a target, because the MNAzyme ligase forms only in the presence of the target, wherein the target comprises the assembly<br><br>
facilitator. A wide variety of assays based on the basic principles outlined above are provided herein. Compositions comprising oligonucleotides capable of forming MNAzyme ligases, and MNAzyme ligases of various sequences are also provided herein.<br>
[ 00217 ] For example, the MNAzyme ligase structures are based on one or more DNAzyme ligases. More preferred are those MNAzyme ligase structures which are based on a particular DNAzyme ligase structure. Presently preferred structures are based on DNAzyme ligases including the 7Q10, 7Z81, 7Z48 and 9DB1 DNAzyme ligases.<br>
[ 00218 ] An assembly facilitator such as a target nucelcic acid may be detected using an MNAzyme with ligase activity. In one scenario, the provision of target nucleic acid is the initiating step which results in the formation of an MNAzyme with ligase activity. With reference to Figure 12, the initiating target nucleic acid would be the assembly facilitator B and the assembly facilitator A would be a synthetic molecule included to drive the assembly of MNAzyme A. MNAzyme A would then have the sole purpose of generating a 5' ligatable substrate B with a 2'3' cylic phosphate terminus in a reaction which could then be added to the reaction which detects the target assembly facilitator B using an MNAzyme B with ligase activity. This approach could provide a practical method for producing substrates with specific termini such as 2*3' cyclic phosphates which may be expensive to synthesise and may be relatively unstable. Further, with reference to Figure 12, the MNAzyme A partzymes and the assembly facilitator A could be replaced by a DNAzyme with cleavage activity present either in a separate preliminary reaction for ligatable substrate generation or the DNAzyme could be present in the same reaction mix as the MNAzyme with ligase activity. Example 11 demonstrates that cleavage and ligation using a combination of an MNAzyme and a DNAzyme can be performed in a single reaction mix.<br>
[ 00219 ] As provided herein, MNAzymes and inactive MNA complexes may contain one or more substitutions such as analogues, derivatives, modified or altered bases, ribonucleotides, alterations of the sugar or phosphate backbone, various deletions, insertions, substitutions, duplications or other modifications, or any combination of these, well known to those skilled in the art. Such modifications, substitutions, deletions, insertions, etc may be made in the sensor and/or substrate arms and/or in the catalytic<br><br>
core portions, such that the molecule retains catalytic activity. Substitutions and modifications to arms that bind the substrate or assembly facilitator may be well tolerated and in fact are the basis of allowing tailoring of the molecules to different substrates/assembly facilitators. For example, modification of the sensor arms will allow tailoring to different assembly facilitators, while modification of the substrate arms will allow tailoring to different substrates. Analysis of multiple substrates (e.g generic substrates) allows the simultaneous monitoring of multiple "detectable effects" or "output" signals as an indication of the presence of the assembly of multiple active MNAzymes in resj&gt;onse to an input event/s or entity/ies (e.g. provision of assembly facilitator/s) which are then able to modify multiple substrates<br>
[ 00220 ] In certain preferred embodiments, the invention envisages MNAzymes with catalytic activity that are comprised of deoxyribonucleotides or which are derived fh)m such molecules by certain modifications/ substitutions etc. As a general rule, replacement of the whole molecule with, for example, ribonucleotides, will render the molecule inactive because it relies for its activity on certain key deoxyribonucleotides. In a corresponding fashion, some ribonucleotides in a ribozyme may be substituted with deoxyribonucleotides but replacement of the whole molecule with, for example, deoxyribonucleotides, will render the molecule inactive.<br>
[ 00221 ] The skilled artisan will appreciate that MNAzymes with cleavage, ligation or other enzymatic activities, and inactive MNA complexes comprise either deoxyribonucleotides or ribonucleotides, or even both. Those MNAzymes and inactive MNA complexes comprising at least one and more preferably, all, deoxyribonucleotide component oligonucleotides are presently preferred. Also preferred are those MNAzymes and inactive MNA complexes comprising at least one deoxyribonucleotide base, or its analogue, within at least one of the partial catalytic cores of the partzymes. Even more preferred are those embodiments where such a base is required for catalytic activity of an MNAzyme with cleavage, ligase or other enzymatic activity.<br>
[ 00222 ] In certain embodiments at least one component of an MNA conq)lex may comprise a region of self complementarity that may under some conditions form a hairpin structure. In one embodiment, a region of self complementarity may be located in one or<br><br>
both of the partzyme sensor arms. In another embodiment the region of self complementarity may be located in one or both of the paitzyme substrate arms. In another embodiment a region or regions of self complementarity may be present in an assembly facilitator, an assembly inhibitor or an activity inhibitor component, or any combination thereof. In other embodiments MNA complexes may bind substrates which may contain regions of self complementarity.<br>
[ 00223 ] In any of the preceding aspects the catalytic modification of at least one substrate is selected from the group consisting of cleavage of nucleic acids, ligation of nucleic acids, phosphorylation of nucleic acids, nucleic acid capping, amino acid adenylation, cofactor synthesis, RNA polymerization, template-directed polymerization, RNA-protein conjugation, aldol reaction, alcohol oxidation, aldehyde reduction, purine and pyrimidine nucleotide synthesis, alkylation, amide synthesis, urea synthesis, formation .of peptide bonds, peptidyl-RNA synthesis, acyl transfer, aminoacylation, carbonate hydrolysis, phosphorothioate alkylation, porphyrin metallation, formation of carbon-caibon bonds, Pd nanoparticle formation, biphenyl isomerization, formation of ester bonds, formation of amide bonds, DNA deglycosylation, thymine dimer photoreversion and phosphoramidate cleavage<br>
[ 00224 ] The skilled artisan will also appreciate that MNAzymes comprised of DNA have advantages over uni-molecular or multipartite ribozymes, for example with respect to stability and ease of use. It is also to be appreciated that in some embodiments, MNAzymes offer advantages over uni-molecular nucleic acid enzymes, for example DNAzymes, which can only recognize one molecule which functions as both the target and the substrate. Further, catalytic modification, for example cleavage, can destroy the DNAzyme's target/substrate and as such the target is not available for another round of recognition. In contrast a single MNAzyme, for example an MNAzyme with cleavage activity, can recognize at least two molecules, namely at least one assembly facilitator and at least one substrate. As such, an assembly facilitator, for example a target nucleic acid present in a sample, is not catalytically modified in the reaction and remains available for further rounds of substrate modification by the MNAzyme. Further, a single MNAzyme which can act on two substrates (e.g. an MNAzyme with ligase activity) can recognize at least three molecules, namely at least one assembly facilitator and at least<br><br>
two substrates. These properties of MNAzymes make them adaptable for systems or processes that require the components to be able to "read" an "input" signal, for example detect the presence of an assembly facilitator, and "write" an "output" signal, for example producing a detectable signal indicative of enzymatic modification of a substrate, or by the creation of a new product (e.g. a new assembly facilitator or component) which can be useful as a new input signal. Such processes include cascade reactions, logic gates and other processes. MNAzymes therefore could provide a mechanism for transduction of information, for example, to receive an input signal and respond with an appropriate output response.<br>
2. Methods for regulating the assembly and disassembly of MNA complexes and applications for their use.<br>
[ 0022S ] MNAzyme assembly and disassembly may be controlled by changing the microenvironment. Examples of such changes include, but are not limited to, temperature, divalent cation type and concentration, salt concentration, pH, additives, and the presence or absence of critical components essential for assembly and/or activity of an active MNAzyme.<br>
[ 00226 ] The assembly of catalytically active MNAzymes may be regulated in a temperature-dependent manner. In one embodiment the assembly of catalytically active MNAzymes may be regulated in a pH-dependent manner. In one embodiment the assembly of catalytically active MNAzymes may be regulated in a manner dependent on the presence or absence of divalent cations or chelators.. In one embodiment the assembly of catalytically active MNAzymes may be regulated in a salt-dependent manner. In one embodiment the assembly of catalytically active MNAzymes may be regulated in a manner dependent on ionic concentration.<br>
[ 00227 ] The assembled MNAzyme represents an "on" state, whereas the disassembled MNA complex components represent an "off state. The assembly and disassembly of MNA complexes can be controlled by temperature. The "on" state can be induced by switching the temperature to one within the range that is compatible with both assembly and catalytic activity of an MNAzyme or MNAzyme complex. Conversely, the "off" states can be induced by switching the temperature to outside the range that is compatible<br><br>
with either assembly and/or catalytic activity of an MNAzyme or MNAzyme complex. The melting temperatures of the components of MNA complexes can be adjusted to only allow assembly within a restricted temperature range. Oligonucleotides which are particularly useful in this aspect of the invention, include but are not restricted to, stabilizer arm components, partzyme components with truncated sensor arms, and components of assembly facilitator and/or modulator oligonucleotides. Great flexibility is afforded by MNA complexes in which the components of the basic design (Figure I) have been further, split into smaller component subunits or portions (Figure 2), the sequence of which can be tailored with respect to the mehing temperature, the sequence composition and complementarity, or lack thereof, with other component oligonucleotides. With reference to Figure 2 it would be appreciated by one skilled in the art that the partzyme arm which is truncated could be any of the following: the partzyme A sensor arm, the partzyme B sensor arm, the partzyme A substrate arm, the partzyme B substrate arm, or any combination thereof.<br>
[00228] With reference to Figure 6, additional exemplary designs for active MNAzyme complexes are shown. MNAzyme complexes with the structures as exemplified in Figure 6 could also be used to modify substrates in ways other than cleavage, for example, the MNAzyme structures could also ligate substrates.<br>
[ 00229 ] The sensitivity of an MNAzyme to temperature can be exploited to build thermo-sensors and rheostats. If the temperature were either too high, or too low, for the assembly (hybridization) of the component oligonucleotides, and/or for catalytic activity, then the MNAzyme substrate would not be modified (eg cleaved or ligated). If the temperature were permissive for MNAzyme activity then the substrate would be modified and a signal would be generated. A rise or fall in temperature from one that is incompatible with MNAzyme activity, to another which is compatible with MNAzyme activity, would be detected by a signal generated following substrate modification by the MNAzyme. MNAzymes can thus provide a device capable of detecting temperature changes. One skilled in the art would appreciate that the invention of simple devices using MNAzymes for temperature sensing could be ^plied in many industries including, for example, the pharmaceutical, food and agricultural industries<br><br>
[ 00230 ] In other embodiments, a magnetic force can regulate cation concentration and hence provide a switch for modulating MNAzyme activity on and off. Positively charged cations are required for the catalytic activity of some MNAzymes. A magnetic force could alternatively switch the MNAzyme activity off by physically separating the negatively charged MNA oligonucleotides, for example the partzymes, assembly facilitators and substrates or components thereof, from the positively charged cations, for example Mg^"*". The MNAzyme could then be switched back on by allowing the MNA oligonucleotides and cations to come back in contact.<br>
[ 00231 ] In some embodiments the active "on" states (MNAzyme) can be induced using a pH within the range that is compatible with activity. Conversely, an "off state can be induced using a pH outside the range that is compatible with activity. pH may further be used to control activity of MNA complexes by inducing hydrolysis of labile sequences, and thus either creating or destroying a new component for an MNAzyme and/or inactive MNA complex.<br>
[ 00232 ] The presence or absence of any component of the MNA complexes can provide either an "on" or "off' switch. Changing, for example, the oligonucleotide sequence, the melting temperature and or concentration can achieve finer regulation. The broad scope for designs of components which can assemble into MNA complexes, for example two-part assembly facilitators and/or two-part partzyme components (e.g. with truncated sensor domains and stabilizer arms), introduces flexibility into systems which can allow tailoring (fme tuning) of conditions compatible with hybridization and hence MNA complex assembly. Further, the hybridization strength and stringency of binding of specific oligonucleotides within an MNA complex is affected by many factors, including but not limited to, salt concentration, cation concentration, temperature and the presence or absence of additives (e.g. DMSO). As such, entities that affect hybridization can provide a tool for controlling the assembly and disassembly of MNAzyme and/or inactive MNA complexes.<br>
[ 00233 ] Physical manipulation of components can be achieved, for example, by exploiting either physical properties of attached moieties as molecular "hooks", and/or by exploiting inherent properties of the oligonucleotides, for example, negative charge, or<br><br>
sequence complementarity. In another embodiment, the attached moiety allows oligonucleotides to be selectively captured, for example using a biotin group. In another embodiment the moiety contains a radio-frequency magnetic field radio to facilitate remote electronic control of hybridisation. This approach could be designed to allow the selective removal of component molecules by targeted thermal denaturation of specific oligonucleotides within an MNA complex, thus allowing activation, or inhibition, of enzymatic activity depending on whether the component molecule is itself an activator or an inhibitor sequence. For example, the activity inhibitor can be selectively denatured from an MNAi complex, allowing transition to the active MNAzyme state.<br>
[ 00234} Other strategies could be used to remove the influence of an activator or inhibitor molecule and thus promote assembly or disassembly of active MNAzymes and inactive MNA complexes such as MNAi complexes. For example, hybridization between two oligonucleotides at single stranded termini can cause ONA branch migration and unzipping of regions of double stranded nucleic acid. The process of unzipping a double stranded region of nucleic acid by branch migration can also be initiated at single stranded region which is not at the termini of a nucleic acid duplex. For example branch migration may be initiated at a single stranded region which lies between two double stranded regions. In one embodiment, an activity inhibitor could be removed fiom an MNAi complex by a modulator oligonucleotide such as a displacer oligonucleotide which functions by branch migration. In another embodiment, a ligase substrate could be removed from an MNAzyme complex with ligase activity by a modulator oligonucleotide such as a displacer oligonucleotide which functions by branch migration.<br>
[ 0023 S ] In other embodiments, complementary oligonucleotides can be used to out-competc and hence switch "off* or shut down oligonucleotide components, which in themselves may comprise either active MNAzymes or MNAzyme complexes or inactive MNA complexes. The components which are inhibited by this approach may comprise activator or inhibitor components of either MNAzymes or inactive MNA complexes.<br>
[ 00236 ] An alternate exemplary strategy is designed to facilitate switching an MNA complex between the "on state" of an active MNAzyme to the "off state" of an MNAi complex using a second MNAzyme with ligase activity (or a DNAzyme with ligase<br><br>
activity). With reference to Figure 13, an active MNAzyme A, which could be cq)able of modifying (eg cleaving or ligating) a substrate(s) A could be formed in the presence of assembly facilitator component 1 (AFC 1) and assembly facilitator component 2 (AFC 2). A second MNAzyme B, which has ligase activity, could form in the presence of assembly facilitator 3 (AF3) and could then ligate AFC 2 with an activity inhibitor component (AIC) causing the formation of an activity inhibitor (AI). This AJ could bind to the partzyme components for the MNAzyme A and result in the formation of an MNAi A complex which is inactive. As such the MNAzyme ligase in this example could operate as an off switch to inactivate the MNAzyme A.. Alternatively, a DNAzyme ligase could replace the MNAzyme B (MNAzyme ligase) and could operate as an off switch to inactivate the MNAzyme A. The inactive MNAi complex and the catalytically active MNAzyme would represent two alternate states for the assembled components, namely an "off' state and the "on" state respectiveiy.The skilled artisan will recognize that the various methods provided herein can generally be used to modulate the assembly or activity of single MNA complexes or of multiple MNA complexes in a single reaction or assay.<br>
3.	Use of the compositions as molecular switches<br>
[ 00237 ] Persons skilled in the art will recognize and understand that the present invention may be equated with a biological "switch", the applications of which are herein contemplated. Exemplary examples of mechanisms for switching on and off MNAzyme activity are listed in Table 2 below.<br>
Table 2. Active and inactive MNA states and mechanisms for switching between the two states.<br>
Type	"On" active	"Off" inactive	Example of a mechanism, which<br>
state	state	can induce transition between<br>
active and inactive states.<br>
MNAzyme or     Fully	Fully or	Temperature may be compatible<br>
MNAzyme	assembled	partially	or incompatible with assembly of<br>
complex	MNAzyme	disassembled	MNAzymes or MNAzyme<br>
assembly and        or	components	complexes<br><br>
disassembly	MNAzyme      for an	An essential MNAzyme or<br>
complex	MNAzyme	MNAzyme complex<br>
complex	component may be provided or<br>
removed.<br>
Activation or       Activator        Activator	Activator ligand provides a switch<br>
inactivation of     ligand	ligand absent      by removing assembly inhibitor<br>
MNA	present<br>
complexes	Assembly        Assembly	Removal, displacement or<br>
containing	inhibitor	inhibitor	modification of the assembly<br>
components for    removed	provided	inhibitor e.g. by branch migration<br>
an apta-MNAzymeand an assembly inhibitor<br>
Alternate MNA    MNAzyme      MNAi	Removal, displacement or<br>
complex	complex	modification of the activity<br>
structures	inhibitor e.g. by branch migration<br>
or using a second MNAzyme with<br>
ligase activity to create an activity<br>
inhibitor<br>
Catalysis	MNAzyme      MNAzyme	Separation of positive cations<br>
Inhibition	plus cation      minus cations     from negative charged DNA (e.g.<br>
e.g. Mg ^^       e.g. Mg ^^	the MNA components) using, for<br>
example, magnetic force<br>
[ 00238 ] In this regard, the presence or absence of any component of the multi-component nucleic acid complexes can provide either an "activator" or "on" switch or it may provide an inhibitor or "off switch.<br>
[ 00239 ] In some embodiments, the presence or addition of a stabilizer arm can provide an "on" switch. In one embodiment, new stabilizer arms can be generated in the<br><br>
system during a reaction, for example by cleavage of a substrate, which may comprise, for example another MNA complex component. In other embodiments the absence, modification or removal of a stabilizer arm can provide an "off' switch.<br>
[00240] In some embodiments, the presence of an assembly facilitator, or a component thereof, can provide an "on" switch. In some embodiments, new assembly facilitators can be generated by MNAzyme cleavage of MNA complex components, for example, by cleavage of a substrate including, but not limited to, those substrates which can serve as another MNA complex component such as an activity inhibitor prior to modification by cleavage. In some embodiments, the assembly facilitators can provide specific "input" signal systems, encoded within the sequence. In some embodiments the assembly facilitator can be recognized or "read". In some embodiments, the partzyme sensor arm can "read" assembly facilitator sequences including those differing by one or more single bases. In other embodiments, the absence or removal of an assembly facilitator, or a component thereof, can provide an "ofF' switch.<br>
[ 00241 ] In some embodiments components of MNA complexes may be generated by ligation of components present in the reaction milieu. In some embodiments a partzyme is created by ligation of oligonucleotides thereby generating a new partzyme component which can associate to form, for example, an active MNAzyme. In some embodiments an assembly facilitator is created by ligation of oligonucleotides thereby generating a new assembly facilitator component which can facilitate assembly of an MNA complex. In some embodiments a substrate or other nucleic acid enzyme or component thereof may be generated by ligation of components.<br>
[ 00242 ] Transition between states of activation and inactivation can provide a mechanism for creating a molecular switch, which can be regulated by alternating between the active and inactive conformations. Such molecular switches may, for example, be applied to the control of nucleic acid replication cascades, or to the regulation of autonomous therapeutic, diagnostic and computational molecular scale devices.<br>
[ 00243 ] The present invention provides compositions comprising the components for self-assembling MNAi complexes that self-assemble in the presence of one or more<br><br>
MNAzyme assembly facilitator molecules and one or more assembly inhibitor molecules to form catalytically inactive MNAi complexes.<br>
[ 00244 ] Aspects of the invention may be better understood by reference to the figures. Figure 1 depicts an example of a basic method for assembling an MNAzyme using an assembly facilitator. More specifically, partzyme A and partzyme B are shown in Figure I, each comprising a (i) sensor arm portion, (ii) a substrate arm portion, and (iii) a catalytic core portion. In the presence of an assembly facilitator, the sensor arm portions of partzyme A and partzyme B can hybridize to, and base pair with complementary portions of the assembly facilitator, for example a DNA or RNA target sequence. Upon contacting the assembly facilitator in this fashion, the MNAzyme self-assembles forming a catalytic core which can modify a substrate which is boimd by the substrate arms. Preferably the presence of the MNAzyme is detected through the detection or measurement of its catalytic activity. The substrate arms of the assembled MNAzyme can engage a substrate, for example the reporter substrate shown in Figure 1, through the interaction of the complementary sequences on the substrate arms and the substrate. Once the substrate is so engaged with the substrate arms, the catalytic core can promote the modification (eg. cleavage) of the substrate, which can in turn be measured or detected, directly or indirectly. The MNAzyme can be alternatively assembled (switched on) and dissembled (switched off) using various methods.<br>
[ 00245 ] One or more of the partzyme sensor arms, the partzyme substrate arms, the assembly facilitator and the substrate may comprise at least two molecules. In one embodiment one or more components for an MNA complex including for example the partzyme sensor arms, the partzyme substrate arms, the assembly facilitator, the activity inhibitor and the substrate may further comprise at least one aptamer.<br>
[ 00246 ] The assembly facilitator may comprise synthetic oligonucleotides which are added to the mix to drive the assembly of MNA complexes.<br>
[ 00247 ] With reference to Figiue 2, additional exemplary designs for active MNAzyme complexes are shown. The exemplary structure for one MNAzyme complex is depicted in panel (i) where multiple assembly facilitator components are required for formation of an MNAzyme.    In this design, one component (Fl) of the assembly<br><br>
facilitator is complementary to regions of the sensor arms of both partzyme A and B, whereas a second assembly facilitator component (F2) has complementarity with either partzyme B only (as per Figure 2(i)), or partzyme A only. The two assembly facilitator components together direct the assembly of an active MNAzyme which can modify (eg cleave) a substrate. Panel (ii) depicts an exemplary design of an MNAzyme complex where the assembly of partzyme A with a bi-partite partzyme B component in the presence of assembly facilitator produces an active MNAzyme capable of modifying (eg cleaving) a substrate. In this design, partzyme B has a truncated sensor arm (T), which is insufficient to allow stable MNAzyme assembly in the absence of a second component, referred to herein as a stabiliser arm component (S). However, when a stabiliser arm component hybridises to the assembly facilitator in a location adjacent to that where the truncated sensor arm of the partzyme binds, this allows assembly into an active MNAzyme.<br>
[ 00248 ] The active MNAzyme formed by the assembly of a partzyme A, a partzyme B component with a truncated sensor arm, and stabiliser arm component in the presence of an assembly facilitator represents an "on" state. Omission, removal or modification of either of the partzymes, the partzyme stabiliser arm component, or an assembly facilitator component, may result in a catalytically active "off' state. As such, the MNAzyme catalytic activity can be regulated by the presence or absence of various oligonucleotides and/or by the ability of such oligonucleotide components to be functionally active, for example to be capable of hybridizing with other oligonucleotide components to form stable MNA complexes. The truncated arm is designed to be insufficient to allow stable MNAzyme assembly under the reaction conditions, unless accompanied by a stabiliser arm component. The partzymes, stabiliser arm component, and the assembly facilitator, can thus function as "on" switches for MNAzyme activity.<br>
[ 00249 ] The reactions illustrated in Figure 3 represent two alternative states for the MNA complexes. The active MNAzyme (reaction (i)) represents the "on" state. Those reactions where either a partzyme stabiliser arm component (reaction (ii)), or an assembly facilitator component (reaction (iii)) is omitted are inactive MNA complexes representative of "off states. As such, the MNAzyme catalytic activity can be regulated by the presence or absence of various oligonucleotides and/or by the ability of such<br><br>
oligonucleotide components to be functionally active, for example, to be capable of hybridising with other oligonucleotide components to form stable MNAzymes. The truncated arm is designed to be insufficient to allow stable MNAzyme assembly under the reaction conditions, unless accompanied by a stabiliser arm component. The stabiliser arm and the assembly facilitator can function as "on" switches for MNAzyme activity.<br>
[ 00250 ] It would be appreciated by one skilled in the art that the partzyme arm which is truncated could be any of the following: the partzyme A sensor arm, the partzyme B sensor arm (as illustrated in Figure 2 (ii) and Figure 3 (i)), the partzyme A substrate arm, or the partzyme B substrate, or any combination thereof<br>
[ 00251 ] One skilled in the art would recognise that MNAzymes can be used in strategies for creating molecular sensors, molecular switches, and/or modulators or propagators of autocatalytic self-replicating cascades and other iterative processes. Areas of use include, but are not limited to, medical, veterinary, agricultural, food technology, imaging and anti-bioterrorism applications.<br>
[ 00252 ] With reference to Figure 4, an MNAi complex is formed when partzymes A and B complex with an assembly facilitator and an activity inhibitor (left hand side). The inactive MNAi complex is capable of interacting with, but not catalytically modifying, the substrate. In some embodiments, the activity inhibitor may further include a labile or cleavable linker, which may separate two or more domains within the activity inhibitor. Such domains may include, for example, (i) an inhibitor portion which is substantially non-complementary to the partzyme components and which exerts an inhibitory effect by disrupting the secondary structure required for formation of a catalytically active MNAzyme and (ii) an activator portion which, if separated from the inhibitor portion, may function as an additional assembly facilitator component and direct the assembly of an active MNAzyme.<br>
[ 00253 ] With reference to Figure 4, an active MNAzjmie complex (right hand side) can be derived from the components of the MNAi complex, following modification of the activity inhibitor such as to cleave or bisect the molecule and separate the (i) activity inhibitor portion and the (ii) activator assembly facilitator portion. The released activator<br><br>
assembly facilitator portion is then able to function as a second assembly facilitator component, which in concert with a first assembly facilitator component, can direct the assembly of partzyme components A and B into an active MNAzyme capable of catalytically modifying a substrate.<br>
[ 00254 ] Conversely an active MNAzyme complex (right hand side of Figure 4) can be converted to an inactive MNAi complex (left hand side of Figure 4), by ligation or joining of an activity inhibitor portion (i) to the activator assembly facilitator portion (ii) thus creating an activity inhibitor.<br>
[ 00255 ] The inactive MNAi complex and the catalytically active MNAzyme represent two alternate states for the assembled components, namely an "off state and the "on" state respectively.<br>
[ 00256 ] With reference to Figure 13 an exemplary strategy for switching an MNA complex from the "on state" of an active MNAzyme to the "off state" of an MNAi complex using a second MNAzyme with ligase activity is described. An active MNAzyme A, which could be capable of modifying (eg cleaving or ligating) a substrate(s) A could be formed in the presence of assembly facilitator component 1 (AFC 1) and assembly facilitator component 2 (AFC 2). A second MNAzyme B, which has ligase activity, could form in the presence of assembly facilitator 3 (AF3} and could then ligate AFC 2 with an activity inhibitor component (AIC) resuhing in the formation of an activity inhibitor (AI). This AI can bind to the partzjone components for MNAzyme A resulting in the formation of an MNAi A complex which is inactive. As such the MNAzyme ligase in this example can operate as an off switch to inactivate an MNAzyme A. The inactive MNAi complex and the catalytically active MNAzyme represent two alternate states for the assembled components, namely an "off" state and the "on" state respectively.<br>
4.     Methods using insoluble and solid supports<br>
[ 00257 ] It is also to be understood that generally the methods, whether multiplexed or not, are applicable in solution, or combined with an insoluble support or solid support on which one or more of any MNA complex component(s), including the substrate or portion thereof, may be attached. Further, additional enzymes or components thereof<br><br>
present in an MNAzyme cascade, including DNAzymes with for example,either cleavage and/or ligase activity may be attached to an insoluble support or solid support. Accordingly, at least one of a DNAzyme, a partzyme component, a substrate, an assembly inhibitor, an assembly facilitator or assembly facilitator component, a stabilizer arm and/or an inhibitor such as an activity inhibitor or assembly inhibitor may be bound, attached or tethered. Further, a portion of at least one of a partzyme component, a substrate or substrates, an assembly inhibitor, an assembly facilitator or assembly facilitator component, a stabilizer arm and/or an inhibitor such as an activity inhibitor or an assembly inhibitor may be bound, attached or tethered.<br>
[ 00258 ] Embodiments of the present invention encompass an insoluble support. By way of example the insoluble support could be in the form of a "chip", otherwise known as an array or microarray, which typically comprise a plurality of oligonucleotides coupled, tethered or otherwise attached to the chip. In particular embodiments, the oligonucleotides comprise nucleic acid.<br>
[ 002S9 ] In particular embodiments, the plurality of oligonucleotides could comprise a plurality of components, or portions thereof, for MNA complexes which include but are not limited to, substrates, and/or partzymes, and/or assembly facilitators, and/or assembly inhibitors and/or activity inhibitors and/or any combination thereof In particular embodiments, the plurality of oligonucleotides could comprise a plurality of oligonucleotides which are complementary to a domain contained within, and/or appended to, any of the components comprising the plurality of components, or portions thereof, for MNA complexes. In particular embodiments, the plurality of components, or portions thereof, for MNA complexes could comprise oligonucleotides with more than one functional domain which may comprise a substrate, and/or a partzyme, and/or an assembly facilitator, and/or an assembly inhibitor and/or an activity inhibitor and/or any combination therof.<br>
[ 00260 ] A plurality of oligonucleotides may be positioned upon a solid support, for example a chip, bead, plate or array by any suitable method known in the art, for example, by pipette, ink-jet printing, contact printing or photolithography. The solid support may be comprised of at least one element, with each element comprising at least<br><br>
one nucleic acid oligonucleotide. The at least one element may be comprised of a plurality of oligonucleotides of the same sequence. The number of elements comprising the solid support may be any nimiber, and where a plurality of elements is positioned on a solid support, the elements may be spaced apart at a uniform or a variable distance, or a combination thereof. In some embodiments, the elements may be positioned randomly, with the respective location of each element then determined. The size and shape of the elements will depend upon the particular application of the present invention, and different sized and shaped elements may be combined into a single solid support. The surface of the solid support may be substantially planar or may have features such as depressions or protuberances, and the elements may be positioned either into the depressions or onto the protuberances. Such depressions may provide a reservoir for solutions into, which the elements are inunersed, or such protuberances may facilitate drying of the elements. For example, elements may be placed in each well of a 96 well plate. In some embodiments, the solid support may include unique identifiers such as indicia, radio frequency tags, integrated devices such as microprocessors, barcodes or other markings in order to identify each of the elements. The unique identifiers may additionally or alternatively comprise the depressions or protuberances on the surface of the array. Furthermore, the unique identifiers can provide for correct orientation or identification of the chip. The unique identifiers may be read directly by a data capture device or by an optical scanner or detector.<br>
5.      Substrate systems<br>
[ 00261 ] Also provided in accordance with the present invention are generic reporter substrate systems, which allow rapid system development by allowing facile design changes to create new MNAzymes and inactive MNA complexes which recognize different assembly facilitators. As discussed herein, the substrate arm portion and the catalytic core portion of the partzymes may remain unchanged, with changes only to the sensor arm portion of one or more partzymes required for new assembly facilitators. A generic substrate sequence or sequences is provided and the same substrate or substrates can therefore be incorporated in systems for various diverse applications. Further, the same substrate(s} can be incorporated into the methods in various embodiments herein, including assays where the substrate(s) is free in solution or is tethered or attached to a<br><br>
support. A series of generic substrates can be used in a multiplex reaction allowing simultaneous detection of multiple assembly facilitators.<br>
[ 00262 ] In some cases, substrates which have been cleaved by an MNAzyme or DNAzyme with endonuclease or cleavage activity can be reconstituted and hence recycled using a DNAzyme ligase or a MNAzyme ligase. Substrates which have been ligated by a DNAzyme or MNAzyme with ligase activity can be recycled as substrates for a DNAzyme or MNAzyme with cleavage activity provided the DNAzyme or MNAzyme cleaver can cleave the link created by ligation by the DNAzyme or MNAzyme ligase. Alternatively, an MNAzyme with cleavage activity may cleave at a site within a ligation product which is not at the junction of the two ligatable substrates provided the cleavage site requirements are met at this site.<br>
[ 00263 ] As described in more detail below, MNA complexes have an advantageous property in certain embodiments of being able to utilize a universal or generic substrate or substrates. Such a substrate is shown in Figure 1 in a presently preferred configuration wherein the substrate for an MNAzyme with cleavage activity comprises both a detectable portion and a quencher portion. The quencher portion is adapted to diminish or eliminate a detectable signal from the detectable portion of the substrate until the substrate is cleaved by an MNAzyme. For example, the quencher portion may comprise "Black Hole Quencher 1" (BHQl) or "Black Hole Quencher 2" (BHQ2).<br>
[ 00264 ] Thus, the MNAzyme cleaves the substrate between the detectable portion and the quencher portion allowing the two portions to separate in solution, thereby allowing the detectable signal to appear or increase as the quencher portion is distanced from, or effectively removed from the local environment of the detectable portion.<br>
[ 00265 ] Further, substratc(s) for an MNAzyme with ligase activity, as shown in Figure 10, can be designed such that one substrate comprises a detectable portion and one substrate comprises a quencher portion. The quencher portion is adapted to diminish or eliminate a detectable signal from the detectable portion when the substrates are ligated into a ligation product thus bringing the detectable and quencher portions into close proximity. For example, the quencher portion may comprise "Black Hole Quencher 1" CBHQl) or "Black Hole Quencher 2" (BHQ2). Further, either an MNAzyme with ligase<br><br>
activity or a DNAzyme with ligase activity could ligate substrates bringing the detectable portion of one substrate in close proximity with the quencher portion of the second substrate, thereby allowing the detectable signal to decrease.<br>
[ 00266 ] In another example, if the 5' ligation substrate and the 3' ligation substrate were each labelled with either a fluorophore or quencher dye respectively (or vice versa) then ligation would result in a decrease in fluorescence, which could serve as an indicator of the presence of the tai^get analyte.<br>
[ 00267 ] In an alternative format, if the 5' ligation substrate, for example, were labelled v*dth a detectable moiety, for example a fluorophore, and the 3' ligation substrate were attached to a discrete location, for example on a solid support such as a chip or a bead, then ligation would result in the appearance of a signal at the discrete location in the presence of target analyte.<br>
[ 00268 ] The use of the generic or universal substrate(s) is enabled through the design of the MNAzyme's component partzymes. By altering only the sensor arms of the partzymes, but by leaving the substrate arms unchanged, a large variety of MNAzymes or inactive MNA complexes specific for each of a plurality of assembly facilitators can be designed all of which utilize a universal generic substrate(s) to produce an output signal. The skilled artisan will appreciate the advantages that this offers in terms of eliminating the need for customized or unique substrates to respond to each input event or entity. Detection of each new assembly facilitator requires only one or more changes in one or more of the sensor arm portions; while the substrate arm portion and the catalytic core portion may remain constant. Thus, a single substrate (e.g. reporter substrate) for an NfNAzyme cleaver, or a single pair of ligatable substrates for an NfNAzyme ligase, can be used for detection of a single assembly facilitator or other input event using an MNAzyme, and/or detection of multiple assembly facilitators in a series of systems using altered MNAzymes. A plurality of generic cleavablc reporter substrates, or generic ligatable substrate pairs, allows multiplexed monitoring of multiple MNAzymes within one system.<br>
[ 00269 ] In some instances it may be useful to have different MNAzymes in a mix which have different sensor arms and at least one substrate arm in common.    For<br><br>
example, with reference to the feedback cascade illustrated in Figure 16, the MNAzyme cleaver 3 could have different sensor arms than MNAzyme cleaver 1 but it may have one substrate arm (on one of the partzymes) that is the same as the sequence of one substrate arm of the partzyme of MNAzyme cleaver 1 which interacts with that portion of the sequence on substrate 1 (here, the 5* portion of the substrate) which is subsequently ligated by the ligasc. Therefore instead of cleaving substrate 1, MNAzyme cleaver 3 would cleave another substrate (referred to as substrate 3) which has the same sequence at the 5* portion but which has a different portion at the 3' end (or vice versa) with respect to cleavable substrate 1. The cleavage site of substrate 3 would lie at the junction of those sequences which are shared between the cleavable substrates 1 and 3 and those which differ. In this case, when MNAzyme cleaver 3 was assembled in the presence of an assembly facilitator component generated by cleavage by the MNAzyme cleaver 2, and if MNAzyme cleaver 3 cleaved substrate 3 then the 5' cleavage product of MNAzyme cleaver 3 could also serve as a 5' substrate for the DNAzyme ligase (or the assembled MNAzyme ligase) and a feedback amplification cascade reaction could be initiated.<br>
[ 00270 ] Further, the substrates may incorporate additional entities such as labelled nucleic acids, nanoparticles, microparticles, proteins, antibodies, RNA, DNA, nucleic acid analogues, biotin group, glycoproteins, lipoproteins, peptide nucleic acids, locked nucleic acids, peptide-nucleic acid chimeras, moiety for radio-frequency magnetic field, or any combination thereof.<br>
[ 002711 Substrates can be modified by an MNAzyme thereby providing a "detectable effect" or "output" signal. In the detection process, the substrate modification by an MNAzyme may involve, for example, cleavage of nucleic acids, ligation of nucleic acids, phosphorylation of nucleic acids, nucleic acid capping, amino acid adenylation, cofactor synthesis, RNA polymerization, template-directed polymerization, RNA-protein conjugation, aldol reaction, alcohol oxidation, aldehyde reduction, purine and pyrimidine nucleotide synthesis, alkylation, amide synthesis, urea synthesis, formation of peptide bonds, peptidyl-RNA synthesis, acyl transfer, aminoacylation, carbonate hydrolysis, phosphorothioate alkylation, porphyrin metallation, formation of carbon-carbon bonds, Pd nanoparticle formation, biphenyl isomerization, formation of ester bonds, formation of<br><br>
amide bonds, DNA deglycosylation, thymine dimer photoreversion or phosphoramidate cleavage,<br>
[ 00272 ] . As a consequence of substrate modification by an MNAzyme, a detectable effect may be generated and the magnitude of the effect may therefore be indicative of the quantity of the input signal. The detectable effect may be detected by a variety of methods, including fluorescence spectroscopy, surface plasmon resonance, mass spectroscopy, NMR, electron spin resonance, polarization fluorescence spectroscopy, circular dichroism, immunoassay, chromatography, radiometry, photometry, scintigraphy, electronic methods, UV, visible light or infra red spectroscopy, enzymatic methods or any combination UV, visible light or infra red spectroscopy, enzymatic methods or any combination thereof<br>
[ 00273 ] The method may further comprise quantifying the magnitude of the output signal or detectable effect. In one embodiment the method may further comprise determining the magnitude of the input event from the quantified output signal.<br>
[ 00274 ] The determination of catalytic activity, for example by determination of the presence of detectable effect or output signal, may be performed in a manner permitting the catalytic activity to be quantified. In one embodiment the magnitude of the detectable event may be determined from the quantified catalytic activity.<br>
[ 00275 ] Several groups have reported detection of nucleic acid targets, and other analytes with colourimetric readouts (Elghanian et ai, 1997, Mirkin et al, 1996, and Liu and Lu, 2004). The strategy involves preparation of batches of gold nanoparticles, each of which has a distinct DNA oligonucleotide sequence attached to its surface. Gold particles can then be aggregated by the addition of a "bridging oligonucleotide", which has complementarity with the sequences that are attached to the gold particles. Particle aggregation results in a concomitant change in colour from red to blue (Mirkin et al, 1996). Inclusion of a DNAzyme substrate sequence within the bridging oligonucleotide can provide a mechanism for reversing the aggregation of the gold particles (Liu and Lu, 2004). Activation of the lead-dependent DNAzyme by the addition of lead, caused cleavage of the bridging oligonucleotide, dissociation of the gold particles and a change in colour from blue to red.<br><br>
[ 00276 ] Simple detectors for monitoring changes could be developed usmg this principle and an MNAzyme with endonuclease or Hgase activity. Changes in temperature or other entities or events could activate MNAzymes which could modify bridging oligonucleotides causing the dissociation (in the case of a cleaver) or association (in the case of a ligase) of nanoparticles and a change in colour.<br>
6.	Optimization of the methods<br>
[ 00277 ] The skilled artisan will readily understand that the methods described herein may be optimized using a variety of experimental parameters. The particular experimental parameters that are optimized, and the level of such optimization, will depend upon the particular method being employed and/or the particular event to be detected. Such parameters include, but are not limited to, time, temperature, concentration of salts, detergents, cations and other reagents including but not limited to dimethylsulfoxide (DMSO), and length, complementarity, GC content and melting point (Tm) of nucleic acids. The temperature at which such methods may be performed may be in the range of about 20°C to about 96°C, about 20°C to about 75*'C, about 20°C to about 60'*C or about 20 °C to about 55°C. The temperature may be constant or may be cycled between one temperature which is compatible with assembly of an MNA complex and/or catalytic activity of an MNAzyme and one temperature which is incompatible with assembly of an MNA complex and/or catalytic activity of an MNAzyme.<br>
[ 00278 ] Additionally or alternatively, a variation in a parameter, and/or the microenvironment, may be used to trigger activation of the MNA complex from an inactive to an active state. Thus, a parameter in the microenvironment may comprise an "activator" as herein deflned, including but not limited to events such as change in temperature, wavelength, concentration of salts, detergents, cations, and concentration of structural and modulator components which include but are not limited to assembly facilitators or assembly facilitator components, paitzymes or partzyme components, substrates, assembly inhibitors, activity inhibitors and activator oligonucleotide components. Accordingly, such optimization of parameters and/or microenvironment may be undertaken in order to achieve use of the MNA complexes as elements of cascade reactions or molecular switches.<br><br>
[ 00279 ] In one preferred embodiment, optimized reactions for practicing the methods of using MNA complexes are provided herein. In such optimized reactions, activation of catalytic activity, which in some cases may be by one or more nucleic acid enzymes in a cascade, is increased by up to 10, 20, or 30 % above unoptimized reactions. More preferred reaction conditions improve catalytic activity by at least 35%, or 40%, and preferably up to 50% or more. In still more preferred embodiments, optimized reactions have an increase in activation of catalytic activity of more than 50%, and up to 66%, 75% or even 100%. In yet more preferred embodiments, a fully optimized reaction method will offer 100, 200 or even 300% or more increase in activation of catalytic activity. Other preferred reaction conditions can improve the activation of catalytic activity by up to 1000% or more over methods practiced with unoptimized reaction conditions. A highly preferred reaction condition for optimizing the methods provided herein is the inclusion of certain divalent cations. The catalytic activity of most nucleic acid enzymes may be influenced in a concentration-dependent fashion by the concentration of divalent cations. Preferred optitnized reactions are optimized for one or more of Ba^^ Si^^ Mg^^ Ca^^ Ni^*, Co^^ Mn^ Zn^^ and Pb^^<br>
7.	Methods using aptamers<br>
[ 00280 ] With reference to Figure 7, a method is illustrated whereby an activator ligand can be used to switch "on" or "ofP' the activity of the apta-MNAzyme. The method uses an assembly inhibitor to block activity of apta-MNAzymes in the absence of an activator as illustrated. These methods use aptamers which may comprise a nucleic acid, protein, polypeptide, phylomer, peptide or combination thereof that has the ability to recognize one or more ligands. Aptamers may bind, for example, proteins, polypeptides, peptides or nucleic acids, glycoproteins, lipids, lipoproteins, cells, viruses, bacteria, archaea, fungi, antibodies, metabolites, pathogens, toxins, contaminants, poisons, entire organisms, small molecules, polymers, metal ions, metal salts, prions or any derivatives, portions or combinations thereof, or any other entity (Lee et al., 2004).<br>
[ 00281 ] Preferred aptamers herein may comprise short single-stranded DNA or RNA oligomers that can be isolated from complex libraries of synthetic nucleic acids by an iterative process of adsorption, recovery, and reamphfication.  Other preferred aptamers<br><br>
may comprise protein, polypeptide, peptide or phylomers or a combination thereof that has the ability to recognize one or more ligands. Aptamers may be generated which may recognize and bind almost any target, ranging from small molecules such as amino acids or antibiotics, to protein and nucleic acid structures. In preferred embodiments, aptamers include, for example, nucleic acid binding molecules which are preferably generated by evolution and selection techniques. Preferably, aptamers may comprise DNA or RNA molecules, or a combination of both, including but not limited to the nucleotide analogues as per, for example. Table 1 above.<br>
[ 00282 ] At least one component of the MNA complex may contain at least one aptamer or portion thereof wherein said aptamer or portion thereof binds a ligand selected from the group comprising nucleic acids, polypeptide, peptide, entire organisms, proteins, glycoproteins, lipids, lipoproteins, cells, viruses, bacteria, archaea, fungi, antibodies, metabolites, pathogens, toxins, contaminants, poisons, small molecules, polymers, metal ions, metal saUs, prions or any derivatives, portions or combinations thereof, or any other entity.<br>
[ 00283 ] An apta-MNAzyme, for example, with either cleavage or ligase activity may incorporate one or more aptamers. The assembly of the apta-MNAzyme may therefore be directed by the presence or absence of one or more analytes which bind the aptamer and cause dissociation of an assembly inhibitor.<br>
[ 00284 ] One skilled in the art will appreciate that one or more aptamers may be incorporated into either end of the assembly facilitator molecule or molecules, and or the activity inhibitor. Further it will be appreciated that multiple aptamers could be incorporated into one or more of the partzyme oligonucleotide components. Still further it will be appreciated that aptamers could also be incorporated into at least one of the partzyme oligonucleotide components. The assembly facilitator in the strategies illustrated in Figure 7 can comprise DNA, RNA, LNA, PNA or a sequence containing one or more nucleotide base analogues. One skilled in the art would appreciate that it could be possible to recognise two molecules, for example, one target nucleic acid acting as the assembly facilitator, and one ligand which binds the aptamer.<br><br>
[ 00285 ] In the strategy shown in Figure 7, an aptamer sequence is incorporated at the end of a partzyme (apta-partzyme) in a configuration whereby an active apta-MN Azyme is only formed in the presence of the activator.<br>
[ 00286 ] It will also be appreciated by one skilled in the art that one or more aptamers could be incorporated into any of the oligonucleotide components, including the partzymes, the assembly facilitator, activity inhibitor or the substrate. Further the aptamer could be incorporated into either end of any one of these oligonucleotides.<br>
[ 00287 ] The strategy illustrated in Figure 7 can be used either (i) to provide a method to control apta-MN Azyme activity using ligands as activator molecules, and/or (ii) to provide a method for detection of non-nucleic acid, and nucleic acid targets using apta-MNAzymes. Further, the apta-MNAzyme could be the first nucleic acid enzyme in a cascade reaction designed to detect a range of ligand targets. The nucleic acid oligonucleotides required for one schema for apta-MNAzyme detection strategy include;<br>
a)	a partzyme;<br>
b)	an apta-partzyme which is a partzyme with an aptamer incorporated into one of its ends;<br>
c)	an assembly facilitator which is an oligonucleotide which binds to both the apta-partzyme and the partzyme enabling assembly of an active MNAzyme;<br>
d)	a substrate e.g. a reporter substrate; and<br>
e)	an assembly inhibitor oligonucleotide which hybridises to the apta-partzyme in a region which spans at least part of the aptamer sequence and part of the substrate binding arm of the apta-partzyme sequence.<br>
[ 00288 ] In the absence of an activator ligand (Figure 7; left hand panel), the assembly inhibitor oligonucleotide binds to the apta-partzyme thus competing with and blocking binding of the reporter substrate. When an activator ligand is present (Figure 7; right hand panel), it binds to the aptamer sequence of the apta-partzyme, blocking the binding of the assembly inhibitor oligonucleotide, and thus allowing assembly of an active apta-MNAzyme complex and and subsequent modification, for example cleavage of the substrate (as illustrated in Figure 7) or ligation of substrates.  As such, an active apta-<br><br>
MNAzyme complex can only form and cause fluorescent signal generation in the presence of activators or ligands that can bind an aptamer portion of an apta-partzyme.<br>
[ 00289 ] In a fiirther embodiment of the apta-MNAzyme schema illustrated in Figure 7 the system could be used to simultaneously detect a combination of nucleic acid and non-nucleic acid molecules. By way of example, the apta-MNAzyme may be used to detect the presence of (i) a nucleic acid target, which functions as an assembly facilitator and directs the formation of the apta-MNA complex, and (ii) a target activator ligand (e.g. a protein or small molecule) which could displace the assembly inhibitor oligonucleotide and allow assembly of an active apta-MNAzyme complex and subsequent substrate modification.<br>
[ 00290 ] In another strategy using an apta-MNAzyme with ligase activity, at least two ligatable substrates could be present in the reaction, one of which could be labelled with a fluorophore and the other of which could be labelled with a quencher. When an activator ligand is present, it would bind to the aptamer sequence of the apta-partzyme, blocking the binding of the assembly inhibitor oligonucleotide, and thus allowing binding and ligation of the substrates. As such, apta-MNAzyme complexes could only form and cause a decrease in fluorescent signal in the presence of activators that can bind aptamers.<br>
[ 00291 ] In some embodiments, modulation of MNAzyme activity can be achieved using either a nucleic acid or a non-nucleic acid target activator ligand as a switch mechanism. In other embodiments, the assembly inhibitor molecule is manipulated by other means so as to modulate activity. For example, the assembly inhibitor could be removed by several strategies including selective thermal denaturation or methods that use oligonucleotides to compete for binding and/or displacer olignucleotides that use branch migration to displace oligonucleotide components of MNA complexes.<br>
[ 00292 ] It would be appreciated by one skilled in the art that the aptamer sequences for an apta-MNAzyme with modifying activity, for example cleavage or ligation activity, could be appended to either partzyme A or partzyme B or both. In the schema above, the assembly inhibitor oligonucleotide bound to portions of the aptamer and the substrate arm of at least one of the partzymes.<br><br>
[ 00293 ] It would also be appreciated by one skilled in the art that the aptamer sequences for an apta-NfNAzyme could be appended to the sensor arms of at least one of either partzyme A or partzyme B or both. In this embodiment the assembly inhibitor oligonucleotide would bind to portions of the aptamer and the sensor arm. In the absence of an activator ligand, the assembly inhibitor oligonucleotide binds to the apta-partzyme thus competing with and blocking binding of an assembly facilitator, or assembly facilitator component. When an activator ligand was present, it would bind to the aptamer sequence of the apta-partzyme, blocking the binding of the assembly inhibitor oligonucleotide, and thus allowing binding of the assembly facilitator or assembly facilitator component and directing assembly of an active apta-MNAzyme capable of modifying, for example cleaving or ligating, the substrate or substrates. As such, apta-MNAzymes could only form and cause a change in fluorescent signal, for example an increase or decrease in fluorescence, in the presence of activators that can bind aptamers.<br>
[ 00294 ] When the apta-MNAzyme has cleavage activity as illustrated in Figure 7 then the modification can cause for example an increase in fluorescence by cleavage of a dual labelled reporter substrate which results in separation of a fluorophore/quehcher dye pair.<br>
[ 00295 ] It would be appreciated that the modification performed by an apta-MNAzyme could be a modification other than cleavage, for example, ligation of two substrates. In this case the signal change could be, for example, a decrease in fluorescence following ligation of two ligatable substrates each labelled with either the fluorophore or the quencher of a fluorophore/quencher dye pair.<br>
[ 00296 ] This apta-MNAzyme approach could also be used to develop target detection methods or molecular switches that can turn on and off the catalytic activity of the MNA complex. Further, the apta-MNAzyme approach may comprise the first step in a linear nucleic acid enzyme cascade or a feedback nucleic acid enzyme amplification cascade. The apta-MNAzyme could also be applied to simultaneous detection of both nucleic acid targets such as assembly facilitator and non-nucleic acid target ligands such as those which can bind to aptamers.<br><br>
8.	Cascades<br>
[ 00297 ] Cascades are any succession of processes or operations that occur in successive stages, wherein the occurrence of each stage is typically dependent on the occurrence of a preceding stage. In a linear cascade, stages (or reaction steps) occur in one direction and the occurrence of each stage (or step) is dependent on the occurrence of a preceding stage or step. In feedback cascades or feedback amplification cascades there may be a succession of processes or operations that occur in successive stages, wherein the occurrence of each stage is typically dependent on the occurrence of a preceding stage and whereby the occurrence of a preceding stage further depends on processes or operations that occur in subesequent stages. In feedback cascades, the product of any preceding stage could serve as a component or substrate for any subsequent stage and the product of any subsequent stage could serve as a component or substrate for any preceding stage.<br>
[ 00298 ] A cascade may therefore include, but is not limited to, an enzymatic cascade or any other signal transduction cascade. In some embodiments, a cascade may comprise amplification of a signal resulting from catalytic activity of an NfNAzyme. In some embodiments, a cascade may comprise amplification of a signal resulting fix&gt;m catalytic activity of an MNAzyme with cleavage activity. In some embodiments, a cascade may comprise amplification of a signal resulting from catalytic activity of an MNAzyme with ligase activity. In some embodiments, a cascade may comprise amplification of a signal resulting from catalytic activity of DNAzymcs or MNAzymes with ligase activity and DNAzymes or MNAzymes with cleavage activity. In prefored embodiments, such an amplification cascade may involve repeated and therefore cyclic amplification of a signal, wherein catalytic activity of a first nucleic acid enzyme creates a molecule required for modification of a substrate(s) by a second nucleic acid enzyme, which can in turn create a molecule required for modification of a substrate(s) by one or more additional nucleic acid enzymes. In some embodiments, the required molecule may include but is not limited to a partzyme, a nucleic acid enzyme, for example a DNAzyme, an assembly facilitator, an assembly facilitator component, a substrate, a stabiliser arm, a portion or component thereof or a combination thereof. In some embodiments, a cascade may therefore involve production of a cumulative effect, and thus detect a target of low<br><br>
abundance by generating a signal to a level at which it may be detected. Cascades may be employed to amplify a signal, for example, in applications where an input event is of low intensity, for example when a target is in low abundance, and may not otherwise provide for a output signal that is detectable. In other embodiments, more than two catalytic stages may be employed. The cascade may be linear. In a preferred embodiment, the cascade may be exponential.<br>
[ 00299 ] In linear cascades (e.g. as illustrated in Figure 9, 12, IS), each step in the cascade employs a nucleic acid enzyme, some or all of which may be undergoing multiple turnover whereby a single nucleic acid enzyme (e.g. a DNAzyme or MNAzyme) may perform multiple cleavages or ligations or other modifications at one or more steps in the cascade. Multiple turnover may facilitate amplification at one or more steps in the cascade resulting in amplification of the signal detected during the detection step(s).<br>
[00300] Further, an extension of the linear cascade could result in a feedback amplification cascade. If, for example, the sequence of a product of a third nucleic acid enzyme was the same as a product of a first nucleic acid enzyme then the said product of the third nucleic acid enzyme could also serve as a molecule permitting a second nucleic acid complex to fimction as a second nucleic acid enzyme and a feedback amplification cascade could be initiated. In this reaction the third nucleic acid enzyme would constantly generate a product which could in turn serve as a molecule permitting a second nucleic acid complex to fimction as a second nucleic acid enzyme thus allowing the creation of more product by the second nucleic acid enzyme that could permit the third nucleic acid complex to function as a third nucleic acid enzyme. It is envisaged this strategy could provide a mechanism for feedback signal amplification following initiation of a reaction by an assembly facilitator (eg a target analyte) which could allow assembly of a first nucleic acid enzyme. The strategy could allow detection of assembly facilitators (eg target analytes) followed by signal amplification using for example, a DNAzyme (or MNAzyme) which could ligate substrates and MNAzymes which could cleave a substrate. One possible feedback cascade is illustrated in Figure 16. Other variations of this schema are described throughout the specification.<br><br>
[ 00301 ] Persons skilled in the art will appreciate that the methods described herein may be used to perform a cascade as herein defined. It will be also be appreciated by one skilled in the art that other strategies based on Figures 9, 12, 15 and 16 could involve modification of at least one substrate by at least one MNAzyme by a variety of means including, for example cleavage or ligation. Unlike target amplification techniques such as the polymerase chain reaction, or the ligase chain reaction, linear and feedback nucleic acid enzyme (DNAzyme and NfNAzyme) cascades allow amplification in formats that require no protein enzymes to facilitate the process. This provides a major advantage over these commonly used target amplification protocols. Further, the ability to control and regulate the catalytic activity of MNAzymes using several oligonucleotide components, such as stabiliser arm components or assembly facilitator components allows the assembly of MNAzymes to be tightly regulated by conditions and components in the microenvironment. Persons skilled in the art will appreciate that the cascades described herein may be initiated by formation of an active MNAzyme or apta-MNAzyme as herein defined.<br>
[ 00302 ] An additional tool useful in MNAzyme cascade reactions is the use of displacer oligonucleotides. Displacer oligonucleotides can bind to single stranded regions within a DNA complex which has both single stranded and double stranded regions. When the displacer binds the single stranded region of an oligonucleotide (e.g. Oligo 1) it can displace the strand or strands complementary to Oligo 1 (e.g. cOligol) within the double stranded regions provided the displacer has the same sequence as the cOligo 1 within the double stranded regions. This can result in a new double stranded structure being formed between the displacer oligonucleotide and Oligo 1 and can cause the release of cOligo 1 as a single stranded oligonucleotide.<br>
[ 00303 ] This strategy could be useful in MNAzyme cascade reactions. For example, a ligated product bound to an MNAzyme ligase has several non paired bases (a single stranded region) between the double stranded regions formed by complementarity between the substrate arms of the partzymes and the substrates. If a displacer oligonucleotide with complementarity to the substrate (Oligo 1) binds to the single stranded region of the substrate it could displace the complementary partzyme sensor arms (e.g. cOligo la and cOligo lb) by branch migration and separate the partzymes fix}m<br><br>
the iigated product. As such, this could increase the turnover of the ligase enzyme. The process would result in a double stranded molecule comprising the displacer and the Iigated product. However if the Iigated product were required for a subsequent step in the reaction, for example, if it were to act as an assembly facilitator for a subsequent MNAzyme, then it would be necessary to separate the Iigated product from the displacer. If the displacer oligonucleotide was shorter than the Iigated product then this would result in single stranded regions at the termini of the duplex formed by the displacer oligonucleotide and the ligation product. In the next step, the partzyme sensor arm could bind to these single stranded regions and remove the displacer oligonucleotide by branch migration thus resulting in a new MNAzyme assembled with partzymes (which themselves would have functioned as displacer oligonucleotides in this instance) and Iigated product (now functioning as an assembly facilitator).<br>
[ 00304 ] Many variations on the strategy are useful in cascades. For example, with reference to Figure 4 (left hand panel) a displacer could be used to remove the activity inhibitor by binding to the activity inhibitor domain of the activity inhibitor and displace it by branch migration such that the partzyme sensor arm would be rendered single stranded and thus available for binding of an assembly facilitator component.<br>
[ 00305 ] MNAzyme cascade reactions could be applied to a range of biotechnological applications, especially in diagnostics, but also in other areas such as medical, veterinary, agricultural, food technology, imaging and anti-bioterrorism applications. They could allow detection of proteins and nucleic acids for disease diagnosis, especially so if they facilitate signal amplification. Catalytic nucleic acids and/or cascade reactions can be used for applications other than diagnostics, for example, within the field of computation analysis and biomolecular engineering of nano-scale devices and molecular switches which may be used in therapeutics.<br>
[ 00306 ] MNAzymes offer advantages over DNAzymes for use in various cascades. DNAzymes can only recognize one molecule which is both its target and the substrate. While the DNAzyme tar^et/substrate(s) can be considered to provide the "input" and the enzyme product (modified target/substrate(s)) can be considered the "output" the DNAzyme is limited in its ability to transduce more complex processes in cascades since<br><br>
both the input and output are merely alternate (modified and unmodified) forms of the same target substrate(s) molecule(s). Further, catalytic modification, for example cleavage, can destroy the DNAzyme's target/substrate and as such the target is not available for another round of recognition. In contrast a single MNAzyme, for example an MNAzyme with cleavage activity, can recognize at least two molecules, namely at least one assembly facilitator and at least one substrate. An assetnbly facilitator, for example a target nucleic acid present in a sample, would not be catalytically modified in the reaction and would remain available for ftirther rounds of substrate modification by the MNAzyme. Further, a single MNAzyme which can act on two substrates (e.g. an MNAzyme with ligase activity) can recognize at least three molecules, namely at least one assembly facilitator and at least two substrates. These properties of MNAzyme complexes make them useful in processes such as in cascade reactions which may involve input (e.g. a molecule which permits a nucleic acid complex to function as a nucleic acid enzyme) and output (e.g. a molecule which is produced and permits a subsequent nucleic acid complex to function as a nucleic acid enzyme). Further, these properties of MNAzyme complexes make them highly suited to use in processes that require the multiple components to be able to "read" (recognise) an "input" signal (e.g. a molecule which may be a target assembly facilitator or an MNA complex component) and "write" an "output" signal (e.g. to recognise an input and act on a molecule to provide an output; e.g. to recognise an input and modify a substrate to create a new reaction product useful in a subsequent step). The properties of MNAzymes and MNAzyme complexes are useful in one step processes (e.g. detection of a target) and also in, for example, systems employing cascades, logic gates and other processes. MNAzyme complexes provide a mechanism for transduction of complex information, for example, to receive an input signal and respond with an appropriate output response. As highlighted throughout this specification, DNAzymes may still be useful in cascades. Some examples include production of a DNAzyme or a substrate for a DNAzyme complex by a preceding nucleic acid enzyme such as an MNAzyme. MNAzyme ligases are particularly useful in making, for example, new DNAzymes. As such the "output" information of one nucleic acid enzyme generated by, for example ligation or cleavage by an MNAzyme can provide an "input" for the next nucleic acid enzyme complex for<br><br>
example by creating a new DNAzyme or substrate, which can participate in the formation of a DNAzyme complex useful in a following, and /or preceding reaction stage.<br>
[ 00307 ] An exemplary mechanism for generating a linear cascade using an MNAzyme and a nucleic acid enzyme with ligase activity in a format designed for target detection is depicted in Figure 9. This figure also describes a general method for target detection. This method relies on a DNAzyme with ligase activity (DNAzyme ligase) to control the assembly of a partzyme for one MNAzyme from at least one of the products of the catalytic activity of one other MNAzyme.<br>
[ 00308 ] The strategy of Figure 9 uses.a combination of MNAzymes with cleavage activity and a DNAzyme with ligase activity. In step (i) an assembly facilitator, for example a target nucleic acid, can direct the formation of a first MNAzyme A. MNAzyme A can cleave a substrate A thus generating a S'cleavage product A which can in turn be used as a substrate for a DNAzyme ligase B. In step (ii) a DNAzyme ligase B can ligate the S' cleavage product A (also referred to as 5' substrate B) generated in step (i) to another oligonucleotide, 3' ligation substrate B, thus creating a new partzyme for an MNAzyme C. In step (iii) the new partzyme/ligation product generated in step (ii) together with another partzyme can form a new MNAzyme C in the presence of assembly facilitator C which can cleave a substrate C into two products, 5'product C and 3' product C. A detectable signal can be generated following cleavage of substrate C if this substrate were labelled, for example, with a fluorophore and quencher dye pair. This cascade provides a mechanism for signal amplification following initiation of a reaction by an assembly facilitator (eg a target analyte) which allows assembly of MNAzyme A. In this Imear cascade format, each step employs a nucleic acid enzyme, some or all of which may be undergoing multiple turnover whereby a single nucleic acid enzyme (e.g. a DNAzyme or MNAzyme) may perform muhiple cleavages, ligations or other modifications at one or more steps in the cascade. Multiple turnover may facilitate amplification at one or more steps in the cascade resulting in amplification of the signal detected during one or more steps.<br><br>
[ 00309 ] Another mechanism for generating an MNAzyme cascade in a format designed for target detection is depicted in Figure 12. This figure also describes a general method for target detection. This method relies on a combination of MNAzymes with ligase and cleavage activity. The MNAzyme ligase controls the assembly of a partzyme for one MNAzyme cleaver from at least one of the products of the catalytic activity of one other MNAzyme cleaver.<br>
[ 00310 ] In step (i) an assembly facilitator A, for example a target nucleic acid, can direct the formation of a first MNAzyme A. MNAzyme A can cleave a substrate A thus generating a 5'cleavage product A which can in turn be used as a substrate for a MNAzyme B which has ligase activity. In step (ii) an MNAzyme ligase B (formed by the partzyme components'of MNAzyme B in the presence of assembly facilitator B) can ligate the 5' cleavage product A (also referred to as S' substrate B) generated in step (i) to another oligonucleotide, 3' substrate B, thus creating a new partzyme for an MNAzyme C. In step (iii) the new partzyme/ligation product generated in step (ii) together with another partzyme can form a new MNAzyme C in the presence of assembly facilitator C which can cleave a substrate C into two products, 5'product C and 3' product C. A detectable signal can be generated following cleavage of substrate C if this substrate were labelled, for example, with a fluorophore and quencher dye pair.<br>
[ 00311 ] Further in the context of Figures 9 and 12 if the sequence of 5' product C were the same as 5' product A then this product could also serve as a 5' substrate B and a feedback amplification cascade could be initiated. In this reaction, MNAzyme C would constantly generate a 5' substrate B which could in turn be ligated by the nucleic acid enzyme ligase (DNAzyme B in Figure 9 or MNAzyme B in Figure 12) to create more partzymes for formation of more MNAzyme C. This strategy could provide a mechanism for feedback signal amplification following initiation of a reaction by an assembly facilitator (eg a target analyte) which allows assembly of MNAzyme A. These feedback cascade strategies could allow detection of assembly facilitators followed by signal amplification using multiple types of nucleic acid enzymes, for example, DNAzyme ligases or MNAzyme ligases in combination with MNAzymes which can cleave a substrate (MNAzyme cleavers). It will be recognised that this kind of feedback cascade could be used in combination with other strategies described throughout the specification.<br><br>
It would be appreciated that there are many variations that could be applied to cascades which follow a strategy similar to those demonstrated in figures 9 and 12. For example, the ligase could be used to create ligation products which serve as for example, assembly facilitators for another MNAzyme with cleavage, ligase or other enzymatic activity.<br>
[ 00312 ] An example of one possible cascade following a strategy similar to Figures 9 and 12 would be where two or more MNAzymes are responsive to different assembly facilitators (e.g. targets) and could be used in a reaction, with each modification reaction catalysed by each MNAzyme providing a different modification product (e.g. a cleavage or ligation product) which could be used to participate in a following reaction. By way of example, with reference to the cascades of Figures 9 and 12, two or more initial MNAzymes responsive to different assembly facilitators (e.g. targets) could be used to provide different modified products which could participate in forming a corresponding number of MNA ligase complexes that can function as nucleic acid enzymes with ligase activity in the next step of the reaction. The nucleic acid ligases could then make various elements for the MNAzyme C reaction, such as an assembly facilitator C, a partzyme A, B, or components thereof So, for example, one ligase reaction driven by the product of one initial MNAzyme could make an assembly facilitator C for MNAzyme C and another ligase reaction driven by the product of another initial MNAzyme could make a partzyme for MNAzyme C. In such an example, the assay could be arranged such that only when both targets are present to drive both of the ligase reactions and hence make both the assembly facilitator and the partzyme, then MNAzyme C is assembled and cataytically active. This would be a mechanism for detection of multiple targets. Only in the presence of the required number of targets would all the required elements of MNAzyme C be available to modify the substrate C. In this example, the added elements of the reaction mix would be tailored according to how many MNAzyme C elements are made by virtue of recognition of the original targets. So, for example, if two targets were monitored and this in turn drives reactions which result in creation of an assembly facilitator C and partzyme A for MNAzyme C, then only the component partzyme B for MNAzyme C and the substrate C would need to be present to allow the formation of MNAzyme C complexes and the third step in the reaction to proceed.<br><br>
[ 00313 ] Similarly, one or more assembly facilitators (e.g. targets) could be detected and feed a number of different elements into one subsequent (e.g. ligation) step. For example, with reference to figure 9, one initial reaction could provide the 5' ligase substrate and another initial reaction could produce the 3' ligase substrate. Alternatively, in another example similar to Figure 12, one initial MNAzyme cleaver could produce the S' ligase substrate for MNAzyme B and one initial MNAzyme ligase could create a partzyme for MNAzyme ligase B. Only when both products are available to participate in the formation of a MNAzyme complex will the ligase substrates be modified and the cascade reaction continue. It will be understood that the different assembly facilitators (e.g. targets) used in such strategies may comprise different molecules or different sequences within one molecule, or a combination. For example, three different SNPs within one molecule could be detected by three different MNAzymes.<br>
[ 00314 ] It will be recognised that there are many other variations to this strategy. The general concept of using multiple targets to provide a number of different elements to a reaction which are all required for the reaction to proceed is applicable in many reactions and cascades. The number of targets that are detected can vary e.g. two or more targets may be detected. It could be applied to one or many steps of a cascade. It could also be applied "directly" by providing a component or components for the next step in the cascade, or "indirectly" by allowing a reaction later in the cascade to occur, or various combinations thereof. So, for example, one MNAzyme directed by one target could provide a molecule required for the next reaction and another MNAzyme directed by another target could provide a product required for the next or other following reaction. It would be obvious that other variations may also apply. It would be obvious that this strategy can also be applied to enzymes which effect different modifications (e.g cleavage or ligation) as described above. In addition various combinations of products made by a varying number of nucleic acid enzymes responding to various numbers of targets are possible, which include but are not limited to, partzymes, assembly facilitators and other MNA complex components. A number of nucleic acid enzymes responding to a number of targets may produce one or more components of an additional nucleic acid enzyme or nucleic acid enzyme complex or may produce all the components of an additional nucleic acid enzyme or nucleic acid enzyme complex.<br><br>
[00315 ] Another example of a cascade reaction which uses nucleic acid enzymes which can cleave substrates, together with nucleic acid enzymes which can ligate substrates is illustrated in Figure 15. In this example the process of ligation results in the generation of a new assembly facilitator. In step (1) the MNAzyme cleaver 1 assembles only in the presence of an assembly facilitator (e.g. a target nucleic acid) and then cleaves MNAzyme substrate 1 releasing a 5' cleavage product with a 2\ 3' cyclic phosphate tenninus. In step (2) a DNAzyme ligase (or an MNAzyme ligase) ligates the 5' cleavage product from step (1) to a 3' ligation substrate thus creating a ligation product which can serve as an assembly facilitator for another MNAzyme cleaver 2. In step (3) the assembly facilitator formed by ligation in step (2) directs the assembly of partzymes which form MNAzyme cleaver 2 which can cleave substrate 2 into two products, a 5' cleavage product with a 2', 3' cyclic phosphate terminus and 3' cleavage product. A detectable signal can be generated following cleavage of substrate 2 if this substrate were labelled, for example, with a fluorophore and quencher dye pair. Similar to the linear versions of the cascades illustrated in Figure 9 and 12, each step in the cascade in Figure IS employs a nucleic acid enzyme, some or all of which may be undergoing multiple turnover whereby a single nucleic acid enzyme (e.g. a DNAzyme or MNAzyme) may perform multiple cleavage or ligation or other modifications at one or more steps in the cascade. Multiple turnover may facilitate amplification at one or more steps in the cascade resulting in amplification of the signal detected during one or more step(s).<br>
[ 00316 ] Further, an extension of the linear cascade illustrated in Figure 15 could result in a feedback amplification cascade. If the sequence of the S' cleavage product of MNAzyme cleaver 2 were the same as the 5' cleavage product of MNAzyme cleaver 1 then the 5' cleavage product of MNAzyme cleaver 2 could also serve as a 5' substrate for the DNAzyme ligase (or MNAzyme ligase) and a feedback amplification cascade reaction could be initiated. In this reaction MNAzyme cleaver 2 would constantly generate S' cleavage products which could in turn serve as a substrate for ligation by the DNAzyme ligase (or MNAzyme ligase) thus allowing the creation of more assembly facilitators that could direct the assembly of more MNAzyme cleaver 2. It is envisaged this strategy could provide a mechanism for feedback signal amplification following initiation of a reaction by an assembly facilitator (eg a target analyte) which could allow<br><br>
assembly of MNAzyme cleaver 1. The strategy could allow detection of assembly facilitators (eg target analytes) followed by signal amplification using a DNAzyme (or MNAzyme) which can ligate substrates and MNAzymes which can cleave a substrate. Other variations of this schema are described throughout the specifications.<br>
[ 00317 ] Another exemplary method for generating a feedback cascade reaction using two types of nucleic acid enzymes namely nucleic acid enzymes which can cleave ' substrates, and nucleic acid enzymes which can ligate substrates is proposed in Figure 16. Figure 16 is but one of a number of variations of this general strategy. In step (1) the MNAzyme cleaver 1 assembles only in the presence of an assembly facilitator (e.g. a target nucleic acid) and then cleaves MNAzyme substrate 1 releasing a S' cleavage product with a 2\ 3' cyclic phosphate terminus and a 3' waste product. In step (2) the DNAzyme ligase (or MNAzyme ligase) ligates the 5' cleavage product from step (1) to a 3' ligation substrate thus creating a ligation product which can serve as an assembly facilitator for an MNAzyme cleaver 2. In step (3) the assembly facilitator formed by ligation in step (2) directs the assembly of partzymes to form MNAzyme cleaver 2 which can cleave substrate 2 into two products, a S' cleavage product with a 2', 3' cyclic phosphate terminus and 3' cleavage product. A detectable signal can be generated following cleavage of substrate 2 if this substrate were labelled, for example, with a fluorophore and quencher dye pair. Further, if the sequence of one of the cleavage products of MNAzyme cleaver 2 were useful as an assembly facilitator component for an MNAzyme cleaver 3 this could facilitate a feedback cascade. In this case, MNAzyme cleaver 3 may have different sensor arms than MNAzyme cleaver 1 but would require at least one substrate arm in common with MNAzyme cleaver 1 in order to provide at least the same S' ligation substrate to participate in the ligase reaction of step (it). (In Figure 16, given it is the same substrate as MNAzyme cleaver 1 (substrate 1), the substrate arms will be the same for MNAzyme cleavers 1 and 3.) In step 4 an MNAzjone cleaver 3 could assemble in the presence of an assembly facilitator component generated by cleavage by the MNAzyme cleaver 2. Further, MNAzyme cleaver 3 could cleave a substrate and generate a S* cleavage product which could be the same as the 5' cleavage product produced by cleavage of substrate 1 by MNAzyme cleaver 1. In turn, this product could also serve as a 5' substrate for the DNAzyme (or MNAzyme) ligase and a feedback<br><br>
amplification cascade reaction could be initiated. In this reaction MNAzyme cleaver 3 would be constantly generating S' cleavage product which in turn could serve as a substrate for ligation by the DNAzyme (or MNAzyme) ligase to create more assembly facilitators that could direct the assembly of more MNAzyme cleaver 2. MNAzyme cleaver 2 could then produce more assembly facilitator components for more MNAzyme cleaver 3 and a feedback cascade would be created. This strategy could provide a mechanism for feedback signal amplification following initiation of a reaction by an assembly facilitator (eg a target analyte) which directs assembly of MNAzyme cleaver 1. The strategy would allow detection of assembly facilitators (eg target analytes) followed by signal amplification using a DNAzyme (or MNAzyme) which could ligate substrates and MNAzymes which could cleave substrates.<br>
[ 00318] It will be imderstood that while a number of the above examples refer to use of a S' cleavage product as a ligation substrate in a subsequent reaction, that it would also be possible to use a 3' cleavage product as a ligation substrate in a subsequent reaction, or both. Additionally, either a S' or 3' cleavage product, or both, could be used as a component of a subsequent or preceding nucleic acid enzyme complex. It would also be understood that ligatable substrates may have different termini depending on the ligase used. For example, in addition to 2'3' cyclic phosphate and 5' hydroxyl termini, they may have, for example, 2'3' diol termini and 5' triphosphate termini (see Table 4). MNAzymes with cleavage activity can be selected based on the linkage to be cleaved.<br>
[00319] In further variations of this and other examples in the specification, the DNAzyme (or MNAzyme) with ligase activity could ligate fragments and form a DNAzyme capable of cleaving substrate 2 to create an assembly facilitator or other component for another MNAzyme (labelled as MNAzyme cleaver 3 in Figure 16). As such, in this variation MNAzyme cleaver 2 could be replaced with a DNAzyme cleaver. In another exemplary strategy the DNAzjmie (or MNAzyme) ligase could ligate fragments and create a new partzyme capable of associating with another partzyme and an assembly facilitator present in the reaction thus forming an MNAzyme cleaver 2 which could cleave substrate 2 to create an assembly facilitator component for an MNAzyme 3. Other variations on this scheme are possible and are described elsewhere in the specification.<br><br>
[ 00320 ] The skilled addressee will appreciate that a number of alternative MNAzyme complexes are possible and may be used in cascades. Several of these designs are illustrated in Figure 6 where some examples of active MNAzyme complexes are shown in Panels A to D. These structures are all capable of forming catalytically active enzymes, which can modify (e.g. cleave) substrate (S). The illustration contains schemes for MNAzymes, which include one assembly facilitator Fl (panel A), two assembly facilitator components FI and F2 (panels B and D) or three assembly facilitator components Fl, F2 and F3 (panel C). The example shown in panel A includes sensor arms with self complementary regions within the partzyme sensor arms. MNAzymes may also include one or more stabilising arms (sA) as shown in panel D. Variations in one panel could be used in combination with a variation in another panel.<br>
[ 00321 ] The skilled addressee will appreciate that a number of cascades are demonstrated throughout the specification. It will albo be appreciated that the cascades are modular in nature and thus numerous variations to the cascades are possible. Table 3, below provides a non-exhaustive list of possible variations. The skilled addressee will realise that other variations are possible and are included within the scope of the invention.<br>
Table 3. Some possible cascades using combinations of MNAzymes or MNAzymes and<br>
DNAzymes.	<br>
*Step       Enzyme      Function<br>
Assay set up        1	MNAzyme     Detect assembly facilitator (eg target) and<br>
variation 1	cleaver 1      cleave a substrate to create a product<br>
(e.g. Example	useful as a substrate in step 2.	<br>
6, Figure 9;	2	MNAzyme    Use the product fiom step 1 as a substrate,<br>
Example 7 and	or	which can be ligated to a second ligation<br>
9, Figure 12)	DNAzyme     substrate to create a new partzyme for an<br>
	ligase	MNAzyme cleaver 2 required for step 3.<br>
3             MNAzyme    Use the partzyme/ligation product from<br>
cleaver 2      step 2 as a component for an MNAzyme<br>
	cleaver 2 which can cleave a substrate.<br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br>
[ 00323 ] All reactions can use a combination of nucleic acid enzymes, namely multiple MNAzymes or a mix of MNAzymes and DNAzyme(s). The modifications by these enzymes involve a combination of various numbers of cleavage and/or ligation steps. In each case, the product of one step serves as a substrate, an enzyme or an enzyme component for the subsequent step. Nucleic acid enzymes which cleave can be used to create, for example, a new substrate for ligation or a new assembly facilitator component for a new MNAzyme. Nucleic acid enzymes which ligate can be used to create, for example, a new substrate for cleavage in a next step by either a DNAzyme or an MNAzyme enzyme. Further, nucleic acid enzymes which ligate can be used to create, for example, a new DNAzyme or a new component for an MNAzyme, such as an assembly facilitator, a new partzyme or a new stabiliser arm. One skilled in the art will realise that there are a large number of variations on assays which use combinations of nucleic acid enzymes and resultant products which can be used in nucleic acid enzyme cascades. For example, in some of the variations listed in Table 3 where a nucleic acid enzyme is indicated as a cleaver it could be possible to replace this with a ligase (or vice versa). Numerous individual steps useful in a number of such cascade reactions have been demonstrated in the examples and summarised in Scheme 1.<br>
[ 00324 } It will be obvious to one skilled in the art that multiple products produced by a single nucleic acid enzyme could be used to permit one or more nucleic acid complex(es) to function as one or more nucleic acid enzyme(s) in one or more subsequent steps.<br>
[ 0032S ] In some of the steps in Table 3 where the product of one reaction is indicated as being useful as a particular component of a nucleic acid enzyme complex in a subsequent reaction, it could be possible to replace the component with another type of component. For example, in step 1 of Table 3 an MNAzyme could create a product that is useful as an assembly facilitator, rather than as a substrate, for assembly of a nucleic acid enzyme complex.<br>
[ 00326 ] It will be obvious to one skilled in the art that a signal, for example a fluorescent signal can be generated at a step where a catalytic modification occurs. For<br><br>
example, a fluorescent signal could be generated at any step where there is cleavage by a DNAzyme or MNAzyme one could separate a fluorophore quencher dye pair and cause an increase in fluorescence. Further, a fluorescent signal could be generated at any step where there is, for example, ligation by a DNAzyme or MNAzyme one could bring a fluorophore quencher dye pair into close proximity and cause a decrease in fluorescence. Further, a fluorescent signal could be generated at multiple steps within a cascade reaction.<br>
[ 00327 ] In all these examples of cascade reactions there is a succession of processes or operations that occur in successive stages, wherein the occurrence of each stage is typically dependent on the occurrence of a preceding stage. Further in examples whereby feedback cascades are described there would be a succession of processes or operations that occur in successive stages, whereby the occurrence of each stage is typically dependent on the occurrence of a preceding stage and whereby the occurrence of at least one preceding stage typically depends on processes or operations that occur in subsequent stages. In feedback cascades, the product of any preceding stage could serve as a component or substrate for any subsequent stage and the product of any subsequent stage could serve as a component or substrate for any preceding stage.<br>
[00328] A cascade could be initiated by nucleic acid targets (DNA/RNA) in the presence of components for one or more MNAzyme(s), or a cascade could be initiated by other target analytes (proteins, small molecules etc) in the presence of components for one or more apta-MNAzyme(s)s. Since the reaction would only be initiated in the presence of target analytes, it could provide a technique for detection and/or identiflcation of target analytes. The method is based on the transition between states of inactivation and activation of nucleic acid complexes. Unlike target amplification techniques such as the PCR, or the ligase chain reaction, cascades involving involving amplification using MNAzyme(s) require no protein enzymes to facilitate the process. Further, the ability to control and regulate the catalytic activity of MNAzyme(s) using several oligonucleotide components, such as stabiliser arm components or assembly facilitator components allows the assembly of MNAzyraes to be tightly regulated by conditions and components in the microenvironment.<br><br>
[ 00329 ] In relation to cascades initiated by non-nucleic acid target analytes aptamers are uscfijl. Aptamers are DNA, UNA or peptide sequences that have the ability to recognize one or more ligands with great affinity and specificity due to their high level structure, for example, a 3-D binding domain or pocket. Many aptamers have been evolved in vitro for their capacity to bind to ligands, including for example, nucleic acids, proteins, prions, small organic compounds, and/or entire organisms. "Aptazymes" are sequences which have been previously described which have sequences comprised of both aptamer and nucleic acid enzyme sequences, for example an aptamer appended to a DNAzyme or ribozyme. Apta-MNAi^ymes are MNAzymes which have an aptamer appended to one of the components in an apta- MNAcomplex. For example an aptamer may be appended to a partzyme, an assembly facilitator, an MNAzyme substrate or any other component of an MNA complex. Association of a non-nucleic acid target analyte with the aptamer may facilitate activity of the nucleic acid enzyme. Thus a cascade may be initiated by a non-nucleic acid target analyte associating with an apta-NfNAzyme, for example, as the first step in a cascade. Furthermore, in a cascade, the product of a reaction catalysed by a nucleic acid enzyme in that cascade may associate with an aptamer present on a component of a nucleic acid enzyme other than the initiating MNAzyme or apta-MNAzyme, thereby facilitating activity of that nucleic acid enzyme.<br>
[ 00330 ] MNAzyme and/or apta-MNAzyme mediated amplification cascades could be applied to a range of biotechnology applications, especially in diagnostics, but also in other medical, veterinary, agricultural, food technology, imaging and anti-bioterrorism applications. They could allow detection of proteins, nucleic acids or other molecules for disease diagnosis by facilitating signal amplification. Catalytic nucleic acids and/or cascade reactions can be used for applications other than diagnostics, for example, within the field of computational analysis and biomolecular engineering of nanoscale devices and switches which may be used in therapeutics.<br><br>
9.	Method which allows switching between active and inactive states of MNA<br>
complexes by removal or creation of components such as the activity inhibitor or the assembly facilitator,<br>
[ 00331 ] A transition between active MNAzymes and inactive MNA complexes, or vice versa, can be achieved by the provision or removal of MNA complex components, including but not limited to, one or more activity inhibitors, assembly facilitators, assembly inhibitors, partzymes or stabiliser arms, or parts thereof. In some embodiments, the activity inhibitor may further include a labile or cleavable linker or substrate, which may be located between two or more domains within the activity inhibitor, for example an activity inhibitor domain and an activator assembly facilitator domain. Cleavage at the linker site may allow separation of an activity inhibitor domain from an activator assembly inhibitor domain, which may then function as an assembly facilitator component and direct the assembly of an active MNAzyme. Cleavage of the linker could be achieved by several methods, including but not limited to, MNAzyme cleavage, protein enzyme cleavage, or hydrolysis induced by changes in the pH and or temperature.<br>
[00332] Alternatively the assembly inhibitor and/or assembly facilitator could be selectively removed using a process involving branch migration and/or complementarity to component oligonucleotides. Modulator oligonucleotides which function through complementarity may do so by altering the secondary structure of oligonucleotides to which they bind. In some embodiments this may result in an alternate conformation where an activator sequence is now able to assemble with other components to form active MNAzymes. By way of example, such a modulator oligonucleotide may cause the disruption of intramolecular structures such as hairpins which constrain activator molecules in non-fimctional conformations.<br>
[ 00333 ] An alternate exemplary strategy is designed to facilitate switching an MNA complex between the "on state" of an active MNAzyme to the "off state" of an MNAi complex using a second MNAzyme with ligase activity (or a DNAzyme with ligase activity). With reference to Figure 13, an active MNAzyme A, which could be capable of modifying (eg cleaving or ligating) a substrate(s) A could be formed in the presence of assembly facilitator component 1 (AFC 1) and assembly facilitator component 2 (AFC<br><br>
2). A second MNAzyme B, which has Hgase activity, could form in the presence of assembly facilitator 3 (AF3) and could then ligate AFC 2 with an activity inhibitor component (AIC) causing the formation of an activity inhibitor (AI). This AI could bind to the partzyme components for MNAzyme A and resuJt in the formation of an MNAi A complex which is inactive. As such the MNAzyme Hgase in this example could operate as an off switch to inactivate an MNAzyme A.. Alternatively a DNAzyme ligase could replace the MNAzyme ligase and be used to operate as an off switch to inactivate an MNAzyme A. The inactive MNAi complex and the catalytically active MNAzyme would represent two alternate states for the assembled components, namely an "off' state and the "on" state respectively.<br>
10.	Kits<br>
[ 00334 ] The present invention also provides kits for practising the methods disclosed herein. Typically, kits for carrying out the methods of the present invention contain ail the necessary reagents to carry out the method. Typically, kits may comprise at least a first container containing a first nucleic acid complex (e.g. an MNA complex), wherein formation of a first nucleic acid enzyme requires the addition of a sample putatively containing a target to be detected, identified and/or quantitated. Formation of the enzyme may lead to the production of a component, for example a substrate, suitable for participation in a subsequent reaction. In the subsequent reaction the component allows a further nucleic acid complex to function as a nucleic acid enzyme. Reagents to allow formation of further nucleic acid complexes which form nucleic acid enzyme complexes when contacted by a component produced by a nucleic acid enzyme of a previous reaction may also be provided.<br>
[ 0033S ] In certain embodiments components for each of the nucleic acid complexes which can allow formation of active nucleic acid enzyme complexes may be present in the same or separate containers. In other embodiments components for each of the nucleic acid complexes which can allow formation of active nucleic acid enzyme complexes may be present in separate containers. For example, in one embodiment a kit may comprise one or more required components for each of the nucleic acid complexes in separate containers.<br><br>
[ 00336 ] It will be understood that not all components for all nucleic acid complexes are required to be in the kit as they may be generated as part of a cascade reaction. Components which allow multiplex reactions are also envisaged.<br>
[ 00337 ] In other embodiments components for additional nucleic acid enzyme complexes with, for example, either cleavage or ligase activity may also form part of the kits of the present invention. In yet other embodiments the kits of the present invention may include DNAzymes or parts thereof.<br>
[ 00338 ] Typically, the kits of the present invention will also comprise one or more other containers, containing for example, wash reagents, and/or other reagents as required in the performance of the methods of the invention.<br>
[ 00339 ] In the context of the present invention, a compartmentalised kit includes any kit in which reagents are contained in separate containers, and may include small glass containers, plastic containers or strips of plastic or paper. Such containers may allow the eflicient transfer of reagents from one compartment to another compartment whilst avoiding cross-contamination of the samples and reagents, and the addition of agents or solutions of each container from one compartment to another in a quantitative fashion. Such kits may also include a container which will accept the test sample, a container which contains the reagents used in the assay, containers which contain wash reagents, and containers which contain a detection reagent. Typically, a kit of the present invention will also include instructions for using the kit components to conduct the appropriate methods. Kits and methods of the invention may be used in conjunction with automated analysis equipment and systems, for example, including but not limited to, real time PCR machines.<br>
[ 00340 ] For application to detection, identification or quantitation of different targets or events, a single kit of the invention may be applicable, or alternatively different kits, for example containing reagents specific for each target, may be required. Methods and kits of the present invention find application in any circumstance in which it is desirable to detect, identify or quantitate any entity or event.<br><br>
/ /.	Methods of making MNAzymes<br>
[ 00341 ] The present invention also provides methods for engineering MNAzymes from uni-molecular DNAzymes.<br>
I 00342 ] DNAzymes and/or ribozymes are known which are capable of catalytic modification of at least one substrate wherein the modification may be selected from the group comprising cleavage of nucleic acids, ligation of nucleic acids, phosphorylation of nucleic acids, nucleic acid capping, amino acid adenylation, cofactor synthesis, RNA polymerization, template-directed polymerization, RNA-protein conjugation, aldol reaction, alcohol oxidation, aldehyde reduction, purine and pyrimidine nucleotide synthesis, alkylation, amide synthesis, urea synthesis, formation of peptide bonds, peptidyl-RNA synthesis, acyl transfer, aminoacylation, carbonate hydrolysis, phosphorothioate alkylation, porphyrin metallation, formation of catbon-carbon bonds, Pd nanoparticle formation, biphenyl isomerization, formation of ester bonds, formation of amide bonds, DNA deglycosylation, thymine dimer photoreversion and phosphoramidate cleavage<br>
[ 00343 ] The methods of making MNAzymes described herein are applicable to a plurality of nucleic acid enzymes such as those above and including but not limited to those described in Hobaitner and Silverman (2007) and those listed in the table below.<br>
Table 4: Some uni-molecular nucleic acids enzymes which may be used to form an MNAzyme.<br>
Reaction   Specific nucleic       Substrate	Product<br>
catalysed   acid enzyme (&amp;       requirements primary<br>
	composition)	<br>
Cleavage   hammeihead	NU*X	2'3' cyclic phosphate<br>
	ribozyme (RNA)       3'-5' RNA linkage	and 5' hydroxyl	<br>
10:23 DNAzyme      R*Y	2'3' cyclic phosphate<br>
	 (DNA)	3'-S' RNA linkage	and 5' hydroxyl<br>
8:17 DNAzyme        A*G	2'3'cyclic phosphate<br>
	I (DNA)	I 3'-5' RNA linkage         | and 5' hydroxyl<br><br>
7Q10 DNAzyme      UA*GR	2'3' cyclic phosphate<br>
	(DNA)	 2'-5' RNA linkage	and 5'hydroxyl<br>
7Z81 DNAzyme       UA*GR	2'3'cyclic phosphate<br>
	 (DNA)	 2'-5' RNA linkage	and 5'hydroxyl<br>
Ligation     7Q10 DNAzyme      UA*GR	2'-5' RNA linkage<br>
(DNA)	2'3'cyclic phosphate<br>
	and 5' hydroxyl	<br>
7Z81 DNAzyme       UA*GR	2'-5' RNA linkage<br>
(DNA)	2'3' cyclic phosphate<br>
	and S' hydroxyl	<br>
9DB1 DNAzyme      D*RA	3'-5' RNA linkage<br>
(DNA)	2'3'dioland5'-<br>
	I	I triphosphate	|	<br>
[ 00344 ] The skilled addressee will recognise that many DNAzymes have similar basic structures with multiple domains. These DNAzymes have a conserved catalytic domain (catalytic core) flanked by two non-conserved substrate-binding domains ("substrate arms"), which specifically recognize and hybridise to the substrate. The substrate binding domain can be tailored to any substrate provided the substrate contains a site which can be modified by the DNAzyme. Several catalytic nucleic acids, including the hammerhead ribozyme, the 10:23 and 8:17 DNAzymes, the "7Z81", "7Z48" and "7Q10" ligases, the "UVIC" thymine dimer photoreversion DNAzyme and the "DAB22" carbon-carbon bond forming DNAzyme, have similar basic structures with multiple domains.<br>
[ 00345 ] With reference to Figure 14, a method for engineering an MNAzyme from a uni-molecular DNAzyme (top left hand side structure) is illustrated whereby in the first step, positions are identified within the DNAzyme catalytic core at which it can be split, such that each partial portion of the catalytic core can be distributed between two partial sequences such that the two partial cores together constitute an entire catalytic core (top right hand side structure). Two oligonucleotides A and B (candidate partzymes) can then be synthesised (bottom left hand side structure). An oligonucleotide A can be synthesised to contain (i) one substrate binding arm portion capable of binding to a substrate, (ii) one partial catalytic core portion, and (iii) one sensor arm portion capable of binding to an<br><br>
assembly facilitator molecule. A second oligonucleotide B can be synthesised such that it contains (i) one substrate binding arm capable of binding to the same substrate as oligonucleotide A, whereby oligonucleotide B binds the substrate in a position adjacent to that of oligonucleotide A, (ii) one partial catalytic core portion which contains those bases fh&gt;m the entire DNAzyme catalytic core which are not incorporated into oligonucleotide A and (iii) one sensor arm sequence capable of binding to the same assembly facilitator as oligonucleotide A, whereby oligonucleotide B binds the assembly facilitator in a position adjacent to that of oligonucleotide A. This process can be repeated thus making a series of pairs of oligonucleotides A and B which incorporate the structure and domains of partzymes, but may or may not have catalytic activity in the presence of a substrate and an assembly facilitator.<br>
[ 00346 ] Some or all of the candidate partzyme pairs (pairs of oligonucleotides A and B from the series) can then be incubated in the presence of the assembly facilitator complementary to the sensor arm portions of the partzymes and a substrate which is complementary to the substrate arm portions of the partzymes. Incubation is performed under conditions which are compatible with modification of a substrate by the DNAzyme from which the partial catalytic core sequences were derived. Pairs of oligonucleotides A and B, which can perform the same type of modification (e.g. cleavage) to the substrate as the DNAzyme from which partial sequences were derived, are useful as partzymes which can be assembled into MNAzymes and MNAzyme complexes (bottom right hand side structure). The sequences of the partial catalytic cores are suitable for incorporation into other partzymes of new MNAzymes. New partzymes can be synthesised which are tailored to new substrates (by changing the substrate binding domains of each partzyme) and/or to new assembly facilitators (by changing the sensor binding domains of each partzyme).<br>
[ 00347 ] The method for engineering an MNAzyme that acts on a first and second substrate e.g. a MNAzyme ligase starting from a uni-molecular nucleic acid enzyme (e.g.DNAzyme ligase) requires the similar steps whereby in the first step, positions are identified within the DNAzyme catalytic core at which it can be split, such that each partial portion of the catalytic core can be distributed between two partial sequences such that   the   two   partial   cores   together   constitute   an   entire   catalytic   core.   Two<br><br>
oligonucleotides A and B (candidate partzymes) can then be synthesised. An oligonucleotide A can be synthesised to contain (i) one substrate binding arm portion capable of binding to a first substrate, (ii) one partial catalytic core portion, and (iii) one sensor arm portion capable of binding to an assembly facilitator molecule. A second oligonucleotide B can be synthesised such that it contains (i) one substrate binding arm capable of binding to a second substrate, (ii) one partial catalytic core portion which contains those bases from the entire DNAzyme catalytic core which are not incorporated into oligonucleotide A and (iii) one sensor arm sequence capable of binding to the same assembly facilitator as oligonucleotide A, whereby oligonucleotide B binds the assembly facilitator in a position adjacent to that of oligonucleotide A. This process can be repeated thus making a series of pairs of oligonucleotides A and B which incorporate the structure and domains of partzymes, but may or may not have catalytic activity in the presence of the substrates and an assembly facilitator.<br>
[ 00348 ] Some or all of the candidate partzyme pairs (pairs of oligonucleotides A and B from the series) can then be incubated in the presence of the assembly faciUtator complementary to the sensor arm portions of the partzymes and the two substrates which are complementary to the substrate arm portions of the partzymes. Incubation is performed under conditions which are compatible with modification (e.g. ligation) of the substrates by the DNAzyme from which the partial catalytic core sequences were derived. Pairs of oligonucleotides A and B which can perform the same type of modification (e.g. ligation) to the substrates as the DNAzyme from which partial sequences were derived are useful as partzymes which can be assembled into MNAzymes and MNAzyme complexes. The sequences of the partial catalytic cores are suitable for incorporation into other partzymes for other MNAzymes. New partzymes can be synthesised which are tailored to new substrates (by changing the substrate binding domains of each partzyme) and/or to new assembly facilitators (by changing the sensor binding domains of each partzyme).<br>
[ 00349 ] Further, a similar method can be used to examine tolerance to changes in the partial sequence derived from splitting the DNAzyme catalytic core that can be incorporated into component partzymes. For example, modifications to the partial catalytic cores incorporated into partzymes can be made which include, by way of<br><br>
example, insertions, deletions, substitution with alternate DNA bases, or substitution with DNA analogues such as ribonucleotides, inosine or other analogue bases. Candidate partzyme pairs containing one or more modification(s) can then be assayed as above to determine whether or not they contain a pair of partial catalytic core sequences which are suitable for incorporation into partzymes that can assemble into catalytically active MNAzymes.<br>
[ 00350 ] The present invention will now be further described in greater detail by reference to the following speciflc examples, which should not be construed as in any way limiting the scope of the invention.<br>
EXAMPLES<br>
Example 1: Example of an MNAzyme where partzyme B comprises two molecules.<br>
[ 00351 ] Many variations on the basic design of an MNAzyme are contemplated in the present invention. In this example, MNAzymes were assembled in the presence of an assembly facilitator from partzyme A, and partzyme B which contained rwo components, namely one partzyme component with a truncated sensor arm, and one component that functions as a stabiliser arm. MNAzyme assembly occurs via Watson-Crick base recognition of the partzyme sensor arms and the assembly facilitator sequence. In the following example, the use of a truncated arm and stabiliser arm was demonstrated.<br>
[ 00352 ] The MNAzyme detection strategy used in this example is illustrated in Figure 2 (panel (ii)) and in Figure 3 (i).<br>
[ 00353 ] An example of the oligonucleotides required for this strategy are described below:<br>
a)	a partzyme A;<br>
b)	a partzyme B comprising a first component which contains a substrate arm, a partial catalytic core and truncated sensor arm; and a second stabiliser arm component, which hybridizes to the assembly' facilitator, adjacent to the truncated sensor arm of the partzyme. This stabiliser arm is designed to facilitate MNAzyme assembly when the truncated sensor arm of the partzyme is hybridized to the assembly facilitator, and<br>
c)	a substrate, for example a reporter substrate.<br><br>
[ 00354 ] Alternatively, oligonucleotides useful in this strategy are a partzyme B; a partzyme A comprising a first component which contains a substrate arm, a partial catalytic core and truncated sensor arm; and a second stabiliser arm component, which can hybridize to the assembly facilitator, adjacent to the truncated sensor arm of the partzyme.<br>
[ 003SS ] Active MNAzyme assembly also requires the presence of an assembly facilitator.<br>
1.1.	Partzyme oligonucleotides and stabiliser arm<br>
[ 00356 ] In this example, the truncated sensor arm of partzyme B was only 5 nucleotides in length. The sequences of partzymes A and the two partzyme B components are shown below (5' to 3'). In the following sequences the bases in bold hybridize with the assembly facilitator, bases underlined form part of the catalytic core of the assembled MNAzyme, and bases in italics hybridize to the substrate. The "-P" indicates 3' phosphorylation of the oligonucleotide.<br>
SEQ ID NO: 1 Partzyme A4 XdA4/2-P: ACTGGATQTCCATCTGTCTQACAACGAGAGGAAACCTT-P<br>
SEQ ID NO: 2 Partzyme B5 component 1 XdB5/2-P: TGCCCAGGGAGGCTAGCTTATAC-P<br>
SEQ ID NO: 3 Partzyme B stabiliser arm component XdS-P; CTTCOTGAGOOTGAG-P<br>
1.2.	Reporter substrate<br>
[ 00357 ] The reporter substrate used in this example was SubBi-2 end-labelled with a 6-FAM moiety at the 5' end, a BHQl moiety at the 3' end and designated SubBi-2-FB. Cleavage of SubBi-2-FB was monitored at 520 ran (FAM emission wavelength) with excitation at 490 nm (FAM excitation wavelength). The sequence of SubBi-2-FB is listed below (5' to 3'); the lower case bases represent RNA and the upper case bases represent DNA.<br>
SEQ ID NO: 4 SubBi-2-FB:<br><br>
AAGGTTTCCTCguCCCTGGGCA<br>
1.3.	Assembfy facilitator molecule<br>
[ 00358 ]   The sequence of the synthetic oligonucleotide used as the assembly facilitator<br>
is below (5' to 3').   This assembly facilitator was fully matched with the partzyme B<br>
sensor arm. Nuclease-free water was used in place of target as a "no target" control.<br>
SEQ	ID	NO:	5	Assembly	facilitator	Xd-T:<br>
TGCCCCCTCACCCTCACGAAGGTATACAGACAGATGGACATCCAGTTGGTGA<br>
1.4	Reaction Conditions<br>
[ 00359 ] Detection of the assembly facilitator was measured by an increase in fluorescent signal caused by cleavage of the reporter substrate by the catalytically active MNAzyme. Reactions were initiated by the addition of substrate and the total volume of all reactions was 50 \iL. Reactions were conducted at 55°C on a FLUOstar OPTIMA (BMG Biotech). Fluorescence for each reaction was read every 2 seconds for a total of 5 minutes. All reactions contained 200 nM SubBl-2-FB, 200 nM XdA4/2-P, 200 nM XdB5/2-P, 1 X PCR Buffer II (Applied Biosystem) and 25 mM MgCb. In addition, reactions contained oligonucleotides as listed in Table 5.<br>
Table 5. Additional reagents in MNAzyme reactions.<br>
Reaction     Assembly Facilitator	Stabiliser arm<br>
(i)	200 nM of Xd-T	200nMofXdS-P<br>
(ii)	200 nM of Xd-T	No stabilizer aim<br>
(iii)             No assembly facilitator (water    200 nM of XdS-P<br>
	I control)	I	<br>
1.5	Results: Assembfy of active MNAzymes in the presence of the partzymes and an<br>
assembly facilitator,<br>
[ 00360} When both the assembly facilitator, and a partzyme stabiliser arm component were included in the reaction (Reaction (i): Figure 3), active MNAzymes assembled and cleaved the substrate, resulting in an increase in fluorescence over time. In contrast, there was no increase in fluorescence in the absence of the assembly facilitator (Reaction (iii): Figure 3).   Further, the presence of the stabiliser arm was shown to be essential for<br><br>
formation of active MNAzymes or MNAzyme complexes. A reaction containing all reaction components including the assembly facilitator, but which lacked the stabiliser arm component, gave no increase in fluorescence over time (Reaction (ii): Figure 3). As such, the 5 bases of the sensor arm of partzyme B were insufficient to form a stable MNAzyme or MNAzyme complex but the presence of a stabiliser arm component was shown to be capable of compensating for the short length (truncation) of the partzyme sensor arm and allowing stable MNAzyme formation under stringent temperature conditions (SS^C in this example). The stabiliser arm component is thus an essential oligonucleotide for assembly of active MNAzymes in this system, which uses a partzyme with a truncated sensor arm.<br>
[00361] Further, when an alternative assembly facilitator, which had a single nucleotide mismatch with respect to tfie partzyme A sensor arm was included in a reaction containing partzyme A and the two components of partzyme B, the fluorescent signal did not increase over time (data not shown).<br>
[ 00362 ] This example demonstrates that MNAzymes could only form in the presence of a fully matched assembly facilitator under the conditions of the experiment. One skilled in the art will appreciate that transition between an active MNAzyme and an inactive multi-component nucleic acid complex can be regulated by providing fully matched or mismatched assembly facilitators. Further, the example demonstrates the use of two-component partzymes, which comprise a first molecule that contains truncated sensor arms, and a second stabiliser arm molecule. The requirement for the presence of a stabiliser arm molecule in such systems, provides another tool with which one can regulate the assembly of MNAzymes.<br>
[ 00363 ] Great flexibility is afforded by MNA complexs which contain multiple oligonucleotide components, the sequences of which can be tailored with respect to the melting temperature, the sequence composition and complementarity or lack thereof with other component oligonucleotides. Shorter sequences, including but not limited to, partzyme components with truncated arms, stabilizer arms, and assembly facilitator components are particularly useful in this aspect of the invention. While this example uses an MNAzyme with cleavage activity, it would be appreciated that similar structural<br><br>
designs for MNAzymes or MNAzyme complexes could be used in MNAzymes which cause modifications other than cleavage, for example, MNAzyme which could ligate.<br>
Example 2: Detection of target analytes using MNAzymes with ligase activity.<br>
[ 00364 ] An example of an MNAzyme which can modify at least two substrates, such as an MNAzyme with ligase activity could be used for detection of a nucleic acid target. An MNAzyme with ligase activity could be assembled from two partzymes which comprise active partial catalytic core pairs derived from, for example, the 7Q10 DNAzyme or the 7Z81 DNAzyme as exemplified in example 7 and 9 respectively. The partzymes could have sensor arms complemenary to the target nucleic acid which could act as an assembly facilitator for an active MNAzyme ligase. The substrate arms could be complementary to two substrate molecules which contain ligatable sequences with appropriate termini (e.g.Table 4).<br>
[ 00365 ] If the 5'ligation substrate and the 3' ligation substrate were each labelled with either a fluorophore and quencher dye respectively (or vice versa) then ligation could result in a decrease in fluorescence which could serve as an indicator of the presence of the target nucleic acid. In an alternative format, if the 5' ligation substrate were labelled with a detectable moiety, for example a fluorophore, and the 3' ligation substrate were attached to a discrete location, for example on a chip, then ligation could result in the appearance of a signal at the discrete location in the presence of target nucleic acid.<br>
[ 00366 ] The strategy could also be used for detection of other ligands if the MNAzyme were an apta-MNAzyme with ligase activity which contained an aptamer sequence capable of specifically binding the target ligand.<br>
[ 00367 ] Further, the ligation of two substrates by an MNAzyme in the presence of a target analyte could constitute the first step in a cascade reaction.<br><br>
Example 3: Mechanisms for facilitating and inhibiting the assembly of active MNAzymes or MNAi complexes.<br>
[ 00368 ] An MNAzyme is composed of partzymes, which assemble in the presence of one or more assembly facilitators, to form an active enayme (e.g. Figures 1, 2, 4 (right hand panel) and Figure S (i and ii)). The assembly facilitator(s), which binds to partzyme sensor arms, can be a target analyte, or can be a synthetic nucleic acid molecule(s) added to the reaction mix to drive MNAzyme assembly. In addition to their capacity to contribute to active MNAzyme assembly, partzymes can assemble into an inactive, non-catalytic, MNAi complex when they hybridise with an "activity inhibitor" molecule (Figure 4 (left hand panel) and Figure S (iii). Various alternative oligonucleotide sequences were tested for their capacity to regulate the assembly of either active MNAzyme or inactive MNAi complexes.<br>
[ 00369 1 In this example (depicted in Figures 4 and 5), MNAzyme assembly was examined in the presence of (i) a single molecule (assembly facilitator Fl/2), or (ii) two molecules (assembly facilitator component Fi and assembly facilitator component F2), whose sequences together comprise that present in facilitator Fl/2. Facilitator F2 binds to the whole sensor arm of one partzyme and overlaps to bind 4 base pairs of the sensor arm of the second partzyme. In another reaction, an "activity inhibitor" molecule, which has a sequence that includes that of facilitator F2 plus an additional activity inhibitor domain, was tested for its ability to drive the reaction towards assembly of inactive MNAi complexes.<br>
3.1	Partzyme Oligonucleotides<br>
[00370] Thesequenccsofpartzymes A and partzyme Bare shown below (5'to 3')- In the following sequences the bases in bold hybridize with the assembly facilitator, bases underlined form part of the catalytic core of the assembled MNAzyme, and bases in italics hybridize to the substrate. The "-P" indicates 3' phosphorylation of the oligonucleotide.<br>
SEQ ID NO 6: Partzyme A R05A4/2-P:<br>
CAAACaAOTCCTGOCCTTOTCTACAACGAGAGGAAACCTT-P<br><br>
SEQ ID NO 7: Partzyme B R05B5/2-P: TYSCCCAGGGAGGCTAGCTOTGOAOACGGATTACACCTTCCCACTTGC-P<br>
3.2	Reporter Substrate<br>
[ 00371 ] The reporter substrate for this example was SubBi-2-FB with the sequence, 5' to 3', as below. SubBi-2-FB was end-labelled with a 6-FAM moiety at the 5' end and a BHQl moiety at the 3' end. Cleavage of SubBi-2-FB was monitored at 530 nm (FAM emission wavelength) with excitation at 485 nm (FAM excitation wavelength). The lower case bases represent RNA and the upper case bases represent DNA.<br>
SEQ ID NO: 4      SubBi-2-FB: AAGGTTTCCTCguCCCTGGGCA<br>
3.3	Regulator Oligonucleotides.<br>
[ 00372 ] Several molecules were tested in this example for their ability to regulate the assembly of MNAzymes and/or MNAi complex. The sequence, which constitutes assembly facilitator F2, was also contained within the sequences of facilitator Fl/2 and the activity inhibitor and is in bold and underlined.<br>
SEQ ID NO 8: Assembly Facilitator Fl/2: GCAAGTGGGAAGGTGTAATCCGTCTCCACAGACAAGGCCAGGACTCGTTTG<br>
SEQ ID NO: 9 Assembly Facilitator Fl: GCAAGTGGGAAGGTGTAATCCGTCT<br>
SEQ ID NO: 10 Assembly Facilitator F2: CCACAGACAAGGCCAGGACTCGTTTQ<br>
SEQ ID NO: U Activity inhibitor molecule:<br>
AAGGTTTCCTCGTCCCTGGGCACCACAGACAAGGCCAGQACTCGTTTQ<br>
3.4	Reaction Components and monitoring ofMNAzyme activity.<br>
[ 00373 ] Real time monitoring of MNAzyme activity was performed on a BMG LabTech FluoStar fluorometer in a total reaction volume of 50 jiL. Reactions were monitored isothermally at 55°C for 4 minutes. The reaction was initiated by injection of the fluorescent substrate SubBi-2-FB (10 jil of 1 ^M solution) into reaction mixture containing Ix PCR Buffer II (Applied Biosystems), 25 mM MgCb, 200 nM of Partzyme A R05A4/2-P, 200 nM of Partzyme B R05B5/2-P and either (i) 400 nM of Assembly<br><br>
Facilitator Fl/2, or (ii) 400 nM of Assembly Facilitator component Fl and 400 nM of Assembly Facilitator component F2, or (iii) 400 nM of Activity inhibitor and 400 nM of Assembly Facilitator component Fl, or (iv) no Assembly facilitator.<br>
3.5	Results:<br>
[ 00374 ] The results using the combinations of various regulatory or structural component oligonucleotides are shown in Figure S. A rapid increase in fluorescent signal, indicative of high level of MNAzyme cleavage activity, was seen in reactions containing either assembly facilitator Fl/2 (Figure 5 (i)), or assembly facilitator components Fl and F2 (Figure 5 (ii)). No increase in fluorescence over time was observed in the absence of a facilitator (Figure 5 (iv)). This demonstrates that an assembly facilitator need not always be an unbroken oligonucleotide, but rather can be split into multiple shorter facilitator components, which align adjacent to each other on one of the sensor arms of a partzyme.<br>
[ 00375 ] No increase in fluorescent signal was observed over time in reactions containing the activity inhibitor and facilitator Fl (Figure 5 (iii)). Since the activity inhibitor molecule includes the sequence of facilitator F2, then the additional non-complementary inhibitory sequence adjoined to facilitator F2 is the element driving the assembly of inactive MNAi complexes. The MNAi complex can bind to a substrate but cannot catalytically modify it. As a result, the substrate was not cleaved and fluorescence did not increase over time in the presence of MNAi complexes (Figure 5 (iii)). Comparison between reactions containing the assembly facilitator components Fl and F2 (Figure 5 (ii)), with those containing the assembly facilitator Fl and the activity inhibitor (which incorporates the F2 sequence)(Figure 5 (iii)), demonstrates that the presence of an inhibitory domain within an activity inhibitor can provide a tool with which to regulate enzymatic activity by driving the assembly of inactive MNAi complexes and preventing the formation of active MNAzymes,<br>
[ 00376 ] Thus an MNAzyme, designed to form in the presence of an assembly facilitator Fl/2, generated fluorescence. The example demonstrated that the assembly facilitator Fl/2 could be split into two parts (assembly facilitator components Fl and F2) and retain the capacity to direct the assembly of catalytic active MNAzymes.  Together<br><br>
the two assembly facilitator components can stabilise active MNAzyme formation and cause fluorescence, provided they bind adjacent to each other on the partzyme sensor arm. Subsequent experiments, performed under identical reaction conditions, demonstrated that no increase in fluorescence over time was observed in the presence of assembly facilitator component F2 only (data not shown). As such, this example demonstrates the assembly of partzymes into active MNAzymes can require the presence of multiple component assembly facilitators. When multi-component assembly facilitators are required, the presence or absence of one or more of these components can be used to control the assembly of active MNAzymes and as such switch them on and off.<br>
[ 00377 ] It was further discovered that an activity inhibitory molecule could prevent MNAzyme assembly by hybridising to a partzyme and disrupting the secondary structure at the junction of the two assembly facilitator component on a partzyme sensor arm which is required for enzymatic activity (see Figures 4 and 5 as examples). It would be appreciated by one skilled in the art, that an inhibitory molecule could be designed to hybridise to either partzyme A or B, and to either the sensor or substrate arm of partzyme.<br>
[ 00378 ] Molecules including activity inhibitors, partzyme stabilizer arm components and assembly facilitators or components thereof, can be used to regulate the assembly of active MNAzymes and inactive MNAi complex. Transition between states of activation (MNAzyme) and inactivation (MNAi complex) can provide a mechanism for creating a molecular switch, which can be regulated by alternating between the active and inactive conformations. Such molecular switches could be applied to the control of nucleic acid replication cascades or to the regulation of autonomous therapeutic, diagnostic and computational molecular scale devices. Various protocols which can be employed to induce the dissociation of a speciflc oligonucleotide component are discussed throughout this document.<br>
Example 4: Application of MNAzymes to detect non-nucleic acid analytes including small molecules such as adenosine S'-trqthospltate.<br>
[ 00379 ] Aptamers are single-stranded DNA or RNA molecules evolved in vitro from large pools of random-sequence oligonucleotides for their capacity to bind target analytes<br><br>
with high affinity and specificity. Aptamers have been selected for their abiHty to bind specifically to many types of analytes including proteins, carbohydrates, lipids, nucleotides, whole cells and viruses. In this example, an aptamer sequence was incorporated at the end of a partzyme (apta-partzyme) in a configuration whereby an active apta-MNAzyme was only formed in the presence of the activator ligand. There are several ways of achieving this goal, including the strategy used in the following example, which is illustrated in Figure 7.<br>
[ 00380 ] The nucleic acid oligonucleotides included in this exemplary apta-MNAzyme detection strategy are illustrated in Figure 7 and include;<br>
a)	a partzyme;<br>
b)	an apta-partzyme which is a partzyme with an aptamer incorporated into one of its ends;<br>
c)	an assembly facilitator which binds to both the apta-partzyme and the partzyme enabling assembly of an active MNAzyme;<br>
d)	a substrate e.g. a reporter substrate; and<br>
e)	an assembly inhibitor oligonucleotide which hybridises to the apta-partzyme in a region which spans at least part of the aptamer sequence and part of the substrate binding arm of the partzyme sequence.<br>
[00381 ] In the absence of an activator ligand (Figure 7, panel (i)), the assembly inhibitor oligonucleotide binds to the apta-partzyme thus preventing it from binding to the substrate. In the presence of an activator ligand (Figure 7, panel (ii)), the activator ligand can interact with the aptamer sequence of the apta-partzyme, thus preventing binding of the assembly inhibitor and allowing an active apta-MNAzyme to assemble, then bind to and cleave the substrate. As such, apta-MNAzymes can only form and cause fluorescent signal generation in the presence of an activator ligand.<br>
[ 00382 ] The strategy was demonstrated using detection of a small molecule, ATP. The 27 nucleotide long aptamer sequence used in this example has been previously reported as being highly specific for binding of ATP and dATP (Achenbach, et al 2005, Huizenga and Szostak, 199S).<br><br>
4.1	Partzyme oligonucleotides, itssembly facilitator and inhibitory oligonucleotides<br>
[ 00383 ] In this example the ATP aptamer sequence was adjoined to the substrate arm of a partzyme, to produce an apta-partzyme molecule (Figure 7). The sensor arms of the apta-partzyme and another partzyme were designed to bind to a synthetic assembly facilitator, included in the reaction to drive the assembly of MNAzymes when targets or regulatory analytes are present. The sequences of apta-partzyme AtpA2/l and partzyme AtpB3/l are shown below (5' to 3'). In the following sequences the bases in bold hybridize with the assembly facilitator, bases underlined form part of the catalytic core of the assembled MNAzyme, and bases in italics hybridize to the substrate. In addition, bases in plain text in partzyme AtpA2/l indicate a DNA aptamer sequence that can bind toATPordATP.<br>
SEQIDNO: 12 Apta-Partzyme A2 AtpA2/l: AAaaTACACTGCACGCGGTCGAAATAGTGAGTACCTGGGGGAGTATTGCGGAGGAAGGT<br>
SEQ ID NO: 13 Partzyme B3 AtpB3/l: CATCTCTTCTCCGAGCGTCTGTACCGTGTAC<br>
[ 00384 ]   The sequence of the assembly facilitator is shown below (5' to 3')<br>
SEQ ID NO: 14 Assembly facilitator AtpC/1: GTACACGGTACAGACCGTGCAGTGTACGTT<br>
[ 00385 ]   The sequence of the "assembly inhibitor" oligonucleotide is shown below (5' to 3').<br>
SEQ ID NO: 15 Assembly Inhibitor AtpR/I: CCAGGTACTCACTATTT<br>
4.2	Reporter substrate<br>
[ 00386 ] Apta-MNAzyme activity was monitored by cleavage of a dual-labelled nucleic acid reporter substrate. The reporter substrate for this example is SubBi-I-FB with the sequence, 5' to 3', as below. The lower case bases represent RNA and the upper case bases represent DNA. The underlined bases indicate the position of a 6-FAM moiety at the 5' end and a BHQl moiety at the 3' end. Changes in fluorescence due to cleavage of SubBi-1-FB at the ribonucleotide between the FAM and BHQl  were<br><br>
monitored at 520 nm (FAM emission wavelength) with excitation at 490 nm (FAM excitation wavelength).<br>
SEQ ID NO: 16	SubBi- 1-FB: ACTCACTATaGGAAGAGATG<br>
4.3	Target or regulatory anafytes and control molecules<br>
[ 00387 ] The activator ligands used for this example were adenosine S'-triphosphate (ATP) and deoxyadenosine 5'-triphosphate (dATP). Guanosine S'-triphosphate (GTP) and cytosine 5'-triphosphate (CTP) were used as negative control molecules. All molecules were purchased from Bioline. Nuclease-free water was used as a no analyte control.<br>
4.4	Reaction Conditions<br>
[ 00388 ] The presence of the activator ligand was measured by an increase in fluorescent signal caused by cleavage of the reporter substrate by the catalytically active apta-MNAzyme. Reactions were initiated by the addition of substrate and the total volume of all reactions was SO ^L. Prior to substrate injection, all reactions were pre-incubated at 60 °C for 5 minutes (to reduce secondary structure). Reactions were conducted at 47 "C on a FLUOstar OPTIMA (BMG Biotech). Fluorescence for each reaction was read every 3 seconds for a total of 10 minutes. Each reaction contained a final concentration of 200 nM AtpA2/l, 200 nM AtpB3/l, 200 nM AtpC/1, 200 nM AtpR/1, 200 nM SubBi-1-FB, 25 mM MgCb, 50 mM Tris HCl pH 7.5 and 2 mM of either ATP, or dATP, or GTP, or CTP or no analyte (water control).<br>
4.5	Results: Detection and cleavage ofSubBi-1-FB reporter substrate<br>
[ 00389 ] In the absence of ATP or dATP a low level of fluorescence was seen which did not increase over time, demonstrating that in the absence of ATP, the assembly inhibitor prevented the assembly of active apta-MNAzyme complexes. In the presence of ATP or dATP, the fluorescent signal was higher and it increased over time. This indicates that the inhibitor oligonucleotide was displaced by dATP and ATP and an active apta-MNAzyme was formed. Assembly of the apta-MNAzyme was ligand-dependent. In the presence of GTP or CTP a low level of fluorescence was seen which did not increase over time.  The fluorescence observed in the presence of GTP or CTP<br><br>
was similar to that observed in the absence of ATP or dATP i.e. in the no ligand water control. This example demonstrates that MNAzymes can be coupled to aptamers for the detection of analytes in an approach that is highly specific for the target analyte. This example further demonstrates that an assembly inhibitor molecule can be used to control the assembly of apta-MNAzymes, and that a molecule e.g. ATP can serve as a activator ligand or molecular regulator in this system.<br>
[ 00390 ] One skilled in the art will recognise that the design of this strategy can be flexible. The aptamer can be incorporated, for example, into either end (5' or 3') of either of the two partzymes containing partial catalytic core sequences. As such, the assembly inhibitor can bind to the aptamer region and to either the substrate arm (that binds the substrate) or the sensor arm (that binds the assembly facilitator). In the former design (Figure 7 and this example), the inhibitor blocks binding of the substrate. In the latter design, the inhibitor would prevent binding of the assembly facilitator with the apta-partzyme and therefore would prevent assembly of active MNAzymes.<br>
[ 00391 ] The literature contains sequences for a large number of aptamers capable of detecting many types of analytes. These include proteins, carbohydrates, lipids, prions, nucleotides, whole cells and viruses. Aptamers to all these types of analytes could be linked to partzymes to detect a very diverse range of molecules. Reaction conditions (buffer, temperature, divalent cation concentration etc), which are compatible with both binding of analytes to aptamers (or apta-partzymes) and cleavage of a reporter substrate by an MNAzyme, can be determined by empirical testing.<br>
[ 00392 ] MNAzyme activity can be modulated by the removal or addition of the assembly inhibitor. Changing the oligonucleotide sequence, the melting temperature and or concentration can achieve finer regulation. The hybridization of the assembly inhibitor within an MNA is affected by many factors, including but not limited to, salt concentration, cation concentration, temperature and the presence or absence of additives (e.g. DMSO). As such, entities that affect hybridization can provide a tool for controlling the assembly and disassembly of MNAzymes,<br>
[ 00393 ]   The assembly facilitator can be removed by physical manipulation of components and can be achieved, for example, by exploiting either physical properties of<br><br>
attached moieties as molecular "hooks", and/or by exploiting inherent properties of the oligonucleotides, for example, negative charge, or sequence complementarity.<br>
[ 00394 ] MNAzymes and apta-MNAzymes can be used for detection of target analytes including nucleic acid targets and/or non-nucleic acid ligands. Further they can be used to detect analytes in a direct detection format (as demonstrated in this example) or they can be used in nucleic acid enzyme cascade reactions. In one embodiment an MNAzyme and/or an apta-MNAzyme could facilitate the first step in the cascade. In one embodiment the nucleic acid cascade could use a combination of MNAzymes, or a combination of MNAzyme(s) and DNAzyme(s) which could modify one or more substrate(s) by, for example, cleavage and/or ligation.<br>
Example 5: A cascade reaction which uses a nucleic acid enzyme with ligase activity and a nucleic acid enzyme with cleavage activity.<br>
[ 0039S ] A reaction useful for inclusion in cascade reactions and/or molecular switches exploiting the catalytic activities of two nucleic acid enzymes is outlined in Figure 8. One reaction could be mediated by a DNAzyme ligase, which can ligate a 5' substrate with a 2', 3'-cyclic phosphate and a 3' substrate with a 5' hydroxl to form a ligation product with a non-native 2'-S' linkage at the junction of the two substrates. Examples of such DNAzyme ligases are known in the art (see summary Table 4). The other reaction could be mediated by an MNAzyme cleaver which could use the ligation product as a substrate and could cleave it at the non-native 2'-5' linkage created by ligation. Cleavage could produce two cleavage products, a 5' product which would have a 2', 3'-cyclic phosphate terminus and a 3' product which would have a 5'-hydroxyl terminus. In turn, one or both of the cleavage products could serve as a substrate or substrates for the DNAzyme ligase.<br>
[ 00396 ] Alternatively a reaction could be mediated by a DNAzyme ligase, which can ligate a 5' substrate \yith a 2',3'-diol and a 3' substrate with a S'-triphosphate terminus to form a ligation product with a native 3'-5' linkage at the junction of the two substrates. Examples of such DNAzyme ligases are known in the art (see summary Table 4). The other reaction could be mediated by an MNAzyme cleaver which could use the ligation<br><br>
product as a substrate and could cleave it at the native 3'-5' linkage created by ligation. Cleavage could produce two cleavage products, a 5' product which would have a 2', 3'-cyclic phosphate terminus and a 3' product which would have a 5'-hydroxyl terminus. In turn, one or both of the cleavage products could serve as a substrate or substrates for a DNAzyme ligase that can ligate such termini.<br>
[ 00397 ] Alternatively, an MNAzyme with cleavage activity may cleave at a site within a ligation product which is not at the junction of the two ligatable substrates provided the cleavage site requirement are met at this site.<br>
[ 00398 ] In a preferred format (Figure 8), a DNAzyme ligase could ligate a first oligonucleotide (oligo 1) to a second oligonucleotide (oligo 2) to create a third ligation product (LP) oligonucleotide (oligo 3), provided oligo 1 and oligo 2 have 2\ 3'-cycIic phosphate and S'-hydroxyl termini. In turn, LP-oligo 3 could be cleaved by an MNAzyme into cleavage products (CP), CP-oligo 1 and CP-oligo 2, thus regenerating oligonucleotides with 2', 3'-cyclic phosphate and S'-hydroxyl termini, which could participate in further rounds of ligation.<br>
[ 00399 ] In a multiplex format, several oligonucleotides, for example four oligonucleotide, each with 2',3'-cyclic phosphate and a 5'-hydroxyl termini, could be ligated by a series of DNAzyme ligases into 16 new unique LP sequences (i.e. each combination of oligonucleotides 1,2, 3 and 4). A series of MNAzymes could then cleave LP-oligo I through to LP-oligo 16 back into component oligonucleotides, now called CP-oligo 1, CP-oligo 2, CP-oligo 3 and CP-oligo 4. Alternatively, an MNAzyme with cleavage activity may cleave at a site within a LP which is not at the junction of the two ligatable substrates provided the cleavage site requirements are met at this site. In this senario complex mixtures of oligonucleotide sequences could be created, shuffled and/or rearranged.<br>
[ 00400 ] A set of MNAzymes could use a constant assembly facilitator (as illustrated) in Figure 8, and/or the MNAzyme could use the LP sequences from previous ligation rounds as its assembly facilitator. As such, new information (input) data produced by the ligation of oligonucleotides, could be recognized "read" by the MNAzyme. The MNAzyme  could  then  cleave  "write"  to  produce  new  output  products,   and/or<br><br>
information. Systems where MNAzymes could read the input LP, and then cleave them into CP-oligos, other than those originally in the pool of starting molecules, could be used to rewrite or recode new output sequences.<br>
[ 00401 ] In some embodiments, ligation by a DNAzyme could "write" input data, for example, by making new assembly facilitators, or components thereof. An MNAzyme could "read" the data, by interrogating the information encoded within the assembly facilitator using the partzyme sensor arms. An MNAzyme could then "write" data, for example, a new sequence (e.g. a cleavage product) thus creating new output data which could then be "read" by a DNAzyme ligase (e.g. by ascertaining the suitability of MNAzyme cleavage products to serve as ligation substrates for the DNAzyme ligase).<br>
[ 00402 ] In other embodiments, cleavage by an MNAzyme could "write" input data, for example, by creating hew ligatable substrates. A DNAzyme ligase could "read" the data, by interrogating the information encoded within the substrates using the substrate binding arms. A DNAzyme ligase can then "write" data, for example, by making a new ligation product sequence. The new ligation product could serve as a component necessary for formation of an active MNAzyme cleaver, for example it could serve as a new assembly facilitator or component thereof The ligation product could thus provide new output data which could then be "read" by an MNAzyme cleaver (by ascertaining the suitability of DNAzyme ligation product to serve as an assembly facilitator or assembly facilitator component for the MNAzyme cleaver).<br>
[ 00403 ] As such, this MNAzyme/DNAzyme ligase cascade could form an automaton. Such devices are capable of converting information from one form into another, according to a defmed procedure. In this case the procedures would be encoded and directed by the substrate arms and/or the sensor arms of the MNAzymes and DNAzyme ligases.<br>
[ 00404 ] An automaton, which was capable of solving computational problems was developed by Benenson et al, 2001, using DNA and protein enzymes (a restriction endonuclease and a ligase). The restriction endonuclease cleaved the double stranded DNA and the protein ligase ligated the cleavage products in a cascade reaction. The protein enzymes served as the "hardware" and the DNA encoded the "software".  The<br><br>
input and automaton were programmed by selection of the appropriate DNA software sequences. The automaton proceeded via a cascade of restriction endonuclease cleavage, hybridization and ligation cycles, producing a detectable output molecule that encoded the automaton's final state and thus the computational result (Benenson et al, 2001).<br>
[ 00405 ] The DNAzyme ligase/MNAzyme cascade could be used in a similar manner to the cascade used by Benenson et al (2001) and thus provide a device capable of solving computational problems. Unlike Benenson's device, a DNAzyme ligase/ MNAzyme cascade, requires no protein enzymes to achieve the same results. While Benenson's device was programmed by double stranded DNA, a DNAzyme ligase/MNAzyme cascade, would be encoded by the various sequences, including for example, the initial input oligonucleotide(s), the substrate arms and/or the sensor arms of the MNAzymes and the substrate arms of DNAzyme ligases.<br>
[ 00406 ] In another embodiment, the DNAzyme ligase/MNAzyme cascade could also be used to "shuffle" oligonucleotide sequences as a method of constructing, and/or increasing the diversity of, molecular libraries.<br>
[ 00407 ] In some embodiments, a DNAzyme ligase could be used to create or destroy " components of MNAzyme and or inactive MNA. By way of example a DNA ligase could attach an "activity inhibitor" to an "assembly facilitator component" resulting in the assembly of MNAi complex ("off' state). Alternatively a DNAzyme ligase could attach a sensor or substrate arm to a partzyme component to create an "on" switch for MNAzymes by promoting assembly. In another embodiment, the DNAzyme ligase could be attached to a sequence labeled with a moiety that allows oligonucleotides to be selectively captured, for example using a biotin group or the moiety could contain a radio-frequency magnetic field radio to facilitate remote electronic control of hybridisation. This approach could allow selective removal of component molecules allowing activation or inhibition of enzymatic activity. For example, the activity inhibitor could be selectively denatured from an MNAi complex resulting in transition to the active MNAzyme state.<br><br>
[ 00408 ] It would be appreciated that the DNAzyme ligase used in this example (as illustrated in Figure 8) could be replaced by an MNAzyme ligase (as illustrated in Figure 11).<br>
Example 6: Demonstration of a reaction which exploits the activity of multiple<br>
nucleic acid enzymes, in this example specifically MNAzymes which cleave nucleic<br>
acid substrates and DNAzymes which ligate nucleic acid substrates, whereby the<br>
cleavage or ligation products created by one type of nucleic acid enzyme forms a<br>
new substrate or enzyme component which participates in the next nucleic acid<br>
enzyme reaction.<br>
[ 00409 ] The strategy used in this example is illustrated in Figure 9. This example demonstrates (i) using target nucleic acid to direct the formation of an MNAzyme (MNAzyme A) which can cleave a substrate (MNAzyme substrate A) thus generating a 5'cleavage product A which can in turn be used as a substrate for a DNAzyme B with ligase activity; (ii) using a DNAzyme ligase B to ligate the S' cleavage product A generated in step (i) to another oligonucleotide ligation substrate thus creating a new partzyme; and (iii) using the new partzyme/ligation product generated in step (ii) to form a new MNAzyme (MNAzyme C) which can cleave a substrate (MNAzyme substrate C). In this example the labelling of MNAzyme substrate C with a fluorophore and quencher dye pair allowed fluorescent monitoring of the reaction.<br>
[ 00410 ]   The DNAzyme ligase B used in this example has previously been reported to ligate RNA through the formation of a 2'-5* phosphodiester linkage firom a 2', 3'-cyclic phosphate and a 5'-hydroxyl group (Prior et al, 2004). Apart from the requirement of a S' substrate with a 2', 3'-cyclic phosphate end, the DNAzyme ligase used in this example also requires a specific sequence motif at the ligation junction, being ua*g(A or G) (where lower case denotes RNA bases and uppercase notes an RNA or DNA base and * denotes the ligation site).<br>
[ 00411 ] In the first step, MNAzyme A forms in the presence of target nucleic acid and is used to cleave MNAzyme substrate A (preSubS) thus generating a cleavage fragment which has a 2', 3* - cyclic phosphate end. In the second step the DNAzyme ligase was<br><br>
then used to ligate the 5' product A (preSubS cleavage fragment) with another oligonucleotide ligation substrate (preSub3). In the third step the ligated product functions as a new partzyme, which together with a second partzyme and an assembly facilitator can form MNAzyme C and cleave a fluorescently labelled substrate C (SubBi-2-FB).<br>
6.1	DNAzyme Ligase B<br>
[ 00412 ] The DNA oligonucleotide sequence which acts as a DNAzyme ligase is listed below.<br>
SEQ ID NO: 17 DNAzyme ligase 7Z81-10/10:<br>
CCTCTCGTTGACGGCGGAGTGATTGGGAGGTTAGCTCTAGTGAGTGC<br>
6.2	Partzyme oligonucleotides<br>
[ 00413 ] In the following sequences the bases in bold hybridize with the target nucleic acid sequence, bases underlined form part of the catalytic core of the assembled MNAzyme, and bases in italics hybridize to the substrate.<br>
[ 00414 ] The following are sequences of the partzymes which form components of MNAzyme A:<br>
SEQ ID NO: 18 Partzyme A2/MNAzyme A miR20A2/l: TACCTGCACTACGGTCGAA^r.4GfG^Gr<br>
SEQ ID NO: 19 Partzyme B3/MNAzyme A miR20B3/l: C/i rcrCTTCTCCGAGCTAAGCACTTTA<br>
[ 00415 ] The following sequence corresponds to the partzyme which associates with the ligation product/partzyme to form a component of MNAzyme C.<br>
SEQ ID NO: 20 Partzyme B5/MNAzyme C STB5/2(21): rGCCC4GGC//JGGCTAGCTCTGTCGTCGGAGTGGTCGTCG<br>
6.3	MNAzyme substrate A (substrate for MNAzyme A)<br>
[00416] In the following sequence, the lower case bases represent RNA and the uppercase bases represent DNA.<br><br>
SEQIDNO:21preSub5: CTGTAGCACTCACTAuaGGAAGAGATG<br>
6.4	DNAzyme Ligase substrate<br>
[00417] In the following sequence, the lower case base represents RNA and the uppercase bases represent DNA. The 3' DNAzyme iigase substrate was synthesised to have the sequence below and a 5' hydroxyl group:<br>
SEQ ID NO: 22 preSubS: gGAACAACGAGAGGAAACCTT<br>
6.5	MNAzyme substrate C (Fluorescent Reporter substrate for MNAzyme C)<br>
[00418] The reporter substrate used in this example was SubBi-2. In the current example, SubBi-2 was end-labelled with a 6-FAM moiety at the 5' end, a BHQl moiety at the 3' end and designated SubBi-2-FB. Cleavage of SubBi-2-FB was monitored at 520 nm (FAM emission wavelength) with excitation at 490 nm (FAM excitation wavelength). The sequence of SubBi-2-FB is listed below (5' to 3'); the lower case bases represent RNA and the upper case bases represent DNA.<br>
SEQ ID NO: 4 SubBi-2-FB: AAGGTTTCCTCguCCCTGGGCA<br>
6.6	Target sequence for MNAzyme A<br>
[00419] The target sequence recognised by MNAzyme A was a synthetic DNA oligonucleotide homologous to the human microRNA miR-20 RNA sequence. It had the following sequence:<br>
SEQ ID NO: 23 D-20 target (MNAzyme A target): TAAAGTGCTTATAGTGCAGGTA<br>
6.7	Assembly facilitator for MNAzyme C<br>
[ 00420 ] The assembly facilitator required for MNAzyme C formation was a synthetic DNA oligonucleotide with the following sequence:<br>
SEQ ID NO: 24 Assembly Facilitator for MNAzyme C: CGACGACCACTCCGACGACAGTCCTATAGTGAGTGCTACAG<br><br>
6.8 Reaction Conditions<br>
[ 00421 3 The reaction was performed in three sequential steps (i), (ii) and (iii) in three separate tubes as described below.<br>
(i)	Cleavage of preSubS  by  MNAiyme  A  to  generate  the  5'   cleavage<br>
product/iigase substrate with the required 2\ 3'-cyclic phosphate end was performed at 40°C for I hour in a total reaction volume of 100 jil. The reactions contained 500 nM preSubS, 50 mM Tris HCl (pH 9.0 at 25°C). 25 mM MgCljand:<br>
a)	Partzymes for MNAzyme A (100 nM miR20A2/l and 100 nM miR20B3/l) and 100 nM of D-20 target; or<br>
b)	no additional oligonucleotides (no cleavage control)<br>
(ii) Ligation was performed at 40°C for 2 hours in a total reaction volume of 25 ^l. Reactions contained 10 nl of previous cleavage reaction mix (i-a) or (i-b), 200 nM preSub3, 150 mM NaCl, 2 mM KCl, 50 mM Tris HCl (pH 9.0 at 25*'C) and 40 mM MgCh and:<br>
a)	200 nM of DNAzyme ligase 7Z81 -10/10, or<br>
b)	no additional oligonucleotide (no ligation control)<br>
(iii)	End-point fluorescent detection due to MNAzyme cleavage mediated by a<br>
partzyme formed by ligation was performed isothermally at 55X for 45 minutes on the Cepheid Smartcyclcr, Reactions were initiated by the addition of the fluorescently labelled substrate SubBi-2-FB. Reactions contained 10 (il of previous ligation reaction mix, 200 nM of partzyme STB5/2(21), 200 nM of the assembly facilitator for MNAzyme C, 50 mM Tris HCl (pH 9.0 at 25°C). 25 mM MgCb and 200 nM of the substrate SubBi-2-FB in a total reaction volume of 25^1.<br>
[ 00422 ] The increase in fluorescence due to cleavage of MNAzyme substrate C (SubBi-2-FB) was monitored over time for reactions that had contained the foUowmg oligonucleotide reagents (Table 6).<br><br>
Table 6. Increase in fluorescence due to cleavage ofMNAzyme substrate C (SubBi-2-FB) monitored over time.<br>
fetep (i)	IStcp (ii)	fctep (iii)	I Result<br>
" Substrate A    • lO^il from step      10^1 from step (ii)       An increase<br>
Reaction 1   " MNAzyme      (i)	MNAzyme C	in<br>
Apartzymes   * Ligase substrate   partzymeB	fluorescence<br>
(Test!-a,      "Target	• DNAzyme	Substrate C	was observed<br>
ii-a)	ligase	Assembly facilitator    over time<br>
	jC	<br>
Reaction 2   " Substrate A    * lOjil from step      10^1 from step (ii)      No increase<br>
(i)	MNAzyme C	in<br>
(Control (i-	* Ligase substrate   partzyme B	fluorescence<br>
b, ii-a - no	* DNAzyme	' Substrate C	was observed<br>
MNAzyme	ligase	' Assembly facilitator    overtime<br>
_A)	^	C.	<br>
Reactions   '"Substrate A     •lOul from step      10^1 frY&gt;m step (ii)       No increase<br>
" MNAzyme      (i)	MNAzyme C	in<br>
(Control (i-   Apartzymes   * Ligase substrate	partzymeB	fluorescence<br>
a, ii-b) - no '" Target	Substrate C	was observed<br>
DNAzyme	Assembly facilitator    over time<br>
ligase)	C<br>
[ 00423 ] The increase in fluorescence over time was measured for step (iii) of reactions 1, 2 and 3 (Table 6). Reaction 1, which had contained all oligonucleotide reaction components for steps (i), (ii) and (iii), showed an increase in fluorescence over time. In contrast Reaction 2, which lacked both the MNAzyme A oligonucleotide partzyme components and the target sequence for MNAzyme A for step (i) showed no increase in fluorescence over time. Similarly, Reaction 3, which lacked the DNAzyme ligase oligonucleotide required for step (ii), showed no increase in fluorescence over time. Together these reactions indicate that the following events have occurred in Reaction 1.<br>
[ 00424 ] Firstly, MNAzyme A cleaved MNAzyme substrate A in the presence of specific target (D-20) producing two fragments, one of which is the 5' fiiigment with the sequence CTGTAGCACTCACTAua (SEQ ID NO: 40) that had a 2', 3'-cyclic phosphate terminus. Secondly, this S' fragment was ligated to a second oligonucleotide ligase substrate present in the reaction mix of step (ii) and this resulted in the formation of a new       oligonucleotide       (ligation       product)       with       the       sequence       of<br><br>
CTGTAGCACTCACTAuagGAACAACGAGAGGAAACCTT (SEQ ID NO: 51) (where upper case represent DNA bases and lower case represent RNA bases). This ligation product in turn functioned as a partzyme component for MNAzyme C. This newly created partzyme associated with a second partzyme in the presence of an assembly facilitator and created MNAzyme C. MNAzyme C cleaved MNAzyme substrate C (SubBi-2-FB) resulting in separation of a fluorophore/quencher dye pair thus causing an increase in fluorescence.<br>
[ 00425 ] The "no MNAzyme A cleavage" control (Reaction 2) demonstrated that cleavage and generation of the 5' fragment with the 2', 3'-cyclic phosphate terminus was essential for subsequent ligation and formation of one of the partzymes required for MNAzyme C. The "no ligation" control (Reaction 3) demonstrated that the ligation of the 5' cleavage product A with 2', 3'-cyclic phosphate terminus to a second 3' ligase substrate was essential for the formation of one of the components of MNAzyme C.<br>
[ 00426 ] The strategy demonstrated in this example allows detection of target analytes in a cascade reaction which used two types of nucleic acid enzymes, namely DNAzymes which can ligate substrates (DNAzyme ligases) and MNAzymcs which can cleave substrates (MNAzyme cleavers). Multiple turnover at each enzymatic step in the cascade could allow signal ampliflcation.<br>
[ 00427 ] The cascade strategy demonstrated in this example and illustrated in Figure 9 could be extended and used to create a feedback amplification cascade. If the sequence of 5' cleavage product C was the same as 5' cleavage product A then this product could also serve as a 5' substrate B and a feedback cascade reaction could be initiated. In this reaction, MNAzyme cleaver C would be constantly generating 5' substrate B which in turn would be ligated by DNAzyme ligase B to create more partzymes for formation of more MNAzyme cleaver C. This strategy could provide a mechanism for feedback amplification following initiation of a reaction by an assembly facilitator e.g. a target analyte which could direct the assembly of MNAzyme cleaver A. The strategy would allow detection of target analytes followed by signal amplification by feedback cascade using two types of nucleic acid enzymes, namely DNAzymes which can ligate substrates (DNAzyme ligases) and MNAzymes which can cleave a substrate (MNAzyme cleavers).<br><br>
[ 00428 ] One skilled in the art would appreciate that there are many variants on this basic cascade strategy. By way of example, the DNAzyme B could be used to create ligation products which sWe as assembly facilitators for another MNAzyme which could have, for example, cleavage, ligase or other enzymatic activity. Further variations are included throughout the specification.<br>
Example 7. Sequences and conditions which allow the formation ofMNAzymes with ligase activity and testing of these MNAzymes with ligase activity in a format useful in<br>
cascade reactions.<br>
[ 00429 ] DNAzyme ligase 7Q10 was used as a basis for generating candidate MNAzyme ligase partzyme sequences in this example. As illustrated in Figure 10 each ligase partzyme was comprised of three sequence domains, including (i) a sensor arm which hybridizes to the assembly facilitator (eg a target nucleic acid), (ii) a domain that constitutes part of a catalytic core and (iii) a substrate binding arm. The partial core sequences were designed to incorporate portions of the catalytic region of the 7Q10 DNAzyme ligase which has previously been reported to ligate RNA through the formation of a 2'-5' phosphodiester linkage from a 2', 3'-cyclic phosphate and a 5'-hydroxyl group (Flynn-Charlebois et ai, 2003).<br>
[ 00430 ] A multi-step assay was used to assess the enzymatic activity of combinations of candidate partzyme sequences. The general strategy is outlined in Figure 12 and can also form the basis of a signal amplification technique. In the first step, MNAzyme A forms in the presence of assembly facilitator A and is used to cleave MNAzyme substrate A thus generating a 5' cleavage product A which has a 2', 3'- cyclic phosphate end. In the second step active MNAzyme B v/ith ligase activity, assembles in the presence of assembly facilitator B and then ligates the 5' product A (5' substrate B) to a 3' substrate B. In the third step, the ligated product B functions as a new partzyme, which together with another partzyme and an assembly facilitator C can form MNAzyme C and then cleave a substrate C which may be, for example, fluorescently labelled. The catalytic activity of several ligase partzyme pairs was assayed.<br><br>
7.1 Partzyme oligonucleotides which incorporate domains that correspond to incomplete catalytic core sequences of a DNAzyme with ligase activity (7Q10).<br>
[ 00431 ] In this experiment the series of partzyme pairs, for the second step, were all synthesized with sensor arms designed to hybridize to an assembly facilitator (AFMzLB), and with substrate arms designed to hybridize to two substrate sequences, SLSubB and 3LSubB. The sequences of the partzyme pairs used in this experiment are listed below (S' to 3'). The bases underlined form part of the catalytic core of the assembled candidate MNAzymes, bases in bold (sensor domain) hybridize with the assembly facilitator and bases in italics (substrate domain) hybridize to the substrates.<br>
SEQ ID NO: 25 Split 1 Partzyme A - R05LS1 A: AACGAGTCCTGGCCTTGTCTfiGCrCTMGT'G/lGT'GC<br>
SEQ ID NO: 26 Split 1 Partzyme B - R05LSIB: TlCrCGTyGTyACGTGGAGGTGGTGGAGACGGATTACACCTT<br>
SEQ ID NO: 27 Split 2 Partzyme A - R05LS2A: AACGAGTCCTGGCCTTGTCTfiGG£IC7MGrG.4(77'GC<br>
SEQ ID NO: 28 Split 2 Partzyme B - R05LS2B: fCrCGTTGTTACGTGGAGGTGTGGAGACGGATTACACCTT<br>
SEQ ID NO: 29 Split 3 Partzyme A - R05LS3A: AACGAGTCCTGGCCTTGTCTTGGGCTCr/4GrG^GrGC<br>
SEQ ID NO: 30 Split 3 Partzyme B - R05LS3B: rCrCGTTGTTACGTGGAGGGTGGAGACGGATTACACCTT<br>
SEQ ID NO: 31 Split 4 Partzyme A - R05LS4A: AACGAGTCCTGGCCTTGTCTTGGAGGTGGGCTCTMGyG/JGTlGC<br>
SEQ ID NO: 32 Split 4 Partzyme B - R05LS4B: rCT-CGTTGTTACeGTGGAGACGGATTACACCTT<br>
SEQ ID NO: 33 Split 5 Partzyme A - R05LS5A: AACGAGTCCTGGCCTTGTCTGGAGGTGGGCTCTMGrGyJGrGC<br>
SEQ ID NO: 34 Split 5 Partzyme B - R05LS5B:<br><br>
rCrCGTTGTTACGTGTGGAGACGGATTACACCTT<br>
SEQ ID NO: 35 Split 6 Partzyrae A - R05LS6A: AACGAGTCCTGGCCTTGTCTGTGGAGGTGGGCTCTXGrG/JGrGC<br>
SEQ ID NO: 36 Split 6 Partzyme B - R05LS6B: 71C7TCC?7T(77TACGTGGAGACGGATTACACCTT<br>
SEQ ID NO: 37 Split 7 Partzyme A - R05LS7A: AACGAGTCCTGGCCTTGTCTGAGGTGGGCTCr^GrOJGrGC<br>
SEQ ID NO: 38 Split 7 Partzyme B - R05LS7B: rCrCGZTGTTACGTGGTGGAGACGGATTACACCTT<br>
7.2	Control DNAzyme with ligase activity.<br>
[ 00432 ] A DNAzyme ligase 7Q10 with substrate arms designed to hybridize to the substrate sequences SLSubB and 3LSubB was used in this experiment and the sequence is listed below (5' to 3'). The bases underlined form the catalytic core and bases in italics hybridize to the substrates.<br>
SEQ ID NO: 39 DNAzyme ligase - 7Q10-10/10:<br>
TCTCGTTGTTACGTGGAGGTGGGCYCTAGTGAGTGC<br>
7.3	MNAzyme substrate A (substrate for MNAzyme A)<br>
[ 00433 ] In the following sequence, the lower case bases represent RNA and the uppercase bases represent DNA.<br>
SEQ ID NO: 21 SubA/PreSLSubB: CTGTAGCACTCACTAuaGGAAGAGATG<br>
7.4	The 3'substrate Bfor ligation<br>
[ 00434 ] The sequence of the 3' substrate for ligation used in this experiment is listed below (5' to 3'). The oligonucleotide has a 5' hydroxyl group. The lower case bases represent RNA and the upper case represent DNA.<br>
SEQ ID NO: 22 Ligase 3' Substrate B - 3LSubB: gGAACAACGAGAGGAAACCTT<br><br>
7.5	Substrate for MNAzyme C (Fluorescent Reporter substrate C for detection)<br>
[ 00435 ] The rqxjrter substrate used in this example was SubBi-2. In the current example, SubBi-2 was end-labelled with a 6-FAM moiety at the 5' end, a BHQl moiety at the 3' end and designated SubBi-2-FB. Cleavage of SubBi-2-FB was monitored at 520 nm (FAM emission wavelength) with excitation at 490 nm (FAM excitation wavelength). The sequence of SubBi-2-FB is listed below (5' to 3'); the lower case bases rqiresent RNA and the upper case bases represent DNA.<br>
SEQ ID NO: 4 SubBi-2-FB: AAGGTTTCCTCguCCCTGGGCA<br>
7.6	Assembly facilitators<br>
[ 00436 ] The assembly facilitator A recognized by MNAzyme A sensor domains was a synthetic DNA oligonucleotide with the following sequence:<br>
SEQ ID NO: 42 AF-MzCA:<br>
CGTCGTGAGATGAGGAAGAGATGGATGGGCAC<br>
[ 00437 ] The assembly facilitator B recognized by MNAzyme B sensor domains was a synthetic DNA oligonucleotide with the following sequence:<br>
SEQ ID NO: 43 AF-MzLB: AAGGTGTAATCCGTCTCCACAGACAAGGCCAGGACTCGTTTG<br>
[ 00438 ] The assembly facilitator C recognized by NfNAzyme C sensor domains was a synthetic DNA oligonucleotide with the following sequence:<br>
SEQIDNO:24AF.MzCC: CGACGACCACTCCGACGACAGTCCTATAGTGAGTGCTACAG<br>
7.7	Partzymesfor MNAzymes with cleavage activity.<br>
[ 00439 ]   Partzymes for MNAzyme A were synthesized with the following sequence:<br>
SEQ ID NO: 45 Partzyme A/MNAzyme A - 4SyntA2/l(16): GTGCCCATCCATCTCCGGTCGAAATAGTGAGT<br>
SEQ ID NO: 46 Partzyme B/MNAzyme A - 4SyntB3/l(16):<br><br>
CATCTCTTCTCCGAGCTTCCTCATCTCACGACG<br>
[ 00440 ]   One partzyme for MNAzyme C was synthesized with the following sequence:<br>
SEQ ID NO: 20 Partzyme B/MNAzymc C - STB5/2(2I)<br>
TGCCCAGGGAGGCTAGCTCTGTCGTCGGAGTGGTCGTCG<br>
7.8 Detection ofligase activity.<br>
7.8.1 Reaction Conditions<br>
[ 00441 ] The reaction was performed in three sequential steps (i), (ii) and (iii) in three separate tubes as described below.<br>
(i) Cleavage of SubA/preSLSubB by MNAzyme A to generate the 5'ligase substrate (5LSubB) with the required 2*,3'-cyclic phosphate end was performed at 40°C for 1 hour. The reactions contained 100 nM of partzyme 4SyntA2/I(16), 100 nM of partzyme 4SyntB3/l(16), 100 nM of the assembly facilitator AF-MzCA, 500 nM SubA/Pre5LSubB, 50 mM Tris HCl (pH 9.0 at 25°C) and 50 mM MgCU.<br>
(ii) Ligations were performed at 40°C for 2 hours in a total reaction volume of 25 ^l. Reactions contained 10^1 of the cleavage reaction mix ftom step (i), 200 nM 3LSubB, 200 nM AF-MzLB, 150 mM NaCl, 2 mM KCl, 50 mM Tris HCl (pH 9.0 at 25°C) and 40 mM MgCh and:<br>
a)	200 nM of control DNAzyme ligase 7Q10-10/10, or<br>
b)	200 nM of each of Split 1 partzymes A and B (R05LSIA and R05LS1B), or<br>
c)	200 nM of each of Split 2 partzymes A and B (R05LS2 A and R05LS2B), or<br>
d)	200 nM of each of Split 3 partzymes A and B (R05LS3A and R05LS3B), or<br>
e)	200 nM of each of Split 4 partzymes A and B (R05LS4A and R05LS4B), or<br>
f)	200 nM of each of Split 5 partzymes A and B (R05LS5A and R05LS5B), or<br>
g)	200 nM of each of Split 6 partzymes A and B (R05LS6A and R05LS6B), or<br>
h)	200 nM of each of Split 7 partzymes A and B (R05LS7A and R05LS7B), or<br>
i)	200 nM of each of Split 2 partzyme A (R05LS2A) and Split 4 partzyme B<br>
(R05LS4B).<br><br>
[ 00442 ] An additional "no 5' ligation substrate" control reaction lacked 10 jil from the cleavage reaction mix from step (i). This reaction mix contained 200 nM 3LSubB, 200 nM of each of Split 4 partzymes A and B (R05LS4A and R05LS4B), 150 mM NaCl, 2mM KCl, 50 mM Tris HCl (pH 9.0 at 25°C), 40 mM MgCl2,200 nM AF-MzLB and 200 nMofSubA/Pre5LsubB<br>
(iii) Fluorescent detection due to MNAzyme C cleavage mediated by a partzyme formed by ligation in step (ii) was performed isothermally at 55°C for up to 3.5 hours on the Cepheid Smartcycler with data collection points at 5 minute intervals. Reactions were initiated by the addition of the fluorescently labelled substrate SubBi-2-FB. Reactions contained 10 ^1 of previous ligation reaction mix (ii-a or ii-b or ii-c or ii-d or ii-e or ii-f or ii-g or ii-h or ii-i or "no-ligation substrate" control), 200 nM of partzyme B for MNAzyme C (STB5/2(21)), 200 nM of the assembly facilitator for MNAzyme C (AF-MzCC), 50 mM Tris HCI (pH 9.0 at 25°C), 25 mM MgCb and 200 nM of the substrate SubBi-2-FB in a total reaction volume of 25 (il. The increase in fluorescence due to cleavage of MNAzyme substrate C (SubBi-2-FB) was monitored over time for reactions.<br>
7.8.2 Results<br>
a) Control reactions.<br>
[ 00443 ] The positive control DNAzyme iigase reaction (steps i, ii-a and iii) showed an increase in fluorescence over time. This reaction confirms that MNAzyme A can form and cleave SubA/Pre5LSubB to generate a product (5LSubB) which has a 2', 3'-cyclic phosphate which makes it suitable for use as a substrate by the DNAzyme ligase. The sequence of 5LSubB is CTGTAGCACTCACTAua (SEQ ID NO: 40) where upper case represents DNA bases and lower case represents RNA bases. This control reaction demonstrated the uni-molecular DNAzyme ligase 7Q10-10/10 ligated 5LSubB and 3LSubB and formed a new oligonucleotide (ligation product) with the sequence of CTGTAGCACTCACTAuagGAACAACGAGAGGAAACCTT (SEQ ID NO: 51) (where upper case represents DNA bases and lower case represents RNA bases).  This ligation<br><br>
product in turn fiinctioned as a partzyme component of MNAzyme C which cleaved substrate C (SubBi-2-FB) resulting in an increase in fluorescence. This increase in fluorescence acted as a surrogate marker of ligase activity.<br>
[ 00444 ] The "no 5' ligation substrate" control reaction (steps ii and iii only) did not show an increase in fluorescence over time. This demonstrates that if SubA/PreSLSubB is not cleaved by MNAzyme A to generate SLSubB then no appropriate 5' ligation substrate is available for ligation to 3LSubB. As a result no partzyme for MNAzyme C is generated and no increase in fluorescence is observed.<br>
b) Reactions designed to test for the presence of ligase activity when using candidate , partzyme pairs which may or may not have capacity to form functional MNAzyme ligases.<br>
[ 00445 ] Reactions that incorporated steps i and iii along with either step ii-e (Split 4) or ii-f (Split S) showed an increase in fluorescence over time. Splitting the catalytic core of the DNAzyme ligase 7Q10 at these positions and then reassembling the partial cores in the partzymes in the presence of an assembly facilitator produced active MNAzyme ligases under these reaction conditions.<br>
[ 00446 ] Reactions which incorporated steps i and iii along with either step ii-b (Split 1) or ii-c (Split 2) or ii-d (Split 3) or ii-g (Split 6) or ii-h (Split 7) or ii-i (Split 2/4) showed no increase in fluorescence after 35 min of monitoring by MNAzyme C cleavage. This demonstrates that splitting the catalytic core of the DNAzyme ligase 7Q10 at these positions and then reassembling the partial cores with an assembly facilitator does not produce detectable MNAzyme ligase activity under these reaction conditions.<br>
[ 00447 ] The example has generated a number of partzyme sequences that can be used to assemble MNAcomplexes, which are inactive or have various levels of enzymatic activity depending on the speciflc sequences and combinations thereof. The general design for candidate partzyme sequences tested are summarized in Table 7.<br>
Table 7. General design for partzymes sequences tested.	<br>
Sequences of the partzyme sequence pairs (based on the 7Q10 DNAzyme ligase) that<br>
were tested for potential MNAzyme ligase activity where N is any nucleotide, (N)x is<br>
any number of nucleotides which bind to a substrate sequence (substrate domain) and<br>
(N)y is any number of nucleotides which bind to an assembly facilitator (sensor	<br><br>
domain). The SEQ ID NO's refer only to the core sequences.	<br>
Split #      Partzyme A	Partzyme B<br>
Split I       (N)yGGCTC(N)x                      (N)xACGTGGAGGTG (N)y<br>
	 (SEQ ID NO: 101)	(SEQ ID NO: 41)	<br>
Split 2       (N)y GGGCTC(N)x                    (N)x ACGTGGAGGT (N)y<br>
	 (SEQ ID NO: 102)	(SEQ ID NO: 44)     	<br>
Split 3       (N)y TGGGCTC(N)x                  (N)xACGTGGAGG(N)y<br>
	(SEQ ID NO: 103)	 (SEQ ID NO: 104) 	<br>
Split 4      CN)yTGGAGGTGGGCTC         (N)xACG(N)y<br>
	(N)x   (SEQ ID NO: 47)	(SEQ ID NO: 48)	<br>
Splits       (N)y GGAGGTGGGCTC (N)x   (N)xACGT(N)y<br>
	 (SEQ ID NO : 49)	(SEQ ID NO: SO)	<br>
Split 6      (N)yGTGGAGGTGGGCTC      (N)xAC(N)y<br>
	(N)x (SEQ ID NO: 53 )	(SEQ ID NO: 105)	<br>
Split 7       (N)y GAGGTGGGCTC (N)x      (N)xACGTG(N)y<br>
	 (SEQ ID NO: 54)	(SEQ ID NO: 106)	<br>
Split 2/4    (N)yGGGCTC (N)x                    (N)xACG (N)y<br>
	I (SEQ ID NO; 102)	| (SEQ ID NO: 48)	<br>
Sequence of the DNAzyme ligase7Q10 where N is any nucleotide and (N)x is any<br>
number of nucleotides which bind to a substrate sequence.	<br>
(N)x ACGTGGAGGTGGGCTC(N)x (SEQ ID NO: 52)<br>
[ 00448 ] Active MNAzymes, including examples as demonstrated above with splits equivalent to Split 4 or Split 5 would have a wide number of applications. By way of example, the MNAzyme ligases could be used to detect target analytes. MNAzyme ligases can be designed to assemble only in the presence of an analyte such as an RNA or DNA target sequence (the assembly facilitator). If the 5'ligation substrate and the 3' ligation substrate were each labelled with either a fluorophore and quencher dye respectively (or vice versa) then ligation would result in a decrease in fluorescence which could serve as an indicator of the presence of the target analyte. In an alternative format, if the S' ligation substrate were labelled with a detectable moiety, for example a fluorophore, and the 3' ligation substrate were attached to a discrete location, for example on a chip or bead, then ligation would result in the appearance of a signal at the discrete location in the presence of target analyte.<br>
[ 00449 ] The strategy used to assay for ligase activity in this example is illustrated in Figure 12,   Once active partzymes have been identified the same cascade reaction can<br><br>
then be used to detect the presence of an assembly facilitator, for example a target analyte. The cascade reaction allowing detection uses two types of nucleic acid enzymes, namely MNAzymes which can ligate substrates and MNAzymes which can cleave substrates. Multiple turnover at each enzymatic step in the cascade could allow signal amplification.<br>
[ 00450 ] The cascade demonstrated in this example and illustrated in Figure 12 could be extended and used to create a feedback amplification cascade. If the sequence of S' cleavage product C was the same as 5' cleavage product A then this product could also serve as a 5' substrate B and a feedback cascade reaction could be initiated. In this reaction, MNAzyme cleaver C could be constantly generating S' substrate B which could in turn be ligated by the MNAzyme ligase B to create more partzymes for formation of more MNAzyme cleaver C. This strategy could provide a mechanism for feedback amplification following initiation of a reaction by a target analyte which could direct the assembly of MNAzyme cleaver A. The strategy would allow detection of target analytes followed by signal amplification by a feedback cascade using two types of MNAzyme, namely MNAzyme ligases and MNAzyme cleavers.<br>
[ 00451 ] Further, this example can also be considered as an example of a reaction where the initiating step results in the formation of an MNAzyme with ligase activity. In this scenario, with reference to Figure 12, the initiating target nucleic acid would be the assembly facilitator B (AF-MzLB) and the assembly facilitator A (AF-MzCA) would be a synthetic molecule included to drive the assembly of MNAzyme A. MNAzyme A would then have the sole purpose of generating a 5' ligatable substrate B with 2'3' cylic phosphate termini in a reaction which could then be added to the reaction which detects the target assembly facilitator B using an MNAzyme B with ligase activity. This could provide a practical method for producing substrates with a 2'3' cyclic phosphate terminus which are expensive to synthesise and relatively unstable. Further the MNAzyme A partzymes and the assembly facilitator A could be replaced by a DNAzyme with cleavage activity present either in a separate preliminary reaction for ligatable substrate generation or the DNAzyme could be present in the same-reaction mix as the MNAzyme with ligase activity. Example 11 demonstrates that cleavage and ligation using a combination of an MNAzyme and a DNAzyme can be performed in a single reaction mix.<br><br>
[ 00452 ] One skilled in the art would appreciate that there are many variants on this basic cascade strategy. By way of example, the MNAzyme B could be used to create ligation products which serve as assembly facilitators for another MNAzyme with cleavage, ligase or other enzymatic activity. In another example, MNAzyme C could modify a substrate by means other than cleavage. Further embodiments are included throughout the specification.<br>
Example 8: An exemplary strategy for switching an MNA complex between the "on state" of an active MNAzyme to the "off state" of an MNAi complex using a second<br>
MNAzyme with ligase activity.<br>
[ 004S3 ] Figure 13 illustrates an exemplary strategy for svtdtching an MNA complex between the "on state" of an active MNAzyme to the "off state" of an MNAi complex using a second MNAzyme with ligase activity. In this strategy, an active MNAzyme A, which could be capable of modifying (e.g. cleaving or ligating) a substrate(s) A could be formed in the presence of assembly facilitator component 1 (AFC 1) and assembly facilitator component 2 (AFC 2). A second MNAzyme B, which has ligase activity, could form in the presence of, for example, an assembly facilitator 3 (AF3) which could comprise one or more components, and could then ligate AFC 2 with an activity inhibitor component (AIC) causing the formation of an activity inhibitor (AI). This AI could bind to the partzymc components for the MNAzyme A and result in the formation of an MNAi A complex which is inactive. As such the MNAzyme ligase in this example could operate as an off switch to inactivate an MNAzyme A. The inactive MNAi complex and the catalytically active MNAzyme would represent two alternate states for the assembled components, namely an "off' state and the "on" state respectively.<br>
Example 9. Sequences and conditions which allow the formation ofMNAzymes with ligase activity and testing of these MNAzymes with ligase activity in a format useful in<br>
a signal amplification cascade.<br>
[ 00454 ] DNAzyme ligase 7Z81 was used as a basis for generating candidate MNAzyme ligase partzyme sequences in this example. As illustrated in Figure 10 each ligase partzyme was comprised of three sequence domains, including (i) a sensor arm<br><br>
which hybridizes to the assembly facilitator (e.g. a target nucleic acid), (ii) a domain that constitutes part of a catalytic core and (iii) a substrate binding arm. The partial core sequences were designed to incorporate portions of the catalytic region of the 7Z81 DNAzyme ligase which has previously been reported to ligate RNA through the formation of a 2'-5' phosphodiester linkage from a 2', 3'-cyclic phosphate and a 5'-hydroxyl group (Prior et al, 2004).<br>
[ 00455 ] A multi-stq) assay was used to assess the enzymatic activity of combinations of candidate partzyme sequences. The general strategy is outlined in Figure 12 and can also form the basis of a signal amplification technique. In the first step, MNAzyme A forms in the presence of assembly facilitator A and is used to cleave MNAzyme substrate A thus generating a 5' cleavage product A which has a 2', 3'- cyclic phosphate end. In the second step active MNAzyme B with ligase activity, assembles in the presence of assembly facilitator B and then ligates the 5' product A (5* substrate B) to a 3' substrate B. In the third step, the ligated product B fimctions as a new partzyme, which together with another partzyme and an assembly facilitator C can form.MNAzyme C and then cleave substrate C, which may be, for example, fluorescently labelled. The catalytic activity of several candidate ligase partzyme pairs was assayed.<br>
9.1	Partzyme oligonucleotides which incorporate domains that correspond to<br>
incomplete catalytic core sequences of a DNAzyme with ligase activity (7Z81).<br>
[ 00456 ] In this experiment the series of partzyme pairs, for the second step, were all synthesized with sensor arms designed to hybridize to an assembly facilitator (AFMzLB), and with substrate arms designed to hybridize to two substrate sequences, 5LSubB and 3LSubB. The sequences of the partzyme pairs used in this experiment are listed below (S' to 3'). The bases underlined form part of the catalytic core of the assembled candidate MNAzymes, bases in bold (sensor domain) hybridize with the assembly facilitator and bases in italics (substrate domain) hybridize to the substrates.<br>
SEQ ID NO: 55 Split 1 Partzyme A - R05S1 A: CAAACGAGTCCTGGCCTTGTCTGGAGGTTAGCTCL4G71G.4GrGC<br>
SEQ ID NO: 56 Split I Partzyme B - ROSS IB: CCrCrCGTTGACGGCGGAGTGATTGGTGGAGACGGATTACACCTT<br><br>
SEQ ID NO: 57 Split 2 Partzyme A - R05S2A: CAAACGAGTCCTGGCCTTGTCTTGGGAGGTTAGCTCry<grg></grg>
SEQ ID NO: 58 Split 2 Partzyme B - R05S2B: CCrCrCGTTGACGGCGGAGTGATGTGGAGACGGATTACACCTT<br>
SEQ ID NO: 59 Split 3 Partzyme A - R05S3A: CAAACGAGTCCTGGCCTTGTCTATTGGGAGGTTAGCTCTVJGT'G/JGrGC<br>
SEQ ID NO: 60 Split 3 Partzyme B - R05S3B: CCrcrCGTTGACGGCGGAGTGGTGGAGACGGATTACACCTT<br>
SEQ ID NO: 61 Split 4 Partzyme A - R05S4A: CAAACGAGTCCTGGCCTTGTCTTGATTGGGAGGTTAGCTC7MG7'G/1(771GC<br>
SEQ ID NO: 62 Split 4 Partzyme B - R05S4B: CCrcrCGTTGACGGCGGAGGTGGAGACGGATTACACCTT<br>
SEQ ID NO: 63 Split 5 Partzyme A - R05S5A: CAAACGAGTCCTGGCCTTGTCTAGTGATTGGGAGGTTAGCTC7:4Gro.4GrGC<br>
SEQ ID NO: 64 Split 5 Partzyme B - R05S5B: CCrCTTGTTGACGGCGGGTGGAGACGGATTACACCTT<br>
SEQ ID NO: 65 Split 6 Partzyme A - R05S6A: CAAArGAGTCCTGGCCTTGTCTGGAGTGATTGGGAGGTTAGCTC7MG7X?^G TGC<br>
SEQ ID NO: 66 Split 6 Partzyme B - R05S6B: CCrCrCGTTGACGGCGTGGAGACGGATTACACCTT<br>
9.2    Control DNAzyme with ligase activity.<br>
[ 00457 ] The DNAzyme ligase used in this example has previously been reported to ligate RNA and form a 2'-5' phosphodiester linkage using a 5' ligation substrate with a 2\ 3'-cyclic phosphate and a 3' ligation substrate with a 5'-hydroxyI group (Prior et al, 2004). Apart from the requirement for specific termini on the substrates, the DNAzyme ligase used in this example also requires a specific sequence motif at the ligation junction.<br><br>
[ 00458 ] A DNAzyme ligase 7Z81 with substrate arms designed to hybridize to the substrate sequences SLSubB and 3LSubB was used in this experiment and the sequence is listed below (5' to 3'). The bases underlined form the catalytic core and bases in italics hybridize to the substrates.<br>
SEQ ID NO: 17 DNAzyme ligase - 7Z81-10/10:<br>
CCTCTCGTTGACGGCGGAGTGATTGGGAGGTTAGCICTAGTGAGTGC<br>
9.3	MNAzyme substrate A (substrate for MNAxyme A)<br>
[ 00459 ] The sequence of Substrate A (Pre5LSubB2-FB) is below. It is internally labelled 5' of the ribobases with a 6-FAM moiety and labelled 3' of the ribobases with BHQl moiety (underlined bases). The lower case bases represent RNA and the upper case bases represent DNA.<br>
SEQ ID NO: 67 Substrate A - Pre5LSubB2-FB: CTGTAGCACTCACTAuaGGAAGAGATGAG<br>
9.4	The 3' substrate Bfor ligation<br>
[ 00460 ] The sequence of the 3' substrate for ligation used in this experiment is listed below (5' to 3'). The oligonucleotide has a 5' hydroxyl group. The lower case bases represent RNA and the upper case bases represent DNA.<br>
SEQ ID NO: 22 Ligase 3' Substrate B - 3LSubB: gGAACAACGAGAGGAAACCTT<br>
9.5	Substrate for MNAzyme C (Fluorescent Reporter substrate Cfor detection)<br>
[ 00461 ] The reporter substrate used in this example was SubBi-2. In the current example, SubBi-2 was end-labelled with a Quasar 670 moiety at the 5' end, a BHQ2 moiety at the 3' end and designated SubBi-2-Q6B2. Cleavage of SubBi-2-Q6B2 was monitored at 665 nm with excitation at 635 lun. The sequence of SubBi-2-Q6B2 is listed below (5' to 3'); the lower case bases represent RNA and the upper case bases represent DNA.<br><br>
SEQ ID NO: 4 SubBi-2-Q6B2: AAGGTTTCCTCguCCCTGGGCA<br>
9.6	Assembly facilitators<br>
[ 00462 ]   The assembly facilitator A recognized by MNAzyme A sensor domains was a synthetic DNA oligonucleotide with the following sequence:<br>
SEQ ID NO: 68 AF-MzCA2:<br>
GCCATTGTCGAACACCTGCTGGATGACCAGC<br>
[ 00463 ]   The assembly facilitator B recognized by MNAzyme B sensor domains was a synthetic DNA oligonucleotide with the following sequence:<br>
SEQ ID NO: 43 AF-MzLB: AAGGTGTAATCCGTCTCCACAGACAAGGCCAGGACTCGTTTG<br>
[ 00464 ]   The assembly facilitator C recognized by MNAzyme C sensor domains was a synthetic DNA oligonucleotide with the following sequence:<br>
SEQ ID NO: 24 AF-MzCC: CGACGACCACTCCGACGACAGTCCTATAGTGAGTGCTACAG<br>
9.7	Partzymesfor MNAzymes with cleavage activity.<br>
[ 00465 ]   Partzymes for MNAzyme A were synthesized with the following sequence:<br>
SEQ ID NO: 69 Partzyrae A/MNAzyme 1 - R04A2/1(11); GCTGGTCXTCCAGCAGCGGICGAAATAGTGAGTGC SEQ ID NO: 70 Partzyme B/MNAzyme 1 - R04B3/1(12): CTCA TCTCTTCTCSQMiSGTGTTCGXCWTGGC<br>
[ 00466 ]   One partzyme for MNAzyme C was synthesized with the following sequence: SEQ ID NO: 20 Partzyme B/MNAzyme C - STB5/2(21) TGCCCAGGGAGGCTAGCTCTGTCGTCGGAGTGGTCGTCG<br><br>
9.8    Detection ofligase activity.<br>
9.8.1 Reaction Conditions<br>
[ 00467 ] The reaction was performed in three sequential steps (i), (ii) and (iii) in three separate tubes as described below.<br>
(i) Cleavage of Pre5LSubB2-FB by MNAzyme A to generate the 5'ligase substrate (SLSubB) with the required 2', 3'-cyclic phosphate end was performed at 40°C for 1 hour. The reactions contained 50 nM of partzymc R04A2/I(11), 50 nM of partzyme R04B3/1(12), 100 nM of the assembly facilitator AF-MzCA2, 1000 nM Pre5LSubB2-FB, 50 mM Tris HCl (pH 9.0 at 25°C) and 40 mM MgClj.<br>
(ii) Ligations were performed at 40°C for 2 hours in a total reaction volume of 25 ^1. Reactions contained 10 yA of the cleavage reaction mix from step (i), 1000 nM 3LSubB, 500 nM AF-MzLB, 150 mM NaCl, 2 mM KCl, 50 mM Tris HCl (pH 9.0 at 25°C) and 40 mM MgCh and either:<br>
a)	500 nM of control DNAzyme ligase 7Z81-10/10, or<br>
b)	500 nM of each of Split 1 partzymes A and B (R05SIA and R05S1B), or<br>
c)	500 nM of each of Split 2 partzymes A and B (R05S2A and R05S2B), or<br>
d)	500 nM of each of Split 3 partzymes A and B (R05S3A and R05S3B), or<br>
e)	500 nM of each of Split 4 partzymes A and B (R05S4A and R05S4B), or<br>
f)	500 nM of each of Split 5 partzymes A and B (R05S5A and R05S5B), or<br>
g)	500 nM of each of Split 6 partzymes A and B (R05S6A and R05S6B), or h) no DNAzyme or partzymes (no ligase control)<br>
(iii) Fluorescent detection due to MNAzyme C cleavage mediated by a partzyme formed by ligation in step (ii) was performed isothennally at SS'C for 2 hours on the Cepheid Smartcycler with data collection points at 72 second intervals. Reactions were initiated by the addition of the fluorescently labelled substrate SubBi-2-Q6B2. Reactions contained 10 \x\ of previous ligation reaction mix (ii-a or ii-b or ii-c or ii-d or ii-e or ii-f or ii-g or ii-h, 500 nM of partzyme B for MNAzyme C (STB5/2(21)), 100 nM of the assembly facilitator for MNAzyme C (AF-MzCC), PCRII Buffer (Applied Biosystems),<br><br>
50 mM MgCb and 1000 nM of the substrate SubBi-2-Q6B2 in a total reaction volume of 25 jxl. The increase in fluorescence due to cleavage of MNAzyme substrate C (SubBi-2-Q6B2) was monitored over time for each reaction.<br>
9.8.2 Results<br>
a)      Control reactions.<br>
[ 00468 ] The positive control DNAzyme ligase reaction (steps i, ii-a and iii) showed an<br>
increase in fluorescence over time. The signal produced by the ligase dependent<br>
formation of MNAzyme C was measured in arbitrary fluorescent units. The overall<br>
change in fluorescence (flnal fluorescence minus initial fluorescence) and the time at<br>
which the reaction came to completion (plateau in fluorescence signal) is shown in Table<br>
8. This change in fluorescence observed in the control DNAzyme reaction confirmed that<br>
the steps in the cascade reaction have been successfully achieved. In this reaction<br>
NiNAzyme A formed and cleaved Pre5LSubB2-FB to generate a product (SLSubB)<br>
which has a 2', 3'-cyclic phosphate which makes it suitable for use as a substrate by the<br>
DNAzyme ligase. The sequence of the cleavage product 5LSubB is<br>
CTGTAGCACTCACTAua (SEQ ID NO: 40) where upper case represents DNA bases<br>
and lower case represents RNA bases. This control reaction demonstrated the uni-<br>
molecular DNAzyme ligase 7Z81-10/10 ligated 5LSubB and SLSubB and formed a new<br>
oligonucleotide	(ligation	product)	with	the	sequence	of<br>
CTGTAGCACTCACTAuagGAACAACGAGAGGAAACCTT (SEQ ID NO: 51) (where upper case represents DNA bases and lower case represents RNA bases). This ligation product in turn functioned as a partzyme component of MNAzyme C, which cleaved substrate C (SubBi-2-Q6B2) resulting in an increase in fluorescence. This increase in fluorescence acted as a surrogate marker of ligase activity.<br>
[ 00469 ] The no ligase control reaction (steps i, ii-h and iii) showed an insignificant increase in fluorescence over time (Table 8). This demonstrates that if no ligase is present 5LSubB cannot be ligated to 3LSubB. As a result no partzyme for MNAzyme C is generated and no increase in fluorescence is observed.<br><br>
b) Reactions designed to test for the presence ofligase activity when using candidate partzyme pairs, which may or may not have the capacity to form functional MNAzyme iigases.<br>
[ 00470 ] Reactions that incoiporated stqjs i and iii along with step ii-b (Split 1), ii-c (Split 2), ii-d (Split 3), ii-e (Split 4), ii-f (Split 5) and ii-g (Split 6) showed an increase in fluorescence over time. The capacity of each candidate partzyme pair to ligate substrates was measured by a change in arbitrary fluorescent units. The overall change in fluorescence (final fluorescence minus initial fluorescence) and the time at which the reaction came to completion (plateau in fluorescence signal) is shown in Table 8, Splitting the catalytic core of the DNAzyme ligase 7Z81 at those positions (Table 9) and then reassembling the partial cores in the partzymes in the presence of an assembly facilitator produced active MNAzyme Iigases under these reaction conditions.<br>
Table 8. Change in fluorescence over time for various partzymes.<br>
Change in fluorescence     Time to reaction<br>
	(fluorescent units)	completion<br>
7Z81 control DNAzyroe	1545	25 minutes<br>
	Partzymes with Split 1	479	&gt; 2 hours<br>
Partzymes with Split 2	1294		&gt; 2 hours<br>
Partzymes with Split 3	1337	&gt; 2 hours<br>
	Partzymes with Split 4	1284	84 minutes<br>
	Partzymes with Split 5	1755	35 minutes<br>
	Partzymes with Split 6	1123	83 minutes<br>
Control - no ligase (no DNAzyme	^.	^,.<br>
or candidate partzymes )        |	j	<br>
The example has generated a number of partzyme sequences that can be used to assemble MNAzymes, which have various levels of activity depending on the speciflc sequences and combinations thereof. The general design for the candidate partzymes sequences which were tested are summarized in Table 9.<br><br>
Table 9. General design for partzymes sequences tested.<br>
Sequences of the partzyme sequence pairs (based on the 7Z81 DNAzyme ligase) that were tested for potential MNAzyme ligase activity where N is any nucleotide, (N)x is any number of nucleotides which bind to a substrate sequence (substrate domain) and (N)y is any number of nucleotides which bind to an assembly facilitator (sensor<br>
domain). The SEQ ID NQ's refer only to the core sequences.	<br>
Split #     Partzyme A	Partzyme B<br>
Split 1     (N)y GGAGGTTAGCTC (N)x                  (N)x ACGGCGGAGTGATTG<br>
	(SEQ ID NO: 71   )	 (N)y (SEQ ID NO : 72)	<br>
Split 2     (N)y TGGGAGGTTAGCTC (N)x             (N)x ACGGCGGAGTGAT (N)y<br>
	(SEQ ID NO: 73)	(SEQ ID NO: 74)	<br>
Split 3     (N)y ATTGGGAGGTTAGCTC (N)x        (N)x ACGGCGGAGTG (N)y<br>
	(SEQ ID NO : 75 )	 (SEQ ID NO: 76 )	<br>
Split 4     (N)yTGATTGGGAGGTTAGCTC(N)     (N)x ACGGCGGAG (N)y<br>
	X (SEQ ID NO: 77 )	 (SEQ ID NO: 78 )	<br>
Splits     (N)y AGTGATTGGGAGGTTAGCTC      (N)x ACGGCGG (N)y<br>
	(N)x (SEQ ID NO: 79)	 (SEQ ID NO: 80 )	<br>
Split 6     (N)yGGAGTGATTGGGAGGTTAGCT   (N)x ACGGC (N)y<br>
	|c(N)x(SEQIDN0:81 )	| (SEQ ID NO : 82)	<br>
Sequence of the DNAzyme ligase 7Z81 where N is any nucleotide and (N)x is any<br>
number of nucleotides which bind to a substrate sequence.<br>
(N)x ACGGCGGAGTGATTGGGAGGTTAGCTC (N)x (SEQ ID NO: 83)<br>
[00471] Active MNAzymes, including examples as demonstrated above, and particularly the MNAzyme with the division equivalent to Split 5, would have a wide number of applications. By way of example, the MNAzyme ligases could be used to detect target analytes. MNAzyme ligases can be designed to assemble only in the presence of an analyte such as an RNA or DNA target sequence (the assembly facilitator). As an example, if the 5' ligation substrate and the 3* ligation substrate were each labelled with either a fluorophore or quencher dye respectively (or vice versa) then ligation would result in a decrease in fluorescence, which could serve as an indicator of the presence of the target analyte. In an alternative format, if the S' ligation substrate were labelled with a detectable moiety, for example a fluoropbore, and the 3' ligation substrate were attached to a discrete location, for example on a chip or bead, then ligation would result in the appearance of a signal at the discrete location in the presence of target analyte.<br><br>
[ 00472 ] The strategy used to assay for ligase activity in this example, as illustrated in Figure 12, constitutes a linear cascade which could be used to create a feedback signal amplification cascade. If the sequence of S' product C were the same as that of S' product A then this product could also serve as a 5' substrate B and a feedback cascade reaction could be initiated. In this reaction, MNAzyme C could constantly be generating a S' substrate B which could in turn be ligated by MNAzyme ligase B to create more partzymes for formation of more MNAzyme C. This strategy could provide a mechanism for feedback ampliflcation following initiation of a reaction by an assembly facilitator (e.g. target analyte) which could direct assembly of MNAzyme A. The strategy could allow detection of assembly facilitators (e.g. target analytes) followed by signal amplification using two types of MNAzymes, namely MNAzymes which could ligate substrates (MNAzyme ligases) and MNAzymes which could cleave a substrate (MNAzyme cleavers).<br>
[ 00473 ] One skilled in the art would appreciate that there are many variants on this basic strategy. By way of example, the MNAzyme B could be used to create Ugation products, which serve as assembly facilitators for another MNAzyme with cleavage, ligase or other enzymatic activity. In another example, MNAzyme C could modify a substrate by means other than cleavage. Further embodiments are included throughout the specification.<br>
Example 10. General method for engineering MNAzymes from DNAzymes,<br>
[ 00474 ] In vitro evolution techniques have facilitated the discovery and development of many DNAzymes (reviewed Emillson and Breaker, 2002). In vitro evolved DNAzymes have been discovered which have the capacity to catalyse a broad range of reactions including cleavage of nucleic acids, ligation of nucleic acids, porphyrin metallation, formation of carbon-carbon bonds, DNA phosphorylation, ester bonds, amide bonds, DNA deglycosylation, thymine dimer photoreversion and phosphoramidate cleavage (reviewed Silverman, 2007).<br>
[ 00475 ] Many DNAzymes have similar basic structures with multiple domains. These DNAzymes comprise a conserved catalytic domain (catalytic core) flanked by two non-<br><br>
conserved substrate-binding domains ("arms"), which specifically recognize and hybridise to the substrate (for example, for DNAzyme cleavers or those that act on one substrate) or at least two substrates (for example, for DNAzyme ligases or those that act on at least two substrates). The substrate binding domains can be tailored to any substrate provided the substrate or substrates contains a site(s) which can be modified by the DNAzyme. This example describes a general strategy for engineering new MNAzymes based on any known DNAzyme. This general strategy is illustrated schematically in Figure 14.<br>
[ 00476 ] The method is described firstly for deriving MNAzymes from DNAzymes with modifiying activity, for example cleavage activity. In the first step, positions within the DNAzyme catalytic core at which it can be split are identified, in order that each partial portion of the catalytic core can be distributed between two sequences such that the two partial cores together constitute an entire catalytic core. An oligonucleotide A can then be synthesised to contain (i) one substrate binding arm portion capable of binding to a substrate, (ii) one partial catalytic core portion, and (iii) one sensor arm portion capable of binding to an assembly facilitator molecule. A second oligonucleotide B is synthesised such that it contains (i) one substrate binding arm portion capable of binding to the same substrate as oligonucleotide A, whereby oligonucleotide B binds the substrate in a position adjacent to that of oligonucleotide A, (ii) one partial catalytic core portion which contains those bases from the entire DNAzyme catalytic core which are not incorporated into oligonucleotide A and (iii) one sensor arm portion capable of binding to the same assembly facilitator as oligonucleotide A, whereby oligonucleotide B binds the assembly facilitator in a position adjacent to that of oligonucleotide A. This process can be repeated thus making a series of pairs of oligonucleotides A and B which incorporate the structure and domains of partzymes, but may or may not have catalytic activity in the presence of a substrate and an assembly facilitator.<br>
[ 00477 ] Some or all of the candidate partzyme pairs (pairs of oligonucleotides A and B from the series) can then be incubated in the presence of the assembly facilitator complementary to the sensor arm portions of the partzymes and a substrate which is complementary to the substrate arm portions of the partzymes. Incubation is performed under conditions which are compatible with modification of a substrate by the DNAzyme<br><br>
from which the partial catalytic core sequences were derived. Pairs of oligonucleotides A and B which can perform the same type of modification to the substrate as the DNAzyme from which partial sequences were derived are useful as partzymes which can be assembled into MNAzymes. The sequences of the partial catalytic cores are suitable for incorporation into other partzymes. New partzymes can be synthesised which are tailored to new substrates (by changing the substrate binding domains of each partzyme) and/or to new assembly facilitators (by changing the sensor binding domains of each partzyme).<br>
[ 00478 ] The method for engineering an MNAzyme that acts on two substrates (e.g. an MNAzyme ligase) from a uni-molecular nucleic acid enzyme (e.g. a DNAzyme ligase) requires similar steps whereby in the first step, positions are identified within the DNAzyme catalytic core at which it can be split, such that each partial portion of the catalytic core can be distributed between two partial sequences such that the two partial cores together constitute an entire catalytic core. Two oligonucleotides A and B (candidate partzymes) can then be synthesised. An oligonucleotide A can be synthesised to contain (i) one substrate binding arm portion capable of binding to a first substrate (e.g. ligatable substrate), (ii) one partial catalytic core portion, and (iii) one sensor arm portion capable of binding to an assembly facilitator molecule. A second oligonucleotide B can be synthesised such that it contains (i) one substrate binding arm capable of binding to a second substrate (e.g. ligatable substrate), (ii) one partial catalytic core portion which contains those bases fiom the entire DNAzyme catalytic core which are not incorporated into oligonucleotide A and (iii) one sensor arm sequence capable of binding to the same assembly facilitator as oligonucleotide A, whereby oligonucleotide B binds the assembly facilitator in a position adjacent to that of oligonucleotide A. This process can be repeated thus making a series of pairs of oligonucleotides A and B which incorporate the structure and domains of partzymes, but may or may not have catalytic activity in the presence of the substrates and an assembly facilitator.<br>
[ 00479 ] Some or QU of the candidate partzyme pairs (pairs of oligonucleotides A and B from the series) can then be incubated in the presence of the assembly facilitator complementary to the sensor arm portions of the partzymes and substrates (e.g. ligatable substrates) which are complementary to the substrate arm portions of the partzymes. Incubation is performed under conditions which are compatible with modification of the<br><br>
substrates (e.g. by ligation) by the DNAzyme from which the partial catalytic core sequences were derived. Pairs of oligonucleotides A and B which can perform the same modification (e.g. ligation) to the substrates as the DNAzyme from which partial sequences were derived are useful as partzymes which can be assembled into MNAzymes. The sequences of the partial catalytic cores are suitable for incorporation into other partzymes. New partzymes can be synthesised which are tailored to new substrates (by changing the substrate binding domains of each partzyme) and/or to new assembly facilitators (by changing the sensor binding domains of each partzyme).<br>
[ 00480 ] Specific examples of the use of this approach to identify partial catalytic cores suitable for incorporation into partzymes capable of forming active MNAzymes are demonstrated for two DNAzymes with ligase activity (examples 7 and 9 and one DNAzyme-with cleavage activity (example 14).<br>
[ 00481 ] Further, a similar method can be used to examine tolerance to changes in the partial sequence derived from splitting the DNAzyme catalytic core that can be incorporated into component partzymes. For example, modifications to the partial catalytic cores incorporated into partzymes can be made which include, by way of example, insertions, deletions, substitution with alternate DNA bases, or substitution with DNA base analogues such as ribonucleotides, inosine bases or other analogues. Candidate partzyme pairs containing one or more modification(s) can then be assayed as above to determine whether or not they are suitable as partzymes that can assemble into catalytically active MNAzymes.<br>
Example 11. A cascade reaction which exploits the activity of multiple nucleic acid<br>
euTymes.<br>
[ 00482 ]   The general strategy used in this example is outlined in Figure 9 and can also<br>
form the basis of a signal amplification technique. In this specific example the reaction<br>
was performed in two steps. In the first step (Figure 9 (i) and (ii)), an MNAzyme A<br>
formed in the presence of assembly facilitator (e.g. a target nucleic acid) and was used to<br>
cleave a substrate (MNAzyme substrate A) thus generating a 5' cleavage product which<br>
had a 2', 3'- cyclic phosphate end. Simultaneously a DNAzyme ligase was used to ligate<br>
the 5' cleavage product to another oligonucleotide ligation substrate B thus creating a<br><br>
ligation product that functioned as a new NdNAzyme component (e.g. a partzyme). In the second step (Figure 9 (iii)), the partzyme ligation product generated in the first step participates in the formation of a new MNAzyme (MNAzyme C), which can cleave a substrate (MNAzyme substrate C). In this example the labelling of MNAzyme substrate C with a fluorophore and quencher dye pair allowed fluorescent monitoring of the reaction.<br>
11.1	Partzymes for MNAzyme A<br>
[ 00483 ] In the first step, MNAzyme A formed in the presence of an assembly facilitator (AF-MzCA2) and was used to cleave MNAzyme substrate A (Pre5LSubB2-FB). The sequences of the MNAzyme A partzymes (RC)4A2/1(11) and R04B3/1(12)) ar« listed below. The bases underlined form part of the catalytic core, bases in bold hybridize with the assembly facilitator and bases in italics hybridize to the substrates.<br>
SEQIDNO:69   Partzyme A/MNAzyme A-R04A2/l(ll); GCTGGTCATCCAGCAGCGGTCGAA^ TA GTGAGTGC SEQ ID NO: 70  Partzyme B/MNAzyme A - R04B3/1(12): CTCA TCTCTTCTCCQAGCGTGTTCGACWTGGC<br>
11.2	DNAzyme with Ligase activity<br>
[ 00484 ] The DNAzyme ligase used in this example has previously been reported to ligate RNA through the formation of a 2'-5' phosphodiester linkage fi-om a 2', 3'-cyclic phosphate and a 5'-hydroxyl group (Prior et al, 2004). Apart fit)m the requirement of a 5' substrate with a 2', 3'-cyclic phosphate end, the DNAzyme ligase used in this example also requires a specific sequence motif at the ligation junction.<br>
[ 00485 1 In the first step, the DNAzyme ligase 7Z81 (7Z81-10/10) with substrate arms designed to hybridize to the substrate sequences SLSubB and 3LSubB was used in this experiment and the sequence is listed below (S' to 3*). The bases underlined form the catalytic core and bases in italics hybridize to the substrates.<br><br>
SEQ ID NO: 17 DNAzyme ligase- 7Z81-10/10: CCrCrCG7TGACGGCGGAGTGATTGGGAGGTTAGCTC7:4GrGyJGr(;C<br>
11.3	PartzymesforMNAzymeC<br>
[ 00486 ] In the second step the ligated product functioned as a new partzyme, which together with a partzyme B (STB5/2(21)) and assembly facilitator (AF-MzCC), formed MNAzyrae C and cleaved a fluorescently labelled substrate C (SubBi-2-Q6B2). The control partzyme A (LS817A4/2), which was the same sequence as the ligated product also formed MNAzyme C with partzyme B (STB5/2(21)). The following sequences correspond to the partzymes, which form part of MNAzyme C. The bases underlined form part of the catalytic core, bases in bold hybridize with the assembly facilitator and bases in italics hybridize to the substrates.<br>
SEQ ID NO: 20 Partzyme B/MNAzyme C - STB5/2(21) TGCCC/f GC7G.4GGCTAGCTCTGTCGTCGGAGTGGTCGTCG SEQ ID NO: 84 Partzyme A/MNAzyme C - LS817A4/2 CTGTAGCACTCACTATAGGAACAACGAG^C?G/iA4CC7T<br>
11.4	MNAzyme substrate A (substrate for MNAzyme cleaver A)<br>
[ 00487 ] MNAzyme substrate A was cleaved by the MNAzyme cleaver A. The resultant 5' fragment denoted SLSubB was then ligated to 3LSubB by the DNAzyme ligase. The sequence of substrate A (Pre5LSubB2-FB) is below. It is internally labelled 5' of the ribobases with a 6-FAM moiety and labelled 3' of the ribobases with BHQl moiety (underlined bases). The lower case bases represent RNA and the upper case bases represent DNA.<br>
SEQ ID NO: 67 Substrate A - Pre5LSubB2-FB: CTGTAGCACTCACTAuaGGAAGAGATGAg<br><br>
11.5	DNAzyme Ligase substrate B<br>
[ 00488 ] The sequence of the 3' substrate for ligation used in this experiment is listed below (5' to 3'). The oligonucleotide has a 5' hydroxyl group. The lower case bases represent RNA and the upper case bases represent DNA.<br>
SEQ ID NO: 22 Ligase 3* Substrate B - 3LSubB: gGAACAACGAGAGGAAACCTT<br>
11.6	Substrate for MNAzyme C (Fluorescent Reporter substrate Cfor detection)<br>
[ 00489 ] The reporter substrate used in this example was SubBi-2. In the current example, SubBi-2 was end-labelled with a Quasar 670 moiety at the S' end, a BHQ2 moiety at the 3* end and designated SubBi-2-Q6B2. Cleavage of SubBi-2-Q6B2 was monitored at 66S nm with excitation at 63S tun. The sequence of SubBi-2-Q6B2 is listed below (S' to 3'): the lower case bases represent RNA and the upper case bases represent DNA.<br>
SEQ ID NO: 4 SubBi-2-Q6B2: AAGGTTTCCTCguCCCTGGGCA<br>
11.7	Assembly facilitators<br>
[ 00490 ] The assembly facilitator A recognized by MNAzyme A sensor domains was a synthetic DNA oligonucleotide with the following sequence:<br>
SEQ ID NO: 68   AF-MzCA2:<br>
GCCATTGTCGAACACCTGCTGGATGACCAGC<br>
[00491 ] The assembly facilitator C recognized by MNAzyme C sensor domains was a synthetic DNA oligonucleotide with the following sequence:<br>
SEQ ID NO: 24 AF-MzCC: CGACGACCACTCCGACGACAGTCCTATAGTGAGTGCTACAG<br>
11.8	Reaction Conditions<br>
[ 00492 ] The reaction was performed in two sequential steps, 1 and 2, in two separate tubes as described below.<br><br>
(1)	Cleavage of Pre5LSubB2-FB by MNAzyme A to generate the 5' cleavage<br>
product/Iigase substrate (SLSubB) with the required 2', 3'-cyclic phosphate end, and<br>
ligation of this fragment with 3LSubB by the DNAzyme ligase, was performed at 45°C<br>
for 2 hours in a total reaction volume of 50 jil. The reactions contained, 70 mM Tris HCl<br>
(pH 9.0 at 25°C). 50 mM MgCh, 150 mM NaCI, 2mM KCl, 40 nM R04B3/1(12), 40 nM<br>
R04A2/1(11), 200 nM Pre5LSubB2-FB and either:<br>
a)	40 nM AF-MzCA2, 200 nM DNA Ligase 7Z81-10/10 and 100 nM 3LSubB; or<br>
b)	40 nM AF-MzCA2 and 100 nM SLSubB; or<br>
c)	40 nM AF-MzCA2 and 200 nM DNA Ligase 7Z81-10/10; or<br>
d)	200 nM DNA Ligase 7Z81-10/10 and 100 nM 3LSubB; or<br>
e)	100 nM 3LSubB only; or<br>
f)	200 nM DNA Ligase 7Z81-10/10 only<br>
(2)	Fluorescent detection resulting from cleavage by MNAzyme C was performed<br>
isothermally at 55°C for 1 hour on the Cepheid Smartcycler. Reactions were initiated by<br>
the addition of the fluorescently labelled substrate SubBi-2-Q6B2. Reactioits contained<br>
10 jil of one of the previous ligation reaction mixes {1-a or 1-b or 1-c or 1-d or 1-e or 1-f)<br>
and 200 nM of partzyme B (STB5/2(21)) and 200 nM assembly facilitator (AF-MzCC)<br>
for MNAzyme C, and 200 nM of the substrate SubBi-2-Q6B2 in a total reaction volume<br>
of 25 jil. A positive control reaction (1-g) was also included where 200 nM of the control<br>
partzyme A for MNAzyme C (LS817A4/2) was also added to 10 ^1 of reaction (1-f).<br>
11.9 Results<br>
[ 00493 ] The increase in fluorescence due to cleavage of MNAzyme substrate C (SubBi-2-Q6B2) was monitored over time for reactions that had contained the following oligonucleotide reagents (Table 10).<br><br>
Table 10. Increase in fluorescence due to cleavage ofMNAzyme substrate C (SubBi-2' Q6B2) monitored over time.<br>
Step 1	[Step 2	I Result<br>
Reaction I	" KfNAzyme A	*lQ\i\ fiom step 1       An increase in<br>
partzymes	* MNAzyme C	fluorescence was<br>
(test 1-a)	"Substrate A	partzymeB	observed over time<br>
" Assembly	" Assembly	and the reaction<br>
facilitator A (target)	Facilitator C	came to completion<br>
" DNAzyme ligase	* Substrate C	in 15 minutes.<br>
* Ligase substrate B<br>
Reaction 2	" MNAzyme A	" 10 |il from step 1       No increase in<br>
partzymes	* MNAzyme C	fluorescence was<br>
(1-b control-no       "Substrate A	partzymeB	observed over<br>
DNAzyme ligase)     * Assembly	* Assembly<br>
facilitator A (target)	Facilitator C<br>
" Ligase substrate B	* Substrate C<br>
Reaction 3	" MNAzyme A	" 10 |J1 from step 1      No increase in<br>
partzymes	" MNAzyme C	fluorescence was<br>
(1 -c control - no      " Substrate A	partzyme B	observed over time<br>
3'ligase substrate     " Assembly	" Assembly<br>
B)	facilitator A (target)	Facilitator C<br>
" DNAzyme ligase	» Substrate C<br>
Reaction 4	* MNAzyme A	" 10 ^1 from step 1       A minimal increase<br>
partzymes	" MNAzyme C	in fluorescence was<br>
(1 -d control - no       * Substrate A	partzyme B	observed over time<br>
assembly facilitator  ' DNAzyme ligase	' Assembly<br>
A)	" Ligase substrate B	Facilitator C<br>
!■ Substrate C<br>
Reaction 5	* MNAzyme A	" 10 nl from step 1       No increase in<br>
partzymes	^ MNAzyme C	fluorescence was<br>
(1 -e control - no      " Substrate A	partzyme B	observed over time<br>
DNAzyme ligase      " Ligase substrate B	* Assembly<br>
and no assembly	Facilitator C<br>
facilitator A)	" Substrate C<br>
Reaction 6	" MNAzyme A	" 10 jil from step 1       No increase in<br>
partzymes	" MNAzyme C	fluorescence was<br>
(1 -f control - no       " Substrate A	partzyme B	observed over time<br>
ligase substrate B      " DNAzyme ligase	* Assembly	<br><br>
and no assembly	Facilitator C<br>
facilitator A)	* Substrate C<br>
Reaction 7	^ MNAzyme A	MO ^l from step 1       An increase in<br>
partzymes	" MNAzyme C	fluorescence was<br>
(1-g positive	" Substrate A	partzyme B	observed over time<br>
control with	* DNAzyme ligase     * Assembly	and the reaction<br>
MNAzyme C	Facilitator C	came to completion<br>
partzyme A	" Substrate C	in less than 1<br>
LS817A4/2)	* MNAzyme C	minute<br>
partzyme A<br>
LS817A4/2<br>
[ 00494 ] The fluorescence level was measured over time for step (2) of reactions 1 to 7 (Table 10). Reaction 1 (1-a), which had contained all oligonucleotide reaction components for steps (1) and (2), showed an increase in fluorescence over time. The change in fluorescence observed was greater than 484 arbitrary units in this reaction resulting in an increase from approximately 46 at time 0 to over S30 after I hour of monitoring fluorescence.<br>
[ 00495 ] In contrast Reaction 2 (1 -b), which lacked the DNAzyme ligase (7Z81-10/10) oligonucleotide for step (1) showed no increase in fluorescence over time. Reaction 3(1-c), which lacked ligase substrate B (3LSubB) but contained the DNAzyme ligase from step (1) showed no increase in fluorescence over time. Reaction 4 (1-d) which lacked assembly facilitator A (AF-MzCA2) from step (1) showed a minimal increase in fluorescence over time above the background drift observed of approximately 40 arbitrary units over the course of 1 hour of fluorescence monitoring. Reaction S (1-e) which lacked both the assembly facilitator A and the DNAzyme ligase fiom step (1) showed no increase in fluorescence over time. Reaction 6 (1-i) which lacked both the ligase substrate B and the assembly facilitator A showed no increase in fluorescence over time. Reaction 7 (1-g) was the positive control that contained the control MNAzyme C partzyme A (LS817A4/2), and showed an increase in fluorescence over time whereby the reaction was completed in less than 1 minute. Together these reactions indicate that the following events have occurred in Reaction 1 (1-a). •<br><br>
[ 00496 ] Firstly, MNAzyme A cleaves Substrate A (Pre5LSubB2-FB) in the presence of specific target assembly facilitator A (AF-MzCA2) producing two fragments, one of which was the 5' fragment SLSubB (CTGTAGCACTCACTAua) (SEQ ID NO: 40) that has a 2', 3'-cyclic phosphate terminus. Secondly, SLSubB was ligated to a second DNAzyme oligonucleotide ligase substrate B (3LSubB) present in the reaction mix of step (1) and this resulted in the formation of a new oligonucleotide (ligation product) with the sequence of CTGTAGCACTCACTAuagGAACAACGAGAGGAAACCTT (SEQ ID NO: 51) (where upper case represent DNA bases and lower case represent RNA bases). This ligation product in turn functioned as a partzyme for MNAzyme C. This newly ligated partzyme A associated with MNAzyme C partzyme B and assembly facilitator C to create MNAzyme C. MNAzyme C cleaved MNAzyme substrate C (SubBi-2-Q6B2) resulting in separation of a fluorophore /quencher dye pair thus causing an increase in fluorescence.<br>
[ 00497 ] The "no ligation" controls (Reaction 2 (1-b) and 3 (l-c)) demonstrated that the ligation of the 5' cleavage (SLSubB) fragment with 2', 3'-cyclic phosphate terminus to a second ligase substrate B (3LSubB) is essential for the formation of the partzyme component of MNAzyme C.<br>
[ 00498 J The "no assembly facilitator A (AF-MzCA2)" controls (Reactions 4 (1-d), 5 (1-e) and 6 (I-f)) demonstrated that cleavage of MNAzyme substrate A (PreSLSubB2-FB) by MNAzyme A and generation of a 5' fragment with the 2', 3'-cyclic phosphate terminus was essential for subsequent ligation and formation of the partzyme required for assembly and activity of MNAzyme C.<br>
[ 00499 ] The cascade reaction in this example allowed the detection of the presence of an assembly facilitator, in this example a target nucleic acid sequence. The cascade reaction allowing detection uses two types of nucleic acid enzymes, namely DNAzymes which can ligate substrates and MNAzymes which can cleave substrates. Multiple turnover at each enzymatic step in the cascade could allow signal amplification.<br>
[ 00500 ] The cascade demonstrated in this example and illustrated in Figure 9 could be extended and used to create a feedback amplification cascade. If the sequence of S' cleavage product C were the same as S' cleavage product A then this product could also<br><br>
serve as a S' substrate B and a feedback cascade reaction could be initiated. In this reaction, MNAzyme cleaver C could be constantly generating 5' substrate B which could in turn be ligated by the DNAzyme ligase B to create more partzymes for formation of more MNAzyme cleaver C. This strategy could provide a mechanism for feedback amplification following initiation of a reaction by a target analyte which could direct the assembly of MNAzyme cleaver A. The strategy could allow detection of target analytes followed by signal ampliflcation by a feedback cascade using two types of nucleic acid enzymes namely DNAzyme ligases and MNAzyme cleavers.<br>
[ 00501 ] Further, it would be obvious to one skilled in the art that the DNAzyme ligase could ligate two fragments and create several possible enzymes, enzyme components or substrates useful in this cascade which include but are not limited to, a) a partzyme for a second MNAzyme (e.g. an MNAzyme cleaver as shown in this example and labelled as MNAzyme C in Figure 9); or b) a new assembly facilitator for a second MNAzyme as described in example 12, and labelled as MNAzyme Cleaver 2 in Figure IS; or c) a new DNAzyme capable of modifying a substrate or substrates (e.g. by cleavage, ligation etc) or a substrate suitable for modification by an MNAzyme or DNAzyme. Further it would be obvious to one skilled in the art that the DNAzyme with ligase activity could be substituted with an MNAzyme with ligase activity as demonstrated in example 7 and 9. The strategy is versatile and could be performed as either a sequential cascade of the first and second steps or as a feedback cascade reaction where step 2 (Figure 9 (iii)) creates a component for step 1 (Figure 9 (i) and (ii)). Additionally, step 2 (Figure 9 (iii)) could create a component for a further subsequent reaction.<br>
Example 12: An example of a cascade reaction which exploits the activity of multiple<br>
nucleic acid enzymes.<br>
[ 00S02 ]   The general strategy used in this example is outlined in Figure IS and can<br>
also form the basis of a signal amplification technique.  In the first step, an MNAzyme<br>
(MNAzyme cleaver 1) forms in the presence of assembly facilitator (e.g. a target nucleic<br>
acid) and is used to cleave a substrate (MNAzyme substrate 1) thus generating a 5'<br>
cleavage product which has a 2\ V- cyclic phosphate end. In the second step a<br>
DNAzyme ligase is used to ligate the cleavage product generated in step one to another<br><br>
oligonucleotide ligation substrate thus creating a ligation product that functions as an assembly facilitator. In the third step, the assembly facilitator ligation product generated in step two participates in the formation of a new MNAzyme (MNAzyme cleaver 2), which can cleave a substrate (NfNAzyme substrate 2). In this example the labelling of MNAzyme substrate 2 with a fluorophore and quencher dye pair allowed fluorescent monitoring of the reaction.<br>
12.1	Partzymesfor MNAzyme cleaver 1<br>
[ 00503 ] In the first step, MNAzyme cleaver 1 formed in the presence of an assembly facilitator (AF-MzCA2) and was used to cleave MNAzyme substrate 1 (Pre5LSubB2-FB). The sequences of the MNAzyme cleaver 1 partzymes (R04A2/l(ll) and R04B3/1(12)) are listed below. The bases underlined form part of the catalytic core, bases in bold hybridize with the assembly facilitator and bases in italics hybridize to the substrates.<br>
SEQ ID NO: 69 Partzyme A/MNAzyme 1 - R04A2/1(11); GCTGGTCATCCAGCAGCGGICGAA/ir^GrCJ^GrCPC SEQ ID NO: 70 Partzyme B/MNAzyme 1 - R04B3/1(12): CTCA fCrCTTCTCCGAGCGTGTTCGAC AATGGC<br>
12.2	DNAzyme with Ligase activity<br>
[ 00504 ] The DNAzyme ligase used in this example has previously been reported to ligate RNA through the formation of a 2'-5' phosphodiester linkage from a 2', 3'-cyclic phosphate and a 5'-hydroxyl group (Prior et al, 2004). Apart fix)m the requirement of a 5' substrate with a 2\ 3'-cyclic phosphate end, the DNAzyme ligase used in this example also requires a specific sequence motif at the ligation junction.<br>
[ 00505 ] In the second step, the DNAzyme ligase 7Z81 (7Z81-10/10) with substrate arms designed to hybridize to the substrate sequences 5LSub6 and 3LSubB was used in this experiment and the sequence is listed below (5' to 3'). The bases underlined form the catalytic core and bases in italics hybridize to the substrates.<br>
173<br><br>
SEQ ID NO: 17 DNAzyme ligase - 7Z81-10/10: CCTCTCGTTGACGGCGGAGTGATTGGGAGGTTAGCTCTAGrGAGTGC<br>
12.3	Partzymesfor MNAzynte cleaver 2<br>
[ 00S06 ] In the third step the ligated product functions as a new assembly facilitator, which together with a pair of partzymes can form MNAzyme cleaver 3 and cleave a fluorescently labelled substrate 3 (SubBi-3-TxB2). The following sequences correspond to the partzymes, which form components of MNAzyme cleaver 2. The bases underlined form part of the catalytic core, bases in bold hybridize with the assembly facihtator and bases in italics hybridize to. the substrates.<br>
SEQ ID NO: 85 Partzyme A/MNAzyme 2 - LIGFACA5/3: AAGGTTTCCTCTCGTTGTTCCTACAACGAGG77X/7X?C7'Cr<br>
SEQ ID NO: 86 Partzyme B/MNAzyme 2 - LIGFACB6/3: CGGTTGGrG/iGGCTAGCTATAGTGAGTGCTACAG<br>
12.4	MNAzyme substrate 1 (substrate for MNAzyme cleaver 1)<br>
{00507 ] MNAzyme substrate 1 was cleaved by the MNAzyme cleaver 1. The resultant 5' fragment denoted 5LSubB was then ligated to 3LSubB by the DNAzyme ligase. The sequence of substrate 1 (Pre5LSubB2-FB) is below. It is internally labelled 5* of the ribobases with a 6-FAM moiety and labelled 3' of the ribobases with BHQI moiety (underlined bases). The lower case bases represent RNA and the upper case bases represent DNA.<br>
SEQ ID NO: 67 Substrate I - Pre5LSubB2-FB: CTGTAGCACTCACTAuaGGAAGAGATGAe<br><br>
12.5	DNAzyme Ligase substrate<br>
[ 00508 ] The sequence of the 3' substrate for ligation used in this experiment is listed below (5' to 3'). The oligonucleotide has a S' hydroxyl group. The lower case bases represent RNA and the upper case bases represent DNA.<br>
SEQ ID NO: 22 Ligase 3' Substrate - 3LSubB: gGAACAACGAGAGGAAACCTT<br>
12.6	MNAzyme substrate 2 (Fluorescent Reporter substrate for MNAzyme cleaver 2)<br>
[ 00509 ] The reporter substrate used in this example was SubBi-3. In the current example, SubBi-3 was end-labelled with a Texas Red moiety at the 5' end, a BHQ2 moiety at the 3' end and designated SubBi-3-TxB2. Cleavage of SubBi-3-TxB2 was monitored at 610 nm (TxB2 emission wavelength) with excitation at S85 lun (TxB2 excitation wavelength). The sequence of SubBi-3-TxB2 is Usted below (5' to 3'); the lower case bases represent RNA and the upper case bases represent DNA.<br>
SEQ ID NO: 87 SubBi-3-TxB2: CAGCACAACCguCACCAACCG<br>
12.7	Assembly Facilitator sequence for MNAzyme cleaver 1<br>
[ 00510 ] The assembly facilitator recognized by MNAzyme cleaver 1 sensor domains was a synthetic DNA oligonucleotide with the following sequence:<br>
SEQ ID NO: 68 AF-MzCA2:<br>
GCCATTGTCGAACACCTGCTGGATGACCAGC<br>
12.% Reaction Conditions<br>
[ 00511 ] The reaction was performed in three sequential steps (i), (ii) and (iii) in three separate tubes as described below.<br>
(i) Cleavage of Pre5LSubB2-FB by MNAzyme cleaver 1 to generate the 5' cleavage product/ligase substrate (5LSubB) with the required 2\ 3'-cyclic phosphate end was performed at 40'C for 66 minutes in a total reaction volume of 100 nl. The reactions<br><br>
contained 50 mM Tris HCI (pH 9.0 at 25°C), 50 mM MgCb. 150 mM NaCl, 2 mM KCl, 50 nM R04B3/1(12), 50 nM R04A2/1(11) and either:<br>
a)	1000 nM Pre5LSubB2-FB, 50 nM AF-MzCA2; or<br>
b)	1000 nM Pre5LSubB2-FB; or<br>
c)	50 nM AF-MzCA2.<br>
(ii) Ligation was performed at 40°C for 2 hours in a total reaction volume of 50 nl. Reactions contained 25 \il of previous cleavage reaction mix (i-a) or (i-b), or (i-c), 500 nM 3LSubB, in a reaction with a final concentration of 150 mM NaCl, 2 mM KCl, 50 mM Tris HCI (pH 9.0 at 25'C) and 50 mM MgCb and:<br>
a)	500 nM of DNAzyme ligase 7Z81 -10/10, or<br>
b)	no DNAzyme Hgase (no ligation control)<br>
(iii) Fluorescent detection due to MNAzyme cleavage resulting firom assembly of partzymes for MNAzyme cleaver 2 in the presence of the assembly facilitator formed by ligation of the cleavage product of Pre5LSubB2-FB (SLSubB) and the ligation substrate (SLSubB) was performed isothermally at 50°C for 2 hours on the Cepheid Smartcycler. Reactions were initiated by the addition of the fluorescently labelled substrate SubBi-3-TxB2. Duplicate reactions contained 10 ^1 of one of previous ligation reaction mixes ((i-a/ii-a) or (i-a/ii-b) or (i-b/ii-a) or (i-b/ii-b) or (i-c/ii-a) or (i-c/ii-b)) and 200 nM of partzyme A (LIGFACA5/3) and 200 nM partzyme B (LIGFACB6/3) for MNAzyme cleaver 2, and 400 nM of the substrate SubBi-3-TxB2 in a total reaction volume of 25 ^l.<br>
12.9 Results<br>
[00512] The increase in fluorescence due to cleavage of MNAzyme substrate 2 (SubBi-3-TxB2) was monitored over time for reactions that had contained the following oligonucleotide reagents (Table 11).<br>
Table U. Increase in fluorescence due to cleavage of MNAzyme substrate 2 (SubBi-3-<br>
T)cB2) monitored over time.<br>
btep (i)             [Step (ii)              fctep (iii)                I Result<br>
"MNAzyme 1    * 25 jil from         10 |xl from step      An increase in<br>
Reaction 1	partzymes	step (i)	(jQ	fluorescence<br><br>
^Substrate 1	*Ligase	'MNAzyme 2         was observed<br>
(Test i-a/ii-a)    "Assembly         substrate	partzymes	over time<br>
facilitator 1	*DNAzyme	*Substrate 2<br>
(target)            ligase<br>
Reaction 2       "MNAzyme 1	* 25 ^1 from	10^1 from step      No increase in<br>
partzymes        step(i)	(ii)	fluorescence<br>
(Control i-a/ii-  "Substrate 1	*Ligase	MNAzyme 2         was observed<br>
b-no              "Assembly         substrate	partzymes	overtime<br>
DNAzyme         facilitator 1	'Substrate 2<br>
ligase)                (target)<br>
Reactions       "MNAzyme 1	*25}ilfrom	10^1 from step      No increase in<br>
partzymes         step (i)	(ii)	fluorescence<br>
(Control i-b/ii-  "Substrate 1	*Ligase	MNAzyme 2          was observed<br>
a-no                                       substrate	partzymes	overtime<br>
Assembly	'DNAzyme	Substrate 2<br>
facilitator 1	ligase<br>
(target))	<br>
Reaction 4       "MNAzyme 1	* 25^1 from	lO^il from step      No increase in<br>
partzymes        step(i)	(ii)	fluorescence<br>
(Control i-b/ii- "Substrate 1	*Ligase	MNAzyme 2         was observed<br>
b-no                                       substrate	partzymes	overtime<br>
Assembly	Substrate 2<br>
facilitator 1<br>
(target) and no<br>
DNAzyme<br>
ligase)	<br>
Reactions       "MNAzyme 1	•25|ilfix)m	iO^lfiomstep       No increase in<br>
partzymes         step (i)	(ii)	fluorescence<br>
(Control i-c/ii-  "Assembly	'Ligase	MNAzyme 2         was observed<br>
a - no                facilitator 1       substrate	partzymes	over time<br>
Substrate 1)        (target)	'DNAzyme	Substrate 2<br>
ligase<br>
Reaction 6       "MNAzyme 1	'25filfrom	' lOjil from step      No increase in<br>
partzymes	step(i)	(ii)                       fluorescence<br>
(Control i-c/ii-  "Assembly	'Ligase	MNAzyme 2         was observed<br>
b-no                facilitator 1	substrate	partzymes             over time<br>
Substrate 1         (target)	Substrate 2<br>
and no<br>
DNAzyme<br>
I'gase)	I	I	I	I	<br><br>
[ 00513 J The fluorescence level was measured over time for step (iii) of reactions 1, 2, 3, 4, 5 and 6 (performed in duplicate) (Table 10). Reaction 1 (i-a/ii-a), which had contained all oligonucleotide reaction components for steps (i), (ii) and (iii), showed an increase in fluorescence over time. The change in fluorescence observed was greater than 900 arbitrary units in this reaction resulting in an increase from approximately 180 at time 0 to over 1100 after 2 hours.<br>
[ OOS14 ] In contrast, reactions 2, 3, 4, S and 6 showed no increase in fluorescence over time above the background drift observed of approximately 50 arbitrary units over the course of 2 hours of fluorescence monitoring. As such. Reaction 2 (i-a/ii-b), which lacked the ligase DNAzyme oligonucleotide for step (ii) showed no increase in fluorescence over time. Reaction 3 (i-b/ii-a), which lacked the assembly facilitator 1 (AF-MzCA2) but contained the DNAzyme ligase fix)m step (ii) showed no increase in fluorescence over time. Reaction 4 (i-b/ii-b), which lacked both the assembly facilitator 1 (AF-MzCA2) and the DNAzyme ligase from step (ii) showed no increase in fluorescence over time. Reaction 5 (i-c/ii-a), which lacked the Substrate 1 (PreSLSubB2-FB) but contained the DNAzyme ligase from step (ii) showed no increase in fluorescence over time. Reaction 6 (i-c/ii-b), which lacked both the Substrate 1 (Pre5LSubB2-FB) and the DNAzyme ligase showed no increase in fluorescence over time.. Together these reactions indicate that the following events have occurred in reaction 1.<br>
[00515] Firstly, MNAzyme cleaver 1 cleaves Substrate 1 (Pre5LSubB2-FB) in the presence of specific assembly facilitator 1 (AF-MzCA2) producing two fragments, one of which was the 5' fragment SLSubB (CTGTAGCACTCACTAua) (SEQ ID NO: 40) that has a 2\ 3'-cyclic phosphate terminus. Secondly, SLSubB was ligated to a second DNAzyme oligonucleotide ligase substrate (3LSubB) present in the reaction mix of step (ii) and this resulted in the formation of a new oligonucleotide (ligation product) with the sequence of CTGTAGCACTCACTAuagGAACAACGAGAGGAAACCTT (SEQ ID NO: 51) (where upper case represent DNA bases and lower case represent RNA bases). This ligation product in turn functioned as an assembly facilitator for MNAzyme cleaver 2. This newly ligated assembly facilitator associated with the partzymes to create MNAzyme cleaver 2.   MNAzyme cleaver 2 cleaved MNAzyme substrate 2 (SubBi-3-<br><br>
TxB2) resulting in separation of a fluorophore/quencher dye pair thus causing an increase in fluorescence.<br>
[ 00516 ] The "no ligation" controls (Reaction 2,4 and 6) demonstrated that the ligation of the 5' cleavage fragment (SLSubB) with 2', 3'-cyclic phosphate terminus to a second ligase substrate (3LSubB) is essential for the formation of the assembly facilitator component of MNAzyme cleaver 2.<br>
[00517] The "no assembly facilitator (AF-MzCA2)" controls (Reaction 3 and 4) demonstrated that cleavage of Substrate 1 (Pre5LSubB2-FB) by MNAzyme cleaver 1 and generation of a 5' fragment with the 2', 3'-cyclic phosphate terminus (SLSubB) was essential for subsequent ligation and formation of an assembly facilitator required for assembly of MNAzyme cleaver 2.<br>
[ 00518 ] The strategy demonstrated in this example could be used to create a feedback signal amplification cascade. If cleavage by MNAzyme cleaver 2 generated the same 5' cleavage product as was generated by MNAzyme cleaver 1 then this cleavage product could also be ligated to form a ligation product that could in turn function as another new MNAzyme component (e.g. partzyme, assembly facihtator or substrate) for the assembly of additional MNAzyme cleaver 2 complexes.<br>
[ 00519 ] Further, it would be obvious to one skilled in the art that the DNAzyme ligase could ligate two fragments and create several possible enzymes or enzyme components useful in this cascade which include but are not limited to, a) a new assembly facilitator for a second MNAzyme cleaver (as shown in this example) or b) a new partzyme for a second MNAzyme as shown in examples 6, 7, 9, and 11, or c) a new DNAzyme capable of modifying a substrate or substrates (e.g. by cleavage, ligation etc). Further it would be obvious to one skilled in the art that the DNAzyme with ligase activity could be substituted with an MNAzyme with ligase activity as demonstrated in example 7 and 9. The strategy is versatile and could be performed as either a sequential cascade of the first, second and third steps or as a feedback cascade reaction where step 3 creates a component for step 2 and step 2 creates components for step 3. Additionally, step 3 could create a component for a further subsequent reaction.<br><br>
Example 13: A cascade reaction with possible feedback<br>
[ 00520 ] A schematic representation of one variation of a feedback amplification cascade is illustrated in Figure 16. The initial steps of a cascade involving NfNAzyme cleavage, DNAzyme ligation and NdNAzyme cleavage are illustrated in Figure 15 and demonstrated in example 6,1,9, 11 and 12. The schema described in this example and illustrated in Figure 16 extends the strategy of the cascade in example 6, 7, 9, 11 and 12 and describes a strategy for a feedback amplification cascade.<br>
[00521 ] As illustrated in Figure 16, in a first step an MNAzyme cleaver 1 could assemble only in the presence of an assembly facilitator (e.g. a target nucleic acid) and then cleave MNAzyme substrate 1 to release a 5' cleavage product with a 2\ 3' cyclic phosphate terminus. In a second step a DNAzyme ligase (or an assembled MNAzyme ligase) could ligate the 5' cleavage product from the first step to a 3' ligation substrate to create a ligation product which could serve as an assembly facilitator for another MNAzyme cleaver 2. In a third step an assembly facilitator formed by ligation in the second step could direct the assembly of partzymes which could form an MNAzyme cleaver 2 which could cleave substrate 2 into two products, a 5' product cleavage with a 2', 3' cyclic phosphate terminus and 3' cleavage product. A detectable signal could be generated following cleavage of substrate 2 if this substrate were labelled, for example, with a fluorophore and quencher dye pair.<br>
[ 00522 ] Further, if the sequence of either of the cleavage products of MNAzyme cleaver 2 was usefiil as an assembly facilitator component for an MNAzyme cleaver 3 then this could initiate a feedback cascade. In this case MNAzyme cleaver 3 could have different sensor arms than MNAzyme cleaver 1 but could have the same substrate arms as MNAzyme cleaver 1. Thus if MNAzyme cleaver 3 were assembled in the presence of an assembly facilitator component generated by cleavage by the MNAzyme cleaver 2, and if MNAzyme cleaver 3 cleaved substrate 1 then the 5' cleavage product of MNAzyme cleaver 3 could also serve as a 5' substrate for the DNAzyme ligase (or the assembled MNAzyme ligase) and a feedback amplification cascade reaction could be initiated. In this reaction MNAzyme cleaver 3 would constantly generate 5' cleavage product which in turn could serve as a substrate for ligation by the DNAzyme ligase (or<br><br>
the assembled MNAzyme ligase) to create more assembly facilitators that could direct the assembly of more MNAzyme cleaver 2. This strategy could provide a mechanism for feedback signal amplification following initiation of a reaction by an assembly facilitator (eg a target analyte) which allowed assembly of MNAzyme cleaver 1. The strategy could allow detection of one or more assembly facilitators (eg target analytes) followed by signal amplification using a ONAzyme or an MNAzyme which can ligate substrates and MNAzymes which can cleave a substrate.<br>
[ 00523 ] In a further variation the DNAzyme (or MNAzyme) with ligase activity could ligate fragments to form a DNAzyme capable of cleaving substrate 2 to create an assembly facilitator component for an MNAzyme 3. As such, in this variation MNAzyme cleaver 2 could be replaced with a DNAzyme cleaver created by ligation from the second step.<br>
[ 00S24 ] It would also be obvious to one skilled in the art that signal could be generated by cleavage of substrate 1 and/or substrate 2 provided the substrates were labelled, for example, with fluorophore quencher dye pairs. In this case, cleavage would result in an increase in fluorescence. Further, it would be possible to measure a change in fluorescence following ligation of the ligation substrate and the 5' product of MNAzyme cleaver 1 and/or MNAzyme cleaver 3 if the fragments were each labelled with either a fluorophore or a quencher. In this scenario ligation would result in a change in fluorescence either in solution or at a specific location in an assay where, for example, one ligatable substrate is attached to a solid support.<br>
[ 00S25 ] The assembly facilitator component generated by cleavage (or ligation) in step 2 or step 3 by an MNAzyme or a DNAzyme could be any one of numerous designs. By way of example, they could resemble any the configurations for assembly facilitator and/or partzyme sensor arms as illustrated in Figure 6.<br>
[ 00526 ]   In another variation of feedback cascade strategy wherein the products of MNAzyme cleaver 2 could be used as an assembly facilitator component for an MNAzyme cleaver 3, the MNAzyme cleaver 3 could have a modified structxire. In this variant, MNAzyme cleaver 3 could have different sensor arms than MNAzyme cleaver I but it may have one substrate arm (on one of the partzymes) for MNAzyme 3 that is the<br><br>
same as the sequence of the partzyme of MNAzyme cleaver 1 which interact with that portion of the sequence on substrate 1 (the 5' portion of the substrate) which is subsequently ligated by the ligase. As such MNAzyme cleaver 3 would cleave a substrate (cleavable substrate 3) which has the same sequence at the 5' portion but which has a different portion at the 3' end or vice versa with respect to cleavable substrate 1. The cleavage site of substrate 3 would lie at the junction of those sequences which are shared between the cleavable substrates 1 and 3 and those which differ.' In this case, when MNAzyme cleaver 3 was assembled in the presence of an assembly facilitator component generated by cleavage by the MNAzyme cleaver 2, and if MNAzyme cleaver 3 cleaved substrate 3 then the 5' cleavage product of MNAzyme cleaver 3 could also serve as a 5' substrate for the DNAzyme ligase (or the assembled MNAzyme ligase) and a feedback amplification cascade reaction could be initiated. In this reaction MNAzyme cleaver 3 would constantly generate S' cleavage product which in turn could serve as a substrate for ligation by the DNAzyme ligase (or the assembled MNAzyme ligase) to create more components, for example, an assembly facilitator that could direct the assembly of more MNAzyme cleaver 2. This strategy could provide a mechanism for feedback signal amplification following initiation of a reaction by an assembly facilitator (eg a target analyte) which allowed assembly of MNAzyme cleaver 1. The strategy could allow detection of an assembly facilitator (eg a target analyte) followed by signal amplification using a DNAzyme or an MNAzyme which can ligate substrates and MNAzymes which can cleave a substrate.<br>
[ 00527 ] It would also be understood by one skilled in the art that nucleic acid enzymes used in the cascade could create different components that could be used in a subsequent reaction such as in a feedback situation. By way of example, with reference to Figure 16 an MNAzyme cleaver 2 could create a stabiliser arm for MNAzyme cleaver 3, instead of the assembly facilitator component as illustrated. In another variation, MNAzyme cleaver 2 could be replaced by an MNAzyme ligase that could create, for example a partzyme or assembly facilitator for MNAzyme cleaver 3, or a DNAzyme cleaver as opposed to the illustration in Figure 16 whereby an MNAzyme cleaver 2 creates an assembly facilitator component.<br><br>
[ 00528 ] In another variant strategy the DNAzyme (or MNAzyme) ligase could ligate fragments and create a new partzyme capable of associating with another partzyme and an assembly facilitator present in the reaction thus forming an MNAzyme cleaver 2 which could cleave substrate 2 to create an assembly facilitator component for an MNAzyme 3.<br>
[ 00529 j Variations described can also be used as appropriate in other cascades throughout the specification. Further variations described elsewhere in the specifications could also be applied here.<br>
[ 00530 ] The nucleic acid enzyme cascades, including the feedback cascades, allow signal generation following detection of target analytes. The first MNAzyme in the cascade may be an MNAzyme or apta-MNAzyme thus MNAzyme systems can be designed for detection of nucleic acid targets and non-nucleic acid analytes, for example proteins or lipids.<br>
[ 00531 j Further, feedback cascades could further amplify the output or detectable signal. Signal amplification cascades using nucleic acid enzymes have advantages over target amplification and detection technologies, for example real time PCR. While target amplification technologies are powerful tools which have been widely used in research and/or in clinical diagnostics, each has inherent disadvantages. They all require the use of protein enzymes (e.g. DNA polymerase, RNA polymerase, reverse transcriptase, and/ or ligase). The inclusion of protein enzymes increases the complexity and cost of reagent manufacture and decreases the shelf life of kits containing reagents. Odier associated technical challenges include contamination by replicons (target amplicons) from previous reactions leading to false positive signal, and/or background signal caused by replication of primer sequences (primer-dimers) or background caused by target-independent ligation.<br>
Example 14: Testing MNAzyme activity from a series ofpartxyme pairs which contain variant partial catalytic core sequences derived from the 10:23 catalytic core<br>
[ 00532 ] Multi-component nucleic acid enzymes (MNAzymes) can be made which incorporate partial sequences from a variety of in vitro evolved DNAzymes. Active MNAzymes, based on partial sequences from the 8:17 and 10:23 DNAzymes, have been<br><br>
demonstrated. Further, several alternative partzyme designs based on the 8:17 and 10:23 DNAzymes have been shown to either have or lack cleavage activity (PCT/AU2006/001473, Johnson &amp; Johnson Research Pty Limited). This example identifles both active and inactive partzyme sequences based on partial catalytic core sequences from the 10:23 DNAzyme which can cleave a substrate. Further, the example provides a general protocol for the steps necessary to identify the optimal place(s) to split a catalytic core sequence such that, when the partial catalytic core sequences are incorporated into partzymes, functional active MNAzymes are generated.<br>
[ 00533 ] Other DNAzymes, either known or evolved in vitro, could be subjected to a similar analysis to that described in this example and in examples 7 and 9. Therefore these examples provide a method that one skilled in the art could use to identify partial catalytic core sequences for active MNAzymes capable of many functions including cleavage (this example) or ligation (examples 7 and 9), or other in vitro evolved ftinctions, including but not limited to phosphorylation of nucleic acids, nucleic acid capping, amino acid adenylation, cofactor synthesis, RNA polymerization, template-directed polymerization, RNA-protein conjugation, aldol reaction, alcohol oxidation, aldehyde reduction, purine and pyrimidine nucleotide synthesis, alkylation, amide synthesis, urea synthesis, formation of peptide bonds, peptidyl-RNA synthesis, acyl transfer, aminoacylation, carfjonate hydrolysis, phosphorothioate alkylation, porphyrin metallation, formation of carbon-carbon bonds, Pd nanoparticle formation, biphenyl isomerization, formation of ester bonds, formation of amide bonds, DNA deglycosylation, thymine dimer photoreversion and phosphoramidate cleavage.<br>
14.1        Partzyme Oligonucleotides<br>
[ 00534 ] The method in this example was used to investigate which positions within the 10:23 catalytic core sequence are suitable for splitting into partial catalytic core sequences which, upon itKSorporation into partzymes, result in functionally active MNAzymes. The 10:23 sequence was split at various points and then the two partial sequences were incorporated into a series of partzyme pairs which were designed to cleave a substrate in the presence of target (human RPLPO gene). The partial catalytic<br><br><br><br>
[ 00535 ] In this experiment the series of partzyme pairs were all synthesized with sensor arms designed to hybridise to exon 5 of the human RPLPO gene, and with substrate arms directed against the substrate, SubBi-2. The part^^roe pairs used in this experiment were synthesized by Sigma-Proligo and their sequences are listed below (5' to 3'). The bases underlined form part of the catalytic core of the assembled MNAzyme, bases in bold hybridise with the nucleic acid target and bases in italics hybridise to the substrate. The "-P" indicates 3' phosphorylation of the oligonucleotide.<br>
RPLPO Partzvme Pair A4:B5 SEQ  ID NO:   6  R05A4/2-P<br>
CAAACGAGTCCTGGCCTTGTCTACAACGAGAGGAAACCTT -P SEQ  ID NO:   88 R05B5(16)/2-P TGCCCAGGGAGGCTAGCTOTGGAGACGGATTACA  -P<br>
RPLPO Partzvme Pair A5:B6 SEQ  ID NO:   89 R05A5/2(22)-P<br>
CAAACGAGTCCTGGCCTTGTCTTACAACGAGAGGAMCCrr -P SEQ  ID NO:   90 R05B6(16)/2-P rGCCCi^GGGAGGCTAGCGTOGAGACGGATTACA -P<br>
RPLPO Partzvme Pair A6:B7 SEQ  ID NO:   91  R05A6(22)/2-P CAAACGAGTCCTGGCCTTGTCTACGAGAGGAAACCTT -P SEQ  ID NO:   92   R05B7(16)/2-P TGCCCAGGGAGGCTAGCTACAGTGGAGACGOATTACA -P<br>
RPLPO Partzvme Pair A7:B8 SEQ  ID NO:   93  R05A7(22)/2-P CAAACGAGTCXJTGGCCTTGTCTCAACQAGAGGAflACCrr -P SEQ  ID NO:   94  R05B8(16)/2-P TGCCCAGGGAGGCTAGCTAGTGGAGACGGATTACA -P<br><br>
RPLPO Partzvme Pair A8:B9 SEQ ID NO: 95 R05A8(22)/2-P<br>
CAAACaAOTCCTGCCCTTOTCTCTACAACGAGAGGAAACCTT -P SEQ ID NO: 96 R05B9(16)/2-P TGCCCAGGGAGGCTAGGTGQAGACQQATTACA -P<br>
RPLPO Partzvme Pair A9:B1Q<br>
SEQ ID NO: 97 R05A9(22)/2-P<br>
CAAACGAOTCCTGGCCTTGTCTGCTACAACGAGAGGAAACCTT -P SEQ ID NO: 98 RO5Bl0{16)/2-P TGCCCAGGGAGGCTAOTGOAGACOOATTACA -P<br>
14.2.	Reporter Substrate<br>
[ 00536 ] The reporter substrate for this example is SubBi-2 with the sequence, 5' to 3', as below. In the current example, SubBi-2 was end-labelled with a 6-FAM moiety at the 5' end and a BHQl moiety at the 3' end and was designated SubBi-2-FB. Cleavage of SubBi-2-FB was monitored at 530nm (FAM emission wavelength) with excitation at 485nm (FAM excitation wavelength). The lower case bases represent RNA and the upper case bases represent DNA.<br>
SEQ ID NO: 4 SubBi-2-FB AAGGTTTCCTCguCCCTGGGCA<br>
14.3.	PCR Primers for the amplification ofexon 5 of the human RPLPO gene.<br>
[ 00537 ]   The sequences of the primers are shown, 5' to 3', below.<br>
SEQ ID NO: 99    5' Primer 5R05/2<br>
GCTACCCAACTGTTGCATC<br>
SEQ ID NO: 100   3' Primer 3R05/2<br>
AGCAGCCACAAAGGCAGA<br><br>
14.4.	Target template<br>
[ 00538 ] Human genomic DNA extracted from K562 cells was used as template in the PCR reaction.<br>
14.5.	Reaction conditions<br>
[ 00539 ] Real time amplification of the target sequence and detection of catalytic activity of the partzyme pairs was conducted in a 25 ^L reaction cycled on an ABI 7700 thermocycler (Applied Biosystems). The cycling parameters were 95°C for 7 minutes, 10 cycles of 95°C for 15 seconds and 65°C for 30 seconds (with a 1° C decrease in temperature per cycle), and finally 50 cycles of 95'C for 15 seconds and SO'C for 30 seconds. Each reaction contained 0.04 ^M 5R05/2 and 0.2 \M of 3R05/2, 10 niM MgCh, 50 \M each dNTP (dATP, dCTP, dGTP, dTTP), Ix Immobuffer (Bioline), 0.2 \xM SubBi-2-FB, Ix Rox reference dye (Invitrogen), 10 Units of Rnasin (Progema) and 1 Unit of Immolase Polymerase (Bioline) and 100 ng of genomic DNA. In addition each reaction contained a pair of partzymes 0.2 ^M of partzyme A and 0.2 ^M of partzyme B (RPLPO Partzyme Pair A4:B5 or A5:B6 or A6:B7 or A7:B8 or A8:B9 or A9:B10).<br>
14.6.	Results<br>
[ 00540 ] Using a real time MNAzyme-PCR method, catalytic activity was detected from three of the six RPLPO partzyme pairs. Partzyme pair A4:B5 and A5:B6 showed high levels of catalytic activity, allowing detection of target after 22 cycles (Table 13). The A7:B8 partzyme pair was also active, although less active than A4;B5 and A5:B6. No catalytic activity was detected from partzyme pairs A6:B7, A8:B9 or A9:B10 under the conditions of this experiment.<br>
Table 13: Threshold Cycle (Ct) values obtained using various partzyme pairs<br>
Core Split	I	Ct	I	Comment<br>
(see table above, this<br>
	example)	<br>
A4:BS (T8-A9)	19.3	This combination of partial catalytic core sequences in these<br>
partzymes is compatible with formation of active MNAzymes.<br>
AS:B6(C7-T8)               21.6	This combination ofpaitial catalytic core sequences in these<br>
	partzymes is compatible with formation of active MNAzymes.<br><br>
A6:B7 (A 11-Al 2)	No signal at SO    This combination of partial catalytic core sequences in these<br>
cycles	partzymes is not compatible with formation of active<br>
	MNAzymcs under these experimental conditions.	<br>
A7:B8 (A9-C10)	31.7	This combination ofpaitial catalytic core sequences in these<br>
	partzymes is compatible with fonnation of active MNAzymes.<br>
A8:B9 (G6-C7)	No signal at 50   This combination of partial catalytic core sequences in these<br>
cycles	partzymes is not compatible with formation of active<br>
	MNAzymes under these experimental conditions.	<br>
A9:B10 (AS-G6)	No signal at SO   This combination ofpartial catalytic core sequences in these<br>
cycles                partzymes is not compatible with formation of active<br>
	I	I MNAzymes under these experimental conditions.	<br>
The Ct values are averaged from triplicate reactions, when the threshold fluorescence level was set at 0.2<br>
and the baseline background fluorescence was subtracted between cycles I and 14.<br>
[ 00541 ] This example demonstrates that there arc several ways of splitting the catalytic core of the 10:23 DNA2yme such that when the partial core sequences are incorporated into partzymes, they can form catalytically active MNAzymes. The example also demonstrates that not all partial core sequences form active MNAzymes when incorporated into partzymes. The method demonstrated in this example can be used to identify partial core sequences suitable for incorporation into partzymes capable of assembling into active MNAzymes.<br>
Patents and Patent Publications:<br>
PCT International Publication No. WO 99/50452 Other References:<br>
Achenbach, J., Nutiu, R. and Li, Y. (2005) Structure-switching allosteric deoxyribozymes. Analytica Chimica Acta. 534(1): 41-51.<br>
Benenson, Y., Paz-Elizur, T., Adar, R., Keinan, E., Livneh, Z. and Shapiro, E. (2001) Programmable and autonomous computing machine made of biomolecules. Nature. Nov 22; 414(6862):430-4<br>
Breaker, R.R. and Joyce, G.F. (1994) A DNA enzyme that cleaves RNA. Chem Biol. Dec; 1(4): 223-9.<br><br>
Cairns, M., King, A. and Sun, L. (2000) Nucleic acid mutation analysis using catalytic DNA. Nucl Acids Res. 28(3): e9.<br>
Cairns, M., King, A. and Sun, L. (2003) Optimisation of the 10-23 DNAzyme-substrate pairing interactions enhanced RNA cleavage activity at purine-cytosine target sites. Nucl Acids Res. Jun l;31(ll): 2883-9.<br>
Carmi, N., Shultz, L.A. and Breaker, R.R. (1996) In vitro selection of self-cleaving DNAs. Chem Biol. 3(12): 1039-46.<br>
Coppins, R.L. and Silverman, S.K. (2004) Rational modification of a selection strategy leads to deoxyribozjones that create native 3'-5' RNA linkages<br>
Cniz, R.P., Withers, J.B. and Li, Y. (2004) Dinucleotide junction cleavage versatility of 8-17 deoxyribozyme. Chem Biol. Jan; 11(1): 57-67.<br>
Elghanian, R., Storhoff, J., Mucic, R., Letsinger, R. and Mirkin, C. (1997) Selective colorimetric detection of polynucleotides based on the distance-dependent optical properties of gold nanoparticles. Science. 277: 1078-1079.<br>
Emillson, G.M. and Breaker, R.R. (2002) Deoxyribozymes: new activities and new applications. Cell. Mol. Life Sci. 59, 596-607.<br>
Flynn-Charlebois, A., Prior, T.K., Hoadley, K.A. and Silverman, S.K. (2003) In vitro evolution of an RNA-cleaving DNA enzyme into an RNA ligase switches the selectivity from 3'-5' to 2'-5'. J. Am. Chem. Soc. 125: 5346-50<br>
Haseloff, J. and Gerlach, W.L. (1988) Simple RNA enzymes with new and highly specific endoribonuclease activities. Nature. Aug 18; 334(6183): 585-'91.<br>
Hobartner, C. and Silverman, S.K. (2007) Recent advances in DNA catalysis. Biopolymers. 87 (5-6): 279-291.<br>
Huizenga, D. and Szostak, J. (1995) A DNA aptamer that binds adenosine and ATP. Biochemistry. 34: 656-665<br>
Kurata, H., Miyagishi, M., Kuwabara, T., Warashima M. and Taira, K.(2000) Maxizymes: Allosterically controllable ribozymes with biosensor function. Journal of Biochemistry and Molecular Biology, 33: 5, 359-363<br><br>
Lee, J.F., Hesselberth, J.R,, Meyers, L.A. and Ellington, A.D. (2004) Aptamer Database. Nucl Acids Res. 32(90001): 095-100.<br>
Levy, M. and Ellington, A. (2003) Exponential growth by cross-catalytic cleavage of deoxyribo2ymogens. Proc Natl Acad Sci USA. 100(11): 6416-21.<br>
Liu, J. and Lu, Y. (2004) Adenosine-dependent assembly of aptazyme-functionalized gold nanoparticles and its application as a colorimetric biosensor. Analytical Chemistry. 76:1627-1632.<br>
Mirkin, C, Letsinger, R., Mucic, R. and StorhofF, J. (1996) A DNA-based method for rationally assembling nanoparticles into macroscopic materials. Nature. 382: 607-609.<br>
Paul, N. and Joyce, G. (2004) Minimal self-replicating systems. Current Opinion in Chemical Biology. 8(6): 634-639.<br>
Perreault, J., Labuda, D., Usman, N., Yang, J. and Cedergren, R. (1991) Relationship between 2'-hydroxyls and magnesium binding in the hammerhead RNA domain: a model for ribozyme catalysis. Biochemistry. 30(16): 4020-5.<br>
Perreault, J., Wu, T., Cousineau, B.. Ogilvie, K. and Cedergren, R. (1990) Mixed deoxyribo- and ribo-oligonucleotides with catalytic activity. Nature. 344(6266): 565-7.<br>
Prior, T.K., Semlow, D.R. Flynn-Charlebois, Rashid, L And Silverman, S.K. (2004) Structure-function correlations derived from faster variants of a RNA ligase deoxyribozyme. Nucleic Acids Research, 32, 1075-1082.<br>
Raillard, S.A. and Joyce, G.F. (1996) Targeting sites within HIV-1 cDNA with a DNA-cleaving ribozyme. Biochemistry. 35(36): 11693-701.<br>
Santoro, S. and Joyce, G. (1997) A general purpose RNA cleaving DNA enzyme. Proc Natl Acad Sci USA. 94: 4262-4266.<br>
Santoro, S.W. and Joyce, G.F. (1998) Mechanism and utility of an RNA-cleaving DNA enzyme. Biochem. 37(38): 13330-42.<br>
Schubert, S., Furste, J., Werk, D., Grunert, H., Zeichhardt, H., Erdmann, V. and Kurreck, J. (2004) Gaining target access for deoxyribozymes. J Mol Biol. May 28;339(2): 355-63.<br><br>
Sidorov, A., Grasby, J. and Williams, D. (2004) Sequence-specific cleavage of RNA in the absence of divalent metal ions by a DNAzyme incorporating imidazolyl and amino functionalities. Nucl Acids Res. Mar 5;32(4): 1591-601.<br>
Silverman, S, (2004) Breaking up is easy to do (if you're a DNA enzyme that cleaves RNA). Chcm Biol. Jan;l 1(1): 7-8.<br>
Silverman, S.K. (2007) In vitro selection and application of nucleic acid enzymes (Ribozymes and deoxyribozymes). Wiley Encyclopedia of Chemical Biology. In Press.<br>
Tabor, J.J., Levy, M. and Ellington, A.D. (2006) Deoxyribozymes that recode sequence information. Nucleic Acids Res. 34(8): 2166-2172<br>
Warashina, M., Kuwabara, T., Nakamatsu, Y. and Taira, K. (1999) Extremely high and specific activity of DNA enzymes in cells with a Philadelphia chromosome. Chem Biol. Apr;6(6): 237-50.<br>
Zaborowska, Z., Furste, J., Erdmann, V. and Kurreck, J. (2002) Sequence requirements in the catalytic core of the "10-23" DNA enzyme. J Biol Chem. 277(43): 240617-22.<br><br><br>
1.	A method for detecting the presence of at least one target comprising<br>
(a)	providing at least two or more oligonucleotide components and at least one substrate of at least a first nucleic acid complex wherein at least a first oligonucleotide component and a second oligonucleotide component are capable of self-assembly in the presence of a target to form at least a first nucleic acid enzyme which comprises a catalytically active multi-component nucleic acid enzyme with ligase activity (MNAzyme ligase),<br>
(b)	contacting the two or more oligonucleotide components and at least one substrate with a sample putatively containing the at least one target under conditions permitting:<br><br>
(1)	the self-assembly of said at least one catalytically active NfNAzyme ligase and<br>
(2)	the catalytic activity of said at least one MNAzyme ligase; and<br>
(c)	determining the presence of the catalytic activity of said at least one MNAzyme<br>
ligase, wherein the presence of the catalytic activity is indicative of the presence of said<br>
at least one target.<br>
2.	A method for the detection of a target comprising<br>
a)	providing a first nucleic acid complex comprising components which require association with said target to facilitate formation of a first nucleic acid enzyme, wherein said first nucleic acid complex further comprises at least one substrate, and<br>
b)	providing at least one component of a second nucleic acid complex capable of associating with a product of a reaction catalysed by the first nucleic acid enzyme<br>
and forming a second nucleic acid complex capable of functioning as a second nucleic acid enzyme<br>
wherein at least one of said nucleic acid enzymes has ligase activity and at least one of said nucleic acid enzymes is an MNAzyme, and wherein said method comprises:<br>
i) contacting a sample putatively containing a target with the first nucleic acid complex, wherein said target associates with said first nucleic acid complex to facilitate formation of said first nucleic acid enzyme, and<br><br>
ii) contacting a product of the reaction catalysed by the first nucleic acid enzyme with said at least one component of the second nucleic acid complex thereby permitting formation of the second nucleic acid complex which functions as said second nucleic acid enzyme, and<br>
wherein activity of at least one of the first or second nucleic acid enzymes or both is indicative of the presence of said target.<br>
3.	The method of claim 2 further comprising<br>
providing at least one component of a third nucleic acid complex capable of associating with a product of a reaction catalysed by the first or second nucleic acid enzyme or both<br>
and forming a third nucleic acid complex capable of functioning as a third nucleic acid enzyme wherein said method comprises<br>
contacting a product of the reaction catalysed by the first or second nucleic acid enzyme or both with said at least one component of the third nucleic acid complex thereby permitting formation of a third nucleic acid complex which functions as said third nucleic acid enzyme, and<br>
wherein activity of the first, second, or third nucleic acid enzymes or any combination thereof is indicative of the presence of said target.<br>
4.	The method of claim 3 further comprising<br>
providing at least one component of at least one additional nucleic acid complex<br>
capable of associating with a product of at least one of said first, second or third nucleic acid enzymes<br>
and forming at least one additional nucleic acid complex capable of functioning as at least one additional nucleic acid enzyme wherein said method comprises<br>
contacting a product of the reaction catalysed by at least one of said first, second or third nucleic acid enzymes with said at least one component of the at least one additional nucleic acid complex thereby permitting formation of at least one additional nucleic acid complex which functions as said at least one additional nucleic acid enzyme, and<br><br>
wherein activity of the first, second, third or at least one additional nucleic acid enzymes or any combination thereof is indicative of the presence of said target.<br>
5.	The method of any one of claims 2-4 wherein at least one of said nucleic acid enzymes has cleavage activity.<br>
6.	The method of any one of claims 2-S wherein at least said second nucleic acid enzyme has ligase activity.<br>
7.	The method of any one of claims 2-6 wherein said first nucleic acid enzyme is an MNAzyme.<br>
8.	The method of any one of claims 2-7 wherein at least one of said nucleic acid enzymes is a DNAzyme.<br>
9.	The method of any one of claims 2-8 wherein a product of a reaction catalysed by any one of the nucleic acid enzymes is selected from the group consisting of an assembly facilitator, partzyme, substrate, DNAzyme, stabiliser arm or component thereof<br>
10.	The method of any one of claims 1-9 wherein the target is a nucleic acid.<br>
11.	The method of any one of claims 1-10 wherein at least one of said components further comprises at least one aptamer and wherein said method further comprises providing at least one assembly facilitator and at least one assembly inhibitor of said first nucleic acid complex wherein said target associates with said aptamer allowing said first nucleic acid complex to function as a first nucleic acid enzyme .<br>
12.	The method of claim 11 wherein said target is a non-nucleic acid target.<br>
13.	The method of any one of claims 2-12 wherein a product of a reaction catalysed by at least one of said nucleic acid enzymes, or any combination thereof, associates with at least one component of at least one of the preceding nucleic acid complexes thereby permitting formation of further at least one of said preceding nucleic acid complexes which functions as further said preceding nucleic acid enzyme.<br>
14.	A method for the detection of a first assembly facilitator using a cascade<br>
comprising<br><br>
a)	providing components for a first NfNAzyme, at least a first MNAzyrae substrate, a first DNAzyme ligase, a first DNAzyme ligase substrate, a second assembly facilitator, at least a second MNAzyme substrate, and a first partzyme for a second MNAzyme wherein said method comprises:<br>
b)	associating said first assembly facilitator with said components for said first MNAzyme under conditions permitting catalytic activity of said first MNAzyme thus facilitating the catalytic activity of said first MNAzyme thereby allowing modification of said first MNAzyme substrate wherein;<br>
the modification of the first MNAzyme substrate results in the production of a second DNAzyme ligase substrate; and<br>
wherein the first DNAzyme ligase assembles with the first and second DNAzyme ligase substrates; and<br>
wherein catalytic activity of the DNAzyme ligase produces a second partzyme for the second MNAzyme, and wherein;<br>
association of said second partzyme for the second MNAzyme with said second assembly facilitator, and the first partzyme for the second MNAzyme under conditions permitting catalytic activity of the second MNAzyme facilitates the catalytic activity of said second MNAzyme thereby allowing modification of at least the second MNAzyme substrate wherein modification of any of said substrates, or a combination thereof, is indicative of the presence of said first assembly facilitator.<br>
IS.    A method  for the detection of a first assembly facilitator using a cascade comprising<br>
a)	providing components for a first MNAzyme, at least a first MNAzyme substrate,<br>
a DNAzyme ligase, a first DNAzyme ligase substrate, at least a second MNAzyme<br>
substrate, and first and second partzymes for a second MNAzyme and wherein said<br>
method comprises:<br>
b)	associating said first assembly facilitator with said components for the first<br>
MNAzyme under conditions permitting catalytic activity of said first MNAzyme thus<br><br>
facilitating the catalytic activity of said first MNAzyme thereby allowing modification of said first MNAzyme substrate wherein;<br>
the modification of the first MNAzyme substrate results in the production of a second DNAzyme ligase substrate; and<br>
wherein the first DNAzyme ligase assembles with the first and second DNAzyme ligase substrates; and<br>
wherein catalytic activity of the DNAzyme ligase produces a second assembly facilitator for the second MNAzyme, and<br>
wherein association of said second assembly facilitator with said first and second partzymes for the second MNAzyme under conditions permitting catalytic activity of the said second MNAzyme facilitates the catalytic activity of said second MNAzyme thereby allowing modification of at least the second MNAzyme substrate wherein modification of any of said substrates, or a combination thereof, is indicative of the presence of said first assembly facilitator.<br>
16.	The method of claim 14 or IS wherein a product of the reaction catalysed by the second MNAzyme is a component required to permit function of a third nucleic acid enzyme.<br>
17.	The method of 16 wherein a product of the reaction catalysed by the third nucleic acid enzyme is a component which participates in a reaction catalysed by the second MNAzyme or the DNAzyme ligase, or both.<br>
18.	A method for the detection of a first assembly facilitator using a cascade comprising<br><br>
a)	providing components for a first MNAzyme, at least a first MNAzyme substrate, components for a second MNAzyme wherein said second MNAzyme has ligase activity, a second assembly facilitator for the second MNAzyme, a second MNAzyme substrate, a third assembly facilitator for a third MNAzyme, at least a third MNAzyme substrate, and a first partzyme for the third MNAzyme and wherein said method comprises:<br>
b)	associating said first assembly facilitator with said components for said first MNAzyme under conditions permitting catalytic activity of said first MNAzyme thus<br><br>
facilitating the catalytic activity of said first MNAzyme thereby allowing modification of said first MNAzyme substrate wherein;<br>
the modification of the first MNAzyme substrate results in the production of a fourth MNAzyme substrate; and<br>
wherein the second MNAzyme assembly facilitator facilitates assembly of the components for a second MNAzyme with the second and fourth MNAzyme substrates;<br>
^   and<br>
wherein catalytic ligase activity of the second MNAzyme produces a second partzyme for the third MNAzyme, and wherein;<br>
association of said first and second partzymes for the third MNAzyme with said third assembly facilitator, under conditions permitting catalytic activity of the third MNAzyme facilitates the catalytic activity of said third MNAzyme thereby allowing modification of at least the third MNAzyme substrate wherein modification of any of said substrates, or a combination thereof, is indicative of the presence of said first assembly facilitator.<br>
19.	A method for the detection of a first assembly facilitator using a cascade<br>
comprising<br>
a)	providing components for a first MNAzyme, at least a first MNAzyme substrate,<br>
components for a second MNAzyme wherein said second MNAzyme has ligase activity,<br>
a second assembly facilitator for the second MNAzyme, a second MNAzyme substrate, a<br>
first and a second partzyme for a third MNAzyme and at least a third MNAzyme<br>
substrate, and wherein said method comprises:<br>
b)	associating said first assembly facilitator with components for said first<br>
MNAzyme imder conditions permitting catalytic activity of said first MNAzyme thus<br>
facilitating the catalytic activity of said first MNAzyme thereby allowing modification of<br>
said first MNAzyme substrate wherein;<br>
the modification of said first MNAzyme substrate results in the production of a fourth MNAzyme substrate; and<br><br>
wherein the second MNAzyme assembly facilitator facilitates assembly of the said components for second MNAzyme with the second and fourth MNAzyme substrates; and<br>
wherein catalytic ligase activity of the second MNAzyme produces a third assembly facilitator for the third MNAzyme, and wherein;<br>
association of said first and second partzjones for the third MNAzyme with said third assembly facilitator, under conditions permitting catalytic activity of the third MNAzyme facilitates the catalytic activity of said third MNAzyme thereby allowing modification of at least the third MNAzyme substrate wherein modification of any one of said substrates, or a combination thereof, is indicative of the presence of said fu^t assembly facilitator.<br>
20.	The method of claim 18 or 19 wherein a product of the reaction catalysed by the third MNAzyme is a component required to permit fiinction of a fourth nucleic acid enzyme.<br>
21.	The method of 20 wherein a product of the reaction catalysed by the fourth nucleic acid enzyme is a component which participates in a reaction catalysed by the second or third MNAzyme or both.<br>
22.	The method of any one of claims 14-21 where the modification of any one of said substrates, or a combination thereof, provides a detectable effect.<br>
23.	The method of any one of claims 14-22 wherein the modification of at least one substrate is selected &amp;om the group consisting of cleavage of nucleic acids, ligation of nucleic acids, phosphorylation of nucleic acids, nucleic acid capping, amino acid adenylation, cofactor synthesis, RNA polymerization, template-directed polymerization, RNA-protein conjugation, aldol reaction, alcohol oxidation, aldehyde reduction, purine and pyrimidine nucleotide synthesis, alkylation, amide synthesis, urea synthesis, formation of peptide bonds, peptidyl-RNA synthesis, acyl transfer, aminoacylation, carbonate hydrolysis, phosphorothioate alkylation, porphyrin metallation, formation of carbon-carbon bonds, Pd nanoparticle formation, biphenyl isomerization, formation of<br><br>
ester bonds, formation of amide bonds, DNA deglycosylation, thymine dimer photoreversion and phosphoramidate cleavage.<br>
24.	The method of any one of claims 14-23 wherein the first assembly facilitator is a target to be identified, detected or quantitated.<br>
25.	The method of any one of claims 14-17 wherein said DNAzyme ligase is selected from the group consisting of DNAzyme ligases 7Z81,7Z48,7Q10 and 9DB1.<br>
26.	The method of claim 14 wherein a product of the second MNAzyme is the second DNAzyme ligase substrate, and, wherein the DNAzyme ligase facilitates the ligation of the first and the second DNAzyme ligase substrates thereby producing further second partzyme for the second MNAzyme.<br>
27.	The method of claim IS wherein a product of the second MNAzyme is the second DNAzyme ligase substrate, and wherein the DNAzyme ligase facilitates the ligation of the first and the second DNAzyme ligase substrates thereby producing further second assembly facilitator for the second MNAzyme.<br>
28.	The method of claim 18 wherein a product of the third MNAzyme is the fourth MNAzyme substrate, and wherein the second MNAzyme facilitates the ligation of the second and the fourth MNAzyme substrates thereby producing a further second partzyme for the third MNAzyme.<br>
29.	The method of claim 19 wherein a product of the third MNAzyme is the fourth MNAzyme substrate, and wherein the second MNAzyme facilitates the ligation of the second and the fourth MNAzyme substrates thereby producing a further third assembly facilitator for the third MNAzyme.<br>
30.	Use of a pair of isolated nucleic acids as catalytic core sequences incorporated into<br>
a partzyme pair for an MNAzyme ligase wherein said pairs of nucleic acids are selected<br>
from the group consisting of TGGAGGTGGGCTC (SEQ ID NO : 47) and ACG (SEQ<br>
ID NO: 48), GGAGGTGGGCTC (SEQ ID NO: 49) and ACGT (SEQ ID NO: 50).<br>
GGAGGTTAGCTC (SEQ ID NO: 71) and ACGGCGGAGTGATTG (SEQ ID NO:<br>
72), TGGGAGGTTAGCTC (SEQ ID NO: 73) and ACGGCGGAGTGAT (SEQ ID<br>
NO : 74), ATTGGGAGGTTAGCTC (SEQ ID NO : 75) and ACGOCGGAGTG (SEQ<br><br>
ID NO : 76), TGATTGGGAGGTTAGCTC (SEQ ID NO: 77) and ACGGCGGAG (SEQ ID NO: 78), AGTGATTGGGAGGTTAGCTC (SEQ ID NO: 79) and ACGGCGG (SEQ ID NO : 80), GGAGTGATTGGGAGGTTAGCTC (SEQ ID NO : 81) and ACGGC (SEQ ID NO: 82).<br>
31.	An MNAzyme ligase comprising a pair of isolated nucleic acids as catalytic core<br>
sequences incorporated into a partzyme pair wherein said pairs of nucleic acids are<br>
selected firom the group consisting of TGGAGGTGGGCTC (SEQ ID NO : 47) and ACG<br>
(SEQ ID NO: 48). GGAGGTGGGCTC (SEQ ID NO : 49) and ACGT (SEQ ID NO: 50),<br>
GGAGGTTAGCTC (SEQ ID NO: 71) and ACGGCGGAGTGATTG (SEQ ID NO :<br>
72), TGGGAGGTTAGCTC (SEQ ID NO: 73) and ACGGCGGAGTGAT (SEQ ID<br>
NO: 74). ATTGGGAGGTTAGCTC (SEQ ID NO : 75) and ACGGCGGAGTG (SEQ<br>
ID NO : 76), TGATTGGGAGGTTAGCTC (SEQ ID NO : 77) and ACGGCGGAG<br>
(SEQ ID NO: 78). AGTGATTGGGAGGTTAGCTC (SEQ ID NO: 79) and<br>
ACGGCGG (SEQ ID NO: 80), GGAGTGATTGGGAGGTTAGCTC (SEQ ID NO :<br>
81) and ACGGC (SEQ ID NO : 82).<br>
32.	Use of an MNAzyme ligase to ligate two substrates wherein said substrates are ligated by said MNAzyme ligase to form a product following the engagement of said substrates by the substrate arms of said MNAzyme ligase.<br>
33.	The product of claim 32 wherein said product associates with the components of at least one nucleic acid complex.<br>
34.	The product of claims 32 or 33 wherein said product is selected from the group consisting of assembly facilitator, partzyme, substrate, DNAzyme, stabiliser arm, activity inhibitor, assembly inhibitor, or component thereof<br>
35.	The product of claim 32-34 wherein the product can participate in a cascade.<br>
36.	The use of claim 32 wherein at least one substrate is the pixxiuct of a reaction catalysed by a nucleic acid enzyme.<br>
37.	A method for the detection of a plurality of targets comprising:<br>
a) providing a plurality of first nucleic acid complexes each comprising components which require association with one of said plurality of targets to facilitate formation of a<br><br>
plurality of first nucleic acid enzymes, wherein each of said plurality of first nucleic acid complexes further comprises at least one substrate, and<br>
b) providing at least one component of each of a plurahty of second nucleic acid complexes capable of associating with a plurality of products of reactions catalysed by the plurality of first nucleic acid enzymes and forming a plurality of second nucleic acid complexes capable of functioning as a plurality of second nucleic acid enzymes, wherein at least one of said nucleic acid enzymes has ligase activity and at least one of said nucleic acid enzymes is an MNAzyme, and wherein said method comprises:<br>
i) contacting a sample putatively containing said plurality of targets with the plurality of first nucleic acid complexes, wherein said plurality of targets associate with said plurality of first nucleic acid complexes to facilitate formation of said plurality of first nucleic acid enzymes, and<br>
ii) contacting a plurality of products of the reactions catalysed by the plurality of first nucleic acid enzymes with at least one component of each of the plurality of second nucleic acid complexes thereby permitting formation of the plurality of said second nucleic acid complexes which function as said plurality of second nucleic acid enzymes, and<br>
wherein activity of the first or second or both pluraUties of nucleic acid enzymes is indicative of the presence of said plurality of targets.<br>
37.	The method of claim 36 wherein at least one of the products of the reactions<br>
catalysed by any one of the plurality of first or second nucleic acid enzymes is a<br>
component required to permit function of at least one additional nucleic acid enzyme.<br>
38.	The method of claim 36 or 37 wherein at least one of the products of the<br>
reactions catalysed by any one of the nucleic acid enzymes associates with the<br>
components of any one of the preceding nucleic acid complexes thereby permitting at<br>
least one of the nucleic acid complexes to function as further nucleic acid enzyme.<br>
39.	Use of at least two oligonucleotides as multi-component nucleic acid enzyme ligase<br>
(^4NAzyme ligase) substrates wherein said substrates are modified by an MNAzyme<br><br>
ligase following the engagement of said substrates by the substrate arms of said MNAzyme ligase.<br>
40.	A method for the detection of at least two targets wherein said method comprises:<br>
a)	providing at least two nucleic acid complexes each comprising components which<br>
require association with at least one of said targets to facilitate formation of at least two<br>
nucleic acid enzyme complexes, and<br>
b)	contacting a sample putatively containing said at least two targets with said at<br>
least two nucleic acid complexes, wherein said targets associate with said nucleic acid<br>
complexes to facilitate formation of said nucleic acid enzyme complexes, and wherein at<br>
least one product created by the activity of each nucleic acid enzyme complex is required<br>
for formation of an additional nucleic acid enzyme complex, and<br>
wherein activity of the additional nucleic acid enzyme complex is indicative of the presence of said at least two targets.<br>
41.	The method of claim 40 wherein at least one of said nucleic acid enzyme complexes has ligase activity.<br>
42.	The method of claim 40 wherein at least one of said nucleic acid enzyme complexes is an MNAzyme complex.<br><br></td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/viewdoc.aspx?id=ZYG9lI4v5xoYCsBTruRvew==&amp;amp;loc=egcICQiyoj82NGgGrC5ChA==" target="_blank" style="word-wrap:break-word;">http://ipindiaonline.gov.in/patentsearch/GrantedSearch/viewdoc.aspx?id=ZYG9lI4v5xoYCsBTruRvew==&amp;amp;loc=egcICQiyoj82NGgGrC5ChA==</a></p>
		<br>
		<div class="pull-left">
			<a href="269871-adjustable-industrial-antenna-mount.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="269873-sea-water-flue-gas-desulfurization-system.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>269872</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>6261/CHENP/2009</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>47/2015</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>20-Nov-2015</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>13-Nov-2015</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>22-Oct-2009</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>JOHNSON &amp; JOHNSON RESEARCH PTY LIMITED</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td>45 JONES STREET ULTIMO, NEW SOUTH WALES 2007</td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>MOKANY, ELISA</td>
											<td>1/3A, TRICKETT ROAD, WOOLOOWARE, NEW SOUTH WALES 2230</td>
										</tr>
										<tr>
											<td>2</td>
											<td>YOUNG, PAUL, EAN</td>
											<td>53 LAURINA AVENUE, ENGADINE, NEW SOUTH WALES 2233</td>
										</tr>
										<tr>
											<td>3</td>
											<td>TODD, ALISON, VELYIAN</td>
											<td>10 CONEILL PLACE GLEBE, NEW SOUTH WALES 2037</td>
										</tr>
										<tr>
											<td>4</td>
											<td>DOAN, TRAM, BICH</td>
											<td>231 HAMILTON ROAD, FAIRFIELD WEST, NEW SOUTH WALES 2165</td>
										</tr>
										<tr>
											<td>5</td>
											<td>SWAIN, HELEN, ANNE</td>
											<td>18/1, SPAINS WHARF ROAD, NEUTRAL BAY, NEW SOUTH WALES 2089</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>C12Q1/68</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>PCT/AU2008/000492</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td>2008-04-04</td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td>11/697,021</td>
									<td>2007-04-05</td>
								    <td>U.S.A.</td>
								</tr>
								<tr>
									<td>2</td>
									<td>2007901833</td>
									<td>2007-04-05</td>
								    <td>U.S.A.</td>
								</tr>
								<tr>
									<td>3</td>
									<td>60/910,427</td>
									<td>2007-04-05</td>
								    <td>U.S.A.</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/269872-nucleic-acid-enzymes-and-complexes-and-methods-for-their-use by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 06:16:51 GMT -->
</html>
