<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/269830-a-method-for-analysing-the-frequency-of-interaction-of-a-target-nucleotide-sequence by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 06:23:38 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 269830:&quot;A METHOD FOR ANALYSING THE FREQUENCY OF INTERACTION OF A TARGET NUCLEOTIDE SEQUENCE.&quot;</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">&quot;A METHOD FOR ANALYSING THE FREQUENCY OF INTERACTION OF A TARGET NUCLEOTIDE SEQUENCE.&quot;</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>The present invention relates in one aspect to a method for analysing the frequency of interaction of a target nu-cleotide sequence with one or more nucleotide sequences of interest (eg. one or more genomic loci) comprising the steps of: (a) providing a sample of cross-linked DNA; (b) digesting the cross-linked DNA with a primary restriction enzyme; (c) ligating the cross-linked nucleotide sequences; (d) reversing the cross linking; (e) digesting the nucleotide sequences with a secondary restriction enzyme; (f) ligating one or more DNA sequences of known nucleotide composition to the available secondary restriction enzyme digestion site(s) that flank the one or more nucleotide sequences of interest; (g) amplifying the one or more nucleotide sequences of interest using at least two oligonucleotide primers, wherein each primer hybridises to the DNA sequences that flank the nucleotide sequences of interest; (h) hybridising the amplified sequence(s) to an array; and (i) determining the frequency of interaction between the DNA sequences.</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td>FIELD OF INVENTION<br>
The present invention relates to the analysis of the. frequency of interaction of two or more nucleotide sequences in the nuclear space.<br>
BACKGROUND TO THE INVENTION<br>
Studies on mammalian nuclear architecture aim to understand how 2 meters of DNA is<br>
folded into a nucleus of 10 urn across, while allowing accurate expression of the genes<br>
that specify the cell-type, and how this is faithfully propagated during each cell cycle.<br>
Progress in this field has largely come from microscopy studies, which revealed that<br>
genomes are non-randomly arranged in the nuclear space. For example, densely<br>
packed heterochromatin is separated from more open euchromatin and chromosomes<br>
occupy distinct territories in the nuclear space. An intricate relationship exists between<br>
nuclear positioning  and transcriptional activity.     Although transcription occurs<br>
throughout  the   nuclear   interior,   active   genes   that   cluster   on  chromosomes<br>
preferentially locate at the edge or outside of their chromosome territory. Individual<br>
genes may migrate upon changes in their transcription status, as measured against<br>
relatively large nuclear landmarks such as chromosome territories, centromeres or the<br>
nuclear periphery.  Moreover, actively transcribed genes tens of megabases apart on<br>
the chromosome can come together in the nucleus, as demonstrated recently by<br>
fluorescence in situ hybridization (FISH) for the p-globin locus and a few, selected,<br>
other genes. Besides transcription, genomic organisation is associated with the<br>
coordination of replication, recombination and the probability of loci to translocate<br>
(which can lead to malignancies) and the setting and resetting of epigenetic programs.<br>
Based on these observations it is thought that the architectural organisation of DNA in<br>
the cell nucleus is a key contributor to genomic function.<br>
Different assays have been developed to allow an insight into the spatial organisation of genomic loci in vivo. One assay, called RNA-TRAP has been developed (Carter et<br>
al. (2002) Nat. Genet. 32, 623) which involves targeting of horseradish peroxidase (HRP) to nascent RNA transcripts, followed by quantitation of HRP-catalysed biotin deposition on chromatin nearby.<br>
Another assay that has been developed is called chromosome conformation capture (3C) technology, which provides a tool to study the structural organisation of a genomic region. 3C technology involves quantitative PCR-analysis of cross-Unking frequencies between two given DNA restriction fragments, which gives a measure of their proximity in the nuclear space (see Figure 1). Originally developed to analyse the conformation of chromosomes in yeast (Dekker et al., 2002), this technology has been adapted to investigate the relationship between gene expression and chromatin folding at intricate mammalian gene clusters (see, for example, Tolhuis et al., 2002; Palstra et al., 2003; and Drissen et al., 2004). Briefly, 3C technology involves in vivo formaldehyde cross-linking of cells and nuclear digestion of chromatin wim a restriction enzyme, followed by ligation of DNA fragments that were cross-linked into one complex. Ligation products are then quantified by PCR. The PCR amplification step requires the knowledge of the sequence information for each of the DNA fragments that are to be amplified. Thus, 3C technology provides a measure of interaction frequencies between selected DNA fragments.<br>
There is an important need for high-throughput technology that can systematically screen the whole genome in an unbiased manner for DNA loci that contact each other in the nuclear space.<br>
The present invention seeks to provide improvements in 3C technology. SUMMARY OF THE INVENTION<br>
3C technology as currently applied only allows analysis of a limited number of selected DNA-DNA interactions owing to the limitations of the PCR amplification step, which requires knowledge of specific sequence information for each fragment to be analysed. Moreover, selecting restriction fragments as candidates for long-range<br>
DNA interactions requires a substantial amount of prior knowledge (e.g. the location of hypersensitive sites) of the locus of interest, which is usually not available. Given the functional relevance of many long-range DNA-DNA interactions described so far, the ability to randomly screen for DNA elements that loop to a sequence of interest -such as a gene promoter, enhancer, insulator, silencer, origin of replication or MAR/SAR - or a genomic region of interest - such as a gene-dense or gene-poor region or repetitive element - can greatly facilitate the mapping of sequences involved in a regulatory network.<br>
The present invention relates to 4C technology (ie. capture and characterise co-localised chromatin), which provides for the high-throughput analysis of the frequency of interaction of two or more nucleotide sequences in the nuclear space.<br>
4C (capture and characterize co-localized chromatin) technology is a modified version of 3C technology that allows an unbiased genome-wide search for DNA fragments that interact with a locus of choice. Briefly, 3C analysis is performed as usual, but omitting the PCR step. The 3C template contains a bait (e.g. a restriction fragment of choice that encompasses a gene of interest) ligated to many different nucleotide sequences of interest (representing this gene's genomic environment). The template is cleaved by another, secondary, restriction enzyme, and ligated. Advantageously, the one or more nucleotide sequences of interest that are ligated to the target nucleotide sequence are amplified using at least one (preferably, at least two) oligonucleotide primer, wherein the at least one primer hybridises to a DNA sequence that flanks the nucleotide sequences of interest. Typically, this yields a pattern of PCR fragments that is highly reproducible between independent amplification reactions and specific for a given tissue. In one embodiment, HindHl and DpriR are used as primary and secondary restriction enzyme. Next, the amplified fragments may be labeled and optionally hybridised to an array, typically against a control sample containing genomic DNA digested with the same combination of restriction enzymes.<br>
In one preferred embodiment of the present invention, the ligated fragments that are cleaved by a secondary restriction enzyme are subsequently religated to form small DNA circles.<br>
3C technology has therefore been modified such that all nucleotide sequences of interest that interact with a target nucleotide sequence are amplified- Practically this means that instead of performing an amplification reaction with primers that are specific for the fragments that one wishes to analyse, an amplification is performed using oligonucleotide primer(s) which hybridise to a DNA sequence that flanks the nucleotide sequences of interest. Advantageously, 4C is not biased towards the design of PCR primers that are included in the PCR amplification step and can therefore be used to search the complete genome for interacting DNA elements.<br>
SUMMARY ASPECTS OF THE PRESENT INVENTION<br>
Aspects of the present invention are presented in the accompanying claims.<br>
In a first aspect, there is provided a method for analysing the frequency of interaction of a target nucleotide sequence with one or more nucleotide sequences of interest (eg. one or more genomic loci) comprising the steps of: (a) providing a sample of cross-linked DNA; (b) digesting the cross-linked DNA with a primary restriction enzyme; (c) ligating the cross-linked nucleotide sequences; (d) reversing the cross linking; (e) digesting the nucleotide sequences with a secondary restriction enzyme; (f) ligating one or more DNA sequences of known nucleotide composition to the available secondary restriction enzyme digestion site(s) that flank the one or more nucleotide sequences of interest; (g) amplifying the one or more nucleotide sequences of interest using at least two oligonucleotide primers, wherein each primer hybridises to the DNA sequences that flank the nucleotide sequences of interest; (h) hybridising the amplified sequence(s) to an array; and (i) determining the frequency of interaction between the DNA sequences.<br>
In a second aspect, there is provided a method for analysing the frequency of interaction of a target micleotide sequence with one or more nucleotide sequences (eg. one or more genomic loci) comprising the steps oft (a) providing a sample of cross-linked DNA; (b) digesting the cross-linked DNA with a primary restriction enzyme; (c) ligating the cross-linked nucleotide sequences; (d) reversing the cross linking; (e) digesting fee nucleotide sequences with a secondary restriction enzyme; (f) circularising the nucleotide sequences; (g) amplifying the one or more nucleotide sequences that are ligated to the target nucleotide sequence; (h) optionally hybridising the amplified sequences to an array; and (i) determining the frequency of interaction between the DNA sequences.<br>
In a third aspect there is provided a circularised nucleotide sequence comprising a first and a second nucleotide sequence, wherein each end of the first and a second nucleotide sequences are separated by different restriction enzyme recognition sites, and wherein said first nucleotide sequence is a target nucleotide sequence and said second nucleotide sequence is obtainable by cross-Unking genomic DNA.<br>
In a fourth aspect there is provided a method for preparing a circularised nucleotide sequence comprising the steps of: (a) providing a sample of cross-linked DNA; (b) digesting the cross-linked DNA with a primary restriction enzyme; (c) ligating the cross-linked nucleotide sequences; (d) reversing the cross linking; (e) digesting the nucleotide sequences with a secondary restriction enzyme; and (f) circularising the nucleotide sequences.<br>
In a fifth aspect there is provided a method for analysing the frequency of interaction of a target nucleotide sequence with one or more nucleotide sequences (eg. one or more genomic loci) comprising the use of the circularised nucleotide sequence.<br>
In a sixth aspect there is provided an array of probes immobilised on a support comprising one or more probes that hybridise or are capable of hybridising to the circularised nucleotide sequence.<br>
In a seventh aspect there is provided a set of probes complementary in sequence to the nucleic acid sequence adjacent to each one of the primary restriction enzyme recognition sites of a primary restriction enzyme in genomic DNA.<br>
In an eighth aspect there is provided a process for preparing a set of probes comprising the steps of: (a) identifying each one of the primary restriction enzyme recognition sites for a primary restriction enzyme in genomic DNA; (b) designing probes that are capable of hybridising to the sequence adjacent each one of the primary restriction enzyme recognition sites in the genomic DNA; (c) synthesising the probes; and (d) combining the probes together to form a set of probes or substantially a set of probes.<br>
In a ninth aspect there is provided a set of probes or substantially a set of probes obtained or obtainable by the process described herein.<br>
In a tenth aspect there is provided an array comprising the array of probes or substantially the set of probes described herein<br>
In an eleventh aspect mere is provided an array comprising the set of probes according described herein.<br>
In a twelfth aspect there is provided a process for preparing an array comprising the step of immobilising on a solid support substantially the array of probes or substantially the set of probes described herein.<br>
In an thirteenth aspect there is provided a process for preparing an array comprising the step of immobilising on a solid support the array of probes or the set of probes described herein.<br>
In an fourteenth aspect there is provided an array obtained or obtainable by the method described herein.<br>
In a fifteenth aspect there is provided a method for identifying one or more DNA-DNA interactions that are indicative of a particular disease state comprising the step of performing steps (a)-(i) of the first and second aspects of the present invention, wherein in step (a) a sample of cross-linked DNA is provided from a diseased and a non-diseased cell, and wherein a difference between the frequency of interaction between the DNA sequences from the diseased and non-diseased cells indicates that the DNA-DNA interaction is indicative of a particular disease state.<br>
In an sixteenth aspect there is provided a method of diagnosis or prognosis of a disease or syndrome caused by or associated with a change in a DNA-DNA interaction comprising the step of performing steps (a)-(i) of the first and second aspects of the present invention, wherein step (a) comprises providing a sample of cross-linked DNA from a subject; and wherein step (i) comprises comparing the frequency of interaction between the DNA sequences with that of an unaffected control; wherein a difference between the value obtained from the control and the value obtained from the subject is indicative that the subject, is suffering from the disease or syndrome or is indicative that the subject will suffer from the disease or syndrome.<br>
In a seventeenth aspect there is provided a method of diagnosis or prognosis of a disease or syndrome caused by or associated with a change in a DNA-DNA interaction comprising the step of: performing steps (a)-(i) of the first and second aspects of the present invention, wherein step (a) comprises providing a sample of cross-linked DNA from a subject; and wherein said method comprises the additional step of: (j) identifying one or more loci that have undergone a genomic rearrangement that is associated with a disease.<br>
In an eighteenth aspect there is provided an assay method for identifying one or more agents that modulate a DNA-DNA interaction comprising the steps of: (a) contacting a sample with one or more agents; and (b) performing steps (a) to (i) of the first and second aspects of the present invention, wherein step (a) comprises providing cross-linked DNA from the sample;<br>
wherein a difference between (i) the frequency of interaction between the DNA sequences in the presence of the agent and (ii) the frequency of interaction between the DNA sequences in the absence of the agent is indicative of an agent that modulates the DNA-DNA interaction.<br>
In a nineteenth aspect there is provided a method for detecting the location of a balanced and/or unbalanced breakpoint (eg. a translocation) comprising the step of: (a) performing steps (a) to (i) of the first and second aspects of the present invention; and (b) comparing the frequency of interaction between the DNA sequences with that of a control; wherein a transition from low to high DNA-DNA interaction frequency in the sample as compared to the control is indicative of the location of a breakpoint.<br>
In a twentieth aspect there is provided a method for detecting the location of a balanced and/or unbalanced inversion comprising the steps of: (a) performing steps (a) to (i) of the first and second aspects of the present invention; and (b) comparing the frequency of interaction between the DNA sequences with that of a control; wherein an inversed pattern of DNA-DNA interaction frequencies for the sample as compared to the control is indicative of an inversion.<br>
In a twenty-first aspect there is provided a method for detecting the location of a deletion comprising the steps of: (a) performing steps (a) to (i) of the first and second aspects of the present invention; (b) comparing the frequency of interaction between the DNA sequences with that of a control; wherein a reduction in the DNA-DNA interaction frequency for tie sample as compared to the control is indicative of deletion.<br>
In a twenty-second aspect there is provided a method for detecting the location of a duplication comprising the steps of: (a) performing steps (a) to (i) of the first and second aspects of the present invention; and (b) comparing the frequency of interaction between the DNA sequences with that of a control; wherein an increase or a decrease in DNA-DNA interaction frequency for the subject sample as compared to the control is indicative of a duplication or insertion.<br>
In a twenty-third aspect there is provided an agent obtained or obtainable by the assay method described herein.<br>
In a twenty-fourth aspect there is provided the use of the circularised nucleotide sequence for identifying one or more DNA-DNA interactions in a sample.<br>
In a twenty-fifth aspect there is provided the use of the circularised nucleotide sequence for the diagnosis or prognosis of a disease or syndrome caused by or associated with a change in a DNA-DNA interaction.<br>
In a twenty-sixth aspect there is provided the use of the array of probes or the set of probes described herein for identifying one or more DNA-DNA interactions in a sample.<br>
In a twenty-seventh aspect there is provided the use of the array of probes or the set of probes described herein for. the diagnosis or prognosis of a disease or syndrome caused by or associated with a change in a DNA-DNA interaction.<br>
In a twenty-eighth aspect there is provided the use of the array described herein for identifying one or more DNA-DNA interactions in a sample.<br>
In a twenty-ninth aspect there is provided the use of the array described herein for the diagnosis or prognosis of a disease or syndrome caused by or associated with a change in a DNA-DNA interaction.<br>
In a thirtieth aspect there is provided a method, an array of probes, a set of probes, a process, an array, an assay method, an agent, or a use substantially as described herein and with reference to any of the Examples or Figures.<br>
PREFERRED EMBODIMENTS<br>
Preferably, the ligation reaction in step (f) results in the formation of DNA circles.<br>
Preferably, the target nucleotide sequence is selected from the group consisting of a genomic rearrangement, promoter, an enhancer, a silencer, an insulator, a matrix attachment region, a locus control region, a transcription unit, an origin of replication, a recombination hotspot, a traaslocation breakpoint, a centromere, a telomere, a gene-dense region, a gene-poor region, a repetitive element and a (viral) integration site.<br>
Preferably, the target nucleotide sequence is a nucleotide sequence that is associated with or causes a disease, or is located up to or greater than 15Mb on a linear DNA template from a locus liiat is associated with or causes a disease.<br>
Preferably, the target nucleotide sequence is selected from the group consisting of: AML1, MLL, MTC, BCL, BCR, ABU, IGH, LYL1, TALI, TAL2, IM02, TCRa/6, TCRfi and HOX or other loci associated with disease as described in "Catalogue of Unbalanced Chromosome Aberrations in Man" 2nd edition. Albert Schinzel. Berlin: Walter deGruyter, 2001. ISBN. 3-11-011607-3.<br>
Preferably, the primary restriction enzyme is a restriction enzyme that recognises a 6-8 bp recognition site.<br>
Preferably, the primary restriction enzyme is selected from the group consiting of BgHl, HindHL, EcoW, BamSi, Spel, Psil and Ndel.<br>
Preferably, the secondary restriction enzyme is a restriction enzyme that recognises a 4 or 5 bp nucleotide sequence recognition site.<br>
Preferably, the secondary restriction enzyme recognition site is located at greater than about 350bp from the primary restriction site in the target nucleotide sequence.<br>
Preferably, the nucleotide sequence is labelled.<br>
Preferably, the probes are complementary in sequence to the nucleic acid sequence adjacent each side of each one of the primary restriction enzyme recognition sites of a primary restriction enzyme in genomic DNA.<br>
Preferably, the probes are complementary in sequence to the nucleic acid sequence that is less than 300 base pairs from each one of lie primary restriction enzyme recognition sites of a primary restriction enzyme in genomic DNA.<br>
Preferably, the probes are complementary to the sequence that is less then 300 bp from each one of the primary restriction enzyme recognition sites of a primary restriction enzyme in. genomic DNA.<br>
Preferably, the probes are complementary to the sequence that is between 200 and 300 bp from each one of the primary restriction enzyme recognition sites of a primary restriction enzyme in genomic DNA.<br>
Preferably, the probes are complementary to the sequence that is between 100 and 200 bp or 0 to 100 bp from each one of the primary restriction enzyme recognition sites of a primary restriction enzyme in genomic DNA.<br>
Preferably, two or more probes are capable of hybridising to the sequence adjacent each primary restriction enzyme recognition site of a primary restriction enzyme in the genomic DNA.<br>
Preferably, the probes overlap or partially overlap. Preferably, the overlap is less than 10 nucleotides.<br>
Preferably, the probe sequence corresponds to all or part of the sequence between each one of the primary restriction enzyme recognition sites of a primary restriction enzyme<br>
and each one of the first neighbouring secondary restriction enzyme recognition sites of a secondary restriction enzyme.<br>
Preferably, each probe is at a least a 25 mer.<br>
Preferably, each probes is a 25-60 mer.<br>
Preferably, Hie probes are PCR amplification products.<br>
Preferably, the array comprises about 300,000-400,000 probes.<br>
Preferably, the array comprises about 385,000 or more probes, preferably, about 750,000 probes, more preferably, 6 x 750,000 probes.<br>
Preferably, the array comprises or consists of a representation of the complete genome of a given species at lower resolution.<br>
Preferably, one out of every 2, 3, 4, 5, 6, 7, 8, 9 or 10 probes as ordered on a linear chromosome template is contained in the array.<br>
Preferably, a transition from low to high interaction frequencies is indicative of the location of a balanced and/or unbalanced breakpoint.<br>
Preferably, an inversed pattern of DNA-DNA interaction frequencies for the subject sample as compared to the control is indicative of an balanced and/or unbalanced inversion.<br>
Preferably, a reduction in the DNA-DNA interaction frequency for the subject sample as compared to the control, in combination with an increase in DNA-DNA interaction frequency for more distant regions, is indicative of a balanced and/or unbalanced deletion.<br>
Preferably, an increase or a decrease in DNA-DNA interaction frequency for the subject sample as compared to the control is indicative of a balanced and/or unbalanced duplication or insertion.<br>
Preferably, spectral karyotyping and/or FISH is used prior to performing said method.<br>
Preferably, the disease is a genetic disease.<br>
Preferably, the disease is cancer.<br>
Preferably, the two or more amplified sequences are differentially labelled.<br>
Preferably, the two or more amplified sequences are identically labelled when the sequences reside on different chromosomes.<br>
Preferably, the two or more amplified sequences are identically labelled when the sequences reside on the same chromosome at a distance that is far enough for minimal overlap between DNA-DNA interaction signals.<br>
Preferably, wherein the diagnosis or prognosis is prenatal diagnosis or prognosis. ADVANTAGES<br>
The present invention has a number of advantages. These advantages will be apparent in the following description.<br>
By way of example, the present invention is advantageous since it provides inter alia commercially useful nucleotides sequences, processes, probes and arrays.<br>
By way of further example, the present invention is advantageous since it provides for the high throughput analysis of the frequency of interaction of two or more nucleotide sequences in the nuclear space.<br>
By way of further example, the present invention is advantageous since using conventional 3C technology, each single DNA-DNA interaction must be analysed by a unique PCR reaction containing a unique pair of primers. High-throughput analysis is therefore only possible if PCR is automated, but the costs of so many primers will be too high. Accordingly, high-throughput (genome-wide) analysis of DNA-DNA interactions is not viable with, conventional 3C technology. In contrast, the present invention now allows the simultaneous screening of thousands of DNA-DNA interactions. High-throughput analysis of DNA-DNA interactions according to the present invention -will greatly increase the scale and resolution of analysis.<br>
By way of further example, the present invention is advantageous since using conventional 3C technology, the screen is biased towards those DNA sequences for which oligonucleotide primers were designed, ordered and included in the analysis. The choice of such oligonucleotide primers is typically based on knowledge concerning the position of, for example, (distant) enhancers and/or other regulatory elements/hypersensitive sites that it is believed will cross-link •with the rmcleotide sequence that is being investigated. Thus, conventional 3C is biased towards the design of PCR primers that are included in the PCR amplification step, whereas 4C is unbiased and can be used to search the complete genome for interacting DNA elements. This is because amplification of cross-linked sequences in 4C is not based on the predicted knowledge of sequences that cross-link with the nucleotide sequence being investigated. Rather, in one embodiment of 4C, sequences that cross link to the first (target) nucleotide sequence can be amplified using PCR primers that hybridise to that nucleotide sequence. Thus, the present invention allows an unbiased genome-wide screen for DNA-DNA interactions.<br>
By way of further example, the present invention is advantageous because using conventional 3C technology only allows the selective amplification of a single DNA-DNA interaction. This is not informative when hybridised to an array. The technology has been improved such that all fragments that interact with a first (target) nucleotide sequence are now amplified eg. selectively amplified.<br>
By way of further example, the present invention is advantageous because 4C technology can be used to detect balanced or unbalanced genetic aberrations - such as all types of translocations, deletions, inversions, duplications and other genomic rearrangements - in nucleic acid, for example, chromosomes. 4C technology (which measures proximity of DNA fragments) can even determine a subject's predisposition to acquire certain translocations, deletions, inversions, duplications and other genomic rearrangements (eg. balanced or unbalanced translocations, deletions, inversions, duplications arid other genomic rearrangements). An advantage over current strategies is that it is not required to know the exact position of the change because the resolution of 4C technology is such that it can be used to detect rearrangements even when the '4C-bait' (as defined by the primary and secondary restriction enzyme recognition sites that are analysed) is located away (eg. up to one megabase or even more) from the change. Another advantage is that 4C technology allows the accurate mapping of changes since it can. be used to define the two (primary) restriction sites between which changes occurred. Another advantage is that cells need not to be cultured before fixation. Thus, for example solid tumours can also be analysed for genomic rearrangements.<br>
By way of further example, the present invention is advantageous because the 4C technology can also detect changes (eg. rearrangements) in a pre-malignant state, i.e. before all the cells contain these changes. Thus, the technology can be used not only in the diagnosis of disease but also in the prognosis of disease.<br>
By way of further example, the array design according to the present invention is particularly advantageous as compared to existing genomic tiling arrays - such as Nimblegen genomic tiling arrays - since the design allows representation of a much larger part of the genome per single array. By way of example, for a restriction enzyme recognising a hexa-nucleotide sequence about 3 arrays with about 385,000 probes each will be sufficient to cover, for example, the complete human or mouse genome. For a restriction enzyme recognising more than 6bp, a single array of about 385,000 probes can be used to cover, for example, the complete human or mouse<br>
genome. The advantages of the array design are that: (1) each probe is informative since each analyses an independent ligation event, greatly facilitating the interpretation of the results; and (2) a large representation of the genome can be spotted on a single array which is cost-effective.<br>
4C technology can advantageously be vised for the fine-mapping of poorly characterised rearrangements originally detected by cytogenetic approaches (light microscopy, FISH, SKY, etc).<br>
4C technology can advantageously be used for the simultaneous screening on a single array for combinations of rearrangements that have occurred near multiple loci.<br>
BRIEF DESCRIPTION OF THE FIGURES<br>
Figure 1<br>
The principle of 3C technology<br>
Figure 2<br>
(a) The principle of one embodiment of 4C technology. 3C analysis is performed as usual, with e.g. HindSi (H) as restriction enzyme. After reversal of cross-links, DNA mix will contain a first (target) nucleotide sequence ligated to many different fragments. These fragments will be amplified and labelled by using amplification methods - such as inverse PCR - on eg., DpnTL circles, using first (target) nucleotide sequence-specific primers. Labelled amplification products may be hybridised to the arrays as described herein. HindUI and DpnH are given as examples, but other combinations of restriction enzymes - such as 6 or 8- and 4 or 5-cutters - can also be used, (b) PCR results separated by gel electrophoresis from two independent fetal liver (LI, L2) and brain (Bl, B2) samples, (c) Schematic representation of the location of the microarray probes. Probes were designed within 100 bp of HindUI sites. Thus, each probe analyzes one possible ligation partner.<br>
Figure 3<br>
4C Technology detects the genomic environment of P-globin (chromosome 7). Shown are unprocessed ratios (4C signals for p-globin HS2 divided by signals obtained for control sample) for probes located in ~35 Mb genomic regions on mouse chromosome 10,11,12,14, 15, 7 and 8 (top to bottom; regions shown are at identical distance from each corresponding centromere). Note the large cluster of strong signals around the (globin) bait on chromosome 7 (row 6), which demonstrates that 4C technology detects genomic fragments close on the linear chromosome template (in agreement with the fact that interaction frequencies are inversely proportional to the genomic site separation). Note that the region linked in cis around me bait that shows high signal intensities is large (&gt;5Mb), implying for example that translocations can be detected even with baits more than 1MB away from the breakpoint.<br>
Figure 4<br>
4C technology detects the genomic environment of Rad23 A (chromosome 8). Shown are unprocessed ratios (4C signals for Rad23A divided by signal obtained for control sample) for probes located in ~15 Mb or more genomic regions on mouse chromosome 10,11,12,14, 15, 7 and 8 (top to bottom; regions shown are at identical distance from each corresponding centromere). Note the large cluster of strong signals around the (Rad23A) bait on chromosome 8 (row 7), which demonstrates mat 4C technology detects genomic fragments close on the linear chromosome template (in agreement with the fact that interaction frequencies are inversely proportional to the genomic site separation). Note that the region linked in cis around tibe bait that shows high signal intensities is large (&gt;5Mb), implying for example that translocations can be detected even with baits more than 1MB away from the breakpoint.<br>
Figure 5<br>
4C interactions of p-globin on chromosome 7 (~ 13 5Mb) for a transcribing tissue (fetal liver) and a non-transcribing tissue (fetal brain) (analysed by a running mean approach). Note that long-range interactions with P-globin differ between tissues (likely dependent on the transcription status of the gene). Independent of the tissue strong 4C signals demarcate a large region (&gt;5 Mb) around the bait<br>
Figure 6<br>
Uros and Eraf interact with p-globin in fetal liver cells. The 4C approach reveals that two genes, Eraf and Uros, interact over &gt;30 Mb with the P-globin locus located ~30 Mb away. These two interactions were previously found by a different technology (Fluorescence In Situ Hybridisation) as described in Osbome et a/., Nature Genetics 36, 1065 (2004). This example shows that long-range interactions detected by 4C technology can be verified by FISH and truly reflect nuclear proximity.<br>
Figure 7<br>
4C technology accurately identifies transitions between unrelated genomic regions that are linked hi cis. For these experiments transgenic mice were used that contain a human p-globin Locus Control Region (LCR) cassette (~20 kb) inserted (via homologous recombination) into the Rad23A locus on mouse chromosome 8. 4C technology was performed on El 4.5 fetal livers of transgenic mice that were homozygous for this insertion. A HindTTt fragment within the integration cassette (HS2) was used as '4C-bait'. The data show that 4C technology accurately defines both ends of the transgenic cassette (bottom row: only probes in the human LCR (~20kb) give 4C-signals and not probes in the remainder of ~380 kb human (3-globin sequence) and clearly reveals the position of integration on mouse chromosome 8 (upper panel: compare signals on chromosome 8 (for position of integration, see arrow) with signals on 6 other mouse chromosomes) (complete chromosomes are depicted). This example shows that 4C technology can be used to detect the genomic position of ectopically integrated DNA fragments (virus, transgene, etc.). It shows that transitions between unrelated genomic regions that are linked in cis can be identified accurately, which can be used to identify genomic breakpoints and translocation partners.<br>
Figure 8<br>
AC technlogy produces reproducible data since the profile for HS2 and P-globin are very similar. Four biologically independent 4C experiments were performed on E14.5 fetal livers, using either the p-globin gene p-major (upper 2 rows) or P-globin<br>
HS2(bottom two rows) as the bait These baits are ~40 kb apart on the linear chromosome template but were previously shown to be close in the nuclear space (Tolhuis et al, Molecular Cell 10, 1453 (2002)) Depicted is a ~5 Mb region on mouse chromosome 7 that is 20-20 Mb away from the (J-globin locus. The data show high reproducibility between independent experiments and demonstrate that two fragments close in the nuclear space share interacting partners located elsewhere in the genome.<br>
Figure 9<br>
4C technology is applied to measure DNA-DNA interaction frequencies with sequence X (on chromosome A) in cells from a healthy person (top) and a patient with translocation (A;B) (bottom). Signal intensities representing DNA-DNA interaction frequencies (Y-axis) are plotted for probes ordered on linear chromosome templates (X-axis). In normal cells, frequent DNA-DNA interactions are detected on chromosome A around sequence X. In patient cells, a 50% reduction in interaction frequencies is observed for probes on chromosome A located on the other side of the breakpoint (BP) (compare grey curve (patient) with black line (healthy person). Moreover, the translocation brings part of chromosome B in close physical proximity to sequence X, and frequent DNA-DNA interactions are now observed for this region on chromosome B. The abrupt transition from low to high interaction frequencies on this chromosome marks the location of its breakpoint.<br>
Figure 10<br>
(Balanced) inversion(s) can be detected by 4C technology. Inversed patterns of DNA-DNA interaction frequencies (measured by 4C technology as hybridization signal intensities) are observed in diseased (solid curve) as compared to non-diseased (stippled curve) subject, which reveals the presence and size of the inversion.<br>
Figure 11<br>
Heterozygous deletion(s) detection by 4C technology. Probes with reduced DNA-DNA interaction frequencies (measured by 4C technology as hybridization signal intensities) in diseased (grey curve) as compared to non-diseased (black curve) subjects, reveal the position and size of the deleted region. Residual hybridization<br>
signals in the deleted region of the diseased subject come from intact allele (heterozygous deletion). Deletion is typically accompanied by an increase in signal intensities for probes located directly beyond the deleted region (note that the grey curve is above the black curve at right hand of the deletion), since these regions come in closer physical proximity to the 4C sequence (bait).<br>
Figure 12<br>
Duplication detected by 4C technology. Probes with increased hybridization signals in a patient (solid curve) as compared to a normal subject (stippled curve) indicate the position and size of duplication. Duplication as detected by 4C technology is typically accompanied by decreased hybridization signals in diseased versus non-diseased subjects for probes beyond the duplicated region (duplication increases their genomic site separation from the 4C sequence).<br>
Figure 13<br>
Long-range interactions with j3-globin revealed by 4C technology, a, Unprocessed ratios of 4C over control hybridization signals, revealing interactions of p-globin HS2 with chromosome 7 and two unrelated chromosomes (8 and 14). b-c, Unprocessed data for two independent fetal liver (top, in red) and fetal brain samples (bottom, in blue) plotted along two different 1-2 Mb regions on chromosome 7. Highly reproducible clusters of interactions are observed either in the two fetal liver samples (b) or the two brain samples (c). d-e, Running mean data for the same regions. False discovery rate was set at 5% (stippled line), f, Schematic representation of regions of interaction with active (fetal liver, top) and inactive (fetal brain, bottom) P-globin on chromosome 7.<br>
Figure 14<br>
Active and inactive p-globin interact with active and inactive chromosomal regions, respectively, a, Comparison between P-globin long-range interactions in fetal liver (4C running mean, top), microarray expression analysis in fetal liver (log scale, middle) and the location of genes (bottom) plotted along a 4 Mb region that contains the gene Uros (~30 Mb away from P-globin), showing that active P-globin preferentially<br>
interacts with other actively transcribed genes, b, The same comparison in fetal brain around a OR gene cluster located -38 Mb away from globin, showing that inactive P-globin preferentially interacts with inactive regions, c, Characterization of regions interacting with fj~globin in fetal liver (left) and brain (right) in terms of gene content and activity.<br>
Figure 15<br>
Ubiquitously expressed Rad23A interacts with very similar, active, regions in fetal liver and brain, a, Schematic representation of regions on chromosome 8 interacting with active Rad23A in fetal liver (top, red) arid brain (bottom, blue), b, Comparison between Rad23A long-range interactions (4C running mean) and microarray expression analysis (log scale) in fetal liver (top two panels), Rad23A long-range interactions (4C running mean) and microarray expression analysis (log scale) in fetal brain (panel 3 and 4) and the location of genes (bottom panel) plotted along a 3 Mb region of chromosome 8. c, Characterization of regions interacting with Rad23A in fetal liver (left) and brain (right) in terms of gene content and activity.<br>
Figure 16<br>
Cryo-FISH confirms that 4C technology truly identifies interacting regions, a, example of part of a (200 nm) cryo-section showing more than 10 nuclei, some of which containing the p-globin locus (green) and/or Uros.(red). Due to sectioning, many nuclei do not contain signals for these two loci, b-d, examples of completely (b) and partially (c) overlapping signals and contacting signals (d), which were all scored as positive for interaction, e-g, examples of nuclei containing non-contacting alleles (e-f) and a nucleus containing only (5-globin (g), which were all scored as negative for interaction, h-i, Schematic representation of cryo-FISH results. Percentages of interaction with p-globin (h) and Rad23A (i) are indicated above the chromosomes for regions positively identified (red arrowhead) and negatively identified (blue arrowhead) by 4C technology. The same BACs were used for the two tissues. Interaction frequencies measured by cryo-FISH between two distant OR gene clusters in fetal liver and brain are indicated below the chromosomes.<br>
Figure 17<br>
4C analysis of HS2 and ^-major give highly similar results. (a( Unprocessed 4C data of four independent E14.5 liver samples show a very similar pattern of interaction with HS2 (top) and p-major (bottom), (b) A large overlap exists between probes scored positive for interaction in the HS-2 experiment and probes that scored positive for interaction in the [}-major experiment.<br>
Figure 18<br>
A comparison between interactions in cis and in trans, (a) Unprocessed 4C data from two independent experiments showing [3-globin interactions with a region positively identified in cis (chromosome 7, top) and a region hi trans containing the oc-globin locus (chr.ll, bottom), (b) Unprocessed 4C data from two independent experiments showing Rad23A interactions with a region positively identified in cis (chromosome 8, top) and a region in trans that appeared on top when ranked according to highest running mean value. None of the regions in trans met the stringent conditions that allowed the identification of long-interacting regions in cis.<br>
Figure 19<br>
Regions that interact with fJ-globin also frequently contact each other. Two regions<br>
(almost 60 Mb apart), containing actively transcribed genes and identified by 4C<br>
technology to interact with (3-globin in fetal liver, showed co-localization frequencies<br>
by cryo-FISH of 5.5%, which was significantly more than background co-localization<br>
frequencies.<br>
DETAILED DESCRIPTION OF THE INVENTION 3C TECHNOLOGY<br>
The 3C method has been described in detail in Dekker et al. (2002), Tolhuis et al. (2002), Palstra et al. (2003), Splinter et al. (2004) and Drissen et al. (2004). Briefly, 3C is performed by digesting cross-linked DNA with a primary restriction enzyme<br>
followed by ligation at very low DNA concentrations. Under these conditions, ligation of cross-linked fragments, which is intramolecular, is strongly favoured over ligation of random fragments, which is intennolecular. Cross-linking is then reversed and individual ligation products are detected and quantified by the polymerase chain reaction (PCR) using locus-specific primers. The cross linking frequency (X) of two specific loci is determined by quantitative PCR reactions using control and cross-linked templates, and X is expressed as the ratio of the amount of the product obtained with the cross-linked template and with the control template.<br>
In accordance with the present invention, a 3C template is prepared using the methods described by Splinter et a/., (2004) Methods Enzymol. 375, 493-507. (i.e. formaldehyde fixation, (primary) restriction enzyme digestion, re-ligation of cross-linked DNA fragments and DNA purification). Briefly, a sample - such as cells,tissues or nuclei — is fixed using a cross-linking agent - such as formaldehyde. The primary restriction enzyme digestion is then performed such that the DNA is digested in the context of the cross-linked nucleus. Intramolecular ligation is then performed at low DNA concentrations (for example, about 3.7ng/nl), which favours ligation between cross-linked DNA fragments (ie. intramolecular ligation) over ligation between non-cross-linked DNA fragments (ie. intermolecular or random ligation). Next, the cross links are reversed and the DNA can be purified. The 3C template that is yielded contains restriction fragments that are ligated because they were originally close in the nuclear space.<br>
Since a primary restriction enzyme is used to digest the DNA prior to the intramolecular ligation step, an enzyme recognition site for the primary restriction enzyme will separate the first (target) nucleotide sequence and the nucleotide sequence that has been ligated. Accordingly, the primary recognition site is located between the first (target) nucleotide sequence and the ligated nucleotide sequence (ie. the ligated second sequence).<br>
NUCLEOTIDE SEQUENCE<br>
The present invention involves the use of nucleotide sequences (eg. 3C templates, 4C templates, DNA templates, amplification templates, DNA fragments and genomic DNA), "which may be available in databases.<br>
The nucleotide sequence may be DNA or RNA of genomic, synthetic or recombinant origin e.g. cDNA. For example, recombinant nucleotide sequences may be prepared using a PCR cloning techniques. This will involve making a pair of primers flanking a region of the sequence which it is desired to clone, bringing the primers into contact with mRNA or cDNA obtained from, for example, a mammalian (eg. animal or human cell) or non-mammalian cell, performing a polymerase chain reaction (PCR) under conditions which bring about amplification of the desired region, isolating the amplified fragment (e.g. by purifying the reaction mixture on an agarose gel) and recovering the amplified DNA. The primers may be designed to contain suitable restriction enzyme recognition sites so that the amplified DNA can be cloned into a suitable cloning vector.<br>
The nucleotide sequence may be double-stranded or single-stranded whether representing the sense or antisense strand or combinations thereof.<br>
For some aspects, it is preferred that the nucleotide sequence is single-stranded DNA — such as single stranded primers and probes.<br>
For some aspects, it is preferred that the nucleotide sequence is double-stranded DNA - such as double stranded 3C and 4C templates.<br>
For some aspects, it is preferred that the nucleotide sequence is genomic DNA — such as one or more genomic loci.<br>
For some aspects, it is preferred that the nucleotide sequence is chromosomal DNA.<br>
The nucleotide sequence may comprise a first (target) nucleotide sequence and/or a second nucleotide sequence.<br>
Ihe primary and secondary restriction enzyme recognition sites will be different to sach other and will typically occur only once in the nucleotide sequence.<br>
In one aspect, there is provided a circularised nucleotide sequence comprising a first nucleotide sequence and (eg. ligated to) a second nucleotide sequence separated (eg. divided or parted) by a primary and a secondary restriction enzyme recognition site, wherein said first nucleotide sequence is a target nucleotide sequence and said second nucleotide sequence is obtainable by cross-linking genomic DNA (eg. in vivo or in vitro). The primary and secondary restriction enzyme recognition sites will be different to each other and will typically occur only once in liie nucleotide sequence.<br>
In a further aspect, there is provided a circularised nucleotide sequence comprising a first nucleotide sequence and (eg. ligated to) a second nucleotide sequence separated (eg. divided or parted) by a primary and a secondary restriction enzyme recognition site, wherein said first nucleotide sequence is a target nucleotide sequence and wherein said first and second nucleotide sequences are obtainable by a process comprising the steps of: (a) cross-linking genomic DNA (eg. in vivo or in vitro) ; (b) digesting the cross-linked DNA with a primary restriction enzyme; (c) ligating the cross-linked nucleotide sequences; (d) reversing the cross linking; and (e) digesting the nucleotide sequences with a secondary restriction enzyme to circularise the nucleotide sequences.<br>
Preferably, the second nucleotide sequence intersects (eg. bisects) the first (target) nucleotide sequence. Accordingly, the nucleotide sequence comprises the second nucleotide sequence, which separates the first (target) nucleotide sequence into two portions or fragments - such as approximately two equally sized portions or fragments. Typically, the portions or fragments will be at least about 16 nucleotides in length.<br>
FIRST NUCLEOTIDE SEQUENCE<br>
The first nucleotide sequence is a target nucleotide sequence.<br>
As used herein, the term "target nucleotide sequence" refers to the sequence that is used as a bait sequence in order to identify the one or more sequences to which it cross-links (eg. one or more nucleotide sequences of interest or one or more sequences of unknown nucleotide sequence composition).<br>
The target nucleotide sequence is of known sequence.<br>
Cross-Unking is indicative that the target nucleotide sequence and sequence cross-linked thereto were originally close in the nuclear space. By determining the frequency by which sequences are close to each other, it is possible to understand, for example, the conformation of chromosomes and chromosomal regions in the spatial context of the nucleus (eg. in vivo or in vitro). Moreover, it is possible to understand the intricate structural organisations within the genome, for example, when enhancers or other transcriptional regulatory elements communicate with distant promoters located in cis or even in trans. Furthermore, it is even possible to understand the positioning of a given genbmic region relative to nucleotide sequences present on the same chromosome (in cis) as well as to nucleotide sequences on other chromosomes (in trans). Thus, it is possible to map nucleotide sequences on different chromosomes mat frequently share sites in the nuclear space. Furthermore, it is even possible to detect balanced and/or unbalanced genetic aberrations - such as balanced and/or unbalanced translocations, deletions, inversions, duplications and other genomic rearrangements (eg. deletions or translocations in one or more chromosomes). In this regard, genetic aberrations result in changes hi the DNA-DNA interactions at the position that the change has occurred, which can be detected.<br>
The first (target) nucleotide sequence in accordance with the present invention can be any sequence in which it is desired to determine the frequency of interaction in the nuclear space with one or more other sequences.<br>
In one embodiment, the first (target) nucleotide sequence will be greater than about<br>
• 350 bp in length since a secondary restriction enzyme is chosen that cuts the first<br>
(target) nucleotide sequence at about 350 bp or more from the primary restriction site.<br>
This may minimise a bias in circle formation due to topologjcal constraints (Rippe et al. (2001) Trends in Biochem. Sciences 26, 733-40).<br>
Suitably, the first (target) nucleotide sequence following amplification comprises at least about 32 bp by virtue of the fact that the minimum length of the at least two amplification primers used to amplify the second nucleotide sequence are about 16 bases each.<br>
In a preferred embodiment, the first (target) nucleotide sequence may comprise completely or partially (eg. a fragment), or be close to (eg. in the proximity of), a promoter, an enhancer, a silencer, an insulator, a matrix attachment region, a locus control region, a transcription unit, an origin of replication, a recombination hotspot, a translocation breakpoint, a centromere, a telomere, a gene-dense region, a gene-poor region, a repetitive element, a (viral) integration site, a nucleotide sequence hi which deletions and/or mutations are related to an effect (e.g. disease, physiological, functional or structural effect - such as an SNP (single nucleotide polymorphism), or nucleotide sequence(s) containing such deletions and/or mutations, or any sequence in which it is desired to determine the frequency of interaction in the nuclear space with other sequences.<br>
As mentioned above, the first (target) nucleotide sequence may comprise completely or partially (eg. a fragment), or be close to (eg. in the proximity of) a nucleotide sequence in which genetic aberrations - such as deletions and/or mutations - are related to an effect (e.g. a disease). According to this embodiment of the invention the first (target nucleotide sequence) may therefore be a nucleotide sequence (eg. a gene or a locus), adjacent to (on the physical DNA template), or in the genomic region in which changes have been associated with or correlated to a disease - such as a genetic or congenital disease. In other words, the first (target) nucleotide sequence may be or may be chosen based on its association with a clinical phenotype. In a preferred embodiment, the changes are changes in one or more chromosomes and the disease may be as a consequence of, for example, one or more deletions, one or more translocations, one or more duplications, and/or one or more inversions etc therein.<br>
Non-limiting examples of such genes/loci are AML1, MLL, MYC, BCL, BCR, AJBL1, immunoglobulin loci, LYL1, TALI, TAL2, LMO2, TCRa/S, TCRfrHOX and other loci in various lymphoblastic leukemias.<br>
Other examples are described in electronic databases - such as:<br>
http://www.ncbi.nlm.nih.gov/entrez/query .fcgi?db=cancerchromosomes<br>
http://cgap.nci.nih.gov/Chromosomes/Mitelman<br>
htip://www.progenetk.net/progenetix/P 1460343 7/ideogram.html<br>
http://www.changbioscience.com/cytogenetics/cyto 1 .pr?query=47,xy<br>
http://www.possum.net.au/<br>
http://www.kndatabases.com/<br>
http://www.wiley.corn/legacy/products/subject/hWborgaonkar/mdex.html<br>
http^/www.ncbi.nhn.nih.gov/entrez/query.fcgi?db===OMIM<br>
http://www.sanger.ac.uk/PostGenomics/decipher/<br>
http://agserverO 1 .a2n.nl:8080/ecaruca/ecaruca.jsp<br>
Other examples are described in "Catalogue of Unbalanced Chromosome Aberrations in Man" 2nd edition. Albert Schinzel. Berlin: Walter de Gruyter, 2001. ISBN 3-11-011607-3.<br>
In one embodiment, the term "adjacent" means "directly adjacent" such that there are no intervening nucleotides between two adjacent sequences.<br>
In another embodiment, the term "adjacent" in the context of the nucleic acid sequence and the primary restriction enzyme recognition site means "directly adjacent" such that there are no intervening nucleotides between the nucleic acid sequence and the primary restriction enzyme recognition site.<br>
SECOND NUCLEOTIDE SEQUENCE<br>
The second nucleotide sequence is obtainable, obtained, identified, or identifiable by cross-linking genomic DNA (eg. in vivo or in vitro).<br>
The second nucleotide sequence (eg. nucleotide sequence of interest) becomes ligated to the first (target) nucleotide sequence after treating a sample with a cross-Unking agent and digesting/ligating the cross-linked DNA fragments. Such sequences are cross-linked to the first (target) nucleotide sequence because they were originally close in the nuclear space and ligated to the first (target) nucleotide sequence because ligation conditions favour ligation between cross-linked DNA fragments (intramolecular) over random ligation events.<br>
Diseases based on alterations - such as translocations, deletions, inversions, duplications and other genomic rearrangements - are generally caused by aberrant DNA-DNA interactions. 4C technology measures DNA-DNA interaction frequencies, which primarily are a function of the genomic site separation, ie. DNA-DNA interaction frequencies are inversely proportional to the linear.distance (in kilobases) between two DNA loci present on the same physical DNA template (Dekker et al., 2002). Thus, alteration^) which create new and/or physically different DNA templates, is accompanied by altered DNA-DNA interactions and this can be measured by 4C technology.<br>
Suitably, the second nucleotide sequence is at least 40 base paks.<br>
Cross-Unking agents — such as formaldehyde — can be used to cross rink proteins to other neighbouring proteins and nucleic acid. Thus, two or more nucleotide sequences can be cross-linked only via proteins bound to (one of) these nucleotide sequences. Cross-linking agents other than formaldehyde can also be used in accordance with the present invention, including those cross-linking agents that directly cross link nucleotide sequences. Examples of agents that cross-link DNA include, but are not limited to, UV light, mitomycin C, nitrogen mustard, melphalan, 1,3-butadiene diepoxide, cis diarninedichloroplatinurn(n) and cyclophosphamide.<br>
Suitably, the cross-linking agent will form cross-links that bridge relatively short distances - such as about 2 A - thereby selecting intimate interactions that can be reversed.<br>
Cross-linking may be performed by, for example, incubating the cells in 2% formaldehyde at room temperature - such as by incubating 1 x 107 cells in 10 ml of DMEM-10% FCS supplemented with 2% formaldehyde for 10 min at room temperature.<br>
PRIMARY RESTRICTION ENZYME<br>
As used herein, the term "primary restriction enzyme" refers to a first restriction enzyme that is used to digest the cross-linked DNA.<br>
The primary restriction enzyme will be chosen depending on the type of target sequence (eg. locus) to be analysed. It is desirable that preliminary experiments are performed to optimise the digestion conditions.<br>
The primary restriction enzyme may be selected from restriction enzymes recognising at least 6 bp sequences or more of DNA.<br>
Restriction enzymes that recognise 6 bp sequences of DNA include, but are not limited to, Acll, HindUI, Sspl, BspLUl II, Agel, MM, Spel, BgUI, Eco47m, StuI, Seal, Clal, AvaHI, Vspl, Mfel, PmaCI, PvuH, Ndel, Ncol, Smal, SacE, Avrll, Pvul, XmaDI, SplI, Xhol, PstI, Afin, EcoRI, Aattt, SacI, EcoRV, SphI, Nael, BsePI, Nhel, Bamffl, Narl, Apal, Kpnl, Snal, Sail, ApaLI, Hpal, SnaBI, Bspffl, BspMH, Nrul, Xbal, Bell, Msfl, BaU, Bspl407I,PsiI,AsuHandAham.<br>
Restriction enzymes that recognise more than a 6 bp sequence of DNA include, but are not limited to BbvC I, AscI, AsiS I, Fse I, Not I, Pac I, Pme I, Sbf I, SgrA I, Swa I, Sap I, Cci NI, FspA I, Mss I, Sgf I, Smi I, Srf I and Sse83871.<br>
For some aspects of the present invention, in the case of restriction enzymes recognizing 6 bp sequences, BglR, HindUL or EcoTHI are preferred<br>
The term "primary restriction enzyme recognition site" refers to the site in a nucleotide sequence that is recognised and cleaved by the primary restriction enzyme.<br>
SECONDARY RESTRICTION ENZYME<br>
As used herein, the term "secondary restriction enzyme" refers to a second restriction enzyme that is used after primary restriction enzyme digestion, ligation of cross-linked DNA, de-cross-Mnking and (optional) DNA purification. In one embodiment, the secondary restriction enzyme is used to provide defined DNA ends to the nucleotide sequences of interest, which allows for the ligation of sequences of known nucleotide composition to the secondary restriction enzyme recognition sites that flank the nucleotide sequences of interest.<br>
In one embodiment, ligation of sequences of known nucleotide composition to the secondary restriction enzyme recognition sites that flank (eg. are at each side or end of) the nucleotide sequences of interest involves ligation under diluted conditions to favour the intra-molecular ligation between the secondary restriction enzyme recognition sites that flank target nucleotide sequences and the linked nucleotide sequences of interest. This effectively results in the formation of DNA circles in which known target nucleotide sequences flank unknown sequences of interest.<br>
In another embodiment, ligation of sequences of known nucleotide composition to the secondary restriction enzyme recognition sites that flank (eg. are at each side or end of) the nucleotide sequences of interest involves the addition of unique DNA sequences of known nucleotide composition, followed by ligation under conditions that favour inter-molecular ligation between the secondary restriction enzyme recognition sites that flank the nucleotide sequences of interest and introduced unique DNA sequences of known nucleotide composition.<br>
In. one embodiment, the secondary restriction enzyme is chosen such that no secondary restriction enzyme sites are within about 350bp (eg. 350-400bp) of the primary restriction site.<br>
In another embodiment, the secondary restriction enzyme is chosen such that the same secondary restriction enzyme site is likely to be located in the ligated nucleotide sequence (ie. the ligated cross-linked sequence). Since the ends of the first (target) nucleotide sequence and the ligated nucleotide sequence may be compatible cohesive (or blunt) ends, the sequences may even be ligated in order to circularise the DNA. Accordingly, the digestion step is followed by ligation under diluted conditions that favour intra-molecular interactions and optional circularisation of the DNA via the compatible ends.<br>
Preferably, the secondary restriction enzyme recognition site is a 4 or 5 bp nucleotide sequence recognition site. Enzymes that recognise 4 or 5 bp sequences of DNA include, but are not limited to, TspEI, Maell, AM, NlaDI, HpaH, FnuDH, Mael, Dpnl, Mbol, Hhal, Haem, Rsal, TaqI, CviRI, Msel, Sthl32I, Acil, DpnE, Sau3AI and Mrdl.<br>
hi a preferred embodiment, the secondary restriction enzyme is Nlain and/or Dprdl.<br>
The term "secondary restriction enzyme recognition site" refers to the site in the nucleotide sequence that is recognised and cleaved by the secondary restriction enzyme.<br>
Following the digestion with the secondary restriction enzyme, a further ligation reaction is performed. In one embodiment, this ligation reaction links DNA sequences of known nucleotide sequence composition to the secondary restriction enzyme digestion site of the one or more sequences that are ligated to the target nucleotide sequence.<br>
TERTIARY RESTRICTION ENZYME<br>
As used herein, the term "tertiary restriction enzyme" refers to a third restriction enzyme that can be optionally used after the secondary restriction enzyme step in order to linearise circularised DNA prior to amplification.<br>
Preferably, the tertiary restriction enzyme is an enzyme that recognises a 6bp or more nucleotide recognition site.<br>
Preferably, the tertiary restriction enzyme digests the first (target) nucleotide sequence between the primary and secondary restriction enzyme recognition sites. As will be understood by a skilled person, it is desirable that the tertiary restriction enzyme does not digest the first (target) nucleotide sequence too close to the primary and secondary restriction enzyme recognition sites such that the amplification primers can no longer hybridise. Accordingly, it is preferred that the tertiary restriction enzyme recognition site is located at least the same distance away from the primary and secondary restriction enzyme recognition sites as the length of the primer to be used such that the amplification primer(s) can still hybridise.<br>
In a preferred embodiment, the tertiary restriction enzyme is one that recognises a 6-bp sequence of DNA.<br>
The term "tertiary restriction enzyme recognition site" refers to the site in the nucleotide sequence that is recognised and cleaved by the tertiary restriction enzyme.<br>
RECOGNITION SITE<br>
Restriction endonucleases are enzymes that cleave the sugar-phosphate backbone of DNA. In most practical settings, a given restriction enzyme cuts both strands of duplex DNA within a stretch of just a few bases. The substrates for restriction enzymes are sequences of double-stranded DNA called recognition sites/sequences.<br>
The length, of restriction recognition sites varies, depending on the restriction enzyme that is used. The length of the recognition sequence dictates how frequently the enzyme will cut in a sequence of DNA.<br>
By way of example, a number of restriction enzymes recognise a 4 bp sequence of DNA. The sequences and the enzyme that recognise the 4 bp sequence of DNA include, but are not limited to, AATT (TspEI), ACGT (Maell), AGCT (AM), CATG (NlaHI), CCGG (HpaH), CGCG (FnuDII), CTAG (Mael), GATC (Dpnl, DpnH, Sau3AI &amp; Mbol), GCGC (Hhal), GGCC (HaeD!), GTAC (Rsal), TCGA (TaqI), TGCA (CviRI), TTAA (Msel), CCCG (Sthl32I), CCGC (Acil) and CCTC (Mnll)<br>
By way of further example, a number of restriction enzymes recognise a 6 bp sequence of DNA. The sequences and the enzyme that recognise the 6 base-pair bp sequence of DNA include, but are not limited to, AACGTT (AclI), AAGCTT (HindHf), AATATT (Sspl), ACATGT (BspLUllI), ACCGGT (Agel), ACGCGT (MM), ACTAGT (Spel), AGATCT (BgHI), AGCGCT (Eco47m), AGGCCT (StuI), AGTACT (Seal), ATCGAT (Clal), ATGCAT (AvalU), ATTAAT (Vspl), CAATTG (Mfel), CACGTG (PmaCI), CAGCTG (PvuIT), CATATG (Ndel), CCATGG (Nfcol), CCCGGG (Smal), CCGCGG (SacD), CCTAGG .(Avrll), CGATCG (Pvul), CGGCCG (XmalU), CGTACG (SplI), CTCGAG (Xhol), CTGCAG (PstI), CTTAAG (Afill), GAATTC (EcoRI), GACGTC (Aaffl), GAGCTC (SacI), GATATC (EcoRV), GCATGC (SphT), GCCGGC (Nael), GCGCGC (BsePI), GCTAGC (Nhel), GGATCC (Bamffl), GGCGCC (Narl), GGGCCC (Apal), GGTACC (Kpnl), GTATAC (Snal), GTCGAC (Sail), GTGCAC (ApaLI), GTTAAC (HpaT), TACGTA (SnaBI), TCATGA (Bspffl), TCCGGA (BspMH), TCGCGA (Nrul), TCTAGA (Xbal), TGATCA (Bell), TGCGCA (MstI), TGGCCA (Ball), TGTACA (Bspl407I), TTATAA (Psil), TTCGAA (AsuH) and TTTAAA (Ahain).<br>
By way of further example, a number of restriction enzymes recognise a 7 bp sequence of DNA. The sequences and the enzyme that recognise the 7 bp sequence of DNA include, but are not limited to CCTNAGG (Saul), GCTNAGC (Espl), GGTNACC BstEH and TCCNGGA Pfol.<br>
By way of further example, a number of restriction enzymes recognise an 8 bp sequence of DNA. The sequences and the enzyme that recognise the 8 bp sequence of DNA include, but are not limited to ATTTAAAT (Swal), CCTGCAGG (Sse8387I), CGCCGGCG (Sse232I), CGTCGACG (SgrDI), GCCCGGGC (Stfl), GCGATCGC (Sgfl), GCGGCCGC (Nofl), GGCCGGCC (Fsd), GGCGCGCC (AscI), GTTTAAAC (Pmel) andTTAATTAA (PacI).<br>
A number of these enzymes contain the sequence CG that may be methylated in vivo. A number of restriction enzymes are sensitive to this methylation and will not cleave the methylated sequence, e.g. Hpall will not cleave the sequence CCmGG whereas its isoschizomer Mspl is insensitive to this modification and will cleave the methylated sequence. Accordingly, in some instances the eukaryotic methylation sensitive enzymes are not used.<br>
In one embodiment, a recognition site is a digestion site.<br>
In one embodiment, a restriction enzyme recognition site is a restriction enzyme digestion site.<br>
CIRCULARISING<br>
In accordance with one embodiment of the present invention, the material for 4C is prepared by creating DNA circles by digesting the 3C template with a secondary restriction enzyme, followed by ligation.<br>
Preferably, a secondary restriction enzyme is chosen that cuts the first (target) nucleotide sequence at greater than about 350bp (eg. 350-400bp) from the primary restriction site. Advantageously, this rninimises a bias in circle formation due to topological constraints (Rippe et al. (2001) Trends in Biochem. Sciences 26,733-40).<br>
Preferably, the secondary restriction enzyme is a frequent cutter recognising a 4 or a 5 bp restriction enzyme recognition site. Thus it is possible to obtain the smallest restriction fragments for equal amplification efficiencies of all ligated fragments during amplification.<br>
Prior to the secondary restriction enzyme digest and ligation, the DNA template will comprise one secondary enzyme recognition site in the first (target) nucleotide sequence located at greater than about 350-400bp from the primary restriction site and another secondary enzyme recognition site located in the nucleotide sequence that has been ligated (ie in the second nucleotide sequence).<br>
Preferably, the secondary restriction enzyme digestion step is performed for more 1 hour to overnight and followed by heat-inactivation of the enzyme.<br>
Preferably, the DNA in this reaction mixture is purified using conventional methods/kits that are known in the art.<br>
Following the secondary restriction enzyme digestion step, a secondary restriction enzyme site will be located at greater than 350-400bp from the primary restriction site in the first (target) nucleotide sequence and another secondary restriction enzyme site will be located in the ligated nucleotide sequence (ie. the second nucleotide sequence). Since the ends of the first (target) nucleotide sequence and the ligated nucleotide sequence have compatible ends, the sequences can be ligated in order to circularise the DNA.<br>
The digestion step is then followed by ligation under diluted conditions that favour intra-molecular interactions and circularisation of the DNA via the compatible ends.<br>
Preferably, the ligation reaction is performed at a DNA concentration of about 1-<br>
Preferably, the ligation reaction is performed for more than 1 hr (eg. 2, 3, 4 or more hrs) at about 16-25 °C.<br>
Accordingly, following the ligation reaction, circularised DNA may be prepared. The circularised DNA will comprise the recognition sites for at least the secondary restriction enzyme or the primary and the secondary restriction enzymes. In circularised DNA containing the first (target) nucleotide sequence, me primary restriction enzyme recognition site and the secondary restriction enzyme recognition sites will define the ends of the first (target) nucleotide sequence and the ligated. nucleotide sequence (ie. the second nucleotide sequence). Accordingly the first (target) nucleotide sequence and the ligated nucleotide sequence are separated (eg. divided) by the primary restriction enzyme recognition site and the secondary restriction enzyme recognition site.<br>
AMPLIFICATION<br>
One or more amplification reactions may be performed in order to amplify the 4G DNA templates.<br>
DNA amplification may be performed using a number of different methods that are known in the art. For example, DNA can be amplified using the polymerase chain reaction (Saiki et al., 1988); ligation mediated PCR, Qb replicase amplification (Cahill, Foster and Mahan, 1991; Chetverin and Spirin, 1995; Katanaev, Rurnasov and Spirin, 1995); the ligase chain reaction (LCR) (Landegren et al., 1988; Barany, 1991); the self-sustained sequence replication system (Fahy, Kwoh and Gingeras, 1991) and strand displacement amplification (Walker et aL, 1992).<br>
Preferably, DNA is amplified using PCR. "PCR" refers to the method of JL B. Mullis U.S. Pat Nos. 4,683,195, 4,683,202, and 4,965,188 that describe a method for increasing the concentration of a segment of a nucleotide sequence in a mixture of genomic DNA without cloning or purification.<br>
In one embodiment, inverse PCR is used. Inverse PCR (IPCR) (described by Ocbman et al (1988) Genetics 120(3), 621-3) is a method for the rapid in vitro amplification of DNA sequences that flank a region of known sequence. The method uses the polymerase chain reaction (PCR), but it has the primers oriented in the reverse direction of the usual orientation. The template for the reverse primers is a restriction fragment that has been ligated upon itself to form a circle. Inverse PCR has many applications in molecular genetics, for example, flae amplification and identification of sequences flanking transposable elements. To increase the efficiency and reproducibility of the amplification it is preferred that the DNA circles are linearised before amplification using a tertiary restriction enzyme. Preferably, a tertiary restriction enzyme that is a 6 bp or more cutter is used. Preferably, the tertiary restriction enzyme cuts the first (target) nucleotide sequence between the primary and secondary restriction enzyme sites.<br>
Digestion of the 3C template with the secondary restriction enzyme, optional circularisation, ligation (eg. ligation under diluted conditions) and optional linearisation of first (target) nucleotide sequence-containing circles yields a DNA template for amplification ("4C DNA template").<br>
For the amplification step, at least two oligonucleoti.de primers are used in which each primer hybridises to a DNA sequence that flanks the nucleotide sequences of interest. In a preferred embodiment, at least two oligonucleotide primers are used in which each, primer hybridises to the target sequence flanking the nucleotide sequences of interest-In one embodiment, the term "flank" in the context of primer hybridisation means that at least one primer hybridises to a DNA sequence adjacent one end (eg. the 5' end) of the nucleotide sequence of interest and at least one primer hybridises to a DNA sequence at the other end (eg. the 3' end) of the nucleotide sequence of interest. Preferably, at least one forward primer hybridises to a DNA sequence adjacent one end (eg. the 5' end) of the nucleotide sequence of interest and at least one reverse primer hybridises to a DNA sequence at the other end (eg. the 3' end) of the nucleotide sequence of interest.<br>
la a preferred embodiment, the term "flank" in the context of primer hybridisation means that at least one primer hybridises to a target sequence adjacent one end (eg. the 5' end) of the nucleotide sequence of interest and at least one primer hybridises to a target sequence at the other end (eg. the 3° end) of the nucleotide sequence of interest. Preferably, at least one forward primer hybridises to a target sequence adjacent one end (eg. the 5' end) of the nucleotide sequence of interest and at least one reverse primer hybridises to a target sequence at the other end (eg. the 3' end) of the nucleotide sequence of interest<br>
As used herein, the term "primer" refers to an oligonucleotide, whether occurring naturally as in a purified restriction digest or produced synthetically, which is capable of acting as a point of initiation of synthesis when placed under conditions in which synthesis of a primer extension product which is complementary to a nucleic acid strand is induced, (i.e., in the presence of nucleotides and an inducing agent such as DNA polymerase and at a suitable temperature and pH). The primer is preferably single stranded for maximum efficiency in amplification, but may be double stranded. If double stranded, the primer is first treated to separate its strands before being used to prepare extension products. Preferably, the primer is an oligodeoxyribonucleotide. The primer must be sufficiently long to prime the synthesis of extension products in the presence of the inducing agent. The exact lengths of the primers will depend on many factors, including temperature, source of primer and the use of the method.<br>
Suitably, the primers will be at least 15, preferably at least 20, for example at least 25, 30 or 40 nucleotides in length. Preferably, the amplification primers are from 16 to 30 nucleotides in length.<br>
Preferably, the primers are designed to be as close as possible to the primary and secondary restriction enzyme recognition sites that separate the first (target) nucleotide sequence and the second nucleotide sequence. The primers may be designed such that they are within about 100 nucleotides - such as about 90, 80, 70,60, 50,40, 30, 20, 10,<br>
9, 8, 7, 6, 5,4, 3,2 or 1 nucleotide(s) away from the primary and secondary restriction enzyme recognition sites.<br>
Suitably, the amplification primers are designed such that their 3' ends face outwards towards the primary and secondary restriction enzyme recognition sites so mat extension proceeds immedktely across the restriction sites into the second nucleotide sequence.<br>
If the amplification method that is used is inverse PCR, then it is preferred that the amplification reactions are carried out on about 100-400 ng of DNA of 4C template (per about 50 pJ PCR reaction mix) or other amounts of DNA for which replicate PCR reactions give reproducible results (see Figure 1) and include a maximum number of ligation events per PCR reaction.<br>
Preferably, the inverse PCR amplification reaction is performed using the Expand Long Template PCR System (Roche), using Buffer 1 according to the manufacturer's instructions.<br>
SAMPLE<br>
The term "sample" as used herein, has its natural meaning. A sample may be any physical entity comprising DNA that is or is capable-.of being cross-linked. The sample may be or may be derived from biological material.<br>
The sample may be or may be derived from one of more entities - such as one or more cells, one or more nuclei^ or one or more tissue samples. The entities may be or may be derivable from any entities in which DNA — such as chromatin - is present The sample may be or may be derived from one or more isolated cells or one or more isolated tissue samples, or one or more isolated nuclei.<br>
The sample may be or may be derived from living cells and/or dead cells and/or nuclear lysates and/or isolated chromatin.<br>
The sample may be or may be derived from diseased and/or non-diseased subjects.<br>
The sample may be or may be derived from a subject that is suspected, to be suffering from a disease.<br>
The sample may be or may be derived from a subject that is to be tested for the likelihood that they will suffer from a disease in the future.<br>
The sample may be or may be derived from viable or non-viable patient material.<br>
The fixation of cells and tissues for use in preparing the 3C template is described in detail in Splinter et al, (2004) Methods Enzymol. 375,493-507.<br>
Preferably, the nucleotide sequences (eg, amplified 4C DNA templates, primers or probes etc.) are labelled in order to assist in their downstream applications - such as array hybridisation. By way of example, the 4C DNA templates may be labelled using random priming or nick translation.<br>
A wide variety of labels (eg. reporters) may be used to label the nucleotide sequences described herein, particularly during the amplification step. Suitable labels include radionuclides, enzymes, fluorescent, chemiluminescent, or chromogenic agents as well as substrates, cofactors, inhibitors, magnetic particles and the like. Patents teaching the use of such labels include US-A-3817837; US-A-3850752; US-A-3939350; US-A-3996345; US-A-4277437; US-A-4275149 andUS-A-4366241.<br>
Additional labels include but are not limited to p-galactosidase, invertase, green fluorescent protein, luciferase, chloramphenicol, acetyltransferase, p-glucuronidase, exo-glucanase and glucoamylase. Fluorescent labels may also be used, as well as fluorescent reagents specifically synthesised with particular chemical properties. A<br>
ivide variety of ways to measure fluorescence are available. For example, some fluorescent labels exhibit a change in excitation or emission spectra, some exhibit resonance energy transfer where one fluorescent reporter looses fluorescence, while a second gains in fluorescence, some exhibit a loss (quenching) or appearance of fluorescence, while some report rotational movements.<br>
In order to obtain sufficient material for labelling, multiple amplifications may be pooled, instead of increasing the number of amplification cycles per reaction. Alternatively, labelled nucleotides can be incorporated in to the last cycles of the amplification reaction (e.g. 30 cycles of PCR (no label) +10 cycles of PCR (plus label)).<br>
ARRAY<br>
In a particularly advantageous embodiment, the 4C DNA templates that are prepared in accordance with the methods described herein can be hybridised to an array. Accordingly, array (eg. micro-array) technology can be used to identify nucleotide sequences — such as genomic fragments - that frequently share a nuclear site with a first (target) nucleotide sequence.<br>
Existing arrays - such as expression and genomic arrays - can be used in accordance with the present invention. However, the present invention also seeks to provide novel arrays (eg. DNA arrays) as described herein.<br>
An "array" is an intentionally created collection of nucleic acids which can be prepared either synthetically or biosynthetically and screened for biological activity in a variety of different formats (e.g., libraries of soluble molecules; and libraries of oligos tethered to resin beads, silica chips, or other solid supports). Additionally, the term "array" includes those libraries of nucleic acids which can be prepared by spotting nucleic acids of essentially any length (e.g., from 1 to about 1000 nucleotide monomers in length) onto a substrate.<br>
Array technology and the various techniques and applications associated with, it is described generally in numerous textbooks and documents. These include Lemieux et al, 1998, Molecular Breeding 4, 277-289, Schena and Davis. Parallel Analysis with Biological Chips, in PCR Methods Manual (eds. M. Tmris, D. Gelfand, J. Sninsky), Schena and Davis, 1999, Genes, Genomes and Chips. In DNA Microarrays: A Practical Approach (ed M. Schena), Oxford University Press, Oxford, UK, 1999), The Chipping Forecast (Nature Genetics special issue; January 1999 Supplement), Mark Schena (Ed), Microarray Biochip Technology, (Eaton Publishing Company), Cortes, 2000, The Scientist 14[17]:25, Gwynne and Page, Microarray analysis: the next revolution in molecular biology, Science, 1999 August 6; and Eakins and Chu, 1999, Trends in Biotechnology, 17, 217-218.<br>
Array technology overcomes the disadvantages with traditional methods in molecular biology, •which generally work on a "one gene in one experiment" basis, resulting in low throughput and the inability to appreciate the "whole picture" of gene function. Currently, the major applications for array technology include the identification of sequence (gene/gene mutation) and the determination of expression level (abundance) of genes. Gene expression profiling may make use of array technology, optionally in combination with proteomics techniques (Celis et al, 2000, FEES Lett, 480(1):2-16; Lockhart and Winzeler, 2000, Nature 405(6788): 827-836; Khan et al., 1999, 20(2):223-9). Other applications of array technology are also known in the art; for example, gene discovery, cancer research (Marx, 2000, Science 289: 1670-1672; Scherf, et al, 2000, Nat Genet;24(3):236-44; Ross et al, 2000, Nat Genet. 2000 Mar;24(3):227-35), SNP analysis (Wang et al, 1998, Science, 280(5366): 1077-82), drug discovery, pharmacogenomics, disease diagnosis (for example, utilising microfluidics devices: Chemical &amp; Engineering News, February 22, 1999, 77(8):27-36), toxicology (Rockett and Dix (2000), Xenobiotica, 30(2): 155-77; Afshari et al., 1999, Cancer Resl;59(19):4759-60) and toxicogenomics (a hybrid of functional genomics and molecular toxicology).<br>
In general, any library may be arranged in an orderly manner into an array, by spatially separating the members of the library. Examples of suitable libraries for arraying<br>
include nucleic acid libraries (including DNA, cDNA, oligonucleotide, etc libraries), peptide, polypepti.de and protein libraries, as well as libraries comprising any molecules, such as ligand libraries, among others.<br>
The samples (e.g., members of a library) are generally fixed or immobilised onto a solid phase, preferably a solid substrate, to limit diffusion and admixing of the samples. In a preferred embodiment, libraries of DNA binding ligands may be prepared. In particular, the libraries may be immobilised to a substantially planar solid phase, including membranes and non-porous substrates such as plastic and glass. Furthermore, the samples are preferably arranged in such a way that indexing (i.e., reference or access to a particular sample) is facilitated. Typically the samples are applied as spots in a grid formation. Common assay systems may be adapted for this purpose. For example, an array may be immobilised on the surface of a microplate, either with multiple samples in a well, or with a single sample in each well. Furthermore, the solid substrate may be a membrane, such as a nitrocellulose or nylon membrane (for example, membranes used in blotting experiments). Alternative substrates include glass, or silica based substrates. Thus, the samples are immobilised by any suitable method known hi the art, for example, by charge interactions, or by chemical coupling to the walls or bottom of the wells, or the surface of the membrane. Other means of arranging and fixing may be used, for example, pipetting, drop-touch, piezoelectric means, ink-jet and bubblejet technology, electrostatic application, etc. In the case of silicon-based chips, photolithography may be utilised to arrange and fix the samples on the chip.<br>
The samples may be arranged by being "spotted" onto the solid substrate; this may be done by hand or by making use of robotics to deposit the sample. In general, arrays may be described as macroarrays or microarrays, the difference being the size of the sample spots. Macroarrays typically contain sample spot sizes of about 300 microns or larger and may be easily imaged by existing gel and blot scanners. The sample spot sizes in microarrays are typically less than 200 microns in diameter and these arrays usually contain thousands of spots. Thus, microarrays may require specialized robotics<br><br>
and imaging equipment, which may need to be custom made. Instrumentation is described generally in a review by Cortese, 2000, The Scientist 14[11]:26.<br>
Techniques for producing immobilised libraries of DNA molecules have been described in the art Generally, most prior art methods described how to synthesise single-stranded nucleic acid molecule libraries, using for example masking techniques to build up various permutations of sequences at the various discrete positions on the solid substrate. U.S. Patent No. 5,837,832 describes an improved method for producing DNA arrays immobilised to silicon substrates based on very large scale integration technology. In particular, U.S. Patent No. 5,837,832 describes a strategy called "tiling" to synthesise specific sets of probes at spatially-defined locations on a substrate which may be used to produced the immobilised DNA libraries of the present invention. U.S. Patent No. 5,837,832 also provides references for earlier techniques that may also be used.<br>
Arrays may also be built using photo-deposition chemistry.<br>
Arrays of peptides (or peptidomimetics) may also be synthesised on a surface in a manner that places each distinct library member (e.g., unique peptide sequence) at a discrete, predefined location in the array. The identity of each library member is determined by its spatial location in the array. The locations in the array where binding interactions between a predetermined molecule (e.g., a target or probe) and reactive library members occur is determined, thereby identifying the sequences of the reactive library members on the basis of spatial location. These methods are described in U.S. Patent No. 5,143,854; WO90/15070 and WO92/10092; Fodor et at. (1991) Science, 251: 767; Dower andFodor (1991) Ann. Rep. Med. Chem., 26: 271.<br>
To aid detection, labels are typically used (as discussed above) - such as any readily detectable reporter, for example, a fluorescent, bioluminescent, phosphorescent, radioactive, etc reporter. Such reporters, their detection, coupling to targets/probes, etc are discussed elsewhere in this document. Labelling of probes and targets is also disclosed in Shalon et al, 1996, Genome Res 6(7):639-45.<br>
Specific examples of DNA arrays are as follow:<br>
Format I: probe cDNA (500-5,000 bases long) is immobilized to a solid surface such as glass using robot spotting and exposed to a set of targets either separately or in a mixture. This method is widely considered as having been developed at Stanford University (Skins and Chu, 1999, Trends in Biotechnology, 1999,17,217-218).<br>
Format II: an array of oligonucleotides (20-25-mer oligos, preferably, 40-60 mer oligos) or peptide nucleic acid (PNA) probes are synthesised either in situ (on-chip) or by conventional synthesis followed by on-chip immobilization. The array is exposed to labelled sample DNA, hybridised, and the identity/abundance of complementary sequences are determined. Such a DNA chip is sold by Affymetrix, Inc., under the GeneChip® trademark. Agilent and .Nimblegen also provide suitable arrays (eg. genomic tiling arrays).<br>
Examples of some commercially available microarray formats are set out in Table 1 below (see also Marshall and Hodgson, 1998, Nature Biotechnology &gt; 16(1), 27-31).<br>
(Table Removed)<br>
Table 1: Examples of currently available hybridization microarray formats<br>
In order to generate data from array-based assays a signal is detected that signifies the presence of or absence of hybridisation between, a probe and a nucleotide sequence. The present invention further contemplates direct and indirect labelling techniques. For example, direct labelling incorporates fluorescent dyes directly into the nucleotide sequences that hybridise to the array associated probes (e.g., dyes are incorporated into nucleotide sequence by enzymatic synthesis in the presence of labelled nucleotides or PCR primers). Direct labelling schemes yield strong hybridisation signals, typically using families of fluorescent dyes with similar chemical structures and characteristics, and are simple to implement In preferred embodiments comprising direct labelling of nucleic acids, cyanine or alexa analogs are utilised in multiple-fluor comparative array analyses. In other embodiments, indirect labelling schemes can be utilised to incorporate epitopes into the nucleic acids either prior to or after hybridisation to the microarray probes. One or more staining procedures and reagents are used to label the hybridised complex (eg., a fluorescent molecule that binds to the epitopes, thereby providing a fluorescent signal by virtue of the conjugation of dye molecule to the epitope of the hybridised species).<br>
Data analysis is also an important part of an experiment involving arrays. The raw data from an array experiment typically are images, which need to be transformed into matrices - tables where rows represent for example genes, columns represent for example various samples such as tissues or experimental conditions, and numbers in each cell for example characterise the expression of a particular sequence (preferably, a second sequence that has ligated to the first (target) nucleotide sequence) in the particular sample. These matrices have to be analysed further, if any knowledge about the underlying biological processes is to be extracted. Methods of data analysis (including supervised and unsupervised data analysis as well as bioinfoimatics approaches) are disclosed in Brazma and Vilo J (2000) FEBS Lett 480(1): 17-24.<br>
As described herein the one or more nucleotide sequences (eg. the DNA template) feat are labelled and subsequently hybridised to an array comprises a nucleotide sequence<br>
that is enriched for small stretches of sequences with a distinct signature ie. spanning the nucleotide sequence between the primary restriction enzyme recognition site that was ligated during the 3C procedure to the first (target) nucleotide sequence, and their respective neighbouring secondary restriction enzyme recognition sites.<br>
A single array may comprise multiple (eg. two or more) bait sequences. PROBES<br>
As used herein, the term "probe" refers to a molecule (e.g., an oligonucleotide, whether occurring naturally as in a purified restriction digest or produced synthetically, recombinantly or by PCR amplification), that is capable of hybridising to another molecule of interest (e.g., another oligonucleotide). When probes are oligonucleotides they may be single-stranded or double-stranded. Probes are useful in the detection, identification and isolation of particular targets (e.g., gene sequences). As described herein, it is contemplated that probes used in the present invention may be labelled with a label so that is detectable in any detection system, including, but not limited to enzyme (e.g., ELJSA, as well as enzyme-based histochemical assays), fluorescent, radioactive, and luminescent systems.<br>
With respect to arrays and microarrays, the term "probe" is used to refer to any •hybridisable material that is affixed to the array for the purpose of detecting a nucleotide sequence that has hybridised to said probe. Preferably, these probes are 25-60 mers or longer.<br>
Strategies for probe design are described in WO95/11995, EP 717,113 and WO97/29212.<br>
Since 4C allows an unbiased genome-wide search for interactions, it is advantageous to prepare an array with probes interrogating every possible (eg. unique/non-repetitive) primary restriction enzyme recognition site in the genome. Thus, array design only<br>
depends on the choice of primary restriction enzyme and not on the actual first or secondary nucleotide sequences.<br>
Whilst existing arrays can be used in accordance with the present invention, it is preferred to use alternative configurations.<br>
In one configuration, one or more probes on the array are designed such that they can hybridise close to the sites that are digested by the primary restriction enzyme. More preferably, the probe(s) are within about 20 bp of the primary restriction enzyme recognition site. More preferably, the probe(s) are within about 50 bp of the primary restriction enzyme recognition site.<br>
Suitably, the probe(s) are within about 100 bp (eg. about 0-100 bp, about 20-100 bp) of the primary restriction enzyme recognition site.<br>
In a preferred configuration, a single, unique, probe is designed within 100 bp at each side of the sites that are digested by the primary restriction enzyme.<br>
m another preferred configuration, the positions of sites digested by the secondary restriction enzyme relative to the positions of sites digested by the primary restriction sites are taken into account. In tihis configuration, a single, unique, probe is designed only at each side of the sites digested by the primary restriction enzyme that have the nearest secondary restriction enzyme recognition site at a distance large enough for a probe of a given length to be designed in between the primary and secondary restriction enzyme recognition site. In this configuration, for example, no probe is designed at the side of a particular primary restriction enzyme recognition site that has a secondary restriction enzyme recognition site within 10 bp at that same side.<br>
In another configuration, the probes on the array are designed such that they can hybridise at either side of the sites that are digested by the primary restriction enzyme. Suitably, a single probe at each side of the primary restriction enzyme recognition site can be used.<br>
In yet another configuration, two or more probes (eg. 3, 4, 5, 6, 7 or 8 or more) can be designed at each side of the primary restriction enzyme recognition site, which can then be used to investigate the same ligation event. For the number and position of probes relative to each primary restriction enzyme recognition site, the exact genomic location of its neighbouring secondary restriction enzyme recognition site can be taken into account.<br>
In yet another configuration, two or more probes (eg. 3, 4, 5, 6, 7 or 8 or more) can be designed near each primary restriction enzyme recognition site irrespective of the nearest secondary restriction enzyme recognition site. In this configuration, all probes should still be close to the primary restriction enzyme recognition sites (preferably within 300 bp of the restriction site).<br>
Advantageously, the latter design and also the design that uses 1 probe per (side of a) primary restriction enzyme recognition site, allows the use of different secondary restriction enzymes in combination with a given primary restriction enzyme.<br>
Advantageously, the use of multiple (eg. 2, 3, 4, 5, 6, 7 or 8 or more) probes per primary restriction enzyme recognition site can minimise the problem of obtaining false negative results due to poor performance of individual probes. Moreover, it can also increase the reliability of data obtained with a single chip experiment and reduce the number of arrays required to draw statistically sound conclusions.<br>
The probes for use in the array may be greater than 40 nucleotides in length and may be iso-thermal.<br>
Preferably, probes containing repetitive DNA sequences are excluded.<br>
Probes diagnostic for the restriction sites that directly flank or are -near to the first nucleotide sequence are expected to give very strong hybridisation signals and may also be excluded from the probe design.<br>
The array may cover any genome including mammalian (eg. human, mouse (eg. chromosome 7)), vertebrate (e.g. zebrafish)), or non-vertebrate (eg. bacterial, yeast, fungal or insect (eg. Drosophila)) genomes.<br>
In a further preferred embodiment, the array contains 2-6 probes around every unique primary restriction site and as close as possible to the site of restriction enzyme digestion.<br>
Preferably, the maximum distance from the site of restriction enzyme digestion is about 300 bp.<br>
In a further preferred embodiment of the present invention, arrays for restriction enzymes - such as HindSL, EcoSI, BglU and Notl - that cover the mammalian or non-mammalian genomes are provided. Advantageously, the design of the arrays described herein circumvent the need to re-design arrays for every target sequence, provided analysis is performed in the same species.<br>
SETS OF PROBES<br>
As used herein, the term "set of probes" refers to a suite or a collection of probes mat hybridise to each one of the primary restriction enzyme recognition sites for a primary restriction enzyme in a genome.<br>
Accordingly, there is provided in a further aspect, a set of probes complementary in sequence to the nucleic acid sequence adjacent to each one of the primary restriction enzyme recognition sites for a primary restriction enzyme in genomic DNA.<br>
Suitably, the set of probes are complementary in sequence to the first 25-60 (eg. 35-60, 45-60, or 50-60) or more nucleotides that are adjacent to each one of the primary restriction enzyme recognition sites in genomic DNA. The set of probes may be complementary in sequence to one (eg. either) side or both sides of the primary<br>
restriction enzyme recognition site. Accordingly, the probes maybe complementary in sequence to the nucleic acid sequence adjacent each side of each one of the primary restriction enzyme recognition sites in the genomic DNA.<br>
It is also possible to define a window (eg. 300bp or less - such as 250bp, 200bp, 150bp or lOObp - from the primary restriction enzyme recognition site) in which one or more probes for the set can be designed. Such factors that are important in defining the window within which to design the probes are, for example, GC-content, absence of palindromic sequences that can form hairpin structures, maximum size to stretches of a single type of nucleotide. Accordingly, the set of probes can be complementary in sequence to the nucleic acid sequence that is less than 300bp from each one of the primary restriction enzyme recognition sites in genomic DNA<br>
It is also possible to define a window of about 100 bp from the primary restriction enzyme recognition site in order to identify optima] probes near each restriction site.<br>
In further embodiments of the present invention, the set of probes are complementary to the sequence that is less then 300 bp from each one of the primary restriction enzyme recognition sites in genomic DNA, complementary to the sequence that is between 200 and 300 bp from each one of the primary restriction enzyme recognition sites in genomic DNA and/or complementary to the sequence that is between 100 and 200 bp from each one of the primary restriction enzyme recognition sites in genomic DNA.<br>
In further embodiments of the present invention, the set of probes are complementary to the sequence that is from 0 to 300 bp from each one of the primary restriction enzyme recognition sites in genomic DNA, complementary to the sequence that is between 0 to 200 bp from each one of the primary restriction enzyme recognition sites in genomic DNA and/or complementary to the sequence that is between 0 to 100 bp from each one of the primary restriction enzyme recognition sites 'm genomic DNA (eg. about 10, 20, 30, 40, 50, 60, 70, 80 or 90 bp from each one of the primary restriction enzyme recognition sites in genomic DNA).<br>
Two or more probes may even be designed that are capable of hybridising to the sequence adjacent each primary restriction enzyme recognition site in the genomic<br>
DNA.<br>
The probes may overlap or partially overlap. If the probes overlap then the overlap is preferably, less than 10 nucleotides.<br>
PCR fragments representing the first 1-300 nucleotides (eg. 1-20, 1-40, 1-60, 1-80, 1-100, 1-120,1-140,1-160,1-180,1-200,1-220, 1-240,1-260 or 1-280 nucleotides) mat flank each primary restriction enzyme recognition site can also be used.<br>
PCR fragments may also be used as probes that exactly correspond to each genomic site that is flanked by the primary restriction enzyme recognition site and the first neighboring second restriction enzyme recognition site. Accordingly, the probe sequence may correspond to all or part of the sequence between each one of the primary restriction enzyme recognition sites and each one of the first neighbouring secondary restriction enzyme recognition sites.<br>
Typically, the probes, array of probes or set of probes will be immobilised on a support. Supports (eg. solid supports) can be made of a variety of materials - such as glass, silica, plastic, nylon or nitrocellulose. Supports are preferably rigid and have a planar surface. Supports typically have from about 1-10,000,000 discrete spatially addressable regions, or cells. Supports having about 10-1,000,000 or about 100-100,000 or about 1000-100,000 cells are common. The density of cells is typically at least about 1000, 10,000, 100,000 or 1,000,000 cells within a square centimeter. In some supports, all cells are occupied by pooled mixtures of probes or a set of probes. In other supports, some cells are occupied by pooled mixtures of probes or a set of probes, and other cells are occupied, at least to the degree of purity obtainable by synthesis methods, by a single type of oligonucleotide.<br>
Preferably, the array described herein comprises more than, one probe per primary restriction enzyme recognition site, which in the case of a 6 bp cutting restriction enzyme occurs, for example, approximately 750,000 times per human or mouse genome.<br>
For a restriction enzyme recognising a &gt;6 bp recognition sequence, a single array of about 2 x 750,000 probes can be used to cover, for example, the complete human or mouse genome, with 1 probe at each side of each restriction site.<br>
In a preferred, array design, the total number of probe molecules of a given nucleotide sequence present on the array is in large excess to homologous fragments present in the 4C sample to be hybridized to such array. Given the nature of 4C technology, fragments representing genomic regions close to the analyzed nucleotide sequence on the linear chromatin template will be hi large excess in the 4C hybridization sample (as described in Figure 2). To obtain, quantitative information about hybridization efficiencies of such abundant fragments, it may be necessary to reduce the amount of sample to be hybridized and/or increase the number of molecules of a given oligonucleotide sequence probe on the array.<br>
Thus, for the detection of regulatory DNA elements that frequently contact, for example, a gene promoter element it may be necessary to use an array with probes that represent only the selected genomic region (eg. about 0.5-10 Mb), but with each unique probe present at multiple (eg. about 100, 200, 1000) positions on the array. Such designs may also be preferred for diagnostic purposes to detect local (eg. within about 10 Mb) genomic rearrangements - such as deletions, inversions, duplications, etc. - around a site (e.g. gene of interest).<br>
The array may comprise about 3 x 750,000 probes, 4 x 750,000 probes, 5 x 750,000 probes, or preferably, 6 X 750,000 probes. More preferably, the array comprises 6 x 750,000 probes with 2, 3,4, 5, 6, 7 or 8 or more probes at each side of each restriction site. Most preferably, the array comprises 6 x 750,000 probes with 3 probes at each side of each restriction site.<br>
Arrays of probes or sets of probes may be synthesised in a step-by-step manner on a support or can be attached in presynthesized form. One method of synthesis is VLSIPS.TM. (as described in US 5,143,854 and EP 476,014), which entails the use of light to direct the synthesis of oligonucleotide probes in high-density, miniaturised arrays. Algorithms for design of masks to reduce the number of synthesis cycles are described in US 5,571,639 and US. 5,593,839. Arrays can also be synthesised in a combinatorial fashion by delivering monomers to cells of a support by mechanically constrained flowpaths, as described in EP 624,059. Arrays can also be synthesised by spotting reagents on to a support using an ink jet printer (see, for example, EP 728,520).<br>
In the context of the present invention, the terms "substantially a set of probes" "substantially the array of probes" means that the.set or the array of probes comprises at least about 50, 60, 70, 80, 90, 95, 96, 97, 98 or 99% of the full or complete set or array of probes. Preferably, the set or the array of probes is a full or complete set of probes (ie. 100%).<br>
In a preferred embodiment, the array comprises a single unique probe per side of each primary restriction enzyme recognition site that is present in a given genome. If this number of probes exceeds the number of probes that can be contained by a single array, the array may preferably still contain a representation of the complete genome of a given species, but at lower resolution, with for example one out of every 2, 3,4, 5, 6, 7, 8, 9,10,102,103 or 104 etc. probes as ordered on the linear chromosome template present on the array. Such arrays that cover the complete human, or other, genome at sub-optimal resolution may be preferred over high-resolution arrays that cover part of the same genome, for example in cases where translocation partners are to be found.<br>
Preferably, the representation of the complete genome of a given species at lower resolution is obtained by probes on the array that each represent a single restriction fragment as obtained after digestion with a primary restriction enzyme. Preferably, this is obtained by ignoring every second, third, fourth, fifth, sixth, seventh, eighth,<br>
ninth, tenth, twentieth, thirtieth, fortieth, fiftieth, sixtieth, seventieth, eightieth, ninetieth or one hundredth etc. probe that hybridises to the same restriction fragment.<br>
Preferably, the representation of the complete genome of a given species at lower resolution comprises probes that are distributed equally along the linear chromosome templates. Preferably, this is obtained by ignoring one or more probes in those genomic regions that show highest probe density.<br>
HYBRIDISATION<br>
The term "hybridisation" as used herein shall include "the process by which a strand of nucleic acid joins with a complementary strand through base pairing" as well as the process of amplification as carried out in, for example, polymerase chain reaction (PCR) technologies.<br>
Nucleotide sequences capable of selective hybridisation will be generally be at least 75%, preferably at least 85 or 90% and more preferably at least 95% or 98% homologous to the corresponding complementary nucleotide sequence over a region of at least 20, preferably at least 25 or 30, for instance at least 40, 60 or 100 or more contiguous nucleotides.<br>
"Specific hybridisation" refers to the binding, duplexing, or hybridising of a molecule only to a particular nucleotide sequence under stringent conditions (e.g. 65°C and O.lxSSC (IxSSC = 0.15 M NaCl, 0.015 M Na-citrate pH 7.0}). Stringent conditions are conditions under which a probe will hybridise to its target sequence, but to no other sequences. Stringent conditions are sequence-dependent and are different in different circumstances. Longer sequences hybridise specifically at higher temperatures. Generally, stringent conditions are selected to be about 5 °C lower than the thermal melting point (Tm) for the specific sequence at a defined ionic strength and pH. The Tm is the temperature (under defined ionic strength, pH, and nucleic acid concentration) at which 50% of the probes complementary to a target sequence hybridise to the target sequence at equilibrium. (As the target sequences are generally<br>
present in excess, at Tin, 50% of the probes are occupied at equilibrium). Typically, stringent conditions include a salt concentration of at least about 0.01 to 1.0 M Na ion concentration (or other salts) at pH 7.0 to 8.3 and the temperature is at least about 30 °C for short probes. Stringent conditions can also be achieved with the addition of destabilising agents - such as formamide or tetraalkyl ammonium salts.<br>
As will be understood by those of skill in the art, a maximum, stringency hybridization can be used to identify or detect identical nucleotide sequences while an intermediate (or low) stringency hybridization can be used to identify or detect similar or related polynucleotide sequences.<br>
Methods are also described for the hybridisation of arrays of probes to labelled or unlabeled nucleotide sequences. The particular hybridisation reaction conditions can be controlled to alter hybridisation (e.g., increase or decrease probe/target binding stringency). For example, reaction temperature, concentrations of anions and cations, addition of detergents, and the like, can all alter the hybridisation characteristics of array probes and target molecules.<br>
FREQUENCY OF INTERACTION<br>
Quantifying ligation frequencies of restriction fragments gives a measure of their cross-linking frequencies. Suitably, this can be achieved using PCR as used in conventional 3C technology as described by Splinter et al. (2004) (supra). Briefly, the formation of PCR products can be measured by scanning the signal intensities after separation on ethidium bromide stained agarose gels, using a Typhoon 9200 imager (Molecular Dynamics, Sunnyvale, CA). Suitably, several controls are used for the correct interpretation of data as also described in Splinter et al. (2004) (supra).<br>
Since the 4C technology described herein provides for the high-throughput analysis of the frequency of interaction of two or more nucleotide sequences in the nuclear space, it is preferred that the ligation frequencies of restriction fragments are quantified using the arrays described herein.<br>
For quantitation, signals obtained for a 4C sample can be normalised to signals obtained for a control sample. 4C sample and control sample(s) will be labelled with, different and discemable labels (eg. dyes) and will be simultaneously hybridised to the array. Control sample(s) will typically contain all DNA fragments (i.e. all potential second nucleotide sequences that have ligated to the first (target) nucleotide sequence) in equimolar amounts and, to exclude a bias in hybridisation efficiency, they should be similar in size to the second nucleotide sequence(s). Thus, control template will typically contain genomic DNA (of the same genetic background as that used to obtain the 4C template), digested with both the primary and the secondary restriction enzyme and labelled by the same method (e.g. random priming) as the 4C template. Such control template makes it possible to correct for probe-to-probe differences in hybridisation efficiency. Normalising 4C array signals to control array signals makes it possible to express results in terms of enrichment over random events.<br>
Labeled 4C template may even be hybridized to an array with or without a differentially labeled control sample and with or without one or more differentially labeled other 4C templates. Other 4C templates can be unrelated to this 4C template, for example it may be obtained from different tissue and/or obtained with a different set of inverse PCR primers. For example, the first 4C template may be patient material and the second 4C template may be obtained from a healthy subject or a control sample.<br>
Given the striking hybridisation patterns that are to be expected for genetic rearrangements it will not always be necessary to compare diseased subjects with healthy subjects. Accordingly, multiple (eg. two or more) 4C templates, each interrogating a different locus from the same patient or subject may be hybridized to one (eg. one or more) array.<br>
The 4C templates may be differentially labeled (eg. with two or multi-color hybridization) and/or may be identically labeled in case such loci normally reside on different chromosomes or on the same chromosome at a distance far enough for<br>
minimal overlap between DNA-DNA interaction signals. As an example, material from a subject with T-cell leukemia may be processed to obtain 4C templates for TCRa/5 (labeled in one color, in order to detect translocations), and MLL, TALI, HOX11 and LMO2 (each labeled in the same second color, in order to detect other genetic rearrangements). These five 4C templates may be hybridized to one array, which will allow the simultaneous analysis at multiple loci for a genomic rearrangement associated with the disease.<br>
For quantification of interaction frequencies, absolute signal intensities or ratios over control sample may also be considered. In addition, signals of probes adjacent on the linear chromosome template may be used to identify interacting chromosomal regions. Such positional information is preferably analyzed by ordering the probes on the linear chromosome template and analysing the absolute signal intensities, or ratios over control template signals, by sliding window approaches, using for example running mean or running median approaches.<br>
ASSAY METHOD<br>
In a further aspect of the present invention, there is a provided an assay method for identifying one or more agents that modulate a DNA-DNA interaction.<br>
As used herein, the term "modulate" refers to preventing, decreasing, suppressing, restorating, elevating, increasing or otherwise affecting the DNA-DNA interaction.<br>
In some cases, it may be desirable to evaluate two or more agents together for use in modulating the DNA-DNA interaction. In these cases, assays may be readily modified by adding such additional agent(s) either simultaneously with, or subsequently to, the first agent.<br>
The method of the present invention may also be a screen, whereby a number of agents are tested for modulating the activity of the DNA-DNA interaction.<br>
It is expected that the assay methods of the present invention will be suitable for both small and large-scale screening of agents as well as in quantitative assays.<br>
Medical uses of such therapeutic agents are within the scope of the present invention as are the drug development programs themselves and pharmaceutical compositions comprising such agents. A drag development program may, for example, involve taking an agent identified or identifiable by the methods described herein, optionally modifying it (e.g. modifying its structure and/or providing a novel composition comprising said moiety) and performing further studies (e.g. toxicity studies and /Or studies on activity, structure or function). Trials may be performed on non-human animals and may eventually be performed on humans. Such trials will generally include determining the effect(s) of different dosage levels. Drug development programs may utilise computers to analyse moieties identified by screening (e.g. to predict structure and/or function, to identify possible agonists or antagonists, to search for other moieties that may have similar structures or functions, etc.).<br>
DIAGNOSTIC TESTING<br>
Currently, various genomic rearrangements remain difficult to detect by available molecular-cytogenetic techniques. Although the array comparative genomic hybridization technique (array-CGH) is a newly developed technique for the detection of chromosomal amplification and/or deletions with a resolution of 35-300 Kb, this technique is not suitable to detect balanced translocations and chromosomal inversions. On the other hand, spectral karyotyping (SKY) or conventional karyotyping is often performed on patient material for the detection of chromosomal translocations as well as numerical changes, but the resolution to define translocation breakpoints is low, usually 10-50 Mb and 5-10 Mb, respectively. Consequently, results obtained by both methods and especially SKY will lead to time-consuming and labor-intensive validations experiments Eke fluorescence in situ hybridization (FISH) and molecular breakpoint cloning strategies.<br>
4C technology involves a procedure that can detect any chromosomal rearrangements on the basis of changed interaction frequencies between physically linked DNA sequences. 4C technology is therefore useful for the identification of (recurrent) chromosomal rearrangements for most human malignancies/multiple congenital malformations or mental retardation. An important advantage of 4C technology is that it allows for the very accurate mapping of the breakpoint to a region of only several thousands of basepairs. Another advantage of 4C technology is that no prior knowledge is required on the exact position of the breakpoint, since breakpoints will be detectable even when the 4C-bait sequence is located 1-5 Mb away from the breakpoint. This has also 1he advantage that the same bait sequence can be used for the detection of specific chromosomal rearrangements covering large breakpoint areas. The accurate mapping of genomic rearrangements by 4C technology will greatly facilitate the identification of aberrantly expressed gene(s) underlying diseases or genetic disorders, which will importantly contribute to a better understanding of the genotype-phenotype correlations, assist in treatment decision-making and add important prognostic information.<br>
In one embodiment of the present invention, in order to provide a basis for the diagnosis or prognosis of disease, normal or standard values from a subject are established. This may be accomplished by testing samples taken from normal subjects - such as animals or humans. The frequency of the DNA-DNA interaction may be quantified by comparing it to a dilution series of positive controls. Then, standard values obtained from normal samples may be compared with values obtained from samples from subjects affected or potentially affected by a disease or a disorder. Deviation between standard and subject values establishes the presence of the disease state.<br>
Such diagnostic assays may be tailored to evaluate the efficacy of a particular therapeutic treatment regime and may be used in animal studies, in clinical trials, or in monitoring the treatment of an individual patient. In order to provide a basis for the diagnosis of disease, a normal or standard profile for the DNA-DNA interaction may be established. Standard values obtained from normal samples may be compared with<br>
values obtained from samples from subjects potentially affected by a disorder or disease. Deviation between standard and subject values establishes the presence of the disease state. If disease is established, an existing therapeutic agent may be administered, and treatment profile or values may be generated. Finally, the method may be repeated on a regular basis to evaluate whether the values progress toward or return to the normal or standard pattern. Successive treatment profiles may be used to show the efficacy of treatment over a period of several days or several months.<br>
4C technology accurately detects at least 5Mb of genomic DNA linked in cis to the nucleotide sequence that is analysed (see Figure 2-3 and 5). Advantageously, 4C technology may be used to detect any genomic aberration that is accompanied by a change in genomic site separation between rearranged sequences and a 4C sequence (bait) of choice. Such change may be, for example, an increase or decrease in genomic site separation or may be an under-representation (as in deletions) or over-representation (as in duplications) of sequences proximal (eg. up to or greater than 15 Mb) to the 4C sequence (bait). Typically, such genomic aberrations or rearrangements are a cause of or are associated with diseases - such as cancer (eg. leukaemia) and other genetic or congenital diseases as described herein.<br>
Genetic aberrations (eg. genomic or chromosomal aberrations - such as balanced and/or or unbalanced genomic or chromosomal aberrations) include, but are not limited to rearrangements, translocations, inversions, insertions, deletions and other mutations of nucleic acid (eg. chromosomes) and also losses or gains of part or whole chromosomes. They are a leading cause of genetic disorders or diseases, including congenital disorders and acquired diseases - such as malignancies. In many rearrangements, two different chromosomes are involved. In this way, genes (or fragments of genes) are removed from the normal physiological context of a particular chromosome and are located to a recipient chromosome, adjacent to non-related genes or fragments of genes (often oncogenes or proto-oncogenes).<br>
Malignancies can include acute leukemias, malignant lymphomas and solid tumours. Non-limiting examples of alterations are t(14;18) which occurs frequently in NHL;<br>
t(12;21) which is frequently found in childhood precursor-B-ALL; and the presence of Ilq23 (MLL (myeloid-lymphoid leukaemia or mixed-lineage leukaemia) gene) aberrations in acute leukemias.<br>
The MLL gene ia chromosome region Ilq23 is involved in several translocations in both ALL and acute myeloid leukemias (AML). To date, at least ten partner genes have been identified. Some of these translocations, - such as t(4;ll) (q21;q23), t(ll;19) (q23;p!3) and t(l;ll) (p32;q23), predominantly occur in ALL, where as others, like t(l;ll) (q21;q23), t(2;ll) (p21;q23), t(6;ll) (q27;q23) and t(9;ll) (p22;q23) are more often observed in AML. Rearrangements involving the Ilq23 region occur very frequently in infant acute leukemias (around 60-70%), and to a much lesser extent in childhood and adult leukemias (each around 5%).<br>
Rearrangements in lymphoid malignancies often involve Ig or TCR genes. Examples include the three types of translocations (t(8;14), t(2;8), and t(8;22)) that are found in Burkitt's lymphomas, in which the MYC gene is coupled to Ig heavy chain (IGH), Ig kappa (IGK), or Ig lambda (IGL) gene segments, respectively. Another common.type of translocation in this category is t(14;18) (q32;q21) which is observed in about 90% of follicular lymphomas, one of the major NHL types. In this translocation the BCL2 gene is rearranged to regions within the IGH locus within or adjacent to the JH gene segments. The result of this chromosome aberration is the overexpression of the BCL2 protein, which plays a role as a survival factor in growth control by inhibiting programmed cell death.<br>
The BCL2 gene consists of three exons, but these are scattered over a large area. Of these the last exon encodes a large 3' untranslated region (31 UTR). This 3' UTR is one of the two regions in which many t(14;18) breakpoints are clustered and is called the "major breakpoint region"; the other breakpoint region involved in t(14;18) translocations, is located 20-30 kb downstream of me BCL2 locus and is called the "minor cluster region". A third BCL2 breakpoint area, the VCR (variant cluster region), is located at the 5' side of the BCL2 locus and is amongst others involved in<br>
variant translocations, i.e., t(2;18) and t(18;22), in which IGK andlGL gene segments are the partner genes.<br>
Thus, by way of example, 4C technology can be applied to me screening of patient material for genetic aberrations near or in loci that were chosen based on their frequent association with a given clinical phenotype. Further non-limiting examples of such loci are AML1, MIL, MFC, BCL, BCR, ABU, immunoglobulin loci, LTL1, TALI, TAL2, LMO2, TCRa/8, TCRJ3, HOX and other loci in various lymphoblastic leukemias.<br>
Advantageously, if a genetic aberration is suspected, 4C technology can be applied as the first and only screen to verify and map the presence of the aberration as explained herein.<br>
Detection ofgenomic rearrangements<br>
In a particularly preferred embodiment of the present invention, the methods described herein can be used for the detection of genomic rearrangements.<br>
Currently, genomic rearrangements - such as translocation breakpoints - are very difficult to detect. For example, comparative genomic hybridization (CGH) micro-arrays can detect several types of rearrangements but fail to detect translocations. If translocation is suspected in a patient but chromosome partners are unknown, spectral karyotyping (SKY) may be performed to find translocation partners and obtain an approximate estimate of breakpoint locations. However, the resolution is very poor (usually not better than -50 Mb) and additional fine-mapping (which is both time consuming and expensive) is usually required. This is normally done using Fluorescence In Situ Hybridization (FISH), which again provides limited resolution. Using FISH, breakpoints can be located to +/-50 kb region at maximum resolution.<br>
DNA-DNA interaction frequencies primarily are a function of the genomic site separation, i.e. DNA-DNA interaction frequencies are inversely proportional to the<br>
linear distance (in kilobases) between, two DNA loci present on the same physical DNA template (Dekker et al., 2002). Thus, a translocation, which creates one or more new physical DNA templates, is accompanied by altered DNA-DNA interactions near the breakpoints, and this can be measured by 4C technology. Diseases based on translocations are typically caused by aberrant DNA-DNA interactions, as translocation is the result of the physical linkage (interaction) of broken chromosome (DNA) arms.<br>
Accordingly, for the detection of translocations, 4C technology may be used to identify those DNA-DNA interactions that are different between diseased and non-diseased subjects.<br>
By way of example, 4C technology can be applied to the screening of patient material for translocations near loci that were chosen based on- their frequent association with a given clinical phenotype as described herein.<br>
If translocation is suspected in a patient but chromosome partners are unknown, an initial mapping may be performed using currently available methods like spectral karyotyping (SKY). This may identify the translocation partners and provide a very rough estimate of breakpoint locations (usually not better than ~50 Mb resolution). 4C technology can then be applied, using 'bait'-sequences in this region located for example at every 2 Mb, 5Mb, 10Mb, 20Mb (or other intervals as described herein) to fine map the breakpoint and identify for example the gene(s) that are mis-expressed as a consequence of the translocation.<br>
Typically a translocation will be identified by way of an abrupt transition from low to high interaction frequencies on a chromosome other than the one containing the 4C-bait sequence, or elsewhere on that same chromosome.<br>
In a preferred embodiment, the sample from the subject is in a pre-malignant state.<br>
In a preferred embodiment, the sample from the subject consists of cultured or uncultured amniocytes obtained by amniocentesis for prenatal diagnosis.<br>
In a preferred array design, probes present on a single array represent the complete genome of a given species at maximum resolution. Thus, arrays to detect translocations and the like by 4C technology contain probes as described herein complementary, to every side of every primary restriction enzyme recognition site in the genome of a given species (e.g. human).<br>
In another preferred design, probes present on a single array represent the complete genome of a given species, but not at maximum resolution. Thus, arrays to detect translocations and the like by 4C technology contain probes as described herein that are complementary to only one side of every primary restriction enzyme recognition site in the genome of a given species (e.g. human).<br>
In another preferred design, probes present on a single array represent the complete genome of a given species, but not at maximum resolution. Thus, arrays to detect translocations, deletions, inversions, duplications and other genomic rearrangements by 4C technology contain probes as described herein that are •complementary to one side of every other primary restriction enzyme recognition site as ordered along the linear template of the genome of a given species (e.g. human).<br>
Thus, arrays to detect translocations, deletions, inversions, duplications and other genomic rearrangements by 4C technology contain probes as described herein that each represent a single restriction fragment as obtained after digestion with a primary restriction enzyme. Preferably, this is obtained by ignoring every second, third, fourth, fifth, sixth, seventh, eight, ninth, tenth, twentieth, thirtieth, fortieth, fiftieth, sixtieth, seventieth, eightieth, ninetieth, or one hundredth etc probe that hybridizes to the same restriction fragment. Arrays to detect translocations, deletions, inversions, duplications and other genomic rearrangements by 4C technology may contain probes as described herein that are distributed equally along the linear chromosome templates. Preferably,<br>
this is obtained by ignoring one or more probes in those genomic regions that show highest probe density.<br>
In another preferred design, probes present on a single array represent the complete genome of a given species, but not at maximum resolution. Thus, arrays to detect translocations, deletions, inversions, duplications and other genomic rearrangements by 4C technology contain probes as described herein complementary to one side of every third, fourth, fifth, sixth, seventh, eight, ninth, tenth, twentieth, thirtieth, fortieth, fiftieth, sixtieth, seventieth, eightieth, ninetieth, or one hundredth etc primary restriction enzyme recognition site as ordered along the linear template of the genome of a given species (e.g. human). Arrays to detect translocations, deletions, inversions, duplications and other genomic rearrangements by 4C technology may contain, probes as described herein, which represent the complete genome, but with a single probe every 100 kilobases. Arrays to detect translocations, deletions, inversions, duplications and other genomic rearrangements by 4C technology may contain probes as described herein which represent every single primary restriction enzyme recognitiorrsite in-the genome that can be represented by a unique probe sequence.<br>
In another preferred array design, probes as described herein on a single array represent genomic regions of a given size - such as about 50 kb, 100 kb, 200 kb, 300 kb, 400 kb, 500 kb, 1 Mb, 2 Mb, 3Mb, 4Mb, 5Mb, 6Mb, 7Mb, 8Mb, 9Mb or 10Mb -(eg. from about 50kb-10Mb) around all loci known to be involved in translocations, deletions, inversions, duplications and other genomic rearrangements.<br>
In another preferred array design, probes as described herein on a single array represent genomic regions of a given size - such as about 50 kb, 100 kb, 200 kb, 300 kb, 400 kb, 500 kb, 1 Mb, 2 Mb, 3Mb, 4Mb, 5Mb, 6Mb, 7Mb, 8Mb, 9Mb or 10Mb -(eg. from about 50kb-10Mb) around a selection of loci known to be involved in translocations, deletions, inversions, duplications and other genomic rearrangements. Selections can be made on educated criteria, for example they can represent only the loci that are implicated in a given type of disease.<br>
In another preferred array design, probes as described herein on a single array represent a genomic region of interest of, for example, 100 kb, 200 kb, 300 kb, 400 kb, 500 kb, 600 kb, 700 kb, 800 kb, 900 kb, 1 Mb, 2 Mb, 3 Mb, 4 Mb, 5 Mb, 6 Mb, 7 Mb, 8 Mb, 9 Mb, 10 Mb, 20 Mb, 30 Mb, 40 Mb, 50 Mb, 60 Mb, 70 Mb, 80 Mb, 90 Mb, or 100 Mb (eg. lOOkb-lOMb) (part of) a chromosome or multiple chromosomes, with each probe being represented multiple (eg. 10, 100, 1000) times to allow quantitative measurements of hybridisation signal intensities at each probe sequence.<br>
In a preferred experimental design, the 4C sequence (bait) is within about Okb, 10kb, 20kb, 30kb, 40kb, 50kb, 100kb, 200kb, 300kb, 400 kb, 500 kb, 1 Mb, 2 Mb, 3Mb, 4Mb, 5Mb, 6Mb, 7Mb, 8Mb, 9Mb 10Mb, 11Mb, 12Mb, 13Mb, 14Mb or 15Mb (eg. from about 0-15Mb) or more from the actual rearranged sequence (i.e. breakpoint in case of a translocation).<br>
In a preferred hybridization, two differentially labeled 4C templates obtained with one sequence (4C bait) from a diseased and non-diseased subject are hybridized simultaneously to the same array. Differences in.DNA-DNA interactions allow the detection of the breakpoint in cis (on the same chromosome as the 4C-bait) and in ti'ans (on the translocation partner).<br>
In a preferred hybridization, multiple differentially labeled 4C templates obtained with one sequence (4C bait) from diseased and non-diseased subjects are hybridized simultaneously to the same array. Differences in DNA-DNA interactions allow the detection of the breakpoint in cis (on the same chromosome as the 4C-bait) and in trans (on the translocation partner).<br>
Advantageously, multi-color, instead of dual color analysis on micro-arrays may be utilised allowing the simultaneous hybridization of more than two samples to a single array. Accordingly, multi-color hybridization can be used in 4C technology.<br>
hi a preferred hybridisation, multiple differentially labeled 4C templates obtained with one sequence (4C bait) from diseased subjects and one differentially labeled 4C<br>
template from a non-diseased subject are hybridised simultaneously to the same array. Differences in DNA-DNA interactions allow the detection of the breakpoint in cis (on the same chromosome as the 4C-bait) and in trans (on the translocation partner).<br>
In another preferred hybridisation, two differentially labeled 4C templates from the same non-diseased subject, obtained with two different sequences (4C-baits) that each represent another possible translocation partner, are hybridised simultaneously to the same array. Clusters of strong hybridisation signals observed on the linear template of chromosomes unrelated to the chromosome carrying the sequence of interest (4C-bait) will identify the translocation partner chromosome and the breakpoint on the translocation partner.<br>
In another preferred hybridisation, multiple differentially labeled 4C templates from the same non-diseased subject, obtained with multiple different sequences (4C-baits) that each represent another possible translocation partner, are hybridised simultaneously to the same array. Clusters of strong hybridisation signals observed on the linear template of chromosomes unrelated to the chromosome carrying the sequence of interest (4C-bait) will identify the translocation partner chromosome and its breakpoint for the sequence of interest.<br>
Material used for the detection of translocations, deletions, inversions, duplications and other genomic rearrangements by 4C technology can be obtained by cross-linking (and further processing, as described) of living cells and/or dead cells and/or nuclear lysates and/or isolated chromatin etc. (as described herein) from diseased and/or non-diseased subjects.<br>
Detection of inversions<br>
Inversions (eg. balanced inversions) cannot be detected by methods - such as Comparative Genomic Hybridization techniques - but can be detected by 4C technology particularly when the (balanced) inversion is close (eg. up to about 1-15 Mb or more) to the 4C sequence (bait).<br>
Detection of (balanced) inversions is based on identifying those DNA-DNA interactions that were different between diseased and non-diseased subjects. Inversions will change the relative position (in kilobases) on the physical DNA template of all (but the most centrally located) sequences of the rearranged region as measured against a sequence nearby on the same chromosome that is taken as 4C sequence (bait). Since DNA-DNA interaction frequencies are inversely related to genomic site separation, diseased subjects will give inversed patterns of hybridization intensities for all probes located in the rearranged genomic region, as compared to a non-diseased subject. Thus, 4C technology allows the identification of position and size of (balanced) inversions.<br>
According to mis aspect of the present invention, a preferred dedicated array design comprises probes on a single array representing genomic regions of a given size - such as about 50 kb, 100 kb, 200 kb, 300 kb, 400 kb, 500 kb, 1 Mb, 2 Mb, 3Mb, 4Mb, 5Mb, 6Mb, 6Mb, 7Mb, 8Mb, 9Mb or 10Mb) (eg. 50kb-10Mb) around the locus at which the inversion or other rearrangement is suspected.<br>
In another preferred dedicated array design, probes on a single array represent genomic regions of a given size (50 kb, 100 kb, 200 kb, 300 kb, 400 kb, 500 kb, 1 Mb, 2 Mb etc) around the locus at which the inversion or other rearrangement is suspected. For reliable quantitative analysis of signal intensities the amount of probe present on the array is typically in large excess to the amount of cognate fragments that are hybridized to the array. Therefore, it may be necessary to have each probe present multiple times (eg 10, 20, 50, 100, 1000 times etc) on the array. In addition, it may be necessary to titrate the amount of template that is to be hybridized to the array.<br>
Detection of deletions<br>
Detection of deletions is based on identifying those DNA-DNA interactions that were different between diseased and non-diseased subjects. Deletions will result in the absence of DNA mteractions with a 4C sequence (bait) located near (eg. about 1, 2, 3,<br>
4, 5, 6,7, 8, 9, 10,11,12,13,14 or 15 Mb or more) the deleted region. This may result in the complete absence of hybridization signals for all probes located in the rearranged region if the deletion is present on both alleles (homozygous), or a reduction for diseased versus non-diseased subjects of signal intensities if the deletion is present on only one allele (heterozygous). Deletion brings more distal sequences into closer proximity on the physical DNA template to the 4C sequence analyzed (bait), which will result in stronger hybridization signals for probes located directly beyond the deleted region.<br>
Detection of duplication^)<br>
Detection of duplication is typically based on identifying those DNA-DNA interactions that are different between diseased and non-diseased subjects. Probes in the duplicated region will show increased hybridization signals with a 4C sequence (bait) located near (eg. about 1, 2, 3, 4, 5, 6, 1, 8, 9, 10, 11, 12, 13, 14 or 15 Mb or more) the rearranged region, as compared to signals from a control non-diseased subject. Probes beyond the duplicated region are further apart from the 4C sequence and consequently will show decreased hybridization signals as compared to signals from a control non-diseased subject.<br>
Preferably, an increase or a decrease DNA-DNA interaction frequency for the subject sample as compared to the control is indicative of a duplication or insertion.<br>
Preferably, an increase in DNA-DNA interaction frequency for the subject sample as compared to the control and/or a reduction in DNA-DNA interaction frequency for more distant regions is indicative of a duplication or insertion.<br>
Prenatal Diagnosis<br>
Advantageously, 4C technology can also be used in prenatal diagnosis.<br>
Nucleic acid can be obtained from a fetus using various methods that are known in the .art. By way of example, amniocentesis can be used to obtain amniotic fluid from which fetal cells in suspension are extracted and cultured for several days (Mercier &amp; Bresson (1995) Ann. Gnt., 38, 151-157). Nucleic acid from the cells can be then extracted. The collection of chorial villi may make it possible to dispense with the culturing step and avoids the collection of amniotic fluid. These techniques may be applied earlier (up to 7 weeks of gestation for the collection of chorial villi and 13-14 weeks for amniocentesis), but with a slightly increased risk of abortion.<br>
A dkect collection of fetal blood at the level of the umbilical cord can also be used to obtain nucleic acid, but typically requires a team of clinicians specialised in this technique (Donner et al. (1996) Fetal Diagn. Ther., 10, 192-199).<br>
Advantageously, genetic aberrations (eg. genomic or chromosomal aberrations) - such as rearrangements, translocations, inversions, insertions, deletions and other mutations in chromosomes and nucleic acid - may be detected at this stage.<br>
Preferably, genetic aberrations (eg. genomic or chromosomal aberrations) - such as rearrangements, translocations, inversions, insertions, deletions and other mutations in chromosomes 21, 18, 13, X or Y and also losses or gains of part or whole chromosomes 21,18, 13, X or Y may be detected since these are the chromosomes in which the majority of aberrations occur in the foetus.<br>
Determination of genomic integration sites<br>
4C technology also allows the determination of genomic integration sites of viruses and transgenes, etc, also when multiple copies are inserted at different positions in the genome (as described in Figure 4).<br>
Determining predisposition to acquiring certain translocations<br>
Advantageously, 4C technology can also be applied to non-diseased subjects to measure the genomic environment of loci frequently involved in genetic aberrations. In this way, it is possible to determine the predisposition of the subject to acquire certain genetic aberrations.<br>
Thus, in addition to the medical uses described herein, the present invention can be used in diagnosis.<br>
SUBJECT<br>
The term "subject" includes mammals - such as animals and humans.<br>
AGENT<br>
The agent may be an organic compound or other chemical. The agent may be a compound, which is obtainable from or produced by any suitable source, whether natural or artificial. The agent may be an amino acid molecule, a polypeptide, or a chemical derivative thereof, or a combination thereof. The agent may even be a polynucleotide molecule - which may be a sense or an anti-sense molecule, or an antibody, for example, a polyclonal antibody, a monoclonal antibody or a monoclonal humanised antibody.<br>
Various strategies have been developed to produce monoclonal antibodies with human character, which bypasses the need for an antibody-producing human cell line. For example, useful mouse monoclonal antibodies have been "humanised" by unking rodent variable regions and human constant regions (Winter, G. and Milstein, C. (1991) Nature 349, 293-299). This reduces the human anti-mouse immunogenicity of the antibody but residual immunogenicity is retained by virtue of the foreign V-region framework. Moreover, the antigen-binding specificity is essentially that of the murine donor. CDR-grafting and framework manipulation (EP 0239400) has improved and refined antibody manipulation to the point where it is possible to produce humanised murine antibodies which are acceptable for therapeutic use in humans. Humanised<br>
mtibodies may be obtained using other methods well known in the art (for example as lescribed in US-A-239400).<br>
Fhe agents may be attached to an entity (e.g. an organic molecule) by a linker which may be a hydrolysable bifunctional linker.<br>
Ihe entity may be designed or obtained from a library of compounds, which may comprise peptides, as well as other compounds, such as small organic molecules.<br>
By way of example, the entity may be a natural substance, a biological macromolecule, or an extract made from biological materials such as bacteria, fungi, or animal (particularly mammalian) cells or tissues, an organic or an inorganic molecule, a synthetic agent, a semi-synthetic agent, a structural or functional mimetic, a peptide, a peptidorniroetics, a peptide cleaved from a whole protein, or a peptides synthesised synthetically (such as, by way of example, either using a peptide synthesizer or by recombinant techniques or combinations thereof, a recombinant agent, an antibody, a natural or a non-natural agent, a fusion protein or equivalent thereof and mutants, derivatives or combinations thereof.<br>
Typically, the entity will be an organic compound. For some instances, the organic compounds will comprise two or more hydrocarbyl groups. Here, the term "hydrocarbyl group" means a group comprising at least C and H and may optionally comprise one or more other suitable substiruents. Examples of such substituents may include halo-, alkoxy-, nitro-, an alkyl group, a cyclic group etc. In addition to the possibility of the substituents being a cyclic group, a combination of substituents may form a cyclic group. If the hydrocarbyl group comprises more than one C then those carbons need not necessarily be linked to each other. For example, at least two of the carbons may be linked via a suitable element or group. Thus, the hydrocarbyl group may contain hetero atoms. Suitable hetero atoms will be apparent to those skilled in the art and include, for instance, sulphur, nitrogen and oxygen. For some applications, preferably the entity comprises at least one cyclic group. The cyclic group may be a polycyclic group, such as a non-fused polycyclic group. For some applications, the<br>
entity comprises at least the one of said cyclic groups linked to another hydrocarbyl group.<br>
The entity may contain halo groups - such as fluoro, chloro, bromo or iodo groups.<br>
The entity may contain one or more of alkyl, alkoxy, alkenyl, alkylene and alkenylene groups — which may be unbranched- or branched-chain.<br>
PROPRUG<br>
It will be appreciated by those skilled in the art that the entity may be derived from a prodrug. Examples of prodrugs include certain protected group(s) which may not possess pharmacological activity as such, but may, hi certain instances, be administered (such as orally or parenterally) and thereafter metabolised in the body to form an. entity that is pharmacologically active.<br>
Suitable pro-drugs may include, but are not limited to, Doxorubicin, Mitomycui, Phenol Mustard, Methotraxate, Antifolates, Chloramphenicol, Camptothecin, 5-Fluorouracil, Cyanide, Quinine, Dipyridamole and Paclitaxel.<br>
It will be further appreciated that certain moieties known as "pro-moieties", for example as described in "Design of Prodrugs" by H. Bundgaard, Elsevier, 1985, may be placed on appropriate functionalities of the agents. Such prodrugs are also included within the scope of the invention.<br>
The agent may be in the form of a pharmaceutically acceptable salt - such as an acid addition salt or a. base salt - or a solvate thereof, including a hydrate thereof. For a review on suitable salts see Berge etal, J. Pharm. Sci., 1977, 66,1-19.<br>
The agent may be capable of displaying other therapeutic properties.<br>
The agent may be used in combination with one or more other phairnaceutically active agents.<br>
If combinations of active agents are administered, then the combinations of active agents may be administered simultaneously, separately or sequentially.<br>
STEREO AND GEOMETRIC ISOMERS<br>
The entity may exist as stereoisomers and/or geometric isomers - e.g. the entity may possess one or more asymmetric and/or geometric centres and so may exist in two or more stereoisomeric and/or geometric forms. The present invention contemplates the use of all the individual stereoisomers and geometric isomers of those entities, and mixtures thereof.<br>
PHARMACEUTICAL SALT<br>
The agent may be administered in the form of a pharmacetitically acceptable salt.<br>
Pharmaceutically-acceptable salts are well known to those skilled in the art, and for example, include those mentioned by Berge et al, in J.Pharm.Sci., 66, 1-19 (1977). Suitable acid addition salts are formed from acids which form non-toxic salts and include the hydrochloride, hydrobromide, hydroiodide, nitrate, sulphate, bisulphate, phosphate, hydrogenphosphate, acetate, trifluoroacetate, gluconate, lactate, salicylate, citrate, tartrate, ascorbate, succinate, maleate, fumarate, gluconate, formate, benzoate, methanesulphonate, ethanesulphonate, benzenesulphonate and p-toluenesulphonate salts.<br>
When one or more acidic moieties are present, suitable pharrnaceutically acceptable base addition salts can be formed from bases which form non-toxic salts and include the aluminium, calcium, lithium, magnesium, potassium, sodium, zinc, and pharmaceuticaHy-active amines such as diethanolamine, salts.<br>
A. pharmaceutically acceptable salt of an agent may be readily prepared by mixing together solutions of the agent and the desired acid or base, as appropriate. The salt may precipitate from solution and be collected by filtration or may be recovered by evaporation of the solvent.<br>
The agent may exist in polymorphic form.<br>
The agent may contain one or more asymmetric carbon atoms and therefore exists in two or more stereoisomeric forms. Where an agent contains an alkenyl or alkenylene group, cis (E) and trans (Z) isomerism may also occur. The present invention includes the individual stereoisomers of the agent and, where appropriate, the individual tautomeric forms thereof, together with mixtures thereof.<br>
Separation of diastereoisomers or cis and trans isomers may be achieved by conventional techniques, e.g. by fractional crystallisation, chromatography or H.P.L.C. of a stereoisomeric mixture of the agent or a suitable salt or derivative thereof. An individual enantiomer of the agent may also be prepared from a corresponding optically pure intermediate or by resolution, such as by H.PX.C. of the corresponding racemate using a suitable chiral support or by fractional crystallisation of the diastereoisomeric salts formed by reaction of the corresponding racemate with a suitable optically active acid or base, as appropriate.<br>
The agent may also include all suitable isotopic variations of the agent or a pharmaceutically acceptable salt thereof. An isotopic variation of an agent or a pharmaceutically acceptable salt thereof is defined as one in which at least one atom is replaced by an atom having the same atomic number but an atomic mass different from the atomic mass usually found in nature. Examples of isotopes that can be incorporated into the agent and pharmaceutically acceptable salts thereof include isotopes of hydrogen, carbon, nitrogen, oxygen, phosphorus, sulphur, fluorine and chlorine such as 2H, 3H, 13C, 14C, 15N, nO, 180,31P, 32P, 3SS, 18F and 36C1, respectively. Certain isotopic variations of the agent and pharmaceutically acceptable salts thereof, for example, those in which a radioactive isotope such as 3H or 14C is incorporated, are<br>
useful in drug and/or substrate tissue distribution studies. Tritiated, i.e., 3H, and carbon-14, i.e., 14C, isotopes are particularly preferred for their ease of preparation and detectability. Further, substitution with isotopes such as deuterium, i.e., 2H, may afford certain therapeutic advantages resulting from greater metabolic stability, for example, increased in vivo half-life or reduced dosage requirements and hence may be preferred in some circumstances. Isotopic variations of the agent and pharmaceutically acceptable salts thereof of this invention can generally be prepared by conventional procedures using appropriate isotopic variations of suitable reagents.<br>
PHARMACEtmCALLY ACTIVE SALT<br>
The agent may be administered as a pharmaceutically acceptable salt Typically, a pharmaceutically acceptable salt may be readily prepared by using a desired acid or base, as appropriate. The salt may precipitate from solution and be collected by filtration or may be recovered by evaporation of the solvent<br>
CHEMICAL SYNTHESIS METHODS<br>
The agent may be prepared by chemical synthesis techniques.<br>
It will be apparent to those skilled in the art that sensitive functional groups may need to be protected and deprotected during synthesis of a compound of the invention. This may be achieved by conventional techniques, for example, as described in "Protective Groups in Organic Synthesis" by T W Greene and PGM Wuts, John Wiley and Sons Inc. (1991), and by P.J.Kocienski, in "Protecting Groups", Georg Thieme Verlag (1994).<br>
It is possible during some of the reactions that any stereocentres present could, under certain conditions, be racemised, for example, if a base is used in a reaction with a substrate having an having an optical centre comprising a base-sensitive group. This is possible during e.g. a guanylation step. It should be possible to circumvent potential problems such as this by choice of reaction sequence, conditions, reagents, protection/deprotection regimes, etc. as is well-known in the art.<br>
The compounds and salts may be separated and purified by conventional methods.<br>
Separation of diastereomers may be achieved by conventional techniques, e.g. by fractional crystallisation, chromatography or H.P.L.C. of a stereoisomeric mixture of a compound of formula (I) or a suitable salt or derivative thereof. An individual enantiomer of a compound of formula 0) may also be prepared from a corresponding optically pure intermediate or by resolution, such as by H.P.L.C. of the corresponding racemate using a suitable chiral support or by fractional crystallisation of the diastereomeric salts formed by reaction of the corresponding racemate with a suitably optically active acid or base.<br>
The agent may be produced using chemical methods to synthesise the agent in whole or in part. For example, if me agent comprises a peptide, then the peptide can be synthesised by solid phase techniques, cleaved from the resin, and purified by preparative high performance liquid chromatography (e.g., Creighton (1983) Proteins Structures And Molecular Principles, WH Freeman and Co, New York NY). The composition of the synthetic peptides may be confirmed by amino acid analysis or sequencing (e.g., the Edman degradation procedure; Creighton, supra).<br>
Synthesis of peptide inhibitor agents (or variants, homologues, derivatives, fragments or mimetics thereof) can be performed using various solid-phase techniques (Roberge JY et al (1995) Science 269: 202-204) and automated synthesis may be achieved, for example, using the ABI43 1 A Peptide Synthesizer {Perkin Elmer) in accordance with the instructions provided by the manufacturer. Additionally, the amino acid sequences comprising the agent, may be altered during direct synthesis and/or combined using chemical methods with a sequence from other subunits, or any part thereof, to produce a variant agent.<br>
CHEMICAL DERIVATIVE<br>
The term "derivative" or "derivatised" as used herein includes chemical modification of an agent Illustrative of such chemical modifications would he replacement of hydrogen hy a halo group, an alkyl group, an acyl group or an amino group.<br>
CHEMICAL MODIFICATION<br>
The agent may be a modified agent - such as, but not limited to, a chemically modified agent<br>
The chemical modification of an agent may either enhance or reduce hydrogen bonding interaction, charge interaction, hydrophobic interaction, Van Der Waals interaction or dipole interaction.<br>
In one aspect, the agent may act as a model (for example, a template) for the development of other compounds.<br>
PHARMACEUTICAL COMPOSITIONS<br>
In a further aspect, there is provided a pharmaceutical composition comprising an agent identified by the assay method described herein admixed with a pharmaceutically acceptable carrier, diluent, excipient or adjuvant and/or combinations thereof.<br>
In a further aspect, there is provided a vaccine composition comprising an agent.<br>
In a further aspect, mere is provided a process of preparing a pharmaceutical composition comprising admixing an agent identified by the assay with a pharmaceutically acceptable diluent, carrier, excipient or adjuvant and/or combinations thereof.<br>
In a further aspect, there is provided a method of preventing and/or treating a disease comprising administering an agent or a pharmaceutical composition or a vaccine to a subject<br>
The pharmaceutical compositions may be for human or animal usage in human and veterinary medicine and will typically comprise any one or more of a pharmaceutically acceptable diluent, carrier, or excipient. Acceptable carriers or diluents for therapeutic use are well known in the pharmaceutical art, and are described, for example, in Remington's Pharmaceutical Sciences, Mack Publishing Co. (A. R. Gennaro edit 1985). The choice of pharmaceutical carrier, excipient or diluent can be selected with regard to the intended route of administration and standard pharmaceutical practice. The pharmaceutical compositions may comprise as - or in addition to - the carrier, excipient or diluent any suitable binder(s), lubricants), suspending agent(s), coating agent(s), solubilising agent(s).<br>
Preservatives, stabilisers, dyes and even flavouring agents may be provided in the pharmaceutical composition. Examples of preservatives include sodium benzoate, sorbic acid and esters of p-hydroxybenzoic acid. Antioxidants and suspending agents may be also used.<br>
There may be different composition/formulation requirements dependent on the different delivery systems. By way of example, the pharmaceutical composition of the present invention may be formulated to be admhiistered using a mini-pump or by a mucosal route, for example, as a nasal spray or aerosol for inhalation or ingestable solution, or parenterally in which the composition is formulated by an injectable form, for delivery, by, for example, an intravenous, intramuscular or subcutaneous route. Alternatively, the formulation may be designed to be administered by a number of routes.<br>
If the agent is to be administered mucosally through the gastrointestinal mucosa, it should be able to remain stable during transit though the gastrointestinal tract; for example, it should be resistant to proteolytic degradation, stable at acid pH and resistant to the detergent effects of bile.<br>
Where appropriate, the pharmaceutical compositions may be administered by inhalation, in the form of a suppository or pessary, topically in the form of a lotion, solution, cream, ointment or dusting powder, by use of a skin patch, orally in the form of tablets containing excipients such as starch or lactose, or in capsules or ovules either alone or in admixture with excipients, or in the form of elixirs, solutions or suspensions containing flavouring or colouring agents, or the pharmaceutical compositions can be injected parenterally, for example, intravenously, intramuscularly or subcutaneously. For parenteral administration, the compositions may be best used in the form of a sterile aqueous solution which may contain other substances, for example, enough salts or monosaccharides to make the solution isotonic with blood. For buccal or sublingual administration the compositions may be administered in the form of tablets or lozenges which can be formulated in a conventional manner.<br>
The agents may be used in combination with a cyclodextrin. Cyclodextrins are known to form inclusion and non-inclusion complexes with drug molecules. Formation of a drug-cyclodextrin complex may modify the solubility, dissolution rate, bioavailability and/or stability property of a drug molecule. Drug-eyclodextrin complexes are generally useful for most dosage forms and administration routes. As an alternative to direct complexation with the drug the cyclodextrin may be used as an auxiliary additive, e.g. as a carrier, diluent or solubiliser. Alpha-, beta- and gamma-cyclodextrins are most commonly used and suitable examples are described in WO-A-91/11172, WO-A-94/02518 and WO-A-98/55148.<br>
If the agent is a protein, then said protein may be prepared in situ in the subject being treated. In this respect, nucleotide sequences encoding said protein may be delivered by use of non-viral techniques (e.g. by use of liposomes) and/or viral techniques (e.g. by use of retroviral vectors) such that the said protein is expressed from said nucleotide sequence.<br>
The pharmaceutical compositions of the present invention may also be used in combination with conventional treatments.<br>
ADMINISTRATION<br>
The term "administered" includes delivery by viral or non-viral techniques. Viral delivery mechanisms include but are not limited to adenoviral vectors, adeno-associated viral (AAV) vectos, herpes viral vectors, retroviral vectors, lentiviral vectors, and baculoviral vectors. Non-viral delivery mechanisms include lipid mediated transfection, liposomes, immunoliposomes, lipofectin, cationic facial amphiphiles (CFAs) and combinations thereof.<br>
The components may be administered alone but will generally be administered as a pharmaceutical composition - e.g. when the components are is in admixture with a suitable pharmaceutical excipient, diluent or carrier selected with regard to the intended route of administration and standard pharmaceutical practice.<br>
For example, the components can be administered in the form of tablets, capsules, ovules, elixirs, solutions or suspensions, which may contain flavouring or colouring agents, for immediate-, delayed-, modified-, sustained-, pulsed- or controlled-release applications.<br>
If the pharmaceutical is a tablet, then the tablet may contain excipients such as microcrystalline cellulose, lactose, sodium citrate, calcium carbonate, dibasic calcium phosphate and glycine, disrntegrants such as starch (preferably corn, potato or tapioca starch), sodium starch glycollate, croscarmellose sodium and certain complex silicates, and granulation binders such as polyvinylpyrrolidone, hydroxypropylmethylcellulose (HPMC), hydroxypropylcellulose (HPC), sucrose, gelatin and acacia. Additionally, lubricating agents such as magnesium stearate, stearic acid, glyceryl behenate and talc may be included<br>
Solid compositions of a similar type may also be employed as fillers in gelatin capsules. Preferred excipients in this regard include lactose, starch, a cellulose, milk sugar or high molecular weight polyethylene glycols. For aqueous suspensions and/or elixirs, the agent may be combined with various sweetening or flavouring agents,<br>
colouring matter or dyes, with emulsifying and/or suspending agents and with diluents such as water, ethanol, propylene glycol and glycerin, and combinations thereof.<br>
The routes for administration (delivery) may include, but are not limited to, one or more<br>
of oral (e.g. as a tablet, capsule, or as an ingestable solution), topical, mucosal (e.g. as<br>
a nasal spray or aerosol for inhalation), nasal, parenteral (e.g. by an injectable form),<br>
gastrointestinal, intraspinal, intraperitoneal, intramuscular, intravenous, intrauterine,<br>
intraocular,	intradermal,	intracranial,	intratracheal,	intravaginal,<br>
intracerebroventricular, intracerebral, subcutaneous, ophthalmic (including intravitreal or intracameral), transdermal, rectal, buccal, vaginal, epidural, sublingual.<br>
DOSE LEVELS<br>
Typically, a physician will determine the actual dosage which will be most suitable for an individual subject. The specific dose level and frequency of dosage for any particular patient may be varied and will depend upon a variety of factors including the activity of the specific compound employed, the metabolic stability and length of action of that compound, the age, body weight, general health, sex, diet, mode and time of administration, rate of excretion, drug combination, the severity of the particular condition, and the individual undergoing therapy.<br>
FORMULATION<br>
The components) may be formulated into a pharmaceutical composition, such as by mixing with one or more of a suitable carrier, diluent or excipient, by using techniques that are known in the art.<br>
DISEASE<br>
Aspects of the present invention may be used for the treatment and/or prevention and/or diagnosis and/or prognosis of a disease - such as those listed in WO-A-98/09985.<br>
For ease of reference, part of (hat list is now provided; macrophage inhibitory and/or T cell inhibitory activity and thus, anti-inflammatory activity; anti-immune activity, i.e. inhibitory effects against a cellular and/or humoral immune response, including a response not associated with inflammation; diseases associated with viruses and/or other intracellular pathogens; inhibit the ability of macrophages and T cells to adhere to extracellular matrix components and fibronectin, as well as up-regulated fas receptor expression in T cells; inhibit unwanted immune reaction and inflammation including arthritis, including rheumatoid arthritis, inflammation associated with hypersensitivity, allergic reactions, asthma, systemic lupus erythematosus, collagen diseases and other autoimmune diseases, inflammation associated with atherosclerosis, arteriosclerosis,   atherosclerotic heart  disease,  reperfusion injury,  cardiac  arrest, myocardial infarction, vascular inflammatory disorders, respiratory distress syndrome or other cardiopulmonary diseases,  inflammation associated with peptic  ulcer, ulcerative colitis and other diseases of the gastrointestinal tract, hepatic fibrosis, liver cirrhosis   or   other   hepatic   diseases,   thyroiditis   or   other   glandular   diseases, glomerulonephritis or other renal and urologjc diseases, otitis or other oto-rhino-laryngological diseases, dermatitis or other dermal diseases, periodontal diseases or other dental diseases, orchitis or epididimo-orchitis, infertility, orchidal trauma or other    immune-related    testicular    diseases,    placental    dysfunction,    placental insufficiency, habitual abortion, eclampsia, pre-eclampsia and other immune and/or inflammatory-related gynaecological diseases, posterior uveitis, intermediate uveitis, anterior uveitis, conjunctivitis, chorioretinitis, uveoretinitis, optic neuritis, intraocular inflammation, e.g. retinitis or cystoid macular oedema, sympathetic ophthalmia, scleritis, retinitis pigmentosa, immune and inflammatory components of degenerative fondus disease, inflammatory components of ocular trauma, ocular inflammation caused   by   infection,   proliferative   vitreo-retinopathies,   acute   ischaemic   optic neuropathy, excessive scarring, e.g. following glaucoma filtration operation, immune and/or  inflammation reaction  against ocular implants  and other immune  and inflammatory-rekted ophthalmic diseases, inflammation associated with autoimmune diseases or conditions or disorders where, both in the central nervous system (CNS) or in any other organ, irnmune and/or hiflammation suppression would be beneficial,<br>
Parkinson's disease, complication and/or side effects from treatment of Parkinson's disease, AIDS-related dementia complex HTV-related encephalopathy, Devic's disease, Sydenham chorea, Alzheimer's disease and other degenerative diseases, conditions or disorders of the CNS, inflammatory components of stokes, post-polio syndrome, immune and inflammatory components of psychiatric disorders, myelitis, encephalitis, subacute sclerosing pan-encephalitis, encephalomyelitis, acute neuropathy, subacute neuropathy,   chronic   neuropathy,   Guillaim-Barre   syndrome,   Sydenham   chora, myasthenia gravis, pseudo-tumour cerebri, Down's Syndrome, Huntington's disease, amyotrophic lateral sclerosis, inflammatory components of CNS compression or CMS trauma or infections of the CNS, inflammatory components of muscular atrophies and dystrophies, and immune and inflammatory related diseases, conditions or disorders of the central and peripheral nervous systems, post-traumatic inflammation, septic shock, infectious diseases, inflammatory complications or side effects of surgery, bone marrow transplantation or other transplantation complications and/or side effects, inflammatory and/or immune complications and side effects of gene therapy, e.g. due to infection with a viral carrier, or inflammation associated with AIDS, to suppress or inhibit a humoral and/or cellular immune response, to treat or ameliorate monocyte or leukocyte proliferative diseases, e.g. leukaemia, by reducing the amount of monocytes or lymphocytes, for the prevention and/or treatment of graft rejection in cases of transplantation of natural or artificial cells, tissue and organs such as cornea, bone marrow, organs, lenses, pacemakers, natural or artificial skin tissue.   Specific cancer related disorders include but not limited to: solid tumours; blood born tumours such as leukemias; tumor metastasis; benign tumours, for example hemangiomas, acoustic neuromas, neurofibromas, trachomas, and pyogenic granulomas; rheumatoid arthritis; psoriasis; ocular angiogenic diseases, for example, diabetic retinopathy, retinopathy of prematurity, macular degeneration, corneal graft rejection, neovascular glaucoma, retrolental fibroplasia, rubeosis; Osier-Webber Syndrome; myocardial angiogenesis; plaque neovascularization; telangiectasia;   hemophiliac joints; angiofibroma; wound granulation; coromay collaterals; cerebral collaterals; arteriovenous malformations; ischeniic limb angiogenesis; neovascular glaucoma; retrolental fibroplasia; diabetic neovascularization;    heliobacter    related    diseases,    fractures,    vasculogenesis, hematopoiesis, ovulation, menstruation and placentation.<br>
Preferably, the disease is cancer - such as acute lymphocytic leukemia (ALL), acute myeloid leukemia (AML), adrenocortical cancer, anal cancer, bladder cancer, blood cancer, bone cancer, brain tumor, breast cancer, cancer of me female genital system, cancer of the male genital system, central nervous system lymphoma, cervical cancer, childhood rhabdomyosarcoma, childhood sarcoma, chronic lymphocytic leukemia (CLL), chronic myeloid leukemia (CML), colon and rectal cancer, colon cancer, endometrial cancer, endometrial sarcoma, esophageal cancer, eye cancer, gallbladder cancer, gastric cancer, gastrointestinal tract cancer, hairy cell leukemia, head and neck cancer, hepatocellular cancer, Hodgkin's disease, hypopharyngeal cancer, Kaposi's sarcoma, kidney cancer, laryngeal cancer, leukemia, liver cancer, lung cancer, malignant fibrous histiocytoma, malignant thymoma, melanoma, mesothelioma, multiple myeloma, myeloma, nasal cavity and paranasal sinus cancer, nasopharyngeal cancer, nervous system cancer, neuroblastoma, non-Hodgkin's lymphoma, oral cavity cancer, oropharyngeal cancer, osteosarcoma, ovarian cancer, pancreatic cancer, parathyroid cancer, penile cancer, pharyngeal cancer, pituitary tumor, plasma cell neoplasm. primary-CNS lymphomaj prostate cancer, rectal cancer, respiratory system, retinoblastoma, salivary gland cancer, skin cancer, small intestine cancer, soft tissue sarcoma, stomach cancer, stomach cancer, testicular cancer, thyroid cancer, urinary system cancer, uterine sarcoma, vaginal cancer, vascular system, Waldenstrom's macroglobulinemia and Wilms' tumor.<br>
KITS<br>
The materials for use in the methods of the present invention are ideally suited for preparation of kits.<br>
Such a kit may comprise containers, each with one or more of the various reagents (typically in concentrated form) utilised in the methods described herein, including, for example, a primary restriction enzyme, a secondary restriction enzyme, a cross-linking agent, a ligation enzyme (eg. a ligase) and an agent to reverse the cross-Unking (eg. proteinase K).<br>
Oligonucleotid.es may also be provided in containers which can be in any form, e.g., lyophilized, or in solution (e.g., a distilled water or buffered solution), etc.<br>
In a preferred aspect of the present invention, there is provided a kit comprising a set of probes as described herein, an array and optionally one or more labels.<br>
A set of instructions will also typically be included.<br>
Advantageously, the present invention can be used in order to obtain information about the spatial organisation of nucleotide sequences - such as genomic loci in vitro or in vivo.<br>
By way of example, 4C technology can be used to study the three dimensional organisation of one or more gene loci.. In particular, this technology can be used to study the role of one or more transcription factors in the three dimensional organisation of one or more gene loci.<br>
By way of further example, 4C technology can be used to study the role of transacting factors and czs-regulatory DNA elements.<br>
By way of further example, 4C technology can be used to study long range gene regulation in vitro or in vivo.<br>
By way of further example, 4C technology can be used to study intra-chromosomal proximity and interaction.<br>
By way of further example, 4C technology can be used to study inter-chromosomal proximity and interaction.<br>
By way of further example, 4C technology can be used to identify nucleotide sequences that function with a promoter, enhancer, silencer, insulator, locus control region, origin of replication, MAR, SAR, centromere, telomere or any other sequence of interest in a regulatory network.<br>
By way of further example, 4C technology can be used to identify genes responsible for a phenotype (disease) in cases where a mutation and/or deletion happens to affect a distant regulatory element and their mapping therefore fails to provide such information.<br>
By way of further example, 4C technology can be used to eventually reconstruct the spatial conformation of gene loci, large genomic regions or even complete chromosomes.<br>
By way of further example, 4C technology can be used to define potential anchor sequences that keep certain chromosomes together in the nuclear space.<br>
By way of further example, 4C technology can be used to eventually reconstruct at high resolution the positioning of chromosomes with respect to each other.<br>
By way of further example, 4C technology can be used in diagnosis (eg. prenatal diagnosis) to detect or identify genomic rearrangements and/or aberrations - such as translocations, deletions, inversions, duplications.<br>
GENERAL RECOMBINANT DNA METHODOLOGY TECHNIQUES<br>
The present invention employs, unless otherwise indicated, conventional techniques of chemistry, molecular biology, microbiology, recombinant DNA and immunology, which are within the capabilities of a person of ordinary skill in the art Such techniques are explained in the literature. See, for example, J. Sambrook, E. F. Fritsch, and T. Maniatis, 1989, Molecular Cloning: A Laboratory Manual, Second Edition, Books 1-3, Cold Spring Harbor Laboratory Press; Ausubel, F. M. et al. (1995<br>
and periodic supplements; Current Protocols in Molecular Biology, ch. 9, 13, and 16, John Wiley &amp; Sons, New York, N.Y.); B. Roe, J. Crabtree, and A. Kahn, 1996, DNA Isolation and Sequencing: Essential Techniques, John Wiley &amp; Sons; M. J. Gait (Editor), 1984, Oligonucleotide Synthesis: A Practical Approach, M Press; and, D. M. J. Lilley and J. E. Dahlberg, 1992, Methods of Enzymology: DNA Structure Part A: Synthesis and Physical Analysis of DNA Methods in Enzymology, Academic Press. Each of these general texts is herein incorporated by reference.<br>
The invention will now be further described by way of Example, which are meant to serve to assist one of ordinary skill in the art in carrying out the invention and are not intended hi any way to limit the scope of the invention.<br>
EXAMPLE 1<br>
Materials &amp; Methods<br>
4C technology<br>
The initial steps of the 3C technology procedure were performed as described previously (Splinter et al. (2004). Methods Enzymol 375, 493-507 (2004), yielding iigation-products- between HindDI-fi:agments-This--Hindni-ligated -3G- template-(~50 -ug) was digested overnight at 100ng/|jl with 50U of a secondary, frequent cutting, restriction enzyme, being either DpnII (HS2, Rad23A) or Nlain (|3-major). To avoid constraints in DNA circle formation (Rippe et al. (1995) Trends Biochem Sci 20, 500-6), care was taken to choose a secondary restriction enzyme that did not cut within about 350-400 bp from the Hindm restriction site that demarcates the restriction fragment of interest (Le. the 'bait'). After secondary restriction enzyme digestion, DNA was phenol extracted, ethanol precipitated and subsequently ligated at low concentration (50 (ig sample hi 14 ml using 200 U ligase (Roche), 4 hours at 16°C) to promote DpnII- or DpnU-circle formation. Ligation products were phenol extracted and ethanol precipitated, using glycogen (Roche) as a carrier (20|ig/ml). The circles of interest were linearised by digesting overnight with a SOU of a tertiary restriction enzyme that cuts the bait in between the primary and secondary restriction eruzyme<br>
recognition sites, using the following restriction enzymes: Spel (HS2), Psfl (Rad23A) and Pflml (p-major). This linearisation step was performed to facilitate subsequent primer hybridization during the first rounds of PCR amplification. Digested products were purified using a QIAquick nucleotide removal (250) column (Qiagen).<br>
PCR reactions were performed using the Expand Long Template PCR system (Roche), using conditions carefully optimized to assure linear amplification of fragments sized up to 1.2 kb (80% of 4C-PCR fragments are smaller than 600 bp). PCR conditions were as follows: 94°C for 2 minutes, 30 cycles of 94°C for 15 seconds, 55°C for 1 minute and 68°C for 3 minutes, followed by a final step of 68°C for 7 minutes. The maximum amount of template that still shows linear range of amplification was determined. For this, serial dilutions of template were added to PCR reactions, amplified DNA material was run out on an agarose gel and PCR products were quantified using ImageQuant software. Typically, 100-200 ng of template per 50 \i\ PCR reaction gave products in the linear range of amplification. 16 to 32 PCR reactions were pooled and purified this 4C template using the QIAquick nucleotide removal (250) system (Qiagen). Purified 4C template was labeled and hybridized to arrays according to standard CbJP-chip protocols (Nimblegen Systems of Iceland, LLC). Differentially labeled genomic DNAj which was digested with the primary and secondary enzyme used in the 4C procedure, served as a control template to correct for differences in hybridisation efficiencies. For each experiment two independently processed samples were labeled with alternate dye orientations.<br>
4C-Primer-sequences used:<br>
4C arrays<br>
Arrays and analysis were based on NCBI build m34. Probes (60-mers) were selected from the sequences 100 bp up -and downstream of HindHI sites. The CG-content was optimized towards 50%, for uniform hybridization signals. To prevent cross-hybridization, probes that had any similarity with highly abundant repeats (RepBase 10.09)3 were removed from the probe set. In addition, probes that gave more than two BLAST hits in the genome were also removed from the probe set. Sequence alignments were performed using MegaBLAST (Zhang et al. (2000) J Comput Biol 1, 203-14) using the standard settings. A hit was defined as an alignment of 30 nt or longer.<br>
(Sequence Removed)<br>
4C data analysis<br>
The signal ratio 4C-sample/genomic DNA was calculated for each probe and the data was visualized with SignalMap software provided by Nimblegen Systems. Data were analyzed using the R package (http://www.r-proiect.org). Spotfire and Excel. ' Unprocessed hybridization~ratios~showed~clusters~ofL-20-50 positive-46=signals-along the chromosome template. To define these clusters, a running mean was applied. Various window sizes were used, ranging from 9-39 probes, which all identified the same clusters. Results shown were based on a window size of 29 probes (on average 60 kb) and were compared to the running mean performed across randomized data. This was done for each array separately. Consequently, all measurements were appreciated relative to the amplitude and noise of that specific array. The False Discovery Rate (FDR), defined as (no. false positives) / (no. of false positives + no. of true positives) was determined as follows: (number of positives in the randomised set) / (number of positives in the data). The threshold level was determined using a top down approach to establish the minimal value for which: FDRO.05.<br>
Next, biological duplicate experiments were compared, Windows that met the threshold in both duplicates were considered positive. When comparing randomized data, no windows were above threshold in both duplicates. Positive windows directly adjacent on the chromosome template were joined (no gaps allowed), creating positive areas.<br>
Expression analysis<br>
For each tissue, three independent microarrays were performed according to Affymetrix protocol (mouse 430_2 arrays). Data were normalized using RMA ca-tools; www.bioconductor.org) and for each probe-set the measurements of the three microarrays were averaged. In addition, when multiple probe-sets represented the same gene, they were also averaged. MasScalls (Affy library: www.bioconductor.org) was used to establish "present", "absent" and "marginal" calls. Genes with a "present" call in all three arrays and an expression value bigger than 50 were called expressed. 'Fetal liver-specific genes' were classified as genes that met our criteria of being expressed in fetal liver and had more than five times higher expression values compared to fetal brain. To provide a measure of overall transcriptional activity around each gene, a running sum was applied. For this, we used log-transformed expression values. For each gene we calculated the sum of the expression of all genes found hi a window 100 kb upstream of the start and 100 kb downstream of the end of the gene, including the gene itself. Resulting values for active genes found inside positive 4C regions (n = 124, 123 and 208 respectively for HS2 in liver, Rad23A in brain and Rad23A in liver) were compared to the values obtained for active genes outside positive 4C areas (n = 153, 301 and 186, respectively, where n=153 corresponds to the number of active, non-interacting, genes present between the most centromeric interacting region and the telomere of chromosome 7); the two groups were compared using a one tailed Wilcoxon rank sum test.<br>
FISH probes<br>
The following BAG clones (BACPAC Resources Centre) were used; RP23-370E12 for Hbb-1, RP23-317H16 for chr.Vat 80.1Mb (OR gene cluster), RP23-334E9 for Uros, RP23-32C19 for chr.7 at 118.3 Mb, RP23-143F10 for chr.7 at 130.1Mb, RP23-470N5<br>
for chr.7 at 73.1Mb, RP23-247L11 for chr.7 at 135.0Mb (OR gene cluster), RP23-136A15 for Rad23A5 RP23-307P24 for chr.8 at 21.8 Mb and RP23-460F21 for chr.8 at 122.4 Mb. For a chromosome 7 centromere specific probe we used PI clone 5279 (Genome Systems Inc.) that anneals to DNA segment D7Mit21. Random prime labeled probes were prepared using BioPrime Array CGH Genomic Labeling System (Tnvitrogen). Prior to labeling, DNA was digested with Dpnll and purified with a DNA clean and concentrator-5 kit (Zymo research). Digested DNA (300 ng) was labeled with SpectrumGreen dUTP (Vysis) or Alexa fluor 594 dUTP (Molecular probes) and purified through a GFX PCR DNA and Gel Band Purification kit (Amersham Biosciences) to remove unincorporated nucleotides. Specificity of labeled probes was tested on metaphase spreads prepared from murine ES cells.<br>
Crvo-FISH<br>
Cryo-FISH was performed as described before 5. Briefly, E14.5 liver and brain, were fixed for 20 min. in 4% paraformaldehyde/250 mM HEPES, pH 7.5 and cut into small tissue blocks, followed by another fixation step of 2 hrs in 8% paraformaldehyde at 4°C. Fixed tissue blocks were immersed in 2.3 M sucrose for 20 min at room temperature, mounted on a specimen holder and snap-frozen hi liquid nitrogen. Tissue blocks were stored in liquid nitrogen until sectioning. Ultrathin cryosections of approximately 200 run were cut using an Reichert Ultramicrotome E equipped with cryo-attachment (Leica). Using a loop filled with sucrose, sections were transferred to coverslips and stored at -20°C. For hybridization, sections were washed with PBS to remove sucrose, treated with 250 ng/ml RNase in 2xSSC for 1 hr at 37°C, incubated for 10 min in 0.1 M HCL, dehydrated in a series of ethanol and denatured for 8 min at 80°C in 70% formamide/2xSSC, pH 7.5. Sections were again dehydrated directly prior to probe hybridization. 500 ng labeled probe was co-precipitated with 5 ng of mouse Cotl DNA (Thvitrogen) and dissolved in hybmix (50% formamide, 10% dextran sulfate, 2xSSC, 50 mM phosphate buffer, pH 7.5). Probes were denatured for 5 min at 95°C, reannealed for 30 min at 37°C and hybridized for at least 40 hrs at 37°C. After posthybridization washes, nuclei were counterstained with 20 ng/ml DAPI (Sigma) in PBS/0.05% Tween-20 and mounted in Prolong Gold antifade reagent (Molecular Probes).<br>
Images were collected with a Zeiss Axio Imager Zl epifluorescence microscope (xlOO plan apochrornat, 1.4 oil objective), equipped with a CCD camera and Isis FISH Imaging System software (Metasystems). A minimum of 250 p-globin or Rad23A alleles was analyzed and scored as overlapping or non-overlapping with BACs located elsewhere in the genome, by a person not knowing the probe combination applied to the sections. Replicated goodness-of-fit tests (G-statistic) 6 were performed to assess significance of differences between values measured for 4C-positive versus 4C negative regions. Overview of the results is provided in Table 2.<br>
Although we found statistically significant differences between background (0.4-3.9%) and true (5-20.4%) interaction frequencies, it may be clear that frequencies measured by cryo-FISH are lower than those measured by others using different FISH protocols. Sectioning may separate some interacting loci and cryo-FISH measurements will therefore slightly underestimate true interaction frequencies. On the other hand, current 2D- and 3D FISH procedures will overestimate these percentages due to limited resolution in the z-direction. La the future, improved microscopy techniques in combination with more specific FISH probes will better reveal true interaction frequencies.<br>
EXAMPLE2<br>
The 3C procedure (i.e. formaldehyde fixation, (primary) restriction enzyme digestion, re-ligation of cross-linked DNA fragments and DNA purification) is carried out essentially as described (Splinter et al, (2004) Methods Enzymol. 375: 493-507), yielding a DNA mixture ('3C template') containing restriction fragments that are ligated because they were originally close in the nuclear space.<br>
Inverse PCR is performed to amplify all fragments ligated to a given restriction fragment ('bait'; chosen because it contains a promoter, enhancer, insulator, matrix attachment region, origin of replication or any other first (target) nucleotide sequence).<br>
For this, DNA circles are created by digesting the 3C template with a secondary restriction enzyme (preferably a frequent cutter recognizing tetra- or penta-nucleotide sequences), followed by ligation under dilute conditions such that intra-molecular interactions are favoured. To minimise a bias in circle formation due to topological constraints (Rippe et al, (2001) Trends in Biochem. Sciences 26, 733-40), a secondary restriction enzyme should be chosen that preferably cuts the bait at &gt;350-400bp from the primary restriction site. To increase inverse PCR amplification efficiency and reproducibility, circles are best linearised before PCR amplification by a restriction enzyme (eg. a 6 or more bp cutter) that cuts the bait between the diagnostic primary and secondary restriction site.<br>
Digestion of the 3C template with the secondary restriction enzyme, circularisation through ligation under diluted conditions and linearisation of bait-containing circles are performed under conditions standard for such DNA manipulations to yield a DNA template for inverse PCR amplification ('4C template').<br>
Accordingly, 10 (ig of 3C template is digested in 100 p.1 with 20U of the secondary restriction enzyme (overnight), followed by heat-inactivation of the enzyme and DNA purification. Ligation is performed in 10 ml (1 ng/(il DNA) with SOU T4 ligase (4 hrs at 16°C, 30 min at RT), followed by DNA purification. Finally, linearisation of the circles of .interest is done in 100 ul with 20U of restriction enzyme (overnight), followed again by DNA purification.<br>
For inverse PCR, two bait-specific primers are designed, each as close as possible to the primary and directly neighbouring secondary restriction enzyme recognition site, respectively, and each with its 3'end facing outwards so that extension proceeds immediately across the restriction sites into a fragment ligated to the bait. Inverse PCR with these primers is preferably carried out on 100-400 ng DNA of 4C template (per 50 fil PCR reaction mix), to include a maximum number of ligation events per PCR reaction. We perform inverse PCR applying the Expand Long Template PCR System (Roche), using buffer 1 according to manufacturer's procedures.<br>
The following PCR cycles are performed:<br>
1.2min94°C<br>
2.	15 sec 94°C<br>
3.	lmin55°C<br>
4.3min68°C<br><br>
5.	repeat step 2-4 29x (or anything between 25-40x)<br>
6.	7 min 68°C<br>
7.	end<br>
Gel electrophoresis is performed to analyse reproducibility between individual PCR reactions. Typically, identical product patterns should be obtained.<br>
In order to obtain sufficient material for labelling by random priming and array hybridisation, multiple PCR reactions (each obtained after 30 cycles of PCR) can be pooled, (instead of increasing the number of PCR cycles per reaction). As an alternative for random primed labelling, labelled nucleotides can be incorporated in the last cycles of PCR (e.g. 30 cycles (no label) + 10 cycles (label)).<br>
EXAMPLE3<br>
Detection oftranslocation using 4C technology<br>
4C technology is used to measure the interaction frequencies for a given sequence X present on a given chromosome A in cells from a healthy subject and in cells from a patient carrying a single, reciprocal, translocation between chromosome A and B with the breakpoint being close to sequence X (as shown in Figure 9).<br>
In normal cells this analysis reveals elevated hybridization signals (z.e. frequent interactions with X) for (almost) every probe located within 0.2-10Mb of sequence X on chromosome A (the actual size of the chromosomal region showing strong cross-linking signals depends mostly on the complexity of the sample that was hybridized to<br>
the array). Elsewhere on the same chromosome A, as well as on other chromosomes, no such large region (on the linear DNA template) of probes wiih elevated hybridization signals is observed.<br>
In patient cells however, hybridization signals with all chromosome A probes located<br>
i<br>
on the other side of the breakpoint are reduced by ~50% (one copy of chromosome A is still intact and will produce normal signals), while a unique (i.e. not present in normal cells) concentration of elevated hybridization signals is observed for probes bordering the breakpoint on chromosome B. In fact, the abrupt transition between probes showing no versus strong hybridization signals on chromosome B reveals the location of the breakpoint on chromosome B.<br>
EXAMPLE 4<br>
Analysis of4C technology results<br>
4C technology was used to characterise the genomic environment of the mouse fi-globin locus control region (LCR), focusing on a restriction fragment containing its hypersensitive site 2 (HS2). The LCR is a strong erythroid-specific transcription regulatory element required for high levels of p-globin gene expression. The f}-globin locus is present on chromosome 7 at position 97 Mb, where it resides in a large, 2.9 Mb, cluster of olfactory receptor genes that are transcribed only in olfactory neurons. Interactions were analysed in two tissues: E14.5 fetal liver, where the LCR is active and the |3-globin genes are transcribed highly, and E14.5 fetal brain, where the LCR is inactive and the globin genes are silent. In both tissues, the great majority of interactions were found with sequences on chromosome 7 and very few LCR interactions were detected with six unrelated chromosomes (8, 10, 11, 12, 13, 14) (Figure I3a). The strongest signals on chromosome 7 were found within a 5-10 Mb region centered around the chromosomal position of P-globin, in agreement with the idea that interaction frequencies are inversely proportional to the distance (hi basepairs) between physically linked DNA sequences. It was not possible to interpret the interactions in this region quantitatively. We reasoned that these nearby sequences<br>
were together with P-globin so frequently that their large overrepresentation in our hybridisation samples saturated the corresponding probes. This was confirmed when we performed hybridisations with samples diluted 1:10 and 1:100 and found that signal intensity was reduced at probes outside and at the edge, but not inside this region (data not shown).<br>
The 4C procedure yielded highly reproducible data. Figure 2b-c shows unprocessed ratios of 4C-signals over control hybridisation signals for two 1.5 Mb regions on chromosome 7, roughly 25 Mb and 80 Mb away from the P-globin gene. At mis level of resolution the results from independently processed samples were almost identical. Both in fetal liver and in brain, clusters of positive signals were identified on chromosome 7, often at chromosomal locations tens of megabases away from |3-globin. These clusters typically consisted of minimally 20-50 probes with increased signal ratios juxtaposed on the chromosome template (Figure 13b-c). Each probe on the array analyses an independent ligation event. Moreover, only two copies of the HS2 restriction fragment are present per cell, each of which can only ligate to one other restriction fragment. Therefore, the detection of independent ligation events with 20 or more neighbouring restriction fragments strongly indicates that the corresponding locus contacts the P-globin LCR in multiple cells.<br>
To determine the statistical significance of these clusters, data of individual experiments were ordered on chromosomal maps and analysed using a running mean algorithm with a window size of approximately 60 kb. The running mean distribution of randomly shuffled data was used to set a threshold value, allowing a false discovery rate of 5%. This analysis identified 66 clusters in foetal liver and 45 in brain that reproducibly were found in duplicate experiments (Figure, 13d-f). Indeed, high resolution FISH confirmed that such clusters truly represent loci that interact frequently (see below).<br>
Thus, 4C technology identifies long-range interacting loci by the detection of independent ligation events with multiple restriction fragments clustered at a chromosomal position.<br>
A completely independent series of 4C experiments was performed with a different inverse PCR primerset that investigated the genomic environment of the p major gene, located ~50 kb downstream of HS2.'In foetal liver, the (3 major gene is highly transcribed and frequently contacted by the LCR. Almost identical clusters of long-range interactions with |3 major as with HS2 were found, both m foetal liver and in brain, further substantiating that these loci frequently contact the (3-gIobin locus (Figure 17).<br>
EXAMPLES<br>
The active and inactive P-globin locus occupy distinct genomic environments.<br>
A comparison between the two tissues revealed that the actively transcribed (5-globin locus in foetal liver interacts with a completely different set of loci than its transcriptionally silent counterpart in brain (T=-0.03; Spearman's Rank correlation) (Figure 13f). This excluded that results were influenced by the sequence composition of the probes. In foetal liver, the interacting DNA segments were located within a 70 Mb region centred around the P-globin locus, with the majority (40/66) located towards the telomere of chromosome 7. In foetal brain, interacting loci were found at similar or even larger distances from p-globin compared to foetal liver and with the great majority of interactions (43/45) located towards the centromere of chromosome 7. These data demonstrated that the active and inactive P-globin locus contact different parts of chromosome 7.<br>
Six other chromosomes (8, 10, 11, 12, 13 and 14) were represented on the micro-arrays. Strong hybridisation signals on these chromosomes were rare, typically appeared isolated on the linear DNA template and often were absent from duplicate experiments. Also, running mean levels across these chromosomes never reproducibly came close to the levels scored for chromosome 7 (Figure 19). Thus, our data showed that the p-globin locus mostly contacted loci elsewhere on the same chromosome, in<br>
agreement with the preferred location of this locus inside its own chromosome territory. We note that the a-globin locus was also present on the array (chromosome 11) and did not score positive for interaction with J3-globin, in agreement with the recent demonstration by FISH that mouse a- and P-globin do not frequently meet in the nuclear space (Brown, J. M. et d. (2006) J Cell Biol 172, 177-87).<br>
hi order to better understand the relevance of the observed long-range interactions on chromosome 7, we compared the interacting loci to the chromosomal positions of genes, hi addition, Affymetrix expression array analysis was performed to determine transcription activity at these positions in the two tissues. Although the average size of interacting areas in foetal liver and brain was comparable (183 kb and 159 kb, respectively), dramatic differences were observed in their gene content and activity. In foetal liver, 80% of the P-globin interacting loci contained one or more actively transcribed genes, while in foetal brain the great majority (87%) showed no detectable gene activity (Figure 15). Thus, the P-globin locus is embedded in a very different genomic environment in the two tissues. In brain, where the locus is not active, it primarily contacts transcriptional silent loci located towards the centromere of chromosome 7. In foetal liver, where the locus is highly active, it interacts preferentially with actively transcribed regions located more prominently towards the telomeric side of chromosome 7. Importantly, 4C technology identified both Uros and Eraf, (~30Mb away from P-globin) as genes interacting with the active p-globin locus in fetal liver, in agreement with previous observations made by FISH (Osborne, C. S. et al. (2004) Nat Genet 36, 1065-71 (2004)). Interestingly, in brain contacts were observed with the two other olfactory receptor gene clusters present on chromosome 7 that were located at each side of, and 17 and 37 Mb away from, P-globin.<br>
Not all transcribed regions on chromosome 7 interact with the active P-globin locus in foetal liver. Therefore, we searched for a denominator shared exclusively by the interacting loci but not by other active regions in fetal liver. The P-globin genes, Uros and Eraf are all erythroid-specific genes that may be regulated by the same set of transcription factors, and it is an attractive idea that these factors co-ordinate the<br>
expression of their target genes in the nuclear space. We compared Afrymetrix expression array data from E14.5 foetal liver with, that of foetal brain to identify genes expressed preferentially (5-fold more) in foetal liver. As such, 28% of the active genes on chromosome 7 were classified as "foetal liver-specific", of which 25% were found in a co-localising area. Thus, we found no enrichment of "foetal liver-specific" genes' in the co-localising areas. More importantly, 49 out of 66 (74%) interacting regions did not contain a "foetal liver-specific" and it is therefore concluded that our data showed no evidence for co-ordinate expression of tissue-specific genes in the nuclear space. The p-globin genes are transcribed at exceptional high rates and it was next asked whether the locus preferentially interacted with other regions of high transcriptional activity, being either highly expressed genes or areas with a high density of active genes. Using Afrymetrix counts as a measure for gene activity, we performed a running sum algorithm to measure overall transcriptional activity within 200 kb regions around actively transcribed genes. This analysis revealed that transcriptional activity around interacting genes was not higher than around non-interacting active genes on chromosome 7 (p = 0.9867; Wilcoxon Rank sum).<br>
EXAMPLE 6<br>
The genomic environment of a housekeeping gene is largely conserved between the tissues<br>
It was next investigated whether a gene that is expressed similarly in both tissues also switches its genomic environment. Rad23A is a ubiquitously expressed gene that resides in a gene-dense cluster of mostly housekeeping genes on chromosome 8. Both in E14.5 foetal liver and in brain, this gene and many of its direct neighbours are active. 4C analysis was performed and identified many long-range interactions with loci up to 70 Mb away from Rad23A. Importantly, interactions with Rad23A were highly correlated between foetal liver and brain (T=0.73; Spearman's Rank correlation) (Figure 15a). A shared hallmark of these loci was again that they contained actively transcribed genes. Thus, in both tissues roughly 70% contained at least one active gene (Figure 15b-c). Regions around interacting genes displayed statistically<br>
significant higher levels of gene activity compared to active genes elsewhere on the chromosome, as determined by a running sum algorithm (p 
EXAMPLE?<br>
Validation of4C technology by high-resolution microscopy<br>
To validate the results obtained by 4C technology, cryo-FISH experiments were performed. Cryo-FISH is a recently developed microscopy technique, which has the advantage over current 3D-FISH protocols that it better preserves the nuclear ultra-structure while offering improved resolution in the z-axis by the preparation of ultra-thin cryo-sections (Branco, M. R. &amp; Pombo, A (2006). PLoS Biol 4, e!38). 4C data were verified by measuring how frequent (3-globin or Rad23A alleles (always n&gt;250) co-localised with more than 15 selected chromosomal regions hi 200 ran ultra-thin sections prepared from E14.5 liver and brain. Importantly, all interaction frequencies measured by cryo-FISH were in perfect agreement with the 4C results (Figure 17). For example, distant regions that were identified to interact with (i-globm by 4C technology co-localised more frequently than intervening areas not detected by 4C (7.4% and 9.7%, versus 3.6% and 3.5%, respectively). Also, the two distant olfactory receptor gene clusters identified by 4C technology to interact with (3-globin in foetal brain but not liver scored co-localisation frequencies respectively of 12.9% and 7% in.<br>
brain, versus 3.6% and 1.9% in liver sections. In summary, co-localisation frequencies measured for loci positively identified by 4C technology were all significantly higher than frequencies measured for background loci (p
EXAMPLE 8<br>
Nuclear organisation of active and inactive chromatin domains<br>
The observations described herein demonstrate that not only active, but also inactive genomic regions form distinct regions in the nuclear space that involve many long-range contacts, strongly suggesting that each DNA segment has its own preferred set of interactions. Our data suggest that when the (3-globin locus is switched on, it leaves a transcriptional silent genomic environment and enters a nuclear area where interactions with active domains are favoured. It is anticipated that such a dramatic repositioning upon transcriptional activation may well be a hallmark only of tissue-specific genes that reach a certain expression level and, more importantly, lie isolated from other active genes on the linear chromosome template, as is the case for fi-globin. It is proposed that the extensive network of long-range interactions that are identified both between inactive and between active genomic loci, reflects cell-to-cell differences in chromosome conformations more than being a consequence of dynamic movements<br>
during interphase (Chakalova et al. (2005) Nat Rev Genet 6, 669-77 (2005). Presumably, different degrees of de-condensation after cell division drive the active genomic regions away from inactive chromatin (Gilbert, N. et al. (2004) Cell 118, 555-66 (2004)) and contacts between distant loci of similar chromatin composition are stabilised mostly through affinities between chromatin-bound proteins. Spatial juxtaposition between distant loci may be functional, but may also simply be the consequence of the unfolding patterns of a chromosome. While individual loci can move within a restricted nuclear volume, the general conformation of a chromosome would largely be maintained throughout the cell cycle and requiring cell division for resetting. This idea is in agreement with life cell imaging studies showing restricted motion of tagged DNA loci in the nuclear interior (Chubb et al. (2002) Curr Biol 12, 439-45 (2002)) and fits well with studies showing that nuclear chromatin position information is frequently propagated during the cell division without being conserved in the population of cells (Essers, J. et al. Mol Biol Cell 16, 769-75 (2005); Gerlich, D. et al. Cell 112, 751-64 (2003)).<br>
FURTHER ASPECTS 1<br>
Further aspects of the present invention are set forth below in the numbered paragraphs.<br>
1.	A set of probes complementary to every side of every primary restriction enzyme<br>
recognition site in the genome of a given species (e.g. human).<br>
2.	A set of probes complementary to only one side of every primary restriction enzyme<br>
recognition site in the genome of a given species (e.g. human),<br>
3.	A set of probes complementary to one side of every other primary restriction<br>
enzyme recognition site as ordered along the linear template of the genome of a given<br>
species (e.g. human).<br>
4.	A set of probes complementary to one side of every third, fourth, fifth, sixth,<br>
seventh, eight, ninth, tenth, twentieth, thirtieth, fortieth, fiftieth, sixtieth, seventieth<br>
eightieth, ninetieth or one hundredth primary restriction enzyme recognition site as<br>
ordered along the linear template of the genome of a given species (e.g. human).<br>
5.	A set of probes representing genomic regions of a given size (eg. about 50 kb, 100<br>
Kb, 200 kb, 300 kb, 400 kb, 500 kb, 1 Mb, 2 Mb, 3Mb, 4Mb, 5Mb, 6Mb, 6Mb, 7Mb,<br>
8Mb, 9Mb or 10Mb) (eg. 50kb-10Mb) around all loci known to be involved in<br>
translocations, deletions, inversions, duplications and other genomic rearrangements,<br>
6.	A set of probes representing genomic regions of a given size (eg. about 50 kb, 100<br>
kb, 200 kb, 300 kb, 400 kb, 500 kb, 1 Mb, 2 Mb, 3Mb, 4Mb, 5Mb, 6Mb, 6Mb, 7Mb,<br>
8Mb, 9Mb or 10Mb) (eg. 50kb-10Mb) around a selection of loci known to be involved<br>
in    translocations,    deletions,    inversions,    duplications    and    other    genomic<br>
rearrangements.<br>
7.	Preferably, the 4C sequence (bait) is within about 50 kb, 100 kb, 200 kb, 300 kb,<br>
400 kb, 500 kb, 1 Mb, 2 Mb, 3Mb, 4Mb, 5Mb, 6Mb, 6Mb, 7Mb, 8Mb, 9Mb, 10Mb,<br>
11Mb, 12Mb, 13Mb, 14Mb or 15Mb or more from the actual rearranged sequence (i.e.<br>
breakpoint in case of a translocation).<br><br>
8.	A set of probes representing the complete genome of a given species, with each<br>
probe representing a single restriction fragment as obtained or obtainable after<br>
digestion with a primary restriction enzyme.<br>
9.	A set of probes representing the complete genome of a given species, with probes<br>
equally distributed along the linear chromosome templates.<br>
10.	An array comprising the set of probes according to any of paragraphs 1-10.<br>
11.	A method for analysing the frequency of interaction of a target nucleotide<br>
sequence with one or more nucleotide sequences (eg. one or more genomic loci)<br>
comprising the use of a nucleotide sequence or an array of probes or a set of probes or an array as described herein.<br>
12.	A method for identifying one or more DNA-DNA interactions that are indicative<br>
of a particular disease state comprising the use of a nucleotide sequence or an array of<br>
probes or a set of probes or an array as described herein.<br>
13.	A method of diagnosis or prognosis of a disease or syndrome caused by or<br>
associated with a change in a DNA-DNA comprising the use of a nucleotide sequence<br>
or an array of probes or a set of probes or an array as described herein.<br>
14.	An assay method for identifying one or more agents that modulate a DNA-DNA<br>
interaction comprising the use of a nucleotide sequence or an array of probes or a set<br>
of probes or an array as described herein.<br><br>
15.	A method for detecting the location of a breakpoint (eg.  a translocation)<br>
comprising the use of a nucleotide sequence or an array of probes or a set of probes or<br>
an array as described herein.<br>
16.	A method for detecting the location of an inversion comprising the use of a<br>
nucleotide sequence or an array of probes or a set of probes or an array as described<br>
herein.<br>
17.	A method for detecting the location of a deletion comprising the use of a<br>
nucleotide sequence or an array of probes or a set of probes or an array as described<br>
herein.<br>
18.	A method for detecting the location of a duplication comprising the use of a<br>
nucleotide sequence or an array of probes or a set of probes or an array as described<br>
herein.<br>
19.	The use of microarrays in 4C technology to identify (all) DNA segments that are<br>
in close spatial proximity to a DNA segment of choice.<br>
20.	A microarray containing probes homologous to DNA sequences directly adjacent<br>
to the primary restriction enzyme recognition sites present in the genomic region that<br>
is included in the analysis (which can be the complete genome or part of the genome):<br>
each probe locates preferably within 100 bp from, or maximally within 300 bp from, a<br>
unique primary restriction enzyme recognition site, or alternatively is designed<br>
between each primary restriction enzyme recognition site and its closest secondary<br>
restriction enzyme recognition site.<br>
21.	An array as described herein comprising probes complementary to sequences of<br>
selected loci, wherein said array is representative of the complete genome of a given<br>
species.<br>
22.	An array according to paragraph 21, wherein the loci are loci associated with one<br>
or more diseases.<br>
23.	An array according to paragraph 21 or paragraph 22, wherein the sequences of<br>
selected loci included sequences that are up to 20Mb away from said loci.<br>
24.	A method for analysing the frequency of interaction of a target nucleotide<br>
sequence with one or more nucleotide sequences of interest (eg. one or more genomic<br>
loci) comprising the steps of:<br>
(a)	providing a sample of cross-linked DNA;<br>
(b)	digesting the cross-linked DNA with a primary restriction enzyme;<br>
(c)	ligating the cross-linked nucleotide sequences;<br>
(d)	reversing the cross linking;<br>
(e)	digesting the nucleotide sequences with a secondary restriction enzyme;<br>
(f)	ligating the nucleotide sequences;<br>
(g)	amplifying the one or more nucleotide sequences of interest that are ligated to the<br>
target nucleotide sequence using at least two oligonucleotide primers, wherein each<br>
primer hybridises to a known DNA sequence that flanks the nucleotide sequences of<br>
interest;<br>
(h) hybridising the amplified sequence(s) to an array; and<br>
(i) determining the frequency of interaction between the DNA sequences.<br>
FURTHER ASPECTS 2<br>
Still further aspects of the present invention are set forth below in the numbered paragraphs.<br>
1.	A circularised nucleotide sequence comprising a first and a second nucleotide<br>
sequence separated by a primary and a secondary restriction enzyme recognition site,<br>
wherein said first nucleotide sequence is a target nucleotide sequence and said second<br>
nucleotide sequence is obtainable by cross-linking genomic DNA.<br>
2.	The circularised nucleotide sequence according to paragraph 1, wherein the target<br>
nucleotide sequence is selected from the group consisting of a promoter, an enhancer,<br>
a silencer, an insulator, a matrix attachment region, a locus control region, a<br>
transcription unit, an origin of replication, a recombination hotspot, a translocarion<br>
breakpoint, a centromere, a telomere, a gene-dense region, a gene-poor region, a<br>
repetitive element and a (viral) integration site.<br>
3.	The circularised nucleotide sequence according to paragraph 1, wherein the target<br>
nucleotide sequence is a nucleotide sequence that is associated with or causes a<br>
disease, or is located less then 15Mb on a linear DNA template from a locus that is<br>
associated with or causes a disease.<br>
4.	The circularised nucleotide sequence according to any of paragraphs 1-3, wherein<br>
the target nucleotide sequence is selected from the group consisting of: AML1, MLL,<br>
MYC, BCL, BCR, ABL1, IGH, LYL1, TALI, TAL2, LMO2, TCRoc/8, TCRfi and HOX<br>
or other loci associated with disease as described in "Catalogue of Unbalanced<br>
Chromosome Aberrations in Man" 2nd edition. Albert Schinzel. Berlin: Walter de Gruyter, 2001. ISBN 3-11-011607-3.<br>
5.	The circularised nucleotide sequence according to any of paragraphs 1-4, wherein<br>
the primary restriction enzyme recognition site is a 6-8 bp recognition site, preferably<br>
selected from the group consiting of BgKL, HindSl, EcdEI, BarriSl, Spel, Pstl and<br>
Ndel.<br>
6.	The circularised nucleotide sequence according to any of the preceding paragraphs,<br>
wherein the secondary restriction enzyme recognition site is a 4 or 5 bp nucleotide<br>
sequence recognition site.<br>
7.	The circularised nucleotide sequence according to any of the preceding paragraphs,<br>
wherein the secondary restriction enzyme recognition site is located at greater than<br>
about 350bp from the primary restriction site.<br>
8.	The circularised nucleotide sequence according to any of the preceding paragraphs,<br>
wherein the nucleotide sequence is labelled.<br>
9.	A nucleotide sequence comprising a first and a second nucleotide sequence<br>
separated by a primary and a secondary restriction enzyme recognition site, wherein<br>
said first nucleotide sequence is a target nucleotide sequence, the second nucleotide<br>
sequence is obtainable by cross-linking genomic DNA and wherein said second<br>
nucleotide sequence intersects the target nucleotide sequence.<br>
10.	A method for preparing a circularised nucleotide sequence comprising the steps<br>
of:<br>
(a)	providing a sample of cross-linked DNA;<br>
(b)	digesting the cross-linked DNA with a primary restriction enzyme;<br>
(c)	ligating tih.e cross-linked nucleotide sequences;<br>
(d)	reversing the cross linking;<br>
(e)	digesting the nucleotide sequences with a secondary restriction enzyme; and<br>
(f)	circularising the nucleotide sequences.<br>
11.	A method for preparing a nucleotide sequence comprising the steps of:<br>
(a)	providing a sample of cross-linked DNA;<br>
(b)	digesting the cross-linked DNA with a primary restriction enzyme;<br>
(c)	ligating the cross-linked nucleotide sequences;<br>
(d)	reversing the cross linking;<br>
(e)	digesting the nucleotide sequences with a secondary restriction enzyme;<br>
(f)	circularising the nucleotide sequences; and<br>
(g)	amplifying the one or more nucleotide sequences ligated to the target nucleotide<br>
sequence.	<br>
12.	A method according to paragraph 11, wherein the circularised target nucleotide<br>
sequence is linearised before amplification.<br>
13.	A method according to paragraph 12, wherein the circularised target nucleotide<br>
sequence is linearised using a restriction enzyme that recognises a 6 bp or more<br>
recognition site.<br>
14.	A method according to any of paragraphs 10-13, wherein the cross-linked<br>
nucleotide sequence is amplified using PCR.<br>
15.	A method according to paragraph 14, wherein the cross-linked nucleotide sequence<br>
is amplified using inverse PCR.<br>
16.	A method according to paragraph 14 or paragraph 15, wherein the Expand Long<br>
Template PCR System (Roche) is used.<br>
17.	A method for analysing the frequency of interaction of a target nucleotide<br>
sequence with one or more nucleotide sequences (eg. one or more genomic loci)<br>
comprising the use of a nucleotide sequence according to any of paragraphs 1-9.<br>
18.	An array of probes immobilised on a support comprising one or more probes that<br>
hybridise or are capable of hybridising to a nucleotide sequence according to any of<br>
paragraphs 1-9.<br>
19.	A set of probes complementary in sequence to the nucleic acid sequence adjacent<br>
to each one of the primary restriction enzyme recognition sites of a primary restriction<br>
enzyme in genomic DNA,<br>
20.	A set of probes according to paragraph 19, wherein the probes are complementary<br>
in sequence to the nucleic acid sequence adjacent each side of each one of the primary<br>
restriction enzyme recognition sites of a primary restriction enzyme in genomic DNA.<br>
21.	A set of probes according to paragraph 19 or paragraph 20, wherein said probes<br>
are complementary in sequence to the nucleic acid sequence that is less than 300 base<br>
pairs from each one of the primary restriction enzyme recognition sites of a primary<br>
restriction enzyme in genomic DNA.<br>
22.	A set of probes according to any of paragraphs 19-21, wherein the probes are<br>
complementary to the sequence that is less then 300 bp from each one of the primary<br>
restriction enzyme recognition sites of a primary restriction enzyme ha genomic DNA.<br>
23.	A set of probes according to any of paragraphs 19-22, wherein the probes are<br>
complementary to the sequence that is between 200 and 300 bp from each one of the<br>
primary restriction enzyme recognition sites of a primary restriction enzyme hi<br>
genomic DNA.<br>
24.	A set of probes according to any of paragraphs 19-23, wherein the probes are<br>
complementary to the sequence that is between 100 and 200 bp from each one of the<br>
primary restriction eozyme recognition sites of a primary restriction enzyme in<br>
genomic DNA.<br>
25.	A set of probes according to any of paragraphs 19-24, wherein two or more probes<br>
are designed that are capable of hybridising to the sequence adjacent each primary<br>
restriction enzyme recognition site of a primary restriction enzyme in the genomic<br>
DNA.<br>
26.	A set of probes according to paragraph 25, wherein the probes overlap or partially<br>
overlap.<br>
27.	A set of probes to paragraph 26, wherein the overlap is less than 10 nucleotides.<br>
28.	A set of probes according to any of paragraphs 19-27, wherein the probe sequence<br>
corresponds to all or part of the sequence between each one of the primary restriction<br>
enzyme recognition sites of a primary restriction enzyme and each one of the first<br>
neighbouring secondary restriction enzyme recognition sites of a secondary restriction<br>
enzyme.<br>
29.	A set of probes according to any of paragraphs 19-28, wherein each probe is at a<br>
least a 25 mer.<br>
30. A set of probes according to any of paragraphs 19-29, wherein each probes is a 25-50 mer.<br>
31. A process for preparing a set of probes comprising the steps of:<br>
a) identifying each one of the primary restriction enzyme recognition sites for a primary restriction enzyme in genomic DNA;<br>
(b)	designing probes that are capable of hybridising to the sequence adjacent each one<br>
of the primary restriction enzyme recognition sites in the genomic DNA;<br>
(c)	synthesising the probes; and<br>
(d)	combining the probes together to form a set of probes or substantially a set of<br>
probes.<br><br>
32.	A process according to paragraph 31, wherein the probes are PCR amplification<br>
products.<br>
33.	A set of probes or substantially a set of probes obtained or obtainable by the<br>
process according to paragraph 31 or paragraph 32.<br>
34.	An array comprising the array of probes according to paragraph 18 or substantially<br>
the set of probes according to any of paragraphs 19-30 or 33.<br>
35.	An array comprising the set of probes according to any of paragraphs 19-30 or 33.<br>
36.	An array according to paragraph 34 or paragraph 35, wherein the array comprises<br>
about 300,000-400,000 probes.<br>
37.	An array according to any of paragraphs 34-36, wherein the array comprises about<br>
385,000 or more probes, preferably, about 750,000 probes, more preferably, 6 x<br>
750,000 probes.<br>
38.	An array according to any of paragraphs 34-37, wherein if the number of probes<br>
exceeds the number of probes that can be contained in a single array, then the array<br>
comprises or consists of a representation of the complete genome of a given species at<br>
lower resolution.<br>
39.	An array according to paragraph 38, wherein one out of every 2, 3, 4, 5, 6, 7, 8, 9<br>
or 10 probes as ordered on a linear chromosome template is contained in the array.<br><br>
40.	A process for preparing an array comprising the step of immobilising on a solid<br>
support substantially the array of probes according to paragraph 18 or substantially the<br>
set of probes according to any of paragraphs 15-30 or 33.<br>
41.	A process for preparing an array comprising the step of immobilising on a solid<br>
support the array of probes according to paragraph 18 or the set of probes according to<br>
any of paragraphs 19-30 or 33.<br>
42.	An array obtained or obtainable by the method according to paragraph 40 or -<br>
paragraph 41.<br>
43.	A method for analysing the frequency of interaction of a target nucleotide<br>
sequence with one or more nucleotide sequences (eg. one or more genomic loci)<br>
comprising the steps of:<br>
(a)	providing a sample of cross-linked DNA;<br>
(b)	digesting the cross-linked DNA with a primary restriction enzyme;<br>
(c)	ligating the cross-linked nucleotide sequences;<br>
(d)	reversing the cross linking;<br>
(e)	digesting the nucleotide sequences with a secondary restriction enzyme;<br>
(f)	circularising the nucleotide sequences;<br>
(g)	ampliiying the one or more nucleotide sequences that are ligated to the target<br>
nucleotide sequence;<br>
(h) optionally hybridising the amplified sequences to an array; and<br>
(i) determining the frequency of interaction between the DNA sequences.<br>
44. A method for identifying one or more DNA-DNA interactions that are indicative of a particular disease state comprising the steps of:<br>
(a)	providing a sample of cross-linked DNA from a diseased and a non-diseased cell;<br>
(b)	digesting the cross-linked DNA in each of the samples with a primary restriction<br>
enzyme;<br>
(c)	ligating the cross-linked nucleotide sequences;<br>
(d)	reversing the cross linking;<br>
(e)	digesting the nucleotide sequences with a secondary restriction enzyme;<br>
(f)	circularising the nucleotide sequences;<br>
(g)	amplifying the one or more sequences that are ligated to the target nucleotide<br>
sequence;<br>
 (h) optionally hybridising the amplified nucleotide sequences to an array, and (i) determining the frequency of interaction between the DNA sequences,<br>
wherein a difference between the frequency of interaction between the DNA sequences from the diseased and non-diseased cells indicates that the DNA-DNA interaction is indicative of a particular disease state.<br>
45. A method of diagnosis or prognosis of a disease or syndrome caused by or associated with a change in a DNA-DNA interaction comprising the steps of:<br>
(a)	providing a sample of cross-linked DNA from a subject;<br>
(b)	digesting the cross-linked DNA with a primary restriction enzyme;<br>
(c)	ligating the cross-linked nucleotide sequences;<br>
(d)	reversing the cross linking;<br>
(e)	digesting the nucleotide sequences with a secondary restriction enzyme;<br>
(f)	circularising the nucleotide sequences;<br>
(g)	amplifying the one or more sequences that are ligated to the target nucleotide<br>
sequence;<br>
(h) optionally hybridising the amplified nucleotide sequences to an array; (i) determining the frequency of interaction between the DNA sequences; and<br>
0) comparing the frequency of interaction between the DNA sequences with that of an unaffected control;<br>
wherein a difference between the value obtained from the control and the value obtained from the subject is indicative that the subject is suffering from the disease or syndrome or is indicative that the subject will suffer from the disease or syndrome.<br>
46.	A method according to paragraph 45, wherein a transition from low to high<br>
interaction frequencies is indicative of the location of a breakpoint.<br>
47.	A method according to paragraph 45 wherein an inversed pattern of DNA-DNA<br>
interaction frequencies for the subject sample as compared to the control is indicative<br>
of an inversion.<br>
48.	A method according to paragraph 45 wherein a reduction in the DNA-DNA<br>
interaction frequency for the subject sample     as compared to the  control, in<br>
combination with an increase in DNA-DNA interaction frequency for more distant<br>
regions, is indicative of deletion.<br>
49.	A method according to paragraph 45, wherein an increase or a decrease in DNA-<br>
DNA interaction frequency for the subject sample as compared to the control is<br>
indicative of a duplication or insertion.<br>
50.	A method according to any of paragraphs 45-49, wherein spectral karyotyping<br>
and/or FISH is used prior to performing said method.<br>
51.	A method according to any of paragraphs 45-50, wherein the disease is a genetic<br>
disease.<br>
52.	A method according to any of paragraphs 45-51, wherein the disease is cancer.<br>
53. A method of diagnosis or prognosis of a disease or syndrome caused by or associated with a change in a DNA-DNA interaction comprising the steps of:<br>
(a)	providing a sample of cross-linked DNA from a subject,<br>
(b)	digesting the cross-linked DNA with a primary restriction enzyme;<br>
(c)	ligating the cross-linked micleotide sequences;<br>
(d)	reversing the cross linking;<br>
(e)	digesting the nucleotide sequences with a secondary restriction enzyme;<br>
(f)	circularising the nucleotide sequences;<br>
(g)	amplifying two or more sequences that are ligated to the target nucleotide<br>
sequence(s);<br>
(h) labelling the two or more amplified sequences;<br>
(i) hybridising the nucleotide sequences to an array;<br>
(j) determining the frequency of interaction between the DNA sequences; and<br>
(j) identifying one or more loci that have undergone a genomic rearrangement that is associated with a disease.<br>
54.    A method according to paragraph 53, wherein the two or more amplified sequences are differentially labelled.<br>
55.	A method according to paragraph 54, wherein the two or more amplified<br>
sequences   are   identically   labelled   when   the   sequences   reside   on   different<br>
chromosomes.<br>
56.	A method according to paragraph 53, wherein the two or more amplified<br>
sequences are identically labelled when the sequences reside on the same chromosome<br>
at a distance that is far enough for minimal overlap between DNA-DNA interaction<br>
signals.<br>
57.	An assay method for identifying one or more agents that modulate a DNA-DNA<br>
interaction comprising the steps of:<br>
(a)	contacting a sample with one or more agents;<br>
(b)	providing cross-linked DNA from the sample;<br>
(c)	digesting the cross-linked DNA with a primary restriction enzyme;<br>
(d)	ligating the cross-linked nucleotide sequences;<br>
(e)	reversing the cross linking;<br>
(f)	digesting the nucleotide sequences with a secondary restriction enzyme;<br>
(g)	circularising the nucleotide sequences;<br>
(h) amplifying the one or more nucleotide sequences that are ligated to the target nucleotide sequence;<br>
(i) optionally hybridising the amplified nucleotide sequences to an array; and (j) determining the frequency of interaction between the DNA sequences,<br>
wherein a difference between (i) the frequency of interaction between the DNA sequences in the presence of the agent and (ii) the frequency of interaction "between the DNA sequences in the absence of the agent is indicative of an agent that modulates the DNA-DNA interaction.<br>
58. A method for detecting the location of a breakpoint (eg. a translocation) comprising the steps of:<br>
(a)	providing a sample of cross-linked DNA;<br>
(b)	digesting the cross-linked DNA with a primary restriction enzyme;<br>
(c)	ligating the cross-linked nucleotide sequences;<br>
(d)	reversing the cross linking;<br>
(e)	digesting the nucleotide sequences with a secondary restriction enzyme;<br>
(f)	circularising the nucleotide sequences;<br>
(g)	amplifying the one or more sequences that are ligated to the target nucleotide<br>
sequence;<br>
(h) optionally hybridising the amplified nucleotide sequences to an array; (i) determining the frequency of interaction between the DNA sequences; and<br>
(j) comparing the frequency of interaction between the DNA sequences with that of a control;<br>
wherein a transition from low to high DNA-DNA interaction frequency in the sample as compared to the control is indicative of the location of a breakpoint.<br>
59. A method for detecting the location of an inversion comprising the steps of:<br>
(a)	providing a sample of cross-linked DNA;<br>
(b)	digesting the cross-linked DNA with a primary restriction enzyme;<br>
(c)	ligating the cross-linked nucleotide sequences;<br>
(d)	reversing the cross linking;<br>
(e)	digesting the nucleotide sequences with a secondary restriction enzyme;<br>
(f)	circularising the nucleotide sequences;<br>
(g)	amplifying the one or more sequences that are ligated to the target nucleotide<br>
sequence;<br>
(h) optionally hybridising the amplified nucleotide sequences to an array; (i) determining the frequency of interaction between the DNA sequences; and<br>
(j) comparing the frequency of interaction between the DNA sequences with that of a control;<br>
wherein an inversed pattern of DNA-DNA interaction frequencies for the sample as compared to the control is indicative of an inversion.<br>
60. A method for detecting the location of a deletion comprising the steps of:<br>
(a)	providing a sample of cross-linked DNA;<br>
(b)	digesting the cross-linked DNA with a primary restriction enzyme;<br>
(c)	ligating the cross-linked nucleotide sequences;<br>
(d)	reversing the cross linking;<br>
(e)	digesting the nucleotide sequences with a secondary restriction enzyme;<br>
(f)	circularising the nucleotide sequences;<br>
(g)	amplifying the one or more sequences that are ligated to the target nucleotide<br>
sequence;<br>
(h) optionally hybridising the amplified nucleotide sequences to an array; (i) determining the frequency of interaction between the DNA sequences; and<br>
(j) comparing the frequency of interaction between the DNA sequences with that of a control;<br>
wherein a reduction in the DNA-DNA interaction frequency for the sample as compared to the control is indicative of deletion.<br>
61. A method for detecting the location of a duplication comprising the steps of:<br>
(a)	providing a sample of cross-linked DNA;<br>
(b)	digesting the cross-linked DNA with a primary restriction enzyme;<br>
(c)	ligating the cross-linked nucleotide sequences;<br>
(d)	reversing the cross linking;<br>
(e)	digesting the nucleotide sequences with a secondary restriction enzyme;<br>
(f)	circularising the nucleotide sequences;<br>
(g)	amplifying the one or more sequences that are ligated to the target nucleotide<br>
sequence;<br>
(h) optionally hybridising the amplified nucleotide sequences to an array; (i) determining the frequency of interaction between the DNA sequences; and<br>
(j) comparing the frequency of interaction between the DNA sequences with that of a control;<br>
wherein an increase or a decrease in DNA-DNA interaction frequency for the subject sample as compared to the control is indicative of a duplication or insertion.<br>
62.	An agent obtained or obtainable by the assay method according to paragraph 57.<br>
63.	Use of a nucleotide sequence according to any of paragraphs 1-9 for identifying<br>
one or more DNA-DNA interactions in a sample.<br>
64.	Use of a nucleotide sequence according to any of paragraphs 1-9 for the diagnosis<br>
or prognosis of a disease or syndrome caused by or associated with a change in a<br>
DNA-DNA interaction.<br>
65.	Use of an array of probes according to paragraph 18 or the set of probes according<br>
to any of paragraphs 19-30 or 33 for identifying one or more DNA-DNA interactions<br>
in a sample.<br>
66.	Use of an array of probes according to paragraph 18 or the set of probes according<br>
to any of paragraphs 19-30 or 33 for the diagnosis or prognosis of a disease or<br>
syndrome caused by or associated with a change in a DNA-DNA interaction.<br>
67.	Use of an array according to any of paragraphs 34-39 or 42 for identifying one or<br>
more DNA-DNA interactions in a sample.<br>
68.	Use of an array according to any of paragraphs 34-39 or 42 for the diagnosis or<br>
prognosis of a disease or syndrome caused by or associated with a change in a DNA-<br>
DNA interaction.<br>
69.	Use according to any of paragraphs 64, 66 or 68, wherein the diagnosis or<br>
prognosis is prenatal diagnosis or prognosis.<br>
70.	A method substantially as described herein and. with reference to any of the<br>
Examples or Figures.<br>
71.	An array of probes substantially as described herein and with reference to any of<br>
the Examples or Figures.<br>
72.	A set of probes substantially as described herein and with reference to any of the<br>
Examples or Figures.<br>
73.	A process substantially as described herein and with reference to any of the<br>
Examples or Figures.<br>
74.	An array substantially as described herein and with reference to any of the<br>
Examples or Figures.<br>
75.	An assay method substantially as described herein and with reference to any of the<br>
Examples or Figures.<br>
76.	An agent substantially as described herein and with reference to any of the<br>
Examples or Figures.<br>
77.	A use substantially as described herein and with reference to any of the Examples<br>
or Figures.<br>
TABLE 2<br><br>
(Table Removed)<br>
REFERENCES<br>
Blanton J, Gaszner M, Schedl P. 2003. Proteinrprotein interactions and the pairing of boundary elements in vivo. Genes Dev 17:664-75.<br>
Dekker, J., Rippe, K., Dekker, M., and Kleckner, N. 2002. Capturing chromosome conformation. Science 295: 1306-11.<br>
Drissen R, Palstra RJ, Gillemans N, Splinter E, Grosveld F, Philipsen S, de Laat W. 2004. The active spatial organization of the beta-globin locus requires the transcription factor EKLF. Genes Dev 18:2485-90.<br>
Horike S, Cai S, Miyano M, Cheng JF, Kohwi-Shigematsu T. 2005. Loss of silent-chromatin loophig and impaired imprinting of DLX5 in Rett syndrome. Nat Genet 37:31-40.<br>
Murrell A, Heeson S, Reik W. 2004. Interaction between differentially methylated regions partitions the imprinted genes Igf2 and H19 into parent-specific chromatin loops. Nat Genet 36:889-93.<br>
Palstra, R.J., Tolhuis, B.} Splinter, E., Nijmeijer, R,, Grosveld, F., and de Laat, W. 2003. The beta-globin nuclear compartment in development and erythroid differentiation. Nat Genet 35: 190-4.<br>
Patrinos, G.P., de Krom, M., de Boer, E., Langeveld, A., Imam, A.M.A, Strouboulis, J., de Laat, W., and Grosveld, F.G. (2004). Multiple interactions between regulatory regions are required to stabilize an active chromatin hub. Genes &amp; Dev. 18: 1495-1509.<br>
Spilianakis CG, Flavell RA. 2004. Long-range intrachromosomal interactions in the T helper type 2 cytokine locus. Nat Immunol 5:1017-27.<br>
Tolhuis, B., Palstra, RJ., Splinter; E., Grosveld, F., and de Laat, W. 2002. Looping and interaction between hypersensitive sites in the active beta-globin locus. Molecular Cell 10: 1453-65.<br>
Vakoc CR, Letting DL, Gheldof N, Sawado T, Bender MA, Groudine M, Weiss MJ, Dekker J, Blobel GA. 2005. Proximity among distant regulatory elements at the beta-globin locus requires GATA-1 and FOG-1. Mol Cell. 17:453-62<br>
All publications mentioned in the above specification are herein incorporated by reference. Various modifications and variations of the described methods and system of the invention will be apparent to those skilled in the art without departing from the scope and spirit of the invention. Although the invention has been described in connection with specific preferred embodiments, it should be understood that the invention as claimed should not be unduly limited to such specific embodiments. Indeed, various modifications of the described modes for carrying out the invention which are obvious to those skilled in molecular biology or related fields are intended to be within the scope of the following claims.<br><br><br><br><br><br>
CLAIMS<br>
1. A method for analysing the frequency of interaction of a target nucleotide sequence with one or more nucleotide sequences of interest (eg. one or more genomic loci) comprising the steps of:<br>
(a)	providing a sample of cross-linked DNA;<br>
(b)	digesting the cross-linked DNA with a primary restriction enzyme;<br>
(c)	ligating the cross-linked nucleotide sequences;<br>
(d)	reversing the cross linking;<br>
(e)	digesting the nucleotide sequences with a secondary restriction enzyme;<br>
(i) ligating one or more DNA sequences of known nucleotide composition to the available secondary restriction enzyme digestion site(s) that flank the one or more nucleotide sequences of interest;<br>
(g) amplifying the one or more nucleotide sequences of interest using at least two oligonucleotide primers, wherein each primer hybridises to the DNA sequences that flank the nucleotide sequences of interest;<br>
(h) hybridising the amplified sequence(s) to an array; and<br>
(i) determining the frequency of interaction between the DNA sequences.<br>
2. The method according to claim 1, wherein the ligation reaction in step (f) results in the formation of DNA circles.<br>
3.	The method according to claim 1 or claim 2, wherein the target nucleotide sequence<br>
is selected from the group consisting of a genomic rearrangement, promoter, an<br>
enhancer, a silencer, an insulator, a matrix attachment region, a locus control region, a<br>
transcription unit, an origin of replication, a recombination hotspot, a translocation<br>
breakpoint, a centromere, a telomere, a gene-dense region, a gene-poor region, a<br>
repetitive element and a (viral) integration site.<br>
4.	The method according to any of the preceding claims, wherein the target nucleotide<br>
sequence is a nucleotide sequence that is associated with or causes a disease, or is<br>
located up to or greater than 15Mb on a linear DNA template from a locus that is<br>
associated with or causes a disease.<br>
5.	The method according to any of the preceding claims, wherein the target nucleotide<br>
sequence is selected from (he group consisting of: AML1, MLL, MYC, BCL, BCR,<br>
ABL1, IGH, LYL1, TALI, TAL2,   LMO2, TCRa/S, TCRJ3 and HOX or other loci<br>
associated with disease as described in "Catalogue of Unbalanced Chromosome<br>
Aberrations in Man" 2nd edition. Albert Schinzel. Berlin: Walter de Gmyter, 2001.<br>
ISBN 3-11-011607-3.<br><br>
6.	The method according to any of the preceding claims, wherein the primary<br>
restriction enzyme is a restriction enzyme that recognises a 6-8 bp recognition site.<br>
7.	The method according to claim 6, wherein the primary restriction enzyme is<br>
selected from the group consiting of BglR, HindUl, EcoSI, BamHL, Spel, Pstl and<br>
Ndel.<br>
8.	The method according to any of the preceding claims, wherein the secondary<br>
restriction enzyme is a restriction enzyme that recognises a 4 or 5 bp nucleotide<br>
sequence recognition site.<br>
9.	The method according to any of the preceding claims, wherein the secondary<br>
restriction enzyme recognition site is located at greater than about 350bp from the<br>
primary restriction site in the target nucleotide sequence.<br>
10.	The method according to any of the preceding claims, wherein the nucleotide<br>
sequence is labelled.<br>
11.	A method for analysing the frequency of interaction of a target nucleotide<br>
sequence with one or more nucleotide sequences (eg. one or more genomic loci)<br>
comprising the steps of:<br><br>
(a)	providing a. sample of cross-linked DNA;<br>
(b)	digesting the cross-linked DNA with a primary restriction enzyme;<br>
(c)	ligating the cross-linked nucleotide sequences;<br>
(d)	reversing the cross linking;<br>
(e)	digesting the nucleotide sequences with a secondary restriction enzyme;<br>
(f)	circularising the nucleotide sequences;<br>
(g)	amplifying the one or more nucleotide sequences that are ligated to the target<br>
nucleotide sequence;<br>
(h) optionally hybridising the amplified sequences to an array; and<br>
(i) determining the frequency of interaction between the DNA sequences.<br>
12.   A circularised nucleotide sequence comprising a first and a second nucleotide sequence, wherein each end of the first and a second nucleotide sequences are<br>
separated "by different restriction enzyme recognition sites, and wherein said first nucleotide sequence is a target nucleotide sequence and said second nucleotide sequence is obtainable by cross-Unking genomic DNA.<br>
13.	A method for preparing a circularised nucleotide sequence comprising the steps<br>
of:<br>
(a)	providing a sample of cross-linked DNA;<br>
(b)	digesting the cross-linked DNA with a primary restriction enzyme;<br>
(c)	ligating the cross-linked nucleotide sequences;<br>
(d)	reversing the cross linking;<br>
(e)	digesting the nucleotide sequences with a secondary restriction enzyme; and<br>
(f)	circularising the nucleotide sequences,<br><br>
14.	A method for analysing the frequency of interaction of a target nucleotide<br>
sequence with one or more nucleotide sequences (eg. one or more genomic loci)<br>
comprising the use of a nucleotide sequence according to claim 12.<br>
15.	An array of probes immobilised on a support comprising one or more probes that<br>
hybridise or are capable of hybridising to a nucleotide sequence according to claim 12.<br>
16.	A set of probes complementary in sequence to the nucleic acid sequence adjacent<br>
to each one of the primary restriction enzyme recognition sites of a primary restriction<br>
enzyme in genomic DNA.<br>
17.	The set of probes according to claim 16, wherein the probes are complementary in<br>
sequence to the nucleic acid sequence adjacent each side of each one of the primary<br>
restriction enzyme recognition sites of a primary restriction enzyme in genomic DNA.<br>
18.	The set of probes according to claim 16 or claim 17, wherein said probes are<br>
complementary in sequence to the nucleic acid sequence that is less than 300 base<br>
pairs from each one of the primary restriction enzyme recognition sites of a primary<br>
restriction enzyme in genomic DNA.<br>
19.	The set of probes according to any of claims 16-18, wherein the probes are<br>
complementary to the sequence that is less then 300 bp from each one of the primary<br>
restriction enzyme recognition sites of a primary restriction enzyme in genomic DNA.<br><br>
20.	The set of probes according to any of claims 16-19, wherein the probes are<br>
complementary to the sequence that is between 200 and 300-bp from each one of the<br>
primary restriction enzyme recognition sites of a primary restriction enzyme in<br>
genomic DNA-<br>
21.	The set of probes according to any of claims 16-20, wherein the probes are<br>
complementary to the sequence that is between 100 and 200 bp or 0 to 100 bp from<br>
each one of the primary restriction enzyme recognition sites of a primary restriction<br>
enzyme in genomic DNA.<br>
22.	The set of probes according to any of claims 16-21, wherein two or more probes<br>
are capable of hybridising to the sequence adjacent each primary restriction enzyme<br>
recognition site of a primary restriction enzyme in the genomic DNA.<br>
23.	The set of probes according to claim 22, wherein the probes overlap or partially<br>
overlap.<br>
24.	The set of probes to claim 23, wherein the overlap is less than 1Q nucleotides.<br>
25.	The set of probes according to any of claims 16-24, wherein the probe sequence<br>
corresponds to all or part of the sequence between each one of the primary restriction<br>
enzyme recognition sites of a primary restriction enzyme and each one of the first<br>
neighbouring secondary restriction enzyme recognition sites of a secondary restriction<br>
enzyme.<br>
26.	The set of probes according to any of claims 16-25, wherein each probe is at a<br>
least a 25 mer.<br>
27.	The set of probes according to any of claims 16-26, wherein each probes is a 25-<br>
60 mer.<br>
28.	A process for preparing a set of probes comprising the steps of:<br>
(a)	identifying each one of the primary restriction enzyme recognition sites for a<br>
primary restriction enzyme in genormc DNA;<br>
(b)	designing probes that are capable of hybridising to the sequence adjacent each one<br>
of the primary restriction enzyme recognition sites in the genomic DNA;<br>
(c)	synthesising the probes; and<br>
(d)	combining the probes together to form a set of probes or substantially a set of<br>
probes.<br><br>
29.	The process according to claim 28, wherein the probes are PCR amplification<br>
products.<br>
30.	A set of probes or substantially a set of probes obtained or obtainable by the<br>
process according to claim 28 or claim 20.<br>
31.	An array comprising the array of probes according to claim 15 or substantially the<br>
set of probes according to any of claims 16-27 or 30.<br>
32.	An array comprising the set of probes according to any of claims 16-27 or 30.<br>
33.	The array according to claim 32 or claim 32, wherein the array comprises about<br>
300,000-400,000 probes.<br>
34.	The array according to any of claims 32-33, wherein the array comprises about<br>
385,000 or more probes, preferably, about 750,000 probes, more preferably, 6 x<br>
750,000 probes.<br>
35.	The array according to any of claims 31-34, wherein the array comprises or<br>
consists of a representation of the complete genome of a given species at lower<br>
resolution.<br>
36.	The array according to claim 35, wherein one out of every 2, 3, 4, 5, 6, 7, 8, 9 .or<br>
10 probes as ordered on a linear chromosome template is contained in the array.<br><br>
37.	A process for preparing an array comprising the step of immobilising on a solid<br>
support substantially the array of probes according to claim 15 or substantially the set<br>
of probes according to any of claims 16-27 or 30.<br>
38.	A process for preparing an array comprising the step of immobilising on a solid<br>
support the array of probes according to claim 16 or the set of probes according to any<br>
of claims 16-27 or 30.<br><br>
39.	An array obtained or obtainable by the method according to claim 37 or claim 38.<br>
40.	A method for identifying one or more DNA-DNA interactions that are indicative<br>
of a particular disease state comprising the step of performing steps (a)-(i) of claims 1-<br>
11, wherein in step (a) a sample of cross-linked DNA is provided from a diseased and<br>
a non-diseased cell, and wherein a difference between the frequency of interaction between the DNA sequences from the diseased and non-diseased cells indicates that the DNA-DNA interaction is indicative of a particular disease state.<br>
41.	A method of diagnosis or prognosis of a disease or syndrome caused by or<br>
associated with a change in a DNA-DNA interaction comprising the step of<br>
performing steps (a)-(i) any of claims 1-11, wherein step (a) comprises providing a<br>
sample of cross-linked DNA from a subject; and wherein step (i) comprises comparing<br>
the frequency of interaction between the DNA sequences with that of an unaffected<br>
control; wherein a difference between the value obtained from the control and the<br>
value obtained from the subject is indicative that the subject is suffering from the<br>
disease or syndrome or is indicative that the subject will suffer from the disease or<br>
syndrome.<br>
42.	The method according to claim 41, wherein a transition from low to high<br>
interaction frequencies is indicative of the location of a balanced and/or unbalanced<br>
breakpoint.<br>
43.	The method according to 41, wherein an inversed pattern of DNA-DNA interaction<br>
frequencies for the subject sample as compared to the control is indicative of an<br>
balanced and/or unbalanced inversion.<br>
44.	The method according to claim 41, wherein a reduction in the DNA-DNA<br>
interaction frequency for the subject sample    as compared to the control, in<br>
combination with an increase in DNA-DNA interaction frequency for more distant<br>
regions, is indicative of a balanced and/or unbalanced deletion.<br>
45.	The method according to claim 41, wherein an increase or a decrease in DNA-<br>
DNA interaction frequency for the subject sample as compared to the control is<br>
indicative of a balanced and/or unbalanced duplication or insertion.<br>
46.	A method according to any of claims 41-45, wherein spectral karyotyping and/or<br>
FISH is used prior to performing said method.<br>
47.	The method according to- any of claims 41-46, wherein the disease is a genetic<br>
disease.<br>
48.	The method according to any of claims 41-47, wherein the disease is cancer.<br>
49.	A method of diagnosis or prognosis of a disease or syndrome caused by or<br>
associated with a change in a DNA-DNA interaction comprising the step  of:<br>
performing steps (a)-(i) of any of claims 1-11, wherein step (a) comprises providing a<br>
sample of cross-linked DNA from a subject; and wherein said method comprises the<br>
additional step of: (j) identifying one or more loci that have undergone a genomic<br>
rearrangement that is associated with a disease.<br>
50.	The method according to claim 49, wherein the two or more amplified sequences<br>
are differentially labelled.<br>
51.	The method according to claim 49, wherein the two or more amplified sequences<br>
are identically labelled when the sequences reside on different chromosomes.<br>
52.	The method according to claim 49, wherein the two or more amplified sequences<br>
are identically labelled when the sequences reside on the same chromosome at a<br>
distance that is far enough for minimal overlap between DNA-DNA interaction<br>
signals.<br>
53.	An assay method for identifying one or more agents that modulate a DNA-DNA<br>
interaction comprising the steps of:<br>
(a) contacting a sample with one or more agents; and<br>
 (b) performing steps (a) to (i) of any of claims 1-11, wherein step (a) comprises providing cross-linked DNA from the sample;<br>
wherein a difference between (i) the frequency of interaction between the DNA sequences in the presence of the agent and (ii) the frequency of interaction between the DNA sequences in the absence of the agent is indicative of an agent that modulates the DNA-DNA interaction.<br>
54.	A method for detecting the location of a balanced and/or unbalanced breakpoint<br>
(eg. a translocation) comprising the step of:<br>
(a)	performing steps (a) to (i) of any of claims 1-11; and<br>
(b)	comparing the frequency of interaction between the DNA sequences with that of a<br>
control;<br>
wherein a transition from low to high DNA-DNA interaction frequency in the sample as compared to the control is indicative of the location of a breakpoint<br>
55.	A method for detecting the location of a balanced and/or unbalanced inversion<br>
comprising the steps of:<br>
(a)	performing steps (a) to (i) of any of claims 1-11; and<br>
(b)	comparing the frequency of interaction between the DNA sequences with that of a<br>
control;<br>
wherein an inversed pattern of DNA-DNA interaction frequencies for the sample as compared to the control is indicative of an inversion.<br>
56.	A method for detecting the location of a deletion comprising the steps of:<br>
(a)	performing steps (a) to (i) of any of claims 1-11; and<br>
(b)	comparing the frequency of interaction between the.DNA sequences witb that of a<br>
control;<br>
wherein a reduction in the DNA-DNA interaction frequency for the sample as compared to the control is indicative of deletion..<br>
57. A method for detecting the location of a duplication comprising the steps of:<br>
(a)	performing steps (a) to .(i) of any of claims 1-11; and<br>
(b)	comparing the frequency of interaction between the DNA sequences with that of a<br>
control;<br>
wherein an increase or a decrease in DNA-DNA interaction frequency for the subject sample as compared to the control is Indicative of a duplication or insertion.<br>
58. An agent obtained or obtainable by (he assay method according to claim 53.<br>
59. A method, an. array of probes, a set of probes, a process, an array, an assay method,. an agent, or a use substantially as described herein and with reference to any of the Examples or Figures.</td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/viewdoc.aspx?id=picVd6BxcN2SG7VMjbIV+w==&amp;amp;loc=+mN2fYxnTC4l0fUd8W4CAA==" target="_blank" style="word-wrap:break-word;">http://ipindiaonline.gov.in/patentsearch/GrantedSearch/viewdoc.aspx?id=picVd6BxcN2SG7VMjbIV+w==&amp;amp;loc=+mN2fYxnTC4l0fUd8W4CAA==</a></p>
		<br>
		<div class="pull-left">
			<a href="269829-multimedia-playback-control-apparatus-and-method.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="269831-method-for-recovering-rare-metals-in-a-zinc-leaching-process.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>269830</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>9959/DELNP/2007</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>46/2015</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>13-Nov-2015</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>09-Nov-2015</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>24-Dec-2007</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>ERASMUS UNIVERSITY MEDICAL CENTRE</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td>FACULTY OF MEDICINE, DEPT.OF CELL BIOLOGY AND GENETICS, P.O.BOX 1738, NL-3000 DR ROTTERDAM, NETHERLANDS.</td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>WOUTER DE LAAT</td>
											<td>DEPT. OF CELL BIOLOGY, ERASMUS MC,PO BOX 1738,NL-3000 DR ROTTERDAM NETHERLANDS.</td>
										</tr>
										<tr>
											<td>2</td>
											<td>FRANK GROSVELD</td>
											<td>DEPT. OF CELL BIOLOGY, ERASMUS MC, PO BOX 1738 NL-3000 DR ROTTERDAM, NETHERLANDS.</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>C12Q 1/68</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>PCT/GB2006/002268</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td>2006-07-03</td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td>0605449.8</td>
									<td>2006-03-17</td>
								    <td>U.K.</td>
								</tr>
								<tr>
									<td>2</td>
									<td>0513676.7</td>
									<td>2005-07-04</td>
								    <td>U.K.</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/269830-a-method-for-analysing-the-frequency-of-interaction-of-a-target-nucleotide-sequence by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 06:23:39 GMT -->
</html>
