<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/270506-means-and-methods-for-mediating-protein-interference by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 04:20:46 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 270506:MEANS AND METHODS FOR MEDIATING PROTEIN INTERFERENCE</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">MEANS AND METHODS FOR MEDIATING PROTEIN INTERFERENCE</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>The present invention belongs to the field of functional proteomics and more particularly to the field of protein aggregation. The invention discloses a method for interfering with the function of a target protein and uses a non-naturally, user-designed molecule, designated as interferor, that has a specificity for a target protein and which induces aggregation upon contact with said target protein. The present invention also discloses such interferor molecules and their use in therapeutic applications.</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td>Means and methods for mediating protein interference<br>
Field of the invention<br>
The present invention belongs to the field of functional proteomics and more particularly to the field of protein aggregation. The invention discloses a method for interfering with the function of a target protein and uses a non-naturally, user-designed molecule, designated as interferon that has a specificity for a target protein and which induces aggregation upon contact with said target protein. The present invention also discloses such interferor molecules and their use in therapeutic applications.<br>
Background of the invention<br>
Biology is entering an exciting era brought about by the increase in genome-wide information. As genome sequencing and high-throughput functional genomics approaches generate more and more data, researchers need new ways to tease out biological relevant information. Functional genomics in particular is making rapid progress in assigning biological meaning to genomic data. The information encoded in the genome comprises genes, the protein products of which mediate most of the functions in organisms, and control elements. Proteins were thought to be the most important effectors in the cells, although recently non-coding RNAs have also been identified as important players in regulatory processes.<br>
Several key biological questions are central to continuing genome projects and are relevant to any cellular organism, from bacteria to humans. One challenge is to understand how genes that are encoded in a genome operate and interact to produce a complex living system. A related challenge is to determine the function of all the sequence elements in the genome. The toolbox of functional genomics has enabled several systematic approaches that can provide the answers to a few basic questions for the majority of genes in a genome, including when is a gene expressed, where its product is localized, with which other gene products does it interact and what phenotype results if a gene is mutated. Phenotypic analysis of mutants has been a powerful approach for determining gene function. Gene function can be altered through gene deletions, insertional mutagenesis and RNA interference (RNAi). RNAi is a relatively recent development for reducing gene expression. It follows reports of gene silencing in plants and other model organisms, and is based on the observation from C. elegans that adding double-stranded RNA (dsRNA) to cells often interferes with gene function in a sequence-specific manner. In many cases, the level of functional reduction cannct be adequately controlled, is incomplete, the level of specificity is not entirely precdictable and in some organisms RNAi does not work (e.g. in the yeast Candida albicans).<br>
It is obvious that functional genomics has changed the way biology is done and yet the field is still in its infancy in terms of detailing the complexity that underlies biological systems, such as<br>
the complex network of genetic regulation, protein interactions and biochemical reactions that make up a cell. Clearly there is a need to develop innovative technologies, especially in the field of functional proteomics, in order to accelerate discoveries and to maximize the potential offered by complementary methods in functional genomics. It would be desirable to possess a flexible technology that can directly target the biological function of a particular extracellular or intracellular protein instead of targeting the mRNA that translates it or manipulating the gene that encodes it.<br>
The conversion of normally soluble proteins into conformationally altered insoluble proteins is thought to be a causative process in a variety of diseases such as for example the occurrence of amyloid beta peptide in Alzheimer's disease and cerebral amyloid angiopathy, alpha-synuclein deposits in Lewy bodies of Parkinson's disease, prions in Creutzfeldt-Jacob disease, superoxide dismutase in amyotrophic lateral sclerosis and tau in neurofibrillary tangles in frontal temporal dementia and Pick's disease. Thus far, protein aggregation has mainly been studied as an unwanted, disease-causing phenomenon and it is widely accepted that cross-beta mediated aggregation is the most frequently occurring and biologically relevant mechanism of aggregation2. Cross-beta aggregation is the term used to indicate that aggregation is nucleated via the formation of intermolecular beta-sheets to which each molecule in the aggregate contributes an identical strand of typically comprising at least three contiguous amino acids. There is now abundant data to show that the individual strands interact to form an intermolecular beta sheet and that this structure forms the backbone of the aggregate34. Self-association regions in target proteins can be determined by computer programs, such as TANGO6, which were developed for predicting the aggregation propensity of peptides and proteins. One specific form of aggregation, namely the highly ordered amyloid fibre, is already being explored in the art for potential use in the material sciences5. In addition, WO03102187 (Scegen, Pty Ltd) discloses a method for enhancing the activity of a molecule by fusing said molecule with a membrane translocating sequence, whereby the resulting chimeric molecule self-assembles into a higher molecular weight aggregate. US20050025165 (Arete Associates) discloses the use of conformational peptides, able to interact with the beta-sheet conformation of insoluble proteins such as prions, as a diagnostic tool for prion diseases.<br>
Summary of the Invention<br>
The present invention relates to a technology for the controlled and inducible protein aggregation of specific target proteins. The invention also provides de novo designed molecules, herein designated as interferor molecules, which comprise at least one aggregation region of which said aggregation region is derived from a target protein. In a preferred embodiment the interferor molecule comprises at least one self-association region that is fused to a moiety that prevents aggregation of said self-association region. Upon contact between a<br>
chosen target protein and a specifically designed interferor molecule, a specific co-aggregation occurs between the target and the interferor resulting in a functional knock-out or a down-regulation of the biological function for said target protein. This protein knock-down is conditional upon the presence of aggregates, which are induced by the presence of the interferor molecule. An additional advantage is that the strength of the protein interference can be experimentally controlled by varying the number of aggregation regions in the interferor molecule. The invention does not only provide an efficient tool to down-regulate the biological function of a specific extra- or intracellular protein but has also important therapeutic, agricultural and diagnostic applications.<br>
Figure legends<br>
Figure 1: Protein interference in E. coll using recombinant expression of 4 different interferor constructs which target specific enzymes involved in amino acid biosynthesis. Part B of each interferor molecule consists of 3 synthetic self-association sequences, separated by linkers of 2 amino acids and one specific self-associating region derived from the enzyme. The self-association regions (synthetic and specific) are coupled to the protein NusA which serves as a moiety to prevent aggregation (id est part A of the interferor molecule) of the self-association regions.<br>
Figure 2: S. cerevisiae cells with an endogenous wild type copy of URA3 were transformed with the empty plasmid, the plasmid with only the aggregator sequence or the plasmid with the aggregator-Ura3 fusion construct. The cells were grown overnight in glucose containing medium, washed and then plated on either glucose (left) or galactose (right) containing medium. 5 ul of 10 fold dilutions are plated (OD6oo=1 as highest concentration).<br>
Figure 3: C. albicans cells with two endogenous wild type copies of TUP1 were transformed with the empty plasmid and the plasmid with the interferor-Tupl fusion construct. The cells were grown overnight in glucose containing medium, washed and then 20 colonies were plated on either glucose (left) or casamino acid (right) containing medium. The upper panel are those with an empty plasmid, the lower panel with the interferor construct ("+ aggreg.-TUP1" means interferor-TUP1 construct). Pictures were taken after 4 days of growth.<br>
Aims and detailed description of the invention<br>
In the present invention we have developed a process for down-regulating the biological<br>
function of a protein through the use of interferor molecules that have a specificity for a target<br>
protein. Upon contact with a target protein a co-aggregation occurs between the interferor molecule and the target. The aggregation withdraws the target from its soluble environment and results in a functional knock-down of the target protein.<br>
Thus in one embodiment the invention provides a method for down-regulating the biological function of a protein comprising contacting said protein with a non-naturally occurring molecule comprising at least one self-association region isolated from said protein. In another embodiment the invention provides a method for down-regulating the biological function of a protein comprising contacting said protein with a non-naturally occurring molecule consisting of at least one self-association region isolated from said protein. In yet another embodiment the invention provides a method for down-regulating the biological function of a protein comprising contacting said protein with a non-naturally occurring molecule comprising at least one self-association region isolated from said protein wherein said self-association domain is fused to a moiety that prevents aggregation of said self-association region.<br>
In yet another embodiment the invention provides a method for down-regulating the biological function of a protein comprising contacting said protein with a non-naturally -occurring molecule consisting of at least one self-association region isolated from said protein wherein said self-association domain is fused to a moiety that prevents aggregation of said self-association region.<br>
In yet another embodiment the invention provides a method for down-regulating the biological function of a protein comprising contacting said protein with a non-naturally occurring molecule which comprises part A and part B wherein i) part A is a peptide, or a protein domain or an agarose bead preventing aggregation of part B and ii) part B which comprises at least 1 self-association region consisting of at least 3 contiguous amino acids and wherein said region is isolated from S3id protein which function is to be down-regulated with, and wherein a linker is optionally present between parts A and B.<br>
In yet another embodiment the invention provides a method for down-regulating the function of a protein comprising contacting said protein with a non-naturally occurring molecule which comprises part A and part B wherein i) part A is a peptide, or a protein domain or an agarose bead preventing aggregation of part B so that part B is in direct contact with the solvent wherein said molecule and said protein are present and ii) part B which comprises at least 1 self-association region wherein said region consists of at least 3 contiguous amino acids and wherein said region is isolated from said protein which function is to be down-regulated with, and wherein a linker is optionally present between parts A and B.<br>
In another embodiment part B of the non-naturally occurring molecuje comprises at least 2 self-association regions wherein at least one of said regions is derived from said protein which function is to be interfered with.<br>
The term 'non-naturally occurring molecule' refers to the fact that such an interferor molecule is man made. For instance, when an interferor molecule is polypeptide (id est both part A and B are peptides) such polypeptide is designed by isolating part B from a target protein (id est the self association region) and by coupling said part B to a part A which can be derived (i) from another protein or (ii) from the same target protein in which case said part A is not present immediately adjacent to part B. In still other words the self-association region derived from the target fused to a moiety (when the interferor is a polypeptide said moiety is also a polypeptide) that prevents the aggregation of the self-association region is different from a naturally occurring fusion between part A and B by at least one natural amino acid. Typically, such interferor molecule will not exist as a contiguous polypeptide in a protein encoded by a gene in a non-recombinant genome.<br>
It should be clear that interferor molecules can be designed in a modular fashion, by introducing repetition and changing the order of the parts A and B. A non-limiting Iist of the following combinations is: an interferor with the A-B - structure, an interferor with the B-A -structure, an interferor with the A-B-A - structure, an interferor with the B-A-B - structure, an interferor with the A'-B-A" structure and an interferor with the B'-A-B" structure wherein a linker (spacer) is optionally present between parts A, A', A" and B, B', B". A, A' and A" are different of similar moieties (e.g. different peptide sequences). B, B' and B" are different or similar self association sequences (e.g. B is a self-association sequence derived from the target protein and B' is a synthetic self-association sequence).<br>
In still other words the invention provides a method for down-regulating the biological function of a protein comprising contacting said protein with a molecule comprising at least one self-association region isolated from said protein wherein said self-association region is fused to a moiety that prevents aggregation of said self-association region so that said self-association region is in direct contact with the solvent wherein said molecule and said protein are present. From the above it should be clear that said 'moiety' is equivalent with the term,-part A and part B is equivalent with the wording 'at least one self association region'.<br>
The wording 'down-regulating the function of a protein' means that the normal biological activity of a protein is reduced (inhibited, down-regulated, reduced and disrupted are equivalent words here) or that the protein is withdrawn from its normal biological environment<br>
(e.g. a protein which is a normal resident of the endoplasmic reticulum is not present through down-regulation of its function). Thus, by applying the method of the invention the function of a protein is disrupted through an aggregation of said protein by contacting said protein with the non-natural molecule of the present invention. Said non-natural molecule is herein designated as 'the interferor' or the 'interferor molecule'. Aggregation refers to the fact that a protein which is normally soluble is changed into an insoluble protein or an aggregated protein in its normal biological environment through direct contact or binding with the interferor. The wording 'down-regulating the function of a protein' can also be interchanged by the wording 'knocking down the function of a protein' or 'negatively interfering with the function of a protein'. The down-regulation of the function of a protein can also mean that a protein is not present anymore in a soluble form in the cell or that a protein is not present anymore in a soluble form in its normal biological environment (e.g. (sub)-cellular or extra-cellular localization). In addition, it can also mean that the aggregated protein is degraded through the natural clearance mechanisms of the cell and is no longer detectable in soluble or insoluble form. In addition, it can also mean that a transmembrane receptor protein cannot bind its normal ligand anymore, through interferor induced aggregation of said transmembrane protein. Thus the down-regulation of the function of a protein can also mean that a protein which is a normal resident of..e.g. the mitochondria is not present there anymore through the method of protein interference. In a particular embodiment the 'down-regulation of the function of a protein' or 'the negative interference with the function of a protein' or 'knocking down the function of a protein' is at least a 20%, at least a 30%, at least a 40%, at least a 50%, at least a 60%, at least a 70%, at least a 80%, at least a 90%, at least a 95% or even a 100% loss of function as compared to the normal (100%) function of the protein.<br>
The function of a protein or the lack of presence of a protein in its normal biological environment (localization) can conveniently be determined by methods known in the art. For example, depending on the target protein of interest, the function can be determined by measuring the reduced enzymatic activity. The reduced presence of a protein in its normal biological localization can for example be measured by the lack of formation of a complex, the lack of the occurrence of a target protein in a sub-cellular compartment, the presence of the target protein in soluble form, the presence of the target protein in an aggregated (insoluble is an equivalent term here) form. Alternatively, the effect of the down-reguletion of a target protein can be measured in a cellular assay (e.g. loss or gain of growth, loss or gain of invasion, loss or gain of proteolytic activity).<br>
In a particular embodiment such normal biological activity (or normal function or normal localization) of a protein can be interfered with intracellularly or extracellularly. 'Intracellularly'<br>
refers to the localization of a protein inside the cell of an organism orhost (e.g. the cytoplasm, the mitochondria, the lysosome, the vacuole, the nucleus, the chloroplast, the endoplasmic reticulum (ER), the cellular membrane, the mitochondrial membrane, the chioroplast membrane, ...). 'Extracellularly' not only refers to the localization of a protein in the extracellular medium of the cell but also refers to proteins which contact the extracellular medium such as a membrane-anchored proteins, a transmembrane protein etc. Non-limiting examples of extracellular proteins are secreted proteins (e.g. proteases, antibodies and cytokines present in the blood or plasma) or proteins present in the extracellular matrix (e.g. matrix metalloproteins and transmembrane proteins (e.g. a growth factor receptor)).<br>
Cells or hosts which can be targeted with the method of the invention comprise prokaryotic and eukaryotic cells. Non-limiting examples are viruses, bacteria, yeasts, fungi, protozoa, plants and mammals including humans.<br>
It should be clear that the method of down-regulation the biological function of a protein can be used to interfere with the biological function with 1, 2, 3, 4, 5 or even more proteins simultaneously. Particularly since part B comprises at least one self-association region, part B can for example comprise different self-association regions each specific for a different-protein. The interferor used for interference with the biological function of at least one target protein is not naturally present in nature and can be made through chemical synthesis or through recombinant protein expression or through a combination of the latter.<br>
Thus an interferor molecule comprises at least one self-association region-(thus part B comprises at least one self-association region). A 'self-association region' is herein defined as a contiguous sequence of amino acids that has a high tendency to form a tight molecular assembly with identical or very closely related sequences. The wording 'has a high tendency to form a tight molecular assembly' can also be construed as 'has a high affinity'. Affinity is usually translated into values of dissociation (Kd- values). Kd-values between interferor and target proteins are typically lying between micromolar and nanomolar ranges, but can be sub-nanomolar or supra-micromolar. Examples of self-association regions are intermolecular beta sheet regions, alpha-helical elements, hairpin loops, transmembrane sequences and signal sequences. In a particular embodiment at least one self-association region is present in part B. In another particular embodiment at least two self-association regions are present in part B. In another particular embodiment 3, 4, 5, 6 or more self-association regions are present in part B. Said self-association regions can be interconnected by a linker region (e.g. a spacer of about 2 to about 4 amino acids). One (or at least one) self-association region present in part B is derived from a target protein. In a particular embodiment 2, 3, 4, 5, 6 or more self-association<br>
regions in part B are derived from a target protein. In another particular embodiment 2, 3, 4, 5, 6 or more self-association regions in part B are derived from more than 1 target protein. In another particular embodiment the at least two self-association regions present in part B are derived from the same target protein. The target protein is defined herein as the protein with which one wants to interfere with its function. Thus, in order to make part B specific for at least one protein at least one self-association region in part B should be 'derived from' the target protein or at least one self-association region should be present in said target protein. 'Derived from' means that at least one contiguous self-associating region should be identical or homologous in amino acid sequence to a contiguous region of said target protein. In a preferred embodiment, said at least one self-associating region is at least 60%, at least 65%, at least 70%, at least 75%, at least 80%, at least 85%, at least 90%, at least 95% or at least 100% identical to the self-association region present in said target protein region. It is preferred that the length of a self-association region consists of at least 3 contiguous amino acids. In a preferred embodiment said region consists of about 3 to about 30 amino acids. In another preferred embodiment said region consists of about 3 to about 25 amino acids. In a particularly preferred embodiment said region consists of about 5 to about 20 amino acids.<br>
Self-association regions present in part B of the interferor molecule also can be determined and isolated from proteins other than the target protein and said self-association regions are coupled with at least one self-association region derived from the target protein, optionally with a spacer ( or linker) between said self-association regions. For example, self-association regions that can be used can be derived from self-association regions of proteins which do not normally occur in the host in which the down-regulation of the biological function of a target protein is performed (thus some self-association regions in part B can be taken from an unrelated organism). The nature of the self-association regions determine the level of inhibition (id est the strength of inhibition) of a target protein through induced aggregation. More than one self-association region can be used from a target protein in an interferor molecule but also synthetic self-association regions or self-association regions derived from a different target protein can be used in combination with one or more self-association regions from a target protein.<br>
In a particular embodiment such self-association regions consist of a synthetic sequence which is not derived from existing proteins and hence does not occur in nature. Examples of such synthetic self association regions are described in Lopez de la Paz M. et al (2002) PNAS 99, 25, p. 16053, table 1 which is herein incorporated by reference.<br>
If at least one self-association region (id est the part B of the interferor molecule) has a hydrophobic character (because of its aggregation inducing properties) it is preferably fused (or linked or coupled which are equivalent terms) to a moiety (id est part A of the interferor molecule) that prevents aggregation of said self-association region and exposes said self-association region in direct contact with the solvent in which the interferor is present. As such, in certain embodiments part A has a solubilizing function to keep part B in solution. In such embodiments said part A is for example a peptide, a protein domain, a protein (preferably different from the target protein, see example 2), a glycosylation structure, a (hydrophilic) chemical group or a cyclodextrine or derivative thereof. In certain other embodiments said part A is an agarose bead, a latex bead, a cellulose bead, a magnetic bead, a silica bead, a polyacrylamide bead, a microsphere, a glass bead or any solid support (e.g. polystyrene, plastic, nitrocellulose membrane, glass).<br>
In the interferor molecules part B and part A may be optionally linked (or coupled) by means of a linker region (a spacer is an equivalent word). Said linker region can for instance be an unnatural linker made by chemical synthesis (e.g. a flexible linker such as a hydroxy-substituted alkane chain, dextran, polyethylene glycol or the linker can also consist of amino acid homologues) or said linker can exist of natural amino acids such as a poly(thneonine) or poly(serine). Preferentially when the linker comprises amino acids, the length of said linker region is between about 3 and about 15 amino acids, more preferably between about 5 and about 10 amino acids. Often a flexible linker can be chosen but it is envisaged that a stiff linker will also work. Flexible linker sequences can be taken from nature, mostly such regions connect domains in naturally occurring proteins, such as the linker between the SH2 and SH3 domains src tyrosine kinase or the linker between the BRCT domains of BRCA1.<br>
The term 'contacting' refers to the process in which the interferor and the target protein interact. In one form the interferor is added (e.g. interferor is present at a particular concentration in a solution) to a sample comprising the target protein. In another form the interferor molecule is injected into an organism comprising the target protein. Contacting can for example also be carried out through the process of transformation of a cell comprising the target protein, e.g. an isolated cell, e.g. in cell culture, a unicellular microorganism or a cell or a plurality of cells within a multicellular organism. Transformation implies that the interferor molecule is introduced in a host (e.g. a cell) through commonly known transfection or transformation methods (e.g. by gene transfer techniques including calcium phosphate, DEAE-dextran, electroporation, microinjection, viral methods, the use of cationic liposomes (see for example Feigner, P. L. et al. (1987), Proc. Natl. Acad. Sci USA 84, 7413), commercially available cationic lipid formulations e.g. Tfx 50 (Promega) or Lipofectamin2000 (Life Technologies), particle bombardment, etc.). The interferor molecule may be encoded by a<br>
recombinant vector (e.g. a plasmid, cosmid, viral vector) and can be synthesized inside a host. In an alternative embodiment the interferor molecule can be introduced into a cell through carrier-mediated delivery, e.g. by liposomal carriers or nano-particles or by injection. In yet another alternative embodiment the interferor molecule can enter a cell through a sequence which mediates cell penetration (or cell translocation). In the latter case the interferor molecule is further modified through the recombinant or synthetic attachment of a cell penetration sequence. Thus, the interferor molecule (e.g. as a polypeptide) may be further fused or chemically coupled to a sequence facilitating transduction of the fusion or chemical coupled proteins into prokaryotic or eukaryotic cells. Sequences facilitating protein transduction are known to the person skilled in the art and include, but are not limited to Protein Transduction Domains. Preferably, said sequence is selected from the group comprising of the HIV TAT protein, a polyarginine sequence, penetratin and pep-1. Still other commonly used cell-permeable peptides (both natural and artificial peptides) are disclosed in Joliot A. and Prochiantz A. (2004) Nature Cell Biol. 6 (3) 189-193.<br>
In a particular embodiment the interferor essentially consists of amino acids. In some embodiments the sequences of parts A and B from the interferor molecule are deriv$d4rom the same target protein. In other embodiments the interferor is a chimeric molecule meaning that the sequences from parts A and B are derived from different proteins, e.g. part A is derived from one protein and at least one aggregation region of part B is derived from the target protein. A "Polypeptide" refers to a polymer in which the monomers are amino acids and are joined together through amide bonds, alternatively referred to as a peptide. When the amino acids are alpha-amino acids, either the L-optical isomer or the D-optical isomer can be used. Additionally, unnatural amino acids, for example, beta-alanine, phenylglycine and homoarginine are also included. Commonly encountered amino acids that are not gene-encoded may also be used in the present invention. All or part of the amino acids used in the interferors may be either the D- or L-isomer. In addition, other peptidomimetics are also useful in the present invention. We specifically refer and incorporate herein the review of the development and use of peptidomimetics as antagonists for protein-protein interactions from Sillerud LO and Larson RS (2005) Curr Protein Pept Sci. 6(2):151-69. Furthermore, D-amino acids can be added to the peptide sequence to stabilize turn features (especially in the case of glycine). In another approach alpha, beta, gamma or delta turn mimics (such as alpha, beta, gamma, or delta di-peptides can be employed to mimic structural motifs and turn features in a peptide and simultaneously provide stability from proteolysis and enhance other properties such as, for example, conformational stability and solubility.<br>
Isolation of a self association region from a target protein:<br>
Self-association sequences are often hydrophobic but this is not always the case. For example, the self-associating regions of the yeast prions are rather polar. In fact cross-beta aggregation of an amino acid region derived from a polypeptide or protein can be initiated when (1) it has a high hydrophobicity, (2) it has a good ÃŸ-sheet propensity, (3) it has a low net charge and (4) it is solvent-exposed. Thus, self-association protein regions ('segment' is an equivalent term for 'region') are most often buried in the folded state and are not exposed to the solvent. The latter is confirmed by the experimental finding that in many globular proteins, aggregation occurs during refolding or under conditions in which denatured or partially folded states are significantly populated, i.e. at high concentration or as a result of destabilizing conditions or mutations.<br>
Based on these findings computer algorithms were developed that are able to predict self-association regions ("(3-aggregating stretches or segments" is an equivalent wording) in proteins. One such algorithm, TANGO, is based on a statistical mechanics algorithm that considers the three physico-chemical parameters described above but also considers competition between different, structural conformations: beta-turn, alpha-helix, beta-sheet aggregates and the folded state (Fernandez-Escamilla, AM et al (2004) Nat. Biotechncl. 22, 1302-1306, especially the Methods section on pages 1305 and 1306 are herein specifically incorporated by reference and also the Supplementary Notes 1 and 2 of the same article for further details on the methods and the data sets used for the calibration and the testing of the TANGO algorithm). Thus, self-association regions present in target proteins are obtainable by computer algorithms such as TANGO. Self-association regions are often buried inside the core of the target proteins10, effectively shielding the peptide from intermolecular association by an energy barrier corresponding to the stability of the target proteins11. In its normal environment (e.g. cytoplasm, extracellular matrix) the target protein has assistance from molecular chaperones that assist the protein in keeping its functional, monomeric form12 The model used by the TANGO algorithm6 is designed to predict beta-aggregation in peptides and proteins and consists of a phase-space encompassing the random coil and the native conformations as well as other major conformational states, namely beta-turn, alpha-helix and beta-aggregate. Every segment of a peptide can populate each of these states according to a Boltzmann distribution. Therefore, to predict self-association regions of a peptide, TANGO simply calculates the partition function of the phase-space. To estimate the aggregation tendency of a particular amino acid sequence, the following assumptions are made: (i) in an ordered beta-sheet aggregate, the main secondary structure is the beta-strand, (ii) the regions involved in the aggregation process are fully buried, thus paying full solvation costs and gains, full entropy and optimizing their H-bond potential (that is, the number of H-bonds made in the aggregate is related to the number of donor groups that are compensated by acceptors. An excess of<br>
donors or acceptors remains unsatisfied), (iii) complementary chargesin the selected window establish favorable electrostatic interactions, and overall net charge of the peptide inside but also outside the window disfavors aggregation. TANGO can be accessed on the World Wide Web at http://tango.embl.de/. The zyggregator algorithm is another example (Pawar AP et al (2005) J. Mol. Biol. 350, 379-392). These algorithms identifiy aggregation prone sequences by comparing the aggregation propensity score of a given amino acid sequence with an average propensity calculated from a set of sequences of similar length.<br>
In the present invention we estimate that a self-association region identified within a target protein with a TANGO score of 5% corresponds to an aggregation risk in vitro of 95%6. We have calculated that 85 % of proteins from the human proteome that are not related to disease have at least one region with a TANGO score above the experimentally determined threshold of 5%. This shows that although more than 85% of the human proteins contain at least one single self-association region that aggregation is prevented because of the normal stability of the protein and the assistance from the chaperone machinery. The present invention isolates these self-association regions from target proteins for the preparation of interferes molecules which are used for the specific induction of protein aggregation. The B-part of the interferor molecules comprises at least 1 aggregation region and at least one aggregatiorwjegion is derived from a target protein. It is possible to control the strength of the protein interference (the strength of protein interference is for example the % of loss of biological function of a target protein when said protein or cell comprising said protein is contacted with a specific interferor molecule) through the incorporation of more than one aggregation region of a target protein in the B-part of the interferor molecule. Indeed, aggregation regions derived from a target protein with a low TANGO score (typically between 5 % to about 20%) can be repeated in the B-part of the interferor to 2, 3, 4 or more aggregation regions. As an alternative embodiment 1, 2 or 3 or 4 or more different aggregation regions with a low TANGO score derived from the same protein can be incorporated into the B-part of the interferor. As another alternative embodiment 1, 2, 3, 4 or more synthetic aggregation regions (thus-not derived from the target protein) can be combined with 1, 2, 3, 4, or more aggregation regions derived from the target protein into the B-part to enhance the down-regulation of a target protein with a low TANGO score.<br>
Thus in another embodiment the invention provides a non-naturally occurring molecule capable of aggregating a target protein. In a particular embodiment said non-naturally molecule is proteinaceous in nature. Proteinaceous means that the molecule comprises L-amino acids or D-amino acids or a mixture of L- and D- amino acids or a combination of natural amino acids and peptidomimetics.<br>
In yet another embodiment the invention provides a non-naturally occurring molecule comprising at least one self-association region isolated from a protein domain capable of being soluble' in water wherein said self-association region is fused to a moiety that prevents aggregation of said self-association region.<br>
In yet another embodiment the invention provides a non-naturally occurring molecule comprising at least one self-association region isolated from a protein domain capable of being soluble in water wherein said self-association region is fused to a moiety that prevents aggregation of said self-association region so that said self-association region is in direct contact with the solvent wherein it is present.<br>
In yet another embodiment the invention provides a non-naturally occurring molecule consisting of at least one self-association region isolated from a protein domain capable of being soluble in water wherein said self-association region is fused to a moiety that prevents aggregation of said self-association region.<br>
In yet another embodiment the invention provides a non-naturally occurring molecule consisting of at least one self-association region isolated from a protein domain capable of being soluble in water wherein said self-association region is fused to a moiety that prevents aggregation of said self-association region so that said self-association region is in direct contact with the solvent wherein it is present.<br>
In a particular embodiment such a moiety is for example a peptide, an agarose bead, a protein domain or a protein. In another particular embodiment said non-naturally occurring molecule comprises at least two self-association regions of which at least one self-association region is derived from a target protein.<br>
In other words the invention provides a non-naturally occurring molecule, which comprises part A and part B wherein i) part A comprises a region, such as a peptide, protein domain, protein or agarose bead preventing the aggregation of part B, and ii) part B which comprises at least 1 self-association region wherein said region consists of at least 3 contiguous amino acids and wherein said region is isolated from said protein which function is to be interfered with, and wherein a linker is optionally present between parts A and B.<br>
In still other words the invention provides a non-naturally occurring molecule which comprises part A and part B wherein i) part A comprises a region, such as a peptide, protein domain or agarose bead preventing the aggregation of part B, and ii) part B which comprises at least 1 self-association region consisting of at least 3 contiguous amino acids and wherein at least one self-assocation region is isolated from a protein which function is to be interfered with and wherein said region is isolated from a domain from said protein which is capable of being soluble in water, and wherein a linker is optionally present between parts A and B, and wherein part B is in direct contact to the environment wherein said molecule and said protein are present.<br>
In still other words the invention provides a non-naturally occurring molecule which comprises part A and part B wherein i) part A comprises a region, such as a peptide, protein domain or agarose bead preventing the aggregation of part B, and ii) part B which consists of at least 1 self-association region consisting of at least 3 contiguous amino acids and wherein said at least one self-assocation region is isolated from a protein which function is to be interfered with and wherein said region is derived from a domain from said protein which is capable of being soluble in water, and wherein a linker is optionally present between parts A and B, and wherein part B is in direct contact to the environment wherein said molecule and said protein are present.<br>
The wording 'isolated (or derived form) from a domain from said protein which is capable of being soluble in water' means that a self association region is a contiguous amino acid sequence isolated from a soluble domain of a protein. The latter also means that self-association regions derived from transmembrane regions or self-association regions derived from signal sequences are specifically excluded in the claim scope of these interferor molecule products in such embodiments.<br>
In the present invention the at least one self-association region of the interferor molecule (id est part B of the interferor molecule), is 'in direct contact' with the environment (e.g, solvent, cytosol) in which said interferor molecule is present. The importance of this is clarified further. In globular proteins self-association sequences (also designated as 'aggregation nucleating regions') are generally buried in the hydrophobic core of the globular protein and as such kept protected from the solvent by a dense network of cooperative interactions stabilizing the native state. Hence, under normal circumstances there is no 'direct contact' between said self-association region and the environment (for example the solvent). Only when the protein is unfolded, for example when it is synthesized on the ribosome or destabilized by mutation, change of temperature, pH or loss of a specific chaperone, thereby favoring the unfolded state, will it expose its self-association regions to the environment. Self-association regions are normally buried inside proteins (in order to prevent aggregation) and in*the non-natural interferor molecule said self-association regions have been isolated and exposed to the environment by linking said regions to a moiety that prevents aggregation (id est part A of the interferor molecule). In still other words, the non-naturally interferor molecule does not fold into a globular structure and therefore the at least one self-association region (id est part B) in the non-natural interferor molecule is in direct contact with the solvent in which said interferor molecule is present. Hence, 'in direct contact' refers to the opposite of 'being buried and kept protected from'.<br><br>
In a specific embodiment the interferor molecules that comprise at least one self-association<br>
region derived from a soluble protein domain are polypeptides.<br>
In another specific embodiment the invention provides a recombinant vector comprising a<br>
polynucleotide encoding such interferor molecules.<br>
In another specific embodiment the interferor molecules of the invention are used as a<br>
medicament.<br>
Therapeutic applications of the interferor molecules<br>
Proteins are responsible for biological activities ranging from numerous enzymatic reactions, over transduction of signals to providing structure. Changes in protein structure, abundance or activity are at the root cause or many diseases. Many drugs act via specific interference with one or a limited number of proteins. The present invention provides a method to develop a novel class of compounds able to specifically interfere with a target protein of choice. These novel compounds are designated as interferors.<br>
Thus, in yet another embodiment the invention provides the use as a medicament pf a non-naturally occurring molecule comprising at least one self-association region derived from a protein domain capable of being soluble in water wherein said self-associaticn regios, is fused to a moiety that prevents aggregation of said self-association region.<br>
In yet another embodiment the invention provides the use as a medicament of a non-naturally occurring molecule comprising at least one self-association region derived from a protein domain capable of being soluble in water wherein said self-association region is fused to a moiety that prevents aggregation of said self-association region so that said self-association region is in direct contact with the solvent wherein said molecule is present. In still other words the invention provides the use as a medicament of non-n3turally occurring interferor molecule, which comprises part A and part B wherein i) part A comprises a region, such as a peptide or protein domain preventing the aggregation of part B, and ii) part B comprises at least 1 self-association region wherein said region comprises at least 3 contiguous amino acids derived from the target protein and wherein a linker is optionally present between parts A and B.<br>
In still other words the invention provides the use as a medicament of non-naturally occurring interferor molecule, which comprises part A and part B wherein i) part A comprises a region, such as a peptide or protein domain preventing the aggregation of part B, and ii) part B comprises at least 1 self-association region wherein said region comprises at least 3 contiguous amino acids derived from the target protein and wherein a linker is optionally present between parts A and B and wherein part B is in direct contact to the solvent wherein said interferor molecule is present.<br>
Said interferor molecules can be used for treating diseases and/or ir the manufacturing of a medicament to treat diseases, such as cancer, associated with the aberrant expression of at least one target protein, such as an oncogenic protein. The term 'aberrant expression' refers to for example the (over)expression of an oncogenic protein in the case of cancer, it also includes the expression of a dominant negative protein, the undesired localization of a particular protein or splice variant of a particular protein, the undesired expression of a particular splice variant of a particular protein, the higher activity of a mutant protein or the higher activity of a particular protein.<br>
In a particular embodiment the "aberrant expression" refers to the unwanted presence of a post-translationally modified protein or to the undesired presence of a non-post-translationally modified protein. Post-translational modifications alter the physico-chemical properties of the modified amino acids, and as such they have the potential of altering the aggregation tendency of a given polypeptide segment that can be exploited to specifically target the form that has the strongest aggregation tendency. So if a post-translational modification significantly; decreases the aggregation tendency of the self-association region, then interference will be most efficient with the unmodified protein. In contrast, in case of post-translational modtficatipns that increase the aggregation tendency of the self-association region, then interference will be most efficient with the modified protein. Based on the hydrophobicity alone, it is assumed that modifications such as phosphorylation and glycosylation will decrease aggregation tendency, whereas lipid attachment will increase aggregation tendency.<br>
The target protein to which the interferor molecule of the invention is directed may be associated with a pathological condition. For example, the protein may be a pathogen-associated protein e.g. a viral protein, a tumor-associated protein, or an autoimmune disease-associated protein. In one aspect, the invention features a method of treating^ subject at risk for or afflicted with unwanted cell proliferation, e.g. malignant or non-malignant cell proliferation. The method includes: providing an interferor molecule, e.g. an interferor having a structure as described herein, wherein said interferor molecule is capable of interfering with (inhibiting) the function and/or presence of a protein that promoted unwanted cell proliferation and administering said interferor to a subject, preferably a human subject, thereby treating the subject.<br>
In a preferred embodiment, the protein is a growth factor or growth factor receptor, a kinase (e.g. a protein tyrosine, serine or threonine kinase), an adaptor protein, a protein from the G protein coupled receptor super-family, or a transcription factor. In a preferred embodiment, the interferor molecule interferes with the biological function of the PDGF-beta protein, and thus<br>
can be used to treat a subject having or at risk for a disorder characterized by unwanted PDGF-beta expression, e.g. testicular and lung cancers. In another preferred embodiment the interferor inhibits (knocks down) the function and/or presence of the Erb-B protein, and thus can be used to treat a subject having or at risk for a disorder characterized by unwanted Erb-B expression, e. g. breast cancer. In another preferred embodiment, the interferor inhibits the function (or "interferes with the function" which is equivalent) of (or interferes with the presence of) the Src protein, and thus can be used to treat a subject having or at risk for a disorder characterized by unwanted Src expression, e.g. colon cancers. In another preferred embodiment, the interferor inhibits the function and/or presence of the CRK protein, and thus can be used to treat a subject having or at risk for a disorder characterized by unwanted CRK expression, e.g. colon and lung cancers. In another preferred embodiment, the interferor interferes with the function and/or presence of the GRB2 protein, and thus can be used to treat a subject having or at risk for a disorder characterized by unwanted GRB2 expression, e.g. squamous cell carcinoma. In another preferred embodiment the interferor molecule interferes with the function and/or presence of the RAS gene, and thus can be used to treat a subject having or at risk for a disorder characterized by unwanted RAS expression e.g. pancreatic, colon and lung cancers, and chronic leukemia. In another preferred embodiment thesinterferor molecule interferes with the function and/or presence of the MEKK protein, and thus can be used to treat a subject having or at risk for a disorder characterized by unwanted MEKK expression, e.g. squamous cell carcinoma, melanoma or leukemia. In another preferred embodiment the interferor molecule interferes with the function and/or presence of the JNK protein, and thus can be used to treat a subject having or at risk for a disorder characterized by unwanted JNK expression, e.g. pancreatic or breast cancers. In another preferred embodiment, the interferor molecule interferes with the function and/or presence of the RAF protein and thus can be used to treat a subject having or at risk for a disorder characterized by unwanted RAP expression, e.g. lung cancer or leukemia. In another preferred embodiment, the interferor molecule interferes with the function and/or presence of the Erkl/2 protein, and thus can be used to treat a subject having or at risk for a disorder characterized by unwanted Erkl/2 expression, e.g. lung cancer. In another preferred embodiment the interferor molecule interferes with the function and/or presence of the PCNA (p2l) protein, and thus can be used to treat a subject having or at risk for a disorder characterized by unwanted RCNA expression, e.g. lung cancer. In another preferred embodiment, the interferor molecule interferes with the function and/or presence of the MYB protein, and thus can be used to treat a subject having or at risk for a disorder characterized by unwanted MYB expression, e.g., colon cancer or chronic myelogenous leukemia. In a preferred embodiment the interferor molecule interferes with the function and/or presence of the c-MYC protein, and thus can be used to treat a subject having or at risk for a disorder characterized by unwanted c-MYC expression, e.g., Burkitt's lymphoma<br>
or neuroblastoma. In another preferred embodiment the interferor molecule interferes with the function and/or presence of the JUN protein, and thus can be used to treat a subject having or at risk for a disorder characterized by unwanted JUN expression, e. g. ovarian, prostate or breast cancers. In another preferred embodiment the interferor molecule interferes with the function and/or presence of the FOS protein, and thus can be used to treat a subject having or at risk for a disorder characterized, by unwanted FOS expression, e.g. skin or prostate cancers. In another preferred embodiment, the interferor molecule inhibits the function and/or presence of the BCL-2 protein, and thus can be used to treat a subject having or at risk for a disorder characterized by unwanted BCL-2 expression, e.g., lung or prostate cancers or non-Hodgkin lymphoma. In another preferred embodiment, the interferor molecule interferes with the function and/or presence of the Cyclin D protein, and thus can be used to treat a subject having or at risk for a disorder characterized by unwanted Cyclin D expression, e.g. esophageal and colon cancers. In another preferred embodiment, the interferor molecule interferes with the function and/or presence of the VEGF protein, and thus can be used to treat a subject having or at risk for a disorder characterized by unwanted VEGF expression, e.g., esophageal, colon cancers or pathological angiogenesis. In a preferred embodiment,* the interferor molecule interferes with the function and/or presence of the EGFR proteinr-and thus can be used to treat a subject having or at risk for a disorder characterized by unwanted EGFR expression, e.g. breast cancer. In another preferred embodiment the interferor molecule interferes with the function and/or presence of the Cyclin A protein, and thus can be used to treat a subject having or at risk for a disorder characterized by unwanted Cyclin A expression, e.g., lung and cervical cancers. In another preferred embodiment the interferor molecule interferes with the function and/or presence of the Cyclin E protein, and thus can be used to treat a subject having or at risk for a disorder characterized by unwanted Cyclin E expression, e.g. lung and breast cancers. In another preferred embodiment the interferor molecule interferes with the function and/or presence of the WNT-1 protein, and thus can be used to treat a subject having or at risk for a disorder characterized by unwanted WIWT-1 expression, e.g., basal cell carcinoma. In another preferred embodiment the interferor molecule interferes with the function and/or presence of the beta-catenin protein, and thus can be used to treat a subject having or at risk for a disorder characterized by unwanted beta-catenin expression, e.g., adenocarcinoma or hepatocellular carcinoma. In another preferred embodiment the interferor molecule interferes with the function and/or presence of the c-MET protein, and thus can be used to treat a subject having or at risk for a disorder characterized by unwanted c-MET expression, e.g. hepatocellular carcinoma. In another preferred embodiment the interferor molecule interferes with the function and/or presence of the protein kinase C (PKC) protein, and thus can be used to treat a subject having or at risk for a disorder characterized by unwanted  PKC  expression,   e.g.,   breast cancer.   In  another preferred  embodiment,  the<br>
interferor molecule interferes with the function and/or presence of theNFKappa-B protein, and thus can be used to treat a subject having or at risk for a disorder characterized by unwanted NFKappa-B expression, e.g., breast cancer.<br>
In another preferred embodiment, the interferor molecule interferes with the function and/or presence of the STAT3 protein, and thus can be used to treat a subject having or at risk for a disorder characterized by unwanted STAT3 expression, e.g. prostate cancer. In another preferred embodiment the interferor molecule interferes with the function and/or presence of the survivin protein, and thus can be used to treat a subject having or at risk for a disorder characterized by unwanted survivin expression, e.g. cervical or pancreatic cancers. In another preferred embodiment the interferor molecule interferes with the function and/or presence of the Her2/Neu protein, and thus can be used to treat a subject having or at risk for a disorder characterized by unwanted Her2/Neu expression, e.g. breast cancer. In another preferred embodiment the interferor molecule interferes with the function and/or presence of the topoisomerase I protein, and thus can be used to treat a subject having or at risk for a disorder characterized by unwanted topoisomerase I expression, e.g. ovarian and colon xancers. In another preferred embodiment, the interferor molecule interferes with the function and/or presence of the topoisomerase II alpha protein, and thus can be used to treat a subject having or at risk for a disorder characterized by unwanted topoisomerase II expression, e.g., breast and colon cancers.<br>
In another aspect, the invention provides a method of treating a subject, e.g. a human, at risk for or afflicted with a disease or disorder that may benefit by angiogenesis inhibition, e.g. cancer. The method includes: providing an interferor molecule e.g., an interferor molecule having a structure described herein, which interferor molecule can inhibit (or interfere with the function) a protein which mediates angiogenesis and administering the interferor molecule to a subject, thereby treating the subject. In a preferred embodiment, the interferor molecule interferes with the function and/or presence of the alpha v-integrin protein, and thus can be used to treat a subject having or at risk for a disorder characterized by unwanted alpha v-integrin, e.g. brain tumors or tumors of epithelial origin. In another preferred embodiment, the interferor molecule interferes with the function and/or presence of the Flt-1 receptor protein, and thus can be used to treat a subject having or at risk for a disorder characterized by unwanted Flt-I receptors e.g. cancer and rheumatoid arthritis. In another preferred embodiment, the interferor molecule interferes with the function and/or presence of the tubulin protein, and thus can be used to treat a subject having or at risk for a disorder characterized by unwanted tubulin, e. g., cancer and retinal neovascularization.<br>
In another aspect, the invention provides a method of treating a subject infected with a virus or at risk for or afflicted with a disorder or disease associated with a viral infection. The method<br>
includes: providing an interferor molecule, e.g. an interferor molecule having a structure described herein, which interferor molecule is homologous to and can silence, a viral protein or a cellular protein which mediates viral function, e.g. entry or growth; and administering said interferor molecule to a subject, preferably a human subject, thereby treating the subject. As such the invention provides methods of using interferors for the manufacture of a medicament to treat patients infected by viruses including the Human Papilloma Virus, Human Immunodeficiency Virus (HIV), Hepatitis B Virus (HBV), Hepatitis A Virus (HAV), Hepatitis C Virus (HCV), Respiratory Syncytial Virus (RSV), Herpes Simplex Virus (HSV), Cytomegalovirus (CMV), Epstein Barr-Virus (EBV), a rhinovirus, West Nile Virus, Tick-borne encephalitis virus, measles virus (MV), or poliovirus.<br>
In another aspect, the invention features methods of treating a subject infected with a pathogen, e.g. a bacterial, amoebic, parasitic, or fungal pathogen. The method includes: providing an interferor molecule, e.g. an interferor molecule having a structure described herein, wherein said interferor molecule is capable of interfering with the function of a pathogenic protein derived from said pathogen and administering said interferor molecule-to a subject, preferably a human subject, thereby treating the subject. The target proteitllrom the pathogen can be one involved in growth, cell wall synthesis, protein synthesis, transcription, energy metabolism (e.g. the Krebs cycle) or toxin production. Thus, the present invention provides for a method of treating patients infected by for example Plasmodium falciparum, Mycobacterium ulcerans, Mycobacterium tuberculosis, Mycobacterium leprae, Staphylococcus aureus, Streptococcus pneumoniae, Streptococcus pyogenes, Chlamydia pneumoniae, or Mycoplasma pneumoniae.<br>
In another aspect, the invention provides a method of treating a subject, e.g. a human, at risk for or afflicted with a disease or disorder characterized by an unwanted immune response, e.g. an inflammatory disease or disorder, or an autoimmune disease or disorder. The method includes: providing an interferor molecule, e.g. an interferor molecule having a structure described herein, which interferor molecule is capable of inhibiting (down-regulating) the function and/or presence of a protein which mediates an unwanted immune response, and administering said interferor molecule to a subject, thereby treating the subject. In a preferred embodiment, the disease or disorder is an ischemia or reporfusion injury, e.g. ischemia reperfusion or injury associated with acute myocardial infarction, unstable angina, cardiopulmonary bypass, surgical intervention (e.g. angioplasty, such as percutaneous transluminal coronary angioplasty), a response to a transplanted organ or tissue (e.g. transplanted cardiac or vascular tissue), or thrombolysis. In another preferred embodiment, the disease or disorder is restenosis, e.g. restenosis associated with surgical intervention (e.g.,<br>
angioplasty, such as percutaneous transluminal coronary angioplasty. In another preferred embodiment, the disease or disorder is Inflammatory Bowel Disease e.g. Crohn's Disease or Ulcerative Colitis. In another preferred embodiment, the disease or disorder is inflammation associated with an infection or injury. In another preferred embodiment, the disease or disorder is asthma, lupus, multiple sclerosis, diabetes, e.g. type II diabetes, arthritis, e.g. rheumatoid or psoriatic. In another preferred embodiment the interferor molecule interferes with the function of an integrin or co-ligand thereof, e.g. VLA4, VCAM, ICAM. In another preferred embodiment the interferor molecule interferes with the function of a selectin or co-ligand thereof, e.g. P-selectin, E-selectin (ELAM), L-selectin, or P-selectin glycoprotein-(PSGLI). In another preferred embodiment the interferor molecule interferes with the function of a component of the complement system, e.g., C3, C5, C3aR, C5aR, C3 convertase, C5 convertase. In another preferred embodiment, the interferor molecule interferes with the function of a chemokine or receptor thereof e.g. TNF-a, IL-1Î±, IL-1, IL-2, IL-2R, IL-4, IL-4R, IL-5, IL-6, IL-8, TNFRI, TNFRII, IgE, SCYA11 or CCR3.<br>
In another aspect, the invention provides a method of treating a subject, e.g., a human, at risk for or afflicted with acute pain or chronic pain. The method includes providing an-innterferor molecule, e.g. an interferor molecule having a structure described herein, which interferor molecule is capable of interfering with a protein which mediates the processing of pain and administering said interferor molecule to a subject, thereby treating the subject. In another preferred embodiment the interferor molecule interferes with the function of a component of an ion channel. In another particularly preferred embodiment, the interferor molecule interferes with the function of a neurotransmitter receptor or ligand.<br>
In another aspect, the invention features a method of treating a subject e.g. a human, at risk for or afflicted with a neurological disease or disorder. The method includes providing an interferor molecule, e.g. an interferor molecule having a structure described herein, which interferor molecule is capable of interfering with a protein which mediates a neurological disease or disorder and administering said interferor molecule to a subject, thereby treating the subject. In particular embodiments said diseases (or disorders) which can be treated include Alzheimer's Disease (in this case the interferor molecule interferes with the function of a secretase which leads to the processing of APP e.g. a protein involved in the gamma-secretase complex (e.g. presenilin protein 1 or 2, an Aph1 protein, nicastrin, BACE1 or BACE2). The interferor inhibits the processing of APP and prevents the formation of insoluble amyloid beta. The same strategy can be used to prevent and/or to treat other neurodegenerative diseases such as Huntington's disease, a spinocerebellar ataxia (e.g. SCA1, SCA2, SCA3 (Machado-Joseph disease), SCA7 orSCA8).<br>
Thus in one aspect the invention provides a method for the production or manufacture of a medicament or a pharmaceutical composition comprising at least one interferor molecule and furthermore mixing said interferor molecule with a pharmaceutically acceptable carrier. In a preferred embodiment the interferor molecule is a polypeptide and can be made synthetically or as a recombinant protein. The recombinant protein may be manufactured using recombinant expression systems comprising bacterial cells, yeast cells, animal cells, insect cells, plant cells or transgenic animals or plants. The recombinant protein may be purified by any conventional protein purification procedure close to homogeneity and/or be mixed with additives.<br>
The administration of a pharmaceutical composition comprising an interferor molecule may be by way of oral, inhaled, transdermal or parenteral (including intravenous, intratumoral, intraperitoneal, intramuscular, intracavity, and subcutaneous) administration. The active compound may be administered alone or preferably formulated as a pharmaceutical composition. A unit dose will normally contain 0.01 to 500 mg, for example 0.01 to 50 mg, or 0.01 to 10 mg, or 0.05 to 2 mg of compound or a pharmaceutically acceptable salt thereof.-Unit doses will normally be administered once or more than once a day, for example 2, 3, or 4 times a day, more usually 1 to 3 times a day, such that the total daily dose is normally in the range of 0.0001 to 10 mg/kg; thus a suitable total daily dose for a 70 kg adult is 0.01 to 700 mg, for example 0.01 to 100 mg, or 0.01 to 10 mg or more usually 0.05 to 10 mg.<br>
It is preferred that the compound or a pharmaceutically acceptable salt thereof is administered in the form of a unit-dose composition, such as a unit dose oral, parenteral, transdermal or inhaled composition. Such compositions are prepared by admixture and are suitably adapted for oral, inhaled, transdermal or parenteral administration, and as such may be in the form of tablets, capsules, oral liquid preparations, powders, granules, lozenges, reconstitutable powders, injectable and infusable solutions or suspensions or suppositories onaerosols.<br>
Tablets and capsules for oral administration are usually presented in a unit dose, and contain conventional excipients such as binding agents, fillers, diluents, tabletting agents, lubricants, disintegrants, colourants, flavourings, and wetting agents. The tablets may be coated according to well-known methods in the art. Suitable fillers for use include cellulose, mannitol, lactose and other similar agents. Suitable disintegrants include starch,' polyvinylpyrrolidone and starch derivatives such as sodium starch glycollate. Suitable lubricants include, for example, magnesium stearate. Suitable pharmaceutically acceptable wetting agents include sodium lauryl sulphate. These solid oral compositions may be prepared by conventional methods of blending, filling, tabletting or the like. Repeated blending operations may be used to distribute<br>
the active agent throughout those compositions employing large quantities of fillers. Such operations are, of course, conventional in the art.<br>
Oral liquid preparations may be in the form of, for example, aqueous or oily suspensions, solutions, emulsions, syrups, or elixirs, or may be presented as a dry product for reconstitution with water or other suitable vehicle before use. Such liquid preparations may contain conventional additives such as suspending agents, for example sorbitol, syrup, methyl cellulose, gelatin, hydroxyethylcellulose, carboxymethyl cellulose, aluminium stearate gel or hydrogenatedi edible fats, emulsifying agents, for example lecithin, sorbitan monooleate, or acacia; non-aqueous vehicles (which may include edible oils), for example, almond oil, fractionated coconut oil, oily esters such as esters of glycerine, propylene glycol, or ethyl alcohol; preservatives, for example methyl or propyl p-hydroxybenzoate or sorbic acid, and if desired conventional flavouring or colouring agents. Oral formulations also include conventional sustained release formulations, such as tablets or granules having an enteric coating.<br>
Preferably, compositions for inhalation are presented for administration to the respiratory tract as a snuff or an aerosol or solution for a nebulizer, or as a microfine powder for insufflation, alone or in combination with an inert carrier such as lactose. In such a case the particles of active compound suitably have diameters of less than 50 microns, preferably less than 10 microns, for example between 1 and 5 microns, such as between 2 and 5 microns. Alternatively, coated nanoparticles can be used, with a particle size between 30 and 500 nm. A favored inhaled dose will be in the range of 0.05 to 2 mg, for example 0.05 to 0.5 mg, 0.1 to 1 mg or 0.5 to 2 mg.<br>
For parenteral administration, fluid unit dose forms are prepared containing a compound of the present invention and a sterile vehicle. The active compound, depending on the vehicle and the concentration, can be either suspended or dissolved. Parenteral solutions are normally prepared by dissolving the compound in a vehicle and filter sterilising before filling into a suitable vial or ampoule and sealing. Advantageously, adjuvants such as a local anaesthetic, preservatives and buffering agents are also dissolved in the vehicle. To enhance the stability, the composition can be frozen after filling into the vial and the water removed under vacuum. Parenteral suspensions are prepared in substantially the same manner except that the compound is suspended in the vehicle instead of being dissolved and sterilised by exposure to ethylene oxide before suspending in the sterile vehicle. Advantageously, a surfactant or wetting agent is included in the composition to facilitate uniform distribution of the active compound.    Where    appropriate,    small    amounts    of    bronchodiiators    for    example<br>
sympathomimetic amines such as isoprenaline, isoetharine, salbutamol, phenylephrine and ephedrine; xanthine derivatives such as theophylline and aminophylline and corticosteroids such as prednisolone and adrenal stimulants such as ACTH may be included.<br>
As is common practice, the compositions will usually be accompanied by written or printed directions for use in the medical treatment concerned.<br>
In a preferred embodiment the interferor molecule further comprises a protein transduction domain. It has been shown that a series of small protein domains, termed protein transduction domains (PTDs), cross biological membranes efficiently and independently of transporters or specific receptors, and promote the delivery of peptides and proteins into cells. For example, the TAT protein from human immunodeficiency virus (HIV-1) is able to deliver biologically active proteins in vivo. Similarly, the third alpha-helix of Antennapedia homeodomain, and VP22 protein from herpes simplex virus promote the delivery of covalently linked peptides or proteins into cells (reviewed in Ford KG et al (2001) Gene Ther. 8, 1-4). Protein delivery based on a short amphipathic peptide carrier, Pep-1, is efficient for delivery of a variety of peptides and proteins into several cell lines in a fully biologically active form, without the neeisl for prior chemical covalent coupling (Morris MC et al, (2001) Nat. Biotechnoi. 19, 1173-1176). The capacity of VP22 chimeric proteins to spread from the primary transduced cell to surrounding cells can improve gene therapy approaches (Zender L ef al (2002) Cancer Gene Ther. 9, 489-496). Sequences, facilitating protein transduction are known to the person skilled in the art and include, but are not limited to Protein Transduction Domains. Preferably, said sequence is selected from the group comprising of the HIV TAT protein, a polyarginine sequence, penetratin and pep-1. Still other commonly used cell-permeable peptides (both natural and artificial peptides) are disclosed in Joliot A. and Prochiantz A. (2004) Nature Cell Biol. 6 (3) 189-193.<br>
A second aspect of a pharmaceutical composition is the use of a nucleotide sequence encoding the interferor molecules. In case a nucleic acid sequence encoding the interferor molecule is used, said medicament is preferably intended for delivery of said nucleic acid into the cell, in a gene therapy treatment. A large number of delivery methods are well known to those of skill in the art. Preferably, the nucleic acids are administered for in vivo or ex vivo gene therapy uses. Non-viral vector delivery systems include DNA plasmids, naked nucleic acid, and nucleic acid complexed with a delivery vehicle such as a liposome. Viral vector delivery systems include DNA and RNA viruses, which have either episomal or integrated genomes after delivery to the cell. Methods of non-viral delivery of nucleic acids include lipofection, microinjection, biolistics, virosomes, liposomes, immunoliposomes, polycation or<br>
Iipid:nucleic acid conjugates, naked DNA, artificial virions, and agent-eahanced uptake of DNA. Lipofection is described in, e.g., US Pat. No. 5,049,386, US Pat No. 4,946,787; and US Pat. No. 4,897,355 and lipofection reagents are sold commercially (e.g., Transfectamâ„¢ and Lipofectinâ„¢). Cationic and neutral lipids that are suitable for efficient receptor-recognition lipofection of polynucleotides include those of Flegner, WO 91/17424, WO 91/16024. Delivery can be to cells (ex vivo administration) or target tissues {in vivo administration). The preparation of lipid: nucleic acid complexes, including targeted liposomes such as immunolipid complexes, is well known to one of skill in the art (see, e.g., Crystal, 1995; Blaese et al., 1995; Behr, 1994; Remy et al., 1994; Gao and Huang, 1995; U.S. Pat. Nos. 4,186,183, 4,217,344, 4,235,871, 4,261,975, 4,485,054, 4,501,728, 4,774,085, 4,837,028, and 4,946,787).<br>
The use of RNA or DNA viral based systems for the delivery of nucleic acids take advantage of highly evolved processes for targeting a virus to specific cells in the body and trafficking the viral payload to the nucleus. Viral vectors can be administered directly to patients {in vivo) or they can be used to treat cells in vitro and the modified cells are administered to patients (ex vivo). Conventional viral based systems for the delivery of nucleic acids include amongst others retroviral, lentivirus, adenoviral, adeno-associated and herpes simplex virus vectors for gene transfer. Viral vectors are currently the most efficient and versatile method of gene transfer in target cells and tissues. Integration in the host genome is possible with the retrovirus, lentivirus, and adeno-associated virus gene transfer methods, often resulting in long-term expression of the inserted transgene. Additionally, high transduction efficiencies have been observed in many different cell types and target tissues. In cases where transient expression of the nucleic acid is preferred, adenoviral based systems, including replication deficient adenoviral vectors may be used. Adenoviral based vectors are capable of very high transduction efficiency in many cell types and do not require cell division. With such vectors, high titer and levels of expression have been obtained. This vector can be produced in large quantities in a relatively simple system. Adeno-associated virus ("AAV") vectors, including recombinant adeno-associated virus vectors are also used to transduce cells with target nucleic acids, e.g., in the in vitro production of nucleic acids and peptides, and for in vivo and ex vivo gene therapy procedures {see, e.g., U.S. Patent No. 4,797,368; WO 93/24641; Kotin, 1994; The construction of recombinant AAV vectors is described in a number of publications, including U.S. Pat. No. 5,173,414; Hermonat &amp; Muzyczka, 1984; Samulski er al., 1939).<br>
Gene therapy vectors can be delivered in vivo by administration to an individual patient, typically by systemic administration (e.g., intravenous, intraperitoneal, intramuscular, intratracheal, subdermai, or intracranial infusion) or topical application. In a particular embodiment the invention also envisages the use of a hydrodynamic gene therapeutic method.<br>
Hydrodynamic gene therapy is disclosed in US6627616 (Mirus Corporation, Madison) and involves the intravascular delivery of non-viral nucleic acids encoding an interferor whereby the permeability of vessels is increased through for example the application of an increased pressure inside said vessel or through the co-administration of vessel permeability increasing compounds such as for example papaverine.<br>
Alternatively, vectors can be delivered to cells ex vivo, such as cells explanted from an individual patient (e.g., lymphocytes, bone marrow aspirates, tissue biopsy) or universal donor hematopoietic stem cells, followed by reimplantation of the cells into a patient, usually after selection for cells which have incorporated the vector. Ex vivo cell transfection for diagnostics, research, or for gene therapy (e.g., via re-infusion of the transfected cells into the host organism) is well known to those of skill in the art. In a preferred embodiment, cells are isolated from the subject organism, transfected with a nucleic acid (gene or cDNA), and re-infused back into the subject organism (e.g., patient). Various cell types suitable for ex vivo transfection are well known to those of skill in the art (see, e.g., Freshney et al., 1994 and the references cited therein for a discussion of how to isolate and culture cells from patients).<br>
In yet another embodiment the method of protein interference of the invention may bo used for determining the function of a protein in a cell or an organism being capable of mediating protein interference. The cell can be a prokaryotic cell or can be a eukaryotic cell or can be a cell line, e.g. a plant cell or an animal cell, such as a mammalian cell, e.g. an embryonic cell, a pluripotent stem cell, a tumor cell, e.g. a teratocarcinoma cell or a virus-infected cell. The organism is preferably a eukaryotic organism, e.g. a plant or an animal, such as a mammal, particularly a human.<br>
The target protein to which the interferor molecule of the invention is directed may be associated with a pathological condition. For example, the protein may be a pathogen-associated protein, e.g. a viral protein, a tumor-associated protein or an autoimmune disease-associated protein. The target protein may also be a heterologous gene expressed in a recombinant cell or a genetically altered organism. By inhibiting the function of such a protein valuable information and therapeutic benefits in the agricultural field or in the medicine or veterinary medicine field may be obtained. In a particularly preferred embodiment the method of the invention is used with an eukaryotic cell or a eukaryotic non-human organism exhibiting a target protein-specific knockout phenotype comprising an at least partially deficient expression of at least one endogenous target protein wherein said cell or organism is contacted with at least one interferor molecule capable of inhibiting the function of at least one endogenous target protein or with a vector encoding at least interferor molecule capable of<br>
interfering with the function and/or presence of at least one endogenous protein. It should be noted that the present invention also allows a target-specific knockout of several different endogenous proteins due to the specificity of the interferor molecule.<br>
Protein-specific knockout phenotypes of cells or non-human organisms, particularly of human cells or non-human mammals may be used in analytic procedures, e.g. in the functional and/or phenotypical analysis of complex physiological processes such as analysis of proteomes. For example, one may prepare the knock-out phenotypes of human proteins in cultured cells which are assumed to be regulators of alternative splicing processes. Among these proteins are particularly the members of the SR splicing factor family, e.g. ASF/SF2, SC35, SRp20, SRp40 or SRp55. Further, the effect of SR proteins on the mRNA profiles of predetermined alternatively spliced genes such as CD44 may be analysed.<br>
Using the protein based knockout technologies described herein, the expression of an endogenous target protein may be inhibited in a target cell or a target organism. The endogenous protein may be complemented by an exogenous target nucleic acid coding foj the target protein or a variant or mutated form of the target protein, e.g. a gene or axDMA, which may optionally be fused to a further nucleic acid sequence encoding a detectable peptide or polypeptide, e.g. an affinity tag, particularly a multiple affinity tag. Variants or mutated forms of the target protein differ from the endogenous target protein in that they differ from the endogenous protein by amino acid substitutions, insertions and/or deletions of single or multiple amino acids. The variants or mutated forms may have the same biological activity as the endogenous target protein. On the other hand, the variant or mutated target protein may also have a biological activity, which differs from the biological activity of the endogenous target protein, e.g. a partially deleted activity, a completely deleted activity, an enhanced activity etc. The complementation may be accomplished by co-expressing the polypeptide encoded by the exogenous nucleic acid, e.g. a fusion protein comprising the target protein and the affinity tag and the interferor molecule for knocking out the endogenous protein in the target cell. This co-expression may be accomplished by using a suitable expression vector expressing both the polypeptide encoded by the exogenous nucleic acid, e.g. the tag-modified target protein and the interferor molecule or alternatively by using a combination of expression vectors or alternatively the interferor molecule may contact the target cell from the outside of the cell. Proteins and protein complexes which are synthesized cfe novo in the target cell will contain the exogenous protein, e.g. the modified fusion protein. In order to avoid suppression of the exogenous protein function with the interferor molecule, the exogenous protein must have sufficient amino acid differences in the aggregation region that is selected for the design of the interferor molecule. Alternatively, the endogenous target protein may be complemented<br>
by corresponding proteins from other species, or the endogenousetarget protein may be complemented by a splice form of said target protein. The combination of knockout of an endogenous protein and rescue by using mutated, e.g. partially deleted exogenous target has advantages compared to the use of a knockout cell. Further, this method is particularly suitable for identifying functional domains of the target protein.<br>
In a further preferred embodiment a comparison, e.g. of gene expression profiles and/or proteomes and/or phenotypic characteristics of at least two cells or organisms is carried out. These organisms are selected from: (i) a control cell or control organism without target protein inhibition, (ii) a cell or organism with target protein inhibition and (iii) a cell or organism with target protein inhibition plus target protein complementation by an exogenous target nucleic acid encoding said target protein.<br>
The methods of the invention are also suitable in a procedure for identifying and/or characterizing pharmacological agents, e.g. identifying new pharmacological agents from a collection of test substances and/or characterizing mechanisms of action and/or side effects of known pharmacological agents. Thus, the present invention also relates to a system for identifying and/or characterizing pharmacological agents acting on at least one target protein comprising: (a) a eukaryotic cell or a eukaryotic non-human organism capable of expressing at least one endogenous target gene coding for said target protein, (b) at least one interferor molecule capable of inhibiting the expression of said at least one endogenous target gene, and (c) a test substance or a collection of test substances wherein pharmacological properties of said test substance or said collection are to be identified and/or characterized. Further, the system as described above preferably comprises: (d) at least one exogenous target nucleic acid coding for the target protein or a variant or mutated form or splice form of the target protein wherein said exogenous target protein differs from the endogenous target protein on the amino acid level of the aggregation regions such that the function of the exogenous target protein is substantially less inhibited by the interferor molecule than the expression of the endogenous protein.<br>
In addition, the invention also comprises cells and organisms comprising an interferor molecule. An organism can for example be a transgenic plant which carries the genetic information that encodes an interferor. Such a transgenic plant is in a preferred embodiment a silenced plant (id est in which a particular target protein is down-regulated because of the presence of a specific interferor in a sub-set of cells or organs or present in all cells and organs of said plant). Cells comprising an interferor can be produced by contacting said cells or by electroporation of said cells with a particular interferor molecule. In a particular embodiment cells comprising an interferor are generated through transfection (or transformation) wherein<br>
the interferor is encoded by a recombinant expression vector such as a plasmid or a viral vector.<br>
ISOLATION: SEPARATION and DETECTION<br>
In another embodiment the invention provides a method to isolate a protein from a sample comprising contacting said sample with a non-naturally occurring molecule comprising at least one self-association region present in said protein and isolating the resulting co-aggregated molecule-protein complex from said sample.<br>
In yet another embodiment the invention provides a method to isolate a protein from a sample comprising contacting said sample with a non-naturally occurring molecule comprising at least one self-association region isolated from said protein wherein said self-association region is fused to a moiety that prevents aggregation of said self-association region and isolating the resulting co-aggregated molecule-protein complex from said sample.<br>
In yet another embodiment the invention provides a method to isolate a protein from a sample comprising contacting said sample with a non-naturally occurring molecule comprising at least one self-association region isolated from said protein wherein said self-association region is fused to a moiety that prevents aggregation of said self-association region so that .said self-association region is in direct contact with the solvent wherein said self-association region fused to said moiety and said protein are present and isolating the resulting co-aggregated molecule-protein complex from said sample.<br>
In other words the invention provides a method for the isolation of a protein from a sample<br>
comprising:<br>
contacting said protein with a non-naturally occurring molecule which comprises part A and part B wherein i) part A comprises a peptide, protein domain or agarose bead, preventing the aggregation of part B, and ii) part B which comprises at least 1 self-association region wherein said region consists of at least 3 contiguous amino acids and wherein said region is isolated from said protein and wherein a linker is optionally present between parts A and B and isolating the resulting co-aggregated molecule-protein complex from said sample.<br>
In still other words the invention provides a method for the isolation of a protein from a sample<br>
comprising:<br>
contacting said protein with a non-naturally occurring molecule which comprises part A and part B wherein i) part A comprises a peptide, protein domain or agarose bead, preventing the aggregation of part B, and ii) part B which comprises at least 1 self-association region wherein said region consists of at least 3 contiguous amino acids and wherein said region is isolated from said protein and wherein a linker is optionally present<br>
between parts A and B, and wherein part B is in direct contact to the environment<br>
wherein said molecule and protein are present and<br>
isolating the resulting co-aggregated molecule-protein complex from r.aid sample.<br>
In still other words the invention provides a rqethod for the isolation of a protein from a sample<br>
comprising:<br>
contacting said protein with a non-naturally occurring molecule which comprises part A and part B wherein i) part A comprises a peptide, protein domain or agarose bead, preventing the aggregation of part B, and ii) part B which consists of at least 1 self-association region wherein said region consists of at least 3 contiguous amino acids and wherein said region is isolated from said protein and wherein a linker is optionally present between parts A and B, and wherein part B is in direct contact to the environment wherein said molecule and protein are present and isolating the resulting co-aggregated molecule-protein complex frcm said sample.<br>
In still other words the invention provides a method for the isolation of a protein from a sample<br>
comprising:<br>
contacting said protein with a non-naturally occurring molecule which comprises part A and part B wherein i) part A comprises a peptide, protein domain or agarose bead, preventing the aggregation of part B, and ii) part B which consists of at least 1 self-association region wherein said region consists of at least 3 contiguous amino acids and wherein said region is isolated from said protein and wherein a linker is optionally present between parts A and B and isolating the resulting co-aggregated molecule-protein complex from said sample.<br>
SEPARATION<br>
In a further embodiment the method for the isolation of at least one protein further comprises<br>
the separation of at least one protein from a sample.<br>
One application of the separation of at least one protein from a sample is the removal (or depletion) of highly abundant proteins from a sample. Indeed, a major challenge in protein target discovery and validation is how to specifically dissect complex protein samples (e.g. plasma, urine, cerebrospinal fluid) and measure trace targets. The abundant proteins are often 6-10 orders of magnitude more concentrated than low abundant proteins. Thus, highly abundant proteins must be removed to detect and measure trace proteins of medical importance. Since albumin, IgG, antitrypsin, IgA, transferrin and haptoglobin make up approximately 90% of the total protein content in human serum, there is a critical need for diagnostic tools to rapidly deplete these unwanted abundant proteins and unmask the less<br>
abundant, low molecular weight protein biomarkers. Several methods-are already used in the art: 1) immunoglobulin G (IgG) as affinity reagents to capture and separate abundant protein targets, 2) immunoglobulin yolk (IgY) are IgG-like antibodies isolated from egg yolks of immunized birds, 3) pre-fractionation is used to separate a mixture of proteins into different fractions to remove certain proteins in the original mixture, and 4) protein A and protein G are bacterial cell wall proteins with a specificity to IgG antibodies, hence protein A and G affinity resins provide a removal of IgG and 5) IgG- and IgY-microbeads are used for protein detection.<br>
DETECTION<br>
In another specific embodiment the method for the isolation of at least one protein further<br>
comprises the detection of at least one protein in said molecule-protein complex.<br>
Detection can be carried out by separating the interferor molecule-target protein complex by for example electrophoresis, column chromatography, filtration, electrostatic attraction, magnetic or paramagnetic attraction, mass spectrometry and the like.<br>
The most broadly used biodetection technologies are based on the use of antibodies. Antibodies recognize and bind to other molecules based on their shape and phy&amp;icochemical properties. Antibodies are highly suited for detecting small quantities of target proteins in the presence of complex mixtures of proteins. The present invention shows that the use ,of interferor molecules (part B has the specificity and recognition for at least one specific protein) is an alternative for the use of antibodies (as the recognition element) for the specific capture of target proteins. Indeed, interferor molecules can be used in numerous applications in which antibodies typically are used. To name only a few, applications are envisaged in diagnosis, micro- analytics, forensics and in the specific detection of pathogens.<br>
For the detection and separation applications of the invention it is preferred that part B of the interferor molecule is bound to a carrier which is herein designated as part A. A carrier can be a flat surface such as plastic or nitrocellulose or a chromatographic column but is preferably a bead such as microsphere beads. A general discussion on various types of beads and microspheres, which serve the purpose of being part A of the interferor molecules, is described on pages 9 and 10 of US6682940 and is herein specifically incorporated by reference.<br>
In a particular embodiment part A of the interferor molecule is a carbohydrate type of carrier, e.g. cellulose or agarose. Part B can be covalently bound to said carbohydrate carrier with a cross-linking agent such as glutaraldehyde.<br>
In another particular embodiment part A is a support such as cellulose, glass or a synthetic polymer. Covalent attachment between part A and part B can be carried out via amino acid residues of part B and an azide, carbodiimide, isocyanate or other chemical derivatives present on part A.<br>
In yet another particular embodiment part A is a porous silanised glass micro bead. Part B can be covalently bonded to part A via its peptide amine groups (by Schiff reaction followed by reduction with sodium borohydride) to aldehyde groups formed by periodate oxidation of glycidoxypropylsilane groups chemically linked to the silica atoms (this coupling is described in Sportsman and Wilson (1980) Anal. Chem. 52, 2013-2018).<br>
In a specific embodiment the carrier part A is enveloped by a proteinaceous film to which part A is crosslinked (see claims 1-50 and examples relating to the carrier in US4478946).<br>
In another specific embodiment part A is a fluorescent bead such as a fluorescent latex particle. The patent US4550017, and especially page 4 therein, describes fluorescent compounds which can be used for the manufacturing of fluorescent beads.<br>
In another specific embodiment the beads, part A, vary in size and may also contain or be impregnated with fluorescent dyes. Because of varying sizes and dyes of the beads multiple proteins can be detected and quantitated in a single reaction. Procedures for the development of such beads are described in US6159748.<br>
In yet another particular embodiment the coupling between part A (the bead) and part B is via a poly(threonine), a poly(serine), dextran or poly(ethylene glycol). Examples 6, 7, 8 and 9 of US6399317 illustrate how this coupling can be carried out.<br>
In yet another particular embodiment part A is a magnetic bead. Magnetic beads, coupling between the magnetic beads and a protein agent and their uses are described on page 8 of application US6489092.<br>
Definitions<br>
Unless defined otherwise, all technical and scientific terms used herein have the same meaning as commonly understood by those of ordinary skill in the art to which the invention belongs. Although any methods and materials similar or equivalent to those described herein can be used in the practice or testing of the present invention, preferred methods and<br>
materials are described. For the purposes of the present invention,,the following terms are defined below.<br>
The articles "a" and "an" are used herein to refer to one or to more than one (i.e. to at least one) of the grammatical object of the article. By way of example, "a target protein" means one target protein or more than one target protein.<br>
As used herein, the term "about" refers to a quantity, level, value, dimension, size, or amount that varies by as much as 30%, preferably by as much as 20%, and more preferably by as much as 10% to a reference quantity, level, value, dimension, size, or amount.<br>
"Bifunctional crosslinking reagent" means a reagent containing two reactive groups, the reagent thereby having the ability to covalently link two elements such as part A and part B of the interferor molecule. The reactive groups in a crosslinking reagent typically belong to the classes of functional groups including succinimidyl esters, maleimides and halcacetamides such as iodoacetamides. Throughout this specification, unless the context requires otherwise, the words "comprise", "comprises" and "comprising" will be understood to imply the inclusion of a stated step or element or group of steps or elements but not the exclusion of any-other step or element or group of steps or elements.<br>
By "expression vector" or "recombinant vector" is meant any autonomous genetic element capable of directing the synthesis of an interferor molecule encoded by the vector. Such expression vectors are known to practitioners in the art.<br>
By "derivative" is meant an interferor molecule that has been derived from the basic sequence by modification, for example by conjugation or complexing with other chemical moieties (e.g. pegylation) or by post-translational modification techniques as would be understood in the art. The term "derivative" also includes within its scope alterations that have been made to a parent sequence including additions, or deletions that provide for functionally equivalent molecules.<br>
By "effective amount", in the context of modulating an activity or of treating or preventing a condition is meant the administration of that amount of an interferor molecule to an individual in need of such modulation, treatment or prophylaxis, either in a single dose or as part of a series, that is effective for modulation of that effect or for treatment or prophylaxis of that condition. The effective amount will vary depending upon the health and physical condition of the individual to be treated, the taxonomic group of individual to be treated, the formulation of<br>
the composition, the assessment of the medical situation, and other relevant factors. It is expected that the amount will fall in a relatively broad range that can be determined through routine trials.<br>
By "isolated" is meant material that is substantially or essentially free from components that normally accompany it in its native state. For example, an "isolated polypeptide", as used herein, refers to a polypeptide, which has been purified from the sequences which flank it in a naturally-occurring state, e.g., a self-association sequence which has been removed from the sequences that are normally adjacent to said sequence. A self-association sequence (optionally coupled to a moiety that prevents aggregation) can be generated by amino acid chemical synthesis or can be generated by recombinant production.<br>
The term "oligonucleotide" as used herein refers to a polymer composed of a multiplicity of nucleotide units (deoxyribonucleotides or ribonucleotides, or related structural variants or synthetic analogues thereof) linked via phosphodiester bonds (or related structural variants or synthetic analogues thereof). An oligonucleotide is typically rather short in length, generally from about 10 to 30 nucleotides, but the term can refer to molecules of any length, although the term "polynucleotide" or "nucleic acid" is typically used for large oligonucleotides: The term "polynucleotide" or "nucleic acid" as used herein designates mRNA, RNA, cRNA, cDNA or DNA. The term typically refers to oligonucleotides greater than 30 nucleotides in length.<br>
The term "recombinant polynucleotide" as used herein refers to a polynucleotide,formed in vitro by the manipulation of nucleic acid into a form not normally found in nature. For example, the recombinant polynucleotide may be in the form of an expression vector. Generally, such expression vectors include transcriptional and translational regulatory nucleic acid operably linked to the nucleotide sequence.<br>
By "operably linked" is meant that transcriptional and translational regulatory nucleic acids are positioned relative to a polypeptide-encoding polynucleotide in such a manner that the polynucleotide is transcribed and the polypeptide is translated.<br>
The terms "subject" or "individual" or "patient", used interchangeably herein, refer to any subject, particularly a vertebrate subject, and even more particularly a mammalian subject, for whom therapy or prophylaxis is desired. Suitable vertebrate animals that fall within the scope of the invention include, but are not restricted to, primates, avians, fish, reptiles, livestock animals (e.g., sheep, cows, horses, donkeys, pigs), laboratory test animals (e.g., rabbits, mice, rats, guinea pigs, hamsters), companion animals (e.g., cats, dogs) and captive wild animals<br>
(e.g., foxes, deer, dingoes). However, it will be understood that the aforementioned terms do not imply that symptoms are present.<br>
By "pharmaceutically acceptable carrier" is meant a solid or liquid filler, diluent or encapsulating substance that can be safely used in topical or systemic administration to a patient,<br>
"Polypeptide", "peptide" and "protein" are used interchangeably herein to refer to a polymer of amino acid residues and to variants and synthetic analogues of the same. Thus, these terms apply to amino acid polymers in which one or more amino acid residues is a synthetic non-naturally occurring amino acid, such as a chemical analogue of a corresponding naturally occurring amino acid, as well as to naturally-occurring amino acid polymers.<br>
By "recombinant polypeptide" is meant a polypeptide made using recombinant techniques, i.e., through the expression of a recombinant or synthetic polynucleotide. When the chimeric polypeptide or biologically active portion thereof is recombinantly produced, it is also preferably substantially free of culture medium, i.e., culture medium represents less than about 20%, more preferably less than about 10%, and most preferably less than about 5% of the-volume of the protein preparation.<br>
The term "sequence identity" as used herein refers to the extent that sequences are identical on a nucleotide-by-nucleotide basis or an amino acid-by-amino acid basis over a window of comparison. Thus, a "percentage of sequence identity" is calculated by comparing two optimally aligned sequences over the window of comparison, determining the number of positions at which the identical nucleic acid base (e.g., A, T, C, G, I) or the identical amino acid residue (e.g., Ala, Pro, Ser, Thr, Gly, Val, Leu, He, Phe, Tyr, Trp, Lys, Arg, His, Asp, Glu, Asn, Gin, Cys and Met) occurs in both sequences to yield the number of matched positions, dividing the number of matched positions by the total number of positions in the window of comparison (i.e., the window size), and multiplying the result by 100 to yield the percentage of sequence identity. For the purposes of the present invention, "sequence identity" will be understood to mean the "match percentage" calculated by the DNASIS computer program (Version 2.5 for windows; available from Hitachi Software engineering Co., Ltd., South San Francisco, Calif., USA) using standard defaults as used in the reference manual accompanying the software. "Similarity" refers to the percentage number of amino acids that are identical or constitute conservative substitutions. Similarity may be determined using sequence comparison programs such as GAP (Deveraux et al. 1984, Nucleic Acids Research 12, 387-395). In this way, sequences of a similar or substantially different length to those cited herein might be<br>
compared by insertion of gaps into the alignment, such gaps being determined, for example, by the comparison algorithm used by GAP.<br>
The term "transformation" means alteration of the genotype of an organism, for example a bacterium, yeast or plant, by the introduction of a foreign or endogenous nucleic acid. Vectors for transformation include plasmids, retroviruses and other animal viruses, YACs (yeast artificial chromosome), BACs (bacterial artificial chromosome) and the like. By "vector" is meant a polynucleotide molecule, preferably a DNA molecule derived, for example, from a plasmid, bacteriophage, yeast or virus, into which a polynucleotide can be inserted or cloned. A vector preferably contains one or more unique restriction sites and can be capable of autonomous replication in a defined host cell including a target cell or tissue or a progenitor cell or tissue thereof, or be integrable with the genome of the defined host such that the cloned sequence is reproducible. Accordingly, the vector can be an autonomously replicating vector, i.e., a vector that exists as an extrachromosomal entity, the replication of which is independent of chromosomal replication, e.g., a linear or closed circular plasmid, an extrachromosomal element, a minichromosome, or an artificial chromosome. The vector can contain any means for assuring self-replication. Alternatively, the vector can be one which, when introduced into the host cell, is integrated into the genome and replicated together with the chromo30me(s) into which it has been integrated. A vector system can comprise a single vector or plasmid, two or more vectors or plasmids, which together contain the total DNA to be introduced into the genome of the host cell, or a transposon. The choice of the vector will typically depend on the compatibility of the vector with the host cell into which the vector is to be introduced. In a preferred embodiment, the vector is preferably a viral or viral-derived vector, which is operably functional in animal and preferably mammalian cells. The vector can also include a selection marker such as an antibiotic resistance gene that can be used for selection of suitable transformants. Examples of such resistance genes are known to those of skill in the art and include the nptll gene that confers resistance to the antibiotics kanamycin and G418 (GeneticinÂ®) and the hph gene which confers resistance to the antibiotic hygramycin B.<br>
Unless otherwise defined, all technical and scientific terms used herein have the same meaning as commonly understood by one of ordinary skill in the art to which this invention belongs. Although methods and materials similar or equivalent to those described herein can be used in the practice or testing of the present invention, useful methods and materials are described below. The materials, methods, and examples are illustrative only and not intended to be limiting. Other features and advantages of the invention will be apparent from the detailed description and from the claims.<br>
Examples<br>
1.	Design of a synthetic peptide interferor molecule<br>
We constructed an interferor molecule wherein part B consists of three synthetic self-association regions with short linkers of two amino acids (STLIVL-QN-STVIFE-QN-STVIFE) interconnecting the self-association regions. Said three self-association regions are hexapeptides which have a strong tendency to aggregate, see Fig. 1 for the design of the interferor molecule. Note: in the text of this invention all amino acid sequences are depicted starting from the amino-terminal part and read in the direction of the carboxy-terminal part -thus, "STLIVL" reads as "NH2-STLIVL-COOH"). Part B of the synthetic interferor molecule was N-terminally fused to a moiety (part A) that prevents aggregation and brings the self-association regions in direct contact with the environment (here the cytosol of E. coli). (Fig. 1 depicts the structure of the synthetic interferor design). Said moiety is the NusA protein, which is frequently used as a solubilising tag in recombinant protein production13. The resulting synthetic interferor molecule (A-B structure) could be made and purified in a recombinant way in E. coli.<br>
We have shown that overexpression of the synthetic interferor molecule (without any specific self-association sequence specific for a particular E. coli protein) does not prevent bacterial growth. Therefore, BL21 E. coli cells were transformed with the synthetic interferor construct present in the pETM60 plasmid (gift from G. Stier, EMBL). In the latter plasmid the interferors are under control of the chimeric T7 promoter (see materials and methods section). Recombinants were grown to a density of 0.6 OD, the interferor was expressed upon the addition of 0.5H uM IPTG for 3 hours at 37 Â°C and the bacterial suspension was plated on agar plates. The plates were inspected after 12h incubation at 37 Â°C and showed abundant bacterial growth.<br>
In a next step a flexible linker sequence ("KPGAAKG" - depicted as 'linker'-in figure 1) was coupled to the COOH-terminus of the synthetic interferor construct to allow fusion of self-association sequences derived from a target protein.<br>
2.	Protein interference in prokaryotes<br>
In the present example E. coli proteins were chosen to be down-regulated of which a functional protein interference confers a selectable trait. Target proteins from the E. coli proteome were selected with a cytosolic localisation and the presence of an aggregation region with a suitable high TANGO score. Since conditional auxotrophy for a single amino acid can conveniently be tested using growth media with controlled composition, we selected four candidate enzymes involved in the synthesis of isoleucine (UniProt15 entry: ILVI_ECOLI), methionine (UniProt15<br><br>
entries: METE_ECOLI and METK_ECOLI) and leucine (UniProt15 entry: LEU1_ECOLI). The self-association sequences based on the TANGO prediction score of the four target proteins was for ILVI_ECOLI: "GWLVTSG", TANGO score: 44, for METE_ECOLI: "LLLTTYF", TANGO score: 32 , METK_ECOLI: ""LTLLV", TANGO score: 20 and LEU1_ECOLI: "LAFIG", TANGO score: 15. The genetic information for the synthetic interferor molecule of example 1 was fused to the DNA-sequence encoding the respective self-association regions of the four biosynthetic enzymes resulting in four specific interferor molecules. To show in vivo protein interference (which is essentially a co-aggregation between the specific interferor (for a biosynthetic enzyme) and the biosynthetic enzyme itself) we proceeded as follows. E. coli were transformed with the plasmid comprising the respective interferor constructs and grown in rich medium until the start of the exponential growth phase, when interferor protein expression was induced with IPTG (isopropyl-beta-D-thiogalactopyranoside). Protein expression was allowed to proceed at 37 Â°C, cells were harvested, washed with a salt solution to remove excess IPTG and rich medium and plated on agar plates containing minimal M9 medium completed with the twenty naturally occurring amino acids (called M9 complete medium) and on minimal M9 medium with all amino acids except the one for which auxotrophy is being tested (called M9 select). It was shown that for three out of four targeted enzymes, a complete functional knockout could be achieved, i.e. conditions were found in which the bacteria formed colonies on M9 complete but not on M9 select agar plates. Expression of the interferor constructs and their exclusive presence in the insoluble phase of the cell-lysate was confirmed by western blot. For the four enzymes tested here, a clear relationship is observed between their sensitivity for the co-aggregation approach and their predicted aggregation propensity according to the TANGO algorithm, further confirming the quality of the TANGO predictions as well as their relevance in a functional cellular context. The ILVI protein, which has the highest TANGO score, was almost completely knocked out by the leaky expression from the T7 promoter in the absence of any IPTG, and one hour of induced overexpression leads to a full functional knock-out. The METE and METK enzymes display an intermediate TANGO score and were not affected by a single hour of overexpression of the interferor. However after three hours of IRTG induction the function was lost completely and no colony formation could be detected. The weakest aggregation score was observed for the LEU1 enzyme and overexpression in this case produced only a modest down-regulation of its activity. Remarkably the functional knock-out of the targeted enzymes is reversible. When cells loaded with a high level of overexpressed interferor material were plated on LB agar, they displayed normal colony growth. When these colonies were copied unto M9 select, normal growth was again observed. This indicates that during colony growth the aggregates were lost, reinstating the cellular network to normal.<br>
3. Protein interference of targets comprising self-association regions with a low self-association score<br>
Self association regions are often flanked or contain charged residues such as R, K, D and E but also P and G (so called gatekeeper amino acid residues) (see Rousseau, Serrano &amp; Schymkowitz (2006) How Evolutionary Pressure Against Protein Aggregation Shaped Chaperone Specificity, J Mol Biol, doi:10.1016/j.jmb.2005.11.035). These gate keeper residues reduce the self association propensity of the sequences they are associated to. In order to optimize the sensitivity of the B-part of an interferor molecule to co-aggregate with a given target protein, the self association region of the target protein that is included in the B-part of the interferor can be mutated so that the above mentioned residues are replaced by aggregation promoting residues such as L,V,I,F,W,Y but also other residues that can increase the self-association propensity of the self association region can be included. The mutated self association region (derived from the target protein) which is included in the B-part of the interferor molecule has a sequence homology of at least 60%, preferably at least 70%, more preferably at least 80% and most preferably at least 90%, with the self aggregation, region of the target protein. In addition, some amino acids are neutral in terms of aggregation,, and replacing said amino acids with amino acids that favour aggregation will also increase the self-association tendency of a region (for example S,T,C but also Q,N,H,M can be replaced by aggregation prone residues such as L,V,I,F,W,Y. These optimized integrator rnoiecules increase the protein interference of targets with a predicted low aggregation score. In example 2 we showed that the protein interference of the LEU1 enzyme of E. coli is less efficient. In this example we optimize the interferor molecule with a specificity for the LEU1 enzyme. The identified self association sequence in LEU1 is "LAFIG". The latter sequence is flanked by the gatekeeper residues "...DYDLEALAFIGKQQEE..." therefore in order to mutationaly enhance the target sequence we employ a strategy based on degenerate per as follows: complementary primers are designed that have an overlap of 20-25 bp on each side of the codon that will be mutated to allow efficient annealing of the primers ot the template. A degenarate codon is introduced by incorporating an equimolar ratio of the four bases during primer synthesis (a so called NNS codon). Using the Quickchange PCR protocol with this degenerate primer, a library containing the 20 point mutations of the flanking position is obtained. This library is amplified in Top10 cells (Invitrogen) and the plasmid DNA purified using the miniPrep kit (Qiagen). To test if knockout efficiency is increased, mutant interferors (design is carried out as in example 2) against the LEU1 target are transformed into BL21 cells (Invitrogen) and plated on LB agar plates. In a 96 well plate containing 0.2 mL LB + antibiotic per well, each well is inoculated by picking individual colonies. The plate is incubated at 37 Â°C until an OD of 0.6 is reached, when expression of the mutant interferors is induced by adding 0.5H uM IPTG for 3 hours at 37 CC. The complete content of each well, except 1 uL is plated<br>
on a selective minimal medium that contains all amino acids except leucine. For clones that that show different gradations of impaired growth on the selective plates, TempliPhi reagent (GE Health Science) is added for DNA amplification and the plate is transferred to the sequencing facility. The sequence information provides us the full spectrum of optimized LEU1 mutated interferor molecules.<br>
4.	Use of interferor molecules for the depletion of immunoglobulin G from serum<br>
In this depletion experiment an agarose bead is chosen as a moiety (part A) to which self-association regions derived from a target protein are fused via amino reactive chemical linking. Such agarose materials are commercially available, such as NHS-activated Sepharoseâ„¢ 4 Fast Flow from GE healthcare. Human immunoglobulin G has two strong tango regions (I) IIVAWIATAVAAIVAAWALIY and II) LTVLLLLASA) that can be used as self-association regions. Since peptide expense scales with length in amino acids, peptides are designed to contain 10 amino acid fractions from the first target region. The target sequences (self association regions) are preceded with the linker sequence ADPRGAAEGA and synthesised with unprotected ends to maintain the reactive N-terminal amino group. The designed sequences are a) ADPRGAAEGAIIVAWIATA, b) ADPRGAAEGAWIATAVAAI, c) ADPRGAAEGAIVAAWALIY (a), b) and c) comprise decapeptides derived from the strong tango region I) and d) ADPRGAAEGALTVLLLLASA comprises tango region II). For depletion, 10 ml of serum is incubated with 1 mg of immobilised peptide at 25, 30, 37 and 45 Â°C for 1 hour. Agarose beads are collected by centrifugation and the serum is removed, agarose beads are washed in PBS buffer in order to remove remaining impurity. The beads are subsequently transferred to SDS buffer and incubated at 95 Â°C for 10 minutes and vortexed extensively. The , -presence of IgG is investigated using SDS-PAGE. The identity of target is confirmed using mass spectrometry.<br>
5.	Use of interferor molecules for detection<br>
Interferor molecules were designed for the specific detection of 3 commercially available recombinant proteins (citrate synthase from porcine heart (Roche), beta-galactosidase from E. coli (Sigma) and glucose-6-phosphate dehydrogenase from Leuconostoc mcsenteroides (Sigma). Thereto in a first step self-association regions were determined with the TANGO algorithm from the following target proteins:<br>
a)	citrate synthase (CISY_PIG): "ALFWLLVT" (TANGO score 60),<br>
b)	beta-galactosidase (BGAL_ECOLI): "AVIIWSLGN" (TANGO score 30) and "ALAWLQ"<br>
(TANGO score 42), and<br>
c) glucose-6-phosphate dehydrogenase (G6PD_LEUME): "AFVDAISAVYTA" (TANGO score<br>
41).<br>
In a next step biotine was amino-terminally coupled to the 4 different self-associaiion regions resulting in 4 different interferor molecules: i) biotine-ALFWLLVT with a specificity for citrate synthase, ii) biotine-AVIIWSLGN and biotine-ALAWLQ with a specificity for beta-galactosidase and iii) biotine-AFVDAISAVYTA with a specificity for glucose-6-phosphate dehydrogenase. Biotinylated peptides were acquired from Jerine Peptide Technologies. Note that the interferor design: biotine - self-association region corresponds with the A-B structure wherein biotine (part A) prevents the aggregation of the self-association region (part B) and brings the self-association region in direct contact with the solvent (PBST) wherein said biotine-self-association region is present.<br>
Individual dotblots were prepared by spotting 0.3 mg of each target protein on nitrocellulose membrane, followed by air drying and overnight incubation in 1% BSA-PBST (PBS with 0.1% Tween-20) to block non-specific binding sites. The membrane was immersed in a 10 mM solution of biotinylated detection peptide and incubated for 3 hours at room temperature with agitation. After repeat washing with buffer, the binding of the peptide to the preiein was confirmed by visualisation of the biotin moiety using streptavidin-HRP (Horse Radish Peroxidase, Pierce) and chemiluminescence detection using a CCD camera system.<br>
6. Use of interferor molecules against murine VEGF for the treatment of pathological retinal angioqenesis<br>
Retinal neovascularization is a major cause of blindness in the world and pathological retinal angiogenesis is a final common pathway leading to vision loss in diseases such as retinopathy of prematurity (ROP), diabetic retinopathy andNage-related macular degeneration. It is known that vascular endothelial growth factor (VEGF) is one of the key players in the development of pathological angiogenesis. We study the effect of interferor molecules against VEGF in two murine induced retinopathy models. In a first model neonatal mice (with an immature retinal vasculature) are exposed to hyperoxia, resulting in obliteration of the developing b'ood vessels supplying oxygen to the retina. When the mice are then returned to normoxia, the retina, distal to the occluded vessels, becomes ischemic, inducing VEGF production and ultimately resulting in reproducible and quantifiable proliferative retinal neovascularization (the model is detailed in Pierce EA et al (1995) Proc. Natl. Acad. Sci. 92(3)905-9, see experimental procedures on page 905 - mouse model). In short, mouse pups of 7 days (P7) together with their nursing mothers, are subjected to hyperoxia (75% oxygen) in specially designed oxygen chambers for 5 days, without opening the cages. On P12, the animals are returned to room air until P17, when the retinas are assessed for maximal neovascular response. On P12 half of the animals<br>
are treated with an interferor molecule against VEGF and half are left untreated. Half of the treated mice receive VEGF interferor by way of intravitreous injection while the other half of the treated group receive VEGF interferor by way of periocular injection (the periocular or intravitreous injection is carried out as described in Shen J et al (2006) Gene Therapy, advance online publication 29 September). Three different interferor molecules against the murine VEGF165 isoform are used in a concentration range of 1-100 Âµg/ml.<br>
a)	REAG-FLLSWVHWTLALLLYLHH-GGEERAG; this interferor molecule has the A-B-A"<br>
structure of an interferor molecule. The self association region derived from murine VEGF165<br>
(underlined) is flanked by solubilizing regions A (REAG and GGEERAG), or in other words the<br>
regions A and A' prevent the aggregation of the self-association region (B-part of the interferor<br>
molecule).<br>
b)	STVIIE-GGAG-NHN/TLS-GGAGQ-FLLSVWHWTLALLLYLHH-GEftAG; this interferor<br>
molecule has the B-A structure of an interferor molecule. The solubilizing part A (GERAG) is<br>
shown in italics. The B-part has the following structure: STVIIE (=synthetic self association<br>
region) - GGAG (=a linker) - NHVTLS(=synthetic self association region) - GGAGQ (=a linker)<br>
- FLLSWVHWTLALLLYLHHG (=the self association region derived from murine VEÂ£F165).<br>
c)	STVIIE-GGAG-FLLSvWHWTLALLLYLHH-GERAG: this interferor molecule has the B-A<br>
structure of an interferor molecule. The solubilizing part A (GERAG) is shown in italics. The B-<br>
part has the following structure: STVIIE (=synthetic self association region) - GGAG (=a linker<br>
between the self association regions) - FLLSWVHWTLALLLYLHH (=the self association<br>
region derived from murine VEGF165).<br>
On P17 anesthetized mice are perfused through the left ventricle with 1 ml of phosphate buffered saline containing 50 mg of 2x106 molecular weight fluorescein-dextran. The eyes are removed and fixed in 4% paraformaldehyde for 3 (right eye) or 24 (left eye) hrs. Of the right eyes, lenses are removed and peripheral retinas cut to allow flat mounting with glycerol-gelatin. The flat mounted retinas are analyzed by fluorescence microscopy. The left eyes are embedded in paraffin and serial 6 urn sections are cut sagittally throughout the cornea, parallel to the optic nerve, and stained with hematoxylin-eosin. The proliferative neovascuiar response is quantified by counting the number of new vessels (= tufts) and the number of endothelial cells extending from the internal limiting membrane of the retina into the vitreum on the stained sagittal cross-sections. The angiographic technique using fluorescein-dextran perfusion is used in conjunction with this counting method for rapid screening of retinas or as an alternative grading system for quantitative evaluation. In a second model retinal neovascularization is experimentally mimicked by laser-induced venous thrombosis in the retina. The model is described in Saito Y et al (1997) Curr. Eye Res. 16(1): 26-33. Chi-Chun Lai et al (2005) Acta<br>
Ophtalmologica Scandinavica 83:590-594, describe in the materials and methods section on pages 591-592 that the model can be quantitated. Application of the VEGF interferor molecules is carried out as described herein before.<br>
7.	Protein interference in a human cell line<br>
The modulation of apoptosis (induction or suppression) is easy to monitor in a cellular system. It is known that staurosporin induces apoptosis in a p53 dependent manner. Thus, the down-regulation of p53 or the down-regulation of proteins which enhance the function of p53 (e.g. ASPP1) suppresses staurosporin induced apoptosis in animal (e.g. human) cell lines. Recombinant expression vectors, that encode interferor molecules, are constructed based on the design of the synthetic interferor molecule described in example 1, except that the A part, the NusA protein, is changed to the green fluorescent protein (GFP) and that the promoter is a constitutive mammalian promoter such as the actin or the CMV promoter. The self-association sequence for p53 is ILTIITLE (which has a tango score of 72) and this sequence is additionally comprised into the B-part of the synthetic interferor leading to an interferor molecule with a specificity for p53. The self-association sequence for ASPP1 is MILTVFLSN (which as a tango score of 63) and this sequence is additionally comprised into the B-part of the -,synthetic interferor molecule leading to an interferor molecule with a specificity for ASPP1. HeLa cells are cultured, and transfected with the recombinant vectors. The GFP (A-part) allows the visualisation of the over-expressed interferor molecules. Addition of 1 uM staurosporin to transfected and non-transfected control cells induces a differential apoptotic response.<br>
8.	Protein interference of vascular endothelial growth factor (VEGF) in zebrafish<br>
Interferor molecules were developed that directed to zebrafish VEGF. Specific inactivation<br>
(through aggregation) of the secreted VEGF can be followed by a disturbance of.the vascular<br>
development in zebrafish embryos.<br>
In a first step the self association regions present in the zebrafish VEGF protein were<br>
determined with the TANGO algorithm. The aggregating region with the highest TANGO-score<br>
is NH2-FLAALLHLSA-COOH. Based on this self association sequence we developed four<br>
synthetic interferor molecules:<br>
Interferor A: NH7-RLFLAALLRFLAALLHLSAR-COOH<br>
Interferor B: NH7-RFLAALLHLSARLFLAALLR-COOH<br>
Interferor C: NH2-RYLAILAGIRLFLAALLR-COOH<br>
Interferor D: NH?-RYLAILAGIRFLAALLHLSAR-COOH<br>
Interferor E (NH2-EALWYLIQLAGR-COOH) served as a control sequence and is derived from<br>
a sequence outside this high TANGO region<br>
Note that interferors A, B and D comprise the whole TANGO-region while interferor C comprises only a part of the TANGO-region. The sequences derived from the TANGO-regions are underlined.<br>
These interferor molecules were added to the medium of transgenic Tg(fli1:EGFP)y1 zebrafish embryos at different concentrations. Transgenic Tg(fH1:EGFP)y1 zebrafish express enhanced Green Fluorescent Protein (GFP) in their endothelial cells, said fish are described in Lawson ND and Weinstein BM (2002) Dev. Biol. 248, 307-318), are provided by the Zebrafish International Resource Center (University of Oregon) and are maintained as described in the zebrafish book, a guide for the laboratory use of zebrafish (Danio rerio), Univ. Oregon Press, Eugene, 1994).<br>
Dechorionated embryos of 20 hours post fertilization (hpf) were arrayed in 24-well plate (10 embryos/well) and exposed at several concentrations of the selected interferor molecules (starting at 50 uM) for 24 hours. Live embryos were analyzed at 28 and 48 hpf (hours post fertilization) using confocal imaging which performed using a Zeiss laser-scanning microscope LSM510.<br>
Although monitoring the development of different vascular structures, we paid partipular attention to (i) the structure of the dorsal aorta (DA), posterior cardinal vein (PCV), (ii) the sprouting of the intersomitic vessels (ISV) and the formation of the vascular plexus (RV) in the posterior region of the trunk. A summary of the dose-dependant experiments is shown in Table 1. It is clear that interferors A and C induce clear vascular defects in the developing zebrafish larvae. Surprising is the fact that the interferor molecules are taken up by the zebrafish larvae through the skin and that no injection of the interferors is necessary. Interferors B and D need to be administered at still lower concentrations in order to be able to monitor the specific vascular defects.<br>
9. Protein interference in the yeast Saccharomyces cerevisiae<br>
We have used the knock-down of the yeast Ura3 enzyme to show that protein interference works in eukaryotes because targeted inactivation (through aggregation) of the Ura3 protein gives an easy readout. The S. cerevisiae Ura3 enzyme is an essential enzyme involved in the uracil biosynthesis pathway. Hence, S. cerevisiae mutants lacking the URA3 gene are not able to grow on medium without uracil but growth can be restored by addition of uracil to the medium. In a first step the self associations regions present in the Ura3 protein were determined with the TANGO algorithm. The self association region (or the aggregation region) with the best TANGO-score is NH2-VIGFIAQ-COOH (TANGO-score: 74). This peptide sequence was used to generate an  interferor expression construct.  To  clone the self<br>
association sequence encoding this peptide in frame with the synthetic interfesor construct (see example 1) we used the following two oligonucleotides:<br>
URA3aggregatorFor: 5' CCTCTAGAATGAAAGAAATTTTGGCTGTAG 3' and URA3aggregatorRev:  5'  CCGTCGACTTA AGC TTG AGC  AAT AAA  GCC  GAT  AAC GCCAGCAGCGCCCGGTTTAGCAGC 3'.<br>
The Xba\ and Sail restriction sites are underlined. The start codon of the NusA protein is highlighted in bold. The stop codon is depicted in italics and the sequence encoding the seven Ura3 amino acids is depicted in bold.<br>
As a template for the PCR we have used the pETM60 plasmid (gift from G. Stier, EMBL) that contains the NusA protein coupled to the synthetic interferor/linker construct (see example 1). This vector contains a T7 promotor, confers kanamycin resistance and provides an N-terminai expression tag of six histidines. The resulting PCR product was subcloned in tho pBEVY/GT vector (Miller CA 3rd, Martinat MA and Hyman LE (1988) Nucleic Acids Res. 26: 3577-3583) using the Xba\ and Sa/I restriction sites, in this vector the "NusA-synlhetic injerferor-Hnker-Ura3 self association sequence" - expression cassette was under the contro!Â«5f the S. cerevisiae GAL1/10 promoter. The selection marker of this vector is the TRP1 gene. Sequence verified constructs with and without the DNA encoding the seven Ura3 amino acids (derived from the self association region) were introduced into the S. cerevisiae strain PVD2 by transformation and selection of transformants was based on trpl complementation..The PVD2 strain is derived from the W303-1A strain (Thomas BJ and Rothstein R. (1989) Cell 56, 619-630) but the PVD2 strain is transformed with wild type alleles of both HIS3 and URA3. The PVD2 strain is still auxotrophic for leucine {LEU2), tryptophane (TRP1) and adenine (ADE2). Transformants were selected on SDglu-Trp medium (minimal yeast medium with 2% glucose but without tryptophane). Colonies were re-streaked on fresh SDglu-Trp plates for single colonies. Two independent colonies were grown overnight in liquid SDglu-Trp medium. The OD600 of the culture was then adjusted to 1 and 5 microliters of a 10-fold serial dilution were spotted on SDglu-Ura-Trp (SD-medium with 2% glucose but without uracil and without tryptophane) or SDgal-Ura-Trp (SD-medium with 2% galactose but without uracil and without tryptophane) plates. The result of this experiment is shown in Fig. 2. This experiment has been repeated three times with similar results. Expression of the empty pBEVY/GT vector or the vector expressing only the NusA-synthetic interferor construct (without a self-association sequence from ura3) does not show any growth inhibition on medium without uracil. Expression of the NusA-synthetic interferor-Ura3 association region construct, however, strongly inhibits growth on medium without uracil (when uracil is added to the growth medium<br>
there is no growth defect), showing that the endogenous Ura3 protein is specifically inactivated by protein interference.<br>
10. Protein interference in the yeast Candida albicans<br>
Candida albicans causes 40% of the fungal infections in humans. This commensal possesses a number of virulence factors. Apart from the capacity to adhere to all kinds of plastics (a major problem in intensive care units) or the production of lipases and proteinases, it is the capacity to adopt various morphologies that has been studied most extensively because it is one of the major virulence factors. Many transcription factors can induce the transition from yeast-like cells to hyphae or pseudohyphae. Other transcription factors are required to keep the cells in the yeast-like form. One example of such a repressor of hyphal formation is Tup1. As an example of protein interference in Candida albicans we have used the Tup1 protein as a target. Down-regulation of the biological function of Tup1 should induce hyphal formation. In a first step the self association regions present in the Tup1 protein were determined with the TANGO algorithm. The self association region (or the aggregation region) with the best TANGO score (TANGO-score: 30) is NH2-VISVAVSL-COOH. This peptide was used to generate an interferor expression construct. To clone the self association sequence-encoding this peptide in frame with the synthetic interferor construct of example 1, we used the-following two oligonucleotides:<br>
TUP1 aggregatorFor:<br>
5'ATTGTACAAATATCCGTATGTGCCTGACTACGCAATGGCTCAGTGGCAGAAC 3' and TUPIaggregatorRev: 5' GCGCTAGC7LA AGC TAG GGA TAC AGC GAC TGA GAT GAC GCCAGCAGCGCCCGGTTTA 3'.<br>
The SsrGI and Nhe\ sites are underlined. The stop codon is depicted in italics and the reverse sequence encoding the target peptide is depicted in bold.<br>
We have used the pETM60 plasmid containing the NusA-synthetic interferor construct (from example 1) as a template for the PCR. The resulting PCR product was subcloned in the pPCK1-GFP plasmid (Barelle CJ, Manson CL, MacCallum DM, Odds F, Gow NAR, Brown AJP. Yeast 21:333-340, 2004) using the SsrGI and Nhe\ restriction sites. In this vector the interferor construct was cloned in frame with the GFP gene present on the vector and the resulting interferor expression cassette "GFP-synthetic interferor-linker-'Tupl self-association region") is under the control of the PCK1 promoter. In the latter construct green fluorescent protein (GFP) is replaced by NusA and GFP serves as part A of the interferor molecule. The PCK1 promoter is strongly induced in casamino acids containing medium and repressed in glucose containing medium (Leuker CE, Sonneborn A, Delbruck S, Ernst JF. Gene 192:235-<br>
240, 1997). Sequence verified plasmids were then transformed into the C. albicans strain CAW (Fonzi WA, Irwin MY. Genetics 134:717-728, 1993). Transformants were selected on SDglu-ura (yeast minimal medium comprising 2% glucose but without uracil). The transformants were grown overnight in glucose containing minimal medium, the cells were diluted to obtain about 20 cells/100 microliter and this volume was plated on SDglu-ura or SDcasamino acid-ura agar plates. Colony morphology was scored after 4 and 6 days of growth (see Fig. 3). As can be seen in Fig. 3, down-regulation of Tup1 occurs in medium with casamino acids and hyphal formation is clearly visible at the edge of the colonies. Hyphal formation is not seen in the control transformants ((pPCK1-GFP plasmid without interferor expression cassette)) neither on the medium comprising glucose. The example shows that the endogenous Tup1 is specifically inactivated by protein interference.<br>
11. Application of protein interference in plants<br>
We demonstrate protein interference in tobacco BY2-cells by using already transformed BY2-cells with several GFP-fusion genes (said genes are depicted in Table 2). A list of said Arabidopsis thai/ana genes together with their corresponding identified self-association regions and tango scores is depicted in table 2.<br>
Specific interferor molecules against each of the targets of table 2 are designed based on the synthetic interferor molecule described in example 1, except that the NusA protein is changed by the Red Fluorescent Protein (RFP) and that the B-part additionally comprises the specific self association regions of the targets depicted in table 2.<br>
Constructs encoding said specific interferor molecules are introduced in appropriate vectors for over-expression using the Gatewayâ„¢ technology (InVitrogen Life Technologies). To this end a set of Gateway compatible binary vectors for plant transformation was developed. For over-expression the pK7WGD2 vector is used in which the gene is put under the control of the p35S promoter. For plant cell transformations the ternary vector system is applied. The plasmid pBBR1MCS-5.virGN54D is used as a ternary vector. The binary plasmid is introduced into Agrobacterium tumefaciens strain LBA4404 already bearing the ternary plasmid by electro-transformation. Fresh BY-2 culture is established before the transformation with the particular construct. Five-day-old BY-2 is inoculated 1:10 and grown for three days (28 Â°C, 130 rpm, dark). The liquid culture of Agrobacterium tumefaciens transformed with pK7WGD2-GUS (control vector), pK7WGD2-interferor 1 (e.g. specific for aurora 1, pK7WGD2-interferor 2 (e.g. specific for aurora 2) etc. etc. is established two days before the transformation of BY-2. A loopfull of bacteria from the solid medium is inoculated in 5 ml of liquid LB medium with the antibiotics (rifampicin, gentamycin, streptomycin and spectinomycin). The culture is grown for two days (28 Â°C, 130 rpm). The transformation of BY-2 is performed in empty petri dish (0 4,6<br>
cm) with the cocultivation method. Three-day-old BY-2 (3 ml) is pipetted into plate and either<br>
50 or 200 (0,1 of bacterial suspension was added. The plates are gently mixed and left to stand in the laminar bench in the dark for three days. After cccultivation the cells are plated on the solid BY-2 -medium with the selections (50 Âµg/ml kanamycin, and 500 Âµg/ml vancomycin and 500 Âµg/ml carbenicillin to kill the excess of bacteria). The plates are sealed with Millipore tape and incubated at 28 Â°C in the dark for approximately two weeks after which the calli become visible. The efficiency of protein interference (here the aggregation between the GFP-construct and the RFP-construct) is visualised by checking the expression of GFP, RFP and the co-localisation of GFP and RFP under the fluorescence microscope.<br>
Material and methods related to examples 1 and 2<br>
Constructs, cells and media<br>
The interferor molecules were cloned into the vector pETM60 (gift from G. Stier, EMBL). This vector is under RNA polymerase T7 control (the T7 RNA polymerase is under the control of regulatory elements from the E. coli lac operon), confers kanamycin resistance and provides an N-terminal expression tag of six histidines followed by NusA. A series of overlapping oligos coding for the sequence of interferor part B were used to create a 'synthetic genel by PCR. This gene was ligated into pETM60 via Nco1 and BamH1 restriction sites. Interferor genes were created by PCR of interferor part B, using a long, anti-coding oligo to include the sequence for the flexible linker and the protein specific self-association region (the bait) at the C-terminus. These were ligated into pETM60 via Nco1 and BamH1 restriction sites. All oligos were purchased from Sigma-Genosys, all restriction enzymes from Fermentas, and ligations were performed using the Quick Ligation Kit from Roche.<br>
Chemically competent BL21 (DE3) cells were produced in house and transformed following standard protocols. Standard LB-agar plates were prepared, and all agar plates contained 50 ug/mL kanamycin. M9 complete plates were prepared by supplementing standard M9 media with all 20 amino acids (50 ug/mL) and nucleotides adenine, guanine, uracil and xanthine (20 Âµg/mL). M9 select plates were identical to M9 complete in all but one amino acid. To control for possible amino acid degradation upon storage, plates were prepared one day before use. LB was prepared following standard protocols and kanamycin was included at 50 Âµg/mL. All amino acids were purchased from Sigma, kanamycin and IPTG from Duchefa.<br>
Protocols<br>
BL21 (DE3) cells were transformed with the expression constructs, plated on LB-agar plus kanamycin and incubated at 37 Â°C overnight. A single colony was used to inoculate 10 mL LB plus kanamycin and this was grown overnight at 37 Â°C, shaking. The following day this culture<br>
was used to inoculate 10 mL LB plus kanarnycin 1:100 and this was grown until an OD60Q of 0.6 was obtained. Cultures were then divided in two; IPTG (50 uM) was added to one culture to induce interferor expression, and both cultures were further incubated at 37 Â°C for the desired expression time. Cells were then harvested by centrifugation and washed twice (by resuspension and centrifugation) with salt solution (0.85% w/v NaCI). Cells were then resuspended in salt solution to give a final OD600 of 0.05, and 200 uL of this cell suspension was plated onto agar plates. Agar plates were incubated at 37 Â°C overnight and colony growth was noted the following day. Where necessary, colonies were picked with a sterile tooth pick onto fresh agar plates.TABLES<br><br>
(Table Removed)<br>
Table 1: Summary of the dose-response effects of different VEGF interferor molecules (used in example 8) on zebrafish vasculature<br><br>
(Table Removed)<br>
Table 2: proteins derived from Arabidopsis thaliana, identified self association regions and corresponding tango scores<br>
References<br>
1.	Dobson, C. M. Protein-misfolding diseases: Getting out of shape. Nature 418, 729-730 (2002).<br>
2.	Dobson, C. M. Principles of protein folding, misfolding and aggregation. Sernin Cell Dev Biol 15, 3-16(2004).<br>
3.	Nelson, R. et al. Structure of the cross-beta spine of amyloid-like fibrils. Mature 435, 773-8 (2005).<br>
4.	Makin, O. S., Atkins, E., Sikorski, P., Johansson, J. &amp; Serpell, L. C. Molecular basis for amyloid fibril formation and stability. Proc Natl Acad Sci U S A 102, 315-20 (2005).<br>
5.	Hamada, D., Yanagihara, I. &amp; Tsumoto, K. Engineering amyloidogenicity towards the development of nanofibrillar materials. Trends Biotechnol 22, 93-7 (2004).<br>
6.	Fernandez-Escamilla, A. M., Rousseau, F., Schymkowitz, J. &amp; Serrano, L. Prediction of sequence-dependent and mutational effects on the aggregation of peptides and proteins. Nat Biotechnol 22, 1302-6 (2004).<br>
7.	Chiti, F., Stefani, M., Taddei, N., Ramponi, G. &amp; Dobson, C. M. Rationalization of the effects of mutations on peptide and protein aggregation rates. Nature 424, 505-8 (2003).<br>
8.	Pawar, A. P. et al. Prediction of "aggregation-prone" and "aggregation^sftsceptible" regions in proteins associated with neurodegenerative diseases. J Mol Biol 3Â£0, 379-92 (2005).<br>
9.	Lopez de la Paz, M. &amp; Serrano, L. Sequence determinants of amyloid fibril formation. Proc Natl Acad Sci U S A 101, 87-92 (2004).<br>
10.	Linding, R., Schymkowitz, J., Rousseau, F., Diella, F. &amp; Serrano, L. A comparative study of the relationship between protein structure and beta-aggregation in globular and intrinsically disordered proteins. J Mol Biol 342, 345-53 (2004).<br>
11.	Clark, L. A. Protein aggregation determinants from a simplified model: cooperative folders resist aggregation. Protein Sci 14, 653-62 (2005).<br>
12.	Barral, J. M., Broadley, S. A., Schaffar, G. &amp; Hartl, F. U. Roles of molecular chaperones in protein misfolding diseases. Semin Cell Dev Biol 15, 17-29 (2004).<br>
13.	De Marco, V., Stier, G., Blandin, S. &amp; de Marco, A. The solubility and stability of recombinant proteins are increased by their fusion to NusA. Biochem Biophys Res Commun 322, 756-71 (2004).<br>
14.	Houry, W. A., Frishman, D., Eckerskom, C, Lottspeich, F. &amp; Hartl, F. U. Identification of in vivo substrates of the chaperonin GroEL. Nature 402, 147-54 (1999).<br>
15.	Bairoch, A. et al. The Universal Protein Resource (UniProt). Nucleic Acids Res 33, D154-9 (2005).<br>
Kopp,   J.   &amp;   Schwede,   T.   The   SWISS-MODEL   Repository   of  annotated   three-dimensional protein structure homology models. Nucleic Acids Res 32, D230-4 (2004). Guex, N. &amp; Peitsch, M. C. SWISS-MODEL and the Swiss-PdbViewer: An environment for comparative protein modeling. Electrophoresis 18, 2714-2723 (1997).<br><br><br><br><br><br>
We claim:<br>
1.	A method for down-regulating the biological function of a protein comprising contacting said protein with a non-naturally occurring molecule comprising at least one self-association region present in said protein.<br>
2.	A method according to claim 1 wherein said self-association region is fused to a moiety that prevents aggregation of said self-association region.<br>
3.	A method according to claim 2 wherein said moiety is a peptide, a protein domain or an agarose bead.<br>
4.	A method according to claims 1-3 wherein said self-association region consists of at least 3 contiguous amino acids.<br>
5.	A method according to claims 1-4 wherein a linker is present between said&gt;self-association region and said moiety.<br>
6.	A method according to claim 5 wherein said linker is a polypeptide or is, of non-polypeptide nature.<br>
7.	A method according to claim 1-6 wherein said molecule is a polypeptide and is encoded by a nucleotide sequence present on a recombinant vector and which, upon transformation to a cell or organism, produces said polypeptide in said cell or organism.<br>
8.	A non-naturally occurring molecule comprising at least one self-association region isolated from a protein domain capable of being soluble in water wherein said self-association region is fused to a moiety that prevents aggregation of said self-association region.<br>
9.	A molecule according to claim 8 wherein said moiety is a peptide, a protein domain or an agarose bead.<br>
10.	A recombinant vector comprising a polynucleotide encoding a polypeptide according to claims 8 and 9.<br>
11 A molecule according to claims 8-10 for use as medicament,<br>
12.	A method to isolate a protein from a sample comprising contacting said sample with a molecule comprising at least one self-association region isolated from said protein and isolating the resulting co-aggregated molecule-protein complex from said sample.<br>
13.	A method according to claim 12 wherein said self-association region is fused to a moiety that prevents aggregation of said self-association region.<br>
14.	A method according to claim 13 wherein said moiety is a peptide, a protein domain or an agarose bead.<br>
15.	A method according to claims 12-14 further comprising the detection of a protein in said sample.<br></td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/viewdoc.aspx?id=WyutQc+RqqUc7ncemxgfMQ==&amp;amp;loc=+mN2fYxnTC4l0fUd8W4CAA==" target="_blank" style="word-wrap:break-word;">http://ipindiaonline.gov.in/patentsearch/GrantedSearch/viewdoc.aspx?id=WyutQc+RqqUc7ncemxgfMQ==&amp;amp;loc=+mN2fYxnTC4l0fUd8W4CAA==</a></p>
		<br>
		<div class="pull-left">
			<a href="270505-an-inhalter.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="270507-a-homeopathic-composition-to-be-used-as-a-substitute-of-chemical-nitrogenous-fertilizers.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>270506</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>6265/DELNP/2008</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>01/2016</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>01-Jan-2016</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>28-Dec-2015</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>17-Jul-2008</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>VIB VZW</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td>RIJVISSCHESTRAAT 120, B-9052 ZWIJNAARDE, BELGIUM</td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>SCHYMKOWITZ, JOOST</td>
											<td>ATTENRODESTRAAT 15, B-3391, MEENSEL-KIEZEGEM, BELGIUM</td>
										</tr>
										<tr>
											<td>2</td>
											<td>ROUSSEAU, FREDERIC</td>
											<td>STICHELBERG 18, B-1702 GROOT-BIJGAARDEN, BELGIUM</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>A61K 38/16</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>PCT/EP2006/070184</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td>2006-12-22</td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td>60/872,079</td>
									<td>2006-12-01</td>
								    <td>U.S.A.</td>
								</tr>
								<tr>
									<td>2</td>
									<td>60/753,245</td>
									<td>2005-12-22</td>
								    <td>U.S.A.</td>
								</tr>
								<tr>
									<td>3</td>
									<td>06125189.8</td>
									<td>2006-12-01</td>
								    <td>U.S.A.</td>
								</tr>
								<tr>
									<td>4</td>
									<td>05112761.1</td>
									<td>2005-12-22</td>
								    <td>U.S.A.</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/270506-means-and-methods-for-mediating-protein-interference by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 04:20:47 GMT -->
</html>
