<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/271189-production-of-isoprenoids-and-isoprenoid-precursors by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 02:18:35 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 271189:&quot;PRODUCTION OF ISOPRENOIDS AND ISOPRENOID PRECURSORS&quot;</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">&quot;PRODUCTION OF ISOPRENOIDS AND ISOPRENOID PRECURSORS&quot;</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>The present invention provides genetically modified host cells and use of same for producing isoprenoid compounds.</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td>PRODUCTION OF ISOPRENOIDS AND LSOPKENOID PRECUPSORS<br>
CROSS-REFERENCE<br>
[0001]	This application claims the benefit of priority of U.S. Provisional Application No. 60/826,970,<br>
filed on September 26,2006, which application is incorporated herein by reference in its entirety.<br>
BACKGROUND<br>
[0002]	Isoprenoids are ubiquitous in nature. They comprise a diverse family of over 40,000 individual<br>
products, many of which are vital to living organisms. Isoprenoids serve to maintain cellular fluidity, electron transport, and other metabolic functions. A vast number of natural and synthetic isoprenoids are useful as pharmaceuticals, cosmetics, perfumes, pigments and colorants, fungicides, antiseptics, nutraceuticals, and fine chemical intermediates.<br>
[0003]	An isoprenoid product is typically composed of repeating five-carbon isopentenyl diphosphate<br>
(IPP) units, although irregular isoprenoids and polyterpenes have been reported. In nature, isoprenoids are synthesized by consecutive condensations of their precursor IPP and its isomer dimethylallyl pyrophosphate (DMAPP). Two pathways for these precursors are known. Eukaryotes, with the exception of plants, generally use the mevalonate-dependent (MEV) pathway to convert acetyl coenzyme A (acetyl-CoA) to IPP, which is subsequently isomerized to DMAPP. Prokaryotes, with some exceptions, typically employ only the mevalonate-independent or deoxyxylulose-5-phosphate (DXP) pathway to produce EPP and DMAPP. Plants use both the MEV pathway and the DXP pathway.<br>
[0004]	Traditionally, isoprenoids have been manufactured by extraction from natural sources such as<br>
plants, microbes, and animals. However, the yield by way of extraction is usually very low due to certain limitations. First, most isoprenoids accumulate in nature in only small amounts. Second, the source organisms in general are not amenable to the large-scale cultivation that is necessary to produce commercially viable quantities of a desired isoprenoid. Third, the requirement of certain toxic solvents for isoprenoid extraction necessitates special handling and disposal procedures, thus complicating the commercial production of isoprenoids.<br>
[0005]	The elucidation of the MEV and DXP metabolic pathways has made biosynthetic production of<br>
isoprenoids feasible. For instance, microbes have been engineered to overcxpress a part of or the entire mevalonate pathway for production of the isoprenoid amorpha-4,11 -diene. Other efforts have focused on balancing the pool of glyceraldehyde-3-phosphate and pyruvate, or on increasing the expression of l-deoxy-D-xylulose-5-phosphate synthase (dxs) and IPP isomerase (idi).<br>
[0006]	Nevertheless, given the very large quantities of isoprenoid products needed for many<br>
commercial applications, there remains a need for expression systems and fermentation procedures that produce even more isoprenoids than available with current technologies.<br>
Literature<br>
[0007]	U.S. Patent Nos. 7,172,886 and 7,192,751; Farmer et al. (2001) Biotechnol. Prog. 17:57-61;<br>
Kajiwara et al. (1997) Biochem. J. 324:421-426; and Kim et al. (2001) Biotechnol. Bioeng. 72:408-415; Martinet al. (2003)Nat. Biotech. 21(7):796-802; U.S. Patent No. 7,183,089;<br>
SUMMARY OF THE INVENTION<br>
[0008]	The present invention provides genetically modified host cells and use of same for producing<br>
isoprenoid compounds.<br>
BRIEF DESCRIPTION OF THE DRAWINGS<br>
[0009]	Figure 1 is a schematic representation of the mevalonate (MEV) pathway for the production of<br>
isopentenyl diphosphate (IPP).<br>
[0010]	Figure 2 is a schematic representation of isoprenoid metabolic pathways that result in the<br>
production of the isoprenoid biosynthetic pathway intermediates polyprenyl diphosphates geranyl diphosphate (GPP), famesyl diphosphate (FPP), and geranylgeranyl diphosphate (GGPPP), from IPP and DMAPP.<br>
[0011]	Figure 3 is a map of expression plasmid pAM39.<br>
[0012]	Figure 4 is a map of expression plasmid pAM40.<br>
[0013]	Figure 5 is a map of expression plasmid pAM45.<br>
[0014]	Figure 6 is a map of expression plasmid pTrc99A-ADS.<br>
[0015]	Figure 7 is a map of expression plasmid pAM94.<br>
[0016]	Figures 8A and 8B show production of amorpha-4,11-diene in host strains in which the MBIS<br>
operon is expressed from a higher copy number plasmid and a stronger promoter.<br>
[0017]	Figure 9 shows amorpha-4,11 -diene production by host strains in which the gene dosage of<br>
mevalonate kinase is increased.<br>
[0018]	Figure 10 shows amorpha-4,11-diene production in host strains in which the gene dosage and<br>
expression of mevalonate kinase are increased.<br>
DEFINITIONS<br>
[0019]	The terms "isoprenoid," "isoprenoid compound," "terpene," "terpene compound," "terpenoid,"<br>
and "terpenoid compound" are used interchangeably herein, and refer to any compound that is capable of being derived from IPP. The number of C-atoms present in the isoprenoids is typically evenly divisible by five (e.g., C5, CIO, CI 5, C20, C25, C30 and C40). Irregular isoprenoids and polyterpenes have been reported, and are also included in the definition of "isoprenoid." Isoprenoid compounds include, but are not limited to, monoterpenes, diterpenes, triterpenes, sesquiterpenes, and polyterpenes.<br>
[0020]	As used herein, the term "prenyl diphosphate" is used interchangeably with "prenyl<br>
pyrophosphate," and includes monoprenyl diphosphates having a single prenyl group (e.g., IPP and DMAPP), as well as polyprenyl diphosphates that include 2 or more prenyl groups. Monoprenyl diphosphates include isopentenyl pyrophosphate (IPP) and its isomer dimethylallyl pyrophosphate (DMAPP).<br>
[0021]	As used herein, the term "terpene synthase" (also referred to a "terpene cyclase") refers to any<br>
enzyme that enzymatically modifies IPP, DMAPP, or a polyprenyl pyrophosphate, such that a terpenoid precursor compound is produced. The term "terpene synthase" includes enzymes that catalyze the conversion of a prenyl diphosphate into an isoprenoid or isoprenoid precursor.<br>
[0022]	The word "pyrophosphate" is used interchangeably herein with "diphosphate." Thus, e.g., the<br>
terms "prenyl diphosphate" and "prenyl pyrophosphate" are interchangeable; the terms "isopentenyl pyrophosphate" and "isopentenyl diphosphate" are interchangeable; the terms famesyl diphosphate" and farnesyl pyrophosphate" are interchangeable; etc.<br>
[0023]	The term "mevalonate pathway" or "MEV pathway" is used herein to refer to the biosynthetic<br>
pathway that converts acetyl-CoA to IPP. The mevalonate pathway comprises enzymes that catalyze the following steps: (a) condensing two molecules of acetyl-Co A to acetoacetyl-CoA (e.g., by action of acetoacetyl-CoA thiolase); (b) condensing acetoacetyl-CoA with acetyl-CoAto form hydroxymethylglutaryl-CoenzymeA (HMG-CoA) (e.g., by action of HMG-CoA synthase (HMGS)); (c) converting HMG-CoA to mevalonate (e.g., by action of HMG-CoA reductase (HMGR)); (d) phosphorylating mevalonate to mevalonate 5-phosphate (e.g., by action of mevalonate kinase (MK)); (e) converting mevalonate 5-phosphate to mevalonate 5-pyrophosphate (e.g., by action of phosphomevalonate kinase (PMK)); and (f) converting mevalonate 5-pyrophosphate to isopentenyl pyrophosphate (e.g., by action of mevalonate pyrophosphate decarboxylase (MPD)). The mevalonate pathway is illustrated schematically in Figure 1. The "top half' of the mevalonate pathway refers to the enzymes responsible for the conversion of acetyl-CoA to mevalonate.<br>
[0024]	The term "1-deoxy-D-xylulose 5-diphosphate pathway" or "DXP pathway" is used herein to<br>
refer to the pathway that converts glyceraldehyde-3-phosphate and pyruvate to IPP and DMAPP<br>
through a DXP pathway intermediate.<br>
[0025]	As used herein, the term "prenyl transferase" is used interchangeably with the terms "isoprenyl<br>
diphosphate synthase" and "polyprenyl synthase" (e.g., "GPP synthase," "FPP synthase," "GGPP<br>
synthase," etc.) to refer to an enzyme that catalyzes the consecutive 1M condensation of isopentenyl<br>
diphosphate with allylic primer substrates, resulting in the formation of prenyl diphosphates of various<br>
chain lengths.<br>
[0026]	The terms "polynucleotide" and "nucleic acid," used interchangeably herein, refer to a<br>
polymeric form of nucleotides of any length, either ribonucleotides or deoxynucleotides. Thus, this term includes, but is not limited to, single-, double-, or multi-stranded DNA or RNA, genomic DNA,<br>
cDNA, DNA-RNA hybrids, or a polymer comprising purine and pyrimidine bases or other natural,<br>
chemically or biochemically modified, non-natural, or derivatized nucleotide bases.<br>
[0027]	The terms "peptide," "polypeptide," and "protein" are used interchangeably herein, and refer to<br>
a polymeric form of amino acids of any length, which can include coded and non-coded amino acids,<br>
chemically or biochemically modified or derivatized amino acids, and polypeptides having modified<br>
peptide backbones.<br>
[0028]	The term "naturally-occurring" as used herein as applied to a nucleic acid, a cell, or an<br>
organism, refers to a nucleic acid, cell, or organism that is found in nature. For example, a polypeptide<br>
or polynucleotide sequence that is present in an organism (including viruses) that can be isolated from a<br>
source in nature and which has not been intentionally modified by a human in the laboratory is naturally<br>
occurring.<br>
[0029]	As used herein the term "isolated" is meant to describe a polynucleotide, a polypeptide, or a<br>
cell that is in an environment different from that in which the polynucleotide, the polypeptide, or the<br>
cell naturally occurs. An isolated genetically modified host cell may be present in a mixed population<br>
of genetically modified host cells.<br>
[0030]	As used herein, the term "exogenous nucleic acid" refers to a nucleic acid that is not normally<br>
or naturally found in and/or produced by a given bacterium, organism, or cell in nature. As used herein,<br>
the term "endogenous nucleic acid" refers to a nucleic acid that is normally found in and/or produced<br>
by a given bacterium, organism, or cell in nature. An "endogenous nucleic acid" is also referred to as a<br>
"native nucleic acid" or a nucleic acid that is "native" to a given bacterium, organism, or cell. For<br>
example, the nucleic acids encoding HMGS, mevalonate kinase, and phosphomevalonate kinase in<br>
represent exogenous nucleic acids to E. coli.<br>
[0031]	The term "heterologous nucleic acid," as used herein, refers to a nucleic acid wherein at least<br>
one of the following is true: (a) the nucleic acid is foreign ("exogenous") to (i.e., not naturally found<br>
in) a given host microorganism or host cell; (b) the nucleic acid comprises a nucleotide sequence that is<br>
naturally found in (e.g., is "endogenous to") a given host microorganism or host cell (e.g., the nucleic<br>
acid comprises a nucleotide sequence that is endogenous to the host microorganism or host cell) but is<br>
either produced in an unnatural (e.g., greater than expected or greater than naturally found) amount in<br>
the cell, or differs in sequence from the endogenous nucleotide sequence such that the same encoded<br>
protein (having the same or substantially the same amino acid sequence) as found endogenously is<br>
produced in an unnatural (e.g., greater than expected or greater than naturally found) amount in the cell;<br>
(c) the nucleic acid comprises two or more nucleotide sequences or segments that are not found in the<br>
same relationship to each other in nature, e.g., the nucleic acid is recombinant.<br>
[0032]	The term "heterologous polypeptide," as used herein, refers to a polypeptide that is not<br>
naturally associated with a given polypeptide. For example, an isoprenoid precursor-modifying enzyme that comprises a "heterologous transmembrane domain" refers to an isoprenoid precursor-modifying enzyme that comprises a transmembrane domain that is not normally associated with (e.g., not normally<br>
contiguous with; not normally found in the same polypeptide chain with) the isoprenoid precursor-<br>
modifying enzyme in nature.<br>
[0033]	"Recombinant," as used herein, means that a particular nucleic acid (DNA or RNA) is the<br>
product of various combinations of cloning, restriction, and/or ligation steps resulting in a construct having a structural coding or non-coding sequence distinguishable from endogenous nucleic acids found in natural systems. Generally, DNA sequences encoding the structural coding sequence can be assembled from cDNA fragments and short oligonucleotide linkers, or from a series of synthetic oligonucleotides, to provide a synthetic nucleic acid which is capable of being expressed from a recombinant transcriptional unit contained in a cell or in a cell-free transcription and translation system. Such sequences can be provided in the form of an open reading frame uninterrupted by internal non-translated sequences, or introns, which are typically present in eukaryotic genes. Genomic DNA comprising the relevant sequences can also be used in the formation of a recombinant gene or transcriptional unit. Sequences of non-translated DNA may be present 5' or 3' from the open reading frame, where such sequences do not interfere with manipulation or expression of the coding regions, and may indeed act to modulate production of a desired product by various mechanisms (see "DNA regulatory sequences", below).<br>
[0034]	Thus, e.g., the term "recombinant" polynucleotide or "recombinant" nucleic acid refers to one<br>
which is not naturally occurring, e.g., is made by the artificial combination of two otherwise separated segments of sequence through human intervention. This artificial combination is often accomplished by either chemical synthesis means, or by the artificial manipulation of isolated segments of nucleic acids, e.g., by genetic engineering techniques. Such is usually done to replace a codon with a redundant codon encoding the same or a conservative amino acid, while typically introducing or removing a sequence recognition site. Alternatively, it is performed to join together nucleic acid segments of desired functions to generate a desired combination of functions. This artificial combination is often accomplished by either chemical synthesis means, or by the artificial manipulation of isolated segments of nucleic acids, e.g., by genetic engineering techniques.<br>
[0035]	Similarly, die term "recombinant" polypeptide refers to a polypeptide which is not naturally<br>
occurring, e.g., is made by the artificial combination of two otherwise separated segments of amino sequence through human intervention. Thus, e.g., a polypeptide that comprises a heterologous amino acid sequence is recombinant.<br>
[0036]	By "construct" or "vector" is meant a recombinant nucleic acid, generally recombinant DNA,<br>
which has been generated for the purpose of the expression and/or propagation of a specific nucleotide sequence(s), or is to be used in the construction of other recombinant nucleotide sequences.<br>
[0037]	As used herein, the terms "operon" and "single transcription unit" are used interchangeably to<br>
refer to two or more contiguous coding regions (nucleotide sequences that encode a gene product such as an RNA or a protein) that are coordinately regulated by one or more controlling elements (e.g., a promoter). As used herein, the term "gene product" refers to RNA encoded by DNA (or vice versa) or<br>
protein that is encoded by an RNA or DNA, where a gene will typically comprise one or more<br>
nucleotide sequences that encode a protein, and may also include introns and other non-coding<br>
nucleotide sequences.<br>
[0038]	The term "coding region," as used herein, refers to a contiguous stretch of nucleotides (a<br>
nucleotide sequence) that encodes a polypeptide. For example, an "MK-encoding coding region" or an<br>
"MK coding region" comprises a nucleotide sequence encoding mevalonate kinase.<br>
[0039]	The terms "DNA regulatory sequences," "control elements," and "regulatory elements," used<br>
interchangeably herein, refer to transcriptional and translational control sequences, such as promoters,<br>
enhancers, polyadenylation signals, terminators, protein degradation signals, and the like, that provide<br>
for and/or regulate expression of a coding sequence and/or production of an encoded polypeptide in a<br>
host cell.<br>
[0040]	The term "transformation" is used interchangeably herein with "genetic modification" and<br>
refers to a permanent or transient genetic change induced in a cell following introduction of new<br>
nucleic acid (i.e., DNA exogenous to the cell). Genetic change ("modification") can be accomplished<br>
either by incorporation of the new DNA into the genome of the host cell, or by transient or stable<br>
maintenance of the new DNA as an episomal element. Where the cell is a eukaryotic cell, a permanent<br>
genetic change is generally achieved by introduction of the DNA into the genome of the cell. In<br>
prokaryotic cells, permanent changes can be introduced into the chromosome or via extrachromosomal<br>
elements such as plasmids and expression vectors, which may contain one or more selectable markers<br>
to aid in their maintenance in the recombinant host cell. Suitable methods of genetic modification<br>
include viral infection, transfection, conjugation, protoplast fusion, electroporation, particle gun<br>
technology, calcium phosphate precipitation, direct microinjection, and the like. The choice of method<br>
is generally dependent on the type of cell being transformed and the circumstances under which the<br>
transformation is taking place (i.e. in vitro, ex vivo, or in vivo). A general discussion of these methods<br>
can be found in Ausubel, et al, Short Protocols in Molecular Biology, 3rd ed., Wiley &amp; Sons, 1995.<br>
[0041]	"Operably linked" refers to a juxtaposition wherein the components so described are in a<br>
relationship permitting them to function in their intended manner. For instance, a promoter is operably<br>
linked to a coding sequence if the promoter affects its transcription or expression.   As used herein, the<br>
terms "heterologous promoter" and "heterologous control regions" refer to promoters and other control<br>
regions that are not normally associated with a particular nucleic acid in nature. For example, a<br>
"transcriptional control region heterologous to a coding region" is a transcriptional control region that is<br>
not normally associated with the coding region in nature.<br>
[0042]	A "host cell," as used herein, denotes an in vivo or in vitro eukaryotic cell, a prokaryotic cell, or<br>
a cell from a multicellular organism (e.g., a cell line) cultured as a unicellular entity, which eukaryotic or prokaryotic cells can be, or have been, used as recipients for a nucleic acid (e.g., an expression vector that comprises a nucleotide sequence encoding one or more biosynthetic pathway gene products such as mevalonate pathway gene products), and include the progeny of the original cell which has been<br>
genetically modified by the nucleic acid. It is understood that the progeny of a single cell may not necessarily be completely identical in morphology or in genomic or total DNA complement as the original parent, due to natural, accidental, or deliberate mutation. A "recombinant host cell" (also referred to as a "genetically modified host cell") is a host cell into which has been introduced a heterologous nucleic acid, e.g., an expression vector. For example, a subject genetically modified prokaryotic host cell (e.g., a bacterium) is a prokaryotic host cell that, by virtue of introduction into a suitable prokaryotic host cell a heterologous nucleic acid, e.g., an exogenous nucleic acid that is foreign to (not normally found in nature in) the prokaryotic host cell, or a recombinant nucleic acid that is not normally found in the prokaryotic host cell; and a subject genetically modified eukaryotic host cell is a eukaryotic host cell that, by virtue of introduction into a suitable eukaryotic host cell a heterologous nucleic acid, e.g., an exogenous nucleic acid that is foreign to the eukaryotic host cell, or a recombinant nucleic acid that is not normally found in the eukaryotic host cell.<br>
[0043]	The term "conservative amino acid substitution" refers to the interchangeability in proteins of<br>
amino acid residues having similar side chains. For example, a group of amino acids having aliphatic side chains consists of glycine, alanine, valine, leucine, and isoleucine; a group of amino acids having aliphatic-hydroxyl side chains consists of serine and threonine; a group of amino acids having amide-containing side chains consists of asparagine and glutamine; a group of amino acids having aromatic side chains consists of phenylalanine, tyrosine, and tryptophan; a group of amino acids having basic side chains consists of lysine, arginine, and histidine; and a group of amino acids having sulfur-containing side chains consists of cysteine and methionine. Exemplary conservative amino acids substitution groups are: valine-leucine-isoleucine, phenylalanine-tyrosine, lysine-arginine, alanine-valine, and asparagine-glutamine.<br>
[0044]	"Synthetic nucleic acids" can be assembled from oligonucleotide building blocks that are<br>
chemically synthesized using procedures known to those skilled in the art. These building blocks are ligated and annealed to form gene segments which are then enzymatically assembled to construct the entire gene. "Chemically synthesized," as related to a sequence of DNA, means that the component nucleotides were assembled in vitro. Manual chemical synthesis of DNA may be accomplished using well-established procedures, or automated chemical synthesis can be performed using one of a number of commercially available machines. The nucleotide sequence of the nucleic acids can be modified for optimal expression based on optimization of nucleotide sequence to reflect the codon bias of the host cell. The skilled artisan appreciates the likelihood of successful expression if codon usage is biased towards those codons favored by the host. Determination of preferred codons can be based on a survey of genes derived from the host cell where sequence information is available.<br>
[0045]	A polynucleotide or polypeptide has a certain percent "sequence identity" to another<br>
polynucleotide or polypeptide, meaning that, when aligned, that percentage of bases or amino acids are the same, and in die same relative position, when comparing the two sequences. Sequence similarity can be determined in a number of different manners. To determine sequence identity, sequences can be<br>
aligned using the methods and computer programs, including BLAST, available over the world wide<br>
web at ncbi.nlm.nih.gov/BLAST. See, e.g., Altschul et al. (1990), J. Mol. Biol. 215:403-10.   Another<br>
alignment algorithm is FAST A, available in the Genetics Computing Group (GCG) package, from<br>
Madison, Wisconsin, USA, a wholly owned subsidiary of Oxford Molecular Group, Inc. Other<br>
techniques for alignment are described in Methods in Enzymology, vol. 266: Computer Methods for<br>
Macromolecular Sequence Analysis (1996), ed. Doolittle, Academic Press, Inc., a division of Harcourt<br>
Brace &amp; Co., San Diego, California, USA. Of particular interest are alignment programs that permit<br>
gaps in the sequence. The Smith-Waterman is one type of algorithm that permits gaps in sequence<br>
alignments. See Meth. Mol. Biol. 70:173-187 (1997). Also, the GAP program using the Needleman<br>
and Wunsch alignment method can be utilized to align sequences. See J. Mol. Biol. 48: 443-453<br>
(1970).<br>
[0046]	A nucleic acid is "hybridizable" to another nucleic acid, such as a cDNA, genomic DNA, or<br>
RNA, when a single stranded form of the nucleic acid can anneal to the other nucleic acid under the appropriate conditions of temperature and solution ionic strength. Hybridization and washing conditions are well known and exemplified in Sambrook, J., Fritsch, E. F: and Maniatis, T. Molecular Cloning: A Laboratory Manual, Second Edition, Cold Spring Harbor Laboratory Press, Cold Spring Harbor (1989), particularly Chapter 11 and Table 11.1 therein; and Sambrook, J. and Russell, W., Molecular Cloning: A Laboratory Manual, Third Edition, Cold Spring Harbor Laboratory Press, Cold Spring Harbor (2001). The conditions of temperature and ionic strength determine the "stringency" of the hybridization. Stringency conditions can be adjusted to screen for moderately similar fragments, such as homologous sequences from distantly related organisms, to highly similar fragments, such as genes that duplicate functional enzymes from closely related organisms. Hybridization conditions and post-hybridization washes are useful to obtain the desired determine stringency conditions of the hybridization. One set of illustrative post-hybridization washes is a series of washes starting with 6 x SSC (where SSC is 0.15 M NaCl and 15 mM citrate buffer), 0.5% SDS at room temperature for 15 minutes, then repeated with 2 x SSC, 0.5% SDS at 45°C for 30 minutes, and then repeated twice with 0.2 x SSC, 0.5% SDS at 50°C for 30 minutes. Other stringent conditions are obtained by using higher temperatures in which the washes are identical to those above except for the temperature of the final two 30 minute washes in 0.2 x SSC, 0.5% SDS, which is increased to 60°C. Another set of highly stringent conditions uses two final washes in 0.1 x SSC, 0.1% SDS at 65°C. Another example of stringent hybridization conditions is hybridization at 50°C or higher andO.lxSSC (15 mM sodium chloride/1.5 mM sodium citrate). Another example of stringent hybridization conditions is overnight incubation at 42°C in a solution: 50% formamide, 5 x SSC (150 mM NaCl, 15 mM trisodium citrate), 50 mM sodium phosphate (pH 7.6), 5 x Denhardt's solution, 10% dextran sulfate, and 20 ug/ml denatured, sheared salmon sperm DNA, followed by washing the filters in 0.1 x SSC at about 65°C. Stringent hybridization conditions and post-hybridization wash conditions are hybridization conditions<br>
and post-hybridization wash conditions that are at least as stringent as the above representative<br>
conditions.<br>
[0047]	Hybridization requires that the two nucleic acids contain complementary sequences, although<br>
depending on the stringency of the hybridization, mismatches between bases are possible. The appropriate stringency for hybridizing nucleic acids depends on the length of the nucleic acids and the degree of complementation, variables well known in the art. The greater the degree of similarity or homology between two nucleotide sequences, the greater the value of the melting temperature (Tm) for hybrids of nucleic acids having those sequences. The relative stability (corresponding to higher Tm) of nucleic acid hybridizations decreases in the following order RNA:RNA, DNA:RNA, DNA:DNA. For hybrids of greater than 100 nucleotides in length, equations for calculating Tm have been derived (see Sambrook et al., supra, 9.50-9.51). For hybridizations with shorter nucleic acids, i.e., oligonucleotides, the position of mismatches becomes more important, and the length of the oligonucleotide determines its specificity (see Sambrook et al., supra, 11.7-11.8). Typically, the length for a hybridizable nucleic acid is at least about 10 nucleotides. Illustrative minimum lengths for a hybridizable nucleic acid are: at least about 15 nucleotides; at least about 20 nucleotides; and at least about 30 nucleotides. Furthermore, the skilled artisan will recognize that the temperature and wash solution salt concentration may be adjusted as necessary according to factors such as length of the probe.<br>
[0048]	Before the present invention is further described, it is to be understood that this invention is not<br>
limited to particular embodiments described, as such may, of course, vary. It is also to be understood that the terminology used herein is for the purpose of describing particular embodiments only, and is not intended to be limiting, since the scope of the present invention will be limited only by the appended claims.<br>
[0049]	Where a range of values is provided, it is understood that each intervening value, to the tenth of<br>
the unit of the lower limit unless the context clearly dictates otherwise, between the upper and lower limit of that range and any other stated or intervening value in that stated range, is encompassed within the invention. The upper and lower limits of these smaller ranges may independently be included in the smaller ranges, and are also encompassed within the invention, subject to any specifically excluded limit in the stated range. Where the stated range includes one or both of the limits, ranges excluding either or both of those included limits are also included in the invention.<br>
[0050]	Unless defined otherwise, all technical and scientific terms used herein have the same meaning<br>
as commonly understood by one of ordinary skill in the art to which this invention belongs. Although any methods and materials similar or equivalent to those described herein can also be used in the practice or testing of the present invention, the preferred methods and materials are now described. All publications mentioned herein are incorporated herein by reference to disclose and describe the methods and/or materials in connection with which the publications are cited.<br>
[0051J	It must be noted that as used herein and in the appended claims, the singular forms "a," "and,"<br>
and "the" include plural referents unless the context clearly dictates otherwise. Thus, for example, reference to "a genetically modified host cell" includes a plurality of such host cells and reference to "the isoprenoid compound" includes reference to one or more isoprenoid compounds and equivalents thereof known to those skilled in the art, and so forth. It is further noted that the claims may be drafted to exclude any optional element. As such, this statement is intended to serve as antecedent basis for use of such exclusive terminology as "solely," "only" and the like in connection with the recitation of claim elements, or use of a "negative" limitation.<br>
[0052]	The publications discussed herein are provided solely for their disclosure prior to the filing date<br>
of the present application. Nothing herein is to be construed as an admission that the present invention is not entitled to antedate such publication by virtue of prior invention. Further, the dates of publication provided may be different from the actual publication dates which may need to be independently confirmed.<br>
DETAILED DESCRIPTION<br>
[0053]	The present invention provides genetically modified host cells and use of same for producing<br>
isoprenoid or isoprenoid precursor compounds.<br>
[0054]	One method of making an isoprenoid or an isoprenoid precursor is to culture a host cell, where<br>
the host cell is capable of making the isoprenoid or isoprenoid precursor. Because the biosynthetic pathway for making an isoprenoid or an isoprenoid precursor involves multiple enzymes, the flux through the pathway may not be optimum or properly balanced. One method of correcting such imbalance is to modulate the activity levels of the pathway enzymes relative to one another. As described in more detail herein, increasing the level of mevalonate kinase, relative to the level of one or more other enzymes in the mevalonate pathway, provides for an increase in production levels of isoprenoid compounds, or precursors of isoprenoid compounds.<br>
[0055]	Isoprenoid compounds are synthesized from a universal five carbon precursor, isopentenyl<br>
pyrophosphate (IPP). There are two major pathways for converting a substrate to IPP: 1) the "mevalonate" pathway," which converts acetyl-CoA to IPP; and the "1-deoxy-D-xylulose 5-diphosphate pathway" (also referred to as the "DXP pathway"), which converts D-glyceraldehyde-3-phosphate and pyruvate to IPP and DMAPP.<br>
[0056]	Mevalonate pathway enzymes are depicted in Figure 1. The mevalonate pathway comprises the<br>
following enzymatic reactions: (a) condensing two molecules of acetyl-CoA to acetoacetyl-CoA; (b) condensing acetoacetyl-CoA with acetyl-CoAto formHMG-CoA; (c) converting HMG-CoAto mevalonate; (d) phosphorylating mevalonate to mevalonate 5-phosphate; (e) converting mevalonate 5-phosphate to mevalonate 5-pyrophosphate; and (f) converting mevalonate 5-pyrophosphate to isopentenyl pyrophosphate. Enzymes that carry out these reactions include acetoacetyl-CoA thiolase, hydroxymethylglutaryl-CoA synthase (HMGS), hydroxymethylglutaryl-CoA reductase (HMGR),<br>
mevalonate kinase (MK), phosphomevalonate kinase (PMK), and mevalonate pyrophosphate<br>
decarboxylase (MPD).<br>
[0057]	In the DXP pathway, pyruvate and D-glyceraldehyde-3-phosphate are converted via a series of<br>
reactions to BPP and DMAPP. The pathway involves action of the following enzymes: 1 -deoxy-D-<br>
xylulose-5-pbosphate synthase (Dxs), l-deoxy-D-xylulose-5-phosphate reductoisomerase (IspC), 4-<br>
diphosphocytidyl-2-C-methyl-D-erythritol synthase (IspD), 4-diphosphocytidyl-2-C-mefhyl-D-<br>
erythritol kinase (IspE), 2C-methyl-D-erythritol 2,4-cyclodiphosphate synthase (IspF), l-hydroxy-2-<br>
methyl-2-(E)-butenyl 4-diphosphate synthase (IspG), and isopentenyl diphosphate isomerase (IspH).<br>
[0058]	Eukaryotic cells other than plant cells use the mevalonate pathway exclusively to convert<br>
acetyl- CoA to IPP, which is subsequently isomerized to DMAPP. Plants use both the mevalonate and<br>
the DXP pathways for isoprenoid synthesis. Prokaryotes, with some exceptions, use the DXP pathway<br>
to produce IPP and DMAPP separately through a branch point.<br>
[0059]	The IPP produced by the mevalonate pathway can be isomerized to produce DMAPP. The IPP<br>
and/or the DMAPP can be acted on by prenyltransfcrases to produce polyprenyl pyrophosphates. For<br>
example, as shown in Figure 2, IPP or DMAPP can be modified by prenyl transferases to generate the<br>
polyprenyl diphosphates geranyl diphosphate (GPP), famesyl diphosphate (FPP), and geranylgeranyl<br>
diphosphate (GGPP). GPP and FPP are further modified by terpene synthases to generate monoterpenes<br>
and sesquiterpenes, respectively; and GGPP is further modified by terpene synthases to generate<br>
diterpenes and carotenoids. IPP and DMAPP are generated by one of two pathways: the mevalonate<br>
(MEV) pathway and the l-deoxy-D-xylulose-5-phosphate (DXP) pathway.<br>
GENETICALLY MODIFIED HOST CELLS<br>
[0060]	The present invention provides genetically modified host cells, and methods of using same to<br>
produce isoprenoid compounds. In some embodiments, a subject genetically modified host cell is<br>
genetically modified such that it produces mevalonate kinase at a level that is higher than the level of at<br>
least one of acetoacetyl-CoA thiolase, hydroxymediylglutaryl-CoA synthase (HMGS),<br>
hydroxymethylglutaryl-CoA reductase (HMGR), phosphomevalonate kinase (PMK), and mevalonate<br>
pyrophosphate decarboxylase (MPD). A subject method, described in more detail below, generally<br>
involves culturing a subject genetically modified host cell in vitro in a suitable medium, such that the<br>
genetically modified host cell converts a substrate to IPP and produces an isoprenoid compound.<br>
[0061]	In some embodiments, the level of MK produced in a subject genetically modified host cell is<br>
at least about 10%, at least about 15%, at least about 20%, at least about 25%, at least about 30%, at least about 35%, at least about 40%, at least about 45%, at least about 50%, at least about 60%, at least about 70%, at least about 80%, at least about 90%, at least about 2-fold, at least about 2.5-fold, at least about 3-fold, at least about 4-fold, at least about 5-fold, at least about 7-fold, or at least about 10-fold, or more, higher than the level of one or more of acetoacetyl-CoA thiolase, HMGS, HMGR, PMK, and MPD.<br>
(0062]	For example, in some embodiments, the level of MK produced in a subject genetically<br>
modified host cell is at least about 10%, at least about 15%, at least about 20%, at least about 25%, at<br>
least about 30%, at least about 35%, at least about 40%, at least about 45%, at least about 50%, at least<br>
about 60%, at least about 70%, at least about 80%, at least about 90%, at least about 2-fold, at least<br>
about 2.5-fold, at least about 3-fold, at least about 4-fold, at least about 5-fold, at least about 7-fold, or<br>
at least about 10-fold, or more, higher than the level of HMGS in the cell.<br>
[0063]	In other embodiments, the level of MK produced in a subject genetically modified host cell is at<br>
least about 10%, at least about 15%, at least about 20%, at least about 25%, at least about 30%, at least<br>
about 35%, at least about 40%, at least about 45%, at least about 50%, at least about 60%, at least about<br>
70%, at least about 80%, at least about 90%, at least about 2-fold, at least about 2.5-fold, at least about<br>
3-fold, at least about 4-fold, at least about 5-fold, at least about 7-fold, or at least about 10-fold, or<br>
more, higher than the level of HMGR in the cell.<br>
[0064]	In other embodiments, the level of MK produced in a subject genetically modified host cell is at<br>
least about 10%, at least about 15%, at least about 20%, at least about 25%, at least about 30%, at least<br>
about 35%, at least about 40%, at least about 45%, at least about 50%, at least about 60%, at least about<br>
70%, at least about 80%, at least about 90%, at least about 2-fold, at least about 2.5-fold, at least about<br>
3-fold, at least about 4-fold, at least about 5-fold, at least about 7-fold, or at least about 10-fold, or<br>
more, higher than the level of both HMGS and HMGR in the cell.<br>
[0065]	In some embodiments, the level of MK produced by a subject genetically modified host cell is a<br>
least about 10%, at least about 15%, at least about 20%, at least about 25%, at least about 30%, at least<br>
about 35%, at least about 40%, at least about 45%, at least about 50%, at least about 60%, at least about<br>
70%, at least about 80%, at least about 90%, at least about 2-fold, at least about 2.5-fold, at least about<br>
3-fold, at least about 4-fold, at least about 5-fold, at least about 10-fold, at least about 25-fold, at least<br>
about 50-fold, or at least about 100-fold, or more, higher than the level of MK produced by a host<br>
genetically modified with pMBI or with pMBIS. pMBI and pMBIS are described in U.S. Patent No.<br>
7,192,751.<br>
10066]	Thus, for example, in some embodiments, where a subject genetically modified host cell is a<br>
prokaryotic cell, the level of MK produced by a subject genetically modified host cell is a least about<br>
10%, at least about 15%, at least about 20%, at least about 25%, at least about 30%, at least about 35%,<br>
at least about 40%, at least about 45%, at least about 50%, at least about 60%, at least about 70%, at<br>
least about 80%, at least about 90%, at least about 2-fold, at least about 2.5-fold, at least about 3-fold, at<br>
least about 4-fold, at least about 5-fold, at least about 10-fold, at least about 25-fold, at least about 50-<br>
fold, or at least about 100-fold, or more, higher than the level of MK produced by the same cell<br>
genetically modified with pMBI or pMBIS.<br>
[0067]	In some embodiments, ths molar ratio of MK polypeptide in a subject genetically modified host<br>
cell relative to one or more of acetoacetyl-CoA thiolase, HMGS, HMGR, PMK, and MPD polypeptide is from about 1.25:1 to about 100:1, or greater than 100:1. For example, in some embodiments, the<br>
molar ratio of MK relative to one or more of acetoacetyl-CoA thiolase, HMGS, HMGR, PMK, and<br>
MPD is from about 1.25:1 to about 1.5:1, from about 1.5:1 to about 2:1, from about 2:1 to about 2.5:1,<br>
from about 2.5:1 to about 3:1, from about 3:1 to about 5:1, from about 5:1 to about 10:1, from about<br>
10:1 to about 25:1, from about 25:1 to about 50:1, or from about 50:1 to about 100:1.<br>
[0068]	The level of MK produced in a subject genetically modified host cell can be controlled in<br>
various ways. In some embodiments, the copy number of coding regions comprising nucleotide<br>
sequences encoding MK is higher than the copy number of coding regions encoding one or more of<br>
acetoacetyl-CoA thiolase, HMGS, HMGR, PMK, and MPD. In other embodiments, a nucleotide<br>
sequence encoding MK is under transcriptional control of (e.g., is operably linked to) a stronger<br>
promoter than the promoter to which one or more of a nucleotide sequence encoding one or more of<br>
acetoacetyl-CoA thiolase, HMGS, HMGR, PMK, and MPD is operably linked. In other embodiments,<br>
the level of MK is increased relative to the level of one or more of acetoacetyl-Co A thiolase, HMGS,<br>
HMGR, PMK, and MPD by increasing both the copy number of MK-encoding coding regions relative<br>
to the copy number of coding regions comprising nucleotide sequences encoding one or more of<br>
acetoacetyl-CoA thiolase, HMGS, HMGR, PMK, and MPD, and by increasing promoter strength of a<br>
promoter to which an MK coding sequence is operably linked, relative to the promoter strength of a<br>
promoter to which a nucleotide sequence encoding one or more of acetoacetyl-CoA thiolase, HMGS,<br>
HMGR, PMK, and MPD is operably linked.<br>
[0069]	In some embodiments, a subject genetically modified host cell comprises a nucleic acid<br>
comprising a nucleotide sequence encoding MK, where the MK-encoding nucleotide sequence is<br>
operably linked to a stronger promoter than the promoter to which an operon comprising nucleotide<br>
sequences encoding acetoacetyl-CoA thiolase, HMGS, and HMGR is operably linked. In other<br>
embodiments, a subject genetically modified host cell comprises a first nucleic acid comprising a<br>
nucleotide sequence encoding MK, where the first nucleic acid is a high copy number expression<br>
vector; and a second nucleic acid comprising nucleotide sequences encoding acetoacetyl-CoA thiolase,<br>
HMGS, and HMGR, where the second nucleic acid is a low copy number expression vector.<br>
Controlling MK levels by copy number of MK-encoding coding regions<br>
[0070]	In some embodiments, a subject genetically modified host cell comprises a plurality of coding<br>
regions comprising a nucleotide sequence encoding MK; a nucleic acid comprising a nucleotide sequence encoding one or more mevalonate pathway en2ymes other than mevalonate kinase; and a heterologous nucleic acid comprising a nucleotide sequence encoding a terpene synthase. The plurality of (e.g., two or more) MK-encoding coding regions can be on the same nucleic acid or on physically separated nucleic acids. For example, the two or more MK-encoding coding regions are in some embodiments all present in genomic DNA of the host cell. As another example, in some embodiments, a first MK-encoding coding regions is present in genomic DNA of the host cell; and a second MK-encoding coding region is present on an extra-chromosomal recombinant vector. As yet another<br>
example, in some embodiments, the two or more MK-encoding coding regions are both included in one<br>
or more extrachromosomal recombinant vectors).<br>
[0071]	In some embodiments, a subject genetically modified host cell comprises a single copy of a<br>
coding region comprising nucleotide sequences encoding one or more of acetoacetyl-CoA thiolase,<br>
HMGS, HMGR, PMK, and MPD; two or more MK-encoding coding regions, each of which comprises<br>
nucleotide sequences encoding MK; and a heterologous nucleic acid comprising a nucleotide sequence<br>
encoding a terpene synthase. For example, in some embodiments, a subject genetically modified host<br>
cell comprising a single copy of a nucleic acid coding region comprising nucleotide sequences<br>
encoding one or more of acetoacetyl-CoA thiolase, HMGS, HMGR, PMK, and MPD, where the coding<br>
region is integrated into the host cell genome; two or more MK-encoding coding regions, each of which<br>
comprises nucleotide sequences encoding MK, where the two or more MK-encoding coding regions are<br>
extrachromosomal, such as extrachromosomal expression vectors that are not integrated into the host<br>
genome; and a heterologous nucleic acid comprising a nucleotide sequence encoding a terpene<br>
synthase.<br>
[0072]	In other embodiments, a subject genetically modified host cell comprises a nucleic acid<br>
comprising two or more coding regions encoding MK; a nucleic acid coding region comprising nucleotide sequences encoding one or more of acetoacetyl-CoA thiolase, HMGS, HMGR, PMK, and MPD; and a heterologous nucleic acid comprising a nucleotide sequence encoding a terpene synthase, where the ratio of the copy number of the MK coding regions to the copy number of the nucleic acid coding region comprising nucleotide sequences encoding one or more of acetoacetyl-CoA thiolase, HMGS, HMGR, PMK, and MPD is from about 1.5:1 to about 100:1, e.g., the ratio of the copy number of the MK-encoding coding regions to the copy number of the nucleic acid coding regions comprising nucleotide sequences encoding one or more of acetoacetyl-CoA thiolase, HMGS, HMGR, PMK, and MPD is from about 1.5:1 to about 2:1, from about 2:1 to about 2.5:1, from about 2.5:1 toabout3:l, from about 3:1 to about 5:1, from about 5:1 to about 10:1, from about 10:1 to about 25:1, from about 25:1 to about 50:1, or from about 50:1 to about 100:1.<br>
[0073]	For example, in some embodiments, a subject genetically modified host cell comprises at least<br>
two MK-encoding coding regions and a single copy of a coding region encoding one or more of acetoacetyl-CoA thiolase, HMGS, HMGR, PMK, and MPD. In some embodiments, a subject genetically modified host cell comprises two MK-encoding coding regions and a single copy of a coding region encoding acetoacetyl-CoA thiolase, HMGS, and HMGR.<br>
[0074]	In some embodiments, a subject genetically modified host cell comprises a plurality of MK-<br>
encoding coding regions, where the an MK-encoding coding region is present on a high copy number expression vector, such as a high copy number plasmid; a nucleic acid coding region comprising nucleotide sequences encoding one or more of acetoacetyl-CoA thiolase, HMGS, HMGR, PMK, and MPD, where the coding region encoding one or more of acetoacetyl-CoA thiolase, HMGS, HMGR, PMK, and MPD is present on a low copy number expression vector, such as a low copy number<br>
plasmid; and a heterologous nucleic acid comprising a nucleotide sequence encoding a terpene<br>
synthase.<br>
[0075]	In some embodiments, the copy number of an MK-encoding coding region in a subject<br>
genetically modified host cell is higher than the copy number of MK-encoding coding regions in a<br>
control host cell genetically modified with pMBI. In some embodiments, a subject genetically modified<br>
host cell comprises a plurality of coding regions comprising a nucleotide sequence encoding MK,<br>
where the MK-encoding coding regions are present on a nucleic acid comprising a p15A origin of<br>
replication (for a pl5A ori nucleotide sequence, see, e.g., nucleotides 4525-5321 of pAM39 (SEQ ID<br>
NO:3); and Selzcr ct al. (1983) Cell 32:119-129).<br>
{0076]	In some embodiments, a subject genetically modified host cell comprises a plurality of MK-<br>
encoding coding regions, where the MK-encoding coding regions are present in a high copy number<br>
plasmid; a nucleic acid coding region comprising nucleotide sequences encoding one or more of<br>
acetoacetyl-CoA thiolase, HMGS, HMGR, PMK, and MPD, where the coding region comprising<br>
nucleotide sequences encoding one or more of acetoacetyl-CoA thiolase, HMGS, HMGR, PMK, and<br>
MPD is present in a medium copy number plasmid; and a heterologous nucleic acid comprising a<br>
nucleotide sequence encoding a terpene synthase.<br>
[0077]	In some embodiments, a subject genetically modified host cell comprises a plurality of MK-<br>
encoding coding regions, where the MK-encoding coding regions are present in a medium copy number<br>
plasmid; a nucleic acid coding region comprising nucleotide sequences encoding one or more of<br>
acetoacetyl-CoA thiolase, HMGS, HMGR, PMK, and MPD, where the coding region comprising<br>
nucleotide sequences encoding one or more of acetoacetyl-Co A thiolase, HMGS, HMGR, PMK, and<br>
MPD is present in a low copy number plasmid; and a heterologous nucleic acid comprising a nucleotide<br>
sequence encoding a terpene synthase.<br>
[0078]	In other embodiments, a subject genetically modified host cell comprises a first nucleic acid<br>
comprising an MK-encoding coding region, where the first nucleic acid is a high copy number<br>
expression vector; a second nucleic acid comprising nucleotide sequences encoding acetoacetyl-CoA<br>
thiolase, HMGS, and HMGR, where the second nucleic acid is a low copy number expression vector;<br>
and a heterologous nucleic acid comprising a nucleotide sequence encoding a terpene synthase.<br>
[0079]	Low copy number plasmids generally provide fewer than about 20 plasmid copies per cell (e.g.,<br>
from about 1 plasmid copy per cell to about 5 plasmid copies per cell, from about 5 plasmid copies per cell to about 10 plasmid copies per cell, from about 10 plasmid copies per cell to about 15 plasmid copies per cell, or from about 15 plasmid copies per cell to about 20 plasmid copies per cell); medium copy number plasmids generally provide from about 20 plasmid copies per cell to about 50 plasmid copies per cell, or from about 20 plasmid copies per cell to about 80 plasmid copies per cell; and high copy number plasmids generally provide from about 80 plasmid copies per cell to about 200 plasmid copies per cell, or more than 200 plasmid copies per cell.<br>
[0080]	Suitable low copy expression vectors for prokaryotic cells (e.g., Escherichia coif) include, but<br>
are not limited to, pACYC184, pBeloBacl 1, pBR332, pBAD33, pBBRlMCS and its derivatives, pSClOl, SuperCos (cosmid), andpWE15 (cosmid). Suitable medium copy expression vectors for prokaryotic cells (e.g., E. colt) include, but are not limited to pTrc99A, pBAD24, and vectors containing a ColEl origin of replication and its derivatives. Suitable high copy number expression vectors for prokaryotic cells (e.g., E. coli) include, but are not limited to, pUC, pBluescript, pGEM, and pTZ vectors.<br>
[0081]	Suitable low-copy (centromeric) expression vectors for eukaryotic cells (e.g., yeast cells)<br>
include, but are not limited to, pRS415 and pRS416 (Sikorski &amp; Hieter (1989) Genetics 122:19-27). Suitable high-copy 2 micron expression vectors for eukaryotic cells (e.g., yeast cells) include, but are not limited to, pRS425 andpRS426 (Christainson et al. (1992) Gene 110:119-122). Alternative 2 micron expression vectors include non-selectable variants of the 2 micron vector (Bruschi &amp; Ludwig (1988) Curr. Genet. 15:83-90) or intact 2 micron plasmids bearing an expression cassette (as exemplified in U.S. Pat. Application No. 20050084972). Controlling MK levels bv promoter strength<br>
10082]	In some embodiments, a subject genetically modified host cell comprises a plurality of nucleic<br>
acid coding regions comprising nucleotide sequences encoding MK, where the MK-encoding coding regions are each operably linked to (e.g., under transcriptional control of) a first transcriptional control element (e.g., a first promoter); a nucleic acid comprising a nucleotide sequence encoding one or more mevalonate pathway enzymes other than mevalonate kinase (e.g., encoding one or more of acetoacetyl-CoA thiolase, HMGS, HMGR, PMK, and MPD), where the nucleotide sequence(s) encoding one or more of acetoacetyl-CoA thiolase, HMGS, HMGR, PMK, and MPD is operably linked to (e.g., under transcriptional control of) a second transcriptional control element (e.g., a second promoter); and a heterologous nucleic acid comprising a nucleotide sequence encoding a terpene synthase, where the first transcriptional control element is stronger than the second transcriptional control element.<br>
[0083]	In some embodiments, a subject genetically modified host cell comprises a plurality of nucleic<br>
acid coding regions comprising nucleotide sequences encoding MK, where the MK-encoding coding<br>
regions are each operably linked to (e.g., under transcriptional control of) a first promoter; a nucleic<br>
acid comprising a nucleotide sequence encoding one or more mevalonate pathway enzymes other than<br>
mevalonate kinase (e.g., encoding one or more of acetoacetyl-CoA thiolase, HMGS, HMGR, PMK, and<br>
MPD), where the nucleotide sequences) encoding one or more of acetoacetyl-CoA thiolase, HMGS,<br>
HMGR, PMK, and MPD is operably linked to (e.g., under transcriptional control of) a second promoter,<br>
where the first promoter is stronger than the second promoter; and a heterologous nucleic acid<br>
comprising a nucleotide sequence encoding a terpene synthase.<br>
[0084]	In some embodiments, a coding region is contained within an operon. In some embodiments, a<br>
subject genetically modified host cell comprises a nucleic acid comprising a first operon comprising nucleotide sequences encoding MK, PMK, and MPD, where the first operon is operably linked to a first<br>
promoter; a nucleic acid comprising a second operon comprising nucleotide sequences encoding<br>
acetoacetyl-CoA thiolase, HMGS, and HMGR, where the second operon is operably linked to a second<br>
promoter, where the firet promoter is stronger than the second promoter; and a heterologous nucleic<br>
acid comprising a nucleotide sequence encoding a terpene synthase.<br>
[0085]	Regulatory elements include, for example, promoters and operators. A promoter is a sequence<br>
of nucleotides that initiates and controls the transcription of a nucleic acid sequence by an RNA<br>
polymerase enzyme. An operator is a sequence of nucleotides adjacent to the promoter that functions to<br>
control transcription of the desired nucleic acid sequence. The operator contains a protein-binding<br>
domain where a specific repressor protein can bind. In the absence of a suitable repressor protein,<br>
transcription initiates through the promoter. In the presence of a suitable repressor protein, the repressor<br>
protein binds to the operator and thereby inhibits transcription from the promoter.<br>
[0086]	In some embodiments of the present invention, promoters used in expression vectors are<br>
inducible. In other embodiments, the promoters used in expression vectors are constitutive. In some<br>
embodiments, one or more nucleic acid sequences are operably linked to an inducible promoter, and<br>
one or more other nucleic acid sequences are operably linked to a constitutive promoter.<br>
[0087]	Non-limiting examples of suitable promoters for use in prokaryotic host cells include a<br>
bacteriophage T7 RNA polymerase promoter, a trp promoter; a lac operon promoter; a hybrid promoter,<br>
for example, a lac/tac hybrid promoter, a tac/trc hybrid promoter, a trp/lac promoter, a T7/lac promoter;<br>
a trc promoter, a tac promoter, and the like; an araBAD promoter; in vivo regulated promoters, such as<br>
an ssaG promoter or a related promoter (see, for example, U.S. Patent Publication No. 20040131637), a<br>
pagC promoter (Pulkkinen and Miller, J. Bacterid. (1991) 173(l):86-93; Alpuche-Aranda et al. (1992)<br>
Proc. Natl. Acad. Sci. USA. 89(21):10079-83), a nirB promoter (Harbome et al. (1992) Mol. Micro.<br>
6:2805-2813), and the like (see, for example, Dunstan et al. (1999) Infect. Immun. 67:5133-5141;<br>
McKelvie et al. (2004) Vaccine 22:3243-3255; and Chatfield et al. (1992) Biotechnol. 10:888-892); a<br>
sigma70 promoter, for example, a consensus sigma70 promoter (see, for example, GenBank Accession<br>
Nos. AX798980, AX798961, and AX798183); a stationary phase promoter, for example, a dps<br>
promoter, an spv promoter, and the like; a promoter derived from the pathogenicity island SPI-2 (see,<br>
for example, W096717951); an actA promoter (see, for example, Shetron-Rama et al. (2002) Infect.<br>
Immun. 70:1087-1096); an rpsM promoter (see, for example, Valdivia and Falkow (1996) Mol.<br>
Microbiol. 22:367 378); a tet promoter (see, for example, Hillen et al. (1989) In Saenger W. and<br>
Heinemann U. (eds) Topics in Molecular and Structural Biology, Protein—Nucleic Acid Interaction.<br>
Macmillan, London, UK, Vol. 10, pp. 143—162); an SP6 promoter (see, for example, Melton et al.<br>
(1984) Nucl. Acids Res. 12:7035-7056); and the like.<br>
[0088]	In another embodiment of the present invention, the total activity of a heterologous mevalonate<br>
kinase relative to other mevalonate pathway enzymes in a host microorganism is increased by expressing the enzyme from a strong promoter. In some embodiments, the mevalonate kinase-encoding nucleotide sequence is operably linked to a promoter that is a stronger promoter than the PLAC promoter<br><br>
(SEQ ID N0:21), e.g., the MK-encoding nucleotide sequence is operably linked to a promoter that is at<br>
least about 10%, at least about 25%, at least about 50%, at least about 2-fold, at least about 5-fold, or at<br>
least about 10-fold, or more, stronger than a PLAC promoter having a nucleotide sequence as set forth in<br>
SEQIDNO:21.<br>
[0089]	Suitable strong promoters for use in prokaryotic cells (e.g., Escherichia coli) include, but are<br>
not limited to, a lacUV5 promoter (see, e.g., SEQ ID NO:22), Trc, Tac, T5, T7, and Plambda. In another embodiment of the present invention, the total activity of the mevalonate kinase in a host microorganism is increased by expressing the enzyme from a strong promoter on a high copy number plasmid. Suitable examples, for use in prokaryotic cells (e.g., Escherichia coli) include, but are not limited to using Trc, Tac, T5, T7, and Pumbd. promoters with pBAD24, pBADl 8, pGEM, pBluescript, pUC, and pTZ vectors.<br>
[0090]	Non-limiting examples of suitable promoters for use in eukaryotic host cells include, but are<br>
not limited to, a CMV immediate early promoter, an HSV thymidine kinase promoter, an early or late SV40 promoter, LTRs from retroviruses, and a mouse metallothionein-I promoter.<br>
[0091]	Non-limiting examples of suitable constitutive promoters for use in prokaryotic host cells<br>
include a sigma70 promoter (for example, a consensus sigma70 promoter). Non-limiting examples of suitable inducible promoters for use in bacterial host cells include the pL of bacteriophage .; Plac; Ptrp; Ptac (Ptrp-lac hybrid promoter); an isopropyl-beta-D44 thiogalactopyranoside (IPTG)-inducible promoter, for example, a lacZ promoter; a tetracycline inducible promoter; an arabinose inducible promoter, for example, PBAD (see, for example, Guzman et al. (1995) J. Bacteriol. 177:4121-4130); a xylose-inducible promoter, for example, Pxyl (see, for example, Kim et al. (1996) Gene 181:71-76); a GAL1 promoter; a tryptophan promoter; a lac promoter; an alcohol-inducible promoter, for example, a methanol-inducible promoter, an ethanol-inducible promoter; a raffinose-inducible promoter; a heat-inducible promoter, for example, heat inducible lambda PL promoter, a promoter controlled by a heat-sensitive repressor (for example, CI857-repressed lambda-based expression vectors; see, for example, Hoffmann et al. (1999) FEMS Microbiol Lett. 177(2):327-34); and the like.<br>
[0092]	Non-limiting examples of suitable constitutive promoters for use in yeast host cells include an<br>
ADH1, an ADH2, a PGK, or a LEU2 promoter. Non-limiting examples of suitable inducible promoters for use in yeast host cells include, but are not limited to, a divergent galactose-inducible promoter such as a GAL 1 or a GAL 10 promoter (West at al. (1984) Mol. Cell. Biol. 4(11):2467-2478), or a CUP1 promoter. Where desired, the promoter that is stronger man a native E. coli Lac promoter.<br>
[0093]	Non-limiting examples of operators for use in bacterial host cells include a lactose promoter<br>
operator (Lad repressor protein changes conformation when contacted with lactose, thereby preventing the Lacl repressor protein from binding to the operator), a tryptophan promoter operator (when complexed with tryptophan, TrpR repressor protein has a conformation that binds the operator; in the absence of tryptophan, the TrpR repressor protein has a conformation that does not bind to the<br>
operator), and a tac promoter operator (see, for example, deBoer et al. (1983) Proc. Natl. Acad. Sci. U.S.A. 80:21-25.). Mevalonate pathway enzymes<br>
[0094]	The mevalonate pathway comprises: (a) condensing two molecules of acetyl-CoA to<br>
acetoacetyl-CoA; (b) condensing acetoacetyl-CoAwith acetyl-CoAto form HMG-Co A; (c) converting HMG-CoA to mevalonate; (d) phosphorylating mevalonate to mevalonate 5-phosphate; (e) converting mevalonate 5-phosphate to mevalonate 5-pyrophosphate; and (f) converting mevalonate 5-pyrophosphate to isopentenyl pyrophosphate. The mevalonate pathway enzymes required for production of IPP vary, depending on the culture conditions.<br>
[0095]	In some embodiments, a subject genetically modified host cell comprises a nucleic acid<br>
comprising nucleotide sequences encoding a mevalonate kinase, as described above; a nucleic acid comprising a nucleotide sequence encoding one or more mevalonate pathway enzymes other than mevalonate kinase; and a heterologous nucleic acid comprising a nucleotide sequence encoding a terpene synthase. Nucleotide seauences encoding mevalonate pathway enzymes<br>
[0096]	Nucleotide sequences encoding mevalonate (MEV) pathway gene products are known in the<br>
art, and any known MEV pathway gene product-encoding nucleotide sequence can used to generate a<br>
subject genetically modified host cell. For example, nucleotide sequences encoding acetoacetyl-CoA<br>
thiolase, HMGS, HMGR, MK, PMK, MPD, and EDI are known in the art. The following are non-<br>
limiting examples of known nucleotide sequences encoding MEV pathway gene products, with<br>
GenBank Accession numbers and organism following each MEV pathway enzyme, in parentheses:<br>
acetoacetyl-CoA thiolase: (NC_000913 REGION: 2324131..2325315; E. coli), (D49362; Paracoccus<br>
denitrificans), and (L20428; Saccharomyces cerevisiae); HMGS: (NC_001145. complement<br>
19061..20536; Saccharomyces cerevisiae), (X96617; Saccharomyces cerevisiae), (X83882;<br>
Arabidopsis thaliand), (AB037907; Kitasatospora griseola), and (BT007302; Homo sapiens); HMGR:<br>
(NM_206548; Drosophila melanogaster), (NM_204485; Gallus gallus), (AB015627; Streptomyces sp.<br>
KO-3988), (AF542543; Nicotiana attenuata), (AB037907; Kitasatospora griseola), (AX128213,<br>
providing the sequence encoding a truncated HMGR; Saccharomyces cerevisiae), and (NC_001145:<br>
complement (115734.. 118898; Saccharomyces cerevisiae)); MK: (L77688; Arabidopsis thaliand), and<br>
(X55875; Saccharomyces cerevisiae); PMK: (AF429385; Hevea brasiliensis), (NM006556; Homo<br>
sapiens), (NC_001145. complement 712315..713670; Saccharomyces cerevisiae); MPD: (X97557;<br>
Saccharomyces cerevisiae), (AF290095; Enterococcus faecium), and (U49260; Homo sapiens); and<br>
IDI: (NC_000913, 3031087..3031635; E. colt), and (AF082326; Haematococcuspluvialis).<br>
[0097]	A non-limiting example of nucleotide sequences encoding aceoacetyl-CoA thiolase, HMGS,<br>
and HMGR is set forth in Figures 13A-C (SEQ ID NO:l) of U.S. Patent No. 7,183,089. A non-limiting<br>
example of nucleotide sequences encoding MK, PMK, MPD, and isopentenyl diphosphate isomerase<br>
(TDI) is set forth in Figures 16A-D of U.S. Patent No. 7,183,089.	..<br>
[0098]	In some embodiments, the HMGR coding region is set forth in SEQ ID NO: 13 of U.S. Patent<br>
No. 7,183,089 (see also Figures 20A-C of U.S. Patent No. 7,183,089), which encodes a truncated form of HMGR ("tHMGR") that lacks the transmembrane domain of wild-type HMGR. The transmembrane domain of HMGR contains the regulatory portions of the enzyme and has no catalytic activity.<br>
[0099]	The coding sequence of any known MEV pathway enzyme may be altered in various ways<br>
known in the art to generate targeted changes in the amino acid sequence of the encoded enzyme. The amino acid of a variant MEV pathway enzyme will usually be substantially similar to the amino acid sequence of any known MEV pathway enzyme, i.e. will differ by at least one amino acid, and may differ by at least two, at least 5, at least 10, or at least 20 amino acids, but typically not more than about fifty amino acids. The sequence changes may be substitutions, insertions or deletions. For example, as described below, the nucleotide sequence can be altered for the codon bias of a particular host cell. In addition, one or more nucleotide sequence differences can be introduced that result in conservative amino acid changes in the encoded protein. DXP pathway enzvmes<br>
[00100]	The DXP pathway comprises: l-deoxy-D-xylulose-5-phosphate synthase (Dxs), 1-deoxy-D-<br>
xylulose-5-phosphate reductoisomerase (IspC), 4-diphosphocytidyl-2-C-methyl-D-erythritol synthase<br>
(IspD), 4-diphosphocytidyl-2-C-memyl-D-eiythritol kinase (IspE), 2C-methyl-D-erythritol 2,4-<br>
cyclodiphosphate synthase (IspF), and 1 -hydroxy-2-methyl-2-(£)-butenyl 4-diphosphate synthase<br>
(IspG).<br>
[00101]	In some embodiments, a subject genetically modified host cell comprises a nucleic acid<br>
comprising nucleotide sequences encoding a mevalonate kinase, as described above; a nucleic acid<br>
comprising a nucleotide sequence encoding one or more mevalonate pathway enzymes other than<br>
mevalonate kinase; and a heterologous nucleic acid comprising a nucleotide sequence encoding a<br>
terpene synthase; where an endogenous DXP pathway in the host cell is functionally disabled.<br>
Prenyltransferases<br>
[00102]	In some embodiments, a subject genetically modified host cell comprises a nucleic acid<br>
comprising nucleotide sequences encoding a mevalonate kinase, as described above; a nucleic acid<br>
comprising a nucleotide sequence encoding one or more mevalonate pathway enzymes other than<br>
mevalonate kinase; a heterologous nucleic acid comprising a nucleotide sequence encoding a<br>
prenyltransferase; and a heterologous nucleic acid comprising a nucleotide sequence encoding a terpene<br>
synthase.<br>
[00103]	Prenyltransferases constitute a broad group of enzymes catalyzing the consecutive<br>
condensation of IPP resulting in the formation of prenyl diphosphates of various chain lengths. Suitable prenyltransferases include enzymes that catalyze the condensation of IPP with allylic primer substrates to form isoprenoid compounds with from about 2 isoprene units to about 6000 isoprene units or more, e.g., 2 isoprene units (Geranyl Pyrophosphate synthase), 3 isoprene units (Famesyl pyrophosphate synthase), 4 isoprene units (geranylgeranyl pyrophosphate synthase), 5 isoprene units, 6<br>
isoprene units (hexadecylpyrophosphate synthase), 7 isoprene units, 8 isoprene units (phytoene<br>
synthase, octaprenyl pyrophosphate synthase), 9 isoprene units (nonaprenyl pyrophosphate synthase, 10<br>
isoprene units (decaprenyl pyrophosphate synthase), from about 10 isoprene units to about 15 isoprene<br>
units, from about 15 isoprene units to about 20 isoprenc units, from about 20 isoprene units to about 25<br>
isoprene units, from about 25 isoprene units to about 30 isoprene units, from about 30 isoprene units to<br>
about 40 isoprene units, from about 40 isoprene units to about 50 isoprene units, from about 50 isoprene<br>
units to about 100 isoprene units, from about 100 isoprene units to about 250 isoprene units, from about<br>
250 isoprene units to about 500 isoprene units, from about 500 isoprene units to about 1000 isoprene<br>
units, from about 1000 isoprene units to about 2000 isoprene units, from about 2000 isoprene units to<br>
about 3000 isoprene units, from about 3000 isoprene units to about 4000 isoprene units, from about<br>
4000 isoprene units to about 5000 isoprene units, or from about 5000 isoprene units to about 6000<br>
isoprene units or more.<br>
[00104]	Suitable prenyltransferases include, but are not limited to, an 2s-isoprenyl diphosphate synthase,<br>
including, but not limited to, geranyl diphosphate (GPP) synmase, farnesyl diphosphate (FPP) synthase,<br>
geranylgeranyl diphosphate (GGPP) synthase, hexaprenyl diphosphate (HexPP) synthase, heptaprenyl<br>
diphosphate (HepPP) synthase, octaprenyl (OPP) diphosphate synthase, solanesyl diphosphate (SPP)<br>
synthase, decaprenyl diphosphate (DPP) synthase, chicle synthase, and gutta-percha synthase; and a Z-<br>
isoprenyl diphosphate synthase, including, but not limited to, nonaprenyl diphosphate (NPP) synthase,<br>
undecaprenyl diphosphate (UPP) synthase, dehydrodolichyl diphosphate synthase, eicosaprenyl<br>
diphosphate synthase, natural rubber synthase, and other Z-isoprenyl diphosphate synthases.<br>
[00105]	The nucleotide sequences of a numerous prenyl transferases from a variety of species are<br>
known, and can be used or modified for use in generating a subject genetically modified host cell.<br>
Nucleotide sequences encoding prenyl transferases are known in the art. See, e.g., Human famesyl<br>
pyrophosphate synthetase mRNA (GenBank Accession No. J05262; Homo sapiens); famesyl<br>
diphosphate synthetase (FPP) gene (GenBank Accession No. J05091; Saccharomyces cerevisiae);<br>
isopentenyl diphosphate:dimethylallyl diphosphate isomerase gene (J05090; Saccharomyces<br>
cerevisiae); Wang and Ohnuma (2000) Biochim. Biophys. Acta 1529:33^18; U.S. Patent No. 6,645,747;<br>
Arabidopsis thaliana farnesyl pyrophosphate synthetase 2 (FPS2) / FPP synthetase 2 / farnesyl<br>
diphosphate synthase 2 (At4gl 7190) mRNA (GenBank Accession No. NM_202836); Ginkgo biloba<br>
geranylgeranyl diphosphate synthase (ggpps) mRNA (GenBank Accession No. AY371321);<br>
Arabidopsis thaliana geranylgeranyl pyrophosphate synthase (GGPS1) / GGPP synthetase /<br>
farnesyltranstransferase (At4g36810) mRNA (GenBank Accession No. NM119845); Synechococcus<br>
elongatus gene for farnesyl, geranylgeranyl, geranylfarnesyl, hexaprenyl, heptaprenyl diphosphate<br>
synthase (SelF-HepPS) (GenBank Accession No. ABO 16095); etc.<br>
Terpene synthases<br>
[00106]	A nucleic acid comprising a nucleotide sequence encoding any known terpene synthase can be<br>
used. Suitable terpene synthases include, but are not limited to, amorpha-4,1 l-diene synthase (ADS),<br>
beta-caryophyllene synthase, germacrene A synthase, 8-epicedrol synthase, valencene synthase, (+•)-<br>
delta-cadinene synthase, germacrene C synthase, (E)-beta- famesene synthase, Casbene synthase,<br>
vetispiradiene synthase, 5-epi-aristolochene synthase, Aristolchcne synthase, beta-caryophyllene, alpha-<br>
humulene, (E,E)-alpha-famesene synthase, (-)-beta-pinene synthase, Gamma-terpinene synthase,<br>
limonene cyclase, Linalool synthase,l,8-cineole synthase, (+)-sabinene synthase, E-alpha-bisabolene<br>
synthase, (+)-bornyl diphosphate synthase, lcvopimaradiene synthase, Abietadiene synthase,<br>
isopimaradiene synthase,(E)-gamma-bisabolenc synthase, taxadiene synthase, copalyl pyrophosphate<br>
synthase, kaurene synthase, longifolene synthase, gamma-humulene synthase, Delta-selinene synthase,<br>
beta-phellandrene synthase, limonene synthase, myrcene synthase, terpinolene synthase, (-)-camphene<br>
synthase, (+)-3-carene synthase, syn-copalyl diphosphate synthase, alpha-terpineol synthase, syn-<br>
pimara-7,15-dienc synthase, ent-sandaaracopimaradiene synthase, stemer-13-ene synthase, E-beta-<br>
ocimene, S-linalool synthase, geraniol synthase, gamma-terpinene synthase, linalool synthase, E-beta-<br>
ocimene synthase, epi-cedrol synthase, alpha-zingiberene synthase, guaiadiene synthase, cascarilladiene<br>
synthase, cis-muuroladiene synthase, aphidicolan-16b-ol synthase, elizabethatriene synthase, sandalol<br>
synthase, patchoulol synthase, Zinzanol synthase, cedrol synthase, scareol synthase, copalol synthase,<br>
manool synthase, and the like.<br>
[00107]	Nucleotide sequences encoding terpene synthases are known in the art, and any known terpene<br>
synthase-encoding nucleotide sequence can used to genetically modify a host cell. For example, the<br>
following terpene synthase-encoding nucleotide sequences, followed by their GenBank accession<br>
numbers and the organisms in which they were identified, are known and can be used: (-)-germacrene<br>
D synthase mRNA (AY438099; Populus balsamifera subsp. trichocarpa x Populus deltoids); E,E-<br>
alpha-farnesene synthase mRNA (AY640154; Cucumis sativus); 1,8-cineole synthase mRNA<br>
(AY691947; Arabidopsis thaliana); terpene synthase 5 (TPS5) mRNA (AY518314; Zea mays); terpene<br>
synthase 4 (TPS4) mRNA (AY518312; Zea mays); myrcene/ocimene synthase (TPS10) (At2g24210)<br>
mRNA (NM_127982; Arabidopsis thaliana); geraniol synthase (GES) mRNA (AY362553; Ocimum<br>
basilicum); pinene synthase mRNA (AY237645; Picea sitchensis); myrcene synthase le20 mRNA<br>
(AY195609; Antirrhinum majus); (E)-0-ocimene synthase (0e23) mRNA (AY195607; Antirrhinum<br>
majus); E-p-ocimene syntiiase mRNA (AY151086; Antirrhinum majus); terpene synthase mRNA<br>
(AF497492; Arabidopsis thaliana); (-)-camphene synthase (AG6.5) mRNA (U87910; Abies grandis); (-<br>
)-4S-limoncne synthase gene (e.g., genomic sequence) (AF326518; Abies grandis); delta-selinene<br>
synthase gene (AF326513; Abies grandis); amorpha-4,11 -diene synthase mRNA (AJ251751; Artemisia<br>
annua); E-a-bisabolene synthase mRNA (AF006195; Abies grandis); gamma-humulene synthase<br>
mRNA (U92267; Abies grandis); 8-selinene synthase mRNA (U92266; Abies grandis); pinene syndiase<br>
(AG3.18) mRNA (U87909; Abies grandis); myrcene synthase (AG2.2) mRNA (U87908; Abies<br>
grandis); etc.<br>
[00108]	Amino acid sequences of the following terpene synthases are found under the GenBank<br>
Accession numbers shown in parentheses, along with the organism in which each was identified,<br>
following each terpene synthase: (-)-germacrene D synthase (AAR99061; Populus balsamifera subsp.<br>
trichocarpa x Populus deltoids); D-cadinene synthase (P93665; Gossypium hirsutum); 5-epi-<br>
aristolochene synthase (Q40577; Nicotiana tabacum); E,E-alpha-farnesene synthase (AAU05951;<br>
Cucumis sativus); 1,8-cineole synthase (AAU01970; Arabidopsis thaliana); (R)-limonene synthase 1<br>
(Q8L5K3; Citrus limori); syn-copalyl diphosphate synthase (AAS98158; Oryza sativa); a taxadiene<br>
synthase (Q9FT37; Taxus chinensis; Q93YA3; Taxus bacca; Q41594; Taxus brevifolia); a D-cadinene<br>
synthase (Q43714; Gossypium arboretum); terpene synthase 5 (AAS88575; Zea mays); terpene<br>
synthase 4 (AAS88573; Zea mays); terpenoid synthase (AAS79352; Vttis vinifera); geraniol synthase<br>
(AAR11765; Ocimum basilicum); myrcene synthase le20 (AA041727; Antirrhinum majus); 5-epi-<br>
aristolochene synthase 37 (AAP05762; Nicotiana attenuata); (+)-3-carene synthase (AA073863; Picea<br>
abies); (-)-camphene synthase (AAB70707; Abies grandis); abietadiene synthase (AAK83563; Abies<br>
grandis); amorpha-4,11-diene synthase (GAB94691; Artemisia annua); trichodiene synthase<br>
(AAC49957; Myrothecium roridum); gamma-humulene synthase (AAC05728; Abies grandis); 5-<br>
selinene synthase (AAC05727; Abies grandis); etc.<br>
Codon optimization<br>
[00109]	In some embodiments, a nucleotide sequence encoding an enzyme (e.g., MK; a<br>
mevalonate pathway enzyme other than MK; a prenyltransferase; a terpene synthase) is<br>
modified to reflect the codon preference for the particular host cell. For example, the<br>
nucleotide sequence will in some embodiments be modified for yeast codon preference.  See,<br>
e.g., Bennetzen and Hall (1982) /. Biol. Chem. 257(6): 3026-3031. As another non-limiting<br>
example, the nucleotide sequence will in other embodiments be modified for E. coli codon<br>
preference. See, e.g., Gouy and Gautier (1982) Nucleic Acids Res. 10(22): 7055-7074; Eyre-<br>
Walker (1996) Mol. Biol. Evol. 13(6):864-872. See also Nakamura et al. (2000) Nucleic Acids<br>
Res. 28(1):292. Such codon modification is also referred to as "codon optimization."<br>
Expression constructs<br>
[00110]	In some embodiments, a subject genetically modified host cell comprises a nucleic acid<br>
comprising nucleotide sequences encoding MK (as described above); a nucleic acid comprising a nucleotide sequence encoding one or more mevalonate pathway enzymes other than MK; and a heterologous nucleic acid comprising a nucleotide sequence encoding a terpene synthase; where one or more of the nucleic acids is present in an expression vector. In some embodiments, a subject genetically modified host cell comprises a nucleic acid comprising nucleotide sequences encoding MK (as described above); a nucleic acid comprising a nucleotide sequence encoding one or more mevalonate pathway enzymes other than MK; a heterologous nucleic acid comprising a nucleotide sequence encoding a terpene synthase; and a heterologous nucleic acid comprising a nucleotide sequence encoding a prenyltransferase; where one or more of the nucleic acids is present in an expression vector.<br>
[00111]	Suitable expression vectors include, but are not limited to, baculovirus vectors, bacteriophage<br>
vectors, plasmids, phagemids, cosmids, fosmids, bacterial artificial chromosomes, viral vectors (e.g.<br>
viral vectors based on vaccinia virus, poliovirus, adenovirus, adeno-associated virus, SV40, herpes<br>
simplex virus, and the like), PI-based artificial chromosomes, yeast plasmids, yeast artificial<br>
chromosomes, and any other vectors specific for specific hosts of interest (such as E. coli and yeast).<br>
Suitable vectors include chromosomal, nonchromosomal and synthetic DNA sequences.<br>
{00112]	Numerous suitable expression vectors are known to those of skill in the art, and many are<br>
commercially available. The following vectors are provided by way of example; for bacterial host cells:<br>
pQE vectors (Qiagen), pBluescript plasmids, pNH vectors, lambda-ZAP vectors (Stratagene); pTrc99a,<br>
pKK223-3, pDR540, and pRIT2T (Pharmacia); for eukaryotic host cells: pXTl, pSG5 (Stratagene),<br>
pSVK3, pBPV, pMSG, and pSVLSV40 (Pharmacia). However, any other plasrnid or other vector may<br>
be used so long as it is compatible with the host cell.<br>
[00113]	Depending on the host/vector system utilized, any of a number of suitable transcription and<br>
translation control elements, including constitutive and inducible promoters, transcription enhancer<br>
elements, transcription terminators, etc. may be used in the expression vector (see e.g., Bitter et al.<br>
(1987) Methods in Enzymology, 153:516-544).<br>
[00114]	Suitable promoters for use in prokaryotic host cells include, but are not limited to, a<br>
bacteriophage T7 KNA polymerase promoter; a trp promoter; a lac operon promoter; a hybrid promoter,<br>
e.g., a lac/tac hybrid promoter, a tac/trc hybrid promoter, a trp/lac promoter, a T7/lac promoter; a trc<br>
promoter; a tac promoter, and the like; an araBAD promoter; in vivo regulated promoters, such as an<br>
ssaG promoter or a related promoter (see, e.g., U.S. Patent Publication No. 20040131637), zpagC<br>
promoter (Pulkkinen and Miller, J. Bacteriol., 1991: 173(1): 86-93; Alpuche-Aranda et al., PNAS,<br>
1992; 89(21): 10079-83), a nirB promoter (Harborne et al. (1992) Mol. Micro. 6:2805-2813), and the<br>
like (see, e.g., Dunstan et al. (1999)/«/ecf. Immun. 67:5133-5141; McKelvie et al. (2004) Vaccine<br>
22:3243-3255; and Chatfield et al. (1992) Biotechnol. 10:888-892); a sigma70 promoter, e.g., a<br>
consensus sigma70 promoter (see, e.g., GenBank Accession Nos. AX798980, AX798961, and<br>
AX798183); a stationary phase promoter, e.g., a dps promoter, an spv promoter, and the like; a<br>
promoter derived from the pathogenicity island SPI-2 (see, e.g., W096/17951); an actA promoter (see,<br>
e.g., Shetron-Rama et al. (2002) Infect. Immun. 70:1087-1096); an rpsM promoter (see, e.g., Valdivia<br>
and Falkow (1996). Mol. Microbiol. 22:367-378); a tet promoter (see, e.g., Hillen,W. and<br>
Wissmann.A. (1989) In Saenger.W. and Heinemann,U. (eds), Topics in Molecular and Structural<br>
Biology, Protein-Nucleic Acid Interaction. Macmillan, London, UK, Vol. 10, pp. 143-162); an SP6<br>
promoter (see, e.g., Melton et al. (1984) Nucl. Acids Res. 12:7035-7056); and the like.<br>
[00115]	Non-limiting examples of suitable eukaryotic promoters include CMV immediate early, HSV<br>
thymidine kinase, early and late SV40, LTRs from retrovirus, and mouse metallothionein-I. Suitable promoters for expression in yeast include, but are not limited to, CYC1, HIS3, GAL1, GAL10, ADH1, PGK, PH05, GAPDH, ADC1, TRP1, URA3, LEU2, ENO, and TP1; and, e.g., AOX1 (e.g., for use in<br>
Pichia). Selection of the appropriate vector and promoter is well within the level of ordinary skill in the art. The expression vector may also contain a ribosome binding site for translation initiation and a transcription terminator. The expression vector may also include appropriate sequences for amplifying expression.<br>
[00116]	In addition, the expression vectors include one or more selectable marker genes to provide a<br>
phenotypic trait for selection of transformed host cells such as dihydrofolate reductase or neomycin resistance for eukaryotic cell culture, or such as tetracycline or ampicillin resistance in prokaryotic host cells such as E. coli.<br>
[00117]	Generally, an expression vectors will include origins of replication and selectable markers<br>
permitting transformation of the host cell, e.g., the ampicillin resistance gene of E. coli, the S. cerevisiae TRP1 gene, etc.; and a promoter derived from a highly-expressed gene to direct transcription of the coding sequence. Such promoters can be derived from operons encoding glycolytic enzymes such as 3-phosphoglycerate kinase (PGK), a-factor, acid phosphatase, or heat shock proteins, among others.<br>
[00118]	In some embodiments, a nucleotide sequence encoding an enzyme (e.g., a mevalonate pathway<br>
enzyme; a terpene synthase; a prenyltransferase) is operably linked to ah inducible promoter. Inducible promoters are well known in the art. Suitable inducible promoters include, but are not limited to, the pL of bacteriophage X; Plac; Ptrp; Ptac (Ptrp-lac hybrid promoter); an isopropyl-beta-D-thiogalactopyranoside (IPTG)-inducible promoter, e.g., a lacZ promoter; a tetracycline-inducible promoter, an arabinose inducible promoter, e.g., PBAD (see, e.g., Guzman et al. (1995) J. Bacteriol. 177:4121-4130); a xylose-inducible promoter, e.g., Pxyl (see, e.g., Kim et al. (1996) Gene 181:71-76); a GAL1 promoter; a tryptophan promoter; a lac promoter; an alcohol-inducible promoter, e.g., a methanol-inducible promoter, an ethanol-inducible promoter; a raffinose-inducible promoter, a heat-inducible promoter, e.g., heat inducible lambda PL promoter, a promoter controlled by a heat-sensitive repressor (e.g., CI857-repressed lambda-based expression vectors; see, e.g., Hoffmann et al. (1999) FEMS Microbiol Lett. 177(2):327-34); and the like.<br>
[00119]	In some embodiments, a nucleotide sequence encoding an enzyme (e.g., a mevalonate pathway<br>
enzyme; a terpene synthase; a prenyltransferase) is operably linked to a constitutive promoter. Suitable constitutive promoters for use in prokaryotic cells are known in the art and include, but are not limited to, a sigma70 promoter, e.g., a consensus sigma70 promoter.<br>
[00120]	In yeast, a number of vectors containing constitutive or inducible promoters may be used. For a<br>
review see, Current Protocols in Molecular Biology, Vol. 2, 1988, Ed. Ausubel, et al., Greene Publish. Assoc. &amp; Wiley Interscience, Ch. 13; Grant, et al., 1987, Expression and Secretion Vectors for Yeast, in Methods in Enzymology, Eds. Wu &amp; Grossman, 31987, Acad. Press, N.Y., Vol. 153, pp.516-544; Glover, 1986, DNA Cloning, Vol. II, IRL Press, Wash., D.C., Ch. 3; and Bitter, 1987, Heterologous Gene Expression in Yeast, Methods in Enzymology, Eds. Berger &amp; Kimmel, Acad. Press, N.Y., Vol. 152, pp. 673-684; and The Molecular Biology of the Yeast Saccharomyces, 1982, Eds. Strathcm et al., Cold Spring Harbor Press, Vols. I and n. A constitutive yeast promoter such as ADH or LEU2 or an<br>
inducible promoter such as GAL may be used (Cloning in Yeast, Ch. 3, R. Rothstein In: DNA Cloning<br>
Vol. 11, A Practical Approach, Ed. DM Glover, 1986, IRL Press, Wash., D.C.). Alternatively, vectors<br>
may be used which promote integration of foreign DNA sequences into the yeast chromosome.<br>
[00121]	In some embodiments, a subject genetically modified host cell comprises a nucleic acid<br>
comprising nucleotide sequences encoding enzymes (e.g., a mevalonate pathway enzyme; a terpene<br>
synthase; a prenyltransferase), as described above, where each of the nucleic acids is contained on<br>
separate expression vectors. In other embodiments, two or more of the nucleic acids are contained in a<br>
single expression vector. Where two or more nucleic acids are contained in a single expression vector,<br>
in some embodiments, the nucleotide sequences will be operably linked to a common control element<br>
(e.g., a promoter). Where two or more nucleic acids are contained in a single expression vector, in some<br>
embodiments, the nucleotide sequences will be operably linked to different control elements (e.g., a<br>
promoters), e.g., different control elements are operably linked to enzyme-encoding nucleotide<br>
sequences separately on a single expression vector. For example, as noted above, in some<br>
embodiments, an MK-encoding nucleotide sequence is operably linked to a first promoter; and a<br>
nucleotide sequence encoding acetoacetyl-CoA thiolase, HMGS, and HMGR is operably linked to a<br>
second promoter.<br>
Host cells<br>
[00122]	In some embodiments, the genetically modified host cell is a eukaryotic cell. Suitable<br>
eukaryotic host cells include, but are not limited to, fungal cells, algal cells, insect cells, and plant cells.<br>
Suitable fungal host cells include, but are not limited to, Ascomycota, Basidiomycota, Deuteromycetes,<br>
Fungi imperfecti, Saccharomyces cerevisiae, Saccharomyces sp., Schizosaccharomyces pombe, Pichia<br>
pastoris, Pichia finlandica, Pichia trehalophila, Pichia kodamae, Pichia membranaefaciens, Pichia<br>
opuntiae, Pichia thermotolerans, Pichia salictaria, Pichia quercuum, Pichia pijperi, Pichia stipitis,<br>
Pichia methanolica, Pichia sp., Pichia angusta, Kluyveromyces sp., Kluyveromyces lactis, Candida<br>
albicans, Aspergillus nidulans. Aspergillus niger, Aspergillus oryzae, Trichoderma reesei,<br>
Chrysosporium lucknowense, Fusarium sp., Fusarium graminearum, Fusarium venenatum, and<br>
Neurospora crassa. Suitable algal host cells include, but are not limited to, Chlamydomonas reinhardtii<br>
and Phormidium sp. ATCC29409.<br>
[00123]	In other embodiments, the host cell is a prokaryotic cell. Suitable prokaryotic cells include, but<br>
are not limited to, any of a variety of laboratory strains of Escherichia coli, Lactobacillus sp., Salmonella sp., Shigella sp., and the like. See, e.g., Carrier et al. (1992) J. Immunol. 148:1176-1181; U.S. Pat. No. 6,447,784; and Sizemore et al. (1995) Science 270:299-302. Examples of Salmonella strains which can be employed in the present invention include, but are not limited to, Salmonella typhi and S. typhimurium. Suitable Shigella strains include, but are not limited to, Shigella flexneri, Shigella sonnei, and Shigella disenteriae. Typically, the laboratory strain is one that is non-pathogenic. Non-limiting examples of other suitable bacteria include, but are not limited to, Bacillus subtilis, Pseudomonas pudita, Pseudomonas aeruginosa, Pseudomonas mevalonii, Rhodobacter sphaeroides,<br>
Rhodobacter capsulatus, Rhodospirillum rubrum, Rhodococcus sp., and the Uke. In some embodiments,<br>
the host cell is Escherichia coli.<br>
[00124]	In other embodiments, the host cell employed in the production method is a bacterial cell.<br>
Suitable bacterial hosts include, but are not limited to, any of a variety of gram-positive, gram-negative, or gram-variable bacteria such as microorganisms belonging to the genera Escherichia, Corynebacterium, Brevibacterium, Bacillus, Microbacterium, Serratia, Pseudomonas, Agrobacterium, Alicyclobacillus, Anabaena, Anacystis, Arthrobacter, Azobacter, Chromatium, Erwinia, Methylobacterium, Rhodobacter, Rhodopseudomonas, Rhodospirillum, Scenedesmun, Strepromyces, Synnecoccus, and Zymomonas. Examples of suitable host microorganisms used herein include Escherichia coli, LactoBacillus sp., Lactococcus lactis, Salmonella sp., Salmonella enterica, Salmonella typhi, Salmonella typhimurium. Shigella sp.. Shigella flexneri. Shigella sonnei. Shigella dysenteriae, Enterobacter sakazakii, Pseudomonas sp. D-0110, Pseudomonas pudica, Pseudomonas aeruginosa, Pseudomonas mevalonii, Rhodobacter sphaeroides, Rhodobacter capsulatus, Rhodospirillum rubrum, Rhodospirillum salexigens, Rhodospirillum salinarum, Rhodococcus sp., Mesorhizobium loti, Clostridium acetobutylicum, Clostridium tetani E88, Clostridium lituseburense, Clostridium saccharobutylicum, Clostridium perfringens, Clostridium beijerinckii, Fusobacterium nucleatum, Thermoanaerobacterium thermosaccharolyticum, Butyrivibrio fibrisolvens. Bacillus thuringiensis, Bacillus anthracis. Bacillus megaterium, Bacillus subtilis, Bacillus amyloliquefacines, LactoBacillus johnsonii, Acinetobacter, Roseburia sp., Faecalibacteriumprausnitzii, and Coprococcus sp., Staphylococcus epidermidis, Staphylococcus haemolyticus. Staphylococcus aureus, Brevibacterium ammoniagenes, Brevibacterium immariophilum ATCC14068, Brevibacterium saccharolyticum ATCC14066, Brevibacterium flavum ATCC14067, Brevibacterium lactofermentum ATCC13869, Corynebacterium glutamicum ATCC13032, Corynebacterium glutamicum ATCC14297, Corynebacterium acetoacidophilum ATCC13870, Microbacterium ammoniaphilum ATCC15354, Serratia ficaria, Serratia fonticola, Serratia liquefaciens, Serratia marcescens, Agrobacterium radiobacter, Agrobacterium rhizogenes, Agrobacterium rubi, Anabaena cylindrica, Anabaena doliolum, Anbaenaflos-aquae, Arthrobacter aurescens, Arthrobacter citreus, Arthrobacter globformis, Arthrobacter hydrocarboglutamicus, Arthrobacter mysorens, Arthrobacter nicotianae, Arthrobacter paraffineus, Arthrobacterprotophonniae, Arthrobacter roseoparaffinus, Arthrobacter sulfureus, Arthrobacter ureafaciens, Chromatium buderi, Chromatium tepidum, Chromatium vinosum, Chromatium warmingii, Chromatiumfluviatile, Erwinia uredovora, Erwinia carotovora, Erwnia ananas, Erwinia herbicola, Erwinia punctata, Erwinia terreus, Methylobacterium rhodesianum, Methylobacterium extorquens, Rhodopseudomonas blastica, Rhodopseudomonas marina, Rhodopseudomonas palustris, Streptomyces ambofaciens, Streptomyces aureofaciens, Streptomyces aureus, Streptomyces fungicidicus, Streptomyces griseochromogenes, Streptomyces griseus, Streptomyces lividans, Streptomyces olivogriseus, Streptomyces rameus, Streptomyces tanashiensis, Streptomyces vinaceus, Zymomonas mobilis, and the like (see, for example, Carrier et al. (1992) J.<br>
Immunol. 148:1176-1181; U.S. Patent No. 6,447,784; and Sizemore et al. (1995) Science 270:299-<br>
302).<br>
[00125]	In yet other embodiments, the bacterial host strain is non-pathogenic to humans. Examples of<br>
Escherichia coli strains that can be employed include common cloning strains such as DH1, B,<br>
MG1655, W3110, BL21, DH10B, JM109, DH5alpha, XLl-Blue, XL2-Blue, MC1000, KY3276,<br>
W1485, HB101, No.49, NY49, MP347, NM522, and derivatives thereof. In one embodiment, a RecA-<br>
strain of Escherichia coli is employed in the methods of the invention. In some embodiments, the<br>
Escherichia coli strain also produces IPP via the DXP pathway. In other embodiments, the Escherichia<br>
coli strain comprises a DXP pathway that is functionally disabled.<br>
[00126]	In some embodiments of the present invention, a subject genetically modified host cell is<br>
genetically modified such that an endogenous pathway enzyme is functionally disabled. Enzymes that<br>
can be inactivated in a host microorganism to increase production of IPP and compounds derived<br>
therefrom include, but are not limited to, pyruvate synthase, which when active uses acetyl-CoA (and<br>
carbon dioxide and reduced ferrodoxin) to produce pyruvate and thus reduces the supply of acetyl-CoA<br>
available for the production of IPP and compounds derived therefrom; acetyl-CoA synthetase, which<br>
when active uses coenzyme A (and propionate and ATP) to produce propionyl-CoA and thus reduces<br>
the supply of coenzyme A available for the production of acetyl-CoA; and pyruvate formate lyase,<br>
which when active uses coenzyme A (and 2-oxobutanoate) to produce propionyl-CoA and thus reduces<br>
the supply of coenzyme A available for the production of acetyl-CoA.<br>
[00127]	Methods for disabling genes encoding such enzymes are well known in the art, and include, but<br>
are not limited to, insertion of a mobile genetic element (for example, a transposon); deletion of all or<br>
part of the gene, such that the gene product is not made, or is truncated and is non-functional (that is,<br>
enzymatically inactive); mutation of the gene such that the gene product is not made, or is truncated and<br>
is nonfunctional; deletion or mutation of one or more control elements that control expression of the<br>
gene such that the gene product is not made; and the like.<br>
METHODS OF PRODUCING AN ISOPRENOID COMPOUND<br>
[00128]	The present invention provides methods of producing an isoprenoid compound, the methods<br>
generally involving culturing a subject genetically modified host cell in a suitable medium under<br>
conditions that provide for production of the mevalonate pathway enzymes and the tcrpene synthase<br>
(and optionally also a heterologous prenyltransferase, as described above), such that an isoprenoid<br>
compound is produced by the cell in a recoverable amount. In some embodiments, a subject method<br>
further comprises recovering the isoprenoid compound, e.g., from the cell, from cell culture medium, or<br>
from both the cell and the cell culture medium.<br>
[00129]	Production of an isoprenoid or an isoprenoid precursor is increased in a subject genetically<br>
modified host cell, compared to a control, parent cell that is not so genetically modified. Thus, e.g., production of an isoprenoid or isoprenoid precursor is increased by at least about 10%, at least about 20%, at least about 50%, at least about 2-fold, at least about 2.5-fold, at least about 5-fold, at least about<br>
10-fold, at least about 20-fold, at least about 30-fold, at least about 40-fold, at least about 50-fold, at<br>
least about 75-fold, at least about 100-fold, at least about 200-fold, at least about 300-fold, at least about<br>
400-fold, or at least about 500-fold, or more, in the genetically modified host cell, compared to the<br>
control host cell.<br>
[00130]	For example, production of an isoprenoid compound in a subject genetically modified host cell<br>
is at least about 10%, at least about 20%, at least about 50%, at least about 2-fold, at least about 2.5-<br>
fold, at least about 5-fold, at least about 10-fold, at least about 20-fold, at least about 30-fold, at least<br>
about 40-fold, at least about 50-fold, at least about 75-fold, at least about 100-fold, at least about 200-<br>
fold, at least about 300-fold, at least about 400-fold, or at least about 500-fold, or more, higher than die<br>
level of production of the isoprenoid compound in a host cell genetically modified with pMevT,<br>
pMBIS, and a heterologous nucleic acid encoding a terpene synthase. The nucleotide sequences of<br>
pMevT and pMBIS can be found in U.S. Patent Nos. 7,192,751 and 7,183,089; see, e.g., pMevT, SEQ<br>
IDNO:3 of U.S. Patent No. 7,183,089; MecT operon, SEQ IDNO:8 of U.S. Patent No. 7.192,751;<br>
pMBIS, SEQ ID NO:4 of U.S. Patent No. 7,183,089; and MBIS operon, SEQ ID NO:13 of U.S. Patent<br>
No. 7,192,751).<br>
[00131]	As another example, in some embodiments, the level of an isoprenoid compound produced in a<br>
subject genetically modified host cell is at least about 500 mg/L, at least about 600 mg/L, at least about 700 mg/L, at least about 800 mg/L, at least about 900 mg/L, at least about 1000 mg/L, at least about 1200 mg/L, at least about 1400 mg/L, at least about 1600 mg/L, at least about 1800 mg/L, or at least about 2000 mg/L, after about 10 hours to about 20 hours, about 20 hours to about 30 hours, about 30 hours to about 40 hours, or about 40 hours to about 50 hours, in culture. For example, in some embodiments, the level of an isoprenoid compound produced in a subject genetically modified host cell is from about 500 mg/L to about 600 mg/L, from about 600 mg/L to about 700 mg/L, from about 700 mg/L to about 800 mg/L, from about 800 mg/L to about 1000 mg/L, from about 1000 mg/L to about 1200 mg/L, from about 1200 mg/L to about 1400 mg/L, from about 1400 mg/L to about 1600 mg/L, from about 1600 mg/L to about 1800 mg/L, or from about 1800 mg/L to about 2000 mg/L, or greater than 2000 mg/L, after about 10 hours to about 20 hours, about 20 hours to about 30 hours, about 30 hours to about 40 hours, or about 40 hours to about 50 hours, in culture. In some embodiments, the level of an isoprenoid compound produced in a subject genetically modified host cell is at least about 500 mg/L, at least about 600 mg/L, at least about 700 mg/L, at least about 800 mg/L, at least about 900 mg/L, at least about 1000 mg/L, at least about 1200 mg/L, at least about 1400 mg/L, at least about 1600 mg/L, at least about 1800 mg/L, or at least about 2000 mg/L, where the level is after about 10 hours to about 20 hours, about 20 hours to about 30 hours, about 30 hours to about 40 hours, or about 40 hours to about 50 hours, after induction (e.g., where one or more of the nucleotide sequences encoding an enzyme (e.g., MK, mevalonate enzyme other than MK, prenyltransferasc, terpene synthase) is operably linked to an inducible promoter).<br><br>
[00132]	In some embodiments, the growth rate of a subject genetically modified host cell is greater than<br>
the growth rate of a control cell. For example, in some embodiments, a subject genetically modified host cell grows at a rate that is at least about 10%, at least about 15%, at least about 20%, at least about 25%, at least about 30%, at least about 35%, at least about 40%, at least about 45%, at least about 50%, at least about 60%, at least about 70%, at least about 80%, at least about 90%, at least about 100% or 2-fold, at least about 5-fold, at least about 10-fold, or more, higher than the growth rate of a control cell. Cell growth o is readily determined using well-known methods, e.g., optical density (OD) measurement at about 600 nm (OD600) of liquid cultures of bacteria; colony size; growth rate; and the like.<br>
[00133)	Isoprenoids that can be produced using the method of the invention include, but are not limited<br>
to, monoterpenes, including but not limited to, limonene, citranellol, geraniol, menthol, perillyl alcohol, linalool, thujone; sesquiterpenes, including but not limited to, periplanone B, gingkolide B, amorphadiene, artemisinin, artemisinic acid, valencene, nootkatone, epi-cedrol, epi-aristolochene, famesol, gossypol, sanonin, periplanone, and forskolin; diterpenes, including but not limited to, casbene, eleutherobin, paclitaxel, prostratin, and pseudopterosin; triterpenes, including but not limited to, arbrusideE, bruceantin, testosterone, progesterone, cortisone, digitoxin. Isoprenoids also include, but are not limited to, carotenoids such as lycopene, a- and p-carotene, a- and p-cryptoxanthin, bixin, zeaxanthin, astaxanthin, and lutein. Isoprenoids also include, but are not limited to, triterpenes, steroid compounds, and compounds that are composed of isoprenoids modified by other chemical groups, such as mixed terpene-alkaloids, menaquinones (e.g., vitamin K-2), and coenzyme Q-10.<br>
[00134]	In some embodiments, a subject genetically modified host cell is cultured in a suitable medium<br>
(e.g., Luria-Bertoni broth, optionally supplemented with one or more additional agents, such as an inducer (e.g., where one or more enzyme-encoding nucleotide sequences (e.g., a nucleotide sequence encoding MK, a mevalonate pathway enzyme(s) other than MK, a prenyltransferase, a terpene synthase) is under the control of an inducible promoter), etc.); and the culture medium is overlaid with an organic solvent, e.g. dodecane, formjng an organic layer. The isoprenoid compound produced by the genetically modified host cell partitions into the organic layer, from which it can be purified. In some embodiments, where the isoprenoid-modifying enzyme-encoding nucleotide sequence is operably linked to an inducible promoter, an inducer is added to the culture medium; and, after a suitable time, the isoprenoid compound is isolated from the organic layer overlaid on the culture medium.<br>
[00135]	In some embodiments, the isoprenoid compound produced by a genetically modified host cell<br>
will be separated from other products which may be present in the organic layer. Separation of the isoprenoid compound from other products that may be present in the organic layer is readily achieved using, e.g., standard chromatographic techniques.<br>
[00136]	In some embodiments, an isoprenoid compound synthesized by a subject method is further<br>
chemically modified in a cell-free reaction. For example, in some embodiments, artemisinic acid is isolated from culture medium and/or a cell lysate, and the artemisinic acid is further chemically modified in a cell-free reaction to generate arternisinin.<br>
[00137]	In some embodiments, the isoprenoid compound is pure, e.g., at least about 40% pure, at least<br>
about 50% pure, at least about 60% pure, at least about 70% pure, at least about 80% pure, at least about 90% pure, at least about 95% pure, at least about 98%, or more than 98% pure, where "pure" in the context of an isoprenoid compound refers to an isoprenoid compound that is free from other isoprenoid compounds, macromolecules, contaminants, etc.<br>
EXAMPLES<br>
[00138]	The following examples are put forth so as to provide those of ordinary skill in the art with a<br>
complete disclosure and description of how to make and use the present invention, and are not intended to limit the scope of what the inventors regard as their invention nor are they intended to represent that the experiments below are all or the only experiments performed. Efforts have been made to ensure accuracy with respect to numbers used (e.g. amounts, temperature, etc.) but some experimental errors and deviations should be accounted for. Unless indicated otherwise, parts are parts by weight, molecular weight is weight average molecular weight, temperature is in degrees Celsius, and pressure is at or near atmospheric. Standard abbreviations may be used, e.g., bp, base pair(s); kb, kilobase(s); pi, picoliter(s); s or sec, second(s); min, minute(s); h or hr, hour(s); aa, amino acid(s); kb, kilobase(s); bp, base pair(s); nt, nucleotide(s); i.m., intramuscular(ly); i.p., intraperitoneal(ly); s.c, subcutaneous(ly); and the like.<br>
Example 1: Generating expression plasmids encoding mevalonate pathway enzymes<br>
[00139]	Expression plasmid pMevT was generated by inserting the MevT operon into the pBAD33<br>
vector. The MevT operon encodes the set of MEV pathway enzymes that together transform the ubiquitous precursor acetyl-CoA to (R)-mevalonate, namely acetoacetyl-CoA thiolase, HMG-CoA synthase, and HMG-CoA reductase. The MevT operon was generated by PCR amplifying from Escherichia coli genomic DNA the coding sequence of the atoB gene (GenBank accession number NC_000913 REGION: 2324131..2325315) (encodes an acetoacetyl-CoA thiolase), from Saccharomyces cerevisiae genomic DNA the coding sequence of the ERG 13 gene (GenBank accession number X96617, REGION: 220..1695) (encodes a HMG-CoA synthase), and from Saccharomyces cerevisiae genomic DNA a segment of the coding region of the HMG1 gene (GenBank accession number M22002, REGION: 1660..3165) (encodes a truncated HMG-CoA reductase (tHMGR)). The upstream PCR primer used for the amplification of the HMG1 gene fragment included an artificial start codon. The amplified fragments were spliced together using overlap extensions (SOEing), during which process ribosome binding sites were introduced after the atoB and the ERG 13 coding sequences. After the addition of 3' A overhangs, the MevT operon was ligated into the TA cloning vector pCR4 (Invitrogen, Carlsbad, CA). The MevT operon was subsequently ligated into the Xmal PstI restriction site of vector pBAD33 (Guzman et al. (1995)7. Bacteriol. 177(14): 4121-4130). To place the operon under the control of the PLae promoter, the araC-PBADNsiI-XmaI fragment of pBAD33 was replaced<br>
with the Nsil-Xmal fragment of pBBRlMCS, yielding expression plasmid pMevT (see U.S. Patent<br>
Number 7,192,751).<br>
[00140]	Expression plasmid pAM36-MevT66 was generated by inserting the MevT66 operon into the<br>
pAM36 vector. The pAM36 vector was generated by inserting an oligonucleotide cassette containing<br>
AscI-Sfil-AsiSI-XhoI-PacI-FsIl-Pmel restriction sites into the pACYCl 84 vector (GenBank accession<br>
number XO6403), and by removing the tetramycin resistance conferring gene in pACYCl 84. The<br>
MevT66 operon was synthetically generated using SEQ ID NO: 1 as a template, which comprises the<br>
atoB gene from Escherichia coli (GenBank accession number NC000913 REGION:<br>
2324131 ..2325315), the ERG 13 gene from Saccharomyces cerevisiae (GenBank accession number<br>
X96617, REGION: 220.. 1695), and a truncated version of the HMG1 gene from Saccharomyces<br>
cerevisiae (GenBank accession number M22002, REGION: 1777..3285), all three sequences being<br>
codon-optimizcd for expression in Escherichia coli. The synthetically generated MevT66 operon was<br>
flanked by a 5' EcoRI restriction site and a 3' Hind III restriction site, and could thus be cloned into<br>
compatible restriction sites of a cloning vector such as a standard pUC or pACYC origin vector. From<br>
this construct, the MevT66 operon was PCR amplified with flanking Sfil and AsiSI restriction sites, the<br>
amplified DNA fragment was digested to completion using Sfil and AsiSI restriction enzymes, the<br>
reaction mixture was resolved by gel electrophoresis, the approximately 4.2 kb DNA fragment was gel<br>
extracted using a gel purification kit (Qiagen, Valencia, CA), and the isolated DNA fragment was<br>
ligated into the Sfil AsiSI restriction site of the pAM36 vector, yielding expression plasmid pAM36-<br>
MevT66.<br>
[00141]	Expression plasmid pAM25 was generated by inserting the MevT66 operon into the pAM29<br>
vector. The pAM29 vector was created by assembling the pl5A origin of replication and kanamycin<br>
resistance conferring gene from pZS24-MCSl (Lutz and Bujard (1997) Nucl Acids Res. 25:1203-1210)<br>
with an oligonucleotide-generated lacUVS promoter. The nucleotide sequence of pAM29 is given as<br>
SEQ ED NO: 2. The DNA synthesis construct comprising the MevT66 operon (see description for<br>
pAM36-MevT66 above) was digested to completion using EcoRI and Hind III restriction enzymes, the<br>
reaction mixture was resolved by gel electrophoresis, the approximately 4.2 kb DNA fragment was gel<br>
extracted, and the isolated DNA fragment was ligated into the EcoRI Hindlll restriction site of pAM29,<br>
yielding expression plasmid pAM25.<br>
(00142]	Expression plasmid pMevB-Cm was generated by inserting the MevB operon into the<br>
pBBRlMCS-1 vector. The MevB operon encodes the set of enzymes that together convert (R)-mevalonate to EPP, namely mevalonate kinase, phosphomevalonate kinase, and mevalonate pyrophosphate decarboxylase. The MevB operon was generated by PCR amplifying from Saccharomyces cerevisiae genomic DNA the coding sequences of the ERG 12 gene (GenBank accession number X55875, REGION: 580..1911) (encodes a mevalonate kinase), the ERGS gene (GenBank accession number Z49939, REGION: 3363..4718) (encodes a phosphomevalonate kinase), and the MVD1 gene (GenBank accession number X97557, REGION: 544..1734) (encodes a mevalonate<br>
pyrophosphate decarboxylase), and by splicing the PCR fragments together using overlap extensions (SOEing). By choosing appropriate primer sequences, the stop codons of ERG]2 and ERG8 were changed from TAA to TAG during amplification to introduce ribosome binding sites. After the addition of 3' A overhangs, the MevB operon was ligated into the TA cloning vector pCR4 (Invitrogen, Carlsbad, CA). The MevB operon was excised by digesting the cloning construct to completion using PstI restriction enzyme, resolving the reaction mixture by gel electrophoresis, gel extracting the approximately 4.2 kb DNA fragment, and ligating the isolated DNA fragment into the PstI restriction site of vector pBBRlMCS-1 (Kovach etal., Gene 166(1): 175-176 (1995)), yielding expression plasmid pMevB-Cm.<br>
[00143]	Expression plasmid pMBI was generated by inserting the MBI operon into the pBBRlMCS-3<br>
vector. In addition to the enzymes of the MevB operon, the MBI operon also encodes an isopentenyl pyrophosphatase isomerase, which catalyzes the conversion of IPP to DMAPP. The MBI operon was generated by PCR amplifying from. Escherichia coli genomic DNA the coding sequence of the idi gene (GenBank accession number AF119715) using primers that contained an Xtnalrestriction site at their 5' ends, digesting the amplified DNA fragment to completion using Xmal restriction enzyme, resolving the reaction mixture by gel electrophoresis, gel extracting the approximately 0.5 kb fragment, and ligating the isolated DNA fragment into the Xmal restriction site of expression plasmid pMevB-Cm, thereby placing idi at the 3' end of the MevB operon. The MBI operon was subcloned into the Sail SacI restriction site of vector pBBRlMCS-3 (Kovach et al., Gene 166(1): 175-176 (1995)), yielding expression plasmid pMBI (see U.S. Patent Number 7,192,751).<br>
[00144]	Expression plasmid pMBIS was generated by inserting the ispA gene into pMBI. The ispA gene<br>
encodes a famesyl pyrophosphate synthase, which catalyzes the condensation of two molecules of EPP with one molecule of DMAPP to make famesyl pyrophosphate (FPP). The coding sequence of the ispA gene (GenBank accession number D00694, REGION: 484..1383) was PCR amplified from Escherichia coli genomic DNA using a forward primer with a SacJI restriction site and a reverse primer with a Sad restriction site. The amplified PCR product was digested to completion vising SacII and Sad restriction enzymes, the reaction mixture was resolved by gel electrophoresis, and the approximately 0.9 kb DNA fragment was gel extracted, and the isolated DNA fragment was ligated into the SadI Sad restriction site of pMBI, thereby placing the ispA gene 3' of idi and the MevB operon, and yielding expression plasmid pMBIS (see U.S. Patent Number 7,192,751).<br>
[00145]	Expression plasmid pAM47 was generated by inserting the MBIS operon into the pAM37<br>
vector. Vector pAM37 was generated by assembling the pl5A origin of replication and the chloramphenicol resistance conferring gene from vector pZA31 luc (Lutz and Bujard (1997) Nucl. Acids Res. 25:1203-1210) with a strong constitutive promoter synthetically generated by linking oligonucleotides. The MBIS operon was PCR-amplified from pMBIS using primers 9-38A (SEQ ID NO: 7) and 9-38B (SEQ ID NO: 8), the approximately 5.5 kb PCR product was purified and digested to<br>
completion using Apal and Mlul restriction enzymes, and the DNA fragment was ligated into the Apal<br>
Mlul restriction enzyme site of pAM37, yielding expression plasmid pAM47.<br>
[00146]	Expression plasmids pAM39 and pAM40 were generated by combining expression plasmids<br>
pAM47 and pAM25. Expression plasmids pAM47 and pAM25 were digested using Sad restriction<br>
enzyme, and ligated to each other in two orientations, yielding expression plasmids pAM39 and<br>
pAM40. The nucleotide sequences of expression plasmids pAM39 and pAM40 are given as SEQ ID<br>
NO: 3 and SEQ ID NO: 4, respectively, and plasmid maps are shown in Figures 3 and 4, respectively.<br>
[00147]	Expression plasmid pAM45 was generated by inserting the MBIS operon into pAM36-MevT66<br>
and adding lacUVS promoters in front of the MBIS and MevT66 operons. The MBIS operon was PCR amplified from pMBIS using primers comprising a 5' Xhol restriction site and a 3' Pad restriction site, the amplified PCR product was digested to completion using Xhol and Pad restriction enzymes, the reaction mixture was resolved by gel electrophoresis, the approximately 5.4 kb DNA fragment was gel extracted, and the isolated DNA fragment was ligated into the Xhol Pad restriction site of pAM36-MevT66, yielding expression plasmid pAM43. A DNA fragment comprising a nucleotide sequence encoding the lacUV5 promoter was synthesized from oligonucleotides, and Ligated into the Asd Sfil and AsiSIXhoIrestriction sites of pAM43, yielding expression plasmid pAM45. The nucleotide sequence of pAM45 is given as SEQ ID NO: 5 and a plasmid map in Figure 5.<br>
[00148]	Expression plasmid pAM29-MK was generated by inserting a nucleotide sequence encoding a<br>
mevalonate kinase ("MK") into the pAM29 vector. A nucleotide sequence encoding a mevalonate kinase was PCR-amplified from pMBIS using primers MK-SR (SEQ ID NO: 9) and MK-HR (SEQ ID NO: 10). The PCR product was purified, digested to completion using Sail and HindHI restriction enzymes, and ligated into the Sail Hindlll restriction enzyme site of pAM29, yielding expression plasmid pAM29-MK.<br>
[00149]	Expression plasmid pAM29-PMK was generated by inserting a nucleotide sequence encoding a<br>
phosphomevalonate kinase ("PMK") into the pAM29 vector. A nucleotide sequence encoding a phosphomevalonate kinase was PCR-amplified from pMBIS using primers PMK-SR (SEQ ID NO: 11) and PMK-HR (SEQ ID NO: 12). The PCR product was purified, digested to completion using Sail and Hindlll restriction enzymes, and ligated into the SallHindiii restriction enzyme site of pAM29, yielding expression plasmid pAM29-PMK.<br>
[00150]	Expression plasmid pAM29-MPD was generated by inserting a nucleotide sequence encoding a<br>
mevalonate pyrophosphate decarboxylase ("MPD") into the pAM29 vector. A nucleotide sequence encoding a mevalonate pyrophosphate decarboxylase was PCR-amplified from pMBIS using primers MPD-ER (SEQ ID NO: 13) and MPD-SR (SEQ ID NO: 14). The PCR product was purified, digested to completion using SaIl and HindHI restriction enzymes, and ligated into the EcoRI Sail restrection enzyme site of pAM29, yielding expression plasmid pAM29-MPD.<br>
[00151]	Expression plasmid pAM29-idi was generated by inserting a nucleotide sequence encoding an<br>
isopentenyl pyrophosphate isomerase ("idi") into the pAM29 vector. A nucleotide sequence encoding<br>
an isopentenyl pyrophosphate isomerase was PCR-amplified from pMBIS using primers idi-EF (SEQ ID NO: 15) and idi-SR (SEQ ID NO: 16). The PCR product was purified, digested to completion using Sail and HindlU restriction enzymes, and ligated into the EcoRI Sail restriction enzyme site of pAM29, yielding expression plasmid pAM29-idi.<br>
[00152]	Expression plasmid pAM29-ispA was generated by inserting a nucleotide sequence encoding a<br>
farnesyl pyrophosphate synthase ("ispA") into the pAM29 vector. A nucleotide sequence encoding a famesyl pyrophosphate synthase was PCR-amplified from pMBIS using primers ispA-EF (SEQ ID NO: 17) and ispA-SR (SEQ ED NO: 18). The PCR product was purified, digested to completion using Sail and Hindlll restriction enzymes, and ligated into the EcoRI Sail restriction enzyme site of pAM29, yielding expression plasmid pA%M29-ispA. Example 2: Generation of expression plasmids that encode enzymes that convert FPP.<br>
[00153]	Expression plasmid pTrc99A-ADS was generated by inserting a nucleotide sequence encoding<br>
an amorpha-4,11-diene synthase ("ADS") into vector pTrc99A. The amorpha-4,11-diene synthase sequence was generated synthetically, so that upon translation the amino acid sequence would be identical to that described by Merke et al. (2000) Ach. Biochem. Biophys. 381:173-180, so that the nucleotide sequence encoding the amorpha-4,11-diene syntiiase was optimized for expression in Escherichia coli, and so that the nucleotide sequence was flanked by a 5' Ncol and a 3' Xmal restriction enzyme site (see U.S. Patent Number 7,192,751). The nucleotide sequence was digested to completion using Ncol and Xmal restriction enzymes, the reaction mixture was resolved by gel electrophoresis, the approximately 1.6 kb DNA fragment was gel-extracted, and the isolated DNA fragment was inserted into the Ncol Xmal restriction enzyme site of the pTrc99 A vector (Amman et al. (1985) Gene40:183-190), yielding expression plasmid pTrc99A-ADS (see Figure 6 for a plasmid map).<br>
[00154]	Expression plasmid pAM94 was generated by inserting a nucleotide sequence encoding the<br>
Saccharomyces cerevisiae melavonate kinase (MK), codon-optimized for expression in Escherichia<br>
coli, into vector pTrc99A. The codon-optimized nucleotide sequence was PCR amplified from<br>
pMBISopt using primers 9-153C (SEQ ID NO: 19) and 9-153D (SEQ ID NO: 20). The PCR product<br>
was digested to completion using BamHI and HindlU restriction enzymes, and ligated into the BamHI<br>
Hindlll restriction site of expression plasmid pTrc99A-ADS, yielding expression plasmid pAM94. The<br>
nucleotide sequence of pAM94 is given is SEQ ID NO: 6 and a plasmid map in Figure 7.<br>
Example 3: Generation of Escherichia coli host strains<br>
[00155]	As detailed in Table 1, the host strains were created by transforming chemically competent<br>
Escherichia coli DH1 parent cells with one or more expression plasmids of Examples 1 and 2.<br>
Table 1 - Escherichia coli host strains<br><br>
(TABLE REMOVED)<br><br><br>
[001561	Host cell transformants were selected on Luria Bertoni (LB) agar containing antibiotics as<br>
detailed in Table 1. Single colonies of strains B59 and B60 were transferred from LB agar to culture tubes containing 5 mL of LB liquid medium and antibiotics. Single colonies of strains B32, B125, 1 through 6, and B177 were transferred from LB agar to culture tubes containing 5 mL of M9-MOPS (Table 2) containing 8-10 g/L of D-glucose. The cultures were incubated at 30°C on a rotary shaker at 250 rpm for 30 hours, at which point cell growth was arrested by chilling the cultures on ice. The cells were stored at -80°C in cryo-vials in 1 mL stock aliquots made up of 400 uL sterile 50% glycerol and 600 µL liquid culture. Example 4: Production of amorpha-4,11-diene in Escherichia coli host strains<br>
[00157]	Production cultures of host strains B32, B59, B60, and B125 were established by adding a<br>
stock aliquot of each strain to separate 250 mL flasks containing 40 mL of medium (TB-1 % glycerol medium for strains B32, B59, and B60; M9-MOPS medium containing 20 g/L D-glucose for strains B32 and B125) and antibiotics as detailed in Table 1. The cultures were incubated at 30°C on a rotary shaker at 250 rpm until they reached an OD600 of approximately 0.2 to 0.3, at which point the production of amorpha-4,11 -diene in the host cells was induced by adding 40 uL of 1 M IPTG to the<br>
culture medium. At the time of induction, the cultures were overlain with 8 mL of dodecane to capture<br>
the amorpha-4,11-diene. Samples were taken at various time points by adding 10 uL of the dodecane<br>
layer to 990 uL ethyl acetate in a clean glass GC vial and vortexing for 30 seconds.<br>
[00158]	The ethyl acetate culture extracts were analyzed on a Hewlett-Packard 6890 gas<br>
chromatograph/mass spectrometer (GC/MS) as described in Martin et al. ((2001) Biotechnol. Bioeng. 75:497-503), by scanning for the molecular ion (204 m/z) and the 189 m/z ion. To expedite run times, the temperature program and column matrix was modified to achieve optimal peak resolution and the shortest overall runtime. Compounds in a 1 uL sample were separated using a DB-XLB column (available from Agilent Technologies, Inc., Palo Alto, CA) and helium carrier gas. The oven cycle for each sample was either 80°C hold for 2 minutes, increasing temperature at 30°C/minute to a temperature of 160°C, increasing temperature at 3°C/minute to a temperature of 170°C, increasing temperature at 50°C/minute to 300°C, and a hold at 300°C for 2 minutes (GC protocol 1), or 100°C hold for 0.75 minutes, increasing temperature at 60°C/minute to a temperature of 300°C, and a hold at 300°C for 0.5 minutes (GC protocol 2). The resolved samples were analyzed by a Hewlett-Packard model 5973 mass selective detector. Previous mass spectra demonstrated that the amorpha-4,11-diene synthase product was amorpha-4,11-diene, and that amorpha-4,11-diene had a retention time of 7.9 minutes using GC protocol 1, and 3.7 minutes using GC protocol 2. Beta- or trans-caryophyllene (Sigma-Aldrich, St. Louis, MO) was used as an internal standard for quantitation. Amorpha-4,11-diene titer was calculated based upon a quantitative calibration curve of purified amorpha-4,11 -diene in caryophyllene-spiked ethyl acetate.<br>
[00159]	Figures 8A and 8B show that host strains in which the MBIS operon is expressed from a higher<br>
copy number plasmid and a stronger promoter produce higher levels of amorpha-4,11 -diene. Example 5: Production of amorpha-4,11-diene in Escherichia coli host strains that harbor a plurality of copies of nucleotide sequences encoding mevalonate kinase<br>
[00160]	Production cultures of host strains 1 through 6 were established by adding a stock aliquot of<br>
each strain to separate 250 mL flasks containing 50 mL of M9-MOPS medium containing 10 g/L D-glucose and antibiotics as detailed in Table 1 at a starting OD6oo of approximately 0.5. The cultures were incubated at 30°C on a rotary shaker at 250 rpm until they reached an OD6oo of approximately 0.2 to 0.3, at which point the production of amorpha-4,11-diene in the host cells was induced by adding 40 uL of 1 M IPTG to the culture medium. At the time of induction, the cultures were also supplemented with 20 mM of D,L-mevalonate and were overlain with 8 mL of dodecane to capture the amorpha-4,11-diene. Samples were taken at various time points by transferring 10 uL of the dodecane overlay to 990 uL of ethyl acetate, and analyzed using GC protocol 1 as described in Example 4.<br>
[00161]	Figure 9 shows that host strains in which the gene dosage of mevalonate kinase is increased<br>
produce higher levels of amorpha-4,11-diene.<br>
Example 6: production of amorpha-4,1 l-diene in Escherichia coli host strains that harbor a plurality of<br>
copies of nucleotide sequences encoding mevalonate kinase<br>
[00162)	Host strains B32 and Bl77 were cultured and amorpha-4,11 -diene production analyzed as<br>
described in Example 4.<br>
[00163]	Figure 10 shows that host strains in which the gene dosage and expression of mevalonate<br>
kinase is increased produce higher levels of amorpha-4,1 l-diene.<br>
[00164]	While the present invention has been described with reference to the specific embodiments<br>
thereof, it should be understood by those skilled in the art that various changes may be made and equivalents may be substituted without departing from the true spirit and scope of the invention. In addition, many modifications may be made to adapt a particular situation, material, composition of matter, process, process step or steps, to the objective, spirit and scope of the present invention. All such modifications are intended to be within the scope of the claims appended hereto.<br><br><br><br>
We claim:<br>
1.	A genetically modified host cell capable of producing an isoprenoid, said host<br>
cell comprising:<br>
a nucleic acid comprising a nucleotide sequence encoding a mevalonate kinase;<br>
a nucleic acid comprising a nucleotide sequence encoding one or more mevalonate pathway enzymes other than mevalonate kinase, wherein the encoded mevalonate kinase is produced in the cell at a level that is higher than the level of at least one mevalonate pathway enzyme other than mevalonate kinase; and<br>
a heterologous nucleic acid comprising a nucleotide sequence encoding a terpene synthase.<br>
2.	A genetically modified host cell according to claim 1, said host cell comprising:<br>
at least two coding regions comprising nucleotide sequences encoding a mevalonate<br>
kinase;<br>
a nucleic acid comprising a nucleotide sequence encoding one or more mevalonate pathway enzymes other than mevalonate kinase; and<br>
a heterologous nucleic acid comprising a nucleotide sequence encoding a terpene synthase.<br>
The host cell of Claim 2, wherein the one or more mevalonate pathway enzymes are a plurality of mevalonate pathway enzymes.<br>
The host cell of Claim 2, wherein the one or more MEV pathway enzymes are selected from an acetoacetyl-CoA thiolase, a hydroxymethylglutaryl-CoA synthase (HMGS), a hydroxymethylglutaryl-CoA reductase (HMGR), a phosphomevalonate kinase, and a mevalonate pyrophosphate decarboxylase.<br>
The host cell of Claim 2, wherein the terpene synthase is selected from an amorpha-4,11-diene synthase; a beta-caryophyllene synthase; a germacrene A synthase; a 8-epicedrol synthase; a valencene synthase; a (+)-delta-cadinene synthase; a germacrene C synthase; a (E)-beta- farnesene synthase; a casbene synthase; a vetispiradiene synthase; a 5-epi-aristolochene synthase; an aristolchene synthase alpha-humulene synthase; an (E,E)-aIpha-<br>
farnesene synthase; a (-)-beta-pinene synthase; a gamma-terpinene synthase; a limonene cyclase; a linalool synthase; a 1,8-cineole synthase; a (+)-sabinene synthase; an E-alpha-bisabolene synthase; a (+)-bornyl diphosphate synthase; a levopimaradiene synthase; an abietadiene synthase; an isopimaradiene synthase; a (E)-gamma-bisabolene synthase; a taxadiene synthase; a copalyl pyrophosphate synthase; a kaurene synthase; a longifolene synthase; a gamma-humulene synthase; a Delta-selinene synthase; a beta-phellandrene synthase; a limonene synthase; a myrcene synthase; a terpinolene synthase; a (-)-camphene synthase; a (+)-3-carene synthase; a syn-copalyl diphosphate synthase; an alpha-terpineol synthase; a syn-pimara-7,15-diene synthase; an ent-sandaaracopimaradiene synthase; a stemer-13-ene synthase; a E-beta-ocimene; a S-linalool synthase; a geraniol synthase; a gamma-terpinene synthase; a linalool synthasel; a E-beta-ocimene synthase; an epi-cedrol synthase; an alpha-zingiberene synthase; a guaiadiene synthase; a cascarilladiene synthase; a cis-muuroladiene synthase; an aphidicolan-16b-ol synthase; an elizabethatriene synthase; a sandalol synthase; a patchoulol synthase; a zinzanol synthase; a cedrol synthase; a scared synthase, copalol synthase; and a manool synthase.<br>
The host cell of Claim 2, further comprising a nucleotide sequence encoding an isopentenyl pyrophosphate (IPP) isomerase.<br>
The host cell of Claim 2, further comprising a heterologous nucleic acid comprising a nucleotide sequence encoding a prenyltransferase selected from a farnesyl diphosphate synthase, a geranyl diphosphate synthase, and a geranylgeranyl diphosphate synthase.<br>
The host cell of Claim 2, wherein the isoprenoid is a diterpene, a triterpene, or a sesquiterpene.<br>
The host cell of Claim 2, wherein one or more of the nucleotide sequences encoding the mevalonate kinase or mevalonate pathway enzymes is codon-optimized for expression in said host cell.<br>
The host cell of Claim 2, wherein the host cell is a prokaryotic cell.<br>
The host cell of Claim 2, wherein the host cell is a eukaryotic cell.<br><br>
12.	A genetically modified host cell according to claim 1, said host cell comprising:<br>
a coding region encoding a mevalonate kinase (MK), wherein the MK coding region is<br>
under the control of a first promoter;<br>
a nucleic acid comprising a nucleotide sequence encoding one or more mevalonate pathway enzyme other than mevalonate kinase under the control of a second promoter wherein the second promoter is weaker than the first promoter; and<br>
a heterologous nucleotide sequence encoding a terpene synthase.<br>
The host cell of Claim 12 wherein the first promoter is a constitutively active promoter.<br>
The host cell of Claim 12 wherein the first promoter is an inducible promoter.<br>
The host cell of Claim 12 wherein the first promoter is stronger than a native Escherichia coli Lac promoter or a PLac promoter.<br>
The host cell of Claim 12 wherein the first promoter is a lacUV5 promoter.<br>
The host cell of Claim 12 wherein the first promoter is a trc promoter.<br>
A genetically modified Escherichia coli host cell according to claim 1, said host cell comprising:<br>
a first expression plasmid comprising a nucleotide sequence encoding a mevalonate kinase;<br>
a second expression plasmid comprising a nucleotide sequence encoding one or more MEV pathway enzymes other than mevalonate kinase wherein the copy number of the second expression plasmid is lower than that of the first expression plasmid; and<br>
a heterologous nucleotide sequence encoding a terpene synthase.<br>
The host cell of Claim 18 wherein the first expression plasmid comprises a pl5A origin of replication.<br>
The host cell of Claim 18 wherein the terpene synthase is selected from an amorpha-4,11-diene synthase; a beta-caryophyllene synthase; a germacrene A synthase; a 8-<br>
epicedrol synthase; a valencene synthase; a (+)-delta-cadinene synthase; a germacrene C synthase; a (E)-beta- farnesene synthase; a casbene synthase; a vetispiradiene synthase; a 5-epi-aristolochene synthase; an aristolchene synthase alpha-humulene synthase; an (E,E)-alpha-farnesene synthase; a (-)-beta-pinene synthase; a gamma-terpinene synthase; a limonene cyclase; a linalool synthase; a 1,8-cineole synthase; a (+)-sabinene synthase; an E-alpha-bisaboIene synthase; a (+)-bornyl diphosphate synthase; a levopimaradiene synthase; an abietadiene synthase; an isopimaradiene synthase; a (E)-gamma-bisabolene synthase; a taxadiene synthase; a copalyl pyrophosphate synthase; a kaurene synthase; a longifolene synthase; a gamma-humulene synthase; a Delta-selinene synthase; a beta-phellandrene synthase; a limonene synthase; a myrcene synthase; a terpinolene synthase; a (-)-camphene synthase; a (+)-3-carene synthase; a syn-copalyl diphosphate synthase; an alpha-terpineol synthase; a syn-pimara-7,15-diene synthase; an ent-sandaaracopimaradiene synthase; a stemer-13-ene synthase; a E-beta-ocimene; a S-linalool synthase; a geraniol synthase; a gamma-terpinene synthase; a linalool synthasel; a E-beta-ocimene synthase; an epi-cedrol synthase; an alpha-zingiberene synthase; a guaiadiene synthase; a cascarilladiene synthase; a cis-muuroladiene synthase; an aphidicolan-16b-ol synthase; an elizabethatriene synthase; a sandalol synthase; a patchoulol synthase; a zinzanol synthase; a cedrol synthase; a scareol synthase, copalol synthase; and a manool synthase.<br>
21.        The host cell of Claim 18 wherein the isoprenoid is a diterpene, a triterpene, or a sesquiterpene.</td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/viewdoc.aspx?id=KbgaWIMTJOAf5D2mQkW2Yw==&amp;amp;loc=+mN2fYxnTC4l0fUd8W4CAA==" target="_blank" style="word-wrap:break-word;">http://ipindiaonline.gov.in/patentsearch/GrantedSearch/viewdoc.aspx?id=KbgaWIMTJOAf5D2mQkW2Yw==&amp;amp;loc=+mN2fYxnTC4l0fUd8W4CAA==</a></p>
		<br>
		<div class="pull-left">
			<a href="271188-compositions-for-the-treatment-of-neoplastic-diseases.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="271190-network-terminal-with-a-low-duty-cycle.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>271189</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>2242/DELNP/2009</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>07/2016</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>12-Feb-2016</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>08-Feb-2016</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>06-Apr-2009</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>AMYRIS BIOTECHNOLOGIES, INC.</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td>5980 HORTON STREET, SUITE 450, EMERYVILLE, CALIFORNIA 94608, U.S.A.</td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>KEASLING, JAY D.</td>
											<td>1160 STERLING AVENUE, BERKELEY,CALIFORNIA 94708, U.S.A.</td>
										</tr>
										<tr>
											<td>2</td>
											<td>NOWROOZI, FARNAZ</td>
											<td>7727 CLAREMONT AVENUE, BERKELEY, CALIFORNIA 94705, U.S.A.</td>
										</tr>
										<tr>
											<td>3</td>
											<td>PITERA, DOUGLAS J.</td>
											<td>4099 HOWE STREET, UNIT #302, OAKLAND, CALIFORNIA 94611, U.S.A.</td>
										</tr>
										<tr>
											<td>4</td>
											<td>NEWMAN, JACK D.</td>
											<td>1715 SACRAMENTO STREET, BERKELEY, CALIFORNIA 94702, U.S.A.</td>
										</tr>
										<tr>
											<td>5</td>
											<td>ANTHONY, JENNIFER</td>
											<td>501 CORY LANE, ASTON, PENNSYLVANIA 19014, U.S.A.</td>
										</tr>
										<tr>
											<td>6</td>
											<td>ANTHONY, LARRY</td>
											<td>501 CORY LANE, ASTON, PENNSYLVANIA 19014, U.S.A.</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>C12N 1/00</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>PCT/US2007/020790</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td>2007-09-25</td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td>60/826,970</td>
									<td>2006-09-26</td>
								    <td>U.S.A.</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/271189-production-of-isoprenoids-and-isoprenoid-precursors by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 02:18:36 GMT -->
</html>
