<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/271440-a-method-for-obtaining-a-lettuce-plant-resistant-to-bremia-lactucae by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 01:30:40 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 271440:A METHOD FOR OBTAINING A LETTUCE PLANT RESISTANT TO BREMIA LACTUCAE</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">A METHOD FOR OBTAINING A LETTUCE PLANT RESISTANT TO BREMIA LACTUCAE</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>The present invention relates to a plant, which is resistant to a pathogen of viral, bacterial, fungal or oomycete origin, wherein the plant has a reduced level, reduced activity or complete absence of DMR6 protein as compared to a plant that is not resistant to the said pathogen, in particular organisms of the Fungi or the phylum Oomycota. The invention further relates to a method for obtaining a plant, which is resistant to a pathogen of viral, bacterial, fungal or oomycete origin, comprising reducing the endogenous level or activity of DMR6 protein in the plant. In addition, the invention relates to the use of a DMR6 promotor for providing disease resistant plants.</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td> <br><br>
DISEASE RESISTANT PLANTS<br>
The present invention relates to disease resistant plants, in particular plants resistant to organisms of the kingdom Fungi and the phylum Oomycota,   the oomycetes. The invention further relates to.plant genes conferring disease resistance and methods of obtaining such disease resistant plants for providing protection to Oomycota pathogens.<br>
Resistance of plants to fungal and oomycete pathogens has been extensively studied, for both pathogen specific and broad resistance. In many cases resistance is specified by dominant genes for resistance. Many of these race-specific or gene-for-gene resistance genes have been identified that mediate pathogen recognition by directly or indirectly interacting with avirulence gene products or other molecules from the pathogen. This recognition leads to the activation of a wide range of plant defence responses that arrest pathogen growth.<br>
In plant breeding there is a constant struggle to identify new sources of mostly monogenic dominant resistance genes. In cultivars with newly introduced single resistance genes, protection from disease is often rapidly broken, because pathogens evolve and adapt at a high frequency and regain the ability to successfully infect the host plant. Therefore, the availability of new sources of disease resistance is highly needed.<br>
Alternative resistance .mechanisms act for example through the modulation of the defence response in plants, such as the resistance mediated by the recessive mlo  gene in ) barley to the powdery mildew pathogen Blumeria  graminis  f.sp. hordei.   Plants carrying mutated alleles of the wildtype MLO gene exhibit almost complete resistance coinciding with the abortion of attempted fungal penetration of the cell wall of<br>
CONFIRMATION COPY<br><br>
single attacked epidermal cells. The wild type MLO  gene thus acts as a negative regulator of the pathogen response. This is described in WO9804586.<br>
Other examples are the recessive powdery mildew resistance genes, found in a screen for loss of susceptibility to Erysiphe  cichoracearum.   Three genes have been cloned so far, named PMR6,   which encodes a pectate lyase-like protein, PMR4  which encodes a callose synthase, and PMR5  which encodes a protein of unknown function. Both mlo  and pmr  genes appear to specifically confer resistance to powdery mildew and not to oomycetes such as downy mildews.<br>
Broad pathogen resistance, or systemic forms of resistance such as SAR, has been obtained by two main ways. The first is by mutation of negative regulators of plant defense and cell death, such as in the cpr,   Isd  and acd mutants of Arabldopsls.   The second is by transgenic overexpression of inducers or regulators of plant defence, such as in NPRl  overexpressing plants.<br>
The disadvantage of these known resistance mechanisms is that, besides pathogen resistance, these plants often show detectable additional and undesirable phenotypes, such as stunted growth or the spontaneous formation of cell death.<br>
It is an object of the present invention to provide a form of resistance that is broad, durable and not associated with undesirable phenotypes.<br>
In the research that led to the present invention, an Arabidopsis  thaliana  mutant screen was performed for reduced susceptibility to the downy mildew pathogen Hyaloperonospora parasitica.   EMS-rautants were generated in the highly susceptible Arabidopsis  line Ler edsl-2.   Eight downy mildew resistant {dmr)   mutants were analysed in detail, corresponding to 6 different loci. Microscopic analysis showed that in all mutants H.   parasitica  growth was severely<br><br>
reduced. Resistance of dmr3,   dmr4  and dmr5  was associated with constitutive activation of plant defence. Furthermore, the dmr3  and dmx4,   but not dmrS  mutants, were also resistant to Pseudomonas  syringae  and Colovinomyces  orontii.<br>
In contrast, enhanced activation of plant defence was not observed in the dmrl,   dmr2,   and dmrS  mutants. The results of this research have been described in Van Damme et al. (2005) Molecular Plant-Microbe Interactions 18(6) 583-592. This article does not disclose the identification and characterization of the DMR genes.<br>
The dmr6  mutant was identified in a loss-of-susceptibility screen in the Arabidopsis Ler edsl-2 background. The DMR6  gene now has been cloned and characterized. Thus, it was found that DMR6  is the gene At5g24530, encoding for an oxidoreductase (DNA and amino acid sequence are depicted in Figure 2). Oxidoreductases are enzymes that catalyze the transfer of electrons from one molecule, the oxidant, to another, the reductant. According to the present invention, it has been found that lacl
The present invention thus provides a plant, which is resistant to a pathogen of viral, bacterial, fungal or oomycete origin, characterized in that the plant has a reduced level, reduced activity or complete absence of the DMR6 protein as compared to a plant that is not resistant to the said pathogen.<br>
This formi ^f res** ^tan^e is in '^articular effective against pathogens of the phylum Oomycota, such as Albugo, Aphanomyces,   Basidiophora,   Bremia,   Hyaloperonospora, Pachymetra,   Paraperonospora,   Perofascia,   Peronophythora, Peronospora,   Peronosclerospora,   Phytium,   Phytophthora, Plasmopara,   Protobremia,   Pseudoperonospora,   Sclerospora,<br><br>
Viennotia   species, as well as to pathogens belonging to the Fungi.<br>
The resistance according to the invention is based on an altered, in particular a reduced level, reduced activity or complete absence of the DMR6 protein in planta.   The term "DMR6 protein" in this respect relates to the DMR6  gene product, such as the protein encoded by the At5g24530 gene in Arabidopsis.   Such alterations can be achieved in various ways.<br>
In one embodiment of the invention, the reduced level of DMR6 protein is the result of a reduced endogenous DMR6 gene expression. Reducing the expression of the DMR6 gene can be achieved, either directly, such as by gene silencing, or indirectly by modifying the regulatory sequences thereof, or by stimulating repression of the gene.<br>
Modulating the DMR6 gene to lower its activity or expression can be achieved at various levels. First, the endogenous gene can be directly mutated. This can be achieved by means of a mutagenic treatment. Alternatively, a modified DMR6 gene can be brought into the plant by means of transgenic techniques or by introgression, or the expression of DMR6 can be reduced at the regulatory level, for example by modifying the regulatory sequences or by gene silencing.<br>
In another embodiment of the invention, the reduced level of DMR6 protein is the result of a mutation in the DMR6 gene resulting in a reduced DMR6 expression as compared to<br>
•t-ho    T.^-i 1H — ■(- \TT\a    HMP ^    rTon<a i.thoyohn no qiioh miih r></a>
"••—      •■—— —  -jt'^  —....-  -3—..—  	_..	_		J.	-,<br>
or resulting in a reduced mRNA or protein stability. In a particular embodiment this is achieved by mutations in the DMR6 coding sequence that result in a non-functional DMR6 protein, i.e. without or with reduced enzymatic activity.<br>
In another embodiment of the invention, reduced expression can be achieved by down-regulation of DMR6  gene<br><br>
expression either at the transcriptional or the translational level, e.g. by gene silencing or by mutations that affect the expression of the DMR6  gene.<br>
This invention is based on research performed on resistance to Hyaloperonospora parasitica  in Arabidopsis  but is a general concept that can be more generally applied in plants, in particular in crop plants that are susceptible to infections with pathogens, such as Oomycota and Fungi.<br>
The invention is suitable for a large number of plant diseases caused by oomycetes such as, but not limited to, Bremia  lactucae  on lettuce, Peronospora  farinosa  on spinach, Pseudoperonospora  cubensis  on members of the Cucurbitaceae family, e.g. cucumber and melon, Peronospora   destructor  on onion, Hyaloperonospora parasitica  on members of the Brasicaceae  family, e.g. cabbage, Plasmopara  viticola  on grape, Phytopbthora   infestans  on tomato and potato, and Phytophthora   sojae on soybean.<br>
When the modification of DMR6  gene expression in a plant is to be achieved via genetic modification of the DMR6 gene or via the identification of mutations in the DMR6 gene, and the gene is not yet known it must first be identified. To generate pathogen-resistant plants, in particular crop plants, via genetic modification of the DMR6 gene or via the identification of mutations in the DMR6 gene, the orthologous DMR6 genes must be isolated from these plant species.<br>
Various methods are available for the identification<br>
-*-hr-.-\ ^.nr-.'.is  sequences in other plants,<br>
A method for the identification of DMR6 orthologous sequences in a plant species, may for example comprise identification of DMR6 ESTs of the plant species in a database; designing primers for amplification of the complete DMR6 transcript or cDNA; performing amplification experiments with the primers to obtain the corresponding complete<br><br>
transcript or cDNA;"and determining the nucleotide sequence of the transcript or cDNA. Suitable methods for amplifying the complete transcript or cDNA in situations where only part of the coding sequence is known are the advanced PCR techniques 5'RACE, 3'RACE, TAIL-PCR, RLM-RACE and vectorette PCR.<br>
Alternatively, if no nucleotide sequences are available for the plant species of interest, primers are designed on the DMR6 gene of a plant species closely related to the plant of interest, based on conserved domains as determined by multiple nucleotide sequence alignment, and used to PCR amplify the orthologous sequence. Such primers are suitably degenerate primers.<br>
Another reliable method to assess a given sequence as being a DMR6 ortholog is by identification of the reciprocal best hit. A candidate orthologous DMR6 sequence of a given plant species is identified as the best hit from DNA databases when searching with the Arabidopsis  DMR6 protein or DNA sequence, or that of another plant species, using a Blast programme. The obtained candidate orthologous nucleotide sequence of the given plant species is used to search for homology to all Arabidopsis  proteins present in the DNA databases (e.g. at NCBI or TAIR) using the BlastX search method. If the best hit and score is to the Arabidopsis  DMR6 protein, the given DNA sequence can be described as being an ortholog, or orthologous sequence,<br>
deduced from the complete genome sequence that is publicly available. In the genome of rice 3 orthologs, and in poplar 2 orthologs have been identified. In most other plant species tested so far, DMR6 appears to be encoded by a single gene, as determined by the analysis of mRNA sequences and EST data from public DNA databases. The orthologous genes and proteins<br><br>
are identified in these plants by nucleotide and amino acid comparisons with the information that is present in public databases.<br>
Alternatively, if no DNA sequences are available for the desired plant species, orthologous sequences are isolated by heterologous hybridization using DNA probes of the DMR6 gene of Arabidopsis  or another plant or by PCR methods, making use of conserved domains in the DMR6 coding sequence to define the primers. For many crop species, partial DMR6 mRNA sequences are available that can be used to design primers to subsequently PCR amplify the complete mRNA or genomic sequences for DNA sequence analysis.<br>
In a specific embodiment the ortholog is a gene of which the encoded protein shows at least 50% identity with the Arabidopsis  DMR6 protein {At5g24530) or that of other plant DMR6 proteins. In a more specific embodiment the identity is at least 55%, more specifically 60%, even more specifically 65%, 70%, 75%, 80%, 85%, 90%, 95% or 99%.<br>
Figure 1 shows orthologous DMR6 sequences (described in Table 1) that have been identified in publicly available databases and obtained by PCR amplification on cDNA and subsequent sequencing. After orthologous DMR6 sequences are identified, the complete nucleotide sequence of the regulatory and coding sequence of the gene is identified by standard molecular biological techniques. For this, genomic libraries of the plant species are screened by DNA hybridization or PCR with probes or primers derived from a known DMR6  gene to identify the genomic clones containing the DMR6  gene. Alternatively, advanced PCR methods, such as RNA ligase-mediated RACE (RLM-RACE) , can be used to directly amplify gene and cDNA sequences from genomic DNA or reverse-transcribed mRNA. DNA sequencing subsequently results in the characterization of the complete gene or coding<br><br>
sequence.<br>
Once the DNA sequence of the gene is known this information is used to prepare the means to modulate the expression of the DMR6  gene.<br>
To achieve a reduced DMR6 protein level, the expression of the DMR6 gene can be down-regulated or the enzymatic activity of the DMR6 protein can be reduced by amino acid substitutions resulting from nucleotide changes in the DMR6 coding sequence.<br>
In a particular embodiment of the invention, downregulation of DMR6 gene expression is achieved by gene-silencing using RNAi. For this, transgenic plants are generated expressing a DMR6 anti-sense construct, an optimized micro-RNA construct, an inverted repeat construct, or a combined sense-anti-sense construct, so  as   to generate dsRNA corresponding to DMR6 that leads to gene silencing.<br>
In an alternative embodiment, one or more regulators of the DMR6 gene are downregulated (in case of transcriptional activators) by RNAi.<br>
In another embodiment regulators are upregulated (in case of repressor proteins) by transgenic overexpression. Overexpression is achieved in a particular embodiment by expressing repressor proteins of the DMR6 gene from a strong promoter, e.g. the 35S promoter that is commonly used in plant biotechnology.<br>
The downregulation of the DMR6 gene can also be achieved by mutagenesis of the regulatory elements in the promoter, terminator region, or potential introns. Mutations in the DMR6  coding sequence in many cases leads to amino acid substitutions or premature stop codons that negatively affect the expression or activity of the encoded DMR6 protein.<br>
These mutations are induced in plants by using mutagenic chemicals such as ethyl methane sulfonate (EMS), by<br><br>
irradiation of plant material with gamma rays or fast neutrons, or by other means. The resulting nucleotide changes are random, but in a large collection of mutagenized plants the mutations in the DMR6 gene can be readily identified by using the TILLING (Targeting Induced Local Lesions IN Genomes) method (McCallum et-al. (2000) Targeted screening for induced mutations. Nat. Biotechnol. 18, 455-457, and Henikoff et al. (2004) TILLING. Traditional mutagenesis meets functional genomics. Plant Physiol. 135, 630-636). The principle of this method is based on the PCR amplification of the gene of interest from genomic DNA of a large collection of mutagenized plants in the M2 generation. By DNA sequencing or by looking for point mutations using a single-strand specific nuclease, such as the CEL-I nuclease (Till et al. (2004) Mismatch cleavage by single-strand specific nucleases. Nucleic Acids Res. 32, 2632-2641) the individual plants that have a mutation in the gene of interest are identified.<br>
By screening many plants, a large collection of mutant alleles is obtained, each giving a different effect on gene expression or enzyme acitivity. The gene expression or protein levels can for example be tested by analysis of DMR6 transcript levels (e.g. by RT-PCR) or by guantification of DMR6 protein levels with antibodies.<br>
Plants with the desired reduced DMR6 level or DMR6 expression are then back-crossed or crossed to other breeding lines to transfer only the desired new allele into the background of the crop wanted.<br>
The invention further relates to mutated DMR6 genes.<br>
In a particular embodiment, the invention relates to dmr6 alleles with premature stop codons, such as the dmr6-l allele.<br><br>
In another embodiment, the invention relates to mutated versions of the DMR6 genes of Lactuca  sativa,   Cucumis sativus,   and Spinacia  oleracea  as shown in Figures 3-5.<br>
The present invention demonstrates that plants having no or a reduced level of functional DMR6 gene product show resistance to pathogens, in particular of oomycete and fungal origin. With such knowledge the skilled person can identify so far unknown natural variants of a given plant species that have variants of the DMR6 gene that lead to a reduced level or absence of a of functional DMR5 protein, or mutated versions of the DMR6 protein, and to use these natural variants according to the invention.<br>
The present invention further relates to the use of a DMR6 promoter for providing disease resistance into plants, i.e. for providing plants with a resistance to a pathogen of viral, bacterial, fungal or oomycete origin. According to the present invention, the transcriptional up-regulation of DMR6 in response to pathogen infection has been demonstrated. Both transcript analysis as well as promoter DMR6-reporter lines support this finding (see Example 1, below). The pathogen-inducible DMR6 promotor according to the invention thus is particularly useful to control the expression of inducible systems that lead to disease resistance in plants.<br>
One example of such inducible system that leads to disease resistance in plants, and in which the DMR6 promotor according to the present invention may be effective, has e.g. bean described in WO 99/45125, v.'herein an antisense nucleotide sequence for a gene involved in the regulation of the C-5 porphyrin metabolic pathway is operably linked to a pathogen-inducible promoter and used to transform plant cells. Expression of the antisence nucleotide sequence in response to the pathogen effectively disrupts porphyrin metabolism of the transformed plant cell, and development of<br><br>
a localized lesion wherein the spread of the pathogen is contained. WO 96/35697 also discloses inducible systems leading to disease resistance in- plants, wherein an inducible promotor controls the expression of a protein capable of evoking the hypersensitivity response in a plant. EP 0474857 furthermore discloses a method for the induction of pathogen resistance in plants, comprising transforming plants with polynucleotide sequences encoding a pair of pathogen-derived-avirulence-gene/plant-derived-resistance gene, wherein the expression of one of or both the elicitor peptide and the resistance gene is regulated by a pathogen inducible promotor. Further examples of inducible systems leading to resistance to pathogens in plants have been described in e.g. WO 98/32325.<br>
In a particular preferred embodiment, the present invention relates to a method of providing disease resistance in a plant, comprising transforming a plant cell with a DNA construct comprising at least one expressible nucleic acid which is operably linked to a pathogen-inducible promotor that is operable within a plant cell, and regenerating transformed plants from said plant cells, wherein the pathogen-inducible promotor is a DMR6 promotor, and wherein the expression of the expressible nucleic acid confers disease resistance to the transgenic plant.<br>
The invention also relates to disease resistance plants, obtainable by said method, as well as to plant tissue  and seeds obtained from said plants.<br>
The invention in particular relates to plants, which are resistant to a pathogen of viral, bacterial, fungal or oomycete origin, wherein the plant comprises in its genome a DNA construct, comprising at least one expressible nucleic acid which is operably linked to a pathogen-inducible promotor, wherein the pathogen-inducible promotor is a DMR6<br><br>
promotor.<br>
The present invention also relates to the DNA construct per se, comprising at least one expressible nucleic acid which is operably linked to a pathogen-inducible promotor, wherein the pathogen-inducible promotor is a DMR6 promotor. The construct of the invention can be used to transform plant cells which may be regenerated into transformed plants. Furthermore, transformed plant tissue and seed may be obtained. Suitable methods for introducing the construct of the invention into plant cells are known to the skilled person.<br>
According to the invention, by "operably linked" is meant that a promotor and an expressible nucleic acid, e.g. a gene, are connected in such way as to permit initiation of transcription of the expressible nucleic acid (e.g. gene) by the promotor.<br>
By "expressible nucleic acid" is meant a nucleic acid (e.g. a gene, or part of a gene) that can be expressed in the cell, i.e. that can be transcribed into mRNA, and eventually may be translated into a protein. The expressible nucleic acid may be genomic DNA, cDNA, or chemically synthesized DNA or any combination thereof.<br>
According to the present invention, a DNA construct comprises all, necessary nucleic acid elements which permit expression (i.e. transcription) of a particular nucleic acid in a cell. Typically, the construct includes an expressible nucleic acid, i.e. a nucleic acid to be transcribed, and a promotor. The construct can suitably be incorporated into e.g a plasmid or vector.<br>
The expressible nucleic acid preferably is a gene involved in a plant defence response, e.g. a gene associated with the hypersensitivity response of a plant. In the hypersensitivity response (HR) of a plant, the site in the<br><br>
plant where the pathogen invades undergoes localized cell death by the induced expression of a suicide mechanism that triggers said localized cell death in response to pathogens. In this way, only a few plant cells are sacrificed and the spread of the pathogen is effectively arrested. Examples of said genes involved in a plant defence response are the regulatory protein NPRl/NIMl (Friedrich et al., Mol. Plant Microbe Interact. 14(9): 1114-1124, 2001) and the transcription factor MYB30 (Vailleau et al., Proc. Natl. Acad. Sci. USA 99(15).- 10179-10184, 2002).<br>
In a particular embodiment, the expressible nucleic acid encodes an autologous or heterologous polypeptide capable of conferring disease-resistance to a plant. By "autologous polpeptide" is meant any polypeptide that is expressed in a transformed plant cell from a gene that naturally occurs in the transformed plant cell. By "heterologous polypeptide" is meant any polypeptide that is expressed in a transformed plant cell from a gene that is partly or entirely foreign (i.e. does not naturally occur in) to the transformed plant cell. Examples of such polypeptides are the mammalian Bax protein, which encodes a pro-apoptotic protein and results in cell death in plants (Lacomme and Santa Cruz, Proc. Natl. Acad. Sci. USA 96(14): 7956-61, 1999) and fungal chitinases (de las Mercedes Dana et al., Plant Physiol. 142(2): 722-730, 2006).<br>
Preferably, the DMR6 promotor is the Arabidopsis DMR6 promotor. The DMR6 promotor comprises a region of 3000 bp that is upstream of the Arabidopsis DMR6  coding sequence (ATG start codon) and includes the 5'UTR. Preferably the DMR6 promotor comprises a nucleotide sequence as defined in Figure 11, and/or any functional fragment thereof, i.e. any fragment (or part) of said sequence which still is capable of initiating transcription of the expressible nucleic acid(s)<br><br>
to which it is ope5f^bly linked, and/or natural variants thereof, i.e. natural variants of this promoter which may contain small polymorphisms, but which are generally at least 90% identical.<br>
In a further preferred embodiment, the DMR6 promoter is an orthologous DMR6 promoter, i.e. a promoter of an orthologous DMR6 gene. Methods for identifying DMR6 orthologs have been described in Example 2 below. Once the DMR6 orthologs have been identified, the skilled person will be able to isolate the respective promoter of said orthologs, using standard molecular biological techniques.<br>
According to the present invention, the DMR6 promoter has been shown to be strongly pathogen-induced, and the DMR6 promotor is net highly expressed in other non-infected tissues. Thus, it is a very suitable promotor for use in inducible system.s for providing resistance to pathogens of viral, bacterial, fungal or oomycete origin in plants. Examples of specific pathogens and plants for which the inducible system, using the DMR6 promotor of the present invention, suitably can be used, have been given above.<br>
The present invention is illustrated in the following examples that are net intended to limit the invention in any way. In the examples reference is made to the following figures.<br>
lahlB 1  shows the Genbank accession numbers and Genlnfo identifiers of the Arabidopsis  DMR6 mRNA and orthologous sequences from other plant species.<br>
Table 2 shows the PCR primers for the markers used for the map-based cloning of DMR6.<br>
Table 3 shows primer pairs for cloning dmr6  orthologs in a suitable plant expression vector.<br>
Figure 1 shows the alignment of the amino acid sequences of the DMR6 protein of Arabidopsis   thaliana  and<br><br>
orthologs from Aquilegia   species,   Citrus  sinensis,   Coffea canephora,   Cucumis  sativus,   Gossypium hirsitum,   Lactuca sativa,   Medicago   truncatula,   Oryza   sativa   (3),   Populvs trichocarpa   (2),   Solanum  lycopersicum   (2) ,   Sorghum bicolor, Spinacia   oleracea,   Vitis   vinifera,   Zea mays,   and Zingiber officinale,   using the CLUSTAL W (1.83) multiple sequence alignment programme (EBI). Below the sequences the conserved amino acids are indicated by the dots, and the identical amino acids are indicated by the asteriks.<br>
Figure 2 shows the nucleotide and amino acid sequence of the DMR6 gene (At5g24530, gi 42568064, Genbank NM_122361) and protein (gi 15238567, Genbank NP_197841) of Arabidopsis thaliana,   respectively.<br>
Figure 3 shows the nucleotide and derived amino acid sequence of the DMR6 ortholog of Lactuca  sativa, respectively.<br>
Figure 4 shows the nucleotide and derived amino acid sequence of the DMR6 ortholog of Spinacia   oleracea, respectively.<br>
Figure 5 shows the nucleotide and derived amino acid sequence of the DMR6 ortholog of Cucumis sativus and Cucumis melo.<br>
Figure 6 shows the downy mildew resistance of the Arabidopsis  dmr6  mutants, (a) Quantification of sporangiophores of H.  parasitica  isolate Waco9, 7 days post inoculation, on the dmr6-l  mutant (BCj, line E37) compared to its oarental line her edsl~2  and OTI  the dmr6~2  mutant (FLAG_445D09 T-DNA line) compared to its parental line Ws-4. (b) Restoration of susceptibility by complementation with the At5g24530 gene in the dmr6-l  mutant. H.   parasitica   spores per mg seedling weight were quantified on her edsl-2,   dmr6-l  and 5 complementation lines (#121, 122, 211,231, and 241).<br><br>
Figure 7 shows the structure of the Arabidopsis  DMR6 gene and dmrS-l   and dmr6-2  mutations. The DMR6  gene contains four exons and a coding sequence of 1026 bases. The two alleles are indicated; dmr6-l  with a base change in exon 2, and dinr6-2  with a T-DNA insertion into intron 2.<br>
Figure 8 shows the rilative transcript levels of DMR6 in Ler plants either mock treated or inoculated with a compatible or incompatible H.  parasitica  isolate. Transcript levels were determined at different days post inoculation. The difference in cycle threshold (ACT) values reflect the number of additional PCR amplification cycles required to reach an arbitrary threshold product concentration as compared to ACTIN2.   A  lower ACT value indicates a higher transcript level.<br>
Figure 9 shows the expression of the DMR6 promoter-reporter (pDMR6::GUS) construct in transgenic Arabidopsis lines, visualized with only X-gluc as substrate (Figure d and e) or Magenta-Xgluc (Figure a-c) and trypan blue staining of H.   parasitica   growth (a) Ler edsl-2   (pDMR6::GUS)   3dpi with H. parasitica,   Cala2 isolate, (b) Col-0 (pDMK6::GUS) 3dpi with H.   parasitica,   Waco9 isolate, (c) Ler edsl-2   {pDMR6::GUS) 3dpi with H.  parasitica,   Emoy2 isolate, (d) Col-0 {pDMR6::GUS]   3 dp wounding, (e) Col-0 {pDMR6:iGUS)   3   dp  BTH application.<br>
Figure 10 shows the Q-PCR analysis of the transcript levels of the genes; At4gl4365, Atlgl4880, ACD6, PR-1, PR-2 and PR-5, selected as up regulated in the dmr6-l micro array analysis.(a) Transcription levels of the six genes in dmr6-l compared to Ler edsl-2 and additionally the DMR6 transcript, (b) Elevated gene transcripts of six defence-associated genes in dinr6-2 versus Ws-4. ACT reflects the number of additional PCR amplification cycles required to reach the level of<br><br>
ACTIN2  transcripts. A lower ACT value indicates a higher transcript level.<br>
Figure 11 shows the nucleotide sequence of the 3 kb region upstream of the start codon of the DMR6 gene.<br>
(at5g24530) of Arabidopsis thaliana, including the promoter and 5'-UTR (underlined).<br>
Figure 12 shows the nucleotide and derived amino acid sequence of the DMR6 ortholog of SoJanum Jycopersicum, respectively.<br>
Figure 13 shows the nucleotide and derived amino acid sequence of the DMR6 ortholog of Nicotiana  benthamiana ,   respectively.<br>
Figure 14 shows complementation of Arabidopsis thaliana dmr6-l  with DMR6 derived from Cucumis  sativa   (Cs), Spinacia   oleracea   (So), Lactuca   sativa   (Ls) and Solanum lycopersicum   (SI).<br>
EXAMPLE 1<br>
The Arabidopsis DMR6   (At5q24530) gene is required for downy mildew susceptibility<br>
Experimental procedures<br>
Hyaloperonospora parasitica  growth  and infection<br>
H.  parasitica  isolate Waco9 was provided by Dr. M. Aarts (WUR, Wageningen, NL) and isolate Cala2 provided by Dr. E. Holub (Warwick HRI, Wellsbourne, UK) and maintained on Arabidopsis  Ws-0 and Ler, respectively. Inocula (400.000 spores per ml) were weekly transferred to 10 day old healthy seedlings (Holub, E. B. et al., Mol. Plant Microbe Interact. 7: 223-239, 1994) by use of a spray gun. Seedlings were air-dried for approximately 45 minutes and incubated under a<br><br>
sealed lid at 100% relative humidity in a growth chamber at 16°C  with 9 hours of light per day (100mE/m2/s). The sporulation levels were quantified 7 days post inoculation (dpi) by counting the number of sporangipphores per seedling, for at least 4 0 seedlings per tested line (Figure 6a) or by isolating spores in water 5 dpi and determining the spore concentration to give the number per mg leaf tissue (Figrure 6b) .<br>
Generation   of backcrossed dmr6 lines<br>
The dmrS  mutants were back crossed twice (BC2) to the parental line Ler edsl-2  as well as Ler. The BC2 lines generated with Ler were selected for the presence of the wild type EDSl   gene by PCR analysis.<br>
Fine mapping of the dmr6  gene was done with PCR markers designed using the Cereon database to identify insertion and deletion (IND) differences between Col-0 and Ler. The markers: IND_MOP9 in gene At5G24210; IND_K16H17 in gene At5G24420; IND_T4C12 in gene At5G24820; IND_T11H3 in between genes At5G24950_60 and IND_F21J6 in gene At5G25270 were used for mapping (Table 2). An additional screen for new recombinants was initiated on 300 F2 plants resulting in eight F2 recombinant plants between the two IND based markers IND_M0P9 and IND T4C12, which flanked a region of 61 genes.<br><br>
0\    ^,-.^,.„^^   4-U_<br>
*_ ;  4_ <s v ulic></s>
Seven additional markers (M450-M590; Tabic<br>
region to eighteen candidate genes for the dmr6  locus, between At5g24420 and At5g24590. Sequence analysis of At5g24530 indicated a point mutation leading to a stop codon in exon 2 in the dmr6-l  mutant.<br>
Identification   of a   dmr6  T-DNA  insertion  line<br><br>
A second dmr6  allele was identified, 445D09  a   FLAG T-DNA insertion line generated by INRA Versailles in the Ws-4 accession background. The T-DNA insertion was confirmed by PCR using a primer designed in the At5g24530 gene, LP primer (5'-caggtttatggcatatctcacgtc-3')^ in combination with the T-DNA right border primer, Tag3' (5'-ctgataccagacgttgcccgcataa-3') or RB4 (5'-tcacgggttggggtttctacaggac-3') , The exact T-DNA insertion in the second intron of At5g24530 was confirmed by sequencing of amplicons generated with the T-DNA primers from both the left and right border in combination with the gene specific primers LP or RP (5'-atgtccaagtccaatagccacaag-3') .<br>
cDNA  synthesis<br>
RNA was isolated (from approximately 100 mg leaf tissue from 10 day old seedlings) with the RNaesy kit (Qiagen, Venlo, The Netherlands) and treated with the RNase-free DNase set (Qiagen). Total RNA was quantified using an UVmini-1240 spectrophotometer (Shimadzu, Kyoto, Japan). cDNA was synthesized with Superscript III reverse transcriptase (Invitrogen, Carlsbad, CA, USA) and oligo(dT)15 (Promega, Madison, WI, USA), according manufactures instructions.<br>
Complementation  of  the  dmr6-l  mutant<br>
Complementation lines were generated by transforming dmr6  plants by the floral dip method with Agrobacterium tumefaciens   (Clough and Bent, 1998) containing the At5g24530 gene from Col-0 behind the 35S promoter. The construct was generated by PCR amplification of the full length At5g24530 from Col-0 cDNA with primers which included restriction sites that were used for directional cloning. A forward primer (5'-ttctqaaatccaATGGCGGCAAAGCTGATATC-3') containing a BamHI restriction site near the start codon (ATG), amplified the 5'-end of DMR6  and at the 3'-end after the stop codon an<br><br>
EcoRI site was gene"rated with a reverse primer (5'-qatatataaattcttaattatttagaaaattctcaaaac-3' ). The 35S-DMR6-Tn was cloned into the pGreenII0229 (Hellens,R.P., Edwards,E.A., Leyland,N.R., Bean,S., and Mullineaux,P.M. (2000)). pGreen: a versatile and flexible binary Ti vector for Agrobacterium-mediated plant transformation. Plant Mol. Biol. 42, 819-832). 300 liM DL-Phosphinothricin (BASTA) resistant seedlings were isolated and analyzed for H.  parasitica  susceptibility and for DMR6 expression levels by RT-PCR.<br>
Knock down   lines   of DMR6 by RNAi<br>
RNAi lines were generated in the Ler edsl-2  and Col-0 background. A 782 bp long cDNA ampiicon of Col-0 At5g24530 gene was generated. The PCR was done with the Phusion DNA polymerase (2U/pL) and two different primer combinations. The ampiicon from the first DMR6 gene specific primer combination (RNAiDMR6F: 5'- aaaaaqcaagctGACCGTCCACGTCTCTCTGAA -3' and RNAiDMR6R: 5'- AGAAAGCTGGGTGAAACGATGCGACCGATAGTC -3') was used as a template for the second PCR amplificaton with general primers allowing recombination into the pDONR? vector of the Gateway cloning system. For the second PCR 10 yl of .the first PCR (denaturation for 30 sec. at 98 "C followed by 10 cycles of: 10 sec. at 98'C; 30 sec. at 58 'C; 30 sec. at 72"O in a total volume of 20 ul was used as template. The second PCR (denaturation for 30 sec, at 98 'C followed by 5 cycles of: 10 sec. at 98'C; 30 sec. at 45 'C; 30 sec. at 72'C<br>
_^,.^^    on    ,-..T^-.T*&gt;*-.     /-^ -c    TO     r^ ^ ,^        —4-    rvo*/*i.     on    	~~        —i-     c c      *'~'.     '^ f\     -,__,<br>
an^    i^\j    o_yi--xco    \_&gt;x     x\j    oc\_-.     at.    :? O    \^ t     -f <j isct-:. ci c>-'^        ^/      O VJ    tstiC .<br>
at 72'C finished by a final extention of 10 min. at 72*C) with the attBl (5'-GGGGACAAGTTTGTACAAAAAAGCAGGCT-3') and the attB2 (5'-qqqgaccactttqtacaaaaaaqctqqat -3') were performed in a 50 yl reaction volume. PCR product was gel purified and 50 rig insert was recombined into 150 r)g pD0NR7 vector with the clonase BP enzyme. The vector was transformed into<br><br>
electrocompotent DH5a E.coli cells and plasmids containing the correct insert were isolated and 100 rig of the pD0NR7 with the DMR6 amplicon were used in the LR reaction to recombine the insert in two opposite direction into 150 rjg pHellsgateS vector. After transformation into E.coli, Spectomycin resistant clones'were selected and the isolated plasmids were verified by a NotI digest for the right insert size and by colony PCR with a single internal primer for At5G24530 (DfragmentF: 5'-gagaagtgggatttaaaatagaggaa-3'), if the inserts vjas inserted tvjice in opposite direction an amplicon of 1420 bp could be detected. Correct pHellsgateS plasmids with the double insert in opposite directions were transformed into electrocompotent Agrobacterium strain, C58C1. Plasmids were isolated from the Agrobacterium and retransformed into the E.coli   to confirm the right size of the plasmid and the insert by NotI digestion. The reconfirmed Agrobacterium' strains were used for the floral dip transformation of the Col-0 and Ler edsl-2  plants. The developed seeds were screened for Kanamycin resistance on ^x GM plates, the T^ seedlings were transferred and the next generation of seeds the T2 was analysed for DMR6 expression and H.   parasitica   susceptibility.<br>
Gene  expression profiling of  the dmr6 mutant<br>
Total RNA was isolated as described above. mRNA was amplified with the MessageAmp aRNA kit (Ambion). CATMA array (Crowe et al., 2003) slides containing approximately 25.000 gene specific tags were hybridized according to standardized conditions described by de Jong et al. (de Jong M., van Breukelen B., Wittink,F.R., Menke,F.L., Weisbeek,P.J., and Van den Ackerveken G. (2006). Membrane-associated transcripts in Arabidopsis; their isolation and characterization by DNA microarray analysis and bioinformatics. Plant J. 4 6, 708-<br><br>
721). For quantitative PCR, cDNA templates were generated as described previously. Cycle thresholds were determined per transcript in triplicate using the ABI PRISM 7700 sequence detection system (Applied Biosystems, Foster City, CA, USA) using SYBR Green I (Applied Biosystems, Foster City, CA, USA) as reporter dye. Primer sets"for the transcripts are DMR6 (QDMR6F:5'-TGTCATCAACATAGGTGACCAG-3' and QDMR6R: 5'-CGATAGTCACGGATTTTCTGTG-3'), Atlgl4880 (QAtlgl4880F:5'-CTCAAGGAGAATGGTCCACA-3' and QAtlgl4880R: 5'-CGACTTGGCCAAATGTGATA-l'), At4gl4365 (QAt4gI4365F: 5'-TGGTTTTCTGAGGCATGTAAA-3' and QAt4gl4365R:5'-AGTGCAGGAACATTGGTTGT-3'), ACD6 (QACD6F:5'-TGGACAGTTCTGGA GCAGAT-3' and QACD6R: 5'-CAACTCCTCCGCTGTGAG-3'), PR-5 (QPR-5F:5'-GGCAAATATCTCCAGTATTCACA-3' andQPR-5R: 5'-GGTAGGGCAAT TGTTCCTTAGA-3'), PR-2 {QPR-2 F:5'-AAGGAGCTTAGCCTCACCAC-3' and QPR-2R: 5'- GAGGGAAGCAAGAATGGAAC -3'), PR-1 (QPR-1F:5'-GAACACGTGCAATGGAGTTT-3'and QPR-IR: 5'-GGTTCCACCATTGTTACACCT-3') and ACT-2 (QACT2 F:5'- AATCACAGCACTTGCACCA-3' and QACT2R: 5'- GAGGGAAGCAAGAATGGAAC-3') generating 100 base pair fragments.<br>
Resul'ts<br>
Characterization  of the  gene responsible  for pathogen resistance  in   the dmr6 mutant<br>
Van Damme et al■, 2005, supra disclose a dmr6 mutant that is resistant to H.  parasitica.   The level of resistance can be examined by counting the number of sporangiophores per seedling seven day post inoculation with the H.  parasitica (isolate Waco9 or Cala2, obtainable from Dr. G. Van den Ackerveken, Plant-Microbe Interactions Group, University of Utrecht, Utrecht, NL). The parental line, Ler edsl-2   (Parker et al., 1996, Plant Cell 8:2033-2046), which is highly<br><br>
susceptible, is used as a positive control (and is set at 100%) .<br>
The reduction in sporangiophore formation on the infected dmr6  mutants compared to seedlings of the parental lines is shown in Fig. 6a, wherein the results of the quantification of Hyaloperonospora parasitica,   Waco9 sporulation (sporangiophores/ seedling) on the downy mildew resistant dmr6-l  mutant, back-crossed twice to the parental line Ler edsl-2,   and on mutant dmr6-2   (FLAG_445D09 T-DNA line) compared to the control lines is shown.<br>
According to the invention, the gene responsible for resistance to H.  parasitica  in the cMr6  mutants of van Damme et al., 2005, supra, has been cloned by a combination of mapping and sequencing of candidate genes. Previously, the recessive cMr6  mutation was mapped near the ngal39 marker on chromosome 5 to a region encompassing 14   genes. Fine mapping linked the dmr6 locus to a mapping interval containing the BACs T13K7 and K18P6 between the markers At5g24420 and At5g24590 located in the corresponding genes. This allowed the dmr6  interval to be confined to a region of 18 candidate genes. Comparative sequence analysis of the 18 genes in dmr6 and the parental line, Ler edsl-2  revealed a point mutation in the second exon of the At5g24530 gene. This single base change of G to A, typical for an EMS mutation, changes a TGG a (trp codon) to a TGA (premature stop codon) at nucleotide position 691 of the coding sequence (Figure 7). The early stop codon truncates the predicted oxidoreductase enzyme of 342 aa at position 141 before the conserved catalytic domain suggesting that dmrS  is a null-allele. The At5g24530 coding } sequence (Figure 2) is predicted to encode a protein with a mass of 39.4 kDa. No biological role has so far been described for At5g24530.<br><br>
At5g24530 is DMR6<br>
A second allele, dmr6-2,   was identified in a T-DNA insertion line (FLAG_445D09) from the mutant collection from INRA, Versailles. The presence and location of the T-DNA insert in the second intron of At5g24530 (Figure 7) was confirmed by PCR and sequence analysis (data not shown). Progeny of the FLAG_445D09 line homozygous for the T-DNA insertion was resistant to H.   parasitica  isolate Waco9, whereas the parental line (Ws-4) was susceptible (Figure 6a). The At5g24530 transcript could be amplified by RT-PCR using primers in exon 2 and 3 in Ws-4, but not in the homozygous dmr6-2  line (data not shown), indicating that 
To corroborate the idea that At5g24530 is required for susceptibility to H.   parasitica  the dmr6-l  mutant was transformed with the cDNA from At5g24530 cloned under control of the 35S promoter. In five independent dmr6-l  Tj seedlings the strong overexpression of At5g24530 was confirmed by RT-PCR (data not- shown) . All T3 lines, homozygous for the transgene, showed restoration of susceptibility to H. parasitica  isolate Cala2 (Figure 6b), confirming that At5g24530 is DMR6.   The complementation, together with the identification of two independent dmr6  mutants clearly indicates that a functional DMR6  gene is required for susceptibility to H.  parasitica.<br>
DMR6 is   transcriptionally' activated during H.   'oarasitica infection<br>
To study the expresssion of DMR6  during infection I with H.  parasitica  relative transcript levels were measured by quantitative PCR at six different time points from 0 days (2 hours) post inoculation to 5 days post inoculation (dpi) (Figure 8). RNA was isolated from ten day old Ler seedlings<br><br>
that were spray inoculated with water (mock), compatible, or incompatible H.   parasitica  isolate. At 2 hours post inoculation (0 dpi) the levels of DMR6 transcripts were equal in the different treatments. Starting from 1 dpi, the level of DMR6  transcript was significantly increased in both the compatible and incompatible interaction compared to mock-treated seedlings. The DMR6  transcript level was slightly but significantly higher at 1 dpi in the incompatible interaction (ACT of 3.5, approximately 11 fold induction) than in the compatible (ACT of 3.0, approximately 8 fold induction). The expression level increased further in time to reach a stable high level at 4-5 dpi. At these time points the DMR6 transcript level was higher in the compatible than in the incompatible interaction. The elevated DMR6  transcript levels during compatible and incompatible H.   parasitica   interactions suggest a role of DMR6  in plant defence. The defence-associated expression of DMR6  could be confirmed in our three enhanced-defence mutants, dmr3, dmr4,   and dmr5 (Van den Ackerveken et al., unpublished). Furthermore, in silico analysis of DMR6  levels in the Genevestigator Mutant Surveyor (Zimmermann,P., Hennig,L., and Gruissem,W. (2005). Gene-expression analysis and network discovery using Genevestigator. Trends Plant Sci. 10, 407-409) showed that the gene is strongly induced in the pathogen resistant mutants mpk4  and cpr5. In the cpr5/nprl  double mutant the DMR6 transcript level remained high indicating that the induction of DMR6  expression is mostly NFRl  independent. Salicylic acid appears to be an important signal in the induction of DMR6  expression during senescence as nahG transgenic plants (expressing the bacterial salicylate hydroxylase gene) showed only low levels of DMR6  transcript. To investigate in more detail how the expression of DMR6  is activated during biotic and abiotic stress, DMR6<br><br>
reporter lines were generated. The localisation of DMR6 expression was studied in transgenic Col-0 and Ler edsl-2 plants containing the DMR6  promoter linked to the uidA   ((3-glucuronidase, GUS) reporter gene {pDMR6::GUS) . To visualise both H.  parasitica  hyphal growth, by staining with trypan blue, as well as GUS -activity, magenta-Xgluc was used as a 3-glucuronidase substrate yielding a magenta precipitate. In uninfected plants no GUS expression could be detected in the different plant organelles; roots, meristem, flower, pollen and seed. The expression of DMR6  was induced in the compatible interactions, Ler edsl-2  infected with Cala2 (Figure 9a), and Col-0 infected with isolate Waco9 (Figure 9b). GUS expression was also induced in the incompatible interaction Ler edsl-2  inoculated with isolate Emoy2 (Figure 9c). As shown in figure 9a and 9b DMR6  expression was confined to the cells in which H.   parasitica  had formed haustoria. Plant cells containing the most recently formed haustoria did not show detectable levels of GUS activity (Figure 9a, indicated by asterisk). During the incompatible interaction (Figure 9c) activity of the DMR6  promoter could only be detected in the cells that were in contact with the initial invading hyphae. In death cells, resulting from the hypersensitive response (HR, visualized by trypan blue staining indicated in Figure 9c by asterisk) no GUS activity could be detected, possibly due to protein degradation in these cells. To test if the DMR6  expression in haustoria-containing cells is caused by a wound-like response, seedlings were wound by incision with scissors and stained for GUS activity 3 days later. No detectable promoter DMR6 GUS expression was seen, indicating that the expression of DMR6  is not induced by wounding (Figure 9d). Furthermore the local induction of DMR6  expression was tested in response to treatment with benzothiadiazole (BTH), a functional analogue<br><br>
of salicylic acid (SA). At 3 days post BTH treatment GUS activity was mainly localized in the newly formed, but not in the mature leaves (Figure 9e). Analysis of pDMRS::GUS lines confirm the expression data described above and highlights the strictly localized induction of DMR6  in response to H. parasitica  infection.<br>
The  dmr6~l mutant  constitutively expresses  defence  associated transcripts<br>
To elucidate how the lack of DMR6  results in H. parasitica  resistance, the transcriptome of the dmr6-l  mutant compared to the Ler edsl~2  parental line was analysed. Probes derived from mRNA of the above-ground parts of 14 day old dmr6-l  and Ler edsl-2  seedlings were hybridised on whole genome CATMA micro arrays. A total of 58 genes were found to be significantly differentially expressed in dmro-l,   of which 51 genes had elevated and 7 genes had reduced transcript levels. A pronounced set of the 51 induced transcripts have been identified as genes associated with activated plant defence responses, e.g., ACD6, PR-5, PR-4/HEL and PAD4. These data indicate that the loss of DMR6  results in the activation of a specific set of defence-associated transcripts. The finding that DMR6  is among the dmr6-l-induced genes corroborates the idea that DMR6  is defence-associated. To test if the induced expression of the defence-associated genes was due to the loss of DMR6  and not due to additional ethajie methyl sulfonate (Er-'S) mutations remaining in the backcrossed dmr6~l  mutant the transcript level of a selection of genes (At4gl4365, Atlgl4880, ACD6,   PR-1,   PR-2  and PR-5) was verified by quantitative PCR in both the dmrS-l   and dmrS-2  mutant (Figure 10). We could only test DMR6  transcript levels in the dmr6-l  mutant (Figure 10a) as the dmr6-2  mutant (Figure 10b) has a T DNA insertion disrupting the DMR6<br><br>
transcript. The induction of DMR6  as observed in the micro array analysis was confirmed by Q-PCR in dmr6-l compared to Ler edsl-2   (Figure 10a). Figure 10a and b show that all six selected genes were elevated in both ^dmr6  mutants compared to the parental lines. The observed elevated expression of the selected defence-associated genes in the dmr6  mutants indicates that lack of DMR6  activates a plant defence response. The activation of this set of defence-associated transcripts could be the primary cause of resistance to H. parasitica   in the dinrS  mutants.<br><br>
EXAMPLE 2<br>
Identification of DMR6 ortholoas in crops<br>
1.	Screening of libraries  on  the basis  of sequence homology<br>
The nucleotide and amino acid sequences of the DMR6 coding sequence and protein of Arabidopsis  thaliana  are shown in Pig. 2. Public libraries of nucleotide and amino acid sequences were compared with the sequences of Fig. 2. This comparison resulted in identification of the complete DMR6 coding sequences and predicted amino acid sequences in Aquilegia  species,   Citrus sinensis,   Coffea  canephora,   Cucumis sativus,   Gossypium hirsitum,   Lactuca  sativa,  Medicago truncatula,   Oryza   sativa    (3),   Populus   trichocarpa    (2), Solanum  lycopersicum   (2),   Sorghum bicolor,   Spinacia   oleracea, Vitis  vinifera,   Zea mays,   and Zingiber officinale.   The sequence information of the orthologous proteins thus identified is given in Table 1 and visualized in an multiple alignment in Fig. 1. For many other plant species orthologous DNA fragments could be identified by BlastX as reciprocal best hits to .the Arabidopsis  or other plant DMR6 protein sequences.<br>
2.	Identification  of orthologs by means  of heterologous<br>
I hybridisation<br>
The DMR6 DNA sequence of Arabidopsis   thaliana  as shown in Fig. 2 is used as a probe to search for hoiriologous sequences by hybridization to DNA of any plant species using standard molecular biological methods. Using this method )  orthologous genes are detected by southern hybridization on restriction enzyme-digested DNA or by hybridization to genomic or cDNA libraries. These techniques are well known to the person skilled in the art. As an alternative probe the<br><br>
DMR6 DNA sequence of any other more closely related plant species can be used as a probe.<br>
3. Identification  of orthologs by means of PCR<br>
For many crop species, partial DMR6 mRNA or gene sequences are available that'are used to design primers to subsequently PCR amplify the complete cDNA or genomic sequence. When 5' and 3' sequences are available the missing internal sequence is PCR amplified, by a DMR6 specific 5' forward primer and 3' reverse primer. In cases where only 5', internal or 3' sequences are available, both forward and reverse primers are designed. In combination with available plasmid polylinker primers, inserts are amplified from genomic and cDNA libraries of the plant species of interest. In a similar way, missing 5' or 3' sequences are amplified by advanced PCR techniques; 5'RACE, 3' RACE, TAIL-PCR, RL^a-RACE or vectorette PCR.<br>
As an  example the sequencing of the Lactuca  satlva (lettuce) DMR6 cDNA is provided. From the Genbank EST database at NCBI several Lactuca  DMR6 ESTs were identified using the tblastn tool starting with the y^lraJbidopsis DMR6 amino acid sequence. Clustering and alignment of the ESTs resulted in a consensus sequence for a 5' DMR6 fragment. To obtain the complete lettuce DMR6 cDNA the RLM-RACE kit (Ambion) was used on mRNA from lettuce seedlings. The 3' mRNA sequence was obtained by using two primers that were designed in the 5' DMR6 consensus sequence derived front ESTs (Lsat_dmr6_fwl: CGATCAAGGTCAACACATGG, and Lsat_dmr6_fw2: TCAACCATTACCCAGTGTGC) and the 3'RACE primers from the kit. Based on  the assembled sequence new primers were designed to amplify the complete DMR6 coding sequence from cDNA to provide the nucleotide sequence and derived protein sequence as presented, in Figure 3.<br><br>
The complete DMR6 coding sequences from more than 10 different plants species have been identified from genomic and EST databases. From the alignment of the DNA sequences, conserved regions in the coding sequence were selected for the design of degenerate oligonucleotide primers (for the degenerate nucleotides the abbreviations are according to the TUB nucleotide symbols that are standard codes used by all companies synthesizing oligonucleotides; G = Guanine, A = Adenine, T = Thymine, C = Cytosine, R = A or G, Y = C or T, M = A or C, K = G or T, S = C or G, W = A or T, B = C or G or T, D = G or A or T, H = A or C or T, V = A or C or G, N = A or C or G or T) .<br>
The procedure for obtaining internal DMR6 cDNA sequences of a given plant species is as follows:<br>
1.	mRNA is isolated using standard methods,<br>
2.	cDNA is synthesized using an oligo dT prim.er and standard methods,<br>
3.	using degenerate forward and reverse oligonucleotides a PCR reaction is carried out,<br>
4.	PCR fragments are separated by standard agarose gel electrophoresis and fragments of the expected size are isolated from the gel,<br>
5.	isolated PCR fragments are cloned in a plasmid vector using standard methods,<br>
6.	plasmids with correct insert sizes, as determined by PCR, are analyzed by DNA sequencing,<br>
7.	Sequence analysis using blastX reveals which fragments contain the correct internal DMR6 sequences,<br>
8.	The internal DNA sequence can then be used to design gene-and species- specific primers for 5' and 3' RACE to obtain the complete DMR6 coding sequence by RLM-RACE {as described above).<br><br>
As an example the sequencing of the Cucumis  sativus (cucumber) DMR6 cDNA is provided. For cucumber several primer combinations between the following primers were successful in amplifying a stretch of internal coding sequence from cDNA; forward primers dmr6_deg_fwlB (TTCCAGGTDATTAAYCAYGG), dmr6_deg_fw2B (CATAAYTGGAGRGAYTAYCT), dmr6_deg_fw3B (GARCAAGGRCARCAYATGGC) and dmr6_deg_fw4<br>
(AATCCTCCTTCHTTCAAGGA) and reverse primers dmr6_deg_rv3B (AGTGCATTKGGGTCHGTRTG) , dmr6_deg_rv4 (AATGTTRATGACAAARGCAT) and dmr6_deg_rv5 (GCCATRTGYTGYCCTTGYTC). After cloning and sequencing of.the amplified fragments cucumber DMR6-specific primers were designed for 5' RACE (Cuc_dmr6_rvl: TCCGGACATTGAAACTTGTG and Cuc_dmr6_rv2: TCAAAGAACTGCTTGCCAAC) and 3' RACE (Cuc_dmr6_fwl: CGCACTCACCATTCTCCTTC and Cuc_dmr6_fw2: GGCCTCCAAGTCCTCAAAG). Finally the complete cucumber DMR6 cDNA sequence was amplified and sequenced (Figure 5). A similar approach was a used for spinach, Spinacia  oleracea   (Figure 4), Solanum lycopersicum   (Figure 12) and Nicotlana benthamiana   (Figure 13) .<br>
Orthologs identified as described in this example can be modified using well-known techniques to induce mutations that reduce the DMR6 expression or activity, to obtain non-genetically modified plants resistant to Fungi or Oomycota. Alternatively, the genetic information of the orthologs can be used to design vehicles for gene silencing, and to transform the corresponding crop plants to obtain plants that are resistanL Lo Oomycota.<br>
EXAMPLE 3<br>
Mutation of seeds<br>
Seeds of the plant species of interest are treated with a mutagen in order to introduce random point mutations<br><br>
in the genome. Mutated plants are grown to produce seeds and the next generation is screened for the absence of reduction of DMR6 transcript levels or activity. This is achieved by monitoring the level of DMR6 gene expression, or by searching for nucleotide changes (mutations) by the TILLING method, by DNA sequencing, or by any other method to identify nucleotide changes. The selected plants are homozygous or are made homozygous by selfing or inter-crossing. The selected homozygous plants with absent or reduced DMR5 transcript activity are tested for increased resistance to the pathogen of interest to confirm the increased disease resistance.<br>
EXAMPLE 4<br>
Transfer of a mutated allele into the background of a desired crop<br>
Introgression of the desired mutant allele into a crop is achieved by crossing and genotypic screening of the mutant allele. This is a standard procedure in current-day marker assistant breeding of crops.<br>
EXAMPLE 5<br>
Use of the DMR6  promotor for pathogen-induced gene expression and the generation of disease resistant plants<br>
Precise control of transgene expression is pivotal Lo the engineering of plants with increased disease resistance. In the past, constitutive overexpression of transgenes )  frequently has resulted in poor quality plants. It has<br>
therefor been suggested to use pathogen-inducible promoters, by which the transgenes are expressed only when and where they are needed - at infection sites.<br><br>
Local and i'nducible expression of engineered genes, e.g. master switch genes, elicitor or Avr genes, anti¬microbial genes, or toxic genes, results in the activation of defense or cell death that will lead to pathogen resistance, such as described by Gurr and Rushton (Trends in Biotechnology 23: 275-282, 2D05). A good example is provided by De wit (Annu. Rev. Phytopathol. 30: 391-418, 1992) who proposes the use of the Avr9-Cf9 combination to achieve induced cell death leading to disease resistance. The tissue-specificity and inducibility of expression is of prime importance for such approaches, as described by Gurr and Rushton (Trends in Biotechnology 23: 283-290, 2005).<br>
According to the present invention, the DMR6 promoter has been demonstrated to show a strong, inducible, localized expression based on promoter-GUS analysis. Thus, the DMR6 promoter is very suitable for engineering disease resistance in transgenic plants. The DMR6 promoter consists of a region of 2.5 kb that is upstream of the Arabidopsis DMR6 coding sequence (ATG start codon) and includes the 5'UTR (as depicted in Figure 11). This pathogen-inducible promoter is then used to engineer suitable transgene constructs, using standard techniqus known the person skilled in the art.<br>
Using orthologous DNA sequences from a given plant species primers are designed for PCR. These are then used to screen genomic libraries of the plant species of interest to identify the genomic clones that contain the DMR6 ortholog with its promoter and regulatory sequences. Alternatively, the genomic clones are isolated by screening a library with a labelled PCR fragment corresponding to the DMR6 orthologous gene. Sequencing reveals the nucleotide sequence of the promoter. The region of 2-5 kb upstream the DMR6 orthologous coding sequence (ATG start codon), so including the 5'UTR, is then amplified by PCR to engineer transgene constructs for<br><br>
plant transformation EXAMPLE 6<br>
This example demonstrates the complementation of mutant c±nr6-l  in Arabidopsis'thaliana  by DMR6  orthologs from 4 different crop species. For this, DMR6  orthologs of Cucumis sativa   (Cs), Spinacia  oleracea   (So), Lactuca  sativa   (Ls) and Solanum lycopersicum   (SI) were cloned into a plant expression vector under the control of the 35S promoter and, subsequently, this vector was transformed into a Arabidopsis thaliana  mutant dmr6-l.<br>
Briefly, mRNA was isolated using standard methods and cDNA was synthesized using an oligo dT primer and standard methods. Subsequently, PCR fragments were generated using primer pairs for each crop as depicted in table 3 below. The generated PCR products were cloned into a pENTR/D-TOPO vector using the pENTR/D-TOPO cloning kit from Invitrogen and resulting plasmids with correct insert sizes, as determined by PCR, were analyzed by DMA sequencing. Recombination to the pB7WG2,0 vector was done using LR clonase II from Invitrogen and the resulting plasmids were analyzed by PCR and digestion with restriction enzymes. Suitable plasmids were transformed into Agrobacterium  tumefaciens  C58C1 PGV2260 and plasmids from Agrobacterium were analyzed by PCR and digestion with restriction enzymes.<br>
Arabidopsis  thaliana  dmr6-l plants were transformed with the above constructs by dipping into Agrobacterium solution and overexpression of crops DMR6 in Arabidopsis Tl )  plants is verified by RT-PCR using the crops DMR6 cloning<br>
primers (table 3). Finally, Arabidopsis T2 and T3 plants were infected with Hyaloperonospora parasitica  Cala2 to confirm complementation. The results are shown in figure 14.<br><br>
As shown in" figure 14, all DMR6  orthologs tested were capable of complementing Arabidopsis  thaliana  mutant dmr6-l indicating that the DMR6  orthologs identified encode DMR6 proteins with a similar functionality as Arabidopsis  thaliana DMR6.<br>
TABLES<br>
Table 1 lists the GI numbers (Genlnfo identifier) and Genbank accession number for Expressed Sequence Tags (ESTs) and mRNA or protein sequences of the Arabidopsis DMR6 mRNA and orthologous sequences from other plant species. A GI number (genlnfo identifier, sometimes written in lower case, "gi") is a unique integer which identifies a particular sequence. The GI number is a series of digits that are assigned consecutively to each sequence record processed by NCBI. The GI number will thus change every time the sequence changes. The NCBI assigns GI numbers to all sequences processed into Entrez, including nucleotide sequences from<br>
DDBJ/EMBL/GenBank, protein sequences from SWISS-PROT, PIR and many others. The GI number thus provides a unique sequence identifier which is independent of the database source that specifies an exact sequence. If a sequence in GenBank is modified, even by a single base pair, a new GI number is assigned to the updated sequence. The accession number stays the same. The GI number is always stable and retrievable. Thus, the reference to GI numi)ers in the table provides a clear and unambiguous identification of the corresponding sequence.<br><br><br>
CLAIMS<br>
1.	Plant, which is resistant to a pathogen of viral, bacterial, fungal or oomycete origin, characterized in that the plant has a reduced level, reduced activity or complete absence of DMR6 protein as compared to a plant that is not resistant to the said pathogen.<br>
2.	Plant as claimed in claim 1, wherein the pathogen is an organism belonging to the Fungi or the phylum Oomycota.<br>
3.	Plant as claimed in claim 2,   wherein the pathogen is a Albugo,   Aphanomyces,   Basidiophora,   Bremia, Hyaloperonospora,   Pachymetra,   Paraperonospora,   Perofascia, Peronophythora,   Peronospora,   Peronosclerospora,   Phytium, Phytophthora,   Plasmopara,   Protobremia,   Pseudoperonospora, Sclerospora,   Viennotia  species.<br>
4.	Plant as  claimed in claim 2 or 3, wherein the plant and the pathogen are selected from Bremia  lactucae  on lettuce, Peronospora  farinosa  on spinach, Pseudoperonospora cubensis  on members of the CucurJbitaceae family, e.g. cucumber and melon, Peronospora  destructor  on onion, Hyaloperonospora parasitica  on members of the Brasicaceae family,   e.g.   cabbage,   Plasmopara  viticola  on  grape, Phytophthora  infestans  on tomato and potato, and Phytophthora sojae  on soybean.<br>
5.	Plant as claimed in any one of the claims 1-4, having a mutation in its DMR6  gene resulting in a DMR6 protein with reduced enzymatic activity as compared to the DMR6 protein encoded by the wild-type DME6  gene wherein no such mutation is present.<br>
6.	Plant as claimed in claim 5, wherein the mutation in the DMR6  gene leads to an amino acid substitution in the encoded protein.<br><br>
7.	Plant as claimed in any one of the claims 1-6, having a mutation in its DMR6  gene resulting in a reduced DMR6 expression as compared to the wild-type DMR6 gene wherein no such mutation is.present.<br>
8.	Plant as claimed in any one of the claims 1-4, having a mutation in the regQlatory sequences of its DMR6 gene that affects the expression of the encoded DMR6 protein,<br>
9.	Plant as claimed in any of the claims 5-8, wherein the gene is an orthologous gene of the Arabidopsis DMR6  gene (At5g24530) as shown in Figure 2.<br><br>
10.	Plant as claimed in any one of the claims 1-8, wherein the gene is a DMR6 orthologous gene as identified in the list of Table 1.<br>
11.	Plant as claimed in any one of the claims 1-8, wherein the gene is a DMR6 orthologous gene of Lactuca   sativa having the nucleotide sequence and amino acid sequence as shown in Figure 3.<br>
12.	Plant as claimed in any one of the claims 1-8, wherein the gene is a DMR6 orthologous gene of Spinacia oleracea  having the nucleotide sequence and amino acid sequence as shown in Figure 4.<br>
13.	Plant as claimed in any one of the claims 1-8, wherein the gene is a DMR6 orthologous gene of Cucumis sativus   (cucumber), or other Cucumis  species, e.g. melon, having the nucleotide sequence and amino acid sequence as shown in Figure 5.<br>
14.	Method for obtaining a plant, v.-hich is resistant to a pathogen of viral, bacterial, fungal or oomycete origin, comprising reducing the endogenous level of DMR6 protein in the plant.<br>
15.	Method as claimed in claim 14, wherein reducing the level of DMR6 protein is achieved by mutation of the DMR6 gene of the plant.<br><br>
16.	Method as claimed in claim 15, wherein reducing the level of DMR6 protein is achieved by mutation of the DME6 gene to induce one or more amino acid changes that lead to a DMR6 protein with a reduced enzymatic activity.<br>
17.	Method as claimed in claim 15 or 16; wherein the mutation is effected by mutagenic treatment of the plant, in particular with mutagens or radiation.<br>
18.	Method as claimed in claim 14, wherein reducing the endogenous level in the plant is achieved by reducing the expression of the DMR6  gene of the plant.<br>
19.	Method as claimed in claim 18, wherein reducing the expression of the DMR6  gene of the plant is achieved by gene silencing or RNAi.<br>
20.	Method as claimed in claim 18, wherein reducing the expression of the DMR6  gene of the plant is achieved by mutagenesis of regulatory elements in the promoter region, terminator region or an intron.<br>
21.	Method as claimed in claim 18, wherein reducing the expression of the DMR6  gene of the plant is achieved by overexpressing repressor proteins of the DMR6  gene.<br>
22.	Method as claimed in claim 18, wherein reducing the expression of the DMR6  gene of the plant is achieved by silencing or mutation of plant genes encoding activating or regulatory proteins.<br>
23.	Method as claimed in any of the claims 14-22, wherein the DMR6 gene to be mutated is an orthologue of the Arabidopsis DMR6  gene {At5g24530) as shov;n in Figure 2.<br>
24.	Method as claimed claim 23, wherein the gene is a DMR6 orthologous gene as identified in the list of Table 1.<br>
25.	Method as claimed in claim 23, wherein the gene is a DMR6 orthologous gene of Lactuca  sativa  having the nucleotide sequence and amino acid sequence as shown in Figure 3.<br><br>
25. Method as claimed in claim 23, wherein the gene is a DMR6 orthologous gene of Spinacia  oleracea  having the nucleotide sequence and amino acid sequence as shown in Figure 4.<br>
27.	Method as claimed in claim 23, wherein the gene is a DMR6 orthologous gene of Cucumis  sativus  having the nucleotide sequence and amino acid sequence as shown in Figure 5.<br>
28.	Mutated plant DMR6  gene encoding a DMR6 protein having a reduced enzymatic activity.<br>
29.	Mutated plant DMR6  gene as claimed in claim 28, comprising a dmr6  allele with a premature stop codon.<br>
30.	Mutated plant gene as claimed in claim 29, comprising the dmr6-l   allele.<br>
31.	Use of a DMR6 promotor for providing disease resistance in plants.<br>
32.	Method of providing disease resistance in a plant, comprising transforming a plant cell with a DNA construct comprising at least one expressible nucleic acid which is operably linked to a pathogen-inducible promotor that is operable within a plant cell, and regenerating transformed plants from said plant cells, wherein the pathogen-inducible promotor is a DMR6 promotor, and wherein the expression of the expressible nucleic acid confers disease resistance to the transgenic plant.<br>
33.	Method as claimed in claim 32, wherein the expressible nucleic acid is a gene involved, in olant defence response.<br>
34.	Method as claimed in claim 32 or 33, wherein the expressible nucleic acid encodes an autologous or heterologous polypeptide capable of conferring disease-resistance to said plant.<br><br>
35.	Method as claimed in claim 32, 33 or 34,<br>
wherein the DMR6 promotor is the Arabidopsis DMR6 promoter.<br>
36.	Method as claimed in claim 35, wherein the DMR6<br>
promotor comprises a nucleotide sequence as defined in Figure<br>
11, and/or a functional fragment, and/or a natural variant<br>
thereof.<br>
37.	Method as claimed in claim 32, 33 or 34,<br>
wherein the DMR6 promotor is an orthologous DMR6 promotor.<br>
38.	DNA construct, comprising at least one<br>
expressible nucleic acid which is operably linked to a<br>
pathogen-inducible promotor, wherein the promotor is a DMR6<br>
promotor.<br>
39.	DNA construct as  claimed in claim 38, wherein<br>
the expressible nucleic acid is a gene involved in plant<br>
defence response.<br>
40.	DNA construct as claimed in claim 38 or 39,<br>
wherein the expressible nucleic acid encodes an autologous or<br>
heterologous polypeptide capable of conferring disease-<br>
resistance to said plant.<br>
41.	DNA construct as claimed in claim 38, 39 or 40,<br>
wherein the DMR6 promotor is the Arabidopsis DMR6 promotor.<br>
42.	Method as claimed in claim 41, wherein the DMR6<br>
promotor comprises a nucleotide sequence as defined in Figure<br>
11, and/or a functional fragment, and/or a natural variant<br>
p thereof<br>
43.	Method as claimed in claim 38, 39 or 40,<br>
wherein the DMRfi promotor is an orthologous DMR6 promotor.<br><br><br><br><br><br><br><br><br><br><br><br><br></j></td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/viewdoc.aspx?id=lWesLK5hzb3KzEBHhzaf3g==&amp;amp;loc=egcICQiyoj82NGgGrC5ChA==" target="_blank" style="word-wrap:break-word;">http://ipindiaonline.gov.in/patentsearch/GrantedSearch/viewdoc.aspx?id=lWesLK5hzb3KzEBHhzaf3g==&amp;amp;loc=egcICQiyoj82NGgGrC5ChA==</a></p>
		<br>
		<div class="pull-left">
			<a href="271439-functional-starch-powder.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="271441-multi-stage-process-for-producing-polyethylene-with-lowered-gel-formation.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>271440</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>4525/CHENP/2009</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>09/2016</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>26-Feb-2016</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>22-Feb-2016</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>31-Jul-2009</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>ENZA ZADEN BEHEER B.V.</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td>HALING 1 E, NL-1602, DB KHUIZEN,</td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>VAN DAMME, MIREILLE, MARIA, AUGUSTA</td>
											<td>VERVERSSTRAAT 2G, 1011 JA AMSTERDAM,</td>
										</tr>
										<tr>
											<td>2</td>
											<td>VAN DEN ACKERVEKEN, AUGUSTINUS, FRANCISCUS, JOHANNES, MARIA</td>
											<td>DIJKGRAAFPOLDER 12, 3991 ZC HOUTEN,</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>C12N15/82</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>PCT/EP08/00718</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td>2008-01-30</td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td>PCT/EP07/50976</td>
									<td>2007-02-01</td>
								    <td>PCT</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/271440-a-method-for-obtaining-a-lettuce-plant-resistant-to-bremia-lactucae by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 01:30:41 GMT -->
</html>
