<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/271544-mannitol-induced-promoter-systems-in-bacterial-host-cells by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 00:59:45 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 271544:&quot;MANNITOL INDUCED PROMOTER SYSTEMS IN BACTERIAL HOST CELLS&quot;</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">&quot;MANNITOL INDUCED PROMOTER SYSTEMS IN BACTERIAL HOST CELLS&quot;</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>The present invention provides methods for producing recombinant peptides in a bacterial host utilizing a mannitol, arabitol, glucitol, or glycerol-inducible promoter, wherein the host bacterial cell that produces the peptide has been rendered incapable of degrading or metabolizing mannitol, arabitol, or glucitol, or derivatives or analogues thereof. The present invention provides bacterial cells that have been genetically altered to inhibit the metabolism or degradation of mannitol, glucitol, or arabitol, or derivatives or analogues thereof. The present invention utilizes mannitol, arabitol, glucitol, or glycerol to induce expression of a target polypeptide from an inducible promoter, allowing for the use of an inexpensive and stable carbon source inducer in the fermentation processes for the production of recombinant peptides.</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td>MANNITOL INDUCED PROMOTER SYSTEMS IN BACTERIAL HOST CELLS<br>
FIELD OF THE INVENTION<br>
The present invention relates to the field of recombinant peptide production. In particular, the present invention provides for the improved production of recombinant peptides utilizing a mannitol, glucitol, or arabitol-inducible promoter, wherein the host cell expressing the peptide from the promoter is rendered deficient in its ability to metabolize or degrade mannitol, glucitol, or arabitol, which is utilized to induce the promoter. In addition, the invention provides for improved bacterial host cells for the production of recombinant peptides, wherein the bacterial cell has been rendered deficient in its ability to metabolize or degrade mannitol, glucitol, or arabitol, or analogues or derivatives thereof.<br>
BACKGROUND OF THE INVENTION<br>
The use of bacterial cells to produce recombinant peptides is increasing in commercial importance. One of the goals in developing a bacterial expression system is the production of high quality target polypeptides quickly, efficiently, and abundantly. An ideal host cell for such an expression system would be able to efficiently utilize a carbon source for growth, efficiently produce a target polypeptide, quickly grow to high cell densities in a fermentation reaction, express the target polypeptide only when induced, and allow for the induction of the target polypeptide in an inexpensive and efficient manner.<br>
One hurdle to the creation of the ideal host cell is overcoming inefficient and low level production of target polypeptides in the fermentation process. Controlling expression of the target peptide until optimal host cell densities and fermentation conditions are reached allows for a more efficient and larger yield of polypeptide. The reasons for this are several fold, including a more efficient utilization of a carbon source and the reduction of extended metabolic stresses on the host cell.<br>
One way to control expression of target polypeptides during the fermentation process includes the use of inducible or regulatable promoters to control expression of the peptides. Promoters are regulatory nucleic acid sequences generally located upstream from the desired peptide coding sequence, directing the transcription of the nucleic acid. Promoters are generally classified as constitutive or regulated promoters. Regulated promoters include: (1) activatable promoters, which are inactive until an activator peptide binds to the 5' regulatory regions; and (2) repressible promoters, which are inactive while the 5' regulatory region is<br><br>
bound by a represser peptide.   Some genes or operons are regulated by more than one mechanism.<br>
An inducible or regulatable promoter can be an essential component for the production of high levels of recombinant peptides in an efficient manner, since such regulatable promoters allow for cell densities to reach optimal levels prior to the induction of peptide production. Attributes of an ideal promoter can include: tight repression so that little or no target peptide is made during the growth phase; strong promoter activity after the addition of the inducer; low cost of induction; stability of the inducer in the cell culture medium so that the inducer need only be added once during the peptide accumulation phase of a fermentation process; easy passage of the inducer of the promoter through the cell membrane so that the external concentrations of the inducer are directly related to the internal concentration within the cell, and linearly related to peptide production; and ability to be used in tandem with other promoters. Such an ideal promoter allows recombinant peptides to be induced at high levels efficiently and inexpensively.<br>
One regulatable promoter that has found widespread use in bacterial fermentation process for the production of recombinant peptides is the lac promoter, and its derivatives, especially the tac and trc promoters. In commercial fermentation systems using a lac-type promoter, the inducer isopropyl-p-D-1-thiogalactopyranoside ("IPTG") is almost universally employed. IPTG is, however, expensive and must be carefully controlled since it is significantly toxic to biological systems. Standard IPTG preparations are currently available at about USD $18 per gram or about USD $125 per 10 grams. In addition, standard IPTG preparations may contain dioxane, a toxin. Dioxane-free IPTG is available on the market, but costs roughly twice the price of standard IPTG. Furthermore, environmental and health regulatory issues arise in regard to the presence of IPTG in the fermentation, or in the protein purified from the fermentation, given the IPTG toxicity risks to humans, animals, and other biological organisms.<br>
Because of the toxicities and costs associated with IPTG, alternative inducible promoter systems have been proposed for use in bacterial fermentation processes for the production of recombinant peptides. For example, promoters induced by high temperatures such as ^PR and ^PL, tryptophan starvation such as trp, 1-arabinose such as araBAD, phosphate starvation such as phoA, nalidixic acid such as recA, osmolarity such as proU, glucose starvation such as cst-1, tetracycline such as tetA, pH such as cadA, anaerobic conditions such as nar, T4 infection such as T4 gene32, alkyl- or halo-benzoates such as Pm, alkyl- or halo-toluenes such as Pu, salicylates such as Psal, and oxygen such as VHb, have all<br><br>
been examined as alternatives to IPTG inducible promoters. See, for example, Makrides, S.C. (1996) Microbiol. Rev. 60, 512-538; Hannig G. &amp; Makrides, S.C. (1998) TIBTECH 16, 54-60; Stevens, R.C. (2000) Structures 8, R177-R185; J. Sanchez-Romero &amp; V. De Lorenzo, Genetic Engineering of Nonpathogenic Pseudomonas strains as Biocatalysts for Industrial and Environmental Processes, in Manual of Industrial Microbiology and Biotechnology (A. Demain &amp; J. Davies, eds.) pp.460-74 (1999) (ASM Press, Washington, D.C.); H. Schweizer, Vectors to express foreign genes and techniques to monitor gene expression for Pseudomonads, Current Opinion in Biotechnology, 12:439-445 (2001); and R. Slater &amp; R. Williams, The Expression of Foreign DNA in Bacteria, in Molecular Biology and Biotechnology (J. Walker &amp; R. Rapley, eds.) pp. 125-54 (2000) (The Royal Society of Chemistry, Cambridge, UK). Several problems exist with these types of promoters. For example: high temperature induction may be harmful to cells, and may not be practical for large scale fermentation due to equipment limitations; oxygen manipulation may affect the overall dynamics of the cell growth density aspects of the fermentation, reducing ideal conditions; the use of toluenes or other similar types of potentially toxic chemicals may require further purification to ensure that these compounds are not present in the final product; and pH may affect the ability of the peptide of interest to correctly fold or be solubilized in the host, making purification more costly and difficult.<br>
Promoters that can be induced by inexpensive and non-toxic carbon sources remain attractive for use in bacterial fermentation processes. One potential advantage to such a promoter is that bacterial host may have endogenous mechanisms to efficiently uptake the inducer. Potential carbon sources that can be used as inducers include maltose, maltodextrin, glucose, arabinose, fructose, galactose, sucrose, glycerol, mannose, acetate, and lactose. Other potential carbon source inducers include the alcohol forms of carbon sugars, such as mannitol, glucitol, and arabitol.<br>
Mannitol as Potential Inducer<br>
Mannitol is the alcohol form of mannose, and is an inexpensive carbon source for a number of bacteria, including Pseudomonads. The operon involved with uptake and degradation of mannitol in Pseudomonas fluorescens (P. fluorescens) contains seven genes, four of which (mtlEFGK) encode proteins involved in mannitol/glucitol/arabitol uptake and transport, and three of which (mtlDYZ) encode proteins involved in the catabolism of mannitol, glucitol, and arabitol. See Brunker et al. (1998) "Structure and function of the<br><br>
genes involved in mannitol, arabitol, and glucitol utilization from Pseudomonas fluorescens DSM50106," Gene 206(1): 117-126 and Figure 1.<br>
Brunker et al. have further identified a sequence similar to the consensus for E. coll sigma 70 promoters 90 bp upstream from the start codon of the first gene in the P. fluorescens DSM50106 operon, mtlE. See Figure 2. A 660 bp fragment containing the putative promoter was cloned upstream of a luciferase gene upon which it conferred mannitol-inducible expression. Arabitol induced expression of the gene to the same level, and glucitol induced expression to a level half as high.<br>
The benefits of carbon source inducible promoters, however, is not without its limitations. One potential problem associated with utilizing carbon source inducible promoters is the ability of the host cell to metabolize the inducer and reduce the effectiveness of the inducer, or require it to be continually added to the media during induction. In addition, the inducer may compromise carbon utilization parameters of the fermentation process, and the constant inducer flux may result in less than desirable peptide production yields. The requirement of continually adding an inducer to the media during induction has the further disadvantage of increasing the cost of the fermentation process.<br>
SUMMARY OF THE INVENTION<br>
The present invention provides bacterial cells and methods for producing recombinant peptides in a bacterial host utilizing a mannitol, glucitol, or arabitol-inducible promoter operably attached to a nucleic acid encoding a target polypeptide, wherein the bacterial host cell has been rendered incapable of degrading or metabolizing mannitol, glucitol, or arabitol. Because the bacterial cell lacks the ability to metabolize or degrade a carbon source selected from the group consisting of mannitol, glucitol, arabitol, and derivatives or analogues thereof, these carbon sources can be utilized to induce expression of a polypeptide, wherein the nucleic acid encoding the polypeptide is operably attached to a mannitol, glucitol, or arabitol-inducible promoter, without the inducer being continuously removed from the media by the host cell. The inducer, therefore, is capable of providing continuous and stable induction levels during fermentation without the requirement of adding additional inducer to the media.<br>
Embodiments of the present invention includes bacterial cells that have been genetically altered to inhibit the metabolism or degradation of a carbon source selected from the group consisting of mannitol, glucitol, arabitol, and derivatives or analogues thereof. The bacterial cells can be transformed with a nucleic acid construct encoding a recombinant peptide of interest operably attached to a inducible promoter selected from the group<br><br>
mannitol-, glucitol, or arabitol-inducible promoters, allowing for the use of mannitol, glucitol, or arabitol, or derivatives or analogues thereof, as an inexpensive and stable inducer in the fermentation processes for the production of recombinant peptides without the subsequent metabolization or degradation of the inducer in the fermentation process.<br>
The promoter utilized in the present invention is capable of induction by a carbon source selected from the group consisting of a carbon source sugar, alcohol, or derivatives thereof. The promoter can be induced by the group selected from mannitol, arabitol, or glucitol. Alternatively, the promoter can also be induced by glycerol. In some embodiments the promoter can be a mannitol inducible promoter. In other embodiments the promoter can include the putative promoter of an endogenous bacterial mannitol operon, and is induced by mannitol derivatives or analogues thereof. In additional embodiments the promoter can include the putative promoter of the Pseudomonas fluorescens mtlEFGKDYZ operon. In yet further embodiments the promoter can further includs a nucleic acid sequence that acts as an mtl activator protein (MtlR) binding region. Additional embodiments include wherein the promoter is the nucleic acid sequence selected from the group consisting of Seq. ID. Nos. 1-13. In other embodiments the mannitol promoter is capable of being induced by a carbon source selected from the group consisting of mannitol, glucitol, arabitol, glycerol, or derivatives or analogues thereof.<br>
Embodiments of the present invention also include bacterial cells for use in expressing recombinant peptides of interest are provided that have been rendered incapable of metabolizing or degrading a carbon source selected from the group consisting of mannitol, glucitol, arabitol, and derivative or analogue thereof. In some embodiments the mutation is a result of exogenous genetic manipulation through the mutation of at least one gene, or the deletion of at least one gene encoding an enzyme required for the metabolism or degradation of the carbon source used as an inducer. In other embodiments the mutated or deleted gene(s) is a gene(s) encoding for an enzyme(s) required in the metabolism or degradation of mannitol, arabitol, or glucitol, or derivative or analogue thereof. In another embodiment, the mutated or deleted gene(s) is a gene(s) required in the metabolism of mannitol, or derivatives or analogues thereof. In further embodiments the gene is selected from the mtl operon. The mutated or deleted gene can be selected from the group consisting of mtlD, mtlY, and mtlZ. In an alternative embodiment, the mutation or deletion includes any combination of mtlD, mtlY, and mtlZ (mtlDYZ). In additional embodiments the mutation or deletion can include mtlD, mtlY, and mtlZ (mtlDYZ).<br><br>
In some embodiments the bacterial cell further comprises a nucleic acid construct including a mannitol, arabitol, or glucitol-inducible promoter operably linked to a peptide of interest. Alternatively, one embodiment includes the bacterial cell further comprising a nucleic acid construct or constructs including more than one mannitol, arabitol, or glucitol-inducible promoter operably linked to the same, or different, peptide or peptides of interest. In additional embodiments at least one of the promoters is a mannitol-inducible promoter, and the bacterial cell expressing the recombinant peptide of interest has been genetically manipulated to inhibit degradation or metabolism of mannitol. Alternatively, the promoter is capable of induction by glycerol, as well as mannitol, and the bacterial cell expressing the recombinant peptide of interest has been genetically manipulated to inhibit degradation or metabolism of mannitol.<br>
In some embodiments the bacterial cell is selected from any bacterial cell capable of expressing a nucleic acid operably linked to the carbon source inducible promoter. In other embodiments the bacterial cell is Escherichia coll. In another embodiment, the bacterial cell is selected from the Pseudomonads and closely related bacteria. In other embodiments the bacterial cell is a Pseudomonas. In another embodiment, the bacterial cell is Pseudomonas fluorescens.<br>
The present invention further provides methods for producing recombinant peptides comprising providing a bacterial cell that has been rendered incapable of degrading or metabolizing mannitol, glucitol, or arabitol, or derivative thereof, transforming the cell with at least one nucleic acid construct comprising at least one nucleic acid encoding at least one peptide of interest operably attached to at least one carbon source-inducible promoter, wherein the inducer is selected from the group consisting of mannitol, glucitol, arabitol, glycerol, or derivative or analogue thereof. In certain embodiments, the expression of the peptide of interest is controlled in a linear fashion directly correlated to the inducer concentration. In some embodiments at least one promoter is a mannitol-inducible promoter, wherein the bacterial cell has been rendered incapable of metabolizing or degrading mannitol, and the inducer is mannitol, arabitol, glucitol, glycerol, or derivatives or analogues thereof.<br>
In other embodiments, the present invention provides the expression of a nucleic acid encoding a peptide operably attached to a promoter induced by mannitol, glucitol, arabitol, or glycerol, or analogue thereof, in combination with other inducible promoters in a bacterial cell that has been rendered deficient in its ability to degrade or metabolize mannitol, glucitol, arabitol, or analogues or derivatives thereof. In other embodiments, the additional inducible promoters can be operably linked in tandem to the mannitol, glucitol, or arabitol-inducible<br><br>
promoter, or independent of each other and operably linked to separate nucleic acids encoding peptides of interest. Additional embodiments of the present invention include the use of multiple promoters, including at least one mannitol, glucitol, or arabitol-inducible promoter, wherein the promoters are independent of each other and operably attached to nucleic acids encoding different peptides of interest and differentially induced, either at different times, or at different expression levels controlled by the utilization of inducer concentration gradients.<br>
Emodiments of the present invention also provides methods for producing recombinant peptides in bacterial host cells utilizing a carbon source inducible promoter. Additional embodiments provide alternative promoters for use in large scale bacterial fermentation processes that are induced by low cost carbon source chemicals. Other embodiments of the present invention provide for improved bacterial host cells for the expression of recombinant proteins from promoters induced by a carbon source.<br>
BRIEF DESCRIPTION OF THE FIGURES<br>
Figure 1 illustrates a pathway of mannitol, glucitol, and arabitol degradation, as described by Brunker et al. (1998) "Structure and function of the genes involved in mannitol, arabitol, and glucitol utilization from Pseudomonas fluorescens DSM50106," Gene 206(1): 117-126.<br>
Figure 2 illustrates the structure of the P. fluorescens mannitol (mtl) operon in DSM50106, identifying a putative -10 and -35 sites in the promoter (bold) and a region of inverted homology (arrows), as described by Brunker et al. (1998) "Structure and function of the genes involved in mannitol, arabitol, and glucitol utilization from Pseudomonas fluorescens DSM50106," Gene 206(1): 117-126. The mtlR activator is present at a site distant from the mtl operon.<br><br>
Figure 3 illustrates a comparison of the mannitol operon in MB 101 and other Pseudomonads. The figure shows an alignment of the mtl operon in four different Pseudomonas strains, P. fluorescens MB 101, P. fluorescens DSM50601, P. fluorescens PfO-1, and P. aeruginosa PAO-1.. The mtlR gene, encoding the transcriptional activator of the mtl operon, is located directly upstream of the mtl operon in all cases except in MB 101, where it occurs several kB upstream and DSM50601, where it occurs at an unknown place in the genome.<br>
Figure 4 illustrates a comparison of the genomic region upstream of mtlE in various Pseudomonads. The region upstream of the mtlE gene in MB101, DSM50106, P. fluorescens PfO-1, and P. aeruginosa PAO-1 was aligned. Regions of perfect homology among the four strains are indicated by small case font. The -35 and -10 regions, putative ribosome binding site (RBS) and start codon are underlined. The dotted-line box shows the minimal sequence required for mannitol induction.<br>
Figure 5 illustrate the cloning of mtl promoter fragment in front of the CopGFP reporter gene. A 299 bp region upstream of the MB 101 mtlE gene (SEQ. ID. No. 9) was cloned by PCR amplification with primers mtlE3 (SEQ. ID. No. 14) and mtlE4 (SEQ. ID. No. 15) (see Table 8), to make pDOWl365-1, using Herculase thermostable DNA polymerase (Stratagene) and MB214 (a strain derived from MB 101) genomic DNA as a template.<br>
Figure 6 illustrates the comparison of activity of various fragment from the region upstream of mtlE. Strain DC283AmtlDYZ carrying either pDOWl365-1, pDOW1377, pDOW1369, pDOW1378, or pDOW2212 (with no CopGFP gene) were patched on M9 agar plates containing 1% mannitol with supplemental uracil. Induction of the fluorescent protein was determined by exposure to blue light. The two 15-bp regions of homology between the four strains (see Figure 4) are indicated by hatched boxes. The -35 and -10 regions are indicated, and the primers used to clone each sequence are noted at the edge of the open box which indicates the region cloned upstream of the CopGFP reporter gene.<br>
Figure 7 illustrates the effect of mannitol on expression of CopGFP from Pmtl in a wildtype strain background grown on standard production medium. Strain DC283 pDOWl365-1 was cultivated in shake flask medium containing 9.5% (v/v) glycerol and uracil supplementation. Mannitol 1% (w/v) was added to one set at 24h EFT (filled squares) or not added (open triangles). Fluorescence after induction was monitored over time. Fluorescence was measured on samples that were normalized to ODeoo = 1 •<br>
Figure 8 illustrates the effect of mannitol on expression of CopGFP from Pmtl in a wildtype strain background in medium with various glycerol concentrations. Strain DC283<br><br>
pDOW1365-l was cultivated in shake-flask medium containing 2% (squares), 5% (triangles), or 9.5% (circles) glycerol. Cultures were uninduced (open symbols) or induced (closed symbols) with 1% mannitol at 20h EFT. Fluorescence after induction was monitored over time. Fluorescence was measured on samples that were normalized to ODeoo - 1 •<br>
Figure 9 illustrates a vector map of mtlDYZ deletion vector pDOWl373-3. The vector pDOW1373-3 was designed to delete the mtlDYZ genes by allele exchange mutagenesis. The mtlDYZ flanking regions were amplified using MB214 genomic DNA as a template and joined using the splicing by overlap extension technique, using primers manl (SEQ. ID. No. 18) and man2 (SEQ. ID. No. 19) for the region upstream and man3 (SEQ. ID. No. 20) and man4 (SEQ. ID. No. 21) for the region downstream. Primers man2 and man3 contains complementary overlaps. The joined product was cloned into pDOW\26\ISrfl to make pDOWl373-3. Colonies were screened with primer H3seq (SEQ. ID. No. 22) and man4; those with the expected 1170 bp product were sequenced to ensure that no PCR-derived mutations were incorporated.<br>
Figure 10 illustrates growth and carbon source (mannitol or glucose) utilization of strain DC283AmtlDYZ. Cultures were grown in M9 medium with 1% carbon source, and supplemented with uracil (250 ug/ml) and proline (250 u,g/ml). Glucose (open circle) and mannitol (open triangle) concentrations and optical density at 600 nm (closed triangle for mannitol, closed circle for glucose) after induction were monitored over time.<br>
Figure 11 illustrates the effect of glycerol concentration on expression of CopGFP from Pmtl in the mannitol-degradation-defective strain after induction with 1% mannitol. Strain DC283&amp;mtlDYZ/ pDOW1365-l was cultivated in shake flask medium with an inital concentration of 2% (squares), 5% (triangles), or 9.5% (circles) glycerol with supplemental uracil. Cultures were either induced with 1% mannitol at 22h EFT (closed symbols) or left uninduced (open symbols). Fluorescence after induction was monitored over time. Fluorescence was measured on samples that were normalized to ODeoo - 1 •<br>
Figure 12 illustrates comparison of Pmtl activity in the mtlDYZ deletion mutant and wildtype in various carbon sources. Pmtl activity was compared in the mannitol-degradation-defective strain (D283AmtfDFZ/ pDOWl365-1) (closed symbols) and the wildtype (DC283/ pDOWl365-1) (open symbols) in shake flask medium with initial concentrations of 5% (squares) or 9.5% glycerol (triangles) (a), or 10% glucose (circles) (b). Cultures were induced with 1% mannitol at 24h EFT and fluorescence after induction was monitored over time. Fluorescence was measured on samples that were normalized to ODsoo = 1.<br><br>
Figure 13 illustrates Pmtl activity after induction with various concentrations of mannitol. Various amounts of mannitol (0, 0.01%.0.1%, 0.5%, 1%) were added as an inducer at 25h EFT to D283ΔmtlDYZ/ pDOWl365-1 grown with 9.5% glycerol and supplemental uracil (triangles) or D283kmtlDYZ/ pDOW1365-l pDOW1339 grown with 12.5% glucose (squares). Fluorescence after induction was monitored over time on samples that were normalized to ODeoo - 1 •<br>
Figure 14 illustrates Pmtl activity in glucose-grown cultures after induction with glycerol. The strain DC283 kmtlDYZ/ pDOW1365-l pDOW1339 was cultivated in shake-flask medium containing 12.5% glucose. Various concentrations of glycerol (0, 0.1%, 0.5%, 1%, 2%, 5%, 9.5%) were added at 12 h EFT and fluorescence was measured over time on normalized samples (above). The fluorescence level at 17 h (squares) or 35 h (triangles) were plotted.<br>
Figure 15 illustrates a flow cytometry analysis using varying concentrations of inducer. Fluorescence of the cells from glucose-grown cultures induced with varying levels of mannitol (see Figure 13) (a) or with glycerol (see Figure 14) (b) were analyzed by flow cytometry and the results overlaid in a histogram. The x-axis is fluorescence and the y-axis is cell count at each fluorescence value. For comparison, fluorescence after induction with 2% mannitol is included in (b).<br>
Figure 16 illustrates the induction by mannitol of CopGFP Pmtl in 20-L corn syrup (glucose) or glycerol fermentor cultures. Optical densities (A) and fluorescence (B) analysis of strain DC283 Aw//D7Z/pDOW1365-l pDOW1339 (= DC390) grown in 20L-fermentors with corn syrup (glucose) (diamonds) or glycerol (squares) is shown. Cultures were induced with 1% mannitol at 20.5 h EFT and optical density at 575 nm and fluorescence were measured over time on normalized samples.<br>
Figure 17 illustrates a flow cytometry analysis of the induction by mannitol of expression of CopGFP from Pmtl in 20-L corn syrup (glucose) or glycerol fermentor cultures. Flow cytometry analysis of strain DC283 Δmtl/DYZ/pDOW1365-l pDOW1339 (=DC390) grown in 20L-fermentors with corn syrup (glucose) (above) or glycerol (below). Cultures were induced with 1% mannitol at 20.5 h EFT. Samples were removed for flow cytometry analysis at 0 and 48 hr after induction. The x-axis of the histogram is fluorescence and the y-axis is cell count at each fluorescence value.<br><br>
The present invention provides methods for producing recombinant peptides in a bacterial host utilizing a mannitol, arabitol, glucitol, or glycerol-inducible promoter, wherein the host bacterial cell that produces the peptide has been rendered incapable of degrading or metabolizing mannitol, arabitol, or glucitol, or derivatives or analogues thereof. The present invention provides bacterial cells that have been genetically altered to inhibit the metabolism or degradation of mannitol, glucitol, or arabitol, or derivatives or analogues thereof. In certain embodiments, mannitol, arabitol, glucitol, or glycerol can then be used to induce expression of a target polypeptide from an inducible promoter, allowing for the use of an inexpensive and stable carbon source inducer in the fermentation processes for the production of recombinant peptides. Because the bacterial cell lacks the ability to metabolize or degrade mannitol, glucitol, or arabitol, if these carbon sources are used as an inducer, then the inducer is not continuously removed from the media by the host cell, and therefore is capable of providing continuous and stable induction levels during fermentation without the requirement of adding additional inducer to the media. In some embodiments the bacterial cell is rendered incapable of degrading or metabolizing mannitol, glucitol, or arabitol, or derivatives or analogues thereof, and the promoter operably linked to a peptide of interest is capable of induction by glycerol, as well as mannitol, glucitol, or arabitol, or derivatives or analogues thereof.<br>
I. MANNITOL INDUCIBLE PROMOTERS<br>
The promoters for use in the present invention are nucleic acid sequences typically of greater than 25 nucleic acids located upstream of the nucleic acid sequence encoding a peptide of interest. The promoters generally contain a -35 region, which is generally a 5-6 nucleic acid sequence beginning approximately 35 base pairs upstream of the transcription start site for a peptide or interest. The transcription start site of the peptide of interest is generally numbered as +1.<br>
In the present invention, an inducible promoter is utilized that is capable of induction by a carbon source. In some embodiments the promoter is capable of induction by a carbon source selected from the group consisting of mannitol, arabitol, glucitol, and glycerol, or derivatives or analogues thereof. In other embodiments the promoter is a mannitol, or mannitol derivative, inducible promoter. In additional embodiments the promoter is the putative promoter of an endogenous bacterial mannitol operon. In some embodiments the promoter is the putative promoter, and related regulatory regions, of the Pseudomonas fluorescens mtlEFGKDYZ operon. In other embodiments of the present invention the<br><br>
mannitol-inducible promoter is capable of being induced by a carbon source other than mannitol. Such carbon sources include, but are not limited to glucitol, arabitol, and glycerol. In some embodiments the carbon source is glycerol.<br>
In some embodiments the mannitol inducible promoter of the present invention comprises the -35 region of the Pseudomonad native mtl operon promoter attached upstream of the -10 region of the native promoter, via a 15-20 nucleotide linker. In other embodiments the linker is 17-18 nucleotides long. In another embodiment, the -35 region is comprised of the nucleic acid sequence 5'-TTGTCA-3'. In yet another embodiment, the -10 region is comprised of the nucleic acid sequence 5'-TGTAAT-3'. In still another embodiment, the promoter is comprised of the -35 region 5'-TTTGTC-3', linked to a nucleotide sequence of between 15-20 nucleotides, which is further linked to the -10 region 5'-TGTAAT-3'.<br>
In alternative embodiments the mannitol-inducible promoter includes the nucleotide sequence comprising 5'-TTGTCACAACCCCGTTTGAAGGCTGTAAT-3' (SEQ. ID. No. 1) (Table 1). In another embodiment, the promoter includes the nucleotide sequence comprising 5'-TTGTCACCGCCGTTTTTGAAGGCTGTAAT-3' (SEQ. ID. No. 2) (Table 1). In still another embodiment, the promoter includes the nucleotide sequence comprising 5'-TTGTCAGCCCTGCGTCAGAAGGCTGTAAT-3' (SEQ. ID. NO. 3) (Table 1). In still another embodiment, the promoter includes the nucleic acid sequence 5'-TTGTCGGTTGCGTGACGCGCCTGTGTAA-3' (SEQ. ID. No. 4) (Table 1).<br>
In some embodiments the mannitol-inducible promoter of the present invention further includes nucleic acid sequences that act as an activator or represser peptide binding site. In some embodiments the nucleic acid sequences act as an activator peptide binding site. In other embodiments the activator site is a nucleic acid sequence from an endogenous Pseudomonad MtlR protein binding site. In other embodiments the activator site is a nucleic acid sequence from an endogenous Pseudomonas fluorescens MtlR protein binding site.<br>
In some embodiments the activator binding site includes the nucleic acid sequence 5'-ACGAGTGCAAAAAAGTATCAGTAAGCGTGCTCCCAAGGAT-3' (SEQ. ID. No. 5) (Table 2). In another embodiment, the activator binding site includes the nucleic acid sequence 5'-ACGAGTGCAAAAAAGTATCAGTCCAAGTGCTCCCAAGGAT -3' (SEQ. ID. No. 6) (Table 2). Alternatively, the activator binding site includes the nucleic acid sequence 5'-CCGAGTGCAAAAAAGTATCGATTCAAGTGCTAGGGATGAT-3' (SEQ. ID. No. 7) (Table 2). In another embodiment, the activator binding site includes 5'-GGCGGTGCAAAAAAGTATCGGTCGAAGTGCAGTCGAGGCT-3' (SEQ. ID. No. 8) (Table 2).<br><br>
In additional embodiments the mannitol inducible promoter of the present invention includes SEQ. ID. No. 9, a 299 base pair region of an endogenous Pseudomonad mtl operon (Table 3). In another embodiment, the mannitol promoter of the present invention includes SEQ. ID No. 10, a 203 base pair region of an endogenous Pseudomonad mtl operon (Table 3). In yet another embodiment, the promoter of the present invention includes SEQ. ID. No. 11, a 126 base pair region from an endogenous Pseudomonad mtl operon (Table 3). In another embodiment, the promoter of the present invention includes SEQ. ID. No. 12, a 147 base pair region from an endogenous Pseudomonad mtl operon (Table 3). In additional embodiments the promoter of the present invention includes SEQ. ID. No. 13, a 77 base pair region from an endogenous Pseudomonad mtl operon (Table 3)<br>
Table 1. Mannitol Inducible Promoter -35 to -10 Nucleic Acid Regions<br>
(Table Removed)<br><br>
Table 2: Mannitol Inducible Promoter Activator Nucleic Acid Regions<br>
(Table Removed)<br><br><br>
Table 3: Mannitol Inducible Promoters mtl Operon Nucleic Acid Regions<br><br>
(Table Removed)<br><br><br>
The sequences recited in this application may be homologous (have similar identity). Proteins and/or protein sequences are "homologous" when they are derived, naturally or artificially, from a common ancestral protein or protein sequence. Similarly, nucleic acids and/or nucleic acid sequences are homologous when they are derived, naturally or artificially, from a common ancestral nucleic acid or nucleic acid sequence. For example, any naturally occurring nucleic acid can be modified by any available mutagenesis method to include one or more selector codon. When expressed, this mutagenized nucleic acid encodes a polypeptide comprising one or more unnatural amino acid. The mutation process can, of course, additionally alter one or more standard codon, thereby changing one or more standard amino acid in the resulting mutant protein as well. Homology is generally inferred from sequence similarity between two or more nucleic acids or proteins (or sequences thereof). The precise percentage of similarity between sequences that is useful in establishing homology varies with the nucleic acid and protein at issue, but as little as 25% sequence similarity is routinely used to establish homology. Higher levels of sequence similarity, e.g., 30%, 40%, 50%, 60%, 70%, 80%, 90%, 95%, 96%, 97%, 98% or 99% or more can also be used to establish homology. Methods for determining sequence similarity percentages (e.g., BLASTP and BLASTN using default parameters) are described herein and are generally available.<br>
As noted above, polypeptides may comprise a signal (or leader) sequence at the N-terminal end of the protein, which co-translationally or post-translationally directs transfer of the protein. The polypeptide may also be conjugated to a linker or other sequence for ease of synthesis, purification or identification of the polypeptide (e.g., poly-His), or to enhance binding of the polypeptide to a solid support.<br>
When comparing polypeptide sequences, two sequences are said to be "identical" if the sequence of amino acids in the two sequences is the same when aligned for maximum correspondence, as described below. Comparisons between two sequences are typically<br>
performed by comparing the sequences over a comparison window to identify and compare local regions of sequence similarity. A "comparison window" as used herein, refers to a segment of at least about 20 contiguous positions, usually 30 to about 75, 40 to about 50, in which a sequence may be compared to a reference sequence of the same number of contiguous positions after the two sequences are optimally aligned.<br>
Optimal alignment of sequences for comparison may be conducted using the Megalign program in the Lasergene suite of bioinformatics software (DNASTAR, Inc., Madison, Wis.), using default parameters. This program embodies several alignment schemes described in the following references: Dayhoff, M. O. (1978) A model of evolutionary change in proteins -Matrices for detecting distant relationships. In Dayhoff, M. O. (ed.) Atlas of Protein Sequence and Structure, National Biomedical Research Foundation, Washington D.C. Vol. 5, Suppl. 3, pp. 345 358; Hein J. (1990) Unified Approach to Alignment and Phylogenes pp. 626 645 Methods in En/ymology vol. 183, Academic Press, Inc., San Diego, Calif; Higgins, D. G. and Sharp, P. M. (1989) CABIOS 5:151 153; Myers, E. W. and Muller W. (1988) CABIOS 4:11 17; Robinson, E. D. (1971) Comb. Theor 11:105; Santou, N. Nes, M. (1987) Mol. Biol. Evol. 4:406 425; Sneath, P. H. A. and Sokal, R. R. (1973) Numerical Taxonomy—the Principles and Practice of Numerical Taxonomy, Freeman Press, San Francisco, Calif.; Wilbur, W. J. and Lipman, D. J. (1983) Proc. Natl. Acad., Scl USA 80:726 730.<br>
Alternatively, optimal alignment of sequences for comparison may be conducted by the local identity algorithm of Smith and Waterman (1981) Add. APL. Math 2:482, by the identity alignment algorithm of Needleman and Wunsch (1970) J. Mol. Biol. 48:443, by the search for similarity methods of Pearson and Lipman (1988) Proc. Natl. Acad. Sci. USA 85: 2444, by computerized implementations of these algorithms (GAP, BESTFIT, BLAST, FASTA, and TFASTA in the Wisconsin Genetics Software Package, Genetics Computer Group (GCG), 575 Science Dr., Madison, Wis.), or by inspection.<br>
One example of algorithms that can be suitable for determining percent sequence identity and sequence similarity are the BLAST and BLAST 2.0 algorithms, which are described in Altschul et al. (1977) Nucl. Acids Res. 25:3389 3402 and Altschul et al. (1990) J. Mol. Biol. 215:403 410, respectively. BLAST and BLAST 2.0 can be used, for example with the parameters described herein, to determine percent sequence identity for the polynucleotides and polypeptides of the invention. Software for performing BLAST analyses is publicly available through the National Center for Biotechnology Information. For amino acid sequences, a scoring matrix can be used to calculate the cumulative score. Extension of the word hits in each direction are halted when: the cumulative alignment score falls off by the quantity X from its<br><br>
maximum achieved value; the cumulative score goes to zero or below, due to the accumulation of one or more negative-scoring residue alignments; or the end of either sequence is reached. The BLAST algorithm parameters W, T and X determine the sensitivity and speed of the alignment.<br>
In one approach, the "percentage of sequence identity" is determined by comparing two optimally aligned sequences over a window of comparison of at least 20 positions, wherein the portion of the polypeptide sequence in the comparison window may comprise additions or deletions (i.e., gaps) of 20 percent or less, usually 5 to 15 percent, or 10 to 12 percent, as compared to the reference sequences (which does not comprise additions or deletions) for optimal alignment of the two sequences. The percentage is calculated by determining the number of positions at which the identical amino acid residue occurs in both sequences to yield the number of matched positions, dividing the number of matched positions by the total number of positions in the reference sequence (i.e., the window size) and multiplying the results by 100 to yield the percentage of sequence identity.<br>
II.      HOST CELLS<br>
The present invention also provides bacterial cells that have been genetically modified to reduce their ability to metabolize or degrade a carbon source selected from the group consisting of mannitol, glucitol, arabitol, and derivative or analogue thereof, which carbon source can be utilized to induce expression from an inducible promoter. The genetic modification can be to a gene or genes encoding an enzyme that is operative in a metabolic pathway involved in metabolizing or degrading a carbon source selected from the group consisting of mannitol, glucitol, arabitol, and derivatives or analogues thereof utilized to induce expression from the inducible promoter. Preferably, the host cell has genetic modifications to genes encoding enzymatic activity related to the metabolization or degradation of the inducer, while genes related to the transport of the inducer are unaffected, allowing for transport into the cell without subsequent degradation or metabolization. In some embodiments the bacterial cell is an E. coli cell. In some embodiments the bacterial cell is selected from Pseudomonads and closely related cells. In some embodiments the bacterial cell is a Pseudomonasfluorescens cell.<br>
A bacterial host cell selected for use in the present invention can be rendered deficient in its ability to metabolize or degrade any inducer utilized to induce expression of a peptide of interest from an inducible promoter. For example, where mannitol is selected as the inducer, the host cell can be rendered deficient in it ability to express an enzyme required for metabolizing mannitol, or any effective replacement enzyme capable of metabolizing<br><br>
mannitol. In some embodiments the host cell will be made deficient for the metabolization or degradation of the inducer by altering its genome so that the cell cannot express, from its genome, a functional enzyme involved in the metabolization or degradation of the inducer. This alteration can be done by altering the cell's genomic coding sequence of the gene encoding the metabolizing or degrading enzyme or enzymes. In other embodiments the coding sequence alteration will be accomplished by introducing: insertion or deletion mutations that change the coding sequence reading frame; substitutions or inversion mutations that alter a sufficient number of codons; and/or deletion mutations that delete a sufficiently large group of contiguous codons therefrom capable of producing a nonfunctional enzyme.<br>
Such knock-out strains can be prepared according to any of the various methods known in the art as effective. For example, homologous recombination vectors containing homologous targeted gene sequences 5' and 3' of the desired nucleic acid deletion sequence can be transformed into the host cell. Ideally, upon homologous recombination, a desired targeted enzymatic gene knock-out can be produced.<br>
Specific examples of gene knock-out methodologies are well known in the art. For example, gene inactivation by insertion of a polynucleotide has been previously described. See, e.g., DL Roeder &amp; A Collmer, Marker-exchange mutagenesis ofapectate lyase isozyme gene in Erwinia chrysanthemi, J Bacteriol. 164(l):51-56 (1985). Alternatively, transposon mutagenesis and selection for desired phenotype (such as the inability to metabolize a particular carbon source) can be used to isolate bacterial strains in which target genes have been insertionally inactivated. See, e.g., K Nida &amp; PP deary, Insertional inactivation of streptolysin S expression in Streptococcus pyogenes, J Bacteriol. 155(3): 1156-61 (1983). Specific mutations or deletions in a gene can be constructed using cassette mutagenesis, for example, as described in JA Wells et al., Cassette mutagenesis: an efficient method for generation of multiple mutations at defined sites, Gene 34(2-3):315-23 (1985); whereby direct or random mutations are made in a selected portion of a gene, and then incorporated into the chromosomal copy of the gene by homologous recombination.<br>
In additional embodiments of the present invention the bacterial cell is rendered deficient in an enzyme necessary to metabolize the carbon source used as an inducer for recombinant peptide expression. In some embodiments the bacterial cell is rendered deficient in an enzyme necessary to metabolize the carbon source mannitol, glucitol, arabitol, or derivatives thereof. In some embodiments the bacterial cell is constructed to lack enzyme activity encoded by a gene from an endogenous mannitol operon - mtl. In another<br><br>
embodiment, the bacterial cell is constructed to lack enzyme activity encoded by the endogenous mtlD gene, or homologue thereof. In an alternative embodiment, the bacterial cell is constructed to lack enzyme activity encoded by an endogenous mtlY gene, or homologue thereof. In still another embodiment, the bacterial cell is constructed to lack enzyme activity encoded by an endogenous mtlZ gene, or homologue thereof. Alternatively, the bacterial cell can be constructed to lack activity of any combination of enzymes encoded by the mtlDYZ genes, or homologues thereof. In additional embodiments the bacterial cell is constructed to lack enzyme activity encoded by an endogenous mtlD, mtlY, and mtlZ genes.<br>
In an alternative embodiment, the bacterial cell can be a natural mutant, wherein the mutant is incapable of degrading or metabolizing a carbon source used as an inducer for recombinant peptide expression due to a naturally arising mutation in an endogenous gene necessary for such carbon source degradation of metabolization.<br>
In addition, the present invention further provides bacterial cells comprising a nucleic acid construct including a nucleic acid sequence encoding for at least one peptide of interest operably linked to at least one carbon source-inducible promoter, wherein the carbon source inducible promoter is a mannitol, arabitol, or glucitol-inducible promoter. In some embodiments the carbon source-inducible promoter is a mannitol-inducible promoter. In other embodiments the mannitol-inducible promoter is capable of being induced by glycerol.<br>
Alternatively, the bacterial cell comprises a first nucleic acid construct including a nucleic acid sequence encoding for at least one peptide of interest operably linked to at least one carbon source-inducible promoter, and a second nucleic acid construct including a nucleic acid sequence encoding at least one peptide of interest operably linked to at least one inducible promoter. In some embodiments the carbon-source inducible promoter is a mannitol, glucitol, or arabitol-inducible promoter. The multiple promoters can be located on the same nucleic acid constructs, or on separate nucleic acid constructs. During the fermentation process, expression of the peptides of interest can be induced in a differential manner, including at different times or at different expression levels. In certain embodiments, the inducible promoters (and related inducers) used in combination with the mannitol-, arabitol, or glucitol-inducible promoters of the present invention can include, for example, lac (IPTG), lacUVS (IPTG), tac (IPTG), trc (IPTG), Psyn (IPTG), trp (tryptophan starvation), araBAD (1-arabinose), lppa (IPTG), Ipp-lac (IPTG), phoA (phosphate starvation), recA (nalidixic acid), proU (osmolarity), cst-1 (glucose starvation), tetA (tretracylin), cadA (pH), nar (anaerobic conditions), PL (thermal shift to 42° C), cspA (thermal shift to 20° C), T7 (thermal induction), T7-lac operator (IPTG), T3-lac operator (IPTG), T5-lac operator (IPTG),<br><br>
T4 gene32 (T4 infection), nprM-lac operator (IPTG), Pm (alkyl- or halo-benzoates), Pu (alkyl- or halo-toluenes), Psal (salicylates), ant (anthranilate), ben (benzoate) or VHb (oxygen) promoter. See, for example, Makrides, S.C. (1996) Microbiol. Rev. 60, 512-538; Hannig G. &amp; Makrides, S.C. (1998) TIBTECH 16, 54-60; Stevens, R.C. (2000) Structures 8, R177-R185. See, e.g.: J. Sanchez-Romero &amp; V. De Lorenzo, Genetic Engineering of Nonpathogenic Pseudomonas strains as Biocatalysts for Industrial and Environmental Processes, in Manual of Industrial Microbiology and Biotechnology (A. Demain &amp; J. Davies, eds.) pp.460-74 (1999) (ASM Press, Washington, D.C.); H. Schweizer, Vectors to express foreign genes and techniques to monitor gene expression for Pseudomonads, Current Opinion in Biotechnology, 12:439-445 (2001); and R. Slater &amp; R. Williams, The Expression of Foreign DNA in Bacteria, in Molecular Biology and Biotechnology (J. Walker &amp; R. Rapley, eds.) pp. 125-54 (2000) (The Royal Society of Chemistry, Cambridge, UK).<br>
The nucleic acid constructs contained in the bacterial cell may be maintained in an episomal fashion, or be integrated into the genome of the cell.<br>
Bacterial Organisms<br>
The present invention provides bacterial cells that are deficient in an enzyme necessary for metabolism of a carbon source selected from the group consisting of mannitol, glucitol, and arabitol, or derivative or analogue thereof, which carbon source is utilized to induce expression of a peptide of interest, wherein the nucleic acid sequence encoding the peptide of interest is operably attached to a promoter induced by a carbon source, the carbon source selected from the group consisting of mannitol, glucitol, arabitol, and derivative or analogue thereof. The present invention contemplates utilizing any bacterial cell capable of expressing a recombinant peptide. Such cells are well known in the art. In some embodiments the host cell can be a Escherichia coli cell. In another embodiment, the cell is selected from a Pseudomonad or closely related cell. In other embodiments the bacterial cell is a Pseudomonas fluorescens. In some embodiments the bacterial cell can be deficient in an enzyme necessary for the metabolism of a carbon source selected from the group consisting of mannitol, glucitol, or arabitol, or derivatives and analogues thereof.<br>
Pseudomonads and closely related bacteria, as used herein, is co-extensive with the group defined herein as "Gram(-) Proteobacteria Subgroup 1." " Gram(-) Proteobacteria Subgroup 1" is more specifically defined as the group of Proteobacteria belonging to the families and/or genera described as falling within that taxonomic "Part" named "Gram-Negative Aerobic Rods and Cocci" by R.E. Buchanan and N.E. Gibbons (eds.), Sergey's<br><br>
Manual of Determinative Bacteriology, pp. 217-289 (8th ed., 1974) (The Williams &amp; Wilkins Co., Baltimore, MD, USA) (hereinafter "Bergey (1974)"). Table 4 presents the families and genera of organisms listed in this taxonomic "Part."<br>
TABLE 4. FAMILIES AND GENERA LISTED IN THE PART, "GRAM-NEGATIVE AEROBIC RODS AND Cocci" (IN BERGEY (1974))<br>
(Table Removed)<br><br><br>
"Gram(-) Proteobacteria Subgroup 1" contains all Proteobacteria classified there under, as well as all Proteobacteria that would be classified according to the criteria used in forming that taxonomic "Part." As a result, "Gram(-) Proteobacteria Subgroup 1" excludes, e.g.: all Gram-positive bacteria; those Gram-negative bacteria, such as the Enterobacteriaceae, which fall under others of the 19 "Parts" of this Bergey (1974) taxonomy; the entire "Family V. Halobacteriaceae" of this Bergey (1974) "Part," which family has since been recognized as being a non-bacterial family of Archaea; and the genus, Thermus, listedwithin this Bergey (1974) "Part," which genus which has since been recognized as being a non-Proteobacterial genus of bacteria.<br>
"Gram(-) Proteobacteria Subgroup 1" further includes those Proteobacteria belonging to (and previously called species of) the genera and families defined in this Bergey (1974) "Part," and which have since been given other Proteobacterial taxonomic names. In some cases, these re-namings resulted in the creation of entirely new Proteobacterial genera. For example, the genera Acidovorax, Brevundimonas, Burkholderia, Hydrogenophaga, Oceanimonas, Ralstonia, and Stenotrophomonas, were created by regrouping organisms belonging to (and previously called species of) the genus Pseudomonas as defined in Bergey (1974). Likewise, e.g., the genus Sphingomonas (and the genus Blastomonas, derived therefrom) was created by regrouping organisms belonging to (and previously called species of) the genus Xanthomonas as defined in Bergey (1974). Similarly, e.g., the genus Acidomonas was created by regrouping organisms belonging to (and previously called species of) the genus Acetobacter as defined in Bergey (1974). Such subsequently reassigned species are also included within "Gram(-) Proteobacteria Subgroup 1" as defined herein.<br>
In other cases, Proteobacterial species falling within the genera and families defined in this Bergey (1974) "Part" were simply reclassified under other, existing genera of Proteobacteria. For example, in the case of the genus Pseudomonas, Pseudomonas enalia (ATCC 14393), Pseudomonas nigrifaciens (ATCC 19375), and Pseudomonas putrefaciens (ATCC 8071) have since been reclassified respectively as Alteromonas haloplanktis, Alteromonas nigrifaciens, and Alteromonas putrefaciens. Similarly, e.g., Pseudomonas acidovorans (ATCC 15668) and Pseudomonas testosteroni (ATCC 11996) have since been reclassified as Comamonas acidovorans and Comamonas testosteroni, respectively; and Pseudomonas nigrifaciens (ATCC 19375) and Pseudomonas piscicida (ATCC 15057) have since been reclassified respectively as Pseudoalteromonas nigrifaciens and Pseudoalteromonas piscicida. Such subsequently reassigned Proteobacterial species are also included within "Gram(-) Proteobacteria Subgroup 1" as defined herein.<br>
"Gram(-) Proteobacteria Subgroup 1" also includes Proteobacterial species that have since been discovered, or that have since been reclassified as belonging, within the Proteobacterial families and/or genera of this Bergey (1974) "Part." In regard to Proteobacterial families, "Gram(-) Proteobacteria Subgroup 1" also includes Proteobacteria classified as belonging to any of the families: Pseudomonadaceae, Azotobacteraceae (now often called by the synonym, the "Azotobacter group" of Pseudomonadaceae), Rhizobiaceae, and Methylomonadaceae (now often called by the synonym, "Methylococcaceae").<br>
Consequently, in addition to those genera otherwise described herein, further Proteobacterial genera falling within "Gram(-) Proteobacteria Subgroup 1" include: 1) Azotobacter group bacteria of the genus Azorhizophilus; 2) Pseudomonadaceae family bacteria of the genera Cellvibrio, Oligella, and Teredinibacter; 3) Rhizobiaceae family bacteria of the genera Chelatobacter, Ensifer, Liberibacter (also called "Candidates Liberibacter"}, and Sinorhizobium; and 4) Methylococcaceae family bacteria of the genera Methylobacter, Methylocaldum, Methylomicrobium, Methylosarcina, and Methylosphaera.<br>
Embodiments of the present invention include wherein the host cell is selected from "Gram(-) Proteobacteria Subgroup 1," as defined above.<br>
In another embodiment, the host cell is selected from "Gram(-) Proteobacteria Subgroup 2." "Gram(-) Proteobacteria Subgroup 2" is defined as the group of Proteobacteria of the following genera (with the total numbers of catalog-listed, publicly-available, deposited strains thereof indicated in parenthesis, all deposited at ATCC, except as otherwise indicated): Acidomonas (2); Acetobacter (93); Gluconobacter (37); Brevundimonas (23); Beijerinckia (13); Derxia (2); Brucella (4); Agrobacterium (79); Chelatobacter (2); Ensifer (3); Rhizobium (144); Sinorhizobium (24); Blastomonas (1); Sphingomonas (27); Alcaligenes (88); Bordetella (43); Burkholderia (73); Ralstonia (33); Acidovorax (20); Hydrogenophaga (9); Zoogloea (9); Methylobacter (2); Methylocaldum (1 at NCIMB); Methylococcus (2); Methylomicrobium (2); Methylomonas (9); Methylosarcina (1); Methylosphaera; Azomonas (9); Azorhizophilus (5); Azotobacter (64); Cellvibrio (3); Oligella (5); Pseudomonas (1139); Francisella (4); Xanthomonas (229); Stenotrophomonas (50); and Oceanimonas (4).<br>
Exemplary host cell species of "Gram(-) Proteobacteria Subgroup 2" include, but are not limited to the following bacteria (with the ATCC or other deposit numbers of exemplary strain(s) thereof shown in parenthesis): Acidomonas methanolica (ATCC 43581); Acetobacter aceti (ATCC 15973); Gluconobacter oxydans (ATCC 19357); Brevundimonas diminuta (ATCC 11568); Beijerinckia indica (ATCC 9039 and ATCC 19361); Derxia gummosa (ATCC 15994); Brucella melitensis (ATCC 23456), Brucella abortus (ATCC 23448); Agrobacterium tumefaciens (ATCC 23308), Agrobacterium radiobacter (ATCC 19358), Agrobacterium rhizogenes (ATCC 11325); Chelatobacter heintzii (ATCC 29600); Ensifer adhaerens (ATCC 33212); Rhizobium leguminosarum (ATCC 10004); Sinorhizobium fredii (ATCC 35423); Blastomonas natatoria (ATCC 35951); Sphingomonas paucimobilis (ATCC 29837); Alcaligenes faecalis (ATCC 8750); Bordetella pertussis (ATCC 9797); Burkholderia cepacia (ATCC 25416); Ralstonia pickettii (ATCC 27511); Acidovorax facilis (ATCC 11228); Hydrogenophaga flava (ATCC 33667); Zoogloea ramigera (ATCC 19544);<br>
Methylobacter luteus (ATCC 49878); Methylocaldum gracile (NCIMB 11912); Methylococcus capsulatus (ATCC 19069); Methylomicrobium agile (ATCC 35068); Methylomonas methanica (ATCC 35067); Methylosarcina flbrata (ATCC 700909); Methylosphaera hansonii (ACAM 549); Azomonas agilis (ATCC 7494); Azorhizophilus paspali (ATCC 23833); Azotobacter chroococcum (ATCC 9043); Cellvibrio mixtus (UQM 2601); Oligella urethralis (ATCC 17960); Pseudomonas aeruginosa (ATCC 10145), Pseudomonas fluorescens (ATCC 35858); Francisella tularensis (ATCC 6223); Stenotrophomonas maltophilia (ATCC 13637); Xanthomonas campestris (ATCC 33913); and Oceanimonas doudoroffii (ATCC 27123).<br>
In another embodiment, the host cell is selected from "Gram(-) Proteobacteria Subgroup 3." "Gram(-) Proteobacteria Subgroup 3" is defined as the group of Proteobacteria of the following genera: Brevundimonas; Agrobacterium; Rhizobium; Sinorhizobium; Blastomonas; Sphingomonas; Alcaligenes; Burkholderia; Ralstonia; Acidovorax; Hydrogenophaga; Methylobacter; Methylocaldum; Methylococcus; Methylomicrobium; Methylomonas; Methylosarcina; Methylosphaera; Azomonas; Azorhizophilus; Azotobacter; Cellvibrio; Oligella; Pseudomonas; Teredinibacter; Francisella; Stenotrophomonas; Xanthomonas; and Oceanimonas.<br>
In another embodiment, the host cell is selected from "Gram(-) Proteobacteria Subgroup 4." "Gram(-) Proteobacteria Subgroup 4" is defined as the group of Proteobacteria of the following genera: Brevundimonas; Blastomonas; Sphingomonas; Burkholderia; Ralstonia; Acidovorax; Hydrogenophaga; Methylobacter; Methylocaldum; Methylococcus; Methylomicrobium; Methylomonas; Methylosarcina; Methylosphaera; Azomonas; Azorhizophilus; Azotobacter; Cellvibrio; Oligella; Pseudomonas; Teredinibacter; Francisella; Stenotrophomonas; Xanthomonas; and Oceanimonas.<br>
In an embodiment, the host cell is selected from "Gram(-) Proteobacteria Subgroup 5." "Gram(-) Proteobacteria Subgroup 5" is defined as the group of Proteobacteria of the following genera: Methylobacter; Methylocaldum; Methylococcus; Methylomicrobium; Methylomonas; Methylosarcina; Methylosphaera; Azomonas; Azorhizophilus; Azotobacter; Cellvibrio; Oligella; Pseudomonas; Teredinibacter; Francisella; Stenotrophomonas; Xanthomonas; and Oceanimonas.<br>
The host cell can be selected from "Gram(-) Proteobacteria Subgroup 6." "Gram(-) Proteobacteria Subgroup 6" is defined as the group of Proteobacteria of the following genera: Brevundimonas; Blastomonas; Sphingomonas; Burkholderia; Ralstonia; Acidovorax;<br>
Hydrogenophaga; Azomonas; Azorhizophilus; Azotobacter; Cellvibrio; Oligella; Pseudomonas; Teredinibacter; Stenotrophomonas; Xanthomonas; and Oceanimonas.<br>
The host cell can be selected from "Gram(-) Proteobacteria Subgroup 7." "Gram(-) Proteobacteria Subgroup 7" is defined as the group of Proteobacteria of the following genera: Azomonas; Azorhizophilus; Azotobacter; Cellvibrio; Oligella; Pseudomonas; Teredinibacter; Stenotrophomonas; Xanthomonas; and Oceanimonas.<br>
The host cell can be selected from "Gram(-) Proteobacteria Subgroup 8." "Gram(-) Proteobacteria Subgroup 8" is defined as the group of Proteobacteria of the following genera: Brevundimonas; Blastomonas; Sphingomonas; Burkholderia; Ralstonia; Acidovorax; Hydrogenophaga; Pseudomonas; Stenotrophomonas; Xanthomonas; and Oceanimonas.<br>
The host cell can be selected from "Gram(-) Proteobacteria Subgroup 9." "Gram(-) Proteobacteria Subgroup 9" is defined as the group of Proteobacteria of the following genera: Brevundimonas; Burkholderia; Ralstonia; Acidovorax; Hydrogenophaga; Pseudomonas; Stenotrophomonas; and Oceanimonas.<br>
The host cell can be selected from "Gram(-) Proteobacteria Subgroup 10." "Gram(-) Proteobacteria Subgroup 10" is defined as the group of Proteobacteria of the following genera: Burkholderia; Ralstonia; Pseudomonas; Stenotrophomonas; and Xanthomonas.<br>
The host cell can be selected from "Gram(-) Proteobacteria Subgroup 11." "Gram(-) Proteobacteria Subgroup 11" is defined as the group of Proteobacteria of the genera: Pseudomonas; Stenotrophomonas; and Xanthomonas.<br>
The host cell can be selected from "Gram(-) Proteobacteria Subgroup 12." "Gram(-) Proteobacteria Subgroup 12" is defined as the group of Proteobacteria of the following genera: Burkholderia; Ralstonia; Pseudomonas.<br>
The host cell can be selected from "Gram(-) Proteobacteria Subgroup 13." "Gram(-) Proteobacteria Subgroup 13" is defined as the group of Proteobacteria of the following genera: Burkholderia; Ralstonia; Pseudomonas; and Xanthomonas.<br>
The host cell can be selected from "Gram(-) Proteobacteria Subgroup 14." "Gram(-) Proteobacteria Subgroup 14" is defined as the group of Proteobacteria of the following genera: Pseudomonas and Xanthomonas..<br>
The host cell can be selected from "Gram(-) Proteobacteria Subgroup 15." "Gram(-) Proteobacteria Subgroup 15" is defined as the group of Proteobacteria of the genus Pseudomonas.<br>
The host cell can be selected from "Gram(-) Proteobacteria Subgroup 16." "Gram(-) Proteobacteria Subgroup 16" is defined as the group of Proteobacteria of the following<br>
Pseudomonas species (with the ATCC or other deposit numbers of exemplary strain(s) shown in parenthesis): Pseudomonas abietaniphila (ATCC 700689); Pseudomonas aeruginosa (ATCC 10145); Pseudomonas alcaligenes (ATCC 14909); Pseudomonas anguilliseptica (ATCC 33660); Pseudomonas citronellolis (ATCC 13674); Pseudomonas flavescens (ATCC 51555); Pseudomonas mendocina (ATCC 25411); Pseudomonas nitroreducens (ATCC 33634); Pseudomonas oleovorans (ATCC 8062); Pseudomonas pseudoalcaligenes (ATCC 17440); Pseudomonas resinovorans (ATCC 14235); Pseudomonas straminea (ATCC 33636); Pseudomonas agarici (ATCC 25941); Pseudomonas alcaliphila; Pseudomonas alginovora; Pseudomonas andersonii; Pseudomonas asplenii (ATCC 23835); Pseudomonas azelaica (ATCC 27162); Pseudomonas beijerinckii (ATCC 19372); Pseudomonas borealis; Pseudomonas boreopolis (ATCC 33662); Pseudomonas brassicacearum; Pseudomonas butanovora (ATCC 43655); Pseudomonas cellulosa (ATCC 55703); Pseudomonas aurantiaca (ATCC 33663); Pseudomonas chlororaphis (ATCC 9446, ATCC 13985, ATCC 17418, ATCC 17461); Pseudomonas fragi (ATCC 4973); Pseudomonas lundensis (ATCC 49968); Pseudomonas taetrolens (ATCC 4683); Pseudomonas cissicola (ATCC 33616); Pseudomonas coronafaciens; Pseudomonas diterpeniphila; Pseudomonas elongata (ATCC 10144); Pseudomonas flectens (ATCC 12775); Pseudomonas azotoformans; Pseudomonas brenneri; Pseudomonas cedrella; Pseudomonas corrugata (ATCC 29736); Pseudomonas extremorientalis; Pseudomonas fluorescens (ATCC 35858); Pseudomonas gessardii; Pseudomonas libanensis; Pseudomonas mandelii (ATCC 700871); Pseudomonas marginalis (ATCC 10844); Pseudomonas migulae; Pseudomonas mucidolens (ATCC 4685); Pseudomonas orientalis; Pseudomonas rhodesiae; Pseudomonas synxantha (ATCC 9890); Pseudomonas tolaasii (ATCC 33618); Pseudomonas veronii (ATCC 700474); Pseudomonas frederiksbergensis; Pseudomonas geniculata (ATCC 19374); Pseudomonas gingeri; Pseudomonas graminis; Pseudomonas grimontii; Pseudomonas halodenitrificans; Pseudomonas halophila; Pseudomonas hibiscicola (ATCC 19867); Pseudomonas huttiensis (ATCC 14670); Pseudomonas hydrogenovora; Pseudomonas jessenii (ATCC 700870); Pseudomonas kilonensis; Pseudomonas lanceolata (ATCC 14669); Pseudomonas lini; Pseudomonas marginata (ATCC 25417); Pseudomonas mephitica (ATCC 33665); Pseudomonas denitriflcans (ATCC 19244); Pseudomonas pertucinogena (ATCC 190); Pseudomonas pictorum (ATCC 23328); Pseudomonas psychrophila; Pseudomonas fulva (ATCC 31418); Pseudomonas monteilii (ATCC 700476); Pseudomonas mosselii; Pseudomonas oryzihabitans (ATCC 43272); Pseudomonas plecoglossicida (ATCC 700383); Pseudomonas putida (ATCC 12633); Pseudomonas reactans; Pseudomonas spinosa (ATCC<br>
14606); Pseudomonas balearica; Pseudomonas luteola (ATCC 43273); Pseudomonas stutzeri (ATCC 17588); Pseudomonas amygdali (ATCC 33614); Pseudomonas avellanae (ATCC 700331); Pseudomonas caricapapayae (ATCC 33615); Pseudomonas cichorii (ATCC 10857); Pseudomonas flcuserectae (ATCC 35104); Pseudomonas fuscovaginae; Pseudomonas meliae (ATCC 33050); Pseudomonas syringae (ATCC 19310); Pseudomonas viridiflava (ATCC 13223); Pseudomonas thermocarboxydovorans (ATCC 35961); Pseudomonas thermotolerans; Pseudomonas thivervalensis; Pseudomonas vancouverensis (ATCC 700688); Pseudomonas wisconsinensis; and Pseudomonas xiamenensis.<br>
The host cell can be selected from "Gram(-) Proteobacteria Subgroup 17." "Gram(-) Proteobacteria Subgroup 17" is defined as the group of Proteobacteria known in the art as the "fluorescent Pseudomonads" including those belonging, e.g., to the following Pseudomonas species: Pseudomonas azotoformans; Pseudomonas brenneri; Pseudomonas cedrella; Pseudomonas corrugata; Pseudomonas extremorientalis; Pseudomonas fluorescens; Pseudomonas gessardii; Pseudomonas libanensis; Pseudomonas mandelii; Pseudomonas marginalis; Pseudomonas migulae; Pseudomonas mucidolens; Pseudomonas orientalis; Pseudomonas rhodesiae; Pseudomonas synxantha; Pseudomonas tolaasii; and Pseudomonas veronii.<br>
The host cell can be selected from "Gram(-) Proteobacteria Subgroup 18." "Gram(-) Proteobacteria Subgroup 18" is defined as the group of all subspecies, varieties, strains, and other sub-special units of the species Pseudomonas fluorescens, including those belonging, e.g., to the following (with the ATCC or other deposit numbers of exemplary strain(s) shown in parenthesis): Pseudomonas fluorescens biotype A, also called biovar 1 or biovar I (ATCC 13525); Pseudomonas fluorescens biotype B, also called biovar 2 or biovar II (ATCC 17816); Pseudomonas fluorescens biotype C, also called biovar 3 or biovar III (ATCC 17400); Pseudomonas fluorescens biotype F, also called biovar 4 or biovar IV (ATCC 12983); Pseudomonas fluorescens biotype G, also called biovar 5 or biovar V (ATCC 17518); Pseudomonas fluorescens biovar VI; Pseudomonas fluorescens PfO-1; Pseudomonas fluorescens Pf-5 (ATCC BAA-477); Pseudomonas fluorescens SBW25; and Pseudomonas fluorescens subsp. cellulosa (NCIMB 10462).<br>
The host cell can be selected from "Gram(-) Proteobacteria Subgroup 19." "Gram(-) Proteobacteria Subgroup 19" is defined as the group of all strains of Pseudomonas fluorescens biotype A. One strain of this biotype is P. fluorescens strain MB101 (see US Patent No. 5,169,760 to Wilcox), and derivatives thereof.<br>
In some embodiments the host cell is any of the Proteobacteria of the order Pseudomonadales. In other embodiments the host cell is any of the Proteobacteria of the family Pseudomonadaceae.<br>
In additional embodiments the host cell is selected from "Gram(-) Proteobacteria Subgroup 1." In some embodiments the host cell is selected from "Gram(-) Proteobacteria Subgroup 2." In other embodiments the host cell is selected from "Gram(-) Proteobacteria Subgroup 3." In additional embodiments the host cell is selected from "Gram(-) Proteobacteria Subgroup 5." For some embodiments the host cell is selected from "Gram(-) Proteobacteria Subgroup 7." Yet, in other embodiments the host cell is selected from "Gram(-) Proteobacteria Subgroup 12", "Gram(-) Proteobacteria Subgroup 15", "Gram(-) Proteobacteria Subgroup 17", "Gram(-) Proteobacteria Subgroup 18", and/or "Gram(-) Proteobacteria Subgroup 19".<br>
Additional P. fluorescens strains that can be used in the present invention include Pseudomonas fluorescens Migula and Pseudomonas fluorescens Loitokitok, having the following ATCC designations: [NCIB 8286]; NRRL B-1244; NCIB 8865 strain CO1; NCIB 8866 strain C02; 1291 [ATCC 17458; IFO 15837; NCIB 8917; LA; NRRL B-1864; pyrrolidine; PW2 [ICMP 3966; NCPPB 967; NRRL B-899]; 13475; NCTC 10038; NRRL B-1603 [6; IFO 15840]; 52-1C; CCEB 488-A [BU 140]; CCEB 553 [IBM 15/47]; IAM 1008 [AHH-27]; IAM 1055 [AHH-23]; 1 [IFO 15842]; 12 [ATCC 25323; NIH 11; den Dooren de Jong 216]; 18 [IFO 15833; WRRL P-7]; 93 [TR-10]; 108 [52-22; IFO 15832]; 143 [IFO 15836; PL]; 149 [2-40-40; IFO 15838]; 182 [IFO 3081; PJ 73]; 184 [IFO 15830]; 185 [W2 L-l]; 186 [IFO 15829; PJ 79]; 187 [NCPPB 263]; 188 [NCPPB 316]; 189 [PJ227; 1208]; 191 [IFO 15834; PJ 236; 22/1]; 194 [Klinge R-60; PJ 253]; 196 [PJ 288]; 197 [PJ 290]; 198 [PJ 302]; 201 [PJ 368]; 202 [PJ 372]; 203 [PJ 376]; 204 [IFO 15835; PJ 682]; 205 [PJ 686]; 206 [PJ 692]; 207 [PJ 693]; 208 [PJ 722]; 212 [PJ 832]; 215 [PJ 849]; 216 [PJ 885]; 267 [B-9]; 271 [B-1612]; 401 [C71A; IFO 15831; PJ 187]; NRRL B-3178 [4; IFO 15841]; KY 8521; 3081; 30-21; [IFO 3081]; N; PYR; PW; D946-B83 [BU 2183; FERM-P 3328]; P-2563 [FERM-P 2894; IFO 13658]; IAM-1126 [43F]; M-l; A506 [A5-06]; A505 [A5-05-1]; A526 [A5-26]; B69; 72; NRRL B-4290; PMW6 [NCIB 11615]; SC 12936; Al [IFO 15839]; F 1847 [CDC-EB]; F 1848 [CDC 93]; NCIB 10586; P17; F-12; AmMS 257; PRA25; 6133D02; 6519E01; Nl; SC15208; BNL-WVC; NCTC 2583 [NCIB 8194]; H13; 1013 [ATCC 11251; CCEB 295]; IFO 3903; 1062; or Pf-5.<br>
III.     NUCLEIC ACID CONSTRUCTS<br><br>
Nucleic acid constructs are provided for use in the present invention, comprising nucleic acids encoding peptides of interest operably linked to carbon source inducible promoters. In some embodiments at least one carbon source inducible promoter is contained on a nucleic acid construct and operably linked to a nucleic acid encoding at least one peptide of interest. In other embodiments the carbon source inducible promoter is mannitol, glucitol, arabitol, or derivative thereof. The peptide of interest can be a monomer. In an alternative embodiment, the carbon source inducible promoter is operably linked to a nucleic acid sequence encoding more than one monomer. In other embodiments more than one mannitol, glucitol, or arabitol-inducible promoter is contained on a nucleic acid construct, wherein the promoters are covalently linked in tandem, and operably linked to a nucleic acid encoding a peptide of interest. In another embodiment, more than one mannitol, glucitol, or arabitol-inducible promoter is contained on the nucleic acid construct, wherein each promoter is separately and operably linked to a nucleic acid encoding a peptide of interest. In some embodiments the peptides of interest may be the same peptides or different peptides.<br>
In another embodiment, the nucleic acid construct can contain a mannitol, glucitol, or arabitol-inducible promoter and another promoter that is not induced by mannitol, glucitol, arabitol, or a derivative thereof. The mannitol-inducible promoter can be covalently linked in tandem, or separate and operably linked to a separate nucleic acid encoding a peptide of interest. For example, one promoter can be a mannitol, glucitol, or arabitol-inducible promoter and the other promoter can be a lac (IPTG), lacUVS (IPTG), tac (IPTG), trc (IPTG), Psyn (IPTG), trp (tryptophan starvation), araBAD (1-arabinose), lppa (IPTG), Ipp-lac (IPTG), phoA (phosphate starvation), recA (nalidixic acid), proU (osmolarity), cst-1 (glucose starvation), tetA (tretracylin), cadA (pH), nar (anaerobic conditions), PL (thermal shift to 42° C), cspA (thermal shift to 20° C), T7 (thermal induction), T7-lac operator (IPTG), T3-lac operator (IPTG), T5-lac operator (IPTG), T4 gene32 (T4 infection), nprM-lac operator (IPTG), Pm (alkyl- or halo-benzoates), Pu (alkyl- or halo-toluenes), Psal (salicylates), ant (anthranilate), ben (benzoate) or VHb (oxygen) promoter. See, for example, Makrides, S.C. (1996) Microbiol. Rev. 60, 512-538; Hannig G. &amp; Makrides, S.C. (1998) TIBTECH 16, 54-60; Stevens, R.C. (2000) Structures 8, R177-R185. See, e.g.: J. Sanchez-Romero &amp; V. De Lorenzo, Genetic Engineering of Nonpathogenic Pseudomonas strains as Biocatalysts for Industrial and Environmental Processes, in Manual of Industrial Microbiology and Biotechnology (A. Demain &amp; J. Davies, eds.) pp.460-74 (1999) (ASM Press, Washington, D.C.); H. Schweizer, Vectors to express foreign genes and techniques to monitor gene<br><br>
expression for Pseudomonads, Current Opinion in Biotechnology, 12:439-445 (2001); and R. Slater &amp; R. Williams, The Expression of Foreign DNA in Bacteria, in Molecular Biology and Biotechnology (J. Walker &amp; R. Rapley, eds.) pp. 125-54 (2000) (The Royal Society of Chemistry, Cambridge, UK).<br>
Other Elements<br>
Other regulatory elements can be included in the nucleic acid expression construct. Such elements include, but are not limited to, for example, transcriptional enhancer sequences, translational enhancer sequences, other promoters, activators, translational start and stop signals, transcription terminators, cistronic regulators, polycistronic regulators, tag sequences, such as nucleotide sequence "tags" and "tag" peptide coding sequences, which facilitates identification, separation, purification, or isolation of an expressed polypeptide, including His-tag, Flag-tag, T7-tag, S-tag, HSV-tag, B-tag, Strep-tag, polyarginine, polycysteine, polyphenylalanine, polyaspartic acid, (Ala-Trp-Trp-Pro)n, thioredoxin, beta-galactosidase, chloramphenicol acetyltransferase, cyclomaltodextrin gluconotransferase, CTP:CMP-3-deoxy-D-manno-octulosonate cytidyltransferase, trpE or trpLE, avidin, streptavidin, T7 gene 10, T4 gp55, Staphylococcal peptide A, streptococcal peptide G, GST, DHFR, CBP, MBP, galactose binding domain, Calmodulin binding domain, GFP, KSI, c-myc, ompT, ompA, pelB,, NusA, ubiquitin, and hemosylin A.<br>
A peptide-encoding gene according to the present invention can include, in addition to the peptide coding sequence, the following regulatory elements operably linked thereto: a promoter, a ribosome binding site (RBS), a transcription terminator, translational start and stop signals. Useful RBSs can be obtained from any of the species useful as host cells in expression systems according to the present invention, preferably from the selected host cell. Many specific and a variety of consensus RBSs are known, e.g., those described in and referenced by D. Frishman et al., Starts of bacterial genes: estimating the reliability of computer predictions, Gene 234(2):257-65 (8 Jul 1999); and B.E. Suzek et al., A probabilistic method for identifying start codons in bacterial genomes, Bioinformatics 17(12):! 123-30 (Dec 2001). In addition, either native or synthetic RBSs may be used, e.g., those described in: EP 0207459 (synthetic RBSs); O. Ikehata et al., Primary structure of nitrile hydratase deduced from the nucleotide sequence of a Rhodococcus species and its expression in Escherichia coli, Eur. J. Biochem. 181(3):563-70 (1989) (native RBS sequence of AAGGAAG). Further examples of methods, vectors, and translation and transcription elements, and other elements useful in the present invention are described in, e.g.: US Patent<br><br>
No. 5,055,294 to Gilroy and US Patent No. 5,128,130 to Gilroy et al.; US Patent No. 5,281,532 to Rammler et al.; US Patent Nos. 4,695,455 and 4,861,595 to Barnes et al.; US Patent No. 4,755,465 to Gray et al.; and US Patent No. 5,169,760 to Wilcox.<br>
Vectors<br>
Generally, the nucleic acid construct will be contained on an expression vector, and will include origins of replication and selectable markers permitting transformation of the bacterial host cell. The recombinant peptide of interest is assembled in appropriate phase with translation initiation and termination sequences, and in certain embodiments, a leader sequence capable of directing secretion of the translated polypeptide can be included in the nucleic acid construct. Optionally, and in accordance with the present invention, the recombinant peptide sequence can encode a fusion polypeptide including an N-terminal identification peptide imparting desired characteristics, e.g., stabilization or simplified purification of expressed recombinant product. In certain embodiments, the vector may be maintained in an episomal fashion (extra-chromosomally), or be inserted into the genome of the host bacterial cell.<br>
Useful expression vectors for use with bacteria in expressing recombinant peptides are constructed by inserting a structural DNA sequence encoding a desired target peptide together with suitable translation initiation and termination signals in operable reading phase with the carbon source-inducible promoter. The vector will comprise one or more phenotypic selectable markers and an origin of replication to ensure maintenance of the vector and to, if desirable, provide amplification within the host. Alternatively, the vector can further provide sequences that allow for integration into the genome of the host cell. Suitable hosts for transformation in accordance with the present disclosure include any bacterial cell capable of expressing a recombinant peptide driven by a carbon source inducible promoter. In some embodiments the bacteria can include various species within the genera Escherichia and Pseudomonas.<br>
Vectors are known in the art as useful for expressing recombinant peptides in host cells, and any of these may be modified and used for expressing recombinant peptides of interest according to the present invention. Such vectors include, e.g., plasmids, cosmids, and phage expression vectors. Examples of useful plasmid vectors that can be modified for use in the present invention include, but are not limited to, the expression plasmids pBBRlMCS, pDSK519, pKT240, pML122, pPSlO, RK2, RK6, pRO1600, and RSF1010. Further examples can include pALTER-Exl, pALTER-Ex2, pBAD/His, pBAD/Myc-His, pBAD/glll,<br><br>
pCal-n, pCal-n-EK, pCal-c, pCal-Kc, pcDNA 2.1, pDUAL, pET-3a-c, pET 9a-d, pET-1 la-d, pET-12a-c, PET-14b, pET15b, pET-16b, pET-17b, pET-19b, pET-20b(+), pET-21 a-d(+), pET-22b(+), pET-23a-d(+), pET24a-d(+), pET-25b(+), pET-26b(+), pET-27b(+), pET28a-c(+), pET-29a-c(+), pET-30a-c(+), pET31b(+), pET-32a-c(+), pET-33b(+), pET-34b(+), pET35b(+), PET-36b(+), pET-37b(+), pET-38b(+), pET-39b(+), pET-40b(+), pET-41a-c(+), pET-42a-c(+), pET-43a-c(+), pETBlue-1, pETBlue-2, pETBlue-3, pGEMEX-1, pGEMEX-2, pGEXlXT, pGEX-2T, pGEX-2TK, pGEX-3X, pGEX-4T, pGEX-5X, pGEX-6P, pHAT10/ll/12, pHAT20, pHAT-GFPuv, pKK223-3, pLEX, pMAL-c2X, pMAL-c2E, pMAL-c2g, pMAL-p2X, pMAL-p2E, pMAL-p2G, pProEX HT, pPROLar.A, pPROTet.E, pQE-9, pQE-16, pQE-30/31/32, pQE-40, pQE-50, pQE-70, pQE-80/81/82L, pQE-100, pRSET, and pSE280, pSE380, pSE420, pThioHis, pTrc99A, pTrcHis, pTrcHis2, pTriEx-1, pTriEx-2, pTrxFus. Other examples of such useful vectors include those described by, e.g.: N. Hayase, in Appl. Envir. Microbiol. 60(9):3336-42 (Sep 1994); A.A. Lushnikov et al., in Basic Life Sci. 30:657-62 (1985); S. Graupner &amp; W. Wackernagel, in Biomolec. Eng. 17(1): 11-16. (Oct 2000); H.P. Schweizer, in Curr. Opin. Biotech. 12(5):439-45 (Oct 2001); M. Bagdasarian &amp; K.N. Timmis, in Curr. Topics Microbiol. Immunol. 96:47-67 (1982); T. Ishii et al., in FEMS Microbiol. Lett. 116(3):307-13 (Mar 1, 1994); I.N. Olekhnovich &amp; Y.K. Fomichev, in Gene 140(l):63-65 (Mar 11, 1994); M. Tsuda &amp; T. Nakazawa, in Gene 136(1-2):257-62 (Dec 22, 1993); C. Nieto et al., in Gene 87(1): 145-49 (Mar 1, 1990); J.D. Jones &amp; N. Gutterson, in Gene 61(3):299-306 (1987); M. Bagdasarian et al., in Gene 16(l-3):237-47 (Dec 1981); H.P. Schweizer et al., in Genet. Eng. (NY) 23:69-81 (2001); P. Mukhopadhyay et al., in J. Bact. 172(1):477-80 (Jan 1990); D.O. Wood et al., in J. Bact. 145(3):1448-51 (Mar 1981); and R. Holtwick et al., in Microbiology 147(Pt 2):337-44 (Feb 2001).<br>
Further examples of expression vectors that can be useful in bacterial host cells include those listed in Table 5 as derived from the indicated replicons:<br>
Table 5. Some Examples of Useful Expression Vectors<br>
(Table Removed)<br><br><br><br>
The expression plasmid, RSF1010, is described, e.g., by F. Heffron et al., in Proc. Nat'l Acad. Sci. USA 72(9):3623-27 (Sep 1975), and by K. Nagahari &amp; K. Sakaguchi, in J. Bact. 133(3): 1527-29 (Mar 1978). Plasmid RSF1010 and derivatives thereof are can be used as vectors in the present invention. Exemplary, useful derivatives of RSF1010, which are known in the art, include, e.g., pKT212, pKT214, pKT231 and related plasmids, and pMYC1050 and related plasmids (see, e.g., US Patent Nos. 5,527,883 and 5,840,554 to Thompson et al.), such as, e.g., pMYC1803. Plasmid pMYC1803 is derived from the RSFlOlO-based plasmid pTJS260 (see US Patent No. 5,169,760 to Wilcox), which carries a regulated tetracycline resistance marker and the replication and mobilization loci from the RSF1010 plasmid. Other exemplary useful vectors include those described in US Patent No. 4,680,264 to Puhler et al.<br>
In additional embodiments an expression plasmid is used as the expression vector. In another embodiment, RSF1010 or a derivative thereof is used as the expression vector. In still another embodiment, pMYClOSO or a derivative thereof, or pMYC1803 or a derivative thereof, is used as the expression vector.<br>
VI.     EXPRESSION OF RECOMBINANT POLYPEPTIDES IN A BACTERIAL HOST CELL<br>
Embodiments of the present invention include processes for expressing recombinant peptides for use in peptide production. In general, the process provides expression of a nucleic acid construct comprising nucleic acids encoding at least one recombinant polypeptide operably linked to at least one mannitol, arabitol, or glucitol-inducible promoter, wherein the nucleic acid construct is expressed in a host cell that is rendered deficient in its ability to metabolize or degrade mannitol, glucitol, or arabitol, or derivatives or analogues thereof. In some embodiments the inducible promoter can be a mannitol inducible promoter, or derivative thereof. In other embodiments the host cell is rendered deficient for the metabolization or degradation of mannitol, which is used as an inducer in promoter induction. In one embodiment, the host cell is Escherichia coli. In another embodiment, the host cell<br><br>
can be a Pseudomonad or closely related bacterial cell. In additional embodiments the host cell can be a Pseudomonas jluorescens cell. In some embodiments the host cell is deficient for the metabolization or degradation of mannitol, arabitol, or glucitol, or analogues or derivatives thereof, and expresses a nucleic acid encoding at least one recombinant polypeptide operably linked to at least one mannitol, arabitol, glucitol, or glycerol-inducible promoter.<br>
The method generally includes:<br>
a)	providing a host cell, preferably an E. coli or Pseudomonas jluorescens, as<br>
described in the present invention,<br>
b)	transfecting the host cell with at least one nucleic acid expression vector<br>
comprising at least one recombinant polypeptide of interest operably linked to at<br>
least one mannitol, glucitol, or arabitol-inducible promoter, and growing the host<br>
cell in a sufficient growth medium; and<br>
c)	adding a mannitol, glucitol, or arabitol, or analogues or derivatives thereof to the<br>
growth medium in an amount capable of inducing the expression of the peptide of<br>
interest, wherein the host cell is rendered incapable of degrading or metabolizing<br>
the added mannitol, glucitol, or arabitol, or analogues or derivatives thereof; and<br>
d)	expressing the target recombinant polypeptide of interest.<br>
The method can further comprise e) transfecting the host cell with at least one nucleic acid expression construct comprising an additional inducible promoter operably linked to at least one peptide of interest. In addition, the method can further comprise f) isolating at least one recombinant peptide. The method can also include g) purifying at least one recombinant peptide.<br>
Alternatively, the method can comprise in step c) adding glycerol, or a derivative or analogue thereof, to the growth medium in an amount capable of inducing the peptide of interest, wherein the host cell is rendered incapable of degrading or metabolizing mannitol, glucitol, or arabitol, or analogues or derivatives thereof.<br>
In some embodiments the inducible promoter is a mannitol-inducible promoter. In other embodiments, the mannitol, glucitol, or arabitol-inducible promoter is also capable of induction by glycerol.<br>
Wherein the method further comprises e) transfecting the host cell with at least one nucleic acid expression construct comprising an additional inducible promoter operably linked to at least one peptide of interest, the peptide of interest can be the same or a different peptide of interest from the one encoded by the nucleic acid operably linked to the mannitol,<br><br>
arabitol, or glucitol-inducible promoter. Other inducible promoters can be, for example, a lac or tac family promoter, including Plac, Ptac, Ptrc, PtacII, PlacUVS, lppa, lpp-PlacUV5, Ipp-lac, nprM-lac, T71ac, TSlac, T31ac, and Pmac, trp (tryptophan starvation), araBAD (1-arabinose), phoA (phosphate starvation), recA (nalidixic acid), proU (osmolarity), cst-1 (glucose starvation), tetA (tretracylin), cadA (pH), nar (anaerobic conditions), PL (thermal shift to 42° C), cspA (thermal shift to 20° C), T7 (thermal induction), T4 gene32 (T4 infection), Pm (alkyl- or halo-benzoates), Pu (alkyl- or halo-toluenes), Psal (salicylates), ant (anthranilate), ben (benzoate) or VHb (oxygen) promoter.<br>
In some embodiments the present invention can employ more than one mannitol, glucitol, or arabitol-inducible promoter in producing a peptide, or more than one peptide, of interest. In other embodiments the multiple promoters can have different sequences, with each promoter capable of induction by the same carbon source at a different rate based on the specific sequence utilized. For example, one mannitol-inducible promoter sequence may be capable of expressing a peptide of interest at a higher expression than a mannitol-inducible promoter with a differing nucleic acid sequence based on differing inherent characteristics or modifications. Alternatively, enhancer elements may be contained, in addition to the promoter sequence, within the promoter region that affect the rate of expression of the promoter sequences differently based on the spacing or linkage of the respective promoters to the enhancer elements, whether the promoter sequences are the same or different.<br>
In embodiments of the present invention the mannitol, glucitol, or arabitol-inducible promoter is used concomitantly in a bacterial host cell with a lac or tac family promoter including, for example, Plac, Ptac, Ptrc, PtacII, PlacUVS, Ipp-lac, nprM-lac, T71ac, TSlac, T31ac, and Pmac.<br>
In some embodiments the expression system is capable of expressing the target polypeptide at a total productivity of polypeptide of at least 0.1 g/L to at least 80 g/L. In other embodiments the recombinant polypeptide is expressed at a level of at least 0.3 g/L, 0.4 g/L, 0.7 g/L, 1.0 g/L, 1.5 g/L, 2 g/L, 3 g/L, 4g/L, Sg/L, 6 g/L, 7 g/L, 8 g/L, 9 g/L, 10 g/L, 12 g/L, 15 g/L, 20 g/L, 25 g/L, 30 g/L, 35 g/L, 40 g/L, 45 g/L, 50 g/L, 60 g/L, 70 g/L, or at least 80 g/L.<br>
In one embodiment, at least one recombinant peptide can be expressed in a bacterial cell that is incapable of degrading or metabolizing mannitol, arabitol, or glucitol, wherein expression of the recombinant polypeptide is induced utilizing mannitol, arabitol, glucitol, or glycerol, or derivatives or analogues thereof. Alternatively, more than one recombinant peptide can be expressed in a cell of the present invention, wherein the nucleic acids<br><br>
encoding the recombinant peptides can be contained on the same vector, or alternatively, on multiple vectors, and operably linked to more than one promoter, including at least one mannitol, arabitol, or glucitol-inducible promoter.<br>
In yet another embodiment, nucleic acid constructs encoding different target polypeptides can be maintained in a bacterial host cell incapable of degrading or metabolizing mannitol, glucitol, or arabitol, or derivatives or analogues thereof, wherein mannitol, glucitol, arabitol, or glycerol is utilized to induce expression of the target polypeptides. In such embodiments, a first nucleic acid encoding a first peptide of interest, and a second nucleic acid encoding second peptide of interest are regulated independently of each other through the use of different promoters, wherein at least one promoter is a mannitol, glucitol, or arabitol-inducible promoter. Such multi-target gene expression systems allow for independent regulation and optimization of expression of each peptide. In certain embodiments, one expressed peptide can modulate the expression or resultant peptide characteristic of the other peptide of interest.<br>
Examples of such a multi-target gene system include, but are not limited to: (1) systems in which the expression product of one of the target genes interacts with the other target gene itself; (2) systems in which the expression product of one of the target genes interacts with the other target gene's expression product, e.g., a peptide and its binding peptide or the a- and (3- polypeptides of an an-pn peptide; (3) systems in which the two expression products of the two genes both interact with a third component, e.g., a third component present in the host cell; (4) systems in which the two expression products of the two genes both participate in a common biocatalytic pathway; and (5) systems in which the two expression products of the two genes function independently of one another, e.g., a bi-clonal antibody expression system.<br>
In one example of a multi-target gene system of the above-listed type (1), a first target gene can encode a desired target peptide, wherein the first target gene is under the control of a regulatable promoter; the second target gene may then encode a peptide involved in regulating the promoter of the first target gene, e.g., the second target gene may encode the first target gene's promoter activator or represser peptide. In an example in which the second gene encodes a promoter regulatory peptide for the first gene, the coding sequence of the second gene can be under the control of a mannitol-inducible promoter. In some embodiments the second gene will be part of a separate DNA construct that is a maintained in the cell as a high-copy-number construct with a copy number of at least 10, 20, 30, 40, 50, or<br><br>
more than 50 copies being maintained in the host cell, or that has been inserted into the genome of the cell.<br>
In another example of a dual-target gene system, the second target gene can encode a peptide that assists in the folding of the first target gene product, or assists in directing the peptide of interest to a cell compartment (e.g., a chaperone protein). For example, the first target gene product can be a peptide that is normally expressed in a bacterial host cell in an incorrectly folded form. Alternatively, the first target gene product can be a peptide that is normally expressed in a bacterial cell in an insoluble form. The second target gene, in these cases, can encode for a protein that assists in properly folding the protein, or directing the protein to a specific location in the cell (e.g., such as the periplasm). Examples of such peptides and proteins include, but are not limited to: cbpA, htpG, dnaK, dnaJ, fkbP2,groES, groEL, htpG or cbpA, HSP70 proteins, HSP110/SSE proteins, HSP40 (DNAJ-related) proteins, GRPE-like proteins, HSP90 proteins, CPN60 and CPN10 proteins, Cytosolic chaperonins, HSP100 proteins, Small HSPs, Calnexin and calreticulin, PDI and thioredoxin-related proteins, Peptidyl-prolyl isomerases, Cyclophilin PPIases, FK-506 binding proteins, Parvulin PPIases, Individual chaperonins, Protein specific chaperones, or intramolecular chaperones. Other proteins that can be expressed in the current invention include folding modulators that are generally described in "Guidebook to Molecular Chaperones and Protein-Folding Catalysts" (1997) ed. M. Gething, Melbourne University, Australia.<br>
Transformation<br>
Transformation of the host cells with the vector(s) may be performed using any transformation methodology known in the art, and the bacterial host cells may be transformed as intact cells or as protoplasts (i.e. including cytoplasts). Exemplary transformation methodologies include poration methodologies, e.g., electroporation, protoplast fusion, bacterial conjugation, and divalent cation treatment, e.g., calcium chloride treatment or<br>
CaCl2/Mg2+ treatment, or other well known methods in the art. See, e.g., Morrison, J. Bact, 132:349-351 (1977); Clark-Curtiss &amp; Curtiss, Methods in Enzymology, 101:347-362 (Wu et al., eds, 1983), Sambrook et al., Molecular Cloning, A Laboratory Manual (2nd ed. 1989); Kriegler, Gene Transfer and Expression: A Laboratory Manual (1990); and Current Protocols in Molecular Biology (Ausubel et al., eds., 1994)).<br>
Selection<br><br>
Preferably, cells that are not successfully transformed are selected against following transformation, and continuously during the fermentation. The selection marker can be an auxotrophic selection marker or a traditional antibiotic selection marker. When the cell is auxotrophic for multiple nutrient compounds, the auxotrophic cell can be grown on medium supplemented with all of those nutrient compounds until transformed with the prototrophy-restoring vector. Where the selection marker is an antibiotic resistance gene, the associated antibiotic can be added to the medium to select against non transformed and revertant cells, as well known in the art.<br>
Fermentation<br>
As used herein, the term "fermentation" includes both embodiments in which literal fermentation is employed and embodiments in which other, non-fermentative culture modes are employed. Fermentation may be performed at any scale. In some embodiments the fermentation medium may be selected from among rich media, minimal media, a mineral salts media; a rich medium may be used, but is preferably avoided. In another embodiment either a minimal medium or a mineral salts medium is selected.<br>
Mineral salts media consists of mineral salts and a carbon source such as, e.g., glucose, sucrose, or glycerol. Examples of mineral salts media include, e.g., M9 medium, Pseudomonas medium (ATCC 179), Davis and Mingioli medium (see, BD Davis &amp; ES Mingioli, in J. Bact. 60:17-28 (1950)). The mineral salts used to make mineral salts media include those selected from among, e.g., potassium phosphates, ammonium sulfate or chloride, magnesium sulfate or chloride, and trace minerals such as calcium chloride, borate, and sulfates of iron, copper, manganese, and zinc. No organic nitrogen source, such as peptone, tryptone, amino acids, or a yeast extract, is included in a mineral salts medium. Instead, an inorganic nitrogen source is used and this may be selected from among, e.g., ammonium salts, aqueous ammonia, and gaseous ammonia. A typical mineral salts medium will contain glucose as the carbon source. In comparison to mineral salts media, minimal media can also contain mineral salts and a carbon source, but can be supplemented with, e.g., low levels of amino acids, vitamins, peptones, or other ingredients, though these are added at very minimal levels.<br>
Ideally, the media selected allows for high cell density cultivation (HCDC) for growth of bacterial cells. The HCDC can start as a batch process which is followed by two-phase fed-batch cultivation. After unlimited growth in the batch part, growth can be controlled at a reduced specific growth rate over a period of 3 doubling times in which the biomass<br><br>
concentration can increased several fold. Further details of such cultivation procedures is described by Riesenberg, D.; Schulz, V.; Knorre, W. A.; Pohl, H. D.; Korz, D.; Sanders, E. A.; Ross, A.; Deckwer, W. D. (1991) "High cell density cultivation of Escherichia coli at controlled specific growth rate" J Biotechnol: 20(1) 17-27.<br>
The expression system according to the present invention can be cultured in any fermentation format. For example, batch, fed-batch, semi-continuous, and continuous fermentation modes may be employed herein.<br>
The expression systems according to the present invention are useful for transgene expression at any scale (i.e. volume) of fermentation. Thus, e.g., microliter-scale, centiliter scale, and deciliter scale fermentation volumes may be used; and 1 Liter scale and larger fermentation volumes can be used. In some embodiments the fermentation volume will be at or above 1 Liter. In other embodiments the fermentation volume will be at or above 5 Liters, 10 Liters, 15 Liters, 20 Liters, 25 Liters, 50 Liters, 75 Liters, 100 Liters, 200 Liters, 500 Liters, 1,000 Liters, 2,000 Liters, 5,000 Liters, 10,OOOLiters or 50,000 Liters.<br>
In the present invention, growth, culturing, and/or fermentation of the transformed host cells is performed within a temperature range permitting survival of the host cells, preferably a temperature within the range of about 4°C to about 55°C, inclusive.<br>
Cell Density<br>
In embodiments of the present invention Pseudomonads and closely related bacteria are utilized in the present invention. Pseudomonads, and particularly Pseudomonas fluorescens, can be grown in high cell densities. To this end, Pseudomonas fluorescens expressions systems according to the present invention can provide a cell density of about 20 g/L or more. The Pseudomonas fluorescens expressions systems according to the present invention can likewise provide a cell density of at least about 70 g/L, as stated in terms of biomass per volume, the biomass being measured as dry cell weight.<br>
In some embodiments the cell density will be at least 20 g/L. In another embodiment, the cell density will be at least 25 g/L, 30 g/L, 35 g/L, 40 g/L, 45 g/L, 50 g/L, 60 g/L, 70 g/L, 80 g/L, 90 g/L., 100 g/L, 110 g/L, 120 g/L, 130 g/L, 140 g/L, or at least 150 g/L.<br>
In another embodiments, the cell density at induction will be between 20 g/L and 150 g/L;, 20 g/L and 120 g/L; 20 g/L and 80 g/L; 25 g/L and 80 g/L; 30 g/L and 80 g/L; 35 g/L and 80 g/L; 40 g/L and 80 g/L; 45 g/L and 80 g/L; 50 g/L and 80 g/L; 50 g/L and 75 g/L; 50 g/L and 70 g/L; 40 g/L and 80 g/L.<br><br>
Expression Levels of Recombinant Peptides<br>
The expression systems according to the present invention can express transgenic polypeptides at a level at between 5% and 80% total cell peptide (%tcp). In some embodiments the expression level will be at or above 5%, 8%, 10%, 12%, 15%, 20%, 25%, 30%, 35%, 40%, 45%, 50%, 55%, 60%, 65%, 70%, 75%, or 80% tcp.<br>
Isolation and Purification<br>
The recombinant peptides produced according to this invention may be isolated and purified to substantial purity by standard techniques well known in the art, including, but not limited to, ammonium sulfate or ethanol precipitation, acid extraction, anion or cation exchange chromatography, phosphocellulose chromatography, hydrophobic interaction chromatography, affinity chromatography, nickel chromatography, hydroxylapatite chromatography, reverse phase chromatography, lectin chromatography, preparative electrophoresis, detergent solubilization, selective precipitation with such substances as column chromatography, immunopurification methods, and others. For example, peptides having established molecular adhesion properties can be reversibly fused a ligand. With the appropriate ligand, the peptide can be selectively adsorbed to a purification column and then freed from the column in a relatively pure form. The fused peptide is then removed by enzymatic activity. In addition, peptide can be purified using immunoaffinity columns or Ni-NTA columns. General techniques are further described in, for example, R. Scopes, Peptide Purification: Principles and Practice, Springer-Verlag: N.Y. (1982); Deutscher, Guide to Peptide Purification, Academic Press (1990); U.S. Pat. No. 4,511,503; S. Roe, Peptide Purification Techniques: A Practical Approach (Practical Approach Series), Oxford Press (2001); D. Bollag, et al., Peptide Methods, Wiley-Lisa, Inc. (1996); AK Patra et al., Peptide Expr Purif, 18(2): p/ 182-92 (2000); and R. Mukhija, et al., Gene 165(2): p. 303-6 (1995). See also, for example, Ausubel, et al. (1987 and periodic supplements); Deutscher (1990) "Guide to Peptide Purification," Methods in Enzymology vol. 182, and other volumes in this series; Coligan, et al. (1996 and periodic Supplements) Current Protocols in Peptide Science Wiley/Greene, NY; and manufacturer's literature on use of peptide purification products, e.g., Pharmacia, Piscataway, N.J., or Bio-Rad, Richmond, Calif. Combination with recombinant techniques allow fusion to appropriate segments, e.g., to a FLAG sequence or an equivalent which can be fused via a protease-removable sequence. See also, for example., Hochuli (1989) Chemische Industrie 12:69-70; Hochuli (1990) "Purification of Recombinant Peptides<br><br>
with Metal Chelate Absorbent" in Setlow (ed.) Genetic Engineering, Principle and Methods 12:87-98, Plenum Press, NY; and Crowe, et al. (1992) QIAexpress: The High Level Expression &amp; Peptide Purification System QIAGEN, Inc., Chatsworth, Calif.<br>
Detection of the expressed peptide is achieved by methods known in the art and includes, for example, radioimmunoassays, Western blotting techniques or immunoprecipitation.<br>
The recombinantly produced and expressed peptide can be recovered and purified from the recombinant cell cultures by numerous methods, for example, high performance liquid chromatography (HPLC) can be employed for final purification steps, as necessary.<br>
Certain peptides expressed in this invention may form insoluble aggregates ("inclusion bodies"). Several protocols are suitable for purification of peptides from inclusion bodies. For example, purification of inclusion bodies typically involves the extraction, separation and/or purification of inclusion bodies by disruption of the host cells, e.g., by incubation in a buffer of 50 mM TRIS/HCL pH 7.5, 50 mM NaCl, 5 mM MgCl.sub.2, 1 mM DTT, 0.1 mM ATP, and 1 mM PMSF. The cell suspension is typically lysed using 2-3 passages through a French Press. The cell suspension can also be homogenized using a Polytron (Brinknan Instruments) or sonicated on ice. Alternate methods of lysing bacteria are apparent to those of skill in the art (see, e.g., Sambrook et al., supra; Ausubel et al., supra).<br>
If necessary, the inclusion bodies can be solubilized, and the lysed cell suspension typically can be centrifuged to remove unwanted insoluble matter. Peptides that formed the inclusion bodies may be renatured by dilution or dialysis with a compatible buffer. Suitable solvents include, but are not limited to urea (from about 4 M to about 8 M), formamide (at least about 80%, volume/volume basis), and guanidine hydrochloride (from about 4 M to about 8 M). Although guanidine hydrochloride and similar agents are denaturants, this denaturation is not irreversible and renaturation may occur upon removal (by dialysis, for example) or dilution of the denaturant, allowing re-formation of immunologically and/or biologically active peptide. Other suitable buffers are known to those skilled in the art.<br>
Alternatively, it is possible to purify the recombinant peptides from the host periplasm. After lysis of the host cell, when the recombinant peptide is exported into the periplasm of the host cell, the periplasmic fraction of the bacteria can be isolated by cold osmotic shock in addition to other methods known to those skilled in the art. To isolate recombinant peptides from the periplasm, for example, the bacterial cells can be centrifuged to form a pellet. The pellet can be resuspended in a buffer containing 20% sucrose. To lyse the cells, the bacteria can be centrifuged and the pellet can be resuspended in ice-cold 5 mM MgSO.sub.4 and kept<br><br>
in an ice bath for approximately 10 minutes. The cell suspension can be centrifuged and the supernatant decanted and saved. The recombinant peptides present in the supernatant can be separated from the host peptides by standard separation techniques well known to those of skill in the art.<br>
An initial salt fractionation can separate many of the unwanted host cell peptides (or peptides derived from the cell culture media) from the recombinant peptide of interest. One such example can be ammonium sulfate. Ammonium sulfate precipitates peptides by effectively reducing the amount of water in the peptide mixture. Peptides then precipitate on the basis of their solubility. The more hydrophobic a peptide is, the more likely it is to precipitate at lower ammonium sulfate concentrations. A typical protocol includes adding saturated ammonium sulfate to a peptide solution so that the resultant ammonium sulfate concentration is between 20-30%. This concentration will precipitate the most hydrophobic of peptides. The precipitate is then discarded (unless the peptide of interest is hydrophobic) and ammonium sulfate is added to the supernatant to a concentration known to precipitate the peptide of interest. The precipitate is then solubilized in buffer and the excess salt removed if necessary, either through dialysis or diafiltration. Other methods that rely on solubility of peptides, such as cold ethanol precipitation, are well known to those of skill in the art and can be used to fractionate complex peptide mixtures.<br>
The molecular weight of a recombinant peptide can be used to isolated it from peptides of greater and lesser size using ultrafiltration through membranes of different pore size (for example, Amicon or Millipore membranes). As a first step, the peptide mixture can be ultrafiltered through a membrane with a pore size that has a lower molecular weight cutoff than the molecular weight of the peptide of interest. The retentate of the ultrafiltration can then be ultrafiltered against a membrane with a molecular cut off greater than the molecular weight of the peptide of interest. The recombinant peptide will pass through the membrane into the filtrate. The filtrate can then be chromatographed as described below.<br>
Recombinant peptides can also be separated from other peptides on the basis of its size, net surface charge, hydrophobicity, and affinity for ligands. In addition, antibodies raised against peptides can be conjugated to column matrices and the peptides immunopurified. All of these methods are well known in the art. It will be apparent to one of skill that chromatographic techniques can be performed at any scale and using equipment from many different manufacturers (e.g., Pharmacia Biotech).<br>
Renaturation and Refolding<br><br>
Insoluble peptide can be renatured or refolded to generate secondary and tertiary peptide structure conformation. Peptide refolding steps can be used, as necessary, in completing configuration of the recombinant product. Refolding and renaturation can be accomplished using an agent that is known in the art to promote dissociation/association of peptides. For example, the peptide can be incubated with dithiothreitol followed by incubation with oxidized glutathione disodium salt followed by incubation with a buffer containing a refolding agent such as urea.<br>
Recombinant peptide can also be renatured, for example, by dialyzing it against phosphate-buffered saline (PBS) or 50 mM Na-acetate, pH 6 buffer plus 200 mM NaCl. Alternatively, the peptide can be refolded while immobilized on a column, such as the Ni NTA column by using a linear 6M-1M urea gradient in 500 mM NaCl, 20% glycerol, 20 mM Tris/HCl pH 7.4, containing protease inhibitors. The renaturation can be performed over a period of 1.5 hours or more. After renaturation the peptides can be eluted by the addition of 250 mM immidazole. Immidazole can be removed by a final dialyzing step against PBS or 50 mM sodium acetate pH 6 buffer plus 200 mM NaCl. The purified peptide can be stored at 4.degree. C. or frozen at -80.degree. C.<br>
Other methods include, for example, those that may be described in MH Lee et al., Peptide Expr. Purif, 25(1): p. 166-73 (2002), W.K. Cho et al., J. Biotechnology, 77(2-3): p. 169-78 (2000), Ausubel, et al. (1987 and periodic supplements), Deutscher (1990) "Guide to Peptide Purification," Methods in Enzymology vol. 182, and other volumes in this series, Coligan, et al. (1996 and periodic Supplements) Current Protocols in Peptide Science Wiley/Greene, NY, S. Roe, Peptide Purification Techniques: A Practical Approach (Practical Approach Series), Oxford Press (2001); D. Bollag, et al., Peptide Methods, Wiley-Lisa, Inc. (1996).<br>
VI.     RECOMBINANT POLYPEPTIDES<br>
The present invention provides for the production of recombinant peptides in bacterial expression systems. Examples of recombinant polypeptides that can be used in the present invention include polypeptides derived from prokaryotic and eukaryotic organisms. Such organisms include organisms from the domain Archea, Bacteria, Eukarya, including organisms from the Kingdom Protista, Fungi, Plantae, and Animalia.<br>
Types of peptides that can be utilized in the present invention, in addition to the proteins and peptides described above, include non-limiting examples such as enzymes, which are responsible for catalyzing the thousands of chemical reactions of the living cell;<br><br>
keratin, elastin, and collagen, which are important types of structural, or support, peptides; hemoglobin and other gas transport peptides; ovalbumin, casein, and other nutrient molecules; antibodies, which are molecules of the immune system; peptide hormones, which regulate metabolism; and peptides that perform mechanical work, such as actin and myosin, the contractile muscle peptides.<br>
Other specific non-limiting polypeptides include molecules such as, e.g., renin, a growth hormone, including human growth hormone; bovine growth hormone; growth hormone releasing factor; parathyroid hormone; thyroid stimulating hormone; lipopeptides; .alpha. 1-antitrypsin; insulin A-chain; insulin B-chain; proinsulin; thrombopoietin; follicle stimulating hormone; calcitonin; luteinizing hormone; glucagon; clotting factors such as factor VIIIC, factor IX, tissue factor, and von Willebrands factor; anti-clotting factors such as Peptide C; atrial naturietic factor; lung surfactant; a plasminogen activator, such as urokinase or human urine or tissue-type plasminogen activator (t-PA); bombesin; thrombin; hemopoietic growth factor; tumor necrosis factor-alpha and -beta; enkephalinase; a serum albumin such as human serum albumin; mullerian-inhibiting substance; relaxin A-chain; relaxin B-chain; prorelaxin; mouse gonadotropin-associated peptide; a microbial peptide, such as beta-lactamase; Dnase; inhibin; activin; vascular endothelial growth factor (VEGF); receptors for hormones or growth factors; integrin; peptide A or D; rheumatoid factors; a neurotrophic factor such as brain-derived neurotrophic factor (BDNF), neurotrophin-3, -4, -5, or -6 (NT-3, NT-4, NT-5, or NT-6), or a nerve growth factor such as NGF-.beta.; cardiotrophins (cardiac hypertrophy factor) such as cardiotrophin-<br>
1	(CT-1); platelet-derived growth factor (PDGF); fibroblast growth factor such as aFGF and<br>
bFGF; epidermal growth factor (EGF); transforming growth factor (TGF) such as TGF-alpha<br>
and TGF-beta, including TGF-.beta.l, TGF-.beta.2, TGF-.beta.3, TGF-.beta.4, or TGF-<br>
.beta.5; insulin-like growth factor-I and -II (IGF-I and IGF-II); des(l-3)-IGF-I (brain IGF-I),<br>
insulin-like growth factor binding peptides; CD peptides such as CD-3, CD-4, CD-8, and CD-<br>
19; erythropoietin; osteoinductive factors; immunotoxins; a bone morphogenetic peptide<br>
(BMP); an interferon such as interferon-alpha, -beta, and -gamma; colony stimulating factors<br>
(CSFs), e.g., M-CSF, GM-CSF, and G-CSF; interleukins (ILs), e.g., IL-1 to IL-10; anti-HER-<br>
2	antibody; superoxide dismutase; T-cell receptors; surface membrane peptides; decay<br>
accelerating factor; viral antigen such as, for example, a portion of the AIDS envelope;<br>
transport peptides;  homing receptors;  addressins;  regulatory peptides;  antibodies;  and<br>
fragments of any of the above-listed polypeptides.<br><br>
The recombinant peptides to be expressed by according to the present invention can be expressed from polynucleotides in which the target polypeptide coding sequence is operably attached to transcription and translation regulatory elements to form a functional gene from which the host cell can express the peptide or peptide. The coding sequence can be a native coding sequence for the target polypeptide, if available, but will more preferably be a coding sequence that has been selected, improved, or optimized for use in the selected expression host cell: for example, by synthesizing the gene to reflect the codon use bias of a Pseudomonas species such as Pseudomonas fluorescens. The gene(s) that result will have been constructed within or will be inserted into one or more vector, which will then be transformed into the expression host cell. Nucleic acid or a polynucleotide said to be provided in an "expressible form" means nucleic acid or a polynucleotide that contains at least one gene that can be expressed by the selected bacterial expression host cell.<br>
Extensive sequence information required for molecular genetics and genetic engineering techniques is widely publicly available. Access to complete nucleotide sequences of mammalian, as well as human, genes, cDNA sequences, amino acid sequences and genomes can be obtained from GenBank at the URL address http://www.ncbi.nlm.nih.gov/Entrez. Additional information can also be obtained from GeneCards, an electronic encyclopedia integrating information about genes and their products and biomedical applications from the Weizmann Institute of Science Genome and Bioinformatics (http://bioinformatics.weizmann.ac.i1/cards/X nucleotide sequence information can be also obtained from the EMBL Nucleotide Sequence Database ( http://www.ebi.ac.uk/embl/) or the DNA Databank or Japan (DDBJ, http://www.ddbj.nig.ac.jp/; additional sites for information on amino acid sequences include Georgetown's peptide information resource website (http://www-nbrf.georgetown.edu/pir/) and Swiss-Prot (http://au.expasy.org/sprot/sprot-top.html).<br>
The present invention is explained in greater detail in the Examples that follow. These examples are intended as illustrative of the invention and are not to be taken are limiting thereof.<br>
EXAMPLES<br>
Materials and Methods<br>
Unless otherwise noted, standard techniques, vectors, control sequence elements, and other expression system elements known in the field of molecular biology are used for nucleic acid manipulation, transformation, and expression. Such standard techniques, vectors,<br><br>
and elements can be found, for example, in: Ausubel et al. (eds.), Current Protocols in Molecular Biology (1995) (John Wiley &amp; Sons); Sambrook, Fritsch, &amp; Maniatis (eds.), Molecular Cloning (1989) (Cold Spring Harbor Laboratory Press, NY); Berger &amp; Kimmel, Methods in Enzymology 152: Guide to Molecular Cloning Techniques (1987) (Academic Press); and Bukhari et al. (eds.), DNA Insertion Elements, Plasmids and Episomes (1977) (Cold Spring Harbor Laboratory Press, NY).<br>
Unless noted otherwise, PCR reactions were performed using a PTC225 thermocycler (MJ Research, South San Francisco, CA, USA) according to the protocol in Table 6.<br>
Table 6. PCR protocol<br>
(Table Removed)<br><br><br>
* (Stratagene, La Jolla, Ca. USA, hereinafter "Stratagene")<br>
Strains and Plasmids<br>
Strains utilized in the present Examples are described in Table 7. Plasmids were prepared using the QIAprep Spin Miniprep Kit from Qiagen; Valencia, CA) and transformed into Pseudomonas fluorescens DC283 or DC388 by electroporation of freshly-plated cells as follows. See for example, Enderle et al. (1998) "Electroporation of freshly plated Escherichia coli and Pseudomonas aeruginosa cells," Biotechniques 25: 954-958. Cells were scraped off an LB-agar plate with an inoculation loop, washed 3 times in 500 ^il Milli-Q water, and resuspended in 40 ul Milli-Q water. One jal of plasmid DNA was mixed with 40 ul of cells and transferred to an ice-cold electroporation cuvette. Electroporation was performed with a Gene Pulser II (Bio-Rad, Hercules, CA) with a field strength of 2.25 kV/cm. Cells were immediately transferred to 1 ml SOC medium (Invitrogen; Carlsbad, CA), incubated for 1-1.5 hours at 30°C, and subsequently spread on selective agar plates (M9 glucose plus uracil). Agar plates were incubated at 30°C.<br><br>
Table 7. Strains used in this report<br>
Strain	Genotype<br>
name<br>
DC283	ApyrF AproC AbenAB Iscr.lacf2'<br>
DC388	ApyrF AproC AbenAB lsc::lacfil AmtlDYZ<br>
DC389	ApyrF AproC AbenAB lsc::lacIQIAmtlDYZ+ pDOW1365-l<br>
DC390	ApyrF AproC AbenAB lsc::lacfi'AmtlDYZ + pDOW1365-l pDOW1339<br>
MB214	MB 101 lacl lacZYA<br>
Growth experiments<br>
All percentage concentrations for glycerol are expressed as v/v and those for glucose and mannitol as w/v. Growth experiments were performed in 1L bottom-baffled shake flasks containing 200 ml of mineral salts medium with 5 g/L yeast extract and trace elements. Standard medium contained 9.5% glycerol, which corresponds to a concentration of 1.3 M. For some experiments, the glycerol concentration was reduced to 2% (=274 mM) or 5% (=685 mM). When glucose was used as a carbon source, a solution of 25% glucose in shake-flask medium without glycerol was filter-sterilized and added to autoclaved medium to give a final concentration of 5% (=278 mM) or 12.5% (=694 mM) glucose. If needed, uracil (750 fo.g/ml) was added to the shake-flask medium. Four mL of an overnight culture of the strain grown in M9 glucose (plus uracil 250 u£/ml if required) was used as inoculum for shake flasks. Growth was monitored by measuring the optical density of the cultures at 600 nm at defined time intervals in 1-cm plastic cuvettes with an Eppendorf BioPhotometer (Eppendorf AG; Hamburg, Germany). Induction of the mannitol promoter was typically performed at 24 hours elapsed fermentation time by addition of 1% (=55 mM) mannitol from a 20% stock solution.<br>
Cultivation ofDC390 in 20-liter fermentors<br>
Strain DC390 was grown in standard aerated 20-L research fermentation vessels in a mineral salts medium described by Risenberg et al. (1991) "High cell density cultivation of Escherichia coli at controlled specific growth rate," J. Biotech. 20(1): 17-27, with slight modification. Cultures were grown for 72h at 32°C; the pH was maintained at 6.5 through the addition of aqueous ammonia. Agitation and sparged air flow rates were initially increased to control dissolved oxygen at a positive level but were fixed at maximum levels when these<br><br>
were reached. Glucose or glycerol was fed throughout the fermentation process to maintain a slight excess. The fed-batch high density fermentation process was divided into an initial growth phase of approximately 24h and a gene expression (induction) phase in which mannitol at 1% (w/v) concentration was added to initiate recombinant gene expression at a target cell density of 170 OD units at 575nm. Optical densities and cell fluorescence were monitored over time. Final cell densities varied depending on the carbon source.<br>
Fluorescence analyses<br>
Fluorescence of cultures was measured with a Spectramax Gemini microplate fluorimeter (Molecular Devices Corporation; Sunnyvale, CA) using the following settings: excitation 485 nm, emission 538 nm, bandpass filter 530 nm. Before analysis, the optical density (ODeoo) of samples was measured in 1-cm cuvettes as described above. Samples were diluted to ODeoo = 5 with shake-flask medium in Eppendorf tubes and then to ODeoo ~ 1 with water directly in the 96-well microplate. Diluted shake-flask medium (1:5) was used as a blank. The optical density at 600 nm of the samples in the 96-well plate was measured with a Spectramax Plus microplate spectrophotometer (Molecular Devices Corporation; Sunnyvale, CA), and fluorescence values were reported as RFUs - relative fluorescence units.<br>
For flow cytometry, culture samples were fixed by formaldehyde as follows: 1 mL of cells were pelleted and resuspended in phosphate-buffered saline (PBS) (pH 7.2). In the fume hood, 37% formaldehyde (stock concentration) was added to a concentration of 2 %. Cells were incubated for 5 min at room temperature, then centrifuged and resuspended in PBS for a total of three times. The ODeoo was adjusted to 0.01 and cells were stored at 4°C until analysis. All washes were collected as hazardous waste. Flow cytometry analyses were performed by Cytometry Research, San Diego, CA. Data were plotted using PCS Express2 software (DeNovo Software; Thornhill, Ontario, Canada).<br>
Analysis of mannitol, glycerol, and glucose concentration<br>
The concentration of mannitol, glycerol, and glucose in culture broth was determined with a Dionex ion chromatograph fitted with a CarboPac PA 10 analytical column and an ED50 pulsed amperometric detector (PAD) (Dionex Corporation; Sunnyvale, CA). Cells and culture broth were separated by centrifugation, and supernatant samples were stored at -20°C until used. Samples were diluted several thousand-fold with water to an estimated concentration of about 10 µg/ml and injected onto the column (injection volume 25 ul) with an auto-sampler. The column was equilibrated with 18 mM sodium hydroxide in water. At a<br><br>
flow rate of 1 ml/min, run conditions were as follows: 18 mM NaOH for 10 min; increase to 100 mM NaOH within 1 min; 100 mM NaOH for 1 min; 18 mM NaOH for 8 min to condition the column. Approximate retention times were as follows: Glycerol 2.2 min; mannitol 3.4 min; glucose 12.8 min. Concentrations were calculated based on standards of known concentrations.<br>
Example 1:    Identification of the mtl operon in various Pseudomonads<br>
A BLAST search of the P. fluorescens MB 101 genome in the ERGO database, using the DSM50106 mtlE sequence as described by Brunker et al. (1998) "Structure and function of the genes involved in mannitol, arabitol, and glucitol utilization from Pseudomonas fluorescens DSM50106," Gene 206(1):117-126, showed that MB101 has an mtl operon that is similar to the one described in DSM50106, starting with RXF01195 (mtlE). A gene alignment of the mtl operon in four different Pseudomonas strains, P. aeruginosa PAO-1, P. fluorescens DSM50106, P. fluorescens PfO-1, and P. fluorescens MB 101, is shown in Figure 3. There are two types of gene arrangement - one where the transcriptional activator gene, mtlR, is located directly upstream of the mtl operon, and one where mtlR is not present directly upstream of the mtl operon, which is the case for MB 101 and DSM50106.<br>
A sequence alignment between the regions directly upstream of the mtlE ATG start codon in MB101, DSM50106, P. fluorescens PfO-1, and P. aeruginosa PAO-1 is shown in Figure 4. Perfectly conserved sequences that match the -35 and -10 promoter region consensus sequence were found in the four strains, which supports the conclusion that the region serves as the promoter for the mtl operon. In addition, two perfectly conserved regions were found, one of 15 bp, upstream of the -35 region, and one of 14 bp downstream of the -10 region.<br>
Example 2:    Cloning of the promoter region for the mtl operon.<br>
A 299 bp region upstream of the MB101 mtlE gene (Seq.ID.No. 9, Table 3) was<br>
amplified	by	PCR	using	primers	mtlES	(5'-<br>
ATATGAGCTCGAGCGTGGGAACGATCAAGTGT-3' - SEQ. ID. No. 14) and mtlE4 (5'-ATATCCGCGGTTCCGCGCCCAGAGGGCTAC-3'-SEQ. ID. No. 15), (see Table 8) and cloned in front of the CopGFP reporter gene to make pDOWl 365-1 (Figure 5). The cloned region contains all of the sequences that are conserved between DSM50106 and MB 101 as well as sequences farther upstream. Two pDOW1365-l clones were sequenced; one (pDOWl365-1) had the same sequence as the genome reference (sequenced by The Dow<br><br>
Chemical Company), whereas the other had fourteen differences between the clone and the reference sequence. Since this large number of changes is unlikely to result solely from the error-prone PCR amplification process, we speculate that the DNA polymerase apparently switched templates among the four and a half direct repeats in the upstream region between cycles in the PCR amplification. The sequence of the repeats varies slightly, leading to sequence diversity in the PCR product.<br>
A promoter region of 126 bp from the same region (SEQ. ID. NO. 11, Table 3) was<br>
amplified	by	PCR	using	primers	mtlES	(5'-<br>
ATATGAGCTCTAAGCGTGCTCCCAAGGATTTGTCA-3'-SEQ. ID. No. 16) and mtlE4, and cloned in front of the CopGFP gene to make pDOW1369 (map not shown).<br>
The 203 bp region between mtlR and mtlE in P. fluorescens PfO-1 (SEQ. ID. No. 10) was cloned by PCR amplification using primers mtlE6 (SEQ. ID. No. 17) and mtlE7 (SEQ. ID. No. 18) (Table 8) and inserted upstream of the CopGFP reporter gene, resulting in pDOWl 370-7 (map not shown). The sequence of the cloned promoter region was the same as the reference sequence obtained from the ERGO database.<br>
A promoter region of 147 bp from the same region (SEQ. ID. NO. 12, Table 3) was<br>
amplified	by	PCR	using	primers	mtlE9	(5'-<br>
ATATGAGCTCACGAGTGCAAAAAAGTATCAGTAAG 3'-SEQ. ID. No. 19) and mtlE4 (Table 8), and cloned in front of the CopGFP gene to make pDOW1377 (map not shown)..<br>
A promoter region of 77 bp from the same region (SEQ. ID. NO. 13, Table 3) was amplified by PCR using primers mtlE9 and mtlEll (5'-ATATCCGCGGGTGCGTTGATTACAGCCTTCAAA 3'-SEQ. ID. No. 20) (Table 8), and cloned in front of the CopGFP gene to make pDOW1378 (map not shown)..<br>
To measure relative promoter activity on agar plates, DC283, a uracil and proline auxotrophic mutant of P. fluorescens MB 101, was transformed with the plasmids and streaked on a M9 plates containing 250 ug/mL uracil, 100 u,M FeCh (to suppress production of fluorescent pyoverdine which is produced by cells when iron-limited), and various carbon sources and incubated at 30°C. Colonies were examined by a blue light transilluminator (Dark Reader, Clare Chemical Research). Only the 299 bp MB 101 fragment in pDOW1365-l or pDOW1365-2 and the 147 bp fragment in pDOW1377 induced the CopGFP gene in the presence of mannitol (See Table 9 and Figure 6) - the other fragments did not express any CopGFP under any of the conditions tested. All of the fragments contained the -35 and -10 promoter regions. When mannitol was supplemented with 1% glycerol or glucose, the pDOW 1365-1 construct still produced CopGFP, which indicated that the promoter was not<br><br>
catabolite-repressed by either carbon source. The pDOWl365-1 promoter had no activity when 1% glycerol or 1% glucose was the sole carbon source. However, the promoter was active when glycerol was used at levels of 5% and 9.5% (Table 9), indicating that glycerol may act as a fortuitous inducer of the mtl promoter.<br>
Table 8. Oligonucleotides with engineered restriction sites are underlined.<br>
(Table Removed)<br><br><br>
pDOW 1361 control (no Cop gene)<br>
Example 3     Mannitol Induction<br>
Experiments to examine mannitol induction were performed in Pseudomonas fluorescens strain DC283, a uracil and proline auxotrophic mutant of P. fluorescens MB101. See, for example, JC Schneider et al. (2005) "Auxotrophic markers pyrF and proC can replace antibiotic markers on protein production plasmids in high cell density Pseudomonas fluorescens fermentation," Biotech Prog 21:343. Growth experiments with strain DC283/ pDOWl365-1 were initially conducted in standard shake flask medium containing 9.5% glycerol. After cultivation of the cells for about 24 hours, 1% mannitol (55 mM) was added to induce expression of CopGFP, which was followed over time by measuring the fluorescence of the culture. Interestingly, the cells showed a high degree of fluorescence before induction with mannitol, and cell fluorescence did not increase after induction (Figure 7). Further experiments revealed that the fluorescence level before induction and the inducibility of the mannitol promoter was correlated with the amount of glycerol in the medium. Increasing the glycerol concentration in the shake flask medium from 2% to 5% to 9.5% resulted in increasing levels of uninduced fluorescence and decreasing levels of induced fluorescence (Figure 8). CopGFP expression was only immediately inducible by mannitol in medium containing a 2% glycerol concentration. Using these low levels of glycerol is not satisfactory for shake flask analysis because all of the carbon source is consumed in 24 hours.<br>
Example 4:    Deletion of the wtfDYZ mannitol degradation genes in P. fluorescens.<br>
To prevent mannitol degradation by strain DC283 we deleted three genes (mtlDYZ) from the mannitol operon on the chromosome. The suicide vector pDOWl373-3 (Figure 9) was designed to delete the mtlDYZ genes by allele exchange mutagenesis. The mtlDYZ flanking regions were amplified using MB214 genomic DNA as a template and joined using the splicing by overlap extension technique, using primers manl (5'-GCCGACAAGGTAGTGGTGCTCAACA-3') (SEQ. ID. No. 21) and man2 (5'-TGCCCGCTCGCCTCACATCGGGAAATACTC-3') (SEQ. ID. No. 22) for the region upstream and man3 (GAGTATTTCCCGATGTGAGGCGAGCGGGCA-3') (SEQ. ID. No. 23) and man4 (5'-ACCGATAGTGCCACCGCTCTGGTAG-3') (SEQ. ID. No. 24) for the region downstream. Primers man2 and man3 contain complementary overlaps. The joined product was cloned into pDOW\26\ISrfl to make pDOW1373-3. (Figure 9). Colonies were screened with primer H3seq (5'-GTCCTGCAATTTCAGCCCGA-3') (SEQ. ID. No. 25) and man4; those with the expected 1170 bp product were sequenced to confirm that no PCR-generated mutations had been incorporated.<br>
Plasmid pDOW1373-3 was transformed into strain DC283, in which it cannot replicate independently and integrants into the genome were selected by tetracycline resistance. Transformants with a successful integration of the plasmid into the chromosome either upstream or downstream of the mtlDYZ gene cluster were identified by carrying out PCR amplification. A few upstream and downstream integrants were then cultivated in the absence of tetracycline to enrich for clones that had successfully looped-out the plasmid (with or without mtlDYZ genes) from the chromosome by a second cross-over. Losing the plasmid caused the clones to be resistant to 5'-fluoroorotic acid, which was used in agar cultivation medium to select for clones with the desired genotype. Clones that did not have the plasmid (and the mtlDYZ genes) integrated in the chromosome anymore had their uracil auxotrophy restored. PCR amplification using primers man5 (5'-TGTTCGACGAACCGCTGTCCA-3') (SEQ. ID. No. 26) and man6 (5'-TTCAATGGTCCCCCCGGTCATTTCATA-3') (SEQ. ID. No. 27), which lie outside of the region amplified for the allele exchange, resulted in PCR product of the expected size (1308 bp) in several isolated clones, which indicated that the plasmid plus the mtlDYZ genes were successfully deleted from the DC283 chromosome. PCR sequencing of the chromosomal region of interest confirmed that mtlDYZ genes were absent in strain DC283 ΔM/D7Z(=DC388).<br>
Strain DC388 was analyzed for growth and carbon metabolism in M9 mineral salts medium supplemented with uracil and proline (250µg/ml each) and 1% (w/v) glucose or 1% (w/v) mannitol as the sole source of carbon and energy. The mutant grew in medium with glucose to an OD600 of about 2.2 within 24 hours and glucose was depleted concomitantly. In contrast, only a minor increase in optical density was observed in medium with mannitol as the sole carbon source, and the mannitol concentration did not decrease over time (Figure 10). These results demonstrated that the mtlDYZ deletion mutant had the expected phenotype and could not degrade mannitol.<br>
Example 5 - Mannitol-induced CopGFP expression in the mf/DYZ deletion mutant under various growth conditions.<br>
Plasmid pDOWl365-1 (which carries the CopGFP gene controlled by the 299 bp mtl promoter) was transformed into strain DC283 AmtlDYZ. The resulting construct, strain DC389, was tested in shake flask medium with various concentrations of glycerol (2%, 5%, 9.5%). In the mannitol-degradation-defective strain, mannitol induced CopGFP at all glycerol concentrations (Figure 11), whereas in the wildtype strain, only cultures that were grown on 2% initial glycerol showed immediate induction (Figure 7). The amount of<br>
promoter activity was positively correlated with the carbon source concentration; the highest induction levels were achieved using 9.5% glycerol. The deletion removed the first steps in mannitol assimilation, therefore the induction of Pmtl by mannitol indicates that mannitol, and not a breakdown product, is the inducer of Pmtl.<br>
In a side-by-side comparison of the deletion mutant and the wildtype strain it was confirmed that the mutant strain was inducible by mannitol in medium with 9.5% glycerol, whereas the wildtype strain was not (Figure 12). Mannitol concentration stayed constant in the mutant strain but declined to zero in the wildtype strain (data not shown). The mutant grew to lower OD6oo which may be attributed to its inability to use mannitol as a carbon source and the higher expression of the CopGFP gene in the mutant, which may place a burden on cell growth. The mutant showed higher pre-induction levels of expression than the wildtype which may result from the inability of the mutant strain to degrade trace amounts of inducers present in the yeast extract.<br>
The mannitol promoter also achieved higher levels of induction when the mutant strain was grown on 10% glucose as a carbon source (Figure 12b). Glucose-grown cultures experienced a rapid loss in CopGFP activity after 48 hours, which is attributed to acidification of the medium and concomitant death and protein denaturation.<br>
Example 6: mtl promoter activity after induction with various mannitol concentrations.<br>
To induce CopGFP expression from the mtl promoter, a concentration of 1% (= 55 mM) mannitol was added to the culture medium in previous experiments. In this experiment, the mannitol concentration added at 24 hr to glucose-grown or glycerol-grown cultures was varied between 0.01% and 1% to determine if lower inducer concentrations could be used to induce expression. Fluorescence after induction increased with increasing concentration of mannitol (Figure 13). Mannitol at 0.01% was sufficient to cause an increase in promoter activity in both glycerol-grown and glucose-grown cells, although higher levels were reached at 1% mannitol.<br>
Example 7:     Glycerol induction of the mtl promoter.<br>
In shake flask experiments, it was observed that both the wildtype and the mutant strain showed background fluorescence before induction in glycerol-containing medium but not in glucose-containing medium.. Glycerol shares structural similarities with mannitol and it is suspected that glycerol could act as a fortuitous inducer of the mtl promoter if the<br>
concentration of glycerol is sufficiently high. The hypothesis was tested by performing shake flask experiments with strain DC283 ΔmtlDYZ/ pDOW1339 pDOW1365-l in medium with 12.5% glucose and adding glycerol instead of mannitol at various concentrations at the time of induction. Mannitol was added as an inducer to control flasks. Induction was performed at 12h EFT instead of 24h EFT to prevent interference from the pH shift that may occur at later time points in glucose-grown cultures.<br>
When the glycerol inducer concentration was 2% or below, the promoter activity was not detectably different from no additions. At higher glycerol concentrations (5% and 9.5%) fluorescence increased to 2 to 3 times above background levels (Figure 14) by 17 hr after induction. This result clearly demonstrates that glycerol can induce mil promoter activity (although at lower level than mannitol) which resulted in CopGFP expression and explains the background fluorescence observed in medium containing a high glycerol concentration.<br>
Example 8: Uniform induction of the mtl promoter at less than saturating inducer concentrations.<br>
A portion of the shake flask cultures from Examples 6 and 7 were analyzed by flow cytometry in order to determine whether sub-saturating levels of inducer created a bimodal "all-or-none" style of induction, or whether induction was uniform among all cells. In the cultures tested, with 0 to 1% mannitol as an inducer, or 0 to 9.5% glycerol as an inducer, in cultures using glucose as the carbon source, the distribution of fluorescence created a single, normal-shaped distribution (Figure 15), indicating that all cells were induced to a similar level within a culture.<br>
Example 9:    mtl promoter activity in 20-L fermentors.<br>
Strain DC390 (=DC283Awr/DFZ/pDOW1365-l pDOW1339) was cultivated in 20L-fermentors in mineral salts medium with either glycerol or corn syrup (glucose) as sole carbon source. Glycerol and glucose were spiked frequently into the medium at various time points according to a standardized feeding regimen that is dependent on the cells' oxygen uptake rate. The concentration of glycerol or glucose never exceeded 1% and 0.1%, respectively, and the pH of the culture medium was kept constant at 6.5. In medium with glycerol, the culture reached an optical density OD575 of about 250. The maximum optical density was slightly lower (OD575 about 230), when cells were grown with corn syrup (Figure 16a).<br>
At an optical density of about 170 at 20.5 hours elapsed fermentation time (EFT), fermentor cultures were induced with 1% (w/v) mannitol. As shown in (Figure 16b), cell fluorescence increased to about 3000 RFUs in medium with corn syrup and to about 1500 RFUs in medium with glycerol. Both background fluorescence, increase in fluorescence over time, and maximum RFUs were comparable to results obtained in shake flask experiments with 12.5% glucose (maximum 3500 RFUs) or 2% glycerol (maximum 1000 RFUs) (data not shown), i.e., the depressed levels were low. Peptide analysis by SDS-PAGE demonstrated the appearance of CopGFP (with an expected molecular size of 25 kDa) after induction with mannitol (data not shown). In both fermentors, expression of CopGFP among cells in the culture was uniformly induced, as assessed by flow cytometry (Figure 17).<br>
Example 10: Use of the mtl promoter to co-express folding modulators GrpE, DnaK and DnaJ to increase human growth hormone solubility.<br>
Human growth hormone (hGH), when expressed in P. fluorescens, accumulates almost entirely as insoluble inclusion bodies. Based on transcriptional profiling data, the expression of P. fluorescens folding modulators (FMs) DnaK and DnaJ was increased in strains after induction of expression of recombinant hGH and accumulation as inclusion bodies in the cytoplasm of P. fluorescens, compared to strains that did not express the target protein (see, for example, U.S. Provisional Application No. 60/591,489). We therefore engineered a strain with the putative operon encoding GrpE, DnaK and DnaJ under control of the mtl promoter on a plasmid in order to co-express the FMs with hGH.<br>
Using chromosomal DNA isolated from MB214 (DNeasy; Qiagen, Valencia, CA) as a template and primers RC199 (SEQ. ID. NO:28) and RC200 (SEQ. ID. NO:29), grpE dnaKJ were amplified using PfuTurbo (Stratagene, La Jolla, CA) as per the manufacturer's recommendations. The resulting PCR product (4 kb) was digested with Spel and Xbal (restriction sites underlined in the primers above) and ligated to pDOW2236 (a derivative of pDOWl306-6) (Schneider, J. C., A. F. Jenings, D. M. Mun, P. M. McGovern and L. C. Chew (2005). "Auxotrophic markers pyrF and proC can replace antibiotic markers on protein production plasmids in high-cell-density Pseudomonas fluorescens fermentation." Biotech Prog 21: 343-348) to create pDOW2240 containing the grpEdnaKJ operon under control of the tac promoter. pDOW2240 was digested with Spel and Hindlll and the resulting grpEdnaKJ-con.la.Mng 4.0 kb fragment was gel-purified using Qiaquick (Qiagen, Valencia, CA) and ligated to pDOW2247 (a derivative of pDOW1365-l) also digested with Spel and Hindlll. The resulting plasmid, pDOW3501, containing grpEdnaKJ under the control of the<br><br>
mannitol  promoter,  was  transformed  into  DCS88  by  selecting  on  M9glucose  plates<br>
supplemented with 250 µg/ml uracil.  Finally, pDOW1426 [ref?] was electroporated into the<br>
above strain (DC462) and selected on M9 glucose plates, resulting in strain DC463 with two<br>
inducible plasmids:     1)    pDOW1426 carrying PlaehGH and 2)    pDOW3501  carrying<br>
Vm\\grpEdnaKJ.<br>
Table 10. Oligonucleotides with engineered restriction sites are underlined.<br>
(Table Removed)<br>
Duplicate cultures of DC463 were grown in mineral salts medium and ODeoo was recorded for each culture at various time points. Induction was accomplished by addition of 0.1 mM IPTG for hGH and 0.5% mannitol for GrpEDnaKJ at varying times after inoculation. Samples were collected at 0, 4, 8, 24 and 48 hours after induction. At each time point, 20 ODeoo normalized in 1 mL was harvested, lysed using EasyLyse™ (Epicentre, Madison, WI) and separated into soluble and insoluble fractions by centrifugation at 14000 rpm for 30 minutes. Equal volumes of samples were combined with BioRad (Hercules, CA) 2x Laemmli buffer, heated at 95°C for 5 minutes with 30 j^L loaded onto a BioRad 15% Tris-HC1 Criterion gel using Ix Tris Glycine SDS running buffer (BioRad). The proteins were visualized with Simply Blue Safestain (Invitrogen, Carlsbad, CA) and hGH production was quantitated using Prediluted BSA standards (Pierce Biotechnology, Rockford, IL) loaded on the same gel. The resulting Coomassie stained gels were scanned using a Molecular Devices Personal Densitometer (Molecular Devices, Sunnyvale, CA) with analyses performed using ImageQuant and Excel. As shown in Table 11, co-expression of GrpE DnaKJ significantly increased the solubility of hGH, converting almost 100% of the target protein into the soluble fraction. Additional experiments repeating growth and induction of DC463 using the simultaneous addition of IPTG and mannitol closely mimicked the results shown here, with 50-100% of hGH found in the soluble fraction (data not shown) when co-overexpressed with GrpE DnaKJ, compared to 100% insoluble otherwise. These results were observed for either the simultaneous addition of IPTG and mannitol, or staggered addition, but cell density for cultures with early induction of GrpE DnaKJ did not grow as well, indicating that a high level of expression of these FMs is deleterious to cell health. Since lower levels of expression may be optimal for the least deleterious effect on cell metabolism and highest yield of soluble<br><br>
hGH; the control of these FM genes from a promoter that is regulated independently of the target protein will allow independent optimization of expression.<br>
Table 11. Quantitation of hGH resulting from from co-expression with GrpE, DnaKJ<br>
The values listed here are the average of duplicate flasks, shown in mg/ml based on the densitometry results shown in Figure x. Only hGH corresponding bands were quantitated in each lane. *EFT refers to elapsed fermentation time.<br>
(Table Removed)<br><br><br><br><br>
CLAIMS<br>
What is claimed is:<br>
1.	An expression system comprising at least one isolated nucleic acid comprising<br>
a mannitol promoter, wherein the nucleic acid sequence comprises a range of at least 299<br>
continuous nucleotides upstream of the MB101 mtlE gene.<br>
2.	An  isolated  nucleic  acid  construct  comprising  the  nucleic  sequence  of a<br>
Pseudomonas Fluorescens gene promoter wherein said nucleic acid sequence is selected<br>
from the group consisting of<br>
(a)	SEQ ID NO: 9 or SEQ ID NO 12;<br>
(b)	a nucleotide sequence having at least 90% sequence identity to the amino acid<br>
sequence set forth in SEQ ID NO: 9 or SEQ ID NO 12; and<br>
(c)	a nucleotide sequence that hybridizes to any of a) or b) under conditions of high<br>
stringency.<br><br>
3.	A recombinant nucleic acid construct comprising the promoter of Claim 2 operably<br>
linked to a gene of interest.<br>
4.	An expression system comprising a construct according to Claim 3.<br>
5.	A vector comprising the nucleic acid construct of Claim 2 wherein said nucleic<br>
construct is expressed in a Pseudomonas Fluorescens cell.<br>
6.	A Pseudomonas fluorescens cell for use in producing a recombinant nucleic acid<br>
comprising:<br>
at least one nucleic acid construct encoding a recombinant polypeptide operably linked to a carbon source inducible promoter selected from the group consisting of mannitol, glucitol, and arabitol-inducible promoters,<br>
wherein the bacterial cell has been genetically altered to inhibit the metabolism or degradation of a carbon source capable of inducing the carbon source inducible promoter.<br>
7.	The cell of claim 6, wherein the promoter is a mannitol-inducible promoter.<br><br>
8.	The cell of claim 7, wherein the promoter comprises the putative promoter of an<br>
endogenous bacterial mannitol operon.<br>
9.	The cell of claim 8, wherein the promoter comprises the putative promoter of the<br>
Pseudomonasfluorescens mtlEFGKDYZ operon.<br>
10.	The cell of claim 9, wherein the promoter further comprises a nucleic acid<br>
sequence that acts an mil activator protein binding region.<br>
11.	The cell of claim 10, wherein the promoter comprises the nucleic acid sequence<br>
selected from the group consisting of Seq. ID. Nos. 1-13.<br>
12.	The cell of claim 6, wherein the genetic alteration is a mutation or deletion of a<br>
gene required in the metabolism of mannitol, arabitol, or glucitol, or a derivative or analogue<br>
thereof.<br><br>
13.	The cell of claim 12, wherein the gene is selected from the mtl operon.<br>
14.	The cell of claim 13, wherein the gene is selected from the group consisting of<br>
mtlD, mtlY, and mtlZ, or a combination thereof.<br>
15.	A method for expressing a recombinant polypeptide in a Pseudomonas<br>
fluorescens cell comprising:<br>
a)	providing a host cell capable of expressing a recombinant polypeptide;<br>
b)	transfecting the host cell with at least one nucleic acid construct encoding a<br>
recombinant polypeptide operably linked to a carbon source inducible promoter<br>
selected from the group consisting of mannitol, glucitol, and arabitol-inducible<br>
promoters;<br>
c)	growing the host cell in a growth medium;<br>
d)	adding a carbon source to the growth medium in an amount capable of inducing the<br>
expression of the peptide of interest from the carbon source inducible promoter, and<br>
e)	expressing the recombinant polypeptide of interest by inducing the carbon source<br>
inducible promoter; and,<br>
f)	isolating the expressed recombinant polypeptide;<br><br>
wherein the bacterial cell has been genetically altered to inhibit the metabolism or degradation of a carbon source capable of inducing the carbon source inducible promoter.<br>
16.	The method of claim 15, wherein the promoter is a mannitol-inducible promoter.<br>
17.	The method of claim 16, wherein the promoter comprises the putative promoter<br>
of an endogenous bacterial mannitol operon.<br>
18.	The method of claim 17, wherein the promoter comprises the putative promoter of<br>
the Pseudomonasfluorescens mtlEFGKDYZ operon.<br>
19.	The method of claim 18, wherein the promoter further comprises a nucleic acid<br>
sequence that acts an mtl activator protein binding region.<br>
20.	The method of claim 15, wherein the promoter comprises the nucleic acid<br>
sequence selected from the group consisting of Seq. ID. Nos. 1-13.<br>
21.	The method of claim 15, wherein the genetic alteration is a mutation or deletion<br>
of a gene required in the metabolism of mannitol, arabitol, or glucitol, or a derivative or<br>
analogue thereof.<br>
22.	The method of claim 19, wherein the gene is selected from the mtl operon.<br>
23.	The method of claim 22, wherein the gene is selected from the group consisting<br>
of mtlD, mtlY, and mtlZ, or a combination thereof.<br>
24.	The method of Claim 15, wherein the bacterial cell has been genetically altered to<br>
inhibit the metabolism or degradation of mannitol, glucitol, or arabitol.</td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/viewdoc.aspx?id=/R5wNfrC5QabC5VqfMCQOQ==&amp;amp;loc=+mN2fYxnTC4l0fUd8W4CAA==" target="_blank" style="word-wrap:break-word;">http://ipindiaonline.gov.in/patentsearch/GrantedSearch/viewdoc.aspx?id=/R5wNfrC5QabC5VqfMCQOQ==&amp;amp;loc=+mN2fYxnTC4l0fUd8W4CAA==</a></p>
		<br>
		<div class="pull-left">
			<a href="271543-improved-process-for-the-production-of-nucleotide-sugars.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="271545-motorcycle.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>271544</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>9206/DELNP/2007</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>09/2016</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>26-Feb-2016</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>25-Feb-2016</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>29-Nov-2007</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>DOW GLOBAL TECHNOLOGIES LLC</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td>2040 Dow Center, Midland, MI 48674, USA</td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>CARRIE J. SCHNEIDER</td>
											<td>2621 32ND, STREET, SAN DIEGO, CA 92104, USA.</td>
										</tr>
										<tr>
											<td>2</td>
											<td>BETTINA ROSENER</td>
											<td>5001, REBEL ROAD, SAN DIEGO, CA 92117, USA.</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>C07H 21/04</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>PCT/US2006/021958</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td>2006-06-06</td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td>60/687,763</td>
									<td>2005-06-06</td>
								    <td>U.S.A.</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/271544-mannitol-induced-promoter-systems-in-bacterial-host-cells by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 00:59:46 GMT -->
</html>
