<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/271718-process-for-obtaining-optically-active-vicinal-diols-from-meso-epoxides by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 00:47:10 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 271718:&quot;PROCESS FOR OBTAINING OPTICALLY ACTIVE VICINAL DIOLS FROM MESO-EPOXIDES&quot;</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">&quot;PROCESS FOR OBTAINING OPTICALLY ACTIVE VICINAL DIOLS FROM MESO-EPOXIDES&quot;</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>The invention provides yeast strains, and polypeptides encoded by genes of such yeast strains, that have enantiospecific meso-epoxide hydrolase activity. The invention also features nucleic acid molecules encoding such polypeptides, vectors containing such nucleic acid molecules, and cells containing such vectors. Also embraced by the invention are methods for obtaining optically active vicinal diols from meso-epoxides.</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td>TECHNICAL FIELD<br>
This invention relates to biocatalytic reactions, and more particularly to the use of enantiomer selective hydrolases to obtain optically active epoxides and vicinal diols.<br>
BACKGROUND<br>
Optically active epoxides and vicinal diols are versatile fine chemical intermediates for use in the production of Pharmaceuticals, agrochemicals, ferro-electric liquid crystals and flavours and fragrances. Epoxides are highly reactive electrophiles because of the strain inherent in the three-membered ring and the electronegativity of the oxygen. Epoxides react readily with various O-, N-, S-, and C-nucleophiles, acids, bases, reducing and oxidizing agents, allowing the production to bifunctional molecules. Vicinal diols, employed as their highly reactive cyclic sulfites and sulfates, act like epoxide-like synthons with a broad range of nucleophiles. The possibility of double nucleophilic displacement reactions with amidines and azide, allow access to dihydroimidazole derivatives, aziridines, diamines and diazides.<br>
Desymmetrisation of meso-epoxides by enantioselective addition of nucleophiles gives access to bifuncional molecules in 100 % theoretical yield. For example, optically active diols prepared from meso epoxides which possess a C2-axis of symmetry such as trans,s-2,3-butanediol, trans-cyclopentane-l,2-diol, trans-cyclohexane-l,2-diol and hydrobenzoin have versatile applications as chiral intermediates, chiral auxiliaries or chiral ligands. In addtion, for example, enantiomerically pure trans cyclohexane-1,2-diol can be employed as chiral auxiliary in conjugate addition reactions (e.g. a,p-unsaturated esters), asymmetric alkylation via chiral acetals, auxiliary for enantioselective rearrangement and cyclopropanation of allyl ethers, and preparation of chiral phosphines and crown ethers.<br>
Epoxide hydrolases (EC 3.3.2.3) are hydrolytic enzymes that convert epoxides to vicinal diols by ring-opening of the epoxide with water. Epoxide hydrolases are present in mammals, plants, insects and microorganisms.<br>
SUMMARY<br>
2<br><br>
The invention is based in part on the surprising discovery by the inventors that certain microorganisms express epoxide hydrolases with high enantio selectivity. These microorganisms and the associated enantioselective meso-epoxide hydrolase (YEMH) polypeptides of the invention hydrolyse meso-epoxides in favour of a particular optically active vicinal diol enantiomer product. In this respect, the genomes of the microorganims encode a polypeptide having highly enantioselective epoxide hydrolase activity.<br>
Even though meso-epoxides have two chiral centers in the epoxide ring, the molecule has a plane of symmetry and the respective isomers can be superimposed on each other, and therefore these type of molecules are not optically active. Vicinal diols produced by hydrolysis of meso-epoxides can, with respect to their corresponding two chiral centers, be R,R; S,S; R,S; or S,R diols. The first two vicinal diols (R,R and S,S) are trans diols and are asymmetric molecules that cannot be superimposed on each other, and are thus optically active. On the other hand, the second two vicinal diols (R,S and S,R), are m-diols and because they (like the meso-epoxides) have a plane of symmetry, can be superimposed on each other. For this reason, the latter vicinal diols (R,S and S,R), which are actually the same compound, are referred to as meso-diols.<br>
Even though, as explained above, meso-epoxides are not optically active, and thus do not exist as optical enantiomers, reactions and enzymes (e.g., epoxide hydrolases) that effect the selective production of one optically active enantiomer of a vicinal diol over the other, are referred to herein as "enantioselective". Because the relevant reactions convert symmetrical meso-epoxidcs into asymmetrical vicinal diols, they are sometimes referred to herein as "desymmetrisations" or "desymmetrisation reactions."<br>
More specifically, the invention provides a process for obtaining an optically active vicinal diol, which process includes the steps of: providing a meso-epoxide (MEO); creating a reaction mixture by mixing with the weso-epoxide a polypeptide, or a functional fragment thereof, having enantioselective meso-epoxide hydrolase activity, the polypeptide being a polypeptide encoded by a gene of a yeast cell or a gene derived from a yeast cell incubating the reaction mixture; and recovering from the reaction mixture an enantiopure, or a substantially enantiopure, vicinal diol.<br>
Another aspect of the invention is a process for obtaining an optically active vicinal diol, which process includes the steps of: providing a meso-epoxide; creating a<br>
3<br><br>
reaction mixture by mixing with the meso-epoxide a cell comprising a nucleic acid encoding, and capable of expressing, a polypeptide having enantioselective meso-epoxide hydrolase activity, the polypeptide being a polypeptide encoded by a gene of a yeast cell; incubating the reaction mixture; and recovering from the reaction mixture an enantiopure, or a substantially enantiopure, vicinal diol.<br>
The following embodiments apply to both of the above processes. The cell can be a yeast cell. The polypeptide can be encoded by an endogenous gene of the cell or the cell can be a recombinant cell, the polypeptide being encoded by a nucleic acid sequence with which the cell is transformed. The nucleic acid sequence can be an exogenous nucleic acid sequence, a heterologous nucleic acid sequence, or a homologous nucleic acid sequence. The polypeptide can be a ftill-length yeast epoxide hydrolase or a functional fragment of a full length yeast epoxide hydrolase.<br>
Moreover both processes can be carried out at a pH from 5 to 10. They can be carried out at a temperature of 0°C to 70 °C. In the processes, the concentration of the styrene epoxide can be at least equal to the solubility of the styrene epoxide in water.<br>
In both processes, the mesoepoxide can be a-compound of the general formula (I), (II), (V), (VI) or (VII) and the vicinal diol produced by the process is a compound of the general formula (III), (IV), (VIII), (IX), or (X),<br><br>
4<br><br><br>
wherein,<br>
R is selected from the group consisting of a variably substituted straight-chain or branched alkyl group, a variably substituted straight-chain or branched alkenyl group, a variably substituted straight-chain or branched alkynyl group, a variably substituted cycloalkyl group as well as cycloalkenyl groups, a variably substituted aryl group, a variably substituted aryl alkyl group, a variably substituted heterocyclic group, a variably substituted straight-chain or branched alkoxy group, a variably substituted straight-chain or branched alkenyloxy group, a variably substituted aryloxy group, a variably substituted aryl alkyloxy group, a variably substituted alkylthio group, a variably substituted alkoxycarbonyl group, a variably substituted straight chain or branched alkylamino or alkenyl amino group, a variably substituted arylamino or arylalkylamino group, a variably substituted carbamoyl group, a variably substituted acyl group, and a<br>
5<br><br>
functional group.<br>
The processes can include adding to the reaction mixture water and at least one water-immiscible solvent, including, for example, toluene, 1,1,2-trichlorotrifluoroethane, methyl tert-buty\ ether, methyl isobutyl ketone, dibutyl-o-phtalate, aliphatic alcohols containing 6 to 9 carbon atoms, aliphatic hydrocarbons containing 6 to 16 carbon atoms, or tributyl phosphate. Alternatively, or in addition, the processes can include adding to tltie reaction mixture water and at least one water-miscible organic solvent, for example, acetone, methanol, ethanol, propanol, isopropanol, acetonitrile, dimethylsulfoxide, N,N-dimethylformamide, or A^-methylpyrrolidine. In addition, or alternatively, one or more surfactants, one or more cyclodextrins, or one or more phase-transfer catalysts can be added to the reaction mixtures. Both processes can include stopping the reaction when one enantiomer of the vicinal diol is in excess compared to the other enantiomer of the vicinal diol. Furthermore, the processes can include recovering continuously during the reaction the optically active vicinal diol produced by the reaction directly from the reaction mixture.<br>
In both processes the yeast cell can be of one of the following exemplary genera: Arxula, Brettanomyces, Bullera, Bulleromyces, Candida, Cryptococcus, Debaryomyces, Dekkera, Exophiala, Geotrichum, Hormonema, Jssatchenkia, Kluyveromyces, Lipomyces, Mastigomyces, Myxozyma, Pichia, Rhodosporidium, Rhodotorula, Sporidiobolus, Sporobolomyces, Trichosporon, Wingea, or Yarrowia<br>
Moreover, in the processes, the yeast cell can be of one of the following exemplary species; Arxula adeninivorans, Arxula terrestris, Brettanomyces bruxellensis, Brettanomyces naardenensis, Brettanomyces anomalus, Brettanomyces species (e.g.NCYC 3151J, Bullera dendrophila, Bulleromyces albus, Candida albicans, Candida fabianii, Candida glabrata, Candida haemulonii, Candida intermedia, Candida magnoliaeCandida parapsilosis, Candida rugosa, Candida tenuis, Candida tropicalis, Candida famata, Candida kruisei, Candida sp. (new) rel to C. sorbophila, Cryptococcus albidus, Cryptococcus amylolentus, Cryptococcus bhutanensis, Cryptococcus curvatus, Cryptococcus gastricus, Cryptococcus humicola, Cryptococcus hungaricus, Cryptococcus laurentii, Cryptococcus luteolus, Cryptococcus macerans, Cryptococcus podzolicus, Cryptococcus terreus, Cryptococcus macerans, Debaryomyces hansenii,<br>
6<br><br>
Dekkera anomala, Exophiala dermatitidis, Geotrichum species (e.g. UOFS Y-0111), Hormonema species (e.g. NCYC 3171), Issatchenkia occidentalism Kluyveromyces marxianus, Lipomyces species (e.g.UOFS Y-2159), Lipomyces tetrasporus, Mastigomyces philipporiU Myxozyma melibiosi, Pichia anomala, Pichia finlandica, Pichia guillermondii, Pichia haplophila, Rhodosporidium lusitaniae, Rhodosporidium paludigenum, Rhodosporidium sphaerocarpum, Rhodosporidium toruloides, Rhodosporidium paludigenum,.Rhodotorula araucariae, Rhodotorula glutinis, Rhodotorula minuta, Rhodotorula minuta var. minuta, Rhodotorula mucilaginosa, Rhodotorulaphilyla, Rhodotorula rubra, Rhodotorula species (e.g. UOFS Y-2042), Rhodotorula species (e.g. UOFS Y-0448), Rhodotorula species (e.g. NCYC 3193), Rhodotorula species (e.g. UOFS Y-0139), Rhodotorula secies (e.g. UOFS Y-0560), Rhodotorula aurantiaca, Rhodotorula species (e.g. NCYC 3224), Rhodotorula sp. "mucilaginosa", Sporidiobolus salmonicolor, Sporobolomyces holsaticus, Sporobolomyces roseus, Sporobolomyces tsugae, Trichosporon beigelii, Trichosporon cutaneum var. cutaneum, Trichosporon delbrueckii, Trichosporon jirdvecii, Trichosporon mucoides, Trichosporon ovoides, Trichosporon pullulans, Trichosporon species (e.g. UOFS Y-0861), Trichosporon species (e.g. UOFS Y-1615), Trichosporon species (e.g. UOFS Y-0451), Trichosporon species (e.g. NCYC 3212), Trichosporon species (e.g. UOFS Y-0449,), Trichosporon species (e.g. NCYC 3211), Trichosporon species (e.g. UOFS Y-2113), Trichosporon species (e.g. NCYC 3210), Trichosporon moniliiforme, Trichosporon montevideense, Wingea robertsiae, or Yarrowia lipolytica.<br>
The yeast cell can also be of any of the other genera, species, or strains disclosed herein.<br>
Another aspect of the invention is a method for producing a polypeptide, which process includes the steps of: providing a cell comprising a nucleic acid encoding and capable of expressing a polypeptide that has enantioselective we^-epoxide hydrolase activity; culturing the cell; and recovering the polypeptide from the culture. Recovering the polypeptide from the culture includes, for example, recovering it from the medium in which the cell was cultured or recovering it from the cell/w se. The cell can be a yeast cell. The polypeptide can be encoded by an endogenous gene of the cell or the cell can be a recombinant cell, the polypeptide being encoded by a nucleic acid sequence with<br>
7<br><br>
which the cell is transformed. The nucleic acid sequence can be an exogenous nucleic acid sequence, a heterologous nucleic acid sequence, or a homologous nucleic acid sequence. The polypeptide can be a full-length yeast epoxide hydrolase or a functional fragment of a full-length yeast epoxide hydrolase. The cell can be of any of the yeast genera, species, or strains disclosed herein or any recombinant cell disclosed herein.<br>
The invention also features a crude or pure enzyme preparation which includes an isolated polypeptide having enantioselective meso-epoxide hydrolase activity. The polypeptide can be one encoded by any of the yeast genera, species, or strains disclosed herein or one encoded by a recombinant cell.<br>
In another aspect, the invention features a substantially pure culture of cells, a substantial number of which comprise a nucleic acid encoding, and are capable of expressing, a polypeptide having enantioselective meso-epoxide hydrolase activity. The cells can be recombinant cells or cells of any of the yeast genera, species, or strains disclosed herein.<br>
Another embodiment of the invention is an isolated cell, the cell comprising a nucleic acid encoding a polypeptide having enantioselective meso-epoxide hydrolase activity, the cell being capable of expressing the polypeptide. The cell can be any of those disclosed herein.<br>
The invention also features an isolated DNA that includes: (a) a nucleic acid sequence that encodes a polypeptide that has enantioselective meso-epoxide hydrolase activity and that hybridizes under highly stringent conditions to the complement of a sequence that can be SEQ ID NO: 10, 11,12, 13, 14,15, 16,17, or 18; or (b) the complement of the nucleic acid sequence. The nucleic acid sequence can encode a polypeptide that includes an amino acid sequence that can be SEQ ID NO: 1, 2, 3, 4, 5, 6, 7, 8, or 9. The nucleic acid sequence can be, for example, one of those with SEQ ID NO: 10, 11, 12, 13,14, 15,16, 17, or 18.<br>
Also provided by the invention is an isolated DNA that includes: (a) a nucleic acid sequence that is at least 55% identical to a sequence that can be SEQ ID NO: 10, 11, 12,13, 14, 15,16,17, or 18; or (b) the complement of the nucleic acid sequence, the nucleic acid sequence encoding a polypeptide that has enantioselective meso-epoxidc hydrolase activity.<br>
8<br><br>
Another aspect of the invention is an isolated DNA that includes: (a) a nucleic acid sequence that encodes a polypeptide consisting of an amino acid sequence that is at least 55% identical to a sequence that can be SEQ ID NO: 1, 2, 3, 4, 5, 6, 7, 8, or 9; or (b) the complement of the nucleic acid sequence, the polypeptide having enantioselective meso-epoxide hydrolase activity.<br>
Also included are vectors (e.g., those in which the coding sequence is operably linked to a transcriptional regulatory element) containing any of the above DNAs and cells (e.g., eukaryotic or prokaryotic cells) containing such vectors.<br>
Also provided by the invention is an isolated polypeptide encoded by any of the above DNAs. The polypeptide can include an amino acid sequence that is at least 55% identical to SEQ ID NO: 1, 2, 3, 4, 5, 6, 7, 8, or 9, the polypeptide having enantioselective styrene epoxide hydrolase activity. The polypeptide can also include:<br>
(a) a sequence that can be SEQ ID NO: 1, 2, 3, 4, 5, 6, 7, 8, or 9, or a functional fragment of the sequence; or (b) the sequence of (a), but with no more than five conservative substitutions, the polypeptide having enantioselective meso-epoxide hydrolase activity.<br>
In another embodiment the invention features an isolated antibody (e.g., a polyclonal or a monoclonal antibody) that binds to any of the above-described polypeptides.<br>
The term "exogenous" as used herein with reference to nucleic acid and a particular host cell refers to any nucleic acid that does not occur in (and cannot be obtained from) that particular cell as found in nature. Thus, a non-naturally-occurring nucleic acid is considered to be exogenous to a host cell once introduced into the host cell. It is important to note that non-naturally-occurring nucleic acids can contain nucleic acid subsequences or fragments of nucleic acid sequences that are found in nature provided the nucleic acid as a whole does not exist in nature. For example, a nucleic acid molecule containing a genomic DNA sequence within an expression vector is non-naturally-occurring nucleic acid, and thus is exogenous to a host cell once introduced into the host cell, since that nucleic acid molecule as a whole (genomic DNA plus vector DNA) does not exist in nature. Thus, any vector, autonomously replicating plasmid, or virus (e.g., retrovirus, adenovirus, or herpes virus) that as a whole does not exist in nature<br>
9<br><br>
is considered to be non-naturally-occurring nucleic acid. It follows that genomic DNA fragments produced by PCR or restriction endonuclease treatment as well as cDNAs are considered to be non-naturally-occurring nucleic acid since they exist as separate molecules not found in nature. It also follows that any nucleic acid containing a promoter sequence and polypeptide-encoding sequence (e.g., cDNA or genomic DNA) in an arriangement not found in nature is non-naturally-occurring nucleic acid. Nucleic acid that is naturally-occurring can be exogenous to a particular cell. For example, an entire chromosome isolated from a cell of yeast x is an exogenous nucleic acid with respect to a cell of yeast y once that chromosome is introduced into a cell of yeast y.<br>
It will be clear from the above that "exogenous" nucleic acids can be "homologous" or "heterologous" nucleic acids. As used herein, "homologous" nucleic acids are those that are derived from a cell of the same species as the host cell and "heterologous" nucleic acids are those that are derived from a species other than that of the host cell.<br>
In contrast, the term "endogenous" as used herein with reference to nucleic acids or genes and a particular cell refers to any nucleic acid or gene that does occur in (and can be obtained from) that particular cell as found in nature.<br>
The MEO used by the methods of the invention can be a compound of the general^ formula (I) or (II) or (V) or (VI) or (VII) and the vicinal diol produced by the process can be a compound of the general formula (III), or (IV), or (VIII) or (IX) or (X),<br><br>
10<br><br>
wherein: <br>
R can be selected from the group consisting of a variably substituted straight-chain or branched alkyl group, a variably substituted straight-chain or branched alkenyl group, a variably substituted straight-chain or branched alkynyl group, a variably substituted cycloalkyl group as well as cycloalkenyl groups, a variably substituted aryl group, a variably substituted aryl alkyl group, a variably substituted heterocyclic group, a variably substituted straight-chain or branched alkoxy group, a variably substituted straight-chain or branched alkenyloxy group, a variably substituted aryloxy group, a variably substituted aryl alkyloxy group, a variably substituted alkylthio group, a variably substituted alkoxycarbonyl group, a variably substituted straight chain or branched alkylamino or alkenyl amino group, a variably substituted arylamino or arylalkylamino group, a variably substituted carbamoyl group, a variably substituted acyl group, and a functional group.<br>
11<br><br>
R can be present as a carbocycle or a heterocycle having 4 to 7 carbon atoms.<br>
In an alternative embodiment, in compounds presented by general formulae (VI), (VII), (IX), and (X 'ten'), X can be in a 5-10 membered heterocyclic group containing nitrogen, oxygen, or sulfur atoms.<br>
The cyclic meso epoxides presented by the general formula II can contain a saturated or unsaturated (single or more than one unsaturated bond) cycloalkyl group that can be symmetrically substituted as further exemplified in Fig 1.<br>
The heterocyclic meso-epoxides presented by general formula VI containing one or more heteroatoms such as O, N or S in the cycloalkyl ring, as well as substitutents on the ring, are further exemplified in Fig 2. .<br>
The alkyl group can be a straight chain or branched alkyl group with 1 to 12 carbon atoms. The alkyl group can be selected from the group consisting of: methyl-; ethyl-; propyl-; isopropyl-; butyl-; isobutyl-; s-butyl-; t-butyl-; pent-1-yl-; pent-2-yl-; pent-3-yl-; 2-methylbut-l-yl-; 3-methylbut-l-yl-; 2-methylbut-2-yl-; 3-methylbut-2-yl-; hex-l-yl-;hex-2-yl-; hex-3-yl-; 1-methylpent-l-yl-; 2-methylpent-l-yl-; 3-methylpent-l-yl-; 2-methylpent-2-yls 3-methylpent-2-yl-; 4-methylpent-2-yl-; 2-methylpent-3-yl-; 3-methylpent-3-yl-; 2-ethylbut-l-yl-; hept-1-yl-; hept-2-yl-; hept-3-yl-; hept-4-yl-; 1-methylhex-1-yl-; 2-methylhex-l-yl-; 3-methylhex-l-yl-; 4-methylhex-l-yl-; 5-methylhex-1-yl-; 2-methylhex-2-yl-; 3-methylhex-2-yl-; 4-methylhex-2-yl-; 5-methylhex-2-yl-; 2-methylhex-3-yl-; 3-methylhex-3-yl-; 4-methylhex-3-yl-; 5-methylhex-3-yl-; 2-methylhex-4-yl-; 1,1-dimethylpent-l-yl-; 1,2-dimethylpent-l-yl-; 1,3-dimethylpent-l-yl-; 1,4-dimethylpent-1-yl-; 2,2-dimethylpent-l-yl-; 2,3-dimethylpent-l-yl-; 2,4-dimethylpent-l-yl-; 2,5-dimethylpent-l-yl-; 3,3-dimethylpent-l-yls 3,4-dimethylpent-l-yl-; 3,5-dimethylpent-1-yl-; 4,4-dimethylpent-l-yl-; 4,5-dimethylpent-l-yl-; 5,5-dimethylpent-l-yl-; 2,2-dimethylpent-2-yl-; 2,3-dimethylpent-2-yl-; 2,4-dimethylpent-2-yl-; 3,3-dimethylpent-2-yl-; 3,4-dimethylpent-2-yl-; 2,2-dimethylpent-3-yl-; 2,3-dimethylpent-3-yl-; 2,4-dimethylpent-3-yl-; 2,2-dimethylpent-4-yl-; 2-ethylpent-l-yl-; 3-ethylpent-l-yl-; 1,1,2-trimethylbut-l-yl-; 1,2,2-trimethylbut-l-yl-; l,2s3-trimethylbut-l-yl-; 2,2,3-trimethylbut-1-yl-; 2,3,3-trimethylbut-l-yl-; 2,3,3-but-2-yl-; 2-isopropylbut-l-yl-; 2-isopropylbut-2-yl-; oct-l-yl-; oct-2-yl-; oct-3-yl-; oct-4-yl-; 2-methylhept-l-yl-; 3-metliylhept-1-yl-; 4-methylhept-l-yl-; 5-methylhept-l-yl-; 6-methylhept-l-yl-; 2-<br>
12<br><br>
methylhept-2-yls 3-methylhept-2-yl-; 4-methylhept-2-yl-; 5-methylhept-2-yl-; 6-methylhept-2-yl-; 2-methylhept-3-yl-; 3-methylhept-3-yl-; 4-methylhept-3-yls 5-methylhept-3-yl-; 6-methylhept-3-yl-; 2-methylhept-4-yl-; 3-methylhept-4-yl-; 4-methylhept-4-yl-; 2,2-dimethylhex-l-yl-; 2,3-dimethylhex-l-yl-; 2,4-dimethylhex-l-yl-; 2,5-dimethylhex-l-yl-; 3,3-dimethylhex-l-yl-; 3,4-dimethylhex-l-yl-; 3,5-dimethylhex-l-yl-; 4,4-dimethyIhex-l-yls 4,5-dimethylhex-l-yl-; 5,5-dimethylhex-l-yl-; 2,3-dimethylhex-2-yl-; 2,4-dimethylhex-2-yls 2,5:dimethylhex-2-yl-; 3,3-dimethylhex-2-yl-; 3,4-dimethylhex-2-yl-; 3?5-dimethylhex-2-yl-; 4,4-dimethylhex-2-yl-; 4,5-dimethylhex-2-yl-; 5}5-dimethylhex-2-yls 2,2-dimethylhex-3-yl-; 2?3-dimethylhex-3-yl-; 2,4-dimethylhex-3-yl-; 2?5-dimethylhex-3-yl-; 3,3-dimethylhex-3-yl-; 3s4-dimethylhex-3-yl-; 3,5-dimethylhex-S-yl-; 4?4-dimethylhex-3-yl-; 4,5-dimethylhex-3-yl-; 5,5-dimethylhex-3-yl-; 2,2,3-trimethylpent-l-yl-; 2,2,4-trimethylpent-l-yl-; 2s3,3-trimethylpenM-yl-; 2,3,4-trimethylpent-1-yl-; 3,3,4-trimethylpent-l-yl-; 3,4,4-trimethylpent-l-yl-; 2,4,4-trimethylpent-1-yl-; 2,3,3-trimethylpent-2-yl-; 2,3,4-trimethylpent-2-yl-; 3,3,4-trimethylpent-2-yl-; 3,4,4-trimethylpent-2-yl-; 2,4,4-trimethylpent-2-yl-; 2,2,3-trimethyIpent-3-yI-; 2-methyl-3-ethylpen-l-yl-; 3-ethyl-3-methylpent-l-yl-; 3-ethyl-4-methylpent-1-yl-; (3-methylhex-3-yl)methyl-; (4-methylhex-3-yl)methyl-; (5-methylhex-3-yl)methyl-; (2-methylhex-2-yl)methyl-; 2-methyl-3-ethylpent-2-yl-; 3-ethyl-3-methylpent-2-yl-; 3-ethyI-4-methylpent-2-yl-; 2-methyl-2-ethylpent-3-yl-; 2-methyl-3-ethylpent-3-yl-; 2s2?3,3-tetramethylbut-l-yl-; 2-ethyl-3s3-dimethylbut-2-yl; 2-isopropyl-3-methylbut-2-yl-; (3-ethylpent-3-yl)methyl-; (2,3-dimethylpent-3-yl)methyl-; (2,4-dimethylpent-3-yl)methyl-; non-l-yl-; non-2-yI-; non-3-yl-; non-4-yI-; non-5-yl-; 2-methyloct-1-yl; 3-methyloct-l-yl-; 4-methyloct-l-yl-; 5-methyloct-l-yl-; 6-methyloct-l-yl-; 7-methyloct-l-yl-; 2-methyloct-2-yl; 3-methyloct-2-yl-; 4-methyloct-2-yl-; 5-methyIoct-2-yl-; 6-methyloct-2-yl-; 7-methyloct-2-yl-; 2-methyloct-3-yl; 3-methyloct-3-yl-; 4-methyloct-3-yl-; 5-methyloct-3-yU; 6-methyloct-3-yl-; 7-methyloct-3-yl-; 2-methyloct-4-yl; 3-methyloct-4-yl-; 4-methyloct-4-yl-; 5-methyloct-4-yl-; 6-methyloct-4-yl-; 7-methyloct-4-yl-; 2,2-dimethylhept-l-yl-; 2,3-dimethylhept-l-yl-; 2,4-dimethylhept-l-yl-; 2,5-dimethylhept-l-yl-; 2,6-dime%lhept-l-yl-; 3,3-dime%lhept-l-yl-; 3,4-dimethylhept-1-yl-; 3,5-dimethylhept-l-yl-; 3,6-dimethylhept-l-yl-; 4,4-dimethyIhept-l-yl-; 4,5-dimethylhept-l-yl-; 4,6-dimethylhept-l-yl-; 5,5-dimethylhept-l-yl-; 5,6-<br>
13<br><br>
dimethylhept-1-yl-; 6,6-dimethylhept-l-yl-; 2,3-dimethylhept-2-yl-; 2,4-dimethylhept-2-yl-; 2,5-dimethylhept-2-yl-; 2,6-dimethylhept-2-yl-; 3,3-dimethylhept-2-yl-; 3,4-dimethylhept-2-yl-; 3,5-dimethylhept-2-yl-; 3,6-dimethylhept-2-yl-; 4,4-dimethylhept-2-yl-; 4,5-dimethylhept-2-yl-; 4,6-dimethylhept-2-yl-; 5,5-dimethylhept-2-yl-; 5,6-dimethylhept-2-yl-; 6,6-dimethylhept-2-yl-; 2,2-dimethylhept-3-yl-; 2,3-dimethylhept-3-yl-; 2,4-dimethylhept-3-yl-; 2,5-dimethylhept-3-yl-; 2,6-dimethylhept-3-yl-; 3,4-dimethylhept-3-yl-; 3,5-dimethylhept-3-yl-; 3,6-dimethylhept-3-yl-; 4,4-dimethylhept-3-yl-; 4,5-dimethylhept-3-yl-; 4,6-dimethylhept-3-yl-; 5,5-dimethylhept-3-yl-; 5,6-1 diiftethylhept-3-yl-; 6,6-dimethylhept-3-yl-; 3-ethylhept-l-yl-; 3-ethylhept-l-yl-; 4-ethylhept-1-yl-; 3-ethylhept-2-yl-; 4-ethylhept-2-yl-; 5-ethylhept-2-yl-; 3-ethylhept-3-yl-; 4-ethylhept-3-yl-; 5-ethylhept-3-yl-; 3-ethylhept-4-yi-; 4-ethylhept-4-yl-; 2,2,3-trimethylhex-1-yl-; 2,2,4-trimethylhex-l-yl-; 2,2,5-trimethylhex-l-yl-; 2,3,3-trimethylhex-1-yl-; 2,3,4-trimethylhex-l-yl-; 2,3,5-trimethylhex-l-yl-; 2,4,4-trimethylhex-1-yl-; 2,4,5-trimethylhex-l-yl-; 2,5,5-trimethylhex-l-yl-; 3,3,4-trimethylhex-1-yl-; 3,3,5-trimethylhex-l-yl-; 4,4,5-trimethylhex-l-yl-; 4,5,5-trimethylhex-1-yl-; 2,3,3-trimethylhex-2-yl-; 2,3,4-trimethylhex-2-yl-; 2,3,5-trimethylhex-2-yl-; 2,4,4-trimethylhex-2-yl-; 2,4?5-trimethylhex-2-yl-; 2,5,5-trimethylhex-2-yl-; 3,3,4-trimethylhex-2-yI-; 3,3,5-trimethylhex-2-yl-; 3,4,4-trimethylhex-2-yl-; 3,4,5-trimethylhex-2-yl-; 3,5,5-trimethylhex-2-yl-; 4,4,5-trimethylhex-2-yl-; 4,5,5-trimethylhex-2-yl-; 2,2,3-trimethylhex-3-yl-; 2,2,4-trimethylhex-3-yl-; 2,2,5-trimethylhex-3-yl-; 2,3,4-trimethylhex-3-yl-; 2,3,5-trimethylhex-3-yl-; 2s4,4-trimethylhex-3-yl-; 2,4,5-trimethylhex-3-yl-; 2,5,5-trimethylhex-3-yl-; 4,4,5-trimethylhex-3-yl-; 4,5,5-trimethylhex-3-yl-; (2-methylhex-3-yl)methyl-; 3-ethyl-2-methylhexrl-yl-; 3-ethyl-3-methylhex-l-yl-; 3-ethyl-4-methyIhex-1-yl-; 3-ethyl-5-methylhex-l-yl-; 4-ethyl-2-methylhex-l-yl-; 4-ethyl-3-methylhex-l-yl-; 4-ethyl-4-methylhex-l-yl-; 4-ethyl-5-methylhex-l-yl-; (2-methylhex-l-yl)methyl-; (3-methylhex-l-yl)methyl-; (4-methylhex-l-yl)methyl-; (5-methylhex-l-yl)methyl-; (6-methylhex-l-yl)methyl-; 3-isopropylhex-l-yl-; 4-ethyl-5-methylhex-l-yl-; 3-ethyl-3-methylhex-2-yl-; 3-ethyl-4-methylhex-2-yl-; 3-ethyl-5-methylhex-2-yl-; 4-ethyl-2-methylhex-2-yl-; 4-ethyl-3-methylhex-2-yl-; 4-ethyl-4-methylhex-2-yl-; 4-ethyl-5-melhylhex-2-yl-; 3-isopropylhex-2-yl-; 4-ethyl-5-methylhex-2-yl-; 3-ethyl-2-methylhex-<br>
14<br><br>
3-yl-; 3-ethyl-4-rnethylhex-3-yl-; 3-ethyl-5-methylhex-3-yls 4-ethyl-2-methylhex-3-yl-; 4-ethyl-3-methylhex-3-yl-; 4-ethyl-4-methylhex-3-yl-; 4-ethyl-5-rnethylhex-3-yl-; 4-isopropyihex-1-yls 2,2,3,3-tetramethylpent-l-yl-; 2,2,3,4-tetramethylpenM-yl-; 2,2,4,4-tetramethylpem-l-yl-^^^^-tetramethylpent-l-yl-^^^^-tetramethylpent-l-yls 2,3,4,4-tetramethylpent-l-ylsS^^^^etramethylpent-l-yl-^^^^-tetramethylpent^-yl-<br>
; 2?3,4,4-tetramethyIpent-2-yl-; 2,3,4,4-tetramethyIpent-2-yl-; 3,3,4,4-tetramethylpent-2-yl-; 2,2,3,4-tetrarnethylpent-3-yl-; 2,2,4,4-tetramethylpent-3-yl-; 2,3,4,4-tetramethylpent-3-yl-; 2,3,4s4-tetramethylpent-3-yl-; (3-ethylhex-3-yl)methyl-; (4-ethylhex-3-yl)methyl-; (5-me%lhept-3-yl)methyls2,4-dimethyl-3-ethylpent-l-yls3?4-dimethyl-3-ethyIpent-l-<br>
yl-; 4,4-dimethyI-3-ethylpent-l-yl~; 2-ethyl-2-methylhex-l-yl-; 3-ethyl-2-methylhex-l-yl-; 4-ethyl-2-methylhex-l-yl-; 2-ethyl-3-methylhex~l-yl-; 2-ethyl-4-methylhex-l-yl-; 3-ethyl-3-methylhex-l-yl-; 3-ethyl-4-methylhex-l-yl-; 3-ethyl-5-methylhex-l-yl-; 4-ethyl-3-methylhex-l-yls 4-ethyI-4-methylhex-l-yl-; 4-ethyl-5-methylhex-l«yl-; and the like from dec-l-yl-; dec-2-yl-; dec-3-yl-; dec-4-yl-; dec-5-yl-; dec-6-yl-; undec-1-yl-; undec-2-yl-; undec-3-yl-; undec-4-yl-; undec-5-yl-; undec-6-yl-; undec-7-yl-; dodec-1-yl; dodec-2-yI; dodec-3-yI; dodec-4-yl; dodec-5-yI; dodec-6-yl groups. Preferably, the alkyl group is a straight chain or branched alkyl group with 1 to 8 carbon atoms.<br>
The aikenyl group can be a straight chain or branched alkenyl group having 2-12 carbon atoms. The alkenyl group can be selected from the group consisting of: vinyl-; allyl-; a-methallyl-; p-methallyl-; 1-propenyl-; isopropenyl-; 1-butenyl-; 2-butenyl-; 3-butenyl; l-buten-2-yl-; l-buten-3-yls 1-methyl-l-propenyl-; 2-methyl-l-propenyl-; 1-pentenyl-; 2-pentenyl-; 3-pentenyl-; 4-pentenyl-; l-penten-2-yl-; 1-penten-3-yl-; 2-methyl- 1-butenyl-; 1-hexenyl-; 2-hexenyl-; 3-hexenyl-; 4-hexenyl-; 5-hexenyl-; 1-heptenyl-; 2-heptenyl-; 3-heptenyl-; 4-heptenyl-; 5-heptenyl-; 6-heptenyl-; 1-octenyl-; 2-octenyj-; 3-octenyI-; 4-octenyi-; 5-octenyl-; 6-octenyl-; 7-octenyl-; 1-nonenyl-; 2-nonenyl-; 3-nonenyl-; 4^nonenyl-; 5-nonenyl-; 6-nonenyl-; 7-nonenyl-; 8-nonenyl-; 1-decenyl-; 2-decenyl-; 3-decenyl-; 4-decenyl-; 5-decenyl-; 6-decenyl-; 7-decenyl-; 8-decenyl-; 9-decenyl-; 1-undecenyl; 2-undecenyl; 3-undecenyl; 4-undecenyl; 5-vmdecmyl; 6-undecenyl; 7-undecenyl; 8-undecenyl; 9-undecenyl; 10-undecenyl; 1-dodecenyl; 2-dodecenyl; 3-dodecenyl; 4-dodecenyl; 5-dodecenyl; 6-dodecenyl; 7-dodecenyl; 8-dodecenyl; 9-dodecenyl; 10-dodecenyl; 11-dodecenyl groups; and related branched<br>
15<br><br>
isomers. Preferably, the group is a straight chain or branched alkenyl group with 2 to 8 carbon atoms.<br>
The alkynyl group can a straight chain or branched alkynyl group having 2-12 carbon atoms. The alkynyl group can be selected from the group consisting of: ethynyl-; 1-propynyl-; 2-propynyl-; 1-butynyl-; 2-butynyl-; 3-butynyl-; 1-pentynyl-; 2-pentynyl-; 3-pentynyl-; 4-pentynyl-; 1-hexynyl-; 2-hexynyl-; 3-hexynyl-; 4-hexynyl-; 5-hexynyl-; 1-heptynyl-; 2-heptynyl-; 3-heptynyl-; 4-heptynyl-; 5-heptynyl-; 6-heptynyl-; 1-octynyl-; 2-octynyl-; 3-octynyl-; 4-octynyl-; 5-octynyl-; 6-octynyl-; 7-octynyl-; 1-nonynyl-; 2-nonynyl-; 3-nonynyl-; 4-nonynyl-; 5-nonynyl-; 6-nonynyl-; 7-nonynyl-; 8-nonynyl-; 1-decynyl-; 2-decynyl-; 3-decynyl-; 4-decynyl-; 5-decynyl-; 6-decynyl-; 7-decynyl-; 8-decynyl-; 9-decynyI-; 1-undecynyl-; 2-undecynyls 3-undecynyl-; 4-undecynyl-; 5-undecynyl-; 6-undecynyl-; 7-undecynyl-; 8-undecynyl-; 9-undecynyl-; 10-undecynyl-; 1-dodecynyl-; 2-dodecynyl-; 3-dodecynyl-; 4-dodecynyl-; 5-dodecynyl-; 6-dodecynyl-; 7-dodecynyl-; 8-dodecynyl-; 9-dodecynyl-; 10-dodecynyl-; 11-dodecynyl- groups; and related branched isomers. Preferably, the alkynyl group is a straight chain or branched alkenyl group with 2 to 8 carbon atoms.<br>
The cycloalkyl group can be cycloalkyl groups with 3 to 10 carbon atoms. The cycloalkyl group can be selected from the group consisting of: cyclopropyl-; cyclobutyl-; cyclopentyl-; cyclohexyl-; cycloheptyl-; and cyclooctyl- groups. These groups can be variably substituted at any position(s) around the ring. Preferably, the cycloalkyl group is a cycloalkyl group with 5 to 8 carbon atoms.<br>
The cycloalkenyl group can be cycloalkenyl groups with 3 to 10 carbon atoms. The cycloalkenyl group can be selected from the group consisting of: cyclobutenyl-; cyclopentenyl-; cyclohexenyl-; cycloheptenyl-; and cyclooctenyl- groups. This group can variably be substituted at any position(s) around the ring. Preferably, the cycloalkenyl group is a cycloalkenyl group with 5 to 7 carbon atoms.<br>
The aryl group can be selected from the group consisting of phenyl; biphenyl; naphtyl; anthracenyl groups; and the like. Preferably; the aryl group is a phenyl group.<br>
The aryl alkyl group can be a group with 7 to 18 carbons. The aryl alkyl group can be selected from the group consisting of: benzyl-; 1 -methylbenzyl-; 2-phenylethyl-; 3-phenylpropyl-; 4-phenylbutyl-; 5-phenylpentyl-; 6-phenylhexyl-; 1-naphtylmethyl; and<br>
16<br><br>
2-(l-naphtyl)-ethyl groups; and the like. Preferably, the aryl alkyl group is an aryl alkyl group with 7 to 12 carbon atoms.<br>
The heterocyclic group represented by R in structure of general formula I, or in alternative embodiments of X in the compounds of general structure (VI), (VII), (IX), and (X), can include 5- to 10-membered heterocyclic groups containing, as heteroatom X, nitrogen, oxygen, or sulfur. The heterocyclic ring can be fused with a cyclic or aromatic ring having 3 to 7 carbon atoms such as: benzene; cyclopropyl; cyclobutane; cyclopentane; and cyclohexane ring systems. Preferably the heterocyclic ring has 5 or 6 carbon atoms. The heterocyclic ring can be selected from the group consisting of: furyl-; dihydrofuranyl-; tetrahydrofuranyl-; dioxolanyl-; oxazolyl-; dihydrooxazolyl-; oxazolidinyl-; isoxazolyl-; dihydroisoxazolyl-; isoxazolidinyl-; oxathiolanyl-; thienyl-; tetrahydrothienyl-; dithiolanyl-; thiazolyl-; dihydrothiazolyl-; thiazolidinyl-; isothiazolyl-; dihydroisothiazolyl-; isothiazolidinyl-; pyrrolyl-; dihydropyrrolyl-; pyrrolidinyl-; pyrazolyl-; dihydropyrazolyl-; pyrazolidinyl-; imidazolyl-; dihydroimidazolyl-; imidazolidinyl-; triazolyl-; dihydrotriazolyl-; triazolidinyl-; tetrazolyl-; dihydrotetrazolyl-; tetrazolidinyl-; pyridyl-; dihydropyridyl-; piperidinyl-; morpholinyl-; dioxanyl-; oxathianyl-; trioxanyl-; thiomorpholinyl-; pyridazinyl-; dihydropyridazinyl-; tetrahydropyridazinyl-; hexahydropyridazinyl-; pyrimidinyl-; dihydropyrimadinyl-; tetrahydropyrimadinyl-; hexahydropyrimadinyl-; pyrazinyl-; piperazinyl-; pyranyl-; dihydropyranyl-; tetrahydropyranyl-; thiopyranyl-; dihydrothiopyranyl-; tetrahydrothiopyranyl-; dithianyl-; purinyl-; pyrimidinyl-; pyrrolizinyl-; pyrrolizidinyl; indolyl-; dihydroindolyl-; isoindolyl-; indolizinyl-; indolizidinyl-; quinolyl-; dihydroquinolyl-; tetrahydroquinolyl-; isoquinolyl-; dihydroquinolyl-; tetrahydroquinolyl-; quinolizinyl-; quinolizidinyl-; phenanthrolinyl-; chromenyl-; chromanyl-; isochromenyl-; isochromanyl-; benzofuranyl-; carbazolyl- groups; and tetrahydrofurane; 1,3-dioxolane; tetrahydrotiophene; pyrrolidine; dioxolane; 2-imidazolidinethione; 2-imidazolidene; dioxane, piperazine; and dithione (see Fig. 2). In compounds of general formula (I) and (III), the heterocyclic ring can befused with a cyclic or aromatic ring having 3 to 7 carbon atoms such as:benzene; cyclopropyl; cyclobutane; cyclopentane; and cyclohexane ring systems .<br>
17<br><br>
The alkoxy group can be a straight chain or branched alkoxy group having 2-12 carbon atoms such as: methoxy; ethoxy; propyloxy; isopropyloxy; butyloxy; isobutyloxy; tert-butyloxy; pentyloxy; hexyloxy; heptyloxy; or octyloxy.<br>
The alkenyloxy group can be a straight chain or branched alkenyloxy group having 2-12 carbon atoms. The alkenyloxy group can be selected from the group consisting of: ethynyloxy-; 1-propynyloxy-; 2-propynyloxy-; 1-butynyloxy-; 2-butynyloxy-; 3-butynyloxy-; 1-pentynyloxy-; 2-pentynyloxy-; 3-pentynyloxy-; 4-pentynyloxy-; 1-hexynyloxy-; 2-hexynyloxy-; 3-hexynyloxy-; 4-hexynyloxy-; 5-hexynyloxy-; 1-heptynyloxy-; 2-heptynyloxy-; 3-heptynyloxy-; 4-heptynyloxy-; 5-heptynyloxy-; 6-heptynyloxy-; 1-octynyloxy-; 2-octynyloxy-; 3-octynyloxy-; 4-ociynyloxy-; 5-octynyloxy-; 6-octynyloxy-; 7-octynyloxy-; 1-nonynyloxy-; 2-nonynyloxy-; 3-nonynyl-oxy-; 4-nonynyloxy-; 5-nonynyloxy-; 6-nonynyloxy-; 7-nonynyloxy-; 8-nonynyloxy-; 1-decynyloxy-; 2-decynyloxy-; 3-decynyloxy-; 4-decynyloxy-; 5-decynyloxy-; 6-decynyloxy-; 7-decynyloxy-; 8-decynyloxy-; 9-decynyloxy-; 1-undecynyloxy-; 2-undecynyloxy-; 3-undecynyloxy-; 4-undecynyloxy-; 5-undecynyloxy-; 6-undecynyloxy-; 7-undecynyloxy-; 8-undecynyloxy-; 9-undecynyloxy-; 10-undecynyloxy-; 1-dodecynyloxy-; 2-dodecynyloxy-; 3-dodecynyloxy-; 4-dodecynyloxy-; 5-dodecynyloxy-; 6-dodecynyloxy-; 7-dodecynyloxy-; 8-dodecynyloxy-; 9-dodecynyloxy-; 10-dodecynyloxy-; 11-dodecynyloxy- groups; and related branched isomers. Preferably, the alkenyloxy group is a straight chain or branched alkenyloxy groups with 2 to 8 carbon atoms.<br>
The aryloxy group can be an aryloxy group, such as a phenoxy or naphtyloxy group (e.g.: phenoxy; 2-methylphenoxy; 3-methylphenoxy; 4-methylphenoxy; 2-allylphenoxy; 2-chlorophenoxy; 3-chlorophenoxy; 4-chlorophenoxy; 4-methoxyphenoxy; 2-allyloxyphenoxy; naphtyloxy; and the like). The group can optionally be substituted with an alkyl or alkenyl or alkoxy group having 1 to 4 carbon atoms or halogens.<br>
The aryl alkyloxy group can be benzyloxy or 2-phenylethyloxy.<br>
The alkyhimino group can be a straight chain or branched alkylamino group having 2-12 carbon atoms such as: methylamino; ethylamino; propylamino; isopropylamino; butylamino; isobutylamino; tert-butylamino; pentylamino; hexylamino; heptylamino; or octylamino.<br>
18<br><br>
The alkenyl amino group can be a straight chain or branched alkenylamino group having 2-12 carbon atoms. The alkenyl amino group can be selected from the group consisting of: ethynylamino-; 1-propynylamino-; 2-propynylamino-; 1-butynylamino-; 2-butynylamino-; 3-butynylamino-; 1-pentynylamino-; 2-pentynylamino-; 3-pentynylamino-; 4-pentynylamino-; 1-hexynylamino-; 2-hexynylamino-; 3-hexynylamino-; 4-hexynylamino-; 5-hexynylamino-; 1-heptynylamino-; 2-heptynylamino-; 3-heptynylamino-; 4-heptynylamino-; 5-heptynylamino-; 6-heptynylamino-; 1-octynylamino-; 2-octynylamino-; 3-octynylamino-; 4-octynylamino-; 5-octynylamino-; 6-octynylamino-; 7-octynylamino-; 1-nonynylamino-; 2-nonynylamino-; 3-nonynyl-amino; 4-nonynylamino-; 5-nonynylamino-; 6-nonynylamino-; 7-nonynylamino-; 8-nonynylamino-; 1-decynylamino-; 2-decynylamino-; 3-decynylamino-; 4-decynylamino-; 5-decynylamino-; 6-decynylamino-; 7-decynylamino-; 8-decynylamino-; 9-decynylamino-; 1-undecynylamino-; 2-undecynylamino-; 3-undecynylamino-; 4-undecynylamino-; 5-undecynylamino-; 6-undecynylamino-; 7-undecynylamino-; 8-undecynylamino-; 9-undecynylamino-; 10-undecynylamino-; 1-dodecynylamino-; 2-dodecynyIamino-; 3-dodecynylamino-; 4-dodecynylamino-; 5-dodecynylamino-; 6-dodecynylamino-; 7-dodecynylamino-; 8-dodecynylamino-; 9-dodecynylamino-; 10-dodecynylamino-; and 11-dodecynylamino- groups; and related branched isomers. Preferably, the alkenyl amino group is a straight chain or branched alkenylamino group with 2 to 8 carbon atoms.<br>
The arylamino group can be an arylamino group such as a phenylamino or naphtylamino group, optionally substituted with an alkyl or alkenyl or alkoxy group having 1 to 4 carbon atoms, or halogens. The arylamino group can be selected from the group consisting of: phenylamino; 2-methylphenylamino; 3-methylphenylamino; 4-methylphenylamino; 2-allylphenylamino; 2-chlorophenylamino; 3-chlorophenylamini; 4-chiorophenylamino; 4-methoxyphenylamino; 2-allyloxyphenylamino; naphtylamino; and the like.<br>
The arylalkylamino group can be benzylamino or 2-phenylethylamino.<br>
The alkylthio group can be an alkylthio group having 1 to 8 carbon atoms. The alkylthio group can be selected from the group consisting of: methylthio; ethylthio; propylthio; butylthio; isobutylthio; and pentylthio.<br>
19<br><br>
The alkenylthio group can be a straight chain or branched alkenylthio group having 1 to 8 carbon atoms. The alkenylthio group can be selected from the group consisting of: ethynylthio-; 1-propynylthio-; 2-propynylthio-; 1-butynylthio-; 2-butynylthio-; 3-butynylthio-; 1-pentynylathio-; 2-pentynylthio-; 3-pentynylthio-; 4-pentynylthio-; 1-hexynylthio-; 2-hexynylthio-; 3-hexynylthio-; 4-hexynylthio-; 5-hexynylthio-; and the like.<br>
The arylrthio group can be an alkenylthio group having 1 to 8 carbon atoms such, as a phenylthio or naphtylthio group, which can optionally be substituted with an alkyl or alkenyl or alkoxy group having 1 to 4 carbon atoms; and also halogens, e.g.: phenylthio; 2-methylphenylthio; 3-methylphenylthio; 4- methylphenylthio; 2-allylphenylthio; 2-chlorophenylthio; 3-chlorophenyIamini; 4-chlorophenylthio; 4-methoxyphenylthio; 2-allyloxyphenylthio; naphtylthio; and the like.<br>
The arylalkylthio group can be an alkenylthio group having 1 to 8 carbon atoms such as a benzylthio- group or a 2-phenylethylthio-group.<br>
The alkoxv car bony 1 group can be: methoxycarbonyl; ethoxycarbonyl; or the like.<br>
The substituted or unsubstituted carbamoyl group can be: carbamoyl; methylcarbamoyl; dimethylcarbampyl; diethylcarbamoyl; or the like.<br>
The acyl group can be an acyl group with 1 to 8 carbon atoms such as: formyl; acetyl; propionyl; or benzoyl groups; or the like.<br>
The alkyl, alkenyl, alkynyl, cycloalkyl, aryl, aryl alkyl, heterocyclic, alkoxy, alkenyloxy, aryloxy, aryl alkyloxy, alkylamino, alkenylamino, arylamino, arylalkylamino, alkylthio, alkenylthio, arylthio, arylalkylthio, alkoxycarbonyl, substituted and unsubstituted carbamoyl and acyl groups mentioned above can optionally be substituted. Examples of substituents include: halogens (F; Cl; Br; I); hydiroxyl groups; mercapto groups; carboxylates; nitro groups; cyano groups; substituted or unsubstituted amino groups (including amino, methylamino, dimethylamino, ethylamino, diethylamino, and various protected amines such as tert-butoxycarbonyl- and arylsulfonamido groups); alkoxy groups (having 1 to 8 carbon atoms such as methoxy, ethoxy, propyloxy, isopropyloxy, butyloxy, isobutyloxy, tert-butyloxy, pentyloxy, hexyloxy, heptyloxy or octyloxy); alkenyloxy groups (having 2 to 8 carbon atoms such as a vinyloxy; allyloxy;<br>
20<br><br>
3-butenyloxy or 5-hexenyloxy); aryloxy groups (such as a phenoxy or naphtyloxy group which can be optionally substituted with an alkyl or alkenyl or alkoxy group having 1 to 4 carbon atoms; and also halogens, e.g., phenoxy, 2-methylphenoxy, 3-methylphenoxy, 4-methylphenoxy, 2-allylphenoxy, 2-chlorophenoxy, 3-chlorophenoxy, 4-chlorophenoxy, 4-methoxyphenoxy, 2-allyloxyphenoxy, naphtyloxy, and the like); aryl alkyloxy groups (e.g., benzyloxy and 2-phenylethyloxy); alkylthio groups (having 1 to 8 carbon atoms such as methylthio, ethylthio, propylthio, butylthio, isobutylthio, pentylthio); alkoxycarbonyl groups (e.g. methoxycarbonyl, ethoxycarbonyl, and the like); substituted or unsubstituted carbamoyl group (e.g. carbamoyl, methylcarbamoyl, dimethylcarbamoyl, diethylcarbamoyl and the like); acyl groups (with 1 to 8 carbon atoms such as formyl, acetyl, propionyl, or benzoyl groups); and others.<br>
The above-mentioned cycloalkyl, cycloalkenyl, aryl, aryl alkyl, heterocyclic, alkoxy, alkenyloxy, aryloxy, aryl alkyloxy, alkylthio, and alkoxycarbonyl groups can also be substituted with alkyl groups having 1 to 10 carbon atoms, alkenyl groups with 2 to 10 carbon atoms, or haloalkyl groups with 1 to 10 carbon atoms in addition to the substituents specified above.<br>
The number of substituents can be one or more than one. The substituents can be the same or different.<br>
The R group can be a functional group. The functional group is selected from the group consisting of: halo, pseudohalo, hydroxyl, variably substituted mercapto, variably substituted sulfinyl, variably substituted sulfonyl, carboxylates, variably substituted amino, variably substituted amido, variably substituted ureido, variably substituted carbamoyl, and variably substituted urethano. Pseudohalo is nitro, cyano, azido, cyanato, isocyanato, or isothiocyanato.<br>
The R group can also be present as a saturated or unsaturated optionally substituted carbocycle or a heterocycle having 5 to 12 carbon atoms, such as: cyclopentane; cyclohexane; cyclohexene; cyclohexadiene; cycloheptane; cyclooctane; cyclononane; or cyclodecane.<br>
As used herein, a "variably substituted" group is a group that is unsubstituted or is substituted with one or more, in another embodiment one to five, in another embodiment one, two or three, substituents.<br>
21<br><br>
"n" in general formulae (II) and (IV) can be 2-8, i.e., 2, 3,4 5, 6, 7 or 8.<br>
"Polypeptide" and "protein" are used interchangeably and mean any peptide-linked chain of amino acids, regardless of length or post-translational modification. The invention also features YEMH polypeptides with conservative substitutions. Conservative substitutions typically include substitutions within the following groups: glycine and alanine; valine, isoleucine, and leucine; aspartic acid and glutamic acid; asparagine, glutamine, serine and threonine; lysine, histidine and arginine; and phenylalanine and tyrosine.<br>
The term "isolated" polypeptide or peptide fragment, as used herein, refers to a polypeptide or a peptide fragment which either has no naturally-occurring counterpart or has been separated or purified from components which naturally accompany it, e.g., microorganism cellular components such as yeast cell cellular components. Typically, the polypeptide or peptide fragment is considered "isolated" when it is at least 70%, by dry weight, free from the proteins and other naturally-occurring organic molecules with which it is naturally associated. Preferably, a preparation of a polypeptide (or peptide fragment thereof) of the invention is at least 80%, more preferably at least 90%, and most preferably at least 99%, by dry weight, the polypeptide (or the peptide fragment thereof), respectively, of the invention. Thus, for example, a preparation of polypeptide z is at least 80%, more preferably at least 90%, and most preferably at least 99%, by dry weight, polypeptide z. Since a polypeptide that is chemically synthesized is, by its nature, separated from the components that naturally accompany it, the synthetic polypeptide is "isolated."<br>
An isolated polypeptide (or peptide fragment) of the invention can be obtained, for example, by: extraction from a natural source (e.g., from yeast cells); expression of a recombinant nucleic acid encoding the polypeptide; or chemical synthesis. A polypeptide that is produced in a cellular system different from the source from which it naturally originates is "isolated," because it will necessarily be free of components which naturally accompany it. The degree of isolation or purity can be measured by any appropriate method, e.g., column chromatography, polyacrylamide gel electrophoresis, or HPLC analysis.<br>
An "isolated DNA" is either (1) a DNA that contains sequence not identical to<br>
22<br><br>
that of any naturally occurring sequence, or (2), in the context of a DNA with a naturally-occurring sequence (e.g., a cDNA or genomic DNA), a DNA free of at least one of the genes that flank the gene containing the DNA of interest in the genome of the organism in which the gene containing the DNA of interest naturally occurs. The term therefore includes a recornbinant DNA incorporated into a vector, into an autonomously replicating plasmid or virus, or into the genomic DNA of a prokaryote or eukaryote. The term also includes a separate molecule such as: a cDNA (e.g., SEQ ID NOs: 10,11,12,13, 14,15, 16,17, or 18) where the corresponding genomic DNA can have introns and therefore can have a different sequence; a genomic fragment that lacks at least one of the flanking genes; a fragment of cDNA or genomic DNA produced by polymerase chain reaction (PCR) and that lacks at least one of the flanking genes; a restriction fragment that lacks at least one of the flanking genes; a DNA encoding a non-naturally occurring protein such as a fusion protein, mutein, or fragment of a given protein; and a nucleic acid which is a degenerate variant of a cDNA or a naturally occurring nucleic acid. In addition, it includes a recombinant nucleotide sequence that is part of a hybrid gene, i.e., a gene encoding a non-naturally occurring fusion protein. Also included is a recombinant DNA that includes a portion of SEQ ID NOs: 10-18. It will be apparent from the foregoing that isolated DNA does not mean a DNA present among hundreds to millions of other DNA molecules within, for example, cDNA or genomic DNA libraries or genomic DNA restriction digests in, for example, a restriction digest reaction mixture or an electrophoretic gel slice.<br>
As used herein, a "functional fragment" of a YEMH polypeptide is a fragment of the polypeptide that is shorter than the full-length polypeptide and has at least 20% (e.g., at least: 30%; 40%; 50%; 60%; 70%; 80%; 90%; 95%; 98%; 99%; 100%, or more) of the ability of the full-length polypeptide to enantioselectively hydrolyse a MEO of interest. Fragments of interest can be made by either recombinant, synthetic, or proteolytic digestive methods and tested for their ability to enantioselectively hydrolyse a MEO.<br>
As used herein, "operably linked" means incorporated into a genetic construct so that an expression control sequence effectively controls expression of a coding sequence of interest.<br>
Unless otherwise defined, all technical and scientific terms used herein have the<br>
23<br><br>
same meaning as commonly understood by one of ordinary skill in the art to which this invention pertains. In case of conflict, the present document, including definitions, will control. Preferred methods and materials are described below, although methods and materials similar or equivalent to those described herein can be used in the practice or testing of the present invention. All publications, patent applications, patents and other references mentioned herein are incorporated by reference in their entirety. The materials, methods, and examples disclosed herein are illustrative only and not intended to be limiting.<br>
Other features and advantages of the invention, e.g., vicinal diols substantially enriched for one optical enantiomer, will be apparent from the following description, from the drawings and from the claims.<br>
DESCRIPTION OF DRAWINGS<br>
Fig. 1 is a depiction of the chemical structures of exemplary MEO represented by general formula (II) that are useful for the invention.<br>
Fig. 2 is a depiction of the chemical structures of exemplary MEO represented by general formula (VI) and (VII) that have heteroatoms in the cyclic ring and are useful for the invention.<br>
Fig. 3 is a schematic representation of desymmetrisation of five exemplary MEO (cis-2,3- epoxybutane, cyclohexene oxide, cyclopentene oxied, cis-epoxysuccinic acid and 3,4-epoxytetrahydrofuran) by epoxide hydrolases to the corresponding R,R-diols.<br>
Fig. 4 is a restriction map of the pYLHmA (pINA1291) expression vector. The positions of the hp4d promoter and LIP2 terminator and of unique restriction sites available for the insertion of coding sequences are indicated.<br>
Fig. 5 is a restriction map of the p YLTsA (pIN A3 313) expression vector. The positions of the TEF promoter and LIP2 terminator and of unique restriction sites available for the insertion of coding sequences are indicated.<br>
Figs. 6-8 (Samples 146-148) are line graphs showing the desymmetrisation of cis-2,3- epoxybutane by the indicated wild-type yeast strains. The left y-axis (and lines with shaded circle and triangle data points) in each graph show the changes in concentrations of the optically active vicinal diol products at the time points indicated on the x-axis and<br>
24<br><br>
the right y-axis (and lines with shaded diamond-shaped data points) in each graph show the enantiomeric excesses ("ee") of the formed optically active vicinal diol at the time points indicated on the x-axis. The biocatalyst loadings are indicated in brackets beside the strain names in each figure. The percentage biocatalyst loading refers to a percentage wet weight of yeast cells or an equivalent wet weight of lysed/broken yeast cells in the aqueous fraction of the reaction matrix. In all these instances the value of the percentage wet weight of biocatalyst is approximately five-fold the value of the equivalent dry weight of biocatalyst. The biocatalyst loading are indicated in a similar fashion in all the biotransformation reaction profile graphs shown in the figures described below.<br>
Figs. 9-12 (Samples 149-152) are line graphs showing the desymmetrisation of cis-2,3- epoxybutane by yeast host strains transformed with vectors expressing YEMH polypeptides of selected wild-type yeast strains. The left y-axis (and lines with shaded circle, square, and triangle data points) in each graph show the changes in concentrations of the meso-vicinal diol and optically active vicinal diol products at the time points indicated on the x-axis and the right y-axis (and lines with shaded diamond-shaped data points) in each graph show the enantiomeric excesses ("ee") of the formed optically active vicinal diol at the time points indicated on the x-axis.<br>
Figs. 13-15 (Samples 153-155) are line graphs showing the desymmetrisation of cyclohexene oxide by the indicated wild-type yeast strains. The left y-axis (and lines with shaded square, circle, and triangle data points) in each graph show the changes in concentrations of the cyclohexene oxide and the optically active vicinal diol products at the time points indicated on the x-axis and the right y-axis (and lines with shaded diamond-shaped data points) in each graph show the enantiomeric excesses ("ee") of the formed optically active vicinal diol at the time points indicated on the x-axis.<br>
Figs. 16-20 (Samples 156-160) are line graphs showing the desymmetrisation of cyclohexene oxide by yeast host strains transformed with vectors expressing YEMH polypeptides of selected wild-type yeast strains. The left y-axis (and lines with shaded square, circle, and triangle data points) in each graph show the changes in concentrations of the cyclohexene oxide and the optically active vicinal diol products at the time points indicated on the x-axis and the right y-axis (and lines with shaded diamond-shaped data points) in each graph show the enantiomeric excesses ("ee") of the formed optically<br>
25<br><br>
active vicinal diol at the time points indicated on the x-axis.<br>
Figs. 21-24 (Samples 161-164) are line graphs showing the desymmetrisation of cyclohexene oxide by whole (Figs 21 and 24) and lysed/broken (Figs. 22 and 23) yeast host cells transformed with vectors expressing three different YEMH polypeptides of selected wild-type yeast strains. The left y-axis (and lines with shaded square, circle, and triangle data points) in each graph show the changes in concentrations of the cyclohexene oxide and the optically active vicinal diol products at the time points indicated on the x-axis and the right y-axis (and lines with shaded diamond-shaped data points) in each graph show the enantiomeric excesses ("ee") of the formed optically active vicinal diol at the time points indicated on the x-axis.<br>
Figs. 25-31. (Samples 165-171) are line graphs showing the desymmetrisation of cyclopentene oxide by yeast host strains transformed with vectors expressing YEMH polypeptides of selected wild-type yeast strains. The left y-axis (and lines with shaded square, circle, and triangle data points) in each graph show the changes in concentrations of the cyclopentene oxide and the optically active vicinal diol products at the time points indicated on the x-axis and the right y-axis (and lines with shaded diamond-shaped data points) in each graph show the enantiomeric excesses ("ee") of the formed optically active vicinal diol at the time points indicated on the x-axis.<br>
Figs. 32 (Samples 183) is a line graph showing the desymmetrisation of cis-epoxysuccinic acid disodium salt by a yeast host strain transformed with a vector expressing YEMH polypeptides of a selected wild-type yeast strains. The left y-axis (and lines with shaded "square, and triangle data points) in each graph show the changes in concentrations of the optically active vicinal diol products at the time points indicated on the x-axis and the right y-axis (and lines with shaded diamond-shaped data points) in each graph show the enantiomeric excesses ("ee") of the formed optically active vicinal diol at the time points indicated on the x-axis.<br>
Figs. 33 (Samples 184) is a line graph showing the desymmetrisation of 3,4-epoxytetrahydrofuran by a yeast host strain transformed with a vector expressing YEMH polypeptides of a selected wild-type yeast strains. The left y-axis (and lines with shaded square, circle, and triangle data points) in each graph show the changes in concentrations of the cyclopentene oxide and the optically active vicinal diol products at the time points<br>
26<br><br>
indicated on the x-axis and the right y-axis (and lines with shaded diamond-shaped data points) in each graph show the enantiomeric excesses ("ee") of the formed optically active vicinal diol at the time points indicated on the x-axis.<br>
Fig. 34 is a depiction of the amino acid sequence (SEQ ID NO:1) of a YEMH polypeptide encoded by cDNA derived from a Rhodosporidium toruloides strain (assigned accession no. NCYC 3181).<br>
Fig. 35 is a depiction of the amino acid sequence (SEQ ID NO:2) of a YEMH polypeptide encoded by cDNA derived from a Rhodosporidium toruloides strain (assigned identification no. UOFS Y-0471).<br>
Fig. 36 is a depiction of the amino acid sequence (SEQ ID NO:3) of a YEMH polypeptide encoded by cDNA derived from a Rhodotorula araucariae strain (assigned accession no. NCYC 3183).<br>
Fig. 37 is a depiction of the amino acid sequence (SEQ ID NO:4) of a YEMH polypeptide encoded by cDNA derived from a Cryptococcus curvatus strain (assigned accession no. NCYC 3158).<br>
Fig. 38 is a depiction of the amino acid sequence (SEQ ID NO;5) of a YEMH polypeptide encoded by cDNA derived from a Rhodosporidium paludigenum strain (assigned accession no. NCYC 3179).<br>
Fig. 39 is a depiction of the amino acid sequence (SEQ ID NO:6) of a YEMH polypeptide encoded by a cDNA derived from a Rhodotorula mucilaginosa strain (assigned accession no. NCYC 3190).<br>
Fig. 40 is a depiction of the amino acid sequence (SEQ ID NO:7) of a YEMH polypeptide encoded by cDNA derived from a Yarrowia lipolytica strain (assigned accession no. NCYC 3229-2 (Polh-2)).<br>
Fig. 41 is a depiction of the amino acid sequence (SEQ ID NO:8) of a YEMH polypeptide encoded by cDNA derived from a Debaromyces hartseniijsirain (assigned<br>
accession no. NCYC 3167).<br>
.I<br>
Fig. 42 is a depiction of the amino acid sequence (SEQ ID NO:9) of a YEMH polypeptide encoded by cDNA derived from a Filobasidium neoformans var neoformans strain (assigned GenBank accession no. XM_568708 (#777)).<br>
Fig. 43 is a depiction of the nucleotide sequence (SEQ ID NO:10) of a YEMH<br>
27<br><br>
polypeptide-encoding cDNA derived from a Rhodosporidium toruloides strain (assigned accession no. NCYC 3181).<br>
Fig. 44 is a depiction of the nucleotide sequence (SEQ ID NO: 11) of a YEMH polypeptide-encoding cDNA derived from a Rhodosporidium toruloides strain (assigned identification no. UOFS Y-0471).<br>
Fig. 45 is a depiction of the nucleotide sequence (SEQ ID NO:12) of a YEMH polypeptide-encoding cDNA derived from a Rhodotorula araucariae strain (assigned accession no. NCYC 3183).<br>
Fig. 46 is a depiction of the nucleotide sequence (SEQ ID NO: 13) of a YEMH polypeptide-encoding by cDNA derived from a. Cryptococcus curvatus strain (assigned accession no. NCYC 3158).<br>
Fig. 47 is a depiction of the nucleotide sequence (SEQ ID NO: 14) of a YEMH polypeptide-encoding cDNA derived from a Rhodosporidium paludigenum strain (assigned accession no. NCYC 3179).<br>
Fig. 48 is a depiction of the nucleotide sequence (SEQ ID NO: 15) of a YEMH polypeptide-encoding cDNA derived from a Rhodotorula mucilaginosa strain (assigned accession no. NCYC 3190).<br>
Fig. 49 is a depiction of the nucleotide sequence (SEQ ID NO: 16) of a YEMH polypeptide-encoding cDNA derived from a Yarrowia lipolytica strain (assigned accession no. NCYC 3229-2 (Polh-2)).<br>
Fig. 50 is a depiction of the nucleotide sequence (SEQ ID NO: 17) of a YEMH polypeptide-encoding cDNA derived from a Debaromyces hansenii strain (assigned accession no. NCYC 3167).<br>
Fig. 51 is a depiction of the nucleotide sequence (SEQ ID NO: 18) of a YEMH polypeptide-encoding cDNA derived from a Filobasidium neoformans var neoformans strain (assigned GenBank accession no. XM_568708 (#777)).<br>
Fig. 52 is a table showing the homology at the amino acid level of the YESH polypeptides with SEQ ID NOs: 1-9.<br>
Fig. 53 is a table showing the homology at the nucleotide level of YEMH-encoding cDNA molecules with SEQ ID NOs: 10-18.<br>
Fig. 54 is a depiction of the amino acid sequences of eight enantioselective epoxide<br>
28<br><br>
hydrolases aligned for maximum homology. Also shown are consensus amino acids. The sequences labeled #1, #46, #25, CarO54, #692, #23, and Jen 46-2 and #777 correspond to SEQ ID NOs: 1, 2, 3, 4, 5, 6, 7, and 9. The consensus catalytic triad is composed of a nucleophile, an acid and a base, the positions of which are indicated by N, A and B, respectively. "HGXP" represents the region of the oxy-anion hole of the enzymes. "sxNxss" represents the genetic motif found in a/p-hydrolase fold enzymes. (| = homology; | = identity of 75-100%; | = identity of 50-75%;. = gap).<br>
DETAILED DESCRIPTION<br>
Various aspects of the invention are described below.<br>
Nucleic Acid Molecules<br>
The YEMH nucleic acid molecules of the invention can be cDNA, genomic DNA, synthetic DNA, or RNA, and can be double-stranded or single-stranded (i.e., either a sense or an antisense strand). Segments of these molecules are also considered within the scope of the invention, and can be produced by, for example, the polymerase chain reaction (PCR) or generated by treatment with one or more restriction endonucleases. A ribonucleic acid (RNA) molecule can be produced by in vitro transcription. Preferably, the nucleic acid molecules encode polypeptides that, regardless of length, are soluble under normal physiological conditions.<br>
The nucleic acid molecules of the invention can contain naturally occurring sequences, or sequences that differ from those that occur naturally, but, due to the degeneracy of the genetic code, encode the same polypeptide (for example, one of the polypeptides with SEQ ID NOS:1 - 9). In addition, these nucleic acid molecules are not limited to coding sequences, e.g., they can include some or all of the non-coding sequences that lie upstream or downstream from a coding sequence.<br>
The nucleic acid molecules of the invention can be synthesized (for example, by phosphoramidite-based synthesis) or obtained from a biological cell, such as the cell of a eukaryote (e.g., a mammal such as human or a mouse or a yeast such as any of the genera, species, and strains of yeast disclosed herein) or a prokarote (e.g., a bacterium such as Escherichia coli). The nucleic acids can be those of a yeast such as any of the<br>
29<br><br>
genera, species, and strains of yeast disclosed herein. Combinations or modifications of the nucleotides within these types of nucleic acids are also encompassed.<br>
In addition, the isolated nucleic acid molecules of the invention encompass segments that are not found as such in the natural state. Thus, the invention encompasses recombinant nucleic acid molecules (for example, isolated nucleic acid molecules encoding the polypeptides of SEQ ID NOs: 1- 9) incorporated into a vector (for example, a plasmid or viral vector) or into the genome of a heterologous cell (or the genome of a homologous cell, at a position other than the natural chromosomal location). Recombinant nucleic acid molecules and uses therefor are discussed further below.<br>
Techniques associated with detection or regulation of genes are well known to skilled artisans. Such techniques can be used, for example, to test for expression of a YEMH gene in a test cell (e.g., a yeast cell) of interest.<br>
A YEMH family gene or protein can be identified based on its similarity to the relevant YEMH gene or protein, respectively. For example, the identification can be based on sequence identity* The invention features isolated nucleic acid molecules which are, or are at least 50% (e.g., at least: 55%; 60%; 65%; 75%; 85%; 95%; 98%; or 99%) identical to: (a) a nucleic acid molecule that encodes the polypeptide of SEQ ID NOs: 1-9; (b) the nucleotide sequence of SEQ ID NOs: 10-18; (c) a nucleic acid molecule which includes a segment of at least 15 (e.g., at least: 20; 25; 30; 35; 40; 50; 60; 80; 100; 125; 150; 175; 200; 250; 300; 350; 400; 500; 600; 700; 800; 900; 1,000; 1,030; 1,035; 1,040; 1,042; 1,043; 1,050; 1,055; 1,058; 1,060; 1,061; 1,100; 1,150; 1,160; 1,170; 1,175; 1,178; 1,180; 1,181; 1,190; 1,190; 1,195; 1,198; 1,200; 1,201; 1,220; 1,224; 1,225; 1,226; 1,227; 1,228; 1,230; 1,231; or 1,232) nucleotides of SEQ ID NOs: 10-18; (d) a nucleic acid molecule encoding any of the polypeptides or fragments thereof disclosed below; and (e) the complement of any of the above nucleic acid molecules. The complements of the above molecules can be full-length complements or segment complements containing a segment of at least 15 (e.g., at least: 20; 25; 30; 35; 40; 50; 60; 80; 100; 125; 150; 175; 200; 250; 300; 350; 400; 500; 600; 700; 800; 900; 1,000; 1,100; 1,200; 1,220; 1,225; 1,228; 1,230; 1,231; or 1,232) consecutive nucleotides complementary to any of the above nucleic acid molecules. Identity can be over the full-length of SEQ ID NOs: 10-18, or over one or more contiguous or non-contiguous segments.<br>
30<br><br>
The determination of percent identity between two sequences is accomplished using the mathematical algorithm of Karlin and Altschul, Proc. Natl. Acad. Set USA 90, 5873-5877,1993. Such an algorithm is incorporated into the BLASTN and BLASTP programs of Altschul et al. (1990) J. Mol Biol. 215, 403-410. BLAST nucleotide searches are performed with the BLASTN program, score = 100, wordlength = 12, to obtain nucleotide sequences homologous to YEMH-encoding nucleic acids. BLAST protein searches are performed with the BLASTP program, score = 50, wordlength = 3, to obtain amino acid sequences homologous to the YEMH polypeptide. To obtain gapped alignments for comparative purposes, Gapped BLAST is utilized as described in Altschul et al. (1997) Nucleic Acids Res. 25, 3389-3402. When utilizing BLAST and Gapped BLAST programs, the default parameters of the respective programs (e.g., XBLAST and NBLAST) are used.<br>
Hybridization can also be used as a measure of homology between two nucleic acid sequences. A YEMH-encoding nucleic acid sequence, or a portion thereof, can be used as a hybridization probe according to standard hybridization techniques. The hybridization of a YEMH probe to DNA or RNA from a test source (e.g., a mammalian cell) is an indication of the presence of YEMH DNA or RNA in the test source. Hybridization conditions are known to those skilled in the art and can be found in Current Protocols in Molecular Biology, John Wiley &amp; Sons, N.Y., 6.3.1-6.3.6,1991. Moderate hybridization conditions are defined as equivalent to hybridization in 2X sodium chloride/sodium citrate (SSC) at 30°C, followed by a wash in 1 X SSC, 0.1% SDS at 50°C. Highly stringent conditions are defined as equivalent to hybridization in 6X sodium chloride/sodium citrate (SSC) at 45°C, followed by a wash in 0.2 X SSC, 0.1% SDS at 65°C.<br>
The invention also encompasses: (a) vectors (see below) that contain any of the foregoing YEMH coding sequences (including coding sequence segments) and/or their complements (that is, "antisense" sequences); (b) expression vectors that contain any of the foregoing YEMH coding sequences (including coding sequence segments) operably linked to one or more transcriptional and/or translational regulatory elements (TRE; examples of which are given below) necessary to direct expression of the coding sequences; (c) expression vectors encoding, in addition to a YEMH polypeptide (or a<br>
31<br><br>
fragment thereof), a sequence unrelated to YEMH, such as a reporter, a marker, or a siignal peptide fused to YEMH; and (d) genetically engineered host cells (see below) that contain any of the foregoing expression vectors and thereby express the nucleic acid molecules of the invention.<br>
Recombinant nucleic acid molecules can contain a sequence encoding a YEMH polypeptide or YEMH polypeptide having an heterologous signal sequence. The full length YEMH polypeptide, or a fragment thereof, can be fused to such heterologous signal sequences or to additional polypeptides, as described below. Similarly, the nucleic acid molecules of the invention can encode a YEMH polypeptide that includes an exogenous polypeptide that facilitates secretion.<br>
The TRE referred to above and further described below include but are not<br>
limited to inducible and non-inducible promoters, enhancers, operators and other<br>
elements that are known to those skilled in the art and that drive or otherwise regulate<br>
gene expression. Such regulatory elements include but are not limited to the<br>
cjtomegalovirus hCMV immediate early gene, the early or late promoters of SV40<br>
adenovirus, the lac system, the trp system, the TAC system, the TRC system, the major<br>
operator and promoter regions of phage A, the control regions of fd coat protein, the<br>
promoter for 3-phosphoglycerate kinase, the promoters of acid phosphatase, and the<br>
promoters of the yeast a-mating factors. Other useful TRE are listed in the examples<br>
below.	:<br>
Similarly, the nucleic acid can form part of a hybrid gene encoding additional polypeptide sequences, for example, a sequence that functions as a marker or reporter. Examples of marker and reporter genes include (3-lactamase, chloramphenicol acetyltransferase (CAT), adenosine deaminase (ADA), aminoglycoside phosphotransferase (neor, G418r), dihydrofolate reductase (DHFR), hygromycin-B-phosphotransferase (HPH), thymidine kinase (TK), lacZ (encoding p-galactosidase), xanthine guanine phosphoribosyltransferase (XGPRT), and green, blue, or yellow fluorescent protein. As with many of the standard procedures associated with the practice of the invention, skilled artisans will be aware of additional useful reagents, for example, additional sequences that can serve the function of a marker or reporter. Generally, the hybrid polypeptide will include a first portion and a second portion; the first portion<br>
32<br><br>
being a YEMH polypeptide (or any of YEMH fragments described below) and the second portion being, for example, the reporter described above or an Ig heavy chain constant region or part of an Ig heavy chain constant region, e.g., the CH2 and CH3 domains of IgG2a heavy chain. Other hybrids could include an antigenic tag or a poly-His tag to facilitate purification.<br>
The expression systems that can be used for purposes of the invention include, but are not limited to, microorganisms such as yeasts (e.g, any of the genera, species or strains listed herein) or bacteria (for example, E. coli and B. subtilis) transformed with recombinant bacteriophage DNA, plasmid DNA, or cosmid DNA expression vectors containing the nucleic acid molecules of the invention; yeast (for example, Saccharomyces, Yarrowia, Pichia, and other genera, species, and strains listed herein) transformed with recombinant yeast expression vectors containing the nucleic acid molecule of the invention; insect cell systems infected with recombinant virus expression vectors (for example, baculovirus) containing the nucleic acid molecule of the invention; pliant cell systems infected with recombinant virus expression vectors (for example, cauliflower mosaic virus (CaMV) or tobacco mosaic virus (TMV)) or transformed with recombinant plasmid expression vectors (for example, Ti plasmid) containing a YEMH nucleotide sequence; or mammalian cell systems (for example, COS, CHO, BHK, 293, VERO, HeLa, MDCK, WI38, and NIH 3T3 cells) harboring recombinant expression constructs containing promoters derived from the genome of mammalian cells (for example, the metallothionein promoter) or from mammalian viruses (for example, the adenovirus late promoter and the vaccinia virus 7.5K promoter). Also useful as host cells axe primary or secondary cells obtained directly from a mammal and transfected with a plasmid vector or infected with a viral vector.<br>
The invention includes wild-type and recombinant cells including, but not limited to, yeast cells (e.g., any of those disclosed herein) containing any of the above YEMH genes, nucleic acid molecules, and genetic constructs. The cells are preferably isolated cells. As used herein, the term "isolated" as applied to a microorganism (e.g., a yeast cell) refers to a microorganism which either has no naturally-occurring counterpart (e.g., a recombinant microorganism such as a recombinant yeast) or has been extracted and/or purified from an environment in which it naturally occurs. Thus, an "isolated .<br>
33<br><br>
microorganism" does not include one residing in an environment in which it naturally occurs, for example, in the air, outer space, the ground, oceans, lakes, rivers, and streams and the like, ground'at the bottom of oceans, lakes, rivers, and streams and the like, snow, ice on top of the ground or in/on oceans lakes, rivers, and streams and the like, man-made structures (e.g., buildings), or in natural hosts (e.g., plant, animal or microbial hosts) of the microorganism, unless the microorganism (or a progenitor of the microorganism) was previously extracted and/or purified from an environment in which it naturally occurs and subsequently returned to such an environment or any other environment in which it can survive. An example of an isolated microorganism is one in a substantially pure culture of the microorganism.<br>
Moreover the invention provides a substantially pure culture of a microorganism (e.g., a microbial cell such as a yeast cell). As used herein, a "substantially pure culture" of a microorganism is a culture of that microorganism in which less than about 40% (i.e., less than about: 35%; 30%; 25%; 20%; 15%; 10%; 5%; 2%; 1%; 0.5%; 0.25%; 0.1%; 0.01%; 0.001%; 0.0001%; or even less) of the total number of viable microbial (e.g., bacterial, fungal (including yeast), mycoplasmal, or protozoan) cells in the culture are viable microbial cells other than the specified microorganism. The term "about" in this context means that the relevant percentage can be 15% percent of the specified percentage above or below the specified percentage. Thus, for example, about 20% can be 17% to 23%. Such a culture of microorganisms includes the microorganisms and a growth, storage, or transport medium. Media can be liquid, semi-solid (e.g., gelatinous media), or frozen. The culture includes the cells growing in the liquid or in/on the semi-solid medium or being stored or transported in a storage or transport medium, including a frozen storage or transport medium. The cultures are in a culture vessel or storage vessel or substrate (e.g., a culture dish, flask, or tube or a storage vial or tube).<br>
The microbial cells of the invention can be stored, for example, as frozen cell suspensions, e.g. in buffer containing a cryoprotectant such as glycerol or sucrose, as lyophiiized cells. Alternatively, they can be stored, for example, as dried cell preparations obtained, e.g., by fluidised bed drying or spray drying, or any other suitable drying method. Similarly the enzyme preparations can be frozen, Iyophilised, or immobilized and stored under appropriate conditions to retain activity.<br>
34<br><br>
Polypeptides and Polvpeptide Fragments<br>
The YEMH polypeptides of the invention include all the YEMH and fragments of YEMH disclosed herein. They can be, for example, the polypeptides with SEQ ID NOs:l-9 and functional fragments of these polypeptides. The polypeptides embraced by the invention also include fusion proteins that contain either full-length or a functional fragment of it fused to unrelated amino acid sequence. The unrelated sequences can be additional functional domains or signal peptides.<br>
The invention features isolated polypeptides which are, or are at least 50% (e.g., at least: 55%; 60%; 65%; 75%; 85%; 95%; 98%; or 99%) identical to the polypeptides with amino acid SEQ ID NOs: 1-9. The identity can be over the full-length of the latter polypeptides or over one or more contiguous or non-contiguous segments.<br>
Fragments of YEMH are segments of the full-length YEMH that are shorter than full-length YEMH. .Fragments of YEMH can contain 5-410 (e.g., 10, 20, 30, 40, 50, 60, 70, 80, 90,100,120,150, 250, 300, 340, 342, 344, 346, 347, 349, 350, 351, 352, 353, 380, 390, 391, 392, 393, 393, 395, 396, 397, 400, 405, 406, 407, 408, or 409) amino acids of SEQ ID NOs:l-9. Fragments of YEMH can be functional fragments or antigenic fragments.<br>
The polypeptides can be any of those described above but with not more 50 (e.g., not more than 50,45, 40, 35, 30, 25, 20,17, 14, 12, 10, nine, eight, seven, six, five, four, three, two, or one) conservative substitution(s). Such substitutions can be made by, for example, site-directed mutagenesis or random mutagenesis of appropriate YEMH coding sequences.<br>
"Functional fragments" of a YEMH polypeptide (and, optionally, any of the above-described YEMH polypeptide variants) have at least 20% (e.g., 30%, 40%, 50%, 60%, 70%, 80%, 90%, 95%, 98%, 99%, 100%, or more) of the ability of the full-length, wild-type YEMH polypeptide to enantioselectively hydrolyse a MEO of interest. One of skill in the art will be able to predict YEMH functional fragments using his or her own knowledge and information provided herein, e.g., the amino acid alignments in Fig. 54 showing highly conserved domains and residues required for epoxide hydrolase activity.<br>
Fragments of interest can be made by either recombinant, synthetic, or proteolytic<br>
35<br><br>
digestive methods and tested for their ability to enantioselectively hydrolyse a MEO.<br>
Antigenic fragments of the polypeptides of the invention are fragments that can bind to an antibody. Methods of testing whether a fragment of interest can bind to an antibody are known in the art.<br>
The polypeptides can be purified from natural sources (e.g., wild-type or recombinant yeast cells such as any of those described herein). Smaller peptides (e.g., those less than about 100 amino acids in length) can also be conveniently synthesized by standard chemical means. In addition, both polypeptides and peptides can be produced by standard in vitro recombinant DNA techniques and in vivo transgenesis, using nucleotide sequences encoding the appropriate polypeptides or peptides. Methods well-known to those skilled in the art can be used to construct expression vectors containing relevant coding sequences and appropriate transcriptional/translational control signals. See, for example, the techniques described in Sambrook et al., Molecular Cloning: A Laboratory Manual (2nd Ed.) [Cold Spring Harbor Laboratory, N.Y., 1989], and Ausubel et al., Current Protocols in Molecular Biology [Green Publishing Associates and Wiley Interscience, N.Yt, 1989].<br>
Polypeptides and fragments of the invention also include those described above, but modified by the addition, at the amino- and/or carboxyl-terminal ends, of a blocking agent to facilitate survival of the relevant polypeptide. This can be useful in those situations in which the peptide termini tend to be degraded by proteases. Such blocking agents can include, without limitation, additional related or unrelated peptide sequences that can be attached to the amino and/or carboxyl terminal residues of the peptide to be administered. This can be done either chemically during the synthesis of the peptide or by recombinant DNA technology by methods familiar to artisans of average skill.<br>
Alternatively, blocking agents such as pyroglutamic acid or other molecules known in the art can be attached to the amino and/or carboxyl terminal residues, or the amino group at the amino terminus or carboxyl group at the carboxyl terminus can be replaced with a different moiety. Likewise, the peptides can be covalently or noncovalently coupled to pharmaceutically acceptable "carrier" proteins prior to administration.<br>
Also of interest are peptidomimetic compounds that are designed based upon the<br>
36<br><br>
amino acid sequences of the functional peptide fragments. Peptidomimetic compounds are synthetic compounds having a three-dimensional conformation (i.e., a "peptide motif) that is substantially the same as the three-dimensional conformation of a selected peptide. The peptide motif provides the peptidomimetic compound with the ability to enantioselectively hydrolyse a MEO of interest in a manner qualitatively identical to that of the YEMH functional fragment from which the peptidomimetic was derived. Peptidomimetic compounds can have additional characteristics that enhance their therapeutic utility, such as increased cell permeability and prolonged biological half-life.<br>
The peptidomimetics typically have a backbone that is partially or completely non-peptide, but with side groups that are identical to the side groups of the amino acid residues that occur in the peptide on which the peptidomimetic is based. Several types of<br>
chemical bonds, e.g., ester, thioester, thioamide, retroamide, reduced carbonyl,<br>
i dirnethylene and ketbmethylene bonds, are known in the art to be generally useful<br>
substitutes for peptide bonds in the construction of protease-resistant peptidomimetics.<br>
The invention also provides compositions and preparations containing one or more (e.g., two, three, four, five, six, seven, eight, nine, ten, 12, 15, 20, 25, or more) of the above-described polypeptides, polypeptide variants, and polypeptide fragments. The composition or preparation can be, for example a crude cell (e.g. a yeast cell) extract or culture supernatant, a crude enzyme preparation, a highly purified enzyme preparation. The compositions and preparations can also contain one or more of a variety of carriers or stabilizers known 'in the art. Carriers and stabilizers are known in the art and include, for example: buffers, such as phosphate, citrate, and other non-organic acids; antioxidants such as ascorbic acid; low molecular weight (less than 10 residues) polypeptides; proteins such as serum albumin, gelatin or immunoglobulins; hydrophilic polymers such polyvinylpyrrolidone; amino acids such as glycine, glutamine, asparagine, arginine, or lysine; monosaccharides, disaccharides, and other carbohydrates including glucose, mannose, or dextrans; chelating agents such as ethylenediaminetetraacetic acid (EDTA); sugar alcohols such as mannitol, or sorbitol; salt-forming counterions such as sodium; and/or nonionic surfactants such as Tween and Pluronics.<br>
Methods of Producing Optically Active Vicinal Diols<br>
37<br><br>
The invention provides methods for obtaining enantiopure, or substantially enantiopure, optically active vicinal diols. Enantiopure optically active vicinal diol preparations are preparations containing one enantiomer of the vicinal diol and none of the other enantiomer of the vicinal diol. "Substantially enantiopure" optically active vicinal diol preparations are preparations containing at least 55% (e.g., at least: 60%; 70%; 80%; 85%; 90%; 95%; 97%; 98%; 99%; 99.5%; 99.8%; or 99.9%), relative to the total amount of both vicinal diol enantiomers, of the particular enantiomer of the vicinal diol.<br>
The method involves exposing a MEO to a YEMH polypeptide (e.g., isolated or in a microbial cell), which selectively catalyzes the conversion of the MEO to one optically active vicinal diol enantiomer (R,R or S,S). In this way the desired optically active vicinal diol is produced. YEMH polypeptides useful for the invention (i.e., those with enantioselective activity) will catalyze the conversion of a MEO to one optically active vicinal diol enantiomer with less than 80% (e.g., less than: 70%, 60%, 50%, 40%, 30%; 20%; 10%; 5%; 2.5%; 1%; 0.5%; 0.01%), or even none, of the efficiency that it catalyzes the conversion of the MEO to the other optically active enantiomer of the vicinal diol. Useful concentrations of the MEO and conditions of incubation will vary from one YEMH polypeptide to another and from one MEO to another. Given the teachings of the working examples contained herein, one skilled in the art will know how to select working conditions for the production of a desired optically active enantiomer of a desired vicinal diol.<br>
The method can be implemented by, for example, incubating (e.g., culturing) the MEO with a wild-type yeast cell or a recombinant cell (yeast or any other host species listed herein) containing a nucleic acid sequence (e.g. a gene, or a recombinant nucleic acid sequence) encoding a YEMH polypeptide, a crude extract from such cells, a semi-purified preparation of aYEMH polypeptide, or an isolated YEMH polypeptide. The strain of the yeast cell may be selected from the following genera: Arxula, Brettanomyces, Bullera, Bulleromyces, Candida, Cryptococcus, Debaryomyces, Dekkera, Exophiala, Geotrichwn, Hormonema, Issatchenkia, Kluyveromyces, Lipomyces, Mastigomyces, Myxozyma, Pichia, Rhodosporidium, Rhodotorula, Sporidiobolus, Sporobolomyces,<br>
38<br><br>
Trichosporon, Wingea, or Yarrowia.<br>
For the MEO cis-2,3-epoxybutane, yeast strains can be selected from, for example, the following genera: Brettanomyces, Candida, Cryptococcus, Debaryomces, Dekkera, Geotricum, Issatchenkia, Lipomyces, Myxozyma, Pichia, Rhodosporidium, Rhodotorula, or Trichosporon.<br>
For the MEO cyclohexene-oxide, yeast strains can be selected from, for example, the following genera: Bullera, Cryptococcus, Rhodosporidium, Rhodotorula, Sporobolomyces, or Trichosporon.<br>
For the MEO cis-epoxysuccinate, the yeast strain can be selected from, for example, the following genera: Cryptococcus, Pichia, Rhodosporidium, Rhodotorula, or Trichosporon.<br>
In other words, the polypeptide or functional fragment thereof having enantioselective epoxide hydrolase activity can be derived from any yeast strain able to produce a polypeptide which converts or hydrolyses the epoxide shown by the general formula (I), (II), (V), (VI) or (VII) to optically vicinal diols.<br>
Yeast strains innately capable of producing a polypeptide which catalyses the hydrolysis of MEO (e.g., the hydrolysis of cis-2,3-epoxybutane to optically active 2,3-butanediol) can be selected from the following yeast species:<br><br>
No.	Genus	Species<br>
] 2 3	Brettanomyces	B. anomalus B. bruxellensis B, species(e.g., NCYC 3151)<br>
45	Candida	C. fabianii C. kruisii<br>
6 7 8	Cryptococcus	C. gastricus C. hungaricus C. poazolicus<br>
910	Debaryomces	D. hansenii D. napalensis<br>
11	Dekkera	D. anomala<br>
12	Geotricum	G. species (e.g., UOFS Y-0111)<br>
13	Issatchenkia	/. orientalis<br>
39<br><br>
14	Lipomyces	L species (e.g., UOFS Y-2159)<br>
15	Myxozyma	M. melibiosi<br>
16	Pichia	P. finlandica<br>
17		P. haplophila<br>
18	Rhodosporidium	R. lusitaniae<br>
19		R. sphaerocarpum<br>
20		R. toruloides<br>
21	Rhodotorula	R. araucariae<br>
22		R. aurantiaca<br>
23		R. glutinis<br>
24		R. minuta var. minuta<br>
25		R. species (e.g., NCYC 3224)<br>
26		R. sp. minuta / mucilaginosa<br>
27	Trichosporon	T. delbrueckii<br>
28		T. moniliiforme<br>
29		T. pullulans<br>
30		T. species(p.g., NCYC 3210)<br>
31	Wingea	W. robertsiae<br>
Yeast strains innately capable of producing a polypeptide which catalyses the hydrolysis of MEO (e.g., the hydrolysis of cyclohexene oxide or cyclopentene oxide to optically active cyclohexane diol or cyclopentane diol) can be selected from the following yeast species:<br><br>
	Genus	Species<br>
1	Bullera	B. dendrophila<br>
2 3 4	Cryptococcus	C. laurentii C. podzolicus C. terrens<br>
5 6	Rhodosporidium	R. paludigenum R. toruloides<br>
7 8 9	Rhodotorula	R. araucariae R. glutinis R. mucilaginosa<br>
10	Sporobolomyces	¦ S. roseus<br>
11	Trichosporon	T. species (e.g., NCYC 3211)<br>
40<br><br>
Yeast strains innately capable of producing a polypeptide which catalyses the hydrolysis of MEO (e.g., the hydrolysis cis-epoxysuccinate to optically active tartaric acid) can be selected from the following yeast species:<br><br>
	Genus	Species<br>
12 3	Cryptococcus	C. curvatus C. laurentii C. podzolicus<br>
4	Pichia	P. haplophila<br>
5 6	Rhodosporidium	R. paludigenum R. toruloides<br>
7 8 9 10	Rhodotorula	R. araucariae R. glutinis R. sp. minuta / mucilaginosa R. species (e.g., NCYC 3192)<br>
11 12	Trichosporon	T. montevideense T. species (e.g., NCYC 3212)<br>
The yeast strain can be, for example, one selected from Tables 2-4 (see below).<br>
Cultivation in bioreactors of wild-type or recombinant yeast strains expressing YEMH polypeptides (with the purpose of preparing yeasts stocks or for the enantio selective preparative methods of the invention) can be carried out under conditions that provide useful biomass and/or enzyme titer yields. Cultivation can be by batch, fed-batch or continuous culture methods. Useful cultivation conditions are dependent on the yeast strain used. General procedures for establishing useful growth conditions of yeasts, fungi and bacteria in bioreactors are known to those skilled in the art. The MEO can be added directly to the culture. The concentration of the MEO in the reaction mixtureshould preferentially be at least equal to the solubility of the MEO in the aqueous phase of the reaction mixture. The starting amount of MEO added to the reaction mixture is not critical. The MEO can be metered out continuously or in batch mode to the reaction mixture.<br>
The amount of the yeast cells, crude yeast cell extract, or partially purified or isolated polypeptide having MEO enantioselective activity added to the reaction depends on the kinetic parameters of the specific reaction and the amount of epoxide that is to be hydrolysed. In the case of product inhibition, it can be advantageous to remove the<br>
41<br><br>
formed vicinal diol from the reaction mixture or to maintain the concentration of the vicinal diol at levels that allow reasonable reaction rates. Techniques used to enhance enzyme and biomass yields include the identification of useful (or optimal) carbon sources, nitrogen sources, cultivation time, dilution rates (in the case of continuous culture) and feed rates, carbon starvation, addition of trace elements and growth factors to the culture medium, and addition of inducers for example substrates or substrate analogs of the epoxide hydrolases during cultivation. In the case of recombinant hosts, the conditions under which the promoters function workably for transcription of the gene encoding the polypeptide with epoxide hydrolase activity are taken into account. At the end of fermentation (culture), biomass and culture medium can be separated by methods known to one skilled in the art, such as filtration or centrifugation.<br>
The processes are generally performed under mild conditions. For example, the reactions can be carried out at a pH from 5 to 10, preferably from 6.5 to 9, and most preferably from 7 to 8.5. The temperature for hydrolysis can be from 0 to 70 °C, preferably from 0 to 50 °C, most preferably from 4 to 40 °C. It is also known that lowering of the temperature of the reaction can enhance enantioselectivity of an enzyme.<br>
The reaction mixture can also contain mixtures of water with at least one water-miscible solvent (e.g., organic water-miscible solvents). Preferably water-miscible solvents are added to the reaction mixture such that epoxide hydrolase activity remains measurable. Water-miscible solvents are preferably organic solvents and can be, for example, acetone, methanol, ethanol, propanol, isopropanol, acetonitrile, dimethylsulfoxide, N, JV-dimethylformamide, A^-methylpyrolidine, and the like.<br>
The reaction mixture can also contain mixtures of water with at least one water-immiscible organic solvent. Examples of water-immiscible solvents that can be used include, for example, toluene, 1,1,2-trichlorotrifluoroethane, methyl tert-butyl ether, methyl isobutyl ketone, dibutyl-o-phtalate, aliphatic alcohols containing 6 to 9 carbon atoms (for example hexanol or octanol), aliphatic hydrocarbons containing 6 to 16 carbon atoms (for example cyclohexane, w-hexane, n -octane, n -decane, n -dodecane, n -tetradecane and n -hexadecane or mixtures of the aforementioned hydrocarbons), and the like. Thus, the reaction mixture can include water with at least one water-immiscible<br>
42<br><br>
organic solvent selected from the group consisting of toluene, 1,1,2-trichlorotrifluoroethane, methyl tert-buty\ ether, methyl isobutyl ketone, dibutyl-o-phtalate, aliphatic alcohols containing 6 to 9 carbon atoms, and aliphatic hydrocarbons containing 6 to 16 carbon.<br>
The reaction mixture can also contain surfactants (for example Tween 80), cyclodextrins or phase-transfer catalysts and the like that can increase, selectively or otherwise, the solubility of the meso-epoxide in the reaction mixture.<br>
The reaction mixture can also contain a buffer. Buffers are known in the art and include, for example, phosphate buffers, citrate buffers, Tris buffers, or HEPES buffers.<br>
The production of the YEMH polypeptides, including functional fragments, can be, for example, as recited above in the section on Polypeptides and Polypeptide Fragments. Thus they can made by production in a natural host cell, production in a recombinant host cell, or synthetic production. Recombinant production can be carried out in host cells of microbial origin. Preferred yeast host cells are selected from, but are not limited to, the genera Saccharomyces, Arxula, Kluyveromyces, Hansenula, Pichia, Yarrowia and Candida. Preferred bacterial host cells include Escherichia coli, Agrobacterium species, Bacillus species and Streptomyces species. Preferred filamentous fungal host cells are selected from the group consisting of the genera Aspergillus, Trichoderma, and Fusarium. The production of the polypeptide can be, e.g., intra- or extra- cellular production and can be by, e.g., secretion into the culture medium.<br>
In the fermentation reactions, the polypeptides (including functional fragments) can be immobilized on a solid support or free in solution. Procedures for immobilization of the yeast or preparation thereof include, but are not limited to, adsorption; covalent attachment; cross-linked enzyme aggregates; cross-linked enzyme crystals; entrapment in hydrogels; and entrapment into reverse micelles.<br>
The progress of the reaction can be monitored by standard procedures known to one skilled in the art, which include, for example, gas chromatography or high-pressure liquid chromatography on columns containing chiral stationary phases. The vicinal diol formed can be removed from the reaction mixture at one or more stages of the reaction<br>
The reaction can be stopped when one enantiomer of the vicinal diol is found to be in excess compared to the other enantiomer of the vicinal diol. Preferably, the<br>
43<br><br>
reaction is stopped when one enantiomer of a vicinal diol of general formula (III), (IV), (VIII), (IX), or (X) is found to be in an enantiomeric excess of at least 90%. In a more preferred embodiment of the invention, the reaction is stopped when one enantiomer of a vicinal diol of general formula (III), (IV), (VIII), (IX), or (X) is found to be in an enantiomeric excess of at least 95%. The reaction can be stopped by the separation (for example centrifugation, membrane filtration and the like) of the yeast, or a preparation thereof, from the reaction mixture or by inactivation (for example by heat treatment or addition of salts and/or organic solvents) of the yeast or polypeptide, or preparation thereof. Thus, the reaction can be stopped for by, for example, the separation of the catalytic agent from the reactants and products in the mixture, or by ablation or inhibition of the catalytic activity, by techniques known to one skilled in the art.<br>
The optically active vicinal diols produced by the reaction can be recovered from the reaction mixture, directly or after removal of the yeast, or preparation thereof. Preferably, the process can include continuously recovering the optically active vicinal diol produced by the reaction directly from the reaction mixture. Methods of removal of the optically active vicinal diol produced by the reaction include, for example, extraction with an organic solvent (such as hexane, toluene, diethyl ether, petroleum ether, dichloromethane, chloroform, ethyl acetate and the like), vacuum concentration, crystallisation, distillation, membrane separation, column chromatography and the like.<br>
Thus, the present invention provides an efficient process with economical advantages compared to other chemical and biological methods for the production, in high enantiomeric purity, of optically active vicinal diols of the general formula (III), (IV), (VIII), (IX) or (X) from mesoepoxides in the presence of a yeast strain having enantioselective epoxide hydrolase activity or a polypeptide having such activity.<br>
Yeast epoxide hvdrolase antibodies<br>
The invention features antibodies that bind to yeast epoxide hydrolase polypeptides or fragments (e.g., antigenic or functional fragments) of such polypeptides. The polypeptides are preferably yeast epoxide polypeptides with enantioselective activity, and in particular those with mejo-epoxide enantioselective activity (i.e.,YEMH), e.g., those with SEQ ID NOs: 1-9. The antibodies preferably bind specifically to yeast<br>
44<br><br>
epoxide hydrolase polypeptides, i.e., not to epoxide hydrolase polypeptides of species other than yeast species. More preferably, they can bind specifically to yeast epoxide polypeptides with enantioselective activity, and in particular to YEMH polypeptides, e.g., those with SEQ ID NOs: 1-9. They can moreover bind specifically to one or more of polypeptides with SEQ ID NOs: 1, 2, 3, 4, 5, 6, 7, 8, or 9.<br>
Antibodies can be polyclonal or monoclonal antibodies; methods for producing both types of antibody are known in the art. The antibodies can be of any class (e.g., IgM, IgG, IgA, IgD, or IgE). They are preferably IgG antibodies. Moreover, polyclonal antibodies and monoclonal antibodies can be generated in, or generated from B cells from, animals any number of vertebrate (e.g., mammalian) species, e.g., humans, non-human primates (e.g., monkeys, baboons, or chimpanzees), horses, goats, camels, sheep, pigs, bovine animals (e.g., cows, bulls, or oxen), dogs, cats, rabbits, gerbils, hamsters, guinea pigs, rats, mice, birds (such as chickens or turkeys), or fish.<br>
Recombinant antibodies specific for YEMH polypeplides, such as chimeric monoclonal antibodies composed of portions derived from different species and humanized monoclonal antibodies comprising both human and non-human portions, are also encompassed by the invention. Such chimeric and humanized monoclonal antibodies can be produced by recombinant DNA techniques known in the art, for example, using methods described in Robinson et al., International Patent Publication PCT/US86/02269; Akira et al., European Patent Application 184,187; Taniguchi, European Patent Application 171,496; Morrison et al., European Patent Application 173,494; Neuberger et al., PCT Application WO 86/01533; Cabilly et al., U.S. Patent No. 4,816,567; Cabilly et al., European Patent Application 125,023; Better et al. (1988) Science 240,1041-43; Liu et al. (1987) J. Immunol. 139, 3521-26; Sun et al. (1987) PNAS 84, 214-18; Nishimura et al. (1987) Cane. Res. 47, 999-1005; Wood et al. (1985) Nature 314,446-49; Shaw et al. (1988) J. Natl. Cancer Inst. 80,1553-59; Morrison, (1985) Science 229,1202-07; Oi et al. (1986) BioTechniques 4, 214; Winter, U.S. Patent No. 5,225,539; Jones et al. (1986) Nature 321, 552-25; Veroeyan et al. (1988) Science 239,1534; and Beidler et al. (1988) J. Immunol. 141, 4053-60.<br>
Also useful for the invention are antibody fragments and derivatives that contain at least the functional portion of the antigen-binding domain of an antibody that binds to<br>
45<br><br>
a YEMH polypeptide. Antibody fragments that contain the binding domain of the molecule can be generated by known techniques. Such fragments include, but are not limited to: F(ab')2 fragments that can be produced by pepsin digestion of antibody molecules; Fab fragments that can be generated by reducing the disulfide bridges of F(ab')2 fragments; and Fab fragments that can be generated by treating antibody molecules with papain and a reducing agent. See, e.g., National Institutes of Health, 1 Current Protocols In Immunology, Coligan et al., ed. 2.8, 2.10 (Wiley Interscience, 1991). Antibody fragments also include Fv fragments, i.e., antibody products in which there are few or no constant region amino acid residues. A single chain Fv fragment (scFv) is a single polypeptide chain that includes both the heavy and light chain variable regions of the antibody from which the scFv is derived. Such fragments can be produced, for example, as described in U.S. Patent No. 4,642,334, which is incorporated herein by reference in its entirety. The antibody can be a "humanized" version of a monoclonal antibody originally generated in a different species.<br>
The above-described antibodies can be used for a variety of purposes including, but not limited to, YEMH polypeptide purification, detection, and quantitative measurement.<br>
The following examples serve to illustrate, not limit, the invention.<br>
EXAMPLES<br>
Example I. Materials and Methods Determination of concentrations and enantiomeric excesses<br>
In the examples, quantitative determinations of the compounds and determination of enantiomeric excesses were carried out by gas chromatography or HPLC. Gas chromatography (GLC) was performed on a Hewlett-Packard 6890 gas chromatograph equipped with FID detector and using H2 as carrier gas. HPLC was performed on a Hewlett-Packard HP 1100 high pressure liquid chromatograph equipped with a UV detector. Determination of the enantiomeric excesses of 2,3-butanediol, cyclohexane diol and tetrahydro-3,4-furandiol was performed by GLC using fused silica cyclodextrin capillary columns (Supelco) (30 m length, 25 mm ID and 25 pm film thickness).<br>
46<br><br>
Determination of the enantiomeric excesses of tartaric acid formed from cis epoxysuccinic acid was performed by HPLC using an Astec CLC-D column (150 x 4.6 mm). Determination of the enantiomeric excess of l,2-diphenyl-l,2-ethanediol was performed by HPLC using a Chiralcel OJ-R column (150 x 4.6 mm). Reaction conditions and retention times were as follows:<br>
2,3-butanediol: p-DEX 110 at 90°C isotherm, head pressure 14 psi, split flow 80 ml/min. Retention times (min): R, (S,S)-diol = 4.50, Rt (R,R)-diol = 4.65, R* (meso)-diol = 5.26.<br>
cyclohexane diol: p-DEX 225 at 110°C isotherm, head pressure 14 psi, split flow 80 ml/min. Retention times (min): Rt (S,S)-diol = 8.7, Rt (R,R)-diol = 9.6.<br>
cyclopentane diol: p-DEX 225 at 100°C isotherm, head pressure 14 psi, split flow 80 ml/min. Retention times (min): Rt(S,S)-diol = 8.7, Rt(R,R)-diol = 9.6.<br>
tetrahydro-3,4-furandiol: a-DEX 120 at 120°C isotherm, head pressure 14 psi, split flow 80 ml/min. Retention times (min):: Rt(R,R)-diol = 13.7, Rt(S,S)-diol = 14.5.<br>
tartaric acid: Astec CLC-D column. Mobile phase: 5 mM O1SO4, pH 3.1. Flow rate: 1 ml/min. Detection: UV, 254 nm. Rt (D)-tartaric acid = 5.66, Rt (L)-tartaric acid = 6.66.<br>
l^-diphenyl-l^-ethanediol: Chiralcel OJ-R, mobile phase: Hexane / 2-propanol 90:10, flow rate 0.5 ml/min. Retention times (min): Rt (S,S) = 27.3, Rt (R,R) = 30.2.<br>
Concentrations of the diol enantiomers were derived from calibration curves obtained from extractions of the diol from buffer in the absence of active cells.<br>
Preparation of frozen yeast cells for screening<br>
Yeasts were grown at 30°C in 1 L shake-flask cultures containing 200 ml yeast<br>
47<br><br>
extract/malt extract (YM) medium (3% yeast extract, 2% malt extract, 1% peptone w/v) supplemented with 1% glucose (w/v). At late stationary phase (48 - 72 h), the cells were harvested by centrifugation (10 000 g, 10 min, 4°C), washed with phosphate buffer (50 mM, pH7.5), pelleted by centrifugation, and frozen in phosphate buffer containing glycerol (20%) at -20°C as 20% (w/v) cell suspensions. The cells were stored for several months without significant loss of activity.<br>
Yeast isolate (strain) screening<br>
Epoxide (10 jxl of a 1M stock solution in EtOH or DMSO) was added to a final concentration of 20 or 50 mM to 500 ul cell suspension (20 or 50 % w/v) in phosphate buffer (50 mM, pH 7.5). The reaction mixtures were incubated at 30°C for 1 to 8 hours. The reaction mixtures were extracted with EtOAc (300 jal) and centrifuged. Vicinal diol formation was evaluated by TLC (silica gel Merck 60 F254). Compounds were visualized by spraying with vanillin/cone. H2SO4 (5g/l). Reaction mixtures that showed substantial vicinal diol formation were evaluated for asymmetric hydrolysis of the epoxide by chiral GLC or HPLC analysis. Some reactions were repeated over longer or shorter times in order to analyse the reactions at suitable conversions.<br>
General procedure for the hydrolysis of mgso-epoxides<br>
Frozen cells were thawed, washed with phosphate buffer (50 mM, pH 7,5) and resuspended in buffer. Cell suspensions (10 ml, 20% or 50% w/v) were placed in 20 ml glass bottles with screw caps fitted with septa. The substrate (100 or 250 jal of a 2M (v/v) stock solution in EtOH or DMSO) was added to final concentrations of 20 mM or 50 mM, The mixtures were agitated on a shaking water bath at 30°C. The course of the byconversions'of epoxides was followed by withdrawing samples (500 p\) at appropriate time intervals. For chiral GLC analysis, samples were extracted with 300 ul EtOAc and centrifuged (3000 x g, 2 min). The organic layer was dried over anhydrous MgSO4 prior to analysis by chiral GLC. For HPLC analysis of tartaric acid, the reactions were stopped by removal of the catalyst by centrifugation (10 000 x g, 10 min) and the supernatant analysed directly. For HPLC analysis of diphenylethanediol (hydrobenzoin), the samples<br>
48<br><br>
were extracted with hexane and centrifuged (3000 x g, 2 min). The organic layer was dried over anhydrous MgSO-i, prior to analysis by chiral HPLC.<br>
Determination of the absolute configuration of diols<br>
Absolute configurations of cyclopentane diol, cyclohexane diol, 2,3-butanediol and tartaric acid were determined by co-injection on GC of the commercially available (1R,2R)- and (1S,2S)- trans diols (Fluka). Absolute configurations of tetrahydrofurandiol and l,2-diphenyl-l,2-ethanediol (hydrobenzoin) were deduced from the elution orders on the same chiral columns previously reported (Zhao et al., 2004; Li et. al., 2004).<br>
Yeast strains<br>
Yeast strains designated "Jen" and numerical screen numbers were obtained from the Yeast Culture Collection of the University of the Free State. Yeast strains with screen numbers containing "AB" or "Car" or "Alf' or "Poh" were isolated from soil from specialised ecological niches selected based on the inventors' belief that selectivity for specific classes of epoxides in microorganisms would be determined by environmental factors such as terpene-rich environments or highly contaminated soil. "AB" and "Alf strains were isolated from Cape Mountain fynbos, an ecological environment unique to South Africa, "Car" strains were isolated from soil under pine trees, and "Poh" strains from soil contaminated by high concentrations of cyanide. It seemed likely that microorganisms existing in these contaminated soil would have alternative respiratory mechanisms.<br>
All new isolates that produced optically active vicinal diols during hydrolysis of MEO were identified by known biochemical characterisation methods, and most of the isolates were also subjected to molecular identification by sequence analyses of the D1/D2 region of the large subunit rDNA. These new isolates were subsequently deposited at the Yeast Culture Collection of University of the Orange Free State (UOFS) and assigned UOFS numbers. Some of the isolates were deposited at the National Collection of Yeast Cultures (NCYC), Norwich, United Kingdom.<br>
Example II. Sequencing- cloning and overexpression of wild type yeast epoxide<br>
49<br><br>
hydrolases in Yarrowia lipolvtica as production host under the control of different<br>
promoters 1. Vectors, Strains and Primers (Table 1)<br>
The following features are common to all the E. colilY, Hpolytica auto-cloning integrative vectors used:<br>
•	LIP2 terminator<br>
•	Zeta regions<br>
•	Kanamycin resistance for E. coli selection<br>
•	mono-copy auto cloning vectors (pINA 1311, pINA 1313, pINA 3313) with a<br>
fully functional selection marker gene carry the fully functional ura3dl allele<br>
from the URA3 selection marker gene<br>
o multi-copy auto cloning vectors (pINA 1291, pINA 1293, pINA 3293) with a defective selection marker gene (copy number amplification) carry the defective ura3d4 allele from the URA3 selection marker gene<br>
Table 1. Vectors, strains and nrimers<br><br><br>
YL-rev	5'-GAT CCC CAC CGG AAT TG-3' (SEQ ID NO:20)	<br>
pINA-1	5'- CAT ACA ACC ACA CAC ATC CA-31 (SEQIDNO:21)	PYLHmA fwd primer<br>
pINA-2	5'-TAA ATA GCT TAG ATA CCA CAG-3' (SEQ ID NO:22)	pYLTsA/pYLHmA rev primer<br>
pINA-3	5'-CTC TCT CTC CTT GTC AAC T-3P (SEQIDNO:23)	pYLTsA fwd primer<br>
51<br><br>
	Transform ants (multi-copy and single-copy)<br>
	LTninsformants^^H|	^^^^^^HI^^^^^^^^^LG cue o r I &amp;i n^^^^^^^Hi^^^^^^^^H<br>
		LVector: pYIi3313.(1313) = (pYLTsAjkfiMfflBi<br>
	YL23TsA	Rhodotorula mucilaginosa NCYC 3190<br>
	YL25TsA	Rhodotorula araucariae NCYC 3183<br>
	YL46TsA	Rhodosporidium toruloides UOFS Y-0471<br>
	YL692TsA	Rhodosporidium paludigenum NCYC 3179<br>
	YLlTsA	Rhodosporidium toruloides NCYC 3181<br>
	YLCar54 TsA	Cryptococcus curvatus NCYC 3158<br>
	YLJen 46-2TsA	Yarrowia lipolytica NCYC 3229<br>
	YL777TsA	Filobasidium neoformans var. neoformans (GenBank accession no. XM 568708) (#777).<br>
	rHp4dfproSter^B	<br>
	YL25HmA	Rhodotorula araucariae NCYC 3183<br>
	YL46HmA	Rhodosporidium toruloides UOFS Y-0471<br>
	YL692HmA	Rhodosporidium paludigenum NCYC 3179<br>
3.	Vector preparation<br>
pINA1291 (Fig. 4) was obtained from Dr Madzak of Labo de Gene"tique, INRA, CNRS. This vector was renamed pYLHmA (Yarrowia Lipolytica expression vector, with Hp4d promoter, multi-copy integration selection, Absent secretion signal).<br>
pINA3313 (pKOV93) (Fig. 5) was prepared by the inventors. This vector was renamed pYLTsA (Yarrowia Lipolytica expression vector, with TEF promoter, single-copy integration selection, Absent secretion signal).<br>
Examples and figures that make reference to recombinant strains utilize recombinant strain identification codes in such a way as to iterate host (e.g. YL), gene origin reference (e.g. #25), promoter (e.g. Hp4d), copy insertion (e.g. multi), and absence of secretion signal (e.g. Absent); e.g. YL-25HmA. (Note the variation of code text does not change the strain identification ; i.e. YL-25HmA = YL25HmA = YL 25HJTIA) .<br>
To prepare the vectors for ligation with an epoxide hydrolase coding sequence (or other insert to be expressed in Y. lipolytica), DNA was digested with BarriRl and AvrW.<br>
52<br><br>
4. Insert preparation<br>
Total RNA was isolated from selected yeast strain cells and messenger RNA (mRNA) was purified from it. The mRNA was used as a template to synthesise complementary DNA (cDNA) using reverse transcriptase. The cDNA was then used as a template for Polymerase Chain Reaction (PCR) using appropriate primers. PCR primers were selected by repeated experimentation using multiple test primers for each yeast strain, the sequences of which were based on previously described epoxide hydrolase sequences from a variety of species. The nucleotide sequences of the forward and reverse primers used to generate cDNA coding sequences from mRNA from seven different yeast strains with appropriate restriction enzyme recognition sites at their termini are shown below. Restriction enzyme recognition sequences are underlined and<br>
thp rpifvant rpctrir.tinn pn7vmfts arp shown in nnrpnthfisps<br><br>
Strain	Forward primer	Reverse primer<br>
R.toruloides~NCYC3m #1 R. toruloides UOFS Y-0471 #46 C. curvatus NCYC 3158 Car 054	GTGGATCCATGGCG ACACACA (BamHl)(SEQIDNO:24)	GACCTAGGCTACTTC TCCCACA (Blnl)(SEQIDNO:25)<br>
R araucariaeNCYC 3183 #25	GATTAATGATCAAT GAGCGAGCA (Bell)(SEQ ID NO:26)	GACCTAGGTCACGAC GACAG (Blnl)(SEQIDNO:27)<br>
R. paludigenum NCYC 3179 #692	GTGGATCCATGGCT GCCC A (BamHT)(SEQIDNO:28)	GAGCTAGCTCAGGCC TGG (Nhel)(SEQ ID NO:29)<br>
R. mucilaginosa NCYC 3190 # 23	GTATATCTATGCCC GCCCGCT (BglU)(SEQIDNO:30)	GACCTAGGCTACGAT TTTTGCT (Blnl)(SEQIDNO:31)<br>
Y. lipolytica NCYC 3229 Jen 46-2	GCAGATCTATGTCA TCACTCG (Bglll) (SEQIDNO:32)	GACCTAGGCTACAAC TTCGACG (Blnl)(SEQ1DNO:33)<br>
Debaromyces hansenii NCYC 3167 (#113)	GTGGATCCATGATGCAAGG(Bam¥Ll) (SEQ ID NO:34)	GACCTAGGCTAAGGA TATT (Blnl) (SEQIDNO:35)<br>
Filobasidium neoformans (# 777J	GAGGATCCATGTCG TATTCAGA (BamHT)(SEQIDNO:36)	GAGCTAGCTCAGTAA TTACCTTTG (Nhe)\ (SEQIDNO:37)<br>
53<br><br>
Each PCR reaction contained 200 uM dNTPs, 250 nM of each primer, 2 mM of MgCh, cDNA and 2.5 U of Taq polymerase in a 50 ul reaction volume. The PCR profile used was: 95°C for 5 minutes, followed by 30 cycles of: 95°C - 1 min, 50°C - 1 min, 72°C - 2 min, then a final extension of 72°C for 10 minutes. The PCR products were purified and digested with the restriction enzymes whose recognition sites are engineered at the end of the primers. The cDNA fragment was cloned into a vector and sequenced for confirmation.<br>
Coding seqences to be inserted in either pYLHmA or pYLTsA were prepared with BamHl and Avrll at their termini. The above PCR primers were designed with these restriction sites, unless the sites were also present in the gene to be inserted. If this occurred, appropriate compatible restriction enzymes were selected. PCR template DNA was either the insert cloned into a different vector, or cDNA synthesized from the original host organism. PCR reactions consisted of 200 uM dNTP's, 250 nM each primer, IX Taq polymerase buffer, and 2.5 units Taq polymerase per 100 ul reaction. The amplification programme used was: 95°C for 5 minutes, 30 cycles of 95°C for 1 minute, 50°C for 1 minute, and 72°C for 2 minutes, followed by 72°C for 10 minutes.<br>
PCR products were purified and digested with the relevant restriction enzymes. The digested DNA was subsequently repurified and was ready for ligation into the prepared vector.<br>
5.	Preparation of pYLHmA or pYLTsA constructs<br>
Vector and insert were ligated at pmol end ratios of 3:1 - 10:1 (inserfcvector), using commercial T4 DNA Ligase. Ligations were electroporated into any laboratory strain of Escherichia coli, using the Bio-Rad GenePulser, or equivalent electroporator. Transformants were selected on LM media (10 g/L yeast extract, 10 g/L tryptone, 5 g/L NaCl), supplemented with kanamycin (50 (xg/ml). Transformants were selected based on restriction enzyme digests of purified plasmid DNA.<br>
6.	Yarrowia lipolytica transformation<br>
6.1.1. Preparation of DNA - method 1<br>
Digestion of the pINA-series of plasmids with Notl resulted in the release of a<br>
54<br><br>
bacterial DNA-free expression cassette, containing the ura3d4 (pYLHmA) or the ura3dl (pYLTsA) marker gene and the promoter-gene-terminator.<br>
Scaled-up quantities of each plasmid were isolated. Not! was used to restrict the plasmid DNA, and the digested DNA was run on an agarose gel. Notl digests resulted in generation of the bacterial fragment of the plasmid as a band at 2210 bp, and the expression cassette as a band of 2760 bp + size of insert (pYLHmA) or 2596 bp + size of insert (pYLTsA). The expression cassette fragments were excised from the gel and purified from the agarose. The purified fragment was used for transformation of Y. lipolytica Polh.<br>
6.11.2. Preparation of DNA - method 2<br>
Primers YL-Fwd and YL-Rev (Table 1) were used to amplify the expression cassette. PCR reactions consisted of 200 uM dNTP's, 250 pmol each primer, IX Taq buffer and 2.5 units Taq peri 00 ul reaction. The amplification programme used was 95°C for 5 minutes, 30 cycles of 95°C for 1 minute, 50°C for 1 minute, and 72°C for 3 Vi minutes, followed by a single incubation at 72°C for 10 minutes. The PCR product was purified from the PCR reaction mix and used for transformation of Y. lipolytica Polh.<br>
6.1.3.	Preparation of carrier DNA<br>
DNA from salmon testes was made up as a 10 mg/ml stock in TE (10 mM Tris-HC1, pH 8.0,1 mM EDTA) and sonicated to result in fragments that ranged from approximately 15 kb to 100 bp, with most fragments in a range of 6 to 10 kb. The DNA was denatured by boiling. Aliquots were stored at -20°C.<br>
6.1.4.	Transformation of Yarrowia lipolytica with pYLHmA or pYLTsA<br>
An adaptation of the method of Xuan et al (1988) was used for the transformation of Y. lipolytica Polh. The yeast was inoculated into 50 ml YPD (10 g/L yeast extract, 20 g/L peptone, 20 g/L glucose). The culture was incubated at 30°C, 220 rpm until cell densities of 8 x 107 - 2 x 108 cells/ml were reached. The entire culture was harvested, the pellet resuspended in 10 ml TE and reharvested. 1 ml TE + 0.1 M LiOAc was used to resuspend the pellet and the culture was incubated at 28°C in a ProBlot Jr (Labnet)<br>
55<br><br>
hybridisation oven, set at 4 rpm (or similar incubator) for 1 hour. Transformation mixes were set up with 0.5 - 2 ug of transforming DNA + 5 ug of carrier DNA with 100 ui of<br>
treated cells.<br>
Each mix was set up in a 1.5 ml microfuge tube, and incubated in a 28°C heating block for 30 minutes. 7 volumes of PEG reagent (40% PEG 4,000, 0.1 M LiOAc, 10 mM Tris, 1 mM EDTA, pH 7.5, filter-sterilised) were added to each, mixed carefully and incubated at 28°C for a further 1 hour. The tubes were transferred to a 37°C heating block for 15 minutes, and then the cells "were pelleted by centrifugation for 1 minute at 13,000 rpm. The cell pellets were carefully resuspended in 100 ul dH2O. The transformations were plated on Y. lipolytica selective plates (17 g/1 Difco yeast nitrogen base without amino acids and without (NH^SO^ 20 g/L glucose, 4 g/L NH4C1, 2 g/L casamino acids, 300 mg/L leucine) and incubated at 28°C.<br>
Colonies which appeared on the selective plates after 3-7 days were transferred onto fresh plates and regrown.<br>
6.1.5. Confirmation of integration of pYLHmA or pYLTsA<br>
Colonies that grew on the newly-streaked selective plates were inoculated into 5 ml of YPD medium and grown at 30°C, 200 rpm for 24-48 hours. A small-scale genomic DNA isolation was performed.<br>
PCR was performed using this genomic DNA as template, with either pINA-1 and pINA-2 as primers (transformants with pYLHmA), or pINA-3 and pINA-2 (pYLTsA) (Table 1). Each PCR reaction contained 200 fiM dNTPs, 250 nM of each primer, 2 mM of MgCl2, genomic DNA and 2.5 U/50ul of Taq polymerase. The PCR profile was as described above in this example. These primer sets gave products the size of the inserted genes.<br>
Example III. Selection of wild type veasts that are able to produce optically active<br>
vicinal diols from MEO<br>
Yeasts were cultivated, harvested and frozen as 20% or 50 % (w/v) cell suspensions as described above. The MEO was added to a final concentration of 20 mM<br>
56<br><br>
and the screening was performed as described above. Strains with the highest activities as judged by TLC from diol formation were subjected to chiral GC or HPLC analysis as described above. The enantiomeric excesses of the formed diol after incubation for 2-20 hours of three MEO (cis-2,3-epoxybutane; cyclohexene oxide; and cis-epoxysuccinic acid) with the different yeast strains are shown as Samples 1-145 in Tables 2-4. The generation of the optically active vicinal diols from these three MEO is illustrated in Fig. 3. Enantiomeric excesses (ee) were determined from the formula: E.e. = [[R,R]-[SSS]}/{[R,R]+[S,S]} x 100. [S,S] is the concentration at the appropriate time point of the (S,S) diol enantiomer and [R,R] is the concentration at the appropriate time point of the (R,R)-diol enantiomer.<br>
57<br><br>
Table 2. (Samples 1-47) Enantiomeric excesses (ee) of formed diol obtained after incubation of different yeast strains with cis-2,3-epoxybutane (50 mM, 50% w/v cells, 16 hours).<br>
Sample	Internal	Strain/species	Culture	Abs	ee<br>
No.	strain No.		collection No.	Conf.	(%)<br>
I	K01	Brettanomyces anomalus	NCYC3149	R,R	71.0<br>
2	52	Brettanomyces bruxellensis	NCYC3150	R,R	19.3<br>
3	N13	Brettanomyces sp.	NCYC3151	R,R	74.0<br>
4	704	Candida fabianii	UOFS Y-0963	R,R	84.1<br>
5	X19	Candida kruisii	NCYC3153	R,R	77.7<br>
6	Jen 13	Cryptococcus gastricus	UOFS Y-0477	R,R	31.9<br>
7	AB54	Cryptococcus podzolicus	UOFS Y-1900	R,R	27.4<br>
8	110	Debaryomces hansenii	UOFS Y-0614	R,R	59.5<br>
9	108	Debaryomces hansenii	UOFSY-0612	R,R	86.0<br>
10	93	Debaryomyces hansenii	NCYC3169	R,R	55.6<br>
11	105	Debaryomyces hansenii	UOFS Y-0608	R,R	93.9<br>
12	111	Debaryomyces hansenii	UOFSY-0615	R,R	95.8<br>
13	TVN119	Debaryomyces napalensis	NCYC 3220	R,R	&gt;98%<br>
14	88	Dekkera anomaia	NCYC3170	R,R	17.4<br>
15	227	Geotricum sp.	UOFSY-0111	R,R	33.4<br>
16	TVN228	Jssatchenkia orientalis	NCYC 3221	R,R	60.1<br>
17	TVN229	Issatchenkia orientalis	NCYC 3222	R,R	63.0<br>
18	TVN231	Issatchenkia orientalis	NCYC 3223	R,R	62.5<br>
19	45a	Lipomyces sp.	UOFS Y-2159	R,R	77.5<br>
20	41	Myxozyma melibiosi	NCYC 3172	R,R	18.9<br>
21	520*	Pichia finlandica	NCYC 3173	R,R	&gt;98%<br>
22	673	Pichia haplophila	NCYC 3177	R,R	65.7<br>
23	169	Rhodosporidium lusitaniae	NCYC 3178	R,R	31.7<br>
24	40	Rhodosporidium sphaerocarpum	NCYC 3180	R,R	27.4<br>
25	Alf 01	Rhodosporidium toruloides	NCYC 3181	R,R	60.0<br>
26	46	Rhodosporidium toruloides	UOFSY-0471	R,R	45.0<br>
27	2	Rhodosporidium toruloides	UOFSY-0518	R,R	88.7<br>
28	Car 067	Rhodosporidium toruloides	UOFS Y-2236	R,R	49.6<br>
29	Car 076	Rhodosporidium toruloides	UOFS Y-2238	R,R	85.3<br>
30	Car 131	Rhodosporidium toruloides	UOFS Y-2252	R.R	83.9<br>
31	Car 142	Rhodosporidium toruloides	UOFS Y-2255	R,R	43.3<br>
32	Car 209	Rhodosporidium toruloides	UOFS Y-2260	R.R	84.0<br>
33	25	Rhodotorula araucariae	NCYC 3183	R,R	81.1<br>
34	EP230	Rhodotorula aurantiaca	NCYC 3185	R,R	41.8<br>
35	50	Rhodotorula glutinis	NCYC 3186	R,R	86.1<br>
36	680	Rhodotorula glutinis	NCYC 3203	R,R	77.9<br>
37	713	Rhodotorula glutinis	UOFS Y-0489	R,R	45.2<br>
38	Alf 6	Rhodotorula glutinis	UOFSY-0513	R,R	95.5<br>
39	185	Rhodotorula glutinis	UOFS Y-0559	R,R	49.5<br>
58<br><br>
40	686	Rhodotorula minuta var. minuta	NCYC3188	R,R	67.2<br>
41	Rh27	Rhodotorula sp.	NCYC 3224	R,R	89.0<br>
42	697	Rhodotorulasp. minuta/ mucilaginosa	UOFS Y-0958	R,R	46.7<br>
43	698	Rhodotorula sp. minuta/ mucilaginosa	UOFS Y-0959	R,R	50.2<br>
44	BV04	Trichosporon moniliiforme	NCYC 3214	R,R	33.6<br>
45	61	Trichosporon pullulans	NCYC 3209	R,R	91.4<br>
46	231	Trichosporon sp.	NCYC 3210	R,R	97.6<br>
47	39	Wingea robertsiae	NCYC 3228	R,R	22.1<br>
Table 3.	(Samples 48-97) Enantiomeric excesses (ee) of formed diol obtained	after<br>
incubation of different yeast strains with cyclohexene	oxide (20 mM, 20% w/v .	sells, 6<br>
hours).					<br>
Sample	Internal	Strain/species	Culture	Abs.	ee<br>
No.	strain No.		Collection No.	Conf.	(%)<br>
48	43	Bullera dendrophila	NCYC 3208	R,R	71<br>
49	Jen 12	Cryptococcus laurentii	NCYC 3160	R,R	67<br>
50	AB58	Cryptococcus podzolicus	NCYC 3164	R,R	71<br>
51	AB-50	Cryptococcus podzolicus	UOFS Y-l 883	R,R	65<br>
52	AB-30	Cryptococcus podzolicus	UOFS Y-l904	R,R	77<br>
53	385	Cryptococcus terreus	NCYC 3166	R,R	54<br>
54	692	Rhodosporidium paludigenum	NCYC 3179	R,R	64<br>
55	Alf 01	Rhodosporidium toruloides	NCYC 3181	R,R	66<br>
56	46	Rhodosporidium toruloides	UOFSY-0471	R,R	83<br>
57	Alf 2	Rhodosporidium toruloides	UOFSY-0518	R,R	83<br>
58	Car-003	Rhodosporidium toruloides	UOFS Y-2222	R,R	53<br>
59	Car-006	Rhodosporidium toruloides	UOFS Y-2223	R,R	67<br>
60	Car-020	Rhodosporidium toruloides	UOFS Y-2226	R,R	66<br>
61	Car-038	Rhodosporidium toruloides	UOFS Y-2228	R,R	88<br>
62	Car-052	Rhodosporidium toruloides	UOFS Y-2230	R,R	87<br>
63	Car-059	Rhodosporidium toruloides	UOFSY-2231	R,R	78<br>
64	Car-067	Rhodosporidium toruloides	UOFS Y-2236	R,R	75<br>
65	Car-070	Rhodosporidium toruloides	UOFS Y-2237	R,R	91<br>
66	Car-076	Rhodosporidium toruloides	UOFS Y-2238	R,R	66<br>
67	Car-077	Rhodosporidium toruloides	UOFS Y-2239	R,R	70<br>
68	Car-078	Rhodosporidium toruloides	UOFS Y-2240	R,R	93<br>
69	Car-092	Rhodosporidium toruloides	UOFSY-2241	R,R	72<br>
70	Car-093	Rhodosporidium toruloides	UOFS Y-2242	R,R	74<br>
59<br><br>
71	Car-094	Rhodosporidium toruloides	UOFS Y-2243	R,R	78<br>
72	Car-100	Rhodosporidium toruloides	UOFS Y-2245	R,R	83<br>
73	Car-103	Rhodosporidium toruloides	UOFS Y-2246	R.R	86<br>
74	Car-108	Rhodosporidium toruloides	UOFS Y-2247	R,R	62<br>
75	Car-118	Rhodosporidium toruloides	UOFS Y-2248	R,R	70<br>
76	Car-120	Rhodosporidium toruloides	UOFS Y-2249	R,R	86<br>
77	Car-121	Rhodosporidium toruloides	UOFS Y-2250	R,R	86<br>
78	Car-126	Rhodosporidium toruloides	UOFSY-2251	R,R	80<br>
79	Car-131	Rhodosporidium toruloides	UOFS Y-2252	R,R	52<br>
80	Car-134	Rhodosporidium toruloides	UOFS Y-2253	R,R	52<br>
81	Car-142	Rhodosporidium toruloides	UOFS Y-2255	R,R	74<br>
82	Car-204	Rhodosporidium toruloides	UOFS Y-2257	R,R	92<br>
83	Car-205A	Rhodosporidium toruloides	UOFS Y-2258	R,R	89<br>
84	Car-209	Rhodosporidium toruloides	UOFS Y-2260	R,R	78<br>
85	Car-210	Rhodosporidium toruloides	UOFS Y-2261	R,R	70<br>
86	25	Rhodotorula araucariae	NCYC3183	R,R	83<br>
87	50	Rhodotorula glutinis	NCYC3186	R,R	55<br>
88	713	Rhodotorula glutinis	UOFS Y-0489	R,R	82<br>
89	Alf6	Rhodotorula glutinis	UOFSY-0513	R,R	78<br>
90	681	Rhodotorula glutinis	UOFS Y-0653	R,R	93<br>
91	Car-022	Rhodotorula glutinis	UOFS Y-2227	R,R	80<br>
92	Car-060	Rhodotorula glutinis	UOFS Y-2232	R,R	70<br>
93	Car-061	Rhodotorula glutinis	UOFS Y-2233	R,R	73<br>
94	Car-062	Rhodotorula glutinis	UOFS Y-2234	R,R	76<br>
95	Car-075	Rhodotorula glutinis	UOFS Y-2265	R,R	73<br>
96	285	Sporobolomyces roseus	NCYC3197	R,R	72<br>
97	225	Trichosporon sp.	NCYC3211	R,R	77<br>
60<br><br>
Table 4, (Examples 98-145) Enantiomeric excesses (ee) of formed diol obtained after<br>
incubation of different yeast strains with cis-epoxysuccinic acid (20 mM, 50% w/v cells,<br>
6 hours).	<br>
Abs. ee Conf (%)<br>
Sample<br>
No.<br>
Internal strain No.<br>
Strain/species<br>
Culture Collection No.<br><br>
98	Car 054	Cryptococcus curvatus	NCYC3158	R,R	98<br>
99	Jen 12	Cryptococcus laurentii	NCYC3160	R,R	98<br>
100	AB24	Cryptococcus laurentii	NCYC3161	R,R	98<br>
101	AB25	Cryptococcus laurentii	UOFSY-1884	R,R	98<br>
102	AB58	Cryptococcus podzolicus	NCYC3164	R,R	98<br>
103	AB49	Cryptococcus podzolicus	UOFSY-1882	R,R	98<br>
104	AB50	Cryptococcus podzolicus	UOFSY-1883	R,R	98<br>
105	AB52	Cryptococcus podzolicus	UOFSY-1895	R,R	98<br>
106	AB47	Cryptococcus podzolicus	UOFSY-1908	R,R	98<br>
107	676	Pichia haplophila	NCYC3176	R,R	44<br>
108	692	Rhodosporidium paludigenum	NCYC3179	R.R	98<br>
109	Car 118	Rhodosporidium toruloides	NCYC3182	R,R .	98<br>
110	POH28	Rhodosporidium toruloides	NCYC3215	R.R	98<br>
111	46	Rhodosporidium toruloides	UOFSY-0471	R,R	22<br>
112	Car 003	Rhodosporidium toruloides	UOFS Y-2222	R,R	98<br>
113	Car 020	Rhodosporidium toruloides	UOFS Y-2226	R,R	98<br>
114	Car 038	Rhodosporidium toruloides	UOFS Y-2228	R,R	98<br>
115	Car 052	Rhodosporidium toruloides	UOFS Y-2230	R,R	98<br>
116	Car 059	Rhodosporidium toruloides	UOFSY-2231	R,R	98<br>
117	Car 076	Rhodosporidium toruloides	UOFS Y-2238	R,R	98<br>
118	Car 078	Rhodosporidium toruloides	UOFS Y-2240	R,R	98<br>
119	Car 092	Rhodosporidium toruloides	UOFSY-2241	R»R	98<br>
120	Car 093	Rhodosporidium toruloides	UOFS Y-2242	R,R	98<br>
121	Car 094	Rhodosporidium toruloides	UOFS Y-2243	R,R	98<br>
122	Car 100	Rhodosporidium toruloides	UOFS Y-2245	R,R	98<br>
123	Car 103	Rhodosporidium toruloides	UOFS Y-2246	R,R	98<br>
124	Car 108	Rhodosporidium toruloides	UOFS Y-2247	R,R	98<br>
125	Car 120	Rhodosporidium toruloides	UOFS Y-2249	R,R	98<br>
126	Car 131	Rhodosporidium toruloides	UOFS Y-2252	R,R	98<br>
127	Car 134	Rhodosporidium toruloides	UOFS Y-2253	R,R	98<br>
128	Car 142	Rhodosporidium toruloides	UOFS Y-2255	R,R	98<br>
129	Car 200	Rhodosporidium torulcides	UOFS Y-2256	R,R	98<br>
61<br><br>
130	Car 204	Rhodosporidium toruloides	UOFS Y-2257	R,R	98<br>
131	Car 205 A	Rhodosporidium toruloides	UOFS Y-2258	R,R	98<br>
132	Car 209	Rhodosporidium toruloides	UOFS Y-2260	R,R	98<br>
133	Car210	Rhodosporidium toruloides	UOFS Y-2261	R,R	54<br>
134	25	Rhodotorula araucariae	NCYC3183	R,R	98<br>
135	50	Rhodotorula glutinis	NCYC3186	R,R	76<br>
136	713	Rhodotorula glutinis	UOFS Y-0489	R,R	98"<br>
137	AlfO6	Rhodotorula glutinis	UOFSY-0513	R,R	98<br>
138	681	Rhodotorula glutinis	UOFS Y-0653	R,R	98<br>
139	Car 022	Rhodotorula glutinis	UOFS Y-2227	R,R	98<br>
140	Car 066	Rhodotorula glutinis	UOFS Y-2235	R,R	98<br>
141	Car 075	Rhodotorula glutinis	UOFS Y-2265	R,R	98<br>
142	172	Rhodotorula sp.	NCYC3192	R,R	98<br>
143	158	Rhodotorula sp. minuta / mucilaginosa	NCYC3194	R,R	98<br>
144	Car 205B	Trichosporon montevideense	NCYC 3225	R,R	23<br>
145	155	Trichosporon sp.	NCYC3212	R,R	98<br>
All the yeast strains referred to in this and the following examples are kept and maintained at the University of the Orange Free State (UOFS), Department of Microbial, Biochemical and Food Biotechnology, Faculty of Natural and Agricultural Sciences, P.O. Box 339, Bloemfontein 9300, South Africa (Tel +27 51 401 2396, Fax + 27 51 444 3219) and are readily identified by the yeast species and culture collection number as indicated. Representative examples of strains belonging to the different species have been deposited under the Budapest Treaty at National Collection of Yeast Cultures (NCYC), Institute of Food Research Norwich Research Park Colney Norwich NR4 7UA, U.K. ( Tel: +44-(0)1603-255274 Fax: +44-(0)1603-458414 Email: <a href="../cdn-cgi/l/email-protection.html" class="__cf_email__" data-cfemail="6e000d170d2e0c0c1d1c0d400f0d401b05">[email&#160;protected]</a>) and are readily identified by the yeast species and culture collection accession number as indicated. The samples deposited with the NCYC are taken from the same deposit maintained by the South African Council for Scientific and Industrial Research (CSIR) since prior to the filing date of this application. The deposits will be maintained without restriction in the NCYC depository for a period of 30 years, or 5 years after the most recent request, or for the effective life of the patent, whichever is longer, and will be replaced if the deposit becomes non-viable during that period. Samples of the yeast strains not deposited at<br>
62<br><br>
NCYC will be made available upon request on the same basis and conditions of the Budapest Treaty.<br>
Example IV. Desvmmetrisation of cis-2,3-epoxybutane. cyclohexene oxide, cyclopentene oxide, cis-epoxy succinie acid, and 3,4-epoxvtetrahydrofuran.<br>
Fig. 3 gives examples of desymmetrisation reactions performed using various MEO as substrates for hydrolase-catalysed reactions giving rise to various corresponding vicinal diol products.<br>
Example V. Desvmmetrisation of cis-23-epoxvbutane to produce (R.RV23-<br>
butanediol<br>
(a)	Various wild-type yeast strains selected from Tables 2-4 were tested (in this example<br>
as Samples 146 -148) for their ability to produce optically active 2,3-butanediol from<br>
cis-2,3-epoxy butane. For each example, a line graph is supplied that shows the change<br>
in concentrations of the vicinal diol enantiomers with time (left y-axis) and the<br>
enantiomeric excess (ee) of the formed diol at different times (right y-axis) (Figs. 6 - 8).<br>
As expected for desymmetrisation reactions, the ee remained essentially constant<br>
throughout the reaction.<br>
(b)	Figs. 9 -12 show the desymmetrisation of cis-2,3-epoxybutane by various<br>
recombinant yeast strains (tested in this Example as Samples 149-152) expressing, under<br>
control of the constitutive TEF promoter (Madzak et al., 2004), exogenous epoxide<br>
hydrolases from selected wild-type yeast strains. For each example, a line graph is<br>
supplied that shows the change in concentrations of the vicinal diol enantiomers with<br>
time (left y-axis) and the enantiomeric excess (ee) of the formed diol at different times<br>
(right y-axis). As expected for desymmetrisation reactions, the ee remained essentially<br>
constant throughout the reaction.<br>
Example VI. Desvmmetrisation of cvclohexene oxide to produce (R,R)-cyclohexane<br>
diol<br>
(a) Various wild-type yeast strains selected from Tables 2-4 were tested (in this example<br>
63<br><br>
as Samples 153 - 155) for their ability to produce optically active cyclohexane diol from cyclohexene oxide. For each example, a line graph is supplied that shows the change in concentrations of the vicinal diol enantiomers with time (left y-axis) and the enantiomeric excess (ee) of the formed diol at different times (right y-axis) (Figs. 13-15). As expected for desymmetrisation reactions, the ee remained essentially constant throughout the reaction.<br>
(b) Figs. 16 -20 show the desymmetrisation of cyclohexene oxide by various recombinant yeast strains (tested in this Example as Samples 156-160) expressing, under control of the constitutive TEF promoter (Madzak et al., 2004), exogenous epoxide hydrolases from selected wild-type yeast strains. For each example, a line graph is supplied that shows the change in concentrations of the vicinal diol enantiomers with time (left y-axis) and the enantiomeric excess (ee) of the formed diol at different times (right y-axis). As expected for desymmetrisation reactions, the ee remained essentially constant throughout the reaction.<br>
Example VII. Production of optically active cyciohexane diol from cvclohexene oxide using whole or lysed yeast host cells transformed with the epoxide hydrolase<br>
genes from selected wild type yeasts<br>
Whole cells (Fig. 21 and 24) and lysed cell suspensions (Figs. 22 and 23) of yeast host cells transformed with the epoxide hydrolase gene from Rhodosporidium paludigenum (NCYC 3179) (Figs 21 and 22) or Rhodotorula araucariae (NCYC 3183) (Fig. 23) or Rhodosporidium toruloides (UOFS Y-0471) (Fig. 24) strains under control of the quasi-constitutive hp4d promoter (Madzak et al., 2004) were tested (as Samples 161-164) for their ability to produce optically active cyclohexane diol from cyclohexene oxide. Lysed cell suspensions were prepared by treatment of the cells with Y-PER®, a yeast protein extraction reagent from Sigma-Aldrich (St. Louis, MO, U.S.A.) according to the instructions of the manufacturer. No significant differences in activity or enantioselectivity between and whole and lysed yeast cells were observed.<br>
Example VIII. Desymmetrisation of cyclopentene oxide to produce (R,R)-<br>
cyclopentane diol<br>
64<br><br>
Figs. 25-31 show the desymraetrisation of cyclopentene oxide by various recombinant yeast strains (tested in this Example as Samples 165-171) expressing, under control of the single-copy constitutive TEF promoter or the quasi-constitutive multi-copy hp4d promoter (Madzak et ah, 2004), exogenous epoxide hydrolases from selected wild-type yeast strains. For each example, a line graph is supplied that shows the change in concentrations of the vicinal diol enantiomers with time (left y-axis) and the enantiomeric excess (ee) of the formed diol at different times (right y-axis). As expected for desymmetrisation reactions, the ee remained essentially constant throughout the reaction.<br>
Example IX. Desvmmcti isation of cis-epoxvsuccinate to produce (R,R)-tartaric acid<br>
Table 5 shows the desymmetrisation of cis-epoxysuccinate by various recombinant yeast strains (tested in this Example as Samples 172-182) expressing, under control of the single-copy constitutive TEF promoter or the quasi-constitutive multi-copy hp4d promoter (Madzak et al.t 2004), exogenous epoxide hydrolases from selected wild-type yeast strains.<br>
Reaction conditions: Biomass (25 % w/v) of Yarrowia lipolytica transformants expressing the epoxide hydrolases of different yeast strains was suspended in phosphate buffer (pH 7.5, 50 mM) and incubated with cis-epoxysuccinic acid disodium salt for 4 hours at 25°C. The catalysts was removed by centrifugation and the supernatant was analysed by chiral HPLC to determine the ee of the L-tartaric acid formed.<br>
Figs. 32 show the desymmetrisation of cis epoxysuccinate by a recombinant yeast strain (tested in this Example as Sample 183) expressing, under control of the single-copy constitutive TEF promoter (Madzak et al., 2004), exogenous epoxide hydrolases from thewild-type yeast strain Cryptococcus curvatus NCYC 3158. For this example, a line graph is supplied that shows the change in concentrations of the vicinal diol enantiomers with time (left y-axis) and the enantiomeric excess (ee) of the formed diol at different times (right y-axis). As expected for desymmetrisation reactions, the ee remained essentially constant throughout the reaction.<br>
Table 5. Enantiomeric excesses (ee) of L-tartaric acid after 4 hours of reaction.<br>
65<br><br>
SampleNo.	Yeast strain	Promoter	ee (%)<br>
172	Rhodotorvla mucilaginosa NCYC 3190	TEF	52.7<br>
173	Rhodotorula araucariae'NCYC 3183	TEF	19.6<br>
174	Rhodotorula araucariae NCYC 3183	hp4d	13.0<br>
1175	Rhodosporidium toruloides UOFS Y-0471	TEF	10.5<br>
176	Rhodosporidium paludigenum NCYC 3179	TEF	63.4<br>
177	Rhodosporidium paludigenum NCYC 3179	hp4d	21.3<br>
178	Filobasidium neoformans var neoformans	TEF	47.3<br>
179	Rhodosporidium toruloides NCYC 3181	TEF	23.3<br>
180	Cryptococcus curvatus NCYC 3158	TEF	82.9<br>
181	Yarrowia lipolytica NCYC 3229	TEF	20.7<br>
182	Yarrowia lipolytica NCYC 3229	TEF	79.9<br>
Example X. Desymmetrisation of 3,4-epoxvtetrahvdrofuran to produce (R,R1-<br>
tetrahydro-3«4-fu ran diol<br>
Figs. 33 show the desymmetrisation of 3,4-epoxytetrahydrofuan by a recombinant yeast strain (tested in this Example as Sample 184) expressing, under control of the single-copy constitutive TEF promoter (Madzak et al, 2004), exogenous epoxide hydrolases from thewild-type yeast strain Rhodotorula mucilaginosa NCYC 3190. For this example, a line graph is supplied that shows the change in concentrations of the vicinal diol enantiomers with time (left y-axis) and the enantiomeric excess (ee) of the formed diol at different times (right y-axis). As expected for desymmetrisation reactions, the ee remained essentially constant throughout the reaction.<br>
References<br>
Madzak, C, Gaillardin, C, Beckerich, J-M. (2004). Heterologous protein expression and secretion in the non-conventional yeast Yarrowia lipolytica: a review. Journal of Biotechnology 109: 63-81.<br>
Nicaud J-M, Madzak C, Van den Broek P, Gysler C, Duboc P, Niederberger P, Gaillardin C. (2002) Protein expression and secretion in the yeast Yarrowia lipolytica. FEMS<br>
66<br><br>
Yeast Research 2, 371-279.<br>
Zhao, L., Han, B., Huang, Z., Miller, M., Huang, H., Malashock, D.S., Zhu, Z., Milan, A., Robertson, D.E., Weiner, D.P. &amp; Burk, M.J. (2004). Epoxide hydrolase-catalyzed enantioselective synthesis of chiral ls2-diols via desymmetrization of meso-epoxides. J. Am. Chem. Soc, 126: 11156-11157.<br>
A number of embodiments of the invention have been described. Nevertheless, it will be understood that various modifications may be made without departing from the spirit and scope of the invention. Accordingly, other embodiments are within the scope of the following claims.<br>
67<br><br>
SEQ. ID. NO. 1<br>
Rhodosporidium toruloides NCYC 3181 (#1)<br>
Amino acid sequence<br>
ORIGIN<br>
1	MATHTFASPP TRFTVDIPQS	ELDELDFRLD	KTRWPATEIV PEDGADDPTA FGLGAGPTLP<br>
61	LMKELAKGWR EFDWKKAQDH	LNTFEHYTVE	IEDLSIHFLH HRSTRPNAVP LILCHGWPGH<br>
121	FGEFLNVIPL LTEPSDPSAQ	AFHVVAPSMP	GYAWSSPPPS SKWSMPDTAR VFDKLMTGLG<br>
181	YEKYMAQGGD WGSIAARCLG	SLHKDHCKAV	HLNFLPVFPP VPMWLINPHT LLAWAPRFLV<br>
241	PEKQAARMKR GLAYLEKGSA	YYVMQQLTPR	TPAYGLTDSP VGLLAWIGEK FEPTIQEASK<br>
301	QAQPTLTRDE LYFTCSLYWF	TRSIGTSFLP	YSLNPHFTTF LTDSSTTCPT LPCPSTRARS<br>
361	TARQNGTLSV PATSSDQGRA	GGRTLCCARE	ARRVCRPSQG GVRRHVGEV<br>
SEQ. ID. NO. 2<br>
Rhodosporidium toruloides UOFS Y-0471 (#46)<br>
Amino acid sequence<br>
ORIGIN<br>
1	MKMATHTFAS	PPTRFTVDIP QSELDELHSR LDKTRWPATE	IVPEDGTDDP TAFGLGAGPT<br>
61	LPLMKELAKG	WREFDWKKAQ DHLNTFEHYM VEIEDLSIHF	LHHRSTRPNA VPLILCHGWP<br>
121	GHFGEFLNVI	PLLTEPSDPS AQAFHVVAPS MPGYAWSLPP	PSSKWNMPDT ARVFDKLMTG<br>
181	LGYEKYMAQG	GDWGSIAARC LGSLHKDHCK AVHLNFLPVF	PPVPMWLINP HTLLAWAPRF<br>
241	LVPEKQAARM	KRGLAYLEKG SAYYVMQQLT PRTPAYGLTD	SPVGLLAWIG EKFEPTIQEA<br>
301	SKQAQPTLTR	DELYFTCSLY WFTRSIGTSF LPYSLNPHFT	TFLTDSKYHL PNFALSLYPG<br>
361	EIYCPAERDA	KRTGNLKWIK DAPEGGHFAA LEKPDVFVEH	LREAFGVMWE K<br>
68<br><br>
SEQ. ID. NO. 3<br>
Rhodotorula araucariae NCYC 3183 (#25)<br>
Amino acid sequence<br>
ORIGIN<br>
1	MSEHSFEAPP QPFTVDFAPH IEDLHRRLDN ARWPTQEIVP VDVSETEHHN AFGLGMGPQL<br>
61	NLMKELANGW RAFDQSALQD HLNSFNNWKV EIEGLSIHFL HHRSTRAGAL PLILCHGWPG<br>
121	GYHEFLHVVQ LLTEPEGADA QAFHLVVPSM PGYAFSSPPP TAKWGMEDTA RVFDKLMTGL<br>
181	GYNKYVAQGG DWGSITARCL GALHKDHCIA VHLNFCPVPP PFPFDQFNPR TLLNWMPRFV<br>
241	ISDQQRAKLE RGLAYLEKGS AYYVMQQLTP RTPAYALNDS PIGLLAWIGE KMIPGINEAS<br>
301	AQPNPTLNRD ALYTTLSLYW FTNSIGTSFL PYSLNPHFST FLTSPRYRLP NFALSSFPGE<br>
361	LFSPTPRDAA RTGVMRWYKE ADDGGHFAAL EKPDVFSQHV REAVKAMLSS<br>
SEQ. ID. NO. 4<br>
Crvptococcus curvatus NCYC 3158 (Car 054)<br>
Amino acid sequence<br>
ORIGIN<br>
1	MATHTFASPP TRFTVDIPQS EVDQLHSRLD KTRWPGTEIV	PEDGADDPTA FGLGAGPTLP<br>
61	LMKELAKGWR DFDWKKAQDH LNTFEHYTVE IEDLSIHFLH	HRSTRPNAVP LILCHGWPGH<br>
121	FGEFLHVIPL LTEPSDPSAQ AFHVVAPSMP GYAWSSPPPT	SKWNMPDTAR VSDKLMNGLG<br>
181	YEKYMAQGGD WGSIAARCLG ALHKDHCKAV HLNFLPVFPP	VPMWLINPHT LLAWAPRFLV<br>
241	PEKQAARMKR GLAYLEKGSA YYVMQQLTPR TSAYGLTDSP	VGLLAWIGEK FEPTIQEASK<br>
301	QAQPTLTRDE LYFTCSLYWF TRSIGTSFLP YSLNPHFTTF	LTDSKYYLPN FALSLYPGEI<br>
361	YCPAERDAKR TGNLKWIKDA PEGGHFAALE KPDVFVDHLR	EAFGVMWEK<br>
SEQ. ID. NO.5<br>
Rhodosporidium paludiqenum NCYC 3179 (#692)<br>
Amino acid sequence<br>
ORIGIN<br>
1	MAAHSFTAPP APYNIDFAPQ	VNDLHRRLDA ARWPEHDVVP DDVDHGEHGA FGLGAGPSLA<br>
61	LMKELAQEWR GQDQKQLQDH	LNSYKNYRVE IEGLNIHFLH YPSSRADAFP LILCHGWPGG<br>
121	YHEFLHVLER LTEPEDQGSR	AFHVVVPSMP GYAFSSPPKT AKWGMEDTAR VFDKLMTGLG<br>
181	YAKYAAQGGD WGSITARCLG	SLHKENCVAV HLNFCPVPPP FPLNMFNPRT LLDWMPRFVL<br>
241	PDQRRAKIER GVAYIERGSA	YYAMQNLTPR TPAYGLNDSP IGLLAWIGEK MIPGIDKAVK<br>
301	HPNATLNREA LFTTLSIYWF	TGSIGSSFLP YALNPHFSTF LVSPRHQLPN FALSNFPDEL<br>
69<br><br>
361 FTPEERDARR TGNLRWYKDA EDGGHFAALE KPEVFAEHVR EAMGVLLSNQ A<br>
SEQ. ID. NO. 6<br>
Rhodotorula mucilaainosa NCYC 3190 (#23)<br>
Amino acid sequence<br>
ORIGIN<br>
1	MPARSLTLRP FSPSFTAPEL DGLARSLESS RLPAETYASR QAKYGIKHAW	MKNALQRWKD<br>
61	GFDWKKHEQD INEVDHYMVQ VQSDGIQHDL HVIYHESKDP NAIPLLLLHG	WPGSAFEFIE<br>
121	AIKILRKSTS PAFHLIAPME PGYGWSTPPP LDRGFNMNDC TALMNDLMVG	LGYGDGYAAQ<br>
181	GGDIGSGLAR LLAVNYDACK CININYMPAV APPEDAPERH QIKPHEEDAL	RRADEFQKTG<br>
241	RGYANMHATR PGTVGIVVGS SPVALLAWIA EKYLAWTDED PPLDTILAIC	TIWWIRDSYP<br>
301	SSIWAYADFL ETGISALHND PKYKLDKKPF GFSSFKEEIS ATPEAWAGRN	GNLQFYRYHD<br>
361	KGGHFAALEQ PEAFAQDMQD CFGKIWPLSQ EQKS<br>
SEQ. ID. NO. 7<br>
Yarrowia I'molvtica NCYC 3229 -2<br>
Amino acid sequence<br>
ORIGIN<br>
1	MSSLDLSYYE	PFHRNAVLNG	KRWHYVDIPG DSSGSLGRGK TLLLVHGFPD TWYGWRHQVP<br>
61	VFRKHGFRLL	IPSLLGFPRS	EAPLTHPGVA TGEKFDGHNV HKELGLEDEN IQELECYTAD<br>
121	FFAKSMVQLL	DQLGIDKVCS	FGHDWGAVFA PRLWLNYPER VECVSSACWY YQMPMEGFVD<br>
181	LKDFAEVAPS	LRYQLYWGGD	APQEVIQKPL KEFMDRMYHH PTENHLLTSE AEYNNILKEF<br>
241	QYGGKTIAPM	FPLYKSRKVA	YDIDERDFLT KGRTEEDLAV DVPYLFIGAE FDIALQPGME<br>
301	SVLEGYVKEG	LLEKQWVPCG	HWVLFEEPEK MNKIYIDFLK KVFGSSKL<br>
SEQ. ID. NO. 8<br>
Debaromvces hansenii NCYC 3167 (#113)<br>
Amino acid sequence<br>
ORIGIN<br>
1	MMQGQYKIQL RNGVRIFSTF SNYDNHDVFC SGGERKWTRV	IFLLHGFPDN NSSYNSVWTV<br>
61	ILDSVPEDEK VLLLAPSMRG YEPSSQGEQC DYKMSDLAGD	VSSWIKNISP DGIPVHLVGH<br>
121	DWGAIVAFKT ASMYPEMITS MACLAIPYIT NIRLWEFAWY	FPDQIYHSSY MFTMQFGWLY<br>
181	KKRLSDTSSN SYLDGLWRCW SPTWNYTVDE IDSVKRTLQE	PGVIDASTAY YRCIANPLNI<br>
241	SNRRWNVDFK KVPTLLLGGE RDGCMNEKLY RLEASKLANN	PYVKVKTISN VGHFLHREDP<br>
301	TKVGELITDW FSEYP*<br>
70<br><br>
SEQ. ID. NO. 9<br>
Filobasidium neoformans var neoformans (#777) (GenBank XM 568708)<br>
Amino acid sequence<br>
MSYSDLPHKP TIPVEPFKLS VPHEDLNGLL TLLKSTRIAK ESYENVSAHE NKFGITRKWL<br>
VNMKDEWIKQ DWRKQEERIN SLPAFKAKVK NSDGSVFSIH FTALFSKKKV AIPIILSHGW<br>
I PGSFYEFVPM MEMVKKKYSP EDLPFHLIVP SLPGWLFSTP PPNDREFNVT DVGYLFNGLM<br>
EGLGFGDGYV AQGGDIGSYV TNELGAKYPA CKIIHVNYSN PPPRPLPSPG	SPGQEASpPS<br>
AEDLLELLQK FGYALEHSTR PATVGLVVGS NPLSLLAWVG EKFLEWTDES	PSEETILTMT<br>
SLYWFTDCFT TSIYTYRYGL GAKRHESAKQ ASYQKCPLGY SQFPKEIVEI	PAEWVKAQSN<br>
MVWSKKHESG GHFAALEKPE LLWADIEEFV HSQWENYKGN Y<br>
SEQUENCE ID NO 10<br>
Rhodosporidium toruloides NCYC 3181 (#1)<br>
Nucleotide sequence<br>
ORIGIN<br>
1	ATGGCGACAC ACACATTCGC TTCGCCTCCC ACGCGCTTCA CCGTCGACAT CCCACAGTCA<br>
61	GAACTCGACG AACTCGACTT CCGACTCGAC AAGACCCGCT GGCCGGCGAC AGAGATCGTT<br>
121	CCAGAGGATG GGGCGGACGA CCCGACGGCG TTTGGGCTCG GAGCAGGGCC GACGCTGCCG<br>
181	CTCATGAAGG AACTGGCGAA GGGTTGGCGC GAGTTCGACT GGAAGAAGGC GCAGGACCAC<br>
241	CTCAACACCT TTGAGCACTA CACGGTCGAA ATCGAGGACC TCTCCATCCA CTTCCTCCAC<br>
301	CACCGCTCGA CTCGCCCGAA TGCTGTTCCG CTCATCCTCT GCCACGGCTG GCCAGGCCAC<br>
361	TTCGGCGAGT TCCTGAACGT CATACCGCTC TTGACGGAGC CGTCGGACCC GTCCGCCCAG<br>
421	GCGTTCCACG TCGTCGCGCC TTCGATGCCC GGTTATGCTT GGTCTTCGCC TCCTCCGTCC<br>
481	TCCAAGTGGA GCATGCCTGA TACCGCGAGG GTCTTCGACA AGCTCATGAC CGGGCTTGGC<br>
541	TACGAGAAGT ACATGGCGCA GGGCGGAGAC TGGGGCAGCA TCGCTGCTCG CTGCCTTGGA<br>
601	TCGCTTCACA AAGACCACTG CAAAGCCGTC CACCTCAACT TCCTCCCCGT CTTCCCACCC<br>
661	GTCCCGATGT GGCTTATCAA CCCGCACACG CTCCTTGCCT GGGCACCGCG CTTCCTCGTG<br>
721	CCGGAGAAGC AGGCTGCGCG TATGAAGCGC GGGTTGGCGT ACCTTGAGAA GGGCTCCGCC<br>
781	TACTACGTCA TGCAGCAGTT GACGCCTCGC ACGCCTGCGT ACGGCCTGAC CGACAGTCCC<br>
841	GTCGGCTTGC TGGCCTGGAT CGGCGAGAAG TTCGAGCCGA CCATTCAGGA GGCGAGCAAG<br>
901	CAAGCCCAGC CGACCCTGAC TCGCGACGAG CTCTACTTCA CCTGCTCGCT CTACTGGTTC<br>
961	ACCCGCTCAA TCGGCACCTC CTTCCTTCCC TACTCGCTCA ACCCGCACTT CACCACCTTC<br>
1021	CTGACCGACA GCAGGTACCA CCTGCCCAAC TTTGCCCTGT CCCTCTACCC GGGCGAGATC<br>
1081	TACTGCCCGG CAGAACGGGA CGCCAAGCGT ACCGGCAACC TCAAGTGGAT CAAGGACGCG<br>
1141	CCCGATGGAG GACACTTTGC TGCGCTCGAG AAGCCCGATG TGTTTGTCGA GCATCTCAGG<br>
1201	GAGGCGTTTG GCGTCATGTG GGAGAAGTAG<br>
71<br><br>
SEQUENCE ID 11<br>
Rhodosooridium toruloides UQFS Y-0471 (#46)<br>
Nucleotide sequence<br>
ORIGIN<br>
1	ATGGCGACAC ACACATTCGC TTCGCCTCCC ACCCGCTTCA	CCGTCGACAT CCCACAGTCG<br>
61 GAACTCGACG AACTTCACTC GCGACTCGAC AAGACCCGCT	GGCCGGCGAC AGAGATCGTT<br>
121 CCAGAGGATG GGACGGACGA TCCGACGGCG TTTGGGCTCG	GAGCAGGGCC GACGCTGCCG<br>
181 CTCATGAAGG AATTGGCGAA GGGTTGGCGC GAGTTCGACT	GGAAAAAGGC GCAGGACCAC<br>
2	41 CTCAACACCT TCGAGCACTA CATGGTCGAA ATTGAGGACC	TCTCGATCCA CTTCCTCCAC<br>
301 CATCGCTCGA CTCGCCCGAA CGCTGTTCCC CTCATCCTCT	GCCACGGCTG GCCAGGCCAC<br>
361 TTTGGCGAGT TCCTGAACGT TATCCCGCTC TTGACGGAGC	CGTCGGACCC CTCCGCTCAG<br>
421 GCGTTCCACG TCGTCGCCCC TTCGATGCCT GGCTATGCTT	GGTCTTTGCC TCCTCCGTCC<br>
4 81 TCCAAGTGGA ACATGCCTGA CACCGCGAGG GTCTTCGACA AGCTCATGAC CGGGCTTGGC<br>
541 TACGAGAAGT ACATGGCGCA GGGCGGAGAC TGGGGAAGCA TCGCCGCTCG CTGCCTTGGA<br>
601 TCGCTGCACA AGGACCATTG CAAAGCCGTC CACCTCAACT	TCCTCCCCGT CTTCCCACCC<br>
661 GTCCCGATGT GGCTTATCAA CCCGCACACG CTCCTTGCCT	GGGCACCGCG CTTCCTCGTG<br>
721 CCGGAGAAGC AGGCTGCGCG TATGAAGCGC GGGTTGGCGT	ACCTTGAGAA GGGCTCCGCC<br>
7 81 TACTACGTCA TGCAGCAGTT GACGCCTCGC ACGCCTGCGT	ACGGCCTGAC CGACAGTCCC<br>
841 GTCGGCTTGC TGGCCTGGAT CGGCGAGAAG TTCGAGCCGA	CCATTCAGGA GGCGAGCAAG<br>
901 CAAGCCCAGC CGACCCTGAC TCGCGACGAG CTCTACTTCA	CCTGCTCGCT CTACTGGTTC<br>
961 ACCCGCTCAA TCGGCACCTC CTTCCTTCCC TACTCGCTCA	ACCCGCACTT CACCACCTTC<br>
1021 CTGACCGACA GCAAGTACCA CCTGCCCAAC TTTGCCCTCT	CGCTTTACCC AGGCGAGATC<br>
1081 TACTGCCCCG CCGAGCGGGA CGCCAAGCGC ACCGGCAACC	TCAAGTGGAT CAAGGACGCG<br>
1141 CCTGAGGGAG GACACTTTGC TGCGCTCGAA AAGCCGGATG	TGTTTGTCGA GCACCTCAGG<br>
1201 GAGGCGTTTG GCGTCATGTG GGAGAAGTAG<br>
72<br><br>
SEQUENCE ID NO 12<br>
Rhodotorula araucahae NCYC 3183 (#25)<br>
Nucleotide sequence<br>
ORIGIN<br>
1	ATGAGCGAGC ACAGCTTCGA GGCCCCGCCA CAGCCGTTTA CGGTCGACTT TGCTCCCCAC<br>
61	ATCGAGGATC TCCACCGCCG TCTCGACAAT GCGCGCTGGC CGACGCAAGA GATTGTCCCC<br>
121	GTCGACGTGT CCGAGACGGA GCATCACAAC GCGTTCGGAC TCGGGATGGG CCCGCAGCTC<br>
181	AACCTTATGA AGGAGCTCGC CAACGGCTGG CGCGCGTTCG ACCAGTCGGC GCTCCAGGAC<br>
241	CACCTCAACA GCTTCAACAA CTGGAAGGTC GAGATCGAGG GATTGTCGAT CCACTTCCTC<br>
301	CACCATCGCT CGACGCGCGC CGGCGCTCTC CCGCTCATCC TGTGCCATGG CTGGCCCGGC<br>
361	GGGTACCACG AGTTCCTCCA CGTCGTCCAG CTCCTCACCG AACCAGAGGG GGCGGATGCG<br>
421	CAGGCGTTTC ACCTCGTCGT CCCCTCGATG CCCGGGTACG CCTTCTCGTC TCCGCCGCCG<br>
4 81	ACGGCCAAGT GGGGCATGGA AGACACTGCA AGGGTTTTTG ACAAGCTCAT GACCGGTTTG<br>
541	GGGTACAACA AGTATGTCGC GCAGGGCGGT GACTGGGGGT CCATCACGGC GCGATGCCTC<br>
601	GGCGCGCTGC ACAAGGACCA CTGCATTGCT GTCCACCTCA ACTTCTGCCC CGTCCCGCCG<br>
661	CCGTTCCCAT TCGACCAGTT CAACCCGCGC ACGCTGCTCA ACTGGATGCC GCGCTTCGTG<br>
721	ATCTCGGACC AGCAGCGTGC GAAGCTCGAG CGTGGGCTGG CGTACCTCGA GAAGGGGTCT<br>
781	GCTTACTATG TCATGCAGCA GCTCACACCG CGTACCCCGG CCTACGCTCT CAATGACAGC<br>
841	CCGATTGGCC TGCTCGCCTG GATTGGCGAA AAGATGATCC CAGGCATCAA CGAGGCGAGC<br>
901	GCGCAGCCGA ACCCGACGCT CAATCGCGAT GCGTTGTACA CCACGCTCTC GCTGTACTGG<br>
961	TTCACCAACT CCATCGGCAC CTCTTTCCTC CCCTACTCGC TTAACCCGCA CTTCAGCACG<br>
1021	TTCCTCACCT CGCCCCGCTA TCGCCTGCCG AACTTTGCGC TGTCTTCCTT CCCGGGCGAG<br>
1081	CTGTTCTCGC CGACGCCGCG CGATGCTGCG AGGACGGGCG TGATGCGCTG GTACAAGGAG<br>
1141	GCGGACGATG GCGGGCACTT TGCGGCGCTC GAGAAGCCCG ATGTGTTCAG CCAGCATGTC<br>
1201	AGGGAGGCAG TCAAGGCCAT GCTGTCGTCG TGA<br>
73<br><br>
SEQUENCE ID NO 13<br>
Cryptococcus curvatus NCYC 3158 (Car 054)<br>
Nucleotide sequence<br>
ORIGIN<br>
1	ATGGCGACAC ACACATTCGC TTCGCCTCCC ACCCGCTTCA CCGTCGACAT TCCGCAGTCG<br>
61	GAAGTTGACC AACTTCACTC GCGACTCGAT AAGACTCGCT GGCCAGGGAC AGAGATCGTT<br>
121	CCAGAGGATG GGGCGGACGA CCCGACGGCG TTTGGACTCG GAGCAGGACC AACGCTGCCG<br>
181	CTCATGAAGG AACTGGCGAA GGGTTGGCGC GACTTCGACT GGAAGAAGGC GCAGGACCAC<br>
241	CTCAACACCT TCGAGCACTA CACGGTGGAG ATCGAGGACC TCTCGATCCA CTTCCTCCAC<br>
301	CATCGCTCGA CTCGTCCGAA TGCTGTTCCC CTCATCCTCT GCCACGGCTG GCCAGGACAC<br>
361	TTCGGCGAGT TTCTACACGT TATACCACTC TTGACGGAGC CGTCGGACCC GTCCGCTCAG<br>
421	GCGTTTCACG TCGTCGCGCC TTCGATGCCT GGTTATGCTT GGTCTTCGCC TCCTCCTACC<br>
481	TCCAAGTGGA ATATGCCTGA CACCGCAAGG GTGTCCGACA AGCTTATGAA CGGACTCGGC<br>
541	TACGAGAAGT ACATGGCGCA GGGCGGAGAC TGGGGCAGCA TCGCCGCTCG CTGCCTCGGA<br>
601	GCGCTGCACA AAGATCACTG CAAAGCCGTC CACCTCAACT TCCTCCCCGT CTTCCCGCCC<br>
661	GTCCCTATGT GGCTCATCAA CCCACACACA CTCCTCGCAT GGGCTCCGCG CTTCCTTGTG<br>
721	CCGGAGAAGC AGGCTGCGCG CATGAAGCGC GGGTTGGCGT ACCTCGAAAA GGGCTCCGCC<br>
781	TACTACGTCA TGCAGCAATT GACGCCTCGC ACGTCTGCGT ACGGCCTTAC TGACAGTCCC<br>
841	GTCGGCTTGC TGGCCTGGAT CGGCGAGAAG TTTGAGCCGA CCATTCAGGA AGCGAGCAAG<br>
901	CAAGCCCAGC CGACCCTCAC TCGCGACGAG CTCTACTTCA CCTGCTCCTT GTACTGGTTC<br>
961	ACCCGCTCAA TCGGCACCTC CTTCCTGCCC TACTCGCTCA ACCCGCACTT CACGACCTTC<br>
1021	CTGACCGACA GCAAGTATTA CCTGCCCAAT TTTGCCCTCT CCCTGTACCC GGGCGAGATC<br>
1081	TACTGCCCTG CCGAGCGGGA CGCCAAGCGC ACTGGCAACC TCAAGTGGAT CAAGGACGCG<br>
1141	CCTGAGGGAG GACACTTTGC GGCGCTCGAG AAGCCCGACG TGTTTGTCGA CCATCTCAGG<br>
1201	GAGGCATTTG GCGTTATGTG GGAGAAGTAG<br>
74<br><br>
SEQUENCE ID NO 14<br>
Rhodosporidium paludiaenum NCYC 3179 ( #692)<br>
Nucleotide sequence<br>
ORIGIN<br>
1	ATGGCTGCCC ATTCCTTTAC TGCACCTCCT GCACCCTACA ACATCGACTT TGCGCCCCAG<br>
61	GTAAATGACC TGCACCGCCG TCTCGATGCT GCCCGCTGGC CGGAACACGA CGTGGTGCCC<br>
121	GACGATGTGG ATCACGGAGA GCACGGCGCA TTCGGACTCG GCGCTGGTCC CAGCCTCGCC<br>
181	CTCATGAAGG AGCTCGCGCA GGAATGGAGG GGCCAGGACC AGAAGCAGCT GCAGGACCAC<br>
241	CTCAACTCCT ACAAGAACTA TCGCGTCGAG ATCGAGGGTC TCAACATCCA CTTCCTGCAC<br>
301	TACCCGTCGT CTCGCGCCGA TGCGTTCCCG CTCATCCTGT GCCACGGCTG GCCTGGCGGC<br>
361	TACCACGAGT TCCTGCACGT CCTAGAGCGC CTCACGGAGC CCGAGGATCA GGGGTCGCGG<br>
421	GCCTTCCATG TCGTCGTGCC TTCCATGCCG GGTTACGCCT TCTCCTCGCC GCCCAAGACG<br>
481	GCAAAATGGG GCATGGAGGA CACGGCTCGC GTGTTCGACA AGCTCATGAC GGGGCTAGGT<br>
541	TACGCCAAGT ATGCGGCCCA AGGCGGTGAC TGGGGGTCTA TCACGGCGCG CTGCCTAGGT<br>
601	TCGCTGCACA AGGAGAACTG CGTCGCTGTC CACCTCAACT TCTGCCCGGT TCCCCCGCCG<br>
661	TTCCCGCTCA ACATGTTCAA CCCGCGCACA CTTCTGGACT GGATGCCTCG CTTTGTCCTG<br>
721	CCTGATCAAC GGCGGGCCAA GATTGAGCGC GGCGTGGCCT ATATCGAGCG CGGCTCTGCC<br>
781	TACTACGCCA TGCAAAACTT GACGCCGCGC ACGCCTGCGT ACGGCTTGAA CGATAGTCCG<br>
841	ATTGGTTTGC TCGCGTGGAT TGGCGAGAAG ATGATTCCGG GCATTGACAA GGCTGTCAAG<br>
901	CATCCGAACG CAACCCTCAA TCGCGAAGCT CTTTTCACGA CACTCTCGAT CTACTGGTTC<br>
961	ACGGGCTCGA TTGGCTCCTC CTTCCTGCCA TACGCTCTCA ACCCGCACTT CTCTACCTTC<br>
1021	CTCGTCTCGC CGCGGCACCA ACTGCCGAAC TTTGCTCTGT CCAACTTTCC CGACGAGCTG<br>
1081	TTCACGCCCG AAGAACGCGA TGCTCGCCGA ACCGGAAACT TGCGGTGGTA CAAGGATGCA<br>
1141	GAGGATGGAG GGCACTTCGC GGCGCTGGAG AAGCCCGAGG TCTTCGCCGA GCACGTAAGG<br>
1201	GAGGCGATGG GGGTCTTGCT GTCGAACCAG GCCTGA<br>
75<br><br>
SEQUENCE ID NO 15<br>
Rhodotorula mucitaginosa NCYC 3190 (#23)<br>
Nucleotide sequence<br>
ORIGIN<br>
1	ATGCCCGCCC GCTCGCTCAC GCTGCGCCCG TTCTCGCCGT CGTTCACGGC TCCGGAACTG<br>
61	GACGGTCTCG CTCGCTCGCT CGAGTCGTCG CGCTTGCCCG CCGAGACGTA CGCTTCCCGC<br>
121	CAGGCCAAAT ACGGCATCAA GCATGCTTGG ATGAAGAATG CCCTCCAACG GTGGAAGGAC<br>
181	GGGTTCGATT GGAAGAAGCA CGAGCAGGAC ATCAACGAGG TCGACCACTA TATGGTGCAG<br>
241	GTCCAGTCCG ATGGCATTCA ACACGACCTC CATGTGATCT ATCACGAATC GAAAGACCCG<br>
301	AATGCGATCC CGCTCTTGCT GCTGCATGGG TGGCCCGGTT CCGCGTTCGA GTTTATCGAG<br>
361	GCGATCAAGA TCCTTCGCAA GAGTACCTCG CCCGCGTTCC ACCTGATCGC GCCCATGGAG<br>
421	CCCGGCTACG GGTGGAGTAC TCCGCCGCCA CTCGACCGCG GTTTCAACAT GAACGATTGC<br>
481	ACAGCGCTCA TGAACGACTT GATGGTTGGA CTCGGGTACG GAGACGGTTA CGCTGCTCAG<br>
541	GGTGGCGACA TCGGTTCGGG ACTCGCAAGA CTCCTCGCCG TCAACTATGA CGCATGCAAA<br>
601	TGCATCAACA TCAACTACAT GCCTGCCGTT GCACCACCAG AGGACGCTCC GGAGCGGCAC<br>
661	CAGATCAAAC CGCACGAGGA GGATGCGCTC CGACGTGCGG ACGAGTTTCA GAAGACGGGC<br>
721	AGGGGGTATG CCAACATGCA TGCAACGAGA CCCGGTACGG TCGGCATCGT CGTCGGTAGT<br>
781	TCGCCGGTCG CACTGCTCGC TTGGATCGCT GAAAAGTACC TCGCTTGGAC CGATGAGGAT<br>
841	CCGCCCCTCG ACACGATCCT CGCAATCTGC ACCATCTGGT GGATCCGCGA CTCGTACCCT<br>
901	TCTTCAATCT GGGCCTACGC CGACTTTCTC GAGACGGGCA TCTCGGCCCT GCACAACGAC<br>
961	CCGAAGTACA AACTTGACAA GAAACCGTTC GGGTTCTCGA GCTTCAAGGA GGAGATCAGC<br>
1021	GCGACTCCCG AGGCGTGGGC GGGCAGGAAC GGCAACTTGC AGTTCTATCG GTACCACGAC<br>
1081	AAGGGAGGTC ACTTTGCGGC GCTCGAGCAG CCGGAAGCGT TCGCGCAAGA CATGCAGGAT<br>
1141	TGCTTCGGCA AAATCTGGCC TCTCTCTCAG GAGCAAAAAT CGTAG<br>
76<br><br>
SEQUENCE ID NO 16<br>
Yarrowia lipolvtica NCYC 3229-2<br>
Nucleotide sequence<br>
ORIGIN<br>
1	ATGTCATCAC TCGACCTATC TTATTACGAA CCGTTCCACA GAAATGCCGT TCTTAATGGC<br>
61	AAAAGATGGC ACTATGTAGA CATTCCTGGA GACTCCAGTG GCTCTCTTGG TAGAGGAAAG<br>
121	ACACTTTTGC TGGTCCATGG GTTCCCAGAC ACTTGGTACG GGTGGCGGCA TCAAGTGCCT<br>
181	GTGTTCCGTA AGCACGGATT TCGGCTCTTG ATTCCCTCCT TGCTTGGTTT CCCAAGAAGT<br>
241	GAAGCTCCTC TTACTCACCC TGGAGTTGCC ACAGGAGAGA AGTTCGATGG TCACAATGTT<br>
301	CACAAGGAGC TGGGTCTGGA AGATGAAAAC ATTCAGGAAC TTGAATGTTA CACGGCTGAC<br>
361	TTCTTTGCCA AGAGCATGGT CCAACTGCTG GACCAATTGG GGATCGACAA GGTCTGTTCC<br>
421	TTTGGACATG ACTGGGGAGC TGTCTTTGCT CCCCGACTCT GGCTCAATTA TCCTGAGAGA<br>
4 81	GTCGAATGTG TGTCATCAGC ATGCTGGTAC TACCAAATGC CAATGGAGGG CTTCGTCGAC<br>
541	TTGAAAGATT TCGCAGAGGT GGCTCCTTCA TTGAGATACC AATTGTACTG GGGAGGAGAC<br>
601	GCTCCTCAAG AAGTCATCCA GAAGCCGCTC AAGGAATTTA TGGACAGAAT GTACCACCAT<br>
661	CCTACAGAGA ATCATCTTCT GACATCGGAA GCTGAATACA ACAACATTCT CAAGGAGTTT<br>
721	CAATATGGAG GAAAGACCAT TGCTCCCATG TTTCCTCTCT ACAAGTCACG AAAGGTAGCC<br>
781	TACGATATCG ATGAAAGGGA CTTTCTGACC AAGGGGAGAA CAGAGGAAGA CTTGGCGGTG<br>
841	GATGTGCCGT ACTTGTTTAT CGGGGCCGAG TTTGACATTG CTCTTCAGCC CGGAATGGAG<br>
901	TCTGTTTTGG AGGGTTATGT GAAGGAGGGA CTACTAGAGA AGCAATGGGT CCCCTGTGGA<br>
961	CATTGGGTCT TGTTTGAGGA GCCTGAGAAG ATGAACAAGA TTTATATTGA CTTTCTGAAG<br>
1021	AAGGTGTTTG GATCGTCGAA GTTGTAG<br>
77<br><br>
SEQ.ID.NO. 17<br>
Debaromvces hansenii NCYC 3167 (#113)<br>
Nucleotide sequence<br>
ORIGIN<br>
1 ATGATGCAAG GTCAATACAA GATTCAACTT CGCAACGGAG TAAGAATATT	TTCTACCTTC<br>
61 AGCAATTATG ATAACCACGA TGTATTTTGT AGTGGTGGTG AGCGTAAATG	GACTCGAGTT<br>
121 ATATTCCTCC TCCATGGGTT TCCCGATAAT AACAGTTCAT ATAATAGCGT	TTGGACAGTG<br>
181 ATATTAGATT CTGTCCCAGA AGATGAAAAG GTATTGTTGC TAGCGCCATC	AATGAGAGGT<br>
241 TACGAACCGT CTAGTCAAGG TGAACAATGT. GATTATAAGA TGTCAGATCT	TGCGGGCGAT<br>
301 GTAAGCTCAT GGATTAAAAA TATTTCACCA GATGGTATTC CAGTACACCT	TGTTGGTCAT<br>
361 GATTGGGGAG CGATAGTGGC TTTTAAGACC GCTAGTATGT ACCCTGAGAT	GATAACATCA<br>
421 ATGGCTTGTT TGGCCATTCC ATATATCACC AATATTCGAT TATGGGAGTT	TGCATGGTAT<br>
481 TTTCCAGACC AGATTTATCA TTCCAGCTAT ATGTTCACCA TGCAGTTTGG	ATGGCTCTAT<br>
541 AAAAAGAGGC TTTCTGATAC GAGTAGCAAC AGTTATTTGG ACGGATTATG	GCGCTGCTGG<br>
601 TCTCCGACAT GGAATTATAC CGTAGATGAA ATCGATAGTG TTAAGAGAAC	ACTTCAAGAA<br>
661 CCTGGAGTTA TCGATGCATC AACTGCATAC TACAGATGTA TTGCTAACCC	CCTAAACATT<br>
721 TCTAATAGGC GCTGGAACGT AGACTTTAAA AAAGTTCCAA CGTTGTTGCT	CGGCGGAGAG<br>
781 CGGGATGGGT GCATGAATGA AAAGTTGTAT CGTTTAGAAG CCAGTAAGCT	TGCAAATAAT<br>
841 CCATATGTTA AGGTCAAGAC AATTTCAAAT GTCGGGCATT TCTTGCATCG	TGAAGACCCT<br>
901 ACAAAAGTTG GTGAATTGAT AACTGACTGG TTCAGTGAAT ATCCTTAG<br>
78<br><br>
SEQ.ID.NO. 18<br>
Filobasidium neoformans var neoformans (#777)<br>
Nucleotide sequence<br>
ORIGIN<br>
1	ATGTCGTATT CAGACCTTCC CCACAAGCCG ACCATTCCTG TCGAGCCCTT CAAACTATCC<br>
61	GTACCCCATG AGGACCTCAA TGGGCTCCTC ACTCTACTCA AATCTACCAG AATTGCCAAA<br>
121	GAAAGCTATG AGAATGTCTC TGCGCACGAG AACAAATTTG GAATAACGAG GAAATGGCTG<br>
181	GTGAATATGA AAGATGAATG GATCAAACAG GACTGGCGGA AACAAGAAGA GCGAATCAAT<br>
241	TCTCTTCCTG CGTTCAAGGC AAAGGTAAAA AACTCAGATG GTTCGGTATT TTCAATCCAT<br>
301	TTTACCGCCC TTTTCTCGAA AAAGAAGGTC GCAATCCCTA TCATCCTCAG TCACGGCTGG<br>
361	CCAGGTAGCT TTTACGAATT CGTTCCAATG ATGGAGATGG TGAAGAAGAA ATATAGTCCT<br>
421	GAGGATCTTC CGTTCCATTT AATCGTACCA TCCCTTCCCG GATGGCTCTT CTCTACACCT<br>
481	CCTCCTAATG ATCGAGAATT CAATGTGACC GATGTTGGGT ATCTCTTCAA TGGTCTAATG<br>
541	GAGGGCCTAG GTTTCGGAGA CGGCTATGTT GCCCAAGGCG GTGATATTGG GAGCTACGTG<br>
601	ACGAACGAAC TTGGTGCCAA GTACCCAGCT TGCAAAATCA TTCACGTCAA TTACTCTAAC<br>
661	CCTCCTCCTC GCCCACTTCC CTCTCCAGGC TCTCCTGGAC AAGAGGCTTC ACCGCCCTCT<br>
721	GCTGAAGATC TTCTTGAGCT ACTGCAGAAG TTTGGATATG CTCTCGAGCA TTCTACTCGA<br>
781	CCAGCCACTG TAGGCTTGGT AGTAGGCTCC AATCCACTCA GCTTACTAGC ATGGGTCGGC<br>
841	GAGAAATTCC TCGAGTGGAC TGATGAGTCC CCMGTGAGG AAACAATTTT GACAATGACA<br>
901	AGCCTTTATT GGTTCACTGA TTGTTTCACG ACGTCGATTT ACACATACAG ATATGGCCTT<br>
961	GGTGCTAAAC GCCATGAAAG TGCCAAACAA GCTTCTTACC AGAAATGTCC TCTGGGCTAC<br>
1021	AGCCAGTTCC CCAAAGAGAT CGTCGAGATA CCTGCCGAGT GGGTCAAGGC GCAATCCAAT<br>
1081	ATGGTTTGGT CTAAGAAACA TGAATCCGGG GGTCATTTTG CTGCTCTCGA AAAGCCAGAA<br>
1141	TTGTTGTGGG CGGATATAGA GGAGTTTGTA CACTCGCAGT GGGAGAATTA CAAAGGTAAT<br>
1201	TACTGA<br>
79<br><br>
WE CLAIM:<br>
1.	A process for obtaining an optically active vicinal diol, which process includes<br>
the steps of:<br>
providing a mew-epoxide;<br>
creating a reaction mixture by mixing with the meso-epoxide a polypeptide, or a functional fragment thereof, having enantioselective epoxide hydrolase activity, the polypeptide being a polypeptide encoded by a gene of a yeast cell;<br>
incubating the reaction mixture; and<br>
recovering from the reaction mixture an enantiopure, or a substantially enantiopure, vicinal diol.<br>
2.	A process for obtaining an optically active vicinal diol, which process includes<br>
the steps of:<br>
providing a meso-epoxide;<br>
creating a reaction mixture by mixing with the meso-epoxide a cell comprising a nucleic acid encoding, and capable of expressing, a polypeptide having enantioselective epoxide<br>
hydrolase activity;<br>
incubating the reaction mixture; and<br>
recovering from the reaction mixture an enantiopure, or a substantially enantiopure, vicinal diol.<br>
3.	The process of claim 2, wherein the cell is a yeast cell.<br>
4.	The process of claims 2 or 3, wherein the polypeptide is encoded by an<br>
endogenous gene of the cell.<br>
5.	The process of claims 2 or 3, wherein the cell is a recombinant cell and the<br>
polypeptide is encoded by a nucleic acid sequence with which the cell is transformed.<br>
6.	The process of claim 5, wherein the nucleic acid sequence is a heterologous<br><br>
nucleic acid sequence.<br>
7.	The process of claim 5, wherein the nucleic acid sequence is a homologous<br>
nucleic acid sequence.<br>
8.	The process of any of claims 1 to 7, wherein the polypeptide is a full-length yeast<br>
epoxide hydrolase.<br>
9.	The process of any of claims 1 to 7, wherein the polypeptide is a functional<br>
fragment of yeast epoxide hydrolase.<br>
10.	The process of any one of claims 1 to 9, wherein the process is carried out at a pH<br>
from 5 to 10.<br>
11.	The process of any one of claims 1 to 10, wherein the process is carried out at a<br>
temperature of 0°C to 70 °C.<br>
12.	The process of any one of claims 1 to 11, wherein the concentration of the meso-<br>
epoxide in the reaction matrix is at least equal to the solubility of the meso-epoxide in water.<br>
13.	The process of any one of claims 1-12, wherein the meso-epoxide is a compound<br>
of the general formula (I), (II), (V), (VI) or (VII) and the vicinal diol produced by the process is<br>
a compound of the general formula (III), (IV), (VIII), (IX), or (X),<br>
The invention provides yeast strains, and polypeptides encoded by genes of such yeast strains, that have enantiospeciflc meso-epoxide hydrolase activity. The invention also features nucleic acid molecules encoding such polypeptides, vectors containing such nucleic acid molecules, and cells containing such vectors. Also embraced by the invention are methods for obtaining optically active vicinal diols from meso-epoxides.<br></td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/viewdoc.aspx?id=0Ljl8iwMCewqRglC3GCaZg==&amp;amp;loc=wDBSZCsAt7zoiVrqcFJsRw==" target="_blank" style="word-wrap:break-word;">http://ipindiaonline.gov.in/patentsearch/GrantedSearch/viewdoc.aspx?id=0Ljl8iwMCewqRglC3GCaZg==&amp;amp;loc=wDBSZCsAt7zoiVrqcFJsRw==</a></p>
		<br>
		<div class="pull-left">
			<a href="271717-purification-of-proteins-with-cationic-surfactant.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="271719-communications-network-for-distributed-sensing-and-therapy-in-biomedical-applications.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>271718</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>3317/KOLNP/2006</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>10/2016</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>04-Mar-2016</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>01-Mar-2016</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>13-Nov-2006</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>CSIR</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td>Meiring Naude Road, Brummeria, Pretoria, South Africa</td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>BOTES,Adriana,Leonora;</td>
											<td>34 Ormonde Street,Muckleneuk 0002 Pretoria,South Africa</td>
										</tr>
										<tr>
											<td>2</td>
											<td>LABUSCHAGNE,Michel</td>
											<td>C/O CSIR,Scientia,0002 PRETORIA,South Africa</td>
										</tr>
										<tr>
											<td>3</td>
											<td>LOTTER,Jeanette</td>
											<td>18 The Cottages,Thornhill Road, Thornhill Estate,Extension 2, 1644 Modderfontein,South Africa.</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>C12N9/14; C12P7/18</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>PCT/IB2005/001022</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td>2005-04-18</td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td>2004/2958</td>
									<td>2004-04-19</td>
								    <td>South Africa</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script data-cfasync="false" src="../cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/271718-process-for-obtaining-optically-active-vicinal-diols-from-meso-epoxides by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 00:47:11 GMT -->
</html>
