<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/272072-processes-using-dual-specificity-oligonucleotide-and-dual-specificity-oligonucleotide-and-dual-specificity-oligonucleotide by HTTrack Website Copier/3.x [XR&CO'2014], Thu, 04 Apr 2024 23:36:04 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 272072:&quot;PROCESSES USING DUAL SPECIFICITY OLIGONUCLEOTIDE AND DUAL SPECIFICITY OLIGONUCLEOTIDE AND DUAL SPECIFICITY OLIGONUCLEOTIDE&quot;</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">&quot;PROCESSES USING DUAL SPECIFICITY OLIGONUCLEOTIDE AND DUAL SPECIFICITY OLIGONUCLEOTIDE AND DUAL SPECIFICITY OLIGONUCLEOTIDE&quot;</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>The present invention relates to various processes by a template-dependent extension reaction using a dual specificity oligonucleotide and a dual specificity oligonucleotide composed of three different Tm portions therefor. Demonstrated in the present invention are the features of the dual specificity oligonucleotide, which are high hybridization specificity and mismatch tolerance.</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td>Form 2<br>
THE PATENTS ACT, 1970<br>
(39 of 1970)<br>
&amp;<br>
The Patent Rules, 2003<br>
COMPLETE SPECIFICATION (Section 10; Rule 13)<br>
1.      TITLE OF THE INVENTION:-<br>
"PROCESSES USING DUAL SPECIFICITY OLIGONUCLEOTIDE AND DUAL SPECIFICITY OLIGONUCLEOTIDE"<br>
2.	APPLICANTS:-<br>
Name	: -      SEEGENE, INC.<br>
Nationality  : -      South Korean<br>
Address       : -      8, 9f, Taewon Bldg., 65-5, Bangyi-dong,<br>
Songpa-gu,  Seoul 138-050, Republic of Korea.<br>
3.	PREAMBLE OF THE DESCRIPTION:-<br>
The following specification particularly describes the invention and the manner in which it is to be performed.<br><br>
PROCESSES USING DUAL SPECIFICITY OLIGONUCLEOTIDE AND DUAL SPECIFICITY OLIGONUCLEOTIDE<br>
      BACKGROUND OF THE INVENTION FIELD OF THE INVENTION<br>
The present invention relates to various processes using a dual specificity oligonucleotide and a dual specificity oligonucleotide therefor. More particularly, the present invention relates to various processes by a template-dependent extension reaction using a dual specificity       oligonucleotide and a dual specificity oligonucleotide therefor.<br>
DESCRIPTION OF THE RELATED ART<br>
Nucleic acid amplification is a pivotal process for a wide variety of methods in molecular biology, such that various amplification methods have been proposed. For example, Miller, H.<br>
         I. et al. (WO 89/06700) amplified a nucleic acid sequence based on the hybridization of a promoter/primer sequence to a target single-stranded DNA ("ssDNA") followed by transcription of many RNA copies of the sequence. Other known nucleic acid amplification procedures include transcription-based amplification systems (Kwoh, D. et al., Proc. Natl. Acad. Sci. U.S.A., 86:1173(1989); and GingerasT.R. etal., WO 88/10315).<br>
	The most predominant process for nucleic acid amplification known as polymerase chain<br>
reaction (hereinafter referred to as "PCR"), is based on repeated cycles of denaturation of double-stranded DNA, followed by oligonucleotide primer annealing to the DNA template, and primer extension by a DNA polymerase (Mullis et al. U.S. Pat. Nos. 4,683,195, 4,683,202, and 4,800,159; Saiki et al, (1985) Science 230, 1350-1354). The oligonucleotide primers used in<br>
         PCR are designed to anneal to opposite strands of the DNA template. The primers are extended by DNA polymerase, from which the product of one primer can serve as the template strand for the other primer in subsequential reactions. The PCR amplification process results in the exponential increase of discrete DNA fragments whose length is defined by the 5' ends of the oligonucleotide primers.<br>
 	The success in nucleic acid amplifications, in particular PCR amplification, relies on the<br>
specificity with which a primer anneals only to its target (and not non-target) sequences; therefore, it is important to optimize this molecular interaction. Whether a primer can anneal<br><br><br>
only to its perfect complement or also to sequences that have one or more mismatches depends critically upon the annealing temperature. In general, higher annealing temperature will lead to more specific annealing of the primer to its perfectly matched template, which in turn increases the likelihood of amplifying the target sequence only. On the other hand, more mismatches  between the template and primer can be tolerated at lower annealing temperatures. In consideration of such phenomenon, adjusting the annealing temperature can alter the specificity of pairing of the template and primer. For example, if there is no product, the temperature may be too high for annealing. If there are several products different in size where only one primer is present, this indicates that the single primer is annealing to more than one region of the<br>
         template. In this case, the annealing temperature should be increased.<br>
In addition to annealing temperature, several "primer search parameters", such as primer length, GC content, and PCR product length, should be considered for primer annealing specificity. A primer satisfying all such parameters will result in significant enhancement of primer annealing specificity during target DNA amplification, while resolving the problems of<br>
         backgrounds and non-specific products arising from primers used in the experiments. It is usual that well-designed primers can help avoid non-specific annealing and backgrounds as well as distinguish between cDNAs or genomic templates in RNA-PCR.<br>
Many approaches have been developed to improve primer annealing specificity and therefore accomplish the amplification of the desired product. Examples arc touchdown PCR<br>
         (Don et al., (1991) Touchdown PCR to circumvent spurious priming during gene amplification. Nucleic Acids Res., 19, 4008), hot start PCR (DAquila et al., (1991) Maximizing sensitivity and specificity of PCR by pre-amplificafion heating. Nucleic Acids Res., 19, 3749), nested PCR (Mullis and Faloona, (1987) Specific synthesis of DNA in vitro via a polymerase-catalyzed chain reaction. Methods Enzymol.  155, 335-350), and booster PCR (Ruano et al., (1989)<br>
          Biphasic amplification of very dilute DNA samples via booster PCR. Nucleic Acids Res. 17, 540). Other alternative approaches have been also reported that various 'enhancer' compounds can improve the specificity of PCR. The enhancer compounds include chemicals that increase the effective annealing temperature of the reaction, DNA binding proteins, and commercially available reagents. However, there is no 'magic' additive that will ensure the success in every<br>
          PCR, and it is very tedious to test different additives under different conditions such as annealing temperature. Although these approaches have contributed to the improvement of primer annealing specificity in some cases, they have not accessed fundamentally to a solution<br><br>
for the problems arising from primers used in the PCR amplification, such as non-specific products and high backgrounds.<br>
PCR-based techniques have been widely used not only for amplification of a target DNA sequence, but also for scientific applications or methods in the fields of biological and medical  research, such as reverse transcriptase PCR (RT-PCR), differential display PCR (DD-PCR), cloning of known or unknown genes by PCR, rapid amplification of cDNA ends (RACE), arbitrary priming PCR (AP-PCR), multiplex PCR, SNP genome typing, and PCR-based genomic analysis (McPherson and Moller, (2000) PCR. BIOS Scientific Publishers, Springer-Verlag New York Berlin Heidelberg, NY).<br>
	As   described   above,   all   these   methods   and   techniques   involving   nucleic   acid<br>
amplification, notably PCR amplification, could not be completely free from the limitations and problems resulting from the non-specificity of the primers used in each method, such as false positives, poor reproducibility, high backgrounds, although improved approaches to each method have been continuously introduced. Therefore, there remains a need of novel primer<br>
         and methods for improving annealing specificity, which can give rise to true amplification results.<br>
Meanwhile, DNA hybridization is a fundamental process in molecular biology and is affected by ionic strength, base composition, length of fragment to which the nucleic acid has been reduced, the degree of mismatching, and the presence of denaturing agents. DNA<br>
          hybridization-based technologies would be a very useful tool in specific nucleic acid sequence determination and clearly be valuable in clinical diagnosis, genetic research, and forensic laboratory analysis. For example, Wallace and coworkers showed that sequence differences as subtle as a single base change are sufficient to enable discrimination of short (e.g., 14-mer) oligomers and demonstrated how this could be applied in molecular analysis of point mutation<br>
          in the b-globin gene (Wallace, B.R., et al., (1981) The use of synthetic oligonucleotides as hybridization probes. Hybridization of oligonucleotides of mixed sequence to rabbit p-globin DNA. Nucleic Acids Res. 9, 879-894; and Conner, B.J., et al. (1983) Detection of sickle cell 0-globin allele by hybridization with synthetic oligonucleotides. Proc. Natl. Acad. Sci. USA 80, 278-282).<br>
	In   spite   of the   power   of  oligonucleotide   hybridization   to   correctly   identify   a<br>
complementary strand, researchers still face limitations. Hybrids containing oligonucleotides are much less stable than hybrids of long nucleic acids. This is reflected in lower melting<br><br><br>
temperature. The instability of the hybrids is one of the most important factors to be considered when designing oligonucleotide hybridization. The stability difference between a perfectly matched complement and a complement mismatched at only one base can be quite small, corresponding to as little as 0.5°C difference in their Tms (duplex melting temperature). The  shorter the oligomer of interest (permitting identification of a complementary strand in a more complex mixture), the stronger the effect of a single-base mismatch on overall duplex stability. However, the disadvantage of using such short oligonucleotides is that they hybridize weakly, even to a perfectly complementary sequence, and thus must be used under the conditions of reduced stringency, which results in decreasing hybridization specificity seriously.<br>
	There have been many efforts to improve the specificity of oligonucleotide hybridization.<br>
A method for chemically modifying bases of DNA for high-sensitivity hybridization (Azhikina et al., (1993) Proc. Natl. Acad. Set, USA, 90:11460-11462) and a method in which the washing after the hybridization is conducted at low temperatures for a long period to enhance the ability of discriminating the mismatch (Drmanac et al., (1990) DNA and Cell Biology, 9:527-534)<br>
         have been proposed. Recently, another method has been introduced for increasing the resolution power of single nucleotide polymorphisms (SNPs) in DNA hybridization by means of artificial mismatches (Guo et al., (1997) Nature Biotechnology, 15:331-5). In addition, many U.S. Patents including U.S. Pat. Nos. 6,077,668, 6,329,144, 6,140,054, 6,350,580, 6,309,824, 6,342,355 and 6,268,128 disclose the probe for hybridization and its applications. Although the<br>
          improved approaches to each method have been continuously introduced, all these methods and techniques involving oligonucleotide hybridization could not be completely free from the limitations and problems arising from non-specificity of oligonucleotide hybridization.<br>
There is still a possibility that artificial factors, such as the failures of spotting and immobilization of oligonucleotide on substrate and establishment of optimal hybridization<br>
         conditions, would affect the negative data of hybridization; the effect of erroneous results is more vulnerable to the results generated from high-throughput screening method. Such artificial factors inherent to spotting and hybridization are main practical drawbacks in oligonucleotide-based DNA microarrays.<br>
Furthermore, the development of DNA sequence determination techniques with enhanced<br>
          speed, sensitivity, and throughput are of utmost importance for the study of biological systems. Conventional DNA sequencing technique originally developed more than two decades ago (Sanger et al. (1977) Proc. Natl. Acad. Sci., 74:5463-5467) faces limitations in both throughput<br><br>
and cost for future applications. Therefore, several new techniques have been proposed. Three methods that hold great promise are sequencing by hybridization (Brain and Smith, (1988) J. Theor. Biol, 135:303-307); Drmanac et al., (1989) Genomics, 4:114-128); and Southern, E.M. (1989) Patent WO/10977), parallel signature sequencing based on ligation and cleavage  (Brenner et al., (2000) Proc. Natl. Acad. Sci., 97:1665-1670), and pyrosequencing (Ronaghi et al., (1996) Anal. Biochem., 242:84-89; and (1998) Science 281:363-365). For all aforementioned techniques, the success of sequencing reactions absolutely depends upon the hybridization specificity of the sequencing primer to a target nucleic acid. In consideration of the hybridization specificity of the sequencing primer, current methods are subject to limitation<br>
          of the length of template nucleic acids supplied for sequencing reactions. In general, sequencing reactions are conducted using template nucleic acids preferably less than a few hundred base pairs in length such that the specific hybridization of the sequencing primer is achieved to certain extent.<br>
For  advanced  studies,  however,  DNA sequencing  reactions  with  enhanced  speed,<br>
          sensitivity, and throughput should not be hindered by the size of template nucleic acids. In light of this, direct sequencing of a target nucleic acid from a population of template nucleic acid is allowed, provided that the sequencing primers are hybridized with the target nucleic acids at high specificity.<br>
	Throughout this application, various patents and publications are referenced, and citations<br>
are provided in parentheses. The disclosure of these patents and publications in their entities are hereby incorporated by references into this application in order to more fully describe this invention and the state of the art to which this invention pertains.<br>
           SUMMARY OF THE INVENTION<br>
To eliminate the problems and shortcomings of such conventional oligonucleotides, used<br>
as primers or probes, and various methods involving nucleic acid hybridization, the present<br>
inventor has developed a dual specificity oligonucleotide that allows a template-dependent<br>
     reaction to proceed with much higher specificity and found its excellent applications to a wide<br>
          variety of processes involving oligonucleotide hybridization or annealing.<br>
Accordingly, it is an object of this invention to provide a method for synthesizing a nucleic acid molecule using a dual specificity oligonucleotide by a template-dependent extension<br><br><br>
reaction.<br>
It is another object of this invention to provide a method for selectively amplifying a target nucleic acid sequence from a DNA or a mixture of nucleic acids.<br>
It is still another object of this invention to provide a method for amplifying two or more  target nucleotide sequences simultaneously using two or more pairs of primers in the same reaction.<br>
It is further object of this invention to provide a method for sequencing a nucleic acid molecule from a DNA or a mixture of nucleic acids.<br>
It is still further object of this invention to provide a method for detecting a nucleic acid       molecule with genetic diversity by a template-dependent extension reaction.<br>
It is another object of this invention to provide a method for detecting a target nucleotide sequence in a nucleic acid sample by use of dual specificity oligonulelotide-immobilized microarray.<br>
It is still another object of this invention to provide a dual specificity oligonucleotide for       synthesizing a nucleic acid molecule by a template-dependent extension reaction.<br>
It is further object of this invention to provide a method for enabling an annealing specificity of an oligonucleoide to be dually determined through a structure of the oligonucleoide.<br>
It is still further object of this invention to provide a method for improving an annealing       specificity of an oligonucleoide.<br>
Other objects and advantages of the present invention will become apparent from the detailed description to follow taken in conjugation with the appended claims and drawings.<br>
           BRIEF DESCRIPTION OF THE DRAWINGS<br>
FIGs.  1A and  IB schematically represent the principle of the dual specificity (DS)<br>
oligonucleotide of this invention in a template-dependent extension reaction. FIG. 1A shows<br>
high hybridization specificity of the DS oligonucleotide under high stringency conditions. FIG.<br>
	IB shows mismatch tolerance of the DS oligonucleotide.<br>
	FIG. 2 shows schematic representations for selectively amplifying a target nucleic acid of<br>
double-strand DNA using DS oligonucleotide primers of this invention.<br>
FIG. 3 shows schematic representations for selectively amplifying a target nucleic acid of<br><br><br>
mRNA using DS oligonucleotide primers of this invention.<br>
FIG. 4 shows schematic representations for template-dependent extension reactions in<br>
selective determination of a target nucleic acid using dual specificity oligonucleotides on<br>
oligonucleotide microarray.<br>
	FIG. 5 is an agarose gel photograph showing the results of PCR amplification for cytokine<br>
family genes, IL-1b and IL-19, using a set of conventional primers (IL-lb-5'-0 and IL-lb-3'-0,<br>
lane l; IL-19-5'-0 and EL-19-3'-0, lane 3) and the dual specificity oligonucleotides (DL-lb-5'<br>
and IL-lb-3', lane 2; IL-19-5' and IL-19-3', lane 4). M is a 100-bp size marker generated by<br>
Forever 100-bp Ladder Personalizer (Seegene, Inc. Seoul, Korea).<br>
	FIG. 6A is an agarose gel photograph showing the results of the 3'-RACE (rapid<br>
amplification of cDNA ends) of the DEG 10 gene, demonstrating PCR specificity of the dual<br>
specificity oligonucleotide. Lane 1 represents a primer with perfect match sequence, lanes 2-4<br>
represent primers with 3, 2 and  1  base mismatch at its 3'- low Tm specificity portion,<br>
respectively, and lanes 5-7 represent primers with 5, 3 and 2 base mismatch at its 5'-high Tm       specificity portion, respectively. M is a 100-bp size marker generated by Forever 100-bp<br>
Ladder Personalizer.<br>
FIG. 6B is an agarose gel photograph showing the results of the 3'-RACE of the DEG 10<br>
gene, demonstrating the mismatch tolerance of the dual specificity oligonucleotide in PCR.<br>
Lane 1 represents a primer with perfect match sequence, lanes 2-4 represent primers with 3, 2       and 1 base mismatch at its 3'- low Tm specificity portion, respectively, and lanes 5-7 represent<br>
primers with 5, 3 and 2 base mismatch at its 5'-high Tm specificity portion, respectively. M is a<br>
100-bp size marker generated by Forever 100-bp Ladder Personalizer.<br>
FIG. 7A represents the sequences of 5' primers for 3'-RACE of the mouse placenta-specific<br>
homeobox family genes Psxl and Psx2. Psxl-5'-40 and Psx2-5'-40 are conventional <br>
primers and Psxl-5'-41  and Psx2-5'-41  are primers designed according to the  present<br>
invention.<br>
FIG. 7B is an agarose gel photograph showing the results of the 3'-RACE of Psxl and<br>
Psx2. Lane 1, 3'-RACE of Psxl using the dual specificity oligonucleotide primer Psxl-5'-41;<br>
Lane 2, 3'-RACE of Psx2 using the dual specificity oligonucleotide primer Psx2-5'-41; Lane 3,       3'-RACE of Psxl using conventional primer Psxl-5'-40; and Lane 4, 3'-RACE of Psxl using<br>
conventional primer Psx2-5'-40. M is a 100-bp size marker generated by Forever 100-bp<br>
Ladder Personalizer.<br><br><br>
FIG. 8 shows the results of direct cycling sequencing for Psxl and Psxl genes from mouse placenta cDNA pool using the dual specificity oligonucleotide primers Psxl-5'-41 (for Psxl) and Psx2-5'-41 (for Psx2) as a sequencing primer.<br>
FIG. 9 shows the results of multiplex PCR using 9 sets of cytokine family gene-specific  dual specificity primers. Lane 1, multiplex PCR for 9 cytokine genes; Lane 2, monoplex PCR for EL-3 (200 bp); Lane 3, monoplex PCR for EL-15 (250 bp); Lane 4, monoplex PCR for EL-18 (300 bp); Lane 5, monoplex PCR for EL-25 (350 bp); Lane 6, monoplex PCR for IL-2 (400 bp); Lane 7, monoplex PCR for EL-6 (450 bp); Lane 8, monoplex PCR for EL-19 (500 bp); Lane 9, monoplex PCR for EL-ip (550 bp); and Lane 10, monoplex PCR for EL-10 (600 bp). M is a       100-bp size marker generated by Forever 100-bp Ladder Personalizer.<br>
FIG. 10 shows the results of PCR amplifications of the fusion glycoprotein (F) gene of<br>
human metapneumovirus (hMPV) by use of the dual specificity oligonucleotide primers. Lane<br>
1, target PCR using hMPV5'-585 and hMPV 3'-698 primer set; Lane 2, target PCR using<br>
hMPV5'-585 and hMPV 3'-1007 primer sets; Lane 3, target PCR using human {3-actin<br>
      primers; Lane 4, target PCR without template; and Lane 5, target PCR without template.<br>
DETAILED DESCRIPTION OF THIS INVETNION<br>
The present invention is generally directed to (a) a wide variety of processes using a dual specificity oligonucleotide and (b) a dual specificity oligonucleotide therefor. The dual  specificity oligonucleotide of this invention (hereinafter referred to as "DS oligo") allows primer or probe to be annealed to its target sequence with improved specificity, such that the specificity of nucleic acid amplification (in particular, PCR) and hybridization reaction can be significantly improved.<br>
         Dual Specificity Oligonucleotide (DS Oligo)<br>
In one aspect of this invention, there is provided a dual specificity oligonucleotide for synthesizing a nucleic acid molecule by a template-dependent extension reaction, which is represented by the following general formula:<br>
5 -Xp-Yq-Zr-3<br>
	wherein, Xp represents a 5'-high Tm specificity portion having a hybridizing nucleotide<br>
sequence substantially complementary to a site on a template nucleic acid to hybridize therewith, Yq represents a separation portion comprising at least two universal bases, Zr<br><br><br>
represents a 3'-low Tm specificity portion having a hybridizing nucleotide sequence substantially complementary to a site on the template nucleic acid to hybridize therewith, p, q and r represent the number of nucleotides, and X, Y, and Z are deoxyribonucleotide or ribonucleotide; Tm of the 5'-high Tm specificity portion is higher than that of the 3'-low Tm  specificity portion, the separation portion has the lowest Tm in the three portions; the separation portion forms a non base-pairing bubble structure under conditions that the 5'-high Tm specificity portion and the 3 '-low Tm specificity portion are annealed to the template nucleic acid, enabling the 5'-high Tm specificity portion to separate from the 3'-low Tm specificity portion in terms of annealing specificity to the template nucleic acid, whereby the annealing  specificity of the oligonucleotide is determined dually by the 5'-high Tm specificity portion and the 3'-low Tm specificity portion such that an overall annealing specificity of the oligonucleotide is enhanced.<br>
The term "dual specificity" with referring to the DS oligonucleotide (hereinafter referred<br>
to as "DS oligo") of this invention used herein is coined to describe its prominent feature that its annealing specificity to a target sequence is dually determined by its separate two portions, i.e., the 5'-high Tm specificity portion and the 3'-low Tra specificity portion. In general, the annealing specificity of primers or probes is governed by their overall consecutive sequence. In contrast, the annealing specificity of the DS oligo is dually determined by its two portions (the<br>
5'-high Tm specificity portion and the 3'-low Tm specificity portion) separated by the separation portion, in which these three portions are located in one oligonucleotide sequence. Such dual specificity permits the DS oligo to serve as a primer and probe exhibiting much higher specificity, rendering the present invention to be novel and unobvious over prior art.<br>
Meanwhile, the present inventor has already developed the ACP (annealing control<br>
primer) to improve annealing specificity as disclosed in WO 03/050303, the teachings of which are incorporated herein by reference. The DS oligo of this invention is distinctly different from the ACP in light of the following: (i) the DS oligo has two specificity portions to be hybridized with a target sequence whereas the ACP has one specificity portion; (ii) three portions in the DS oligo are distinctly discriminated in view of Tm whereas portions in the ACP are not; (iii) the<br>
DS primer is extended to synthesize a nucleic acid molecule complementary to the template only when annealing occurs by both the 5'-high Tm specificity portion and 3'- low Tra specificity portion, whereas the ACP is extended even when annealing occurs by the 3'-end<br><br><br>
portion; and (iv) thus the annealing or hybridizing specificity of the DS oligo is determined dually by the two separate portions, i.e., the 5'-high T,„specificity portion and the 3'-low Tm specificity portion, whereas that of the ACP is governed only by the 3'-end portion. Accordingly, it could be appreciated that the annealing or hybridizing specificity of the DS  oligo to its target sequence is much higher than that that of the ACP, addressing that the DS oligo is novel and unobvious over the ACP.<br>
The striking feature of the DS oligo is to have three different portions with distinct properties within one oligonucleotide molecule: 5'-high Tra specificity portion, 3'-low Tm specificity portion and separation portion.<br>
	The DS oligo is useful in a wide variety of processes and analyses involving a template-<br>
dependent extension reaction. The term used herein "a template-dependent extension reaction" means a reaction to extend an oligonucleotide molecule hybridized to a target sequence by incorporating successive nucleotides into its end moiety in which the extended sequence is determined by a complementary template sequence.<br>
	A schematic representation for the principles governing the hybridization (annealing)<br>
specificity of the DS oligo is illustrated in Fig. 1A. Referring to Fig. 1A, the DS oligo will be described in more detail.<br>
Where only the 5'-high Tm specificity portion of the DS oligo is annealed to a template, it cannot serve as a priming site for a template-dependent extension, resulting in no occurrence of<br>
           extension.<br>
While the 5'-high Tm specificity portion of the DS oligo is annealed to a non-target sequence, the 3'-low Tm specificity portion having a shorter sequence is unlikely to anneal to the non-target sequence. The reasons for that are that the 5'-high Tm specificity portion and the 3'-low Tm specificity portion are separated by the separation portion in terms of annealing<br>
         events. In other words, the 3'-low Tm specificity portion is involved in annealing events in a relatively independent manner from the 5'-high Tra specificity portion and the annealing of the 3'-low Tm specificity portion is less affected by the annealing of the 5'-high Tm specificity portion. In this connection, the likelihood of annealing of the 3'-low Tm specificity portion to a non-target sequence becomes much lower.<br>
	Where only the 3'-low Tm specificity portion has a sequence complementary to a non-<br>
target site, annealing either does not occur under certain high stringent conditions, e.g., stringent conditions for annealing of the 5'-high Tm specificity portion. According to a preferred<br><br><br>
embodiment, it is advantageous to perform template-dependent extension reactions using the DS oligo under stringent conditions with annealing temperature much higher than Tm of the 3'-low Tm specificity portion.<br>
Where both 5'-high Tm specificity portion and 3'-low Tm specificity portion have a       sequence substantially complementary to a template, the DS oligo can be annealed to the template and hence the successful extension occurs.<br>
Without wising to be bound by theory, it is believed that the separation portion makes the 3'-low Tm specificity portion more sensitive to annealing conditions (e.g., temperature and sequence complementarity). In this regard, the incidence of the non-specific hybridization<br>
between the 3'-low Tm specificity portion and non-target sequences becomes much lower under certain annealing (or stringent) conditions. Where the 3'-low Tm specificity portion as well as the 5'-high Tm specificity portion is annealed to its target sequence, the 3'-end of the 3'-low Tm specificity portion is more likely to generate a site extendible by DNA polymerases.<br>
The term "oligonucleotide" as used herein refers to a linear oligomer of natural or<br>
modified monomers or linkages, including deoxyribonucleotides, ribonucleotides and the like, capable of specifically hybridizing with a target nucleotide sequence, whether occurring naturally or produced synthetically. The oligonucleotide is preferably single stranded for maximum efficiency in hybridization. Preferably, the oligonucleotide is an oligodeoxyribonucleotide. The oligonucleotide of this invention can be comprised of naturally<br>
          occurring dNMP (i.e., dAMP, dGM, dCMP and dTMP), nucleotide analogs, or nucleotide derivatives. The oligonucleotide can also include ribonucleotides. For example, the oligonucleotide of this invention may include nucleotides with backbone modifications such as peptide nucleic acid (PNA) (M. Egholm et al., Nature, 365:566-568(1993)), phosphorothioate DNA, phosphorodithioate DNA, phosphoramidate DNA, amide-linked DNA, MMI-linked<br>
         DNA, 2'-0-methyl RNA, alpha-DNA and methylphosphonate DNA, nucleotides with sugar modifications such as 2'-0-methyl RNA, 2'-fluoro RNA, 2'-amino RNA, 2'-0-alkyl DNA, 2'-0-allyl DNA, 2'-0-alkynyl DNA, hexose DNA, pyranosyl RNA, and anhydrohexitol DNA, and nucleotides having base modifications such as C-5 substituted pyrimidines (substituents including fluoro-, bromo-, chloro-, iodo-, methyl-, ethyl-, vinyl-, formyl-, ethynyl-, propynyl-,<br>
alkynyl-, thiazolyl-, imidazolyl-, pyridyl-), 7-deazapurines with C-7 substituents (substituents including fluoro-, bromo-, chloro-, iodo-, methyl-, ethyl-, vinyl-, formyl-, alkynyl-, alkenyl-, thiazolyl-, imidazolyl-, pyridyl-), inosine, and diaminopurine.<br><br><br>
The term "primer" as used herein refers to an oligonucleotide, which is capable of acting as a point of initiation of synthesis when placed under conditions in which synthesis of primer extension product which is complementary to a nucleic acid strand (template) is induced, i.e., in the presence of nucleotides and an agent for polymerization, such as DNA polymerase, and at a  suitable temperature and pH. The primer is preferably single stranded for maximum efficiency in amplification. Preferably, the primer is an oligodeoxyribonucleotide. The primer of this invention can be comprised of naturally occurring dNMP {i.e., dAMP, dGM, dCMP and dTMP), modified nucleotide, or non-natural nucleotide. The primer can also include ribonucleotides. The primer must be sufficiently long to prime the synthesis of extension<br>
products in the presence of the agent for polymerization. The exact length of the primers will depend on many factors, including temperature, application, and source of primer. The term "annealing" or "priming" as used herein refers to the apposition of an oligodeoxynucleotide or nucleic acid to a template nucleic acid, whereby the apposition enables the polymerase to polymerize nucleotides into a nucleic acid molecule which is complementary to the template<br>
nucleic acid or a portion thereof. The term used "hybridizing" used herein refers to the formation of a double-stranded nucleic acid from complementary single stranded nucleic acids. There is no intended distinction between the terms "annealing" and "hybridizing", and these terms will be used interchangeably.<br>
The term used herein "probe" refers to a single-stranded nucleic acid molecule comprising<br>
a portion or portions that are substantially complementary to a target nucleotide sequence.<br>
The term "portion" used herein in conjunction with the DS oligo of this invention refers to a nucleotide sequence separated by the separation portion. The term "5'-high Tm specificity portion" or "3'-low Tm specificity portion" refers to a nucleotide sequence at the 5'-end or 3'-end of the DS oligo of this invention, respectively, which is separated by the separation portion.<br>
The term "5'-high Tra specificity portion" in conjunction with the DS oligo is intended to refer to a portion with the highest Tm among three portions and having a hybridizing nucleotide sequence substantially complementary to a site on the template nucleic acid. The term "3'-low Tm specificity portion" in reference to the DS oligo means a portion with a lower Tm than the 5'-high Tm specificity portion but higher Tm than the separation portion and having a hybridizing<br>
nucleotide sequence substantially complementary to a site on the template nucleic acid.<br>
The term "Tm" used herein refers to the melting temperature at which half the molecules of a nucleic acid duplex are single stranded. The terms "high Tm" and "low Tm" in conjunction<br><br><br>
with portions in the DS oligo are intended to describe a relative Tm value yet not an absolute Tm value. That is, it is only required that the Tm of the 5'-high Tm specificity portion is high relative to that of the 3'-low Tm specificity portion.<br>
The 5'-high Tm specificity portion and 3'-low Tm specificity portion are designed to have a hybridizing nucleotide sequence substantially complementary to a site on a template nucleic acid to hybridize therewith. The term "substantially complementary" in reference to the DS oligo is used herein to mean that the oligonucleotide molecule is sufficiently complementary to hybridize selectively to a template nucleic acid sequence under the designated annealing conditions or stringent conditions, such that the annealed oligonucleotide can be extended by a<br>
polymerase to form a complementary copy of the template. Therefore, this term has a different meaning from "perfectly complementary" or related terms thereof. It will be appreciated that the 5'-high Tm specificity portion and 3'-low Tm specificity portion of the DS oligo can have one or more mismatches to a template to an extent that the DS oligo can serve as primer or probe. Most preferably, the 5'-high Tm specificity portion and/or 3'-low Tm specificity portion<br>
of the DS oligo have a nucleotide sequence perfectly complementary to a site on a template, i.e., no mismatches.<br>
For successful performance of the DS oligo, it is essential that the Tm of the 5'-high Tm specificity portion is higher than that of the 3'-low Tm specificity portion. It is preferred that the Tm of the 5'-high Tm specificity portion ranges from 40°C to 80°C, more preferably, 40°C to<br>
75°C, still more preferably, 50°C to 68°C, and most preferably, 50°C to 65°C. It is preferred that Tm of the 3'-low Tm specificity portion ranges from 10°C to 40°C, more preferably, 15°C to 40°C, and most preferably, 20°C to 35°C. Preferably, the Tm of the 5'-high Tm specificity portion is higher at least 5°C, more preferably at least 10°C, still more preferably at least 15°C, and  most  preferably   at   least   20°C  than   that   of the   3'-low   Tm specificity   portion.<br>
Advantageously, the Tm of the 5'-high Tm specificity portion is higher 5-70°C, preferably, 10-70°C, more preferably, 10-60°C, still more preferably, 10-50°C, still yet more preferably, 10-40°C and most preferably, 20-40°C than that of the 3'-low Tm specificity portion.<br>
According to a preferred embodiment, the 5'-high Tm specificity portion is longer than the 3'-low Tm specificity portion. The length of 5'-high Tm specificity portion is preferably 15 to 40<br>
nucleotide residues, more preferably, 15 to 30 nucleotide residues, and most preferably, 20 to 25 nucleotide residues. The length of 3'-low Tn, specificity portion is preferably 3 to 15 nucleotide residues, more preferably, 5 to 15 nucleotide residues, and most preferably, 6 to 12<br><br>
nucleotide residues.<br>
The separation portion comprising at least two universal bases is partly responsible for advantages and features of the DS oligo. The term "universal base" used herein refers to one capable  of forming base pairs with  each  of the natural DNA/RNA bases  with  little       discrimination between them.<br>
It has been widely known that nucleotides at some ambiguous positions of degenerate primers have been replaced by universal base such as deoxyinosine (Ohtsuka, E. et al., (1985) J. Biol. Chem. 260, 2605-2608; and Sakanari, J.A. et al, (1989) Proc. Natl. Acad. Sci. 86, 4863-4867),   l-(2'-deoxy-beta-D-ribofuranosyl)-3-nitropyrrole  (Nichols,  R.   et  al.,  (1994)<br>
Nature 369, 492-493) and 5-nitroindole (Loakes, D. et al, (1994) Nucleic Acids Res. 22, 4039-4043) for solving the design problems associated with the degenerate primers, because such universal bases are capable of non-specific base pairing with all four conventional bases. However, there has not been any report that these universal bases allow forming a portion in an oligonucleotide molecule to generate a bubble structure during annealing (hybridization) or<br>
amplification and then separate two opposite adjacent sequences, resulting in the elevation of the annealing specificity of primer or probe to a target sequence by dual specificity through two separate specificity (annealing) portions.<br>
According to a preferred embodiment, the universal base in the separation portion is selected from the group consisting of deoxyinosine, inosine, 7-deaza-2'-deoxyinosine, 2-aza-<br>
	2'-deoxyinosine, 2'-OMe inosine, 2'-F inosine, deoxy 3-nitropyrrole, 3-nitropyrrole, 2'-OMe 3-<br>
nitropyrrole, 2'-F 3-nitropyrrole, l-(2'-deoxy-beta-D-ribofuranosyl)-3-nitropyrrole, deoxy 5-<br>
nitroindole, 5-nitroindole, 2'-OMe 5-nitroindole, 2'-F 5-nitroindole, deoxy 4-<br>
nitrobenzimidazole,	4-nitrobenzimidazole,	deoxy	4-aminobenzimidazole,	4-<br>
aminobenzimidazole, deoxy nebularine, 2'-F nebularine, 2'-F 4-nitrobenzimidazole, PNA-5-<br>
	introindole, PNA-nebularine, PNA-inosine, PNA-4-nitrobenzimidazole, PNA-3-nitropyrrole,<br>
morpholino-5-nitroindole, morpholino-nebularine, morpholino-inosine, morpholino-4-<br>
nitrobenzimidazole,	morpholino-3-nitropyrrole,	phosphoramidate-5-nitroindole,<br>
phosphoramidate-nebularine,	phosphoramidate-inosine,	phosphoramidate-4-<br>
nitrobenzimidazole,     phosphoramidate-3-nitropyrrole,     2'-0-methoxyethyl     inosine,     2'0-<br>
methoxyethyl nebularine, 2'-0-methoxyethyl 5-nitroindole, 2'-0-methoxyethyl 4-nitrobenzimidazole, 2'-0-methoxyethyl 3-nitropyrrole, and combinations thereof. More preferably, the universal base or non-discriminatory base analog is deoxyinosine, l-(2'-deoxy-beta-D-<br><br><br>
ribofuranosyl)-3-nitropyrrole or 5-nitroindole, most preferably, deoxyinosine.<br>
Such universal bases may be contained in the separation portion in a contiguous manner or<br>
interrupted manner with other nucleotides such as dNMPs. It is preferable that the separation<br>
portion comprises contiguous nucleotides having universal bases, preferably, deoxyinosine.<br>
	It is critical that the separation portion in the DS oligo has the lowest Tm in the three<br>
portions, in order that the separation portion forms a non base-pairing bubble structure under conditions that the 5'-high Tra specificity portion and the 3'-low Tm specificity portion are annealed to the template nucleic acid, enabling the 5'-high Tm specificity portion to separate from the 3 '-low Tm specificity portion in terms of annealing specificity to the template nucleic<br>
acid, whereby the annealing specificity of the oligonucleotide is determined dually by the 5'-high Tm specificity portion and the 3'-low Tm specificity portion such that the overall annealing specificity of the oligonucleotide is considerably enhanced. Preferably, the Tm of the separation portion ranges from 3°C to 15°C, more preferably, 4°C to 15°C, and most preferably 5°C to 10°C.<br>
	According to a preferred embodiment, the separation portion between the 5'-high Tm<br>
specificity portion and the 3'-low Tm specificity portion contains at least 3 universal bases, more preferably at least 4 universal bases, and most preferably at least 5 universal bases. According to a preferred embodiment, the separation portion contains 2-10 universal bases, more preferably 3-10 universal bases, still more preferably, 4-8 universal bases, and most<br>
          preferably, 5-7 universal bases.<br>
Where a primer or probe having a longer sequence is required, the advantages of the DS oligo are most highlighted. For example, according to a conventional technique, a primer having a nucleotide sequence longer than 35 bp as a hybridizing sequence is very liable to generate non-specific amplicons. By contrast, the DS oligo can generate specific amplicons<br>
even with long sequences, since it carries two hybridizing sequences (i.e., the 5'-high Tm specificity portion and the 3'-low Tm specificity portion) separated from each other in terms of molecular interaction with templates (i.e., annealing). For example, the DS oligo may contain 35-45 bp of a hybridizing sequence complementary to a target sequence. In this regard, it could be appreciated that the present invention permits primers to be designed with much longer<br>
sequences considered to be non-practicable in conventional primer design strategies.<br>
According to a preferred embodiment, the 5'-high Tm specificity portion is 15 to 25 nucleotides in length, the separation portion is 3 to 15 nucleotides in length, and the 3'-low Tm<br><br><br>
specificity portion is 3 to 15 nucleotides in length.<br>
More preferably, the 5'-high Tm specificity portion is 15 to 25 nucleotides in length; the separation portion is 3 to 10 nucleotides in length; and the 3'-low Tm specificity portion is 5 to 15 nucleotides in length. Most preferably, the 5'-high Tm specificity portion is 15 to 25  nucleotides in length; the separation portion is 5 to 7 nucleotides in length; and the 3'-low Tm specificity portion is 6 to 10 nucleotides in length. According to the exemplary and illustrative DS oligo described in Examples, the 5'-high Tm specificity portion is about 20 nucleotides in length; the separation portion is about 5 nucleotides in length; and the 3'-low Tm specificity portion is about 8-10 nucleotides in length.<br>
	In the most preferred embodiment, the DS oligo is represented by the following general<br>
formula: 5'-Xp-(dI)q-Zr-3' (definition and characteristics of Xp and Zr are the same as described previously, dl represents deoxyinosine, (dl)q represents a separation portion comprising contiguous nucleotides having universal bases and q is an integer between 5-7).<br>
Interestingly, the present DS oligo also has mismatch tolerance under the stringent<br>
conditions sufficient to tolerate mismatching to its target sequence.<br>
A schematic representation for the principles governing mismatch tolerance of the DS oligo is illustrated in Fig. IB. One or more, preferably one to three base mismatches in the 5'-high Tm specificity portion can be tolerated under the condition that both 5'-high Tm specificity portion and 3'-low Tm specificity portion are annealed to the template. One or more, preferably<br>
one to two base mismatches in the 3'-low Tm specificity portion can be tolerated under the condition that both 5'-high Tm specificity portion and 3'-low Tm specificity portion are annealed to the template. Further, one or more, preferably one to five base mismatches in both the 5'-high Tm specificity portion and 3'-low Tm specificity portion can be tolerated under the condition that both 5'-high T„, specificity portion and 3'-low Tm specificity portion are annealed<br>
to the template.<br>
For imposing mismatch tolerance on the DS oligo, the annealing condition, notably, the annealing temperature is important. The annealing is performed under conditions that annealing solely by the 3 '-low Tm specificity portion does not occur, yet annealing by all the portions occurs when the 5'-high Tm specificity portion and/or the 3'-low Tm specificity portion has one<br>
or more, yet limited, mismatched bases to its target site. The DS oligos having mismatch tolerance is required to amplify or detect a nucleotide sequence with genetic diversity. The DS oligos with mismatch tolerance can be annealed to target sequences showing genetic diversity<br><br><br>
and result in successful amplification and detection of nucleotide sequences of interest. In other words, the DS oligo originally developed to dramatically enhance specificity of annealing and hybridization can be also used in processes to require mismatch tolerance where annealing or stringent conditions are suitably adjusted. <br>
In another aspect of this invention, there is provided a method for enabling an annealing specificity of an oligonucleoide to be dually determined through a structure of the oligonucleoide, which comprises the steps of: (a) selecting a target nucleic acid sequence; (b) designing a sequence of an oligonucleotide having (i) a hybridizing sequence substantially<br>
complementary to the target nucleic acid and (ii) a separation portion comprising at least two universal bases, such that the separation portion intervenes in the hybridizing sequence to form three portions in the oligonucleotide; and (c) determining the position of the separation portion in the oligonucleotide to allow a portion at the 5 '-direction of the separation portion to have a higher Tm than a portion at the 3'-direction of the separation portion and to allow the separation<br>
portion to have the lowest Tra in the three portions, thereby providing an oligonucleotide having three distinct portions with different Tm values from one another in which (i) a 5'-high Tra specificity portion of the oligonucleotide has a hybridizing nucleotide sequence substantially complementary to the target nucleic acid, (ii) a 3'-low Tm specificity portion of the oligonucleotide has a hybridizing nucleotide sequence substantially complementary to the<br>
target nucleic acid; and (iii) the separation portion of the oligonucleotide between the 5'-high Tm specificity portion and 3'-low Tm specificity portion comprises at least two universal bases; and the Tm of the 5'-high Tm specificity portion is higher than that of the 3'-low Tm specificity portion and the separation portion has the lowest Tm in the three portions, whereby the annealing specificity of the oligonucleotide to the target nucleic acid is determined dually by<br>
both the 5'-high Tm specificity portion and the 3'-low Tm specificity portion.<br>
The present method is directed to provide a novel approach to dramatically increase the annealing specificity of an oligonucleotide to be hybridized with its target sequence. The present method is also expressed as a method for improving an annealing specificity of an oligonucleoide. Moreover, the present invention is expressed as a method using a separation<br>
portion comprising at least two universal bases for improving the annealing specificity of an oligonucleotide hybridized to a target sequence.<br>
The present method is performed to  prepare the DS  oligo  discussed hereinabove.<br><br><br>
Therefore, in the interest of avoiding unnecessary redundancy, the common descriptions between them are not being repeated but they are incorporated into this description of the method as if they were repeated.<br>
Most of the conventional methods for designing primers or probes merely use a sequence      as hybridizable to their target sequences as available. Moreover, to increase the annealing specificity of oligonucleotides, it has been conventionally attempted to adjust amplification or hybridization conditions such as temperature and ion concentration.<br>
By contrast, the present method provides a novel strategy for increasing annealing specificity by introducing novel characteristics into oligonucleotide sequences per se. The term<br>
"through a structure of the oligonucleotide" used herein with reference to enabling the annealing specificity of oligonucleoides to be dually determined means that the structure of oligonucleotides contributes heavily to the increase in the annealing specificity of oligonucleoides by imposing on oligonucleotides a novel feature to be dually determined in terms of annealing specificity.<br>
	It is critical in the present method to design a sequence of an oligonucleotide having (i) a<br>
hybridizing sequence substantially complementary to the target nucleic acid and (ii) a separation portion comprising at least two universal bases. In this step, the structural outline of the oligonucleotide is presented to show a 5'-end portion/separation portion/3'-end portion in the oligonucleotide. Both the 5'-end and 3'-end portions carry a hybridizing sequence<br>
substantially complementary to the target nucleic acid and are intervened by the separation portion.<br>
The most critical step in the present invention is to determine the position of the separation portion in the oligonucleotide to allow a portion at the 5'-direction of the separation portion to have a higher Tm than a portion at the 3'-direction of the separation portion and to allow the<br>
separation portion to have the lowest Tm in the three portions, thereby providing an oligonucleotide having three distinct portions with different Tm values from one another.<br>
The novel structural characteristics introduced into oligonucleotides by the present method are: (i) three distinct portions (5'-high Tra specificity portion, separation portion and 3'-low Tm specificity portion) in oligonucleotide sequences; (ii) different Tra values of the three portions<br>
from one another; (iii) separation portion between the 5'-high Tm specificity portion and 3'-Low Tm specificity portion comprising at least two universal bases; (iv) two portions involved in molecular interaction with targets in annealing step, which is separated in terms of annealing<br><br><br>
event by the separation portion; (v) Tm values following the order of the 5'-high Tm specificity portion, 3'-low Tm specificity portion and separation portion. Such structural features ensure the annealing specificity of oligonucleotides finally provided by the present invention to be determined dually by both the 5'-high Tm specificity portion and the 3'-low Tm specificity  portion, permitting the dramatic increase in the annealing specificity of oligonucleotides to their target sequence.<br>
The oligonucleotides designed and prepared according to the present method exhibit much higher annealing specificity than those not having such three portions.<br>
	The features and advantages of the DS oligo will be described as follows:<br>
(a)	the separation portion of the DS oligo comprises at least two universal bases which<br>
generates the lowest Tm region in the DS oligo, so that it forms a non base-pairing bubble<br>
structure under conditions that the 5'-high Tm specificity portion and the 3'-low Tm specificity<br>
portion are annealed to the template nucleic acid. Such non base-pairing bubble structure<br>
enables the 5'-high Tm specificity portion to separate from the 3'-low Tm specificity portion in terms of annealing specificity to the template nucleic acid;<br>
(b)	the Tm of the 5'-high Tm specificity portion is higher than that of the 3'-low Tm<br>
specificity portion, and the separation portion shows the lowest Tm, which makes it possible to<br>
establish stringent conditions under that annealing solely by the 3'-low Tm specificity portion<br>
          does not occur;<br>
(c)	thus, the overall annealing specificity of the DS oligo is determined dually by both 5'-<br>
high Tm specificity portion and 3 '-low Tm specificity portion; and<br>
(d)	consequently, the overall annealing specificity of the DS oligo is dramatically<br>
improved.<br>
	It may be appreciated that the DS oligo of the present invention is very useful in a variety<br>
of (i) primer-based nucleic acid amplification methods such as the methods of Miller, H. I. (WO 89/06700) and Davey, C. et al. (EP 329,822), Ligase Chain Reaction (LCR, Wu, D.Y. et al., Genomics 4:560 (1989)), Polymerease Ligase Chain Reaction (Barany, PCR Methods and Applic, 1:5-16(1991)), Gap-LCR (WO 90/01069), Repair Chain Reaction (EP 439,182), 3SR<br>
(Kwoh et al., PNAS, USA, 86:1173(1989)) and NASBA (U.S. Pat. No. 5,130,238), (ii) primer extension-based technologies such as cycle sequencing (Kretz et al., (1994) Cycle sequencing. PCR Methods Appl. 3:S107-S112) and pyrosequencing (Ronaghi et al, (1996) Anal. Biochem.,<br><br><br>
242:84-89; and (1998) Science 281:363-365), and (iii) hybridization-based technologies such as detection of a target nucleotide sequence using oligonucleotide microarray.<br>
        The DS oligo of the subject invention can be applied to a variety of nucleic acid amplification, sequencing and hybyridization-based technologies. Representative examples to      prove the effect of the DS oligo are as follows:<br>
I.   Application to Synthesizing a Nucleic Acid Molecule<br>
In another aspect of this invention, there is provided a method for synthesizing a nucleic acid molecule using a dual specificity oligonucleotide by a template-dependent extension      reaction, which comprises the steps of:<br>
(a)	annealing the dual specificity oligonucleotide to a template nucleic acid molecule,<br>
wherein the dual specificity oligonucleotide has three portions in which a 5'-high Tm specificity  portion has a hybridizing nucleotide sequence substantially complementary to a site on the  template nucleic acid to hybridize therewith, a separation portion comprises at least two universal bases and a 3'-low Tm specificity portion has a hybridizing nucleotide sequence substantially complementary to a site on the template nucleic acid to hybridize therewith, Tm of the 5'-high Tm specificity portion is higher than that of the 3'-low Tm specificity portion, the separation portion has the lowest Tm in the three portions; the separation portion forms a non base-pairing bubble structure under conditions that the 5'-high Tm specificity portion and the<br>
3'-low Tm specificity portion are annealed to the template nucleic acid, wherein the annealing is performed under conditions that annealing solely by the 3'-low Tra specificity portion does not occur; and<br>
(b)	extending the dual specificity oligonucleotide to synthesize a nucleic acid molecule   complementary to the template nucleic acid.<br>
		Since the synthesis method of this invention employs the DS oligo of this invention, the  common descriptions between them are omitted in order to avoid the complexity of this specification leading to undue multiplicity.<br>
This application using the DS oligo of the subject invention can provide an improved <br>
method for selectively synthesizing a nucleic acid sequence complementary to a target sesuence<br>
by a template-dependent extension reaction involving annealing and extending steps. In particular, the synthesis of a nucleic acid sequence complementary to a target sesuence can be achieved by repeating the process of the template-dependent extension reaction in which the<br><br>
annealing and extension steps are followed by denaturation step.<br>
The method of the present invention can be used to synthesize a nucleic acid molecule complementary to any template nucleic acid molecule. Such molecule may be either DNA or RNA. The molecule may be in either a double-stranded or single-stranded form. Where the nucleic acid as starting material is double-stranded, it is preferred to render the two strands into a single-stranded or partially single-stranded form. Methods known to separate strands includes, but not limited to, heating, alkali, formamide, urea and glycoxal treatment, enzymatic methods (e.g., helicase action), and binding proteins. For instance, strand separation can be achieved by heating at temperature ranging from 80°C to  105°C. General methods for<br>
accomplishing this treatment are provided by Joseph Sambrook, et al., Molecular Cloning, A Laboratory Manual, Cold Spring Harbor Laboratory Press, Cold Spring Harbor, N.Y.(2001).<br>
Where a mRNA is employed as starting material, a reverse transcription step is necessary prior to performing annealing step, details of which are found in Joseph Sambrook, et al., Molecular Cloning, A Laboratory Manual, Cold Spring Harbor Laboratory Press, Cold Spring<br>
Harbor, N.Y.(2001); and Noonan, K. F. et al., Nucleic Acids Res. 16:10366 (1988)). For reverse transcription, an oligonucleotide dT primer hybridizable to poly A tail of mRNA is used. The oligonucleotide dT primer is comprised of dTMPs, one or more of which may be replaced with other dNMPs so long as the dT primer can serve as primer. Reverse transcription can be done with reverse transcriptase that has RNase H activity. If one uses an enzyme having RNase H<br>
activity, it may be possible to omit a separate RNase H digestion step by carefully choosing the reaction conditions.<br>
The present methods do not require that the template nucleic acid molecules have any particular sequence or length. In particular, the molecules include any naturally occurring prokaryotic, eukaryotic (for example, protozoans and parasites, fungi, yeast, higher plants,<br>
           lower and higher animals, including mammals and humans), viral (for example, Herpes viruses,<br>
HTV, influenza virus, Epstein-Barr virus, hepatitis virus, polio virus, etc.), or viroid nucleic<br>
acid. The nucleic acid molecule can also be any nucleic acid molecule which has been or can be<br>
chemically synthesized. Thus, the nucleic acid sequence may or may not be found in nature.<br>
The DS oligo used for the present invention is hybridized or annealed to a site on the<br>
template such that double-stranded structure is formed. Conditions of nucleic acid annealing suitable for forming such double stranded structures arc described by Joseph Sambrook, et al., Molecular Cloning, A Laboratory Manual, Cold Spring Harbor Laboratory Press, Cold Spring<br><br><br>
Harbor, N.Y.(2001) and Haymes, B. D., et al., Nucleic Acid Hybridization, A Practical Approach, IRL Press, Washington, D.C. (1985). The sequences of the 5'-high Tm specificity portion and the 3'-low Tm specificity portion of the DS oligo need not to exhibit precise complementarity, yet need only be substantially complementary in sequence to be able to form a stable double-stranded structure. Thus, departures from complete complementarity are permissible, so long as such departures are not sufficient to completely preclude hybridization to form a double-stranded structure. Annealing of the DS oligo to a site on the template nucleic acid is a prerequisite for its template-dependent polymerization with polymerases. Factors (see Joseph Sambrook, et al., Molecular Cloning, A Laboratory Manual, Cold Spring Harbor<br>
Laboratory Press, Cold Spring Harbor, N.Y.(2001); and Haymes, B.D., et. al., Nucleic Acid Hybridization, A Practical Approach, IRL Press, Washington, D.C.(1985)) which affect the base pairing of the DS oligo to its complementary nucleic acids subsequently affect priming efficiency. The nucleotide composition of the DS oligo can affect the temperature at which annealing is optimal and therefore can affect its priming efficiency.<br>
	The 5 '-high Tm specificity portion and 3 '-low Tm specificity portion of DS oligo during the<br>
annealing step play a role as a hybridizing portion or a specificity determining site (i.e., dual specificity determining site), while the separation portion does not serve as a hybridizing site and does not interact with the template for base pairing.<br>
A variety of DNA polymerases can be used in the extension step of the present methods,<br>
which includes "Klenow" fragment of E. coli DNA polymerase I, a thermostable DNA polymerase, and bacteriophage T7 DNA polymerase. Preferably, the polymerase is a thermostable DNA polymerase which may be obtained from a variety of bacterial species, including Thermus aquaticus (Taq), Thermits thermophilic (Tth), Tliermtts filiformis, Thermis flavus, Thermococcus literalis, and Pyrococcus furiosus (Pfu). When a polymerization reaction<br>
is being conducted, it is preferable to provide the components required for such reaction in excess in the reaction vessel. Excess in reference to components of the extension reaction refers to an amount of each component such that the ability to achieve the desired extension is not substantially limited by the concentration of that component. It is desirable to provide to the reaction mixture an amount of required cofactors such as Mg2+, dATP, dCTP, dGTP, and dTTP<br>
in sufficient quantity to support the degree of the extension desired.<br>
Annealing or hybridization in the present method is performed under stringent conditions that allow for specific binding between the DS oligo and the template nucleic acid. Such<br><br>
stringent conditions for annealing will be sequence-dependent and varied depending on environmental parameters. In the present method, the annealing step is generally performed under high stringent conditions. However, if the present method is applied to processes requiring mismatch tolerance, it is preferable that the annealing step is carried out under  stringent conditions that the 5'-high Tm specificity portion and the 3'-low Tm specificity portion are annealed to the template despite the presence of one or more, yet limited, base pair mismatches. Such mismatch tolerance is very useful in the amplification or detection of a gene with genetic diversity. Stringent conditions may be readily determined from the standard known in the art.<br>
	It is advantageous to perform the annealing step at an annealing temperature higher than<br>
the Tm of the 3'-low Tm specificity portion, ensuring that annealing by only the 3'-low Tm specificity portion does not occur. Preferably, the annealing temperature is higher at least 5°C, more preferably at least 10°C, still more preferably at least 15°C, and most preferably at least 20°C than the Tm of the 3'-low Tm specificity portion.<br>
	In a preferred embodiment, the annealing temperature ranges from about 40°C to 75°C,<br>
more preferably, 45°C to 72°C, still more preferably, 50°C to 68°C, and most preferably, 55°C to 65°C. Annealing temperatures suitable in the present method may be determined by considering independently the Tm values of the 5'-high Tm specificity portion and the 3'-low Tm specificity portion. In other words, annealing temperatures in the present method are not be<br>
determined by the total length and nucleotide compositions of both the 5'-high Tm specificity portion and the 3'-low Tm specificity portion but by the individual length and nucleotide composition of the 5'-high Tm specificity portion and/or the 3'-low Tm specificity portion. Usually, the annealing temperature determined by considering only the Tm of the 5'-high Tm specificity portion may be much higher than the Tm of the 3 '-low Tm specificity portion and<br>
become an optimal one.<br>
If mismatch tolerance is required in the annealing step, it is preferred that the annealing temperature is adjusted to become lower than those indicated above.<br>
The present method may be combined with many other processes known in the art to achieve a specific aim. For example, the isolation (or purification) of synthesized product may<br>
follow the extension reaction. This can be accomplished by gel electrophoresis, column chromatography, affinity chromatography, or hybridization. In addition, the synthesized product of this invention may be inserted into suitable vehicle for cloning. Furthermore, the synthesized<br><br><br>
product of this invention may be expressed in a suitable host-harboring expression vector.<br>
II   Application to Amplifying Target Nucleic Acid Sequence<br>
In still another aspect of this invention, there is provided a method for selectively  amplifying a target nucleic acid sequence from a DNA or a mixture of nucleic acids, which comprises amplifying the target nucleic acid sequence by performing at least two cycles of primer annealing, primer extending, and denaturing, using a pair of dual specificity oligonucleotides as a primer; wherein the dual specificity oligonucleotide has three portions in which a 5'-high Tm specificity portion has a hybridizing nucleotide sequence substantially<br>
complementary to a site on the target nucleic acid to hybridize therewith, a separation portion comprises at least two universal bases and a 3 '-low Tm specificity portion has a hybridizing nucleotide sequence substantially complementary to a site on the target nucleic acid to hybridize therewith, Tm of the 5'-high Tm specificity portion is higher than that of the 3'-low Tm specificity portion, the separation portion has the lowest Tm in the three portions; the separation<br>
portion forms a non base-pairing bubble structure under conditions that the 5'-high Tm specificity portion and the 3'-low Tm specificity portion are annealed to the target nucleic acid; wherein annealing in the amplification reaction is performed under conditions that annealing solely by the 3 '-low Tm specificity portion does not occur.<br>
Since the amplifying method of this invention employs the DS oligo of this invention, the<br>
common descriptions between them are omitted in order to avoid the complexity of this specification leading to undue multiplicity. In addition, since this method involves annealing and extension processes, the descriptions as to the two processes are omitted in order to avoid the complexity of this specification leading to undue multiplicity. For instance, the composition and structure of the DS oligo used and the conditions for annealing and extension are common<br>
between this process and the method for synthesizing nucleic acid molecule previously discussed.<br>
This application using the DS oligo of the subject invention can provide an improved method for selectively amplifying a target nucleic acid sequence from a nucleic acid or a mixture of nucleic acids (DNA or mRNA) by performing nucleic acid amplifications,<br>
preferably, PCR(polymerase chain reaction).<br>
A schematic representation for selectively amplifying a target nucleic acid of double-stranded DNA using the DS oligo as described above is illustrated in Figs. 2. As shown in Fig.<br><br><br>
2, one pair of the DS oligos is annealed to a denatured ds DNA template. The 5'-high Tm specificity portion and 3'-low Tm specificity portion during annealing play a role as a hybridizing portion or a specificity determining site (i.e., dual specificity determining site), while the separation portion does not serve as a hybridizing site and not interacted with the  template for base pairing. At this time, the separation portion forms a bubble structure in the DS oligo, in order that two end portions, i.e., the 5'-high Tm specificity portion and 3'-low Tm specificity portion can be spatially separated to dually determine the overall specificity of the DS oligo. Details of subsequent reactions are similar to that of conventional primer-based nucleic acid amplifications known in the art described hereinabove.<br>
	The present method for amplifying a nucleic acid sequence may be carried out in<br>
accordance with various primer-based nucleic acid amplifications known in the art. Preferably, the methods are carried out according to PCR process disclosed in U.S. Pat. Nos. 4,683,195, 4,683,202, and 4,800,159, more preferably, hot start PCR method.<br>
Fig. 3 illustrates a schematic representation for selectively amplifying a target nucleic acid<br>
of mRNA using the DS oligo. In the first step, mRNA obtained from various biological samples is reverse transcribed using oligo dT primer hybridizable to poly A tail of mRNA and reverse transcriptase. Details of reverse transcription are found in Joseph Sambrook, et al, Molecular Cloning, A Laboratory Manual, Cold Spring Harbor Laboratory Press, Cold Spring Harbor, N.Y.(2001); and Noonan, K. F. et al., Nucleic Acids Res. 16:10366 (1988)). The description of<br>
subsequent reactions is similar to that of conventional primer-based nucleic acid amplifications known in the art discussed previously.<br>
III.   Application to Multiplex DNA Amplification<br>
In further aspect of this invention, there is provided a method for amplifying two or more<br>
target nucleotide sequences simultaneously using two or more pairs of primers in the same reaction, which comprises amplifying the target nucleotide sequences by performing at least two cycles of primer annealing, primer extending and denaturing, using two or more pairs of dual specificity oligonucleotides as a primer, characterized in that the dual specificity oligonucleotides have three portions in which a 5'-high Tm specificity portion has a hybridizing<br>
nucleotide sequence substantially complementary to a site on the target nucleotide sequence to hybridize therewith, a separation portion comprises at least two universal bases and a 3'-low Tm specificity portion has a hybridizing nucleotide sequence substantially complementary to a site<br><br>
on the target nucleotide sequence to hybridize therewith, Tm of the 5'-high Tm specificity portion is higher than that of the 3'-low Tm specificity portion, the separation portion has the lowest Tm in the three portions; the separation portion forms a non base-pairing bubble structure under conditions that the 5'-high Tm specificity portion and the 3'-low Tm specificity  portion are annealed to the target nucleotide sequence; wherein annealing in the amplification reaction is performed under conditions that annealing solely by the 3'-low Tm specificity portion does not occur.<br>
This application using the DS oligo of the subject invention can also provide an improved<br>
method for amplifying more than one target sequence using more than one pair of primers in the same reaction. In general, it is extremely difficult to set up multiplex PCR conditions to amplify more than 10 target sequences in parallel, because an optimal PCR reaction is required to amplify even one specific locus without any unspecific by-products. Since annealing needs to take place at a sufficiently high temperature to allow the perfect DNA-DNA matches to<br>
occur in the reaction, the DS oligo of the subject invention is ideal in the optimization of multiplex DNA amplification due to its function of improving the specificity of amplification. "Multiplex PCR" as used herein refers to the simultaneous amplification of multiplex DNA targets in a single polymerase chain reaction (PCR) mixture.<br>
In  a  specific  embodiment  of this  invention,  this  Multiplexing  process  comprises<br>
performing an amplification reaction comprising at least two cycles of primer annealing, primer extending and denaturing, using the primer pairs of the DS oligo, characterized in that the primers are a dual specificity oligonucleotide having three portions in which a 5'-high Tm specificity portion has a hybridizing nucleotide sequence substantially complementary to a site on the target nucleotide sequence to hybridize therewith, a separation portion comprises at least<br>
two universal bases and a 3'-low Tm specificity portion has a hybridizing nucleotide sequence substantially complementary to a site on the target nucleotide sequence to hybridize therewith, Tm of the 5'-high Tm specificity portion is higher than that of the 3'-low Tm specificity portion, the separation portion has the lowest Tm in the three portions; the separation portion forms a non base-pairing bubble structure under conditions that the 5'-high Tm specificity portion and<br>
the 3'-low Tm specificity portion are annealed to the target nucleotide sequence; wherein annealing in the amplification reaction is performed under conditions that annealing solely by the 3'-low Tm specificity portion does not occur.<br><br><br>
Since this application using the DS oligo of this invention is carried out in accordance with the present method for amplification of nucleic acid sequence previously discussed, except for using more than one target nucleotide sequence and primer pairs, the common descriptions between them are omitted in order to avoid the complexity of this specification leading to  undue multiplicity. For instance, the composition and structure of the DS oligo used and the conditions for amplification, are common between this process and the present methods for amplification of nucleic acid sequence previously discussed.<br>
According to a preferred embodiment, the annealing temperature ranges from about 40°C to 70°C, more preferably, 45°C to 68°C, still more preferably, 50°C to 65°C, and most      preferably, 55°C to 65°C.<br>
In a preferred embodiment, the amplified products from each of target nucleotide sequences are different in size for subsequent analysis. According to a preferred embodiment, the amplification products of multiplex target nucleotide sequences may be analyzed through size separation. The size separation comparison is performed using a variety of method known in the art, such as electrophoresis through a polyacrylamide gel matrix or agarose gel matrix and nucleotide sequencing. The nucleotide sequencing may be rapidly carried out with an automatic sequencer available from various manufacturers.<br>
As exemplified in Example below, the multiplexing of this invention permits the final amplified products to be free from the background problems as well as non-specificity arising from conventional multiplex processes known in the art.<br>
	The advantage of the multiplex amplification is that numerous diseases or specific<br>
nucleotide sequence alterations (e.g., single nucleotide polymorphism or point mutation) can be<br>
assayed in the same reaction. The number of analyses that can be run simultaneously is<br>
unlimited; however, the upper limit is probably about 20 and is likely to be dependent on the<br>
size difference required for resolution and methods that are available to resolve the amplified<br>
product.<br>
	The method of the present invention may be applied to the diagnosis of genetic and<br>
infectious diseases, gender determination, genetic linkage analysis, and forensic studies.<br>
IV.    Application to DNA Sequencing<br>
The improved specificity allows DS oligo to be used in direct sequencing as a primer of<br>
      solution-phase sequencing (in particular, cycling sequencing) or as a probe of solid-phase<br><br><br>
sequencing (in particular, oligonucleotide chip sequencing) by using the principle of the template-dependent extension of DS oligo.<br>
In still further aspect of this invention, there is provided a method for sequencing a target nucleic acid molecule using a dual specificity oligonucleotide from a DNA or a mixture of       nucleic acids, which comprises the steps of:<br>
(a)	synthesizing a complementary nucleic acid molecule to the target nucleic acid<br>
molecule to be sequenced by performing at least two cycles of primer annealing, primer<br>
extending and denaturing, using the dual specificity oligonucleotide as a sequencing<br>
primer; wherein the dual specificity oligonucleotide has three portions in which a 5'-<br>
	high  Tm specificity  portion  has  a hybridizing  nucleotide  sequence  substantially<br>
complementary to a site on the target nucleic acid molecule to hybridize therewith, a separation portion comprises at least two universal bases and a 3'-low Tm specificity portion has a hybridizing nucleotide sequence substantially complementary to a site on the target nucleic acid molecule to hybridize therewith, Tm of the 5'-high Tm specificity<br>
	portion is higher than that of the 3'-low Tm specificity portion, the separation portion<br>
has the lowest Tra in the three portions; the separation portion forms a non base-pairing bubble structure under conditions that the 5'-high Tra specificity portion and the 3'-low Tm specificity portion are annealed to the target nucleic acid molecule; wherein annealing in the synthesizing reaction is performed under conditions that annealing<br>
	solely by the 3 '-low Tm specificity portion does not occur; and<br>
(b)	determining a nucleotide sequence of the synthesized complementary nucleic acid<br>
molecule.<br>
Generally, DNA sequencing has been carried out by various methodologies such as Maxam-Gilbert sequencing, Sanger sequencing, pyrosequencing, and exonuclease digestion       sequencing. The present sequencing method intends to improve the pyrosequencing as well as thermal cycle sequencing.<br>
The present method can be performed in accordance with variations of Sanger dideoxy<br>
 method. Thermal cycle sequencing of the present invention can be done on PCR amplified<br>
          nucleic acid templates. In addition, according to the invention, thermal cycle sequencing will be<br>
              performed on a nucleic acid template which has not been PCR amplified just prior to<br>
sequencing.<br><br><br>
In brief, Sanger sequencing is based on the principle that DNA polymerase will incorporate 2',3'-dideoxynucleotides into nucleic acid chains resulting in chain termination (Sanger et al., (1977) PNAS. USA 74:5463). The method developed by Sanger is referred to as the dideoxy chain termination method. In the most traditional one of this method, a DNA  segment for which the sequence is desired is cloned into a single-stranded DNA phage, such as M13. These phage DNAs can serve as templates for the primed synthesis of the complementary strand by the Klenow fragment of DNA polymerase I. The primer is a synthetic oligonucleotide to hybridize specifically with a region of the Ml 3 vector near the 3' end of the cloned insert. In each of four sequencing reactions, the primed synthesis is carried out in the presence of<br>
sufficient dideoxy analog of one of the four possible deoxynucleotides, in order that the growing chains are randomly terminated by the incorporation of these dead-end nucleotides. The relative concentration of dideoxy to deoxy forms is adjusted to give a spread of termination events corresponding to all the possible chain lengths that can be resolved by gel electrophoresis. Tags incorporated in the growing chains are used to develop an autoradiogram<br>
image of the pattern of the DNA in each electrophoresis track. The sequence of the deoxynucleotides in the cloned nucleic acid template is determined from an examination of the pattern of bands in the four lanes<br>
As a variation of Sanger method, thermal cycle sequencing method normally involves the use of solutions containing a nucleic acid sequencing primer, deoxynucleoside triphosphates,<br>
one or more dideoxynucleoside triphosphates (ddNTPs), a suitable buffer solution, a thermal stable DNA polymerase (e.g., Taq polymerase), and the nucleic acid template to be sequenced. The details of thermal cycle sequencing may be found in U.S. Pat. Nos. 5,432,065, 5,723,298, 5,756,285, 5,817,797 and 5,831,065, the teachings of which are incorporated herein by references in their entity. The processes of the method are generally performed under thermal<br>
cycling conditions similar to common PCR.<br>
Once a sequencing reaction has been performed on a nucleic acid template, the determination of the sequence of the molecule requires that the reaction products be identified. A considerable number of detection methods are known in the art. These methods generally involve   the   detection   of   tags   including   radionucleotide,   fluorescent,   infrared   and<br>
chemiluminescent labels as described in Ausubel, F. M. et al., Current Protocols in Molecular Biology, (1993) John Wiley &amp; Sons, Inc., New York, N.Y. The labels may be tagged with primers or ddNTP, preferably, ddNTP. The most preferred label is fluorescent one including 6-<br><br><br>
carboxyfluorescein,	6-carboxy-X-rhodamine,	3-(E-carboxypentyl)-3'-ethyl-5,5'-<br>
dimethyloxacarbocyanine,       6-carboxy-X-rhodamine,       4,4-difIuoro-4-bora-3a,4a-diaza-s-indacene-3-propionic acid derivatives, and 4,7-dichlororhodamine dyes.<br>
Preferably, the annealing temperature ranges from about 40°C to 70°C, more preferably,      45°C to 68°C, and most preferably, 50°C to 65°C.<br>
The present method proves a high specific sequencing of a target nucleic acid molecule as demonstrated in Example hereunder. More specifically, the mouse placenta-specific homeobox family genes, Psxl and Psx2 can be differentially sequenced using their sequencing primer designed to have a unique structure of the DS oligo. Such differential sequencing is highlighted  in the senses that the overall sequences of the sequencing primers are different in only one base in the 3 '-low Tm specificity portion.<br>
Surprisingly, the present invention permits a target nucleic acid molecule contained in a genomic DNA or a population of cDNAs to be directly sequenced without purification or isolation. The success in direct sequencing of a target nucleic acid molecule in a genomic DNA  or a population of cDNAs has not yet been reported. Where the present sequencing method is employed to directly sequence a target nucleic acid molecule contained in a population of cDNAs from a total RNA, this method comprises the steps of:<br>
	(a) contacting a population of mRNAs with an oligonucleotide dT primer which is<br>
hybridized to polyA tail of the mRNAs under conditions sufficient for template driven<br>
enzymatic deoxyribonucleic acid synthesis to occur;<br>
(b)	reverse transcribing the mRNAs to which the oligonucleotide dT pirmer hybridizes<br>
to produce a population of a first cDNA strands that are complementary to the mRNAs to<br>
which the oligonucloetide dT primer hybridizes;<br>
(c)	synthesizing a complementary nucleic acid molecule to the first cDNA strand to be<br>
sequenced by performing at least two cycles of primer annealing, primer extending and<br>
	denaturing, using the dual specificity oligonucleotide as a sequencing primer; wherein the<br>
dual specificity oligonucleotide has three portions in which a 5'-high Tm specificity<br>
portion has a hybridizing nucleotide sequence substantially complementary to a site on the  the first cDNA strand to hybridize therewith, a separation portion comprises at least two<br>
	universal bases and a 3'-low Tm specificity portion has a hybridizing nucleotide sequence<br>
substantially complementary to a site on the first cDNA strand to hybridize therewith, Tra of the 5'-high Tm specificity portion is higher than that of the 3'-low Tm specificity portion,<br><br><br>
the separation portion has the lowest Tra in the three portions; the separation portion forms<br>
a non base-pairing bubble structure under conditions that the 5'-high Tm specificity portion<br>
and the 3'-low Tm specificity portion are annealed to the first cDNA strand; wherein<br>
annealing in the synthesizing reaction is performed under conditions that annealing solely<br>
by the 3 '-low Tm specificity portion does not occur; and<br>
(b)   determining a nucleotide sequence of the synthesized the first cDNA strand.<br>
V.   Application to Detection of Nucleic Acid Molecule with Genetic Diversity<br>
In another aspect of this invention, there is provided a method for detecting a nucleic acid<br>
molecule with genetic diversity by a template-dependent extension reaction of a dual specificity oligonucleotide, which comprises the steps of:<br>
(a) annealing the dual specificity oligonucleotide to a template nucleic acid molecule, wherein the dual specificity oligonucleotide has three portions in which a 5'-high Tm specificity portion has a hybridizing nucleotide sequence substantially complementary<br>
	to a site on the template nucleic acid to hybridize therewith, a separation portion<br>
comprises at least two universal bases and a 3'-low Tm specificity portion has a hybridizing nucleotide sequence substantially complementary to a site on the template nucleic acid to hybridize therewith, Tm of the 5'-high Tm specificity portion is higher than that of the 3'-low Tm specificity portion, the separation portion has the lowest Tm in<br>
	the three portions; the separation portion forms a non base-pairing bubble structure<br>
under conditions that the 5'-high Tm specificity portion and the 3'-low Tm specificity portion are annealed to the template nucleic acid, wherein the annealing is performed under conditions that annealing solely by the 3'-low Tm specificity portion does not occur and annealing occurs when the 5'-high Tm specificity portion and/or the 3'-low Tm<br>
	specificity portion has one or more mismatched bases to its target site; and<br>
(b)	extending the dual specificity oligonucleotide to synthesize the nucleic acid<br>
molecule complementary to the template.<br>
(c)	detecting the occurrence of the template-dependent extension of the dual specificity<br>
oligonucleotide<br><br>
Since this application using the DS oligo of this invention is carried out in accordance with the present methods for synthesizing nucleic acid sequence previously discussed, the<br><br><br>
common descriptions between them are omitted in order to avoid the complexity of this specification leading to undue multiplicity.<br>
This application using the DS oligo of the subject invention can provide an improved method for selectively detecting a nucleic acid sequence with genetic diversity by a template- dependent extension reaction involving annealing and extending steps. In particular, the detection of a target nucleic acid sequence with genetic diversity can be achieved by repeating the process of the template-dependent extension reaction in which the annealing and extension steps are followed by denaturation step.<br>
	The present invention is based on mismatch tolerance of the DS oligo.<br>
Genetic diversity has been reported for various genomes. This phenomenon has been considered an obstacle to detect a gene or genome of interest without failure. The present invention is directed to provide an approach to overcome such conventional problems by use of the DS oligo with mismatch tolerance. The DS oligo with a defined sequence can be annealed<br>
to several target sequences showing genetic diversity and result in successful amplification and detection of nucleotide sequences of interest. In other words, the DS oligo originally developed to dramatically enhance specificity of annealing and hybridization can be also used in processes requiring mismatch tolerance where annealing or stringent conditions are suitably adjusted.<br>
To provide the DS oligo demonstrating mismatch tolerance, it should be designed on the<br>
basis of a conserved region of nucleic acid molecules generated by aligning all available nucleotide sequences. The term "conserved region" as used herein refers to a segment of nucleotide sequence of a gene or amino acid sequence of a protein that is significantly similar between various different nucleotide sequences of a gene. This term is interchangeably used with the term "conserved sequence".<br>
	In a preferred embodiment, the most conserved sequence within the conserved region is<br>
located in the 3'-end portion of the DS oligo and the lowest conserved sequence in the separation portion. The 5'-high Tm specificity portion and/or the 3'-low Tm specificity portion, preferably 5'- high Tm specificity portion may one or more, preferably, one to three, more preferably one or two mismatched bases to its target site due to mismatch tolerance of the DS<br>
oligo.<br>
For imposing mismatch tolerance on the DS oligo, the annealing condition, in particular, annealing temperature is important. The annealing is performed under conditions that annealing<br><br><br>
solely by the 3'-low Tm specificity portion does not occur, yet annealing by all the portions occurs when the 5'-high Tm specificity portion and/or the 3'-low Tm specificity portion has one or more mismatched bases to its target site.<br>
Preferably, the annealing temperature ranges from about 40°C to 70°C, more preferably,       45°C to 68°C, and most preferably, 50°C to 65°C.<br>
According to a preferred embodiment, the present invention is performed in accordance with polymerase chain reaction (PCR).<br>
The detection step of the present method may be carried out by a multitude of conventional techniques. For example, the detection of the product of the template-dependent<br>
extension may be readily performed by conventional gel electrophoresis if the preset method is executed in a repeated manner to generate products sufficient to be detected in a gel. If labeled materials including those detectable by spectroscopic measurement, photochemical measurement, biochemical measurement, bioelectronic measurement, immunochemical measurement, electronic measurement, and chemical measurement, a suitable measurement<br>
may be performed for detecting the occurrence of the template-dependent extension.<br>
Genetic diversity is most frequently found and generated in viral genome (Nathalie B. et al, (2004) Journal of Clinical Microbiology, 42, 3532; Tersa C. et al., (2002) Journal of Infectious Diseases, 185, 1660; Takashi E. et al., (2004) Journal of Clinical Microbiology, 42, 126; and Elizabeth R. et al., (2001) Clinical Infectious Diseases, 32, 1227). In this regard, it is<br>
preferred that the nucleic acid molecule with genetic diversity to be detected is a nucleic acid of a virus exhibiting genetic diversity. For example, where the present invention is applied to detect human metapneumoviruses exhibiting genetic diversity by PCR, the most preferable primer set designed to have the structure of the DS oligo is set forth in SEQ ID NOs: 39 (for 5' primer) and 40 (for 3' primer) or SEQ ID NOs: 39 and 41 (for 3' primer).<br><br>
VI. Application to Detection of Target Nucleotide Sequence Using DS Oligos Immobilized on Microarray<br>
This application is a novel process for detecting target nucleotide sequence by repeating the template-dependent reaction on DS oligo-immobilzed microarray.<br>
	In another aspect of this invention, there is provided a method for detecting a target<br>
nucleotide sequence in a nucleic acid sample by a template-dependent extension reaction, comprising the steps of:<br><br><br>
(a)	extending a dual specificity oligonucleotide as a probe immobilized on a substrate<br>
comprising at least one cycle of a hybridization, a template-dependent extension and a<br>
denaturation, wherein the hybridization is performed by contacting the dual specificity<br>
oligonucleotide to the nucleic acid sample, wherein the dual specificity oligonucleotide<br>
	has three portions in which a 5'-high Tm specificity portion has a hybridizing nucleotide<br>
sequence substantially complementary to a site on the target nucleotide sequence to hybridize therewith, a separation portion comprises at least two universal bases and a 3'-low TOT specificity portion has a hybridizing nucleotide sequence substantially complementary to a site on the target nucleotide sequence to hybridize therewith, Tm of<br>
	the 5'-high Tm specificity portion is higher than that of the 3’-low Tm specificity portion,<br>
the separation portion has the lowest Tm in the three portions; the separation portion forms a non base-pairing bubble structure under conditions that the 5'-high Tm specificity portion and the 3'-low Tm specificity portion are annealed to the target nucleotide sequence, wherein the hybridization is performed under conditions that hybridization<br>
	solely by the 3'-low Tm specificity portion does not occur; and<br>
(b)	analyzing the occurrence of the template-dependent extension.<br>
A schematic representation for detecting a target nucleotide sequence in a nucleic acid sample by use of DS oligo-immbolized microarray is illustrated Fig. 4.<br>
This process using the DS oligos may be carried out under suitable hybridization<br>
conditions routinely determined by optimization procedures. Conditions such as temperature, concentration of components, hybridization and washing times, buffer components, and their pH and ionic strength may be varied depending on various factors, including the length and GC content of oligonucleotide and target nucleotide sequence. For instance, when a relatively short oligonucleotide is used, it is preferable that low stringent conditions are adopted. The detailed<br>
conditions for hybridization can be found in Joseph Sambrook, et al., Molecular Cloning, A Laboratory Manual, Cold Spring Harbor Laboratory Press, Cold Spring Harbor, N.Y.(2001); and M.L.M. Anderson, Nucleic Acid Hybridization, Springer-Verlag New York Inc. N.Y.(1999).<br>
The DS oligos are immobilized on a substrate. A preferable substrate includes suitable<br>
solid or semi-solid supporters, such as membrane, filter, chip, slide, wafer, fiber, magnetic or nonmagnetic bead, gel, tubing, plate, macromolecule, microparticle and capillary tube. Such immobilization may occur through  chemical binding or covalent binding by ultra-violet<br><br><br>
radiation. In an embodiment of this invention, the DS oligos are bound to a glass surface modified to contain epoxy compounds or aldehyde groups or to a polylysin-coated surface. Furthermore, the DS oligos are bound to a substrate through linkers (e.g. ethylene glycol oligomer and diamine). Immobilized DS oligos may be fabricated to produce array or arrays for a given application by conventional fabrication technologies such as photolithography, ink-jetting, mechanical microstting, and derivatives thereof.<br>
According to the present method, dNTPs used in the extension step are preferably labeled. For labeling, materials detectable by spectroscopic measurement, photochemical measurement, biochemical   measurement,   bioelectronic   measurement,   immunochemical   measurement,<br>
electronic measurement, or chemical measurement are used. For instance, the labels include, yet are not limited to, radioisotopes like P32 and S35, chemilluminescent compounds, spectroscopic markers, such as fluorescence markers and dyes, and magnetic labels. The dyes, for example, include, but not limited to, quinoline dye, triarylmethane dye, phthalein, azo dye, and   cyanine  dye.   The  fluorescence  makers   include,   but  not  limited  to,   fluorescein,<br>
phycoerythrin, rhodamine, lissamine, Cy3 and Cy5 (Pharmacia). Labeling is performed according to various methods known in the art.<br>
Target nucleotide sequences in a nucleic acid sample are hybridized with the DS oligos as probes immobilized in a substrate, preferably, a solid support, and in turn the DS oligos hybridized   with   target   nucleotide   sequences   are   extended   using   dNTPs,   preferably<br>
fluorescence- labeled dNTPs and DNA polymerase in a template-dependent fashion. The step (a) is preferably repeated to perform hybridization reactions to the extent that all or most of the DS oligos are hybridized with target nucleotide sequences, making the results of hybridization analysis more reproducible.<br>
The occurrence of hybridization is verified with various methods known in the art<br>
depending on the types of labels used. For example, fluorescence microscope, preferably, confocal fluorescence microscope is used for fluorescence labels, and the intensity of the signal detected with such instruments increases proportionally to the extent of hybridization. Fluorescence microscopes, in general, are equipped with a scanning device which builds up a quantitative two dimensional image of hybridization intensity. The intensity of the signal<br>
detected with such instruments increases proportionally to the extent of hybridization and then the extent of a template-dependent extension.<br><br><br>
The present invention will now be described in further detail by examples. It would be obvious to those skilled in the art that these examples are intended to be more concretely illustrative and the scope of the present invention as set forth in the appended claims is not limited to or by the examples. <br>
EXAMPLE 1: PCR Specificity using Dual Specificity (DS) Oligonucleotides<br>
The DS oligonucleotides developed by the present invention were applied as primers to amplify target nucleotide sequences of mouse cytokine family genes EL-19 and IL-lbeta. The process and results for the amplification of the target nucleotide sequences of IL-19 and IL-   lbeta using DS primers are described herein.<br>
The following conventional primer sequences were chosen and used for comparing to DS oligonucleotides in view of PCR specificity:<br>
The IL-19-specific conventional primers used in the Example (500 bp) are: IL19-5'-0    5'-GTCTCATCTGCTGCCCTTAAGTCTCTAGGAGAACT-3' (SEQ ID NO:l); 15      and<br>
IL19-3'-0    5'-CATAGGCCTGGAAGAAGCCGCTTTACAATAAGTTAG-3' (SEQ ID NO:2).<br>
The IL-lbeta-specific conventional primers used in the Example (550 bp) are: ILlb-5'-0  5'-GGAGAGTGTGGATCCCAAGCAATACCCAAAGAAG-3'  (SEQ  ID NO:3); and       ILlb-3'-0    5'-AGACCTCAGTGCAGGCTATGACCAATTCATCCC-3'(SEQIDNO:4).<br>
The DS oligonucleotides of the subject invention were applied to these conventional<br>
primer sequences to demonstrate whether the DS oligonucleotides can overcome the main<br>
problems arising from these conventional primer sequences, such as generation of background<br>
         and non-specific products.<br>
	The following DS primers comprise sequences identical to the above conventional<br>
primers except the separation portion having a polydeoxyinosine [poly(dl)] linker between the 5' portion and the 3' portion. The DS primers are designed to comprise a 5'-high Tm specificity portion and a 3'-low Tm specificity portion in a way that Tm of the 5'-high Tm specificity portion is higher than that of the 3'-low Tni specificity portion, while the separation portion has the 30      lowest Tm among the three portions.<br>
The DS primers for IL-19 used in the Example (500 bp) are: IL19-5'     5'-GTCTCATCTGCTGCCCTTAAinnTAGGAGAACT-3' (SEQ ID NO:5); and<br><br><br>
IL19-3' 5'-CATAGGCCTGGAAGAAGCCGIIIIICAATAAGTTAG-3' (SEQ ID NO:6), wherein I is deoxyinosine.<br>
The DS primers for IL-lbeta used in the Example (550 bp) are: <br>
ILlb-5'     5'-GGAGAGTGTGGATCCCAAGCITIIICCAAAGAAG-3' (SEQ ID NO:7); and      ILlb-3'     5'-AGACCTCAGTGCAGGCTATGIIIIITTCATCCC-3' (SEQ ID NO:8), wherein I is deoxyinosine.<br>
The target PCR amplification was conducted in the final volume of 20 ml containing 2 p.1 (50 ng) of the genomic DNA isolated from placenta tissues of mouse strain ICR, 2 ml of 10 x PCR reaction buffer containing 15 mM MgCl2 (Roche), 2 u.1 of dNTP (2 mM each dATP, dCTP,<br>
dGTP and dTTP), 1 ml of 5' DS or conventional primer (10 mM), 1 ml of 3' DS or conventional primer (10 mM), and 0.5 ul of Taq polymerase (5units/ml; Roche); the tube containing the reaction mixture was placed in a preheated (94°C) thermal cycler; samples were denatured for 5 min at 94°C and subjected to 30 cycles of 1 min at 94°C, 1 min at 60°C, and 1 min at 72°C, followed by a 7-min incubation at 72°C.<br>
	The amplified products were analyzed by electrophoresis in a 2% agarose gel and were<br>
detected by staining with ethidium bromide. The resulting PCR products could be also detected on a denaturing polyacrylamide gel by autoradiography or non-radioactive detection methods, such as silver staining (Gottschlich et al., (1997) Res. Commun. Mol. Path. Pharm. 97, 237-240; Kociok, N., et al. (1998) Mol. Bioteclmol. 9, 25-33), or by using fluoresenscent-labelled<br>
oligonucleotides (Bauer D., et al., (1993) Nucleic Acids Res. 21, 4272-4280; Ito, T. et al., (1994) FEES Lett. 351, 231-236. Luehrsen, K.R. et al., (1997) BioTechniques 22, 168-174; Smith, N.R. et al., (1997) BioTechniques 23, 274-279), and the use of biotinylated primers (Korn, B. et al., (1992) Hum. Mol. Genet. 1, 235-242; Tagle, D.A. et al., (1993) Nature 361. 751-753; Rosok, O. etal., (\996) BioTechniques 21, 114-121).<br>
	As shown in FIG. 5, the target PCR amplifications for cytokine family genes IL-lb and IL-<br>
19 using each primer set of ILlb-5' and ILlb-3', and IL19-5' and IL19-3' generate a single band which corresponds to the expected size of 550-bp for IL-lbeta (lane 2) and the expected size of 500-bp for IL-19 (lane 4), respectively. Subsequent cloning and sequence analysis of the clones confirmed that the bands are IL-lbeta and IL-19 fragments. In contrast, the conventional<br>
primer sets (IL19-5'-0 and IL19-3'-0; l Llb-5'-0 and ILlb-3'-0), which do not contain the [poly(dl)], produced non-specific products (FIG 5, lanes 1 and 3). These results indicate that the DS primers designed to have three different Tm portions (5'-high Tm specificity portion, 3'-<br><br><br>
low Tm specificity portion and separation portion) can overcome the main problems arising from the conventional primer sequences such as background and non-specific products and enhance PCR specificity remarkably.<br>
       EXAMPLE 2: Evaluation of PCR Specificity using Dual Specificity (DS) Oligonucleotides<br>
The dual specificity oligonucleotides are characterized by high hybridization specificity and mismatch tolerance arising from their unique structure, depending on hybridization stringency. High hybridization specificity of the DS oligonucleotides is achieved under high stringency conditions that both 5'- and 3'-portions are annealed to the template. Meanwhile,<br>
mismatch tolerance of the DS oligonucleotides is achieved under stringency conditions that both 5'- and 3'-portions are annealed to the template despite the presence of one or more, yet limited, base pair mismatches.<br>
The dual specificity of DS oligonucleotides developed according to the present invention was evaluated in terms of hybridization specificity and mismatch tolerance by the 3'-RACE of<br>
a novel gene, DEG10, which was identified to be expressed in mouse placenta (Kim, YJ. et al., (2004) Annealing control primer system for identification of differentially expressed genes on agarose gels. BioTechniques 36:424-434; XM_129567). For this evaluation, a few nucleotides in both portions were replaced with other nucleotides as to be mismatched to the target template sequence.<br>
           The 5'-DEG10-specific DS primers used in the Example are:<br>
DEG10-5'-108: 5'-TGTAGTTTTGGGTTTCCTCCIIHICTCCGATG-3' (SEQ ID NO:9); <br>
DEG10-5'-103: 5 '-TGTAGTTTTGGGTTTCCTCCIIIIIICTGCCATC-3' (SEQ ID NO: 10); <br>
DEG10-5'-102: 5'-TGTAGTTTTGGGTTTCCTCCIIIIICTCCCATC-3'(SEQ ID NO: 11); <br>
DEG10-5'-101: 5'-TGTAGTTTTGGGTTTCCTCCIIIIICTCCCATG-3: (SEQ ID NO: 12);<br>
DEG10-5'-158: 5'-TGTACTTATGCGTATCGTCCIIIIICTCCGATG-3' (SEQ ID NO.13);<br>
DEG10-5'-138: 5'-TGTACTTTTGCGTTTCGTCCIIIIICTCCGATG-3'  (SEQ ID NO:14);<br>
 and<br>
DEG10-5'-128:     5'-TGTAGTTATGGGTATCCTCCIIIIICTCCGATG-3' (SEQ ID NO: 15), wherein the replaced nucleotides are underlined and bold and I is deoxyinosine.<br><br><br><br>
A. Enhancement in PCR specificity under high stringency<br>
Total RNAs from 17.5-dpc (E17.5) placenta tissues of mouse strain ICR were isolated<br><br><br>
and used for the synthesis of first-strand cDNAs by reverse transcriptase, as described previously (Hwang, I.T., et al., (2003) Annealing control primer system for improving specificity of PCR amplification. BioTechniques 35:1180-1184). Reverse transcription reaction was performed using the total RNAs for 1.5 hr at 42 °C in a reaction volume of 20 ml composed  of the following: 3 mg of total RNA, 4 ml of 5 x reaction buffer (Promega, USA), 5 m1 of dNTPs (each 2 mM), 2 m1 of 10 mM cDNA synthesis primer (oligo (dT)2o-Adaptor), 0.5 ml of RNase inhibitor (40 units/ml, Promega), and 1 ml of reverse transcriptase (200 units/m1, Promega). First-strand cDNAs were diluted by adding 180 m1 of ultra-purified H20. The cDNA synthesis primer oligo (dT)i8-ACPl is: 5'-CTGTGAATGCTGCGACTACGATirrn(T)18-3', wherein I is<br>
          deoxyinosine.<br>
The 3'-RACE of DEG10 was conducted in a final volume of 20 ml containing 2 ml (30 ng) of the diluted first strand cDNA, 2ml of 10 x PCR reaction buffer containing 15 mM MgCl2 (Roche), 2m1 of dNTP (2 mM each dATP, dCTP, dGTP, dTTP), ml of one of DEGlO-specific DS primers (10 mM), 1 ml of oligo (dT)15-ACP2 (10 mM), and 0.5 m.1 of Taq polymerase<br>
         (5units/ml; Roche); the tube containing the reaction mixture was placed in a preheated (94°C) thermal cycler; samples were denatured for 5 min at 94°C and subjected to 30 cycles of 1 min at 94°C, 1 min at 68°C, and 1 min at 72°C, followed by a 7-min incubation at 72°C. The oligo (dT)15-ACP2 is: 5'-CTGTGAATGCTGCGACTACGATIIIII(T)I5-3\ wherein I is deoxyinosine.<br>
         B. Mismatch tolerance of DS oligonucleotides<br>
The sample DS primers, template, and PCR conditions for the 3'-RACE of DEG10 used in the Example 2A were employed except for the annealing temperature. PCR amplification was conducted under the following conditions: one cycle of 94°C for 5 min, 60°C for 3 min, and 72°C for 3 min; followed by 29 cycles of 94°C for 40 s, 65°C for 1 min, and 72°C for 40s, and<br>
         a 7-min final extension cycle at 72°C.<br>
As a result, FIG. 6A shows the high hybridization specificity of DS oligonucleotide primers by the 3'-RACE of DEG10. The intact 5'-DEG10-specific DS primer (DEG10-5'-108) generated an expected 677-bp product of the DEG10 3'-RACE (lane 1). In contrast, the other  primers (DEG10-5'-103, DEG10-5'-102, DEG10-5'-101, DEG10-5'-158, DEG10-5'-138, and DEG10-5'-128) with mismatched sequences at the 5' portion or the 3' portion did not generate any product: the mismatch of three (lane 2), two (lane 3), or one (lane 4) base(s) at the 3'<br><br><br>
portion; the mismatch of five (lane 5), three (lane 6), or two (lane 7) base(s) at the 5' portion.<br>
	These results demonstrate that the dual specificity of the DS primers can discriminate the<br>
mismatched bases not only at the 3'-end but also at the 5'-end under such high stringency<br>
conditions.<br>
	In general, the region of primers that should be perfectly complementary to the template is<br>
the 3 '-end, because this end is the region extended by DNA polymerase and is therefore the most important for ensuring annealing to the correct target sequence to occur. Meanwhile, the 5'-end of primers is less important in determining the specificity of annealing to the target sequence and can be modified to carry additional sequence such as restriction sites and<br>
promoter sequences that are not complementary to the template (McPherson, M J., Moller, S.G. (2000) PCR. BIOS Scientific Publishers, Springer-Verlag New York Berlin Heidelberg, N.Y.). In contrast to these, the exceptional advantage of the DS primers is demonstrated by the dual specificity due to their unique structure, allowing discrimination of mismatched bases in 5'-end as well as in 3'-end.<br><br>
FIG. 6B shows an example of the mismatch tolerance of DS oligonucleotide primers by the 3'-RACE of DEG10. Although the DS primers (DEG10-5-108, DEG10-5M01, DEG10-5'-128, and DEG10-5'-138) having no or a few mismatch nucleotides at their 5'- or 3'-end portions still generated an expected 677 bp product of the DEG10 3'-RACE (lanes 1, 4, 6 and<br>
    	 7). In contrast, the other primers (DEG10-5'-103, DEG10-5M02, and DEG10-5'-158) with more mismatched nucleotides at their 5' or 3' portions did not generate any product (lanes 2, 3 and 5). These results indicate that the DS primers can be also applied to amplifications of diverse nucleotide sequences requiring mismatch tolerance.<br>
	In summary, these results support the following principles of DS primers:<br>
1)	only when annealing occurs by both 5'-high Tm specificity portion and 3'- low Tm<br>
specificity portions, the DS primer is extended to synthesize a nucleic acid molecule<br>
complementary to the template (lane 1); however,<br>
2)	when annealing occurs only by the 5'-high Tm specificity portion and not by the 3'- low<br>
          Tm specificity portion, the DS primer is not extended to synthesize a nucleic acid molecule<br>
complementary to the template (lanes 2-4); and<br>
3)	even though the sequence of the 3'- low Tm specificity portion has a perfect match to<br><br><br>
the template, the annealing solely by the 3'-low Tm specificity portion does not occur under high stringent conditions (lanes 5-7). Regarding to the annealing portion, the DS primer is distinctly different from the annealing control primer (ACP) that is extended only by annealing of the 3'-end portion in an initial PCR step (Hwang, I.T., et al., (2003) Annealing control      primer system for improving specificity of PCR amplification. BioTechniques 35:1180-1184).<br>
EXAMPLE 3: Single-base Discrimination Using Dual Specificity (DS) Oligonucleotides<br>
To demonstrate the dual specificity of DS oligonucleotides for single-base discrimination, the mouse placenta-specific homeobox family genes Psxl and Psx2 cDNAs were amplified<br>
with either conventional primers or DS primers. Overall sequence identity between the two Psx cDNAs was 91% at the nucleotide level (Han, Y.J., et al., (2000) Identification and characterization of Psx2, a novel member of the Psx (placenta-specific homeobox) family. Gene 241:149-155). The 5'-primers were designed to distinguish Psxl and Psxl by one- or two-base discrimination (FIG 7A). However, the 3'-primer was designed to have a conserved<br>
sequence for both Psx cDNAs. The Psxl- and Psx2-specific  conventional and DS primer sequences are:<br><br>
Psxl-5’-10; 5'-AAGGAAGACATGCTGGTGATGGTGCTTCTAGCT-3' (SEQ ID NO: 16); <br>
Psx2-5'-10; 5'-AAGGAAGACATGCTGGTGATGGTGCTTCTGGCC-3' (SEQ ID NO:17); <br>
Psxl-5'-11; 5'-AAGGAAGACATGCTGGTGATIIIIITTCTAGCT-3' (SEQ ID NO: 18);<br>
Psx2-5'-11; 5'-  -AAGGAAGACATGCTGGTGATIIIIITTCTGGCC-3' (SEQ ID NO:19); <br>
Psxl-5'-40; 5'- TCTTGCACGATGGATGGGTGTGGATGAATGTGA-3' (SEQ ID NO:20); <br>
Psxl-5'-41; 5'-TCTTGCACGATGGATGGGTGTGGATGAATCTGA-3' (SEQ ID NO:21);<br>
Psx2-5'-41;5'- TCTTGCACGATGGATGGGTGIIinGAAIGIGA-3' (SEQ ID NO:22); <br>
Psx2-5’-41;5'-TCTTGCACGATGGATGGGTGIIHIGAAICIGA-3'(SEQ ID NO:23) and <br>
          Psx-3'-2:        5'-TTCATCCACACCCATCCATCIIIIIAGATCCCT-3' (SEQ ID NO:24), wherein the Psxl- or Psx 2-specific nucleotides are underlined and bold.<br>
A.	First-strand cDNA Synthesis<br>
The mouse placenta first-strand cDNA synthesized in Example 2 was used as a starting       material for 3'-RACE and a target PCR of Psx cDNA.<br>
B.	3'-RACE of Psxl and Psx2 using Psxl- and Psx-2-Specific DS primer<br>
The 3'-RACE of Psxl and Psx2 were conducted in a final volume of 20 ul containing 2 ml<br><br>
(30 ng) of the diluted first strand cDNA, 2 p.1 of 10 x PCR reaction buffer containing 15 mM MgCl2 (Roche), 2 pi of dNTP (2 mM each dATR dCTP, dGTP and dTTP), 1 p.1 of one of 5'-Psxl- or 5'-P.sx2-specific DS or conventional primers (10 uM), 1 pi of oligo (dT)i5-ACP2 (10 pM), and 0.5 pi of Taq polymerase (5 units/pl; Roche); the tube containing the reaction mixture  was placed in a preheated (94°C) thermal cycler; samples were denatured for 5 min at 94°C and subjected to 30 cycles of 1 min at 94°C, 1 min at 60-65°C, and 1 min at 72°C, followed by a 7-min incubation at 72°C.<br>
C. Target nucleic Acid Amplification of Psxl and Psxl using Psxl- and Psx^-Specific DS<br>
          primers<br>
The target PCR amplification of Psxl and Psx2 were conducted in a final volume of 20 p.1 containing 2 pi (30 ng) of the diluted first strand cDNA, 2 p.1 of 10 x PCR reaction buffer containing 15 mM MgCl2 (Roche), 2 pi of dNTP (2 mM each dATP, dCTP, dGTP and dTTP), 1 pi of one of S'-Psxl- or 5'-P.sx2-specific DS or conventional primers (10 p.M), 1 p.1 of Psx-3'-2<br>
(10 p,M), and 0.5 pi of Taq polymerase (5units/ui; Roche); the tube containing the reaction mixture was placed in a preheated (94°C) thermal cycler; samples were denatured for 5 min at 94°C and subjected to 30 cycles of 1 min at 94°C, 1 min at 60-65°C, and 1 min at 72°C, followed by a 7-min incubation at 72°C.<br>
 	As a result, FIG. 7B shows the products of the 3'-RACE and target PCR generated by 5'-<br>
Psxl- or -Ax2-specific primers. Since the two Psx cDNAs differ from each other by 29-bp deletion or insertion toward their 3'-end, the products different in size by 29-bp are expected to be amplified by their 3'-RACE. The 3'-RACE of Psxl cDNA using the Psxl- or P^2-specific DS primers Psxl-5'-41 and Psx2-5'-41 each generated a single band which corresponds to the<br>
 expected size 311-bp (lane 1) and 282-bp (lane 2), respectively. Subsequent sequence analysis of the products generated by the 3'-RACE confirmed that the S'-Psxl- and -ftr2-specific primers amplified Psxl and Psx2 cDNAs, respectively. In contrast, the conventional primers (Psxl-5'-40 and Psx2-5'-40), which do not comply with the principles of DC oligonucleotides, did not distinguish two Psx cDNAs (lanes 3 and 4).<br>
 	These results indicate that the DS primers according to the present invention can<br>
discriminate a single-base mismatch. Therefore, DS oligonucleotides can be applied for identifying point mutations or single nucleotide polymorphism genotyping.<br><br><br>
EXAMPLE 4: Direct Sequencing of a Target cDNA from cDNA Pool Using Dual Specificity (DS) Oligonucleotides<br>
Most attempts to identify and isolate a novel cDNA result in the acquisition of clones that  represent only a part of the mRNA's sequence. Once the partial sequence has been identified, the remainder of the transcript can be often obtained through either typical cDNA library screening or PCR-based methods such as RACE (rapid amplification of cDNA ends), followed by the sequencing of the obtained cDNA. Thus, all current methods are a prerequisite step for obtaining the sequence information of the remainder of the transcript. If the missing sequence  information is directly obtained from a population of cDNAs generated from a target cell, these time-consuming prerequisite steps can be completely bypassed and the sequence of the target cDNA can be determined directly from a crude biological sample.<br>
The DS oligonucleotides of the subject invention were applied as primers to directly<br>
sequence mouse placenta-specific homeobox gene Psx cDNAs using placenta first-strand<br>
      cDNA pool. The process and results for direct sequencing of the placenta-specific gene cDNAs<br>
from placenta cDNA pool are described herein. The same S'-i^-specific DS primers used in<br>
Example 3 were used. They are Psxl-5'-ll, Psx2-5'-ll, Psxl-5'-41, and Psx2-5'-41.<br>
A.	First-strand cDNA Synthesis<br>
	The mouse placenta first-strand cDNA synthesized in Example 2 was used as a template<br>
for direct sequencing of Psx cDNAs.<br>
B.	Direct Sequencing of Psx cDNA from Placenta cDNA Pool using Psx Specificity DS primer<br>
       The cycle sequencing reaction were conducted in a final volume of 20 pi containing 13 u.1 25 (150 ng) of the diluted first strand cDNA, 2 p.1 of ABI PRISM Big Dye Terminator Reaction mix (Applied Biosystems, USA), 3 pi of 5 x sequence reaction buffer (Applied Biosystems), and 1.6 ul of one of 5'-Psxl- or 5'-P5x2-specific DS primers (1 uM); the tube containing the reaction mixture was placed in a preheated (94°C) thermal cycler; samples were denatured for 5 min at 94°C and subjected to 40-50 cycles of 10 sec at 94°C, 3 min at 50-60°C, and 4 min at 30 60-65°C. The sequencing products were purified as the following: 1) add 2 p.1 of 3 M sodium acetate (pH 4.6) and 50 p.1 of fresh cold 100% EtOH, 2) keep at -75°C for 30 min, 3) centrifuge for 15-30 min at 13,000 g and remove the supernatant, 4) wash with 200 p.1 of 70% EtOH, 5)<br><br><br>
centrifuge for 15-30 min at 13,000 g and remove the supernatant carefully, and dry. The pellet was re-suspended in 10 JJ.1 of HiDi Formamide just before running the sequencing product into ABI PRISM 3100 Genetic Analyzer.<br>
Surprisingly, the Psx DS primers precisely sequenced their-specific Psx cDNAs. In other<br>
words, /toi-specific DS primer (Psxl-5'-41) sequenced only Psxl cDNA and Ps^-specific<br>
DS primer (Psx2-5'-41) sequenced only Psx2 cDNA (FIG. 8). The 29-bp deletion region in<br>
Psx2 is shown by a black bar. In contrast, the conventional primers (Psxl-5'-40 and Psx2-5'-<br>
40), which do not comply with the principles of DS oligonucleotides, did not distinguish two<br>
Psx cDNAs.<br>
10	These results indicate that DS primers can discriminate single-base mismatch even in<br>
cycle sequencing as well as in PCR amplification.<br>
EXAMPLE 5: Multiplex PCR using Dual Specificity (DS) Oligonucleotides<br>
To demonstrate the application of DS oligonucleotide primers in multiplex PCR, nine 15      different cytokine family genes were amplified with DS primers. The process and results for the multiplex PCR amplification using DS primers are described herein. The cytokine family gene-specific DS primers were designed to generate a 50 bp ladder by using the longest exon sequence of each cytokine gene.<br>
The DS primers for EL-3 used in the Example (200 bp) are: 20      IL3-5'       5'-GCTGCCAGGGGTCTTCATTCimiCTGGATGA-3' (SEQ ID NO:25); and IL3-3'       5'-GGCCATGAGGAACATTCAGAIIIIIGGTGCTCT-3' (SEQ ID NO:26).<br>
The DS primers for EL-15 used in the Example (250 bp) are:<br>
IL15-5'     5'-ATGTAGCAGAATCTGGCTGCmnATGTGAGG-3' (SEQ ID NO:27); and<br>
IL15-3'     5'-ATGTGATCCAAGTGGCTCATIUIICCTTGTTAGG-3' (SEQ ID NO:28).<br>
25	The DS primers for IL-18 used in the Example (300 bp) are:<br>
IL18-5'     5 '-AGGAAATGGATCCACCTGAAniIITGATGATATA-3' (SEQ ID NO:29); and IL18-3'     5'-ATGGAAATACAGGCGAGGTCIiniAAGGCGCA-3' (SEQ ID NO:30).<br>
The DS primers for IL-25 used in the Example (350 bp) are: IL25-5'     5'-AGCTCTCCAAGCTGGTGATCfflHCAAGGCGG-3' (SEQ IDNO:31); and 30      IL25-3'     5'-GAGCTGCCCTGGATGGGGTTITniGTGGTCCT-3'(SEQIDNO:32). The DS primers for IL-2 used in the Example (400 bp) are: IL2-5'       5'-CTCTGACAACACATTTGAGTGCniIICGATGATGAG-3' (SEQ ID NO:33);<br>
45<br><br>
and<br>
IL2-3'       5'-GTGCTGTCCTAAAAATGACAGAimiGAGCTTATTT-3' (SEQ ID NO:34).<br>
The DS primers for EL-6 used in the Example (450 bp) are: IL6-5'       5'-CCAATGCTCTCCTAACAGATAAHinAGTCACAGAA-3' (SEQ ID NO:35); 5      and<br>
IL6-3'       5'-AGGTAAACTTATACATTCCAAGAAAiniITGGCTAGG-3' (SEQ ID NO:36).<br>
The DS primers for IL-19 used in the Example (500 bp) are:<br>
IL19-5'     5'-GTCTCATCTGCTGCCCTTAAIIIIITAGGAGAACT-3' (SEQ ID NO:5); and<br>
IL19-3'     5'-CATAGGCCTGGAAGAAGCCGIiniCAATAAGTTAG-3' (SEQ ID NO:6).<br>
10	The DS primers for EL-lbeta used in the Example (550 bp) are:<br>
ILlb-5'     5'-GGAGAGTGTGGATCCCAAGCIIIIICCAAAGAAG-3' (SEQ ID NO:7); and ILlb-3'     S'-AGACCTCAGTCCAGGCTATGIIIIITTCATCCC-S' (SEQ ID NO:8).<br>
The DS primers for IL-10 used in the Example (600 bp) are: TL10-5'     5'-AAGGCCATGAATGAATTTGAniIITCATCAACTG-3' (SEQ ID NO:37); and 15      IL10-3'     5'-TGACAGTAGGGGAACCCTCTiniIGCTGCAGG-3'(SEQIDNO:38).<br>
A.	Monoplex PCR using one set of cytokine family gene-specific DS primers<br>
The single target PCR amplification for each cytokine family gene was conducted in a final volume of 20 ul containing 2 ul (50 ng) of the mouse genomic DNA, 2 ul of 10 x PCR<br>
20 reaction buffer containing 15 mM MgCl2 (Roche), 2 ul of dNTP (2 mM each dATP, dCTP, dGTP, dTTP), 1 ul of each cytokine family gene-specific 5' DS primer (10 uM), 1 ul of each cytokine family gene-specific 3' DS primer (10 uM), and 0.5 ul of Taq polymerase (5units/ul; Roche); the tube containing the reaction mixture was placed in a preheated (94°C) thermal cycler; samples were denatured for 5 min at 94°C and subjected to 30 cycles of 1 min at 94°C,<br>
25      1 min at 60-65°C, and 1 min at 72°C, followed by a 7-min incubation at 72°C.<br>
B.	Multiplex PCR using 9 sets of cytokine family gene-specific DS primers<br>
The multiplex PCR amplification was conducted in a single tube by using 9 sets of<br>
cytokine family gene-specific DS primers; the reaction mixture was in the final volume of 50 ul<br>
30      containing 100 ng of mouse genomic DNA, 5 ul of 10 x PCR reaction buffer containing 15 mM<br>
MgCl2 (Roche), 5 ul of dNTP (2 mM each dATP, dCTP, dGTP and dTTP), 1 ul of each<br>
cytokine family gene-specific 5' DS primer (0.2-5 uM), 1 JJ.1 of each cytokine family gene-<br>
46<br><br>
specific 3' DS primer (0.5-5 pJVi), and 0.5 ju.1 of Taq polymerase (5 units/ul; Roche); the tube containing the reaction mixture was placed in a preheated (94°C) thermal cycler; the PCR conditions are one cycle of 94°C for 5 min, 50°C for 3 min, and 72°C for 3 min; followed by 29 cycles of 94°C for 40 s, 60°C for 1 min, and 72°C for 40 s, and a 5-min final extension cycle<br>
5      at 72°C.<br>
As shown in FIG. 9, multiplex PCR amplification generates multiple bands which correspond to the expected sizes from 200 bp to 600 bp for 9 different cytokine gene products (FIG. 4, lane 1). Each monoplex PCR amplification generated a single band which corresponds<br>
10 to the expected size of 200 bp for IL-3 (FIG. 4, lane 2), 250 bp for IL-15 (FIG. 4, lane 3), 300 bp for IL-18 (FIG. 4, lane 4), 350 bp for TL-25 (FIG. 4, lane 5), 400 bp for IL-2 (FIG 4, lane 6), 450 bp for TL-6 (FIG. 4, lane 7), 500 bp for EL-19 (FIG. 4, lane 8), 550 bp for IL-lbeta (FIG. 4, lane 9), and 600 bp for IL-10 (FIG. 4, lane 10), respectively.<br>
Accordingly, it could be appreciated that the DS primers developed by the present<br>
15 invention can be successfully applied to multiplex PCR. The unique structure of DS oligonucleotides allows to overcome the common problem of any conventional multiplex PCR, namely primer interference and dimmer formation.<br>
EXAMPLE 6: Detection of Human Metapneumovirus using Mismatch Tolerance of Dual 20      Specificity (DS) Oligonucleotides<br>
To demonstrate the application of DS oligonucleotide primers in mismatch tolerance, DS primers were applied to detect human metapnemovirus (hMPV) in clinical samples. The process and results for the detection of human metapnemovirus using DS primers are described herein. This Example should not be construed as limiting the applications of the invention to<br>
25      the detection of the specific virus.<br>
The DS primers were designed based on the conserved region of the fusion glycoprotein (F) gene generated by aligning sequences of all available hMPV isolates (see Table 1). To tolerate the genetic diversity of these isolates, the primers were designed based on the following criteria: (a) conserved regions should be at least 30 nucleotides in length despite the<br>
30 presence of one or more, yet limited, base pair mismatches (Table 1); (b) most mismatched sequences within the conserved regions are preferably located in the separation portion in the DS primer (e.g., hMPV 5'-585, hMPV 3'-698, and hMPV 3'-1007); (c) otherwise, the<br>
47<br><br>
mismatched nucleotides are located in the 5'-end portion, some of which could be replaced with universal bases such as deoxyinosines (e.g., hMPV 3'-698 and hMPV 3'-1007); and (d) one or two mismatched nucleotide(s) in the 3'-end portion can be replaced with degenerate nucleotide(s) or universal base(s) (e.g., hMPV 3 '-1007 ).<br>
TABLE 1. hMPV-Specific DS Oligonucleotide Primers Based on the Conserved Region of the Fusion Glycoprotein (F) Gene of All Available hMPV Isolates<br><br>
Primer<br>
Virus sequences<br><br>
Sequence<br>
•AGCTTCAGTCAATTCAACAGAAGGTTCCTAAATGTTG' •AGCTTCAGTCAATTCAACAGAAGATTTCTAAATGTTG-•AGCTTCAGTCAATTCAACAGAAGATTCCTAAATGTTG-•AGCTTCAGTCAATTCAACAGAAGATTTCTAAATGTTG-■AGCTTCAGTCAATTCAACAGAAGATTCCTAAATGTTG-■AGCTTCAGTCAATTCAACAGAAGGTTTCTAAATGTTG-<br><br>
No. of Isolates<br>
1 3 4 1 9 14<br><br><br><br>
5'-primer (585):<br><br>
5'-AGCTTCAGTCAATTCAACAGAAIIIIICTAAATGTTG-3'<br><br><br><br>
Virus sequences<br><br>
•AACATCAGTTTTATTTGTCCTGCAGATGTTGGCATGT-■AACATCAGTTTTATCTGCCCTGCAGATGTTGGCATGT-■AACATTAGTTTTATCTGTCCTGCAGATGTTGGCATGT-•AACATCAGTTTTATCTGTCCTGCAGATGTTGGCATGT-•AACATCAATTTTATTTGTCCTGCAGATGTTGGCATGT-•AACATCAATTTTATTTGTCCTGCAGATGTTGGCATGT-■AACATCAATTTTATTTGTCCTGCAGATGTTGGCATGT-<br><br>
4<br>
3<br>
12<br>
2<br>
3<br>
3<br>
5<br><br><br><br>
3'-Primer (698):<br><br>
5'-AACATCAITTTTATITGTCCTGCAIIIIITGGCATGT-3'<br><br>
	TTGACTGCTCAGCAACATTGATTCCTGCTGCTGTGTC	<br>
	TTGATTGCTCAGCAACATTGATCCCTGCTGCTGTGTC	<br>
	TTGATTGCTCAGCAACATTGATCCCTGCAGCTGTGTC	<br>
Virus sequences      J<br>
	TTGATTGCTCAGCGACATTGATCCCTGCTGCTGTGTC	<br>
	TTGATTGTTCAGCAACATTGATCCCTGCTGCTGTGTC	<br>
	TTGATTGCTCAGCAACATTAATTCCTGCTGCTGTGTC	<br>
	TTGATTGCTCAGCAACATTAATTCCCGCTGCTGTGTC	<br>
5'-TTGAITGCTCAGCIACATTGATIIIIICWGCTGTGTC-3'<br>
3'-Primer (1007):<br>
* Genetic diversity between hMPV isolates is shown as underlined nucleotides. The DS primers specific to hMPV F gene used in the Example are: hMPV    5'-585    5'-AGCTTCAGTCAATTCAACAGAAIIinCTAAATGTTG-3'    (SEQ    ID NO:39);<br>
48<br><br>
hMPV 3 '-698    5 '-AACATCAITTTTATITGTCCTGCAiniITGGCATGT-3' (SEQ ID NO:40); and<br>
hMPV   3'-1007       5'-TTGAITGCTCAGCIACATTGATfflIICWGCTGTGTC-3'   (SEQ   ID NO:41), 5      wherein W can be A or T,   and I is deoxyinosine.<br>
Viral total RNA was extracted by using the RNAzol B method according to the manufacturer's protocol (RNAzol LS; Tel-Test, Inc.). Reverse transcription reaction was performed to synthesize cDNA using the viral RNA for 1.5 hr at 42 °C in a reaction volume of<br>
10 20 pi composed of the following: 5 pi of total RNA (approximately 100 ng), 4 ul of 5 x reaction buffer (Invitrogen, USA), 5 pi of dNTPs (each 5 mM), 2 JLII of 10 pM random hexadeoxynuclotides, 0.5 ul of RNase inhibitor (40 units/pl, Promega), and 1 pi of Moloney murine leukemia virus reverse transcriptase (200 units/pl, Promega).<br>
The target PCR amplification of hMPV F gene was conducted in a final volume of 20 pi<br>
15 containing 2 p.1 (30 ng) of the first strand cDNA, 2 pi of 10 x PCR reaction buffer containing 15 mM MgCl2 (Roche), 2 pi of dNTP (2 mM each dATP, dCTP, dGTP and dTTP), 1 pi of 5' hMPV-specific DS primer (hMPV 5'-585; 10 pM), 1 pi of 3' hMPV-specific DS primer (hMPV 3'-698 or hMPV 3M007; 10 pM), and 0.5 pi of Taq polymerase (5 units/pl; Roche); the tube containing the reaction mixture was placed in a preheated (94°C) thermal cycler;<br>
20 samples were denatured for 5 min at 94°C and subjected to 30 cycles of 1 min at 94°C, 1 min at 60-65°C, and 1 min at 72°C, followed by a 7-min incubation at 72°C.<br>
As shown in FIG. 10B, each pair of hMPV-specific DS primers (hMPV 5'-585 and hMPV 3'-698, and hMPV 5'-585 and hMPV 3'-1007) generates a single band which corresponds to the expected sizes of 150-bp and 459-bp, respectively (lanes 1 and 2). Subsequent sequence<br>
25 analysis of the products confirmed that they are the partial sequences of the hMPV fusion glycoprotein (F) gene. In contrast, these primers generate no product in a negative control PCR without the template (lanes 4 and 5). As a positive control, human bcta-actin-specific primer set was used (lane 3).<br>
These results indicate that the DS primers can be applied to detect hMPVs from patient<br>
30 with respiratory infections. Thus, it could be realized that the DS oligonucleotides can be adapted, as primers in PCR amplification or probes in oligonucleotide chip, to all potential situations encountered in the field.<br>
49<br><br>
EXAMPLE 7: Detection of Target Nucleotide Sequence Using DS Oligo-Immobilized Microarray<br>
The DS oligos complementary to a region of a target nucleic acid molecule are synthesized 5 by means of a DNA synthesizer (Expedite 8900 Nucleic Acid Synthesis System, Applied Biosystems (ABI)) according to a standard protocol. The synthesized DS oligos are immobilized on a glass slide for microarray. Then, a template-dependent extension reaction mixture containing 50-200 ng of DNA sample, 5 p.1 of 10 x PCR reaction buffer (Promega), 5 ul of 15 mM MgCl2, 5 ul of fluorescence-labeled dNTP (2 mM each dATP, dCTP, dGTP and<br>
10 dTTP) and 0.5 u.1 of Taq polymerase (5 units/jil; Promega) is added to the microarray, after which the microarray is placed in a preheated (94°C) thermal cycler. The template-dependent extension reaction is conducted according to the following thermal cycle: denaturation for 5 min at 94°C, and 15-50 cycles of 1 min at 94°C, 1-3 min at 50-65°C and 1-4 min at 60-72°C, followed by a 5-min extension at 72°C. Following the template-dependent extension reaction,<br>
15 the extended DS oligos are washed and detected by its fluorescent images through a microarray scanner, followed by analysis of the images.<br>
EXAMPLE 8: SNP (Single Nucleotide Polymorphism) Genotyping Using DS Oligonucleotides<br>
20	The DS oligos are synthesized by means of a DNA synthesizer (Expedite 8900 Nucleic<br>
Acid Synthesis System, Applied Biosystems (ABI)) according to a standard protocol with placing a polymorphic base (interrogation site) in the center of 3'-low Tm specificity portion. The synthesized oligonucleotides are immobilized on a glass slide for microarray. Then, a template-dependent extension reaction mixture containing 50-200 ng of DNA sample, 5 u.1 of<br>
25 10 x PCR reaction buffer (Promega), 5 ul of 15 mM MgCl2, 5 p.1 of fluorescence-labeled dNTP (2 mM each dATP, dCTP, dGTP and dTTP) and 0.5 u.1 of Taq polymerase (5 units/ul; Promega) is added to the microarray, after which die microarray is placed in a preheated (94°C) thermal cycler. The template-dependent extension reaction is conducted according to the following thermal cycle: denaturation for 5 min at 94°C, and 15-50 cycles of 1 min at 94°C, 1-3 min at<br>
30 50-65°C and 1-4 min at 60-72°C„ followed by a 5-min extension at 72°C. Following the template-dependent extension reaction, the extended DS oligos are washed and detected by its fluorescent images through a microarray scanner, followed by analysis of the images.<br>
50<br><br>
Having described a preferred embodiment of the present invention, it is to be understood that variants and modifications thereof falling within the spirit of the invention may become apparent to those skilled in this art, and the scope of this invention is to be determined by 5      appended claims and their equivalents.<br>
51<br><br>
What is claimed is:<br>
1.    A method for synthesizing a nucleic acid molecule using a dual specificity oligonucleotide by a template-dependent extension reaction, which comprises the steps of:<br>
(a)	annealing the dual specificity oligonucleotide to a template nucleic acid molecule,<br>
                    wherein the dual specificity oligonucleotide has three portions in which a 5'-high Tm<br>
specificity portion has a hybridizing nucleotide sequence substantially complementary to a site on the template nucleic acid to hybridize therewith, a separation portion comprises at least two universal bases and a 3'-low Tm specificity portion has a hybridizing nucleotide sequence substantially complementary to a site on the template<br>
	nucleic acid to hybridize therewith; Tm of the 5'-high Tm specificity portion is higher<br>
than that of the 3'-low Tm specificity portion, the separation portion has the lowest Tm in the three portions; the separation portion forms a non base-pairing bubble structure under conditions that the 5'-high Tm specificity portion and the 3'-low Tm specificity portion are annealed to the template nucleic acid, wherein the annealing is performed<br>
	under conditions that annealing solely by the 3'-low Tm specificity portion does not<br>
occur; and<br>
(b)	extending the dual  specificity oligonucleotide to synthesize the nucleic acid<br>
molecule complementary to the template nucleic acid.<br>
     2. A method for selectively amplifying a target nucleic acid sequence from a DNA or a mixture of nucleic acids, which comprises amplifying the target nucleic acid sequence by performing at least two cycles of primer annealing, primer extending and denaturing, using a primer set which comprises at least one dual specificity oligonucleotide; wherein the dual specificity oligonucleotide has three portions in which a 5'-high Tm specificity portion has a<br>
           hybridizing nucleotide sequence substantially complementary to a site on the target nucleic acid to hybridize therewith, a separation portion comprises at least two universal bases and a 3'-low Tm specificity portion has a hybridizing nucleotide sequence substantially complementary to a site on the target nucleic acid to hybridize therewith, Tm of the 5'-high Tm specificity portion is higher than that of the 3'-low Tm specificity portion, the separation portion has the lowest Tm in<br>
          the three portions; the separation portion forms a non base-pairing bubble structure under conditions that the 5'-high Tn, specificity portion and the 3'-low Tm specificity portion are annealed to the target nucleic acid; wherein annealing in the amplification reaction is performed<br><br><br>
under conditions that annealing solely by the 3'-low Tm specificity portion does not occur.<br>
3. A method for amplifying two or more target nucleotide sequences simultaneously using two or more pairs of primers in the same reaction, which comprises amplifying the target  nucleotide sequences by performing at least two cycles of primer annealing, primer extending and denaturing, using two or more primer pairs in which at least one primer of the primer pair is a dual specificity oligonucleotide, characterized in that the dual specificity oligonucleotides have three portions in which a 5'-high Tm specificity portion has a hybridizing nucleotide sequence substantially complementary to a site on the target nucleotide sequence to hybridize<br>
therewith, a separation portion comprises at least two universal bases and a 3'-low Tm specificity portion has a hybridizing nucleotide sequence substantially complementary to a site on the target nucleotide sequence to hybridize therewith, Tm of the 5'-high Tm specificity portion is higher than that of the 3'-low Tm specificity portion, the separation portion has the lowest Tm in the three portions; the separation portion forms a non base-pairing bubble<br>
structure under conditions that the 5'-high Tm specificity portion and the 3'-low Tm specificity portion are annealed to the target nucleotide sequence; wherein annealing in the amplification reaction is performed under conditions that annealing solely by the 3'-low Tm specificity portion does not occur.<br>
   4. A method for sequencing a target nucleic acid molecule using a dual specificity oligonucleotide from a DNA or a mixture of nucleic acids, which comprises the steps of:<br>
(a) synthesizing a complementary nucleic acid molecule to the nucleic acid molecule to be sequenced by performing at least two cycles of primer annealing, primer extending and denaturing, using the dual specificity oligonucleotide as a sequencing primer;<br>
	wherein the dual specificity oligonucleotide has three portions in which a 5'-high Tm<br>
specificity portion has a hybridizing nucleotide sequence substantially complementary to a site on the the target nucleic acid molecule to hybridize therewith, a separation portion comprises at least two universal bases and a 3 '-low Tm specificity portion has a hybridizing nucleotide sequence substantially complementary to a site on the target<br>
	nucleic acid molecule to hybridize therewith, Tm of the 5'-high Tm specificity portion is<br>
higher than that of the 3'-low Tm specificity portion, the separation portion has the lowest Tra in the three portions; the separation portion forms a non base-pairing bubble<br><br><br>
structure under conditions that the 5'-high Tm specificity portion and the 3'-low Tm<br>
specificity portion are annealed to the target nucleic acid molecule; wherein annealing<br>
in the synthesizing reaction is performed under conditions that annealing solely by the<br>
3 '-low Tra specificity portion does not occur; and<br>
(b)   determining a nucleotide sequence of the synthesized complementary nucleic acid<br>
molecule.<br>
5.   A method for detecting a nucleic acid molecule with genetic diversity by a template-dependent extension reaction of a dual specificity oligonucleotide, which comprises the steps       of:<br>
(a)   annealing a dual specificity oligonucleotide to a template nucleic acid molecule,<br>
wherein the dual specificity oligonucleotide has three portions in which a 5'-high Tm<br>
specificity portion has a hybridizing nucleotide sequence substantially complementary<br>
                 to a site on the template nucleic acid to hybridize therewith, a separation portion<br>
          comprises at least two universal bases and a 3'-low Tm specificity portion has a<br>
hybridizing nucleotide sequence substantially complementary to a site on the template<br>
nucleic acid to hybridize therewith, Tm of the 5'-high Tm specificity portion is higher<br>
than that of the 3 '-low Tm specificity portion, the separation portion has the lowest Tm in<br>
the three portions; the separation portion forms a non base-pairing bubble structure<br>
	under conditions that the 5'-high Tm specificity portion and the 3'-low Tm specificity<br>
portion are annealed to the template nucleic acid, wherein the annealing is performed<br>
under conditions that annealing solely by the 3'-low Tm specificity portion does not<br>
occur and annealing occurs when the 5'-high Tm specificity portion and/or the 3'-low Tm<br>
specificity portion has one or more mismatched bases to its target site;<br>
	(b)    extending the dual specificity oligonucleotide to synthesize the nucleic acid<br>
molecule complementary to the template; and .<br>
(c)    detecting the occurrence of the template-dependent extension of the dual specificity oligonucleotide.<br>
      6. A method for detecting a target nucleotide sequence in a nucleic acid sample by a template-dependent extension reaction of a dual specificity oligonucleotide, which comprises the steps of:<br><br><br>
(a)	extending a dual specificity oligonucleotide as a probe immobilized on a substrate<br>
comprising at least one cycle of a hybridization, a template-dependent extension and a<br>
denaturation, wherein the hybridization is performed by contacting the dual specificity<br>
oligonucleotide to the nucleic acid sample, wherein the dual specificity oligonucleotide<br>
	has three portions in which a 5'-high Tm specificity portion has a hybridizing nucleotide<br>
sequence substantially complementary to a site on the target nucleotide sequence to hybridize therewith, a separation portion comprises at least two universal bases and a 3'-low Tm specificity portion has a hybridizing nucleotide sequence substantially complementary to a site on the target nucleotide sequence to hybridize therewith, Tm of<br>
	the 5'-high Tm specificity portion is higher than that of the 3'-low Tm specificity portion,<br>
the separation portion has the lowest Tm in the three portions; the separation portion forms a non base-pairing bubble structure under conditions that the 5'-high Tm specificity portion and the 3'-low Tm specificity portion are annealed to the target nucleotide sequence, wherein the hybridization is performed under conditions that hybridization<br>
1	solely by the 3 '-low Tm specificity portion does not occur; and<br>
(b)	analyzing the occurrence of the template-dependent extension.<br>
7. The method according to any one of claims 1-6, wherein the universal base in the separation portion is selected from the group consisting of deoxyinosine, inosine, 7-deaza-2'-<br>
         deoxyinosine, 2-aza-2'-deoxyinosine, 2'-OMe inosine, 2'-F inosine, deoxy 3-nitropyrrole, 3-nitropyrrole, 2'-OMe 3-nitropyrrole, 2'-F 3-nitropyrrole, l-(2'-deoxy-beta-D-ribofuranosyl)-3-nitropyrrole, deoxy 5-nitroindole, 5-nitroindole, 2'-OMe 5-nitroindole, 2'-F 5-nitroindole, deoxy 4-nitrobenzimidazole, 4-nitrobenzimidazole, deoxy 4-aminobenzimidazole, 4-aminobenzimidazole, deoxy nebularine, 2'-F nebularine, 2'-F 4-nitrobenzimidazole, PNA-5-<br>
          introindole, PNA-nebularine, PNA-inosine, PNA-4-nitrobenzimidazole, PNA-3-nitropyrrole,<br>
morpholino-5-nitroindole, morpholino-nebularine, morpholino-inosine, morpholino-4-<br>
nitrobenzimidazole,	morpholino-3-nitropyrrole,	phosphoramidatc-5-nitroindole,<br>
phosphoramidate-nebularine,	phosphoramidate-inosine,	phosphoramidate-4-<br>
nitrobenzimidazole,     phosphoramidate-3-nitropyrrole,     2'-0-methoxyethyl     inosine,     2'0-<br>
          methoxyethyl nebularine, 2'-0-methoxyethyl 5-nitroindole, 2'-0-methoxyethyl 4-nitrobenzimidazole, 2'-0-methoxyethyl 3-nitropyrrole, and combinations thereof.<br><br><br>
8.	The method according to claim 7, wherein the universal base is deoxyinosine, l-(2'-deoxy-<br>
beta-D-ribofuranosyl)-3-nitropyrrole or 5-nitroindole.<br>
9.	The method according to claim 8, wherein the universal base is deoxyinosine.<br><br>
10.	The method according to any one of claims 1-6, wherein the separation portion comprises<br>
contiguous nucleotides having universal bases.<br>
11.	The method according to any one of claims 1-6, wherein the 5'-high Tm specificity portion<br>
           is longer than the 3 '-low Tm specificity portion.<br>
12.	The method according to any one of claims 1-6, wherein the 3'-low Tm specificity portion<br>
has the hybridizing nucleotide sequence perfectly complementary to the site on the template<br>
nucleic acid to hybridize therewith.<br><br>
13.	The method according to any one of claims 1-6, wherein the 5'-high Tm specificity portion<br>
is 15 to 40 nucleotides in length.<br>
14.	The method according to claim 13, wherein the 5'-high Tra specificity portion is 15 to 25<br>
          nucleotides in length.<br>
15.	The method according to any one of claims 1-6, wherein the separation portion is at least<br>
3 nucleotides in length.<br>
         16.    The method according to any one of claims 1-6, wherein the separation portion is 3 to 10 nucleotides in length.<br>
17.	The method according to any one of claims 1-6, wherein the 3'-low Tm specificity portion<br>
is 3 to 15 nucleotides in length.<br><br>
18.	The method according to any one of claims 1-6, wherein the 5'-high Tm specificity portion<br>
is 15 to 25 nucleotides in length, the separation portion is 3 to 10 nucleotides in length, and the<br><br><br>
3 '-low Tm specificity portion is 3 to 15 nucleotides in length.<br>
19.	The method according to any one of claims 1-6, wherein the Tra of the 5'-high Tm<br>
specificity portion ranges from 40°C to 80°C.<br><br>
20.	The method according to any one of claims 1-6, wherein the Tm of the 3'-low Tm<br>
specificity portion ranges from 10°C to 40°C.<br>
21.	The method according to any one of claims 1-6, wherein the Tm of the separation portion<br>
           ranges from 3°C to 15°C.<br>
22.	The method according to any one of claims 1-6, wherein the annealing is performed at<br>
temperature ranging from 45°C to 68°C.<br>
           23. The method according to claim 1 or 5, wherein the synthesis of a target nucleic acid sequence is achieved by repeating the process of the template-dependent extension reaction in which the dual specificity oligonucleotide annealing and extension steps are followed by denaturation step.<br>
          24. The method according to claim 2 or 3, wherein the amplification is performed in accordance with polymerase chain reaction.<br>
25.	The method according to claim 4, wherein the target nucleic acid molecule to be<br>
sequenced is contained in a genomic DNA or a population of cDNAs.<br><br>
26.	The method according to claim 5, wherein the nucleic acid molecule with genetic<br>
diversity is a nucleic acid of a virus exhibiting genetic diversity.<br>
27.	A dual specificity oligonucleotide for synthesizing a nucleic acid molecule by a template-<br>
                dependent extension reaction, which is represented by the following general formula:<br>
5 -Xp-Yq-Zr-3 wherein, Xp represents a 5'-high Tra specificity portion having a hybridizing nucleotide<br><br><br>
sequence substantially complementary to a site on a template nucleic acid to hybridize therewith, Yq represents a separation portion comprising at least two universal bases, Zr represents a 3'-low Tm specificity portion having a hybridizing nucleotide sequence substantially complementary to a site on the template nucleic acid to hybridize therewith, p, q 5 and r represent the number of nucleotides, and X, Y, and Z are deoxyribonucleotide or ribonucleotide; the Tm of the 5'-high Tm specificity portion is higher than that of the 3'-low Tm specificity portion, the separation portion has the lowest Tm in the three portions; the separation portion forms a non base-pairing bubble structure under conditions that the 5'-high Tm specificity portion and the 3'-low Tra specificity portion are annealed to the template nucleic<br>
          acid, enabling the 5'-high Tm specificity portion to separate from the 3'-low Tm specificity portion in terms of annealing specificity to the template nucleic acid, whereby the annealing specificity of the oligonucleotide is determined dually by the 5'-high Tm specificity portion and the 3'-low Tm specificity portion such that the overall annealing specificity of the oligonucleotide is enhanced.<br><br>
28.	The dual specificity oligonucleotide according to claim 27, wherein the universal base in<br>
the separation portion is selected from the group consisting of deoxyinosine, inosine, 7-deaza-<br>
2'-deoxyinosine, 2-aza-2'-deoxyinosine, 2'-OMe inosine, 2'-F inosine, deoxy 3-nitropyrrole, 3-<br>
nitropyrrole, 2'-OMe 3-nitropyrrole, 2'-F 3-nitropyrrole, l-(2'-deoxy-beta-D-ribofuranosyl)-3-<br>
           nitropyrrole, deoxy 5-nitroindole, 5-nitroindole, 2'-OMe 5-nitroindole, 2'-F 5-nitroindole, deoxy 4-nitrobenzimidazole, 4-nitrobenzimidazole, deoxy 4-aminobenzimidazole, 4-aminobenzimidazole, deoxy nebularine, 2'-F nebularine, 2'-F 4-nitrobenzimidazole, PNA-5-introindole, PNA-nebularine, PNA-inosine, PNA-4-nitrobenzimidazole, PNA-3-nitropyrrole, morpholino-5-nitroindole,     morpholino-nebularine,     morpholino-inosine,     morpholino-4-<br>
           nitrobenzimidazole,	morpholino-3-nitropyrrole,	phosphoramidate-5-nitroindole,<br>
phosphoramidate-nebularine,	phosphoramidate-inosine,	phosphoramidate-4-<br>
nitrobenzimidazole, phosphoramidate-3 -nitropyrrole, 2'-0-methoxyethyl inosine, 2'0-methoxyethyl nebularine, 2'-0-methoxyethyl 5-nitroindole, 2'-0-methoxyethyl 4-nitrobenzimidazole, 2'-0-methoxyethyl 3-nitropyrrole, and combinations thereof.<br><br>
29.	The dual specificity oligonucleotide according to claim 28, wherein the universal base is<br>
deoxyinosine, l-(2'-deoxy-beta-D-ribofuranosyl)-3-nitropyrrole or 5-nitroindole.<br><br><br>
30. The dual specificity oligonucleotide according to claim 29, wherein the universal base is deoxyinosine.<br>
        31.    The dual specificity oligonucleotide according to claim 27, wherein the separation portion comprises contiguous nucleotides having universal bases.<br>
32.	The dual specificity oligonucleotide according to claim 27, wherein the 5'-high Tm<br>
specificity portion is longer than the 3'-low Tm specificity portion.<br><br>
33.	The dual specificity oligonucleotide according to claim 27, wherein the 3'-low Tm<br>
specificity portion has the hybridizing nucleotide sequence perfectly complementary to the site<br>
on the template nucleic acid to hybridize therewith.<br>
         34.    The dual specificity oligonucleotide according to claim 27, wherein p represents an integer of 15 to 40.<br>
35.	The dual specificity oligonucleotide according to claim 34, wherein p represents an<br>
integer of 15 to 25.<br><br>
36.	The dual specificity oligonucleotide according to claim 27, wherein q is at least 3.<br>
37.	The dual specificity oligonucleotide according to claim 27, wherein q represents an integer of 3 to 10.<br><br>
38.	The dual specificity oligonucleotide according to claim 27, wherein r represents an<br>
integer of 3 to 15.<br>
39.	The dual specificity oligonucleotide according to claim 27, wherein p is an integer of 15       to 25, q is an integer of 3 to 10, and r is an integer of 3 to 15.<br>
40.	The dual specificity oligonucleotide according to claim 27, wherein the Tmof the 5'-high<br><br><br>
Tm specificity portion ranges from 400 C to 80°C.<br>
41.	The dual specificity oligonucleotide according to claim 27, wherein the Tm of the 3'-low<br>
Tm specificity portion ranges from 10°C to 40°C.<br><br>
42.	The dual specificity oligonucleotide according to claim 27, wherein the Tm of the<br>
separation portion ranges from 3°C to 15°C.<br>
43.	A method for enabling an annealing specificity of an oligonucleoide to be dually<br>
           determined through a structure of the oligonucleoide, which comprises the steps of:<br>
(a)	selecting a target nucleic acid sequence;<br>
(b)	designing a sequence of an oligonucleotide having (i) a hybridizing sequence<br>
substantially complementary to the target nucleic acid and (ii) a separation portion<br>
comprising at least two universal bases, such that the separation portion intervenes in the<br>
 	hybridizing sequence to form three portions in the oligonucleotide; and<br>
(c)	determining the position of the separation portion in the oligonucleotide to allow a<br>
portion at the 5'-direction of the separation portion to have a higher Tm than a portion at the<br>
3'-direction of the separation portion and to allow the separation portion to have the lowest<br>
Tm in the three portions, thereby providing an oligonucleotide having three distinct<br>
 	portions with different Tm values from one another in which (i) a 5'-high Tm specificity<br>
portion of the oligonucleotide has a hybridizing nucleotide sequence substantially complementary to the target nucleic acid, (ii) a 3'-low Tm specificity portion of the oligonucleotide has a hybridizing nucleotide sequence substantially complementary to the target nucleic acid; and (iii) the separation portion of the oligonucleotide between the 5'-<br>
 	high Tm specificity portion and 3'-low Tm specificity portion comprises at least two<br>
universal bases; and the Tm of the 5'-high Tm specificity portion is higher than that of the 3'-low Tm specificity portion and the separation portion has the lowest Tm in the three portions,<br>
whereby the annealing specificity of the oligonucleotide to the target nucleic acid is<br>
 	determined dually by both the 5'-high Tm specificity portion and the 3'-low Tm<br>
specificity portion.<br><br><br>
44.	The method according to claim 43, wherein the universal base in the separation portion is<br>
selected from the group consisting of deoxyinosine, inosine, 7-deaza-2'-deoxyinosine, 2-aza-<br>
2'-deoxyinosine, 2'-OMe inosine, 2'-F inosine, deoxy 3-nitropyrrole, 3-nitropyrrole, 2'-OMe 3-<br>
nitropyrrole, 2'-F 3-nitropyrrole, l-(2'-deoxy-beta-D-ribofuranosyl)-3-nitropyrrole, deoxy 5-<br>
       nitroindole, 5-nitroindole, 2'-OMe 5-nitroindole, 2'-F 5-nitroindole, deoxy 4-nitrobenzimidazole, 4-nitrobenzimidazole, deoxy 4-aminobenzimidazole, 4-aminobenzimidazole, deoxy nebularine, 2'-F nebularine, 2'-F 4-nitrobenzimidazole, PNA-5-introindole, PNA-nebularine, PNA-inosine, PNA-4-nitrobenzimidazole, PNA-3-nitropyrrole, morpholino-5-nitroindole,     morpholino-nebularine,     morpholino-inosine,     morpholino-4-<br>
          nitrobenzimidazole,	morpholino-3-nitropyrroIe,	phosphoramidate-5-nitroindole,<br>
phosphoramidate-nebularine,	phosphoramidate-inosine,	phosphoramidate-4-<br>
nitrobenzimidazole, phosphoramidate-3-nitropyrrole, 2'-0-methoxyethyI inosine, 2'0-methoxyethyl nebularine, 2'-0-methoxyelhyl 5-nitroindole, 2'-0-methoxyethyl 4-nitrobenzimidazole, 2'-0-methoxyethyl 3-nitropyrrole, and combinations thereof.<br><br>
45.	The method according to claim 44, wherein the universal base is deoxyinosine, l-(2'-<br>
deoxy-beta-D-ribofuranosyl)-3-nitropyrrolc or 5-nitroindole.<br>
46.	The method according to claim 45, wherein the universal base is deoxyinosine.<br><br>
47.	The method according to claim 43, wherein the separation portion comprises contiguous<br>
nucleotides having universal bases.<br>
48.	The method according to claim 43, wherein the 5'-high Tm specificity portion is longer<br>
          than the 3'-low Tra specificity portion.<br>
49.	The method according to claim 43, wherein the 3'-low Tm specificity portion has the<br>
hybridizing nucleotide sequence perfectly complementary to the site on the template nucleic<br>
acid to hybridize therewith.<br><br>
50.	The method according to claim 43, wherein p represents an integer of 15 to 40.<br><br><br>
51.	The method according to claim 50, wherein p represents an integer of 15 to 25.<br>
52.	The method according to claim 43, wherein q is at least 3.<br>
       53.	The method according to claim 43, wherein q represents an integer of 3 to 10.<br>
54.	The method according to claim 43, wherein r represents an integer of 3 to 15.<br>
55.	The method according to claim 43, wherein p is an integer of 15 to 25, q is an integer of 3       to 10, and r is an integer of 3 to 15.<br>
56.	The method according to claim 43, wherein the Tm of the 5'-high Tra specificity portion<br>
ranges from 40°C to 80°C.<br>
       57.    The method according to claim 43, wherein the Tmof the 3'-low Tm specificity portion ranges from 10°C to 40°C.<br>
58.	The method according to claim 43, wherein the Tmof the separation portion ranges from<br>
3°C to l5° C.<br><br>
59.	The method according to claim 43, wherein the template-dependent extension reaction is<br>
a polymerase chain reaction.<br><br><br>
Abstract: -<br>
The present invention relates to various processes by a template-dependent extension reaction using a dual specificity oligonucleotide and a dual specificity oligonucleotide composed of three different Tm portions therefor. Demonstrated in the present invention are the features of the dual specificity oligonucleotide, which are high hybridization specificity and mismatch tolerance.<br><br></td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/viewdoc.aspx?id=uWOJIfbqV1ZBWEJuT0btVQ==&amp;amp;loc=vsnutRQWHdTHa1EUofPtPQ==" target="_blank" style="word-wrap:break-word;">http://ipindiaonline.gov.in/patentsearch/GrantedSearch/viewdoc.aspx?id=uWOJIfbqV1ZBWEJuT0btVQ==&amp;amp;loc=vsnutRQWHdTHa1EUofPtPQ==</a></p>
		<br>
		<div class="pull-left">
			<a href="272071-method-for-preparing-tetrapyrrole-compounds-and-tetrapyrrole-compounds.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="272073-promoter.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>272072</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>1621/MUMNP/2007</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>12/2016</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>18-Mar-2016</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>16-Mar-2016</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>04-Oct-2007</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>SEEGENE INC.</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td>8, 9F, TAEWON BLDG., 65-5, BANGYI-DONG, SONGPA-GU, SEOUL 138-050.</td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>CHUN JONG-YOON</td>
											<td>319-1301 OLYMPIC SEONSUGIJACHON APARTMENT, 89, BANGI-DONG, SONGPA-GU, SEOUL 138-050.</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>C12N15/11 C12Q1/68</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>PCT/KR2006/00746</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td>2006-03-03</td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td>PCT/KR2005/001206</td>
									<td>2005-04-26</td>
								    <td>U.S.A.</td>
								</tr>
								<tr>
									<td>2</td>
									<td>10-2005-0018419</td>
									<td>2005-03-05</td>
								    <td>U.S.A.</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/272072-processes-using-dual-specificity-oligonucleotide-and-dual-specificity-oligonucleotide-and-dual-specificity-oligonucleotide by HTTrack Website Copier/3.x [XR&CO'2014], Thu, 04 Apr 2024 23:36:05 GMT -->
</html>
