<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/272180-plants-having-improved-growth-characteristics-and-methods-for-making-the-same by HTTrack Website Copier/3.x [XR&CO'2014], Thu, 04 Apr 2024 23:01:56 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 272180:PLANTS HAVING IMPROVED GROWTH CHARACTERISTICS AND METHODS FOR MAKING THE SAME</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">PLANTS HAVING IMPROVED GROWTH CHARACTERISTICS AND METHODS FOR MAKING THE SAME</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>Plants having improved growth characteristics and methods for making the same The present invention relates generally to the field of molecular biology and concerns a method for improving various plant growth characteristics by modulating expression in a plant of a nucleic acid encoding a GRP (Growth-Related Protein). The present invention also concerns plants having modulated expression of a nucleic acid encoding a GRP, which plants have improved growth charactenstics relative to corresponding wild type plants or other control plants. The invention also provides constructs useful in the methods of the invention. The GRP may be one of the following: Seed Yield Regulator (SYR), FG-GAP, CYP90B, CDC27, AT-hook transcription factors, DOF transcnption factors and Cyclin Dependent Kinase Inhibitors (CKIs).</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td>Plants having improved growth characteristics and methods for<br>
making the same<br>
The present invention relates generally to the field of molecular biology and concerns a method for improving various plant growth characteristics by modulating expression in a plant of a nucleic acid encoding a GRP (Growth-Reiated Protein). The present invention also concerns plants having modulated expression of a nucleic acid encoding a GRP, which plants have improved growth characteristics relative to corresponding wild type plants or other control plants. The invention also provides constructs useful in the methods of the invention.<br>
Given the ever-increasing world population, and the dwindling area of land available for agriculture, it remains a major goal of research to improve the efficiency of agriculture and to increase the diversity of plants in horticulture. Conventional means for crop and horticultural improvements utilise selective breeding techniques to identify plants having desirable characteristics. However, such selective breeding techniques have several drawbacks, namely that these techniques are typically labour intensive and result in plants that often contain heterogeneous genetic complements that may not always result in the desirable trait being passed on from parent plants. Advances-in molecular biology have allowed mankind to manipulate the germplasm of animals and plants. Genetic engineering of plants entails the isolation and manipulation of genetic material (typically in the form of DNA or RNA) and the subsequent introduction of that genetic material into a plant. Such technology has led to the development of plants having various improved economic, agronomic or horticultural traits. Traits of particular economic interest are growth characteristics such as high yield. Yield is normally defined as the measurable produce of economic value from a crop. This may be defined in terrns of quantity and/or quality. Yield is directly dependent on several factors, for example, the number and size of the-organs, plant architecture (for example, the number of branches), seed production and more. Root development, nutrient uptake and stress tolerance may also be important factors in determining yield.<br>
Seed yield is a particulariy important trait, since the seeds of many plants are important for human and animal nutrition. Crops such as, corn, rice, wheat, canola and soybean account for over half the total human caloric intake, whether through direct consumption of the seeds themselves or-through consumption of meat products raised on processed seeds. They are also a source of sugars, oils and many kinds of metabolites used in industrial processes. Seeds contain an embryo (the source of new shoots and roots) and an endosperm (the source of nutrients for embryo growth during germination and during eariy growth of seedlings). The development of a seed involves many genes, and requires the transfer of metabolites from the<br>
2<br><br>
roots, leaves and stems into the growing seed. The endosperm, in particular, assimilates the metabolic precursors of carbohydrates, oils and proteins and synthesizes them into storage macromolecules to fill out the grain.<br>
Another important trait for many crops is early vigour. Improving early vigour is an important objective of modern rice breeding programs in both temperate and tropical rice cultivars. Long roots are important for proper soil anchorage in water-seeded hce. Where rice is sown directly into flooded fields, and where plants must emerge rapidly through water, longer shoots are associated with vigour. Where drill-seeding is practiced, longer mesocotyls and coleoptiles are important for good seedling emergence. Early vigour may also result from increased plant fitness due to, for example, the plants being better adapted to their environment (i.e. being more able to cope with various abiotic or biotic stress factors). Plants having early vigour also show better establishment of the crop (with the crop growing in a more uniform manner, i.e. with the majority of plants reaching the various stages of development at substantially the same time), and show better growth and often better yield.<br>
A further important trait is that of improved abiotic stress tolerance. Abiotic stress is a primary cause of crop loss worldwide, reducing average yields for most major crop plants by more than 50% (Wang et al., Planta (2003) 218: 1-14). Abiotic stresses may be caused by drought, salinity, extremes of temperature, chemical toxicity and oxidative stress. The ability to improve plant tolerance to abiotic stress would be of great economic advantage to farmers woridwide and would allow for the cultivation of crops during adverse conditions and in territories where cultivation of crops may not otherwise be possible.<br>
Crop yield may therefore be increased by optimising one of the above-mentioned factors.<br>
Depending on the end use, the modification of certain yield traits may be favoured over others. For example for applications such as forage or wood production, or bio-fuel resource, an increase in the leafy parts of a plant may be desirable, and for applications such as flour, starch or oil production, an increase in seed parameters may be particularly desirable. Even amongst the seed parameters, some may be favoured over others, depending on the application. Various mechanisms may contribute to increasing seed yield, whether that is in the form of increased seed size or increased seed number.<br>
One approach to increasing (seed) yield in plants may be through modification of the inherent grov/th mechanisms of a plant. One such mechanism is the cell cycle.<br><br>
!t has now been found that various growth characteristics may be improved in plants by modulating expression in a plant of a nucleic acid encoding a GRP (Growth-Related Protein) in a plant. The GRP may be one of the following: Seed Yield Regulator (SYR), FG-GAP, CYP90B, CDC27, AT-hook transcription factors, DOF transcription factors and Cyciin Dependent Kinase Inhibitors (CKis),<br>
BACKGROUND<br>
Seed Yieid Regulator (SYR)<br>
There is a continuous need to find new seed yield enhancement genes and several approaches have been used so far, for example through manipulation of plant hormone levels (WO 03/050287), through manipulation of the cell cycle (WO 2005/051702), through manipulation of genes involved in salt stress response (WO 2004/058980) amongst other<br>
strategies.<br>
SYR is a new protein that has hitherto not been characterised. SYR shows some homology (around 48% sequence identity on the DNA level, around 45% on the protein level) to an Arabidopsis protein named ARGOS (Hu et al., Plant Cell 15, 1951-19.61, 2003; US 2005/0108793). Hu et a!, postulated that ARGOS is a protein of unique function and is encoded by a single gene. The major phenotypes of ARGOS overexpression in Arabidopsis are increased leafy biomass and delayed flowering.<br>
FG-GAP<br>
FG-GAP proteins are putative transmembrane proteins. They are characterised by the presence of one or more FG-GAP domains (Pfam accession number PF01839) and by the presence of an N-terminal signal peptide and a transmembrane domain in the C-terminal half of the protein.<br>
One such protein, DEX1, was isolated from Arabidopsis and was reported to play a role during pollen development (Paxson-Sowders et a!. Plant Physiol. 127, 1739-1749, 2001). Dex1 mutant plants were shown to be defective in pollen wall pattern formation. The DEX1 gene encodes an 895-amino acid protein that is predicted to localize to the plasma membrane, with residues 1 through to 850 being located outside of the cell, residues 880 through to 895 on the cytoplasmic side of the rrrembrane, and amino acids 861 through to 879 representing a potential membrane-spanning domain. Twelve potential N-glycosylation sites are present in DEX1. Therefore, the protein has the potential to be heavily modified and interact with various components of the cell wall. DEX1 shows the greatest sequence similarity to a hemolysin-iike protein from V. cholerae, whereas an approximately 200-amino acid segment of DEX1 (amino<br><br>
acids 439-643) also shows limited similarity to the calcium-binding domain of aipha-integrins. In this region are at least two sets of putative calcium-binding iigands that are also present in a predicted Arabidopsis calmodulin protein (AC009853). Therefore, it appears that DEX1 may be a calcium-binding protein. DEX1 appears to be a unique plant protein; homologs are not present in bacteria, fungi, or animals.<br>
The alterations observed in dex1 plants, as well as the predicted structure of DEX1, raise several possibilities for the role of the protein in pollen wall formation (Paxson-Sowders et a!., 2001):<br>
(a)	DEX1 could be a linker protein. It may associate with the microspore membrane and participate in attaching either the phmexine or sporopolienin to the plasma membrane. Absence of the protein from the microspore surface could result in structural alterations in the primexine. The numerous potential N-giycosyiation sites are consistent with attachment of DEX1 to the caliose wall, the intine, or both.<br>
(b)	DEX1 may be a component of the primexine matrix and play a role in the initial polymerization of the primexine. Changes in Ca+2 ion concentrations appear to be important for pollen wall synthesis; beta-glucan synthase is activated by micromolar concentrations of Ca+2 during caliose wall forrnation.<br>
(c)	DEX1 could be part of the rough ER and be involved in processing and/or transport of primexine precursors to the membrane. The delayed appearance and general alterations in the primexine are consistent with a general absence of primexine precursors. The primexine matrix is initially composed of polysaccharides, proteins, and cellulose, followed by the incorporation of more resistant materials. Therefore, DEX1 may participate in the formation or transport of any number of different components.<br>
CYP90B<br>
Brassinosteroids (BRs) are a class of plant hormones that are important for promoting plant growth, division and development. The term BR collectively refers to more than forty naturally occurring poly-hydroxylated sterol derivatives, with structural similarity to animal steroid hormones. Among these, brassinolide has been shown to be the most biologically active (for review, Clouse (2002) Brassinosteroids. The Arabidopsis Book: 1-23).<br>
The BR biosynthetic pathv\/ay has been elucidated using biochemical and mutational analyses. BRs are synthesized via at least two branched biochemical pathways starting from the same<br>
initial precursor, campesterol (Fujioka et al. (1937) Physiol Plant 100:710-715). The discovered BR biosynthesis genes have been found to encode mostly cytochrome P450<br><br>
monooxygenases (CYP) (Bishop and Yokota (2001) Plant Cell Physiol 42:114-120). CYP supsrfamlly of enzymes catalyses the oxidation of many chemicals, and in the present case more specifically catalyse essential oxidative reactions in the biosynthesis of BRs. One of the important steps identified consists in the hydroxyiation of the steroid side chain of BR intermediates campestanol and 5-oxocampestanol to form 6-deoxocathasterone and cathasterone respectively. These two parallel oxidative steps are also collectively called the early steroid C-22 alpha-hydroxylation step (Choe et al. (1998) Plant Cell 10; 231-243). in ArabidopsiS: a specific CYP enzyme, CYP90B1 or DWF4, performs this step (for general reference on plant CYP nomenclature, Nelson et al. (2004) Plant Phys 135: 756-772).<br>
ArabidopsiS mutant plants lacking steroid 22 alpha hydroxylase activity due insertion of a T-DNA in the DWF4 locus displayed a dwrarfed phenotype due to lack of cell elongation (Choe et al. (1998) Plant Cell 10: 231-243). Biochemical feeding studies with BR biosynthesis intermediates showed that all of the downstream compounds rescued the phenotype, whereas the known precursors failed to do so.<br>
Transgenic ArabidopsiS .and tobacco plants, both dicotyledonous, were generated .that . ectopically overexpressed an ArabidopsiS DWF4 genomic fragment, using the cauliflower mosaic virus 35S promoter (Choe ef al. (2001) Plant J 26(5): 573-582). Phenotypic characterisation of the plants showed that the hypocotyl length, plant height at maturity, total number of branches and total number of seeds were increased in the transgenics compared to control plants. Choe ef al. found that the increased seed production was due to a greater number of seeds per plant, seed size increase being within the range of standard deviation. These experiments are further described in WOOO/47715.<br>
Patent US 6,545,200 relates to isolated nucleic acid fragments encoding sterol biosynthetic genes, and more specifically claims a nucleotide sequence encoding a polypeptide having C-8,7 sterol isomerase activity.  Partial nucleotides sequences encoding DWF4 are disclosed.<br>
US 2004/0050079 relates to a method of producing a modified monocotyledonous plant having a desired trait. An example is provided in which the rice DWF4-encoding nucleotide sequence (referred to either OsDWF4 or CYP90B2) is placed under the control of a constitutive promoter, the rice actin promoter. Fourteen of the thirty-six transgenic rice plants expressing the chimeric construct show an increased number of grains per spike as compared to non-transformed control plants. According to the inventors, the yield increase in the transgenics compared to the wild types is due to an increase in total number of seeds, as no significant difference is found in the "weight of 10 grains".<br><br>
CDC27<br>
Depending on the end use, the modification of certain yield traits may be favoured over others. For example for applications such as forage or wood production, or bio-fuei resource, an increase in the leafy parts of a plant may be desirable, and for applications such as flour, starch or oil production, an increase in seed parameters may be particularly desirable. Even within the seed parameters, some may be favoured over others, depending on the application. Various mechanisms may contribute to increasing seed yield, whether that is in the form of increased seed size or Increased seed number. One such mechanism is the cell cycle.<br>
Progression through the cell cycle is fundamental to the growth and development of all multicellular organisms and is crucial to cell proliferation. The major components of the cell cycle are highly consen/ed in yeast, mammals,' and plants, The cell cycle is typically divided into the following sequential phases: GO - C3i - S - G2 - M. DNA replication or synthesis generally takes place during the S phase ("S" is for DNA synthesis) and mitotic segregation of the chromosomes occurs during the M phase (the "M" is for mitosis), with intervening gap phases, G1 (during which cells grow before DNA replication) and G2 (a period after DNA replication during which the cell prepares for division). Cell division is completed after cytokinesis, the last step of the M phase. Cells that have exited the cell cycle and that have become quiescent are said to be in the GO phase. Cells in this phase can be stimulated to renter the cell cycle at the G1 phase. The "G" in G1, G2 and GO stands for "gap". Completion of the cell cycle process allows each daughter cell during cell division to receive a full copy of the parental genome.<br>
Cell division is controlled by two principal cell cycle events, namely initiation of DNA synthesis and initiation of mitosis. Each transition to each of these key events is controlled by a checkpoint represented by specific protein complexes (involved in DNA replication and division). The expression of genes necessary for DNA synthesis at the G1/S boundary is regulated by the E2F family of transcription factors in mammals and plant cells (La Thangue, 1994; Muiler et al., 2001; De Veylder et al., 2002). Entry into the ceil cycle is regulated/triggered by an E2F/Rb complex that integrates signals and allows activation of transcription of cell cycle genes. The transition between the different phases of the cell cycle. and therefore progression through the cell cycle, is driven by the formation and activation of different heterodimeric serine/threonine protein kinases, generally referred to as cyciin-dependent kinases (CDKs). A prerequisite for activity of these kinases is the physical association with a specific cyclin, the timing of activation being largely dependent upon cyciin expression.    Cyciin-binding induces conformational changes in the N-terminal lobe of the<br><br>
associating CDK and contributes to the localisation and substrate specificity of the complex. Monomeric CDKs are activated when they are associated with cyclins and thus have a kinase activity. Cyciin protein levels fluctuate in the cell cycle and therefore represent a major factor in determining timing of CDK activation. The periodic activation of these complexes containing ■ cyclins and CDK during cell cycle mediates the temporal regulation of cell cycle transitions (checkpoints).<br>
Mechanisms exist to ensure that DNA replication occurs only once during the cell cycle. For example, CDC1S, CDC23 and CDC27 proteins are-part of a high molecular weight complex known as the anaphase promoting complex (APC) or cyclosome, (see Romanowski and Madine, Trends in Cell Biology 6, 184-188, 1996, and Wuarin and Nurse, Cell 85, 785-787 (1996). The complex in yeast is composed of at least eight proteins, the TPR-(tetratrico peptide repeat) containing proteins CDC16, CDC23 and CDC27, and five other subunits named APC1, APC2, APC4, APC5 and APC7 (Peters et al. 1996, Science 274, 1199-1201). The APC targets its substrates for proteolytic degradation by catalyzing the ligation of ubiquitin molecules to these substrates. APC-dependent proteolysis is required for the separation of the sister chromatids at meta- to anaphase transition and for the finalexit from mitosis. Among the APC-substrates are the anaphase inhibitor protein Pdsip and mitotic cyclins such as cyciin B, respectively (Ciosk et al. 1998, Cell 93, 1057-1078; Cohen-Fix et al. 1996, Genes Dev 10, 3081-3093; Sudakin et al. 1995, Mol Biol Cell 6, 185-198; Jorgensen et al. 1998, Mol Cell Biol 18, 468-476; Townsley and Ruderman 1998, Trends Cell Biol 8, 238-244). To become active as an ubiquitin-iigase, at least CDC16, CDC23 and CDC27 need to be phosphorylated in the M-phase (Ollendorf and Donoghue 1997, J Biol Chem 272, 32011-32018). Activated APC persists throughout G1 of the subsequent cell cycle to prevent premature appearance of B-type cyclins, which would result in an uncontrolled entry into the S-phase (Irniger and Nasmyth 1997, J Cell Sci 110, 1523-1531). It has been demonstrated in yeast that mutations in either of at least two of the APC components, CDC15 and CDC27, can result in DNA overrepiication without intervening passages through M-phases (Heichman and Roberts 1996, Cell 85, 39-48). This process of replication of nuclear DNA without subsequent mitosis and cell division is called DNA endoredupiication, and leads to increased cell size.<br>
CDC16, CDC23 and CDC27 all are tetratrico peptide repeat (TPR; 34 amino acids long) ■ containing proteins. A suggested minimal consensus sequence of the TPR motif is as follows: Xs-W-Xz-L-G-X^-Y-Xe-A-Xj-F-Xj-A-X^-P-Xz, v/here X is any amino acid (Lamb et a!. 1994, EMBO J 13, 4321-4328). The consensus residues can exhibit significant degeneracy and little or no homology is present in non-consensus residues. It is the hydrophobicity and size of the consensiis residues, rather than their identity, that seems to be of importance.  TPR motifs are<br><br>
present in a wide variety of proteins functional in yeast and higher eukaryotes in mitosis (including the APC protein components CDC16, CDC23 and CDC27), transcription, splicing, protein import and neurogenesis (Goebl and Yanagida 1991, Trends Biochem Sci 16, 173-177). The TPR forms an a-helical structure; tandem repeats organize into a superheiical structure ideally suited as interfaces for protein recognition (Groves and Barford 1999, Curr Opin Struct Biol 9, 383-389). Within the a-helix, two amphipathic domains are usually present, one at the NH2 terminal region and the other near the COOH terminal region (Sikorski et al. 1990, Cell 60, 307-317).<br>
CDC27 (also known as Hobbit; others names include CDC27, BimA, Nuc2 or makos) has been isolated from various organisms, including Aspergillus nidulans, yeast, drosophila, human and various plants (such as Arabidopsis thaliana and Oryza sativa). The gene encoding CDC27 is present as a single copy in most genomes, but two copies may exceptionally be found within the same genome, for example in Arabidopsis thaliana. The two genes encoding CDC27 proteins have been named CDC27A and CDC27B (MIPS references At3g 16320 and At2g20000 respectively).<br>
Published International Patent Application, WO0.1/Q2430 describes CDC27A (CDC27A1 and.     	<br>
CDC27A2) and CDC27B sequences. Also described in this document is a truncated CDC27B amino acid sequence in which 161 amino acids are missing from the NH2 terminal region. Reference is made in this document to GenBank accession number AC006081 for the CDC27B gene encoding a CDC27B polypeptide truncated at the NH2 terminal region. The document reports the NH2 terminal region to be conserved in CDC27 homoiogues of different origin. The CDC27 sequences mentioned in WO01/02430 are described to be useful in modifying endoredupiication.<br>
DMA endoredupiication occurs naturally in flowering plants, for example during seed . development. DMA endoredupiication leads to enlarged nuclei with elevated DNA content. It has been suggested that the increased DNA content during endoredupiication may provide for increased gene expression during endosperm development and kernel filling, since it coincides with increased enzyme activity and protein accumulation at this time (Kowles et al., (1992) Genet. Eng. 14:55-88). In cereal species, the cellular endosperm stores the resen/es of the seed during a phase marked by endoredupiication. The magnitude of DNA endoredupiication is highly correlated with endosperm fresh weight, which implies an important role of DNA endoredupiication in the determination of endosperm mass (Engeien-Eigies et al. (2000) Plant Cell Environ. 23:657-563). In maize for example, the endosperm makes up 70 to 90% of kernel mass: tnus. factors that mediate endosperm development to a great extent also determine grain<br><br>
yield of maize, via individual seed weight. Increased endoreduplication is therefore typically indicative of increased seed biomass but is in no way related to increased seed number.<br>
AT-hook transcription factor<br>
An AT-hook domain is found in polypeptides belonging to a family of transcription factors associated with Chromatin remodeling. The AT-hook motif is made up of 13 or so (sometimes about 9) amino acids which participate in DMA binding and which have a preference for A/T rich regions. In Arabidopsis there are at least 34 proteins containing AT-hook domains. These proteins share homology along most of the.sequence, with the AT-hook domain being a particularly highly conserved region.<br>
International Patent application WO 2005/030966 describes several plant transcription factors<br>
comprising AT-hook domains and the use of these transcription factors to produce plants<br>
having increased biomass and increased stress tolerance. The application concerns members<br>
of the G1073 clade of transcription factors and states that, "Use of tissue-specific or inducible<br>
'	promoters mitigates undesirable morphological effects that may be associated with constitutive<br>
overexpression of-G1073 .clade members (e.gwhen increased size is undesirable)."   The. data provided in this application relate to dicotyledonous plants.<br>
In contrast to these teachings, it has now been found that expression in a monocotyledonous (monocot) plant of a polynucleic acid encoding an AT-hook transcription factor comprising a DUF296 domain (which includes members of clade G1073), gives plants having little or no increase in biomass compared with suitable control plants, regardless of whether that expression is driven by a constitutive promoter or in a tissue-specific manner. This suggests that teachings concerning expression of such transcription factors in dicots may not be so readily applicable to monocots. It has also now been found that the extent or nature of any increase in seed yield obtained is dependent upon the tissue-specific promoter used.<br>
DOF transcription factors<br>
Dof domain proteins are plant-specific transcription factors with a highly conserved D.NA-binding domain with a single C2-C2 zinc finger. During the past decade, numerous Dof domain proteins have been identified in both monocots and dicots including maize, barley, wheat, rice. tobacco, Arabidopsis, pumpkin, potato and pea. Dof domain proteins have been shown to function as transcriptional activators or repressors in diverse piani-specific biobgicai processes.<br><br>
Cvclin Dependent kinase inhibitors (CKI)<br>
The abiiity to increase plant seed yield, whether through seed number, seed biomass, seed development, seed filling or any other seed-related trait would have many applications in agriculture, and even many non-agricultural uses such as in the biotechnological production of substances such as pharmaceuticals, antibodies or vaccines. One approach to increasing seed yield in plants may be through modification of the inherent growth mechanisms of a plant.<br>
The inherent growth mechanisms of a plant reside in a highly ordered sequence of events collectively known as the 'cell cycle'. Progression through the cell cycle is fundamental to the growth and development of all multi-cellular organisms and is crucial to cell proliferation. The major components of the cell cycle are highly conserved in yeast, mammals, and plants. The cell cycle is typically divided into the following sequential phases: GO - G1 - S - G2 - M. DNA replication or synthesis generally takes place during the S phase ("S" is for DNA synthesis) and mitotic segregation of the chromosomes occurs during the M phase (the "M" is for mitosis), with intervening gap phases, G1 (during which cells grow before DNA replication) and G2 (a period after DNA replication during which the cell prepares for division). Cell division is completed after cytokinesis,, the last step of the M phase. Cells that have exited the cell cycle . and that have become quiescent are said to be in the GO phase. Cells in this phase can be stimulated to renter the cell cycle at the G1 phase. The "G" in G1, G2 and GO stands for "gap". Completion of the cell cycle process allows each daughter cell during cell division to receive a full copy of the parental genome.<br>
Cell division is controlled by two principal cell cycle events, namely initiation of DNA synthesis and initiation of mitosis. Each transition to each of these key events is controlled by a checkpoint represented by specific protein complexes (involved in DNA replication and division). The expression of genes necessary for DNA synthesis at the G1/S boundary is regulated by the E2F family of transcription factors in mammals and plant cells (La Thangue, 1994; Muller ef a/.. 2001; De Veylder et al., 2002). Entry into the cell cycle is regulated/triggered by an E2F/Rb complex that integrates signals and allows activation of transcription of cell cycle genes. The transition between the different phases of the cell cycle, and therefore progression through the cell cycle, is driven by the formation and activation of different heterodimeric serine/threonine protein kinases, generally referred to as cyclin-dependent kinases (CDKs). A prerequisite for activity of these kinases Is the physical association with a specific cyclin, the timing of activation being largely dependent uDon cyclin expression. Cyclin binding induces conformational changes in the N-terminal lobe of the associating CDK and contributes to the localisation and substrate specificity of the complex. Monomeric CDKs are activated when they are associated with cyclins and thus have kinase<br><br>
activity. Cyciin protein levels usually fluctuate in the cell cycle and therefore represent a major factor in determining timing of CDK activation. The periodic activation of these complexes containing cyclins and CDK during cell cycle mediates the temporal regulation of cell-cycle transitions (checkpoints). Other factors regulating CDK activity include cyciin dependent kinase inhibitors (CKts or ICKs, KIPs, CIPs, INKs), CDK activating kinases (CAKs), a CDK phosphatase (Cdc25) and a CDK subunit (CKS) (Mironov et al. 1999; Reed 1996).<br>
The existence of an inhibitor of mitotic CDKs was inferred from experiments with endosperm of maize seed (Grafi and Larkins (1995) Science 269, 1262-1264). Since then, several CKIs have been identified in vanous plant species, such as Arat)/c/ops/s (Wang et al. (1997) Nature 386(6624): 451-2; De Veylder et al. (2001) Plant Cell 13: 1653-1668; Lui et al. (2000) Plant J 21: 379-385), tobacco (Jasinski et al. (2002) Plant Physiol 2002 130(4): 871-82), Chenopodium rubrum (Fountain et al. (1999) Plant Phys 120: 339) or corn (Coelho et al. (2005) Plant Physiol 138: 2323-2335). The encoded proteins are characterized by a stretch of approximately 45 carboxy-terminal amino acids showing homology to the amino-terminal cyclin/Cdk binding domain of animal CKIs of the -types.   Outside this<br>
carboxy-terminal region^ plant CKIs show little homology. _<br>
Published International patent application WO 2005/007829 in the name of Monsanto Technology LLC describes various isolated nucleic acid molecules encoding polypeptides having cyciin dependent kinase inhibitor activity.<br>
Published International patent applications, WO 02/28893 and WO 99/14331, both in the name of CropDesign N.V., describe various plant cyciin dependent kinase inhibitors. The use of these inhibitors to increase yield is mentioned in these applications.<br>
SUMMARY OF THE INVENTION<br>
It has now surprisingly been found that increasing activity of a SYR protein and/or expression of a nucleic acid encoding a SYR protein in plants results in plants having increased seed yield and or increased growth rate, relative to corresponding wild type plants. It has also now surprisingly been found that overexpression of SYR in rice primarily increases seed yield, whereas the leafy biomass and flowering time are not obviously affected (in contrast to the major phenotypes of ARGOS overexpression in Arabidopsis, which were shown to be increased leafy biomass and delayed flowering 2005/0108793)). <br><br>
According to one embodiment of the present invention there is provided a method for increasing seed yieid and/or growth rate of a plant comprising increasing activity of a SYR polypeptide or a homoiogue thereof in a plant and/or expression of a nucleic acid encoding such a protein; and optionally selecting for plants having improved growth characteristics.<br>
Advantageously, performance of the methods of the invention insofar as they concern SYR, result in plants having a vahety of improved growth charactehstics, such as improved seed yieid without effect on the biomass of vegetative plant parts, when compared to corresponding control plants, and a life cycle comparable to corresponding control piants, without delay in flowering time. Further advantageously, performance of the methods according to the present invention result in plants having improved tolerance to abiotic stress relative to corresponding wild type (or other control) plants.<br>
It has now surprisingly been found that modulating activity of an FG-GAP protein and/or expression of a nucleic acid encoding an FG-GAP protein in plants results in plants having improved growth characteristics, and in particular increased yieid, relative to corresponding wild type plants.    -.    .<br>
According to another embodiment of the present invention there is provided a method for improving grow'th characteristics of a plant comprising modulating activity of an FG-GAP polypeptide or a homoiogue thereof and/or modulating expression of a nucleic acid encoding an FG-GAP polypeptide or a homoiogue thereof in a plant and optionally selecting for piants having improved growth charactehstics.<br>
Advantageously, performance of the methods according to the present invention, insofar as they concern an FG-GAP polypeptide or a homoiogue thereof, result in plants having a variety of improved growth characteristics, such as improved growth, improved yield, improved biomass. improved architecture or improved cell division, each relative to corresponding wild type plants. Preferably, the improved growth characteristics comprise at least increased yield relative to corresponding wild type plants.<br>
It has now surprisingly been found that increasing non-constitutive expression in a plant of a nucleic acid encoding a CYP90B polypeptide or a homoiogue thereof gives piants having<br>
increased yield relative to suitable control plants.<br><br>
According to a further embodiment of the present invention, there is provided a method for increasing plant yield comprising increasing non-constitutive expression in a plant of a nucleic acid encoding a CYP90B polypeptide or a homologue thereof.<br>
It has now been found that preferentially increasing expression in the shoot apical meristem tissue of plants of a nucleic acid encoding a CDC27 polypeptide having at least one inactive TPR domain in the NH2 terminal region of the polypeptide gives plants having increased seed number relative to suitable control plants.<br>
The invention therefore provides a method for increasing the seed number of plants relative to that of suitable control plants, comprising preferentially increasing expression in plant shoot apical meristem tissue of a nucleic acid encoding a CDC27 polypeptide having at least one inactive TPR domain in the NH2 terminal region of the polypeptide.<br>
It has no\N been found that preferentially increasing expression of a nucleic acid encoding a polypeptide comprising an AT-hook domain and a DUF296 domain in endosperm tissue of a monocotyledonous plant gives plants having increased seed yield relative to suitable control plants.<br>
A further embodiment of the present invention therefore provides a method for increasing seed yield in monocotyledonous plants relative to suitable control plants, comprising preferentially increasing expression in endosperm tissue of a monocotyledonous plant of a nucleic acid encoding a polypeptide comprising an AT-hook domain and a DUF296 domain.<br>
It has now been found that increasing expression in a plant of a nucleic acid encoding a DOF transcription factor polypeptide gives plants having increased yield relative to suitable control plants.<br>
According to a further embodiment of the present invention, there is provided a method for increasing plant yield comprising increasing expression in a plant of a nucleic acid encoding a DOF transcription factor polypeptide.<br>
It has.now been found that preferential reduction in expression of an endogenous CKI gene in endosperm tissue of a plant gives plants with better seed yield than seed yield in plants where there is no preferential reduction in expression of an endogenous CKI gene in plant endosperm tissue.   The present invention therefore provides a method for increasing seed<br><br>
yield in plants relative to suitable control plants, comprising preferentially reducing-expression of an endogenous CKl gene in endosperm tissue of a plant,<br>
DETAILED DESCRIPTION OF THE INVENTION<br>
The term'"increased yield" as defined herein is taken to mean an increase in biomass (weight) of one or more parts of a plant (particularly harvestable parts) relative to corresponding wild type or other control plants, which increase in biomass may be aboveground or underground. An increase in biomass underground may be due to an increase in the biomass of plant parts, such as tubers, rhizomes, bulbs etc. Particularly preferred is an increase in any one or more of the following: increased root biomass, increased root volume, increased root number, increased root diameter and increased root length. The term increased yield also encompasses an increase in seed yield.<br>
The term "increased seed yield" as defined herein is taken to mean an increase in any one or more of the following, each relative to corresponding wild type plants: (i) increased total seed yield, which includes an increase in seed biomass (seed weight) and which may be an increase in the seed weight per plant or on an individual seed basis; (ii) increased number of . . _ flowers ("florets") per panicle (iii) increased number of filled seeds; (iv) increased seed size; (v) increased seed volume; (vi) increased individual seed area; (vii) increased individual seed length and/or width; (viii) increased harvest index, which is expressed as a ratio of the yield of harvestable parts, such as seeds, over the total biomass; (ix) increased fill rate, (which is the number of filled seeds divided by the total number of seeds and multiplied by 100); and (x) increased thousand kernel weight (TKW), which is extrapolated from the number of filled seeds counted and their total weight. An increased TKW may result from an increased seed size and/or seed weight. An increased TKW may result from an increase in embryo size and/or endosperm size.<br>
Taking corn as an example, a yield increase may be manifested as one or more of the following: an increase in the number of ears per plant, an increase in the number of rows, number of kernels per row, kernel weight, TKW, ear length/diameter, among others. Taking rice as an example, a yield increase may be manifested by an increase in one or more of the following: number of panicles per plant, number of spikelets per panicle, number of flowers per panicle, increase in the seed filling rate, increase in TKW, among others. An increase in yield may also result in modified architecture, or may occur as a result of modified architecture.<br>
The improved grov^ah characteristics obtained by performing the methods of the invention. insofar as they concern use of CDC27, result in plants having increased seed number.   An<br><br>
V<br>
increased seed number encompasses an increase in the total number of seeds and/or tiie-number of filled seeds and/or an increase in the seed filling rate (which is the number of filled seeds divided by the total number of seeds and multiplied by 100), each relative to suitable control plants, which increase may be per plant and/or per hectare or acre. Taking corn as an example, an increase in the number of seeds is typically manifested by an increase in the number of ears per plant, an increase in the number of rows, number of kernels per row, increase in the seed filling rate, among others. Taking nee as an example, an increase in the number of seeds is typically manifested by an increase in number of panicles per plant, number of spikelets per panicle, number of flowers (florets) per panicle (which is expressed as a ratio of the number of filled seeds over the number of primary panicles), increase in the seed filling rate.<br>
The invention therefore provides a method for increasing the seed number of plants relative to that of suitable control plants, comprising preferentially increasing expression in plant shoot apical meristem tissue of a nucleic acid encoding a CDC27 polypeptide having at least one inactive TPR domain in the NH2 terminal region of the polypeptide.<br>
Insofar as the methods of the invention concern SYR, preferably performance of the methods result in plants having increased seed yield. Further preferably, the increased seed yield comprises an increase in one or more of number of (filled) seeds, total seed weight, seed size, thousand kernel weight, fill rate and harvest index, each relative to control plants. Therefore, according to the present invention, there is provided a method for increasing plant seed yield, which method comprises increasing activity of a SYR polypeptide and/or expression in a plant of a nucleic acid encoding a SYR polypeptide or a homologue thereof.<br>
Insofar as the methods of the invention concern FG-GAP, preferably performance of the methods result in plants having increased yield and, more particularly, increased biomass and/or increased seed yield. Preferably, the increased seed yield comprises an increase in one or more of number of (filled) seeds, total seed weight, seed size, thousand kernel weight and harvest index, each relative to control plants. Therefore, according to the present invention, there is provided a method for increasing plant yield, particularly, increased biomass and/or increased seed yield, which method comprises modulating activity of an FG-GAP polypeptide and/or expression in a plant of a nucleic acid encoding an FG-GAP polypeptide or a homologue thereof,<br>
insofar as the methods of the invention concern CYP903, preferably the increased yield includes one or more of the following: increased HI, increased TKW, increased seed area and<br><br>
increased seed length, each relative to suitable control plants. Therefore, according to the present invention, there is provided a method for increasing plant yield, particularly seed yield, relative to suitable control plants, which method comprises increasing non-constitutive expression in a plant of a nucleic acid encoding a CYP90B polypeptide or a homologue<br>
thereof.<br>
Insofar as methods of the invention concern AT-hook transcription factors, seed yield in monocotyledonous plants is increased. There is therefore provided a method for increasing seed yield in monocotyledonous plants relative to suitable control plants, comprising preferentially increasing expression in endosperm tissue of a monocotyledonous plant of a nucleic acid encoding a polypeptide comprising an AT-hook domain and a DUF296 domain.<br>
Insofar as the methods of the invention concern DOF transcription factors, preferably the increased yield is increased seed yield. According to a preferred feature of the present invention, there is provided a method for increasing plant seed yield relative to seed yield of suitable control plants,which method comprises increasing expression in a plant of a nucleic acid encoding a DOF transcription factor polypeptide.<br>
Insofar as the methods of the invention concern CKIs, the improved growth characteristic is increased seed yield. The present invention therefore provides a method for increasing seed yield in plants relative to suitable control plants, comprising preferentially reducing expression of an endogenous CKI gene in endosperm tissue of a plant.<br>
Since the improved plants according to the present invention have increased yield (seed yield), it is likely that these plants exhibit an increased growth rate (during at least part of their life cycle), relative to the growth rate of corresponding wild type plants at a corresponding stage in their life cycle. The increased growth rate may be specific to one or more parts or cell types of a plant (including seeds), or may be throughout substantially the whole plant. Plants having an increased growth rate may have a shorter life cycle. The life cycle of a plant is taken to mean the time needed to grow from a dry mature seed up to the stage where the plant has produced dry mature seeds, similar to the starting material. This fife cycle may be influenced by factors such as early vigour, growth rate, flowering time and speed of seed maturation. An increase in growthh rate may take place at one or more stages in the life cycle of a plant or during substantially the whole plant life cycle, increased growth rate during the earty stages in the life cycle of a plant may reflect enhanced vigour. The increase in growth rate may alter the har\'55t cycle of a plant allowing plants to be sown later and/or harvested sooner than would otherwise be possible.  If the growth rate is sufficiently increased, it may allow for the sowing of<br><br>
further seeds of the same plant species (for example sowing and harvesting of rice plants followed by sowing and harvesting of further rice plants all within one conventional growing period). Similarly, if the grovirth rate is sufficiently increased, it may allow for the further sowing of seeds of different plants species (for example the sowing and harvesting of rice plants followed by, for example, the sowing and optional harvesting of soy bean, potatoes or any other suitable plant). Harvesting additional times from the same rootstock in the case of some plants may also be possible. Altering the harvest cycle of a plant may lead to an increase in annual biomass production per acre (due to an increase in the number of times (say in a year) that any particular plant may be grown and harvested). An increase in growth rate may also allow for the cultivation of transgenic plants in a wider geographical area than their wild-type counterparts, since the territorial limitations for growing a crop are often determined by adverse environmental conditions either at the time of planting (eariy season) or at the time of harvesting (late season). Such adverse conditions may be avoided if the harvest cycle is shortened. The growth rate may be determined by deriving various parameters from growth curves plotting growth experiments, such parameters may be: T-Mid (the time taken for plants to reach 50% of their maximal size) and T-90 (time taken for plants to reach 90% of their maximal size), amongst others. The term "flowering time" as used herein shaJ! meanIhe time period between the start of seed germination and the start of flowering.<br>
Performance of the methods of the invention gives plants having an increased growth rate.<br>
Therefore, according to the present invention, there is provided a method for increasing the growth rate of plants, which method comprises increasing activity in a plant of a SYR polypeptide or a homoiogue thereof and/or expression of a nucleic acid encoding such a protein.<br>
According to the present invention, there is provided a method for increasing the growth rate of plants, which method comprises modulating (preferably increasing) activity in a plant of an FG-GAP polypeptide or a homoiogue thereof and/or modulating (preferably increasing) expression of a nucleic acid encoding such protein.<br>
According to the present invention, there is provided a method for increasing the growth rate of plants which method comprises increasing non-constitutrve expression in a plant of a nucleic acid encoding a CYP90B polypeptide or a homoiogue thereof.<br><br>
According to the present invention, there is provided a method for increasing the growth rate of plants, which method comprises increasing expression in a plant of a nucleic acid encoding a DOF transcription factor polypeptide.<br>
According to the present invention, there is provided a method for increasing the growth rate of plants relative to suitable control plants, which method comprises preferentially reducing expression of an endogenous Cyciin Dependent Kinase Inhibitor (CKI) gene in endosperm tissue of a plant.<br>
An increase in yield and/or seed yield and/or growth rate occurs whether the plant is under non-stress conditions or whether the plant is exposed to various stresses compared to control plants. Plants typically respond to exposure to stress by growing more slowly, in conditions of severe stress, the plant may even stop growing altogether. Mild stress on the other hand is defined herein as being any stress to which a plant is exposed which does not result in the plant ceasing to grow altogether without the capacity to resume growth. Mild stress in the sense of the invention leads to a reduction in the growth of the stressed plants of less than 40%, 35% or 30%, preferably less than 25%, 20% or 15%, more preferably less than 14%, 13%, 12%, 11% or 10% or less in comparison to the control plant under non-stress conditions. Due to advances in agricultural practices (irrigation, fertilization, pesticide treatments) severe stresses are not often encountered in cultivated crop plants. As a consequence, the compromised grovW:h induced by mild stress is often an undesirable feature for agriculture. Mild stresses are the typical stresses to which a plant may be exposed. These stresses may be the everyday biotic and/or abiotic (environmental) stresses to which a plant is exposed. Typical abiotic or environmental stresses include temperature stresses caused by atypical hot or cold/freezing temperatures; salt stress; water stress (drought or excess water), anaerobic stress, chemical toxicity and oxidative stress. The abiotic stress may be an osmotic stress caused by a water stress (particulariy due to drought), salt stress, oxidative stress or an ionic stress. Chemicals may also cause abiotic stresses (for example too high or too low concentrations of minerals or nutrients). Biotic stresses are typically those stresses caused by pathogens, such as bacteria, viruses, fungi and insects. The term "non-stress conditions" as used herein are those environmental conditions that do not significantly go beyond the everyday climatic and other abiotic conditions that plants may encounter, and which allow optimal growth of the plant. Persons skilled in the art are aware of norm.al soil conditions and climatic conditions for a given geographic location.<br>
Insofar as the methods of the invention concern SYR, performance of the methods result in plants having increased tolerance to abiotic stress.  As reported in Wang et al. (Planta (2003)<br><br>
218: 1-14), abiotic stress leads to a series of morphological, physiological, biochemical and molecular changes that adversely affect plant growth and productivity. Drought, salinity, extreme temperatures and oxidative stress are known to be interconnected and may induce growth and cellular damage through similar mechanisms. For example, drought and/or salinisation are manifested primarily as osmotic stress, resulting in the disruption of homeostasis and ion distribution in the cell. Oxidative stress, which frequently accompanies high or low temperature, salinity or drought stress may cause denaturation of functional and structural proteins. As a consequence, these diverse environmental stresses often activate similar cell signaling pathways and cellular responses, such as the production of stress proteins, up-regulation of anti-oxidants, accumulation of compatible solutes and growth arrest.<br>
Since diverse environmental stresses activate similar pathways, the exemplification of the present invention with drought stress (insofar as the invention concerns the use of SYR polypeptides and their encoding nucleic acids) should not be seen as a limitation to drought stress, but more as a screen to indicate the involvement of SYR polypeptides or homologues thereof in abiotic stresses in general. Furthermore, the methods of the present invention may be performed-under x)on--Stress conditions or under conditions of mild drought to give, plants _ ._ having improved growth characteristics (particularly increased yield) relative to corresponding wild type or other control plants.<br>
A particulariy high degree of "cross talk" is reported between drought stress and high-salinity stress (Rabbani et al. (2003) Plant Physiol 133: 1755-1767). Therefore, it would be apparent that a SYR polypeptide or a homologue thereof would, along with its usefulness in conferring drought-tolerance in plants, also find use in protecting the plant against various other abiotic stresses. Similarly, it would be apparent that a SYR protein (as defined herein) would, along with its usefulness in conferring salt-tolerance in plants, also find use in protecting the plant against various other abiotic stresses. Furthermore, Rabbani et al. (2003, Plant Physiol 133: 1755-1767) report that similar molecular mechanisms of stress tolerance and responses exist between dicots and monocots. The methods of the invention are therefore advantageously applicable to any plant.<br>
The term "abiotic stress' as defined herein is taken to mean any one or more of: water stress (due to drought or excess water), anaerobic stress, salt stress, temperature stress (due to hot, coid or freezing temperatures), chemical toxicity stress and oxidative stress. According to one aspect of the invention, the abiotic stress is an osmotic stress, selected from water stress, salt stress, oxidative stress and ionic stress.   Preferably, the water "stress is drought stress.   The<br><br>
"V<br>
term salt stress is not restricted to common salt (NaCi), but may be any one or more of: NaCl, KCl, LiCl, MgCI2, CaCl2, amongst others.<br>
Increased tolerance to abiotic stress is manifested by increased plant yieid in abiotic stress conditions. Insofar as the invention concerns the use of SYR polypeptides and their encoding nucleic acids, such increased yield may include one or more of the following: increased number of filled seeds, increased total seed yield, increased number of flowers per panicle, increased seed fill rate, increased Harvest Index, increased Thousand Kernel Weight, increased root length or increased root diameter, each relative to corresponding wild type plants.<br>
Performance of the methods of the invention gives plants having increased tolerance to abiotic stress. Performance of the methods of the invention gives plants grown under non-stress conditions or under mild drought conditions improved growth characteristics (particularly increased yield and/or increased emergence vigour (or early vigour)) relative to corresponding wild type plants or other control plants grown under comparable conditions.<br>
According to the present invention, there is provided a method for increasing abiotic stress tolerance in plants which method comprises modulating expression in a plant of a nucleic acid encoding a SYR polypeptide or a homologue thereof. According to one aspect of the invention, the abiotic stress is osmotic stress, selected from one or more of the following: water stress, salt stress, oxidative stress and ionic stress. Preferably, the water stress is drought stress.<br>
The present invention also provides a method for improving abiotic stress tolerance in plants, comprising increasing activity in a plant of a SYR protein or a homologue thereof.<br>
Insofar as the methods of the invention concern DOF transcription factors, the methods may be performed under conditions of mild drought to give plants having increased yield relative to suitable control plants. As reported in Wang et a!. (Planta (2003) 218: 1-14), abiotic stress leads to a series of morphological, physiological, biochemical and molecular changes that adversely affect plant growth and productivity. Drought, salinity, extreme temperatures and oxidative stress are known to be interconnected and may induce grovW:h and cellular damage through similar mechanisms. Rabbani ei al. (Plant Physioi (2003) 133: 1755-1767) describes a particularly high degree of "cross talk" between drought stress and high-salinity stress. For example, drought and/or salinisation are manifested primarily as osmotic stress, resulting in the disruption of homeostasis and ion distribution in the cell. Oxidative stress, which frequently<br><br>
accompanies high or low temperature, salinity or drought stress, may cause denaturing of functional and structural proteins. As a consequence, these diverse environmental stresses often activate similar cell signaling pathways and cellular responses, such as the production of stress proteins, up-regulation of anti-oxidants, accumulation of compatible solutes and growth<br>
arrest.<br>
Performance of the methods of the invention gives plants grown under mild drought conditions increased yield relative to suitable control plants grown under comparable conditions. Therefore, according to the present invention, there is provided a method for increasing yield in plants grown under mild drought conditions, which method comprises increasing expression in a plant of a nucleic acid encoding a DOF transcription factor polypeptide.<br>
The abovementioned improved growth characteristics may advantageously be improved in any plant. Insofar as the methods of the invention concern the use of AT-hook transcription factors, the methods are applicable to monocotyiedonous plants.<br>
The term "plant" as used herein encompasses whole plants, ancestors and progeny of the plants and plant parts, including seeds, shoots, stems, leaves, roots (including tubers), flowers, and tissues and organs, wherein each of the aforementioned comprise the gene/nucleic acid of interest or the genetic modification in the gene/nucleic acid of interest. The term "plant" also encompasses plant ceils, suspension cultures, callus tissue, embryos, meristematic regions, gametophytes, sporophytes, pollen and microspores, again wherein each of the aforementioned comprise the gene/nucleic acid of interest.<br>
Plants that are particulariy useful in the methods of the invention include all plants which belong to the superfamily Viridiplantae, in particular monocotyiedonous and dicotyledonous plants including fodder or forage legumes, ornamental plants, food crops, trees or shrubs selected from the list comprising Acer spp., Actinidia spp., Abelmoschus spp., Agropyron spp., Allium spp., Amaranthus spp.. Ananas comosus, Annona spp., Apium graveolens, Arachis spp, Artocarpus spp.. Asparagus officinalis., Avena spp. (e.g. Avena sativa., Avena fatua, Avena byzantina. Avena fatua var. sativa, Avena hybrida), Averrhoa carambola, aanincasa hispida, Bertholletia excelsea, Beta vulgaris, Brassica spp. (e.g. Brassica napus, Brassica rapa ssp. [canola, oilseed rape, turnip rape]), Cadaba farinosa, Camellia sinensis. Canna indica. Capsicum spp.., Carex elata, Carica papaya. Carissa m,acrocarpa, Cary-a spp., Carthamus tinctorius, Castanea spp., Cichohum endivia, Cinnamomum spp., Citrullus lanatus, Citrus spp., Cocos spp., Coffea spp., Colooasia esculenta. Cola spp., Coriandrum sativum, Coryius spp.. Crataegus spp., Crocus sativus, Cucurbita spp., Cucumis spp., Cynara spp.. Caucus carota.<br><br>
Desmodium spp., Dimocarpus longan, Dioscorea spp., Diospyros spp., Echinochloa spp., Elaeis (e.g. Elaeis guineensis, Elaeis oleifera), Eleusine coracana, Eriobotrya japonica, Eugenia uniflora, Fagopyrum spp., Fagus spp., Ficus carica, Fortunelia spp., Fragaria spp., Ginkgo biioba, Giycine spp. (e.g. Glycine max, Soja iiispida or Soja max), Gossypium hirsutum, Helianthus spp. (e.g. Helianthus annuus), Hemerocaliis fulva, Hibiscus spp., Hordeum spp. (e.g. Hordeum vulgare), Ipomoea batatas, Juglans spp., Lactuca sativa, Lathyrus spp.. Lens culinaris, Linum usitatissimum, Litchi chinensis, Lotus spp., Luffa acutangula, Lupinus spp., Luzula sylvatica, Lycopersicon spp. (e.g. Lycopersicon esculentum, Lycopersicon lycopersicum, Lycopersicon pyriforme), Macrotyloma spp., Ma/us spp., Malpighia emarginata, Mammea americana, Mangifera indica, Manitiot spp., Manilikara zapota, Medicago sativa, Melilotus spp., Mentha spp., Momordica spp., Moras nigra,- Musa spp., Nicotiana spp., Olea spp., Opuntia spp., Ornittiopus spp., Oryza spp. (e.g. Oryza sativa, Oryza latifolia), Panicum miliaceum, Passiflora edulis, Pastinaca sativa, Persea spp,, Petroselinum crispum, Ptiaseolus spp., Phoenix spp., Physalis spp., P/nus spp., Pistacia vera, Pisum spp., Poa spp., Populus spp., Prosopis spp., Prunus spp., Psidium spp., Punica granatum, Pyrus communis, Quercus spp., Raphanus sativus. Rheum rhabarbarum, Ribes spp., Ricinus communis, Rubus spp., Saccharuw spp., Sambucus spp., Secale cereale, Sesamum spp., Sinapis sp., S.olanum spp. (e.g. Solanum tuberosum, Solanum integrifolium or Solanum lycopersicum), Sorghum bicolor, Spinacia spp., Syzygium spp., Tagetes spp., Tamarindus indica, Theobroma cacao, Trifolium spp., Triticosecale rimpaui, Triticum spp. (e.g. Triticum aestivum, Triticum durum, Triticum turgidum, Triticum hybernum, Triticum macha, Triticum sativum or Triticum vulgare), Tropaeolum minus, Tropaeolum majus, Vaccinium spp., vicia spp., Vigna spp., V/o/a odorata, Vitis spp., Zea mays, Zizania palustris, Ziziphus spp., amongst others.<br>
Preferably, the plant is a crop plant such as soybean, sunflower, canola, alfalfa, rapeseed, cotton, tomato, potato or tobacco. Further preferably, the plant is a monocotyiedonous plant, such as sugarcane. More preferably the plant is a cereal, such as rice, maize, wheat, bariey, millet, rye, sorghum or oats.<br>
Where the methods of the invention concern use of an AT-hook transcription factor, the monocotyiedonous plant is a cereal, such as rice, maize, sugarcane, wheat, bariey, miliet, rye, sorghum, grasses or cats,<br>
DEFINITIONS<br>
Polypeptide<br>
The terms '^poiypeptide'^ and 'protein" are used interchangeably herein and refer to amino<br>
acids in  a  polymieric form  of any length.     The terms "polynucleotide(s)",  "nucleic acid<br>
23<br><br>
sequence(s)", "nucleotide sequence(s)" are used interchangeabiy herein and refer to nucleotides, either ribonucleotides or deoxyribonucleotides or a combination of both, in a polymeric form of any length.<br>
Contro! Plant<br>
The choice of suitable control plants is a routine part of an experimental setup and may include corresponding wild type plants or corresponding plants without the gene of interest. The control plant is typically of the same plant species or even of the same variety as the plant to be assessed. The control plant may also be a nullizygote of the plant to be assessed. A "control plant" as used herein refers not only to whole plants, but also to plant parts, including seeds and seed parts.<br>
Increase. Improve<br>
The terms "increase", "improving" or "improve" are used interchangeably herein and are taken<br>
to mean at least a 5%, 6%, 7%, 8%, 9% or 10%, preferably at least 15% or 20%, more<br>
preferably 25%, 30%, 35% or 40% more yield and/or growth in comparison to correspondiag	.   .<br>
wild type or other control plants as defined herein.<br>
Hybridisation<br>
The term "hybridisation" as defined herein is a process wherein substantially homologous complementary nucleotide sequences anneal to each other. The hybridisation process may occur entirely in solution, i.e. both complementary nucleic acids are in solution. The hybridisation process may also occur with one of the complementary nucleic acids immobilised to a matrix such as magnetic beads, Sepharose beads or any other resin. The hybridisation process may furthermore occur with one of the complementary nucleic acids immobilised to a solid support such as a nitro-celluiose or nylon membrane or immobilised by e.g. photolithography to, for example, a siliceous glass support (the latter known as nucleic acid arrays or microarrays or as nucleic acid chips). In order to allow hybridisation to occur, the nucleic acid molecules are generally thermally or chemically denatured to melt a double strand into two single strands and/or to remove hairpins or other secondary structures from single stranded nucleic acids. The stringency of hybridisation is influenced by conditions such as temperature, salt concentration, ionic strength and hybridisation buffer composition.<br>
"Stringent hybridisation conditions" and "stringent hybridisation wash conditions" in the context of nucieic acid hybridisation experiments such as Southern and Northern hybridisations are sequence dependent and are different under different environmental parameters.  The skilled<br><br>
artisan is aware of various parameters which may be altered during hybridisation and washing and which will either maintain or change the stringency conditions.<br>
The Tm is the temperature under defined ionic strength and pH, at which 50% of the target sequence hybridises to a perfectly matched probe. The To-, is dependent upon the solution conditions and the base composition and length of the probe. For example, longer sequences hybridise specifically at higher temperatures. The maximum rate of hybridisation is obtained from about 16OC up to 32°C below Tm. The presence of monovalent cations in the hybridisation solution reduce the electrostatic repulsion between the two nucleic acid strands thereby promoting hybrid formation; this effect is visible for sodium concentrations of up to 0.4M. Formamide reduces the melting temperature of DNA-DNA and DNA-RNA duplexes with 0,6 to 0.7°C for each percent formamide, and addition of 50% formamide allows hybridisation to be performed at 30 to 45°C, though the rate of hybridisation will be lowered. Base pair mismatches reduce the hybridisation rate and the thermal stability of the duplexes. On average and for large probes, the Tm decreases about 1 °C per % base mismatch. The Tm may be calculated using the following equations, depending on the types of hybrids;<br><br>
^ or for other monovalent cation, but only accurate in the 0.01-0.4 M range.<br>
* only accurate for %GC in the 30% to 75% range.<br>
^ L = length of duplex in base pairs.<br>
"^ Oligo, oligonucleotide; /„, effective length of primer = (no. of G/C)+(no. of A/T).<br>
Note: for each 1% formamide, the Tm is reduced by about 0.6 to 0.7C, while the presence of 6M urea reduces the Tm by about 30°C<br>
Specificity of hybridisation is typically the function of post-hybridisation vvashes. To remove background resulting from non-specific hybridisation, samples are washed with dilute salt solutions. Critical factors of such washes include the ionic strength and temperature of the final wash solution: the lower the salt concentration and the higher the vv-ash temperature, the higher the stringency of the wash.    Wash conditions are typically performed at or below<br><br>
■ hybridisation stringency. Generally, suitable stringent conditions for nucleic acid hybridisation assays or gene amplification detection procedures are as set forth above. More or less stringent conditions may also be selected. Generally, low stringency conditions are selected to be about 50°C lower than the thermal melting point (Tm) for the specific sequence at a defined ionic strength and pH. Medium stringency conditions are when the temperature is 20°C below Tm„ and high stringency conditions are when the temperature is 10°C below Tm. For example, stringent conditions are those that are at least as stringent as, for example, conditions A-L; and reduced stringency conditions are at least as stringent as, for example, conditions M-R. Non¬specific binding may be controlled using any one of a number of known techniques such as, for example, blocking the membrane with protein containing solutions, additions of heterologous RNA, DNA, and SDS to the hybridisation buffer, and treatment with Rnase.<br>
Examples of hybridisation and wash conditions are listed in Table 1;<br><br><br><br>
*	The "hybrid length" is the anticipated length for the hybridising nucleic acid. When nucleic<br>
acids of known sequence are hybridised, the hybrid length may be determined by aligning the<br>
sequences and identifying the conserved regions described herein.<br>
^ SSPE (1xSSPE is 0.15M NaCl, 10mM NaH2P04, and 1.25mM EDTA, pH7.4) may be substituted for SSC (1 xSSC is 0.15M NaCI and 15mM sodium citrate) in the hybridisation and wash buffers; washes are performed for 15 minutes after hybridisation is complete. The hybridisations and washes may additionally include 5 x Denhardt's reagent. 0.5-1.0% SDS, 100 µg/ml denatured, fragmented salmon sperm DNA, 0.5% sodium pyrophosphate, and up to 50% formamide.<br>
*	Tb-Tr: The hybridisation temperature for hybrids anticipated to be less than 50 base pairs in<br>
length should be 5-10°C less than the melting temperature Tm of the hybrids; the Tm is<br>
determined according to the above-mentioned equations.<br>
" The present invention also encompasses the substitution of any one, or more DNA or RNA hybrid partners with either a RNA, or a modified nucleic acid.<br>
For the purposes of defining the level of stringency, reference may conveniently be made to<br>
Sambrook et a!. (2001) Molecular Cloning: a laboratory manua!, 3'^ Edition Cold Spring Harbor<br><br>
Laboratory Press, CSH, New York or to Current Protocols in Molecular Biology, John Wiley &amp; Sons, N.Y, (1989).<br>
T-DNA Activation Tagging<br>
T-DNA activation tagging (Hayashi et al. Science (1992) 1350-1353) involves insertion of T-DNA, usually containing a promoter (may also be a translation enhancer or an intron), in the genomic region of the gene of interest or 10 kb up- or down stream of the coding region of a gene in a configuration such that the promoter directs expression of the targeted gene. Typically, regulation of expression of the targeted gene by its natural promoter is disrupted and the gene falls under the control of the newly introduced promoter. The promoter is typically embedded in a T-DNA. This T-DNA is randomly inserted into the plant genome, for example, through Agrobacterium infection and leads to overexpression of genes near the inserted T-DNA. The resulting transgenic plants show dominant phenotypes due to overexpression of genes close to the introduced promoter. The promoter to be introduced may be any promoter capable of directing expression of a gene in the desired organism, in this case a plant. For example, constitutive, tissue-preferred, cell type-preferred and inducible promoters are all suitable for use in T-DNA activation.<br>
TILLING<br>
TILLING (Targeted Induced Local Lesions In Genomes) is a mutagenesis technology useful to generate and/or identify and/or to eventually isolate mutagenised variant nucleic acids. TILLING also allows selection of plants carrying such mutant variants. These mutant variants may even exhibit higher activity than that exhibited by the gene in its natural form. TILLING combines high-density mutagenesis with high-throughput screening methods. The steps typically followed in TILLING are: (a) EMS mutagenesis (Redei GP and Koncz C (1992) In Methods in Arabidopsis Research, Koncz C, Chua NH, Schell J, eds. Singapore, Worid Scientific Publishing Co, pp. 16-82; Feidmann et aL, (1994) In Meyerowitz EM, Somerville CR, eds, Arabidopsis. Cold Spring Harbor Laboratory Press, Cold Spring Harbor, NY, pp 137-172; Lightner J and Caspar T (1998) In J Martinez-Zapater, J Salinas, eds. Methods on Molecular Biology, Vol. 82. Humana Press, Totowa, NJ, pp 91-104); (b) DNA preparation and pooling of individuals: (c) PCR amplification of a region of interest: (d) denaturation and annealing to allow formation of heteroduplexes; (e) DHPLC. where the presence of a heterodupiex in a pool Is detected as an extra peak in the chromatogram; (f) identification of the mutant Jndividual; and (g) sequencing of the mutant PCR product. Methods for TILLIhvJG are well known in the art (McCallum et al., (2000) Nat Biotechnol 18: 455-457; reviewed by Stemple (2004) Nat Rev Genet 5(2): 145-50),<br><br>
Site-DirectedMutagenesis<br>
Site-directed mutagenesis may be used to generate variants of SYR nucleic acids. Several methods are available to achieve site-directed mutagenesis; the most common being PCR based methods (Current Protocols in Molecular Biology. Wiley Eds. http://www.4ulr.com/products/currentprotocols/index.html).<br>
Transposon Mutagenesis<br>
Transposon mutagenesis is a mutagenesis technique based on the insertion of transposons in genes, which frequently results in gene-knockout. The technique has been used for several plant species, including rice (Greco et al., Plant Physiol, 125, 1175-1177, 2001), corn (McCarty et al.. Plant J. 44, 52-61, 2005) and Arabidopsis (Parinov and Sundaresan, Curr. Opin. Biotechnol. 11, 157-161, 2000).<br>
Directed Evolution<br>
Directed evolution or gene shuffling consists of iterations of DNA shuffling followed by appropriate screening and/or selection to generate variant nucleic acids or portions thereof, or polypeptides or homologues thereof having a modified biological activity (Castle et al., (2004) Science 304(5674): 1151-4; US patents 5,811,238 and 6,395,547).<br>
Homologous Recombination<br>
Homologous recombination allows introduction in a genome of a selected nucleic acid at a defined selected position. Homologous recombination is a standard technology used routinely in biological sciences for lower organisms such as yeast or the moss Physcomitrella. Methods for performing homologous recombination in plants have been described not only for model plants (Offringa et al. (1990) EMBO J 9(10): 3077-84) but also for crop plants, for example rice (Terada et al. (2002) Nat Biotech 20(10): 1030-4; lida and Terada (2004) Curr Opin Biotech 15(2): 132-8). Tne nucleic acid to be targeted (which may be any of the nucleic acids or variant defined .herein) needs to be targeted to the particular gene locus. The nucleic acid to be targeted may be an improved allele used to replace the endogenous gene or may be introduced in addition to the endogenous gene.<br>
Homologues<br>
"Homologues" of a protein encompass peptides, oligopeptides, polypeptides, proteins and enzymes having amino acid substitutions, deletions and/or insertions relative to the unmodified protein in question and having similar biological and functional activity as the unmodified protein fromwhich they are derived. To produce such homologues. amino acids of the protein may   be   replaced   by   other   amino   acids   having   similar   properties   (such   as   similar<br><br>
hydrophobicity, hydrophiiicity. antigenicity, propensity to form or break a-heiical structures or {3-sheet structures). Conservative substitution tables are well known in the art (see for example Creighton (1984) Proteins. W.H. Freeman and Company and Table 2 below).<br>
Ortholoques and Paralogues<br>
Encompassed by the term "homologues" are orthologous sequences and paralogous sequences, two special forms of homology which encompass evolutionary concepts used to describe ancestral relationships of genes.<br>
The term 'paralogous' relates to gene-duplications within the genome of a species leading to paralogous genes. Paralogues may easily be identified by performing a BLAST analysis against a set of sequences from the same species as the query sequence.<br>
The term "orthologous" relates to homologous genes in different organisms due to speciation. Orthologues in, for example, dicot plant species may easily be found by performing a so-called reciprocal blast search. This may be done by a first blast involving blasting a query sequence (for example, SEQ !D NO: 1 or SEQ ID NO: 2) against any sequence database, such as-the,. publicly available NCBI database which may be found at: http://www.ncbi.nlm.nih.gov. BLASTN or TBLASTX (using standard default values) may be used when starting from a nucleotide sequence and BLASTP or TBLASTN (using standard_ default values) may be used when starting from a protein sequence. The BLAST results may optionally be filtered. The full-length sequences of either the filtered results or non-filtered results are then BLASTed back (second BLAST) against sequences from the organism from which the query sequence is derived (where the query sequence is SEQ ID NO: 1 or SEQ ID NO: 2 the second blast would therefore be against 0/yza sativa sequences! The results of the first and second BLASTs are then compared. A paralogue is identified if a high-ranking hit from the second blast is from the same species as from which the query sequence is derived; an orthoiogue is identified if a high-ranking hit is not from the same species as from which the query sequence is derived. High-ranking hits are those having a low E-value. The lower the E-value, the more significant the score (or in other words the lower the probability that the hit was found by chance). Computation of the E-value is vv'ell known in the art. In the case of large families, CiustalW may be used, followed by a neighbour joining tree, to help visualize clustering of related genes and to identify orthologues and paralogues.<br>
A homologue may be in the form of a "substitutional variant" of a protein, i.e. where at least one residue In an amino acid sequence has been removed and a different residue inserted in f:s place.   Amino acid substitutions are typically of single residues, but may be clustered<br><br>
depending upon functional constraints placed upon the polypeptide; insertions will usually be of the order of about 1 to 10 amino acid residues. Preferably, amino acid substitutions comprise conservative amino acid substitutions. Less conserved substitutions may be made in case the above-mentioned amino acid properties are not so critical. Conservative substitution tables are readily available in the art. The table below gives examples of conserved amino acid substitutions.<br><br>
A homologue may also be in the form of an 'insertional variant" of a protein, i.e. v^here one or more amino acid residues are introduced into a predetermined site in a protein. Insertions may comprise N-terminal and/or C-terminal fusions as w'ell as intra-sequence insertions of single or multiple amino acids. Generally, insertions within the amino acid sequence will be smaller than N- or C-terminal fusions, of the order of about 1 to 10 residues. Examples of N- or C-termina! fusion proteins or peptides include the binding domain or activation domain of a transcriptional activator as used in the yeast two-hybrid system, phage coat proteins, (histidine)-6-tag, glutathione S-transferase-tag, protein A, maltose-binding protein, dihydrofolate reductase, Tag-100 epitope, c-myc epitope, FLAG®-epitope, lacZ, CMP (calmoduiin-binding peptide), HA epitope, protein C epitope and VSV epitope.<br>
Homologues in the form of "deletion variants' of a protein are characterised by the removal of one or more amino acids from a protein,<br>
Amino acid variants of a protein may readily be made using peptide synthetic techniques well<br>
knovv: in the art, such as solid phase peptide synthesis and the like, or by recombinant DNA manipulations.    Methods for the manipulation of DNA sequences to produce substitution.<br><br>
insertion or deletion variants of a protein are well known in the art. For example, techniques for making substitution mutations at predetermined sites in DNA are well known to those skilled in the art and include Ml3 mutagenesis, T7-Gen in vitro mutagenesis (USB, Cleveland, OH), QuickChange Site Directed mutagenesis (Stratagene, San Diego, CA}, PCR-mediated site-directed mutagenesis or other site-directed mutagenesis protocols.<br>
Derivatives<br>
"Derivatives" are polypeptides or proteins which may comprise naturally modified and/or non-naturally modified amino acid residues compared to the amino acid sequence of a naturally-occurring form (that is not having undergone post-translational modifications) of the protein, for example, as presented in SEQ ID NO: 2. "Derivatives" of a protein encompass polypeptides or proteins which may comprise naturally occurring altered, glycosylated, acylated, prenylated or non-naturaliy occurring amino acid residues compared to the amino acid sequence of a naturally-occurring form of the polypeptide. A derivative may also comprise one or more non-amino acid substituents compared to the amino acid sequence from which it is derived, for example a reporter molecule or other iigand, covalently or non-covalently bound to the amino acid sequence, such as a reporter molecule which is bound to facilitate its detection, and non-naturally occurring amino acid residues relative to the amino acid sequence of a naturally-occurring protein.<br>
Alternative Splice Variants<br>
The term "alternative splice variant" as used herein encompasses variants of a nucleic acid sequence in which selected introns and/or exons have been excised, replaced or added, or in which introns have been shortened or lengthened. Such variants will be,ones in which the biological activity of the protein is retained, which may be achieved by selectively retaining functional segments of the protein. Such splice variants may be found in nature or may be manmade. Methods for making such splice variants are known in the art.<br>
Allelic Variant<br>
Allelic variants exist in nature, and encompassed within the methods of the present invention is the use of these natural alleles. Allelic variants encompass Single Nucleotide Polymorphisms (SNPs). as well as Small Insertion/Deletion Polymorphisms (INDELs). The size of INDELs is usually less than 100 bp. SNPs and INDELs form the largest set of sequence variants in naturally occurring polymorphic strains of most organisms.<br><br>
Promoter<br>
The terms 'regulatory element", "control sequence" and "promoter" are all used interchangeably herein and are to be taken in a broad context to refer to regulatory nucleic acid sequences capable of effecting expression of the sequences to which they are ligated. Encompassed by the aforementioned terms are transcriptional regulatory sequences dehved from a classical eukaryotic genomic gene (including the TATA box which is required for accurate transchption initiation, with or without a CCAAT box sequence) and additional regulatory elements (i.e. upstream activating sequences, enhancers and silencers) which alter gene expression in response to developmental and/or external stimuli, or in a tissue-specific manner. Also included within the term is a transcriptional regulatory sequence of a classical prokaryotic gene, in which case it may include a -35 box sequence and/or -10 box transcriptional regulatory sequences. The term "regulatory element" also encompasses a synthetic fusion molecule or derivative that confers, activates or enhances expression of a nucleic acid molecule in a cell, tissue or organ. The term "operably linked" as used herein refers to a functional linkage between the promoter sequence and the gene of interest, such that the promoter sequence is able to initiatatranscription of the gene of interest.<br>
The promoter may be an inducible promoter, i.e. having induced or increased transcription initiation in response to a developmental, chemical, environmental or physical stimulus,<br>
A tissue-preferred or tissue-specific promoter is one that Is capable of preferentially initiating transcription in certain tissues, such as the leaves, roots, seed tissue etc, or even in specific ceils.<br>
The term "constitutive" as defined herein refers to a promoter that is expressed predominantly in at least one tissue or organ and predominantly at any life stage of the plant. Preferably the promoter is expressed predominantly throughout the plant.<br>
Examples of other constitutive promoters are shown in Table 3 below.<br><br><br><br><br><br><br><br><br><br><br>
The term "selectable marker gene" as referred to herein includes any gene that confers a phenotype on a cell in which it is expressed to facilitate the identification and/or selection of cells that are transfected or transformed with a nucleic acid construct of the invention. Suitable markers may be selected from markers that confer antibiotic or herbicide resistance, that introduce a new metabolic trait or that allow visual selection. Examples of selectable marker genes include genes conferring resistance to antibiotics (such as nptll that phosphorylates neomycin and kanamycin, or hpt, phosphorylating hygromycin), to herbicides (for example bar which provides resistance to Basta™; aroA or gox providing resistance against glyphosate), or genes that provide a metabolic trait (such as manA that allows plants to use mannose as sole carbon source). Visual marker genes result in the formation of colour (for example ^-glucuronidase, GUS), luminescence (such as luciferase) or fluorescence (Green Fluorescent Protein, GFP, and derivatives thereof).<br>
Transformation<br>
The term "transformation" as referred to herein encompasses the transfer of an exogenous polynucleotide into a host cell, irrespective of the method used for transfer. Plant tissue ■capable of subsequent clonal-propagation^whether by-organogenesis or embryogenesis, may be transformed with a genetic construct of the present invention and a whole plant regenerated from there. The particular tissue chosen will vary depending on the clonal propagation systems available for, and best suited to, the particular species being transformed. Exemplary-tissue targets include leaf disks, pollen, embryos, cotyledons, hypocotyis, megagametophytes. callus tissue, existing meristematic tissue (e.g., apical meristem, axillary buds, and root meristems), and induced meristem tissue (e.g., cotyledon meristem and hypocoty! meristem). The polynucleotide may be transiently or stably introduced into a host cell and may be maintained non-integrated, for example, as a plasmid. Alternatively, it may be integrated into the host genome. The resulting transformed plant cell may then be used to regenerate a transformed plant in a manner known to persons skilled in the art.<br>
Transformation of plant species is now a fairly routine technique. Advantageously, any of several transformation methods may be used to introduce the gene of interest into a suitable ancestor cell, Transform.ation methods include the use of liposomes, electroporation, chemicals that increase free DNA uptake, injection of the DNA directly into the plant, particle gun bombardment, transformation using viruses or pollen and microproiection. Methods may be selected from the calcium/polyethylene glycol method for protoplasts (Krens. F.A. et al.. (1932) Nature 296, 72-74; Negrutiu I et al. (1987) Plant Mol Biol 8; 363-373); electroporation of protoplasts (Shillito R.D. et al. (1985) Bio/Techno! 3, 1093-1102); microinjection into plant materia! (Crossway A et al., (1385) Mol. Gen Genet 202: 173-185); Dfsl,A or RNA-coated<br><br>
particle bombardment (Klein TM et al., (1987) Nature 327: 70) infection with (non-integrative) viruses and the like. Transgenic rice plants are preferably produced via Agrobacterium-mediated transformation using any of the well known methods for rice transformation, such as described in any of the following: published European patent application EP 1198985 Al, Aidemita and Hodges (Planta 199: 612-617, 1996); Chan et al. (Plant Mol Biol 22 (3): 491-505, 1993), Hiei et al. (Plant J 6 (2): 271-282, 1994), which disclosures are incorporated by reference herein as if fully set forth. In the case of corn transformation, the preferred method is as described in either ishida et al. (Nat. Biotechnol 14(5): 745-50, 1995) or Frame et al. (Plant Physiol 129(1): 13-22, 2002), which disclosures are incorporated by reference herein as if fully set forth.<br>
Generally after transformation, plant cells or cell groupings are selected for the presence of one or more markers which are encoded by plant-expressible genes co-transferred with the gene of interest, following which the transformed material is regenerated into a whole plant.<br>
Following DNA transfer and regeneration, putatively transformed plants may be evaluated, for instance using Southern analysis, for the presence .of the gene of interest, copy number and/.or _    _.   _ genomic organisation.  Alternatively or additionally, expression levels of the newly introduced DNA may be monitored using Northern and/or Western analysis, both techniques being well known to persons having ordinary skill in the art.<br>
The generated transformed plants may be propagated by a variety of means, such as by clonal propagation or classical breeding techniques. For example, a first generation (or T1) transformed plant may be selfed and homozygous second-generation (or T2) transformants selected, and the T2 plants may then further be propagated through classical breeding techniques.<br>
The generated transformed organisms may take a variety of forms. For example, they may be chimeras of transformed cells and non-transformed cells; clonal transform.ants (e.g.. all cells transformed to contain the expression cassette); grafts of transformed and untransformed tissues (e.g., in plants, a transformed rodtstock grafted to an untransformed scion).<br>
Detailed Description Seed Yield Regulator (SYR)<br>
The activity of a SYR protein may be increased by increasing levels of the SYR polypeptide. Alternatively, activity may also be increased when there is no change in levels of a SYR, or even when there is a reduction in levels of a SYR protein.   This may occur when the intrinsic<br><br>
properties of the polypeptide are altered, for example, by making a mutant or selecting a variant that is more active that the wild type.<br>
The term "SYR protein or homoiogue thereof as defined herein refers to a polypeptide of about 65 to about 200 amino acids, comprising (i) a leucine rich domain that resembles a leucine zipper in the C-terminal half of the protein, w/hich leucine rich domain is (ii) preceded by a tripeptide with the sequence YFS (conserved motif la, SEQ ID NO: 6), or YFT (conserved motif 1b, SEQ ID NO: 7), or YFG (conserved motif 1c, SEQ ID NO: 8) or YLG (conserved motif 1d. SEQ ID NO: 9), and (iii) followed by a conserved motif 2 ((V/A/!)LAFMP(T/S), SEQ ID NO: 10). Preferably, the conserved motif 2 is (AA/)LAFMP(T/S), most preferably, the conserved motif is VLAFMPT. The "SYR protein or homoiogue thereof preferably also has a conserved C-terminus peptide ending with the conserved motif 3 (SYL or PYL, SEQ ID NO: 11). The leucine rich domain of the SYR protein or its homoiogue is about 38 to 48 amino acids long, starting immediately behind the conserved motif 1 and stopping immediately before the conserved motif 2, and comprises at least 30% of leucine. The Leu rich domain preferably has a motif that resembles the Leucine Zipper motif (L-Xe-L-Xe-L-Xe-L, wherein Xe is a sequence of 5 consecutive.amino acids). A preferred example of-a SYR protein is represented-by.SEQ ID NO: 2, an overview of its domains is given in Figure 1. It should be noted that the term "SYR protein or homoiogue thereof does not encompass the ARGOS protein from Arabidopsis thaliana (SEQ ID NO: 26).<br>
Further preferably, SYR proteins have two transmembrane domains, with the N-terminal part and C-terminal part of the protein located inside and the part between the transmembrane domains located outside.<br><br><br><br><br><br>
Claims:<br>
1.	An isolated SYR protein selected from the group consisting of;<br>
(a)	a polypeptide as given in SEQ ID NO 44;<br>
(b)	a polypeptide with an amino acid sequence which has at least, in increasing order of preference, 85%, 85%, 87%, 88%, 89%, 90%, 91%, 92%, 93%, 94%, 95%, 96%, 97%, 98% or 99% or more sequence identity to the amino acid sequence as given in SEQ ID NO 44; <br>
(c)	a derivative of a polypeptide as defined in (a) or (b),<br>
2.	An isolated nucleic acid sequence comprising:<br>
(a)	a nucleic acid sequence represented by SEQ ID NO: 43, or the complement strand thereof;<br>
(b)	a nucleic acid sequence encoding the amino acid sequence represented by SEQ ID NO: 44;<br>
(c)	a nucleic acid sequence capable of hybridising (preferably under stringent conditions) with a nucleic acid sequence of (a) or (b), which hybridising sequence preferably encodes a SYR protein;<br>
(d)	a nucleic acid which is an allelic variant to the nucleic acid sequences according to (a) or (b);<br>
(e)	a nucleic acid which is an alternative splice variant to the nucleic acid sequences according to (a) or (b);<br>
(f)	a nucleic acid sequence which has 50%, 60%, 70%, 75%, 80%, 85%, 90%, 95%, 96%, 97%, 98% or 99% sequence identity to the sequence defined in (a) or (b).<br><br>
3.	Method for increasing seed yield and/or increasing growth rate of plants relative to corresponding wild type plant's, comprising modulating expression in a plant of a nucleic acid encoding a SYR polypeptide or a homologue thereof, and optionally selecting for plants having improved growth characteristics; provided that said SYR protein or homologue thereof is not the protein of SEQ ID NO: 26.<br>
4.	Method according to claim 3, wherein said modulated expression is effected by introducing a genetic modification preferably in the locus of a gene encoding a SYR polypeptide or a homologue thereof.<br><br>
5.	Method according to claim 4, wherein said genetic modification is effected by one of: T-DNA activation, TILLING, site-directed mutagenesis, homologous recombination or directed evolution.<br>
6.	Method for increasing seed yield and/or increasing growth rate relative to corresponding wild type plants, comprising introducing and expressing in a plant a SYR nucleic acid or a variant thereof; provided that said SYR protein or homologue thereof is not the protein of SEQ ID NO: 26.<br>
7.	Method according to claim 6, wherein said nucleic acid encodes a homologue of the SYR protein of SEQ ID NO: 2, preferably said nucleic acid encodes an orthologue or paralogue of the SYR protein of SEQ ID NO: 2.<br>
8.	Method according to claim 6, wherein said variant is a portion of a SYR nucleic acid or a sequence capable of hybridising to a SYR nucleic acid, which portion or hybridising sequence encodes a polypeptide of about 65 to about 200 amino acids, comprising a leucine rich domain, preceded by the conserved tripeptide motif 1 (one of SEQ ID NO: 6, 7, __ 8 or 9)) and followed by the conserved motif 2 (SEQ ID NO: 10) and preferably also by the conserved motif 3 (SEQ ID NO: 11).<br>
9.	Method according to any of claims 6 to 8, wherein said nucleic acid comprises the conserved motifs of SEQ ID NO: 6, SEQ ID NO: 10 and SEQ ID NO: 11, wherein the motif of SEQ ID NO: 10 is VLAFMPT and wherein the motif of SEQ ID NO: 11 is PYL, preferably wherein said nucleic acid comprises the sequence of SEQ ID NO: 1.<br>
10.	Method according to any of claims 6 to 9, wherein said SYR nucleic acid or variant thereof is overexpressed in a plant.<br>
11.	Method according to any one of claims 6 to 10, wherein said SYR nucleic acid or variant thereof is of plant origin, preferably from a monocotyledonous plant, more preferably from the family Poaceae, most preferably the nucleic acid is from Oryza sativa.<br>
12.	Method according to any one of claims 6 to 11, Vi'herein said SYR nucleic acid or variant thereof is operabiy linked to a constitutive promoter.<br>
13.	Method according to claim 12, wherein said constitutive promoter is a GOS2 promoter or a high mobility group protein promoter.<br><br>
14.	Method according to any one of claims 1 to 13, wherein said increased seed yield is selected from: increased total weight of seeds, increased number of filled seeds, seed fill rate or increased harvest index.<br>
15.	Method according to any of claims 1 to 14, wherein said increased growth rate comprises at least increased seed yield obtained without delay in flowering time.<br>
16.	Method according to any of claims 1 to 15, wherein said plants are grown under non-stress conditions.<br>
17.	Method according to any of claims 1 to 15, wherein said plants are grown under abiotic stress conditions.<br>
18.	Method of claim 17, wherein said abiotic stress conditions are conditions of osmotic stress.<br>
-19. Plant or plant cell obtainable by a method according to any one of claims 1 to 18.<br>
20.	Construct comprising:<br>
(i)  a SYR nucleic acid or a variant thereof;<br>
(ii) one or more control sequences capable of driving expression of the nucleic acid<br>
sequence of (i); and optionally (iii) a transcription termination sequence, provided that said SYR nucleic acid does not encode the protein of SEQ ID NO: 26.<br>
21.	Construct according to claim 20, wherein said control sequence is a constitutive promoter.<br>
22.	Construct according to claim 21, wherein said constitutive promoter is a G0S2 promoter or a High Mobility Group Protein (HMGP) promoter.<br>
23.	Construct according to claim 22, wherein said G0S2 promoter is as represented by SEQ ID NO: 5.<br>
24.	Construct according to claim 22, wherein said HMGP promoter is as represented by SEQ ID NO: 33.<br>
25.	Plant or plant cell transformed with a construct according to any one of claims 20 to 24,<br><br>
26.	Method for the production of a transgenic plant having increased yield compared to<br>
corresponding wild type plants, which method comphses:<br>
1.	introducing and expressing in a plant or plant cell a SYR nucleic acid or variant<br>
thereof; and<br>
2.	cultivating the plant cell under conditions promoting plant growth and development,<br>
with the proviso said SYR nucleic acid or variant thereof does not encode the protein of<br>
SEQ ID NO; 26.<br>
27.	Transgenic plant or plant cell having increased seed yield and/or increased growth rate<br>
resulting from a SYR nucleic acid or a variant thereof introduced into said plant, provided<br>
that said SYR nucleic acid or variant thereof does not encode the protein of SEQ ID NO:<br>
26.<br>
28.	Transgenic plant or plant cell according to claim 19, 25 or 27, wherein said plant is a monocotyledonous plant, such as sugarcane or wherein the plant is a cereal, such as rice, maize, wheat, barley,.millet, rye, oats or sorghum, or wherein said transgenic plant cell is    .    . denved from a monocotyledonous plant, such as sugar cane or wherein the plant is a cereal, such as rice, maize, wheat, bariey, millet, rye, oats or sorghum.<br>
29.	Harvestable parts of a plant according to any one of claims 19, 25, 27 or 28.<br>
30.	Harvestable parts of a plant according to claim 29 wherein said harvestable parts are seeds.<br>
31.	Products directly derived from a plant according to claim 27 and/or from harvestable parts of a plant according to claims 29 or 30.<br>
32.	Use of a SYR nucleic acid/gene or variant thereof, or use of a SYR polypeptide or a homologue thereof, in improving seed yield, relative to corresponding wild type plants.<br>
33.	Use according to claim 32, wherein said seed yield is one or more of: increased total weight of seeds, increased number of filled seeds or increased harvest index.<br>
34.	Use of a SYR nucleic acid/gene or variant thereof, or use of a SYR polypeptide or a homologue thereof, in improving resistance of plants to abiotic stress: relative to corresponding wild type plants.<br><br>
35.	Use of a SYR nucleic acid/gene or variant thereof, or use of a SYR polypeptide or a homologue thereof, as a molecular marker.<br>
36.	Method for improving growth characteristics of plants relative to control plants, comprising increasing expression in a plant of a nucleic acid encoding an FG-GAP polypeptide or a homologue thereof, and optionally selecting for plants having improved growth characteristics.<br>
37.	Method according to claim 36, wherein said increased expression is effected by introducing a genetic modification preferably in the locus of a gene encoding an FG-GAP polypeptide or a homologue thereof.<br>
38.	Method according to claim 37, wherein said genetic modification is effected by one of: T-DNA activation, TILLING, site-directed mutagenesis, homologous recombination or directed evolution.<br>
39.	Method for improving grov\/th characteristics relative to corresponding wild type plants, comprising introducing and expressing in a plant an FG-GAP nucleic acid or a variant thereof.<br>
40.	Method according to claim 39, wherein said nucleic acid encodes a homologue of the FG-GAP protein of SEQ ID NO: 46, preferably said nucleic acid encodes an orthologue or paralogue of the FG-GAP protein of SEQ ID NO: 46.<br>
41.	Method according to claim 39, wherein said variant is a portion of an FG-GAP nucleic acid or a sequence capable of hybridising to an FG-GAP nucleic acid, which portion or hybridising sequence encodes a polypeptide comprising a signal peptide, one or more FG-GAP domains and a transmembrane domain located in the C-terminal half of the protein.<br>
42.	Method according to any of claims 39 to 41, wherein said nucleic acid comprises one or more of the conserved motifs of SEQ ID NO: 50 to 52.<br>
43.	Method according to any of claims 39 to 42, wherein said FG-GAP nucleic acid or variant thereof is overexpressed in a plant.<br><br>
44,	Method according to any one of claims 39 to 43, wherein said FG-GAP nucleic acid or variant thereof is of plant origin, preferably from a dicotyledonous plant, more preferably from the family Brassicaceae, most preferably the nucleic acid is from Arabidopsis thaliana.<br>
45,	Method according to any one of claims 39 to 44, wherein said FG-GAP nucleic acid or variant thereof is operably linked to a constitutive promoter,<br>
46,	Method according to claim 45, wherein said constitutive promoter is a GOS2 promoter.<br>
47,	Method according to any one of claims 36 to 46, wherein said improved growth characteristic is increased yield.<br>
48,	Method according to claim 47, wherein said increased yield is increased seed yield.<br>
49,	Method according to claim 48, wherein said increased seed yield is selected from: increased total weight of seeds, increased number of filled seeds or increased harvest index.<br>
50,	Plant, plant part or plant cell obtainable by a method according to any one of claims 36 to<br>
49,<br>
51,	Construct comprising:<br>
(a)	an FG-GAP nucleic acid or a variant thereof;<br>
(b)	one or more control sequences capable of driving expression of the nucleic acid sequence of (a); and optionally<br>
(c)	a transcription termination sequence, .<br>
provided that said construct is not a pPZP-type gene construct.<br>
52,	Construct according to claim 51, wherein said control sequence is a constitutive promoter,<br>
53,	Construct according to claim 52, wherein said constitutive promoter is a G0S2 promoter.<br>
54,	Construct according to claim 53, wherein said G0S2 promoter is as represented by nucleotides 1 to 2183 in SEQ ID NO: 49,<br>
. 55, Plant, plant part or plant cell transformed with a construct according to any one of claims 51 to 54,<br><br>
55. Method for the production of a transgenic plant having increased yield compared to corresponding wild type plants, which method comprises:<br>
(a)	introducing and expressing in a plant or plant cell an FG-GAP nucleic acid or variant thereof;<br>
(b)	cultivating the plant cell under conditions promoting plant growth and development.<br><br>
57.	Transgenic plant, plant part or plant ceil having improved growth characteristics resulting from an FG-GAP nucleic acid or a variant thereof introduced into said plant.<br>
58.	Transgenic plant, plant part or plant cell according to claim 50, 55 or 57, wherein said plant is a monocotyiedonous plant, such as sugarcane or wherein the plant is a cereal, such as rice, maize, wheat, barley, millet, rye, oats or sorghum, or wherein said transgenic plant cell is derived from a monocotyiedonous plant, such as sugarcane or wherein the plant is a cereal, such as rice, maize, wheat, barley, millet, rye, oats or sorghum.<br>
59.	Harvestable parts of a plant according to any one of claims 50, 55, 57 or 58.	. .<br>
60.	Harvestable parts of a plant according to claim 59 wherein said harvestable parts are seeds.<br>
61.	Products derived from a plant according to claim 58 and/or from harvestable parts of a plant according to claims 59 or 60.<br>
62.	Use of an FG-GAP nucleic acid/gene or variant thereof, or use of an FG-GAP polypeptide or a homologue thereof, in improving growth characteristics of plants, preferably in improving yield, especially seed yield, relative to corresponding wild type plants.<br><br>
53.	Use of a construct according to any one of claims 51 to 54 in improving growth characteristics of plants, preferably in improving yield, especially seed yield, relative to corresponding wild type plants.<br>
54.	Use according to claim 62 or 63, wherein said seed yield is one or more of: increased total weight of seeds, increased number of filled seeds or increased harvest index.<br>
65. Use of an FG-GAP nucleic acid/gene or variant thereof, or use of an FG-GAP polypeptide or a homologue thereof, as a molecular marker.<br><br>
66.	Isolated FG-GAP protein selected from the group consisting of;<br>
(a)	a protein encoded by the nucleic acid of SEQ ID NO: 72;<br>
(b)	a protein comprising a signal sequence, one or more FG-GAP domains and a transmembrane domain located in the C-terminal half of the protein, wherein said protein comprises at least one of SEQ ID NO: 73 to SEQ ID NO: 76;<br>
(c)	an active fragment of an amino acid sequence as defined in (a) or (b), which active fragment comprises a signal sequence, one or more FG-GAP domains and a transmembrane domain located in the C-terminal half of the protein.<br>
67.	Isolated nucleic acid encoding an FG-GAP protein, selected from the group consisting of;<br>
(i)        the nucleic acid as represented in SEQ ID NO: 72;<br>
(ii)       a nucleic acid encoding a protein as defined in (a) to (c) above;<br>
(iii) a nucleic acid sequence capable of hybridising (preferably under stringent conditions) with a nucleic acid sequence of (i) or (ii) above, which hybridising sequence preferably encodes a protein comprising a signal peptide, one or more FG-GAP domains and a transmembrane domain located in the C-terminal,half of the proteiri;<br>
(iv) a nucleic acid which is an allelic variant to the nucleic acid sequences according to (i) to (iii);<br>
(v) a nucleic acid which is an alternative splice variant to the nucleic acid sequences according to (i) to (iii);<br>
(vi) a portion of a nucleic acid sequence according to any of (!) to (v) above, which portion preferably encodes a protein comprising a signal peptide, one or more FG-GAP domains and a transmembrane domain located in the C-terminal half of the protein.<br>
58.	An isolated CYP90B protein selected from the group consisting of:<br>
(i)        a protein encoded by the nucleic acid of SEQ ID NO: 117;<br>
(ii) a protein comprising comprising the following: (i) CYP domains A to D; (ii) an N-terminal hydrophobic anchor domain; (iii) a transition domain; and (iv) within the A domain, the consensus sequence Phe-Ala-Gly-His-Glu-Thr-Ser-Ser. allowing for one amino acid change at any position, and having in increasing order of preference at least 85%, 85%, 87%, 88%, 89%. 90%, 91%. 92%, 93%, 94%, 95%. 96%, 97%, 98%, 99% identity to the amino acid sequence of SEQ ID NO: 118.<br>
59.	An isolated nucleic acid encoding a CYP90B protein, selected from the group consisting of:<br><br>
(i)        a nucleic acid as represented by SEQ ID HO: 117<br>
(ii)       a nucleic acid encoding a protein as defined in (i) and (ii) of claim 68;<br>
(iii) a nucleic acid having in increasing order of preference at least 82%, 83%, 84%, 85%, 86%, 87%, 88%, 89%, 90%, 91%. 92%, 93%, 94%, 95%. 96%, 97%, 98%, 99% or more identity to the nucleic acid represented by SEQ ID NO: 117;<br>
(iv) a nucleic acid sequence capable of hybridising under stringent conditions with a nucleic acid sequence of (i) to (iii) above, which hybridising sequence encodes a protein comprising (a) CYP domains A to D; (b) an N-terminal hydrophobic anchor domain; (c) a transition domain; and (d) within the A domain, the consensus sequence Phe-Ala-Gly-His-Glu-Thr-Ser-Ser, allowing for one amino acid change at any position, and having in increasing order of preference at least 85%i, 86%., 87%, 88%, 89%, 90%, 91%, 92%, 93%, 94%, 95%, 96%, 97%, 98%, 99% or more to the amino acid sequence of SEQ ID NO: 118;<br>
(v) a nucleic acid which is an allelic variant or a splice variant of the nucleic acid sequences according to (!) to (iv);<br>
(vi)      a portion of a nucleic acid sequence according to any of (i) to (v) above, which<br>
- portion encodes.a protein comprising: (i) CYP domains A to D; (ii) an N-terminal __ ,. . hydrophobic anchor domain; (iii) a transition domain; and (iv) within the A domain, the consensus sequence Phe-Ala-Gly-His-Glu-Thr-Ser-Ser, allowing for one amino acid change at any position, and having in increasing order of preference at least 85%, 86%, 87%, 88%, 89%, 90%, 91%, 92%, 93%, 94%, 95%, 96%, 97%, 98%, 99% to the amino acid sequence of SEQ ID NO: 118.<br>
70.	Method for increasing plant yield relative to suitable control plants comprising increasing non-constitutive expression in a plant of a nucleic acid encoding nucleic acid encoding a cytochrome P450 (CYP) monooxygenase CYP90B or a homologue thereof, and optionally selecting for plants having increased yield, wherein said CYP90B polypeptide or homologue comphses the following: (a) CYP domains A to D; (b) an N-terminal hydrophobic anchor domain; (c) a transition domain; and (d) within the A domain, the consensus sequence Phe-Ala-Gly-His-Glu-Thr-Ser-Ser, allowing for one amino acid change at any position.<br>
71.	Method according to claim 70, wherein said CYP90B polypeptide or homologue thereof additionally comprises: (i) a sequence with more than 50% identity to SEQ ID NO: 78; and (ii) steroid 22-alpha hydroxylase enzymatic activity.<br><br>
72.	Method according to claim 70 or 71, wherein said increased non-constitutive expression is effected by introducing a genetic modification preferably in the locus of a gene encoding a CYP90B polypeptide or a homologue thereof,<br>
73.	Method according to claim 72, wherein said genetic modification is effected by one of; T-DNA activation, TILLING, site-directed mutagenesis or directed evolution.<br>
74.	Method for increasing plant yield relative to suitable control plants comprising introducing and expressing non-constitutively in a plant a CYP90B nucleic acid or a variant thereof.<br>
75.	Method according to claim 74, wherein said variant is a portion of a CYP90B nucleic acid, which portion encodes a polypeptide comprising: (a) CYP domains A to D; (b) an N-terminal hydrophobic anchor domain; (c) a transition domain; and (d) within the A domain, the consensus sequence Phe-Ala-Gly-His-Glu-Thr-Ser-Ser, allowing for one amino acid change at any position.<br>
75. Method according to claim 74, wherein said variant is a sequence capable of hybridising to a CYP90B nucleic acid, which hybhdising sequence encodes a polypeptide comprising: (a) CYP domains A to D; (b) an N-terminal hydrophobic anchor domain; (c) a transition domain; and (d) within the A domain, the consensus sequence Phe-Ala-Gly-His-Glu-Thr-Ser-Ser, allowing for one amino acid change at any position.<br>
77.	Method according to any one of claims 74 to 76, wherein said CYP90B nucleic acid or variant thereof is of plant origin, preferably from a monocotyledon plant, further preferably from the family Poaceae, more preferably from the genus Oryza, most preferably from Oryza sativa.<br>
78.	Method according to any one of claims 74 to 77, wherein said variant encodes an orthologue or paralogue of the CYP90B protein of SEQ ID NO: 78.<br>
79.	Method according to any one of claims 74 to 78, wherein said CYP90B nucleic acid or variant thereof is operabty linked to a non-constitutive promoter.<br>
30. Method according to claim 79, wherein said non-constitutive promoter is a seed specific promoter,  preferably said seed-specific promoter is an endosperm-specific promoter.<br>
further preferably the endosperm-specific promoter is a prolamin promoter.<br><br>
81.	Method according to claim 80, wherein said endosperm-specific promoter is a rice RP5 proiamln promoter, more preferably the endosperm-specific promoter is represented by a nucleic acid sequence substantially similar to SEQ ID NO: 109, most preferably the endosperm-specific promoter is as represented by SEQ ID NO: 109.<br>
82.	Method according to any one of claims 70 to 81, wherein said increased yield is selected from one or both of: increased total seed yield or increased harvest index (HI), each relative to suitable control plants.<br>
83.	Method according to claim 79, wherein said non-constitutive promoter is a seed specific promoter, preferably said seed-specific promoter is an embryo/aleurone-specific promoter, further preferably the embryo/aleurone-specific promoter is an oleosin promoter.<br>
84.	Method according to claim 83, wherein said embryo/aleurone-specific promoter is a rice oleosin 18 kDa promoter, more preferably the embryo/aleurone-specific promoter is represented by a nucleic add sequence substantially similar to SEQ ID NO: 110, most preferably the embryo/aleurone-specific.promoter is as. represented by SEQ ID NO: 110. .	_<br>
85.	Method according to any one of claims 70 to 79, 83 or 84, wherein said increased yield is selected from one or more of: increased thousand kernel weight (TKW), increased seed area or increased seed length, each relative to suitable control plants.<br>
86.	Method according to any one of claims 70 to 85, wherein said plant is a monocotyledonous<br>
plant.<br>
87.	Plant, plant part or plant cell obtainable by a method according to any one of claims 70 to<br>
86.<br>
88.	Use of a construct comphsing:<br>
(i)        a CYP90B nucleic acid or variant thereof, as defined hereinabove;<br>
(ii)       one or more control sequences capable of driving non-constitutive expression of the<br>
nucleic acid sequence of (i); and optionally (iii)       a transcription termination sequence in a method according to any one of claims 74 to 35.<br>
39. Use according to claim 88, wherein said control sequence is a non-constitutive promoter.<br><br>
90.	Use according to claim 89, wherein said non-constitutive promoter is a seed specific promoter, preferably said seed-specific promoter is an endosperm-specific promoter, further preferably the endosperm-specific promoter is a prolamin promoter.<br>
91.	Use according to claim 90, wherein said endosperm-specific promoter is a rice RP6 prolamin promoter, more preferably the endosperm-specific promoter is represented by a nucleic acid sequence substantially similar to SEQ ID NO: 109, most preferably the endosperm-specific promoter is as represented by SEQ ID NO: 109.<br>
92.	Plant, plant part or plant cell transformed with a construct comprising a CYP90B nucleic acid or variant thereof under the control of a RP6 prolamin promoter.<br>
93.	Use according to claim 89, wherein said non-constitutive prompter is a seed specific promoter, preferably said seed-specific promoter is an embryo/aleurone-specific promoter, further preferably the embryo/aleurone-specific promoter is an oleosin promoter.<br>
-94. Use according to. claim 93, wherein said embryo/aleurone-specific promoter .is a rice .    .. oleosin  18 kDa  promoter,  more preferably the embryo/aleurone-specific promoter is represented by a nucleic acid sequence substantially similar to SEQ ID NO: 110, most preferably the embryo/aleurone-specific promoter is as represented by SEQ ID NO: 110.<br>
95.	Plant, plant part or plant cell transformed with a construct comprising a CYP90B nucleic acid or variant thereof under the control of an oleosin 18kDa promoter.<br>
96.	Method for the production of a transgenic plant having increased yield relative suitable control plant, which method comprises:<br><br>
(a)	introducing and expressing non-constitutively in a plant or plant cell a CYP90B nucleic acid or variant thereof;<br>
(b)	cultivating   the   plant   cell   under   conditions   promoting   plant   growth   and development.<br>
97.	Transgenic plant having increased yield relative to suitable control plant, said increased<br>
yield resulting from a CYP90B nucleic acid or a variant thereof introduced and expressed<br>
non-constitutively into said plant.<br><br>
1   "<br>
98.	Transgenic plant according to any one of claims 87, 92, 95 or 97, wherein said plant is a monocotyiedonous plant, such as sugarcane or wherein the plant is a cereal, such as rice, maize, wheat, barley, millet, rye, oats or sorghum.<br>
99.	Harvestable parts of a plant according to any one of claims 87, 92, 95, 97 or 98.<br><br>
100.	Harvestable parts of a plant according to claim 99, wherein said harvestable parts are seeds.<br>
101.	Products derived from a plant according to claim 98 and/or from harvestable parts of a plant according to claims 99 or 100.<br>
102.	Use of a construct according to claim 90 or 91 in increasing plant yield relative to suitable control plants.<br>
103.	Use according to claim 102, wherein said increased yield is selected from one or both<br>
.of:.increased total seed yield orjncreased HI, each relative to suitable control plants..	....<br>
104.	Use of a construct according to claim 93 or 94 in increasing plant yield relative to suitable control plants.<br>
105.	Use according to claim 104, wherein said increased yield is selected from one or more of: increased TKW, increased average seed area or increased average seed length, each relative to suitable control plants.<br>
106.	Method for increasing seed number in plants relative to suitable control plants, comprising preferentially increasing expression in the shoot apical meristem tissue of a plant of a nucleic acid encoding a CDC27 polypeptide having at least one inactive TPR domain in the NH2 terminal region of the polypeptide, and optionally selecting for plants having increased seed number.<br>
107.	Method according to claim 106, wherein said increased expression is effected by introducing a genetic modification preferably in the locus of a gene encoding a CDC27 polypeptide having at least one inactive TPR domain in the NH,2 terminal region of the polypeptide.<br><br>
108.	Method according to claim 107, wherein said genetic modification is effected site-directed mutagenesis or directed evolution.<br>
109.	Method for increasing seed number in plants relative to suitable control plants. comprising introducing and preferentially expressing in the shoot apical meristem tissue of a plant a nucleic acid encoding a CDC27 polypeptide having at least one inactive TPR domain in the NH2 terminal region of the polypeptide.<br>
110.	Method according to claim 109,wherein said nucleic acid introduced is a splice variant or an allelic vahant of a nucleic acid represented by SEQ ID NO: 129 or SEQ ID NO: 131, which splice variant or allelic variant encodes a CDC27 polypeptide having at least one inactive TPR domain in the NH2 terminal region of the polypeptide.<br>
111.	Method according to claim 109 or 110, wherein said nucleic acid introduced is capable of hybridizing to a nucleic acid as represented by SEQ ID NO: 129 or SEQ ID NO: 131 or to a splice variant or allelic variant according to claim  110, wherein said hybridizing<br>
-sequence.encodes a CDC27 polypeptide having at least one inactive TPR docnain-in the.. . NH2 terminal region of the polypeptide.<br>
112.	Method according to any one of claims 106 to 111, wherein said CDC27 nucleic acid encoding a CDC27 polypeptide having at least one inactive TPR domain in the NH2 terminal region of the polypeptide, or the polypeptide itself, is of plant origin, preferably from a dicotyledonous plant, preferably from the family Brassicaceae, further preferably from Arabidopsis thaliana.<br>
113.	Method according to any one of claims 106 to 112, wherein said CDC27 nucleic acid encoding a CDC27 polypeptide having at least one inactive TPR domain in the NH2 terminal region of the polypeptide encodes an orthologue or paralogue of a CDC27 polypeptide represented by SEQ ID NO: 130 or SEQ ID NO: 132.<br>
114.	Method according to any one of claims 109 to 113, wherein said CDC27 nucleic acid encoding a GDC27 polypeptide having at least one inactive TPR domain in the NH2 terminal region of the polypeptide is operably linked to a shoot apical meristem promoter, preferably to an early shoot apical meristem promoter.<br>
115.	Method according to claim 114, wherein said shoot apical meristem promoter is an 0SH1 promoter.<br><br>
106 to 115.<br>
117.	Use of a construct comprising:<br>
(i) a CDC27 nucleic acid encoding a CDC27 polypeptide having at least one inactive TPR domain in the NH2 terminal region of the polypeptide; and<br>
(ii) one or more control sequences capable of preferentially increasing expression of the nucleic acid sequence of (i) in shoot apical meristem tissue of a plant; and optionally<br>
(iii)       a transcnption termination sequence<br>
in a method according to any one of claims 109 to 115.<br>
118.	Use of a construct according to claim 117, wherein said control sequence is an 0SH1 promoter.<br>
119.	Plant, plant.part or plant cell.transformed with.a construct according to claim. 11.7 or 118.<br>
120.	Method for the production of a transgenic plant having increased seed number relative to suitable control plants, which method comprises:<br><br>
(a)	introducing and expressing in a plant a CDC27 nucleic acid encoding a CDC27 polypeptide having at least one inactive TPR domain in the NH2 terminal region of the polypeptide operably linked to an shoot apical meristem-specific promoter; and<br>
(b)	cultivating the plant cell under conditions promoting plant growth and development.<br><br>
121.	Transgenic plant having increased seed number relative to suitable control plants, said increased seed number resulting from a CDC27 nucleic acid encoding a CDC27 polypeptide having at least one inactive TPR domain in the NH2 terminal region of the polypeptide which nucleic acid is operably linked to an apical shoot meristem-specific promoter.<br>
122.	Transgenic plant according to claim 116, 119 or 121, wherein said plant is a monocotyledonous plant, such as sugarcane or wherein the plant is a cereal, such as rice, maize, wheat, barley, millet, rye, oats or sorghum.<br><br>
123.	Harvestable parts of a plant according to any one of claims 116, 119, 121 or 122,<br>
124.	Han/estabie parts according to claim 123, wherein said harvestable parts are seeds.<br>
125.	Products derived from a plant according to claim 122 and/or from harvestable parts of a plant according to claim 123 or 124.<br>
126.	Use of a CDC27 nucleic acid encoding a CDC27 polypeptide having at least one inactive TPR domain in the NH2 terminal region of the polypeptide operabiy linked to an apical shoot meristem-specific promoter, or use of such polypeptide, in increasing plant seed number relative to suitable control plants.<br>
127.	Method for increasing seed yield in a monocotyledonous plant relative to the seed yield of suitable control plants, comprising preferentially increasing expression in endosperm tissue of a monocotyledonous plant of a nucleic acid encoding a polypeptide comprising an AT-iiookdomain.and a DUF296 domain.<br>
128.	Method according to claim 127, wherein said polypeptide further comprises one of the following motifs; Motif 1; QGQ V/l GG; or Motif 2: ILSLSGSFLPPPAPP; or Motif 3; NATYERLP; or Motif 4; SFTNVAYERLPL with zero or one amino acid change at any position; or Motif 5; GRFEILSLTGSFLPGPAPPGSTGLTIYLAGGQGQWGGSWG with zero, one or two amino acid changes at any position.<br>
129.	Method according to claim 127 or 128, wherein said increased expression is effected by introducing a genetic modification preferably in the locus of a gene encoding a polypeptide comprising an AT-hook domain and a DL)F295 domain.<br>
130.	Method according to claim 129, wherein said genetic modification is effected by one or more of; T-DNA activation, TILLING and homologous recombination.<br>
131.	Method for increasing seed yield in a monocotyledonous plant relative to the seed yield of suitable control plants, comprising introducing and preferentially expressing in endosperm tissue of a monocotyledonous plant of a nucleic acid encoding a polypeptide comprising an AT-hook domain and a DUF296 domain.<br><br>
132.	Method according to claim 131, wherein said polypeptide further comprises one of the following motifs: Motif 1: QGQ V/l GG; or Motif 2: ILSLSGSFLPPPAPP; or Motif 3: NATYERLP; or Motif 4: SFTNVAYERLPL with zero or one amino acid change at any position; or Motif 5: GRFEILSLTGSFLPGPAPPGSTGLTIYLAGGQGQWGGSWG with zero, one or two amino acid changes at any position.<br>
133.	Method according to claim 131 or 132, wherein said nucleic acid is operabiy linked to an endosperm-specific promoter, preferably to a prolamin promoter.<br>
134.	Method according to any one of claims 127 to 133, wherein said nucleic acid is a portion or a an allelic variant or a splice variant or a sequence capable of hybridising to a sequence according to any one of SEQ ID NO: 152, SEQ ID NO: 153, SEQ ID NO; 154, SEQ ID NO: 156, SEQ ID NO: 158, SEQ ID NO: 160, SEQ ID NO: 162, SEQ ID NO: 154, SEQ ID NO: 166, SEQ ID NO: 168 and SEQ ID NO: 170, wherein said portion, allelic vahant, splice variant or hybridising sequence encodes a polypeptide comprising an AT-hook domain and a DUF296 domain.<br>
135.	Method according to claim 134, wherein said portion, allelic variant, splice variant or hybridising sequence encodes an orthoiogue or paralogue of the AT-hook protein of SEQ ID NO: 153.<br>
136.	Method according to any one of claims 127 to 135, wherein said nucleic acid encoding a polypeptide comprising an AT-hook domain and a DUF296 domain is of plant origin, preferably from a monocotyledonous plant, further preferably from the family Poaceae, more preferably from the genus Oryza, most preferably from Oryza satrva.<br>
137.	Method according to any one of claims 127 to 136, wherein said increased yield is selected from one or more of: increased total seed weight, increased number of filled seeds, increased total number of seeds increased number of flowers per panicle, increased harvest index (HI).<br>
13B. Plant or part thereof including plant cell obtainable by a method according to any one of claims 127 to 137, wherein said plant or part thereof comprises a nucleic acid encoding a polypeptide comprising an AT-hook domain and a DUF296 domain which nucleic acid is operabiy linked to an endosperm-specific promoter.<br>
139,    Gene construct comprising:<br><br>
(a)	A nucleic acid encoding a polypeptide comprising an AT-iiook domain and a DUF296 domain;<br>
(b)	One or more control sequences capable of driving expression of the nucleic acid sequence of (i) in endosperm tissue of a monocotyledonous plant; and optionally<br>
(c)	A transcription termination sequence.<br><br>
140.	Use of a construct according to claim 139 for increasing seed yield in monocotyledonous plants.<br>
141.	Construct according to claim 139, wherein said control sequence is a prolamin promoter.<br>
142.	Plant, plant part or plant cell transformed with a construct according to claim 139.<br>
143.	Method for the production of a transgenic monocotyledonous plant having increased seed yield relative to suitable control plants which method comprises:<br><br>
(a)	introducing and preferentially increasing expression in endosperm tissue of a monocotyledonous plant of a nucleic acid encoding a polypeptide comprising an AT-hook domain and a DUF296 domain; and<br>
(b)	cultivating the plant cell under conditions promoting plant growth and development.<br><br>
144.	Transgenic monocotyledonous plant having increased seed yield relative to suitable control plants, said increased seed yield resulting from preferential expression in endosperm tissue of a monocotyledonous plant of a nucleic acid encoding a polypeptide comprising an AT-hook domain and a DUF296 domain.<br>
145.	Transgenic monocotyledonous plant according to claim 138, 142 or 144, wherein said plant is a cereal, such as rice, maize, sugarcane, wheat, barley, millet, rye, sorghum, grasses or oats.<br>
146.	Harvestable parts of a plant according to any one of claims 138, 142. 144 or 145, wherein said harvestable parts are seeds.<br>
147.	Products derived from a plant according to claim 145 and/or from harvestable parts of a plant according to claim 146.<br><br>
148.	Use of a nucleic acid encoding a polypeptide comprising an AT-hook domain and a DUF296 domain, which nucleic acid is operably linked to an endosperm-specific promoter, in increasing seed yield in a monocotyledonous plant compared to seed yield in a suitable control plant.<br>
149.	Use according to claim 148, wherein said increased seed yield is selected from one or more of: increased total seed weight, increased number of filled seeds, increased total number of seeds increased number of flowers per panicle, increased harvest index (HI).<br>
150.	Method for increasing plant yield relative to suitable control plants, comprising increasing expression in a plant of a nucleic acid encoding a DOF (DNA-binding with one finger) domain transcription factor polypeptide comprising feature (i) as follows, and additionally either feature (ii) or (lii) as follow:<br>
(i)        at least 60% sequence identity to either the DOF domain represented by SEQ ID<br>
NO: 200 or SEQ ID NO: 228; and (ii)       at least 70% sequence identity to the DOF domain represented by SEQ ID NO:<br>
200; or (ill)       Motif I: KALKKPDKILP (SEQ ID NO: 229) with no changes; or with one or more<br>
conservative change at any position; or with one, two or three non-conservative<br>
change(s) at any position; and/or Motif II: DDPGiKLFGKTIPF (SEQ ID NO: 230) with no changes; or with one or more conservative change at any position; or with one, two or three non-conservative change(s) at any position.<br>
151.	Method according to claim 150, wherein said polypeptide comprising feature (i) and<br>
feature (iii) further comprises any one, any two or all three of the following motifs:<br>
-	Motif III: SPTLGKHSRDE (SEQ ID NO: 231) with no changes; or with one or more conservative change at any position; or with one, two or three non-conservative change(s) at any position: and/or<br>
-	Motif (V: LQANPAALSRSQNFQE (SEQ ID NO: 232) with no changes; or with one or more conservative change at any position; or with one, two or three non-conservative change(s) at any position; and,/or<br>
-	Motif V: KGEGCLWVPKTLRIDDPDEAAKSSIWTTLGIK (SEQ ID NO: 233) with no changes; or with one or more conservative change at any position; or with one, two, three, four or five non-conservative change(s) at any position.<br><br>
152.	Method according to claim 150 or 151, wherein said polypeptide comprising feature (i) and feature (iii) comprises both Motif I and II.<br>
153.	Method according to any one of claims 150 to 152, wherein said increased expression is effected by introducing a genetic modification preferably in the locus of a gene encoding a DOF transcription factor polypeptide.<br>
154.	Method according to claim 153, wherein said genetic modification is effected by one of: T-DNA activation, TILLING and homologous recombination.<br>
155.	Method for increasing plant yield relative to suitable control plants, comprising introducing and expressing in a plant a nucleic acid or variant thereof encoding a DOF transcription factor polypeptide comprising feature (i) as follows, and additionally either feature (ii) or (iii) as follow:<br>
(i)        at least 60% sequence identity to either the DOF domain represented by SEQ ID<br>
NO: 200 or SEQ ID NO: 228; and<br>
(ii)        at least 70% sequence identity to the DOF domain represented by SEQ ID NO:     	<br>
200; or (iii)       Motif I: KALKKPDKILP (SEQ ID NO: 229) with no changes; or with one or more<br>
conservative change at any position; or with one, two or three non-conservative<br>
change(s) at any position; and/or Motif II: DDPGIKLFGKTIPF (SEQ ID NO: 230) with no changes; or with one or more conservative change at any position; or with one, two or three non-conservative change(s) at any position.<br>
156.	Method according to claim 155, wherein said polypeptide comprising feature (i) and<br>
feature (iii) further comprises any one, any two or all three of the following motifs:<br>
-	Motif III: SPTLGKHSRDE (SEQ ID NO: 231) with no changes; or with one or more conservative change at any position; or with one, two or three non-conservative change(s) at any position; and/or<br>
-	Motif !V: LQANPAALSRSQNFQE (SEQ ID NO: 232) with no changes; or with one or more conservative change at any position; or with one, two or three non-conserv'ative change(s) at any position; and/or<br>
-	Motif V: KGEGCLWVPKTLRlDDPDEAAKSSIWTTLGlK (SEQ ID NO: 233) with no changes; or with one or more conservative change at any position; or with one, two, three, four or five non-conservative change(s) at any position.<br><br>
157.	Method according to ciaim 155 or 156, wherein'said polypeptide comprising feature (i) and feature (iii) comprises both Motif I and II,<br>
158.	Method according to claims 155 to 157, wherein said nucleic acid or variant thereof DOF transcription factor is overexpressed in a plant.<br>
159.	Method according to any one of claims 155 to 158, wherein said nucleic acid or variant thereof is of plant origin, preferably from a dicotyledonous plant, further preferably from the family Brassicaceae, more preferably the nucleic acid is from Arabidopsis thaliana.<br>
160.	Method according to any one of claims 155 to 159, wherein said variant encodes a homologue of a DOF transcription factor protein of SEQ ID NO: 199 or SEQ ID NO: 227.<br>
161.	Method according to claim 160, wherein said homologue of a DOF transcription factor protein of SEQ ID NO: 199 is represented by any one of SEQ ID NO: 202, SEQ ID NO: 204, SEQ ID NO: 206, SEQ ID NO: 208, SEQ ID NO: 210, SEQ ID NO: 212, SEQ ID NO: 214.,..SEQIDNO:216,^EQIDNO:.218, SEQID NO:220andSEQlDNOL222_	<br>
162.	Method according to claim 160, wherein said homologue of a DOF transcription factor protein of SEQ ID NO: 227 is represented by any one of SEQ ID NO: 235, SEQ ID NO: 237, SEQ ID NO: 239, SEQ ID NO: 241, SEQ ID NO: 243, SEQ ID NO: 245, SEQ ID NO: 247, SEQ ID NO: 249, SEQ ID NO: 251, SEQ ID NO: 253 and SEQ ID NO: 255.<br>
153. Method according to any one of claims 155 to 152, wherein said nucleic acid or variant thereof encoding a DOF transcription factor polypeptide is operably linked to a constitutive promoter.<br>
164. Method according to claim 153, wherein said constitutive promoter is a G0S2 promoter, preferably a G0S2 promoter from rice.<br>
155.	Method according to any one of claims 155 to 152, wherein said nucleic acid or variant thereof encoding a DOF transcription factor polypeptide is operably linked to a seed-specific promoter.<br>
156.	Method according to claim 155, wherein said seed-specific promoter is an endosperm-specific promoter, preferably a prolamin promoter.<br><br>
157.	Method according to claim 163 or 154.. wherein said constitutive promoter drives expression of a nucleic acid encoding a DOF transcription factor polypeptide comprising said features (i) and (ii).<br>
158.	Method according to claim 155 or 166, wherein said seed-specific promoter drives expression of a nucleic acid encoding a DOF transcription factor polypeptide comprising said features (i) and (iii).<br><br>
169.	Method according to any one of claims 150 to 168, wherein said increased yield is selected from one or more of: increased number of filled seeds, increased seed weight, increased number of flowers per panicle, increased seed fill rate, increased han*/est index (HI), increased thousand kernel weight (TKW), increased root biomass, increased root length and increased root diameter.<br>
170.	Method according to any one of claims 150 to 169, wherein said increased yield occurs under mild drought stress.<br>
171.	Plant or part thereof obtainable by a method according to any one of claims 150 to 170.<br>
172.	Construct comprising:<br>
(i)        a nucleic acid or variant thereof encoding a DOF transcription factor polypeptide as<br>
defined in claim 155; (ii)        one or more control sequences capable of driving expression of the nucleic acid<br>
sequence of (a), and optionally (iii)       a transcription termination sequence.<br>
173.	Construct according to claim 172, wherein said control sequence is a constitutive promoter.<br>
174.	Construct according to claim 172, wherein said control sequence is a seed-specific promoter.<br>
175.	Plant, plant part or plant cell transformed with a construct according to any one of claims 172 to 174.<br>
176.	Method for the production of a transgenic plant having increased yield relative<br>
corresponding wild type plant, which method comprises:<br><br>
(a)	introducing and expressing in a plant, plant part or plant cell a nucleic acid or variant thereof encoding a DOF transcription factor polypeptide as defined in claim 155;<br>
(b)	cultivating the plant cell under conditions promoting plant growth and development.<br><br>
177.	Method according to claim 176 wherein said increased yield occurs under conditions of mild drought stress.<br>
178.	Transgenic plant having increased yield relative to corresponding wild type plant, said increased yield resulting from a nucleic acid or variant thereof encoding a DOF transcription factor polypeptide according to claim 155 introduced into said plant.<br>
179.	Transgenic plant according to claim 171, 175 or 178, wherein said plant is a<br>
monocotyledonous plant, such as sugar cane or wherein the plant is a cereal, such as rice,<br>
maize, wheat, barley, millet, rye, oats or sorghum.<br>
180.	Harvestable parts of a plant according to any one of claims 171, 175, 178 or 179.<br>
181.	Harvestable parts of a plant according to claim 180, wherein said harvestable parts are seeds.<br>
182.	Products derived from a plant according to claim 179 and/or from harvestable parts of a plant according to claims 180 or 181.<br>
183.	Use of a nucleic acid or variant thereof encoding a DOF transcription factor polypeptide, or use of a DOF transcription factor polypeptide, in increasing plant yield relative to suitable control plants. ■<br>
184.	Use according to claim 183, wherein said increased yield is selected from one or more of: increased number of filled seeds, increased seed weight, increased number of flowers per panicle, increased seed fill rate, increased harvest index (HI), increased thousand kernel weight (TKW), increased root biomass, increased root length and increased root diameter.<br>
185.	Use according to claims 183 or 184, wherein said increased yield occurs under mild drought stress conditions.<br><br>
186.	Use of a nucleic acid or variant thereof encoding a DOF transcription factor polypeptide, or use of a DOF transcription factor polypeptide, as a molecular marker,<br>
187.	Method for increasing seed yield in plants relative to suitable control plants, comprising preferentially reducing expression of an endogenous CKI gene in endosperm tissue of a plant,<br>
188.	Method according to claim 187, wherein said preferentially reducing expression is effected by RNA-mediated downregulation of gene expression,<br>
189.	Method according to claim 188, wherein said RNA-mediated downregulation is effected by co-suppression,<br>
190.	Method according to claim 188, wherein said RNA-mediated downregulation is effected by use of antisense CKI nucleic acid sequences,<br>
191.	Method according to claim 187, wherein said preferentially reducing expression is effected using an inverted repeat of a CKI gene or fragment thereof, preferably capable of forming a hairpin structure.<br>
192.	Method according to claim 187, wherein said preferentially reducing expression is effected using ribozymes with specificity for a CKI nucleic acid.<br>
193.	Method according to claim 187, wherein said preferentially reducing expression is effected by insertion mutagenesis.<br>
194.	Method according to any one of claims 187 to 193, wherein said preferentially reducing expression of an endogenous CKI gene in endosperm tissue of a plant is effected by an endosperm-specific promoter, preferably a prolamin promoter.<br>
195.	Method according to any one of claims 187 to 194, wherein said endogenous CKI gene is a CKI gene as found in a plant in its natural form or is an isolated CKI gene subsequently introduced into a plant,<br>
196.	Method according to any one of claims 187 to 197, wherein said CKI gene/nucleic is from a plant source or artificial source, preferably wherein CKI nucleic acid sequences from monocotyledonous plants are used for transformation of monocotyledonous plants, further<br><br>
preferably wherein CKI sequences from the family Poaceae are used for transformation into plants of the family Poaceae, more preferably wherein CKI nucleic acid sequences from rice are used to transform hce plants.<br>
197.	Method according to claim 196, wherein said CKI nucleic acid sequence from nee comprises a sufficient length of substantially contiguous nucleotides of SEQ ID NO: 267 (OsCK14) or comprises a sufficient length of substantially contiguous nucleotides of a nucleic acid sequence encoding an orthologue or paralogue of OsCKI4 (SEQ ID NO; 267),<br>
198.	Method according to claim 197, wherein said orthologues or paralogues of OsCKI4 are represented by SEQ ID NO: 270, SEQ ID NO: 272, SEQ ID NO: 274, SEQ ID NO: 276, SEQ ID NO: 278 and SEQ ID NO: 280.<br>
199.	Method according to claim 197 or 198, wherein said substantially contiguous nucleotides of a nucleic acid sequence encoding an orthologue or paralogue of OsCKI4 (SEQ ID NO: 267) are substantially contiguous nucleotides from nucleic acid sequences represented by<br>
SEQ ID NO:-269, SEQ !D NO: 271, SEQ ID NO: 273, SEQ ID NO: 275, SEQ ID NO: 277   	<br>
and SEQ ID NO: 279.<br>
200.	Method according to any one of claims 187 to 199, wherein said increased seed yield is selected from one or more of the following: a) increased seed biomass; b) increased number of flowers per plant; c) increased number of (filled) seeds; and d) increased harvest index.<br>
201.	Method according to any one of claims 187 to 200, wherein said increased yield occurs under mild stress conditions.<br>
202.	Plant or part thereof obtainable by a method according to any one of claims 187 to 201.<br>
203.	Method for the production of a transgenic plant having increased seed yield relative to suitable control plahts, which method comprises:<br>
(a) introducing and expressing in a plant, plant part or plant cell a gene construct comprising one or more control sequences capable of preferentially driving expression of a sense and/or antisense CKI nucleic acid sequence in plant endosperm tissue so as to silence an endogenous CKI gene in endosperm tissue of a plant; and<br><br>
(b) cultivating the plant, plant part or piant cell under conditions promoting plant growth and development.<br>
204.	Use of CKI nucleic acids for the reduction or substantial elimination of endogenous CKI<br>
gene expression in plant endosperm tissue to increase seed yield in plants relative to<br>
suitable control plants.<br>
205.	Use according to claim 204, wherein said increased yield is selected from one or more of:<br>
selected from one or more of the following: a) increased seed biomass; b) increased<br>
number of flowers per plant; c) increased number of (filled) seeds; and d) increased<br>
harvest index.<br>
206.	Use according to claim 204 or 205, wherein said seed yield occurs under mild stress<br>
conditions.<br>
Dated this 30 day'-of May 2008<br></td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/viewdoc.aspx?id=lUbrT4Ucz3mzklhVSH69jA==&amp;amp;loc=egcICQiyoj82NGgGrC5ChA==" target="_blank" style="word-wrap:break-word;">http://ipindiaonline.gov.in/patentsearch/GrantedSearch/viewdoc.aspx?id=lUbrT4Ucz3mzklhVSH69jA==&amp;amp;loc=egcICQiyoj82NGgGrC5ChA==</a></p>
		<br>
		<div class="pull-left">
			<a href="272179-method-for-treatment-of-sludge.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="272181-methods-and-compositions-for-oral-administration-of-proteins.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>272180</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>2721/CHENP/2008</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>13/2016</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>25-Mar-2016</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>21-Mar-2016</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>30-May-2008</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>CROPDESIGN N.V.</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td>TECHNOLOGIEPARK 3, B-9052 ZWIJNAARDE, BELGIUM</td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>FRANKARD, VALERIE</td>
											<td>BIERENBERG 47, B-1640 SINT-GENESIUS-RODE, BELGIUM</td>
										</tr>
										<tr>
											<td>2</td>
											<td>REUZEAU, CHRISTOPHE</td>
											<td>LA CHAPELLE GONAGUET, F-24350 TOCANE, FRANCE</td>
										</tr>
										<tr>
											<td>3</td>
											<td>SANZ MOLINERO, ANA, ISABEL</td>
											<td>BERNHEIMLAAN 38, B-9050 GENTBRUGGE, BELGIUM</td>
										</tr>
										<tr>
											<td>4</td>
											<td>DAMMANN, CHRISTIAN</td>
											<td>4205 PINE BARK TRAIL, DURHAM, NC 27705, USA</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>C12N5/10</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>PCT/US06/45721</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td>2006-11-29</td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td>60/753,650</td>
									<td>2005-12-30</td>
								    <td>EUROPEAN UNION</td>
								</tr>
								<tr>
									<td>2</td>
									<td>60/742,352</td>
									<td>2005-12-05</td>
								    <td>EUROPEAN UNION</td>
								</tr>
								<tr>
									<td>3</td>
									<td>60/756,042</td>
									<td>2006-01-04</td>
								    <td>EUROPEAN UNION</td>
								</tr>
								<tr>
									<td>4</td>
									<td>05113110.0</td>
									<td>2005-12-30</td>
								    <td>EUROPEAN UNION</td>
								</tr>
								<tr>
									<td>5</td>
									<td>05111996.4</td>
									<td>2005-12-12</td>
								    <td>EUROPEAN UNION</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/272180-plants-having-improved-growth-characteristics-and-methods-for-making-the-same by HTTrack Website Copier/3.x [XR&CO'2014], Thu, 04 Apr 2024 23:01:57 GMT -->
</html>
