<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/272136-nucleic-acid-construct-comprising-nucleic-acid-sequence-encoding-polypeptide-having-arabinofuranosidase-activity by HTTrack Website Copier/3.x [XR&CO'2014], Thu, 04 Apr 2024 23:15:52 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 272136:NUCLEIC ACID CONSTRUCT COMPRISING NUCLEIC ACID SEQUENCE ENCODING POLYPEPTIDE HAVING ARABINOFURANOSIDASE ACTIVITY</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">NUCLEIC ACID CONSTRUCT COMPRISING NUCLEIC ACID SEQUENCE ENCODING POLYPEPTIDE HAVING ARABINOFURANOSIDASE ACTIVITY</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>The present invention relates to isolated polypeptides having alpha-L-arabinofuranosidase activity and isolated nucleic acid sequences encoding the polypeptides. The invention also relates to nucleic acid constructs, vectors, and host cells comprising the nucleic acid sequences as well as methods for producing and using the polypeptides.</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td>ARABINOFURANOSIDASES<br>
FIELD OF THE INVENTION<br>
The present invention relates to isolated polypeptides having alpha-L-arabinofuranosidase activity and isolated nucleic acid sequences encoding the polypeptides. The invention also relates to nucleic acid constructs, vectors, and host cells comprising the nucleic acid sequences as well as methods for producing and using the polypeptides.<br>
BACKGROUND OF THE INVENTION<br>
Arabinofuranosidases are capable of hydrolyzing terminal non-reducing alpha-L-arabinofuranoside residues in alpha-L-arabinosides and are classified as EC 3.2.1.55. Arabinofuranosidases have been isolated from several organisms including filamentous fungi. However, very few alpha-arabinofuranosidases able to liberate arabinose from di-substituted xyloses are known. From Bifidobacterium adolescentis an intracellular enzyme able to release arabinose from C3 of di-substituted xylose (also internally) has been described (Van Laere, 1997, Appl.Microbiol.Biotechnol, 47, 231-235 and Van den Broek, 2005, Applied Microbiology and Biotechnology). From barley an enzyme active on mono-and terminally di-sustituted xyloses has been Isolated, but it has little activity on internally di- substituted xyloses (Ferre, 2000, Eur.J.Biochem., 267, 6633-6641). An enzyme from Trichoderma reesei is possibly active on terminally di-substituted residues (activity seen on 3,5-di-O-alpha-Larabinofuranosyl-alpha-L-arabinofuranoside). but has no activity towards an oligo-substrate vwth internally C3 substituted arabinose (Nogawa, 1999, Appl. Environ. Microbiol.. 65. 3964-3968).<br>
A comparison with full-length prior-art sequences shows that the mature amino acid sequence shown in SEQ ID N0;2 has 72 % homology with an amino acid sequence from Cf)aetomium globosum, and the corresponding DNA sequence in SEQ ID N0:1 shows 73 % homology with that of the corresponding Chaetomium globosum DNA sequence. The homology between the sequence shown in SEQ ID N0:2 and the bacterial GH43 alpha-L-arabinofuranosidase from Bifidobacterium sp. is 25 %. The mature amino acid sequence shown in SEQ ID N0:4 has 38 % homology with the arabinofuranosidase from Aspergillus niger.<br>
SUMMARY OF THE INVENTION<br>
The inventors have isolated alpha-L-arabinofuranosidases from strains of the filamentous fungi Humicola insolens (SEQ ID NO:2) and Meripilus giganteus (SEQ ID N0:4).<br><br>
The inventors also isolated the genes encoding the novel alpha-L-arabinofuranosidases. The enzymes are extracellular. The alpha-L-arabinofuranosidases from Humicola insolens belongs to GH43 and the alpha-L-arabinofuranosidases from Meripilus giganteus belongs to GH51.<br>
The alpha-L-arabinofuranosidase from Humicola insolens is able to liberate arabinose from di-substituted xyloses, i.e. the alpha-L-arabinofuranosidase is active on xylose units of wheat arabinoxylan with arabinose attached to C2 and C3. The activity towards di-substJtuted xyloses is essential for total hydrolysis of arabinoxylan to monosaccharides e.g. in production of ethanol from biomass.<br>
Accordingly, in a first aspect the invention provides an arabinofuranosidase derived from a fungus, preferably a species within Humicola, said arabinofuranosidase capable of releasing arabinose from di-substitued xyloses.<br>
Accordingly, in a second aspect the invention provides an arabinofuranosidase which is: a) a polypeptide having an amino acid sequence as the mature peptide shown in SEQ ID NO: 2, or which can be obtained there from by substitution, deletion, and/or insertion of one or more amino acids; b) an analogue of the polypeptide defined in (i) or (i) which: i) has at least 80 % homology with said polypeptide, ii) is an allelic variant of said polypeptide, c) a polypeptide which is encoded by a nucleic acid sequence which hybridizes under high stringency conditions with a complementary strand of the nucleic acid sequence of SEQ ID N0:2 encoding the mature polypeptide or a subsequence thereof having at least 100 nucleotides.<br>
Accordingly, in a third aspect the invention provides an arabinofuranosidase which is: a) a polypeptide having an amino acid sequence as the mature peptide shown in SEQ ID NO: 4, or which can be obtained there from by substitution, deletion, and/or insertion of one or more amino acids; b) an analogue of the polypeptide defined in (i) or (ii) which: i) has at least 60 % homology with said polypeptide, 11) is an allelic variant of said polypeptide, c) a polypeptide which is encoded by a nucleic acid sequence which hybridizes under high stringency conditions with a complementary strand of the nucleic acid sequence of SEQ ID N0:2 encoding the mature polypeptide or a subsequence thereof having at least 100 nucleotides.<br>
In a fourth aspect the invention provides a nucleic acid sequence comprising a nucleic acid sequence which encodes the arabinofuranosidase of the first or second aspect.<br>
In a fifth aspect the invention provides a nucleic acid sequence which comprises: a) the DNA sequence encoding any of the arabinofuranosidases shown in SEQ ID N0:2 and SEQ ID N0:4, b) an analog DNA sequence which i) has at least 80 % homology with any of said DNA sequences, or ii) hybridizes at high stringency with a complementary strand of any<br><br>
of said DNA sequences or a subsequence thereof having at least 100 nucleotides, iii) is an allelic variant thereof, or c) a complementary strand to a) or b).<br>
In a sixth aspect the invention provides a nucleic acid sequence which has at least 80 % homology with any of the DNA sequences shown in SEQ ID N0:1 and SEQ ID N0:3, or a) hybridizes at high stringency with a complementary strand of any of said DNA sequences or a subsequence thereof having at least 100 nucleotides, b) is an allelic variant thereof, or c) a complementary strand to a) or b).<br>
In a seventh aspect the invention provides a nucleic acid construct comprising the nucleic acid sequence of the second, third and fourth aspect operably linked to one or more control sequences capable of directing the expression of the arabinofuranosidase in a suitable expression host.<br>
In an eight aspect the invention provides a recombinant expression vector comprising the nucleic acid construct of the sixth aspect<br>
In an ninth aspect the invention provides a recombinant host cell comprising the nucleic acid construct of the sixth aspect.<br>
In a ninth aspect the invention provides a method for producing an arabinofuranosidase comprising cultivating the host cell of the eight aspect under conditions conducive to production of the arabinofuranosidase, and recovering the arabinofuranosidase.<br>
in a tenth aspect the invention provides a composition comprising the arabinofuranosidase of the first and second aspect.<br>
In further aspects the Invention provides uses of the arabinofuranosidases of the first and second aspect.<br>
BRIEF DESCRIPTION OF THE DRAWING<br>
Fig. 1A-C show arabinoxylan polymers Fig. 1A shows intact arabinoxylan. Fig. IB shows di-substituted arabinoxylan. Fig. 1C shows singly substituted arabinoxylan.<br>
Fig. 2 A-C show arabinoxylo-oligosaccharides Fig. 2A shows arabinosyl groups linked to internal C-3. Fig. 2B shows arabinosyl groups linked to terminal C-3 Fig. 2C shows arabinosyl groups linked to internal C-2<br><br>
DETAILED DESCRIPTION OF THE INVENTION<br>
In a embodiment of the second aspect of the present invention, the isolated polypeptide has an amino acid sequence which has at least 80 % identity with the amino acid sequence shown as amino acids 1 to 558 of SEQ ID N0:2. in an interesting embodiment of the invention the polypeptide has at least 75 %, at least 85 %, at least 90 %. at least 95 %, at least 96 %, at least 97 %, at least 98 %, or at least 99 % identity with the amino acid sequence shown as amino acids 1 to 558 of SEQ ID NO;2.<br>
In an embodiment of the third aspect of the present invention, the isolated polypeptide has an amino acid sequence which has at least 50 % identity with the amino acid sequence shown as amino acids 1 to 643 of SEQ ID N0:4. In an interesting embodiment of the invention the polypeptide has at least 60%, at least 70 %, at least 80 % at least 85 %, at least 90 %, at least 95 %, at least 96 %, at least 97 %, at least 98 %, or at least 99 % identity with the amino acid sequence shown as amino acids 1 to 643 of SEQ ID N0:4.<br>
In a preferred embodiment, the isolated polypeptide have an amino acid sequence which differs by five amino acids, e.g. by four amino adds, such as by three amino acids, by two amino acids, or by one amino acid from any of the dmino acid sequences shown as amino acids 1 to 558 of SEQ ID N0:2 and 1 to 643 of SEQ ID N0:4.<br>
In an embodiment of the second aspect of the invention, the isolated polypeptide is an alpha-L-arabinofuranosidase able to liberate arabinose from di-substituted xyloses, e.g. the alpha-L-arabinofuranosidase is active on xylose units of wheat arabinoxylan with arabinose attached to C2 and C3.<br>
Alignments of sequences and calculation of homology may suitably be determined by means of computer programs Icnown in the art such as GAP provided in the GCG program package (Program Manual for the Wisconsin Package, Version 8, August 1994, Genetics Computer Group, 575 Science Drive, Madison, Wisconsin, USA 53711) (Needleman, S.B. and Wunsch, CD., (1970), Journal of Molecular Biology, 48, 443-453. The following settings for amino acid sequence comparison are used: GAP creation penalty of 3.0 and GAP extension penalty of 0.1. The relevant part of the amino acid sequence for the homology determination is the mature polypeptide, i.e. without the signal peptide.<br>
Preferably, the polypeptides of the present invention comprise any of the amino acid sequences shown as amino acids 1 to 558 of SEQ ID N0:2 and 1 to 643 of SEQ ID N0;4, an allelic variant thereof, or a fragment thereof that has arabinofuranosidase activity. Obviously, the polypeptide of the invention may also consist of any of the amino acid sequences shown as amino acids 1 to 558 of SEQ ID N0:2 and 1 to 643 of SEQ ID N0:4.<br>
An allelic variant denotes any of two or more alternative fonns of a gene occupying the same chromosomal locus. Allelic variation arises naturally through mutation, and may<br><br>
result in polymorphism within populations. Gene mutations can be silent (no change in the encoded polypeptide) or may encode polypeptides having altered amino acid sequences. An allelic variant of a polypeptide is a polypeptide encoded by an allelic variant of a gene.<br>
In an embodiment of the invention, the isolated polypeptide is encoded by a nucleic acid sequence which hybridizes under low stringency conditions, preferably under medium stringency conditions, more preferably under high stringency conditions with (i) a complementary strand of any of the nucleic acid sequences shown as nucleotides 1 to 1677 of SEQ ID N0:1 and 46 to 1974 of SEQ ID N0:3, or (ii) a subsequence of (i) of at least 100 nucleotides (J. Sambroolc, E.F. Fritsch, and T. Maniatus, 1989, Molecular Cloning, A Laboratory Manual, 2d edition, Cold Spring Harbor, New York).<br>
The subsequence of the complementary strand of any of the nucleic acid sequences shown as nucleotides 1 to 1677 of SEQ ID N0:1 and 46 to 1974 of SEQ ID N0:3, may be at least 100 nucleotides or preferably at least 200 nucleotides. Moreover, the subsequence should encode a polypeptide fragment which has arabinofuranosidase activity. The polypeptides may also be allelic variants or fragments of the polypeptides that have arabinofuranosidase activity.<br>
The variants of the polypeptides specified comprising a substitution, deletion, and/or insertion of one or more amino acids. In a particular embodiment, the polypeptides are thermostable variants of the polypeptides specified.<br>
The amino acid sequences of the variant polypeptides may differ from the amino acid sequence specified in amino acids 1 to 558 of SEQ ID N0:2 or 1 to 643 of SEQ ID N0:4 by an insertion or deletion of one or more amino acid residues and/or the substitution of one or more amino acid residues by different amino acid residues. Preferably, amino acid changes are of a minor nature, that is conservative amino acid substitutions that do not significantly affect the folding and/or activity of the protein; small deletions, typically of one to about 30 amino acids; small amino- or carboxyl-terminal extensions, such as an amino-terminal methionine residue; a small linker peptide of up to about 20-25 residues; or a small extension that facilitates purification by changing net charge or another function, such as a poiy-histidine tract, an antigenic epitope or a binding domain.<br>
Examples of conservative substitutions are within the group of basic amino acids (arginine, lysine and histidine), acidic amino acids (glutamic acid and aspartic acid), polar amino acids (glutamine and asparagine), hydrophobic amino acids (leucine, isoleucine and valine), aromatic amino acids (phenylalanine, tryptophan and tyrosine), and small amino acids (glycine, alanine, serine, threonine and methionine). Amino acid substitutions which do not generally alter the specific activity are known in the art and are described, for example, by H. Neurath and R.L. Hill, 1979, In, The Proteins, Academic Press, New York. The most commonly occurring exchanges are Ala/Ser, Val/lle, Asp/Glu, Thr/Ser. Ala/Giy, AlaA'hr,<br><br>
Ser/Asn, AlaA/al, Ser/Gly. Tyr/Phe, Ala/Pro, Lys/Arg. Asp/Asn, Leu/lle. LeuA/al, Ala/Glu, and Asp/Gly as well as these in reverse.<br>
The polypeptides referred to herein may comprise the amino acid sequence specified, or they may be an allelic variant thereof; or a fragment thereof that has the relevant enzyme activity. In one embodiment, the polypeptides comprise the amino acid sequence specified or an allelic variant thereof; or a fragment thereof that has the relevant enzyme activity. In another embodiment, the polypeptides consist of the amino acid sequence specified, or an allelic variant thereof; or a fragment thereof that has the relevant enzyme activity.<br>
A fragment of a specified amino acid sequence is a polypeptide having one or more amino adds deleted from the amino and/or carboxyl tenninus of this amino acid sequence. In one embodiment, a fragment contains at least 60 amino acid residues, or at least 68, or at least 70, or at least 75, or at least 100, or at least 150, or at least 160, or at least 170, or at least 180, or at least 190, or at least 200, or at least 210, or at least 220, or at least 240, or at least 260, or at least 280, or at least 300, or at least 310, or at least 320, or at least 330, or at least 334, or at least 350, or at least 375, or at least 400, or at least 425, or at least 430 amino acid residues.<br>
An allelic variant denotes any of two or more alternative forms of a gene occupying the same chromosomal locus. Allelic variation arises naturally through mutation, and may result in polymorphism within populations. Gene mutations can be silent (no change in the encoded polypeptide) or may encode polypeptides having altered amino acid sequences. An allelic variant of a polypeptide is a polypeptide encoded by an allelic variant of a gene.<br>
A mature polypeptide or a mature amino acid sequence refers to that part of an amino acid sequence which remains after a potential signal peptide part has been cleaved off. And analogously, a mature polypeptide encoding part of a gene refers to that part of a gene, which con^esponds to a mature polypeptide.<br>
The nucleic acid sequence of any of SEQ ID N0:1 and SEQ ID N0:3 or a subsequence thereof; as well as the amino acid sequence of any of of SEQ ID N0:2 and SEQ ID N0:3 or a fragment thereof, may be used to design a nucleic acid probe to identify and clone DNA encoding polypeptides having arabinofuranosidase activity from strains of different genera or species according to methods well known in the art. In particular, such probes can be used for hybridization with the genomic or cDNA of the genus or species of interest, following standard Southern blotting procedures, in order to identify and isolate the corresponding gene therein. Such probes can be considerably shorter than the entire sequence, but should be at least 15, preferably at least 25, and more preferably at least 35 nucleotides in length. Longer probes can also be used. Both DNA and RNA probes can be<br><br>
used. The probes are typically labeled for detecting the corresponding gene (for example, with 32P, 3H, 35S, biotin, or avidin). Such probes are encompassed by the present invention.<br>
Thus, a genomic DNA or cDNA library prepared from such other organisms may be screened for DNA which hybridizes with the probes described above and which encodes a polypeptide having arabinofuranosidase activity. Genomic or other DNA from such other organisms may be separated by agarose or poiyacrylamide gel electrophoresis, or other separation techniques known by the skilled person. DNA from the libraries or the separated DNA may be transferred to and immobilized on nitrocellulose or other suitable carrier materials. In order to identify a clone or DNA which is homologous with SEQ ID N0:1 or SEQ ID N0:3, or a subsequence thereof, the carrier material is used in a Southern blot. For purposes of the present invention, hybridization indicates that the nucleic acid sequence hybridizes to a labeled nucleic acid probe corresponding to any of the nucleic acid sequences shown in SEQ ID NO:1 and SEQ ID NO:3, its complementary strand, or a subsequence thereof, under low to high stringency conditions. Molecules to which the nucleic acid probe hybridizes under these conditions are detected using X-ray film.<br>
In another interesting embodiment, the nucleic acid probe is a nucleic acid sequence which encodes any of the (mature) polypeptides of SEQ ID N0:2 and SEQ ID N0:4, or a subsequence thereof. In a third interesting embodiment, the nucleic acid probe is any of SEQ ID N0:1 and SEQ ID N0:3. In a fourth interesting embodiment, the nucleic acid probe is any of the mature polypeptide coding regions of SEQ ID N0:1 and SEQ ID N0:3.<br>
For long probes of at least 100 nucleotides in length, low to high stringency conditions are defined as prehybridization and hybridization at 42**C in 5X SSPE, 0.3 % SDS, 200 ^g/ml sheared and denatured salmon sperm DNA, and either 25 % formamide for low stringency, 35 % formamide for medium stringency, or 50 % formamide for high stringency, following standard Southern blotting procedures.<br>
For long probes of at least 100 nucleotides in length, the carrier material is finally washed three times each for 15 minutes using 2 x SSC, 0.2 % SDS preferably at least at 50*C (low stringency), more preferably at least at 55°C (medium stringency), even more preferably at least at 65°C (high stringency).<br>
For short probes which are about 15 nucleotides to about 70 nucleotides in length, stringency conditions are defined as prehybridization, hybridization, and washing post-hybridization at 5°Cto 10°C below the calculated Tm using the calculation according to Bolton and McCarthy (1962. Proceedings of the National Academy of Sciences USA 48:1390) in 0.9 M NaCI, 0.09 M Tris-HCI pH 7.6, 6 mM EDTA, 0.5 % NP-40, 1X Denhardt's solution, 1 mM sodium pyrophosphate, 1 mM sodium monobasic phosphate, 0.1 mM ATP, and 0.2 mg of yeast RNA per ml, following standard Southern blotting procedures.<br><br>
For short probes which are about 15 nucleotides to about 70 nucleotides in lengxn, the carrier material is washed once in 6X SCC plus 0.1 % SDS for 15 minutes and twice each for 15 minutes using 6X SSC at 5°C to 10°C below the calculated Tm.<br>
As indicated above, the polypeptide of the invention may be a polypeptide having any of the amino acid sequences of SEQ ID N0:2 and SEQ ID NO:4. or the mature polypeptide thereof, wherein one or more amino acid(s) has (have) been substituted by another (other) amino acid(s), wherein one or more amino acid(s) has (have) been deleted, and/or wherein one more amino acid(s) has (have) been inserted.<br>
Preferably, amino acid changes are of a minor nature, that is conservative amino acid substitutions that do not significantly affect the folding and/or activity of the protein; small deletions, typically of one to about 30 amino acids; small amino- or carboxyl-terminal extensions, such as an amino-terminal methionine residue; a small linlcer peptide of up to about 20-25 residues; or a small extension that facilitates purification by changing net charge or another function, such as a poly-histidine tract, an antigenic epitope or a binding domain.<br>
Examples of conservative substitutions are within the group of basic amino acids (arginine, lysine and histidine), acidic amino acids (glutamic acid and aspartic acid), polar amino acids (glutamine and asparagine), hydrophobic amino acids (leucine, isoleucine and valine), aromatic amino acids (phenylalanine, tryptophan and tyrosine), and small amino acids (glycine, alanine, serine, threonine and methionine). Amino add substitutions which do not generally alter the specific activity are known in the art and are descrit&gt;ed, for example, by H. Neurath and R.L Hill, 1979, In, The Proteins, Academic Press, New York. The most commonly occurring exchanges are Ala/Ser, Val/lle, Asp/Glu, Thr/Ser, Ala/Gly, Ala/Thr, Ser/Asn, AlaA/al, Ser/Gly, Tyr/Phe, Ala/Pro, Lys/Arg, Asp/Asn, Leu/lle, LeuA/al, Ala/Glu, and Asp/Gly as well as these in reverse.<br>
In general, it is preferred that the polypeptides of the invention have at least 20 % of the arabinofuranosidase activity of the polypeptide having any of the amino acid sequences shown as amino acids 1 to 558 of SEQ ID N0:2 and amino acids 1 to 643 of SEQ ID N0:4. Particular preferred are polypeptides, which have at least 30 %, such as at least 40 %, e.g. at least 50 %, preferably at least 60 %, such as at least 70 %, e.g. at least 80 %, more prefenred at least 90 %. or at least 95 % of the arabinofuranosidase activity of the polypeptide any of the amino acid sequences shown as amino acids 1 to 558 of SEQ ID N0:2 and amino acids 1 to 643 of SEQ ID N0:4.<br>
A polypeptide of the present invention may be obtained from microorganisms of any genus. For purposes of the present invention, the term "obtained from" as used herein in connection with a given source shall mean that the polypeptide encoded by the nucleic acid sequence is produced by the source or by a cell in which the nucleic acid sequence from the<br><br>
source has been inserted. In a preferred embodiment, the polypeptide is secreted extracellulariy.<br>
A polypeptide of the present Invention may be a fungal polypeptide, and more preferably a filamentous fungal polypeptide such as an Acremorium, Aspergillus, Aureobasidium, Cryptococcus, Flllbasidium, Fusarium, Humicola, Magnaporthe, Meripilus, , Mucor,. Myceliophthora, Neocallimastix, Neurospora, Paecilomyces, PeniciWum, Piromyces, Schizophyllum, Talaromyces, Thermoascus, Thielavia, Tolypocladium, or Trichoderma polypeptide.<br>
In another preferred embodiment, the polypeptide is an Aspergillus aculeatus, Aspergillus awamori, Aspergillus foetidus, Aspergillus Japonicus, Aspergillus r^iduians, Aspergillus niger, Aspergillus oryzae, Fusarium bactridloides, Fusarium cerealis, Fusarium crookwellense, Fusarium culmorum, Fusarium graminearum, Fusarium graminum, Fusarium heterosporum, Fusarium negundi, Fusarium oxysporum, Fusarium reticulatum, Fusarium roseum, Fusarium sambucinum, Fusarium sarcochroum, Fusarium sporotrichioides, Fusarium sulpiiureum, Fusarium torulosum, Fusarium trichothecioides, Fusarium venenatum, Humicola insolens, Humicola lanuginosa, Meripilus giganteus, Mucor miehei, Myceliophthora thermophila, Neurospora crassa, Penicillium purpurogenum, Trichoderma harzianum, Trichoderma koningii, Trichoderma longibrachiatum, Trichoderma reesei, or Trichoderma viride polypeptide.<br>
In a preferred embodiment of the invention the polypeptide of the invention is derived from a strain within the Ascomycota, e.g.. within the genus Humicola, such as within H. lanuginosa H. fuscoatra, K grisea, H. lutea, H, nigrescens and in particular within H, insolenSj or from as strain within Basidiomycota, such as within the genus Meripilus, such as within the M, giganteus.<br>
It will be understood that for the aforementioned species, the invention encompasses both the perfect and imperfect states, and other taxonomic equivalents, e.g., anamorphs, regardless of the species name by which they are known. Those skilled in the art will readily recognize the identity of appropriate equivalents.<br>
Strains of these species are readily accessible to the public in a number of culture collections, sucfi as the American Type Culture Collection (ATCC), Deutsche Sammlung von Mlkroorganismen und Zellkulturen GmbH (DSM), Centraalbureau Voor Schimmelcultures (CBS), and Agricultural Research Service Patent Culture Collection, Northern Regional Research Center (NRRL).<br>
In a particularly preferred embodiment of the invention the polypeptide of the invention is derived from the H. insolens strain described in W09117243 and deposited April 14 1980 at the German Collection of l\4icroorganisms and Cell cultures (DSM. Deutsche<br><br>
Sammlung von Microorganismen und Zeilkulturen) Gotlingen, Germany, under the DSM number 1800) in accordance with the provisions of the Budapest Treaty.<br>
In a second particularly preferred embodiment of the invention the polypeptide of the invention is derived the specific strain of Meripilus giganteus having the accession number CBS 52195 in Centraalbureau Voor Schimmelcultures (CBS), Uppsalalaan 8, 3584 CT Utrecht. The Netheriands (alternatively P.O.Box 85167. 3508 AD Utrecht, The Netheriands).<br>
Furthermore, such polypeptides may be identified and obtained from other sources including microorganisms isolated from nature (e.g., soil, composts, water, etc.) using the above-mentioned probes. Techniques for isolating microorganisms from natural habitats are well known in the art. The nucleic acid sequence may then be derived by similarly screening a genomic or cDNA library of another microorganism. Once a nucleic add sequence encoding a polypeptide has been detected with the probe(s), the sequence may be isolated or cloned by utilizing techniques which are known to those of ordinary skill in the art (see, e.g., Sambrook et aL, 1989. supra).<br>
Polypeptides encoded by nucleic acid sequences of tiie present invention also include fused polypeptides or cleavabie fusion polypeptides in which another polypeptide is fused at the N-terminus or the C-terminus of the polypeptide or fragment thereof. A fused polypeptide Is produced by fusing a nucleic acid sequence (or a portion thereof) encoding another polypeptide to a nucleic acid sequence (or a portion thereof) of the present invention. Techniques for producing fusion polypeptides are known in the art, and include ligating the coding sequences encoding the polypeptides so that they are In frame and that expression of the fused polypeptide is under control of the same promoter(s) and terminator.<br>
Nucleic Acid Sequences<br>
The present invention also relates to isolated nucleic acid sequences which encode a polypeptide of the present invention.<br>
In one interesting embodiment, the nucleic acid sequence has at least 80 % identity with any of the nucleic acid sequences shown as nucleotides 1 to 1677 of SEQ ID N0:1 and 46 to 1974 of SEQ ID N0:2. Preferably, the nucleic acid sequence has at least at least 85 %, at least 90 %. at least 95 %, at least 96 %, at least 97 %, at least 98 %, or at least 99 % identity with any of the nucleic acid sequences shown as nucleotides 1 to 1677 of SEQ ID N0:1 and 46 to 1974 of SEQ ID N0:2, In another interesting embodiment of the invention the nucleic acid sequence comprises any of the amino acid sequences shown as nucleotides 1 to 1677 of SEQ ID N0:1, an allelic variant thereof, or a fragment thereof capable of encoding a polypeptide according to the invention. Obviously, the nucleic acid sequence may<br><br>
consist of any of the amino acid sequences shown as nucleotides 1 to 1677 of SEQ ID NO:1 and 46 to 1974 of SEQ ID NO:2.<br>
The present invention also encompasses nucleic acid sequences which encode a polypeptide having any of the amino acid sequences of SEQ ID NO:2 or SEQ ID NO:4. or the mature polypeptides thereof, which differ from SEQ ID NO:1 or SEQ ID NO:3 by virtue of the degeneracy of the genetic code. The present invention also relates to subsequences of SEQ ID NO:1 or SEQ ID NO:3 which encode fragments of SEQ ID NO:2 or SEQ ID NO:4 that have arabinofuranosidase activity. A subsequence of SEQ ID NO:1 or SEQ ID NO:3 is a nucleic acid sequence encompassed by nucleotides 1 to 1677 of SEQ ID NO:1 or nucleotides 46 to 1974 of SEQ ID NO:3 except that one or more nucleotides from the 5* and/or 3' end have been deleted.<br>
The present invention also relates to isolated nucleic acid sequences encoding a polypeptide of the present invention, which hybridize under low stringency conditions, preferably under medium stringency conditions, more preferably under high stringency conditions, with (i) a complementary strand of any of the nucleic acid sequences shown as nucleotides 1 to 1677 of SEQ ID NO:1 and 46 to 1974 SEQ ID NO:3. or (ii) a subsequence of (i) of at least 100 nucleotides. The present invention also relates to complementary strands of (i), (ii), and (iii).<br>
The techniques used to isolate or clone a nucleic acid sequence encoding a polypeptide are l<nown in the art and include isolation from genomic dna preparation cdna or a combination thereof. cloning of nucleic acid sequences present invention such can be effected e.g. by using well known polymerase chain reaction antibody screening expression libraries to detect cloned fragments with shared structural features. see innis ef pcr: guide methods application academic press new york. other amplification procedures as ligase ligated activated transcription sequence-based may used. sequence strain humicola insolens miripilus giganteus another related organism for example an allelic species variant polypeptide encoding region sequence.></nown>
An isolated nucleic acid sequence can, for example, be obtained by standard cloning procedures used in genetic engineering to relocate- the nucleic acid sequence from its natural location to a different site where it will be reproduced. The cloning procedures may involve excision and isolation of a desired nucleic add fragment comprising the nucleic acid sequence encoding the polypeptide, insertion of the fragment into a vector molecule, and incorporation of the recombinant vector into a host cell where multiple copies or clones of the<br><br>
nucleic acid sequence will be replicated. The nucleic acid sequence may be of genomic, cDNA. RNA, semisynthetic, synthetic origin, or any combinations thereof.<br>
For purposes of the present invention, the degree of identity between two nucleic acid sequences is determined as described above.<br>
Modification of a nucleic acid sequence encoding a polypeptide of the present invention may be necessary for the synthesis of polypeptides substantially similar to the polypeptide. The temi "substantially similar" to the polypeptide refers to non-naturally occurring forms of the polypeptide. These polypeptides may differ in some engineered way from the polypeptide isolated from its native source, e.g., variants that differ in specific activity, thermostability, pH optimum, or the like. The variant sequence may be constructed on the basis of the nucleic acid sequence presented as the polypeptide encoding part of SEQ ID NO:1, e.g., a subsequence thereof, and/or by introduction of nucleotide substitutions which do not give rise to another amino acid sequence of the polypeptide encoded by the nucleic acid sequence, but which correspond to the codon usage of the host organism intended for production of the enzyme, or by introduction of nucleotide substitutions which may give rise to a different amino acid sequence. For a general description of nucleotide substitution, see, e.g., Ford et aL, 1991, Protein Expression and Purification 2: 95-107.<br>
It will be apparent to those skilled in the art that such substitutions can be made outside the regions critical to the function of the molecule and still result in an active polypeptide. Amino acid residues essential to the activity of the polypeptide encoded by the isolated nucleic acid sequence of the invention, and therefore preferably not subject to substitution, may be identified according to procedures known in the art, such as site-directed mutagenesis or alanine-scanning mutagenesis (see, e.g.. Cunningham and Wells, 1989. Science 244: 1081-1085). In the latter technique, mutations are introduced at every positively charged residue in the molecule, and the resultant mutant molecules are tested for arabinofuranosidase activity to identify amino acid residues that are critical to the activity of the molecule. Sites of substrate-enzyme interaction can also be determined by analysis of the three-dimensional structure as determined by such techniques as nuclear magnetic resonance analysis, crystallography or photoaffinity labelling (see, e.g., de Vos et ai, 1992, Science 255: 306-312; Smith et aL, 1992, Journal of Moiecular Biology 224: 899-904; Wlodaver ef a/., 1992, FEBS Letters 309: 59-64).<br><br>
Nucleic Acid Constructs<br>
The present invention also relates to nucleic acid constructs comprising a nucleic acid sequence of the present invention operably linked to one or more control sequences capable of directing the expression of the polypeptide in a suitable host cell.<br>
An isolated nucleic acid sequence encoding a polypeptide of the present invention may be manipulated in a variety of ways to provide for expression of the polypeptide. Manipulation of the nucleic acid sequence prior to its insertion into a vector may be desirable or necessary depending on the expression vector. The techniques for modifying nucleic acid sequences utilizing recombinant DNA methods are well known in the art.<br>
The control sequences include all components which are necessary or advantageous for the expression of a polypeptide of the present invention. Each control sequence may be native or foreign to the nucleic acid sequence encoding the polypeptide. Such control sequences include, but are not limited to, a leader, polyadenylation sequence, propeptide sequence, promoter, signal peptide sequence, and transcription terminator. At a minimum, the control sequences include a promoter, and transcriptional and translational stop signals. The control sequences may be provided with linkers for the purpose of introducing specific restriction sites facilitating ligation of the control sequences with the coding region of the nucleic acid sequence encoding a polypeptide.<br>
The control sequence may be an appropriate promoter sequence, a nucleic acid sequence which is recognized by a host ceil for expression of the nucleic acid sequence. The promoter sequence contains transcriptional control sequences which mediate the expression of the polypeptide. The promoter may be any nucleic acid sequence which shows transcriptional activity in the host cell of choice including mutant, truncated, and hybrid promoters, and may be obtained from genes encoding extracellular or intracellular polypeptides either homologous or heterologous to the host cell.<br>
Examples of suitable promoters for directing the transcription of the nucleic acid constructs of the present invention, especially in a bacterial host cell, are the promoters obtained from the £. coli lac operon, Streptomyces coelicolor agarase gene (dagA), Bacillus subtiiis levansucrase gene (sacS), Bacillus liciieniformis alpha-amylase gene {amyL\ Bacillus stearotfiermophilus maltogenic amylase gene [amyM), Bacillus amyloliquefaciens alpha-amylase gene {amyQ), Bacillus licheniformis penicillinase gene {perjp), Bacillus subtiiis xylA and xylB genes, and prokaryotic beta-lactamase gene (Villa-Kamaroff et aL, 1978, Proceedings of the National Academy of Sciences USA 75: 3727-3731), as well as the tac promoter (DeBoer et aL, 1983, Proceedings of the National Academy of Sciences USA 80: 21-25). Further promoters are described in "Useful proteins from recombinant bacteria" in Scientific American, 1980, 242: 74-94; and in Sambrook ef a/., 1989, supra.<br><br>
Examples of suitable promoters for directing the transcription of the nucleic acid constructs of the present invention in a filamentous fungal host cell are promoters obtained from the genes for Aspergillus oryzae TAKA amylase, Rhizomucor miehei aspartic proteinase, Aspergillus niger neutral alpha-amylase, Aspergillus niger acid stable alpha-amylase, Aspergillus niger or Aspergillus awamor/glucoamylase {glaA), Rhizomucor miehei lipase, Aspergillus oryzae alkaline protease, Aspergiilus oryzae triose phosphate isomerase, Aspergillus nidulans acetamidase, and Fusarium oxysporum trypsin-like protease (WO 96/00787). as well as the NA2-tpi promoter (a hybrid of the pnamoters from the genes for Aspergillus n/ger neutral alpha-amylase and Aspergillus oryzae triose phosphate isomerase), and mutant, truncated, and hybrid promoters thereof.<br>
In a yeast host, useful promoters are obtained from the genes for Saccharomyces oerevislae enolase (ENO-1), Saccharomyces cerevisiae galactokinase (GAL1), Saccharomyces cerevisiae alcohol dehydrogenase/glyceraldehyde-3-phosphate dehydrogenase (ADH2/GAP), and Saccharomyces cerevisiae 3-phosphoglycerate kinase. Other useful promoters for yeast host cells are described by Romanes et aL, 1992, Yeast 8: 423-488.<br>
The control sequence may also be a suitable transcription terminator sequence, a sequence recognized by a host cell to tenninate transcription. The terminator sequence is operably linked to the 3' terminus of the nucleic acid sequence encoding the polypeptide. Any terminator which is functional in the host cell of choice may be used in the present invention.<br>
Preferred terminators for filamentous fungal host cells are obtained from the genes for Aspergillus oryzae TAKA amylase, Aspergillus niger glucoamylase. Aspergillus nidulans anthranilate synthase. Aspergillus niger alpha-glucosidase, and Fusarium oxysporum trypsin-like protease.<br>
Preferred terminators for yeast host cells are obtained from the genes for Saccharomyces cerevisiae enolase, Saccharomyces cerevisiae cytochrome 0 {CYC1), and Saccharomyces cerevisiae glyceraldehyde-3-phosphate dehydrogenase. Other useful tenninators for yeast host cells are described by Romanes et ai, 1992, supra.<br>
The control sequence may also be a suitable leader sequence, a nontranslated region of an mRNA which is important for translation by the host cell. The leader sequence is operably linked to the 5' terminus of the nucleic acid sequence encoding the polypeptide. Any leader sequence that is functional in the host cell of choice may be used in the present invention.<br>
Pretended leaders for filamentous fungal host cells are obtained from the genes for Aspergillus oryzae TAKA amylase and Aspergillus nidulans triose phosphate isomerase.<br><br>
Suitable leaders for yeast host cells are obtained from the genes for Saccharomyces cerevisiae enolase (ENO-1), Saccharomyces cerevisiae 3-phosphoglycerate kinase. Saccharomyces cerevisiae alpha-factor, and Saccharomyces cerevisiae alcohol dehydrogenase/glyceraldehyde-3-phosphate dehydrogenase (ADH2/GAP).<br>
The control sequence may also be a polyadenylation sequence, a sequence operably linked to the 3' terminus of the nucleic acid sequence and which, when transcribed, is recognized by the host cell as a signal to add polyadenosine residues to transcribed mRNA. Any polyadenylation sequence which is functional in the host cell of choice may be used in the present invention.<br>
Preferred polyadenylation sequences for filamentous fungal host ceils are obtained from the genes for Aspergillus oryzae TAKA amylase, Aspergillus niger glucoamylase, Aspergillus nidulans anthranilate synthase, Fusarium oxysporum trypsin-like protease, and Aspergillus n/ger alpha-glucosidase.<br>
Useful polyadenylation sequences for yeast host cells are described by Guo and Shemnan, 1995, Molecular Cellular Biology ^5: 5983-5990.<br>
The control sequence may also be a signal peptide coding region that codes for an amino acid sequence linked to the amino terminus of a polypeptide and directs the encoded polypeptide into the cell's secretory pathway. The 5' end of the coding sequence of the nucleic acid sequence may inherently contain a signal peptide coding region naturally linked in translation reading frame with the segment of the coding region virtiich encodes the secreted polypeptide. Alternatively, the 5' end of the coding sequence may contain a signal peptide coding region which is foreign to the coding sequence. The foreign signal peptide coding region may be required where the coding sequence does not naturally contain a signal peptide coding region. Alternatively, the foreign signal peptide coding region may simply replace the natural signal peptide coding region in order to enhance secretion of the polypeptide. However, any signal peptide coding region which directs the expressed polypeptide into the secretory pathway of a host cell of choice may be used in the present invention.<br>
Effective signal peptide coding regions for bacterial host cells are the signal peptide coding regions obtained from the genes for Bacillus NCIB 11837 maltogenic amylase, Bacillus stearothermophilus alpha-amytase. Bacillus licheniformis subtllisin. Bacillus ficheniformis beta-lactamase, Bacillus stearothermophilus neutral proteases [nprT, nprS, nprM), and Bacillus subtllis prsA. Further signal peptides are described by Simonen and Paiva, 1993, Microbiological Reviews 57:109-137.<br>
Effective signal peptide coding regions for filamentous fungal host cells are the signal peptide coding regions obtained from the genes for Aspergillus oryzae TAKA amylase,<br><br>
Aspergillus r)iger neutral amylase, Aspergillus niger glucoamylase, Rhizomucor miehei aspartic proteinase, Humicoia hsoleris cellulase, and Humicola lanuginosa lipase.<br>
Useful signal peptides for yeast host cells are obtained from the genes for Saccharorr)yces cerevisiae alpha-factor and Saccharomyces cerews/ae invertase. Other useful signal peptide coding regions are described by Romanes etaL, 1992, supra.<br>
The control sequence may also be a propeptide coding region that codes for an amino acid sequence positioned at the amino temriinus of a polypeptide. The resultant polypeptide is known as a proenzyme or propolypeptide (or a zymogen in some cases). A propolypeptide is generally inactive and can be converted to a mature active polypeptide by catalytic or autocatalytic cleavage of the propeptide from the propolypeptide. The propeptide coding region may be obtained from the genes for Bacillus subWis alkaline protease {aprE), Bacillus subtilis neutral protease (nprT), Sacctiaromyces cerevisiae alpha-factor, Rhizomucor mieiiei aspartic proteinase, and Myceliophthora thermophila laccase (WO 95/33836).<br>
Where both signal peptide and propeptide regions are present at the amino terminus of a polypeptide, the propeptide region is positioned next to the amino terminus of a polypeptide and the signal peptide region is positioned next to the amino terminus of the propeptide region.<br>
It may also be desirable to add regulatory sequences which allow the regulation of the expression of the polypeptide relative to the growth of the host cell. Examples of regulatory systems are those which cause the expression of the gene to be turned on or off in response to a chemical or physical stimulus, including the presence of a regulatory compound. Regulatory systems in prokaryotic systems include the lac, tac, and trp operator systems. In yeast, tiie ADH2 system or GAL1 system may be used. In filamentous fungi, the TAKA alpha-amylase promoter, Aspergillus niger glucoamylase promoter, and Aspergillus oryzae glucoamylase promoter may be used as regulatory sequences. Other examples of regulatory sequences are tiiose which allow for gene amplification. In eukaryotic systems, these include the dihydrofolate reductase gene which is amplified in the presence of methotrexate, and the metallothioneln genes which are amplified with heavy metals. In these cases, the nucleic acid sequence encoding tiie polypeptide would be operabiy linked with the regulatory sequence.<br>
Expression Vectors<br>
The present invention also relates to recombinant expression vectors comprising a nucleic acid sequence of the present invention, a promoter, and transcriptional and translational stop signals. The various nucleic acid and control sequences described above may be joined together to produce a recombinant expression vector which may include one<br><br>
or more convenient restriction sites to allow for insertion or substitution of the nucleic acid sequence encoding the polypeptide at such sites. Alternatively, the nucleic acid sequence of the present invention may be expressed by inserting the nucleic acid sequence or a nucleic acid construct comprising the sequence into an appropriate vector for expression. In creating the expression vector, the coding sequence is located in the vector so that the coding sequence is operably linked with the appropriate control sequences for expression.<br>
The recombinant expression vector may be any vector (e.g., a plasmid or virus) which can be conveniently subjected to recombinant DNA procedures and can bring about the expression of the nucleic acid sequence. The choice of the vector will typically depend on the compatibility of the vector with the host cell into which the vector is to be introduced. The vectors may be linear or closed circular plasmids.<br>
The vectar may be an autonomously replicating vector, /.e., a vector which e)dsts as an extrachromosomal entity, the replication of which is independent of chromosomal replication, e.g., a plasmid. an extrachromosomal element, a minichromosome, or an artificial chromosome. The vector may contain any means for assuring self-replication. Alternatively, the vector may be one which, when introduced into the host cell, is integrated into the genome and replicated together with the chromosome(s) into which It has been integrated. Furthemnore, a single vector or plasmid or two or more vectors or plasmids which together contain the total DNA to be introduced into the genome of the host cell, or a transposon may be used.<br>
The vectors of the present invention preferably contain one or more selectable markers which pennit easy selection of transformed cells. A selectable marker is a gene the product of which provides for biocide or viral resistance, resistance to heavy metals, prototrophy to auxotrophsi, and the like. Examples of bacterial selectable markers are the dal genes from Bacillus subtilis or Bacillus licheniformis, or markers which confer antibiotic resistance such as ampicillin, kanamycin, chloramphenicol or tetracycline resistance. Suitable markers for yeast host cells are ADE2, HIS3. LEU2, LYS2. MET3. TRP1. and URA3. Selectable markers for use in a filamentous fungal host cell include, but are not limited to, amdS (acetamidase). argB (omithine carbamoyltransferase), bar (phosphinothricin acetyltransferase). iiygB (hygromycin phosphotransferase), niaD (nitrate reductase), pyrG (orotidine-5'-phosphate decarboxylase), sC (sulfate adenyltransferase), trpC (anthranllate synthase), as well as equivalents thereof. Preferred for use in an Aspergillus cell are the amdS and pyrG genes of Aspergillus nidular)s or Aspergillus oryzae and the bar gene of Streptomyces hygroscopicus.<br>
The vectors of the present invention preferably contain an element(s) that pemiits stable integration of the vector into the host cell's genome or autonomous replication of the vector in the cell independent of the genome.<br><br>
For integration into the host cell genome, the vector may rely on the nucleic acid sequence encoding the polypeptide or any other element of the vector for stable integration of the vector into the genome by homologous or nonhomologous recombination. Alternatively, the vector may contain additional nucleic add sequences for directing integration by homologous recombination into the genome of the host cell. The additional nucleic acid sequences enable the vector to be integrated into the host ceil genome at a precise location(s) in the chromosome(s). To increase the likelihood of integration at a precise location, the integrational elements should preferably contain a sufficient number of nucleic acids, such as 100 to 1,500 base pairs, preferably 400 to 1,500 base pairs, and most preferably 800 to 1,500 base pairs, &gt;Artiich are highly homologous with the corresponding target sequence to enhance the probability of homologous recombination. The integrational elements may be any sequence that is homologous with the target sequence in the genome of the host cell.<br>
Furthennore, the integrational elements may be non-encoding or encoding nucleic acid sequences. On the other hand, the vector may be integrated into the genome of the host cell by non-homologous recombination.<br>
For autonomous replication, the vector may further comprise an origin of replication enabling the vector to replicate autonomously in the host cell in question. Examples of bacterial origins of replication are the origins of replication of plasmids pBR322, pUC19, pACYC177, and pACYC184 pennitting replication in £ cofi, and pUBHO, pE194, pTAIOSO, and pAMBI permitting replication in Bacillus. Examples of origins of replication for use in a yeast host cell are the 2 micron origin of replication. ARS1, ARS4, the combination of ARS1 and CEN3, and the combination of ARS4 and CEN6. The origin of replication may be one having a mutation which makes its functioning temperature-sensitive in the host cell (see, e.g., Ehriich, 1978, Proceedings of the National Academy of Sciences USA 75:1433).<br>
More than one copy of a nucleic acid sequence of the present invention may be inserted into the host cell to increase production of the gene product. An increase in the copy number of the nucleic acid sequence can be obtained by integrating at least one additional copy of the sequence into the host cell genome or by including an amplifiable selectable marker gene with the nucleic acid sequence where cells containing amplified copies of the selectable marker gene, and thereby additional copies of the nucleic acid sequence, can be seiected for by cultivating the cells in the presence of the appropriate selectable agent.<br>
The procedures used to ligate the elements described above to construct the recombinant expression vectors of the present invention are well known to one skilled in the art (see, e.g., Sambrook et a/., 1989, supra).<br>
Host Cells<br><br>
The present invention also relates to recombinant host cells, comprising a nucleic acid sequence of the invention, which are advantageously used in the recombinant production of the polypeptides.<br>
A vector comprising a nucleic acid sequence of the present invention is introduced into a host cell so that the vector Is maintained as a chromosomal Integrant or as a self-replicating extra-chromosomal vector as described eariier. The choice of a host cell will to a large extent depend upon the gene encoding the polypeptide and its source.<br>
The host cell may be a unicellular microorganism, e.g., a prokaryote, or a non-unicellular microorganism, e.g.. a eukaryote.<br>
Useful unicellular cells are bacterial cells such as gram positive bacteria including, but not limited to, a Bacillus cell, e.g., Bacillus alkalophllus, Bacillus amyloliquefaciens, Bacillus brevis, Bacillus cimulans, Bacillus clausii, Bacillus coagulans. Bacillus lautus, Bacillus lentus. Bacillus licheniformis, Bacillus megaterium, Bacillus stearothermophilus, Bacillus subtilis, and Bacillus thuringiensis] or a Streptomyces cell, e.g., Streptomyces lividans or Streptomyces murinus, or gram negative bacteria such as £. coll and Pseudomonas sp. In a preferred embodiment, the bacterial host cell is a Bacillus lentus, Bacillus lichenlformis, Bacillus stearothenvophilus, or Bacillus subtilis cell. In another prefered embodiment, the Bacillus cell is an alkalophilic Bacillus,<br>
The introduction of a vector into a bacterial host cell may, for instance, be effected by protoplast transfomnation (see, e.g., Chang and Cohen, 1979, Molecular General Genetics 168: 111-115). using competent cells (see, e.g.. Young and Spizizin, 1961. Journal of Bacteriology 81: 823-829, or Dubnau and Davidoff-Abelson, 1971, Journal of Molecular Biology 56: 209-221). electroporation (see, e.g., Shigekawa and Dower, 1988, Biotechniques 6: 742-751), or conjugation (see, e.g., Koehler and Thome, 1987, Journal of Bacteriology 169:5771-5278).<br>
The host cell may be a eukaryote, such as a mammalian, insect, plant, or fungal cell. In a preferred embodiment, the host cell is a fungal celL "Fungi" as used herein includes the phyla Ascomycota, Basidiomycota, Chytridiomycota, and Zygomycota (as defined by Hawksworth et ai, In, Ainsworth and Bisby's Dictionary of The Fungi, 8th edition, 1995, CAB Internationa!. University Press, Cambridge, UK) as well as the Oomycota (as cited in Hawksworth etaL, 1995, supra, page 171) and all mitosporic fungi (Hawksworth etaL, 1995, supra).<br>
In a more preferred embodiment, the fungal host cell is a yeast cell. 'Yeasf as used herein includes ascosporogenous yeast (Endomycetales), basidiosporogenous yeast, and yeast belonging to the Fungi Imperfect! (Blastomycetes). Since the classification of yeast may change in the future, for the purposes of this invention, yeast shall be defined as<br><br>
described in Biology and Activities of Yeast (Skinner, F.A., Passmore, S.M., and Davenport, R.R., eds, Soc. App. BacterioL Symposium Series No. 9,1980).<br>
In an even more prefen'ed embodiment, the yeast host cell is a Candida, Hansenula, Kluyveromyces, PIcfila, Saccharomyces, Schizosacctiaromyces, or Yarrowia<br>
cell.<br>
In a most preferred embodiment, the yeast host cell is a Saccharomyces carlsbergensis, Saccharomyces cerevisiae, Saccliaromyces diastaticus, Sacctiaromyces douglasii, Saccharomyces l<luyverl saccharomyces norbensis or oviformis cell. in another most preferred embodiment the yeast host cell is a kluyveromyces lactis prefen-ed yarrowia lipolytica></luyverl>
In another more preferred embodiment, the fungal host cell is a filamentous fungal cell. "Filamentous fungi" include all filamentous forms of the subdivision Eumycota and Oomycota (as defined by Hawksworth et al., 1995, supra). The filamentous fungi are characterized by a mycelial wall composed of chitin, cellulose, glucan, chitosan, mannan, and other complex polysaccharides. Vegetative growth is by hyphal elongation and carbon catabolism is obligately aerobic. In contrast, vegetative growth by yeasts such as Saccharomyces cerevlslae is by budding of a unicellular thallus and carbon catabolism may be fermentative.<br>
In an even more preferred embodiment, the filamentous fungal host cell is a cell of a species of, but not limited to, Acremonium, Aspergillus, Fusarium, Humicola, Mucor, Myceliophthora, Neurospora, Penicillium, Thielavia, Tolypocladium, or Trichoderma,<br>
In a most preferred embodiment, the filamentous fungal host cell is an Aspergillus awamori, Aspergillus foetidus, Aspergillus japonicus, Aspergillus nidulans, Aspergillus niger or Aspergillus oryzae cell. In another most prefen^ed embodiment, the filamentous fungal host cell is a Fusarium bactridioides, Fusarium cerealis, Fusarium crookwellense, Fusarium culmorum, Fusarium graminearum, Fusarium gramlnum, Fusarium heterosporum, Fusarium negundi, Fusarium oxysporum, Fusarium reticulatum, Fusarium roseum, Fusarium sambucinum, Fusarium sarcochroum, Fusarium sporotrichioldes, Fusarium sulphureum, Fusarium torulosum, Fusarium trichothecioides, or Fusarium venenatum cell. In an even most preferred embodiment, the filamentous fungal parent cell is a Fusarium venenatum (Nirenberg sp. nov.) cell. In another most preferred embodiment, the filamentous fungal host cell is a Humicola insolens, Humicola lanuginosa, Mucor miehei, Myceliophthora thermophlla, Neurospora crassa, Penicillium purpurogenum, Thielavia terrestris, Trichoderma harzianum, Trichoderma koningii, Trichoderma longibrachiatum, Trichoderma reesei, or Trichoderma virlde cell.<br>
Fungal cells may be transformed by a process involving protoplast formation, transformation of the protoplasts, and regeneration of the cell wall in a manner known per<br><br>
se. Suitable procedures for transfomaation of Aspergillus host cells are described in EP 238 023 and Yelton et al, 1984, Proceedings of the National Academy of Sciences USA 81: 1470-1474. Suitable methods for transforming Fusarium species are described by Malardler et ai., 1989, Gene 78: 147-156 and WO 96/00787. Yeast may be transfomied using the procedures described by Becker and Guarente, in Abelson, J.N. and Simon, M.I., editors, Guide to Yeast Genetics and Molecular Biology, Mettiods in Enzymology, Volume 194, pp 182-187. Academic Press, Inc., New York; Ito et aL, 1983, Journal of Bacteriology 153: 163; and Hinnen ef a/., 1978, Proceedings of the National Academy of Sciences USA 75:1920.<br>
Methods of Production<br>
The present invention also relates to methods for producing a polypeptide of the present invention, the method comprising (a) cultivating a strain from the genus Humicola or the genus Meripilus, to produce a supernatant comprising the polypeptide; and (b) recovering the polypeptide. Preferably, the strain is of the species (Humicola insolens or the species Meripilus giganteus.<br>
The present invention also relates to a method for producing a polypeptide of the invention, the method comprising (a) cultivating a recombinant host cell as described above under conditions conducive to the production of the polypeptide, and (b) recovering the polypeptide from the ceils and/or the culture medium.<br>
In the production methods of the present invention, the cells are cultivated in a nutrient medium suitable for production of the polypeptide using methods known in the art. For example, the cell may be cultivated by shake flask cultivation, small-scale or large-scale femnentation (including continuous, batch, fed-batch, or solid state fermentations) in laboratory or industrial fermentors performed in a suitable medium and under conditions allowing the polypeptide to be expressed and/or isolated. The cultivation takes place in a suitable nutrient medium comprising carbon and nitrogen sources and inorganic salts, using procedures known in the art Suitable media are available from commercial suppliers or may be prepared according to published compositions (e.g., in catalogues of the American Type Culture Collection). If the polypeptide is secreted into the nutrient medium, the polypeptide can be recovered directly from the medium. If the polypeptide is not secreted, it can be recovered from cell lysates.<br>
The polypeptides may be detected using methods known in the art: that are specific for the polypeptides. These detection methods may include use of specific antibodies, formation of an en2yme product, or disappearance of an enzyme substrate. For example, an enzyme assay may be used to determine the activity of the polypeptide as described herein.<br><br>
The resulting polypeptide may be recovered by methods l<nown in the art. for example polypeptide may be recovered from nutrient medium by conventional procedures including but not limited to centrifugation filtration extraction spray-drying evaporation or precipitation.></nown>
The polypeptides of the present invention may be purified by a variety of procedures Icnown in the art including, but not limited to, chromatography (e.g., ion exchange, affinity, hydrophobic, chromatofocusing, and size exclusion), electrophoretic procedures (e.g., preparative isoelectric focusing), differential solubility (e.g., ammonium sulfate precipitation), SDS-PAGE, or extraction (see, e.g., Protein Purification, J.-C. Janson and Lars Ryden, editors, VCH Publishers. New York. 1989).<br>
Expression of the enzymes in plants<br>
A DNA sequence encoding a polypeptide of interest, such as an arabinofuranosidase of the present invention, may be transfonned and expressed In transgenic plants as described below.<br>
The transgenic plant can be dicotyledonous or monocotyledonous. for short a dicot or a monocot Examples of monocot plants are grasses, such as meadow grass (blue grass, Poa). forage grass such as Festuca, Lolium, temperate grass, such as Agrostis, and cereals, e.g. wheat, oats, rye, bariey, rice, sorghum and maize (com).<br>
Examples of dicot plants are tobacco, legumes, such as lupins, potato, sugar beet, pea, bean and soybean, and cruciferous plants (family Brassicaceae), such as cauliflower, oil seed rape and the closely related model organism Arabidopsis ttialiana.<br>
Examples of plant parts are stem, callus, leaves, root, fruits, seeds, and tubers as well as the individual tissues comprising these parts, e.g. epidermis, mesophyll, parenchyme, vascular tissues, meristems. In the present context, also specific plant cell compartments, such as chloroplast, apoplast, mitochondria, vacuole, peroxisomes and cytoplasm are considered to be a plant part. Furthennofe, any plant cell, whatever the tissue origin, is considered to be a plant part. Likewise, plant parts such as specific tissues and cells isolated to facilitate the utilisation of the invention are also considered plant parts e.g. embryos, endospemns, aleurone and seeds coats.<br>
Also included within the scope of the invention are the progeny of such plants, plant parts and plant cells.<br>
The transgenic plant or plant cell expressing the polypeptide of interest may be constructed in accordance with methods known in the art. In short the plant or plant cell is constructed by incorporating one or more expression constructs encoding the polypeptide of interest into the plant host genome and propagating the resulting modified plant or plant cell into a transgenic plant or plant cell.<br><br>
Conveniently, the expression construct is a DNA construct which comprises a gene encoding the polypeptide of interest in operable association with appropriate regulatory sequences required for expression of the gene in the plant or plant part of choice. Furthermore, the e)q3ression construct may comprise a selectable marker useful for identifying host cells into which the expression construct has been integrated and DNA sequences necessary for introduction of the construct into the plant in question (the latter depends on the DNA introduction method to be used).<br>
The choice of regulatory sequences, such as promoter and terminator sequences and optionally signal or transit sequences is determined, e.g. on tiie basis of when, where and how the enzyme is desired to be expressed. For instance, the expression of the gene encoding the enzyme of the invention may be constitutive or inducible, or may be developmental, stage or tissue specific, and the gene product may be targeted to a specific cell compartment, tissue or plant part such as seeds or leaves. Regulatory sequences are, e.g. described by Tague et al, Plant, Phys., 86, 506,1988.<br>
For constitutive expression the 35S-CaMV, the maize ubiquitin 1 and the rice actin 1 promoter may be used (Francis et al. 1980. Cell 21: 285-294, Christensen AH, Shanroclc RA and Quail 1992. Maize polyubiquitin genes: structure, thermal perturbation of expression and transcript splicing, and promoter activity following transfer to protoplasts by electroporation. Plant Mo. Biol. 18, 675-689.; Zhang W, McElroy D. and Wu R 1991, Analysis of rice Act1 5' region activity in transgenic rice plants. Plant Cell 3, 1155-1165). Organ-specific promoters may, e.g. be a promoter from storage sink tissues such as seeds, potato tubers, and fruits (Edwards &amp; Coruzzi, 1990. Annu. Rev. Genet. 24: 275-303), or from metabolic sink tissues such as meristems (Ito et al., 1994. Plant Mol. Biol. 24; 863-878), a seed specific promoter such as the glutelin, prolamin, globulin or albumin promoter fn^m rice (Wu et al., Plant and Cell Physiology Vol. 39, No. 8 pp. 885-889 (1998)), a Vicia faba promoter from the legumin 84 and the unknown seed protein gene from Vicia faba described by Conrad U. et al. Journal of Plant Physiology Vol. 152, No. 6 pp. 708-711 (1998), a promoter from a seed oil body protein (Chen et al.. Plant and cell physiology vol. 39, No. 9 pp. 935-941 (1998), the storage protem napA promoter from Brassica napus, or any other seed specific promoter known in the art, e.g. as described in WO 91/14772. Furttiemnore, the promoter may be a leaf specific promoter such as the rbcs promoter from rice or tomato (Kyozuka et aL, Plant Physiology Vol. 102, No, 3 pp. 991-1000 (1993), the chlorella virus adenine methyltransferase gene promoter (Mitra, A. and Higgins, DW, Plant Molecular Biology Vol. 26, No. 1 pp. 85-93 (1994), or the aldP gene promoter from rice (Kagaya et al., Molecular and General Genetics Vol. 248, No. 6 pp. 668-674 (1995), or a wound inducible promoter such as the potato pin2 promoter (Xu et al. Plant Molecular Biology Vol. 22, No. 4 pp. 573-588 (1993). Likewise, the promoter may Indudble by abiotic treatments such as temperature,<br><br>
drought or alterations in salinity or induced by exogenousiy applied substances that activate the promoter e.g. ethanoi, oestrogens, plant hormones like ethylene, abscisic acid and gibberellic acid and heavy metals.<br>
A promoter enhancer element may be used to achieve higher expression of the enzyme in the plant For instance, the promoter enhancer element may be an intron which is placed between the promoter and the nucleotide sequence encoding ttie enzyme. For instance, Xu et al. op cit disclose the use of the first intron of the rice actin 1 gene to enhance expression.<br>
The selectable marker gene and any other parts of the expression construct may be chosen from those available in the art.<br>
The DNA construct is incorporated into the plant genome according to conventional techniques known in the art, including Agrobacterium-mediated transformation, virus-mediated transformation, micro injection, particle bombardment, biolistic transformation, and electroporation (Gasser et al. Science, 244, 1293; Potrykus, Bio/Techn. 8, 535, 1990; Shimamoto et al, Nature, 338, 274,1989).<br>
Presently, Agrobacterium tumefaciens mediated gene transfer is the method of choice for generating transgenic dicots (for review Hooykas &amp; Schilperoort, 1992. Plant Mol. Biol. 19: 15-38), and can also be used for transforming monocots, although other transformation methods often are used for these plants. Presently, the method of choice for generating transgenic monocots supplementing the Agrobacterium approach Is particle bombardment (microscopic gold or tungsten particles coated with the transforming DNA) of embryonic calii or developing embryos (Christou, 1992. Plant J. 2: 275-281; Shimamoto, 1994. Cun-. Opin. Biotechnol. 5: 158-162; Vasil et al., 1992. Bio/Technology 10: 667-674). An alternative method for transformation of monocots is based on protoplast transformation as described by Omirulleh S, et al., Plant Molecular biology Vol. 21, No. 3 pp. 415-428 (1993).<br>
Following transfonnation, the transformants having incorporated the expression construct are selected and regenerated into whole plants according to methods well-known in the art. Often the transformation procedure is designed for the selective elimination of selection genes either during regeneration or in the following generations by using e.g. co-transformation with two separate T-DNA constructs or site specific excision of the selection gene by a specific recombinase.<br>
Use of arabinofuranosidases<br>
The present invention also relates to use of the polypeptides of the invention, i.e. the arabinofuranosidases, in various industrial application, e.g. in biomass conversion, such as in production of fuel ethanol from cellulose containing biomass, in pnDduction of fuel<br><br>
and/or potable ethanol from starch, in mashing for beer production, in a dough for bread making, or in manufacture of an animal feed product,.<br>
The invention furthermore provides a process wherein a arabinoxylan containing substrate and/or a biomass is contacted with an arabinofuranosidase capable of releasing arabinose from di-substitued xyloses. Preferably the alpha-L-arabinofuranosidase is an arabinofuranosidase of GH43. The alpha-L-arabinofuranosidase of GH43 is preferably derived of baterial, of fungal or of plant origin. Preferably the arabinoxylan containing substrate and/or the biomass is selected from the list consisting of herbaceous and/or woody energy crops, agricultural food and feed crops, animal feed products, tubers, roots, stems, legumes, cassava peels, cocoa pods, rice husks and/or hulls, rice bran from rice polishing, cobs, straw, hulls and/or husks from cereal grain, pressed sugar cane stalk, sugar beet pulp, locust bean pulp, vegetable or fruit pomaces, cereals or whole grain agricultural crop waste, straw, stalks, leaves, com bran, husks, cobs, rind, shells, pods, wood waste, bark, shavings, sawdust, wood pulp, pulping liquor, waste paper, cardboard, construction and demolition wood waste, industrial or municipal waste water solids or sludge, manure, by-product from brewing and/or fermentation processes, wet distillers grain, dried distillers grain, spent grain, vinasse and bagasse.<br>
The present invention also relates to compositions comprising the polypeptides of the invention, i.e. the arabinofuranosidase, as well as to uses of such compositions.<br>
MATERIALS AND METHODS<br>
Arabinose and xylose were purchased from Merck (Darmstadt, Germany). Water soluble and water insoluble wheat arabinoxylans were obtained from Megazyme (Bray, County Wicklow, Ireland).<br>
Enzymes<br>
The alpha-L-arabinofuranosidases were cloned using basic molecular techniques (Sambrook et al., 1989, Molecular Clor}ing, A Laboratory Manual, 2d edition. Cold Spring Harbor, New York. Christgau et al. 1995, Cun". Genet. 27. 135-141, Ausubel et al., 2003, Cun". Prot. Mol.BioL. John Wiley &amp; Sons, Cambridge. USA.).<br>
The Bifidobacterium adolescentis alpha-L-arabinofuranosidases of Van Laere et al. (1997) and Van den Broek et al. (2005) was obtained from Megazyme (Ireland).<br>
Shearzyme (GH10) and Pentopan Mono (GH11), monocomponent endo-1,4-p-xylanase preparations produced by Aspergillus aculeatus and Thermomyces lar^uginosus, respectively, were commercial products from Novozymes A/S (Bagsvaerd, Denmark).<br><br>
Preparation of specific arabinoxylan polymers and oligosaccharides<br>
Doubly substituted arabinoxylan was prepared by incubating soluble wheat arabinoxylan (1g) in 0.1 M acetate buffer (100 mL), pH 6.0 with 0.167 g a-L-arabinofuranosjdase from Meripilus giganteus (GH51)'kg""' water soluble wheat arabinoxylan for 48 hours at 30°C. Singly substituted arabinoxylan was prepared by incubating water soluble wheat arabinoxylan (1g) in 0.1 M acetate buffer (42 mL), pH 6.0 with 0.147 g a-L-arabinofuranosidase from Humicola insolens (GH43) -kg"^ water soluble wheat arabinoxylan for 48 hours at SO'^C, To halt the enzymatic reactions the mixtures were heated to 100°C for 10 min. Arabinoxylan polymers were precipitated by addition of ethanol (126 ml). The precipitates were filtered (Miracloth) and dryed in vacuum<br>
Oligosaccharides containing arabinosyl groups linked to terminal (1-^3) were prepared by incubating the water insoluble wheat arabinoxylan (1g) in 0.1 l\^ acetate buffer (100 mL), pH 6.0 with 6,67 g Shearzyme (xylanase GH10) •kg"' water insoluble wheat arabinoxylan for 2 hours at 30°C, Oligosaccharides containing arabinosyl groups linked to internal (1-*3) were prepared by incubating water insoluble wheat arabinoxylan (1g) in 0.1 M acetate buffer (100 mL), pH 6.0 with 0.03 g Pentopan Mono (xylanase GH11) •kg"' water insoluble wheat arabinoxylan for 2 hours at 30°C. Oligosaccharides containing arabinosyl groups linked to internal (1-)-2) were prepared by incubating water insoluble wheat arabinoxylan (1g) in 0.1 M acetate buffer (100 mL), pH 6.0 with 0.03 g Pentopan Mono (xylanase GH11) •kg'"' water insoluble wheat arabinoxylan and alpha-L-arabinofuranosidase from H. insolens (GH43) ■kg"' water soluble wheat arabinoxylan for 2 hours at 30°C. To halt the enzymatic reactions the mixtures were heated to 100°C for 10 min. The arabinoxylo-oligosaccharides were concentrated on a rotary evaporator and evaluated by ""H-NMR.<br>
Assay for activity towards alpha-L»arabinofuranosidase activity<br>
Alpha-L-arabinofuranosidase activity may be assessed as described by Poutanen et al. (Appl. Microbiol. Biotechnol. 1988. 28, 425-432) using 5 mM p-nitrophenyl alpha-L-arabinofuranoside as substrates. The reactions may be carried out in 50 mM citrate buffer at pH 6.0, 40^0 with a total reaction time of 30 min. The reaction Is stopped by adding 0.5 ml of 1 M sodium carbonate and the liberated p-nltrophenol is measured at 405 nm. Activity is expressed in U/ml.<br>
Assay for activity towards C2- and C3-di-substltuted xylan<br>
Medium viscosity water-soluble wheat arabinoxylan (Megazyme. Bray, Ireland) was treated with an alpha-arabinofuranosidase of GH51 from Meripilus giganteus (SEQ ID NO:2) to remove single alpha-arabinofuranosyl substituents attached to the C(0)-3 arabinose of the  arabinoxylan  in  order to  produce  an  dl-substituted  arabinoxylan  substrate with<br><br>
arabinofuranosyl substituents attached to both C(0)-2,3 of the xylose residues. The substrate was dialysed and freeze dried.<br>
A 0.1% solution of the di-substituted arabinoxylan was prepared and the alpha-arabinofuranosidase activity was measured by mixing 0.1 ml enzyme, 0.9 ml buffer (0.12 M Succinic acid, pH 6.0) and 1.0 ml substrate solution in an eppendorf tube. The eppendorf tube was incubated at 60°C for 1 hour with shaking. The amount of liberated arabinose was measured by HPAEC (high-performance anion-exchange chromatography).<br>
HPAEC<br>
Hydrolysates (10 jil) were applied onto a Dionex BioLC system fitted with a Dionex CarboPac™ PA1 guard column (4 x 250 mm) (Dionex Corporation, Sunnyvale, CA, USA) combined with a CarboPac™ PA1 precolumn (4 x 50 mm). The monosaccharides were separated isocratically with 10 mlW KOH for 15 min. flow; 1 mL'min"\ Monosaccharides were detected by a pulsed eiectrochemical detector in the pulsed amperiometric detection mode. The potential of the electrode was pnagrammed for +0.1 V (t = 0-0.4 s) to -2.0 V (t = 0,41-0.42 s) to 0.6 V (t = 0.43 s) and finally -0.1 V (t = 0.44-0.50 s). while integrating the resulting signal from t = 0.2-0.4 s. A mixture of arabinose and xylose (concentration of each component: 0.0025-0.1 g-L""*) was used as standard.<br>
^H-NMR analysis<br>
All degradation products were lyophillzed twice from 99.9% D2O and re-dissolved in 99.9% D2O. Some hydrolysates were dialyzed (Spectra/Por membrane molecular weight cut-off 1000) to remove free arabinose prior to the spectral analysis. The ^H-NMR spectra were recorded at 30°C in a Varian Mercury-VX instrument operated at 400 MHz and equipped with a 4-nucleus auto-switchable probe. Data were collected over 128-512 scans and the HDO signal was used as a reference signal (4.67 ppm).<br>
EXAMPLES Example 1<br>
Wheat arabinoxylan comprises arabinofuranoside as a monosubstituent linked to the 3-position of intemal xylose (A) and arabinofuranoside linked to the 3- (B) and 2-position (C) on di-substituted xylose, respectively. Substrates were produced each comprising only one of the 3 types of arabinofuranoside linkages. The activity of arabinofuranosidases towards these substrates was investigated using ^i-l NMR.<br><br><br>
Example 2<br>
Soluble wheat arabinoxylan was incubated with 0.1 g enzyme protein per kg DM of alpha -L-arabinofuranosidase from H. insolens (GH43), S. adolescentjs (GH43), H. insolens (GH51), and M. giganteus (GH51) for 24 hrs. The released arabinose was measured. Results are expressed in mg ar&amp;binose per g water soluble wheat arabinoxylan, as the average of :riplicate detenminations, coefficient of variation on mean 
 <br><br>
Example 3<br>
Arabionofuranosidases may be applied in animal feed compositions to increase digestibility. Com arabinoxylan is heavily di-substituted with arabinose. In order to facilitate the xylan degradation it is advantageous to remove as many as possible of the arabinose substituents. The in vitro degradation of arabinoxylans in a com based diet supplemented with the GH51 arabinofuranosidase from Humicola insolens and a commercial xylanase derived from Temiomyces lanuginosus was studied in an in vitro digestion system.<br>
Conditions for the vitro digestion system:<br>
Substrate:	800 mg 30:70 Soy bean meal/Com diet milled and pre-<br>
blended<br>
pH:	pH 5, 4 and 3 (gastric) and 4.0 (transition step) followed<br>
by a small intestinal step at pH 6.8<br>
Pepsin:	3000 U/g diet.<br>
Pancreatin	8 mg/g diet<br>
Temperature:	40 **C<br>
Number of replicates:	4<br>
Enzyme dosage:	Arabinofuranosidase and xylanase 50 and 67 mg EP/l<g></g>
diet respectively<br>
Samples were precipitated with 80 % ethanol, centrifuged and supematant discarded. The pellet was washed in 80 % ethanol and centrifuged and supematant discarded. The pellet was dissolved in acetate buffer (pH 5) and starch was removed before dietary fibre analysis.<br><br>
Table 4. Content of arabinose and xylose In residual total non-starch polysaccharide<br><br>
CLAIMS<br>
1	An arabinofuranosidase capable of releasing arabinose from di-substitued xylan,<br>
said arabinofuranosidase derived from a fungus.<br>
An arabinofuranosidase which is:<br>
a)	a polypeptide having an amino acid sequence as the mature peptide shown in<br>
SEQ ID NO: 2. or which can be obtained therefrom by substitution, deletion, and/or<br>
insertion of one or more amino acids;<br>
b)	an analogue of the polypeptide defined in (!) or (ii) which:<br>
i) has at least 80 % homology with said polypeptide,<br>
ii)       is an allelic variant of said polypeptide,<br>
c)	a polypeptide which is encoded by a nucleic acid sequence which hybridizes<br>
under high stringency conditions with a complementary strand of the nucleic acid<br>
sequence of SEQ ID N0:2 encoding the mature polypeptide or a subsequence<br>
thereof having at least 100 nucleotides.<br>
An arabinofuranosidase which is:<br>
a)	a polypeptide having an amino acid sequence as the mature peptide shown in<br>
SEQ ID NO: 4, or which can be obtained therefrom by substitution, deletion, and/or<br>
insertion of one or more amino acids;<br>
b)	an analogue of the polypeptide defined in (i) or (ii) which:<br>
i) has at least 60 % homology with said polypeptide,<br>
Ii)       is an allelic variant of said polypeptide,<br>
c)	a polypeptide which is encoded by a nucleic acid sequence which hybridizes<br>
under high stringency conditions with a complementary strand of the nucleic acid<br>
sequence of SEQ ID NO:2 encoding the mature polypeptide or a subsequence<br>
thereof having at least 100 nucleotides.<br>
4.	The arabinofuranosidase of any of claims 1 to 3 which is native to a strain of Humicola, preferably K insolens or native to a strain of Menpilus, preferably Menpilus giganteus.<br>
5.	A nucleic acid sequence comprising a nucleic acid sequence which encodes the arabinofuranosidase of any of claims 1 to 4.<br>
6.	A nucleic acid sequence which comprises:<br><br>
a)	the DNA sequence encoding any of the arabinofuranosidases shown in SEQ ID NO:2 and SEQ ID N0:4,<br>
b)	an analog DNA sequence which has<br>
i)        has at least 80 % homology with any of said DNA sequences, or<br>
ii)       hybridizes at high stringency with a complementary strand of said DNA<br>
sequence or a subsequence thereof having at least 100 nucleotides, iii)       is an allelic variant thereof, or<br>
c) a complementary strand to a) or b).<br>
J, The nucleic acid sequence according to claim 6, wherein the polypeptide is an artificial variant comprising an amino acid sequence that has one or more truncation{s), and/or at least one substitution, deletion, and/or insertion of an amino acid as compared to amino acids 1 to 558 of SEQ ID N0:2.<br>
8. A nucleic acid sequence which has at least 80 % homology with any of the DNA<br>
sequences shown in SEQ ID N0:1 and SEQ ID N0:3, or<br>
a) hybridizes at high stringency with a complementary strand of any of said DNA sequences or a subsequence thereof having at least 100 nucleotides,<br>
b) is an allelic variant thereof, or<br>
c) a complementary strand to a) or b).<br>
9. The nucleic acid sequence according to claim 8, wherein the polypeptide is an<br>
artificial variant comprising an amino acid sequence that has one or more<br>
truncation(s), and/or at least one substitution, deletion, and/or insertion of an amino<br>
acid as compared to amino acids 1 to 643 of SEQ ID N0:4.<br>
10. A nucleic acid construct comprising the nucleic acid sequence of any of claims 5 to 9 operably linked to one or more control sequences capable of directing the expression of the arabinof uranosidase in a suitable expression host.<br>
11. A recombinant expression vector comprising the nucleic acid construct of claim 10.<br>
12. A recombinant host cell comprising the expression vector of claim 11.<br>
13. A method for producing an arabinofuranosidase comprising cultivating the host cell of<br>
claim 12 under conditions conducive to production of the arabinofuranosidase, and<br>
recovering the arabinofuranosidase.<br>
14. A composition comprising the arabinofuranosidase according to any of claims 1 to 4.<br>
15. A use of the arabinofuranosidase according to any of claims 1 to 4 in a dough.<br>
16. A use of the arabinofuranosidase according to any of claims 1 to 4 in an ethanol process.<br>
17. A use of the arabinofuranosidase according to any of claims 1 or 4 in a mashing process.<br>
18. A use of the arabinofuranosidase according to any of claims 1 or 4 in a process for producing a feed product.</td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/viewdoc.aspx?id=l+JxsdOkmCOGxXg1+LKlcg==&amp;amp;loc=egcICQiyoj82NGgGrC5ChA==" target="_blank" style="word-wrap:break-word;">http://ipindiaonline.gov.in/patentsearch/GrantedSearch/viewdoc.aspx?id=l+JxsdOkmCOGxXg1+LKlcg==&amp;amp;loc=egcICQiyoj82NGgGrC5ChA==</a></p>
		<br>
		<div class="pull-left">
			<a href="272135-hexarylene-and-pentarylene-tetracarboxylic-acid-diimides.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="272137-enhanced-process-for-the-purification-of-anyhydrous-hydrogen-chloride-gas.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>272136</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>4795/CHENP/2007</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>13/2016</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>25-Mar-2016</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>18-Mar-2016</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>26-Oct-2007</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>NOVOZYMES A/S</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td>KROGSHOEJVEJ 36, DK-2880 BOULEVARD,</td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>SOERENSEN, HANNE, RISBJERG</td>
											<td>VEJLEGAARDSVEJ 6, DK-2840 HOLTE,</td>
										</tr>
										<tr>
											<td>2</td>
											<td>JOERGENSEN, CHRISTEL, THEA</td>
											<td>FUGLEVADSVEJ 10, DK-2800 KRS. LYNGBY, DENMARK</td>
										</tr>
										<tr>
											<td>3</td>
											<td>CHRISTENSEN, LARS, HYLLING</td>
											<td>SANDHOLMGAARDSVEJ 49, DK-3450 ALLEROED, DENMARK</td>
										</tr>
										<tr>
											<td>4</td>
											<td>HANSEN, CARSTEN, HOERSLEV</td>
											<td>TOMSGAARDSVEJ 105, 1, TV., DK-2400 COPENHAGEN NV, DENMARK</td>
										</tr>
										<tr>
											<td>5</td>
											<td>KOFOD, LENE, VENKE</td>
											<td>BRORFELDEVEJ 8, DK-44350 UGGERLOESE, DENMARK</td>
										</tr>
										<tr>
											<td>6</td>
											<td>JOERGENSEN, CHRISTIAN, ISAK</td>
											<td>BONDEHAVEVEJ 92, DK-2880 BAGSVAERD, DENMARK</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>C12N 9/24</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>PCT/DK06/00213</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td>2006-04-25</td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td>PA 2005 00609</td>
									<td>2005-04-26</td>
								    <td>Denmark</td>
								</tr>
								<tr>
									<td>2</td>
									<td>PA 2005 01562</td>
									<td>2005-11-10</td>
								    <td>Denmark</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/272136-nucleic-acid-construct-comprising-nucleic-acid-sequence-encoding-polypeptide-having-arabinofuranosidase-activity by HTTrack Website Copier/3.x [XR&CO'2014], Thu, 04 Apr 2024 23:15:53 GMT -->
</html>
