<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/272361-method-for-producing-an-acidified-milk-drink by HTTrack Website Copier/3.x [XR&CO'2014], Thu, 04 Apr 2024 23:03:51 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 272361:&quot;METHOD FOR PRODUCING AN ACIDIFIED MILK DRINK&quot;</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">&quot;METHOD FOR PRODUCING AN ACIDIFIED MILK DRINK&quot;</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>The present invention relates to a method for producing an acidified milk drink using an enzyme which reduces the isoelectric point of the milk proteins. The invention also relates to a novel enzyme having deamidase activity and its use in production of an acidified milk drink.</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td>METHOD FOR PRODUCING AN ACIDIFIED MILK DRINK<br>
REFERENCE TO A SEQUENCE LISTING<br>
This application contains a Sequence Listing in computer readable form. The computer readable form is incorporated herein by reference.<br>
REFERENCE TO A DEPOSIT OF BIOLOGICAL MATERIAL<br>
This application contains a reference to a deposit of biological material, which deposit is incorporated herein by reference. For complete information see last page of the description.<br>
TECHNICAL FIELD<br>
The present invention relates to a method for producing an acidified milk drink using an enzyme which reduces the isoelectric point of the milk proteins. The invention also relates to a novel enzyme having deamidase activity and its use in production of an acidified milk drink.<br>
BACKGROUND OF THE INVENTION<br>
The market for acidified milk drinks, which includes fermented milk drinks and liquid yoghurt, is increasing worldwide and there is an interest in improving the quality and economics of this product.<br>
Acidified milk drinks are generally produced by mixing acidified milk with a sugar syrup solution, and subjecting the mixture to a homogenization treatment. Acidification may take place through addition of a chemical, such as glucono delta-lactone (GDL), or it may be caused by fermentation of the milk with lactic acid bacteria. When such products are stored, however, casein, an ingredient of milk, often precipitates or associates and aggregates, and as a result the drinks tend to separate so that liquid whey collects on the surface. This process, which is often referred to as syneresis, decreases the quality of the acidified milk drinks.<br>
Pectin, starch, modified starch, CMC, etc., are often used as stabilizers in acidified milk drinks, but due to the relatively high cost of such stabilizers, there is an interest in finding other and perhaps even better solutions. It is therefore an object of the present invention to provide a method for manufacturing of a stable acidified milk drink where the separation into curd and whey upon storage is reduced. The aim is to provide an alternative method for stabilization to either complement or partly replace the use of the stabilizers used in the art today.<br>
 <br>
Enzymes which are able to reduce the isoelectric point of proteins in a manner that does not alter the amino acid chain length of the protein are known in the prior art. For instance, deamidating enzymes which increase the negative charge of proteins by acting on the amide groups to generate carboxyl groups have been found (EP976829, US6756221). Deamidation of milk proteins by such enzymes has been described {WO2006075772, JP2003250460, WO2002068671). These publications teach that proteins, e.g. milk proteins, can be deamidated to improve a range of properties when the modified proteins are used in food items, e.g. dairy products containing such modified proteins are reported to have better taste and texture. However, these publications are silent with respect to the use of protein modifying enzymes in the production of acidified milk drinks.<br>
SUMMARY OF THE INVENTION<br>
The present inventors have surprisingly found that syneresis of acidified milk drinks can be decreased by adding an enzyme during the manufacture of the product which reduces the isoelectric point of the casein in the milk.<br>
Consequently, the present invention relates to a method for producing an acidified milk drink, said method comprising:<br>
a)	acidifying a milk substrate comprising milk protein; and<br>
b)	treating with an enzyme having a protein modifying activity which reduces the isoelectric point of the milk protein;<br>
wherein step b) is performed before, during or after step a).<br>
Furthermore, the inventors have identified a novel enzyme having deamidase activity which is applicable in the method of the invention.<br>
In another aspect, the invention therefore relates to an isolated polypeptide having deamidase activity, selected from the group consisting of:<br>
a)	a polypeptide having an amino acid sequence which is at least 91% identical to (i) SEQ ID NO: 2, or (ii) amino acids 133-318 thereof;<br>
b)	a polypeptide which is encoded by a polynucleotide which is at least 90% identical to (i) SEQ ID N0:1, or (ii) nucleotides 397-954 thereof;<br>
c)	a polypeptide which is encoded by a polynucleotide whose complement hybridizes under high stringency conditions with nucleotides 397-954 of SEQ ID NO: 1;<br>
d)	a polypeptide which is encoded by the plasmid contained in E. coli DSM 19445; and<br>
 <br>
e) a polypeptide having an amino acid sequence modified by substitution, deletion, and/or<br>
'9<br>
insertion of one or several amino acids in (i) SEQ ID NO: 2, or (ii) amino acids 133-318 thereof..<br>
BRIEF DESCRIPTION OF THE FIGURE<br>
Figure 1 shows a coomassie stained lEF gel with deamidase treated casein. The following samples were loaded on the lEF gel (from left): Lane 1: lEF marker; Lane 2: casein treated with supernatant of the deamidase clone; Lane 3: casein treated with supernatant prepared for loading to preparative lEF (this sample is about 4 times diluted compared to lane 2); Lanes 4-11: casein treated with pools of fractions from preparative lEF (pool 4 (lane 7) contains the deamidase enzyme); Lane 12 is the casein standard.<br>
DETAILED DISCLOSURE OF THE INVENTION<br>
Acidified milk drinks<br>
"Acidified milk drinks" according to the present invention include any drinkable products<br>
based on acidified milk substrates which can be produced according to the method of the<br>
invention. Acidification may take place because of addition of a chemical, such as lactic acid,<br>
citric acid  or glucono delta-lactone  (GDL),  or it  may be as  a fermentation with  a<br>
microorganism. Acidified milk drinks thus include fermented milk drinks and liquid yoghurt<br>
drinks.<br>
Acidified milk drinks according to the invention are drinkable in the sense that they are in liquid form and consumed as beverages, i.e. they are suitable for drinking instead of being eaten with a spoon. "In liquid form" means that the products are in the fluid state of matter thus exhibiting a characteristic readiness to flow. Thus, the shape of a liquid is usually determined by the container it fills, in contrary to, e.g., a gel-like substance, which is soft but not free flowing, such as e.g. yoghurt or pudding.<br>
An acidified milk drink according to the present invention may have a viscosity which is lower than 40 Pa (obtained at shear rate 300 pr. s), preferable less than 30 Pa, more preferably less than 20 Pa, and even more preferably between 15 and 20 Pa at a shear rate of 300 pr. s.<br>
Viscosity of acidified milk drinks may be determined as follows:<br>
Principle: This method is based on characterisation of texture by a viscometry measurement<br>
(constant rate). By a constant rate measurement, viscosity and shear rate are registered as<br>
<br>
 <br>
function of shear rate. Selected and calculated parameters from the flow curves are<br>
extracted.<br>
Materials: StressTech rheometer with CC 25 (bop / cup) measurement system.<br>
Procedure: Before the measurements are started, the samples must be tempered to the right<br>
temperature at 13°C.<br>
A flow curve is registered by increasing the deformation (shear rate: 0.2707 to 300 s'^)<br>
folllowed by a decreasing of deformation (shear rate : 300 to 0,2707 s-1).<br>
Settings:<br>
Normal force:<br>
•	Method: To Gab<br>
•	Max loading force: 10N<br>
•	Start measurement when normal force is below: 10N<br>
•	Timeout: lOOOsec.<br>
•	Approx. sample height: 1.000mm Shear rate:<br>
<br>
•	21 steps - up and down: 0.2707 - 0.3304 -0.4923-0.7334-1-2-4-6-10-25-50 - 75 - 100 - 125 -150 - 175 - 200-225 - 250 - 275 - 300.<br>
•	Delay time: 5sec.<br>
<br>
•	Integration time: lOsec. Temperature and time:<br>
•	Equilibrium time: -<br>
•	Manual: 1 repetition<br>
•	Manual control: 15° C<br>
•	Temp, after measurement: 13°C Advanced position:<br>
•	Position resolution: 85 y rad<br>
•	Approach to steady state, interval-: 0,010<br>
•	Strength: 60%<br>
•	Reference curve: off<br>
An acidified milk drink according to the present invention may have a pH of less than 4.6, preferably less than 4.4, more preferably less than 4.2 and even more preferably about pH 4 or less. In one aspect, the acidified milk drink has a pH of less than 3.8, such as less than 3.6.<br>
An acidified milk drink according to the invention may have a fat content of 0 to 2%, preferably below 1.5%, below 1%, below 0.5% or of about 0.1% or less. The acidified milk<br>
 <br>
drink may have a milk solid non-fat content of less than 20%, preferably less than 8.5%, less than 8%, less than 7.5%, less than 7%, less than 6.5%, less than 6%, or of about 5%.<br>
An acidified milk drink according to the invention may have a milk protein content of less than 8%, preferably less than 4%, less than 3.5% or less than 3%. In one preferred aspect, the acidified milk drink has a milk protein content of about 2.5 to about 3%. in another preferred aspect, the acidified milk drink has a milk protein content of less than 2%, preferably a milk protein content of about 1 %.<br>
An acidified milk drink according to the invention may have a shelf life of more than 7 days, preferably more than 14 days, more preferably more than 28 days, such as more than 3 months.<br>
Preferably, the acidified milk drink according to the present invention has an increased stability. The stability may be determined after having stored the acidified milk drink for an appropriate number of days by measuring the height of the whey collecting on the surface because of syneresis. It may also be determined after accelerated syneresis, such as by centrifugation.<br>
Method of producing an acidified milk drink<br>
As mentioned above, the method for producing an acidified milk drink according to the<br>
present invention comprises:<br>
a)	acidifying a milk substrate comprising milk protein; and<br>
b)	treating with an enzyme having a protein modifying activity which reduces the isoelectric point of the milk protein;<br>
wherein step b) is performed before, during or after step a).<br>
"Milk substrate", in the context of the present invention, may be any raw and/or processed milk material that can be subjected to acidification according to the method of the invention. Thus, useful milk substrates include, but are not limited to solutions/suspensions of any milk or milk like products comprising milk protein, such as whole or low fat milk, skim milk, buttermilk, reconstituted milk powder, condensed milk, dried milk, whey, whey permeate, lactose, mother liquid from crystallization of lactose, whey protein concentrate, or cream. Obviously, the milk substrate may originate from any mammal.<br>
Preferably, the milk substrate is a lactose solution/suspension and more preferably the substrate is milk. In a preferred embodiment, the milk substrate is an aqueous solution of skim milk powder.<br>
<br>
 <br>
r<br>
The milk substrate comprises milk protein, e.g. casein and whey protein. "Milk protein" in the context of the present invention may be any protein naturally occurring in milk. When referring to the milk protein in the acidified milk drink, the milk protein includes the enzymatically modified milk protein.<br>
The term "milk" is to be understood as the lacteal secretion obtained by milking any mammal, such as cows, sheep, goats, buffaloes or camels.<br>
In one embodiment, the milk substrate may be more concentrated than raw milk and it may thus have a protein content of more than 5%, preferably more than 6%, more than 7%, more than 8%, more than 9%, or more than 10%, and a lactose content of more than 7%, preferably more than 8%, more than 9%, or more than 10%.<br>
Prior to acidification, the milk substrate may be homogenized and pasteurized according to methods known in the art.<br>
"Homogenizing" as used herein means intensive mixing to obtain a soluble suspension or emulsion. If homogenization is performed prior to acidification, it may be performed so as to break up the milk fat into smaller sizes so that it no longer separates from the milk. This may be accomplished by forcing the milk at high pressure through small orifices.<br>
"Pasteurizing" as used herein means heating to and maintaining at a specified high temperature for a specified period of time. Such high temperature may be, e.g., at least 65°C, at least 70°C, at least 75°C, at least 80°C, at least 85°C, at least 90°C or at least 95°C, and the milk substrate may be maintained at such high temperature for, e.g., at least 15 sec, at least 20 sec, at least 30 sec, at least 5 minutes or at least 10 minutes. Such pasteurization will reduce or eliminate the presence of live organisms, such as microorganisms, in the milk substrate. Pasteurization of the milk substrate may also denature wholly or partly some of the milk protein. If performed after the enzyme treatment, pasteurization may also inactivate the enzyme. A rapid cooling step may follow.<br>
In the method of the present invention, the milk substrate is acidified. Such acidification may be a chemical acidification, such as by the addition of an acid, such as lactic acid or citric acid, or glucono delta-lactone (GDL), which is a cyclic ester of D-gluconic acid commonly used for acidification of food.<br>
<br>
 <br>
In a preferred aspect of the method of the invention, the acidification comprises fermentation<br>
of the milk substrate with a microorganism.	"<br>
"Fermentation" in the method of the present invention means the conversion of carbohydrates into alcohols or acids through the action of a microorganism. Preferably, fermentation in the method of the present invention comprises conversion of lactose to lactic acid.<br>
In the context of the present invention, "microorganism" may Include any bacterium or fungus being able to ferment the milk substrate.<br>
The microorganisms used for most fermented milk products are selected from the group of bacteria generally referred to as lactic acid bacteria. As used herein the term "lactic acid bacterium" designates a gram-positive, microaerophllic or anaerobic bacterium, which ferments sugars with the production of acids including lactic add as the predominantly produced acid, acetic acid and propionic acid. The industrially most useful lactic add bacteria are found within the order "Lactobacillales" which includes Lactococcus spp.. Streptococcus spp., Lactobacillus spp., Leuconostoc spp., Pseudoleuconostoc spp., Pediococcus spp., Brevibactehum spp., Enterococcus spp. and Propionibacterium spp. Additionally, lactic acid producing bacteria belonging to the group of the strict anaerobic bacteria, bifidobacteria. I.e. Bifidobacterium spp., which are frequently used as food cultures alone or in combination with lactic acid bacteria, are generally included in the group of lactic acid bacteria.<br>
Lactic acid bacteria are normally supplied to the dairy industry either as frozen or freeze-dried cultures for bulk starter propagation or as so-called "Direct Vat Set" (DVS) cultures, intended for direct inoculation into a fermentation vessel or vat for the production of a dairy product, such as an acidified milk drink. Such cultures are in general referred to as "starter cultures" or "starters".<br>
Commonly used starter culture strains of lactic acid bacteria are generally divided into mesophilic organisms having optimum growrth temperatures at about 30°C and thermophilic organisms having optimum growth temperatures in the range of about 40 to about 45°C. Typical organisms belonging to the mesophilic group include Lactococcus lactis, Lactococcus lactis subsp. cremoris, Leuconostoc mesenteroides subsp. cremoris, Pseudoleuconostoc mesenteroides subsp. cremoris, Pediococcus pentosaceus, Lactococcus lactis subsp. lactis biovar diacetylactis, Lactobacillus easel subsp. casei and Lactobacillus paracasei subsp. paracasei. Thermophilic lactic acid bacterial species include as examples Streptococcus thermophilus, Enterococcus faecium, Lactobacillus delbrueckii subsp. lactis, Lactobacillus helveticus, Lactobacillus delbrueckii subsp. bulgaricus and Lactobacillus acidophilus.<br>
<br>
 <br>
Also the strict anaerobic bacteria belonging to the genus Bifidobacterium including Bifidobacterium bifidum and Bifidobacterium longum are commonly used as dairy starter cultures and are generally included in the group of lactic acid bacteria. Additionally, species of Propionibacteria are used as dairy starter cultures, in particular in the manufacture of cheese. Additionally, organisms belonging to the Brevibacterium genus are commonly used as food starter cultures.<br>
Another group of microbial starter cultures are fungal cultures, including yeast cultures and cultures of filamentous fungi, which are particularly used in the manufacture of certain types of cheese and beverage. Examples of fungi include Penicillium roqueforti, Penicillium candidum, Geotrichum candidum, Torula kefir, Saccharomyces kefir and Sacctiaromyces cerevisiae.<br>
In a preferred embodiment of the present invention, the microorganism used for fermentation of the milk substrate is Lactobacillus casei or a mixture of Streptococcus thermophilus and Lactobacillus delbrueckii subsp. bulgaricus.<br>
Fermentation processes to be used in production of acidified milk drinks are well known and the person of skill in the art will know how to select suitable process conditions, such as temperature, oxygen, amount and characteristics of microorganism/s, additives such as e.g. carbohydrates, flavours, minerals, enzymes (e.g. rennet and phospholipase) and process time. Obviously, fermentation conditions are selected so as to support the achievement of the present invention, i.e. to obtain a fermented milk product suitable in the production of an acidified milk drink.<br>
In one embodiment of the method of the present invention, a syrup is added to the milk substrate, either before or after acidification.<br>
In a preferred embodiment, a syrup is added to the milk substrate after acidification, such as after fermentation.<br>
"Syrup" in the context of the present invention is any additional additive ingredient giving flavour and/or sweetness to the final product, i.e. the acidified milk drink. It may be a solution comprising, e.g., sugar, sucrose, glucose, liquid sugar of fructose, aspartame, sugar alcohol, fruit concentrate, orange juice, strawberry juice and/or lemon juice.<br>
 <br>
The mixture of the acidified milk substrate and the syrup may be homogenized using any method known in the art. The homogenization may be performed so as to obtain a liquid homogenous solution which is smooth and stable. Homogenization of the mixture of the acidified milk substrate and the syrup may be performed by any method known in the art, such as by forcing the milk at high pressure through small orifices.<br>
In one embodiment of the invention, water is added to the acidified milk substrate, such as to the fermented milk substrate, and the mixture of acidified milk substrate, such as fermented milk substrate, and water is homogenized.<br>
In another embodiment of the method of the invention, the acidification of the milk substrate takes place because of addition of the syrup, which may be acidic, i.e. by mixing the milk substrate with syrup in the form of, e.g., fruit concentrate, orange juice, strawberry juice and/or lemon juice.<br>
Step b) of the method of the present invention comprises an enzyme treatment. The enzyme treatment may be performed prior to step a), i.e. the milk substrate may be subjected to enzyme treatment before acidification, such as before addition of the chemical acidifier or before inoculation with the microorganism. The enzyme treatment may be performed during step a), i.e. the milk substrate may be subjected to enzyme treatment at the same time as it is being acidified. In one embodiment, the enzyme is added before or after inoculation of the milk substrate with a microorganism, and the enzyme reaction on the milk substrate takes place at the same time as it is being fermented.<br>
Alternatively, the enzyme treatment may be performed after step a), i.e. the milk substrate may be subjected to enzyme treatment after acidification. If the acidified milk substrate is mixed and optionally homogenized with the syrup, the enzyme treatment may be performed before or after this. The enzyme may be added at the same time or after the syrup, but before homogenization, or it may be added after the acidified milk substrate and the syrup have been mixed and homogenized.<br>
In a preferred embodiment, step b) is performed before or during step a). In a more preferred embodiment, the milk substrate is subjected to pasteurization prior to step a), and the enzyme treatment is performed after pasteurization but before or during acidification, e.g., the enzyme may be added before or after inoculation with the microorganism, and the enzyme reaction may take place at essentially the same time as the fermentation.<br>
 <br>
in an even more preferred embodiment, the milk substrate is subjected to pasteurization prior to step a), and the enzyme treatment is performed after pasteurization but before acidification. In an even more preferred embodiment, another pasteurization is performed after the enzyme treatment but before acidification, i.e., pasteurization is performed before and after step b), and step a) is performed after the second pasteurization.<br>
The protein modifying enzyme is added in a suitable amount to achieve the desired degree of protein modification under the chosen reaction conditions. The enzyme may be added at a concentration of betw/een 0.0001 and 1 g/L milk substrate, preferably between 0.01 and 0.1 g/L milk substrate.<br>
The enzymatic treatment in the process of the invention may be conducted by adding the enzyme to the milk substrate and allowing the enzyme reaction to take place at an appropriate holding-time at an appropriate temperature. The enzyme treatment may be carried out at conditions chosen to suit the selected protein modifying enzyme according to principles well known in the art. The treatment may also be conducted by contacting the milk substrate with an enzyme that has been immobilised.<br>
The enzyme treatment may be conducted at any suitable pH, such as e.g., in the range 2-10, such as, at a pH of 4-9 or 5-7. It may be preferred to let the enzyme act at the natural pH of the milk substrate, or, if acidification is obtained because of fermentation, the enzyme may act at the natural pH of the milk substrate during the fermentation process, i.e. the pH will gradually decrease from the natural pH of the unfermented milk substrate to the pH of the fermented milk substrate.<br>
The enzyme treatment may be conducted at any appropriate temperature, e.g. in the range 1-TO'C, such as 2-60°C. If the milk substrate is acidified because of fermentation with a microorganism, the enzyme treatment may be conducted during the fermentation, e.g. at 35-45'C.<br>
Optionally, after the enzyme has been allowed to act on the milk substrate, the enzyme protein may be removed, reduced, and/or inactivated by any method known in the art.<br>
Optionally, other ingredients may be added to the acidified milk drink, such as colour;<br>
stabilizers, e.g. pectin, starch, modified starch, CMC, etc.; or polyunsaturated fatty acids, e.g.<br>
omega-3 fatty acids. Such ingredients may be added at any point during the production<br>
process, i.e. before or after acidification, before or after enzyme treatment, and before or after<br>
the optional addition of syrup.<br>
<br>
 <br>
f<br>
Enzyme having a protein modifying activity<br>
In the method of the present invention, an enzyme is used to reduce the isoelectric point of<br>
milk protein in acidified milk drinks, thus decreasing the syneresis upon storage.<br>
It is to be understood that in the context of the method of the Invention, "milk protein" means any milk protein, i.e. any protein naturally occurring in milk. Thus, in the method of the present invention, the isoelectric point of any milk protein is reduced. In other words, in the method of the present invention, an enzyme is used to reduce the isoelectric point of at least one milk protein, i.e. at least one of the proteins naturally occurring in milk.<br>
Thus, the method for producing an acidified milk drink according to the present invention in one embodiment comprises:<br>
a)	acidifying a milk substrate comprising a milk protein; and<br>
b)	treating with an enzyme having a protein modifying activity which reduces the isoelectric point of the milk protein;<br>
wherein step b) is performed before, during or after step a).<br>
In a preferred embodiment, the milk protein is casein.<br>
Preferentially, the enzyme to be used has an activity which reduces the isoelectric point of the protein in a manner that does not alter the amino acid chain length of the protein, i.e. without hydrolysing the peptide bonds in the protein and without adding new amino acids to the N-terminal or the C-terminal of the amino acid chain.<br>
In one aspect of the present invention, the isoelectric point of the milk protein is reduced by introducing more negative charges in the milk protein at low pH. Such negative charges may be added on the surface of milk protein, e.g., casein.<br>
In a preferred aspect of the method of the invention, the enzyme having a protein modifying activity which reduces the isoelectric point of the milk protein is an enzyme which introduces additional negative charges into the protein in a manner that does not alter the amino acid chain length of the protein. I.e., the additional negative charges are not introduced by, e.g., increasing the amino acid chain length by addition of more (negatively charged) amino acids at the C-terminal or the N-terminal of the amino acid chain. The additional negative charges are preferentially introduced by modifying amino acid side chains in the protein. In a preferred aspect, such modification does not comprise cross-linking of the protein. The enzyme may have an activity which deamidates amide groups in the protein by directly acting upon such<br>
<br>
 <br>
groups without hydrolysing peptide bonds of the protein and without cross-linking of the protein. The enzyme may, e.g., be an asparaginase, a glutaminase, an amidase or a deamidase.<br>
In a more preferred aspect, the enzyme has deamidase activity.<br>
The term "deamidase" refers to an enzyme that catalyzes the hydrolysis of the gamma-amide of the amino acid glutamine or asparagine incorporated in a polypeptide chain (protein) and releases side chain carboxyl groups and ammonia. Such an enzyme may also be referred to as peptidoglutaminase, protein-deamidase, protein deamidating enzyme, protein glutamine glutaminase, protein glutamine amidohydrolase. The group of deamidases comprises but is not limited to the enzymes assigned to subclasses EC 3.5.1.44 (described in Kikuchi, M., Hayashida, H., Nakano, E. and Sakaguchi, K. Peptidoglutaminase. Enzymes for selective deamidation of y-amide of peptide-bound glutamine. Biochemistry AO (1971) 1222-1229).<br>
Enzymes having a protein modifying activity to be used in the method of the present invention may be of animal, of plant or of microbial origin. Preferred enzymes are obtained from microbial sources, in particular from a filamentous fungus or yeast, or from a bacteria.<br>
The enzyme may, e.g., be derived from a strain of Aspergillus, e.g. A. niger, A. awamori, A. foetidus, A.japonicus, A. oryzae; Dictyostelium, e.g. D. discoideum; Mucor, e.g. M.Javanicus, M. mucedo, M. subtilissimus; Neurospora, e.g. N. crassa; Rhizomucor, e.g. R. pusillus; Rhizopus, e.g. R. arrhizus, R. japonicus, R. stolonifer; Sclerotinia, e.g. S. libertiana; Trichophytor), e.g. 7. rubrum; Whetzelinia, e.g. W. sclerotiorum; Bacillus, e.g. S. megaterium, B. subtilis, B. pumilus, B. stearothermophilus; Chryseobacterium; Citrobacter, e.g. C. freur)dii; Enterobacter, e.g. £ aerogenes, E. cloacae Edwardsiella, E. tarda; Erwinia, e.g. E. herbicola; Escherichia, e.g. £. coli; Klebsiella, e.g. K. pneumoniae: Proteus, e.g. P. vulgaris; Providencia, e.g. P. stuartii; Salmonella, e.g. S. typhimurium; Serratia, e.g. S. liquefasciens, S. marcescens; Shigella, e.g. S. flexneri; Streptomyces, e.g. S. violeceoruber; Yersinia, e.g. y. enterocolitica.<br>
In a preferred embodiment, the enzyme is bacterial, e.g. from the class Flavobacteria, such as from the order Flavobacteriales, such as from the family Flavobacteriaceae, such as from a strain of Chryseobacterium. A preferred enzyme is a deamidase having a sequence which is at least 50%, such as at least 60%, at least 70%, at least 80% or at least 90% identical to SEQ ID NO: 2 or SEQ ID NO; 4.<br>
<br>
 <br>
.Novel enzyme having deamidase activity<br>
The present inventors have identified a novel enzyme from a strain of Chryseobacterium having deamidase activity. The DNA sequence of the open reading frame of the gene encoding the enzyme is disclosed as SEQ ID NO: 1. The amino acid sequence of the protein encoded by the gene is shown as SEQ ID NO: 2. SEQ ID NO: 3 shows the N-terminal sequence of the active deamidase as has been experimentally determined. SEQ ID NO: 4 shows the 186 amino acids long amino acid sequence of the mature enzyme. The mature enzyme has a molecular weight of -20 kDa, as has been determined by SDS-PAGE. SEQ ID NO: 5 shows the DNA sequence of the enzyme encoding gene without the region encoding the signal peptide.<br>
Thus, one aspect of the present invention relates to a polypeptide having deamidase activity, selected from the group consisting of:<br>
a)	a polypeptide having an amino acid sequence which is at least 75%, preferably at least 80% or at least 90%, more preferably at least 91%, at least 95% or at least 97%, identical to (i) SEQ ID NO: 2, or (ii) amino acids 133-318 thereof;<br>
b)	a polypeptide which is encoded by a polynucleotide which is at least 80%, preferably at least 90%, more preferably at least 95% or at least 97%, identical to (i) SEQ ID N0:1, or (ii) nucleotides 397-954 thereof;<br>
c)	a polypeptide which is encoded by a polynucleotide whose complement hybridizes under high stringency, preferably very high stringency, conditions, with nucleotides 397-954 of SEQ IDN0:1;<br>
d)	a polypeptide which is encoded by the plasmid contained in £ coli DSM 19445; and<br>
e)	a polypeptide having an amino acid sequence modified by substitution, deletion, and/or insertion of one or several amino acids in (i) SEQ ID NO: 2, or (ii) amino acids 133-318 thereof.<br>
The polypeptide may be isolated.<br>
The term "isolated polypeptide" as used herein refers to a polypeptide which is at least 20% pure, preferably at least 40% pure, more preferably at least 60% pure, even more preferably at least 80% pure, most preferably at least 90% pure, and even most preferably at least 95% pure, as determined by SDS-PAGE.<br>
In a preferred aspect, a polypeptide of the present invention has an amino acid sequence which differs by ten amino acids, preferably by five amino acids, more preferably by four amino acids, even more preferably by three amino acids, most preferably by two amino acids,<br>
<br>
 <br>
and even most preferably by one amino acid from SEQ ID NO: 2 or amino acids 133 to 318 thereof.<br>
In another preferred aspect, a polypeptide of the present invention comprises the amino acid sequence of SEQ ID NO: 2 or an allelic variant thereof; or a fragment thereof that has deamidase activity, the fragment comprising at least 100, such as at least 150 or 200 amino acids. In a more preferred aspect, the polypeptide comprises the amino acid sequence of SEQ ID NO: 2. In another preferred aspect, the polypeptide comprises amino acids 133 to 318 of SEQ ID NO: 2, or an allelic variant thereof, or a fragment thereof that has deamidase activity, the fragment comprising at least 100, such as at least 150 or 200 amino acids. In another preferred aspect, the polypeptide consists of the amino acid sequence of SEQ ID NO: 2 or an allelic variant thereof; or a fragment thereof that has deamidase activity, the fragment comprising at least 100, such as at least 150 or 200 amino acids. In another preferred aspect, the polypeptide consists of amino acids 133 to 318 of SEQ ID NO: 2, or an allelic variant thereof, or a fragment thereof that has deamidase activity, the fragment comprising at least 100, such as at least 150 or 200 amino acids.<br>
An "allelic variant" of a polypeptide is a polypeptide encoded by an allelic variant of a gene. An allelic variant of a gene is any of two or more alternative forms of a gene occupying the same chromosomal locus. Allelic variation arises naturally through mutation, and may result in polymorphism within populations. Gene mutations can be silent (no change in the encoded polypeptide) or may encode polypeptides having altered amino acid sequences, i.e. allelic variants of the polypeptide.<br>
Polynucleotide encoding a deamidase<br>
Another aspect of the present invention relates to an isolated polynucleotide encoding a polypeptide having deamidase activity, selected from the group consisting of:<br>
a)	a polynucleotide encoding a polypeptide having an amino acid sequence which is at least 75%, preferably at least 80% or at least 90%, more preferably at least 91%, at least 95% or at least 97%, identical to (i) SEQ ID NO: 2, or (ii) amino acids 133 to 318 thereof;<br>
b)	a polynucleotide which is at least 80%, preferably at least 90%, more preferably at least 95% or at least 97%, identical to (i) SEQ ID N0:1, or (ii) nucleotides 397-954 thereof; and<br>
c)	a polynucleotide whose complement hybridizes under high stringency conditions,<br>
preferably very high stringency conditions, with (i) SEQ ID NO: 1, (ii) nucleotides 397 to 954<br>
of SEQ ID NO: 1, or (iii) a subsequence of (i) or (ii).<br>
 <br>
.The term "subsequence" is defined herein as a nucleotide sequence having one or more nucleotides deleted from the 5' and/or 3' end of SEQ ID NO: 1 or a homologous sequence thereof. A subsequence of SEQ ID NO; 1 preferably contains at least 100 contiguous nucleotides, more preferably at least 200, 300, 400 or 500 contiguous nucleotides. A preferred subsequence consists of nucleotides 397 to 954 of SEQ ID NO: 1. Moreover, the subsequence may encode a polypeptide fragment which has deamldase activity.<br>
For purposes of the present invention, hybridization indicates that the nucleotide sequence or its complement hybridizes to a labelled nucleic acid probe corresponding to the nucleotide sequence shown in SEQ ID NO: 1, its complementary strand, or a subsequence of any of these, under high to very high stringency conditions. A preferred subsequence consists of nucleotides 397 to 954 of SEQ ID NO: 1 or its complementary strand. Molecules to which the nucleic acid probe hybridizes under these conditions can be detected using X-ray film.<br>
In a preferred aspect, the nucleic acid probe is a polynucleotide sequence which encodes the polypeptide of SEQ ID NO: 2, or a subsequence thereof, such as amino acids 133 to 318, or the complementary strand of any of these. In another preferred aspect, the nucleic acid probe is SEQ ID NO: 1 or nucleotides 397 to 954 thereof, or the complement of any of these. In another preferred aspect, the nucleic acid probe is the mature polypeptide coding region of SEQ ID NO: 1 or its complement.<br>
For long probes of at least 100 nucleotides in length, high to very high stringency conditions are defined as prehybridization and hybridization at 42°C in 5X SSPE, 0.3% SDS, 200 [iglm\ sheared and denatured salmon sperm DNA, and 50% formamide, following standard Southern blotting procedures for 12 to 24 hours optimally (J. Sambrook, E.F. Fritsch, and T. Maniatis, 1989, Molecular Cloning, A Laboratory Manual, 2d edition, Cold Spring Harbor, New York).<br>
For long probes of at least 100 nucleotides in length, the carrier material is finally washed three times each for 15 minutes using 2X SSC, 0.2% SDS preferably at least at 65°C (high stringency), or more preferably at least at 70°C (very high stringency).<br>
In a particular embodiment, the wash is conducted using 0.2X SSC, 0.2% SDS preferably at least at 65°C (high stringency), or more preferably at least at 70°C (very high stringency). In another particular embodiment, the wash is conducted using 0.1X SSC, 0.2% SDS preferably at least at 65°C (high stringency), or more preferably at least at 70'C (very high stringency).<br>
<br>
 <br>
For short probes which are about 15 nucleotides to about 70 nucleotides in length, stringency conditions are defined as prehybridization, hybridization, and washing post-hybridization at about 5°C to about 10°C below the calculated Tm using the calculation according to Bolton and McCarthy (1962, Proceedings of the National Academy of Sciences USA 48:1390) in 0.9 M NaCI, 0.09 M Tris-HCI pH 7.6, 6 mM EDTA, 0.5% NP-40, 1X Denhardt's solution, 1 mM sodium pyrophosphate, 1 mM sodium monobasic phosphate, 0.1 mM ATP, and 0.2 mg of yeast RNA per ml following standard Southern blotting procedures.<br>
For short probes which are about 15 nucleotides to about 70 nucleotides in length, the carrier material is washed once in 6X SCC plus 0.1% SDS for 15 minutes and twice each for 15 minutes using 6X SSC at 5°C to 10°C below the calculated Tm.<br>
Under salt-containing hybridization conditions, the effective Tm is what controls the degree of identity required between the probe and the filter bound DNA for successful hybridization.<br>
The effective Tm may be determined using the formula below to determine the degree of identity required for two DNAs to hybridize under various stringency conditions.<br>
Effective T^ = 81.5 + 16.6{log M[Na1) + 0.41 (%G+C) - 0.72(% formamide) (Seewww.ndsu.nodak.edu/instruct/mcclean/plsc731/dna/dna6.htm)<br>
The G+C content of SEQ ID NO: 1 is 39.3%. For medium stringency, the formamide is 35% and the Na* concentration for 5X SSPE is 0.75 M. Applying this formula to these values, the Effective Tm is 70.3°C.<br>
Another relevant relationship is that a 1% mismatch of two DNAs lowers the Tm by 1.4'C. To determine the degree of identity required for two DNAs to hybridize under medium stringency conditions at 42°C, the following formula is used:<br>
% Homology = 100 - [(Effective Tm - Hybridization Temperature)/1.4] (See www.ndsu.nodak.edu/instruct/mcclean/plsc731/dna/dna6.htm)<br>
Applying this formula to the values, the degree of identity required for two DNAs to hybridize under medium stringency conditions at 42°C is 100 - [(70.3 - 42)/1.4] = 79.8%.<br>
Sequence Identity<br>
 <br>
1<br>
The relatedness between two amino acid sequences is described by the parameter "identity" or "homology". For purposes of the present invention, the alignment of two amino acid sequences is determined by using the Needle program from the EMBOSS package (Rice, P., Longden, I. and Bleasby, A. (2000) EMBOSS: The European Molecular Biology Open Software Suite. Trends in Genetics 16, (6) pp276—277; http://emboss.org) version 2.8.0. The Needle program implements the global alignment algorithm described in Needleman, S. B. and Wunsch, C. D. (1970) J. Mol. Biol. 48, 443-453. The substitution matrix used is BLOSUM62, gap opening penalty is 10, and gap extension penalty is 0.5.<br>
The degree of identity or homology between an amino acid sequence of the present invention ("invention sequence") and a different amino acid sequence ("foreign sequence") is calculated as the number of exact matches in an alignment of the two sequences, divided by the length of the "invention sequence" or the length of the "foreign sequence", whichever is the shortest. The result is expressed in percent identity.<br>
An exact match occurs when the "invention sequence" and the "foreign sequence" have identical amino acid residues in the same positions of the overlap. The length of a sequence is the number of amino acid residues in the sequence.<br>
In a particular embodiment, the percentage of identity of an amino acid sequence of a polypeptide with, or to, SEQ ID NO: 2 is determined by i) aligning the two amino acid sequences using the Needle program, with the BLOSUM62 substitution matrix, a gap opening penalty of 10, and a gap extension penalty of 0.5; ii) counting the number of exact matches in the alignment; iii) dividing the number of exact matches by the length of the shortest of the two amino acid sequences, and iv) converting the result of the division of iii) into percentage. The percentage of identity to, or with, other sequences of the invention are calculated in an analogous way.<br>
For purposes of the present invention, the degree of identity between two nucleotide sequences is determined by the Wilbur-Lipman method (Wilbur and Lipman, 1983, Proceedings of tlie National Academy of Science USA 80: 726-730) using the LASERGENE™ MEGALIGN™ software (DNASTAR, Inc., Madison, Wl) with an identity table and the following multiple alignment parameters: Gap penalty of 10 and gap length penalty of 10. Pairwise alignment parameters are Ktuple=3, gap penalty=3, and wlndows=20.<br>
Expression vectors and host cells<br>
 <br>
Yet another aspect of the invention relates to nucleic acid constructs, recombinant expression vectors, and recombinant host cells comprising the polynucleotides, and to methods for producing such polypeptides having deamidase activity comprising (a) cultivating a recombinant host cell comprising a nucleic acid construct comprising a polynucleotide encoding the polypeptide under conditions conducive for production of the polypeptide; and (b) recovering the polypeptide.,<br>
The term "nucleic acid construct" as used herein refers to a nucleic acid molecule, either single- or double-stranded, which is isolated from a naturally occurring gene or which is modified to contain segments of nucleic acids in a manner that would not otherwise exist in nature. The term nucleic acid construct is synonymous with the term "expression cassette" when the nucleic acid construct contains the control sequences required for expression of a coding sequence of the present invention.<br>
The term "control sequences" is defined herein to include all components, which are necessary or advantageous for the expression of a polynucleotide encoding a polypeptide of the present invention. Each control sequence may be native or foreign to the nucleotide sequence encoding the polypeptide. Such control sequences include, but are not limited to, a leader, polyadenylation sequence, propeptide sequence, promoter, signal peptide sequence, and transcription terminator. At a minimum, the control sequences include a promoter, and transcriptional and translational stop signals. The control sequences may be provided with linkers for the purpose of introducing specific restriction sites facilitating ligation of the control sequences with the coding region of the nucleotide sequence encoding a polypeptide.<br>
The term "coding sequence" means a nucleotide sequence, which directly specifies the amino acid sequence of its protein product. The boundaries of the coding sequence are generally determined by an open reading frame, which usually begins with the ATG start codon or alternative start codons such as GTG and TTG. The coding sequence may be a DNA, a cDNA, or a recombinant nucleotide sequence.<br>
The term "expression" includes any step involved in the production of the polypeptide including, but not limited to, transcription, post-transcriptional modification, translation, post-translational modification, and secretion.<br>
The term "expression vector" is defined herein as a linear or circular DNA molecule that comprises a polynucleotide encoding a polypeptide of the invention, and which is operably linked to additional nucleotides that provide for its expression.<br>
<br>
 <br>
 The term "operably linked" denotes herein a configuration in which a control sequence is placed at an appropriate position relative to the coding sequence of the polynucleotide sequence such that the control sequence directs the expression of the coding sequence of a polypeptide.<br>
The term "host cell", as used herein, includes any cell type which is susceptible to transformation, transfection, transduction, and the like with a nucleic acid construct comprising a polynucleotide of the present invention.<br>
Sources of Polypeptides Having Deamidase Activity<br>
The nucleotide sequence of SEQ ID NO: 1 or a subsequence thereof, as well as the amino acid sequence of SEQ ID NO: 2 or a fragment thereof, may be used to design a nucleic acid probe to identify and clone DNA encoding polypeptides having deamidase activity from strains of different genera or species according to methods well known in the art. In particular, such probes can be used for hybridization with the genomic or cDNA of the genus or species of interest, following standard Southern blotting procedures, in order to identify and isolate the corresponding gene therein. Such probes can be considerably shorter than the entire sequence, but should be at least 14, preferably at least 25, more preferably at least 35, and most preferably at least 70 or at least 100 nucleotides in length. For example, the nucleic acid probe may be at least 200 nucleotides, preferably at least 300 nucleotides, more preferably at least 400 nucleotides, or most preferably at least 500 nucleotides in length. Even longer probes may be used, e.g., nucleic acid probes which are at least 600 nucleotides, preferably at least 700 nucleotides, more preferably at least 800 nucleotides, or most preferably at least 900 nucleotides in length. Both DNA and RNA probes can be used. The probes are typically labeled for detecting the corresponding gene (for example, with "P, 'H, ^^S, biotin, or avidin). Such probes are encompassed by the present invention.<br>
A genomic DNA or cDNA library prepared from other organisms of interest may be screened for DNA which hybridizes with the probes described above and which encodes a polypeptide having deamidase activity. Genomic or other DNA from such other organisms may be separated by agarose or polyacrylamide gel electrophoresis, or other separation techniques. DNA from the libraries or the separated DNA may be transferred to and immobilized on nitrocellulose or other suitable carrier material. In order to identify a clone or DNA which is homologous with SEQ ID NO: 1 or a subsequence thereof, the carrier material is used in a Southern blot.<br>
<br>
 <br>
A polypeptide of the present invention may be obtained from microorganisms of any genus. For purposes of the present invention, the term "obtained from" as used in connection with a given source shall mean that the polypeptide encoded by a nucleotide sequence is produced by the source or by a strain in which the nucleotide sequence from the source has been inserted. In a preferred aspect, the polypeptide obtained from a given source is secreted extracellularly.<br>
A polypeptide of the present invention may be a bacterial polypeptide, and preferably it may be obtained from or obtainable from the class Flavobacteria, more preferably from the order Flavobacteriales and even more preferably from the family Flavobacteriaceae. The polypeptide may be obtained from or obtainable from any of the genera Bergeyella, Capnocytophaga, Cellulophaga, Chryseobacterium, Coenonia, Dokdonia, Empedobacter, Flavobacterium, Gelidibacter, Ornithobacterium, Polaribacter, Psychroflexus, Psychroserpens, Riemerella, Saligentibacter, or Weeksella, preferably from Chryseobacterium.<br>
For the aforementioned species, the invention encompasses both the perfect and imperfect states, and other taxonomic equivalents, e.g., anamorphs, regardless of the species name by which they are known. Those skilled in the art will readily recognize the identity of appropriate equivalents.<br>
Strains of the genera mentioned above are readily accessible to the public in a number of culture collections, such as the American Type Culture Collection (ATCC), Deutsche Sammlung von Mikroorganismen und Zellkulturen GmbH (DSMZ), Centraalbureau Voor Schimmelcultures (CBS), and Agricultural Research Service Patent Culture Collection, Northern Regional Research Center (NRRL).<br>
Furthermore, polypeptides of the present invention may be identified and obtained from other sources including microorganisms isolated from nature (e.g., soil, composts, water, etc.) using the above-mentioned probes. Techniques for isolating microorganisms from natural habitats are well known in the art. The polynucleotide may then be obtained by similarly screening a genomic or cDNA library of another microorganism. Once a polynucleotide sequence encoding a polypeptide has been detected with the probe(s), the polynucleotide can be isolated or cloned by utilizing techniques which are well known to those of ordinary skill in the art (see, e.g., Sambrook et al., 1989, supra).<br>
Polypeptides of the present invention also include fused polypeptides or cleavable fusion polypeptides in which another polypeptide is fused at the N-terminus or the C-terminus of the<br>
<br>
 <br>
  polypeptide or fragment thereof. A fused polypeptide is produced by fusing a nucleotide sequence (or a portion tliereof) encoding another polypeptide to a nucleotide sequence (or a portion thereof) of the present invention. Techniques for producing fusion polypeptides are known in the art, and include ligating the coding sequences encoding the polypeptides so that they are in frame and that expression of the fused polypeptide is under control of the same promoter(s) and terminator.<br>
EXAMPLES<br>
Example 1<br>
SKMP solution<br>
20 ml water + 4.5 g skim milk powder (instant dispersibility from Kerry) was incubated at 50°C<br>
for 10 min before use, so a homogeneous solution was obtained.<br>
Sugar solution<br>
3.3 g sucrose<br>
10.5 g glucose<br>
These sugars were added to 46 ml 20 mM lactic acid buffer, pH 4.0 and incubated at 90°C for<br>
5 mIn with stirring and then cooled down to 5°C.<br>
Enzyme<br>
Chromatographically purified Chryseobacterium deamidase (Example 3), 0.9 mg/ml, was<br>
diluted to give the final concentrations indicated in the Tables.<br>
Procedure A (Enzyme added before pasteurization)<br>
250 ul SKMP solution was transferred to eppendorf tubes. 20 ul Enzyme or water (control)<br>
was added and incubation was performed for 120 min at SO'C.<br>
The solution was incubated at SS'C for 30 min with 1000 rpm and hereafter incubated at<br>
43'C for 10 min with 1000 rpm.<br>
30 ul 4 U/l YF-3331  (mixed strain culture containing Streptococcus thermophilus and<br>
Lactobacillus delbrueckii subsp. bulgaricus from Chr. Hansen A/S, Denmark) was added and<br>
incubation was performed for 16 hours at 43°C.<br>
Hereafter the samples were incubated at 0-5°C ice/water bath for 20 min.<br>
600 ul sugar solution (ice bath) was added and sucked up and down with a pipette five times.<br>
500 ul glass beads (5C) were added to each tube.<br>
The samples were placed in an eppendorf thermomixer at 5°C with 1400 rpm for 10 min.<br>
The samples were placed at 5°C for 4 days and syneresis was measured.<br>
<br>
 <br>
^ Procedure B (Enzyme added after pasteurization) 250 ul SKMP solution was transferred to eppendorf tubes and incubated for 120 min at 50°C. The solution was incubated at 85°C for 30 min with 1000 rpm and hereafter incubated at AS'C for 10 min with 1000 rpm.<br>
30 ul 4 U/l YF-3331  (mixed strain culture containing Streptococcus thermophilus and Lactobacillus delbrueckii subsp. bulgaricus from Chr. Hansen A/S, Denmark) + 20 ul Enzyme or water (control) was added and incubation was performed for 16 hours at 43°C. Hereafter the samples were incubated at 0-5°C ice/water bath for 20 min. 600 ul sugar solution (ice bath) was added and sucked up and down with a pipette five times. 500 ul glass beads (5°C) were added to each tube.<br>
The samples were placed in an eppendorf thermomixer at 5°C with 1400 rpm for 10 min. The samples were placed at 5°C for 4 days and syneresis was measured.<br>
Procedure C (Enzyme added after homooenization)<br>
250 ul SKMP solution was transferred to eppendorf tubes and incubated for 120 min at SO'C.<br>
The solution was incubated at 85°C for 30 min with 1000 rpm and hereafter incubated at<br>
43-0 for 10 min with 1000 rpm.<br>
30 ul 4 U/l YF-3331  (mixed strain culture containing Streptococcus thermophilus and<br>
Lactobacillus delbrueckii subsp. bulgaricus from Chr. Hansen A/S, Denmark) was added and<br>
incubation was performed for 16 hours at 43°C.<br>
Hereafter the samples were incubated at 0-5'C ice/water bath for 20 min.<br>
600 ul sugar solution (ice bath) was added + 20 ul Enzyme or water (control) and sucked up<br>
and down with a pipette five times.<br>
500 ul glass beads (5'C) were added to each tube.<br>
The samples were placed in an eppendorf thermomixer at S'C with 1400 rpm for 10 min.<br>
The samples were placed at 5°C for 4 days and syneresis was measured.<br>
The data (double determinations) in Table 1 shows a decreased syneresis when deamidase is added compared to the water control. The most predominant effect can be seen when the enzyme is added after pasteurization.<br>
Deamidase	Average	deviation<br>
Table 1	syneresis	(+/-)<br>
ug/ml	mm	mm<br>
Procedure A<br>
Added before pasteurization     57	2.8	0.08<br>
<br>
 <br>
,    Water control	0	3.3	0.04<br>
Procedure B<br>
Added after pasteurization        57	1.1	0.03<br>
Water control	0	3.5	0.25<br>
Procedure C<br>
Added after homogenization     57	3.9	0.12<br>
Water control	0	3.8	0.13<br>
Example 2<br>
A dose response experiment was made according to Procedure B as described in Example 1. Table 2 shows a decrease in syneresis with increasing deamidase concentration.<br>
Deamidase Average	deviation<br>
Table 2	syneresis (+/-)<br>
ug/ml	Mm	mm<br>
Added after pasteurization   0	2.5	0.11<br>
Added after pasteurization   13	2.2	0.10<br>
Added after pasteurization   29	1.6	0.00<br>
Added after pasteurization   57	1.2	0.03<br>
Example 3<br>
Cloning and production of deamidase<br>
Cloning and production<br>
Chryseobacterium sp. was isolated from a soil sample from Antarctica from 19/01/1989.<br>
The deamidase gene was PCR amplified based on primers designed from the sequences of known deamidases. (Genbanl
<br>
 <br>
The signal peptide from the SavinaseTM gene (the subtilisin protease of B .clausii) was fused by PCR in frame to the gene encoding the deamidase (SEQ ID NO: 5). The fused gene was integrated by homologous recombination into the genome of the Bacillus subtilis host. The gene construct was expressed under the control of a triple promoter system (as described in WO 99/43835), consisting of the promoters from Bacillus licheniformis alpha-amylase gene (amyL), Bacillus amyloliquefaciens alpha-amylase gene (amyQ), and Bacillus thuringiensis crylllA promoter including stabilizing sequence. The gene coding for Chloramphenicol acetyl-transferase was used as marker. (Described e.g in Diderichsen et al., A useful cloning vector for Bacillus subtilis. Plasmid, 30, p. 312, 1993). Chloramphenicol resistant transformants were analyzed by DNA sequencing to verify the correct DNA sequence of the construct. One such clone was selected.<br>
Fermentations of the deamidase expression clone was performed on a rotary shaking table In 500 ml baffled Erienmeyer flasks each containing 100 ml LB medium (or another medium suitable for fermentation of Bacillus subtilis) supplemented with 34 mg/l chloramphenicol. The clone was fermented for 5 days at 30°C.<br>
Purification of deamidase by preparative lEF<br>
50 ml of supernatant was added in 2.5 ml aliquots to PD10TM (Pharmacia) gelflltration columns to remove salt. 70 ml was collected from the PD10 TM (Pharmacia) columns. Sample was prepared for RotoforTM (Biorad) separation. 59 ml gelfiltrated supernant was mixed with 3 ml ampholyte Biolyte 3/10 (Biorad art 163 1112). The preparative isoelectric focusing (lEF) was conducted as recommended by the manufacturer. Sample was loaded in the RotoforTM and electrophoresed for 4 hours with constant effect of 15W. Twenty fractions obtained by the preparative lEF were evaluated on SDS PAGE, and based on the banding patterns the fractions were pooled. The activity was evaluated on casein by a Qualitative assay as described below.<br>
Qualitative deamidase assay<br>
Casein was deamidated by adding 10 ul sample to 1 ml 1% casein in Britten Robinson buffer pH 7, followed by incubation at 37'C for 1 h with agitation. Reaction was stopped by taking the sample to ice.<br>
The deamidation of casein was evaluated by analytical lEF. The enzyme treated samples (and an untreated casein control) were mixed 1:1 with a solution of 9 M Urea, 2% Triton X-100 to denature the casein micelles. Samples were loaded on Pharmacia lEF 3/10. The lEF gel was subjected to electrophoresis as recommended by the manufacturer, followed by<br>
<br>
 <br>
  coomassie staining to visualise the protein (Figure 1). The following samples were loaded on the lEF gel (from left) Lane 1: lEF marker; Lane 2: supernatant of the deamidase clone; Lane 3 is the supernatant prepared for loading to preparative lEF (this sample is about 4 times diluted compared to lane 2); Lanes 4-11 are pools of fractions from preparative lEF (pool 4 (lane 7) contains the deamidase enzyme); Lane 12 is the casein standard.<br>
DEPOSIT OF BIOLOGICAL MATERIAL<br>
The following biological material has been deposited under the terms of the Budapest Treaty with the Deutsche Sammlung von Mikroorganismen und Zeilkuituren GmbH (DSM), Mascheroder Weg 1 B, D-38124 Braunschweig, Germany, and given the following accession number:<br>
Deposit	Accession Number	Date of Deposit<br>
E. coli	DSM 19445	June 21, 2007<br>
The strain has been deposited under conditions that assure that access to the culture will be available during the pendency of this patent application to one determined by foreign patent laws to be entitled thereto. The deposit represents a substantially pure culture of the deposited strain. The deposit is available as required by foreign patent laws in countries wherein counterparts of the subject application, or its progeny are filed. However, it should be understood that the availability of a deposit does not constitute a license to practice the subject invention in derogation of patent rights granted by governmental action.<br>
<br>
<br>
. WE CLAIM:<br>
1.	A method for producing an acidified miik drink, said method comprising:<br>
a)	acidifying a milk substrate comprising milk protein; and<br>
b)	treating with an enzyme having a protein modifying activity which reduces the isoelectric point of the milk protein;<br>
wherein step b) is performed before, during or after step a).<br>
2.	The method of claim 1, wherein the acidification comprises fermentation with a lactic acid bacterium.<br>
3.	The method of claim 1, wherein step a) comprises chemical acidification.<br>
4.	The method of any of the preceding claims, wherein step b) is performed before or during step a).<br>
5.	The method of any of the preceding claims, wherein the milk substrate is subjected to pasteurization before step a) and step b) is performed after pasteurization.<br>
6.	The method of any of the preceding claims, wherein the acidified milk substrate is mixed with a syrup and the mixture is subjected to homogenization.<br>
7.	The method of any of the preceding claims, wherein the acidified milk drink is in liquid form.<br>
8.	The method of any of the preceding claims, wherein the acidified milk drink has a viscosity of less than 40 Pa at a shear rate of 300 pr. s.<br>
9.	The method of any of the preceding claims, wherein the milk substrate has a protein content of more than 5%.<br>
<br>
10.	The method of any of the preceding claims, wherein the acidified milk drink has a milk protein content of less than 3%.<br>
11.	The method of any of the preceding claims, wherein the milk protein is casein.<br>
12.	The method of any of the preceding claims, wherein the enzyme has deamidase activity.<br>
 <br>
^  13. The method of any of the preceding claims, wherein the amino acid sequence of the enzyme has at least 50% identity to SEQ ID: NO 2 or SEQ ID NO: 4.<br>
14.	An acidified milk drink obtainable by any of the preceding claims.<br>
15.	An isolated polypeptide having deamidase activity, selected from the group consisting of:<br>
<br>
a)	a polypeptide having an amino acid sequence which Is at least 91% identical to (i) SEQ ID NO: 2, or (ii) amino acids 133-318 thereof;<br>
b)	a polypeptide which is encoded by a polynucleotide which is at least 90% identical to (i) SEQ ID N0:1, or (ii) nucleotides 397-954 thereof;<br>
c)	a polypeptide which is encoded by a polynucleotide whose complement hybridizes under high stringency conditions with nucleotides 397-954 of SEQ ID NO: 1;<br>
d)	a polypeptide which is encoded by the piasmid contained in E. coli DSM 19445; and<br>
e)	a polypeptide having an amino acid sequence modified by substitution, deletion, and/or insertion of one or several amino acids in (i) SEQ ID NO: 2, or (ii) amino acids 133-318 thereof.<br>
 <br>
</td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/viewdoc.aspx?id=28l7NdMTGFluFy5XkSW19g==&amp;amp;loc=egcICQiyoj82NGgGrC5ChA==" target="_blank" style="word-wrap:break-word;">http://ipindiaonline.gov.in/patentsearch/GrantedSearch/viewdoc.aspx?id=28l7NdMTGFluFy5XkSW19g==&amp;amp;loc=egcICQiyoj82NGgGrC5ChA==</a></p>
		<br>
		<div class="pull-left">
			<a href="272360-three-dimensional-two-way-acoustic-wave-equation-pre-stack-imaging-systems-and-methods.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="272362-method-and-apparatus-for-transmitting-and-receiving-control-channels-by-restricting-a-set-of-the-control-channels-in-a-wireless-communication-system.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>272361</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>6623/CHENP/2009</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>14/2016</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>01-Apr-2016</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>30-Mar-2016</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>10-Nov-2009</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>NOVOZYMES A/S</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td>KROGSHOEJVEJ 36, DK-2880 BAGSVAERD</td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>TAMS, JEPPE, WEGENER</td>
											<td>FENNEVANGEN 11, DK-2820 GENTOFTE</td>
										</tr>
										<tr>
											<td>2</td>
											<td>NIELSEN, PREBEN</td>
											<td>ALLIKEVEJ 4, DK-2970 HOERSHOLM</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>C12N9/08</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>PCT/EP08/55772</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td>2008-05-09</td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td>07108070.9</td>
									<td>2007-05-11</td>
								    <td>EUROPEAN UNION</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/272361-method-for-producing-an-acidified-milk-drink by HTTrack Website Copier/3.x [XR&CO'2014], Thu, 04 Apr 2024 23:03:52 GMT -->
</html>
