<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/272373-molecular-biosensors-for-detecting-macromolecules-and-other-analytes by HTTrack Website Copier/3.x [XR&CO'2014], Thu, 04 Apr 2024 23:07:39 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 272373:MOLECULAR BIOSENSORS FOR DETECTING MACROMOLECULES AND OTHER ANALYTES</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">MOLECULAR BIOSENSORS FOR DETECTING MACROMOLECULES AND OTHER ANALYTES</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>Abstract MOLECULAR BIOSENSORS FOR DETECTING MACROMOLECULES AND OTHER ANALYTES The invention generally provides molecular biosensors. The molecular biosensors are useful in several methods including in the identification and quantification of target molecules.</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td>MOLECULAR BIOSENSORS FOR DETECTING MACROMOLECULES<br>
AND OTHER ANALYTES<br>
GOVERNMENTAL RIGHTS<br>
[0001]	The present invention was supported by a Phased Innovation<br>
Grant (R21/R33 CA 94356) and a STTR Grant {1 R41 GM079891-01) from the National Institutes of Health. The United States Government has certain rights in this invention.<br>
FIELD OF THE INVENTION<br>
[0002]	The invention relates to molecular biosensors and methods for<br>
detecting several types of target molecules, such as polypeptides, analytes,<br>
macromolecular complexes, or combinations thereof.<br>
BACKGROUND OF THE INVENTION<br>
[0003]	The detection, identification and quantification of specific<br>
molecules in our environment, food supply, water supply and biological samples (blood, cerebral spinal fluid, urine, et cetera) can be very complex, expensive and time consuming. Methods utilized for detection of these molecules include gas chromatography, mass spectroscopy, DNA sequencing, immunoassays, cell-based assays, biomolecular blots and gels, and myriad other multi-step chemical and physical assays.<br>
[0004]	There continuea to be a high demand for convenient<br>
methodologies for detecting and measuring the levels of specific proteins in biological and environmental samples. Detecting and measuring levels of proteins Is one of the most fundamental and most often performed methodologies in biomedical research. While antibody-based protein detection methodologies are enormously useful in research and medical diagnostics, they are not well adapted to rapid, high-throughput parallel protein detection.<br>
[0005]	Previously, the inventor had developed a fluorescent sensor<br>
methodology for detecting a specific subclass of proteins, i.e., sequence-specific DNA binding proteins (Heyduk, T.; Heyduk, E. Nature Biotechnology 2002. 20,171-176; Heyduk, E,; Knoll, E.; Heyduk, T. Analyt. Biochem. 2003, 316, 1-10; U.S. Patent<br>
 <br>
No, 6,544,746 and copending patent applications number 10/062,064, PCT/US02/24822 and PCT/US03/02157, which are incorporated herein by reference). This methodology is based on splitting the DNA binding site of proteins into two DNA "half-sites." Each of the resulting "half-sites" contains a short complementary single-stranded region of the length designed to introduce some propensity for the two DNA "half-sites" to associate recreating the duplex containing the fully functional protein binding site. This propensity is designed to be low such that in the absence of the protein only a small fraction of DNA half-sites will associate. When the protein is present in the reaction mixture, It will bind only to the duplex containing fully functional binding site. This selective binding will drive association of DNA half-sites and this protein-dependent association can be used to generate a spectroscopic signal reporting the presence of the target protein. The term "molecular beacons" is used in the art to describe the above assay to emphasize that selective recognition and generation of the signal reporting the recognition occur in this assay simultaneously. Molecular beacons for DNA binding proteins have been developed for several proteins illustrating their general applicability (Heyduk, T.; Heyduk, E. Nature Biotechnology 2002, 20, 171-176, which is herein incorporated by reference). Their physical mechanism of action has been established and they have also been used as a platform for the assay detecting the presence of ligands binding to DNA binding proteins (Heyduk, E.; Knoll, E.; Heyduk, T. Analyt. Biochem. 2003, 316, 1-10; Knoll. E.; Heyduk, T. Analyt. Chem. 2004, 76, 1156-1164; Heyduk, E.; Per, Y.; Heyduk, T. Combinatorial Chemistry and High-throughput Screening 2003,6, 183-194, which are incorporated herein by reference.) While already very useful, this assay is limited to proteins that exhibit natural DNA binding activity.<br>
Aptamers in "Molecular Beacons"<br>
[0006]	Development of convenient, specific, sensitive high-throughput<br>
assays for detecting proteins remains an extremely important goal. Such assays find applications in research, drug discovery and medical diagnosis. Antibodies recognizing target proteins are the centerpieces of the great majority of protein detection assays so far. Development of in vitro methods for selecting aptamers recognizing target proteins from a population of random sequence nucleic acids<br>
 <br>
provided the first real alternative to antibodies. One of the potentially important advantages of aptamers is that they are made of easy to propagate and synthesize oligonucleotides, Additionally, standard nucleic acid chemistry procedures can be used to engineer aptamers to contain reporter groups such as, for example, fluorescent probes. Thus, it is no wonder that there is significant interest in utilizing aptamers in various formats of protein detection assays. One of the most promising routes is the development of aptamer-based sensors combining recognition of the target protein with generation of an optical signal reporting the presence of the protein.<br>
[0007]	There are several published reports that document ingenious<br>
designs of aptamer-based "molecular beacons" which produced a fluorescent signal upon binding to a specific target protein. All of these designs rely on target protein-induced conformational transition in the aptamer to generate fluorescence signal change. Yamomoto and Kumar (Genes to Cells 2000, 5, 389-396) described a molecular beacon aptamer that produced an increase of fluorescence upon recognition of HIV Tat protein. Fluorescence signal was generated due to a change in proximity of a fluorophore-quencher pair resulting from Tat protein-induced transition between hairpin and duplex forms of the aptamer. Hamaguchi et al. (Analyt. Biochem. 2001, 294,126-131) described a molecular beacon aptamer that produced an increase of fluorescence upon recognition of thrombin. In the absence of the target protein, the beacon was designed to form a stem-loop structure bringing the fluorophore and the quencher into close proximity. In Iho presence of the protein, the beacon was forced into a ligand-binding conformation resulting in increased separation between the fluorophore and the quencher and therefore, increased fluorescence signal. Li et al. (Biochem. Biophys. Res. Commun. 2002, 292, 31-40) described a molecular beacon aptamer that underwent a transition from loose random coil to a compact unimolecular quadruplex in the presence of a target protein. This protein-induced change in aptamer conformation resulted in a change of proximity between the fluorescence probes attached to the ends of the aptamer generating a fluorescence signal change. An analogous approach was used by Fang et al. (ChemBioChem. 2003, 4, 829-834) to design a molecular beacon aptamer recognizing PDGF. These examples illustrate the great potential of<br>
 <br>
ptamers for designing sensors, which could transduce the presence of the protein itoan optical signal.<br>
UMMARY OF THE INVENTION<br>
[0008]	One aspect of the invention encompasses a three-component<br>
lolecular biosensor. The molecular biosensor generally comprises two epitope inding agents and an oligonucleotide construct, as further detailed herein.<br>
[0009]	Another aspect of the invention provides a composition. The<br>
omposition typically comprises a surface immobilized with an oligonucleotide onstruct, and two epitope binding agents, as described more fully herein.<br>
[0099]	Yet another aspect of the invention provides a method for detecting<br>
target in a sample. The method comprises contacting a surface immobilized with an ligonulceotide construct, two epitope binding agents, and a sample; and detecting 'hether the epitope binding agents bind to the oligonucleotide construct. Binding of the pitope binding agents to the oligonucleotide constnjct indicates the presence of target 1 the sample.<br>
[00100]	A further aspect of the invention encompasses a molecular<br>
iosensor. The molecular biosensor comprises two epitope binding agents, which igether have formula (VI)<br>
p51   p52_p53  p54.<br>
(VI) wherein:<br>
R""^ is an epitope-binding agent that binds to a first epitope on a target molecule;<br>
R"^ is a flexible linker attaching R"^ to R"^; R"*^ and R^^ are a pair of complementary nucleotide sequences having a free energy for association from about 5.5 kcal/mole to about 8.0 kcal/mole at a temperature from about 2r C to about 40° C and at a salt concentration from about 1 mM lo about 100 mM;<br>
R^" and R'^' together comprise a detention moans such that when R"*^ and R^^ associate a detectable signal is produced; R°^ is an epitope binding agent that binds to R'*^; and<br>
 <br>
R^^ is a flexible linker attaching R^^ to R'^^<br>
[0012]	Another aspect of the invention provides a molecular biosensor.<br>
The molecular biosensor comprises two epitope binding agents, which together have formula (Vll)<br>
p51   D52_rD53  p54.<br>
(VII) wherein:<br>
R'^ is a peptide, a small molecule, or protein epitope-binding agent that binds to a first epitope on a target molecule; R**^ is a flexible linker attaching R"' to R"^; R^^ and H^^ are a pair of complementary nucleotide sequences having a free energy for association from about 5.5 kcal/mole to about 8.0 kcal/mole at a temperature from about 21° C to about 40° C and at a salt concentration from about 1 mM to about 100 mlVI;<br>
R^ and R^ together comprise a detection means such that when R"*^ and R^^ associate a detectable signal is produced;<br>
R^^ is an antibody or antibody fragment epitope binding agent that binds to R"'; and<br>
R^^ is a flexible linker attaching R^^ to R'^^<br>
[013]	Yet another aspect of the invention provides a method for<br>
determining the presence of a target molecule in a sample. The method comprises measuring the signal of a molecular biosensor having either formula (VI) or (VI I) without the target molecule being present. The molecular biosensor is then combined with the sample and the signal of the molecular biosensor is measured. A decrease in signal indicates the presence of a target molecule.<br>
[014]	Other features and aspects of the invention are described in<br>
more detail herein.<br>
DESCRIPTION OF THE FIGURES<br>
[015]	Fig. 1. Overall design of molecular beacons for detecting<br>
proteins. (A) Variant of the design for targets lacking natural DNA binding activity. The beacon in this case will be composed of two aptamers developed to recognize<br>
 <br>
two different epitopes of the protein. (B) Variant of the design for a target exhibiting natural DNA binding activity. The beacon in this case will be composed of a short double-stranded DNA fragment containing the DNA sequence corresponding to the DNA-binding site and an aptamer developed to recognize a different epitope of the protein.<br>
[016]	Fig. 2. Methods for preparing aptamers to be used in molecular<br>
biosensors. (A) Selection of an aptamer in the presence of a known aptamer construct. The in vitro evolution process is initiated with a nucleic acid construct, an aptamer construct (composed of a known aptamer (thick black line), a linker (thin black line), and a short oligonucleotide sequence (light gray l:)ar)), nnd the target (gray). The light gray bars depict complementary short oligonucleotide sequerices. (B) Simultaneous selection of two aptamers that bind distinct epitopes of the same target (gray). The in vitro evolution process is initiated with two types of nucleic acid constructs (the prlmer1-2 construct and the primer3-4 construct) and the target. The light gray bars depict short complementary sequences at the end of the two types of nucleic acid constructs. (C) Alternative design for simultaneous selection of two aptamers that bind distinct epitopes of the same target (gray). An additional pair of short oligonucleotides (light gray bars) connected by a flexible linker is present during the selection process. These oligonucleotides will be complementary to short oligonucleotide sequences at the end of the nucleic acid constructs (in primer 1 and primer 4). Their presence during selection will provide a bias towards selecting pairs of aptamers capable of simultaneously binding to the target. Before cloning of the selected nucleic acid constructs the pairs of selected sequences will be ligated to preserve the information regarding the preferred pairs between various selected constructs. (D) Selection of an aptamer in the presence of a known antibody construct. The in vitro evolution process is initiated with a nucleic acid construct, an antibody construct (composed of a known antibody (black), a linker, and a Short oligonucleotide sequence (light gray)), and the target (gray). The light gray colored bars depict complementary short oligonucleotide sequences. (E) Selection of an aptamer in the presence of a known double-stranded DNA construct. The in vitro evolution process is initiated with a nucleic acid construct, an aptamer construct (composed of a known double-stranded DNA sequence (black), a linker, and a short<br>
 <br>
oligonucleotide sequence (light gray)), and the target (gray). The light gray bars depict complementary short oligonucleotide sequences.<br>
[017]	Fig. 3. Comparison of the design of molecular beacons for DNA<br>
binding proteins (A) and molecular beacons for detecting proteins based on aptamers directed to two different epitopes of the protein (B).<br>
[018]	Fig. 4. Aptamer constructs containing aptamers binding<br>
thrombin at fibrinogen exosite (60-18 [29]) (gray arrow) and at heparin exosite {G15D) (black arrow).<br>
[019]	Fig. 5. Binding of fluorescein-labeled aptamers to thrombin. (A)<br>
Binding of 60-18 [29] aptamer (THR1) (50 nM) detected by fluorescence polarization; (B) Binding of G15D aptamer (THR2) (50 nM) detected by change in fluorescence intensity; (C) Quantitative equilibrium titration of fluorescein-labeled G15D aptamer (THR2) (20 nM) with thrombin, Solid line represents nonlinear fit of experimental data to an equation describing formation of 1:1 complex between the aptamer and ttirombin; (D) Quantitative equilibrium titration of fluorescein-labeled G15D aptamer {THR2) (20 nM) with thrombin In the presence of ten fold excess of unlabeled 60-18 [29] aptamer (THR3). Solid line represents nonlinear fit of experimental data to an equation describing formation of 1:1 complex between the aptamer and thrombin.<br>
[020]	Fig. 6. Illustration of the competition between thrombin aptamer<br>
constructs and fluorescein-labeled G15D aptamer (THR2) for binding to thrombin. Fluoresconce spectra of 50 nM fluorescein-labeled G15D (THR2) with and without thrombin in the absence of competilor (A), in the presonnR of 150 nM THR3 (B), in the presence of 150 nM THR4 (C), and in the presence of 150 nM THR7 (D).<br>
[021]	Fig. 7. Summary of experiments probing competition between<br>
thrombin aptamer constructs and fluorescein-labeled G15D aptamer (THR2) for binding to thrombin. Fluorescence intensity of fluorescein-iabeled G15D aptamer (THR2) (50 nM) in the absence and the presence of the competitor (250 nM) was used to determine % of THR2 bound in the presence of the competitor. Thrombin concentration was 75 nM. The values of dissociation constants shown in the figure were calculated from a separate experiment in which 200 nM fluorescein-labeled G15D aptamer (THR2), 200 nM competitor and 150 nM thrombin were used.<br>
[022]	Fig. 8. The effect of 60-18 [29] aptamer {THR3) on the<br>
competition between fluorescein-labeled G15D aptamer (THR2) and THR5 construct<br>
 <br>
for binding to thrombin. Fluorescence spectra of 200 nM fluorescein-labeled G15D (THR2) with and without thrombin (150nM)in the absence of the competitor (A), in the presence of 1000 nM THR3 and 200 nM THR5 (B), in the presence of 1000 nM THR3 (C), and in the presence of 200 nM THR5 (D).<br>
[023]	Fig. 9. Binding of THR7 aptamer construct to thrombin detected<br>
by gel electrophoresis mobility shift assay. Samples of 417 nM THR7 were incubated with various amounts of thrombin (0 to 833 nM) and after 15 min incubation were loaded on a native 10% polyacrylamide gel. (A) Image of the gel stained with Sybr Green. (B) Intensity of the band corresponding to THR7-thrombin complex as a function of thrombin concentration.<br>
[024]	Fig. 10. Family of bivalent thrombin aptamer constructs in which<br>
G15D (black arrows) and 60-18 [29] (gray arrows) aptamers were connected to a 20 bp DNA duplex by a 9-27 nt long poly T linker.<br>
[025]	Fig.11. Binding of thrombin to bivalent aptamer constructs (33<br>
nM each) illustrated in Fig. 10 detected by electrophoretic mobility shift assay (EMSA). Asterisk marks the lane best illustrating preferential binding of thrombin to constructs with 27 and 17 nt poly T linker over the constructs with 9 nt poly T linker. Thrombin concentration was varied from 0 to 400 nM.<br>
[026]	Fig. 12. Thrombin beacon design using G15D (black arrows)<br>
and 60-18 [29] (gray arrows) aptamers connected to 9 bp fluorophore (or quencher)-labeled "signaling" duplex through 17 nt poly T linker. (A) Nucleotide sequence of the fluoroscoin-labeled G15D construct (THR9) and dabcyl-labeled 60-18 [29] construct (THRB). (B) Mechanism of signaling by thrombin beacon. (C) Fluorescence signal change detected upon addition of thrombin to the thrombin beacon. For comparison, titration of the fluorescein-labeled G15D construct (THR9) with thrombin in the absence of dabcyl-labeled 60-18 [29] constnjct (THR8) is also shown (donor only curve).<br>
[027]	Fig. 13. A thrombin beacon design. G15D (black arrows) and<br>
60-18 [29] (gray arrows) aptamers were connected to 7 bp fluorophore (or quencher)-labeled "signaling" duplex through a linker containing 5 Spacer18 units.<br>
(A)	Nucleotide sequence of the fluorescein-labeled G15D construct (THR21) and dabcyl-lal)eled 60-18 [29] construct (THR20). X corresponds lo Spacer 18 moiety.<br>
(B)	Mnchanism of signaling by thrombin beacon. (C) Fluoroyconce sicinal change<br>
 <br>
deteclod upon addition of thrombin to the thrombin beacon, For comparison, titration of the fluorescein-labeled G15D construct (THR21) with thrombin in the absence of dabcyl-labeled 60-18 [29] construct (THR20) is also shown {donor only curve). Signal change (%) was calculated as 100* {lo -l)/lo where I and lo correspond to dilution-corrected fluorescence emission intensity observed in the presence and absence of a given thrombin concentration, respectively. Inset shows fluorescence emission spectra recorded at various concentrations of thrombin corresponding to data points in the main graph.<br>
[028]	Fig. 14. Binding of thrombin to the beacon illustrated in Fig. 13<br>
(THR20/THR21) detected by gel electrophoresis mobility shift assay. The gel was imaged for fluorescein emission (i.e. only THR21 component of the beacon is visible).<br>
[029]	Fig. 15. (A) Sensitivity of thrombin detection at two different<br>
concentrations of the beacon. Squares: 50 nM THR21 and 95 nM THR2D. Circles: 5 nM THR21 and 9.5 nM THR20. (B) Specificity of the beacon for thrombin. 50 nM THR21 and 95 nM THR20 were titrated with thrombin (open circles) and trypsin (closed circles).<br>
[030]	Fig. 16. Reversal of thrombin beacon signal by competitor<br>
aptamer constructs. Fluorescence intensity of 50 nMTHR21, 95 nM THR20, and 100 nM thrombin was measured at increasing concentrations of competitor DNA's. The data are plotted as a relative fluorescence increase with respect to a signal (Fo) of a starting beacon and thrombin mixture. Open squares: THR7; filled circles: THR14/THR15; filled squares: THR16/THR17; filled triangles: THR18/THR19; open triangles: THR3; gray filled inverted triangles; THR4; open triangles: nonspecific single stranded DNA.<br>
[031]	Fig. 17. The binding of aptamer constructs to thrombin. (A)<br>
Binding of G15D aptamer (THR2) (50 nM) detected by change in fluorescence intensity of 5' fluorescein moiety. Solid line represents the best fit of the experimental data to a simple 1:1 binding isotherm. (B) Binding ofG15D aptamer (THR2) in the presence of 10x excess of unlabeled 60-18 [29] aptamer. Solid line represents the best fit of the experimental data to a simple 1:1 binding isotherm. (C) Summary of experiments probing competition between thrombin aptamer constructs and fluorescein-labeled G15D aptamer (THR2). Fluorescence intensity of THR2 (200 nM)<br>
 <br>
was used to dGtermine % THR2 bomd in the presence of competitor (200 nM). Thrombin was 150 nM. The labels above each bar indicate relative affinity (expressed as fold increase of affinity constant) of the competitor compared to the affinity of THR2 aptamer. (D) Binding of THR7 aptamer construct to thrombin detected by gel electrophoresis mobility shift assay. Intensity of the band corresponding to THR7-thrombin complex is plotted as a function of thrombin concentration. Inset: Image of the gel stained with Sybr Green. Fluorescence change (%) was calculated as 100* (l-lo)/lo where I and lo correspond to dilution-corrected fluorescence emission intensity observed in the presence and absence of a given thrombin concentration, respectively.<br>
[032]	Fig. 18. Variants of thrombin beacon with various combinations<br>
of donor-acceptor fluorophores. (A) fluorescein-dabcyl; (B) fluorescein-Texas Red;<br>
(C)	fluorescein-Cy5, (D) Cy3-Cy5. Emission spectra of the beacon in the absence (solid line) and presence (line with Xs) of thrombin are shown. Insets show images of microplate wells containing corresponding beacon and indicated concentrations of thrombin. The images were obtained on Bio-Rad Molecular Imager FX using the following excitation-emission settings: (A) 488 nm laser - 530 nm bandpass filter; (B) 488 nm laser - 640 nm bandpass filter; (C) 488 nm laser ~ 895 nm bandpass filter;<br>
(D)	532 nm laser - 695 nm bandpass filter. Fluorescence is in arbitrary units (corrected for instrument response) and is plotted in a linear scale.<br>
[033]	Fig. 19. Response curves for the beacon with various<br>
combinations of donor-acceptor pairs. (A) fluorescein-dabcyl, (B) fluorescein-Texas Red, (C) Cy3-Cy5, (D) fluorescein-Cy5, (E) europium chelate-Cy5, (F) Fold signal change observed for indicated donor-acceptor pair at saturating thrombin concentration. Insets show expanded view of data points at low thrombin concentrations. In all experiments 5 nM donor-labeled and 5.5 nM acceptor-labeled aptamer constructs were used. Signal change (fold) was calculated as l/lo where I and lo correspond to dilution-corrected acceptor fluorescence emission intensity (measured with donor excitation) observed in the presence and absence ofa given thrombin concentration, respectively. Buffer background was subtracted from I and lo before calculating signal change.<br>
[034]	Fig. 20. The dependence of the sensitivity of the thrombin<br>
beacon on a donor-acceptor pair. Response of 10 nM donor-labeled and 11 nM<br>
10<br>
 <br>
acceptor-labeled beacon was determined at low thrombin concentrations using beacon labeled with fluorescein-dabcyl pair (triangles), fluorescein-Texas Red pair (inverted triangles), and fluorescein-Cy5 pair (circles). Averages and standard deviations of four independent experiments are shown.<br>
[035]	Fig. 21. The reproducibility and stability of thrombin beacon. (A)<br>
Five independent determinations of beacon signal at four different thrombin concentrations were performed. Data shown represent mean +/- standard deviation. (B) Thrombin beacon signal at four thrombin concentrations was monitored over time up to 24 hours. Data shown represent mean +/- standard deviation of 5 independent measurements. Beacon containing 5 nM lluorescein-labeled aptamer (THR21) and 5.5 nM Texas Red-labeled aptamer (THR27) was used in this experiment.<br>
[036]	Fig. 22. shows the determination of Z'-factor for thrombin<br>
beacon. Panel in the middle of the plot shows an image of wells of the microplate corresponding to the experiment shown in a graph (the upper half of wells are + thrombin, the lower half of the wells is - thrombin. Beacon containing with 5 nM fluorescein-labeled aptamer (THR21) and 5.5 nM Texas Red-labeled aptamer (THR27) was used in this experiment. Signal corresponds to a ratio of acceptor to donor emission (in arbitrary units) measured with donor excitation.<br>
[037]	Fig. 23. The detection of thrombin in complex mixtures. (A)<br>
Response of thrombin beacon at 1 nM thrombin concentration in the absence and presence of the excess of unrelated proteins. The data shown are averages and standnrd deviation of 4 independent experiments. (R) DelRctinn of thrombin in HeLa extract "spiked" with various amounts of thrombin. Data shown aro averages and standard deviation from 3 independent measurements. Concontralions of thrombin in cell extract were (from left to right); 0, 1.88 nM, 3.75 nM, and 7.5 nM. Signal for beacon mixture alone was - 25% lower then when cell extract (no thrombin added) was present (not shown) which was essentially the same as the signal observed in the presence of cell extract and specific competitor. (C) Time course of prothrombin to thrombin conversion catalyzed by Factor Xa monitored by thrombin beacon. (D) Detection of thrombin in plasma. Data shown are averages and standard deviation from 4 independent measurements. The volumes of plasma used (per 20 ml assay mixture) were (from left to right): 0 ml, 0.005 ml, 0.015 ml, and 0.045 ml. "Specific" refers to unlabeled thrombin aptamer competitor (THR7) whereas "nonspecific"<br>
11<br>
 <br>
refers to random sequence 30 nt DNA. Signal in panels A, B and D corresponds to a ratio of acceptor to donor emission measured with donor excitation. Signals were normalized to value of 1 for beacon mixture alone (panels A and D) and beacon mixture in the presence of cell extract (panel B). Panel C shows raw acceptor fluorescence intensity (with donor excitation).<br>
[038]	Fig. 24. Various formations of molecular biosensors.<br>
[039]	Fig. 25. The experimental demonstration of the sensor design<br>
shown in Fig. 24F. (A) Principle of sensor function. (B) Increase of sensitized acceptor fluorescence upon titration of increasing concentrations of DNA binding protein to the mixture of donor and acceptor labeled sensor components.<br>
[040]	Fig. 26. The experimental demonstration of a functioning sensor<br>
design shown in Fig. 24G. (A) Principle of sensor function. (B) Increase of sensitized acceptor fluorescence (emission spectrum labeled with"+"} upon addition of single-stranded DNA containing two distinct sequence elements complementary to sensor elements to the mixture of two donor and acceptor labeled sensor components (spectrum labeled with"-").<br>
[041]	Fig. 27. The experimental demonstration of the increased<br>
specificity of the sensor design compared to assays based on a single, target macromolecule-recognizing element. (A) Three molecular contacts providing free energy (AG). (B) Nonspecific binding. (C) No signal with the beacon.<br>
[042]	Fig. 28. Summarizes the selection of an aptamer that binds to<br>
thrombin at an epitope distinct from the binding site of the G15D aptamer. (A) An illustration of the reagents used to begin the process of selection. (B) The graph indicates the increase in thrombin binding with successive rounds of selection. (C) The sequences represent aptamers developed after 12 rounds of selection.<br>
[043]	Fig. 29. The demonstration of the functional thrombin sensor<br>
comprising Texas Red-labeled THR27 and fluorescein-labeled THR35 or THR36 (both contain the sequence corresponding to that of clones 20, 21, 24, and 26 of Figure 28C). The fluorescence image represents the specificity of either 20nM (panel A) or 10OnM (panel B) of the indicated biosensor.<br>
[044]	Fig. 30. Summarizes the simultaneous selection of two<br>
aptamers that bind to thrombin at distinct epitopes. (A) An illustration of the reagents used to begin the process of selection. (B) The graph Indicates the increase in<br>
12<br>
 <br>
thrombin binding with successive rounds of selection. (C) The sequences represent aptamors developed after 13 rounds of selection.<br>
[045]	Fig. 31. Summarizes the selection of an aptamer that binds to<br>
CRP at an epitope distinct from the DNA-binding site. (A) An illustration of the reagents used to begin the process of selection. (B) The graph indicates the increase in thrombin binding with successive rounds of selection. (C) The sequences represent aptamers developed after 11 rounds of selection.<br>
[046]	Fig. 32. A diagram of methods for permanently linking the two<br>
aptamers recognizing two distinct epitopes of the target. (A) Two complimentary oligonucleotides are attached using linkers to each of the aptamers. These oligonucleotides are long enough (typically &gt;15bps) to form a stable duplex permanently linking the two aptamers. (B) The two aptamers are connected directly via a linker.<br>
[047]	Fig. 33. Example of a potential sensor design utilizing three<br>
sensing components. In this design the target is a complex of three components (black, dark gray, and light gray ovals). Each of the aptamers recognizes one of the components of the complex. Signals of different color from each of the two signaling oligonucleotide pairs could be used to discriminate between the entire complex containing all three components with alternative sub-complexes containing only two of the components.<br>
[048]	Fig. 34. Depicts an experiment demonstrating the feasibility of<br>
an antibody-based molecular biosensor as shown in Fig. 24D. (A) Design of the model system used; (B) Signal generated by the sensor at various concentrations of biotin labeled CRP. The signal corresponds to intensity of emission at 670 nm (Cy5) upon excitation at 520 nm (fluorescein) (C) Specificity of the sensor response. The FRET signal is responsive to both cAMP and streptavidin.<br>
[049]	Fig. 35. Depicts an experiment demonstrating the feasibility of<br>
an antibody-based molecular biosensor composed of two antibodies recognizing distinct epitopes of the same target. (A) Design of the model system used; (B) Signal generated by the sensor at various concentrations of biotin-DNA-digoxin. Signal corresponds to intensity of emission at 670 nm (Cy5) upon excitation at 520 nm (fluorescein) (C) Specificity of pincer response.<br>
13<br>
 <br>
[050]	Fig. 36. Illustrates (he procedure for attaching signaling<br>
oligonucleotides to antibodies. Peaks 1-3 denote three peaks eluting from a ResourceQ column. Samples from these peaks were run on a native polyacrylamide gel and visualized using a Molecular Imager FX with fluorescein emission settings (the signaling oligonucleotide used to label the antibody was labeled with fluorescein). No fluorescence was found in peak 1 (suggesting that it contained unlabeled antibody. Peaks 2 and 3 produced fluorescent bands of different mobility indicating that they contain antibody labeled with one {peak 2) or more (peak 3) signaling oligonucleotides.<br>
[051]	Fig. 37. Depicts the relative signal change for a thrombin sensor<br>
obtained with various combinations of donor-acceptor pairs (Hoyduk and Heyduk. AnalChem, 77:1147-56, 2005).<br>
[052]	Fig. 38. (A) Design of the model for a competitive molecular<br>
sensor for detecting a protein. (B) Design of a competitive sensor for detecting an antigen.<br>
[053]	Fig. 39. Depicts an experiment demonstrating the feasibility of<br>
an antibody-based competitive molecular biosensor. (A) Design of the model for a competitive molecular sensor. (B) Titration of the beacon with unlabeled competitor (biotin-labeled oligonucleotide).<br>
[054]	Fig. 40. Design of competitive antibody beacon utilizing the<br>
antigen attached to fluorochrome-labeled complementary signaling oligonucleotides. Binding of the antigens to the bivalent antibody results in proximity-driven hybridization of the signaling oligonucleotides generating a FRET signal. In the presence of the competitor antigen, fluorochrome-labeled antigen is displaced by the competitor resulting in a decrease in FRET signal. This decrease in FRET signal can be used to detect the presence of the antigen.<br>
[055]	Fig. 41. Implementation of the design illustrated in Fig. 40 for<br>
detection of proteins. Synthetic peptide containing the epitope recognized by the antibody is attached to fluorochrome-labeled complementary signaling oligonucleotides. Binding of the peptide to the bivalent antibody results in proximity-driven hybridization of the signaling oligonucleotides generating a FRET signal, in the presence of the protein containing the same epitope, fluorochrome-labeled<br>
14<br>
 <br>
peptide is displaced by the competitor resulting in a decrease in FRET signal. This decrease in FRET signal can be used to detect the presence of the protein.<br>
[056]	Fig. 42. (A) Design of a model system to test the design of<br>
competitive antibody beacon illustrated in Fig. 40. Fluorochrome-labeled complementary signaling oligonucleotides attached to long flexible linkers were labeled with biotin. In the presence of anti-biotin antibody the two constructs will bind to the antibody resulting in a FRET signal. (B) Experimental verification of the reaction shown in panel A. A mixture of biotinylated signaling oligonucleoWdes (50 nM biotinylated ANTB8 labeled with fluorescein and 50 nM biotinylated ANTB6 labeled with Cy5) was titrated with polyclonal anti-biotin antibody (upper curve). As a control, the same mixture of biotinylated oligonucleotides was also titrated with a monovalent Fab anti-biotin antibody fragment. No FRET signal was observed in this case (lower line) consistent with the need for a bivalent antibody to generate the FRET signal.<br>
[057]	Fig. 43. Proof-of-princip!e evidence for the feasibility of the<br>
competitive antibody beacon illustrated in Fig. 40. (A) Design of the assay. (B) Mixture of 50 nM biotinylated ANTS8 labeled with fluorescein, 50 nM biotinylated ANTB6 labeled with Cy5 and 50 nivl anti-biotin antibody was titrated with a specific competitor (unrelated biotinylated oligonucleotide) resulting in expected concentration-dependent decrease in FRET signal (gray circles). No decrease in FRET was observed upon titration with the same oligonucleotide lacking biotin (black circles). In the absence of anti-biotin antibody only background signal was observed which was unaffected by addition of the specific competitor (white circles).<br>
[058]	Fig. 44. Table depicting blood clotting times for a thrombin<br>
beacon and its individual component aptamers,<br>
[059]	Fig. 45. Sensor for p53 protein comprising a DNA molecule<br>
containing a p53 binding site and an anti-p53 antibody. (A) FRET signal in the presence of varying concentrations of full-length recombinant p53. The sensor design is shown schematically in the inset. (B) Specificity of sensor signal in the presence of p53.<br>
[060]	Fig. 46. Sensor for cardiac Troponin I (CTnl) based on two<br>
antibodies recognizing nonoverlapping epitopes of the protein. Plotted is the FRET<br>
15<br>
 <br>
signal with 50 nM of the sensor components in the presence of increasing concentrations of troponin I.<br>
[061]	Fig. 47. Response of troponin sensor at various concentrations<br>
of sensor components to different concentration of troponin I (Ctnl).<br>
[062]	Fig. 48. Competitive sensor for cardiac Troponin I (CTnl). (A)<br>
FRET signal in the presence of increasing concentration of the anti-troponin antibody. The inset demonstrates competition by the unlabeled N-terminal CTnl peptide. (B) Competitions with intact CTnl protein. (0) Design of the sensor.<br>
[063]	Fig. 49. Comparison of the two-component sensor design (A)<br>
and the three-component sensor design (B).<br>
[064]	Fig. 50. Proof-of-principle for the three-component sensor<br>
design. The FRET signal is plotted as a function of various concentrations of the target.<br>
[065]	Fig. 51. Insensitivity of the three-component sensor design to<br>
the concentration of the S3 component. (A) Schematics illustrating the binding of SI and S2 in the absence (top) and presence (bottom) of target. (B) Experimental confirmation of the principles described in (A), FRET signal of the sensor was measured in the presence and absence of target (T) at various concentrations of S3. Inset plots the background signal in the absence of T at various concentrations of S3.<br>
[066]	Fig. 52. Homogenous signal amplification utilizing the three-<br>
component sensor design. Hybridized S3 comprises a restriction endonuclease recognition site.<br>
[067]	Fig. 53. Proof-of-principle for the signal amplification scheme<br>
depicted in Fig. 52. Cleavage of S3 by Hinc II was monitored by native gel electrophoresis at various concentrations of target (T) for 4 hours (A) or 24 hrs (B). (C) The proportion of cleaved S3 after 4 hrs is plotted as a function of T concentration.<br>
[066]	Fig. 54. Solid-surface implementation of the three-component<br>
biosensor design.<br>
[069]	Fig. 55. Proof-of-principle for the solid-surface implementation<br>
of the three-component biosensor design. (A) Design of the sensor employing TIRF detection. (B) FRET signals in the presence and absence of target (T) over time.<br>
16<br>
 <br>
[070]	Fig. 56. Use of the three-component biosensor design for a<br>
microarray detection of a target.<br>
DETAILED DESCRIPTION OF THE PREFERRED EMBODIMENTS<br>
[071]	The present invention is directed to molecular biosensors that<br>
may be utilized in several different methods, such as the detection of a target molecule. In one design, the biosensor is comprised of two components, which comprise two epitope-binding agent constructs. In the two-component design. detection of a target molecule typically involves target-molecule induced co-association of two epitope-binding agent constructs that each recognize distinct epitopes on the target molecule. The epitope-binding agent constructs each comprise complementary signaling oligonucleotides that are labeled with detection means and are attached to the epitope-binding agents through a flexible linker, Co-association of the two epitope-binding agent constructs with the target molecule results in bringing the two signaling oligonucleotides into proximity such that a detectable signal is produced.<br>
[072]	Alternatively, in another design the biosensor is comprised of<br>
three components, which comprise two epitope-binding agent constructs and an oligonucleotide construct. In the three-component design, analogous to the two-component design, detection of a target molecule typically involves target-molecule induced co-association of two epitope-binding agent constructs that each recognize distinct epitopes on the target molecule. Unlike the two-component design, however, the epitope-binding agent constructs each comprise non-complementary signaling oligonucleotides that are labeled with detection means and are attached to the epitope-binding agents through a flexible linker. Each signaling oligonucleotide is complementary to two distinct regions on the oligonucleotide construct. Co-association of the two epitope-binding agent constructs with the target moiecuie results in hybridization of each signaling oligonucleotide to the oligonucleotide construct. Binding of the two signaling oligonucleotides to the oligonucleotide construct brings them into proximity such that a detectable signal is produced.<br>
[073]	Advantageously, the molecular biosensors, irrespective of the<br>
design, provide a rapid homogeneous means to detect a variety of target molecules, including but not limited to proteins, carbohydrates, macromolecules, and analytes.<br>
17<br>
 <br>
In particular, as illustrated in the Examples, the three-component biosensors are<br>
useful in several applications involving solid surfaces.<br>
(I) Two-Component Molecular Biosensors<br>
[074]	One aspect of the invention, accordingly, encompasses a two-<br>
component molecular biosensor. Several molecular configurations of biosensors are suitable for use in the invention as illustrated by way of non-limiting example in Figs. 24, 33, and 38. In one embodiment, the molecular biosensor will be monovalent comprising a single epitope-binding agent that binds to an epitope on a target molecule. The molecular biosensor of the invention, however, is typically multivalent. It will be appreciated by a skilled artisan, depending upon the target molecule, that the molecular biosensor may comprise from about 2 to about 5 epitope binding agents. Typically, the molecular biosensor will comprise 2 or 3 epitope binding agents and more typically, will comprise 2 epitope binding agents. In one alternative of this embodiment, therefore, the molecular biosensor will be bivalent comprising a first epitope binding agent that binds to a first epitope on a target molecule and a second epitope binding agent that binds to a second epitope on the target molecule. In yet another alternative of this embodiment, the molecular biosensor will be trivalent comprising a first epitope binding agent that binds to a first epitope on a target molecule, a second epitope binding agent that binds to a second epitope on a target molecule and a third epitope binding agent that binds to a third epitope on a target molecule.<br>
(a)      bivalent molecular sensors<br>
[075]	In one alternative of the invention, the molecular biosensor will<br>
be bivalent. In a typical embodiment, the bivalent construct will comprise a first epitope binding agent that binds to a first epitope on a target molecule, a first linker, a first signaling oligo, a first detection means, a second epitope binding agent that binds to a second epitope on the target molecule, a second linker, a second signaling oligo, and a second detection means.<br>
[076]	In one preferred embodiment, the molecular biosensor<br>
comprises two nucleic acid constructs, which together have formula (I):<br>
R'-R^-R^-R-^; and<br>
18<br>
 <br>
R^-R^-R^-R^       (i) wherein:<br>
R^ is an epitope-binding agent tinat binds to a first epitope on a target molecuie;<br>
R^ is a flexible linker attaching R^ to R^; R^ and R^ are a pair of complementary nucleotide sequences having a free energy for association from about 5.5 kcal/mole to about 8.0 kcal/mole at a temperature from about 21° C to about 40° C and at a salt concentration from about 1 mM to about 100 mM;<br>
R^ and R^ together comprise a detection means such that when R^ and R'^ associate a detectable signal is produced;<br>
R^ is an epitope binding agent that binds to a second epitope on the target molecule; and<br>
R^ is a flexible linker attaching R^ to R^<br>
[077]	As will be appreciated by those of skill in the art, the choice of<br>
epitope binding agents, R^ and R^, in molecular biosensors having formula (I) can and will vary depending upon the particular target molecule. By way of example, when the target molecule is a protein, R' and R'^ may be an aptamer, or antibody. By way of further example, when R^ and R^ are double stranded nucleic acid the target molecule is typically a macromolecule that binds to DNA or a DNA binding protein. In general, suitable choices for R"" and R^ will include two agents that each recognize distinct epitopes on the same target molecule. In certain embodiments, however, it Is also envisioned that R"" and R^ may recognize distinct epitopes on different target molecules. Non-limiting examples of suitable epitope binding agents, depending upon the target molecule, include agents selected from the group consisting of an aptamer, an antibody, an antibody fragment, a double-stranded DNA sequence, modified nucleic acids, nucleic acid mimics, a ligand, a Iigand fragment, a receptor, a receptor fragment, a polypeptide,.a peptide, a coenzyme, a coreguiator, an allosteric molecule, and an ion. In an exemplary embodiment, R' and R' are each aptamers having a sequence ranging in length from about 20 to about 110 bases. In another embodiment, R^ and R^ are each antibodies selected from the group consisting of polyclonal antibodies, ascites. Fab fragments, Fab' fragments, monoclonal antibodies, and humanized antibodies. In an alternative embodiment, R"*<br>
19<br>
 <br>
and R^ are peptides. In a preferred embodiment, R' and R^ are each monoclonal antibodies. In an additional embodiment, R^ and R^ are each double stranded DNA. In a further embodiment, R^ is a double stranded nucleic acid and R^ is an aptamer. In an additional embodiment, R^ is an antibody and R^ is an aptamer. In another additional embodiment, R^ is an antibody and R^ is a double stranded DNA.<br>
[078]	In an additional embodiment for molecular biosensors having<br>
formula {!), exemplary linkers, R^ and R^, will functionally keep R^ and R^ in close proximity such that when R^ and R^ each bind to the target molecule, R^ and R' associate in a manner such that a detectable signal is produced by the detection means, R"* and R®. R^ and R^ may each be a nucleotide sequence from about 10 to about 100 nucleotides in length. In one embodiment, R^ and R^ are from 10 to about 25 nucleotides in length. In another embodiment, R^ and R° are from about 25 to about 50 nucleotides in length. In a further embodiment, R^ and R^ are from about 50 to about 75 nucleotides in length. In yet another embodiment, R^ and R^ are from about 75 to about 100 nucleotides in length. In each embodiment, the nucleotides comprising the linkers may be any of the nucleotide bases in DNA or RNA (A, C, T, G in the case of DNA, or A, C, U, G in the case of RNA).   In one embodiment R^ and R^ are comprised of DNA bases. In another embodiment, R^ and R** are comprised of RNA bases. In yet another embodiment, R^ and R*" are comprised of modified nucleic acid bases, such as modified DNA bases or modified RNA bases. Modifications may occur at, but are not restricted to, the sugar 2' position, the C-5 position of pyrimidines, and the 8-position of purines. Examples of suitable modified DNA or RNA bases include 2'-fluoro nucleotides, 2'-amino nucleotides, 5'-aminoallyl-2'-fluoro nucleotides and phosphorothioate nucleotides {monothiophosphate and dithiophosphate). In a further embodiment, R^ and R^ may be nucleotide mimics. Examples of nucleotide mimics include locked nucleic acids (LNA), peptide nucleic acids (PNA), and phosphorodiamidate morpholino oligomers (PMO). Alternatively, R^ and R^ may be a polymer of bifunctional chemical linkers. In one embodiment the bifunctional chemical linker is heterobifunctional. Suitable hetGrobifunctional chemical linkers include sulfoSMCC (Sulfosuccinimidyl-4-(N-maleimidomethyl}cyclohexane-1-carboxylate), and lc-SPDP( N-Succinimidyl-6-(3'-(2-PyridylDithio)-Propionamido)-hexanoate). In another embodiment the bifunctional chemical linker is homobifunctional. Suitable homobifunctional linkers include<br>
20<br>
 <br>
disuccinimidyl suberate, disuccinimidyl glutarate, and disucclnimidyl tartrate. Additional suitable linkers are illustrated in the Examples, such as the phosphoramidate form of Spacer 18 comprised of polyethylene glycol. In one embodiment, R^ and R^ are from 0 to about 500 angstroms in length. In another embodiment, R^ and R^ are from about 20 to about 400 angstroms in length. In yet another embodiment, R^ and R^ are from about 50 to about 250 angstroms in length.<br>
[079]	In a further embodiment for molecular biosensors having formula<br>
(I), R^ and R^ are complementary nucleotide sequences having a length such that they preferably do not associate unless R^ and R^ bind to separate epitopes on the target molecule. When R^ and R^ bind to separate epitopes of the target molecule, R^ and R^ are brought to relative proximity resulting in an increase in their local concentration, which drives the association of R^ and R^. R^ and R^ may be from about 2 to about 20 nucleotides in length. In another embodiment, R^ and R^ are from about 4 to about 15 nucleotides in length. In an exemplary embodiment, R^ and R^ are from about 5 to about 7 nucleotides in length. In one embodiment, R^ and R^ have a free energy for association from about 5.5 kcal/mole to about 8.0 kcal/mole as measured in the selection buffer conditions, defined below. In another embodiment, R^ and R' have a free energy for association from about 6.0 kcal/mole to about 8.0 kcal/mote as measured in the selection buffer conditions defined below. In yet another embodiment, R^ and R^ have a free energy for association from about 7.0 kcal/mole to 8.0 kcal/mole in the selection buffer conditions. In a preferred embodiment, R^ and R^ have a free energy for association of 7.5 kcal/mole in the selection buffer conditions described below. Preferably, in each embodiment R^ and R^ are not complementary to R^ and R^.<br>
[080]	In a typical embodiment for molecular biosensors having formula<br>
(I), R" and R^ may together comprise several suitable detection means such that when R^ and R^ associate, a detectable signal is produced. Exemplary detections means suitable for use in the molecular biosensors include fluorescent resonance energy transfer (FRET), lanthamide resonance energy transfer (LRET), fluorescence cross-correlation spectroscopy, flourescence quenching, fluorescence polarization, flow cytometry, scintillation proximity, luminescence resonance energy transfer, direct quenching, ground-state complex formation, ohemiluminescence energy transfer, bioluminescence resonance energy transfer, excimer formation, colorimetric<br>
21<br>
 <br>
substrates detection, phosphorescence, electro-chemical changes, and redox potential changes.<br>
[081]	In a further embodiment, the molecular biosensor will have formula<br>
(I)<br>
wherein:<br>
R^ is an epitope-binding agent that binds to a first epitope on a target molecule and is selected from the group consisting of an aptamer, an antibody, a peptide, and a double stranded nucleic acid;<br>
R^ is a flexible linker attaching R"" to R^ by formation of a covalent bond with each of R"" and R^, wherein R^ comprises a bifunctional chemical cross linker and is from 0 lo 500 angstroms in length;<br>
R^ and R^ are a pair of complementary nucleotide sequences from about 4 to about 15 nucleotides in length and having a free energy for association from about 5.5 kcal/mole lo about 8.0 kcal/mole at a temperature from about 21° C to about 40" C and at a salt concentration from about 1 mM to about 100 mM;<br>
R"* and R^ together comprise a detection means selected from the group consisting of fluorescent resonance energy transfer (FRET), lanthamide resonance energy transfer (LRET), fluorescence cross-correlation spectroscopy, flourescence quenching, fluorescence polarization, flow cytometry, scintillation proximity, luminescence resonance energy transfer, direct quenching, ground-state complex formation, ohemiluminescence energy transfer, bioluminescence resonance energy transfer, excimer formation, colorimetric substrates detection, phosphorescence, electro-chemical changes, and redox potential changes;<br>
R^ is an epitope binding agent that binds to a second epitope on the target molecule and is selected from the group consisting of an aptamer, an antibody, a peptide, and a double stranded nucleic acid; and<br>
R^ is a flexible linker attaching R^ to R' by formation of a covalent bond with each of R^ and R^, wherein R^ connprlses a<br>
22<br>
 <br>
bifunctional chemical cross linker and is from 0 to 500 angstroms in<br>
length.<br>
[082]	Yet another embodiment of the invention encompasses a<br>
molecular biosensor having formula (I) wherein:<br>
R^ is an aptamer that binds to a first epitope on a target molecule;<br>
R^ is a flexible linker attaching R^ to R^; R^ and R' are a pair of complementary nucleotide sequences having a free energy for association from about 5.5 kcal/mole to about 8.0 kcal/mole at a temperature from about 21 ° C to about 40° C and at a salt concentration from about 1 mM to about 100 mM;<br>
R"* and R^ together comprise a detection means such that when R^ and R^ associate a detectable signal is produced;<br>
R^ is an aptamer that binds to a second epitope on the target molecule; and<br>
R^ is a flexible linker attaching R^ to R^.<br>
[083]	A further embodiment of the invention encompasses a<br>
molecular biosensor having formula (I) wherein:<br>
R^ is an aptamer that binds to a first epitope on a target molecule;<br>
R^ is a flexible linker attaching R^ to R^ by formation of a covalent bond with each of R^ and R^, wherein R^ comprises a bifunctional chemical cross linker and is from 0 to 500 angstroms in length;<br>
R^ and R^ are a pair of complementary nucleotide sequence from about 4 to about 15 nucleotides in length and having a free energy for association from about 5.5 kcal/mole to about 8.0 kcal/mole at a temperature from about 21" C to about 40" C and at a salt concentration from about 1 mM to about 100 mM;<br>
R"* and R^ together comprise a detection means selected from the group consisting of fluorescence resononance energy transfer<br>
23<br>
 <br>
(FRET), lanthamide resonance energy transfer (LRET), fluorescence cross-correlation spectroscopy, flourescence quenching, fluorescence polarization, flow cytometry, scintillation proximity, luminescence resonance energy transfer, direct quenctiing, ground-state complex formation, chemiluminescence energy transfer, bioluminescence resonance energy transfer, excimer formation, colorimetric substrates detection, phosphorescence, electro-chemical changes, and redox potential changes;<br>
R^ is an aplamer that binds to a second epitope on the target molecule; and<br>
R^ is a flexible linker attaching R^ to R^ by formation of a<br>
covalent bond with each of R^ and R^, wherein R^ comprises a<br>
bifunctional chemical cross linker and is from 0 to 500 angstroms in<br>
length.<br>
[084]	Yet another embodiment of the invention encompasses a<br>
molecular biosensor having formula (I) wherein:<br>
R^ is an peptide that binds to a first epitope on a target molecule; R^ is a flexible linker attaching R^ to R^ R^ and R^ are a pair of complementary nucleotide sequences having a free energy for association from about 5.5 kcal/mole to about 8.0 kcal/mole at a temperature from about 21° C to about 40° C and at a salt concentration from about 1 mM to about 100 mM;<br>
R"* and R^ togettier comprise a detection means such that when R^ and R^ associate a detectable signal is produced;<br>
R^ is an peptide that binds to a second epitope on the target molecule; and<br>
R^ is a flexible linker attaching R^ to R^<br>
[085]	Yet another embodiment of \he invention encompasses a<br>
molecular biosensor having formula (!) wherein:<br>
R' is an antibody that binds to a first epitope on a target molecule;<br>
24<br>
 <br>
R^ is a flexible linker attaching R^ to R^; R^ and R^ are a pair of complementary nucleotide sequences having a free energy for association from about 5.5 kcal/mole to about 8.0 kcal/mole at a temperature from about 21" C to about 40" C and at a salt concentration from about 1 mM to about 100 mM;<br>
R" and R^ together comprise a detection means such that when R"* and R^ associate a detectable signal is produced;<br>
R^ is an antibody that binds to a second epitope on the target molecule; and<br>
R^ is a flexible linker attaching R^ to R^<br>
[086]	In each of the foregoing embodiments for molecular biosensors<br>
having formula (I), the first nucleic acid construct, R^-R^-R^-R**, and the second nucleic acid construct, R^-R^-R'^-R^, may optionally be attached to each other by a linker R*^ to create tight binding bivalent ligands. Typically, the attachment is by covalent bond formation. Alternatively, the attachment may be by non covalent bond formation. In one embodiment, R^ attaches R^ of the first nucleic acid construct to R^ of the second nucleic acid construct to form a molecule comprising:<br>
R'-R^-R'^R"<br>
 <br>
R<br>
 <br>
I.A<br>
 <br>
R^-R*^-R^-R**<br>
[087]	In a further embodiment, R^ attaches R^ of the first nucleic acid<br>
construct to R^ of the second nucleic acid construct to form a molecule comprising:<br>
 <br>
2 D-' D4<br>
.1<br>
R'-R--R'-R<br>
R^-R'^-R^-R^<br>
[088]	In yet another embodiment, R'-* attaches R^ of the first nucleic<br>
acid construct to R^ of the second nucleic acid construct lo form a molecule comprising:<br>
25<br>
 <br>
<br>
[089]	Generally speaking, R^ may be a nucleotide sequence from<br>
about 10 to about 100 nucleotides in length. The nucleotides comprising R"-"^ may be any of the nucleotide bases in DNA or RNA (A, C, T, G in the case of DNA, or A, 0. U, G in the case of RNA).   In one embodiment, R'-'^ is comprised of DNA bases. In another embodiment, R"^ is comprised of RNA bases. In yet another embodiment, R^ is comprised of modified nucleic acid bases, such as modified DNA bases or modified RNA bases. Modifications may occur at, but are not restricted to, the sugar 2' position, the C-5 position of pyrimidines, and the 8-position of purines. Examples of suitable modified DNA or RNA bases include 2'-fluoro nucleotides, 2'-amino nucleotides, 5'-aminoallyl-2'-fluoro nucleotides and phosphorothioate nucleotides (monothiophosphate and dithiophosphate), In a further embodiment, R^ and R' may be nucleotide mimics. Examples of nucleotide mimics Include locked nucleic acids (LNA), peptide nucleic acids (PNA), and phosphorodiamidate morpholino oligomers (PMO). Alternatively, R"^ may be a polymer of bifunctional chemical linkers. In one embodiment the bifunctional chemical linker is heterobifunctionat. Suitable heterobifunctional chemical linkers include sulfoSMCC {Sulfosuccinimidyl-4-{N-maleimidomethyl)cyclohexane-1-carboxylate), and lc-SPDP( N-Succinimidyl-6'(3'-(2-PyridylDithio)-Propionamido)-hexanoate). In another embodiment the bifunctional chemical linker is homobifunotional. Suitable homobifunctional linkers include disuccinimidyl suberate, disuccinimidyl glutarate, and disuccinimidyi tartrate. An exemplary R"-* is the phosphoramidate form of Spacer 18 comprised of polyethylene glycol.   In one embodiment, R'-'^ is from about 1 to about 500 angstroms in length. In another embodiment, R'-'^ is from about 20 to about 400 angstroms in length- In yet another embodiment, R'"* is from about 50 to about 250 angstroms in length.<br>
26<br>
 <br>
(b) trivalent molecular sensors<br>
[090]	In an additional alternative embodiment, the molecular<br>
biosensor will be trivalent. In a typical embodiment, the trivalent sensor will comprise a first epitope binding agent that binds to a first epitope on a target molecule, a first linker, a first signaling oligo. a first detection means, a second epitope binding agent that binds to a second epitope on the target molecule, a second linker, a second signaling oligo, a second detection means, a third epitope binding agent that binds to a third epitope on a target molecule, a third linker, a third signaling oligo, and a third detection means.<br>
[091 ]	In one preferred embodiment, the molecular biosensor<br>
comprises three nucleic acid constructs, which together have formula (II);<br>
R16-R1^.R1«-R19.   and<br>
wherein:<br>
R^ is an epitope-binding agent that binds to a first epitope on a target molecule;<br>
R^° is a flexible linker attaching R^ to R"; R" and R^^ are a first pair of complementary nucleotide sequences having a free energy fur association from about 5.5 kcal/mole to about 8.0 kcal/mole at a temperature from about 21" C to about 40° C and at a salt concentration from about 1 mM to about 100 nfiM;<br>
R^^ and R" together comprise a detection means such that when R^^ and R^^ associate a detectable signal Is produced; R^^ is a flexible linker attaching R^' to R^"; R^* and R^^ are a second pair of complementary nucleotide sequences having a free energy for association from about 5.5 kcal/mole to about 8.0 kcal/mole at a temperature from about 21" C to about 40° C and at a salt concentration from about 1 mM to about 100 mM;<br>
27<br>
 <br>
R^ and R^^ together comprise a detection means such that when R^" and R^^ associate a detectable signal is produced;<br>
R^^ is an epitope-binding agent that binds to a second epitope on a target molecule;<br>
R^^ is a flexible linker attaching R^^ to R^^;<br>
R^" is an epitope binding agent that binds to a third epitope on a target molecule; and<br>
R^^ is a flexible linker attaching R^" to R^^<br>
[092]	The choice of epitope binding agents, R^, R""^ and R^, in<br>
molecular biosensors having formula {II) can and will vary depending upon the particular target molecule. Generally speaking, suitable choices for R", R""* and R^° will include three agents that each recognize distinct epitopes on the same target molecule or on different target molecules. Non-limiting examples of suitable epitope binding agents, depending upon the target molecule(s), include agents selected from the group consisting of an aptamer, an antibody, an antibody fragment, a double-stranded DNA sequence, modified nucleic acids, nucleic acid mimics, a ligand, a ligand fragment, a receptor, a receptor fragment, a polypeptide, a peptide, a coenzyme, a coregulator, an allosteric molecule, and an ion. In one embodiment, R®, R^^ and R^ are each aptamers having a sequence ranging in length from about 20 to about 110 nucleotide bases. In another embodiment, R°, R^^, and R^" are peptides. In yet another embodiment, R^, R^^, and R^" are antibodies or antibody fragments.<br>
[093)	In an additional embodiment for molecular biosensors having<br>
formula (II), exemplary linkers, R'" and R^\ will functionally keep R^"* and R^^ in close proximity such that when R^ and R^ each bind to the target molecule(s), R" and R^^ associate in a manner such that a detectable signal is produced by the detection means, R^^ and R". In addition, exemplary linkers, R" and R", will functionally keep R^" and R^" in close proximity such that when R^ and R^^ each bind to the target molecule(s), R^"* and R^° associate in a manner such that a detectable signal is produced by the detection means, R^^ and R^^. In one embodiment, the linkers utilized in molecular biosensors having formula (II) may each be a nucleotide sequence from about 10 to about 100 nucleotides in length. In one embodiment, the linkers are from 10 to about 25 nucleotides in length. In another embodiment, the<br>
28<br>
 <br>
linkers are from about 25 to about 50 nucleotides in length. In a further embodiment, the linkers are from about 50 to about 75 nucleotides in length. In yet another embodiment, the linkers are from about 75 to about 100 nucleotides In length. In each embodiment, the nucleotides comprising the linkers may be any of the nucleotide bases in DNA or RNA (A, C, T, G in the case of DNA, or A, C, U, G in the case of RNA).   (n one embodiment, the linkers are comprised of DNA bases. In another embodiment, the linkers are comprised of RNA bases. In yet another embodiment, the linkers are comprised of modiiled nucleic acid bases, such as modified DNA bases or modified RNA bases. Modifications may occur at, but are not restricted to, the sugar 2' position, the C-5 position of pyrimidines, and the 8-position of purines. Examples of suitable modified DNA or RNA bases include 2'-fluoro nucleotides, 2'-amino nucleotides, 5'-aminoallyl-2'-fluoro nucleotides and phosphorothioate nucleotides (monothiophosphate and dithiophosphate). In a further embodiment, R^ and R^ may be nucleotide mimics. Examples of nucleotide mimics inciude Iocl
[094]	In a further embodiment for molecular biosensors having<br>
formula (II), R" and R^^ are complementary nucleotide sequences having a length such that they preferably do not associate unless R" and R^° bind to separate epitopes on the target molecule(s). In addition, R^" and R^^ are complementary nucleoUde sequences having a length such that they preferably do not associate<br>
29<br>
 <br>
unless R° and R^^ bind to separate epitopes on the target molecuie(s). R^^ and R^^ and R^" and R^^ may be from about 2 to about 20 nucleotides in length. In another embodiment, R^' and R^^ and R^"* and R'^ are from about 4 to about 15 nucleotides in length. In an exemplary embodiment, R" and R^^ and R'" and R^^ are from about 5 to about 7 nucleotides in length. In one embodiment, R" and R^^ and R^" and R^^ have a free energy for association from about 5.5 kcal/mole to about 8,0 kcal/mole as measured in the selection buffer conditions, defined below. In another  . embodiment, R" and R^^ and R^* and R^^ have a free energy for association from about 6.0 kcal/mole to about 8.0 kcal/mole as measured in the selection buffer conditions defined below. In yet another embodiment, R" and R^^ and R^" and R'° have a free energy for association from about 7.0 kcal/mole to 8.0 kcal/mole in the selection buffer conditions. In a preferred embodiment, R^^ and R^^ and R^' and R'" have a free energy for association of 7,5 kcal/mole in the selection buffer conditions described below. Preferably, in each embodiment R" and R^^ and R" and R^° are not complementary to any of R^. R^^ or R^'.<br>
[095]	In a typical embodiment for molecular biosensors having<br>
formula (II), R"*^ and R^^ may together comprise several suitable detection means such that when R^^ and R^^ associate, a detectable signal is produced. In addition, R'^ and R^" may together comprise several suitable detection means such that when R^'* and R'" associate, a detectable signal is produced. Exemplary detections means suitable for use in the molecular biosensors include fluorescent resonance energy transfer (FRET), lanthamide resonance energy transfer (LRET), fluorescence cross-correlation spectroscopy, flourescence quenching, fluorescence polarization, flow cytometry, scintillation proximity, luminescence resonance energy transfer, direct quenching, ground-state complex formation, chemiluminescence energy transfer, bioluminescence resonance energy transfer, excimer formation, colorimetric substrates detection, phosphorescence, electro-chemical changes, and redox potential changes,<br>
(II) Three-Component Molecular Biosensors<br>
[096]	Another aspect of the invention comprises three-component<br>
molecular biosensors. In certain embodiments, the three-component molecular<br>
biosensor will comprise an endonuclease restriction site. In alternative<br>
30<br>
 <br>
embodiments, the three-component molecular biosensor will not have an endonuclease restriction site.<br>
(a) biosansors with no endonuclease restriction site<br>
[097]	In one embodiment, the three-component biosensor will<br>
comprise: (1) a first epitope binding agent construct that binds to a first epitope on a target molecule, a first linker, a first signaling oligo, and a first detection means; (2) a second epitope binding agent construct that binds to a second epitope on the target molecule, a second linker, a second signaling ollgo, and a second detection means; and (3) an oligonucleotide construct that comprises a first region that is complementary to the first oligo and a second region that is complementary to the second oligo. The first signaling oiigo and second signaling oligo, as such, are not complementary to each other, but are complementary to two distinct regions on the oligonucleotide construct. Co-association of the two epitope-binding agent constructs with the target molecule results in hybridization of each signaling oiigos to the oligonucleotide construct. Binding of the two signaling oligo to the oligonucleotide construct brings them into proximity such that a detectable signal is produced.<br>
[098]	In an exemplary embodiment, the three-component molecular<br>
biosensor comprises three nucleic acid constructs, which together have formula (III):<br>
R28_D29_R30_D31.<br>
O	(III)<br>
wherein:<br>
R^"* is an epitope-binding agent that binds to a first epitope on a target molecule;<br>
R^^ is a flexible linker attaching R^ to R^*"; R^^ and R^" are a pair of nucleotide sequences that are not complementary to each other, but are complementary to two distinct regions on O;<br>
R^^ and R^' together comprise a detection means such that when R^^ and R^° associate a detectable signal is produced;<br>
31<br>
 <br>
R^^ is an epitope-binding agent that binds to a second epitope<br>
on the target molecule;<br>
R^^ is a flexible linker attaching R^** to R^"; and<br>
O is a nucleotide sequence comprising a first region that is<br>
complementary to R^, and a second region that is complementary to<br>
[099]	The choice of epitope binding agents, R^" and R^^, in molecular<br>
biosensors having formula (III) can and will vary depending upon the particular target molecule. By way of example, when the target molecule is a protein, R^' and R^^ may be an aptamer, or antibody. By way of further example, when R^^ and R^° are double stranded nucleic acid the target molecule is typically a macromolecule that binds to DNA or a DNA binding protein. In general, suitable choices for R^' and R^^ wilt include two agents that each recognize distinct epitopes on the same target molecule. In certain embodiments, however, it is also envisioned that R^"* and R^" may recognize distinct epitopes on different target molecules. Non-limiting examples of suitable epitope binding agents, depending upon the target molecule, include agents selected from the group consisting of an aptamer, an antibody, an antibody fragment, a double-stranded DNA sequence, modified nucleic acids, nucleic acid mimics, a ligand, a ligand fragment, a receptor, a receptor fragment, a polypeptide, a peptide, a coenzyme, a coregulator, an allostehc molecule, and an ion. In an exemplary embodiment, R^' and R^° are each aptamers having a sequence ranging in length from about 20 to about 110 bases. In another embodiment, R^" and R^° are each antibodies selected from the group consisting of polyclonal antibodies, ascites. Fab fragments. Fab' fragments, monoclonal antibodies, and humanized antibodies, In an alternative embodiment, R^"" and R^" are peptides, in a preferred alternative of this embodiment, R^' and R^" are each monoclonal antibodies. In an additional embodiment, R^" and R^^ are each double stranded DNA. In a further embodiment, R^"* is a double stranded nucleic acid and R^ is an aptamer. In an additional embodiment, R^' is an antibody and R^^ is an aptamer. In another additional embodiment, R^* is an antibody and R^^ is a double stranded DNA.<br>
[0100]	In an additional embodiment for molecular biosensors having<br>
formula (III), exemplary linkers, R^^ and R^° may each be a nucleotide sequence from about 10 to about 100 nucleotides in length. In one embodiment, R^^ and R^^<br>
32<br>
 <br>
are from 10 to about 25 nucleotides in length. In another embodiment, R^ and R^ are from about 25 to about 50 nucleotides in length. In a further embodiment, R^^ and R^^ are from about 50 to about 75 nucleotides in length. In yet another embodiment, R^^ and R^ are from about 75 to about 100 nucleotides in length. In each embodiment, the nucleotides comprising the linkers may bo any of the nucleotide bases in DNA or RNA (A, C, T, G in the case of DNA, or A, C, U, G in the case of RNA).   In one embodiment R^^ and R^ are comprised of DNA bases. In another embodiment, R^^ and R^ are comprised of RNA bases. In yet another embodiment, R^^ and R^' are comprised of modified nucleic acid bases, such as modified DNA bases or modified RNA bases. Modifications may occur at, but are not restricted to, the sugar 2' position, the C-5 position of pyrimldines, and the 8-position of purines. Examples of suitable modified DNA or RNA bases include 2'-fluoro nucleotides, 2'-amino nucleotides, 5'-aminoallyl-2'-fluoro nucleotides and phosphorothioate nucleotides (monothiophosphate and dithiophosphate). In a further embodiment, R^^ and R^' may be nucleotide mimics. Examples of nucleotide mimics include locked nucleic acids (LNA), peptide nucleic acids (PNA), and phosphorodiamidate morpholino oligomers (PMO).<br>
[0101]	Alternatively, R^^ and R^^ may be a polymer of bifunctional<br>
chemical linkers. In one embodiment the bifunctional chemical linker is heterobifunctional. Suitable heterobifunotional chemical linkers include sulfpSMCC (Sulfosuccinimidyl-4-(N-maleimidomethyl)cyclohexane-1 -carboxylate), and lc-SPDP( N-SucGinimidyl-6-(3'-(2-PyndylDithio)-Propionamido)-hexanoate). In another embodiment the bifunctional chemical linker is homobifunctional. Suitable homobifunctional linkers include disuccinimidyl suberate, disuccinimidyl glutarate, and disuccinimidyl tartrate. Additional suitable linkers are illustrated in the Examples, such as the phosphoramidate form of Spacer 18 comprised of polyethylene glycol. In one embodiment, R^^ and R^^ are from 0 to about 500 angstroms in length. In another embodiment, R^^ and R^ are from about 20 to about 400 angstroms in length. In yet another embodiment, R^^ and R^ are from about 50 to about 250 angstroms in length.<br>
[01021	In a further embodiment for molecular biosensors having<br>
formula (III), R^^ and R^" are nucleotide sequences that are not complementary to each other, but that are complementary to two distinct regions of O. R^^ and R^<br>
33<br>
 <br>
may be from about 2 to about 20 nucleotides in length. In another embodiment, R^^ and R^ are from about 4 to about 15 nucleotides in length. In an exemplary embodiment. R^^ and R™ are from about 5 to about 7 nucleotides in length. Preferably, in each embodiment R^' and R^" are not complementary to R^'* and R^".<br>
[0103]	In a typical embodiment for molecular biosensors having<br>
formula (III), R^ and R^^ may together comprise several suitable detection means such that when R^ and R^° each bind to complementary, distinct regions on O, a detectable signal is produced. Exemplary detections means suitable for use in the molecular biosensors include fluorescent resonance energy transfer (FRET), lanthamide resonance energy transfer (LRET), fluorescence cross-correlation spectroscopy, flourescence quenching, fluorescence polarization, flow cytometry, scintillation proximity, luminescence resonance energy transfer, direct quenching, ground-state complex formation, chemiluminescence energy transfer, bioluminescence resonance energy transfer, excimer formation, colorimetric substrates detection, phosphorescence, electro-chemicai changes, and redox potential changes.<br>
[0104]	For molecular biosensors having formula (III), O comprises a<br>
first region that is complementary to R^^, and a second region Ihat is complementary to R^". O may be from about 8 to about 100 nucleotides In length. In other embodiments, O is from about 10 to about 15 nucleotides In length, or from about 15 to about 20 nucleotides in length, or from about 20 to about 25 nucleotides in length, or from about 25 to about 30 nucleotides in length, or from about 30 to about 35 nucleotides in length, or from about 35 to about 40 nucleotides in length, or from about 40 to about 45 nucleotides in length, or from about 45 to about 50 nucleotides in length, or from about 50 to about 55 nucleotides in length, or from about 55 to about 60 nucleotides in length, or from about 60 lo about 65 nucleotides in length, or from about 65 to about 70 nucleotides in length, or from about 70 to about 75 nucleotides in length, or from about 75 to about 80 nucleotides in length, or from about 80 to about 85 nucleotides in length, or from about 85 to about 90 nucleotides in length, or from about 90 to about 95 nucleotides in length, or greater than about 95 nucleotides in length.<br>
[0105}	In an exemplary embodiment, O will comprise formula'(IV)-.<br>
34<br>
 <br>
(IV]<br>
wherein:<br>
R^^, R^, and R^ are nucleotide sequences not complementary to any of R^^ R^", R^^ or R^^ R^^ R^, and R^^ may independently be from about 2 to about 20 nucleotides in length. In other embodiments, R^^, R^, and R^^ may independently be from about 2 to about 4 nucleotides in length, or from about 4 to about 6 nucleotides in length, or from about 6 to about 8 nucleotides in length, or from about 8 to about 10 nucleotides in length, or from about 10 to about 12 nucleotides in length, or from about 12 to about 14 nucleotides in length, or from about 14 to about 16 nucleotides in length, or from about 16 to about 18 nucleotides in length, or from about 18 to about 20 nucleotides in length, or greater than about 20 nucleotides in length;<br>
R^^ is a nucleotide sequence complementary to R^^, and<br>
R^^ is a nucleotide sequence that is complementary to R^".<br>
R^^ and R^^ generally have a length such that the free energy of association between R^^ and R^^ and R^^ and R^ is from about -5 to about -12 l<cal at a temperature from about c to and salt concentration mm mm. in other embodiments the free energy of association between r is kcal or greater than additional may range nucleotides length. atiout></cal>
35<br>
 <br>
(b) biosensors with an endonuclease restriction site<br>
[0106]	In an alternative embodiment, the three-component biosensor<br>
will comprise: (1) a first epitope binding agent construct that binds to a first epitope on a target molecule, a first linker, and a first signaling oligo; (2) a second epitope binding agent constmct that binds to a second epitope on the target molecule, a second linker, a second signaling oligo and (3) an oligonucleotide construct that comprises a first region that is complementary to the first oligo, a second region that is complementary to the second oligo, two flexible linkers, an endonuclease restriction site overlapping the first and the second regions complementary to the first and the second oligos, and a pair of complementary nucleotides with detection means. The first signaling oligo and second signaling oligo are not comptementary to each other, but are complementary to two distinct regions on the oligonucleotide constmct. Referring to Fig. 52, when the oligonucleotide construct is intact, the complementary nucleotides are annealed and produce a detectable signal. Co-association of the two epitope-binding agent constructs with the target molecule results in hybridization of each signaling oligo to the oligonucleotide construct. The signaling oligos hybridize to two distinct locations on the oligonucleotide construct such that a double-stranded DNA molecule containing the restriction site is produced, with a gap between the signaling oligos located exactly at the site of endonuclease cleavage in one strand of the double-stranded DNA substrate. When a restriction endonuclease is present, accordingly, it will cleave the oligonucleotide construct only when the target is present (i.e., when the signaling oligos ard bound to the oligonucleotide construct). Upon this cleavage, the detection means present on the oligonucleotide are separated-resulting in no detectable signal. Upon dissociation of the cleaved oligonucleotide construct, another oligonucleotide construct may hybridize with the signaling oligos of the two epitope-binding agents co-associated with the target and the cleavage reaction may be repeated. This cycle of hybridization and cleavage may be repeated many times resulting in cleavage of multiple oligonucleotide constructs per one complex of the two epitope-binding agents with the target.<br>
[0107]	In exemplary alternative of this embodiment, the three-<br>
component molecular biosensor comprises three nucleic acid constructs, which together have formula (V):<br>
36<br>
 <br>
p36_D37_Q3a, R39.R40.R41.<br>
O	(V)<br>
wherein:<br>
R^ is an epitope-binding agent that binds to a first epitope on a target molecule;<br>
R" is a flexible linker attaching R^^ to R-'"; R^^ and R"*^ are a pair of nucleotide sequences that are not complementary to each other, but are complementary to two distinct regions on O;<br>
R^^ is an epitope-binding agent that binds to a second epitope on the target molecule;<br>
R"" is a flexible linker attaching R^" to R'^ and O comprises:<br>
R'^ is a nucleotide construct comprising an endonuclease restriction site, a first region that is complementary to R^**, and a second region that is complementary to R'*^ R"^ is a first flexible linker;<br>
R^'*is a first nucleotide sequence that is complementary to R**^ attached to a detection means;<br>
R'^is a second flexible linker;<br>
R'^^is a second nucleotide sequence that is complementary to R'*' attached to a second detection means; and<br>
R*'^ attaches R"*^ to R'" and R"^ attaches R"^ to R'^<br>
[0108]	Suitable linkers, epitope binding agents, and detection means<br>
for three-component molecular biosensors having formula (V) are the same as three component molecular biosensors having formula (III). Suitable, endonuclease restriction sites comprising R"^ include sites that are recognized by restriction enzymes that cleave double stranded nucleic acid, but not single stranded nucleic acid. By way of non-limiting example, these sites include AccI, Agel, BamHI, Bgl, Bgll, BsiWI, BstBI, Clal, CviQI, Ddel, Dpnl, Dral, EagI, EcoRI, EcoRV, Fsel, Fspl, Haell, Haelil, Hhal. Hinc II, HinDIII, Hpal, Hpall, Kpnl, Kspl, Mbol, Mfel, Nael. Narl, Ncol, Ndel, Nhel, NotI, Phol, PstI, Pvul, Pvull, Sad, Sacll, Sail, Sbfl, Smal,.Spel,<br>
37<br>
 <br>
SphI, StuI, TaqI, Tfil, Tlil, Xbal, Xhol, Xmal, Xmnl. and Zral. Optionally, R'^ may comprise nucleotide spacers that precede or follow one or more of the endonuclease restriction site, the first region that is complementary to R^^ and/or the second region that is complementary to R'\ Suitable nucleotide spacers, for example, are detailed in formula (IV).<br>
(Ill) Methods for Selecting Epitope Binding Agents<br>
[0109]	A further aspect of the invention provides methods for selecting<br>
epitope-binding agents, and in particular aptamers for use in making any of the molecular biosensors of the present invention. Generally speaking, epitope binding agents comprising aptamers, antibodies, peptides, modified nucleic acids, nucleic acid mimics, or double stranded DNA may be purchased if commercially available or may be made in accordance with methods generally known in the art.<br>
[0110]	For example, in vitro methods of selecting peptide epitope<br>
binding agents include phage display (Ozawa et al., J. Vet. Med. Sci. 67(12):1237-41, 2005), yeast display (Boderetal., Nat. Biotech. 15:553-57, 1997), ribosome display {Hanes et al., PNAS 94:4937-42, 1997; Lipovsek et al., J. Imm. Methods, 290:51-67, 2004), bacterial display (Francisco et al., PNAS 90:10444-48. 1993; Georgiou et al.. Nat. Biotech. 15:29-34,1997), mRNA display (Roberts et al., PNAS 94:12297-302, 1997; Keefe et al.. Nature 410:715-18, 2001), and protein scaffold libraries (Hosseetal., Protein Science 15:14-27,2006). In one embodiment, the peptide epitope binding agents are selected by phage display. In another embodiment, the peptide epitope binding agents are selected by yeast display, In yet another embodiment, the peptide epitope binding agents are selected via ribosome display. In still yet another embodiment, the peptide epitope binding agents are selected via bacterial display. In an alternative embodiment, the peptide epitope binding agents are selected by mRNA display. In another alternative embodiment, the peptide epitope binding agents are selected using protein scaffold libraries.<br>
[0111]	The invention, however, provides methods for simultaneously<br>
selecting two or more aptamers that each recognize distinct epitopes on a target molecule or on separate target molecules. Alternatively, the invention also provides novel methods directed to selecting at least one aptamer in the presence of an<br>
38<br>
 <br>
epitope binding agent construct. The aptamer and epitope binding agent construct also each recognize distinct epitopes on a target molecule.<br>
(a) method for selection of an aptamer in the presence of an epitope binding agent construct<br>
[0112]	One aspect of the invention encompasses a method for<br>
selecting an aptamer in the presence of an epitope binding agent construct. The aptamer and epitope binding agent construct are selected so that ttiey each bind to the same target at two distinct epitopes. Typically, the method comprises contacting a plurality of nucleic acid constmcts and epitope binding agent constructs with a target molecule to form a mixture. The mixture will generally comprise complexes having target molecule bound with nucleic acid constructs and epitope binding agent constructs. According to the method, the complex is isolated from the mixture and the nucleic acid construct is purified from the complex. The aptamer selected by the method of tl^e invention will comprise the purified nucleic acid construct.<br>
[0113]	In this method of selection, a plurality of nucleic acid constaicts<br>
is utilized in the presence of the epitope binding agent construct to facilitate aptamer selection. The nucleic acid constructs comprise:<br>
A-B-C-D<br>
[0114]	The epitope binding agent construct comprises:<br>
P-Q-R wherein:<br>
A and C are each different DNA sequences from about 10 to about 30 nucleotides in length, A and C together comprising a' sequence to prime a polymerase chain reaction for amplifying the aptamer sequence;<br>
B is a single-stranded nucleotide of random sequence from about 20 to about 110 nucleotides in length that contains specific sequences binding to a first epitope of the target molecule;<br>
D and R are a pair of complementary nucleotide sequences from about 2 to about 20 nucleotides in length, wherein D and R have a free energy for association from about 5.5 kcal/mole to about 8.0<br>
39<br>
 <br>
kcal/mole at a temperature from approximately 21° C to about 40° C and at a salt concentration of approximately 1 mM to about 100 mM;<br>
P is an epitope-binding agent that binds to a second epitope on the target molecule. The epitope binding agent will vary depending upon the embodiment, but is selected from the group comprising an aptamer, an antibody, an antibody fragment, a double-stranded DNA sequence, modified nucleic acids, nucleic acid mimics, a ligand, a ligand fragment, a receptor, a receptor fragment, a polypeptide, a peptide, a coenzyme, a coregulator, an allosteric molecule, and an ion; and<br>
Q is a flexible linker.<br>
[0115]	Generally speaking, A and C are each different DNA sequences<br>
ranging from about 7 to about 35 nucleotides in length and function as polymerase chain reaction primers to amplify the nucleic acid construct. In another embodiment. A and C range from about 15 to about 25 nucleotides in length. In yet another embodiment, A and C range from about 15 to about 20 nucleotides in length. In still another embodiment, A and C range from about 16 to about 18 nucleotides in length. In an exemplary embodiment, A and C are 18 nucleotides in length. Typically, A and C have an average GC content from about 53% to 63%. In another embodiment, A and C have an average GC content from about 55% to about 60%. In a preferred embodiment, A and C will have an average GC content of about 60%.<br>
[0116]	B is typically a single-stranded nligonunlRotido synthesized by<br>
randomly selecting and inserting a nucleotide base (A, C, T, G in the case of DNA, or A, C, U, G in the case of RNA) at every position of the oligonucleotide. In one embodiment, B encodes an aptamer sequence that binds to the first epitope on the target. In another embodiment B is comprised of DNA bases. In yet another embodiment, B is comprised of RNA bases. In another embodiment, B is comprised of modified nucleic acid bases, such as modified DNA bases or modified RNA bases. Modifications may occur at, but are not restricted to, the sugar 2' position, the C-5 position of pyrimidines, and the 8-position of purines. Examples of suitable modified DNA or RNA bases include 2'-fluoro nucleotides, 2'-amino nucleotides, 5'-aminoailyl-2'-fluoro nucleotides and phosphorothioate nucleotides (monothiophosphate and dithiophosphate). In a further embodiment, B is about 20 to 110 nucleotides In<br>
40<br>
 <br>
length. In another embodiment, B is from about 25 to about 75 nucleotides in length. In yel another embodiment, B is from about 30 to about 60 nucleotides in length.<br>
[0117]	In one embodiment, D and R are complementary nucleotide<br>
sequences from about 2 to about 20 nucleotides in length. In another embodiment, D and R are from about 4 to about 15 nucleotides in length. In a preferred embodiment, D and R are from about 5 to about 7 nucleotides in length. In one embodiment, D and R have a free energy for association from about 5.2 kcal/mole to about 8.2 kcal/mole as measured in the selection buffer conditions, defined below. In another embodiment, D and R have a free energy for association from about 6.0 kcal/mole to about 8.0 kcal/mole as measured in the selection buffer conditions defined below. In yet another embodiment, D and R have a free energy for association from about 7.0 kcal/mole to 8.0 kcal/mole in the selection buffer conditions. In a preferred embodiment, D and R have a free energy for association of 7.5 kcal/mole in the selection buffer conditions described below.<br>
[0118]	Q may be a nucleotide sequence from about 10 to about 100<br>
nucleotides in length. In one embodiment, Q is from 10 to about 25 nucleotides in length. In another embodiment, Q is from about 25 to about 50 nucleotides in length. In a further embodiment, Q is from about 50 to about 75 nucleotides in length. In yet another embodiment, Q is from about 75 to about 100 nucleotides in length. In each embodiment, the nucleotides may be any of the nucleotide bases in DNA or RNA (A. C, T, G in the case of DNA, or A, C, U, G in the case of RNA). In one embodiment Q is comprised of DNA bases, tn another embodiment, Q is comprised of RNA bases. In yet another embodiment, Q is comprised of modified nucleic acid bases, such as modified DNA bases or modified RNA bases. Modifications may occur at, but are not restricted to, the sugar 2' position, the C-5 position of pyrimidines, and the 8-position of purines. Examples of suitable modified DNA or RNA bases include 2'-fluoro nucleotides, 2'-amino nucleotides, 5'-aminoallyl-2'-fluoro nucleotides and phosphorothioate nucleotides {monothiophosphate and dithiophosphate). In a further embodiment, R^ and R^ may be nucleotide mimics. Examples of nucleotide mimics include locked nucleic acids (LNA), peptide nucleic acids (PNA), and phosphorodiamidate morpholino oligomers (PMO). Alternatively, Q may be a polymer of bifunclional chemical linkers. In one embodiment the bifunctional chemical linker is heterobifunctional. Suitable heterobifunctional chemical linkers<br>
41<br>
 <br>
include sulfoSMCC (Sulfosuccinimidyl-4-(N-maleimidomelhyl)cyc(ohexane-1 -oarboxylate), and lc-SPDP( N-SuGcinimidyt-6-(3'-{2-PyridylDithio)-PropiDnamido)-hexanoate). In another embodiment the bifunctional chemical linker is homobifunclional. Suitable homobifunctional linkers include disuccinimidyl suberate, disuccinimidyl glutarate, and disuccinimidyl tartrate. Additional suitable linkers are illustrated in the Examples, such as the phosphoramidate form of Spacer 18 comprised of polyethylene glycol. In one embodiment, Q is from 0 to about 500 angstroms in length. In another embodiment, Q is from about 20 to about 400 angstroms in length. In yet another embodiment, Q is from about 50 to about 250 angstroms in length.<br>
[0119]	In a preferred embodiment, A and C are approximately 18<br>
nucleotides in length and have an average GC content of about 60%; B is about 30 to about 60 nucleotides in length; Q is a linker comprising a nucleotide sequence that is from about 10 to 100 nucleotides in length or a bifunctional chemical linker; and D and R range from about 5 to about 7 nucleotides in length and have a free energy of association of about 7.5 kcal/mole.<br>
[0120]	As will be appreciated by those of skill in the art, the choice of<br>
epitope binding agent, P, can and will vary depending upon the particular target molecule. By way of example, when the target molecule is a protein P may be an aptamer, or antibody. By way of further example, when P Is double stranded nucleic add the target molecule is typically a macromolecule that binds to DNA or a DNA binding protein. Suitable epitope binding agents, depending upon the target molecule, include agents selected from the group consisting of an aptamer, an antibody, an antibody fragment, a double-stranded DNA sequence, modified nucleic acids, nucleic acid mimics, a ligand, a liggnd fragment, a receptor, a receptor fragment, a polypeptide, a peptide, a coenzyme, a coregulator, an allosteric molecule, and an ion. In an exemplary embodiment, P is an aptamer sequence ranging in length from about 20 to about 110 bases. In another embodiment, P is an antibody selected from the group consisting of polyclonal antibodies, ascites. Fab fragments. Fab' fragments, monoclonal antibodies, and humanized antibodies. In a preferred embodiment, P is a monoclonal antibody. In an additional embodiment, P is a double stranded DNA. In yet another embodiment, P is a peptide.<br>
42<br>
 <br>
[0121]	Typically in the method, a plurality of nucleic acid constructs, A-<br>
B-C-D. are contacted with the epitope bind agent construct, P-Q-R, and the target molecular in the presence of a selection buffer to form a mixture. Several selection buffers are suitable for use in the invention. A suitable selection buffer is typically one that facilitates non-covalent binding of the nucleic acid construct to the target molecule in the presence of the epitope binding agent construct. In one embodiment, the selection buffer is a salt buffer with salt concentrations from about ImM to lOOmM. In another embodiment, the selection buffer is comprised of Tris-HCI, NaCI, KCI, and MgCb- In a preferred embodiment, the selection buffer is comprised of 50 mM Tris-HCI, 100 mM NaCI, 5 mM KCI, and 1 mM MgCb- In one embodiment, the selection buffer has a pH range from about 6.5 to about 8.5. In another embodiment, the selection buffer has a pH range from about 7.0 to 8.0. In a preferred embodiment, the pH is 7.5. Alternatively, the selection buffer may additionally contain analytes that assist binding of the constructs to the target molecule. Suitable examples of such analytes can include, but are not limited to, protein co-factors, DNA-binding proteins, scaffolding proteins, or divalent ions.<br>
[0122]	The mixture of the plurality of nucleic acid constructs, epitope-<br>
binding agent constnjcts and target molecules are incubated in selection buffer from about 10 to about 45 min. In yet another embodiment, the incubation is performed for about 15 to about 30 min. Typically, the incubation is performed at a temperature range from about 21" C to about 40° C.   In another embodiment, the incubation is performed at a temperature range from about 20" C to about 30" C.  In yet another embodiment, the incubation is performed at 35° C. In a preferred embodiment, the incubation is performed at 25" C for about 15 to about 30 min. Generally speaking after incubation, the mixture will typically comprise complexes of the target molecule having nucleic acid construct bound to a first epitope and epitope binding agent construct bound to a second epitope of the target molecule. The mixture will also comprise unbound nucleic acid constructs and epitope binding agent constructs.<br>
[0123]	The complex comprising the target molecule having bound<br>
nucleic acid construct and bound epitope binding agent construct is preferably isolated from the mixture. In one embodiment, nitrocellulose filters are used to separate the complex from the mixture. In an alternative embodiment magnetic beads are used to separate the complex from the mixture. In yet another<br>
43<br>
 <br>
embodiment sepharose beads can be used to separate the complex from the mixture. In an exemplary embodiment, streptavidin-linked magnetic beads are used to separate the complex from the mixture.<br>
[0124]	Optionally, the target molecules are subjected to denaturation<br>
and then the nucleic acid constructs purified from the complex. In one embodiment, urea is used to denature the target molecule. In a preferred embodiment, 7 M urea In 1M NaCI is used to denature the target molecule. The nucleic acid constructs may be purified from the target molecule by precipitation. In another embodiment, the nucleic acid constructs are precipitated with ethanol. In yet another embodiment, the nucleic acid constructs are precipitated with isopropanol. In one embodiment, the precipitated DNA is resuspended in water. Alternatively, the precipitated DNA is resuspended in TE buffer.<br>
[0125]	Generally speaking, the purified, resuspended nucleic acid<br>
constructs are then amplified using the polymerase chain reaction (PCR). If the nucleic acid construct contains a B comprised of RNA bases, reverse transcriptase is preferably used to convert the RNA bases to DNA bases before initiation of the PCR. The PCR is performed with primers that recognize both the 3' and the 5' end of the nucleic acid constructs in accordance with methods generally known in the art. In one embodiment, either the 3' or 5' primer is attached to a fluorescent probe. In an alternative embodiment, either the 3' or the 5' primer is attached to fluorescein. In another embodiment, either the 3' or 5' primer is biotinylated. In a preferred embodiment, one primer is labeled with fluorescein, and the other primer is biotinylated.<br>
[0126]	In addition to primers, the PCR reaction contains buffer,<br>
deoxynucleotide triphosphates, polymerase, and template nucleic acid. In one embodiment, the PCR can be performed with a heat-stable polymerase. In a preferred embodiment, the concentrations of PCR reactants are outlined in the examples section as follows; 80 pL of dd H20,10 pL of lOx PCR buffer, 6 pL of MgCb, 0.8 pL 25 mM dNTPs, 1 pL 50 ^JM primer 1(modified with fluorescein), 1 pL 50 pM primer 2 (biotinylated), 0.5 pL Taq polymerase, and 1 pL of template.<br>
[0127]	In another embodiment, the PCR consists of a warm-up period,<br>
where the temperature is held in a range between about 70° C and about 74" C. Subsequently, the PCR consists of several cycles (about 8 to about 25) of a)<br>
44<br>
 <br>
incubating the reaction at a temperature between about 92° C and about 97" C for about 20 sec to about 1 min; b) incubating the reaction at a temperature between about 48" C and about 56" C for about 20 sec to about 1 min; and c) incubating the reaction at a temperature between about 70" C and about 74° C for about 45 sec to about 2 min. After the final cycle, the PCR is concluded with incubation between about 70° C and about 74° C for about 3min to about 10 min. In an alternative embodiment, the reaction consists of 12-18 cycles. A preferred embodiment of the PCR, as outlined in the examples section, is as follows: 5 min at 95" C, sixteen cycles of 30s at 95° C, 30s at 50° C, and 1 min at 72" C, and then an extension period of 5 min at 72° C.<br>
[0128]	Typically after PCR amplification, the double-stranded DNA<br>
PCR product is separated from the remaining PCR reactants. One exemplary embodiment for such separation is subjecting the PCR product to agarose gel electrophoresis. In another embodiment, the PCR product is separated in a low melting point agarose gel. In a preferred embodiment, the gel is a native 10% acrytamide gel made in TBE buffer. In one embodiment, the band(s) having the double-stranded DNA PCR product are visualized in the gel by ethidium bromide staining. In another embodiment, the band(s) are visualized by fluorescein fluorescence. Irrespective of the embodiment, the bands are typically excised from the gel by methods generally known in the art.<br>
[0129]	Generally speaking, the double-stranded get-purified PCR<br>
product is separated into single-stranded DNA in accordance with methods generally known in the art. One such embodiment involves using a basic pH to denature the double helix. In another embodiment, 0.15N NaOH is used to denature the helix. In still another embodiment, streptavidin linked beads are used to separate the denatured DNA strands. In a preferred embodiment, magnetic streptavidin beads are used to separate the denatured DNA strands.<br>
[0130]	The method of the invention typically involves several rounds of<br>
selection, separation, amplification and purification in accordance with the procedures described above until nucleic acid constructs having the desired binding affinity for the target molecule are selected. In accordance with the method, the single-stranded DNA of estimated concentration is used for the next round of selection. In one embodiment, the cycle of selection, separation, amplification,<br>
45<br>
 <br>
purification, and strand separation is performed from about 4 to about 20 times. In another embodiment, the said cycle is performed from about 12 to about 18 tinges. In yet another embodiment, the said cycle is performed until the measured binding-activity of the selected nucleic acid constructs reaches the desired strength.<br>
[0131]	Alternatively, the single DNA strand attached to the streptavidin-<br>
linked beads is used as a template for RNA polymerase. In this embodiment, after the RNA polymerase is finished, the supernatant contains the RNA nucleic acid constmct that can be used in another round of RNA aptamer selection.<br>
[0132]	In an alternative method, if a RNA aptamer is being selected,<br>
the double-stranded, gel-purified PCR DNA product is transcribed vt/ith RNA polymerase to produce a single-stranded RNA construct. In such a case, A will typically contain a sequence encoding a promoter recognized by RNA polymerase. In one embodiment, double-stranded, gel-purified PCR DNA product attached to streptavidin-linked beads is used as a template for RNA polymerase. In this embodiment, after the RNA polymerase reaction, the supernatant containing the RNA nucleic acid construct can used in another round of RNA aptamer selection,<br>
[0133]	Generally speaking, after the nucleic acid constructs have<br>
reached the desired binding specificity, the nucleic acid constructs are cloned, and the cloned DNA is sequenced. In one embodiment, the sequences are used in aptamer constructs either alone or as part of a molecular biosensor.<br>
fb) method for simultaneous selection of two or more aptamers<br>
[0134]	Another aspect of the invention Is a method for simultaneously<br>
selecting two or more aptamers for use in making molecular biosensors having two or more aptamers. The aptamers selected by the method each bind to the same target molecule at distinct epitopes. Typically, the method comprises contacting a plurality of pairs of nucleic acid constructs with a target molecule to form a mixture. The mixture will generally comprise complexes having target molecule bound with a pair of nucleic acid constructs at distinct epitope sites. According to the method, the complex is isolated from the mixture and the nucleic acid constructs are purified from the complex. The aptamers selected by the method of the invention will comprise the pair of purified nucleic acid constructs.<br>
46<br>
 <br>
[0135]	In the method of the invention, the first nucleic acid constructs<br>
comprises:<br>
A-B-C-D The second nucleic acid construct comprises:<br>
E-F-G-H. wherein:<br>
A, C, E, and G are each different DNA sequences from about 10 to about 30 nucleotides in length, A and C together comprising a sequence to prime a polymerase chain reaction for amplifying a first aptamer sequence, and E and G together comprising a sequence to prime a polymerase chain reaction for amplifying a second aptamer sequence;<br>
B is a single-stranded nucleotide random sequence from about 20 to about 110 nucleotides in length that contains specific sequences binding to a first epitope of the target molecule;<br>
D and H are a pair of complementary nucleotide sequences from about 2 to about 20 nucleotides in length, wherein D and H have a free energy for association from about 5.5 kcal/mole to about 8.0 kcal/mole at a temperature from approximately 21" C to about 40° C and at a salt concentration of approximately 1 mM to about 100 mM; and<br>
F is a single-stranded nucleotide random sequence from about<br>
20 to about 110 nucleotides in length that contains specific sequences<br>
binding to the second epitope of the target molecule.<br>
[0136]	In another embodiment, A, C, E and G are each different DNA<br>
sequences ranging from about 7 to about 35 nucleotides in length. In another embodiment. A, C, E, and G range from about 15 to about 25 nucleotides in length. In yet another embodiment, A, C, E, and G range from about 15 to about 20 nucleotides in length. In still another embodiment. A, C, E and G range from about 16 to about 18 nucleotides in length. In an exemplary embodiment. A, C, E and G are 18 nucleotides in length. Generally speaking. A, C, E and G have an average GC content from about 53% to 63%. In another embodiment. A, C, E and G have an<br>
47<br>
 <br>
average GC content from about 55% to about 60%. In a preferred embodiment. A, C. E and G will have an average GC content of about 60%.<br>
[0137]	In one embodiment, B and F are single-stranded<br>
oligonucleotides synthesized by randomly selecting and inserting a nucleotide base (A, C, T, G in the case of DNA, or A, C, U, G in the case of RNA) at every position of the oligonucleotide. In a preferred embodiment, B and F encode an aptamer sequence, such that B binds to the first epitope on the target molecule and F binds to the second epitope on the target molecule. In one embodiment B and F are comprised of DNA bases. In another embodiment, B and F are comprised of RNA bases. In yet another embodiment, B and F are comprised of modified nucleic acid bases, such as modified DNA bases. Modifications may occur at, but are not restricted to, the sugar 2' position, the C-5 position of pyrimidines, and the 9-posttion of purines. Examples of suitable modified DNA or RNA bases include 2'-fluoro nucleotides, 2'-amino nucleotides, 5'-aminoallyl-2'-fluoro nucleotides and phosphorothioate nucleotides (monothiophosphate and dithiophosphate). In typical embodiments, B and F are about 20 to 110 nucleotides in length. In another embodiment, B and F are from about 25 to about 75 nucleotides in length. .In yet another embodiment, B and F are from about 30 to about 60 nucleotides in length.<br>
[0138]	D and H are complementary nucleotide sequences from about 2<br>
to about 20 nucleotides in length. In another embodiment, D and H are from about 4 to about 15 nucleotides in length. In a preferred embodiment, D and H are from about 5 to about 7 nucleotides in length. In one embodiment, D and H have a free energy for association from about 5.2 kcal/mole to about 8.2 kcal/mole as measured in the selection buffer conditions, defined below. In another embodiment, D and H have a free energy for association from about 6.0 kcal/mole to about 8.0 kcal/mole as measured in the selection buffer conditions. In yet another embodiment, D and H have a free energy for association from about 7.0 kcal/mole to 8.0 kcal/mole in the selection buffer conditions. In a preferred embodiment, D and H have a free energy for association of 7.5 kcal/mole in the selection buffer conditions.<br>
[0139]	In a preferred embodiment. A, C, E and G are approximately 18<br>
nucleotides in length and have an average GC content of about 60%, B and F are about 30 to about 60 nucleotides in length, and D and H range from about 5 to about 7 nucleotides in length and have a free energy of association of about 7,5 kcal/mole.<br>
48<br>
 <br>
[0140]	The method for simultaneous selection is initiated by contacting<br>
a plurality of pairs of the nucleic acid constructs A-B-C-D and E-F-G-H with the target molecule in the presence of a selection buffer to form a complex. Generally speaking, suitable selection buffers allow non-covalent simultaneous binding of the nucleic acid constructs to the target molecule. The method for simultaneous selection then involves the same steps of selection, separation, amplification and purification as described in section (a) above involving methods for the selection of an aptamer in the presence of an epitope binding agent construct, with the exception that the PCR is designed to amplify both nucleic acid constructs (A-B-C-D and E-F-G-H), using primers to A, C, E, and F.   Typically several rounds of selection are performed until pairs of nucleic acid constructs having the desired affinity for the target molecule are selected. In one embodiment, the cycle of selection, separation, amplification, purification, and strand separation is performed from about 4 to about 20 times. In another embodiment, the cycle is performed from about 12 to about 18 times. After the pair of nucleic acid constructs has reached the desired binding specificity, the nucleic acid constructs are cloned, and the cloned DNA is sequenced. The resulting nucleic acid constructs comprise a first aptamer that binds to a first epitope on the target molecule and a second aptamer that binds to a second epitope on the target molecule.<br>
[0141]	In another aspect of the invention, two aptamers can be<br>
simultaneously selected in the presence of a bridging construct comprised of S-T-U. In one embodiment, S and U are complementary nucleotide sequences from about 2 to about 20 nucleotides in length. In another embodiment, S and U are from about 4 to about 15 nucleotides in length. In a preferred embodiment, S and U are from about 5 to about 7 nucleotides in length. In one embodiment, S and U have a free energy for association from about 5.2 kcal/mole to about 8.2 kcal/mole as measured in the selection buffer conditions, defined below. In another embodiment, S and U have a free energy for association from about 6.0 kcal/mole to about 8.0 kcal/mole as measured in the selection buffer conditions. In yet another embodiment, S and U have a free energy for association from about 7.0 kcal/mole to 8.0 kcal/mole in the selection buffer conditions. In a preferred embodiment, S and U have a free energy for association of 7.5 kcal/mole in the selection buffer conditions.<br>
49<br>
 <br>
[0142]	T may be a nucleotide sequence from about 10 to about 100<br>
nucleotides in length. In one embodiment, T is from 10 to about 25 nucleotides in length. In another embodiment, T is from about 25 to about 50 nucleotides in length. In a further embodiment, T is from about 50 to about 75 nucleotides in length. In yet another embodiment, T is from about 75 to about 100 nucleotides in length. In each embodiment, the nucleotides may be any of the nucleotide bases in DNA or RNA (A, C, T, G in the case of DNA, or A, C, U, G in the case of RNA). In one embodiment T is comprised of DNA bases. In another embodiment, T is comprised of RNA bases. In yet another embodiment, T is comprised of modified nucleic acid bases, such as modified DNA bases or modified RNA bases. Modifications may occur at, but are not restricted to, the sugar 2' position, the C-5 position of pyrimidines, and the 8-position of purines. Examples of suitable modified DNA or RNA bases include 2'-fluoro nucleotides, 2'-amtno nucleotides, 5'-aminoallyl-2'-fluoro nucleotides and phosphorothioate nucleotides (monothiophosphate and dithiophosphate). In a further embodiment, R^ and R^ may be nucleotide mimics. Examples of nucleotide mimics include locked nucleic acids (LNA), peptide nucleic acids (PNA), and phosphorodiamidate morpholino oligomers (PMO). Alternatively, T may be a polymer of bifunctlonal chemical linkers. In one embodiment the bifunctional chemical linker is heterobifunctional. Suitable heterobifunotional chemical linkers include sulfoSMCC (Sulfosuccinimidyl-4-(N-maleimidomethyl)cyclohexane-1-carboxylate), and lc-SPDP( N-Succinimidyl-6-(3'-(2-PyridylDithlo)-Propionamido)-hexanonto). In another embodiment the bifunctional chemical linker is homobifunotional. Suitable homobifunctional tinkers Include dlBuccinimidyl suberate, disucclnimidyl glutarate, and disuccinlmidyl tartrate. Additional suitable linkers are illustrated in the Examples, such as the phosphoramidate form of Spacer 18 comprised of polyethylene glycol. In one embodiment, Q is from 0 to about 500 angstroms in length. In another embodiment, Q is from about 20 to about 400 angstroms in length. In yet another embodiment, Q is from about 50 to about 250 angstroms in length.<br>
[0143]	In one embodiment, S is complementary to D and U is<br>
complementary to H. In another embodiment, S and U will not bind to D and H unless 8, U, D, and H are brought in close proximity by the A-B-C-D constmct and the E-F-G-H construct binding to the target.<br>
50<br>
 <br>
[0144]	In this embodiment of the invention utilizing the bridging<br>
construct, the method is initiated in the presence of nucleic acid constructs A-B-C-D and E-F-G-H, and the bridging construct S-T-U. Generally speaking, the method is performed as described with the same steps detailed above. In one embodiment, after the final round of selection, but before cloning, the bridging construct is ligated to the A-B-C-D construct and the E-F-G-H construct. This embodiment allows the analysis of pairs of selected nucleic add sequences that are best suited for use in a molecular biosensor.<br>
(c)      selection of aptamers by in vitro evolution<br>
[0145]	A further aspect of the invention encompasses selection of<br>
aptamers by in vitro evolution in accordance with methods generally known in the art.<br>
[0146]	In another embodiment, the invention is directed to a method of<br>
making a set of aptamer constructs, comprising a first and second aptamer construct, comprising the steps of (a) selecting a first aptamer against a first substrate, which comprises a first epitope, and selecting a second aptamer against a second substrate, which comprises a second epitope, wherein the first aptamer is capable of binding to the first epitope and the second aptamer is capable of binding to the second epitope, (b) attaching a first label to the first aptamer and attaching a second label to the second aptamer, (c) attaching a first signaling oligo to the first aptamer and attaching a second signaling oligo to the second aptamer, wherein the second signaling oligo is complementary to the first signaling oligo, and (d) such that (i) the first aptamer construct comprises the first aptamer, the first label and, the first signaling oligo, and (ii) the second aptamer construct comprises the second aptamer, the second label and the second signaling oligo. Preferably, the aptamers are selected using in vitro evolution methods, however, natural DNA binding elements may be used in the practice of this invention.<br>
[0147]	In a preferred embodiment, the first substrate is a polypeptide<br>
and the second substrate is the polypeptide bound to the first aptamer, wherein the first aptamer masks the first epitope, such that the first epitope is not available for the second aptamer to bind. Alternatively, the first aptamer may be replaced by a first aptamer construct that contains (i) the first aptamer and signaling oligo, or (11) the first<br>
51<br>
 <br>
aptamer, signaling oligo and label, thereby producing a second substrate that allows for tho selection of the optimum second aptamer or aptamer construct for signal detection. As a further step, the first and second aptamer constructs may then be joined together by a flexible linker, as described above.<br>
[0148]	In an altemate preferred embodiment, the first substrate is a<br>
peptide consisting essentially of the first epitope and the second substrate is a peptide consisting essentially of the second epitope. Thus, in this alternate embodiment, there is no need to mask an epitope in the production or selection of aptamers. Again, the first and second aptamer constructs created by this method may be linked together by a flexible linker, as described above.<br>
(IV) methods utilizing the Molecular Biosensors<br>
[0149]	A further aspect of the invention encompasses the use of the<br>
molecular biosensors of the invention in several applications. In certain embodiments, the molecular biosensors are uUlized in methods for detecting one or more target molecules. In other embodiments, the molecular biosensors may be utilized in kits and for therapeutic and diagnostic applications.<br>
(a) detection methods<br>
[0150]	In one embodiment, the molecular biosensors may be utilised<br>
for detection of a target molecule. The method generally involves contacting a molecular biosensor of the invention with the target molecule. To detect a target molecule utilizing two-component biosensors, the method typically involves target-molecule induced co-association of two epitope-binding agents (present in the molecular biosensor of the invention) that each recognize distinct epitopes on the target molecule. The epitope-binding agents each comprise complementary signaling oligonucleotides that are labeled with detection means and are attached to the epitope-binding agents through a flexible linker. Co-association of the two epitope-binding agents with the target molecule results in bringing the two signaling oligonucleotides into proximity such that a detectable signal is produced. Typically, the detectable signal is produced by any of the detection means known in the art or as described herein. Altematively, for three-component biosensors, co-association of the two epitope-binding agent constructs with the target molecule results in<br>
52<br>
 <br>
hybridization of each signaling oligos to the oligonucleotide construct. Binding of the two signaling oligo to the oligonucleotide construct brings them into proximity such that a detectable signal is produced.<br>
[0151]	In one particular embodiment, a method for the detection of a<br>
target molecule that is a protein or polypeptide is provided. The method generally involves detecting a polypeptide in a sample comprising the steps of contacting a sample with a molecular biosensor of the invention. By way of non-limiting example, several useful molecular biosensors are illustrated in Figs. 24, 33 and 38.   Panel 24A depicts a molecular biosensor comprising two aptamers recognizing two distinct epitopes of a protein. Panel 248 depicts a molecular biosensor comprising a double stranded polynucleotide containing binding site for DNA binding protein and an aptamer recognizing a distinct epitope of the protein. Panel 24C depicts a molecular biosensor comprising an antibody and an aptamer recognizing distinct epitopes of the protein. Panel 24D depicts a molecular biosensor comprising a double stranded polynucleotide containing a binding site for a DNA binding protein and an antibody recognizing a distinct epitope of the protein. Panel 24E depicts a molecular biosensor comprising two antibodies recognizing two distinct epitopes of the protein. Panel 24F depicts a molecular biosensor comprising two double stranded polynucleotide fragments recognizing two distinct sites of the protein. Panel 24G depicts a molecular biosensor comprising two single stranded polynucleotide elements recognizing two distinct sequence elements of another single stranded polynucleotide. Panel 24H depicts a molecular biosensor that allows for the direct detection of formation of a protein-polynucleotide complex using a double stranded polynucleotide fragment {containing the binding site of the protein) labeled with a first signaling oligonucleotide and the protein labeled with a second signaling oligonucleotide. Panel 241 depicts a molecular biosensor that allows for the direct detection of the formation of a protein-protein complex using two corresponding proteins labeled with signaling oligonucleotides. Fig. 33 depicts a tri-valent biosensor that allows for detection of a target molecule or complex with three different epitope binding agents. Fig. 38 depicts a competitive biosensor that allows detection of a target competitor in a solution.<br>
[0152]	In another embodiment, the molecular biosensors may be used<br>
to detect a target molecule that is a macromolecular complex in a sample. In this<br>
53<br>
 <br>
embodiment, the first epitope is preferably on one polypeptide and the second epitope is on another polypeptide, such that when a macromolecular complex is formed, the one and another polypeptides are bought into proximity, resulting in the stable interaction of the first aptamer construct and the second aptamer cohstruct to produce a detectable signal, as described above.   Also, the first and second aptamer constructs may be fixed to a surface or to each other via a flexible linker, as described above.<br>
[0153]	In another embodiment, the molecular biosensors may be used<br>
to detect a target molecule that is an analyte in a sample. In this embodiment, when the analyte is bound to a polypeptide or macromolecular complex, a first or second epitope is created or made available to bind to a first or second aptamer construct. Thus, when an analyte is present in a sample that contains its cognate polypeptide or macromolecular binding partner, the first aptamer construct and the second aptamer construct are brought into stable proximity to produce a detectable signal, as described above. Also, the first and second aptamer constructs may be'fixed to a surface or to each other via a flexible linker, as described above.<br>
(b) solid surfaces<br>
[0154]	Optionally, the invention also encompasses a solid surface<br>
having the molecular constructs of the invention attached thereto. For example, in an embodiment for two-component biosensors, the first epitope binding agent construct may be fixed to a surface, the second epitope binding agent construct may be fixed to a surface, or both may be fixed to a surface. Non-limiting examples of suitable surfaces include microtitre plates, test tubes, beads, resins and other polymers, as well as other surfaces either known in the art or described herein. In a preferred embodiment, the first aptamer construct and the second aptamer construct may be joined with each other by a flexible linker to form a bivalent aptamer. Preferred flexible linkers include Spacer 18 polymers and deoxythymidine ("dT") polymers.<br>
[0155]	Referring to Figs. 54 and 56, in an exemplary embodiment the<br>
solid surface utilizes a three-component biosensor. In this embodiment, the oligonucleotide construct (e.g., O as described in (II), and S3 as described in the examples and figures) may be immobilized on a solid surface. Tfie first epitope<br>
54<br>
 <br>
binding agent and second epitope binding agent (e.g., SI and S2 in the figure) are contacfed with the surface comprising immobilized O and a sample that may comprise a target {e.g., T in figure). In the presence of target, the first epitope binding agent, second epitope binding agent, and target bind to immobilized O to form a complex. Several methods may be utilized to detect the presence of the complex comprising target. The method may include detecting a probe attached to the epitope-binding agents after washing out the unbound components. Alternatively, several surface specific real-time detection methods may be employed, including but not limited to surface plasmon resonance (SPR) or total internal reflection fluorescence (TIRF).<br>
[0156]	The oligonucleotide construct, O, may be immobilized to several<br>
types of suitable surfaces. The surface may be a material that may be modified to contain discrete individual sites appropriate for the attachment or association of the three-component biosensor and is amenable to at least one detection method. Non-limiting examples of surface materials include glass, modified or functionalized glass, plastics (including acrylics, polystyrene and copolymers of styrene and other materials, polypropylene, polyethylene, polybutylene, polyurethanes, TeflonJ, etc.), nylon or nitrocellulose, polysaccharides, nylon, resins, silica or silica-based materials including silicon and modified silicon, carbon, metals, inorganic glasses and plastics. The size and shape of the surface may also vary without departing from the scope of the invention. A surface may be planar, a surface may be a welt, i.e. a 364 well plate, or alternatively, a surface may be a bead or a slide.<br>
[0157]	The oligonucleotide construct, O, may be attached to the<br>
surface in a wide variety of ways, as will be appreciated by those in the art. O, for example, may either be synthesized first, with subsequent attachment to the surface, or may be directly synthesized on the surface. The surface and O may be derivatized with chemical functional groups for subsequent attachment of the two. For example, the surface may be derivatized with a chemical functional group including, but not limited to, amino groups, carboxyl groups, oxo groups or thiol groups. Using these functional groups, the O may be attached using functional groups either directly or indirectly using linkers. Alternatively, O may also be attached to the surface non-covalenlly. For example, a biotinylated O can be prepared, which may bind to surfaces covalently coated with streptavidin, resulting in attachment. Alternatively, O<br>
55<br>
 <br>
may be synthesized on the surface using techniques such as photopolymerization and photolithography. Additional methods of attaching O to a surface and methods of synthesizing O on surfaces are well known in the art, i.e. VLSIPS technology from Affymetrix (e.g., see U.S. Patent 6,566,495, and Rockett and Dix, "DNA arrays: technology, options and toxioological applications," Xenobiotica 30(2):155-177, all of which are hereby incorporated by reference in their entirety).<br>
(c) competition assays<br>
[0158]	In a further embodiment, a competitive molecular biosensor can<br>
be used to detect a competitor in a sample. Typically, the molecular biosensor used for competition assays will be a two-component molecular biosensor, as detailed in section (1) above. In an exemplary embodiment, the competitive molecular biosensor will comprise two epitope binding agents, which together have formula (Vt)<br>
p47_p48_f^49.p50. g^d R51.R".R53.R54.<br>
(VI)<br>
wherein:<br>
R^^ is an epitope-binding agent that binds to a first epitope on a target molecule;<br>
R'^^ is a flexible linker attaching R" to R"^; R'"' and R^^ are a pair of complementary nucleotide sequences having a free energy for association from about 5.5 kcal/moie to about 8.0 kcal/mole at a temperature from about 21 ° C to about 40° C and at a salt concentration from about 1 mM to about 100 mfVl;<br>
R^ and R" together comprise a detection means such that when R'^ and R^^ associate a detectable signal is produced; R^^ is an epitope binding agent that binds to R"; and R^^ is a flexible linker attaching R^^ to R".<br>
[0159]	In another alternative, the competitive molecular biosensor will<br>
comprise formula (VI) wherein:<br>
R"*^ is a peptide, a small molecule, or protein epitope-binding agent that binds to a first epitope on a target molecule; R"' is a flexible linker attaching R^^ to R"^;<br>
56<br>
 <br>
R"*^ and R^^ are a pair of complementary nucleotide sequences having a free energy for association from about 5.5 kcal/mole to about 8.0 kcal/mole at a temperature from about 21' C to about 40° C and at a salt concentration from about 1 mM to about 100 mM;<br>
R™ and R^ together comprise a detection means such that when R'^ and R^^ associate a detectable signal Is produced;<br>
R^^ is an antibody or antibody fragment epitope binding agent that binds to R"^; and<br>
R^^ is a flexible linker attaching R^^ to R^^<br>
[0160]	For each embodiment for competitive molecular biosensors<br>
having formula (VI), suitable flexible linkers, complementary nucleotide sequences, detection means, and epitope binding agents are described in section (1) for two-component molecular biosensors having formula (I).<br>
[0161]	To delect the presence of a target, referring to Figs. 38 and 39,<br>
the molecular biosensor is comprised of two epitope binding agents - the first epitope binding agent is a peptide that is a solvent exposed epitope of a target protein, and the second epitope binding agent is an antibody which binds to the first epitope binding agent. When the biosensor is in solufion without the target, a signal is created because the first epitope binding agent and the second epitope binding agent bind, thereby bringing the first signaling oligo and the second signaling oligo into close proximity, producing a detectable signal from the first and second label. When liie target competitive protein {comprising the solvent exposed epitope used for the first epitope binding agent) is added to the biosensor, the target protein competes with the first epitope binding agent for binding to the second epitope binding agent. This competition displaces the first epitope-binding agent from the second epitope binding agent, which destabilizes the first signaling oligo from the second signaling oligo, resulting in a decrease in signal. The decrease in signal can be used as a measurement of the concentration of the competifive target, as illustrated in example 5.<br>
(d) use of biosensors with no detection means<br>
[0162]	Alternatively, in certain embodiments it is contemplated that the<br>
molecular biosensor may not include a detections means. By way of example, when<br>
57<br>
 <br>
the molecular biosensor is a bivalent aptamer construct, the bivalent aptamer construct may not have labels for detection. It is envisioned that these alternative bivalent aptamer constructs may be used much like antibodies to detect molecules, bind molecules, purify molecules (as in a column or pull-down type of procedure), block molecular interactions, facilitate or stabilize molecular interactions, or confer passive immunity to an organism. It is further envisioned that the bivalent aptamer construct can be used for therapeutic purposes. This invention enables the skilled artisan to build several combinations of aptamers that recognize any two or more disparate epitopes form any number of molecules into a bivalent, trivalent, or other multivalent aptamer construct to pull together those disparate molecules to test the effect or to produce a desired therapeutic outcome. For example, a bivalent aptamer constnjct may be constructed to facilitate the binding of a ligand to its receptor in a situation wherein the natural binding kinetics of that ligand to the receptor is not favorable (e.g., insulin to insulin receptor in patients suffering diabetes.)<br>
(e)	kits<br>
[0163]	In another embodiment, the invention is directed to a kit<br>
comprising a first epitope binding agent, to which is attached a first label, and a second epitope binding agent, to which is attached a second label, wherein (a) when the first epitope binding agent and the second epitope binding agent bind to a first epitope of a polypeptide and a second epitope of the polypeptide, respectively, (b) the first Inhel and the second label interact to produce a detectable signal. In a preferred embodiment the epitope-binding agent is an aptamer construct, which comprises an aptamer, a label and a signaling oligo. However, the epitope-binding agent may be an antibody, antibody fragment, or peptide. The kit Is useful in the detection of polypeptides, analytes or macromolecular complexes, and as such, may be used in research or medical/veterinary diagnostics applications.<br>
(f)	diagnostics<br>
[0164]	In yet another embodiment, the invention is directed to a<br>
method of diagnosing a disease comprising the steps of (a) obtaining a sample from a patient, (b) contacting the sample with a first epitope binding agent construct and a second epitope binding agent construct, and (c) detecting the presence of a<br>
58<br>
 <br>
polypeptide, analyte or macromolecular complex in the sample using a detection method, wherein the presence of the polypeptide, analyte or macromolecular complex in the sample indicates whether a disease is present in the patient. In a one embodiment, (a) the first epitope binding agent construct is a first aptamer to which a first label and a first signaling oligo are attached, (b) the second epitope binding agent construct is a second aptamer to which a second label and a,second signaling oligo, which is complementary to the first signaling oligo, are attached, and<br>
(c)	the detection method is a fluorescence detection method, wherein, (d) when the first aptamer binds to the polypeptide and the second aptamer binds to the polypeptide, (e) the first signaling oligo and the second signaling oligo associate with each other, and (f) the first label is brought into proximity to the second label such that a change in fluorescence occurs. In another embodiment, (a) the first epitope binding agent construct is a first peptide to which a first label and a first signaling oligo are attached, (b) the second epitope binding agent construct Is a second peptide to which a second label and a second signaling oligo, which is complementary to the first signaling oligo, are attached, and (c) the detection method is a fluorescence detection method, wherein, (d) when the first aptamer binds to the polypeptide and the second aptamer binds to the polypeptide, (e) the first signaling oligo and the second signaling oligo associate with each other, and (f) the first label is brought into proximity to the second label such that a change in fluorescence occurs. In yet another embodiment, (a) the first epitope binding agent construct is a first antibody to which a first label and a first signaling oligo are attached, (b) the second epitope binding agent construct is a second antibody to which a second label and a second signaling oligo, which is complementary to the first signaling oligo, are attached, and (c) the detection method is a fluorescence detection method, wherein,<br>
(d)	when the first aptamer binds to the polypeptide and the second aptamer binds to the polypeptide, (e) the first signaling oiigo and the second signaling oligo associate with each other, and (f) the first label is brought into proximity to the second label such that a change in fluorescence occurs, In other embodiments, the first epitope binding agent and the second epitope-binding agents are different types of epitope binding agents (i.e. an antibody and a peptide, an aptamer and an antibody, etc.). Preferred samples include blood, urine, ascites, cells and tissue samples/biopsies. Preferred patients include humans, farm animals and companion animals.<br>
59<br>
0[0165]	In yet another embodiment, the invention is directed to a<br>
method of screening a sample for useful reagents comprising the steps of (a) contacting a sample with a first epitope binding agent construct and a second epitope binding agent construct, and (b) detecting the presence of a useful reagent in the sample using a detection method. Preferred reagents include a polypeptide, which comprises a first epitope and a second epitope, an analyte that binds to a polypeptide (in which case the method further comprises the step of adding the polypeptide to the screening mixture) and a potential therapeutic composition. In one embodiment, (a) the first epitope binding agent is a first aptamer to which a first label and a first signaling oligo are attached, (b) the second epitope binding agent is a second aptamer to which a second label and a second signaling oligo, which is complementary to the first signaling oligo, are attached, and (c) the detection method is a fluorescence detection method, wherein, (d) when the first aptamer binds to the polypeptide and the second aptamer binds to the polypeptide, (e) the first signaling oligo and the second signaling oligo associate with each other, and (f) the first label is brought into proximity to the second label such that a change in fluorescence occurs. In another embodiment, (a) the first epitope binding agent is a first peptide to which a first label and a first signaling oligo are attached, (b) the second epitope binding agent is a second peptide to which a second label and a second signaling oligo, which is complementary to the first signaling oligo, are attached, and (c) the detection method is a fluorescence detection method, wherein, (d) when the first aptamer binds to the polypeptide and the second aptamer binds to the polypeptide, (e) the first signaling oligo and the second signaling oligo associate with each other, and (f) the first label is brought into proximity to the second label such that a change in fluorescence occurs. In yet another embodiment, (a) the first epitope binding agent is a first antibody to which a first label and a first signaling oligo are attached, (b) the second epitope binding agent is a second antibody to which a second label and a second signaling oligo, which is complementary to the first signaling oligo, are attached, and (c) the detection method is a fluorescence detection method, wherein, (d) when the first aptamer binds to the polypeptide and the second aptamer binds to the polypeptide, (e) the first signaling oligo and the second signaling oligo associate with each other, and (f) the first label is brought into proximity to the second label such that a change in fluorescence occurs. In other embodiments, the first epitope<br>
60<br>
 <br>
binding agent and the second epitope-binding agents are different types of epitope binding agents (i.e. an antibody and a peptide, an aptamer and an antibody, etc.).<br>
DEFINITIONS<br>
[0166]	As used herein, the term "analyte" refers generally to a ligand,<br>
chemical moiety, compound, ion, salt, metal, enzyme, secondary messenger of a cellular signal transduction pathway, drug, nanoparticle, environmental contaminant, toxin, fatty acid, steroid, hormone, carbohydrate, amino acid, peptide, polypeptide, protein or other amino acid polymer, microbe, virus or any other agent which is capable of binding to a polypeptide, protein or macromolecular complex in such a way as to create an epitope or alter the availability of an epitope for binding to an aptamer,<br>
[0167]	The term "antibody" generally moans a polypeptide or protein<br>
that recognizes and can bind to an epitope of an antigen. An antibody, as used herein, may be a complete antibody as understood in the art, i.e., consisting of two heavy chains and two light chains, or be selected from a group comprising polyclonal antibodies, ascites, Fab fragments. Fab' fragments, monoclonal antibodies, chimeric antibodies, humanized antibodies, and a peptide comprising a hypervariabte region of an antibody.<br>
[0168]	The term "aptamer" refers to a polynucleotide, generally a RNA<br>
or a DNA that has a useful biological activity in terms of biochemical activity, molecular recognition or binding attributes. Usually, an aptamer has a molecular activity such as binding to a target molecule at a specific epitope (region). It is generally accepted that an aptamer, which is specific in its binding to any polypeptide, may be synthesized and/or identified by in vitro evolution methods.<br>
[0169]	As used herein, "detection method" means any of several<br>
methods known in the art to detect a molecular Interaction event. The phrase "detectable signal", as used herein, is essentially equivalent to "detection method." Detection methods include detecting changes in mass (e.g., plasmin resonance), changes in fluorescence (e.g., fluorescent resonance energy transfer (FRET), lanthamide resonance energy transfer (LRET), FCCS, fluorescence quenching or increasing fluorescence, fluorescence polarization, flow cytometry), enzymatic activity (e.g., depletion of substrate or formation of a product, such as a detectable<br>
61<br>
 <br>
dye - NBT'BCIP system of alkaline phosphatase is an example), changes in chemiluminescence or scintiJIalion (e.g., scintillation proximity assay, luminescence resonance energy transfer, bioluminescence resonance energy transfer and the like), and ground-state complex formation, excimer formation, colohmetric substance detection, phosphorescence, electro-chemical changes, and redox potential changes.<br>
[0170]	The term "epitope" refers generally to a particular region of a<br>
target molecule. Examples include an antigen, a hapten, a molecule, a polymer, a prion, a microbe, a cell, a peptide, polypeptide, protein, or macromolecular complex. An epitope may consist of a small peptide derived from a larger polypeptide. An epitope may be a two or three-dimensional surface or surface feature of a polypeptide, protein or macromolecular complex that comprises several non¬contiguous peptide stretches or amino acid groups.<br>
[0171]	The term "epitope binding agent" refers to a substance that is<br>
capable of binding to a specific epitope of an antigen, a polypeptide, a protein or a macromolecular complex. Non-limiting examples of epitope binding agents include aplamers, thioaptamers, double-stranded DNA sequence, peptides and polypeptides, ligands and fragments of ligands, receptors and fragments of receptors, antibodies and fragments of antibodies, polynucleotides, coenzyfnes, coregulators, allosteric molecules, peptide nucleic acids, locked nucleic acids, phosphorodiamidate morpholino oligomers (PMO) and ions. Peptide epitope binding agents include ligand regulated peptide epitope binding agents.<br>
[0172]	The term "epitope binding agent construct" refers to a construct<br>
that contains an epitope-binding agent and can serve in a "molecular biosensor" with another molecular biosensor. Preferably, an epitope binding agent construct also contains a "linker," and a "signaling oligo". Epitope binding agent constructs can be used to initiate the aptamer selection methods of the invention. A first epitope binding agent construct and a second epitope binding agent construct may be joined together by a "linker" to form a "bivalent epitope binding agent construct." An epitope binding agent construct can aiso be referred to as a molecular recognition construct. An aptamer construct is a special kind of epitope binding agent construct wherein the epitope binding agent is an aptamer.<br>
62<br>
 <br>
[0173]	The phrase "in vitro evolution" generally means any method of<br>
selecting for an aptamer tfiat binds to a biomolecule, particularly a peptide or polypeptide. In vitro evolution is also known as "in vitro selection", "SELEX" or "systematic evolution of ligands by exponential enrichment." Briefly, in vitro evolution involves screening a pool of random polynucleotides for a particular polynucleotide that binds to a biomolecule or has a particular activity that is selectable. Generally, the particular polynucleotide (i.e., aptamer) represents a very small fraction of the pool, therefore, a round of aptamer amplification, usually via polymerase chain reaction, is employed to increase the representation of potentially useful aptamers. Successive rounds of selection and amplification are employed to exponentially increase the abundance of the particular and useful aptamer. In vitro evolution is described in Famulok, M.; Szostak. J. W., In Vitro Selection of Specific Ligand Binding Nucleic Acids. Angew. Chem. 1992, 104, 1001. (Angew. Cham. Int. Ed, Engl. 1992. 31. 979-988.); Famulok. M.; Szostak. J. W.. Selection of Functional RNA and DNA Molecules from Randomized Sequences, Nucleic Acids and Molecular Biology. Vol 7, F. Eckstein. D. M. J. Lilley, Eds., Springer Verlag, Berlin, 1993. pp. 271; Klug, S.; Famulok. M.. All you wanted to know about SELEX; Mol. Biol. Reports 1994,20, 97-107; and Burgstaller. P.; Famulok, M. Synthetic ribozymes and the first deoxyribozyme; Angew. Chem. 1995, 107.1303-1306 (Angew. Chem. Int. fd. Engl. 1995. 34,1189-1192). which are incorporated herein by reference.<br>
[0174]	In the practice of certain embodiments of the invention, in vitro<br>
evolution is used to generate aptamers that bind to distinct epitopes of any given polypeptide or macromolecular complex. Aptamers are selected against "substrates", which contain the epitope of interest. As used herein, a "substrate" is any molecular entity that contains an epitope to which an aptamer can bind and that is useful in the selection of an aptamer.<br>
[0175]	The term "label", as used herein, refers to any substance<br>
attachable to a polynucleotide, polypeptide, aptamer, nucleic acid component, or other substrate material, in which the substance is detectable by a detection method. Non-limiting examples of labels applicable to this invention inctude but are not limited to luminescent molecules, chemilumlnescent molecules, fluorochromes, fluorescent quenching agents, colored molecules, radioisotopes, scintillants, massive labels (for detection via mass changes), biotin, avidin, streptavidin, protein A, protein G,<br>
63<br>
 <br>
antibodies or fragments thereof, Grb2, polyhistidine, Ni2+, Flag tags, myc tags, heavy metals, enzymes, alkaline phosphatase, peroxidase, luciferase. electron donors/acceptors, acridinium esters, and colorimetric substrateK. The skilled artisan would readily recognize other useful labels that are not mentioned above, which may be employed in the operation of the present invention.<br>
[0176]	As used herein, the term "macromolecular complex" refers to a<br>
composition of matter comprising a macromolecule. Preferably, these are complexes of one or more macromolecules, such as polypeptides, lipids, carbohydrates, nucleic acids, natural or artificial polymers and the like, in association with each other. The association may involve covalenl or non-covalent interactions between components of the macromolecular complex. Macromolecular complexes may be relatively simple, such as a ligand bound polypeptide, relatively complex, such as a lipid raft, or very complex, such as a cell surface, virus, bacteria, spore and the like. Macromolecular complexes may be biological or non-biological in nature.<br>
[0177]	The term "molecular biosensor" and "molecular beacon" are<br>
used interchangeably herein to refer to a construct comprised of at least two epitope binding agent constructs. The molecular biosensor can be used for detecting or quantifying the presence of a target molecule using a chemical-based system for detecting or quantifying the presence of an analyte, a prion, a protein, a nucleic acid, a lipid, a carbohydrate, a biomolecule, a macromolecular complex, a fungus, a microbial organism, or a macromolecular complex comprised of hiomoleculBS using a measurable read-out system as the detection method.<br>
[0178]	The phrase "natural cognate binding element sequence" refers<br>
to a nucleotide sequence that serves as a binding site for a nucleic acid binding factor. Preferably the natural cognate binding element sequence is a naturally occumng sequence that is recognized by a naturally occurring nucleotide binding factor.<br>
[0179]	The term "nucleic acid construct" refers to a molecule<br>
comprising a random nucleic acid sequence flanked by two primers. Preferably, a nucleic acid construct also contains a signaling oligo. Nucleic acid constructs are used to initiate the aptamer selection methods of the invention.<br>
61<br>
 <br>
[0180]	The term "signaling oligo" means a short (generally 2 to 15<br>
nucleotides, preferably 5 to 7 nucleotides in length) single-stranded polynucleotide. Signaling oligos are typically used in pairs comprising a first signaling oligo and a second signaling oligo. Preferably, the first signaling otigo sequence is complementary to the second signaling oligo. Preferably, the first signaling oligo and the second signaling oligo can not form a stable association with each other through hydrogen bonding unless the first and second signaling oligos are brought into close proximity to each other through the mediation of a third party agent.<br>
EXAMPLES<br>
[0181]	The following examples illustrate various iterations of the<br>
invention.<br>
Example 1: General Method for Preparing Specific Aptamer Constructs Introduction<br>
[0182]	Disclosed is a method for the rapid and sensitive detection of<br>
proteins, protein complexes, or analytes that bind to proteins. This method is based on the protein-driven association of two constructs containing aptamers that recognize two distinct epitopes of a protein (a.k.a. "aptamer constructs") (Fig. 1 A). These two aptamer constructs contain short complementary signaling oligonucleotides attached to the aptamers through a flexible linker.  Upon the simultaneous binding of the two aptamers to the target protein, the complementary oligonucleotides (a.k.a. "signaling oligos") are brought into relative proximity that promotes their association to form a stable duplex. Attaching fluorescence probes to the ends of the signaling oligos provides a means of detecting the protein-induced association of the two aptamers constructs (Fig. 1A). In the case of proteins that possess natural nucleic acid binding activity, one of the aptamers can be substituted with a nucleic acid sequence containing the DNA-binding sequence that the protein naturally binds to protein (Fig. IB).<br>
[0183]	Development or selection of aptamers directed to two distinct<br>
epitopes of a given protein is an essential step in developing the aptamer constructs depicted in Fig. 1. Review of the available literature on aptamers reveals at least two possible approaches to achieve this goal. The first approach is to perform in vitro<br>
65<br>
 <br>
selection (a.k.a. in vitro evolution) of nucleic acid aptamers using different methods for the separation of protein-bound and protein-unbound nucleic acid aptamers. The rationale here is that in these different partitioning methods different regions of the protein are preferentially displayed resulting in aptamers directed to different regions of the protein surface, Aptamers selected to thrombin {infra) are an example of such an approach.<br>
[0184]	The in vitro selection of a first aptamer using as a substrate<br>
thrombin immobilized on agarose beads resulted in an aptamer binding the thrombin at the heparin exosite. Additional in vitro selection using as a substrate the thrombin-first aptamer complex, which was bound to nitrocellulose as the partitioning method, resulted in a second aptamer binding the thrombin at the fibrinogen exosite.<br>
<br>
<br>
CLAIMS<br>
WHAT IS CLAIMED IS:<br>
1.	A three-component molecular biosensor, the molecular biosensor comprising<br>
two epitome binding agents and an oligonucleotide construct, which together<br>
have formula (III):<br>
wherein: <br>
R24 is an epitope binding agent that binds to a first epitome on a target molecule;<br>
R25 is a flexible linker attaching R^" to R^^;<br>
R26 and R^ are a pair of nucleotide sequences that are not complementary to each other, but are complementary to two distinct regions on O;<br>
R^^ and R^' together comprise a detection means such that when R^ and R^ associate a detectable signal is produced;<br>
R^° is an epitome binding agent that binds to a second epitome on the target molecule;<br>
R^^ is a flexible linker attaching R^" to R'"; and<br>
O is a nucleotide sequence comprising a first region that is complementary to R^^, and a second region that is complementary to<br>
2.	The three-component molecular biosensor of claim 1, wherein O comprises<br>
formula (IV);<br>
p 32_ p33_ □ 34. p35_ p36<br>
(IV)<br>
wherein:<br>
R^^, R^', and R^^ are nucleotide sequences not complementary<br>
to any of R^^ R^", R^^ or R^^;and<br>
R^^ is a nucleotide sequence complementary to R^, and R^^ is a nucleotide sequence that is complementary to R^.<br>
120<br>
 <br>
3.	The three-component molecular biosensor of claim 2, wherein R^^ R^, and R^^ are from about 2 to about 20 nucleotides in length; and R^^ and R^^ comprise a length such that the free energy of association between R^^ and R^^ and R^^ and R^" is from about -5 to about -12 kcal/mole at a temperature from about 21 ° C to about 40° C and at a salt concentration from about 1 mM to about 100 mM.<br>
4.	The three-component molecular biosensor of claim 3, wherein the target molecule is selected from the group consisting of an anaiyte, a prion, a protein, a polypeptide, a nucleic acid, a lipid, a carbohydrate, a macromoiecular complex, a fungus, and a microbial organism.<br>
5.	The three-component molecular biosensor of claim 4, wherein R^** and R^^ are independently selected from the group consisting of an aptamer, an antibody, an antibody fragment, a double-stranded DNA sequence, modified nucleic<br>
acids, nucleic acid mimics, a ligand, a ligand fragment, a receptor, a receptor fragment, a polypeptide, a peptide, a coenzyme, a coregulator, an allosteric molecule, and an ion.<br>
6.	The three-component molecular biosensor of claim 5, wherein R^ and R^^ are<br>
from about 50 to about 250 angstroms in length and are independently<br>
selected from the group consisting of a heterobifunctional chemical linker, a<br>
homobifunctional cheniical linker, polyethylene glycol, and nucleic acid.<br>
7.	The three-component molecular biosensor of claim 6, wherein R^ and R^" are from about 2 to about 20 nucleotides in length.<br>
8.	The three-component molecular biosensor of claim 7, wherein R^^ and R^' are independently selected from the group consisting of fluorescence resonance electron transfer (FRET), lanthamide resonance electron transfer (LRET), fluorescence cross-correlation spectroscopy, flourescence quenching, fluorescence polarization, flow cytometry, scintillation proximity, luminescence resonance energy transfer, direct quenching, ground-state complex formation, chemiluminescence energy transfer, bioluminescence resonance energy<br>
121<br>
 <br>
transfer, excimer formation, colorimetric substrates detection, phosphorescence, electro-chemical changes, and redox potential changes.<br>
9.	A composition, the composition comprising an oligonucleotide construct O<br>
immobilized to a solid surface and two epitope binding agents, the epitope<br>
binding agents comprising:<br>
wherein:<br>
R^"* is an epitope binding agent that binds to a first epitope on a target molecule;<br>
R^^ is a flexible linker attaching R^" to R^*^;<br>
R^^ and R^ are a pair of nucleotide sequences that are not complementary to each other, but are complementary to two distinct regions on O;<br>
R^^ and R^^ together comprise a detection means such that when R^ and R^ associate a detectable signal is produced;<br>
R^^ is an epitope binding agent that binds to a second epitope on the target molecule;<br>
R^ is a flexible linker attaching R^^ to R^; and<br>
O is a nucleotide sequence comprising a first region that is complementary to R^^, and a second region that is complementary to<br>
10.	The composition of claim 9, wherein O comprises formula (IV):<br>
p32_rD33.D34_p3S_p36<br>
{IV)<br>
wherein:<br>
R^^ R^", and R^^ are nucleotide sequences not complementary to<br>
anyofR^,R^°,R'^orR^^and<br>
R^^ is a nucleotide sequence complementary to R^^, and R^^ is a nucleotide sequence that is complementary to R^.<br>
122<br>
 <br>
11.	The composition of claim 10, wherein R^^, R^, and R^^ are from about 2 to about 20 nucleotides in length; and R^^ and R^^comprise a length such that the free energy of association between R^^ and R^ and R^^ and R^" is from about -5 to about -12 kcal/mole at a temperature from about 21° C to about 40" C and at a salt concentration from about 1 mM to about 100 mM.<br>
12.	The composition of claim 11, wherein the target molecule is selected from the group consisting of an analyte, a prion, a protein, a polypeptide, a nucleic add, a lipid, a carbohydrate, a macromolecular complex, a fungus, and a microbial organism.<br>
13.	The composition of claim 12, wherein R^"* and R^" are independently selected from the group consisting of an aptamer, an antibody, an antibody fragment, a double-stranded DNA sequence, modified nucleic acids, nucleic acid mimics, a ligand, a iigand fragment, a receptor, a receptor fragment, a polypeptide, a peptide, a coenzyme, a coregulator, an allosteric molecule, and an ion.<br>
14.	The composition of claim 13, wherein R^^ and R^^ are from about 50 to about 250 angstroms in length and are independently selected from the group consisting of a heterobifunctional chemical linker, a homobifunotional chemical linker, polyethylene glycol, and nucleic acid.<br>
15.	The composition of claim 14, wherein R^^ and R™ are from about 2 to about 20 nucleotides in length.<br>
16.	The composition of claim 15, wherein R" and R^^ are independently selected from the group consisting of fluorescence resonance electron transfer (FRET), lanthamide resonance electron transfer (LRET), fluorescence aoss-correlation spectroscopy, flourescence quenching, fluorescence polarization, flow cytometry, scintillation proximity, luminescence resonance energy transfer, direct quenching, ground-state complex formation, chemiluminescence energy transfer, bioluminescence resonance energy transfer, exclmer formation, colorimetric substrates detection, phosphorescence, electro-chemical changes, and redox potential changes,<br>
123<br>
 <br>
17.	The composition of claim 16, winerein the surface is a material selected from the group consisting of glass, functionalized glass, plastic, nylon, nitrocellulose, polysaccharides, resin, silica, metals, and inorganic glasses.<br>
18.	The composition of claim 16, wherein the surface is selected from the group consisting of a microtilre plate, a test tube, beads, resins, and a slide.<br>
19.	A method for detecting a target in a sample, the method comprising:<br>
(a)	contacting a surface Immobilized with an oligonulceotide construct, two<br>
epitope binding agents, and a sample, the epitope binding agents<br>
comprising:<br>
R24.R25_R26.f^27.<br>
wherein:<br>
R^"" is an epitope binding agent that binds to a first epitope on a target molecule;<br>
R^^ is a flexible linker attaching R^"* to R^^<br>
R^^ and R^° are a pair of nucleotide sequences that are not complementary to each other, but are complementary to two distinct regions on O;<br>
R^^ and R^^ together comprise a detection means such that when R^^ and R^" associate a detectable signal Is produced;<br>
R^^ is an epitope binding agent that binds tn a snnonti epitope on the target molecule;<br>
R^^ is a flexible linker attaching R^' to R^°\ and<br>
O is a nucleotide sequence comprising a first region that is complementary to R^^, and a second region that is complementary to R^°; and<br>
(b)	detecting whether R^^ and R^** bind to O, such binding indicating that<br>
the target is present in the sample.<br>
20.	The method of claim 19, wherein total internal reflection fluorescence is used<br>
to detect whether R^^ and R^" bind to O.<br>
124<br>
 <br>
21.	A molecular biosensor, the molecular biosensor comprising two epitope<br>
binding agents, which together have formula (VI)<br>
p47_R48.R49_f^50. ^^^ p51,p52,R53.R54.<br>
(VI)<br>
wherein:<br>
R"^ is an epitope-binding agent that binds to a first epitope on a target molecule;<br>
R"^ is a flexible linker attaching R"' to R"^; R"*^ and R^^ are a pair of complementary nucleotide sequences having a free energy for association from about 5.5 kcal/mole to about 8.0 kcal/mole at a temperature from about 21° C to about 40° C and at a salt concentration from about 1 mM to about 100 mM;<br>
R^ and R^ together comprise a detection means such that when R'^ and R^^ associate a detectable signal is produced; R^^ is an epitope binding agent that binds to R''; and R^^ is a flexible linker attaching R^^ to R^l<br>
22.	The molecular biosensor of claim 21, wherein R'"' is selected from the group consisting of a peptide, a small molecule, and a protein, and R^^ is selected from the gnDup consisting of an antibody, an antibody fragment, and an aplamer.<br>
23.	The molecular biosensor of claim 22, wherein R^"" is an antibody or antibody fragment selected from the group consisting of polyclonal antibodies, ascites. Fab fragments, Fab' fragments, monoclonal antibodies and humanized antibodies.<br>
24.	The molecular biosensor of claim 22, wherein R"^ and R^^ are from about 50 to about 250 angstroms in length and are selected independently selected from the group consisting of a heterobifunctional chemical linker, a homobifunctionat chemical linker, polyethylene glycol, and nucleic acid.<br>
125<br>
 <br>
25.	The molecular biosensor of claim 22, wherein R"" and R" are from about 2 to about 20 nucleotides in length.<br>
26.	The molecular biosensor of claim 22, wherein R^" and R^"* are independently selected from the group consisting of fluorescence resonance electron transfer (FRET), lanthamide resonance electron transfer (LRET), fluorescence cross-correlation spectroscopy, fluorescence quenching, fluorescence polarization, flow cytometry, scintillation proximity, luminescence resonance energy transfer, direct quenching, ground-state complex fonnation, chemiluminescence energy transfer, bioluminescence resonance energy transfer, excimerfomnation, colorimetric substrates detection, phosphorescence, electro-chemical changes, and redox potential changes.<br>
27.	A molecular biosensor, the molecular biosensor comprising two epitope binding agents, which together have formula (VI)<br>
pj47.R4a.pj49_R50. 3^^<br>
(VI) wherein:<br>
R'^ is a peptide or protein epitope-binding agent thai binds to a first epitope on a target molecule;<br>
R*^ is a flexible linker attaching R**^ to R'^ R"*^ and R^^ are a pair of complementary nucleotide sequences having a free energy for association from about 5.5 kcal/mole to about 8.0 l^cal/mole at a temperature from about 21° G to about 40'. C and at a salt concentration from about 1 mM to about 100 mM;<br>
R^ and R^ together comprise a detection means such that when R**^ and R^^ associate a detectable signal is produced;<br>
R^^ is an antibody or antibody fragment epitope binding agent that binds to R"^; and<br>
R^^ is a flexible linl<er attaching r to></er>
28.	The molecular biosensor of claim 27, wherein R^^ is an antibody or antibody<br>
fragment selected from the group consisting of polyclonal antibodies, ascites,<br>
126<br>
 <br>
Fab fragments, Fab' fragments, monoclonal antibodies and humanized antibodies.<br>
29.	The molecular biosensor of claim 28, wherein R*^ and R^^ are from about 50 to about 250 angstroms in length and are selected independently selected from the group consisting of a heterobifunctional chemical linker, a homobifunctional chemical linker, polyethylene glycol, and nucleic acid.<br>
30.	The molecular biosensor of claim 29, wherein R"^ and R^^ are from about 2 to about 20 nucleotides in length.<br>
31.	The molecular biosensor of claim 30, wherein R^° and R"^ are independently selected from the group consisting of fluorescence resonance electron transfer (FRET), lanthamide resonance electron transfer (LRET), fluorescence cross-correlation spectroscopy, fluorescence quenching, fluorescence polarization, flow cytometry, scintillation proximity, luminescence resonance energy transfer, direct quenching, ground-state complex formation,<br>
chemiluminescence energy transfer, bioluminescence resonance energy transfer, excimerfonnation. colorimefric substrates detection, phosphorescence, electro-chemical changes, and redox potential changes.<br>
32.	A method for determining the presence of a target molecule in a sample, the<br>
method comprising:<br>
a) measuring the signal of a molecular biosensor without the target molecule being present, the molecular biosensor comprising two epitope binding agents, which together have formula (VI)<br>
q51  D52_a53_Q54,<br>
wherein:<br>
R'"' is an epitope-binding agent that binds to a first epitope<br>
on a target molecule;<br>
R"*^ is a flexible linker attaching R"^ to R'^<br>
R""^ and R^^ are a pair of complementary nucleotide<br>
sequences having a free energy for association from about 5.5<br>
127<br>
 <br>
kcal/mole to about 8.0 kcal/mole at a temperature from about 21° C<br>
to about 40° C and at a salt concentration from about 1 mM to<br>
about 100 mM;<br>
R^" and R^ together comprise a detection means such that<br>
when R"^ and R^^ associate a detectable signal is produced; R^^ is an epitope binding agent that binds to R"*^; and R^^ is a flexible linker attaching R'^ to R^^<br>
b)	combining the molecular biosensor with the sample; and<br>
c)	measuring the signal of the biosensor, wherein a decrease in signal indicates the presence of a target molecule.<br>
<br>
33.	The method of claim 32, wherein R"*^ is selected from the group consisting of a peptide, small molecule, and a protein, and R^' is selected from the group consisting of an antibody, an antibody fragment, and an aptamer.<br>
34.	The method of claim 33, wherein R^' is an antibody or antibody fragment selected from the group consisting of polyclonal antibodies, ascites. Fab fragments, Fab' fragments, monoclonal antibodies and humanized antibodies.<br>
35.	The method of claim 34, wherein R"" and R^^ are from about 50 to about 250 angstroms in length and are selected independently selected from the group consisting of a heterobifunclionai chemical tinker, a homobifunotional<br>
chemical linker, polyethylene glyr,ol, and nucleic acid.<br>
36.	The method of claim 35, wherein R^** and R^^ are from about 2 to about 20 nucleotides in length.<br>
37.	The method of claim 36, wherein R^ and R^ are independently selected from the group consisting of fluorescence resonance electron transfer (FRET), lanthamide resonance electron transfer (LRET), fluorescence cross-correlation spectroscopy, fluorescence quenching, fluorescence polarization, flow cytometry, scintillation proximity, luminescence resonance energy transfer, direct quenching, ground-state complex formation, chemiluminescence energy transfer, bioluminescence resonance energy<br>
128<br>
 <br>
transfer, excimer formation, colohmetric substrates detection, phosphorescence, electro-chemical changes, and redox potential changes.<br>
38.	The method of claim 32, wherein the concentration of the target molecule is determined.<br>
39.	The method of claim 32, wherein the target molecule is a macromolecule selected from the group consisting of a protein, a polypeptide, a prion, a nucleic acid, a lipid, and a carbohydrate.<br>
40.	The method of claim 33, wherein the target molecule is selected from a<br>
kinase, a binding protein, and an antigen.<br>
 <br>
I<br>
129<br>
<br>
<br>
</td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/viewdoc.aspx?id=I3j3vj01iWnqt8lZ5EBoaw==&amp;amp;loc=egcICQiyoj82NGgGrC5ChA==" target="_blank" style="word-wrap:break-word;">http://ipindiaonline.gov.in/patentsearch/GrantedSearch/viewdoc.aspx?id=I3j3vj01iWnqt8lZ5EBoaw==&amp;amp;loc=egcICQiyoj82NGgGrC5ChA==</a></p>
		<br>
		<div class="pull-left">
			<a href="272372-securing-device.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="272374-method-for-preventing-cheating-in-network-game.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>272373</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>1337/CHENP/2009</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>14/2016</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>01-Apr-2016</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>30-Mar-2016</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>09-Mar-2009</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>SAINT LOUIS UNIVERSITY</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td>221 N. GRAND BOULEVARD. ST. LOUIS MO 63103</td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>TIAN, LING,</td>
											<td>3209 JANUARY AVENUE, APT. 1 ST. LOUIS, MO 63139</td>
										</tr>
										<tr>
											<td>2</td>
											<td>HEYDUK, TOMASZ,</td>
											<td>2510 JOHNSON PLACE DRIVE, BALLWIN, MO 63021</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>C12Q 1/68</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>PCT/US07/75560</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td>2007-08-09</td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td>60/821,876</td>
									<td>2006-08-09</td>
								    <td>U.S.A.</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/272373-molecular-biosensors-for-detecting-macromolecules-and-other-analytes by HTTrack Website Copier/3.x [XR&CO'2014], Thu, 04 Apr 2024 23:07:40 GMT -->
</html>
