<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/272567-recombinant-proteins-and-production-by-autoproteolytic-cleavage-of-a-fusion-protein by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 00:08:17 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 272567:&quot;Recombinant Proteins and Production by Autoproteolytic Cleavage of a Fusion Protein&quot;</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">&quot;Recombinant Proteins and Production by Autoproteolytic Cleavage of a Fusion Protein&quot;</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>The invention relates to a process for the recombinant production of a heterologous polypeptide of interest, Ccompris-ing, (i) cultivation of a bacteria] host cell which is transformed with an expression vector which comprises a nucleic acid molecule which codes for a fusion polypeptide, the fusion polypeptide comprising a derivative of an autoprotease N of Pestivirus, wherein at least one cysteine residue of the naturally occuring autoprotease N™ of Pestivirus is replaced by another amino acid residue, and a second polypeptide which is connected to the first polypeptide at the C-terminus of the first polypeptide in a manner such, that the second polypeptide is capable of being cleaved from the fusion polypeptide by the autoproteolytic activity of the first polypeptide, said second polypeptide being a heterologous polypeptide, wherein cultivation occurs under conditions which cause expression of the fusion polypeptide and formation of corresponding cytoplasmic inclusion bodies, (ii) isolation of the inclusion bodies from the host cell, (iii) solubilization of the isolated inclusion bodies, (iv) induction of autoproteolytic cleavage of the heterologous polypeptide of interest from the fusion polypeptide, and (v) isolation of the cleaved heterologous polypeptide of interest.</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td>Organic Compounds<br>
Field of Invention<br>
The present invention relates to a process for the recombinant production of a desired heterologous polypeptide of interest with a clearly defined homogeneous N-terminus in a bacterial host cell, wherein initially a fusion polypeptide which comprises a derivative of the autoprotease N1*0 of Pestivirus and the heterologous polypeptide of interest is provided by expression in a host cell. The heterologous polypeptide of interest is produced in the host cell in form of cytoplasmic inclusion bodies, which are then isolated and treated in such a way, that the desired heterologous polypeptide is cleaved from the fusion polypeptide by the NPRO autoproteolytic activity.<br>
Background of invention<br>
In the production of recombinant proteins in heterologous organisms such as the expression of human or other eukaryotic proteins in bacterial cells it is often difficult to obtain a clearly defined N-terminus which is as nearly 100% homogeneous as possible. This applies in particular to recombinant pharmaceutical proteins whose amino acid sequence in many cases ought to be identical to the amino acid sequence naturally occurring in humans/animals.<br>
On natural expression, for example in humans, many pharmaceutical proteins which are in use for therapy as well are transported into the extracellular space. A signal sequence is present in the precursor protein for this purpose and cleavage of this signal sequence results in a clearly defined N-terminus. For several reasons such homogeneous N-termini are not always easy to produce, for example in bacterial cells.<br>
For production on industrial scale, many pharmaceutical proteins are produced in the cytoplasm of bacterial cells (for example Escherichia coll). In these host cells the pharmaceutical proteins are accumulated in adequate quantities and are often deposited as insoluble inclusion bodies (IBs) inside the cell. These IBs have great advantages in working up and purification of the target protein. In addition, the protein expressed in the form of IBs is protected from protease degradation by intracellular proteases.<br>
As used herein the term "inclusion bodies" shall refer to aggregates containing heterologous polypeptides present in the cytoplasm of transformed host cells. These appear as bright spots under the microscope and can be recovered by separation of the cytoplasm.<br>
However, production of IB material requires in vitro refolding of the expressed protein. This can in many cases be effected by methods known per se.<br>
Only in rare cases is export of the target protein into the bacterial periplasm with the aid of a pro- or eukaryotic signal sequence suitable. Because of the low transport capacity of the bacterial export machinery it is usually only possible to accumulate very small quantities of product here.<br>
However, the bacterial cytoplasm differs considerably from the extracellular space of eukaryotes. One difference is that within the bacterial cytoplasm reducing conditions are predominant; also a mechanism for cleaving N-terminal leader sequences to form mature proteins is lacking. Synthesis of all cytoplasmic proteins starts with a methionine which is specified by the appropriate start codon (ATG = initiation of translation). This N-terminal methionine is retained in many proteins, while in others it is cleaved by the rrjethionine aminojaeptidase (MAP) present in the cytoplasm and intrinsic to the host The efficiency of the cleavage depends essentially on two parameters: 1. the nature of the following amino acid, and 2. the location of the N-terminus in the three-dimensional structure of the protein. The N-terminal methionine is preferentially deleted when the following amino add is serine, alanine, glycine, methionine or valine and when the N-terminus is exposed, i.e. not "hidden" inside the protein, if the following amino acid is a different one, in particular a charged one (glutamic acid, aspartic acid, lysine, arginine), or if the N-terminus is located inside the protein, in most cases cleavage of the N-terminal methionine does not occur.<br>
Even if an amino acid that promotes cleavage is present at position 2, the cleavage is rarely complete. It is usual for a not inconsiderable proportion (1-50%) of the target protein to remain unaffected by the MAP.<br>
This in-homogeneity or deviation from the natural sequence is, however, unacceptable in many cases because these products frequently show different immunological (for example induction of antibody formation) and pharmacological (half-life, pharmacokinetics) properties. For these reasons, it is necessary in most cases to produce a nature-identical product (homogeneous and without foreign amino acids at the N-terminus). In the case of cytoplasmic expression, the remedy here in most cases is to fuse a cleavage sequence (leader) for a specific endopeptidase (for example factor Xa, enterokinase, KEX endopeptidases, IgA protease) or aminopeptidase (for example dipeptidyl aminopeptidase) to the N-terminus of the target protein. However, this makes an additional step necessary during further working up, the so called down stream processing of the protein, with expenditure of costs and materials. In addition, in the presence of IBs there is in many cases interference with or even complete prevention of the refolding by the leader sequence.<br>
Fusion polypeptides comprising the autoprotease N"™ of Pestivirus are especially useful in this respect. The autoprotease N1*0 of Pestivirus always cleaves off the fusion partner at a clearly determined site, releasing a polypeptide of interest with homogenous N-terminus. In addition, the autoproteolytic activity of NPRO can be induced in vitro, by application of special buffers, so that the polypeptide of interest can be obtained by cleavage of fusion polypeptides that are expressed in IBs.<br>
Pestiviruses are small enveloped viruses with a genome which acts directly as mRNA and is 12.3 kb in size and from which the viral gene products are transcribed in the cytoplasm. This takes place in the form of a single polyprotein which comprises about 4000 amino acids and which is broken down both by viral and by cellular proteases into about 12 mature proteins.<br>
Pestiviruses comprise the subclasses CSFV (classical swine fever virus), BDV (border disease virus) and BVDV (bovine viral diarrhoea virus).<br>
Npre is an autoprotease with a length of 168 amino acids and an apparent M, of about 20,000 (in vivo). It is the first protein in the polyprotein of Pestiviruses and undergoes autoproteolytic cleavage from the following nucleocapsid protein C. This cleavage takes place after the last amino acid in the sequence of NPRO, Cys168.<br>
Use of the naturally occurring autoprotease Npraof Pestivirus for production of heterologous polypeptides of interest may be limited though, as activation of autoproteolytic function of N""3 in vitro is susceptible only to specific renaturazing conditions. These conditions that allow for the cleavage activity of N"10 in vitro are inhibitory for certain other interactions which are necessary or desirable in some settings for production of heterologous polypeptides of interest As an example of such interactions certain bio-specific affinities as e.g. selective peptide-protein affinity can be named. Also, due to other requirements of parameters, certain processes do not permit to create the favourable renaturazing conditions for N1*0 and as a result N1*0 can not be used in these processes. Therefore the naturally occurring N1*0 of Pestivirus may be unsuitable for the production of certain polypeptides of interest and for use under certain conditions. Accordingly the need for an N1*0 of Pestivirus with improved properties exists, in order to enhance cleavage efficiency, to obtain higher yields of polypeptide of interest, and in order to be able to use N1"0 in a wider range of conditions, which allow for the application of new production processes.<br>
Summary of invention<br>
Within the present invention it has surprisingly been found, that certain derivatives of the naturally occurring autoprotease NPRO of Pestivirus, which have a lower pi and enhanced solubility are suitable for use in a wider range of conditions. In addition these derivatives were surprisingly found to have improved autoproteolytic activity in vitro. Accordingly within the scope of the present invention an improved process for the production of heterologous polypeptides of interest with homogenous N-termini is provided, which makes use of the derivatives of autoprotease N1*0 of Pestivirus, also part of the present invention.<br>
Detailed description of invention<br>
The present invention relates to a process for the recombinant production of a heterologous<br>
polypeptide of interest, comprising<br>
(i)	cultivation of a bacterial host cell which is transformed with an expression vector which<br>
comprises a nucleic add molecule which codes for a fusion polypeptide, the fusion<br>
polypeptide comprising a derivative of an autoprotease N1*0 of Pestivirus, wherein at least<br>
one cysteine residue of the naturally occuring autoprotease N"10 of Pestivims is replaced by<br>
another amino acid residue, and a second polypeptide, which is connected to the derivative<br>
at the C-terminus of the derivative in a manner such, that the second polypeptide is capable<br>
of being cleaved from the fusion polypeptide by the autoproteolytic activity of the derivative,<br>
and the second polypeptide being a heterologous polypeptide, wherein cultivation occurs<br>
under conditions which cause expression of the fusion polypeptide and formation of<br>
corresponding cytoplasmic inclusion bodies,<br>
(ii) isolation of the inclusion bodies from the host cell,<br>
(iii) solubilization of the isolated inclusion bodies,<br>
(iv) induction of autoproteolytic cleavage of the heterologous polypeptide from the fusion<br>
polypeptide, and<br>
(v) isolation of the cleaved heterologous polypeptide.<br>
A preferred autoprotease Isl"*0 of pestivirus from which a derivative according to the present invention is derived, is the autoprotease N1*0 of CSFV, having the following amino acid sequence:<br>
SEQIDN01:<br>
(1)-MELNHFELLYKTSKQKPVGVEEPVYDTAGRPLFGNPSEVHPQSTLKLPHDRGRGDIRTT LRDLPRKGDCRSGNHLGPVSGIYIKPGPVYYQDYTGPVYHRAPLEFFDEAQFCEVTKRIGR VTGSDGKLYHIWCVDGCILLKLAKRGTPRTLKWIRNFTNCPLWvTSC-(168)<br>
Within the present invention the above sequence is mutated in order to generate fusion polypeptides with improved properties, the fusion polypeptides comprising a derivative of an autoprotease N1*0 of Pestivirus, wherein at least one cysteine residue of the naturally occuring autoprotease N"10 of Pestivirus is replaced by another amino acid residue, and a second polypeptide, which is connected to said derivative at its C-terminus in a manner such, that the second polypeptide is capable of being cleaved from the fusion polypeptide by the autoproteolytic activity of the autoprotease derivative, said second polypeptide being a heterologous polypeptide.<br>
Accordingly, the present invention relates to such derivatives of the naturally occurring of Pestivirus, which are used in the process of the present invention as N-terminal part of the fusion protein. The derivatives are part of the invention in the sense that they are part of the fusion protein used within the process for the production of heterologous proteins, to which the present invention also relates.<br>
In another aspect the present invention relates to derivatives of the naturally occurring N1*0 Pestivirus, which have reduced tendency to aggregate.<br>
Within the present invention such derivatives of the naturally occurring N1"0 of Pestivirus are preferred, wherein the number of cysteine residues is reduced.<br>
Within the present invention derivatives of the naturally occurring NPRO of CSFV are particularly preferred.<br>
Accordingly the present invention relates to a derivative of an autoprotease N1*0 of CSFV, wherein at least one cysteine residue of the naturally occuring autoprotease N*"0 of CSFV is replaced by another amino acid residue.<br>
Thus the present invention also relates in another aspect to a process as described above, wherein the fusion polypeptide comprises a derivative of an autoprotease IM**0 of CSFV, wherein at least one cysteine residue of the naturally occuring autoprotease N""3 of CSFV is replaced by another amino acid residue.<br>
Preferably the present invention relates to a derivative of an autoprotease N1™ of CSFV, wherein at least one cysteine residue of the naturally occuring autoprotease NPRO of CSFV selected from the group consisting of C112, C134 and C138, is replaced by another amino acid residue.<br>
Thus in another aspect the present invention preferably relates to a process as described above, wherein the fusion polypeptide comprises a derivative of an autoprotease NPRO of<br>
CSFV, wherein at least one cysteine residue of the naturally occuring autoprotease N1*0 of CSFV, selected from the group consisting of C112, C134 and C138, is replaced by a another amino acid residue.<br>
Even further preference is given to a derivative of an autoprotease N*"0 of CSFV, wherein at least one cysteine residue of the naturally occuring autoprotease N1"0 of CSFV selected from the group consisting of C112, C134 and C138, is replaced by a glutamic acid residue.<br>
Thus in another aspect the present invention more preferably relates to a process as described above, wherein the fusion polypeptide comprises a derivative of an autoprotease N"™ of CSFV, wherein at least one cysteine residue of the naturally occuring autoprotease N1*0 of CSFV, selected from the group consisting of C112, C134 and C138, is replaced by a glutamic acid residue.<br>
Further preference is given to a derivative of the autoprotease N""* of CSFV comprising the following amino acid sequence:<br>
SEQ ID NO 2:<br>
(I)-MELNHFELLYKTSKQKPVGVEEPVYDTAGRPLFGNPSEVHPQSTLKLPHDRGRGDIRTT LRDLPRKGDCRSGNHLGPVSGIYIKPGPVYYQDYTGPVYHRAPLEFFDEAQFEEVTKRIGR WGSDGKLYHIWEVDGEILLKLAKRGTPRTLKWIRNFTNCPLWVTSCK168)<br>
Thus in another aspect the present invention relates with further preference to a process as described above, wherein the fusion polypeptide comprises a derivative of an autoprotease N1*0 of CSFV having a sequence according to SEQ ID NO 2.<br>
Solubility of the derivatives is determined in the following way:<br>
After 72 hours a concentrated solution of the respective Nf*° derivative is centrifuged, the pellet dissolved and applied to SDS gel-electrophoresis. A part of the supernatant is combined with probe buffer and applied to SDS gel-electrophoresis. After electrophoresis the bands are stained with coomassie blue, quantified by densitometry with an<br>
AlphaDigiDoc™ system and the amount of precipitated material is calculated. For experimental details see example 2.<br>
Ionic strength of the buffer is often limiting to certain production processes. Therefore the present invention relates in a further aspect to derivatives of the naturally occurring NPRO of Pestivirus, which have a more neutral pi than the naturally occurring N1*0 of Pestivirus. It is preferred to adapt the pi of the H*"0 of Pestivirus moiety of the fusion polypeptide to be expressed as close as feasible to the pi of the second polypeptide (the polypeptide of interest). For example, the N""3 of Pestivirus moiety of the fusion polypeptide may have a pi of from 5.5 to 9.5, especially from 6.0 to 9.0.<br>
Accordingly, within the present invention further preference is given to a derivative of an autoprotease N*"0 of CSFV, wherein in addition to the replacement of at least one cysteine residue as described above, at least one basic amino acid residue is replaced by an acidic amino acid residue.<br>
Thus in another aspect the present invention relates with further preference to a process as described above, wherein the fusion polypeptide comprises a derivative of an autoprotease N1*0 of CSFV, wherein in addition to the replacement of at least one cysteine residue as described above, at least one basic amino acid residue is replaced by an acidic amino acid residue.<br>
Further preference is given to a derivative of an autoprotease N1*0 of CSFV, wherein in addition to the replacement of at least one cysteine residue as described above, furthermore, at least one of the following amino acids are exchanged: H5, K16, N35, R53, G54, R57, L143, K145 and R150. A preferred example is a derivative wherein the following amino acids are exchanged: arginine (R) 53 with glutamic acid (E), glycine (G) 54 with aspartic acid (D), arginine (R) 57 with glutamic acid (E), and leucine (L) 143 with glutamine<br>
(Q).<br>
Thus in another aspect the present invention relates with further preference to a process as described above, wherein the fusion polypeptide comprises a derivative of an autoprotease<br>
Npno of CSFV, wherein in addition to the replacement of at least one cysteine residue as described above, the following amino acids are exchanged: arginine (R) 53 with glutamic acid (E), glycine (G) 54 with aspartic acid (D), arginine (R) 57 with glutamic acid (E), and leucine (L) 143 with glutamine (Q).<br>
In another preferred embodiment of the present invention a derivative of the autoprotease of CSFV comprises the following amino acid sequence:<br>
SEQ ID NO 3:<br>
(1)-MELNHFELLYKTSKQKPVGVEEPVYDTAGRPLFGNPSEVHPQSTLKLPHDRGEDDIETT LRDLPRKGDCRSGNHLGPVSGIYIKPGP\ATQDYTGPVYHRAPLEFFDEAQFEEVTKRIGR VTGSDGKLYHIYVEVDGEILLKQAKRGTPRTLKWIRNFTNCPLWVTSC-{ 1 68).<br>
Thus in another aspect the present invention also relates to a process as described above, wherein the fusion polypeptide comprises a derivative of an autoprotease N1™ of CSFV having a sequence according to SEQ ID NO 3. .<br>
In jet another aspect the present invention relates to a derivative of the naturally occurring NPRO of a Pestivirus, which shows in addition to the reduced aggregation and more neutral pi further enhanced solubility, as compared to the naturally occurring Npreof a Pestivirus.<br>
Solubility is determined as described above.<br>
Accordingly the present invention relates to a derivative of an autoprotease N1*0 of CSFV, wherein, in addition to the replacement of at least one cysteine residue as described above, at least one hydrophobic amino acid residue is replaced by a hydrophilic residue.<br>
Thus in another aspect the present invention also relates to a process as described above, wherein the fusion polypeptide comprises a derivative of an autoprotease NPRO of CSFV, wherein in addition to the replacement of at least one cysteine residue as described above, at least one hydrophobic amino acid residue is replaced by a hydrophilic residue.<br>
Preferred within the present Invention is a derivative of an autoprotease N1"0 of CSFV, wherein in addition to the replacement of at least one cysteine residue as described above furthermore at least one of the following amino adds are replaced: V24, A27, L32, G54, L75, A109, V114, V121, L143, 1155 and F158. A preferred example is a derivative wherein the following amino acids are exchanged by threonine (T): alanine (A) 109, valine (V) 114, isoleucine (I) 155 and phenylalanine (F)158.<br>
Thus in another aspect the present invention relates preferably to a process as described above, wherein the fusion polypeptide comprises a derivative of an autoprotease M1*0 of CSFV, wherein in addition to the replacement of at least one cysteine residue as described above, the following amino adds are replaced by threonine (T): alanine (A) 109, valine (V) 114, isoleucine (1)155 and phenylalanine (F)158. Another, within the present invention more preferred derivative of an autoprotease NPRO of CSFV, comprises the following amino acid sequence:<br>
SEQ ID NO 4:<br>
(1)-MELNHFELLYKTSKQKPVGVEEPVYDTAGRPLFGNPSEVHPQSTLKLPHDRGRGDIRTT LRDLPRKGDCRSGNHLGPVSGIYIKPGPNATQDnGPVWRAPLEFFDETQFEETTKRIGRV TGSDGKLYHIYVEVDGEILlKLAKRGTPRTUCWTFU^TTNCPLVvVrSC-(168)<br>
Thus in another aspect the present invention more preferably relates to a process as described above, wherein the fusion polypeptide comprises a derivative of an autoprotease N"10 of CSFV having a sequence according to SEQ ID NO 4.<br>
Even more preferred within the present invention is a derivative of an autoprotease N1"0 of CSFV, wherein in addition to the replacement of at least one cysteine residue as described above the following amino adds have been exchanged: alanine (A) 109, valine (V) 114, isoleucine (f) 155 and phenylalanine (F) 158 by threonine (T), arginine (R) 53 with glutamic acid (E), glycine (G) 54 with aspartic acid (D), arginine (R) 57 with glutamic acid (E), and leucine (L) 143 with glutamine (Q).<br>
Thus in another aspect the present invention relates even more preferably to a process as<br>
described above, wherein the fusion polypeptide comprises a derivative of an autoprotease tf*° of CSFV, wherein in addition to the replacement of at least one cysteine residue as described above the following amino acids have been exchanged: alanine (A) 109, valine (V) 114, isoleucine (I) 155 and phenylalanine (F) 158 by threonine (T); arginine (R) 53 with glutamic acid (E), glycine (G) 54 with aspartic acid (D), arginine (R) 57 with glutamic acid (E), and leucine (L) 143 with glutamine (Q).<br>
Most preferably the derivative of an autoprotease NPRO of CSFV according to the present invention comprises the following amino acid sequence:<br>
SEQ ID NO 5:<br>
(1)-MELNHFELLYKTSKQKPVGVEEPVYDTAGRPLFGNPSEVHPQSTLKLPHDRGEDDIETT LRDLPRKGDCRSGNHLGPVSGIYIKPGPNAYQDYTGPVYHRAPLEFFDETQFEETTKRIGRV TGSDGKLYHIWEVDGEILLKQAKRGTPRTLKWTRNTTNCPLWVTSC-(168).<br>
Thus in another, most preferred aspect the present invention also relates to a process as described above, wherein the fusion polypeptide comprises a derivative of an autoprotease NP10 of CSFV having a sequence according to SEQ ID NO 5.<br>
In another equally preferred aspect the present invention relates to a process for the production of heterologous proteins as described above, wherein the fusion polypeptide comprises a derivative of an autoprotease NPRO of CSFV having a sequence according to SEQ. ID NO. 5, wherein in addition asparagine (N) 35 is replaced with threonine (T), and threonine (T) 158 is replaced with serine (S).<br>
The derivative of an autoprotease N1"0 of CSFV which is utilized in the process according to the above aspect of the present invention forms also part of the present invention and comprises the following amino acid sequence:<br>
SEQ ID NO 32:<br>
 (1)-MELNHFELLYKTSKQKPVGVEEPVYDTAGRPLFGTPSEVHPQSTLKLPHDRGEDDIETT<br>
LRDLPRKGDCRSGNHLGPVSGIYIKPGPVYYQDYTGPVYHRAPLEFFDETQFEETTKRIGRV<br>
TGSDGKLYHIYVEVDGEILLKQAKRGTPRTLKWTRNSTNCPLWVTSO{168).<br>
In another preferred aspect the present invention relates to a process for the production of heterologous proteins as described above, wherein the fusion polypeptkle comprises a derivative of an autoprotease N1*0 of CSFV having a sequence according to SEQ. ID NO. 32, wherein in addition alanine (a) 28 is replaced with glutamic acid (E), serine (S) 71 is replaced with phenylalanine (F) and arginine (R) 150 is replaced with histidine (H).<br>
The derivative of an autoprotease N1™ of CSFV which is utilized in the process according the above aspect of the present invention forms also part of the present invention and comprises the following amino acid sequence:<br>
SEQ ID NO 33:<br>
(1)-MELNHFELLYKrSKQKPVGVEEPVYDTEGRPLFGTPSEVHPQSTLKLPHDRGEDDIETT<br>
LRDLPRKGDCRFGNHLGPVSGmKPGPVYYQDYTGPWHRAPLEFFDETQFEETTKRIGRV<br>
TGSDGKLYHIYVEVDGEILLKQAKRGTPHTLKV\nTlNSTNCPLVVVTSC-(168).<br>
Preferably in the process according to the present invention the derivative of an autoprotease N1"0 of CSFV with the sequence according to SEQ ID NO 32 is used in fusion with a protein that contains at least the three first amino acids of proinsulin, more preferably with proinsulin, further more preferably with human proinsulin, most preferably with recombinant human proinsulin, for the production of proinsulin.<br>
It is preferred according to the present invention if the derivative of an autoprotease N1"0 of CSFV has in addition to the replacement of at least one cysteine residue as described above at least one of the following amino acids exchanged: arginine (R) 53, glycine (G) 54, arginine (R) 57, threonine (T) 109, 114, 155, 158 and leucine (L) 143. Preferred derivatives of the autoprotease N1*0 of CSFV according to the present invention have in addition to the<br>
replacement of at least one cysteine residue as described above, the following amino acids are exchanged: arginine (R) 53 with glutamic acid (E), glycine (G) 54 with aspartic acid (D), arginine (R) 57 with glutamic acid (E), threonine (T) 109,114.155,158 with serine (S) and leucine (L) 143 with glutamine (Q) or asparagine (N) or aspartic acid (D) or serine (S) or histidine:<br>
Such preferred derivatives of an autoprotease N*™ of CSFV which are utilized in the process according the above aspect of the present invention forms also part of the present invention and comprise the following amino acid sequences:<br>
SEQ ID 92:<br>
(1)-MELNHFELLYKTSKQKPVGVEEPWDTAGRPLFGNPSEVHPQSTLKLPHDRGEDDIETT LRDLPRKGDCRSGNHLGPVSGIYIKPGPVYYQDYTGPVYHRAPLEFFDESQFEESTKRIGR VTGSDGKLYHIYVEVDGEILLKSAKRGTPRTLKWSRNSTNCPLWVTSC-(168).<br>
SEQ ID 95:<br>
(I)-MELNHFELLYKTSKQKPVGVEEPVYDTAGRPLFGNPSEVHPQSTLKLPHDRGEDDIETT LRDLPRKGDCRSGNHLGPVSGIYIKPGPVYYQDYTGPNAHRAPLEFFDESQFEESTKRIGR VTGSDGKLYHIYVEVDGEILLKNAKRGTPRTLKWSRNSTNCPLWVTSC-(168).<br>
SEQ ID 96:<br>
(I)-MELNHFELLYKTSKQKPVGVEEPVYDTAGRPLFGNPSEVHPQSTLKLPHDRGEDDIETT LRDLPRKGDCRSGNHLGPVSGIYIKPGPNATQDYTGPVYHRAPLEFFDESQFEESTKRIGR VTGSDGKLYHIYVEVDGEILLKDAKRGTPRTLKWSRNSTNCPLWVTSC-(168).<br>
SEQ ID 97:<br>
(1)-MELNHFELLYKTSKQKPVGVEEPVYDTAGRPLFGNPSEVHPQSTLKLPHDRGEDDIETT LRDLPRKGDCRSGNHLGPVSGIYIKPGPVYYQDYTGPVYHRAPLEFFDESQFEESTKRIGR VTGSDGKLYHIWEVDGEILLKHAKRGTPRTLKWSRNSTNCPLWVTSC-(168).<br>
SEQ ID 98:<br>
 (1)-MELNHFELLYKrSKQKPVGVEEP\AT&gt;TAGRPLFGNPSEVHPQSTLKLPHDRGEDDIETT<br>
LRDLPRKGDCRSGNHLGPVSGIYIKPGPVYYQDYTGPNAHRAPLEFFDESQFEESTKRIGC<br>
VTGSDGKLYHIYVEVDGEILLKQAKRGTPRTLKWSRNSTNCPLWVTSC-(168).<br>
The derivatives of the naturally occurring N""0 of CSFV described above which are part of the present invention, have improved properties over the naturally occurring N"™ of CSFV and are therefore suitable to enhance efficiency of protein production. Refolding of the derivatives described in this invention can be induced in vitro in a wide range of conditions, e.g. under lower ionic strengths, where the natural occurring N"10 would by dysfunctional. Therefore the derivates described above are suitable for use under reaction conditions that do not allow for successful use of the naturally occurring N1*0. The derivatives which are last described herein is particularly preferred within the present invention, due to its suitability for use in a particularly wide range of reaction conditions.<br>
In a further aspect the present invention relates to the use of any of the derivatives of an autoprotease N"™ of CSFV described above in a process for the production of heterologous polypeptides of interest according to the present invention.<br>
Thus in the process for recombinant production of heterologous polypeptides of interest according to the present invention, the fusion polypeptide comprises any one of the above described derivatives of an autoprotease N1*0 of CSFV.<br>
In a preferred embodiment of the present invention, induction of autoproteolytic cleavage of the heterologous polypeptide is performed by diluting the fusion polypeptide under conditions which promote refolding. Thereby the inactive fusion polypeptide is refolded and thus activated.<br>
In a particularly preferred embodiment, the solubilizate is diluted with an arginine-containing buffer so that the final concentration of arginine is up to 1.0M, preferably 0.4-0.6 M. Alternatively, dilution is also possible by dialysing the solubilized inclusion bodies against an appropriate arginine-containing cleavage buffer.<br>
The temperature of the reaction solution for the cleavage is, for example, between 0°C and 30°C. The temperature can preferably be 10°C-20°C.<br>
The pH of the reaction solution is, for example, 5.0-9.0. The pH is preferably 7.0-8.0, in particular 7.0-7.5. Most preferably the pH is 7.4.<br>
Where appropriate, the reaction solution contains DTT in a concentration of 0.5-100 mM. The DDT concentration is preferably about 5.5 mM.<br>
The protein concentration in the reaction solution during the cleavage can be, for example, in the region of 20-150 jig/ml. The protein concentration is preferably less than 40 ng/ml.<br>
The reaction solution can contain tris/HCI in a concentration of, for example, up to 1.5M during the cleavage. The tris/HCI concentration is preferably between 0.4M and 1.2M.<br>
The reaction solution can contain glycerol in a concentration range of for example between 0,2 and 1%. More preferably the glycerol concentration is 5%.<br>
Also, the reaction solution can contain EDTA in a range of about 1-3 mM EDTA. Preferably the EDTA concentration is 2mM.<br>
Other buffer systems are also possible in place of arginine-containing and/or tris/HCI-containing buffers.<br>
In a particularly preferred embodiment, the pH in the cleavage buffer is 7.4, the temperature during the cleavage is 10°C-20°C, and the cleavage buffer contains about 10mM DTT as reducing agent, 0.5M NaCI, 5% glycerol, and 2mM EDTA.<br>
Finally, the heterologous polypeptide which has been cleaved from the fusion polypeptide is isolated in a manner known per se.<br>
In another preferred embodiment of the present invention, induction of autoproteolytic cleavage of the heterologous polypeptide is performed by binding the fusion polypeptide to an affinity chromatographic system, with its autoproteolytic part in an inactive form, and subsequent application of a refolding buffer. In particular, in a first step the fusion polypeptide is bound to the chromatographic system. Binding is maintained while conditions are changed such, that the autoproteolytic part of the fusion polypeptide regains its activity. The polypeptide of interest is cleaved and eluted while the autoproteolytic part of the fusion polypeptide remains bound to the chromatographic system.<br>
In a particularly preferred embodiment of the present invention the affinity chromatographic system is a column, and the fusion polypeptide is inactivated by denaturation. Therefore reactivation of the autoproteolytic activity is induced by application of refolding buffer and thus refolding of the fusion polypeptide.<br>
In a particularly preferred embodiment of the present invention, refolding is conducted in a buffer with the following composition: 0.5M NaCI, 20 mM sodium phosphate, 5% glycerol, 2mM EDTA, 10 mM DTT 0,01% Brij, pH 7.4.<br>
As used herein the following terms shall have the meanings described below:<br>
The term "heterologous polypeptide of interest" means a polypeptide which is not naturally cleaved by an autoprotease N1*0 of Pestivirus from a naturally occurring fusion polypeptide or polyprotein. Examples of heterologous polypeptides are industrial enzymes (process enzymes) or polypeptides with pharmaceutical, in particular human pharmaceutical, activity.<br>
The term "fusion polypeptide" refers to polypeptides consisting of two or more polypeptides. In particular herein fusion polypeptides can comprise an affinity tag, an autoproteolytic part, preferably an autoprotease, and the polypeptide of interest.<br>
The term "polypeptide of interest" refers to the polypeptide to be produced with a homogenous N-terminus.<br>
According to the present invention an expression vector can be used, which encodes the polypeptide of interest as the part of the fusion polypeptide which is to be cleaved off autoproteolytically. In accordance with the present invention, a variety of polypeptides of interest can be produced by use of such an expression vector. For example, the polypeptide of interest is one, that exerts pharmacological activity and can for example be selected from the group consisting of an interferon, an interteukin, a growth hormone, a growth factor, a cytokine, an enzyme, an enzyme inhibitor, an antibody and an antibody fragment, and the like, for example interferon alpha 2A, interferon alpha 2B, interteukin-3, interteukin-6, human growth hormone, (pro)insulin, insulin like growth factor, granulocyte-colony stimulating factor, granulocyte macrophage-colony stimulating factor, macrophage-colony stimulating factor, interferon beta 1, bovine somatropin, porcine somatropin, interteukin 11, interleukin-2, a Fab-fragment, and small peptides such as calcitonin, parathyroid hormone (PTH), or a glucagon, CD 40 ligand soluble form, plasminogen activator, sex steroid binding protein, epidermal growth factor, tissue factor extra cellular domain.<br>
In addition the polypeptide of interest can be any other kind of polypeptide in particular a polypeptide which is especially suited for analytical methods, e.g. Green Ruorescent Protein.<br>
In the expression vector to be employed in the process according to the present invention, the fusion polypeptide is operably linked to at least one expression control sequence. Expression control sequences are, in particular, promoters (such as the lac, tac, T3, T7, tip, gac, vhb, lambda pL or phoA promoter), ribosome binding sites (for example natural ribosome binding sites which belong to the abovementioned promoters, cro or synthetic ribosome binding sites), or transcription terminators (for example rmB T1T2 or bla).<br>
The vector may also contain sequences encoding fusion domains, as described below, that are present at the N-terminal end of the fusion polypeptide and that are required for its binding to the affinity chromatography system, e.g. polyamino acids like polylysine or, for immunoaffmity chromatography, so-called "epitope tags", which are usually short peptide sequences for which a specific antibody is available. Well known epitope tags for which specific monoclonal antibodies are readily available include FLAG, influenza virus haemagglutinin (HA), and c-myc tags.<br>
In a preferred embodiment of the present invention, the expression vector is a plasmld.<br>
A bacterial host cell to be employed in the process according to the present invention can be selected, for example, from the group of the following micro organisms: Gram-negative bacteria such as Escherichia species, for example E. coli, or other Gram-negative bacteria, for example Pseudomonas sp., such as Pseudomonas aeruginosa, or Cautobacter sp., such as Caulobacter crescendos, or Gram-positive bacteria such as Bacillus sp., in particular Bacillus subtilis. E. coli is particularly preferred as host cell.<br>
As used herein the term "transformed host cell" shall refer to a cell containing a vector coding for a heteroiogous polypeptide.<br>
The bacterial host cell, i.e. the expression strain, is cultivated in accordance with microbiological practice known per se. The strain is generally brought up starting from a single colony on a nutrient medium, but it is also possible to employ cryo-preserved cell suspensions (cell banks). The strain is generally cultivated in a multistage process in order to obtain sufficient biomass for further use.<br>
On a small scale, this can take place in shaken flasks, it being possible in most cases to employ a complex medium (for example LB broth). However, it is also possible to use defined media (for example citrate medium). For the cultivation, a small-volume pre-culture of the host strain (inoculated with a single colony or with cell suspension from a cryo-culture) is grown, the temperature for this cultivation not generally being critical for the later expression result, so that it is possible routinely to operate at relatively high temperatures (for example 30°C or 37°C). The main culture is set up in a larger volume (for example 500 ml), where it is in particular necessary to ensure good aeration (large volume of flask compared with the volume of contents, high speed of rotation). Since it is intended that expression take place in the form of insoluble inclusion bodies, the main culture will in most cases also be carried out at relatively high temperature (for example 308C or 37°C). Inducible systems are particularly suitable for producing inclusion bodies (for example with trp, lac, tac or phoA promoter). After the late logarithmic phase has been reached (usually at<br>
an optical density of 0.5 to 1.0 in shaken flasks), in these cases the inducer substance (for example indoleacrylic acid, isopropyl p-D-thiogalactopyranoside = IPTG) is added and incubation is continued for 1 to 5 hours. During this time, most of the Npra fusion polypeptide is deposited as inclusion bodies in the bacterial cytoplasm. The resulting cells can be harvested and processed further.<br>
On a larger scale, the multistage system consists of a plurality of bioreactors (fermenters), it being preferred to employ defined nutrient media in this case in order to be able to improve the process engineering control of the process. In addition, it is possible greatly to increase biomass and product formation by metering in particular nutrients (fed batch). Otherwise, the process is analogous to the shaken flask. For example, a preliminary stage fermenter and a main stage fermenter are used, the cultivation temperature being chosen similar to that in the shaken flask. The preliminary stage fermenter is inoculated with a so-called inoculum which is generally grown from a single colony or a cryoculture in a shaken flask. Good aeration and a sufficient inducer concentration must also be ensured in the fermenter - and especially in the main stage thereof. The induction phase must, however, in some cases be made distinctly longer compared with the shaken flask. The resulting cells are once again delivered for further processing.<br>
In the process according to the present invention, the inclusion bodies are isolated from the host cell in a manner, known per se.<br>
For example, after the fermentation has taken place, the host cells are harvested by centrifugation, micro filtration, flocculation or a combination thereof, preferably by centrifugation. The wet cell mass is disintegrated by mechanical, chemical or physical means such as high pressure homogenizer, beads mills, french press, hughes press, osmotic shock, detergents, enzymatic lysis or a combination thereof. Preferably, disruption of the cells takes place by high pressure homogenization. In the favoured case that the recombinant fusion polypeptide is deposited as inclusion bodies, the inclusion bodies can be obtained for example by means of high-pressure dispersion or, preferably, by a simple centrifugation at low rotor speed. The inclusion bodies are separated by centrifugation or microfiltration or a combination thereof. The purity in relation to the desired polypeptide of<br>
interest can then be improved by multiple resuspension of the inclusion bodies in various buffers, for example in the presence of NaCI (for example 0.5-1.0 M) and/or detergent (for example Triton X-100). Preferably the purity of the inclusion body preparation is improved by several washing steps with various buffers (e.g. 0.5 % Deoxycholate followed by two times 1 M NaCI solution — and finally distilled water). This usually results in removal of most of the foreign polypeptides in the inclusion bodies.<br>
As used herein the term "solubilization" shall refer to the process necessary to dissolve the inclusion bodies. Solubilization results in a monomolecular dispersion of the polypeptides with minimum intra- and inter-molecular interactions.<br>
A preferred way of solubilization of inclusion bodies within the scope of the present invention, is conducted by suspension in 50 mM Tris/HCI, 8 M urea, pH 7.3, adding a reducing agent, e.g. 50 mM DTT, 4-8M Guanidinium.HCI or Guanidinium SON, to prevent oxidation of eventually present cysteine residues. Where necessary it is possible to remove potentially insoluble material, for example by centrifugation.<br>
in the case that the inactive fusion polypeptide is produced soluble within the cell, the clarified cell homogenate is subjected to the further work up described above for the solubilized inclusion bodies, except for the step of dilution since the cell homogenate is already diluted.<br>
In a preferred embodiment, the solubilizate is diluted with a Tris/HCI containing buffer so that the final concentration of Tris/HCI is up to 1.5 M, preferably 0.4-1.2 M. Alternatively, dilution is also possible by dialysing the solubilized inclusion bodies against an appropriate Tris/HCI containing cleavage buffer. Tris/HCI can be replaced by other salts eg. 0.2 - 1.5 M NaCI if an appropriate buffer substance is added eg. 20 mM sodium phosphate.<br>
The temperature of the reaction solution for the cleavage is, for example, between 0°C and 30°C. The temperature can preferably be 10°C-20°C.<br>
The pH of the reaction solution is, for example, 5.0-9.0. The pH is preferably 7.0-8.0, in<br>
particular 7.0-7.5. Most preferably the pH is 7.4.<br>
Where appropriate, the reaction solution contains DTT in a concentration of 0.5-100 mM. The DDT concentration is preferably about 10 mM.<br>
The protein concentration in the reaction solution during the cleavage can be, for example, in the region of 20-150 iig/ml. The protein concentration is preferably less than 40 \igfml<br>
The reaction solution can contain arginine in a concentration of, for example, up to 1.0M during the cleavage. The tris/HCI concentration is preferably between 0.4M and 0.6M.<br>
The reaction solution can contain glycerol in a concentration range of for example between 0,2 and 30%. More preferably the glycerol concentration is 5%.<br>
Also, the reaction solution can contain EDTA in a range of about 1-3 mM EDTA. Preferably the EDTA concentration is 2mM.<br>
Other buffer systems are also possible in place of arginine-containing and/or tris/HCI-containing buffers.<br>
In a particularly preferred embodiment, the pH in the cleavage buffer is 7.4, the temperature during the cleavage is 10°C-200C, the cleavage buffer contains about 10mM DTT as reducing agent, 0.5M NaCI, 20 mM sodium phosphate 5% glycerol, and 2mM EDTA.<br>
Finally, the heterologous polypeptide which has been cleaved from the fusion protein is isolated in a manner known per se.<br>
The present invention is described further with reference to the following examples, which are illustrative only and non-limiting. In particular, the examples relate to preferred embodiments of the present invention.<br>
EXAMPLES<br>
With the present invention it is possible to express and produce a wide variety of recombinant proteins (or "polypeptides of interest"), especially such proteins which are problematic to express in usual systems, e.g. proteins with toxic effects on the host cells, proteins which are insoluble or have low solubility, proteins which have other solubility disadvantages (e.g. shorter proteins). The present derivatives also show - in the form of specific fusion constructs or specific activation conditions improvements in cleavage rates, expression rates, overall production rates. Moreover, expression in inclusion bodies with the present constructs show advantageous results for the above mentioned problems (see e.g. example 11).<br>
For example, small proteins have generally low expression rates in E. coli, because they are rapidly degraded in bacterial cells; the constructs according to the present invention allow elevated expression levels (see example 14).<br>
Examples 1, 3, 4, 5, 8 and 9 describe the production of proinsulin utilizing different aspects of the process according to the present invention. The sequence of proinsulin is given below in SEQ ID NO 6, forming the non-bold part of the sequence. In the following for convenience proinsulin is sometimes referred to as insulin.<br>
Example 1<br>
Production of a heterologous polypeptide of interest (insulin) by refolding, using the<br>
N^-derivative with SEQ ID NO 5 (EDDIE)<br>
1.1 Generation of derivatives<br>
1.1.1 Mutational PCR<br>
From the construct containing the DMA sequence for NPRO-pro-insulin (SEQ ID NO 6):<br>
ATGGAACTCAATCATTTCGAACTGCTCTACAAAACTAGCAAGCAAAAACCTGTTGGCGT<br>
TGAAGAGCCGGTCTAC6ATACTGCAGGTCGTCCTCTTTTTGGGAATCCGTCCGAAGTG<br>
CACCCCCAGTCAACCCTCAAGCTTCCCCATGACCGCGGACGCGGTGACATTCGTACAA<br>
CGCTGCGCGATCTGCCTCGTAAAG6CGATTGTCGCTCTGGAAACCACCTAGGTCCGGT<br>
GTCGGGCATTTACATTAAACCAGGTCCCGTCTATTACCAAGACTACACTGGTCCGGTTT<br>
ACCATCGTGCACCTCTGGAATTCTTTGATGAAGCTCAATTTTGCGAAGTGACTAAACGT<br>
ATTGGCCGTGTAACCGGTTCGGACGGGAAACTGTACCACATCTACGTGTGCGTTGATG<br>
GCTGTATCCTGCTGAAACTCGCGAA6CGCGGAACCCCTCGCACCCTGAAATGGATCCG<br>
TAACTTCACTAACTGTCCACTGTOGGTCACTAGTTGCTTCGTTAACCAACATCTGTGCG<br>
GTTCACACCTTGTGGAAGCCCTGTATCTGGTGTGTGGCGAACGCGGATTCTTTTATACC<br>
CCGAAAACGCGGCGCGAAGCCGAAGATCTTCAGGTTGGTCAAGTGGAACTGGGCGGA<br>
GGTCCGGGAGCCGGGAGCCTGCAACCGCTGGCGCTTGAAGGGTCGCTGCAAAAACGC<br>
GGTATTGTTGAACAGTGCTGTACCTCCATCTGCTCTCTGTATCAGCTGGAAMCTACTG<br>
CAATTAATAA<br>
that is custom-synthesized and inserted into pUC119 ( NCBI #1107650: National Centre for Biotechnology Information Plasmid Database, National Library of Medicine, Building 38A, Bethesda, MD 20894, USA) by Operon Biotechnologies Inc. (1000 Atlantic Avenue, Suite 108 Alameda , CA 94501, USA). From this construct the, the required NPRO-sequence, indicated in bold, is amplified by PCR using the following primer pain UP™ -F-Ndel, (SEQ ID NO 20) and N1*0 -R-Sall, (SEQ ID NO 21) and inserted via the newly created restriction sites for A/del and Sail (bold letters, table 1 below) into the vector pETSOa (# 69909-3, 2002-2003 catalogue, Novagen, CN Biosciences Inc., Merck KgaA, Darmstadt, Germany) creating S-Np-6H-pET30a. From S-Np-6H-pET30a the N1"0 sequence is amplified in by two standard 50 ul PCR reactions: one with 50 pmol N1™ -F-Ndel primer (SEQ ID NO 20) and 50 pmol of one reverse mutation primer selected from Tablel, (SEQ ID NO 8, 10, 12, 14, 16, 18), 5 units Taq DNA-polymerase (# GC 002004, catalog 2004 Genecraft, Treskow StraRe 10, D-48163 Munster, Germany), 1x PCR buffer (# GC 002006 catalog 2004, Genecraft and 20 nmol each dNTP mixture (# GC 013004, catalog 2004, Genecraft); the second with 50 pmol N""" -R-Sall primer (SEQ ID NO 21) and 50 pmol of one forward mutation primer, selected from Tablel, (SEQ ID NO 7, 9,11, 13, 15, 17) 5 units Taq DNA-polymerase, 1x PCR buffer and 20 nmol each dNTP mixture. PCR reaction takes place in a<br>
heated lid thermocyder using the following program: 94°C for 3 min; 25 cycles: 94eC for 30 sec, 54°C for 30 sec, 68°C for 1 min; final incubation at 68°C for 7 min.<br>
1.1.2 Amplification of mutant by PCR<br>
The mutation primers given in Tablel are used to introduce the respective amino acid changes. One-hundredth of both PCRs is combined and amplified in a standard 50 \i\ PCR reaction with 50 pmol N"10 -F-Ndel primer (SEQ ID NO 20) and 50 pmol N1*0 -R-Sall primer (SEQ ID NO 21), as described above. Free primers are removed by QIAquick PCR Purification Kit (Qiagen GmbH, Qiagen Strasse 1, D 40724 Hilden, Cat. Nr.28104, Quiagen product guide 2005) according to the manufacturers recommendations. The PCR fragments are inserted via the Ndel and Sail restriction sites into vector pET30a. The construct is then used for the next mutational step. This is done in a number of consecutive steps to introduce the amino acid changes necessary to create the desired N1*0 derivative. In the case of this example the process is repeated six times. The respective amino acid exchanges are indicated in table 1. The outcoming plasmid of each step is controlled by DMA sequence analysis as described (see 4.1) The mutations I155T and F158T are introduced by a single PCR reaction with the primer pair N"10 -F-Ndel (SEQ ID NO 20) and 3'J155T, F158T (SEQ ID NO 19) and the resulting PCR product is inserted via the Ndel and Spel restriction sites into S-Np-6H-pET30a. The combination of all eleven amino acid changes results in EDDIE-6H-pET30a, where EDDIE stands for the mutant of the autoprotease NPRO of CSFV with SEQ ID NO 5.<br>
Tablel: Mutation primers with corresponding amino acid changes:<br>
(Table Removed)<br>
1.2	Construction of plasmid<br>
This process is conducted analogous to the one described under 4.1.<br>
1.3	Transformation of host cells<br>
This process is conducted analogous to the one described under 4.2 below.<br>
1.4	Expression and fermentation<br>
These processes are conducted analogous to the one described under 4.3 below.<br>
1.5	Cleavage<br>
1 ml of over night culture of host cells transformed as described in 4.2. with construct 6H-EDDIE-SDDIns-pET30a (for construction see 4.1) is transferred into 100 ml M9-KAN medium (50 mM Na2HPO4, 20 mM KH2PO4, 10 mM NaCI, 20 mM NH4CI, 1mM MgSO4, 0,4 % w/v Glucose, 50ug/ml Kanamycin), incubated at 37°C and 225 rpm to an OD of 0.5 and induced for expression with 1 mM IPTG at 37°C for 2 h. Cells are spun down at 2500 g<br>
for 15 min. The pellet is suspended in 8 ml lysis-buffer (20mM Na2HPO4,75 mM NaCI, 5 mM EDTA, 2 mM MgCI2), transferred into a pre-cooled press chamber and incubated at 1380 bar for 5 min. The valve is opened slowly and 500 pi aliquots poured drop by drop (2-4 drops/10 sec) into 1,5 ml tubes. The homogenate is spun for 15 min at 19000g and 4°C, the supernatant discarded and the pellet suspended in 30 pi lysis-buffer (or H2O). 500 ul Guanidinium HCI-solution (5 M Guanidinium HCI, 120 mM Tris pH7.3, 25 mM DTT) are added and incubated for 40 min at room temperature. 10 ul are transferred into a clean reaction tube for TCA-precipitation (IB control), another 10 pi are transferred into a clean tube for in-vitro renaturation by 1:50 dilution with 490 pi refolding buffer (0.5M NaCI, 5 % glycerol, 2 mM EDTA, 10 mM DTT, pH 7.4) for 40 min at RT followed by TCA-precipitation. The TCA precipitates are spun down, the SN discarded, the pellet dissolved in 10 ul 1x SDS-PAGE probe buffer and the success of renaturation and cleavage analyzed by SDS-PAGE. The gel is stained with Coomassie Brilliant Blue R250 (Fluka cat. n. 27816, Laborchemikalien und analytische Reagentien 2005/2006, Ruka Chemie GmbH, Industriestrasse 25, CH-9471 Buchs, Switzerland), the bands of undeaved fusion polypeptides and cleaved autoprotease are quantified by densitometry based on measurement of absorption of white light by the stain and amount of cleavage is calculated.<br>
Example 2 Determination of solubility<br>
The pellet of an 800 mi culture of E. coll BL21 (DE3) transformed with EDDIE-6H-pET30a (for construction see 1.1.2) is prepared as described under 4.3. The pellet is suspended in 40 ml/g lysis puffer (20 mM Na2HPO4, 75 mM NaCI, 5 mM EDTA, 2 mM MgCI2, 10 mM 2-Mercaptoethanol pH 8). Lysis of the cells is achieved by two passages through pressure cell (1380 bar). After incubation for 15 min. with 1% Triton X-100 (solubilized in 5ml/g lysis puffer) the cell homogenate is centrifuged with 25000 g for 45 min, the supernatant discarded and the inclusion bodies (IB) stored at -20 °C. Inclusion bodies are dissolved to 1,3 ml/g IB in Guanidinium chloride solution (5 M GuCI, 120 mM Tris, 25 mM DTT, pH 7.5) incubated for 3.5 h at room temperature and centrifuged with 25000g for 15 min. The supernatant is diluted to 30 ml/g IB in refolding puffer (0.4 M Tris, 10 mM DTT, 2 mM EDTA, 5 % Glycerol, 7.3 pH), incubated over night at room temperature, centrifuged and sterile<br>
filtered. The NPRO derivative is purified by ion exchange chromatography on an SP Sepharose column with a volume of 50 ml. The column is equilibrated with 3 CV of 0.4 mM Tris pH 7.3 and after application of the refolding solution washed with 150 mM NaCI 20 mM Na2HPO4, pH 7.5. Elution is carried out with 3 CV 600 mM NaCI, 20 mM Na2HPO4, 5% Glycerol, pH 7.5. Fractions 8 and 9 (8.5 ml each) containing the protein are combined and concentrated by membrane filtration (Amicon Centricon plus-20, #UFC2LGC24, product catalogue 2004, Millipore Corporation, 290 Concord Rd. Billerica, MA 01821, USA) using centrifugation (30 min 805g) and the resulting solution is subjected to a second concentration step (Amicon Microcon YM-10, #42407, product catalogue 2004, Millipore Corporation) for 30min at 17000g and room temperature. After 72 hours the concentrated solution is centrifuged (10 min, 17000 g, room temperature), the pellet dissolved in 10 ul 1x SDS-PAGE probe buffer and applied to SDS gel-electrophoresis. 10 ul of the supernatant are combined with 10 ul 2x SDS-PAGE probe buffer and applied to SDS gel-electrophoresis. After electrophoresis the bands are stained with Coomassie Brilliant Blue R250, quantified as described (2) and the amount of precipitated material is calculated.<br>
Example 3<br>
Production of a heterologous polypeptide of interest (insulin) by refolding, using one<br>
of the NPRO-derivatives with SEQ ID NO 2,3 or 4, respectively<br>
The different steps of the process are performed analogous for each of the three derivatives with SEQ ID NO 2,3 or 4. The outcome for these derivatives is according to the results achieved with the derivative with SEQ ID NO 5 (see example 1).<br>
3.1 Generation of derivatives<br>
3.1.1	Mutational PCR<br>
This process is conducted analogous to the one described under 1.1.1.<br>
3.1.2	Amplification of mutant by PCR<br>
This process is conducted analogous to the one described under 1.1.2.<br>
3.2	Construction of plasmid<br>
This process is conducted analogous to the one described under 4.1.<br>
3.3	Transformation of host cells<br>
This process is conducted analogous to the one described under 4.2 below.<br>
3.4	Expression and fermentation<br>
These processes are conducted analogous to the one described under 4.3 below.<br>
3.5	Cleavage<br>
This process is conducted analogous to the one described under 1.5.<br>
Solubility and cleavage efficiency can be tested using the techniques disclosed under 1.5<br>
and example 2.<br>
Example 4<br>
Production of a heterologous polypeptide of interest (insulin) by on column refolding,<br>
using the N^-derivative with SEQ ID NO 5 (EDDIE)<br>
In the following "EDDIE" indicated the mutant of the naturally occurring autoprotease N1"0 of CSFV with the sequence according to SEQ ID NO 5.<br>
For this experiment the construct pET30-6H-EDDIE-SDD-lns is used to express the fusion polypeptide 6H-EDDIE-SDD-lns. This fusion polypeptide comprises an N-terminally Bxhistidine tagged mutant form of the pestiviral autoprotease N1*0, SEQ ID NO 5, followed by a SDD-linker (serine, aspartic add, aspartic acid) and the sequence of pro-insulin.<br>
4.1 Construction of plasmids<br>
The DNA sequence for Npra-pro-insuline (SEQ ID NO 6) is custom-synthesized and inserted into pUC119 (NCBI #U07650: National Center for Biotechnology Information Plasmid Database, National Library of Medicine, Building 38A, Bethesda, MD 20894, USA) by Operon Biotechnologies, Inc. (1000 Atlantic Avenue, Suite 108 Alameda , CA 94501, USA).<br>
From this construct the N^-sequence (indicated in bold) which is required is amplified by PCR using the following primer pair. N^-F-Ndel (SEQ ID NO 20) and Ins-R-Sall (SEQ ID NO 22), (5'- CTT TCG TC6 ACT TAT TM TTG CAG TAG TTT TC-3') and the resulting fragment inserted via the newly created restriction sites for A/del and Sail (bold letters) into the vector pET30a. Transformation (see 4.2) into E. coli strain DHSalpha (# 10643-013, Invitrogen catalogue 2003, Invitrogen Life Technologies Corporation, 1600 Faraday Avenue, PO Box 6482 Carlsbad, California 92008), isolation of plasmid DNA from selected clones and DNA sequence analysis verifies S-Np-lns-pET30a. From EDDIE-6H-pETSOa (see for construction under 1.1.2) EDDIE (SEQ ID NO 5) is amplified by PCR using the following primer pain 6H- N"™ -F-Ndel (SEQ ID NO 23), (5'- CTC TCA TAT GCA TCA CCA TCA TCA TCA CGA ACT CAA TCA TTT CGA ACT GCT C-3' and N"10 -R-Sall (SEQ ID NO 21) and the resulting fragment used to replace N"10 via restriction sites for Ndel and Spel (bold letters) in the construct S-Np-lns-pET30a creating 6H-EDDIE-lns-pET30a. To create a suitable cleavage site for N1"0 autoprotease the pro-insulin sequence is amplified from plasmid 6H-EDDIE-lns-pET30a by PCR using the following primer pain SDDIns-F-Spe (SEQ ID NO 24) (5'-GTA ACT AGT TGC AGC GAT GAG TTC GTT AAC CAA CAT CTG TGC-3') and Ins R Sail, (SEQ ID NO 22) and the resulting fragment used to replace the pro-insulin sequence via restriction sites for Spel and Sa/l (bold letters) in the construct 6H-EDDIE-lns-pETSOa to create 6H-EDDIE-SDDIns-pET30a. The sequences of the constructs are verified by DNA sequencing according to standard techniques.<br>
4.2 Transformation<br>
Electrocompetent cells are prepared from one liter of bacterial culture (grown at 37 °C and 225 rpm to OD6oo=0,5 ). The cell suspension is cooled on ice for 15 min (continuous agitation) pelleted (4 °C, 2500 g,10 min) and the supernatant removed. The remaining pellet is resuspend in one liter of deionized water at 4 °C, spun down (4 °C, 2500 g ,10 min) again and washed 2 times in 50 ml de-ionized water (4 °C) with intermittent centrifuging steps (4°C, 2500 g , 10 min). The pellet is finally washed with 50 ml 10% sterilized glycerol solution (4 °C) pelleted (4 °C, 2500 g ,10 min) and resuspended in 2.5 ml 10 % sterilized glycerol solution (4°C), frozen and stored in 40 ul aliquots at -80 °C. One aliquot of electrocompetent cells is thawed on ice, 1ul of ligation reaction containing 5ng DNA added<br>
and transferred without air bubbles to an electroporation cuvette with 1mm electrode gap. Electroporation takes place with a BIO-RAD Gene Pulser™ (Bio-Rad Laboratories Inc., 2000 Alfred Nobel Drive, Hercules, CA 94547, USA; cat n. 1652077, Life Science Research Products 1998) including BIO-RAD pulse controller (Bio-Rad Laboratories Inc., 2000 Alfred Nobel Drive, Hercules, CA 94547, USA; cat. n. 1652098, Life Science Research Products 1998) set to 1,5 kV, 25 uF, 200 Ohms with a time constant longer than 4.4 ms whereby a plasmid constructed as described under 4.1 is transferred into the cell. Immediately thereafter 180 pi TY-broth (1.0% w/v Peptone, 0.7% w/v Yeast extract, 0.25% w/v NaCI) is added and the suspension transferred to a sterile 14ml plastic tube and incubated for 30 min (37 °C, 225 rpm). The suspension is then plated on selection medium. After incubation over night at 37 °C colonies are picked, transferred to 2 ml TY-broth and incubated over night at 37 °C and 225 rpm. 1 ml of the overnight culture is used for plasmid preparation by standard methods and the plasmid preparation subjected to restriction analysis and DNA sequencing. After verification by sequence analysis the plasmid is used for further transformation in expression strains by the method described herein.<br>
4.3	Expression and fermentation<br>
10 ml of an over night expression culture of cells transformed as described above under 4.2 are diluted by 10 with TY-medium (see 1.1.2) and incubated for 30 minutes at 37°C, 225 rpm, followed by induction of protein expression with 1 mM IPT6 (Isopropyl-thiogalactoside) for 2 hours at 37°C, 225 rpm. Cells are harvested by centrifugation at 2500 g for 10 minutes and the pellet is resuspended in 8 ml lysis buffer (20 mM Na2HP04, 75 mM NaCI, 5 mM EDTA, 2 mM MgCfe, pH 8.0). The suspension is then transferred into a precooled pressure cell and incubated at 1380 bar for 5 minutes. After that the valve is slowly opened and the suspension of disrupted cells is poured drop by drop (2-4 drops/10 seconds) into a clean collection tube. After a second passage through the pressure cell the suspension is divided into aliquots of 500 pi and inclusion bodies are isolated by centrifugation at 4°C, 20000 g for 30 minutes and stored at -20°C (supernatant is removed before freezing).<br>
4.4	On column cleavage of insulin<br>
One of these aliquots is resuspended in 30 pi H2O and subsequently dissolved by adding 500 ul of 5M guanidine hydrochloride. After incubation for 40 min at room temperature the inclusion bodies that are dissolved are then applied onto a 500 ul column filled with an immobilized metal affinity matrix, (Quiagen GmbH, Quiagen Strasse 1, D 40724 Hilden, Cat. Nr 30210). After application the column is washed with 5 column volumes (CV) of 5M guanidine hydrochloride and renaturation of the mutated N*"0 is induced by rapid buffer exchange to refolding buffer (20 mM sodium phosphate pH 7.3, 500 mM NaCI, 5% glycerine, 2 mM EDTA). Refolding buffer is applied until no guanidine hydrochloride is detectable in the flow through, afterwards the column is sealed. The sealed column is incubated for at least 80 minutes, then SDD-lns is washed out, simply by applying 1 CV of refolding buffer.<br>
Example 5<br>
Production of a heterologous polypeptide of interest (insulin) by on column refolding,<br>
using the Npf°-derivative with SEQ ID NO 2,3 or 4, respectively<br>
For this experiment a construct analogous to that described in example 4 is used. This fusion polypeptide comprises an N-terminally 6xhistidine tagged mutant form of the pestiviral autoprotease N1*0, (SEQ ID NO 2,3,4 respectively), followed by an SDD-linker (serine, aspartic acid, aspartic acid) and the sequence of pro-insulin.<br>
5.1	Construction of plasmids<br>
The construction of the plasmids is performed analogous to the process described under 4.1.<br>
5.2	Transformation<br>
The transformation of the host cells is performed analogous to the process described under 4.2.<br>
5.3	Expression and fermentation<br>
The expression and fermentation is performed analogous to the process described under 4.3.<br>
5.4 On column cleavage of insulin<br>
The on column cleavage of Insulin is performed analogous to the process described under 4.4 with similar results.<br>
Example 6<br>
Production of a heterologous polypeptide of interest (domain D of protein A from<br>
Staphylococcus auret/s) by refolding, using the N^-derivative with SEQ. ID NO 5<br>
(EDDIE)<br>
For this experiment the construct pET30-EDDIE-sSpA-D is used to express the fusion protein EDDlE-sSpA-D. This fusion protein comprises a mutant form of the pestiviral autoprotease N""0 with (SEQ ID NO 5), (EDDIE) followed by domain D of Staphylococcus aureus protein A.<br>
6.1 Construction of plasmid<br>
A codon optimized DMA sequence for domain D of Staphylococcus aureus protein A, (SEQ ID NO 25):<br>
GCAGACGCACMCAGMTMGTTTAACAAAGACCAGCAGAGCGCATTCTACGAAATTCT GMCATGCCGMTCTGMTGAGGMCMCGTMTGGCnTATTCAGTCTTTAAAAGACG ACCCATCTCAGAGCACCMCGTTCTGGGCGAAGCAAAGAAACTGAACGAATCTCAGGC ACCAAAA<br>
is assembled by PCR of six partially overlapping oligonudeotktes<br>
SpAD1Spe(SEQIDNO26): ATATACTAGTTGCGCAGACGCACAACAGAATAAGTTTAACAAAGACCAGCAG;<br>
SpA-D2 (SEQ ID NO 27): CATGTTCAGAATTTCGTAGAATGCGCTCTGCTGGTCTTTGTTAAACTTAT;<br>
SpA-D3(SEQIDNO28): CATTCTACGAMTTCTGAACATGCCGAATCTGAATGAGGAACAACGTAAT;<br>
SpA-D4(SEQIDNO29): GGGTCGTCTTTTAMGACTGMTAMGCCATTACGTTGTTCCTCATTCAG;<br>
SpA-D5(SEQIDNO30): TCAGTCTTTAAAAGACGACCCATCTCAGAGCACCAACGTTCTGGGCGAAG;<br>
SpA-D6 Sal (SEQ ID NO 31): TTTTGGTGCCTGAGATTCGTTCAGTTTCTTTGCTTCGCCCAGAACGTT<br>
in a 50 pi PCR reaction with 5 units Taq DNA-polymerase (Biotherm Kat. Nr. GC-002, Genecraft GmbH, Raiffeisenstr. 12, 59348 Liidinghausen, Germany), 1x PCR buffer (delivered with Biotherm, Genecraft), 20 nmol each dNTP mixture (GC-013-002, Genecraft) using the following program: initial incubation at 94 °C 3min, 25 cylces of 94°C 30 sec, 54°C 30sec, 688C 30 sec, and final incubation at 68°C for 7 min. 1 pi of the first PCR is directly amplified in a standard 50 ul PCR reaction with 50 pmol of 5'- and 3'-flanking primers (SpA-D1 Spe and SpA-D6_Sal). The success of the gene assembly procedure is analyzed by 1% agarose gel electrophoresis in a manner known per se. The purified sSpA-D PCR product is digested with Spel and Sail and ligated into dephosphorylated pET30-EDDIE-6Ha (for construction see under 1.1.2) according to standard methods. Transformation into £ co// strain DHSalpha (# 10643-013, Invitrogen catalogue 2003, Invitrogen Life Technologies Corporation, 1600 Faraday Avenue, PO Box 6482 Carlsbad, California 92008), is performed analogous to the procedure described under 4.2. Isolation of plasmid DNA from selected clones in a manner known per se and DNA sequence analysis as known in the art verify pET30-EDDIE-sSpA-D.<br>
6.2 Transformation<br>
The transformation of the host cells is performed analogous to the process described under 4.2.<br>
6.3	Expression and Fermentation<br>
The expression and fermentation is performed analogous to the process described under 4.3.<br>
6.4	Cleavage of domain D of protein A from Staphylococcus aureus<br>
Cleavage of domain D of protein A from Staphylococcus aureus is performed analogous to the process described under 1.5.<br>
Example 7<br>
Generation  of the derivative according to SEQ ID NO 32 (EDDIEN35TJ158S;<br>
asparagine 35 replaced by threonine, and threonine 158 replaced by serine):<br>
Starting from the derivative comprising SEQ ID NO 5 (EDDIE) a derivative wherein in addition N35 is replaced by T, and T 158 is replaced by S is constructed by mutational PCR as described in 1.1.2. Two consecutive steps are performed using the primer pairs: 5'_N35T (5'CTC TTT TTG GGA CCC CGT CCG AAG TG3) and 3'_N35T (5'CAC TTC GGA CGG GGT CCC AAA AAG AG3') as well as E 5'_T158S (5'GGA CCC GTA ACA GCA CTA ACT GTC C3') and E 3'_T158S (5'GGA CAG TTA GTG CTG TTA CGG GTC C3'). The resulting fragment is used to replace EDDIE in the vector 6H-EDDIE-lns-pet30a via the Ndel and Spel restriction sites. The DMA sequence of derivative EDDIEN35T.T158S is verified by DMA sequencing.<br>
Example 8<br>
Production of a neterologous polypeptide of interest (proinsulin), using the N1"0-<br>
derivative with SEQ ID NO 33:<br>
8.1 Generation of the derivative according to SEQ ID NO 33:<br>
1ng of EDDIEN35T,T158S-lns-pet30a is used for random mutagenesis with the GeneMorph PCR II random mutagenesis kit (Stratagene, 11011 North Torrey Pines Road, La Jolla, CA 92037, USA, Cat#200550 catalog 2005,). In detail, 5ul 10x buffer (GeneMorph 11), 1ul 40 mM dNTP-mix (GeneMorphll), 2,5ul (103 ng each) primer-mix IF-Np-Nde-F (5'-AAG GAG ATA TAC ATA TGG AAC TCA ATC ATT TCG AAC TG-3') and IF-Np-lns-Spe-R (5'-TAA CGA AGO AAC TAG TGA CCC ACA GTG GAG AGT TAG T-31), 1ul Mutazyme® (GeneMorphll), 1ng EDDIEN35T, T158S-ins-pet30a, A. dest. ad 50ul. This mixture is subjected to the following PCR-Program: 1 min 94°C; step 1 to 30:30 sec 94°C, 30 sec 55°, 1 min 72°C; final step: 10 min 72°C; hold 10°C. The reaction with the given amount of DMA leads to 4 mutations per N1*0 gene in average. After PCR the reaction mix is purified using the QIAquick PCR Purification Kit (Qiagen GmbH, Qiagen Strasse 1, D 40724 Hilden, Cat#28104, Qiagen product guide 2005) according to the manufacturer's recommendations.<br>
8.2	Construction of plasmids<br>
The construction of the plasmids is performed analogous to the process described under 4.1. The fragments generated according to 8.1 are used to replace the N""0 gene in the plasmid EDDIE -Ins-pet30a via the Ndel and Spel restriction sites thus creating a random mutagenesis pool of N1*0 derivatives.<br>
8.3	Transformation<br>
The transformation of the host cells is performed analogous to the process described under 4.2.<br>
8.4	Expression and fermentation<br>
The expression and fermentation is performed analogous to the process described under 4.3.<br>
8.5	Cleavage analysis<br>
Cleavage analysis is conducted as described under 1.5.<br>
Example 9<br>
Production of a heterologous polypeptide of interest (proinsulin) using the N1"0-derivative according to SEQ ID NO 32:<br>
Alternatively the derivative according to SEQ ID NO 32 can be used in the process described<br>
above. The derivative is produced as described in example 7.<br>
The steps described under 8.2 to 8.5 are performed analogous for the derivative with SEQ<br>
ID NO 32.<br>
Example 10<br>
10. 1. Generation of Threonine-Serine derivatives of EDDIE:<br>
To further increase to polarity of EDDIE the amino acids Threonine (T) in positions 109, 114, 155, 158 and Glutamine (Q) 143 are replaced by Serine (S) by gene assembly. To this the gene for EDDIE is split into the following set of 15 overlapping oligonucleotides and assembled by PCR as described in 6.1 :<br>
Primer List:<br>
The resulting fragment is used to replace the N1"0 gene in s-Np-6H-pet30a via the Ndel and Spel restriction sites. The DMA sequence of derivative 92 is verified by DMA sequencing. Transformation into bacterial cells and expression and fermentation in done as described under 4.2 and 4.3. Cleavage analysis is conducted as described under 1.5.<br>
no.Amino acid changes in N1"0 derivative<br>
92 R53E, G54D, R57E, A109S, C112E, V114S, C134E, C138E, L143S. I155S, F158S<br>
SEQ ID 92:<br>
(I)-MELNHFELLYKTSKQKPVGVEEPNA'DTAGRPLFGNPSEVHPQSTLKLPHDRGEDDIETT LRDLPRKGDCRSGNHLGPVSGIYIKPGPVYYQDYTGPNA^RAPLEFFDESQFEESTKRIGR VTGSDGKLYHIWEVDGEILLKSAKRGTPRTLKWSRNSTNCPLWVTSC-(168).<br>
EDDIE 143 derivatives:<br>
To exchange the amino acid S143 with a number of other polar amino acids (D, G, H, K, N, Q) oligonucleotide e14 is repaced by the degenerated oligonucleotide e14vaw containing the nucleotide composition VAW for the codon in position 143 (V: ACG; W:AT). Since e14vaw is a reverse oligonucleotide it contains the reverese complementary triplett WTB.<br>
e14vav«CCATTTCAGGGTGCOAGGGGTTCCGCGCTTCG(a«rn3TTTCAGCAGGATCTCXKX^TCAAC<br>
The same gene assembly process and insertion in s-Np-6H-pet30a resulted in the mutants described in table of mutants.<br>
Transformation into bacterial cells and expression and fermentation in done as described under 4.2 and 4.3. Cleavage analysis is conducted as described under 1.5.<br>
Table of mutants:<br>
(Table Removed)<br>
SEQ ID 95:<br>
 (1)-MELNHFELLYKTSKQKPVGVEEP\ADTAGRPLFGNPSEVHPQSTLKLPHDRGEDDIETT<br>
URDLPRKGDCRSGNHLGPVSGIYIKPGPVYYQDYTGPNAHRAPLEFFDESQFEESTKRIGR<br>
VTGSDGKLYHIYVEVDGEILLKNAKRGTPRTLKWSRNSTNCPLWVTSC-(168).<br>
SEQ ID 96:<br>
(1)-MELNHFELLYKTSKQKPVGVEEPVYDTAGRPLFGNPSEVHPQSTLKLPHDRGEDDIETT LRDLPRKGDCRSGNHLGPVSGIYIKPGPVYYQDYTGPVYHRAPLEFFDESQFEESTKRIGR VTGSDGKLYHIYVEVDGEILLKDAKRGTPRTLKWSRNSTNCPLWVTSC-(168).<br>
SEQ ID 97:<br>
(1J-MELNHFEUYKTSKQKPVGVEEPWDTAGRPLFGNPSEVHPQSTLKLPHDRGEDDIETT LRDLPRKGDCRSGNHLGPVSGIYIKPGPVYYQDYTGPVYHRAPLEFFDESQFEESTKRIGR VTGSDGKLYHIYVEVDGEILLKHAKRGTPRTLKWSRNSTNCPLWVTSC-(168).<br>
SEQ ID 98:<br>
(1)-MELNHFELLYKTSKQKPVGVEEPVYDTAGRPLFGNPSEVHPQSTLKLPHDRGEDDIETT LRDLPRKGDCRSGNHLGPVSGIYIKPGPVYYQDYTGPVYHRAPLEFFDESQFEESTKRIGC VTGSDGKLYHIWEVDGEILLKQAKRGTPRTLKWSRNSTNCPLWVTSCK168).<br>
Construction of sNp-FVN-6H-pet30a<br>
To insert the peptide FVN-6H containing the first three amino acids of insulin and the 6His tag (FVNVDKLAAALEHHHHHH) NpfD is amplified from plasmid s-Np-6H-pet30a with the primer pair sNp FVN R Sal (5'-GAG AGT CGA CGT TM CGA AGO AAC TAG TGA CCC ACA GTG-31) and N^ -F-Ndel primer (SEQ ID NO 20) by a standard PCR reaction and the resulting fragments used to replace NPRO-6H via the restriction sites Ndel and Salt by standard procedures creating plasmid sNp-FVN-6H-pet30a.<br>
Example 11<br>
Production of a heterologous polypeptide of interest (double domain D of Staphylococcus aureus protein A) using the N^-derivative according to SEQ ID NO 5 (EDDIE) and the NPRO-derivative containing the amino acid substitutions C134E and C138E<br>
11.1 Construction of pET30-6H-EDDIE-sSpA-D-sSpA-D<br>
The domain D of Staphylococcus aureus protein A generated by gene assembly (see<br>
Example 6) is amplified by PCR from pET30-EDDIE-sSpA-D (6.1) by use of the primer pair<br>
(SpA-D1;GCAGACGCACAACAGAATAAGTTTAAC	and	SpA-D6;<br>
TTTTGGTGCCTGAGATTCGTTCAGTTTCTTTGCTTCGCCCAGAACGTT) using essentially the same PCR reaction conditions as in Example 6 and subjected to a domain assembly process. In a first step single domains are linked together by PCR with a link-primer pair (SpA-Dlink2RC;<br>
CTGCTGGTCTTTGTTAMCTTATTCTGTTGTGCGTCTGCTTTTGGTGCCTGAGATTCGTT<br>
C	and	SpA-DLink;<br>
GAACGAATCTCAGGCACCAAAAGCAGACGCACAACAGAATAAGTTTAACAAAGACCAGC<br>
AG). In this PCR reaction the link-primer concentration is reduced to 0.5 pmol, while the that<br>
of template (single domain D) is elevated to 10 - 25 pmol. The reverse-link-primer attaches a<br>
reverse complementary sequence of the 5' end to the 3'end of the monomer and the forward<br>
link-primer attaches a reverse complementary sequence of the 3' end to the 5* end,<br>
respectively. These new 5' and 3' linking-ends of domain D anneal with the complementary<br>
3'and 5' iinking-sequences of another domain D, respectively. Hence, many units of one<br>
specific domain are linked together producing synthetic genes with multiple repeats of<br>
domain D. To allow subsequent isolation and cloning and to get rid of the 5' and 3' ends<br>
attached during the first PCR reaction, anchor and restriction sites are incorporated by a<br>
second	PCR	with	the	adaptor-primer pair:	fish-R-Sal-<br>
SpA;GATCTTCAGGTTGGTCMGTGGGTCGACTTATTTTGGTGCCTGAGATTCGTTCAGT<br>
TTC	and	fish2-F-Spe-SpA;	gagaGAAGAgTGGCTACTGTAgAG<br>
ACTAGTTGCGCAGACGCACAACAGAATAAGTTTMC. One tenth of the first PCR reaction is directly added to the second PCR mixture containing 0.5 pmol adaptor-primer. The reaction products are separated by agarose gel electrophoresis and the fragments containing the double domain D extracted from the gel by QIAquick Gel Extraction Kit. The double domain D genes are amplified by PCR using 50 pmol of anchor-primers (fish2-<br>
F;gagaGAAGAgTGGCTACTGTAgAG and fish-R;GATCTTCAGGTTGGTCAAGTGG), purified by gel electrophoresis and digested with Spel/Sall and cloned into pET30-6H-EDDIE-lns digested with the same enzymes, which results in a replacement of the sequence of proinsulin with the double domain D sequence, thereby giving rise to the construct pET30-6H-EDDIE- sSpA-D-sSpA-D (fusion of EDDIE with with double domain D of Staphyiococcus aureus protein A).<br>
11.2	Expression of pET30-6H-EDDIE-sSpA-D-sSpA-D<br>
Transformation into bacterial cells, expression and fermentation is done as described under 4.2 and 4.3. Cleavage analysis conducted as described under 1.5 reveales, that besides of the cleaved N^-EDDIE protein and the double domain D the majority of the uncleaved fusion protein (about 90 percent) is also found in the soluble fraction. Therefore it is considered to use the N1"0 derivative containing the amino acid substitutions C134E and C138E which showed very tow in vivo cleavage rate.<br>
11.3	Construction of pET30-Npr°C134E/C138E-sSpA-D-sSpA-D<br>
The NPROC134E/C138E DNA sequence is amplified with the primer pair IF Np-Nde-F (5rAAGGAGATATACATATGGAACTCAATCATTTCGAACTG3') and IF Np SpAD-Spe-R (5'CGTCTGCGCMCTAGTGACCCACAGTGGACAGTTAGT3*) cut with the restriction enzymes Ndel/Spel and inserted into the vector pET30-6H-EDDIE- sSpA-D-sSpA-D digested with Ndel/Spel thereby replacing 6H-EDDIE with NPROC134E/C138E and giving rise to the construct pET30-NPROC134eC138E-sSpA-D-sSpA-D.<br>
11.4	Expression of NPRO C134E, C138E-sSpA-D-sSpA-D<br>
Transformation into bacterial cells, expression and fermentation is done as described under 4.2 and 4.3. Cleavage analysis conducted as described under 1.5 reveales, that most of the uncleaved fusion protein was found in the insoluble fraction after cell disruption via French Press. This result shows that by use of different N"™ derivatives the amount of in vivo cleavage rates and direction of the expression of the fusion proteins into inclusion bodies can be controlled. Moreover, refolding of Npr°C134E/C138E-sSpA-D-sSpA-D shows besides almost zero in vivo cleavage still approx. 33 % cleaved products in vitro.<br>
Example 12:<br>
Production of a heterologous polypeptide of interest (JAC, a direct target of oncogenic transcription factor Jun) using the NPRO-derivative with SEQ ID NO 5 (EDDIE)<br>
12.1 Construction of 6H-EDDIE-JAC<br>
The gene for JAC, a direct target of oncogenic transcription factor Jun which is involved in cell transformation and tumorigenesis, is amplified from a cDNA clone pAC01(Markus Hartl et. al. JAC, a direct target of oncogenic transcription factor Jun, is involved in cell transformation and tumorigenesis. PNAS 98, 13601-13606, 2001) by PCR with the oligonudeotide primers JACI(GATCACTAGTTGCATGCCCAACGGAGG) and JAC2 (GATCGTCGACTTAGTTGCCACAGCCACA) containing the Spel and Sail restriction sites according to the protocol described in 1.1.1. The resulting fragment is used to replace the insulin gene from 6H-EDDIE-lns-pet30a to create 6H-EDDIE-JAC-pet30a. The sequences of the constructs are verified by DNA sequencing according to standard techniques.<br>
Transformation into bacterial cells and expression and fermentation in done as described under 4.2 and 4.3. Cleavage analysis is conducted as described under 1.5.<br>
Example 13:<br>
Production of a heterologous polypeptide of interest (Interferon alpha 1, IFNA1) using<br>
the NPRO-derivative with SEQ ID NO 5 (EDDIE)<br>
13.1 Construction of 6H-EDDIE-slFNA1-pet30a:<br>
The gene encoding IFNA1 (gene bank accession number NM_024013) is assembled by PCR as described (10.1) using the following oligonudeotide set<br>
IFNA1-1 ATA TAG TAG TTG CAT GGC ACC GAC CTC T<br>
IFNA1-2 AAA TGG CAT TGC AGC TTA ACA GAA CTA ATG CCG TCA GAA AGG CAG AGG TCG GTG CCA TGC<br>
IFNA1-3 QTT CTG TTA AGO TOC AAT GCC ATT TGT TCT TTA GGC TGC GAT CTG CCA CM ACC CAC TCT IFNA1-4 CAT TTG TGC CAG CAG AGO TAA GGC ACG CGT ATG GGC CAG AGA GTG GGT TTG TOG CAG ATC IFNA1 -5 CTT ACG TCT GOT GGC ACA AAT GCG TCG CAT TAG CCC ATT CTC TTG TCT GGA TCA TCG CCG IFNA1-6 TGG TTA CCA CCA AAG GCC TCG TGC GGA GAG CCG AAA TCA CGG CGA TGA TCC AGA CAA GAG IFNA1-7 GAG GCC TTT GGT GGT AAC CAA GTC CAA AAG GCC CAG GCA ATG GCC TTA GTG CAT GAG ATG IFNA1-8 TGC CCT CCG TGC TGA ATA ACT GAA AGO TCT GTT GCA GCA TCT CAT GCA CTA AGG CCA TTG IFNA1-9 GTT ATT CAG CAC GGA GGG CAG CGC AGC GGC CTG GAA TGA AAG CTT ACT GCA CCA ATT TTG IFNA1-10 TCT AAA TCG CGC ACT TGT TGG TCC AGA CCG GTA CAA AAT TGG TGC ACT AAG CTT TCA TTC IFNA1-11 ACC AAC AAC TGC GCG ATT TAG AAG CCT GCG TCA TGC AAG AAG CGG GCT TAG AAG GTA CCC IFNA1-12 ATA CTT GCG CAC CGC TAA AAT AGA GTC TTC CTC TAA TAA TGG GGT ACC TTC TAA GCC CGC IFNA1-13 CTC TAT TTT AGC GGT GCG CAA GTA TTT CCA TCG TTT AAC CTT ATA CTT ACA GGA AAA ATC IFNA1-14 ACG ATC TCC CAT GCG CAC GGG CTG TAA GAT TTT TCC TGT AAG TAT AAG GTT AAA CGA TGG IFNA1-15 GTG CGC ATG GGA GAT CGT TCG CGC GGA GGT CAT GCG TAG CTT CAG CAG CTC TCG TAA TCT IFNA1-16 ATA TGT CGA CTT ATT CCT TCT TAC GCA GAG GGT CTT GCA GAT TAC GAG AGC TGC TGA AGC<br>
The resulting fragment is digested with the restriction enzymes Spel and Sail and used to replace the insulin gene from 6H-EDDIE-lns-pet30a to create 6H-EDDIE-slFNA1-pet30a. The sequences of the constructs are verified by DNA sequencing according to standard techniques.<br>
Transformation into bacterial cells and expression and fermentation in done as described under 4.2 and 4.3. Cleavage analysis is conducted as described under 1.5.<br>
Example 14:<br>
Production of a heterologous polypeptide of interest (Hepcidin), using 6H-EDDIE-lns:<br>
The DNA sequence of hepcidin is amplified by PCR from the template "huhep in pCR2.1" (S. Ludwiczek, Department of Internal Medicine, University of Innsbruck) using the primer pair "Hep25 F Spe"(5'-TCG ACT AGT TGC GAG ACC CAC TTC CCC ATC-3')'/"Hep R Sal" (5'-ATC GTC GAC TTA CGT CTT GCA GCA CAT CCC AC-3').<br>
The resulting DNA fragment is digested by Spel/Sall and cloned into pET30-6H-EDDIE-lns digested with the same enzymes, which results in a replacement of the sequence of proinsulin with the hepcidin25 sequence, thereby giving rise to the construct pET30-6H-EDDIE-Hep25 (fusion of EDDIE with mature hepcidin).<br>
Transformation of pET30-6H-EDDIE-Hep25 into E. coli BL21-CodonPlus(DE3)-RIL (Cat. Nr. 230245, Stratgene, 11011 N.Torrey Pines Road, La Jolla, CA 92037, USA, 2004 Catalog) bacterial cells, expression and fermentation is done as descnbed under 4.2 and 4.3. After that cell-harvest, cell-disruption, isolation of IBs, renaturation and cleavage analysis is conducted as described under 1.5. The results show about 80% cleavage of EDDIE-Hepcidin25.<br>
<br>
<br>
<br>
<br>
<br>
<br>
<br>
<br>
<br>
<br>
<br>
<br>
<br>
<br>
(New) Claims:<br>
1.	The derivative of an autoprotease NPRO of classical swine fever virus (CSFV), wherein at least<br>
one cysteine residue of the naturally occuring autoprotease NPRO of CSFV selected from the<br>
group consisting of C112, C134 and C138, is replaced by a glutamic acid residue.<br>
2.	The derivative of the autoprotease NPRO of CSFV according to claim 1 , comprising the follow<br>
ing amino acid sequence:<br>
SEQ ID NO 2:<br>
(I)-MELNHFELLYKTSKQKPVGVEEPVYDTAGRPLFGNPSEVHPQSTLKLPHDRGRG-<br>
DIRTTLRDLPRKGDCRSGNHLGPVSGIYIKPGPVYYQDYTGPVYHRAPLEFF-<br>
DEAQFEEVTKRIGRVTGSDGKLYHIWEVDGEILLKLAKRGTPRTLKWIRNFTNCPLWVTSC-<br>
(168)<br>
3.	The derivative of an autoprotease NPRO of CSFV according to one of the claims 1 or 2,<br>
wherein in addition to the replaced cysteine residues at least one basic amino acid residue is<br>
replaced by an acidic amino acid residue.<br>
4.	The derivative of an autoprotease NPRO of CSFV according to claim 3, wherein in addition to<br>
the replaced cysteine residues the following amino acids are exchanged: R 53 with E, G 54<br>
with D, R 57 with E, and L 143 with Q.<br>
5.	The derivative of an autoprotease NPRO of CSFV according to claim 4, comprising the following<br>
amino acid sequence:<br>
SEQ ID NO 3:<br>
(I)-MELNHFELLYKTSKQKPVGVEEPVYDTAGRPLFGNPSEVHPQSTLKLPHDRGEDDI-ETTLRDLPRKGDCRSGNHLGPVSGIYIKPGPVYYQDYTGPVYHRAPLEFFDEAQFEEVTKRI-GRVTGSDGKLYHIYVEVDGEILLKQAKRGTPRTLKWIRNFTNCPLWVTSC-(168).<br>
6.	The derivative of an autoprotease NPRO of CSFV according to one of the claims 1 or 2,<br>
wherein in addition to the replaced cysteine residues at least one hydrophobic amino acid<br>
residue is replaced by a hydrophilic residue.<br>
7.	The derivative of an autoprotease NPRO of CSFV according to claim 6, wherein in addition to<br>
the replaced cysteine residues the following amino acids are replaced by T: A109, V114, 1155 and F158.<br>
8. The derivative of an autoprotease NPRO of CSFV according to claim 7 comprising the following amino acid sequence:<br>
SEQIDNO4:<br>
(I)-MELNHFELLYKTSKQKPVGVEEPWDTAGRPLFGNPSEVHPQSTLKLPHDRGRG-<br>
DIRTTLRDLPRKGDCRSGNHLGPVSGIYIKPGPVYYQDYTGPVYHRAPLEFFDE-<br>
TQFEETTKRIGRVTGSDGKLYHIYVEVDGEILLKLAKRGTPRTLKWTRNTTNCPLWVTSC-<br>
(168)<br>
9. The derivative of an autoprotease NPRO of CSFV according to one of the claims 1 or 2, wherein in addition to the replaced cysteine residues the following amino acids have been exchanged: A109, V114, 1155 and F158 by T, R 53 with E, G 54 with D, R 57 with E, and L 143 with Q.<br>
1 0. The derivative of an autoprotease N"™ of CSFV according to claim 9 comprising the following amino acid sequence:<br>
SEQ ID NO 5:<br>
(I)-MELNHFELLYKTSKQKPVGVEEPVYDTAGRPLFGNPSEVHPQSTLKLPHDRGEDDI-<br>
ETTLRDLPRKGDCRSGNHLGPVSGIYIKPGPVYYQDYTGPWHRAPLEFFDE-<br>
TQFEETTKRIGRVTGSDGKLYHIYVEVDGEILLKQAKRGTPRTLKWTRNTTNCPLWVTSC-<br>
(168).<br>
11 . The derivative of an autoprotease NPRO of CSFV according to claim 9, wherein in addition the following amino acids have been exchanged: N35 by T and T158 by S.<br>
12. The derivative of an autoprotease NPRO of CSFV according to claim 11 comprising the following amino acid sequence:<br>
SEQ ID NO 32:<br>
(I)-MELNHFELLYKTSKQKPVGVEEPVYDTAGRPLFGTPSEVHPQSTLKLPHDRGEDDI-<br>
ETTLRDLPRKGDCRSGNHLGPVSGIYIKPGPVYYQDYTGPVYHRAPLEFFDE-<br>
TQFEETTKRIGRVTGSDGKLYHIYVEVDGEILLKQAKRGTPRTLKWTRNSTNCPLWVTSC-<br>
(168).<br>
13. The derivative of an autoprotease NP™ of CSFV according to claim 1 1 , wherein in addition the following amino acids have been exchanged: A28 by E, S71 by F and R150 by H.<br>
14. The derivative of an autoprotease N1*0 of CSFV according to claim 13 comprising the following amino acid sequence: SEQ ID NO 33:<br>
(I)-MELNHFELLYKTSKQKPVGVEEPWDTEGRPLFGTPSEVHPQSTLKLPHDRGEDDIET-<br>
TLRDLPRKGDCRFGNHLGPVSGIYIKPGPVYYQDYTGPVYHRAPLEFFDETQFEETTKRI-<br>
GRVTGSDGKLYHIWEVDGEILLKQAKRGTPHTLKWTRNSTNCPLvWTSC-(168).<br>
15.	The derivative of an NPRO of CSFV according to claim 1 wherein at least one of the following<br>
amino acids have been replaced in addition to the replaced cysteine residues: arginine (R)<br>
53, glycine (G) 54, arginine (R) 57, threonine (T) 109,114,155,158 and leucine (L) 143.<br>
16.	The derivative of an N1*0 of CSFV according to claim 1 wherein at least one of the following<br>
amino acids have been replaced in addition to the replaced cysteine residues: arginine (R)<br>
53 with glutamic acid (E), glycine (G) 54 with aspartic acid (D), arginine (R) 57 with glutamic<br>
acid (E), threonine (T) 109, 114, 155, 158 with serine (S) and leucine (L) 143 with glutamine<br>
(Q) or asparagine (N) or aspartic acid (D) or serine (S) or histidine.<br>
17.	The derivative of an autoprotease N"™ of CSFV according to claim 1 comprising the follow<br>
ing amino acid sequence:<br>
SEQ ID NO 92:<br>
(I)MELNHFELLYKTSKQKPVGVEEPVYDTAGRPLFGNPSEVHPQSTLKLPHDRGEDDI-<br>
ETTLRDLPRKGDCRSGNHLGPVSGIYIKPGPVYYQDYTGPVYHRAPLEFFDE-<br>
SQFEESTKRIGRVTGSDGKLYHIYVEVDGEILLKSAKRGTPRTLKWSRNSTNCPLWVTSC-<br>
(168).<br>
18.	The derivative of an autoprotease NPRO of CSFV according to claim 1 comprising the follow<br>
ing amino acid sequence:<br>
SEQ ID NO 95:<br>
(I)MELNHFELLYKTSKQKPVGVEEPVYDTAGRPLFGNPSEVHPQSTLKLPHDRGEDDI-ETTLRDLPRKGDCRSGNHLGPVSGIYIKPGPVYYQDYTGPVYHRAPLEFFDE-SQFEESTKRIGRVTGSDGKLYHIWEVDGEILLKNAKRGTPRTLKWSRNSTNCPLWVTSC-(168).<br>
19.	The derivative of an autoprotease NPRO of CSFV according to claim 1 comprising the follow<br>
ing amino acid sequence:<br>
SEQ ID NO 96:<br>
(I)MELNHFELLYKTSKQKPVGVEEPVYDTAGRPLFGNPSEVHPQSTLKLPHDRGEDDI-<br>
ETTLRDLPRKGDCRSGNHLGPVSGIYIKPGPVYYQDYTGPWHRAPLEFFDE-<br>
SQFEESTKRIGRVTGSDGKLYHIWEVDGEILLKDAKRGTPRTLKWSRNSTNCPLWVTSC-<br>
(168).<br>
20. The derivative of an autoprotease NPRO of CSFV according to claim 1 comprising the following amino acid sequence:<br>
SEQ ID NO 97:<br>
(I)MELNHFELLYKTSKQKPVGVEEPWDTAGRPLFGNPSEVHPQSTLKLPHDRGEDDI-<br>
ETTLRDLPRKGDCRSGNHLGPVSGIYIKPGPNATQDYTGPWHRAPLEFFDE-<br>
SQFEESTKRIGRVTGSDGKLYHIWEVDGEILLKHAKRGTPRTLKWSRNSTNCPLWVTSC-<br>
(168).<br>
21. The derivative of an autoprotease N"™ of CSFV according to claim 1 comprising the following amino acid sequence:<br>
SEQ ID NO 98:<br>
(I)MELNHFELLYKTSKQKPVGVEEPVYDTAGRPLFGNPSEVHPQSTLKLPHDRGEDDI-<br>
ETTLRDLPRKGDCRSGNHLGPVSGIYIKPGPVYYQDYTGPVYHRAPLEFFDE-<br>
SQFEESTKRIGCVTGSDGKLYHIYVEVDGEILLKQAKRGTPRTLKWSRNSTNCPLWVTSC-<br>
(168).<br>
22. A process for the recombinant production of a heterologous polypeptide of interest, comprising,<br>
(i) cultivation of a bacterial host cell which is transformed with an expression vector which comprises a nucleic acid molecule which codes for a fusion polypeptide, the fusion polypeptide comprising a derivative of an autoprotease N*"™ of Pestivirus according to any one of claims 1 to 21, and a second polypeptide which is connected to the first polypeptide at the C-terminus of the first polypeptide in a manner such, that the second polypeptide is capable of being cleaved from the fusion polypeptide by the autoproteolytic activity of the first polypeptide, said second polypeptide being a heterologous polypeptide, wherein cultivation occurs under conditions which cause expression of the fusion polypeptide and formation of corresponding cytoplasmic inclusion bodies, (ii) isolation of the inclusion bodies from the host cell, (iii) solubilization of the isolated inclusion bodies,<br>
(iv) induction of autoproteolytic cleavage of the heterologous polypeptide of interest from the fusion polypeptide, and<br>
 (v) isolation of the cleaved heterologous polypeptide of interest.<br>
23.	Use of a derivative of the autoprotease N1™ of CSFV according to one of the claims 1 to 21<br>
in a process according to claim 22.<br>
24.	The process according to claim 22, wherein the fusion polypeptide comprises a derivative of<br>
an autoprotease NPRO of CSFV, according to one of the claims 1 to 21.<br>
25.	Use of a derivative of the autoprotease NPRO of CSFV according to one of the claims 2 to<br>
24 in a process according to claim 1.<br>
26.	The process according to claim 1, wherein the fusion polypeptide comprises a derivative<br>
of an autoprotease NPRO of CSFV, according to one of the claims 2 to 24.</td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/viewdoc.aspx?id=2DusEoMeHE4QyAivsvY7Gg==&amp;amp;loc=+mN2fYxnTC4l0fUd8W4CAA==" target="_blank" style="word-wrap:break-word;">http://ipindiaonline.gov.in/patentsearch/GrantedSearch/viewdoc.aspx?id=2DusEoMeHE4QyAivsvY7Gg==&amp;amp;loc=+mN2fYxnTC4l0fUd8W4CAA==</a></p>
		<br>
		<div class="pull-left">
			<a href="272566-polynucleotide-encoding-acetolactate-synthase.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="272568-apparatus-and-method-for-fast-simulation-of-manufacturing-effects-during-integrated-circuit-design.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>272567</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>7911/DELNP/2007</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>16/2016</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>15-Apr-2016</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>08-Apr-2016</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>12-Oct-2007</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>SANDOZ AG.</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td>LICHTSTRASSE 35, CH-4056 BASEL SWITZERLAND.</td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>ACHMULLER CLEMENS</td>
											<td>SCHNEEBURGGASSE 59, A-6020 INNSBRUCK AUSTRIA</td>
										</tr>
										<tr>
											<td>2</td>
											<td>WECHNER PHILIPP</td>
											<td>LINDENSTRASSE 16, A-6020 INNSBRUCK AUSTRIA</td>
										</tr>
										<tr>
											<td>3</td>
											<td>AUER BERNHARD</td>
											<td>HORTNAGLSTRASSE 20,A-6020 INNSBRUCK AUSTRIA</td>
										</tr>
										<tr>
											<td>4</td>
											<td>PODMIRSEG SILVIO</td>
											<td>HOTTINGER AU 16, A-6020 INNSBRUCK AUSTRIA</td>
										</tr>
										<tr>
											<td>5</td>
											<td>WERTHER FLORIAN</td>
											<td>JAHNSTRASSE 23, A-6020 INNSBRUCK AUSTRIA</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>C12N 15/62</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>PCT/AT2006/000165</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td>2006-04-25</td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td>0605379.7</td>
									<td>2006-03-16</td>
								    <td>U.K.</td>
								</tr>
								<tr>
									<td>2</td>
									<td>0508435.5</td>
									<td>2005-04-26</td>
								    <td>U.K.</td>
								</tr>
								<tr>
									<td>3</td>
									<td>0508434.8</td>
									<td>2005-04-26</td>
								    <td>U.K.</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/272567-recombinant-proteins-and-production-by-autoproteolytic-cleavage-of-a-fusion-protein by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 00:08:18 GMT -->
</html>
