<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/272773-a-mehtod-of-producing-r-r-montain-a-salt-of-r-r-montain-or-a-combination-thereof by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 01:09:22 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 272773:A MEHTOD-OF PRODUCING R,R-MONTAIN-A SALT OF R,R MONTAIN, OR A COMBINATION THEREOF</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">A MEHTOD-OF PRODUCING R,R-MONTAIN-A SALT OF R,R MONTAIN, OR A COMBINATION THEREOF</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>ALDOLASES, NUCLEIC ACIDS ENCODING THEM AND METHODS FOR MAKING AND USING THEM ABSTRACT This invention relates to polypeptides having aldolase activity, including pyruvate activity such as, without limitation, HMG find/or K.HG aldolase activity, polynucleotides encoding these polypeptides, and methods of making and using these polynucleotides and polypeptides, In some embodiments, the invention is directed to polypeptides having aldolase activity, including pyruvate activity such as, without limitation, HMG and/or KHG aldolase activity, including thermostable and thermotoierant activity, and polynucleotides encoding these enzymes, and making and using these polynucleotides and polypeptides. The polypeptides in accordance with the invention can be used in a variety of pharmaceutical, agricultural and industrial contexts. In some embodiments, the invention provides polypeptides and biosynthetic pathways that are useful m the production of R-2-hydroxy 2-(indol-3ylmethyl)-4-keto glutaric acid (R-MP) and certain stereoisomers of raonatin, such as R,R and S,R monatin, and salts thereof, as well as certain stereoisomers of monatin derivatives, such as the R,R and S,R configurations, and salts thereof.</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td> <br>
<br>
ALDOLASES, NUCLEIC ACIDS ENCODING THEM AND METHODS FOR<br>
MAKING AND USING THEM<br>
Field in accordance with the invention<br>
[0001]	This invention relates to molecular and cellular biology and biochemistry.  More<br>
specifically, the invention relates to polypeptides having aldolase activity, polynucleotides encoding these polypeptides, and methods of making and using these polynucleotides and polypeptides.<br>
BACKGROUND OF THE INVENTION<br>
[0002]	Monatin is a high-intensity sweetener having the chemical formula:<br>
 <br>
[0003]	Monatin includes two chirai centers leading to four potential stereoisomeric<br>
configurations. The R,R configuration (the "R,R stereoisomer" or "R,R monatin"); the S,S configuration (the "S,S stereoisomer" or "S,S monatin"); the R,S configuration (the "R,S stereoisomer" or "R,S monatin"); and the S,R configuration (the "S,R stereoisomer" or "S,R monatin"). As used herein, unless stated otherwise, the term "monatin" is used to refer to compositions including all four stereoisomers of monatin, compositions including any combination of monatin stereoisomers, (such as a composition including only the R,R and S,S, stereoisomers of monatin), as well as a single isomeric form.<br>
[0004]	For purposes of this disclosure, the monatin carbon backbone will be numbered as<br>
illustrated above, with the carbon directly covalently attached to the alcohol group being identified as the 2-position carbon and the carbon directly covalently attached to the amino group being identified as the 4-position carbon. Consequently, references herein to R,R monatin, S,S monatin, R,S monatin, and S,R monatin mean: 2R,4R monatin, 2S,4S monatin, 2R,4S monatin, and 2S,4R monatin, respectively, unless otherwise indicated.<br>
[0005]	It should be noted that in the literature, the monatin carbon backbone has also<br>
been numbered using an alternative convention, with the carbon attached to the alcohol<br>
 <br>
group being the 4-position carbon, and the carbon attached to the amino group being the 2-position carbon. Accordingly, for example, references to 2S,4R monatin in this disclosure would be the same as references to 2R,4S monatin in literature using the alternative numbering convention.<br>
[0006]	Furthermore, because of various naming conventions, monatin is known by a<br>
number of altemative chemical names, including: 2-hydroxy-2-(indol-3-yImethyl)-4-aminoglutaric acid; 4-amino-2-hydroxy-2-(lH-indol-3'ylmethy!)-pentanedioic acid; 4-hydroxy-4-(3-indolylmethyl)glutamic acid; and, 3-{I-amino-l,3-dicarboxy-3-hydroxy-but-4-yl)indole.<br>
[0007]	Certain isomeric fomis of monatin can be found in the bark of roots of the<br>
Schlerochiton ilicifolius plant located in the Transvaal Region of South Afiica. U.S. Patent Application Nos. 10/422,366 ("the '366 Application") and 10/979,821 ("the '821 Application"), which are hereby incorporated by reference, disclose, inter alia, polypeptides, pathways, and microorganisms for in vitro and in vivo production of monatin.<br>
SUMMARY<br>
[0008]	The   invention   provides   polypeptides   havmg   aldolase   activity   (hereinafter<br>
"aldolases"), including pyruvate aldolase activity such as, without limitation, HMG and KHG aldolase activity, polynucleotides encoding the polypeptides, and methods for making and using the polypeptides and polynucleotides. In some embodiments, the invention also provides compositions (such as pharmaceutical compositions, fuel and fuel additive compostions, foods and food additives, beverage and beverage additives, feeds and feed additives, drugs and drug additives, dietary supplements) comprising the polypeptides or polynucleotides in accordance with the invention. These compositions can be formulated in a variety of forms, sixch as as tablets, gels, pills, implants, liquids, sprays, films, micelles, powders, food, feed pellets or as any type of encapsulated fomi.<br>
[0009]	In some embodiments, the aldolases and/or compositions thereof may be useful in<br>
pharmaceutical, industrial, and/or agricultural contexts.<br>
[0010]	In some embodiments, the aldolases and/or compositions thereof may be useful<br>
for forming or cleaving carbon-carbon bonds.<br>
 <br>
[0011]	In some embodiments, aldolases are provided that catalyze carbon-carbon bond<br>
forming reactions between an alpha-keto acid acceptor and a pyruvate or a p&gt;Tuvate<br>
derivative donor (see reaction scheme below). In some embodiments, the acceptor can<br>
also be a ketone or an aldehyde. In some embodiments, aldolases are provided that have<br>
4-hydroxy-2-oxogIutarate ald( Aldolase	keto-4-hydroxyglutarate aldolase, 2-oxo-4--<br>
hydroxyglutarate aldolase, KHG-aldolase, EC 4.1.3.16) activity and catalyze the following reaction: 4-hydroxy-2-oxoglutarate  pyruvate -s- glyoxylate. In some embodiments, aldolases are provided that have HMG-aldoiase (such as 4-hydroxy-4-methyl-2-oxoglutarate aldolase, pyruvate aldolase, gamma-methyl-gamma-hydroxy-alpha-ketoglutaric aldolase, 4-hydroxy-4-methyl-2-ketoglutarate aldolase, EC 4.1.3.17) activity and catalyze the following reaction: 4-hydroxy-4-methyl-2-oxoglutarate  2 pyruvate. An HMG aldolase will also act on 4-hydroxy-4-methyl-2-oxoadipate and 4-carboxy-4-hydroxy-2-oxohexadioate.<br>
 <br>
[0012]	In some embodiments, aldolases, such as a pyruvate aldolase, such as, without<br>
limitation a HMG and/or a KHG aldolase, are provided that facilitate the production of a 3, 4-substituted 2-keto-glutarate. In one embodiment, the invention provides a method of making a 3, 4-substituted 2-keto-gJutarate comprising: (a) providing a polypeptide having an aldolase activity, such as a pyruvate aldolase activity, such as, without limitation, a HMG aldolase and/or a KMG aldolase activity; (b) providing a donor and an acceptor compound; and (c) contacting the polypeptide of step (a) with the compounds of step (b) under conditions wherein the aldolase catalyzes the synthesis of a 3, 4-substituted 2-keto-<br>
 <br>
glutarate, wherein optionally the donor and the acceptor are a   pyruvate or a pyruvate donor and an a-keto acid acceptor, a ketone and/'or an aldehyde.<br>
[0013]	hi some embodiments, aldolases are provided that facilitate the production of R-2-<br>
hydroxy 2-(indo]-3yJmethy])-4-keto glutaric acid (R-MP), a monatin precursor. In some embodiments, a pyruvate aldolase, such as a HMG and/or a KHG aldolase, can be used in conjunction with a D-aminotransferase to make a 4-substituted D-g]utamic acid or a derivative thereof A 4-substituted D-glutamic acid and/or a derivative thereof can be used as an antibiotic, as these compoionds have been found to inhibit bacterial glutamate racemase (WO0214261 A3). In one embodiment, the invention provides a method of making a 4-substituted D-glutamic acid comprising: (a) providing a polypeptide having an aldolase activity, such as a pyruvate aldolase activity, such as, without limitation, a HMG aldolase and/or a KMG aldolase activity; (b) providing an a-keto acid acceptor and a pyruvate or a pyruvate donor; and (c) contacting the polypeptide of step (a) with the compounds of step (b) under conditions wherein the aldolase catalyzes the sjTithesis of a 4-substituted D-glutamic acid, wherein optionally the polypeptide has pyruvate aldolase, HMG aldolase and/or KHG aldolase activity and wherein optionally the method further comprises use of a D-aminotransferase.<br>
[0014]	The   invention   provides   isolated,   synthetic   or   recombinant   nucleic   acids<br>
comprising a nucleic acid sequence having at least about 50%, 51%, 52%, 53%, 54%, 55%, 56%, 57%, 58%, 59%, 60%, 61%, 62%, 63%, 64%, 65%, 66%, 67%, 68%, 69%, 70%, 71%, 72%, 73%, 74%, 75%, 76%, 77%, 78%, 79%, 80%, 81%, 82%, 83%, 84%, 85%, 86%, 87%., 88%, 89%, 90%, 91%, 92%, 93%, 94%, 95%, 96%, 97%, 98%, 99%, or more, or complete (100%)) sequence identity to a nucleic acid in accordance ™th the invention, including SEQ ID N0:1, SEQ ID N0:3, SEQ ID N0:5, SEQ ID N0:7, SEQ ID N0:9, SEQ ID NOiII, SEQ ID N0:13, SEQ ID NO:15, SEQ ID N0:17, SEQ ID N0:19, SEQ ID N0:21, SEQ ID NO:23, SEQ ID NO:25, SEQ ID NO:27, SEQ ID NO:29, SEQ ID NO:3I, SEQ ID NO:33, SEQ ID NO;35, SEQ ID NO;37, SEQ ID NO:39, SEQ ID N0:41, SEQ ID NO:43, SEQ ID NO:45, SEQ ID NO:47, SEQ ID NO:49, SEQ ID N0;51, SEQ ID NO:53, SEQ ID NO:55, SEQ ID NO;57, SEQ ID NO;59, SEQ ID N0:6i, SEQ ID NO:63, SEQ ID NO:65, SEQ ID NO:67, SEQ ID NO;69, SEQ ID N0:71, SEQ ID NO:73, SEQ ID NO:75, SEQ ID NO;77, SEQ ID NO:79, SEQ ID N0:8L SEQ ID NO:83, SEQ ID NO:85, SEQ ID NO:87, SEQ ID<br>
 <br>
NO:89, SEQ ID N0:91, SEQ ID NO:93, SEQ ID NO:95, SEQ ID NO:97, SEQ ID NO:99, SEQIDNO:10U SEQIDNO:103, SEQIDNO;105, SEQIDNO;107. SEQ ID NO:109, SEQ ID NO:UL SEQ ID N0:113, SEQ IDNO:I15. SEQ ID NO:117, SEQ ID N0:U9, SEQ ID N0:121, SEQ ID NO:123, SEQ ID NO:125, SEQ ID NO:127, SEQ ID N0:129, SEQ ID N0:131, SEQ ID NO;133, SEQ ID NO:135, SEQ ID NO:i37, SEQ ID NO:139, SEQ ID N0:141, SEQ ID NO:143, SEQ ID NO:145, SEQ ID NO:147, SEQ ID NO:149, SEQ ID N0:151, SEQ ID NO:I53, SEQ ID NO:155, SEQ ID NO:157, SEQ ID NO:159, SEQ ID N0:16I, SEQ ID NO:163, SEQ ID NO:165, SEQ ID NO:167, SEQ ID NO:169, SEQ IDN0:171, SEQ IDNO:173, SEQ IDNO:175, SEQ ID NO:177, SEQ ID NO:179, SEQ ID N0:181, SEQ ID NO:183, SEQ ID NO:185, SEQ ID NO:I87, SEQ ID NO:189, SEQ ID N0:191, SEQ ID NO:193, SEQ ID NO:195, SEQ ID NO:197, SEQ ID NO:199, SEQ ID NO:201, SEQ ID NO:203, SEQ ID NO:205, SEQ ID NO:207, SEQ ID NO:209, SEQ ID N0:211, SEQ ID NO:213, SEQ ID NO:215, SEQ ID NO:217, SEQ ID NO:219, SEQ ID NO:221, SEQ ID NO:223, SEQ ID NO:225, SEQ ID NO:227, SEQ ID NO:229, SEQ ID NO:231, SEQ ID NO:233, SEQ ID NO:235, SEQ ID NO:237, SEQ ID NO:239, SEQ ID NO:241, SEQ ID NO:243, SEQ ID NO:245, SEQ ID NO:247, SEQ ID NO:249, SEQ ID NO:251, SEQ ID NO:253, SEQ ID NO;255, SEQ ID NO:257, SEQ ID NO:259, SEQ ID NO:261, SEQ ID NO:263, SEQ ID NO:265, SEQ ID NO:267, SEQ ID NO:269, SEQ ID NO:271, SEQ ID NO:273, SEQ ID NO:275, SEQ ID NO:277, SEQ ID NO:279, SEQ ID NO:28U SEQ ID NO:283, SEQ ID NO:285, SEQ ID NO:287, SEQ ID NO:289, SEQ ID NO:291, SEQ ID NO:293, SEQ ID NO:295, SEQ ID NO:297, SEQ ID NO:299, SEQ ID NO:301, SEQ ID NO:303, SEQ ID NO:305, SEQ ID NO:307, SEQ ID NO:309, SEQ IDN0:3n, SEQ IDNO:313, SEQ IDNO:315, SEQ IDNO:317, SEQ ID NO:319, SEQ ID NO:321, SEQ ID NO:323, SEQ ID NO:325, SEQ ID NO:327, SEQ ID NO:329, SEQ ID NO:331, SEQ ID NO:333, SEQ ID NO:335, SEQ ID NO:336, SEQ ID NO:337, and SEQ ID NO:338, over a region of at least about 10, 15, 20, 25, 30, 35, 40, 45, 50, 75, 100, !50, 200, 250, 300, 350, 400, 450, 500, 550, 600, 650, 700, 750, 800, 850, 900, 950, 1000, 1050, 1100,1150, 1200, 1250,1300,1350,1400, 1450, 1500,1550, 1600, 1650, 1700, 1750, 1800, 1850, 1900, 1950, 2000, 2050, 2100, 2200, 2250, 2300, 2350, 2400, 2450, 2500, or more residues. In some embodiments, one or more nucleic acids encode at let one polypeptide having an aldolase activity, including pyruvate activity such as, without limitation, HMG and/or KHG aldolase activity.    In some<br>
 <br>
embodiments, the sequence identities are detennined by analysis with a sequence comparison algorithm or by a visual inspection.<br>
[0015]	In alternative embodiments, one or more nucieic acids encode at least one<br>
polypeptide capable of generating an antibody that can specifically bind to a polypeptide of the invention, or, these nucleic acids can be used as probes for identifying or isolating aldolase-encoding nucleic acids, or to inhibit the expression of aldolase-expressing nucleic acids.<br>
[0016]	Nucleic acids in accordance with the invention also include isolated, synthetic or<br>
recombinant nucleic acids encoding enzymes in accordance with the invention, such as enzymes including one or more polypeptides having a sequence as set fortii in SEQ ID N0:2, SEQ ID N0:4, SEQ ID N0:6, SEQ ID N0:8, SEQ ID NO:10, SEQ ID N0:12, SEQ ID N0:14, SEQ ID N0:16, SEQ ID N0:]8, SEQ ID NO:20, SEQ ID NO:22, SEQ ID N0:24, SEQ ID NO:26, SEQ ID NO:28, SEQ ID NO;30, SEQ ID NO:32, SEQ ID NO:34, SEQ ID NO:36, SEQ ID NO:38, SEQ ID NO:40, SEQ ID NO:42, SEQ ID NO:44, SEQ ID NO:46, SEQ ID NO:48, SEQ ID NO:50, SEQ ID NO:52, SEQ ID NO:54, SEQ ID NO:56, SEQ ID NO:58, SEQ ID NO:60, SEQ ID NO:62, SEQ ID NO:64, SEQ ID NO:66, SEQ ID NO:68, SEQ ID NO:70, SEQ ID NO:72, SEQ ID NO:74, SEQ ID NO:76, SEQ ID NO:78, SEQ ID NO:80, SEQ ID NO:82, SEQ ID NO:84, SEQ ID NO:86, SEQ ID NO:88, SEQ ID NO:90, SEQ ID NO:92, SEQ ID NO:94, SEQ ID NO:96, SEQ ID NO:98, SEQ ID NO:100, SEQ ID NO:102, SEQ ID NO:I04, SEQ ID NO:106, SEQ ID NO;108, SEQ ID NOillO, SEQ ID N0:112, SEQ ID N0:114, SEQ ID N0:116, SEQ ID N0:118, SEQ ID NO:120, SEQ ID NO:122, SEQ ID NO:I24, SEQ ID NO:126, SEQ ID NO:128, SEQ ID NO:130, SEQ ID NO:132, SEQ ID NO:134, SEQ ID NO:136, SEQ ID NO:138, SEQ ID NO:140, SEQ ID NO:142, SEQ ID NO:144, SEQ ID NO:146, SEQ ID NO:148, SEQ ID NO:150, SEQ ID NO:152, SEQ ID NO:154, SEQ ID NO:156, SEQ ID NO:158, SEQ ID NO:160, SEQ ID NO:I62, SEQ ID NO:164, SEQ ID NO:166, SEQ ID NO:168, SEQ ID NO:170, SEQ ID NO:172, SEQ ID NO:174, SEQ ID NO:176, SEQ ID NO:178, SEQ ID NO:180, SEQ ID NO:182, SEQ ID NO:184, SEQ IDNO:186, SEQIDNO;188, SEQ IDNO:190, SEQ IDNO:192, SEQ ID NO:194, SEQ ID NO:196, SEQ ID NO:198, SEQ ID NO:200, SEQ ID NO:202, SEQ ID NO:204, SEQ ID NO:206, SEQ ID NO:208, SEQ ID NO:210, SEQ ID NO:212, SEQ ID NO:214, SEQ ID NO:216, SEQ ID NO:218, SEQ ID NO:220, SEQ ID NO:222, SEQ ID<br>
 <br>
NO:224, SEQ ID NO:226, SEQ ID NO:228, SEQ ID NO:230, SEQ ID NO:232, SEQ ID<br>
NO:234, SEQ ID NO:236, SEQ ID NO:238, SEQ ID NO:240, SEQ ID NO:242, SEQ ID<br>
NO:244, SEQ ID NO:246, SEQ ID NO:248, SEQ ID NO:250, SEQ ID NO:252, SEQ ID<br>
NO:254, SEQ ID NO:256, SEQ ID NO:258, SEQ ID NO:260, SEQ ID NO:262, SEQ ID<br>
NO:264, SEQ ID NO:266, SEQ ID NO:268, SEQ ID NO:270, SEQ ID NO;272, SEQ ID<br>
NO:274, SEQ ID NO:276, SEQ ID NO:278, SEQ ID NO:280, SEQ ID NO:282, SEQ ID<br>
NO:284, SEQ ID NO:286, SEQ ID NO:288, SEQ ID NO:290, SEQ ID NO:292, SEQ ID<br>
NO:294, SEQ ID NO:296, SEQ ID NO:298, SEQ ID NOi300, SEQ ID NO:302, SEQ ID<br>
NO:304, SEQ ID NO:306, SEQ ID NO:308, SEQ ID NO:310, SEQ ID NO:3]2, SEQ ID.<br>
NO:314, SEQ ID NO:316, SEQ ID NO:318, SEQ ID NO:320, SEQ ID NO:322, SEQ ID<br>
NO:324, SEQ ID NO:326, SEQ ID NO:328, SEQ ID NO:330, SEQ ID NO;332, and SEQ<br>
ID NO:334, and subsequences thereof, variants thereof and enzjmatically active<br>
fragments thereof. In some embodiments, the polypeptide has an aidoiase activity,<br>
including pyruvate activity such as, without limitation, HMG and/or KHG aldolase<br>
activity.<br>
[0017]	In some embodiments, the invention provides aldolase-encoding, such as pyruvate<br>
aldolase-, such as HMG and/or KHG aldolase-encoding nucleic acids preferably derived from mixed cultures. In some embodiments, the invention provides carbon-carbon bond forming or cleaving enzyme-encoding nucleic acids isolated from mixed cultures comprising polynucleotides in accordance with the invention, such as a sequence having at least about 10%, 15%, 20%, 25%, 30%, 35%, 40%, 45%, 50%, 51%, 52%, 53%», 54%, 55%, 56%, 57%,, 58%, 59%, 60%, 61%, 62%, 63%, 64%, 65%, 66%, 67%, 68%, 69%, 70%, 71%, 72%, 73%, 74%, 75%, 76%, 77%, 78%, 79%, 80%, 81%, 82%, 83%, 84%, 85%*, 86%, 87%, 88%, 89%, 90%, 91%, 92%, 93%, 94%, 95%, 96%, 97%, 98%, 99%, or more, or complete (100%)) sequence identity to a nucleic acid in accordance with the invention, such as SEQ ID N0:1, SEQ ID N0:3, SEQ ID N0:5, SEQ ID N0:7, SEQ ID N0:9, SEQ ID N0:11, SEQ ID N0:13, SEQ ID N0:15, SEQ ID N0;17, SEQ ID N0:19, SEQ ID N0:21, SEQ ID NO:23, SEQ ID NO:25, SEQ ID NO:27, SEQ ID NO:29, SEQ ID N0:31, SEQ ID NO:33, SEQ ID NO:35, SEQ ID NO:37, SEQ ID NO:39, SEQ ID N0:41, SEQ ID NO:43, SEQ ID NO:45, SEQ ID NO:47, SEQ ID NO:49, SEQ ID N0:51, SEQ ID NO:53, SEQ ID NO:55, SEQ ID NO:57, SEQ ID NO:59, SEQ ID N0:6I, SEQ ID NO;63, SEQ ID NO:65, SEQ ID NO:67, SEQ ID NO:69, SEQ ID<br>
 <br>
N0:7!, SEQ ID NO:73, SEQ ID NO:75, SEQ ID NO:77, SEQ ID NO:79, SEQ ID<br>
N0:8L SEQ ID NO;83, SEQ ID NO:85, SEQ ID NO:87, SEQ ID NO:89, SEQ ID<br>
N0:91, SEQ ID NO:93, SEQ ID NO:95, SEQ ID NO:97, SEQ ID NO:99, SEQ ID<br>
NO:[0L SEQ ID NO:103, SEQ ID NO:105, SEQ ID NO:107, SEQ ID NO:109. SEQ ID<br>
N0:111, SEQ ID N0:113, SEQ ID N0:115, SEQ ID N0:117, SEQ ID N0:119, SEQ ID<br>
N0:121, SEQ ID NO:I23, SEQ ID NO:125, SEQ ID NO:127, SEQ ID NO:I29, SEQ ID<br>
N0:131, SEQ ID NO:133, SEQ ID NO;135, SEQ ID NO:I37, SEQ ID 10:139, SEQ ID<br>
N0:14l, SEQ ID NO:143, SEQ ID NO:145, SEQ ID NO:147, SEQ ID NO:149, SEQ ID<br>
N0:151, SEQ ID NO:153, SEQ ID NO:155, SEQ ID NO:157, SEQ ID NO:159, SEQ ID<br>
H0:16l, SEQ ID NO:]63, SEQ ID NO:]65, SEQ ID NO;167, SEQ ID NO:]69, SEQ ID<br>
N0:171, SEQ IDNO:173, SEQ IDNO:175, SEQIDNO:177, SEQ IDNO:179, SEQ ID<br>
N0:181, SEQ ID NO:183, SEQ ID NO:185, SEQ ID NO:187, SEQ ID NO:189, SEQ ID<br>
N0:191, SEQ ID NO:193, SEQ ID NO:195, SEQ ID NO:197, SEQ ID NO:199, SEQ ID<br>
NO:201, SEQ ID NO:203, SEQ ID NO:205, SEQ ID NO:207, SEQ ID NO:209, SEQ ID<br>
N0:211, SEQ ID NO:2!3, SEQ ID NO:2i5, SEQ ID NO:2I7, SEQ ID NO:219, SEQ ID<br>
NO:221, SEQ ID NO:223, SEQ ID NO:225, SEQ ID NO:227, SEQ ID NO:229, SEQ ID<br>
NO:231, SEQ ID NO:233, SEQ ID NO:235, SEQ ID NO:237, SEQ ID NO:239, SEQ ID<br>
NO:24U SEQ ID NO:243, SEQ ID NO:245, SEQ ID NO:247, SEQ ID NO:249, SEQ ID<br>
NO:251, SEQ ID NO:253, SEQ ID NO:255, SEQ ID NO:257, SEQ ID NO:259, SEQ ID<br>
NO:261, SEQ ID NOi263, SEQ ID NO:265, SEQ ID NO:267, SEQ ID NO:269, SEQ ID<br>
NO:271, SEQ ID NO:273, SEQ ID NO:275, SEQ ID NO:277, SEQ ID NO:279, SEQ ID<br>
NO:281, SEQ ID NO:283, SEQ ID NO:285, SEQ ID NO:287, SEQ ID NO:289, SEQ ID<br>
NO:291, SEQ ID NO:293, SEQ ID NO:295, SEQ ID NO:297, SEQ ID NO:299, SEQ ID<br>
NO:301, SEQ ID NO:303, SEQ ID NO:305, SEQ ID NO-.307, SEQ ID NO:309, SEQ ID<br>
N0:311, SEQ ID NO:313, SEQ ID NO:315, SEQ ID NO:317, SEQ ID NO:319, SEQ ID<br>
NO;32I, SEQ ID NO:323, SEQ ID NO;325, SEQ ID NO:327, SEQ ID NO;329, SEQ ID<br>
NO:331, SEQ ID NO:333, SEQ ID NO;335, SEQ ID NO:336, SEQ ID NO:337, and SEQ<br>
ID NO:338 over a region of at least about 50, 75, 100, 150, 200, 250, 300, 350, 400, 450,<br>
500, 550, 600, 650, 700, 750, 800, 850, 900, 950,1000,1050,1100, 1150, or more.<br>
[0018]	In some embodiments, the invention provides aldolase enzyme-, such as pyruvate<br>
aldolase   enzyme-,   HMG   and/or   KHG   enzyme-encoding   nucleic   acids,   including polynucleotide sequences in accordance with the invention and the polypeptides encoded<br>
 <br>
by them, including enzymes in accordance with the invention, such as polypeptides in accordance with the invention, such as SEQ ID N0:2, SEQ ID N0:4, SEQ ID N0:6, SEQ ID N0:8, SEQ ID NO:10, SEQ ID N0:12, SEQ ID N0:14, SEQ ID NO:16, SEQ ID NO; 18, SEQ ID NO:20, SEQ ID NO:22, SEQ ID NO:24, SEQ ID NO:26, SEQ ID NO;28, SEQ ID NO:30, SEQ ID NO:32, SEQ ID NO:34, SEQ ID NO:36, SEQ ID NO:38, SEQ ID NO:40, SEQ ID NO:42, SEQ ID NO:44, SEQ ID NO:46, SEQ ID NO:48, SEQ ID NO:50, SEQ ID NO:52, SEQ ID NO:54, SEQ ID NO:56, SEQ ID NO:58, SEQ ID NO:60, SEQ ID NO:62, SEQ ID NO:64, SEQ ID NO:66, SEQ ID NO:68, SEQ ID NO:70, SEQ ID NO:72, SEQ ID NO:74, SEQ ID NO:76, SEQ ID NO:78, SEQ ID NO:80, SEQ ID NO:82, SEQ ID NO:84, SEQ ID NO:86, SEQ ID NO:88, SEQ ID NO:90, SEQ ID NO:92, SEQ ID NO:94, SEQ ID NO:96, SEQ ID N0:9S, SEQ ID NOilOO, SEQ ID NO:102, SEQ ID NO:104, SEQ ID NO:106, SEQ ID NO:108, SEQ ID NO:U0, SEQ IDNO:112, SEQ IDN0:114, SEQ ID NO:116, SEQ ID N0:118, SEQ ID NO:120, SEQ ID NO:122, SEQ ID NO:124, SEQ ID NO:126, SEQ ID NO:128, SEQ ID NO:I30, SEQ ID NO:132, SEQ ID NO:134, SEQ ID NO:136, SEQ ID NO:138, SEQ ID NO:U0, SEQ ID NO:142, SEQ ID NO:144, SEQ ID NO:146, SEQ ID NO:148, SEQ ID NO:150, SEQ ID NO:I52, SEQ ID NO:154, SEQ ID NO:I56, SEQ ID N0:15S, SEQ ID NO:160, SEQ ID NO: 162, SEQ ID NO:164, SEQ ID NO:166, SEQ ID NO:168, SEQ ID NO:nO, SEQ ID NO:172, SEQ ID NO:174, SEQ ID NO:176, SEQ ID NO:178, SEQ ID NO:180, SEQ ID NO:182, SEQ ID NO:184, SEQ ID NO:186, SEQ ID NO:188, SEQ ID NO:190, SEQ ID NO:192, SEQ ID NO:194, SEQ ID NO:196, SEQ ID NO:198, SEQ ID NO:200, SEQ ID NO:202, SEQ ID NO:204, SEQ ID NO:206, SEQ ID NO:208, SEQ ID NO:210, SEQ ID NO;212, SEQ ID NO:214, SEQ ID NO:216, SEQ ID NO:2I8, SEQ ID NO:220, SEQ ID NO:222, SEQ ID NO:224, SEQ ID NO:226, SEQ ID NO:228, SEQ ID NO:230, SEQ ID NO:232, SEQ ID NO:234, SEQ ID NO:236, SEQ ID NO:238, SEQ ID NO:240, SEQ ID NO:242, SEQ ID NO:244, SEQ ID NO;246, SEQ ID NO:248, SEQ ID NO:250, SEQ ID NO:252, SEQ ID NO:254, SEQ ID NO:256, SEQ ID NO:258, SEQ ID NO:260, SEQ ID NO:262, SEQ ID NO:264, SEQ ID NO:266, SEQ ID NO:268, SEQ ID NO:270, SEQ ID NO:272, SEQ ID NO;274, SEQ ID NO:276, SEQ ID NO:278, SEQ ID NO:280, SEQ ID NO:282, SEQ ID NO:284, SEQ ID NO:286, SEQ ID NO:288, SEQ ID NO:290, SEQ ID NO:292, SEQ ID NO:294, SEQ ID NO:296, SEQ ID NO:298, SEQ ID NO:300, SEQ ID NO:302, SEQ ID NO:304, SEQ ID NO:306, SEQ ID<br>
 <br>
NO:308, SEQ ID NO:310, SEQ ID NOG 12, SEQ ID NO:314, SEQ ID NO:316, SEQ ID NO:318, SEQ ID NO:320, SEQ ID NO:322, SEQ ID NO:324, SEQ ID NO:326, SEQ ID NO:328, SEQ ID NO:330, SEQ ID NO:332, or SEQ ID NO:334, and enzymatically-active fragmeats thereof, preferably derived from a common source, such as an environmental source. In some embodiments, the invention also provides aldolase enzyme- , such as pyruvate aldolase enzyme-, HMG and/or KHG enzyme-encoding nucleic acids preferably derived from environmenta! sources, such as mixed environmental sources.<br>
[0019]	In some embodiments, the sequence comparison algorithm is a BLAST version<br>
2.2.2 algorithm where a filtering setting is set to blastall -p blastp -d "nr pataa" -F F, and all other options are set to default.<br>
[0020]	Other embodiments of the invention are isolated, synthetic or recombinant nucleic<br>
acids including at least 10, 15, 20, 25, 30, 35, 40, 45, 50, 75, 100, 150, 200, 250, 300, 350, 400,450, 500, 550, 600. 650, 700, 750, 800, 850, 900, 950, 1000, 1050,1100, 1150, 1200, 1250, 1300, 1350, 1400, 1450, 1500, 1550, 1600, .1650, 1700, 1750, 1800, 1850, 1900, 1950, 2000, 2050, 2100, 2200, 2250, 2300, 2350, 2400, 2450, 2500, or more consecutive bases of a nucleic acid sequence in accordance with the invention, sequences substantially identical thereto, and the sequences complementary thereto.<br>
[0021]	In some embodiments, the isolated, synthetic or recombinant nucleic acids in<br>
accordance with the invention encodes a polypeptide having an aldolase activity, including pyruvate activity such as, without limitation, HMG and/or KHG aldolase activity, which is thermostable. The thermostable polypeptide according to the invention can retain an aldolase activity, such as a pyruvate aldolase activity, such as a HMG and/or a KHG aldolase activity, under conditions comprising a temperature range from about -100°C to about -80°C, about -80°C to about -40°C, about -40°C to about -20°C, about -20°C to about 0°C, about 0°C to about 37°C, about OX to about SC, about 5°C to about 15°C, about 15°C to about 25'C, about 25°C to about 37°C, about 37°C to about 45°C, about 45°C to about 55°C, about 55°C to about 70°C, about 70'C to about 75°C, about 75°C to about 85°C, about 85°C to about 90*0, about 90°C to about 95°C, about 95°C to about 100°C, about 100°C to about I05°C, about 105°C to about 110°C, about 110°C to about 120°C, or 95°C, 96°C, 97°C, 98°C, 99T, 100°C, 101°C, 102'C, 103T, 104°C, 105°C, 106°C, 107°C, 10S°C, 109°C, 110°C, 111°C, U2X, 113°C, 114°C, 115°C or<br>
 <br>
more. The thermostable polypeptides according to the invention can retain an aldolase<br>
activity, such as a pyruvate aldolase activity, such as a HMG and/or a KHG aldolase<br>
activity, in temperatures in the range from about -100°C to about -80°C, about -80°C to<br>
about -40°C, about -40°C to about -20°C, about -20°C to about 0°C, about OX to about<br>
5°C, about 5°C to about 15°C, about 15°C to about 25°C, about 25'C to about 37°C, about<br>
37X to about 45°C, about 45°C to about 55°C, about DSX to about 70°C, about 70°C to<br>
about 15°C, about 75°C to about %5°C, about 85°C to about 90°C, about 90°C to about<br>
95°C, about 95°C to about 100°C, about 100°C to about 105°C, about lOSC to about<br>
110°C, about 110°C to about I20°C, or 95°C, 96'C, 97°C, 98°C, 99°C, 100°C, lOrC,<br>
102°C, lOSX, 104°C, 105X, 106°C, 107°C, lOSX, 109°C, 110°C, UTC, 1120, 113°C,<br>
114°C, 115°C or more. In some embodiments, the thermostable polypeptides accordmg<br>
to the invention retains an aldolase activity at a temperature in the ranges described<br>
above, at about pH,3-0, about pH 3.5, about pH 4.0, about pH 4.5, about pH 5.0, about pH<br>
5.5, about pH 6.0, about pH 6.5, about pH 7.0, about pH 7.5, about pH 8.0, about pH 8.5,<br>
about pH 9.0, about pH 9.5, about pH 10.0, about pH 10.5, about pH 11.0, about pH 11.5,<br>
about pH 12.0 or more.<br>
[0022]	In.other embodiments, the isolated, synthetic or recombinant nucleic acids encode<br>
a polypeptide having an aldolase activity, including pyruvate activity such as, without limitation, HMG and/or KHG aldolase activity, which is thermotolerant. The thermotolerant polypeptides according to the invention can retain an aldolase activity, such as a pyruvate aldolase activity, such as a HMG and/or a KHG aldolase activity, after exposure to conditions comprising a temperature in the range from about -100°C to about -80°C, about -80°C to about -40°C, about -40°C to about -20°C, about -20°C to about 0°C, about OT to about 5°C, about 5°C to about 15X, about !5X to about 25°C, about 25 to about 37°C, about ZTC to about 45°C, about 45°C to about 55°C, about 55°C to about 70°C, about 70T to about 75°C, about 75°C to about 85°C, about SS'C to about 90X, about 90°C to about 95°C, about 95°C to about 100°C, about 100°C to about 105°C, about 105°C to about n0°C, about 110°C to about 120°C, or 95°C, 96°C, 97°C, 98°C, 99°C, 100°C, lore, 102°C, 103°C, 104'C, 105°C, 106°C, 107°C, 108°C, 109'C, 110°C, lire, 112'e, 113°C, lUX, 1I5°C or more. The thermotolerant polypeptides according to the invention can retain an aldolase activity, such as a pyruvate aldolase activity, such as a HMG and/or a KHCJ aldolase activity, after exposure to a temperature<br>
 <br>
in the range from about -100°C to about -80°C, about -SO'C to about -40X, about -40°C<br>
to about -20°C, about -20°C to about OC, about O'C to about SC, about SC to about<br>
15C, about 15T to about 25°C, about 25°C to about 37T, about 37'C to about 45'C,<br>
about 45°C to about 55°C, about 55°C to about 70°C, about 70C to about 75°C, about<br>
75°C to about 85°a about 85°C to about 90°C, about 90°C to about 95°C, about 95°C to<br>
about lOOC, about 100°C to about 105°C, about !05°C to about 110°C, about 110°C to<br>
about 120°C, or 95°C, 96°C, 97°C, 98°Q 99°Q 100°C. lOrC, 102'C, 103°a 104°C,<br>
lOS'C, 106°C, 107'C, 108°C, 109°C, llOX, IITC, 112°C, 113°C, 114°C, USX or<br>
more. In some embodiments, the thermotolerant polypeptides according to the invention<br>
retains an aldoiase actii'ity after exposure to a temperature in the ranges described above,<br>
at about pH 3.0, about pH 3.5, about pH 4.0, about pH 4.5, about pH 5.0, about pH 5.5,<br>
about pH 6.0, about pH 6.5, about pH 7.0, about pH 7.5, about pH 8.0, about pH 8.5,<br>
about pH 9.0, about pH 9.5, about pH .10.0, about pH 10.5, about pH 11.0, about pH 11.5,<br>
about pH 12.0 or more.<br>
[0023]	The   invention   provides   isolated,   synthetic   or   recombinant   nucleic   acids<br>
comprising a sequence that hybridizes under stringent conditions to nucleic acids in accordance with the invention, including a sequence as set forth in SEQ ID N0:1, SEQ ID N0:3, SEQ ID N0:5, SEQ ID N0:7, SEQ ID N0:9, SEQ ID N0:11, SEQ ID N0:i3, SEQ ID N0:15, SEQ ID N0-.17, SEQ ID N0:19, SEQ ID N0:21, SEQ ID NO:23, SEQ ID NO:25, SEQ ID NO:27, SEQ ID NO:29, SEQ ID N0:3], SEQ ID NO;33, SEQ ID NO:35, SEQ ID NO:37, SEQ ID NO:39, SEQ ID NO:41, SEQ ID NO:43, SEQ ID NO:45, SEQ ID NO:47, SEQ ID NO:49, SEQ ID N0:51, SEQ ID NO:53, SEQ ID NO:55, SEQ ID NO:57, SEQ ID NO:59, SEQ ID NO;6I, SEQ ID NO:63, SEQ ID NO:65, SEQ ID NO:67, SEQ ID NO;69, SEQ ID N0:71, SEQ ID NO:73, SEQ ID NO:75, SEQ ID NO;77, SEQ ID NO:79, SEQ ID N0:81, SEQ ID NO:83, SEQ ID NO:85, SEQ ID NO:87, SEQ ID NO:89, SEQ ID N0:91, SEQ ID NO:93, SEQ ID NO:95, SEQ ID NO:97, SEQ ID NO:99, SEQ ID NO:101, SEQ ID NO:103, SEQ ID NO:105, SEQ ID NO:107, SEQ ID NO:109, SEQ ID N0:111, SEQ IDN0:113, SEQ ID N0:115, SEQ IDN0:117, SEQ ID N0:119, SEQ IDN0:121. SEQ IDNOa23, SEQ ID NO:125, SEQ ID NO:127, SEQ ID NO:I29, SEQ ID NO:131, SEQ ID NO:133, SEQ ID NO:135, SEQ ID NO:137, SEQ ID NO:I39, SEQ ID N0:141, SEQ ID NO:143, SEQ ID NO:145, SEQ ID NO:147, SEQ ID NO:149, SEQ ID N0:151, SEQ ID NO:153, SEQ ID<br>
 <br>
NO:155, SEQ ID NO:157, SEQ ID NO:159, SEQ ID N0:161, SEQ ID NO:i63, SEQ ID<br>
NO:165, SEQ ID NO:167, SEQ ID NO:169, SEQ ID N0:17L SEQ ID NO:173, SEQ ID<br>
N0:I75, SEQ ID NO:177, SEQ ID NO:I79, SEQ ID N0:I81, SEQ ID NO:i83, SEQ ID<br>
NO:185, SEQ ID N0:1S7, SEQ ID NO:189, SEQ ID NO:193, SEQ ID NO.i93, SEQ ID<br>
NO:195, SEQ ID NO:197, SEQ ID NO:199, SEQ ID NO:201, SEQ ID NO:203, SEQ ID<br>
NO:205, SEQ ID NO:207, SEQ ID NO:209, SEQ ID N0:2n, SEQ ID NO:213, SEQ ID<br>
NO:215, SEQ ID NO:217, SEQ ID NO:219, SEQ ID NO:22i, SEQ ID NO:223, SEQ ID<br>
NO:225, SEQ ID NO:227, SEQ ID NO:229, SEQ ID NO:231, SEQ ID NO:233, SEQ ID<br>
NO:235, SEQ ID NO:237, SEQ ID NO:239, SEQ ID NO:241, SEQ ID NO:243, SEQ ID<br>
NO:245, SEQ ID NO:247, SEQ ID NO:249, SEQ ID NO:251, SEQ ID NO:253, SEQ ID<br>
NO:255, SEQ ID NO-.257, SEQ ID NO:259, SEQ ID NO:261, SEQ ID NO:263, SEQ ID<br>
NO:265, SEQ ID NO;267, SEQ ID NO:269, SEQ ID NO;271, SEQ ID NO:273, SEQ ID<br>
NO:275, SEQ ID NO:277, SEQ ID NO:279, SEQ ID NO:281, SEQ ID NO:283, SEQ ID<br>
NO:285, SEQ ID NO:287, SEQ ID NO:289, SEQ ID NO:291, SEQ ID NO:293, SEQ ID<br>
NO:295, SEQ ID NO:297, SEQ ID NO:299, SEQ ID NO:301, SEQ ID NO:303, SEQ ID<br>
NO:305, SEQ ID NO:307, SEQ ID NO:309, SEQ ID N0:3n, SEQ ID NO:313, SEQ ID<br>
NO:315, SEQ ID NO:317, SEQ ID NO:319, SEQ ID NO:321, SEQ ID NO:323, SEQ ID<br>
NO:325, SEQ ID NO-.327, SEQ ID NO:329, SEQ ID NO:331, SEQ ID NO:333, SEQ ID<br>
NO:335, SEQ ID NO:336, SEQ ID NO:337, or SEQ ID NO;338, or fragments or<br>
subsequences thereof. In some embodiments, the nucleic acids encode polypeptides<br>
having an aldolase activity, including pyruvate activity such as, without limitation, HMG<br>
and/or SCHG aldolase activity. The nucleic acids can be at let about 10, 15, 20, 25, 30,<br>
35, 40, 45, 50, 75, 100,150, 200, 250, 300, 350, 400, 450, 500, 550, 600, 650, 700, 750,<br>
800, 850, 900, 950, 1000, 1050, 1100, 1150, 1200 or more residues m length or the Ml<br>
length of the gene or transcript. In some embodiments, the stringent conditions comprise<br>
a wash step comprising a wash in 0.2X SSC at a temperature of about 65°C for about 15<br>
minutes.<br>
[0024]	The invention provides nucleic acid probes for identifying or isolating nucleic<br>
acids encoding polypeptides having an aldolase activity, including pyruvate activity such as, without limitation, HMG and/or KHG aldolase activity-, wherein the probes comprise about 10, 15, 20, 25, 30, 35, 40, 45, 50, 55, 60, 65, 70, 75, 80, 85, 90, 95, 100, 150, 200, 250, 300, 350, 400, 450, 500, 550, 600, 650, 700, 750, 800, 850, 900, 950, 1000 or more.<br>
 <br>
consecutive bases of a sequence in accordance with the invention, and wherein the probes identify the nucleic acid by binding or hybridization. The probes can comprise an oligonucleotide comprising between about 10-100 consecutive bases of a sequence in accordance with the invention,or fragments or subsequences thereof, for example, 10, 15, 20, 25, 30, 35, 40, 45, 50, 55, 60, 65, 70, 75, 80, 85, 90, 95 or 100 bases or more, or, any desired length in between.<br>
[0025]	The invention provides nucleic acid probes for identifj'ing or isolating nucleic<br>
acids encoding polypeptides having an aldolase activity, including pyruvate activity such as, without limitation, HMG and/or KHG aldolase activity, wherein the probes comprise nucleic acids comprising a sequence ai least about 10, 15, 20, 30, 40, 50, 60, 70, 80, 90, 100, 150, 200, 250, 300, 350, 400, 450, 500, 550, 600, 650, 700, 750, 800, 850, 900, 950, 1000 or more residues of a nucleic acid in accordance with the invention, such as a polynucleotide havmg at least about 50%, 51%, 52%, 53%, 54%, 55%, 56%, 57%, 58%, 59%, 60%, 61%, 62%, 63%, 64%, 65%, 66%, 67%, 68%, 69%, 70%, 71%, 72%, 73%, 74%, 75%, 76%, 77%, 78%, 79%, 80%, 81%, 82%, 83%, 84%, 85%, 86%, 87%, 88%, 89%, 90%, 91%, 92%, 93%, 94%, 95%, 96%, 97%, 98%, 99%, or more, or complete (100%) sequence identity to a nucleic acid of the invention. In some embodiments, the sequence identities are determined by analysis with a sequence comparison algorithm or by visual inspection. In other embodiments, the probes can comprise an oligonucleotide comprising between at least about 10-100 corecutive bases of a nucleic acid sequence in accordance with the invention, or a subsequence thereof, for example ] 0, 15, 20, 25, 30, 35, 40, 45, 50, 55, 60, 65, 70, 75, 80, 85, 90, 95, or 100 bases or more, or, any desired letigth in between.<br>
[0026]	The invention provides amplification primer pairs for amplifying (such as by<br>
PCR) a nucleic acids encoding polypeptides having aldolase activity, including pyruvate activity such as, without limitation, HMG and/or KHG aldolase activity, wherein each primer pair is capable of amplifying a nucleic acid comprising a sequence in accordance with the invention, or fragments or subsequences thereof (see the Sequence Listing). One or each member of the amplification primer sequence pair can comprise an oligonucleotide comprising at least about 10 to 50, or more, consecutive bases of the sequence, or about 10, 11, 12, 13, 14, 15, 16, 17, 18, 19, 20, 21, 22, 23, 24, 25, 26, 27, 28, 29, 30, 31, 32, 33, 34, 35, 36 or more consecutive bases of the sequence.   In some<br>
 <br>
embodiments, the invention provides amplification primer pairs, wherein each primer pair comprises a first member having a sequence as set forth by about the first (the 5') 12, 13. 14, 15, 16, 17, 18, 19, 20, 21, 22, 23, 24, 25, 26, 27, 28, 29, 30, 31, 32, 33, 34, 35, 36 or more residues of a nucleic acid in accordance with the invention, and a second member having a sequence as set forth by about the first (the 5') 12, 13, 14, 15,16, 17,18,19, 20, 21, 22, 23, 24, 25, 26, 27, 28, 29, 30, 31, 32, 33, 34, 35, 36 or more residues of the complementary strand of the first member.<br>
[0027]	The invention provides aldolase-encoding, such as pyruvate aldolase-encoding,<br>
HMG and/or KHG aldolase-encoding nucleic acids generated by amplification, such as polymerase chain reaction (PCR), using an ampJification primer pair in accordance with the invention. In some embodiments, the invention provides aldolase-encoding , such as pyruvate aldolase-encoding, HMG and/or KHG aldolase-encoding nucleic acids generated by amplification, such as polymerase chain reaction (PCR), using an amplification primer pair in accordance with the invention. In some embodiments, the invention provides methods of maldng an aldolase, such as pyruvate aldolase, HMG and/or KHG aldolase enzyme by amplification, such as polymerase chain reaction (PCR), using an amplification primer pair in accordance with the invention. In some embodiments, the amplification primer pair amplifies a nucleic acid from a library, such as a gene library, such as an environmental library.<br>
[0028]	The  invention provides methods  of amplifying a nucleic  acid  encoding a<br>
polypeptide having an aldolase activity, including pyruvate activity such as, without lunitation, HMG and/or KHG aldolase activity comprising amplification of a template nucleic acid with an amplification primer sequence pair capable of amplifying a nucleic acid sequence in accordance with the invention, or fi"agments or subsequences thereof<br>
[0029]	The  invention  provides  expression  cassettes  comprising  a nucleic  acid  in<br>
accordance with the invention or a subsequence thereof. In some embodiments, the expression cassette can comprise the nucleic acid that is operably Imked to a promoter. The promoter can be a viral, bacterial, mammalian, flingal, yeast, or plant promoter. In some embodiments, the plant promoter can be a potato, rice, com, wheat, tobacco or barley promoter. The promoter can be a constitutive promoter. The constitutive promoter can comprise CaMV35S. In other embodiments, the promoter can be an inducible promoter.     In some embodiments, the promoter can be a tissue-specific<br>
 <br>
proraoier or an environmentally regulated or a developmentally regulated promoter. Thus, ihe promoter can be, such as a seed-specific, a leaf-specific, a root-specific, a stem-specific or an abscission-induced promoter, hi some embodiments, the expression cassette can flirther comprise a plant or plant virus expression vector.<br>
(0030]	The invention provides cloning vehicles comprising an expression cassette {such<br>
as a vector) in accordance with the invention or a nucleic acid in accordance with the invention. The cloning vehicle can be a viral vector, a plasmid, a phage, a phagemid, a cosmid, a fosmid, a bacteriophage or an artificial chromosome. The viral vector can comprise an adenovirus vector, a retroviral vector or an adeno-associated viral vector. The cloning vehicle can comprise a bttcterial artificial chromosome (BAC), a plasmid, a bacteriophage Pl-derived vector (PAC), a yeast artificial chromosome (YAC), or a mammalian artificial chromosome (MAC).<br>
[0031]	The invention provides transformed cells comprising nucleic acids in accordance<br>
with tiie invention or expression cassettes (such as vectors) in accordance with the invention, or cloning vehicles in accordance with the invention. In some embodiments, the transformed cell can be a bacterial cell, a mammalian cell, a fimgal cell, a yeast cell, an insect cell or a plant ceil. In some embodiments, the plant cell can be soybeans, rapeseed, oilseed, tomato, cane sugar,, a cereal, a potato, wheat, rice, com, tobacco or barley cell.<br>
[0032]	The invention provides transgenic non-human animals comprising a nucleic acid<br>
in accordance with the invention or an expression cassette (such as a vector) in accordance with the invention. In some embodiments, the animal is a mouse, a rat, a pig, a goat or a sheep.<br>
[0033]	The invention provides transgenic plants comprising a nucleic acid in accordance<br>
with the invention or an expression cassette (such as a vector) in accordance with the invention. The transgenic plant can be a cereal plant, a com plant, a potato plant, a tomato plant, a wheat plant, an oilseed plant, a rapeseed plant, a soybean plant, a rice plant, a barley plant or a tobacco plant.<br>
[0034]	The invention provides transgenic seeds comprising a nucleic acid in accordance<br>
with the invention or an expression cassette (such as a vector) in accordance with the invention.   The transgenic seed can be a cereal plant, a com seed, a wheat kernel, an<br>
 <br>
oilseed, a rapeseed, a soybean seed, a. palm kernel, a sunflower seed, a sesame seed, a peanut or a tobacco plant seed.<br>
[00351	The   invention  provides  antisense  oligonucleotides   comprising  nucleic  acid<br>
sequences complementary to or capable of hybridizing under stringent conditions to nucleic acids in accordance with the invention. In some embodiments, the invention provides methods of inhibiting the translation of an aldolase, such as pyruvate aldolase, HMG and/or KHG aldolase enzyme message in a cell comprising administering to the cell or expressing in the cell an antisense oligonucleotide comprising a nucleic acid sequence complementary to or capable of hybridizing under stringent conditions to a nucleic acid in accordance with the invention, hi some embodiments, the antisense oligonucleotide is about 10 to about 50, about 20 to about 60, about 30 to about 70, about 40 to about 80, or about 60 to about 100 bases in length, such as 10, 15, 20, 25, 30, 35, 40,45, 50, 55, 60, 65, 70, 75, 80, 85, 90,95,100 or more bases in length.<br>
[0036]	The invention provides methods of inhibiting the translation of an aldolase<br>
enzyme, such as pyruvate aldolase, such as HMG and/or KHG aldolase enzyme message in a cell comprisii administering to the cell or expressing in the cell an antisense oligonucleotide comprising a nucleic acid sequence complementary to or capable of hybridizing under stringent conditions to a nucleic acid in accordance witii the invention.<br>
[0037}	The   invention  provides  double-stranded  inhibitory  RNA   (RNAi,   or  RNA<br>
interference) molecules (including small interfering RNA, or siRNAs, for inhibiting transcription, and microRNAs, or miRNAs, for inhibiting translation) comprising a subsequence of a sequence in accordance with the invention. In some embodiments, the siRNA is about 21 to about 24 residues, or, about at least 15, 16, 17, 18, 19, 20, 21, 22, 23, 24, 25, 26, 27, 28, 29, 30, 31, 32, 33, 34, 35, 40, 45, 50, 55, 60, 65, 70, 75, 80, 85, 90, 95, 100 or more duplex nucleotides in length. In some embodiments, the invention provides methods of inhibiting the expression of an aldolase enzyme, such as pyruvate aldolase, such as HMG and/or KHG aldolase enzyme in a cell comprising administering to the cell or expressing in the cell a double-stranded inhibitory RNA (siRNA or miRNA), wherein the RNA comprises a subsequence of a sequence in accordance with the invention.<br>
[0038]	The   invention   provides   isolated,   synthetic   or   recombinant   polypeptides<br>
comprising an amino acid sequence having at least about 50%, 51%, 52%, 53%i, 54%,<br>
 <br>
55%, 56%, 57%, 58%, 59%&gt;, 60%, 61%, 62%, 63%, 64%, 65%,, 66%, 61%, 68%, 69%, 70%, 71%, 72%, 73%, 74%, 75%, 76%, 77%, 78%, 79%, 80%, 81%, 82%, 83%, 84%, 85%, 86%, 87%, 88%, 89%, 90%, 91%, 92%, 93%, 94%,, 95%o, 96%, 97%, 98%, 99%, or more, or complete (100%)) sequence identity to a polypeptide or peptide in accordance with the invention over a region of at least about 10, 15, 20, 25, 30, 35, 40, 45, 50, 55, 60, 65, 70, 75, 80, 85, 90, 95, 100, 125, 150, 175, 200, 225, 250, 275, 300, 325, 350 or more residues, or over the full length of the polypeptide. In some embodiments, the sequence identities are determined by analysis with a sequence comparison algorithm or by a visual inspection. Polypeptide or peptide sequences in accordance with the invention include SEQ ID N0:2, SEQ ID N0:4, SEQ ID N0:6, SEQ ID N0:8, SEQ ID NO:10, SEQ ID N0:12, SEQ ID N0:14, SEQ ID N0:16, SEQ ID K0:18, SEQ ID NO:20, SEQ ID NO:22, SEQ ID NO:24, SEQ ID NO:26, SEQ ID NO:28, SEQ ID NO:30, SEQ ID NO:32, SEQ ID NO:34, SEQ ID NO:36, SEQ ID NO:38, SEQ ID NO:40, SEQ ID ,NO;42, SEQ ID NO.-44, SEQ ID NO:46, SEQ IT&gt; NO:4S, SEQ ID NO:50, SEQ ID NO:52, SEQ ID NO:54, SEQ ID NO:56, SEQ ID NO:58, SEQ ID NO:60, SEQ ID NO:62, SEQ ID NO:64, SEQ ID NO:66, SEQ ID NO:68, SEQ ID NO:70, SEQ ID NO:72, SEQ ID NO;74, SEQ ID NO:76, SEQ ID NO:78, SEQ ID NO:80, SEQ ID NO:82, SEQ ID NO:84, SEQ ID NO:86, SEQ ID NO:88, SEQ ID NO:90, SEQ ID MO:92, SEQ ID NO:94, SEQ ID NO:96, SEQ ID N0:9S, SEQ ID NO:100, SEQ ID NO:102, SEQ ID NO:104, SEQ ID NO:i06, SEQ ID NO:108, SEQ ID NO:110, SEQ ID N0:112, SEQ ID N0:114, SEQ ID N0:116, SEQ ID N0:118, SEQ ID NO:120, SEQ ID NO:122, SEQ ID NO:124, SEQ ID NO:126, SEQ ID NO:128, SEQ ID NO:130, SEQ ID NO:132, SEQ ID NO:134, SEQ ID NO:136, SEQ ID NO:138, SEQ ID NO:140, SEQ ID NO:142, SEQ ID NO:I44, SEQ ID NO:146, SEQ ID NO:148, SEQ ID NO:I50, SEQ ID NO:152, SEQ ID NO:154, SEQ ID NO:156, SEQ ID NO:158, SEQ ID NO:160, SEQ ID NO:162, SEQ ID NO:164, SEQ ID NO:166, SEQ ID NO:168, SEQ ID NO:170, SEQ ID NO:172, SEQ ID NO:174, SEQ ID ND:]76, SEQ ID NO:178, SEQ ID NO:180, SEQ ID NO:I82, SEQ ID NO:184, SEQ ID NO:186, SEQ ID NO:188, SEQ ID NO:190, SEQ ID NO;192, SEQ ID NO:194, SEQ ID NO:196, SEQ ID NO:198, SEQ ID NO:200, SEQ ID NO:202, SEQ ID NO:204, SEQ ID NO:206, SEQ ID NO:208, SEQ ID NO:210, SEQ ID NO:212, SEQ ID NO:214, SEQ ID NO:216, SEQ ID NOi218, SEQ ID NO:220, SEQ ID NO:222, SEQ ID NO:224, SEQ ID NO;226, SEQ ID NO:228, SEQ ID NO;230, SEQ ID<br>
 <br>
NO:232, SEQ ID NO:234, SEQ ID NOi236, SEQ ID NO:238, SEQ ID NO:240, SEQ ID NO:242, SEQ ID NO:244, SEQ ID NO:246, SEQ ID NO:248, SEQ ID NO:250, SEQ ID NO:252, SEQ ID NO:254, SEQ ID NO:256, SEQ ID NO:258, SEQ ID NO:260, SEQ ID NO:262, SEQ ID NO:264, SEQ ID NO:266, SEQ ID NO:268, SEQ ID NO:270, SEQ ID NO:272, SEQ ID NO:274, SEQ ID NO;276, SEQ ID NO:27S, SEQ ID NO:280, SEQ ID NO:282, SEQ ID NO:284, SEQ ID NO:286, SEQ ID NO:288, SEQ ID NO:290, SEQ ID NO:292, SEQ ID NO:294, SEQ ID NO:296, SEQ ID NO:298, SEQ ID NO:300, SEQ ID NO:302, SEQ ID NO:304, SEQ ID NO:306, SEQ ID NO:308, SEQ ID NO:310, SEQ ID NO:312, SEQ ID NO;314, SEQ ID NO:316, SEQ ID NO:318, SEQ ID NO:320, SEQ ID NO:322, SEQ ID NO:324, SEQ ID NO:326, SEQ ID NO:328, SEQ ID NO:330, SEQ ID NO:332, and SEQ ID TS(0:334, and subsequences thereof, variants thereof and enzymatically active fragments thereof Polypeptides in accordance with the invention also include fragments of at least about 10, 15, 20, 25, 30, 35, 40, 45, 50, 75, 80, 85, 90, 95, 100, 150, 200, 250, 300, 350, 400, 450, 500, 550, 600 or more residues in length, or over the full length of an enzyme. Polypeptide or peptide sequences in accordance with the invention include sequence encoded by a nucleic acid in accordance with the invention. Polypeptide or peptide sequences in accordance with the invention include polypeptides or peptides specifically bound by an antibody in accordance with the mvention (such as epitopes), or polypeptides or peptides that can generate an antibody in accordance with the invention (such as an immunogen).<br>
[0039]	In some embodiments, a polypeptide in accordance with the invention has at least<br>
one aldolase enzyme activity, such as pyruvate aldolase, such as HMG and/or KHG aldolase, enzyme activity. In other embodiments, a polynucleotide in accordance with the invention encodes a polypeptide that has at least one aldolase enzyme activity, such as pyruvate aldolase, such as HMG and/or KHG aldolase enzyme activity.<br>
f0040]	Another embodiment of the invention provides isolated, synthetic or recombinant<br>
polypeptides or peptides comprising at least 10, 15, 20, 25, 30, 35, 40, 45, 50, 55, 60, 65, 70, 75, 80, 85, 90, 95, 100,125, 150 or more consecutive bases of polypeptide or peptide sequences in accordance with the invention, sequences substantially identical thereto, and the sequences complementary thereto. The peptide can be, such as an immunogenic fragment, a motif (such as a binding site), a signal sequence, a prepro sequence or an active site.<br>
 <br>
{0041]	The   invention   provides   isolated,   synthetic   or   recombinant   nucleic   acids<br>
comprising a sequence encoding a pol&gt;peptide having an aldolase, such as pyruvate aldolase, such as HMG and/or KHG aldolase, enzyme activity- and a signal sequence, wherein the nucleic acid comprises a sequence iu accordance wath the invention. A "signal sequence" means a secretion signal or other domain that facilitates secretion of the aldolase in accordance with the invention fi:om the host cell. The signal sequence can be derived from another aldolase, such as pyruvate aldolase, such as HMG and/or KHG aldolase enzyme or a non-aldolase, such as non-pyruvate aldolase, such as non-HMG and/or non-KHG-aldolase enzyme (a lieterologous) enzyme. In some embodiments, the invention provides isolated, synthetic or recombinant nucleic acids comprising a sequence encoding a polypeptide having an aldolase, such as pyruvate aldolase, such as HMG and/or KHG aldolase, enzyme activitj', wherein the sequence does not contain a signal sequence and the nucleic acid comprises a sequence in accordance with the invention. In some embodiments, the invention provides isolated, synthetic or recombinant polypeptides comprising polypeptides in accordance with the invention lacking all or part of a signal sequence. In some embodiments, the isolated, synthetic or recombinant polypeptide can comprise the polypeptide in accordance with the invention comprising a heterologous signal sequence, such as a heterologous aldolase, such as pyruvate aldolase, such as HMG and/or KHG dolase enzyme signal sequence or non-aldolase, such as non-pyruvate aldolase, such as non-HMG and/or non-KHG-aldolase enzyme signal sequence.<br>
[0042]	In some embodiments, the invention provides chimeric proteins comprising a first<br>
domain comprising a signal sequence in accordance with the invention and at least a second domain. The protein can be a fusion protein. The second domain can comprise an enzyme. The protein can be a non-enzyme.<br>
[0043]	The invention provides chimeric polypeptides comprising at least a first domain<br>
comprising signal peptide (SP), a prepro sequence and/or a catalytic domain (CD) in accordance with the invention and at least a second domain comprising a heterologous polypeptide or peptide, wherein the heterologous polypeptide or peptide is not naturally associated with the signal peptide (SP), prepro sequence and/ or catalytic domain (CD). In some embodiments, the heterologous polypeptide or peptide is not an aldolase, such e&amp; pyruvate aldolase, HMG and/or KHG aldolase enzyme.  The heterologous polypeptide or<br>
 <br>
peptide can be amino terminaj to, carboxy lerminal to or on both ends of the signal peptide (SP), prepro sequence and/or catalytic domain (CD).<br>
[0044]	The invention provides isolated, synthetic or recombinant nucleic acids encoding<br>
a chimeric polypeptide, wherein the chimeric polypeptide comprises at least a first domain comprising signal peptide (SP), a prepro domain and/or a catalytic domain (CD) in accordance with the invention and at least a second domain comprising a heterologous polypeptide or peptide, wherein the heterologous polypeptide or peptide is not naturally associated with the signal peptide (SP), prepro domain and/ or catalytic domain (CD).<br>
[0045]	The invention provides isolated, synthetic or recombinant signal sequences (such<br>
as signal peptides) comisting of or comprising a sequence as set forth in residues 1 to 14, 1 to 15, 1 to 16, 1 to 17, 1 to 18, 1 to 19, 1 to 20, 1 to 21, 1 to 22, 1 to 23, 1 to 24, 1 to 25, 1 to 26, 1 to 27, 1 to 28, 1 to 28, 1 to 30, 1 to 31, ] to 32, 1 to 33, 1 to 34, 1 to 35, J to 36, 1 to 37, 1 to 38,1 to 40, 1 to 41, 1 to 42,1 to 43,1 to 44,1 to 45, 1 to 46 or 1 to 47, of a polypeptide in accordance with the invention, such as SEQ ID N0:2, SEQ ID NO;4, SEQ ID N0:6, SEQ ID N0:8, SEQ ID NO:10, SEQ ID N0:12, SEQ ID NO:14, SEQ ID N0:I6, SEQ ID N0:18, SEQ ID NO:20, SEQ ID NO:22, SEQ ID NO:24, SEQ ID NO:26, SEQ ID NO:28, SEQ ID NO:30, SEQ ID NO:32, SEQ ID NO:34, SEQ ID NO:36, SEQ ID 10:38, SEQ ID NO:40, SEQ ID NO:42, SEQ ID NO:44, SEQ ID NO:46, SEQ ID NO:48, SEQ ID NO:50, SEQ ID NO:52, SEQ ID NO:54, SEQ ID NO:56, SEQ ID NO:58, SEQ ID NO:60, SEQ ID NO:62, SEQ ID NO:64, SEQ ID NO:66, SEQ ID NO:68, SEQ ID NO:70, SEQ ID NO:72, SEQ ID NO:74, SEQ ID NO:76, SEQ ID NO:78, SEQ ID NO:80, SEQ ID NO:82, SEQ ID NO:84, SEQ ID NO:86, SEQ ID 10:88, SEQ ID NO:90, SEQ ID NO:92, SEQ ID NO:94, SEQ ID NO:96, SEQ ID NO:98, SEQ ID NO:100, SEQ ID NO:102, SEQ ID NO:104, SEQ ID NO; 106, SEQ ID NO: 108, SEQ ID NO: 110, SEQ ID NO: 112, SEQ ID NO: 1J 4, SEQ ID N0:116, SEQ ID NO:! 18, SEQ ID NO:120, SEQ ID NO-.122, SEQ ID NO:124, SEQ ID NO;126, SEQ ID NO:128, SEQ ID NO:130, SEQ ID NO:132, SEQ ID NO:I34, SEQ ID NO:!36, SEQ ID NO:138, SEQ ID NO:140, SEQ ID NO:142, SEQ ID NO:144, SEQ ID NO:146, SEQ ID NO:148, SEQ ID NO:150, SEQ ID NO:152, SEQ ID NO:154, SEQ ID NO:156, SEQ ID NO:158, SEQ ID NO:160, SEQ ID NO: 162, SEQ ID NO:164, SEQ ID NO;166, SEQ ID NO:168, SEQ ID NO:170, SEQ ID NO:i72, SEQ ID NO:174, SEQ ID NO:176, SEQ ID NO:178, SEQ ID NG:180, SEQ ID NO:182, SEQ ID NO:184, SEQ ID<br>
 <br>
NO:186, SEQ ID NO:188, SEQ ID NO:190, SEQ ID NO:192, SEQ ID NO:194, SEQ ID<br>
NO:196, SEQ ID NO:I98, SEQ ID NO:200, SEQ ID NO:202, SEQ ID NO:204, SEQ ID<br>
NO:206, SEQ ID NO:208, SEQ ID NO:210, SEQ ID NO:212, SEQ ID NO:214, SEQ ID<br>
NO:216, SEQ ID NO:218, SEQ ID NO:220, SEQ ID NO:222, SEQ ID NO:224, SEQ ID<br>
NO:226, SEQ ID NO:228, SEQ E) NO:230, SEQ ID NO:232, SEQ ID NO:234, SEQ ID<br>
NO:236, SEQ ID NO:238, SEQ ID NO:240, SBQ ID NO:242, SEQ ID NO:244, SEQ ID<br>
NO:246, SEQ ID NO;248, SEQ ID NO:250, SEQ ID NO:252, SEQ ID NO:254, SEQ ID<br>
NO:256, SEQ ID NO:258, SEQ ID NO:260, SEQ ID NO;262, SEQ ID NO:264, SEQ ID<br>
NO:266, SEQ ID NO:268, SEQ ID NO:270, SEQ ID NO:272, SEQ ID NO:274, SEQ ID<br>
NO:276, SEQ ID NO:278, SEQ ID NO:280, SEQ ID NO:282, SEQ ID NO:284, SEQ ID<br>
NO:286, SEQ ID NO:288, SEQ ID NO-.290, SEQ ID NO:292, SEQ ID NO:294, SEQ ID<br>
NO:296, SEQ ID NO:298, SEQ ID NO:300, SEQ ID NO:302, SEQ ID NO:304, SEQ ID<br>
NO:306, SEQ ID NO:308, SEQ ID NO:310, SEQ ID NO:312, SEQ ID NO:314, SEQ ID<br>
NO;316, SEQ ID NO:318, SEQ ID NO:320, SEQ ID NO:322, SEQ ID NO;324, SEQ ID<br>
NO:326, SEQ ID NO:328, SEQ ID NO:330, SEQ ID NO:332, or SEQ ID NO:334. In<br>
some embodiments, the invention provides signal sequences comprising the first 14, 15,<br>
16, 17, 18, 19, 20, 21, 22, 23, 24, 25, 26, 27, 28, 29, 30, 31, 32, 33, 34, 35, 36, 37, 38, 39,<br>
40, 41,42, 43, 44, 45, 46, 47, 48,49, 50, 51, 52, 53, 54, 55, 56, 57, 58, 59, 60, 61, 62, 63,<br>
64, 65, 66, 67, 68, 69, 70 or more amino terminal residues of a polypeptide in accordance<br>
with the invention.<br>
[0046]	In some embodiments, the aldolase, such as pyruvate aldolase, such as HMG<br>
and/or KHG aldolase enzyme, activity comprises a specific activity fi"om about 10 to about 12,000 units per milligram of protein. In other embodiments, the aldolase, such as pyruvate aldolase, such as HMG and/or KHG aldolase, enzyme activity comprises a specific activity from about 1000 to about 10,000 units per milligram of protein, or, from about 5000 to about 7500 units per milligram of protein. Altematively, the aldolase, such as pyruvate aldolase, such as HMG and/or KHG aldolase enzyme, activity comprises a specific activity in the range from about 10 to about 7500 units per milligram of protein, or, from about 5000 to about 12,000 units per milligram of protein. In some embodiments, the aldolase, such as pyruvate aldolase, such as HMG and/or KHG aldolase enzyme, activity comprises a specific activity in the range from about 10 to about 5000 units per milligram of protein, or, from about 7500 to about 10,000 units per milligram of<br>
 <br>
protein. In order embodiments, the aldolase, such as pyruvate aldolase, such as HMG and/or K-HG aldolase, enzyme activity comprises a specific activity in the range from about 10 to about 2500 units per miiiigram of protein. Alternatively, the aldojase, such as pyruvate aldolase, such as HMG and/or KHG aldolase enzyme, activity comprises a specific activity in the range fi'om about 10 to about 1000 units per milligram of protein. An exemplary method to measure the activity of different aldolases, such as pyruvate aldolases, such as HMG and/or KHG aldolase enzymes, uses a general substrate, 4-carboxy-4-hydroxy-2-oxoadipate ("CHA"). A typical assay comprises 50 mM sodium phosphate pH 7.5, 1 mM MgC]2, 1 mM CHA, 10 figi D-lactate dehydrogenase ("LDH") fi-om Lactobacillus leichmanii (Sigma-Aldrich, St. Louis, MO), 0.5 mM NADH. The assay is started by adding the enzyme to be measured. Liberation of pyruvate, coupled to the formation of NAD+ is monitored continuously in a spectrophotometer at 340rmi. A unit of enzyme activity is defined as the amount that liberates sufficient pyruvate to lower the absorbance at 340nm by 1 OD per minute..<br>
[0047]	In other embodiments, the thermotolerance of the aldolase, such as pyruvate<br>
aldolase, such as HMG and/or KHG aldolase enzyme, comprises retention of at least half of the specific activity of the aldolase, such as pyruvate aldolase, such as HMG and/'or KHG aldolase enzyme after being heated to an elevated temperature, such as a temperature from about 0°C to about 20°C, about 20°C to about 37°C, about 37°C to ahout 50°C, about 50°C to about 70°C, about 70°C to about 75°C, about 75°C to about 80°C, about 80°C to about 85°C, about 85°C to about 90°C, about 90°C to about 95°C, about 95°C to about 100°C, about 100°C to about 110°C, or higher. Alteniatively, the thermotolerance can comprise retention of specific activity fi-om about 10 to about 12,000 imits per milligram of protein, or, from about 5000 to about 10,000 units per milligram of protein, after being heated to an elevated temperature, as described above. In other embodiments, the thermotolerance can comprise retention of specific activity in the range from about. 10 to about 5000 units per milligram of protein after being heated to an elevated temperature, as described above.<br>
[0048]	The  invention  provides  isolated,   synthetic  or recombinant  polypeptides  in<br>
accordance with the invention, wherein the polypeptides comprise at least one glycosylatjon   site.      In   some   emtjodiments,   glj'cosylation   can   be   an   N-ljnked<br>
 <br>
glycosylation. In some embodiments, the polypeptide can be glycosylated after being expressed in a P. pastoris or a S. pombe host or in a mammalian host cell.<br>
[0049]	In some embodiments, the polypeptide can retain aldolase, such as pyruvate<br>
aldolase, such as HMG and/or KHG aldolase enzyme, activity under conditions comprising about pH 6.5, pH 6, pH 5.5, pH 5, pH 4.5, pH 4.0, pH 3.5, pH 3.0 or less (more acidic) pH. In other embodiments, the polypeptide can retain an aldolase such as pyruvate aldolase, such as HMG and/or KHG aldolase enzyme, activity under conditions comprising about pH 7, pH 7.5 pH 8.0, pH 8.5, pH 9, pH 9.5, pH 10, pH 10.5, pH 11.0, pH 11.5, pH 12, pH 12.5 or more (more basic) pH. In some embodiments, the polypeptide can retain an aldolase, such as pyruvate aldolase, such as HMG and/or KHG aldolase enzyme, activity after exposure to conditions comprising about pH 6.5, pH 6, pU 5.5, pH 5, pH 4.5, pH 4.0, pH 3.5, pH 3.0 or less (more acidic) pH. In other embodiments, the po]ypepcide can retain an aldolase, such as pyruvate aldolase, such as HMG and/or KHG aldolase enzyme, activity after exposure to conditions comprising about pH 7, pH 7.5 pH 8.0, pH 8.5, pH 9, pH 9.5, pH 10, pH 10.5, pH 11.0, pH II.5, pH 12, pH 12.5 or more (more basic) pH.<br>
[OOSOJ	In some embodiments, the aldolase, such as pyruvate aldolase, such as HMG<br>
and/or KHG aldolase enzyme in accordance with the invention has activity at under alkaline conditions, such as the alkaline conditions of the gut, such as the small intestine. In some embodiments, the polypeptide can retains activity after exposure to the acidic pH of the stomach.<br>
[0051]	The invention provides protein preparations comprising a polypeptide (including<br>
peptides) in accordance with the invention, wherein the protein preparation comprises a liquid, a solid or a gel. In some embodiments, the invention provides heterodimers comprising a polypeptide in accordance with the invention and a second member, such as a polypeptide or other (second) domain. The second member of the heterodimer can be a different aldolase, such as pyruvate aldolase, such as HMG and/or KHG aldolase enzyme, a different enzyme or another protein. In some embodiments, the second domain can be a polypeptide and the heterodimer can be a flision protein. In some embodiments, the second domain can be an epitope or a lag. In some embodiments, the invention provides homomultimers, including, but not limited to, homodimers, homotrimers, homotetramers.<br>
 <br>
homopentamers, and homohexamers, comprising a polypeptide in accordance with the invention.<br>
[0052]	The invention provides immobilized polypeptides (including peptides) having<br>
aldolase, such as pyruvate aldolase, such as HMG and/or KHG aldolase enzyme, activity, wherein the immobilized polypeptide comprises a polypeptide in accordance with the invention, a polypeptide encoded by a nucleic acid in accordance with the invention, or a polypeptide comprising a polypeptide in accordance with the invention and a second domain. In some embodiments, the polypeptide can be inmiobilized on a cell, a metal, a resin, a polymer, a ceramic, a glass, a microelectrode, a graphitic particle, a bead, a gel, a plate, an array or a capillary tube.<br>
[0053]	The invention also provides arrays comprising an immobilized nucleic acid in<br>
accordance with the invention, including, such as probes in accordance with the invention. In some embodiments, the invention also provides arrays comprising an antibody in accordance with the invention.<br>
[0054]	The   invention  provides   isolated,   synthetic   or  recombinant   antibodies  that<br>
specifically bind to a polypeptide in accordance with the invention or to a polypeptide encoded by a nucleic acid in accordance with the invention. These antibodies in accordance with the invention can be a monoclonal or a polyclonal antibody. In some embodiments, the invenlion provides hybridomas comprising an antibody in accordance with the invention, such as an antibody that specifically binds to a polypeptide in accordance with the invention or to a polypeptide encoded by a nucleic acid in accordance with the invention. In some embodiments, the invention provides nucleic acids encoding these antibodies.<br>
[0055]	The invention provides methods of isolating or identifying polypeptides having<br>
aldolase, such as pyruvate aldolase, such as HMG and/or KHG aldolase, enzyme activity comprising the steps of: (a) providing an antibody in accordance with the invention; (b) providing a sample comprising polypeptides; and (c) contacting the sample of step (b) with the antibody of step (a) under conditions wherein the antibody can specifically bind to the polypeptide, thereby isolating or identifying a polypeptide having an aldolase, such as pyruvate aldolase, such as HMG and/or KHG aldolase enzyme, activit;'.<br>
[0056]	The invention provides methods of making an anti-aldolase, such as anti-pyruvate<br>
aldolase, such as anti-HMG and/or anti-KHG aldolase enzyme antibody comprising<br>
 <br>
administering to a non-human animal a nucleic acid in accordance with the invention or a polypeptide in accordance with the invention or subsequences thereof in an amount sufficient to generate a humoral immrnie response, thereby making an anti-aldolase, such as anti-pyruvate aldolase, such as anti-HMG and/or anti-KHG aldolase enz&gt;'me antibody. In some embodiments, the invention firovides methods of making an anti-aldolase, such as anti-pyruvate aldolase, such as anti-HMG and/or anti-KHG aldolase immune response (cellular or humoral) comprising administering to a non-human animal a nucleic acid in accordance with the invention or a polypeptide in accordance with the invention or subsequences thereof in an amount sufficient to generate an immune response (cellular or humoral).<br>
[0057]	The   invention  provides   methods   of producing   a  recombinant  polypeptide<br>
comprising the steps of: (a) providing a nucleic acid in accordance with the invention operably linked to a promoter; and (b) expressing the nucleic acid of step (a) under conditions that allow expression of the polypeptide, thereby prodxxcing a recombinant polypeptide- In some embodiments, the method can further comprise tiansformmg a host eel! with the nucleic acid of step (a) followed by expressing the nucleic acid of step (a), thereby producing a recombinant polypeptide in a transformed cell.<br>
[0058]	The invention provides methods for identifying a polypeptide having aldolase,<br>
such as pyruvate aldolase, such as HMG and/or KHG aldolase enzyme, activity comprising the following steps: (a) providing a polypeptide in accordance with the invention; or a polypeptide encoded by a nucleic acid in accordance with the invention; (b) providing aldolase, such as pyruvate aldolase, such as HMG and/or KHG aldolase enzyme substrate; and (c) contacting the polypeptide or a fragment or variant thereof of step (a) with the substrate of step (b) and detecting a decrease in the amount of substrate or an increase in the amount of a reaction product, wherein a decrease in the amount of the substrate or an increase in the amount of the reaction product detects a polypeptide having an aldolase, such as pyruvate aldolase, such as HMG and/or KHG aldolase enzyme, activity, hi some embodiments, the substrate is a carbohydrate, a carbohydrate-comprising compound and/or a carbohydrate mimetic.<br>
[0059]	The invention provides methods for identifying aldolase,  such as pyruvate<br>
aldolase, such as HMG and/'or KHG aldolase enzyme substrate comprising the following steps: (a) providing a polypeptide in accordance with the invention; or a polypeptide<br>
 <br>
encoded by a nucleic acid in accordance with the invention; (b) providing a test substrate: and (c) contacting the polypeptide of step (a) with the test substrate of step (b) and detecting a decrease in the amount of substrate or an increase in the amount of reaction product, wherein a decrease in the amount of the substrate or an increase in the amount of a reaction product identifies the test substrate as an aldolase, such as pyruvate aldolase, HMG and/or KHG aldolase enzyme substrate.<br>
[0060]	The  invention provides  metliods  of determining whether a test compound<br>
specifically binds to a polypeptide comprising the following steps: (a) expressing a nucleic acid or a vector comprising the nucleic acid under conditions permissii'e for translation of the nucleic acid to a polypeptide, wherein the nucleic acid comprises a nucleic acid in accordance with the invention, or, providing a polypeptide in accordance with the invention; (b) providing a test compound; (c) contacting the polypeptide with the test compound; and (d) determining whether the test compound of step (b) specifically binds to the polypeptide.<br>
[0061]	The invention provides methods for identifying a modulator of an aldolase, such<br>
as pyruvate aldole, such as HMG and/or KHG aldolase enzyme, activity comprising the following steps: (a) providing a polypeptide in accordance with the invention or a polypeptide encoded by a nucleic acid in accordance with Che invention; (b) providing a test compound; (c) contacting the polypeptide of step (a) with the test compound of step (b) and measuring an activity of the aldolase, such as pyruvate aldolase, such as HMG and/or KHG aldolase enzyme, wherein a change in the aldolase, such as pyruvate aldolase, such as HMG and/or KHG aldolase en2yme, activity measured in the presence of the test compound compared to the activity in the absence of the test compound provides a determination that the test compound modulates the aldolase, such as pyruvate aldolase, such as HMG and/or KHG aldolase enzyme, activity. In some embodiments, the aldolase, such as pyruvate aldolase, such as HMG and/or KHG aldolase enzyme, activity can be measured by providing an aldolase, such as pyruvate aldolase, HMG and/or KHG aldolase enzyme substrate and detecting a decrease in the amount of the substrate or an increase in the amount of a reaction product, or, an increase in the amount of the substrate or a decrease in the amount of a reaction product. A decrease in the amount of the substrate or an increase in the amount of the reaction product with the test compound as compared to the amount of substrate or reaction product without the test<br>
 <br>
compound identifies the test compound as an activator of aldolase, such as pyruvate<br>
aldolase, such as HMG and/or KHC aldolase enzyme, activity. An increase in the<br>
amount of the substrate or a decrease m the amount of the reaction product with the test<br>
compound as compared to the amount of substrate or reaction product without the test<br>
compound identifies the test compound as an inhibitor of aldolase, such as pyruvate<br>
aldolase, such as HMG and/or KHG aldolase enzyme, activity.<br>
[0062]	The invention provides computer systems comprising a processor and a data<br>
storage device wherein said data storage device has stored thereon a polypeptide sequence or a nucleic acid sequence in accordance with the invention (such as a polypeptide or peptide encoded by a nucleic acid in accordance with the invention). Li some embodiments, the computer system can further comprise a sequence comparison algorithm and a data storage device having at least one reference sequence stored thereon, hi other embodiments, the sequence comparison algorithm comprises a computer program that indicates polymorphisms. In some embodiments, the computer system can further comprise an identifier that identifies one or more features in said sequence. In some embodiments, the invention provides computer readable media having stored thereon a polypeptide sequence or a nucleic acid sequence in accordance with the invention. In some embodiments, tlie invention provides methods for identifying a feature in a sequence comprising the steps of: (a) reading the sequence using a computer program which identifies one or more features in a sequence, wherein the sequence comprises a polypeptide sequence or a nucleic acid sequence in accordance with the invention; and (b) identifying one or more features in the sequence with the computer program. In some embodiments, the invention provides methods for comparing a first sequence to a second sequence comprising the steps of; (a) reading the first sequence and the second sequence through use of a computer program which compares sequences, wherein the first sequence comprises a polypeptide sequence or a nucleic acid sequence in accordance with the invention; and (b) determining differences between the first sequence and the second sequence with the computer program. The step of determining differences between the first sequence and the second sequence can farther comprise the step of identifying poJymorphisms. In some embodiments, the method can further comprise an identifier that identifies one or more features in a sequence.    In other<br>
 <br>
embodiments, the method can comprise reading the first sequence using a computer program and ideniiiying one or more features in the sequence.<br>
[0063]	The invention provides methods for isolating or recovering a nucleic acid<br>
encoding a polypeptide having an aldolase, such as pyruvate aldolase, such as HMG and/or KHG aldolase enzyme, activity from a sample, such as an environmental sample, comprising the steps of: (a) providing an amplification primer sequence pair for am.plifying a nucleic acid encoding a polypeptide having an aldolase, such as pyruvate aldolase, such as HMG and/or KHG aldolase enzyme, activity, wherein the primer pair is capable of amplifying a nucleic acid in accordance with the invention; (b) isolating a nucleic acid from the sample or treating the sample such that nucleic acid in the sample is accessible for hybridization to the ajnplification primer pair; and, (c) combining the nucleic acid of step (b) with tbe amj)lification primer pair of step (a) and amplifying nucleic acid from the sample, thereby isolating or recovering a nucleic acid encoding a polypeptide having an aldolase, such as pyruvate aldolase, such as HMG and/or KHG aldolase enzyme, activity from a sample. One or each member of the amplification primer sequence pair can comprise an oligonucleotide comprising an amplification primer sequence pair in accordance with tiie invention, such as having at least about 10 to 50 consecutive bases of a sequence in accordance with the invention. In one embodiment of the invention, the sample is an environmental sample.<br>
{0064]	The invention provides methods for isolating or recovering a nucleic acid<br>
encoding a polypeptide having an aldolase, such as pyruvate aldolase, such as HMG and/or KHG aldolase enzyme, activity from a sample, such as an environmental sample, comprising the steps of: (a) providing a polynucleotide probe comprising a nucleic acid in accordance with the invention or a subsequence thereof; (b) isolating a nucleic acid from the sample or treating the sample such that nucleic acid in the sample is accessible for hybridization to a polynucleotide probe of step (a); (c) combining the isolated nucleic acid or the treated sample of step (b) with the polynucleotide probe of step (a); and (d) isolating a nucleic acid that specifically hybridizes with the polynucleotide probe of step (a), thereby isolating or recovering a nucleic acid encoding a polypeptide having an aldolase, such as pyruvate aldolase, such as HMG and'or KHG aldolase enzyme, activltj' from a sample. The sample can comprise a water sample, a liquid sample, a soil sample, an air sample or a biological sample. In some embodiments, the biological sample can be<br>
 <br>
derived from a bacteria! ceil, a protozoan cell, an insect cell, a yeast cell, a plant cell, a fungal cell or a mammalian cell. In one embodiment of the invention, the sample is an environmental sample.<br>
{0065]	The invention provides methods of generating a variant of a nucleic acid encoding<br>
a polypeptide having an aldolase, such as pyruvate aldolase, such as HMG and/or KHG aldolase enzyme, activity comprising the steps of: (a) providing a template nucleic acid comprising a nucleic acid in accordance with the invention; and (b) modiiying, deleting or adding one or more nucleotides in the template sequence, or a combination thereof, to generate a variant of the template nucleic acid. In some embodiments, the method can further comprise expressing the variant nucleic acid to generate a variant aldolase, such as pyruvate aldolase, such as HMG and/or KHG aldolase enzyme polypeptide. The modifications, additions or deletions can be introduced by a method comprising error-prone PCR, shufHing, oiigonucleotide-directed mutagenesis, assembly FCR, sexual PCR mutagenesis, in vivo mutagenesis, cassette mutagenesis, recursive ensemble mutagenesis, exponential ensemble mutagenesis, site-specific mutenesis, gene reassembly. Gene Site Saturation Mutagenesis (GSSM), synthetic Hgation reassembly (SLR), Chromosomal Saturation Mutagenesis (CSM) or a combination thereof In other embodiments, the modifications, additions or deletions are introduced by a method comprising recombination, recursive sequence recombination, phosphothioate-modified DNA mutagenesis, uracil-containing template mutagenesis, gapped duplex mutagenesis, point mismatch repair mutagenesis, repair-deficient host strain mutagenesis, chemical mutagenesis, radiogenic mutagenesis, deletion mutagenesis, restriction-selection mutagenesis, restriction-purification mutagenesis, artificial gene synthesis, ensemble mutagenesis, chimeric nucleic acid multimer creation and a combination thereof<br>
[0066]	In some embodiments, the method can be iteratively repeated until an aldolase,<br>
such as pyruvate aldolase, HMG and/or KHG aldolase enzyme having an altered or different activity or an altered or different stability from that of a polypeptide encoded by the template nucleic acid is produced. In some embodiments, the variant aldolase, such as pyruvate aldolase, such as HMG and/or KHG aldolase enzyme polypeptide is thermotolerant, and retains some activity after being exposed to an elevated temperature. In other embodiments, the variant aldolase, such as pyruvate aldolase, such as HMG and/or KHG aldolase enzyme polypeptide has increased g)}'Cos}'lation as compared to the<br>
 <br>
aldolase, such as pyruvate aldolase, such as HMG and/or KHG aldolase enzyme encoded by a template nucleic acid. Alternatively, the variant aldolase, such as pyruvate aldolase, such as HMG and/or KHG aldolase polypeptide has an aldolase, such as pyruvate aldolase, such as HMG and/or KHG aldolase enzyme, activity under a high temperature, wherein the aldolase, such as pyruvate aldolase, such as HMG and/or KHG aldolase enzyme encoded by the template nucleic acid is not active under the high temperature. In some embodiments, the method can be iteratively repeated until an aldolase, such as pyruvate aldolase, HMG and/or KHG aldolase en2yme coding sequence having an altered codon usage ftom that of the template nucleic acid is produced, hi other embodiments, the method can be iteratively repeated imtil an aldolase, such as pyruvate aldolase, HMG and/or KHG aldolase enzyme gene having higher or lower level of message expression or stability from that of the template nucleic acid is produced.<br>
[0067]	The invention provides methods for modifying codons in a nucleic acid encoding<br>
a polypeptide having an aldolase, such as pyruvate aldolase, such as HMG and/or KHG aldolase enzyme, activity to incree its expression in a host cell, the method comprising the following steps: (a) providing a nucleic acid in accordance with the invention encoding a polypeptide having an aldolase, such as pyruvate aldolase, such as HMG and/or KHG aldolase enzyme, activity; and, (b) identifying a non-preferred or a less preferred codon in the nucleic acid of step (a) and replacing it with a preferred or neutrally used codon encoding the same amino acid as the replaced codon, wherein a preferred codon is a codon over-represented in coding sequences in genes in the host cell and a non-preferred or less preferred codon is a codon under-represented in coding sequences in genes in the host cell, thereby modifying the nucleic acid to increase its expression in a host cell.<br>
[0068]	The invention provides methods for modifying codons in a nucleic acid encoding<br>
a polypeptide having an aldolase, such as pyruvate aldolase, such as HMG and/or KHG aldolase enzyme, activity, the method comprising the following steps: (a) providing a nucleic acid in accordance with the invention; and, (b) identifying a codon in the nucleic acid of step (a) and replacing it with a different codon encoding the same amino acid as the replaced codon, thereby modifying codons in a nucleic acid encoding an aldolase, such as pyruvate aldolase, HMG and/or KHG aldolase enzyme.<br>
 <br>
[0069]	The invention provides methods for modifying codons in a nucleic acid encoding<br>
a polypeptide having an aldolase, such as pyruvate aldolase, such as HMG and/or KHG aldolase enzyme, activity to increase its expression in a host cell, the method comprising the following steps: (a) providing a nucleic acid in accordance with the invention encoding an aldolase, such as pyruvate aldolase, HMG and/or KHG aldolase enzyme polypeptide; and, (b) identifying a non-preferred or a less preferred codon in the nucleic acid of step (a) and replacing it with a preferred or neutrally used codon encoding the same amino acid as the replaced codon, wherein a preferred codon is a codon over-represented in coding sequences in genes in the host cell and a non-preferred or less preferred codon is a codon under-represented in coding sequences in genes in the host cell, thereby modifying the nucleic acid to increase its expression in a host cell.<br>
[0070]	The invention provides methods for modifying a codon in a nucleic acid encoding<br>
a polypeptide having an aldolase, such as pyruvate aldolase, such as HMG and/or KHG aldolase enzyme, activity to decrease its expression in a host cell, the method comprising the following steps: (a) providing a nucleic acid in accordance with the invention; and (b) identifying at least one preferred codon, in the nucleic acid of step (a) and replacing it with a non-preferred or less preferred codon encoding the same amino acid as the replaced codon, wherein a preferred codon is a codon over-represented in coding sequences in genes in a host cell and a non-preferred or less preferred codon is a codon under-represented in coding sequences in genes in the host cell, thereby modifying the nucleic acid to decrease its expression in a host cell. In some embodiments, the host cell can be a bacterial cell, a fiingal cell, an insect cell, a yeast cell, a plant cell or a mammalian cell.<br>
[0071]	The invention provides methods for producing a library of nucleic acids encoding<br>
a plurality of modified aldolase, such as pyruvate aldolase, such as HMG and/or KHG aldolase enzyme active sites or substrate binding sites, wherein the modified active sites or substrate binding sites are derived from a first nucleic acid comprising a sequence encoding a first active site or a first substrate binding site the method comprising the following steps; (a) providing a first nucleic acid encoding a first active site or first substrate binding site, wherein the first nucleic acid sequence comprises a sequence that hybridizes xmder stringent conditions to a nucleic acid in accordance with the invention, and the nucleic acid encodes an aldolase, such as pyruvate aldolase, HMG and/or KHG aldolase enzyme active site or an aldolase, such as pyruvate aldolase, HMG and/or KHG<br>
 <br>
aldolase enzyme substrate binding site; (b) providing a set of mutagenic oligonucleotides<br>
that encode naturally-occurring amino acid variants at a plurality of targeted codons in the<br>
first nucleic acid; and, (c) using the set of mutagenic oligonucleotides to generate a set of<br>
active site-encoding or substrate binding site-encoding variant nucleic acids encoding a<br>
range of amino acid variations at each amino acid codon that was mutagenized, thereby<br>
producing a library of nucleic acids encoding a plurality of modified aldolase, such as<br>
pyruvate aldolase, such as HMG and/or KHG aldolase enzyme active sites or substrate<br>
binding sites. In some embodiments, the method comprises muiagenizing the first nucleic<br>
acid of step (a) by a method comprising an optimized directed evolution system. Gene<br>
Site Saturation Mutagenesis (GSSM), synthetic ligation reassembly (SLR), error-prone<br>
PCR, shuffling, oligonucleotide-direi:ted mutagenesis, assembly PCR, sexual PCR<br>
mutagenesis, in vivo mutagenesis, cassette mutagenesis, recursive ensemble mutagenesis,<br>
exponential ensemble mutagenesis, site-specific mutagenesis, gene reassembly, and a<br>
combination thereof In other embodiments, the method comprises mutagenizing the first<br>
nucleic acid of step (a) or variants by a method comprising recombination, recursive<br>
sequence recombination, phosphothioate-modified DNA mutagenesis, uracil-containing<br>
template mutagenesis, gapped duplex mutagenesis, point mismatch repair mutagenesis,<br>
repair-deficient host strain mutagenesis, chemical mutagenesis, radiogenic mutagenesis,<br>
deletion mutagenesis, restriction-selection mutagenesis, restriction-puriiication<br>
mutagenesis., artificial gene synthesis, ensemble mutagenesis, chimeric nucleic acid<br>
multimer creation and a combination thereof.<br>
[0072]	The invention provides methods for making a small molecule comprising the<br>
following steps: (a) providing a plurality of biosynthetic enzymes capable of synthesizing or modifying a small molecule, wherein one of the enzymes comprises an aldolase, such as pyruvate aldolase, HMG and/or KHG aldolase enzyme encoded by a nucleic acid in accordance with the invention; (b) providing a substrate for at least one of the enzymes of step (a); and (c) reacting the substrate of step (b) with the enzymes under conditions that facilitate a plurality of biocatalytic reactions to generate a small molecule by a series of biocatalytic reactions. In some embodiments, the invention provides methods for modifying a small molecule comprising the following steps: (a) providing an aldolase, such as pyruvate aldolase, HMG and/or KHG aldolase enzyme, wherein the enzyme comprises a polypeptide in accordance with the invention, or, a polypeptide encoded by a<br>
 <br>
nucleic acid in accordance with the invention, or a subsequence thereof; (b) providing a<br>
small molecule; and (c) reacting the enzyme of step (a) with the small molecule of step<br>
fb) under conditions that facilitate an enzymatic reaction catalyzed by the aldolase, such<br>
as pyruvate aldolase, such as HMG and/or KHG aldolase enzyme, thereby modifying a<br>
small molecule by an aldolase, such as pyruvate aldolase, HMG and/or KHG aldolase<br>
enzymatic reaction. In some embodiments, the method can comprise a plurality of small<br>
molecule substrates for the enzyme of step (a), thereby generating a library' of modified<br>
small molecules produced by at least one enzymatic reaction catalyzed by the aldolase,<br>
such as pyruvate aldolase, such as HMG and/or KHG aldolase enzyme. In some<br>
embodiments, the method can comprise a plurality of additional enzymes under<br>
conditions that facilitate a plurality of biocatalytic reactions by the enzymes to form a<br>
library of modified small molecules produced by the plurality of enzymatic reactions. In<br>
other embodiments, the method can fiuther comprise the step of testing the library to<br>
determine if a particular modified small molecule that exhibits a desired activity is<br>
present within the library. The step of testing the library can further comprise the steps of<br>
systematically eliminating all but one of the biocatalytic reactions used to produce a<br>
portion of the plurality of the modified small molecules within the library by testing the<br>
portion of the modified small molecule for the presence or absence of the particular<br>
modified small molecule with a desired activity, and identifying at least one specific<br>
biocatalytic reaction that produces the particular modified small molecule of desired<br>
activity.<br>
[0073]	The invention provides methods for determining a functional fi-agment of an<br>
aldolase, such as pyruvate aldolase, HMG and/or KHG aldolase enzyme comprising the steps of: (a) providing an aldolase, such as pyruvate aldolase HMG and/or KHG aldolase enzyme, wherein the enzyme comprises a polypeptide in accordance with the invention, or a polypeptide encoded by a nucleic acid in accordance with the invention, or a subsequence thereof; and (b) deleting a plurality of amino acid residues fi-om the sequence of step (a) and testing the remaining subsequence for an aldolase, such as pyruvate aldolase, such as HMG and/or KHG aldolase enzyme, activity, thereby determining a functional fragment of an aldolase, such as pyruvate aldolase, HMG and/or KHG aldolase enzyme. In some embodiments, the aldolase, such as pyruvate aldolase, such as HMG and/or KHG aldolase enzyme, activity is measured by providing an<br>
 <br>
aldolase, such as pyruvate aldolase. JIMG and'or KHG aldolase enzyms substrate and detecting a decrease in the amount of tlie substrate or an increase in the amount of a reaction product.<br>
[0074]	The invention provides methods for whole cell engineering of new or modified<br>
phenotypes by using real-time metabolic flux analysis, the method comprising the following steps; (a) making a modified eel) by modiiying the genetic composition of a cell, wherein the genetic composition is modified by addition to the cell of a nucleic acid in accordance with the invention; (b) culturing the modified cell to generate a plurality of modified cells; (c) measuring at least one metabolic parameter of the celt by monitoring the cell culture of step (b) in real time; and, (d) analyzing the data of step (c) to determine if the measured parameter differs from a comparable measurement in an unmodified cell under similar conditions, thereby identifying an engineered phenotype in the eel! using real-time metabolic ffux analysis, hi some embodiments, the genetic composition of the cell can be modified by a method comprising deletion of a sequence or modification of a sequence in the cell, or, knocking out the expression of a gene. In some embodiments, the method can further comprise selecting a cell comprising a newly engineered phenotype. In other embodiments, the method can comprise culturing the selected cell, thereby generating a new cell strain comprising a newly engineered phenot}'pe.<br>
[0075]	The invention provides methods of increasing thermotolerance or thermostability<br>
of an aldolase, such as pyruvate aldolase, HMG and/or KHG aldolase enzyme polypeptide, the method comprising glycosylating an aldolase, such as pyruvate aldolase, HMG and/or KHG aldolase enzyme polypeptide, wherein the polypeptide comprises at least thirty contiguous amino acids of a polypeptide in accordance with the invention; or a polypeptide encoded by a nucleic acid sequence in accordance with the invention, thereby increasing the thermotolerance or thermostability of the aldolase, such as pyruvate aldolase, such as HMG and/or KHG aldolase polypeptide. In some embodiments, the aldolase, such as pyruvate aldolase, such as HMG and/or KHG aldolase enzyme specific activity can be thermostable or thermotolerant at a temperature in the range fi-om greater than about 37°C to about 95X.<br>
(0076]	The invention provides methods for overexpressing a recombinant aldolase, such<br>
as pyruvate aldolase, such as HMG and/or KHG aldolase polypeptide in a cell comprising expressing a vector comprising a nucleic acid comprising a nucleic acid in accordance<br>
 <br>
with the invention or a nucleic acid sequence in accordance with tiie invention, wherein<br>
the sequence identities are determined by analysis with a sequence comparison algorithm<br>
or by visual inspection, wherein overexpression is effected by use of a high activity<br>
promoter, a dicistronic vector or by gene amplification of the vector.<br>
[0077]	The invention provides methods of making a transgenic plant comprising the<br>
following steps: (a) introducing a heterologous nucleic acid sequence into the cell, wherein the heterologous nucleic sequence comprises a nucleic acid sequence in accordance with the invention, thereby producing a transformed plant cell; and (b) producing a transgenic plant from the transformed cell. In some embodiments, the step (a) can further comprise introducing the heterologous nucleic acid sequence by electroporation or microinjection of plant cell protoplasts. In other embodiments, the step<br>
(a)	can further comprise introducing the heterologous nucleic acid sequence directly to<br>
plant tissue by DNA particle bombaniment. Alternatively, the step (a) can fiirther<br>
comprise infroducing the heterologous nucleic acid sequence into the plant cell DNA<br>
using an Agrobacterium tumefaciens host. In some embodiments, the plant cell can be a<br>
cane sugar, beet, soybean, tomato, potato, com, rice, wheat, tobacco or barley cell.<br>
(0078]	The invention provides methods  of expressing a heterologous nucleic acid<br>
sequence in a plant cell comprising tiie following steps; (a) transforming the plant cell with a heterologous nucleic acid sequence operably linked to a promoter, wherein the heterologous nucleic sequence comprises a nucleic acid in accordance with the invention;<br>
(b)	growing the plant under conditions wherein the heterologous nucleic acids sequence is<br>
expressed in the plant cell. In some embodiments, the invention provides methods of<br>
expressing a heterologous nucleic acid sequence in a plant cell comprising the following<br>
steps: (a) transforming the plant cell with a heterologous nucleic acid sequence operabiy<br>
linked to a promoter, wherein the heterologous nucleic sequence comprises a sequence in<br>
accordance with the invention; (b) growing the plant under conditions wherein the<br>
heterologous nucleic acids sequence is expressed in the plant cell.<br>
[0079]	The invention provides feeds or foods comprising a polypeptide in accordance<br>
with the invention, or a polypeptide encoded by a nucleic acid in accordance with the mvention. In some embodiments, the invention provides foods, feeds, liquids, such as beverages (such as fruit juices or beer), breads or doughs or bread products, or beverage precursors (such as wort), comprising a polypeptide in accordance with the invention. In<br>
 <br>
other embodiments, the invention provides foods, feeds, or beverage additives comprising a polypeptide in accordance with the invention. In some embodiments, the invention provides foods or nutritional supplements, such as for a human or an animal, comprising a polypeptide in accordance with the invention, such as a polypeptide encoded by the nucleic acid in accordance with the invention.<br>
[0080]	In some embodiments, the polypeptide in the food or nutritional supplement can<br>
be glycosylated. In some embodiments, the invention provides edible enzyme delivery matrices comprising a polypeptide in cordance with the invention, such as a polypeptide encoded by the nucleic acid in accordance with the invention. In some embodiments, the delivery matrix comprises a pellet. In some embodiments, the polypeptide can be glycosylated. In some embodiments, the aldolase, such as pyruvate aldolase, such as HMG and/or KHG aldolase enzyme, activity is thermotolerant. In other embodiments, the aldolase, such as pyruvate aldolase, such as HMG and/or KHG aldolase enzyme, activity is thermostable.<br>
[0081]	The invention provides foods, feeds or nutritional supplements comprising a<br>
polypeptide in accordance with the invention. In some embodiments, the invention provides methods for utilizing an aldolase, such as pyruvate aldolase, HMG and'or KHG aldolase enzyme as a nutritional supplement in an animal diet, the method comprising; preparing a nutritional supplement containing an aldolase, such as pyruvate aldolase, HMG and/or KHG aldolase enzyme comprising at least thirty contiguous amino aci6s of a polypeptide in accordance with the invention; and administering the nutritional supplement to an animal. The animal can be a human, a ruminant or a monogastric animal. The aldolase, such as pyruvate aldolase, such as HMG and/or KHG aldolase enzjone can be prepared by expression of a polynucleotide encoding the aldolase, such as pyruvate aldolase, such as HMG and/or KHG aldolase enzyme in an organism selected from the group consisting of a baoteiium, a yeast, a plant, an insect, a fungus and an animal. The organism can be selected ftom the group consisting of an S. pombe, S. cerevisiae, Pichia pastoris, E. coli, Streptomyces sp., Bacillus sp. Pseudomonas sp., Aspergillus sp. and Lactobacillus sp.<br>
[0082]	The   invention   provides   edible   enzyme   deliver}'   matrices   comprising   a<br>
thermostable recombinant aldolase, such as pyruvate aldolase, such as HMG and/or KHG aldolase enzyme, such as a polypeptide in accordance with the invention.    In some<br>
 <br>
embodiments, the invention provides methods for delivering an aldolase, such as pyruvate aldolase, HMG and/or KHG aldolase enzyme supplement to an animal, the method comprising: preparing an edible enzjTne delivery matrix in the form of pellets comprising a granulate edible carrier and a thsmiostabk recombinant aldolase, such as pyruvate aldolase, such as HMG and/or KHG aldolase en2yme, wherein the pellets readily disperse the aldolase, such as pyruvate aldolase, such as HMG and/or KHG aldolase enzyme contained therein into aqueous media, and administering the edible enzyme delivery matrix to the animal. The recombinant aldolase, such as pyruvate aldolase, such as HMG and/or KHG aldolase enzyme can comprise a polypeptide in accordance with the invention. The aldolase, such as pyruvate aldolase, such as HMG and/or KHG aldolase enzyme can be glycosylated to provide thermostability at pelletizing conditions. The deliver' matrix can be formed by pelletizing a mixture comprising a grain germ and an aldolase, such as pyruvate aldolase, HlvlG and/or KHG aldolase enzyme. The pelletizing conditions can include application of steam. The pelletizing conditions can comprise application of a temperature in excess of about 80'C for about 5 minutes and the enzy'me retains a specific activity of at least 3 50 to about 900 units per milligram of enzyme.<br>
|0083]	In some embodiments,  the  invention provides  pharmaceutical  compositions<br>
comprising an aldolase, such as pjTuvate aldolase, HMG and/or KHG aldolase enzyme in accordance with the invention, or a polypeptide encoded by a nucleic acid in accordance with the invention, hi some embodiments, the pharmaceutical composition acts as a digestive aid.<br>
[0084]	In some embodiments, a carbon-carbon bond-containing compound is contacted a<br>
polypeptide in accordance with the invention having aldolase en2yrae activity, such as pyruvate aldolase, such as HMG and/or KHG aldolase enzyme activity, at a pH ranging from about pH 3.0 to about 9.0, about 3.0 to about 10.0, about 3.0 to about 11.0 or more. In other embodiments, a carbon-carbon bond-containing compound is contacted with the aldolase, such as pyruvate aldolase, such as HMG and/or KHG aldolase enzyme, at a temperature of at least about 55°C, 60°C, 65°C, 70°C, 75°C, 80°C, 85°C, 90°C, or more.<br>
[0085]	This disclosure provides, among other things, polypeptides that are useful in<br>
facilitating a reaction in processes for producing monatin, monatin derivatives, and salts thereof, for example in the production of R-2-hydroxy 2-(indol-3yhnethyl)-4-keto glutaric acid {also referred to as R-alpha keto acid monatin, R-monatin precursor, R-MP, and the<br>
 <br>
alpha keto form of monatin), a precm-sor for certain stereoisomers of monatin, such as R,R and S.R monatin. The disclosure also provides methods of making monatin, monatin derivatives, and salts and internal condensation products thereof using one or more polypeptides of the invention. The methods of synthesizing R-MP, stereoisomers of monatin and/or stereoisomers of monatin derivatives include the use of one or more polypeptides with aldolase activity of any of SEQ ID N0:2, SEQ ID N0:4, SEQ ID N0:6, SEQ ID N0:8, SEQ ID NO:10, SEQ ID N0:12, SEQ ID N0:14, SEQ ID N0:16, SEQ ID NO: 18, SEQ ID NO:20, SEQ ID NO;22, SEQ ID NO:24, SEQ ID NO:26, SEQ ID NO:28, SEQ ID NO:30, SEQ ID NO:32, SEQ ID NO:34, SEQ ID NO:36, SEQ ID NO:38, SEQ ID NO:40, SEQ ID NO:42, SEQ ID NO:44, SEQ ID NO:46, SEQ ID NO:48, SEQ ID NO:50, SEQ ID NO:52, SEQ ID NO:54. SEQ ID NO:56, SEQ ID NO:58, SEQ ID NO:60, SEQ ID NO:62, SEQ ID NO:64, SEQ ID NO:66, SEQ ID NO:68, SEQ ID NO:70, SEQ ID NO:72, SEQ ID NO:74, SEQ ID NO:76, SEQ ID NO:78, SEQ ID NO:80, SEQ ID NO:82, SEQ ID NO:84, SEQ ID NO:86, SEQ ID NO:88, SEQ ID NO:90, SEQ ID NO:92, SEQ ID NO:94, SEQ ID NO:96, SEQ ID NO:98, SEQ ID NO.IOO, SEQ ID NO:I02, SEQ ID NO:I04, SEQ ID NO:106, SEQ ID NO:I08, SEQ ID NO:110, SEQ ID N0:112, SEQ ID N0:114, SEQ ID N0:116, SEQ ID N0:118, SEQ ID NO:120, SEQ ID NO:122, SEQ ID NO;124, SEQ ID NO:126, SEQ ID NO:128, SEQ ID NO:130, SEQ ID NO:132, SEQ ID NO:134, SEQ ID NO:136, SEQ ID NO:]38, SEQ ID NO:140, SEQ ID NO:142, SEQ ID N0:144, SEQ ID NO:146, SEQ ID NO:I48, SEQ ID NO:150, SEQ ID NO:!52, SEQ ID NO:154, SEQ ID NO:156, SEQ ID NO:I58, SEQ ID NO:160, SEQ ID NO:162, SEQ ID NO:164, SEQ ID NO:166, SEQ ID NO:I68, SEQ ID NO:170, SEQ ID NO:172, SEQ ID NO:174, SEQ ID NO:176, SEQ ID NO:178, SEQ ID NO:180, SEQ ID NO:182, SEQ ID NO:184, SEQ ID NO:186, SEQ ID NO:188, SEQ ID NO:190, SEQ ID NO:192, SEQ ID NO:194, SEQ ID NO:196, SEQ ID NO:198, SEQ ID NO:200, SEQ ID NO:202, SEQ ID NO:204, SEQ ID NO:206, SEQ ID NO:208, SEQ ID NO:210, SEQ ID NO:212, SEQ ID NO:214, SEQ ID NO:216, SEQ ID NO:218, SEQ ID NO:220, SEQ ID NO:222, SEQ ID NO:224, SEQ ID NO:226, SEQ ID NO:228, SEQ ID NO:230, SEQ ID NO:232, SEQ ID NO:234, SEQ ID NO:236, SEQ ID NO:238, SEQ ID NO:240, SEQ ID NO:242, SEQ ID NO:244, SEQ ID NO:246, SEQ ID NO:248, SEQ ID NO:250, SEQ ID NO:252, SEQ ID NO:254, SEQ ID NO:256, SEQ ID NO:258, SEQ ID NO:260, SEQ ID NO:262, SEQ ID NO:264, SEQ ID NO:266, SEQ ID<br>
 <br>
NO;268, SEQ ID NO:270, SEQ ID NO:272, SEQ ID NO:274, SEQ ID NO:276, SEQ ID<br>
NO:278, SEQ ID NO:280, SEQ ID NO:282, SEQ ID NO:284, SEQ ID NO:286, SEQ ID<br>
NO:288, SEQ ID NO:290, SEQ ID NO:292, SEQ ID NO:294, SEQ ID NO:296, SEQ ID<br>
NO:298, SEQ ID NO;300, SEQ ID NO:302, SEQ ID NO:304, SEQ ID NO:306, SEQ ID<br>
NO:308, SEQ ID NO:310, SEQ ID NO:312, SEQ ID NO:314, SEQ ID NO:316, SEQ ID<br>
NO;318, SEQ ID NO:320, SEQ ID NO:322, SEQ ID NO:324, SEQ ID NO:326, SEQ ID<br>
NO:328, SEQ ID NO:330, SEQ ID NO:332, and SEQ ID NO:334, and enzymatically<br>
active fi-agments thereof.<br>
[0086}	Also,  the methods of synthesizing R-MP,  stereoisomers  of monatin and/or<br>
stereoisomers of monatin derivatives may include the use of a polypeptide with aldolase activity encoded by a nucleic acid sequence having at least about 50%, 51%, 52%, 53%, 54%, 55%, 56%,, 57%, 58%, 59%, 60%, 61%, 62%, 63%, 64%, 65%, 66%., 67%., 68%, 69%, 70%, 71%, 72%,, 73%, 74%, 75%, 76%, 77%, 78%, 79%, 80%, 81%, 82%, 83%, 84%, 85%, 86%, 87%, SS%, 89%, 90%, 91%, 92%, 93%, 94%, 95%, 96%, 91%, 98%, 99%, or more, or complete (100%) sequence identity to nucleic acid in accordance with the invention, including SEQ ID N0:1, SEQ ID N0:3, SEQ ID N0:5, SEQ ID N0:7, SEQ ID N0:9, SEQ IDN0:11, SEQ ID N0:13, SEQ IDN0:15, SEQ ID N0:17, SEQ ID N0:19, SEQ ID N0:21, SEQ ID NO:23, SEQ ID NO:25, SEQ ID NO:27, SEQ ID NO:29, SEQ ID N0:31, SEQ ID NO:33, SEQ ID NO:35, SEQ ID NO:37, SEQ ID NO:39, SEQ ID N0:41, SEQ ID NO:43, SEQ ID NO:45, SEQ ID NO:47, SEQ ID NO:49, SEQ ID N0:51, SEQ ID NO:53, SEQ ID NO:55, SEQ ID NO:57, SEQ ID NO:59, SEQ ID N0:61, SEQ ID NO;63, SEQ ID NO;65, SEQ ID NO;67, SEQ ID NO:69, SEQ ID N0:71, SEQ ID NO:73, SEQ ID NO:75, SEQ ID NO:77, SEQ ID NO:79, SEQ ID N0-.81, SEQ ID NO:83, SEQ ID NO:85, SEQ ID NO:87, SEQ ID NO:89, SEQ ID N0:91, SEQ ID NO:93, SEQ ID NO:95, SEQ ID NO:97, SEQ ID NO:99, SEQ ID NO:101, SEQ ID NO:103, SEQ ID NO:105, SEQ ID NO:107, SEQ ID NO:]09, SEQIDNO:Jn, SEQIDNO.-ID, SEQIDNO-'llS, SEQ ID NO:] 17, SEQ ID N0:l]9, SEQ ID N0:121, SEQ ID NO:I23, SEQ ID NO:125, SEQ ID NO:127, SEQ ID NO:129, SEQ ID N0:131, SEQ ID NO:133, SEQ ID NO:135, SEQ ID NO:137, SEQ ID NO:139, SEQ ID N0:141, SEQ ID NO:143, SEQ ID NO:145, SEQ ID NO:147, SEQ ID NO:149, SEQ ID N0:151, SEQ ID NO:153, SEQ ID NO:155, SEQ ID NO:157, SEQ ID NO:I59, SEQIDN0;I6I, SEQ ID NO;163, SEQlDNO:165, SEQ ID NO.i67, SEQ ID<br>
 <br>
NO;J69, SEQ ID NO:]7I, SEQ ID NO:I73, SEQ ID NO;I75, SEQ ID NO:177, SEQ ID<br>
NO:179, SEQ ID N0:18U SEQ ID NO:183, SEQ ID NO:185, SEQ ID NO:187, SEQ ID<br>
NO:189, SEQ IDN0:19I, SEQ ID NO:193, SEQ ID NO:195, SEQ IDNO:i97, SEQ ID<br>
NO:199, SEQ ID NO:201, SEQ ID NO:203, SEQ ID NO:205, SEQ ID NO:207, SEQ ID<br>
NO:209, SEQ ID N0:2U, SEQ ID NO:213, SEQ ID NO:215, SEQ ID NO:217, SEQ ID<br>
NO:219, SEQ ID NO:221, SEQ ID NO:223, SEQ ID NO:225, SEQ ID NO:227, SEQ ID<br>
NO:229, SEQ ID NO:23I, SEQ ID NO:233, SEQ ID NO:235, SEQ ID NO:237, SEQ ID<br>
NO:239, SEQ ID NO:241, SEQ ID NO:243, SEQ ID NO:245, SEQ ID NO:247, SEQ ID<br>
NO:249, SEQ ID NO:251, SEQ ID NO:253, SEQ ID NO:255, SEQ ID NO:257, SEQ ID<br>
NO:259, SEQ ID NO:26L SEQ ID NO:263, SEQ ID NO:265, SEQ ID NO:267, SEQ ID<br>
NO:269, SEQ ID NO:27I, SEQ ID NO:273, SEQ ID NO:275, SEQ ID NO:277, SEQ ID<br>
NO:279, SEQ ID NO:281, SEQ ID ]sIO:283. SEQ ID NO:285, SEQ ID NO:287, SEQ ID<br>
NO:289, SEQ ID NO:291, SEQ ID NO:293, SEQ ID NO:295, SEQ ID NO:297, SEQ ID<br>
NO:299, SEQ ID NO:301, SEQ ID NO:303, SEQ ID NO:305, SEQ ID NO:307, SEQ ID<br>
NO;309, SEQIDNO:3n,SEQIDNO:3]3, SEQ IDNO:315, SBQIDNO;3I7, SEQ ID<br>
NO:319, SEQ ID NO:321, SEQ ID NO:323, SEQ ID NO:325, SEQ ID NO:327, SEQ ID<br>
NO:329, SEQ ID NO:33I, SEQ ID NO:333, SEQ ID 0:335, SEQ ID NO:336, SEQ ID<br>
NO:337, and SEQ ID NO;338 over a region of at least about 10, 15, 20, 25, 30, 35, 40,<br>
45, 50, 75, !00, 150, 200, 250, 300, 350, 400, 450, 500, 550, 600, 650, 700, 750, 800,<br>
850, 900, 950, 1000, 1050, 1100, 1150, 1200, 1250, 1300, 1350, 1400, 1450, 1500, 1550,<br>
1600, 1650, 1700, 1750, 1800, 1850, 1900, 1950, 2000, 2050, 2100, 2200, 2250, 2300,<br>
2350, 2400, 2450, 2500, or more residues,<br>
[0087]	Furthermore, the methods of synthesizing R-MP, stereoisomers of monatin and/or<br>
stereoisomers of monatin. derivatives may include the use of a polypeptide with aldolase activity encoded by a nucleic acid sequence that hybridizes imder stringent condition to a nucleic acid of SEQ ID N0:1, SEQ ID N0:3, SEQ ID N0:5, SEQ ID N0:7, SEQ ID N0:9, SEQ ID N0:11, SEQ ID N0:13, SEQ ID N0:15, SEQ ID N0il7, SEQ ID N0:19. SEQ ID N0-.21, SEQ ID NO:23, SEQ ID NO:25, SEQ ID NO:27, SEQ ID NO:29, SEQ ID N0:3I, SEQ ID NO:33, SEQ ID NO:35, SEQ ID NO:37, SEQ ID NO:39, SEQ ID N0:41, SEQ ID NO:43, SEQ ID NO:45, SEQ ID NO:47, SEQ ID NO:49, SEQ ID N0:51, SEQ ID NO:53, SEQ ID NO:55, SEQ ID NO:57, SEQ ID NO:59, SEQ ID N0:61, SEQ ID NO:63, SEQ ID NO:65, SEQ ID NO:67, SEQ ID NO:69, SEQ ID<br>
 <br>
N0:71, SEQ ID NO:73, SEQ ID NO:75, SEQ ID NO:77, SEQ ID NO:79, SEQ ID<br>
H0:81, SEQ ID NO:83, SEQ ID NO:85, SEQ ID NO:87, SEQ ID NO:89, SEQ ID<br>
N0:9!, SEQ ID NO:93, SEQ ID NO:95, SEQ ID NO:97, SEQ ID NO:99, SEQ ID<br>
NO:I01, SEQ ID NO:I03, SEQ ID NO:I05, SEQ ID NO:I07, SEQ ID NO:i09, SEQ ID<br>
N0:111. SEQ ID N0:113, SEQ ID NOiHS, SEQ ID N0:117, SEQ ID N0:119. SEQ ID<br>
N0:12I, SEQ ID NO:123, SEQ ID NO:125, SEQ ID NO:I27, SEQ ID NO:129, SEQ ID<br>
N0;131, SEQ ID NO:133, SEQ ID NO:I35, SEQ ID NO;I37, SEQ ID NO:I39, SEQ ID<br>
N0:141, SEQ IDNO:143, SEQIDNO:145, SEQ IDNO:147, SEQ IDNO:I49, SEQ ID<br>
N0:151, SEQ ID NO:153, SEQ ID NO:I55, SEQ ID NO:157, SEQ ID NO-.159, SEQ ID<br>
N0:161, SEQ ID NO;I63, SEQ ID NO:165, SEQ ID NO:167, SEQ ID NO:]69, SEQ ID<br>
N0:171, SEQ ID NO:I73, SEQ ID NO:I75, SEQ ID NO:177, SEQ ID NO:179, SEQ ID<br>
N0:181, SEQ ID NO:183, SEQ ID NO:185, SEQ ID NO:i87, SEQ ID NO:189, SEQ ID<br>
N0:19l, SEQ ID NO:193, SEQ ID NO:195, SEQ ID NO:197, SEQ ID NO:199, SEQ ID<br>
NO;201, SEQ ID NO:203, SEQ ID NO:205, SEQ ID NO:207, SEQ ID NO:209, SEQ ID<br>
NO:2n, SEQ ID NO:213, SEQ ID NO:2I5, SEQ ID NO:2i7, SEQ ID NO:219, SEQ ID<br>
NO:221, SEQ ID NO:223, SEQ ID NO:225, SEQ ID NO:227, SEQ ID NO:229, SEQ ID<br>
NO:231, SEQ ID NO:233, SEQ ID NO:235, SEQ ID NO:237, SEQ ID NO:239, SEQ ID<br>
NO:241, SEQ ID NO:243, SEQ ID NO:245, SEQ ID NO:247, SEQ ID NO:249, SEQ ID<br>
NO:251, SEQ ID NO:253, SEQ ID NO:255, SEQ ID NO:257, SEQ ID NO:259, SEQ ID<br>
NO:261, SEQ ID NO:263, SEQ ID NO:265, SEQ ID NO:267, SEQ ID NO:269, SEQ ID<br>
NO:271, SEQ ID NO:273, SEQ ID NO:275, SEQ ID NO:277, SEQ ID NO:279. SEQ ID<br>
NO:281, SEQ ID NO:283, SEQ ID NO:285, SEQ ID NO:287, SEQ ID NO:289, SEQ ID<br>
NO:29!, SEQ ID NO:293, SEQ ID NO:295, SEQ ID NO;297, SEQ ID NO:299, SEQ ID<br>
NO:301, SEQ ID NO;303, SEQ ID NO;305, SEQ ID NO:307, SEQ ID NO:309, SEQ ID<br>
N0:311, SEQ ID NO:313, SEQ ID NO:315, SEQ ID NO:317, SEQ ID NO:319, SEQ ID<br>
NO:321, SEQ ID NO;323, SEQ ID NO:325, SEQ ID NO:327, SEQ ID NO:329, SEQ ID<br>
NO:331, SEQ ID NO:333, SEQ ID NO:335, SEQ ID NO:336, SEQ ID NO:337, and SEQ<br>
IDNO:338.<br>
[0088]	The invention provides a method, comprising: producing a product chosen from<br>
monatin, monatin derivatives, salts tliereof, and combinations thereof in a multi-step pathviJay, wherein a reaction in the pathway is facilitated by one or more polypeptides chosen from isolated or recombinant polypeptides comprising the amino acid sequence of<br>
 <br>
SEQ ID N0:2, SEQ ID N0:4, SEQ JD N0:6, SEQ ID NO:8, SEQ ID NO:10, SEQ ID N0:]2, SEQ ID N0:14, SEQ ID N0:16, SEQ ID N0:18, SEQ ID NO:20, SEQ ID NO;22, SEQ ID NO;24, SEQ ID NO;26, SEQ ID NO:28, SEQ ID NO:30, SEQ ID NO:32, SEQ ID NO:34, SEQ ID NO:36, SEQ ID NO:38, SEQ ID NO:40, SEQ ID NO:42, SEQ ID NO:44, SEQ ID NO:46, SEQ ID NO:48, SEQ ID NO:50, SEQ ID NO:52, SEQ ID NO:54, SEQ ID NO:56, SEQ ID NO:58, SEQ ID NO:60, SEQ ID NO:64, SEQ ID NO:66, SEQ ID NO:68, SEQ ID NO;70, SEQ ID NO:72, SEQ ID NO:74, SEQ ID NO:76, SEQ ID NO:78, SEQ ID NO:80, SEQ ID NO:82, SEQ ID N0:S4, SEQ ID NO:86, SEQ ID NO:88, SEQ ID NO:90, SEQ ID NO:92, SEQ ID NO:94, SEQ ID NO:96, SEQ ID KO:98, SEQ ID NO:100, SEQ ID NO;102, SEQ ID NO:106, SEQ ID NO:108, SEQ ID NO:110, SEQ ID N0:1I2, SEQ ID N0:114, SEQ ID N0:116, SEQ ID N0:l!8, SEQ ID NO:120, SEQ ID NO:122, SEQ ID NO:124, SEQ ID NO:i26, SEQ ID NO:128, SEQ ID NO:130, SEQ ID NO:132, SEQ ID NO:134, SEQ ID NO;136, SEQ ID NO:138, SEQ ID NO:140, SEQ ID NO:142, SEQ ID NO:144, SEQ ID NO:I46, SEQ ID 10:148, SEQ ID NO:150, SEQ ID NO:152, SEQ ID NO:I54, SEQ ID NO:156, SEQ ID NO:158, SEQ ID NO:160, SEQ ID NO:162, SEQ ID NO:164, SEQ ID NO:166, SEQ ID NO:168, SEQ ID NO:170, SEQ ID NO:172, SEQ ID NO:174, SEQ ID NO:176, SEQ ID NO:178, SEQ ID NO:180, SEQ ID NO:182, SEQ ID NO:184, SEQ ID NO:186, SEQ ID NO:188, SEQ ID NO:190, SEQ ID NO:192, SEQ ID NO:194, SEQ ID NO: 196, SEQ ID NO: 198, SEQ ID NO:200, SEQ ID NO:204, SEQ ID NO:206, SEQ ID NO:208, SEQ ID NO:210, SEQ ID NO:212, SEQ ID NO:214, SEQ ID NO:216, SEQ ID NO:218, SEQ ID NO:220, SEQ ID NO:222, SEQ ID NO:224, SEQ ID NO:226, SEQ ID NO:228, SEQ ID NO:230, SEQ ID NO:232, SEQ ID NO:234, SEQ ID NO:236, SEQ ID NO:238, SEQ ID NO:240, SEQ ID NO:242, SEQ ID NO:244, SEQ ID NO:246, SEQ ID NO:248, SEQ ID NO:250, SEQ ID NO:252, SEQ ID NO:254, SEQ ID NO:256, SEQ ID NO:258, SEQ ID NO:260, SEQ ID NO:262, SEQ ID NO:264, SEQ ID NO:266, SEQ ID NO:268, SEQ ID NO:270, SEQ ID NO:272, SEQ ID NO:274, SEQ ID NO:276, SEQ ID NO:278, SEQ ID NO:282, SEQ ID NO:284, SEQ ID NO:286, SEQ ID NO:288, SEQ ID NO:290, SEQ ID NO:292, SEQ ID NO:294, SEQ ID NO:296, SEQ ID NO:298, SEQ ID NO:300, SEQ ID NO:302, SEQ ID NO:304, SEQ ID NO:306, SEQ ID NO:308, SEQ ID NO:310, SEQ ID NO:312, SEQ ID NO:314, SEQ ID NO:316, SEQ ID NO:318, SEQ ID NO:320, SEQ ID NO:322, SEQ ID NO:324, SEQ ID NO:326, SEQ ID NO:32S, SEQ ID<br>
 <br>
NO:330, SEQ ID NO:332, or SEQ ID NO:334, or fragments or subsequences thereof having aldolase activity. In some embodiments, the fragments or subsequences thereof have an aldolase activitj' of at least 0.2 mg MP/mg protein/hr. In other embodiments, the fragments or subsequences thereof have an aldolase activity of at least 0.1 mg MP/mg protein/hr. In some embodiments, the reaction facilitated by one or more polypeptides in accordance with the invention is performed in about 1.0 to about 30.0 mM MgC!2. In other embodiments, the reaction facilitated by one or more polypeptides in accordance with the invention is performed at about pH 7.0 to about pH 11.5. In still other embodiments, the reaction facilitated by one or more polypeptides in accordance with the invention is performed in about 0.005% to about 1% polysorbate detergent.<br>
[0089]	In some embodiments, the reaction is a reaction between indole-3-pyruvate and a<br>
C3 carbon source. In some embodiments, the reaction preferentially produces R-2-hydroxy-2-(indo]-3-yl-methy])-4-ketoglutaric acid over S—2-hydroxy-2-(indol-3-yl-methyl)-4-ketogIutaric acid.<br>
[0090]	In some embodiments, the product made by the multi-step pathway is monatin,<br>
salts thereof and combinations thereof.<br>
[0091]	In  other  embodiments,   at  least  one   of R,R-monatin,  R-S   monatin,  or  a<br>
combination thereof is produced in greater quantity than either S,S-monatin or S,R-monatin in the multi-step pathway. In some embodiments, R,R monatin is produced in greater quantity than R,S-monatin, S,S-monatin and S,R-monatin in the multi-step pathway.<br>
[0092]	The invention provides a method, comprising; producing a product chosen from<br>
monatin, monatin derivatives, salts thereof, and combinations thereof in a multi-step pathway, wherein a reaction in the pathway is facilitated by at least one polypeptide encoded by a nucleic acid sequence that comprises a sequence having a percent sequence identity of at least 50% to SEQ ID N0:1, SEQ ID N0:3, SEQ ID N0:5, SEQ ID N0:7, SEQ ID N0:9, SEQ ID N0:11, SEQ ID N0:13, SEQ ID N0:15, SEQ ID N0:17, SEQ ID N0:19, SEQ ID N0:21, SEQ ID NO:23, SEQ ID NO:25, SEQ ID NO:27, SEQ ID NO:29, SEQ ID NO:3I, SEQ ED NO;33, SEQ ID NO:35, SEQ ID NO:37, SEQ ID NO:39, SEQ ID N0:41, SEQ ID NO:43, SEQ ID NO:45, SEQ ID NO:47, SEQ ID NO:49, SEQ ID N0:51, SEQ ID N0;61, SEQ ID NO:63, SEQ ID NO:65, SEQ ID NO:67, SEQ ID NO:69, SEQ ID N0:7I, SEQ ID NO:73, SEQ ID NO:75, SEQ ID<br>
 <br>
NO:77, SEQ ID NO:79, SEQ ID N0:81, SEQ ID NO:83, SEQ ID NO:85, SEQ ID<br>
N0:S7, SEQ ID NO:89, SEQ ID N0:91, SEQ ID NO:93, SEQ ID NO:95, SEQ ID<br>
NO:97, SEQ ID NO:99, SEQ ID NO:103, SEQ ID NO:105, SEQ ID NO:107, SEQ ID<br>
NO:109, SEQ ID NOiUL SEQ IDN0:113, SEQ ID N0:115, SEQ ID N0:117, SEQ ID<br>
NO:U9, SEQ ID NO:12I, SEQ ID NO:I23, SEQ ID NO:125, SEQ ID NO:127, SEQ ID<br>
NO:129, SEQ ID N0:131, SEQ ID NO:133, SEQ ID NO:135, SEQ ID NO:137, SEQ ID<br>
NO:139, SEQ ID N0:141, SEQ ID NO:143, SEQ ID NO:145, SEQ ID NO:147, SEQ ID<br>
NO:149, SEQ ID N0;]5I, SEQ ID NO:153, SEQ ID NO:]55, SEQ ID NO:157, SEQ ID<br>
NO:159, SEQ ID N0:161, SEQ ID NO:163, SEQ ID NO:165, SEQ ID NO:167, SEQ ID<br>
NO:169, SEQ ID N0:171, SEQ ID NO:173, SEQ ID NO:175, SEQ ID NO:177, SEQ ID<br>
NO:179, SEQ ID N0:]81, SEQ ID NO:]83, SEQ JD NO:J85, SEQ ID NO:]87, SEQ ID<br>
NO:189, SEQ ID N0:19U SEQ ID NO:193, SEQ ID NO:195, SEQ ID NO:197, SEQ ID<br>
NO:199, SEQ ID NO:203, SEQ ID NO:205, SEQ ID NO:207, SEQ ID NO:209, SEQ ID<br>
N0:211, SEQ ID NO:213, SEQ ID NO:215, SEQ ID NO:2i7, SEQ ID NO:219, SEQ ID<br>
NO:221, SEQ ID NO:223, SEQ ID NO:225, SEQ ID NO:227, SEQ ID NO:229, SEQ ID<br>
NO:231, SEQ ID NO:233, SEQ ID NO:235, SEQ ID NO:237, SEQ ID NO:239, SEQ ID<br>
NO:241, SEQ ID NO:243, SEQ ID NO:245, SEQ ID NO:247, SEQ ID NO:249, SEQ ID<br>
NO:251, SEQ ID NO:253, SEQ ID NO:255, SEQ ID NO:257, SEQ ID NO:259, SEQ ID<br>
NO:261, SEQ ID NO:263, SEQ ID NO:265, SEQ ID NO:267, SEQ ID NO:269, SEQ ID<br>
NO:27E SEQ ID NO:273, SEQ ID NO:275, SEQ ID NO:277, SEQ ID NO:281, SEQ ID<br>
NO:283, SEQ ID NO:285, SEQ ID NO:287, SEQ ID NO:289, SEQ ID NO:291, SEQ ID<br>
NO:293, SEQ ID NO:295, SEQ ID NO:297, SEQ ID NO:299, SEQ ID NO:30I, SEQ ID<br>
NO:303, SEQ ID NO:305, SEQ ID NO:307, SEQ ID NO:309, SEQ ID N0:311, SEQ ID<br>
NO:313, SEQ ID NO:315, SEQ ID NO:317, SEQ ID NO:319, SEQ ID NO:321, SEQ ID<br>
NO:323, SEQ ID NO:325, SEQ ID NO:327, SEQ ID NO:329, SEQ ID NO;33I, SEQ ID<br>
NO:333, SEQ ID NO:335, SEQ ID NO:336, SEQ ID NO:337, or SEQ ID NO:338. In<br>
some embodiments, the percent sequence identity is at least 95%. In other embodiments,<br>
the percent sequence identity is 100%.<br>
[0093]	The  invention provides a method comprising a reaction that preferentially<br>
produces R-2-hydroxy-2-(indol-3-y]-methyI)-4-ketogiutaric acid over S~2-hydroxy-2-(indo]-3-yJ-methy])-4-ketogiutaric acid wherein at least one polypeptide encoded by a nucleic acid sequence that comprises a sequence having at least 95% sequence identit&gt;' to<br>
 <br>
SEQ ID NO: 28, SEQ ID N0:116, SEQ ID NO:298, SEQ ID NO: 44, SEQ ID NO:54,<br>
SEQ ID NO: 148, SEQ ID NO: 46, SEQ ID NO;134, SEQ ID NO:142, SEQ ID NO:122,<br>
SEQ ID NO:74, SEQ ID NO: 64, SEQ ID NO: 108, SEQ ID NO:96, SEQ ID NO:126,<br>
SEQ ID NO:80, SEQ ID NO:36, SEQ ID NO:62, SEQ ID N0:112, SEQ ID NO:130,<br>
SEQ ID NO:94, SEQ ID NO;58, SEQ ID NO:50, SEQ ID NO:106, SEQ ID NO:42, SEQ<br>
ID NO:278, SEQ ID NO:162, SEQ ID NO:276, SEQ ID NO:178, SEQ ID NO:166, SEQ<br>
ID NO:2]8, SEQ ID NO:224, SEQ ID NO;226, SEQ ID NO:244, SEQ ID NO:250, SEQ<br>
ID NO:252, SEQ ID NO:264, SEQ ID NO:268, SEQ ID NO:272, SEQ ID NO: 184, SEQ<br>
ID NO:282, SEQ ID NO:186, SEQ ID NO:192, SEQ ID NO:200, SEQ ID NO:284, SEQ<br>
ID NO:172, SEQ ID NO:180, SEQ ID NO:168, SEQ ID NO:228, SEQ ID NO:236, SEQ<br>
ID NO-.238, SEQ ID NO:240, and SEQ ID NO:156 is utilized to faciliate one reaction in a<br>
multi-step pathway.<br>
[0094]	The invention provides a method comprising: producing a product chosen from<br>
monatin, monatin derivatives, salts thereof, and combinations thereof in a multi-step pathway, wherein a reaction in the pathway is facilitated by at least one polypeptide encoded by a nucleic acid sequence that comprises a sequence that hybridizes under stringent condition to a nucleic acid of SEQ ID N0:1, SEQ ID N0:3, SEQ ID N0:5, SEQ ID N0:7, SEQ ID N0:9, SEQ ID NO;U, SEQ ID N0:13, SEQ ID NO:I5, SEQ ID N0:17, SEQ ID N0:19, SEQ ID N0:21, SEQ ID NO:23, SEQ ID NO:25, SEQ ID NO:27, SEQ ID NO:29, SEQ ID N0:3i, SEQ ID NO:33, SEQ ID NO:35, SEQ ID N0:37, SEQ ID N0:39, SEQ ID N0:41, SEQ ID NO:43, SEQ ID NO:45, SEQ ID NO:47, SEQ ID NO:49, SEQ ID N0:51, SEQ ID NO:53, SEQ ID NO:55, SEQ ID NO:57, SEQ ID NO:59, SEQ ID NO:63, SEQ ID NO;65, SEQ ID NO:67, SEQ ID NO:69, SEQ ID N0:71, SEQ ID NO:73, SEQ ID NO:75, SEQ ID NO:77, SEQ ID N0:79, SEQ ID N0:81, SEQ ID NO:83, SEQ ID N0:85, SEQ ID NO:87, SEQ ID NO:89, SEQ ID N0:91, SEQ ID NO:93, SEQ ID NO:95, SEQ ID NO:97, SEQ ID NO;99, SEQ ID NO:103, SEQ ID NO:105, SEQ ID NO:107, SEQ ID NO:109, SEQ ID NOill 1, SEQ ID N0:113, SEQ ID N0:115, SEQ ID N0:117, SEQ ID N0:119, SEQ ID NO:]22, SEQ ID J
 <br>
N0:161, SEQ ID NO;]63, SEQ ID NO:165, SEQ ID NO:I67, SEQ ID NO:169, SEQ ID<br>
N0:171, SEQ ID NO:173, SEQ ID NO:175, SEQ ID NO:177, SEQ ID NO:179, SEQ ID<br>
N0:I81, SEQ ID NO:I83, SEQ ID NO:185, SEQ ID NO:187, SEQ ID NO:189, SEQ ID<br>
N0:191, SEQ ID NO:193, SEQ ID NO:195, SEQ ID NO:197, SEQ ID NO:199, SEQ ID<br>
NO:203, SEQ ID NO:205, SEQ ID NO:207, SEQ ID NO;209, SEQ ID N0:2n, SEQ ID<br>
NO:213, SEQ ID NO:215, SEQ ID NO:217, SEQ ID NO:219, SEQ ID NO:221, SEQ ID<br>
NO:223, SEQ ID NO:225, SEQ ID NO:227, SEQ ID NO:229, SEQ ID NO:23U SEQ ID<br>
NO;233, SEQ ID NO:235, SEQ ID NO:237, SEQ ID NO:239, SEQ ID NO:242, SEQ ID<br>
NO:243, SEQ ID NO:245, SEQ ID NO:247, SEQ ID NO:249, SEQ ID NO:251, SEQ ID<br>
NO;253, SEQ ID NO:255, SEQ ID NO:257, SEQ ID NO:259, SEQ ID NO:26I, SEQ ID<br>
NO;263, SEQ ID NO:265, SEQ ID NO:267, SEQ ID NO:269, SEQ ID NO:271, SEQ ID<br>
NO:273, SEQ ID NO:275, SEQ ID NO:277, SEQ ID NO:281, SEQ ID NO:283, SEQ ID<br>
NO;285, SEQ ID NO;287, SEQ ID NO:289, SEQ ID NO:291, SEQ ID NO:293, SEQ ID<br>
NO:295, SEQ ID NO:297, SEQ ID NO:299, SEQ ID NO:301, SEQ ID NO:303, SEQ ID<br>
NO:305, SEQ ID NO;307, SEQ ID NO;309, SEQ ID N0:3] I, SEQ ID NO:313, SEQ ID<br>
NO:315, SEQ ID NO:317, SEQ ID NO:319, SEQ ID NO:321, SEQ ID NO:323, SEQ ID<br>
NO:325, SEQ ID NO;327, SEQ ID NO:329, SEQ ID NO:33L SEQ ID NO:333, SEQ ID<br>
NO:335, SEQ ID NO:336, SEQ ID NO:337, or SEQ ID NO:338.<br>
[0095]	The invention also provides a method comprising: producing a product chosen<br>
from monatin precursor, salts thereof, and combinations thereof, in a multi-step pathway, wherein a reaction in the pathway is facilitated by one or more polypeptides chosen from isolated or recombinant polypeptides comprising the amino acid sequence of SEQ ID NO:2, SEQ ID N0:4, SEQ ID N0:6, SEQ ID NO:S, SEQ ID NO:10, SEQ ID N0:I2, SEQ ID N0;14, SEQ ID NO:i6, SEQ ID NO;I8, SEQ ID NO:20, SEQ ID NO;22, SEQ ID NO:24, SEQ ID NO:26, SEQ ID NO:28, SEQ ID NO:30, SEQ ID NO:32, SEQ ID NO:34, SEQ ID NO:36, SEQ ID NO:38, SEQ ID NO:40, SEQ ID NO:42, SEQ ID NO:44, SEQ ID NO:46, SEQ ID NO:48, SEQ ID NO:50, SEQ ID NO:52, SEQ ID NO:54, SEQ ID NO:56, SEQ ID NO:58, SEQ ID NO:60, SEQ ID NO:64, SEQ ID NO:66, SEQ ID NO:68, SEQ ID NO:70, SEQ ID NO:72, SEQ ID NO:74, SEQ ID NO:76, SEQ ID NO:78, SEQ ID NO:80, SEQ ID NO:82, SEQ ID NO:84, SEQ ID N0:S6, SEQ ID NO:88, SEQ ID NO:90, SEQ ID NO:92, SEQ ID NO:94, SEQ ID NO:96, SEQ ID NO:98, SEQ ID NO:100, SEQ ID NO:104, SEQ ID NO;106, SEQ ID<br>
 <br>
NO:i08, SEQ ID NO:1IO, SEQ ID NO:II2, SEQ ID N0:n4, SEQ ID NO:I16, SEQ ID<br>
N0:l]8, SEQIDNO:120, SEQIDNO:122, SEQ ID NO:124, SEQ ID NO:126, SEQ ID<br>
NO:128, SEQIDNO:130, SEQ IDM):I32, SEQ IDNO:134, SEQ IDNO:136, SEQ ID<br>
NO:138, SEQ IDNO:140, SEQ IDNO:142, SEQ IDNO:144, SEQ IDNO:146, SEQ ID<br>
NO:148, SEQ ID NO:150, SEQ ID NO:152, SEQ ID NO:154, SEQ ID NO:156, SEQ ID<br>
NO:158, SEQ ID NO:160, SEQ ID NO:162, SEQ ID NO:164, SEQ ID NO:166, SEQ ID<br>
NO:168, SEQ ID NO:170, SEQ ID NO:172, SEQ ID NO:174, SEQ ID NO:176, SEQ ID<br>
NO:178, SEQIDNO:180, SEQ IDNO:182, SEQ IDNO:184, SEQ IDNO:186, SEQ ID<br>
NO:lS8, SEQ ID NO:I90, SEQ ID NO;192, SEQ ID NO:I94, SEQ ID NO;I96, SEQ ID<br>
NO:198, SEQ ID NO:200, SEQ ID NO;204, SEQ ID NO:206, SEQ ID NO:208, SEQ ID<br>
NO:210, SEQ ID NO:212, SEQ ID NO;214, SEQ ID NO;216, SEQ ID NO:218, SEQ ID<br>
NO:220, SEQ ID NO:222, SEQ ID NO:224, SEQ ID NO:226, SEQ ID NO:228, SEQ ID<br>
NO:230, SEQ ID NO:232, SEQ ID NO:234, SEQ ID NO:236, SEQ ID NO:238, SEQ ID<br>
NO:240, SEQ ID NO:242, SEQ ID NO:244, SEQ ID NO:246, SEQ ID NO:248, SEQ ID<br>
NO:250, SEQ ID NO:252, SEQ ID NO:254, SEQ ID NO;256, SEQ ID NO:258, SEQ ID<br>
NO:260, SEQ ID NO:262, SEQ ID NO:264, SEQ ID NO:266, SEQ ID NO:268, SEQ ID<br>
NO:270, SEQ ID NO:272, SEQ ID NO;274, SEQ ID NO;276, SEQ ID NO;278, SEQ ID<br>
NO:282, SEQ ID NO:284, SEQ ID NO:286, SEQ ID NO:288, SEQ ID NO:290, SEQ ID<br>
NO:292, SEQ ID NO:294, SEQ ID NO:296, SEQ ID NO:298, SEQ ID NO:300, SEQ ID<br>
NO:302, SEQ ID NO:304, SEQ ID NO:306, SEQ ID NO:308, SEQ ID NO:310, SEQ ID<br>
NO:312, SEQ ID NO:314, SEQ ID NO:316, SEQ ID NO:318, SEQ ID NO:320, SEQ ID<br>
NO:322, SEQ ID NO:324, SEQ ID NO:326, SEQ ID NO:328, SEQ ID NO:330, SEQ ID<br>
NO:332, or SEQ ID NO:334, or fragments or subsequences thereof having aldolase<br>
activity, wherein said monatin precursor, salts thereof, and combinations thereof is sweet.<br>
[0096]	The invention additionally provides a method comprising: producing a product<br>
chosen from monatin precursor, salts thereof, and combinations thereof, in a multi-step pathway, wherein a reaction in the pathway is facilitated by at least one polypeptide encoded by a nucieic acid sequence that comprises a sequence having a percent sequence identity of at least 50% to SEQ ID N0:1, SEQ ID N0:3, SEQ ID N0:5, SEQ ID N0:7, SEQ ID N0:9, SEQ ID N0:11, SEQ ID N0:13, SEQ ID N0:i5, SEQ ID N0:17, SEQ ID N0:19, SEQ ID N0:21, SEQ ID NO:23, SEQ ID NO:25, SEQ ID NO:27, SEQ ID NO:29, SEQ ID N0:31, SEQ ID NO:33, SEQ ID NO:35, SEQ ID NO:37, SEQ ID<br>
 <br>
NO:39, SEQ ID N0:4U SEQ ID NO:43, SEQ ID NO:45, SEQ ID NO:47, SEQ ID NO:49, SEQ ID N0:51, SEQ ID NO:53, SEQ ID NG:55, SEQ ID NO:57, SEQ ID NO:59, SEQ ID NO:63, SEQ ID NO:65, SEQ ID NO:67, SEQ ID NO:69, SEQ ID N0:71, SEQ ID NO:73, SEQ ID NO:75, SEQ ID NO;77, SEQ ID NO:79, SEQ ID N0:81, SEQ ID NO:83, SEQ ID NO:85, SEQ ID NO:87, SEQ ID NO:89, SEQ ID N0:91, SEQ ID NO:93, SEQ ID NO:95, SEQ ID NO:97, SEQ ID NO:99, SEQ ID NO:103, SEQ ID NO:105, SEQ ID NO:107, SEQ ID NO:109, SEQ ID N0:111, SEQ ID N0:113, SEQ ID N0:115, SEQ ID N0:117, SEQ ID N0:119, SEQ ID N0:12I, SEQ ID NO:123, SEQ ID NO:125, SEQ ID NO:127, SEQ ID NO:129, SEQ ID N0:I31, SEQ ID NO:133, SEQ ID NO:135, SEQ ID NO:137, SEQ ID NO:139, SEQ ID N0:141, SEQ ID NO:143, SEQ ID NO:145, SEQ ID NO:I47, SEQ ID NO:149, SEQ ID N0:151, SEQ ID NO:153, SEQ ID NO:155, SEQ ID NO:157, SEQ ID NO:159, SEQ ID N0:161, SEQ ID NO:163, SEQ ID NO:165, SEQ ID NO:167, SEQ ID NO:169, SEQ ID N0:171, SEQ ID NO:173, SEQ ID NO:175, SEQ ID NO:177, SEQ ID NO:179, SEQ ID N0:181, SEQ ID NO:I83, SEQ IDNO:I85, SEQIDNO:187, SEQIDNO:189, SEQ IDN0:191, SEQ ID NO:193, SEQ ID N0:I9S, SEQ ID NO:197, SEQ ID NO:I99, SEQ ID NO:203, SEQ ID NO:205, SEQ ID NO:207, SEQ ID NO:209, SEQ ID N0:211, SEQ ID NO:213, SEQ ID NO:215, SEQ ID NO:217, SEQ ID NO:219, SEQ ID NO:221, SEQ ID NO:223, SEQ ID NO:225, SEQ ID NO:227, SEQ ID NO:229, SEQ ID NO:231, SEQ ID NO:233, SEQ ID NO:235, SEQ ID NO:237, SEQ ID NO:239, SEQ ID NO:241, SEQ ID NO:243, SEQ ID NO;245, SEQ ID NO;247, SEQ ID NO:249, SEQ ID NO;25], SEQ ID NO;253, SEQ ID NO:255, SEQ ID NO:257, SEQ ID NO:259, SEQ ID NO:261, SEQ ID NO:263, SEQ ID NO:265, SEQ ID NO:267, SEQ ID NOi269, SEQ ID NO:271, SEQ ID NO:273, SEQ ID NO:275, SEQ ID NO;277, SEQ ID NO:281, SEQ ID NO:283, SEQ ID NO:285, SEQ ID NO:287, SEQ ID NO:289, SEQ ID NO:291, SEQ ID NO:293, SEQ ID NO:295, SEQ ID NO:297, SEQ ID NO:299, SEQ ID NO;30], SEQ ID NO:303, SEQ ID NO:305, SEQ ID NO:307, SEQ ID NO:309, SEQ ID N0:311, SEQ ID NO:313, SEQ ID NO:315, SEQ ID NO;317, SEQ ID NO:319, SEQ ID NO:321, SEQ ID NO:323, SEQ ID NO:325, SEQ ID NO:327, SEQ ID NO:329, SEQ ID NO:331, SEQ ID NO:333, SEQ ID NO:335, SEQ ID NO:336, SEQ ID NO:337, or SEQ ID NO:338, wherein said raonatin precursor, salts thereof, and combinations thereof is sweet.<br>
 <br>
[0097]	The invention further provides a method, comprising: producing a product chosen<br>
from monatin precursor, salts thereof, and combinations thereof, in a multi-step pathway, wherein a reaction in the pathway is facilitated by at least one polypeptide encoded by a nucleic acid sequence that comprises a sequence that hybridizes under stringent condition to a nucleic acid of SEQ ID N0:1, SEQ ID N0:3, SEQ ID N0:5, SEQ ID N0:7, SEQ ID N0:9, SEQ ID N0:1], SEQ ID N0:13, SEQ ID N0:15, SEQ ID N0:17, SEQ ID N0:19, SEQ ID N0:21, SEQ ID NO:23, SEQ ID NO:25, SEQ ID NO:27, SEQ ID NO:29, SEQ ID N0:31, SEQ ID NO:33, SEQ ID NO:35, SEQ ID NO:37, SEQ ID NO:39, SBQ ID N0:41, SEQ ID NO:43, SEQ ID "NO:45, SEQ ID NO:47, SEQ ID NO:49, SEQ ID N0:5h SEQ ID NO:53, SEQ ID NO:55, SEQ ID NO:57, SEQ ID NO:59, SEQ ID NO:63, SEQ ID NO:65, SEQ ID NO:67, SEQ ID NO:69, SEQ ID NO:71, SEQ ID NO:73, SEQ ID NO:75, SEQ ID NO:77, SEQ ID NO:79, SEQ ID N0:81, SEQ ID NO:83, SEQ ID NO:85, SEQ ID NO:87, SEQ ID NO:89, SEQ ID N0:91, SEQ ID NO:93, SEQ ID NO:95, SEQ ID NO:97, SEQ ID NO:99, SEQ ID NO:103, SEQ ID NO:105, SEQ IDNO:107, SEQ ID NO:109, SEQ IDNOiIll, SEQ ID N0:113, SEQ ID N0:115, SEQ ID NO:!!?, SEQ ID N0:119, SEQ ID N0:12U SEQ ID NO:123, SEQ ID N0:U5, SEQ ID NO:127, SEQ ID NO:129, SEQ ID N0:131, SEQ ID NO:133, SEQ ID NO:135, SEQ ID NO:137, SEQ ID NO:139, SEQ ID N0:i41, SEQ ID NO:143, SEQ ID NO:145, SEQ ID NO:147, SEQ ID NO:149, SEQ ID N0:151, SEQ ID NO:153, SEQ ID NO:155, SEQ ID NO:157, SEQ ID NO:159, SEQ ID N0:!61, SEQ ID NO:163, SEQ ID NO:165, SEQ ID NO:167, SEQ ID NO:169, SEQ ID N0:171, SEQ ID NO:173, SEQ ID N0:n5, SEQ ID NO:177, SEQ ID NO:179, SEQ ID N0:181, SEQ ID NO:183, SEQ ID NO:185, SEQ ID NO:187, SEQ ID NO:189, SEQ ID N0:19U SEQ ID NO:193, SEQ ID NO:195, SEQ ID NO:197, SEQ ID NO:199, SEQ ID NO:203, SEQ ID NO:205, SEQ ID NO:207, SEQ ID NO:209, SEQ ID N0:211, SEQ ID NO:213, SEQ ID NO:215, SEQ ID NO:217, SEQ ID NO:219, SEQ ID NO:221, SEQ ID NO:223, SEQ ID NO:225, SEQ ID NO:227, SEQ ID NO:229, SEQ ID NO:231, SEQ ID NO:233, SEQ ID NO:235, SEQ ID NO:237, SEQ ID NO:239, SEQ ID NO:24I, SEQ ID NO:243, SEQ ID NO:245, SEQ ID NO:247, SEQ ID NO:249, SEQ ID NO:251, SEQ ID NO:253, SEQ ID NO:255, SEQ ED NO:257, SEQ ID NO:259, SEQ ID NO:261, SEQ ID NO:263, SEQ ID NO:265, SEQ ID NO:267, SEQ ID NO:269, SEQ ID NO:271, SEQ ID NO:273, SEQ ID NO:275, SEQ ID NO:277, SEQ ID NO:281, SEQ ID NO:283, SEQ ID NO:285, SEQ ID NO:287, SEQ ID<br>
 <br>
NO:289, SEQ ID NO:291, SEQ ID NO:293, SEQ ID NO:295, SEQ ID NO:297, SEQ ID NO:299, SEQ ID NO:30], SEQ ID NO:303, SEQ ID NO:30S, SEQ ID NO:307 SEQ ID NO:309, SEQ ID N0:3] 1, SEQ ID NO:313, SEQ ID NO;315, SEQ ID NO:317, SEQ ID NO:319, SEQ ID NO:321, SEQ ID NO;323, SEQ ID NO:325, SEQ ID NO:327, SEQ ID NO:329, SEQ ID NO:331, SEQ ID NO:333, SEQ ID NO:335, SEQ ID NO:336, SEQ ID NO:337, or SEQ ID NO;338, wherein said monatin precursor, salts thereof, and combinations thereof is sweet.<br>
[0098]	In an effort to be concise, where ever intermediates/products are identified in the<br>
specification and claims (such as monatin, monatin precursor, or monatin der!vative(s)) as being formed, the term "and/or salts thereof should be understood to be included where applicable. In other words, for example, the phrase "indo!e-3-pyruvate is converted to MP" should be understood to read "indoIe-3-pyruvic acid is converted to MP and/or salts thereof." A person of ordinary skill, in fact, would appreciate that under reaction conditions shown the salts of the intermediates/products are in fact present.<br>
[0099]	According to some embodiments, the method produces a monatin or monatin<br>
derivative composition wherein the monatin or monatin derivative component of the composition includes only the R,R and S,R forms of monatin or monatin derivative. The term "only" when used to indicate that only certain isomers are formed, means that the pathway would produce only the identified isomers if racemization did not occur. Consequently, the term "only" should not be taken to mean absence of other isomers, but rather a person of ordinary skill would understand that other isomeric forms may be present in a relatively small amount due to racemization which may occur. According to some embodiments, the method produces a composition wherein the monatin or monatin derivative component of the composition includes only the R,R form of monatin or monatin derivative (except to the extent racemization occm resulting in other isomeric forms).<br>
[OlOOJ	As used herein, the phrase "monatin composition" means compositions including<br>
one or more isomers of monatin; the term can also mean only a single isomeric form of monatin and nothing else.<br>
[0101]	As used herein, the phrase "monatin derivative composition" means compositions<br>
including one or more isomers of a monatin derivative; the term can also mean only a single isomeric form of the monatin derivative and nothing else.<br>
 <br>
 <br>
amino group, or a halogen, atom, such iis an iodine atom, bromine atom, chlorine atom, or fluorine atom. However, R, Rt,, Re, Rd, and R cannot simultaneously all be hydrogen. Alternatively, Rb and Re, and/or R4 and Re may together form a C1-C4 alkylene group, respectively.<br>
[0103]	As used herein, "substituted indole-3-pyruvate" means one or more carbon atoms<br>
of the indole ring of the indole-3-pyruvate is independently substituted with one or more of the Ra, Rb, Re, R(j5 and R substituent groups defined above. However, Ra, Rb, Re, R-d, and Re cannot simultaneously all be hydrogen. Alternatively, Rb and Re, and/or Rd and Re may together form a C1-C4 alkylene group, respectively.<br>
[0104|	As used herein, "substituted tryptophan" means one or more carbon atoms of the<br>
indole ring of the tryptophan is independently substituted with one or more of the Ra, Rb, Re, Rdt and Re substituent groups defined above. However, Ra, Rb, Re, Ra, and Re cannot simultaneously all be hydrogen. Alternatively, Rt, and Re, and/or Rd and Re may together form a C1-C4 alkylene group, respectively. In one embodiment, the substituted tryptophan contains the same substituent group{s) on the indole ring as the final monatin derivative.<br>
[0105]	Furthermore, the biosynthetic pathways for producing monatin described herein<br>
can utilize a substituted tryptophan to yield monatin derivatives that are likeiy to be sweet. In some embodiments, the substituted tryptophan to be used in the biosynthetic pathways described herein include chlorinated tryptophan and 5-hydrDxytryplophan.<br>
 <br>
[0106]	For example, chlorinated D-tryptophans, which have structurai similarities to R,R<br>
monatin, have been identified as non-nutritive sweeteners (particularly 6-ch!oro-D-tryptophan). Similarly, halogenated and hydroxy-substituted forms of monatin have been found to be sweet U.S. Published Patent Application No. 2005/0118317. Halogens and hydroxyl groups could be substitutable for hydrogen, particularly on positions 1-4 of the benzene ring in the indole of tryptophan, without interfering in subsequent conversions to D- or L-tryptophan, indole-3-pyruvate, MP, or monatin. Substituted indoles have been shown in the literature to be suitable substrates for PLP-enzymes and have yielded substituted tryptophans. Fukuda, D.S., el at, "Production of Substituted L-Tryptophans by Fermentation," Appl Environ. Microbiol., 21:841-43 (1971). The halogen does not appear to sterically hinder the tryptophan synthase beta subunits catalytic mechanism and the enantiospecifici was also intact.<br>
[0107]	In some embodiments of the present invention, a process for producing a monatin<br>
composition is provided, which includes producing indoie-3-pyruvate from L-tiyptophan, producing 2-hydroxy 2-(indol-3yImetliyl)-4-keto glutaric acid ("monatin precursor" or "MP") from indole-3-pyruvate, and producing monatin from MP. The reaction of L-tryptophan to produce indole-3-pyruvate is facilitated by an enzyme having greater specificity, greater activity, or both for L-tryptophan than for R-MP, R,R monatin, or both. According to certain embodiments, the reaction of indoIe-3-pyruvate is facilitated by an enzyme having R-specific aldolase activity and consequently produces R-MP. According to certain embodiments, a racemase enzyme is provided which can facilitate the epimerization of the amino acid byproduct of the tryptophau reaction from one isomeric form to another isomeric form.<br>
[0108]	In some embodiments according to the invention, a process for producing a<br>
monatin composition is provided, which includes producing indole-3-pyruvate from L-tryptophan, producing 2-hydroxy 2-(indol-3ylmethyl)-4-keto glutaric acid ("monatin precursor" or "MP") from indole-3-pyruvate, and producing monatin from MP. The reaction of L-tryptophan to produce indole-3-pyruvate is facilitated by an enzyme having greater specificity, greater activity, or both for L-tiyptophan than for R-MP, R,R monatin, or both, and the reaction of MP to form monatin is facilitated by an enzyme, which is stereoselective for R-MP. The term "stereoselective" means that an enzyme has greater specificity, greater activity, or both for one isomer—in this case for R-MP versus S-MP—<br>
 <br>
over another. In preferred embodiments, a stereoselective enzyme has limited activity for one isomer as compared to another. "Limited" activity means activity that is minimally or not perceptible, for example as determined according to experiments provided herein.<br>
[0109]	It should be noted that, where references are made to a series of reactions such as<br>
in the preceding paragraphs, the invention does not require each step to be explicitly performed; it is sufficient that the step: may be implicitly performed. In other words, for example, the process for producing a monatin composition, which includes producing indole-3-pyruvate from L-tryptophan, producing 2-hydroxy 2-(indol-3ylmethyl)-4-keto glutaric acid ("monatin precursor" or "MP") from indole-3-pyruvate, and producing monatin from MP, wherein each reaction is facilitated by an appropriate enzyme, can be performed by combining L-tryptophan with the enzymes and setting conditions so that the enumerated reactions could occur. In such an instance L-tryptophan could react to produce indole-3-pyruvate, the indole-3-pyruvate produced from the L-tryptophan reaction could react to form MP, and the MP produced from the indole-3-pyruvate reaction could react to form monatin. The process could also be performed, by way of example, by providing a compound that can produce L-tryptophan, under conditions suitable for L-tryptophan production to occur and combining that compound with enzymes capable of facilitating the series of reactions set forth under conditions which would be suitable for those reactions to occur. As yet another example, the process could be performed by providing a microorganism genetically engineered to produce monatin according to the described pathway, and providing appropriate conditions for the fermentation process to occur. For example, a microorganism, which naturally produces large amounts of L-tryptophan could be genetically engineered to produce or over¬produce one or more of the enzymes used to facilitate reactions in the pathway to monatin, and appropriate conditions could be provided so that the microorganism would thereby produce monatin.<br>
[0110]	In other embodiments according to the invention, a process for producing monatin<br>
is provided, in which a substrate forms an L-amino acid when L-tryptophan is converted to indole-3-pyruvate, mdoie-3-pyruvate reacts to form MP (which can include both R-MP and S-MP though preferably includes only or predominately R-MP), and the L-amino acid reacts to regenerate (also referred to as "recycle") the substrate when R-MP is converted to R,R monatin.   The reaction of R-MP to form R,R monatin is facilitated by a<br>
 <br>
stereoinverting aminotransfere such as D-methionine aminotransferase (EC 2.6.1.41) or an enzyme having D-pheny!g]ycine aminotransferase activily.<br>
[0111]	In other embodiments according to the invention, a process for producing a<br>
monatin composition is provided, which includes producing D-tryptophan from L-tryptophan, producing indoIe-3-pyruvate from D-tryptophan, producing R-MP from indole-3-pyruvate, and producing R,R monatin from R-MP. The production of the D-tryptophan from the L-tryptophan is facilitated by a tryptophan racemase and functional equivalents thereof In certain further embodiments, the reactions of D-tryptophan to form indole-3-pyruvate and of MP to form monatin are facilitated by the same enzyme. In yet other further embodiments, the reaction of indole-3-pyruvate is facilitated by an enzyme having R-specific aldolase activity and consequently R-MP is formed, and the reactions of D-tryptophan to form indoie-3-pyruvate and of R-MP to form R,R monatin are facilitated by the same enzyme.<br>
[0112]	hi some embodiments according to the invention, a process for producing a<br>
monatin derivative is provided, which includes producing the monatin derivative firom a substituted indoie-3-pyruvate and pjTuvate, using an enzyme having R-specific aldolase activity" to catalyze the reaction.<br>
[0113]	ITie details of one or more embodiments of the invention are set forth in the<br>
accompanying drawings and the description below. Other features, objects, and advantages in accordance with the invention will be apparent firom the description and drawings, and from the claims. As should be realized from the description herein, the mvention is capable of modifications in various embodiments, all without departing from the spirit and scope of the present invention. Accordingly, the drawings and detailed description are to be regarded as illustrative in nature and not restrictive.<br>
[0114]	All publications, patents, patent applications, GenBank sequences and ATCC<br>
deposits, cited herein are hereby expressly incorporated by reference for all purposes.<br>
BRIEF DESCRIPTION OF DRAWINGS<br>
[0115]	The following drawings are illustrative of embodiments of the invention and are<br>
not meant to limit the scope of the invention as encompassed by the claims.<br>
[0116]	Figure 1 is a flow chart that shows an example of an enzymatic process for<br>
producing R,R monatin from L-tryptophan in accordance with the invention.   In this<br>
 <br>
example, the process includes using an L-aminotransferase (examples of which include an L-tryptophan aminotransferase, an L-aromatic aminotransferase, an L-aspartate aminotransferase, and an L-alanine aminotransferase) in the reaction of L-tryptophan that has greater specificity and/or selectivity for L-tryptophan as a substrate than for R-MP and/or the process includes using an L-amino acid oxidase with limited activity and/or specificity for R,R monatin as a substrate. In the specific example diagrammed in Figure I, an L-aminotransferase or L-amino acid oxidase converts L-tryptophan to indole-3-pyruvate. indole-3-pyruvate is reac;ted with an R-specific aldolase and pyruvate to produce R-alpha-keto acid monatin (R-MP), and R-MP is converted to R,R monatin by a D-aminotransferase or a D-amino acid dehydrogenase. As shown on Figure 1, the reactions are reversible, but for the purposes of the invention, it is not required that the reactions proceed in the reverse direction.<br>
[0117]	Figure 2 is a flow chart tha! shows an example of another process for producing<br>
R,R monatin in accordance with the invention. In this example, the process mcludes using an enzyme to convert R-MP to monatin which is stersoselective for R-MP. In the specific example diagrammed in Figure 2, tryptophan is shovra to be converted to indole-3-pyruvate in a reversible reaction. The indoIe-3-pyruvate can be reacted vth a non-stereospecific aldolase to reversibly form alpha-keto acid monatin (both R- and S-MP). The R-MP is reversibly converteid to R,R monatin by a stereoselective D-aminotransferase or a stereoselective D-amino acid dehydrogenase. Any S-MP that is formed by the non-stereospecific aldolase can be converted back into indole-3-pyruvate if a stereoselective D-aminotiansferase or D-amino acid dehydrogenase is utilized. For the purposes of the invention, it is not I'equired that the reactions shown as being reversible proceed in the reverse direction.<br>
[0118]	Figure 3 is a flow chart that shows an example of yet another process for<br>
producing R,R monatin from L-tryptophan in accordance with the invention In this example, the process includes converting L-tryptophan to D-tryptophan using a tryptophan racemase and using a D-amino acid product in the reaction coupled to the reaction forming indole-3-pyruvate as a substrate in the reaction coupled to the reaction forming R,R monatin. In the specific example diagrammed in Figure 3, L-tryptophan is converted to D-tryptophan by a tryptophan racemase in a reversible reaction. The D-tryptophan  is  reacted with  alpha-ketoglutarate  (a-KG)  and  a broad  specificit}' D-<br>
 <br>
aminotransferase to produce indoie-3-pyruvate and D-giutamate. IndoIe-3-pyruvate is reacted with pyruvate and an R-specific aldolase and converted to R-alpha-keto acid monatin (R-MP), and R-MP is reacted with a broad specificity D-aminotransferase and D-glutamate to form R,R monatin d alpha-ketoglutarate (a-KG). As shown on Figure 3, each of the reactions are reversible, but for the purposes of the invention, it is not required that the reactions proceed in tlie reverse direction.<br>
[0119]	Figure 4 is a flow chart thai shows an example of yet another process for<br>
producing R,R monatin from L-tryptophan in accordance with the invention. In this example, the process includes con-i'erting the L-amino acid formed in the reaction coupled with the L-tryptophan reaction to a D-amino acid; this D-amino acid acts as an amino donor for the reaction in which R-MP is converted to R,R monatin. In the specific example diagrammed in Figure 4, L-trj'ptophan is reacted with an L-aminotransferase and alpha-ketoglutarate to produce indole--3-pyruvate and L-glutamate. Indole-3-pyruvate is reacted with pyruvate and an R-specific aldolase and converted to R-aJpha-keto acid monatin (R-MP), and R-MP is reacted with a broad specificity D-aminotransferase and D-glutamate to form R,R monatin and alpha-ketoglutarate. As shown on Figure 4, the reactions are reversible, but for the purposes of the invention, it is not required that the reactions proceed in the reverse direction.<br>
[0120]	Figure 5 is a flow chart that shows an example of yet another process for<br>
producing R,R monatin from L-tryptophan in accordance with the invention. Ln this example, the process includes enzymatically facilitating the conversion of R-MP to R,R monatin using a stereoinverting enayme so that the L-amino acid formed by the reaction coupled to the L-tryptophan reaction can be used as a substrate for the reaction coupled to the R-MP to R,R monatin reaction, hi the specific example diagrammed in Figure 5, L-tryptophan is reacted with an L-aminotransferase and oxaloacetate, pyruvate or alpha-ketoglutarate (a-KG) to produce indDle-3-pyruvate, and L-aspartate (if oxaloacetate is used), L-alanine (if pyruvate is used) or L-glutamate (if a-KG is used). Indole-3-pyruvate is reacted with pyruvate and an R-specific aldolase and converted to R-alpha-keto acid monatin (R-MP), and R-MP is reacted wilb a stereoinverting aminotransferase and L-aspartate, L-alanine or L-glutamate to form R,R monatin and oxaloacetate (if L-partate is used), pyruvate (if L-alanine is used) or alpha-ketoglutarate (a-KG, if L-glutamate is<br>
 <br>
used). As shown on Figure 5, the re.ax;tions are reversible, but for the purposes of the invention, it is not required that the rea&gt;;tions proceed in the reverse direction.<br>
[0121]	Figure 6 is a flow chart tliat shows an example of yet another process for<br>
producing R,R monatin in accordance with the present invention. In this example, the process includes recycling the L-amino acid produced in the reaction forming indole-3-pyruvate with the D-amino acid used as a reactant with R-MP in the reaction forming R,R monatin through a series of conversion reactions. In the specific example diagrammed in FIG. 6, L-tryptophan is reversibly reacted with an L-aminotransferase and oxaloacetate to produce indoIe-3-pyruvate and L-aspartate. IndoIe-3-pyruvate is reacted in a reversible manner with pyruvate and an R-specific aldolase and converted to R-alpha-keto acid monatin (R-MP), and R-MP is reversibly reacted with a D-aminotransferase and D-aJanine to form R,R monatin and pyruvate. The L-aspartate is converted to L-alanine and CO2 using an aspartate 4-decarboxylase. The L-alanine is converted to D-alanine with an alanine racemase. For the purposes of the invention, it is not required that the reactions shown as being reversible proceed in the reverse direction.<br>
[0122]	Figure 7 is a flow chart tliat shows an example of yet another process for<br>
producing R,R monatin in accordance with the present invention. In tliis example, the process includes pushing the L-tryptophan reaction forward (i.e., driving the reaction toward the production of indole-3-pyruvate) by converting the L-amino acid byproduct of that reaction into another product. In this example, the L-amino acid L-aspartate byproduct is converted into L-alanine in an irreversible reaction using a decarboxylase. In the specific example diagrammed in Figure 7, L-tryptophan is reversibly reacted with an L-aminotransferase and with alpha-ketoglutarate (a-KG) or oxaloacetate to produce indole-3-pyruvate and L-glutamate {if a-KG is used) or L-aspartate (if oxaloacetate is used). Indole-3-pyruvate is reversibly reacted viith pyruvate and an R-specific aldolase and converted to R-alpha-keto acid monatin (R-MP). R-MP is reacted in a reversible manner with a D-aminotransferase and a D-amino acid to form R,R monatin and any of oxaloacetate, pyruvate or a-KG. The L-glutamate or L-aspartate that was a product of the L-aminotransferase reaction is converted to either 4-aininobutanoate and CO2 (if L-glutamate is the substrate) or to |3-alanine and CO2 (if L-aspartate is the substrate) using a glutamic acid or an aspartate decarboxylase. For the purposes of the invention, it is not required that the reactions shown as being reversible proceed in the reverse direction.<br>
 <br>
[0123]	Figure 8 is a flow chart thai shows an example of yet another process for<br>
producing R,R monatin in accordance with the present invention. In this example, the process includes recycling the amino acid byproduct of the L-tryptophan reaction with the amino acid reactant of the R-MP reaction through a series of conversion reactions. In the specific example diagrammed in FIG. 8, L-trptophan is reacted reversibly with an L-aminotransferase and with alpha-ketoglutarate (a-KG) to produce indole-3-pyruvate and L-glutamate. Indole-3-pyruvate is reversibly reacted with pyruvate and an R-specific aldolase and converted to R-alpha-keto acid monatin (R-MP). R-MP is reacted in a reversible manner with a D-aminotransferase and D-alanine to form R,R monatin and pyruvate. An L-alanine aniinoti-ansferase and pyruvate are used to reversibly convert the L-glutamate that was a product of the L-aminotransferase reaction back to a-KG, with L-alanine as a co-product. An alanine racemase reversibly converts the L-alanine to the D-alanine that is useful in the third reaction, (the D-aminotransferase reaction. For the purposes of the invention, it is not required that the reactions shown as being reversible proceed in the reverse direction.<br>
[0124]	Figure 9 is a block diagram of a computer system.<br>
[0125J	Figure 10 is a flow diagram illustrating one embodiment of a process for comparing<br>
a new nucleotide or protein sequence with a database of sequences in order to determine the homology levels between the new sequence and the sequences in the database.<br>
J0126]	Figure 11  is a flow diagram illustrating one embodiment of a process in a<br>
computer for determining whether two sequences are homologous.<br>
[0127]	Figure 12 is a flow diagram i])ustrating one embodiment of an identifier process<br>
300 for detecting the presence of a feature in a sequence.<br>
[0128]	Figures 13 and 14 together illustrate the activities of 58 different aldolases (each<br>
identified by its specific SEQ ID number) in the formation of monatin precursor (MP) as measured by LC/MS/MS.<br>
[0129]	Figure 15 illustrates the effect of dithiothreitol on the production of monatin by<br>
the polypeptide with aldolase activity of SEQ ID NO:88.<br>
[0130]	Like reference symbols in the various drawings indicate like elements.<br>
 <br>
DETAILED DESCRIPTION<br>
[0131]	A number of embodiments have been described above and are described in more<br>
detail infra. Embodiments of the invention include one or more of the described aspects.<br>
Abbreviations and Terms<br>
[0132]	The following explanations of terms and methods are provided to better describe<br>
the present disclosure and to guide those of ordinary skill in the art m the practice of the present disclosure. As used herein, 'including" means "comprising." In addition, the singular forms "a" or "an" or "the" include plural references unless ihe context clearly dictates othervdse. For example, reference to "comprising a protein" includes one or a plurality of such proteins, and reference to "comprising the cell" includes reference to one or more cells and equivalents thereof known to those skilled in the art, and so forth. The term "about" encompasses the range of experimental error that occurs in any measurement. Unless otherwise stated, all measurement numbers are presumed to have the word "about" in front of them even if the word "about" is not expressly used.<br>
(0133]	Conservative substitution:   a substitution of one amino acid for another amino<br>
acid in a polypeptide, which substitution has little to no impact on the activity of the polypeptide. The substitution is considered conservative independent of whether the exchanged amino acids appear structui'ally or functionally similar. For example, ideally, a tryptophan aminotransferase polypeptide including one or more conservative substitutions retains tryptophan aminotransferase activity, A polypeptide can be produced to contain one or more conservative substitutions by manipulating the nucleotide sequence that encodes that polypeptide using, for example, standard procedures such as site-directed mutagenesis or PCR or other methods known to those in the art.<br>
[0134]	Non-limiting examples of amino acids which may be substituted for an original<br>
amino acid in a protein and which may be regarded as conservative substitutions if there is little to no impact on the activity of the polypeptide include: Ala substituted with ser or thr; arg substituted with gin, his, or lys; asn substituted with glu, gin, lys, his, asp; asp substituted with asn, gJu, or gin; cys substituted with ser or ala; gin substituted with asn, glu, lys, his, asp, or arg; glu substituted with asn, gin lys, or asp; gly substituted with pro;<br>
 <br>
his substituted with asn, lys, gin, arg, tyr; ile substituted with leu, met, val, phe; leu substituted with ile, met, va!, phe; (ys substituted with asn, glu, gin, his, arg; met substituted with ile, leu, val, phe; plie substituted with trp, tyr, met, ile, or leu; ser substituted with thr, ala; thr substituted with ser or ala; trp substituted with phe, tyr: tyr substituted with his, phe, or trp; and val substituted with met, ile, feu.<br>
[0135]	Further information about conservative substitutions can be found in, among other<br>
locations, Ben-Bassat et al, (J. Bacteriol. 169:751-7, 1987), O'Regan et al, {Gene 77:237-51, 1989), Sahin-Toth et al, (Protein Sci. 3:240-7, J994), Hochuli el al., (Bio/Technology 6:1321-5, 1988), WO 00/67796 (Curd et al.) and in standard textbooks of genetics and molecular biofogy.<br>
[0136]	Derived:  For purposes of the specification and claims, a substance is "derii'ed"<br>
from an organism or source if any one or more of the following are true: 1) the substance is present m the organism/source; 2) the substance is removed from the native host; or, 3) the substance is removed from the native host and is evolved, for example, by mutagenesis.<br>
[0137]	Isolated:   The term "isolated" as used herein refere to any substance removed<br>
from its native host; the substance need not be purified. For example "isolated nucleic acid" refers to a naturally-occurring nucleic acid that is not immediately contiguous with both of the sequences with which it is immediately contiguous (one on the 5' end and one on the 3' end) in the naturally-occurring genome of the organism from which it is derived. For example, an isolated nucleic acid can be, without limitation, a recombinant DNA molecule of any length, provided one of the nucleic acid sequences normally found immediately flanking that recombinant DNA molecule in a naturally-occurring genome is removed or absent. Thus, an isolated nucleic acid includes, without limitation, a recombinant DNA that exists as a separate molecule (such as a cDNA or a genomic DNA fragment produced by PCR or restriction endonuclease treatment) independent of other sequences as well as recombinant DNA that is incorporated into a vector, an autonomously replicating plasmid, a -virus (such as a retrovirus, adenovirus, or herpes virus), or into the genomic DNA of a prokaryote or eukaryote. In addition, an isolated nucleic acid can Include a recombinant DNA molecule that is part of a hybrid or fusion nucleic acid sequence.<br>
 <br>
[0138]	As used herein, the tenn "isolated" means that the material (such as a protein or<br>
nucleic acid in accordance with the invention) is removed from its original environment (such as the natural environment if it is naturally occurring). For example,, a naturally-occurring polynucleotide or polypeptide present in a living animal is not isolated, but the same polynucleotide or polypeptide, separated from some or all of the coexisting materials in the natural system, is isolated. Such polynucleotides could be part of a vector and/or such polynucleotides or polypeptides could be part of a composition and still be isolated in that such vector or composition is not part of its natural environment.<br>
[0139J	The term "isolated" as used herein with reference to nucleic acid also includes any<br>
non-naturally-occurring nucleic acid because non-naturally-occurring nucleic acid sequences are not found in nature and do not have immediately contiguous sequences in a naturally-occurring genome. For example, non-naturally-occiirring nucleic acid such as an engineered nucleic acid is considered to be isolated nucleic acid. Engineered nucleic acid can be made using comrnon molecular cloning or chemical nucleic acid synthesis techniques. Isolated non-naturally-occurring nucleic acid can be independent of other sequences, or incorporated into a vector, an autonomously replicating plasmid, a virus (such as a retrovirus, adenovirus, or herpes virus), or the genomic DNA of a proitaryote or eukaryote. In addition, a non-naturaJly-occurring nucleic acid can include a nucleic acid molecule that is part of a hybrid or fusion nucleic acid sequence.<br>
[0140]	Purified: The term "purified" as ied herein does not require absolute purity, but<br>
rather is intended as a relative term. Thus, for example, a purified polypeptide or nucleic acid preparation can be one in which the subject polypeptide or nucleic acid is at a higher concentration than the polypeptide or nucleic acid would be in its natural environment within an organism or at a higher concentration than in the environment irom which it was removed.<br>
[0141]	Individual nucleic acids obtained from a library have been conventionally purified<br>
to electropfaoretic homogeneity. The sequences obtained from these clones could not be obtained directly either from the library or from total human DNA. The purified nucleic acids in accordance with the invention have been purified from the remainder of the genomic DNA in the organism by ac least 10'-10 fold. In some embodiments, the teim "purified" includes nucleic acids which have been purified from the remainder of the genomic DNA or from other sequences in a library or other environment by at least one<br>
 <br>
order of magnitude, such as, in some embodiments, two or three orders, or, four or five orders of magnitude.<br>
(0142J	Amino acid:  "Amino acid" or "amino acid sequence" as used herein refer to an<br>
ohgopeptide, peptide, polypeptide, or protein sequence, or to a fragment, portion, or subunit of any of these and to naturally occurring or synthetic molecules. "Amino acid" or "amino acid sequence" include an oligopeptide, peptide, polypeptide, or protein sequence, or to a fragment, portion, or subunit of any of these, and to naturally occurring or synthetic molecules. The term "polypeptide" as used herein, refers to amino acids joined to each other by peptide bonds or modified peptide bonds, i.e., peptide isosteres and may contain modified amino acids other than the 20 gene-encoded amino acids. The polypeptides may be modified by either natural processes, such as post-translational processing, or by chemical modification techniques which are well known in the art. Modifications can occur anywhere in the polypeptide, including the peptide backbone, the amino acid side-chains and the amino or carboxyl termini. It will be appreciated that the same type of modification may be present in the same or varying degrees at several sites in a given polypeptide. Also, a given polypeptide may have many types of modifications. Modifications include acetylation, acylation, .ADP-ribosylation, amidation, covaJent attachment of flavin, covalent attachment of a heme moiety, covalent attachment of a nucleotide or nucleotide derivative, covalent attachment of a lipid or lipid derivative, covalent attachment of a phosphatidylinosito!, cross-linking cyclization, disulfide bond formation, demethylation, formation of covalent cross-links, formation of cysteine, formation of pyroglutamate, fomiylation, gamma-carboxylation, glycosylation, GPI anchor formation, hydroxylation, iodinafion, methylation, myristolyation, oxidation, pegylaUon, glucan hydrolase processing, phosphorylation, prenylation, raceraization, selenoylation, sllfation and transfer-RNA mediated addition of amino acids to protein such as arginylation. (See Creighton, T.E., Proteins - Structure and Molecular Properties 2nd Ed., W.H. Freeman and Company, New York (1993); Posttranslational Covalent Modification of Proteins, B.C. Johnson, Ed., Academic Press, New York, pp. 1-12 (1983)). The peptides and polypeptides in accordance with the invention also include all "mimetic" and "peptidomimetic" forms, as described in further detail, below.<br>
[0143]	Polypeptide Having an Aldolase Activity: By a "polypeptide having an aldolase<br>
activity" is meant a polypeptide that either by itself, or in association with one or more<br>
 <br>
additional polypeptides (having the same or a different sequence), is a protein with the enzymatic activity of an aldolase.<br>
[0144J	Recombinant:   "Recombinant" polypeptides or proteins refer to polypeptides or<br>
proteins produced by recombinant DNA techniques; i.e., produced from cells transformed by an exogenous DNA construct encoding the desired polypeptide or protein. "Synthetic" polypeptides or protein are those prepared by chemical synthesis. Solid-phase chemical peptide synthesis methods can also be used to synthesize the polypeptide or fragments in accordance with the invention. Such method have been known in the art since the early 1960's (Merrifield, R. B.. J. Am. Chem. Soc, 85:2149-2154, 1963) (See also Stewart, J. M. and Yotuig, J. D., Solid Phase Peptide Synthesis, 2nd Ed., Pierce Chemical Co., Rockford, 111., pp. U-12)) and have recently been employed in commercially available laboratory peptide design and synthesis kits (Cambridge Research Biochemicals). Such commercially available laboratory kits have generally utilized the teachings of H. M. Geysen et al. Proc. Natl. Acad. ScL, USA, 81:3998 (1984) and provide for synthesizing peptides upon the tips of a multitude of "rods" or "pins" all of which are connected to a single plate.<br>
[0145]	Substantially identical:   The phrase "substantially identical" in the context of<br>
two nucleic acids or polypeptides, refers to two or more sequences that have, such as at least about 50%, 51%, 52%, 53%, 54%, 55%, 56%, 57%, 58%, 59%, 60%, 61%, 62%, 63%, 64%, 65%, 66%, 67%, 68%, 69%, 70%, 71%, 72%, 73%., 74%, 75%, 76%., 77%, 78%, 79%, 80%, 81%, 82%, 83%, 84%, 85%, 86%, 87%, 88%, 89%, 90%,, 91%, 92%, 93%, 94%, 95%, 96%, 97%, 98%., 99%, or more nucleotide or amino acid residue (sequence) identity, when compared and aligned for maximum correspondence, as measured using one of the known sequence comparison algorithms or by visual inspection. In other embodiments, the substantial identity exists over a region of at least about J 00 or more residues and most commonly the sequences are substantially identical over at least about 150 to 200 or more residues. In some embodiments, the sequences are substantially identical over the entire length of the coding regions.<br>
[0146]	Additionally a "substantially identical" amino acid sequence is a sequence that<br>
differs from a reference sequence by one or more conservative or non-conser\'ative amino acid substitutions, deletions, or insertions. In some embodiments, the substitution occurs at a site that is not the active site of the molecule, or, alternatively the substitution occurs<br>
 <br>
at a site that is the active site of the molecule, provided that the polypeptide essentially retains its functional (enzymatic) properties, A conservative amino acid substitution, for example, substitutes one amino acid for another of the same class {such as substitution of one hydrophobic amino acid, such as isoleucine, valine, leucine, or methionine, for another, or substitution of one polar amino acid for another, such as substitution of arginine for lysine, glutamic acid for aspartic acid or glutamine for asparagine). One or more amino acids can be deleted, for example, from an aldolase, such as pyruvate aldolase, HMG and/or KHG aldolase polypeptide, resulting in modification of the structure of the polypeptide, without significantly altering its biological activity. For example, amino- or carboxyl-terminal .amino acids that are not required for aldolase, such as pyruvate aldolase, such ar. HMG andor KHG aldolase enzyme, biological activity can be removed. Modified polypeptide sequences in accordance with the invention can be assayed for aldolase, such as pyruvate aldolase, such as HMG and/or KHG aldolase enzyme, biological activity by any number of methods, including contacting the modified polypeptide sequence with a substrate and determining whether the modified polypeptide decreases the amount of specific substrate in the assay or increases the bioproducts of the enzymatic reaction of a functional aldolase, such as pyruvate aldolase, such as HMG and/or KHG aldolase polypeptide with the substrate.<br>
[0147]	Fragment:   A "fragment" as used herein with regard to a protein or polypeptide<br>
or nucleic acid is a portion of the protein, polypeptide or nucleic acid, respectively. Fragments can have the same or substantially the same amino acid or nucleic acid sequence as the longer protein, poiypeptide or nucleic acid sequence from which the fragment is derived. Fragments which have different three dimensional structures as compared to that of the longer protein, polypeptide or nucleic acid are also included. An example of this, is a "pro-form" molecule, such as a low activity proprotein that can be modified by cleavage to produce a mature enzyme with significantly higher activity. A fragment of a protein or polypeptide can be an en2ymatically active portion of a protein or polypeptide.<br>
[0148]	Stereoinverting aminotransferase:    A 'stereomverting aminotransferase" is a<br>
polypeptide capable of preferentially or selectively producing a chiral amino acid product (such as monatin) while using an opposite chirality substrate as the amino donor. For example, a stereoinverting aminotransferase may be a D-phenylglycine aminotransferase<br>
 <br>
(also called D-4-hydroxyphenylglyciiie aminotransferase) that preferentially or se)ectivel}' uses L-glutamate as a substrate to produce R,R monatin. Non-limitJng examples of stereoinverting aminotransferases include D-methionine aminotransferase (EC 2.6,1.41) and enzymes having E'-phenylglycine aminotransferase activit&gt;' or D-4-hydroxyphenyjglycine aminotransferase activity.<br>
[0149]	The invention provides polypeptides with aldolase, including pyruvate activity<br>
such as, without limitation, HMG and/or KHG aldolase activity, polynucleotides encoding them, and methods ol" making and using these polynucleotides and polypeptides. In some embodiments, the invention also provides aldolase enzymes, such as pyruvate aldolase, such as HMG and/or KHG aldolase en2ymes, polynucleotides encoding these enzymes, the use of such polynucleotides and polypeptides.<br>
[0150]	In some embodiments, the mvention provides modified or evolved aldolases, such<br>
as pyruvate aldolases, HMG and/or KHG aldolases, with an increased specific activity as compared to the unmodified or unevolved aldolases, respectively.<br>
[0151]	In some embodiments, aldolases, such as a pyruvate aldolase, such as, without<br>
limitation a HMG and/or a KHG aldolase, are provided that facilitate the production of a 3, 4-substituted 2-keto-giutarate. In one embodiment, the invention provides a method of making a 3, 4-substituted 2-keto-glutarate comprising: (a) providing a polypeptide having an aldolase activity, such as a pyruvate aldolase activity, such as, without limitation, a HMG aldolase and/or a KMG aldolase activity; (b) providing a donor and an acceptor compound; and (c) contacting the polypeptide of step (a) with the compounds of step (b) under conditions wherein the aldolase catalyzes the synthesis of a 3, 4-suhstituted 2-keto-glutarate, wherein optionally the donor and the acceptor are a pyruvate or a pyruvate donor and an a-keto acid acceptor, a ketone and/or an aldehyde.<br>
[0152]	In another embodiment of the invention, a p)Tuvate aldolase, such as a HMG<br>
and/or a KHG aldolase, can be used in conjunction with a D-aminotransferase to make a 4-substituted D-glutamic acid or a derivative thereof. A 4-substituted D-glutamic acid and/or a derivative thereof can be used as an antibiotic, as these compounds have been found to inhibit bacterial glutamate racemase. In one embodiment, the invention provides a method of making a 4-substituted D-glutamic acid comprising: (a) providing a polypeptide having an aldolase activity, such as a pyruvate aldole activity, such as, without limitation, a HMG aldolase and/or a KMG aldolase activity; (b) providing an a-<br>
 <br>
keto acid acceptor and a pyruvate or a pyruvate donor: and (c) contacting the polypeptide of step (a) with the compounds of step (b) under conditions wherein the aldolase catalyzes the synthesis of a 4-substituted D-glutamic acid, wherein optionally the polypeptide has pyruvate aldolase, HMG aldolase and'or KHG aldolase activity and wherein optionally the method further comprises use of a D-aminotransferase.<br>
10153]	In some embodiments the invention provides compositions (such as enzyme<br>
preparations, foods and food additives, feeds and feed additives, beverage and beverage additives, drugs and drug additives, and dietary supplements) comprising the enzymes, polypeptides or polynucleotides in act;ordance with the invention. These compositions can be formulated in a variety of forms, such as as liquids, gels, pills, tablets, sprays, films, micelles, powders, food, feed pellets or encapsulated forms, including nanoencapsulated forms.<br>
[0154]	Assays for measuring aldolase activity, including pyruvate activity such as,<br>
without limitation, HMG and/or KHG aldolase activity, such as for determining if a polypeptide has aldolase activity, including pyruvate activity such as, vrithout limitation, HMG and/or KHG aldolase activity, are well known in the art and are within the scope in accordance with the invention; see E.E. Dekker &amp; R.P. Kitson, J. Biol. Chem. 267, 10507-10514, 1992; Taha TS, Deits TL, Purification and characterization of 2-keto-3-deoxy-6-phosphogluconate aldolase from Azotobacter vinelandii: evidence that the enzyme is bifunctional towards 2-keto-4-hydroxy glutarate cleavage, Biochem Biophys Res Commun. 1994 Apr 15;200(l):459-66; Dekker EE, Kobes RD, Grady SR, 2-keto-4-hydroxyglutarate aldolase fi-om bovine liver, Methods Enzymol. 1975;42:280-5; Dekker EE, Nishihara H, Grady SR, Methods Enzymol. 1975;42:285-90, 2-keto-4-hydroxyglutarate aldolase from Escherichia coli; Nishihara H, Dekker EE, Biochim Biophys Acta. 1969 Jul 8;185(l):255-7, A stereospecific 2-keto-4-hydroxyglutarate aldolase from Escherichia coli. One example of a suitable assay for determining if a polypeptide has aldolase activity, such as pyruvate aldolase, such as HMG and/or KHG aldolase activity is described in Example 3.<br>
[0155]	In some embodiments, the aldolases of the invention can be used effectively at a<br>
variety of pH conditions, including for example, from a range of about 3.0 to about 12.0. In other embodiments, the aldolases oi'the invention can be used at about pH is 3.0, 4.0, 4.5, 5.0, 5.5, 6.0, 6.5, 7.0, 7.5, 8.0, 8.5, 9.0, 9.5, lO.O, 10.5, 11.0, 11.5, or about 12.0.<br>
 <br>
Reaction conditions conducted under acidic or alkaline conditions also can be advantageous, such as in some industrial or pliarmaceutica] applications of enzymes in accordance with the invention.<br>
[0156]	The invention provides aldolase, such as pyruvate aldolase, such as HMG and/or<br>
KHG aldolase polypeptides in accordance with the invention in a variety of forms and formulations. In the methods in accordance with the invention, aldolase, such as pyruvate aldolase, such as HMG and/or KHG aldolase polypeptides in accordance with the invention are used in a variety of forms and formulations. For example, purified aldolase, such as pyruvate aldolase, such as HVIG and/or KHG aldolase polypeptides can be used in enzyme preparations deployed in the production of R-2-hydroxy 2-(indol-3yIme£hyl)-4-keto giutaric acid (R-MP) and certain stereoisomers of monatin, such as R,R and S,R monatin, and salts thereof, as well as certain stereoisomers of monatin derivatives, such as the R,R and S,R monatin derivative configurations, and salts thereof, or in pharmaceutical or dietary aid applications. Alternatively, the enzymes in accordance with the invention can be used directly in processes to produce R-2-hydroxy 2-(indol-3ylmethy!)-4-keto giutaric acid (R-MP) and certain stereoisomers of monatin, such as R,R and S,R monatin, and salts thereof, as well as certain stereoisomers of monatin derivatives, such as the R,R and S,R monatin derivative configurations, and salts thereof, to process foods, liquids or feeds, and the like.<br>
[0157]	In some embodiments, aldolase, such as pyruvate aldolase, such as HMG and/or<br>
KHG aldolase polypeptides in accordance with the invention can be expressed in a microorganism using procedures knov/n in the art. In some embodiments, the aldolase, such as pyruvate aldolase, such as HMG and/or KHG aldolase polypeptides in accordance with the invention can be immobilized on a solid support prior to use in the methods in accordance with the invention. Methods for immobilizing enzymes on solid supports are commonly known in the art, for example J. Mol. Cat. B: Enzymatic 6 (1999) 29-39; Chivata et al. Biocatalysis: Immobilized cells and enzymes, J Mol. Cat. 37 (1986) 1-24: Sharma et al.. Immobilized Biomaterials Techniques and Applications, Angew. Chem. Int. Ed. Engl. 2J (J 982) 837-54: Laskin (Ed.), Enzymes and Immobilized Cells in Biotechnology.<br>
 <br>
Nucleic Acids. Probes and Inhibitor' Molecules<br>
[01581	The invention provides isolated and recombinant nucleic acids, such as see<br>
Sequence Listing; nucleic acids encoding polypeptides, including the polynucleotide sequences in accordance with the invention, such as see Sequence Listing; including expression cassettes such as expression vectors and various cloning vehicles comprising nucleic acids in accordance with the invention. In some embodiments, the invention'also includes methods for discovering, identifying or isolated new aldolase, such as pyruvate aldole, HMG and/or KHG aldolase polypeptide sequences using the nucleic acids in accordance with the invention. In some embodiments, the invention also includes methods for inhibiting the expression of aldolase, such as pyruvate aldolase, such as HMG and/or KHG aldolase encoding genes and transcripts using the nucleic acids in accordance with the invention.<br>
[0159]	Also provided are methods for modifying the nucleic acids in accordance with the<br>
invention, including making variants of nucleic acids in accordance with the invention, by, such as synthetic ligation reassembly, optimized directed evolution system and/or saturation mutagenesis such as gene site saturation mutagenesis (GSSM). The term "saturation mutagenesis". Gene Site Saturation Mutagenesis, or "GSSM" includes a method that uses degenerate oligonucleotide primers to introduce point mutations into a polynucleotide, as described in detail, below. The term "optimized directed evolution S}'Stem" or "optimized directed evolution" includes a method for reassembling fragments of related nucleic acid sequences, such as related genes, and explained in detail, below. The term "synthetic ligation reassembly" or "SLR" includes a method of ligating oligonucleotide fragments in a non-stochastic fashion, and explained in detail, below. The term "variant" refers to polynucleotides or polypeptides in accordance with the invention modified at one or more base pairs, codons, introns, exons, or amino acid residues (respectively) yet still retain the biological activity of an aldolase, such as pyruvate aldolase, HMG and/or KHG aldolase in accordance with the invention. Variants can be produced by any number of means included methods such as, for example, error-prone PCR, shuffling, oHgonucleotide-directed mutagenesis, assembly PCR, sexual PCR mutagenesis, in vivo mutagenesis, cassette mutagenesis, recursive ensemble mutagenesis, exponential ensemble mutagenesis, site-specific mutagenesis, gene reassembly, GSSM and any combination thereof.<br>
 <br>
[0160]	The nucleic acids in accordance with tiie invention can be made, isolated and/or<br>
manipulated by, such as cloning and expression of cDNA libraries, amplification of message or genomic DNA by PCR, arid the like. For example, sequences in accordance with the invention were initially derived from environmenta] sources. Thus, in some embodiments, the invention provides aldolase-, such as pyruvate aldolase-, such as HMG and/or KHG aldolase enz}'me-encodir?g nucleic acids, and the polypeptides encoded by them, preferably derived from a common source, such as an environmental, mixed culture, or a bacterial source.<br>
[0161]	In practicing the methods in accordance with the invention, homologous genes can<br>
be modified by manipulating a template nucleic acid, as described herein. In some embodiments, the invention can be prticticed in conjunction with any method or protocol or device known in the art, which are well described in the scientific and patent literature.<br>
[0162]	The phrases "nucleic acid" or "nucleic acid sequence" as used herein refer to an<br>
oligonucleotide, nucleotide, polynucleotide, or to a fragment of any of these, to DNA or RNA of genomic or synthetic origin which may be stngle-stranded or double-stranded and may represent a sense or antisense (complementary) strand, to peptide nucleic acid (PNA), or to any DNA-like or RNA-like material, natural or synthetic in origin. The phrases "nucleic acid" or "nucleic acid sequence" includes oligonucleotide, nucleotide, polynucleotide, or to a fragment of any of these, to DNA or RNA (such as mRNA, rRNA, tRNA, iRNA) of genomic or sjTithetic origin which may be single-stranded or double-sfranded and may represent a sense or antisense sfrand, to peptide nucleic acid (PNA), or to any DNA-like or RNA-like material, natural or synthetic in origin, including, such as iRNA, ribonucleoproteins (such as double stranded IRNAs, such as iRNPs). The term encompasses nucleic acids, i.e., oligonucleotides, containing known analogues of natural nucleotides. The term also encompasses nucleic-acid-iike structures with synthetic backbones, see such as Mata (1997) Toxicol. Appl. Pharmacol. 144:189-197; Strauss-Soukup (1997) Biochemistry 36:8692-8698; Samstag (1996) Antisense Nucleic Acid Drug Dev 6:153-156. "Oligonucleotide" includes either a single sfranded polydeoxynucleotide or two complementary polydeoxynucleotide sfrands which may be chemically sjTithesized. Such synthetic oligonucleotides have no 5' phosphate and thus will not ligate to another oligonucleotide without adding a phosphate with an ATP in the<br>
 <br>
presence of a kinase. A synthetic ciligonucleotide can iigate to a fragment that has not been dephosphorylated.<br>
{0163]	A "coding  sequence  of or a "nucleotide  sequence encoding"  a particular<br>
polypeptide or protein, is a nucleic acid sequence which can be transcribed and translated into a polypeptide or protein when placed under the control of appropriate regulatory sequences. The term "gene" means the segment of DNA involved in producing a polypeptide chain; it includes regions preceding and foUowmg the coding region (leader and trailer) as well as, where applicable, intervening sequences (introns) between individual coding segments (exons). A promoter sequence is "operably linked to" a coding sequence when RNA polymerase which initiates transcription at the promoter will transcribe the coding sequence into mRNA. "Operably linked" as used herein refers to a functional relationship between two or more nucleic acid (such as DNA) segments. It can refer to the functional relationship of transcriptional regulatory sequence to a transcribed sequence. For example, a promoter is operably linked to a coding sequence, such as a nucleic acid in accordance with the invention, if it stimulates or modulates the transcription of the coding sequence in an appropriate host cell or other expression system. Generally, promoter transcriptional regulatory sequences that are operably linked to a transcribed sequence are physically contiguous to the transcribed sequence, i.e., they are cis-acting. However, some transcriptional regulatory sequences, such as enhancers, need not be physically contiguous or located in close proximity to the coding sequences whose transcription they enhance.<br>
(01641	The term "expression cassette" as used herein refers to a nucleotide sequence<br>
which is capable of affecting expression of a structural gene (i.e., a protein coding sequence, such as an aldolase, such as pyruvate aldolase, HMG and/or KHG aldolase enzyme in accordance with the invention) in a host compatible with such sequences. Expression cassettes include at least a promoter operably linked with the polypeptide coding sequence; and, optionally, with other sequences, such as transcription termination signals. Additional factors necessary or helpful in effecting expression may also be used, such as enhancers, alpha-factors. Thus, expression cassettes also include plasmids, expression vectors, recombinant viruses, any form of recombinant "naked DNA" vector, and the like. A "vector" comprises a nucleic acid which can infect, transfect, transiently or permanently transduce a cell. It will be recognized that a vector can be a naked nucleic<br>
 <br>
acid, or a nucleic acid complexed with protein or lipid. The vector optionally comprises viral or bacterial nucleic acids and/or proteins, and/or membranes (such as a ceil membrane, a vira! lipid envelope, etc.). Vectors include, but are not limited to replicons (such as RNA repiicons, bacteriophag,es) to which fragments of DNA may be attached and become replicated. Vectors thus include, but are not limited to RNA, autonomous self-replicating circular or linear D&gt;[A or RNA (such as plmids, viruses, and the like, see U.S. Patent No. 5,217,879), and include both the expression and non-expression plasmids. Where a recombinant microorganism or cell culture is described as hosting an "expression vector" this includes both extra-chromosomal circular and linear DNA and DNA that has been incorporated into the host chromosome(s). Where a vector is being maintained by a host cell, the vector may either be stably replicated by the cells during mitosis as an autonomous structure, or is incorporated within the host's genome.<br>
[01651	As used herein, the term "recombinant" encompasses nucleic acids adjacent to a<br>
"backbone" nucleic acid to which it is not adjacent in its natural environment. In some embodiments, to be "enriched" the nucleic acids will represent about 5% or more of the number of nucleic acid inserts in a population of nucleic acid backbone molecules. Backbone molecules according to the invention include nucleic acids such as expression vectors, self-replicating nucleic acids, viruses, integrating nucleic acids and other vectors or nucleic acids used to maintain or manipulate a nucleic acid insert of interest. In some embodiments, the enriched nucleic acids represent about 15% or more of the number of nucleic acid inserts in the population of recombinant backbone molecules. In some embodiments, the enriched nucleic acids represent about 50% or more of the number of nucleic acid inserts in the population of recombinant backbone molecules. In some embodiments, the enriched nucleic acids represent about 90% or more of the number of nucleic acid inserts in the population of recombmant backbone molecules.<br>
[0166]	One embodiment of the invention is an isolated, synthetic or recombinant nucleic<br>
acid comprising one of the sequences in accordance with the invention, or a fragment comprising at least 10, 15, 20, 25, 30, 35, 40, 50, 75, 100, 150, 200, 300, 400, or 500 or more consecutive bases of a nucleic acid in accordance with the invention. The isolated, synthetic or recombinant nucleic acids may comprise DNA, including cDNA, genomic DNA and synthetic DNA.   The DNA may be double-stranded or single-sttanded and if<br>
 <br>
single stranded may be the coding strand or non-coding (anti-sense) strand. Alternaliveiy, the isolated, synthetic or recombinant Eucleic acids comprise RNA.<br>
[0167]	The isolated, synthetic or recombinant nucleic acids in accordance with the<br>
invention may be used to prepare one of the polypeptides in accordance with the invention, or fragments comprising at least 5, 10, 15, 20, 25, 30, 35, 40, 50, 75, 100, or 150 or more consecutive amino acids of one of the polypeptides in accordance with the invention. Accordingly, another embodiment of the invention is an isolated, synthetic or recombinant nucieic acid which encodes one of the polypeptides in accordance with the invention, or fragments comprising at least 5, 10, 15, 20, 25, 30, 35, 40, 50, 75, 100, or 150 or more consecutive amino acids of one of the polypeptides in accordance with the invention. The coding sequences of these nucieic acids may be identical to one of the coding sequences of one of the nucleic acids in accordance with the invention or may be different coding sequences which encode one of the in accordance with the invention having at least 5, 10, 15, 20, 25, 30, 35, 40, 50, 75, 100, or 150 or more consecutive amino acids of one of the polypeptides in accordance with the invention, as a result of the redundancy or degeneracy of the genetic code. The genetic code is well known to those of skill in the art and can be obtained, such as on page 214 of B. Lewin, Genes VI, Oxford University Press, 1997.<br>
[0168]	The isolated nucleic acid which encodes one of the polypeptides of the invention<br>
and sequences substantially identical thereto, may include, but is not limited to: the coding sequence of a nucleic acid in accordance with the invention and additional coding sequences, such as leader sequences or proprotein sequences and non-coding sequences, such as introns or non-coding sequences 5' and/or 3' of the coding sequence. Thus, as used herein, the term "polynucleotide encoding a polypeptide" encompasses a polynucleotide which includes the coding sequence for the polypeptide as well as a polynucleotide which includes additional coding and/or non-coding sequence.<br>
[0169]	Alternatively,  the  nucleic   acid   sequences   of the  invention  and   sequences<br>
substantially identical thereto, may be mutagenized using conventional techniques, such as site directed mutagenesis, or other techniques familiar to those skilled in the art, to introduce silent changes into the polynucleotides o in accordance with the invention. As used herein, "silent changes" include, for example, changes which do not alter the amino acid sequence encoded by the polynucleotide.  Such changes ma)' be desirable in order to<br>
 <br>
increase the level of the polypeptide produced by host cells containing a vector encoding<br>
the polypeptide by introducing codons or codon paks which occur frequently in the host<br>
organism.<br>
[0170]	The invention also relates to polynucleotides which have nucleotide changes<br>
which result in amino acid substitutions, additions, deletions, fusions and truncations in the polypeptides in accordance with the invention. Such nucleotide changes may be introduced using techniques such as site directed mutagenesis, random chemical mutagenesis, exonuclease III deletion and other recombinant DNA techniques. Alternatively, such nucleotide changes may be naturally occurring allelic variants which are isolated by identifying nucleic acids which specifically hybridize to probes comprising at least 10, 15, 20, 25, 30, 35, 40, 50, 75, 100, 150, 200, 300, 400, or 500 consecutive bases of one of the sequences in accordance with the invention (or the sequences complementary thereto) under conditions of high, moderate, or low stringency as provided herein.<br>
General Techniques<br>
[0171]	The nucleic acids used to practice this invention, whether RNA, siRNA, miRNA,<br>
antisense nucleic acid, cDNA. genomic DNA, vectors, viruses or hybrids thereof, may be isolated from a variety of sources, genetically engineered, amplified, and/or expressed/ generated recombinantly. Recombinant polypeptides {such as aldolase, such as pyruvate aldolase, such as HMG and/or KHG aldolase enzymes) generated from these nucleic acids can be individually isolated or cloned and tested for a desired activity. Any recombinant expression system can be used, including bacterial, mammalian, yeast, insect or plant cell expression systems.<br>
[0172]	Alternatively, these nucleic acids can be synthesized in vitro by well-known<br>
chemical synthesis techniques, as described in, such as Adams (1983) J. Am. Chem. Soc. 105:661; Belousov (1997) Nucleic Acids Res. 25:3440-3444; Frenkel (1995) Free Radio. Biol. Med. 19:373-380; Blommers (1994) Biochemistry 33:7886-7896; Narang (1979) Meth. Enzymol. 68:90; Brown (1979) Meth. Enzymol. 68:109; Beaucage (1981) Tetra. Lett 22:1859; U.S. Patent No. 4,458,066.<br>
[0173]	Techniques for the manipulation of nucleic acids, such as subcloning, labeling<br>
probes (such as random-primer labeling using Klenow polymerase, nick translation, amplification), sequencing, hybridization and the like are well described in the scientific<br>
 <br>
and patent literature, see Sambrook, ed., MOLECULAR CLONING: A LABORATORY iMANUAL (2ND ED.), Vols. 1-3, Cold Spring Harbor Laborator&gt;', (1989); CURRENT PROTOCOLS IN MOLECULAR BIOLOGY, Ausubel, ed. John Wiley &amp; Sons, Inc., New York (1997); LABORATORY TECHNIQUES IN BIOCHEMISTRY AND MOLECULAR BIOLOGY: HYBRIDIZATION WITH NUCLEIC ACID PROBES, Part I. TheoryandNucleic Acid Preparation, Tijssen,ed. Elsevier, N.Y. (1993).<br>
f0174j	.Another useful means of obtaining and manipulating nucleic acids used to practice<br>
the methods in accordance with the invention is to clone from genomic samples, and, if desired, screen and re-clone inserts isolated or amplified from, such as genomic clones or cDNA clones. Sources of nucleic acid used in the methods in accordance with the invention include genomic or cDNA libraries contained in, such as mammalian artificial chromosomes (MACs), see U.S. Patent Nos. 5,721,118; 6,025,155; human artificial chromosomes, see Rosenfeld (1997) Nat. Genet. 15:333-335; yeast artificial chromosomes (YAC); bacterial artificial ciiromosomes (BAG); PI artificial chromosomes, see Woon (1998) Genomics 50:306-316; Pl-derived vectors (PACs), see Kern (1997) Biotechniques 23:120-124; cosmids, recombinant viruses., phages or piasmids.<br>
|0175]	In some embodiments, a nucleic acid encoding a polypeptide in accordance with<br>
the invention is assembled in appropriate phase with a leader sequence capable of directing secretion of the translated polypeptide or fragment thereof.<br>
[0176]	The invention provides fiision proteins and nucleic acids encoding them.    A<br>
polypeptide in cordance with the invention can be flised to a heterologous peptide or polypeptide, such as N-terminal identification peptides which impart desired characteristics, such as increased stability or simplified purification. Peptides and polypeptides in accordance with the invention can also be synthesized and expressed as fusion proteins with one or more additional domains linked thereto for, such as producing a more immunogenic peptide, to more readily isolate a recombinantly synthesized peptide, to identify and isolate antibodies and antibody-expressing B cells, and the like. Detection and purification facilitating domams mclude, such as metal chelating peptides such as polyhistidine tracts and histidine-tiyptophan modules that allow purification on immobilized metals, protein A domains that allow purification on immobilized immunoglobulin, and the domain utilized in the FLAGS extension/affinity purification<br>
 <br>
system (Immunex Corp, Seattle W.A)- The inclusion of a cleavable linker sequences such as Factor Xa or enterokinase (Invitrogen, Carlsbad, CA) between a purification domain and the motif-comprising peptide or polypeptide to facilitate purification. For example, an expression vector can include an epitope-encoding nucleic acid sequence linked to six histidine residues followed by a thioredoxin and an enterokinase cleavage site (see such as Williams (1995) Biochemistry 34:1787-1797; DobeH (1998) Protein Expr. Purif 12:404-414). The histidine residues facilitate detection and purification while the enterokinase cleavage site provides means for purifying the epitope from the remainder of the fusion protein. Technology pertaining to vectors encoding fusion proteins and appiication effusion proteins are well described in the scientific and patent literature, see such as Kroll (1993) DNA Cell. Biol., 12:441-53.<br>
Transcriptional and translational conti'ol sequences<br>
[0177]	The invention provides nucleic acid (such as DNA) sequences in accordance with<br>
the invention operatively linked to expression (such as transcriptional or translational) control sequence(s), such as promoters or enhancers, to direct or modulate RNA synthesis/ expression. The expression control sequence can he in an expression vector. Exemplary bacterial promoters include lad, lacZ, T3, T7, gpt, lambda PR, VL and trp. Exemplary eukaiyotic promoters include CMV immediate early, HSV thymidine kinase, early and late SV40, LTRs from retrovirus, and mouse metallothionein I.<br>
[0178]	As used herein, the term "promoter" includes all sequences capable of driving<br>
transcription of a coding sequence in a cell, such as a plant or animal cell. Thus, promoters used in the constructs in accordance with the invention include ci5-acting transcriptional control elements and regulatory sequences that are involved in regulating or modulating the timing and/or rate of transcription of a gene. For example, a promoter can be a cis-actin% transcriptional control element, including an enhancer, a promoter, a transcription terminator, an origin of replication, a chromosomal integration sequence, 5' and 3' untranslated regions, or an inironic sequence, which are involved in transcriptional regulation. These cis-acting sequences can interact with proteins or other biomolecules to carry out (tum on/off, regulate, modulate, etc.) transcription. "Constitutive" promoters are those that drive expression continuously lander most environmental conditions and states of development or cell differentiation. "Inducible" or "regulatable' promoters direct expression of the nucleic acid in accordance with the invention under the influence<br>
 <br>
of environmental conditions or devslopmental conditions. Examples of environmenta! conditions that may affect transcription by inducible promoters include anaerobic conditions, elevated temperature, drought, or the presence of light.<br>
(0I79I	"Tissue-specific" promoters are transcriptional control elements that are only<br>
active in particular cells or tissues or organs, such as in plants or animals. Tissue-specific regulation may he achieved by certain intrinsic factors which ensure that genes encoding proteins specific to a given tissue arc expressed. Such factors are known to exist in mammals and plants so as to allow for specific tissues to develop.<br>
[0180]	Promoters suitable for expressing a polypeptide in bacteria include the E. coli lac<br>
or trp promoters, the lad promoter, the lacZ promoter, the T3 promoter, the T7 promoter, the gpt promoter, the lambda PR promoter, the lambda PL promoter, promoters from operons encoding glycolytic enzymes such as 3-phosphog]ycerate kinase (PGK), and the acid phosphatase promoter. Eukaryotic promoters include the CMV immediate early promoter, the HSV thymidine kinase promoter, heat shock promoters, the early and late SV40 promoter, LTRs from retroviruses, and the mouse metallothionein-1 promoter. Other promoters known to control expression of genes in prokaryotic or eukaryotic cells or their viruses may also be used. Promoters suitable for expressing the polypeptide or fragment thereof in bacteria include the E. coli lac or trp promoters, the /ac/promoter, the lacZ promoter, the T3 promoter, the T7 promoter, the gpt promoter, the lambda PR promoter, the lambda Pi promoter, promoters from operons encoding glycolytic enzymes such as 3-phosphoglycerate kinase (PGK) and the acid phosphatase promoter. Fungal promoters include the a-factor promoter. Eukaryotic promoters include the CMV immediate early promoter, the HSV thymidine kinase promoter, heat shock promoters, the early and late SV40 promoter, LTRs from refroviruses and the mouse metallothionein-I promoter. Other promoters known to control expression of genes m prokaryotic or eukaryotic cells or their viruses may also be used.<br>
Tissue-Specific Plant Promoters<br>
[0181]	The invention provides expression cassettes that can be expressed in a tissue-<br>
specific manner, such as that can express an aldolase, such as pyruvate aldolase, HMG and/or KHG aldolase enzyme in accordance with the invention in a tissue-specific manner. In some embodiments, the invention also provides plants or seeds that express an aldolase, such as pyruvate aldolase, HMG and/or KHG aldolase enzyme in accordance<br>
 <br>
with the invention in a tissue-specific manner. The tissue-specificity can be seed specific, stem specific, leaf specific, root specific, fiuit specific and the like.<br>
[0182]	The term "plant" includes whole plants, plant parts (such as leaves, stems, flowers,<br>
roots, etc), plant protoplasts, seeds and plant cells and progeny of same. The class of plants which can be used in the method in accordance with the invention is generally as broad as the class of higher plants junenable to transformation techniques, including angiosperms (monocotyledonous and dicol;yledonous plants), as well as gymnosperms. It includes plants of a variety of ploidy levels, including polyploid, diploid, haploid and hemizygous states. As used herein, the term "transgenic plant" includes plants or plant cells into which a heterologous nucleic acid sequence has been inserted, such as the nucleic acids and various recombinant constructs (such as expression cassettes) in accordance with the invention.<br>
[0183]	In some embodiments, a constitutive promoter such as the CaMV 35S promoter<br>
can be used for expression in specific parts of the plant or seed or throughout the plant. For example, for overexpression, a plant promoter fragment can be employed which will direct expression of a nucleic acid in some or all tissues of a plant, such as a regenerated plant. Such promoters are referred to herein as "constitutive" promoters and are active under most environmental conditions and states of development or cell differentiation. Examples of constitutive promoters include the cauliflower mosaic virus (CaMV) 35S transcription initiation region, the 1'- or 2'- promoter derived from T-DNA of Agrobacterium tumefaciens, and other transcription initiation regions from various plant genes known to those of skill. Such genes include, such as ACTIl from Arabidopsis (Huang (1996) Plant Mol Biol. 33-.125-139); Cal3 fi-om Arabidopsis (GenBank No. U43147, Zhong (1996) Mol. Gen. Genet. 251:196-203); the gene encoding stearoyl-acyi carrier protein desaturase om Brassica napus (GenbankNo. X74782, Solocombe(1994) Plant Physiol 104:1167-1176); GPcl from maize (GenBank No. X15596; Martinez (1989) J. Mol. Biol 208:551-565); the Gpc2 from maize (GenBank No. U45855, Manjunath (1997) Plant Mol Biol 33:97-112); plant promoters described in U.S. Patent Nos. 4,962,028; 5,633,440.<br>
10184]	The invention uses tissue-specific or constitutive promoters derived from viruses<br>
which can include, such as the tobamovirus subgenomic promoter (Kmnagai (1995) Proc. Nati. Acad. Sci. USA 92:1679-1683; tiie rice tungro bacillifoim virus (RTBV), which<br>
 <br>
replicates only in pMoeni cells in infected rice plants, with its promoter which drives strong phloem-specific reporter gene expression; the cassava vein mosaic virus (CVMV) promoter, with highest activity in vascular elements, in leaf mesophyll ceils, and in root tips (Verdaguer (1996) Plant Mol. Biol. 31:1129-1139).<br>
[01851	In some embodiments, the plant promoter directs expression of aldolase, such as<br>
pyruvate aldolase, such as HMG and/or KHG aldolase enzyme-expressing nucleic acid in a specific tissue, organ or cell type (ie. tissue-specific promoters) or may be otherwise imder more precise environmental or developmental control or under the control of an inducible promoter. Examples of environmental conditions that may affect transcription include anaerobic conditions, elevated temperature, the presence of light, or sprayed with chemicals/hormones. For example, the invention incorporates the drought-inducible promoter of maize (Busk (1997) supra); the cold, drought, and high salt inducible promoter from potato (Kirch (1997) Plant Mol. Biol. 33:897 909).<br>
[0186]	In some embodiments, tissue-specific promoters promote transcription only within   .<br>
a certain time frame of developmental stage within that tissue. See Blazquez (] 998) Plant Cell 10:791-800, characterizing the Arabidopsis LEAFY gene promoter. See also Cardon (1997) Plant J 12:367-77, describing the transcription factor SPL3, which recognizes a conserved sequence motif in the promoter region of the A. thaliana floral meristem identity gene API; and Mandel (1995) Plant Molecular Biology, Vol. 29, pp 995-1004, describing the meristem promoter eIF4. Tissue specific promoters which are active throughout the life cycle of a particular tissue can be used. In some embodiments, the nucleic acids in accordance with the invention are operably linked to a promoter active primarily only in cotton fiber cells. In some embodiments, the nucleic acids in accordance with the invention are operably linked to a promoter active primarily during the stages of cotton fiber cell elongation, such as described by Rinehart (1996) supra. The nucleic acids can be operably linked to the Fbl2A gene promoter to be preferentially expressed in cotton fiber cells (Ibid) . See also, John (1997) Proc. Natl. Acad. Sci. USA 89:5769-5773; John, et a!., U.S. Patent Nos. 5,608,148 and 5,602,321, describing cotton fiber-specific promoters and methods for the construction of transgenic cotton plants. Root-specific promoters may also be used to express the nucleic acids in accordance with the invention. Examples of root-specific promoters include the promoter from the alcohol dehydrogenase gene (DeLisle (1990) hit. Rev, Cytol. 123:39-60).   Other promoters that<br>
 <br>
can be used to express the nucleic acids in accordance with the invention include, such as ovule-specific, embryo-specific, end(3spenn-specific, integument-specific, seed coat-specific promoters, or some combination thereof; a leaf-specific promoter (see Busk (1997) Plant J. 11:1285 1295, describing a leaf-specific promoter in maize): the 0RFI3 promoter from Agrobacterium rhizogenes (which exhibits high activity in roots, see Hansen (1997) supra); a maize pollen specific promoter (see Guerrero (1990) Mol. Gen. Genet. 224:]6] 168); a tomato promoter active during fruit ripening, senescence and abscission of leaves and, to a lesser extent, of flowers can be used (see Blume (1997) Plant J. 12:731 746); a pistil-specific promoter from the potato SK2 gene (see Picker (1997) Plant Mol. BioL 35:425 431); the Blec4 gene from pea, which is active in epidermal tissue of vegetative and floral shoot apices of transgenic alfalfa making it a useful tool to target the expression of foreign genes to the epidermal layer of actively growing shoots or fibers; the ovule-specific BELl gene (see Reiser (1995) Cell 83:735-742, GenBank No. U39944); and/or, the promoter in Klee, U.S. Patent No. 5,589,583, describing a plant promoter region is capable of conferring high levels of transcription in meristematic tissue and/or rapidly dividing cells.<br>
[6187]	In some embodiments, plant jjromoters which are inducible upon exposure to<br>
plant hormones, such as auxins, are used to express the nucleic acids in accordance vinth the invention. For example, the invention can use the auxin-response elements El promoter fragment (AuxREs) in the soybean (Glycine max L.) (Liu (1997) Plant Physiol. 115:397-407); the auxin-responsive Arabidopsis GST6 promoter (also responsive to salicylic acid and hydrogen peroxide) (Chen (1996) Plant J, 10: 955-966); the auxin-inducible parC promoter fi-om tobacco (Sakai (1996) Plant Cell Physiol. 37:906-913); a plant biotin response element (Streit (1997) Mol. Plant Microbe Interact. 10:933-937); and, the promoter responsive to the stress hormone abscisic acid (Sheen (1996) Science 274:1900-1902).<br>
[0188]	The nucleic acids in accordance with the invention can also be operably linked to<br>
plant promoters which are inducible upon exposure to chemicals reagents which can be applied to the plant, such as herbicides or antibiotics. For example, the maize ln2-2 promoter, activated b}-' benzenesulfonaiaide herbicide safeners, can be used fDe Veylder (1997) Plant Cell Physiol. 38:568-577); application of different herbicide safeners induces distinct gene expression patterns, including expression in the root, hydathodes.<br>
 <br>
and the shoot apical meristem. uociing sequence can oe imaer xne control oi. sucn as a tetracycline-inducible promoter, such as described with transgenic tobacco plants containing the Avena sativa L. (oat) arginine decarboxylase gene (Masgrau (1997) Plant J. 11:465-473); or, a salicylic acid-responsive element (Stange (1997) Plant J. 11:1315-1324). Using chemically- (such as hormone- or pesticide-) induced promoters, i.e., promoter responsive to a chemicaJ which can be applied to the transgenic plant in the field, expression of a polypeptide in accordance with the invention can be induced at a particular stage of development of tlie plant. Thus, the invention also provides for transgenic plants containing an inducible gene encoding for polypeptides in accordance with the invention whose host range is limited to target plant species, such as com, rice, barley, soybean, tomato, wheat, potato or other crops, inducible at any stage of development of the crop.<br>
[0189]	One of skill will recognize that a tissue-specific plant promoter may drive<br>
expression of operably linked sequences in tissues other than the target tissue.  Thus, In , some embodiments, a tissue-specific promoter is one that drives expression preferentially in the target tissue or cell type, but may also lead to some expression in other tissues as well.<br>
[0190]	The nucleic acids in accordance with the invention can also be operably hnked to<br>
plant promoters which are inducible upon exposure to chemicals reagents. These reagents include, such as herbicides, s&gt;Tithetic auxins, or antibiotics which can be applied, such as sprayed, onto transgenic plants. Inducible expression of the aldolase, such as pyruvate aldolase, such as HMG and/or KHG aldolase enzyme-producing nucleic acids in accordance with the invention will aiJow the grower to select plants with the optimal aldolase, such as pyruvate aldolase, such as HMG and/or KHG aldolase enzyme expression and/or activity. The development of plant parts can thus controlled, hi this way the invention provides the means to facilitate the harvesting of plants and plant parts. For example, in various embodiments, the maize In2-2 promoter, activated by benzenesulfonamide herbicide safeners, is used (De Veylder (1997) Plant Cell Physiol. 38:568-577); appHcation of different herbicide safeners induces distinct gene expression patterns, including expression in the root, hydathodes, and the shoot apical meristem. Coding sequences in accordance with the mvention are also under the control of a tetracycline-inducible  promoter,   such  as  described  with  transgenic  tobacco   plants<br>
 <br>
containing the Avena sativa L. (oat) arginine decarboxylase gene {Masgrau (1997) Plant<br>
J.   11:465-473):   or,   a   salicylic   acid-responsive   element   (Stange   (1997)   Plant   J.<br>
11:1315-1324).<br>
[0191J	In     some    embodiments,    proper    polypeptide    expression    may    require<br>
polyadenylation region at the 3'-end -of the coding region. The polyadenylatlon region can be derived from the natural gene,.from a variety of other plant (or animal or other) genes, or from genes in the Agrobacterial T-DNA.<br>
Expression vectors and cloning vehicles<br>
[0192]	The invention provides expression vectors and cloning vehicles  comprising<br>
nucleic acids in accordance with the invention, such as sequences encoding the aldolase, such as pyruvate aldolase, such as HIvfG and/or KHG aldolase enzymes in accordance with the invention. Expression vectors and cloning vehicles in accordance with the invention can comprise viral particles, baculcvirus, phage, plasmids, phagemids, cosmids, fosmids, bacterial artificial chromosomes, viral DNA (such as vaccinia, adenovirus, foul pox virus, pseudorabies and derivatives of SV40), Pl-based artificial chromosomes, yeast plasmids, yeast artificial criromosomes, and any other vectoi specific for specific hosts of interest (such as bacillus, Aspergillus and yeast). Vectors in accordance with the invention can include chromosomal, non-chromosomal and synthetic DNA sequences. Large numbers of suitable vectors are known to those of skill in the art, and are commercially avlable. Exemplary vectors include: bacterial: pQE'*' vectors (Qiagen, Valencia, CA), pBLUESCRIPTT' plasmids, pNH vectors, lambda-ZAP vectors (Stratagene, La Jolla, CA); ptrc99a, pKK223-3, pDR540, pRJT2T (GE Healthcare, Piscataway, NJ), pET vectors (Novagen, Madison, WI); Eukaryotic: pXTl, pSG5 (Stratagene, La Jolla, CA), pSVK3, pBPV, pMSG, pSVLSV40 (Pharmacia). However, any other plasmid or other vector may be used so long as they are replicable and viable in the host. Low copy number or high copy number vectors may be employed with the present invention. "Plasmids" can be commercially available, publicly available on an unrestricted basis, or can be constructed from available plasmids in accord with published procedures. Equivalent plasmids to those described herein are known in the art and will be apparent to the ordinarily skilled artisan.<br>
[0193]	The expression vector can comprise a promoter, a ribosome binding site for<br>
translation initiation and a transcription terminator.    The vector may also include<br>
 <br>
appropriate sequences for amplifying expression. Mammalian expression vectors can comprise an origin of replication, anj' necessary' ribosome binding sites, a poIyadenyJation site, splice donor and acceptor sites, transcriptionai termination sequences, and 5' flanking non-tranS'Cribed sequences. In some embodiments, DNA sequences derived from the SV40 splice and polyadenylation sites may be used to provide the required non-transcribed genetic elements.<br>
[0194]	In some embodiments, the expression vectors contain one or more selectable<br>
marker genes to permit selection of host cells containing the vector. Such selectable markers include genes encoding dihydrofolate reductase or genes conferring neomycin resistance for eukaryotic cell culture, genes conferring tetracycline or ampicillin resistance in E. coli, and the S. cerevisiae TRPl gene. Promoter regions can be selected from any desired gene using chloramphenicol transferase (CAT) vectors or other vectors with selectable markers.<br>
[0195]	In some embodiments, vectors for expressing the polypeptide or fragment thereof<br>
in eukaryotic cells contain enhancers to increase expression levels. Enhancers are cis-acting elements of DNA that can be from about 10 to about 300 bp in length. They can act on a promoter to increase its transcription. Exemplary enhancers include the SV40 enhancer on the late side of the replication origin bp 100 to 270, the cytomegalovirus early promoter enhancer, the polyoma enhancer on the late side of the replication origin, and the adenovirus enhancers.<br>
[0196]	A nucleic acid sequence can be inserted into a vector by a variety of procedures.<br>
In general, the sequence is ligated to the desired position in the vector following digestion of the insert and the vector with appropriate restriction endonucleases. Alternatively, blunt ends in both the insert and the vector may be ligated. A variety of cloning techniques are known in the art, such as described in Ausubel e( al. Current Protocols in Molecular Biology, John Wiley &amp; Sons, inc. 1997 and Sambniok et al, Molecular Cloning: A Laboratory Manual 2nd Ed., Cold Spring Harbor Laboratory Press (1989). Such procedures and others are deemed to be within the scope of those skilled in the art.<br>
[0197]	The vector can be in the form of a plasmid, a viral particle, or a phage.   Other<br>
vectors include chromosomal, non-chromosomal and synthetic DNA sequences, derivatives of SV40; bacterial plasmids, phage DNA, baculovirus, yeast plasmids, vectors derived from combinations '■' "'"■''' .A niiao* riNA. viral DNA such as vaccinia.<br>
 <br>
adenovirus, fowl pox vims, and pseudorabies. A variety of cloning and expression vectors for use with prokaryotic and eukaryotic hosts are described by, such as Sambrook.<br>
[0198]	Particular bacterial vectors which can be used include the commercially available<br>
plasmids comprising genetic elements of the well known cloning vector pBR322 (ATCC 37017), pKK223-3 (Pharmacia Fine Chemicals, Uppsala, Sweden), GEMl (Promega Biotec, Madison, WI, USA) pQE70, pQE60, pQE-9 (Qiagen, Valencia, CA), pDlO, psiX174 pBLUESCRIPT II KS, pNH8A, pNH16a, pNHlSA, pNH46A (Shtagene, La JoUa, CA), ptrc99a, pKK223-3, pKK233-3, DR540, pRIT5 (Pharmacia), pKK232-8, pET (Novagen, Madison, WI), and pCM7. Patticuiar eukaryotic vectors include pSV2CAT, pOG44, pXTl, pSG (Stratagene, La Jolla, CA) pSVKS, pBPV, pMSG, and pSVL (Pharmacia). However, any other vector may be used as long as it is replicable and viable in the host cell.<br>
[0199]	The nucleic acids in accordance with the invention can be expressed in expression<br>
cassettes, vectors or viruses and transiently or stably expressed in plant cells and seeds. One exemplary transient expression system uses episomal expression systems, such as cauliflower mosaic virus (CaMV) viral RNA generated in the nucleus by transcription of an episomal mini-chromosome containing supercoJJed DNA, see Covey (1990) Proc. Natl. Acad. Sci. USA 87:1633-1637. Altemativeiy, coding sequences, i.e., all or sub-fragments of sequences in accordance with the invention can be inserted into a plant host cell genome becoming an integral part of the host chromosomal DNA. Sense or antisense transcripts can be expressed in this manner. A vector comprising the sequences (such as promoters or coding regions) from nucleic acids in accordance with the invention can comprise a marker gene that confers a selectable phenotype on a plant cell or a seed. For example, the marker may encode biocide resistance, such as antibiotic resistance, such as resistance to kanamycin, G418, bleomycin, hygromycin, or herbicide resistance, such as resistance to chlorosulfuron or Basta.<br>
[0200]	Expression vectors capable of expressing nucleic acids and proteins in plants are<br>
well known in the art, and can include, such as vectors from Agrobacterium spp., potato virus X (see Angell (1997) EMBO J. 16:3675-3684), tobacco mosaic virus (see Casper (1996) Gene 173:69-73), tomato bushy stunt virus (see Hilknan (1989) Virology 169:42-50), tobacco etch virus (see Dolja (1997) Virology 234:243-252), bean golden mosaic virus (see Morinaga (1993) Microbiol Immunol. 37:471-476), cauliflower mosaic<br>
 <br>
virus (see Cecchini (1997) Mol. Plant Microbe Interact. 10:1094-1101), maize Ac/Ds transposable element (see Rubin (1997) MoL Cell Biol. 17:6294-6302; Kunze (1996) Curr. Top. Microbiol. Immunol. 204:161-194), and the maize suppressor-mutator (Spm) transposable element (see Schiappi (1996) Plant Mol. Biol. 32:717-725); and derivatives thereof.<br>
[0201]	In some embodiments, the expression vector can have two replication systems to<br>
allow it to be maintained in two organisms, for example in mammalian or insect cells for expression and in a prokaryotic host for cloning and amplification. Furthermore, for integrating expression vectors, the expression vector can contain at let one sequence homologous to the host cell genome. It can contain two homologous sequences which flank the expression construct. The integrating vector can be directed to a specific locus in the host cell by selecting the apprijpriate homologous sequence for inclusion in the vector. Constructs for integrating vectors are well known in the art.<br>
[0202]	Expression vectors in accordance with the invention may also include a selectable<br>
marker gene to allow for the selection of bacterial strains that have been transformed, such as genes which render the bacteria resistant to drugs such as ampicillin, chloramphenicol, erythromycin, kanamycin, neomycin and tetracycline. Selectable markers can also include biosynthetic genes, such as those in the histidine, tryptophan and leucine biosynthetic pathways.<br>
[0203]	The DNA sequence in the expression vector is operatively linked to an appropriate<br>
expression control sequence(s) (promoter) to direct RNA synthesis. Particular named bacterial promoters include lad, lacZ, T3, T7, gpt, lambda PR, PL and trp. Eukaryotic promoters include CMV immediate early, HSV thymidine kinase, early and late SV40, LTRs from retrovirus and mouse metallothionein-I. Selection of the appropriate vector and promoter is well within the level of ordinary skill in the art. The expression vector also contains a ribosome binding site for translation initiation and a transcription terminator. The vector may also include appropriate sequences for amplifying expression. Promoter regions can be selected from any desired gene using chloramphenicol transferase (CAT) veaors or other vectors vvth selectable markers. In addition, the expression vectors in some embodiments contain one or more selectable marker genes to provide a phenotypic trait for selection of transformed host cells such as<br>
 <br>
dihydrofolate reductase or neomycin resistance for eukaryotic cell culture, or such as tetracycline or ampicillin resistance in E. coli.<br>
[0204]	Mammalian expression vectors may also comprise an origin of replication, any<br>
necessar&gt;' ribosome binding sites, a polyadenylation site, splice donor and acceptor sites, transcriptional tenninarion sequences and 5' flanking nontranscribed sequences. In some embodiments, DNA sequences derived from the SV40 splice and polyadenylation sites may be used to provide the required noatranscribed genetic elements.<br>
[0205J	Vectors for expressing the polypeptide or fragment thereof in eukaryotic cells may<br>
also contain enhancers to increase expression levels. Enhancers are cis-acting elements of DNA, usually from about 10 to about 300 bp in length that act on a promoter to increase its transcription. Examples include the SV40 enhancer on the late side of the replication origin bp 100 to 270, the cytomegalovirus early promoter enhancer, the polyoma enhancer on the late side of the replication origin and the adenovirus enhancers.<br>
[0206]	In addition, the expression vectors can contain one or more selectable marker<br>
genes to permit selection of host cells containing the vector. Such selectable markers include genes encoding dihydrofolate reductase or genes conferring neomycin resistance for eukaryotic cell culture, genes conferring tetracycline or ampicillin resistance in E. coli and the S. cerevisiae TRPl gene.<br>
[0207]	In some embodiments, the nucleic acid encoding one of the polypeptides in<br>
accordance with the invention, and sequences substantially identical thereto, or fragments comprising at least about 5, 10, 15, 20, 25, 30, 35, 40, 50, 75, 100, or 150 or more consecutive amino acids thereof is assembled in appropriate phase with a leader sequence capable of directing secretion of the translated polypeptide or fragment thereof In some embodiments, the nucleic acid can encode a fusion polypeptide in which one of the polypeptides in accordance with the invention, or fragments comprising at least 5, 10, 15, 20, 25, 30, 35, 40, 50, 75,100, or 150 or more consecutive amino acids thereof is fused to heterologous peptides or polypeptides, such as N-terminal identification peptides which impart desired characteristics, such as increased stability or simplified purification.<br>
[0208}	The appropriate DNA sequence may be inserted into the vector by a variety of<br>
procedures. In general, the DNA sequence is ligated to the desired position in the vector following digestion of the insert and the vector with appropriate restriction endonucleases. Alternatively, blunt ends in both the insert and the vector may be ligated.<br>
 <br>
A variety of cloning techniques are disclosed in Ausubel et al. Current Protocols in<br>
Motecular Biology, John Wiley &amp; Sons, Inc. 1997 and Sambrook et al, Molecular Cloning:<br>
A Laboratory Manual 2nd Ed., Cold Spring Harbor Laborator&gt;' Press {1989).    Such<br>
procedures and others are deemed to be: within the scope of those skilled in the art.<br>
[0209]	The vector may be, for example, in the form of a plasmid, a viral particle, or a<br>
phage. Other vectors include chromosomal, nonchromosomal and synthetic DNA sequences, derivatives of SV40; bacterial plasniids, phage DNA, baculovirus, yeast plasmids, vectors derived from combinations of plasmids and phage DNA, viral DNA such as vaccinia, adenovirus, fowl pox. virus and pseudorabies. A variety of cloning and expression vectors for use with prokaryotic and eukaryotic hosts are described by Sambrook, et al., Molecular Cloning: A Laboratory Manual, 2nd Ed., Cold Spring Harbor, N.Y., (1989).<br>
Host cells and transformed cells<br>
[0210]	The invention also provides transformed cells comprising nucleic acid sequences<br>
in accordance with the invention, such as sequences encoding aldolases, such as pyruvate aldolases, HMG and/or KHG aldolase enzymes in accordance with the invention, or vectors in accordance with the invenlion. The host cell may be any of the host cells familiar to those skilled in the ari:, including prokaryotic cells, eukaryotic cells, such as bacterial cells, fungal cells, yeast cells, mammalian cells, insect cells, or plant cells. Exemplary bacterial cells include any species of Streptomyces, Staphylococcus, Pseudomonas or Bacillus, including E. coli. Bacillus subtilis, Pseudomonas fluorescens. Bacillus cereus, or Salmonella typhimurium. Exemplary fungal cells include any species of Aspergillus. Exemplary yeast cells include any species of Pichia, Saccharomyces, Schizosaccharomyces, or Schwanniornyces„ including Pichia pastoris, Saccharomyces cerevisiae, or Schizosaccharomyces pombe. Exemplary insect cells include any species of Spodoptera or Drosophila, including Drosophila S2 and Spodoptera Sf9. Exemplary animal cells include CHO, COS or Bowes melanoma or any mouse or human cell line. The selection of an appropriate host is within the abilities of those skilled in the art. Techniques for transforming a wide variety of higher plant species are well known and described in the technical and scientific literature. See Weising (1988) Ann. Rev. Genet. 22:421-477; U.S. Patent No. 5,750,870.<br>
 <br>
[0211]	The vector can be introduced into the host cells using any of a variety of<br>
techniques, including transformation, transfection, transduction, viral infection, gene guns, or Ti-mediated gene transfer. Particular methods include calcium phosphate transfection, DEAE-Dextran mediated transfection, lipofection, or electroporation (Davis, L., Dibner, M., Battey, I., Basic Methods in Molecular Biology, (1986)).<br>
[5212]	In some embodiments, the nucleic acids or vectors in accordance with the<br>
invention are introduced into the cells for screening, thus, the nucleic acids enter the cells in a manner suitable for subsequent expression of the nucleic acid. The method of introduction is largely dictated by the targeted eel! type. Exemplary methods include CaP04 precipitation, liposome fiision, lipofection (such as LIPOFECTIN""), electroporation, viral infection, etc. The candidate nucleic acids may stably integrate into the genome of the host cell (for example, with retroviral introduction) or may exist either transiently or stably in the cytoplasm (i.e. through the use of traditional plasnuds, utilizing standard regulatory sequences, selection markers, etc.). As many pharmaceutically important screens require human or model mammalian cell targets, retroviral vectors capable of transfecting such targets can be used.<br>
[02131	Where appropriate, the engineered host cells can be cultured in conventional<br>
nutrient media modified as appropriate for activating promoters, selecting transformants or amplifying the genes in accordance with the invention. Following transformation of a suitable host strain and growth of the host strain to an appropriate cell density, the selected promoter may be induced by appropriate means (such as tempemture shift or chemical induction) and the cells may be cultured for an additional period to allow them to produce the desired polypeptide or fiagment thereof.<br>
[0214]	Cells can be harvested by centrifugation, disrupted by physical or chemical<br>
means, and the resulting crude extract is retained for fiirther purification. Microbial cells employed for expression of proteins can be disrupted by any convenient method, including freeze-thaw cycling, sonication, mechanical dismption, or use of cell lysing agents. Such methods are well known to those skilled in the art. The expressed polypeptide or Augment thereof can be recovered and purified iirom recombinant cell cultures by methods including ammonium sulfate or ethanol precipitation, acid extraction, anion or cation exchange chromatography, phosphoceJlulose chromatography, hydrophobic   interaction   chromatography,   affinity   chromatography,   hydroxylapatite<br>
 <br>
chromatography and lectin chromatography. Protein refolding steps can be used, as necessary, in completing configuration of the polypeptide. If desired, high performance liquid chromatography (HPLC) can be employed for final purification steps.<br>
10215]	The constructs in host cells can be used in a conventional manner to produce the<br>
gene product encoded by the recombinant sequence. Depending upon the host employed in a recombinant production procedure, the polypeptides produced by host cells containing the vector may be glycosylated or may be non-glycosylated. Polypeptides in accordance with the invention may or may not also include an initial methionine amino acid residue.<br>
[0216]	Cell-fi-ee translation systems can also be employed to produce a polypeptide in<br>
accordance with the invention. Cell-free translation systems can use mRNAs transcribed from a DNA construct comprising a promoter operably linked to a micleic acid encoding the polypeptide or fragment thereof In some embodiments, the DNA construct may be linearized prior to conducting an in vitfo transcription reaction. The franscribed mRNA is then incubated with an appropriate cell-fi-ee translation extract, such as a rabbit reticulocyte extract, to produce the desired polypeptide or fragment thereof<br>
[0217]	The expression vectors can contain one or more selectable marker genes to.<br>
provide a phenotypic trait for selection of transformed host cells such as dihydrofolate reductase or neomycin resistance for eukaryotic cell culture, or such as tetracycline or ampicillin resistance in E. coli.<br>
[0218]	Host cells containing the po]}'nucleot3des of interest, such ss nucleic acids in<br>
accordance with the invention, can be cultured in conventional nutrient media modified as appropriate for activating promoters, selecting transformants or amplifying genes. The culture conditions, such as temperature, pH and the like, are those previous]y used with the host cell selected for expression and will be apparent to the ordinarily skilled artisan. The clones which are identified as having the specified enzyme activity may then be sequenced to identjiy the polynucleotide sequence encoding an enzyme having the enhanced activity.<br>
[0219]	The invention provides methods for overexpressing recombinant aldolases, such<br>
as pyruvate aldolases, such as HMG <md khg aldolase enzymes in cells comprising expressing a vector nucleic acid accordance with the invention such as sequence at least about></md>
 <br>
54%, 55%, 56%, 57%, 58%, 59%, 60%, 61%, 62%), 63%., 64%, 65%, 66%), 67%, 68%, 69%, 70%, 71%, 72%, 73%, 74%, 75%, 76%, 77%, 78%, 79%, 80%, 81%, 82%, 83%,, 84%., 85%, 86%, 87%, 88%, 89%, 90%, 91%, 92%o, 93%&gt;, 94%, 95%, 96%, 97%, 98%, 99%o, or more sequence identity to a sequence in accordance with the invention over a region of at least about 100 residues, wherein the sequence identities are determined by analysis with a sequence comparison algorithm or by visual inspection, or, a nucleic acid that hybridizes under stringent conditions to a nucleic acid sequence in accordance with the invention, or a subsequence thereof. The overexpression can be effected by any means, such as use of a high activity promoter, a dicistronic vector or by gene amplification of the vector.<br>
[0220]	The  nucleic acids in  accordance with the  invention can be  expressed,  or<br>
overexpressed, in any in vitro or in vivo expression system. Any cell culture systems can be employed to express, or over-express, recombinant protein, including bacterial, insect, yeast, fungal or mammalian cultures. Over-expression can be effected by appropriate choice of promoters, enhancers, vectors (such as use of replicon vectors, dicistronic vectors (see Gurtu (1996) Biochem. Biophys. Res. Commun. 229:295-8), media, culture systems and the like. In some embodiments, gene amplification using selection markers, such as glutamine synthetase (see Sanders (1987) Dev, Biol, Stand. 66:55-63), in cell systems are used to overexpress the polypeptides in accordance with the invention.<br>
[0221]	Additional details regarding this approach are in the public literature and/or are<br>
known to the skilled artisan. In a particular non-limiting exemplification, such publicly available literature includes EP 0659215 (WO 9403612 Al) (Nevalainen et al.); Lapidot, A., Mechaly, A., Shoham, Y., "Overexpression and single-step purification of a thermostable xylanase from Bacillus stearothermophilus T-6," J. Biotechnol Nov 51:259-64 (1996); Luthi, E., Jasmat, N.B., Bergquist, P.L., "Xylanase from the extremely thermophilic bacterium Caldocellum saccharolyticuni: overexpression of the gene in Escherichia coli and characterization of the gene product," Appl. Environ. Microbiol. Sep 56:2677-83 (1990); and Sung, W.L., Luk, C.K., Zahab, D.M., Wakarchuk, W., "Overexpression of the Bacillus subtilis and circulans xylanases in Escherichia coli,'" Protein Expr. Purif. Jun 4:200-6 (1993), although these references do not teach tlie inventive enzymes of the instant application.<br>
[0222]	The host cell may be any of the host cells familiar to those skilled in the art,<br>
including prokaryotic cells, eukaryotic cells, mammalian cells, insect cells, or plant cells.<br>
 <br>
As representative examples of appropriate iiosts, there may be mentioned: bacteria! cells, such as £, colU Streptomyces, Bacillus subtilis, Bacillus cereus. Salmonella typhimurium and various species within the genera Pseudomonas, Streptomyces and Staphylococcus, fiingal cells, such as Aspergillus, yeast such as any species of Pichia, Saccharomyces, Schizosaccharomyces, Schwanniomyces, including Pichia pastohs, Saccharomyces cerevisiae, or Schizosaccharomyces pombe, insect cells such as Drosophila S2 and Spodoptera Sf9, animal cells such as CHO, COS or Bowes melanoma and adenoviruses. The selection of an appropriate host is within the abilities of those skilled in the art.<br>
[0223]	The vector may be introduced into the host cells using any of a variety of<br>
techniques, including transformation, transfection, transduction, viral infection, gene guns, or Ti-mediated gene transfer. Paiticuhtr methods include calcium phosphate transfection, DEAE-Dextran mediated ti"ansfection, lipofection, or electroporation (Davis, L., Dibner, M., Battey, I., Basic Methods in Molecular Biology, (1986)).<br>
[0224]	Where appropriate, the engineered host cells can be cultured in conventional<br>
nutrient media modified as appropriate for activating promoters, selecting transformants or amplifying the genes in accordance with the invention. Following transformation of a suitable host strain and growth of the host strain to an appropriate cell density, the selected promoter may be induced bj' appropriate means (such as temperature shift or chemical induction) and the cells may be cultured for an additional period to allow them to produce the desired polypeptide or fi-agment thereof<br>
10225]	Cells can be harvested by centrifugation, disrupted by physical or chemical means<br>
and the resulting crude extract is R:tained for further purification. Microbial cells employed for expression of protein:! can be disrupted by any convenient method, including freeze-thaw cycling, sonicadon, mechanical disruption, or use of cell iysing agents. Such methods are well known to those skilled in the art. The expressed polypeptide or fragment thereof can be recovered and purified firom recombinant cell cultures by methods including ammonium sulfate or ethanol precipitation, acid extraction, anion or cation exchange chromatography, phosphocelJulose chromatography, hydrophobic interaction chromatography, affinity chromatography, hydroxylapatite chromatography and lectin chromatography. Protein refolding steps can be used, as necessary, in completing configuration of the polypeptide. If desired, high performance liquid chromatography (HPLC) can be employed for final purification steps.<br>
 <br>
[0226]	Various mammalian eel!  culture systems can also be employed to express<br>
recombinant protein. Examples of mammalian expression systems include the COS-7 lines of monkey kidney fibroblaste (described by Gluzman, Cell, 23.:175, 1981) and other cell lines capable of expressing proteins from a compatible vector, such as the C127, 3T3, CHO, HeLa and BHK cell lines.<br>
[0227]	The constructs in host cells can be used in a conventional manner to produce the<br>
gene product encoded by the recombinant sequence. Depending upon the host employed in a recombinant production procedure, the polypeptides produced by host cells containing the vector may be glycosylated or may be non-giycosyiated. Polypeptides in accordance with the invention may or may not also include an initial methionine amino acid residue.<br>
[0228]	Alternatively, the polypeptides in accordance with the invention, or fragments<br>
comprising at least 5, 10, 15, 20, 25, 30, 35, 40, 50, 75, 100, or 150 or more consecutive amino acids thereof can be synthetically produced by conventional peptide synthesizers, such as discussed below. In other embodiments, fragments or portions of the polypeptides may be employed for producing the corresponding full-length polypeptide by peptide synthesis: therefore, the iragments may be employed as intermediates for producing the ftill-length po]}7&gt;eptides.<br>
[0229]	Cell-iree translation  systems can also be employed  to produce one of the<br>
polypeptides in accordance with the invention, or fragments comprising at least 5, 10,15, 20, 25, 30, 35, 40, 50, 75, 100, or 150 or more consecutive amino acids thereof using mRNAs transcribed from a DNA construct comprisii a promoter operably linked to a nucleic acid encoding the polypeptide or fragment thereof. In some embodiments, the DNA construct may be linearized prior to conducting an in vitJ-o transcription reaction. The transcribed mRNA is then incubated with an appropriate ceil-free translation extract, such as a rabbit reticulocyte extract, to produce the desired polypeptide or fragment thereof.<br>
Amplification of Nucleic Acids<br>
[0230]	In practicing the invention, nucleic ids in accordance with the invention and<br>
nucleic acids encoding the aldolase, such as pyruvate aldolase, such as HMG and/or KHG aldolase enzymes in accordance with the invention, or modified nucleic acids in accordance with the invention, can be reproduced by amplification, such as PCR.<br>
 <br>
Amplification can also be used to clone or modify the nucleic acids in accordance witii the invention. Thus, the invention provides amplification primer sequence pairs for amplifying nucleic acids in accordance with the invention. One of skill in the art can design amplification primer sequence pairs for any part of or the fiili length of these sequences.<br>
[0231]	In  some  embodiments,  the  invention  provides  nucleic   acids  amplified  by<br>
amplification primer pairs in accordance with the invention, such as primer pairs as set forth by about the first (the 5') 12, 13, 14, 15, 16, 17, !8, 19, 20, 21, 22, 23, 24, or 25 or more residues of nucleic acids in accordance with the invention, and about the first (the 5') 15, 16, 17, 18, 19, 20, 21, 22, 23, 24, or 25 or more residues of the complementary strands. In some embodiments, the invention provides amplification primer sequence pairs for amplifying a nucleic acid encoding a polypeptide having an aldolase, such as pyruvate aldolase, such as HMG and/or KHG aldolase enzyme, activity, wherein the primer pair is capable of amplifying a nucleic acid comprising a sequence in accordance with the invention, or fragments or subsequences thereof One or each member of the amplification primer sequence pair can comprise an oligonucleotide comprising at least about 10 to 50 or more consecutive ba.ses of the sequence, or about 12, 13, 14, 15, 16, 17, 18, 19, 20, 21, 22, 23, 24, or 25 or more consecutive bases of the sequence. In some embodiments, the invention provides amplification primer pairs, wherein the primer pair comprises a fust member having a sequence as set forth by about the first (the 5') 12, !3, 14, 15, 16, 17, 18, 19, 20, 21, 22, 23, 24, or 25 or more residues of a nucleic acid in accordance with the invention, and a second member having a sequence as set forth by about the first (the 5') 12, 13, 14, 15, 16, 17, 18, 19, 20, 21, 22, 23, 24, or 25 or more residues of the complementary strand of the fust member.<br>
[0232]	The invention provides aldolase, such as pyruvate aldolase, such as HMG and/or<br>
KHG aldolase enzymes generated by amplification, such as polymerase chain reaction (PCR), using an amplification primer pair m accordance with the invention. In some embodiments, the invention provides methods of making an aldolase, such as pyruvate aldolase, HMG and/or KHG aldolase enzyme by amplification, such as PCR, using an amplification primer pair in accordance with the invention, hi some embodiments, the amplification primer pair amplifies a nucleic acid from a library, such as a gene libraiy, such as an environmental library.<br>
 <br>
10233]	Amplification reactions can also be used to quantify the amount of nucleic acid in<br>
a sample (such as the amount of mef;sa.ge in a cell sample), label the nucleic acid (such as to apply it to an array or a blot), detect the nucleic acid, or quantify the amount of a specific nucleic acid in a sample. ]_n some embodiments of the invention, message isolated from a cell or a cDNA library are amplified.<br>
[0234]	The skilled artisan can select and design suitable oligonucleotide amplification<br>
primers. Amplification methods are also well known in the art, and include, such as polymerase chain reaction, PCR (see PCR PROTOCOLS, A GUIDE TO METHODS AND APPLICATIONS, ed. Innis, Academic Press, N.Y. (1990) and PCR STRATEGIES (1995), ed. Innis, Academic Press, Inc., N.Y., ligase chain reaction (LCR) (see Wu (1989) Genomics 4:560; Landegren (1988) Science 241:1077; Barringer (1990) Gene 89:117); transcription amplification (see Kwoh (1989) Proc. Natl. Acad. Sci. USA 86:1173); and, self-sustained sequence replication (see Guatelli (1990) Proc. Natl. Acad. Sci. USA 87:1874); Q Beta replicase amplification (see Smith (1997) J. Clin. Microbiol. 35:1477-I49I), automated Q-beta replicase amplification assay (see Burg (1996) Mol. Cell. Probes 10:257-271) and other RNA polymerase mediated techniques (such asNASBA, Cangene, Mississauga, Ontario); see also Berger (1987) Methods Enzymol. 152:307-316; Ausubel et a!. Current Protocols in Molecular Biology, John Wiley &amp; Sons, Inc. 1997 and Sarabrook et at. Molecular Cloning: A Laboratory Manual 2nd Ed., Cold Spring Harbor Laboratory Press (1989). U.S. Patent Nos. 4,683,195 and 4,683,202; Sooknanan (1995) Biotechnology 13:563-564.<br>
Determining sequence identity in nucleic acids and polypeptides<br>
[0235]	The invention provides nucleic acids comprising sequences having at least about<br>
50%, 51%, 52%, 53%, 54%, 55%, SSy, 57%, 58%, 59%, 60%, 61%., 62%, 63%, 64%, i&gt;SV, d"/, 67%, 68%, 69%, 70%, 71%, 72%, 73%, 74%, 75%, 76%, 77%, 78%, 79%, 80%, 81%, 82%, 83%, 84%, 85%, 86%, 87%, 88%, 89%, 90%, 91%, 92%, 93%, 94%, 95%, 96%, 97%, 98%, 99%, or more, or complete (100%) sequence identity (homology) to a nucleic acid in accordance with the invention (see Sequence Listing) over a region of at least about S% 75, 100, 150, 200, 250, 300, 5Q, 400, 450, 500, 550, 600, 650, 700, 750, 800, 850, 900, 950, 1000, 1050, 1100, 1150, 1200, 1250, 1300, 1350, 1400, 1450, 1500, 1550 or more, residues. In some embodiments, the invention provides polypeptides comprising sequences having at least about 50%, 51%, 52%, 53%, 54%, 55%, 56%, 57%,<br>
 <br>
58%, 59%, 60%, 61%, 62%, 63%, 64%, 65%, 66%, 67%, 68%, 69%, 70%, 71%&gt;, 72%, 73%. 74%. 75%, 76%, 77%, 78%, 79%, 80%, 81%, 82%o, 83%, 84%. 85%, 86%, 87%. 88%, 89%. 9Q%o, 91%, 92%, 93%, 94%, 95%&gt;, 96%, 97%, 98%. 99%, or more, or complete (100%) sequence identity- to a polypeptide in accordance with the invention (see Sequence Listing). The extent of sequence identity (homology) may be determined using any computer program and associated parameters, including those described herein, such as BLAST 2.2.2. or FASTA version 3.0t78, with the default parameters.<br>
[0236]	Nucleic acid sequences in accordance with the invention can comprise at least 10,<br>
15, 20, 25, 30, 35, 40, 50, 75, 100, 150, 200, 300, 400, or 500 or more consecutive nucleotides of a sequence in accordance with the invention and sequences substantJaiJy identical thereto. Homologous sequences and fragments of nucleic acid sequences in accordance with the invention can refer to a sequence having at least about 50%), 51%i, 52%, 53%, 54%, 55%o, 56%, 57%,, 58%&gt;, 59%, 60%, 61%, 62%, 63%*, 64%, 65%c, 66%, 67%, 68%,, 69%., 70%o, 71%, 72%, 73%, 74%., 75%, 76%, 77%o, 78%, 79%, 80%), 81%, 82%o, 83%, 84%, 85%, 86%, 87%, 88%,, 89%, 90%, 91%., 92%, 93%, 94%, 95%., 96%, 97%o, 98%), 99%o, or more sequence identity (homology) to these sequences. Homology (sequence jdentit}') may be detennined using an)' of the computer programs and parameters described herein, including FASTA version 3.0t78 with the default parameters. Homologous sequences also include RNA sequences in which uridines replace the thymines in the nucleic aciii sequences in accordance with the invention. The homologous sequences may be obtained using any of the procedures described herein or may restilt from the correction of a sequencing error. It will be appreciated that the nucleic acid sequences in accordance with the invention can be represented in the traditionaJ single character formal (See the inside back cover of Stryer, LuberL Biochemistry, 3rd Ed., W. H Freeman &amp; Co., New York.) or in any other format which records the identity of the nucleotides in a sequence.<br>
[0237]	In some embodiments, sequenc;e comparison programs identified herein are used<br>
in this aspect in accordance with tlie invention, i.e., to determine if a nucleic acid or polypeptide sequence is within the scope in accordance with the invention. However, protein and/or nucleic acid sequence identities (homologies) may be evaluated using any sequence comparison algorithm or program known in the art. Such algorithms and programs include, but are by no means limited to, TBLASTN, BLASTP, FASTA,<br>
 <br>
TFASTA Md CLUSTALW (see Pearson and Lipman, Proc. Natl Acad. Sci. USA 85(8):2444-2448, 1988; Altschul et aL, J. Mol. Biol. 2I5(3):403-410, 1990: Thompson Nucleic Acids Res. 22(2):4673-4680, 1994; Higgins ei aL, Methods Enzymol. 266:383-402, 1996; Altschul el al, J. Mol. Biol. 2i5(3):403-410, 1990: Altschul et al. Nature Genetics 3:266-272,1993).<br>
[0238]	In some embodiments, homology or identity is measured using sequence analysis<br>
software (such as Sequence Analysis Software Package of the Genetics Computer Group, University of Wisconsin Biotechnology Center, 1710 University Avenue, Madison, Wl 53705). Such software matches similar sequences by assigning degrees of homology to various deletions, substitutions and other modifications. In some embodiments, the terms "homology" and "identity" in the context of two or more nucleic acids or polypeptide sequences, refer to two or more sequences or subsequences that are the same or have a specified percentage of amino acid residues or nucleotides that are the same when compared and aligned for maximum correspondence over a comparison window or designated region as measured using any number of sequence comparison algorithms or by manual alignment and visual inspection, hi some embodiments, for sequence comparison, one sequence acts as a reference sequence, to which test sequences are compared. When using a sequence comparison algorithm, test and reference sequences are entered into a computer, subsequence coordinates are designated, if necessary and sequence algorithm program parameters M-e designated. Default program parameters can be used, or alternative parameters can be designated. The sequence comparison algorithm then calculates the percent sequence identities for the test sequences relative to the reference sequence, based on the program parameters.<br>
(0239]	A "comparison window", as u.'jed herein, includes reference to a segment of any<br>
one of the number of contiguous positions selected from the group consisting of from 20 to 600, usually about 50 to about 200, more usually about 100 to about 150 in which a sequence may be compared to a reference sequence of the same number of contiguous positions after the two sequences are optimally aligned. Methods of alignment of sequence for comparison are well-knowTi in the art. Optimal aligmnent of sequences for comparison can be conducted, such as by the Jocal homology algorithm of Smith &amp; Waterman, Adv. Appl. Math. 2:482, 1981, by the homology alignment algorithm of Needleman &amp; Wunsch, J. Mol. Biol 48:443, 1970, by the search for similarity method of<br>
 <br>
person &amp; Lipman, Proc. Nat'l. Acad. Sci. USA 85:2444, 1988, by computerized implementations of these algorithms (GAP, BESTFIT, FASTA and TFASTA in the Wisconsin Genetics Software Package, Genetics Computer Group, 575 Science Dr., Madison, WI), or by manual alignment and visual inspection. Other algorithms for determining homology or identity include, for example, in addition to a BLAST program (Basic Local Alignment Search Tool at the National Center for Biological Information), ALIGN, AMAS (Analysis of Multiply Aligned Sequences), AMPS (Protein Multiple Sequence Alignment), ASSET (Aligned Segment Statistical Evaluation Tool), BANDS, BESTSCOR, BIOSCAN (Biological Sequence Comparative Analysis Node), BLIMPS (BLocks IMProved Searcher), FA.STA, Intervals &amp; Points, BMB, CLUSTAL V, CLUSTAL W, CONSENSUS, LCONSENSUS, WCONSENSUS, Smith-Watennan algorithm, DARWIN, Las Vegas algorithm, FNAT (Forced Nucleotide Alignment Tool), Framealign, Framesearch DYNAMIC, FILTER, FSAP (Fristensky Sequence Analysis Package), GAP (Global Alignment Program), GBNAL, GIBBS, GenQuest, ISSC (Sensitive Sequence Comparison), LALIGN (Local Sequence Alignment), LCP (Local Content Program), AdACAW (Multiple Alignment Construction &amp; Analysis Workbench), MAP (Multiple Alignment Program), MBLKP, MBLKN, PIMA (Pattern-induced Multi-sequence Alignment), SAGA (Sequence Aligmnent by Genetic Algorithm) and WHAT-IF. Such alignment programs can also be used to screen genome databases to identify polynucleotide sequences having substantially identical sequences. A number of genome databases are available, for example, a substantial portion of the human genome is available as part of the Human Genome Sequencing Project (Gibbs, 1995). At least twen-one other genomes have already been sequenced, including, for example, M, genitalium (Fraser el al. Science 270:397-403 (1995)), M. jannaschii (Bult et al. Science 23:1058-73 (1996)), H. infiuenzae (Fleischmann et at. Science 2(5P:496-512 (1995)), E. coll {Blattner er al, Science 277:H53-74 (1997)) and yeast (S. cerevisiae) (Mewes ef al. Nature 387:1-65 (1997)) and D. melanogaster (Adams et al. Science 287:2185-95 (2000)). Significant progress has also been made in sequencing the genomes of model organism, such as mouse, C. elegans iind Arabadopsis sp. Several databases containing genomic information annotated with some functional information are maintained by different organizations and may be accessible via the internet.<br>
 <br>
J0240]	In some embodiments, BLAST and BLAST 2.0 algorithms are used, which are<br>
described in Altschul et al, Nuc. Acids Res, 25:3389-3402, 1977 and Altschul ei al.., J. Mol. Biol 215:403-410, 1990, respectively. Software for performing BLAST analyses is publicly available through the National Center for Biotechnology Information. This algorithm involves first identifying high scoring sequence pairs (HSPs) by identifying short words of length W in the query sequence, which either match or satisfy some positive-valued threshold score T when aligned with a word of the same length in a database sequence. T is referred to as the neighborhood word score threshold (Altschul et al, supra). These initial neighborhood word hits act as seeds for initiating searches to fmd longer HSPs containing them. The word hits are extended in both directions along each sequence for as far as the cumulative alignment score can be increased. Cumulative scores are calculated using, for nucleoiide sequences, the parameters M (reward score for a pair of matching residues; aJways &gt;0). For amino acid sequences, a scoring matrix is used to calculate the cumulative score. Extension of the word hits in each direction are halted when: the cimiulative alignment score falls off by the qimntify X from its maximum achieved value; the cumulative score goes to zero or below, due to the accumulation of one or more negative-scoring residue alignments; or the end of either sequence is reached. The BLAST algorithm parameters W, T and X determine the sensitivity and speed of the alignment. The BLASTN program (for nucleotide sequences) uses as defaults a wordlength (W) of 11, an expectation (E) of 10, M=5, N=-4 and a comparison of both strands. For amino acid sequences, the BLAST? program uses as defaults a wordlength of 3 and expectations (E) of 10 and the BLOSUM62 scoring matrix (see Henikoff &amp; Henikoff, Proc, Natl. Acad. Sci. USA 89:10915, 1989) alignments (B) of 50, expectation (E) of 10, M=5, N= -4 and a comparison of both strands.<br>
[0241J	The BLAST algorithm also performs a statistical analysis of the similarity<br>
between two sequences (see Karlin &amp; Altschul, Proc. Natl. Acad. Sci. USA 90:5S73, 1993). One measure of similarity provided by BLAST algorithm is the smallest sum probability (P(N)), which provides an indication of the probability by which a match between two nucleotide or amino acid sequences would occur by chance. For example, a nucleic acid is considered similar io a reference sequence if the smallest sum probability in a comparison of the test nucleic acid to the reference nucleic acid is less than about 0.2,<br>
 <br>
n some embodiments less than aboui 0.01 and in other embodiments less than about<br>
0.001.<br>
[0242]	In  some  embodiments,  protein  and  nucleic  acid  sequence  homologies  are<br>
evaluated using the Basic Local Alignment Search Tool ("BLAST") In particular, five specific BLAST programs are used to perform the following task:<br>
(1)	BLASTP and BLASTS compare an amino acid query sequence against a protein sequence database;<br>
(2)	BLASTN compares a nucleotide query sequence against a nucleotide sequence database;<br>
(3)	BLASTX compares the six-frame conceptual translation products of a query nucleotide sequence {both strands) against a protein sequence database;<br>
(4)	TBLASTN compares  a query  protein sequence  against a nucleotide sequence database translated in all six reading frames (both strands); and<br>
(5)	TBLASTX compares the six-frame translations of a nucleotide query sequence against the six-frame translations of a nucleotide sequence database.<br>
[0243]	The BLAST programs identify homologous sequences by identifying similar<br>
segments, which are referred to herein as "high-scoring segment pairs,' between a query amino or nucleic acid sequence and a test sequence which is, in some embodiments, obtained from a protein or nucleic acid sequence database. High-scoring segment pairs are, in some embodiments, identified (/.e., aligned) by nieans of a scoring matrix, many of which are known in the art. In some embodiments, the scoring matrix used is the BLOSUM62 matrix (Gonnet (1992) Science 256:1443-1445; Henikoff and Henikoff (1993) Proteins 17:49-61). Less In some embodiments, the PAM or PAM250 matrices may also be used (see Schwartz and Dayhoff, eds., 1978, Matrices for Detecting Distance Relationships: Atlas of Protein Sequence and Structure, Washington: National Biomedical Research Foundation). BLAST programs are accessible through the U.S. National Library of Medicine.<br>
[0244]	The parameters used with the above algorithms may be adapted depending on the<br>
sequence length and degree of homology studied. In some embodiments, the parameters may be the default parameters used by the algorithms in the absence of instructions from the user.<br>
 <br>
Computer systems and computer program products<br>
[0245]	The   invention   provides   computers,   computer   systems,   computer   readable<br>
mediums, computer programs products and the like recorded or stored thereon the nucleic acid and polypeptide sequences in Eiccordance with the invention. Additionally, in practicing the methods in accordanC' with the invention, such as to determine and identify sequence identities (to determine whether a nucleic acid is within the scope in accordance with the invention), structural homologies, motiis and the like in silico, a nucleic acid or polypeptide sequence in accordance with the invention can be stored, recorded, and manipulated on any medium which can be read and accessed by a computer.<br>
[0246]	As used herein, the words "recorded" and "stored" refer to a process for storing<br>
information on a computer medium. A skilled artisan can readily adopt any known methods for recording information on a computer readable medium to generate manufactures comprising one or more of the nucleic acid and/or polypeptide sequences in accordance with the invention. As used herein, the terms "computer," "computer program" and "processor" are used in their broadest general contexts and incorporate aJl such devices, as described in detail, below. A "coding sequence of" or a "sequence encodes" a particular polypeptide or protein, is a nucleic acid sequence which is transcribed and translated into a polypeptide or protein when placed under the control of appropriate regulatory sequences.<br>
[0247]	The   polypeptides   in   accordance   with  the  invention   include   sequences   in<br>
accordance with the invention and sequences substantially identical thereto, and subsequences and enzymatically active fragments of any of the preceding sequences. In some embodiments, substantially identical, or homologous, polypeptide sequences refer to a polypeptide sequence having at least 50%, 51%, 52%, 53%, 54%, 55%, 56%, 57%, 58%, 59%, 60%, 61%, 62%, 63%, 64%, 65%, 66%, 67%, 68%, 69%, 70%, 71%, 72%, 73%, 74%, 75%, 76%, 77%, 78%, 79%, 80%, 81%, 82%, 83%, 84%, 85%, 86%, S7%, 88%, 89%, 90%, 91%, 92%, 93%, 94%, 95%, 96%, 97%, 98%, 99%, or more, or complete (100%)) sequence identity (homology) to sequence in accordance with the invention.<br>
[0248]	Homology (sequence identity) may be determmed using any of the computer<br>
programs and parameters described herein.   A nucleic acid or polypeptide sequence in<br>
 <br>
accordance with the invention can be stored, recorded and manipulated on y medium which can be read and accessed by a computer. As used herein, the words "'recorded" and "stored" refer to a process for storing mformation on a computer medium. A skilled artisan can readily adopt any of theipiesently known methods for recording information on a computer readable medium to generate manufactures comprising one or more of the nucleic acid sequences in accordance mth the invention, one or more of the polypeptide sequences in accordance with the invention. Another embodiment of the mvention is a computer readable medium having recorded thereon at least 2, 5, 10, 15, or 20 or mors nucleic acid or polypeptide sequences in accordance with the invention.<br>
[0249]	Another embodiment of the invention is a computer readable medium having<br>
recorded thereon one or more of the nucleic acid sequences in accordance with the invention. Another embodiment of the invention is a computer readable medium having recorded thereon one or more of the polypeptide sequences in accordance with the invention. Another embodiment of the invention is a computer readable medium having recorded thereon at least 2, 5, 10, 15, or 20 or more of the nucleic acid or polypeptide sequences as set forth above.<br>
[0250]	Computer readable media include magnetically readable media, optically readable<br>
media, electronically readable media and magnetic/optical media. For example, the computer readable media may be a hard disk, a floppy disk, a magnetic tape, CD-ROM, Digital Versatile Disk (DVD), Random Access Memory (RAM), or Read Only Memory (ROM) as well as other types of other media known to those skilled in the art.<br>
[0251]	Some embodiments of the invention include systems (such as internet based<br>
systems), such as computer systems wliich store and manipulate the sequence information described herein. One example of a computer system 100 is illustrated in block diagram form in Figure 9. As used herein, "a computer system" refers to the hardware components, software components and data storage components used to analyze a nucleotide sequence of a nucleic acid sequence in accordance with the invention, or a polypeptide sequence in accordance with the invention. In some embodiments, the computer system 100 includes a processor for processing, accessing and manipulating the sequence data. The processor 105 can be any well-known t&gt;'pe of central processing unit, such as, for example, the Pentium III from Intel Corporation, or similar processor from Sun, Motorola, Compaq, AMD or International Business Machines.<br>
 <br>
[0252]	In some embodiments, the computer system 100 is a generaj purpose system that<br>
comprises the processor 105 and one or more internal data storage components 110 for storing data and one or mors data retrieving devices for retrieving the data stored on the data storage components. A skilled artisan can readily appreciate that any one of the currently available computer systems are suitable.<br>
[0253]	In one embodiment, the computer system 100 includes a processor 105 connected<br>
to a bus which is connected to a main memory 115 (in one embodiment implemented as RAM) and one or more internal data storage devices 110, such as a hard drive and/or other computer readable media having data recorded thereon. In some embodiments, the computer system 100 further includes one or more data retrieving device 118 for reading the data stored on the internal data storage devices 110.<br>
[0254]	The data retrieving device 118 may represent, for example, a floppy disk drive, a<br>
compact disk drive a magnetic tape drive, or a modem capable of connection to a remote data storage system (such as via the internet) etc. In some embodiments, the internal data storage device 110 is a removable computer readable medium such as a floppy disk, a compact disk, a magnetic tape, etc. containing control logic and/or data recorded thereon. The computer system 100 may advantageously include or be. programmed by appropriate software for reading the control logic and/or the data from the data storage component once inserted in the data retrieving device.<br>
[0255]	The computer system 100 includes a display 120 which is used to display output<br>
to a computer user. It should also be noted that the computer system 100 can be linked to other computer systems 125a-c in a network or wide area network to provide centralized access to the computer system 100.<br>
[0256]	Software for accessing and processing the nucleotide sequences of a nucleic acid<br>
sequence in accordance with the invention and sequences substantially identical thereto, or a polypeptide sequence in accordance with the invention and sequences substantially identical thereto, (such as search tools, compare tools and modeling tools etc.) may reside in main memory 115 during execution.<br>
[0257]	In some embodiments, the computer system 100 may further comprise a sequence<br>
comparison algorithm for comparing a nucleic acid sequence in accordance with the invention and sequences substantiallj' identical thereto, or a polypeptide sequence in accordance with the invention and sequences substantially identical thereto, stored on a<br>
 <br>
computer readable medium to a reference nucleotide or polypeptide sequence(s) stored on a computer readable medium. A "sequence comparison algorithm" refers to one or more programs which are implemented (locally or remotely) on the computer system 100 to compare a nucleotide sequence with other nucleotide sequences and/or compounds stored within a data storage means. For example, the sequence comparison algorithm may compare the nucleotide sequences of a nucleic acid sequence in accordance with the invention and sequences substantiall)' identical thereto, or a polypeptide sequence in accordance with the invention and sequences substantially identical thereto, stored on a computer readable medium to reference sequences stored on a computer readable medium to identify homologies or structural motifs.<br>
[0258]	Figure 10 is a flow diagram illustrating one embodiment of a process 200 for<br>
comparing a new nucleotide or protein sequence with a database of sequences in order to determine the homology levels between the new sequence and the sequences in the database. The database of sequences can be a private database stored within the computer system 100, or a public database such as GENBANK that is available through the Internet.<br>
[0259]	The process 200 begins at a start state 201 and then moves to a state 202 wherein<br>
the new sequence to be compared is stored to a memory in a computer system 100. As discussed above, the memory could be any type of memory, including RAM or an interna! storage device.<br>
[0260]	The process 200 then moves to a state 204 wherein a database of sequences is<br>
opened for analysis and comparison. The process 200 then moves to a state 206 wherein the first sequence stored in the database is read into a memory on the computer. A comparison is then performed at a statu 210 to determine if the first sequence is the same as the second sequence. It is important to note that this step is not limited to performing an exact comparison between the new sequence and the first sequence in the database. Well-known methods are known to those of skill in the art for comparing two nucleotide or protein sequences, even if they are not identical. For example, gaps can be introduced into one sequence in order to raise the homology level between the two tested sequences. The parameters that control whether gaps or other features are introduced into a sequence during comparison are normally entered by the user of the computer system.<br>
 <br>
[0261]	Once a comparison of the tv/o sequences has been performed at the state 210, a<br>
determination is made at a decision state 210 whether the two sequences are the same. Of course, the term "same" is not limited to sequences that are absolute]}' identical. Sequences that are within the homology parameters entered by the user will be marked as "same" in the process 200.<br>
I0262J	If a determination is made thai the two sequences are the same, the process 200<br>
moves to a state 214 wherein the namti of the sequence irom the database is displayed to the user. This state notifies the user tliat the sequence with the displayed name fulfills the homology constraints that were entered. Once the name of the stored sequence is displayed to the user, the process 200 moves to a decision state 218 wherein a determination is made whether more sequences exist in the database. If no more sequences exist in the database, then the process 200 terminates at an end state 220. However, if more sequences do exist in the database, then the process 200 moves to a state 224 wherein a pointer is moved to the next sequence in the database so that it can be compared to the new sequence. In this manner, the new sequence is aligned and compared with every sequence in the database.<br>
[0263]	It should be noted that if a determination had been made at the decision state 212<br>
that the sequences were not homologous, then the process 200 would move immediately to the decision state 218 in order to determine if any other sequences were available in the database for comparison.<br>
[0264]	Accordingly, one embodiment of the invention is a computer system comprising a<br>
processor, a data storage device having stored thereon a nucleic acid sequence in accordance with the invention and sequences substantially identical thereto, or a polypeptide sequence in accordance with the invention and sequences substantially identical thereto, a data storage device having retrievably stored thereon reference nucleotide sequences or polypeptide sequences to be compared to a nucleic acid sequence in accordance with the invention and sequences substantially identical thereto, or a polypeptide sequence in accordance with the invention and sequences substantially identical thereto and a sequence comparer for conducting the comparison. The sequence comparer may indicate a homology' level between the sequences compared or identify structural motifs in die above described nucleic acid code a nucleic acid sequence in accordance  with the  invention  and  sequences  substantially  identical thereto,  or a<br>
 <br>
polypeptide sequence in accordance with the invention and sequences substantially identical thereto, or it may identify structural motifs in sequences which are compared to these nucleic acid codes and polypeptide codes. In some embodiments, the data storage device may have stored thereon the sequences of at least 2, 5, iO, 15, 20, 25, 30 or 40 or more of the nucleic acid sequences in accordance with the invention and sequences substantially identical thereto, or the polypeptide sequences in accordance with the invention and sequences substantially identical thereto.<br>
[0265]	Another embodiment of the invention is a method for determining the level of<br>
homology between a nucleic acid sequence in accordance with the invention and sequences substantially identical thereto, or a polypeptide sequence in accordance with the invention and sequences substantially identical thereto and a reference nucleotide sequence. The method including reading the nucleic acid code or the polypeptide code and the reference nucleotide or polypeptide sequence through the use of a computer program which determines homologj' levels and determining homology between the nucleic acid code or polypeptide code and the reference nucleotide or polypeptide sequence with the computer program. The computer program may be any of a nnber of computer programs for determining homolog}' levels, including those specificaJlj' enumerated herein, (such as BLAST2N with the default parameters or with any modified parameters). The method may be implemented using the computer systems described above. The method may also be perfoimed by reading at least 2, 5, 10, 15, 20, 25, 30 or 40 or more of the above described nucleic acid sequences in accordance with the invention and sequences substantially identical thereto, or the pol&gt;'peptide sequences in accordance with the invention and sequences substantially identical thereto through use of the computer program and determining homology between the nucleic acid codes or polypeptide codes and reference nuclectide sequences or polypeptide sequences.<br>
[0266]	Figure 11 is a flow diagram illustrating one embodiment of a process 250 in a<br>
computer for determining whether two sequences are homologous. The process 250 begins at a start state 252 and then moves to a state 254 wherein a first sequence to be compared is stored to a memory. The second sequence to be compared is then stored to a memory at a state 256. The process 250 then moves to a state 260 wherein the first character in the first sequence is read and then to a state 262 wherem the first character of the second sequence is read.  It should, be understood that if the sequence is a nucleotide<br>
 <br>
sequence, then the character would noimally be either A, T, C, G or U. If the sequence is a protein sequence, then it is, in some embodiments, in the single Setter amino acid code so that the first and sequence sequencer can be easily compared,<br>
[0267]	A determination is then made at a decision state 264 whether the two characters<br>
are the same. If they are the same, tliea the process 250 moves to a state 268 wherein the next characters in the first and second sequences are read. A determination is then made whether the next characters are the same. If they are, then the process 250 continues this loop until two characters are not the; same. If a determination is made that the next two characters are not the same, the process 250 moves to a decision state 274 to detennine whether there are any more characters either sequence to read.<br>
[0268]	If there are not any more characters to read, then the process 250 moves to a state<br>
276 wherein the level of homology between the fu-st and second sequences is displayed to the user. The level of homology is determined by calculating the proportion of characters between the sequences that v'ere the same out of the total number of sequences in the first sequence. Thus, if every character in a, first 100 nucleotide sequence aligned with a every character in a second sequence, the homology level would he 100%.<br>
10269]	Allemativei}', the computer program may be a computer program -s'hich compares<br>
the nucleotide sequences of a nucleic acid sequence as set forth in the invention, to one or more reference nucleotide sequences in order to determine whether the nucleic acid code in accordance with the invention and sequences substantially identical thereto, differs from a reference nucleic acid sequence at one or more positions. Optionally such a program records the length and identity of inserted, deleted or substituted nucleotides with respect to the sequence of either the reference polynucleotide or a nucleic acid sequence in accordance with the invention and sequences substantially identical thereto. In some embodiments, the computer program may be a program which determines whether a nucleic acid sequence in accordance with the invention and sequences substantially identical thereto, contains a single nucleotide polymorphism (SNP) with respect to a reference nucleotide sequence.<br>
[0270]	Accordingly, another embodunent of the invention is a method for determining<br>
whether a nucleic acid sequence in accordance with the invention and sequences substantially identical thereto, differs at one or more nucleotides from a reference nucleotide sequence comprising the steps of reading the nucleic acid code and the<br>
1  i\'-7<br>
 <br>
reference nucleotide sequence through use of a computer program which identifies differences between nucleic acid sequences and identifying differences between the nucleic acid code and the reference nucleotide sequence with the computer program. In some embodiments, the computer program is a program which identifies single nucleotide polymorphisms. The method may be implemented by the computer systems described above and the method illustrated in Figure 11. The method may also be performed by reading at least 2, 5, 10, 15, 20, 25, 30, or 40 or more of the nucleic acid sequences in accordance with the invention and sequences substantially identical thereto and the reference nucleotide sequences through the use of the computer program and identifying differences between the nucleic acid codes and the reference nucleotide sequences with the computer program.<br>
In other embodiments, the computer based system may further comprise an identifier for identifying features within a nucleic acid sequence in accordance with the invention or a polypeptide sequence in accwdance with the invention and sequences substantially identical thereto. An "identifier" refers to one or more programs which identifies certain features within a nucleic acid sequence in accordance with the invention, or a polypeptide sequence in accordance with the invention. In some embodiments, the identifier may comprise a program which identifies an open reading frame in a nucleic acid sequence in accordance with the invention and sequences substantially identical thereto.<br>
Figure 12 is a flow diagram illustrating one embodiment of an identifier process 300 for detecting the presence of a feature in a sequence. The process 300 begins at a start state 302 and then moves to a state 304 wherein a first sequence that is to be checked for features is stored to a memoiy 115 in the computer system 100. The process 300 then . moves to a state 306 wherein a database of sequence features is opened. Such a database would include a list of each feature's attributes along with the name of the feature. For example, a feature name could be "Initiation Codon" and the attribute would be "ATG". Another example would be the feature name "TAATAA Box" and the feature attribute would be "TAATAA". An example of such a database is produced by the University of Wisconsin Genetics Computer Group. Alternatively, the features may be structural polypeptide motifs such as alpha helices, beta sheets, or functional polypeptide motifs<br>
 <br>
such as enzymatic active sites, helix-tum-helix motifs or other motifs known to those skilled in the art.<br>
[0273]	Once the database of features i5; opened at the state 306, the process 300 moves to<br>
a state 308 wherein the first feature is read from the database. A comparison of the attribute of the first feature with the first sequence is then made at a state 310. A determination is then made at a decision state 316 whether the attribute of the feature was found in the first sequence. If the attiibute was found, then the process 300 moves to a state 318 wherein the name of the found feature is displayed to the user.<br>
[0274]	The process 300 then moves to a decision state 320 wherein a determination is<br>
made whether move features exist in the database. If no more features do exist, then the process 300 terminates at an end state 324. However, if more features do exist in the database, then the process 300 reads the next sequence feature at a state 326 and loops back to the state 310 wherein the attribute of the next feature is compared against the first sequence. It should be noted, that if the feature attribute is not found in the first sequence. at the decision state 316, the process 300 moves directly to the decision state 320 in order to determine if an}' more features exist in the database.<br>
[0275]	Accordingly, another embodiment of the invention is a method of identifying a<br>
feature within a nucleic acid sequence in accordance with the invention and sequences substantially identical thereto, or a polypeptide sequence in accordance with the invention and sequences substantially identical tliereto, comprising reading the nucleic acid code{s) or polypeptide code(s) through the use of a computer program which identifies features therein and identifying features within the nucleic acid code(s) with the computer program. In some embodiments, the computer program comprises a computer program which identifies open reading frames. The method may be performed by reading a single sequence or at least 2, 5,10,15, 20, 25, 30, or 40 or more of the nucleic acid sequences in accordance with the invention and sequences substantially identical thereto, or the polypeptide sequences in accordance wilh the invention and sequences substantially identical thereto, through the use of the computer program and identifying features within the nucleic acid codes or polypeptide codes with the computer program.<br>
{0276]	A  nucleic  acid  sequence  in  accordance  with the  invention and  sequences<br>
substantially identical thereto, or a polypeptide sequence in accordance with the invention and sequences substantially identical thereto, may be stored and manipulated in a variety<br>
 <br>
of data processor programs in a variety of fomiats. For example, a nucleic acid sequence<br>
in accordance with the invention and sequences substantially identical thereto, or a<br>
polypeptide sequence in accordance with the invention and sequences substantialiy<br>
identical thereto, may be stored as text in a word processing file, such as Microsoft<br>
WORD™ or WORDPERFECT or m an ASCII file in a varietj- of database programs<br>
familiar to those of skill in tiie art, such as DB2TM, SYBASET", or ORACLE™. In<br>
addition, many computer programs and databases may be used as sequence comparison<br>
algorithms, identifiers, or sources of reference nucleotide sequences or polypeptide<br>
sequences to be compared to a nucleic acid sequence in accordance with the invention<br>
and sequences substantially identical thereto, or a polypeptide sequence in accordance<br>
with the invention and sequences sutistantially identical thereto. The following list is<br>
intended not to limit the invention but to provide guidance to programs and databases<br>
which are useful with the nucleic acid sequences in accordance with the invention and<br>
sequences substantially identical thereto, or the polypeptide sequences in accordance with<br>
the invention and sequences substantially identical thereto.<br>
[0277]	The programs and databases which may be used include, but are not limited to:<br>
MACPATTERN™ (EMBL), DISCOVERYBASETM (Molecular Applications Group), GENEMINETM (Molecular Applications Group), LOOKT" (Molecular Applications Group), MACLOOK™ (Molecular Applications Group), BLAST and BLAST2 (NCBI), BLASTT
 <br>
Report data base, the Comprehensi\'e Medicinal Chemistiy database, Derwents's World<br>
Drug Index database, the BioByleMe,sterFile database, the Genbank database and the<br>
Genseqn database. Many other programs and daia bases would be apparent to one of skill<br>
in the art given the present disclosure.<br>
[0278]	Motifs which may be detected using the above programs include sequences<br>
encoding leucine zippers, helix-tum-helix motifs, glycosylation sites, ubiquitination sites, alpha helices and beta sheets, signal sequences encoding signal peptides which direct the secretion of the encoded proteins, sequences Implicated in transcription regulation such as homeoboxes, acidic stretches, enzymatic active sites, substrate binding sites and enzymatic cleavage sites.<br>
Hybridization of nucleic acids<br>
[0279]	The invention provides isolated, synthetic or recombinant nucleic acids that<br>
hybridize under stringent conditions to a sequence in accordance with the invention (such as SEQ ID N0:1, SEQ ID N0:3, SEQ ED N0:5, SEQ ID N0:7, SEQ ID N0:9, SEQ ID NO.-n, SEQ ID N0:I3, SEQ ID N0;I5, SEQ ID NO:I7, SEQ ID NO:]9, SEQ ID N0:21, SEQ ID NO:23, SEQ ID NO:25, SEQ ID NO:27, SEQ ID NO:29, SEQ ID N0:31, SEQ ID NO:33, SEQ ID NO:35, SEQ ID NO:37, SEQ ID NO;39, SEQ ID N0:41, SEQ ID NO:43, SEQ ID NO:45, SEQ ID NO:47, SEQ ID NO:49, SEQ ID N0:5U SEQ ID NO:53, SEQ ID NO:55, SEQ ID NO:57, SEQ ID NO:59, SEQ ID N0:61, SEQ ID NO:63, SEQ ID NO:65, SEQ ID NO;67, SEQ ID NO:69, SEQ ID N0:71, SEQ ED NO:73, SEQ ID NO:75, SEQ ID NO:77, SEQ ID NO:79, SEQ ID N0:81, SEQ ID NO:83, SEQ ID NO:85, SEQ ID NO:87, SEQ ID NO:89, SEQ ID N0:9I, SEQ ID NO:93, SEQ ID NO:95, SEQ ID NO:97, SEQ ID NO:99, SEQ ID NO:101, SEQ ID NO:103, SEQ ED NO:105, SEQ ID NO:107, SEQ ID NO:109, SEQ ID NO:ni, SEQ IDN0:113, SEQ ID N0:n5, SEQ IDN0:I17, SEQ ID N0:119, SEQ ID N0:12I, SEQ ID NO:123, SEQ ID NO:125, SEQ ID NO:127, SEQ ID NO:I29, SEQ ID N0:131, SEQ ID NO:I33, SEQ ID NO:135, SEQ ID NO:137, SEQ ID NO:139, SEQ ID N0:141, SEQ ID NO:143, SEQ ID NO:145, SEQ ID NO:147, SEQ ID NO:149, SEQ ID N0:15!, SEQ ID NO:i53, SEQ ID NO:155, SEQ ID NO:157, SEQ ID NO:159, SEQ ID N0:161, SEQ ID NO:163, SEQ ID NO:165, SEQ ID NO;167, SEQ ID NO:169, SEQ ID N0:171, SEQ ID NO:173, SEQ ID NO:175, SEQ ID NO:177, SEQ ID NO:179, SEQ ID N0:181, SEQ ID NO:183, SEQ ID NO:]85, SEQ ID NO:187, SEQ ID NO:189, SEQ ID<br>
 <br>
N0:i9i, SEQ ID NO:I93, SEQ ID NO:I95, SEQ ID NO:197, SEQ ID NO:I99, SEQ ID<br>
NO:201, SEQ ID NO:203, SEQ ID NO:205, SEQ ID NO:207, SEQ ID NO:209. SEQ ID<br>
N0:211, SEQ ID NO:213, SEQ ID NO:215, SEQ ID NO:217, SEQ ID NO:219, SEQ ID<br>
NO:221, SEQ ID NO:223, SEQ ID NO:225, SEQ ID NO:227, SEQ ID NO:229, SEQ ID<br>
NO:231, SEQ ID NO:233, SEQ ID NO:235, SEQ ID NO:237, SEQ ID NO:239, SEQ ID<br>
NO:241, SEQ ID NO:243, SEQ ID NO:245, SEQ ID NO:247, SEQ ID NO:249, SEQ ID<br>
NO:251, SEQ ID NO:253, SEQ ID NO:255, SEQ ID NO:257, SEQ ID NO:259, SEQ E)<br>
NO:26l, SEQ ID NO:263, SEQ ID NO:265, SEQ ID NO:267, SEQ ID NO:269, SEQ ID<br>
NO:27I, SEQ ID NO:273, SEQ ID NO:275, SEQ ID NO:277, SEQ ID NO:279, SEQ ID<br>
NO:28l, SEQ ID NO:283, SEQ ID NO:285, SEQ ID NO:287, SEQ ID NO:289, SEQ ID<br>
NO:291, SEQ ID NO:293, SEQ ID NO:295, SEQ ID NO:297, SEQ ID NO:299, SEQ ID<br>
NO:301, SEQ ID NO:303, SEQ ID NO:305, SEQ ID NO:307, SEQ ID NO:309, SEQ ID<br>
N0:311, SEQ ID N0:313, SEQ ID NO:315, SEQ ID NO:317, SEQ ID NO:319, SEQ ID<br>
NO:32U SEQ ID NO:323, SEQ ID NO:325, SEQ ID NO:327, SEQ ID NO:329, SEQ ID<br>
NO:33U SEQ ID NO:333, SEQ ID NO:335, SEQ ID NO:336, SEQ ID NO:337, or SEQ<br>
ID NO:338. The stringent conditiC'OS can be highly stringent conditions, medium<br>
stringent conditions and/or low stringent conditions, including the high and reduced<br>
stringency conditions described herein. In some embodiments, it is the stringency of the<br>
wash conditions that set forth the conditions which determine whether a nucleic acid is<br>
within the scope in accordance with the invention, as discussed below.<br>
[0280]	"Hybridization" refers to the process by which a ntfcleic acid strand joins with a<br>
complementary strand through base pairing. Hybridization reactions can be sensitive and selective so that a particular sequence of interest can be identified even in samples in which it is present at low concentrations. Suitably stringent conditions can be defined by, for example, the concentrations of salt or formamide in the prehybridization and hybridization solutions, or by the hybridization temperature and are well known in the art. In other embodiments, stringency can be increased by reducing the concentration of salt, increasing the concentration of formamide, or raising the hybridization temperature. In other embodiments, nucleic acids in accordance with the invention are defined by their ability to hybridize under various stringency conditions (such as high, medium, and low), as set forth herein.<br>
 <br>
[0281]	In some embodiments, hybridization under high stringency conditions comprise<br>
about 50% formamide at about 3TC to 42°C. In some embodiment, hybridization conditions comprise reduced stringency conditions in about 35% to 25% formamide at about 30°C to 35°C. In some embodiments, hybridization conditions comprise high stringency conditions, such as at 42C in 50% formamide, 5X SSPE, 0.3% SDS and 200 |ig/mi sheared and denatured salmon sperm DNA. In some embodiments, hybridization conditions comprise these reduced stringency conditions, but in 35% formamide at a reduced temperature of 35°C. The temperature range corresponding to a particular level of stringency can be further narrowed by calculating the purine to pyrimidine ratio of the nucleic acid of interest and adjusting the temperature accordingly. Variations on the above ranges and conditions are well known in the art.<br>
[0282]	In other embodiments, nucleic acids in accordance with the invention as defined<br>
by their ability to hybridize under stringent conditions can be between about five residues and the full length of nucleic acid in accordance with the invention; such as they can be at least 5, 10, 15, 20, 25, 30, 35, 40, 50, .55, 60, 65, 70, 75, 80, 90, 100, 150, 200, 250, 300, 350, 400, 450, 500, 550, 600, 650, 700, 750, BOO, 850, 900, 950, 1000, or more, residues in length. Nucleic acids shorter than full length are also included. These nucleic acids can be useful as, such as hybridization probes, labeling probes, PCR oligonucleotide probes, siRNA or miRKA (single or double stranded), antisense or sequences encoding antibody binding peptides (epitopes), motifs, active sites and the like.<br>
[0283]	In some embodiments, nucleic acids in accordance with the invention are defined<br>
by their ability to hybridize under high stringency comprises conditions of about 50% formamide at about 37°C to 42°C. In some embodiments, nucleic acids in accordance with the invention are defined by their ability to hybridize under reduced stringency comprising conditions in about 35% to 25% formamide at about 30°C to 35°C.<br>
[0284]	Altematively, nucleic acids in accordance with the invention are defmed by their<br>
ability to hybridize under high stringency comprising conditions at 42°C in 50% formamide, 5X SSPE, 0.3% SDS, and a repetitive sequence blocking nucleic acid, such as cot-1 or salmon sperm DNA (such as 200 ug/ml sheared and denatured salmon sperm DNA). In some embodiments, nucleic acids in accordance with the invention are defined by their ability to hybridize under reduced stringency conditions comprising 35% or 40% formamide at a reduced temperature of 35°C or 42'C.<br>
 <br>
[0285]	In nucleic acid hybridiTation reactions, the conditions used to achieve a particular<br>
level of stringency will vary, depending on the nature of the nucleic acids being hybridized. For exainple, the length, degree of complementarity, nucleotide sequence composition (such as GC v. AT content) and nucleic acid type (such as RNA v. DNA) of the hybridizing regions of the nucleic acids can be considered in selecting hybridization conditions. An cidditional consideration is whether one of the nucleic acids is immobilized, for example, on a filter,<br>
[0286]	Hybridization may be carried out under conditions of low stringency, moderate<br>
stringency or high stringency. As jin example of nucleic acid hybridization, a polymer membrane containing immobilized denatured nucleic acids is first prehybridized for 30 minutes at 45'C in a solution consisting of 0.9 M NaCl, 50 mM NaH2P04, pH 7.0, 5.0 mM NaiEDTA, 0.5% SDS, lOX Denhardt's and 0.5 mg/mJ polyriboadenylic acid. Approximately 2X10 cpm {specific activity 4-9 X 10 cpm/fxg) of "'P end-labeled oligonucleotide probe are then added to the solution. After 12-16 hours of incubation, the membrane is washed for 30 minutes at room temperature in IX SET (150 mM NaCl, 20 mM Tris hydrochloride, pH 7.8, ! mM Na2EDTA) containing 0.5% SDS, followed by a 30 minute wash in fresh JX SET at Tm-ICC for the oligonucleotide probe. The membrane is then exposed to auto-radiographic film for detection of hybridization signals. All of the foregoing hybridizations would be considered to be under conditions of high stringency.<br>
[0287]	Following hybridization, a filter can be washed to remove any non-specifically<br>
bound detectable probe. The stringency tised to wash the filters can also be varied depending on the nature of the nucleic acids bemg hybridized, the length of the nucleic acids being hybridized, the degree of complementarity, the nucleotide sequence composition (such as GC v. AT content) and the nucleic acid type (such as RNA v. DNA). Examples of progressively higher stringency condition washes are as follows: 2X SSC, 0.1% SDS at room temperature for 15 minutes (low stringency); O.IX SSC, 0.5% SDS at room temperature for 30 minutes to 1 hour (moderate stringency); O.IX SSC, 0.5% SDS for 15 to 30 minutes at between the hybridization temperaUire and 68°C (high stringency); and 0.I5M NaCI for 15 minutes at 72'C (very high stringency). A fmal low stringency wash can be conducted in O.IX SSC at room temperature. The examples above are merely illustrative of one set of conditions that can be used to wash filters. One<br>
 <br>
of skil] in the an would know that tiiere are numerous recipes for different stringency washes. Some other examples are given below.<br>
[0288]	In some embodiments, hybridization conditions comprise a wash step comprising<br>
a wash for 30 minutes at room temperature in a solution comprising IX 150 mM NaCl, 20 mM Tris hydrochloride, pH 7.8, 1 mM Na2EDTA, 0.5% SDS, followed by a 30 minute wash in fresh solution.<br>
[0289]	Nucleic   acids   which   have   hybridized   to   the   probe   are   identified   by<br>
autoradiography or other conventionaJ techniques.<br>
[0290]	The   above  procedures  may  be  modified to   identify   nucleic  acids  having<br>
decreasing levels of sequence identity (homolog}') to the probe sequence. For example, to obtain nucleic acids of decreasing sequence identity (homology) to the detectable probe, less stringent conditions may be used. For example, the hybridization temperature may be decreased in increments oi' 5°C from 68°C to 42°C in a hybridization buffer having a Na" concentration of approximately IM. Following hybridization, the filter may be washed with 2X SSC, 0-5% SDS at tie temperature of hybridization. These conditions are considered to be "moderate" conditions above 50°C and "low" conditions below 50°C. A specific example of "moderate" hybridization conditions is when the above hybridization is conducted at 55°C. A specific example of "low stringency" hybridization conditions is when the above hybridization is conducted at 45°C.<br>
[0291]	Alternatively, the hybridization may be carried out in buffers, such as 6X SSC,<br>
containing formamide at a temperature of 42°C. In this case, the concentration of formamide in the hybridization buffer may be reduced in 5% increments from 50Vo to 0% to identify clones having decreasing levels of homology to the probe. Following hybridization, the filter may be washed with 6X SSC, 0.5% SDS at 50°C. These conditions are considered to be "moderate" conditions above 25%) formamide and "low" conditions below 25% formamide. A specific example of "moderate" hybridization conditions is when the above hybridization is conducted at 30% formamide. A specific example of "low stringency" hybridization conditions is when the above hybridization is conducted at 10% formamide.<br>
[0292]	However, the selection of a hybridization format may not be critical - it is the<br>
stringency of the wash conditions that set forth the conditions which determine whether a nucleic acid is within the scope in accordance with the invention.  Wash conditions used<br>
 <br>
to identify nucleic acids within the scope in accordance with the invention include, such<br>
as: a salt concentration of about 0.02 molar at pH 7 and a temperature of at least about<br>
SO'C or about 55°C to about 60°C; or, a salt concentration of about 0.15 MNaCl at 72°C<br>
for about 15 minutes; or, a salt conceutration of about 0.2X SSC at a temperature of at<br>
least about 50°C or about 55°C to a,bout 60°C for about 15 to about 20 minutes; or, the<br>
hybridization complex is washed twice with a solution with a salt concentration of about<br>
2X SSC containing 0. ] % SDS at room temperature for 15 minutes and then washed twice<br>
by O.IX SSC containing 0.1% SDS at 6B°C for 15 minutes; or, equivalent conditions. See<br>
Sambrook ed.. MOLECULAR CLONING; A LABORATORY MANUAL (2nd ed.), vols. 1-3,<br>
Cold Spring Harbor Laborator;' (1989), LABORATORY TECHNIQUES IN<br>
BIOCHEMISTRY AND MOLECULAR BIOLOGY: HYBRIDIZATION WITH<br>
NUCLEIC ACID PROBES, Part I. Theory and Nucleic Acid Preparation, Tijssen, ed.<br>
Elsevier, N.Y. (1993) and Ausubel, ed. John Wiley &amp; Sons, hic. New York (1997) for a<br>
description of SSC buffer and equivalent conditions.<br>
[0293]	These methods may be used o isolate or identify nucleic acids in accordance with<br>
the invention and sequences substitially identical thereto. For example, the preceding methods may be used to isolate or identify nucleic acids having a sequence with at least about 50%, 51%, 52%, 53%, 54%, 55%, 56%, 57%, 58%, 59%, 60%, 61%, 62%, 63%, 64%, 65%, 66%, 67%, 68%, 69%, 70%, 71%, 72%, 73%, 74%, 75%, 76%, 77%, 78%. 79%, 80%, 81%, 82%, 83%, 84%, 85%, 86%, 87%, 88%, 89%, 90%, 91%, 92%, 93%, 94%, 95%, 96%, 97%, 98%, 99%, or more sequence identity (homology') to a nucleic acid sequence selected from the group consisting of one of the sequences in accordance with the invention and sequences substantidly identical thereto, or fragments comprising at least about 10, 15, 20, 25, 30, 35, 40, 50, 75, 100, 150, 200, 300, 400, or 500 consecutive bases thereof and the sequences complementary thereto. Sequence identify (homology) may be measured using the alignment algorithm. For example, the homologous polynucleotides may have a coding sequence which is a naturally occurring allelic variant of one of the coding sequences described herein. Such allelic variants may have a substitution, deletion or addition of one or more nucleotides when compared to the nucleic acids in accordance with the invention. Additional)}', the above procedures may be used to isolate nucleic acids which encode polypeptides having at least about 99%, 95%, at least 90%, at least 85%, at least 80%, at least 75%, at least 70%, at least 65%, at<br>
 <br>
least 60%, at least 55%, or at least 50% sequence identity (homology) to apolypeptide in accordance witli the invention, or fragments comprising at least 5, 10, 15, 20, 25, 30, 35. 40, 50, 75, 100, or 150 consecutive amino acids thereof as determined using a sequence alignment algorithm (such as the FSTA version 3.0t78 algorithm with the default parameters).<br>
Oligonucleotides probes and methods for using them<br>
{0294J	The invention also provides nucleic acid probes that can be used, such as for<br>
identifying, amplifying, or isolating nucleic acids encoding a polypeptide having an aldolase, such as pyruvate aldolase, such as HMG and/or KHG aldolase enzyme activity or fragments thereof or for identifying aldolase, such as pyruvate aldolase, such as HMG and/or KHG aldolase enzyme, genes. ![n some embodiments, the probe comprises at least about 10 consecutive bases of a nucleic acid in accordance with the invention. Alternatively, a probe in accordance with the invention can be at least about 5, 6, 7, 8, 9, 10, n, 32, 13, 14, 15, 16, J7,38, 19, 20, 2], 22, 23, 24, 25, 30, 35, 40, 45, 50, 60, 70, SO, 90, 100, 110, 120, 130, 150 or about 10 to 50, about 20 to 60 about 30 to 70, consecutive bases of a sequence as set forth in a nucleic acid in accordance with the invention. The probes identify a nucleic acid by binding and/or hybridization. The probes can be used in arrays in accordance with the invention, see discussion below, including, such as capillary arrays. The probes in accordance with the invention can also be used to isolate other nucleic acids or polypeptides.<br>
[0295]	The isolated, synthetic or rec;ombinant nucleic acids in accordance with the<br>
invention, the sequences complementary thereto, or a fragment comprising at least about 10,15, 20, 25, 30, 35, 40, 50, 75,100,150, 200, 300,400, or 500 consecutive bases of one of the sequences in accordance with the invention, or the sequences complementary thereto may also be used as probes to determine whether a biological sample, such as a soil sample, contains an organism having a nucleic acid sequence in accordance with the invention or an organism from which the nucleic acid was obtained. In such procedures, a biological sample potentially harboring the organism from which the nucleic acid was isolated is obtained and nucleic acids are obtained from the sample. The nucleic acids are contacted with the probe under conditions which permit the probe to specifically hybridize to any coraplementaiy sequeijces from which are present therein.<br>
 <br>
[0296]	Where necessary, conditions which permit the probe to specifically hybridize to<br>
coraplementary sequences may be determined by placing the probe in contact with complementary sequences from samples known to contain the complementary sequence as well as control sequences which do not contain the complementary' sequence. Hybridization conditions, such as the salt concentration of the hybridization buffer, the formamide concentration of the hybridization buffer, or the hybridization temperature, may be varied to identify conditions which allow the probe to hybridize specifically to complementary nucleic acids.<br>
[0297]	If the sample contains the org,anism fi-om which the nucleic acid was isolated,<br>
specific hybridization of the probe is then detected. Hybridization may be detected by labeling the probe with a detectable agent such as a radioactive isotope, a fluorescent dye or an enzyme capable of catalyzing tlie formation of a detectable product.<br>
(0298]	Many  methods   for   using   the   labeled   probes   to   detect   the   presence   of<br>
complementary nucleic acids in a sample are familiar to those skilled in the art. These include Southern Blots, iSIorthem Blots, colony hybridization procedures and dot blots. Protocols for each of these procedures are provided in Ausubel et al. Current Protocols in Molecular Biology, John Wiley &amp; Sons, Inc. (1997) and Sambrook et al. Molecular Cloning: A Laboratory Manual 2nd Ed., Cold Spring Harbor Laboratory Press (1989).<br>
[0299]	Alternatively, more than one probe (at least one of which is capable of specifically<br>
hybridizing to any complementary sequences which are present in the nucleic acid sample), may be used in an amplifi<::ation reaction to determine whether the sample contains an organism containing a nucleic acid sequence in accordance with invention as from which was isolated some embodiments probes comprise oligonucleotides. embodhnents amplification may pcr reaction. jrotocols are described ausubel and sambrook supra. alternatively ligase chain or strand displacement barany f. worid methods applications e. fahy et al replication isothermal transcription-based system alternative method walker g.t. amplification-an vitro dna technique research hi such procedures acids></::ation>
 <br>
sample are contacted with the probes, the amplification reaction is performed and any resulting amplification product is detected. The amplification product may be detected by performing gel electrophoresis on liie reaction products and staining the gel with an intercalator such as ethidium bromide. Alternatively, one or more of the probes may be labeled with a radioactive isotope and the presence of a radioactive amplification product may be detected by autoradiography after gel electrophoresis.<br>
[0300]	Probes derived fi'om sequences near the ends of the sequences in accordance with<br>
the invention, may also be used in clu-omosome walking procedures to identify clones containing genomic sequences located adjacent to the sequences in accordance with the invention. Such methods allow the isolation of genes which encode additional proteins fi'om the host organism.<br>
[0301]	In some embodiments, the isolated, synthetic or recombinant nucleic acids in<br>
accordance with the invention, the sequences complementary thereto, or a figment comprising at least 10, 15, 20, 25, 30, 35, 40, 50, 75, 100, 150, 200, 300, 400, or 500 or more consecutive bases of one of the sequences in accordance with the invention, or the sequences complementary thereto are used as probes to identify and isolate related nucleic acids. In some embodiments, the related nucleic acids may be cDNAs or genomic DNAs irom organisms other than the one from which the nucleic acid was isolated. For example, the other organisms may be related organisms. In such procedures, a nucleic acid sample is contacted with the probe xmder conditions which permit the probe to specifically hybridize to related sequences. Hybridization of the probe to nucleic acids fi'om the related organism is then detected using any of the methods described above.<br>
[0302]	By varying the stringency of the hybridization conditions used to identify nucleic<br>
acids, such as cDNAs or genomic DNVs, which hybridize to the detectable probe, nucleic acids having different levels of homology to the probe can be identified and isolated. Stringency may be varied by conducting the hybridization at varying temperatures below the melting temperatures of the probes. The melting temperature, Tm, is the temperature (under defined ionic strength and pH) at which 50% of the target sequence hybridizes to a perfectiy complementary probe. Very stringent conditions are selected to be equal to or about 5°C lower than the T, for a particular probe. The melting temperature of the probe may be calculated using the following formulas:<br>
 <br>
[0303]	For probes between 14 and 70 nucleotides in length the melting temperature (Tm) is<br>
calculated using the foimula: Tn,=81.5--]6.6(log [Na+]&gt;0,41(fi-action G+CH600..'N) where N is the length of the probe.<br>
[6304]	If the hybridization is carried out in a solution contaming formaniide, the melting<br>
temperamre may be calcidated using the equation: Tn,=8I.5+16.6(log [Na+])+0.41 (fraction G+C)-(0.63% formamide)-(600/N) where N is the length of the probe.<br>
[0305]	Prehybridization may be carried out in 6X SSC, 5X Denhardt's reagent, 0.5% SDS,<br>
lOOfxg/m] denatured fragmented salmon sperm DNA or 6X SSC, 5X Denhardt's reagent, 0.5% SDS, 100|j,g/nil denatured fragmented salmon sperm DNA, 50% formamide. The formulas for SSC and Denhardt's solutions are listed in Sambrook et al. supra.<br>
[0306]	In some embodiments, hybridization is conducted by adding the detectable probe to<br>
the prehybridization solutions listed above. Where the probe comprises double stranded DNA, it is denatured before addition to the hybridization solution. In some embodiments, the filter is contacted with the hybridization solution for a sufficient period of tune to allow the probe to hybridize to cDNAs or genomic DNAs containing sequences compiementaiy thereto or homologous thereto. For probes over 200 nucleotides in length, the hybridization may be carried out at 15-25°C belov/ the Tn,. For shorter probes, such as oligonucleotide probes, the hybridization may be conducted at 5-10°C below the Tm- hi some embodiments, for hybridizations in 6X SSC, the hybridization is conducted at approxbnately 68°C. Usually, for hybridizations in 50% formamide containing solutions, the hybridization is conducted at approximately 42°C.<br>
Inhibiting Expression of Aldolase EnzTnes<br>
[0307]	The invention provides  nucleic acids  complementary to  (such as antisense<br>
sequences to) the nucleic acids in accordance with the invention, such as aldolase enzyme-encoding nucleic acids, such as nucleic acids comprising antisense, siRNA, miRNA, ribozymes. Nucleic acids in accordance with the invention comprising antisense sequences can be capable of inhibiting the transport, splicing or transcription of aldolase enzyme-encoding genes. The inhibition can be effected through the targeting of genomic DNA or messenger RNA. The transcription or function of targeted nucleic acid can be inhibited, for example, by hybridization and/or cleavage. One exemplary set of inhibitors provided by the present invention includes oligonucleotides which are able to either bind<br>
 <br>
aldolase, such as pyruvate aldolase, such as HMG and/or KHG aldolase enzyme gene or<br>
message, in either case preventing or iahibiting the production or function of an aldolase,<br>
such as pyruvate aldolase, HMG and/or KHG aldolase enzyme. The association can be<br>
through sequence specific hybridization. Another usefiil cls of inhibitors includes<br>
oligonucleotides which cause inactivation or cleavage of aldolase, such as pyruvate<br>
aldolase, such as HMG and/or KHG aldolase enzyme message. The oligonucleotide can<br>
have enzyme activity which causes such cleavage, such as ribozymes. The<br>
oligonucleotide can be chemically modified or conjugated to an enzyme or composition<br>
capable of cleaving the complementary nucleic acid. A poo! of many different such<br>
oligonucleotides can be screened for tliose with the desired activity. Thus, the invention<br>
pro-i'ides various compositions for the inhibition of aldolase, such as pyrvvate aldolase,<br>
such as HMG and/or KHG aldolase enzyme expression on a nucleic acid and/or protein<br>
levelj such as antisense, siRNA, miRNA and ribozymes comprising aldolase, such as<br>
pyruvate aldolase, such as HMG and/or KHG aldolase enzyme sequences in accordance<br>
with the invention and the anti-aldolase, such as anti-pyruvate aldolase, such as anti-<br>
HMG and/or anti-KHG aldolase antibodies in accordance with the invention.<br>
[0308]	Inhibition of aldolase, such as pyruvate aldolase, such as HMG and/or KHG<br>
aldolase enzyme expression can have a variety of industrial applications. For example, inhibition of aldolase, such as pyruvate aldolase, such as HMG and/or KHG aldolase enzyme expression can slow or prevent spoilage. lo some embodiments, use of compositions in accordance with the- invention that inhibit the expression and/or activity of aldolase, such as pyruvate aldolase, such as HMG and/or KHG aldolase enzymes, such as antibodies, antisense oligonucleotides, ribozymes, siRNA and miRNA are used to slow or prevent spoilage. Thus, in some embodiments, the invention provides methods and compositions comprising application onto a plant or plant product (such as a cereal, a grain, a fruit, seed, root, leaf, etc.) jmtibodies, antisense oligonucleotides, ribozymes, siRNA and miRNA in accordance widi the invention to slow or prevent spoilage. These compositions also can be expressed by the plant (such as a transgenic plant) or another organism (such as a bacterium or other microoianism transformed with an aldolase, such as pyruvate aldolase, HMG and/or KHG aldolase enzyme gene in accordance with the invention).<br>
 <br>
[0309]	The compositions in accordance with the invention for the inhibition of aldolase,<br>
such as pyruvate aldolase, such as I-IMG and/or KHG aldolase enzyme expression (such as antisense, iRNA, ribozymes, antibodies) can be used as pharmaceutical compositions, such as anti-pathogen agents or in other therapies, such as anti-microbials for, such as Salmonella.<br>
Antisense Oligonucleotides<br>
[0310]	The invention provides antisense oligonucleotides capable of binding aldolase,<br>
such as pyruvate aldolase, such as HMG and/or KHG aldolase enzyme message which. In some embodiment, can inhibit aldola;;e, such as pyruvate aldolase, such as HMG and/or KHG aldolase enzyme, activity by targeting mRNA. Strategies for designing antisense oligonucleotides are well described in the scientific and patent literature, and the skilled artisan can design such aldolase, such as pyruvate aldolase, such as HMG and/or KHG aldolase enzyme oligonxxcleotides using the novel reagents in accordance with the invention. For example, gene walking/ RNA mapping protocols to screen for effective antisense oligonucleotides are well known in the art, see Ho (2000) Methods Enzymol. 314:168-183, describing an RNA mapping assay, which is based on standard molecular techniques to provide an easy and reliable method for potent antisense sequence selection-See also Smith (2O00) Eur. J. Phann, Sci. 1} .■191-19S.<br>
[0311]	Naturally occurring nucleic acids are tised as antisense oligonucleotides.   The<br>
antisense oligonucleotides can be of any length; for example, in other embodiments, the antisense oligonucleotides are about 5 to about 100, about 10 to about 80, about 15 to about 60, about 18 to about 40. The optimal length can be detennined by routine screening. The antisense oligonucleotides can be present at any concentration. The optimal concentration can be detemdned by routine screening. A wide variety of synthetic, non-naturally occurring nucleotide and nucleic acid analogues are known which can address this potential problem. For example, peptide nucleic acids (PNAs) containing non-ionic backbones, such as N-(2-aminoethyl) glycine units can be used. Antisense oligonucleotides having phosphorothioate linkages can also be used, as described in WO 97/03211; WO 96/39154; Mata (1997) Toxicol Appl Pharmacol 544:189-197; Antisense Therapeutics, ed. Agrawal (Humana Press, Totowa, N.J., 1996). Antisense oligonucleotides having synthetic DNA backbone analogues provided by the invention can also include phosphoro-dithioate, methylphosphonate, phosphoramidate.<br>
 <br>
aikyl phosphotriester, sulfamate, 3'-tliioacetal, metl!ylene(methyiimino), 3'-N-carbamate,<br>
and morpholino carbamate nucleic acids, as described above.<br>
[0312]	Combinatorial chemistry methodology can be used to create vast numbers of<br>
oligonucleotides that can be rapidly screened for specific oligonucleotides that have appropriate binding affinities and speiiificities toward any target, such as the sense and antisense aldolase, such as pyruvate aldolase, such as HMG and/or KHG aldolase enzyme sequences in accordance with the invention (see Gold (1995) J. of Biol. Chem. 270:13581-13584).<br>
Inhibitory Ribozymes<br>
[0313]	The invention provides ribozymes capable of binding aldolase, such as pyruvate<br>
aldolase, such as HMG and/or KHG aldolase enzyme message. These ribozymes can inhibit aldolase, such as pyruvate aldolase, such as HMG and/or KHG aldolase enzyme ■ activity by, such as targeting mRNA. Strategies for designing ribozymes and selecting the aldolase, such as pyruvate aldolase, such as HMG and/or KHG aldolase enzyme-specific antisense sequence for targeting are well described In the scientific and patent literature, and the skilled artisan can design such ribozymes using the novel reagents in accordance with the invention. Ribozymes act by binding to a target RNA through the target RNA binding portion of a ribozyme which is held in close proximity to an enzymatic portion of the RNA that cleaves the target RNA. Thus, the ribozyme recognizes and binds a target RNA through complementary base-pairing, and once bound to the correct site, acts enzymatically to cleave and inactivate the target RNA. Cleavage of a target RNA in such a manner will destroy its ability to direct synthesis of an encoded protein if the cleavage occurs in the coding sequence. After a ribozyme has bound and cleaved its RNA target, it can be released fi-om that RNA to bind and cleave new targets repeatedly.<br>
[0314]	In some circumstances, the enzymatic nature of a ribozyme can be advantageous<br>
over other technologies, such as antisense technology (where a nucleic acid molecule simply binds to a nucleic acid target to block its transcription, translation or association with another molecule) as the effective concentration of ribozyme necessary to effect a therapeutic treahnent can be lower than that of an antisense oligonucleotide. This potential advantage reflects the ability of the ribozyme to act enzymatically. Thus, a single ribozyme molecule is able to cleave many molecules of target RNA.   In some<br>
 <br>
embodiments, a ribozyme is a highly ;tpecific inhibitor, with the specificity of inhibition<br>
depending not only on the base pairing meciianisra of binding, but also on the mechanism<br>
by which the molecule inhibits the expression of the RNA to which it binds. That is, the<br>
inhibition is caused by cleavage of the RNA target and so specificity is defined as the<br>
ratio of the rate of cleavage of the targeted RNA over the rate of cleavage of no n-targeted<br>
RNA. This cleavage mechatiism is dependent upon factors additional to those involved in<br>
base pairing. Thus, the specificity of action of a ribozyme can be greater than that of<br>
antisense oligonucleotide binding ih&amp; same RNA site.<br>
J0315]	The ribozyme in accordance with the invention, such as an enzymatic ribozyme<br>
RNA molecule, can be forosed in a hanimerhead motif, a hairpin motif, as a hepatitis delta virus motif, a group I intron motif and/or an RNaseP-Iike RNA in association with an RNA guide sequence. Examples of hammerhead motifs are described by, such as Rossi (1992) Aids Research and Human Retroviruses 8:183; hairpin motifs by Hampel (1989) Biochemistry 28:4929, and Hampel (1990) Nuc. Acids Res. 18:299; the hepatitis delta virus motif by Perrotta(1992) Biochemistry 31:16; theRNaseP motif by Guerrier-Takada (1983) Cell 35:849; and the group I intron by Cech U.S. Pat. No. 4,987,071. The recitation of these specific motifs is not intended to be limiting. Those skilled in the art will recognize that a ribozyme in accordance with the invention, such as an enzymatic RNA molecule of this invention, can have a specific substrate binding site complementary to one or more of the target gene RNA regions. A ribozyme in accordance with the invention can have a nucleotide sequence within or surrounding that substrate binding site which imparts an RNA cleaving activity to the molecule.<br>
RNA interference (RNAi)<br>
[0316]	In some embodiments, the invention provides RNA inhibitory molecules, so-<br>
called "RNAi" molecules, comprising aldolase, such as pyruvate aldolase, HMG and/or KHG aldolase enzyme sequences in accordance with the invention. The RNAi molecule can comprise a double-stranded RNA (dsRNA) molecule, such as siRNA, miRNA and/or short hairpin RNA (shRNA) molecules. The RNAi molecule, such as siRNA (small inhibitor)- RNA) and/or miRNA (micro RNA), can inhibit expression of an aldolase, such as pyruvate aldolase, HMG and/or KHG aldolase enzyme gene. In some embodiments, the RNAi molecule, such as siRNA and/or miRNA, is about 11, 12,13, 14, 15, 16, 17,18, 19, 20, 21, 22, 23, 24, 25, 26, 27, 28, 29 or more duplex nucleotides in length. While the<br>
 <br>
invention is not limited by any particular mechanism of action, the RNAi can enter a cell<br>
and cause the degradation of a single-stranded RNA (ssRNA) of similar or identical<br>
sequences, including endogenous mRNAs. When a cell is exposed to double-stranded<br>
RNA (dsRNA), mRNA from the homologous gene is selectively degraded by a process<br>
called RNA interference (KNAi). A possible basic mechanism behind RNAi is the<br>
breaking of a double-stranded RNA (dsRNA) matching a specific gene sequence into<br>
short pieces called short interfering RNA, which trigger the degradation of mRNA that<br>
matches its sequence. In some embodiments, the RNAi's in accordance with the<br>
invention are used in gene-silencing therapeutics, see Shuey (2002) Drug Discov. Today<br>
7:1040-1046. In some embodiments, the invention provides methods to selectively<br>
degrade RNA using the RNAi's molecules, such as siRNA and/or miRNA, in accordance<br>
with the invention. The process may be practiced in vitro, ex vivo or in vivo. In some<br>
embodiments, the RNAi molecules in aw;ordance with the invention can be used to<br>
generate a loss-of-function mutation in a cell, an organ or an animal.<br>
[0317]	In one aspect, intracellular introduction of the RNAi is by internalization of a<br>
target cell specific ligand bonded to an RNA binding protein comprising an RNAi (such as microRNA) is adsorbed. The ligand is specific to a unique target cell surface antigen. The ligand can be spontaneously internalized after binding to the cell surface antigen. If the xmique cell surface antigen is not naturally internalized after binding to its ligand, internalization can be promoted by the incorporation of an arginine-rich peptide, or other membrane permeable peptide, into the structure of the ligand or RNA binding protein or attachment of such a peptide to the ligand or RKA binding protein. See U.S. Patent App. Pub. Nos. 20060030003; 20060025361; 20060019286; 20060019258. hi one aspect, the invention provides lipid-based formulations for delivering, such as introducing nucleic acids of the invention as nucleic acid-lipid particles comprising an RNAi molecule to a cell, see .g., U.S. Patent App. Pub. No. 20060008910.<br>
Modification of Nucleic Acids - Making Variant Enzymes of the Invention<br>
[0318]	The invention provides methods of generating variants of the nucleic acids in<br>
accordance with the invention, such as those encoding an aldolase, such as pyruvate aldolase, HMG and/or KHG aldolase enzyme. These methods can be repeated or used in various combinations to generate aldolase, such as pyruvate aldolase, such as HMG and/or KHG aldolase enzymes having an altered or different activity or an altered or<br>
 <br>
different stability from that of an aldolase, such as pyruvate aldolase, HMG and/or KHG aldolase enzyme encoded by the template nucleic acid. These methods also can be repeated or used in. various combinations, such as to generate variations in gene/ message expression, message translation or message stabiiity. In other embodiments, the genetic composition of a cell is altered by., such as modification of a homologous gene ex vivo. followed by its reinsertion into the cell,<br>
[0319]	A nucleic acid in accordance with the invention can be altered by any means. For<br>
example, random or stochastic methods, or, non-stochastic, or "directed evolution," methods, see U.S. Patent No. 6,361,974. Methods for random mutation of genes are well known in the art, see U.S. Patent No. 5,830,696. For example, mutagens can be used to randomly mutate a gene. Mutagens include, such as ultraviolet light or ganuna irradiation, or a chemical mutagen, such as mitomycin, nitrous acid, photoactivated psoralens, alone or in combination, to induce DNA breaks amenable to repair by recombination. Other chemical mutagens include, for example, sodium bisulfite, nitrous acid, hydroxylamine, hydrazine or formic acid. Other mutagens are analogues of nucleotide precursors, such as mtrosoguanidine, 5-bromouraciI, 2-aminopurine, or acridine. These agents can be added to a PCR reaction in place of the nucleotide precursor thereby mutatii the sequence. Intercalating agents such as proflavine, acriflavine, quinacrine and the like can also be used.<br>
[0320]	Any technique in  molecular  biology can  be  used,   such  as  random  PCR<br>
mutagenesis, see Rice (1992) Proc. Natl. Acad. Sci. USA 89:5467-5471; or, combinatorial multiple cassette mutagenesis, see Crameri (1995) Biotechniques 18:194-196. Alternatively, nucleic acids, such as genes, can be resembled after random, or "stochastic," fragmentation, see U.S. Patent Nos. 6,291,242; 6,287,862; 6,287,861; 5,955,358; 5,830,721; 5,824,514; 5,811,238; 5,605,793. In other embodiments, modifications, additions or deletions are introduced by error-prone PCR, shuffling, oligonucleotide-directed mutagenesis, assembly PCR, sexual PCR mutagenesis, in vivo mutagenesis, cassette mutagenesis, recursive ensemble mutagenesis, exponential ensemble mutagenesis, site-specific mutagenesis, gene reassembly (such as GeneReassembly, see U.S. Patent No. 6,537,776), Gene Site Saturation Mutagenesis (GSSM), synthetic ligation reassembly (SLR), recombination, recursive sequence recombination, phosphothioate-modified DNA mutagenesis, uracil-containing template<br>
 <br>
mutagenesis, gapped duplex mutagenesis, point mismatch repair mutagenesis, repair-<br>
deficient host strain mutagenesis, chemical mutagenesis, radiogenic mutagenesis, deletion<br>
mutagenesis, restriction-selection mutagenesis, restriction-purification mutagenesis,<br>
artificial gene synthesis, ensemble mutagenesis, chimeric nucleic acid multimer creation.<br>
Chromosomal Saturation Mutagenesis (CSM) and/or a combination of these and other<br>
methods.<br>
[0321]	The   following  publications   describe  a  variety  of recursive  recombination<br>
procedures and/or methods which can be incorporated into the methods in accordance with the invention: Stemmer (1995') "Molecular breeding of viruses for targeting and other clinical properties" Tumor Targeting 4:1-4; Ness (1999) Nature Biotechnology' 17:893-896; Chang (1999) "Evolution of a cytokme using DNA family shuffling" Nature Biotechnology 17:793-797; Minshull (1999) "Protein evolution by molecular breeding" Current Opinion in Chemical Biology 3:284-290; Christians (1999) "Directed evolution of thymidine kinase for AZT phosphorylation using DNA family shuffling" Nature Biotechnology 17:259-264; Crameri (1998) "DNA shuffling of a family of genes from diverse species accelerates directed evolution" Nature 391:288-291; Crameri (1997) "Molecular evolution of an arsenate detoxification pathway by DNA shuffling," Nature Biotechnology 15:436-438; Zhang (1997) "Directed evolution of an effective fiicosidase from a galactosidase by DNA shuffling and screening" Proc. Natl. Acad. Sci. USA 94:4504-4509; Patten et al. (1997) "Applications of DNA Shuffling to Phaimaceuticals and Vaccines" Current Opinion in Biotechnology 8:724-733; Crameri et al. (1996) "Construction and evolution of antibody-phage libraries by DNA shuffling" Nature Medicine 2:100-103; Gates et al. (1996) "Affinity selective isolation of ligands from peptide libraries through display on a lac repressor 'headpiece dimer'" Journal of Molecular Biology 255:373-386; Stemmer (1996) "Sexual PCR and Assembly PCR" hi; The Encyclopedia of Molecular Biology. VCH Publishers, New York, pp.447-457; Crameri and Stemmer (1995) "Combmatorial multiple cassette mutagenesis creates all the permutations of mutant and wildtype cassettes" BioTechniques 18:194-195; Stemmer et al. (1995) "Single-step assembly of a. gene and entire plasmid form large numbers of oligodeoxyribonucleotides" Gene, 164:49-53; Stemmer (1995) "The Evolution of Molecular Computation" Science 270: 1510; Stemmer (1995) "Searching Sequence Space" Bio/Technology 13:549-553; Stemmer (1994) "Rapid evolution of a protein in<br>
 <br>
vitro by DNA shuffling" Nature 370:389-391; and Stemmer (1994) "DNA shuffling by<br>
random fragmentation and reassembly: In vitro recombination for molecular evolution."<br>
Proc. Natl. Acad. Sci. USA 91:10747-10751.<br>
[0322]	Mutational methods of generating diversity include, for example, site-directed<br>
mutagenesis (Ling et al. (1997) "Approaches to DNA mutagenesis: an overview" Anal Biochem. 254(2): 157-178; Dale et al. (1996) "Oligonucleotide-directed random mutagenesis using the phosphorothioate method" Methods Mo], Biol. 57:369-374; Smith<br>
(1985)	"In vitro mutagenesis" Ann. Rev. Genet. 19:423-462; Botstein &amp; Shortle (1985)<br>
"Strategies and applications of in vitro mutagenesis" Science 229:n93-]201; Carter<br>
(1986)	"Site-directed mutagenesis" Biochem. J. 237:1-7; and Kunkel (1987) "The<br>
efficiency of oligonucleotide directed mutagenesis" in Nucleic Acids &amp; Molecular<br>
Biology (Eckstein, F. and Lilley, D. M. J. eds., Springer Verlag, Berlin)); mutagenesis<br>
using uracil containing templates (JIunkel (1985) "Rapid and efficient site-specific<br>
mutagenesis without phenotypic selection" Proc. Natl. Acad. Sci. USA 82:488-492;<br>
Kunkel et al. (1987) "Rapid and efficient site-specific mutagenesis without phenotypic<br>
selection" Methods in Enzymol. 154, 367-382; and Bass et al. (1988) "Mutant Trp<br>
repressors with new DNA-binding specificities" Science 242:240-245); oligonucleotide-<br>
directed mutagenesis (Methods in Enzymol. 100: 468-500 (1983); Methods in Enzymol.<br>
154: 329-350 (1987); Zoller (1982) "Oligonucleotide-directed mutagenesis using M13-<br>
derived vectors: an efficient and general procedure for the production of point mutations<br>
in any DNA fragment" Nucleic Acids Res. 10:6487-6500; Zoller &amp; Smith (1983)<br>
"Oligonucleotide-directed mutagenesis of DNA fragments cloned into M13 vectors"<br>
Methods in Enzymol. 100:468-500; and Zoller (1987) Oligonucleotide-directed<br>
mutagenesis: a simple method using two oligonucleotide primers and a single-stranded<br>
DNA template" Methods in Enzymol. 154:329-350); phosphorothioate-modified DNA<br>
mutagenesis (Taylor (1985) "The use of phosphorothioate-modified DNA in restriction<br>
enzyme reactions to prepare nicked DNA" Nucl. Acids Res. 13: 8749-8764; Taylor<br>
(1985) "The rapid generation of oligonucleotide-directed mutations at high frequency<br>
using phosphorothioate-modified DNA" Nucl. Acids Res. 13: 8765-8787 (1985);<br>
Nakamaye (1986) "Inhibition of restriction endonuclease Nci I cleavage by<br>
phosphorothioate groups and its application to oligonucleotide-directed mutagenesis"<br>
NucL Acids Res. 14; 9679-9698; Sayers (1988) "Y-T Exonucleases in phosphorothioate-<br>
 <br>
based oligonucleotide-directed mutagenesis" Nucl. Acids Res. 16:791-802; and Sayers et<br>
al. (1988) "Strand specific cleavage of phospliorothioate-containing DNA by reaction<br>
with restriction endonucleases in the presence of ethidium bromide" Nucl. Acids Res. 16:<br>
803-814); mutagenesis using gapped duplex DNA (Kramer et al. (1984) "The gapped<br>
duplex DNA approach to oligonucleotide-directed mutation construction" Nucl. Acids<br>
Res. 12: 9441-9456; Kramer &amp; Fritz (1987) Methods in Enzymol. "Oligonucleotide-<br>
directed construction of mutations via gapped duplex DNA" 154:350-367; Kramer (1988)<br>
"Improved enzymatic in vitro reactions in the gapped duplex DNA approach to<br>
oligonucleotide-directed construction of mutations" Nucl. Acids Res. 16: 7207; and Fritz<br>
(1988) "Oligonucleotide-directed construction of mutations: a gapped duplex DNA<br>
procedure without enzymatic reactions in vitro" Nucl. Acids Res. 16: 6987-6999).<br>
[0323]	Additional protocols that can be used to practice the invention include point<br>
mismatch repair (Kramer (1984) "Point Mismatch Repair" Cell 38:879-887), mutagenesis using repair-deficient host strains (Carter et al. (1985) "Improved oligonucleotide site-directed mutagenesis using M13 vectors" Nucl. Acids Res. 13: 4431-4443; and Carter (1987) "Improved oligonucleotide-directed mutagenesis using Ml 3 vectors" Methods in EnTOol. 154; 382-403), deletion mutagenesis (Eghtedarzadeh (1986) "Use of oligonucleotides to generate large deletions" Nucl. Acids Res. 14: 5115), restriction-selection and restriction-selection and restriction-purification (Wells et al. (1986) "Importance of hydrogen-bond formation in stabilizing the transition state of subtilisin" Phil- Trans. R. Soc. Lond. A 317: 415-423), mutagenesis by total gene synthesis (Nambiar et al. (1984) "Total synthesis and cloning of a gene coding for the ribonuclease S protein" Science 223: 1299-1301; Sakamar and Khorana (1988) "Total synthesis and expression of a gene for the a-subunit of bovine rod outer segment guanine nucieotide-binding protein (transducin)" Nucl. Acids Res. 14: 6361-6372; Wells et al. (1985) "Cassette mutagenesis: an efficient method for generation of multiple mutations at defined sites" Gene 34:315-323; and Grundstrom et al. (1985) "Oligonucleotide-directed mutagenesis by microscale 'shot-gun' gene synthesis" Nucl. Acids Res. 13: 3305-3316), double-strand break repair (Mandecki (1986); Arnold (1993) "Protein engineering for unusual environments" Current Opinion in Biotechnolog}' 4:450-455. "Oligonucleotide-directed double-strand break repair in plasmids of Escherichia coli; a method for site-specific mutagenesis" Proc. Natl. Acad. Sci. USA, 83:7177-7181). Additional details on<br>
 <br>
many of the above methods can be found in Methods in Enzymology Voiume 154, which<br>
also describes useful controls for troabie-sfaooting problems with various mutagenesis<br>
methods.<br>
[0324J	Protocols that can be used to practice the invention are described, such as in U.S.<br>
Patent Nos. 5,605,793 to Stemmtir (Feb. 25, 1997), "Methods for In Vilro Recombination;" U.S. Pat. No. 5,811,238 to Stemmer et al. (Sep. 22, 1998) "Methods for Generating Polynucleotides having Desired Characteristics by Iterative Selection and Recombination;" U.S. Pat No. 5,830,721 to Stemmer et ai. (Nov. 3, 1998), "DNA Mutagenesis by Random Fragmentation and Reassembly;" U.S. Pat. No. 5,834,252 to Stemmer, et al. (Nov. 10, 1998) "End-Complementary Polymerase Reaction;" U.S. Pat. No. 5,837,458 to Minshull, et al. (Nov. 17, 1998), "Methods and Compositions for Cellular and Metabolic Engineering;" WO 95/22625, Stemmer and Crameri, "Mutagenesis by Random Fragmentation and Reassembly;" WO 96/33207 by Stemmer and Lipschutz "End Complementaiy Polymerase Chain Reaction;" WO 97/20078 by Stemmer and Crameri "Methods for Generating Polynucleotides having Desired Characteristics by Iterative Selection and Recombination;" WO 97/35966 by Minshull and Stemmer, "Methods and Compositions for Cellular and Metabolic Engineering;" WO 99/41402 by Punnonen et al. "Targeting of Genetic Vaccine Vectors;" WO 99/41383 by Punnonen et al. "Antigen Library Immunization;" WO 99/41369 by Punnonen et al. "Genetic Vaccine Vector Engineering;" WO 99/41368 by Punnonen et al. "Optimization of Immunomodulatory Properties of Genetic Vaccines;" EP 752008 by Stemmer and Crameri, "DNA Mutagenesis by Random Fragmentation and Reassembly;" EP 0932670 by Stemmer "Evolving Cellular DNA Uptake by Recursive Sequence Recombination;" WO 99/23107 by Stemmer et al., "Modification of Virus Tropism and Host Range by Viral Genome Shuffling;" WO 99/21979 by Apt et al., "Human Papillomavirus Vectors;" WO 98/31837 by del Cardayre et al.. "Evolution of Whole Cells and Organisms by Recursive Sequence Recombination;" WO 98/27230 by Patten and Stemmer, "Methods and Compositions for Polypeptide Engineering;" WO 98/27230 by Stenmier et al., "Methods for Optimization of Gene Therapy by Rectirsive Sequence Shuffling and Selection," WO 00/00632, "Methods for Generating Highly Diverse Libraries," WO 00/09679, "Methods for Obtaining in Vitro Recombined Polynucleotide Sequence Banks and   Resulting   Sequences,"   WO   98/42832  by   Arnold  et   al.,   "Recombination   of<br>
 <br>
Polynucleotide Sequences Using Random or Defined Primers," WO 99/29902 by Arnold et al., "Method for Creating Polynucleotide and Polypeptide Sequences," WO 98/41653 by Vind, "An in Vitro Method for Construction of a DNA Library," WO 98/41622 by Borchert ef al., "Method for Constructing a Libraiy Using DNA Shuffling," and WO 98/42727 by Pati and Zarling, "Sequence Alterations using Homologous Recombination."<br>
[0325]	Protocols that can be used to practice the invention (providing details regarding<br>
various diversity generating methods) are described, such as in U.S. Patent application<br>
serial no. (USSN) 09/407,800, "SfflJFFLING OF CODON ALTERED GENES" by<br>
Patten et ai. filed Sep. 28, 1999; "EVOLUTION OF WHOLE CELLS AND<br>
ORGANISMS BY RECURSIVE SEQUENCE RECOMBINATION" by del Cardayre et<br>
al., United States Patent No. 6,379,964.; "OLIGONUCLEOTIDE MEDIATED NUCLEIC<br>
ACID RECOMBINATION" by Crameri et al.. United States Patent Nos. 6,319,714;<br>
6,368,861; 6,376,246; 6,423,542; 6,426,224 and PCT/USOO/01203; "USE OF CODON-<br>
VARIED OLIGONUCLEOTIDE SYNTHESIS FOR SYNTHETIC SHUFFLING" by<br>
Welch et al.. United States Patent No. 6,436,675; "METHODS FOR MAKING<br>
CHARACTER STRINGS, POLYNUCLEOTIDES &amp; POLYPEPTIDES HAVING<br>
DESIRED CHARACTERISTICS" by Selifonov et ai., filed Jan. 18, 2000,<br>
(PCT/USOO/01202) and, such as "METHODS FOR MAKING CHARACTER STRINGS,<br>
POLYNUCLEOTIDES	&amp;	POLYPEPTIDES	HAVING	DESIRED<br>
CHARACTERISTICS" by Selifonov et a!., filed Jul. 18, 2000 (U.S. Ser. No. 09/618,579); "METHODS OF POPULATING DATA STRUCTURES FOR USE IN EVOLUTIONARY SIMULATIONS" by Selifonov and Stemmer, filed Jan. 18, 2O00 (PCT/USOO/01138); and "SINGLE-SIIRANDED NUCLEIC ACID TEMPLATE-MEDIATED RECOMBINATION AND NUCLEIC ACID FRAGMENT ISOLATION" by Affholter, filed Sep. 6, 2000 (U.S. Ser. No. 09/656,549); and United States Patent Nos. 6,177,263; 6,153,410.<br>
[0326]	Non-stochastic, or "directed evolution," methods include, such as saturation<br>
mutagenesis, such as Gene Site Saturation Mutagenesis (GSSM), synthetic ligation reassembly (SLR), or a combination thereof are used to- modify the nucleic acids in accordance with the invention to generate aldolase, such as pyruvate aldolase, such as HMG and/or KHG aldolase enzymes with new or altered properties (such as activity under highly acidic or alkaline conditions, high or low temperatures, and the like).<br>
 <br>
Polypeptides encoded by the modified nucleic acids can be screened for an activity before testing for carbon-carbon bond formation or cleavage or other activity. Any testing modality or protocol can be used, such as using a capillary array platform. See U.S. Patent Nos. 6,361,974; 6,280,926; 5,939,250.<br>
Gene Site Saturation Mutagenesis, or, GSSM<br>
[0327]	The invention also provides methods  for making enzyme using Gene Site<br>
Saturation mutagenesis, or, GSSM, as described herein, and also in U.S. Patent Nos. 6,171,820 and 6,579,258. hi some embodiments, codon primers containing a degenerate N,N,G/T sequence are used to introduce point mutations into a polynucleotide, such as an aldolase, such as pyruvate aldolase, H}dG and/or KHG aldolase enzyme or an antibody in accordance with the invention, so as to generate a set of progeny polypeptides in which a full range of single amino acid substitutions is represented at each amino acid position, such as an amino acid residue in an enzyme active site or ligand binding site targeted to be modified. These oligonucleotides can comprise a contiguous first homologous sequence, a degenerate N,N,G/T sequence, and, optionally, a second homologous sequence. The downstream progeny translational products from the use of such oligonucleotides include all possible amino acid changes at each amino acid site along the polypeptide, because the degeneracy of the N,N,G/T sequence includes codons for all 20 amino acids. In some embodiments, one such degenerate oligonucleotide (comprised of, such as one degenerate N,N,G/T cassette) is used for subjecting each original codon in a parental polynucleotide template to a fall range of codon substitutions. In other embodiments, at least two degenerate cassettes are used - either in the same oligonucleotide or not, for subjet;tiiig at least two original codons in a parental polynucleotide template to a lull range of codon substitutions. For example, more than one N,N,G/T sequence can be contained in one oligonucleotide to introduce amino acid mutations at more than one site. This plurality of NJ<i sequences can be directly contiguous or separated by one more additional nucleotide sequence in other embodiments oligonucleotides serviceable for introducing additions and deletions xised either alone combination with the codons containing an n to introduce any permutation of amino acid substitutions.></i>
 <br>
[0328]	In some embodiments, simultaneous mutagenesis of two or more contiguous<br>
amino acid positions is done using an oligonucleotide that contains contiguous N,N,G/T triplets, i.e. a degenerate (N,N,G/T)n sequence. In other embodiments, degenerate cassettes having less degeneracy than the N,N,G/T sequence are used. For example, it may be desirable in some instances to use (such as in an oligonucleotide) a degenerate triplet sequence comprised of only one N, where said N can be in the first second or third position of the triplet. Any other bases including any combinations and permutations thereof can be used in the remaining two positions of the triplet. Alternatively, it may be desirable in some instances to use (such as in an oligo) a degenerate N,N,N triplet sequence.<br>
[0329]	In some embodiments, use of degenerate triplets (such as N,N,G/T triplets) allows<br>
for systematic and easy generation of a full range of possible natural amino acids (for a total of 20 amino acids) into each and every amino acid position in a polypeptide (in other embodiments, the methods also include generation of less than all possible substitutions per amino acid residue, or codon, position). For example, for a 100 amino acid polypeptide, 2000 distinct species (i.e. 20 possible amino acids per position X 100 amino acid positions) can be generated. Through the use of an oligonucleotide or set of oligonucleotides containing a degenerate N,N,G/T triplet, 32 individual sequences can code for all 20 possible natural amino acids. Thus, in a reaction vessel in which a parental polynucleotide sequence is subjected to saturation mutagenesis using at least one such oligonucleotide, there are generated 32 distinct progeny polynucleotides encoding 20 distinct polypeptides. In contrast, the use of a non-degenerate oligonucleotide in site-directed mutagenesis leads to only one progeny polypeptide product per reaction vessel. Nondegenerate oligonucleotides can optionally be used in combination with degenerate primers disclosed; for example, nondegenerate oligonucleotides can be used to generate specific point mutations in a working polynucleotide. This provides one means to generate specific silent point mutations, point mutations leading to corresponding amino acid changes, and point mutations that cause the generation of stop codons and the corresponding expression of polypeptide fragments.<br>
[0330]	In some embodiments, each saturation mutagenesis reaction vessel contains<br>
polynucleotides encoding at least 20 progeny polypeptide (such as aldolase, such as pyruvate aldolase, such as HMG and/or KHG aldolase enzymes) molecules such that all<br>
 <br>
20 natural amino acids are represented at the one specific amino acid position corresponding to the codon position mutagenized in the parental polynucleotide (other embodiments use less than all 20 natural combinations). The 32-fold degenerate progeny polypeptides generated from each saturation mutagenesis reaction vessel can be subjected to clonal amplification (such as cloned into a suitable host, such as E. coli host, using, such as an expression vector) and subjected to expression screening. When an individual progeny pol3peptide is identified by screening to displaj' a favorable change in propert}' (when compared to the parental polypeptide, such as increased carbon-carbon formation or cleavage activity under alkaline or acidic conditions), it can be sequenced to identify the correspondingly favorable amino acid substitution contained therein.<br>
[0331]	In some embodiments, upon mutagenizing each and every amino acid position in<br>
a parental polypeptide using saturation mutagenesis as disclosed herein, favorable amino acid changes may be identified at more than one amino acid position. One or more new progeny molecules can be generated that contain a combination of all or part of these favorable amino acid substitutions. For example, if 2 specific favorable amino acid changes are identified in each of 3 amino acid positions in a polypeptide, the permutations include 3 possibilities at each position (no change from the original amino acid, and each of two favorable changes) and 3 positions. Thus, there are 3 x 3 x 3 or 27 total possibilities, including 7 that were previously exammed - 6 single point mutations (i.e. 2 at each of three positions) and no change at any position.<br>
[0332]	In yet another embodiment, site-saturation mutagenesis can be used together with<br>
shuffling, chimerization, recombination and other mutagenizing processes, along with screening. This invention provides for the use of any mutagenizing process(es), including saturation mutagenesis, in an iterative manner. In one exemplification, the iterative use of any mutagenizing process(es) is used in combination with screening.<br>
[0333]	The invention also provides for the use of proprietary codon primers (containing a<br>
degenerate N,N,N sequence) to introduce point mutations into a polynucleotide, so as to generate a set of progeny polypeptides in which a fiill range of single amino acid substitutions is represented at eh amino acid position (Gene Site Saturation Mutagenesis (GSSM)). The oligos used are comprised contiguously of a first homologous sequence, a degenerate N,N,N sequence and, in some embodiments but not necessarily, a second homologous sequence.    The downstream progeny translational<br>
 <br>
products from the use of such oligos include all possible amino acid changes at each amino acid site along the polypeptide, because the degeneracy of the N.N,N sequence includes codons for all 20 amino acids.<br>
[0334]	In some embodiments, one such degenerate oligo (comprised of one degenerate<br>
N,N,N cassette) is used for subjecting each original codon in a parental polynucleotide template to a fiill range of codon substitutions. In other embodiments, at least two degenerate N,N,N cassettes are used - either in the same oligo or not, for subjecting at least two original codons in a parental polynucleotide template to a full range of codon substitutions. Thus, more than one I'3,N,N sequence can be contained in one oligo to introduce amino acid mutations at more than one site. This plurality of N,N,N sequences can be directly contiguous, or separated by one or more additional nucleotide sequence(s). In other embodiments, oligos serviceable for introducing additions and deletions can be used either alone or in combination vnth the codons containing an N,N,N sequence, to introduce any combination or permutation of amino acid additions, deletions and/or substitutions.<br>
[0335]	In some embodiments, it is pcissible to simultaneously mutagenize two or more<br>
contiguous amino acid positions using an oligo that contains contiguous N,N,N triplets, i.e. a degenerate (NN,N)TI sequence. In other embodiments, the present invention provides for the use of degenerate cassettes having less degeneracy than the N,N,N sequence. For example, it may be desirable in some instances to use (such as in an oligo) a degenerate triplet sequence comprised of only one N, where the N can be in the first second or third position of the triplet. Any other bases including any combinations and permutations thereof can be used in the remaming two positions of the triplet. Altematively, it may be desirable in some instances to use (such as in an oligo) a degenerate N,N,N triplet sequence, N,N,G/T, or anN,N, G/C triplet sequence.<br>
[0336]	In some embodiments, use of a degenerate triplet (such as N,N,G/T or an N,N,<br>
G/C triplet sequence) is advantageous for several reasons. In some embodiments, this invention provides means to systemati.cally and fairly easily generate the substitution of the full range of possible amino acids (for a total of 20 amino acids) into each and every amino acid position in a polypeptide. Thus, for a 100 amino acid pol&gt;peptide, the invention provides ways to systematically and fairly easily generate 2000 distinct species (/.£., 20 possible amino acids per position times  100 amino acid positions).    It is<br>
 <br>
appreciated that there is provided, Ihrough the use of an oligo containing a degenerate N,N,G,T or an N,N, G/C triplet sequence, 32 individual sequences that code for 20 possible amino acids. Thus, in a reaction vessel in which a parental polynucleotide sequence is subjected to saturation mutagenesis using one such oligo, there are generated 32 distinct progeny polynucleotides ejicoding 20 distinct polypeptides. In contrt, the use of a non-degenerate oligo in site-directed mutagenesis leads to only one progeny polypeptide product per reaction vessel.<br>
[0337]	This invention also provides for the use of nondegenerate oligos, which can<br>
optionally be used in combination with degenerate primers disclosed. It is appreciated that in some situations, it is advantageous to use nondegenerate oHgos to generate specific point mutations in a working polynucleotide. This provides means to generate specific silent point mutations, point mutatJons; leading to corresponding amino acid changes and point mutations that cause the generation of stop codons and the corresponding expression of polypeptide fragments.<br>
[0338]	Thtis,  in some embodiments of this  invention, each saturation mutagenesis<br>
reaction vessel contains polynucleotides encoding at least 20 progeny polypeptide molecules such that all 20 amino acids are represented at the one specific amino acid position corresponding to the codon position mutagenized in the parental polynucleotide. The 32-fold degenerate progenj' polypeptides generated from each saturation mutagenesis reaction vessel can be subjected to clonal amplification (such as cloned into a suitable E. coli host using an expression vector) and subjected to expression screening. When an individual progeny polypeptide is idenified by screening to display a favorable change in property (when compared to the parenlal polypeptide), it can be sequenced to identify the correspondingly favorable amino acid substitution contained therein.<br>
[0339]	In some embodiments, upon mutagenizing each and every amino acid position in<br>
a parental polypeptide using saturation mutagenesis as disclosed herein, a favorable amino acid changes is identified at more than one amino acid position. One or more new progeny molecules can be generated that contain a combination of all or part of these fevorable amino acid substitutions. For example, if 2 specific favorable amino acid changes are identified in each of 3 amino acid positions in a polypeptide, the permutations include 3 possibilities at each position (no change from the original amino acid and each of two favorable changes) and 3 positions. Thus, there are 3 x 3 x 3 or 27<br>
 <br>
totai possibilities, including 7 that were previously examined - 6 single point mutations (i.e., 2 at each of three positions) and no change at any position.<br>
[0340]	The invention provides for the use of saturation mutagenesis in combination with<br>
additional mutagenization processes, such as process where two or more related poiynucieotides are introduced into a iiuitable host ceil such that a hybrid poiynucleotide is generated by recombination and reductive reassortment.<br>
[0341]	In addition to performing mutagenesis along the entire sequence of a gene, the<br>
instant invention provides that mutagenesis can be use to replace each of any number of bases in a polynucleotide sequence, wherein the number of bases to be mutagenized is, in some embodiments every integer from 15 to 100,000. Thus, instead of mutagenizing every position along a molecule, one can subject every or a discrete number of bases (in some embodiments a subset totaling from 15 to 100,000) to mutagenesis. In some embodiments, a separate nucleotide is used for mutagenizing each position or group of positions along a polynucleotide sequence. A group of 3 positions to be mutagenized may be a codon. The mutations can be introduced using a mutagenic primer, containing a heterologous cassette, also referred to as a mutagenic cassette. Exemplary csettes can have from 1 to 500 bases. Each nucleotide position in such heterologous cassettes be N, A, C, G, T, A/C, A/G, A/T, C/G, C/T, G/T, C/G/T, A/G/T, AJC/T, A/C/G, or E, where E is any base that is not A, C, G, or T (E can be referred to  a designer oligo).<br>
[0342]	In some embodiments, saturation mutagenesis is comprised of mutagenizing a<br>
complete set of mutagenic cassettes (wherein each cassette is, in some embodiments, about 1-500 bases in length) in defined polynucleotide sequence to be mutagenized (wherein the sequence to be mutagenized is, in some embodiments, from about 15 to 100,000 bases in length). Thus, a group of mutations (ranging from 1 to 100 mutations) is introduced into each cassette to be mutagenized. A grouping of mutations to be introduced into one cassette can be different or the same from a second grouping of mutations to be introduced into a second cassette during the application of one rotmd of saturation mutagenesis. Such groupings are exemplified by deletions, additions, groupings of particular codons and groupings of particular nucleotide cassettes.<br>
[0343]	In some embodiments, defmed sequences to be mutagenized include a whole<br>
gene, pathway, cDNA, an entire open reading frame (ORP) and entire promoter, enhancer,   repressor/transactivator,   origin   of  replication,   intron,   operator,   or   any<br>
 <br>
polynucleotide functional group. Generally, a "defined sequences" for this purpose may<br>
be any polynucleotide that a 15 base-polynucleotide sequence and polynucleotide<br>
sequences of lengths between 15 bases and 15,000 bases (this invention specifically<br>
names every integer in between). Considerations in choosing groupings of codons<br>
include types of amino acids encoded by a degenerate mutagenic cassette.<br>
J03441	In some embodiments, a grouping of mutations that can be introduced into a<br>
mutagenic cassette, this invention specifjcall}' pro'i'ides for degenerate codon substitutions (using degenerate oligos) that code for 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12,13, 14,15, 16, 17, 18,19 and 20 amino acids at each position and a library of polypeptides encoded thereby.<br>
Synthetic Ligation Reassembly (SLR)<br>
[0345]	The  invention provides  a  non-stochastic  gene  modification  system  termed<br>
"synthetic ligation reassembly," or simply "SLR," a "directed evolution process," to generate polypeptides,, such as aldolase, such as pyruvate aldolase, such as HMG and/or KHG aldolase enzymes or antibodies in accordance with the invention, with new or altered properties.<br>
(0346]	SLR is a method of Hgating oligonucleotide fragments together non-stochastically.<br>
This method differs from stochastic oligonucleotide shuffling in that the nucleic acid building blocks are not shuffled, concatenated or chimerized randomly, but rather are assembled non-stochastically. See U.S. Patent Nos. 6,773,900; 6,740,506; 6,713,282; 6,635,449; 6,605,449; 6,537,776. In some embodiments, SLR comprises the following steps: (a) providing a template polynucleotide, wherein the template polynucleotide comprises sequence encoding a homologous gene; (b) providing a plurality of building block polynucleotides, wherein the building block polynucleotides are designed to cross¬over reassemble with the template polynucleotide at a predetermined sequence, and a building block polynucleotide comprises a seqtience that is a variant of the homologous gene and a sequence homologous to the template polynucleotide flariking the variant sequence; (c) combining a building block polynucleotide with a template polynucleotide such that the building block polynucleotide cross-over reassembles with the template polynucleotide to generate polynucleotides comprising homologous gene sequence variations.<br>
[0347]	SLR does not depend on iht presence of high levels of homology between<br>
polynucleotides to be rearranged.   Thus, this method can be used to non-stochastically<br>
 <br>
generate libraries (or sets) of progeny molecules comprised of over I0'°° different chimeras. SLR can be used to generate-iibraries comprised of over lo'"™ different progeny chimeras. Thus, embodiments of the present invention include non-stochastic methods of producing a set of finalized chimeric nucleic acid molecule shaving an overall assembly order that is chosen by design. This method includes the steps of generating by design a plurality of specific nucleic acid building blocks having serviceable mutually compatible ligatable ends, and assembling these nucleic acid building blocks, such that a designed overall assembly order is achieved.<br>
[0348]	The mutually compatible ligatable ends of the nucleic acid building blocks to be<br>
assembled are considered to be "serviceable" for this type of ordered assembly if they enable the building blocks to be coupled in predetermined orders. Thus, the overall assembly order in which the nucleic acid building blocks can be coupled is specified by the design of the ligatable ends. If more than one assembly step is to be used, then the overall assembly order in which the nucleic acid building blocks can be coupled is also specified by the sequential order of the assembly step{s). In some embodiments, the annealed building pieces are treated with an enzyme, such as a ligase (such as T4 DNA ligase), to achieve covalent bonding of the building pieces.<br>
[0349]	In some embodiments, the design of the oligonucleotide building blocks is<br>
obtained by analyzing a set of progenitor nucleic acid sequence templates that serve as a basis for producing a progeny set of finalized chimeric polynucleotides. These parental oligonucleotide templates thus serve as a source of sequence information that aids in the design of the nucleic acid building blocks that are to be mutagenized, such as chimerized or shuffled. In some embodiments of this method, the sequences of a plurality of parental nucleic acid templates are aligned in order to select one or more demarcation points. The demarcation points can be located at an area of homology, and are comprised of one or more nucleotides. These demarcation points are, in some embodiments, shared by at least two of the progenitor templates. The demarcation points can thereby be used to delineate the boundaries of oligonucleotide building blocks to be generated in order to rearrange the parental polynucleotides. The demarcation points identified and selected in the progenitor molecules serve as potential chimerization points m the assembly of the final chimeric progeny molecules. A demarcation point can be an area of homology (comprised of at least one homologous nucleotide base) shared by at least two parental<br>
 <br>
polynucleotide sequences. Alternatively, a demarcation point can be an area of homology that is shared by at least half of the parental polynucleotide sequences, or, it can be an area of homology that is shared by at least two thirds of the parental polynucleotide sequences. Even more, in some embodiments, a serviceable demarcation points is an area of homology that is shared by at least three fourths of the parental polynucleotide sequences, or, it can be shared by al almost all of the parental polynucleotide sequences, hi some embodiments, a demarcation point is an area of homology that is shared by all of the parental pol3Tiucleotide sequences.<br>
[0350]	hi some embodunents, a ligation reassembly process is performed exhaustively in<br>
order to generate an exhaustive libraiy of progeny chimeric polynucleotides, hi other words, a]] possible ordered combinations of the nucleic acid building blocks are represented in the set of finalized chimeric nucleic acid molecules. At the same time, in other embodiments, the assembly order (i.e. the order of assembly of each building block in the 5' to 3 sequence of each finalized chimeric nucleic acid) in each combination is by design (or non-stochastic) as described above. Because of the non-stochastic nature of this invention, the possibility of unwanted side products is greatly reduced.<br>
[0351]	In    other   embodiments,   the    ligation   reassembly    method    is   performed<br>
systematically. For example, the method is performed in order to generate a systematically compartmentalized library of progeny molecules, with compartments that can be screened systematically, such as one by one. In other words this invention provides that, through the selective and judicious use of specific nucleic acid building blocks, coupled with the selective and judicious use of sequentially stepped assembly reactions, a design can be achieved where specific sets of progeny products are made in each of several reaction vessels. This allows a systematic examination and screening procedxire to be performed. Thus, these methods allow a potentially very large number of progeny molecules to be examined systematically in smaller groups. Because of its ability to perform chimerizations in a manner that is highly flexible yet exhaustive and systematic as well, particularly when there is a low level of homology among the progenitor molecules, these methods provide for the generation of a library (or set) comprised of a large number of progeny molecules. Because of the non-stochastic nature of the instant ligation reassembly invention, the progen)' molecules generated in some embodiments comprise a library of finalized chimeric nucleic acid molecules having an<br>
 <br>
overall assembly order that is chosen by design. The saturation mutagenesis and optimized directed evolution method.'; also can be used to generate different progeny molecular species. It is appreciated that the invention provides freedom of choice and control regarding the selection of demiircation points, the size and number of the nucleic acid building blocks, and the size and design of the couplings. It is appreciated, furthermore, that the requirement for intermolecular homology is highly relaxed for the operability of this invention, in fact, demarcation points can even be chosen in areas of little or no intermolecular homology. For example, because of codon wobble, i.e. the degeneracy of codons, nucleotide substitutions can be introduced into nucleic acid building blocks without altering the amino acid originalJj' encoded in the corresponding progenitor template. Alternatively, a codon can be altered such that the coding for an originally amino acid is altered. This invention provides that such substitutions can be introduced into the nucleic acid building block in order to increase the incidence of intermolecular homologous demarcation points and thus to allow an increased number of couplings to be achieved among the building blocks, which in turn allows a greater number of progeny chimeric molecules to be generated.<br>
Synthetic gene reassembly<br>
[0352]	In some embodiments, the present invention provides a non-stochastic method<br>
termed synthetic gene reassembly, that is somewhat related to stochastic shuffling, save that the nucleic acid building blocks are not shuffled or concatenated or chimerized randomly, but rather are assembled non-stochastically. See U.S. Patent No. 6,537,776.<br>
{0353]	The synthetic gene reassembly method does not depend on the presence of a high<br>
level of homology between polynucleotides to be shuffled. The invention can be used to non-stochastically generate libraries (or sets) of progeny molecules comprised of over lO'"" different chimeras. Conceivably, synthetic gene reassembly can even be used to generate libraries comprised of over 10     different progeny chimeras.<br>
[0354]	Thus, in some embodiments, the invention provides a non-stochastic method of<br>
producing a set of finalized chimeric nucleic acid molecules having an overall assembly order that is chosen by design, which method is comprised of the steps of generating by design a plurality of specific nucleic acid building blocks having serviceable mutually compatible ligatable ends and assembling these nucleic acid building blocks, such that a designed overall assembly order is achieved.<br>
 <br>
[0355]	The mutually compatible ligat£ible ends of the nucleic acid building blocks to be<br>
assembled are considered to be "serviceable" for this type of ordered assembly if they enable the building blocks to be coupled in predetermined orders. Thus, in some embodiments, the overall assembly order in which the nucleic acid building blocks can be coupled is specified by the design of the ligatable ends and, if more than one assembly step is to be used, then the overall assembly order in which the nucleic acid building blocks can be coupled is also specified by the sequential order of the assembly slep(s). In a one embodiment, of the invention, the annealed building pieces are treated with an enzyme, such as a lige (such as T4 DNA ligase) to achieve covalent bonding of the building pieces.<br>
[0356]	In a another embodiment, the design of nucleic acid building blocks is obtained<br>
upon analysis of Ihe sequences of a set of progenitor nucleic acid templates that serve as a basis for producing a progeny set of finalized chimeric nucleic acid molecules. These progenitor nucleic acid templates thus serve as a source of sequence information that aids in the design of the nucleic acid building blocks that are to be mutagenized, ie. chimerized or shuffled.<br>
(0357]	In one exemplification, the invention provides for the chimerization of a family of<br>
related genes and their encoded family of related products. In a particular exemplification, the encoded products are enzymes. The aldolase, such as pyruvate aldolase, such as HMG and/or KHG aldolase enzymes of the present invention can be mutagenized in accordance with the methods described herein.<br>
[0358]	Thus according to one embodiment of the invention, the sequences of a plurality<br>
of progenitor nucleic acid template;s (such as polynucleotides in accordance with the invention) are aligned in order to select one or more demarcation points, which demarcation points can be located at an area of homology. The demarcation points can be used to delineate the boundaries of nucleic acid building blocks to be generated. Thus, the demarcation points identified and selected in the progenitor molecules serve as potential chimerization points in the assembly of the progeny molecules.<br>
[0359]	In some embodiments, a serviceable demarcation point is an area of homology<br>
(comprised of at least one homologous nucleotide base) shared by at least two progenitor templates, but the demarcation point cjm be an area of homology that is shared by at least half of the progenitor templates, at least two thirds of the progenitor templates, at least<br>
 <br>
three fourths of the progenitor templates and in some embodiments at almost all of the progenitor templates. Even more in tiome embodiments still a serviceable demarcation point is an area of homology that is shared by all of the progenitor templates.<br>
[0360]	In a one embodiment, the gene reassembly process is performed exhaustively in<br>
order to generate an exhaustive library. In other words, all possible ordered combinations of the nucleic acid building blocks are represented in the set of finalized chimeric nucleic acid molecules. At the same time, the assembly order (i.e. the order of assembly of each building block in the 5' to 3 sequence of each finalized chimeric nucleic acid) in each combination is by design (or non-stochastic). Because of the non-stochastic nature of the method, the possibility of unwanted side products is greatly reduced.<br>
[0361]	In other embodiments, the method provides that the gene reassembly process is<br>
performed systematically, for example to generate a systematically compartmentalized library, with compartments that can be screened systematically, such as one by one. In other words the invention provides thai, through the selective and judicious use of specific nucleic acid building blocks, coupled with the selective and judicious use of sequentially stepped assembly reactions, an experimental design can be achieved where specific sets of progeny products are.made in each of several reaction vessels. This allows a systematic examination and screening procedure to be performed. Thus, it allows a potentially very large mimber of progeny molecules to be examined systematically in smaller groups.<br>
[0362]	Because of its ability to perform chimerizations in a manner that is highly flexible<br>
yet exhaustive and systematic as well, particularly when there is a low level of homology among the progenitor molecules, the instant invention provides for the generation of a library (or set) comprised of a large number of progeny molecules. Because of the non-stochastic nature of the instant gene reassembly invention, the progeny molecules generated in some embodiments comprise a library of finalized chimeric nucleic acid molecules having an overall assembly order that is chosen by design. In some embodiments, such a generated library is comprised of greater than 10 to greater than IQIOOO jjiggj-gfi; progeny molecular species.<br>
[0363]	In some embodiments,  a set of finalized  chimeric nucleic acid molecules,<br>
produced as described is comprised of a polynucleotide encoding a polypeptide. In one embodiment, this polynucleotide is a gene, which may be a man-made gene.  In another<br>
 <br>
embodiment, this polynucleotide is a gene pathway, which may be a man-made gene pathway. In some embodiments, the invention provides that one or more man-made genes generated by the invention may be incorporated into a man-made gene pathway, such as pathway operable in a eukaryotic oianism (including a plant).<br>
[0364]	In another exemplification, the synthetic nature of the step in which the building<br>
blocks are generated allows the design and introduction of nucleotides (such as one or more nucleotides, which may be, foi example, codons or introns or regulatory sequences) that can later be optionally removed in an in vitro process (such as by mutagenesis) or in an in vivo process (such as by utilizing the gene splicing ability of a host organism). It is appreciated that in many mstances the introduction of these nucleotides may also be desirable for many other reasons in addition to the potential benefit of creating a serviceable demarcation point.<br>
[0365]	Thus, in some embodiments, the invention provides that a nucleic acid building<br>
block can be used to introduce an intron. Thus, the invention provides that functional introns may be introduced into a man-made gene in accordance with the invention. In some embodiments, the invention also provides that functional introns may be introduced into a man-made gene pathway in accordance with the invention. Accordingly, the invention provides for the generation of a chimeric polynucleotide that is a man-made gene containing one (or more) artificially introduced intion(s).<br>
[0366]	The invention also provides for the generation of a chimeric polynucleotide that is<br>
a man-made gene pathway containing one (or more) artificially introduced intron(s). In some embodiments, the artificially introduced mtion(s) are functional in one or more host cells for gene splicing much in the way that naturally-occurring introns serve timctionally in gene splicing. In some embodiments, the invention provides processes of producing man-made intion-containing poiynucleotides to be introduced into host organisms for recombination and/or splicing.<br>
[0367]	A man-made gene produced using the invention can also serve as a substrate for<br>
recombination with anotiier nucleic acid. Likewise, a man-made gene pathway produced using the invention can also serve as a substrate for recombination with another nucleic acid, til some embodiments, tiie recombination is facilitated by, or occurs at, areas of homology between the man-made, inti-on-containing gene and a nucleic acid, which serves as a recombination partner. In some embodiments, the recombination partner may<br>
 <br>
also be a nucleic acid generated by the invention, including a man-made gene or a man-made gene pathway. Recombination may be facilitated by or may occur at areas of homology that exist at the one (or raorp) artificially introduced intron(s) in the man-made gene.<br>
[0368]	In some embodiments, the synthetic gene reassembly method in accordance with<br>
the invention utilizes a plurality of nucleic acid building blocks, each of which, in some embodiments, has two ligatable ends. The two ligatable ends on each nucleic acid building block may be two blunt ends (i.e. each having an overhang of zero nucleotides), or in some embodiments one blunt end and one overhang, or more in some embodiments still two overhangs. In some embodiment, a useful overhang for this purpose may be a 3' overhang or a 5' overhang. Thus, a nucleic acid building block may have a 3' overhang or alternatively a 5' overhang or alternatively two 3' overhangs or alternatively two 5' overhangs. The overall order in which the nucleic acid building blocks are assembled to form a finalized chimeric nucleic acid molecule is determined by purposeful experimental design and Is not random,<br>
[0369]	In some embodiments, a nucleic acid building block is generated by chemical<br>
synthesis of two single-stranded nucleic acids (also referred to as single-stranded oligos) and contacting them so as to allow them to anneal to form a double-stranded nucleic acid building block. A double-stranded nucleic acid building block can be of variable size. The sizes of these building blocks can be small or large. Exemplary sizes for building block range from 1 base pair (not including any overhangs) to 100,000 base pairs (not mcluding any overhangs). Other exemplaiy size ranges are also provided, which have lower limits of from 1 bp to 10,000 bp (including every integer value in between) and upper limits of from 2 bp to 100, 000 bp (including every inter value in between).<br>
[0370]	Many methods exist by which a doiible-stranded nucleic acid building block can<br>
be generated that is serviceable for the invention; and these are known in the art and can be readily performed by the skilled artisan. In some embodiments, a double-sfranded nucleic acid buildmg block is generated by first generating two single stranded nucleic acids and allowing them to anneal to form a double-stranded nucleic acid building block. The two strands of a double-stranded nucleic acid building block may be complementary at every nucleotide apart from any that form an overhang; thus containing no mismatches, apart from any overhang(s). In anotliei- embodiment, the two strands of a doubie-stranded<br>
 <br>
nucleic acid building block are conijlementary at fewer than every nucleotide apart from any that form an overhang. Thus, ticcording to this embodiment, a double-stranded nucleic acid building block can be used to introduce codon degeneracy. In some embodiments the codon degeneracy h introduced using the site-saturation mutagenesis described herein, using one or more N,N,G/T cassettes or alternatively using one or more N,N,N cassettes.<br>
[0371]	The in vivo recombination method in accordance with the invention can be<br>
performed blindly on a pool of unknovm hybrids or alleles of a specific polynucleotide or sequence. However, it is not necessar/ to know the actual DNA or RNA sequence of the specific polynucleotide. The approach of using recombination within a mixed population of genes can be useful for the generation of any useful proteins, for example, an aldolase in accordance with the invention or a variant thereof. This approach may be used to generate proteins having altered specificity or activity. The approach may also be useful for the generation of hybrid nucleic acid sequences, for example, promoter regions, introns, exons, enhancer sequences, 31 untranslated regions or 51 untranslated regions of genes. Thus this approach may be used to generate genes having increased rates of expression. This approach may also be useful in the study of repetitive DNA sequences. Finally, this approach may be useful to make ribozymes or aptamers in accordance with the invention.<br>
[0372]	In some embodiments the invention described herein is directed to the use of<br>
repeated cycles of reductive reassortment, recombination and selection which allow for the directed molecular evolution of highly complex linear sequences, such as DNA, RNA or proteins thorough recombination.<br>
Optimized Directed Evolution System<br>
[0373]	The   invention  provides   a non-stochastic  gene  modification  system  termed<br>
"optimized directed evolution system" to generate polypeptides, such as aldolase, such as pyruvate aldolase, such as HMG and/or KHG aldolase enzymes or antibodies in accordance with the invention, with new or altered properties. In some embodiments, optimized directed evolution is directed to the use of repeated cycles of reductive reassortment, recombination and selection that allow for the directed molecular evolution of nucleic acids through recombination.<br>
 <br>
[0374]	Optimized directed evolution sJiows generation of a Jarge population of evolved<br>
chimeric sequences, wherein the generated population is significantly enriched for sequences that have a predetermined number of crossover events. A crossover event is a point in a chimeric sequence where a shift in sequence occurs from one parental variant to another parental variant. Such a point is normally at the juncture of where oligonucleotides from two parents are ligated together to form a single sequence. This method allows calculation of the comjct concentrations of oligonucleotide sequences so that the final chimeric population of sequences is enriched for the chosen number of crossover events. This provides more control over choosing chimeric variants having a predetermined number of crossover events.<br>
[0375]	In addition, this method provides convenient means for exploring a tremendous<br>
amoiont of the possible protein variant space in comparison to other systems. Previously, if one generated, for example, lO' chimeric molecules during a reaction, it would be extremely difficult to test such a high number of chimeric variants for a particular activity. Moreover, a significant portion of the progeny population would have a very high number of crossover events which resulted in proteins that were less likely to have increased levels of a particular activity. By using these methods, the population of chimerics molecules can be enriched for those variants that have a palicular number of crossover events. Thus, although one can still generate lO' chimeric molecules during a reaction, each of the molecules chosen for further analysis most likely has, for example, only three crossover events. Because the resulting progeny population can be skewed to have a predetermined number of crossover events, the boundaries on the functional variety between the chimeric molecules is reduced. This provides a more manageable number of variables when calculating which oligonucleotide from the original parental polynucleotides might be responsible for affecting a particular trait.<br>
[0376]	One method for creating a chimeric progeny polynucleotide sequence is to create<br>
oligonucleotides corresponding to fragments or portions of each parental sequence. Each oligonucleotide in some embodiments includes a unique region of overlap so that mixing the oligonucleotides together results in a new variant that has each oligonucleotide fragment assembled in the correct order. Altematively protocols for practicing these methods in accordance with the invention can be found in U.S. Patent Nos. 6,773,900; 6,740,506; 6,713,282; 6,635,449; 6,605,449; 6,537,776; 6,361,974.<br>
 <br>
[0377]	The number of oligonucleotides generated for each parental variant bears a<br>
relationship to the total number of resulting crossovers in the chimeric molecule that is ultimately created. For example, three parental nucleotide sequence variants might be provided to undergo a ligation reaction in order to find a chimeric variant having, for example, greater activity at high temperature. As one example, a set of 50 oligonucleotide sequences can be generated corresponding to each portions of each parental variant. Accordingly, during the ligation reassembly process there could be up to 50 crossover events within each of the chimeric sequences. The probability that each of the generated chimeric polynucleotides will contain oligonucleotides from each parental variant in alternating order is very low. If each oligonucleotide fragment is present in the ligation reaction in the same molar quantity it is likely that in some positions oligonucleotides from the same parental polynucleotide will ligate next to one another and thus not result in a crossover event. If the concentration of each oligonucleotide from each parent is kept constant during any ligation step in this example, there is a 1/3 chance (assuming 3 parents) that an oligonucleotide from the same parental variant will ligate within the chimeric sequence and produce no crossover.<br>
[0378]	Accordingly, a probability density function (PDF) can be determined to predict<br>
the population of crossover events that are likely to occur during each step in a ligation reaction given a set number of parental variants, a number of oligonucleotides corresponding to each variant, and the concentrations of each variant during each step in the ligation reaction. The statistics and mathematics behind determining the PDF is described below. By utilizing these methods, one can calculate such a probability density function, and thus enrich the chimeric progeny population for a predetermined number of crossover events resulting from a particular ligation reaction. Moreover, a target number of crossover events can be predetermined, and the system then programmed to calculate the starting quantities of each parental oligonucleotide during each step in the ligation reaction to result in a probability density function that centers on the predetermined number of crossover events. These methods are directed to the use of repeated cycles of reductive reassortment, recombination and selection that allow for the directed molecular evolution of a nucleic acid encoding a polypeptide through recombination. This system allows generation of a large population of evolved chimeric sequences, wherein the generated population is significantly enriched for sequences that have a predetermined<br>
 <br>
number of crossover events. A crossover event is a point in a chimeric sequence where a shift in sequence occurs from one parental variant to another parental variant. Such a point is normally at the juncture of where oligonucleotides from two parents are ligated together to form a single sequence. The method allows calculation of the correct concentrations of oligonucleotide sequences so that the final chimeric population of sequences is enriched for the chosen ntimber of crossover events. This provides more control over choosing chimeric variants having a predetermined number of crossover events.<br>
[0379J	In addition, these methods provide a convenient means for exploring a tremendous<br>
amotmt of the possible protein variant space in comparison to other systems. By using the methods described herein, the population of chimerics molecules can be enriched for those variants that have a particular number of crossover events. Thus, although one can still generate lO' chimeric molecules during a reaction, each of the molecules chosen for further analysis most likely has, for example, only three crossover events. Because the restating progeny population can be skewed to have a predetermined number of crossover events, the boundaries on the functional variety between the chimeric molecules is reduced. This provides a more manageable number of variables when calculating which oligonucleotide from the original parental polynucleotides might be responsible for affecting a particular trait.<br>
[03801	In some embodiments, the method creates a chimeric progeny polynucleotide<br>
sequence by creating oligonucleotides corresponding to fragments or portions of each parental sequence. Each oligonucleotide in some embodiments includes a unique riion of overlap so that mixmg the oligonucleotides together results in a new variant that has each oligonucleotide fragment assembled in the correct order. See also U.S. Patent Nos. 6,773,900; 6,740,506; 6,713,282; 6,635,449; 6,605,449; 6,537,776; 6,361,974.<br>
Determining Crossover Events<br>
[03811	Embodiments of the invention include a system and software that receive a<br>
desired crossover probability density function (PDF), the number of parent genes to be reassembled, and the number of fragments in the reassembly as inputs. The output of this program is a "fragment PDF" that can be used to determine a recipe for producing reassembled genes, and the estimated crossover PDF of those genes. The processing described herein is in some embodiments performed in MATLAB"" (The Mathworks,<br>
 <br>
Natick, Massachusetts) a programming language and development environment for technical computing.<br>
Iterative Processes<br>
10382]	Any process in accordance witli the invention can be iterativeiy repeated, such as<br>
a nucleic acid encoding an altered or new aldolase phenotype, such as pyruvate aldolase, such as HMG and/or KHG aldolase enzyme in accordance with the invention, can be identified, re-isolated, again modified, re-tested for activity. This process can be iterativeiy repeated until a desired phenotype is engineered. For example, an entire biochemical anabolic or catabolic pathway can be engineered into a cell, including, such as aldolase, such as pyruvate aldoiasie, such as HMG and/or KHG aldolase enzyme, activity.<br>
[0383]	Sunilarly, if it is determined tliat a particular oligonucleotide has no affect at all<br>
on the desired trait (such as a new aldolase, such as pyruvate aldolase, such as HMG and/or KHG aldolase enzyme phenotype), it can be removed as a variable b)' synthesizing larger parental oligonucleotides that include the sequence to be removed. Because incorporating the sequence within a larger sequence prevents any crossover events, there will no longer be any variation of this sequence in the progeny polynucleotides. This iterative practice of determining which oligonucleotides are most related to the desired trait, and which are unrelated, allows more efficient exploration all of the possible protein variants that might be provide a particular trait or activity.<br>
In vivo shuffling<br>
[0384]	In various embodiments, in vivo shuffling of molecules is used in methods in<br>
accordance with the invention to provide variants of polypeptides in accordance with the invention, such as antibodies in accordance with the invention or aldolases in accordance with the invention, stich as pyruvate aldolase, HMG and/or KHG aldolase enzymes, and the like. In vivo shuffling can be performed utilizing the natural property of cells to recombine multimers. While recombination in vivo has provided the major natural route to molecular diversity, genetic recombination remains a relatively complex process that involves 1) the recognition of homologies; 2) strand cleavage, strand invasion, and metabolic steps leading to the production of recombinant chiasma; and fmally 3) the<br>
 <br>
resolution of chiasma into discrete recumbined molecules. The formation of the chiasma requires the recognition of homologous; sequences.<br>
[0385]	In other embodiments, the invention includes a method for producing a hybrid<br>
polynucleotide from at least a first polynucleotide and a second polynucleotide. In some embodiments, the invention can be used to produce a hybrid polynucleotide by introducing at least a first polynucleotide and a second polynucleotide (such as one, or both, being an aldolase, such as pyruvate aldolase, such as HMG and/or KHG aldolase enzyme-encoding sequence in accordance with the invention) which share at least one region of partial sequence homology into a suitable host cell. The regions of partial sequence homology promote processes which result in sequence reorganization producing a hybrid polynucleotide. The term "hybrid polynucleotide", as used herein, is any nucleotide sequence which results from the method of the present invention and contains sequence from at least two origbal polynucleotide sequences. Such hybrid polynucleotides can result from intermolecular recombination events which promote sequence integration between DNA molecules. In addition, such hybrid polynucleotides can result from inttamolecular reductive reassortment processes which utilize repeated sequences to alter a nucleotide sequence within a DNA molecule.<br>
[0386]	In some embodiments, vivo reassortment focuses on "inter-molecular" processes<br>
collectively referred to as "recombination"; which in bacteria, is generally viewed as a "RecA-dependent" phenomenon. In some embodiments, the invention can rely on recombination processes of a host cell to recombine and re-assort sequences, or the cells' ability to mediate reductive processes to decrease the complexity of quasi-repeated sequences in the cell by deletion. This process of "reductive reassortment" occurs by an "intra-molecular", RecA-independent process,<br>
[0387]	In other embodunents of the invention, novel polynucleotides can be generated by<br>
the process of reductive reassortment. The method mvolves the generation of constructs containing consecutive sequences (original encoding sequences), their insertion into an appropriate vector and their subsequent introduction into an appropriate host cell. The reassorttnent of the individual molecular identities occurs by combinatorial processes between the consecutive sequences in the constiTict possessing regions of homolog&gt;', or between quasi-repeated units. The reassortment process recombines and/or reduces the complexity and extent of the repeated sequences and results in the production of novel<br>
 <br>
molecular species. Various treatments may be applied to enhance the rate of reassortment. These could include treatment with ultra-violet light, or DNA damaging chemicals and/or the use of host cell lines displaying enhanced levels of "genetic instability". Thus the reassortment process may involve homologous recombination or the natural property of quasi-repeated sequences to direct their own evolution.<br>
(0388]	Repeated or "quasi-repeated" sequences play a role in genetic instability. In some<br>
embodiments, "quasi-repeats" are repeats that are not restricted to their original imit structure. Quasi-repeated units can be presented as an array of sequences in a construct; consecutive tmits of similar sequences. Once ligated, the junctions between the consecutive sequences become essentially invisible and the quasi-repetitive nature of the resulting construct is now continuous at the molecular level. The deletion process the cell performs to reduce the complexity of the resulting construct operates between the quasi-repeated sequences. The quasi-repeated units provide a practically limitless repertoire of templates upon which slippage events can occur. In some embodiments, the constructs containing the quasi-repeats thus effectively provide sufiicient molecular elasticity that deletion (and potentially insertion) events can occur virttmily anywhere within the quasi-repetitive units.<br>
[0389]	When the quasi-repeated sequences are all ligated in the same orientation, for<br>
instance head to tail or vice versa, the cell cannot distinguish individual units. Consequently, the reductive process can occur throughout the sequences. In contrast, when for example, the units are presented head to head, rather than head to tail, the inversion delineates the endpoints of the adjacent unit so that deletion formation will favor the loss of discrete units. Thus, it is preferable with the present method that the sequences are in the same orientation. Random orientation of quasi-repeated sequences will result in the loss of reassortment efficiency, while consistent orientation of the sequences will offer the highest efficiency. However, while having fewer of the contiguous sequences in the same orientation decreases the efficiency, it may still provide sufficient elasticity for the effective recovery of novel molecules. Constructs can be made with the quasi-repeated sequences in the same orientation to allow higher efficiency.<br>
[0390]	Sequences can be assembled in a head to tail orientation using any of a variety of<br>
methods, including the following-<br>
 <br>
a)	Primers that include a poly-A head and poly-T tail which when made single-stranded would provide orienmtion can be utilized. This is accomplished by having the first few bases of the primers made from RNA and hence easily removed RNaseH.<br>
b)	Primers that include unique restriction cleavage sites can be utilized. Multiple sites, a battery of unique sequences and repeated synthesis and ligation steps would be required.<br>
c)	The inner few bases of the primer could be thiolated and an exonuclease used to produce properly tailed molecules.<br>
[0391]	In some embodiments, the recovery of the re-assorted sequences relies on the<br>
identification of cloning vectors with a reduced repetitive index (RI). The re-assorted encoding sequences can then be recovered by amplification. The products are re-cloned and expressed. The recovery of cloning vectors with reduced RI can be affected by:<br>
1)	The use of vectors only stably maintained when the construct is reduced in complexity.<br>
2)	The physical recover&gt;' of shortened vectors by physical procedures. In this case, tlie cloning vector would be recovered using standard plasmid isolation procedures and size fractionated on either an agarose gel, or column with a low molecular weight cut off utilizing standard procedures.<br>
3)	The recovery of vectors containing interrupted genes which can be selected when insert size decreases.<br>
4)	The use of direct selection techniques with an expression vector and the appropriate selection.<br>
[0392]	Encoding   sequences   (for   example,   genes)   from   related   organisms   may<br>
demonstrate a high degree of homology and encode quite diverse protein products. These types of sequences are particularly useful in the present invention as quasi-repeats. However, while the examples illustrated below demonstrate the reassortment of nearly identical original encoding sequences (quasi-repeats), this process is not limited to such nearly identical repeats.<br>
10393]	The following example demonstrates an exemplary method in accordance with the<br>
invention. Encoding nucleic acid sequences (quasi-repeats) derived from three (3) unique species are described.  Each sequence encodes a protein with a distinct set of properties.<br>
 <br>
Each of the sequences differs by a single or a few base pairs at a unique position in the sequence. The qimsi-repeated sequences are separately or coilectiveiy amplified and ligated into random assemblies such that all possible permutations and combinations are available in the population of ligated molecules. The number of quasi-repeat units can be controlied by the assembly conditions. The average number of quasi-repeated units in a construct is defined as the repetitive index (RI).<br>
[0394]	Once formed, the constructs may, or may not be size fractionated on an agarose<br>
gel according to published protocols, uiserted into a cloning vector and transfected into an appropriate host cell. The cells are then propagated and "reductive reassortment" is effected. The rate of the reductive reassortment process may be stimulated by the introduction of DNA damage if desired. Whether the reduction in RI is mediated by deletion formation between repeated sequences by an "intra-molecular" mechanism, or mediated by recombination-like events through "inter-molecular" mechanisms is immaterial. The end result is a reassortment of the molecules into all possible combinations.<br>
[0395]	Optionally, the method comprises the additional step of screening the library<br>
members of the shuffled pool to identify individual shuffled library members having the ability to bind or otherwise interact, or catalyze a particular reaction (such as catalytic domain of an enzyme) with a predetermined macromolecule, such as for example a proteinaceous receptor, an oligosaccharide, virion, or other predetermined compound or structure.<br>
[0396]	The  polypeptides that  are  identified  from  such  libraries  can  be  used  for<br>
therapeutic, diagnostic, research and related purposes (such as catalysts, solutes for increasing osmolarity of an aqueoi solution and the like) and/or can be subjected to one or more additional cycles of shuffling and/or selection.<br>
[0397]	In other embodiments, it is envisioned that prior to or during recombination or<br>
reassortment, polynucleotides generated by the method in accordance with the invention can be subjected to agents or processes which promote the introduction of mutations into the original polynucleotides. The introduction of such mutations would increase the diversity of resulting hybrid polynucleotides and polypeptides encoded therefrom. The agents or processes which promote mutagenesis can include, but are not limited to; (+)-CC-1065, or a synthetic analog such as (+)-CC-1065-(N3-Adenine (See Sun and Huriey,<br>
 <br>
(1992); an N-acetylated or deacetylated 4'-fluro-4-aminobiphenyl adduct capable of<br>
inhibiting DNA synthesis {See , for ex;3mpJe, -van de Pol) e/ al (1992)); or a N-acetyJaled<br>
or deacetylated 4-aminobiphenyl adduct capable of inhibiting DNA synthesis (See also,<br>
van de Poll et al (1992), pp. 751-758); trivalent chromium, a trivalent chromium salt, a<br>
polycyclic aromatic hydrocarbon (PAH) DNA adduct capable of inhibiting DNA<br>
replication, such as 7-bromomethyl-benz[a]anthracene ("BMA"), tris(2,3-<br>
dibromopropyl)phosphate ("Tris-BP"), l,2-dibromo-3-chloropropane ("DHCP"), 2-<br>
bromoacrolein (2BA), benzo[a]pyrene-7,8-dihydrodiol-9-10-epoxide ("BPDE"), a<br>
platinimi(II) halogen salt, N-hydrDxy-2-aniino-3-methy]imidazo[4,5-/]-quino]ine ("N-<br>
hydroxy-lQ") and N-hydroxy-2-amino-l-methyl-6-phenylimidazo[4,5-/|-pyridine ("N-<br>
hydroxy-PhlP"). Exemplary means for slowing or halting PCR amplification consist of<br>
UV light (+)-CC-1065 and (+)-CC-1065-(N3-Adenine). Particularly encompassed means<br>
are DNA adducts or polynucleotides comprising the DNA adducts from the<br>
polynucleotides or polynucleotides pool, which can be released or removed by a process<br>
including heating the solution comprising the polynucleotides prior to further processing.<br>
|0398]	In other embodiments the invention is directed to a method of producing<br>
recombinant proteins having biological activity by treating a sample comprising double-stranded template polynucleotides encoding a wild-type protein under conditions according to the invention which provide for the production of hybrid or re-assorted polynucleotides.<br>
Producing sequence variants<br>
[0399]	The invention also provides additional methods for making sequence variants of<br>
the nucleic acid (such as aldolase, such as pyruvate aldolase, such as HMG and/or KHG aldolase enzyme) sequences in accordance with the invention. In some embodiments, the invention also provides additional methods for isolating aldolase, such as pyruvate aldolase, such as HMG and/or KHG aldolase enzymes using the nucleic acids and polypeptides in accordance with the invention: In some embodiments, the invention provides for variants of an aldolase, such as pyruvate aldolase, HMG and/or KHG aldolase enzyme coding sequence (such as a gene, cDNA or message) in accordance with the invention, which can be altered by any means, including, such as random or stochastic methods, or, non-stochastic, or "directed evolution," methods, as described above.<br>
 <br>
[0400]	The isolated variants may be naturally occurring.   Variant can also be created in<br>
vitro. Variants may be created using genetic engineering techniques such as site directed mutagenesis, random chemical mulagenesis, Exonuclease III deletion procedures, and standard cloning techniques. Alternatively, such variants, fragments, analogs, or derivatives may be created using chemical synthesis or modification procedures. Other methods of making variants are also familiar to those skilled in the art. These include procedures in which nucleic acid sequences obtained from natural isolates are modified to generate nucleic acids which encode polypeptides having characteristics which enhance their value in industrial or laboratorj- applications. In such procedures, a large number of variant sequences having one or more nucleotide differences with respect to the sequence obtained from the natural isolate are generated and characterized. These nucleotide differences can result in amino acid changes with respect to the polypeptides encoded by the nucleic acids from the natural isolates.<br>
[0401]	For example,  variants may be  created  using  error prone  PCR.     In  some<br>
embodiments of error prone PCR, the PCR is performed under conditions where the copying fidelity of the DNA polymerase is low, such that a high rate of point mutations is obtained along the entire length of the PCR product. Error prone PCR is described, such as in Leung, D.W. el al, (1989) Technique 1:11-15; and Caldwell, R.C. &amp; Joyce, G.F., (1992) PCR Methods Applic. 2:28-33. Briefly, in such procedures, nucleic acids to be mutagenized aie mixed with PCR primers, reaction buffer, MgCl2, MnCh, Taq polymerase and an appropriate concentration of dNTPs for achieving a high rate of point mutation along the entire length of the PCR product. For example, the reaction may be performed using 20 fmoles of nucleic acid to be mutagenized, 30 pmole of each PCR primer, a reaction buffer comprising 50mM KCl, lOmM Tris HCl (pH 8.3) and 0.01% gelatin, 7mM MgC12, 0.5mM MnCla, 5 units of Taq polymerase, 0.2mM dGTP, 0.2mM dATP, ImM dCTP, and ImM dTTP. PCR may be performed for 30 cycles of 94°C for 1 minute, 45°C for 1 minute, and 72°C for 1 minute. However, it will be appreciated that these parameters may be varied as appropriate. The mutagenized nucleic acids are cloned into an appropriate vector and the activities of the polypeptides encoded by the mutagenized nucleic acids are evaluated.<br>
[0402]	In  some  embodiments,   variants  are   created  using  oligonucleotide  directed<br>
mutagenesis  to  generate   site-specific   mutations  in  any  cloned  DNA  of interest.<br>
 <br>
Oligonucleotide mutagenesis is described, such as in Reidhaar-Olson (1988) Science 241:53-57. Briefly, in such procedures a plurality of double stranded oligonucleotides bearing one or more mutations to be introduced into the cloned DNA are synthesized and inserted into the cloned DNA to be mutagenized. In some embodiments, clones containing the mutagenized DNA are recovered, expressed, and the activities of the polypeptide encoded therein assessed.<br>
[0403]	Another method for generatiJig variants is assembly PCR.     Assembly PCR<br>
involves the assembly of a PCR product from a mixture of small DNA fragments. A large number of different PCR reactions occur in parallel in the same vial, with the products of one reaction priming the products of another reaction. Assembly PCR is described in the art, such as in U.S. Patent No. 5,965,408.<br>
[0404]	In some embodiments, sexual PCR mutagenesis is an exemplary method of<br>
generating variants in accordance with the invention. In some embodiments of sexual PCR mutagenesis forced homologous recombination occurs between DNA molecules of different but highly related DNA sequence in vitro, as a result of random fragmentation of the DNA molecule based on seqtience homology, followed by fixation of the crossover by primer extension in a PCR reaction. Sexual PCR mutagenesis is described, such as in Stemmer (1994) Proc. Natl. Acad. Sci. USA 91:10747-10751. Briefly, in such procedures a plurality of nucleic acids to be recombined are digested with DNase to generate fragments having an average size of 50-200 nucleotides. Fragments of the desired average size are purified and resuspended in a PCR mixture. PCR is conducted under conditions which facilitate recombination between the nucleic acid fragments. For example, PCR may be performed by resuspendk the purified fragments at a concentration of 10-30ng/|jJ in a solution of 0.2mM of each dNTP, 2.2mM MgCli, 50mM KCL, lOmM Tris HCl, pH 9.0, and 0.1% Triton X-100. 2.5 units of Taq polymerase per 100 \il of reaction mixture is added and PCR is performed using the following regime: 94°C for 60 seconds, 94°C for 30 seconds, 50-55°C for 30 seconds, 72°C for 30 seconds (30-45 times) and 72°C for 5 minutes. However, it will be appreciated that these parameters may be varied as appropriate. In some embodiments, oligonucleotides may be included m the PCR reactions. In other embodiments, the Klenow fragment of DNA polymerase I may be ied in a first set of PCR reactions and Taq polymerase may be used<br>
 <br>
in a subsequent set of PCR reactions. Recombinant sequences are isolated and the activities of the polypeptides they encode are assessed.<br>
[0405]	In some embodiments, varianis are created by in vivo mutagenesis.    In some<br>
embodiments, random mutations in a sequence of interest are generated by propagating the sequence of interest in a bacterial strain, such as an E. coli strain, which carries mutations in one or more of the DNA repair pathways. Such "mutator" strains have a liigher random mutation rate than that of a wild-type parent. Propagating the DNA in one of these strains will eventually generate random mutations within the DNA. Mutator strains suitable for use for in vivo mutagenesis are described in PCT Publication No. WQ 91/16427, published October 31, 1991, entitled "Methods for Phenotype Creation from Multiple Gene Populations".<br>
[0406]	Variants  may  also  be   generated  using   cassette  mutagenesis.     In  cassette<br>
mutagenesis a small region of a double stranded DNA molecule is replaced with a synthetic oligonucleotide "cassette" that differs from the native sequence. The oligonucleotide often contains completely and/or partially randomi2ed native sequence.<br>
[0407)	Recurei-i'e ensemble mutagenesis may also be used to generate variants. Recursive<br>
ensemble mutagenesis is an algorithm for protein engineering (protein mutagenesis) developed to produce diverse populations of phenotypically related mutants whose members differ in amino acid sequence. This method uses a feedback mechanism to control successive rounds of combinatorial cassette mutagenesis. Recursive ensemble mutagenesis is described, such as in Arkin (1992) Proc. Natl. Acad. Sci. USA 89:7811-7815.<br>
[0408]	In   some   embodiments,   variants   are   created   using   exponential   ensemble<br>
mutagenesis. Exponential ensemble mutagenesis is a process for generating combinatorial libraries with a high percentage of unique and functional mutants, wherein small groups of residues are randomized in parallel to identify, at each altered position, amino acids which lead to functional proteins. Exponential ensemble mutagenesis is described, such as in Deiegrave (1993) Biotechnology Res. 11:1548-1552. Random and site-directed mutagenesis are described, such as in Arnold (1993) Current Opinion in Biotechnology 4:450-455.<br>
[0409]	In some embodiments, the variants are created using shuffling procedures wherein<br>
portions of a plurality of nucleic acids which encode distinct polypeptides are fused<br>
 <br>
together to create chimeric nucleic acid sequences which encode chimeric polypeptides as described in U.S. Patent No, 5,965,408, filed July 9, 1996, entitled, "Method of DNA Reassembly by Interrupting Synthesis" and U.S. Patent No. 5,939,250, filed May 22, 1996, entitled, "Production of Enzymes Having Desired Activities by Mutagenesis,<br>
[0410]	The variants of the polypeptides in accordance with the invention may be variants<br>
in which one or more of the amino acid residues of the polypeptides of the sequences in accordance with the invention are substituted with a conserved or non-conserved amino acid residue (in some embodiments, a conserved amino acid residue) and such substituted amino acid residue may or may not be one encoded by the genetic code.<br>
[0411]	In some embodiments, conservative substitutions are those that substitute a given<br>
amino acid in a polypeptide by another amino acid of like characteristics. In some embodiments, conservative substitutions in accordance with the invention comprise the following replacements: replacements of an aliphatic amino acid such as Alanine, Valine, Leucine and Isoleucine with another aliphatic amino acid; replacement of a Serine with a Threonine or vice versa; replacement of an acidic residue such as Aspartic acid and Glutamic acid with another acidic residue; replacement of a residue bearing an amide group, such as Asparagine and Glutamine, with another residue bearing an amide group; exchange of a basic residue such as Lysine and Arginine with another basic residue; and replacement of an aromatic residue such as Phenylalanine, Tyrosine with another aromatic residue.<br>
[0412]	Other variants are those in which one or more of the amino acid residues of a<br>
polypeptide in accordance with the invention includes a substituent group. In some embodiments, other variants are those in which the polypeptide is associated with another compound, such as a compound to increase the half-life of the polypeptide (for example, polyethylene glycol). Additional variants are those m which additional amino acids are fused to the polypeptide, such as a leader sequence, a secretory sequence, a proprotein sequence or a sequence which facilitates purification, enrichment, or stabilization of the polypeptide.<br>
[0413]	In some embodiments, the fragments, derivatives and analogs retain the same<br>
biological function or activity as the jjolypeptides in accordance with the invention and sequences substantially identical thereto. In other embodiments, the fragment, derivative.<br>
 <br>
or analog includes a proprotein, such that the fragment, derivative, or analog can be activated by cleavage of the proprotein portion to produce an active polypeptide.<br>
Optimizing codons to achieve high levels of protein expression in host cells<br>
[0414]	The  invention  provides  metl-iods  for modifying  aldolase,  such  as  pyruvate<br>
aldolase, such as HMG and/or KHG aldolase, enzyme-encoding nucleic acids to modify (such as optimize) codon usage. In some embodiments, the invention provides methods for modifying codons in a nucleic aciii encoding an aldolase, such as pyruvate aldolase, HMG and/or KHG aldolase enzyme to increase or decrease its expression in a host cell. In some embodiments, the invention also provides nucleic acids encoding an aldolase, such as pyruvate aldolase, HMG and/or KHG aldolase enzyme modified to increase its expression in a host cell, aldolase, such as pyruvate aldolase, such as HMG and/or KHG aldolase enzyme so modified, and methods of making the modified aldolase, such as pyruvate aldolase, such as HMG and/or KHG aldolase enzymes. The method comprises identifying a "non-preferred" or a "less preferred" codon in aldolase, such as pyruvate aldolase, such as HMG and/or KHG aldolase, enzyme-encoding nucleic acid and replacing one or more of these non- preferred or less preferred codons with a "preferred codon" encoding the same amino acid as the replaced codon and at least one non-preferred or less preferred codon in the nucleic acid has been replaced by a preferred codon encoding the same amino acid. A preferred codon is a codon over-represented in coding sequences in genes in the host cell and a non- preferred or less preferred codon is a codon under-represented in coding sequences in genes in the host cell.<br>
[0415]	Host cells for expressing the nucleic acids, expression cassettes and vectors in<br>
accordance with the invention include bacteria, yeast, fimgi, plant cells, insect cells and mammalian cells (see discussion, above). Thus, the invention provides methods for optimizing codon usage in all of these cells, codon-altered nucleic acids and polypeptides made by the codon-altered nucleic acids. Exemplary host cells include gram negative bacteria, such as Escherichia coli; gram positive bacteria, such as Streptomyces sp., Lactobacillus gasseri, Lactococcus lactis, Lactococcus cremoris. Bacillus subtilis. Bacillus cereus. Exemplary host cells also include eukaryotic organisms, such as various yeast, such as Saccharomyces sp,, including Saccharomyces cerevisiae, Schizosaccharomyces pombe, Pichia pastoris, and Kluyveromyces lactis, Hansenula polymorpha, Aspergillus niger, and mammalian cells and cell lines and insect cells and<br>
 <br>
cell lines. Thus, the invention also includes nucleic acids and polypeptides optimized for<br>
expression in these organisms and species.<br>
[0416]	For example, the codons of a nucleic acid encoding an aldolase, such as pyruvate<br>
aldolase, HMG and/or KHG aldolase inzyme isolated from a bacterial cell are modified such that the nucleic acid is optimally expressed in a bacterial cell different from the bacteria from which the aldolase, such as pyruvate aldolase, such as HMG and/or KHG aldolase enzyme was derived, a yeast, a fungi, a plant cell, an insect cell or a mammalian cell. Methods for optimizing codom; are well known in the art, see U.S. Patent No. 5,795,737; Baca (2000) Int. J. Parasitol. 30:113-118; Hale (1998) Protein Expr. Purif. 12:185-188; Narum (2001) Infect. Imniun. 69:7250-7253. See also Narum (2001) Infect. Irnmun. 69:7250-7253, describing optimizing codons in mouse systems; Outchkourov (2002) Protein Expr. Purif 24:18-24, describing optimizing codons in yeast; Feng (2000) Biochemistry 39:15399-15409, describing optimizing codons in E. coli; Humphreys (2000) Protein Expr. Purif 20:252-264, describing optimizing codon tage that affects secretion in E. coli.<br>
Transgenic non-htiman animals<br>
[0417J	The invention provides transgenic non-human animals comprising a nucleic acid,<br>
a polypeptide (such as an aldolase, such as pyruvate aldolase, HMG and/or KHG aldolase enzyme), an expression cassette or vector or a transfected or transformed cell in accordance with the invention, hi some embodiments, the invention also provides methods of making and using these transgenic non-human animals.<br>
[0418]	The transgenic non-human Einimals can be, such as dogs, goats, rabbits, sheep,<br>
horses, fish, pigs (including all swine, hogs and related animals), cows, rats and mice, comprising the nucleic acids in accordance with the mvention. These animals can be used, such as in vivo models to study aldolase, such as pyruvate aldolase, such as HMG and/or KHG aldolase enzyme, activity, or, as models to screen for agents that change the aldolase, such as pyruvate aldolase, such as HMG and/or KHG aldolase enzyme, activity in vivo. The coding sequences for the polypeptides to be expressed in the transgenic non-human animals can be designed to be constitutive, or, under the control of tissue-specific, developmental-specific or inducible transcriptional regulatory factors.<br>
[0419]	Transgenic non-himian animals can be designed and generated using any method<br>
known in the art; see U.S. Patent Nos. 6,211,428; 6,187,992; 6,156,952; 6,118,044;<br>
 <br>
6,111,166; 6,107,541; 5,959,171; 5,92:,854; 5,892,070; 5,880,327; 5,891,698; 5,639,940;<br>
5,573,933; 5,387,742; 5,087,571, describing making and using transformed cells and eggs<br>
and transgenic mice, rats, rabbits, sheep, pigs, chickens, goats, fish and cows. See also,<br>
such as Pollock (1999) J. Immunol. Methods 231:147-157, describing the production of<br>
recombinant proteins in the milk of transgenic dairy animals; Baguisi (1999) Nat.<br>
Biotechnol. 17:456-461, demonstrating the production of transgenic goats. U.S. Patent<br>
No. 6,211,428, describes making and using transgenic non-human mammals which<br>
express in their brains a nucleic acid construct comprising a DNA sequence. U.S. Patent<br>
No. 5,387,742, describes injecting cloned recombinant or synthetic DNA sequences into<br>
fertilized mouse eggs, implanting the injected eggs in pseudo-pregnant females, and<br>
growing to term transgenic mice. U.S. Patent No. 6,187,992, describes making and using<br>
a transgenic mouse.<br>
[0420]	"Knockout animals" can also be used to practice the methods in accordance with<br>
the invention. For example, in some embodiments, the transgenic or modified animals in accordance with the invention comprise a "knockout animal," such as a "knockout mouse," engineered not to express an endogenous gene, which is replaced with a gene expressing an aldolase, such as pyruvate aldolase, HMG and/or KHG aldolase enzyme in accordance with the invention, or, a fusion protein comprising an aldolase, such as pyruvate aldolase, HMG and/or KHG aldolase enzyme in accordance with the invention.<br>
Transgenic Plants and Seeds<br>
[0421]	The invention provides transgenic plants and seeds comprising a nucleic acid, a<br>
polypeptide (such as an aldolase, such as pyruvate aldolase, HMG and/or KHG aldolase enzyme), an expression cassette or vector or a transfected or transformed cell in accordance with the invention. The invention also provides plant products or byproducts, such as fiiiits, oils, seeds, leaves, extracts and the like, including any plant part, comprising a nucleic acid and/or a polypeptide (such as a xylanase) of the invention, such as wherein the nucleic acid or polypeptide of the invention is heterologous to the plant, plant part, seed etc. The transgenic plant (which includes plant parts, fruits, seeds etc.) can be dicotyledonous (a dicot) or monocotyledonous (a monocot). In some embodiments, the invention also provides methods of making and using these transgenic plants and seeds.   The transgenic pL-mt or plant cell expressing a polypeptide of the<br>
 <br>
present invention may be constructed in accordance with any method known in the art. See, for example, U.S. Patent No. 6,309,872.<br>
[0422]	Nucleic acids and expression constructs in accordance with the invention can be<br>
introduced into a plant cell by any means. For example, nucleic acids or expression constructs can be introduced into the genome of a desired plant host, or, the nucleic acids or expression constructs can be epistim.es. Introduction into the genome of a desired plant can be such that the host's aldolase, such as pyruvate aldole, such as HMG and/or KHG aldolase enzyme production is regulated by endogenous transcriptional or translational control elements. In some embodiments, the invention also provides "knockout plants" where insertion of gene sequence by, such as homologous recombination, has disrupted the expression of the endogenous gene. Means to generate "knockout" plants are well-known in the art, see Strepp (1998) Proc Natl. Acad. Sci. USA 95:4368-4373; Miao (1995) Plant J 7;359-365. See discussion on transgenic plants, below.<br>
[0423]	The nucleic acids in accordance with the invention can be used to confer desired<br>
traits on essentially any plant, such as on starch-producing plants, such as potato, tomato, soybean, beets, com, wheat, rice, barley, and the like. Nucleic acids in accordance with the invention can be used to manipulate metabolic pathways of a plant in order to optimize or alter host's expression of aldolase, such as pyravate aldolase, such as HMG and/or KHG aldolase enzyme. The nucleic acids in accordance with the invention can change expression or activity levels or alter characteristics of compounds or enzymes naturally produced in a plant. Alternatively, an aldolase, such as pyruvate aldolase, HMG and/or KHG aldolase enone in accordance with the invention can be used in production of a transgenic plant to produce a compound not naturally produced by that plant. This can lower production costs or create a novel product.<br>
[0424J	In some embodiments, the first step in production of a transgenic plant involves<br>
making an expression consfruct for expression in a plant cell. These techniques are well known in the art. They can include selecting and cloning a promoter, a coding sequence for facilitating efficient binding of ribosomes to mRNA and selecting the appropriate gene terminator sequences. One exemplary constitutive promoter is CaMV35S, from the cauliflower mosaic virus, which genenUly results in a high degree of expression in plants. Other promoters are more specific and respond to cues in the plant's internal or external<br>
 <br>
environment. An exemplary light-inducible promoter is the promoter from the cab gene, encoding the major chlorophyll a/b binding protein.<br>
i0425]	,   In some embodiments, the nucleic acid is modified to achieve greater expression<br>
in a plant cell. For example, a sequence in accordance with the invention is likely to have a higher percentage of A-T nucleotide pain; compared to that seen in a plant, some of which prefer G-C nucleotide pau-s. Therefore, A-T nucleotides in the coding sequence can be substituted with G-C nucleotides without significantly changing the amino acid sequence to enhance production of the gene product in plant cells,<br>
[0426]	Selectable marker gene can be added to the gene construct in order to identify<br>
plant cells or tissues that have successfully integrated the transgene. This may be necessary because achieving incorporation and expression of genes in plant cells is a rare event, occurring in jixst a few percent of the targeted tissues or cells. Selectable marker genes encode proteins that provide resistance to agents that are normally toxic to plants, such as antibiotics or herbicides. Only plant cells that have integrated the selectable marker gene will survive when grown on a medium containing the appropriate antibiotic or herbicide. As for other inserted genes, marker genes also require promoter and termination sequences for proper function.<br>
[0427]	In some embodiments, making transgenic plants or seeds comprises incorporating<br>
sequences in accordance with the invention and, optionally, marker genes into a target expression construct (such as a plasmid), along with positioning of the promoter and the terminator sequences. This can involve transferring the modified gene into the plant through a suitable method. For example, a construct may be introduced directly into the genomic DNA of the plant cell using techniques such as electroporation and microinjection of plant cell protoplasts, or the constructs can be introduced directly to plant tissue using ballistic methods, such as DNA particle bombardment. For example, see Christou (1997) Plant Mol. Biol. 35:197-203; Pawlowski (1996) Mol. Biotechnol. 6:17-30; Klein (1987) Nature 327:70-73; Takumi (1997) Genes Genet. Syst. 72:63-69, discussing use of particle bombardment to introduce Iransgenes into wheat; and Adam (1997) supra, for use of particle bombardment to introduce YACs into plant cells. For example, Rinehart (1997) supra, used particle bombardment to generate transgenic cotton plants. Apparatus for accelerating particles is described U.S. Pat. No. 5,015,580; and, the commercially available Bio-Rad (Biolistics) PDS-2000 particle acceleration instrument<br>
 <br>
(Bio-Rad, Hercules, CA); see also, John, U.S. Patent No. 5,608,148; and Ellis, U.S. Patent No. 5, 681,730, describing particle-mediated transformation of gymnosperms.<br>
10428]	In some embodiments, protoplasts can be immobilized and injected with a nucleic<br>
acids, such as an expression construct. Although plant regeneration from protoplasts is not easy with cereals, plant regeneration is possible in legumes using somatic embryogenesis from protoplast derived callus. Organized tissues can be transfonned with naked DNA using gene gun technique, where DNA is coated on tungsten microprojectiles, shot 1/lOOth the size of cells, which carry the DNA deep into cells and organelles. Transformed tissue is tlien induced to regenerate, usuall}' by somatic embryogenesis. This technique has been successful in several cereal species including maize and rice.<br>
[0429]	Nucleic acids, such as expression constructs, can also be introduced into plant<br>
cells using recombinant viruses. Plant cells can be transformed using viral vectors, such as tobacco mosaic virus derived vectors (Rouwendal(I997)PlantMol. Biol. 33:989-999), see Porta (1996) "Use of viral replicons for the expression of genes in plants," Mol. Biotechnol. 5:209-221.<br>
[0430]	Alternatively, nucleic acids, such as an expression construct, can be combined<br>
with suitable T-DNA flanking regions and introduced into a conventional Agi-obacierium tumefaciens host vector. The virulence fiinctions of the Agrobacterium tumefaciens host will direct the insertion of the construct and adjacent marker into the plant cell DNA when the cell is infected by the bacteria. Agrobacterium tumefaciens-m&amp;diatcd fransformation techniques, including disarming and use of binary vectors, are well described in the scientific literature. See Horsch (1984) Science 233:496-498; Fraley (1983) Proc. Natl. Acad ScL USA 80:4803 (1983); Gene Transfer to Plants, Potrykus, ed. (Springer-Verlag, Berlm 1995). The DNA in an A tumefaciens cell is contained in the bacterial chromosome as well as in another structure known as a Ti (tumor-inducir) plasmid. The Ti plasmid contains a stretch of DNA termed T-DNA (-20 kb long) that is fransferred to the plant cell in the infection process and a series of vir (virulence) genes that direct the infection process. A. tumefaciens can only infect a plant through wounds: when a plant root or stem is wounded it gives off certain chemical signals, in response to which, the vir genes of A. tumefaciens become activated and direct a series of events necessary for the transfer of the T-DNA from the Ti plasmid to the plant's chromosome.<br>
 <br>
The T-DNA then enters the plant cell through the wound. One speculation is that the T-DNA waits until the plant DNA is being replicated or transcribed, then inserte itself into the exposed plant DNA. In order to use A. tumefaciens as a transgene vector the tumor-inducing section of T-DNA have to be removed, while retaining the T-DNA border regions and the vir genes. The transgene is then inserted between the T-DNA border regions, where it is transferred to the plant cell and becomes integrated into the plant's chromosomes.<br>
[0431]	The invention provides for the transformation of monocotyledonous plants using<br>
the nucleic acids in accordance with tae invention, including important cereals, see Hiei (1997) Plant Mol. Biol. 35:205-218. See also, Horsch, Science (1984) 233:496; Fraley (1983) Proc. Natl. Acad. Sci USA 80:4803; Thykjaer (1997) supra; Park (1996) Plant Mol. Biol. 32:1135-1148, discussing T-DNA integration into genomic DNA. See also D'Halluin, U.S. Patent No. 5,712,135, describing a process for the stable integration of a DNA comprising a gene that is functional in a cell of a cereal, or other monocotyledonous plant.<br>
(0432}	In some embodiments, the third step involves selection and regeneration of whole<br>
plants capable of transmitting the incorporated target gene to the next generation. Such regeneration techniques may use manipulation of certain phytohormones in a tissue culture growth medium. In some embodiments, the method uses a biocide and/or herbicide marker that has been introduced togeiiier with the desired nucleotide sequences. Plant regeneration from cultured protoplasts is described in Evans et al., Protoplasts Isolation and Culture, Handbook of Plant Cell Culture, pp. 124-176, MacMiUilan Publishing Company, New York, 1983," and Binding, Regeneration of Plants, Plant Protoplasts, pp. 21-73, CRC Press, Boca Ratoi 1985. Regeneration can also be obtained from plant callus, explants, organs, or parts thereof. Such regeneration techniques are described generally in Klee (1987) /Jin. Rev. of P]ant Phys. 38:467-486. To obtain whole plants from transgenic tissues such as immature embryos, they can be grown under controlled environmental conditions in a series of media containing nutrients and hormones, a process known as tissue culture. Once whole plants are generated and produce seed, evaluation of the progeny begins.<br>
[0433]	In some embodiments, after the expression cassette is stably incorporated in<br>
transgenic plants, it can be introduced into other plants by sexual crossing.   Any of a<br>
 <br>
number of standard breeding techniques can be used, depending upon the species to be crossed. Because transgenic expression of the nucleic acids in accordance with the invention leads to phenotypic changes, plants comprising the recombinant nucleic acids in accordance with the invention can be sexually crossed with a second plant to obtain a final product. Thus, the seed in accordance with the invention can be derived from a cross between two transgenic plants in accordance with the invention, or a cross between a plant in accordance with the invention and another plant. The desired effects (such £« expression of the polypeptides in accordance with the invention to produce a plant in which flowering behavior is altered) can be enhanced when both parental plants express the polypeptides (such as an aldolase, such as pyruvate aldolase. HMG and/or KHG aldolase enzjone) in accordance with the invention. The desired effects can be passed to future plant generations by standard propagation means.<br>
[04341	In some embodiments, the nucleic acids and polypeptides in accordance with the<br>
invention are expressed in or inserted in any plant or seed. Transgenic plants in accordance with the invention can be dicotyledonous or monocotyledonous. Examples of monocot transgenic plants in accordance with the invention are grasses, such as meadow grass (blue grass. Pod), forage gratis such as festuca, loiium, temperate grass, such as Agrostis, and cereals, such as wheat, oats, rye, barley, rice, sorghum, and maize (com). Examples of dicot transgenic plants in accordance with the invention are tobacco, legumes, such as lupins, potato, sugar beet, pea, bean and soybean, and cruciferous plants (family Brassicaceae), such as cauliflower, rape seed, and the closely related model organism Arabidopsis thaliana. Thus, the transgenic plants and seeds in accordance with the in'V'ention include a broad range of plants, including, but not limited to, species from the genera Anacardium, Arachis, Asparagus, Atropa, Avena, Brassica, Citrus, Citrullus, Capsicum, Carthamus, Cocos, Coffea, Cucumis, Cucurbita, Daucus, Elaeis, Fragaria, Glycine, Gossypium, Helianthus, Heterocallis, Hordeum, Hyoscyamus, Lactuca, Linum, Loiium, Lupinus, Lycopersicon, Malm, Manihot, Majorana, Medicago, Nicotiana, Olea, Oryza, Panieum, Pannisetum, Persea, Phaseolus, Pistachia, Pisum, Pyrus, Prunus, Raphanus, Ricinus, Secale, Senecio, Sinapis, Solarium, Sorghum, Theobromus, TrigoneUa, Triticum, Vicia, Vitis, Vigna, and Zea.<br>
[0435]	In alternative embodiments, the nucleic acids in accordance with the invention are<br>
expressed in plants which contain fiber cells, including, such as cotton, silk cotton tree<br>
 <br>
(Kapok, Ceiba pentandra), desert willow, creosote bush, winterfat, bajsa, ramie, kenaf, hemp, roseile, jute, sisal abaca and flax. In alternative embodiments, the transgenic plants in accordance with the invention can be members of the genus Gossypium, including membera of any Gossypium species, such as G. arboreum; G. herbaceum, G. barbadense, and G, hirmtum.<br>
[0436]	The invention also provides for transgenic plants to be used for producing large<br>
amounts of the polypeptides (such as an aldolase, such as pyruvate aldolase, HMG and/or KHG aldolase enzyme or antibody) in accordance with the invention. For example, see Palmgren (1997) Trends Genet. 13:348; Chong (1997) Transgenic Res. 6:289-296 (producing human milk protein beta-casein in transgenic potato plants using an auxin-inducible, bidirectional mannopine synthase (masr,2') promoter with Agrobacterium tumefaciens-me.iiiaXe&amp; leaf disc transformation methods).<br>
[0437]	Using known procedures, one of skill can screen for plants in accordance with the<br>
invention by detecting the increase or decrease of transgene mRNA or protein in transgenic plants. Means for detecting and quantitation of mRNAs or proteins are well known in the art.<br>
Polypeptides and peptides<br>
[0438]	In some embodiments, the invention provides isolated, synthetic or recombinant<br>
polypeptides having a sequence identity (such as at least about 50%, 51%, 52%i, 53%, 54%, 55%, 56%, 57%, 58%, 59%, 60%, 61%, 62%, 63%, 64%, 65%, 66%, 67%, 68%, 69%, 70%, 71%, 72%, 73%, 74%, 75%, 76%, 77%, 78%, 79%, 80%, 81%), 82%, 83%, 84%, 85%, 86%, 87%, 88%, 89%, 90%, 91%, 92%, 93%, 94%, 95%, 96%, 97%, 98%, 99%, or more, or complete (100%) sequence identity, or homology) to a sequence in accordance with the invention, such as proteins having a sequence as set forth in SEQ ID N0:2, SEQ ID N0:4, SEQ ID NO:6, SEQ ID NO:S, SEQ ID NO.'IO, SEQ ID N0:]2, SEQ ID N0:14, SEQ ID N0:16, SEQ ID N0:18, SEQ ID NO:20, SEQ ID NO:22, SEQ ID NO:24, SEQ ID NO;26, SEQ ID NO:28, SEQ ID NO:30, SEQ ID NO:32, SEQ ID NO:34, SEQ ID NO:36, SEQ ID NO:38, SEQ ID NO:40, SEQ ID NO:42, SEQ ID NO:44, SEQ ID NO:46, SEQ ID NO:48, SEQ ID NO:50, SEQ ID NO:52, SEQ ID NO:54, SEQ ID NO:56, SEQ ID NO:58, SEQ ID NO:60, SEQ ID NO:62, SEQ ID NO:64, SEQ ID NO:66, SEQ ID NO:68, SEQ ID NO:70, SEQ ID NO:72, SEQ ID NO:74, SEQ ID NO:76, SEQ ID NO:78, SEQ ID NO:80, SEQ ID NO:82, SEQ ID<br>
 <br>
NO:84, SEQ ID NO:86, SEQ ID NO:88, SEQ ID NO:90, SEQ ID NO:92, SEQ ID<br>
NO:94, SEQ ID NO:96, SEQ ID N0:9S, SEQ ID NO:100, SEQ ID NO:102, SEQ ID<br>
NO:104, SEQ ID NO:106, SEQ ID NO:]08, SEQ ID NO.-HO, SEQ ID N0:112, SEQ ID<br>
N0:1I4, SEQ ID N0:I16, SEQ ID N0:118, SEQ ID NO:120, SEQ IDNO;I22, SEQ ID<br>
NO:124, SEQ ID NO:126, SEQ ID NO;128, SEQ ID NO:130, SEQ ID NO:132, SEQ ID<br>
NO:134, SEQ ID NO:136, SEQ ID N0:138, SEQ ID NO.i40, SEQ ID NO:I42, SEQ ID<br>
NO:144, SEQ ID NO:146, SEQ ID NO:I48, SEQ ID NO:150, SEQ ID NO:152, SEQ ID<br>
NO:154, SEQ ID NO:156, SEQ ID NO:158, SEQ ID NO:160, SEQ ID NO:162, SEQ ID<br>
NO:I64, SEQ ID NO:I66, SEQ ID NO:!68, SEQ ID NO:I70, SEQ ID NO:I72, SEQ ID<br>
NO:174, SEQ ID NO:176, SEQ ID NO:178, SEQ ID NO:180, SEQ ID NO:182, SEQ ID<br>
NO:184, SEQ ID NO:186, SEQ IDNO:188, SEQ IDNO:190, SEQ ID NO:192, SEQ ID<br>
NO:194, SEQ ID NO:196, SEQ ID NO:I98, SEQ ID NO:200, SEQ ID NO:202, SEQ ID<br>
NO:204, SEQ ID NO:206, SEQ ID NO:208, SEQ ID NO:210, SEQ ID NO:212, SEQ ID<br>
NO:214, SEQ ID NO:216, SEQ ID NO:218, SEQ ID NO:220, SEQ ID NO:222, SEQ ID<br>
NO:224, SEQ ID NO:226, SEQ ID NO:228, SEQ ID NO:230, SEQ ID NO:232, SEQ ID<br>
NO:234, SEQ ID NO;236, SEQ ID NO;238, SEQ ID NO:240, SEQ ID NO:242, SEQ ID<br>
NO:244, SEQ ID NO;246, SEQ ID NO:248, SEQ ID NO:250, SEQ ID NO:252, SEQ ID<br>
NO;254, SEQ ID NO:256, SEQ ID NO:258, SEQ ID NO:260, SEQ ID NO-.262, SEQ ID<br>
NO:264, SEQ ID NO:266, SEQ ID NO:268, SEQ ID NO:270, SEQ ID NO:272, SEQ ID<br>
NO:274, SEQ ID NO:276, SEQ ID NO:278, SEQ ID NO:280, SEQ ID NO:282, SEQ ID<br>
NO:284, SEQ ID NO:286, SEQ ID NO:288, SEQ ED NO:290, SEQ ID NO:292, SEQ ID<br>
NO:294, SEQ ID NO:296, SEQ ID NO;298, SEQ ID NO:300, SEQ ID NO:302, SEQ ID<br>
NO:304, SEQ ID NO:306, SEQ ID NO:308, SEQ ID NO:310, SEQ ID NO:312, SEQ ID<br>
NO:314, SEQ ID NO:316, SEQ ID NO:318, SEQ ID NO:320, SEQ ID NO:322, SEQ ID<br>
NO:324, SEQ ID NO:326, SEQ ID NO:328, SEQ ID NO:330, SEQ ID NO:332, or SEQ<br>
ID NO:334 and enzymatically active fragments thereof. The percent sequence identity<br>
can be over the fiill length of the polypeptide, or, the identity can be over a region of at<br>
least about 50, 60, 70, 80, 90,100, 150, 200, 250, 300, 350, 400, 450, 500, 550, 600, 650,<br>
700 or more residues.<br>
0439]	Polypeptides in accordance with some embodiments of the invention can also be<br>
shorter than the full length of the polypeptides.   In other embodiments, the invention provides polypeptides (peptides, fragments) ranging in size between about 5 and the fUIl<br>
 <br>
length of a polypeptide, such as an enzyme, such as an aldolase, such as pyruvate aldolase,. HMG and/or KHG aldolase enzyme; exemplary sizes being of about 5, ] 0, 15. 20, 25, 30, 35, 40, 45, 50, 55, 60, 65, 70, 75, 80, 85, 90, 100, 125, 150, 175, 200, 250, 300, 350, 400, 450, 500, 550, 600, 650, 700, or more residues, such as contiguous residues of an aldolase, such as pyru'ate aldolase, such as HMG and/or KHG aldolase enzyme in accordance with the invention. Peptides in accordance with the invention (such as a subsequence of a polypeptide in accordance witli the invention) can be usefUl as, such as labeling probes, antigens (immunogens), toleragens, motifs, aldolase, such as pyruvate aldolase, such as HMG and/or KHG aldolase enzyme active sites (such as "catalytic domains"), signal sequences and/or prepro domains.<br>
[0440]	In other embodiments, polypeptides in accordance with the invention having<br>
aldolase activity, such as pyruvate aldolase, such as HMG and/or KHG aldolase activity are members of a genus of polypeptides sharing specific structural elements, such as amino acid residues, that correlate with aldolase activity, including pyruvate activity such as, without limitation, HMG and/or KHG aldolase activity. These shared structura! elements can be used for the routine generation of aldolase, such as pyruvate aldolase, such as HMG and/or KHG aldolase; variants. These shared structural elements of aldolase, such as pyruvate aJdolase, such as HMG and/or KHG aldolase enzymes in accordance with the invention can be used as guidance for the routine generation of aldolase, such as pyruvate aldolase, such as HMG and/or KHG aldolase enzymes variants within the scope of the genus of polypeptides in accordance with the invention.<br>
[0441]	As used herein, the terms "al'dolase, such as pyruvate aldolase, such as HMG<br>
and/or KHG aldolase" encompass any polypeptide or enzymes capable of catalyzing the aldol addition reaction or the retro-aldol reaction (such as polypeptides in accordance with the invention, see also Table 1 and Examples 4, 5 and 6, below), or any modification of a cariion-carbon bond containing material, such as in the production of R-2-hydroxy 2-(indo!-3yImethyl)-4-keto glutaric acid (R-MP) and certain stereoisomers of monatin, such as R,R and S,R monatin, and salts thereof.<br>
[0442]	Polypeptides in accordance with some embodiments of the invention catalyze the<br>
formation of carbon-carbon bonds in an aldol reaction and have the ability to utilize pyruvate or phosphoenolpyruvate as the nucleophilic component in the synthesis of a 4-hydroxy-2-ketobutyrate framework as shown in the general scheme below.<br>
 <br>
 <br>
[0443]	Without being bound by theory, it is believed that the conserved foiu--carbon<br>
fragment prepared in all pyruvate aldolase-catalyzed condensations is both densely and differentially functionalized. Moreover, in each adduct, four different oxidation states of carbon are contained in four contiguous carbons. The framework prepared by pyruvate aldolases thus allows the preparation of a-amino-y-hydroxycarboxylic acids, p-hydroxycarboxylic acids, a,Y-dihydroxycarboxylic acids, and 2-deoxyaidose sugars as shown in the scheme below.<br>
 <br>
invention can be synthetically versatile and can be used in the preparation of a wide range of products for use in animal feeds, human foods, indmtrial processes, and pharmaceuticais (see, for example, Gijsen, H.J.M. et al. Recent Advances in the Chemoenzymatic Synthesis of Carbohydrates and Carbohydrate Mimetics, Chem. Rev.<br>
 <br>
1996, 96, 443-473; Henderson, D.P. et al J. Org. Chem., Stereospecific Preparation of<br>
the N-Terminal Amino Acid Moieiy of Nikkomycins KX and KZ via a Multiple Enzyme<br>
Synthesis, 1997, 62, 7910-7911; Wynier, N. &amp; Toone, E.J. Enzyme-catalyzed Synthesis<br>
of Carbohydrates. Current Opin. Chemical Biology, 2000, 4, 110-119).<br>
!0445i	Polypeptides in accordance with some embodiments of the invention may have<br>
more than one type of enzymatic activity, specifically aldolase activity and an additional activity, for example, as set forth in Table 1, below. For example, a polypeptide in accordance with the invention can have aldolase activity, pyruvate aldolase, HMG and/or KHG aldolase activity. Additionally, the polypeptide may have, or may be thought to have, additional enzyme activity based on its EC classification. Table 1 includes the column "Predicted EC Nimiber". An EC number is the number assigned to a type of enzyme according to a scheme of standardized enzyme nomenclature developed by the Enzyme Commission of (he Nomenclature Committee of the International Union of Biochemistry and Molecular Biologj' (lUBMB). The results in the "Predicted EC Number" colimm are determined by a BLAST search against the Kegg (Kyoto Encyclopedia of Gen&amp;s and Genomes) database. If the top BLAST match (also called a "hit") has an Evalue equal to or less than e", the EC number assigned to the top match is entered into the table. The EC number of the top hit is used as a guide to what the EC number of the sequence of the invention might be. In instances where only a partial EC number is given, only a broad classification could be assigned based on the top hit. For instance, in the first row, for SEQ ID N0:2, encoded by SEQ ID NO:I, the Predicted EC Number is listed as "2...". Therefore, the classification assigned is broadly a transferase. For SEQ ID NO:26, encoded by SE(J ID NO:25, the most specific classification that could be assigned based on the top hit is as an aldehyde-lyase.<br>
 <br>
 <br>
13,14	Aldolase	HMG	2...	Unknown<br>
15,16	Aldolase	HMG	2...	Unknown<br>
17, 18	Aldolase	HMG	2-,.	Unknown<br>
19.20	Aldolase	HMG	2...	Unknown<br>
21,22	Aldolase	HMG	Unknown<br>
23,24	Aldolase	HMG	2...	Unknown<br>
25,26	Aldolase	HMG	4.1.2.	Unknown<br>
27,28	Aldolase	HMG	2...	Unknown<br>
29,30	Aldolase	HMG	2...	Unknown<br>
31,32	Aldolase	HMG	2...	Unknown<br>
33,34	Aldolase	HMG	2...	Unknown<br>
35,36	Aldolase	HMG	2...	Unknown<br>
37,38	Aldolase	HMG	2...	Unknown<br>
39,40	Aldolase	HMG	2...	Unknown<br>
41,42	Aldolase	HMG	2...	Unknown<br>
43,44	Aldolase	HMG	2...	Unknown<br>
45,46	Aldolase	HMG	2...	Unknown<br>
47,48	Aldolase	HMG	2...	Unknown<br>
49,50	Aldolase	HMG	2...	Unknown<br>
51,52	Aldolase	HMG	2...	Unknown<br>
53,54	Aldolase	HMG	2...	Unknown<br>
55,56	Aldolase	HMG	2...	Unknown<br>
57,58	Aldolase	HMG	2...	Unknown<br>
59,60	Aldolase	HMG	2...	Unknown<br>
61,62	Aldolase	HMG	2...	Unknown<br>
63,64	Aldolase	HMG	2...	Unknown<br>
65.66	Aldolase	HMG	2...	AA1-27	Unknown<br>
67,68	Aldolase	HMG	2...	Unknown<br>
S9,70	Aldolase	HMG	2-.	Unknown<br>
71,72	Aldolase	HMG	2...	Unknown<br>
73,74	Aldolase	HMG	2...	Unknown<br>
75,76	Aldolase	HMG	2...	Unknown<br>
77,78	Aldolase	HMG	2...	Unknown<br>
79,80	Aldolase	HMG	2...	Unknown<br>
81,82	Aldolase	HMG	2...	Unknown<br>
83,84	Aldolase	HMG	2...	Unknown<br>
85,86	Aldolase	HMG	2...	Unknown<br>
87,88	Aldolase	HMG	2...	Unknown<br>
89,90	Aldolase	HMG	2.-	Unknown<br>
91,92	Aldolase	HMG	2...	Unknown<br>
93,94	Aldolase	HMG	2...	Unknown<br>
95, 96	Aldolase	HMG	2...	Unknown<br>
97,98	Aldolase	HMG	2...	Unknown<br>
99, 100	Aldolase	HMG	2...	Unknown<br>
101,102	Aldolase	HMG	2...	Unknown<br>
103, 104	Aldolase	HMG	2...	Unknown<br>
105, 106	Aldolase	HMG	2...	Unknown<br>
107, 108	Aldolase	HMG	2...	Unknown<br>
109,110	Aldolase	HMG	2...	Unknown<br>
111,112	Aldolase	HMG	2...	Unknown<br>
113,114	Aldolase	HMG	2...	Unknown<br>
115,116	Aldolase	HMG	2...	Unknown<br>
117,118	Aldolase	HMG	2...	Unknown<br>
 <br>
119, 120	Aldolase	HMG	2..-	Unknown<br>
121, 122	Aldolase	HMG	2...	Unknown<br>
123, 124	Aldolase	HMG	2..-	Unknown<br>
125, 126	Aldolase	HMG	2...	Unknown<br>
127, 128	Aldolase	HMG	2...	Unknown<br>
129,130	Aldolase	HMG	2...	Unknown<br>
131. 132	Aldolase	HMG	2...	Unknown<br>
133, 134	Aldolase	HMG	2..-	Unknown<br>
135,136	Aldolase	HMG	2..,	Unknown<br>
137, 13S	Aldolase	HMG	2...	Unknown<br>
139,140	Aldolase	HMG	2...	Unknown<br>
141,142	Aldolase	HMG	2...	Unknown<br>
143, 144	Aldolase	HMG	2...	Unknown<br>
145, 146	Aldolase	HMG	2-.,	Unknown<br>
147, 148	Aldolase	HMG	2...	Unknown<br>
149, 150	Aldolase	HMG	2...	Unknown<br>
151, 152	Aldolase	HMG	2...	Unknown<br>
153, 154	Aldolase	HMG	2...	Unknown<br>
155, 156	Aldolase	HMG	2..-	Unknown<br>
157, 158	Aldolase	HMG	2...	Unknown<br>
159, 160	Aldolase	HMG	2...	Unknown<br>
161, 162	Aldolase	HMG	2...	Unknown<br>
163.164	Aldolase	HMG    -   ■  2.,.	Unknown.<br>
165, 166	Aldolase	HMG	2.,.	Unknown<br>
167, 168	Aldolase	HMG	2...	Unknown<br>
169. 170	Aldolase	HMG	2...	Unknown<br>
171, 172	Aldolase	HMG	2...	Unknown<br>
173, 174	Aldolase	HMG	2...	Unknown<br>
175, 176	Aldolase	HMG	2...	Unknown<br>
177,178	Aldolase	HMG	2...	Unknown<br>
179,180	Aldolase	HMG	2..-	Unknown<br>
181, 182	Aldolase	HMG	2...               AA1-31	Unknown<br>
183,184	Aldolase	HMG	2...	Unknown<br>
185, 186	Aldolase	HMG	2.-.	Unknown<br>
187, 188	Aldolase	HMG	2...	Unknown<br>
189,190	Aldolase	HMG	2...	Unknown<br>
191,192	Aldolase	HMG	2...	Unknown<br>
193, 194	Aldolase	HMG	2...	Unknown<br>
195,196	Aldolase	HMG	2...	Unknown<br>
197,198	Aldolase	HMG	2...	Unknown<br>
199,200	Aldolase	HMG	2...	Unknown<br>
201,202	Aldolase	HMG	2.,.	Unknown<br>
203,204	Aldolase	HMG	2...	Unknown<br>
205,206	Aldolase	HMG	2...	Unknown<br>
207,208	Aldolase	HMG	2...	Unknown<br>
209,210	Aldolase	HMG	2...	Unknown<br>
211,212	Aldolase	HMG	2...	Unknown<br>
213,214	Aldolase	HMG	2...	Unknown<br>
215,216	Aldolase	HMG	2.,-	Unknown<br>
217,218	Aldolase	HMG	2...	Unknown<br>
219,220	Aldolase	HMG	2...	Unknown<br>
221,222	Aldolase	HMG	2...	Unknown<br>
223,224	Aldolase	HMG	2...	Unknown<br>
 <br>
225,226	Aldolase	HMG	2...	Unknown<br>
227,228	Aldolase	HMG	2...	Unknown<br>
229,230	Aldolase	HMG	2...	Unknown<br>
231,232	Aldolase	HMG	2..,	Unknown<br>
233,234	Aldolase	HMG	2-..	Unknown<br>
235,236	Aldolase	HMG	2...	Unknown<br>
237,238	Aldolase	HMG	2...	Unknown<br>
239,240	Aldolase	HMG	2...	Unknown<br>
241,242	Aldolase	HMG	2...	Unknown<br>
243,244	Aldolase	HMG	2...	Unknown<br>
245,246	Aldolase	HMG	2...	Unknown<br>
247,248	Aldolase	HMG	2...	Unknown<br>
249,250	Aldolase	HMG	2...	Unknown<br>
251,252	Aldolase	HMG	2.,.	Unknown<br>
253,254	Aldolase	HMG	2...	Unknown<br>
255,256	Aldolase	HMG	2...	Unknown<br>
257,258	Aldolase	HMG	2...	Unknown<br>
259,260	Aldolase	HMG	2...	AA1-18	Unknown<br>
261,262	Aldolase	HMG	2...	Unknown<br>
263,264	Aldolase	HMG	2...	Unknown<br>
265,266	Aldolase	HMG	2...	Unknown<br>
267,268	Aldolase	HMG	2...	Unknown<br>
269,270	Aldolase	HMG	2...	Unknown ,..       ,<br>
271,272	Aldolase	HMG	2...	Unknown<br>
273,274	Aldolase	HMG	2...	Unknown<br>
275,276	Aldolase	HMG	2...	Unknown<br>
277,278	Aldolase	HMG	2...	Unknown<br>
279,280	Aldolase	HMG	2...	Unknown<br>
281,282	Aldolase	HMG	2...	Unknown<br>
283,284	Aldolase	HMG	2...	Unknown<br>
285,286	Aldolase	HMG	2...	Unknown<br>
287,288	Aldolase	HMG	2...	Unknown<br>
289,290	Aldolase	HMG	2..-	Unknown<br>
291,292	Aldolase	HMG	2...	Unknown<br>
293,294	Aldolase	HMG	2...	Unknown<br>
295,296	Aldolase	HMG	2...	Unknown<br>
297,298	Aldolase	HMG	2..,	Unknown<br>
299,300	Aldolase	HMG	2.1..	Unknown<br>
301,302	Aldolase	HMG	2.1.,	Unknown<br>
303,304	Aldolase	HMG	2.1.	Unknown<br>
305,306	Aldolase	KHG	4.1.2.14	Unknown<br>
307.308	Aldolase	KHG	4.1.2.14	Unknown<br>
309,310	Aldolase	KHG	4.1.2.14	Unknown<br>
311,312	Aldolase	KHG	4.1.2.14	Unknown<br>
313,314	Aldolase	KHG	4.1.2.14	Unknown<br>
315,316	Aldolase	KHG	4,1.2.14	Unknown<br>
317,318	Aldolase	KHG	4.1.2.14	Unknown<br>
319,320	Aldolase	KHG	4.1.3.16	Unknown<br>
321,322	Aldolase	KHG	4.1.2.14	Unknown<br>
323,324	Aldolase	KHG	4.1.2.14	Unknown<br>
325,326	Aldolase	KHG	4.1.2.14	Unknown<br>
327,328	Aldolase	KHG	4,1.2.14	Unknown<br>
329,330	Aldolase	KHG	4.1,3.16	Unknown<br>
 <br>
331, 332    Aldolase    KHG	4.1.2-14	Unknown<br>
333, 334    Aldolase    KHG	4,12.14	Unknown<br>
[0446]	Polypeptides and peptides in accordance with the invention can be isolated from<br>
natural sources, be synthetic, or be recombinantly generated polypeptides. Peptides and proteins can be recombinantly expressed in vitro or in vivo. The peptides and polypeptides in accordance with the invention can be made and isolated using any method icnown in the art. Polypeptide and peptides in accordance with the invention can also be synthesized, in whole or m part, using chemical methods well known in the art. See such as Caruthers (1980) Nucleic Acids Res. Symp. Ser. 215-223; Horn (1980) Nucleic Acids Res. Symp. Ser. 225-232; Banga, A.K., Therapeutic Peptides and Proteins, FormuJalion, Processing and Delivery Systems (1995) Technomic Publishing Co., Lancaster, PA. For example, peptide synthesis can be peiformed using various solid-phase techniques (see such as Roberge (1995) Science 269:202; Merrifield (1997) Methods En2ymol. 289:3-13) and automated synthesis may be achieved, such as using the ABI 431A Peptide Synthesizer (Perkin Elmer) in accordance with the instructions provided by the manufacturer.<br>
[0447]	Peptides   and  polypeptides   in  accordance  with  the   invention   can  also  be<br>
glycosylated. The glycosylation can be added post-translationally either chemically or by cellular biosynthetic mechanisms, wherein the later incorporates the use of known glycosylation motifs, which can be native to the sequence or can be added as a peptide or added in the nucleic acid coding sequence. The glycosylation can be 0-linked or N-linked.<br>
[0448]	In some embodiments, when indicated, peptides and polypeptides in accordance<br>
with the invention can include all "mimetic" and "peptidomimetic" forms. The terms "mimetic" and "peptidomimetic" refer to a synthetic chemical compound which has substantially the same structural and/or fimctional characteristics of the polypeptides in accordance with the invention. The mimetic can be either entirely composed of synthetic, non-natural analogues of amino acids, or, is a chimeric molecule of partly natural peptide amino acids and partly non-nalural analogs of amino acids. The mimetic CMI also incorporate any amount of natural amino acid conservative substitutions as long as such substitutions also do not substantially alter the mimetic's structure and/of activity. As with polypeptides in accordance with the invention which are conservative variants or members of a genus of polypeptides in accordance with the invention (such as having<br>
 <br>
about 50% or more sequence identitj' to a sequence in accordance with the invention), routine experimentation will determine whether a mimetic is within the scope in accordance with the invention, i.e., that its structure and/or function is not substantially altered. Thus, in some embodiments, a mimetic composition is within the scope in accordance with the invention if it has an aldolase, such as pyruvate aldolase, such as HMG and/or KHG aldolase enzyme, attivity.<br>
[0449]	Polypeptide mimetic compositions in accordance with the invention can contain<br>
any combination of non-natural structural components. In an alternative embodiment, mimetic compositions in accordance with the invention include one or all of the following three structural groups: a) residue linkage groups other than the natural amide bond ("peptide bond") linkages; b) non-natural residues in place of naturally occurring amino acid residues; or c) residues which induce secondary structural mimicry, i.e., to induce or stabilize a secondary structure, such as a beta turn, gamma turn, beta sheet, alpha helix conformation, and the like. For example, a polypeptide in accordance with the mvention can be characterized as a mimetic when all or some of its residues are joined by chemical means other than natural peptide bonds. Individual peptidomimetic residues can be joined by peptide bonds, other chemical bonds or coupling means, such as glutaraldehyde, N-hydroxysuccinimide esters, bifttnctional maleimides, N,N'-dicyclohexylcarbodiimide (DCC) or N,N'-diisopropylcarbodiimide (DIG). Linking groups that can be an alternative to the traditional amide bond ("peptide bond") linkages include, such as ketomethylene (such as -C(=0)-CH2- for -C(=0)-NH-), aminomethylene (CH2-NH), ethylene, olefin (CH=CH), ether (CH2-O), thioether (CH2-S), tetrazole (CN-), thiazole, retroamide, thioamide, or ester (see Spatola (1983) in Chemistry and Biochemistry of Amino Acids, Peptides and Proteins, Vol. 7, pp 267-357, "Peptide Backbone Modifications," Marcell Dekker, NY).<br>
[0450]	A polypeptide in accordance with the invention can also be characterized as a<br>
mimetic by containing all or some non-natural residues in place of naturally occurring amino acid residues. Non-natural residues are well described in the scientific and patent literature; a few exemplary non-natural compositions useful as mimetics of natural amino acid residues and guidelines are described below. Mimerics of aromatic amino acids can be generated by replacing by, such as D- or L- naphylalanine; D- or L- phenylglycine; D-or L-2 thieneylaianine; D- or L-1, -2, 3-, or 4- pyreneylalanine; D- or L-3 thieneylalanine;<br>
 <br>
D- or L-(2-pyridinyl)-aianine; D- or L-(3-pyridinyl)-alamne; D- or L-(2-pyraziny!)-<br>
alanine; D- or L-(4-isopropyl)-phenylglycine; D-(tiifluoromethyl)-phenyIglycine; D-<br>
(trifluoromethyl)-phenylalaiiine;        D-p-fluoro-phenylalanine;	D-	or        L-p-<br>
biphenylphenylalanine; D- or L-p-methoxy-biphenylphenylalanine; D- or L-2-<br>
indole(alkyl)a!anines; and, D- or L-alkyiainines, where alkyl can be substituted or<br>
unsubstituted methyl, ethyi, propyl, hexyl, butyl, pentyl, isopropyl, iso-butyl, sec-isotyl,<br>
iso-pentyl, or a non-acidic amino acids. Aromatic rings of a non-natural amino acid<br>
include, such as thiazoly], thiophenyl, pyrazolyl, benzimidazolyl, naphthyl, furan}'!,<br>
pyrrolyl, and pyridyl aromatic rings.<br>
[0451|	Mimetics of acidic amino acids can be generated by substitution by, such as non-<br>
carboxylate amino acids while maintaining a negative charge; (phosphono)alanine; sulfated threonine. Carboxyl side groups (such as aspartyl or glutamyl) can also be selectively modified by reaction with carbodiimides (R'-N-C-N-R') such as 1~ cyclohexyl-3(2-niorpholiny]-(4-ethyl) carbodiimide or l-ethyl-3(4-a2onia- 4,4-dimetholpentyl) carbodiimide. Aspartyl or glutamyl can also be converted to asparaginy! and glutaminyi residues by reaction with ammonium ions. Mimetics of basic amino acids can be generated by substitution with, such as (in addition to Jysine and arginine) the amino acids ornithine, citrulline, or (guanidino)-acetic acid, or (guanidino)alkyl-acetic acid, where alkyl is defined above. Nitrile derivative {such as containing the CN-moiety in place of COOH) can be substituted for asparagine or glutamine. Asparaginyl and glutaminyi residues can be deaminated to the corresponding aspartyl or glutamyl residues. Arginine residue mimetics can be generated by reacting arginy! with, such as one or more conventional reagents, including, such as phenylglyoxal, 2,3-butanedione, 1,2-cyclo-hexanedione, or ninhydrin, in some embodiments under alkaline conditions. Tyrosine residue mimetics can be generated by reacting tyrosyl with, such as aromatic diazonium compounds or tetranitromethane. N-acetylimidizoI and tetranitromethane can be used to form 0-acelyl tyrosyl species and 3-nitro derivatives, respectively. Cysteine residue mimetics can be generated by reacting cysteinyl residues with, such as alpha-haloacetates such as 2-chloroacetic acid or chloroacetamide and corresponding amines; to give carboxymethyl or carboxyamidomethyl derivatives. Cysteine residue mimetics can also be generated by reacting cysteinyl residues with, such as bromo-trifluoroacetone, alpha-bromo-beta-(5-imidozoyl) propionic acid; chloroacetyl phosphate, N-alkyimaieimides, 3-<br>
 <br>
nitro-2-pyridy! disulfide; methyl 2-pyridyl disulfide; p-chloromercuribenzoate; 2-chloromercuri-4 nitropheno!; or chbi-o-7-mtrobenzo-oxa-l,3-diazole. Lysine mimetics can be generated (and amino terminal residues can be altered) by reacting lysinyl with, such as succinic or other carboxylic acid anhydrides. Lysine and other alpha-amino-containing residue mimetics can also be generated by reaction with imidoesters, such as methyl picolinimidate, pyridoxal phosphate, pyridoxal, chloroborohydride, trinitro-benzenesulfonic acid, 0-methylisourea, 2,4, pentanedione, and transamidase-catalyzed reactions with glyoxylate. Mimetics of methionine can be generated by reaction with, such as methionine sulfoxide. Mimetics of proline include, such as pipecolic acid, thiazolidine carboxylic acid, 3- or 4- hydroxy proline, dehydroproline, 3- or 4-methylproline, or 3,3,-dimethylproIine. Histidine residue mimetics can be generated by reacting histidyl with, such as diethylprocarbonaie or para-bromophenacyl bromide. Other mimetics include, such as those generated by hydroxylation of proline and lysine; phosphorylation of the hydroxyl groups of seryl or threonyl residues; methylation of the alpha-amino groups of lysine, arginine and liistidine; acetylation of the N-terminal amine; methylation of main chain amide residues or substitution with N-methyi amino acids; or amidation of C-terminal carboxyi groups.<br>
[0452]	In some embodiments, a residue, such as an amino acid, of a polypeptide in<br>
accordance with the invention can also be replaced by an amino acid (or peptidomimetic residue) of the opposite chirality. In some embodiments, any amino acid naturally occurring in the L-configuration (which can also be referred to as the R or S, depending upon the structure of the chemical entity) can be replaced with the amino acid of the same chemical structural type or a peptidomimetic, but of the opposite chirality, referred to as the D- amino acid, but also can be referred to as the R- or S- form.<br>
[0453]	The   invention   also   provides   methods   for  modifying   the   polypeptides   in<br>
accordance with the invention by either natural processes, such as post-translational processing (such as phosphorylation, acylation, etc), or by chemical modification techniques, and the resulting modified polypeptides. Modifications can occur anywhere in the polypeptide, including the peptide backbone, the amino acid side-chains and the amino or carboxyi termini. It will be appreciated that the same type of modification may be present in the same or varying degi-ees at several sites in a given polypeptide. Also a given polypeptide may have many types of modifications.    In some embodiments.<br>
 <br>
modifications include acelation, acylation, ADP-ribosylation, amidation, covalent<br>
attachment of flavin, covalent attachment of a heme moiety, covaJent attachment of a<br>
nucleotide or nucleotide derivative, covalent attachment of a lipid or lipid derivative,<br>
covalent attachment of a phosphatidylinositoi, cross-iinking cyclization, disulfide bond<br>
formation, demethylation, formation of covalent cross-links, formation of cysteine,<br>
formation of pyroglutamate, formylation, gamma-carboxylation, glycosylation, GPI<br>
anchor formation, hydroxylation, iodination, methylation, myristolyation, oxidation,<br>
pegylation, proteolytic processing, phosphorylation, prenylation, racemization,<br>
seJenoylation, sulfation, and transfer-IlNA mediated addition of amino acids to protein<br>
such as arginylation. See, Creighton, T.E., Proteins - Structure and Molecular Properties<br>
2nd Ed., W.H. Freeman and Company, New York (1993); Postlranslational Covalent<br>
Modification of Proteins, B.C. Johnson, Ed., Academic Press, New York, pp. 1-12<br>
(1983).<br>
[0454]	Solid-phase chemical peptide synthesis methods can also be used to synthesize the<br>
polypeptide or fragments in accordance with the invention. Such method have been known in the art since the early 1960's (Merrifield, R. B., J. Am. Chem. Soc, 85:2149-2154, 1963) (See also Stewart, J. M. and Young, J. D., Solid Phase Peptide Synthesis, 2nd Ed., Pierce Chemical Co., Rockford.. III., pp. 11-12)) and have recently been employed in commercially available laboratory peptide design and synthesis kits (Cambridge Research Biochemicals). Such commercially available laboratory kits have generally utilized the teachings of H. M. Geysen et al, Proc. Natl. Acad. Sci., USA, 81:3998 (1984) and provide for synthesizing peptides upon the tips of a multitude of "rods" or "pins" all of which are connected to a single plate. When such a system is utilized, a plate of rods or pins is inverted and inserted into a second plate of corresponding wells or reservoirs which contain solutions for attaching or anchoring an appropriate amino acid to the pin's or rod's tips. By repeating such a process step, i.e., inverting and inserting the rod's and pin's tips into appropriate solutions, amino acids are built into desired peptides. In addition, a number of available FMOC peptide synthesis systems are available. For example, assembly of a polypeptide or fragment can be carried out on a solid support using an Applied Biosystems, Inc. Model 431 A"™ automated peptide synthesizer. Such equipment provides ready access to the peptides in accordance with the invention, either by direct<br>
 <br>
synthesis or by synthesis of a series of fragments that can be coupled using other known techniques.<br>
(0455]	The poiypeptides in accordance with the invention include aldolase, such as<br>
pyruvate ajdolase, such as HMG and/or KHG aldolase enzymes in an active or inactive form. For example, the polypeptides in accordance with the invention include proproteins before "maturation'" or processing of prepro sequences, such as by a proprotein-processing enzyme, such as a proprotein convertase to generate an "active" mature protein. The polypeptides in accordance with the invention include aldolase, such as pyruvate aldolase, such as HMG and/or KHG aldolase enzymes inactive for other reasons, such as before "activation" by a post-translational processing event, such as an endo- or exo-peptidase or proteinase action, a phosphorylation event, an amidation, a glycosylation or a sulfation, a dimerization event, and the like. The polypeptides in accordance with the invention include all active forms, including active subsequences, such as catalytic domains or active sites, of the enzyme.<br>
[0456]	The invention includes immobilized aldolase, such as pjTuvate aldolase, such as<br>
HMG and/or KHG aldolase enzymes, anti-aJdolase, such as anti-pyruvate aldolase, such as anti-HMG and/or anti-KHG aldolase antibodies and fragments thereof In some embodiments, the invention provides methods for inhibiting aldolase, such as pyruvate aldolase, such as HMG and/or KHG aldolase enzyme, activity, such as using dominant negative mutants or anti-aldolase, such as anti-pyruvate aldolase, such as anti-HMG and/or anli-KHG aldolase antibodies in accordance with the invention. In some embodiments, the invention includes heterocomplexes, such as fusion proteins, heterodimers, etc., comprising the aldolase, such as pyruvate aldolase, such as HMG and/or KHG aldolase enzymes in accordance with the invention.<br>
[0457]	In some embodiments, polypeptides in accordance with the invention can have an<br>
aldolase, such as pyruvate aldolase, such as HMG and/or KHG aldolase enzyme, activity under various conditions, such as at extremes in pH and/or temperature or, in some embodiments, in the presence of oxidizing agents. In some embodiments, the invention provides methods leading to alternative aldolase, such as pyruvate aldolase, such as HMG and/or KHG aldolase enzyme preparations with different catalytic efficiencies and stabilities, such as towards temperature, oxidizing agents and changing wash conditions. In some embodiments, aldolase, such as pyruvate aldolase, such as HMG and/or KHG<br>
 <br>
aldolase enzyme variants can be produced using techniques of site-directed mutagenesis and/or random mutagenesis. In some: embodiments, directed evolution can be used to produce a great variety of aldolase, such as pyruvate aldolase, such as HMG and/or KHG aldolase enzyme variants with alternative specificities and stability.<br>
[0458]	The proteins in accordance with the invention are also usefu! as research reagents<br>
to identiiy aldolase, such as pyruvate aldolase, such as HMG and/or KHG aldolase enzyme modulators, such as activators or inhibitors of aldolase, such as pyruvate aldolase, such as HMG and/or KHG aldolase enzyme, activity. Briefly, test samples (compounds, broths, extracts, and the like) are added to aldolase, such as pyruvate aldolase, such as HMG and/or KHG aldolase enzyme assays to determine their ability to inhibit substrate cleavage, hihibitors identified in this way can be used in industry and research to reduce or prevent undesired proteolysis. As with aldolase, such as pyruvate aldolase, such as HMG and/or KHG aldolase enzymes, inhibitors can be combined to . increase the spectrum of activity.<br>
[0459]	The enzymes in accordance with the invention are also useilil as research reagents<br>
to digest proteins or in protein sequencing. For example, the aldolase, such as pyruvate aldolase, such as HMG and/or KHG aldolase enzymes may be used to break polypeptides into smaller iragments for sequencing using, such as an automated sequencer.<br>
[0460]	The invention also provides methods of discovering new aldolase, such as<br>
pyruvate aldolase, such as HMG and/or KHG aldolase enzymes using the nucleic acids, polypeptides and antibodies in accordance with the invention. In some embodiments, phagemid libraries are screened for expression-based discovery of aldolase, such as pyruvate aldolase, such as HMG and/or KHG aldolase enzymes. In other embodiments, lambda phage libraries are screened for expression-based discovery of aldolase, such as pyruvate aldolase, such as HMG and/or KHG aldolase enzymes. Screening of the phage or phagemid libraries can allow the detection of toxic clones; improved access to substrate; reduced need for engineering a host, by-passing the potential for any bias resulting from mass excision of the library; and, faster growth at low clone densities. Screening of phage or phagemid libraries can be in liquid phase or in solid phase. In some embodiments, the invention provides screening in liquid phase. This gives a greater flexibility in assay conditions; additional substrate flexibility; higher sensitivity for weak clones; and ease of automation over solid phase screening.<br>
 <br>
[0461]	The invention provides screening methods using the proteins and nucleic acids in<br>
accordance with the invention and robotic automation to enable the execution of many thousands of biocatalytic reactions and screening assays in a short period of time, such as per day, as well as ensuring a high level of accuracy and reproducibility (see discussion of arrays, below). As a result, a librar,' of derivative compounds can be produced in a matter of weeks. For further teachings on modification of molecules, including small molecules, see PCr/US94/09I74; U.S. Pat. No. 6,245,547.<br>
[04621	hi some embodiments, polypeptides or fragments in accordance with the invention<br>
are obtained through biochemical enrichment or purification procedures. The sequence of potentially homologous polypeptides or fragments may be determined by aldolase, such as pyruvate aldolase, such as HMG and/or KHG aldolase enzyme assays (see Examples 3, 4 and 5, below), gel electrophoresis and/or microsequencing. The sequence of the prospective polypeptide or fragment in accordance with the invention can be compared to a polypeptide in accordance with the invention, or a fragment, such as comprising at least about 5, 10, 15, 20, 25, 30, 35, 40, 50, 75, 100, or 150 or more consecutive amino acids thereof using any of the programs described above,<br>
[0463]	Another embodiment of the invention is an assay for identifying fragments or<br>
variants in accordance with the invention, which retain the enzymatic function of the polypeptides in accordance with the invention. For example the fragments or variants of said polypeptides, may be used to catalyze biochemical reactions, which indicate that the fragment OT variant retains the ensymatic activity of a polypeptide in accordance with the invention. An exemplary assay for determining if fragments of variants retain the enzymatic activity of the polypeptides in accordance with the invention includes the steps of: contacting the polypeptide fragment or variant with a substrate molecule under conditions which allow the polypeptide fragment or variant to function and detecting either a decrease in the level of substrate or an increase in the level of the specific reaction product of the reaction between the polypeptide and substrate.<br>
[0464]	The  present  invention exploits the  unique  catalytic properties  of enzymes.<br>
Whereas the use of biocatalysts (i.e., purified or crude enzymes, non-living or living cells) in chemical transformations normally requires the identification of a particular biocatalyst that reacts with a specific starting compound, the present invention uses selected biocatalysts and reaction conditions that are specific for fmictional groups that<br>
 <br>
are present in many starting compounds, such as small molecules. Each biocatalyst is specific for one functional group, or several related ftmctional groups and can react with many starting compounds containing this functional group.<br>
[0465]	In  some  embodiments,  the  biocatalytic  reactions  produce   a population  of<br>
derivatives ftom a single starting compound. These derivatives can be subjected to another round of biocatalytic reactions to produce a second population of derivative compound. Thousands of variations of the original small molecule or compotmd can be produced with each iteration of biocatalytic derivatization.<br>
[0466}	Enzymes react at specific sites of a starting compound without affecting the rest of<br>
the molecule, a process which is very difficult to achieve using traditional chemical methods. This high degree of biocatalytic specificiiy provides the means to identify a single active compound within the library. The library is characterized by the series of biocatalytic reactions used to produce it, a so-called "biosynthetic history". Screening the library for biological activities and tracing the biosynthetic history identifies the specific reaction sequence producing the active compound. The reaction sequence is repeated and the structure of the synthesized compound determined. This mode of identification, unlike other synthesis and screening approaches, does not require immobilization technologies and compounds can be synthesized and tested free in solution using virtually any type of screening assay. It is important to note, that the high degree of specificity of enzyme reactions on functional groups allows for the "tracking" of specific enzymatic reactions that make up the biocatalytically produced library.<br>
[0467]	hi some embodiments, procedural steps are performed using robotic automation<br>
enabling the execution of man}' thousands of biocatalytic reactions and/or screening assays per day as well as ensuring a high level of accuracy and reproducibility. Robotic automation can also be used to screen for aldolase activity to determine if a poljeptide is within the scope in accordance with the invention. As a result, in some embodiments, a library of derivative compounds can be produced in a matter of weeks which would take years to produce using "traditional" chemical or enzymatic screening methods.<br>
[0468]	In  one  embodiment,  the  invention provides  methods  for  modifying  small<br>
molecules, comprising contacting a polypeptide encoded by a polynucleotide described herein or enzymatically active Augments thereof with a small molecule to produce a modified small molecule. A library of modified small molecules is tested to determine if<br>
 <br>
a modified small molecule is present vfithin the library, which exhibits a desired activity. A specific biocatalytic reaction which produces the modified small molecuie of desired activity is identified by systematically eliminating each of the biocatalytic reactions used to produce a portion of the library and then testing the small molecules produced in the portion of the library for the presence or absence of the modified small molecule with the desired activity. The specific biocatalytic reactions which produce the modified small molecule of desired activit&gt;' is optionally repeated. The biocatalytic reactions are conducted with a group of biocatalysts that react with distinct structural moieties found within the structure of a small molecule, each biocatalyst is specific for one structural moiety or a group of related structural moieties; and each biocatalyst reacts with many different small molecules which contain the distinct structural moiety.<br>
Aldolase, such as pyruvate aldolase, such as HMG and/or KHG aldolase enzyme signal sequences, prepro and catalytic domains<br>
[0469]	The invention provides aldolase, such as pyruvate aldolase, such as HMG and/or<br>
KHG aldolase enzyme signal sequences (such as signal peptides (SPs)), prepro domains and catalytic domains (CDs). The SPs, prepro domains and/or CDs in accordance with the invention can be isolated, synthetic or recombinant peptides or can be part of a fusion protein, such as a heterologous domain in a chimeric protein. In some embodiments, the invention provides nucleic acids encoding these catalytic domains (CDs), prepro domains and signal sequences (SPs, such as a peptide having a sequence comprising/ consisting of amino terminal residues of a polypeptide in accordance with the invention).<br>
|0470]	The invention provides isolated, synthetic or recombinant signal sequences (such<br>
as signal peptides) consisting of or comprising a sequence as set forth m residues 1 to 14, I to 15,1 to 16,1 to 17,1 to 18, [ to 19,1 to 20, 1 to 21, 1 to 22, I to 23,1 to 24, 1 to 25, 1 to 26,1 to 27,1 to 28,1 to 28, 1 to 30,1 to 31,1 to 32,1 to 33,1 to 34,1 to 35,1 to 36, 1 to 37, 1 to 38, 1 to 40, 1 to 41, 1 to 42, 1 to 43, 1 to 44, 1 to 45, 1 to 46, or 1 to 47, or more, of a polypeptide in accordance with the invention, such as polypeptides in accordance with the invention, see also Table 1, Examples 4, 5 and 6, below, and Sequence Listing. For example, Table 1, above, sets forth exemplary signal (leader) sequences in accordance with the invention, such as in the polypeptide having a sequence as set forth in SEQ E) NO-.66, encoded, such as by SEQ ID NO:65, has a signal sequence comprising     (or     consisting     of)     the     amino     terminal     27     residues,     or.<br>
 <br>
MSrVVTKIERAGAAAVAALRTSGVATV (SEQ ID NO:407) which corresponds to the first 27 amino acids of SEQ ID NO:66.<br>
[0471]	In some embodimems, the im'ention provides signal sequences comprising the<br>
first 14,15,16, 17,18,19, 20, 21, 22, 23, 24, 25,26, 27, 28, 29, 30, 31, 32, 33, 34, 35, 36, 37, 38, 39, 40, 41, 42, 43, 44, 45, 46, 47, 48, 49, 50, 51, 52, 53, 54, 55, 56, 57, 58, 59, 60, 61, 62, 63, 64, 65, 66, 67, 68, 69, 70 or more amino terminal residues of a polypeptide in accordance with the invention.<br>
[0472]	The invention includes polypeptides with or without a signal sequence and/or a<br>
prepro sequence. In some embodiments, the invention includes polypeptides with heterologous signal sequences and/or prepro sequences. The prepro sequence (including a sequence in accordance with the invention used as a heterologous prepro domain) can be located on the amino terminal or the carboxy terminal end of the protein. In some embodiments, the invention also includes isolated, synthetic or recombinant signal sequences, prepro sequences and catalytic domains (such as "active sites") comprising sequences in accordance with the invention. The polypeptide comprising a signal sequence in accordance with the invention can be an aldolase, such as pyruvate aldolase, HMG and/or KHG aldolase enzyme in accordance with the invention or another aldolase, such as pyruvate aldolase, such as HMG and/or KHG aldolase enzyme or another enzyme or other polypeptide. Methods for identifying "prepro" domain seqtiences and signal sequences are well known in the art, see Van de Ven (1993) Crit. Rev. Oncog. 4(2):115-136. For example, to identify a piepro sequence, the protein is purified from the extracellular space and the N-teiminai protein sequence is determined and compared to the unprocessed form.<br>
[0473]	The aldolase, such as pyruvate aldolase, such as HMG and/or KHG aldolase<br>
enzyme signal sequences (SPs) and/or prepro sequences in accordance with the invention can be isolated, synthetic or recombinant peptides, or, sequences joined to another aldolase, such as pyruvate aldolase, such as HMG and/or KHG aldolase enzyme or a non-aldolase, such as non-pyruvate aldolase, e.g, non-HMG and/or non-KHG aldolase polypeptide, such as a fusion (chimeric) protein. In some embodiments, the invention provides polypeptides comprising aldolase, such as pyruvate aldolase, such as HMG and/or KHG aldolase enzyme signal sequences in accordance with the invention. In some embodiments, polypeptides comprising aldolase, such as pyruvate aldolase, such as HMG<br>
 <br>
and/or KHG aldolase enzyme signal sequences SPs and/or prepro in accordance with the invention comprise sequences heterologous to an aldolase, such as pyruvate aldolase, HMG and/or KHG aldolase enzyme in accordance with the invention (such as a fusion protein comprising an SP and/or prepro in accordance with the invention and sequences from another aldolase, such as pyruvate aldolase, such as HMG and/or KHG aldolase enzyme or a non-aidolase, such as non-pymvate aldolase, e.g, non-HMG and/or non-KHG aldolase protein). In some embodiments, the invention provides aldolase, such as pyruvate aldolase, such as HMG and/or KHG aldolase enzymes in accordance with the invention with heterologous SPs and/or prepro sequences, such as sequences with a yeast signal sequence. An aldolase, such as pyruvate aldolase, HMG and/or KHG aldolase enzyTne in accordance with the invention can comprise a heterologous SP and/or prepro in a vector, such as a pPIC series vector (Invitrogen, Carlsbad, CA).<br>
[0474]	In some embodiments, SPs and/or prepro sequences in accordance with the<br>
invention are identified following identification of novel aldolase, such as pyruvate aldolase, such as HMG and/or KHG aldolase polypeptides. The pathways by which proteins are sorted and transported to their proper cellular location are often referred to as protein targeting pathways. One of the most important elements in al! of these targeting systems is a short amino acid sequence at the amino terminus of a newly synthesized polypeptide called the signal sequence. This signal sequence directs a protein to ife appropriate location in the cell and is removed during transport or when the protein reaches its final destination. Most lysosomal, membrane, or secreted proteins have an amino-terminal signal sequence that marks them for translocation into the lumen of the endoplasmic reticulum. The signal sequences can vary in length ftom about 10 to 65, or more, amino acid residues. Various methods of recognition of signal sequences are known to those of skill in the art. For example, in some embodiments, novel aldolase, such as pyruvate aldolase, such as HMG and/or KHG aldolase enzyme signal peptides are identified by a method referred to as SignalP. SignalP uses a combined neural network which recognizes both signal peptides and their cleavage sites. (Nielsen (1997) "Identification of prokaryotic and eukaryotic signal peptides and prediction of their cleavage sites." Protein Engineering 10:1-6.<br>
[0475]	In some embodiments, aldolase, such as pyruvate aldolase, such as HMG and/or<br>
KHG aldolase enzymes in accordance with the invention do not have SPs and/or prepro<br>
 <br>
sequences or "domains." In some embodiments, the in-vention provides the aldolase, such<br>
as pyruvate aldolase, such as HMG and/or KHG aldolase enzymes in accordance with the<br>
invention lacking all or part of an SP md/or a prepro domain, hi some embodiments, the<br>
invention provides nucleic acid sequences encoding a signal sequence (SP) and/or prepro<br>
from one aldolase, such as pyruvate aldolase, such as HMG and/or KHG aldolase enzyme<br>
operably linked to a nucleic acid sequence of a different aldolase, such as pyruvate<br>
aldolase, such as HMG and/or KHG aldolase enzyme or, optionally, a signal sequence<br>
(SPs) and/or prepro domain from a non-aldolase, such as non-pyruvate aldolase, e.g, non-<br>
HMG and/or non-KHG aldolase protein may be desired.<br>
(0476]	The invention also provides isolated, synthetic or recombinant polypeptides<br>
comprising signal sequences (SPs), prepro domain and/or catalytic domains (CDs) in accordance with the invention and hetsrologous sequences. The heterologous sequences are sequences not naturally associated (such as to a enzyme) with an SP, prepro domain and/or CD. The sequence to which tlie SP, prepro domain and/or CD are not naturally associated can be on the SP's, prepro domain and/or CD's amino terminal end, carboxy terminal end, and/or on both ends of the SP and/or CD. In some embodiments, the invention provides isolated, synthetic or recombinant polypeptides comprising (or consisting of) a polypeptide comprising a signal sequence (SP), prepro domain and/or catalytic domain (CD) in accordance with the invention with the proviso that it is not associated with any sequence to which it is naturally associated (such as an aldolase, such as pyruvate aldolase, HMG and/or KHG aldolase enone sequence). Similarly, in some embodiments, the invention provides isolated, synthetic or recombinant nucleic acids encoding these polypeptides. Thus, in sonie embodiments, the isolated, synthetic or recombinant nucleic acid in accordance with the invention comprises coding sequence for a signal sequence (SP), prepro domain and/or catalytic domain (CD) in accordance with the invention and a heterologous sequence (i.e., a sequence not naturally associated with the a signal sequence (SP), prepro domain and/or catalytic domain (CD) in accordance with the invention). The heterologous sequence can be on the 3' terminal end, 5' terminal end, and'or on both ends of the SP, prepro domain and/or CD coding sequence.<br>
 <br>
Hybrid (chimeric) aldolase, such as pyruvate aldolase, such as HMG and/or KHG aldolase enzymes and peptide libraries<br>
[0477]	In some embodiments, the invention provides hybrid aldolase, such as pyruvate<br>
aldolase, such as HMG and/or KHG aldolase enzymes and fiision proteins, including peptide libraries, comprising sequences in accordance with the invention. The peptide libraries in accordance with the invention can be used to isolate peptide modulators (such as activators or inhibitors) of targets, such as aldolase, such as pyruvate aldolase, such as HMG and/or KHG aldolase enzyme substrates, receptors, enzymes. The peptide libraries in accordance with the invention can be used to identify formal binding partners of targets, such as ligands, such as cytokines, hormones and the like. In some embodiments, the invention provides chimeric proteins comprising a signal sequence (SP), prepro domain and/or catalytic domain (CD) in accordance with the invention or a combination thereof and a heterologous sequence (see above).<br>
[0478]	In some embodiments, the fusion proteins in accordance with the invention (such<br>
as the peptide moiety) are conformationally stabilized (relative to linear peptides) to allow a higher binding affinity for targets. In some embodiments, the invention provides fusions of aldolase, such as pyruvate aldolase, such as HMG and/or KHG aldolase enzymes in accordance with the invention and other peptides, including known and random peptides. They can be fused in such a manner that the structure of the aldolase, such as pyruvate aldolase, such as HMG and/or KHG aldolase enzymes is not significantly perturbed and the peptide is metabolically or structurally conformationally stabilized. This allows the creation of a peptide library that is easily monitored both for its presence within cells and its quantity.<br>
[0479]	Amino   acid   sequence  variants   in   accordance  with   the   invention   can  be<br>
characterized by a predetermined nature of the variation, a feature that sets them apart from a naturally occurring form, such as an allelic or interspecies variation of an aldolase, such as pyruvate aldolase, HMG and/or KHG aldolase enzyme sequence. In some embodimeni3, the variants in accordance with the invention exhibit the same qualitative biological activity as the naturally occurring analogue. Alternatively, the variants can be selected for having modified characteristics, hi some embodiments, while the site or region for introducing an amino acid sequence variation is predetermined, the mutation per se need not be predetermmed. For example, in order to optimize the performance of a mutation at a given site, random mutagenesis may be conducted at the target codon or<br>
 <br>
region and the expressed aldolase, sucb as pyruvate aldolase, such as HMG and/or KHG<br>
aldolase enzyme variants screened for the optimal combination of desired activity.<br>
Techniques for making substitution mutations at predetermined sites in DNA having a<br>
known sequence are well known, as discussed herein for example, M13 primer<br>
mutagenesis and PCR mutagenesis. Screening of the mutants can be done using, such as<br>
assays of carbon-carbon bond formation or cleavage. In other embodiments, amino acid<br>
substitutions can be single residues; insertions can be on the order of from about I to 20<br>
amino acids, although considerably larger insertions can be done. Deletions can range<br>
from about 1 to about 20, 30, 40, 50, 60, 70 residues or more. To obtain a final derivative<br>
with the optimal properties, substitutions, deletions, insertions or any combination thereof<br>
may be used. Generally, these changes are done on a few amino acids to minimize the<br>
alteration of the molecule. However, larger changes may be tolerated in certain<br>
circumstances.<br>
{0480]	The invention provides aldolase, such as pyruvate aldolase, such as HMG and/or<br>
KHG aldolase enzymes where the structure of the polypeptide backbone, the secondary or the tertiary structure, such as an alpha-helical or beta-sheet structure, has been modified. In some embodiments, the charge or hydrophobicity has been modified. In some embodiments, the bulk of a side chain has been modified. Substantia! changes in ftmction or immunological identity are made by selecting substitutions that are less conservative. For example, substitutions can be made which more significantly affect: the structure of the poljeptide backbone in the area of the alteration, for example a alpha-helical or a beta-sheet structure; a charge or a hydrophobic site of the molecule, which can be at an active site; or a side chain. In some embodiments, the invention provides substitutions in polypeptide in accordance with the invention where (a) a hydrophilic residues, such as seryl or threonyl, is substituted for (or by) a hydrophobic residue, such as leucyl, isoleucyl, phenylalanyl, valyl or alanyl; (b) a cysteine or proline is substituted for (or by) any other residue; (c) a residue having an electropositive side chain, such as lysyl, arginyl, or histidyl, is substituted for (or by) an electronegative residue, such as glutamyl or aspartyl; or (d) a residue having a bulky side chain, such as phenylalanine, is substituted for (or by) one not having a side chain, such as glycine. The variants can exhibit the same qualitative biological activity (i.e., an aldolase, such as pyruvate aldolase, such as HMG mdJor KHG aldolase enzyme, activity) although variants can be ,<br>
 <br>
selected to modify the characteristics of the aldolase, such as pyruvate aldolase, such as HMG and/or KHG aldolase enzymes as needed.<br>
[04811	In some embodiments, aldolase, such as pyruvate aldolase, such as HMG and/or<br>
KHG aldolase enzymes in accordance with the invention comprise epitopes or purification tags, signal sequences or other fusion sequences, etc. In some embodiments, the aldolase, such as pymvate aldolase, such as HMG and/or KHG aldolase enzymes in accordance with the invention can be fused to a random peptide to form a fusion polypeptide. By "fused" or "operably linked" herein is meant that the random peptide and the aldolase, such as pyruvate aldolase, such as HMG and/or KHG aldolase enzyme are linked together, in such a manner as to minimize the disruption to the stability of the aldolase, such as pyravate aldolase, such as HMG and/or KHG aldolase enzyme structure, such as it retains aldolase, such as pyruvate aldolase, such as HMG and/or KHG aldolase enzyme, activity. The fusion polypeptide (or flision polynucleotide encoding the fusion polypeptide) can comprise further components as well, including multiple peptides al multiple loops.<br>
[0482]	In  some  embodiments,  the  peptides  and  nucleic  acids  encoding  them  are<br>
randomized, either fially randomized or they are biased in their randomization, such as in nucleotide/residue frequency generally or per position. "Randomized" means that each nucleic acid and peptide consists of essentially random nucleotides and amino acids, respectively. In some embodiments, the nucleic acids which give rise to the peptides can be chemically synthesized, and thus may incorporate any nucleotide at any position. Thus, when the nucleic acids are expressed to form peptides, any amino acid residue may be incorporated at any position. The synthetic process can be designed to generate randomized nucleic acids, to allow the formation of all or most of the possible combinations over the length of the nucleic acid, thus forming a library of randomized nucleic acids. The library can provide a sufficiently structurally diverse population of randomized expression products to affect a probabilistically sufficient range of cellular responses to provide one or more ceils exhibiting a desired response. Thus, the invention provides interaction libraries large enough so that at least one of its members will have a structure that gives it affinity for some molecule, protein, or other factor.<br>
[0483]	In some embodiments, an aldolase, such as pyruvate aldolase, HMG and/or KHG<br>
aldolase enzyme in accordance with the invention is a multidomain enzyme that<br>
 <br>
comprises a signal peptide, a carbohydrate binding module, an aldolase, such as pyruvate aldolase, HMG and/or KHG aldolase enzyme catalytic domain, a linker and/or another catalytic domain.<br>
[0484]	The   invention   provides   methods   and   sequences   for   generating   chimeric<br>
polypeptides which may encode biologically active hybrid polypeptides (such as hybrid aldolase, such as pyruvate aldolase, such as HMG and/or KHG aldolase enzymes). In some embodiments, the origina] polynucleotides (such as a nucleic acid in accordance with the invention) encode biologically active polypeptides. In some embodiments, a method in accordance with the invention produces new hybrid polypeptides by utilizing cellular processes which integrate the sequence of the original polynucleotides such that the resulting hybrid polynucleotide encodes a polypeptide demonstrating activities derived, but different, from the original bioIogJcalJy active polypeptides (such as aldolase or antibody in accordance with the invention). For example, the original polynucleotides may encode a particular enzyme (such as aldolase) from or found in different microorganisms. An enzjne encoded by a first polynucleotide from one organism or variant may, for example, function effectively under a particular environmental condition, such as high salinity. An enzyme encoded by a second polynucleotide from a different organism or variant may function effectively under a different environmental condition, such as extremely high temperatures. A hybrid polynucleotide containing sequences from the first and second original polynucleotides may encode an enzyme which exhibits characteristics of both enzymes encoded by the original polynucleotides. Thus, the enzyme encoded by the hybrid polynucleotide in accordance with the invention may function effectively under environmental conditions shared by each of the enzymes encoded by the first and second polynucleotides, such as high salinity and extreme temperatures.<br>
[0485]	In some embodiments, a hybrid polypeptide generated by a method in accordance<br>
with the invention may exhibit specialized enzyme activity not displayed in the original enzymes. For example, following recombination and/or reductive reassortment of polynucleotides encoding aldolase, such as pyruvate aldolase, such as HMG and/or KHG aldolase enzymes, the resulting hybrid polypeptide encoded by a hybrid polynucleotide can be screened for specialized non-aldolase, such as non-pyruvate aldolase, such as non-HMG   and/or  non-KHG-aldolase   enzyme   activities,   such  as   hydrolase,   peptidase.<br>
 <br>
phosphorylase, etc., activities, obtained from each of the original enzymes, hi some<br>
embodiments, the hybrid polypeptide is screened to ascertain those chemical<br>
functionalities which distinguish the hybrid polypeptide from the original parent<br>
poiypeptides, such as the temperature, pH or salt concentration at which the hybrid<br>
polypeptide functions.<br>
[0486]	In some embodhnente,  the invention relates to a method  for producing a<br>
biologically active hybrid polypeptide and screening such a polypeptide for enhanced activity by:<br>
1)	introducing at least a first polynucleotide in operable linkage and a second polynucleotide in operable linkage, the at least first polynucleotide and second polynucleotide sharing at least one region of partial sequence homology, into a suitable host cell;<br>
2)	growing the host cell under conditions which promote sequence reorganization resulting in a hybrid polynucleotide in operable linkage;<br>
3)	expressing a hybrid polypeptide encoded by the hybrid polynucleotide;<br>
4)	screening the hybrid polypeptide under conditions which promote identification of enhanced biological activity; and<br>
5)	isolating the a polynucleotide encoding the hybrid polypeptide.<br>
Isolating and discovering aldolase enzymes<br>
[0487]	The invention provides methods for isolatmg and discovering aldolase, such as<br>
pyruvate aldolase, HMG and/or KHG aldolase enTnes and the nucleic acids that encode them. Polynucleotides or enzymes may be isolated from individual organisms ("isolates"), collections of organisms that have been grown in defined media ("enrichment cultures"), or, uncultivated organisms ("environmental samples"). The organisms can be isolated by, such as in vivo biopanning (see discussion, below). The use of a culture-independent approach to derive polynucleotides encoding novel bioactivities from environmental samples is most preferable because it allows one to access untapped resources of biodiversity. Polynucleotides or enzymes also can be isolated from any one of numerous organisms, such as bacteria. In addition to whole cells, polynucleotides or enzymes also can be isolated from crude enzyme preparations derived from cultures of these organisms, such as bacteria.<br>
 <br>
[0488]	"Environnjental   libraries"   are   generated   from   environmental   samples   and<br>
represent the collective genomes of naturally occurring organisms archived in cloning vectors that can be propagated in suitable prokaryotic hosts. Because the cloned DNA is initially extracted directly from environmental samples, the libraries are not limited to the small fraction of prokaryotes that can be grown in pure culture. Additionally, a normalization of the environmental DNA present in these samples could allow more equal representation of the DNA from all of the species present in the original sample. This can dramatically increase the efficiency of finding interesting genes from minor constituents of the sample which may be under-represented by several orders of magnitude compared to the dominant species.<br>
[0489]	In some embodiments, gene libraries generated from one or more uncultivated<br>
microorganisms are screened for an activity of interest. Potential pathways encoding bioactive molecules of interest are first captured in prokaryotic cells in the form of gene expression libraries, hi some embodiments, polynucleotides encoding activities of interest are isolated from such libraries and infroduced into a host cell. The host ceil is grown imder conditions which promote recombination and/or reductive reassortment creating potentially active biomolecules with novel or enhanced activities.<br>
[0490]	In vivo biopanning may be performed utilizing a FACS-based and non-optical<br>
(such as magnetic) based machines. In some embodiments, complex gene libraries are constructed with vectors which contain elements which stabilize franscribed RNA. For example, the inclusion of sequences which result in secondary structures such as hairpins which are designed to flank the franscribed regions of the RNA would serve to enhance their stability, thus increasing their half life within the cell. The probe molecules used in the biopanning process consist of oligonucleotides labeled with reporter molecules that only fluoresce upon binding of the probe to a target molecule. These probes are infroduced mto the recombinant cells from the library using one of several ttansformation methods. The probe molecules bind to the transcribed target mRNA resulting in DNA/RNA heteroduplex molecules. Binding of the probe to a target will yield a fluorescent signal which is detected and sorted by the FACS machine during the screening process.<br>
[0491]	In some embodiments, subcloning is performed to iurther isolate sequences of<br>
interest.  In subcloning, a portion of DNA is amphfied, digested, generally by resfriction<br>
 <br>
enzymes, to cut out the desired sequence, the desired sequence is Hgated into a recipient vector and is amplified. At each step in subcloning, the portion is examined for the activity of interest, in order to ensure tliat DNA that encodes the structural protein has not been excluded. The insert may be purified at any step of the subcloning for example, by gel electrophoresis prior to ligation into a vector or where cells containing the recipient vector and cells not containing the recipient vector are placed on selective media containing, for example, an antibiotic, which will kill the cells not containing the recipient vector. Specific methods of subcloning cDNA inserts into vectors are well-known in the art (Sambrook et al.. Molecular Cloning: A Laboratory Manual, 2nd Ed, Cold Spring Harbor Laboratory Press (1989)). hi other embodiments, the enzymes in accordance with the invention are subclones. Such subclones may differ from the parent clone by, for example, length, a mutation, a tag or a label.<br>
[0492|	The microorganisms from which the polynucleotide may be discovered, isolated<br>
or prepared include prokaryotic microorganisms, such as Eubacteria and Archaehacteria and lower eiikaryotic microorganisms such as fungi, some algae and protozoa. Polynucleotides may be discovered, isolated or prepared from environmental samples in which case the nucleic acid may be recovered without cultuiing of an organism or recovered from one or more cultured organisms. In some embodiments, such microorganisms may be extremophiles, such as hyperthermophiles, psychrophiles, psychrotrophs, halophiles, barophiles and acidophiles. Polynucleotides encoding enzymes isolated from extremophilic microorganisms can be used. Enmes of this invention can function at temperatures above 100°C, such as those found in terrestrial hot springs and deep sea thermal vents, or at temperatures below 0'C, such as those found in arctic waters, in a saturated salt en'ironment, such as those found in the Dead Sea. at pH values around 0, such as those found in coal deposits and geothermal sulfur-rich springs, or at pH values greater than 11, such as those found in sewage sludge. In some embodiments, enzymes in accordance with the invention have high activity throughout a wide range of temperatures and pHs.<br>
[0493]	Polynucleotides selected and isolated as hereinabove described are introduced into<br>
a suitable host cell. A suitable host cell is any cell which is capable of promoting recombination and/or reductive reassortment. The selected polynucleotides are, in some embodiments, already in a vector which includes appropriate control sequences. The host<br>
 <br>
cell can be a higher eukaryotic cell, such as a mammalian cell, or a lower eukaryotic cell, such as a yeast cell, or, in some embodiments, the host cell can be a prokarj'Otic cell, such as a bacterial cell. Introduction of the construct into the host cell can be effected by calcium phosphate transfection, DEAE-Dextran mediated transfection, or electrDporation.<br>
[0494]	Exemplary hosts include bacterial cells, such  E. coli, Streptomyces, Salmonella<br>
typhimurium; fungal cells, such as yet; insect cells such as Drosophila S2 and Spodopiera Sf9; animal cells such as C;H0, COS or Bowes melanoma; adenoviruses; and plant cells; see discussion, above. The selection of an appropriate host is deemed to be within the scope of those skilled in the art from the teachings herein.<br>
[0495]	Various mammalian cell culture systems can be employed to express recombinant<br>
protein; examples of mammalian expression systems include the COS-7 lines of monkey kidney fibroblasts, described in "SV40-transformed simian cells support the replication of early SV40 mutants" (Gluzman, 1981) and other cell lines capable of expressing a compatible vector, for example, the CI27, 3T3, CHO, HeLa and BHK cell lines. Mammalian expression vectors can comprise an origin of replication, a suitable promoter and enhancer and also any necessary ribosome binding sites, polyadenylation site, splice donor and acceptor sites, transcriptional termination sequences and 5' flanking nontranscribed sequences. DNA sequences derived from the SV40 splice and polyadenylation sites may be used to provide the required nontranscribed genetic elements.<br>
[0496]	In other embodiments, nucleic acids, polypeptides and methods in accordance<br>
with the invention are used in biochemical pathways, or to generate novel polynucleotides encoding biochemical pathways from one or more operons or gene clusters or portions thereof For example, bacteria and many eukaryotes have a coordinated mechanism for regulating genes whose products are involved in related processes. The genes are clustered, in structures referred to as "gene clusters," on a single chromosome and are transcribed together under the control of a single regulator}' sequence, including a single promoter which initiates transcription of the entire cluster. Thus, a gene cluster is a group of adjacent genes that are either identical or related, usually as to their function (an example of a biochemical pathway encoded by gene clusters are polyketides).<br>
[0497]	In some embodiments, gene cluster DNA is isolated from different organisms and<br>
iigated into vectors, such as vectoi containing expression regulatory sequences which<br>
 <br>
can control and regulate the production of a detectable protein or protein-related array activit)' from the ligated gene clusters. Use of vectors which have an exceptionally large capacity for exogenous DNA introduction can be appropriate for use with such gene clusters and are described by way of example herein to include the f-factor (or fertility factor) of E. coli. This f-factor of E. coli is a piasmid which affects high-ftequency transfer of itself during conjugation and is ideal to achieve and stably propagate large DNA fragments, such as gene clusters irom mixed microbial samples. In one embodiment, cloning vectors, referred to as "fosmids" or bacterial artificial chromosome (BAG) vectors are used. These are derived from E. coli f-factor which is able to stably integrate large segments of genomic DNA. When integrated with DNA from a mixed uncultured envirormiental sample, this makes it possible to achieve large genomic fragments in the form of a stable "environmental DNA library." Another type of vector for use in the present invention is a cosmid vector. Cosmid vectors were originally designed to clone and propagate large segments of genomic DNA. Cloning into cosmid vectors is described in detail in Sambrook et al. Molecular Cloning: A Laboratory Manual, 2nd Ed., Cold Spring Harbor Laboratory Press (1989). Once ligated into an appropriate vector, two or more vectors containing different polyketide synthase gene clusters can be introduced into a suitable host cell. Regions of partial sequence homology shared by the gene clusters will promote processes which result in sequence reorganization resulting in a hybrid gene cluster. The novel hybrid gene cluster can then be screened for enhanced activities not found in the original gene clusters.<br>
[0498]	Methods for screening for various enzyme activities are known to those of skill in<br>
the art and are discussed throughout the present specification, see Examples 1, 2 and 3, below. Such methods may be employed when isolating the polypeptides and polynucleotides in accordance with the invention.<br>
[0499]	In some embodiments, the invention provides methods for discovering and<br>
isolating aldolases, such as pyruvate aldolase, such as HMG and/or KHG aldolase, or compounds to modify the activity of these enzymes, using a whole cell approach (see discussion, below). Putative clones encoding aldolase, such as pyruvate aldolase, such as HMG and/'or KHG aldolase from genomic DNA library can be screened.<br>
 <br>
Screening Methodologies and "On-line" Monitoring Devices<br>
[0500]	In practicing the methods in accordance with the invention, a variety of apparatus<br>
and methodologies can be used to in conjunction with the polypeptides and nucleic acids in accordance with the invention, such as to screen polypeptides for aldolase, such as pyruvate aldolase, such as HMG and/or KHG aldolase enzyme, activity, to screen compounds as potential modulators, such as activators or inhibitors, of an aldolase, such as pyruvate aldolase, such as HMG and/or KHG aldolase enzyme, activity, for antibodies that bind to a polypeptide in accordance with the invention, for nucleic acids that hybridize to a nucleic acid in accordance with the invention, to screen for cells expressing a polypeptide in accordance with the invention and the like. In addition to the array formats described in detail below for screening samples, alternative formats can also be used to practice the methods in cordance with the invention. Such formats include, for example, mass spectrometers, chromalographs, such as high-throughput HPLC and other forms of liquid chromatography, and smaller formats, such as 1536-wel] plates, 384-weII plates and so on. High throughput screening apparatus can be adapted and used to practice the methods in accordance with the invention, see U.S. Patent Application Nos. 20020001809; 20050272044.<br>
Capillary Arrays<br>
[0501]	Nucleic   acids   or  polypeptides   in   accordance  with   the   invention   can  be<br>
immobilized to or applied to an array. Arrays can be used to screen for or monitor libraries of compositions (such as small molecules, antibodies, nucleic acids, etc.) for their ability to bind to or modulate the activity of a nucleic acid or a polypeptide in accordance with the invention. Capillary arrays, such as the GIGAMATRK'", Diversa Corporation, San Diego, CA; and arrays described in, such as U.S. Patent Application No. 20020080350 Al; WO 0231203 A; WO 0244336 A, provide an alternative apparatus for holding and screening samples. In some embodiments, the capillary array includes a plurality of capiiiaries formed into an array of adjacent capillaries, wherein each capillaiy comprises at least one wall defining a lumen for retaining a sample. The lumen may be cylindrical, square, hexagonal or any other geometric shape so long as the walls form a lumen for retention of a liquid or sample. The capillaries of the capillary airay can be held together in close proximity to form a planar structure.  The capillaries can be bound<br>
 <br>
together, by being ftised (such as where the capillaries are made of glass), glued, bonded, or clamped side-by-side. Additionally, the capillary array can include interstitial material disposed between adjacent capillaries in the array, thereby forming a solid planar device containing a plurality of through-holes.<br>
[0502]	A capillary array can. be formed of any number of individual capillaries, for<br>
example, a range from 100 to 4,000,000 capillaries. Further, a capillary array having about 100,000 or more individtia! capillaries can be formed into the standard size and shape of a Microtiter® plate for fitment into standard laboratory equipment. The lumens are filled manually or automatically using either capillar' action or microinjection using a thin needle. Samples of interest may subsequently be removed from individual capillaries for fiirther analysis or characterization. For example, a thin, needle-like probe is positioned in fluid communication with a selected capillary to either add or withdraw material from the lumen.<br>
[0503]	In a single-pot screening assay, the assay components are mixed yielding a<br>
solution of interest, prior to insertion into the capillary array. The lumen is filled by capillary action when at least a portion of the array is immersed into a solution of interest. Chemical or biological reactions and/or activity in each capillary are monitored for detectable events. A detectable event is often referred to as a "hit", which can usually be distinguished from "non-hit" producing capillaries by optical detection. Thus, capillary arrays allow for massively parallel detection of "hits".<br>
[0504]	In a multi-pot screening assay, a polypeptide or nucleic acid, such as a ligand, can<br>
be introduced into a first component, which is introduced into at least a portion of a capillary of a capillary array. An air bubble can then be introduced into the capillary behind the first component. A second component can then be introduced into the capillary, wherein the second component is separated from the first component by the air bubble. The first and second components can then be mixed by applying hydrostatic pressure to both sides of the capillary array to collapse the bubble. The capillary array is then monitored for a detectable event resulting from reaction or non-reaction of the two components.<br>
[0505]	hi a binding screening assay, a sample of interest can be introduced as a first<br>
liquid labeled with a detectable particle into a capillary of a capillary array, wherein the lumen of the capillary is coated with a binding material for binding the detectable particle<br>
 <br>
to the iumen. The first Uquid may then be removed from the capillary tube, wherein the bound detectable particle is maintained within the capillary, and a second liquid may be introduced into the capillary tube. The capillary is then monitored for a detectable event resulting from reaction or non-reaction of the particle with the second liquid.<br>
Arrays, or "Biochips"<br>
(0506J	Nucleic   acids   or   polypeptides   in  accordance   with   the   invention   can   be<br>
immobilized to or applied to an airay. Arrays can be used to screen for or monitor libraries of compositions (such as small molecules, antibodies, nucleic acids, etc.) for their ability to bind to or modulate Ihe activity of a nucleic acid or a polypeptide in accordance with the invention. For example, in some embodiments of the invention, a monitored parameter is transcript expression of an aldolase, such as pyruvate aldolase, HMG and/or KHG aldolase enzyme gene. One or more, or, all the transcripts of a cell can be measured by hybridization of a sample comprising franscripts of the cell, or, nucleic acids representative of or complementary to transcripts of a cell, by hybridization to immobilized nucleic acids on an array, or "biochip." By using an "array" of nucleic acids on a microchip, some or all of the transcripts of a cell can be simultaneously quantified. Alternatively, arrays comprising genomic nucleic acid can also be used to determine the genotype of a newly engineered strain made by the methods in accordance with the invention. Polypeptide arrays" can also he used to simultaneously quantify a plurality of proteins. The present im-ention can be practiced with any known "array," also referred to as a "microarray" or "nucleic acid array" or "polypeptide array" or "antibody array" or "biochip," or variation thereof. Arrays are generically a plurality of "spots" or "target elements," each target element comprising a defined amount of one or more biological molecules, such as oligonucleotides, immobilized onto a defined area of a substrate surface for specific binding to a sample molecule, such as mRNA franscripts.<br>
[0507]	The terms "array" or "microarray" or "biochip" or "chip" as used herein is a<br>
plurality of target elements, each target element comprising a defined amount of one or more polypeptides (including antibodies) or nucleic acids immobilized onto a defined area of a substrate surface, as discussed in fiirther detail, below.<br>
[0508]	In practicing the methods in accordance with the invention, any known array<br>
and/or method of making and using arrays can be incorporated in whole or in part, or variations thereof, as described, for example, in U.S. Patent Nos. 6,277,628; 6,277,489;<br>
 <br>
6,261J76: 6,258,606; 6,054,270; 6,048,695; 6,045,996; 6,022,963; 6,013,440; 5,965,452; 5,959,098; 5,856,174; 5,830,645; 5,770,456; 5,632,957; 5,556,752; 5,143,854; 5,807.522; 5,800,992; 5,744,305: 5,700,637; 5,556,752; 5,434,049; see also, such as WO 99/51773; WO 99/09217; WO 97/46313; WO 96/17958; see aiso, such as Johnston (1998) Curr. Bio]. S:R]71-R174; Schummer (1997) Biotechniques 23;]087-]092; Kern (1997) Biotechniques 23:120-124; SoUnas-Toldo (1997) Genes, Chromosomes &amp; Cancer 20:399-407; Bowtell (1999) Nature Genetics Supp. 21:25-32. See also published U.S. patent applications Nos. 20010018642; 2D0I0D19827; 20010016322; 20010014449; 20010014448; 20010012537; 20010008765.<br>
Antibodies and Antibody-based screesnine methods<br>
[0509]	The   invention  provides   isolated,   synthetic   or  recombinant  antibodies  that<br>
specifically bind to an aldolase, such as pyruvate aldolase, HMG and/or KHG aldolase enzyme in accordance with the invention. These antibodies can be used to isolate, identify or quantify the aldolase, such as pyruvate aldolase, such as HMG and/or KHG aldolase enzymes in accordance with the invention or related polypeptides. These antibodies can be used to isolate other polypeptides within the scope the invention or other related aldolase, such as pyruvate aldolase, such as HMG and/or KHG aldolase enzymes. TTie antibodies can be designed to bind to an active site of an aldolase, such as pyruvate aldolase, HMG and/or KHG aldolase enzyme. Thus, the invention provides methods of inhibiting aldolase, such as pyruvate aldolase, such as HMG and/or KHG aldolase enzymes using the antibodies in accordance with the in'ention (see discussion above regarding applications for anti-aldolase, such as anti-pyruvate aldolase, such as anti-HMG and/or anti-KHG aldolase enzyme compositions in accordance with the invention).<br>
[0510]	The term "antibody" includes a peptide or polypeptide derived from, modeled<br>
after or substantially encoded by an immunoglobulin gene or immunoglobulin genes, or fragments thereof, capable of specifically binding an antigen or epitope, see Fundamental hnmunology, Third Edition, W.E. Paul, ed., Raven Press, N.Y. (1993); Wilson (1994) J. Immunol. Methods 175:267-273; Yarmush (1992) J. Biochem. Biophys. Methods 25:85-97. The term antibody includes antigen-binding portions, i.e., "antigen binding sites," (such as fragments, subsequences, complementarity determining regions (CDRs)) that retain capacity to bind antigen, including (i) a Fab fragment, a monovalent fragment<br>
 <br>
consisting of the VL, VH, CL and CHI domains; (ii) a F(ab')2 fragment, a bivalent fragment comprising two Fab fragments linked by a disulfide bridge at the hinge regioji; (iii) a Fd fragment consisting of the VH and CHI domains; (iv) a Fv fragment consisting of the VL and VH domains of a single arm of an antibody, (v) a dAb fragment (Ward et al., (1989) Nature 341:544-546), which consists of a VH domain; and (vi) an isolated complementarity determming region (CDR). Single chain antibodies are also included by reference in the term "antibody."<br>
[0511]	The  invention provides  fragments  of the enzymes  in  accordance with the<br>
invention (such as peptides) including immunogenic fragments (such as subsequences) of a polypeptide in accordance with the invention. In some embodiments, the invention provides compositions comprising a polypeptide or peptide in accordance with the invention and adjuvants oi carriers and the like.<br>
[0512]	The antibodies can be used in immunoprecipitation, staining, immunoaffinity<br>
columns, and the. like. If desired, nucleic acid sequences encoding for specific antigens can be generated by immunization followed by isolation of polypeptide or nucleic acid, amplification or cloning and immobilization of polypeptide onto an array in accordance with the invention. Alternatively, the methods in accordance with the invention can be used to modify the structure of an antibody produced by a cell to be modified, such as an antibody's affinity can be increased or decreased. Furthermore, the ability to make or modify antibodies can be a phenotype engineered into a cell by the methods in accordance with the invention.<br>
[0513]	Methods of immunization, producing and isolating antibodies (polyclonal and<br>
monoclonal) are known to those of skill in the art and described in the scientific and patent literature, see Coligan, CURRENT PROTOCOLS IN IMMUNOLOGY, Wiley/Greene, NY (1991); Stites (eds.) BASIC AND CLINICAL IMMUNOLOGY (7th ed.) Lange Medical Publications, Los Altos, CA ("Stites"); Coding, MONOCLONAL ANTIBODIES: PRINCIPLES AND PRACTICE (2d ed.) Academic Press, New York, NY (1986); Kohler (1975) Nature 256:495; Harlow (1988) ANTIBODIES, A LABORATORY MANUAL, Cold Spring Harbor Publications, New York. Antibodies also can be generated in vitro, such as using recombinant antibody binding site expressing phage display libraries, in addition to the traditional in vivo methods using animals.  See,<br>
 <br>
Hoogenboom (1997) Trends Biotechiioi. 15:62-70; Katz (1997) Annu. Rev. Biophys. Biomol. Struct. 26:27-45.<br>
[0514]	The polypeptides in accordance with the invention or fragments comprising at<br>
Jeast 5, 10, 15, 20,25, 30,35,40, 50, 75, 100, or 150 consecutive amino acids thereof, may also be used to generate antibodies which bind specifically to the polypeptides or fragments. The resulting antibodies may be used in immunoaffmity chromatography procedures to isolate or purify the polypeptide or to determine whether the polypeptide is present in a biological sample, hi such procedures, a protein preparation, such as an extract, or a bioiogicai sample is contacted with an antibody capable of specifically binding to one of the polypeptides in accordance with the invention, or fragments comprising at least 5, 10, 15, 20, 25, 30, 35, 40, 50, 75, 100, or 150 consecutive amino acids thereof<br>
[05151	In immunoaffinity procedures, the antibody is attached to a solid support, such as a<br>
bead or other column matrix. The protein preparation is placed in contact with the antibody under conditions in which the antibody specifically binds to one of the polypeptides in accordance with the invention, or fragment thereof. After a wash to remove non-speciiica]]y bound proteins, the speciiically bound polypeptides are eluted.<br>
[0516]	The ability of proteins in a biological sample to bind to the antibody may be<br>
determined using any of a variety of procedures familiar to those skilled in the art. For example, binding may be determined by labeling the antibody with a detectable label such as a fluorescent agent, an enzymatic label, or a radioisotope. Alternatively, binding of the antibody to the sample may be detected using a secondary antibody having such a detectable label thereon. Particular assays include ELISA assays, sandwich assays, rioimmunoassays and Western Blots.<br>
[0517]	Polyclonal antibodies generated agamst the polypeptides in accordance with the<br>
invention, or fragments comprising at least 5,10,15,20,25, 30,35,40, 50, 75,100, or 150 consecutive amino acids thereof can be obtained by direct injection of the polypeptides into an animal or by administering the polypeptides to an animal, for example, a nonhuman. The antibody so obtained can bind the polypeptide itself In this manner, even a sequence encoding only a fragment of the polypeptide can be used to generate antibodies which may bind to the whole native polypeptide. Such antibodies can then be used to isolate the polypeptide from ceils expressing that polypeptide.<br>
 <br>
|0518]	For   preparation  of  monoclonal   antibodies,   any   technique   which   provides<br>
antibodies produced by continuous cell line cultures can be used. Examples include the hybridoma technique (Kohler and Milstein, Nature, 256:495-497, 1975), the tiioma technique, the human B-cell hybridoma technique (Kozbor et al. Immunology Today 4:72, 1983) and the EBV-hybridoma technique (Cole, et al, I9S5, in Monoclonal Antibodies and Cancer Therapy, Alan ll. Liss, Inc., pp. 77-96).<br>
[0519]	Techniques described for the production of single chain antibodies (U.S. Patent<br>
No. 4,946,778) can be adapted to produce single chain antibodies to the polypeptides in accordance with the invention, or fragments comprising at least 5, 10, 15, 20, 25, 30, 35, 40, 50, 75, 100, or 150 consecutive amino acids thereof. Alternatively, transgenic mice may be used to express hxmianized antibodies to these polypeptides or fragments thereof,<br>
[0520]	Antibodies generated against the polypeptides in accordance with the invention, or<br>
fragments comprising at least 5, 10, 15, 20, 25, 30, 35, 40, 50, 75,100, or 150 consecutive amino acids thereof may be used in screening for similar polypeptides from other organisms and samples. In such techniques, polypeptides from the organism are contacted with the antibody and those polypeptides which specifically bind the antibody are detected. Any of the procedures described above may be used to detect antibody binding. One such screening assay is described in Shulman H, Eberhard A, Eberhard C, Ulitzur S, Keinan E, Bioorg Med Chem Lett. 2000 Oct l6;10(20):2353-6. Highly sensitive and rapid detection of antibody catalysis by luminescent bacteria.<br>
Kits<br>
[0521]	The invention provides kits comprising the compositions, such as nucleic acids,<br>
expression cassettes, vectors, cells, transgenic seeds or plants or plant parts, polypeptides (such as an aldolase enzyme) and/or antibodies in accordance with the invention. The kits also can contain instructional material teaching the methodologies and industrial, medical and dietary uses in accordance with the invention, as described herein.<br>
Whole cell engineering and measuring metabolic parameters<br>
[0522]	The methods in accordance with the invention provide whole ceil evolution, or<br>
whole cell engineering, of a cell to develop a new cell strain havmg a new phenotype, such as a new or modified aldolase, such as pyruvate aldolase, such as HMG and/or KHG<br>
 <br>
aldolase enzyme, activity, by modiiying the genetic composition of the cell. See U.S. patent application no. 20040033975.<br>
f0523]	The genetic composition can be modified by addition to the cell of a nucleic acid<br>
in accordance with the invention, such as a coding sequence for an enzyme in accordance with the invention. See WO0229032; WO0196S51.<br>
10524}	To detect the nevi' phenotype, at least one metabolic parameter of a modified cell<br>
is monitored in the cell in a "real time" or "on-line" time frame. In some embodiments, a plurality of cells, such as a cell culture, is monitored in "real time" or "on-line." In some embodiments, a plurality of metabolic parameters is monitored in "real time" or "on¬line." Metabolic parameters can be monitored using the aldolase, such as pyruvate aldolase, such as HMG and/or KHG aldolase enzymes in accordance with the invention.<br>
{0525]	Metabolic flux analysis (MFA) is based on a known biochemistry fiameworic.  A<br>
linearly independent metabolic matrix is constructed based on the law of mass conservation and on the pseudo-steady state hypothesis (PSSH) on the intracellular metabolites. In practicing the methods in accordance with the invention, metabolic networks are established, including the:<br>
" identity of all pathway substrates, products and intermediary metabolites<br>
•	identity of all the chemical reactions interconverting the pathway metabolites, the stoichiometry of the pathway reactions,<br>
•	identitj' of all the enzymes catalyzing the reactions, the enzyme reaction kinetics,<br>
•	the regulatory interactions between pathway components, such as allosteric interactions, enzyme-enzyme interaciions etc,<br>
•	intracellular compartmentalization of enzymes or any other supramolecular<br>
organization of the enzymes, and,<br>
•	the presence of any concentration gradients of metabolites, enzymes or effector<br>
molecules or difftjsion barriers to their movement.<br>
[0526]	Once the metabolic network for a given strain is built, mathematic presentation by<br>
matrix notion can be introduced to estimate the intracellular metabolic fluxes if the on¬line metabolome data is available. Metabolic phenotype relies on the changes of the whole metabolic network within a cell. Metabolic phenotype relies on the change of pathway utilization with respect to environmental conditions, genetic regulation, developmental state and the genotype, etc.   In some embodiments of the methods in<br>
 <br>
accordance with the invention, after tlie on-line MFA calculation, the dynamic behavior<br>
of the cells, their phenotype and other properties are analyzed by investigating the<br>
pathway utilization. For example, if the glucose supply is increased and the oxygen<br>
decreased during the yeast fermentation, the utilization of respirator)' pathways will be<br>
reduced and/or stopped, and the utilization of the fermentative pathways will dominate.<br>
Control of physiological state of cell cultures will become possible after the pathway<br>
analysis. The methods in accordance with the invention can help determine how to<br>
manipulate the fermentation by determining how to change the substrate supply,<br>
temperature, use of inducers, etc. to control the physiological state of cells to move along<br>
desirable direction. In practicing the methods in accordance with the invention, the MFA<br>
results can also be compared with transcriptome and proteome data to design experiments<br>
and protocols for metabolic engineering or gene shuffling, etc,<br>
[0527]	In practicing the methods in accordance with the invention, any modified or new<br>
phenotype can be conferred and detected, including new or improved characteristics in the cell. Any aspect of metabolism or growth can be monitored.<br>
Monitoring expression of an mRNA transcript<br>
[0528]	In some embodiments of the invention, the engineered phenotype comprises<br>
increasing or decreasing the expression of an mRNA transcript (such as an aldolase, such as pyruvate aldolase, HMG and/or KHG aldolase enzyme message) or generating new (such as aldolase, such as pyruvate aldole, such as HMG and/or KHG aldolase enzyme) transcripts in a cell. This increased or decreased expression can be traced by testing for the presence of an aldolase, such as pyruvate aldolase, HMG and/or KHG aldolase enzyme in accordance with the invention or by aldolase, such as pyruvate aldolase, such as HMG and/or KHG aldolase enzyme activity assays. mRNA transcripts, or messages, also can be detected and quantified by any method known in the art, including, such as Northern blots, quantitative ampUfication reactions, hybridization to arrays, and the like. Quantitative amplification reactions include, such as quantitative PCR, including, such as quantitative reverse transcription polymerEee chain reaction, or RT-PCR; quantitative real time R.T-PCR, or "real-time kinetic RT-PCR" (see Kreuzer (2001) Br. J. Haematol. 114:313-318; Xia (2001) Transplantation 72:907-914).<br>
[0529]	In some embodiments of the invention, the engineered phenotype is generated by<br>
knocking out expression of a homologous gene.  The gene's coding sequence or one or<br>
 <br>
more  transcriptional control  elements can be knocked out,  such as promoters or<br>
enhancers.    Thus, the expression of a transcript can be completely ablated or only<br>
decreased.<br>
|0530]	In some embodiments of the invention, the engineered phenotype comprises<br>
increasing the expression of a homologous gene. This can be effected by knocking out of a negative control element, including a transcriptional regulatory element acting in cis- or trans-, or, mutagenizing a positive control element. One or more, or, all the transcripts of a cell can be measured by hybridization of a sample comprising transcripts of the cell, or, nucleic acids representative of or complementary to transcripts of a cell, by hybridization to immobilized nucleic acids on an array.<br>
Monitoring expression of a polypeptides, peptides and amino acids<br>
[0531]	In some embodiments of the invention, the engineered phenotype comprises<br>
increasing or decreasing the expression of a poljeptide (such as an aldolase, such as pyruvate aldolase, HMG and/or KHG aldolase enzyme) or generating new polypeptides in a cell. This increased or decreased expression can be traced by determining the amount of aldolase, such as pyruvate aIdola;ie, such as HMG and/or KHG aldolase enzyme present or by aldolase, such as pypjvate aldolase, such as HMG and/or KHG aldolase enzyme activity assays. Polypeptides, peptides and amino acids also can be detected and quantified by any method known in the art, including, such as nuclear magnetic resonance (NMR), spectrophotometry, radiography (protein radiolabeling), electrophoresis, capillary electrophoresis, high performance liquid chromatography (HPLC), thin layer chromatography (TLC), hyperdifEiision chromatography, various immunological methods, such as immunoprecipitation, immunodiffusion, immuno-electrophoresis, radioimmtmoassays (RIAs), enzyme-linked immunosorbent assays (ELISAs), immimo-fluorescent assays, gel electrophoresis (such as SDS-PAGE), staining with antibodies, fluorescent activated cell sorter (FACS), pyrolysis mass spectrometry, Fourier-Transform Infrared Spectrometry, Raman spectrometry, GC-MS, and LC-Electrospray and cap-LC-tandem-electrospray mass spectrometries, and the like. Novel bioactivities can also be screened using methods, or variations thereof, described in U.S. Patent No. 6,057,103. Furthermore, as discussed below in detail, one or more, or, ail the polypeptides of a ceil can be measured using a protein array.<br>
 <br>
Industrial. Pharmaceutical and other Applications<br>
[0532]	Polypeptides in accordance with the invention (such as having aldolase, such as<br>
pyruvate aldolase, such as HMG and/or KHG aldolase) can catalyze the formation or cleavage of carbon-carbon bonds. The eniymes in accordance with the invention can be highly selective catalysts. In some embodiments, the invention provides industrial processes using enzymes in accordance with the invention, such as in the pharmaceutical or nutrient (diet) supplement industry, in the food and feed industries, such as in methods for making food and feed products and food and feed additives. In some embodiments, the invention provides processes using enzymes In accordance with the invention in the medical industry, such as to make pharmaceuticals or dietary aids or supplements, or food supplements and additives.<br>
Biomass conversion and production of clean bio fuels<br>
[0533]	The invention provides enzymes, such aldolases, including pyruvate aldolases<br>
such as, without limitation, HMG and/or KHG aldolases (including mixtures, or "cocktails" of enzymes) and methods for the conversion of a biomass or any lignocellulosic material (e.g., any composition comprising cellulose, hemicellulose and lignin), to fiiels (e.g., bioethanol, biobutanol, biopropanol, biomethanol, biodiesel), using the enzymes of the invention, in addition to feeds, foods and chemicals. Thus, the compositions and methods of the invention provide effective and sustainable alternatives or adjuncts to use of petroleum-based products, e.g., as a mixture of bioethanol and gasoline. The invention provides organisms expressing enzymes of the invention for participation m chemical cycles involving natural biomass conversion. In one embodiment, enzymes and methods for the conversion are used in enzyme ensembles for the efficient depolymerization of cellulosic and hemicellulosic polymers to metabolizeable carbon moieties. The invention provides methods for discovering and implementing the most effective of enzymes to enable these important new "biomass conversion" and alternative energy industrial processes.<br>
[0534]	The methods of the invention also include taking the converted lignocellulosic<br>
material (processed by enzymes oi the invention) and making it into a fuel (e.g. bioethanol, biobutanol, biopropanol, biomethanol, biodiesel) by fermentation and/or by<br>
 <br>
chemical synthesis.   In one embodiment, the produced sugars are fermented and/or the<br>
non-fermentab!e products are gasified.<br>
[0535]	The  enzymes of the invention (including, for example,  organisms, such as<br>
microorganisms, e.g., fimgi, yeast or bacteria, making and in some embodiments secreting recombinant enzymes of the invention) can be used in or included/ integrated at any stage of any biomass conversion process, e.g., at any one step, several steps, or included in all of the steps, or all of the following methods of biomass conversion processes, or all of these bioftiel alternatives:<br>
•	Direct combustion: the burning of material by direct heat and is the simplest biomass technology; can be very economical if a biomass source is nearby.<br>
•	Pyrolysis: is the thermal degradation of biomass by heat in the absence of oxygen. In one embodiment, biomass is heated io a temperature between about 800 and 1400 degrees Fahrenheit, but no oxygen is introduceid to support combustion resulting in the creation of gas, fuel oil and charcoal.<br>
•	Gasification: biomass can be used to produce methane through heating or anaerobic digestion. Sjngas, a mixture of carbon monoxide and hydrogen, can be derived from biomass.<br>
•	Landfill Gas: is generated by the decay (anaerobic digestion) of buried garbage in landfills. When the organic waste decomposes, it generates gas consisting of approximately 50% methane, the ma,ior component of natural gas.<br>
•	Anaerobic digestion; converts organic matter to a mixture of methane, the major component of natural gas, and carbon dioxide. In one embodiment, biomass such as waterwaste (sewage), manure, or food processing waste, is mixed with water and fed into a digester tank without air.<br>
•	Fermentation<br>
•	Alcohol Fermentation: fuel alcohol is produced by converting starch to sugar, fermenting the sugar to alcohol, then separating the alcohol water mixture by distillation. Feedstocks  such  as wheat, barley, potatoes, and waste paper, sawdust, and straw<br>
 <br>
containing sugar, starcli, or cellulose can be converted to alcohol by fermentation with yeast.<br>
•	Transeslerification; An exemplary reaction for converting oil to biodiesel is called transesterification. The transesterification process reacts an alcohol (like methanol) with the triglyceride oils contained in vegetable oils, animal fats, or recycled greases, forming fatty acid alkyl esters (biodiesel) and glycerin. The reaction requires heat and a strong be catalyst, such as sodium hydroxidis or potassium hydroxide.<br>
•	Biodiesel: Biodiesel is a mixture of fatty acid alkyl esters made from vegetable oils, animal fats or recycled greases. Biodiesel can be used as a fuel for vehicles in its pure form, but it is usually used as a petroleum diesel additive to reduce levels of particulates, carbon monoxide, hydrocarbons and air toxics from diesel-powered vehicles.<br>
•	Hydrolysis: includes hydrolysis of a compound, e.g., a biomass, such as a lignocellulosic material, catalyzed using an enzyme of the instant invention.<br>
•	Congeneration: is the simultaneous production of more than one form of energy using a single fuel and faciJily. In one embodiment, biomass cogeneration has more potential growth than biomass generation alone because cogeneration produces both heat and electricity.<br>
[0536]	hi one embodiment, the polypeptides of the invention have an aldolase activity,<br>
including pyruvate aldolase activity, such as, without limitation, HMG and/or KHG aldolase activity, or other enzymatic activity for generating biodiesel, bioethanol, biobutanol, biopropanol, or biomethauol, from an organic material, e.g., a biomass, such as compositions derived from plants and animals, including any agricultural crop or other renewable feedstock, an agricultural residue or an animal waste, or the organic components of municipal and industrial wastes, or microorganisms such as algae or yeast. In one embodiment, polypeptides of the invention are used in processes for converting lignocellulosic biomass to sthanol, butanol, propanol, methanol or otherwise are used in processes for hydrolyzing or digesting biomaterials such that they can be used as a biofuel (including bioethanol, biobutanol, biopropanol, biomethanol, or biodiesel), or for making it easier for the biomass to be processed into a fuel. In an alternative embodiment,<br>
 <br>
polypeptides of the invention are used in processes for a transesteriiJcatJon process reacting an alcohol (like methanol) with a triglyceride oil contained in a vegetable oil, animal fat or recycled greases, forming fatty acid alkyl esters (biodiesel) and glycerin. In one embodiment, biodiesel is made from soybean oil or recycled cooking oils. Animal's fats, other \'egetable oils, and other recycled oils can also be used to produce biodiesel, depending on their costs and availability. In another embodiment, blends of all kinds of fats and oils are used to produce a biodiesel fuel of the invention.<br>
[0537]	Enzymes of the invention can also be used in glycerin refining. The glycerin by-<br>
product contains unreacted catalyst and soaps that are neutralized with an acid. Water and alcohol are removed to produce SO'ii to 80% crude glycerin. The remaining contaminants include unreacted fats and oils, which can be processes using the polypeptides of the invention. In large biodiesel plants of the invention, the glycerin can be further purified, e.g., to 99% or higher purity, for tiie pharmaceutical and cosmetic industries.<br>
|0538]	Bioethanol, biobutanol, biopropanol, biomethanol, and/or biodiesel are made<br>
using the polypeptides of the invention can be used with fuel oxygenates to improve combustion characteristics. Adding oxygen results in more complete combustion, which reduces carbon monoxide emissions. This is another environmental benefit of replacing petroleum fiiels with biofuels (e.g., a fuel of the invention). A bioethanol, biobutanol, biopropanol, biomethanol, and/or biodiesel made using the compositions and/or methods of this invention can be blended with gasoline to form an ElO blend (about 5% to 10% ethanol and about 90% to 95% gasoline), but it can be used in higher concentiations such as E85 or in its pure form. A bioethanol, biobutanol, biopropanol, biomethanol, and/or biodiesel made usmg the compositions and/or methods of this invention can be blended with petroleum diesel to form a B20 blend (20% biodiesel and 80% petroleum diesel), although other blend levels can be used up to BlOO (pure biodiesel).<br>
[0539]	The invention also provides processes for making ethanol ("bioethanol"), butanol<br>
("biobutanol"), propanol ("biopropanol"), methanol ("biomethanol"), and/or diesel ("biodiesel") from compositions comprising lignocellulosic biomass. The Hgnocellulose biomass material can be obtained from agricultural crops, as a byproduct of food or feed production, or as lignocellulosic waste products, such as plant residues and waste paper. Examples of suitable plant sources or plant residues for treatment with polypeptides of the invention include kelp, algae, grains, seeds, stems, leaves, hulls, husks, com cobs,<br>
 <br>
com stover, straw, grasses (e.g., Indian grass, such as Sorghastrum nutans; or, switch grass, e.g., Panicum species, such as Panicum virgatum), and the like, as well as wood, wood chips, wood pulp, and sawdust. Examples of paper waste suitable for treatment with polypeptides of the invention include discard photocopy paper, computer printer paper, notebook paper, notepad paper, typewriter paper, and the like, as well as newspapers, magazines, cardboard, and paper-based packaging materials.<br>
[0540]	In one embdiment, the enzymes and methods of the invention can be used in<br>
conjunction with more "traditiona]" means of making ethanol, methanol, butanol, propanol and/or diesel from biomass, e.g. as methods comprising hydrolyzing lignocellulosic materials by subjecting dried lignocellulosic material in a reactor to a catalyst comprised of a dilute solution of a strong acid and a metal salt; this can lower the activation energy, or the temperature, of cellulose hydrolysis to obtain higher sugar yields; see, e.g., U.S. PatentNos. 6,660,506; and 6,423,145.<br>
[0541]	Another embodiment that Incorporates use of enzymes of the invention comprises<br>
hydrolyzing lignocellulosic materia! containing hemicellulose, cellulose and lignin by subjecting the material to a first stage hydrolysis step in an aqueous medium at a temperature and a pressure chosen to effect primarily depolymerization of hemicellulose without major depolymerization of cellulose to glucose. This step results in a slurry in which the liquid aqueous phase contains dissolved monosaccharides resulting from depolymerization of hemicellulose and a solid phase containing cellulose and lignin. A second stage hydrolysis step can comprise conditions such that at least a major portion of the cellulose is depolymerized, such step resulting in a liquid aqueous phase containing dissolved/ soluble depolymerization producte of cellulose. See, e.g., U.S. Patent No. 5,536,325. Enzymes of the invention can be added at any stage of this exemplary process.<br>
[0542]	Another embodiment that incorporates use of enzymes of the invention comprises<br>
processing a lignocellulose-containing biomass material by one or more stages of dilute acid hydrolysis with about 0.4% to 2% strong acid; and treating an unreacted solid lignocellulosic component of the acid hydrolyzed biomass material by alkaline delignification to produce precursors for biodegradable thermoplastics and derivatives. See, e.g., U.S. Patent No. 6,409,841. Enzymes of the invention can be added at any stage of tJiis exemplary process.<br>
 <br>
[0543]	Another embodJmeDt that incorporates use of enzymes of the invention comprises<br>
prehydrolyzing lignocellulosic material in a prehydroiysis reactor; adding an acidic liquid to the solid lignocellulosic material to make a mixture; heating the mixture to reaction temperature; maintaining reaction temperature for time suificient to fractionate the lignocellulosic material into a solubilized portion containing at least about 20% of the lignin from the lignocellulosic material and a solid fraction containing cellulose; removing a solubilized portion from the solid fraction while at or near reaction temperature wherein the cellulose in the solid fraction is rendered more amenable to enzymatic digestion; and recovering a solubilized portion. See, e.g., U.S. Patent No. 5,705,369. Enzymes of the invention can be added at any stage of this exemplary process.<br>
[0544]	The invention provides methods for making motor fuel compositions (e.g., for<br>
spark ignition motors) based on liquid hydrocarbons blended with a fuel grade alcohol made by using an enzyme or a method of the invention. In one embodiment, the fuels made by use of an enzyme of the invention comprise, e.g., coal gas liquid- or natural gas liquid-ethanol, methanol, butanol, propanol and/or diesel blends. In one embodiment, a co-solvent is biomass-derived 2-methyltetrahydrofriran (MTHF). See, e.g., U.S. Patent No. 6,712,866.<br>
[0545]	In one embodiment, methods of the invention for the enzymatic degradation of<br>
lignocellulose, e.g., for production of ethanol from lignocellulosic material, can also comprise use of ultrasonic treatment of the biomass material; see, e.g., U.S. Patent No. 6,333,IgI.<br>
[0546]	In another embodiment, methods of the mvention for producing bioethanol,<br>
biobutanol, biopropanol, biomethanolj and/or biodiesel from a cellulosic substrate comprise providing a reaction mixture 'm the form of a slurry comprising cellulosic substrate, an enzyme of this invention and a fermentation agent (e.g., within a reaction vessel, such as a semi-continuousl&gt;' solids-fed bioreactor), and the reaction mixture is reacted under conditions sufficient to initiate and maintain a fermentation reaction (as described, e.g., in U.S. Pat. App. No. 20060014260). In one embodiment, experiment or theoretical calculations can determine an optimum feeding frequency. In one embodiment, additional quantities of the cellulosic substrate and the enzyme are provided into the reaction vessel at an interval(s) according to the optimized feeding frequency.<br>
 <br>
[0547]	One exemplary process for making biofiiels (such as bioethanol, biobutanol,<br>
biopropanol, biometbanol, and/or biodiesel) of the invention is described in U.S. Pat. App. Pub. Nos. 20050069998; 20020164730; and in one embodiment comprises stages of grinding the lignocellulosic biomass (e.g., to a size of 15-30 mm), subjecting the product obtained to steam explosion pre-treatment (e.g., at a temperature of I90-230oC) for between 1 and 10 minutes in a reactor; collecting the pre-treated materiaJ in a cyclone or related product of manufacture; and separating the liquid and solid fractions by filtration in a filter press, introducing the solid fraction in a fermentation deposit and adding one or more enzymes of the invention, e.g., a cellulase and/or beta-giucosidase enzyme (e.g., dissolved In citrate buffer pH 4.8).<br>
[0548]	Another exemplary process for making biofuels (such as bioethanol, biobutanol,<br>
biopropanol, biomethanol, and/or biodiesel) of the invention comprising using enzymes of the invention comprises pretreating a starting material comprising a lignocellulosic feedstock comprising at least hemicellulose and cellulose. In one embodiment, the starting material comprises potatoes, .soybean (rapeseed), barJey&gt; rye, com, oats, wheat, beets or sugar cane or a component oi- waste or food or feed production byproduct. The starting material ("feedstock") is reacted at conditions which disrupt the plant's fiber structure to effect at least a partial hydrolysis of the hemicellulose and cellulose. Disruptive conditions can comprise, e.g., subjecting the starting material to an average temperature of 180 °C to 270 °C at pH 0.5 to 2.5 for a period of about 5 seconds to 60 minutes; or, temperature of 220 °C to 270 X, at pH 0.5 to 2.5 for a period of 5 seconds to 120 seconds, or equivalent. This generates a feedstock with increased accessibility to being digested by an enzyme, e.g., a cellulase enzyme of the invention. U.S. Patent No. 6,090,595.<br>
[0549]	Exemplary conditions for hydrolysis of lignocellulosic material include reactions<br>
at temperatures between about 30 "C and 48 °C, and/or a pH between about 4.0 and 6.0. Other exemplary conditions include a temperature between about 30 "C and 60 °C and a pH between about 4.0 and 8.0.<br>
[0550]	The enzymes in accordance with Hic invention  can catalyze reactions with<br>
exquisite stereo-, regio- and chemo- selectivities. The aldolase, such as pyruvate aldolase, such as HMG and/or KHG aldolase enzymes in accordance with the invention can be engineered to function in various solvents, operate at extreme pHs (for example.<br>
 <br>
high pHs and low pHs) extreme temfteratures (for example, high temperatures and low temperatiires), extreme salinity levels (for example, high salinity and low salinity) and catalyze reactions with compounds that are structurally unrelated to their natural, physiological substrates.<br>
Feeds and food or feed and food additives<br>
[0551]	In addition to providing dietary aids or supplements, or food supplements and<br>
additives, the invention also provides compositions and methods for treating human and animal feeds and foods and food or feed additives using a polypeptide in accordance with the invention, such as a protein having aldolase activity, such as pyruvate aldolase, such as HMG and/or KHG aldolase enzymes in accordance with the invention, and/or the antibodies in accordance with the invention. In some embodiments, the invention provides animal feeds, foods, and additives comprising aldolase, such as pyruvate aldolase, such as HMG and/or KHG aldolase enzymes in accordance with the invention and/or antibodies in accordance with the invention. The animal can be any farm animal or any animal.<br>
[0552]	The animal feed additive in accordance with the invention may be a granulated<br>
enzyme product that may readily be mixed wath feed components. Alternatively, feed additives in accordance with the invention can form a component of a pre-mix. The granulated enzyme product in accordance with the invention may be coated or uncoated. The particle size of the enzyme granulates can be compatible with that of feed and pre-mix components. This provides a sai'e and convenient mean of incorporating enzymes into feeds. Alternatively, the animal feed additive in accordance with the invention may be a stabilized liquid composition. This may be an aqueous or oil-based slurry. See U.S. Patent No. 6,245,546.<br>
[0553]	Aldolase, such as pyruvate aldolase, such as HMG and/or KHG aldolase enzymes<br>
of the present invention, in the modification of feed or a food, can process the food or feed either in vitro (by modiiying components of the feed or food) or in vivo. Polypeptides in accordance with the invention can be added to feed or food compositions.<br>
[0554]	In some embodiments, an enzyme in accordance with the invention is added in<br>
combination with another enzyme, such as beta-galactosidases, catalases, laccases, ceiiuiases, other aldolases, endoglycosidases, endo-beta-l,4-iaccases, amyiogiucosidases, glucosidases,    glucose    isomerases,    glycosyltransferases,    lipases,    phospholipases.<br>
 <br>
tipooxygenases, beta-laccases, endo-beta-I,3(4)-laccases, cutinases, peroxidases, amylases, phytases, glucoamylases, pectinases, reductases, oxidases, decarboxylases, phenoloxidases, ligninases, puUulanases, arabinanases, hemicellulases, mannanases, xylolaccases, xylanases, pectin acetyl esterases, rhanmogalacturonan acetyl esterases, proteases, peptidases, proteinases, polygalacturonases, rhamnogalacturonases, galactanases, pectin lyases, transglutaminases, pectin methylesterases, cellobiohydrolases and/or transglutaminases. These enz&gt;Tne digestion products are more digestible by the animal. Thus, aldolase, such as pyruvate aldolase, such as HMG and/or KHG aldolase enzymes in accordance with the invention can contribute to the available energy- of the feed or food, or to the digestibility of ttie food or feed by breaking down cellulose.<br>
[0555]	In other embodiments, aldolase, such as pyruvate aldolase, such as HMG and/or<br>
KHG aldolase enone in accordance with the invention can be supplied by expressing the enzymes directly in transgenic feed crops (as, such as transgenic plants, seeds and the like), such as grains, cereals, com, so&gt;' bean, rape seed, lupin and the like. As discussed abo'e, the invention provides transgenic plants, plant parts and plant cells comprising a nucleic acid sequence encoding a polypeptide in accordance with the invention. In some embodiments, the nucleic acid is expressed such that the aldolase, such as pyruvate aldolase, such as HMG and/or KHG aldolase enzyme in accordance with the invention is produced in recoverable quantities. The aldolase, such as pyruvate aldolase, such as HMG and/or KHG aldolase enzyme can be recovered ftom any plant or plant part. Alternatively, the plant or plant part containing the recombinant polypeptide can be used as such for improving the quality of a food or feed, such as improving nutritional value, palatability, etc.<br>
[0556]	In  some embodiments,  the enzyme delivery matrix  in accordance with the<br>
invention is in the form of discrete plural particles, pellets or granules. By "granules" is meant particles that are compressed or compacted, such as by a pelletizing, extrusion, or similar compacting to remove water m the matrix. Such compression or compacting of the particles also promotes intraparticle cohesion of the particles. For example, the granules can be prepared by pelletizhig the grain-based substrate in a pellet mill. The pellets prepared thereby are ground of crumbled to a granule size suitable for use as an adjuvant in animal feed. Because the matrix is itself approved for use in animal feed, it can be used as a diluent for delivery of enzymes in animal feed.<br>
 <br>
[0557]	In some embodiments, the aldolase, such as pyruvate aldolase, such as HMG<br>
and/or KHG aldolase enzyme contained in the invention enzyme delivery matrix and methods is a thermostable aldolase, such as pyruvate aldolase, such as HMG and/or KHG aldolase enzyme, as described herein, so as to resist inactivation of the aldolase, such as pyruvate aldolase, such as HMG and/or KHG aldolase enzyme during manufacture where elevated temperatures and/or steam may be employed to prepare the palletized enzyme delivery matrix. During digestion of feed containing the invention enzyme delivery matrix, aqueous digestive fluids will cause release of the active enzyme. Other types of thermostable enzymes and nutritional supplements that are thermostable can also be incorporated in the delivery matrix for release under any type of aqueous conditions.<br>
[0558]	In some embodunents, a coating is applied to the enzyme matrix particles for<br>
many different purposes, such as to add a flavor or nutrition supplement to animal feed, to delay release of animal feed supplements and enzjmes in gastric conditions, and the like. In some embodiments, the coating is applied to achieve a fionctional goal, for example, whenever it is desirable to slow release of the enzyme from the matrix particles or to control the conditions under which the enzyme will be released. The composition of the coating material can be such that it is selectively broken down by an agent to which it is susceptible (such as heat, acid or base, engines or other chemicals). Alternatively, two or more coatings susceptible to different such breakdown agents may be consecutively applied to the matrix particles.<br>
[0559]	The invention is also directed towards a process for preparing an enzyme-<br>
releasing matrix. In accordance with the invention, the process comprises providing discrete plural particles of a grain-based substrate in a particle size suitable for use as an enzyme-releasing matrix, wherein the particles comprise an aldolase, such as pyruvate aldolase, HMG and/or KHG aldolase enzyme encoded by an amino acid sequence in accordance with the invention. In some embodiments, the process includes compacting or compressing the particles of enzyme-releasing matrix into granules, which most In some embodiments is accomplished by pelletizing. The mold inhibitor and cohesiveness agent, when used, can be added at any suitable time, and, in some embodiments are mixed with the grain-based substrats; in the desired proportions prior to pelletizing of the grain-based substrate. Moisture content in the pellet mill feed in some embodiments is in the ranges set forth above with respect to the moisture content in the finished product.<br>
 <br>
and, in some embodiments, is about 14-15%. In some embodiments, moisture is added to<br>
the feedstock in the form of an aqueous preparation of the enzyme to bring the feedstock<br>
to this moisture content. The temperature in the pellet mill in some embodiments is<br>
brought to about 82°C with steam. The pellet mill may be operated under any conditions<br>
that impart sufficient work to the feedstock to provide pellets. The pelleting process itself<br>
is a cost-effective process for removing water from the enzyme-containing composition.<br>
[0560]	The compositions and methods in accordance with the invention can be practiced<br>
in conjunction with administration of prebiotics, which are high molecular weight sugars, such as fructo-oligosaccharides (I'OS); galacto-oligosaccharides (GOS), GRAS (Generally Recognized As Safe) matetial. These prebiotics can be metabolized by some probiotic lactic acid bacteria (LAB). They are non-digestible by the majority of intestinal microbes.<br>
Treating foods and food processing<br>
[0561]	The invention provides foods and feeds comprising enzymes in accordance with<br>
the invention, and methods for using enzymes in accordance with the invention in processing foods and feeds. Aldolase, such as pyruvate aldolase, HMG and/or KHG aldolase enzymes in accordance with the invention have numerous applications in food processing industry. In some embodiment, the invention provides methods for hydrolyzing cellulose-comprising compositions, including, such as a plant cell, a bacterial cell, a yeast cell, an insect cell, or an animal cell, or any plant or plant part, or any food or feed, a waste product and the like.<br>
|0562]	For example, the invention provides feeds or foods comprising an aldolase, such<br>
as pyruvate aldolase, HMG and/or KHG aldolase enzyme the invention, such as in a feed, a liquid, such as a beverage (such as a fruit juice or a beer), a bread or a dough or a bread product, or a drink (such as a beer) or a. beverage precursor (such as a wort).<br>
[0563]	The food treatment processes in accordance with the invention can also include<br>
the use of any combination of other enzymes such as tryptophanases or tyrosine decarboxylases, laccases, catalases, laccases, other aldolases, cellulases, endoglycosidases, endo-beta-1,4-laccases, amyloglucosidases, glucosidases, glucose isomerases, glycosyltransferases, lipases, phospholipases, Jipooxygenases, beta-Iaccases, endo-beta-l,3(4)-laccases, cutinases, peroxidases, amylases, phytases, glucoamylases, pectinases,     reductases,     oxidases,     decarboxylases,     phenoloxidases,     ligninases.<br>
 <br>
puiiuianases, arabinanases, hemiceiiulases, mannanases, xylolaccases, xylanases, pectin acetyl esterases, rhamnogalacturonaji acetyl esterases, proteases, peptidases, proteinases, polygalacturonases, rhamnogalacturonases, galactanases, pectin lyases, transglutaminases, pectin methylesterases, cellobiohydrolases and/or transglutaminases.<br>
Pharmaceutical compositions and dietary supplements<br>
{0564]	The invention also provides phannaceuticat compositions and dietary supplements<br>
(stzch as dietary aids) comprising an aldolase in accordance with the invention. The aldolase activity comprises pyruvate aldolase, HMG and/or KHG aldolase activity. In some embodiments, the pharmaceutical compositions and dietary supplements (such as dietary aids) ans formulated for oral ingestion.<br>
[0565]	Periodontal treatment compounds can comprise an enzyme in accordance with the<br>
invention, such as described in U.S. patent no. 6,776,979. Compositions and methods for the treatment or prophylaxis of acidic gut syndrome can comprise an enzyme in accordance with the invention, such as described in U.S. patent no. 6,468,964.<br>
[0566]	In   other  embodiments,   wound  dressing,   implants   and   the   like  comprise<br>
antimicrobial (such as antibiotic-acting) enzymes, including an enzyme in accordance with the invention (including, such as sequences in accordance with the invention). Enzymes in accondance wath the invention can also be used in alginate dressings, antimicrobial barrier dressings, bum dressings, compression bandages, diagnostic tools, gel dressings, hydro-selective dressings, hydrocellular (foam) dressings, hydrocolloid dressings, I.V dressings, incise drapes, low adherent dressings, odor absorbing dressings, paste bandages, post operative dressings, scar manement, skin care, transparent film dressings and/or wound closure. Enzymes in accordance with the invention can be used in wound cleansing, wound bed preparation, to treat pressure ulcers, leg ulcers, bums, diabetic foot ulcers, scars, IV fixation, surgical wounds and minor wounds. Enzymes in accordance with the invention can be used to in steriJe enzymatic debriding compositions, such as ointments. In various embodiments, the aldolase is formulated as a tablet, gel, pili, implant, liquid, spray, film, micelle, powder, food, feed pellet or as an encapsulated formulation.<br>
[0567]	The pharmaceutical compositions and dietary supplements in accordance with the<br>
invention can also include the use of any combination of other enzymes such as beta-galactosidases, catalases, laccases, cellulases, other aldolases, endogiycosidases, endo-<br>
 <br>
beta-],4-]accases, amyJoglucosidases, glucosidases, glucose isomerases, glycosyltransferases, lipases, phospholipases, lipooxygenases, beta-laccases, endo-beta-l,3(4)-lacces, cutinases, peroxidases, amylases, phytases, glucoamylases, pectinases, reductases, oxidases, decarboxylases, phenoloxidases, ligninases, pullulanases, arabinanases, hemicellulases, mannanases, xylolaccases, xylanases, pectin acetyl esterases, rhamnogalacturonan acetyl esterases, proteases, peptidases, proteinases, polygalacturonases, rhamnogalacturonases, galactanases, pectin lyases, transglutaminases, pectin methylesterases, cellobiohydrolases and/or transglutaminases.<br>
Biosynfhetic Pathways to Produce R,R and Other Stereoisomers ofMonatin<br>
(0568]	As described, inter alia, in WO 03/091396 A2 {see Figures 1-3 and 1M3),<br>
monatin can be produced from tryptophan through a multi-step pathway involving biological conversions (i.e. facilitating the reaction of a substrate to a product with a polypeptide). A pathway described involves biologically converting tryptophan to indoie-3-pyruvate, biologically converting indoie-3-pyruvate to 2-hydroxy 2-(indoi-3-yimethyi)-4-keto glutaric acid ("MP"), and biologically converting MP to monatin. In some embodiments, polypeptides of the invention can be used to facilitate the reaction of indole-3-pyruvate to fonn MP. In some embodiments, polypeptides of the invention can be used to preferentially facilitate the production of R-MP.<br>
[0569]	In  some embodiments,  one or more polypeptides chosen from  isolated  or<br>
recombinant polypeptides of SEQ ID N0:2, SEQ ID N0:4, SEQ ID N0:6, SEQ ID N0:8, SEQ ID NO:10, SEQ ID N0:12, SEQ ID N0:14, SEQ ID N0:16, SEQ ID N0:18, SEQ ID NO:20, SEQ ID NO:22, SEQ ID NO:24, SEQ ID NO:26, SEQ ID NO:28, SEQ ID NO:30, SEQ ID NO:32, SEQ ID NO:34, SEQ ID NO:36, SEQ ID NO:38, SEQ ID NO:40, SEQ ID NO:42, SEQ ID NO:44, SEQ ID NO:46, SEQ ID NO:48, SEQ ID NO:50, SEQ ID NO:52, SEQ ID NO:54, SEQ ID NO:56, SEQ ID NO:58, SEQ ID NO:60, SEQ ID NO:62, SEQ ID NO:64, SEQ ID NO:66, SEQ ID NO:68, SEQ ID NO:70, SEQ ID NO:72, SEQ ID NO:74, SEQ ID NO:76, SEQ ID NO:78, SEQ ID NO:80, SEQ ID NO:82, SEQ ID NO:84, SEQ ID NO:86, SEQ ID NO:88, SEQ ID NO:90, SEQ ID NO:92, SEQ ID NO:94, SEQ ID NO:96, SEQ ID NO:98, SEQ ID NO:100, SEQ ID KO:102, SEQ ID NO:104, SEQ ID NO:106, SEQ ID NO:108, SEQ ID NO:110, SEQ ID N0:112, SEQ ID N0:114, SEQ ID N0:116, SEQ ID N0:118, SEQ ID NO:120, SEQIDNO:122, SEQIDNO:124, SEQIDNOil26, SEQ IDNO:128, SEQ ID<br>
 <br>
NO:i30, SEQ ID NO:!32, SEQ ID NO:134, SEQ ID NO:136, SEQ ID NO:I38, SEQ ID<br>
NO:140, SEQ ID NO-.I42, SEQ ID NO;144, SEQ ID NO:I46, SEQ ID NO:I48, SEQ ID<br>
NO:150, SEQ ID NO:152, SEQ ID NO:154, SEQ ID HO:i56, SEQ ID NO:158, SEQ ID<br>
MO:I60, SEQ ID NO:162, SEQ ID NO:i64, SEQ ID NO:i66, SEQ ID NO:I68, SEQ ID<br>
NO:170, SEQ ID NO:I72, SEQ ID NO:174, SEQ ID NO:I76, SEQ ID NO:I78, SEQ ID<br>
NO:I80, SEQ ID NO:i82, SEQ ID NO:I84, SEQ ID NO:186, SEQ ID NO:I88, SEQ ID<br>
NO;190, SEQ ID NO:I92, SEQ ID NO:194, SEQ ID NO:I96, SEQ ID NO:198, SEQ ID<br>
NO:200, SEQ ID NO:202, SEQ ID NO:204, SEQ ID NO:206, SEQ ID NO:208, SEQ ID<br>
NO:210, SEQ ID NO:212, SEQ ID NO:2I4, SEQ ID NO:216, SEQ ID NO:2I8, SEQ ID<br>
NO:220, SEQ ID NO:222, SEQ ID NO:224, SEQ ID NO:226, SEQ ID NO:228, SEQ ID<br>
NO:230, SEQ ID NO:232, SEQ ID NO:234, SEQ ID NO:236, SEQ ID NO:238, SEQ ID<br>
NO:240, SEQ ID NO:242, SEQ ID NO:244, SEQ ID NO:246, SEQ ID NO:248, SEQ ID<br>
NO:250, SEQ ID NO:252, SEQ ID NOi254, SEQ ID NO:256, SEQ ID NO:258, SEQ ID<br>
NO:260, SEQ ID NO:262, SEQ ID NO:264, SEQ ID NO:266, SEQ ID NO:268, SEQ ID<br>
NO:270, SEQ ID NO:272, SEQ ID NO:274, SEQ ID NO:276, SEQ ID NO:278, SEQ ID<br>
NO:280, SEQ ID NO:282, SEQ ID NO:284, SEQ ID NO:286, SEQ ID NO:288, SEQ ID<br>
NO:290, SEQ ID NO:292, SEQ ID NO:294, SEQ ID NO:296, SEQ ID NO:298, SEQ ID<br>
NO:300, SEQ ID NO:302, SEQ ID NO:304, SEQ ID NO:306, SEQ ID NO:308, SEQ ID<br>
NO:310, SEQ ID NO:312, SEQ ID NO:3I4, SEQ ID NO:316, SEQ ID NO:3I8, SEQ ID<br>
NO:320, SEQ ID NO:322, SEQ ID NO:324, SEQ ID NO:326, SEQ ID NO:328, SEQ ID<br>
NO:330, SEQ ID NO:332, or SEQ ID NO:334, or fragments or subsequences thereof<br>
having aldolase activity may be use&amp;i in facilitating a reaction vithin a multi-step<br>
pathway to produce a product chosen from monatin, monatin derivatives, salts thereof and<br>
combinations thereof. In one embodiment, the polypeptides with aldolase activity may be<br>
usefiil in facilitating a reaction in which indoie-3-pyruvate is converted to MP as one step<br>
within a multi-step pathway to produce a product chosen from monatin, monatin<br>
derivatives, salts thereof and combinations thereof.<br>
[0570]	In another embodiment, one or more polypeptides chosen from isolated or<br>
recombinant polypeptides with HMG aldolase activity of any of SEQ ID N0:2, SEQ ID N0:4, SEQ ID N0:6, SEQ ID N0:8, SEQ ID NO:10, SEQ ID N0:!2, SEQ ID NO:I4, SEQ ID N0:I6, SEQ ID N0:1S, SEQ ID NO:20, SEQ ID NO:22, SEQ ID NO:24, SEQ ID NO:26, SEQ ID NO:28, SEQ ID NO:30, SEQ ID NO:32, SEQ ID NO:34, SEQ ID<br>
 <br>
NO:36, SEQ ID NO:38, SEQ ID NO:40, SEQ ID NO:42, SEQ ID NO:44, SEQ ID NO:46, SEQ ID NO:48, SEQ ID NO:50, SEQ ID NO:52, SEQ ID NO:54, SEQ ID NO:56, SEQ ID NO:58, SEQ ID NO:60, SEQ ID NO:62, SEQ ID NO:64, SEQ ID NO:66, SEQ ID NO:68, SEQ ID NO:70, SEQ ID NO:72, SEQ ID NO:74, SEQ ID NO:76, SEQ ID N0:7S, SEQ ID NO:80, SEQ ID NO:82, SEQ ID NO:84, SEQ ID N0;S6, SEQ ID NO:88, SEQ ID NO:90, SEQ ID NO;92, SEQ ID NO:94, SEQ ID NO:96, SEQ ID NO:98, SEQ ID NO:100, SEQ ID NO:]02, SEQ ID NO:104, SEQ ID NO:106, SEQ ID NO:I08, SEQ IDNOiIlO, SEQ IDN0:112, SEQ ID N0:114, SEQ ID N0:116, SEQ ID N0:118, SEQ ID NO:120, SEQ ID NO:122, SEQ ID NO:124, SEQ ID NO:126, SEQ ID NO:128, SEQ ID NO:130, SEQ ID N0:n2, SEQ ID N0:n4, SEQ ID NO:I36, SEQ ID NO:138, SEQ ID NO:140, SEQ ID NO:142, SEQ ID NO:144, SEQ ID NO:146, SEQ ID NO:I48, SEQ ID NO:150, SEQ ID NO:152, SEQ ID NO:154, SEQ ID NO:156, SEQ ID NO:158, SEQ ID NO:160, SEQ ID NO:162, SEQ ID NO:164, SEQ ID NO:166, SEQ ID NO;168, SEQ ID NO:170, SEQ ID NO:I72, SEQ ID NO:I74, SEQ ID NO:176, SEQ ID NO; 178, SEQ ID NO;ISO, SEQ ID NO: 182, SEQ ID NO; 184, SEQ ID NO: 186, SEQ ID NO; 188, SEQ ID NO; 190, SEQ ID NO: 192, SEQ ID NO; 194, SEQ ID NO;196, SEQ ID NO:198, SEQ ID NO:200, SEQ ID NO:202, SEQ ID NO:204, SEQ ID NO:206, SEQ ID NO:208, SEQ ID NO:210, SEQ ID NO:2I2, SEQ ID NO:214, SEQ ID NO:216, SEQ ID NO:218, SEQ ID NO;220, SEQ ID NO:222, SEQ ID NO:224, SEQ ID NO:226, SEQ ID NO:228, SEQ ID NO:230, SEQ ID NO:232, SEQ ID NO:234, SEQ ID NO:236, SEQ ID NO:238, SEQ ID NO:240, SEQ ID NO:242, SEQ ID NO:244, SEQ ID NO;246, SEQ ID NO:248, SEQ ID NO:250, SEQ ID NO:252, SEQ ID NO:254, SEQ ID NO:256, SEQ ID NO:258, SEQ ID NO:260, SEQ ID NO:262, SEQ ID NO:264, SEQ ID NO:266, SEQ ID NO:268, SEQ ID NO:270, SEQ E5 NO:272, SEQ ID NO:274, SEQ ID NO;276, SEQ ID NO:27S, SEQ ID NO;280, SEQ ID NO;2S2, SEQ ID NO:284, SEQ ID NO;286, SEQ ID NO:288, SEQ ID NO;290, SEQ ID NO:292, SEQ ID NO:294, SEQ ID NO:296, SEQ ID NO:298, SEQ ID NO:300, SEQ ID NO:302, SEQ ID NO:304 or fragments or subsequences thereof having aldolase activity may be useful in facilitating a reaction between indole-3-pyruvate and a C3 carbon source as one step within a multi-step pathway to produce a product chosen from monatin, monatin derivatives, salts thereof and combinations thereof. In one embodiment, the polypeptides with HMG aldolase activity may be useful in facilitating a reaction in which indole-3-pyruvate is<br>
 <br>
converted to MP as one step within £i multi-step pathway to produce a product chosen from monarin, monatm derivatives, salts thereof and combinations thereof.<br>
[0571]	In yet another embodiment, one or more polypeptides chosen from isolated or<br>
recombinant polypeptides with KHG aldolase activity of any of SEQ ID NO:306, SEQ ID NO:308, SEQ ID N0:310, SEQ ID N0:312, SEQ ID N0:314, SEQ ID N0:316, SEQ ID NO:3I8, SEQ ID NO:320, SEQ ID NO:322, SEQ ID NO:324, SEQ ID NO:326, SEQ ID NO:328, SEQ ID NO:330, SEQ ID NO:332, or SEQ ID NO:334 or fragments or subsequences thereof having aldolase activity may be useful in facilitating a reaction between indole-3-pyruvate and a C3 carbon source as one step within a multi-step pathway to produce a product chosen fi:om monatin, monatm derivatives, salts thereof and combinations thereof. In one embodiment, the polypeptides with KHG aldolase activity may be useiiil in facilitating a reaction in which indole-3-pyruvate is converted to MP as one step within a multi-step pathwaj' to produce a product chosen from monatin, monatin derivatives, salts thereof and combinations thereof<br>
[0572]	Additionally, one or more polypeptides encoded by one or more nucleic acids<br>
sequence having at least about 50%, 51%, 52%, 53%, 54%, 55%, 56%, 57%, 58%, 59%, 60%, 61%, 62%, 63%, 64%, 65%, 66%, 67%, 68%, 69%, 70%, 71%, 72%, 73%, 74%, 75%, 76%, 77%, 78%, 79%, 80%, 81%, 82%o, 83%, 84%, 85%, 86%, 87%., 88%o, 89%, 90%, 91%, 92%, 93%, 94%, 95%, 96%o, 97%, 98%, 99%, or more, or complete (100%) sequence identity to a nucleic acid in accordance with the invention, including SEQ ID N0:1, SEQ ID N0-.3, SEQ ID N0:5, SEQ ID N0:7, SEQ ID N0:9, SEQ ID N0:1U SEQ ID NO:I3, SEQ ID N0;I5, SEQ ID NO:I7, SEQ ID N0:19, SEQ ID NO:2I, SEQ ID NO:23, SEQ ID NO:25, SEQ ID NO:27, SEQ ID NO:29, SEQ ID N0:31, SEQ ID NO:33, SEQ ID NOi35, SEQ ID NO:37, SEQ ID NO:39, SEQ ID N0:41, SEQ ID NO:43, SEQ ID NO:45, SEQ ID NO:47, SEQ ID NO:49, SEQ ID N0:51, SEQ ID NO:53, SEQ ID NO:55, SEQ ID NO:57, SEQ ID NO:59, SEQ ID N0:61, SEQ ID NO:63, SEQ ID NO:65, SEQ ID NO:67, SEQ ID NO:69, SEQ ID N0:71, SEQ ID NO:73, SEQ ID NO:75, SEQ ID NO:77, SEQ ID NO:79, SEQ ID N0:81, SEQ ID NO:83, SEQ ID NO:85, SEQ ID NO:87, SEQ ID NO:89, SEQ ID N0:91, SEQ ID NO:93, SEQ ID NO:95, SEQ ID NO:97, SEQ ID NO:99, SEQ ID NO:101, SEQ ID NO:103, SEQ ID NO:105, SEQ ID NO:I07, SEQ ID NO:109, SEQ ID N0:111, SEQ ID N0:n3, SEQ ID 0:115, SEQ ID NO:!!?, SEQ ID NO:! 19, SEQ ID NO;12I, SEQ ID<br>
 <br>
NO:123, SEQ ID NO:125, SEQ ID NO:127, SEQ ID NO:129, SEQ ID N0:131, SEQ ID NO:I33, SEQ ID NO:135, SEQ IDNO:137, SEQ 1DN0:!39, SEQ IDN0:!41, SEQ ID NO-.143, SEQ ID NO:l45, SEQ ID NO:147, SEQ ID NO:149, SEQ ID N0:151, SEQ ID NO:153, SEQ ID N0:I55, SEQ ID NO:157, SEQ ID NO:159, SEQ ID N0:16I, SEQ ID NO:163, SEQ ID NO:165, SEQ ID NO:167, SEQ ID NO:169, SEQ ID NO:171, SEQ ID NO:173, SEQ IDNO:175, SEQIDNO:177, SEQ IDNO:179, SEQ IDN0:181, SEQ ID NO:183, SEQIDN0:185,SEQIDN0:187, SEQIDNO:189, SEQ IDNO:191, SEQ ID NO:193, SEQ ID NO:l95, SEQ ID NO:197, SEQ ID NO;199, SEQ ID NO:201, SEQ ID NO:203, SEQ ID NO:205, SEQ ID NO;207, SEQ ID NO:209, SEQ ID NO:211, SEQ ID NO:213, SEQ ID NO:2l5, SEQ ID NO:217, SEQ ID NO:219, SEQ ID NO:221. SEQ ID NO:223, SEQ ID NO:225, SEQ ID NO:227, SEQ ID NO:229, SEQ ID NO:231, SEQ ID NO:233, SEQ ID NO:235, SEQ ID NO;237, SEQ ID NO:239, SEQ ID NO:241, SEQ ID NO:243, SEQ ID NO:245, SEQ ID NO:247, SEQ ID NO:249, SEQ ID NO:251, SEQ ID NO:253, SEQ ID NO:255, SEQ ID NO:257, SEQ ID NO:259, SEQ ID NO:261, SEQ ID NO;263, SEQ ID NO:265, SEQ ID NO:267, SEQ ID NO;269, SEQ ID NO:271, SEQ ID NO:273, SEQ ID NO:275, SEQ ID NO:277, SEQ ID NO:279, SEQ ID NO:281, SEQ ID NO:283, SEQ ID NO:285, SEQ ID NO:287, SEQ ID NOi289, SEQ ID NO:29E SEQ ID NO:293, SEQ ID NO:295, SEQ ID NO:297, SEQ ID NO;299, SEQ ID NO:30], SEQ ID NO:303, SEQ ID NO;305, SEQ ID NO:307, SEQ ID NO:309, SEQ ID N0:311, SEQ ID NO:313, SEQ ID NO:3l5, SEQ ID NO:317, SEQ ID NO:319, SEQ ID NO:321, SEQ ID NO:323, SEQ ID NO:325, SEQ ID NO:327, SEQ ID NO:329, SEQ ID NO:33I, SEQ ID NO:333, SEQ ID NO:335, SEQ ID NO:336, SEQ ID NO:337, and SEQ ID NO:338 over a region of at least about 10, 15, 20, 25, 30, 35, 40, 45, 50, 75, 100, 150, 200, 250, 300, 350, 400, 450, 500, 550, 600, 650, 700, 750, 800, 850, 900, 950, 1000, 1050, 1100, 1150, 1200, 1250, 1300, 1350, 1400, 1450, 1500, 1550, 1600, 1650, 1700, 1750, 1800, 1850, 1900, 1950, 2000, 2050, 2100, 2200, 2250, 2300, 2350, 2400, 2450, 2500, or more residues may be usefUl in facilitating a reaction between indoIe-_3-pyruvate and a C3 carbon source as one step within a multi-step pathway to produce a product chosen from monatin, monatin derivatives, salts thereof and combinations thereof. In one embodiment, the one or more polypeptides, or fragments or subsequences thereof with aldolase activity may be useful in facilitating a reaction in which indole-3-pyruvate is<br>
 <br>
converted to MP as one step within a multi-step pathway to produce a product chosen<br>
from monatin, monatin derivatives, salts thereof and combinations thereof.<br>
[0573]	In another embodiment of the invention, one or more polypeptides with HMG<br>
aldolase activit&gt;' encoded by a nucleic acid sequence having at least about 50%, 51%, 52%, 53%, 54%, 55%, 56%, 57%, 5g%, 59%, 60%, 61%., 62%, 63%&gt;, 64%, 65%, 66%, 67%, 68%, 69%, 70%», 71%», 72%, 73%., 74%, 75%, 76%, 77%., 78%, 79%, 80%,, 81%, 82%o, 83%, 84%, 85%, 86%, 87%, 88%, 89%, 90%., 91%., 92%, 93%o, 94%,, 95%, 96%, 97%, 98%), 99%i, or more, or complete (100%) sequence identity to a nucleic acid in accordance with the invention, including SEQ ID N0:1, SEQ ID N0:3, SEQ ID N0:5, SEQ ID N0:7, SEQ ID N0:9, SEQ ID N0:1I, SEQ ID N0:13, SEQ ID N0:15, SEQ ID N0:17, SEQ ID N0:19, SEQ ID N0:21, SEQ ID NO:23, SEQ ID NO:25, SEQ ID NO:27, SEQ ID NO:29, SEQ ID N0:31, SEQ ID NO:33, SEQ ID NO;35, SEQ ID NO:37, SEQ ID NO:39, SEQ ID N0:41, SEQ ID NO:43, SEQ ID NO:45, SEQ ID NO:47, SEQ ID NO:49, SEQ ID N0:51, SEQ ID NO:53, SEQ ID NO:55, SEQ ID NO;57, SEQ ID NO:59, SEQ ID N0:6I, SEQ ID NO:63, SEQ ID NO:65, SEQ ID NO:67, SEQ ID NO:69, SEQ ID N0:71, SEQ ID "NO:73, SEQ ID NO:75, SEQ ID NO:77, SEQ ID NO:79, SEQ ID N0:81, SEQ ID NO:83, SEQ ID NO:85, SEQ ID NO:87, SEQ ID NO;89, SEQ ID N0;91, SEQ ID NO;93, SEQ ID NO:95, SEQ ID NO:97, SEQ ID NO:99, SEQ ID NO:101, SEQ ID NO:103, SEQ ID NO:105, SEQ ID NO:107, SEQ ID NO:109, SEQ ID N0:111, SEQ ID NOillS, SEQ ID N0:115, SEQ ID NO:n7, SEQ ID 0:119, SEQ ID NO:22I, SEQ ID NO:I23, SEQ ID NO;125, SEQ ID NO:127, SEQ ID NO:129, SEQ ID N0;131, SEQ ID NO:i33, SEQ ID NO:135, SEQ ID NO:137, SEQ ID NO:139, SEQ ID N0:141, SEQ ID NO:143, SEQ ID NO:145, SEQ ID NO;I47, SEQ ID N0:M9, SEQ ID N0:151, SEQ ID 10:153, SEQ ID NO:]55, SEQ ID NO:157, SEQ IDNO:159, SEQ IDN0;161, SEQ IDNO:163, SEQ IDNO:165, SEQ ID NO:167, SEQ ID NO:169, SEQ ID N0:171, SEQ ID NO:I73, SEQ ID NO:175, SEQ ID NO:177, SEQ ID NO:I79, SEQ ID N0:I8i, SEQ ID NO:183, SEQ ID N0:1S5, SEQ ID NO:187, SEQ ID NO:189, SEQ ID N0:191, SEQ ID NO:193, SEQ ID NO:195, SEQ ID NO:197, SEQ ID NO:I99, SEQ ID NO:201, SEQ ID NO:203, SEQ ID NO:205, SEQ ID NO:207, SEQ ID NO:209, SEQ ID N0:211, SEQ ID NO:2[3, SEQ ID NO:215, SEQ ID NO:217, SEQ ID NO:219, SEQ ID NO:221, SEQ ID NO:223, SEQ ID NO:225, SEQ ID NO:227, SEQ ID NO:229, SEQ ID NO:231, SEQ ID NO:233, SEQ ID NO-.235, SEQ ID<br>
 <br>
NO:237, SEQ ID NO:239, SEQ ID NO:241, SEQ ID NO;243, SEQ ID NO:245, SEQ ID<br>
NO:247, SEQ ID NO:249, SEQ ID NO:25L SEQ ID NO:253, SEQ ID NO:255, SEQ ID<br>
NO:257, SEQ ID NO:259, SEQ ID NO:261, SEQ ID NO:263, SEQ ID NO:265, SEQ ID<br>
NO:267, SEQ ID NO;269, SEQ ID NO;27U SEQ ID NO:273, SEQ ID NO:275, SEQ ID<br>
NO:277, SEQ ID NO:279, SEQ ID NO:281, SEQ ID NO:2g3, SEQ ID NO:285, SEQ ID<br>
NO:287, SEQ ID NO:289, SEQ ID NO:291, SEQ ID NO:293, SEQ ID NO:295, SEQ ID<br>
NO:297, SEQ ID NO:299, SEQ ID NO:301, SEQ ID NO:303, SEQ ID NO:305 over a<br>
region of at least about 10, 15, 20, 25, 30, 35, 40, 45, 50, 75, 100, 150, 200, 250, 300,<br>
350, 400, 450, 500, 550, 600, 650, 700, 750, 800, 850, 900, 950, 1000, 1050, 1100, 1150,<br>
1200, 1250, 1300, 1350, 1400, 1450, 1500, 1550, 1600, 1650, 1700, 1750, 1800, 1850,<br>
1900, 1950, 2000, 2050, 2100, 2200, 2250, 2300, 2350, 2400, 2450, 2500, or more<br>
residues may be useful in facilitating a reaction between indole-3-pyruvate and a C3<br>
carbon source as one step within a multi-step pathway to produce a product chosen from<br>
monatin, monatin derivatives, salts thereof and combinations thereof. In one<br>
embodiment, the one or more polypeptides with HMG aldolase activity may be useful in<br>
facilitating a reaction in which indole-3-pyruvate is converted to MP as one step within a<br>
multi-step pathway to produce a product chosen from monatin, monatin derivatives, salts<br>
thereof and combinations thereof.<br>
[0574]	In yet another embodiment of the invention, one or more polypeptides with KHG<br>
aldolase activity encoded by a nucleic acid sequence having at least about 50%, 51%, 52%, 53%, 54%, 55%, 56%, 57%, 58%, 59%, 60%, 61%, 62%, 63%, 64%, 65%, 66%, 67%, 68%, 69%, 70%, 71%, 72%, 73%, 74%, 75%, 76%, 77%o, 78%, 79%, S0%, 8]%, 82%, 83%, 84%, 85%, 86%, 87%, 88%, 89%, 90%, 91%, 92%, 93%, 94%, 95%., 96%, 97%i, 98%, 99%, or more, or com]ilete (100%)) sequence identity to a nucleic acid in accordance with the invention, including SEQ ID NO:307, SEQ ID NO:309, SEQ ID N0:311, SEQ ID NO:313, SEQ ID NO:315, SEQ ID NO:317, SEQ ID KO:319, SEQ ID NO:321, SEQ ID NO:323, SEQ ID NO:325, SEQ ID NO:327, SEQ ID NO:329, SEQ ID NO:331, SEQ ID NO:333, SEQ ID NO:335, SEQ ID NO:336, SEQ ID NO:337, and SEQ ID NO:338 over a region of at least about 10, 25, 20, 25, 30, 35, 40, 45, 50, 75, 100, 150, 200, 250, 300, 350, 400, 450, 500, 550, 600, 650, 700, 750, 800, 850, 900, 950, 1000, 1050, 1100, 1150, 1200, 1250, 1300, 1350, 1400, 1450, 1500, 1550, 1600, 1650, 1700, 1750, 1800, 1850, 1900, 1950, 2000, 2050, 2100, 2200, 2250, 2300, 2350, 2400, 2450,<br>
 <br>
2500, or more residues may be useful in facilitating a reaction between indole-3-pyruvate<br>
and a C3 carbon source as one step within a multi-step pathway to produce a product<br>
chosen from monatin, monatin derivatives, salts thereof and combinations thereof In one<br>
embodiment, the one or more polypeptides with KHG aldolase activity may be useful in<br>
facilitating a reaction in which indoIe-3-pyruvate is converted to MP as one step within a<br>
multi-step pathway to produce a product chosen from monatin, monatin derivatives, salts<br>
thereof and combinations thereof<br>
[0575]	Furthermore, one or more polypeptides with aldolase activity encoded by a<br>
nucleic acid sequence that hybridizes under stringent condition to a nucleic acid of SEQ ID N0:1, SEQ ID N0:3, SEQ ID NO:5, SEQ ID N0:7, SEQ ID N0:9, SEQ ID N0:11, SEQ ID N0.13, SEQ ID N0:15, SEQ ID N0:17, SEQ ID N0:19, SEQ ID N0:2I, SEQ ID NO:23, SEQ ID NO:25, SEQ ID NO:27, SEQ ID NO:29, SEQ ID N0:31, SEQ ID NO:33, SEQ ID NO:35, SEQ ID NO:37, SEQ ID NO:39, SEQ ID N0:41, SEQ ID NO:43, SEQ ID NO:45, SEQ ID NO:47, SEQ ID NO:49, SEQ ID NO:51, SEQ ID NO:53, SEQ ID NO:55, SEQ ID NO:57, SEQ ID NO:59, SEQ ID N0:61, SEQ ID NO:63, SEQ ID NO:65, SEQ ID NO:67, SEQ ID NO:69, SEQ ID N0:71, SEQ ID NO:73, SEQ ID NO:75, SEQ ID NO:77, SEQ ID NO:79, SEQ ID N0:81, SEQ ID NO:83, SEQ ID NO:85, SEQ ID NO:87, SEQ ID N0:S9, SEQ ID N0:9i, SEQ ID NO:93, SEQ ID NO:95, SEQ ID NO:97, SEQ ID NO:99, SEQ ID NO:101, SEQ ID NO:103, SEQ ID NO:I05, SEQ ID NO:I07, SEQ ID NO:109, SEQ ID N0:111, SEQ ID N0:113, SEQ ID N0;115, SEQ ID NO:l 17, SEQ ID N0:119, SEQ ID N0:121, SEQ ID NO-.123, SEQ ID NO:125, SEQ ID NO:127, SEQ ID NO:129, SEQ ID N0:131, SEQ ID NO:I33, SEQ ID NO:]35, SEQ ID NO:137, SEQ ID NO:139, SEQ ID N0:]41, SEQ ID NO:143, SEQ ID NO:i45, SEQ ID NO:147, SEQ ID NO:149, SEQ ID N0:15U SEQ ID NO:153, SEQ ID NO:155, SEQ ID NO:157, SEQ ID NO:159, SEQ ID N0:161, SEQ ID NO:163, SEQ ID NO:165, SEQ ID NO:167, SEQ ID NO:169, SEQ ID N0:171, SEQ ID NO:173, SEQ ID NO:175, SEQ ID NO:177, SEQ ID NO:179, SEQ ID N0:I81, SEQ ID NO:]83, SEQ ID NO.i85, SEQ ID NO;]87, SEQ ID NO:189, SEQ ID N0:191, SEQ ID NO:193, SEQ ID NO:195, SEQ ID NO:197, SEQ ID NO:199, SEQ ID NO:201, SEQ ID NO:203, SEQ ID NO:205, SEQ ID NO:207, SEQ ID NO:209, SEQ ID N0:2n, SEQ ID NO:213, SEQ ID NO:215, SEQ ID NO:217, SEQ ID NO:219, SEQ ID NO:221, SEQ ID NO:223, SEQ ID NO:225, SEQ ID NO:227, SEQ ID NO:229, SEQ ID NO:231, SEQ ID<br>
 <br>
NO:233, SEQ ID NO:235, SEQ ID NO:237, SEQ ID NO:239, SEQ ID NO:241, SEQ ID<br>
NO:243, SEQ ID NO:245, SEQ ID NO:247, SEQ ID NO:249, SEQ ID NO:251, SEQ ID<br>
NO:253, SEQ ID NO:255, SEQ ID NO:257, SEQ ID NO:259, SEQ ID NO:261, SEQ ID<br>
NO:263, SEQ ID NO:265, SEQ ID NO:267, SEQ ID NO:269, SEQ ID NO:271, SEQ ID<br>
NO:273, SEQ ID NO:27S, SEQ ID NO:277, SEQ ID NO:279, SEQ ID NO:281, SEQ ID<br>
NO:283, SEQ ID NO:285, SEQ ID NO:287, SEQ ID NO;289, SEQ ID NO:29U SEQ ID<br>
NO:293, SEQ ID NO:295, SEQ ID NO:297, SEQ ID NO:299, SEQ ID NO:301, SEQ ID<br>
NO:303, SEQ ID NO:305, SEQ ID NO:307, SEQ ID -NO:309, SEQ ID N0:3n, SEQ ID<br>
NO:313, SEQ IDNO:315, SEQ IDNO:317, SEQ IDNO:319, SEQ IDNO:321, SEQ ID<br>
NO:323, SEQ ID NO:325, SEQ ID NO:327, SEQ ID NO:329, SEQ ID NO:33!, SEQ ID<br>
NO:333, SEQ ID NO:335, SEQ ID NO:336, SEQ ID NO:337, and SEQ ID NO:338 may<br>
be useful in facilitating a reaction between indole-3-pymvate and a C3 carbon source as<br>
one step within a multi-step pathway to produce a product chosen from monatin, monatin<br>
derivatives, salts thereof and combinations thereof. In one embodiment, the one or more<br>
polypeptides with aldolase activity may be useful in facilitating a reaction in which<br>
indole-3-pyruvate is converted to MP as one step within a multi-step pathway to produce<br>
a product chosen from monatin, monatin derivatives, salts thereof and combinations<br>
thereof.<br>
[0576]	In another embodiment of the invention, one or more polypeptides with HMG<br>
aldolase activity encoded by a nucleic acid sequence that hybridizes und stringent condition to a nucleic acid of SEQ ID N0:1, SEQ ID N0:3, SEQ ID N0:5, SEQ ID N0:7, SEQ ID N0;9, SEQ ID N0:11, SEQ ID N0:13, SEQ ID N0:15, SEQ ID N0:17, SEQ ID N0:19, SEQ ID N0:2I, SEQ ID NO:23, SEQ ID NO:25, SEQ ID NO:27, SEQ ID NO:29, SEQ ID N0:31, SEQ ID NO:33, SEQ ID NO:35, SEQ ID NO:37, SEQ ID NO:39, SEQ ID N0:41, SEQ ID NO:43, SEQ ID NO:45, SEQ ID NO-.47, SEQ ID NO:49, SEQ ID N0:51, SEQ ID NO:53, SEQ ID NO:55, SEQ ID NO:57, SEQ ID NO:59, SEQ ID N0:6I, SEQ ID NO:63, SEQ ID NO:65, SEQ ID NO:67, SEQ ID NO:69, SEQ ID N0:71, SEQ ID NO:73, SEQ ID NO:75, SEQ ID NO:77, SEQ ID NO:79, SEQ ID N0;81, SEQ ID NO:83, SEQ ID NO:85, SEQ ID NO:87, SEQ ID NO:89, SEQ ID N0:91, SEQ ID NO:93, SEQ ID NO:95, SEQ ID NO:97, SEQ ID NO:99, SEQ ID NO:I01, SEQ ID NO:103, SEQ ID NO:105, SEQ ID NO;I07, SEQ ID NO:109, SEQ ID N0:111, SEQ IDN0:113, SEQ ID N0:115, SEQ ID N0:117, SEQ ID<br>
 <br>
N0:119, SEQ ID N0:121, SEQ ID NO:!23, SEQ ID NO:I25, SEQ ID NO:127, SEQ ID<br>
NO:I29, SEQ ID N0:I3I, SEQ ID NO:I33, SEQ ID NO:135, SEQ ID NO:137, SEQ ID<br>
NO:139, SEQ IDN0:I41, SEQ ID NO:143, SEQ IDNO:145, SEQ ID NO:147, SEQ ID<br>
NO:149, SEQ ID N0:151, SEQ ID NO-.153, SEQ ID NO:I55, SEQ ID NO:157, SEQ ID<br>
NO:159, SEQ ID N0:16I, SEQ ID MO:I63, SEQ ID NO:165, SEQ ID NO:167, SEQ ID<br>
NO:169, SEQ IDN0:171, SEQ IDNO:I73, SEQ IDNO:175, SEQ IDNO:I77, SEQ ID<br>
NO:I79, SEQ ID N0:18I, SEQ ID NO:183, SEQ ID NO:I85, SEQ ID NO:187, SEQ ID<br>
NO:189, SEQ ID N0:I91, SEQ ID NO:193, SEQ ID NO:195, SEQ ID NO:I97, SEQ ID<br>
NO:199, SEQ ID NO:201, SEQ ID NO:203, SEQ ID NO:205, SEQ ID NO:207. SEQ ID<br>
NO:209, SEQ ID N0:211, SEQ ID NO:2I3, SEQ ID NO:2I5, SEQ ID NO:217, SEQ ID<br>
NO:219, SEQ ID NO:221, SEQ ID NO:223, SEQ ID NO:225, SEQ ID NO:227, SEQ ID<br>
NO:229, SEQ ID NO:23I, SEQ ID NO:233, SEQ ID NO:235, SEQ ID NO:237, SEQ ID<br>
NO:239, SEQ ID NO:24I, SEQ ID NO:243, SEQ ID NO:245, SEQ ID NO:247, SEQ ID<br>
NO:249, SEQ ID NO:25l, SEQ ID NO:253, SEQ ID NO:255, SEQ ID NO:257, SEQ ID<br>
NO:259, SEQ ID NO:261, SEQ ID NO:263, SEQ ID NO:265, SEQ ID NO:267, SEQ ID<br>
NO:269, SEQ ID NO:27I, SEQ ID NO:273, SEQ ID NO:275, SEQ ID NO:277, SEQ ID<br>
NO:279, SEQ ID NO:281, SEQ ID NO:283, SEQ ID NO:285, SEQ ID NO:287. SEQ ID<br>
NO:289, SEQ ID NO:29I, SEQ ID NO:293, SEQ ID NO;295, SEQ ID NO:297, SEQ ID<br>
NO:299, SEQ ID NO:30I, SEQ ID NO:303, SEQ ID NO:305 may be usefiil in<br>
facilitating a reaction between indoIe-3-pyruvate and a C3 carbon source as one step<br>
within a multi-step pathway to produce a product chosen from monatin, monatin<br>
derivatives, salts thereof and combinations thereof. In one embodiment, the one or more<br>
polypeptides with HMG aldolase activity may be useftil in facilitating a reaction in which<br>
indole-3-pyruvate is converted to MP as one step -vidthin a multi-step pathway to produce<br>
a product chosen from monatin, monatin derivatives, salts thereof and combinations<br>
thereof.<br>
(0577]	In yet another embodiment of the invention, one or more polypeptides with KHG<br>
aldolase activity encoded by a nucleic acid sequence that hybridizes under stringent condition to a nucleic acid of SEQ ID NO:307, SEQ ID NO:309, SEQ ID N0:311, SEQ ID NO:323, SEQ ID NO:3I5, SEQ ID NO:3}7, SEQ ID NO:3I9, SEQ ID NO;32], SEQ ID NO:323, SEQ ID NO:325, SEQ ID NO:327, SEQ ID NO:329, SEQ ID NO:331, SEQ ID NO:333, SEQ ID NO:335, SEQ ID NO:336, SEQ ID NO:337, and SEQ ID NO:338<br>
 <br>
may be useful in facilitating a reacticm between indoIe-3-pyruvate and a C3 carbon source as one step within a multi-step pathway to produce a product chosen from monatin, monatin derivatives, sahs thereof and combinations thereof. In one embodiment, the one or more polypeptides with KHG aldolase activity may be useiul in facilitating a reaction in which indoie-3-pyruvate is converted to MP as one step within a multi-step pathway to produce a product chosen fi:om monatin, monatin derivatives, salts thereof and combinations thereof.<br>
[0578]	The polypeptides with aldolase tivity described herein may be useful in<br>
facilitating a reaction between indole-3-pyruvate and a C3 carbon source. The C3 carbon source may be, but is not lunited to, oxaloacetate, pyruvate or a pyruvate derivative, such as phosphoenolpyruvate. In one embotiimeut, the C3 carbon source is pyruvate.<br>
[0579]	Exemplary enzymes useful for the conversion of the reaction product between<br>
indole-3-pyruvate and the C3 carbon source to monatin include members of the enzyme classes: tryptophan aminotransferases (2.6.1.27), tryptophan dehydrogenases (1.4.1.19), D-amino acid dehydrogenases (1.4.99.1), glutamate dehydrogenases (1.4.1.2-4), phenylalanine dehydrogeTiase (EC 1.4.1.20), trj'ptophan-phenylpyruvate transaminases (2.6.1.28), or more generally members of the aminotransferase family (2.6.1.-) such as aspartate aminotransferase (EC 2.6.1.1), tyrosine (aromatic) aminotransferase (2.6.1.5), D-tryptophan aminotransferase, or D-alanine (2.6.1.21) aminotransferase {see Figure 2 of WO 03/091396 A2). This reaction can also be performed using chemical reactions. Amination of the keto acid (MP) is performed by reductive amination using ammonia and sodium cyanoborohydride. Figures 11-13 of WO 03/091396 A2 show additional polypeptides that can be used to convert MP to monatin, as well as providing increased yields of monatin from indole-3-pyruvate or tryptophan. In one embodiment, these enzymes are utilized to catalyze the conversion of MP, the reaction product between indole-3-pyruvate and pyruvate, to monatin.<br>
[0580]	The taste profile of a monatin composition can be altered by controlling the<br>
relative amount of the various stereoisomers of monatin in the composition. The present disclosure provides pathways and substances for producing monatin compositions with a desired percentage of R,R monatin and/or S,R monatin.<br>
[0581]	The chirality of the monatm compounds produced by the pathways disclosed can<br>
be affected both by pH and by the polypeptides used for the biological conversions. The<br>
 <br>
polypqitides with aldolase activity described herein, may be utilized to control the chirality of the monatin carbon-2 (see Fonnula I, above) in the reaction in which indole-3-pynivate is converted to MP.<br>
[0582]	Once the reaction product of the reaction between indole-3-pyruvate and the C3<br>
carbon source is produced, the amino group can be added stereospecifically. Either the R or S configuration of carbon-4 (see Fonnula I above) can be generated depending on whether a D- or L- aromatic acid aminotransferase is used. Many aminotransferases are specific for the L-isomer, however, D-nyptophan aminotransferases exist in certain plants (Kohiba and Mito, Proceedings of the 8th International Symposium on Vitamin Be and Carbonyl Catalysis, Osaka, Japan 1990). Moreover, D-alanine aminotransferases (2.6.1.21), D-methionine-pyruvate aminotransferases (2.6.1.41) and both (R)-3-amino-2-methylpropanoate aminotransferase (2.6.1.61), (S)-3-amino-2-methylpropanoate aminotransferase (2.6.1.22), and D-phenylglycine aminotransferase have been identified. Certain aminotransferases may only accept the substrate for this reaction with a particular configurafion at the C2 carbon. Therefore, even if the conversion to the reaction product between indole-3-pyruvate and the C3 carbon source is not stereospecific, the stereochemistry of the final product can be controlled through the appropriate selection of an aminotransferase. Because the reactions are reversible, the um-eacted reaction product (undesired isomer) can be recycled back to its constituents and a racemic mixture of the reaction product can be reformed.<br>
[0583]	An example of a suitable amino donor for the addition of an amino group to the<br>
reaction product of the reaction between the mdole-3-pyruvate and the C3 carbon source includes, but is not limited to an amino acid, such as alanine, aspartate, lysine, glutamate, glycine, and tryptophan.<br>
[0584]	Referring now to the figures, the following should be noted.   The flow charts<br>
identify pathways for producing monatin, but are not limited to any particular method for practicing the pathways. For example, the pathways may be practiced in vivo, in vitro, or a combination thereof.<br>
[0585]	Furthermore, practice of the pathways does not require that each of the identified<br>
components (such as reactants and enzymes) is explicitly provided by the practitioner, so long as sufficient components, or sources of components, and reaction conditions are provided so that the pathway can potentially proceed.   In other words, for example, if a<br>
 <br>
figure depicts a process for producing a monatin composition, whicli includes producing indo!e-3-pyruvate from L-nyptophsn, producing 2-hydroxy 2-(indol-3ylmethyl)-4-keto glutaric acid ("monatin precursor" or "MP") from indole-3-pyruvate, and producing monatin from MP, wlierein each reaction is facilitated by an appropriate en2yme, it is contemplated that practice of that pathway includes combining L-tryptophan with a-ketoglutarate and enzymes contemplated for facilitating the identified reactions, and under conditions suitable for each of the reactions to occur without also explicitly providing indole-3-pyruvate or MP. ki such an instance L-tryptophan could react with a-ketoglutarate to produce indole-3-p;Titvate. Due to the set conditions and the provided enzyme, the indole-3-pyruvate produced from the L-tryptophan reaction could react to form MP, and then due to the set conditions and the provided enzyme, the MP produced from the indole-3-pyruvate reaction could react to form monatin.<br>
[0586]	It should also be noted that practice of the depicted pathways does not require the<br>
practitioner to explicitly provide the identified startmg materials or enzymes. In other words, it is contemplated that practice of any pathways which identifies L-tryptophan as a starting material would include providing a compound that can produce L-tryptophan, under conditions suitable for L-trypiophan production to occur and combining that compoimd with enzymes capable of facilitating the series of reactions set forth under conditions which would be suitable for those reactions to occur. As another example, it is also contemplated that practicing the identified pathway would include providing a microorganism genetically engineered to produce monatin according to the described pathway, and providing appropriate conditions for the fermentation process to occur. For example, a microorganism, which naturally produces large amounts of L-tryptophan could be genetically engineered to produce or over-produce one or more of the enzymes used to facilitate reactions in the pathway to monatin, and appropriate conditions could be provided so that the microorganism would thereby produce monatin.<br>
[0587]	Figure 1  identifies the particular embodiment wherein an R-speciiic aldolase<br>
facilitates the reaction of indole-3-pyruvate and pyruvate to form R-MP. The flow chart of Figure 1 schematically depicts a process in accordance with the invention for making a monatin composition including R,R monatin. As shown in Figure 1, the overall pathway involves a reaction of tryptophan to form indole-3-pyruvate, a reaction of indole-3-<br>
 <br>
pyruvate to produce MP, and a reaction of MP to produce monatin, including R,R monatin.<br>
[0588J	Figure 1 further illustrates specific permutations of this overall pathway, designed<br>
to increase the production of the R,R form of monatin at the expense of the S,S, R,S and S,R forms of monatin. In particular, Figure 1 illustrates the embodiment wherein: the aminotransferase enzyme utilized in the L-tryptophan reaction has greater activity and/or specificity for that reaction versus the reactions of MP and 4S monatin or the oxidase has greater activity and/or specificity for L-tryptophan than for 4R monatin; the enzyme which facilitates the reaction of indoIe-3-pyruvate is a polypeptide with aldolase activity disclosed herein, and, the enzyme which facilitates the reaction of MP is a broad specificity D-enzyme, preferably evolved to work more efficiently with the R isomer of MP.<br>
[0589]	Figure 1 also illustrates particular permutations designed to make the production<br>
of R,R monatin more economical. For example, in Figure I, L-tryptophan—as opposed to D-tryptophan or combinations of I and D-tryptophan—is identified as the starting material. While the choice of the specific form of tryptophan does not impact the chiralily of the ultimate monatin compounds in die monatin composition (because the tryptophan reaction foims indole-3-pyruvate, which has no chirality), some may prefer utilizing L-tryptophan as a startmg material at least because L-tryptophan is currently less expensive and more easily obtainable than D-tryptophan<br>
[0590]	Focusing now on the first reaction shown in Figure  1, when tryptophan is<br>
converted to indoIe-3-pjTuvate any one or more of alpha-ketoglutarate, oxaloacetate, and pyruvate reacts to form an amino acid (glutamate, aspartate, and alanine respectively). Figure 1 depicts the embodiment wherein the tryptophan starting material is L-tryptophan, and the alpha-ketoglutarate, oxaloacetate, and/or pyruvate produce the L-isomer form. of the amino acid (such as L-glutamate, L-aspartate, and/or L-alanine, respectively).<br>
[0591]	As shown in Figure 1, an approach to enhancing the production of R,R monatin<br>
involves facilitating the reaction of L-tryptophan with an enzyme having greater specificity, greater activity, or both for tryptophan as opposed to MP or monatin, and facilitating the reaction of MP with a D-enzyme. As is disclosed in WO 03/091396 A2, certain enzymes can facilitate the reaction of tryptophan to produce indoie-3-pyruvate, as<br>
 <br>
well as the amination reaction of MP to produce monatin. Use of an L-aminotransferase in the amination step creates an S chiral center at the monatin C-4 position, whereas use of a D-enzyme creates a D chiral center at the monatin C-4 position. Thus, in the instance where an L-aminotransferase, which facilitates the tryptophan reaction, is also active in the MP reaction, R.S and S,S monatin can be formed, depending on the form of MP present. In addition, certain other enzymes—the L-amino acid oxidases—can not only facilitate the reaction of tryptophan to indole-3-pyruvate, but may have a side activity for the degradation of R,R monatin. According to some embodiments- this 4R side activity is minimized or eliminated. An oxidase side activity on 4S forms of monatin would decrease or minimize them from the jlnal product and could be desirable depending on the final composition desired. Consequently, the greater the specificity and/or activity of the L-enzyme chosen for tryptophan versus the MP or monatin, the greater the amount of R,R and S,R produced versus S,S and R,S monatin.<br>
[0592]	Suitable enzymes for the tryptophan reaction, in accordance with the embodiment<br>
illustrated in Figure 1, include: L-aminotransferases capable of facilitating a reaction of L-tryptophan to form indole-3-pyruvate, and which have greater specificity for that reaction over the reaction of R-MP to form 4S isomers of monatin; and, L-amino acid oxidases capable of facilitating a reaction of L-tryptophan to form indole-3-pyruvate, and which have greater specificity and/or activity for that reaction versus the reaction of 4R isomers of monatin to form MP, and functional equivalents of any of the foregoing.. More specifically, non-limiting examples of suitable enzymes can be chosen from L-tryptophan aminotransferases (E.C. 2.6.1.27) and tyrosine (aromatic) aminofransferases (EC 2.6.1.5) and L-amino acid oxidases (EC 1.4.3.2), and mutants derived from enzymes having aspartate aminotransferase activity.<br>
[0593]	Example 16 identifies a specific enzyme, a mutant HEXaspC polypeptide which<br>
includes a Pro 9 to Tyr substitution and an Arg 122 to Gly substitution useful for facilitating the reactions of L-tryptophan and a-KG, oxaloacetate, pyruvate, or combinations thereof to form indole-3-pyruvate and L-glutamate, L-aspartate, and L-alanine, respectively. Another specific enzyme havuig "limited" activity is TatA, the L-tryptophan aminotransferase from S. meliloti. Other enzymes suitable for the tryptophan reaction in accordance with preferred embodiments of the pathway shown in Figure 1 include those with the following characteristics: an enzyme that transaminates MP at 1/10<br>
 <br>
the rate or less than the rate of L-tryplophan as in Example 16 or an enzyme when used with a racemase, as in Example 18, that produces greater than 90% of the 4R isomers of monatin.<br>
[0594]	Examples of enzymes not having greater specificity for the L-tryptophan to<br>
indole-3-p&gt;Tuvate conversion compai'ed to the MP to monatin conversion include: HEXAspC (Example 16), Leishmania major broad specificity aminotransferase (WO 03/091396 A2), the Porcine aminotransferase (WO 03/091396 A2) and Rhodobacter sphaeroides TatA (Example 18). These enzymes may, however, be evolved, for example through mutagenesis to have limited activity for R-MP and/or R,R monatin versus tryptophan.<br>
(0595J	Focusing now on the second reaction identified in Figure 1, the choice of enzyme<br>
for facilitating the reaction of indole-3-pyruvate to MP influences the relative amount of R,R monatin versus S,R monatin produced. In general, the greater the relative amount of R-MP versus S-MP produced, the greater the relative amount of R,R monatin versus S,R monatin produced (when a D-enzyme facilitates the reaction of MP to monatin). Where a monatin composition having the R,R form of monatin as its only monatin component is desired, an enzyme that selectively produces R-MP as opposed to S-MP (an "R-specific enzyme") should be used. The polypeptides with aldolase activity described herein are useful in selectively producing R-MP, as opposed to S-MP. Several examples of highly R-specific aldolase enzymes are demonstrated in Table 1, above. Examples 4, 5 and 6, below, and in the Sequence Listing.<br>
[0596]	Focusing now on the last step of the pathway identified in Figure I, the reaction of<br>
R-MP to form R,R monatin is shown to be facilitated by a broad specificity D-aminotransferase, for example D-alanine ammotrmisfemse: (E.C. 2.6.1.21, also known as D-amino acid aminotransferase or D-aspartate aminotransferase) or a D-amino acid dehydrogenase. As discussed above, the convereion of MP to monatin is an amination reaction, which creates a chiral center at the monatin C-4 carbon. Where the R-chiral form is desired at the C-4 position, enzymes should be used which produce "R" chiral centers in amino acids.<br>
[0597]	According to some embodhnents, the D-aminotransferase has greater specificity,<br>
greater activity, or both for the R-MP than for indole-3-pyruvate. According to some embodiments, the D-aminotramferase has limited activity for the indole-3-pyruvate.<br>
 <br>
Enzymes with such characteristics may be evolved or mutated from existing ensymes, for example as shown in Example 3 6.<br>
[0598]	Examples 9 to 12 illustrate the production of R,R-monatin from D-tryptophan.<br>
[0599]	Figure 1 illustrates a method of producing R,R monatin and S,R monatin.<br>
Whereas in the embodiment of Figure 1, the aldolase used in the reaction of indo!e-3-pyruvate to form R-VIP influences the ratio of R,R:S,R formed, in the embodiment of Figure 2, the D-enzyme that facilitates the conversion of MP to monatin influences the ratio of R,R:S,R formed. According to the pathway of Figure 2, if a non-stereospecific enzyme is used to facilitate the conversion of indole-3-pyruvate to MP, then both S-MP and R-MP can be formed. If a non-stereoselective aldolase is utilized to convert indole-3-pyruvate to MP, then a stereoselective transaminase is required to convert the MP to either R,R monatin or S,R monatin. As shown on Figure 2, use of a D-aminotransferase or D-amino acid dehydrogenase that is stereospecific for R-MP results in the production of RjR monatin.<br>
[0600]	Figure 3 illustrates another alternative pathway for targeting production of R,R<br>
monatin. The pathway of Figure 3 is a modification of the pathway of Figure 1, wherein indole-3-pyruvate is produced indirectly, rather than directly, from L-tryptophan. More specifically, L-tryptophan is converted to D-tiyptophan, and D-tiyptophan is then converted to indole-3-pyruvate.<br>
[0601]	The conversion of L-tryptophan to D-tryptophan can be facilitated by a tryptophan<br>
racemase or functional equivalent thereof. Example 15 provides potential sources of tryptophan racemases and screening methods for identifying such enzymes. It is also contemplated a tiyptophan racemase may be evolved (such as via mutagenesis or recombinant engineering) for improved performance from an existing amino acid racemase.<br>
[0602]	Non-limiting examples of tiyptophan racemases include homolog or mutants of<br>
amino acid racemases (EC 5.1.1.-), for example serine racemase, wherein the homoiogs or mutants are capable of converting L-tiyptophan to D-tryptophan. Non-limiting examples of sources from which the amino acid racemase may be derived include: microorganisms such as Salmonella typhimurium, Escherichia coli. Bacillus subtilis, Pseudomonas aeruginosa, Vibrio cholerae, Schizosaccaroyces pombe. Bacillus cereus, Enterococcus gallinarum, Pediococcus pentosaceus. Bacillus pumilus,  Lactobacillus<br>
 <br>
fermenti. Laclobacillus brevis, Aqup'ex pyrophilus, Lactobacilli, Streptococcus, Anabaena sp-, Pseudomonas striata, Lentinus edodes, Scapharca brouhtonii Desulfurococcus sp., Thermococcus sp., and Pseudomonas striata, AdditionaJ non-limiting examples of sources from which the amino acid racemase may be derived include silkworm, rat brain, or mouse brain.<br>
[0603]	Non-limiting examples of potential  sources from which suitable tryptophan<br>
racemases may be derived include: microorganisms such as Pseudomonas, for example Pseudomonas chlororaphis {Pseudomonas aurereofaciens) (ATCC15926), and Burkholderia pyrrocina (ATCC15958). Additional non-iimiting examples of potential sources from which suitable tryptophan racemases may be derived include plants, for example tobacco plants, such as Nicotiana tabacum, wheat plants, such as Triticum aestivum beets, tomatoes, and Sclerochiton ilicifoUus.<br>
[0604]	The pathway shown in Figure 3 has certain benefits, including that even where<br>
R,R monattn is the desired product, the same enzyme may be used for the reaction producing indole-3-pyruvate as for the reaction producing monatin. That is, in the pathway iiiustrated in Figure 1, an L-aminotransferase (or suitable L-enzyme) facilitates the reaction producmg indole-3-pyruvate, but a D-ammotransferase facilitates the reaction producing monatin. By contrast in the pathway of Figure 3, certain D-aminotransferase that facilitates the reaction producing indoie-3-pyruvate, can also facilitate the reaction producing monatin. Consequently, in pathways according to Figure 3 broad specificity D-aminotransferases may be preferred where there is a desire to use the same enzyme for the reaction forming indole-3-pjruvate as for the reaction forming monatin. By confrast, in pathways according to Figures 1, 2, 4, 6, 7, and 8 production of monatin may proceed forward more efficiently when a D-aminofransferase is chosen that has limited activity and/or specificity for indole-3-pynzvate as compared to R-MP.<br>
[0605]	Another benefit of the pathway schematically represented in Figure 3 is that the<br>
amino acid product of the retion coupled to the reaction producing indoie-3-pyruvate can now be used as a starting material in the reaction coupled to the reaction producing monatin. That is. in the pathvray illustrated in Figure 1, L-tryptophan reacts to produce indole-3-pyruvate and at the same time oxaloacetate, alpha-ketoglutarate and/or pyruvate react to produce an L-amino acid. Because the reaction of R-MP to form monatm is coupled with a reaction utilizhig a D-amino acid as a substrate, the L-amino acid of the<br>
 <br>
reaction forming indoie-3-pyruvate is not, under the conditions shown, recycled for use in the reaction coupled to the R-MP reaction. By contrast, in the pathway illustrated in Figure 3, the reaction of D-tryptophan to form mdoIe-3-pyruvate is coupled to a reaction forming a D-amino acid product, which D-amino acid can be recycled for use in the reaction coupled to the R-MP reactioiL This allows one to use non-stoichlometric amounts of amino acceptor in step one. In some embodiments of the invention, the D-amino acid is D-alanine.<br>
[0606]	Figures 4 and 5 illustrate additional modifications of the pathway shown in Figure<br>
1, which modifications are directed to recycling the amino acid product formed by the reaction coupled with the L-tryptophan reaction with the amino acid reactant of the reaction coupled to the MP to monatin reaction.<br>
[0607]	Turning to Figure 4, the recycling is accomplished providing an enzyme that can<br>
facilitate the conversion of an L-amino acid to a D-amino acid and vice versa. More specifically, where as is shown in Figure 4, a-KG reacts to form L-glutamate when L-tryptophan reacts to form indole-3-pyruvate, a glutamate racemase (EC 5.1.1.3) or functional equivalent can be provided that can facilitate the conversion of L-glutamate to D-glutamate and vice versa. In such an instance, the L-giutamate formed alongside the production of indole-3-pymvate is removed by virtue of its conversion to D-glutamate, and the D-glutamate formed from the conversion of L-glutamate is then available as a substrate for the reaction coupled with the MP to monatin reaction. Similarly, the a-KG formed in the reaction of D-glutamate is available as a substrate for the reaction coupled to the L-tryptophan to indole-3-pyruvate reaction.<br>
[0608]	Non-limited examples of potential sources from which a glutamate racemase may<br>
be derived include Pediococcus pentosaceus, Bacillus pumilus, Lactobacillus fermenti, Lactobacillus brevis, E. coli, Aquifex pyrophilus, and Bacillus subtHis. More specifically (also non-limiting), the glutamate racemase may be expressed from a nucleic acid such as pediococcus pentaosaceus murl gene (Genbank Accession No. L22789), or Lactobacillus brevis glutamate racemase.<br>
[0609]	Where oxaloacetate reacts to form L-aspartate when L-tryptophan reacts to form<br>
indole-3-pyruvate, an aspartate racemase (EC 5.1.1.13) or functional equivalent can be provided to convert L-aspartate to D-aspartate. In such an instance, the L-aspartate alongside the production of indole-3-pyruvate is removed by virtue of its conversion to<br>
 <br>
D-aspartate, and the D-aspartate fonned from the conversion of L-aspartate is then available to as a substrate for the risaction coupled to the MP to monatin reaction. Similarly, the oxaloacetate formed in the reaction of D-aspartate is available to act as a substrate for the reaction coupled to the L-tnptophan to indole-3-pyruvate reaction.<br>
[0610]	Non-iimiting examples of suitable enzymes having aspartate racemase activity<br>
include ASPR-101 (BioCatalytics, Inc., Pasadena, CA) and homologs or mutants of an amino acid racemase (EC 5.1.1.-) which are capable of facilitating the conversion of L-aspartate to D-aspartate.<br>
[0611]	Non-limiting examples of potential sources from which aspartate racemases may<br>
be derived include: Desulfurococcus, Thermococcus, bivalve moUusk Scapharca brouhtonii, Acinetobacter, Agrobacterium, Archaeoglobus, Bacillus, Bordetella, Bradyrhizobium, Brevibacterium, Burkholderia, Campylobacter, Candida, Caulobacter, Clostridium, Desulfitobacterium, Demlfotalea, Enterococcus, Erwinia, Escherichia, Ferroplasma, Helicobacter, Klebsiella, Lactobacillus, Mannheimta. Medicago, Mesorhizobium, Methanococcus, Methanosarcina, Oceanobacillus, Oenococcus, Pediococcus, Polaribacter, Pseudomonas, Pyrococcus, Ralsonia, Shigella, Sinorhizobium, Salmonella, Sphingomonas, Streptococcus, Thermoanaerobacter, Vibrio, WolineUa, Xanthomonas, Xanthobacter, Yersinia and Zymomonas.<br>
(0612]	Where pyruvate reacts to form L-alanine when L-tryptophan reacts to form<br>
indole-3-pyruvate, an alanine racemase or functional equivalent can be provided to convert L-alanine to D-alanine. In such an instance, the L-alanine formed alongside the production of indole-3-pymvate is removed by virtue of its conversion to D-alanine, and the D-alanine formed from the conversion of L-alanine is then available to act as a substrate for the reaction coupled to the MP to monatin reaction. Similarly, the pyruvate formed in the reaction of D-alanine is available to act as a substrate for the reaction couple with the L-tryptophan to indole~3-pyruvate reaction.<br>
[0613]	Non-limiting examples of suitable alanine racemases include A8936 (Sigma-<br>
Aldrich, St. Louis, MO).<br>
[0614]	Non-limiting examples of potential sources from which the alanine racemase may<br>
be derived include: Brucella abortus. Streptococcus faecalis Salmonella typhimurium, Escherichia    coli.    Bacillus    subtilis.    Bacillus   stear other mophilus,    Pseudomonas<br>
 <br>
aeruginosa Vibrio cholerae, Schizosaccaroyces pombe. Bacillus cereus and Lenlinus edodes.<br>
[0615]	Examples IS and 21 illustrate the use of the above racemases, their impact on<br>
increasing the ratio of the desired monatin product, and provide potential sources for the racemase enzymes.<br>
[0616]	Turning to Figure 5, a stereoinverting aminotransferase is used to facilitate the<br>
reaction of R-MP to monatin. Although typically the R-MP {or S-MP) reaction to form R,R monatin (or S,R monatin) is coupled with the reaction of a D-amino acid, a stereoinverting aminotransferase can facilitate the coupled reactions of R-MP (or S-MP) to form R,R monatin (or S,R monatin) using an L-amino acid. In this way, the L-amino acid product of the L-tryptophan aminotransferase reaction can be used as a substrate for the transamination of MP to monatin, and the product (i.e. oxaloacetate, pyruvate, and/or a-KG) of the reaction coupled to the MP to monatin reaction can be used as a starting material for the reaction coupled to the L-tryptophan to indole-3-pyruvate reaction. Non-limiting examples of stereoinverting aminotransferases that may be used include D-phenylglycine aminotransferase (EC 2.6.1.72, also known as D-4-hydroxyphenylglycine aminotransferase) and D-methionine aminotransferase (EC 2.6.1.41, also known as D-met-aminotransferase and D-methionine-pyruvate aminotransferase). Non-limiting examples of potential sources from which the D-phenylglycine aminotransferase may be derived include Pseudomonas, such as Pseudomonas putida LW-4 and Pseudomonas stutzeri ST-201. Non-limiting examples of potential sources from which the D-methionine aminotransferase may be derived include cauliflower and peanut.<br>
[0617]	Examples   19   and 20  together provide  potential  sources  of stereoinverting<br>
enzymes, and methods of making such enzymes. The examples also provide screening methods for identifying such enzymes. It is also contemplated that such enzymes may be evolved from stereoinverting enzymes known or found in nature. As a non-limiting example, the stereoinverting aminotransferase may be a homolog or mutant of a D-amino acid aminotransferase or a homolog or mutant of an amino acid racemase (EC 5.1.1 .-)■<br>
[0618]	Figures 6-8 also illustrate modifications to the pathway of Figure 1. The pathways<br>
illustrated in Figures 6-8 provide methods to push equilibriimi reactions forward by removing byproduct of the tryptophan reaction and in some cases providing substrate for the MP reaction.<br>
 <br>
[0619]	Turning to Figure 6, the pathway shown removes the L-amino acid product of the<br>
reaction coupled to the tryptophan reaction by converting it to a different L-amino acid, and then provides a substrate for reacrion coupled to the MP reaction by converting the newly formed L-amino acid to a D-arnino acid. Specifically, L-tryptophan is shown to react alongside oxaloacetate to form indo!e-3-pyruvate and L-aspartate. An aspartate 4-decarboxylase (EC 4.1.1.12) or functional equivalent is used to facilitate the conversion of L-aspartate to L-alanine and carbon dioxide, and an enzyme with alanine racemase activity is used to facilitate the conversion of L-alanine to D-a!anine, which D-alanine can serve as an amino donor for the conversion of R-MP to monatin.<br>
[0620]	Turning to Figure 7, the pathway shown illustrates additional methods for<br>
removing the L-ammo acid product of the reaction coupled to the tryptophan reaction. Embodiments as presented in the figire produce a byproduct(s) that is unavailable to react in the reverse direction, for example due to volatility (such as carbon dioxide) or by spontaneous convereion to an unreactive endproduct. An example of such an approach includes where a-KG reacts alongside L-tryptophan to produce L-glutamate, a glutamate decarboxylase (EC 4.LI. 15) or functional equivalent can be provided which can facilitate the conversion of L-glutamate to 4-aminobutanoate (with carbon dioxide as a byproduct). Non-limiting examples of potential sources from which the L-glutamate decarboxylase may be derived include: Clostridium perjringens, C. welchii, or E. coli.<br>
[0621]	Another example of such an approach for moving the tryptophan reaction forward<br>
includes where oxaloacetate reacts alongside L-tryptophan, an aspartate decarboxylase (EC 4.1.1.11) or functional equivalent can be provided to facilitate the conversion of L-aspartate to p-aianine (with carbon dioxide as a byproduct).<br>
[0622]	Turning to Figure 8, the pathway shown illustrates yet additional methods for<br>
removing the L-amino acid product of the reaction coupled to the tryptophan reaction and providing a substrate for the reaction coupled to the MP reaction. Specifically, where a-KG reacts alongside L-tryptophan to form L-glutamate, an enzyme with L-alanine aminotransferase activity and pyiixvate can be provided, wherein the L-alanine aminotransferase enzyme facilitates the reaction of pyruvate and L-glutamate to form L-alanine. An alanine racemase or functional equivalent can also be provided in order to facilitate the conversion of die L-alanine to D-alanine, which D-alanine can be used as a substrate along with MP to form monatin and pyruvate. See Examples 18 and 21.<br>
 <br>
Biosynthetic Pathways to Produce R,R and Other Stereoisomers ofMonatin Derivatives<br>
[0623]	The methods of the described invention include using the polypeptides with<br>
aldole activity described herein may be used to facilitate the reaction between a substituted indole-3-pyruvate and a C3 carbon source.<br>
[0624]	Enzymes useful for the facilitating a reaction between a substituted indole-3-<br>
pyruvate and a C3 carbon source include one or more polypeptides with aldolase activity of any of SEQ ID N0:2, SEQ ID N0:4, SEQ ID N0:6, SEQ ID N0;8, SEQ ID NO:10, SEQ ID N0:12, SEQ ID N0:14, SEQ ID N0:16, SEQ ID N0:18, SEQ ID NO:20, SEQ ID NO:22, SEQ ID NO:24, SEQ ID NO:26, SEQ ID NO:28, SEQ ID KO:30, SEQ ID NO:32, SEQ ID NO:34, SEQ ID NO:36, SEQ ID NO:38, SEQ ID NO:40, SEQ ID NO:42, SEQ ID NO:44, SEQ ID NO:46, SEQ ID NO:48, SEQ ID NO:50, SEQ ID NO:52, SEQ ID NO:54, SEQ ID NO:56, SEQ ID NO:58, SEQ ID HO:60, SEQ ID NO:62, SEQ ID NO:64, SEQ ID NO:66, SEQ ID NO:68, SEQ ID NO:70, SEQ ID NO:72, SEQ ID NO:74, SEQ ID NO:76, SEQ ID NO:78, SEQ ID HO:80, SEQ ID N0:S2, SEQ ID NO:84, SEQ ID NO:86, SEQ ID NO:88, SEQ ID NO:90, SEQ ID NO:92, SEQ ID NO:94, SEQ ID NO:96, SEQ ID NO:98, SEQ ID NOrlOO, SEQ ID NO:102, SEQ ID NO:104, SEQ ID NQ:106, SEQ ID NO:i08, SEQ ID N0;110, SEQ ID N0:112, SEQ ID N0:114, SEQ ID N0:116, SEQ ID N0:118, SEQ ID NO:120, SEQ ID NO:122, SEQ ID NO:124, SEQ ID NO:126, SEQ ID NO:128, SEQ ID NO:130, SEQ ID NO:132, SEQ ID NO:134, SEQ ID NO:136, SEQ ID NO:138, SEQ ID N0:14D, SEQ ID NO:142, SEQ ID NO:144, SEQ ID NO:146, SEQ ID NO:148, SEQ ID NO:150, SEQ ID NO:152, SEQ ID NO:154, SEQ ID NO:156, SEQ ID NO:i58, SEQ ID NO:160, SEQ ID NO:162, SEQ ID NO:164, SEQ ID NO:166, SEQ ID NO:168, SEQ ID NO:170, SEQ ID NO:172, SEQ ID NO:174, SEQ ID NO:176, SEQ ID NO:178, SEQ ID NO:180, SEQ ID NO:I82, SEQ ID NO:184, SEQ ID NO:186, SEQ ID NO:188, SEQ ID NO:190, SEQ ID NO:192, SEQ ID NO:194, SEQ ID NO:196, SEQ ID NO:198, SEQ ID NO:200, SEQ ID NO:202, SEQ ID NO:204, SEQ ID NO:206, SEQ ID NO:208, SEQ ID NO:210, SEQ ID NO:212, SEQ ID NO:214, SEQ ID NO:216, SEQ ID NO:218, SEQ ID NO:220, SEQ ID NO:222, SEQ ID NO:224, SEQ ID NO:226, SEQ ID NO:228, SEQ ID NO;230, SEQ ID NO:232, SEQ ID NO:234, SEQ ID NO:236, SEQ ID NO:238, SEQ ID NO;240, SEQ ID NO:242, SEQ ID NO:244, SEQ ID NO:246, SEQ ID NO:248, SEQ ID NO:250, SEQ ID NO:252, SEQ ID NO:254, SEQ ID NO:256, SEQ ID NO:258, SEQ ID NO:260, SEQ ID<br>
 <br>
NO:262, SEQ ID NO:264, SEQ ID NO:266, SEQ ID NO:268, SEQ ID NO;270, SEQ ID<br>
NO:272, SEQ ID NO:274, SEQ ID NO:276, SEQ ID NO:278, SEQ ID NO:280, SEQ ID<br>
NO:282, SEQ ID NO:284, SEQ ID NO:286, SEQ ID NO:288, SEQ ID NO:290, SEQ ID<br>
NO:292, SEQ ID NO:294, SEQ ID NO:296, SEQ ID NO:298, SEQ ID NO:300, SEQ ID<br>
NO:302, SEQ ID NO:304, SEQ ID NO:306, SEQ ID NO:308, SEQ ID NO;3I0, SEQ ID<br>
NO:312, SEQ ID NO:314, SEQ ID NO:3i6, SEQ ID NO:318, SEQ ID NO:320, SEQ ID<br>
NO:322, SEQ ID NO:324, SEQ ID NO:326, SEQ ID NO:328, SEQ ID NO:330, SEQ ID<br>
NO:332, or SEQ ID NO:334, or fragments or subsequences thereof having aldolase<br>
activity.<br>
[0625]	In one embodiment, one or more polypeptides with HMG aldolase activity of any<br>
of SEQ ID N0:2, SEQ ID N0:4, SEQ ID NO:6, SEQ ID N0:8, SEQ ID NO: 10, SEQ ID N0:12, SEQ ID N0:14, SEQ ID N0:16, SEQ ID N0:18, SEQ ID NO:20, SEQ ID NO:22, SEQ ID NO;24, SEQ ID NO:26, SEQ ID NO:28, SEQ ID NO:30, SEQ ID NO:32, SEQ ID NO:34, SEQ ID NO:36, SEQ ID NO:38, SEQ ID NO:40, SEQ ID NO:42, SEQ ID NO:44, SEQ ID NO:46, SEQ ID NO:48, SEQ ID NO:50, SEQ ID NO:52, SEQ ID NO:54, SEQ ID NO:56, SEQ ID NO:58, SEQ ID NO:60, SEQ ID NO:62. SEQ ID NO:64, SEQ ID NO:66, SEQ ID NO:68. SEQ ID NO:70, SEQ ID NO:72, SEQ ID NO:74, SEQ ID NO:76, SEQ ID NO:78, SEQ ID NO:80, SEQ ID NO:82, SEQ ID NO;84, SEQ ID NO:86, SEQ ID NO:88, SEQ ID NO;90, SEQ ID NO:92, SEQ ID NO:94, SEQ ID NO:96, SEQ ID NO:98, SEQ ID NOrlOO, SEQ ID NO:102, SEQ ID NO:104, SEQ ID NO:106, SEQ ID NO:108, SEQ ID NO:110, SEQ ID N0:112, SEQ ID N0:114, SEQ ID N0:116, SEQ ID N0:118, SEQ ID NO:120, SEQ ID NO:122, SEQ ID NO:124, SEQ ID NO:126, SEQ ID NO:128, SEQ ID NO:i30, SEQ ID NO:132, SEQ ID NO:134, SEQ ID NO:136, SEQ ID NO:138, SEQ ID NO:140, SEQ ID NO:142, SEQ ID NO:144, SEQ ID NO:146, SEQ ID NO:148, SEQ ID NO:150, SEQ ID NO:152, SEQ ID NO:154, SEQ ID NO:156, SEQ ID NO:158, SEQ ID NO:160, SEQ ID NO:162, SEQ ID NO:164, SEQ ID NO:166, SEQ ID NO:168, SEQ ID NO:170, SEQ ID NO:172, SEQ ID NO:174, SEQ ID NO:176, SEQ ID NO:178, SEQ ID NO:180, SEQ ID NO:182, SEQ ID NO:184, SEQ ID NO:186, SEQ ID NO:188, SEQ ID NO:190, SEQ ID NO:192, SEQ ID NO:194, SEQ ID NO:196, SEQ ID NO:!98, SEQ ID NO:200, SEQ ID NO:202, SEQ ID NO:204, SEQ ID NO:206, SEQ ID NO:208, SEQ ID NO:2I0, SEQ ID NO:212, SEQ ID NO:214, SEQ ID NO:216, SEQ ID NO:218, SEQ ID NO;220, SEQ ID<br>
 <br>
NO:222, SEQ ID NO:224, SEQ ID NO:226, SEQ ID NO;228, SEQ ID NO:230, SEQ ID NO:232, SEQ ID NO:234, SEQ ID NO:236, SEQ ID NO:238, SEQ ID NO:240, SEQ ID NO:242, SEQ ID NO:244, SEQ ID NO:246, SEQ ID NO:248, SEQ ID NO:250, SEQ ID NO:252, SEQ ID NO:254, SEQ ID NO:256, SEQ ID NO:258, SEQ ID NO:260, SEQ ID NO:262, SEQ ID NO:264, SEQ ID NO:266, SEQ ID NO:268, SEQ ID NO:270, SEQ ID NO:272, SEQ ID NO:274, SEQ ID NO:276, SEQ ID NO:278, SEQ ID NO:280, SEQ ID NO:282, SEQ ID NO:284, SEQ ID NO:286, SEQ ID NO:288, SEQ ID NO:290, SEQ ID NO:292, SEQ ID NO:294, SEQ ID NO:296, SEQ ID NO:298, SEQ ID NO:300, SEQ ID NO:302, SEQ ID NO:304 or fragments or subsequences thereof having aldolase activity may be useful in facilitating a reaction between a substituted indoIe-3-pyruvate and a C3 carbon source.<br>
[0626]	In another embodiment, one or more polypeptides v«th KHG aldolase activity of<br>
any of SEQ ID NO:306, SEQ ID NO:308, SEQ ID NO:310, SEQ ID NO:312, SEQ ID NO:314, SEQ ID NO:316, SEQ ID NO:318, SEQ ID NO;320, SEQ ID NO:322, SEQ ID NO:324, SEQ ID NO:326, SEQ ID NO:328, SEQ ID NO:330, SEQ ID NO:332, or SEQ ID NO:334 or fragments or subsequences thereof having aldolase activity may be useiiil in facilitating a reaction between a substituted indoIe-3-pyruvate and a C3 carbon source.<br>
[0627]	Alternatively, one or more polypeptides with aldolase activity encoded by a<br>
nucleic id sequence having at least about 50%, 51%, 52%, 53%, 54%, 55%, 56%, 57%, 58%, 59%, 60%, 61%, 62%, 63%, 64%, 65%, 66%, 67%, 6S%, 69%, 70%, 71%, 72%, 73%, 74%, 75%, 76%, 77%, 78%, 79%, 80%, 81%, 82%, 83%, 84%, 85%, 86%, 87%, 88%, 89%, 90%, 91%, 92%, 93%, 94%, 95%, 96%, 97%, 98%, 99%, or more, or complete (100%) sequence identity to a nucleic acid in accordance with the invention, including SEQ ID N0:1, SEQ ID N0:3, SEQ ID N0:5, SEQ ID N0:7, SEQ ID N0:9, SEQ ID N0:11, SEQ ID N0:13, SEQ ID N0:15, SEQ ID N0:17, SEQ ID N0:19, SEQ ID N0:2l, SEQ ID NO:23, SEQ ID NO:25, SEQ ID NO:27, SEQ ID NO:29, SEQ ID N0:31, SEQ ID NO:33, SEQ ID ■NO:35, SEQ ID NO:37, SEQ ID NO:39, SEQ ID N0:41, SEQ ID NO:43, SEQ ID NO:45, SEQ ID NO:47, SEQ ID NO:49, SEQ ID N0:5i, SEQ ID NO:53, SEQ ID NO:55, SEQ ID NO:57, SEQ ID NO:59, SEQ ID N0:61, SEQ ID NO:63, SEQ ID NO:65, SEQ ID NO:67, SEQ ID NO:69. SEQ ID N0:71, SEQ ID NO:73, SEQ ID NO:75, SEQ ID NO:77, SEQ ID NO:79, SEQ ID N0:81, SEQ ID NO:83, SEQ ID NO:85, SEQ ID NO:87, SEQ ID NO:89, SEQ ID<br>
 <br>
N0:91, SEQ ID NO:93, SEQ ID NO:95, SEQ ID NO:97, SEQ ID NO:99, SEQ ID ■ NO:101, SEQ ID NO:103, SEQ ID NO:105, SEQ ID NO:107, SEQ ID NO;109, SEQ ID N0:111, SEQ ID N0:il3, SEQ ID N0:115, SEQ ID N0:117, SEQ ID N0:119, SEQ ID N0:121, SEQ ID NO:123, SEQ ID NO:125, SEQ ID NO:127, SEQ ID NO:129, SEQ ID N0:131, SEQ ID NO:133, SEQ ID NO:135, SEQ ID NO:137, SEQ ID NO:I39, SEQ ID N0:141, SEQ ID NO:143, SEQ ID NO:145, SEQ ID NO:147, SEQ ID NO:149, SEQ ID N0:151, SEQ ID NO:i53, SEQ ID NO:155, SEQ ID NO:157, SEQ ID NO:159, SEQ ID N0:161, SEQ ID NO:I63, SEQ ID NO:165, SEQ ID NO:167, SEQ ID NO:I69, SEQ ID N0:17L SEQ ID NO:173, SEQ ID NO:175, SEQ ID NO:I77, SEQ ID NO:179, SEQ ID N0:181, SEQ ID NO:183, SEQ ID NO:185, SEQ ID NO:187, SEQ ID NO:189, SEQ ID N0:19I, SEQ ID NO:193, SEQ ID NO:195, SEQ ID NO:197, SEQ ID NO:199, SEQ ID NO;20I, SEQ ID NO:203, SEQ ID NO:205, SEQ ID NO:207, SEQ ID NO:209, SEQ ID N0;211, SEQ ID NO:213, SEQ ID NO:215, SEQ ID NO:217, SEQ ID NO:219, SEQ ID NO:221, SEQ ID NO:223, SEQ ID NO;225, SEQ ID NO;227, SEQ ID NO:229, SEQ ID NO:231, SEQ ID NO:233, SEQ ID NO:235, SEQ ID NO:237, SEQ ID NO:239, SEQ ID NO:241, SEQ ID NO:243, SEQ ID NO:245, SEQ ID NO:247, SEQ ID NO:249, SEQ ID NO:251, SEQ ID NO:253, SEQ ID NO:255, SEQ ID NO:257, SEQ ID NO:259, SEQ ID NO:261, SEQ ID NO:263, SEQ ID NO:265, SEQ ID NO:267, SEQ ID NO:269, SEQ ID NO:271, SEQ ID NO:273, SEQ ID NO:275, SEQ ID NO:277, SEQ ID NO:279, SEQ ID NO:281, SEQ ID NO:283, SEQ ID NO:285, SEQ ID NO:287, SEQ ID NO:289, SEQ ID NO:291, SEQ ID NO:293, SEQ ID NO;295, SEQ ID NO:297, SEQ ID NO:299, SEQ ID NO:30U SEQ ID NO:303, SEQ ID NO:305, SEQ ID NO:307, SEQ ID NO:309, SEQ ID N0:3I1, SEQ ID NO:313, SEQ ID NO:315, SEQ ID NO:317, SEQ ID NO:319, SEQ ID NO:321, SEQ ID NO:323, SEQ ID NO:325, SEQ ID NO:327, SEQ ID NO:329, SEQ ID NO:331, SEQ ID NO:333, SEQ ID NO:335, SEQ ID NO:336, SEQ ID NO:337, and SEQ ID NO:338 over a region of at least about 10,15, 20, 25, 30, 35, 40, 45, 50, 75, 100,150, 200, 250, 300, 350, 400, 450, 500, 550, 600, 650, 700, 750, 800, 850, 900, 950, 1000, 1050, 1100, 1150, 1200, 1250, 1300, 1350, 1400, 1450, 1500, 1550, 1600, 1650, 1700, 1750, 1800, 1850, 1900, 1950, 2000, 2050, 2100, 2200, 2250, 2300, 2350, 2400, 2450, 2500, or more residues may be usefiil in facilitating a reaction between a substituted indole-3-pyruvate and a C3 carbon source.<br>
 <br>
[06281	In one embodiment of the invention, one or more polypeptides witli HMG<br>
aldolase activity encoded by a nucleic acid sequence having at least about 50%, 51%, 52%, 53%, 54%, 55%, 56%, 57%, 58%, 59%, 60%, 61%, 62%, 63%, 64%, 65%, 66%, 67%, 68%, 69%, 70%, 71%, 72%, 73%, 74%, 75%, 76%. 77%, 78%, 79%, 80%, 81%, 82%, 83%, 84%, 85%, 86%, 87%, 88%, 89%, 90%, 91%, 92%, 93%, 94%, 95%, 96%, 97%, 98%, 99%, or more, or complete (100%) sequence identity to a nucleic acid in accordance with the invention, including SEQ ID N0;1, SEQ ID N0:3, SEQ ID N0:5, SEQ ID N0:7, SEQ ID N0:9, SEQ ID N0:11, SEQ ID N0:13, SEQ ID N0:15, SEQ ID NO:17, SEQ ID N0:19, SEQ ID N0:21, SEQ ID NO:23, SEQ ID NO:25, SEQ ID NO:27, SEQ ID NO:29, SEQ ID N0:31, SEQ ID NO:33, SEQ ID NO:35, SEQ ID NO:37, SEQ ID NO:39, SEQ ID N0:4!, SEQ ID NO:43, SEQ ID NO:45, SEQ ID NO:47, SEQ ID NO:49, SEQ ID N0:51, SEQ ID NO:53, SEQ ID NO:55, SEQ ID NO:57, SEQ ID NO:59, SEQ ID N0:61, SEQ ID NO:63, SEQ ID NO:65, SEQ ID NO:67, SEQ ID, NO:69, SEQ ID N0:71, SEQ ID NO:73, SEQ ID NO:75, SEQ ID NO:77, SEQ ID NO:79, SEQ ID N0:81, SEQ ID NO:83, SEQ ID NO:85, SEQ ID NO:87, SEQ ID NO:89, SEQ ID N0:9L SEQ ID NO:93, SEQ ID NO:95, SEQ ID NO:97, SEQ ID NO:99, SEQ ID NO:101, SEQ ID NO:i03, SEQ ID NO:105, SEQ ID NO:107, SEQ ID NO:109, SEQ ID NOilIl, SEQ ID N0:113, SEQ ID N0:I15, SEQ ID N0:117, SEQ IDN0:119, SEQ IDN0:121, SEQ IDNO:123, SEQ IDNO:125, SEQ ID NO:127, SEQ ID NO:129, SEQ ID NO:131, SEQ ID NO:133, SEQ ID NO:135, SEQ ID NO:137, SEQ ID NO:139, SEQ ID N0:141, SEQ ID NO:143, SEQ ID NO:]45, SEQ ID NO:I47, SEQ IDNO:149, SEQ ID NO:151, SEQ IDNO:153, SEQ IDNO:155, SEQ ID NO:157, SEQ ID NO:159, SEQ ID N0:161, SEQ ID NO:163, SEQ ID NO:165, SEQ ID NO:167, SEQ ID NO:169, SEQ ID N0:171, SEQ ID NO:173, SEQ ID NO:175, SEQ ID NO:177, SEQ ID NO:179, SEQ ID N0:181, SEQ ID NO:183, SEQ ID NO:185, SEQ ID NO:187, SEQ ID NO:189, SEQ ID N0:191, SEQ ID NO:193, SEQ ID NO:195, SEQ ID NO:197, SEQ ID NO;199, SEQ ID NO:201, SEQ ID NO:203, SEQ ID NO:205, SEQ ID NO:207, SEQ ID NO:209, SEQ ID N0:211, SEQ ID NO:213, SEQ ID NO:215, SEQ ID NO:217, SEQ ID NO:219, SEQ ID NO:221, SEQ ID NO:223, SEQ ID NO:225, SEQ ID NO:227, SEQ ID NO;229, SEQ ID NO:231, SEQ ID NO:233, SEQ ID NO:235, SEQ ID NO:237, SEQ ID NO:239, SEQ ID NO:241, SEQ ID NO:243, SEQ ID NO:245, SEQ ID NO:247, SEQ ID NO:249, SEQ ID NO:25I, SEQ ID NO:253, SEQ ID NO:255, SEQ ID<br>
 <br>
NO:257, SEQ ID NO:259, SEQ ID NO:261, SEQ ID NO:263, SEQ ID NO:265, SEQ ID NO:267, SEQ ID NO:269, SEQ ID NO:27L SEQ ID NO:273, SEQ ID NO:275, SEQ ID NO:277, SEQ ID NO:279, SEQ ID NO:28L SEQ ID NO:283, SEQ ID NO:285, SEQ ID NO:287. SEQ ID NO:289, SEQ ID NO:29l, SEQ ID NO:293, SEQ ID NO:295. SEQ ID NO:297, SEQ ID NO:299, SEQ ID NO:301, SEQ ID NO:303, SEQ ID NO:305 over a region of at least about 10, 15, 20, 25, 30, 35, 40, 45, 50, 75, 100, 150, 200, 250, 300, 350, 400, 450, 500, 550, 600, 650, 700, 750, 800, 850, 900, 950, 1000, 1050, 1100, 1150, 1200, 1250, 1300, 1350, 1400, 1450, 1500, 1550, 1600, 1650, 1700, 1750, 1800, 1850, 1900, 1950, 2000, 2050, 2100, 2200, 2250, 2300, 2350, 2400, 2450, 2500, or more residues may be useful in facilitating a rertion between a substituted indole-3-pymvate and a C3 carbon source.<br>
[0629]	In another embodiment of the invention, one or more polypeptides with KHG<br>
aldolase activity encoded by a nucleic acid sequence having at least about 50%, 51%, 52%, 53%, 54%, 55%, 56%, 57%, 58%, 59%, 60%, 61%, 62%., 63%, 64%, 65%., 66%, 67%, 68%, 69%, 70%, 71%, 72%, 73%, 74%, 75%, 76%, 77%, 7S%, 79%, 80%, 81%, 82%, 83%, 84%, 85%, 86%, 87%, 88%, 89%, 90%, 91%, 92%), 93%, 94%, 95%, 96%, 97%, 98%), 99%, or more, or complete (100%) sequence identity to a nucleic acid in accordance with the invention, including SEQ ID NO:307, SEQ ID NO:309, SEQ ID N0:311, SEQ ID NO:313, SEQ ID NO:315, SEQ ID NO:317, SEQ ID NO:319, SEQ ID NO:321, SEQ ID NO:323, SEQ ID NO:325, SEQ ID NO:327, SEQ ID NO:329, SEQ ID NO:331, SEQ ID NO:333, SEQ ED NO:335, SEQ IDNO:336, SEQ ID NO:337, and SEQ ID NO;338 over a region of at least about 10, 15, 20, 25, 30, 35, 40, 45, 50, 75, 100, 150, 200, 250, 300, 350, 400, 450, 500, 550, 600, 650, 700, 750, 800, 850, 900, 950, 1000, 1050, 1100, 1150, 1200, 1250, 1300, 1350, 1400, 1450, 1500, 1550, 1600, 1650, 1700, 1750, 1800, 1850, 1900, 1950, 2000, 2050, 2100, 2200, 2250, 2300, 2350, 2400, 2450, 2500, or more residues may be useful in facilitating a reaction between a substituted indoIe-3-pyruvate and a C3 carbon source.<br>
10630]	One or more polypeptides with aldolase activity encoded by a nucleic acid<br>
sequence that hybridizes under stringent condition to a nucleic acid of SEQ ID NO:!, SEQ ID N0:3, SEQ ID N0:5, SEQ ID N0:7, SEQ ID N0:9, SEQ ID NOill, SEQ ID N0:!3, SEQ ID N0:15, SEQ ID N0:17, SEQ ID N0:19, SEQ ID N0:21, SEQ ID NO:23, SEQ ID NO:25, SEQ ID NO:27, SEQ ID NO:29, SEQ ID N0:31, SEQ ID<br>
 <br>
NO:33, SEQ ID NO:35, SEQ ID NO:37, SEQ ID NO:39, SEQ ID N0:41, SEQ ID NO:43, SEQ ID NO:45, SEQ ID NO:47, SEQ ID NO:49, SEQ ID N0:5L SEQ ID NO:53, SEQ ID NO:55, SEQ ID NO:57, SEQ ID NO:59, SEQ ID N0:6I, SEQ ID NO:63, SEQ ID NO:65, SEQ ID NO:67, SEQ ID NO:69, SEQ ID N0:7L SEQ ID NO:73, SEQ ID NO:75, SEQ ID NO:77, SEQ ID NO:79, SEQ ID N0:81, SEQ ID NO:83, SEQ ID NO:85, SEQ ID NO:87, SEQ ID NO:89, SEQ ID N0:91, SEQ ID NO:93, SEQ ID NO:95, SEQ ID NO:97, SEQ ID NO:99, SEQ ID NO:101, SEQ ID NO:103, SEQ ID NO:105, SEQ ID NO:107, SEQ ID NO:109, SEQ ID NO:!! 1, SEQ ID N0:!!3, SEQ ID N0:l!5, SEQ ID N0:!!7, SEQ ID N0:l!9, SEQ ID N0:12!, SEQ ID NO:223, SEQIDNO;125, SEQIDNO.i27, SEQIDNO;]29, SEQIDN0:131, SEQ ID NO:!33, SEQ ID NO:!35, SEQ ID NO:!37, SEQ ID NO:I39, SEQ ID N0:14!, SEQ ID NO:143, SEQ ID NO:145, SEQ ID NO:!47, SEQ ID NO:149, SEQ ID N0:151, SEQ ID NO:153, SEQ ID NO:155, SEQ ID NO:I57, SEQ ID 0:159, SEQ ID NO:i61, SEQ ID NO:163, SEQ ID NO:!65, SEQ ID NO:167, SEQ ID NO:169, SEQ ID N0:!7!, SEQ ID NO:!73, SEQ ID NO:175, SEQ ID NO:!77, SEQ ID NO:!79, SEQ ID N0:!8I, SEQ ID NO:183, SEQ ID NO:!85, SEQ ID NO:!87, SEQ ID NO:189, SEQ ID N0:!91, SEQ ID NO:193, SEQ ID NO:195, SEQ ID NO;197, SEQ ID NO:199, SEQ ID NO:201, SEQ ID NO:203, SEQ ID NO:205, SEQ ID NO:207, SEQ ID NO:209, SEQ ID N0:2! 1, SEQ ID NO:2!3, SEQ ID NO:2!5, SEQ ID NO:217, SEQ ID NO:2I9, SEQ ID NO:221, SEQ ID NO:223, SEQ ID NO:225, SEQ ID NO:227, SEQ ID NO:229, SEQ ID NO;231, SEQ ID NO:233, SEQ ID NO:235, SEQ ID NO:237, SEQ ID NO:239, SEQ ID NO:241, SEQ ID NO:243, SEQ ID NO:245, SEQ ID NO:247, SEQ ID NO:249, SEQ ID NO:251, SEQ ID NO:253, SEQ ID NO;255, SEQ ID NO:257, SEQ ID NO:259, SEQ ID NO:26U SEQ ID NO:263, SEQ ID NO:265, SEQ ID NO:267, SEQ ID NO:269, SEQ ID NO:271, SEQ ID NO:273, SEQ ID NO:275, SEQ ID NO:277, SEQ ID NO:279, SEQ ID NO:281, SEQ ID NO:283, SEQ ID NO:285, SEQ ID NO:287, SEQ ID NO;289, SEQ ID NO:291, SEQ ID NO:293, SEQ ID NO:295, SEQ ID NO:297, SEQ ID NO:299, SEQ ID NO:301, SEQ ID NO:303, SEQ ID NO:305, SEQ ID NO:307, SEQ ID NO:309, SEQ ID N0:311, SEQ ID NO:313, SEQ ID NO:3t5, SEQ ID NO:317, SEQ ID NO:319, SEQ ID NO:321, SEQ ID NO:323, SEQ ID NO;325, SEQ ID NO:327, SEQ ID NO:329, SEQ ID NO:331, SEQ ID NO:333, SEQ ID NO:335, SEQ ID NO:336, SEQ ID NO:337, and SEQ ID NO:338 may<br>
 <br>
be useful in facilitating a reaction between a substituted indole-3-pyruvate and a C3<br>
carbon source.<br>
[0631]	In one embodiment of the invention, one or more polypeptides with HMG<br>
aldolase activity encoded by a nucleic acid sequence that hybridizes under stringent condition to a nucleic acid of SEQ ID N0:1, SEQ ID N0:3, SEQ ID N0:5, SEQ ID N0:7, SEQ ID N0:9, SEQ ID N0:11, SEQ ID N0:13, SEQ ID N0:15, SEQ ID N0:17, SEQ ID N0:19, SEQ ID N0:21, SEQ ID NO:23, SEQ ID NO:25, SEQ ID NO:27, SEQ ID NO:29, SEQ ID N0:31, SEQ ID NO:33, SEQ ID NO:35, SEQ ID NO:37, SEQ ID NO:39, SEQ ID N0:4i, SEQ ID NO:43, SEQ ID NO:45, SEQ ID NO:47, SEQ ID NO:49, SEQ ID N0:51, SEQ ID NO:53, SEQ ID NO:55, SEQ ID NO:57, SEQ ID NO:59, SEQ roN0:6l, SEQ ID NO:63, SEQ ID NO:65, SEQ ID NO:67, SEQ ID NO:69, SEQ ID N0:71, SEQ ID NO:73, SEQ ID NO:75, SEQ ID NO:77, SEQ ID NO;79, SEQ ID N0:81, SEQ ID NO:83, SEQ ID NO:85, SEQ ID NO:87, SEQ ID NO:89, SEQ ID N0:91, SEQ ID NO:93, SEQ ID NO:95, SEQ ID NO:97, SEQ ID NO:99, SEQ ID NO:101, SEQ ID NO:103, SEQ ID NO:105, SEQ ID NO:107, SEQ ID NO:109, SEQ ID NOiIll, SEQ ID N0:113, SEQ ID N0:115, SEQ ID N0:1I7, SEQ ID N0:!19, SEQ ID N0:121, SEQ ID NO:123, SEQ ID NO:125, SEQ ID NO:127, SEQ ID NO:129, SEQIDN0:131, SEQIDNO:133, SEQIDNO:135, SEQ ID NO:137, SEQ ID NO:139, SEQ ID N0:141, SEQ ID NO:143, SEQ ID NO:145, SEQ ID NO:147, SEQ ID NO:149, SEQ ID N0:1S1, SEQ ID NO:153, SEQ ID NO:155, SEQ ID NO:157, SEQ ID NO:159, SEQ ID N0:161, SEQ ID NO:163, SEQ ID NO:165, SEQ ID NO:167, SEQ ID NO:169, SEQ ID N0:17l, SEQ ID NO:173, SEQ ID NO:I75, SEQ ID NO:I77, SEQ ID NO:179, SEQ ID N0:18l, SEQ ID NO:183, SEQ ID NO:185, SEQ ID NO:187, SEQ ID NO:189, SEQ ID N0:19l, SEQ ID NO:193, SEQ ID NO:195, SEQ ID NO:]97, SEQ ID NO:199, SEQ ID NO:201, SEQ ID NO:203, SEQ ID NO:205, SEQ ID NO:207, SEQ ID NO:209, SEQ ID N0:211, SEQ ID NO:213, SEQ ID NO:2I5, SEQ ID NO:217, SEQ ID NO:219, SEQ ID NO:22l, SEQ ID NO:223, SEQ ID NO:225, SEQ ID NO:227, SEQ ID NO:229, SEQ ID NOi23l, SEQ ID NO:233, SEQ ID NO:235, SEQ ID NO:237, SEQ ID NO:239, SEQ ID NO:241, SEQ ID NO:243, SEQ ID NO:245, SEQ ID NO:247, SEQ ID NO:249, SEQ ID N0:2S1, SEQ ID NO:253, SEQ ID NO;255, SEQ ID NO:257, SEQ ID NO:259, SEQ ID NO:261, SEQ ID NO:263, SEQ ID NO:265, SEQ ID NO:267, SEQ ID NO:269, SEQ ID NO:27l, SEQ ID NO:273, SEQ ID NO:275, SEQ ID NO:277, SEQ ID<br>
 <br>
NO:279, SEQ ID NO:281, SEQ ID NO:283, SEQ ID NO;285, SEQ ID NO:287, SEQ ID NO:289, SEQ ID NO:291, SEQ ID NO:293, SEQ ID NO:295, SEQ ID NO:297, SEQ ID NO:299, SEQ ID NO:301, SEQ ID NO:303, SEQ ID NO:305 may be useful in facilitating the reaction between the substituted indoIe-3-pyruvate and the C3 carbon source.<br>
[0632]	In another embodiment of the invention, one or more polypeptides with KHG<br>
aldolase activity encoded by a nucleic acid sequence that hybridizes under stringent condition to a nucleic acid of SEQ ID NO:307, SEQ ID NO:309, SEQ ID N0:3n, SEQ ID NO:313, SEQ ID NO:3I5, SEQ ID NO:317, SEQ ID NO:319, SEQ ID NO:321, SEQ ID NO:323, SEQ ID NO:325, SEQ ID NO:327, SEQ ID NO:329, SEQ ID NO:331, SEQ ID NO:333, SEQ ID NO:335, SEQ ID NO:336, SEQ ID NO:337, and SEQ ID NO:338 may be useful in faciUtating a reaction between a substituted indole-3-pyruvate and a C3 carbon source.<br>
[0633J	In one embodiment, the substituent group of the substituted indole-3-pyruvate is a<br>
halogen atom attached to any carbon atom of the indole ring. In another embodiment, the substituent group is a chlorine atom attached to any carbon of the indole ring. In yet another embodiment, the monatin derivative is 4-hydroxy-4-(6-methylindole-3-ylmethyl)glutanuc acid.<br>
[0634]	Polypeptides having aldolase activity, and in accordance with some embodiments<br>
of the invention, may be used in a multi-step pathway in which one or more step is a chemical synthesis reaction. For example, in some embodiments, one or more polypeptides having aldolase activity can facilitate a reaction between pyruvate and indole-3-pyruvate to yield monatin precursor. The monatin precursor can then be purified. A reductive amination reaction of the monatin precursor can then be utilized to yield monatin.<br>
[0635]	Polypeptides having aldolase activity, and in accordance with some embodiments<br>
of the invention, as well as the other enzymes used in the process for producing monatin and monatin derivatives may be used in pure, crude, isolated, or ammonium sulfate suspension form.<br>
|0636]	Polypeptides having aldolase activity, and in accordance with some embodiments<br>
of the invention, may be optimized using stabilizing agents, including dithiothreitol ("DTT") and p-mercaptoethanol.<br>
 <br>
[0637]	Monatin or monatin derivative that is produced utilizing one or more of the<br>
polypeptides disclosed herein, is generally at least about 50 to about 99Vo R.,R-monatin or R,R-monatin derivative, by weight of the total monatin or monatin derivative produced. In other embodiments, the monatin or monatin derivative produced utilizing one or more of the polypeptides disclosed herein, is greater than 60% R,R-inonatin or R,R-monatin derivative, by weight of the total monatin produced; for example, the R,R-monatin or R,R-monatin derivative is 70%, 75%, 80%, 85%, 90%, 91%, 92%&gt;, 93%, 94%, 95%, 96%, 97%, 98% or 99% of the total monalin or monatin derivative produced Alternatively, various amounts of two or more preparations of monatin or monatin derivative can be combined so as to result in a preparation that is a desired percentage of R,R-monatin or R,R-monatin derivative. For example, a monatin preparation that is 60% R,R-monatin can be combined with a monatin preparation that is 90% R,R-monatin; if equal amounts of 60% and 90%. R,R-moDatin preparations are combined, the resulting monatin preparation would be 75% R,R~monatm,<br>
[0638]	The monatin or monatin derivative,  or an intermediate  (including monatin<br>
precursor), produced utilizmg one or more of the polypeptides disclosed herein, may be purified from the components of the reaction, hi one embodiment, the monatin, monatin derivative or intermediate, such as monatin precursor, may be purified simply by removing the substance that is to be purified from the enzyme preparation in which it was synthesized.<br>
[0639]	In other embodiments, the intermediate, monatin precursor, monatin or monatin<br>
derivative is purified from a preparation in which it was synthesized so that the resulting "purified" composition or preparation is at least about 5-60% monatin by weight of total organic compoimds. In another embodiment, the monatin, monatin derivative or intermediate, such as monatin precursor, may be purified to a degree of purity of at least about 70%, 80%, 90%, 95% or 99% by weight of total organic compounds. The monatin, monatin derivative or the intermediate (including monatin precursor), produced utilizing one or more of the polypeptides disclosed herein, may be purified from the components of the reaction by any method known to a person of ordinary skill in the art. Optimally, the purified monatin or intermediate may be repeatedly recrystallized until the desired degree of purit&gt;' is achieved.<br>
 <br>
[0640]	The following examples are offered to illustrate, but not to limit the claimed<br>
invention.<br>
EXAMPLES<br>
Example 1<br>
Detection of Monatin, Monatin Precursor, Tryptophan, Alanine, Aspartate, and<br>
Glutamate<br>
(0641]	This example describes methods used to detect the presence of monatin, monatin<br>
precursor ("MP"), tryptophan, aspartate, alanine, and glutamate. It also describes a method for the separation and detection of the four stereoisomers of monatin.<br>
LC/MS/MS Multiple Reaction Monitoring ("MRM") Analysis  of Monatin  and Tryptophan<br>
[0642]	Analyses of mixtures for monatin and tryptophan derived from in vitro or in vivo<br>
biochemical reactions were performed using a Waters/Micromass liquid chromatography-tandem mass spectrometry (LC/MS/MS) instrument including a Waters 2795 liquid chromatograph with a Waters 996 Photo-Diode Array (PDA) absorbance monitor placed in series between the chromatograph iind a Micromass Quattro Ultima triple quadrupole mass spectrometer. LC separations were made using an Xterra MS Cg reversed-phase chromatography column, 2.1mm x 250 mm at 40 "C. The LC mobile phase consisted of A) water containing either (i) 0.05% (v/v) trifluoracetic acid or (ii) 0.3% formic acid and 10 mM ammonium formate and B) methanol containing either (i) 0.05% (v/v) trifluoracetic acid or (ii) 0.3% formic acid and 10 mM ammonium formate.<br>
[0643]	If the LC mobile phase consisted of A) water containing 0.05% (v/v) trifluoracetic<br>
acid and B) methanol containing 0.05% (v/v) trifluoracetic acid, gradient elution was linear from 5% B to 35% B, 0-4 minutes, linear from 35% B to 60% B, 4-6.5 minutes, linear from 60% B to 90% B, 6.5-7 minutes, isocratic at 90% B 7-11 minutes, linear from 90% B to 95% B, 11-12 mmutes, linear from 95% B to 5% B, 12-13 minutes, with a 2 minute re-equilibration period between runs. The flow rate was 0.25 mL/min, and PDA absorbance was monitored from 200 nm to 400 nm. Ail parameters of the ESl-MS were optimized and selected based on generation of protonated molecular ions ([M + H]+) of<br>
 <br>
the analytes of interest, and production of characteristic fragment ions. The following<br>
instrumental parameters were used for LC/MS/MS Multiple Reaction Monitoring (MRM)<br>
analysis of monatin and tryptophan: Capillary; 3.5 kV; Cone: 40 V; Hex 1: 20 V;<br>
Aperture: 0 V; Hex 2: 0 V; Source temperature: 100 °C; Desolvation temperature: 350<br>
°C; Desolvation gas: 500 L/h; Cone gas: 50 L/h; Low mass resolution (Ql): 12.0; High<br>
mass resolution (Ql): 12,0; Ion energy: 0.2; Entrance: -5 V; Collision Energy: S; Exit:<br>
IV; Low mass resolution (Q2): 15; High mass resolution (Q2): 15; Ion energy (Q2): 3.5;<br>
Multiplier: 650. Five monatin-specific parent-to daughter MRM transitions are used to<br>
specifically detect monatin in in vitro and in vivo reactions. The transitions monitored<br>
are 293.1 to 158.3, 293.1 to 168.2, 293.1 to 211.2, 293.1 to 230.2, and 293.1 to 257.2.<br>
Tryptophan is monitored with the MRM transition 204.7 to 146.4. For internal standard<br>
quantification of monatin and tryptophan, four calibration standards containing four<br>
different ratios of each analyte to d5-tryptophan and dS-monatin, are analyzed. These<br>
data are subjected to a linear least squares analysis to form a calibration curve for monatin<br>
and tryptophan. To each sample is added a fixed amount of d5-tryptophan and d5-<br>
monatin (d5-monatin was synthesized from d5-tryptophan according to the methods from<br>
WO03/091396 A2), and the response ratios (monatin/d5-monatin; tryptophan/d5-<br>
tryptophan) used in conjunction with the calibration curves described above to calculate<br>
the amount of each anal3e in the mixtures.<br>
[0644]	If the LC mobile phase was A) water containing 0.3% formic acid and 10 mM<br>
ammonium formate and B) methanol containing 0.3% formic acid and 10 mM ammonium formate, the gradient elution was linear from 5% B to 45% B, 0-8.5 minutes, linear from 45% B to 90% B, 8.5-9 minutes, isocratic from 90% B to 90%o B, 9-12.5 minutes, linear from 95% B to 5% B, 12.5-13 minutes, with a 4 minute re-equilibration period between runs. The flow rate was 0.27 mL/min, aid PDA absorbance was monitored from 210 nm to 400 nm. All parameters of the ESI-MS were optimized and selected based on generation of protonated molecular ions ([M + H]+) of the analytes of interest, and production of characteristic fragment ions. The insfrumental parameters used for this secondary mobile phase are the same as above. Foiir monatin-specific parent-to daughter MRM transitions and one tryptophan specific parent to daughter transition are used to specifically detect monatin and tryptophan in in vitro and in vivo reactions.   The transitions monitored are 293.1 to 158.0, 293.1 to 168.0, 293.1 to 2U.5,<br>
 <br>
and 293.1 to 257.0. Tryptophan is monitored with the MRM transition 205.2 to 146.1. For internal standard quantification of monatin and tryptophan, four calibration standards containing four different ratios of each analyte to d5-tryptophan and d5-monatin, are analyzed. These data are subjected to a iinear iet squares analysis to form a calibration curve for monatin and tryptophan. To each sample is added a fixed amount of d5-tryptophan and d5-monatin (d5-monatin was synthesized from d5-tryptophan according to the methods from ■WO03/091396 A2), and the response ratios (monatin/d5-monatin; tryplophan/d5-tiyptophan) in conjunction with the calibration curves described above are used to calculate the amount of each analyte in the mixtures. Parent to daughter mass transitions monitored for d5-tiyptophaii and d5-monatin are 210.2 to ]51-J, and 298.1 to 172.0 respectively.<br>
Accurate Mass Measurement of Monatin<br>
[0645]	High resolution MS analysis was carried out using an Applied Biosystems-Perkin<br>
Elmer Q-Star hybrid quadrupole/time-of-flight mass spectrometer. The measured mass for protonated monatin used tryptopln as an internal mass calibration standard. The calculated mass of protonated monatin, based on the elemental composition C14H17N205 is 293.1137. Monatin produced using the biocatalytic process described in Examples 2 and 3 showed a measured mass of 293.1144. This is a mass measurement error of less than 2 parts per million ("ppm"), providing conclusive evidence of the elemental composition of monatin produced enzymatically.<br>
Chira! LCTMS/MS ("MRM") Measurement of Monatin<br>
[0646]	Determination of the stereoisomer distribution of monatin in in vitro and in vivo<br>
reactions was accomplished by derivitization with I-fluoro-2-4-dinitrophenyl-5-L-alanine amide ("FDAA"), followed by reversed-phase LC/MS/MS MRM measurement.<br>
Derivatization of Monatin with FDAA<br>
[0647]	To 50 fiL of sample or standard and 10 iL of internal standard was added either<br>
100 jiL or 200 L of a 1% solution of FDAA in acetone. Twenty or forty [iL, respectively, of 1.0 M sodium bicarbonate was added, and the mixture incubated for 1 h at 40 "C with occasional mixing.  The sample was removed and cooled, and neufralized<br>
 <br>
with 20 iiL of 2-0 M HCI (more HCI may be required to effect neutralization of a buffered biological mixture). After degassing w complete, samples were ready for analysis by LC/MS/MS,<br>
LC/MS/MS Multiple Reaction Monitoring for tlie Determination of the Stereoisomer Distribution of Monatin in in vitro and in vivo Reactions<br>
[0648]	Analyses were performed using the LC/MS/MS instrumentation described above.<br>
LC separations capable of separating a]] four stereoisomers of monatin (speciijcall)' FDAA-monatin) were performed on a Phenomenex Luna 2.0 x 250 mm (3 m) C!8 (2) reversed phase chromatography column at 40 °C. The LC mobile phase consisted of A) water containing 0.05% (mass/volume) ammonium acetate and B) acetonitrile. The elution was isocratic at 13% B, 0-2 minutes, linear from 13% B to 30% B, 2-15 minutes, linear ftom 30% B to 80% B, 15-16 minutes, isocratic at 80% B 16-21 minutes, and linear from 80% B to 13% B, 21-22 minutes, with an 8 minute re-eqmlibration period between runs. The flow rate was 0.23 mL/mtn, and PDA absorbance was monitored from 200 nm to 400 nm. All parameters of the ESI-MS were optimized and selected based on generation of deprotonated molecular ions ([M - H]-) of FDAA-monatin, and production of characteristic frment ions.<br>
[0649]	The following instnunental parameters were used for LC/MS analysis of monatin<br>
in the negative ion ESl/MS mode; Capillary: 2.0 kV; Cone: 25 V; Hex 1:10 V; Aperture: 0 V; Hex 2: 0 V; Source temperature: 100°C; Desolvation temperature: 350 "C; Desolvation gas: 500 L/h; Cone gas: 50 L/h; Low mass resolution (Ql): 12.0; High mass resolution (Ql): 12.0; Ion energy: 0.2; Entrance: -5V; Collision Energy: 20; Exit: IV; Low mass resolution (Q2): 12; High mass resolution (Q2): 12; Ion energy (Q2): 3.0; Multiplier: 650. Three FDAA-monatin-specific parent-to daughter transitions are used to specifically detect FDAA-monatin in in vitro and in vivo reactions. The transitions monitored for monatm are 543.2 to 268.1, 543.2 to 499.3, and 543.2 to 525.3. Monatin internal standard derivative mass transition monitored was 548.2 to 530,3. Identification of FDAA-monatin stereoisomers is based on chromatographic retention time as compared to purified synthetic monatin stereoisomers, and mass spectral data. An internal standard is used to monitor the progress of the reaction and for confirmation of retention time of the S,S stereoisomer.<br>
 <br>
Liquid Chromatography-Post Column Fluorescence Detection of Amino Acids Including Giutamate and Alanine<br>
[0650]	Liquid chromatography with post-column fluorescence detection (LC/OPA) for<br>
the determination of giutamate and alanine in in vitro and in vivo reactions was performed on a Waters 2690 LC system or equivalent combined with a Waters 474 scanning fluorescence detector, and a Waters post-column reaction module. Semi¬quantitative analyses of monatin and tryptophan were also performed using this method. LC separations were performed on an Interaction-Sodium loaded ion exchange colxmm at 60°C. Mobile phase A w Pickering Na 328 buffer (Pickering Labomtories, Inc.; Mountain View, CA). Mobile phiise B was Pickering Na 740 buffer. The gradient elution was from 0% B to 100% B, 0-20 minutes, isocratic at 100% B, 20-36 minutes, and linear from 100% B to 0% B, 36-37 minutes, with at least a 5 minute re-equilihration period between runs, depending on sample matrix. The flow rate for the mobile phase was 0.5 mL/min. The flow rate for the OPA post-column derivatization solution was 0.5 mL/min. The fluorescence detector settings were EX 338-340 nm and Em 420-425 nm. Norleucine was employed as an internal standard for the analysis. Identification of amino acids was based on chromatographic retention time data for purified standards.<br>
Detection of L- and D-Amino Acids by LC/MS/MS<br>
[0651]	Samples containing a mixture :of L- and D-amino acids such as lysine, alanine,<br>
methionine, tyrosine, leucine, phenylalanine, tryptophan, giutamate, and aspartate from biochemical reaction experiments were first treated with formic acid to denature protein. The sample was then centrifuged and filtered through a 0.45 m nylon syringe filter prior to LC/MS/MS analysis. Identification of L- and D-amino acids was based on retention time and mass selective detection. LC separation was accomplished by using Waters 2690 liquid chromatography system and an ASTEC 2.1 mm x 250 mm Chirobiotic TAG chromatography column with column temperature set at 45o C. LC mobile phase A and B were 0.25% acetic acid and 0.25% acetic acid in methanol, respectively. Isocratic elution was used for all methods to separate the L and D isomers. Lysine was eluted using 80% mobile phase A, and 20% B. Giutamate, alanine, and methionine were separated with elution of 60%* mobile phase A and 40%i B and a flow rate of 0.25 mL/min.    Aspartate, tryptophan, tyrosine, leucine, and phenylalanine were separated<br>
 <br>
isomerically with 30% mobile phase A. and 70% B with a flow rate of 0.3 mL/min for all<br>
but phenylalanine, which was run at a tlow rate of 0.25 mL/min.<br>
[0652]	The detection system for analysis of L- and D-amino acids included a Waters 996<br>
Photo-Diode Array (PDA) detector and a Micromass Quattro Ultima triple quadrupole mass spectrometer. The PDA, scanning from 195 to 350 nm, was placed in series between the chromatography system and the mass spectrometer. Parameters for the Micromass Quattro Ultima triple quadrupole mass spectrometer operating in positive electrospray ionization mode (+ESI) were set as the following: Capillary: 3.0 kV; Cone: 20 V; Hex 1: 15 V; Aperture: 1 V; Hex 2: 0 V; Source temperature: 100 °C; Desolvation temperature; 350 °C; Desolvation gas: 530 L/h; Cone gas: 30 L/h; Low mass Ql resolution: 12.5; High mass Ql resolution: 12.5; Ion energy 1: 0.2; Entrance: -5; Collision: 8; Exit 1: 10; Low mass Q2 resolution: 12.5; High mass Q2 resolution: 12.5; Ion energy 2: 0.5; Multiplier: 650 V. MS/MS experiments with Multiple Retion Monitoring (MRM) mode were set up to selectively monitor reaction transitions of 147.8 to 84.2 and 147.8 to 102.1 for glutamate, 134.00 to 74.30, and 134.00 to 88.2 for aspartate, 147.3 to 85.0 for lysine, 150.3 to 104.8 for methionine, 182.3 to 137.0 for tyrosine, 132.3 to 87.0 for leucine, and 166.3 to 121.0 for phenylalanine. In the case where two transitions are listed, the lattra- transitions were used for quantification. For tryptophan, MS/MS experiments with Multiple Reaction Monitoring (MRM) mode were set up to selectively monitor reaction transitions of 205.2 to 118.2, 205.2 to 146.1, and 205.2 to 188.2, and the transition from 212.1 to 151.1 for d8-DL tryptophan. Tryptophan quantification was achieved by determining the ratio of anaiyte response of transition 205.2 to 146.1 to that of the internal standard, d8-D,L tryptophan. Alternatively, quantification of tryptophan, glutamate, and aspartic acids were based off signal responses of m/z=I46.5, m/z=102.1, and m/z=88.2, respectively.<br>
Production of Monatin and Monatin Precursor ("MP") for Standards and for Assays Production of Monatin<br>
[0653]	A racemic mixture of R,R and S,S monatin was synthetically produced as<br>
describedinU.S. Patent No. 5,128,482.<br>
[0654]	The R,R and S,S monatin were separated by a derivatization and hydrolysis step.<br>
Briefly, the monatin racemic mixture was esterified, the free amino group was blocked<br>
 <br>
with Cbz, a lactone was formed, and the S,S lactone was selectively hydroiyzed using an immobilized protease eazyme. Tlie monatin can also be separated as described in BassoH, A. et al., Eur. J. Org. Chem., 8:1652-1658, (2005).<br>
MP Production<br>
J0655]	R-MP was produced by the transamination of R,R monatin using AT-103 broad<br>
range D-aminotransferase (BioCatalytics, Pasadena, CA) in 0.1 M potassium phosphate buffer, using sodium pyruvate as the amino acceptor. S-MP w produced by the transamination of S,S monatin using AT-102 L-aminotransferase (BioCatalytics, Pasadena, CA) in 0.1 M potassium phosphate buffer, using sodium pyruvate as the amino acceptor. Both reactions were carried out at 30 'C and at a pH of approximately 8.0-8.3, for approximately 20 hours. Both compounds were purified using preparative scale HPLC with a Rohm and Haas (Philadelphia, PA) hydrophobic resin (XADTM1600), eiuting in water. Samples containing greater than 90% purity monatin precursor were collected and freeze-dried.<br>
Example 2<br>
Defection of Monatin Precursor<br>
[0656]	This example describes methods used for the sqsaration and detection of the two<br>
enantiomers of monatin precursor.<br>
Non-chirat Method for Detection of Monatin Precursor<br>
[0657]	Reaction samples from 96-well plates were injected onto an Agilent Zorbax RX-<br>
ClSi 3.5 am, 3.0 x 150 mm column using a CTCPal auto-sampler (LEAP Technologies, Carrboro, N.C.). Products were separated using a H20/ACN (0.1% Formic acid) gradient:<br>
Time: 0.00 min	5% B<br>
Time: 4.00 min	100% B<br>
Time: 5.00 min	100% B<br>
Time:5.10min	5%B<br>
Time: 6.50 min	5% B<br>
 <br>
[0658]	The gradient was provided by LC-lOADvp pumps (Shimadzu, Kyoto, Japan) ai<br>
0.8 mL/min. Products were detected using API4000 Turbolon-Spray triple-quad mass spectrometer (Applied Biosystems, Foster City, CA). Ion spray and Multiple-ion monitoring were performed for the analytes of interest in the negative ion mode, and each analysis lasted 6.5 minutes.<br>
Pyruvate = 87.1 [M-H+]-Indoie-3-pymvate = 202.1  [U-B+]-Product = 290,0 [M-H+]-<br>
Cbiral CE analysis of R &amp; S Monatin Precursors<br>
(0659]	A   P/ACE     MDQ   capillary   electrophoresis   instrument   (Becknian  Coulter,<br>
Fullerton, CA) was used. The Chiral Development kit was used and includes small amounts of several chiral selectors, necessary buffers and 2 capillaries (Beckman Coulter, Fullerton, CA). Alternatively, for the MP assay only, the following reagents and other supplies can be obtained separately irom Beckman Coulter (Fullerton, CA) or elsewhere:<br>
Coated capillary N-CHO; 50 um ID, 65 cm total length or fused silica capillary.<br>
25 mM phosphate buffer, pH 5<br>
25 mg hydroxypropyl-p-cyclodextrin<br>
Capillary conditioning solution, 10 mL (alternatively, can use 0.5% polyethylene oxide solution, M 600,000 or 300,000 Daltons)<br>
Capillary Electrophoresis ("CE") Analysis<br>
[0660]	A neutral coated capillary, 50 um ID, 60 cm (50 cm to detection) or 30 (20) cm<br>
was used along with DAD detection (or simple UV) at 214 nm. The separation capillary was thermostated at 15°C, samples at 4°C. The separation buffer was 20 mM hydroxypropyl-p-cycylodextrin, 25 mM phosphate, pH 5. Sample injection was typically 0.5 psi, 5 s. Separation was at 500 V/cm, reversed polari (15 kV for 30 cm capillary, 30 kV for 60 cm). Typical current used during separation was -28 |JA. Typical migration times for MP peaks were around 3.5 minutes (20 cm effective length) or 8 minutes (50 cm)<br>
 <br>
[0661]	An optional capillary cleaning/washing/conditioning step, prior to sample runs<br>
used H2O 4 minutes, 0.1 M HCl Irninutes, H2O 1.5 minutes, capillary conditioning<br>
solution 4 minutes, H20,1 minute, separation buffer 4 minutes.<br>
[0662]	A summary of the run method was: separation buffer rinse 1-2 minutes, sample<br>
injection 5 s at 0.5 psi, separation 5-10 minutes at reversed voltage polarity 15 or 30 kV,<br>
depending on capillary length.<br>
Example 3<br>
General assay for pyruvate aldolases<br>
[0663]	An exemplary method to measure the activity of different pyruvate aldolases uses<br>
a general substrate, 4-Carboxy-4-hydTOxy-2-oxoadipate (CHA). The CHA assay was adapted from literature assays (eg. See E.E. Dekker &amp; R.P. Kitson, J. Biol. Chem. 267, 10507-10514, 1992). A typical assay comprised 50 mM sodium phosphate pH 7.5, 1 mM MgClj, 1 mM CHA, 10 jig/ml D-lactate dehydrogenase (LDH) from Lactobacillus leichmanii (Sigma-Aldrich, St. Louis, MO), 0.5 mM NADH. The assay was started by adding enzyme (typically between 1 to 5 pL). Liberation of pyruvate, coupled to the formation of NAD" was monitored continuously in a spectrophotometer at 340nm.<br>
[0664]	The CHA was synthesized according to the procedure described in Tack, B.F.<br>
Chapman, P.J., and S. Dagley. Biol Chem. 247 6438-6443 (1972).<br>
[0665]	A imit of enzyme activity, such as pyruvate aldolase, such as HMG and/or KHG<br>
aldolase enzyme, is defmed as the amount that liberates sufficient pyruvate to lower the absorbance at 340mn by 1 OD per minute.<br>
Example 4<br>
Discovery of Novel Keto-hydroxy-glutarate (KHG) and Hydroxy-methyl-keto-glutarate (HMG) aldolases from Diversa environmental libraries<br>
[0666]	Over 150 unique HMG aldolases and 15 KHG aldolases were discovered by<br>
screening Diversa DNA libraries. These aldolase genes were sequenced and subcloned into a suitable expression vector. This vector was then transformed into a suitable expression host for production of sufficient amounts of the aldolase for enzyme characterization. A selected set of aldolases were tested for the activity on CHA and also<br>
 <br>
for the formation of monatin precursor (MP). AH the enzymes discovered and described in this patent have potential for vse in other carbon-carbon bond forming reactions between an alpha-keto acid acceptor and pyruvate or pyruvate derivative donor as exemplified in the general reaction scheme below.<br>
 <br>
 <br>
[0669]	Aldol reactions were performed with 20 mM indole-3-pyruvate ("I3P"), 50 mM<br>
pyravate, WQ mM sodium phosphate pH 7, ImJVI MgCl2, 100 ug/mL aidoiase. Reactions were incubated at room temperature in the dark. AJiquots (30 fiL) were removed at various times and reactions were stopped by storing the samples on ice. A portion of each aliquot was submitted for CF. analysis while the remaining portion was diluted 1:1000 in 50% acetonitrile and submitted for LC/MS analysis.<br>
 <br>
 <br>
 <br>
(0670]	Note that selectivities for R-MP of 98+% indicate that DO S-MP was detected.<br>
Given the sensitivity of the CE assay, the results indicate that the at least 98% of MP<br>
formed is the R-enantiomer.   Thus, enzymes that are listed as 98+% are at least 98%<br>
selective towards R-MP and may be up to 100% selective.<br>
[0671]	Table 2 also shows the activity of the enzymes on a general aldolase substrate,<br>
CHA, as well as the relative expression of each enzyme, as determined by SDS-PAGE.<br>
Note that several enzymes did not show detectable activity on CHA yet they did exhibit<br>
activity in making MP.<br>
[0672]	In summary, the aldolases show a wide range of activities, expression and<br>
selectivities.     Moreover, there are  numerous aldolases that  show exquisitely high<br>
selectivities (98% or greater) for R-MP.<br>
Example 6 Discovery of plant pyruvate aldolases<br>
[0673]	Degenerate PCR primers (see below) were designed and used to extract aldolase<br>
genes from cDNA prepared from Sclerochiton Umfolius. The 5' and 3" ends of the genes were recovered and the iuU length genes were then PCR amplified.<br>
 <br>
 <br>
Cloning of Bacillus sphaericiis D-amino acid aminotransferase<br>
[0674]	The B. sphaericiis D-amino acid aminotransferase (EC 2.6.I.2I, also known as D-<br>
alanine aminotransferase or D-aspailate aminotransferase) was produced recombinantly for use in coupled assays with the various aldolases. This enzyme is homologous to D-aminotransferases described previously for production of monatin (U.S. Publication No. 20040063175 and U.S. Publication No. 20050282260).<br>
Strains<br>
(0675]	B. sphaericus (ATCC numbei 10208) was grown on Nutrient Agar at 30°C<br>
overnight. Groups of colonies were placed in 100 \iL of sterile water and heated for 5 minutes at 95°C, to disrupt the cells. Three |iL was used in subsequent Polymerase Chain Reaction (PCR) amplifications.<br>
Polymerase Chain Reaction Protocol<br>
[0676]	Primers were designed for cloning into pET 28b and pET 30a vectors (Novagen,<br>
Madison, WI), ming the Ncol and BamHI sites. The pET 30 construct contains an N-terminal His-tag and S-tag, whereas the pET 28 construct is untagged.<br>
Bacillus sphaericus dat primers:<br>
[0677]	N    term:    5'-GATATACCATGGCATACTCATTATGGAATG-3'    (SEQ    ID<br>
NO:383) and C term: 5'-GTTATCGGATCCTTAGGCArTAATTGAAATTG-3' (SEQ ID NO:384).<br>
 <br>
[0678]	The coding region was amplified using the following PCR protocol.  In a 50 [iL<br>
reaction 3 jiL template, 1.6 fjM of each primer, 0.25 mM each dNTP, 3.5 U Expand High Fidelity Polymerase (Roche, Indianapolis, IN), and IX Expand" buffer with Mg were used. The thermocycler program used included a hot start at 94°C for 3 minutes, followed by 8 repetitions of the following steps: 94°C for 30 seconds, 52°C for 30 seconds, and 72'C for 2 minutes. Twenty-two subsequent cycles were done with an annealing temperature of 58°C- After 30 cycles, the sample was maintained at 72°C for 7 minutes and then stored at 4°C. Clean PCR products of the correct size were obtained (approximately 850 bp for the dat gene).<br>
Cloning<br>
[06791	The PCR products were purified using the Qiagen QlAquick PCR purification kit<br>
(Qiagen, Valencia, CA), and digested with BamHI and Ncol in BaniHI buffer (New England Biolabs, Ipswich, MA).<br>
[0680]	Digested vector and inserts were purified using the Qien QlAquick Gel<br>
Extraction Kit (Qiagen, Valencia, CA). Ligations were done usmg the Roche Rapid DNA Ligation Kit (Roche, Indianapolis, IN) and purified using the QlAquick PCR purification kit (Qiagen, Valencia, CA). The ligations were transformed into Escherichia coli DHIOB using a 0.2 cm cuvette and a Bio-Rad Gene Pulser II system as described in the Bio-Rad electroporation manual (Bio-Rad, Hercules, CA). The cells were allowed to recover in 900 \iL SOC medium for 30 minutes at 37'C at 225 rpm. Cells were plated on LB-agar plates containing kanamycin (25 jig/mL).<br>
[06811	Plasmid DNA was purified using the Qiagen spin miniprep kit (Qiagen, Valencia,<br>
CA) and screened for the correct inserts by restriction digest with BamHI and Ncol. The sequences of plasmids that appeared to have the correct insert were verified by dideoxy chain termination DNA sequencing at Agencourt BioScience Corporation (Beverly, MA). Sequencing verified the coding sequence found in NCBI accession number AF081278 Region: 134..985 (gi: 3513754), which produces a protein with amino acid sequence as listed in accession number AAC33964 (gi: 3513755).<br>
Gene Expression and Assays<br>
[0682]	Plasmid DNA was subcloned into E. coli expression host BL21(DE3) (Novagen,<br>
Madison, WI).   The cultures were grown and the plasmids were isolated using Qiagen<br>
 <br>
miniprep kit (Qiagen, Valencia, CAj, and analyzed by restriction digest to confirm<br>
identi'. Induction was typically performed in LB medium containing kanamycin (50<br>
g/mL). The cells were grown to an ODgoo of 0.4-0.8, induced with 0.1 mM IPTG<br>
(isopTopyl thiogalacatoside) and sampled at 4 hours post induction. Cell extracts were<br>
prepared according to the protocol accompanying the Novagen BugBuster' reagent<br>
(Novagen, Madison, Wl) (with benzonase nuclease and Roche complete protease<br>
inhibitor cocktail added (Roche, hidianapolis, IN)). Very high levels of soluble protein<br>
were obtained at the predicted molecular weight, as judged by SDS-PAGE. For some<br>
reactions, the pET 30 gene product was purified using His-Bind cartridges following<br>
manufacturer's protocols (Novagen, Madison, WI). The eluent iractions were desalted on<br>
PD-10 (GE Healthcare, Piscataway, NJ) columns and eluted in 25-100 mM potassium<br>
phosphate buffer, pH 7.5.<br>
[0683]	Cell   extracts   were   analyzed   for   D-aminotransferase   activity  by   following<br>
production of alanine from pyruvate and D-tryptophan using the following protocol. One mL reactions were typically carried out in 100 mM potassium phosphate buffer (pH 7.5), 50 )iM pyridoxal phosphate, 25 mM sodium pyruvate, and 50 mM D-tryptophan. The reactions were initiated by the addition of cell free extracts or purified enzyme and were incubated 15 minutes-overnight at SOC, with mild shaking. Formic acid w added to a fmal concentration of two percent to stop the reaction, and the precipitated protein was removed by centrifligation. Control reactions without added protein were also performed. Zero time points were also used as negative controls. Alanine was detected using OPA derivatization as described in Example I.<br>
Example 8<br>
Comparison of total monatin production and isomeric distribution for the<br>
polypeptides with aldolase activity of SEQ ID NO:8, SEQ ID NO:4, SEQ ID<br>
NO:I2, and SEQ ID NO:28, and C. testosteroni ProA<br>
[0684]	AT-103 transaminase (a broad specificity D-aminotransferase) was purchased<br>
from BioCatalytics (Pasadena, CA) ajid either this enzyme or the recombinant enzyme produced in Example 7 was used in coupled reactions with HMG aldolases to produce monatin from D-tryptophan and pyruvate as described in U.S. PubUshed AppHcation No. 20050282260. The ProA aldolase from C testosteroni was used as a benchmark aldolase<br>
 <br>
for comparative pxirposes, and was prepared as described in U.S. Published Application No. 20040063175 and WO 03091396 A2. The aldolases tested were isolated and trsformed as described above in Example 4.<br>
[0685]	To produce test quantities of each aidolase, 50 niL cultures were grown in LB<br>
medium containing ampicillin (100 g/mL), to an ODoo of approximately 0.5. The strains containing SEQ ID N0:7, SEQ ID N0:3, and SEQ ID N0:11 constructs were induced with 100 nM of PTG. The strain containii the SEQ ID NO:27 construct was induced with 200 ig/L anhydrotetracyclitie. The cells were grown 5 hours post-induction, and cellular extracts were prepared according to manufacturer's protocols (Novagen, Madison, WI, Bugbuster reagent). Benzonticlease and protease inhibitor were also added. The soluble proteins in the celtular extracts were separated on a Bio-Rad Laboratories ExperJon Automated Electrophoresis Station (Bio-Rad, Hercules, CA) and analyzed for concentration and percent expression using the Experion Software version l.i.98.0.<br>
[0686]	The following were added per 1 mL of reaction mixture:   approximately  50 \ig<br>
aldolase (supplied in cellular extracts unless otherwise noted), 4 mM MgCh, 50 mM D-tryptophan, 0.5 mg purified B. sphaericus D-aminotransferase, 200 mM sodium pyruvate, 100 mM potassium phosphate buffer pH 7.5, and 0.05 mM PLP. Experiments were run in duplicate, with negative controls in which no aldolase was added. Samples were incubated 1 hr, 2 hrs, and overnight (17-20 hours) at 30 "C with gentle shaking. Small amounts of monatin (
 <br>
 <br>
 <br>
[0688]	The same experiments were done, side by side, using L-tryptophan as the starting<br>
substrate and coupling the aldolases with HexAspC broad specificity L-aminotransferase produced as described in U.S. Published Application No. 20050282260 and purified. Hiese reactions should yield primarily S,S monatin and R,S monatin. The reactions were also supplemented with 10 mM alpha-ketoglutarate as the amino acceptor for L-tryptophan transamination. Again, duplicate results are averaged below for total monatin (subtracting background levels without aldolase present), and percent S,S monatin is shown based on reversed phase LC peak area. In some cases, because the aldolases are quite R-speciiic and produce little total monatin, the reversed phase estimates of stereoisomeric distribution are less accurate due to some tailing of the tryptophan peak that can co-elute with the S,S/R,R monatin peak. The trends are still informative in comparing R-specificity of the aldolases. Results from fiirther analysis using the FDAA derivatization method are shown in parentheses for several samples, and are more accurate. Total monatin numbers above approximately 400 ppm are higher than the linear range of the scale of the standards used to quantitate the results, so are qualitative results. The C. testosteroni ProA aldolase topically produces 95-100% S,S monatin, as shown in U.S. Published Application No. 20050282260.<br>
 <br>
 <br>
 <br>
ID NO:28 is quite high compared to the benchmark ProA enzyme, this is also reflected in<br>
the low % S,S monatin produced, despite the high degree of specificity of the HexAspC<br>
aminotransferase for S-MP in these reactions. The total monatin numbers, when<br>
comparing S,S monatin production versus R,R monatin production, are not indicative of<br>
the aldolase activity. The D-aminotransferase is less active than HexAspC, particularly at<br>
the concentrations of MP that are present in these reactions.<br>
[0690]	For further comparison of the polypeptide with aldolase activity of SEQ ID NO;28<br>
to the ProA en2yme from C testosteroni, varying ratios of D-aminotransferase to aldolase were utilized in reactions starting with D-tryptophan (no duplicate samples for these experiments). Reactions were carried out as above. For reactions where aldolase concentration was kept constant, approximately 50 g was used. For reactions where D-aminotransferase was kept constant, 0.5 mg was used. For the 2 and 10 mg/mL concentration of IXaminotransferase, lyophilized enzyme was used. For the 2 highest D-aminotransferase concentrations, duplicates vrere run.<br>
 <br>
 <br>
 <br>
standard curve and are approximate values only. The maximum amount of R,R monatin produced, when diluted appropriately, was 1100 ppm. FDAA stereoisomeric analysis was done for the polypeptide with aldolase activity of SEQ ID NO:28 with 10 mg/mL D-aminotransferase samples. At two hours, the sample contained 98.5% R,R monatin. At 17 hours, the sample contained 95.9% R,R monatin. The polypeptide with aldolase activity of SEQ ID NO:28 produced high percentages of R,R monatin, even after long incubation times and using large amounts of aminotransferase. If adequate D-aminotransferase is supplied, the polypeptide with aldolase activity of SEQ ID NO:28 produces as much total monatin as C. testosteroni ProA aldolase, indicating a similar specific activity.<br>
 <br>
 <br>
[0693]	To further examine the R-specifici of the aldolases tested, experiments were<br>
done starting with L-tryptophan and HexAspC aminotransferase, which was produced and purified, as described in U.S. PubHshed Application No. 20050282260. The HexAspC shows a strong selectivity for transamination of S-MP versus R-MP, thus percentages above 50% R,S monatin indicate a highly stereospecific aldolase. Ten mM alpha-ketoglutarate was supplied as an amino acceptor; however, at high concentrations, pyruvate is also utilized by the L-aminotransferase. In these reactions, typically only S,S and R,S monatin are produced within the limits of detection of the FDAA derivatization protocol.<br>
 <br>
 <br>
<br>
 <br>
activity of SEQ ID NO:28, less total mooatin is produced and increasing the amount of aldolase does increase total monatin (as well as % S,S). These aldolases produce less S-MP substrate, the preferred substrate for the L-aminotransferase used. For enzymes that are less R-specific, such as ProA, increasing aldolase does not significantly improve total monatin production or % S,S monatin. Increasing the amount of L-aminotransferase added decreases the percentage of S,S monatin produced. Based on the above analysis, the polypeptide with aldolase activity of SEQ ID N0:8 is between ProA and the polypeptide with aldolase activity of SEQ ID NO:28 in terms of R-specificity, which agrees with data above where % R-MP is measured for the aldol step alone.<br>
Subcloning of SEQ ID NO:27<br>
(0695]	The  following primers  were  used to  PCR amplify the  aldolase  gene:   5'-<br>
gaggagctcgagtcagacgtatttcagtcctttttc-3'	(SEQ        ID        NO:385)        and        5'-<br>
agaagacatatgatttatcagccggggac-3' (SEQ ID NO:386). The aldolase gene SEQ ID NO-.27 encodes the polypeptide with aldolase activity of SEQ ID NO:28. The resulting PCR product was digested with iTioI and Ndel to cut at the sites that had been engineered into the primers. The fJBgment was gel pxirified (QIAquick Gel extraction Kit (Qiagen, Valencia, CA))and iigated (using T4 DNA Hgase) with pET28b that had been digested with Xho\ and Ndel and gel purified. The ligation was transformed into TOPI OF' chemically competent cells. Colonies growmg on the plates were screened for inserts and several isolates with inserts were submitted for DNA sequence analysis (Agencourt, Beverly, MA).<br>
Purification of the Polypeptide with Aldolase Activity of SEQ ID NO:28<br>
[0696]	Confirmed aldolase clones were transformed into either BL21 DE3 or BL21 DE3<br>
pLysS. Overnight cultures grown with the appropriate antibiotic were diluted into fresh media (typically 1:100) and grown to an ODmo "--6 with aeration at 37°C. Cultures were then induced with 1 mM IPTG and shifted to 30°C (with aeration) and incubation was<br>
 <br>
continued overnight. Cells were harvested by centrifugation. The cell pellet was typically subjected to one freeze thaw cycle to assist with cell lysis. The cell pellet was iysed in BugBuster and Benzonase (Novagen, Madison, WI) (according to the manufacturer's protocol). Cell debris was removed by centrifugation. The crude protein extract was applied to a HisBind column (Novagen, Madison, WI) that had been prepared according to the manufacturer's protocol. The column was washed and protein was eluted according to the manufacturer's protocol. The purified protein was desalted with PD-10 columns (GE Healthcare, Piscataway, NJ). The buffer used for the exchange was 50 mM potassium phosphate pH 7.5, 100 rnM NaCl, 4 mM MgCl2. Purified protein was concentrated with Amicon centrifugal concentrators (Millipore, Billerica, MA).<br>
Example 9<br>
Comparison of total monatin production and isomeric distribution for polypeptides<br>
with aldolase activity of SEQ ID NO:40, SEQ ID NO:298, SEQ ID NO:36, SEQ<br>
IDNO:62, SEQIDNO:64, SEQIDNO:96, SEQIDNO:54, SEQlDNO:122,<br>
SEQ ID NO: 142, SEQ ID NO:42, SEQ ID NO: 130, SEQ ID NO: 112, SEQ ID<br>
NO; 108, SEQ ID NO:94, SEQ ID NO:80, and SEQ ID NO:28<br>
[0697]	AT-103 transaminase (a broad specificity D-aminotransferase) was purchased<br>
fi-om BioCatalytics (Pasadena, CA) and either this enzyme or the recombinant enzyme produced in Example 7 was used in coupled reactions with HMG aldolases to produce monatin from D-tryptophan and pyruvate as described in U.S. Published Application No. 20050282260. The polypeptide with aldolase activity of SEQ ID NO:28 (his-tagged) was used as a benchmark aldolase for comparative purposes, and was produced and purified as described at the end of Example 8. The other aldolases tested were isolated and transformed as described above in Example 4.<br>
10698]	To produce test quantities of each aldolase, 25 mL cultures were grown in LB<br>
medium containing ampicilBn (100 mL), to an QDsoo of approximately 0.4. The strains were induced with 100 jM of IPTG. The cells were grown 4 hours post-induction, and cellular extracts were prepared according to manufacturer's protocols (Novagen, Madison, WI, Bugbuster reagent) with benzonuclease. The soluble proteins in the cellular extracts were separated on a Bio-Rad Laboratories Experion Automated<br>
 <br>
Electrophoresis Station (Bio-Rad, Hercules, CA) and analyzed for concentration and<br>
percent expression using the Experion Software version 1.1.98.0.<br>
[0699]	The following were added per 1 mL of reaction tnixture:   approximateSy  50 jj.g<br>
aldolase (supplied in cellular extracts unless otherwise noted), 4 mM MgCb, 50 mg/mL D-tryptophan, 2 mg AT-103 (BioCatalytics, Pasadena, CA), 200 mM sodium pyruvate, 100 mM potassium phosphate buffer pH 7.5, and 0.05 mM PLP. The D-tryptophan is not soluble at this higher concentration, but was used to ensure that the reactions were kept at saturating amounts of D-tryptophan. Experiments were run in duplicate, with negative controls in which no aldolase was added. Samples were incubated 2 hrs and overnight (17-20 hours) at 30 °C with gentle shaking. Small amounts of monatin are produced overnight without aldolase (-0.5 ppm), due to non-enzymatic reactions catalyzed by magnesium and phosphate. Typical values for %R,R monatin are 50% for these samples. The negative control values were subtracted from the numbers shown below, and averaged results are shown. The only stereoisomers detected when producing monatin using these methods are R,R and S,R. The percent R,R is listed below, and was determmed by reversed-phase LC peak area. The same experiment was conducted after storage of the cell extracts and the purified polypeptide with aldolase activity of SEQ ID NO:28 for 2 months at -20 °C, this time 50 mM D-tryptophan was used as in Example 8. Twice the amount of aldole was added with the exception of the polypeptide with aldolase activity of SEQ ID NO:28, for which approximately 50 p-g was utilized again. These results are shown to the right of Table 9. FDAA derivatization restilts for isomeric distribution are shown in parentheses.<br>
 <br>
 <br>
 <br>
giutamate can also be formed during these reactions. The enzymes above were ranked for<br>
their specificity towards monatin production by comparing the peak areas of that versus<br>
the byproduct. The results were the polypeptide with aldolase activity of SEQ ID<br>
NO:122 &gt; SEQ ID NO:42 &gt; SEQ ID NO:80 &gt; SEQ ID NO:108 &gt; SEQ ID NO:96 &gt; SEQ<br>
ID N0:112 &gt; SEQ ID NO:130 &gt; SEQ ID NO:36 &gt; SEQ ID NO:94 &gt; SEQ ID NO:298 &gt;<br>
SEQ ID NO:40 &gt; SEQ ID NO:142 &gt; SEQ ID NO:54 &gt; SEQ ID NO:64 &gt; SEQ ID NO:28<br>
&gt; SEQ ID NO:62.<br>
[0701]	Based on initial experiments, the polypeptides with aldolase activity of SEQ ID<br>
NO:298, SEQ ID NO:54, and SEQ ID NO:42 looked the most promising in terms of activity level and % R,R monatin produced. These enzymes were subcloned into pET expression vectors with and without his-tags.<br>
 <br>
 <br>
digested with appropriate enzymes (Ndel and BamHi for the PCR products containing SEQ ID KO:297 and SEQ ID NO:53, A'col and BamBl for the PCR products containing SEQ ID N0:41) and gel purified (QIAquick Gel extraction Kit (Qiagen, Valencia, CA)). SEQ ID NO:297-and SEQ ID NO:53 were individually ligated into pET2§ that had. been .. digested with Ndel and BamHi and gel purified. SEQ ID N0:41 was ligated to pET30 that had been digested with Ncol and BamHl and gel purified. The ligation was transformed into TOPIO. Colonies were screened for inserts. Isolates with an insert were submitted for DNA sequence analysis (Agencourt, Beverly MA).<br>
Purification of aldolases<br>
[0703]	Confirmed aldolase clones were transformed into either BL21 DE3 or BL21 DE3<br>
pLysS. Overnight cultures grown with the appropriate antibiotic were diluted into fresh media (typically 1:100) and grown to an ODgoo -~0.6 with aeration at 37°C. Cultures were then induced with 1 mM IPTG and shifted to 30°C (with aeration) and incubation was continued overnight. Cells were harvested by centrifugation. The cell pellet was typically subjected to one freeze thaw cycle to assist with cell lysis. The cell pellet was lysed in BugBuster and Benzonase (Novagen, Madison, WI) (according to the manufacturer's protocol). Cell debris was removed by centrifugation. The crude protein extract was applied to a HisBind column (Novagen, Madison, WI) that had been prepared accordmg to the manufacturer's protocol. The column was washed and protein was eluted according to the manufacturer's protocol. The purified protein was desalted with PD-10 columns (GE Healthcare, Piscataway, NJ). The buffer used for the exchange was 50 mM<br>
 <br>
potassium phosphate pH 7.5, 100 mM NaCI, 4 mM MgCb. Purified protein was concentrated with Amicon centri&amp;gal c;oncentrators (Millipore, Billerica, MA).<br>
Testing of purified aldolases<br>
10704]	Purified aldolases were tested for their ability to produce R,R monatin from D-<br>
tryptophan. The foUowitig were added per 1 mL of reaction mixture: approximately 50 ig purified aldolase, 4 mM MgCl2, 50 mM D-tryptophan, 0.5 mg purified B. sphaericm D-aminotransferase, 200 mM sodium pyruvate, 100 mM potassium phosphate buffer pH 7.5, and 0.05 mM PLP. Samples were taken at 2 hours and overnight. Results are shown in Table 11 below.<br>
 <br>
coupling the aldolases with HexAspC broad specificity L-aminotransferase produced and purified as described in U.S. Published Application No. 20050282260 (0-5 mg of purified protein). Results are shown below in Table 12 for total monatin production (subtracting background levels without aldolase present), and percent S,S monatin is shown based on reversed phase hC peak area. Numbers above 400 ppm are outside the linear range of the standard curve, and are approximate.<br>
 <br>
 <br>
 <br>
Example 10<br>
Comparison of total monatin production and isomeric distribution for the<br>
polypeptides with aldolase activity of SEQ ID NO; 116, SEQ ID NO:76, SEQ ID<br>
NO:44, SEQ ID NO:148, SEQ ID NO:46, SEQ ID NO:134, SEQ ID NO:74, SEQ<br>
ID NO:126, SEQ ID NO;102, SEQ ID NO:58, SEQ ID NO:88, SEQ ID NO:50,<br>
SBQ 101/0:106; SEQ ID NO:304, SEQ ID NO:300, and SEQ ID NO:-2S-.-.   .<br>
[0707]	The recombinant enzyme produced in Example 7 was used in coupled reactions<br>
with HMG aldolases to produce monatin from D-tryptophan and pyruvate, as described in U.S. Published Application No. 20050282260. The polypeptide with aldolase activity of SEQ ID NO:28 was used as a benchmark in these assays and had been purified as described in Example 8.<br>
[0708]	To produce test quantities of each aldolase, 25 mL cultures were grown in LB<br>
medium containing ampicillin (100 f.tg/mL), to an ODoo of approximately 0.5. The cultures were induced with 1 mM of TPTG. The cells were shifted to 30°C and were grown overnight Cellular extracts were prepared according to manufacturer's protocols (Novagen, Madison, WI, Bugbuster reagent). Benzonuclease was also added. The soluble proteins m the cellular extracts were separated on a Bio-Rad Laboratories Experion Automated Electrophoresis Station (Bio-Rad, Hercules, CA) and analyzed for concentration and percent expression using the Experion Software version LL98.0.<br>
[0709]	The following were added per 1 mL of reaction mixture;   approximately  50 |xg<br>
aldolase (supplied in cellular extracts unless otherwise noted), 4 mM MgCh, 50 mM D-tryptophan. 0.5 mg purified B. sphaericus D-aminotransferase, 200 mM sodium pyruvate, 100 mM potassium phosphate buffer pH 7.5, and 0.05 mM PLP.  Dithiothreitoi ("DTT")<br>
 <br>
was added (final concentration 2 mJVI) to the samples noted below. Experiments were run in duplicate. Samples were incubated 2 iirs, and overnight (20 hours) at 30 °C with gentie shaking. Averaged results are shown beiow. The only stereoisomers detected when producing monatin using these methods are R.,R and S,R. The percent R,R is listed below, and was determined by reversed-phase LC peak area.<br>
 <br>
 <br>
[0710]	Total monatin production numbers ranged from I ppm to over 200 ppm and %<br>
R,R ranged from 0% to 99%. Because the aminotransferase was the same for all of the aldolases, changing the aldolase can have a significant impact on both the amount of monatin produced and the stereoisomeric distribution of the monatin produced. DTT (as in the samples below) appeared to increase the amount of total monatin produced.<br>
[0711]	The same experiments as above were done using L-tryptophan as the starting<br>
substrate and coupling the aldolases (supplied as cellular extract) with HexAspC broad specificity L-aminotransferase partial!;' purified (0.5 mg of HexAspC). Averaged results (of duplicates) are shown below in Table 14 for total monatin production (subtracting background levels without aldolase present), and percent S,S monatin is shown based on reversed phase LC peak area. Nimibers above 400 ppm are outside the linear range of the standard curve, and are approximate. Purified polypeptide with aldolase activity of SEQ ID N0;2S was used as a benchmark. Polypeptides with aldolase activity of SEQ ID NO:304 and SEQ ID NO:300 (plant derived) were used with and without 2 ml'I DTT. Shannon and Marcus (The Journal of Biological Chemistiy 237: 3342-3347, 3962) used mercaptoethanol as a reducing agent in the original purification of a peanut HMG aldolase.<br>
 <br>
 <br>
 <br>
Effect of Dithiothreitol (DTT) on monatin production<br>
[0712]	Several of the enzymes in Example 10 were chosen for further study.  The plant<br>
derived aldolases showed improvement upon the addition of DTT as a reducing agent. It was noted that the microbially-derived aldolases from environmental samples also contain a high percentage of cysteine residues. Therefore, forther experiments were conducted to see if DTT increased monatin production for non-plant aldolases as well.<br>
[0713}	The following were added per I mL of reaction mixture:   approximately 50 ng<br>
aldolase (supplied in cellular extracts unless otherwise noted), 4 mM MgCU, 50 mM D-tryptophan, 2 mg AT-103, 200 mM sodium pyruvate, 100 mM potassium phosphate buffer pH 7.5, and 0.05 mM PLP. Dithiothreitol was added (final concentration 2 mM) to the samples noted below. Experiments were run in duplicate. Samples were incubated 2 hrs at 30 °C with gentle shaking. Averaged results are shown below for total monatin as determined by LC/MS/MS, with the backgroxmd production of monatin (no aldolase control) subtracted.<br>
Table 15: Total monatin produced from D-tryptophan<br>
	Aldolase	Total monatin (ppm)<br>
 <br>
 <br>
[0714]	The no aldolase control produced 10 ppm of total monatin with and without DTT,<br>
indicating that the DTT is not affecting the overall reaction by reduction of byproducts, and is not affecting the D-aroinotransferase activity. The polypeptides with aldolase activity of SEQ ID NO:46, SEQ ID NO:58, and SEQ ID NO:28 all showed a benefit from the addition of DTT. The polypeptide with aldolase activity of SEQ ID NO:46 showed the highest benefit, approximately 1.8 fold higher activity with 2 mM DTT. Two polypeptides with aldolase activity appear to have been inhibited by DTT (SEQ ID N0:116 and SEQ ID NO;50) while no effect was noted, within experimental error, for the polypeptide with aldolase activity of SEQ ID NO;44. However, it is possible that for each aldolase utilized there is an optimal concentration of DTT in order to detect a benefit of providing the reducing agent.<br>
[0715]	The polypeptide with aldolase activity of SEQ ID NO:88 was chosen to study the<br>
effect of DTT concentration on monatin production. Coupled reactions were carried out as above. Results are plotted in Figure 15. The optimal concentration of DTT in this assay was between 2.5 and 5 mM, for the amount of aldolase added. Interestingly, if no DTT was added the amount of monatin produced was almost as high as the 2.5 mM DTT, but adding suboptima] amounts of DTT (0.5-1 mM) actually appears to be inhibitory, as well as addition of too much DTT (20 mM).<br>
Example 12<br>
Comparison of total monatin production and isomeric distribution for the<br>
polypeptides with aldolase activity of SEQ ID NO:278, SEQ ID NO:162, SEQ ID<br>
NO:276, SEQ ID NO:178, SEQ ID NO:202, SEQ ID NO:166, SEQ ID NO:218,<br>
SEQ ID NO:224, SEQ ID NO:226, SEQ ID NO:244, SEQ ID NO:250, SEQ ID<br>
 <br>
NO:252, SEQ ID NO:264, SEQ ID NO:268, SEQ ID NO:272, SEQ ID NO: 184, SEQ ID NO:282, SEQ ID NO: 186, SEQ ID NO: 192, SEQ ID NO:200, SEQ ID NO:280, SEQ ID NO:284, SEQ ID NO: 172, SEQ ID NO: 180, SBQ ID NO: 168, SEQ ID NO:228, SEQ ID NO:236, SEQ ID NO:238, SEQ ID NO:240, SEQ ID NO:270, SEQ ID NO:156, and SEQ ID NO:28<br>
[0716]	The recombinant enzyme produced in Example 7 was used in coupled reactions<br>
with HMG aldolases to produce monatin from D-tryptophan and pyruvate as described in U.S. Published Application No. 20050282260. The polypeptide with aldolase activity of SEQ ID NO:28 was used as a benchmark in these assays and had been ptirified as described in Example 8.<br>
[0717]	To produce test quantities of (;ach aldolase, 25 mL cultures were grown in LB<br>
medium containing ampicillin (100 |.Lg/mL), to an ODsoo of approximately 0.5. The cultures were induced with 1 mM of IPTG. The cells were shifted to 30°C and were grown overnight. Cellular extracts were prepared using Bugbuster reagent according to manufacturer's protocols (Novagen, Madison, WI). Benzonuclease was also added. The soluble proteins in the cellular extracts were separated on a Bio-Rad Laboratories Experion Automated Electrophoresis Station (Bio-Rad, Hercules, CA) and analyzed for concentration and percent expression using the Experion Software version 1.1.98.0.<br>
[0718]	The following were added per 1 mL of reaction mixture: approximately 200 g of<br>
the polypeptides with aldolase acti\dty of SEQ ID NO:278, SEQ ID NO:162, SEQ ID NO:276, SEQ ID NO:I78, SEQ ID NO:202, SEQ ID NO:166, SEQ ID K0:2IS, SEQ ID NO:224, SEQ ID NO:226, SEQ ID NO:244, SEQ ID NO:250, SEQ ID NO:252, SEQ ID NO:264, SEQ ID NO:268, SEQ ID NO:272, SEQ ID NO: 184, SEQ ID NO:282, SEQ ID NO:186, SEQ ID NO:192, and SEQ ID NO:200 or 50 ig of the polypeptide with aldolase activity of SEQ ID NO:280, SEQ ID NO:284, SEQ ID NO:172, SEQ ID NO:180, SEQ ID NO:168, SEQ ID NO:228, SEQ ID NO:236, SEQ ID NO:238, SEQ ID NO:240, SEQ ID NO:270, and SEQ ID NO:156 (supplied in celltilar extracts unless otherwise noted), 4 mM MgCh, 50 mM D-tryptophan, 0.5 mg purified S. sphaericus D-aminotransferase, 200 mM sodium pyruvate, 100 mM potassium phosphate buffer pH 7.5, and 0.05 mM PLP. Experiments were run in duplicate. Samples were incubated 2 hrs, and overnight (20 hours) at 30 °C wth gentle shaking. Averaged results are shown below. The only stereoisomers detected when producing monatin using these methods are R,R and S,R. The percent R,R is listed below, and was determined by reversed-phase LC peak area.<br>
 <br>
 <br>
<br>
 <br>
[0719]	Total monatin production numbers ranged from undetectable to over 600 ppm and<br>
% R,R ranged from 61% to 100%. Because the aminotransferase was the same for all of the aldolases, changing the aldolase can have a significant impact on both the amount of monatin produced and the stereoisomeric distribution of the monatin produced.<br>
[0720]	The same experiments as above were done using L-tryptophan as the starting<br>
substrate and coupling the aldolases (supplied as cellular extract) with HexAspC broad specificity L-aminotransferase produced and purified as described In U.S. Published Application No. 20050282260 (0.5 mg of purified protein). Results are shown below in Table 17 for total monatin production (subtracting background levels without aldolase present), and percent S,S monatin is shown based on reversed phase LC peak area. Numbers above 400 ppm are outside the Imear range of the standard curve, and are approximate. Table 12 shows results for the benchmark R-specific enzyme, the polypeptide with aldolase activity' of SEQ ID NO;28, which was assayed at the same time.<br>
 <br>
 <br>
Production of monatin from indole--3-pyruvate using a D-aminotransferase<br>
[0721]	AT-103   transaminase  was  part  of a transaminase   library  purebred  from<br>
BioCatalytics (Pasadena, CA) and the enzyme was tested for production of monatin in coupled reactions using the ProA aldolase from C. testosteroni. The aldolase was prepared as described in WO 03/091396 A2. AT-103 is a broad specificity D-transaminase (E.G. 2.6.1.21) from a Bacillus species that requires a D-amino acid (such as D-glutamate, D-aspartate, or D-alanme) as the amino acid donor. Enzymes and additiona] components/substrates wert; added directly to the reaction buffer provided in the kit, which contained 100 mM potassium phosphate buffer pH 7.5, 100 mM amino donor, and 0.1 mM PLP. To one niL of reaction buffer were added: 4 mg indole-3-pyruvate, 20 mg pyruvate, approximately 50 g ProA provided in a cellular extract, 1 fiL 2 M MgCb, and 2 mg of aminotransferase enzyme.    Reactions were performed in<br>
 <br>
duplicate. The reactions were incubated overnight at 30 °C with gentle shaking (100<br>
rpm). The samples were filtered arid submitted for reversed-phase LC/MS/MS analysis<br>
as described in Example 1. The resuhs indicated that approximately 370 |ig/mL monatin<br>
were produced using AT-103 enzyme. The results were further analyzed to determine<br>
ratios of S,R/R,S versus R,R/S,S monatin, on the basis of the peak areas of the two<br>
stereoisomer pools that resolve during the chromatographic separation. Of the total<br>
monatin produced by AT-1D3, 69% was R,R/S,S monatin in comparison to the mixed<br>
isomers. This enzyme is homologous to the Bacillus subtilis DAT enzyme described in<br>
WO 03/091396 A2, which is known to have a broad specificity for D-amino acids. Chiral<br>
analysis was performed using the FDAA methodology described in Example 1, which<br>
verified that the D-aminotransferase was making predominantly R,R monatin, and some<br>
S,R monatin as expected. Further transamination experiments with S,S monatin or R,R<br>
monatin and a-ketoglutarate as substrates verified that the BioCatalytics enzyme was<br>
highly selective for the D-configuration at carbon 4, as expected. In these experiments,<br>
no glutamate was detected in the reaction with S,S monatin and a-ketoglutarate as<br>
substrates.<br>
[0722]	To decrease the amount, of S,S monatin or R,S monatin produced as bj'products in<br>
coupled reactions with AT-103 (the broad range D-transaminase) and the ProA aldolase, the aldolase was purified using His-Bind cartridges, following manufacturer's protocols (Novagen, Madison, WI). The purified enzyme preferably should not contain wildtype L-aminotransferase activities that can be present in cellular extracts (such as the native E. coli AspC or TyrB activities). The His-Bind eluent was desalted to remove imidazole using PD-10 columns (G25 Sephadex, GE Healthcare, Piscataway, NJ) and was eluted in 50 mM Tris-Cl, pH 7. Experiments were carried out in duplicate in a volume of 1 mL and contained 100 mM Tris-Cl buffer, pH 7.8, 50 }ig ProA aldolase, 4 mg indole-3-pyruvate, 1 or 2 mg D-aminotransferase, 200 mM sodium pyruvate, 2 mM MgCb, 3 mM potassium phosphate, 0.1 mM PLP, and 14.7 mg of D-glutamate. The tubes were incubated at 30 "C with gentle shaking. Two-hotir time points were taken and frozen immediately at -20 "C. The pH was adjusted at two hours from 5 to between 7-8 using NaOH, and the assays were incubated overnight. Samples were filtered and analyzed for monatin as described in Example 1. The two-hour samples did not have detectable amounts of monatin, probably due to the low pH.   The overnight samples contained<br>
 <br>
approximately 190 ng/mL monatin when 1 mg of D-aminotransferase was used, and<br>
approximately 84% was R,R monatin and 16% was S,R monatin.   When 2 mg of D-<br>
aminotransferase were used, 540 ng/mL monatin was produced, approximately 71% was<br>
R,R monatin.<br>
[0723]	Similar experiments were conducted using BioCatalytics Aminotransferase buffer<br>
(BioCatalytics, Pasadena, CA), which contained 100 mM potassium phosphate pH 7.5, O.i mM PLP, and 100 mM D-g!utamate. Solid indoIe-3-pyruvate and D-aminotransferase were added as above. ProA aldolase (50 p.g), MgCij, and 50 mM pyruvate were added from stock solutions. The assays were treated as above, although no pH adjustment was required in this case. A negative control was done with just the BioCatalytics supplied enTne and buffer, which did not contain monatin. The experimental results are shown in Table 18.<br>
 <br>
buffered systems.<br>
[0725]	To compare activities of the cloned B. subtilis DAT from WO 03/091396 A2 with<br>
the BioCatalytics enzyme (AT-103) (BioCatalytics, Pasadena, CA), additional assays were done. The B. subtilis dat gene was also subcloned into pET30a to remove the His-6 tag. Untagged and tagged enzyme were produced in BL21(DE3), as described in WO 03/091396 A2. Cellular extracts were made and total protein assays were done to estimate protein concentration as described previously. Duplicate one mL reactions were done which contained: 500 )j.g D-aminotransferase, 50 \xg ProA aldolase, 100 mM potassium phosphate pH 7.5, 3 mM MgCb, 4 mg indole-3-pyravate, 200 mM sodium pyruvate, 7.35 mg (50 mM) D-glutamate, and 0.1 mM PLP. Samples were incubated at 30 "C for 1 hr, 2 hr, and overnight, and were filtered for LC/MS/MS analysis.   The<br>
 <br>
samples contained only the S,R and R,R stereoisomers of monatin, as determined by the FDAA derivitizalion protocol described in Example 1. The results are summarized in Table 19 below. The % RR was determined by peak areas that were separated by reversed phase chromatography.<br>
 <br>
subtilis D-aminotransferase; however, the BioCatalytics D-amlnotransferase homolog<br>
clearly had the highest activity.    The BioCatalytics D-aminoiransferase homolog also<br>
showed greater substrate specificity for the R-monatin precursor,   hicreased incubation<br>
times appear to reduce the enantiomeric excess of R,R monatin that is produced.<br>
[0727]	Because the Bacillus D-aminotransferase enzymes have a preference for pyruvate<br>
as an amino acceptor and D-alanine as an amino donor. It was expected that D-alanine could be utilized as the amino donor for conversion of MP to monatin with similar or . belter results. Duplicate one mL reactions were done which contained: 500 jxg D-aminotransferase, 50 g purified ProA aldolase, 100 mM potassium phosphate pH 7.5, 3 mM MgCb, 4 mg mdole-3-pyruvate, 100 mM sodixmi pyruvate, 25 mM D-glutamate or D-alanine, and 0.1 mM PLP. Samples were incubated for 2 hours, and treated asabove prior to analysis. When D-alanine wa;; used as the amino donor, slightly higher levels of monatin were produced (23 versus 21 ppm) as expected. Additionally, it is expected that high concentrations of pyruvate may inhibit the transamination step, thus dosing in smaller amounts of pyruvate over time may improve the overall rate of monatin production. One can see from the above data that even though one-half of the pyruvate was used in this case compared to the above table, significantly more monatin was<br>
 <br>
produced. AT-103 is an example of an enzyme with limited activity for S-MP. Even though the ProA aldolase used in this study makes greater than 90-95% S-MP, AT-103 makes up to 92% R,R monatin.<br>
Example 14 Production of R,R monatin from D-tryptophan<br>
[0728]	The following were added per I mL of reaction mixture:  approximately 60 g C.<br>
tesiosteroni ProA aldolase (supplied in cellular extracts, as described in WO 03/09)396 A2), 4 mM MgC]2, 50 mM D-tryptophan, 0.5 mg BioCataJytics D-aminoiransferase (AT-103) (BioCatalytics. Pasadena, CA), 100 mM sodium pyruvate, 100 mM potassium phosphate buffer pH 7.5 or 100 mM sodium acetate buffer pH 8, 0.05 mM PLP, 3 mM potassium phosphate (only to the acetate reactions), and 10 mM a-ketoglutarale. Experiments were run in duplicate, with negative controls in which no aldolase was added. Samples were incubated overnight (20 hours) at 30 "Q. with gentle shaking. The actual pH of the sodium acetate samples was approximately 5, while the final pH for the phosphate buffered samples was approximately 7. None of the aldolases appeared to have significant activity at pH 5, the s;imple containing ProA aldolase was slightly above the negative control but probably not above experimental error. In potassium phosphate, the ProA aldolase produced 73.4 ppin monatin with a ratio of R,R:S,R of 1.7:1 (-63% R,R from D-tryptophan).<br>
[0729]	Because the Bacillus D-aminotransferase enzymes have a preference for pyruvate<br>
as an amino acceptor and D-alanine as. an amino donor, it was expected that the addition of alpha-ketoglutarate is unnecessary when producing R,R or S,R monatin &amp;om D-tryptophan. The above experiment was repeated (in 100 mM potassium phosphate buffer) using pimfied ProA aldolase (50-60 |ig), and an incubation time of 2,5 hours. Duplicate experiments were run, with and without alpha-ketoglutarate. With 10 mM alpha-ketogiutarate added, 56.1 ppm monatin was formed from D-tryptophan (79.5 % R,R, 20.5% S,R). Without alpha-ketoglutarate, 102.5 ppm monatin was formed (79% R,R, 21% S,R).<br>
 <br>
Example 15 TryptopJian racemase<br>
[0730]	R,R-monatin has been produced using D-aminotransferase and an aldolase when<br>
D-tryptophan was used as the starting material (Example 14). That notwithstanding, L-tryptophan may be a preferred starting material for several reasons. For example, L-tryptophan may be less expensive and more readily available than D-tryptophan. This disclosure describes several methods for obtaining an active tryptophan racemase. Yields of R,R monatin axe improved by using an R-specific aldolase, i.e., an aldolase that preferentially or selectively produces R-MP. Figure 3 illustrates a method for producing stereoisomerically-enriched R,R monatin from L-tryptophan using a tryptophan racemase, a D-aminotransferase and an R-specific aldolase.<br>
[0731]	A selection for a tryptophan racemase is created by constructing a strain that<br>
would require an active remase for growth. A tryptophan auxotroph needs a source of L-tryptophan when grown on minimiil medium. Supplementing the medium with D-tryptophan is one way to select for a racemase that converts D-tryptophan to L-tryptophan. The tryptophan auxbtrophs were tested for growth on minimal medium supplemented with D-tryptophan. The strains, CAG18455 and CAG18579 from the Coli Genetic Slock Center and NRRL12264 (Also !ipA~, }DE3Jysogeni2ed, and cured of its plasmid), did not grow when supplemented with D-tryptophan but grew when supplemented with L-tryptophan. E. coH may be used as a host organism but other host organisms also may used, such as yeast, other bacteria, or other eukaryotic organisms. A tryptophan auxotroph (specifically NRRL12264 (also UpA, IDES lysogenized and cured of its plasmid)) will grow on D-tryptophan when it has been transformed with a D-aminotransferase. This confmns the ability of E. coli to transport D-tryptophan into the cell.<br>
[0732]	Saicher  and  Lingens  described the  presence  of a tryptophan  racemase  in<br>
Pseudomonas aurereofaciens (ATCC15926). Tryptophan racemase has also been described in several plants including tobacco, beets, tomato, and wheat and the enzyme appears to be induced by conditions of osmotic stress or drought. Tryptophan racemase may play a role in Sclerochiton ilicifolius in the native monatin production pathway. To isolate this racemase activity, an expression library is constructed from ATCC15926 (or<br>
 <br>
another organism with tryptophan racemase activity) and the Hbrary is transformed into the tryptophan auxotroph. A strain is selected that will grow using D-tryptophan as the tryptophan source. A simitar method is also used to screen many strains with known racemases to look for a racemase with activity on D-tryptophan. Examples include: alanine, serine, and glutamate racemases (T, Yoshimura and N. Esaki, "Amino Acid Racemases: Functions and Mechanisms." Joximal of Bioscience and Bioengineering, Vol. 96, No. 2, 103-109, 2003). Alanine racemase is PLP dependent and has been cloned from Salmonella typhimurium (dadB gene). Other sources are Escherichia coli. Bacillus subtilis, Pseudomonas aeruginosa. Vibrio cholerae, Schizosaccaroyces pombe, and Bacillus cereus. A basidiomycetous mushroom, Lentinus edodes, also contains a broad activity alanine racemase. Serine racemase is also PLP dependent and is found in Eukaryotes (such as silkworm, rat brain, mouse brain cDNA) as well as bacteria {Enterococcus gallinarum). Glutamate racemase is PLP-independent and has been cloned from Pediococcus pentosaceus. Bacillus pumilus, Lactobacillus fermenti, Lactobacillus brevis, E. coli, Aquifex pyrophilus, and Bacillus subtilis. The glutamate racemase is very specific and, consequently, even structurally similar amino acids aspartate, asparagine, and glutamine are not substrates for the enzyme. Aspartate racemases also exist and are PLP independent. These enzymes are found in Lactobacilli. Streptococcus strains, and some archaea such as Desulfurococcus and Thermococcus strains. The bivalve moUusk Scapharca brouhtonii also contains an aspartate racemase. Other racemases found in the literature include amino acid racemase (EC 5.1.1.10) from Anabaena sp. and Pseudomonas striata, proline racemase, multifunctional phenylalanine racemase. Related epimerases or racemases are also being tested. Potential racemases are tested to make sure they are not D-tryptophan aminotransferases. This screening is done by sequence analysis and/or an enzyme assay.<br>
[0733]	Enzymes that pass the test as a racemase are screened for activity on monatin as<br>
described in Example 17. Ideally, one obtains an enzyme that is very specific for tryptophan and has little or no racemase activity on monatin.<br>
10734]	A tryptophan racemase also may be evolved and/or improved (via mutagenesis or<br>
recombinant engmeering) from an existing racemase, transaminase, or epimerase. Additionally, because crystal striictures for alanine aminotransferases are known, these may be used as a basis for rational, structure based mutagenesis.  The process described<br>
 <br>
above is used as an initial selection for tryptophan racemase activity and as a screen for improved activity.<br>
Tryptophan racemase libraries Construction of libraries:<br>
[0735]	Burkholderia    pyrrocina    (ATCC15958)    and    Pseudomonas    chlororaphis<br>
(ATCC15926) were obtained from iht American Type Culture CoJlection. They were grown as recommended by ATCC aiid genomic DNA was prepared according to the method of Mekalanos JJ. (Duplication and amplification of toxin genes in Vibrio cholerae. Cell. 1983. 35:253-63). Tlie genomic DNA was partially digested with the SauiPA restriction enzyme. 1-3 Kbp fragments were gel purified using a Qiagen QIAquick Ge) Extraction Kit (Qiagen, Valencia, CA). The purified DNA was ligated into pTrc99a (Amersham, Piscataway, NJ) that had been digested with BarnBi and purified as above. The ligation was done at room temperature with overnight incubation using a 3:1 molar ratio of insert to vector. The ligated library was transformed into TOPIOF' chemically competent cells (Invitrogen, Carlsbad, CA) and plated on LB medium with J OOfig/mJ ampiciJIin. After ovemighi: incubation of the transformation plates, colonies were scraped off of the plates washed with liquid LB medium and an appropriate size cell pellet was mini-prepped using a Qiagen QIAquick mini-prep kit (Qiagen, Valencia, CA). Approximately 30,000 colonies were pooled and mini-prepped.<br>
[0736]	The pooled plasmid was fransformed into CAG18455 (lrpC83::TnlO, rph-l) or<br>
CAG18579 (IrpO.-.TnlOkan, rph-1). Both strains are tryptophan auxotrophs so they will not grow on M9 minimal medium (Difco) xmless the medium is supplemented with tryptophan. The transformants were plated on M9 minimal medium supplemented with D-tryptophan. This selects for a strain that can convert D-tryptophan to L-tryptophan.<br>
[07371	Prior to transformation of the library, the strains were tested for growth on<br>
minimal medium with L- or D-tryptophan, The strains were tested for growth on minimal medium supplemented with D-tryptophan and no growth was observed. Bolh strains grew on identical medium supplemented with L-tryptophan instead of D-tryptophan. Additionally, a derivative of NRRL12264 (the strain used had been cured of the tryptophan operon plasmid, lysogenized with XDE3, and deleted for lipA, in addition to the other chromosomally encoded mutations (serB, AtrpED tnaA2, aroP) (this strain<br>
 <br>
could not grow on minimal mediimi supplemented with D-tryptophan but grew on identical medium supplemented with L-tryptophan instead of D-tryptophan) was transformed with a D specific amini&gt;transferase from Bacillus subtilis (WO 03/091396). Expression of the D-aminotransferase was driven by the T7 promoter. The transfoiroed strain was able to grow on M9 minimal, medium supplemented with D-tryptophan.<br>
(073SJ	The colonies that grow on the D-tryptophan medium are screened. The plasmid is<br>
isolated and retransformed into the parent strain (CAG18455 or CAG18579) to confirm that growth on D-tryptophan medium is dependent on the plasmid and not on a host mutation. The nucleotide sequences of the plasmids that complement the tryptophan auxotrophy are analyzed. Clones tliat are determined to contain a tryptophan racemase gene are further analyzed.<br>
[0739]	The tryptophan racemase from other tissue sources is isolated in a similar fashion.<br>
There are literature reports of tryptophan racemase activity in both tobacco tissue culture cells (Nicotiana tabacum L. var. Wisconsin 38) (Miura, G.A. and Mills, S.E. The conversion of D-tryptophan to L-tryptophan in cell cultures of tobacco. Plant Physiol. 1971. 47:483-487) and in crude protein extracts of wheat (Triiicum aestivum) {Rekoslavskaya, N.I., Yur'eve, O.V., Shibanova, L.A., and Salyaev, R.K. Synthesis and physiological fimction of D-tryptophan during wheat germination. Russian S. Plant Physiol. 1997. 44:196-203). A cDNA expression library is made from tissue, as described in the literature, and the expression library is used to transform a tryptophan atixotroph as described above.<br>
Tryptophan racemase assay<br>
[0740]	Clones that are identified as potentially having a tryptophan racemase are<br>
transformed into a strain of E. coli commonly used for expression of recombinant proteins, such as BL21. The cells are grown in LB broth to an optical density at 600 nm of 0.4-0.6. The promoter driving expression of the racemase is induced with IPTG (0.1 mM final concentration). After induction, the cells are allowed to express the protein for 1-3 hours at 37° C (with aeration). The cells are harvested and lysed by French press, sonication, or by chemical means (such as BugBuster (Novagen, Madison, WI)). The lysed cells are centrifuged to remove the cell debris. The clarified extract is used directly in assays.<br>
 <br>
[0741]	Varying amounts of extract is added to-a solution such that the final concentration<br>
is 50 mM potassium phosphate (pH 7.0) and 2 mM L-tryptophan. Pyridoxai-5'-phosphate is added al a final concentration of 10 fiM. The samples are incubated and then anaiyzed by LC/MS. The presence of a D-tryptophan peak when only L-tryptophan is used as a substrate indicates a positive result. D-tryptophan concentration should increase with increasing time until equilibrium is reached, and the rate should also increase with protein concentration tinti] the concentration of enzyme is high enough that it is no longer saturated with substrate. D-tryptophan may also be converted to L-tryptophan as above.<br>
[0742]	A complementing gene may code for a D-aminotransferase,   (A "complementing<br>
gene" is a gene that, when expressed, nullifies a mutation in an organism. For example, if an organism has a null mutation in one of the genes required for synthesis of tryptophan by the cell, a complementing gene could be one that, when expressed, allows the strain to grow on minimal medium (i.e., without tryptophan). This reaction requires an alpha-keto acid such as a-k.etog!utarate, oxaloacetate, or pyruvate as an amino acceptor. These compounds will likely be present in a cell extract (in small amounts). These compounds may be removed using a PD-10 desalting column (GE Healthcare, Piscataway, NJ) and the assay may still be performed in crude extract. The tryptophan racemase activity is purified using conventional column chromatography. Finally, the open reading frame identified as a potential tryptophan racemase is cloned into an expression vector with an affinity tag. The potential tryptophan racemase is then purified by affinity chromatography. In either case the purified protein is used in enzyme assays essentially as described above.<br>
Reverse genetic engineering of tryptophan racemase<br>
[0743]	The tryptophan racemase may be purified from either plant or microbial sources<br>
by conventional protein purification techniques including ammonium sulfate fractionation, and conventional column chromatography. Once the protein has been purified such that a spot can be isolated on a 2-D gel, peptide microsequencing techniques or conventional Edman type amino acid sequencing are utilized {see golgi.harvard.edu/microchem/ for descriptions of the protocols and equipment used for this type of work), hi some cases, however, the genome sequence of the organism cannot be used as a source of the protein for the orotein purification because such sequence has<br>
 <br>
not been determined yet. In that situation, the first set of degenerate primers may be designed based on sequence available from the closest known relative of the protein source. Degenerate PCR and genome walking is then be performed according to established protocols to isolate the tryptophan racemase coding sequence.<br>
Tryptophan racemase monatin production<br>
[0744]	The following is added per 1 mL of reaction mixture:   approximately 60 jig C.<br>
testosteroni ProA aldolase (supplied in cellular extracts, as described in WO 03/091396 A2), ] 00 \iVmL of tryptophan racemase cellular extract or 1 mg/mL purified tryptophan racemase, 4 mM MgCla, 50 mM L-tryptophai, 0.5 mg BioCatalytics D-aminotransferase (AT-103) (BioCatalytics, Pasadena, CA), 100 mM sodium pyruvate, 100 mM potassium phosphate buffer pH 7.5, 0.05 mM PLP, and 10 mM a-ketoglutarate. Because pyruvate is an acceptable amino acceptor for the broad specificity D-aminotransferase, a-ketoglutarate is optional. Experiments are run in duplicate, with negative controls in which no aldolase was added or no tryptophan racemase was added. Samples are incubated - 1 hour or overnight (20 hours) at 30 C with gentle shaking.<br>
[0745]	The tryptophan racemase is tested for activity on monatin.   An assay similar to<br>
that in Example 17 is used with monatin as a substrate, and compared to the activity of the enzyme on tryptophan. The ideal enzyme has activity on tryptophan but little or no activity on other amino acids particularly glutamate and monatin. If the enzyme has significant activity on monatin, the enzyme may be mutagenized to decrease the activity on monatin and or glutamate while keeping the tryptophan activity unchanged or at a level high enough for the enzyme to be useful in monatin production. Techniques that may be used for mutagenesis include, but are not limited to, error prone PCR, site-directed mutagenesis, modeling to identify site-directed mutagenesis targets (sites that may be involved in substrate binding), passage through mutagenic strains, and DNA shuffling.<br>
[0746]	Mutagenized racemases may be screened for tryptophan activity using a plate<br>
assay as described above. Clones that retain tryptophan activity are then screened for a loss of activity on monatin.<br>
 <br>
Example 16 Site Directed Mutagenesis of HEXAspC<br>
Experimental Overview<br>
[0747]	A hexamutant of E. coli AspC (HEXaspQ was found to have better activity as<br>
compared to AspC for the production of S,S monatin as described in Example 6 of WO 03/091396 A2. HEX (accession numberi/AHFA gi:127190) contains the following mutations from AspC {E. coli numbering): V35L, K37Y, T43I, N64L, T104S, and N285S. Based on structural analysis and literature reports (S. Rothman and J. Kirsch, J. Mol. Biol. (2003), 327, 593-608; S. Rothman et al.. Protein Science (2004), 13: 763-772), 5 more mutants were created that were expected to increase the kinetic activity toward substrates utilized in the monatin production pathway; L-tryptophan, S-MP, or both. Two of the mutants increased transamination rates for both tryptophan and S,S monatin. Two of the mutants showed an increased stereoselectivi for the formation of SS monatin while one w less stereoselective. Based on this, it is expected that the broad specificity D-aminotransferase from Bacillus sp. having similar mutations is useful as the D-aminotransferase in the R,R monatin pathways shown in Figure 3, and described in Example 15. One of the mutants (HEXaspCP9T/R122G) had increased activity for L-tryptophan transamination, but activity in S,S monatin production or S,S monatin transamination was decreased significantly. Thus, it is expected that this enzyme is usefiif in the first step of the R,R monatin production pathway's shown in Figures I, 2, 4, 5, 6, 7, and 8 and described in Examples 18 and 19. In general, an aminotransferase that has activity similar to that of AspC on L-tryptophan and limited activity on R-MP and S-MP would be useful for the processes depicted in Figures. 1, 2, 4, 5, 6, 7, and 8.<br>
Methods and Materials<br>
10748]	The HEX gene cloned m pUC19 was provided by Professor J. F. Kirsch<br>
(Department of Molecular and Cell Biology, University of California, Berkeley, Berkeley, CA 94720-3206) and used as the template for the cloning of the gene into pET23a. See James J. Onuffer and Jack F. Kirsch, Redesign of the substrate specificity of Escherichia coli aspartate aminotransferase to that of Escherichia coli tyrosine aminotransferase by homology modeling and site-directed mutagenesis, Protein Science,<br>
 <br>
4:   1750-1757 (1995).   See also NCBI accession number 1AHF_A GI:1127190 (HEX<br>
amino acid sequence).   The following primers were designed for cloning the HEX gene<br>
into the pET23a vector (Novagen, Madison, WI):<br>
HEXaspC primers:<br>
N term: 5'-GCGGAACATATGTTTGAGAACATTACCGCC-3'(SEQ ID NO:393);<br>
Cterm: 5'-ATAACCGGATCCTTACAGCACTGCCACAATCG-3' (SEQ IDNO:394)<br>
[0749]	The foiiowing PCR protocol was used for gene amplification:    In a iOO |iL<br>
reaction, 50 ng DNA template, 1.0 M-M of each primer, 0.2 mM each dNTP, 1 U Pfu Turbo Polymerase (Stratagene, La JoUa, CA), and IX Cloned Pfu buffer were added. The themiocycler program utilized a hot stiut of 94°C for 5 minutes; followed by 25 cycles of a denaturing step at 94°C (30 sec), an annealing step at 55°C (1 min), and an extension step at 12°C. (2 min) and fmally a firdshing step at 72°C (7 min). Standard molecular biology techniques were utilized to clone the PCR product into pET23a using Ndel and Sam-ffl restriction sites.<br>
[0750]	The tryptophan residue at position 130 is thought to be important for stacking<br>
interactions with the pyridoxyl ring, but also appears to be a source of steric hindrance with the S-monatin precursor (MP) substrate, based on protein modeling observations. Therefore, an amino acid with a smaller hydrophobic side chain (phenylalanine) was used to replace the tryptophan. The rest of the mutations were based on kinetics data in literature, although new combinations of desuble mutations were created. All mutations to HEXaspC, with the exception of W130F, were made using the Stratagene Multi-Change kit (Stratagene, La Jolla, CA) by following the manufacturer's instructions. The WnOF mutation was made using the Stratagene QuikChange kit (Stratagene, La Jolla, CA) according to the manufacturer's instructions with the only exception being that the extension temperature for the PCR reaction was decreased to 66 °C. The primers for the multi-change kit were designed using the QuikChange multi-kit primer design tool on <www.stratagene.com>, except for the W130F single mutation primers.<br>
 <br>
 <br>
Expression of HEXaspC Mutant Genes and Analysis of EnCTme Activity<br>
[07521	Liquid cultures (5 mL) of Novagen Overnight Express   Autoinduction System 2<br>
(Catalog # 71366-3; solutions 1-6) (Novagen, Madison, WI) were inoculated from fresh<br>
plates or frozen glycerol stocks of the following strains:<br>
E. co//BL21(DE3)::HEXaspCpET23a<br>
E. coli BL21(DE3);:HEXaspCW130FpET23a<br>
E. coli BL2KDE3)::HEXaspCT156ApET23a<br>
E. coli BL21(DE3)::HEXaspCP9Tn:i56ApET23a<br>
E. coli BL21(DE3)::HEXaspCP9T/R122GpET23a<br>
E. coliBL21(DE3)::HEXaspCR122G/n56ApET23a.<br>
[0753]	The cultures were incubated at 37°C at 230 rpm for 6 - 8 h.  The ODeoo of each<br>
culture was determined and the volume of culture necessary to obtain an OD600 of 0.03-0.05 in 25 mL was calculated. The calculated volumes of each liquid culture were transferred to flasks containing 25 mL of the same medium. The Overnight Express' Autoinduction System 2 is a complete, chemically defined medium for high-level expression with IPTG-inducible expression systems that uses lactose as the inducing agent and does not require monitoring of cell growth. The Overnight Express cultures were incubated at 30°C with shaking at 230 rpm for 18 h. The cells were harvested by centrifugation and washed once with cold 50 mM MOPS, pH 7.0. The cells were then lysed using Bugbuster™ (primary amine free) Exttaction Reagent (Novagen Catalog #70923-3) (Novagen, Madison, WI) containing 1 iL/mL benzonase nuclease (Novagen Catalog #70746-3) (Novagen, Madison, WI), 5 nL/mL Protease Inhibitor Cocktail Set II<br>
 <br>
(Novagen Catalog #539132) (Novagen, Madison, WI) and 0.33 jiL/IO mL r-Lysozyme (Novagen Catalog #71110-3) (Novagen, Madison, WI) following the Novagen recommended protocol. After incubation at 25°C for 15 minutes with gentle shaking, the ceil debris from each suspension was pelleted by centrifugation at 21,000g for 15 minutes at 4°C. The supernatant was carefully decanted and analyzed as the cell free extract. Inclusion body fractions were isolated by suspending the cell debris fractions in 30% Bugbuster™ (primary amine free) Extraction Reagent, centrifriging at 21,000xg for 10 min; suspending the centriftiged pellets m 10% Bugbuster™ (primary amine free) Extraction Reagent, centrifiaging again to isolate the washed pellets. The eel! free extracts and inclusion body fractions were analyzed for protein expression by SDS-PAGE on 4-15% gradient gels (BioRad # 161-1104, Hercules, CA). For the cell extract samples, twenty micrograms of soluble protein were loaded in each gel lane (premixed with IX protein loading buffer and heated at 95 C for 5 min). The inclusion body fractions were dissolved in IX protein loading buffer (0.2 mL). heated for 10 minutes at 95 C and 5 fjL of each solution was loaded per gel lane. The amount of each HEX mutant in comparison to the total soluble protein loaded into each lane was calculated by band intensity analysis using Labworks Biohnaging ID-gel tool (UVP, Inc. Upland, CA), and is reported below:<br>
 <br>
protein   that   was   found   in   substantial   quantities   as   inclusion   bodies.      The<br>
HEXaspCP9T/T156A protein gave the highest level of expression, approximately 90%<br>
better than HEXaspC protein.  In contrast, the W130F, T156A and P9T/R122G proteins<br>
were expressed in lower concenfrationa than HEXaspC.<br>
[0755]	The activity of the HEXaspC mutant proteins for the production of S,S-monatin<br>
was measured using the following reaction conditions:  Each 1 mL reaction contained 50<br>
 <br>
mM TAPS, pH 8.2, 4 mM MgCh, 3 mM sodium phosphate, pH 8.0, 200 mM sodium pyruvate (pH adjusted to 8). 5 mM a-lietog!utarate (pH adjusted to 8), 50 mM tryptophan, 0.05 mM pyridoxal 3-phosphate, 50 pg/mL ProA aldolase (added as a cell free extract) and varying concentrations (approximately 50 and 500 jxg/mL) of aminotransferase (added as a cell free extract). Deaerated water was used to prepare the stock solutions and to adjust the volume of the reaction mixtures to 1.0 mL. The pyridoxal phosphate was added just prior to the addition of the enzymes. The reaction tubes were incubated at sec with gentle shaking for 4 h. Samples (0.01 mL) were withdravm at 1, 2, and 4 h after the addition of the enzymes, filtered, and analyzed by LC/MSMS, as described in Example 1. Monatin production was normalized based on the amount of aminotransferase present in the reactions. Under the conditions of these assays, the HEXaspC and the HEXaspCT156A produced the most total monatin per mg of aminotransferase while the. P9T/R122G protein produced the least, followed by HEXaspCW13QF. The HEXaspCW130F and P9T/R122G enzymes showed the greatest stereoselectivity for S-MP (greater than 98% S,S-monatin), even when high enzyme concentrations were used (greater than 300 )ig/mL). The percentage of S,S-monatin product decreased to less than 90% in the enzymatic reactions containing the P9T/T156A enzyme at high concentration. The other mutant showed a product stereoselectivity very similar to the original HEXaspC mutant (approximately 95%) S,S-monatin). Analysis of the product of the reaction containing the HEXaspC enzyme using the FDAA derivitazation reagent described in Example I showed that the second stereoisomer formed is R,S-monatin.<br>
Assaying of tryptophan and monatin aminotransferase activity<br>
[07561	The mutants were tested for transamination activity using S,S monatin and L-<br>
ttyptophan as substrates. The ammotransferase activi was measured by following the formation of the co-product of the reaction, glutamate, by HPLC with OPA-post-column derivitization as described in Example 1. The reaction mixture contained, in 1.0 mL, 100 mM HEPPS buffer, pH 8.0, 20 mM alpha-ketoglutarate, 0.08 mM pyridoxal phosphate, 25 mM tryptophan or S,S monatm, and enzyme (supplied as 2.5 mg of in cellular extracts protein). All components except the enzyme were mixed together, the enzyme was added to start the reaction and the reaction solution was incubated at 30°C (gentle shaking) for<br>
 <br>
90 minutes. Reactions were done in duplicate, with negative controls in which no<br>
enzyme was added. The reaction was stopped by the addition of 10% formic acid (final<br>
concentration), the mixture was centrifuged at 21,000 rpm, and the supernatant was<br>
carefully removed and filtered. Thi; data were corrected for background levels of<br>
glutamate and for the dilution from the addition of acid to precipitate the proteins, then<br>
normalized by amount of mutant aminotransferase added. When tryptophan was utilized<br>
as a substrate, HEXaspC produced 13.0 mM glutamate per mg of aminotransferase per<br>
hour. The relative activity, expressed as a percentage, of the mutants is as follows:<br>
HEXaspCWi30F (156%), HEXaspCT156A (151%), HEXaspCP9T/T156A (63.7%),<br>
HEXaspCP9T/R122G (116%), and HEXaspCR122G/T156A (107%). When S,S monatin<br>
was utilized as a substrate, HEXaspC produced 7.43 mM glutamate per mg of<br>
aminotransferase per hour. The relative activity, expressed as a percentage, of the<br>
mutants is as follows: HEXaspCW130F (113%), HEXaspCT156A (87.7%),<br>
HEXaspCP9T/TI 56A	(67.3%),	HEXaspCP9T/R122G	(11.2%),	and<br>
HEXaspCR122G/T156A(114%).<br>
[0757]	The HEXaspCP9T/Rl 22G mutant had increased activity for tryptophan to indole-<br>
3-pyTuvate conversion, but decreased activity for S,S monatin transamination. The ratio of tryptophan to monatin activity was 18.2 in comparison to 1.75 for HEXaspC, making it a desirable candidate for production of R,R monatin using pathways that require an L-aminotransferase such as those described in Examples 18 and 19. As such, the HEXaspCP9T/R122G is an example of an aminotransferase with limited activity on S,S monatin (and MP).<br>
[0758]	Most of tiie mutations improved L-tryptophan activi, but only two mutants<br>
increase activity toward both L-tryptophan and S,S monatin (HEXaspCW130F and HEXaspCR122G/T156A). Because 25 mM of substrate was used in these assays, the enTnes were most likely saturated and the activi is a reflection of the kcai of the enzymes. However, under the conditions in which the assays for S,S monatin production were performed (above) it is unlikely that the concentration of S-MP is sufficient to saturate the enzyme, thus the increase in kcat is not reflected. It has been reported, for similar substrates, that some of the mutations made mcrease the kcat but also increase the apparent K for the amino acid substrate. If increasing concentrations of substrates were used, it is expected that these two mutants would provide a benefit in production rates of<br>
 <br>
S,S monatin in comparison to HEXaspC. The HEXaspCT156A mutation appears to have increased tryptophan transamination rates without having a significant effect on MP transamination rate under the conditions above for S5S monatin production.<br>
[0759]	By comparison of the structures of HEXaspC and one of the Bacillus sp. D-<br>
aminotransferase enzymes (see, for example, S. Sugio, G.A. Petsko, J.M. Manning, K. Soda, and D. Ringe, Biochemistry 34 (1995) pp. 9661-9669), the W130F, R122G, T156A, aid HEX mutations of AspC could be mapped to corresponding residues in the D-aminotransferase structure. It is expected that similar mutations in the broad specificity D-aminotransferase would improve the overall production of R,R monatin as described in Example 14. For example, the functionality provided by tryptophan 130 in AspC is replaced in Bacillus D-aminotransferases by hydrogen bonding between the side chains of serines ] 79-181 and glutamate 166 (YM-1 numbering scheme). To lessen steric hindrance, the glutamate could be mutated to an aspartate residue. Some D-aminotransferases have a threonine residue at position 179, which would increase steric hindrance and should be avoided. The B. sphearicus enzyme has an alanine in place of serine at position 181, which may also reduce steric hindrance.<br>
[0760]	Additional information from studies of aspartate aminotransferase can be applied<br>
to the D-aminotransferase as well. While the AspC enzyme has an arginine in the active site that interts with the side chain of dicarboxylate substrates, the D-aminotransferase has a loop from Ser240 to Ser243. The side chains of Ser240, Thr242, and Ser243 face the same direction and form a pocket with the hydroxyl group of Serl 80 which provides a surface for both nonpolar and polar substrates can interact. SerlSO is involved in PLP binding; however, to improve the activity of a D-aminotransferase with R-MP, one can modify the Ser240, Thr242, or Ser243 residues to accept larger substrates or to favor negatively charged substrates. For instance, Thr242 can be mutated to Ser to reduce the side chain length. One of the residues can be mutated to lysine or arginine, such as Ser243. The residues (YM-1 numbering) Val30-Val36 are located in a beta strand across the active site of the D-aminotransferase, and are also important for activity. Tyr31, Val33, Glu32, and Lys35 are thought to face the active site. Tyr31, Glu32, and Val33 are invariant in all the Bacillus homologs. Ro et al. (FEBS Lett 398 (1996) pp, 141-145) mutagenized Val33 to Ala and found a slightly increased catalytic efficiency for alpha-ketoglutarate transamination and a significantly improved catalytic efficiency for bulkier<br>
 <br>
substrates (less steric hindrance). In s;ome homologs Lys35 is replaced with Arg, but if steric hindrance is a concern the Lys residue is preferable. Valines 34 and 36 are also preferable over conservative substitutions such as isoleucine, again due to less steric hindrance for large molecules such as MP.<br>
Example 17 Cloning, Expression, and Testing of Glutamate and Aspartate Racemases<br>
[0761]	This example describes methods used to clone and test amino acid racemase<br>
enzymes, which can be used to interconvert between L-glutamate and D-glutamate (or L-and D-aspartate or L- and D-alanine). Giutamate, aspartate, or alanine racemases are useful in a biosynthetic pathway to produce R,R monatin when a step in that pathway produces an L-amino acid (such as L-glutamate, L-aspartate, or L-alanine) and another step in the pathway consumes a D-amino acid (such as D-glutamate, D-aspartate, or D-alanine). Figure 4 illustrates a biosynthetic pathway for producing R,R monatin from L-tryptophan using an L-tryptophan-specific aminotransferase, an R-specific aldolase, a D-aminotransferase and a glutamate (or aspartate or alanine) racemase.<br>
[0762]	Genes were cloned into the pET28 and pET30 vector to generate both non-tagged<br>
proteins and fusion proteins with cleavable N-terminal HIS5-Tag/T7-Tags. The resulting proteins were purified using immobilized metal affinity chromatography.<br>
Experimental Overview<br>
[0763]	Genes encoding glutamate racemases (EC 5.1.1.3) from Lactobacillus brevis<br>
(Genbank Accession No. D29627, nucleic acid sequence), and Pediococcus pentosaceus (fflwr/gene) (Genbank Accession No. L22789) were cloned and expressed in E. coU. The extracts were tested for activity in conversion of L-glutamate to D-glutamate and D-glutamate to L-glutamate. BioCatalytics aspartate racemase enzyme (EC 5.LI. 13) (BioCatalytics, Pasadena, CA) was also tested for inierconversion between L- and D-aspartate.<br>
Isolation of Genomic DNA for Cloniag<br>
[0764]	L brevis genomic DNA (ATCC 8287D) was obtained from the American Type<br>
Culture Collection.  P.  pentosaceus  (ATCC 25745) was grown at 37°C in lactobacilli<br>
 <br>
MRS broth and 2 mJ was used for genomic DNA isolation using the method of Mekaianos JJ. Duplication and amplification of toxin genes in Vibrio cholerae. Cell. 1983. 35:253-63.<br>
Polymerase Chain Reaction Protocol<br>
(0765]	Primers were designed with 5' restriction sites and overhangs for cloning into the<br>
pET 28 and pET30 vectors (Novagen, Madison, WI). L. brevis glutamate racemase primers:<br>
N term: 5'-GCGGCGCCATGGAAAATGATCCGATTGGTCTAATG -3' (SEQ ID NO;401), and<br>
C term: 5'-GCGGCGGTCGACGCAATTACAATTGTGTTTGTC-3' (SEQ ID NO:402). P. penlosaceus glutamate racemase primers:<br>
N term: 5'- GCGGCGCCATGGATGTATGTATAATTTTATTTAG -3' (SEQ ID NO;403), and<br>
C term: 5'-GCGGCGGTCGACAAATTTCATTATTCATTCTAATTT -3' (SEQ ID NO:404}.<br>
[0766]	The gene derived from L. brevis was amplified using the following PCR protocol.<br>
In a 50 iL reaction 0.150 p,g template, 1.6 (JM of each primer, 0.4 mM each dNTP, 2.8 U Expand High Fidelity™ Polymerase (Roche, Indianapolis, IN), 0.5 U Pfu polymerase (Stratagene, La JoUa, CA) and IX Expand* buffer with Mg were used. The thermocycler program used included a hot start at 96°C for 3 minutes, 8 repetitions of the following steps: 94°C for 30 seconds, 52'C for 45 seconds, and 72°C for 2 minutes, followed by 22 repetitions of the following steps: 94'C for 30 seconds, 60°C for 45 seconds, and 72°C for 2 minutes. After the 22 repetitions the sample was maintained at 72°C for 7 minutes and then stored at 4°C. This PCR protocol produced a product of ~830 bp, as judged by comparison to DNA size markers.<br>
[0767]	The gene derived from P. pentosaceus was amplified using the foDowing PCR<br>
protocol. In a 50 ij:. reaction, 0.15 |.ig template, 1.6 M of each primer, 0.4 mM each dNTP, 2.8 U Expand High Fidelity™ Polymerase, 0.5 U Pfii polymerase and IX Expand' buffer with Mg were added. The thermocycler program used included a hot start at 96"C for 3 minutes, followed by 8 repetitions of the following steps: 94°C for 30 seconds, 37°C for 45 seconds, and 72°C for 2 mmutes, followed by 8 repetitions of the following steps: 94°C for 30 seconds, 45°C for 45 seconds, and 72°C for 2 minutes,<br>
 <br>
followed by 14 repetitions of the following steps: 94°C for 30 seconds, 55°C for 45 seconds, and 72°C for 2 minutes. After the 14 repetitions the sample was maintained at 72°C for 7 minutes and then stored at 4°C. This PCR protocol produced a product of -840 bp, as judged by comparison to DNA size markers.<br>
Cloning<br>
[0768]	The PCR products were gel purified from 0.8% TAE-agarose gels iing the<br>
Qiagen gel extraction kit (Qiagen, Vfilencia, CA). The PCR products were quantified using a SmartSpec 3000 spectrophotometer. The products were digested with restriction enzymes Ncol and Sail following the manufacturer's recommended protocols (New England Biolabs, Beverly, MA) and gel purified from 0.8% TAE-agarose gels using the Qiagen gel extraction kit (Qiagen, Valencia, CA). Vectors pET28 and pET30 were prepared by digestion with restric;tion enzymes Ncol and Sail followed by treatment with shrimp alkaline phosphatase and purification from 0.8% TAE-agarose gels using the Qiagen gel extraction kit (Qiagen, Valencia, CA).<br>
[0769]	The digested vectors and insierts were ligated using the Rapid ' DNA Ligation<br>
Kit (Roche, Indianapolis, IN). Approximately 50 ng of treated insert, 100 ng of freated vector (3 to 1 molar ratio o£ insert to vector), 5 U of T4 DNA ligase, and IX ligation buffer were incubated for 5 minutes at room temperature. The ligation reactions were purified using the High Pure PCR Product Purification Kit (Roche, Indianapolis, IN) and used to transform E. coli DHJOB eJectrocompetent cells (Invitrogen, Carlsbad, CA). Ten yX of each ligation reaction was added to 40 [il of DHIOB cells and were fransformed by electroporation using the Bio-Rad Gene Pulser II (Bio-Rad, Hercules, CA) under the following conditions: 2.5 kV, 25 \jF, 200 ohm in a Q.2 cm cuvette. The cells were allowed to recover in 1 mL of room temperature SOC for 1 hour at 37X with shaking at 225 rpm. Cells were plated on LB plates containmg kanamycin (50 fig/mL).<br>
[07701	Plasmid DNA was purified fi'om the resulting transformants using the Qiagen spin<br>
miniprep kit (Qiagen, Valencia, CA) and screened for the correct inserts by restriction digest with A'col and SaK. The sequences of plasmids appearing to have the correct insert were verified by dideoxy chain termination DNA sequencing.<br>
 <br>
Gene Expression and Assays<br>
[0771]	Plasmid  DNA,  verified  by  ;iequence  analysis,  was  subcloned  into  E.   coli<br>
expression host BL2](DE3) (Novageri, Madison, WI). The cultures were grown. The plasmids were isolated using a Qiagen miniprep kit (Qiagen, Valencia, CA) and analyzed by restriction digest to confirm identity.<br>
(0772]	Induction   in   BL21(DE3)   was  initially  performed   with  X.   brevis  and  P.<br>
penlosaceus glutamate racemases in both pET28 (untagged) and pET 30 (histidine-tagged) vectors. A time course study was performed with cultures grown in 250 mL LB containing kanamycm (50 mg/L) to an ODeoo of 0.5 - 0.6, induced with 100 mM IPTG (isopropy! thiogalacatoside) and sampled at 0 and 3 hoxu-s post induction. Cells from 600 iL (0 hour) and 275 p,L (3 hour) were resuspended in 40 }iL sodium dodecyl sulfate buffer containing 2-mercaptoethanol. heated at 95°C for 10 minutes, and cooled. Aliquots of these total cellular protem samples were analyzed by SDS-PAGE using a 4-15% gradient gel.<br>
[0773]	Cell extracts were also prepared from the 3 hour cultures by suspending cell<br>
pellets from 5 mL of culture in 0.625 mL Novagen BugBuster reagent (Novagen, Madison, 'Wl) containing 0.625 nL benzonase nuclease and 3 |xL protease inhibitor cocktail set #3 (Calbiochem-Novabiochem Corp., San Diego, CA) at room temperature for 20 minutes with gentle shaking, and centrifuging at 16,000 x g to remove cell debris. TTie supematants (cell extracts) were loaded onto 4-15% gradient gels for analysis of the cellular soluble proteins.<br>
[0774]	The 3-hour samples from cloned L brevis glutamate racemase and P. pentosaceus<br>
glutamate racemase showed both total and soluble protein that corresponded to the correct size (approximately 31 kDa). The L. brevis pET30 (histidine-tagged) gene product was over-expressed at a higher level than, and was also more soluble (&gt;20% of soluble protein) than the Z. brevis pET 28 (untagged) gene product, as well as the P. pentosaceus gene products in both vectors. The P. pentosaceus gene product showed equal overexpression and solubility in the pET28 and pET30 vectors, which was significantly less than that observed for the l. brevis pET30 gene product.<br>
[0775]	Cells from the induced cultures (250 mL) were centrifuged and washed once with<br>
0.85 % NaCl. Cell pellets were resuspended in 5 mL/g wet cell weight of BugBuster™ (Novagen, Madison, WI) reagent containing 5 fiL/mL protease inhibitor cocktail set #3<br>
 <br>
(Calbiochem-Novabiochem Corp., San Diego, CA) and 1 |iL/mL benzonase nuclease. Samples were incubated ar room temperature for 20 minutes on an orbital shaker. Insoluble cell debris was removed by centrifugation at 16,000 X g for 20 minutes at 4'C.<br>
f0776J	Cel! extracts were assayed for glutamate racemase activity using the following<br>
protocol. 400-L reactions were carried out in 10 mM potassium phosphate (pH 8.0), 0.2 mM DTT, and 10 mM L-glutamate or D-glutamate. The reactions were initiated by the addition of 20-100 \xL of cell ftee extracts and were incubated at room temperature. Sample aliquots were taken over a time course of 1 minute, 5 minutes, 10 minutes, 20 minutes and 1 hour (zero minute samples served as control reactions). 2 M formic acid (25 nL) was added to each 40-fiL sample aliquot to stop the reaction and the precipitated protein was removed by centrifugation. Supematants were removed and frozen at -80°C until they were analyzed by LC/MS/lv4S.<br>
[0777]	Assay results from cell extracts from pET30 induction with 100 mM IPTG (3<br>
hours) demonstrate that L. brevis (Genbank Accession No. BAA06106.1 GI:468450) and P. pentosaceus (Genbank Accession No. AAA16761.1 GI:349029) enzymes have significant levels of racemase activity on both glutamate isomers. The P. pentosaceus racemase (20 \iL of cellular extracts) reached equilibrium between L-and D-glutamate in 10-20 minutes starting with either substrate. The L. brevis enzyme (20 pL of cellular extracts) also reached equilibrium in approximately 20 minutes.<br>
|0778]	A partially purified aspartate racemase enzyme (catalog # ASPR-101) purchased<br>
from BioCatalytics, hic. (Pasadena, CA) was assayed for activity on L-aspartate and D-aspartate using a protocol similar to the one above. The commercial enzyme showed racemase activity on both isomers. Using 0.5-1 mg of enzyme, equilibrium was achieved in 20-60 minutes.<br>
(0779]	All three racemases (I. brevis glutamate racemase, P. pentosaceus glutamate<br>
racemase and BioCatalytics aspartate racemase) were also assayed for activity on S,S monatin using the following protocol. 400-jxL reactions were carried out in 10 mM potassium phosphate (pH 8.0), 0.2 mM DTT, and 10 mM S,S monatm. The reactions were initiated by the addition of cell free extracts (I. brevis and P. pentosaceus) or purified enzyme (BioCatalytics aspartate racemase) and were incubated at room temperature. Sample aliquots were taken over a time course of 1 minute, 5 minutes, 10 minutes, 20 minutes and 1 hour (zero minute samples served as control reactions as well<br>
 <br>
as samples without enzyme). 2 M formic acid (25 iiL) was added to each 40-fiL sample aliquot to stop the reaction and the precipitated protein was removed by centriftigation. Supematants were removed and frozen at -80°C imli] they were analyzed by LC/MS/MS (Example 1). No decrease in S.S monatin concentration was noted over time, nor was ere any increase in S,R monatin (present initially as 
Example 18<br>
Production of R,R Monatin from L-tryptophan Using Glutamate or Aspartate<br>
Racemases<br>
[0780]	This example describes methods of producing stereoisomerically-enriched R,R<br>
monatin from L-tryptophan using an L-tryptophan (L-tyrosine, or aromatic) aminotransferase, ProA aldolase, glutamate or aspartate racemase, and a broad speciiicitj' D-amino acid aminotransferase. Figure 5 is a diagram that illustrates the pathway. This approach to production of stereoisomerically enriched R,R monatin requires an enzyme for step 1 that has low activity in the production of monatin from monatin precursor (MP). Based upon earlier results, we used the Sinorhizobium meliloti and Rhodobacter sphaeroides tatA gene products described in Example 1 from WO 03/091396 A2.<br>
Materials and Methods<br>
[0781]	Glutamate racemases from L. brevis and P. pentosaceus were produced in E, coli<br>
as described in Example 17. In some cases the Hisg-tagged version of these enzymes were purified using His-Bind 900 cartridges according to manufacturer's protocols (Novagen, Madison, WI) and were desalted to remove imidazole using PD-10 columns (G25 Sephadex, GE Healthcare, Piscataway, NJ), The enzymes were eluted in 25 mM potassium phosphate pH 8.0. Aspartate racemase (ASPR-101) and D-aminotransferase (AT-103) were purchased from BioCatalytics, Inc. (Pasadena, CA). 5. meliloti and R. sphaeroides tyrosine (aromatic) aminotransferases were prepared as described in Example 1 from WO 03/091396 A2. Comamonas testosteroni ProA aldolase was prepared as described in Example 4 from WO 03/091396 A2. Total protein assays were<br>
 <br>
done   utilizing  the  Bio-Rad  Protein   Assay  according  to  manufacturer's  protocols (Hercules, CA).<br>
Reduction in amount of S,S monatin produced using racemases<br>
f07821	Reaction mixtures (I mL volume, run in duplicate) contained 100 mM potassium<br>
phosphate buffer (pH 8), 2 mM MgCl2, 0.05 mM pyridoxal 5'-phosphate (PLP), 200 mM sodium pyruvate, 5 mM sodium (x-ketoglutarate or oxaloacelate, approximately 280 |ig/mL S. meliloti TatA supplied in a cellular extract, 1 mg/mL BioCatalytics D-aminotransferase (AT-103; (BioCatalytics, Pasadena, CA), \Q0 \ihImL of glutamate racemase cellular extract or 1 mg/mL aspartate racemase, and approximately 100 ng/mL of ProA aldolase provided as a cellular extract. Solid tryptophan was added at a concentration of 10.2 mg/ml. Negative controls did not contain racemase. Samples were incubated at 30 "C (shaking at 250 rpm) for 1 hour, 2 hours, or overnight. Samples were centrifiiged to remove precipitate, syringe filtered, and stored at -80 °C prior to analysis for monatin using the LC/MS/MS method described in Example I. Most of the samples contained &gt;95% S,S monatin, due to tlie amounts of native L-aminotransferase present in the cellular extracts. However, the samples that contained racemase had a reduced amount of total monatin as a result of the racemase enzymes making L-glutamate less available for transamination of MP. Without racemase, 1545-2355 ppm monatin (predominantly S,S) was produced during the timecourse. With the racemases present, only 340-879 ppm (L. brevis enzyme), 444-531 ppm (P. pemosaceus enzyme), and 506-1460 ppm monatin (aspartate racemase) were produced. These data indicate that the racemases are active in the reaction conditions required to produce monatin. To minimize formation of S,S monatin from cellular extract enzymes such as aspartate aminotransferases, further experiments were done with purified enzymes and a higher ratio of D-aminotransferase to L-aminotransferase enzymes.<br>
Conversion of L-tryptophan to 4-R containing isomers of Monatin<br>
[0783]	The above experiments were repeated using approximately 54 p.g of purified L-<br>
aminotransferase (either S. meliloti or R. sphaeroides TatA), I mg aspartate aminotransferase (BioCatalytics, Pasadena, CA), 1 mg D-aminotransferase , oxaloacetate as the amino acceptor, and 75 .g purified aldolase. Reactions were run in duplicate with<br>
 <br>
a 2-hour sampling time and an overnight- incubation time. Negative controls were done with S. meliloti L-aminotransferase but no racemase. In addition to quantification of R,R/S,S and S,R/R,S monatin peak quantification based on reversed phase chromatography, the percentage of each stereoisomer was determined using the FDAA<br>
 <br>
[0784J	Clearly the presence of the racemase increased the total amount of monatin<br>
produced when S. meliloti TatA was used as the enayme for L-tryptophan transamination. Monatin levels increased fi-om an average of 6.4 to 16.5 ppm in the two-hour assay, and from 41-73 ppm in the ovemight assay. Additionally, the percent of R,R fomied increased from about 1% up to as much as 58% by utilizing the racemase enzyme. The S,R stereoisomer of monatin, another potent sweetener, was the other major component, increasing from nearly 0 in the negative controls to 31%. The R. sphaeroides TatA clearly had more activity on S-MP then the S. meliloti L-transaminase, demonstrating the importance of having an enzyme that has a high substrate specificity for L-tryptophan as compared to MP when 4-R isomers of monatin are the desired products. With about 10% of the total monatin being 4S at the two-hour time point, the S. meliloti TatA could be considered as having limited activity on MP.<br>
 <br>
[0785]	The experiments were repeated with the purified S. meliloti TatA (54 )j.g) and the<br>
L brevis gJutamate racemase. iiVhen puriiied glutamate racemase was used, approximately 64 \i% was used per 1 mL reaction. Cellular extracts containing the glutamate racemase were also tested, and 1.4 mg of soluble protein was used. A no racemase negative control was utilized again, and all samples were run in duplicate. The results are as follows:<br>
 <br>
[0786]	Again, it is clear that the addition of the racemase increases the total monatin<br>
produced from L-tryptophan, as well as increasing the relative amounts of 4R-containing isomers of monatin as compared to S,S monatin. The use of purified aldolase, racemase, and L-aminotransferaae greatly improves the ability to control the desired stereoisomer foimation. The ratio of L to D aminotransferase is also a way to manipulate stereochemistry of the final product.<br>
[0787]	When comparing results shown in Tables 18 and 19 in Example 13 to results with<br>
reaction conditions similar to the conditions above, one can see that approximately 7-29 ppm of monatin were formed from indoIe-3-pyruvate and the percentages of R,R monatin formed were approximately 51-90%. Using the aspartate racemase increased the total amount of monatin produced to 16-78 ppm monatin with % R,R of approximately 40-58%.   Additionally, a more stable and less expensive raw material, L-tryptophan, was<br>
 <br>
utilized. In Example 14, approximately 73 ppm monatin was produced from D-<br>
tryptophan at a ratio of R,R:S,R of approximately 1.7:1. The total amount of 4R isomers<br>
was &gt;80% of the total monatin. Because both R,R-monatin and S,R-monatin are potent<br>
sweeteners (&gt;1000 times sweeter tlian sucrose) the ability to enrich for these isomers<br>
without the need for expensive D-amino acid substrates is criticaJ.<br>
[0788]	As described in Examples 13 and 14, D-alanine can serve as the amino donor for<br>
transamination of MP to monatin. Many L-aminotransferases have the ability to utilize pyruvate as an amino acceptor to some extent, and produce L-alanine. Because the above-mentioned reactions use high concentrations of pyruvate, it is likely that some of the pyruvate is converted to L-alanine. For example, during transamination of L-tryptophan, the HexAspC enzyme described in Example 16 has been found to convert 10-18% of pyruvate (50-200 mM initial concentrations) to L-alanine in 2 hours if alpha-ketoglutarate is absent. The enzyme showed a 10-fold preference for alpha-ketoglutarate when both amino acceptors were present at high (&gt;50 mM) concentrations. AspC (described in WO 03/091396 A2) also produced some L-alanine from pyruvate. Therefore, it is expected that one ian omit the addition of alpha-ketoglutarate or oxaloacetate in the above reactions, and utilize an alanine racemase (EC 5.1.1.1) in place of glutamate or aspartate racemase. Alanine racemase enzymes were first identified in Brucella abortus and Streptococcus faecalis (Marr, A.G. and Wilson, P.W. Arch. Biochem. Biophys., 49 (1954) 424-433 and Wood, W.A. and Gunsalus, l.C. J. Biol. Chem., 190 (1951) 403-416). The dadB gene in Salmonella typhimurium was identified as the source of alanine racemase activity, and several hundred homologs can be found in genomics databases. Other known sources of alanine racemase activity are Escherichia coli. Bacillus subtilis, Pseudomonas aeruginosa. Vibrio cholerae, Schizosaccaroyces pombe, and Bacillus cereus. A basidiomycetous mushroom, Lentinus edodes, also contains a broad activity alanine racemase. A thermostable homolog from Bacillus stearothermophilus is available for purchase from Sigma-Aldrich (catalog # A8936) (Sigma-Aldrich, St. Louis, MO) and lias been immobilized for commercial applications (Inagaki, K., Biochemistry, 25: 3268 1986). An alanine racemase is converted to a glutamate or aspartate racemase by random methods such as mutagenic PCR, passage through mutagenic strains, or other methods to those known in the art. A more focused evolution of the alanine racemase is focied on active site residues, including Lysl29,<br>
 <br>
Metl34, and the residues including and between Qly283 and Trp288 (numbering from Bacillus stearothermophilus).<br>
Example 19 D-phenylglycine aminotransferase (D-4-hydroxyphenylglycine aminotransferase)<br>
[0789]	As   shown   in  Figure   3,   D-phenylglycine   aminotransferase   is   useflil   in  a<br>
biosynthetic pathway for the production of monatin. For example, D-phenylglycine aminotransferase produces R,R monatin from R-MP with L-glutamate as the amino donor.<br>
PCR synthesis of P. stutzeri 4 D- hydroxyphenylgiycine aminotransferase from oligonucleotide primers<br>
[0790]	This   example   describes   methods   that   were   used   to   synthesize   4   D-<br>
hydioxyphenylglycine aminotransferase, a stereoinverting enzyme that can be used to convert R monatin precursor to R,R monatin using L-glutamate as the amino donor.<br>
Primer Design<br>
[0791]	The  published  sequence  (Genbank Accession No.  AY319935,  nucleic acid<br>
sequence; Genbank Accession No. AAQ8290, protein sequence) for Pseudomonas stutzeri 4 D-hydroxyphenylglycine aminotransferase (4 D-HPG AT) was used as a template for PCR pruner design. Alternatively, the 4-D-hydroxyphenylglycine aminotransferase from Pseudomonas putida, (CAD42450 (protein), AX467211 (nucleotide)) is used as a sequence template. A total of 34 forward primers and 35 reverse primers were designed; forward and reverse primers were 40-mers sharing 20 overlapping base pairs. In addition, 2 outer primers were designed with 5' restriction sites and overhangs for cloning into the pET 28 and pET30 vectors (Novagen, Madison, WI).<br>
P. stutzeri 4 D-HPG AT outer primers: N term (with Nde\ Site): 5'-GGCCGGCATATGTCGATCCTT/iACGACTACAAACGT -3'  (SEQ ID NO:405), and C term (withol site): 5'-GGAAGGCTCGAGTCATGATTGGTTTCCAGACAAATT -3' (SEQ ID NO:406).<br>
 <br>
Polymerase Chain Reaction Protocol<br>
[0792]	The gene sequence trom P. stvtzeri was amplified using the following protocols.<br>
The primary 100 jxL PCR reaction included 0.05 M of each of the internal 69 primers, OA mM each dNTP, 10 U vTth Polymerase XL (Roche, Indianapolis, IN), 0.625 U Pfu polymerase (Stratagene, La JoUa, CA), IX XL buffer and 1 mM Mg(0Ac)2. The thermocycler program used included a hot start at 94°C for 3 minutes, 15 repetitions of the following steps: 94°C for 30 seconds, ATC for 30 seconds, and 6S°C for 15 seconds, followed by 10 repetitions of the following steps: 94°C for 30 seconds, S2°C for 30 seconds, and 68°C for 30 seconds, followed by 10 repetitions of the following steps: 94*0 for 30 seconds, 60°C for 30 seconds, and 68°C for 1 minute and 15 seconds. After the final JO cycles the sample was maintained at 6B°C for 7 minutes and then stored at 4°C. This PCR protocol produced a smear of product at -0.5 kb on a 0.8% TAE-agarose gel.<br>
[0793]	The secondary PCR reaction was set up using the primary PCR reaction as ,<br>
template. The secondary 100 fiL PCR reaction included 2.5 pL of the primary PCR reaction, 0.5 pM of each of the 2 outer primers (with AWel SD.6.Xho\ restriction sites), 0.4 mM each dNTP, 10 U xTth Polymerase XL, 0.625 U Pfu polymerase, IX XL buffer and 1 mM Mg(0Ac)2. The thermocycler program used included a hot start at 94°C for 3 minutes, 10 repetitions of the following steps: 94°C for 30 seconds, 52°C for 30 seconds, and 68°C for 1 minute 30 seconds, followed by 15 repetitions of the following steps: 94°C for 30 seconds, 60°C for 30 seconds, and 68°C for 1 minute 30 seconds. After the 15 repetitions, the sample was maintained at 68°C for 7 minutes and then stored at 4°C. This PCR protocol produced a distinctive product band at -1.4 kb on a 0.8% TAE-agarose gel.<br>
[0794]	The PCR product was gel purified from 0.8% TAE-agarose gel using the Qiagen<br>
gel extraction kit (Qiagen, Valencia, CA). The product was TOPO cloned and transformed into TOP 10 cells according to manufacturer's protocol (Invitrogen, Carlsbad, CA). Plasmid DNA was purified &amp;om the resulting transformants using the Qiagen spin miniprep kit (Qiagen, Valencia, CA) and screened for the correct inserts by restriction digest with Ndel and Xhol The sequences of plasmids appearing to have the correct insert were verified by dideoxy chain termination DNA sequencing with tiniversal M13 forward and M13 Reverse primers.  Of the 10 clones sequenced, all had at least one<br>
 <br>
mutation from the desired sequence. The best clone had a single base-pair mutation that resulted in an amino acid change. The sequence of this clone was corrected using the QuickChange Mutagenesis protocol according to manufacturer recommendations (Stratagene, La JoUa, CA).<br>
|0795]	The corrected TOPO clone was digested with restriction enzymes Ndel and Xhol<br>
following the manufacturer's recommended protocols (Hew England Biolabs, Beverly, MA) and gel purified from 0.8% TAE-agarose gels using the Qiagen gel extraction kit (Qiagen, Valencia, CA). Vectors pET 28 and pET 30 were prepared by digestion with restriction enzymes Ndel and Xhol followed by treatment with shrimp alkaline phosphatase and purification from 0.8% TAE-agarose gels using the Qiagen gel extraction kit (Qiagen, Valencia, CA).<br>
[0796J	The digested vectors and insert were iigated using the NEB Quick Ligation Kit<br>
(Beverly, MA). Approximately 50 i of freated insert, 100 ng of treated vector (3 to 1 molar ratio of insert tp, vector),. 5 .U of T4 DNA ligase, and IX ligation buffer were incubated for 5 minutes at room temperature. TTie ligation mixture was transformed in to TOPIOF' chemically competent cells (Invitrogen, Carlsbad, CA). The cells were allowed to recover in 0.25 mL of room temperature SOC for 1 hour at 37°C with shaking at 225 rpm. Cells were plated on LB plates containing kanamycin (50 xg/mL). Plasmid DNA is purified from the resulting fransformants using the Qiagen spin rainlprep kit (Qiagen, Valencia, CA) and screened for the correct inserts by restriction digest with Ndel and Xhol.<br>
Gene Expression and Assays<br>
[07971	Plasmid DNA was transformed into E. coli expression host BL21(DE3) (Novagen,<br>
Madison, WI). The cultures were grown and the plasmids were isolated using Qiagen miniprep kit (Qiagen, Valencia, CA), and analyzed by restriction digest to confirm identity.<br>
{0798}	Induction in BL21(DE3) is initially performed with P. stutzeri 4 D - HPG in both<br>
pET 28 (histidine-tagged) and pET 30 (untagged) vectors. A time course study is perforoied with cultures grown in 250 mL LB containing kanamycin (50 mg/L) to m OD600 of 0.5 - 0.6, induced with 100 mM IPTG (isopropyl thiogalacatoside) and sampled at 0 and 3 hours post induction. An appropriate volume of cells from 0 hours and 3 hours is resuspended in 40 L sodium dodfc-.vl sulfate buffer containing 2-mercaptoethanol and<br>
 <br>
heated at 95°C for 10 minutes, and cooled.    Aliquots of these total cellular protein<br>
samples is analyzed by SDS-PAGE using a 4-15% gradient gel.<br>
[0799]	Cell extracts are also prepared from the 3 hour cultures by suspending cell pellets<br>
from 5 mL of culture in 0.625 mL Novagen BugBuster' reagent (Novagen, Madison, WI) containing 0.625 (iL benzonase nuclease and 3 jiL protease inhibitor cocktail set #3 (Calbiochem-Novabiochem Corp., San Diego, CA) at room temperature for 20 minutes with gentle shaking, and centrifiiging at 16,000 x g to remove cell debris. The supematants (cell extracts) are loaded onto 4-15% gradient gels for analysis of the cellular soluble proteins. When required, protein is purified using His-Bind 900 cartridges according to manufacturer's protocols (Novagen, Madison, WI) and is desalted to remove imidazole using PD-10 columns {G25 Sephadex, GE Healthcare, Piscataway, NJ).<br>
Organisms with D-phenylglycine aminotransferase (DPGAT)<br>
(0800]	Organisms of the genus Pseudomonas and like genera, with a stereoinverting D-<br>
phenylglycine aminotransferase (also called D-4-hydroxyphenylglycine aminotransferase) are isolated in the following maimer. Soil samples are incubated on petri plates with the following medium: (per liter) 15 g Agar, 3.4 g KH2PO4, 3.55 g Na2HP04, 0.2 g MgS04-7H20, 8 mg CaCl2-2H20, 10 mg yeast extract, 1 ml lOOOx trace elements solution, 1 g D-phenylglycine (D-4-hydroxyphenylglycine).<br>
fOSOlJ	Isolates are tested by PCR for the presence of a stereoinverting aminotransferase<br>
(primers designed from known D-phenylglycine aminotransferases) or are further enriched for the presence of a stereoinverting aminotransferase as follows: isolated from the plates could be grown in liquid medium as above minus the agar at 30°C with shaking to an ODeoo of about 1.0. Cells are harvested by centrifugation and washed twice with 0.85% NaCl. A 10 mg (wet weight) sample is suspended in 1 ml potassium phosphate buffer (pH 7.0) and 5 mM D-phenylglycme (or D-4-hydroxyphenylglycine). Neutralized 15 mM (aminooxy)acetic acid is added to duplicate samples prepared as described above. Consumption of D-phenylglycine (or D-4-hydroxygiycine) is measured by HPLC. Isolates capable of degrading D-phenlyglycine (or D-4-hydroxyphenylglycine), but do so at a slower rate in the presence of (aminooxy)acetic acid are selected for further analysis. Isolates are tested, by PCR, for the presence of a stereoinverting aminotransferase (primers designed from known D-phenylglycine aminotransferases).<br>
 <br>
[0802]	TJie presence of the stereoin-\'ertmg aminotransferase is confirmed by growing a<br>
culture in liquid medium as described above, harvesting the cells and making a cell free crude extract (CFE) and testing ibr D-phenylgiycine aminotransferase (or D-4-hydroxyphenylglycine aminotransferase) enzyme activity. CFE is added to a reaction mixture with the following final concentrations: 0.1 M CAPS (pH 9.5), 60 mM L-glutamate (sodium salt), 5 mM benzoylformate (or 4-hydroxybenzoate) and 50 p-M PLP. The reverse reaction is measured by adding CFE to a reaction mixture with the following concentrations:, 50 mM potassium phosphate (pH 7.0), 60 mM D-phenylglycine (or D-4-hydroxyphenylglycine), 5 mM a-ketoglutarate, 50 \iM PLP. The assays are incubated at 35°C and aliquots are taken at time points and stopped by boiling for 2 minutes. Product will be quantitated by the HLPC method of Gil-Av, E., Tishbee, A., Hare, P.E., Resolution of underivatizad amino acids by reversed phase chromatography. J. Am. Chem. Soc, 102: 5115-5117 (1980) or by methods described in Example 1 (measurement of glutamate formation).<br>
[0803]	As an alternative to PCR based methods, the stereoinverting D-phenylglycine<br>
aminotransferase is purified from the isolated bacteria by conventional protein purification techniques including ammonium sulfate fractionation, and conventional colximn chromatography. Once the protein has been purified to a reasonable degree peptide microsequencing techniques or conventional Edman type amino acid sequencing are utilized {see golgi.harvard.edu/microchem/ for descriptions of the protocols and equipment used for this type of work). Degenerate primers are designed based on sequence available from the closest known relative of the protein source. Degenerate PCR and genome walking is then performed according to established protocols to isolate the stereomverting D-phenylglycine aminofransferase coding sequence.<br>
DPGAT monatin production<br>
[0S04]	D-hydroxyphenylglycine aminotransferases, as described in (I) and (2) above, are<br>
used in crude cell free protein extracts or purified as described in (1) above. S. meliloti and R. sphaeroides tyrosine (aromatic) aminofransferases are prepared as described in Example 1 from WO 03/091396 A2. Comamonaa lestosteroni ProA aldolase is prepared as described in Example 4 from WO 03/091396 A2. Total protein assays are done utilizing the Bio-Rad Protein Assay according to manufacturer's protocols (Hercules, CA).<br>
 <br>
[0805]	Reaction mixtures (1 mL volume, run in duplicate) contain 100 mM potassium<br>
phosphate buffer (pH 8), 2 mM MgClj, 0.05 mM pyridoxal 5'-phosphate (PLP), 200 mM sodium pyruvate, 5 mM sodium a-ketoglutarate, approximately 280 )ig/mL S. meliloti TatA supplied in a cellular extract, 100 \xLlvaL of D-hydroxyphenylglycine aminotransferase cellular extract or 1 mg/mL purified D-hydroxyphenyJglycine aminotransferase, and approximately 100 ng/mL of ProA aldolase provided as a cellular extract. Solid tryptophan is added at a concentration of 10.2 mg/ml. Negative controls are set up without D-hydroxyphenylglycine aminofransferase. Samples are incubated al 30 "C with gentle shaking for ~1 hour or overnight. Samples are centrifuged to remove precipitate, syringe filtered, and stored at -80 'C prior to analysis for monatin using the LC/MS/MS method described in Example 1.<br>
[0806]	D-hydroxyphenylglycine aminotransferases with improved activity for monatin<br>
production are made my mutagenesis techniques known to those in the art, including: mutagenic PCR, passage through mutagenic straira, or site-directed mutagenesis. The improved D-hydroxyphenylglycinc aminotransferases are selected by growth on minimal medium with R,R-monatiu as the source of nitrogen. Initially the selection is based on growth but as improved aminotransferases are selected the screen is growth rate based. That is, cells with mutated versions of the gene are grown and the gene is expressed in minimal medium with R,R-monatin as the nitrogen source. The growth rates of the cells with the mutated versions of the gene are compared to the unmutated version. Those cells with a faster growth rate are selected and the aminotransferase is analyzed further. The D-hydroxyphenylglycine aminotransferase is mutagenized by available techniques such as error-prone PCR and passage through mutagenic strdns or by methods for which a license has been obtained such as DNA shuffling and other directed evolution technologies.<br>
DPGAT assay<br>
[0807]	Cells  with  the  recombinant  D-hydroxyphenylglycine   aminotransferase  were<br>
grown, the protein expressed, and the cells lysed as described in Example 17 or by standard protocols. The protein is expressed in its native host using described methods (Wiyakrutta, W., Meevootisom, V. A stereoinverting D-phenylglycine aminotransferase<br>
 <br>
from Pseudomonas stutzeri ST-201: purification, characterization, and application forD-<br>
phenylglycine synthesis. J. Biotechnol., 55: 193-203 (1997)).<br>
[0808]	Cell free extract was added to a reaction mixture with the following final<br>
concentrations: 100 mM potassium phosphate (pH 7.0-8.5), 60 mM D-phenylglycine (or D-4-hydroxyphenylglycine), 5 mM a-ketoglutarate, 50 iM PLP. The assays were incubated at room temperature and aliquots were taken at time points and stopped by adding an equal volume of formic acid. Product (L-glutamate) is quantitated by methods described in Example 1.<br>
Example 20 Discovery of a D-methionine aminotransferase gene<br>
Background<br>
f0809]'	D-methionine—pyruvate aminotraiisferase (EC 2.6.1.41) is thought to be another<br>
example, although rare, of a stereoinverting transaminase. This enzyme catalyzes the reversible conversion of D-methionine and pyruvate to L-alanine and 4-methylthio-2-oxobutanoate. Oxaloacetate, phenylpymvate, 2-oxobutyrate, 2-oxovalerate, 2-osoheptanoate, glyoxylate, and oxoglutarate can also serve as amino acceptors.<br>
[0810]	Transamination of D or L methionine is thought to be part of a pathway to<br>
ethylene production m higher plants (cauliflower, tomato, apple, pea stem, banana, peanut) as well as in soil microorganisms {Escherichia coli, Pseudomonas pisi, Pseudomonas aeruginosa. Bacillus mycoides, Acinetobacter calcoaceticus, Aeromonas hydrophila B12E, Rhizobium Irifolii N2P7, Penicillium digitatum Saccharomyces cerevisiae, Corynebacterium D7F). D.C. Billington, B.T. Golding, and S.B. Primrose Biochem J. (1979) 182, 827-836. hi bacteria, L-methionine is typically used as the substrate in the ethylene production studies and broad specificity enzymes such as TyrB or AspC from E. coli are thought to be responsible for the transamination. However, S.B. Primrose J. Gen. Microbiol. (1976), 95(1), 159-65 and S.B. Primrose J. Gen. Microbiol. (1977), 98, 519-528. showed that E. coli strain SPA O (University of Warwick culture collection) produced nearly as much ethylene from D-methionine as from L-methionine in batch cultures. Because no broad specificity D-aminotransferase has been identified in E. coli, one possible explanation could be that the E. coli D-amino acid dehydrogenase<br>
 <br>
(encoded by the dadA gene) converts the D-methionine to 4-methylthio-2-oxobutanoate,<br>
It is also possible that there is a methionine racemase in E. coli; however, no such enzyme<br>
has been described in the literature.<br>
[0811]	In contrast to E, coli, m caiiliiiower florets (mitochondrial extract preparations)<br>
and germinating peanut seeds production of ethylene was higher when D-methionine and pyruvate were supplied to the enzyme extract as compared to L-methionine and pyruvate (L.W. Mapson, J.F. March, and D.A. WardaJe Biochem J. (1969) 115, 653-661; J.I. Durham, P.W. Morgan, J.M. Prescott and CM. Lyman Phytochemistry (1973) 12, 2123-2126). Therefore the possibility of a combination of methionine racemase and an L-aminotransferase is not supported by the data. Dehydrogenase activity was ruled out by dialysis of ceiiuiar extracts of cauliflower, no NAD was present in the assay mixtures. Oxidase activity was ruled out as no consumption of oxygen was noted and there was no requirement for FAD. The D-methionine aminotransferase from peanut tissues was purified, shown to be dependent on PLP, and shown to be independent of L-methionine aminotransferase activity. There is a possibility that these D-methionine—pyruvate aminotransferases actually produce D-aJanine as a byproduct {similar to the Bacillus enzymes described in Examples 13 and 14), and that the cells contain alanine racemase to recycle the D-alanine back to L-alanine (or an analogous amino donor). In either case, discovery of the broad specificity D-aminotransferase from higher plants is advantageous for development of processes that produce R,R monatin or S,R monatin.<br>
Experimental Overview<br>
[0812]	D-methionine aminotransferase is partially purified from cauliflower florets and<br>
germinatmg peanut embryos using standard chromatography protocols and a Pharmacia AKTA Explorer system. The protein sequences of homologous proteins are determined by LC/MS/MS fingerprinting techniques and database searching performed by Harvard Microchemistry fility. The coding regions of the plant genes are cloned from a cDNA library using standard PCR protocols or by synthesis of the gene as described in Example 19.<br>
[0813]	Alternatively, cDNA expression libraries are constructed (Stratagene, La Jolla,<br>
CA) from cauliflower tissue or peanut seeds grown in the presence of D-methionine (and producing ethylene). The libraries are transformed into E. coli methionine auxotrophs from the E. coli Genetic Stock Center (Yale) such as strains RC519 or AB1931. Plasmids<br>
 <br>
of strains capable of growth on minimal media containing D-methionine contain the<br>
coding region of interest (see Example 15, an analogous screening technique).<br>
[0814]	Once the coding regions of interest are obtained and are expressed in a standard E-<br>
coli laboratory strain, the resulting gene products can be used in assays to produce R,R monatin as described in Example 19 in place of the D-hydroxyphenylgiycine aminotransferase, with the exception of the pH being 7.5 (the optimal pH for the aminotransferase). If the D-methionine aminotransferase has a strict requirement for D-amino acid donor substrates, the enzyme can be used to make R,R monatin as described in Examples 13 and 14. The gene can be mutagenlzed and screened for increased activity as described in Example 19.<br>
Methods<br>
Isolation from cauliflower<br>
[0815]	Four hundred grams of freshly picked cauliflower florets are extracted with 400<br>
mL of a 4 "C sucrose/buffer solution (0.4 M sucrose and 0.1 M sodium phosphate buffer pH 7.4) by alternating soaking and mixing using a blender. Cell debris is removed by filtration with cheesecloth and the resulting solution is centrifuged at 40,000Xg for 30 minutes at 4 °C. The solid material (containing mitochondrial organelles) is resuspended in 20 mL 10 mM sodium phosphate buffer pH 7.4, and enzymes are extracted with 200 mL cold (-30 "C) acetone. The suspension is recentrifiiged and the precipitate is dried using a Savant Speed Vac. The solid material is dissolved in 10 mM sodium phosphate buffer pH 7.4, and residual acetone is removed using a PD-10 column (GE Healthcare, Piscataway, NJ).<br>
[0816]	Aminotransferase activity is assayed by incubation of the enzyme preparation with<br>
5 mM D-methionine, 1 mM pyruvate, 0.05 mM PLP and 2 mM EDTA in 0.1 M sodium phosphate buffer pH 7.4. Assays are performed at 25 "C for 16 hours. The 4-Methylthio-2-oxobutanoate is measured by formation of e 2,4-dinitrophenylhydrazone derivative, using LC/MS (m/z of 328) and similar methodology described in Example 1. A 0.4% (w/v) solution of 2,4-dinitrophenylhydrazine in 2M sulfuric acid is prepared, and a half volume is added to the assay mixture after incubation. The mixture is mixed with gentle shaking at 30 "C for 30 minutes and the precipitate is collected by centri&amp;gation and analyzed  by  LC/MS.        Protein  fractions   separated  by   standard  chromatographic<br>
 <br>
techniques are assayed for activity' in a similar manner, but the co-product alanine is measured by the OPA post-column derivitization technique described in Example 1.<br>
Isolation from peanut (Arachia hypogea L. cv. Starr)<br>
[0817]	The D-methionine aminotransferase enzyme from germinating peanut embryo<br>
homogenate (minus the cotyledons) is purified according to the method of J.I. Durham, P.W. Morgan, J.M. Prescott and CM. Lyman Phytochemistry (1973) 12, 2123-2126. Reducing agents are used during the preparation of crude extracts to stabilize the enzymes, and the cell debris is removed by centrifugatlon at 33,OOOXg. A 35-50% ammonium sulfate fraction is further purified by incubation at low temperature, and by removal of the proteins in the precipitate. The supernatant is further fractionated using acetone. The active pools are then frirther purified by gel filtration chromatography (Sephadex 200, GE Healthcare, Piscataway, NJ).<br>
[0818]	As protein fractions become enriched with the transaminase protein, 2D-gel<br>
electrophoresis is utilized to separate the enzyme of interest for microsequencing. After elucidation of homologous coding regions in plant sequences deposited at NCBI, the D-aminofransferase protein is produced recombinantly in Escherichia coli using standard molecular biology techniques. It is expected that the cellular extracts from cauliflower florets or peanut seeds or recombinantly produced homologous enzymes can be used in production of R,R monatin as described in Example 19 (if a stereoinverting transaminase) or Examples 13 and 14 (if a broad specificity D-aminotransferase).<br>
Example 21 L-alanine aminotransferase/alanine racemase/D-alanine aminotransferase<br>
[0819]	Figures    4,    6,    and    8    iUusfrate    biosynthetic    pathways    for   producing<br>
stereoisomerically-enriched R,R monatin from L-tryptophan using a L-amino acid aminotransferase (such as L-alanine-aminofransferase and/or L-tryptophan-aminotransferase), an R-specific aldolase, an alanine racemase and a D-alanine aminotransferase.<br>
[0820]	A    tryptophan-specific    aminotransferase    is    described    in    Example    16.<br>
Alternatively, S. meliloti and R. sphaeroides tyrosine (aromatic) aminotransferases are prepared as described in Example 1 from WO 03/091396 A2.   Comamonas testosteroni<br>
 <br>
ProA aldolase is prepared 3s described in Example 4 from WO 03/091396 A2. Total protein assays are done utilizing the Bio-Rad Protein Assay according to manufacturer's protocols (Bio-Rad, Hercules, CA). Janine racemase is purchased from Sigma-Aldrich (St. Louis, MO) (catalog number A89.'16). D-alanine aminotransferase is purchased from BioCatalytics (catalog number AT-103) (BioCatalytics, Pasadena, CA).<br>
[0821]	L-alanine aminotransferases ai-e widely distributed in eukaryotes, bacteria, and<br>
archaea. The following organisms have been identified (based on sequence homology) as containing an L-alanine aminotransferase (E.G. 2.6.L2): Arabidopsis thaliana, Ashbya gossypii, Azotohacter vineiandii, Bifidobacterium longum, Caenorhdbditis elegans, Candida albicans, Candida glabrata, Chlamydomonas reinhardtii, Cryptococcus neoformans, Debaryomyces kamenii, Homo sapiens, Bordeum vulgare, Kli/yveromyces laclis, Magnaporthe grisea, Medicago truncatula, Mus musculus, Neurospora crassa, Oryza saliva, Phanerochaete chrysosporium, Finns taeda, Pseudomonas putida, Pyrococcus abyssi, Pyrococcus furiosns, Pyrococcus horikoshii, Rattus norvegicus, Saccharomyces cerevisiae, Schizosaccharomyces pombe, Takifugu rubripes, Trypanosoma cruzi, Vibrio cholerae, Vibrio parahaemolyticus. Vibrio vulnificus, Yarrowia lipolytica, and Zea mays. Additionally many aminotransferases have low-level aJanine aminotransferase activity and given high levels of L-glutamate and pyruvate can convert it to L-alanine and a-ketoglutarate. An enzyme with low activity is improved with standard mutagenesis techniques such as error-prone PCR and passage through mutagenic strains or by directed evolution techniques. The gene for an L-aianine aminotransferase is cloned using publicly available to design primers and using standard techniques to amplify, clone, express and purify the gene/enzyme.<br>
[0822]	Reaction mixtures (1 mL volume, run in duplicate) contain 100 mM potassium<br>
phosphate buffer (pH 8), 2 mM MgCl:, 0.05 mM pyridoxal 5'-phosphate (PLP), 200 mM sodium pyruvate, 5 mM sodium a-ketoglutarate, approximately 280 |ig/mL S. meliloti TatA supplied in a cellular extract (or other L-tryptophan specific aminotransferase), 100 IL/mL of alanine racemase cellular extract or 1 mg/mL purified alanine racemase, approximately 280 fxg/mL D-alanine aminotransferase supplied in a cellular extract and approximately 100 ig/mL of ProA aldolase provided as a cellular exu-act. Solid tryptophan is added at a concentration of 10.2 mg/ml. Negative controls are set up without alanine racemase.   Samples are incubated at 30 "C with gentle shaking for ~1<br>
 <br>
hour or overnight. Samples are centrifliged to remove precipitate, syringe filtered, and stored at -80 'C prior to analysis for moitin using the LC/MS/MS method described in Example 1. In these reaction mixtures, if the L-tryptophan aminotransferase can use a-ketoglutarate, but not pyruvate, as an amino acceptor, one can add an L-alanine aminotransferase, which converts L-glutamate and pyruvate and to L-alanine and a-ketoglutarate, as shown in Figure 6.<br>
Example 22<br>
Subcloning of genes encoding the aldolases of SEQ ID NO: 276, 244, 218, 228,<br>
162, 50, 74, 44, and 116<br>
[0823]	The genes encoding aldolases having the amino acid sequences of SEQ ID NO:<br>
276, 244,218,228, 162,50,74,44, and 116 were subcloned into the pET28b expression vector (Novagen, Madison, WI) with N-terminal His-tags to allow for purification. SEQ ID NO:275 is the sequence of the gene that encodes the aldolase having the amino acid sequence of SEQ ID NO:276. SEQ ID NO:243 is the sequence of the gene that encodes the aldolase having the amino acid sequence of SEQ ID NO:244. SEQ ID N0:217 is the sequence of the gene that encodes the aldolase having the amino acid sequence of SEQ ID N0:21S. SEQ ID NO:227 is the sequence of the gene that encodes the aldolase having the amino acid sequence of SEQ ID NO:228. SEQ ID NO: 161 is the sequence of the gene that encodes the aldolase having the amino acid sequence of SEQ ID NO: 162. SEQ ID NO:49 is the sequence of the gene that encodes the aldolase having the amino acid sequence of SEQ ID NO:50. SEQ ID NO:73 is the sequence of the gene that encodes the aldolase having the amino acid sequence of SEQ ID NO: 74. SEQ ID NO;43 is the sequence of the gene that encodes the aldolase having the amino acid sequence of SEQ ID NO:44. SEQ ID N0:115 is the sequence of the gene that encodes the aldolase having the amino acid sequence of SEQ IDN0:116.<br>
{0824]	The primers used for cloning are shown in Table 24.<br>
 <br>
 <br>
 <br>
[0825]	The genes encoding the above aldolases were amplified by PCR and digested with<br>
appropriate enzymes {Me 1 and BamH I) and gel purified (QIAquick® Gel extraction Kit (Qiagen, Valencia, CA)). The digests svem individually ligated into pET28 that had been digested with fde I and BamH I and gel purified. The ligation was transformed into TOPIO cells (Invitrogen, Carlsbad, CA). Miniprep DMA from individual colonies was analyzed for the presence of inserts by size analysis usmg agarose ge! electrophoresis. Isolates with an insert were submitted for DNA sequence analysis (Agencourt, Beverly, MA).<br>
Purification of aldolases<br>
[08261	Confirmed   aldolase   clones   were   transformed   into   either   BL2I(DE3}   or<br>
BL21{DE3) pLysS. Induction was carried out overnight in Terrific Broth at 30 "C. Overnight cultures grown with the appropriate antibiotic were diluted into fi-esh medium (typically 1:100) and grown to an ODoo ~0.6 with aeration at 37°C.  Cultures were then<br>
 <br>
induced with 1 mM IPTG and shifted to 30°C (with aeration) and incubation was<br>
continued overnight. Cells were harvested by centritugation. The cell pellet was<br>
typically subjected to one freeze thaw cycle to assist with cell lysis. The cell pellet was<br>
lysed in BugBuster and Benzonase Nuclease (Novagen, Madison, WI) (according to the<br>
manufacturer's protocol). Cell debris was removed by centrifiigation. The crude protein<br>
extract was appHed to a 10 mg capacity HIS-Bind column (Novagen, Madison, WI) that<br>
had been prepared according to the manufacturer's protocol. The column was washed<br>
and the protein was eluted according to the manufacturer's protocol. The purified protein<br>
was desalted with PD-10 columns (GE Healthcare, Piscataway, NJ) and eluted in 50 mM<br>
potassium phosphate buffer, pH 7.5, containing 4 mM MgCb, 200 mM NaCl. Purified<br>
protein was concentrated with Aniicon centrifugal concentrators (5000 MW cutoff)<br>
(Millipore, Billerica, MA). After concentration, it was noted that the aldolases of SEQ ID<br>
NO: 44, SEQ ID NO: 28, and SEQ ID NO: 276 showed some level of precipitation,<br>
although the activity was still quite high. Proteins were stored at -80 "C until assayed.<br>
[0827]	Protein assays were done using the Pierce BCA kit (Pierce, Rockford, IL) and the<br>
microliter plate protocol with Bovine Serum Albumin ("BSA") as the protein slandard. - The Experion Pro260 electrophoresis system (Bio-Rad, Hercules, CA) or SDS-PAGE was used to estimate the percentage of aldolase in the purified sample, and to evaluate expression levels m the soluble cell extract and in total protein.<br>
Testing of purified aldolases<br>
[0828]	Purified aldolases were tested for their ability to produce R,R monatin from D-<br>
tryptophan, and were compared to the aldolases of SEQ ID NO:28 and SEQ ID NO:54 prepared in the same manner. Assays were run in microcentrifuge tubes (in duplicate) with purified protein, using the same concenfration of enzyme per assay (50 ng/mL) with the exception of SEQ ID NO:244, for which 25 \ig/mL was used. SEQ ID NO:243 did not express well and yielded smaller amounts of purified protein. Two mg/mL of BioCatalytics AT-103 (BioCatalytics, Pasadena, CA) was used as the D-aminotransferase. The foUowmg were added per 1 mL of reaction mixture: aldolase, 4 mM MgCb, 50 mM D-tryptophan, D-ammofransferase, 200 mM sodium pyruvate, 100 mM potassium phosphate buffer pH 7.5, and 0.05 mM PLP. Samples were incubated at 30 °C. Thirty minute, 1 hour, 3 hour, and overnight (19 hour) samples were taken. Table 25 shows the averaged results of total monatin produced at each time point and the % R,R<br>
 <br>
 <br>
<br>
 <br>
highest stereospecificity for production of R,R monatln.   Storage of this enzyme in a<br>
buffer omitting the sodium chloride appears to reduce the level of precipitation noted<br>
earlier. Magnesium concentration in the storage buffer does not appear to have an impact<br>
on the level of precipitation.<br>
[0830]	The aldolases of SEQ ID NO:276, SEQ ID NO: 28, and SEQ ID NO: 244 all<br>
demonstrated high activity and high stereoselectivity for production of R,R monatin. These three enzymes were prepared as described in Example 23 and assayed anaerobicaJIy, as described in Example 24, using 10 mL serum vials. Seven mL assays were done using 0.05 mg/mL of each aldolase (purified) and 2 mg/mL of purified B. ' sphdericus D-aminotransferase prepared as described in Example 24. The activity of each aldolase in production of monatin from D-tryptophan was compared to the S. meliloti HMG aldolase prepared as described in Example 27. Total monatin was estimated using the LC-OPA method described in Example 1. The percentage of R,R monatin formed was determined using the FDAA derivatization method described in Example 1. The results are shown below in Table 26.<br>
 <br>
 <br>
Example 23 Expression and Purification of the SEQ ID NO:276 aldolase<br>
[0832]	The cloning of the E. coli BL21 (DE3)pLysS host carrying the aldolase gene listed<br>
as SEQ ID NO:275 on the pET28b plairaid is described above in Example 22.<br>
[0833]	The SEQ ID NO:276 aldolase with an amino-terminal HlSe-purification tag was<br>
produced using the EMD Biosciences Overnight Express System II (Novagen, Madison, WI) (solutions 1-6) containing 50 [ig/mL kanamycin in shake flasks. This expression system induces the expression of IPTG-inducible systems without the need to monitor cell growth. After inoculation of 200-mL aliquots of the medium (in 1 L flasks) from either liquid cultures or plates of flie aldolase construct, the cultures were incubated at SOC overnight with shaking at 225 rpm. When the ODeoonm had reached a minimum of 6, the cells were harvested by centrifiigation and washed once with buffer.<br>
[0834]	To prepare cell free extract containing the aldolase, the cells were suspended in 3-<br>
4	volumes of 100 mM potassium phosphate, pH 7.8 and then disrupted using a<br>
Microfluidics homogenizer (Microfluidics, Newton, MA) (3 passes at 18,000 psi),<br>
maintaining the temperature of the suspension at less than 15°C. Alternatively, cell free<br>
extract was prepared using EMD Biosciences BugBuster® (primary amine-free)<br>
Extraction Reagent (Novagen, Madison WI) contmning 1 jxL/mL Benzonase  Nuclease,<br>
5	)LIvaL Protease Inhibitor Cocktail Set fl, and 0.033 jiL/mL rLysoTiie" following the<br>
manufacturer's protocol. All subsequent purification steps were carried out at 4°C. The<br>
cell suspension was centrifuged for 20-30 minutes at 15,000-20000 x g to remove the cell<br>
debris. A 20-25 mL aliquot of the cell free extract was applied to a 45 mL column of GE<br>
Healthcare Chelating Sepharose™ Fast Flow resin (nickel (II) form) (GE Healthcare,<br>
Piscataway, NJ) that had been previously equilibrated with 100 mM potassium phosphate<br>
containing 200 mM sodium chloride. To generate the nickel form of the resin, the resin<br>
was washed with 150 mL of 200 mM nickel (II) sulfate hexahydrate and then with 150<br>
mL of distilled water. After loading the sample, the column was washed/eluted with 150<br>
mL of the equilibration buffer containing 25 mM imidazole, 150 mL of the equilibration<br>
buffer containing 50 mM imidazole and 150 mL of the equilibration buffer containing<br>
500 mM imidazole. The ffiSe-tagged protein eluted in the last wash. The 500 mM<br>
imidazole wash was concentrated with Millipore/Amicon Centricon Plus-70 centrifugal<br>
 <br>
filter devices (MWCO 10 kHa) (Millipore, Billerica, MA) to 15-20 mL according to the<br>
manufacturer's instructions. The imidazole and sodium chloride were removed by<br>
passage through disposable GE Healthcare PDIO columns (GE Healthcare, Piscataway,<br>
NJ) (2.5 mL sample per column) previously equilibrated with 100 mM potassium<br>
phosphate, pH 7.8. The purified aldolase was eluted with 3.5 mL per column of the same<br>
buffer.<br>
[0835]	The protein concentration of each fraction was determined using the Pierce BCA<br>
assay kit (Pierce, Rockford, IL) using BSA as the protein standard. The purity of each fi-action and the level of expression in the cell free extract were determined using a Blo-Rad Experion microcapillaiy chip sysiem (Bio-Rad, Hercules, CA) or using Bio-Rad 4-15% SDS-polyacrylamide gradient gels run in a Mini PROTEAN® 3 cell apparatus (Bio-Rad, Hercules, CA). The protein v/ar, visualized in the polyacrylamide gels using Bio-Rad Bio-Safe G-250 Coomassie stain (Bio-Rad, Hercules, CA) and destained with water. .Typically this procedure produces - 50 mg of enzyme from 400 mL of overnight culture that is 85-95% pure as judged by the Experion software. Aliquots (1-5 mL) of the purified enzyme were stored at -80° C until use. Preparation of the enzyme in this manner reduced the level of precipitation of the enzyme previously noted. The presence of magnesium in the storage buffer had no effect on the level of precipitation.<br>
Example 24<br>
Production of R,R-monatin Using the SEQ ID NO:276 Aldolase: Optimization of<br>
Reaction Conditions<br>
[0836]	The Bacillus sphaericus (ATCC strain 10208) D-alanine aminotransferase cloned<br>
in Example 7 was purified as the HlSe-tagged protein as described below using EMD Biosciences Overnight Express System II (Novagen, Madison, WI) for growth and induction. The EMD Biosciences Overnight Express System II (solutions 1-6) (Novagen, Madison, WI) contained 50 p-g/mL kanamycin m shake flasks. This expression system induces the expression of IPTG-inducible systems without die need to monitor cell growth. After inoculation of 200-mL aliquots of the medium (in 1 L flasks) from either liquid cultures or plates of the aminofransferase construct, the cultures were incubated at<br>
 <br>
SOX overnight with shaking at 225 rpm.  When the ODsoonm had reached a minimum of 6, the ceils were harvested by centrifugation and washed once with buffer.<br>
[0837]	To prepare cell free extract containing the D-alanine aminotransferase, the cells<br>
were suspended in 3-4 volumes of 100 mM potassium phosphate, pH 7.8, containing 50 \iM pyridoxal phosphate (PLP) and then disrupted using a Microfluidics homogenizer (Microfluidics, Newton, MA) (3 passes at 18,000 psi), maintaining the temperature of the suspension at less than 15°C. Alternatively, cell free extract was prepared using EMD Biosciences BugBuster® (primary amine-free) Exfraction Reagent (Novagen, Madison, WI) containing 1 L/mL Benzonase® Nuclease, 5 (iL/mL Protease Inhibitor Cocktai! Set n, and 0.033 L/mL rLysozyme following the manufacturer's protocol. All subsequent purification steps were carried out at 4°C. The cell extract was centrifuged for 20-30 minutes at 15,000 x g to remove tlie cell debris. A 20-25 niL aliquot of the cell free extract was applied to a 45 mL column of GE Healthcare Chelating Sepharose" Fast Flow resin (nickel (11) form) (GE HeJilthcare, Piscataway, NJ) that had been previously equilibrated with 100 mM potassium jihosphate containing 200 mM sodium chloride and 50 M PLP. To generate the nickel form of the resin, the resin was washed with 150 mL of 200 mM nickel (II) sulfate hexahydi-ate and then with 150 mL of distilled water. After loading the sample, the column was washed/eluted with 150 mL of the equilibration buffer containing 25 mM imidazole, 150 mL of the equilibration buffer containing 50 mM imidazole and 150 mL of the etjuilibration buffer containing 500 mM imidazole. The HIS6-tagged protein eluted in the last wash. The 500 mM imidazole wash was concentrated with MiUipore/Amicon Centricon P!us-70 centrifugal filter devices (MWCO 10 kDa) (Millipore, Billerica, MA) to 15-20 mL according to the manufacturer's instructions. The imidazole and sodium chloride were removed by passage tiirough disposable GE Healthcare PDIO columns (GE Healthcare, Piscataway, NJ) (2.5 mL sample per colunm) previously equilibrated with 100 mM potassium phosphate, pH 7.8 containing 0.5 |iM PLP. The purified aminotransferase was eluted with 3.5 mL per column of the same buffer.<br>
[0838]	The protein concentration of each fraction was determined using the Pierce BCA<br>
assay kit (Pierce, Rockford, IL) with BSA as the protein standard. The purity of each fraction and the level of expression in the cell free extract fraction were determined using a Bio-Rad Experion microcapillary chip system (Bio-Rad, Hercules, CA) or using Bio-<br>
 <br>
Rad 4-15% SDS-po!yacryIamide gradient gels (Bio-Rad, Hercules, CA) run in a Mini PROTEAN® 3 cell apparatus. The protein was visualized in the polyacrylamide gels using Bio-Rad Bio-Safe G-250 Coomassie stain (Bio-Rad, Hercules, CA) and destained with water. Typically this procedure produces - 50 rog of enzyme irom 200 mL of overnight culture that is 85-90% pure as judged by the Experion software or from analysis of the SDS-PAGE gels. Aliquots (1-5 mL) of the purified enzyme were stored at-80° C untiJ use.<br>
[0839]	The SEQ ID NO:276 aldolase (cloned in Example 22) was purified as the HIS-<br>
tagged protein as described in Example 23.<br>
[0840]	The preferred metal cofactor for the SEQ ID NO:276 aldolase was determined by<br>
screening a variety of divalent metals. The reactions were set up anaerobicaJly in 10 mL serum bottles with 7 mL final volumes. A bulk solution consisting of 100 mM potassium phosphate (pH 7.8), 200 mM sodium pyruvate, 0.05 mM PL? and 0.01% (v/v) Tween 80 was prepared to a final volume of 48.8 mL and sparged with nitrogen for 30 minutes. D-Tryptophan (143 mg; final concentration of 100 mM) was dispensed into seven 10 mL serum vials. To each of the vials was added 0.014 mL of a 2 M stock solution of a divalent metal cation, prepared from the chloride salt (final concentration of 4 mM). For the negative control, 0.014 mL of dHaO was added. The serum viaJs were capped with rubber septa and spaied with nitrogen via 16-18 gauge needles. Under anaerobic conditions, 5.625 mL of the anaerobic bulk solution was added to each anaerobic serum bottle. Subsequently, the B. sphaericus D-alanine aminotransferase and the SEQ ID NO:276 aldolase were added anaerobically to each serum bottle to a final concentration of 2 mg/mL and 0.05 mg/mL, respectively. The solutions were incubated at room temperature with gentle mixing for 18 hours. Final monatin was analyzed according to the methods described in Example I using the Liquid Chromatography-Post Column Fluorescence Detection of Amino Acids method. (Table 27).<br>
 <br>
 <br>
 <br>
10841]	The reaction conditions for the SEQ ID NO:276 aldolase were further investigated<br>
with a two-level fi-actional factorial experiment designed using the statistical software Design Expert 7.0.0 (Stat-Ease, Inc. Minneapolis, MN). The screening design consisted of a single block of five factors at two \evels with four centerpoints (20 runs total). The five factors to be optimized were the metal cofactor concentration, reaction pH, Tween® 80 concentration, pyruvate to tryptophan ratio, and the aldolase concentration (Table 28).<br>
[0842]	Conical polypropylene tubes (14 mL) containing 143 mg of D-tryptophan were<br>
de-oxygenated in an anaerobic glove box (Coy Laboratoiy Products, Inc; Grass Lake, MI) overnight. Stock solutions of 2 M MgCl2; 1 M potassium phosphate at pH 7.0, 7.75, and 8.5; 10% (v/v) Tween® 80; 2 M sodium pyruvate, and 10 mM PLP (pyridoxal 5'-phosphate) were prepared in degassed water and equilibrated in the anaerobic glove box ovemighL Stock solutions of purified B. sphaericus D-aJanine aminotransferase and the SEQ ID NO:276 aldolase were thawed on ice and used in the anaerobic glove box immediately. Stock solutions were added to the 14 mL conical tubes containing the D-tryptophan to obtain the concentrations determined by the statistical design (Table 28). Degassed water was added to each tube to bring the fmal volume, along with the enzyme additions, to 7.0 mL. The tubes were incubated at room temperature in the anaerobic glove box with gentle mixing for up to 24 hours. Monatin concentration and isomeric purity were analyzed according to the methods described in Example 1 using the Liquid Chromatography-Post Column Fluorescence Detection of Amino Acids method and the LC/MS/MS Multiple Reaction Monitoring for the Determination of the Stereoisomer Distribution of Monatin in in vitro and in vivo Reactions method (FDAA derivatization method).<br>
 <br>
 <br>
 <br>
ratio and aldolase concentration were the significant factors affecting monatin titer,<br>
isomeric purity and carbon yield. A desirability graph was generated using the Design<br>
Expert software in which the factors were varied in order to maximize the goals of<br>
highest monatin titer and highest isomeric purity under conditions of excess pyruvate.<br>
The reactions conditions indicated as optimiun were 1 mM MgClj, pH &gt;8.0, 0.01% (v/v)<br>
Tween® 80, and 0.01 mg/mL SEQ ID NO:276 aldolase. This is a 5 fold reduction in the<br>
typical amount of aldolase utilized, as well as a 4-foId reduction in the amount of divalent<br>
metal typically used.<br>
[0844]	Additional experiments were performed to determine the optimum pH range for<br>
the reaction process. Stock solutions of 1 M EPPS bIffer were prepared at increments of 0.2 pH units between pH 7.0 and 9.0. These solutions were degassed and equilibrated in the anaerobic glove box overnight. Polypropylene tubes (14 mL) containing 143 mg of D-tryptophan were de-oxygenated in an anaerobic glove box ovemighL Stock solutions of 2 M MgCli, 10% (v/v) Tween® 80, 2 M sodium pyruvate and 10 mM PL? were prepared in degassed water and equilibrated in the anaerobic glove box. Preparations of purified B. sphaericus D-alanine aminotransferase and SEQ ID NO:276 aldolase were thawed on ice and used immediately in the anaerobic glove box. The stock solutions were added to the 14 mL conical tubes to give a final concentration of 100 mM EPPS, 200 mM pyruvate, 100 mM tryptophan, 1 mM MgCU, 0.01% (v/v) Tween® 80, 0.05 mM PLP, 2 mg/mL B. sphaericus D-alanine aminotransferase, and 0.01 mg/mL SEQ ID NO:276 aldolase in a total volume of 7 mL per tube. The reactions were incubated at room temperature in the anaerobic glove box with gentle agitation for 22 hours. Samples<br>
 <br>
were removed and analyzed for monatm as described in Example 1 using the LC/MS/MS multiple reaction monitoring method (Table 29).<br>
 <br>
[0845]	The results indicated that monatin formation increased with increasing pH<br>
between 7.0 - 8.0. Monatin formation reached a maximum in the range of pH 8.0 - 8.2,<br>
and decreased above pH 8,4.   Additionally, the isomeric purity of monatin decreased<br>
above pH 8.4.<br>
[0846]	Further reaction optimization was done using the aldolase of SEQ ID NO: 276<br>
(0.01 mg/mL), 2 mg/mL of the T243N 4978 DAT (untagged, from Example 26), 1 mM MgCli, 200 mM sodium pyruvate, 0.05 mM PLP, 0.01% Tween-80, and 100 mM D-tryptophan at pH 8.5. The cells used to produce the aldolase and the D-aminotransferase were broken open in 50 mM EPPS, pH 8.4 using a Microfluidics homogenizer (Microfluidics, Newton, MA) (3 passes at 20,000 psi). The cell debris was removed by centrifugation (20,000 x g for 30 minutes) and the clarified cell extracts were used in the enzymatic reactions. No additional buffer «s utilized, but the reaction mixtures were adjusted to pH 8.5 using sodium hydroxide and flushed with nitrogen prior to addition of enzyme. Two-hundred fifty mL reactions were carried out in 0.7 L Sixfor agitated fermenters (Infors AG, Bottmingen, Switzerland) at 30 °C using nitrogen in the headspace. Potassium phosphate was added to a final concentration of 0, 5, 10, 20, and 50 mM. The addition of 5-10 mM phosphate was found to be optimal, producing 3.5 g/L monatin (quantitated by LC/MS/MS as described in Example 1).<br>
 <br>
Example 25 Cloning of A Novel Bacillus D-Amino Acid Aminotransferase<br>
[0847]	A Bacillus D-amino acid aminotransferase ("DAAT" or "DAT") (EC 2.6.1.21,<br>
also known as D-alanine aminotransferase or D-aspartate aminotransferase) was produced<br>
recombinantiy. This aminotransferase was used m coupled assays with aldolases for<br>
production of R,R monatin. This aminotransferase enzyme is homologous to D-<br>
aminotransferases described previously for production of monatin (U.S. Published<br>
Application No. 20040063175 and U.S. Published AppHcation No. 20050282260). The<br>
organism ATCC 4978—Bacillus sphaericus originally deposited as Bacillus rotans-was<br>
ordered from the ATCC and used to prepare genomic DNA. Degenerate primers were<br>
designed in the regions of protein sequence conservation of known Bacillus D-<br>
aminotransferases and used for polymerase chain reaction ("PCR") amplification of<br>
internal DAAT gene sequence from the ATCC strain mentioned above. Genome walking<br>
was performed using the BD Genome Walker™ Universal Kit (Clontech, Mountain View,<br>
CA). Sequence analyses (Agencourt BioScience Corporation, Beverly, MA) verified a<br>
frilMength coding sequence for the DAAT gene from ATCC 497S. The DNA sequence<br>
of the DAAT gene from ATCC 4978 is SEQ ID NO:357 and is shown below. The gene<br>
of SEQ ID NO:357 may be amplified by standard PCR protocols and cloned using<br>
standard recombinant DNA techniques. The gene of SEQ ID NO;357 may also be<br>
reconstructed by any method known to a person of ordinary skill in the art, such as<br>
assembly PCR methods known to one skilled in the art.<br>
10848]	The ATCC 4978 DAAT DNA sequence is:<br>
atgagttata gcttatggaa tgaccaaatt gtgaatgatg aagaagtagt agttgataag gaggaccgtg gctatcaatt tggcgatggt gtttatgaag ttgtaaaagt atataacggt gaattattta cagcggagga gcatgtcgat cgtttttacg cgagtgctga aaaaattcgc gttacgatcc cttatacaaa agacaaattg catcaattat tgcatcagtt agttgaaatg aataaagttc aaacaggaca tatttatttc caaattacgc gtggtgcagg ccctcgtaat catattttcc ctggtgatga agtgaagcca gtattaacag gtaataccaa ggaaaatcca cgtcccgtag caaactttga aaaaggtgtg aaagcaacat ttgtagaaga cattcgttgg ttacgctgtg acattaaatc attaaattta cttggtgcgg tacttgctaa acaagaagca catgaaaaag gatgctatga agcggtttta catcgtgatg aaatcgtaac agaaggctct tcttcaaata tttatggaat taaagatggc gtattataca cacatccagc<br>
 <br>
gaataacttc atcttaaatg gtattacacg icaagtaatc attaaatgtg ctgctgaaat tggcttacca gtgaaggaag aagcaatgac aaaaactcag cttcttgcaa tggatgaagt gattgtttca tcaacgactt cagaagtaac gccaattatc gacatagatg gaacagtaat tggtgcgggt aaaccgggtg actggacacg taaattacaa gcacaatttg atacgaaaat cccaaaaggt aticgcgcat aa<br>
(SEQ ID NO: 357)<br>
[0849]	The amino acid sequence of the DAAT gene from ATCC 4978 as encoded by the<br>
above DNA sequence is SEQ ID NO:358 and is shown below:<br>
Met Ser Tyr Ser Leu Trp Asn Asp Gin He Va! Asn Asp GIu Glu Val Val Va] Asp Lys GIu Asp Arg Gly Tyr Gbi Phe Gly Asp Gly Val Tyr Glu Val Val Lys Val Tyr Asn Gly Glu Leu Phe Thr Ala GIu Glu His Val Asp Arg Phe Tyr Ala Ser Ala Glu Lys He Arg Val Thr lie Pro Tyr Thr Lys Asp Lys Leu His Gin Leu Leu His Gin Leu Val Glu Met Asn Lys Val Gin Thr Gly His He Tyr Phe Gin He Tbr Arg Gly Ala Gly Pro Arg Asn His He Phe Pro Gly Asp Glu Val Lys Pro Val Leu Thr Gly Asn Thr Lys Glu Asn Pro Arg Pro Val Ala Asn Phe Glu Lys Gly Val Lys Ala Thr Phe Val Glu Asp He Arg Trp Leu Arg Cys Asp He L&gt;'s Ser Leu Asn Leu Leu Gly Ala Val Leu Ala Lys Gin Glu Ala His Glu Lys Gly Cys Tyr Glu Ala Val Leu His Arg Asp Glu He Val Thr Glu Gly Ser Ser Ser Asn He Tyr Gly He Lys Asp Gly Val Leu Tyr Thr His Pro Ala Asn Asn Phe He Leu Asn Gly IJe Thr Arg Gin Val He He Lys Cys Ala Ala Glu He Gly Leu Pro Val Lys Glu Glu Ala Met Thr Lys Thr Ghi Leu Leu Ala Met Asp Glu Val He Val Ser Ser Thr Thr Ser Glu Val Thr Pro He lie Asp He Asp Gly Thr Val He Gly Ala Gly Lys Pro Gly Asp Trp Thr Arg Lys Leu Ghi Ala Gin Phe Asp Thr Lys He Pro Lys Gly He Arg Ala<br>
(SEQ ID NO: 358)<br>
[0850]	The novel D-aminotransferase obtained from strain ATCC 4978 has a protein<br>
sequence that has distinct amino acid residue changes when compared to the B.<br>
sphaericus ATCC 10208 published D-aminotransferase sequence.    The DAAT from<br>
ATCC 4978 has only 72% identity with the DAAT from B. sphaericus (ATCC 10208).<br>
While this sfrain is currently listed as B. sphaericus in the ATCC, it was deposited as B.<br>
rotans.   Based on the sequence alignments and the highlighted differences between this<br>
 <br>
novel DAAT and the DAAT from B. sphaericus, a number of candidate residues are identified that can be evaluated tbi' their role (individually or in combination) in increasing DAAT activity for R,R monatin biosynthesis, in these, as well as other DAAT sequences.<br>
Example 26<br>
Characterization of mutants of D-aminotransferase from ATCC 4978<br>
Experimental overview<br>
[0851]	The novel D-aminotransferase gene (described in Example 2S) from Bacillus<br>
strain ATCC 4978 was mutagenized using site-directed methods. The mutant genes were expressed and assayed for activities of interest for monatin production pathways, especially when coupled with one or more aldolases.<br>
(0852]	In addition to the ideas listed in Example \6 for site directed mutagenesis targets,<br>
other ideas were developed b)' the actual docking of R-MP into the active site of the YM-1 ciystal structure primary amino acid sequence alignments were used to determme if the 4978 aminotransferase protein was likely to have similar structural characteristics in that region. It was expected that the following additional mutations would be beneficial (using the 4978 aminotransferase's amino acid numbering). It was thought that mutagenesis of alanine 153 to arginine would stabilize the second carboxyl group of the substrate (R-MP). This change is likely to increase steric hindrance, so to compensate, the serine residues at positions 181 and 182 were changed to alanine or glycine. It was also hypothesized that one could introduce an arginine at position 180, 181, or 182 and convert one or more of the other serine residues to alanine or glycine to make room for the bulkier side chain of arginine. The phenylalanine at amino acid 200 is spatially close to where R.-MP is predicted to dock into the active site and there is a large amount of variability in this residue amongst the D-aminotransferases that catalyze monatin transamination fairly well. It was thought that amino acid modifications at this position could be useful. Mutation of leucine 151 to phenylalanine was predicted to potentially improve interactions with the indole ring of the substrate.<br>
[0853]	Based upon literature, it was hypothesized that mutation of threonine 243 to<br>
asparagine may improve R-MP selectivity for transamination reactions. Likewise, it was<br>
 <br>
thought that mutagenesis of asparagine 100 to alanine may improve the specific activity of the enzyme for monatin transamination reactions (Ro, ei al., FEBS Lett 398:14\A45, (1996); Sugio, S, et al.. Biochemistry 5:9661-9669, (1995); EP1580268).<br>
[0854]	Lee et al. characterized mutants of the 141-144 region (loop) and found that D-<br>
aminotransferases with the EYcY rather than the LRcD (which is native to our protein) tend to have a lower Km for dicarboxylic acid substrates. (Lee SG, Hong SP, Song JJ, Kim SJ, KwaJf MS, Simg MH. Functional and structural characterization of thermostable D-amino acid aminotransferases from Geobacillus spp. Appl Environ Microbiol. 2006 Feb;72(2): 1588-94). Because MP is a dicarboxylic acid substrate, similar to alpha-keto glutarate, and the concentrations of MP are fairly low in a typical monatin production reaction mixture, a decreased K could potentially help the activity of a mutant DAT for monatin production.<br>
[0855]	Below, methods are described for creating the 4978 D-aminotransferase mutants,<br>
s well as assay results using these mutants.<br>
Mutagenesis<br>
[0856]	The primers for mutagenesis were designed following the suggestions listed in the<br>
Stratagene Multi-Change kit (Stratagene, La JoUa, CA). The primers were 5'-phosphorylated. Mutagenesis was done using the Stratagene Multi-Change kit (Stratagene, La JoUa, CA) following the manufacturer's instructions. The templates used for mutagenesis were either the pET30 (untagged) or pET28 (tagged) 4978 DAT constructs described in Example 25. The primers are listed below in Table 30:<br>
Table 30<br>
Mutant Name    Amino acid        Primer<br>
change<br>
4978-22	T243N	GTGATTGTTTCATCAACGAATTCAGAAGTAACG<br>
CC (SEQ ID NO:359)<br>
10	T243R	GTGATTGTTTCATCAACGCGTTCAGAAGTAACG<br>
CC (SEQ ID NO:360)<br>
7	T243S	GTGATTGTTTCATCAACGAGTTCAGAAGTAACG<br>
CC(SEQIDNO:36I)<br>
19	T243A	GTGATTGTTTCATCAACGGCTTCAGAAGTAACG<br>
CC (SEQ ID NO:362)<br>
IS	N1OOA	GTGCAGGCCCTCGTGCTCATATTTTCCCTGG<br>
(SEQIDNO:363)<br>
B	T243Q	GAAGTGATTGTTTCATCAACGCAGTCAGAAGT<br>
 <br>
AACGCCAATTATC (SEQ ID NO:364) T243N/N100     above primers used together<br>
[0857]	E.  coli XLIO-Gold cells (Sti-atagene, La Jolla.  CA) were transformed, and<br>
resultant purified plasmid preparations were sequenced to verify ihaX the correct mutations were incorporated.<br>
Expression and assay<br>
[0858]	Plasmid DNA preparations (;ontaining the correct mutants or the wildtype 4978<br>
DAT were transformed into the E. coli expression host BL21(DE3) (Novagen, Madison, WI). The cultures were grown using the protocols described above, the plasmids were isolated using Qiagen miniprep kit (Qiagen, Valencia, CA) and analyzed by restriction digestion, as described above, to confirm the presence of an insert.<br>
[0859]	Induction of the DAAT gene was typically performed in LB medimn containing<br>
kanamycin (50 jxg/mL). The cells were grown to an ODeoonm of 0.4-0.8 at 37 °C, induced with 0.1 mM IPTG (isopropyl thiogalacatoside) and sampled at 3-4 hours post induction.<br>
[0860]	Cellular extracts were prepared with BugBuster Reagent and Benzonase Nuclee<br>
(Novagen, Madison, Wl). One ml assays were performed at 30 "C with gentle shaking and contained 10.2 mg D-tryptophan, 0.05 mM PLP, 4 mM MgCb, 100 mM potassium phosphate buffer pH 7.5, approximately 50 p-g of aldolase, 200 mM pyruvate, and 0.150-0.5 mg/mL D-aminotransfere supplied as cellular extracts. Total protein assays were done using the Bio-Rad total protein kit (Coomassie) (Bio-Rad, Hercules, CA) or the Pierce BCA kit (Pierce, Rockford, IL), and percent expression of the D-aminotransferase was estimated by SDS-PAGE or the Bio-Rad Experion Automated Electrophoresis System (Bio-Rad, Hercules, CA). Samples were taken at 3 hours and overnight.<br>
[0861]	The R-specific aldolase of SEQ ID NO: 28 was used in assays with approximately<br>
0.150 mg/mL D-aminotransferase.<br>
[0862}	The first assays showed that the following mutants (untagged) had transamination<br>
activity (in order of highest to lowest): T243N, T243S, T243N/N100A, NIOOA. It was also noted that the T243N appeai-ed to raise the stereo-purity of the R,R monatin produced. Assays were repeated using purified Comamonas testosteroni ProA aldolase (100 |ig/mi) and 0.50 mg/ml of D-aminotransferase mutants (untagged, supplied as<br>
 <br>
cellular extract). Samples were ten at 2 hours and overnight. The results for the active proteins are shown below, duplicate results were averaged. The % R,R monatin was determined by peak area on reversed phase HPLC, and then measured using the FDAA derivatization method described in Example 1. In column 3 of Table 31, additional FDAA-derivatization LC/MS/MS analysis as described in Example I was done for some of the reactions and those results are shown in the parentheses. Only R,R and S,R monatin are produced from D-tryptophan. The T243R mutant did not appear to produce monatin under the conditions tested, and the T243A mutant produced very low levels of monatin.<br>
 <br>
[0863]	Although the assays were performed estimating percent D-aminotransferase using<br>
Bio-Rad Experion software, it is clear that the T243S and T243N mutants had increased activity compared to the wildtype enzyme. The T243N mutant also provided an additional benefit of increasing dramatically the % R,R monatin formed. This enzyme has an increased preference for R-MP as compared to S-MP in transamination reactions. The N1OOA mutant did not increase activity alone or in combination with T243N contrary to what was suggested in the literature. A V34A site directed mutant of the untagged 4978 DAT was also created using similar methods, as described above. The V34A sited directed mutant was found to have significantly less activity than the wild-type enzyme under the conditions tested.<br>
[08641	Another point of interest in the initial assays was that the wildtype enzyme<br>
appeared to have more activity when it was produced with an N-terminal His-tag.<br>
 <br>
Subsequent mutagenesis was done on the tagged version of the gene. Additionally, the<br>
most promising mutants above were subcloned into pET28b that has an N-terminal His-<br>
tag. These were purified using Novagen HIS-bind columns and the manufacturer's<br>
protocol with the recommended buffers (Novagen, Madison, WI). The buffer of the<br>
eluent fractions was exchanged, using GE Healthcare PDIO columns (GE Healthcare,<br>
Piscataway, NJ), to the buffer used in the assays<br>
[0865]	One ml assays with purified D-aminotransferase (0.5 mg/ml) and purified R-<br>
specific aldolase SEQ ID NO:276 (50 )ag/ml) were conducted at 30 "C with gentle shaking and contained 10.2 mg D-tryjjtophan, 0.05 mM PLP, 200 mM pyruvate, 4 mM MgCb, and 100 mM potassium phosphate buffer pH 7.5. Duplicate samples were incubated for 2 hours and overnight. As a positive control, the Bacillus sphaericus DAT (cloned in Example 7) was used in the same assays. The results are presented in Table 32:<br>
 <br>
[0866]	The data above show that the T243N mutant clearly produces the highest amount<br>
of monatm at 2 hours. As time mcreases, the ratio of T243N mutant to B. sphaericus DAT positive control is reduced. This result suggests that the T243N mutant is not as stable during the monatin reaction as the B. sphaericus DAT. When assayed under similar conditions, the T243S (purified tagged) enzyme had similar levels of activity to the T243N mutant; however, the percent R,R monatin produced was lower (97.2 % at both 2 h and overnight). The T243N/N100A mutant had less activity than the T243N mutant. However, both T243S and T243N/N100A had higher activity than the wildtype 4978 DAT.<br>
 <br>
[0867]	Transamination assays were performed to determine which reaction rates were<br>
improved when using the T243N mutant in place of the B. sphaericus DAT. One-haJf mL assays were performed at 30 °C taking time points at I holr, 2 hours, and 5 hours. The assays contained 25 mM monatin or D-tryptophan, 25 mM pyruvate, 100 mM potassium phosphate pH 7.5, 50 M PLP, and 0.1 mg D-aminotransferase (tagged, purified). In the case where less than 100 .g DAT was used, the amount of alanine was normalized to 100 |a,g of D-aminotransferase. Samples were treated with formic acid and analyzed by LC-OPA for the presence of the coproduct, alanine. The results are shown in Tables 33 and 34.<br>
 <br>
 <br>
[6H68]	For Ihe D-tryptophan reactions, the results show that some of the enzymes had<br>
reached equilibrium at 2 hotirs.  The R,R monatin reactions are clearly rate-limiting and<br>
improvements to this activity have more of an impact on monatin production rates from<br>
D-tryptophan.<br>
[0869]	Further assays were done to examine the stability of the T243N 4978 DAT<br>
mutant. The wildtype enzyme also loses activity over time. Example 27 describes<br>
methods to improve the stability of the T243N D-aminotransferase mutant. When freshly<br>
prepared untagged and tagged versions of the T243N mutant are prepared and compared<br>
for activity, it was found that the untagged version had a better temporal stability, making<br>
it overall a better version of the enzyme to use in monatin production reactions.<br>
[0870]	Additional mutants of 4978 DAT were made by methods commonly known to<br>
those skilled in the art. However, these mutations all resulted in protein that was insoluble under the conditions that they were prepared, and thus could not be assayed for activity. The mutations that resulted in insoluble protein were;<br>
S180A/S18IA/S182R;<br>
L151F;<br>
V34G<br>
S181R<br>
A153R/S181A/S182A;<br>
A153R/S182A;<br>
A153R/SI82G;<br>
S180R/S181A/S182G;<br>
S180R/S181A/S182A; SI80R/S18IG/SI82G; S180G/S181R/S182G; and<br>
S180A/S181R/S182A.<br>
Additional mutagenesis<br>
[0871]	To create the F200M 4978 DAT mutant, the wildtype 4978 DAT open reading<br>
frame from Example 25 (tagged) was amplified with primers 73 and 80 (below) and PftiTurbo DNA Polymerase (Stratagene, La Jolla, CA) and cloned into pCRII-Blunt (hivitrogen, Carlsbad, CA). Its sequence was verified (Agencourt, Beverly, MA). The 5' and 3' regions were amplified using primers 80 and 96 and 99 and 103, respectively. The<br>
 <br>
amplified DNA was then gel purified using Qiagen QIAquick Gei Extraction Kit (Qiagen, Vaiencia, CA). They amplified DNA was subjected again to PCR using primers 80 and 99. The amplified DNA was gel purified as described above and cloned into pCRII Blunt and its sequence verified. The DAT' open reading fi-ame was subcloned as an Ndel/Xhol restriction digest fragment into pET28b.<br>
Table 35<br>
Primer	Sequence<br>
Number<br>
73        CATATGAGTTATAGCTTATGGAATGACCAAATTGTGAATG (SEQ ID NO:365)<br>
80        CTCGAGTGCGGCCGCAAGCTTGTCGACGGAGCTC (SEQ ID NO:366}<br>
96        AATATTTATGGAATTAAAG.TGGCGTATTATACACACATCCAGCGAATA<br>
ACATGATCTTAAATGGTATTACACGTCAAGTAATCATTAAATGTGC<br>
(SEQIDNO:367) 99        GGCCAGTGAATTGTAATACGACTCACTATAGGGC (SEQ ID NO:368) 103       CGCCATCTTTAATTCCATAAATArTTGAAGAAGAGCCTTCTG (SEQ ID<br>
NO:369)	'<br>
[0872]	TTie following primers were designed for additional site-directed mutagenesis<br>
using the QuikChange Multi Site-Directed Mutagenesis Kit (Stratagene, La Jolia, CA). Mutagenesis was done using the Stratagene Multi-Change kit (Stratagene, La Jolla, CA) following the manufacturer's instructions. The template used for mutagenesis was the pET28 (tagged) 4978 DAT construct described in Example 25. A double mutant was also created using the F200Y mutant as template and doing an additional round of mutagenesis with the T243N (listed above) primer.<br>
Table 36<br>
Mutant	Oligo<br>
141 -LRcD-144 -&gt;       GCAACATTTGTAGAAGACATTCGTTGGGAATACTGTTACATTA<br>
EYcY	AATCATTAAATTTACTTGGTGCG (SEQ ID NO:370)<br>
F200Y	GTATTATACACACATCCAGCGAATAACTACATCTTAAATGGTAT<br>
TACACGTCAAG (SEQ ID NO:371)<br>
S244K	GCAATGGATGAAGTGATTGTTTCATCAACGACTAAAGAAGTAA<br>
CGCCAATTATCGACATAGATG (SEQ IDNOi372) 243-TS-244 -&gt; NK  GCAATGGATGAAGTGATTGTTTCATCAACGAATAAAGAAGTAA<br>
CGCCAATTATCGACATAGATG (SEQ IDNO:373) 243-TS-244 -&gt; NR  GCAATGGATGAAGTGATTGTTTCATCAACGAATCGTGAAGTAA<br>
CGCCAATTATCGACATAGATG (SEQ IDNO:374)<br>
 <br>
[0873]	The mutant coding regions were verified by DNA sequencing (Agencourt).  The<br>
sequence-verified plasmids were transformed into BL21(DE3) cells (Novagen, Madison, WI).<br>
Expression and Assay<br>
[0874]	Cultures containing 100 ml LB with 50 |j.g/ml kanamycin in a 500 ml baffled flask<br>
were inoculated with one ml of an overnight culture and grown at 37 'C to an optical density {at 600 nm) of approximateiy 0.6. Production of the protein was induced by IPTG at a final concentration of 1 mM. Cells were incubated at 30 "C for 4.5 hours after the addition of tlie IPTG. Ceils were centrifuged and fi-ozen at -80 "C. Cells were disrupted (prepared using Novagen BugBuster reagent (Novagen, Madison, WI) contaming 1 pL/mL benzonase nuclease, 5 jiL/mL protease inhibitor cocktail II, and 0.033 L/mL rLysozyme following Novagen's recommended protocol) and analyzed by SDS-PAGE. Mutants (141-LRcD-144 -&gt; EYcY) and (243-TS-244 -&gt; NR) resulted in insoluble proteins under the conditions in which they were prepared. Mutant 243-TS-244~&gt; NK did not have quantifiable activity under the conditions tested, and is probably a weak activity enzyme in comparison to wildtype as is the S244K mutant.<br>
[0875]	His-tagged proteins were pxurified as follows.    HIS-bind columns (Novagen,<br>
Madison, WI) were equilibrated with 10 mL of 100 mM potassium phosphate, pH 7.8, containing 200 mM NaCl and 50 )iM PLP. Cell-fi-ee extracts were loaded on the column. The columns were washed with 10 mL of equilibration buffer, 10 mL equilibration buffer containing 25 mM imidazole, and 10 mL equilibration buffer containing 50 mM imidazole. Proteins were eluted with 5 ml equilibration buffer containing 500 mM imidazole. Proteins were desalted using PDIO columns which were equilibrated in 100 mM potassium phosphate, pH 7.8 containing 50 pM PLP. The purified proteins were concentrated and quantified using the Bradford Assay (Bio-Rad, Hercules, CA).<br>
10876}	The  D-aminotransferase mutants were assayed using  500 jig/ml  of the D-<br>
aminotransferase, 50 ig/ml of the aldolase of SEQ ID NO:276, 4 mM MgCh, 50 mM potassium phosphate pH 8, 200 mM sodiimi pyruvate, 0.05 mM PLP and 20.4 mg/ml D-tryptophan for assay conditions. The final volume was 1.25 ml. Samples (200 fil) were taken after 0.5,1, 2 and 14 hours and frozen until the experiment was complete. Samples were filtered, diluted 1 to 10, and analyzed by LC/MS/MS as described in Example 1.<br>
 <br>
(0877J	The wildtype 4978 D-aminotransferase from Example 25 was used as a reference<br>
for percent relative activity. Table 37 shows relative activity of each mutant at each time point.<br>
 <br>
Example 27<br>
Stabilization of the T243N Mutant of the D-aminotransferase from strain ATCC<br>
4978<br>
|0879]	As shown in Example 25, the initial activity of the T243N mutant DAT is<br>
significantly higher than the B. sphaericus DAT, but activity decreases more rapidly. Additional experiments, using the anaerobic protocol described below, indicated that the<br>
 <br>
initial activity of the T243N mutant DAT was up to 8-fold higher than the B. sphaericus DAT, however the activit}' decreased rapidly even under the anaerobic conditions. The following studies were done to try to maintam the higher activity for an extended period of time.<br>
[0880]	The T243N mutant of the D-aminotransferase from strain 4978 (described in<br>
Example 25) and the S. meliloti HMG aldolase were purified as the HISs-tagged proteins as described below. The SEQ ID NO:276 aldolase was purified as described in Example 23.<br>
Purification of the DAT from ATCC 4978 fT243N mutant<br>
[0881]	The T243N mutant of the D-aminotransferase from ATCC strain 4978 with an<br>
amino-terminal HlSe-purification tag (described in Example 26) was produced using the EMD Biosciences Overnight Express System II (solutions 1-6) (Novagen, Madison, WI) ■--intaining 50 fig/mL kanamycin in shake flasks. This expression system induces the iression of IPTG-inducible systems without the need to monitor cell growth. After iculation of 200-mL aliquots of the medium (in 1 L flasks) from either liquid cultures or plates of the E. coli BL21(DE3) host carrying the gene for the T243N mutant D-aminotransferase from ATCC strain 4978 on the plasmid pET28h, the cultures were incubated at 30°C overnight with shaking at 225 rpm. When the OD600 had reached a minimum of 6, the cells were harvested by centrifiigation and washed once with buffer.<br>
[0882]	Cell free extract was prepared using EMD Biosciences BugBuster® (primary<br>
amine-free) Extraction Reagent (Novagen, Madison, WI) containing 1 iL/mL Benzonase® Nuclease (Novagen, Madison, WI), 5 fiL/mL Protease Inhibitor Cocktail Set II (Calbiochem - Novabiochem Corp., San Diego, CA), and 0.033 \iLlmL rLysozyme'" (Novagen.. Madison, WI) followmg the manufacturer's protocol. All subsequent purification steps were carried out at 4°C. The cell extract was centrifuged for 20-30 minutes at 15,000 x g to remove the cell debris. A 20-25 mL aliquot of the cell free extract was applied to a 45 mL column of GE Healthcare Chelating Sepharose™ Fast Flow resin (nickel (II) form) (GE Healthcare, Piscataway, NJ) that had been previously equilibrated with 100 mM potassium phosphate containing 200 mM sodium chloride and 50 nM PLP. To generate the nickel form of the resin, the resin vj&amp;s washed with 150 mL of 200 mM nickel (II) sulfate hexahydrate and then with 150 mL of distilled water. After<br>
 <br>
loading the sample, the column was washed/eluted with 150 mL of the equilibration buffer containing 25 mM imidazole, 150 mL of the equilibration buffer containing 50 mM imidazole and 150 mL of the equilibration buffer containing 500 mM imidazole. The HlS6-tagged protein eluted in the last wash. The 500 mM imidazole wash was concentrated with Millipore/Amicon Centricon Plus-70 centrifugal filter devices (MWCO 10 kDa) (Millipore, Billerica, MA) to 15-20 mL according to the manufacturer's instructions. The imidazole and sodium chloride were removed by passage through disposable GE Healthcare PDIO columns (GE Healthcare, Piscataway, NJ) (2.5 mL sample per column) previously equilibrated with 100 mM potassium phosphate, pH 7.8 containing 0.5 jxM PLP. The purified aminotransferase was eluted with 3.5 mL per column of the same buffer. The protein concentration of each fraction was detennined using the Pierce BCA assay kit (Pierce, Rockford, IL) with BSA as the protein standard.<br>
(0S83J	The purity of each iraction and the level of expression in the cell fee extrnct<br>
fi-action were determined using a Bio-Rad Experion microcapiUary chip system (Bio-Rad, Hercules, CA) or using Bio-Rad 4-15% SDS-polyacrylamide gradient gels (Bio-Rad, Hercules, CA) run in a Mini PROTEAN® 3 cell apparatus. The protein was visualized in the polyacrylamide gels using Bio-P.ad Bio-Safe G-250 Coomassie stain (Bio-Rad, Hercules, CA) and destained with water. Typically this procedure produces ~ 20 mg of enzyme fi-om 200 mL of ovemigtit culture that is 85-90% pure as judged by the Experion software or from analysis of the SDS-PAGE gels. Aliquots (1-5 mL) of the purified enzyme were stored at -80° C until use.<br>
[0884]	The purification of the SEQ ID NO:276 aldolase is as described in Example 23.<br>
Purification of the S. meliloti HMG aldolase<br>
[0885]	The S.  meliloti HMG aldolase with an ammo-terminal HlSg-purification tag<br>
(cloning described in U.S. Published Application No. 20040063175 and WO 03091396 A2) was produced by induction of cultures grown in Luria-Bertani broth containing 50 mg/L kanamycin with 0.2 mM IPTG. After inoculation of 800-mL aliquots of the medium from either liquid cultures or plates of the E. coli BL21(DE3) host carrying the gene for the S. meliloti HMG aldolase in pET30(Xa/LIC), the cultures were incubated at 37'C with shaking at 225 rpm. When the optical density reached an ODeoonm of 0.5-0.75, the IPTG was added and the cultures were incubated 30°C with shaking at 225 rpm for 4 hours. The cells were harvested by centritugation and washed once with buffer.<br>
 <br>
f0886]	To prepare cell free extract contaming the S. meliloti aldolase, the cells were<br>
suspended in 3-4 volumes of 50 mM EPPS (N-(2-hydroxyethyl)piperazine-N'-(3-propanesulfonic acid), pH 8.2 containing 100 mM NaCl and then disrupted using a Microfluidics homogenizer (Microfliiidics, Newton, MA) (3 passes at 18,000 psi), maintaining the temperature of the suspension at less than \S°C. The cell suspension was centrifuged for 20-30 minutes at 15,000-20,000 x g to remove the cell debris. The HlSe-lagged protein was purified using HMD Biosciences HIS-Bind® columns (Novagen, Madison, Wl) following the manufiicturer's recommended protocol with one exception: the columns were washed with a 1:1 solution of Bmding buifer:Wash buffer instead of the Wash buffer alone. The elution fraction was concentrated with MiUipore/Amicon 15 mL centrifugal filter devices (MWCO 5 kDa) (Millipore, Billerica, MA) to 7-10 mL according to the manufacturer's instructions. The imidazole and sodium chloride were removed by passage through disposable GE Healthcare PDIO columns (GE Healthcare, Piscataway, NJ) (2.5 mL sample per column) previously equilibrated with 50 mM EPPS, pH 8.2 containing 100 mM NaCl. The purified aldolase was eluted with 3.5 mL per column of the same buffer. The protein concentration of each fraction was determined using the Pierce BCA assay kit (Pierce, Rockford, EL) using BSA as the protein standard.<br>
[0887]	The purity of each fraction and the level of expression in the cell free extract<br>
fraction were determined using Bio-Rad 4-15% SDS-polyacrylamide gradient gels (Bio-Rad, Hercules, CA) run in a Mini PROTEAN® 3 cell apparatus. The protein was visualized in the polyacrylamide gels using Bio-Rad Bio-Safe G-250 Coomassie stain (Bio-Rad, Hercules, CA) and destained with water. Typically, this procedure produces -15-20 mg of enzyme from 800 mL of culture and is 85-95% pure. Aliquots (1-5 mL) of the purified enzyme were stored at -80° C until use.<br>
Monatin production assays<br>
[0888]	Conical polypropylene Uibes (14 mL) containing 143 mg of D-tryptophan were<br>
de-oxygenated in an anaerobic glove box overnight. Stock solutions of 1 M EPPS buffer (pH 8.2), 2 M MgCIi, 2 M sodium pyruvate and 10 mM PLP were prepared in degassed water and equilibrated in an anaerobic glove box overnight. Stock solutions of 10% (v/v) Tween 80, 1% (v/v) Tween 20, 1% (v/v) Triton X-100, 100% acetone, 100% ethanol and 50% (w/v) glycerol were equilibrated in die anaerobic glove box along with 0.7 g each of trehalose, inositol, sorbitol and erythritol in 2 mL microcentrifuge tubes. Preparations of<br>
 <br>
the purified enzymes were thawed on ice and used immediatefy in the anaerobic glove box. The stock solutions were added to the 14 mL conical tubes to give a final concentration of 100 mM EPPS, 200 mM pyruvate, 100 niM tryptophan, 1 mM MgCli, 0.05 mM PLP, 0.5 mg/mL D-aminotransferase, and 0.01 mg/mL of SEQ ID NO:276 aldolase or 0.05 mg/mL ofS. meliloti HMG aldolase. The proposed enzyme stabilizing components were added at various final concentrations (Tables 38 and 39) to bring the final reaction volume lo 7 mL per tube. The reactions were incubated at room temperature in the anaerobic glove bo&gt;: with gentle agitation for up to 24 hours. Samples were removed periodically and analyzed for monatin as described in Example 1 using the LCMSfMS multiple reaction monitoring method. The initial rates were calculated from the samples withdrawn between 0 and 3 h after the addition of the enzyme.<br>
 <br>
 <br>
improved the  stability of the T243N D-aminotransferase  over the lifetime of the experiment.<br>
Example 28<br>
A: Cloning of Pseudomonas putida KT2440 Broad-specificity Amino acid<br>
Racemase (BAR)<br>
(0890] A BAR was identified in P. putida KT2440 using information from literature<br>
(Roise, D., Soda, K., Yagi, T., Walsch, C.T., Biochemistry 23, 5195-5201, (1984)). The<br>
active site of a BAR enzyme from P. striata was sequenced and reported -<br>
LTAVLKADAYGXGIGL (SEQ ID NO:375). This sequence was used to BLAST the P.<br>
putida KT2440 genome sequence available m NCBI. A protein with a nearly identical<br>
consensus sequence was identified. Primers were designed to clone the gene from<br>
genomic DNA obtamed from the American Type Culture Collection (ATCC, Manassas,<br>
VA)			.-...,.   .<br>
{5'-AGAAGACATATGCCCTTTCGCCGTAGGG-3' (SEQ ID NO:376 and 5'-AGAAGAGGATCCTCAGTCGACGAGTATCTTCG-3') (SEQ IDNO:377)).<br>
|0891]	PCR was conducted under standard condiiions and d:e PCR product was purified<br>
(QIAquick PCR purification kit, Qiagen, Valencia, CA). The purified PCR product was digested with Nde I and BaniR I. The digested PCR product was gel purified (QIAquick Gel Extraction Kit, Qiagen, Valencia, CA) and ligated to pET30 and pET28 that had been digested and gel purified in a similar maimer. Clones with inserts were sequenced (Agencourt, Beverly, MA) and isolates with the correct sequence were identified (pET30 KT2440 BAR and pET28 KT2440BAJI) and used in later studies.<br>
[0892]	The KT2440 BAR DNA sequence is:<br>
atgccctttcgccgtacccttctggctgcatccctggcacttctgatcaccggacaggcccccctgtatgcg gcaccaccgtlgtcgatggacaacggccaacaccctgaccgtgcaaaacagcaatgcctgggtcga agtcagcgccagcgccctgcagcacaacatccgcacgctgcaggccgagctggccggcaagtccaag ctgtgcgccgtgctcaaggccgatgcctatggccacggtatcggcctggtaatgccatcgatcatcgccc aaggcgtgccctgcgtggcggtggccagcaacgaggaggcccgcgtggtccgcgccagtggcttcac cgggcaactggtgcgggtacgcctggccagcctcagcgagctggaagatggcttgcagtacgacatgg aagagctggtgggcagcgcggaatttgcccgccaggccgatgccatcgccgcgcgccatggcaagac<br>
 <br>
cttgcgcattcacatggcgctcaactccagcggcatgagccgcaacggggtggagatggccacctggtc cggccgtggcgaagcgctgcagatcaccgaccagaagcacctcaagctggtcgcgctgatgacccact tcgccgtggaagacaaggacgatgtacgcaagggcctggcggcattcaacgagcagaccgactggttg atcaagcacgccaggctggaccgcagcaagctcaccctgcacgccgccaactcgttcgctacgctgga agtgccggaagcgcgcctggacatggtacgaacgggtggcgcgctgttcggcgacaccgtgccggcg cgcaccgagtacaaacgtgcgatgcagtUiaaatcgcacgtggcggcggtgcacagctatccggccgg caacaccgtgggctatgaccgcaccttcaccctggcccgtgattcgcggctggccaacattacggtcgg glactccgatggctaccgccgggtattcaccaacaagggccatgtgctgatcaacggccaccgtgtgcc ggtcgtgggcaaggtgtcgatgaacacgctgatggtcgatgtcaccgacttccctgatgtgaaggggggt aacgaagtggtgctgttcggcaagcaggccgggggcgaaatcacccaggccgagatggaagaaatca acggcgcgttgctcgccgatttgtacaccgtatggggcaattccaacccgaagatactcgtcgactga (SEQ ID NO:378).<br>
[0893}	The KT2440 BAR amino acid sequence is:<br>
-Mpfrrtllaaslallitgqaplyaappismclngtntltvqnsnawvevsasalqhnirtlqaelagksklc avlkadayghgiglvmpsiiaqgvpcvavasneearwrasgftgqlvrvrlaslseledglqydmee Ivgsaefarqadaiaarhgktlrihmalnssgmsmgvematwsgrgealqitdqklilklvalmthfa vedkddvrkglaafheqtdwlikharldrskltlhaansfatlevpearldmvrtggalfgdtvpartey kramqfkshvaavlisypagntvgydrtftlardsrlanitvgysdgyrrvftnkghvlinghrvpwg kvsmntlmvdvtdfpdvkggnewlfgkqaggeitqaemeeingalladlytvwgnsnpkilvd (SEQ ID NO:379)<br>
B) Purification of P. putida KT2440 BAR.<br>
[0894]	The pET30 KT2440 BAR plasmid described above was transfonned into BL21<br>
DEB pLysS (Invitrogen, Carlsbad, CA). The resulting strain was grown in LB or Terrific Broth at 37° C with aeration to an ODeoonm of 0.4-0.6 and induced with 1 mM IPTG. Incubation was continued 3-4 hours at 37° C with aeration. The cells were harvested by centrifugation and the cell pellet was stored at -80 °C until use. The cell pellet was thawed on ice. The cells were l&gt;'sed with BugBuster (Novagen, Madison, WI) and Benzonase (Novagen, Madison, WI). Cell debris was removed by centrifugation and the cell free extract was either ;ised immediately or stored at -80° C. The KT2440 BAR gene was also cloned into the Ndel-BamHL sites of pET28 and transformed into BL21 DE3 pLysS. This construct did not appear' to express soluble protein very efficiently so the untagged version (pET30 KT2440 BAR") was used in future studies.<br>
 <br>
[089S]	The extract was applied to an UnoQ column (Bio-Rad, Hercules, CA) that had<br>
been equilibrated with at least 5 column volumes buffer A (25 mM potassium phosphate pH 8.0, 10 ixM pyridoxal-5'-phosphate (PLP)). The column was washed with 2 column volumes of buffer A. The protem was eluted with a linear gradient of buffer B (buffer A + 1 M NaCl) from 0-100% buffer B over 20 column volumes and 5 ml fractions were collected from the time the gradient started. Fractions were assayed using the Amplex Red method described in Example 4-part 7. Briefly, 100 p,g D-amino acid oxidase (Sigma-AIdrich, St. Louis, MO), 0.05 mM FAD, 25 mM L-trp, and a small volume of the fraction to be assayed were combined in 50 |iL H2O and added to 50 iL Amplex Red reaction buffer prepared as directed in the manufacturer's protocol. Fractions with activity were desalted with a PD-10 column (GE Healthcare, Piscataway, NJ) and concentrated with Amicon centrifugal concentrators (MilHpore, Billercia, MA). Purified protein was stored at -SOC.<br>
C) Production and assay of an alanine racemase mutant Y354A of Geobacitlus stearothermophilus with tryptophan racemase activity<br>
[0896]	The   wild-type   Geobacillus   stearothermophilus   alanine   racemase   (SEQ   ID<br>
NO:380, shown below was cloned into pET30 was used as a template for site-directed<br>
mutagenesis to make the Y354A change. The gene of SEQ ID NO:380 can be amplified<br>
by standard PCR protocols and cloned using standard recombinant DNA techniques. The<br>
gene of SEQ ID NO:380 can also be reconstructed by any method known to a person of<br>
ordinary skill in the art, such as assembly PCR methods known to one skilled in the art.<br>
[0897]	The wild-type Geobacillus stearothermophilus alanine racemase DNA and amino<br>
acid sequences is shown below as SEQ ID NO-.380:<br>
atggacgagt ttcaccgcga tacgtgggcg gaagtggatt tggacgccat ttacgacaat gtggagaatt tgcgccgttt gctgccggac gacacgcaca ttatggcggt cgtgaaggcg aacgcctatg gacatgggga tgtgcaggtg gcaaggacag cgctcgaagc gggggcctcc cgcctggcgg ttgccttttt ggatgaggcg ctcgctttaa gggaaaaagg aatcgaagcg ccgattctag ttctcggggc ttcccgtcca gctgatgcgg cgctggccgc ccagcagcgc attgccctga ccgtguccg ctccgactgg ttggaagaag cgtccgccct ttacagcggc ccttttccta ttcatttcca tttgaaaatg gacaccggca tgggacggct tggagtgaaa gacgaggaag agacgaaacg aatcgtagcg ctgattgagc gccatccgca ttttgtgctt gaaggggtgt    acacgcattt    tgcgactgcg    gatgaggtga    acaccgatta    tttttcctat<br>
 <br>
cagtataccc gttttttgca catgctcgaa tggctgccgt cgcgccxigcc gctcgtccat tgcgccaaca gcgcagcgtc gctccgtttc cctgaccgga cgttcaatat ggtccgcttc ggcattgcca tgtatgggct tgccccgtcg cccggcatca agccgctgct gccgtatcca ttaaaagaag cattttcgct ccatagt:cgc ctcgtacacg tcaaaaaact gcaaccaggc gaaaaggtga gctatggtgc gacgKicact gcgcagacgg aggagtggat cgggacgatt ccgatcggct atgcggacgg ctggctccgc cgccigcagc actttcatgt ccttgttgac ggacaaaagg cgccgattgt cggccgcatt tgcatggacc agtgcatgat ccgcctgcct ggtccgctgc cggtcggcac gaaggtgaca ctgattggtc gccaagggga cgaggtaatt tccattgatg atgtcgctcg ccatttggaa acgatcaact acgaagtgcc " ttgcacgatc agttatcgag tgccccgtat ttttttccgc cataagcgta taatggaagt gagaaacgcc gttggccgcg ga (SEQIDNO:380).<br>
[0898]	The  encoded  amino  acid  sequence  of the Alanine  Racemase  (Geobacillus<br>
stearothermophilus) is shown below as SEQ ID NO:381:<br>
Met Asp Glu Phe His Arg Asp Thr Trp Ala Glu VaJ Asp Leu Asp Ala He Tyr Asp Asn Val Giu Asn Leu Arg Arg Leu Leu Pro Asp Asp Thr His He Met Ala Val Val Lys Ala Asn Ala Tyr Gly His Gly Asp Val Gin Val Ala Arg Thr Ala Leu G] u Ala Gly Ala Ser Arg Leu Ala Val Ala Phe Leu Asp Glu Ala Leu Ala Leu Arg Glu Lys Gly He Glu Ala Pro He Leu VaJ Leu Gly Ala Ser Arg Pro Ala Asp Ala Ala Leu Ala Ala Gin Gin Arg He Ala Leu Thr Val Phe Arg Ser Asp Tip Leu Glu Giu Ala Ser Ala Leu Tyr Ser Gly Pro Phe Pro He His Phe His Leu Lys Met Asp Thr Gly Met Gly Arg Leu Gly Val Lys Asp Glu Glu Glu Thr Lys Arg lie Val Ala Leu He Glu Arg His Pro His Phe Val Leu Glu Gly Val Tyr Thr His Phe Ala Thr Aia Asp Giu Val Asn Thr Asp Tyr Phe Ser Tyr Gin Tyr Thr Arg Phe Leu His Met Leu Glu Trp Leu Pro Ser Arg Pro Pro Leu Val His Cys Ala Asn Ser Ala Ala Ser Leu Arg Phe Pro Asp Arg Thr Phe Asn Met Val Arg Phe Gly He Ala Met Tyr Gly Leu Ala Pro Ser Pro Gly He Lys Pro Leu Leu Pro Tyr Pro Leu Lys Glu Ala Phe Ser Leu His Ser Arg Leu Val His Val Lys Lys Leu Gin Pro Gly Glu Lys Val Ser Tyr Gly Ala Thr Tyr Thr Ala Gin Thr Glu Glu Trp He Gly Thr He Pro He Gly Tyr Ala Asp Gly Trp Leu Arg Arg Leu Gin His Phe His Val Leu Val Asp Gty Gin Lys Ala Pro He Val Gly Arg He Cys Met Asp Gin Cys Met He Arg Leu Pro Gly Pro Leu Pro Val Gly Thr Lys Val Thr Leu He Gly Arg Gin Gly Asp Glu Val He Ser He Asp<br>
 <br>
Asp VaJ Ala Arg His Leu Glu Thr He Lsn Tyr Glu Val Pro Cys Thr He Ser Tyr Arg Val Pro Arg He Phe Phe Arg His Lys Arg lie Met Glu Val Arg Asn Ala Val Gly Arg Gly (SEQIDNO:381)<br>
[0899]	The mutagenesis was performed using the QuickChange-Multi site-directed<br>
mutagenesis kit (Stratagene, La Jolla, CA). The following mutagenic primer was used to make the Y354A change, 5'-gccatctggaaacgatcaacgcggaagtgccttgcacgatcag-3' (SEQ ID NO:382). The site-directed mutagenesis was done as described in the manufacturer's protocol. Several isolates were sequenced (Agencourt, Beverly, MA) and an isolate with the correct sequence was selected and used for further analysis.<br>
[0900]	The pET30Y354A single mutant was transformed into E. coli BL21(DE3)pLysS.<br>
Purified protein was prepared in ihs toHowing manner. The strain was grown in LB or Terrific Broth (at 37* C with aeration) to an ODeoonm of 0.4-0.6 and induced with 1 mM IPTG. Incubation was continued at 37° C with aeration for -3 hours. The cells were harvested by centrifugation and the cell pellet was stored at -80° C.<br>
[0901]	The cell pellet was ttiawed on ice and then re-suspended in an appropriate volume<br>
of BugBuster (Novagen, Madison, WI) plus Benzonase nuclease (Novagen, Madison, WI). Cell debris was removed by centrifugation, and the cell-free extract was applied to a HIS-Bind column (Novagen, Madison, WI) that had been equilibrated with Binding buffer. The column was washed with Binding buffer and Wh buffer and the protein was eluted with Elution buffer (as directed in the manufacturer's protocol). The purified protein was desalted using a PD-10 column (GE Healthcare, Piscataway, NJ). The protein was desalted into 50 mM potaasiimi phosphate pH 8.0 and 10 {iM pyridoxal-5'-phosphate according to the manufacturer's protocol. The protein was concentrated using an Amicon centrifugal concentrator (MiUipore, Billercia, MA). The purified and concentrated protein was divided into small aliquots and stored at -80° C until use.<br>
[0902]	The purified Y354A was compared to wild-type alanine racemase (prepared in the<br>
manner described above) in both alanine and tryptophan assays. Assays were performed at 37 °C in 50 mM potassium phosphate buffer, pH 8, and 10 JJM PLP using 400 ML of concentrated purified protein (&gt;1 mg/ml) and 50 mM substrate. Detection of D-alanine and D-tryptophan was performed taing the chiral amino acid methodology described in Example 1.<br>
 <br>
 <br>
[0903]	These data were analyzed without the use of an internal standard; thus, these data<br>
are semi-quantitative and should be used for comparative purposes. Nonetheless, these results show that the Y354A single mutation is sufficient to broaden the specificity of the alanine racemase so that it can catalyze amino acid racemization using altemative substrates.<br>
D) Assay of the BAR enzyme.<br>
[0904]	The BAR eniyme was assayed as follows. Amplex Red assays were set up as<br>
described in this example. P. putida KT2440 BAR was used at 200 \i% (purified as described in this example). Wildtype G. stearothermophilus alanine racemase and the Y354A were purified as described in this example and used at either 200 [xg/mL or 1000 P-g/'mL. CE is cell-free extract that was prepared as described in this example. The results for the 60 minute time point are shown in the Table 41 below.<br>
 <br>
 <br>
0905]	The purified protein was also Eissayed for tryptophan racemase activitj'- in 50 mM<br>
potassiml phosphate pH 8, 10 jxM PLP, and 30 mM L-tryptophan as described above. Either 200 |j,g/mL or 1000 jig/mL of purified enzyme was used in the assays (indicated in parentheses). D-tryptophan was analyzed using the chiral amino acid method described in Example 1 for detection.<br>
 <br>
 <br>
 <br>
[0906]	The assays indicate that the P. puiida KT2440 BAR enzyme is much more active<br>
on tryptophan than the G. stearothermophilus derived enzyme and the Y354A mutant thereof.<br>
E) Monatin production with P. putida KT2440 BAR.<br>
[0907]	A monatin production assay was done with the purified P. putida KT2440 BAR<br>
(as purified above) (100 g) or purified Y354A (as purified above) (500 )ag), D-aminotransferase (BioCatalytics AT-103 (Pasadena, CA)) (500 \ig), and the aldolase of SEQ ID NO:276) (as purified in Example 23) (50 ng). The monatin production experiment starting v«th L-tryptophau was done as follows. In addition to the enzymes above, the following were added per 1 mL of reaction mixture: 4 mM MgCb, 50 mM L-tryptophan, 100 mM sodium pyruvate, 100 mM potsium phosphate buffer pH 7.5, and 0.05 mM PLP.<br>
[0908]	As a control, the experiment was done as described above without racemase and<br>
starting with D-tryptophan instead of L-tryptophan. A summary of the results is presented in Table 43 below.<br>
 <br>
 <br>
 <br>
[0909]	No monatin was detected using Y354A in this experiment. This racemase has<br>
been used in the past (data not shown) to produce monatin, but a much higher level of enzyme was used (at least 2 mg and up to 10 mg to see higher levels of monatin). The P. putida KT2440 BAR was used to produce monatin fi-om L-tryptophan. The 100 )ig used in this experiment was not enough to see monaiin production after two hours but was enough to see monatin production after 18 hours. The stereoisomer distribution indicated that most of the monatin produced is the R,R isomer. There was no R,S isomer produced. This result indicates that KT2440 BAR is not able to detectably racemize the R,R isomer of monatin (racemization of the R,R isomer would produce the R,S isomer). There was a significant amoxmt of the S,S isomer produced in this experiment. This is probably due to the fact that the AT-103 used in this experiment is not highly purified and may contain L-aminotransferases from the cellular extract, and that there is a large amount of L-tryptophan present to serve as an amino donor for transamination of S-MP.<br>
Example 29 Cloning and Expression of Pseudomonas taetrolens Arginine Racemase<br>
Experimental Overview<br>
[0910]	Pseudomonas  taetrolens   (also  known as P.  graveolens)  arginine  racemase<br>
(Genbank Accession No. AB096176, nucleic acid sequence) and an I384M mutant thereof, was cloned, expressed, and tested for activity m conversion of L-tryptophan to D-tryptophan. This gene is 72% identical to the P. putida BAR gene from KT2440 and 73% identical to the P. putida BAR gene fi-om NBRC 12996 described above. The amino acid sequence is 72% identical to both P. putida BAR proteins.<br>
Polymerase Chain Reaction Protocol<br>
[0911]	Pseudomonas taetrolens (ATCC 4683) was grown in nutrient broth at 28 °C with<br>
shaking at 225 rpm.   Polymerase cham reaction was performed on whole cells using<br>
 <br>
primers designed with 5' restriction sites and overliangs for cloning into the pET 28 and pET30 vectors (Novagen, Madison, WI).<br>
[0912]	The primer sequences were:<br>
N terra: 5'-ATAATACATATGCCCTTCTCCCGTACCC -3' (SEQ ID NO:408) and C term: 5'-GCGGCGGGATCCTTACTGATCTTTCAGGATT-3' {SEQ ID NO:409).<br>
[0913]	The gene derived from P. taetrolens was amplified using the following PCR<br>
protocol. Twenty-five (iL of grown cells were lysed at 96 °C for 10 minutes. Cell debris was removed by centrifiigation and the supernatant was used as template for PCR. A 100 IL reaction contained 5 \xL template (lysed cell supernatant), 1.6 yM of each primer, 0.3 mM each dNTP, 10 U rT* Polymerafie XL (Applied Biosystems, Foster City, CA), IX XL buffer and 1 mM Mg(0Ac)2. The thermocycler program used included a hot start at 94 °C for 3 minutes, 8 repetitions of the following steps: 94 °C for 30 seconds, 52 °C for 30 seconds, and 68 °C for 2 minutes, followed by 22 repetitions of the following steps: 94 °C for 30 seconds, 58 °C for 30 seconds, and 68 °C for 2 minutes. After the 22 repetitions, the sample was maintained at 68 °C for 7 minutes and then stored at 4 °C. This PCR protocol produced a product of 1230 bp.<br>
Cloning<br>
[0914]	The PCR product was gel purified from 0.8% TAE-agarose gel using the Qiagen<br>
gel extraction kit (Qiagen, Valencia, CA). The product was TOPO cloned and transformed into TOP 10 cells according to manufacturer's protocol (hivitrogen, Carlsbad, CA). Plasmid DNA was purified from the resulting transformants using the Qiagen spin miniprep kit (Qiagen, Valencia, CA) and screened for the correct inserts by restriction digest with fde I and BamH I. The sequences of piasmids appearing to have the correct insert were verified by dideoxy chain termination DNA sequencing with universal M13 forward and M13 Reverse primers.<br>
[091S]	The correct TOPO clone was digested witii restriction enzymes Nde I and BamH I<br>
following the manufacturer's recommended protocols (New England Biolabs, Beverly, MA) and gel purified from 0.8% TAE-agarose gels using the Qiagen gel extraction kit (Qiagen, Valencia, CA). Vectors pET 28 and pET 30 were prepared by digestion witii reshiction enzymes Nde 1 and BamH I followed by treatment witii shrimp alkaline phosphatase (Roche, hidianapolis, IN) and purification from 0.8% TAE-agarose gels using tile Qiagen gel extraction kit (Qiagen, Valencia, CA).   The digested vectors and<br>
 <br>
insert were ligated using the Rapid " DNA Ligation Kit (Roche, Indianapolis, IN). Approximately 50 ng of treated insert, 100 ng of treated vector (3 to 1 molar ratio of insert to vector), 5 U of T4 DNA ligase, and IX ligation buffer were incubated for 5 minutes at room temperature. The ligation reaction was desalted using the High Pure PCR Product Purification Kit (Roche, Indianapolis, IN) and used to transform E. coli DHIOB eiectrocompetent cells (Invitrogen, Carlsbad, CA). Ten iL of each ligation reaction was added to 40 iL of DHIOB cells, which were transformed by electroporation using the BioRad Gene Pulsar II under the following conditions: 2.5 kV, 25 fxF, 200 ohm in a 0.2 cm cuvette. The cells were allowed to recover in 1 mL of room temperature SOC for 1 hour at 37 "C with shaking at 225 rpm. Cells were plated on LB plates containing kanamycin (50 ftg/mL). Plasmid DNA was purified from the resulting transformants using the Qiagen spin miniprep kit (Qiagen, Valencia, CA) and screened for the correct inserts by restriction digest ',\ith Nde I ;md BamHl.<br>
Gene Expression and Assays	,, .    .<br>
[0916]	Plasmid DNA was transformed into E. coli expression host BL2](DE3) pLysS<br>
(Novagen, Madison, WI). The cultures were grown and the plasmids were isolated using the Qiagen miniprep kit (Qiagen, Valencia, CA) and analyzed by restriction digest to confirm identity.<br>
[0917]	Induction in BL21DE3 pLysS was initially performed in both pET 28 (histidine-<br>
tagged) and pET 30 (untagged) vectors. A time course study was performed with cultures grown at 37 °C in 100 mL LB contaming kanamycin (50 mg/L) to an ODoo of 0.5 and induced with 100 [xM IPTG (isopropyl thiogalacatoside) and sampled at 0 and 3 hours post induction. Cells from 0 hour and 3 hour time points were resuspended in IX sodium dodecyl sulfate buffer containing 2-mercaptoethanoi and heated at 95 °C for 10 minutes, and cooled. Aliquots of these total cellular protein samples were analyzed by SDS-PAGE using a 4-15% gradient gel.<br>
[0918]	Cell extracts were also prepared from the 3 hour cultures by suspending cell<br>
pellets from 5 mL of culture in Novagen BugBuster' reagent containing benzonase nuclease and protease inhibitor cocktail set #3 (Calbiochem-Novabiochem Corp., San Diego, CA) at room temperature for 20 minutes with gentle shaking and centriiliging at 16,000 X g to remove cell debris. The supematants (cell extracts) were loaded onto 4-15% gradient gels for analysis of flip r-el1n!ar soluble nroteins.<br>
 <br>
[0919]	The 3 hour sample ftom cloned P. taetrolens arginine racemase showed a total<br>
protein band that corresponded to the correct size (approximately 45 kDa) in the pET 30 (untagged) vector. The P. taetrolens pET 30 gene product was over-expressed at a higher level than the P. taetrolens pET 28 (histidine-lagged) gene piodnct, but neither of £he vectors gave a visible soluble protein, band.<br>
(0920]	Ceils from the induced cultures (100 mL) were centrifuged and washed once with<br>
0.85 % NaCl. Cell pellets were resuspended in 5 mL/g wet cell weight of BugBuster™ (Novagen, Madison, WI) reagent containing 5 jiL/mL protease inhibitor cocktail set #3 (Calbiochem-Novabiochem Corp., San Diego, CA) and 1 \xVmh benzonase nuclease. Samples were incubated at room temperature for 20 minutes on an orbital shaker. Insoluble celJ debris was removed by centrifugation at 16,000 X g for 20 minutes at 4 "C.<br>
[0921]	Cell extracts were assayed for tryptophan racemase activity using the following<br>
protocol. One mL reactions were carried out in 50 mM potassium phosphate (pH 8.0), 0.05 mM PLP and 30 mM L tryptophan. The reactions were initiated by the addition of cell free extracts and were incubated at 30 °C overnight. Sample aliquots were taken after overnight incubation (zero minute samples served as control reaction). Concentrated formic acid (5 \xL) was added to each 250 |iL sample aliquot to stop the reaction and the precipitated protein was removed by centrifugation. Supematants were removed and frozen at -80 °C until they were analyzed for D-tryptophan by the chiral amino acid method described in Example 1.<br>
[0922]	Assay results from cell extracts from pET28 and pET30 induction with 100 uM<br>
IPTG (3 hours) demonstrate that P. taetrolens clones show racemase activity on L-tryptophan. Again, the tagged version of the BAR appears to be less active and may precipitate or be less soluble than the untagged (pET28). Table 44, below, shows the initial results, although not quantitative as very poor soluble protein was obtained.<br>
 <br>
 <br>
 <br>
[0923]	Induction of the pET30 (untagged) construct was repeated using same conditions<br>
as mentioned above and a visible soluble protein band was observed in SDS-PAGE. The assay was repeated using same the conditions described above and the results, as shown in Table 45 below, were obtained.<br>
 <br>
[0924]	Again, it was noted that doubling of the volumes did not scale to more activity.<br>
For future work, it was determined to remove the proteirj irom Bugbuster as quickly as possible after preparation of cell extracts and to store the protein in 50 mM phosphate buffer pH 8 containii 0.01 mM PLP. The detergent in Bugbuster may inhibit the reaction or may cause a loss of activity upon storage.<br>
[0925]	Induction of the pET30 construct was carried out again and the cell extract was<br>
processed with anion exchange chromatography (as in Example 28) to give a more pure extract. The assay was repeated with this partially purified prep. The numbers in parenthesis in the Racemase extract coltmm of Table 46 below indicate the approximate amount of partially purified racemase enzyme used in the assay. The results of the assay are shown in Table 46 below.<br>
 <br>
 <br>
 <br>
[0926]	The non-linearity of the overnight sample in this case is probably due to the fact<br>
that the reactions are reaching equiHbrium. Clearly, the P. taetrolens BAR has significant<br>
activity for racemization of tryptophan, as do the 12996 BAR and KT2440 BAR.   It<br>
appears that the KT2440 BAR and the P. taetrolens BAR have similar activity, which is<br>
slightly higher than the 12996 BTvR.<br>
[0927|	The DNA Sequence of the P. taetrolens arginine racemase is shown below as<br>
SEQ ID NO:410. The PCR sequence gave two changes as compared with the published NCBI sequence. Specifically, the PCR sequence contained an adenosine rather than a guanine at position 902 and a cytosine rather than a guanine at position 921. These DNA changes resulted in one silent mutation as well as one change from glycine to aspartate at amino acid position 301.<br>
ATGCCCTTCTCCCGTACCCTGCTCGCCCTTTCCCTTGGCAT<br>
GGCATTGCTGCAAAACCCGGCCTTTGCTGCGCCACCCCTG<br>
TCGATGACCGACGGCGTAGCTCAAGTGAATACCCAGGAC<br>
AGCAATGCCTGGGTCGAA.A.TCAATAAAGCCGCGTTCGAG<br>
CACAACATACGGACTCTGCAAACCGCCCTCGCCGGCAAG<br>
TCGCAGATCTGCGCCGTACTCAAGGCGGATGCCTATGGC<br>
CACGGTATCGGCTTGTTGATGCCCTCGGTGATCGCCATGG<br>
GTGTTCCCTGTGTCGGTGTCGCCAGCAACGAAGAAGCCC<br>
GCGTCGTGCGCGAGAGCGGTTTCAAGGGTCAACTGATAC<br>
GCGTGCGCACCGCTGCCCTGAGCGAACTGGAAGCTGCAC<br>
TGCCGTACAACATGGAAGAGCTGGTGGGCAACCTGGACT<br>
TCGCGGTCAAGGCCAGCCTGATTGCCGAGGATCACGGTC<br>
GCCCGCTGGTGGTGCACCTGGGTCTGAATTCCAGCGGCA<br>
TGAGCCGTAACGGAGTGGACATGACCACCGCTCAGGGCC<br>
 <br>
. GTCGTGATGCGGTAGCTATCACCAAGGTGCCAAACCTGG AAGTGCGGGCGATCATGACCCACTTCGCGGTCGAAGATG CTGCCGACGTGCGTGCCGGGCTCAAGGCCTTCAATCAGC AAGCCCAATGGCTGATGAACGTGGCCCAGCTTGATCGCA GCAAGATCACCCTGCACGCGGCCAACTCGTTCGCCACAC TGGAGGTGCCCGAATCGCATCTGGACATGGTCCGCCCCG GCGGCGCGCTGTTCGGCGACACCGTACCGTCCCACACCG AGTACAAGCGGGTCATGCAGTTCAAGTCCCACGTGGCGT CGGTCAACAGCTACCCCAJGGGCAACACCGTCGGTTATG<br>
ACCGCACGTACACCCTGGGCCGCGACTCGCGGCTGGCCA<br>
ACATCACCGTCGGCTACTC:TGACGGCTACCGCCGCGCGTT<br>
TACCAATAAAGGGATTGT(}CTGATCAACGGCCATCGCGT<br>
GCCAGTGGTGGGCAAAGTCTCGATGAACACCCTGATGGT<br>
GGACGTCACTGACGCGCCGGATGTGAAAAGCGGCGATGA<br>
AGTGGTGCTGTTCGGGCACCAGGGCAAGGCCGAGATTAC<br>
CCAGGCTGAGATCGAAGACATCAACGGTGCACTGCTTGC<br>
GGATCTGTATACCGTGTGCIGGCAATTCCAACCCTAAAAT<br>
CCTGAAAGATCAGTAA<br>
(SEQIDNO:410).<br>
[0928]	The protein encoded by the gene of SEQ ID NO:410 was analyzed by the signal<br>
peptide prediction program Signal P 3.0 (www.cbs.dtu.dk/services/SignaIP/) and a leader sequence of 23 amino acids was predicted.<br>
I384M Mutagenesis ofV. taetrolens BAR<br>
[0929]	Mutagenesis was done using the QuickChange-Multi site-directed mutagenesis kit<br>
(Stratagene, La Jolla, CA), using the P. taetrolens BAR gene in pET30 which results in an untagged protein. The following mutagenic primer was used to make the i384M change: 5'-TACCCAGGCTGAGATGGAAGACATCAACG-3' (SEQ ID N0:411).<br>
[0930]	The site-directed mutagenesis was done as described in the manufacturer's<br>
protocol. Several isolates were sequenced (Agencourt, Beverly, MA) and an isolate with the correct sequence was selected and used for further analysis.<br>
 <br>
[0931]	The plasmid was transformed into BL21(DE3) ceils (Novagen, Madison, Wl).<br>
Recombinant protein was produced in Overnight Express II medium (Novagen, Madison, WI) containing 50 ig/mL kanamycin according to manufacturer's protocols. Cell-free extracts were prepared using BugBuster (Novagen, Madison, WI) according to manufacturer's protocols, desalted, and analyzed for percent expression of the target protein using the Experion method described above.<br>
[0932]	Total protein assays were done using a Pierce BCA kit (Pierce, Rockford, IL).<br>
Tryptophan racemase assays with the mutant enzyme were performed using the wild-type enzyme prepared in the same manner as a positive control. Assays contained per mL: 30 mM L-tryptophan, 50 mM potassium phosphate pH 8, 10 pM PLP, and approximately 100 fig of racemase protein in a cell fiee extract. In the case where 100 |ig was not used (based on Experion % expression and Pierce total protein numbers), the results were normalized. Zero, 30 minute, 2 hour, and overnight samples were collected, treated with . 2% formic acid, filtered, and diluted 1:10 for analysis using the chiral amino acid method described in Example 1.<br>
[0933]	The wild-type enzyme appeared to produce 49.1 ppm D-tryptophan in 30 minutes,<br>
whereas the I384M mutant produced 108 ppm. The 2 hour time point was similar - 229.4 ppm D-tryptophan was produced by the wild-type enzyme versus 541-7 for the I384M mutant. The I384M mutation appears to have approximately doubled the activity of the F. taetrolens BAR. The overnight time point for the mutant is also higher, but as the reactions approach equilibrium the difference between activities is reduced. When assays were done for monatin production as in Example 28, the I384M did not appear to provide any benefit over the wild-type P. taetrolens enzyme.<br>
Example 30<br>
A. caviae Extract Assay<br>
[0934]	Aeromonas caviae ATCC 14486 was grown in nutrient broth at 37 "C. Cells from<br>
the culture (200 mL) were centrifuged and washed once with 0.85 % NaCl. Cell pellets were resuspended in 5 mL/g wet ceil weight of BugBuster' (Novagen, Madison, WI) reagent containing 5 [iL/mL protease inhibitor cocktail set #3 (Calbiochem-Novabiochem Corp., San Diego, CA) and 1 |JVmL benzonase nuclease.   Samples were incubated at<br>
 <br>
room temperature for 20 minutes on an orbital shaker. Insoluble cell debris was removed by centriiligation at 16,000 X g for 20 minutes at 4 "C. Cell-free extract was desalted on a PD-10 column (GE Healthcare, Piscataway, NJ).<br>
[0935]	Cell-free extract was assayed for tryptophan racemase activity using the following<br>
protocoi. One mL reactions were carried out in 50 mM potassium phosphate (pH 8.0), 0.05 mM PL? and 30 mM L tryptophan. The reactions were initiated by the addition of cell free extract (either 100 fxL or 500 nL) and were incubated at 30 °C overnight. Sample aliquots were taken at 2 hours and after overnight incubation (zero minute samples served as control reactions). Concentrated formic acid (5 |iL) was added to each 250 |xL sample aliquot to stop the reaction and the precipitated protein was removed by centrifugation. Supematants were removed and frozen at —80 'C until they were analyzed for D-tryptophan by the chiral amino acid method described in Example 1.<br>
[0936]	The assay results from cell extracts of . caviae  demonstrated racemase activity<br>
on L-tryptophan, as shown in Table 47.<br>
 <br>
[0937]	After finding activity in the A. caviae cell extracts, degenerate primers were<br>
designed (based on conserved regions of known BAR homologs) to obtain the BAR gene from this species. Degenerate primer sequences are shown below: Aer deg F2: 5'-GCCAGCAACGARGARGCMCGCGT-3' (SEQ ID NO:412); and AerdegRl: 5'-TGGCCSTKGATCAGCACA-3'(SEQIDNO:413) wherein K indicates G or T, R indicates A or G, S indicates C or G, and M indicates A orC.<br>
 <br>
[0938]	The above primers were used to PCR amplify a 715 bp DNA fragment from A.<br>
caviae (ATCC 14486) genomic DNA. The following PCR protocol was used: A 50 p.L reaction contained 0.5 fiL template (-100 ng of. caviae genomic DNA), 1.6 fxMof each primer, 0.3 mM each dNTP, 10 U rX* Polymerase XL (Applied Biosystems, Foster City, CA), iX XL buffer, 1 mM Mg(0Ac)2 and 2.5 fiL dimethyl sulfoxide. The thermocycler program used included a hot start al: 94 °C for 3 minutes and 30 repetitions of the following steps: 94 °C for 30 seconds, 53 °C for 30 seconds, and 68 °C for 2 mmutes. After the 30 repetitions, the sample was maintained at 68 °C for 7 minutes and then stored at 4 °C. This PCR protocol produced a product of 715 bp.<br>
Cloning<br>
[0939]	The PCR product was gel purified from 0.8% TAE-agarose gel using the Qiagen<br>
gel extraction kit (Qiagen, Valencia, CA). The product was TOPO cloned and<br>
transformed into TOP 10 cells according to manufacturer's protocol (Invitrogen,<br>
Carlsbad, CA). The plasmid DNA was purified from the resulting transformants using<br>
the Qiagen spin miniprep kit (Qiagen, Valencia, CA) and screened for the correct inserts<br>
b}' restriction digest with EcoR I. The sequences of p\asmids appearing to have the<br>
correct insert were verified by dideoxy chain termination DNA sequencing with universal<br>
M13 forward primers.<br>
[0940]	The DNA sequence of the A, caviae PCR product is shown below as SEQ ID<br>
NO:414), with degenerate primer sequence regions underlined;<br>
GCCAGCAACGARGARGCMCGCGTTGCCCGCGAGAAGGGCTTCG<br>
AAGGTCGCCTGATGCGGGTACGTGCCGCCACCCCGGATGAAGT<br>
GGAGCAGGCCCTGCCCTACAAGCTGGAGGAGCTCATCGGCAGC<br>
CTGGAGAGCGCCAAGGGGATCGCCGACATCGCCCAGCGCCATC<br>
ACACCAACATCCCGGTGCACATCGGCCTGAACTCCGCCGGCAT<br>
GAGCCGCAACGGCATCGATCTGCGCCAGGACGATGCCAAGGCC<br>
GATGCCCTGGCCATGCTCAAGCTCAAGGGGATCACCCCGGTCG<br>
GCATCATGACCCACTTCCCGGTGGAGGAGAAAGAGGACGTCAA<br>
GCTGGGGCTGGCCCAGTTCAAGCTGGACTACCAGTGGCTCATC<br>
GACGCCGGCAAGCTGGATCGCAGCAAGCTCACCATCCACGCCG<br>
CCAACTCCTTCGCCACCCTGGAAGTACCGGAAGCCTACTTTGAC<br>
ATGGTGCGCCCGGGCGGCATCATCTATGGCGACACCATTCCCTC<br>
 <br>
CTACACCGAGTACAAGAAGGTGATGGCGTTCAAGACCCAGGTC<br>
GCCTCCGTCAACCACTACCCGGCGGGCAACACCGTCGGCTATG<br>
ACCGCACCTTCACCCTCW5.GCGCGACTCCCTGCTGGCCAACCTG<br>
CCGATGGGCTACTCCGACGGCTACCGCCGCGCCATGAGCAACA<br>
AGGCCTATGTGCTGATCIvl.\SGGCCA<br>
(SEQ ID NO:414), wherein R indicates A or G, S indicates C or G, and M indicates A or C.<br>
[0941]	The amino acid sequence of the partial A. caviae BAR enzyme is shown as SEQ<br>
E)NO:415 below.<br>
ASNEEARVAREKGFEGRLMR.VRAATPDEVEQALPYKLEELI<br>
GSLESAKGIADIAQRHHTNIPVHIGLNSAGMSRNGIDLRQDD<br>
AKADALAMLKLKGITPVGIMTHFPVEEKEDVKLGLAQFKL<br>
DyQWLlDAGKLDRSKLTlHAANSFATLEVPEAYFDMVRPG<br>
GIIYGDTIPSYTEYKKVMAFKITQVASVNHYPAGNTVGYDRT<br>
FTLKRDSLLANLPMGYSDGYRRAMSNKAYVLIXG<br>
wherein X is H, Q, N, or K (SEQ ID NO:415).<br>
[0942]	The consensus protein sequence fragment of SEQ ID NO:415 above is 89%<br>
homologous at the amino acid level to the published TIGR sequence for A. hydrophila. It was expected that because the highly related Aeromonas hydrophila protein exhibited broad specificity racemase activity, as well as the A. caviae cellular extracts, the fiill length coding region for A. caviae, once obtained, would produce a racemase that also would have a broad specificity with activity on tryptophan. Genome Walker methods were utihzed to obtain the full-length gene sequence of the A. caviae BAR gene shown beIowasSEQIDNO;416.<br>
atgcacaaga aaacactgct cgcgaccctg atctttggcc tgctggccgg ccaggcagtc gccgccccct atctgccgct cgccgacgac caccgcaacg gtcaggaaca gaccgccgcc aacgcctggc tggaagtgga tctcggcgcc ttcgagcaca acatccagac cctgaagaat cgcctcgglg acaagggccc gcagatctgc gccatcatga aggcggacgc ctacggtcac ggcatcgacc tgctggtccc ttccgtggtc aaggcaggca tcccctgcat cggcatcgcc agcaacgaag aagcacgtgt tgcccgcgag aagggcttcg aaggtcgcct gatgcgggta<br>
 <br>
cgtgccgcca ccccggatga agtggagcag gccctgccct acaagctgga ggagctcatc<br>
ggcagcctgg agagcgccaa ggggatcgcc gacatcgccc agcgccatca caccaacatc<br>
ccggtgcaca tcggcctgaa ciccgccggc atgagccgca acggcatcga tctgcgccag<br>
gacgatgcca aggccgatgc cctggccatg ctcaagctca aggggatcac cccggtcggc<br>
atcatgaccc acttcccggt ggaggagaaa gaggacgtca agctggggct ggcccagttc<br>
aagctggact accagtggct catcgacgcc ggcaagctgg atcgcagcaa gctcaccatc<br>
cacgccgcca actccttcgc caccctggaa gtaccggaag cctactttga catggtgcgc<br>
ccgggcggca tcatctatgg cgacaccatt ccctcctaca ccgagtacaa gaaggtgatg<br>
gcgttcaaga cccaggtcgc ctccgtcaac cactacccgg cgggcaacac cgtcggctat<br>
gaccgcacct tcaccctcaa gcgcgactcc ctgctggcca acctgccgat gggctactcc<br>
gacggctacc gccgcgccat gagcaacaag gcctatgtgc tgatccatgg ccagaaggcc<br>
cccgtcgtgg gcaagacttc catgaacacc accatggtgg acgtcaccga catcaagggg<br>
atcaaacccg gtgacgaggt ggtcctgttc ggacgccagg gtgatgccga ggtgaaacaa<br>
tctgatctgg aggagtacaa cggtgccctc ttggcggaca tgtacaccgt ctggggctat<br>
accaacccca agaagatcaa gcgctaa<br>
(SEQIDNO:416).<br>
[0943]	The corresponding amino acid sequence for the A. caviae native BAR is shown<br>
belowasSEQIDNO:417:<br>
I mhkktllatl ifgllagqav aapylpladd hmgqeqtaa<br>
41 nawlevdiga fehniqtlkn rlgdkgpqic aimkadaygh<br>
81 gidllvpsw kagipcigia sneearvare kgfegrimrv<br>
121 raatpdeveq alpykleeli gslesakgia diaqrhhtni<br>
161 pvhiglnsag msmgidlrq ddakadalam Iklkgitpvg<br>
201  imthfpveek edvklglaqf kldyqwlida gkldrskiti<br>
241 haansfatle vpeayfdmvr pggiiygdti psyteykkvm<br>
281 afktqvasvn hypagntvgy drtftlkrds llanlpmgys<br>
321 dgyrramsnk ayvlihgqka pwgktsmnt tmvdvtdikg<br>
361 ikpgdewlf grqgdaevkq sdleeyngal ladmytvwgy<br>
401 tnpkkikr<br>
(SEQIDNO:417).<br>
[0944]	The first 21 N-terminal amino acid residues of SEQ ID NO:417 are predicted to<br>
be a signal peptide using die program Signal P 3.0 (www.cbs.dtu.dk/services/SignalP/).<br>
 <br>
Experimental evidence confirmed thai the expression product was secreted into the periplasm of E. coli, and the signal peptide was cleaved as predicted. The full length gene, when cloned and expressed using methods described above, was found to have activity comparable to, but greater than, the P, taetrolens BAR.<br>
Example 31 Production of the aldolase of SEQ ID NO: 276 in an alternative expression host<br>
[0945]	The gene of SEQ ID NO: 275 was subcloned using standard molecular biology<br>
procedures into a derivative of the pET23d vector (Novagen, Madison, WI) contaiiung the E. coli metE gene and promoter inserted at the NgoMT restriction site and a second psil restriction site that was added for facile removal of the beta lactamase gene {bid). The construction of this vector containing an insert for a myo-inositol oxygenase gene is described in PCT WO2006066072 in Examples 2 and 20. The aldolase insert was confirmed by DNA sequencing (Ageiicourt Bioscience Corporation; Beverly, MA) and the plasmJd with the correct insert sequence was transformed into the E. coli expression host BW30384(DE3)AompTAmetE. The construction of this expression host and the transformation protocol are also described in PCT WO2006066072 (Examples 21 and 22). The aldolase gene was expressed using the Novagen Overnight Express' Autoinduction System U (Novagen, Madison, WI) containing 100 mg/L ampicillin. This system was described in Example 24 for the expression of Bacillus sphaericus (ATCC strain 10208) D-alanine aminotransferase. Cell free extracts containing the aldolase were produced using Novagen BugBuster™ Extraction Reagent (primary amine fees) (Novagen, Madison, WI) containing 1 |iL/mL benzonase nuclease, 0.033 [j,L/mL r-Lysozyme, and 5 mL Protease Inhibitor Cocktail Set II following the manufacturer's recommended protocol for ceil lysis. The soluble proteins in the cell free extracts were separated on a Bio-Rad Laboratories Experion" Automated Electrophoresis Station (Bio-Rad, Hercules, CA) and analyzed for percent soluble protein expression using the Experion Software version 1.1.98.0 as described in Example 12.<br>
[0946]	To attempt to improve the aldolase expression in E. coli, codons two through<br>
seven of the DNA coding sequence were mutated to changes suggested from the analysis of the wild type sequence using the Roche ProteoExpert RTS E. coli HY algorithm.  The<br>
 <br>
changes were made using a QuikCliange® Multi Site-Directed Mutagenesis Kit or QuikChange® U XL Site-Directed Mutagenesis Kit (Stratagene, La Jolla, CA) following the manufacturer's suggested protocol with 0.75 |iL of Quik Solution per 25 |iL reaction mixture and transformation into XLIO-Gold® ultracompetent cells. The mutations were generated using primers (each 45 nucleotides m length) designed with Stratagene's web-based QuikChange® Primer Design Program available on-hne at www. stratagene.com.qcprimerdesign.<br>
 <br>
[0947]	The aldolase gene sequences with the above codon changes were transformed into<br>
the E. coli expression host BW30384(DE3)AompTAmetE and then expressed using Novagen Overnight Express' Autoinduction System II (Novagen, Madison, WI). None of these mutations resulted in higher levels of gene expression when compared to the wild type sequence. Typical results from cell free extracts analyzed by the Bio-Rad Experion Pro260 software 1.1.98.0 are shown in Table 49 below, in which the column entitled Protein Expression shows values for % total soluble protein.<br>
[0948]	The cell free extracts (0.025 mg/mL soluble protein per assay) were assayed for<br>
their ability to produce R,R-monatin from 200 mM sodium pyruvate and 100 mM D-tiyplophan using the protocol described in Example 7. The reactions (7 mL total volume) were carried out in 14 mL polypropylene tubes in an anaerobic glove box using purified B. sphaericus (ATCC 10208) D-aminotransferase at 2 mg/mL final concentration. The concentrations of monatin produced at 1, 4 and 20 h after the addition of the enzymes are shown in Table 49 below. Table 49 shows that the cell free exfract generated from the construct containing the wild type aldolase sequence produced a slightly higher concentration of monatin at 20 hours than the cell free extracts from the constructs carrying the ProteoExpert mutations. The monatin concentrations were determined by the LC/MS/MS method described in Example 1.<br>
 <br>
 <br>
[0950]	The bla gene was removed from the vector and subsequent fermentations to<br>
produce aldolase were done without the use of antibiotic as described in PCX WO2006066072 for another enzyme. Expression levels in fed-batch fermentations at 30 °C reached a maximum at 6-8 hours post-induction, producing the aldolase of SEQ ID NO: 276 at 25-30% of the soluble protein, according to Experion data. Stability studies showed no apparent loss of aldolase activity when the fermentation product was left for 6 hours at 15 or 30 "C (under both oxygen limiting conditions as well as aerated conditions) prior to cell concentration and disruption. The aldolase was found to be equally stable stored as either a cell free extract or as a cell pellet when stored for 5 days at -80 °C. Washing the cell pellet in buffer prior to storage at -80 °C was not required and actually caused a slight decrease in activity. Cells resuspended in distilled water, fermentation supernatant, or 100 mM potassium phosphate buffer (pH 7.8) were found to have no loss in activity or protein concentration 0"dged by SDS-PAGE) when stored for 11 days at room temperature or 4 'C. Cell-free extract produced in potassium phosphate buffer showed no loss of aldolase activity when stored at 4 °C or room temperature for 5 days. Cells can be broken open in phosphate buffer, up to 25% culture supernatant or water with comparable recovery of aldolase activity; however, addition of 1 mM MgCb to water was found to slightly improve the aldolase activity. These data show that the aldolase protein is sufficiently stable to be commercially useful.<br>
[0951]	A  number  of embodiments  of the   invention  have  been  described.     The<br>
embodiments of the invention include one or more of the above described aspects. It will be understood that various modifications may be made without departing from the spirit and scope in accordance with the mvention. Accordingly, other embodiments are within the scope of the following claims.<br>
<br>
<br>
WE CLAIM:<br>
1.	A method, comprising: producing a product chosen from monatin, monatin<br>
derivatives, salts thereof, and combinations thereof, in a multi-step pathway, wherein a reaction in the pathway is facilitated by one or more polypeptides chosen from isolated or recombinant polypeptides comprising the amino acid sequence of SEQ ID N0:2, SEQ ID N0:4, SEQ ID N0:6, SEQ ID 'N0:8, SEQ ID NO: 10, SEQ ID NO:12, SEQ ID N0:14, SEQ ID N0:16, SEQ ID N0:18, SEQ ID NO:20, SEQ ID NO:22, SEQ ID NO:24, SEQ ID NO:26, SEQ ID NO:28, SEQ ID NO:30, SEQ ID NO:32, SEQ ID NO:34, SEQ ID NO:36, SEQ ID NO:38, SEQ ID NO:40, SEQ ID NO:42, SEQ ID NO:44, SEQ ID NO:46, SEQ ID NO:48, SEQ ID NO:50, SEQ ID HO;52, SEQ ID NO:54, SEQ ID NO:56, SEQ ID NO:58, SEQ ID NO:60, SEQ ID NO:64, SEQ ID NO:66, SEQ ID NO:68, SEQ ID NO:70, SEQ ID NO:72, SEQ ID NO:74, SEQ ID NO:76, SEQ ID NO:78, SEQ ID NO:80, SEQ ID NO:82, SEQ ID NO:84, SEQ ID NO:86, SEQ ID NO:88, SEQ ID NO:90, SEQ ID NO:92, SEQ ID NO:94, SEQ ID NO:96, SEQ ID NO:98, SEQ ID NO:100, SEQ ID NO;104, SEQ ID NO:106, SEQ ID NO:108, SEQ ID NO:110, SEQ ID N0:1I2, SEQ ID NO:114, SEQ ID N0:116, SEQIDN0:118, SEQ IDNO:120, SEQ IDNO:122, SEQIDNO:124, SEQ ID NO:126, SEQ ID NO:128, SEQ ID NO:130, SEQ ID NO:132, SEQ ID NO:I34, SEQ ID NO:136, SEQ ID NO:138, SEQ ID NO:140, SEQ ID NO:142, SEQ ID NO:144, SEQ ID NO:I46, SEQ ID NO:148, SEQ ID NO:150, SEQ ID NO:152, SEQ ID NO:154, SEQ ID NO:156, SEQ ID NO:158, SEQ ID NO:!60, SEQ ID NO:!62, SEQ ID NO:164, SEQ ID NO:166, SEQ ID NO:168, SEQ ID NO:170, SEQ ID NO:172, SEQ ID NO:174, SEQ ID NO:176, SEQ ID NO:178, SEQ ID NO:180, SEQ ID NO:182, SEQ ID NO:184, SEQ ID NO:186, SEQ ID NO:188, SEQ ID NO:190, SEQ ID NO:192, SEQ ID NO:194, SEQ ID NO:196, SEQ ID NO:198, SEQ ID NO:200, SEQ ID NO:204, SEQ ID NO;206, SEQ ID NO:208, SEQ ID NO:210, SEQ ID NO:212, SEQ ID NO:214, SEQ ID NO:216, SEQ ID NO:218, SEQ ID NO:220, SEQ ID NO:222, SEQ ID NO:224, SEQ ID NO:226, SEQ ID NO:228, SEQ ID NO:230, SEQ ID NO:232, SEQ ID NO:234, SEQ ID NO:236, SEQ ID NO:238, SEQ ID NO:240, SEQ ID NO:242, SEQ ID NO:244, SEQ ID NO:246, SEQ ID NO:248, SEQ ID NO:250, SEQ ID NO:252, SEQ ID NO:254, SEQ ID NO:256, SEQ ID NO:258, SEQ ID NO:260, SEQ ID NO:262, SEQ ID NO:264, SEQ ID NO:266, SEQ<br>
 <br>
ID NO:268, SEQ ID NO:270, SEQ ID NO:272, SEQ ID NO:274, SEQ ID NO:276, SEQ ID NO;278, SEQ ID NO:2!52, SEQ ID NO:284, SEQ ID NO:286, SEQ ID NO:288, SEQ ID NO:290, SEQ ID NO:292, SEQ ID NO:294, SEQ ID NO:296, SEQ ID NO:298, SEQ ID NO:300, SEQ ID NO:302, SEQ ID NO:304, SEQ ID NO:306, SEQ ID NO:308, SEQ ID NO:310, SEQ ID NO:312, SEQ ID N0:3I4, SEQ ID NO:316, SEQ ID NO:318, SEQ ID NO:320, SEQ ID NO:322, SEQ ID NO:324, SEQ ID NO:326, SEQ ID KO:328, SEQ ID NO:330, SEQ ID NO:332, or SEQ ID NO:334, or fragments or subsequences thereof having aldolase activity.<br>
The method of claim 1, wherein the one or more polypeptides are chosen from polypeptides that comprise the amino acid sequence of SEQ ID N0:2, SEQ ID N0:4, SEQ ID N0:6, SEQ ID N0:8, SEQ ID NO:I0, SEQ ID N0:I2, SEQ ID N0:I4, SEQ ID NO: 16, SEQ ID N0:18, SEQ ID NO:20, SEQ ID NO:22, SEQ ID NO:24, SEQ ID NO:26, SEQ ID NO:28, SEQ ID NO:30, SEQ ID NO:32, SEQ ID NO:34, SEQ ID NO:36, SEQ ID NO:38, SEQ ID NO:40, SEQ ID NO:42, SEQ ID NO:44, SEQ ID NO:46, SEQ ID NO:48, SEQ ID NO:50, SEQ ID NO:52, SEQ ID NO:54, SEQ ID NO:56, SEQ ID NO:58, SEQ ID NO:60, SEQ ID NO:64, SEQ ID NO:66, SEQ ID NO:68, SEQ ID NO:70, SEQ ID NO:72, SEQ ID NO:74, SEQ ID NO:76, SEQ ID NO:78, SEQ ID NO:80, SEQ ID "NO:82, SEQ ID NO:84, SEQ ID NO:86, SEQ ID NO:88, SEQ ID NO:90, SEQ ID NO:92, SEQ ID NO:94, SEQ ID NO:96, SEQ ID NO:98, SEQ ID NO: 100, SEQ ID NO: 104, SEQ ID NO: 5 06, SEQ IP NO: 108, SEQ IDNO:nO, SBQIDN0:112, SEQIDN0:1I4, SEQIDN0:1I6, SEQIDN0:118, SEQ ID NO:120, SEQ ID N0:I22, SEQ ID N0:124, SEQ ID N0:126, SEQ ID NO:128, SEQ ID NO:130, SEQ ID NO:I32, SEQ ID N0:134, SEQ ID N0:136, SEQ ID NO:I38, SEQ ID NO:140, SEQ ID NO:I42, SEQ ID NO:I44, SEQ ID NO:I46, SEQ ID NO:I48, SEQ ID NO;I50, SEQ ID NO:152, SEQ ID NO:I54, SEQ ID NO:156, SEQ ID NO:158, SEQ ID NO:160, SEQ ID NO:I62, SEQ ID NO:164, SEQ ID NO:I66, SEQ ID NO:168, SEQ ID NO:170, SEQ ID NO:172, SEQ ID NO:174, SEQ ID NO:176, SEQ ID NO:178, SEQ ID NO:180, SEQ ID NO:182, SEQ ID NO:184, SEQIDNO:186, SEQ IDNO:188, SEQ IDNO:190, SEQ lDNO:192, SEQ ID NO:194, SEQ ID NO:196, SEQ ID NO:198, SEQ ID NO:200, SEQ ID NO:204, SEQ ID NO:206, SEQ ID NO:208, SEQ ID NO:210, SEQ ID NO:212, SEQ ID NO;2!4, SEQ ID NO:216, SEQ ID NO:218, SEQ ID NO:220, SEQ ID NO:222, SEQ<br>
 <br>
ID NO:224, SEQ ID NO:226, SEQ ID NO:228, SEQ ID NO:230, SEQ ID NO:232, SEQ ID NO;234, SEQ ID NO:236, SEQ ID NO:238, SEQ ID NO:240, SEQ ID NO:242, SEQ ID NO:244, SEQ ffii NO:246, SEQ ID NO:248, SEQ ID NO:250, SEQ ID NO;252, SEQ ID NO:254, SEQ ID NO:256, SEQ ID NO;258, SEQ JD NO;260, SEQ ID NO:262, SEQ ID NO:264, SEQ ID NO:266, SEQ ID NO:268, SEQ ID NO:270, SEQ ID NO:272, SEQ IE' NO:274, SEQ ID NO:276, SEQ ID NO:278, SEQ ID NO:282, SEQ ID NO:284, SEQ ID NO:286, SEQ ID NO:288, SEQ ID NO:290, SEQ ID NO:292, SEQ ID NO:294, SEQ ID NO:296, SEQ ID NO:298, SEQ ID NO:300, SEQ ID NO:302, or SEQ ID NO:304, or fragments or subsequences thereof having aldolase activity.<br>
3.	The method of claim 1, wherein one or more steps in said multi-step pathway is a chemical step.<br>
4.	The^method of claim 1, .wherein, the reaction is a reaction between indole-3-pymvate and a C3 carbon source.<br>
5.	The method of claim 4, wherein the one or more poiypetides are chosen from polypeptides that comprise the amino acid sequence of SEQ ID N0:4, SEQ ID N0:8, SEQ JD N0:12, SEQ ID NO:28, SEQ ID NO:36, SEQ ID NO:40, SEQ ID NO:42, SEQ ID NO:44, SEQ ID NO:46, SEQ ID NO:50, SEQ ID NO;54, SEQ ID NO:58, SEQ ID NO:64, SEQ ID NO:74, SEQ ID NO:76, SEQ ID NO:80, SEQ ID NO:88, SEQ ID NO:94, SEQ ID NO:96, SEQ E) NO:106, SEQ ID NO:I08, SEQ ID N0:l]2, SEQ ID NO:n6, SEQ ID NO:122, SEQ ID NO:126, SEQ ID NO:130, SEQ ID NO:I34, SEQ ID NO:142, SEQ ID NO:148, SEQ ID NO:155, SEQ ID NO;]62, SEQ ID NO:166, SEQ ID NO:168, SEQ ID NO:172, SEQ ID NO:178, SEQ ID NO:180, SEQ ID NO:184, SEQ ID NO:186, SEQ ID NO;192, SEQ ID NO:200, SEQ ID NO:218, SEQ ID NO:224, SEQ ID NO:226, SEQ ID NO:228, SEQ ID NO:236, SEQ ID NO;238, SEQ ID NO:240, SEQ ID NO:244, SEQ ID NO-.250, SEQ ID NO:252, SEQ ID NO:264, SEQ ID NO:268, SEQ ID NO:270, SEQ ID NO:272, SEQ ID NO:276, SEQ ID NO:278, SEQ ID NO:282, SEQ ID NO:284, SEQ ID NO:298, SEQ ID NO:300, or SEQ ID NO:304.<br>
 <br>
6.	The method of claim 4, wherein the reaction preferentially produces R-2-hydroxy-2-(indol-3-yl-methyi)-4-ketoglutaric acid over S--2-hydroxy-2-(indol-3-yl-methyl)-4-ketoglutaric acid.<br>
7.	The method of claim 6, wherein the one or more polypeptides are chosen from polypeptides that comprise the amino acid sequence of SEQ ID NO: 28, SEQ ID N0:116, SEQ ID NO:298, SEQ IT) NO: 44, SEQ ID NO:54, SEQ ID NO: 148, SEQ ID NO: 46, SEQ ID NO:134, SEQ ID NO:142, SEQ ID NO:122, SEQ ID NO:74, SEQ ID NO: 64, SEQ ID NO: 108, SEQ ID NO:96, SEQ ID NO:126, SEQ ID NO:80, SEQ ID NO:36, SEQ ID NO:62, SEQ ID N0:112, SEQ ID NO:]30, SEQ ID NO:94, SEQ ID NO:58, SEQ ID NO:50, SEQ ID NO:106, SEQ ID NO:42, SEQ ID NO:278, SEQ ID NO:162, SEQ ID NO:276, SEQ ID NO:178, SEQ ID NO:166, SEQ ID NO;2I8, SEQ ID NO:224, SEQ ID NO:226, SEQ ID NO:244, SEQ ID NO:250, SEQ ID NO:252, SEQ ID NO:264, SEQ ID NO:268, SEQ ID NO:272, SEQ ID NO:184, SEQ ID NO:282, SEQ ID NO:186, SEQ ID NO: 192, SEQ ID NO:200, SEQ ID KO:284, SEQ ID NO:172, SEQ ID NO:180, SEQ ID NO:168, SEQ ID NO:228, SEQ ID NO:236, SEQ IDNO:238, SEQ ID NO:240, or SEQ ID NO:156.<br>
8.	The method of claim 6, wherein the one or more polypeptides are chosen from polypeptides that comprise the amino acid sequence of SEQ ID NO:28, SEQ ID NO:298, SEQ ID NO:54, SEQ ID NO:142, SEQ ID NO:I22, SEQ ID NO:108, SEQ ID NO:96, SEQ ID NO:80, SEQ ID NO:36, SEQ ID NO:62, SEQ ID NO:n2, SEQ ID NO:130, SEQ ID NO:94, SEQ ID NO:58, SEQ ID NO:50, SEQ ID NO:106, SEQ ID NO:42, SEQ ID NO:276, SEQ ID NO:166, SEQ ID NO:224, SEQ ID NO:226, SEQ ID MO:244, SEQ ID NO:I80, SEQ ID NO:I68, SEQ ID NO:22S, SEQ ID NO:238,orSEQIDNO:156.<br>
9.	The method of claim 6, wherem the one or more polypeptides are chosen from polypeptides that comprise the amino acid sequence of SEQ ID NO:28, SEQ ID NO:276, or SEQ ID NO:244.<br>
10.	The method of claim 1, wherein the product is chosen from monatin, salts thereof and combinations thereof<br>
 <br>
11.	The method of claim 1, wherein the reaction facilitated by said one or more polypeptides is performed in about 1.0 to about 10.0 mM MgCh.<br>
12.	The method of claim 1, wliereia the reaction facilitated by said one or more polypeptides is performed at about pH 7.0 to about pH 11.5.<br>
13.	The method of claim 1, wlierein the reaction facilitated by said one or more polypeptides is performed in about 0.005% to about 1% polysorbate detergent.<br>
14.	The method of claim 7, wherein at least one of R,R-monatin, R-S monatin, or a combination thereof is produced in greater quantity than either S,S-nionatin or S,R-monatin.<br>
15.	The method of claim 7, wherein R,R monatin is produced in greater quantity than R,S-monatin, S,S-monatin and S,R-monatin.<br>
16.	The method of claim 15, wherein said R,R monatin produced is at least about 60%, by weight, of total monatin pi'oduced.<br>
17.	The method of claim 16, wherein said R,R monatin produced is at least about 80%, by weight, of total monatin produced.<br>
18.	The method of claim 17, wherein said R,R monatin produced is at least about 90%, by weight, of tola! monatin produced.<br>
19.	The method of claim 17, wherein said R,R monatin produced is at least about 98%i, by weight, of total monatin produced.<br>
20.	The method of claim 1, wherein the fragments or subsequences thereof have an aldolase activity of at least 0.2 mg MP/mg protein/hr.<br>
21.	A method, comprising: producing a product chosen from monatin, monatin derivatives, salts thereof, and combinations thereof, in a multi-step pathway, wherein a reaction in the pathway is facilitated by at least one polypeptide encoded by a nucleic acid sequence that comprises a sequence having a percent sequence identity of at least 50% to SEQ ID N0:1, SEQ ID N0:3, SEQ ID N0:5, SEQ ID N0:7, SEQ ID N0:9, SEQ ID N0:11, SEQ ID N0:I3, SEQ ID N0:15, SEQ ID N0:17, SEQ ID<br>
 <br>
N0:19, SEQ ID N0:21, SEQ ID NO:23, SEQ ID NO:25, SEQ ID NO;27, SEQ ID NO;29, SEQ ID N0:31, SEQ ID NO:33, SEQ ID NO;35, SEQ ID NO;37, SEQ ID NO:39, SEQ ID N0:41, SEQ E) NO:43, SEQ ID NO:45, SEQ ID NO:47, SEQ ID NO:49, SEQ ID N0:51, SEQ ID NO:53, SEQ ID NO:55, SEQ ID NO:57, SEQ ID NO:59, SEQ ID NO:63, SEQ U) NO:65, SEQ ID NO:67, SEQ ID NO:69, SEQ ID N0:71, SEQ ID NO:73, SEQ JD NO:75, SEQ ID NO:77, SEQ ID NO:79, SEQ ID N0:8I, SEQ ID NO:83, SEQ ID NO:85, SEQ ID NO:87, SEQ ID NO:89, SEQ ID N0:91, SEQ ID NO:93, SEQ ID NO:95, SEQ ID NO:97, SEQ ID NO:99, SEQ ID NO:103, SEQIDNO:105, SEQ IDNO:107, SEQIDNO:109, SEQ IDN0:111, SEQ ID N0:113, SEQ ID N0:115, SEQ ID N0:li7, SEQ ID N0:119, SEQ ID N0:121, SEQ m NO:l23, SEQ ID NO:I25, SEQ ID NO:I27, SEQ ID NO:I29, SEQ ID NO:131, SEQIDNO:133, SEQ IDNO:I35, SEQIDNO:]37, SEQ IDNO:I39, SEQ ID N0:141, SEQ ID N0;K3, SEQ ID NO:245, SEQ ID NO:147, SEQ ID NO;]49, SEQ ID NO:151, SEQ ID NO:153, SEQ ID NO:155, SEQ ID NO:157, SEQ ID NO:159, SEQ ID N0:161, SEQ ID NO:163, SEQ ID NO:165, SEQ ID NO:167, SEQ ID NO:169, SEQ ID N0:171, SEQ ID NO:173, SEQ ID NO:175, SEQ ID NO:177, SEQ ID NO:179, SEQ ID N0:181, SEQ ID NO:183, SEQ ID NO:185, SEQ ID NO:187, SEQ IDNO:189, SEQ IDN0:19l, SEQ ID NO:193, SEQ ID NO:195, SEQ ID NO:197, SEQ ID NO:199, SEQ ID NO:203, SEQ ID NO:205, SEQ ID NO:207, SEQ ID NO:209, SEQ ID N0:21I, SEQ ID NO:213, SEQ ID NO:215, SEQ ID NO:2]7, SEQ ID NO:219, SEQ ID NO:221, SEQ ID NO:223, SEQ ID NO:225, SEQ ID NO:227, SEQ ID NO:229, SEQ ID NO:23I, SEQ ID NO:233, SEQ ID NO:235, SEQ ED NO:237, SEQ ID NO:239, SEQ ID NO:24I, SEQ ID NO:243, SEQ ID NO;245, SEQ ID NO:247, SEQ ID NO:249. SEQ ID NO:251, SEQ ID NO:253, SEQ ID NO:255, SEQ ID NO:257, SEQ ID NO:259, SEQ ID NO:261, SEQ ID NO:263, SEQ ID NO:265, SEQ ID NO:267, SEQ ID NO:269, SEQ ID NO:271, SEQ ID NO:273, SEQ ID NO:275, SEQ ID NO:277, SEQ ID NO:281, SEQ ID NO:283, SEQ ID NO:285, SEQ ID NO:287, SEQ ID NO:289, SEQ ID NO;291, SEQ ID NO:293, SEQ ID NO:295, SEQ ID NO:297, SEQ ID NO:299, SEQ ID NO:30U SEQ ID NO:303, SEQ ID NO:305, SEQ ID NO:307, SEQ ID NO;309. SEQ ID N0:3U, SEQ ID NO:313, SEQ ID NO:315, SEQ ID NO:317, SEQ ID NO:319, SEQ ID NO:321, SEQ ID NO:323, SEQ ID NO:325, SEQ ID NO:327, SEQ ID NO:329, SEQ ID<br>
 <br>
NO:331, SEQ ID NO:333, SEQ ID NO:335, SEQ ID NO:336, SEQ ID NO;337, or SEQIDNO:338.<br>
22.	The method of claim 21, wherein the percent sequence identity is at least 95%.<br>
23.	The meUiod of claim 22, wherein the percent sequence identity is 100%.<br>
24.	The method of claim 23, wherein the at least one polypeptide is encoded by a nucleic acid sequence that comprises a sequence having at least 95% identity to SEQ ID N0:4, SEQ ID NO:S, SEQ ID N0:I2, SEQ ID NO:28, SEQ ID NO:36, SEQ ID NO:40, SEQ ID NO:42, SEQ U) NO:44, SEQ ID NO:46, SEQ ID NO:50, SEQ ID NO:54, SEQ ID NO:58, SEQ ID NO:64, SEQ ID NO:74, SEQ ID NO:76, SEQ ID NO:80, SEQ ID NO:88, SEQ ID NO:94, SEQ ID NO:96, SEQ ID NO:106, SEQ ID NO:108, SEQ IDN0:112, SEQ IDN0:116, SEQ IDNO:I22, SEQ ID NO:126, SEQ ID NO:130, SEQ ID NO:134, SEQ ID NO:142, SEQ ID NO:148, SEQ ID NO:156, SEQ ID NO:162, SEQ ID NO;166, SEQ ID NO:168, SEQ ID NO:172, SEQ ID NO:178, SEQ ID NO:180, SEQ ID NO:I84, SEQ ID NO:186, SEQ ID NO:192, SEQ ID NO:200, SEQ ID NO:218, SEQ ID NO:224, SEQ ID NO:226, SEQ ID NO:228, SEQ ID NO;236, SEQ ID NO:2.3S, SEQ ID NO:240, SEQ ID NO:244, SEQ ID NO:250, SEQ ID NO:252, SEQ ID NO:264, SEQ ID NO:268, SEQ ID NO:270, SEQ ID NO:272, SEQ ID NO:276, SEQ ID NO:278, SEQ ID NO:282, SEQ ID NO:284, SEQ ID NO;298, SEQ ID NO:300, and SEQ ID NO:304.<br>
25.	The method of 21, the reaction preferentially produces R-2-hydroxy-2-(indol-3-yJ-methyl )-4-ketog]utaric acid over S—2-hydroxy-2-(indol-3-y]-methyl)-4-ketoglutaric acid and the at least one polypeptide is encoded by a nucleic acid sequence that comprises a sequence having at least 95% sequence identity to SEQ ID NO: 28, SEQ IDNOilie, SEQ ID NO:298, SEQ ID NO: 44, SEQ ID NO:54, SEQ ID NO: 148, SEQ ID NO: 46, SEQ ID NO:134, SEQ ID NO:142, SEQ ID NO:122, SEQ ID NO:74, SEQ ID NO: 64, SEQ ID NO: 108, SEQ ID NO:96, SEQ ID NO:126, SEQ ID NO:80, SEQ ID NO:36, SEQ ID NO:62, SEQ ID N0:l]2, SEQ ID NO:130, SEQ ID NO:94, SEQ ID NO:58, SEQ ID NO:50, SEQ ID NO:106, SEQ ID NO:42, SEQ ID NO:278, SEQ ID NO:162, SEQ ID NO:276, SEQ ID NO:178, SEQ ID NO1I66, SEQ ID NO:218, SEQ ID NO:224, SEQ ID NO:226, SEQ ID NO:244, SEQ ID<br>
 <br>
NO:250, SEQ ID NO:252, SEQ ID NO:264, SEQ ID NO:268, SEQ ID NO:272, SEQ ID NO:I84, SEQ ID NO:282, SEQ ID NO:186, SEQ ID NO:192, SEQ ID NO:200, SEQ ID NO:284, SEQ ID NO:172, SEQ ID NO:180, SEQ ID NO:168, SEQ ID NO:228, SEQ ID NO:236, SEQ ID NO:238, SEQ ID NO:240, and SEQ ID N0:156.<br>
26.	The method of claim 25, wherein the at least one polypeptide is encoded by a nucleic acid sequence that comprises a sequence having at least 95% sequence identity to SEQ ID NO;28, SEQ ID NO:298, SEQ ID NO:54, SEQ ID NO:142, SEQ ID NO:122, SEQ ID NO:108, SEQ ID NO:96, SEQ ID NO:80, SEQ ID NO:36, SEQ ID NO;62, SEQ ID NO:112, SEQ ID NO:I30, SEQ ID NO:94, SEQ ID NO:5S, SEQ IDNO:50, SEQ ID NO:I06, SEQ ID NO:42, SEQ ID NO:276, SEQ ID NO:166, SEQ ID NO;224, SEQ ID NO:226, SEQ ID NO;244, SEQ ID NO:IS0, SEQ ID NO:I68, SEQ ID NO:228, SEQ ID NO:238, and SEQ ID NO:156.<br>
27.	-The. method of claim 26, wherein the at least one polypeptide is encoded by a nucleic acid sequence that comprises a sequence having at least 95% sequence identity to SEQ ID NO:28, SEQ ID NO:276, and SEQ ID NO:244.<br>
28.	A method, comprising: producing a product chosen from monatin, monatin derivatives, salts thereof, and combinations thereof, in a multi-step pathway, wherein a reaction in the pathway is facilitated by at least one polypeptide encoded by a nucleic acid sequence that comprises a sequence that hybridizes under stringent condition to a nucleic acid of SEQ ID NO:I, SEQ ID N0:3, SEQ ID N0:5, SEQ ID N0:7, SEQ ID N0:9, SEQ ID N0:11, SEQ ID N0:13, SEQ ID N0:15, SEQ ID N0:17, SEQ ID N0;I9, SEQ ID N0:21, SEQ ID NO:23, SEQ ID NO-.25, SEQ ID NO:27, SEQ ID NO:29, SEQ ID N0:31, SEQ ID NO:33, SEQ ID NO:35, SEQ ID NO:37, SEQ ID NO:39, SEQ ID N0:4I, SEQ ID NO:43, SEQ ID NO:45, SEQ ID NO:47, SEQ ID NO:49, SEQ ID N0:51, SEQ ID NO:53, SEQ ID NO:55, SEQ ID NO:57, SEQ ID NO:59, SEQ ID NO:63, SEQ ID NO:65, SEQ ID NO:67, SEQ ID NO:69, SEQ ID N0:71, SEQ ID NO:73, SEQ ID NO:75, SEQ ID NO:77, SEQ ID NO:79, SEQ ID N0:81, SEQ ID NO:83, SEQ ID NO:85, SEQ ID NO:87, SEQ ID NO:89, SEQ ID N0:91, SEQ ID NO:93, SEQ ID NO:95, SEQ ID NO:97, SEQ ID NO:99, SEQ ID NO:103, SEQ ID NO:105, SEQ ID NO:107, SEQ ID NO:i09, SEQ IDNOilll, SEQIDN0:113, SEQIDNO:il5, SEQIDN0:117, SEQIDN0:1I9,<br>
 <br>
SEQ ID NO:]23, SEQ ID NO:]23, SEQ ID NO:I25, SEQ ID NO;!27, SEQ ID NO:129, SEQ ID N0:13L SEQ IE' NO:133, SEQ ID NO:135, SEQ ID NO:137, SEQ ID NO:139, SEQ ID N0:141, SEQ ID NO:143, SEQ ID NO:145, SEQ ID NO:147, SEQ ID NO-.149, SEQ ID N0:15I, SEQ ID NO:153, SEQ ID N0:155, SEQ ID NO:157, SEQ ID NO:159, SEQ ID N0:161, SEQ ID NO:163, SEQ ID NO:165, SEQ ID NO:167, SEQ ID NO:169, SEQ ID N0;171, SEQ ID NO:173, SEQ ID NO:175, SEQ ID NO;177, SEQ ID NO;179, SEQ ID N0:181, SEQ ID NO:183, SEQ ID NO:185, SEQ ID NO:187, SEQ ID NO:I89, SEQ ID N0:19U SEQ ID NO:193, SEQ ID NO:195, SEQ ID NO:197, SEQ ID NO:199, SEQ ID NO:203, SEQ ID NO:205, SEQ ID NO:207, SEQ ID NO:209, SEQ ID N0:211, SEQ ID NO:213, SEQ ID NO;215, SEQ ID NO:217, SEQ ID NO:219, SEQ ID NO:221, SEQ ID NO:223, SEQ ID NO:225, SEQ ID NO:227, SEQ ID NO:229, SEQ ID NO:231, SEQ ID NO:233, SEQ ID NO:235, SEQ ID NO:237, SEQ ID NO:239, SEQ ID NO:241, SEQ ID NO:243, SEQ ID NO:245, SEQ ID NO:247, SEQ ID NO:249, SEQ ID NO:251, SEQ ID NO:253, SEQ ID NO:255, SEQ ID NO:257, SEQ ID NO:259, SEQ ID NO:261, SEQ ID NO:263, SEQ ID NO;265, SEQ ID NO:267, SEQ ID NO:269, SEQ ID NO:27i, SEQ ID NO:273, SEQ ID NO:275, SEQ ID NO:277, SEQ ID NO:281, SEQ ID NOi283, SEQ ID NO:285, SEQ ID NO:287, SEQ ID NO:289, SEQ ID NO;291, SEQ ID NO:293, SEQ ID NO:295, SEQ ID NO:297, SEQ ID NO:299, SEQ ID NO:301, SEQ ID NO:303, SEQ ID NO:305, SEQ ID NO:307, SEQ ID NO:309, SEQ ID N0:311, SEQ ID NO:313, SEQ ID NO:315, SEQ ID NO:317, SEQ ID NO;319, SEQ ID NO:321, SEQ ID NO:323, SEQ ID NO:325, SEQ ID NO:327, SEQ ID NO:329, SEQ ID NO:331, SEQ ID NO:333, SEQ ID NO:335, SEQ ID NO:336, SEQ ID NO:337, or SEQ ID NO:338.<br>
29.	A method,comprising: producing R,R monatin in a multi-step pathway, wherein said R,R monatin produced is at least about 95%, by weight, of total monatin produced.<br>
30.	A method, comprising: producmg a product chosen from monatin precursor, salts thereof, and combinations thereof, in a pathway, wherein a reaction in the pathway is facilitated by one or more polypeptides chosen from isolated or recombinant polypeptides comprising the amino acid sequence of SEQ ID N0:2, SEQ<br>
 <br>
ID N0:4, SEQ ID N0:6, SEQ ID N0:8, SEQ ID NO:10, SEQ ID N0:!2, SEQ ID N0:14, SEQ ID N0:16, SEQ ID :MO:18, SEQ ID NO:20, SEQ ID NO:22, SEQ ID NO:24, SEQ ID NO:24 SEQ ID iSJO:28, SEQ ID NO:30, SEQ ID NO;32, SEQ ID NO:34, SEQ ID NO:36, SEQ ID NO:38, SEQ ID NO:40, SEQ ID NO:42, SEQ ID NO:44, SEQ ID NO:46, SEQ ID NO:48, SEQ ID NO:50, SEQ ID NO:52, SEQ ID NO:54, SEQ ID NO:56, SEQ ID NO:58, SEQ ID NO:60, SEQ ID NO:64, SEQ ID NO:66, SEQ ID NO:68, SEQ ID NO:70, SEQ ID NO:72, SEQ ID NO:74, SEQ ID NO:76, SEQ ID NO:78, SEQ ID NO:80, SEQ ID NO:82, SEQ ID KO:84, SEQ ID NO:86, SEQ ID NO:88, SEQ ID NO:90, SEQ ID NO:92, SEQ ID NO:94, SEQ ID NO:96, SEQ IDNO:98, SEQ IDNO:100, SEQ ID NO:104, SEQ ID NO:106, SEQ ID NO:108, SEQ ID NO:110, SEQ ID N0:112, SEQ ID N0:114, SEQ IDN0:116, SEQ ID N0:118, SEQ ID NO:120, SEQ ID NO:122, SEQ ID NO:124, SEQ ID NO:126, SEQ ID NO:128, SEQ ID NO:130, SEQ ID NO:132, SEQ ID NO:134, SEQ ID NO:136, SEQ ID NO:138, SEQ ID NO:140, SEQ ID NO:142, SEQ ID NO:144, SEQ ID NO:146, SEQ ID NO:148, SEQ ID NO:150, SEQ ID NO:152, SEQ ID NO:154, SEQ ID NO:156, SEQ ID NO:158, SEQ ID NO:160, SEQ ID NO:162, SEQ ID NO:164, SEQIDNO:166, SEQ IDNO:168, SEQ IDNO:170, SEQ IDNO:172, SEQ ID NO:174, SEQ ID NO:176, SEQ ID NO:178, SEQ ID NO:]80, SEQ ID NO:182, SEQ ID NO:I84, SEQ ID N0:1S6, SEQ ID NO:188, SEQ ID NO:190, SEQ ID NO:192, SEQ ID NO:194, SEQ ID NO:196, SEQ ID NO:198, SEQ ID NO;200, SEQ ID NO:204, SEQ ID NO:206, SEQ ID NO:208, SEQ ID NO:210, SEQ ID NO:212, SEQ ID NO:214, SEQ ID NO:216, SEQ ID NO:218, SEQ ID NO:220, SEQ ID NO:222, SEQ ID NO:224, SEQ ID NO;226, SEQ ID NO:228, SEQ ID NO:230, SEQ ID NO:232, SEQ ID NO:234, SEQ ID NO:236, SEQ ID NO:23S, SEQ ID NO;240, SEQ ID NO:242, SEQ ID NO:244, SEQ ID NO:246, SEQ ID NO:248, SEQ ID NO:250, SEQ ID NO:252, SEQ ID NO:254, SEQ ID NO:256, SEQ ID NO:258, SEQ ID NO:260, SEQ ID NO:262, SEQ ID NO:264, SEQ ID NO:266, SEQ ID NO:268, SEQ ID NO:270, SEQ ID NO:272, SEQ ID NO:274, SEQ ID NO:276, SEQ ID NO:278, SEQ ID NO:282, SEQ ID NO:284, SEQ ID NO:286, SEQ ID NO:288, SEQ ID NO-.290, SEQ ID NO:292, SEQ ID NO:294, SEQ ID NO:296, SEQ ID NO:298, SEQ ID NO:300, SEQ ID NO:302, SEQ ID NO:304, SEQ ID NO:306, SEQ ID NO:308, SEQ IDNO:310, SEQIDNO:312, SEQ IDNO:3I4, SEQ IDNO:3i6, SEQ<br>
 <br>
ID NO:318, SEQ ID NO:320, SEQ ID NO:322, SEQ ID NO:324, SEQ ID NO:326, SEQ ID NO:328, SEQ ID NO:330, SEQ ID NO:332, or SEQ ID NO:334, or fragments or subsequences thereof having aldolase activity, wherein said monatin precursor, salts thereof, and combinations thereof is used to make a sweetener product.<br>
31.	A method, comprising: producing a product chosen from monatin precursor,<br>
salts thereof, and combinations thereof, in a pathway, wherein a reaction in the pathway is facilitated by at least one polypeptide encoded by a nucleic acid sequence that comprises a sequence having a percent sequence identity of at least 50% to SEQ ID N0:1, SEQ ID N0:3, SEQ n&gt; N0:5, SEQ ID N0:7, SEQ ID N0:9, SEQ ID N0:11, SEQ ID N0:13, SEQ ID N0:15, SEQ ID N0:17, SEQ ID N0:19, SEQ ID N0:21, SEQ ID NO:23, SEQ ID NO:25, SEQ ID NO:27, SEQ ID NO:29, SEQ ID N0:31, SEQ ID NO:33, SEQ ID NO:35, SEQ ID NO:37, SEQ ID NO:39, SEQ ID N0:41, SEQ ID NO:43, SEQ ID NO:45, SEQ ID NO:47, SEQ ID NO:49, SEQ ID N0:51, SEQ ID NO:53, SEQ ID NO:55, SEQ ID NO:57, SEQ ID NO:59, SEQ ID NO:63, SEQ ID NO:65, SEQ ID NO:67, SEQ ID NO:69, SEQ ID N0:71, SEQ ID NO:73, SEQ ID NO:75, SEQ ID NO:77, SEQ ID NO:79, SEQ ID N0:81, SEQ ID NO:83, SEQ ID NO:85, SEQ K NO:87, SEQ ID NO:89, SEQ ID N0:9I, SEQ ID NO:93, SEQ ID NO:95, SEQ ID NO;97, SEQ ID NO:99, SEQ ID NO:103, SEQ ID NO:105, SEQ ID NO:107, SEQ ID NO:109, SEQ ID N0:111, SEQ ID N0:U3, SEQ ID N0:U5, SEQ ID N0:117, SEQ ID N0:119, SEQ ID N0:121, SEQ ID NO:123, SEQ ID NO:125, SEQ ID NO:127, SEQ ID NO:129, SEQ ID N0:131, SEQ ID NO:133, SEQ ID NO:135, SEQ ID NO:137, SEQ ID NO:139, SEQ ID N0:141, SEQ ID NO:143, SEQ ID NO:145, SEQ ID NO:147, SEQ ID NO:I49, SEQ ID N0:151, SEQ ID NO:153, SEQ ID NO:155, SEQ ID NO:157, SEQ ID NO:159, SEQ ID N0;161, SEQIDNO:163, SEQ IDNO:165, SEQIDNO:167, SEQ IDNO:169, SEQ ID N0:171, SEQ ID NO:173, SEQ ID NO:175, SEQ ID NO:177, SEQ ID NO:179, SEQ ID N0:IS1, SEQ ID N0:1S3, SEQ ID NO:185, SEQ ID NO:187, SEQ ID NO:189, SEQ ID N0:191, SEQ ID NO:193, SEQ ID NO:195, SEQ ID NO:197, SEQ ID NO:199, SEQ ID NO:203, SEQ ID NO:205, SEQ ID NO:207, SEQ ID NO:209, SEQ ID N0:2n, SEQ ID NO:213, SEQ ID NO:215, SEQ ID NO:217, SEQ ID NO:219, SEQ ID NO:221, SEQ ID NO:223, SEQ ID NO:225, SEQ ID NO:227, SEQ<br>
 <br>
ID NO:229, SEQ ID NO:23U SEQ ID NO:233, SEQ ID NO:235, SEQ ID NO:237, SBQ ID NO:239, SEQ ID NO:241, SEQ ID NO:243, SEQ ID NO:245, SEQ ID NO:247, SEQ ID NO:249, SEQ ID NO:251, SEQ ID NO:253, SEQ ID NO:255, SEQ ID NO:257, SEQ ID NO:259, SEQ ID NO:261, SEQ ID NO:263, SEQ ID NO:265, SEQ ID NO:267, SEQ ID NO:269, SEQ ID NO:271, SEQ ID NO:273, SEQ ID NO:275, SEQ ID NO:277, SEQ ID NO:281, SEQ ID NO:283, SEQ ID NO:285, SEQ ID NO:287, SEQ ID NO:289, SEQ ID NO:291, SEQ ID NO:293, SEQ ID NO:295, SEQ ID NO:297, SEQ ID NO:299, SEQ ID NO:301, SEQ ID NO:303, SEQ ID NO:305, SEQ ID NO:307, SEQ ID NO;309, SEQ ID NO:311, SEQ ID NO:313, SEQ ID NO:315, SEQ ID NO:317, SEQ ID NO:319, SEQ ID NO:321, SEQ ID NO:323, SEQ ID NO;325, SEQ ID NO:327, SEQ ID NO:329, SEQ ID NO;331, SEQ ID NO:333, SEQ ID NO:335, SEQ ID NO:336, SEQ ID NO:337, or SEQ ID NO:338, wherein said monatin precursor, salts thereof, and combinations thereof is used to make a sweetener product.<br>
32.	A method, comprising: producing a product chosen from monatin precursor,<br>
salts thereof, and combinations thereof, in a pathway, wherein a reaction in the pathway is facilitated by at least one polypeptide encoded by a nucleic acid sequence that comprises a sequence that hybridizes under stringent condition to a nucleic acid of SEQ ID N0;1, SEQ ID N0:3, SEQ ID N0;5, SEQ ID N0:7, SEQ ID N0:9, SEQ ID NOrll, SEQ ID N0:13, SEQ ID N0:15, SEQ IDN0:17, SEQ ID N0:19, SEQ ID N0:21, SEQ ID NO:23, SEQ ID NO:25, SEQ ID NO:27, SEQ ID NO:29, SEQ ID N0:31, SEQ ID NO:33, SEQ ID NO:35, SEQ ID NO:37, SEQ ID NO:39, SEQ ID N0:41, SEQ ID NO:43, SEQ ID NO:45, SEQ ID NO;47, SEQ ID NO:49, SEQ ID N0:51, SEQ ID NO:53, SEQ ID NO:55, SEQ ID NO:57, SEQ ID NO:59, SEQ ID NO:63, SEQ ID NO:65, SEQ ID NO:67, SEQ ID NO:69, SEQ ID N0:71, SEQ ID NO:73, SEQ ID NO:75, SEQ ID NO:77, SEQ ID NO:79, SEQ ID N0:81, SEQ ID NO:83, SEQ ID NO:85, SEQ ID NO;87, SEQ ID NO:89, SEQ ID N0:91, SEQ ID NO:93, SEQ ID NO;95, SEQ ID NO:97, SEQ ID NO:99, SEQ ID NO: 103, SEQ ID NO:105, SEQIDNO:I07, SEQ IDNO:109, SEQ IDN0:111, SEQ IDN0:113, SEQ IDN0:il5, SEQIDN0:117, SEQ ID N0:U9, SEQIDN0:121, SEQ ID NO:123, SEQ ID NO:125, SEQ ID NO:I27, SEQ ID NO:129, SEQ ID N0:131, SEQ ID NO:]33, SEQ ID NO:135, SEQ ID NO:137, SEQ IDNO:139, SEQ ID N0:141, SEQ<br>
 <br>
ID NO:i43, SEQ ID NO:145, SEQ ID NO:147, SEQ ID NO:149, SEQ ID N0:151, SEQ ID NO:153, SEQ ID NO:155, SEQ ID NO:i57, SEQ ID NO:159, SEQ ID N0:161, SEQ ID NO:163, SEQ ID NO:165, SEQ ID NO:167, SEQ ID NO:169, SEQ ID N0:171, SEQ ID NO:173, SEQ ID NO:175, SEQ ID NO:177, SEQ ID NO:179, SEQ ID N0:181, SEQ ID N0:183, SEQ ID NO:185, SEQ ID NO:187, SEQ ID NO:189, SEQ ID N0:191, SEQ ID NO:193, SEQ ID NO:195, SEQ ID NO:197, SEQ ID NO:199, SEQ ID NO;203, SEQ ID NO:205, SEQ ID NO:207, SEQ ID NO:209, SEQ ID N0:211, SEQ ID NO:213, SEQ ID NO:215, SEQ ID NO:217, SEQ ID NO:219, SEQ ID NO:221, SEQ ID NO:223, SEQ ID NO:225, SEQ ID NO:227, SEQ ID NO:229, SEQ ID NO;231, SEQ ID NO:233, SEQ ID NO:235, SEQ ID NO:237, SEQ ID NO:239, SEQ ID NO:241, SEQ ID NO:243, SEQ ID NO:245, SEQ ID NO:247, SEQ ID NO:249, SEQ ID NO:251, SEQ ID NO:253, SEQ ID NO:255, SEQ ID NO:257, SEQ ID NO:259, SEQ ID NO:261, SEQ ID NO:263, SEQ ID NO:265, SEQ ID NO;267, SEQ ID NO:269, SEQ ID NO:271, SEQ ID NO:273, SEQ ID NO:275, SEQ ID NO:277, SEQ ID NO:281, SEQ ID NO:283, SEQ ID NO:285, SEQ ID NO:287, SEQ ID NO:289, SEQ ID NO:291, SEQ ID NO:293, SEQ ID NO:295, SEQ ID NO:297, SEQ ID NO:299, SEQ ID NO:301, SEQ ID NO:303, SEQ ID NO:305, SEQ ID NO:307, SEQ ID NO:309, SEQ ID N0:31U SEQ ID NO:313, SEQ ID NO:315, SEQ ID NO:317, SEQ ID NO:319, SEQ ID NO:321, SEQ ID NO:323, SEQ ID NO;325, SEQ ID NO:327, SEQ ID NO:329, SEQ ID NO:331, SEQ ID NO:333, SEQ ID NOi335, SEQ ID NO:336, SEQ ID NO:337, or SEQ ID NO:338, wherein said monatin prec\irsor, salts thereof, and combinations thereof is used to make a sweetener product.<br>
 <br>
</www.stratagene.com></td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/viewdoc.aspx?id=agttMkbjaiX9ZZNd3bS/dQ==&amp;amp;loc=egcICQiyoj82NGgGrC5ChA==" target="_blank" style="word-wrap:break-word;">http://ipindiaonline.gov.in/patentsearch/GrantedSearch/viewdoc.aspx?id=agttMkbjaiX9ZZNd3bS/dQ==&amp;amp;loc=egcICQiyoj82NGgGrC5ChA==</a></p>
		<br>
		<div class="pull-left">
			<a href="272772-a-vibratory-tip.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="272774-a-process-for-the-preparation-of-alcohol-from-damaged-rice-grains.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>272773</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>5340/CHENP/2008</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>18/2016</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>29-Apr-2016</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>26-Apr-2016</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>06-Oct-2008</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>CARGILL, INCORPORATED</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td>15407 MCGINTY ROAD WEST, WAYZATA, MN 55391</td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>GORT, STEVEN, JOHN</td>
											<td>3207 QUARLWS ROAD, BRIIKLYN CENTER, MN 55429</td>
										</tr>
										<tr>
											<td>2</td>
											<td>HICKS, PAULA, M</td>
											<td>11601 TANGLEWOOD DRIVE, EDEN PRARIE, MN 55347</td>
										</tr>
										<tr>
											<td>3</td>
											<td>LUGINBUEL, PETER</td>
											<td>12802 CALLE DE LA SIENA, SAN DIEGO, CA 92130</td>
										</tr>
										<tr>
											<td>4</td>
											<td>MCFARLAN, SARA.C.</td>
											<td>2151 BAYARD AVENUE, ST. PAUL , MN 55116</td>
										</tr>
										<tr>
											<td>5</td>
											<td>RICHARDSON, TOBY</td>
											<td>11646 CANDY ROSE WAY, SAN DIEGO CA 92131</td>
										</tr>
										<tr>
											<td>6</td>
											<td>SOLHEID, CHRISTOPHER</td>
											<td>4218 WASHBURN AVENUE N., MINNEAPOLIS, MINNESOTA 55412</td>
										</tr>
										<tr>
											<td>7</td>
											<td>WEINER, DAVID</td>
											<td>13416 PORTOFINO DRIVE, DEL MAR, CA 92014</td>
										</tr>
										<tr>
											<td>8</td>
											<td>ZHAO, LISHAN</td>
											<td>6564 RED KNOT STREET, CARSLSBAD, CA 92009</td>
										</tr>
										<tr>
											<td>9</td>
											<td>BURKE, ELLEN</td>
											<td>3428 TALBOT STREET, SAN DIEGO, CA 92106</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>C12P17/00</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>PCT/US07/05718</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td>2007-03-07</td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td>60/853,005</td>
									<td>2006-10-20</td>
								    <td>U.S.A.</td>
								</tr>
								<tr>
									<td>2</td>
									<td>60/779,460</td>
									<td>2006-03-07</td>
								    <td>U.S.A.</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/272773-a-mehtod-of-producing-r-r-montain-a-salt-of-r-r-montain-or-a-combination-thereof by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 01:09:23 GMT -->
</html>
