<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/272721-synthetic-immunoglobulin-domains-with-binding-properties-engineered-in-regions-of-the-molecule-different-from-the-complementarity-determining-regions by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 00:53:47 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 272721:&quot;SYNTHETIC IMMUNOGLOBULIN DOMAINS WITH BINDING PROPERTIES ENGINEERED IN REGIONS OF THE MOLECULE DIFFERENT FROM THE COMPLEMENTARITY DETERMINING REGIONS&quot;</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">&quot;SYNTHETIC IMMUNOGLOBULIN DOMAINS WITH BINDING PROPERTIES ENGINEERED IN REGIONS OF THE MOLECULE DIFFERENT FROM THE COMPLEMENTARITY DETERMINING REGIONS&quot;</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>Method for engineering an immunoglobulin comprising at least one modification in a structural loop region of said immunoglobulin and determining the binding of said immunoglobulin to an epitope of an antigen, wherein the unmodified immunoglobulin does not significantly bind to said epitope, comprising the steps of: - providing a nucleic acid encoding an immunoglobulin com- prising at least one structural loop region, - modifying at least one nucleotide residue of at least one of said structural loop regions, - transferring said modified nucleic acid in an expression system, - expressing said modified immunoglobulin, -contacting the expressed modified immunoglobulin with an epitope, and -determining whether said modified immunoglobulin binds to said epitope, as well as modified immunoglobulins.</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td>SYNTHETIC IMMUNOGLOBULIN DOMAINS WITH BINDING PROPERTIES ENGINEERED IN REGIONS OF THE MOLECULE DIFFERENT FROM THE COMPLEMENTARITY DETERMINING REGIONS<br>
The present invention relates to a method for engineering and manufacturing of a modified imiminoglobulin.<br>
The general field is the engineering of proteins with the aim to impart them with specific binding properties. More specifically, the engineered proteins of relevance here are immuno-globulins (antibodies), and even more specifically, single domains or pairs or combinations of single domains of immuno-globulins. The specific binding properties of immunoglobulins are important features since they control the interaction with other molecules such as antigens, and render immunoglobulins useful for diagnostic and therapeutic applications.<br>
The basic antibody structure will be explained here using as example an intact IgGl immunoglobulin.<br>
Two identical heavy (H) and two identical light (L) chains combine to form the Y-shaped antibody molecule. The heavy chains each have four domains. The amino terminal variable domains (VH) are at the tips of the Y. These are followed by three constant domains: CH1, CH2, and the carboxy terminal CH3, at the base of the Y's stem. A short stretch, the switch, connects the heavy chain variable and constant regions. The hinge connects CH2 and CHS (the Fc fragment) to the remainder of the antibody (the Fab fragments) . One Fc and two identical Fab fragments can be produced by proteolytic cleavage of the hinge in an intact antibody molecule. The light chains are constructed of two domains, variable (VL) and constant (CL), separated by a switch.<br>
Disulfide bonds in the hinge region connect the two heavy chains. The light chains are coupled to the heavy chains by additional disulfide bonds. Asn-linked carbohydrate moieties are attached at different positions in constant domains depending pn the class of immunoglobulin. For IgGl two disulfide bonds in the hinge region, between Cys235 and Cys238 pairs, unite the two heavy chains. The light chains are coupled to the heavy chains by two additional disulfide bonds, between Cys229,s in the CHI domains and Cys214s in the CL domains. Carbohydrate moieties are attached to Asn306 of each CH2, generating a pronounced bulge in the stem of the Y.<br>
These features have profound functional consequences. The variable regions of both the heavy and light chains (VH) and (VL) lie at the "tips" of the Y, where they are positioned to react with antigen. This tip of the molecule is the side on which the N-terminus of the amino acid sequence is located. The stem of the Y projects in a way to efficiently mediate effector functions such as the activation of complement and interaction with Fc receptors, or ADCC and ADCP. Its CH2 and CHS domains bulge to facilitate interaction with effector proteins. The C-terminus of the amino acid sequence is located on the opposite side of the tip, which can be termed "bottom" of the Y. The structure of an intact IgGl is illustrated in Figure la.<br>
Two types of light chain, termed lambda (X) and kappa (K),.. are found in antibodies. A given immunoglobulin either has K chains or X chains, never one of each. No functional difference has been found between antibodies having X or K light chains.<br>
The structural organization of the main human immunoglobulin class monomers is shown in Figure Ib. The classes differ in the composition and sequence of their respective heavy chains. Both IgM and IgE lack a hinge region but each contains an extra heavy-chain domain (CH4). Numbers and locations of the disulfide bonds (lines) linking the chains differ between the isotypes. They also differ in the distribution of N-linked carbohydrate groups, symbolically shown as circles.<br>
Each domain in an antibody molecule has a similar structure of two beta sheets packed tightly against each other in a compressed antiparallel beta barrel. This conserved structure is termed the immunoglobulin fold. The immunoglobulin fold of constant domains contains a 3-stranded sheet packed against a 4-stranded sheet. The fold is stabilized by hydrogen bonding between the beta strands of each sheet, by hydrophobic bonding between residues of opposite sheets in the interior, and by a disulfide bond between the sheets. The 3-stranded sheet comprises strands C, F, and G, and the 4-stranded sheet has strands A, B, E, and D. The letters A through G denote the sequential positions of the beta strands along the amino acid sequence of the immunoglobulin fold.<br>
The fold of variable domains has 9 beta strands arranged in two sheets of 4 and 5 strands. The 5-stranded sheet is structurally homologous to the 3-stranded sheet of constant domains, but<br>
contains the extra strands C1 and C11. The remainder of the strands (A, B, C, D, E, F, G) have the same topology and similar structure as their counterparts in constant domain immuno-globulin folds. A disulfide bond links strands B and F in opposite sheets, as in constant domains. The immunoglobulin fold is illustrated in Figure 2 for a constant and a variable domain of an immunoglobulin.<br>
The variable domains of both light and heavy immunoglobulin chains contain three hypervariable loops, or complementarity-determining regions (CDRs). The three CDRs of a V domain (CDRl, CDR2, CDR3) cluster at one end of the beta barrel. The CDRs are loops that connect beta strands B-C, C'-C'1, and F-G of the immunoglobulin fold. The residues in the CDRs vary from one immunoglobulin molecule to the next, imparting antigen specificity to each antibody.<br>
The VL and VH domains at the tips of antibody molecules are closely packed such that the 6 CDRs (3 on each domain) cooperate in constructing a surface (or cavity) for antigen-specific binding. The natural antigen binding site of an antibody thus is composed of the loops which connect strands B-C, C'-C11, and F-G of the light chain variable domain and strands B-C, C'-C1', and F-G of the heavy chain variable domain.<br>
Using the 3D structure of a protein as an aid for design, amino acid residues located on the surface of many proteins have been randomized using the core structure of the protein as scaffold. Examples for this strategy are described or reviewed in the following references incorporated herein by reference: Nygren PA, Uhlen M., Curr Opin Struct Biol. (1997) 7:463-9; Binz HK, Amstutz P, Kohl A, Stumpp MT, Briand C, Forrer P, Grutter MG, Pluckthun A. Nat Biotechnol. (2004) 22:575-82; Vogt M, Skerra A. Chembiochem. (2004) 5:191-9; US 6,562,617.<br>
The basic principle of this technique is based on the observation that many proteins have a stable core, formed by specific arrangements of secondary structure elements such as beta sheets or alpha helices, which are interconnected by structures such as loops, turns, or random coils. Typically, these latter three structure elements are less crucial for the overall structure of the protein, and amino acid residues in these structure elements can be exchanged often without destroying the general fold of the protein. A naturally occurring example for this design prin-<br>
ciple are the CDRs of antibodies. Artificial examples include lipocalins, ankyrins and other protein scaffolds.<br>
The loops which are not CDR-loops in a native immunoglobulin do not have antigen binding or epitope binding specificity but contribute to the correct folding of the entire immunoglobulin molecule and/or its effector or other functions and are therefore called structural loops for the purpose of this invention.<br>
In United States Patent 6,294,654 it is shown that altered antibodies can be made in which a peptide antigen can be incorporated into a non-CDR loop of an antibody (Ab)in the CHI region between the hinge region and the variable region,- and the resulting Ab can be taken up in an APC so that the peptide antigen is presented on the surface of the APC in the context of MHC II, and thereby produce an immune response. These inserted peptides are epitopes and the overall structure of the carrier molecule is not important. It was demonstrated that a ras peptide can be placed on a (non-CDR) loop of an Immunoglobulin and the Immunoglobulin still be secreted. There is stringent "quality control" in the cells which prevent the Immunoglobulin from being secreted unless it is properly folded, and altering the amino acid sequence of the loop might cause the protein to fold into a structure which the cell would detect as incorrect/ and hence degrade it. Thus, besides the examples shown it was considered to be difficult to further modify the structural loops without changing the nature of the Immunoglobulin.<br>
US Pat Appl 2004/0101905 describes binding molecules comprising a target binding site and a Fc effector peptide. The Fc effector peptide is a peptide which interacts with effector molecule. The insertion of an effector peptide into a non-CDR loop of a CHI-domain of an immunoglobulin fragment has been shown.<br>
Fc effector peptides are structures which are naturally occurring in non-CDR loops of antibodies and are therefore expected not to disturb the structure of the immunoglobulin if grafted to onto different equivalent locations in an immunoglobulin.<br>
Nevertheless every peptide grafted into a non-CDR loop according to this disclosure has a high chance of being inactive by the different structural environment it has been selected.<br>
It is stated in both prior art documents mentioned above that it is difficult to insert peptides into the loop that should retain its structure and function, as it is critical not<br>
to disturb the immunoglobulin folding structure as this is important for function and secretion.<br>
US Patent Applications 2004/0132101 and 2005/0244403 describe mutant immunoglobulins with altered binding affinity to an effector ligand, which are natural ligands for structural loops of antibodies. In this document a number of mutations in various regions across the entire immunoglobulin molecule are described which influence the effector function of the entire antibody.<br>
Other prior art documents show that the immunoglobulin like scaffold has been employed so far for the purpose of manipulating the existing antigen binding site, thereby introducing novel binding properties. So far however, only the CDR regions have been engineered for antigen binding, in other words, in the case of the immunoglobulin fold, only the natural antigen binding site has been modified in order to change its binding affinity or specificity. A vast body of literature exists which describes different formats of such manipulated immunoglobulins, frequently expressed in the form of single-chain Fv fragments (scFv) or Fab fragments, either displayed on the surface'of phage particles or solubly expressed in various prokaryotic or eukaryotic expression systems. Among the leading authors in the field are Greg Winter, Andreas Pluckthun and Hennie Hoogenboom.<br>
It is an object of the present invention to provide immunoglobulins with new antigen binding sites introduced, and methods for engineering and manufacturing said immunoglobulins.<br>
Therefore, the present invention relates to a method for engineering an immunoglobulin comprising at least one modification in a structural loop region of said immunoglobulin and determining the binding of said immunoglobulin to an epitope of an antigen, wherein the unmodified immunoglobulin does not significantly bind to said epitope, comprising the steps of:<br>
-	providing a nucleic acid encoding an immunoglobulin com<br>
prising at least one structural loop region,<br>
-	modifying at least one nucleotide residue of at least one<br>
of said structural loop regions,<br>
-	transferring said modified nucleic acid in an expression<br>
system,<br>
-	expressing said modified immunoglobulin,<br>
-	contacting the expressed modified immunoglobulin with an<br>
epitope, and<br>
-	determining whether said modified iromunoglobulin binds to<br>
said epitope.<br>
In particular, the present invention relates to a method for engineering an immunoglobulin binding specifically to an epitope of an antigen selected from the group consisting of allergens, tumor associated antigens, self antigens, enzymes; bacterial antigens, fungal antigens, protozooal antigen and viral antigens. Through the modification in the structural loop region the immunoglobulin may be engineered bind to the epitope. In a preferred embodiment the immunoglobulin is binding specifically to at least two such epitopes, that differ from each other, either of the same antigen or of different antigens.<br>
For example, the method according to the invention refers to engineering an immunoglobulin binding specifically to at least one first epitope and comprising at least one modification in at least one structural loop region of said immunoglobulin and determining the specific binding of said at least one loop region to at least one second epitope, the epitope being selected from the group of antigens as mentioned above, wherein the unmodified structural loop region (non-CDR region) does not specifically bind to said at least one second epitope, comprising the steps of:<br>
-	providing a nucleic acid encoding an immunoglobulin bind<br>
ing specifically to at least one first epitope comprising at<br>
least one structural loop region,<br>
-	modifying at least one nucleotide residue of at least one<br>
of said loop regions encoded by said nucleic acid,<br>
-	transferring said modified nucleic acid in an expression<br>
system,<br>
-	expressing said modified immunoglobulin,<br>
-	contacting the expressed modified immunoglobulin with said<br>
at least one second epitope, and<br>
-	determining whether said modified immunoglobulin binds<br>
specifically to the second epitope.<br>
The method according to the invention preferably refers to at least one modification in at least one structural loop region of said immunoglobulin and determining the specific binding of said at least one loop region to at least one antigen selected from the group consisting of allergens, tumor associated anti-<br>
gens, self antigens, enzymes, bacterial antigens, fungal antigens, viral antigens and protozooal antigens, wherein the immunoglobulin containing an unmodified structural loop region does not specifically bind to said at least one antigen.<br>
The term "immunoglobulins" according to the present invention to be modified {as used herein the terms immunoglobulin and antibody are interchangeable) may exhibit mono- or multi-specific, or multivalent binding properties, at least two, preferably at least three specific binding sites for epitopes of e.g. antigens, effector molecules/proteins. Immunoglobulins according to the invention are also functional fragments accepted in the art, such as Fc, Fab, scFv, single chain dimers of CH/CL domains, Fv, or other derivatives or combinations of the immunoglobulins, domains of the heavy and light chains of the variable region (such as Fd, VI, Vk, Vh) and the constant region of an intact antibody such as CH1, CH2, CH3, CH4, Cl and Ck, as well as mini-domains consisting of two beta-strands of an immunoglobulin domain connected by a structural loop.<br>
It is understood that the term wimmunoglobulin", "modified immunoglobulin" or "immunoglobulin according to the invention" includes a derivative of immunoglobulins as well. A derivative is any combination of one or more immunoglobulins of the invention and or a fusion protein in which any domain or minidomain of the immunoglobulin of the invention may be fused at any position of one or more other proteins (such as other immunoglobulins, ligands, scaffold proteins, enzymes toxins and the like). A derivative of the immunoglobulin of the invention may also be obtained by binding to other substances by various chemical techniques such as covalent coupling, electrostatic interaction, disulphide bonding etc.<br>
The other substances bound to the immunoglobulins may be lipids, carbohydrates, nucleic acids, organic and anorganic molecules or any combination therof (e.g. PEG, prodrugs or drugs). A derivative is also an immunoglobulin with the same amino acid sequence but made completely or partly from non-natural or chemically modified amino acids.<br>
The engineered molecules according to the present invention will be useful as stand-alone proteins as well as fusion proteins or derivatives, most typically fused in such a way as to be part of larger antibody structures or complete antibody mo-<br>
 <br>
lecules, or parts thereof such as Fab fragments, Fc fragments, Fv fragments and others. It will be possible to use the engineered proteins to produce molecules which are monospecific, bis-pecific, trispecific, and maybe even carry more specificities at the same time, and it will be possible at the same time to control and preselect the valency of binding at the same time according to the requirements of the planned use of such molecules .<br>
According to the present invention, binding regions to antigens or antigen binding sites of all kinds of allergens, tumor associated antigens, self antigens, enzymes, bacterial antigens, fungal antigens, protozooal antigen and viral antigens, may be introduced into a structural loop of a given antibody structure.<br>
The term "antigen" according to the present invention shall mean molecules or structures known to interact or capable of interacting with the CDR-loop region of immunoglobulins. Structural loop regions of the prior art do not interact with antigens but rather contribute to the overall structure and/or to the binding to effector molecules.<br>
The term "allergens, tumor associated antigens, self antigens, enzymes, bacterial antigens, fungal antigens, protozooal antigen and viral antigens" according to the present invention shall include all allergens and antigens capable of being recognised by an antibody structure, and fragments of such molecules {especially substructures generally referred to as "epitopes" (e.g. B-cell epitopes)), as long as they are immunologically relevant, i.e. are also recognisable by natural or monoclonal antibodies.<br>
The term "epitope" according to the present invention shall mean a molecular structure which may completely make up a specific binding partner or be part of a specific binding partner to the binding domain or the immunoglobulin of the present invention.<br>
Chemically, an epitope may either be composes of a carbohydrate, a peptide, a fatty acid, a anorganic substance or derivatives thereof and any combinations thereof. If an epitope is a polypeptide, it will usually include at least 3 araino acids, preferably 8 to 50 amino acids, and more preferably between about 10-20 araino acids in the peptide. There is no critical upper limit to the length of the peptide, which could comprise<br>
<br>
nearly the full length of the polypeptide sequence. Epitopes can be either linear or conformational epitopes. A linear epitope is comprised of a single segment of a primary sequence of a polypeptide chain. Linear epitopes can be contiguous or overlapping. Conformational epitopes are comprised of amino acids brought together by folding of the polypeptide to form a tertiary structure and the araino acids are not necessarily adjacent to one another in the linear sequence.<br>
Specifically, epitopes are at least part of diagnostically relevant molecules, i.e. the absence or presence of an epitope in a sample is qualitatively or quantitatively correlated to either a disease or to the health status or to a process status in manufacturing or to environmental and food status. Epitopes may also be at least part of therapeutically relevant molecules, i.e. molecules which can be targeted by the specific binding domain which changes the course of the disease.<br>
Preferred "allergens, tumor associated antigens, self antigens, enzymes, bacterial antigens, fungal antigens, protozooal antigen and viral antigens," are those allergens or antigens, which have already been proven to be or are capable of being im-munologically or therapeutically relevant, especially those, for which a clinical efficacy has been tested.<br>
On the other hand, according to another aspect of the present invention also other binding capacities may be introduced in the structural loop regions, e.g. binding capacities for small molecules, such as drugs or enzymes, catalytic sites of enzymes or enzyme substrates or for a transition state analog of an enzyme substrate.<br>
Preferably the new antigen binding site in the structural loops is foreign to the unmodified immunoglobulin. Thus targets like effector molecules or Fc-receptors are preferably excluded from the binding molecules and the specificity of the immuno-globulins according to the invention.<br>
Preferably, the new antigen binding sites in the structural loops are introduced by substitution, deletion and/or insertion of the immunoglobulin encoded by the selected nucleic acid.<br>
According to another preferred embodiment of the present invention the modification of at least one nucleotide results in a substitution, deletion and/or insertion of the immunoglobulin encoded by said nucleic acid.<br>
present invention is to be understood in the following manner: immunoglobulins are made of domains with a so called immuno-globulin fold. In essence, anti-parallel beta sheets are connected by loops to form a compressed antiparallel beta barrel. In the variable region, some of the loops of the domains contribute essentially to the specificity of the antibody, i.e. the binding to an antigen. These loops are called CDR-loops. All other loops of antibody domains are rather contributing to the structure of the molecule and/or the effector function. These loops are defined herein as structural loops or non-CDR-loops.<br>
The nucleic acid molecules encoding the modified immunoglobulins (and always included throughout the whole specification below: immunoglobulin fragments) may be cloned into host cells, expressed and assayed for their binding specificities. These practices are carried out using well-known procedures, and a variety of methods that may find use in the present invention are described in Molecular Cloning—A Laboratory Manual, S.sup.rd Ed. (Maniatis, Cold Spring Harbor Laboratory Press, New York, 2001), and Current Protocols in Molecular Biology (John Wiley &amp; Sons) . The nucleic acids that encode the modified imraun-oglobulins of the present invention may be incorporated into an expression vector in order to express said immunoglobulins. Expression vectors typically comprise an immunoglobulin operably linked, that is placed in a functional relationship, with control or regulatory sequences, selectable markers, any fusion partners, and/or additional elements. The modified immunoglobulins of the present invention may be produced by culturing a host cell transformed with nucleic acid, preferably an expression vector, containing nucleic acid encoding the modified immunoglobulins, under the appropriate conditions to induce or cause expression of the modified immunoglobulins. The methods of introducing exogenous nucleic acid molecules into a host are well known in the art, and will vary with the host used. Of course, also acellular or cell free expression systems for the expression of modified immunoglobulins may be employed.<br>
In a preferred embodiment of the present invention, the modified immunoglobulins are purified or isolated after expression. Modified immunoglobulins may be isolated or purified in a variety of ways known to those skilled In the art. Standard purification methods include chromatographic techniques, electrophoret-<br>
The modification of the at least one loop region may result in a substitution, deletion and/or insertion of 1 or more amino acids, preferably a point mutation, exchange of whole loops, more preferred the change of at least 2, 3, 4, 5, 6, 7, 8, 9, 10 up to 30 amino acids.<br>
Also preferred is the site directed random mutation. With this method one or more specific amino acid residues of the loop are exchanged or introduced using randomly generated inserts into such structural loops. Alternatively preferred is the use of combinatorial approaches.<br>
The at least one loop region is preferably mutated or modified by random, semi-random or, in particular, by site-directed random mutagenesis methods. These methods may be used to make amino acid modifications at desired positions of the immuno-globulin of the present invention. In these cases positions are chosen randomly, or amino acid changes are made using simplistic rules. For example all residues may be mutated to alanine, referred to as alanine scanning. Such methods may be coupled with more sophisticated engineering approaches that employ selection methods to screen higher levels of sequence diversity. A preferred method according to the invention refers to the randomly modified nucleic acid molecule which comprises at least one nucleotide repeating unit having the sequence 5'-NNS-3', 5'-NNN-3'or 5'- NNK-3'.<br>
The randomly modified nucleic acid molecule may comprise the above identified repeating units, which code for all known naturally occurring amino acids.<br>
As is well known in the art, there are a variety of selection technologies that may be used for the identification and isolation of proteins with certain binding characteristics and affinities, including, for example, display technologies such as phage display, ribosome display, cell surface display, and the like/ as described below. Methods for production and screening of antibody variants are well known in the art. General methods for antibody molecular biology, expression, purification, and screening are described in Antibody Engineering, edited by Duebel &amp; Konterraann, Springer-Verlag, Heidelberg, 2001; and Hay-hurst &amp; Georgiou, 2001, Curr Opin Chem Biol 5:683-689; Maynard &amp; Georgiou, 2000, Annu Rev Biomed Eng 2:339-76.<br>
A "structural loop" or "non-CDR-loop" according to the<br>
icr immune-logical, precipitation, dialysis, filtration, concentration, and chromatofocusing techniques. Purification can often be enabled by a particular fusion partner. For example, anti-bodys may be purified using glutathione resin if a GST fusion is employed, Ni+2 affinity chromatography if a His-tag is employed or immobilized anti-flag antibody if a flag-tag is; used. For general guidance in suitable purification techniques, see Antibody Purification: Principles and Practice, 3.sup;rd Ed., Scopes, Springer-Verlag, NY, 1994. Of course, it is also possible to express the modified immunoglobulins according to the present invention on the surface of a host, in particular on the surface of a bacterial, insect or yeast cell or on the surface of phages or viruses.<br>
Modified immunoglobulins may be screened using a variety of methods, including but not limited to those that use in vitro assays, in vivo and cell-based assays, and selection technologies. Automation and high-throughput screening technologies may be utilized in the screening procedures. Screening may employ the use of a fusion partner or label, for example an enzyme, an immune label, isotopic label, or small molecule label such as a fluorescent or colorimetric dye or a luminogenic molecule.<br>
In a preferred embodiment, the functional and/or biophysical properties of the immunoglobulins are screened in an in vitro assay. In a preferred embodiment, the antibody is screened for functionality, for example its ability to catalyze a reaction or its binding affinity to its target.<br>
Assays may employ a variety of detection methods including but not limited to chroraogenic, fluorescent, luminescent, or isotopic labels.<br>
As is known in the art, a subset of screening methods are those that select for favorable members of a library. The methods are herein referred to as "selection methods'1, and these methods find use in the present invention for screening modified immunoglobulins. When immunoglobulins libraries are screened using a selection method, only those members of a library that are favorable, that is which meet some selection criteria, are propagated, isolated, and/or observed. As will be appreciated, because only the most fit variants are observed, such methods<br>
\_<br>
enable the screening of libraries that are larger than those screenable by methods that assay the fitness of library members<br>
individually. Selection is enabled by any method, technique, or fusion partner that links, covalently or noncovalently, the phenotype of immunoglobulins with its genotype, that is the function of a antibody with the nucleic acid that encodes it. For example the use of phage display as a selection method is enabled by the fusion of library members to the gene III protein. In this way, selection or isolation of modified immunoglobulins that meet some criteria, for example binding affinity to the immunoglobulin's target, also selects for or isolates the nucleic acid that encodes it. Once isolated, the gene or genes encoding modified iromunoglobulins may then be amplified. This process of isolation and amplification, referred to as panning, may be repeated, allowing favorable antibody variants in the library to be enriched. Nucleic acid sequencing of the attached nucleic acid ultimately allows for gene identification.<br>
A variety of selection methods are known in the art that may find use in the present invention for screening immunoglobulin libraries. These include but are not limited to phage display (Phage display of peptides and antibodies: a laboratory manual, Kay et al., 1996, Academic Press, San Diego, Calif., 1996; Low-man et al., 1991, Biochemistry 30:10832-10838; Smith, 1985, Science 228:1315-1317) and its derivatives such as selective phage infection (Malmborg et al., 1997, J Mol Biol 273:544-551), selectively infective phage (Krebber et al., 1997, J Mol Biol 268:619-630), and delayed infectivity panning (Benhar et al.,<br>
2000,	J Mol Biol 301:893-904), cell surface display (Witrrup,<br>
2001,	Curr Opin Biotechnol, 12:395-399) such as display on bac<br>
teria (Georgiou et al., 1997, Nat Biotechnol 15:29-34; Georgiou<br>
et al., 1993, Trends Biotechnol 11:6-10; Lee et al., 2000, Nat<br>
Biotechnol 18:645-648; Jun et al., 1998, Nat Biotechnol 16:576-<br>
80), yeast (Boder &amp; Wittrup, 2000, Methods Enzymol 328:430-44;<br>
Boder &amp; Wittrup, 1997, Nat Biotechnol 15:553-557), and mammalian<br>
cells (Whitehorn et al., 1995, Bio/technology 13:1215-1219), as<br>
well as in vitro display technologies (Amstutz et al., 2001,<br>
Curr Opin Biotechnol 12:400-405) such as polysome display<br>
(Mattheakis et al., 1994, Proc Natl Acad Sci USA 91:9022-9026),<br>
ribosome display (Hanes et al., 1997, Proc Natl Acad Sci USA<br>
94:4937-4942), mRNA display (Roberts &amp; Szostak, 1997, Proc Natl<br>
Acad Sci USA 94:12297-12302; Nemoto et al., 1997, FEES Lett<br>
414:405-408), and ribosome-inactivation display system (Zhou et<br>
al., 2002, J Am Chem Soc 124, 538-543).<br>
Other selection methods that may find use in the present invention include methods that do not rely on display, such as in vivo methods including but not limited to periplasmic expression and cytometric screening (Chen et al., 2001, Nat Biotechnol 19:537-542), the antibody fragment complementation assay (Johns-son &amp; Varshavsky, 1994, Proc Natl Acad Sci USA 91:10340-10344; Pelletier et al., 1998, Proc Natl Acad Sci USA 95:12141-12146), and the yeast two hybrid screen (Fields &amp; Song, 1989, Nature 340:245-246) used in selection mode (Visintin et al., 1999, Proc Natl Acad Sci USA 96:11723-11728) . In an alternate embodiment, selection is enabled by a fusion partner that binds to a specific sequence on the expression vector, thus linking covalently or noncovalently the fusion partner and associated Fc variant library member with the nucleic acid that encodes them. For example, PCT WO 00/22906; PCT WO 01/49058; PCT WO 02/04852; PCT WO 02/04853; PCT WO 02/08023; PCT WO 01/28702; and PCT WO 02/07466 describe such a fusion partner and technique that may find use in the present invention. In an alternative embodiment, in vivo selection can occur if expression of the antibody imparts some growth, reproduction, or survival advantage to the cell.<br>
A subset of selection methods referred to as "directed evolution" methods are those that include the mating or breeding of favourable sequences during selection, sometimes with the incorporation of new mutations. As will be appreciated by those skilled in the art, directed evolution methods can facilitate identification of the most favourable sequences in a library, and can increase the diversity of sequences that are screened. A variety of directed evolution methods are known in the art that may find use in the present invention for screening antibody variants, including but not limited to DNA shuffling (PCT WO 00/42561 A3; PCT WO 01/70947 A3), exon shuffling (U.S. Pat. No. 6,365,377; Kolkman &amp; Stemmer, 2001, Nat Biotechnol 19:423-428), family shuffling (Crameri et al., 1998, Nature 391:288-291; U.S. Pat. No. 6,376,246), RACHITT.TM. (Coco et al., 2001, Nat Biotechnol 19:354-359; PCT WO 02/06469), STEP and random priming of in vitro recombination (Zhao et al., 1998, Nat Biotechnol 16:258-261; Shao et al., 1998, Nucleic Acids Res 26:681-683), exonuclease mediated gene assembly (U.S. Pat. No. 6,352,842; U.S. Pat. No. 6,361,974), Gene Site Saturation Mutagenesis.TM.<br>
(U.S. Pat. No. 6,358,709), Gene Reassembly.TM. (U.S. Pat. No. 6,358,709), SCRATCHY (Lutz et al., 2001, Proc Natl Acad Sci USA 98:11248-11253), DNA fragmentation methods (Kikuchi et al., Gene 236:159-167), single-stranded DNA shuffling (Kikuchi et al., 2000, Gene 243:133-137), and AMEsystem.TM. directed evolution antibody engineering technology (Applied Molecular Evolution) (U.S. Pat. No. 5,824,514; U.S. Pat. No. 5,817,483; U.S. Pat. No. 5,814,476; U.S. Pat. No. 5,763,192; U.S. Pat. No. 5,723,323).<br>
In a preferred embodiment, antibody variants are screened using one or more cell-based or in vivo assays. For such assays, purified or unpurified modified immunoglobulins are typically added exogenously such that cells are exposed to individual immunoglobulins or pools of immunoglobulins belonging to a library. These assays are typically, but not always, based on the function of the immunoglobulin; that is, the ability of the antibody to bind to its target and mediate some biochemical event, for example effector function, ligand/receptor binding inhibition, apoptosis, and the like. Such assays often involve monitoring the response of cells to the antibody, for example cell survival, cell death, change in cellular morphology, or tran-scriptional activation such as cellular expression of a natural gene or reporter gene. For example, such assays may measure the ability of antibody variants to elicit ADCC, ADCP, or CDC. For some assays additional cells or components, that is in addition to the target cells, may need to be added, for example example serum complement, or effector cells such as peripheral blood monocytes (PBMCs), NK cells, macrophages, and the like. Such additional cells may be from any organism, preferably humans, mice, rat, rabbit, and monkey. Immunoglobulins may cause apoptosis of certain cell lines expressing the target, or they may mediate attack on target cells by immune cells which have been added to the assay. Methods for monitoring cell death or viability are known in the art, and include the use of dyes, immuno-chemical, cytochemical, and radioactive reagents. For example, caspase staining assays may enable apoptosis to be measured, and uptake or release of radioactive substrates or fluorescent dyes such as alamar blue may enable cell growth or activation to be monitored. In a preferred embodiment, the DELFIA.RTM. EuTDA-based cytotoxicity assay (Perkin Elmer, MA) may be used. Alternatively, dead or damaged target cells may be monitored by meas-<br>
uring the release of one or more natural intracellular components, for example lactate dehydrogenase. Transcriptional activation may also serve as a method for assaying function in cell-based assays. In this case, response may be monitored by assaying for natural genes or immunoglobulins which may be upregu-lated, for example the release of certain interleukins may be measured, or alternatively readout may be via a reporter construct. Cell-based assays may also involve the measure of morphological changes of cells as a response to the presence of modified imraunoglobulins. Cell types for such assays may be prokaryotic or eukaryotic, and a variety of cell-lines that are known in the art may be employed. Alternatively, cell-based screens are performed using cells that have been transformed or transfected with nucleic acids encoding the variants. That is, antibody variants are not added exogenously to the cells. For example, in one embodiment, the cell-based screen utilizes cell surface display. A fusion partner can be employed that enables display of modified immunoglobulins on the surface of cells (Witrrup, 2001, Curr Opin Biotechnol, 12:395-399).<br>
In a preferred embodiment, the immunogenicity of the modified immunoglobulins may be determined experimentally using one or more cell-based assays. In a preferred embodiment, ex vivo T-cell activation assays are used to experimentally quantitate im-raunogenicity. In this method, antigen presenting cells and naive T cells from matched donors are challenged with a peptide or whole antibody of interest one or more times. Then, T cell activation can be detected using a number of methods, for example by monitoring production of cytokines or measuring uptake of tritiated thymidine. In the most preferred embodiment, interfer-on gamma production is monitored using Elispot assays (Schraittel et. al., 2000, J. Immunol. Meth., 24: 17-24).<br>
The biological properties of the modified immunoglobulins of the present invention may be characterized in cell, tissue, and whole organism experiments. As is known in the art, drugs are often tested in animals, including but not limited to mice, rats, rabbits, dogs, cats, pigs, and monkeys, in order to measure a drug's efficacy for treatment against a disease or disease model, or to measure a drug's pharmacokinetics, toxicity, and other properties. The animals may be referred to as disease models. Therapeutics are often tested in mice, including but not<br>
limited to nude mice, SCID mice, xenograft mice, and transgenic mice (including knockins and knockouts). Such experimentation may provide meaningful data for determination of the potential of the antibody to be used as a therapeutic. Any organism, preferably mammals, may be used for testing. For example because of their genetic similarity to humans, monkeys can be suitable therapeutic models, and thus may be used to test the efficacy, toxicity, pharmacokinetics, or other property of the modified immunoglobulins of the present invention. Tests of the in humans are ultimately required for approval as drugs, and thus of course these experiments are contemplated. Thus the modified immunoglobulins of the present invention may be tested in humans to determine their therapeutic efficacy, toxicity, imraunogeni-city, pharmacokinetics, and/or other clinical properties.<br>
The modified immunoglobulins of the present invention may find use in a wide range of antibody products. In one embodiment the antibody variant of the present invention is used for therapy or prophylaxis, for preparative or analytic use, as a diagnostic, an industrial compound or a research reagent, preferably a therapeutic. The antibody variant may find use in an antibody composition that is monoclonal or polyclonal. In a preferred embodiment, the modified immunoglobulins of the present invention are used to kill target cells that bear the target antigen, for example cancer cells. In an alternate embodiment, the modified immunoglobulins of the present invention are used to block, antagonize, or agonize the target antigen, for example by antagonizing a cytokine or cytoklne receptor. In an alternately preferred embodiment, the modified immunoglobulins of the present invention are used to block, antagonize, or agonize the target antigen and kill the target cells that bear the target antigen.<br>
In an alternately preferred embodiment, the modified immunoglobulins of the present invention are used to block, antagonize, or agonize growth factors or growth factor receptors and kill the target cells that bear or need the target antigen. In an alternately preferred embodiment, the modified immunoglobulins of the present invention are used to block, antagonize, or agonize enzymes and substrate of enzymes.<br>
The modified immunoglobulins of the present invention may be used for various therapeutic purposes. In a preferred embodi-<br>
ment, an antibody comprising the modified immunoglobulins is administered to a patient to treat a specific disorder. A "patient" for the purposes of the present invention includes both humans and other animals, preferably mammals and most preferably humans. By "specific disorder" herein is meant a disorder that may be ameliorated by the administration of a pharmaceutical composition comprising a modified immunoglobulin of the present invention.<br>
In one embodiment/ a modified immunoglobulin according to the present invention is the only therapeutically active agent administered to a patient. Alternatively, the modified immuao-globulin according the present invention are administered in combination with one or more other therapeutic agents, including but not limited to cytotoxic agents, chemotherapeutic agents, cytokines, growth inhibitory agents, anti-hormonal agents, kinase inhibitors, anti-angiogenic agents, cardioprotectants, or other therapeutic agents. The modified immunoglobulins may be administered concomitantly with one or more other therapeutic regimens. For example, an antibody variant of the present invention may be administered to the patient along with chemotherapy, radiation therapy, or both chemotherapy and radiation therapy. In one embodiment, the modified immunoglobulins of the present invention may be administered in conjunction with one or more antibodies, which may or may not comprise a antibody variant of the present invention. In accordance with another embodiment of the invention, the modified immunoglobulins of the present invention and one or more other anti-cancer therapies are employed to treat cancer cells ex vivo. It is contemplated that such ex vivo treatment may be useful in bone marrow transplantation-and particularly, autologous bone marrow transplantation. It is of course contemplated that the antibodies of the invention can be employed in combination with still other therapeutic techniques such as surgery.<br>
A variety of other therapeutic agents may find use for administration with the modified imraunoglobulins of the present invention. In one embodiment, the modified immunoglobulin is administered with an anti-angiogenic agent, which is a compound that blocks,,or interferes to some degree, the development of blood vessels. The anti-angiogenic factor may, for instance, be a small molecule or a protein, for example an antibody, Fc fu- <br>
sion, or cytokine, that binds to a growth factor or growth factor receptor involved in promoting angiogenesis. The preferred anti-angiogenic factor herein is an antibody that binds to Vascular Endothelial Growth Factor (VEGF). In an alternate embodiment, the modified immunoglobulin is administered with a therapeutic agent that induces or enhances adaptive immune response, for example an antibody that targets CTLA-4. In an alternate embodiment, the modified immunoglobulin is administered with a tyrosine kinase inhibitor, which is a molecule that inhibits to some extent tyrosine kinase activity of a tyrosine kinase. In an alternate embodiment, the modified immunoglobulins of the present invention are administered with a cytokine. By "cytokine" as used herein is meant a generic term for proteins released by one cell population that act on another cell as intercellular mediators including chemokines.<br>
Pharmaceutical compositions are contemplated wherein modified immunoglobulins of the present invention and one or more therapeutically active agents are formulated. Formulations of the antibody variants of the present invention are prepared for storage by mixing said immunoglobulin having the desired degree of purity with optional pharmaceutically acceptable carriers, excipients or stabilizers (Remington's Pharmaceutical Sciences 16th edition, Osol, A. Ed., 1980), in the form of lyophilized formulations or aqueous solutions. The formulations to be used for in vivo administration are preferably sterile. This is readily accomplished by filtration through sterile filtration membranes or other methods. The modified immunoglobulins and other therapeutically active agents disclosed herein may also be formulated as immunoliposoraes, and/or entrapped in microcapsules<br>
Administration of the pharmaceutical composition comprising a modified immunoglobulin of the present invention, preferably in the form of a sterile aqueous solution, may be done in a variety of ways, including, but not limited to, orally, subcu-taneously, intravenously, intranasally, intraotically, trans-dermally, topically (e.g., gels, salves, lotions, creams, etc.), intraperitoneally, intramuscularly, intrapulmonary (e.g., AERx™ inhalable technology commercially available from Aradigm, or In-hance™ pulmonary delivery system commercially available from Inhale Therapeutics), vaginally, parenterally, rectally, or in-traocularly.<br>
As used herein, the term "specifically binds" refers to a binding reaction which is determinative of the cognate ligand of interest in a heterogeneous population of molecules. Thus, under designated conditions (e.g. immunoassay conditions in the case of an immunoglobulin), the specified antibody binds to its particular "target" and does not bind in a significant amount to other molecules present in a sample. Comparable to CDRs of antibodies the modified structural loop regions are antigen- or molecule-binding protein moieties and not antigens as such.<br>
The term "expression system" refers to nucleic acid molecules containing a desired coding sequence and- control sequences in operable linkage, so that hosts transformed or trans-fected with these sequences are capable of producing the encoded proteins. In order to effect transformation, the expression system may be included on a vector; however, the relevant DNA may than also be integrated into the host chromosome.<br>
According to a preferred embodiment of the present invention the expression system comprises a vector. Any expression vector known in the art may be used for this purpose as appropriate.<br>
The modified immunoglobulin is preferably expressed in a host, preferably in a bacterial, a yeast, a plant cell, in an animal cell or in a plant or animal.<br>
A wide variety of appropriate host cells may be used to express the modified immunoglobulin, including but not limited to mammalian cells (animal cells), plant cells, bacteria (e.g. Bacillus subtilis, Escherichia coli), insect cells, and yeast (e.g. Pichia pastoris, Saccharomyces cerevisiae). For example, a variety of cell lines that may find use in the present invention are described in the ATCC cell line catalog, available from the American Type Culture Collection. Furthermore, also plants and animals may be used as hosts for the expression of the immunoglobulin according to the present invention. The expression as well as the transfection vectors or cassettes may be selected according to the host used.<br>
Of course also acellular or cell free protein expression systems may be used. In vitro transcription/translation protein expression platforms, that produce sufficient amounts of protein offer many advantages of a cell-free protein expression, eliminating the need for laborious up- and down-stream steps (e.g. host cell transformation, culturing, or lysis) typically associ-<br>
ated with cell-based expression systems.<br>
Another aspect of the present invention relates to a method for manufacturing an immunoglobulin or a pharmaceutical preparation thereof comprising at least one modification in a structural loop region of said immunoglobulin and determining the binding of said immunoglobulin to an epitope of an antigen, wherein the unmodified immunoglobulin does not significantly bind to said epitope, comprising the steps of:<br>
-	providing a nucleic acid encoding an immunoglobulin com<br>
prising at least one loop region,<br>
-	modifying at least one nucleotide residue of at least one<br>
of said loop regions,<br>
-	transferring said modified nucleic acid in an expression<br>
system,<br>
-	expressing said modified immunoglobulin,<br>
-	contacting the expressed modified immunoglobulin with an<br>
epitope,<br>
-	determining whether said modified immunoglobulin binds to<br>
said epitope, and<br>
-	providing the modified immunoglobulin binding to said epi<br>
tope and optionally finishing it to a pharmaceutical prepara<br>
tion.<br>
In particular the present invention relates to a method for manufacturing a multi-specific immunoglobulin binding specifically to at least one first molecule or a pharmaceutical preparation thereof comprising at least one modification in at least one structural loop region of said immunoglobulin and determining the specific binding of said at least one loop region to at least one second molecule selected from the group consisting of allergens, tumor associated antigens, self antigens, enzymes, bacterial antigens, fungal antigens, protozooal antigens and viral antigens, wherein the immunoglobulin containing an unmodified structural loop region does not specifically bind to said at least one second molecule, comprising the steps of:<br>
-	providing a nucleic acid encoding an immunoglobulin bind<br>
ing specifically to at least one first molecule comprising at<br>
least one structural loop region,<br>
-	modifying at least one nucleotide residue of at least one<br>
of said loop regions encoded by said nucleic acid,<br>
-	transferring said modified nucleic acid in an expression<br>
system,<br>
-	expressing said modified immunoglobulin,<br>
-	contacting the expressed modified immunoglobulin with said<br>
at least one second molecule, and<br>
-	determining whether said modified immunoglobulin binds<br>
specifically to the second molecule and	<br>
-	providing the modified immunoglobulin binding specifically<br>
to said at least one second molecule and optionally finishing it<br>
to a pharmaceutical preparation.<br>
The engineering of more than one specificity into a member of a specific binding pair is preferred (Kufer et al. (2004) Trends in Biotechnology vol. 22 pages 238-244).<br>
Numerous attempts have been made to produce multi-specific, e.g. bispecific, monoclonal antibodies or antibody fragments. One problem in the production of bispecific antibodies made of two different polypeptide chains (heavy and light chain) is the necessity to express four different chains (two heavy and two light chains) in one cell resulting in a number of various combinations of molecules which have to be separated from the desired bispecific molecule in the mixture. Due to their similarity the separation of these molecules is difficult and expensive. A number of techniques have been employed to minimize the occurrence of such unwanted pairings (Carter (2001) Journal of Immunological Methods, vol 248, pages 7-15)<br>
One solution to the problem is the production of one polypeptide chain with two specificities, like e.g. two scFvs linked to each other or the production of so-called diabodies. Such-molecules have been shown to be far away from the fold of a natural molecule and are notoriously difficult to produce (LeGall—et al. (2004) Protein Engineering, Design &amp; Selection vol 17 pages 357-366).<br>
Another problem of the current design of bispecific antibodies is the fact that even if the parent antibodies are bi-valently binding to their respective binding partner (e.g. IgG), the resulting bispecific antibody is monovalent for each of the respective binding partner.<br>
The preferred multi-specific molecules of the present invention solve these problems:<br>
Expression of a bispecific molecule as one pqlypeptide chain is possible (a modified Ig domain with two binding specificit-<br>
ies, see example section), which is easier to accomplish than the expression of two antibody polypeptide chains (Cabilly et al. Proc. Natl. Acad. Sci. USA 81:3273-3277 (1984)).<br>
It can also be produced as an antibody like molecule (i.e. made of 2 polypeptide chains), due to the fact that the second specificity is located in the non-variable part of the molecule there is no need for two different heavy chains or different light chains. Thus, there is no possibility of wrong pairing of the two chains.<br>
An antibody of the present invention may consist of a heavy chain and a light chain, which form together a variable region binding to a specific binding partner the second specificity may be formed by a modified loop of any of the structural loops of either the heavy chain or the light chain. The binding site may also be formed by more than one non-CDR loop which may be structurally neighboured (either on the heavy chain or on the light chain or on both chains).<br>
The modified antibody or derivative may be a complete antibody or an antibody fragment (e.g. Fab, CH1-CH2, CH2-CH3).<br>
It may bind mono- or multi-valently to binding partners or even with different valency for the different binding partners, depending on the design.<br>
As there are a number of various loops available for selection and design of a specific binding site in the non-CDR regions of heavy and light chains it is possible to design antibody derivatives with even more than two specificities without the problems mentioned above.<br>
The specific binding domains within one polypeptide chain may be connected with or without a peptide linker.<br>
Some antibody classes can be regarded as multi-specific, in particular bispecific, by nature: They bind to an antigen (which is typically e.g. either a foreign structure or a cancer associated structure) with the variable region and bind to Fc-effector molecules with the Fc part (e.g. Fc receptors on various immune cells or complement protein) thus enabling effects such as ADCC, ADCP or CDC.<br>
The Fc-effector molecules are bound by the Fc-part of an im-munoglobulin molecule (for IgGl it consists of domains CH2 and CH3) and a number of methods have been described to optimize effector function by improvement of binding of the Fc-part of an<br>
antibody molecule either by glycoengineering techniques<br>
(US 6,602,684) or by protein engineering either directly at the<br>
Fc (US 2005/0054832) or indirectly by engineering outside the Fc<br>
(US 2005/02444403). Both, binding of the Fc region to Fc recept<br>
or and/or binding to complement proteins such Cql has been<br>
altered by such techniques. Usually the binding affinity to "such<br>
Fc-effector molecules is seeked to improve as this correlates<br>
with improved effector functions.	.:<br>
With the current invention it is possible to design antibody binding to Fc-effector molecules outside the natural Fc binding region. Modified loops in antibody domains other than the loops involved in "natural" Fc-effector molecule binding can be selected from a library or designed to bind to one ore more Fc-efc-fector molecule. An antibody with such additional Fc-effector molecule binding sites would either have stronger avidity to a certain Fc-effector molecule or effector-cell displaying an Fc-effector molecule and therefore may have an even stronger effect than glycoengineered antibodies or otherwise improved Fc regions. However, for certain embodiments of the present invention, the effector characteristics of a given antibody to be modified should not directly be changed but remain unaffected by the modification in the structural loop according to the present invention.<br>
Antibody fragments have certain advantages as compared to whole antibodies. Fragments have usually good biodistribution properties and can more easily be produced. However, most of the antibody fragment designs lack effector functions and have short in vivo half life (Holliger P, et al. Nat Biotechnol.(2005) 23:1126-36.).<br>
Neither CHI nor CK or CX domains mediate effector functions which is the reason why Fabs do not show ADCC, ADCP or CDC. The WO 02/44215 describes binding molecules which consists of the antigen binding site of an antibody and a peptide binding Fc-effector molecules. In such a way an antibody fragment displaying effector functions can be constructed. The peptide is being incorporated into the binding molecule at a position that does neither destroy the antigen binding nor the ability of the peptide to bind to an Fc-effector molecule.<br>
According to the present invention however, the binding to Fc-effector molecules may be performed with modified immuno-<br>
globulin domains which have been selected for Fc-effector molecule binding from libraries of random loop sequences within a fixed scaffold of an immunoglobulin domain. Therefore, it is possible to select for specific loop sequences which would not bind to Fc-effector molecules outside the Ig-domain scaffold. The polypeptides resulting from the present invention may therefore preferably consist of more than 100 amino acids.<br>
In order to select for potential effector function of such domains according to the present invention, libraries of mutant CHI, CK or CX domains may be selected for binding to Fc-receptors and/or complement factors such as Clq.<br>
In order to increase in vivo half life of a molecule consisting of or containing such a domain (e.g. CHI, CH2, CH3, CH4, CK or CX), binding to.FcRn may be selected for with libraries of mutant e.g. CH1-, CH2-, CH3-, CH4-, CK- or CX-domains according to the present invention.<br>
FcRn-receptors for selection may be provided either on the surface of cells expressing naturally the respective receptors or by expression and purification of the extracellular part of the respective receptor. For the purpose of this invention a first screening on FcRn may select for mutant domains which can further be tested in vitro and even further characterized in FACS experiments by binding to cells expressing FcRn receptor. It can be further characterized by affinity ranking of binding to various recombinant FcRn, isoforms and allotypes e.g with surface plasmon resonance techniques.<br>
According to a preferred embodiment of the present invention the immunoglobulin is of human or murine origin.<br>
Since the modified immunoglobulin may be employed for various purposes, in particular in pharmaceutical compositions, the immunoglobulin is preferably of human or murine origin. Of course, the modified immunoglobulin may also be a humanized or chimeric immunoglobulin.<br>
According to another preferred embodiment of the present invention the human immunoglobulin is selected from the group consisting of IgAl, IgA2, IgD, IgE, IgGl, IgG2, IgG3, IgG4 and IgM.<br>
The murine immunoglobulin is preferably selected from the group consisting of IgA, IgD, IgE, IgGl, IgG2A, IgG2B, IgG2C, IgG3 and IgM.<br>
The modified immunoglobulin may be derived from one of the<br>
above identified immunoglobulin classes.<br>
The immunoglobulin comprises preferably a heavy and/or light chain of the immuncglobulin or a part thereof.<br>
The modified imraunoglobulin may comprise a heavy and/or light chain, at least one variable and/or constant domain.<br>
The iiranunoglobulin according to the present invention comprises preferably at least one constant and/or at least one variable domain of the immunoglobulin or a part thereof including a minidomain.<br>
A constant domain is an immunoglobulin fold unit of the constant part of an immunoglobulin molecule, also refered to as a domain of the constant region (e.g. CH1, CH2, CH3, CH4, Ck, Cl).<br>
A variable domain is an immunoglobulin fold- unit of the variable part of an immunoglobulin, also refered to as a domain of the variable region (e.g. Vh, Vk, VI, Vd)<br>
A preferred immunoglobulin according to the invention consists of a constant domain selected from the group consisting of CHI, CH2, CHS, CH4, Igk-C, Igl-C, or a part thereof including a minidomain, with at least one loop region, and is characterised in that said at least one loop region comprises at least one amino acid modification forming at least one modified loop region, wherein said at least one modified loop region binds specifically to at least one epitope of an antigen.<br>
Another preferred immunoglobulin according to the invention consists of a variable domain of a heavy or light chain, or a part thereof including a minidomain, with at least one loop region, and is characterised in that said at least one loop region comprises at least one amino acid modification forming at least one modified loop region, wherein said at least one modified loop region binds specifically to at least one epitope of an antigen.<br>
According to a preferred embodiment the constant domain is<br>
selected from the group of CH1, CH2, CH3, CH4, Igk-C, Igl-C and<br>
combinations thereof.	<br>
The modified immunoglobulin according to the present invention may comprise one or more constant domains (e.g. at least two, three, four, five, six, ten domains). If more than one domain is present in the modified immunoglobulin these domains may be of the same type or of varying types (e.g. CH1-CH1-CH2, CH3-CH3). Of course also the order of the single domains may be of<br>
any kind (e.g. CH1-CH3-CH2, CH4-CH1-CH3-CH2).<br>
All numbering of the amino acid sequences of the immuno-globulins is according to the IMGT numbering scheme (IMGT, the international ImMunoGeneTics information <a href="../cdn-cgi/l/email-protection.html" class="__cf_email__" data-cfemail="3c4f454f4859517c55515b48125f5552594f125a4e">[email&#160;protected]</a>; http://imgt.cines.fr; Lefranc et al., 1999, Nucleic Acids Res. 27: 209-212; Ruiz et al., 2000 Nucleic Acids Res. 28: 219-221; Lefranc et "al., 2001, Nucleic Acids Res. 29: 207-209; Lefranc et al., 2003, Nucleic Acids Res. 31: 307-310; Lefranc et al., 2005, Dev Comp Immunol 29:185-203).<br>
According to another preferred embodiment of the present invention the modified loop regions of CH1, CH2, CH3 and CH4 comprise amino acids 7 to 21, amino acids 25 to 39, amino acids 41 to 81, amino acids 83 to 85, amino acids 89 to 103 and amino acids 106 to 117.<br>
The loop regions of Igk-C and Igl-C of human origin comprise preferably amino acids 8 to 18, amino acids 27 to 35, amino acids 42 to 78, amino acids 83 to 85, amino acids 92 to 100, amino acids 108 to 117 and amino acids 123 to 126.<br>
The loop regions of Igk-C and Igl-C of murine origin comprise preferably amino acids 8 to 20, amino acids 26 to 36, amino acids 43 to 79, amino acids 83 to 85, amino acids 90 to 101, amino acids 108 to 116 and amino acids 122 to 125.<br>
The structural loop regions of the variable domain of the immunoglobulin of human origin comprise preferably amino acids 8 to 20, amino acids 44 to 50, amino acids 67 to 76 and amino acids 89 to 101.<br>
According to a preferred embodiment of the present invention the structural loop regions of the variable domain of the immunoglobulin of murine origin comprise amino acids 6 to 20, amino acids 44 to 52, amino acids 67 to 76 and amino acids 92 to 101.<br>
The above identified amino acid regions of the respective immunoglobulins comprise loop regions to be modified.<br>
The immunoglobulin according to the invention is preferably of camel origin.<br>
Camel antibodies comprise only one heavy chain and have the same antigen affinity as normal antibodies consisting of light and heavy chains. Consequently camel antibodies are much smaller than, e.g., human antibodies, which allows them to penetrate dense tissues to reach the antigen, where larger proteins cannot. Moreover, the comparative simplicity, high affinity and<br>
specificity and the potential to reach and interact with active sites, camel's heavy chain antibodies present advantages over common antibodies in the design, production and application of clinically valuable compounds.<br>
The immunoglobulin of camel origin comprises preferably at least one constant domain selected from the group consisting of CHI, CH2 and CH3.<br>
According to a preferred embodiment of the present invention the loop regions of CHI, CH2 and CHS of the camel immunoglobulin comprise amino acids 8 to 20, amino acids 24 to 39, amino acids 42 to 78, amino acids 82 to 85, amino acids 91 to 103 and amino acids 108 to 117.<br>
According to a preferred embodiment of the present invention the specific binding of the modified immunoglobulin to the molecule is determined by a binding assay selected from the group consisting of iramunological assays, preferably enzyme linked im-munosorbent assays (ELISA), surface plasmon resonance assays, saturation transfer difference nuclear magnetic resonance spec-troscopy, transfer NOE (trNOE) nuclear magnetic resonance spec-troscopy, competitive assays, tissue binding assays, live cell binding assays and cellular extract assays.<br>
Binding assays can be carried out using a variety of"methods<br>
known in the art, including but not limited to FRET (Fluores<br>
cence Resonance Energy Transfer) and BRET (Bioluminescence Res<br>
onance Energy Transfer)-based assays, AlphaScreen.TM. (Amplified<br>
Luminescent Proximity Homogeneous Assay), Scintillation Proxim<br>
ity Assay, ELISA (Enzyme-Linked Immunosorbent Assay), SPR (Sur<br>
face Plasmon Resonance, also known as BIACORE.RTM.), isothermal<br>
titration calorimetry, differential scanning calorimetry, gel"<br>
electrophoresis, and chromatography including gel filtration.<br>
These and other methods may take advantage of some fusion part<br>
ner or label.	--<br>
The modified immunoglobulin is preferably conjugated to a label selected from the group consisting of organic molecules, enzyme labels, radioactive labels, colored labels, fluorescent labels, chromogenic labels, luminescent labels, haptens, digoxi-genin, biotin, metal complexes, metals, colloidal gold and mixtures thereof.<br>
The modified immunoglobulin may be conjugated to other molecules which allow the simple detection of said conjugate in,<br>
 <br>
for instance, binding assays (e.g. ELISA) and binding studies.<br>
Another aspect of the present invention relates to a immuno-globulin consisting of a constant domain selected from the group consisting of CH1, CH2, CH3, CH4, Igk-C, Igl-C, or a part thereof including minidomains, or combinations thereof, with at least one loop region, characterised in that said at least one loop region comprises at least one amino acid modification forming at least one modified loop region, wherein said at least one modified loop region binds specifically to at least one epitope of an antigen<br>
It is preferred to combine molecularly at least one modified antibody domain (= binding to the specific partner via the non-variable sequences or structural loop) with at least one other binding molecule which can be an antibody, antibody fragment, a soluble receptor, a ligand or another modified antibody domain.<br>
The molecule is selected from the group consisting of pro-teinaceous molecules, nucleic acids, and carbohydrates.<br>
The loop regions of the modified immunoglobulins may specifically bind to any kind of binding molecules, in particular to proteinaceous molecules, proteins, peptides, polypeptides, nucleic acids, glycans, carbohydrates, lipids, small organic molecules, anorganic molecules. Of course, the modified immunoglobulins may comprise at least two loop regions whereby each of the loop regions may specifically bind to other molecules or epitopes.<br>
According to a preferred embodiment of the present invention the molecule binding to the modified structural loop region is selected from the group consisting of tumor associated antigens, in particular EpCAM, tumor-associated glycoprotein-72 (TAG-72), tumor-associated antigen CA 125, Prostate specific membrane antigen (PSMA), High molecular weight melanoma-associated antigen (HMW-MAA), tumor-associated antigen expressing Lewis Y related carbohydrate, Carcinoembryonic antigen (CEA), CEACAM5, HMFG PEM, mucin MUC1, MUC18 and cytokeratin tumor-associated antigen, bacterial antigens, viral antigens, allergens, fluorescein, lyso-zyrae, toll-like receptor 9, erythropoietin, CD2, CD3, CD3E, CD4, CD11, CDlla, CD14, CD18, CD19, CD20, CD22, CD23, CD25, CD28, CD29, CD30, CD33 (p67 protein), CD38, CD40, CD40L, CD52, CD54, CD56, CD80, CD147, GD3, IL-1, IL-1R, IL-2, IL-2R, IL-4, IL-5, IL-6, IL-6R, IL-8, IL-12, IL-15, IL-18, IL-23, interferon alpha,<br>
interferon beta, interferon gamma; TNF-alpha, TNFbeta2, TNF.alpha., TNFalphabeta, TNF-Rl, TNF-RII, FasL, CD27L, CD30L, 4-1BBL, TRAIL, RANKL, TWEAK, APRIL, BAFF, LIGHT, VEG1, OX40L, TRAIL Re-ceptor-1, Al Adenosine Receptor, Lymphotoxin Beta Receptor, TACI, BAFF-R, EPO; LFA-3, ICAM-1, ICAM-3, integrin betal, integ-rin beta2, integrin alpha4/beta7, integrin alpha2, integrin al-pha3, integrin alpha4, integrin alphaS, integrin alpha6, integrin alphav, alphaVbeta3 integrin, FGFR-3, Keratinocyte Growth Factor, VLA-1, VLA-4, L-selectin, anti-Id, E-selectin, HLA, HLA-DR, CTLA-4, T cell receptor, B7-1, B7-2, VNRintegrin, TGFbetal, TGFbeta2, eotaxinl, BLyS (B-lymphocyte Stimulator.), complement C5, IgE, factor VII, CD64, CBL, NCA 90, EGFR (ErbB-1), Her2/neu (ErbB-2), Her3 (ErbB-3), Her4 (ErbB4), Tissue Factor, VEGF, VEG-FR, endothelin receptor, VLA-4, carbohydrates such as blood group antigens and related carbohydrates, Galili-Glycosylation, Gastrin, Gastrin receptors, tumor associated carbohydrates, Hapten NP-cap or NIP-cap, T cell receptor alpha/beta, E-selectin, digoxin, placental alkaline phosphatase (PLAP) and testicular PLAP-like alkaline phosphatase, transferrin receptor, Heparanase I, human cardiac myosin, Glycoprotein Ilb/IIIa (GPIIb/IIIa), human cytomegalovirus (HCMV) gH envelope glycopro-tein, HIV gp!20, HCMV, respiratory syncital virus RSV F, RSVF Fgp, VNRintegrin, Hep B gp!20, CMV, gpllbllla, HIV IIIB gp!20 V3 loop, respiratory syncytial virus (RSV) Fgp, Herpes simplex virus (HSV) gD glycoprotein, HSV gB glycoprotein, HCMV gB envelope glycoprotein, Clostridium perfringens toxin and fragments thereof.<br>
The modified immunoglobulin according to the present invention may preferably bind to one of the molecules disclosed above. These molecules comprise also allergens.<br>
According to another preferred embodiment of the present invention the araino acid residues of positions 15 to 17, 29 to 34, 85.4 to 85.3, 92 to 94, 97 to 98 and/or 108 to 110 of CH3 are modified.<br>
The modification of the immunoglobulin according to the present invention is preferably a deletion, substitution or an insertion.<br>
According to the present invention at least 1, preferably at least 2, 3, 4, 5, 6, 7, 8, 9, 10 and 15, amino acids are deleted, substituted with other amino acids (also with modified<br>
amino acids) or inserted into the loop region of the immuno-globulin. However, the maximum number of amino acids inserted into a loop region of an immunoglobulin may not exceed the number of 30, preferably 25, more preferably 20, amino acids. The substitution and the insertion of the amino acids occurs preferably randomly by methods known in the art and as disclosed in the present patent application.<br>
The immunoglobulin according to the invention is according to a specific embodiment characterised in that the CH3 region comprises SEQ ID No. 16 or SEQ ID No. 18, when EpCam binds to said imraunoglobulin, SEQ ID No. 20, when fluorescein binds to said immunoglobulin, SEQ ID No. 22, 24, 26, 28, 30 or 32, when lysozyme binds to said immunoglobulin, SEQ ID No. 34, 36, 38 or 40, when TLR9 binds to said immunoglobulin, and SEQ ID No. 42, when lysozyme and/or erythropoietin bind to said immunoglobulin.<br>
According to a specific embodiment of the invention the immunoglobulin is characterised in that it comprises SEQ ID No. 44 or SEQ ID No. 46, when lysozyme and gp41 bind to said immunoglobulin.<br>
The modified immunoglobulin is preferably conjugated to a label or reporter molecule selected from the group consisting of organic molecules, enzyme labels, radioactive labels, colored labels, fluorescent labels, chromogenic labels, luminescent labels, haptens, digoxigenin, biotin, metal complexes, metals, colloidal gold and mixtures thereof.<br>
Modified immunoglobulins conjugated to labels as specified above may be used, for instance, in diagnostic methods.<br>
Another aspect of the present invention relates to the use of an immunoglobulin according to the present invention or obtainable by a method according to the present invention for the preparation of a vaccine for active immunization. Hereby the immunoglobulin is either used as antigenic drug substance to formulate a vaccine or used for fishing or capturing antigenic structures for use in a vaccine formulation.<br>
Another aspect of the present invention relates to the use of an immunoglobulin according to the present invention or obtainable by a method according to the present invention for the preparation of a protein library of immunoglobulins.<br>
Yet another aspect of the present invention relates to a method for specifically binding and/or detecting a molecule com-<br>
prising the steps of:<br>
(a)	contacting a modified immunoglobulin according to the<br>
present invention or a modified immunoglobulin obtainable by a<br>
method according to the present invention with a test sample<br>
suspected to contain said molecule, and<br>
(b)	detecting the potential formation of a specific immuno<br>
globulin/molecule complex.<br>
Another aspect of the present invention relates to a method for specifically isolating a molecule comprising the steps of:<br>
(a)	contacting a modified immunoglobulin according to the<br>
present invention or a modified immunoglobulin obtainable by a<br>
method according to the present invention with a sample contain<br>
ing said molecule,<br>
(b)	separating the specific immunoglobulin/molecule complex<br>
formed, and<br>
(c)	optionally isolating the molecule from said complex. .<br>
The immunoglobulins according to the present invention may<br>
be used to isolate specifically molecules from a sample. If multi-specific immunoglobulins are used more than one molecules may be isolated from a sample. It is especially advantageous using modified immunoglobulins in such methods because it allows, e.g., to generate a matrix having a homogeneous surface with defined amounts of binding partners (i.e. Modified immunoglobulins) immobilised thereon which able to bind to the molecules to be isolated. In contrast thereto, if mono-specific binding partners are used no homogeneous matrix can be generated because the single binding partners do not bind with the same efficiency to the matrix.<br>
Another, aspect of t:he present invention relates to a method for targeting a compound to a target comprising the steps of:<br>
(a)	contacting a modified immunoglobulin according to the<br>
present invention or a modified immunoglobulin obtainable by a<br>
method according to the present invention capable to specific<br>
ally bind to said compound,<br>
(b)	delivering the immunoglobulin/compound complex to the<br>
target.<br>
Modified immunoglobulins according to the present invention may be used to deliver at least one compound bound to the CDRs and/or modified loop regions to a target. Such immunoglobulins may be used to target therapeutic substances to a preferred site<br>
of action in the course of the treatment of a disease.<br>
Another aspect of the present invention relates to a protein library comprising an immunoglobulin according to the present invention or obtainable by the method according to the present invention.<br>
Preferred methods for constructing said library can be found above and in the examples. The library according to the present invention may be used to identify immunoglobulins binding to a distinct molecule.<br>
In particular the present invention relates to the use of a protein library comprising an immunoglobulin according to the present invention or obtainable by the method according to the present invention for the design of immunoglobulin derivatives. An existing immunoglobulin can be changed to introduce antigen binding sites into any domain or minidomain by using a protein library of the respective domain of at least 10, preferably 100, more preferably 1000, more preferably 10000, more preferably 100000, most preferably more than 1000000 variant domains with at least one modified loop. The library is then screened for binding to the specific antigen. After molecular characterization for the desired properties the selected domain or minidomain is cloned into the original immunoglobulin by genetic engineering techniques so that it replaces the wild type region. Alternatively, only the DNA coding for the loops or coding for the mutated amino acids may be exchanged to obtain an immunoglobulin with the additional binding site for the specific antigen.<br>
The choice of the site for the mutated, antigen-specific structural loop is dependent on the structure of the original immunoglobulin and on the purpose of the additional binding site. If, for example, the original molecule is a complete immunoglobulin which needs to have inserted an additional antigen binding site without disturbance of the effector function, the loops to be modified would be selected from domains distant from CH2 and CHS which are the natural binding partners to Fc-effect-or molecules. If the original immunoglobulin is a Fab, modification of loops in constant domains of the light chains or the heavy chains or the respective variable domains is possible. To generate a library one may prepare libraries of mutant original molecules which have mutations in one ore more structural loops<br>
of one or more domains. The selection with complete mutated original molecules may have some advantages as the selection for antigen binding with a modified structural loop will deliver the sterically advantageous modifications if tested also for the other properties the mutated immunoglobulin should show.<br>
The size requirement (i.e. the number of variant proteins)" of a protein library of a mutated domain or a minidomain or a fusion molecule of a domain is dependent on the task. In general, a library to generate an antigen binding site de novo needs to be larger than a library used to further modify an already existing engineered antigen binding site made of a modified structural loop (e.g. for enhancing affinity or changing fine specificity to the antigen).<br>
The present invention also relates to an immunoglobulin library or a nucleic acid library comprising a plurality of immuno-globulins, e.g. a constant or variable domain, a minidomain and/or at least one structural loop region contained in a minidomain, or nucleic acid molecules encoding the same. The library contains members with different modifications, wherein the plurality is defined by the modifications in the at least one structural loop region. The nucleic acid library preferably includes at least 10 different members (resulting in one amino acid exchange) and more preferably includes at least 100, more preferably 1000 or 10000 different members (e.g. designed by randomisation strategies or combinatory techniques). Even more diversified individual member numbers, such as at least 1000000 or at least 10000000 are also preferred.<br>
A further aspect of the invention is the combination of two different domains or minidomains selected from at least two libraries according to the invention in order to generate multispe-cific immunoglobulins. These selected specific immunoglobulins may be combined with each other and with other molecules, similar to building blocks, to design the optimal arrangement of the domains or minidomains to get the desired properties.<br>
Furthermore, one or more modified immunoglobulins according to te invention may be introduced at various or all the different sites of a protein possible without destruction of the structure of the protein. By such a "domain shuffling" technique new libraries are created which can again be selected for the desired properties.<br>
The preferred library contains immunoglobulins according to the invention, selected from the group consisting of domains of an immunoglobulin, minidomains or derivatives thereof.<br>
A preferred embodiment of the present invention is a binding molecule for an antigen (antigen binding molecule) comprising at least one immunoglobulin domain and a structural loop region being modified according to the present invention to bind to the antigen, wherein said binding molecule does not comprise variable domains of an antibody. It may comprise other parts useable for antibody activities (e.g. such as natural or modified effector regions (sequences); however, it lacks the "natural" binding region of antibodies, i.e. the variable domains in their naturally occurring position. These antigen binding molecules according to the present invention have the advantages described above for the present molecules, yet without the specific binding activity of antibodies; however with a newly introduced specific binding activity in the structural loop region.<br>
Preferably, these antigen binding molecules according to the present invention comprise CHI, CH2, CH3, CH4, Igk-C, Igl-C and combinations thereof; said combinations comprising at least two, preferably at least four, especially at least six constant domains and at least one structural loop region modified according to the present invention. Preferably these structural loop regions are either connected via structural loop region modified according to the present invention or the structural loops being naturally present between such two constant domains. An embodiment of these antigen binding molecules according to the present invention consists of the Fc region of an antibody with at least one modification in a structural loop according to the present invention. Also for the antigen binding molecules according to the present invention it is preferred that the new antigen binding sites in the structural loops are introduced by randomising technologies, i.e. by exchanging one or more amino acid residues of the loop by randomisation techniques or by introducing randomly generated inserts into such structural loops. Alternatively preferred is the use of combinatorial approaches.<br>
According to another aspect, the present invention relates to a modified immunoglobulin having an antigen binding site foreign to the unmodified immunoglobulin and incorporated in one or more structural loops. The term "foreign" means that the antigen<br>
binding site is not naturally formed by the specific region of the immunoglobulin, and a foreign binding partner, but not the natural binding partner of an immunoglobulin, is bound by the antigen binding site. This means that a binding partner, such as a Fc-receptor or an effector of the immune system, is not considered to be bound by the antigen binding site foreign to the unmodified immunoglobulin.<br>
Preferably, the antigen is selected from the group consisting of pathogen antigen, tumour associated antigen, enzyme, substrate, self antigen, organic molecule or allergen. More preferred antigens are selected from the group consisting of viral antigens, bacterial antigens or antigens from pathogens of euka-ryots or phages. Preferred viral antigens include HAV-, HBV-, HCV-, HIV I-, HIV II-, Parvovirus-, Influenza-, HSV-, Hepatitis Viruses, Flaviviruses, Westnile Virus, Ebola Virus, Pox-Virus, Smallpox Virus, Measles Virus, Herpes Virus, Adenovirus, Papil-loma Virus, Polyoma Virus, Parvovirus, Rhinovirus, Coxsackie virus, Polio Virus, Echovirus, Japanese Encephalitis virus, Dengue Virus, Tick Borne Encephalitis Virus, Yellow Fever Virus, Coronavirus, respiratory syncytial virus, parainfluenza virus, La Crosse Virus,Lassa Virus,Rabies Viruse, Rotavirus antigens; preferred bacterial antigens include Pseudomonas-, Mycobacteri-um-, Staphylococcus-, Salmonella-, Meningococcal-, Borellia-, Listeria, Neisseria-, Clostridium-, Escherichia-, Legionella-, Bacillus-, Lactobacillus-, Streptococcus-, Enterococcus-, Corynebacterium-, Nocardia-, Rhodococcus-, Moraxella-, Brucella, Campylobacter-, Cardiobacterium-, Francisella-, Helicobacter-, Haemophilus-, Klebsiella-, Shigella-, Yersinia-, Vibrio-, Chlamydia-, Leptospira-, Rickettsia-, Mycobacterium-, Treponema-, Bartonella- antigens. Preferred eukaryotic antigens of pathogenic eukaryotes include antigens from Giardia, Toxoplasma, Cyc-lospora, Cryptosporidium, Trichinella, Yeasts, Candida, Asper-gillus, Cryptococcus,Blastomyces, Histoplasma, Coccidioides.<br>
Preferred immunoglobulins according to the present invention comprise at least two antigen binding sites, the first site binding to a first epitope, and the second site binding to a second epitope.<br>
According to a preferred embodiment, the present immunoglobulin comprises at least two loop regions, the first loop region binding to a first epitope, and the second loop region<br>
binding to a second epitope. Either the at least first or at least second loop region or both may be contain a structural loop. The immunoglobulins according to the present inventions include the fragments thereof known in the art to be functional which contain the essential elements according to the present invention: the structural loop region modified according to the present invention.<br>
Preferably, the immunoglobulin according to the present invention is composed of at least two imraunoglobulin domains, or a part thereof including a minidomain, and each domain contains at least one antigen binding site.<br>
Also preferred is an immunoglobulin according to the invention, which comprises at least one domain of the constant region and/or at least one domain of the variable region of the immunoglobulin, or a part thereof including a minidomain. Thus, a variable domain, which is for example modified in the C-terminal region, or the variable domain linked to a modified CHI region, for instance a modified CHI minidomain, is one of the preferred embodiments.<br>
The preferred immunoglobulin according to the invention comprises a domain that has at least 50% homology with the unmodified domain.<br>
The term "homology" indicates that polypeptides have the same or conserved residues at a corresponding position in their primary, secondary or tertiary structure. The term also extends to two or more nucleotide sequences encoding the homologous polypeptides.<br>
"Homologous immunoglobulin domain" means an immunoglobulin domain according to the invention having at least about 50% araino acid sequence identity with regard to a full-length native sequence immunoglobulin domain sequence or any other fragment of a full-length immunoglobulin domain sequence as disclosed herein. Preferably, a homologous inununoglobulin domain will have at least about 50% amino acid sequence identity, preferably at least about 55% amino acid sequence identity, more preferably at least about 60% amino acid sequence identity, more preferably at least about 65% amino acid sequence identity, more preferably at least about 70% amino acid sequence identity, more preferably at least about 75% amino acid sequence identity, more preferably at least about 80% amino acid sequence identity, more preferably at<br>
least about 85% amino acid sequence identity, more preferably at least about 90% amino acid sequence identity, more preferably at least about 95% amino acid sequence identity to a native immuno-globulin domain sequence, or any other specifically defined fragment of a full-length immunoglobulin domain sequence as disclosed herein.<br>
"Percent (%) amino acid sequence identity" with respect to the immunoglobulin domain sequences identified herein is defined as the percentage of amino acid residues in a candidate sequence that are identical with the amino acid residues in the specific imraunoglobulin domain sequence, after aligning the sequence and introducing gaps, if necessary, to achieve the maximum percent sequence identity, and not considering any conservative substitutions as part of the sequence identity. Alignment for purposes of determining percent amino acid sequence identity can be achieved in various ways that are within the skill in the art, for instance, using publicly available computer software such as BLAST, BLAST-2, ALIGN or Megalign (DNASTAR) software. Those skilled in the art can determine appropriate parameters for measuring alignment, including any algorithms needed to achieve maximal alignment over the full length of the sequences being compared.<br>
% amino acid sequence identity values may be obtained as described below by using the WU-BLAST-2 computer program (Altschul et al., Methods in Enzymology 266:460-480 (1996)). Most of the WU-BLAST-2 search parameters are set to the default values. Those not set to default values, i.e., the adjustable parameters, are set with the following values: overlap span=l, overlap fraction=0.125, word threshold (T)=ll, and scoring raatrix=BLOSUM62. When WU-BLAST-2 is employed, a % amino acid sequence identity value is determined by dividing (a) the number of matching identical amino acid residues between the amino acid sequence of the immunoglobulin domain of interest having a se-<br>
*-<br>
quence derived from the native immunoglobulin domain and the comparison amino acid sequence of interest (i.e., the sequence against which the immunoglobulin domain of interest is being compared which may be the unmodified immunoglobulin domain) as determined by WU-BLAST-2 by (b) the total number of amino acid residues of the non-randomized parts of the immunoglobulin do-<br>
main of interest. For example, in the statement "a polypeptide comprising an amino acid sequence A which has or having at least 80% amino acid sequence identity to the amino acid sequence B", the amino acid sequence A is the comparison amino acid sequence of interest and the amino acid sequence B is the amino acid sequence of the immunoglobulin domain of interest.<br>
In a preferred embodiment the immunoglobulin according to the invention is a bispecific antibody or a bispecific single chain antibody. Further preferred is that the immunoglobulin comprises a bispecific domain or a part thereof including a min-idomain.<br>
The iramunoglobulin according to the present invention may be used for any purpose known in the art for iraraunoglobulins but also enables applications which are depending on the combination of specificities introduced by the present invention. Accordingly, the immunoglobulins according to the present inventions are preferably used for therapeutic and prophylactic use (e.g. as an active or passive immunotherapy); for preparative and analytic use and for diagnostic use.<br>
Another aspect of the present invention relates to a kit of binding partners containing<br>
(a)	a modified immunoglobulin having an antigen binding site<br>
foreign to the immunoglobulin incorporated in one or more struc<br>
tural loops, and<br>
(b)	a binding molecule containing an epitope of said antigen.<br>
Such a binding molecule of this kit according to the present<br>
invention may be used for identifying the binding specificity of the modified imraunoglobulin according to the present invention. By using the binding molecule of this kit according to the present invention, the potency of the modified immunoglobulins according to the present invention may be determined.<br>
Potency as defined here is the binding property of the modified molecule to its antigen. The binding can be determined quantitatively and/or qualitatively in terms of specificity and/or affinity and/or avidity as used for quality control purposes.<br>
Moreover, the binding molecule of a kit according to the present invention may be used for selecting the modified immunoglobulin according to the present invention from a library consisting of at least 10, preferably at least 100, more preferably<br>
at least 1000, more preferred at least 10000, especially at least 100000 immunoglobulins with different modifications in the structural loops.<br>
In accordance with the present invention, one of the key features of the present invention is that the engineering of the immunoglobulin domains takes place in regions which are not normally involved in antigen binding, in other words, in regions other than the CDRs of an antibody. It was observed that the specific fold of immunoglobulin domains allows the introduction of random mutations in regions which are structurally analogous to the CDRs but different in position in sequence. The regions "identified by the present invention are, like CDRs, loop regions connecting the beta strands of the immunoglobulin fold.<br>
More specifically, it is described herein that by introducing random mutations in the loops connecting beta strands A-B and E-F of a human IgGl CHS domain, mutated CHS domains were selected that bind specifically to either Toll like receptor 9-peptide (TLR-9) or to hen egg lysozyme, which are a peptide and a protein respectively that are not normally recognized and bound by human CHS domains of IgGl. The mutations introduced by us include mutations in which selected amino acid residues in the wildtype sequence were replaced by randomly chosen residues, and they also include insertions of extra amino acid residues in the loops mentioned above.<br>
By analogy the immunoglobulin domains from any class of immunoglobulins and from immunoglobulins from any species are amenable to this type of engineering. Furthermore not only the specific loops targeted in the present invention can be manipulated, but any loop connecting beta strands in immunoglobulin domains can be manipulated in the same way.<br>
Engineered inununoglobulin domains from any organism and from any class of immunoglobulin can be used according to the present invention either as such (as single domains), or as part of a larger molecule. For example, they can be part of an intact immunoglobulin, which accordingly would have its "normal" antigen binding region formed by the 6 CDRs and the new, engineered antigen binding region. Like this, a multi-specific, e.g. bispe-cific, immunoglobulin could be generated. The engineered immunoglobulin domains can also be part of any fusion protein. The use of these engineered immunoglobulin domains is in the general<br>
field of the use of inanunoglobulins.<br>
The domains of the following immunoglobulins are understood<br>
as immunoglobulin domains here:<br>
for IgG, IgD and IgA: VL, CL, VH, CH1, CH2, CHS for IgM and IgE : VL, CL, VH, CH1, CH2, CH3, CH4<br>
1.	Single immunoglobulin domains randomized on one side,<br>
i.e. either in loops connecting beta-strands B-C, D-E or F-G<br>
(the "tip", with the exception of variable domains which are<br>
covered by many patents) or beta-strands A-B, C-D, (C-C1 and<br>
C1 '-D in the case of variable domains) or E-F (the "bottom") .<br>
Single loops or any combination of loops can be randomized.<br>
Residues can be changed, deleted, or additional residues can be<br>
inserted.<br>
2.	Single immunoglobulin domains randomized on both sides,<br>
the tip and the bottom.<br>
3.	any protein containing one of the single randomized do<br>
mains, such as:<br>
<br>
a)	"single-chain CH3" dimers (scCH3), scCH2, scCHl/CL,<br>
randomized on one or both sides<br>
b)	single-chain Fv randomized on the "bottom", i.e. on<br>
the side opposite to the CDRs<br>
c)	Fab fragments randomized at the "bottom", i.e. on<br>
the C-terminal end of the CHI and of the CL domain<br>
d)	Fc fragments (i.e. proteins consisting of CH2-CH3)<br>
randomized on one or both sides<br>
e)	complete immunoglobulins randomized on the bottom of<br>
the Fc<br>
f)	other suitable domains<br>
The primary advantages of the single domains: are very similar to all the arguments that are used to promote camel VH molecules ("nanobodies", see www.ablynx.com) . The randomized im-raunoglobulin domains are very small proteins (molecular weight ca. 12-15 kDa, depending on the number of inserted amino acid residues) and therefore will have the following advantages as compared to conventional antibodies or antibody fragments such as scFv and Fabs: recognizing uncommon or hidden epitopes, binding into cavities or active sites of protein targets, ease of<br>
manufacture, and many others. In the case of an immunoglobulin domain that is randomized on both sides, a bivalent or a bispe-cific molecule can be generated.  The main advantages of the single domains as part of fusion proteins is additional binding properties can be engineered on any other protein.<br>
It is contemplated that any expression system can be used to make the proteins. An analogy to the single domains as described here can be found in the antibodies from the camel, which only: has a VH but no VL In these proteins, only 3 CDRs (instead of 6 as in "normal" antibodies are responsible for antigen binding).<br>
The following patent references are incorporated herein by reference as if set forth in their entirety herewith:<br>
US 6,294,654 Modified immunoglobulin molecule incorporating an antigen in a non-CDR loop region<br>
US 5,844,094 Target binding polypeptide  .<br>
US 5,395,750 Methods for producing proteins which bind to predetermined antigens<br>
US 2004/0071690 High avidity polyvalent and polyspecific reagents<br>
US 2004/0018508 Surrogate antibodies and methods of preparation and use thereof<br>
US 2003/0157091 Multi-functional proteins<br>
US 2003/0148372 Method to screen phage display libraries with different ligands<br>
US 2002/0103345 Bispecific immunoglobulin-like antigen binding proteins and method of production<br>
US 2004/0097711 Iramunoglobulin superfamily proteins<br>
US 2004/0082508 Secreted proteins<br>
US 2004/0063924 Secreted proteins<br>
US 2004/0043424 Immunoglobulin superfamily proteins<br>
US 5,892,019 Production of a single-gene-encoded immunoglobulin<br>
US 5,844,094 Target binding polypeptide<br>
The present invention is further illustrated in the following figures and examples without being restricted thereto.<br>
Figure la shows the structure of an intact IgGl. Domains are indicated with arrows.<br>
Figure Ib illustrates the structural organization of the main human immunoglobulin isotype monomers. Disulfide bonds are shown as lines, N-linked carbohydrate groups are shown as<br>
circles.<br>
Figure 2 shows the immunoglobulin fold for a constant (left) and a variable (right) domain of an immunoglobulin. Beta strands are indicated by arrows<br>
Figure 3 shows a molecular model of the engineered CH3 domain according to the present invention, with the randomized part indicated by a solvent accessible surface. The surface is circled.<br>
Figure 4 shows a schematic presentation of the PCRs used for production of the fragments used for assembly of the mutated CH3 domain. PCR primers are indicated by arrows with their respective 5'-3' orientation, and vertical lines indicate the approximate positions of the introduced restriction sites which were used for assembly of the mutated gene.The following restriction sites are contained on the primers for ligations of the PCR fragments: CH3LNCO: Ncol; CH3LSAC and CH3CSAC: Sad; CH3CHIN and CH3RHIN: Hindlll; CH3RNOT: NotI.<br>
Figure 5 shows some examples of how the immunoglobulin domains of the current application could be used. Randomized regions are indicated by a star symbol. Specificities of the randomized regions in one molecule can either be identical or different.<br>
Figure 6 shows a schematic presentation of the design of the bispecific engineered CH3 domain. Names of primers are given in boxes and arrows indicate the direction in which the primers are elongated. Boxes with sloping lines indicate the relative positions of regions that are randomized in this construct, boxes with vertical lines indicate the relative positions of regions that were introduced for generation of clone C24, and restriction sites used for the cloning procedure are given.<br>
Figure 7 shows a schematic presentation of the design of the bispecific engineered CH3 domain. The nucleotide sequence and its translation is shown of the basic design of the bispecific engineered CH3 domain. Red sequences indicate randomized regions in order to generate the bispecific construct, while green boxes indicate regions in which the sequence was randomized in order to generate clone C24.<br>
Figure 8 shows the sequence listing of the sequences disclosed herein.<br>
DESCRIPTION OF SPECIFIC EXAMPLES:<br>
Example 1: Construction of the CHS library and phage surface display<br>
The crystal structure of an IgGl Fc fragment, which is published in the Brookhaven Database as entry lOQO.pdb was used to aid in the design of the mutated CH3 domain.<br>
The sequence which was used as the basis for construction of the CH3 library is given in SEQ ID No. 1. In this sequence, the first amino acid corresponds to Proline 343 of chain A of Brookhaven database entry loqo.pdb. The last residue contained in loqo.pdb is Serine 102 of SEQ ID No. 1. After detailed analysis of the structure of loqo.pdb and by visual inspection of the residues forming the loops which connect the beta strands/ it was decided to randomize residues 17, 18 and 19, which are part of the loop connecting beta strand A-B as well as 71, 72, 73, 76, and 77, which are part of the loop connecting beta strand E-F of SEQ ID No. 1. A molecular model of the engineered CH3 domain, with the randomized part indicated by a solvent accessible surface is shown in Figure 3. The engineered gene was produced by a series of PCR reactions followed by ligation of the resulting PCR products. To facilitate ligation, some of the codons of the nucleotide sequence coding for SEQ ID No. 1 were modified to produce restriction sites without changing the amino acid sequences (silent mutations). For insertion into the cloning vector pHENl (Nucleic Acids Res. 1991 Aug 11;19(15):4133-7. Multi-subunit proteins on the surface of filamentous phage: methodologies for displaying antibody (Fab) heavy and light chains. Hoogenboom HR, Griffiths AD, Johnson KS, Chiswell DJ, Hudson P, Winter G.) in frame with the pelB secretion signal, r extra nucleotide residues encoding Met-Ala were attached at the 5' end of the sequence to create an Ncol restriction site. For the randomized residues, the codon NNS (IUPAC code, where S means C or G) was chosen which encodes all 20 naturally occurring amino acids, but avoids 2 out of 3 stop codons. The engineered sequence is given as a nucleotide sequence in SEQ ID No. 2 and as an amino acid sequence in SEQ ID No. 3. The Letter X in SEQ ID No. 3 denotes randomized amino acid residues. The sequences of the PCR primers used for assembly of the mutated CH3 domain are given in SEQ ID No. 4 through 9. Figure 4 shows a schematic presentation of the PCR fragments generated for assembly of the mutated gene, and the primers used therefor.<br>
cDNA of the heavy chain of the human monoclonal antibody 3D6 (Felgenhauer M, Kohl J, Ruker F. Nucleotide sequences of the cD-NAs encoding the V-regions of H- and L-chains of a human monoclonal antibody specific to HIV-l-gp41. Nucleic Acids Res. 1990 Aug 25;18(16):4927) was used as template for the PCR reactions. The 3 PCR products were digested with SacI and/or HindiII respectively and ligated together. The ligation product was further digested with Ncol and NotI and ligated into the surface display phagemid vector pHENl, which had previously been digested with Ncol and NotI. A number of selected clones were controlled by restriction analysis and by DNA sequencing and were found to contain the insert as planned, including the correctly inserted randomized sequences. For the following steps of phage preparation, standard protocols were followed. Briefly, the ligation mixture was transformed into E. coli TGI cells by electro-poration. Subsequently, phage particles were rescued from E. coli TGI cells with helper phage M13-KO7. Phage particles were then precipitated from culture supernatant with PEG/NaCl in 2 steps, dissolved in water and used for selection by panning or, alternatively, they were stored at minus 80°C.<br>
Example 2: Construction of the CH3+3 library<br>
This library was constructed and cloned in the same way as the CHS library. The amino acid sequence of the construct is given in SEQ ID No. 10, the corresponding nucleotide sequence in SEQ ID No. 11, and the primers used for construction were SEQ ID No. 4-7, SEQ ID No. 9 and SEQ ID No. 12.<br>
Example 3: Construction of the CH3+5 library<br>
This library was constructed and cloned in the same way as the CHS library. The amino acid sequence of the construct is given in SEQ ID No. 13, the corresponding nucleotide sequence in SEQ ID No. 14, and the primers used for construction were SEQ ID No. 4-7, SEQ ID No. 9 and SEQ ID No. 15.<br>
Example 4: Panning of the CH3 - phage library on TLR-9 pep-tide<br>
3 panning rounds were performed according to standard protocols. Briefly, the following method was applied. Maxisorp 96-well plates (Nunc) were coated with a synthetic peptide representing part of the sequence of Toll-like Receptor 9 (TLR-9). 200 μ1 of the following solution were added per well: 0.1M Na-car-bonate buffer, pH 9.6, with the following concentrations of dis-<br>
solved peptide:<br>
1st panning round:	1 rag/ml TLR-9 peptide<br>
2nd panning round:	500 pg/ml TLR-9 peptide<br>
3rd panning round:	100 pg/ml TLR-9 peptide<br>
Incubation was for 1 hour at 37°C, followed by blocking with<br>
2% dry milk (M-PBS) with 200 pi per well for 1 hour at room tem<br>
perature,	<br>
The surface display phage library was then allowed to react<br>
with the bound peptide by adding 100 ul phage suspension and 100<br>
μ1 4% dry milk (M-PBS), followed by incubation for 45 minutest<br>
with shaking and for 90 minutes without shaking at room temper<br>
ature .	<br>
Unbound phage particles were washed away as follows. After the 1st panning round: 10 x 300 μ1 T-PBS, 5x 300 μ1 PBS; after the 2nd panning round: 15 x 300 μ1 T-PBS, lOx 300 μ1 PBS; after the 3rd panning round: 20 x 300 μ1 T-PBS, 20 x 300 μ1 PBS.<br>
Elution of bound phage particles was performed by adding 200 μ1 per well of 0.1 M glycine, pH 2.2, and incubation with shaking for 30 minutes at room temperature. Subsequently, the phage suspension was neutralized by addition of 60 μ1 2M Tris-Base, followed by infection into E. coli TGI cells by mixing 10 ml exponentially growing culture with 0.5 ml eluted phage and incubation for 30 minutes at 37°C. Finally, infected bacteria were plated on TYE medium with 1% glucose and 100 pg/ml ampicillin, and incubated at 30°C overnight.<br>
Table 1: Results of the panning of the CH3 - phage library on TLR-9 peptide (Phage titers) <br>
(Table 1Removed)<br>
Example 5: Cloning of selected clones of CH3 mutants selected against TLR-9 for soluble expression<br>
Phagemid DNA from the phage selected through the 3 panning rounds was isolated with a midi-prep. DNA encoding mutated CH3-<br>
regions was batch-amplified by PCR and cloned NcoI-NotI into the vector pNOTBAD/Myc-His, which is the E. coli expression vector pBAD/Myc-His (Invitrogen) with an inserted NotI restriction site to facilitate cloning. Ligated constructs were transformed into E. coli LMG194 cells (Invitrogen) with electroporation, and grown at 30°C on TYE medium with 1% glucose and ampicillin overnight. Selected clones were inoculated into 200 μ1 2xYT medium with ampicillin, grown overnight at 30°C, and induced by adding L-arabinose to an end concentration of 0.1%. After expression at 16°C overnight, the cells were harvested by centri-fugation and treated with 100 μ1 Na-borate buffer,  pH 8.0, at 4°C overnight for preparation of periplasmic extracts. 50 ul of the periplasmic extracts were used in ELISA (see below).<br>
Example 6: ELISA of CHS mutants selected against TLR-9 Selected clones were assayed for specific binding to the TLR-9 peptide by ELISA.<br>
Coating: Microtiter plate (NUNC, Maxisorp), 100 μ1 per well, 20 pg TLR-9 peptide /ml 0.1 M Na-carbonate buffer, pH 9.6, Ih at 37°C<br>
Wash:    3x 200 μ1 PBS<br>
Blocking: 1% BSA-PBS, Ih at RT<br>
Wash:    3x 200 μ1 PBS<br>
Periplasmic extract binding: 50 μ1 periplasmic extract<br>
50 pi 2% BSA-PBS, at room temperature overnight<br>
Wash:    3x 200 pi PBS<br>
1st antibody: aati-His4 (Qiagen), 1:1000 in 1% BSA-PBS, 90 min at RT, 100 μ1 per well Wash:    3x 200 μ1 PBS<br>
2nd antibody: goat anti mouse*HRP (SIGMA), 1:1000 in 1% BSA-PBS, 90 min at RT, 100 μ1 per well Wash:    3x 200 μ1 PBS<br>
Detection : 3mg/ml OPD in Na-citrate/phosphate buffer, pH 4.5, 0.4 μ1 30%H2O2<br>
Stopping: 100 ml 3M H2SO4 Absorbance read: 492/620 nm<br>
Clones that gave a high signal in this first, preliminary<br>
ELISA were cultured in a 20-ml volume at the same conditions as<br>
described above. Their periplasmic extracts were isolated in<br>
1/20 of the culture volume as described above and tested with<br>
ELISA (as described above) for confirmation.	<br>
Table 2: Results of confirmation ELISA  <br>
(Table 2Removed)<br>
Background (antigen alone) (12 parallel readings): 0.0115<br>
Example 7: Panning of the CH3 and of the CH3+5 - phage library on hen egg lysozyme<br>
3 panning rounds were performed. Maxisorp 96-well plates (Nunc) were coated with hen egg lysozyme, by adding 200 pi of the following solution per well:<br>
PBS, with the following concentrations of dissolved hen egg lysozyme:<br>
1st panning round: 2 mg/ml HEL<br>
2nd panning round: Img/ml HEL<br>
3rd panning round: 1 mg/ml HEL<br>
Incubation was for T hour at 37°C, followed by blocking with 2% dry milk (M-PBS) with 200 ul per well for 1 hour at room temperature .<br>
The surface display phage library was then allowed to react with the bound hen egg lysozyme by adding 100 ul phage suspension and 100 ul 4% dry milk (M-PBS), followed by incubation for 45 minutes with shaking and for 90 minutes without shaking at room temperature.<br>
Unbound phage particles were washed away as follows: 1st panning round: 10 x 300 μ1 T-PBS, 5x 300 μ1 PBS 2nd panning round: 15 x 300 μ1 T-PBS, 10x 300 μ1 PBS 3rd panning round: 20 x 300 μ1 T-PBS, 20 x 300 μ1 PBS<br>
Elution of bound phage particles was performed by adding 200 μ1 per well of 0.1 M glycine, pH 2.2, and incubation with shaking for 30 minutes at room temperature. Subsequently, the phage suspension was neutralized by addition of 60 μ1 2M Tris-Base, followed by infection into E. coli TGI cells by mixture of 10 ml exponentially growing culture with 0.5 ml eluted phage and incubation for 30 minutes at 37°C. Finally, infected bacteria were plated on TYE medium with 1% glucose and 100 μg/ml ampicillin, and incubated at 30°C overnight.<br>
Table 3: Results of the panning of phage library CHS on hen egg lysozyme (Phage titers) <br>
(Table 3Removed)<br>
Table 4: Results of the panning of the phage library CH3 +5 on hen egg lysozyme (HEL) (phage titers)<br>
(Table 4Removed)<br>
 Example 8: Cloning of selected clones of example 7 for soluble expression<br>
The cloning of selected clones for soluble expression was performed as described above for the CH3 mutants selected against TLR-9.        <br>
Example 9: Soluble expression of selected clones of example 7<br>
The soluble expression of selected clones was performed as described above for the CH3 mutants selected against TLR-9. Periplasmic extracts were tested in a preliminary ELISA (protocol see example 10)<br>
Clones that gave a high signal in this first, preliminary ELISA were cultured in a 20-ml volume at the same conditions as<br>
described above. Their periplasmic extracts were isolated in 1/20 of the culture volume as described above and tested with ELISA (as described in example 10) for confirmation.<br>
Example 10: ELISA of CH3 mutants selected against hen egg lysozyme<br>
Coating: Microtiter plate (NUNC, Maxisorp), 100 μ1 per well, 100 μg hen egg lysozyme /ml in PBS, Ih at 37°C<br>
Wash:    3x 200 ul PBS	      <br>
Blocking: 1% BSA-PBS, Ih at RT<br>
Wash:    3x 200 μ1 PBS	<br>
Periplasmic extract binding: 50 ul periplasmic extract 50 μ1 2% BSA-PBS, at room temperature overnight Wash:    3x 200 ul PBS<br>
1st antibody: anti-His4 (Qiagen), 1:1000 in 1% BSA-PBS, 90 rain at RT, 100 μ1 per well Wash:    3x 200 ul PBS<br>
2nd antibody: goat anti mouse*HRP (SIGMA), 1:1000 in 1% BSA-PBS, 90 min at RT (room temperature), 100 μ1 per well Wash:    3x 200 μ1 PBS<br>
Detection: 3mg/ml OPD in Na-citrate/phosphate buffer, pH 4.5, 0.4 μ1 30%H202<br>
Stopping: 100 ml 3M H2SO4 Absorbance read: 492/620 nm<br>
Table 5: Results of confirmation ELISA of CH3 mutants selec-ted against hen egg lysozyme <br>
(Table 5Removed)<br>
Background (antigen alone) (12 parallel readings): 0.1763<br>
Table 6: Results of confirmation ELISA with antigen dilutions of CH3 mutants selected against hen egg lysozyme <br>
(Table 6Removed)<br>
nc: no periplasmic extract added<br>
It is noted that hen egg lysozyme reacts with anti-his4 antibody, therefore a relatively high background was observed.<br>
Table 7: Results of confirmation ELISA of CH3+5 mutants selected against hen egg lysozyme <br>
(Table 7Removed)<br>
Background (antigen alone) (12 parallel readings): 0.1334 Note:  hen egg lysozyme reacts with anti-his* antibody, therefore a relatively high background was observed.<br>
Example 11: CL library<br>
Visual inspection of the crystal structure of an Fab fragment (the structure of the Fab of the human monoclonal antibody 3D6 is used: RSCB Protein Data Bank (http://www.rcsb.org/pdb/) Entry 1DFB.PDB (He XM, et al. Proc Natl Acad Sci USA. 1992 Aug<br>
1;89(15):7154-8) and computer-aided analysis (e.g. Protein Explorer is used for this purpose (http://molvis.sdsc.edu/protex-pl/frntdoor.htm)) of the secondary and tertiary structure of this protein) allows to identify residues located in loop regions which connect the beta-strands of the CL-domain scaffold. These residues comprise amino acids 8 to 18, amino acids 27 to 35, amino acids 42 to 78, amino acids 83 to 85, amino acids 92 to 100, amino acids 108 to 117 and amino acids 123 to 126 (numbering according to the IMGT numbering system (Lefranc MP, et al. Nucleic Acids Res. 2005 Jan 1;33 (Database issue) :D593-7; Lefranc MP, et al. Dev Comp Immunol. 2005;29(3) :185-203)J .<br>
More specifically, residues 11, 12, 14-18, and 92-95 are randomized within the human CL domain (SEQ ID No. 48) . Randomization is achieved by PCR amplification of the coding sequences with PCR primers in which the positions of the relevant codons are encoded by the nucleotide sequence 5'-NNS-3', which potentially encodes for all 20 amino acids while avoiding 2 out of 3 stop codons. The library insert is amplified by two separate PCR reactions, and the two PCR fragments are ligated together via a HpyCH4IV restriction site which is introduced as a silent mutation by the PCR primers. The primers further provide the restriction endonuclease sites Ncol and NotI respectively for cloning into the phage display vector pHEN (Hoogenboom HR, et al. Nucleic Acids Res. 1991 Aug 11;19(15):4133-7). The C-termin-al cystein of the CL domain is not included for the phage display, but can be added later on when a modified CL clone is used e.g. for the construction of an Fab fragment.<br>
As a template for PCR amplification, a plasmid such as pRcCMV-3D6LC (Rtiker F, et al. Ann N Y Acad Sci. 1991 Dec 27;646:212-9), which contains as an insert the complete light chain of the human monoclonal antibody, is used.<br>
For the CL+3 (SEQ ID No. 50, 51) and the CL+5 (SEQ ID No. 52, 53) libraries, which contain additional residues inserted between position 92 and 95 of the CL domain, primer CLRHPY3 and CLRHPY5 are used respectively instead of primer CLRHPY.<br>
The nucleotide and amino acid sequence of the final product of the PCRs and ligations, cloned into the Ncol site of pHENl, which leads to the attachment of a pelB leader sequence to the N-terminus of the construct is shown below (SEQ ID No. 48, 49) :<br>
(Seq No 48,49 Removed)<br>
Primer List for CL library:<br>
cllnco: 5'-cttaccatgg ccgtggctgc accatctgtc ttcatcttcc cgc-catctnn snnscagnns nnsnnsnnsn nsgcctctgt tgtgtgc-31 (SEQ ID No. 56)<br>
cllhpy: 5'-tgacaacgtc agggtgctgc tgaggc-31 (SEQ ID No. 57)<br>
clrhpy: 5'-tcagaacgtt gnnsnnsnns nnstacgaga aacacaaagt c-3' {SEQ ID No. 58)<br>
clrhpy3: 5'-tcagaacgtt gnnsnnsnns nnsnnsnnsn nstacgagaa acacaaagtc-3' (SEQ ID No. 59)<br>
clrhpyS: 5'-tcagaacgtt gnnsnnsnns nnsnnsnnsn nsnnsnnsta cgagaaacac aaagtc-3' (SEQ ID No. 60)<br>
clrnot: 5'-catcgcggcc gcctctcccc tgttgaagct c-3' (SEQ ID No. 61)<br>
A number of selected library clones (mutated CL domains cloned in the phagmid vector pHENl) are controlled by restriction analysis and by DNA sequencing to contain the insert as planned, including the correctly inserted randomized sequences. For the following steps of phage preparation, standard protocols are followed. Briefly, the ligation mixture is transformed into E. coli TGI cells by electroporation. Subsequently, phage particles are rescued from E. coli TGI cells with helper phage M13-K07. Phage particles are then precipitated from culture supernatant with PEG/NaCl in 2 steps, dissolved in water and used for selection by panning or, alternatively, they can be stored at minus 80°C.<br>
Example 12: CHI library<br>
Visual inspection of the crystal structure of an Fab fragment (the structure of the Fab of the human monoclonal antibody 3D6 is used: RSCB Protein Data Bank Entry 1DFB.PDB) and computer-aided analysis (Protein Explorer is used for this purpose) of the secondary and tertiary structure of this protein allows to identify residues located in loop regions which connect the beta-strands of the CHI-domain scaffold. These residues comprise amino acids 7 to 21, amino acids 25 to 39, araino acids 41 to 81, amino acids 83 to 85, amino acids 89 to 103 and amino acids 106 to 117 (numbering according to the IMGT numbering system).<br>
More specifically, residues 12-19 and 93-100 are randomized within the human CHI domain (SEQ ID No. 54, 55). Randomization is achieved by PCR amplification of the coding sequences with PCR primers in which the positions of the relevant codons are encoded by the nucleotide sequence 5'-NNS-3', which potentially encodes for all 20 amino acids while avoiding 2 out of 3 stop codons. The library insert is amplified by two separate PCR reactions, and the two PCR fragments are ligated together via a BstEII restriction site which occurs naturally in the CHI domain. The primers further provide the restriction endonuclease<br>
sites Ncol and NotI respectively for cloning into the phage display vector pHEN. The C-terminal cystein of the CHI domain is not included for the phage display, but can be added later on when a modified CHI clone is used e.g. for the construction of an Fab fragment.<br>
As a template for PCR amplification, a plasmid such as pRcCMV-3D6HC, which contains as an insert the complete heavy chain of the human monoclonal antibody, is used.<br>
The nucleotide and amino acid sequence of the final product of the PCRs and ligations, cloned into the Ncol site of pHENl, which leads to the attachment of a pelB leader sequence to the N-terminus of the construct is shown below (SEQ ID No. 54, 55):<br>
(Seq No 54,55 Removed)<br>
Primer List for CHI library<br>
CH1LNCO: 5'-acgtccatgg ccgcctccac caagggccca tcggtcttcc ccctggcacc ctcctccnns nnsnnsnnsn nsnnsnnsnn sgccctgggc tgcctg-gtc-31 (SEQ ID No. 62)<br>
CH1LBST: 5'-ggcacggtca ccacgctgct gag-31 (SEQ ID No. 63)<br>
CH1RBST: 5'-agcgtggtga ccgtgcccnn snnsnnsnns nnsnnsnnsa cctacatctg caacgtgaat c-31 (SEQ ID No. 64)<br>
CH1RNOT: 5'-catagcggcc gcagatttgg gctcaacttt ettgtc-31 (SEQ ID No. 65)<br>
A number of selected library clones (mutated CHI domains cloned in the phagmid vector pHENl) are controlled by restriction analysis and by DNA sequencing to contain the insert as planned, including the correctly inserted randomized sequences. For the following steps of phage preparation, standard protocols are followed. Briefly, the ligation mixture is transformed into E. coli TGI cells by electroporation. Subsequently, phage particles are rescued from E. coli TGI cells with helper phage M13-KO7. Phage particles are then precipitated from culture supernatant with PEG/NaCl in 2 steps, dissolved in water and used for selection by panning or, alternatively, they can be stored at minus 80°C.<br>
Example 13: Panning of the CHI - phage library on hen egg lysozyme (HEL)<br>
3 panning rounds are performed with the CHI - phage library<br>
(see example 12). Maxisorp 96-well plates (Nunc) are coated with<br>
hen egg lysozyme, by adding 200 μ1 of the following solution per<br>
well: PBS, with the following concentrations of dissolved hen<br>
egg lysozyme:<br>
1st panning round: 2 mg/ml HEL 2nd panning round: Img/ml HEL 3rd panning round: 1 mg/ml HEL<br>
Incubation is for 1 hour at 37°C, followed by blocking with 2% dry milk (M-PBS) with 200 μ1 per well for 1 hour at room temperature .<br>
The surface display phage library is then allowed to react with the bound hen egg lysozyme by adding 100 μ1 phage suspension and 100 μ1 4% dry milk (M-PBS) , followed by incubation for 45 minutes with shaking and for 90 minutes without shaking at room temperature.<br>
Unbound phage particles are washed away as follows:<br>
1st panning round: 10 x 300 μ1 T-PBS, 5x 300 μ1 PBS 2nd panning round: 15 x 300 μ1 T-PBS, 10x 300 μ1 PBS 3rd panning round: 20 x 300 μ1 T-PBS, 20 x 300 μ1 PBS<br>
Elution of bound phage particles is performed by adding 200 μ1 per well of 0.1 M glycine, pH 2.2, and incubation with shaking for 30 minutes at room temperature. Subsequently, the phage suspension is neutralized by addition of 60 μ1 2M Tris-Base, followed by infection into E. coli TGI cells by mixture of 10 ml exponentially growing culture with 0.5 ml eluted phage and incubation for 30 minutes at 37°C. Finally, infected bacteria are plated on TYE medium with 1% glucose and 100 pg/ml ampicillin, and incubated at 30°C overnight.<br>
Cloning of selected clones of 'CHI mutants selected against Lysozyme for soluble expression<br>
Phagmid DNA from the phage selected through the 3 panning rounds is isolated with a midi-prep. DNA encoding mutated CHI-domains is batch-amplified by PCR and cloned NcoI-NotI into the vector pNOTBAD/Myc-His, which is the E. coli expression vector pBAD/Myc-His (Invitrogen) with an inserted NotI restriction site to facilitate cloning. Ligated constructs are transformed into E. coli LMG194 cells (Invitrogen) with electroporation, and grown at 30 °C on TYE medium with 1% glucose and ampicillin overnight. Selected clones are inoculated into 200 μ1 2xYT medium with ampicillin, grown overnight at 30°C, and induced by adding L-arabinose to an end concentration of 0.1%. After expression at 16°C overnight, the cells are harvested by centrifu-gation and treated with 100 μ1 Na-borate buffer, pH 8.0, at 4°C overnight for preparation of periplasmic extracts. 50 μ1 of the periplasmic extracts are used in ELISA.<br>
Clones that give a high signal in this first, preliminary ELISA are cultured in a 20-ml volume at the same conditions as described above. Their periplasmic extracts are isolated in 1/20<br>
of the culture volume as described above and tested with ELISA (as described below) for confirmation.<br>
ELISA of CHI mutants selected against hen egg lysozyme Coating:    Microtiter plate  (NUNC,  Maxisorp),  100 μ1 per<br>
well, 100 ug hen egg lysozyme /ml in PBS, Ih at 37°C Wash:  3x 200 μ1 PBS Blocking:   1% BSA-PBS, Ih at RT Wash:  3x 200 μ1 PBS Periplasmic extract binding:    50 μ1 periplasmic extract<br>
50 μ1 2% BSA-PBS, at room temperature overnight Wash:  3x 200 μ1 PBS 1st antibody: anti-His, (Qiagen), 1:1000 in 1% BSA-PBS, 90 min<br>
at RT, 100 μ1 per well Wash:  3x 200 μ1 PBS<br>
2nd antibody: goat anti mouse*HRP (SIGMA), 1:1000 in 1% BSA-PBS, 90 min at RT, 100 μ1 per well Wash:  3x 200 μ1 PBS Detection  :  3mg/ml  OPD  in  Na-citrate/phosphate  buffer,<br>
pH 4.5, 0.4 μ1 30%H202 Stopping: 100 ml 3M H2S04 Absorbance read: 492/620 nm<br>
Clones are interpreted as positive when their ELISA signal is at least three times that of the background signal.<br>
Example 14: Panning of the CL - phage library on hen egg lysozyme (HEL)<br>
3 panning rounds are performed with the CL - phage library<br>
(see example 11) . Maxisorp 96-well plates (Nunc) are coated with<br>
hen egg lysozyme, by adding 200 μ1 of the following solution per<br>
well: PBS, with the following concentrations of dissolved hen<br>
egg lysozyme:<br>
lat panning round: 2 mg/ml HEL 2nd panning round: Img/ml HEL 3rd panning round: 1 mg/ml HEL<br>
Incubation is for 1 hour at 37°C, followed by blocking with 2% dry milk (M-PBS) with 200 μ1 per well for 1 hour at room temperature.<br>
The surface display phage library is then allowed to react<br>
with the bound hen egg lysozyme by adding 100 μ1 phage suspension and 100 μ1 4% dry milk (M-PBS) , followed by incubation for 45 minutes with shaking and for 90 minutes without shaking at room temperature.<br>
Unbound phage particles are washed away as follows:<br>
1st panning round: 10 x 300 μ1 T-PBS, 5x 300 μ1 PBS 2nd panning round: 15 x 300 μ1 T-PBS, lOx 300 μ1 PBS 3rd panning round: 20 x 300 μ1 T-PBS, 20 x 300 μ1 PBS<br>
Elution of bound phage particles is performed by adding 200 μ1 per well of 0.1 M glycine, pH 2.2, and incubation with shaking for 30 minutes at room temperature. Subsequently, the phage suspension is neutralized by addition of 60 μ1 2M Tris-Base, followed by infection into E. coli TGI cells by mixture of 10 ml exponentially growing culture with 0.5 ml eluted phage and incubation for 30 minutes at 37°C. Finally, infected bacteria are plated on TYE medium with 1% glucose and 100 pg/ml Ampicillin, and incubated at 30°C overnight.<br>
Cloning of selected clones of CL mutants selected against Lysozyme for soluble expression<br>
Phagmid DNA from the phage selected through the 3 panning rounds is isolated with a midi-prep. DNA encoding mutated CL-do-mains is batch-amplified by PCR and cloned NcoI-NotI into the vector pNOTBAD/Myc-His, which is the E. coli expression vector pBAD/Myc-His (Invitrogen) with an inserted NotI restriction site to facilitate cloning. Ligated constructs are transformed into E. coli LMG194 cells {Invitrogen) with electroporation, and grown at 30°C on TYE medium with 1% glucose and ampicillin overnight. Selected clones are inoculated into 200 μ1 2xYT medium with ampicillin, grown overnight at 30°C, and induced by adding L-arabinose to an end concentration of 0.1%. After expression at 16°C overnight, the cells are harvested by centrifu-gation and treated with 100 μ1 Na-borate buffer, pH 8.0, at 4°C overnight for preparation of periplasmic extracts. 50 μ1 of the periplasraic extracts are used in ELISA.<br>
Clones that give a high signal in this first, preliminary ELISA are cultured in a 20-ml volume at the same conditions as described above. Their periplasmic extracts are isolated in 1/20 of the culture volume as described above and tested with ELISA<br>
(as described below) for confirmation.<br>
ELISA of CL mutants selected against hen egg lysozyme Coating:    Microtiter plate  (NUNC,  Maxisorp),  100 μ1 per<br>
well, 100 μg hen egg lysozyme /ml in PBS, In at 37°C Wash:  3x 200 μ1 PBS Blocking:   1% BSA-PBS, Ih at RT Wash:  3x 200 μ1 PBS Periplasmic extract binding:    50 μ1 periplasmic extract<br>
50 μ1 2% BSA-PBS/ at room temperature overnight Wash:  3x 200 μ1 PBS 1st antibody: anti-His
at RT, 100 μ1 per well Wash:  3x 200 μ1 PBS<br>
2nd antibody: goat anti mouse*HRP (SIGMA), 1:1000 in 1% BSA-PBS, 90 min at RT, 100 μ1 per well Wash:  3x 200 μ1 PBS Detection : 3mg/ml OPD in Na-citrate/phosphate buffer,   pH<br>
4.5, 0.4 μ1 30%H202 Stopping: 100 ml 3M H2S04 Absorbance read: 492/620 nm<br>
Clones are interpreted as positive when their ELISA signal is at least three times that of the background signal.<br>
Example 15: Construction of an immunoglobulin domain which is randomized on both sides (bispecific engineered CH3 domain)<br>
This example describes an engineered immunoglobulin domain with two binding specificities.<br>
The design of this engineered immunoglobulin domain comprised the following strategy:<br>
an engineered CH3 domain, clone C24 (see example 10), derived from the CH3+5 library binding specifically to lysozyme was used as starting point<br>
residues to be randomized were identified in this modified CH3 domain which are connecting ß-strands of the immunoglobulin fold, and which lie on the opposite side of the domain compared to the residues that were mutated when generating clone C24.<br>
• PCR primers were designed that allowed randomization of these residues and synthesis of this engineered imrauno-globulin domain in a procedure similar to the one described above for the CH3, the CH3+3 and the CH3+5 libraries.<br>
4 PCR products containing randomised positions were ligated and full-length inserts were amplified by PCR. Subsequently, they were cloned in pHEN-1 via NcoI-NotI sites and transformed into E. coli TG-1 cells to construct the library of about 108 colonies. 20 randomly chosen colonies were sequenced and randomised positions were found to be independently mutated. Also no "wild type "(C24) sequence was observed. The phage library was generated following standard protocols, and a phage titer of 6.32 x. 1010 TU/ml was achieved.<br>
In order to test bispecificity, recombinant human Eryth-ropoietin (rhEPO) was chosen as second antigen, while it was expected that the construct retained its originally engineered specificity for hen egg lysozyme. rhEPO-reactive phage was selected in 4 panning rounds. In order to preserve the population of C24 clones that after mutagenesis still should bind hen egg lysozyme, the first round of selection on rhEPO was followed by a round of panning of the phage population on hen egg lysozyme (1 mg/ml in PBS). 200 μ1 of rhEPO was coated on the 5 wells of microtitre plate (Maxisorp, Nunc) in 0.1 M Na-carbonate buffer, pH 9.6, in decreasing concentrations in subsequent panning rounds (see.Table below). After blocking with 2% M-PBS, phage in the blocking agent was allowed to bind at room temperature for 2 h. After 20 washes with T-PBS and 20 with PBS, it was eluted with 6.1 M glycine, pH 2.2, and neutralised with 2M Tris. Eluted phage was used immediately to infect exponentially growing TG-1. Infected cells were selected on ampicilline-containing medium. Phage particles were rescued from culture supernatants upon superinfection with helper phage M13-K07, concentrated with PEG and used in another panning round. Input and output phage numbers were determined as transforming units of E. coli after every panning round (Table 8).<br>
Table 8: <br>
(Table 8 Removed)Raesulting colonies were scraped off the plates, cultured in 2xYT with ampicilline and their plasmid DNA was isolated with a midi-prep. Inserts were amplified with a PCR, and then subcloned into vector pNOTBAD and transformed into an E. coli strain E104. 4x72 colonies were cultured in 200 μ1 2xYT with ampicilline and induced with 0.1% L-arabinose on the following day. After 24h expression at 16 °C, they were lysed with 200 μ1 Na-borate buffer, pH 8.0 for 6 h at 4 °C and periplasmic extract was used in ELISA.<br>
For ELISA, Maxisorp plates were coated with hen egg lysozyrae in PBS (20 pg/ml) or rhEPO in 0.1 M Na-carbonate buffer, pH 9.6, respectively, for 1 h at 37 °C. After blocking with 1% BSA-PBS, periplasmic extract in the same blocking agent was allowed to bind overnight. Binding was revealed with an anti-His-(4) antibody and and a goat anti-mouse IgG antibody, conjugated with HRP (for hen egg lysozyme detection) or AP (for rhEPO detection). Colour reaction of OPD conversion (HRP) was read at 492/620 run after being stopped with 1.25 M H2SO4, and pNPP conversion (AP) was read at 405/620 nm. 14 clones with promising absorbance values were selected for expression at 20-ml-scale. After 24 h ara-binose induction at 16 °C, the cells were collected and lysed overnight in 1 ml Na-borate buffer at 4 °C, and the lysate was used for ELISA. ELISA was performed as above in 4 parallels, and wells without periplasmic extract and without antigen were used as negative controls. Results (Table 9) were achieved with clone according to SEQ ID No. 42, 43.<br>
Table 9: <br>
(Table 9 Removed)<br>
Example 16: Engineered CH3 domains provide bispecificity in an Fab-like format<br>
In the construct used in this example, both the VL and the VH chain of an antibody are fused to an engineered CH3 domain.<br>
The VL and VH region of the human monoclonal antibody 3D6 (He XM, et al. Proc Natl Acad Sci USA. 1992 89:7154-8.; Kohl J, et al. Ann N Y Acad Sci. 1991 646:106-14.; Felgenhauer Mf et al. Nucleic Acids Res. 1990 18:4927), which recognizes an epi-tope on gp41 of HIV-1 was used as fusion partner for the engineered CH3 domain clone C24 which binds specifically to hen egg lysozyme.<br>
In order to promote the formation of the VL-CH3 / VH-CH3 di-mer via a disulfide bond, the residues Ser-Cys were added to the C-terminus of the C24 sequence.<br>
The nucleotide- and amino acid sequences respectively of the two chains, 3D6VL-C24 and 3D6VH-C24 are given in SEQ ID No. 47, 46 and SEQ ID No. 45, 44, respectively.<br>
Primers were designed that allow the amplification of the coding regions, introducing restriction sites at the same time (silent mutations) which were used to ligate the coding regions together. For expression of the genes, the Pichia pastoris expression system was chosen. Constructs were cloned in suitable Pichia pastoris expression vectors: 3D6VL-C24 was cloned in the pPIC9K (final name: pPIC9K3LC) and 3D6VH-C24 (final name: pPICZ3HC) was cloned in pPICZalphaA. Construct pPICZ3HC was linearized with Bgl II, transformed into Pichia pastoris GS115 and transformants were selected on zeocin-containing solid medium. One of the transformants was subsequently used as a host cell for the Sal I- linearized construct pPIC9K3LC. Double transform-ants were then selected on RDB - medium.<br>
Clones were inoculated into 30 ml YPG medium and grown until OD600=10, and were then induced by the addition of 1% methanol in BMMY medium. The induction was continued for 36 hours at 16 °C. Supernatants were removed by centrifugation and were then concentrated about 10-times. Presence of the recombinant protein<br>
was confirmed by a Western blot with an anti-His (4) antibody, and was estimated to be at a concentration of approximately 50 -100 μg/1 initial culture.<br>
First functional tests wer performed with lOx- concentrated supernatant. Firstly, wells of Maxisorp plates were coated with 20 μg/ml hen egg lysozyme in PBS or 20 μg/ml epitope of the antibody 3D6 in 0.1 M Na-carbonate buffer, pH 9.6, respectively, for Ih at 37 °C. The 3D6 epitope was used in the form of a re-combinantly produced GST-fusion protein.  After blocking with 1% BSA-PBS, concentrated supernatants were allowed to bind overnight in the same blocking agent. Binding was revealed with an anti-His (4) antibody and goat anti-mouse antibody, conjugated to HRP, and visualised as colour reaction resulting from OPD conversion at 492/620 nm (Table 10). (Table 10 Removed)<br>
<br>
<br>
<br>
<br>
<br>
<br>
<br>
Claims:<br>
1.	Method for engineering an immunoglobulin comprising at least<br>
one modification in a structural loop region of said immuno<br>
globulin and determining the binding of said immunoglobulin to<br>
an epitope of an antigen, wherein the unmodified immunoglobulin<br>
does not significantly bind to said epitope, comprising the<br>
steps of:<br>
-	providing a nucleic acid encoding an immunoglobulin com<br>
prising at least one structural loop region,<br>
-	modifying at least one nucleotide residue of at least one<br>
of said structural loop regions,<br>
-	transferring said modified nucleic acid in an expression<br>
system,<br>
-	expressing said modified immunoglobulin,<br>
-	contacting the expressed modified immunoglobulin with an<br>
epitope, and<br>
-	determining whether said modified immunoglobulin binds to<br>
said epitope.<br>
<br>
2.	Method according to claim 1, wherein said immunoglobulin is<br>
binding specifically to at least two different epitopes.<br>
3.	Method according to any of claim 1 or claim 2, comprising at<br>
least one modification in at least one structural loop region of<br>
said immunoglobulin and determining the specific binding of said<br>
at least one loop region to at least one molecule selected from the group consisting of allergens, tumor associated antigens, self antigens, enzymes, bacterial antigens, fungal antigens, viral antigens and protozooal antigens, wherein the immunoglobulin containing an unmodified structural loop region does not specifically bind to said at least one molecule.<br>
4.	Method for manufacturing an immunoglobulin or a pharmaceutic<br>
al preparation thereof comprising at least one modification in a<br>
structural loop region of said immunoglobulin and determining<br>
the binding of said immunoglobulin to an epitope of an antigen,<br>
wherein the unmodified immunoglobulin does not significantly<br>
bind to said epitope, comprising the steps of:<br>
-	providing a nucleic acid encoding an immunoglobulin com-<br>
 <br>
<br>
prising at least one loop region,<br>
-	modifying at least one nucleotide residue of at least one<br>
of said loop regions,<br>
-	transferring said modified nucleic acid in an expression<br>
system,<br>
-	expressing said modified immunoglobulin,<br>
-	contacting the expressed modified immunoglobulin with an<br>
epitope,<br>
-	determining whether said modified immunoglobulin binds to<br>
said epitope, and<br>
-	providing the modified immunoglobulin binding to said epi<br>
tope and optionally finishing it to a pharmaceutical prepara<br>
tion.<br>
5. Method for manufacturing a multi-specific immunoglobulin binding specifically to at least one first molecule or a pharmaceutical preparation thereof comprising at least one modification in at least one structural loop region of said immunoglobulin and determining the specific binding of said at least one loop region to at least one second molecule selected from the group consisting of allergens, tumor associated antigens, self antigens, enzymes, bacterial antigens, fungal antigens, viral antigens and protozooal antigens, wherein the immunoglobulin containing an unmodified structural loop region does not specifically bind to said at least one second molecule, comprising the steps of:<br>
-	providing a nucleic acid encoding an immunoglobulin bind<br>
ing specifically to at least one first molecule comprising at<br>
least one structural loop region,<br>
-	modifying at least one nucleotide residue of at least one<br>
of said loop regions encoded by said nucleic acid,<br>
-	transferring said modified nucleic acid in an expression<br>
system,<br>
-	expressing said modified immunoglobulin,<br>
-	contacting the expressed modified immunoglobulin with said<br>
at least one second molecule, and<br>
-	determining whether said modified immunoglobulin binds<br>
specifically to the second molecule and<br>
-	providing the modified immunoglobulin specifically to said<br>
at least one second molecule and optionally finishing it to a<br>
 <br>
pharmaceutical preparation.<br>
6.	Method according to any of the claims 1 to 5, characterised<br>
in that the immunoglobulin is of human or murine origin.<br>
7.	Method according to claim 6, characterised in that the human<br>
immunoglobulin is selected from the group consisting of IgAl,<br>
IgA2, IgD, IgE, IgGl, IgG2, IgG3, IgG4 and IgM.<br>
8.	Method according to claim 6, characterised in that the murine<br>
immunoglobulin is selected from the group consisting of IgA,<br>
IgD, IgEf IgGl, IgG2A, IgG2B, IgG2C, IgG3 and IgM.<br>
9.	Method according to claim 7 or 8, characterised in that the<br>
immunoglobulin comprises a heavy and/or light chain of the im<br>
munoglobulin or a part thereof.<br>
<br>
10.	Method according to any one of claims 7 to 9f characterised<br>
in that the immunoglobulin comprises at least one constant do<br>
main and/or at least one variable domain of the immunoglobulin,<br>
or a part thereof including a minidomain.<br>
11.	Method according to claim 10, characterised in that the con<br>
stant domain is selected from the group of CH1, CH2, CH3, CH4,<br>
Igk-C, Igl-C and combinations thereof.<br>
12.	Method according to claim 11, characterised in that the mod<br>
ified loop regions of CH1, CH2, CH3 and CH4 comprise amino acids<br>
7	to 21, amino acids 25 to 39, amino acids 41 to 81, amino acids<br>
83 to 85, amino acids 89 to 103 and amino acids 106 to 117.<br>
13.	Method according to claim 11, characterised in that the loop<br>
regions of Igk-C and Igl-C of human origin comprise amino acids<br>
8	to 18, amino acids 27 to 35, amino acids 42 to 78, amino acids<br>
83 to 85, amino acids 92 to 100, amino acids 108 to 117 and<br>
amino acids 123 to 126.<br>
14.	Method according to claim 11, characterised in that the loop<br>
regions of Igk-C and Igl-C of murine origin comprise amino acids<br>
8 to 20, amino acids 26 to 36, amino acids 43 to 79, amino acids<br>
 <br>
83 to 85, amino acids 90 to 101, amino acids 108 to 116 and amino acids 122 to 125.<br>
15.	Method according to claim 10, characterised in that the<br>
structural loop regions of the variable domain of the immuno-<br>
globulin of human origin comprise amino acids 8 to 20, amino<br>
acids 44 to 50, amino acids 67 to 76 and amino acids 89 to 101.<br>
16.	Method according to claim 10, characterised in that the<br>
structural loop regions of the variable domain of the immuno-<br>
globulin of murine origin comprise amino acids 6-to 20, amino<br>
acids 44 to 52, amino acids 67 to 76 and amino acids 92 to 101.<br>
17.	Method according to any of the claims 1 to 5, characterised<br>
in that the immunoglobulin is of camel origin.<br>
18.	Method according to claim 17, characterised in that the im<br>
munoglobulin comprises at least one constant domain selected<br>
from the group consisting of CHI, CH2 and CH3.<br>
19.	Method according to claim 18, characterised in that the loop<br>
regions of CH1, CH2 and CH3 comprise amino acids 8 to 20, amino<br>
acids 24 to 39, amino acids 42 to 78, amino acids 82 to 85,<br>
amino acids 91 to 103 and amino acids 108 to 117.<br>
20.	Method according to any one of claims 1 to 19, characterised<br>
in that the modification of at least one nucleotide results in a<br>
substitution, deletion and/or insertidn of the immunoglobulin<br>
encoded by said nucleic acid.<br>
21.	Method according to any one of claims 1 to 20, characterised<br>
in that the amino acids of at least one loop region are modified<br>
by site-directed random mutation.<br>
22.	Method according to claim 21, characterised in that the ran<br>
domly modified nucleic acid molecule comprises at least one nu<br>
cleotide repeating unit having the sequence 5'-NNS-3', 5'-NUN<br>
S' or 5'- NNK-3'.<br>
23.	Method according to any one of claims 1 to 22, characterised<br>
 <br>
in that the expression system comprises a vector.<br>
24.	Method according to any one of claims 1 to 23 characterised<br>
in that the modified immunoglobulin is expressed in a host,<br>
preferably in a bacterial, a yeast, a plant cell, in an animal<br>
cell or in a plant or animal.<br>
25.	Method according to any one of claims 1 to 24, characterised<br>
in that the specific binding of the modified immunoglobulin to<br>
the molecule is determined by a binding assay selected from the<br>
group consisting of immunological assays, preferably enzyme<br>
linked immunosorbent assays (ELISA), surface plasmon resonance<br>
assays, saturation transfer difference nuclear magnetic reson<br>
ance spectroscopy, transfer NOE (trNOE) nuclear magnetic reson<br>
ance spectroscopy, competitive assays, tissue binding assays,<br>
live cell binding assays and cellular extract assays.<br>
26.	Method according to any one of claims 1 to 25, characterised<br>
in that the modified immunoglobulin is conjugated to a label se<br>
lected from the group consisting of organic molecules, enzyme<br>
labels, radioactive labels, colored labels, fluorescent labels,<br>
chromogenic labels, luminescent labels, haptens, digoxigenin,<br>
biotin, metal complexesr metals, colloidal gold and mixtures<br>
thereof.<br>
27.	Immunoglobulin consisting of a constant domain selected from<br>
the group consisting of CH1, CH2, CH3, CH4, Igk-C, Igl-C, or a<br>
part thereof including a minidomain, with at least one loop re<br>
gion, characterised in that said at least one loop region com<br>
prises at least one amino acid modification forming at least one<br>
modified loop region, wherein said at least one modified loop<br>
region binds specifically to at least one epitope of an antigen.<br>
28.	Immunoglobulin consisting of a varable domain of a heavy or<br>
light chain, or a part thereof including a minidomain, with at<br>
least one loop region, characterised in that said at least one<br>
loop region comprises at least one amino acid modification form<br>
ing at least one modified loop region, wherein said at least one<br>
modified loop region binds specifically to at least one epitope<br>
of an antigen.<br>
29.	Immunoglobulin according to claim 27 or 28, characterised in<br>
that the antigen is selected from the group consisting of pro-<br>
teinaceous molecules, nucleic acids, and carbohydrates.<br>
30.	Immunoglobulin according to any of the claims 27 to 29,<br>
characterised in that the antigen is selected from the group<br>
consisting of tumor associated antigens, in particular EpCAM,<br>
tumor-associated glycoprotein-72 (TAG-72), tumor-associated an<br>
tigen CA 125, Prostate specific membrane antigen (PSMA), High<br>
molecular weight melanoma-associated antigen (HMW-MAA), tumor-<br>
associated antigen expressing Lewis Y related carbohydrate, Car-<br>
cinoembryonic antigen (CEA), CEACAM5, HMFG PEM, mucin MUC1,<br>
MUC18 and cytokeratin tumor-associated antigen, bacterial anti<br>
gens, viral antigens, allergens, fluorescein, lysozyme, toll-<br>
like receptor 9, erythropoietin, CD2, CDS, CD3E, CD4, CD11,<br>
CDlla, CD14, CD18, CD19, CD20, CD22, CD23, CD25, CD28, CD29,<br>
CD30, CD33 (p67 protein), CD38, CD40, CD40L, CD52, CD54, CD56,<br>
CD80, CD147, GD3, IL-1, IL-1R, IL-2, IL-2R, IL-4, IL-5, IL-6,<br>
IL-6R, IL-8, IL-12, IL-15, IL-18, IL-23, interferon alpha, in-<br>
terferon beta, interferon gamma; TNF-alpha, TNFbeta2, TNF.al<br>
pha., TNFalphabeta, TNF-Rl, TNF-RII, FasL, CD27L, CD30L, 4-1BBL,<br>
TRAIL, RANKL, TWEAK, APRIL, BAFF, LIGHT, VEG1, OX40L, TRAIL Re-<br>
ceptor-1, Al Adenosine Receptor, Lymphotoxin Beta Receptor,<br>
TACI, BAFF-R, EPO; LFA-3, ICAM-1, ICAM-3, integrin betal, integ-<br>
rin beta2, integrin alpha4/beta7, integrin alpha2, integrin al-<br>
pha3, integrin alpha4, integrin alphas, integrin alpha6, integ<br>
rin alphav, alphaVbetaS integrin, FGFR-3, Keratinocyte Growth<br>
Factor, VLA-1, VLA-4, L-selectin, anti-Id, E-selectin, HLA, HLA-<br>
DR, CTLA-4, T cell receptor, B7-1, B7-2, VNRintegrin, TGFbetal,<br>
TGFbeta2, eotaxinl, BLyS (B-lymphocyte Stimulator), complement<br>
C5, IgE, factor VII, CD64, CBL, NCA 90, EGFR (ErbB-1) , Her2/neu<br>
(ErbB-2), Her3 (ErbB-3), Her4 (ErbB4), Tissue Factor, VEGF, VEG-<br>
FR, endothelin receptor, VLA-4, carbohydrates such as blood<br>
group antigens and related carbohydrates, Galili-Glycosylation,<br>
Gastrin', Gastrin receptors, tumor associated carbohydrates,<br>
Hapten NP-cap or NIP-cap, T cell receptor alpha/beta, E-se<br>
lectin, digoxin, placental alkaline phosphatase (PLAP) and<br>
testicular PLAP-like alkaline phosphatase, transferrin receptor,<br>
Heparanase I, human cardiac myosin, Glycoprotein Ilb/IIIa<br>
 <br>
 (GPIIb/IIIa), human cytomegalovirus (HCMV) gH envelope glycoprotein, HIV gp!20/ HCMV, respiratory syncital virus RSV F, RSVF Fgp, VNRintegrin, Hep B gpl20, CMV, gpIIbIIIa, HIV IIIB gp120 V3 loop, respiratory syncytial virus (RSV) Fgp, Herpes simplex virus (HSV) gD glycoprotein, HSV gB glycoprotein, HCMV gB envelope glycoprotein, Clostridium perfringens toxin and fragments thereof.<br>
31.	Immunoglobulin according to any one of claims 27 to 30,<br>
characterised in that amino acid residues of positions 15 to 17,<br>
29 to 34, 85.4 to 85.3, 92 to 94, 97 to 98 and/or 108 to 110 of<br>
CH3 are modified.<br>
32.	Immunoglobulin according to any one of claims 27 to 31,<br>
characterised in that the modification is a deletion, a substi<br>
tution, a insertion or a combination thereof.<br>
33.	Immunoglobulin according to any one of claims 27 to 32,<br>
characterised in that the CH3 region comprises SEQ ID No. 16 or<br>
SEQ ID No. 18, when EpCam binds to said immunoglobulin, SEQ ID<br>
No. 20, when fluorescein binds to said immunoglobulin, SEQ ID<br>
No. 22, 24, 26, 28, 30 or 32, when lysozyme binds to said immun<br>
oglobulin, SEQ ID No. 34, 36, 38 or 40, when TLR9 binds to said<br>
immunoglobulin, and SEQ ID No. 42, when lysozyme and/or eryth-<br>
ropoietin bind to said immunoglobulin.<br>
34.	Immunoglobulin according to any one of claims 27 to 33,<br>
characterised in that it comprises SEQ ID No. 44 or SEQ ID No.<br>
46, when lysozyme and gp41 bind to said immunoglobulin.<br>
35.	Immunoglobulin according to any one of claims 27 to 34,<br>
characterised in that the modified immunoglobulin is conjugated<br>
to a label or reporter molecule selected from the group consist<br>
ing of organic molecules, enzyme labels, radioactive labels,<br>
colored labels, fluorescent labels, chromogenic labels, lumines<br>
cent labels, haptens, digoxigenin, biotin, metal complexes,<br>
metals, colloidal gold and mixtures thereof.<br>
36.	Use of an immunoglobulin according to any one of claims 27<br>
to 35 or obtainable by the method according to any one of claims<br>
 <br>
1 to 26 for the preparation of a vaccine for active immunization.<br>
37.	Use of an immunoglobulin according to any one of claims 27<br>
to 35 or obtainable by the method according to any one of claims<br>
1 to 26 for the preparation of a protein library.of immuno-<br>
globulins.<br>
38.	Protein library comprising an immunoglobulin obtainable by<br>
the method according to any one of claims 27 to 35 or obtainable<br>
by the method according to any one of claims 1 to 26.<br>
39.	Method for specifically binding and/or detecting a molecule<br>
comprising the steps of:<br>
<br>
(a)	contacting a modified immaunoglobulin according to any<br>
one of claims 27 to 35 or a modified immunoglobulin obtainable<br>
by a method according to any one of claims 1 to 26 with a test<br>
sample suspected to contain said molecule, and<br>
(b)	detecting the potential formation of a specific immuno-<br>
globulin/molecule complex.<br>
40.	Method for isolating specifically a molecule comprising the<br>
steps of:<br>
(a)	contacting a modified immunoglobulin according to any<br>
one of claims 27 to 35 or a modified immunoglobulin obtainable<br>
by a method according to any one of claims 1 to 26 with a sample<br>
containing said molecule.<br>
(b)	separating the specific immunoglobulin/molecule complex<br>
formed, and<br>
(c)	optionally isolating the molecule from said complex.<br>
41.	Method for targeting a compound to a target comprising the<br>
steps of:<br>
(a) contacting a modified immunoglobulin according to any one of claims 27 to 35 or a modified immunoglobulin obtainable by a method according to any one of claims 1 to 26 binding specifically to said compound,<br>
{b) delivering the specific immunoglobulin/compound complex to the target.<br>
42.	A modified immunoglobulin having an antigen binding site<br>
foreign to the unmodified immunoglobulin and incorporated in one<br>
or more structural loops.<br>
43.	Immunoglobulin according to claim 42, wherein said antigen<br>
is selected from the group consisting of allergens, tumor asso<br>
ciated antigens, self antigens, enzymes, bacterial antigens,<br>
fungal antigens, viral antigens and protozooal antigens.<br>
44.	Immunoglobulin according to any claim 42 or 43, wherein said<br>
immunoglobulin comprises at least two antigen binding sites, the<br>
first site binding to a first epitope, and the second site bind<br>
ing to a second epitope.<br>
45.	Immunoglobulin according to any one of claims 42 to 44,<br>
wherein said immunoglobulin comprises at least two loop regions,<br>
the first loop region binding to a first epitope, and the second<br>
loop region binding to a second epitope.<br>
46.	Immunoglobulin according to claim 45, wherein said loop re<br>
gions contain structural loops.<br>
47.	Immunoglobulin according to any one of claims 42 to 46 being<br>
composed of at least two immunoglobulin domains, or a part<br>
thereof including a minidomain, and each domain contains at<br>
least one antigen binding site.<br>
48.	Immunoglobulin according to any one of claims 42 to 47,<br>
wherein the immunoglobulin comprises at least one domain of the<br>
constant region and/or at least one domain of the variable re<br>
gion of the immunoglobulin, or a part thereof including a min<br>
idomain .<br>
49.	Immunoglobulin according to any one of claims 42 to 48,<br>
wherein said immunoglobulin comprises a domain that has at least<br>
50% homology with the unmodified domain.<br>
50.	Immunoglobulin according to any one of claims 42 to 49,<br>
wherein said immunoglobulin is a bispecific antibody or a bispe-<br>
cific single chain antibody.<br>
51.	Immunoglobulin according to any one of claims 42 to 50,<br>
wherein said immunoglobulin comprises a bispecific domain or a<br>
part thereof including a minidomain.<br>
52.	Immunoglobulin according to any one of claims 42 to 51, for<br>
therapeutic and prophylactic use.<br>
53.	Immunoglobulin according to any one of claims 42 to 51, for<br>
preparative and analytic use.<br>
54.	Immunoglobulin according to any one of claims 42 to 51, for<br>
diagnostic use.<br>
55.	Library consisting of at least 100 immunoglobulins according<br>
to any of the claims 27 to 35 and 42 to 54 with different modi<br>
fications in the structural loops.<br>
56.	Library according to claim 55, wherein said immunoglobulins<br>
are immaunoglobulin selected from the group consisting of domains<br>
of an immunoglobulin, minidomains or derivatives thereof.<br>
57.	Kit of binding partners containing<br>
<br>
(a)	a modified immunoglobulin having an antigen binding site<br>
foreign to the immunoglobulin incorporated in one or more<br>
structural loops, and<br>
(b)	a binding molecule containing an epitope of said antigen.<br>
58.	Use of the binding molecule of a kit according to claim 57<br>
for identifying the binding specificity of said modified immuno<br>
globulin thereof.<br>
59.	Use of the binding molecule of a kit according to claim 57<br>
for determining the potency of said modified immunoglobulin.<br>
60.	Use of the binding molecule of a kit according to claim 57<br>
for selecting said modified immunoglobulin from a library con<br>
sisting of at least 100 immunoglobulins with different modifica<br>
tions in the structural loops.<br>
</td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/viewdoc.aspx?id=QE8pGhdwZvsRo5e1aePA2g==&amp;amp;loc=+mN2fYxnTC4l0fUd8W4CAA==" target="_blank" style="word-wrap:break-word;">http://ipindiaonline.gov.in/patentsearch/GrantedSearch/viewdoc.aspx?id=QE8pGhdwZvsRo5e1aePA2g==&amp;amp;loc=+mN2fYxnTC4l0fUd8W4CAA==</a></p>
		<br>
		<div class="pull-left">
			<a href="272720-method-for-the-enantioselective-enzymatic-reduction-of-hydroxyketo-compounds.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="272722-novel-technique-for-hypersonic-drag-control-using-heat-addition-in-the-shock-layer.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>272721</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>4657/DELNP/2007</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>18/2016</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>29-Apr-2016</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>21-Apr-2016</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>18-Jun-2007</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>F-STAR BIOTECHNOLOGISCHE FORSCHUNGS-UND ENTWICKLUNGSGES.M.B.H.</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td>SCHWRZENBERGPLATZ 7, A-1030 VIENNA, AUSTRIA</td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>RUKER FLORIAN</td>
											<td>MONTIGASSE 6, A-1170 VIENNA AUSTRIA.</td>
										</tr>
										<tr>
											<td>2</td>
											<td>WOZNIAK-KNOPP GORDANNA</td>
											<td>PETER-JORDAN-STRASSE 163/2/1, A-1180 VIENNA AUSTRIA.</td>
										</tr>
										<tr>
											<td>3</td>
											<td>HIMMLER GOTTFRIED</td>
											<td>COLLOREDOGASSE 29/10, A-1180 VIENNA AUSTRIA.</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>C07K 16/00</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>PCT/EP2006/050059</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td>2006-01-05</td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td>60/641,144</td>
									<td>2005-01-05</td>
								    <td>U.S.A.</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script data-cfasync="false" src="../cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/272721-synthetic-immunoglobulin-domains-with-binding-properties-engineered-in-regions-of-the-molecule-different-from-the-complementarity-determining-regions by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 00:53:48 GMT -->
</html>
