<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/272765-a-simple-protocol-for-isolation-of-undegraded-total-rna-from-eucalyptus-and-casuarina-and-cdna-synthesis-from-unpurified-rna by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 01:06:58 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 272765:A SIMPLE PROTOCOL FOR ISOLATION OF UNDEGRADED TOTAL RNA FROM EUCALYPTUS AND CASUARINA AND cDNA SYNTHESIS FROM UNPURIFIED RNA</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">A SIMPLE PROTOCOL FOR ISOLATION OF UNDEGRADED TOTAL RNA FROM EUCALYPTUS AND CASUARINA AND cDNA SYNTHESIS FROM UNPURIFIED RNA</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>&quot;A simple protocol for isolation of undegraded total RNA from Eucalyptus and Casuarina and cDNA synthesis from unpurified RNA&quot;. This invention relates to a short protocol for isolation of undegraded total RNA with high recovery from Eucalyptus tereticomis and Casuarina equisetifolia and cDNA synthesis from unpurified RNA comprising the steps, extracting total RNA from different tissues in extraction buffer including activated charcoal (0.1 to 1%), precipitation of total RNA using 2M - 4M Sodium Chloride instead of Lithium Chloride, final precipitation of total RNA in absolute alcohol without sodium acetate, reverse transcription of unpurified total RNA to complementary DNA using 0.5 -2% PVP and l-2mg/ml BSA in the reaction mixture and generation of single stranded cDNA suitable for PCR reaction.</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td> <br>
<br>
"A simple protocol for isolation of undegraded total RNA from Eucalyptus and Casuarina and cDNA synthesis from unpurified RNA".<br>
FIELD OF INVENTION<br>
This invention relates to isolation of undegraded total RNA from Eucalyptus and Casuarina and cDNA synthesis from unpurified ribonucleic acid using a simple protocol and nonhazardous chemicals.<br>
BACKGROUND OF INVENTION<br>
Isolation of high quality, intact Ribonucleic Acid (RNA) is the first and often the most critical step in performing many fundamental molecular biology experiments, including Northern analysis, nuclease protection assays, RT-PCR, RNA mapping, in vitro translation and cDNA library construction. The separation of RNA from the complex mixtures in which they are often found is necessary before other procedures are undertaken. The presence of large amounts of cellular or other contaminating biomolecules like proteins, carbohydrates and phenolic compounds in such complex mixtures often impedes many of the reactions and techniques used in molecular biology. Also, for RNA isolation, rapid methods are required, since RNA usually are very unstable and rapidly degraded by ubiquitous RNases present in cells, tissues and body fluids. The quality of RNA is probably the most important factor in determining the quality of the final results in protocols utilizing messenger RNA (mRNA), especially for complementary DNA (cDNA) synthesis. The basic total RNA isolation protocols from plant tissues include<br>
 <br>
[l]The Guanidinium based methods using Guanidine thiocyanate / Guanidine Hydrochloride (a strong chaotropic denaturant) in extraction buffer to. disrupt cells, solubilize components and denature endogenous RNases simultaneously, followed by ethanol extraction or ultracentrifugation across cesium chloride gradient (Chirgwin et al., 1979). Chomczynski and Sacchi in 1987 improved the method to a single step protocol through the use of a mixture of guanidinium thiocyanate and phenol-chloroform. The TRIzol (Guanidinium thiocyanate-phenol-chloroform extraction) method described by Chomczynski and Sacchi in 1987 and brand product of Invitrogen is also a commonly used protocol.<br>
[2] Another method is the phenol/SDS method using Tris (Tris hydroxymethyl aminomethane), EDTA (Ethylene diamine tetraacetic acid), SDS (Sodium dodecyl sulphate), lithium chloride (LiCl) and phenol for extraction of total RNA followed by precipitation in Lithium chloride and final precipitation using Ethanol (Ausubel et al., 1998).<br>
Lithium Chloride has been frequently used to precipitate RNA, although precipitation with alcohol and a monovalent cation such as sodium or ammonium ion is also widely used. Lithium Chloride precipitation offers major advantages over other RNA precipitation methods in that it does not efficiently precipitate DNA, protein or carbohydrate (Barlow et al., 1963) and has been widely used for selective precipitation of RNA from plant tissues (Chang et al., 1993; Zheng and Yang, 2002; Azevedo et al., 2003; Kolosova et al., 2004; Wang et al., 2007; Chan et al., 2007; Zamboni et al., 2008). Modified protocols involving two precipitation<br>
 <br>
steps including lithium chloride and ethanol is also described (Hu et al., 2002; Chan et al., 2004; Pateraki and Kanellis, 2004; Rubio-Pina, 2008).<br>
P Mercaptoethanol (BME), a strong reducing agent is often used in total RNA isolation to prevent oxidation of phenolic compounds (Lai et al., 2001). Further, it is used in some RNA isolation procedures to eliminate ribonuclease released during cell lysis. Numerous disulfide bonds make ribonucleases very stable enzymes, so mercaptoethanol is used to reduce these disulfide bonds and irreversibly denature the protein. This prevents them from digesting the RNA during the isolation procedure (Nelson et al., 2005). In most RNA isolation protocol (including commercial kits), BME is freshly added to the extraction buffer (Chirgwin et al., 1979; Chang et al., 1993; Hu et al., 2002; Sharma et al., 2003; Jaiprakash et al., 2003; Azevedo et al., 2003; Gasic et al., 2004; Wang et al., 2007; Zamboni et al., 2008; Vasanthaiah et al., 2008).<br>
Several protocols for RNA isolation from tissues of species with high contents of polyphenols or polysaccharides have been reported, including methods using polyvinylpyrrolidone (PVPP) (Woodhead et al., 1997), soluble PVP and ethanol precipitation (Salzman et al. 1999), hot borate (Wan 8B Wilkins 1994), phenol extraction (Komjanc et al. 1999), calcium precipitation (Dal Cin et al. 2005), Vanadyl ribonucleoside complex (Baba and Ito, 1997) and 2-butoxyethanol (Malnoy et al. 2001, Manning 1990). Alternate procedures using SDS and proteinase K to reduce ribonuclease activity was described by Wan and Wilkins (1994). Protocol using CTAB in extraction buffer is 'routinely used for RNA isolation and does not include steps involving ultracentrifugation (Murray and Thompson,<br>
 <br>
1980). Further, the qualitative and quantitative differences in composition of phenolics and polysaccharides in different plant tissues significantly alter the efficiency of nucleic acid extraction and purification procedures.<br>
Subsequently, the basic protocols have been modified to suite the tissue type for total RNA isolation. Modification of the acid guanidinium thiocyanate-phenol-chloroform method (Vareli and Frangou- Lazaridis, 1996), modification of the hot borate method (Wan and Wilkins, 1994), modified CTAB protocol (Hu et al., 2002; Gasic et al., 2004), modified SDS/ phenol method (Duan et al., 1997) and the acetone treatment of freeze-dried and powdered plant materials (Schneiderbauer et al., 1991) are reported in literature. Combinations of conventional protocols are also reported like Guanidine hydrochloride / phenol protocol for freeze dried tea leaves (Jaiprakash et al., 2003); CTAB/ PVP for pine needles (Chang et al., 1993; Zeng and Yang, 2002) and Sodium borate/CTAB protocol for recalcitrant tree species like Pinus, Populus, Hevea, Swietenia, Sapindus, Dimocarpus, litchi and Musa (Wang et al., 2007). In spruce and Populus (needles, bark, xylem and leaf tissues), a modified protocol was described where CTAB was used to remove the residual polysaccharide from nucleic acid pellet (Kolosova et al., 2004). Recent protocol involving acid phenol - silica method was described by Ding et al. (2008) for tissues recalcitrant to guanidine thiocyanate.<br>
Further, Kits supplied by biotechnology companies extract RNA successfully from many plant tissues and often employ the spin column technology to isolate intact RNA from relatively low quantity of tissue (75 - lOOmg) and the protocols are usually of short duration. However, they<br>
 <br>
can be ineffective on tissues rich in polyphenols or polysaccharides and in tissues recalcitrant to guanidinium salts.<br>
In tree species, the process of isolation of RNA is highly cumbersome due to the presence of high quantity of secondary metabolites which co -precipitate with RNA and interfere with down stream protocols. In the family Casuarinaceae, the RNA isolation protocols are defined for only two species, Casuarina glauca and Allocasuarina verticillata. In C. glauca, total RNA for Northern hybridization was isolated from nodule, uninfected roots, and aerial parts using the protocol described by Bugos et al. in 1995 (Laplaze et al., 1999). The protocol described by Bugos et al. (1995) involved the extraction of total RNA in homogenization buffer (containing Tris, NaCl, EDTA, Sarkosyl and BME and phenol/ chloroform/ isoamyl alcohol) followed by two precipitation, first one with isopropanol and sodium acetate and second with Lithium chloride. Further, RT-PCR was conducted after isolating mRNA using commercial kit. In Allocasuarina verticillata, total RNA isolation from roots was done using Qiagen commercial kit (Gherbi et al., 2008).<br>
In eucalypts, RNA isolation is difficult due to the presence of high polyphenols and polysaccharide content specifically in the leaf tissues. Several protocols have been described in literature for RNA isolation from different tissues of eucalypts. A rapid and simple method was developed for RNA isolation from leaves of eucalypt species by using an extraction buffer containing sodium isoascorbate at a concentration of 500 mM. This method consisted of one or two chloroform extractions, one acid guanidium-phenol-chloroform extraction and isopropanol precipitation (Suzuki et al., 2003). The RNA isolated using this protocol was found<br>
 <br>
suitable for RT PCR reactions. This protocol was also used to isolate total RNA from roots of E. camaldulensis (Koyama et al., 2006). In E. globulus seedlings, RNA isolation was conducted using the protocol described by Chang et al., (1993) using CTAB and PVP. However, RT-PCR was conducted after isolating mRNA from total RNA (Rasmussen-Poblete et al., 2008).<br>
In E. grandis, total RNA was isolated from xylem, phloem, roots, young leaves, mature leaves and apical/lateral meristems using the CTAB protocol described by Chang et al. (1993) followed by a RNA clean up using the commercial RNAeasy Plant Mini Kit (Qiagen USA, Valencia, CA) prior to cDNA synthesis (Novaes et al., 2008). This protocol was also used by Ranik and Myburg (2006) to isolate RNA from xylem tissues from E. grandis.<br>
In E. camaldulensis, RNA was isolated from the differentiating xylem tissues using CTAB protocol with major modifications. An initial extraction of the ground tissues in methanol and DTT was described, followed by isolation of RNA in extraction buffer containing Vanadyl Ribonucleoside Complex followed by addition of NaCl and CTAB and precipitation using isopropanol. Vanadyl ribonucleosides were added to inhibit the binding of polyphenols to RNA and inhibit RNase activity (Baba and Ito, 1997). mRNA was subsequently isolated from total RNA using commercial kit.<br>
 <br>
Reverse Transcription of Total RNA:<br>
Reverse Transcription (RT reaction) is a process in which single-stranded RNA is reverse transcribed into complementary DNA (cDNA). The main problems associated with plant RNA isolation is attributed primarily to the presence and co-purification of plant polyphenolic compounds and polysaccharides (Baker et al., 1990; Levi et al., 1992, Lopez-Gomez and Gomez-Lim, 1992; Schneiderbauer et al., 1991). During cDNA synthesis and Northern analysis, these substances bind to RNA and render it unsuitable for downstream applications (Tesniere and Vayda, 1991). Inhibitors in the reaction mix can sabotage cDNA synthesis, so an important part of optimizing a reaction mixture is dealing with inhibitors. The common inhibitors include salts, ionic detergents such as sodium deoxycholate, sarkosyl and SDS (Weyant et al., 1990), ethanol and isopropanol (Loffert et al., 1997) and phenol (Katcher and Schwartz, 1994). The standard method for dealing with inhibitors is through the process of dilution.<br>
Several chemical components have been used in PCR reactions to alleviate the problem of contamination of nucleic acid. Bovine Serum Albumin (BSA) has many uses as a carrier protein and as a stabilizing agent in enzymatic reactions. BSA is also a common additive for PCR amplifications (Kreader, 1996; Al-Soud and Radstrom, 2000; Wang et al., 2007), footprinting and gel shift assays. In restriction digests, BSA has been shown to enhance enzyme activity. BSA has been used in PCR reactions with cDNA as template for Real-time quantitative PCR (RQ-PCR) to enhance PCR sensitivity (Silvy et al., 2004; Wang et al., 2007). Acetylated BSA addition was found to increase the sensitivity of the RT-<br>
 <br>
PCR by 2 -3 orders of magnitude in conditions with limiting RNA template (Nathan and Fox, 1997).<br>
Polyvinylpyrrolidone (PVP) has been reported to prevent Taq DNA polymerase from inhibition by compounds present in crude DNA preparations (Xin et al. 2003). PVP helps in dissociation of complexes of polysaccharides, phenols and other compounds (Ainsworth, 1994). Inclusion of PVP during grinding the tissue was found to help in recovering higher quantity of RNA (20%) compared to adding PVP in the extraction Buffer (Vasanthaiah, et al. 2008). The PVP solution has been shown to improve PCR in DNA samples extracted from tissues rich in polyphenolics (Koonjul et al., 1999). The addition of PVP to PCR reactions consistently resulted in successful PCR amplification with cotton (Dabo etal. 1993).<br>
In perennial species like trees, several modifications of the existing protocols are made to suite the tissues type as described earlier. Thus biohazardous chemicals like phenol, BME and lithium chloride are commonly used in optimization protocol. Further, the protocols are often time consuming with long hours of precipitations to obtain optimal amount of RNA. In tissues recalcitrant to guanidinium salts like Casuarina, the conventional protocols (including kits) are unsuitable. Further, due to co-precipitation of higher levels of phenolic compounds, the RNA usually requires a clean up procedure or the mRNA requires to be isolated prior to reverse transcription procedure. This increases the cost of reactions during large scale extractions. Hence, a basic RNA isolation protocol using non hazardous cost effective chemicals which is less time consuming with high recovery and suitable for different tissue types for Casuarina and Eucalyptus is described. Further, the process for<br>
 <br>
reverse transcription of the unpurified RNA to cDNA for downstream experiments is also described.<br>
OBJECTS OF INVENTION<br>
An object of the invention is to propose a simple and short protocol for total ribonucleic acid isolation from tissues of Eucalyptus and Casuarina and further conversion to complementary deoxyribonucleic acid without purification.<br>
Other object of this invention is to propose a protocol for ribonucleic acid isolation without using hazardous chemicals.<br>
Another object of this invention is to propose a protocol for ribonucleic acid isolation from tissues recalcitrant to Guanidine salts.<br>
Yet another object of this invention is to propose a protocol for high recovery of total ribonucleic acid from difficult tissues like needles and wood.<br>
Further object of this invention is to propose a protocol for reverse transcription of total ribonucleic acid without isolation of messenger ribonucleic acid.<br>
Yet another object of this invention is to propose a protocol for ribonucleic acid isolation at a significantly reduced cost.<br>
 <br>
STATEMENT OF INVENTION<br>
This invention relates to a short protocol for isolation of undegraded total RNA with high recovery from Eucalyptus tereticomis and Casuarina equisetifolia and cDNA synthesis from unpurified RNA comprising the steps, extracting total RNA from different tissues in extraction buffer including activated charcoal (0.1 to 1%), precipitation of total RNA using 2M - 4M Sodium Chloride instead of Lithium Chloride, final precipitation of total RNA in absolute alcohol without sodium acetate, reverse transcription of unpurified total RNA to complementary DNA using 0.5 -2% PVP and l-2mg/ml BSA in the reaction mixture and generation of single stranded cDNA suitable for PCR reaction.<br>
BRIEF DESCRIPTION OF ACCOMPANYING DRAWINGS<br>
Fig 1. shows undegraded total RNA from tissues of Eucalyptus tereticomis<br>
Fig. 2 shows undegraded total RNA from needle tissues of Casuarina equisetifolia<br>
Fig. 3 demonstrates undegraded total RNA from root tissues of Casuarina equisetifolia<br>
Fig. 4 illustrates undegraded total RNA from wood tissues of Casuarina equisetifolia<br>
 <br>
Fig. 5 illustrates the amplification of cDNA derived from unpurified RNA of Casuarina equisetifolia needles with gene specific primers<br>
Fig. 6 shows the amplification of cDNA derived from unpurified RNA of Casuarina equisetifolia roots with gene specific primer<br>
Fig. 7 shows the amplification of cDNA derived from unpurified RNA of Eucalyptus tereticornis.wood tissues with gene specific primers<br>
Fig. 8 demonstrates the amplification of cDNA derived from unpurified RNA of different tissues with ISSR primer R(CA)7 [GRTRCYGRTRCACACACACACACA]<br>
Table 1: Quality and recovery of undegraded total RNA from different tissue samples<br>
DETAILED DESCRIPTION OF INVENTION<br>
All RNA isolation protocols involve lysing of cell in a chemical environment resulting in denaturation of ribonuclease and fractionation of RNA from other cellular macromolecules. The cell/ tissue type from which RNA is isolated and their resultant usage determine the appropriate protocol for isolation of RNA. A range of methods including conventional and commercial kits are known for RNA isolation from different plant tissues. The procedures include complex series of extraction and precipitation steps which are time consuming and laborious to perform. The relatively large number of steps required increases the risk of RNA degradation, sample loss or cross-<br>
 <br>
contamination of samples when several samples are simultaneously processed. Moreover, the use of materials such as diethyl pyrocarbonate (DEPC) for treating all glasswares used in RNA isolation to inactivate RNase activity, phenol during extraction, Guanidinium salts, a chaotropic denaturant in extraction buffers (including commercial kits) to disrupt/lyse cells and denature protein, p~ Mercaptoethanol in extraction buffer as a reducing agent and Lithium chloride in extraction buffer for selective precipitation of RNA are the common components in most of the protocols. However, most of the above mentioned chemicals are hazardous. Another disadvantage is associated with the use of chaotropes and other similar agents which interfere with the down stream application. Many commercially available RNA isolation kits use mixture of acid phenol and guanidine isothiocyanate extraction technologies. These kits generally involve the use of spin columns to purify the isolated RNA. However, RNA isolation using kits have limited use in tissues with high polyphenolic contents and tissues recalcitrant to Gunidinium salts.<br>
In tree species, due to co precipitation of RNA with polyphenols, most protocols describe the use of RNA clean up kits or mRNA isolation, prior to cDNA synthesis. These additional procedures increase the cost of processing of samples. Protocols devoid of sample processing methods can considerably reduce the cost of experiments. Hence, protocols to convert RNA to cDNA without purification can be a viable option in large scale RNA isolation procedures with reduced cost.<br>
 <br>
With the increasing need for gene expression studies in perennial crops like tree species, the isolation of high quality RNA suitable for downstream applications are a mainstay of tree genomics. In trees, RNA isolation is hindered due to high levels of phenolics, tannins and polysaccharides in the tissues. Further, the need for methods which are quick and simple to perform, devoid of hazardous chemicals and chaotropic agents or use of high concentration of salt and/ or high molecular weight compounds with high viscosity and generating high quantity of un degraded RNA from difficult tissues like wood and needles of tree species is of paramount importance.<br>
The present investigation thus provides a method for isolating undegraded total RNA from needles, root and wood tissues of Casuarina equisetifolia and leaves, root and wood of Eucalyptus tereticornis.<br>
The isolation protocol involves an initial extraction of nucleic acid in preheated extraction buffer followed by a first precipitation using isopropyl alcohol and second with sodium chloride. Finally, the RNA is dissolved in buffer, treated with DNase and processed for reverse transcription to complementary DNA.<br>
The tissues mentioned above were collected and immediately processed for RNA isolation to reduce degradation. The tissues were ground in chilled mortar and pestle using liquid nitrogen to a fine powder. The ground sample was immediately transferred to preheated (at 65°C) extraction buffer containing Tris (200mM), EDTA (25mM), SDS (1%), NaCl (50 - lOOOmM), activated charcoal (0.1 to 1%) and equal volume of chloroform isoamyl alcohol in the ratio of 1:5 (w/v). Five mg of PVP was<br>
 <br>
added while grinding the tissue. The mixture was vortexed thoroughly and centrifuged at 9000 rpm for 10 minutes. The supernatant was subsequently transferred to a fresh tube and chilled isopropyl alcohol was added to equal volume. The contents were mixed by inverting the tube for 5 - 6 times and centrifuged at 12,000 rpm for 10 minutes. The supernatant was discarded and the pellet was dissolved in 200^1 of nuclease free water and to it twice the volume of 2 M NaCl was added. The contents were gently mixed and incubated at -20°C for thirty minutes followed by centrifugation at 12,000 rpm for 10 minutes. The pellet was dissolved in Tris EDTA buffer and thrice the volume absolute ethanol was added to pellet the RNA by centrifugation at 12,000 rpm for 10 minutes. The RNA pellet was further washed with 70% ethanol to remove the remnant salt, air dried and dissolved in Tris EDTA buffer. The sample was briefly centrifuged to pellet the insoluble contaminants and the supernatant was stored at -80°C until further use.<br>
The total RNA was checked for its quality and quantity using agarose gel electrophoresis and by spectrophotometry using Picodrop Microliter UV/Vis Spectrophotometer (Picodrop Ltd., UK).<br>
The recovery of the undegraded total RNA from all tissues are shown in Table 1. The recovery ranged from 4.06 ng/mg fresh stem tissue to 30.19 ng/mg fresh needle tissue in Casuarina equisetifolia while in Eucalyptus tereticornis, the recovery ranged from 1.94 ng/mg fresh wood tissue to 188.68 ng/mg fresh stem tissue. The RNA quality as depicted by the A 260/280 ratio ranged from 1.13 to 2.14 in all tissues. This revealed the co - precipitation of contaminants along with RNA. In general, Reverse<br>
 <br>
Transcriptase reaction is often inhibited by the presence of these contaminants, hence necessitating the process of RNA clean up or isolation of mRNA using commercial kits to convert RNA to cDNA.<br>
Reverse transcription of un purified total RNA: One |ig of total RNA from all tissues were converted to cDNA using the standard protocol containing oligo dT primer, Ribonuclease Inhibitor and M-MuLv Reverse Transcriptase (provided in first strand cDNA synthesis kit, Fermentas, Hanover, MD, USA) for synthesis of first strand cDNA. The RT - PCR reaction was fortified with addition of two components bovine serum albumin (BSA) at concentration of 1 - 2mg/ml and PVP at concentration of 0.5 - 2%.<br>
The cDNA synthesized was further amplified using gene specific primers and arbitrary Inter Simple Sequence Repeat (ISSR) primer in PTC 200 DNA engine (MJ Research Inc., USA) The program used to amplify gene specific primer included an initial denaturation at 94°C for 5 minutes; 30 cycles of 94°C for 1 minutes, annealing at 58°C /60°C for 30 seconds /l minutes and extension at 72°C for 2 minutes with final extension at 72°C for 10 minutes. The program used to amplify ISSR primer included an initial denaturation at 94°C for 3 minutes; 35 cycles of 94°C for 30 seconds, annealing at 50°C for 30 seconds and extension at 72°C for 1 minutes with final extension at 72°C for 10 minutes.<br>
The PCR amplification of single stranded cDNA with the above mentioned primers is illustrated in Figures 5 to Figure 8. The amplification of the cDNA derived from unpurified RNA from all tissues revealed the quenching of the contaminant effect on the polymerase chain reaction by BSA and PVP.<br>
 <br>
Table 1: Quality and recovery of undegraded total RNA from different tissue samples<br>
Sample name	A 260/	280   Recovery	Qssolvedin(µl)	150mg tissue	ng/mgtissue<br>
Casuarina Needle	2.143	226.4 ng/µL	20	4528	30.19<br>
Casuarina stem	1.572	60.9ng/µL	10	609	4.06<br>
Casuarina Root	1.453	64.2 ng/µL	20	1284	8.56<br>
Casuarina wood	1.348	47.6ng/µL	30	1428	9.52<br>
Eucalyptus Leaf	1.135	65.3ng/µl_	30	1959	13.06<br>
Eucalyptusaem	1.144	943.4ng/µl_	30	28302	188.68<br>
Eucalyptus Fbot	1.171	252.7 ng/µL	20	5054	33.69<br>
Eucalyptus Bark	1.197	136.1 ng/µL	20	2722	18.14<br>
Eucalyptuswood(upto1.5cm)	1.245	29.1 ng/µL.	10	291	1.94<br>
Eucalyptus wood (1.5-3.0 cm)	1.216	54.8ng/µL	10	548	3.65<br>
<br>
<br>
We Claim<br>
1. A short protocol for isolation of undegraded total RNA with high recovery from Eucalyptus tereticomis and Casuarina equisetifolia and cDNA synthesis from unpurified RNA comprising the steps: -<br>
i) extracting total RNA from different tissues in extraction buffer including activated charcoal (0.1 to 1%)<br>
ii) precipitation of total RNA using 2M - 4M Sodium Chloride instead of Lithium Chloride<br>
iii) final precipitation of total RNA in absolute alcohol without sodium acetate<br>
iv) reverse transcription of unpurified total RNA to complementary DNA using 0.5 - 2% PVP and l-2mg/ml BSA in the reaction mixture and generation of single stranded cDNA suitable for PCR reaction.<br>
2. A short protocol for isolation of undegraded total RNA with high recovery from Eucalyptus tereticomis and Casuarina equisetifolia and cDNA synthesis from unpurified RNA as claimed in claim 1, wherein no biohazardous chemicals like mercaptoethanol, phenol, DEPC and lithium chloride are used.<br>
 <br>
3.	A short protocol for isolation of undegraded total RNA with high<br>
recovery from Eucalyptus tereticomis and Casuarina equisetifolia<br>
and cDNA synthesis from unpurified RNA as claimed in claim 1,<br>
wherein un degraded RNA can be isolated from tissues recalcitrant<br>
to Guanidine salts.<br>
4.	A short protocol for isolation of undegraded total RNA with high<br>
recovery from Eucalyptus tereticomis and Casuarina equisetifolia<br>
and cDNA synthesis from unpurified RNA as claimed in claim 1,<br>
wherein unpurified total RNA can be converted to cDNA without<br>
mRNA isolation or RNA purification.<br>
5.	A short protocol for isolation of undegraded total RNA with high<br>
recovery from Eucalyptus tereticomis and Casuarina equisetifolia<br>
and cDNA synthesis from unpurified RNA as described and<br>
illustrated herein.<br>
 <br>
</td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/viewdoc.aspx?id=EDaQtCD1+xFAx/eS9TbK5A==&amp;amp;loc=egcICQiyoj82NGgGrC5ChA==" target="_blank" style="word-wrap:break-word;">http://ipindiaonline.gov.in/patentsearch/GrantedSearch/viewdoc.aspx?id=EDaQtCD1+xFAx/eS9TbK5A==&amp;amp;loc=egcICQiyoj82NGgGrC5ChA==</a></p>
		<br>
		<div class="pull-left">
			<a href="272764-process-for-the-generation-of-a-synthesis-gas.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="272766-biological-load-indicator-and-method-of-measuring-biological-load.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>272765</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>1927/CHE/2009</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>18/2016</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>29-Apr-2016</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>26-Apr-2016</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>13-Aug-2009</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>INSTITUTE OF FOREST GENETICS AND TREE BREEDING</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td>P.B. NO.1061, FOREST CAMPUS, R.S.PURAM, COIMBATORE-641002</td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>MODHUMITA DASGUPTA</td>
											<td>INSTITUTE OF FOREST GENETICS AND TREE BREEDING P.B. NO.1061, FOREST CAMPUS, R.S.PURAM, COIMBATORE-641002</td>
										</tr>
										<tr>
											<td>2</td>
											<td>RADHA VELUTHAKKAL</td>
											<td>INSTITUTE OF FOREST GENETICS AND TREE BREEDING P.B. NO.1061, FOREST CAMPUS, R.S.PURAM, COIMBATORE-641002</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>C12N 15/10</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>N/A</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td></td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td></td>
									<td></td>
								    <td>NA</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/272765-a-simple-protocol-for-isolation-of-undegraded-total-rna-from-eucalyptus-and-casuarina-and-cdna-synthesis-from-unpurified-rna by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 01:06:59 GMT -->
</html>
