<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/272763-method-of-producing-transgenic-seeds-with-enhanced-agronomic-traits by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 01:06:22 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 272763:&quot;METHOD OF PRODUCING TRANSGENIC SEEDS WITH ENHANCED AGRONOMIC TRAITS&quot;`</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">&quot;METHOD OF PRODUCING TRANSGENIC SEEDS WITH ENHANCED AGRONOMIC TRAITS&quot;`</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>This disclosure describes screening a population of transgenic plants derived from plant cells transformed with recombinant DNA for expression of proteins with homeobox domains to identify plant cells of specific transgenic events that are useful for imparting enhanced traits to transgenic crop plants. Traits include enhanced nitrogen use efficiency, increased yield, enhanced water use efficiency, enhanced tolerance to cold stress and/or improved seed compositions. Also disclosed are transgenic seeds for growing a transgenic plant having the recombinant DNA in its genome and exhibiting the screened enhance trait. Also disclosed are methods for generating seed and plants based on the transgenic events.</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td>Transgenic Plants With Enhanced Agronomic Traits Cross Reference To Related Applications<br>
This application claims benefit under 35USC ยง 119(e) of United States provisional application Serial No. 60/638,099, filed 12/21/2004, incorporated herein by reference. Incorporation of Sequence Listing<br>
A sequence listing and a computer readable form (CRF) of the sequence listing, on CD-ROM, each containing the text file named "G1543C.ST25.txt", which is 63 KB (measured in MS-WINDOWS) and was created on December 18,2005, are herein incorporated by reference.<br>
Incorporation of Computer Listing<br>
Appended hereto is a Computer Listing on duplicate CD-ROMs containing a folder labeled "hmmer-2.3.2" and two _.HMM files, incorporated herein by reference. Folder hmmer-2.3.2 contains the source code and other associated files for implementing the HMMer software for Pfam analysis. The _.HMM files contains Pfam Hidden Markov Models. The Computer Listings were created on December 18, 2005.<br>
Field Of The Invention<br>
Disclosed herein are inventions in the field of plant genetics and developmental biology. More specifically, the inventions provide plant cells with recombinant DNA for providing an enhanced trait in a transgenic plant, plants comprising such cells, seed and pollen derived from such plants, methods of making and using such cells, plants, seeds and pollen. In particular, the recombinant DNA of the inventions express transcription factors with homeobox domains.<br>
Background Of The Invention<br>
Transgenic plants with improved agronomic traits such as yield, environmental stress tolerance, pest resistance, herbicide tolerance, improved seed compositions, and the like are desired by both farmers and consumers. Although considerable efforts in plant breeding have provided significant gains in desired traits, the ability to introduce specific<br>
DNA into plant genomes provides further opportunities for generation of plants with improved and/or unique traits. Merely introducing recombinant DNA into a plant genome doesn't always produce a transgenic plant with an enhanced agronomic trait. Methods to select individual transgenic events from a population are required to identify those transgenic events that are characterized by the enhanced agronomic trait.<br>
Summary of The Invention<br>
This invention employs recombinant DNA for expression of proteins that are useful for imparting enhanced agronomic traits to the transgenic plants. Recombinant DNA in this invention is provided in a construct comprising a promoter that is functional in plant cells and that is operably linked to DNA that encodes a protein having domains of amino acids in a sequence that exceed the Pfam gathering cutoff for amino acid sequence alignment with a Pfam Homeobox protein domain family and a Pfam HALZ protein domain family. The Pfam gathering cuttoff for the Homeobox protein domain family is -4 and the Pfam gathering cuttoff for the HALZ protein domain family is 17. Other aspects of the invention are specifically directed to transgenic plant cells comprising the recombinant DNA of the invention, transgenic plants comprising a plurality of such plant cells, progeny transgenic seed and transgenic pollen from such plants.   Such plant cells are selected from a population of transgenic plants regenerated from plant cells transformed with recombinant DNA and that express the protein by screening transgenic plants in the population for an enhanced trait as compared to control plants that do not have said recombinant DNA, where the enhanced trait is selected from group of enhanced traits consisting of enhanced water use efficiency, enhanced cold tolerance, increased yield, enhanced nitrogen use efficiency, enhanced seed protein and enhanced seed oil.<br>
In yet another aspect of the invention the plant cells, plants, seeds and pollen further comprise DNA expressing a protein that provides tolerance from exposure to an herbicide applied at levels that are lethal to a wild type of said plant cell. Such tolerance is especially useful not only as an advantageous trait in such plants but is also useful in a selection step in the methods of the invention. In aspects of the invention the agent of such herbicide is a glyphosate, dicamba, or glufosinate compound.<br>
Yet other aspects of the invention provide transgenic plants which are homozygous for the recombinant DNA and transgenic seed of the invention from corn, soybean, cotton, canola, alfalfa, wheat or rice plants, hi other important embodiments for practice of various aspects of the invention in Argentina the recombinant DNA is provided in plant cells derived from corn lines that that are and maintain resistance to the Mai de Rio Cuarto virus or the Puccina sorghi fungus or both.<br>
This invention also provides methods for manufacturing non-natural, transgenic seed that can be used to produce a crop of transgenic plants with an enhanced trait resulting from expression of stably-integrated, recombinant DNA for expressing a protein selected from the group consisting of SEQ ID NO: 5-8. More specifically the method comprises (a) screening a population of plants for an enhanced trait and a recombinant DNA, where individual plants in the population can exhibit the trait at a level less than, essentially the same as or greater than the level that the trait is exhibited in control plants which do not express the recombinant DNA, (b) selecting from the population one or more plants that exhibit the trait at a level greater than the level that said trait is exhibited in control plants, (c) verifying that the recombinant DNA is stably integrated in said selected plants, (d) analyzing tissue of a selected plant to determine the production of a protein having the function of a protein encoded by nucleotides in a sequence of one of SEQ ID NO: 1-4; and (e) collecting seed from a selected plant, hi one aspect of the invention the plants in the population further comprise DNA expressing a protein that provides tolerance to exposure to an herbicide applied at levels that are lethal to wild type plant cells and the selecting is effected by treating the population with the herbicide, e.g. a glyphosate, dicamba, or glufosinate compound. In anotiier aspect of the invention the plants are selected by identifying plants with the enhanced trait. The methods are especially useful for manufacturing corn, soybean, cotton, alfalfa, wheat or rice seed. In a another aspect, the plants further comprise a DNA expressing a second protein that provides plant cells with one or more enhanced agronomic traits.<br>
Another aspect of the invention provides a method of producing hybrid corn seed comprising acquiring hybrid corn seed from a herbicide tolerant corn plant which also has stably-integrated, recombinant DNA comprising a promoter that is (a) functional in plant cells and (b) is operably linked to DNA that encodes a protein selected from the<br>
group consisting of SEQ ID NO: 5-8; wherein a progeny transgenic plant regenerated from a copy of said cell exhibits an enhanced trait as compared to a control plant without said DNA construct; and wherein said cell is selected from a population of cells transformed with said DNA construct by screening progeny plants of cells in said population for an enhanced trait as compared to said control plant, and wherein said enhanced trait is selected from the group consisting of enhanced water use efficiency, enhanced cold tolerance, increased yield, enhanced nitrogen use efficiency, enhanced seed protein and enhanced seed oil resulting from expression of said protein..   The methods further comprise producing corn plants from said hybrid corn seed, wherein a fraction of the plants produced from said hybrid corn seed is homozygous for said recombinant DNA, a fraction of the plants produced from said hybrid corn seed is hemizygous for said recombinant DNA, and a fraction of the plants produced from said hybrid com seed has none of said recombinant DNA; selecting com plants which are homozygous and hemizygous for said recombinant DNA by treating with an herbicide; collecting seed from herbicide-treated-surviving corn plants and planting said seed to produce further progeny corn plants; repeating the selecting and collecting steps at least once to produce an inbred corn line; and crossing the inbred corn line with a second corn line to produce hybrid seed.<br>
Another aspect of the invention provides a method of selecting a plant comprising plant cells of the invention by using an immunoreactive antibody to detect the presence of protein expressed by recombinant DNA in seed or plant tissue. Yet another aspect of the invention provides anti-counterfeit milled seed having, as an indication of origin, a plant cell of this invention.<br>
Still other aspects of this invention relate to transgenic plants with enhanced water use efficiency or enhanced nitrogen use efficiency. For instance, this invention provides methods of growing a corn, cotton or soybean crop without irrigation water comprising planting seed having plant cells of the invention which are selected for enhanced water use efficiency. Alternatively methods comprise applying reduced irrigation water, e.g. providing up to 300 millimeters of ground water during the production of a corn crop. This invention also provides methods of growing a corn, cotton or soybean crop without<br>
added nitrogen fertilizer comprising planting seed having plant cells of the invention which are selected for enhanced nitrogen use efficiency.<br>
Detailed Description of The Invention<br>
As used herein a "plant cell" means a plant cell that is transformed with stably-integrated, non-natural, recombinant DNA, e.g. by Agrobacterium-mediated transformation or by bombardment using microparticles coated with recombinant DNA or other means. A plant cell of this invention can be an originally-transformed plant cell that exists as a microorganism or as a progeny plant cell that is regenerated into differentiated tissue, e.g. into a transgenic plant with stably-integrated, non-natural recombinant DNA, or seed or pollen derived from a progeny transgenic plant.<br>
As used herein a "transgenic plant" means a plant whose genome has been altered by the stable integration of recombinant DNA. A transgenic plant includes a plant regenerated from an originally-transformed plant cell and progeny transgenic plants from later generations or crosses of a transformed plant.<br>
As used herein "recombinant DNA" means DNA which has been a genetically engineered and constructed outside of a cell including DNA containing naturally occurring DNA or cDNA or synthetic DNA.<br>
As used herein "consensus sequence" means an artificial sequence of amino acids in a conserved region of an alignment of amino acid sequences of homologous proteins, e.g. as determined by a CLUSTALW alignment of amino acid sequence of homolog proteins.<br>
As used herein "homolog" means a protein in a group of proteins that perform the same biological function, e.g. proteins that belong to the same Pfam protein family and that provide a common enhanced trait in transgenic plants of this invention. Homologs are expressed by homologous genes. Homologous genes include naturally occurring alleles and artificially-created variants. Degeneracy of the genetic code provides the possibility to substitute at least one base of the protein encoding sequence of a gene with a different base without causing the amino acid sequence of the polypeptide produced from the gene to be changed. Hence, a polynucleotide useful in the present invention may have any base sequence that has been changed from SEQ ID NO: 1 through SEQ ID<br>
NO:4 by substitution in accordance with degeneracy of the genetic code. Homologs are proteins that, when optimally aligned, have at least 60% identity, more preferably about 70% or higher, more preferably at least 80% and even more preferably at least 90% identity over the full length of a protein identified as being associated with imparting an enhanced trait when expressed in plant cells. Homologs include proteins with an amino acid sequence that has at least 90% identity to a consensus amino acid sequence of proteins and homologs disclosed herein.<br>
Homologs are be identified by comparison of amino acid sequence, e.g. manually or by use of a computer-based tool using known homology-based search algorithms such as those commonly known and referred to as BLAST, FASTA, and Smith-Waterman. A local sequence alignment program, e.g. BLAST, can be used to search a database of sequences to find similar sequences, and the summary Expectation value (E-value) used to measure the sequence base similarity. As a protein hit with the best E-value for a particular organism may not necessarily be an ortholog or the only ortholog, a reciprocal query is used in the present invention to filter hit sequences with significant E-values for ortholog identification. The reciprocal query entails search of the significant hits against a database of amino acid sequences from the base organism that are similar to the sequence of the query protein. A hit is a likely ortholog, when the reciprocal query's best hit is the query protein itself or a protein encoded by a duplicated gene after speciation. A further aspect of the invention comprises functional homolog proteins that differ in one or more amino acids from those of disclosed protein as the result of conservative amino acid substitutions, for example substitutions are among: acidic (negatively charged) amino acids such as aspartic acid and glutamic acid; basic (positively charged) amino acids such as arginine, histidine, and lysine; neutral polar amino acids such as glycine, serine, threonine, cysteine, tyrosine, asparagine, and glutamine; neutral nonpolar (hydrophobic) amino acids such as alanine, leucine, isoleucine, valine, proline, phenylalanine, tryptophan, and methionine; amino acids having aliphatic side chains such as glycine, alanine, valine, leucine, and isoleucine; amino acids having aliphatic-hydroxyl side chains such as serine and threonine; amino acids having amide-containing side chains such as asparagine and glutamine; amino acids having aromatic side chains such as phenylalanine, tyrosine, and tryptophan; amino acids having basic side chains such as<br>
lysine, arginine, and histidine; amino acids having sulfur-containing side chains such as cysteine and methionine; naturally conservative amino acids such as valine-leucine, valine-isoleucine, phenylalanine-tyrosine, lysine-arginine, alanine-valine, aspartic acid-glutamic acid, and asparagine-glutamine. A further aspect of the homologs encoded by DNA useful in the transgenic plants of the invention are those proteins that differ from a disclosed protein as the result of deletion or insertion of one or more amino acids in a native sequence.<br>
As used herein, "percent identity" means the extent to which two optimally aligned DNA or protein segments are invariant throughout a window of alignment of components, for example nucleotide sequence or amino acid sequence. An "identity fraction" for aligned segments of a test sequence and a reference sequence is the number of identical components that are shared by sequences of the two aligned segments divided by the total number of sequence components in the reference segment over a window of alignment which is the smaller of the full test sequence or the full reference sequence. "Percent identity" ("% identity") is the identity fraction times 100.<br>
As used herein "Pfam" refers to a large collection of multiple sequence alignments and hidden Markov models covering many common protein families, e.g. Pfam version 18.0 (August 2005) contains alignments and models for 7973 protein families and is based on the Swissprot 47.0 and SP-TrEMBL 30.0 protein sequence databases. See S.R. Eddy, "Profile Hidden Markov Models", Bioinformatics 14:755-763, 1998. Pfam is currently maintained and updated by a Pfam Consortium. The alignments represent some evolutionary conserved structure that has implications for the protein's function. Profile hidden Markov models (profile HMMs) built from the Pfam alignments are useful for automatically recognizing that a new protein belongs to an existing protein family even if the homology by alignment appears to be low. Once one DNA is identified as encoding a protein which imparts an enhanced trait when expressed in transgenic plants, other DNA encoding proteins in the same protein family are identified by querying the amino acid sequence of protein encoded by candidate DNA against the Hidden Markov Model which characterizes the Pfam domain using HMMER software, a current version of which is provided in the appended computer listing. Candidate proteins meeting the gathering cutoff for the alignment of a particular Pfam are in the<br>
protein family and have cognate DNA that is useful in constructing recombinant DNA for the use in the plant cells of this invention. Hidden Markov Model databases for use with HMMER software in identifying DNA expressing protein in a common Pfam for recombinant DNA in the plant cells of this invention are also included in the appended computer listing. The HMMER software and Pfam databases are version 18.0 and were used to determine that the amino acid sequence of SEQ ID NO:5 is characterized by two Pfam domains, i.e. Homeobox domain and HALZ domain. The Homeobox domain was identified as comprising amino acid residues between positions 130 and 193 with a score of 70.1 exceeding the gathering cutoff of-4. The HALZ domain was identified as comprising amino acid residues between positions 194 and 238 with a score of 71.9 exceeding the gathering cutoff of 17.<br>
The HMMER software and databases for identifying the Homeobox and HALZ domains are accessed at any Pfam website and can be provided by the applicant, e.g. in an appended computer listing.<br>
As used herein "promoter" means regulatory DNA for initializing transcription. A "plant promoter" is a promoter capable of initiating transcription in plant cells whether or not its origin is a plant cell, e.g. is it well known that Agrobacterium promoters are functional in plant cells. Thus, plant promoters include promoter DNA obtained from plants, plant viruses and bacteria such as Agrobacterium and Bradyrhizobium bacteria. Examples of promoters under developmental control include promoters that preferentially initiate transcription in certain tissues, such as leaves, roots, or seeds. Such promoters are referred to as "tissue preferred". Promoters that initiate transcription only in certain tissues are referred to as "tissue specific". A "cell type" specific promoter primarily drives expression in certain cell types in one or more organs, for example, vascular cells in roots or leaves. An "inducible" or "repressible" promoter is a promoter which is under environmental control. Examples of environmental conditions that may effect transcription by inducible promoters include anaerobic conditions, or certain chemicals, or the presence of light. Tissue specific, tissue preferred, cell type specific, and inducible promoters constitute the class of "non-constitutive" promoters. A "constitutive" promoter is a promoter which is active under most conditions.<br>
As used herein "operably linked" means the association of two or more DNA fragments in a DNA construct so that the function of one, e.g. protein-encoding DNA, is controlled by the other, e.g. a promoter.<br>
As used herein "expressed" means produced, e.g. a protein is expressed in a plant cell when its cognate DNA is transcribed to mRNA that is translated to the protein.<br>
As used herein a "control plant" means a plant that does not contain the recombinant DNA that expressed a protein that impart an enhanced trait. A control plant is to identify and select a transgenic plant that has an enhance trait. A suitable control plant can be a non-transgenic plant of the parental line used to generate a transgenic plant, i.e. devoid of recombinant DNA. A suitable control plant may in some cases be a progeny of a hemizygous transgenic plant line that is does not contain the recombinant DNA, known as a negative segregant.<br>
As used herein an "enhanced trait" means a characteristic of a transgenic plant that includes, but is not limited to, an enhance agronomic trait characterized by enhanced plant morphology, physiology, growth and development, yield, nutritional enhancement, disease or pest resistance, or environmental or chemical tolerance. In more specific aspects of this invention enhanced trait is selected from group of enhanced traits consisting of enhanced water use efficiency, enhanced cold tolerance, increased yield, enhanced nitrogen use efficiency, enhanced seed protein and enhanced seed oil. In an important aspect of the invention the enhanced trait is enhanced yield including increased yield under non-stress conditions and increased yield under environmental stress conditions. Stress conditions may include, for example, drought, shade, fungal disease, viral disease, bacterial disease, insect infestation, nematode infestation, cold temperature exposure, heat exposure, osmotic stress, reduced nitrogen nutrient availability, reduced phosphorus nutrient availability and high plant density.   "Yield" can be affected by many properties including without limitation, plant height, pod number, pod position on the plant, number of internodes, incidence of pod shatter, grain size, efficiency of nodulation and nitrogen fixation, efficiency of nutrient assimilation, resistance to biotic and abiotic stress, carbon assimilation, plant architecture, resistance to lodging, percent seed germination, seedling vigor, and juvenile traits. Yield can also affected by efficiency of germination (including germination in stressed conditions), growth rate (including growth<br>
rate in stressed conditions), ear number, seed number per ear, seed size, composition of seed (starch, oil, protein) and characteristics of seed fill.<br>
Increased yield of a transgenic plant of the present invention can be measured in a number of ways, including test weight, seed number per plant, seed weight, seed number per unit area (i.e. seeds, or weight of seeds, per acre), bushels per acre (bu/a), tonnes per acre, tons per acre, kilo per hectare. For example, maize yield may be measured as production of shelled corn kernels per unit of production area, for example in bushels per acre or metric tons per hectare, often reported on a moisture adjusted basis, for example at 15.5 percent moisture. Increased yield may result from improved utilization of key biochemical compounds, such as nitrogen, phosphorous and carbohydrate, or from improved responses to environmental stresses, such as cold, heat, drought, salt, and attack by pests or pathogens. Recombinant DNA used in this invention can also be used to provide plants having improved growth and development, and ultimately increased yield, as the result of modified expression of plant growth regulators or modification of cell cycle or photosynthesis pathways. Also of interest is the generation of transgenic plants that demonstrate enhanced yield with respect to a seed component that may or may not correspond to an increase in overall plant yield. Such properties include enhancements in seed oil, seed molecules such as tocopherol, protein and starch, or oil particular oil components as may be manifest by an alteration in the ratios of seed components.<br>
A subset of the nucleic molecules of this invention includes fragments of the disclosed recombinant DNA consisting of oligonucleotides of at least 15, preferably at least 16 or 17, more preferably at least 18 or 19, and even more preferably at least 20 or more, consecutive nucleotides. Such oligonucleotides are fragments of the larger molecules having a sequence selected from the group consisting of SEQ ID NO:1 through SEQ ID NO:4, and find use, for example as probes and primers for detection of the polynucleotides of the present invention.<br>
DNA constructs are assembled using methods well known to persons of ordinary skill in the art and typically comprise a promoter operably linked to DNA, the expression of which provides the enhanced agronomic trait. Other construct components may include additional regulatory elements, such as 5' leaders and introns for enhancing<br>
transcription, 3' untranslated regions (such as polyadenylation signals and sites), DNA for transit or signal peptides.<br>
Numerous promoters that are active in plant cells have been described in the literature. These include promoters present in plant genomes as well as promoters from other sources, including nopaline synthase (NOS) promoter and octopine synthase (OCS) promoters carried on tumor-inducing plasmids of Agrobacteriwn tumefaciens, caulimovirus promoters such as the cauliflower mosaic virus. For instance, see U.S. Patents No. 5,858,742 and 5,322,938, which disclose versions of the constitutive promoter derived from cauliflower mosaic virus (CaMV35S), U.S. Patent 5,641,876, which discloses a rice actin promoter, U.S. Patent Application Publication 2002/0192813Al, which discloses 5', 3' and intron elements useful in the design of effective plant expression vectors, U.S. patent application Serial No. 09/757,089, which discloses a maize chloroplast aldolase promoter, U.S. patent application Serial No. 08/706,946, which discloses a rice glutelin promoter, U.S. patent application Serial No.09/757,089, which discloses a maize aldolase (FDA) promoter, and U.S. patent application Serial No.60/310, 370, which discloses a maize nicotianamine synthase promoter, all of which are incorporated herein by reference. These and numerous other promoters that function in plant cells are known to those skilled in the art and available for use in recombinant polynucleotides of the present invention to provide for expression of desired genes in transgenic plant cells.<br>
In other aspects of the invention, preferential expression in plant green tissues is desired. Promoters of interest for such uses include those from genes such as Arabidopsis thalianaribulose-l,5-bisphosphate carboxylase (Rubisco) small subunit (Fischhoff et al (1992) Plant Mol Biol. 20:81-93), aldolase and pyruvate orthophosphate dikinase (PPDK) (Taniguchi et al. (2000) Plant Cell Physiol. 41(l):42-48).<br>
Furthermore, the promoters may be altered to contain multiple "enhancer sequences" to assist in elevating gene expression. Such enhancers are known in the art. By including an enhancer sequence with such constructs, the expression of the selected protein may be enhanced. These enhancers often are found 5' to the start of transcription in a promoter that functions in eukaryotic cells, but can often be inserted upstream (5') or downstream (31) to the coding sequence. In some instances, these 5' enhancing elements<br>
are introns. Particularly useful as enhancers are the 5' introns of the rice actin 1 (see US Patent 5,641,876)and rice actin 2 genes, the maize alcohol dehydrogenase gene intron, the maize heat shock protein 70 gene intron (U.S. Patent 5,593,874) and the maize shrunken 1 gene.<br>
In other aspects of the invention, sufficient expression in plant seed tissues is desired to effect improvements in seed composition. Exemplary promoters for use for seed composition modification include promoters from seed genes such as napin (U.S. 5,420,034), maize L3 oleosin (U.S. 6,433,252), zein Z27 (Russell et al (1997) Transgenic Res. 6(2): 157-166), globulin 1 (Belanger etal (1991) Genetics 129:863-872), glutelin 1 (Russell (1997) supra), and peroxiredoxin antioxidant (Perl) (Stacy et al. (1996) Plant Mol Biol. 31(6):1205-1216).<br>
Recombinant DNA constructs prepared in accordance with the invention will also generally include a 3' element that typically contains a polyadenylation signal and site. Well-known 3' elements include those from Agrobacterium tumefaciens genes such as nos 3', tml 3', tmr3', tms 3', ocs 3\ tr7 3', for example disclosed in U.S. 6,090,627, incorporated herein by reference; 3' elements from plant genes such as wheat (Triticum aesevitwri) heat shock protein 17 (Hspl? 3')&gt; a wheat ubiquitin gene, a wheat fructose-1,6-biphosphatase gene, a rice glutelin gene a rice lactate dehydrogenase gene and a rice beta-tubulin gene, all of which are disclosed in U.S. published patent application 2002/0192813 Al, incorporated herein by reference; and the pea (Pisum sativum) ribulose biphosphate carboxylase gene (rbs 3'), and 3' elements from the genes within the host plant.<br>
Constructs and vectors may also include a transit peptide for targeting of a gene target to a plant organelle, particularly to a chloroplast, leucoplast or other plastid organelle. For descriptions of the use of chloroplast transit peptides see U.S. Patent 5, 188,642 and U.S. Patent No. 5,728,925, incorporated herein by reference. For description of the transit peptide region of an Arabidopsis EPSPS gene useful in the present invention, see Klee, H.J. et al (MGG (1987) 210:437-442).<br>
Transgenic plants comprising or derived from plant cells of this invention transformed with recombinant DNA can be further enhanced with stacked traits, e.g. a crop plant having an enhanced trait resulting from expression of DNA disclosed herein in<br>
combination with herbicide and/or pest resistance traits. For example, genes of the current invention can be stacked with other traits of agronomic interest, such as a trait providing herbicide resistance, or insect resistance, such as using a gene from Bacillus thuringensis to provide resistance against lepidopteran, coliopteran, homopteran, hemiopteran, and other insects. Herbicides for which transgenic plant tolerance has been demonstrated and the method of the present invention can be applied include, but are not limited to, glyphosate, dicamba, glufosinate, sulfonylurea, bromoxynil and norflurazon herbicides. Polynucleotide molecules encoding proteins involved in herbicide tolerance are well-known in the art and include, but are not limited to, a polynucleotide molecule encoding 5-enolpyruvylshikimate-3-phosphate synthase (EPSPS) disclosed hi U.S. Patent 5,094,945; 5,627,061; 5,633,435 and 6,040,497 for imparting glyphosate tolerance; polynucleotide molecules encoding a glyphosate oxidoreductase (GOX) disclosed in U.S. Patent 5,463,175 and a glyphosate-N-acetyl transferase (GAT) disclosed in U.S. Patent Application publication 2003/0083480 Al also for imparting glyphosate tolerance; dicamba monooxygenase disclosed in U.S. Patent Application publication 2003/0135879 Al for imparting dicamba tolerance; a polynucleotide molecule encoding bromoxynil nitrilase (Bxn) disclosed in U.S. Patent 4,810,648 for imparting bromoxynil tolerance; a polynucleotide molecule encoding phytoene desaturase (crtl) described in Misawa et al, (1993) Plant J. 4:833-840 and Misawa et al, (1994) Plant J. 6:481-489 for norflurazon tolerance; a polynucleotide molecule encoding acetohydroxyacid synthase (AHAS, aka ALS) described in Sathasiivan et al. (1990) Nucl. Acids Res. 18:2188-2193 for imparting tolerance to sulfonylurea herbicides; polynucleotide molecules known as bar genes disclosed in DeBlock, et al. (1987) EMBO J. 6:2513-2519 for imparting glufosinate and bialaphos tolerance; polynucleotide molecules disclosed in U.S. Patent Application Publication 2003/010609 Al for imparting N-amino methyl phosphonic acid tolerance; polynucleotide molecules disclosed in U.S. Patent 6,107,549 for impartinig pyridine herbicide resistance; molecules and methods for imparting tolerance to multiple herbicides such as glyphosate, atrazine, ALS inhibitors, isoxoflutole and glufosinate herbicides are disclosed in U.S. Patent 6,376,754 and U.S. Patent Application Publication 2002/0112260, all of said U.S. Patents and Patent Application Publications are incorporated herein by reference. Molecules and methods for imparting<br>
insect/nematode/virus resistance are disclosed in U.S. Patents 5,250,515; 5,880,275; 6,506,599; 5,986,175 and U.S. Patent Application Publication 2003/0150017 Al, all of which are incorporated herein by reference.<br>
In particular embodiments, the inventors contemplate the use of antibodies, either monoclonal or polyclonal which bind to the proteins disclosed herein. Means for preparing and characterizing antibodies are well known in the art (See, e.g., Antibodies: A Laboratory Manual, Cold Spring Harbor Laboratory, 1988; incorporated herein by reference). The methods for generating monoclonal antibodies (mAbs) generally begin along the same lines as those for preparing polyclonal antibodies. Briefly, a polyclonal antibody is prepared by immunizing an animal with an immunogenic composition in accordance with the present invention and collecting antisera from that immunized animal. A wide range of animal species can be used for the production of antisera. Typically the animal used for production of anti-antisera is a rabbit, a mouse, a rat, a hamster, a guinea pig or a goat. Because of the relatively large blood volume of rabbits, a rabbit is a preferred choice for production of polyclonal antibodies.<br>
As is well known in the art, a given composition may vary in its immunogenicity. It is often necessary therefore to boost the host immune system, as may be achieved by coupling a peptide or polypeptide immunogen to a carrier. Exemplary and preferred carriers are keyhole limpet hemocyanin (KLH) and bovine serum albumin (BSA). Other albumins such as ovalbumin, mouse serum albumin or rabbit serum albumin can also be used as carriers. Means for conjugating a polypeptide to a carrier protein are well known in the art and include using glutaraldehyde, m-maleimidobencoyl-N-hydroxysuccinimide ester, carbodiimide and bis-biazotized benzidine.<br>
As is also well known in the art, the immunogenicity of a particular immunogen composition can be enhanced by the use of non-specific stimulators of the immune response, known as adjuvants. Exemplary and preferred adjuvants include complete Freund's adjuvant (a non-specific stimulator of the immune response containing killed Mycobacterium tuberculosis), incomplete Freund's adjuvants and aluminum hydroxide adjuvant.<br>
The amount of immunogen composition used in the production of polyclonal antibodies varies upon the nature of the immunogen as well as the animal used for<br>
immunization. A variety of routes can be used to administer the immunogen (subcutaneous, intramuscular, intradermal, intravenous and intraperitoneal). The production of polyclonal antibodies may be monitored by sampling blood of the immunized animal at various points following immunization. A second, booster, injection may also be given. The process of boosting and titering is repeated until a suitable titer is achieved. When a desired level of immunogenicity is obtained, the immunized animal can be bled and the serum isolated and stored, and/or the animal can be used to generate mAbs.<br>
mAbs may be readily prepared through use of well-known techniques, such as those exemplified in U.S. Pat. No. 4,196,265, incorporated herein by reference. Typically, this technique involves immunizing a suitable animal with a selected immunogen composition, e.g., a purified or partially purified antifungal protein, polypeptide or peptide. The immunizing composition is administered in a manner effective to stimulate antibody producing cells. Rodents such as mice and rats are preferred animals, however, the use of rabbit, sheep, or frog cells is also possible. The use of rats may provide certain advantages (Coding, 1986, pp. 60-61), but mice are preferred, with the BALB/c mouse being most preferred as this is most routinely used and generally gives a higher percentage of stable fusions.<br>
Following immunization, somatic cells with the potential for producing antibodies, specifically B lymphocytes (B cells), are selected for use in the mAb generating protocol. These cells may be obtained from biopsied spleens, tonsils or lymph nodes, or from a peripheral blood sample. Spleen cells and peripheral blood cells are preferred, the former because they are a rich source of antibody-producing cells that are in the dividing plasmablast stage, and the latter because peripheral blood is easily accessible. Often, a panel of animals will have been immunized and the spleen of animal with the highest antibody titer will be removed and the spleen lymphocytes obtained by homogenizing the spleen with a syringe. Typically, a spleen from an immunized mouse contains approximately 5xl07 to 2xl08lymphocytes.<br>
The antibody-producing B lymphocytes from the immunized, animal are then fused with cells of an immortal myeloma cell, generally one of the same species as the animal that was immunized. Myeloma cell lines suited for use in hybridoma-producing<br>
38-21(53720)C<br>
fusion procedures preferably are non-antibody-producing, have high fusion efficiency, and enzyme deficiencies that render them incapable of growing in certain selective media which support the growth of only the desired fused cells (hybridomas).<br>
Any one of a number of myeloma cells may be used, as are known to those of skill in the art (Coding, 1986, pp. 65-66; Campbell, 1984, pp. 75-83). For example, where the immunized animal is a mouse, one may use P3-X63/Ag8, X63-Ag8.653, NSl/l.Ag 4 1, Sp210-Agl4, FO, NSO/U, MPC-11, MPC11-X45-GTG 1.7 and S194/5XXO Bui; for rats, one may use R210.RCY3, Y3-Ag 1.2.3, IR983F and 4B210; and U-266, GM1500-GRG2, LICR-LON-HMy2 and UC729-6 are all useful in connection with human cell fusions.<br>
One preferred murine myeloma cell is the NS-1 myeloma cell line (also termed P3-NS-l-Ag4-l), which is readily available from the NIGMS Human Genetic Mutant Cell Repository by requesting cell line repository number GM3573. Another mouse myeloma cell line that may be used is the 8-azaguanine-resistant mouse murine myeloma SP2/0 non-producer cell line.<br>
Methods for generating hybrids of antibody-producing spleen or lymph node cells and myeloma cells usually comprise mixing somatic cells with myeloma cells in a 2:1 ratio, though the ratio may vary from about 20:1 to about 1:1, respectively, in the presence of an agent or agents (chemical or electrical) that promote the fusion of cell membranes. Fusion methods using Spend virus have been described (Kohler and Milstein, 1975; 1976), and those using polyethylene glycol (PEG), such as 37% (v/v) PEG, (Gefter et al., 1977). The use of electrically induced fusion methods is also appropriate (Coding, 1986, pp. 71-74).<br>
Fusion procedures usually produce viable hybrids at low frequencies, about 1x10" 6 to IxlO"8. However, this does not pose a problem, as the viable, fused hybrids are differentiated from the parental, unfused cells (particularly the unfused myeloma cells that would normally continue to divide indefinitely) by culturing in a selective medium. The selective medium is generally one that contains an agent that blocks the de novo synthesis of nucleotides in the tissue culture media. Exemplary and preferred agents are aminopterin, methotrexate, and azaserine. Aminopterin and methotrexate block de novo synthesis of both purines and pyrimidines, whereas azasenne blocks only purine<br>
synthesis. Where aminopterin or methotrexate is used, the media is supplemented with hypoxanthine and thymidine as a source of nucleotides (HAT medium). Where azaserine is used, the media is supplemented with hypoxanthine.<br>
The preferred selection medium is HAT. Only cells capable of operating nucleotide salvage pathways are able to survive in HAT medium. The myeloma cells are defective in key enzymes of the salvage pathway, e.g., hypoxanthine phosphoribosyl transferase (HPRT), and they cannot survive. The B-cells can operate this pathway, but they have a limited life span in culture and generally die within about two weeks. Therefore, the only cells that can survive in the selective media are those hybrids formed from myeloma and B-cells.<br>
This culturing provides a population of hybridomas from which specific hybridomas are selected. Typically, selection of hybridomas is performed by culturing the cells by single-clone dilution in microtiter plates, followed by testing the individual clonal supernatants (after about two to three weeks) for the desired reactivity. The assay should be sensitive, simple and rapid, such as radioimmunoassays, enzyme immunoassays, cytotoxicity assays, plaque assays, dot immunobinding assays, and the like.<br>
The selected hybridomas would then be serially diluted and cloned into individual antibody-producing cell lines, which clones can then be propagated indefinitely to provide mAbs. The cell lines may be exploited for mAb production in two basic ways. A sample of the hybridoma can be injected (often into the peritoneal cavity) into a histocompatible animal of the type that was used to provide the somatic and myeloma cells for the original fusion. The injected animal develops tumors secreting the specific monoclonal antibody produced by the fused cell hybrid. The body fluids of the animal, such as serum or ascites fluid, can then be tapped to provide mAbs in high concentration. The individual cell lines could also be cultured in vitro, where the mAbs are naturally secreted into the culture medium from which they can be readily obtained in high concentrations. mAbs produced by either means may be further purified, if desired, using filtration, centrifugation and various chromatographic methods such as HPLC or affinity chromatography. Plant Cell Transformation Methods<br>
Numerous methods for transforming plant cells with recombinant DNA are known in the art and may be used in the present invention. Two commonly used methods for plant transformation are Agrobacterium-mediated transformation and microprojectile bombardment. Microprojectile bombardment methods are illustrated in U.S. Patents 5,015,580 (soybean); 5,550,318 (corn); 5,538,880 (corn); 5,914,451 (soybean); 6,160,208 (com); 6,399,861 (corn) and 6,153,812 (wheat) and Agrobacterium-medialed transformation is described in U.S. Patents 5,159,135 (cotton); 5,824,877 (soybean); 5,591,616 (corn); and 6,384,301 (soybean), all of which are incorporated herein by reference. For Agrobacterium tumefaciens based plant transformation system, additional elements present on transformation constructs will include T-DNA left and right border sequences to facilitate incorporation of the recombinant polynucleotide into the plant genome.<br>
In general it is useful to introduce recombinant DNA randomly, i.e. at a nonspecific location, in the genome of a target plant line. In special cases it may be useful to target recombinant DNA insertion in order to achieve site-specific integration, for example to replace an existing gene in the genome, to use an existing promoter in the plant genome, or to insert a recombinant polynucleotide at a predetermined site known to be active for gene expression. Several site specific recombination systems exist which are known to function implants include cre-lox as disclosed in U.S. Patent 4,959,317 and FLP-FRT as disclosed in U.S. Patent 5,527,695, both incorporated herein by reference.<br>
Transformation methods of this invention are preferably practiced in tissue culture on media and in a controlled environment. "Media" refers to the numerous nutrient mixtures that are used to grow cells in vitro, that is, outside of the intact living organism. Recipient cell targets include, but are not limited to, meristem cells, callus, immature embryos and gametic cells such as microspores, pollen, sperm and egg cells. It is contemplated that any cell from which a fertile plant may be regenerated is useful as a recipient cell. Callus may be initiated from tissue sources including, but not limited to, immature embryos, seedling apical meristems, microspores and the like. Cells capable of proliferating as callus are also recipient cells for genetic transformation. Practical transformation methods and materials for making transgenic plants of mis invention, for example various media and recipient target cells, transformation of immature embryo<br>
cells and subsequent regeneration of fertile transgenic plants are disclosed in U.S. Patents 6,194,636 and 6,232,526, which are incorporated herein by reference.<br>
The seeds of transgenic plants can be harvested from fertile transgenic plants and be used to grow progeny generations of transformed plants of this invention including hybrid plants line for selection of plants having an enhanced trait. In addition to direct transformation of a plant with a recombinant DNA, transgenic plants can be prepared by crossing a first plant having a recombinant DNA with a second plant lacking the DNA. For example, recombinant DNA can be introduced into first plant line that is amenable to transformation to produce a transgenic plant which can be crossed with a second plant line to introgress the recombinant DNA into the second plant line. A transgenic plant with recombinant DNA providing an enhanced trait, e.g. enhanced yield, can be crossed with transgenic plant line having other recombinant DNA that confers another trait, for example herbicide resistance or pest resistance, to produce progeny plants having recombinant DNA that confers both traits. Typically, in such breeding for combining traits the transgenic plant donating the additional trait is a male line and the transgenic plant carrying the base traits is the female line. The progeny of this cross will segregate such that some of the plants will carry the DNA for both parental traits and some will carry DNA for one parental trait; such plants can be identified by markers associated with parental recombinant DNA, e.g. marker identification by analysis for recombinant DNA or, in the case where a selectable marker is linked to the recombinant, by application of the selecting agent such as a herbicide for use with a herbicide tolerance marker, or by selection for the enhanced trait. Progeny plants carrying DNA for both parental traits can be crossed back into the female parent line multiple times, for example usually 6 to 8 generations, to produce a progeny plant with substantially the same genotype as one original transgenic parental line but for the recombinant DNA of the other transgenic parental line<br>
In the practice of transformation DNA is typically introduced into only a small percentage of target plant cells in any one transformation experiment. Marker genes are used to provide an efficient system for identification of those cells that are stably transformed by receiving and integrating a transgenic DNA construct into their genomes. Preferred marker genes provide selective markers which confer resistance to a selective<br>
agent, such as an antibiotic or herbicide. Any of the herbicides to which plants of this invention may be resistant are useful agents for selective markers. Potentially transformed cells are exposed to the selective agent. In the population of surviving cells will be those cells where, generally, the resistance-conferring gene is integrated and expressed at sufficient levels to permit cell survival. Cells may be tested further to confirm stable integration of the exogenous DNA. Commonly used selective marker genes include those conferring resistance to antibiotics such as kanamycin and paromomycin (nptll), hygromycin B (aph IV) and gentamycin (aac3 and aacC4) or resistance to herbicides such as glufosinate (bar or pat) and glyphosate (aroA or EPSPS). Examples of such selectable are illustrated in U.S. Patents 5,550,318; 5,633,435; 5,780,708 and 6,118,047, all of which are incorporated herein by reference. Selectable markers which provide an ability to visually identify transformants can also be employed, for example, a gene expressing a colored or fluorescent protein such as a luciferase or green fluorescent protein (GFP) or a gene expressing a feera-glucuronidase or uidA gene (GUS) for which various chromogenic substrates are known.<br>
Plant cells that survive exposure to the selective agent, or plant cells that have been scored positive in a screening assay, may be cultured in regeneration media and allowed to mature into plants. Developing plantlets regenerated from transformed plant cells can be transferred to plant growth mix, and hardened off, for example, in an environmentally controlled chamber at about 85% relative humidity, 600 ppm CCยป2, and 25-250 microeinsteins m"2 s"1 of light, prior to transfer to a greenhouse or growth chamber for maturation. Plants are regenerated from about 6 weeks to 10 months after a transformant is identified, depending on the initial tissue. Plants may be pollinated using conventional plant breeding methods known to those of skill in the art and seed produced, for example self-pollination is commonly used with transgenic corn. The regenerated transformed plant or its progeny seed or plants can be tested for expression of the recombinant DNA and selected for the presence of enhanced agronomic trait. Transgenic Plants and Seeds<br>
Transgenic plants derived from the plant cells of this invention are grown to generate transgenic plants having an enhanced trait as compared to a control plant and produce transgenic seed and haploid pollen of this invention. Such plants with enhanced<br>
traits are identified by selection of transformed plants or progeny seed for the enhanced trait. For efficiency a selection method is designed to evaluate multiple transgenic plants (events) comprising the recombinant DNA, for example multiple plants from 2 to 20 or more transgenic events. Transgenic plants grown from transgenic seed provided herein demonstrate improved agronomic traits that contribute to increased yield or other trait that provides increased plant value, including, for example, improved seed quality. Of particular interest are plants having enhanced water use efficiency, enhanced cold tolerance, increased yield, enhanced nitrogen use efficiency, enhanced seed protein and enhanced seed oil.<br>
Table 1 provides a list of protein encoding DNA ("genes") that are useful as recombinant DNA for production of transgenic plants with enhanced agronomic trait, the elements of Table 1 are described by reference to:<br>
"PEP SEQ" which identifies an amino acid sequence from SEQ ID NO:5-8. "NUC SEQ" which identifies a DNA sequence from SEQ ID NO:1- 4. "Base Vector" which identifies a base plasmid used for transformation of the recombinant DNA.<br>
"PROTEIN NAME" which is a common name for protein encoded by the recombinant DNA.<br>
"Plasmid ID" which identifies an arbitrary name for the plant transformation plasmid comprising recombinant DNA for expressing the recombinant DNA in plant cells.<br>
Table 1<br>
<br>
(Table Removed)<br>
Screening methods for transgenic plants with enhanced agronomic trait<br>
Many transgenic events which survive to fertile transgenic plants that produce seeds and progeny plants will not exhibit an enhanced agronomic trait. Screening is necessary to identify the transgenic plant of this invention. Transgenic plants having enhanced agronomic traits are identified from populations of plants transformed as described herein by evaluating the trait in a variety of assays to detect an enhanced agronomic trait. These assays also may take many forms, including but not limited to, analyses to detect changes in the chemical composition, biomass, physiological properties, morphology of the plant. Changes in chemical compositions such as nutritional composition of grain can be detected by analysis of the seed composition and content of protein, free amino acids, oil, free fatty acids, starch or tocopherols. Changes in biomass characteristics can be made on greenhouse or field grown plants and can include plant height, stem diameter, root and shoot dry weights; and, for corn plants, ear length and diameter. Changes in physiological properties can be identified by evaluating responses to stress conditions, e.g., assays using imposed stress conditions such as water deficit, nitrogen deficiency, cold growing conditions, pathogen or insect attack or light deficiency, or increased plant density. Changes in morphology can be measured by visual observation of tendency of a<br>
transformed plant with an enhanced agronomic trait to also appear to be a normal plant as compared to changes toward bushy, taller, thicker, narrower leaves, striped leaves, knotted trait, chlorosis, albino, anthocyanin production, or altered tassels, ears or roots. Other screening properties include days to pollen shed, days to silking, leaf extension rate, chlorophyll content, leaf temperature, stand, seedling vigor, intemode length, plant height, leaf number, leaf area, tillering, brace roots, stay green, stalk lodging, root lodging, plant health, barreness/prolificacy, green snap, and pest resistance. In addition, phenotypic characteristics of harvested grain may be evaluated, including number of kernels per row on the ear, number of rows of kernels on the ear, kernel abortion, kernel weight, kernel size, kernel density and physical grain quality. Although preferred seeds for transgenic plants with enhanced agronomic traits of this invention are corn and soybean plants, other seeds are for cotton, canola, wheat, sunflower, sorghum, alfalfa, barley, millet, rice, tobacco, fruit and vegetable crops, and turfgrass.<br>
Screening for Enhanced Nitrogen Use Efficiency<br>
One preferred enhanced agronomic trait in transgenic plants of this invention is enhanced nitrogen use efficiency as compared to control plants. Higher nitrogen soil applications increase seed protein and starch accumulation, and lead to larger seed weight and larger kernel number per ear. Recent improvements in elite high yielding corn hybrid genotypes include the ability to utilize nitrogen efficiently. Genes causing the enhanced nitrogen use efficiency in crop plants are especially useful, e.g., for improving yield. Enhanced nitrogen use efficiency can be assessed by measuring changes in plant growth such as leaf area production, shoot biomass, chlorophyll content in plants grown in nitrogen limiting conditions and/or nitrogen sufficient conditions. It is useful to conduct a first screen in nitrogen limiting conditions and confirm replicate transgenic events in both nitrogen limiting and nitrogen sufficient conditions. Table 2 shows the amount of nutrients in the nutrient solution for nitrogen limiting conditions (low nitrogen growth condition) and nitrogen sufficient conditions (high nitrogen growth condition) useful for nitrogen use efficiency screening. For example in a greenhouse screen pots of transgenic plants and control plants are treated with 100 ml of nutrient solution three times a week<br>
on alternate days starting at 8 and 10 days after planting for high nitrogen and low nitrogen screening, respectively.<br>
Table 2<br>
(Table Removed)<br>
Note: Adjust pH to 5.6 with HC1 or KOH<br>
After 28 days of plant growth for low nitrogen screening and 23 days for high nitrogen screening, measurements are taken for: total shoot fresh mass, leaf chlorophyll, leaf area leaf fresh mass and leaf dry mass. Screening for Increased Yield<br>
Many transgenic plants of this invention exhibit enhanced yield as compared to a control plant. Enhanced yield can result from enhanced seed sink potential, i.e. the number and size of endosperm cells or kernels and/or enhanced sink strength, i.e. the rate of starch biosynthesis. Sink potential can be established very early during kernel development, as endosperm cell number and cell size are determined within the first few days after pollination.<br>
Much of the increase in corn yield of the past several decades has resulted from an increase in planting density. During that period, com yield has been increasing at a rate of 2.1 bushels/acre/year, but the planting density has increased at a rate of 250 plants/acre/year. A characteristic of modern hybrid com is the ability of these varieties to be planted at high density. Many studies have shown that a higher than current planting density should result in more biomass production, but current germplasm does not<br>
perform well at these higher densities. One approach to increasing yield is to increase harvest index (HI), the proportion of biomass that is allocated to the kernel compared to total biomass, in high density plantings.<br>
Effective yield screening of trans genie corn uses hybrid progeny of the transgenic event over multiple locations with plants grown under optimal production management practices, and maximum pest control. A useful target for enhanced yield is a 5% to 10% increase in yield as compared to yield produced by plants grown from seed for a control plant. Useful screening in multiple and diverse geographic locations, e.g., up to 16 or more locations, over one or more plating seasons, e.g., at least two planting seasons to statistically distinguish yield improvement from natural environmental effects. It is to plant multiple transgenic plants, positive and negative control plants, and pollinator plants in standard plots, e.g., 2 row plots, 20 feet long by 5 feet wide with 30 inches distance between rows and a 3 foot alley between ranges. Transgenic events can be grouped by recombinant DNA constructs with groups randomly placed in the field. A pollinator plot of a high quality corn line is planted for every two plots to allow open pollination when using male sterile transgenic events. A useful planting density is about 30,000 plants/acre.<br>
Surrogate indicators for screening for yield improvement include source capacity (biomass), source output (sucrose and photosynthesis), sink components (kernel size, ear size, starch in the seed), development (light response, height, density tolerance), maturity early flowering trait and physiological responses to high density planting, e.g., at 45,000 plants per acre, e.g., as illustrated in Table 3 and 4.<br>
 <br>
Table 3<br>
<br>
(Table Removed)<br>
When screening for yield improvement a useful statistical measurement approach comprises three components, i.e. modeling spatial autocorrelation of the test field separately for each location, adjusting traits of recombinant DNA events for spatial dependence for each location, and conducting an across location analysis. The first step in modeling spatial autocorrelation is estimating the covariance parameters of the semivariograni. A spherical covariance model is assumed to model the spatial autocorrelation. Because of the size and nature of the trial, it is likely that the spatial autocorrelation may change. Therefore, anisotropy is also assumed along with spherical covariance structure. The following set of equations describes the statistical form of the anisotropic spherical covariance model.<br>
(Equaition Removed)<br>
 <br>
where is the indicator function<br>
(Equaition Removed)<br>
where Si = (x), yO are the spatial coordinates of one location and are the spatial coordinates of the second location. There are 5 covariance parameters,<br>
where vis the nugget effect, o2 is the partial sill, pis a rotation in degrees clockwise from north, 6&gt;n is a scaling parameter for the minor axis and o)j is a scaling parameter for the major axis of an anisotropical ellipse of equal covariance. The five covariance parameters that define the spatial trend will then be estimated by using data from heavily replicated pollinator plots via restricted maximum likelihood approach. In a multi-location field trial, spatial trend are modeled separately for each location.<br>
After obtaining the variance parameters of the model, a variance-covariance structure is generated for the data set to be analyzed. This variance-covariance structure<br>
contains spatial information required to adjust yield data for spatial dependence. In this case, a nested model that best represents the treatment and experimental design of the study is used along with the variance-covariance structure to adjust the yield data. During this process the nursery or the seed batch effects can also be modeled and estimated to adjust the yields for any yield parity caused by seed batch differences.<br>
After spatially adjusted data from different locations are generated, all adjusted data is combined and analyzed assuming locations as replications. In this analysis, intra and inter-location variances are combined to estimate the standard error of yield from transgenic plants and control plants. Relative mean comparisons are used to indicate statistically significant yield improvements. Screening for Water Use Efficiency<br>
An aspect of this invention provides transgenic plants with enhanced yield resulting from enhanced water use efficiency and/or drought tolerance. Described in this example is a high-throughput method for greenhouse selection of transgenic corn plants to wild type corn plants (tested as inbreds or hybrids) for water use efficiency. This selection process imposes 3 drought/re-water cycles on plants over a total period of 15 days after an initial stress free growth period of 11 days. Each cycle consists of 5 days, with no water being applied for the first four days and a water quenching on the 5th day of the cycle. The primary phenotypes analyzed by the selection method are the changes ii plant growth rate as determined by height and biomass during a vegetative drought treatment. The hydration status of the shoot tissues following the drought is also measured. The plant heights are measured at three time points. The first is taken just prior to the onset drought when the plant is 11 days old, which is the shoot initial height (SIH). The plant height is also measured halfway throughout the drought/re-water regimen, on day 18 after planting, to give rise to the shoot mid-drought height (SMH). Upon the completion of the final drought cycle on day 26 after planting, the shoot portion of the plant is harvested and measured for a final height, which is the shoot wilt height (SWH) and also measured for shoot wilted biomass (SWM). The shoot is placed in water at 40 degree Celsius in the dark. Three days later, the shoot is weighted to give rise to the shoot turgid weight (STM). After drying in an oven for four days, the shoots are weighted for shoot dry biomass (SDM). The shoot average height (SAH) is the mean<br>
plant height across the 3 height measurements. The procedure described above may be adjusted for +/- ~ one day for each step given the situation.<br>
To correct for slight differences between plants, a size corrected growth value is derived from SIH and SWH. This is the Relative Growth Rate (RGR). Relative Growth Rate (RGR) is calculated for each shoot using the formula   [RGR% = (SWH-SIH)/((SWH+SIH)/2)* 100].   Relative water content (RWC) is a measurement of how much (%) of the plant was water at harvest. Water Content (RWC) is calculated for each shoot using the formula [RWC% = (SWM-SDM)/(STM-SDM)*100j. Fully watered corn plants of this age run around 98% RWC. Screening for Growth Under Cold Stress<br>
An aspect of this invention provides transgenic plants with enhanced growth under cold stress, e.g., in an early seedling growth assay. In an early seedling growth assay 3 sets of seeds are assayed. The first set is a group of transgenic seeds from transgenic plants; the second set is negative segregants of the transgenic seed; and the third seed set is seed from two cold tolerant and two cold sensitive wild-type controls. All seeds are treated with a fungicide as indicated above. Seeds are grown in germination paper (12 inch x 18 inch pieces of Anchor Paper #SD7606), wetted in a solution of 0.5% KNO3 and 0.1% Thyram. For each paper fifteen seeds are placed on the line evenly spaced such that the radical s will grow toward the same edge. The wet paper is rolled up evenly and tight enough to hold the seeds in place. The roll is secured into place with two large paper clips, one at the top and one at the bottom. The rolls are incubated in a growth chamber at 23 degree C for three days in a randomized complete block design within an appropriate container. The chamber is set for 65% humidity with no light cycle. For the cold stress treatment the rolls are then incubated in a growth chamber at 12 degree C for fourteen days. The chamber is set for 65% humidity with no light cycle. For the warm treatment the rolls are incubated at 23 degree C for an additional two days. After the treatment the germination papers are unrolled and the seeds that did not germinate are discarded. The lengths of the radicle and coleoptile for each seed are measured. A coleoptile sample is collected from six individual kernels of each entry for confirming the expression of recombinant DNA. Statistical differences in the length of radical and shoot during pre-shock and cold shock are used for an estimation of the effect<br>
of the cold treatment on com plants. The analysis is conducted independently for the<br>
warm and cold treatments.<br>
Screen for enhanced oil, starch, or protein levels in plant seeds<br>
Oil levels of plant seeds are determined by low-resolution .sup.lH nuclear magnetic resonance (NMR) (Tiwari et al, JAOCS, 51:104-109 (1974); or Rubel, JAOCS, 71:1057-1062 (1994)). Alternatively, oil, starch and protein levels in seeds are determined by near infrared spectroscopy (NIR).<br>
The following examples illustrate aspects of the invention.<br>
Example 1<br>
This example illustrates the construction of plasmids for transferring recombinant DNA into plant cells which can be regenerated into transgenic plants of this invention. Primers for PCR amplification of protein coding nucleotides of recombinant DNA were designed at or near the start and stop codons of the coding sequence, in order to eliminate most of the 5' and 3' untranslated regions. Each recombinant DNA coding for a protein identified in Table 1 was amplified by PCR prior to insertion into the insertion site of one of the base vectors as referenced in Table 1.<br>
A base plant transformation vector pMON65154 was fabricated for use in preparing recombinant DNA for transformation into com tissue using GATEWAYโข Destination plant expression vector systems (available from Invitrogen Life Technologies, Carlsbad, CA). With reference to the elements described in Table 5 below and SEQ ID NO:9, pMON65154 comprises a selectable marker expression cassette and a template recombinant DNA expression cassette. The marker expression cassette comprises a CaMV 35S promoter operably linked to a gene encoding neomycin phosphotransferase II (nptll) followed by a 3' region of an Agrobacterium tumefadens nopaline synthase gene (nos). The template recombinant DNA expression cassette is positioned tail to tail with the marker expression cassette. The template recombinant DNA expression cassette comprises 5' regulatory DNA including a rice actin 1 promoter, exon and intron, followed by a GATEWAYโข insertion site for recombinant DNA, followed by a 3' region of a potato proteinase inhibitor II (pinU) gene. Once recombinant<br>
 <br>
DNA has been inserted into the insertion site, the plasmid is useful for plant transformation, for example by microprojectile bombardment.<br>
Table 5<br>
<br>
(Table Removed)<br>
similar base vector plasmid pMON72472 (SEQ ID NO: 10) was constructed for use in Agrobacterium-mediated methods of plant transformation similar to pMON65154 except (a) the 5' regulatory DNA in the template recombinant DNA expression cassette was a rice actin promoter and a rice actin intron, (b) left and right T-DNA border sequences from Agrobacterium are added with the right border sequence is located 5' to the rice actin 1 promoter and the left border sequence is located 3' to the 35S promoter and (c) DNA is added to facilitate replication of the plasmid in both E. coli and Agrobacterium tumefaciens. The DNA added to the plasmid outside of the T-DNA border sequences<br>
includes an orN wide host range origin of DNA replication functional in Agrobacterium, a pBR322 origin of replication functional in E.coli, and a spectinomycin/stretptomycin resistance gene for selection in both E. coli and Agrobacterium. pMON74775 is constructed in a base vector essentially the same as pMON72472.<br>
Other base vectors similar to those described above were also constructed including pMON81244 containing a pyruvate orthophosphate dikinase (PPDK) promoter (SEQ ID NO: 11) and a maize DnaK intron (SEQ ID NO: 12) as an enhancer.<br>
Plant expression vector for soybean transformation<br>
Plasmids for use in transformation of soybean were also prepared. Elements of an exemplary common expression vector plasmid pMON74532 (SEQ ID NO: 13) are shown in Table 7 below.<br>
A plasmid vector similar to that described above for soy transformation was constructed for use in Agrobacterium-mediated soybean transformation, pMON74537, which contains the Arabidopsis thaliana ribulose-l,5-bisphosphate carboxylase (Rubisco) small subunit promoter (SEQ ED NO: 14)<br>
Protein coding segments of recombinant DNA are amplified by PCR prior to insertion into vectors at the insertion site. Primers for PCR amplification are designed at or near the start and stop codons of the coding sequence, in order to eliminate most of the 5' and 3' untranslated regions.<br>
Table 7<br>
<br>
(Table Removed)<br>
Example 2<br>
This example illustrates plant transformation useful in producing the transgenic corn plants of this invention. Com plants of a readily transformable line are grown in the<br>
greenhouse and ears harvested when the embryos are 1.5 to 2.0 mm in length. Ears are surface sterilized by spraying or soaking the ears in 80% ethanol, followed by air drying. Immature embryos are isolated from individual kernels on surface sterilized ears. Prior to inoculation of maize cells, Agrobacterium cells are grown overnight at room temperature. Immature maize embryos are inoculated with Agrobacterium shortly after excision, and incubated at room temperature with Agrobacterium for 5-20 minutes. Immature embryos are then co-cultured with Agrobacterium for 1 to 3 days at 23ยฐC in the dark. Co-cultured embryos are transferred to selection media and cultured for approximately two weeks to allow embryogenic callus to develop. Embryogenic callus is transferred to culture medium containing 100 mg/L paromomycin and subcultured at about two week intervals. Transformants are recovered 6 to 8 weeks after initiation of selection.<br>
Plasmid vectors are prepared cloning DNA identified in Table 1 in the identified base for use in corn transformation to produce transgenic com plants and seed.<br>
For Agrobacterium-mediated transformation of maize callus, immature embryos are cultured for approximately 8-21 days after excision to allow callus to develop. Callus is then incubated for about 30 minutes at room temperature with the Agrobacterium suspension, followed by removal of the liquid by aspiration. The callus and Agrobacterium are co-cultured without selection for 3-6 days followed by selection on paromomycin for approximately 6 weeks, with biweekly transfers to fresh media, and paromomycin resistant callus identified as containing the recombinant DNA in an expression cassette.<br>
For transformation by microprojectile bombardment, immature maize embryos are isolated and cultured 3-4 days prior to bombardment. Prior to microprojectile bombardment, a suspension of gold particles is prepared onto which the desired recombinant DNA expression cassettes are precipitated. DNA is introduced into maize cells as described in U.S. Patents 5,550,318 and 6,399,861 using the electric discharge particle acceleration gene delivery device. Following microprojectile bombardment,<br>
tissue is cultured in the dark at 27 degrees C.<br>
To regenerate transgenic corn plants trangenic callus resulting from transformation is placed on media to initiate shoot development in plantlets which are transferred to potting soil for initial growth in a growth chamber at 26 degrees C followed by a mist bench before transplanting to 5 inch pots where plants are grown to maturity. The plants are self fertilized and seed is harvested for screening as seed, seedlings or progeny R2 plants or hybrids, e.g., for yield trials in the screens indicated above.<br>
Example 3<br>
This example further illustrates the production and identification of transgenic seed for transgenic corn having an enhanced agronomic trait, i.e. enhanced nitrogen use efficiency, increased yield, enhanced water use efficiency, enhanced tolerance to cold and/or improved seed compositions as compared to control plants. Transgenic corn seed and plants comprising recombinant DNA from each of the genes cloned in one of base vectors as identified in Table 1 are prepared by transformation. Many transgenic events which survive to fertile transgenic plants that produce seeds and progeny plants will not exhibit an enhanced agronomic trait. The transgenic plants and seeds having enhanced agronomic traits of this invention are identified by screening for nitrogen use efficiency, yield, water use efficiency, and cold tolerance. Transgenic plants providing seeds with improved seed compositions are identified by analyzing for seed compositions including protein, oil and starch levels.<br>
A. Enhanced Nitrogen Use Efficiency<br>
The transgenic plants with enhanced nitrogen use efficiency provided by this invention<br>
were selected through the selection process according to the standard procedure described above and the performance of these transgenic plants are shown in Table 8 below.<br>
(Table Removed)<br>
Yield<br>
The transgenic plants with enhanced yield provided by this invention were selected through the selection process according to the standard procedure described above and the performance of these transgenic plants are shown in Tables 9 and 10 below indicating the change in corn yield measured in bushels per acre-Table 9<br>
<br>
(Table Removed)<br>
Table 10<br>
<br>
(Table Removed)<br>
Water Use Efficiency<br>
The transgenic plants with enhanced water use efficiency provided by this invention were<br>
selected through the selection process according to the standard procedure described above and the performance of these transgenic plants are shown in Table 11 below.<br>
Table 11<br>
<br>
(Table Removed)<br>
Cold Tolerance<br>
The transgenic plants with enhanced cold tolerance provided by this invention were selected through the selection process according to the standard procedure described above and the performance of the early seedling growth of these transgenic plants are shown in Table 12 below.<br>
Table 12<br>
<br>
(Table Removed)<br>
Table 13 <br>
Improved Seed Composition<br>
The transgenic plants with improved seed composition provided by this invention were selected through the selection process according to the standard procedure described above and the performance of these transgenic plants are shown in Tables 13-5.<br>
Table 14 Oil<br>
<br>
(Table Removed)<br>
Table 15 Protein<br>
<br>
(Table Removed)<br>
Example 4<br>
This example illustrate transgenic plants with enhanced traits through combinations. As<br>
illustrated in the Example 3, transgenic plants with enhanced agronomic traits are generated employing the recombinant DNA from each of the genes identified in Table 1. To produce further enhancement of agronomic traits in transgenic plants, the genes of Table 1 are combined with one or more additional genes that enhance agronomic traits to generate a transgenic plant with greater enhancement in one or more agronomic traits than either gene alone. This combination is achieved through either through transformation or breeding. The following example illustrates this principle. A transgenic maize plant stably transformed with a construct, pMON74923, containing the Zea mays phytochrome B (phyB) gene (SEQ ID NO: 15) under the control of a maize aldolase (FDA) promoter (U.S. patent application Serial No.09/757,089) was crossed with a transgenic maize plant stably transformed with pMON68392. The cross<br>
demonstrated an increased yield (bu./a) of 7.2% compared to the maize plant containing the phyB gene alone (2.4%).<br>
Example 5. Soybean Plant Transformation<br>
This example illustrates plant transformation useful in producing the transgenic soybean plants of this invention and the production and identification of transgenic seed for transgenic soybean having an enhanced agronomic trait, i.e. enhanced nitrogen use efficiency, enhanced yield, enhanced water use efficiency, enhanced growth under cold stress, and/or enhanced seed oil, protein and/or starch levels as compared to control plants. For Agrobacterium mediated transformation, soybean seeds are germinated overnight and the meristem explants excised. The meristems and the explants are placed in a wounding vessel. Soybean explants and induced Agrobacterium cells from a strain containing plasmid DNA with the gene of interest cassette and a plant selectable marker cassette are mixed no later than 14 hours from the time of initiation of seed germination and wounded using sonication. Following wounding, explants are placed in co-culture for 2-5 days at which point they are transferred to selection media for 6-8 weeks to allow selection and growth of transgenic shoots. Trait positive shoots are harvested approximately 6-8 weeks post bombardment and placed into selective rooting media for 2-3 weeks. Shoots producing roots are transferred to the greenhouse and potted in soil. Shoots that remain healthy on selection, but do not produce roots are transferred to non-selective rooting media for an additional two weeks. Roots from any shoots that produce roots off selection are tested for expression of the plant selectable marker before they are transferred to the greenhouse and potted in soil.<br>
Example 6<br>
This example further illustrates the production and identification of transgenic seed for transgenic soybean having an enhanced agronomic trait, i.e. enhanced nitrogen<br>
use efficiency, increased yield, enhanced water use efficiency, enhanced growth under cold stress, and/or improved seed compositions as compared to control plants. Transgenic soybean seed and plants comprising recombinant DNA from each of the genes cloned in one of base vectors as identified in Table 1 are prepared by transformation. Many 5     transgenic events which survive to fertile transgenic plants that produce seeds and progeny plants will not exhibit an enhanced agronomic trait. The transgenic plants and seeds having enhanced agronomic traits of this invention are identified by screening for nitrogen use efficiency, yield, water use efficiency, and cold tolerance. Transgenic plants providing seeds with improved seed compositions are identified by analyzing for seed 10     compositions including protein, oil and starch levels.<br>
<br>
<br>
<br>
What is claimed is:<br>
1.	A plant cell with stably integrated, recombinant DNA comprising a promoter that<br>
is functional in plant cells and that is operably linked to DNA that encodes a protein having domains of amino acids in a sequence that exceed the Pfam gathering cutoff for amino acid sequence alignment with a Pfam Homeobox protein domain family and a Pfam HALZ protein domain family; wherein the Pfam gathering cuttoff for the Homeobox protein domain family is -4 and the and the Pfam gathering cuttoff for the HALZ protein domain family is 17; wherein said plant cell is selected from a population of plant cells with said recombinant DNA by screening plants that are regenerated from plant cells in said population and that express said protein for an enhanced trait as compared to control plants that do not have said recombinant DNA; and wherein said enhanced trait is selected from group of enhanced traits consisting of enhanced water use efficiency, enhanced cold tolerance, increased yield, enhanced nitrogen use efficiency, enhanced seed protein and enhanced seed oil.<br>
2.	A plant cell of claim 1 wherein said protein has an amino acid sequence with at<br>
least 90% identity to a consensus amino acid sequence in the group of consensus amino<br>
acid sequences consisting of the consensus amino acid sequence constructed from an<br>
alignment of SEQ ID NO:5 through SEQ ID NO:8.<br>
3.	A plant cell of claim 1 wherein said protein is selected from the group of proteins<br>
having an amino acid sequence of SEQ ID NO:5 through SEQ ID NO:8.<br>
4.	A plant cell of claim 1 further comprising DNA expressing a protein that provides<br>
tolerance from exposure to an herbicide applied at levels that are lethal to a wild type of<br>
said plant cell.<br>
5.	A plant cell of claim 4 wherein the agent of said herbicide is a glyphosate,<br>
dicamba, or glufosinate compound.<br>
6.	A transgenic plant comprising a plurality of the plant cell of claim 1<br>
7.	A transgenic plant of claim 6 which is homozygous for said recombinant DNA.<br>
8.	A transgenic seed comprising a plurality of the plant cell of claim 1.<br>
9.	A transgenic seed of claim 8 from a corn, soybean, cotton, canola, alfalfa, wheat<br>
or rice plant.<br>
10.	Non-natural, transgenic corn seed of claim 9 wherein said seed can produce corn<br>
plants that are resistant to disease from the Mai de Rio Cuarto virus or the Puccina sorghi<br>
fungus or both.<br>
11.	A transgenic pollen grain comprising a haploid derivative of a plant cell of <br>
claim 1.<br>
12.	A method for manufacturing non-natural, transgenic seed that can be used to<br>
produce a crop of transgenic plants with an enhanced trait resulting from expression of<br>
stably-integrated, recombinant DNA comprising a promoter that is (a) functional in plant<br>
cells and (b) is operably linked to operably linked to DNA that encodes a protein having<br>
domains of amino acids in a sequence that exceed the Pfam gathering cutoff for amino<br>
acid sequence alignment with a Pfam Homeobox protein domain family and a Pfam<br>
HALZ protein domain family; wherein the Pfam gathering cuttoff for the Homeobox<br>
protein domain family is -4 and the and the Pfam gathering cuttoff for the HALZ protein<br>
domain family is 17; and wherein said enhanced trait is selected from the group of<br>
enhanced traits consisting of enhanced water use efficiency, enhanced cold tolerance,<br>
increased yield, enhanced nitrogen use efficiency, enhanced seed protein and enhanced<br>
seed oil, said method for manufacturing said seed comprising:<br>
<br>
(a)	screening a population of plants for said enhanced trait and said recombinant DNA,<br>
wherein individual plants in said population can exhibit said trait at a level less than,<br>
essentially the same as or greater than the level that said trait is exhibited in control plants<br>
which do not express the recombinant DNA,<br>
(b)	selecting from said population one or more plants that exhibit the trait at a level<br>
greater than the level that said trait is exhibited in control plants,<br>
(c)	verifying that said recombinant DNA is stably integrated in said selected plants,<br>
(d)	analyzing tissue of a selected plant to determine the production of a protein having the<br>
function of a protein encoded by nucleotides in a sequence of one of SEQ ID NO: 1 -4;<br>
and<br>
(e)	collecting seed from a selected plant.<br>
13.	A method of claim 12 wherein plants in said population further comprise DNA<br>
expressing a protein that provides tolerance to exposure to an herbicide applied at levels<br>
that are lethal to wild type plant cells, and wherein said selecting is effected by treating said population with said herbicide.<br>
14.	A method of claim 13 wherein said herbicide comprises a glyphosate, dicamba, or<br>
glufosinate compound.<br>
15.	A method of claim 12 wherein said selecting is effected by identifying plants with<br>
said enhanced trait.<br>
16.	A method of claim 12 wherein said seed is corn, soybean, cotton, alfalfa, wheat or<br>
rice seed.<br>
17.	A method of producing hybrid com seed comprising:<br>
<br>
(a)	acquiring hybrid corn seed from a herbicide tolerant corn plant which also has<br>
stably-integrated, recombinant DNA comprising a promoter that is (a) functional<br>
in plant cells and (b) is operably linked to operably linked to DNA that encodes a<br>
protein having domains of amino acids in a sequence that exceed the Pfam<br>
gathering cutoff for amino acid sequence alignment with a Pfam Homeobox<br>
protein domain family and a Pfam HALZ protein domain family; wherein the<br>
Pfam gathering cuttoff for the Homeobox protein domain family is -4 and the and<br>
the Pfam gathering cuttoff for the HALZ protein domain family is 17;<br>
(b)	producing corn plants from said hybrid corn seed, wherein a fraction of the plants<br>
produced from said hybrid com seed is homozygous for said recombinant DNA, a<br>
fraction of the plants produced from said hybrid corn seed is hemizygous for said<br>
recombinant DNA, and a fraction of the plants produced from said hybrid corn<br>
seed has none of said recombinant DNA;<br>
(c)	selecting corn plants which are homozygous and hemizygous for said<br>
recombinant DNA by treating with an herbicide;<br>
(d)	collecting seed from herbicide-treated-surviving corn plants and planting said<br>
seed to produce further progeny corn plants;<br>
(e)	repeating steps (c) and (d) at least once to produce an inbred corn line;<br>
(f)	crossing said inbred com line with a second corn line to produce hybrid seed.<br>
18.	The method of selecting a plant comprising cells of claim 1 wherein an<br>
immunoreactive antibody is used to detect the presence of said protein in seed or plant<br>
tissue.<br>
19.	Anti-counterfeit milled seed having, as an indication of origin, a plant cell of<br>
claim 1.<br>
20.	A method of growing a corn, cotton or soybean crop without irrigation water<br>
comprising planting seed having plant cells of claim 1 which are selected for enhanced<br>
water use efficiency.<br>
21.	A method of claim 20 comprising providing up to 300 millimeters of ground<br>
water during the production of said crop.<br>
22.	A method of growing a corn, cotton or soybean crop without added nitrogen<br>
fertilizer comprising planting seed having plant cells of claim 1 which are selected for<br>
enhanced nitrogen use efficiency.<br>
</td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/viewdoc.aspx?id=fwhyFdtemO2v6qt4L67Ahg==&amp;amp;loc=+mN2fYxnTC4l0fUd8W4CAA==" target="_blank" style="word-wrap:break-word;">http://ipindiaonline.gov.in/patentsearch/GrantedSearch/viewdoc.aspx?id=fwhyFdtemO2v6qt4L67Ahg==&amp;amp;loc=+mN2fYxnTC4l0fUd8W4CAA==</a></p>
		<br>
		<div class="pull-left">
			<a href="272762-communication-station-communication-method-and-communication-system.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="272764-process-for-the-generation-of-a-synthesis-gas.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>272763</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>5699/DELNP/2007</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>18/2016</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>29-Apr-2016</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>25-Apr-2016</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>23-Jul-2007</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>MONSANTO TECHNOLOGY, LLC</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td>800 NORTH,LINDBERGH BOULEVARD,ST.LOUIS,MISSOURI 63167 USA.</td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>LUND ADRIAN</td>
											<td>800 NORTH,LINDBERGH BOULEVARD,ST.LOUIS,MISSOURI 63167 USA.</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>A01H 1/00</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>PCT/US2005/046013</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td>2005-12-19</td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td>60/638,099</td>
									<td>2004-12-21</td>
								    <td>U.S.A.</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/272763-method-of-producing-transgenic-seeds-with-enhanced-agronomic-traits by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 01:06:23 GMT -->
</html>
