<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/272884-novel-variant-hypocrea-jacorina-cbh2-cellulases by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 01:40:49 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 272884:&quot;NOVEL VARIANT HYPOCREA JACORINA CBH2 CELLULASES&quot;</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">&quot;NOVEL VARIANT HYPOCREA JACORINA CBH2 CELLULASES&quot;</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>Described herein are Variants of H. Jecorina CBG2, a Cel6A Enzyme. The present invention provides novel cellobiohydrolases that have altered thermostability</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td>NOVEL VARIANT HYPOCREA JECORINA CBH2 CELLULASES<br>
CROSS-REFERENCE TO RELATED APPLICATIONS [01]    The present application claims priority to U.S. Provisional Patent Application Serial<br>
No. 60/640,398, entitled Novel Variant Hyprocrea jecorina CBHII Cellulases, filed December 30,2004 (Attorney Docket No. GC864P) and U.S. Provisional Patent Application Serial No. 60/656,863, entitled Novel Van'ant Hyprocrea jecorina CBHII Cellulases, filed February 25, 2005 (Attorney Docket No. GC864-P2).<br>
<br>
STATEMENT AS TO RIGHTS TO INVENTIONS MADE UNDER FEDERALLY<br>
SPONSORED RESEARCH AND DEVELOPMENT [02]    Portions of this work were funded by Subcontract No. ZCO-0-30017-01 with the<br>
National Renewable Energy Laboratory under Prime Contract No. DE-AC36-99GO10337 with the U.S. Department of Energy. Accordingly,,the United States Government may have certain rights in this invention.<br>
FIELD OF THE INVENTION<br>
[03]    The present invention relates to variant cellobiohydrolase enzymes and isolated nucleic acid sequences which encode polypeptides having cellobiohydrolase activity. The invention also relates to nucleic acid constructs, vectors, and host ceils comprising the nucleic acid sequences as well as methods for producing recombinant variant CBH polypeptides.<br>
REFERENCES<br>
1.	Sheehan and Himmel Biotechnology Progress 15, pp 817-827 (1999)<br>
2.	Matti Linko Proceedings of the Second TRICEL Symposium on Trichoderma reesei<br>
Cellulases and Other Hydroiases pp9-11 (1993)<br>
3.	Tuula T. Teeri Trends in Biotechnology 15, pp 160-167 (1997)<br>
4.	T.T. Teeri et al. Spec, Publ. - R. Soc. Chem., 246 (Recent Advances in Carbohydrate<br>
Bioengineering), pp 302-308. (1999)<br>
 <br>
5.	PDB reference 1QK2 '(Cel6A = CBH2) J.-Y. Zou, G.J, Kleywegt, J. Stahlberg, H.<br>
Drigues, W. Nerinckx, M. Claeyssens, A. Koivula, T.T. Teeri,.T.A Jones, Structure<br>
(LONDON), V.  7 p. 1035 (1999)<br>
6.	PDB reference 2BVW Structural changes of the active site tunnel of Humicola<br>
insolens celloblohydrolase, Cel6A, upon oligosaccharide binding., Varrot A, Schuteln<br>
M, Davies GJ, Biochemistry 1999 Jui 13;38(28):8884-91.<br>
7.	PDB reference 1DYS Structure and function of Humicola insolens family 6<br>
cellulases: structure of the endoglucanase, Cel6B, at 1.6 A resblutlon., Davies<br>
GJ, Brzozowski AM, Dauter M, Varrot A, Schulein M, Biochem 3 2000 May<br>
15;348Pt 1:201-7.<br>
BACKGROUND OF THE INVENTION  ;<br>
[04]     Cellulose and hemicellulose are the most abundant plant materials produced by photosynthesis. They can be degraded and used as an energy source by numerous microorganisms, including bacteria, yeast and fungi, that produce extracellular enzymes capable of hydrolysis of the polymeric substrates to monomeric sugars (Aro etal., J. Biol. Chem., vol. 276, no. 26, pp. 24309-24314, June 29,2001). As the limits of non-renewable resources approach, the potential of cellulose to become a major renewable energy resource is enormous (Krishna etal., Bioresource Tech. 77:193-196,2001). The effective utilization of cellulose through biological processes is one approach to overcoming the shortage of foods, feeds, and fuels (Ohmiya etal., Biotechnol. Gen. Engineer. Rev. vol. 14, pp. 365-414,1997).<br>
[05]    Cellulases are enzymes that hydrolyze cellulose (beta-1,4-glucan or beta D-glucosidic linkages.) resulting in the formation of glucose, cellobiose, cellooligosaccharides, and the like. Cellulases have been traditionally divided into three major classes: endoglucanases (EC 3.2.1.4) ("EG"), exoglucanases or cellobiohydrolases (EC 3.2.1.91) ("CBH") and beta-glucosidases ([beta] -D-glucoside glucohydrolase; EC 3.2.1.21) ("BG"). (Knowles et al., TIBTECH 5,255-261,1987; SchQIein, Methods Enzymol., 160,25, pp. 234-243,1988). Endoglucanases act mainly on the amorphous parts of the cellulose fibre, whereas cellobiohydrolases are also able to degrade crystalline cellulose (Nevalainen and Penttila", Mycota, 303-319,1995). Thus, the presence of a cellobiohydrolase in a cellulase system is required for efficient solubilization of crystalline cellulose (Suurnakki, et al. Cellulose 7:189-209,2000). Beta-glucosidase acts to liberate D-glucose units from cellobiose, cello-oligosaccharides, and other glucosides (Freer, J. Biol. Chem. vol. 268, no. 13, pp. 9337-9342,1993).<br>
 <br>
[06]    Cellulases are known to be produced by a large number of bacteria, yeast and fungi. Certain fungi produce a complete cellulase system capable of degrading crystalline forms of cellulose, such that the cellulases are readily produced in large quantities via fermentation. Filamentous fungi play a special role since many yeast, such as Saccharomyces cerevisiae, lack the ability to hydrolyze cellulose. See, e.g., Aro etal., 2001; Biochemistry and Genetics of Cellulose Degradation, eds. Aubert, J. P. et al., Academic Press, 1988; Wood et al., Methods in Enzymology, vol. 160, no. 9, pp. 87-116,1988, and Coughian, et al.,<br>
[07]     "Comparative Biochemistry of Fungal and Bacterial Cellulolytic Enzyme Systems" Biochemistry and Genetics of Cellulose Degradation, pp. 11 -301988. [08]    The fungal cellulase classifications of CBH, EG and BG can be further expanded to include multiple components within each classification. For example, multiple CBHs, EGs and BGs have been isolated from a variety of fungal sources including Trichoderma reesei (also referred to as Hypocrea jecorina) which contains known genes for 2 CBHs, I.e., CBH I ("CBH1") and CBH II ("CBH2"), at least 8 EGs, i.e., EG I, EG II, EG III, EGIV, EGV, EGVI, EGVII and EGVill, and at least 5 BGs, i.e., BG1, BG2, BG3, BG4 and BGS. EGIV, EGVI and EGVill also have xyloglucanase activity.<br>
[09]     In order to efficiently convert crystalline cellulose to glucose the complete cellulase system comprising components from each of the CBH, EG and BG classifications is required, with isolated components less effective in hydrolyzing crystalline cellulose (Filho et al., Can. J. Microbiol. 42:1-5,1996). A synergistic relationship has been observed between cellulase components from different classifications. In particular, the EG-type cellulases and CBH- type cellulases synergistically interact to more efficiently degrade cellulose. See, e.g., Wood, Biochemical Society Transactions, 611th Meeting, Galway, vol. 13, pp. 407-410, 1985.<br>
[10]    Cellulases are known in the art to be useful in the treatment of textiles for the purposes of enhancing the cleaning ability of detergent compositions, for use as a softening agent, for improving the feel and appearance of cotton fabrics, and the like (Kumar etal., Textile Chemist and Cplorist, 29:37-42,1997).<br>
[11]    Cellulase-containing detergent compositions with improved cleaning performance (US Pat. No. 4,435,307; GB App. Nos. 2,095,275 and 2,094,826) and for use in the treatment of fabric to improve the feel and appearance of the textile (US Pat. Nos. 5,648,263, 5,691,178, and 5,776,757; GB App. No. 1,358,599; The Shizuoka Prefectural<br>
 <br>
Hamamatsu Textile Industrial Research Institute Report, Vo|. 24, pp. 54-61,1986), have<br>
been described.<br>
[12]    Hence, cellulases produced in fungi and bacteria have received significant attention.<br>
In particular, fermentation of Trichoderma spp. (e.g., Trichodema longibrachiatum or<br>
Trichoderma reesei) has been shown to produce a complete cellulase system capable of<br>
degrading crystalline forms of cellulose.<br>
[13]    Although cellulase compositions have been previously described, there remains a<br>
need for new and improved cellulase compositions for use in household detergents,<br>
stonewashing compositions or laundry detergents, etc. Cellulases that exhibit improved<br>
performance are of particular interest.<br>
BRIEF SUMMARY OF THE INVENTION<br>
[14]    The. invention provides an isolated cellulase protein, identified herein as variant CBH2, and nucleic adds which encode a variant CBH2.<br>
[15]     In one embodiment the invention is directed to a variant CBH2 cellulase, wherein said variant comprises a substitution or deletion at a position1 corresponding to one or more of residues V94, P9B, G118, M120, M134, T142, L144, M145, T148, T154, L179, Q204, V206, S210,1212, T214, L215, G231, T232, V250, Q276, N285, S291, G308, T312, S316, V323, N325,1333, G334, S343, T349, G360, S380, A381, S386, F411, S413, A416, Q426 and/or A429 in CBH2 from Hypocrea jecorina (SEQ ID NO: 2). In a first aspect, the invention encompasses a variant CBH2 cellulase, wherein said variant comprises a substitution or deletion at a position corresponding to one or more of residues V94E, P98L, G118P, M120L, M134G/L/V, T142V, L.144G/R/S, M145L, T148Y, T154A, L179A, Q204E, V206L, S210L7R, 1212V, T214M/Y, L215I, G231N, T232V, V250I, Q276L, N285Q, S291G, G308A, T312S, S316P, V323L/N/Y, N325D, I333L, G334A, S343P, T349L/V, G360R, S380T, A381T, S386P, F411Y, S413Y, A416G, Q426E and/or A429T in CBH2 from Hypocrea jecorina (SEQ ID NO: 2).<br>
[16]    in a second the invention is directed to a variant CBH2 cellulase, wherein said variant comprises a substitution or deletion at a position corresponding to one or more of residues V94, P98, G118, M120, M134, T142, M145, T148, T154, L179, Q204, V206,1212, L215, G231, T232, V250, Q276, N285, S291, G308, T312, S316, V323, N325,1333, G334, S343, T349, G360, S380, A381, S386, F411, S413, A416, Q426 and/or A429 in CBH2 from Hypocrea jecorina (SEQ ID NO: 2). In a first aspect, the invention encompasses a variant CBH2 cellulase, wherein said variant comprises a substitution or deletion at a position corresponding to one or more of residues V94E, P98L, G118P, M120L, M134V, T142V,<br>
 <br>
M145L, T148Y, T154A, L179A, Q204E, V206L, 1212V, L215I, G231N, T232V, V250I, Q276L, N285Q, 8291G, G308A, T312S, S316P, V323N, N325D, I333L, G334A, S343P, T349L, G360R, S380T, A381T, S386P, F411Y, S413Y, A416G, Q426E and/or A429T In CBH2 from Hypocreajecorina (SEQ ID NO: 2).<br>
[17]    In a third embodiment, the invention encompasses a variant CBH2 cellulase, wherein said variant comprises a substitution or deletion at a position corresponding to one or more of residues P98, M134, V206,1212, T312,8316, F411 and/or 8413 in CBH2 from Hypocrea jecorina (SEQ ID NO: 2). In a first aspect, the invention encompasses a variant CBH2 cellulase, wherein said variant comprises a substitution or deletion at a position corresponding to one or more of residues P98L, M134G/L/V, V206L, 1212V, T312S, S316P, F411Y and/or S413Y in CBH2 from Hypocreajecorina (SEQ ID NO: 2). [18]    In a fourth embodiment, the invention encompasses a variant CBH2 cellulase, wherein said variant comprises a substitution or deletion at a position corresponding to one or more residues in a spatial region and said spatial region is selected from the group consisting of (210,214), (253,255 ,257,258), (411,413,415), (412, 414,416), (312,313), 323, (212,149,152), (134,144) and 98 in CBH2 from Hypocreajecorina (SEQ ID NO: 2). In a first aspect, the invention encompasses a variant CBH2 cellulase, wherein said variant is selected from the group consisting of S316P/V323L, S316P/V323Y, V206L7S21 OR/8316P, V206L/S316P, V206L/S210I7T214M/S316P, V206L/S210R/T214Y/S316P, M134G/L144G/S316P, M134L/L144R/S316P and M134L/L1448/8316P in CBH2from Hypocreajecorina (SEQ ID N0:2).<br>
[19]    In a fifth embodiment, the invention encompasses a variant CBH2 cellulase, wherein said variant comprises a substitution or deletion at a position corresponding to one or more of residues P98, M134, L144, V206, 8210, T214, 8316, V323 and/or 8413 in CBH2 from Hypocreajecorina (SEQ ID NO': 2). In a first aspect, the invention encompasses a variant CBH2 cellulase, wherein said variant comprises a substitution or deletion at a position corresponding to one or more of residues P98L, M134L/V, L144R, V206L, S210L/R, T214Y, 8316P, V323Y and/or 8413Y in CBH2 from Hypocreajecorina (SEQ ID NO: 2). In a second aspect, the invention encompasses a variant CBH2 cellulase, wherein said variant is selected from the group consisting of 98L/134V/206L/21 OR/214Y/316P/413Y, 9817134L/144R/316P/413Y, 98L/134L7144R/206L/21 OR/214Y/316P/413Y, 98U134V/316P/323Y/413Y, 98L/134V/206L/21 OR/214Y/316P/323Y/413Y, 98U13417144R/316P/323Y/413Y, 98L/134L/144R/206I721 OR/214Y/316P/323Y/413Y, 9817134L/144R/21 OR/214Y/316P/323Y/413Y and<br>
 <br>
98L/134L/144R/210L/214Y/316P/323Y/413Y in CBH2 from Hypocrea jecorina (SEQ ID NO:<br>
2).<br>
[20]    In a sixth embodiment, the invention encompasses an isolated nucleic acid encoding<br>
a polypeptide having cellobiohydrolase activity.<br>
[21]    In a first aspect, the invention encompasses an isolated nucleic acid encoding a<br>
polypeptide having cellobiohydrolase activity, which polypeptide is a variant of a glycosyl<br>
hydrolase of family 6, and wherein said nucleic acid encodes a substitution at a residue<br>
which is sensitive to temperature stress in the polypeptide encoded by said nucleic acid,<br>
wherein said variant cellobiohydrolase is derived from H. jecorina cellobiohydrolase. In a<br>
second aspect, the invention encompasses an isolated nucleic acid encoding a polypeptide<br>
having cellobiohydrolase activity, which polypeptide is a variant of a glycosyl hydrolase of<br>
family 6, and wherein said nucleic acid encodes a substitution at a residue which is effects<br>
enzyme processitivity in the polypeptide encoded by said nucleic acid, wherein said variant<br>
cellobiohydrolase is derived from H. jecorina cellobiohydrolase. In a third aspect, the<br>
invention encompasses an isolated nucleic acid encoding a polypeptide having<br>
cellobiohydrolase activity, which polypeptide is a variant of a glycosyl hydrolase of family 6,<br>
and wherein said nucleic acid encodes a substitution at a residue which effects product<br>
inhibition in the polypeptide encoded by said nucleic acid, wherein said variant<br>
cellobiohydrolase is derived from H. jecorina cellobiohydrolase.<br>
[22]    In another aspect the invention is directed to an isolated nucleic acid encoding a<br>
variant CBH2 cellulase, wherein said variant comprises a substitution at a position<br>
corresponding to one or more of residues V94, P98, G118, M120, M134, T142, M145, T148<br>
T154, L179,0204, V206,1212, L215, G231, T232, V250, Q276, N285, S291, G308, T312,<br>
S316, V323, N325,1333, G334, S343, T349, G360, S380, A381, S390, F411, S413, A416,<br>
Q426 and/or A429 in CBH2 from Hypocrea jecorina (SEQ ID NO: 2).<br>
[23]     In an seventh embodiment the invention is directed to an expression cassette<br>
comprising a nucleic acid encoding a variant CBH2. In one aspect there is a construct<br>
comprising the nucleic acid of encoding the variant CBH2 operably linked to a regulatory<br>
sequence.<br>
[24]     In an eighth embodiment the invention is directed to a vector comprising a nucleic<br>
acid encoding a variant CBH2. In one aspect there is a construct comprising the nucleic<br>
acid of encoding the variant CBH2 operably linked to a regulatory sequence.<br>
[25]     In a ninth embodiment the invention is directed to a host cell transformed with the<br>
vector comprising a nucleic acid encoding a CBH2 variant.<br>
 <br>
[26]     In a tenth embodiment the invention is directed to a method of producing a CBH2 variant comprising the steps of:<br>
(a)	culturing a host cell transformed with the vector comprising a nucleic acid<br>
encoding a CBH2 variant in a suitable culture medium under suitable<br>
conditions to produce CBH2 variant;<br>
(b)	obtaining said produced CBH2 variant.<br>
[27]     In an eleventh embodiment the invention is directed to a detergent composition<br>
comprising a surfactant and a CBH2 variant. In one aspect of this embodiment the<br>
detergent is a laundry detergent. In a second aspect of this embodiment the detergent is a<br>
dish detergent In third aspect of this invention, the variant CBH2 cellulase is used in the<br>
treatment of a cellulose containing textile, jn particular, in the stonewashing or indigo dyed<br>
denim.<br>
[28]     In a twelfth embodiment the invention is directed to a feed additive comprising a<br>
CBH2 variant.<br>
[29]     In a, thirteenth embodiment the invention is directed to a method of treating wood<br>
pulp comprising contacting said wood pulp with a CBH2 variant.<br>
[30]     In a fourteenth embodiment the invention is directed to a method of converting<br>
biomass to sugars comprising contacting said biomass with a CBH2 variant.<br>
[31]     In an embodiment, the cellulase is derived from a fungus, bacteria or Actinomycete.<br>
In one aspect, the cellulase is derived from a fungus. In another aspect, the fungus is a<br>
filamentous fungus. It is preferred the filamentous fungus belong to Euascomycete, in<br>
particular, Aspergillus spp., Gliocladlum spp., Fusarium spp., Acremonium spp.,<br>
Myceliophtora spp., Verticillium spp., Myrothecium spp., or Periicillium spp. In a further<br>
aspect of this embodiment, the cellulase is a cellobiohydrolase.<br>
[32]    Other objects, features and advantages of the present invention will become<br>
apparent from the following detailed description. It should be understood, however, that the<br>
detailed description and specific examples, while indicating preferred embodiments of the<br>
invention, are given by way of illustration only, since various changes and modifications<br>
within the scope and spirit of the invention will become apparent to one skilled in the art from<br>
this detailed description.<br>
BRIEF DESCRIPTION OF THE DRAWINGS<br>
[33]     Rgure 1 is the amino acid (SEQ ID NO: 2) sequence of the wild type Cel6A (CBH2) from HJecorina.<br>
 <br>
 [34]     Figure 2 is the nucleic acid (SEQ ID NO: 1) sequence of the wild type H. jecorina CBH2.<br>
[35]    Rgures 3A-C show the amino acid, alignment of the Cel6 family members for which there were crystal structures available. The sequences are: - Humicola insolens CBH2, Acremonium GBH2, Agaricus CBH2, Fusarium oxysporum CBH2, Hypocrea koningii CBH2, Phanerochaete chrysosporum CBH2, Talaromyces emersonllCBH2, 7. reesel (i.e., Hypocrea jecorina CBH2, and the consensus sequence. Alignment has been done by Clustal W with a gap penalty of 10 using Vector NT! Suite software program. [36]     Rgure 4 is the pRAXt vector. This vector is based on the plasmid pGAPT2 except a 5259bp Hindlll fragment of Aspergillus nidulans genomic DMA fragment AMA1 sequent (Aleksenko and Clutterbuck, Molecular Microbiology 199619:565-574) was inserted. Base 1 to 1134 contains Aspergillus nlger glucoamylase gene promoter. Base 3098 to 3356 and 4950 to 4971 contains Aspergillus niger glucoamylase terminator. Aspergillus nidulans pyrQ gene was inserted from 3357 to 4949 as a marker for fungal transformation. There is a multiple cloning site (MCS) into which genes may be inserted. [37]    Rgure 5 is the pRAXdes2 vector backbone. This vector is based on the plasmid vector pRAX1. A Gateway cassette has been inserted into pRAX1 vector (indicated by the arrow on the interior of the circular plasmid). This cassette contains recombination sequence attR1 and attR2 and the selection marker catH and ccdB. The vector has been made according to the manual given in Gateway™ Cloning Technology: version 1 page 34-38 and can only replicate in £ coll DB3.1 from Invitrogen; in other E. coll hosts the ccdB gene is lethal. First a PCR fragment is made with primers containing attB1/2 recombination sequences. This fragment is recombined with pDONR201 (commercially available from Invitrogen); this vector contains attP1/2 recombination sequences with catH and ccdB in between the recombination sites. The BP clonase enzymes from Invitrogen are used to recombine the PCR fragment in this so-called ENTRY vector, clones with the PCR fragment inserted can be selected at 50ug/ml kanamycin because clones expressing ccdB do not survive. Now the att sequences are altered and called attL1 and attL2. The second step is to recombine this clone with the pRAXdes2 vector (containing attR1 and attR2 catH and ccdB in between the recombination sites). The LR clonase enzymes from Invitrogen are used to recombine the insert from the ENTRY vector in the destination vector. Only pRAXCBH2 vectors are selected using 100μg/ml ampicillin because ccdB is lethal and the ENTRY vector is sensitive to ampicillin. By this method the expression vector is now prepared and can be used to transform A. niger.<br>
 <br>
 [38]     Figure 6 provides an illustration of the pRAXdes2cbh2 vector which was used for<br>
expression of the nucleic acids encoding the CBH2 variants in Aspergillus. A nucleic acid<br>
encoding a CBH2 enzyme homolog or variant was cloned into the vector by homologous<br>
recombination of the att sequences.<br>
[39]     Figure 7 provides an illustration of the pENTRY-CBH2 vector.<br>
[40]    Figure 8 is a graph of the dose dependent release of sugars from phosphoric acid<br>
swollen cellulose by different variants. The variants display a wide range of activity on this<br>
substrate.'<br>
[41]    Rgure 9 is a bar graph of the ratio of the (average) total sugar produced by a variant<br>
and (average) total sugar produced by wild-type CBH2 as measured on PASC.<br>
[42]     Figure 10 is a graph showing the amount of sugar released by varying amounts of<br>
enzyme. Wild-type enzyme is denoted as FCA500.3 (open diamonds) and the variant is<br>
FCA543 (P98UM134WT154A/I212V/S316P/S413Y) (open squares). The broth from a<br>
CBH2-deleted strain served as a control.<br>
[43]     Figure 11 is a bar graph of the ratio of the (average) total sugar produced by a<br>
variant and (average) total sugar produced by wild-type CBH2 as measured on pre-treated<br>
com stover. The minimum scale value of the y-axis is 0.6 representing the value of the<br>
(average) total sugar produced by CBH2-delete strain divided by the (average) total sugar<br>
produced by the wild-type CBH2 in combination with the CBH2-delete strain. A value of 1<br>
represents a level of activity similar to wild-type.<br>
[44]     Rgure 12 is a graph of a time course experiment. Total sugar released from PASC<br>
by a CBH2 molecule over time at 53°C is shown. Variant is shown as filled triangles (Δ);<br>
wild-type as open squares (a).<br>
[45]     Figure 13 is a graph of a time course experiment Total sugar released from PASC<br>
by a CBH2 molecule over time at 65°C is shown. Variant is shown as filled triangles (Δ);<br>
wild-type as open squares (a).<br>
[46]    Rgure 14 is a graph of a time course experiment Total sugar released from PASC<br>
•<br>
by a CBH2 molecule over time at 75°C is shown. Variant is shown as filled triangles (A); wild-type as open squares (D).<br>
[47]     Figure 15 is a graph depicting the specific performance of a cellulase mixture at 38°C. The mixture contains the Acidothermus cellulolyticus E1 catalytic core and either wild-type or variant cellobiohydrolases. The wild-type is designated as 301,500 which indicates that the wild-type CBH1 (i.e., 301) and wild-type CBH2 (i.e., 500) were used. The variant is designated as 469,543 which indicates that the variant CBH1 (i.e., 469) and variant CBH2 (i.e., 543) were used.<br>
 <br>
 [48]    Figure 16 is a graph depicting the specific performance of a cellulase mixture at 65°C. The mixture contains the Acidothermus cellulolyticus E1 catalytic core and either wild-type or variant cellobiohydrolases. The wild-type is designated as 301,500 which indicates that the wild-type CBH1 (i.e., 301) and wild-type CBH2 (i.e., 500) were used. The variant is designated as 469,543 which indicates that the variant CBH1 (i.e., 469) and variant CBH2 (i.e., 543) were used.<br>
[491    Figure 17 is a graph of small scale saccharifteation conversion assay results at varying temperatures of the variant cellulase mixture described above in Rgure 16.<br>
DETAILED DESCRIPTION<br>
[50]    The invention will now be described in detail by way of reference only using the following definitions and examples. All patents and publications, including ail sequences disclosed within such patents and publications, referred to herein are expressly incorporated by reference.<br>
[51]    Unless defined otherwise herein, all technical and scientific terms used herein have the same meaning as commonly understood by one of ordinary skill in the art to which this invention belongs. Singleton, etal., DICTIONARY OF MICROBIOLOGY AND MOLECULAR BIOLOGY, 2D ED., John Wiley and Sons, New York (1994), and Hale &amp; Mamam, THE HARPER COLLINS DICTIONARY OF BIOLOGY, Harper Perennial, NY (1991) provide one of skill with a general dictionary of many of the terms used in this invention. Although any methods and materials similar or equivalent to those described herein can be used in the practice or testing of the present invention, the preferred methods and materials are described. Numeric ranges are inclusive of the numbers defining the range. Unless otherwise indicated, nucleic acids are written left to right in 5' to 3* orientation; amino acid sequences are written left to right in amino to carboxy orientation, respectively. Practitioners are particularly directed to Sambrook et ai, MOLECULAR CLONING: A LABORATORY MANUAL (Second Edition), Cold Spring Harbor Press, Piainview, N.Y., 1989, and Ausubel FM etal., Current Protocols in Molecular Biology, John Wiley &amp; Sons, New York, NY, 1993, for definitions and terms of the art. It is to be understood that this invention is not limited to the particular methodology, protocols, and reagents des&amp;ribed, as these may vary. [52]    The headings provided herein are not limitations of the various aspects or embodiments of the invention which can be had by reference to the specification as a whole. Accordingly, the terms defined immediately below are more fully defined by reference to the specification as a whole.<br>
 <br>
 [S3]    All publications cited herein are expressly incorporated herein by reference for the purpose of describing and disclosing compositions and methodologies which might be used in connection with the invention.<br>
I.         DEFINITIONS<br>
[54]    The term "polypeptide" as used herein refers to a compound made up of a single chain of amino acid residues linked by peptide bonds. The term "protein* as used herein may be synonymous with the term "polypeptide".<br>
[55]    "Variant* means a protein which is derived from a precursor protein (e.g., the native protein) by addition of one or more amino adds to either or both the C- and N-terminal end, substitution of one or more amino acids at one or a number of different sites In the amino acid sequence, or deletion of one or more amino acids at either or both ends of the protein or at one or more sites in the amino acid sequence. The preparation of an enzyme variant is preferably achieved by modifying a DNA sequence which encodes for the native protein, transformation of the modified DNA sequence into a suitable host, and expression of the modified DNA sequence to form the variant enzyme. The variant CBH2 enzyme of the invention includes peptides comprising altered amino acid sequences in comparison with a precursor enzyme amino acid sequence wherein the variant CBH2 enzyme retains the characteristic cellulolytic nature of the precursor enzyme but which may have altered properties in some specific aspect. For example, a variant CBH2 enzyme may have an increased pH optimum or increased temperature or oxidative stability but will retain its characteristic cellulolytic activity. It is contemplated that the variants according to the present invention may be derived from a DNA fragment encoding a cellulase variant CBH2 enzyme wherein the functional activity of the expressed cellulase variant is retained. For example, a DNA fragment encoding a cellulase may further include a DNA sequence or portion thereof encoding a hinge or linker attached to the cellulase DNA sequence at either the 5' or 3' end wherein the functional activity of the encoded cellulase domain is retained. The terms variant and derivative may be used interchangeably herein. [56]     "Equivalent residues" may also be defined by determining homology at the level of tertiary structure for a precursor cellulase whose tertiary structure has been determined by x-ray crystallography. Equivalent residues are defined as those for which the atomic coordinates of two or more of the main chain atoms of a particular amino acid residue of a cellulase and Hypocrea jecorina CBH2 (N on N, CA on CA, C on C and O on 0) are within 0.13nm and preferably 0.1 nm after alignment. Alignment is achieved after the best model has been oriented and positbned to give the maximum overlap of atomic coordinates of non-hydrogen protein atoms of the cellulase in question to the H.jecorina CBH2. The best<br>
 <br>
model is the crystallographic model giving the lowest R factor for experimental diffraction data at the highest resolution available.<br>
(Equation Removed)<br>
 [57]     Equivalent residues which are functionally analogous to a specific residue of H. jecorina CBH2 are defined as those amino acids of a cellulase which may adopt a conformation such that they either alter, modify or contribute to protein structure, substrate binding or catalysis in a manner defined and attributed to a specific residue of the H. jecorina CBH2. Further, they are those residues of the cellulase (for which a tertiary structure has been obtained by x-ray crystallography) which occupy an analogous position to the extent that, although the main chain atoms of the given residue may not satisfy the criteria of equivalence on the basis of occupying a homologous position, the atomic coordinates of at least two of the side chain atoms of the residue lie with 0.13nm of the corresponding side chain atoms of H. jecorina CBH2. The crystal structure of H. jecorina CBH2 is shown in Zou et a/. (1999) (Ref. 5, supra).<br>
[58]     The term "nucleic acid molecule" includes RNA, DMA and cDNA molecules. It will be understood that, as a result of the degeneracy of the genetic code, a multitude of nucleotide sequences encoding a given protein such as CBH2 and/or variants thereof may be produced. The present invention contemplates every possible variant nucleotide sequence, encoding variant CBH2, all of which are possible given the degeneracy of the genetic code. [59]    A "heterologous" nucleic acid construct or sequence has a portion of the sequence which is not native to the cell in which it is expressed. Heterologous, with respect to a control sequence refers to a control sequence (i.e. promoter or enhancer) that does not function in nature to regulate the same gene the expression of which it is currently regulating. Generally, heterologous nucleic acid sequences are not endogenous to the cell or part of the genome in which they are present, and have been added to the cell, by infection, transfection, transformation, microinjection, electroporatlon, or the like. A "heterologous" nucleic acid construct may contain a control sequence/DNA coding sequence combination that is the same as, or different from a control sequence/DNA coding sequence combination found in the native cell.<br>
[60]     As used herein, the term "vector" refers to a nucleic acid construct designed for transfer between different host cells. An "expression vector" refers to a vector that has the ability to incorporate and express heterologous DNA fragments in a foreign cell. Many<br>
prokaryotic and eukaryotic expression vectors are commercially available. Selection of appropriate expression vectors is within the knowledge of those having skill in the art [61]    Accordingly, an "expression cassette" or "expression vector" is a nucleic acid construct generated recombinantly or synthetically, with a series of specified nucleic acid elements that permit transcription of a particular nucleic acid in a target cell. The recombinant expression cassette can be incorporated into a plasmid, chromosome, mitochondria! DNA, plastid DMA, virus, or nucleic acid fragment Typically, the recombinarri expression cassette portion of an expression vector includes, among other sequences, a nucleic acid sequence to be transcribed and a promoter.<br>
[62]    As used herein, the term "plasmid" refers to a circular double-stranded (ds) DNA construct used as a cloning vector, and which forms an extrachromosomal self-replicating genetic element in many bacteria and some eukaryotes.<br>
[63]    As used herein, the term "selectable marker-encoding nucleotide sequence* refers to a nucleotide sequence which is capable of expression in cells and where expression of the selectable marker confers to cells containing the expressed gene the ability to grow in the presence of a corresponding selective agent, or under corresponding selective growth conditions.<br>
[64]    As used herein, the term "promoter" refers to a nucleic acid sequence that functions to direct transcription of a downstream gene. The promoter will generally be appropriate to the host cell in which the target gene is being expressed. The promoter together with other transcriptional and translational regulatory nucleic acid sequences (also termed "control sequences") are necessary to express a given gene. In general, the transcriptional and translational regulatory sequences include, but are not limited to, promoter sequences, ribosomal binding sites, transcriptional start and stop sequences, translational start and stop sequences, and enhancer or activator sequences.<br>
[65]    "Chimeric gene" or "heterologous nucleic acid construct", as defined herein refers to a non-native gene (i.e., one that has been introduced into a host) that may be composed of parts of different genes, including regulatory elements. A chimeric gene construct for transformation of a host cell is typically composed of a transcriptional regulatory region (promoter) operably linked to a heterologous protein coding sequence, or, in a selectable marker chimeric gene, to a selectable marker gene encoding a protein conferring, for example, antibiotic resistance to transformed cells. A typical chimeric gene of the present invention, for transformation into a host cell, includes a transcriptional regulatory region that is constitutive or inducible, a protein coding sequence, and a terminator sequence. A<br>
chimeric gene construct may also include a second DNA sequence encoding a signal peptide if secretion of the target protein is desired.<br>
[66]    A nucleic acid is "operably linked" when it is placed into a functional relationship with another nucleic acid sequence. For example, DNA encoding a secretory leader is operably linked to DNA for a polypeptide if it is expressed as a preprotein that participates in the secretion of the polypeptide; a promoter or enhancer is operably linked to a coding sequence if it affects the transcription of the sequence; or a ribosome binding site is operably linked to a coding sequence if it is positioned so as to facilitate translation. Generally, "operably linked" means that the DNA sequences being linked are contiguous, and, in the case of a secretory leader, contiguous and in reading frame. However, enhancers do not have to be contiguous. Linking is accomplished by ligation at convenient restriction sites. If such sites do not exist, the synthetic oligonucieotide adaptors, linkers or primers for PCR are used in accordance with conventional practice.<br>
[67]    As used herein, the term "gene" means the segment of DNA involved in producing a polypeptide chain, that may or may not include regions preceding and following the coding region, e.g., 5' untranslated (5' UTR) or "leader" sequences and 3' UTR or "trailer" sequences, as well as intervening sequences (introns) between individual coding segments (exons).<br>
[68]     In general, nucleic acid molecules which encode the variant CBH2 will hybridize, under moderate to high stringency conditions to the wild type sequence provided herein as SEQ ID NO:1. However, in some cases a CBH2-encoding nucleotide sequence is employed that possesses a substantially different codon usage, while the protein encoded by the CBH2-encoding nucleotide sequence has the same or substantially the same amino acid sequence as the native protein. For example, the coding sequence may be modified to facilitate faster expression of CBH2 in a particular prokaryotic or eukaryotic expression system, in accordance with the frequency with which a particular codon is utilized by the host. Te'o, et al. (FEMS Microbiology Letters 190:13-19,2000), for example, describes the optimization of genes for expression in filamentous fungi.<br>
[69]    A nucleic acid sequence is considered to be "selectively hybridizable" to a reference nucleic acid sequence if the two sequences specifically hybridize to one another under moderate to high stringency hybridization and wash conditions. Hybridization conditions are based on the melting temperature (Tm) of the nucleic acid binding complex or probe. For example, "maximum stringency" typically occurs at about Tm-5°C (5°C below the Tm of the probe); "high stringency" at about 5-10°C below the Tm; "moderate" or "intermediate stringency" at about 10-20°C below the Tm of the probe; and "low stringency" at about 20-<br>
25°C below the Tm. Functionally, maximum stringency conditions may be used to identify<br>
sequences having strict identity or near-strict identity with the hybridization probe; while high<br>
stringency conditions are used to identify sequences having about 80% or more sequence<br>
identity with the probe.<br>
[70]    Moderate and high stringency hybridization conditions are well known in the art (see,<br>
for example, Sambrook, et al, 1989, Chapters 9 and 11, and in Ausubel, P.M., etal., 1993,<br>
expressly incorporated by reference herein). An example of high stringency conditions<br>
includes hybridization at about 42°C in 50% formamide, 5X SSC, 5X Denhardts solution,<br>
0.5% SDS and 100μg/ml denatured carrier DNA followed by washing two times in 2X SSC<br>
and 0.5% SDS at room temperature and two additional times in 0.1 X SSC and 0.5% SOS at<br>
42°C.<br>
[71]    As used herein, "recombinanf includes reference to a cell or vector, that has been<br>
modified by the Introduction of a heterologous nucleic acid sequence or that the ceil is<br>
derived from a cell so modified. Thus, for example, recombinant cells express genes that<br>
are not found in identical form within the native (non-recombinant) form of the cell or express<br>
native genes that are otherwise abnormally expressed, under expressed or not expressed at<br>
all as a result of deliberate human intervention.<br>
[72]    As used herein, the terms transformed", "stably transformed" or Iransgenic" with<br>
reference to a ceil means the cell has a non-native (heterologous) nucleic acid sequence<br>
integrated into its genome or as an eplsomal plasmid that is maintained through multiple<br>
generations.<br>
[73]    As used herein, the term "expression" refers to the process by which a polypeptide is<br>
produced based on the nucleic acid sequence of a gene. The process includes both<br>
transcription and translation.<br>
[74]    The term "introduced" in the context of inserting a nucleic acid sequence into a cell,<br>
means "transfection", or "transformation" or "transduction" and includes reference to the<br>
incorporation of a nucleic acid sequence into a eukaryotlc or prokaryotic cell where the<br>
nucleic acid sequence may be incorporated into the genome of the cell (for example,<br>
chromosome, plasmid, plastid, or mitochondria! DNA), converted into an autonomous<br>
replicon, or transiently expressed (for example, transf ected mRNA).<br>
[75]     it follows that the term "CBH2 expression" refers to transcription and translation of<br>
the cbh2gene or variants thereof, the products of which include precursor RNA, mRNA,<br>
polypeptide, post-translationally processed polypeptides, and derivatives thereof, including<br>
CBH2 from related species such as Trichoderma koningii, Hypocrea jecorina (also known as<br>
Trichoderma longibrachiatum, Trichoderma reesei or Trichoderma viride) and Hypocrea<br>
schweinitzii. By way of example, assays for CBH2 expression include Western blot for CBH2 protein, Northern blot analysis and reverse transcriptase polymerase chain reaction (RT-PCR) assays for cbh2mRNA, and Phosphoric Acid Swollen Cellulose and PAHBAH assays as described in the following: (a) PASC: (Karlsson, J. et al. (2001), Eur. J. Blochem, 268,6498-6507, Wood, T. (1988) in Methods in Enzymology, Vol. 160. Btomass Part a Cellulose and Hemicellulose (Wood, W. &amp; Kellog, S. Eds.), pp. 19-25, Academic Press, San Diego, CA, USA) and (b) PAHBAH: (Lever, M. (t972) Analytical Biochemistry, 47,273, Biakeney, A.B. &amp; Mutton, LL (1980) Journal of Science of Food and Agriculture, 31,889, Henry, R J. (1984) Journal of the Institute of Brewing, 90,37). [76]    The term "alternative splicing" refers to the process whereby multiple polypeptide isoforms are generated from a single gene, and involves the splicing together of nonconsecutive exons during the processing of some, but not all, transcripts of the gene. Thus a particular exon may be connected to any one of several alternative exons to form messenger RNAs. The alternatively-spliced mRNAs produce polypeptides ('splfce variants11) in which some parts are common while other parts are different. [77]    The term "signal sequence" refers to a sequence of amino acids at the N-terminal portion of a protein that facilitates the secretion of the mature form of the protein outside the cell. The mature form of the extracellular protein lacks the signal sequence that is cleaved off during the secretion process. [78]     By the term "host ceil" is meant a cell that contains a vector and supports the<br>
•<br>
replication, and/or transcription or transcription and translation (expression) of the<br>
expression construct. Host cells for use in the present invention can be prokaryotic cells,<br>
such as E co//, or eukaryotic cells such as yeast, plant, insect amphibian, or mammalian<br>
cells. In general, host cells are filamentous fungi.<br>
[79]    The term "filamentous fungi" means any and all filamentous fungi recognized by<br>
those of skill hi the art. A preferred fungus is selected from the group consisting of<br>
Asperglllus, Trichoderma, Fusarlum, Chrysosporium, Penicillium, Humicola, Neurospora, or<br>
alternative sexual forms thereof such as Emericella, Hypocrea. It has now been<br>
demonstrated that the asexual industrial fungus Trichoderma reese it is a clonaf derivative of<br>
the ascomycete Hypocrea jecorina. See Kuhls et al., PNAS (1996) 93:7755-7760.<br>
[80]    The term "cetlooiigosaccharide" refers to oligosaccharide groups containing from 2-8<br>
glucose units and having p-1,4 linkages, e.g., celtobiose.<br>
[81]    The term "cellulase" refers to a category of enzymes capable of hydrolyzing cellulose<br>
polymers to shorter cello-oligosaccharide oligomers, cellobiose and/or glucose. Numerous<br>
examples of cellulases, such as exoglucanases, exoceilobiohydrolases, endoglucanases,<br>
and glucosidases have been obtained from cellulolytic organisms, particularly including fungi, plants and bacteria.<br>
[82]     CBH2 from Hypocrea jecorina is a member of the Glycosyl Hydrolase Family 6 (hence Cel6) and, specifically, was the first member of that family identified in Hypocrea jecorina (hence Cel6A). The Glycosyl Hydrolase Family 6 contains both Endoglucanases and Cellobiohydrolases/exoglucanases, and that CBH2 is the latter. Thus, the phrases CBH2, CBH2-type protein and Cel6 cellobiohydrolases may be used interchangeably herein. [83]    The term "cellulose binding domain" as used herein refers to portion of the arnino acid sequence of a cellulase or a region of the enzyme that is involved in the cellulose binding activity of a cellulase or derivative thereof. Cellulose binding domains generally function by non-covalently binding the cellulase to cellulose, a cellulose derivative or other polysaccharide equivalent thereof. Cellulose binding domains permit or facilitate hydrolysis of cellulose fibers by the structurally distinct catalytic core region, and typically function independent of the catalytic core. Thus, a cellulose binding domain will not possess the significant hydrolytic activity attributable to a catalytic core. In other words, a cellulose binding domain is a structural element of the cellulase enzyme protein'tertiary structure that is distinct from the structural element which possesses catalytic activity. Cellulose binding domain and cellulose binding module may be used interchangeably herein, [84]    As used herein, the term "surfactant" refers to any compound generally recognized in the art as having surface active qualities. Thus, for example, surfactants comprise anionic, cationic and nonionic surfactants such as those commonly found in detergents. Anionic surfactants include linear or branched alkylbenzenesulfonates; alkyl or alkenyl ether sulfates having linear or branched alkyl groups or alkenyl groups; alkyl or alkenyl sulfates; olefinsutfonates; and alkanesulfonates. Ampholytic surfactants include quaternary ammonium salt sulfonates, and betaine-type ampholytic surfactants. Such ampholytic surfactants have both the positive and negative charged groups in the same molecule. Nonionic surfactants may comprise polyoxyalkylene ethers, as well as higher fatty acid alkanolamides or alkylene oxide adduct thereof, fatty acid glycerine monoesters, and the like.<br>
[85]    As used herein, the term "cellulose containing fabric* refers to any sewn or unsewn fabrics, yarns or fibers made of cotton or non-cotton containing cellulose or cotton or non-cotton containing cellulose blends including natural cellulosics and manmade cellulosics (such as jute, flax, ramie, rayon, and lyocell).<br>
 [861    As used herein, the term "cotton-containing fabric" refers to sewn or unsewn fabrics,<br>
yams or fibers made of pure cotton or cotton blends including cotton woven fabrics, cotton<br>
knits, cotton denims, cotton yams, raw cotton and the like.<br>
[87]    As used herein, the term "stonewashing composition" refers to a formulation for use<br>
In stonewashing cellulose containing fabrics. Stonewashing compositions are used*to<br>
modify cellulose containing fabrics prior to sale, I.e., during the manufacturing process. In<br>
contrast, detergent compositions are intended for the cleaning of soiled garments and are<br>
not used during the manufacturing process.<br>
[88]    As used herein, the term "detergent composition" refers to a mixture which is<br>
intended for use in a wash medium for the laundering of soiled cellulose containing fabrics.<br>
In the context of the present invention, such compositions may include, in addition to<br>
cellulases and surfactants, additional hydrolytic enzymes, builders, bleaching agents, bleach.<br>
activators, bluing agents and fluorescent dyes, caking inhibitors, masking agents, cellulase<br>
activators, antioxidants, and solubilizers.<br>
[89]     As used herein, the term "decrease or elimination in expression of the cbh2 gene"<br>
means that either that the cbh2 gene has been deleted from the genome and therefore<br>
/ cannot be expressed by the recombinant host microorganism; or that the cbh2 gene or<br>
transcript has been modified such that a functional CBH2 enzyme is not produced by the host microorganism.<br>
[90] • The term "variant cbh2 gene? or "variant CBH2" means, respectively, that the nucleic acid sequence of the cbh2 gene from H. jecorina has been altered by removing, adding, and/or manipulating the coding sequence or the amino acid sequence of the expressed protein has been modified consistent with the invention described herein. [91]    As used herein, the term "purifying" generally refers to subjecting transgenic nucleic acid or protein containing cells to biochemical purification and/or column chromatography. [92]     As used herein, the terms "active" and "biologically active" refer to a biological activity associated with a particular protein and are used interchangeably herein. For example, the enzymatic activity associated with a protease is proteolysis and, thus, an active protease has proteolytic activity. It follows that the biological activity of a given protein refers to any biological activity typically attributed to that protein by those of skill in the art. [93]     As used herein, the term "enriched" means that the CBH2 is found in a concentration that is greater relative to the CBH2 concentration found in a wild-type, or naturally occurring, fungal cellulase composition. The terms enriched, elevated and enhanced may be used interchangeably herein.<br>
 [94]    A wild type fungal cellulase composition is one produced by a naturally occurring fungal source and which comprises one or more BGL, CBH and EG components wherein each of these components is found at the ratio produced by the fungal source. Thus, an enriched CBH composition would have CBH at an altered ratio wherein the ratio of CBH to other cellulase components (i.e., EGs, beta-glucosidases and other endoglucanases) is elevated. This ratio may be increased by either increasing CBH or decreasing (or eliminating) at least one other component by any means known in the art [95]    The term "isolated" or "purified" as used herein refers to a nucleic acid or amino acid that is removed from at least one component with which it is naturally associated. [96]    Thus, to illustrate, a naturally occurring cellulase system may be purified into substantially pure components by recognized separation techniques well published in the literature, including ion exchange chromatography at a suitable pH, affinity chromatography, size exclusion and the like. For example, in ion exchange chromatography (usually anion exchange chromatography), it is possible to separate the cellulase components by eluting with a pH gradient, or a salt gradient, or both a pH and a salt gradient The purified CBH may then be added to the enzymatic solution resulting in an enriched CBH solution. It is also possible to elevate the amount of CBH produced by a microbe using molecular genetics methods to overexpress the gene encoding CBH, possibly in conjunction with deletion of one or more genes encoding other cellulases.<br>
[97]    Fungal cellulases may contain more than one CBH component. The different components generally have different isoelectric points which allow for their separation via ton exchange chromatography and the like. Either a single CBH component or a combination of CBH components may be employed in an enzymatic solution.<br>
[98]    When employed in enzymatic solutions, the homolog or variant CBH2 component is generally added in an amount sufficient to allow the highest rate of release of soluble sugars from the biomass. The amount of homolog or variant CBH2 component added depends upon the type of biomass to be saccharified, which can be readily determined by the skilled artisan, when employed, the weight percent of the homolog or variant CBH2 component present in the cellulase composition is from preferably between 1 and 100 with illustrative examples being about 1, preferably about 5, preferably about 10, preferably about 15, or preferably about 20 weight percent to preferably about 25, preferably about 30, preferably about 35, preferably about 40, preferably about 45 or preferably about 50 weight percent. Furthermore, preferred ranges may be about 0.5 to about 15 weight percent, about 0.5 to about 20 weight percent, from about 1 to about 10 weight percent, from about 1 to about 15 weight percent, from about 1 to about 20 weight percent, from about 1 to about 25 weight<br>
percent, from about 5 to about 20 weight percent, from about 5 to about 25 weight percent, from about 5 to about 30 weight percent, from about 5 to about 35 weight percent, from about 5 to about 40 weight percent, from about 5 to about 45 weight percent, from about 5 to about 50 weight percent, from about 10 to about 20 weight percent, from about 10 to about 25 weight percent, from about 10 to about 30 weight percent, from about 10 to about 35 weight percent, from about 10 to about 40 weight percent, from about 10 to about 45 weight percent, from about 10 to about 50 weight percent, from about 15 to about 60 weight Dercerrt, from about 15 to about 65 weight percent, from about 15 to about 70 weight percent from about 15 to about 75 weight percent, from about 15 to about 80 weight percent, from about 15 to about 85 weight percent, from about 15 to about 95 weight percent .However, when employed, the weight percent of the homblog or variant CBH2 component relative to any EG type components present in the cellulase composition is from preferably about 1, preferably about 5, preferably about 10, preferably about 15, or preferably about 20 weight percent to preferably about 25, preferably about 30, preferably about 35, preferably about 40, preferably about 45 or preferably about 50 weight percent Furthermore, preferred ranges may be about 0.5 to about 15 weight percent, about 0.5 to about 20 weight percent, from about 1 to about 10 weight percent, from about 1 to about 15 weight percent, from about 1 to about 20 weight percent from about 1 to about 25 weight percent, from about 5 to about 20 weight percent, from about 5 to about 25 weight percent from about 5 to about 30 weight percent, from about 5 to about 35 weight percent, from about 5 to about 40 weight percent, from about 5 to about 45 weight percent, from about 5 to about 50 weight percent, from about 10 to about 20 weight percent, from about 10 to about 25 weight percent, from about 10 to about 30 weight percent, from about 10 to about 35 weight percent, from about 10 to about 40 weight percent, from about 10 to about 45 weight percent, from about 10 to about 50 weight percent, from about 15 to about 20 weight percent from about 15 to about 25 weight percent, from about 15 to about 30 weight percent from about 15 to about 35 weight percent, from about 15 to about 30 weight percent from about 15 to about 45 weight percent, from about 15 to about 50 weight percent<br>
II.        HOST ORGANISMS<br>
[99]    Rlamentous fungi include all filamentous forms of the subdivision Eumycota and Oomycota. The filamentous fungi are characterized by vegetative mycelium having a cell wall composed of chitin, glucan, chitosan, mannan, and other complex polysaccharides, with vegetative growth by hyphal elongation and carbon catabolism that is obligately aerobic.<br>
 [100]   In the present invention, the filamentous fungal parent cell may be a cell of a species of, but not limited to, Trichoderma, e.g., Trichoderma tongibrachlatum, Trichoderma viride, Trichoderma koningii, Trichoderma harzianum; Penlcillium sp.; Humicola sp., including Humicola Insolens and Humicola grisea; Chrysosporium sp., including C. lucknowense, Gliocladium sp.; Asperglllus sp.', Fusarium sp., Neurospora sp., Hypocrea sp., and Emericella sp. As used herein, the term 'Trichoderma' or 'Trichoderma sp* refers to any fungal strains which have previously been classified as Trichoderma or are currently classified as Trichoderma.<br>
[101]   In one preferred embodiment, the filamentous fungal parent cell is an Asperglllus niger, Asperglllus awamori, Asperglllus aculeatus, or Asperglllus nidulans cell. [102]   in another preferred embodiment, the filamentous fungal parent cell is a Trichoderma reese/cell.<br>
III.       CELLULASES<br>
[103]   Cellulases are known in the art as enzymes that hydrolyze cellulose (beta-1,4-glucan or beta D-glucosidic linkages) resulting in the formation of glucose, cellobiose, cellooligosaccharides, and the like. As set forth above, cellulases have been traditionally divided into three major classes: endoglucanases (EC 32.1.4) ("EG"), exoglucanases or cellobiohydrolases (EC 3.2.1.91) ("CBH") and beta-glucosidases (EC 3.2.1.21) ("BG'). (Knowles, etal., TIBTECH 5,255-261,1987; Schulein, 1988). [104]   Certain fungi produce complete cellulase systems which include exo-cellobiohydrolases or CBH-type cellulases, endoglucanases or EG-type cellulases and beta-glucosidases or BG-type cellulases (Schulein, 1988). However, sometimes these systems lack CBH-type cellulases and bacterial cellulases also typically include little or no CBH-type cellulases. In addition, it has been shown that the EG components and CBH components synergistically interact to more efficiently degrade cellulose. See, e.g., Wood, 1985. The different components, I.e., the various endoglucanases and exocellobiohydrolases in a multi-component or complete cellulase system, generally have different properties, such as isoelectric point, molecular weight, degree of glycosylation, substrate specificity and enzymatic action patterns.<br>
[105]   It is believed that endoglucanase-type cellulases hydrolyze internal beta -1,4-glucosidic bonds in regions of low crystailinity of the cellulose and exo-cellobiohydrolase-type cellulases hydrolyze cellobiose from the reducing or non-reducing end of cellulose. It follows that the action of endoglucanase components can greatly facilitate the action of exocellobiohydrolases by creating new chain ends which are recognized by exo-cellobiohydrolase components. Further, beta-glucosidase-type cellulases have been shown<br>
to catalyze the hydrolysis of alkyl and/or aryl (β-D-glucosides such as methyl β-D-glucoside and p-nitrophenyl glucoside as well as glycosides containing only carbohydrate residues, such as cellobiose. This yields glucose as the sole product for the microorganism and reduces or eliminates cellobiose which inhibits cellobiohydrolases and endoglucanases. [106]   Cellulases also find a number of uses in detergent compositions including to enhance cleaning ability, as a softening agent and to improve the feel of cotton fabrics (Hemmpel, ITB Dyeing/Printing/Finishing 3:5-14,1991; Tyndall, Textile Chemist and Colorist 2423-26, 1992; Kumar etal., Textile Chemist and Colorist, 29:37-42,1997). While the mechanism is not part of the invention, softening and color restoration properties of cellulase have been attributed to the alkaline endoglucanase components in cellulase compositions, as exemplified by U.S. Patent Nos. 5,648,263,5,691,178, and 5,776,757, which disclose that detergent compositions containing a cellulase composition enriched in a specified alkaline endoglucanase component impart color restoration and improved softening to treated garments as compared to cellulase compositions not enriched in such a component. In addition, the use of such alkaline endoglucanase components in detergent compositions has been shown to complement the pH requirements of the detergent composition (e.g., by exhibiting maximal activity at an alkaline pH of 7.5 to 10, as described in U.S. Patent Nos. 5,648,263, 5,691,178, and 5,776,757).<br>
[107]   Cellulase compositions have also been shown to degrade cotton-containing fabrics, resulting in reduced strength loss in the fabric (U.S. Patent No. 4,822,516), contributing to reluctance to use cellulase compositions in commercial detergent applications. Cellulase compositions comprising endoglucanase components have been suggested to exhibit reduced strength loss for cotton-containing fabrics as compared to compositions comprising a complete cellulase system.<br>
[108]   Cellulases have also been shown to be useful in degradation of cellulase biomass to ethanol (wherein the cellulase degrades cellulose to glucose and yeast or other microbes further ferment the glucose into ethanol), in the treatment of mechanical pulp (Pere et a/., In Proc. Tappi Pulping Conf., Nashville, TN, 27-31, pp. 693-696,1996), for use as a feed additive (WO 91/04673) and in grain wet milling.<br>
[109]   Most CBHs and EGs have a multidomain structure consisting of a core domain separated from a cellulose binding domain (CBD) by a linker peptide (Suumakki et al., 2000). The core domain contains the active site whereas the CBD interacts with cellulose by binding the enzyme to it (van Tilbeurgh etai, .FEBS Lett. 204:223-227,1986; Tomme etal., Eur. J. Biochem. 170:575-581,1988). The CBDs are particularly important in the hydrolysis of crystalline cellulose. It has been shown that the ability of cellobiohydrolases to degrade<br>
 <br>
crystalline cellulose clearly decreases when the CBD is absent (Under and Teeri, J. Biotechnoi. 57:15-28,1997). However, the exact role and action mechanism of CBDs is still a matter of speculation. It has been suggested that the CBD enhances the enzymatic activity merely by increasing the effective enzyme concentration at the surface of cellulose (Stahlberg et a/., Bio/Technol. 9:286-290,1991), and/or by loosening single cellulose chains from the cellulose surface (Tormo et al., EMBO J. vol. 15, no. 21, pp. 5739-5751,1996). Most studies concerning the effects of cellulase domains on different substrates have been carried out with core proteins of cellpbiohydrolases, as their core proteins can easily be produced by limited proteolysis with papain (Tomme etal., 1988). Numerous cellulases have been described in the scientific literature, examples of which include: from Trichoderma reeseh Shoemaker, S. et al., Bio/Technology, 1:691 -696,1983, which discloses CBH1; Teeri, T. et al., Gene, 51:43-52,1987, which discloses CBH2. Cellulases from species other than Trichoderma have also been described e.g., Ool etal., Nucleic Acids Research, vol. 18, no. 19,1990, which discloses the cDNA sequence coding for endoglucanase F1-CMC. produced by Aspergillus aculeatus; Kawaguchi T etal., Gene 173(2)587-8,1996, which discloses the cloning and sequencing of the cDNA encoding beta-glucosidase 1 from Aspergillus aculeatus; Sakamoto etal., Curr. Genet. 27:435-439,1995, which discloses the cDNA sequence encoding the endoglucanase CMCase-1 from Aspergillus kawachli IFO 4308; Saarilahti etal., Gene 90:9-14,1990, which discloses an endoglucanase from Erwiriia carotovara; Spilliaert R, et al., Eur J Biochem. 224(3):923-30,1994, which discloses the cloning and sequencing of bgIA, coding for a thermostable beta-glucanase from Rhodothermus marinus; and Halldorsdottir S et al., Appl Microbiol Biotechnoi. 49(3):277-84, 1998, which discloses the cloning, sequencing and overexpression of a Rhodothermus marinus gene encoding a thermostable cellulase of glycosyl hydrolase family 12. However, there remains a need for identification and characterization of novel cellulases, with improved properties, such as improved performance under conditions of thermal stress or in the presence of surfactants, increased specific activity, altered substrate cleavage pattern, and/or high level expression In vitro.<br>
[110]   The development of new and improved cellulase compositions that comprise varying amounts CBH-type, EG-type and BG-type cellulases is of interest for use: (1) in detergent compositions that exhibit enhanced cleaning ability, function as a softening agent and/or improve the feel of cotton fabrics (e.g., "stone washing" or "biopolishing"); (2) in compositions for degrading wood pulp or other biomass into sugars (e.g., for bio-ethanol production); and/or (3) in feed compositions.<br>
IV.      MOLECULAR BIOLOGY<br>
[1ll]   In one embodiment this invention provides for the expression of variant cbh2genes under control of a promoter functional in a filamentous fungus. Therefore, this invention relies on routine techniques in the field of recombinant genetics. Basic texts disclosing the general methods of use in this invention include Sambrook et al., Molecular Cloning, A Laboratory Manual (2nd ed. 1989); Kriegler, Gene Transfer and Expression: A Laboratory Manual (1990); and Ausubel et al., eds., Current Protocols in Molecular Biology (1994)).<br>
Methods for Identifying Homologous cbh2 Genes<br>
[112]   The nucleic acid sequence for the wild type H. jecorina CBH2 is shown in Rgure 1. The invention, in one aspect, encompasses a nucleic acid molecule encoding a CBH2 homolog described herein. The nucleic acid may be a DNA molecule. [113]   Techniques that can be used to isolate CBH2-encoding DNA sequences are well known in the art and include, but are not limited to, cDNA and/or genomic library screening with a homologous DNA probe and expression screening with activity assays or antibodies against CBH2. Any of these methods can be found in Sambrook, ef al. or In CURRENT PROTOCOLS IN MOLECULAR BIOLOGY, F. Ausubel, et al., ed. Greene Publishing and Wiley-Interscience, New York (1987) ("Ausubel").<br>
Methods of Mutating cbh2 Nucleic Acid Sequences<br>
[114]   Any method known in the art that can introduce mutations is contemplated by the present invention.<br>
[115]   The present invention relates to the expression, purification and/or isolation and use of variant CBH2. These enzymes are preferably prepared by recombinant methods utilizing the cbh2 gene from H. jecorina. The fermentation broth may be used with or without purification.<br>
[116]   After the isolation and cloning of the cbh2 gene from H. jecorina, other methods known in the art, such as site directed mutagenesis, are used to make the substitutions, additions or deletions that correspond to substituted amino acids in the expressed CBH2 variant. Again, site directed mutagenesis and other methods of incorporating amino acid changes in expressed proteins at the DNA level can be found in Sambrook, et al. and Ausubel, et al.<br>
[117]   DNA encoding an amino acid sequence variant of the H. jecorina CBH2 is prepared by a variety of methods known in the art. These methods include, but are not limited to, preparation by site-directed (or oligonucleotide-mediated) mutagenesis, PCR mutagenesis, and cassette mutagenesis of an earlier prepared DNA encoding the H. jecorina CBH2.<br>
 [118]   Site-directed mutagenesis is a preferred method for preparing substitution variants. This technique is well known in the art (see, e.g.,Carter et al. Nucleic Acids Res. 13:4431-4443 (1985) and Kunkel et al., Proc. Natl. Acad.Sci.USA 82:488 (1987)). Briefly, in carrying out site-directed mutagenesis of DNA, the starting DNA is altered by first hybridizing an oligonucleotide encoding the desired mutation to a single strand of such starting DNA; After hybridization, a DNA polymerase is used to synthesize an entire second strand, using the hybridized oligonucleotide as a primer, and using the single strand of the starting DNA as a template. Thus, the oligonucleotide encoding the desired mutation is incorporated in the resulting double-stranded DNA.<br>
[119]   PCR mutagenesis is also suitable for making amino acid sequence variants of the starting polypeptide, i.e., H. jecorina CBH2. See Higuchi, in PCR Protocols, pp.177-183 (Academic Press, 1990); and Vallette et al., Nuc. Acids Res. 17:723-733 (1989). See, also, for example Cadwell et al., PCR Methods and Applications, Vol 2,28-33 (1992). Briefly, when small amounts of template DNA are used as starting material in a PCR, primers that differ slightly in sequence from the corresponding region in a template DNA can be used to generate relatively large quantities of a specific DNA fragment that differs from the template sequence only at the positions where the primers differ from the template. [120]   Another method for preparing variants, cassette mutagenesis, is based on the technique described by Wells et al., Gene 34:315-323 (1985). The starting material is the plasmld (or other vector) comprising the starting polypeptide DNA to be mutated. The codon(s) in the starting DNA to be mutated are identified. There must be a unique restriction endonuclease site on each side of the identified mutation site(s). If no such restriction sites exist, they may be generated using the above-described oligonucleotide-mediated mutagenesis method to introduce them at appropriate locations in the starting polypeptide DNA. The plasmid DNA is cut at these sites to linearize It. A double-stranded oligonucleotide encoding the sequence of the DNA between the restriction'sites but containing the desired mutation(s) is synthesized using standard procedures, wherein the two strands of the oligonucleotide are synthesized separately and then hybridized together using standard techniques. This double-stranded oligonucleotide is referred to as the cassette. This cassette is designed to have 5' and 3' ends that are compatible with the ends of the linearized plasmid, such that it can be directly ligated to the plasmid. This plasmid now contains the mutated DNA sequence.<br>
[121]   Alternatively, or additionally, the desired amino acid sequence encoding a variant CBH 2 can be determined, and a nucleic acid sequence encoding such amino acid sequence variant can be generated synthetically.<br>
 [122]   The variant CBH2(s) so prepared may be subjected to further modifications, oftentimes depending on the intended use of the cellulase. Such modifications may involve further alteration of the amino acid sequence, fusion to heterologous polypeptide(s) and/or covalent modifications.<br>
V.       cbh2 Nucleic Acids And CBH2 Polypeptides. A.       Variant cbh2-type Nucleic acids<br>
[123]   The nucleic acid sequence for the wild type H. jecorina cbh2 is shown in Rgure 1. The invention encompasses a nucleic acid molecule encoding the variant cellulases described herein. Trie nucleic acid may be a DMA molecule.<br>
[124]   After the isolation and cloning of the cbh2, other methods known in the art, such as site directed mutagenesis, are used to make the substitutions, additions or deletions that correspond to substituted amino acids in the expressed CBH2 variant. Again, site directed mutagenesis and other methods of incorporating amino acid changes in expressed proteins at the DNA level can be found in Sambrook, et al. and Ausubel, et al [125]   After DNA sequences that encode the CBH2 variants have been cloned into DNA constructs, the DNA is used to transform microorganisms. The microorganism to be transformed for the purpose of expressing a variant CBH2 according to the present invention may advantageously comprise a strain derived from Trichoderma sp. Thus, a preferred mode for preparing variant CBH2 cellulases according to the present invention comprises transforming a Trichoderma sp. host cell with a DNA construct comprising at least a fragment of DNA encoding a portion or all of the variant CBH2. The DNA construct will generally be functionally attached to a promoter. The transformed host cell is then grown under conditions so as to express the desired protein. Subsequently, the desired protein product may be purified to substantial homogeneity.<br>
[126]   However, it may in fact be that the best expression vehicle for a given DNA encoding a variant CBH2 may differ from H. jecorina. Thus, it may be that it will be most advantageous to express a protein in a transformation host that bears phylogenetic similarity to the source organism for the variant CBH2. In an alternative embodiment, Aspergillus niger can be used as an expression vehicle. For a description of transformation techniques with A. niger, see WO 98/31821, the disclosure of which is incorporated by reference in its entirety.<br>
[127]   Accordingly, the present description of a Aspergillus spp. expression system is provided for illustrative purposes only and as one option for expressing the variant CBH2 of the invention. One of skill in the art, however, may be inclined to express the DNA encoding<br>
variant CBH2 in a different host cell if appropriate and it should be understood that the source of the variant CBH2 should be considered in determining the optimal expression host. Additionally, the skilled worker in the field will be capable of selecting the best expression system for a particular gene through routine techniques utilizing the tools available in the art.<br>
B.       Variant CBH2 Polypeptides<br>
[128]   The amino acid sequence for the wild type H. jecorina CBH2 is shown in Rgure 1. The variant CBH2 polypeptides comprises a substitution or deletion at a position corresponding to one or more of residues V94, P98, G118, M120, M134, T142, L144, M145, T148, T154, L179, Q204, V206, S210,1212, T214, L215, G231, T232, V250, Q276, N285, S291, G308, T312, S316, V323, N325,1333, G334, S343, T349, G360, S380, A381, S386, F411, S413, A416, Q426 and/or A429 in CBH2 from Hypocrea jecorina (SEQ ID NO2). [129]   In one aspect the invention relates to an isolated CBH2 enzyme of the Cel6A family having at least one amino acid residue substitution or deletion in a region selected from the group consisting of (1) from position 92 to 100, (2) 115-123, (3) 140-155, (4) 160-180, (5) 198-218, (6) 228-235, (7) 240-260, (8) 275-295, (9) 305-318, (10) 322-335, (11) 340-350, (12) 360-370, (13) 378-390 and (14) 410-430. In another aspect the invention relates to an isolated CBH2 enzyme of the CelGA family having at least one amino acid residue substitution in a region selected from the group consisting of (1) from position 92 to 100, (2) 115-123, (3) 140-155, (4) 160-180, (5) 198-218, (6) 228-235, (7) 240-260, (8) 275-295, (9) 305-318, (10) 322-335, (11) 340-350, (12) 360-370, (13) 378-390 and (14) 410-430. [130]   The variant CBH2's of this invention have amino acid sequences that are derived from the amino acid sequence of a precursor CBH2. The amino acid sequence of the CBH2 variant differs from the precursor CBH2 amino acid sequence by the substitution, deletion or insertion of one or more amino acids of the precursor amino acid sequence. In a preferred embodiment, the precursor CBH2 is Hypocrea jecorina CBH2. The mature amino acid sequence of H. jecorina CBH2 is shown in Figure 1. Thus, this invention is directed to CBH2 variants which contain amino acid residues at positions which are equivalent to the particular identified residue in H. jecorina CBH2. A residue (amino acid) of an CBH2 homolog is equivalent to a residue of Hypocrea jecorina CBH2 if it is either homologous (i.e., corresponding in position in either primary or tertiary structure) or is functionally analogous to a specific residue or portion of that residue in Hypocrea jecorina CBH2 (i.e., having the same or similar functional capacity to combine, react, or interact chemically or structurally). As used herein, numbering is intended to correspond to that of the mature CBH2 amino acid sequence as illustrated in Figure 1. In addition to locations within the precursor CBH2,specific residues in the precursor CBH2 corresponding to the amino acid positions that are responsible for instability when the precursor CBH2 is under thermal stress are identified herein for substitution or deletion. The amino acid position number (e.g., +51) refers to the number assigned to the mature Hypocrea jecorina CBH2 sequence presented in Figure 1. [131]   Alignment of amino acid sequences to determine homology is preferably determined by using a "sequence comparison algorithm." Optimal alignment of sequences for comparison can be conducted, e.g., by the local homology algorithm of Smith &amp; Waterman, Adv. Appl. Math. 2:482 (1981), by the homology alignment algorithm of Needleman &amp; Wunsch, J. Mol. Biol. 48:443 (1970), by the search for similarity method of Pearson &amp; Lipman, Proc. NaflAcad. Scl USA 85:2444 (1988), by computerized implementations of these algorithms (GAP, BESTFIT, FASTA, and TFASTA in the Wisconsin Genetics Software Package, Genetics Computer Group, 575 Science Dr., Madison, Wl), or by visual inspection Visual inspection may utilize graphics packages such as, for example, MOE by Chemical Computing Group, Montreal Canada.<br>
[132]   An example of an algorithm that is suitable for determining sequence similarity is the BUST algorithm, which is described in Altschul, et ai, J. Mol. Biol. 215:403-410 (1990). Software for performing BLAST analyses is publicly available through the National Center fc Biotechnology Information (www.ncbi.nlm.nih.gov). This algorithm involves first identifying high scoring sequence pairs (HSPs) by identifying short words of length W in the query sequence that either match or satisfy some positive-valued threshold score T when aligned with a word of the same length in a database sequence! These initial neighborhood word, hits act as starting points to find longer HSPs containing them. The word hits are expanded in both directions along each of the two sequences being compared for as far as the cumulative alignment score can be increased. Extension of the word hits is stopped when: the cumulative alignment score falls off by the quantity X from a maximum achieved value; the cumulative score goes to zero or below; or the end of either sequence is reached. The BLAST algorithm parameters W, T, and X determine the sensitivity and speed of the alignment. The BLAST program uses as defaults a word length (W) of 11, the BLOSUM62 scoring matrix (see Henikoff &amp; Henikoff, Proc. Natl. Acad. Sci. USA 89:10915 (1989)) alignments (B) of 50, expectation (E) of 10, M'5, N'-4, and a comparison of both strands. [133]   The BLAST algorithm then performs a statistical analysis of the similarity between two sequences (see, e.g., Karlin &amp; Altschul, Proc. Nat l. Acad. Scl. USA 90:5873-5787 (1993)). One measure of similarity provided by the BLAST algorithm is the smallest sum probability (P(N)), which provides an indication of the probability by which a match between two nucleotide or amino acid sequences would occur by chance. For example, an amino<br>
acid sequence is considered similar to a protease if the smallest sum probability in a comparison of the test amino acid sequence to a protease amino acid sequence is less than about 0.1, more preferably less than about 0.01, and most preferably less than about 0.001. [134]   For purposes of the present invention, the degree of identity may be suitably determined by means of computer programs known in the art  Such as GAP provided in the GCG program package (Program Manual for the Wisconsin Package, Version 8, August 1994, Genetics Computer Group, 575 Science Drive, Madison, Wis., USA 53711) (Needleman, S. B. and Wunsch, C. D., (1970), Journal of Molecular Biology, 48,443-45), using GAP with the following settings for poiynucleotide sequence comparison: GAP creation penalty of 5.0 and GAP extension penalty of 0.3.<br>
[135]   Sequence searches are typically carried out using the BLASTN program when evaluating a given nucleic acid sequence relative to nucleic acid sequences in the GenBank DNA Sequences and other public databases. The BLASTX program is preferred for searching nucleic acid sequences that have been translated in all reading frames against amino acid sequences in the GenBank Protein Sequences and other public databases. Both<br>
	'	,<br>
BLASTN and BLASTX are run using default parameters of an open gap penalty of 11.0, and an extended gap penalty of 1.0, and utilize the BLOSUM-62 matrix. (See, e.g., Altschul, ef a/., 1997.)<br>
[136]   Additional specific strategies for modifying stability of CBH2 cellulases are provided below.<br>
[137]   (1)      Decreasing the entropy of main-chain unfolding may introduce stability to the enzyme. For example, the introduction of praline residues may significantly stabilize the protein by decreasing the entropy of the unfolding (see, e.g., Watanabe, et al., Eur. J. Biochem. 226:277-283 (1994)). Similarly, glycine residues have no fi-carbon, and thus have considerably greater backbone conformational freedom than many other residues. Replacement of glycines, preferably with alanines, may reduce the entropy of unfolding and improve stability (see, e.g., Matthews, ef a/., Proc. Natl. Acad. Sci. USA 84; 6663-6667 (1987)). Additionally, by shortening external loops it may be possible to improve stability. It has been observed that hyperthermophile produced proteins have shorter external loops than their mesophilic homologues (see, e.g., Russel, ef al., Current Opinions in Biotechnology 6:370-374 (1995)). The introduction of disulfide bonds may also be effective to stabilize distinct tertiary structures in relation to each other. Thus, the introduction of cysteines at residues accessible to existing cysteines or the introduction of pairs of cysteines that could form disulfide bonds would alter the stability of a CBH2 variant<br>
 [138]   (2)      Decreasing internal cavities by increasing side-chain hydrophobicity may alter the stability of an enzyme. Reducing the number and volume of internal cavities increases the stability of enzyme by maximizing hydrophobic interactions and reducing packing defects (see, e.g., Matthews, Ann. Rev. Biochem. 62:139-160 (1993); Burtey, et al, Science 22953-29 (1985); Zuber, Biophys. Chem. 29:171-179 (1988); Kellls, etal., Nature 333:784-786 (1988)). It is known that multimeric proteins from thermophiles often have more hydrophobic sub-unit Interfaces with greater surface complementarity than their mesophilic counterparts (Russel, et a/., supra). This principle is believed to be applicable to domain interfaces of monomeric proteins. Specific substitutions that may improve stability by increasing hydrophobicity include lysine to arginine, serine to alanine and threonine to alanine (Russel, et al,, supra). Modification by substitution to alanine or proline may increase side-chain size with resultant reduction in cavities, better packing and increased hydrophobicity. Substitutions to reduce the size of the cavity, increase hydrophobicity and improve the complementarity the interfaces between the domains of CBH2 may improve stability of the enzyme. Specifically, modification of the specific residue at these positions with a different residue selected from any of phenylalanine, tryptophan, tyrosine, leucine and isoleucine may improve performance.<br>
[139]   (3)      Balancing charge in rigid secondary structure, i.e., a-helices and β-tums may improve stability. For example, neutralizing partial positive charges on a helix N-terminus with negative charge on aspartic acid may improve stability of the structure (see, e.g., Eriksson, etal., Science255:178-183 (1992)). Similarly, neutralizing partial negative charges on helix C-terminus with positive charge may improve stability. Removing positive charge from interacting with peptide N-terminus in β-turns should be effective in conferring tertiary structure stability. Substitution with a non-positively charged residue could remove an unfavorable positive charge from interacting with an amide nitrogen present in a turn.. [140]   (4)      Introducing salt bridges and hydrogen bonds to stabilize tertiary structures may be effective. For example, ion pair interactions, e.g., between aspartic acid or glutamic acid and lysine, arginine or histidine, may introduce strong stabilizing effects and may be used to attach different tertiary structure elements with a resultant improvement in thermostability. Additionally, increases in the number of charged residue/non-charged residue hydrogen bonds, and the number of hydrogen-bonds generally, may improve thermostability (see, e.g., Tanner, etal., Biochemistry 35:2597-2609 (1996)). Substitution with aspartic acid, asparagine, glutamic acid or glutamine may introduce a hydrogen bond with a backbone amide. Substitution with arginine may improve a salt bridge and introduce an H-bond into a backbone carbonyl.<br>
 [141]   (5)      Avoiding thermolabile residues in general may increase thermal stability. For example, asparagine and glutamine are susceptible to deamidation and cysteine is susceptible to oxidation at high temperatures. Reducing the number of these residues in sensitive positions may result in improved thermostability (Russet, et al, supra). Substitution or deletion by any residue other than glutamine or cysteine may increase stability by avoidance of a thermolabile residue.<br>
[142]   (6)      Stabilization or destabilization of binding of a ligand that confers modified stability to CBH2 variants. For example, a component of the matrix in which the CBH2 variants of this invention are used may bind to a specific surfactant/thermal sensitivity site of the CBH2 variant. By modifying the site through substitution, binding of the component to the variant may be strengthened or diminished. For example, a non-aromatic residue in the binding crevice of CBH2 may be substituted with phenylalanine or tyrosine to introduce aromatic side-chain stabilization where interaction of the cellulose substrate may interact favorably with the benzyl rings, increasing the stability of the CBH2 variant. [143]   (7)      Increasing the electronegativity of any of the surfactant/ thermal sensitivity ligands may improve stability under surfactant or thermal stress. For example, substitution with phenylalanine or tyrosine may increase the electronegativity of D (aspartate) residues by improving shielding from solvent, thereby improving stability.<br>
VI.      Expression Of Recomblnant CBH2 Variants<br>
[144] The methods of the invention rely on the use cells to express variant CBH2, with no particular method of CBH2 expression required. The CBH2 is preferably secreted from the cells.<br>
[145]   The invention provides host cells which have been transduced, transformed or transfected with an expression vector comprising a variant CBH-encoding nucleic acid sequence. The culture conditions, such as temperature, pH and the like, are those previously used for the parental host cell prior to transduction, transformation or transfection and will be apparent to those skilled in the art.<br>
[146]   !n one approach, a filamentous fungal cell or yeast cell is transfected with an expression vector having a promoter or biologically active promoter fragment or one or more (e.g., a series) of enhancers which functions in the host cell line, operably linked to a DNA segment encoding CBH2, such that CBH2 is expressed in the cell line.<br>
A.       Nucleic Acid Constructs/Expression Vectors.<br>
[147]   Natural or synthetic poiynucleotide fragments encoding CBH2 ("CBH2-encoding nucleic acid sequences") may be incorporated into heteroiogous nucleic acid constructs or<br>
vectors, capable of introduction into, and replication in, a filamentous fungal or yeast cell. The vectors and methods disclosed herein are suitable for use in host cells for the expression of CBH2. Any vector may be used as long as it is replicable and viable in the ceils into which it is introduced. Large numbers of suitable vectors and promoters are known to those of skill in the art, and are commercially available. Cloning and expression vectors are also described in Sambrook et al., 1989, Ausubel FM et ai, 1989, and Strathem et al., The Molecular Biology of the Yeast Saccharomyces, 1981, each of which is expressly incorporated by reference herein. Appropriate expression vectors for fungi are described in van den Hondel, C.A.M.J J. et al. (1991) In: Bennett, J.W. and Lasure, L.L (eds.) More Gene Manipulations in Fungi. Academic Press, pp. 396-428. The appropriate DNA sequence may be inserted Into a plasmid or vector (collectively referred to herein as "vectors") by a variety of procedures. In general, the DNA sequence is inserted into an appropriate restriction endonuclease site(s) by standard procedures. Such procedures and related sub-cloning procedures are deemed to be within the scope of knowledge of those skilled in the art.<br>
[148]   Recombinant filamentous fungi comprising the coding sequence for variant CBH2 may be produced by introducing a heterologous nucleic acid construct comprising the variant CBH2 coding sequence into the cells of a selected strain of the filamentous fungi. [149]   Once the desired form of a variant cbh2 nucleic acid sequence is obtained, it may be modified in a variety of ways. Where the sequence involves non-coding flanking regions, the flanking regions may be subjected to resection, mutagenesis, etc. Thus, transitions, transversions, deletions, and insertions may be performed on the naturally occurring sequence.<br>
[150]   A selected variant cbh2 coding sequence may be inserted into a suitable vector according to well-known recombinant techniques and used to transform filamentous fungi capable of CBH2 expression. Due to the inherent degeneracy of the genetic code, other nucleic acid sequences which encode substantially the same or a functionally equivalent amino acid sequence may be used to clone and express variant CBH2. Therefore it is appreciated that such substitutions in the coding region fall within the sequence variants covered by the present invention. Any and all of these sequence variants can be utilized in the same way as described herein for a parent CBH2-encoding nucleic acid sequence. [151]   The present invention also includes recombinant nucleic acid constructs comprising one or more of the variant CBH2-encoding nucleic acid sequences as described above. The constructs comprise a vector, such as a plasmid or viral vector, into which a sequence of the invention has been inserted, in a forward or reverse orientation.<br>
 [152]   Heterologous nucleic acid constructs may include the coding sequence for variant cbh2. (i) in isolation; (ii) in combination with additional coding sequences; such as fusion protein or signal peptide coding sequences, where the cbh2 coding sequence is the dominant coding sequence; (iii) in combination with non-coding sequences, such as introns and control elements, such as promoter and terminator elements or 5' and/or 3' untranslated regions, effective for expression of the coding sequence in a suitable host; and/or (iv) in a vector or host environment in which the cbh2 coding sequence is a heterologous gene. [153]   In one aspect of the present invention, a heterologous nucleic acid construct is employed to transfer a variant CBH2-encoding nucleic acid sequence into a cell in vitro, with established filamentous, fungal and yeast lines preferred. For long-term, production of variant CBH2, stable expression is preferred. It follows that any method effective to generate stable transformants may be used in practicing the invention. [154]   Appropriate vectors are typically equipped with a selectable marker-encoding nucleic acid sequence,, insertion sites, and suitable control elements, such as promoter and termination sequences. The vector may comprise regulatory sequences, including, for example, non-coding sequences, such as introns and control elements, i.e., promoter and terminator elements or 5' and/or 3' untranslated regions, effective for expression of the coding sequence in host cells (and/or in a vector or host cell environment in which a modified soluble protein antigen coding sequence is not normally expressed), operably linked to the coding sequence. Large numbers of suitable vectors and promoters are known to those of skill in the art, many of which are commercially available and/or are described in Sambrook, etal., (supra).<br>
[155]   Exemplary promoters include both constitutive promoters and inducible promoters, examples of which include a CMV promoter, an SV40 early promoter, an RSV promoter, an EF-1a promoter, a promoter containing the tet responsive element (TRE) in the tet-on or tet-off system as described (ClonTech and BASF), the beta actin promoter and the metallothionine promoter that can upregulated by addition of certain metal salts. A promoter sequence is a DNA sequence which is recognized by the particular filamentous fungus for expression purposes. It is operabiy linked to DNA sequence encoding a variant CBH2 polypeptide. Such linkage comprises positioning of the promoter with respect to the Initiation codon of the DNA sequence encoding the variant CBH2 polypeptide in the disclosed expression vectors. The promoter sequence contains transcription and translation control sequence which mediate the expression of the variant CBH2 polypeptide. Examples include the promoters from the Aspergillus niger, A awamorior A oryzae glucoamylase, alpha-amylase, or alpha-glucosidase encoding genes; the A nidulans gpdA or trpC Genes; the<br>
Neurospora crassa cbhl or trpl genes; the A. nigeror Rhizomucor miehei aspartc proteinase encoding genes; the H.jecorina (T. reesei) cbhl, cbh2, egh, eg12, or other cellulase encoding genes.<br>
[156]   The choice of the proper selectable marker will depend on the host ceil, and appropriate markers for different hosts are well known in the art. Typical selectable marker genes include argB from A nidulans or 7". reesei, amdS from A nidulans, pyr4 from Neurospora crassa or T. reesei, pyrG from Aspergillus nigeror A. nidulans. Additional exemplary selectable markers include, but are not limited to trpc, trpl, 011C31, niaD or Ieu2, which are included in heterologous nucleic acid constructs used to transform a mutant strain such as trp-, pyr-, leu-and the like.<br>
[157]   Such selectable markers confer to transformants the ability to utilize a metabolite that is usually not metabolized by the filamentous fungi. For example, the amdS gene from H. jecorina which encodes the enzyme acetamidase that allows transf ormant cells to grow on acetamide as a nitrogen source. The selectable marker (e.g. pyrG) may restore the ability of an auxotrophic mutant strain to grow on a selective minima! medium or the selectable marker (e.g. olic31) may confer to transformants the ability to grow in the presence of an inhibitory drug or antibiotic.<br>
[158]   The selectable marker coding sequence is cloned into any suitable plasmid using methods generally employed in the art. Exemplary plasmids include pUC18, pBR322, pRAX and pUCIOO. The pRAX plasmid contains AMA1 sequences from A. nidulans, which make it possible to replicate in A. niger.<br>
[159]   The practice of the present invention will employ, unless otherwise indicated, conventional techniques of molecular biology, microbiology, recombinant DNA, and immunology, which are within the skill of the art. Such techniques are explained fully in the literature. See, for example, Sambrook et al, 1989; Freshney, Animal Cell Culture, 1987; Ausubel, et a/., 1993; and Coligan er a/., Current Protocols in Immunology, 1991.<br>
B.       Host Cells and Culture Conditions For CBH2 Production<br>
(0       Filamentous Fungl<br>
[160]   Thus, the present invention provides filamentous fungi comprising cells which have been modified, selected and cultured in a manner effective to result in variant CBH2 production or expression relative to the corresponding non-transformed parental fungi. [161]   Examples of species of parental filamentous fungi that may be treated and/or modified for variant CBH2 expression include, but are not limited to Trichoderma, e.g., Trichoderma reesei, Trichoderma longibrachiatum, Trichoderma viride, Trichoderma<br>
koningii; Penicllllum sp., Humlcola sp., including Humicola insolens; Aspergillus sp., Chrysosporium sp., Fusarium sp., Hypocreasp., and Emericella sp. [162]   CBH2 expressing cells are cultured under conditions typically employed to culture the parental fungal line. Generally, cells are cultured in a standard medium containing physiological salts and nutrients, such as described in Pourquie, J. et a/., Biochemistry and Genetics of Cellulose Degradation, eds. Aubert, J. P. et a/., Academic Press, pp. 71 -86, 1988 and llmen, M. et al., Appl. Environ. Microbiol. 63:1298-1306,1997. Culture conditions are also standard, e.g., cultures are incubated at 28°C in shaker cultures or fermenters until desired levels of CBH2 expression are achieved.<br>
[163]   Preferred culture conditions for a given filamentous fungus may be found in the scientific literature and/or from the source of the fungi such as the American Type Culture Collection (ATCC; www.atcc.org/). After fungal growth has been established, the cells are exposed to conditions effective to cause or permit the expression of variant CBH2. [164]   In cases where a CBH2 coding sequence is under the control of an inducible promoter, the inducing agent, e.g., a sugar, metal salt or antibiotics, is added to the medium at a concentration effective to induce CBH2 expression.<br>
[165]   In one embodiment, the strain comprises Aspergillus niger, which is a useful strain for obtaining overexpressed protein. For example A. niger varawamori dgr246 is known to secrete elevated amounts of secreted cellulases (Goedegebuur etal, Curr. Genet (2002) 41: 89-98). Other strains of Aspergillus niger var awamori such as GCDAP3, GCDAP4 and GAP3-4 are known Ward et a/(Ward, M, Wilson, L.J. and Kodama, K.H., 1993, Appl. Microbiol. Biotechnol. 39:738-743).<br>
[166]   In another embodiment, the strain comprises Trichoderma reesei, which is a useful strain for obtaining overexpressed protein. For example, RL-P37, described by Sheir-Neiss, etal., Appl. Microbiol. Biotechnol. 20:46-53 (1984) is known to secrete elevated amounts of cellulase enzymes. Functional equivalents of RL-P37 include Trichoderma reesei strain RUT-C30 (ATCC No. 56765) and strain QM9414 (ATCC No. 26921). It is contemplated that these strains would also be useful in overexpressing variant CBH2. [167]   Where it is desired to obtain the variant CBH2 in the absence of potentially detrimental native cellulolytic activity, it is useful to obtain a Trichoderma host cell strain which has had one or more cellulase genes deleted prior to introduction of a DNA construct or plasmid containing the DNA fragment encoding the variant CBH2. Such strains may be prepared by the method disclosed in U.S. Patent No. 5,246,853 and WO 92/06209, which disclosures are hereby incorporated by reference. By expressing a variant CBH2 cellulase in a host microorganism that is missing one or more cellulase genes, the identification and<br>
subsequent purification procedures are simplified. Any gene from Trichoderma sp. which has been cloned can be deleted, for example, the cbhl, cbh2, eg11 and eg12 genes as well as those encoding EG III and/or EGV protein (see e.g., U.S. Patent No. 5,475,101 and WO 94/28117, respectively).<br>
[168]   Gene deletion may be accomplished by inserting a form of the desired gene to be deleted or disrupted into a piasmid by methods known in the art The deletion plasmid is then cut at an appropriate restriction enzyme stte(s), Internal to the desired gene coding region, and the gene coding sequence or part thereof replaced with a selectable marker. Flanking DNA sequences from the locus of the gene to be deleted or disrupted, preferably between about 0.5 to 2.0 kb, remain .on either side of the selectable marker gene. An appropriate deletion plasmid will generally have unique restriction enzyme sites present therein to enable the fragment containing the deleted gene, including flanking DNA sequences, and the selectable marker gene to be removed as a single linear piece. [169]   A selectable marker must be chosen so as to enable detection of the transformed microorganism. Any selectable marker gene that is expressed in the selected<br>
i<br>
microorganism will be suitable. For example,.with Aspergillus sp., the selectable marker is chosen so that the presence of the selectable marker in the transformants will not significantly affect the properties thereof. Such a selectable marker may be a gene that encodes an assayable product. For example, a functional copy of a Aspergillus sp. gene may be used which if lacking in the host strain results in the host strain displaying an auxotrophic phenotype. Similarly, selectable markers exist for Trichoderma sp. [170]   In one embodiment, a pyrG derivative strain of Aspergillus sp. is transformed with a functional pyrG gene, which thus provides a selectable marker for transformation. A pyrG  , derivative strain may be obtained by selection of Aspergillus sp. strains that are resistant to f luoroorotic acid (FOA). The pyrG gene encodes orotidine-5'-monophosphate decarboxylase, an enzyme required for the biosynthesis of uridine. Strains with an intact pyrG gene grow in a medium lacking uridine but are sensitive to fluoroorotic acid. It is possible to select pyrG" derivative strains that lack a functional orotidine monophosphate decarboxylase enzyme and require uridine for growth by selecting for FOA resistance. Using the FOA selection technique it is also possible to obtain uridine-requiring strains which lack a functional orotate pyrophosphoribosyl transferase. It is possible to transform these cells with a functional copy of the gene encoding this enzyme (Berges &amp; Barreau, Curr. Genet. 19:359-365 (1991), and van Hartingsveldt et al., (1986) Development of a homologous transformation system for Aspergillus niger based on the pyrG gene. Mol. Gen. Genet 206:71-75). Selection of derivative strains is easily performed using the FOA<br>
resistance technique referred to above, and thus, the pyrG gene is preferably employed as a selectable marker.<br>
[171]   In a second embodiment, a pyr4~ derivative strain of Hyprocrea sp. (Hyprocrea sp. (Trichoderma sp.)) is transformed with a functional pyr4 gene, which thus provides a selectable marker for transformation. A pyr4 derivative strain may be obtained by selection of Hyprocrea sp. (Trichoderma sp.) strains that are resistant to fluoroorotic add (FOA). The pyr4 gene encodes orotidine-5'-monophosphate decarboxylase, an enzyme required for the biosynthesis of uridine. Strains with an intact pyr4 gene grow in a medium lacking uridine but are sensitive to fluoroorotic acid. It is possible to select pyrf derivative strains that lack a functional orotidine monophosphate decarboxylase enzyme and require uridine for growth by selecting for FOA resistance. Using the FOA selection technique it is also possible to obtain uridine-requiring strains which lack a functional orotate pyrophosphoribosyl transferase. It is possible to transform these cells with a functional copy of the gene encoding this enzyme/Berges &amp; Barreau, 1991). Selection of derivative strains is easily performed using the FOA resistance technique referred to above, and thus, the pyr4 gene is preferably employed as a selectable marker.<br>
[172]   To transform pyrG Aspergillus sp. or pyr4~ Hyprocrea sp. (Trichoderma sp.) so as to be lacking in the ability to express one or more cellulase genes, a single DMA fragment comprising a disrupted or deleted cellulase gene is then isolated from the deletion plasmid and used to transform an appropriate pyf Aspergillus or pyf Trichoderma host. Transformants are then identified and selected based on their ability to express the pyrG or pyr4, respectively, gene product and thus compliment the uridine auxotrophy of the host strain. Southern blot analysis is then carried out on the resultant transformants to identify and confirm a double crossover integration event that replaces part or all of the coding region of the genomic copy of the gene to be deleted with the appropriate pyr selectable markers.<br>
[173]   Although the specific plasmid vectors described above relate to preparation of pyf transformants, the present invention is not limited to these vectors. Various genes can be deleted and replaced in the Aspergillus sp. or Hyprocrea sp. (Trichoderma sp.) strain using the above techniques. In addition, any available selectable markers can be used, as discussed above. In fact, any host, e.g., Aspergillus sp. or Hyprocrea sp., gene that has been cloned, and thus identified, can be deleted from the genome using the above-described strategy.<br>
[174]   As stated above, the host strains used may be derivatives of Hyprocrea sp. (Trichoderma sp.) that lack or have a nonfunctional gene or genes corresponding to the<br>
selectable marker chosen. For example, if the selectable marker of pyrG is chosen for Aspergillus sp., then a specific pyrG derivative strain is used as a recipient in the transformation procedure. Also, for example, if the selectable marker of pyr4 is chosen for a Hyprocrea sp., then a specific pyrf derivative strain is used as a recipient In the transformation procedure. Similarly, selectable markers comprising Hyprocrea sp. (Trichoderma sp.) genes equivalent to the Aspergillus nldulans genes amdS, argB, trpC, niaD may be used. The corresponding recipient strain must therefore be a derivative strain such as argB, trpC, niaD, respectively.<br>
[175]   DMA encoding the CBH2 variant is then prepared for insertion into an appropriate microorganism. According to the present invention, DNA encoding a CBH2 variant comprises the DNA necessary to encode for a protein that has functional cellulolytic activity. The DNA fragment encoding the CBH2 variant may be functionally attached to a fungal promoter sequence, for example, the promoter of the glaA gene in Aspergillus or the promoter of the cbhl or eg!1 genes in Trichoderma.<br>
[176]   It is also contemplated that more than one copy of DNA encoding a CBH2 variant may be recombined into the strain to facilitate overexpression. The DNA encoding the CBH2 variant may be prepared by the construction of an expression vector carrying the DNA encoding the variant. The expression vector carrying the inserted DNA fragment encoding the CBH2 variant may be any vector which is capable of replicating autonomously in a given host organism or of integrating into the DNA of the host, typically a plasmid. In preferred' embodiments two types of expression vectors for obtaining expression of genes are contemplated. The first contains DNA sequences in which the promoter, gene-coding region, and terminator sequence all originate from the gene to be expressed. Gene truncation may be obtained where desired by deleting undesired DNA sequences (e.g., coding for unwanted domains) to leave the domain to be expressed under control of its own transcriptional and translational regulatory sequences. A selectable marker may also be contained on the vector allowing the selection for integration into the host of multiple copies of the novel gene sequences.<br>
[177]   The second type of expression vector is preassembied and contains sequences required for high-level transcription and a selectable marker.. It is contemplated that the coding region for a gene or part thereof can be inserted into this general-purpose expression vector such thai it is under the transcriptional control of the expression cassettes promoter and terminator sequences.<br>
[178]   For example, in Aspergillus, pRAX is such a general-purpose expression vector. Genes or part thereof can be inserted downstream of the strong glaA promoter.<br>
 [179]   For example, in Hypocrea, pTEX is such a general-purpose expression vector. Genes or part thereof can be inserted downstream of the strong cbh1 promoter. [180]   In the vector, the DNA sequence encoding the CBH2 variant of the present invention should be operably linked to transcriptional and translational sequences, i.e., a suitable promoter sequence and signal sequence in reading frame to the structural gene. The promoter may be any DNA sequence that shows transcriptional activity in the host cell and may be derived from genes encoding proteins either homologous or heterologous to the host cell. .An optional signal peptide provides for extracellular production of the CBH2 variant. The DNA encoding the signal sequence is preferably that which is naturally associated with the gene to be expressed, however the signal sequence from any suitable source, for example an exo-cellobiohydrolase or endoglucanase from Trichoderma, is contemplated in the present invention.<br>
[181]   The procedures used to ligate the DNA sequences coding for the variant CBH2 of the present invention with the promoter, and insertion into suitable vectors are well known in the art.<br>
[182]   The DNA vector or construct described above may be introduced in the host cell in accordance with known techniques such as transformation, transfection, microinjection, microporation, biolistic bombardment and the like.<br>
[183]   In the preferred transformation technique, it must be taken into account that the permeability of the cell wall to DNA in Hyprocrea sp. (Trichoderma sp.) Is very low. Accordingly, uptake of the desired DNA sequence, gene or gene fragment is at best minimal. There are a number of methods to increase the permeability of the Hyprocrea sp. (Trichoderma sp.) cell wall in the derivative strain (i.e., lacking a functional gene corresponding to the used selectable marker) prior to the transformation process. [184]   The preferred method in the present invention to prepare Aspergillus sp. or Hyprocrea sp. (Trichoderma sp.) for transformation involves the preparation of protoplasts from fungal mycelium. See Campbell et al. improved transformation efficiency of A.nigef using homologous niaD gene for nitrate reductase. Curr. Genet. 16:53-56; 1989. The mycelium can be obtained from germinated vegetative spores. The mycelium is treated with an enzyme that digests the cell wall resulting in protoplasts. The protoplasts are then protected by the presence of an osmotic stabilizer in the suspending medium. These stabilizers include sorbitol, mannitol, potassium chloride, magnesium sulfate and the like. Usually the concentration of these stabilizers varies between 0.8 M and 1.2 M. It is preferable to use about a 1.2 M solution of sorbitol in the suspension medium.<br>
 [185]   Uptake of the DNA into the host strain, (Aspergillus sp. or Hyprocrea sp. (Trichoderma sp.), Is dependent upon the calcium ion concentration. Generally between about 10mMCaCI2 and 50 mMCaC12 is used in an uptake solution. Besides the need for the calcium ion in the uptake solution, other items generally included are a buffering system such as TE buffer (10 Mm Tris, pH 7.4; 1 mM EDTA) or 10 mM MOPS, pH 6.0 buffer (morpholinepropanesulfonte acid) and polyethylene glycol (PEG). It is believed that the polyethylene glycol acts to fuse the cell membranes thus permitting the contents of the medium to be delivered into the cytoplasm of the host cell, by way of example either Aspergillus sp. or Hyprocrea sp. strain, and the plasmid DNA is transferred to the nucleus. This fusion frequently leaves multiple copies of the plasmid DNA integrated into the host chromosome.<br>
[186]   Usually a suspension containing the Aspergillus sp. protoplasts or cells that have been subjected to a permeability treatment at a density of 105 to 106/mL, preferably 2 x 105/mL are used in transformation. Similarly, a suspension containing the Hyprocrea sp. (Trichoderma spj protoplasts or cells that have been subjected to a permeability treatment at a density of 10s to 109/mL, preferably 2 x 108/mL are used in transformation. A volume of 100 pL of these protoplasts or cells in an appropriate solution (e.g., 1.2 M sorbitol; 50 mM CaCI2) are mixed with the desired DNA. Generally a high concentration of PEG is added to the uptake solution. From 0.1 to 1 volume of 25% PEG 4000 can be added to the protoplast suspension. However, it is preferable to add about 0.25 volumes to the protoplast suspension. Additives such as dimethyl sulfoxide, heparin, spermidine, potassium chloride and the like may also be added to the uptake solution and aid in transformation. [187]   Generally, the mixture is then incubated at approximately 0°C for a period of between 10 to 30 minutes. Additional PEG is then added to the mixture to further enhance the uptake of the desired gene or DNA sequence. The 25% PEG 4000 is generally added in volumes of 5 to 15 times the volume of the transformation mixture; however, greater and lesser volumes may be suitable. The 25% PEG 4000 is preferably about 10 times the volume of the transformation mixture. After the PEG is added, the transformation mixture is then incubated either at room temperature or on ice before the addition of a sorbitol and CaCI2 solution. The protoplast suspension is then further added to molten aliquots of a growth medium. This growth medium permits the growth of transformants only. Any growth medium can be used in the present invention that is suitable to grow the desired transformants. However, if Pyr* transformants are being selected it is preferable to use a growth medium that contains no uridine. The subsequent colonies are transferred and purified on a growth medium depleted of uridine.<br>
 [188]   At this stage, stable transformants may be distinguished from unstable transformants by their faster growth rate and, in Trichoderma, for example, the formation of circular colonies with a smooth, rather than ragged outline on solid culture medium lacking uridine. Additionally, in some cases a further test of stability may made by growing the transformants on solid non-selective medium (i.e. containing uridine), harvesting spores from this culture medium and determining the percentage of these spores which will subsequently germinate and grow on selective medium lacking uridine.<br>
[189]   In a particular embodiment of the above method, the CBH2 variant(s) are recovered in active form from the host cell after growth in liquid media as a result of the appropriate post translational processing of the CBH2 variant.<br>
(il)      Yeast<br>
[190]   The present Invention also contemplates the use of yeast as a host cell for CBH2 production. Several other genes encoding hydrolylic enzymes have been expressed in various strains of the yeast S. cerevisiae. These include sequences encoding for two endoglucanases (Penttila et al., Yeast vol. 3, pp 175-185,1987), two cellobiohydrolases (Penttila etal., Gene, 63:103-112,1988) and one beta-glucosidase from Trichoderma reesei (Cummings and Fowler, Curr. Genet. 29:227-233,1996), a xylanase from Aureobasidlium pullulans(U and Ljungdahl, Appl. Environ. Microbiol. 62, no. 1, pp. 209-213,1996), an alpha-amylase from wheat (Rothstein etal., Gene 55:353-356,1987), etc. In addition, a cellulase gene cassette encoding the Butyrivibrio flbrisolvens endo- [beta] -1,4-glucanase (END1), Phanerochaete chrysosporium cellobiohydrolase (CBH1), the Rumlnococcus flavefaciens cellodextrinase (CEL1) and the Endomyces fibrilizer cellobiase (Bgl1) was successfully expressed in a laboratory strain of S. cerevisiae (Van Rensburg et al., Yeast, vol. 14, pp. 67-76,1998).<br>
C.       Introduction of a CBH2-Encoding Nucleic Acid Sequence into Host Cells.<br>
[191]   The invention further provides cells and cell compositions which have been genetically modified to comprise an exogenously provided variant CBH2-encoding nucleic acid sequence. A parental cell or cell line may be genetically modified (i.e., transduced, transformed or transfected) with a cloning vector or an expression vector. The vector may be, for example, in the form of a plasmid, a viral particle, a phage, etc, as further described above.<br>
[192]   The methods of transformation of the present invention may result in the stable integration of all or part of the transformation vector into the genome of the filamentous<br>
fungus. However, transformation resulting in the maintenance of a self-replicating extra-chromosomal transformation vector is also contemplated.<br>
[193]   Many standard transfection methods can be used to produce Trichoderma reesel cell lines that express large quantities of the heterologus protein. Some of the published methods for the introduction of DNA constructs into cellulase-producing strains of Trichoderma include Lorito, Hayes, DiPietro and Harman, 1993, Curr. Genet 24:349-356; Goldman, VanMontagu and Herrera-Estrella, 1990, Curr. Genet 17:169-174; Penttila, Nevalainen, Ratto, Salminen and Knowles, 1987, Gene 6:155-164, for Asperglllus Yelton, Hamer and Timberlake, 1984, Proc. Natl. Acad. Sci. USA 81:1470-1474, for Fusarium Bajar, Podila and Kolattukudy, 1991, Proc. Natl. Acad. Sci. USA 88:8202-8212, for Streptomyces Hopwood et al., 1985, The John Innes Foundation, Norwich, UK and for Bacillus Brigidi, DeRossi, Bertarini, Riccardi and Matteuzzi, 1990, FEMS Microbiol. Lett 55: 135-138).<br>
[194]   Other methods for introducing a heterologous nucleic acid construct (expression vector) into filamentous fungi (e.g., H. jecorlna) include, but are not limited to the use of a particle or gene gun, permeabiiization of filamentous fungi cells walls prior to the transformation process (e.g., by use of high concentrations of alkali, e.g., 0.05 M to 0.4 M CaC12 or lithium acetate), protoplast fusion or Agrobacterium mediated transformation. An exemplary method for transformation of filamentous fungi by treatment of protoplasts or spheroplasts with polyethylene glycol and CaCI2 is described in Campbell, E.I. et al., Curr. Genet. 16:53-56,1989 and Penttila, M. 01 al., Gene, 63:11-22,1988. [195]   Any of the well-known procedures for introducing foreign nucleotide sequences into host cells may be used. These include the use of calcium phosphate transfection, polybrene, protoplast fusion, electroporation, biolistics, liposomes, microinjection, plasma vectors, viral vectors and any of the other well known methods for introducing cloned genomic DNA, cDNA, synthetic DNA or other foreign genetic material into a host cell (see, e.g., Sambrook et al., supra). Also of use is the Agrobacterium-mediated transfection method described in U.S. Patent No. 6,255,115. It is only necessary that the particular genetic engineering procedure used be capable of successfully introducing at least one gene into the host cell capable of expressing the heterologous gene. [196]   In addition, heterologous nucleic acid constructs comprising a variant CBH2-encoding nucleic acid sequence can be transcribed in vitro, and the resulting RNA introduced into the host cell by well-known methods, e.g., by injection. [197]   The invention further includes novel and useful transformants of filamentous fungi such as H.jecorina and A. niger for use in producing fungal cellulase compositions. The<br>
invention includes transformants of filamentous fungi especially fungi comprising the variant<br>
CBH2 coding sequence, or deletion of the endogenous cbh coding sequence.<br>
[198]   Following introduction of a heterologous nucleic acid construct comprising the coding<br>
sequence for a variant cbh2, the genetically modified cells can be cultured in conventional<br>
nutrient media modified as appropriate for activating promoters, selecting transformants or<br>
amplifying expression of a variant CBH2-encoding nucleic acid sequence. The culture<br>
conditions, such as temperature, pH and the like, are those previously used for the host cell<br>
selected for expression, and will be apparent to those skilled in the art.<br>
[199]   The progeny of cells into which such heterologous nucleic acid constructs have been<br>
introduced are generally considered to comprise the variant CBH2-encoding nucleic acid<br>
sequence found in the heterologous nucleic acid construct.<br>
[200]   The invention further includes novel and useful transformants of filamentous fungi<br>
such as H. jecorina for use in producing fungal cellulase compositions. Aspergillus niger<br>
may also be used in producing the variant CBH2. The invention includes transformants of<br>
filamentous fungi especially fungi comprising the variant cbh 2 coding sequence, or deletion<br>
of the endogenous cbh2 coding sequence.<br>
[201]   Stable transformants of filamentous fungi can generally be distinguished from<br>
unstable transformants by their faster growth rate and, in Trichoderma, for example, the<br>
formation of circular colonies with a smooth rather than ragged outline on solid culture<br>
medium. Additionally, in some cases, a further test of stability can be made by growing the<br>
transformants on solid non-selective medium, harvesting the spores from this culture<br>
medium and determining the percentage of these spores which will subsequently germinate<br>
and grow on selective medium.<br>
VII.     Analysis For CBH2 Nucleic Acid Coding Sequences and/or Protein Expression. [202]   In order to evaluate the expression of a variant CBH2 by a cell line that has been transformed with a variant CBH2-encoding nucleic acid construct, assays can be carried out at the protein level, the RNA level or by use of functional bioassays particular to cellobiohydrolase activity and/or production.<br>
[203]   In one exemplary application of the variant cbh2 nucleic acid and protein sequences described herein, a genetically modified strain of filamentous fungi, e.g., Trichoderma reesei, is engineered to produce an increased amount of CBH2. Such genetically modified filamentous fungi would be useful to produce a cellulase product with greater increased cellulolytic capacity. In one approach, this is accomplished by introducing the coding sequence for cbh2 into a suitable host, e.g., a filamentous fungi such as Aspergillus niger.<br>
 [204]   Accordingly, the invention includes methods for expressing variant CBH2 in a filamentous fungus or other suitable host by introducing an expression vector containing the DNA sequence encoding variant CBH2 into cells of the filamentous fungus or other suitable host.<br>
[205]   In another aspect, the invention includes methods for modifying the expression of CBH2 in a filamentous fungus or other suitable host. Such modification includes a decrease or elimination in expression of the endogenous CBH2.<br>
[206]   In general, assays employed to analyze the expression of variant CBH2 include,. Northern blotting, dot blotting (DNA or RNA analysis), RT-PCR (reverse transcriptase polymerase chain reaction), or in situ hybridization, using an appropriately labeled probe (based on the nucleic acid coding sequence) and conventional Southern blotting and autoradiography,<br>
[207]   In addition, the production and/or expression of variant CBH2 may be measured in a sample directly, for example, by assays for cellobiohydrolase activity, expression and/or production. Such assays are described, for example, in Karlsson, J. era/. (2001), Eur. J. Biochem, 268,6498-6507, Wood, T. (1988) in Methods in Enzymotogy, Vol. 160. Biomass Part a Cellulose and Hemicellulose (Wood, W. &amp; Kellog, S. Eds.), pp. 19-25, Academic Press, San Diego, CA, USA) and, for the PAHBAH assay in (Lever, M. (1972) Analytical Biochemistry, 47,273, Blakeney, A.B. &amp; Mutton, LL (1980) Journal of Science of Food and Agriculture, 31,889, Henry, RJ. (1984) Journal of the Institute of Brewing, 90,37 Substrates useful for assaying cellobiohydrolase, endoglucanase or β-glucosidase activities include crystalline cellulose, filter paper, phosphoric acid swollen cellulose, cellooligosaccharides, methylumbelliferyl lactoside, methylumbelliferyl cellobioside, orthonitrophenyl lactoside, paranitrophenyl lactoside, orthonitrophenyl cellobioside, paranitrophenyl cellobioside.<br>
[208] In addition, protein expression, may be evaluated by immunological methods, such as immunohistochemical staining of cells, tissue sections or immunoassay of tissue culture medium, e.g., by Western blot or ELISA. Such immunoassays can be used to qualitatively and quantitatively evaluate expression of a CBH2 variant. The details of such methods are known to those of skill in the art and many reagents for practicing such methods are commercially available.<br>
[209]   A purified form of a variant CBH2 may be used to produce either monoclonal or polyclonal antibodies specific to the expressed protein for use in various immunoassays, (See, e.g., Hu et al., Mol Cell Biol. vol.11, no. 11, pp. 5792-5799,1991). Exemplary assays include ELISA, competitive immunoassays, radioimmunoassays, Western blot, indirect<br>
immunofluprescent assays and the like. In general, commercially available antibodies and/or kits may be used for the quantitative immunoassay of the expression level of cellobiohydrolase proteins.<br>
VIII.    Isolation And Purification Of Recombinant CBH2 Protein. [210]   In general, a variant CBH2 protein produced in cell culture is secreted into the medium and may be purified or isolated, e.g., by removing unwanted components from the cell culture medium. However, in some cases, a variant CBH2 protein may be produced in a cellular form necessitating recovery from a ceil lysate. In such cases the variant CBH2 protein is purified from the cells in which it was produced using techniques routinely employed by those of skill in the art Examples include, but are not limited to, affinity chromatography (Tilbeurgh et al., FEES Lett. 16:215,1984), ion-exchange chromatographic methods (Goyal et al., Bioresource Technol. 36:37-50,1991; Fliess et al., Eur. J. Appl. Microbiol. Biotechnol. 17:314-318,1983; Bhikhabhai etal., J. Appl. Biochem. 6:336-345, 1984; Eliouz etal., J. Chromatography 396:307-317,1987), including ion-exchange using materials with high resolution power (Medve etal., J. Chromatography A 808:153-165, 1998), hydrophobic interaction chromatography (Tomaz and Queiroz, J. Chromatography A 865:123-128,1999), and two-phase partitioning (Brumbauer, era/., Bioseparation 7:287-295, 1999).<br>
'[211]   Typically, the variant CBH2 protein is fractionated to segregate proteins having selected properties, such as binding affinity to particular binding agents, e.g., antibodies or receptors; or which have a selected molecular weight range, or range of isoelectric points. [212]   Once expression cf a given variant CBH2 protein is achieved/the CBH2 protein thereby produced is purified from the cells or cell culture. Exemplary procedures suitable for such purification include the following: antibody-affinity column chromatography, ion exchange chromatography; ethanol precipitation; reverse phase HPLC; chromatography on silica or on a cation-exchange resin such as DEAE; chromatofocusing; SDS-PAGE; ammonium sulfate precipitation; and gel filtration using, e.g., Sephadex G-75. Various methods of protein purification may be employed and such methods are known in the art and described e.g. in Deutscher, Methods in Enzymology, vol. 182, no. 57, pp. 779,1990; Scopes, Methods Enzymol. 90: 479-91,1982. The purification step(s) selected will depend, e.g., on the nature of the production process used and the particular protein produced.<br>
IX.      Utility of cbh2 and CBH2<br>
[213]. it can be appreciated that the variant cbh nucleic acids, the variant CBH2 protein and compositions comprising variant CBH2 protein activity find utility in a wide variety applications, some of which are described below.<br>
[214]   New and improved cellulase compositions that comprise varying amounts BG-type, EG-type and variant CBH-type cellulases find utility in detergent compositions that exhibit enhanced cleaning ability, function as a softening agent and/or improve the feel of cotton . fabrics (e.g., "stone washing" or "biopollshing"), in compositions for degrading wood pulp into sugars (e.g., for bio-ethanol production), and/or in feed compositions. The isolation and characterization of cellulase of each type provides the ability to control the aspects of such compositions.<br>
[215]   Variant (or mutant) CBHs with increased thermostabillty find uses In all of the above areas due to their ability to retain activity at elevated temperatures. [216]   Variant (or mutant) CBHs with decreased thermostabillty find uses, for example, in areas where the enzyme activity is required to be neutralized at lower temperatures so that other enzymes that may be present are left unaffected. In addition, the enzymes may find utility in the limited conversion of cellulosics, for example, in controlling the degree of crystallinity or of cellulosic chain-length. After reaching the desired extent of conversion the saccharifying temperature can be raised above the survival temperature of the de-stabilized CBH. As the CBH activity is essential for hydrolysis of crystalline cellulose, conversion of crystalline cellulose will cease at the elevated temperature.<br>
[217]   In one approach, the cellulase of the invention finds utility in detergent compositions or in the treatment of fabrics to improve the feel and appearance. [218]   Since the rate of hydrolysis of cellulosic products may be increased by using a transformant having at least one additional copy of the cbh gene inserted into the genome, products that contain cellulose or heteroglycans can be degraded at a faster rate and to a greater extent... Products made from cellulose such as paper, cotton, cellulosic diapers and the like can be degraded more efficiently in a landfill. Thus, the fermentation product obtainable from the transformants or the transformants alone may be used in compositions to help degrade by liquefaction a variety of cellulose products that add to the overcrowded landfills.<br>
[219]   Separate saccharification and fermentation is a process whereby cellulose present in biomass, e.g., com stover, is converted to glucose and subsequently yeast strains convert glucose into ethanol. Simultaneous saccharification and fermentation is a process whereby cellulose present in biomass, e.g., corn stover, is converted to glucose and, at the same time<br>
and in the same reactor, yeast strains convert glucose into ethanol. Thus, in another approach, the variant CBH type cellulase of the invention finds utility in the degradation of biomass to ethanol. Ethanol production from readily available sources of cellulose provides a stable, renewable fuel source.<br>
[220]   Cellulose-based feedstocks are comprised of agricultural wastes, grasses and woods and other low-value biomass such as municipal waste (e.g., recycled paper, yard clippings, etc.). Ethanol may be produced from the fermentation of any of these cellulosic feedstocks. However, the cellulose must first be converted to sugars before there can be conversion to ethanol.<br>
[221]   A large variety of feedstocks may be used with the inventive variant CBH and the one selected for use may depend on the region where the conversion is being done. For example, in the Midwestern United States agricultural wastes such as wheat straw, com stover and bagasse may predominate while in California rice straw may predominate. However, it should be understood that any available cellulosic biomass may be used in any region.<br>
[222]   A cellulase composition containing an enhanced amount of cellobiohydrolase finds utility in ethanol production. Ethanol from this process can be further used as an octane enhancer or directly as a fuel in lieu of gasoline which is advantageous because ethanol as a fuel source is more environmentally friendly than petroleum derived products. It is known that the use of ethanol will improve air quality and possibly reduce local ozone levels and smog. Moreover, utilization of ethanol in lieu of gasoline can be of strategic importance in buffering the impact of sudden shifts in non-renewable energy and petro-chemicai supplies. [223]   Ethanol can be produced via saccharification and fermentation processes from cellulosic biomass such as trees, herbaceous plants, municipal solid waste and agricultural and forestry residues. However, the ratio of individual cellulase enzymes within a naturally occurring cellulase mixture produced by a microbe may not be the most efficient for rapid conversion of cellulose in biomass to glucose. It is known that endoglucanases act to, produce new cellulose chain ends which themselves are substrates for the action of cellobiohydrolases and thereby improve the efficiency of hydrolysis of the entire cellulase system. Therefore, the use of increased or optimized cellobichydrolase activity may greatly enhance the production of ethanol.<br>
[224]   Thus, the inventive cellobiohydrolase finds use in the hydrolysis of cellulose to its sugar components. In one embodiment, a variant cellobiohydrolase is added to the biomass prior to the addition of a fermentative organism. In a second embodiment, a variant<br>
cellobiohydrolase is added to the biomass at the same time as a fermentative organism. Optionally, there may be other cellulase components present In either embodiment. [225]   In another embodiment the cellulosic feedstock may be pretreated. Pretreatment may be by elevated temperature and the addition of either of dilute acid, concentrated acid or dilute alkali solution. The pretreatment solution is added for a time sufficient to at least partially hydrolyze the hemicellulose components and then neutralized. [226]   The major product of CBH2 action on cellulose is cellobiose which is available for conversion to glucose by BG activity (for instance in a fungal cellulase product). Either by the pretreatment of the cellulosic biomass or by the enzymatic action on the biomass, other sugars, in addition to glucose and cellobiose, can be made available from the biomass. The hemi-cellulose content of the biomass can be converted (by hemi-cellulases) to sugars such as xylose, galactose, mannose and arabinose. Thus, in a biomass conversion process, enzymatic saccharification can produce sugars that are made available for biological or. chemical conversions to other intermediates or end-products. Therefore, the sugars generated from biomass find use in a variety of processes in addition to the generation of ethanol. Examples of such conversions are fermentation of glucose to ethanol (as reviewed by M.E. Himmel et al. pp2-45, in "Fuels and Chemicals from Biomass", ACS Symposium Series 666, ed B.C. Saha and J. Woodward, 1997) and other biological conversions of glucose to 2,5-diketo-D-gluconate (US Patent No. 6,599,722), lactic acid (R. Datta and S-P. Tsai pp224-236, Ibid), succinate (R.R. Gokam, MA Etteman and J. Sridhar pp237-263, ibid), 1,3-propanediol (A-P. Zheng, H. Biebl and W-D. Deckwer pp264-279, ibid), 2,3-butanediol (C.S. Gong, N. Cao and G.T. Tsao pp280-293, ibid), and the chemical and biological conversions of xylose to xylose(B.C. Saha and RJ. Bothast pp307-319, ibid). See also, for example, WO 98/21339.<br>
[227]   The detergent compositions of this invention may employ besides the cellulase composition (irrespective of the cellobiohydrolase content, i.e., cellobiohydrolase -free, substantially cellobiohydrolase -free, or cellobiohydrolase enhanced), a surfactant, including anionic, non-ionic and ampholytic surfactants, a hydrolase, building agents, bleaching agents, bluing agents and fluorescent dyes, caking inhibitors, solubilizers, cationic surfactants and the like. All of these components are known in the detergent art. The cellulase composition as described above can be added to the detergent composition either in a liquid diluent, in granules, in emulsions, in gels, in pastes, and the like. Such forms are well known to the skilled artisan. When a solid detergent composition is employed, the cellulase composition is preferably formulated as granules. Preferably, the granules can be formulated so as to contain a cellulase protecting agent For a more thorough discussion,<br>
 <br>
see US Patent Number 6,162,782 entitled "Detergent compositions containing cellulase compositions deficient in CBH2 type components," which is incorporated herein by reference.<br>
[228]   Preferably the cellulase compositions are employed from about 0.00005 weight percent to about 5 weight percent relative to the total detergent composition. More preferably, the cellulase compositions are employed from about 0.0002 weight percent to about 2 weight percent relative to the total detergent composition. [229]   In addition the variant CBH2 nucleic acid sequence finds utility in the identification and characterization of related nucleic acid sequences. A number of techniques useful for determining (predicting or confirming) the function of related genes or gene products include, but are not limited to, (A) DNA/RNA analysis, such as (1) overexpression, ectopic expression, and expression in other species; (2) gene knock-out (reverse genetics, targeted knock-out, viral induced gene silencing (VIGS, see Baulcombe, 100 Years of Virology, Calisher and Horzinek eds., Springer-Verlag, New York, NY 15:189-201,1999); (3) analysis of the methylation status of the gene, especially flanking regulatory regions; and (4) in situ hybridization; (B) gene product analysis such as (1) recombinant protein expression; (2) antisera production, (3) immunolocalization; (4) biochemical assays for catalytic or other activity; (5) phosphorylation status; and (6) interaction with other proteins via yeast two-hybrid analysis; (C) pathway analysis, such as placing a gene or gene product within a particular biochemical or signaling pathway based on its overexpression phenotype or by sequence homology with related igenes; and (D) other analyses which may also be performed to determine or confirm the participation of the isolated gene and its product in a particular metabolic or signaling pathway, and help determine gene function. [230]   All patents, patent applications, articles and publications mentioned herein, are hereby expressly incorporated herein by reference.<br>
EXAMPLES<br>
[231]   The present invention is described in further detail in the following examples which are not in any way intended to limit the scope of the invention as claimed. The attached Figures are meant to be considered as integral parts of the specification and description of the invention. All references cited are herein specifically incorporated by reference for all that is described therein.<br>
EXAMPLE 1 Alignment of known CelGA cellulases<br>
[232]   The choice of several of the mutations was determined by first aligning Hypocrea<br>
jecorina Cel6A to eight (8) family members using structural Information and a modeling program. Figure 3 shows the alignment of CBH2 molecules derived from Humicola insolens (Q9C1S9), Acremonium cellulotyticus (O93837),Agaricus blsporus (P49075), Hypocrea ton//70//(AF315681), Phanerochaete chrysosporium (S76141), Talaromycesemersontt (Q8N1B5), Lentlnula edodes (AF244369), Hypocrea Jecorina (P07987). Alignments were done by Clustal W with a gap penalty of 10 using Vector NT! Suite software program. [233]   Based on the alignments, various single and multiple amino acid mutations were made in the protein by site mutagenesis. Possible mutations were identified that might improve the thermostability of the enzyme by using the consensus sequence. See Rgure 3. A visual inspection of the 3D-structure was performed to check for their compatibility with the structure. All changes, which do either not fit into the CBH2 molecule for sterical reasons or are to close to the active site, were omitted from the set of initial mutations. [234]   The consensus sequence for CBH2 was determined by aligning CBH2 sequences as described herein. The alignment of Figure 3 served as basis for the determinatton of the so-called consensus sequence. The consensus sequence of an alignment is the sequence, which has at each position the amino acid, which is found in the majority of the amino acid sequences, which were used to construct the alignment. Those positions where the consensus sequence deviated from the CBH2 T, reesei amino acid sequence were evaluated by examining the 3D-structure of the protein (PDB-code 1QK2).. The graphical inspection was done using the computer program BRAGI (D. Schomburg, J. Reichelt, J. Mol. Graphics, Vol 6,161-165 (1988)) on a Silicon Graphics Indigo2 Solid Impact computer. Those mutations that - according to the 3D-model -fit into the structure without disturbance and were likely to improve the thermostability of the enzyme were selected as replacement for improved thermostability of H. jecorina CBH2. In some cases the visual inspection of the 3D-structure of the CBH2-molecule made it necessary to replace a non-conserved residue of H. jecorina CBH2 by another amino acid than the sequence alignment suggested. In some cases, the amino acid is glycosylated in the 30-structure. The glycosylated positions, which were investigated based on the alignment, are S109 and N310. These positions were not changed. At position V94 the valine was replaced by a glutamic acid residue, because it might have stabilizing charge interactions with one or two arginines at position 213 and/or 237. At position T142 we decided to introduce a proline, which might fit according to the sequence alignment This amino acid is found at many of the aligned sequences and can stabilize due to the entropic effect of the proline. At position L179 we decided to test the<br>
effect of an introduction of an alanine, which is the only alternative aminp acid in the CBH2 molecules in this alignment. At position Q204, we decided to replace the glutamlne by a glutamic acid residue and not by alanine as suggested by the consensus sequence, because the alanine introduction might destroy favorable interactions in the hydrophobia core, whereas the introduction of a charge through the Q to E mutation should improve the charge network on the surface of the molecule. We replaced V206 by ieucine, because the fit in the hydrophobia core seems better than fitting isoleucine. In case of V250, we decided to replace it by Ieucine and not by the slightly bigger isoleucine due to space constraints. At position N285, we investigated the influence of the side chain length on stability by replacing the asparagine by glutamlne. At position 8291 we decided to test the effect of an introduction of a glycine, which is the only alternative amino acid in the CBH2 molecules in this alignment. At position 8316 we decided to test the effect of an introduction of a proline, which is the only alternative amino acid in the CBH2 molecules in this alignment. At position 8343 we decided to introduce a proline due to its stabilizing effect on the backbone and the fact that this amino acid is the most frequent one at this position in the alignment. At position T349 the threonine was replaced by a Ieucine and not by valine as suggested by the consensus sequence. At position 8413 we decided to test the effect of an aromatic residue on stability at this position by replacing the serine by tyrosine.<br>
EXAMPLE 2 Preparation of cbh2 constructs<br>
[235]   The cDNA sequence of CBH2 presented in Rgure 2 served as the template for<br>
amplifying the gene. It also served as the template for the introduction of mutations. [236]   The following DNA primers were constructed for use in amplification of mutant cbh2 genes from genomic DNA's isolated from various microorganisms. All symbols used herein for protein and DNA sequences correspond to IUPACIUB Biochemical Nomenclature Commission codes.<br>
[237]   Homologous 5' (FRG361) and 3' (FRG362) primers were developed based on the sequence of cbh2from Trichoderma reesel. Both primers contained Gateway cloning sequences from Invitrogen® at the 5' of the primer. Primer 361 contained attB1 sequence and primer FRG362 contained attB2 sequence.<br>
Sequence of FRG361 without the attB1:<br>
ATGATTGTCGGCATTCTCAC (this primes the 5' end of the gene, encoding the<br>
signal sequence of CBH2.H. jecorina) (SEQ ID NO: 3)<br>
Sequence of FRG362 without the attB2:<br>
TTACAGGAACGATGGGTTTGCG (this primes the 3' end of the gene encoding the<br>
catalytic domain of CBH2 H. Jecorina) (SEQ ID NO: 4))<br>
[238]   The H. jecorina cMZcDNA served as template. The cDNA used was derived from a cDNA library prepared as described by Pamela K. Foreman et al, Journal of Biological Chemistry Vol 278 No 34 2003 page 31989. The library was screened with specific CBH2 catalytic domain probe using primers:<br>
Table 1<br>
(Table Removed)<br>
<br>
[239]   PCR conditions were as follows: 10 ML of 10X reaction buffer (10X reaction buffer comprising 100mM Tris HCI, pH 8-8.5; 250 mM KCI; 50 mM (NH4)2SO4; 20 mM MgSO4); 0.2 mM each of dATP, dTTP, dGTP, dCTP (final concentration), 1 uL of 100 ng/pL genomic DNA, 0.5 uL of PWO polymerase (Boehrin£jer Mannheim, Cat # 1644-947) at 1 unit per uL, 0.2uM of each primer, FRG361 and FRG362, (final concentration), 4μl DMSO and water to 100 μL<br>
[240]   The fragments encoding the variants were purified from an agarose gel using the Qiagen Gel extraction KIT. The purified fragments were used to perform a clonase reactbn with the pDONR™201 vector from Invitrogen® using the Gateway™ Technology instruction manual (version C) from Invitrogen®, hereby incorporated by reference herein. The thus prepared pENTRYCBH2 clone is given in figure 4.<br>
[241]   Various sites in H. jecorina CBH2 may be involved in the thermostabiiity of the variants and the H. jecorina cbh2 gene was therefore subjected to mutagenesis using the primers and reaction reagents described below.<br>
[242]   The cycling parameters for each reaction (single site mutagenesis, random mutagenesis, regional mutagenesis or combinatorial) were the same:<br>
Table 2<br>
(Table Removed) <br>
[243]   The amplification products were isolated and characterized as described below (see<br>
Examples 3-6.<br>
[244]   Genes (variant or wild type) having the correct sequence were then transferred from<br>
this ENTRY vector to the destination vector (pRAXdes2) to obtain the expression vector<br>
pRAXdesCBH2.<br>
[245]   Cells were transformed with an expression vector comprising a variant CBH2<br>
cellulase encoding nucleic acid. The constructs were transformed into A. niger var. awamori<br>
according to the method described by Cao et al (Cao Q-N, Stubbs M, Ngo KQP, Ward M,<br>
Cunningham A, Pal EF, Tu G-C and Hofmann T (2000) Penicillopepsin-JT2 a recombinant<br>
enzyme from Penicilllum janthlnellum and contribution of a hydrogen bond in subslte S3 to<br>
kcat Protein Science 9:991-1001).<br>
EXAMPLES<br>
Site Directed Mutagenesis [246]   Based on the above rationale presented in Example 1, Site Directed CBH2 mutants<br>
were made with the following 5' phosphorylated primers that were developed and synthesized using techniques well known in the art;<br>
Table 3; Primers for Single Site Directed Mutants<br>
(Table Removed) <br>
[247]   Codons encoding the mutation are underlined and in bold type. [248]   To develop the Site Directed Mutants the QuikChange Multi Site-Directed Mutagenesis Kit (Stratagene, La Jolla, CA; Cat # 200513) was used. 5     [249]   The mutagenesis reaction was catried out using the following reaction reagents:<br>
(Table Removed 4)<br>
<br>
[251]   The amplification products were recovered (purified from primers, nucleotides, 10    enzymes, mineral oil, salts, and other impurities) were digested with Dpn I. One microliter Dpn I (10 U/μl) was added to the PCR mixture and incubated at 37°C for 90 min. The PCR<br>
reaction products were purified using QIAquick PCR purification Kit (250) (Qiagen, Gat No. 28106) according to manufacturer's instruction. The elution volume was 35 ul eiution buffer. [252]   To remove double stranded non-mutated DNA, the eluted sample was digested a second time with Dpn I restriction enzyme. 1 ul Dpn I from Invitrogen (Cat. No. 15242-019) and 4 pi reaction buffer (React 4 is supplied with Dpn I as a 10X solution; dilute 1:10 In final<br>
*	*<br>
reaction mixture) were added to the sample and Incubated at 37°C for 90 minutes. Two microliters of the reaction sample was used for transformation of 10 pi One-Shot Top10 electro-competent cells (Invitrogen, Cat No. C4040-50). After 1 hour of growth at 37°C in SOC medium (See Hanahan (1983) J. Mol. Btol. 166:557-580), the cells have been plated on selective Kanamycin plates, and incubated at 37°C overnight. [253]   Positive clones were grown in 2*TY medium with 100 ug/ml ampicillin, and plasmid DNA was isolated with QIAprep Spin Miniprep Kit (Cat. No. 27106). The plasmids were sequenced to confirm that the mutation sequence had been incorporated correctly. [254]   Mutants with the right sequence were transferred to the A. nlger expression vector pRAXdest#2 with the LR reaction, according the Gateway Cloning procedure (Invitrogen; Cat. No. 11791019). After protoplast transformation of the expression clones to A. nlger AP4, the Site Directed Mutants (SDM) were screened for altered thermostability (Table 14).<br>
EXAMPLE 4 Combinatorial Libraries<br>
[255]   Two QuikChange libraries (QC2C and QC2D) were developed, based on the results for single site mutations that were identified during the SDM screening: 98,134,206,212, 312, 316,411 and 413.<br>
[256]   The mutations P98L, M134V, V206L, 1212V, T312S, S316P, F411Y and S413Y were randomly combined in libraries using the quick-change (QC) method. To develop QC libraries the Multi Site-Directed Mutagenesis Kit (Cat # 200513 from Strategene) was used. [257]   The primers were prepared as listed in the table below:<br>
Table 5<br>
(Table Removed)<br>
 <br>
 <br>
[258]   A primer mix was prepared as follows:<br>
[259]   Thirty microliters of primers 98 and 134 (see above table) were mixed with the 10 μl<br>
of each other primer (see above table). Two different primer concentrations were tested<br>
resulting in two libraries of which one contained an average of 2 amino acid substitutions<br>
and the second 6 amino acids substitutions per molecule.<br>
[260]   The mutagenesis reaction was carried out using the following reaction reagents:<br>
Tables<br>
(Table Removed)<br>
<br>
<br>
[261]   The amplification products were digested with Dpn I. 1 ul Dpnl (10 U/ul) was added to the PCR mixture and incubated at 37°C for 90 min. The PCR reaction products were purified using QIAquick PCR purification Kit (250) (Qiagen, Cat. No. 28106) according to manufacturer's instruction. The elution volume was 35 M' elution buffer. [262]   To remove double stranded non-mutated DMA, the eluted sample was digested a second time with Dpn I restriction enzyme. 1 μl Dpn I from Invttrogen (Cat. No. 15242-019) and 4 M! reaction buffer (React 4 is supplied with Dpn I as a 10X solution; dilute 1:10 in final reaction mixture) were added to the sample and incubated at 37°C for 90 minutes. Two microliters of the reaction sample was used for transformation of 10 μl One-Shot Topi 0 electro-competent cells (Invitrogen, Cat. No. 04040-50). After 1 hour of growth at 37°C in SOC medium (See Hanahan (1983) J. Mol. Biol. 166:557-580), the cells were plated on selective Kanamycin plates, and incubated at 37°C overnight<br>
 [263]   Positive clones were grown in 2*TY medium with 100 μg/ml ampicillin, and plasmid DNA was isolated with QIAprep Spin Miniprep Kit (Cat. No. 27106). The plasmids were sequenced to confirm that the mutation sequence had been incorporated correctly. [264]   Mutants with the right sequence were transferred to the A niger expression vector pRAXdest#2 with the LR reaction, according the Gateway Cloning procedure (Invitrogen; Cat. No. 11791019). After protoplast transformation of the expression clones to A niger AP4, the QC libraries were screened for altered thermostabilrty (Tables 15,16).<br>
EXAMPLES<br>
Regional Mutagenesis<br>
[265]   As described below, based on the results for single site mutations that were identified<br>
during the SDM screening, regions were identified in the 3D-structure of CBH2, randomly mutated and screened in a thermostability assay. The amino acids, which make up such a spatial region are (in groups): [210,214], [253,255,257,258], [411,413,415], [432,414, 416], [312,313], 323, [212,149,152], [134,144] and 98.<br>
[266]   Fully randomized libraries at the positions above (i.e., [210,214], [253,255,257, 258], [411,413, 415], [412,414,416], [312,313], 323, [212,149,152], [134,144] and 98) were screened. The amino acids in the list above between brackets (e.g., [210,214]) were randomized together, the amino acids 323 and 98 were randomized alone. The variants CBH2-S316P or CBH2-V206L-S316P served as the backbone for these libraries. [267]   NNS primers were constructed and ordered from Invitrogen:<br>
Table 7<br>
(Table Removed)<br>
 [268]   PCR was performed according to the following protocol, with Pfu Ultra DNA polymerase (Strategene; Cat. No. 600380):<br>
Table 8<br>
(Table Removed)<br>
<br>
<br>
[269]   PCR fragments were put on a 1% LMT gel, and purified with a Qiagen Gel Extraction Kit (Strategene Cat.No. 28706). The purified fragments were fused with Pfu Ultra (see above) and the cbh2 primers with attB flanking-sequences.<br>
[270]   Purified cbh2 genes were transferred into the Gateway Entry-Vector pDON201 with the BP reaction, according the manual (Invitrogen; Cat. No. 11789013). Positive clones<br>
were selected on 2*TY plates with kanamycin. (50 μg/ml), and the plates scraped for preparation of the plasmids for transfer to pRAXdest#2 (expressionvector), with the Gateway LR reaction (Invftrogen; Cat. No. 11791019). This vector was digested with Notl, to optimize the transformation frequency of the LR reaction. Protoplast transformation has been used to create 9 CBH2 Regional Libraries in A. nlgerAP4 that were screened for altered thermostability (Table 17).<br>
EXAMPLES Multiple Mutants<br>
[271]   Based on the expression and thermostability results of the single site mutations, a set<br>
of multiple mutants was designed, which were produced in shake flasks only. [272]   Mutations from CBH2 mutant FCA557 (P98L7M134V/S316P/S413Y) (from QC-libraries, Example 4) were combined (TABLE 9) with those from CBH2 mutants FCA 564 (S316P/V323Y), FCA568 (V206L/S210R/T214Y/S316P) and FCA570 (M134LA144R/S316P) (from Region Libraries, Example 5), to obtain CBH2 molecules with improved thermal stability.<br>
Table 9<br>
(Table Removed)<br>
<br>
<br>
 <br>
 [273]   Primers were constructed and ordered from Invrtrogen:<br>
Table 10<br>
(Table Removed)<br>
<br>
<br>
[274]   PCR was performed using the reaction reagents in Table 11 (below), to obtain all fragments (A - M) needed to construct all 9 CBH2 combinatorials. The DNA polymerase Phusion (Finnzymes; Cat. No. F-530) was used. The primer concentration was 10 μM. The cycling conditions are given in Table 2 (above).<br>
Table 11<br>
(Table Removed)<br>
 [275]   PCR fragments were put on a 1 % LMT gel, and purified with a Qiagen Gel Extraction Kit (Strategene Cat.No. 28706). The purified fragments were fused using Phusion DNA polymerase (see above) with the CBH2-attB primers (above), to obtain the complete CBH2 combinatorials according to Table 12.<br>
Table 12<br>
(Table Removed)<br>
 [276]   Complete CBH2-attB molecules were purified from 1% LMT agarose and transferred to the A niger AP4 expression vector pRAXdest#2. The method used was the "One-tube Protocol for cloning .attB-PCR products directly into destination vectors" (according Invttrogen's manual).<br>
[277]   Three microliters ul of the reaction sample was used for transformation of 100 μl DH5a max. efficiency competent cells (Invitrogen Cat. No. 18258012), according the manual. After 1 hour of growth at 37°C in SOC medium, the cells were plated on selective ampiciilin plates (100μg/ml), and incubated at 37°C overnight. Positive clones were grown in 2*TY medium and 100 ug/ml ampiciilin. Plasmid DNA was isolated with QIAprep Spin Miniprep Kit (Qiagen Cat. No. 27106) and sequenced.<br>
[278]   Mutants with the right sequence were transferred to the A niger expression vector pRAXdest#2 with the LR reaction, according the Gateway Cloning procedure (Invitrogen; Cat. No. 11791019). After protoplast transformation of the expression clones to A. niger AP4, the Multiple Mutants were expressed and isolated (as in Example 7), and analyzed for thermostability (Tables 18,19).<br>
EXAMPLE 7 Expression and Isolation of CBH2 and its variants from shake flask growths<br>
[279]   To provide materials for Tm measurements in thermal denaturation studies (Example<br>
9), expression clones were grown in shake flasks and then the CBH2 molecules purified, as<br>
follows:<br>
[280]   Cells were transformed with an expression vector comprising a variant CBH2<br>
cellulase encoding nucleic acid. The constructs were transformed into A niger var. awamori<br>
according to the method described by Cao et al (Cao Q-N, Stubbs M, Ngo KQP, Ward M,<br>
Cunningham A, Pal EF, Tu G-C and Hofmann T (2000) Penicillopepsin-JT2 a recombinant<br>
enzyme from Penicillium janthinellum and contribution of a hydrogen bond in subsite S3 to<br>
kcat Protein Science 9:991-1001).<br>
[281]   A niger var awamori transformants were grown on minimal medium lacking uridine<br>
(Ballance et al. 1983). Transformants were grown by Inoculating 1cm2 of spore suspension<br>
from the sporulated grown agar plate into 100ml shake flasks for 3 days at 37°C as described by Cao et.al. (2000), and then screened for cellulase activity. [282]   The CBH2 activity assay was based on the hydrolysis of the Phosphoric Acid Swollen Cellulose (PASC: 0.5% PASC in 0.5mM Na acetate, pH 4.85). Reducing sugars were measured by PAHBAH assay. (PAHBAH: 2.975 g PAHBAH, 9.75 g Na-K-tartrate in 195 ml 2% NaOH). PASC: (Karlsson, J. et al. (2001), Eur. J. Biochem, 268,6498-6507, Wood, T. (1988) in Methods in Enzymology, Vol. 160. Biomass Part a Cellulose and Hemicellulose (Wood, W. &amp; Kellog, S. Eds.), pp. 19-25, Academic Press, San Diego, CA, USA) and PAHBAH: (Lever, M. (1972) Analytical Biochemistry, 47,273, Blakeney, A.B. &amp; Mutton, L.L (1980) Journal of Science of Food and Agriculture, 31, 889, Henry, RJ. (1984) Journal of the Institute of Brewing, 90,37)<br>
[283]   Cel6A wild type and variants were then purified from cell-free supernatants of these cultures by hydrophobic interaction chromatography (HIC) by one of two procedures*. [284]   For SDM variants (Example 3), Bio-RAD Poly-Prep Columns CAT# 731-1550 were used with Pharmacia Phenyl Sepharose resin (1.6ml=1 ml column) CAT # 17-0973-05. The resin was allowed to settle before washing with 1-2 column volume (CV) water, then equilibrated with 5 CV of 0.020 M sodium phosphate, 0.5 M ammonium sulfate, pH6.8 (Buffer A). 4M Ammonium sulfate was added to the supernatants to a final concentration of approximately 0.5 M. 2 CV of supernatant was loaded and the column then washed with 5 CV of Buffer A, before elution with 4 CV of 0.020 M sodium phosphate, pH6.8. The filtrate contained purified CBH2.<br>
[285]   For Multiple Mutants (Example 6), columns were run on Novagen vacuum manifold using Poros® 20 HP2 resin made by Applied Biosystems. HIC columns were equilibrated with 5 CV of Buffer A. Ammonium sulfate was added to the supernatants to a final concentration of approximately 0.5 M and the pH adjusted to 6.8. After filtration, the supernatant was loaded, the column washed with 10 CV of Buffer A and then eluted with a 10 CV of 0.020 M sodium phosphate, pH 6.80. Fractions were collected and pooled on the basis of the presence of CBH2, as detected by reduced, SDS-PAGE gel analysis. [286]   When desired, CBH2 molecules are de-glycosylated prior to purification by treatment of the supernatant with Endoglycosidase H according to the supplied protocol (Sigma-Aldrich).<br>
EXAMPLES Thermostability of CBH2 variants by thermal inactivation<br>
[287]   CBH2 molecules with altered stability to irreversible thermal inactivation compared to wild type CBH2 were identified by measurement of the PASC activity of equal aliquots of<br>
cell-free supernatants before and after incubation at elevated temperature under stringent conditions. The stringent conditions used were 1 hour incubation at either 61°C or 65°C (as indicated) of a 1:1 dilution of supernatant In 0.1 M sodium acetate, pH4.85, followed by cooling on ice for 10 minutes. The % residual activity (% remaining activity after incubation at elevated conditions) was calculated by dividing the residual activity by the initial activity (CBHS activity on PASC before stringency incubation).<br>
[288]. Screening of CBH2 variants and wild type for stability to thermal inactivation was performed according to the following protocols:<br>
A. Solutions and media<br>
[289]   The following solutions/media were used in the determination of the CBH2 mutants<br>
stability to irreversible thermal inactivation:<br>
1.	Minimal medium plus maltose (MM medium) was prepared as shown in Table<br>
13:<br>
Table 13; Minimal medium + maltose for A. niaer<br>
(Table Removed)<br>
<br>
2.	0.5% PASC solution in 50mM NaAc pH 4.85:<br>
i.     Add 5 gram Avicel PH101 (Fluka 11365) to a 1 liter beaker glass and<br>
add-12 ml water to make a thick slurry. II.    Place beaker on Ice. HI   Add 150 ml ice-cold 85% ortho-phosphoric acid (Art. 1000573 Merck)<br>
and mix with the ultra tun-ax at high speed (prevent splashing) for about<br>
1 hour, iv.   Add 100 ml ice-cold acetone, which causes a slow precipitation of the<br>
amorphous cellulose to a very thick slurry. Eventually use a spatula for<br>
better mixing, v.    Dilute the very thick slurry to -1 liter with water to make it fluid enough to<br>
transfer it to 6 x 250 ml Sorvall containers, vi.    Spin down 15 minutes at 10k and discard the supernatant, vii.   Mix pellets with as much water as the beakers can contain and spin<br>
down again, viii.  Repeat step 5 and 6 at least for 3 times until the pH has increased to<br>
pH4.0-5.0. ix.   To increase the washing of the phosphoric acid a drop of NaOH 4N can<br>
be added to the water.<br>
x.    Mix the last pellets with water to -300 ml and homogenize, xi.    Determine the concentration of the slurry with dry weight measurement. Sterilize the slurry for 20 minutes at 121 °b. Cool down and store in the refrigerator.<br>
3.	PAHBAH reagents were prepared as follows: 1.5 g PAHBAH + 5 g Sodium-<br>
Potassium-tartrate in 100 ml 2% NaOH<br>
4.	Cellobiose stock solutions; In MQ water prepare a 0,0.01,0.05,0.1,0.2,0.5,0.7 and<br>
a 1 mg/ml solution.<br>
B.	Sample preparation<br>
1.	Grow Aniger variants in 96W filter MTP's (Millipore, # MAGVS2210), containing<br>
200pl minimal medium + maltose (above) per well for 7 days at 33°C, on an orbital shaker<br>
(225 rpm) with 80-90% humidity.<br>
2.	After growth incubation, filter the cultures using a vacuum manifold, collecting the<br>
filtrate (supernatant) in a fresh 96W flat MTP's (Greiner, # 655101), store at 4°C.<br>
C.	Stringency incubation at elevated temperatures<br>
1.	Dilute 60μl supernatant (sup) per well with 60μl 10OmM NaAc pH 4.85 (1:1).<br>
(Optional: When remaining supernatant needs to be checked for residual sugars, add 190μl 100mM NaAcpH4.8 to 10μl sup per well and transfer 20ul of this diluted supernatant to 150ul PAHBAH reagents to do the PAHBAH reducing sugar assay (see E)<br>
2.	Transfer 20μl diluted supernatant to a fresh flat 96W MTP and store at 4°C (for initial<br>
activity)<br>
3.	Incubate remain diluted supernatant (ca. 10Oμl) for 1 hr at 61 °C (or 65°C) (for<br>
residual activity)<br>
4.	Cool down on ice for 10V<br>
D.	PASC Incubation; small scale conversion (SSC) assay<br>
1.        Prepare in fresh flat MTP's: 180μl/well of a well-stirred 0.5% PASC-solution in 50mM<br>
NaAc, pH4.85.<br>
2a.      In one plate designed to measure residual activity, transfer (with mixing up and<br>
down) per well 20μl treated diluted supernatant (sup) after pre-incubatlon to the PASC-<br>
MTP's.<br>
2b.      In a second plate designed to measure initial activity, transfer (with mixing up and<br>
down) per well 180μl PASC-solution to the stored 20μl untreated diluted supernatant.<br>
3.	Seal the sup-PASC MTP's and incubate for 2hrs at 50°, stirring 900 rpm.<br>
4.	Cool down on ice for 10'.<br>
5.	Transfer the sup-PASC mix to a fresh filter MTP , filter in a vacuum manifold and<br>
collect the filtrate.<br>
E.	PAHBAH Reducing Sugar assay<br>
1.	Prepare a fresh 96W flat MTP with 150μl PAHBAH reagents per well.<br>
2.	Transfer 20μl of the sup-PASC filtrate to the PAHBAH (mixing up and down)<br>
3.	Put a calibration line in column 1 in first MTP; 20μ! of the cellobiose stock solutions<br>
(see A4, above)<br>
4.	Incubate for 1 hr at 69PC, 900 rpm, cool down to room temperature and spin down at<br>
2000 rpm for 2'.<br>
5.	Measure endpoint OD410 in MTP reader directly in SpectraMax spectrophotometer<br>
(Spectra, Sunnyvale, CA, U.S.A.)<br>
F.	Data processing (Spad-lt)<br>
1.	From the readings on the cellobiose dilution wells, plot a cellobiose calibration line in<br>
mg/ml cellobiose VS.OD410<br>
2.	Use calibration curve and the readings from the sample wells to calculate, in mg/ml<br>
cellobiose, the initial and residual values for each sup<br>
3.	Calculate % residual activity<br>
Residual Activity Measurements for CBH2 Variants from Site Directed Mutaoenesls Table 14; CelSA Wild-Type and Variants Residual Activity<br>
(Table Removed)<br>
Each Wild Type (w.t.) and variant %residual activity number is the average of 3 determinations, stdev = calculated standard deviation for the determinations.<br>
[290]   The above table shows the % residual activity remaining for the single site directed mutants (SDM, Example 3) after 1 hour stringency incubation at 61 °C, or at 65°C. The residual activities of the WT clones are shown in every subset of the data (there was one WT reference on every plate). The average value for the WT was 21.4 % and 6.6 % at 61 °C and 65°C, respectively. It is clear that every mutant with a residual activity higher than WT is a molecule with improved thermostability under the screening conditions.<br>
Residual Activity Measurements for CBH2 Combinatorial Mutants<br>
[291]   The two results columns in Tables 15 and 16 show the % residual activities after<br>
incubation at two different temperatures for the variants produced as in Example 4.<br>
<br>
[292]   Two tables (Tables 15 AND 16) are presented below because the values were generated in two independent experiments.<br>
[293]   It is clear that every mutant with a residual activity higher than WT is a molecule with improved thermostability under the screening conditions.<br>
I aoie 15: Residual Activities for CelSA Wild Type and Variants<br>
(Table Removed)<br>
<br>
<br>
Unless otherwise indicated, the average % residual activities were calculated from 4 determinations. Where indicated, "(8)', 8 determinations were made, stdev = calculated standard deviation for the determinations.<br>
Table 16: Residual Activities for Cel6A Wild Type and Variants<br>
(Table Removed)<br>
<br>
<br>
The average % residual activities were calculated from the number of determinations indicated in parentheses, stdev = calculated standard deviation for the determinations.<br>
Residual Activity Measurements for CBH2 variants from Regional Mutaoenesis [294]  Table 17 shows the % residual activity after incubation at 61 °C for the variants<br>
produced as in Example 5.<br>
 <br>
Table 17: Residual Activities for Cel6A Wild Type and Variants<br>
(Table Removed)<br>
<br>
<br>
Unless otherwise indicated, each % residual activity number is the average of three determinations. Where indicated, "(32)", 32 determinations were made, stdev = calculated standard deviation for the determinations.<br>
[295]   It is obvious that every mutant with a residual activity higher than WT and/or S316P is a molecule with improved thermostability under the screening conditions.<br>
Example 9 Thermostability of CBH2 variants by Tm measurements<br>
[296]   CBH2 cellulase mutants were cloned, expressed and purified as above (Example 7). Thermal denaturation data was collected on a VP-DSC microcalorimeter from Microcal (Nothampton, Massachusetts, US).   Buffer conditions were 50 mM Bis Tris Propane/50 mM ammonium acetate/glacial acetic acid at pH 5.5 or at pH5.0, as indicated. The protein concentrations were approximately 0.25 mgs/ml. Three thermal scans were performed from 25°C - 80*0 at a scan rate of 90 (BC/hr). The first scan showed thermal denaturation of the CBH2 and was used to determine the apparent mid-point of thermal denaturation, Tm: The instrument software generates a Cp (cal/C) versus Temperature (BC) curve and the Tm was determined manually from this curve. The thermal denaturation was irreversible in all cases, as shown by the absence of thermal denaturation in the second and third thermal scans.<br>
Table 18; Tm's bv DSC at PH5.5.<br>
(Table Removed)<br>
<br>
All of the variant data is referenced to the still glycosylated FCA500 (rCBH2 wild type) Tm. 'Tm G" 9 Tm measured on recombinant protein as purified. Tm DG"« Tm measured on recombinant protein de-glycosylated with EndoH before being purified.<br>
TABLE 19 Tm's bv DSC at PH5.0.<br>
(Table Removed)<br>
<br>
<br>
All of the variant data is referenced to the still glycosylated FCA500 (rCBH2 wild type) Tm. Tm G" = Tm measured on recombinant protein as purified. Tm DG" = Tm measured on recombinant protein de-glycosylated with EndoH before being purified.<br>
[297]   The mutations introduced into the CBH2 cellulase mutants affected the thermal stability of the mutant CBH2 cellulase compared to wild type. [298]   De-glycosylated proteins used in this Example and the following Examples were prepared using procedures well known in the art for removal of N-linked glycans (see, for example, Biochem. J. (2001) 358:423-430). See also Tai, T., et al. J. Biol. Chem. (1975) 250, 8569.<br>
Example 10 Specific Activity of CBH2 variants on PASC<br>
[299]   This example examines the specific performance on phosphoric acid swollen cellulose of CBH2 variants as compared to wild-type H. jecorina CBH2 that had been cloned into A. niger.<br>
[300]   Phosphoric acid swollen cellulose (PASC) • PASC was prepared from Avicel according to the method described in Walseth (1971) Tappi 35:228 (1971) and Wood Biochem J. 121:353 (1971). This material was diluted with buffer and water to achieve a 1 % w/v mixture such that the final concentration of sodium acetate was 50mM, pH 5.0. [301]   The relative specific performance of these variants on celluloslc substrates was determined by techniques known in the art. See, for example, Baker et al, Appl Biochem Biotechnol 1998 Spring; 70-72():395-403..<br>
[302]   A standard cellulosic conversion assay was used in the experiments. See Baker, supra. In this assay enzyme and buffered substrate were placed in containers and incubated at a temperature over time. The reaction was quenched with enough 100 mM Glycine, pH 11.0 to bring the pH of the reaction mixture to at least pH10. Once the reaction was quenched, an aliquot of the reaction mixture was filtered through a 0.2 micron membrane to remove solids. The filtered solution was then assayed for soluble sugars by HPLC according to the methods described in Baker et al., Appl. Biochem. Biotechnol. 70-72:395-403(1998).<br>
[303]   The relative specific activity of these variants was determined with 1 % PASC in 50 mM NaOAc pH 5.0 at 53°C with 1400 rpm shaking for 3.5 hours. The enzymes were dosed at 0.75,1.5, and 3 mg/ gram of cellulose. The protein concentration was determined by OD 280 as in Leach and Scheraga 1960 (J. Am. Chem. Soc. 82:4790-4792). Variants that were compared were FCA500.3, FCA523, FCA536, and FCA540-550. For simplicity, Figure 8 only shows FCA540, FCA542, FCA545, FCA547, FCA549, and FCA550. All other variant samples had specific activities bounded by the lines defined by the FCA542 and FCA545 results.<br>
[304]   Several of the new CBH2 variants from the temperature stability screen are as active as wild type.<br>
[305]   To compare the numbers from the dose dependent data above (as shown in Rgure 8), the ratio of the (average) total sugar produced by a variant and (average) total sugar produced by FCA500.3 (wild-type) at the same dose were averaged. These ratios, presented in Figure 9, are all very similar except for the much less active FCA547, FCA549<br>
and FCA550. The error bars are single standard deviation of the average of the ratios.  A ratio of 1 would indicate that the variant has similar activity to the wild-type in this assay. All of the stabilized variants retained activity on this substrate. •<br>
Example 11 Specific Activity of CBH2 variants on PCS .<br>
[306]   This example compares the specific activity on pretreated com stover of the CBH2 variants as compared to wild-type H. jecorina CBH2 that had been cloned into A niger. [307]   Pretreated corn stover (PCS1 - Com stover was pretreated with 2% w/w H2SO4 as<br>
i	'      *<br>
described in Schell, D. et al., J. Appl. Biochem. Biotechnol. 105:69 - 86 (2003) and followed by multiple washes with deionized water to obtain a paste having a pH of 4.5. Sodium acetate buffer (pH 5.0) was then added (to a final concentration of 50mM sodium acetate) and, if necessary, this mixture was then titrated to pH 5.0 using 1N NaOH. The cellulose concentration in the reaction mixture was approximately 7%. The specific performance of the CBH2 was tested using PCS at 53 °C with 700 rpm for 20 hours. Three different doses of CBH2 variants, 0.75,1.5 and 2.5 mg/g cellulose (in the PCS) were added to 8.5 mg CBH2-deleted cellulase strain broth/g cellulose. (For a discussion of deleting the CBH2 gene in Hypocrea jecorina (also referred to as Trichoderma reeset) see U.S. Patent Nos. 5,861,271 and 5,650,322.) Results are shown in Figure 10. The baseline activity of the CBH2-deleted strain (with no CBH2 added back) is shown. The CBH2 variant has similar activity to the CBH2 wild-type in a reconstituted whole cellulase on PCS. This shows that wild-type when added back to a deleted strain gives a certain activity above the deleted strain. The variant reaches approximately the same activity under similar conditions. [308]   Similar assays were run for other variants as described above. The values of total sugar for duplicates were averaged at each dose then divided this value by the average of the corresponding duplicates for FCA500.3 (wild-type). These ratios, presented in Figure 11, are all very similar except for the much less active FCA547, FCA549 and FCA550. The error bars are single standard deviation of the ratios at different doses. A ratio of 1 would indicate that the variant has similar activity to the wild-type in this assay. All of the stabilized variants retained activity on this substrate.<br>
Example 12 CBH2 Variant Specific Activity at Various Temperatures<br>
[309]   This example demonstrates how long each of the enzymes (stabilized variants and wild-type) remained active at various temperatures.<br>
 [310]    The assays described in Example 10 were used in .this Example as modified below. Total sugar produced by CBH2 (0.5 mg/g cellulose) in 1% PASC at 53,65, and 75°C with 300 RPM shaking over various incubation times was used to determine how long each of these enzymes (stabilized variants and wild-type) remained active at these temperatures. [311]   At 53°C the variant possessed approximately the same activity as wild-type enzyme over time (see Figure 12).  Due to the stability of the enzymes at 53C the half-lives of the enzymes could not be determined from the data. At 65°C, the total sugar produced by FCA543 and FCA500 shows that FCA543 is active for a longer period of time than FCA500 (Figure 13). The half-life of the variant was determined to be approximately 24 hours whereas the wild-type half-life was approximately 4 hours. However, both enzymes begin to fail within the 72-hour incubation time. At 75°C, FCA543 produces more sugar than FCA500 in the first hour (see Rgure 14).<br>
Example 13 CBH2 Variant Specific Activity with Other Cellulases<br>
[312]   This example demonstrates the use of the variant (i.e., stabilized) CBH2 in biomass<br>
conversion in combination with other cellulases.<br>
[313]   A three enzyme mixture with Addothermus cellulolyticus E1 core (see WO<br>
05/093050) plus either wild-type CBH1 and 2 (FCA301 and FCA500, respectively) or<br>
stabilized CBH1 and 2 (FCA469 and FCA543, respectively) was tested at both 5 mg/g<br>
cellulose and 10 mg/g cellulose and at 38,53 and 65°C in the standard conversion assay<br>
using PCS as a substrate (see Example 11). Samples were quenched at one, two and five<br>
days.<br>
[314]   CBH1 variants are described in US Patent Publication No. 20050054039, herein<br>
incorporated by reference. The E1 enzyme from Addothermus cellulolyticus is disclosed in<br>
US Patent No. 5,712,142. Reference is also made to the following patent documents WO<br>
91/05039; WO 93/15186; US Patent No. 5,275,944; WO 96/02551; US Patent No. 5,536,65<br>
and WO 00/70031. Also reference is made to GenBank U33212.<br>
[315]   The results show that the specific performance of the variant mix is about the same<br>
as that of the wild type mix at 38°C. (See Figure 15) A similar pattern for performance is<br>
seen at 53°C (data not shown).<br>
[316]   The stabilized variant mix shows a significant increase in specific performance over<br>
the wild type mix at 65°C. (See Figure 16)<br>
[317]   Using the same standard conversion assay as described in Example 11 the specific<br>
performance of the stabilized variant mix was tested at 56,59 and 62°C at 5 and 10 mg/g<br>
cellulose and samples quenched at 24,48 and 120 hours. At all three times 56°C was better<br>
 <br>
than the higher temperatures. See Figure 17. The optimum temperature is below 59°C at all times tested.<br>
[318]   Various modifications and variations of the described methods and system of the invention will be apparent to those skilled in the art without departing from the scope and spirit of the invention. Although the invention has been described in connection with specific preferred embodiments, it should be understood that the invention as claimed should not be unduly limited to such specific embodiments. Indeed, various modifications of the described modes for carrying out the inventbn which are obvious to those skilled in molecular biology or related fields are intended to be within the scope of the claims.<br>
<br>
<br>
<br>
<br>
CLAIMS<br>
1.	A variant CBH2 cellulase, wherein said variant comprises a substitution or deletion at<br>
a position corresponding to one or more of residues V94, P98, G118, M120, M134, T142,<br>
L144, M145, T148, T154, L179, Q204, V206, S210,1212, T214, L215, G231, T232, V250,<br>
Q276, N285, S291, G308, T312, S316, V323, N325,1333, G334, S343, T349, G360, S380,<br>
A381, S386, F411, S413, A416, Q426 and/or A429 in CBH2 from Hypocrea jecorina (SEQ<br>
ID NO: 2).<br>
2.	A variant CBH2 cellulase according to Claim 1, wherein said variant comprises a<br>
substitution at a position corresponding to one or more of residues V94E, P98L, G118P,<br>
M120L, M134(G/L/V), T142V, L144(G/R/S), M145L, T148Y, T154A, L179A, Q204E, V206L,<br>
S210(L/R), 1212V, T214(M/Y), L215I, G231N, T232V, V250I, Q276L, N285Q, S291G,<br>
G308A, T312S, S316P, V323(L/N/Y), N325D, 1333L, G334A, S343P, T349L, G360R,<br>
S380T, A381T, S386P, F411Y, S413Y, A416G, Q426E and/or A429T in CBH2 from<br>
Hypocrea jecorina (SEQ ID NO: 2).<br>
3.	A variant CBH2 cellulase, wherein said variant comprises a substitution at a position<br>
corresponding to one or more of residues V94, P98, G118, M120, M134, T142, L144, M145,<br>
7148, T154, L179, Q204, V206, S210,1212, T214, L215, G231, T232, V250, Q276, N285,<br>
S291, G308, T312, S316, V323, N325,1333, G334, S343, T349, G360, S380, A381, S386,<br>
F411, S413, A416, Q426 and/or A429 in CBH2 from Hypocrea jecorina (SEQ ID NO: 2).<br>
4.	A variant CBH2 cellulase, wherein said variant comprises a substitution at a position<br>
corresponding to a residue selected from the group consisting of V94E, P98L, G118P,<br>
M120L, M134L, T142V, L144R, M145L, T148Y.T154A, L179A, Q204E, V206L, S210R,<br>
1212V, T214Y, L215I, G231N, T232V, V250I, Q276L, N285Q, S291G, G308A, T312S, S316P,<br>
V323(N/Y), N325D, I333L, G334A, S343P, T349L, G360R, S380T, A381T, S386P, F411Y,<br>
S413Y, A416G, A429T and Q426E in CBH2 from Hypocrea jecorina (SEQ ID NO: 2).<br>
5.	A variant CBH2 cellulase, wherein said variant CBH2 consists essentially of the<br>
mutations selected from the group consisting of<br>
i.	I212V/S316P/F411Y;<br>
ii.	M134G/L144G/S316P;<br>
iii.	M134LA144R/S316P;<br>
iv.	M134L/L144S/S316P;<br>
v.	M134V/V206L/I212V/T312S/S316P/F411Y/S413Y;<br>
vi.	P98L/M134LA144R/S21OL/T214Y/S316PA/323Y/S413Y<br>
vii.	P98L7M134L/L144R/S21OR/T214Y/S316PA/323Y/S413Y<br>
 <br>
viii.    P98L/M134IA144R/S316P/S413Y<br>
Ix.      P98L/M134lA144R/S316P/V323Y/S413Y<br>
x.       P98L/M134L/L144RA/206L/S210R/T214Y/S316P/S413Y<br>
xi.      P98L/M134LA144RA/206LyS210FVr214Y/S316PA/323Y/S413Y<br>
xii.     P98L/M134V/I212V/S316P/S413Y<br>
xlii.    P98L/M134V/I212V/T312S/S316P/S413Y<br>
xlv.    P98L/M134V/S316P<br>
xv.     P98L/M134V/S316P/S413Y<br>
xvj.    P98L/M134V/S316P/V323Y/S413Y<br>
xvif.   P98iyMi 34WT154A/1212V/S316P/lr411Y/S413Y<br>
xviii.  P98LyM134V/T154A/l212V/S316P/S413Y<br>
xix.    P98L;M134V/T154A/I212V/T312S/S316P/S413Y<br>
XX.     P98L/M134V^154A/T312S<br>
xxi.    P98UM134V/T154AA^206M212V/S316P/F411Y/S413Y<br>
xxii.   P98L/M134V/T154A/V206L'S316P<br>
xxiii.  P98L/M134VA/206L/F411Y<br>
xxiv.   P98L/M134VA/206L/1212V/S316P/F411Y<br>
xxv.   P98L/M134VA/206Lyi212V/T312S/S316P/S413Y<br>
xxvi.   P98L/M134VA/206L/S210R/T214Y/S316P/S413Y<br>
xxvii. P98UM134VA/206L7S21OR/T214Y/S316PA/323Y/S413Y<br>
xxviii. P98L/M134VA/206US316P/S413Y<br>
xxlx.   P98L/T154A/l212V/F411Y<br>
xxx.   P98LA/206L/I212V/T312S/S316P/F411Y/S413Y<br>
xxxi. S316P/S413Y<br>
xxxii. S316PA/323L<br>
xxxiii.S316P/V323Y<br>
xxxiv. V206L/I212V/S316P<br>
xxxv. V206L/I212V/T312S/S316P<br>
xxxvi. V206L/I212V/T312S/S316P/F411Y/S413Y<br>
xxxvii. V206L/I212V/T312S/S316P/S413Y<br>
xxxviii. V206L/I212WT312S/S316P/S413Y<br>
xxxix. V206LyS21 OI/T214M/S316P<br>
xl.      V206LyS210R/S316P<br>
xli.     V206L/S21OR/T214Y/S316P; and<br>
xlli.    V206L/S316P;<br>
 <br>
in CBH2 from Hypocrea Jecorina (SEQ ID NO:2).<br>
6.	A nucleic acid encoding a CBH2 variant according to claim 1.<br>
7.	A nucleic acid encoding a CBH2 variant according to claim 4.<br>
8.	A nucleic acid encoding a CBH2 variant according to claim 5.<br>
9.	A vector comprising a nucleic acid encoding a CBH2 variant of claim 6.<br>
10.	A vector comprising a nucleic acid encoding a CBH2 variant of claim 7.<br>
11.	A vector comprising a nucleic acid encoding a CBH2 variant of claim 8.<br>
12.	A host cell transformed with the vector of claim 9.<br>
13.	A host cell transformed with the vector of claim 10.<br>
14.	A host cell transformed with the vector of claim 11.<br>
15.	A method of producing a CBH2 variant comprising the steps of:<br>
<br>
(a)	culturing the host cell according to claim 12 in a suitable cultur<br>
medium under suitable conditions to produce CBH2 variant;<br>
(b)	obtaining said produced CBH2 variant.<br>
16.	A method of producing a CBH2 variant comprising the steps of:<br>
(a)	culturing the host cell according to claim 13 in a suitable cultur<br>
medium under suitable conditions to produce CBH2 variant;<br>
(b)	obtaining said produced CBH2 variant.<br>
17.	A method of'producing a CBH2 variant comprising the steps of:<br>
(a)	culturing the host cell according to claim 14 in a suitable culture<br>
medium under suitable conditions to produce CBH2 variant;<br>
(b)	obtaining said produced CBH2 variant.<br>
<br>
18.	A detergent composition comprising a surfactant and a CBH2 variant, wherein said<br>
CBH2 variant comprises a CBH2 variant according to claim 1.<br>
19.	The detergent according to claim 18, wherein said detergent is a laundry detergent.<br>
20.	The detergent according to claim 16, wherein said detergent is a dish detergent.<br>
21.	A feed additive comprising a CBK2 variant according to claim 1.<br>
22.	A method of treating wood puip comprising contacting said wood pulp with a CBH2<br>
variant according to claim 1.<br>
23.	A method of converting biomass to sugars comprising contacting said biomass with a<br>
CBH 2 variant according to claim 1.<br>
 <br>
 <br>
<br>
</td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/viewdoc.aspx?id=AB/9HnsvGgnxIaFeSLvLFw==&amp;amp;loc=+mN2fYxnTC4l0fUd8W4CAA==" target="_blank" style="word-wrap:break-word;">http://ipindiaonline.gov.in/patentsearch/GrantedSearch/viewdoc.aspx?id=AB/9HnsvGgnxIaFeSLvLFw==&amp;amp;loc=+mN2fYxnTC4l0fUd8W4CAA==</a></p>
		<br>
		<div class="pull-left">
			<a href="272883-configurations-and-methods-for-power-generation-in-lng-regasification-terminals.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="272885-a-method-of-treating-a-fluid-with-an-anti-microbial-agent.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>272884</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>4414/DELNP/2007</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>19/2016</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>06-May-2016</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>29-Apr-2016</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>08-Jun-2007</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>GENENCOR INTERNATIONAL,INC.,</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td>925 PAGE MILL ROAD, PALO ALTO, CA 94304 USA.</td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>AEHLE WOLFGANG</td>
											<td>C/O GENENCOR INTERNATIONAL,INC.,925 PAGE MILL ROAD, PALO ALTO, CALIFORNIA 94304 USA</td>
										</tr>
										<tr>
											<td>2</td>
											<td>CALDWELL ROBERT M.,</td>
											<td>C/O GENENCOR INTERNATIONAL,INC.,925 PAGE MILL ROAD, PALO ALTO, CALIFORNIA 94304 USA</td>
										</tr>
										<tr>
											<td>3</td>
											<td>DANKMEYER LYDIA</td>
											<td>C/O GENENCOR INTERNATIONAL,INC.,925 PAGE MILL ROAD, PALO ALTO, CALIFORNIA 94304 USA</td>
										</tr>
										<tr>
											<td>4</td>
											<td>GOEDEGEBUUR FRITS</td>
											<td>C/O GENENCOR INTERNATIONAL,INC.,925 PAGE MILL ROAD, PALO ALTO, CALIFORNIA 94304 USA</td>
										</tr>
										<tr>
											<td>5</td>
											<td>KELEMEN BRADLEY R.,</td>
											<td>C/O GENENCOR INTERNATIONAL,INC.,925 PAGE MILL ROAD, PALO ALTO, CALIFORNIA 94304 USA</td>
										</tr>
										<tr>
											<td>6</td>
											<td>MITCHINSON COLIN</td>
											<td>C/O GENENCOR INTERNATIONAL,INC.,925 PAGE MILL ROAD, PALO ALTO, CALIFORNIA 94304 USA</td>
										</tr>
										<tr>
											<td>7</td>
											<td>NEEFE PAULIEN</td>
											<td>C/O GENENCOR INTERNATIONAL,INC.,925 PAGE MILL ROAD, PALO ALTO, CALIFORNIA 94304 USA</td>
										</tr>
										<tr>
											<td>8</td>
											<td>TEUNISSEN PAULINE</td>
											<td>C/O GENENCOR INTERNATIONAL,INC.,925 PAGE MILL ROAD, PALO ALTO, CALIFORNIA 94304 USA</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>C12N 9/42</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>PCT/US2005/047266</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td>2005-12-22</td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td>60/656,863</td>
									<td>2005-02-25</td>
								    <td>U.S.A.</td>
								</tr>
								<tr>
									<td>2</td>
									<td>60/640,398</td>
									<td>2004-12-30</td>
								    <td>U.S.A.</td>
								</tr>
								<tr>
									<td>3</td>
									<td>60/666,072</td>
									<td>2005-03-28</td>
								    <td>U.S.A.</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/272884-novel-variant-hypocrea-jacorina-cbh2-cellulases by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 01:40:50 GMT -->
</html>
