<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/278035-transgenic-plant-event-detection by HTTrack Website Copier/3.x [XR&CO'2014], Thu, 04 Apr 2024 23:59:38 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 278035:TRANSGENIC PLANT EVENT DETECTION</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">TRANSGENIC PLANT EVENT DETECTION</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>The present invention relates to detection of materials derived from transgenic plant events. Inparticular, the invention provides methods, reagents, kits and reference materials for detecting the presence or absence in a sample of genetic material derived from and attributable to select transgenic plant events.</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td>TRANSGENIC PLANT EVENT DETECTION<br>
FIELD OF THE INVENTION<br>
The present invention relates to detection of materials derived from transgenic plant events. In particular, the invention provides methods, reagents, kits and reference materials for detecting the presence or absence in a sample of genetic material derived from and attributable to select transgenic plant events.<br>
BACKGROUND OF THE INVENTION<br>
Genetically modified organisms (GMOs), in particular genetically modified plants, are gaining importance in agriculture as well as in the production and marketing of foods and feed.<br>
However, due to existing concerns regarding the impact of GMOs on the environment and on consumers' health, the introduction of new GMOs commonly requires regulatory approval and, in many countries including the European Union, food and feed products containing or produced from GMOs are themselves subjected to authorisation and compulsory labelling (see, e.g., Regulation (EC) no. 1829/2003. Off J Eur Communities: Legis. 2003, L268: 1-23; Regulation (EC) no. 1830/2003. Off J Eur Communities: Legis. 2003, L268: 24-28).<br>
Consequently, tracing of GMOs in the environment and through the cultivation process and the food or feed production chain is fundamental for environmental risk assessment, as well as necessary to preserve consumers' confidence and to satisfy or verify the observance of mandatory regulations, including GMO product labelling.<br>
This requires the availability of methods that can determine the presence, identity and/or quantity of GMOs or materials originating or derived therefrom in, for example, food and feed sources, ingredients and/or processed consumables. DNA-based methods are very useful in this respect, not least because DNA often withstands physical and chemical food processing treatments better than other molecules, such as proteins. For example, real-time polymerase chain reaction (PCR) proved to be a specific and sensitive technique for the quantification of nucleic acids indicative of the presence of DNA originating from GMOs.<br>
Assays are available to unambiguously conclude the presence or absence of genetic material derived from a select plant transformation event in a sample. Commonly, such assays can detect the presence of a unique, event-specific nucleotide sequence found in the plant transformation event of interest but absent from other events. For example, such distinctive<br>
 <br>
nucleotide sequences may be present at junctions between the GMO's endogenous genomic sequence and the sequence of the transforming gene construct at the genomic insertion site of the latter; detection may for instance involve PCR amplification across the said junction using specific primers flanking the junction (see, e.g., Hernandez et al. 2003. Transgenic Res 12: 179-189, Hernandez et a!. 2004. J Cereal Sci 39: 99-107; Berdal et al. 2001. Eur Food Res Technol 213: 432-438; Nielsen et al. 2004. Eur Food Res Technol 219: 421-427; or Ronning et al. 2003. Eur Food Res Technol 216: 347-354).<br>
However, these event-specific assays can often be procured only as highly priced kits allowing for a limited number of tests, are confined to detection of material from a single event, and commonly employ very particular reaction conditions. Yet, the number of generated, authorised and marketed and GMOs keeps increasing. Hence, an unambiguous determination of the GMO composition of a sample would in principle require to apply an ever-growing battery of separate event-specific detection assays on each sample, which is both laborious and cost demanding.<br>
Therefore, while it continues to be desirable to employ event-specific assays due to the unambiguous nature of their output, there exists a need for using these assays in a more effective and targeted manner, such as to improve the detection of GMOs detection, e.g., in terms of time expense, cost and labour intensity.<br>
SUMMARY OF THE INVENTION<br>
In aspects, the present invention provides methods, reagents, kits and reference materials that address one or more of the above discussed needs of the art.<br>
In general, the invention hence provides improved methods, reagents, kits and reference materials for detecting material derived from GMOs, and preferably from genetically modified plants, in samples.<br>
More particularly, the inventors realised that by carefully choosing certain sequence features to be assayed in the sample - wherein the said sequence features need not be event-specific and may in fact be shared between two or more events - one can conclude whether the sample potentially contains material derived from one or more transformation events or, conversely, whether the sample does not contain material derived from one or more such transformation events. The assays of the invention can thus provide a valuable indication of the potential GMOs contents of the sample and thereby advantageously reduce the number<br>
 <br>
of subsequent tests (such as, e.g., event-specific assays), needed to verify the presence of material derived from one or more particular transformation events in the sample; this can in turn improve the cost-, time- and/or labour-effectiveness of GMOs detection methods. For example, the assays of the invention may allow to considerably reduce the number of event-specific assays needed to characterise the GMOs composition of the sample.<br>
The inventors also meticulously assessed the known, commercially significant and/or authorised transformation events to select sequence features for being assayed in a sample, such as to reduce the number of individual test reactions needed per sample yet ensuring adequate informativeness of the GMO profiling according to the invention.<br>
The present invention integrates the above relevant realisation in its diverse aspects.<br>
Hence, in a particularly preferred aspect (herein aspect "A1") the invention concerns a method to examine a sample for the presence or absence of material derived from one or more transgenic plant events comprising the steps of:<br>
(1) detecting the presence or absence in the sample of nucleic acids comprising or consisting of:<br>
a)	"Zm": a nucleic acid derived from and specific for Zea mays taxon, preferably Zea mays ssp. mays,<br>
b)	"Bn": a nucleic acid derived from and specific for Brassica napus taxon,<br>
c)	"Gm": a nucleic acid derived from and specific for Glycine max taxon, and one, more than one or all of nucleic acids chosen from:<br>
d)	"p35S": a nucleic acid derived from the 35S promoter of Cauliflower Mosaic Virus,<br>
e)	"tNOS":  a nucleic acid  derived from the 3' terminator of the Agrobacterium<br>
tumefaciens nopaline synthetase gene,<br>
f)	"CrylAb": a nucleic acid derived from the crystal protein gene CrylAb of Bacillus thuringiensis,<br>
g)	"PAT/bar": a nucleic acid derived from the phosphinothricin acetyltransferase (PAT) gene oar of Streptomyces hygroscopicus,<br>
h) "PAT/pat": a nucleic acid derived from the phosphinothricin acetyltransferase (PAT) gene par of Streptomyces viridochromogenes, and<br>
i) "CP4-EPSPS": a nucleic acid derived from the 5-Enol-pyruvylshikimate-3-phosphate synthase EPSPS gene from Agrobacterium sp. CP4; and<br>
 <br>
 (2) concluding the presence or absence in the sample of material derived from one or more transgenic plant events, such as events chosen from the group comprising or consisting of: events Bt176, Bt11, Bt10, MON810, MON863, TC1507, NK603, T25, GA21, DAS-59122, crosses thereof, and related events thereof; events Topas 19/2, MS1, RF1, RF2, RF3, MS8, GT73, T45, Liberator pHoe6/Ac, GS40/90pHoe6/Ac, crosses thereof including MS1/RF1, MS1/RF2, MS8/RF3, and related events thereof, events MON 40-3-2, MON89788, A2704-12, A5547-127, crosses thereof, and related events thereof.<br>
Aspect A1 allows to investigate the potential presence or absence in a sample of material from numerous transgenic plant events belonging to several distinct plant taxons, while performing a relatively limited number of detection steps. Advantageously, the transgenic plant events detected by the method of aspect A1 encompass a large fraction, or even substantially most, transgenic plant events presently authorised and/or commercialised, e.g., in Europe, i.e., transgenic plant events which are particularly likely to be encountered in, e.g., environmental or commercial samples. Hence, in this aspect the invention provides a relatively undemanding assay for inspecting a considerable spectrum of relevant transgenic plant events.<br>
Preferably, step (1) of aspect A1 can involve detecting the presence or absence in the sample of nucleic acids comprising or consisting of all nucleic acid listed under a) - i) as defined above. This advantageously improves the information obtained by testing the sample and permits to conclude the potential presence or absence of material derived from a substantially high number of transgenic plant events.<br>
Whereas in step (2) of aspect A1 detection is particularly foreseen for particular existing transgenic plant events, it shall be appreciated that other events, including any future transgenic events to be generated, can also be detected insofar they belong to the taxons as defined under a) - c) and comprise one or more of the nucleic acids as defined under d) - i). This observation applies mutatis mutandis to detection of events in any aspects of the invention as described in the following.<br>
While the method of aspect A1 permits to simultaneously examine the presence or absence in the sample of transgenic plant events belonging to at least three different taxons, the inventors also envisage adapting the method to individual taxons. This may usefully reduce the number of nucleic acid detection steps if information is only required for a particular taxon.<br>
 <br>
Accordingly, in a further aspect ("A2") the invention provides a method to examine a sample for the presence or absence of material derived from one or more maize transgenic plant events comprising the steps of:<br>
(1)	detecting the presence or absence in the sample of nucleic acids comprising or consisting of nucleic acid listed under a) as defined above and one, more than one or all of nucleic acids chosen from those listed under d) - i) as defined above; and<br>
(2)	concluding the presence or absence in the sample of material derived from one or more maize transgenic plant events, such as events chosen from the group comprising or consisting of events Bt176, Bt11, Bt10, MON810, MON863, TC1507, NK603, T25, GA21, DAS-59122, crosses thereof, and related events thereof.<br>
Preferably, step (1) of aspect A2 can involve detecting the presence or absence in the sample of nucleic acids comprising or consisting of all nucleic acid listed under a) and d) - i) as defined above. This increases the information obtained by testing the sample and permits to conclude the potential presence or absence of material derived from a comparably higher number of maize transgenic plant events.<br>
In a further aspect ("A3") the invention offers a method to examine a sample for the presence or absence of material derived from one or more oilseed rape transgenic plant events comprising the steps of:<br>
(1)	detecting the presence or absence in the sample of nucleic acids comprising or consisting of nucleic acid listed under b) as defined above and one, more than one or all of nucleic acids chosen from those listed under d) - i) as defined above; and<br>
(2)	concluding the presence or absence in the sample of material derived from one or more oilseed rape transgenic plant events, such as events chosen from the group comprising or consisting of events Topas 19/2, MS1, RF1, RF2, RF3, MS8, GT73, T45, Liberator pHoe6/Ac, GS40/90pHoe6/Ac, crosses thereof including MS1/RF1, MS1/RF2, MS8/RF3 and related events thereof.<br>
Preferably, step (1) of aspect A3 can involve detecting the presence or absence in the sample of nucleic acids comprising or consisting of all nucleic acids listed under b) and d) - i) as defined above. This increases the information obtained by testing the sample and permits to<br>
 <br>
conclude the potential presence or absence of material derived from a comparably higher number of oilseed rape transgenic plant events.<br>
In another preferred embodiment, step (1) of aspect A3 involves detecting the presence or absence in the sample of nucleic acids comprising or consisting of nucleic acid listed under b) and d), e), and g) - i) as defined above. It is noted that many oilseed* rape events, e.g., those specifically recited in step (2) of aspect A3, may not include nucleic acid listed under f) as defined above. Therefore, omission of the said nucleic acid listed under f) from step (1) of aspect A3 may reduce the number of tests without loss of information.<br>
In a further aspect ("A4") the invention offers a method to examine a sample for the presence or absence of material derived from one or more soybean transgenic plant events comprising the steps of:<br>
(1)	detecting the presence or absence in the sample of nucleic acids comprising or consisting of nucleic acid listed under c) as defined above and one, more than one or all of nucleic acids chosen from those listed under d) - i) as defined above; and<br>
(2)	concluding the presence or absence in the sample of material derived from one or more soybean transgenic plant events, such as events chosen from the group comprising or consisting of events MON 40-3-2, MON89788, A2704-12, A5547-127, crosses thereof, and related events thereof.<br>
Preferably, step (1) of aspect A4 can involve detecting the presence or absence in the sample of nucleic acids comprising or consisting of all nucleic acids listed under c) and d) - i) as defined above. This increases the information obtained by testing the sample and permits to conclude the potential presence or absence of material derived from a comparably higher number of soybean transgenic plant events.<br>
In another preferred embodiment, step (1) of aspect A4 involves detecting the presence or absence in the sample of nucleic acids comprising or consisting of nucleic acid listed under c) and d), e), h) and i) as defined above. It is noted that many soybean events, e.g., those specifically recited in step (2) of aspect A4, may not include nucleic acids listed under f) and g) as defined above. Therefore, omission of the said nucleic acids listed under f) and g) from step (1) of aspect A4 may reduce the number of tests without loss of information.<br>
 <br>
It shall be also appreciated that further aspects can teach combinations of any two of the above aspects A2 - A4. For example, this may advantageously reduce the number of testing actions in cases where detection of transgenic events from select taxons is intended.<br>
Accordingly, in a further aspect ("A5") the invention offers a method to examine a sample for the presence or absence of material derived from one or more maize and/or oilseed rape transgenic plant events comprising the steps of:<br>
(1)	detecting the presence or absence in the sample of nucleic acids comprising or consisting of nucleic acids listed under a) and b) as defined above and one, more than one or all of nucleic acids chosen from those listed under d) - i) as defined above; and<br>
(2)	concluding the presence or absence in the sample of material derived from one or more maize and/or oilseed rape transgenic plant events, such as events chosen from the group comprising or consisting of events Bt176, Bt11, Bt10, MON810, MON863, TC1507, NK603, T25, GA21, DAS-59122, crosses thereof, and related events thereof; and events Topas 19/2, MS1. RF1, RF2, RF3. MSB, GT73, T45, Liberator pH.oe6/Ac, GS40/90pHoe6/Ac, crosses thereof including MS1/RF1, MS1/RF2, MS8/RF3 and related events thereof.<br>
For reasons clarified above, step (1) of aspect A5 can preferably involve detecting the presence or absence in the sample of nucleic acids comprising or consisting of a!! nucleic acids listed under a), b) and d) - i) as defined above.<br>
In yet a further aspect ("A6") the invention offers a method to examine a sample for the presence or absence of material derived from one or more maize and/or soybean transgenic plant events comprising the steps of:<br>
(1)	detecting the presence or absence in the sample of nucleic acids comprising or consisting of nucleic acids listed under a) and c) as defined above and one, more than one or all of nucleic acids chosen from those listed under d) - i) as defined above; and<br>
(2)	concluding the presence or absence in the sample of material derived from one or more maize and/or soybean transgenic plant events, such as events chosen from the group comprising or consisting of events Bt176, Bt11, Bt10, MON810, MON863, TC1507, NK603, T25, GA21, DAS-59122, crosses thereof, and related events thereof; and events MON 40-3-2, MON89788, A2704-12, A5547-127, crosses thereof, and related events thereof.<br>
 <br>
presence or absence in the sample of nucleic acids comprising or consisting of all nucleic acids listed under a), c) and d) - i) as defined above.<br>
In yet a further aspect ("A7") the invention offers a method to examine a sample for the presence or absence of material derived from one or more oilseed rape and/or soybean transgenic plant events comprising the steps of:<br>
(1)	detecting the presence or absence in the sample of nucleic acids comprising or consisting of nucleic acids listed under b) and c) as defined above and one, more than one or all of nucleic acids chosen from those listed under d) - i) as defined above; and<br>
(2)	concluding the presence or absence in the sample of material derived from one or more oilseed rape and/or soybean transgenic plant events, such as events chosen from the group comprising or consisting of events Topas 19/2, MS1, RF1, RF2, RF3, MS8, GT73, T45, Liberator pHoe6/Ac, GS40/90pHoe6/Ac, crosses thereof including MS1/RF1, MS1/RF2, MS8/RF3 and related events thereof; and events MON 40-3-2, MON89788, A2704-12, A5547-127, crosses thereof, and related events thereof.<br>
For reasons clarified above, step (1) of aspect A7 can preferably involve detecting the presence or absence in the sample of nucleic acids comprising or consisting of all nucleic acids listed under b), c) and d) - i) as defined above.<br>
In another preferred embodiment, step (1) of aspect A7 involves detecting the presence or absence in the sample of nucleic acids comprising or consisting of all nucleic acids listed under b), c) and d), e) and g) - i) as defined above, i.e.. without detecting the nucleic acid listed under f).<br>
Hence, in an aspect ("A8") the invention offers a method to examine a sample for the presence or absence of material derived from one or more maize and/or oilseed rape and/or soybean transgenic plant events comprising the steps of:<br>
(1) detecting the presence or absence in the sample of nucleic acids comprising or consisting of:<br>
- one, more than one or all of nucleic acids chosen from those listed under a), b) and c) as defined above, and<br>
 <br>
- one, more than one or all of nucleic acids chosen from those listed under d) - i) as defined above; and<br>
(2) concluding the presence or absence in the sample of material derived from one or more transgenic plant events, such as events chosen from the group comprising or consisting of: events Bt176, Bt11, Bt10, MON810, MON863, TC1507, NK603, T25, GA21, DAS-59122, crosses thereof, and related events thereof; events Topas 19/2, MS1, RF1, RF2, RF3, MS8, GT73, T45, Liberator pHoe6/Ac, GS40/90pHoe6/Ac, crosses thereof including MS1/RF1, MS1/RF2, MS8/RF3, and related events thereof; and events MON 40-3-2, MON89788, A2704-12, A5547-127. crosses thereof, and related events thereof.<br>
For reasons clarified above, step (1) of aspect A8 can preferably involve detecting the presence or absence in the sample of nucieic acids comprising all nucleic acids listed under d) - i) as defined above.<br>
As explained, methods of the above aspects A1 - A8 are well-suited for detecting the potential presence or absence in a sample of material derived from relevant transgenic piant events, such as ones that have been authorised and/or are commonly used. However, it shall be appreciated that the said methods are also advantageously flexible and, in embodiments, allow for better assessing the presence or absence of material derived from further transgenic events, such as events not specifically recited in step (2) of the above aspects A1 - A8. This thus permits to gather even more information about the composition of the sample.<br>
Hence, in embodiments, step (1) of any of the above aspects A1, A2, A5, A6 or A8 may further comprise detecting the presence or absence in the sample of nucleic acid j) "mCry3A": a nucleic acid derived from the modified crystal protein gene Cry3A of Bacillus thuringiensis; and the group of events as defined in step (2) of any of the respective aspects further comprises maize event MIR604, crosses thereof with other maize events, and events related to MIR604.<br>
In further embodiments, step (1) of any of the above aspects A1, A2, A5, A6 or A8 may further comprise detecting the presence or absence in the sample of one or both nucleic acids k) "cordapA": a nucleic acid derived from the lysine-insensitive dihydrodipicolinate synthase (cDHDPS) gene cordapA of Corynebacterium glutamicum and/or I) "Glb1": a nucleic acid derived from the Glb1 promoter of maize; and the group of events as defined in step (2)<br>
 <br>
of any of the respective aspects further comprises maize event LY038, crosses thereof with other maize events, and events related to LY038.<br>
In yet further embodiments, step (1) of any of the above aspects A1, A2, A5, A6 or A8 may further comprise detecting the presence or absence in the sample of nucleic acid m) "Cry3Bb1": a nucleic acid derived from the crystal protein gene Cry3BM of Bacillus thuringiensis; and the group of events as defined in step (2) of any of the respective aspects further comprises maize event MON88017, crosses thereof with other maize events, and events related to MON88017.<br>
Accordingly, in embodiments, step (1) of any of the above aspects A1, A2, A5, A6 or A8 may further comprise detecting the presence or absence in the sample of one, more than one, or all of nucleic acids listed under j) - m) as defined above; and the group of events as defined in step (2) of any of the respective aspects may further comprise one, more than one, or all of the maize events MIR604, LY038 and MON88017, crosses thereof with other maize events, and events related thereto. However, whereas detection of the respective nucleic acids listed under j) - m) as defined above can provide for improved detection of the presence of material from the maize events MIR604, LY038 and/or MON88017 in the sample, the detection of nucleic acids listed under d) - i) as defined above already allows to draw conclusions concerning the potential presence of material from said events MIR604, LY038 and/or MON88017 in a sample (see also Table 3).<br>
In embodiments, step (1) of any of the above aspects A1, A3, A5, A7 or A8 may further comprise detecting the presence or absence in the sample of nucleic acid n) "Bxn": a nucleic acid derived from the nitrilase gene Bxn of Klebsiella pneumoniae ssp. ozaenae; and the group of events as defined in step (2) of any of the respective aspects further comprises oilseed rape event OXY235, crosses thereof with other oilseed rape events, and events related to OXY235. However, whereas detection of the nucleic acid listed under n) as defined above can provide for improved detection of the presence of material from the oilseed rape event OXY235 in the sample, the detection of nucleic acids listed under d) - i) as defined above already allows to draw conclusions concerning the potential presence of material from said event OXY235 in a sample (see Table 3).<br>
 <br>
Thus, in an aspect ("A9") the invention offers a method to examine a sample for the presence or absence of material derived from one or more maize and/or oilseed rape and/or soybean transgenic plant events comprising the steps of<br>
(1)	detecting the presence or absence in the sample of nucleic acids comprising or consisting<br>
of:<br>
-	one, more than one or all of nucleic acids chosen from those listed under a), b) and c) as defined above, and<br>
-	one, more than one or all of nucleic acids chosen from those listed under d) - i) as defined above, and<br>
-	optionally, one, more than one or all of nucleic acids chosen from those listed under j) - n) as defined above; and<br>
(2)	concluding the presence or absence in the sample of material derived from one or more<br>
transgenic piant events, such as events chosen from the group comprising or consisting of:<br>
events Bt176, Bt11, Bt10, MON810, MON863, TC1507, NK603, T25, GA21, DAS-59122,<br>
MIR604, LY038 and MON88017, crosses thereof, and related events thereof; events Topas<br>
19/2, MS1, RF1. RF2, RF3, MSB, GT73, T45, Liberator pHoe6/Ac, GS40/90pHoe6/Ac,<br>
OXY235, crosses thereof including MS1/RF1, MS1/RF2, MS8/RF3, and related events<br>
thereof; and events MON 40-3-2, MON89788, A2704-12, A5547-127, crosses thereof, and<br>
related events thereof.<br>
For reasons clarified above, step (1) of aspect A9 can preferably involve detecting the presence or absence in the sample of nucleic acids comprising all nucleic acids listed under d)-i) as defined above.<br>
In further advantageous embodiments, the methods of the above aspects A1 - A8 can thus additionally also assess the potential presence or absence in samples of material derived from transgenic events that belong to other taxons.<br>
Accordingly, in embodiments, step (1) of any of the above aspects A1 - A8 (or of the above embodiments of said aspects) may further comprise detecting the presence or absence in the sample of nucleic acid o) "Or": a nucleic acid derived from and specific for Oryza sativa taxon; and the group of events as defined in step (2) of any of the respective aspects further<br>
 <br>
comprises one, more than one, or all of the rice events LL62, LL06 and LL601. crosses<br>
* thereof, and events related thereto.<br>
In further embodiments, step (1) of any of the above aspects A1 - A8 (or of the above embodiments of said aspects) further comprises detecting the presence or absence in the sample of nucleic acid p) "Bv": a nucleic acid derived from and specific for Beta vulgaris taxon; and the group of events as defined in step (2) of any of the respective aspects further comprises one, more than one, or all of the sugar beet events T120-7, H7-1 and A5-15, crosses thereof, and events related thereto.<br>
In still further embodiments, step (1) of any of the above aspects A1 - A8 (or of the above embodiments of said aspects) further comprises detecting the presence or absence in the sample of nucleic acid q) "Gs": a nucleic acid derived from and specific for Gossypium taxon; and the group of events as defined in step (2) of any of the respective aspects further comprises one, more than one, or all of the cotton events LL cotton 25, MON 1445, MON 531, MON 15985, crosses thereof, and events related thereto.<br>
In a further development, step (1) of any of the embodiments of the preceding paragraph may further comprise detecting the presence or absence in the sample of one or both of nucleic acids r) "CrylAc": a nucleic acid derived from the crystal protein gene CrylAc of Bacillus thuringiensis and/or s) "Cry2Ab2": a nucleic acid derived from the crystal protein gene Cry2Ab2 of Bacillus thuringiensis; and the group of events as defined in step (2) can particularly well detect one or both of cotton events MON 531, MON15985, crosses thereof with other cotton events, and events related thereto.<br>
In still further embodiments, step (1) of any of the above aspects A1 - A8 (or of the above embodiments of said aspects) further comprises detecting the presence or absence in the sample of nucleic acid t) "St": a nucleic acid derived from and specific for Sotanum tuberosum taxon; and the group of events as defined in step (2) of any of the respective aspects further comprises the potato event EH92-527-1 and events related thereto.<br>
In a further development, step (1) of any of the embodiments of the preceding paragraph may further comprise detecting the presence or absence in the sampie of nucleic acid u) "GBSS": a nucleic acid derived from the granule bound starch synthase gene Gbss of Soianum tuberosum, further improving the reliability of detection of material from said event EH92-527-1.<br>
 <br>
It shall be appreciated that the preceding methods may be performed in various suitable combinations, allowing desired events to be detected and reducing the effort.<br>
Thus, in an aspect ("A10") the invention offers a method to examine a sample for the presence or absence of material derived from one or more maize and/or oilseed rape and/or soybean and/or rice and/or sugar beet and/or cotton and/or potato transgenic plant events comprising the steps of:<br>
(1)	detecting the presence or absence in.the sample of nucleic acids comprising or consisting<br>
of:<br>
-	one, more than one or all of nucleic acids chosen from those listed under a), b), c), o), p) q) and t) as defined above, and<br>
-	one, more than one or all of nucleic acids chosen from those listed under d) - i) as defined above, and<br>
-	optionally, one more than one or all of nucleic acids chosen from those listed under j) - n), r) s) and u) as defined above, and<br>
(2)	concluding the presence or absence in the sample of material derived from one or more<br>
transgenic plant events, such as events chosen from the group comprising or consisting of:<br>
events Bt176, Bt11, Bt10, MON810, MON863, TC1507, NK603, T25, GA21, DAS-59122,<br>
MIR604, LY038, MON88017, crosses thereof, and related events thereof; events Topas 19/2,<br>
MS1, RF1, RF2, RF3, MS8, GT73, T45, Liberator pHoe6/Ac, GS40/90pHoe6/Ac, OXY235,<br>
crosses thereof including MS1/RF1, MS1/RF2, MS8/RF3, and related events thereof; events<br>
MON 40-3-2, MON89788, A2704-12, A5547-127, crosses thereof, and related events thereof;<br>
events LL62, LL06 and LL601, crosses thereof, and events related thereto; events T120-7,<br>
H7-1 and A5-15, crosses thereof, and events related thereto; events LL cotton 25, MON<br>
1445, MON 531, MON15985, crosses thereof, and events related thereto; and event EH92-<br>
527-1 and events related thereto.<br>
For reasons clarified above, step (1) of aspect A TO can preferably involve detecting the presence or absence in the sample of nucleic acids comprising all nucleic acids listed under d) - i) as defined above!<br>
 <br>
In a related preferred aspect ("A11") the invention offers a method to examine a sample for the presence or absence of material derived from one or more maize and/or oilseed rape and/or soybean and/or rice and/or sugar beet and/or cotton and/or potato transgenic plant events comprising the steps of:<br>
(1)	detecting the presence or absence in the sample of nucleic acids comprising or consisting<br>
of:<br>
-	one, more than one or all of nucleic acids chosen from those listed under a), b), c), o), p) q) and t) as defined above, and<br>
-	one. more than one or all of nucleic acids chosen from those listed under d) - i) as defined above; and<br>
(2)	concluding the presence or absence in the sample of material derived from one or more<br>
transgenic piant events, such as events chosen from the group comprising or consisting of:<br>
events Bt176, Bt11, Bt10, MON810, MON863, TC1507, NK603, T25, GA21, DAS-59122,<br>
optionally MIR604, LY038 and MON88017, crosses thereof, and related events thereof;<br>
events Topas 19/2, MS1, RF1, RF2, RF3, MS8, GT73, T45, Liberator pHoe6/Ac,<br>
GS40/90pHoe6/Ac, optionally OXY235. crosses thereof including MS1/RF1, MS1/RF2,<br>
MS8/RF3, and related events thereof; events MON 40-3-2, MON89788, A2704-12, A5547-<br>
127, crosses thereof, and related events thereof; events LL62, LL06 and LL601, crosses<br>
thereof, and events related thereof; events T120-7, H7-1 and A5-15, crosses thereof, and<br>
events related thereto; events LL cotton 25, MON 1445, MON 531, MON15985, crosses<br>
thereof, and events related thereto; and event EH92-527-1 and events related thereto.<br>
For reasons clarified above, step (1) of aspect A11 can preferably involve detecting the presence or absence in the sample of nucleic acids comprising all nucleic acids listed under d)-i) as defined above.<br>
Preferably, in any of the methods of the above aspects A1 to A11 or of any embodiment of said aspects, the presence or absence of nucleic acids in a sample is detected using amplification of amplicons comprised in the respective nucleic acids, such as, preferably using PCR and, more preferably using real-time PCR.<br>
 <br>
To allow for efficient use of PCR amplification in the above methods, the present inventors meticulously searched for and tested primers and primer pairs so as to select ones particularly advantageous in said methods<br>
The identified preferred primer pairs are listed in Table 1. They entail inter alia the following significant advantages. The primers display substantially same or satisfactorily similar annealing temperature, thus allowing the PCR amplification on the different nucleic acids to be performed at the same temperature cycling conditions and hence in the same apparatus. This greatly facilitates the throughput of the methods and reduces the amount of labour, disposables and apparatus needed. The primer pairs produce amplicons of about 100 bp, and thereby allow to amplify the target nucleic acids even from samples in which the genetic material is largely fragmented, as may be the case for processed food or feed. The primers and primer pairs show adequate specificity of amplification, as well as sensitivity suitable for real-time amplification. In addition, to ensure sensitivity and uniformity of the methods and kits provided by the invention, the primers and primer pairs have been so designed as to generate, at 40 cycles and using 10000 copies of a template, fluorescence of at least 2500 relative fluorescence unit (RFU) and wherein the so-generated fluorescence values for the different primer sets preferably lies within a 3-fold margin. In addition, each primer pair has been so designed as to amplify on the respective nucleic acids from substantially all events of interest, despite the existence of potential sequence disparities (e.g.. due to optimised codon usage) in said nucleic acids between such events.<br>
Table 1. Advantageous primer pairs for detection of nucleic acids listed under a) - i), o), p), q) and t) as defined above, by PCR.<br>
(Table Removed)<br>
Hence, the invention provides methods as defined in any one of the above aspects A1 to A11<br>
or in any embodiment of said aspects, wherein the presence or absence of nucleic acids from among those listed under a) - i), o), p), q) and t) as defined above in a sample is detected using PCR amplification using the respective primer pairs as shown in Table 1.<br>
Where Table 1 lists more than one forward and/or reverse primer for amplification on a certain nucleic acid (such as, e.g., for "CP4-EPSPS"), any one or any combination of said forward primer(s) may be used in conjunction with any one or with any combination of said reverse primer(s) to obtain amplification.<br>
One shall appreciate that while the primer sequences listed in Table 1 have been perfected to provide for optimal amplification, the present methods can perform adequately when using variant primers that display a certain restricted degree of sequence variation vis-a-vis the primers in Table 1.<br>
Hence, the above aspects and embodiments also encompass methods wherein the nucleic acids of interest are detected by PCR amplification using variant primers that include one or more sequence variations vis-a-vis the corresponding primers listed in Table 1, such as, e.g., one or more deletion, insertion and/or substitution, insofar such variant primers / primer pairs can still achieve adequate amplification of their respective amplicons. Preferably, such variant primers would show at least 85%, more preferably at least 90%, even more preferably at least 95%, and yet more preferably at least 96%, at least 97%, at least 98% or at least 99%<br>
sequence identity to the primers listed in Table 1. Sequence alignments and determination of sequence identity can be done, e.g., using the Basic Local Alignment Search Tool (BLAST) originally described by Altschul et al. 1990 (J Mol Biol 215: 403-10), such as the "Blast 2 sequences" algorithm described by Tatusova and Madden 1999 (FEMS Microbiol Lett 174: 247-250).<br>
One shall further appreciate that the primers listed in Table 1 or variants thereof may be advantageously derivatised. For example, said primers or variants thereof may be labelled to allow detection of the amplified products, such as, e.g., in real-time PCR applications. Hence, such derivatisation may for example entail introducing one or mpre label moiety and/or one or more accessory moiety (e.g., a quenching moiety, a FRET moiety, etc.), and optionally where required modifying the primer sequence to allow for the introduction and/or proper working of said moiety or moieties. A skilled person is generally knowledgeable about ways to modify primers and primer pairs for purposes of labelling. The use of so-derivatised primers and primer pairs can be particularly useful in real-time PCR, and particularly where two or more differently labelled amplification products are to be followed in multiplexed amplification reactions.<br>
Hence, the above aspects and embodiments also encompass methods where the nucleic acids of interest are detected by PCR amplification using derivatives of the primers and primer pairs listed in Table 1 or of variants thereof.<br>
By means of an example, in a preferred aspect ("A12"), the invention offers a method to examine a sample for the presence or absence of material derived from one or more transgenic plant events (in particular one or more maize and/or oilseed rape and/or soybean and/or rice and/or sugar beet and/or cotton and/or potato transgenic plant events) comprising the steps of:<br>
(1) detecting the presence or absence in the sample of nucleic acids comprising or consisting<br>
of:<br>
-	one, more than one or all of nucleic acids chosen from those listed under a), b), c), o), p) q) and t) as defined above, and<br>
-	one, more than one or all of nucleic acids chosen from those listed under d) - i) as defined above; and<br>
(2) concluding the presence or absence in the sample of material derived from one or more transgenic plant events, such as events chosen from the group comprising or consisting of: events Bt176, Bt11, Bt10, MON810, MON863, TC1507, NK603, T25, GA21, DAS-59122, optionally MIR604, LY038 and MON88017, crosses thereof, and related events thereof; events Topas 19/2, MS1, RF1, RF2, RF3, MS8, GT73, T45, Liberator pHoe6/Ac, GS40/90pHoe6/Ac, optionally OXY235, crosses thereof including MS1/RF1, MS1/RF2, MS8/RF3, and related events thereof; events MON 40-3-2, MON89788, A2704-12, A5547-127, crosses thereof, and related events thereof; events LL62, LL06 and LL601, crosses thereof, and events related thereof; events T120-7, H7-1 and A5-15, crosses thereof, and events related thereto; events LL cotton 25, MON 1445, MON 531, MON15985, crosses thereof, and events related thereto; and event EH92-527-1 and events related thereto,<br>
wherein in step (1) the presence or absence of nucleic acids in a sample is detected using PCR amplification using the respective primers and primer pairs as shown in Table 1, or variants or derivatives thereof.<br>
For reasons clarified above, step (1) of aspect A12 can preferably involve detecting the presence or absence in the sample of nucleic acids comprising all nucleic acids listed under d) - i) as defined above. In a preferred embodiment, step (1) of aspect A12 can preferably involve detecting the presence or absence in the sample of nucleic acids comprising at least nucleic acids listed under a) - i) as defined above.<br>
In an aspect, the invention further provides primers and primer pairs and variants or derivatives thereof suitable for amplification of amplicons from nucleic acids comprising or consisting of those listed under a) - u) above. In particular, in view of the significant effort invested by the present inventors to the identification of the primer pairs listed in Table 1, the invention provides the primers and primer pairs listed in Table 1, as well as variants and derivatives thereof, optionally in any suitable combination of said primers and/or primer pairs or variants or derivatives thereof.<br>
Additionally, the invention also provides amplification products obtainable using the preferred primers and primer pairs, vectors and plasmids comprising said amplification products, and recombinant microorganisms transformed with and harbouring said vectors and plasmids.<br>
Consequently, in preferred embodiments, the invention also relates to recombinant E. coli as deposited under the Budapest Treaty with the Belgian Coordinated Collections of Microorganisms (BCCM), Laboratorium voor Moleculaire Biologie - Plasmidencollectie (BCCM/LMBP) (Universiteit Gent, Technologiepark 927, B-9052 Gent-Zwijnaarde, Belgium) on January 10, 2007 under LMBP accession numbers LMBP 5452, LMBP 5453, LMBP 5454, LMBP 5455, LMBP 5456, LMBP 5457, LMBP 5458, LMBP 5459 and LMBP 5460, on March 6, 2007 under LMBP accession number LMBP 5451, and on April 19, 2007 under LMBP accession numbers LMBP 5587, LMBP 5588, LMBP 5589 and LMBP 5590, as well as to isolated recombinant plasmids obtainable from the said recombinant E. coli, as well as to isolated inserts of the said plasmids, preferably isolated EcoRI-EcoRI inserts thereof.<br>
Such plasmids or vectors or inserts thereof, and their combinations, are particularly useful as positive controls, references and/or calibrators for the respective amplicons in the amplification reactions of the above methods.<br>
In addition, the invention also covers kits comprising one or more reagents useful in the methods of one or more of the above aspects and embodiments. For example, such kits can comprise one or more of the above primers and/or primer pairs, particularly as listed in Table 1, or variants or derivatives of said primers, and/or one or more of the above amplification products, vectors or plasmids comprising such, or inserts of the said vectors or plasmids, and/or a data carrier comprising instructions for a programmable computing device to carry out the step (2) of the methods of the invention (see below). Such kits may further conveniently contain instructions for the user to perform the methods of the invention and to interpret the data so-obtained.<br>
The invention also teaches an uncomplicated and straightforward way to perform step (2) of the above aspects and embodiments, i.e., to conclude the potential presence or absence in the sample of material derived from the various transgenic plant events of interest. In particular, the results acquired for a sample in step (1) of the above aspects and embodiments are represented as a set "GSW consisting of, i.e., representing a collection of, those nucleic acids chosen from ones comprising or consisting of those listed under a) - u) above that were detected in the said step (1) as being present in the sample (step (i)).<br>
Moreover, any transgenic plant event of interest can be represented as a set "Gx" (where X designates the event of interest), consisting of, i.e., representing a collection of, those nucleic acids chosen from ones comprising or consisting of those listed under a) - u) above that are found in the said event and thus would be detected in step (1) if the sample contained material derived from the said event, or a cross involving such, or a related event (step (ii)).<br>
In particular exemplary embodiments, the set Gx may preferably be chosen from sets<br>
representing the events specifically recited above, e.g., comprising sets: maize event sets<br>
(Formula Removed)<br>
 wherein the designations between brackets {} refer to those nucleic acids as iisted under a) - u) above that are found in such events and may thus be detected in materials derived there from. It is to be understood that if the detection step (1) involves additional nucleic acids other than those listed under a) - u) above, such nucleic acids can be advantageously included in the above sets.<br>
It shall be understood that where the above methods do not test the presence of all nucleic acids under a) - u) above, the sample set as well as the event sets can be defined in terms of<br>
those nucleic acids the presence of which is effectively tested. By means of example and not limitation, where the presence of nucleic acids listed under j) - n), r) s) and u) as defined above, which correspond to specific traits in addition to those defined under d) - i) is not tested, then the sets corresponding to several events may be defined in a narrower manner, in particular: maize event sets <br>
(Formula Removed)<br>
Hence, the presence of material derived from an event of interest X in the sample can then be determined by performing for the respective set of interest Gx logical operations as follows (step iii):<br>
-	if Gx equals GSAM (GX = GSAM), then material derived from the transgenic plant event X, or from a cross thereof, or from an event related thereto, is potentially present in the sample;<br>
-	if Gx is a proper subset of GSAM (Gx c GSAM), then material derived from transgenic plant event X, or from a cross thereof, or from an event related thereto, is potentially present in the sample;<br>
-	if Gx does not equal GSAM and Gx is not a proper subset of GSAM, (GX ^ GSAM and Gx cr GSAM), then material derived from transgenic plant event X, or from a cross thereof, or from an event related thereto, is absent from the sample.<br>
In preferred embodiments, these logical operations can be advantageously carried out by a computing device.<br>
It shall be appreciated that whereas the above algorithm for use in step (2) of the present methods is herein explained in connection with concluding the presence of material from certain events using detection of particular nucleic acids, said algorithm is in principle readily applicable to methods which aim to conclude the presence of any events, such as events other than or additional to those specifically mentioned above, using detection of any nucleic acids, such as nucleic acids other than or additional to those listed under a) - u) above.<br>
These and further aspects and preferred embodiments of the invention are described in the following sections and in the appended claims.<br>
BRIEF DESCRIPTION OF FIGURES<br>
Figure 1 (A-F) illustrates select sequences of amplicons amplified using primers sets listed in Table 4 and present in cloning vectors, particularly in pUC18, such as plasmids listed in Table 5. As can be appreciated, the sequences may include partial sequences from multiple cloning sites flanking the inserted amplicons.<br>
Figure 2 (A) results of the multiplex PCR test for p35S/tNOS, (B) Dissociation curve of p35S/tNOS specific multiplex test.<br>
Figure 3 provides graphical representation of the data using +/- 8300 template copies as listed in Table 8.<br>
DETAILED DESCRIPTION OF THE INVENTION<br>
As used herein, the singular forms "a", "an", and "the" include both singular and plural referents unless the context clearly dictates otherwise.<br>
The terms "comprising", "comprises" and "comprised of as used herein are synonymous with "including", "includes" or "containing", "contains", and are inclusive or open-ended and do not exclude additional, non-recited members, elements or method steps.<br>
The term "about" as used herein when referring to a measurable value such as a parameter, an amount, a temporal duration, and the like, is meant to encompass variations of +/-20% or less, preferably+/-10% or less, more preferably +1-5% or less, even more preferably+/-1 % or less, and still more preferably +/-0.1% or less of and from the specified value, insofar such variations are appropriate to perform in the disclosed invention. It is to be understood that the value to which the modifier "about" refers is itself also specifically, and preferably, disclosed.<br>
The recitation of numerical ranges by endpoints includes all numbers and fractions subsumed within that range, as well as the recited endpoints.<br>
All documents cited in the present specification are hereby incorporated by reference in their entirety. In particular, the teachings of all documents herein specifically referred to are incorporated by reference.<br>
Hence, in some aspects, more particularly aspects A1 - A12 as described in the Summary section, the invention concerns methods to examine a sample for the presence or absence of material derived from one or more transgenic plant events comprising the steps (1) detecting<br>
the presence or absence in the sample of nucleic acids comprising or consisting of one, more than one or all (the particular choice depending on the goal of a given assay) of nucleic acids listed under a) - u) in the Summary section and (2) concluding the presence or absence in the sample of material derived from one or more transgenic plant events chosen from the group comprising or consisting of some or all of those detailed in the Summary section.<br>
A "transgenic plant event" occurs with an independent transformation of plant cells with heterologous nucleic acid, typically a nucleic acid construct comprising one or more transgenes of interest; regeneration of a population of plants resulting from an insertion of the construct or a part thereof including the said transgene(s) of interest into the genome of the plants; and selection of a particular plant characterised by the said insertion into one or more, preferably one, particular genome location. The term "transgenic plant event" thus encompasses the original so-selected transformant plant, as well as any successive progeny thereof comprising the inserted nucleic acid including the transgene(s) of interest, for example, the said progeny may be hemizygous or homozygous for the said insertion; the said progeny may be obtained, e.g., by vegetative propagation or by sexual crossing.<br>
The term "plant" as used herein generally encompasses plant cells; plant protoplasts; plant tissues; plant cells or tissue culture from which a plant organism can be regenerated; plant calii or clumps; plant organisms and parts thereof, such as, without limitation, flowers, tepals, petals, sepals, anthers, pollen, seeds, fruit, pericarp, pods, leaves, petioles, stems, roots, rhizomes, stolons, tubers, shoots, and the like. The term as used herein generally encompasses plants of any taxons classified in the art within the kingdom Plantae. In addition, plant as understood herein may preferably belong to land plants (embryophytes) including, without limitation, non-vascular plants (bryophytes) and vascular plants (tracheophytes); more preferably, to vascular plants (tracheophytes) including, without limitation, lycopodiophyta, equisetophyta, pteridophyta, psilotophyta, ophioglossophyta, and seed plants (spermatophytes); even more preferably to seed plants (spermatophytes) including, without limitation, seed-bearing plants (gymnosperms) such as, e.g., pinophyta, cycadophyta, ginkgophyta and gnetophyta and flowering plants (angiosperms) such as magnoliophyta, including monocots (liliopsida) and dicots (Magnoliopsida). Preferred plants may be ones commonly applied in agriculture or horticulture. By means of example and not limitation, preferred crop plants can include maize, wheat, rice, barley, sorghum, tobacco,<br>
tomato, potato, oilseed rape (rapeseed), soybean, sugar beet, pea, sunflower, cotton, peanut, flowers (e.g., carnation), etc.<br>
The term "material" generally refers to physical matter. The term "material derived from a transgenic plant event" encompasses the transgenic plant event as such, including, by means of example and not limitation, the transgenic plant event organism; any parts of the transgenic plant event organism, such as, e.g., flowers, tepals, petals, sepals, anthers, pollen, seeds, fruits, pericarp, pods, leaves, petioles, stems, roots, rhizomes, stolons, tubers or shoots, or portions thereof; cells, protoplasts, tissues, calli or clumps of the transgenic plant event; or material obtained by subjecting any of the above to one or more downstream processing steps. These downstream processing steps may comprise any actions used to process particular plant materials in agriculture, horticulture or in any downstream industries, e.g., food industry including beverage industry, feed industry, textile industry (e.g., cotton, linen, bamboo, etc.), fuel industry (e.g., oilseed rape), lubricant industry (e.g., castor oil), paint industry (e.g., linseed oil, tung oil), cosmetic and pharmaceutical industry, etc.<br>
By means of example and not limitation, such downstream processing may include harvesting; removal of unwanted outer layers, e.g.. peeling, skinning, etc.; drying, e.g., air drying, sun drying, spray drying, freeze drying, juice concentrating, etc.; diminution, e.g., grinding, milling, chopping, slicing, etc.; liquefaction, e.g., juice collection; preserving, e.g., vacuum bottling, tinning, canning, pasteurization, addition of preservatives, etc.; pressing, e.g., oil collection; hydrogenation, e.g., segetable oil saturation; macerating; emulsifying; heat treatment, e.g., cooking, pressure cooking, steaming, oven baking, smoking, frying, grilling, etc.; fermenting, e.g., by yeast, bacteria and/or fungi; freezing; and the like.<br>
The term "sample" as used herein refers broadly to a representative part or fraction of a larger material whole being presented for inspection, such as, e.g., for quality control. Preferably, a sample to be examined by methods of the invention may be suspected of potentially comprising material derived from one or more transgenic events of interest. By means of example, such suspicion may exist for any sample representative of material that is known as comprising, or suspected of probably or possibly comprising, plants or parts thereof or material derived therefrom, in a preferred example, the sample may be representative of material that is known as comprising, or suspected of probably or possibly comprising, plants or parts thereof or material derived therefrom, wherein the said plants or parts thereof belong to the same taxon as a transgenic event of interest whose presence in the sample is to be<br>
 <br>
examined. For example, a sample may be representative of plants or parts thereof, e.g., collected or harvested plants or parts thereof (such as, without limitation, fruits, seeds, pods, flowers, etc.), or representative of an intermediary or final material obtained by applying one or more downstream processing steps thereto, or of products comprising such material, e.g., of processed food or feed products.<br>
Hence, in an embodiment the sample can comprise plants or parts thereof, including flowers, tepals, petals., sepals, anthers, pollen, seeds, fruits, pericarp, pods, leaves, petioles, stems, roots, rhizomes, stolons, tubers or shoots, or portions thereof, plant cells, plant protoplasts and/or plant tissues, and/or plant-derived material, preferably food or feed material, including processed food or feed material.<br>
The methods of the invention detect the presence or absence in the sample of nucieic acids, i.e., of genetic material and preferably genomic DNA, originating or derived from transgenic plant events of interest. The presence or absence of such nucleic acids is indicatory of, respectively, the presence or absence in the sample of this and potentially further material derived from the respective transgenic plant events; such as, e.g., materials co-purifying or co-separating with the said nucleic acids during the various processing steps applied to the transgenic plant events.<br>
It is known that nucleic acids, and particularly genomic DNA, are comparably durable and can tolerate a variety of processing steps applied to plant materials in agriculture and industry, e.g., in food or feed industry, such that DNA molecules of lengths that permit sequence-specific detection thereof may be found following such steps and even in final products.<br>
Accordingly, a preferred sample of the invention comprises nucleic acids, more preferably genomic DNA, even more preferably genomic DNA having at least sizes that allow for sequence specific-detection thereof, such as. e.g., on average, at least about 20 bp, e.g., at least about 30 bp, preferably at least about 50 bp, e.g., at least about 75 bp, more preferably at least about 100 bp, e.g. about 150 bp, or even more preferably at least about 200 bp, or at least about 300 bp, or at least about 500 bp, or at least about 1 kb, or more.<br>
Although it can be assumed that most processing steps to which a transgenic plant event would be commonly exposed in practice would preserve detectable nucleic acids as above in the eventual product, and thus in the sample thereof to be examined, the inventors also<br>
 <br>
envisage a positive control aimed at verifying whether the said sample comprises detectable plant-derived nucleic acids.<br>
In particular, the method of the invention may further comprise detecting the presence or absence in the sample of a generic plant-derived nucleic acid. In a preferred embodiment, such generic plant-derived nucleic acid is derived from a gene present in and preferably highly conserved (e.g., &gt; 80%, preferably &gt; 90%, more preferably &gt; 95%, even more preferably &gt; 96%, &gt; 97%, &gt; 98% or &gt; 99% sequence identity) between various plant taxons; preferably at least within spermatophytes; or at least within angiosperms, such as between and/or within monocots and dicots: or preferably at least between plant taxons commonly used in agriculture and industry, such as, without limitation, between maize, wheat, rice, barley, sorghum, tobacco, tomato, potato, oilseed rape (rapeseed), soybean, sugar beet, pea, sunflower, cotton, peanut and flowers (e.g., carnation); or preferably at least between plant taxons comprising or consisting of those of the transgenic plant events tested in the sample.<br>
In preferred embodiments, the said generic plant-derived nucleic acid is derived from the chloroplastic small subunit of Rubisco gene or from the CHL-tRNA synthetase gene.<br>
As mentioned, the methods of the invention include in step (1) detecting the presence or absence in the sample of nucleic acids comprising or consisting of one, more than one or all of those listed under a) - u) in the Summary section. The term "detecting" or "detection" as used herein encompasses any means of determining the presence or absence of a given nucleic acid in the sample.<br>
Preferably, the presence of a particular nucleic acid molecule in a sample can be detected using one or more reagents specifically hybridising to respective nucleotide sequence(s) comprised within the said nucleic acid molecule.<br>
The terms "hybridisation" and "hybridise" refer to a process by which a nucleic acid strand anneals with complementary or substantially complementary sequence(s) comprised in the same or another polynucleotide strand through base pairing, preferably Watson-Crick base pairing.<br>
The terms "complementary" and "complementarity" refer to the normal binding of polynucleotides under permissive salt (ionic strength) and temperature conditions by base pairing, preferably Watson-Crick base pairing. By means of example, complementary Watson-Crick base pairing occurs between the bases A and T, A and U or G and C. For<br>
 <br>
example, the sequence A-G-T (i.e., 5'-A-G-T-3') is thus complementary sequence A-C-T (i.e., 5'-A-C-T-3').<br>
The "degree of complementarity" of a nucleic acid strand (A) to a nucleic acid strand (B) can be expressed as the proportion (percentage) of nucleotides of the nucleic acid strand (A) that would be expected to match, i.e., form Watson-Crick base-pairing, with nucleotides of the nucleic acid strand (B), when the said nucleic acid strands (A) and (B) were annealed, preferably in high stringency conditions.<br>
"Complementary" as used herein thus refers to wholly complementary, such that all respective nucleotides of the nucleic acid strands would bind when the strands anneal. By means of example, a relatively shorter nucleic acid strand would show total complementarity to a relatively longer nucleic acid strand, if the latter strand comprised a sequence fully complementary to the sequence of the former strand. "Substantially complementary" refers to largely but not wholly complementary, in particular, to at least 85% complementary, e.g., at least 90% complementary, preferably at least 95% complementary, e.g., at least 96%, 97%: 98% or at least 99% complementary.<br>
"Specifically hybridise" and "specific hybridisation" reflects a situation when a reagent hybridises to a given sequence more readily than it would hybridise to a random, unrelated sequence. For example, a reagent specifically hybridising to a given nucleotide sequence (A) preferably displays little or no hybridisation to other polynucleotides, and preferably to homologues or orthologues of the nucleic acids sequence (A), under conditions where it would specifically hybridise with the said polynucleotide (A).<br>
Preferably, reagents that can specifically hybridise to the respective nucleotide sequence(s) comprised within the nucleic acid molecules listed under a) - u) above can be probes or primers.<br>
A "probe" is an isolated nucleic acid to which is desirably attached a detectable label or reporter moiety, e.g., a radioactive isotope (e.g., 32P, 33P), ligand, chemiluminescent agent, fluorophore (e.g., fluorescein, tetrachloro-fluorescein, TAMRA, ROX, Cy3, Cy3.5, Cy5, Cy5.5, Texas Red, etc.), vitamin (e.g., biotin), steroid (e.g., digoxin), enzyme (e.g., HRP, AP, etc.), etc. Such a probe is complementary or substantially complementary to a sequence comprised in a strand of a target nucleic acid, in the case of the present invention, of a nucleic acid, preferably genomic DNA, listed under a) - u) above. Probes according to the present<br>
 <br>
invention may be deoxyribonucleic acids, ribonucleic acids, or nucleic acids comprising both deoxyribo- and ribonucleotides; may comprise standard purine and/or pyrimidine bases (A, G, T, C, U and/or I), but also other natural, chemically or biochemically modified (e.g., methylated), non-natural, or derivatised nucleotide bases; may include backbone comprising  sugars and phosphate groups, as can typically be found in RNA or DNA, as well as one or more modified or substituted (such as, 2'-0-alkylated, e.g., 2'-0-methylated or 2'-0-ethylated; or 2'-0,4'-C-alkynelated, e.g., 2'-0,4'-C-ethylated) sugars or one or more modified or substituted phosphate groups - for example, backbone analogues in nucleic acids may include phosphodiester, phosphorothioate, phosphorodithioate, methylphosphonate, phosphoramidate, alkyl phosphotriester, sulfamate, 3'-thioacetal, methylene (methylimino), 3'-N-carbamate, morpholino carbamate, and peptide nucleic acids (PNAs).<br>
Probes may be at least 10 nucleotides in length, e.g., at least 11, at least 12, at least 13 or at least 14 nucleotides in length, preferably at least 15 nucleotides, e.g., at least 16, at least 17, at least 18 or at least 19 nucleotides in length, more preferably at least 20 nucleotides in length, e.g., at least 21, at least 22, at least 23, or at least 24 nucleotides in length, even more preferably at least 25 nucleotides in length, e.g.. at least 30, at least 35, at least 40, at least 45, at least 50, at least 75, at least 100 or at least 200 or more nucleotides in length.<br>
"Primers" are isolated nucleic acids, preferably deoxyribonucleic acids or. in other preferred examples, ribonucleic acids or PNAS, complementary or substantially complementary to a sequence comprised in a target nucleic acid (in the case of the present invention, to a sequence comprised in a nucleic acid, preferably genomic DNA, comprising or consisting of those listed under a) - u) above), that can be annealed to the said target nucleic acid and can act as a point of initiation of synthesis of a primer extension product in the presence of nucleotides and an agent for nucleic acid polymerization, such as DNA dependent or RNA dependent polymerase.<br>
A primer needs to be sufficiently long to prime the synthesis of an extension products in the presence of an agent for polymerization. A typical primer may thus be at least 10 nucleotides in length, e.g., at least 11, at least 12, at least 13 or at least 14 nucleotides in length, preferably at least 15 nucleotides in length, e.g., at least 16, at least 17, at least 18 or at least 19 nucleotides in length, more preferably at least 20 nucleotides in length. Further preferred primers are between about 10 and about 40 nucleotides in length, more preferably between<br>
 <br>
about 15 and about 30 nucleotides in length, most preferably between about 20 and about 25 nucleotides long.<br>
"Primer pairs" refer to combinations of primers which are suited for amplification of amplicons from within target nucleic acids (in the case of the present invention, from within nucleic acids, preferably genomic DNA, comprising or consisting of those listed under a) - u) above) by suitable nucleic-acid amplification methods. Accordingly, the ability to amplify an amplicon from within a given target nucleic acid using a primer pair designed to specifically hybridise within the said target nucleic acid indicates the presence (and optionally quantity) of the said target nucleic acid in the sample.<br>
Exemplary but non-limiting methods for preparing and using probes and primers are described, for example, in Molecular Cloning: A Laboratory Manual, 2nd ed.. vol. 1-3, ed. Sambrook el al., Cold Spring Harbor Laboratory Press, Cold Spring Harbor, N.Y., 1989 ("Sambrook et al. 1989"); Current Protocols in Molecular Biology, ed. Ausubel et al., Greene Publishing and Wiley-lnterscience, New York, 1992 (with periodic updates) ("Ausubel et al. 1992"); and Innis et al., PCR Protocols: A Guide to Methods and Applications, Academic Press: San Diego, 1990. PCR-primer pairs can be derived from a known sequence, for example, by using computer programs intended for that purpose such as Primer3 (Rozen and Skaletsky. 2000. Primer3 on the WWW for general users and for biologist programmers. In: Krawetz S, Misener S (eds) Bioinformatics Methods and Protocols: Methods in Molecular Biology. Humana Press, Totowa, NJ, pp 365-386) or the GCG v. 11.1.2 package from Accelrys.<br>
Specific hybridisation of the probes, primers or primer pairs of the invention to their respective complementary sequences within the target nucleic acids, e.g., those comprising or consisting of those listed under a) - u) above, can be preferably achieved under high stringency hybridisation conditions.<br>
"High stringency" conditions include conditions equivalent to the following exemplary conditions for binding or hybridisation at 65 C in a solution consisting of 5xSSPE (43.8 g/l NaCI, 6.9 g/l NaH2PO4.H2O and 1.85 g/l EDTA, pH adjusted to 7.4 with NaOH), 0.1% SDS, 5xDenhardt's reagent (50xDenhardt's contains per 500 ml: 5 g Ficoll (Type 400, Pharmacia), 5 g BSA (Fraction V; Sigma) and 100 ug/ml denatured salmon sperm DNA), followed by washing in a solution comprising 5xSSPE, 0.1% SDS at 65 C when a probe of about 500<br>
 <br>
nucleotides in length is employed. Other exemplary conditions for hybridisation at "high stringency" for nucleic acid sequences over approximately 20-100 nucleotides in length, preferably about 30-100 nucleotides in length, e.g., between 30-50 nucleotides in length, include conditions equivalent to hybridisation in 6xSSC at 45C, followed by one or more washes in 0.2xSSC, 0.1% SDS or even O.lxSSC, 0.1% SDS at 65C. Numerous equivalent conditions may be employed to vary stringency conditions; factors such as the length and nature (DNA, RNA. base composition) of the probe and nature of the target (DNA, RNA, base composition, present in solution or immobilised, etc.) and the concentration of the salts and other components (e.g., the presence or absence of formamide, dextran sulphate, polyethylene glycol) are considered and the hybridisation solution may be varied to generate conditions of low or high stringency hybridisation different from, but equivalent to, the above listed conditions. In addition, the art knows conditions that promote hybridisation under conditions of high stringency (e.g., increasing the temperature of the hybridisation and/or wash steps, the use of formamide in the hybridisation solution, etc.). Guidance for performing hybridisation reactions can be found, for example, in Current Protocols in Molecular Biology, John Wiley &amp; Sons, N.Y., 6.3.1-6.3.6, 1989, and more recent updated editions, all of which are incorporated by reference.<br>
Regarding the amplification of amplicons from within the target nucleic acids using primer pairs, "stringent conditions" or "high stringency conditions" are conditions that permit a primer pair to hybridise only to its respective target nucleic-acid sequence and to produce a unique amplification product, i.e., the amplicon, substantially without producing non-intended, i.e., non-specific amplification products. For example, preferably less than 20%, more preferably less than 10%, even more preferably less than 5%, e.g., less than 4%, less than 3%, or less than 2%, yet more preferably less than 1%, e.g., less than 0.1% or less than 0.01% of the total quantity (e.g., weight) of amplification product in an amplification reaction would be nonspecific when such reaction is performed at stringent conditions (e.g., as analysed by quantitative gel electrophoresis, or by Tm determination, or the like).<br>
Any conventional methods can be used to detect specific hybridisation of a probe to a target nucleic acid in the sample. For example, nucleic acids, preferably DNA. from a sample may be immobilised and denatured on a solid support and so-hybridised with respective probes (e.g.. Southern blotting, slot blotting, dot blotting, etc.). If a probe - and consequently its<br>
 <br>
detectable label - is retained on the solid support, this indicates that the target nucleic for which the probe is specific is present in the sample.<br>
Similarly, any existing nucleic-acid amplification method can be used to amplify amplicons from within target sequences of interest, thereby indicating the presence of the corresponding target sequences in the sample: including but not limited to the polymerase chain reaction (PCR) (US 4,683.202; US 4,965,188), strand displacement amplification (SDA) (US 5.455,166; EP 0684315). LCR, TAS, 3SR, NASBA (US 5,409,818; EP 0329822), RCA and Q-beta amplification<br>
in a particularly preferred embodiment, the methods of the invention employ polymerase-chain reaction (PCR) to amplify ampiicons from within target nucleic acids chosen from ones comprising or consisting of those listed under a) - u) above. The terms "polymerase-chain reaction" and "PCR" broadly covers any method referred as such in the art and in particular methods for amplification of target nucleic acid sequences, especially target DNA sequences, using heat-stabie DNA polymerase(s) and two primers, especially oligonucleotide primers, one complementary to the (+)-strand at one end of the sequence to be amplified and the other complementary to the (-)-strand at the other end. The term encompasses modifications of the prototypic PCR. such as, e.g., high-fidelity PCR, hot-start PCR. touch-down PCR, nested PCR, multiplex PCR, quantitative PCR, quantitative real-time PCR, long-range PCR, RT-PCR, etc. (see, e.g., PCR Protocols: A Guide to Methods and Applications, eds. Innis et aL, Academic Press, San Diego, 1990).<br>
Amplification products obtained using the above amplification methods, and in particular using PCR, can be evaluated, e g., for their presence or absence, for their specificity and/or quantity, by any of the manners common in the art. For example, the size (as an indication of amplification specificity) and/or quantity of an amplification product may be evaluated using gel electrophoresis, such as, e.g., agarose or polyacrylamide gel electrophoresis, where amplification products are visualised using suitable DNA-binding dyes, such as. e.g.. ethidium bromide.<br>
Alternatively, amplification products may be evaluated by methods that employ probes hybridising to specific sequences within the amplification products. For instance, an amplification product may be immobilised and denatured on a solid support and subsequently hybridised with a labelled probe.<br>
 <br>
Otherwise, the amplification product may itself be labelled, e.g., by including a detectable label (e.g.. a fluorophore) in one or both primers and/or by virtue of substrate nucleotides incorporated into the amplification -product during amplification, and the so-obtained PCR product may be subsequently denatured and hybridised with specific (oligonucleotide) probes attached to a solid support. The presence and quantity of the detectable label in select positions on the solid support thus indicates the specificity and quantity, respectively, of the amplification product. For instance, such set-up is suitable for use with probe arrays, e.g., microarrays.<br>
In further exemplary manners, amplification products may be evaluated using cloning and sequencing; direct sequencing; oiigonucleotide-mediated pyrosequencing (Ahmadian et al. 2000. Anal Biochem 280; 103-110): chromatography, e.g.. DHPLC. oligonucleotide ligation assays (Landegren et al. 1988. Science 241: 1077: Eggerding et al. 1995. Hum Mutat 5: 153-165; Nickerson efal. 1990. PNAS 87: 8923-8927); RNAse Protection Assay: etc.<br>
A particularly advantageous manner of evaluating the amplification product is real-time amplification, especially real-time PCR. The term "real-time amplification" is intended to mean any amplification technique, preferably PCR ("real-time PCR"), which makes it possible to monitor the evolution of an ongoing amplification reaction (see, e.g., Real-Time PCR: An Essential Guide, eds Edwards et al., Horizon Scientific Press, 2004; Mamas SAE et al. 2006. Real-time assays with molecular beacons and other fluorescent nucleic acid hybridization probes. Clin Chim Acta 363: 48-60; for discussion of various real-time PCR platforms).<br>
By means of example and not limitation, real-time amplification, especially real-time PCR, as intended herein encompasses fully conventional systems, such as, e.g., the TaqMan system developed by Applied Biosystems. which relies on the release and detection of a fiuorogenic probe during each round of DNA amplification (Holland et al. 1991. Detection of specific polymerase chain reaction product by utilizing the 5'-3' exonuclease activity of Thermus aquaticus DNA polymerase. PNAS 88: 7276-80). The method uses the 5' exonuclease activity of Taq polymerase during primer extension to cleave a dual-labelled, fiuorogenic probe hybridised to the target DNA between the PCR primers. Prior to cleavage, a reporter fluorophore, such as 6-carboxyfluorescein (6-FAM) at the 5' end of the probe is quenched by 6-carboxy-tetramethylrhodaniine (TAMRA) through fluorescent resonance energy transfer (FRET). Following digestion, FAM is released. The resulting fluorescence<br>
 <br>
measured in real-time at around 518 nm during the log phase of product accumulation is proportional to the number of copies of the target sequence<br>
Further real-time amplification, especially real-time PCR, detection systems can also utilise FRET, such as, e.g., systems based on molecular beacons. Molecular beacons are single-stranded polynucleotide probes that possess a stem-and-loop hairpin structure. The loop portion is a probe sequence complementary to a sequence within an amplicon to be evaluated, and the stem is formed by short complementary sequences located at the opposite ends of the molecular beacon. The molecular beacon is labelled with a fluorophore (e.g., 6-FAM) at one end and a quencher (e.g., TAMRA) at the other end. When free in solution, the stem keeps the fluorophore and the quencher in close proximity, causing the fluorescence of the fluorophore to be quenched by FRET. However, when bound to its complementary target, the probe-target hybrid forces the stem to unwind, separating the fluorophore from the quencher, and restoring the fluorescence. Accordingly, when the quantity of an amplicon increases during amplification, this can be monitored as an increase in the fluorescence of the corresponding beacon (see. e.g., Manganelli et al. 2001. Real-time PCR using molecular beacons. Methods Mol Med 54: 295-310; Marras SAE. 2006. Selection of fluorophore and quencher pairs for fluorescent nucleic acid hybridization probes. Methods Mol Biol 335: 3-16; Marras SAE et al. 2006. Real-time assays with molecular beacons and other fluorescent nucleic acid hybridization probes. Clin Chim Acta 363: 48-60 for further discussion of molecular beacons detection).<br>
A particularly advantageous real-time PCR amplification and detection system for use in the present invention is the Light Upon Extension (LUX) system commercialised by Invitrogen (Carlsbad, CA) and described in detail in Nazarenko et al. 2002 (Nucleic Acids Research 30: e37) and Nazarenko et al. 2002 (Nucleic Acids Research 30: 2089-2095). This system employs primer pairs in which usually one of the primers of said primer pair is labelled by a fluorophore (such as, e.g., FAM or JOE or Alexa Fluor 546). The particular structure of the "free" primer quenches the signal of the fluorophore bound thereto, whereas the fluorophore's signal intensity increases when the primer assumes an extended conformation once incorporated into the amplification product. The sequence of the primers of the present invention, such as of the particularly preferred primers as listed in Tables 1 and 4, may be tailored to perform with the LUX technology, following instructions of the above publications of   Nazarenko   et   al.   2002   or   using   software   tools   provided   by   Invitrogen   on<br>
 <br>
www.invitrogen.com/lux. The LUX technology is particularly well suited for multiplexing (i.e., performing in a single reaction) of two or more amplifications using different primer sets, since each of the primer sets may be marked using a different fluorophore.<br>
For description of additional ways to detect and evaluate amplification products in real-time (e.g.. using adjacent probes; 5'-nuclease probes such as Taqman; Light-up probes; Duplex scorpion primers: Amplifluor primers; and further alternative fluorescent hybridisation probe formats see, e.g.. Marras SAE et al. 2006. Real-time assays with molecular beacons and other fluorescent nucleic acid hybridization probes. Clin Chim Acta 363: 48-60, esp. section 6 and references therein)<br>
In preferred embodiments, the real-time amplification, preferably real-time PCR, of the invention uses (substantially) sequence non-specific reagents to detect and evaluate the accumulation of amplification products. These systems usually employ fluorochromes (fluorescent dyes) that bind to DNA (i.e., DNA-binding), preferably to double-stranded DNA, in a substantially sequence non-specific way, and upon such binding their fluorescence is attained or enhanced. Accordingly, the observed increase in fluorescence is indicative of the appearance and quantity of the accumulated (double-stranded) amplification product. If the said fluorochromes preferentially or exclusively bind to double-stranded DNA and/or if their fluorescence is preferentially or exclusively enhanced when bound to double-stranded DNA, then the specificity of the amplification product can be determined by performing its "melting -curve", i.e., fluorescence vs. temperature plot, and determining the Tm .of the product, i.e., temperature at which 50% of the product is melted and has thus lost its enhanced fluorescence. The observation of a single expected Tm for an amplification product (or multiple expected Tm in multiplex amplification) corroborates the specificity of amplification.<br>
Exemplary fluorochromes for use in the above sequence non-specific detection methods include, without limitation, major-groove binders, minor-groove binders, intercalating dyes, or the like; such as, e.g., SYBR Green I (2-[N-(3-dimethylaminopropyl)-N-propyiamino]-4-[2,3-dihydro-3-methyl-(benzo-1,3-thiazol-2-yl)-methyiidene]-1-phenyl-quinolinium; Zipper et al. 2004. Nucleic Acid, Res 32: e103), ethidium bromide, Hoechst 33258, PicoGreen (Molecular Probes), preferably SYBR Green I or PicoGreen.<br>
In a further preferred embodiment, the real-time amplification reactions of the present methods can be multiplexed, i.e.. two or more different amplicons can be amplified using two or more corresponding primer pairs on the same sample(s) in the same reaction. The primers<br>
 <br>
envisaged by the present inventors, in particular the primers listed in any of Table 1 or Table 4 or variants or derivatives thereof, can be particularly suited for multiplexing applications inter alia because they perform adequately at substantially same or similar reaction conditions (e.g., reaction composition and temperature cycling conditions), as well as because they produce similarly sized products.<br>
It is well-understood that multiplexing generally requires that the different amplification products arising in the reaction can be distinguished. In the present context, if non-sequence specific methods (e.g., SYBR Green or other DNA-binding dye) are used to detect the amplification products, multiplexing can be particularly foreseen where the amplification products have sufficiently dissimilar Tm to be distinguished by performing a melting curve (e.g., Tm differing by about 5C or more). However, where amplicon specific detection methods are used (e.g., differently labelled primers or differently labelled probes for different amplicons), far-going multiplexing of the present tests is in principle possible.<br>
Hence, target for the above described detection using probes and/or template for the amplification using primer pairs, would be nucleic acid originating from respective transgenic plant events comprised in the sample. Preferably, this target and/or template nucleic acid would be genomic DNA originating from the said transgenic events which, due to its greater stability, is expected to be better preserved in the sample than other nucleic acid types, such as, e.g., hnRNA or mRNA. Nevertheless, the invention also contemplates the detection of mRNA or cDNA (e.g., -by corresponding RT-PCT assays), or plasmid-derived DNA, where applicable<br>
The present method thus preferably employs amplification using primers pairs designed to specifically hybridise within and amplify corresponding amplicons from within nucleic acids the presence or absence of which in the sample need to be detected, such as, preferably, nucleic acids comprising or consisting of those listed under a) - u) above, as a means to detect the presence or absence of the said nucleic acids in the sample.<br>
In this respect, the present inventors realised that if material of a transgenic plant event has been exposed to one or more downstream processing steps, e.g., as described above, the nucleic acids, such as the genomic DNA thereof, may be fragmented. Therefore, to increase the chances of the respective amplicons to be representatively amplified from so-fragmented DNA, the invention also teaches to advantageously reduce the size of the amplicons.<br>
 <br>
Accordingly, in preferred embodiments, the ampiicons are less than about 300 bp. e.g., less than 280 bp, less than 260 bp. (ess than 240 bp, or less than 220 bp. more preferably less than about 200 bp, e.g., less than 190 bp, less than 180 bp, less than 170 bp or less than 160 bp, even more preferably less than about 150 bp, e.g.. less than 140 bp, less than 130 bp, less than 120 bp or less than 110 bp. yet more preferably less than about 100 bp. and still more preferably less than about 90 bp, e.g. about 85 bp. about 80 bp, about 75 bp, about 70 bp. about 65 bp, about 60 bp, about 55 bp or about 50 bp.<br>
In further preferred embodiments the ampiicons are between about 50 bp and about 150 bp, between about 50 bp and about 100 bp. preferably between about 50 bp and about 90 bp, more preferably between about 60 bp and about 80 bp, and yet more preferably between about 65 bp and about 75 bp.<br>
As explained, the above methods detect particular nucleic acids, preferably genomic DNA, in a sample. It can be appreciated that some robust detection protocols, e.g., some PCR protocols, might be performed on samples without prior purification or enrichment of their constituent nucleic acids. Dn the other hand, the detections of the invention may be performed more consistently if nucleic acids, such as preferably genomic DNA. are first enriched or isolated from the sample. Methods for isolation of nucleic acids, such as genomic DNA, from samples, particularly samples comprising plant material, are long-established in the art and include methods based on, by example and not limitation, organic solvent extraction (e.g., phenol-chloroform extraction) and ethanol or isopropyl alcohol precipitation; binding to ion-exchange resins; caesium chloride density separation; spin-column chromatography, magnetic separation, etc (see, e.g., Milligan 1992. Plant DNA isolation, pp 59-88 in Hoelzel, ed. Molecular genetic analysis of populations: a practical approach. IRL Press, Oxford, UK).<br>
Quality of so-isolated DNA can be assessed, e.g., by gel electrophoresis and/or by measuring the A260/A280 absorbance ratio. Preferably, the A26O/A2BO ratio for nucleic acids used in the detection methods of the invention can be between 1.6 and 2.3, more preferably between 1.6 and 2.0, even more preferably between 1.7 and 1.9. yet more preferably between 1.75 and 1.85 and most preferably about 1.8. Quantity of so-isolated DNA may be assessed, e.g., by measuring absorbance A260 (Sambrook 1989) or preferably using PicoGreen or SYBR Green I fluorochromes (Ahn et al. 1996. Nucleic Acids Res 24: 2623-5: Schneeberger et al. 1995. PCR Methods Appl 4: 234-8).<br>
 <br>
In a preferred embodiment, preparation of nucleic acid material, particularly genomic DNA, for use in the methods of the invention may also entail fragmentation to obtain lower average sizes of the nucleic acid fragments, such as, e.g., using enzymatic digestion or sonication as known in the art Preferable average length of the so-fragmented nucleic acids may be between 200 and 2000 bp, more preferably between 300 and 1500 bp, even more preferably between 500 and 1000 bp, e.g., about 500, about 600. about 700, about 800 about 900 or about 1000 bp.<br>
The above methods thus detect the presence or absence of select nucleic acids in a sample, preferably using detection systems and protocols as described heretofore As used herein, the "presence" of a given nucleic acid in a sample is generally concluded when the signal intensity value (signal quantity) obtained for that nucleic acid using a suitable detection assay (e.g., fluorescent signal, chemiluminiscent signal, enzymatic reaction signal, etc.) in the sample is greater, preferably statistically significantly greater, than that in a negative control.<br>
A negative control does not contain the nucleic acid being assayed, but can advantageously be comparable to the sample in other respects, preferably most other respects, e.g., in the type and quantity of plant material contained therein, in downstream processing steps to which the material has been exposed, etc.<br>
Preferably, the detection methods of the invention may be performed on nucleic acids, more preferably on genomic DNA, isolated from the sample (see above). In such instance, the negative control may consist essentially of isolated nucleic acids, not comprising the nucleic acid subject to detection. Preferably, the detection method would then employ the same or about the same quantity of isolated nucleic acids from both the sample and, for reference, the negative control. Also preferably, the purity and/or quality of nucleic acids isolated from the sample and from the negative control may also be the same or about the same; further preferably, the same isolation method may be used for isolating nucleic acids from both the sample and the negative control.<br>
In the above situations, the presence of a given nucleic acid in a sample may be preferably concluded when the signal intensity value (signal quantity) obtained for that nucleic acid in the sample is at least 2 x greater than that in the negative control, e.g., at least 3 x or at least 4 x greater, more preferably at least 5 x greater, e.g., at least 6 x greater, at least 7 x greater, at least 8 x greater or at least 9 y greater, even more preferably at least 10 x greater, e.g.. at<br>
 <br>
least 15 x greater, yet more preferably at least 20 x greater, e.g., at least 30 x greater or at least 40 x greater, still more preferably at least 50 x greater, e.g.. at least 100 x greater, at least 200 x greater, at least 300 x greater, at least 400 x greater, at least 500 x greater, at least 1000 x greater or even greater than the signal in the negative control.<br>
Hence, when the signal value (signal quantity) obtained for the assayed nucleic acid in the sample is not greater (e.g.. is same, about same or lower) than the signal value obtained for the assayed nucleic acid in the negative control, or preferably is not greater by the above recited preferred multiples, the "absence" of the said nucleic acid in the sample can be generally concluded.<br>
In preferred embodiments, the presence or absence in the sample of a nucleic acid of interest is evaluated by real-time amplification, preferably real-time PCR. In these systems; a common measure used to express the quantity of the template for a given nucleic acid in a sample is the Ct value. Briefly, the C
Accordingly, in real-time amplification, the presence of a given nucleic acid in a sample may be generally concluded when C( of the sample ("SAM C") is lower than Cf of a negative control ("NIC C/'); and may be preferably concluded when <br>
(Formula Removed)<br>
Hence, in real-time ampiification, when SAM C, is not lower (e.g., is same, about same or greater) than NC C,, or preferably is not lower by the preferred differentials recited above, the "absence" of the said nucleic acid in the sample can be generally concluded.<br>
 <br>
As noted above, in real-time amplification systems which do not employ sequence specific probes (e.g., when amplification product is detected using sequence non-specific DNA dyes such as, e.g., SYBR Green I), the specificity of the amplification product can be verified by performing a melting curve analysis and determining the Tm. Presence of specific amplification product can be concluded when the observed Tm is identical or substantially identical (preferably within an interval of  3C, more preferably! 2C, yet more preferably r "PC. even more preferably  0.5C, and still more preferably  0.2C or  0.1C) to the calculated and/or experimentally determined melting temperature of the expected amplicon.<br>
Moreover, specific amplification can be preferably concluded when the melting curve analysis of the amplification product displays a single Tm. Alternatively or in addition, specific amplification can be preferably concluded when the amplification product displays a single, expected size upon gel electrophoresis. However, the appearance of non-specific amplification product(s) might to some extent be tolerable in practice, preferably if such nonspecific amplification product(s) would constitute less than 50%, more preferably less than 40%. even more preferably less than 30%, still more preferably less than 20%. yet more preferably less than 10%, further more preferably less than 5%, even more preferably less than 2% and most preferably less than 1%. e.g., less than 0.5% or less than 0.1%, of the total quantity (w/w) of the amplification product. Such non-specific amplification product(s) may be observed, e.g., by appearance of one or more additional Tm peaks in the melting curve analysis and/or appearance of one or more additional product sizes upon gel electrophoresis.<br>
in a further development of the invention, it is noted that amplicons amplified using the same primer set on nucleic acids originating from different evens may show distinct Tm values, presumably due to some differences in the sequence between such amplicons. By means of example and not limitation, amplicons amplified using primer set SEQ ID NO: 11 and 12 (Table 4) on Bt11 and Bt176 show such a difference in their Tm. Accordingly, the observed Tm value may serve as a further distinguishing information to use in the method of the invention, such that an even more detailed indication of the presence or absence of nucleic acids from such particular events is obtained.<br>
Alternatively or in addition, in a further variation, a particular nucleic acid may be amplifiable using a particular primer set from one or more events, while the same primer would not amplify the corresponding nucleic acid in one or more other events, presumably due to some sequence differences in the primer binding sites While such situation would necessitate the<br>
 <br>
use of two or more primer sets to amplify the corresponding nucleic acid from different events (and may in that respect be less advantageous than choosing a primer set that amplifies the nucleic acid from all plant events of interest), it may give the advantage of providing additional information about the presence or absence of nucleic acids from particular events in the sample. For example, the CrylAb nucleic acid may be advantageously detected from MON810 using the primer set SEQ ID NO. 30 and 31 and from Bt11 and Bt176 using the primer set SEQ ID NO: 11 and 12 (Table 4).<br>
A skilled person can appreciate that amplification reactions, e.g., PCR, may give rise to a non-specific, "primer-dimer" by-product. Such primer-dimer products can be typically readily distinguished from the specific amplification product(s) by their different, most often smaller, size and/or Tm. Hence, a skilled person would be able to recognise and minimise the effect of such artefacts on the analysis of the amplification reaction. The preferred primers of the invention, such as those listed in Tables 1 and 4, have been designed to minimise the effect of primer dimer.<br>
A further issue is the sensitivity of the detection methods used in the present invention, which may be advantageously expressed as "limit of detection" (LOD) thereof, referring herein to the lowest amount or concentration of an analyte (herein of a given nucleic acid to be detected) in a sample which can be reliably detected, although not necessarily quantified. "Reliably detected" as used herein means that samples containing the LOD amount or concentration of the analyte will give a positive outcome in &gt; 50% of repeated detections, preferably in &gt; 60%, more preferably in &gt; 70%, even more preferably in &gt; 80%, yet more preferably in &gt; 90%. and most preferably in &gt; 95% or even in &gt; 99% of repeated detections.<br>
Preferably, LOD of detection methods suitable for use in the invention, expressed herein as the LOD copy number of a given nucleic acid of interest in a sample subjected to the detection, may be 10 000 or less, e.g., 9 000 or less, 8 000 or less, 7 000 or less or 6 000 or less, more preferably 5 000 or less, e.g., 4 000 or less, 3 000 or less or 2 000 or less, even more preferably 1 000 or less. e.g.. 900 or less. 800 or less, 700 or less or 600 or less, yet more preferably 500 or less, e.g., 400 or less, 300 or less or 200 or less, still more preferably 100 or less. e.g.. 90 or less, 80 or less, 70 or less or 60 or less, and most preferably 50 or less, with exemplary preferred embodiments including 40 or less. 30 or less, 20 or less or even 10 or less, such as, e.g., 1, 2, 3. 4, 5, 6. 7, 8 or 9. The preferred primers of the invention.<br>
 <br>
such as those listed in Tables 1 and 4. have been designed to achieve particularly high sensitivity (low LOD)<br>
As used herein, the term "maize" refers to Zea mays, preferably Zea mays ssp mays, and includes all plant varieties that can be bred with maize, including wild maize species; the terms "rapeseed", "oilseed rape" or "canola" refer interchangeably to Brassica napus, and include all plant varieties that can be bred with rapeseed. including wild rapeseed species; the terms "soya", "soy", "soya bean" or "soybean" refer interchangeably to Glycine max.. and include all plant varieties that can be bred with soybean including wild soybean species; the term "rice" refers to Oryza sativa, including, without limitation, ssp. indica and japonica, and includes all plant varieties that can be bred with rice, including wild rice species: the term "sugar beet" refers to Beta vulgaris, and include all plant varieties that can be bred with sugar beet, including wild sugar beet species; the term "cotton" refers to Gossypium species, preferably Gossypium hirsutum. and includes all plant varieties that can be bred with cotton, including wild cotton species; the term "potato" refers to Solanum tuberosum, and includes all plant varieties that can be bred with potato, including wild potato species.<br>
Transgenic plant events are referred to herein by their designations as commonly employed in the art and familiar to a skilled person. Nevertheless, by means of further assistance. Table 2 below summarises further information sufficient for specific identification of recited events.<br>
Table 2. Identification of transgenic plant events.<br>
<br>
(Table Removed)<br>
The above listed unique identifiers (Ul) represent a system for unique classification of transgenic plant events set in place by the Organisation for Economic Co-operation and<br>
 <br>
Development (OECD) and described in its publications ENV/JM/MONO(2002)7: "OECD Guidance for the Designation of a Unique Identifier for Transgenic Plants", of October 20, 2004. and ENV/JM/MONO(2002)7/REV1 of November 7, 2006. The said unique identifiers are used, recognised and assigned internationally, including in Europe (see, e.g., Commission Regulation (EC) No 65/2004 of January 14. 2004 establishing a system for the development and assignment of unique identifiers for genetically modified organisms).<br>
Detailed information regarding the above and further transgenic plant events of interest, such as, e.g.. information on hosttaxons, transforming constructs, inserted sequences, etc. can be accessed through publicly available portals of various regulator)' authorities including, e.g., the OECD BioTrack Product Database (http://yAAww2.oecd.org/biotech/default.aspx), the US FDA (http://www.cfsan.fda.gov/). the European Community Register of GM Food and Feed (http://ec.europa.eu/food/dyna/gm_register/index_en.cfm), the Agbios database (www-.agbios.com). or the EC GMO Compass database (http://www.gmo-compass.org/), as well as in scientific literature, etc. A skilled person is well-aware and can use such database sources.<br>
In addition, event-specific detection methods and construct-specific detection methods (e.g., see Codex Alimentarium) are available and allow to unambiguously distinguish the above and further events. For example, the European Community Reference Laboratory (CRL) (http://gmo-crl.jrc.it/; http://gmo-crl.jrc.it/statusofdoss.htm) lists validated methods for event-specific detection, as provided by manufacturers, described in scientific literature or developed and by the CRL itself.<br>
As noted above, the invention allows to conclude the presence or absence in a sample of material derived from select events, crosses thereof, or related events thereof.<br>
The term "crosses thereof in this context refers to progeny obtained by one or more subsequent crossings of two or more compatible events (i.e., events capable of being crossed, particularly events belonging to the same taxon), such that the said resulting progeny comprises transgenic inserts from the two or more parental events being crossed.<br>
The term "related events thereof in this context refers to events that have been generated by introducing into the same taxons the same transforming constructs as that of the respective recited events, but are nevertheless distinct from the recited events, e.g., may typically differ in the number and/or site of genomic integrations of the transforming construct. Often, such<br>
 <br>
events may themselves be subject to authorisation. In other instances, such related events may be present as unwanted contaminants Hence, by detecting genetic elements from within the transforming constructs, the method of the invention advantageously allows to detect such events.<br>
As noted above, the methods of the invention involve detection of the presence or absence in samples of one or more nucleic acids comprising or consisting of those listed under a) - u) as defined above.<br>
In this respect, the recitation "a nucleic acid derived from and specific for" a given taxon. such as defined for nucleic acids listed under a) "Zm". b) "Bn", c) "Gm", o) "Or", p) "Bv", q) "Gs", and t) "St", means that the said nucleic acid in the sample originates from the genetic material, preferably genomic DNA. of the said given taxon and is not present - and thus would not be detectable using the chosen detection method - in the genetic material originating from other taxons, such as the remaining taxons listed above. By means of example and not limitation, said nucleic acid derived from the given taxon may either have no homoiogues in other taxons or its homologues in other taxons may have sufficiently distinct sequences such that they would not be detected by the chosen detection method. For example, at a site of detection, e.g., at a site to which of a probe or one or more amplification primers would hybridise within the given nucleic acid, such homologues may show less than 100% sequence identity, preferably less than 95%, even more preferably less than 90%, yet more preferably less than 85%, still more preferably less than 80%. and most preferably less than 75%, in preferred exemplary embodiments less than 70%, less than 65%, less than 60%, less than 55% or less than 50% sequence identity with the given nucleic acid.<br>
Further, the term nucleic acid "derived from" as used especially in a) - u) means that the said nucleic acids originate from their respective taxons, genetic elements, genes, open reading frame, etc., albeit they - as found in the sample - may in part structurally differ therefrom.<br>
For instance, downstream processing of plant material may cause alterations in nucleic acid (e.g., DNA) structure, and perhaps most notably may cause fragmentation thereof. In addition, sometimes functional fragments or variants of full-length elements may be used in transgenic plants. Accordingly, nucleic acids "derived from" those recited under a) - u) above may be, e.g., fragments thereof, insofar such fragments allow for their specific attribution to the original nucleic acids. For example, such fragments may include at least 15 continuous bp of the sequence from which they are derived, preferably at least 20 continuous bp, e.g., at<br>
 <br>
least 30 or at least 40 continuous bp, more preferably at least 50 continuous bp, e.g., at least 60 or at least 70 continuous bp, even more preferably at least 80 continuous bp. e.g., at least 90 continuous bp, yet more preferably at least 100 continuous bp, e.g., at least 150 continuous bp, still more preferably at least 200 continuous bp, e.g.. at least 300 continuous bp, at least 400 continuous bp or at least 500 continuous bp or more and may even include full-length elements.<br>
Other nucleic acids modifications expected from downstream processing of plant material, e.g., partial deamination, etc., are also contemplated under the term "derived from".<br>
In a preferred embodiment, the nucleic acid a) "Zm" is derived from the endogenous alcohol dehydrogenase I gene (adh-1) of Zea mays. An exemplary but non-limiting sequence of the adh-1 gene of Zea mays is found under accession number AF123535.1 in the GenBank database (http://www.ncbi.nlm.nih.gov/).<br>
In a further preferred embodiment, the nucleic acid b) "Bn" is derived from the endogenous acetyl-CoA carboxylase gene (ace) or the endogenous cruciferin gene of Brassica napus. An exemplary but non-limiting sequence of the ace gene of Brassica napus is found under accession number X77576.1 in the GenBank database; an exemplary sequence of the cruciferin gene of Brassica napus is found under accession number X14555.1 in the GenBank database.<br>
In a further preferred embodiment, the nucleic acid c) "Gm" is derived from the endogenous lectin gene (lee) of Glycine max. An exemplary but non-limiting sequence of the lee gene of Glycine max is found under accession number K00821.1 in the GenBank database.<br>
In a further preferred embodiment, the nucleic acid o) "Or" is derived from the endogenous phospholipase D gene (pld) of Oryza sativa. An exemplary but non-limiting sequence of the pld gene of Oryza sativa is found under accession number AB001919.<br>
In a preferred embodiment, the nucleic acid p) "Bv" is derived from the endogenous Glutamine synthetase gene (GluA3) of Beta vulgaris. An exemplary but non-limiting sequence of the GluA3 gene of Beta vulgaris is found under accession number AY026353.1 in the GenBank database.<br>
In a preferred embodiment, the nucleic acid q) "Gs" is derived from the endogenous sinopsis arabidopsis homolog 7 gene (sah-7) of Gossypium An exemplary but non-limiting sequence<br>
 <br>
of the sah-7 gene of Gossypium is found under accession number AY117067 (Authors: Senchina et al. 2003 Mol Biol Evol 20 (4) 633-643) in the GenBank database.<br>
In a preferred embodiment, the nucleic acid t) "St" is derived from the endogenous UDP-glucose pyrophosphorylase gene (UGPase) of Solarium tuberosum. An exemplary but non-iimiting sequence of the UGPase gene of Solarium tuberosum is found under accession number U20345 in the GenBank database.<br>
In a further preferred embodiment, the generic plant-derived nucleic acid is derived from the endogenous chloroplastic RBCL gene (rbcl) of Zea mays. An exemplary but non-limiting sequence of the rbcl gene of Zea mays is found under accession number Z11973.1 in the GenBank database<br>
A skilled person would be capable of designing and verifying specific probes"or primer pairs from the above cited sequences for use in the detection steps of the present method.<br>
The references to sequence elements and genes recited in this specification for the nucleic acids listed under d) - n), r), s) and u) are known to a skilled person in the field of genetic modification of plants. Hence, a skilled person understands to what sequences these designations refer and would as well appreciate the level of alterations to these sequences that is common in generation of transgenic plants.<br>
In an embodiment, the nucleic acid present in a transgenic event may include the open reading frame or a functional part thereof (i.e., a part achieving the desired effect in the transgenic plant) of the genes recited herein, esp. the genes listed under f) - n), r), s) and u) above.<br>
Nevertheless, by means of further clarification and not limitation, we here under list exemplary sequences for the particular genes and elements referred to under d) - n), r), s) and u), and an exemplary generic plant-derived nucleic acid:<br>
Sequence element / gene	GenBank accession number<br>
CMV p35S promoter	V00140 (entire CAMV genome)<br>
NOS terminator	V00087.1<br>
CrylAb ORF	AF465640<br>
bar gene ORF	AY346130.1<br>
pat gene ORF	DQ156557.1<br>
 <br>
EPSPS gene ORF	AY 125353 1<br>
modified Cry3A gene ORF	AR836206.1<br>
Glb1 promoter	CS 155614.1<br>
Cry3Bb1 gene ORF	CS410008.1<br>
Bxn gene ORF	E01313.1<br>
Cry 1 Ac gene ORF	EF094884.1<br>
Cry2Ab2 gene ORF	DQ361266.1<br>
Gbss gene ORF	X58453.1<br>
Zea mays chloroplast rbcL gene	Z11973.1<br>
It shall be appreciated that nucleic acids actually present in transgenic plant events can comprise sequences which may differ to some extent from the above exemplary sequences. For example, such differences may include base deletions, additions and/or substitutions, truncations (e.g., inclusion of functional fragments), fusions to other elements (e.g., fusion to membrane transport or organelle sorting signals), etc.<br>
The actual sequences of the above and other elements found in the transgenic events have been in many instances determined and are accessible via GenBank and other sequence databases. Accordingly, sequence alignments shall be performed in order to identify regions within the above nucleic acids most suitable for derivation of probes or ampiicons therefrom.<br>
Hence, a probe or an amplicon (and corresponding primer set) can be advantageously derived from a portions of each of the above nucleic acids which is found in most transgenic events and/or which shows the highest sequence identity between the various transgenic events. Such probe or primer pair will increase the chance of detection of that particular nucleic acid from various transgenic events. A skilled person can follow this instruction to perform the necessary sequence alignments.<br>
As set out in the Summary section, the invention also provides an advantageous manner of concluding the potential presence or absence of material from one or more transgenic plant events of interest to be used in step (2) of the above methods. In particular, the results obtained for the sample in step (1) are represented as a set GSAM symbolising a collection of those nucleic acids, chosen from those comprising or consisting of the ones listed under a) -u) above, that were detected as present in the sample; and comparing the said set GSAM with one or more sets of interest Gx representing the individual events to be evaluated. The sets of interest Gx consist of, i.e., represent a collection of. those nucleic acids chosen from ones<br>
 <br>
comprising or consisting of those listed under a) - u) above that are found in the respective events and thus would be detected in step (1) (in the extent to which said step (1) detects the presence of said nucleic acids) if the sample contained material derived from the respective events, or a cross involving such, or a related event.<br>
The following Table 3 lists which nucleic acids of those listed under a) - u) above are present in particular events of interest:<br>
Table 3. Features of events.<br>
(Table Removed)<br>
 As already explained:<br>
-	if G&gt;; equals GSAM (GX = GSAM), then material derived from the transgenic plant event X, or from a cross thereof, or from an event related thereto, is potentially present in the sample;<br>
-	if Gx is a proper subset of GSAM (G.  GSAM). then material derived from transgenic plant event X, or from a cross thereof, or from an event related thereto, is potentially present in the sample;<br>
-	if Gx does not equal GSAM and Gx is not a proper subset of GSM, (Gx # GSAM and Gx GSAM), then material derived from transgenic plant event X, or from a cross thereof, or from an event related thereto, is absent from the sample.<br>
In a further preferred embodiment, if Gx equals GSAM, and if no one set or the sum of two or more sets representing events (in particular, sets representing events chosen from a group comprising or consisting of<br>
(Formula Removed)<br>
and GEH92~527-I) other than G&gt;.- equals GA-, then material derived from transgenic plant event X or from a cross thereof, or from an event related thereto, is present in the sample.<br>
In a further development of this manner of evaluation, nucleic acids comprising or consisting of those listed under a) - u) above are assigned respective unique values. Hence, the set GSAM is then assigned a value "VGSAM" being a multiple of the unique values assigned to those nucleic acids detected in step (1) as being present in the sample; and a set of interest G&gt;- is assigned a value "VGx" being a multiple of the unique values assigned to those nucleic acids chosen from ones comprising or consisting of those listed under a) - u) above that are found in the said event and thus would be detected in step (1).<br>
Accordingly, in a preferred example, nucleic acids "Zm". "Bn", "Gm", "p35S". "tNOS", "CrylAb". "PAT/bar", "PAT/pat", "CP4-EPSPS", "mCry3A", "cordap A", "Glb1", "Cry3Bb1", "Bxn". "Or", "Bv", "Gs", "CrylAc", "Cry2ab2". "St" and "Gbss" would be assigned respective<br>
(Formula Removed)<br>
and each set of interest Gx would be assigned a respective value "VGx" chosen from values of the group comprising or consisting of values "VGafW- "VGBI", "VGA^C", "VGMONSIO'<br>
(Formula Removed)<br>
Note that insofar the methods do not test the presence of all nucleic acids under a) - u), the above values shall be defined in terms of those nucleic acids the presence of which is effectively tested. By means of example and not limitation, where the presence of nucleic acids listed under j) - n), r) s) and u) as defined above, which correspond to specific traits in addition to those defined under d) - i), is not tested, then the values corresponding to several events may be defined in a narrower manner, in particular, maize event values VGMIRSO* = VZm<br>
(Formula Removed)<br>
VtNosl and potato event value <br>
Consequently, step (iii) of this embodiment would comprise performing for each set of interest G&gt;; logical operations:<br>
-	if VGSAM / VGx equals 1, then material derived from transgenic plant event X or from a cross thereof, or from an event related thereto, is potentially present in the sample;<br>
-	if VGSAM I VGX equals one value or a multiple of two or more values chosen from a group comprising or consisting of values VZm, VBr VGm. Vp35s, Vtw0s- VCo/Mc, VpAmar, VPAVpat. VCP4_<br>
EPSPS,  VmCry3A,   Vcordap A,  VG/b1,  VcrySBbl-  VBxr  Vor, ^Bv,  VGs.  VcrylAc,  ^'c/y2aD2.  Vgt and  VGte5s,  then<br>
material derived from transgenic plant event X or from a cross thereof, or from an event related thereto, is potentially present in the sample;<br>
-	if VGSAM 1 VGx does not equal 1 and does not equal one value or a multiple of two or more<br>
values chosen from a group comprising or consisting of values <br>
(Formula Removed)<br>
2, Vs, and VGtes, then material derived from transgenic plant event X or from a cross thereof, or from an event related thereto, is absent from the sample.<br>
In a further preferred variant of the above embodiment, if VGSAM / VGx equals 1. and if no one value or the multiple of two or more values of sets representing events (in particular, values chosen from a group comprising or consisting of: <br>
(FORMULA REMOVED)<br>
 other than VGx equals VGx. then material derived from transgenic plant event X or from a cross thereof, or from an event related thereto, is present in the sample.<br>
In a further preferred embodiment, the unique values assigned to the nucleic acids comprising or consisting of those listed under a) - u). i.e., values chosen from the group comprising or consisting of <br>
(Formula Removed)<br>
<br>
prime numbers, and the step (iii) comprises performing for each set of interest Gx logical operations,<br>
-	if VGSAM / VGx equals 1, then material derived from transgenic plant event X or from a cross thereof, or from an event related thereto, is potentially present in the sample;<br>
-	if VGSAM IVGx is an integer greater than 1, then material derived from transgenic plant event X or from a cross thereof, or from an event related thereto, is potentially present in the sample;<br>
-	if VGSAW / VGX is not an integer, then material derived from transgenic plant event X or from a cross thereof, or from an event related thereto, is absent from the sample.<br>
The inventors realised that the use of prime numbers to represent the individual assayed traits (nucleic acids), as described in the foregoing paragraph, is particularly straightforward and advantageously streamlines and facilitates the analysis of the data.<br>
In view thereof, the inventors also contemplate the general applicability of the herein explained data analysis method using prime numbers in other applications, such as, e.g., in the field of molecular biology or molecular medicine, etc. Hence, in one application, the method may be generally used for determining the composition of a sample, wherein said sample may contain two or more different events (the word event as used in this paragraph has a general meaning not limited to transgenic plant events) or material thereof, wherein each event is characterised by one or more features or traits (the words features and traits as used in this paragraph carry a general meaning not limited to. albeit preferably including, nucleic acids) Hence, the sample as well as each event can be represented as a product (i.e., operation "x") of unique prime values assigned to each of the tested features or traits, and the (potential) presence or the absence of said event or material thereof in said sample may be tested by dividing (i.e.. operation "/") the value obtained for the sample by the value<br>
calculated and assigned to each individual event, and considering the result of said division as described above mutatis mutandis.<br>
In a preferred embodiment, the above calculation steps of the method step (2) are carried out by a computing device, e.g., a calculator or a computer; and in further aspects the invention also contemplates a computing device performing the above calculations, a data carrier (e.g., a diskette, CD-ROM, etc.) comprising instructions for a programmable computing device to carry out the step (2) of the method of the invention, including the above calculations, a kit comprising such data carrier alongside one or more reagents also useful in the methods of the invention. Also, the invention contemplates a web-based application allowing for performing the algorithm of step (2), optionally wherein information about events potentially present in the sample can be directly accessed in an associated database by means of an embedded hyperlink.<br>
It shall be appreciated that the identifiers "GSAW, "Gx", "VGSAW. "VGX", etc. used in the above description were arbitrarily chosen herein to represent the underlying concepts of inter alia sets and values, respectively, and that other identifiers (e.g., other letters, words or expressions) may be substituted to represent said sets and values.<br>
As mentioned, the method of the invention may preferably evaluate the presence or absence of nucleic acids of interest, especially nucleic acids chosen from the group comprising or consisting of those listed under a) - u) above, using amplification and particularly PCR amplification of amplicons therefrom.<br>
It can be appreciated that a skilled person may be in general capable of designing individual primer sets for specific amplification from known nucleic acid sequences.<br>
Nevertheless, as already described. Table 1 iists particularly advantageous primer pairs that have been carefully designed by the inventors to provide numerous benefits when used in the present methodology. Moreover, Table 4 lists the primers of Table 1 and further includes additional primers and primer pairs which also perform very satisfactorily in the present method, albeit the primers of Table 1 remaining the prime choice of the inventors. These primer sets can advantageously achieve amplification from the respective nucleic acids as present in the relevant plant events and from variously processed plant material, and entail various advantages as described elsewhere in this specification.<br>
Table 4 Primer sequences.<br>
<br>
<br>
(Table Removed)<br>
Where Table 4 includes more than one rows which concern amplification on a certain nucleic acid (such as. e.g., several rows for "CrylAb" and "CP4-EPSPS) by different primer sets, any one or more of said primer sets of said different rows of Table 4 can be used for amplification. Where Table 4 lists, within a single row, more than one forward and/or reverse primers for amplification on a certain nucleic acid (such as. e.g., for "Bn" (ACC), "Gm" (lectin) or "CP4-EPSPS"). any one or any combination of said forward primer(s) may be used in conjunction with any one or with any combination of said reverse primer(s) to obtain amplification.<br>
In addition, the invention also provides variant primers that include one or more sequence variations vis-a-vis the primers listed in Table 4, such as. e.g.. one or more deletion, insertion and/or substitution, insofar such primers / primer pairs can still achieve adequate amplification of their respective amplicons. Preferably, such variant primers would show at least 85%, more preferably at least 90%, even more preferably at least 95%, and yet more preferably at least 96%, at least 97%, at least 98% or at least 99% sequence identity to the primers listed in Table 4. Sequence alignments and determination of sequence identity can be done, e.g., using the Basic Local Alignment Search Tool (BLAST) originally described by Altschul et al. 1990 (J Mol Biol 215: 403-10), such as the "Blast 2 sequences" algorithm described by Tatusova and Madden 1999 (FEMS Microbiol Lett 174: 247-250).<br>
In addition, the invention also contemplates nested primers and primer pairs suitable for amplification of amplicons from within those amplified from the respective nucleic acids using the primer pairs listed in any of Table 1 or Table 4.<br>
The invention also contemplates derivatives, as described elsewhere in this specification, of the primers and primer pairs listed in any of Table 1 or Table 4.<br>
Accordingly, the invention provides the primers and primer pairs listed in any of Table 1 or Table 4, as well as variants and derivatives thereof, optionally in any suitable combination of said primers and/or primer pairs or variants or derivatives thereof.<br>
In a further aspect, the invention provides amplification products obtainable from the respective nucleic acids using the above primers and primer pairs, and particularly using primer pairs set out in any of Table 1 or Table 4 or variants or derivatives thereof. The said amplification products may be further processed, such as, e.g., isolated, cloned in suitable vectors or plasmids, transformed into recombinant hosts, e.g., suitable bacterial hosts such as E. coli, propagates therewith and isolated therefrom, sequenced, etc.<br>
Advantageously, the said amplification products, preferably cloned and re-isolated, and possibly comprised in suitable plasmids or vectors, can be used as positive controls to verify the working of detection steps of the methods of the invention aimed at the detection of the respective nucleic acid in a sample. Alternatively or in addition, such amplification products (cloned or isolated) may be used as calibrators to determine the quantity of particular templates in a sample, e.g., in real-time PCR.<br>
Accordingly, in a further aspect, the invention provides recombinant E. coli as deposited under the Budapest Treaty with the Belgian Coordinated Collections of Microorganisms (BCCM) on January 10. 2007 under LMBP accession numbers LMBP 5452, LMBP 5453, LMBP 5454, LMBP 5455, LMBP 5456. LMBP 5457, LMBP 5458. LMBP 5459 and LMBP 5460, on March 6, 2007 under LMBP accession number LMBP 5451. and on April 19, 2007 under LMBP accession numbers LMBP 5587, LMBP 5588, LMBP 5589 and LMBP 5590.<br>
The said recombinant E. coli comprise the amplification products obtained on template events using primer sets as listed in Table 5, cloned in the EcoRI sites of the pUC18 cloning vector MCS:<br>
Table 5<br>
<br>
<br>
(Table Removed)<br>
In addition to deposited plasmids and E. coli as listed in Table 5, the invention also provides<br>
further exemplary plasmids, such as. e.g., pUC18, containing inserts amplified using, e.g.:<br>
-	the primer pair SEQ ID NO: 3 and 4 for the Brassica ace gene (exemplary insert sequence SEQ ID NO: 44 in Fig. 1 amplified on OSR wt);<br>
-	CP4-EPSP primer pairs SEQ ID NO: 18 and 19, and SEQ ID NO: 50 and 19 (exemplary inserts amplified on, respectively, event 40-3-2 and event NK603, to obtain the respective exemplary insert sequences shown as SEQ ID NO: 45 and 46 in Fig. 1) (primer SEQ ID NO: 17 may also be used in conjunction with primer SEQ ID NO: 19; however, primer SEQ ID NO: 17 contains a sequence mismatch at nucleotide 20 compared to SEQ ID NO: 50 the latter being therefore more preferred);<br>
-	the primer pair SEQ ID NO: 1 and 2 for the maize adh-1 gene (exemplary insert sequence SEQ ID NO: 32 in Fig. 1 amplified on wild-type maize);<br>
-	the primer pairs SEQ ID NO: 5 and 49 for the soybean lectin gene (exemplary insert sequence SEQ ID NO: 33 in Fig. 1 amplified on wild-type soybean);<br>
-	the primer pair SEQ ID NO: 56 and 57 for a shorter amplicon within the Tea mays adh-1 gene (exemplary insert sequence SEQ ID NO: 70 in Fig. 1);<br>
-	the primer pair SEQ ID NO: 58 and 59'(SLTM) for a Glycine max lectin amplicon (exemplary insert sequence SEQ ID NO: 71 in Fig. 1);<br>
 <br>
-	the primer pair SEQ ID NO: 60 and 61 for a shorter amplicon within the p35S element (exemplary insert sequence SEQ ID NO: 72 in Fig. 1):<br>
-	the primer pair SEQ ID NO: 64 and 65 for an amplicon within the Beta vulgaris GluA3 gene (exemplary insert sequence SEQ ID NO' 76 in Fig. 1);<br>
-	the primer pair SEQ ID NO: 66 and 67 for an ampiicon within the Gossypium sah-7 gene (exemplary insert sequence SEQ ID NO: 77 in Fig. 1),<br>
-	the primer pair SEQ ID NO: 68 and 69 for an amplicon within the potato UGPase gene (exemplary insert sequence SEQ ID NO: 78 in Fig. 1);<br>
-	the primer pair SEQ ID NO: 11 and 12 for an amplicon within the CrylAb trait (exemplary insert sequence SEQ ID NO: 73 in Fig. 1 amplified on MON 810 material);<br>
-	the primer pair SEQ ID NO. 22 and 62 (RRS) or 22 and 63 (GT 73) for an amplicon within the CP4-EPSPS trait (exemplary insert sequence in Fig. 1: SEQ ID NO: 74 amplified on GT73 and SEQ ID NO: 75 amplified on RRS).<br>
It is noted that due to the shorter size of the corresponding amplicon, the primer pair SEQ ID NO: 22 and 23 or SEQ ID NO' 22 in conjunction with any one or both of primers SEQ ID NO: 62 and 63, may be preferably used to amplify CP4-EPSPS from both unprocessed plant material (e.g., plant tissue or seeds) as well as processed material (e.g., food or feed), whereas the primer pairs SEQ ID NO: 18 and 19 or 50 and 19, which define a longer ampiicon, can be preferably used on unprocessed material. Hence, the primer pair SEQ ID NO: 22 and 23 or SEQ ID NO. 22 in conjunction with any one or both of primers SEQ ID NO: 62 and 63, may be more preferred in the present methods and kits.<br>
The invention further provides isolated recombinant plasmids obtainable from the recombinant E. coli bacteria listed in Table 5. as well as isolated inserts, preferably EcoRl inserts, thereof. The invention also provides any further recombinant microorganism transformed with the so-isolated plasmids.<br>
A skilled person can also appreciate that the invention may too provide combinations of the said plasmids or inserts thereof, wherein the said combinations are representative of those nucleic acids that one intends to detect in a sample.<br>
In addition, the invention also provides the use of recombinant plasmids or inserts as found in and obtainable from the bacteria of Table 5 or others explained above for use as positive<br>
 <br>
controls and/or calibrators in the methods of the present invention. For example, such plasrnids or inserts thereof may be included in the methods and kits of the invention in suitable quantities to also facilitate making a decision whether nucleic acid of one or more events is present in an amount which justifies further quantification, or is present under an acceptable threshold.<br>
In a further development of the invention, it is also contemplated that any of the primers of the invention, such as in particular the primers listed in any of Table 1 or Table 4 or variants or derivatives thereof, may be used in a genome walking strategy to identify sequences adjacent to said primer in genomic DNA of an event. Similarly, it is contemplated that any primer designed on the basis of the sequence of any amplicon of the present invention, such as in particular any primer located within or overlapping with any of the amplicon sequences SEQ ID NO: 34 to SEQ ID NO: 78, or a variant or derivative of such primer, may be used in a genome walking strategy to identify sequences adjacent to said amplicon in genomic DNA of an event.<br>
This aspect may be particularly useful where the present amplification methods produce an amplicon having characteristics (e.g., length, Tm or sequence) which cannot be unambiguously attributed to a particular event. In such situation, genome walking outside of the amplicon in question would provide additional sequence information about the event and thereby help to identify said event. In particularly preferred embodiments, the specific primer may be derived from p35 or tNOS amplicons.<br>
in general, genome walking strategies are well-known in the art. Examples include inter alia inverse PCR or the vectorette method of Riley et al. 1990 (Nucleic Acids Res 18: 2887-90) commercialised as Universal Vectorette System (Sigma), as well as later simplifications thereof (see, e.g.. Kilstrup &amp; Kristiansen 2000. Nucleic Acids Res 28: e55).<br>
In a particular embodiment, the inventors contemplate a two-step genome walking method. In a first amplification step a LUX (or otherwise) labelled primer of the invention is used together with a mixture of random primers (generally about 8 to 15 bp long), and preferably gradient PCR conditions of increasing stringency are used. The label on the primer allows to follow whether amplification involving the labelled primer is taking place. In a second amplification step, a nested PCR is performed using another nested primer from the amplicon. preferably also LUX or otherwise labelled, together with said random primers, and preferably at high-stringency PCR conditions. The label on the primer allows to follow<br>
 <br>
whether amplification involving the labelled primer is taking place. Eventually, the so-obtained product is sequenced, thereby identifying the sequences adjacent to the amplicon<br>
In a further aspects, the invention provides kits of parts for performing the methods of the present invention, i.e., for examining a sample for the potential presence or absence of material derived from one or more transgenic plant events.<br>
In an embodiment, a kit according to the invention can comprise probes suitable for detection of one or more or all nucleic acids listed under a) - u) above.<br>
In an embodiment, a kit according to the invention can comprise primers, preferably primer pairs, suitable for amplification of amplicons from within one or more or all nucleic acids listed under a) - u) as defined above.<br>
In another embodiment, the kit may comprise primers, preferably primer pairs, suitable for amplification of one, more than one. and preferably all of nucleic acids listed under a) - i) above.<br>
In another embodiment, the kit may comprise primers, preferably primer pairs, suitable for amplification of one, more than one, and preferably all of nucleic acids listed under a) - i), o). p). q) and t) above<br>
In another embodiment, the kit may comprise primers, preferably primer pairs, suitable for amplification of one, more than one, or all of nucleic acids listed under a) - c) above and one, more than one or preferably all of nucleic acids listed under d) - i) above.<br>
In another embodiment, the kit may comprise primers, preferably primer pairs, suitable for amplification of one. more than one, or all of nucleic acids listed under a) - c): o). p), q), t) above and one, more than one or preferably all of nucleic acids listed under d) - i) above.<br>
in addition, the kit may also comprise primers or primer pairs for amplification of a generic plant sequence, as described elsewhere in this specification.<br>
It shall be appreciated that primers and primer pairs for amplification on various nucleic acids taught herein may be included in variously designed kits, in accordance with the preference as to which transgenic events one wants to detect.<br>
In a preferred embodiment, and taking into account the various selections of nucleic acids to be amplified as discussed in the preceding paragraphs, the kit may comprise one or more, or<br>
 <br>
all primers, preferably one, more than one or all primer pairs, as defined in any of Table 1 or Table 4, or variants or derivatives thereof.<br>
In a particularly preferred embodiment, one or both primers of any primers pair included in the kit may be labelled<br>
In another particularly preferred embodiment, any primer pair included in the kit may contain at least one, and usually one. primer labelled with a suitable fluoropnore and designed to allow for real-time detection using the above-described LUX detection system.<br>
In a further embodiment, the kit may also comprise amplification products, e.g., cloned and re-isolated and possibly comprised in suitable plasmids or vectors, that can be used as positive controls and/or calibrators in the methods of the invention, as explained above. For example, precisely" measured quantities or dilutions of such amplification products, e.g., cloned and re-isolated and possibly comprised in suitable plasmids or vectors, may be provided for such purposes. As explained above, various combinations of such reagents may also be envisaged.<br>
In a particular embodiment, the kit may comprise one. more than one or all of the E. coli organisms as listed in Table 5. and/or plasmids obtainable therefrom, and/or isolated inserts, preferably EcoRI inserts, of the said plasmids, or combinations hereof<br>
In a further embodiment, as already explained, the kit may also comprise a data carrier containing instructions for a programmable computing device to perform the actions of step (2) of the methods of the invention.<br>
Understandably, such kits may comprise further components, such as components useful hybridisation, PCR. detection, etc. A skilled person will appreciate the potential use of such components<br>
In a preferred embodiment, the kit may comprise one or more reaction compartments, for example reaction compartments provided within a multi-well strip or plate (multi-well format), each compartment comprising a composition of all components necessary for a PCR amplification reaction (e.g., nucleotides, salts, thermostable polymerase, etc.), and including the desired one or more (where multiplexed) primer pairs as defined above, but not including template DNA. Accordingly, the PCR reaction could be started once the user introduces a sample DNA to be analysed to said composition. Optionally, to ensure stability, Taq or other thermostable polymerase may also be excluded from the composition (but may be separately<br>
 <br>
included in the kit), and may need to be added by the user. Generally, the composition may be liquid, while especially for the purposes of shipment and storage it may be frozen. However, lyophiiised compositions are also envisaged.<br>
EXAMPLES<br>
Example 1: Amplification conditions for select primer pairs listed in Tables 1 and 4<br>
SYBR Green detection real-time PCR was optimised for amplification of amplicons from nucleic acids "p35S", "tNOS", "CrylAb", "PAT/Bar". "PAT/pat" and "CP4-EPSPS" using respective primer pairs listed in Tables 1 and 4.<br>
DNA was isolated from events containing (positive control) or lacking (negative control) these nucleic acids, including events Bt11, Bt176,. MON810, MON40-3-2, TC1507. NK603. MS8/RF3 (see Table 5), using CTAB isolation from leaf tissue material.<br>
PCR reactions contained SYBR Green PCR Master Mix (Diagenode. ref: GMO-GS2X-A300) 1x, Forward primer 250 nM. Reverse primer 250 nM, Template DNA 50 ng in total volume of 20 pi. PCR cycling involved Step 1: 1x [50C, 120 sec]: Step 2: 1x [95C 600 sec]: and Step 3: 40x [95C 15 sec, 60C 60 sec] with fluorescence acquisition. Applied Biosystems Prism 7700 was used. Melting curve analysis was done at gradient from 50C to 95C over 1200 sec.<br>
For all primer pairs, specific amplification products were obtained as shown by a single specific Tm and single band on agarose gel electrophoresis. The assays had low LOD:<br>
RBCL: primers SEQ ID NO: 26 and SEQ ID NO: 27; Tm = 76,5C; size = 95 bp; LOD = <br>
0.039 ng target DNA<br>
Zea mays ADH (longer amplicon): primers SEQ ID NO: 1 and SEQ ID NO: 2; Tm = 79,5DC;<br>
size = 138 bp; LOD =  0.016 ng target DNA ( 6 haploid genomes)<br>
Zea mays ADH (shorter amplicon): primers SEQ ID NO: 56 and SEQ ID NO: 57; Tm =<br>
75,5C: size = 83 bp; LOD =  0.016 ng target DNA ( 6 haploid genomes)<br>
Oilseed rape ACC: primers SEQ ID NO: 79 and SEQ ID NO: 3; Tm = 79C: size = 103 bp;<br>
LOD =  0.05 ng target DNA (20 haploid genomes)<br>
 <br>
Oilseed rape cruciferin: primers SEQ ID NO: 24 and SEQ ID NO: 25; Tm = 80C: size = 85<br>
bp; LOD =  0.015 ng target DNA ( 12 haploid genomes)<br>
Soybean lectin (longer amplicon): primers SEQ ID NO. 5 and SEQ ID NO: 49; Tm =<br>
81,5C; size= 178 bp; LOD = 0.016 ng target DNA ( 13 haploid genomes)<br>
Soybean (lectin): primers SEQ ID NO: 58 and SEQ ID NO: 59 (SLTM); Tm = 79,5C; size =<br>
81 bp; LOD = T 0.063 ng target DNA ( 50 haploid genomes)<br>
Rice PLD: primers SEQ ID NO: 20 and SEQ ID NO: 21; Tm = 76,5C: size = 80 bp: LOD = <br>
0.01 ng target DNA ( 20 haploid genomes)<br>
Sugar beet GluA3: primers SEQ ID NO' 64 and SEQ ID NO: 65; Tm = 77C; size = 118 bp;<br>
LOD = n 0.01 ng target DNA ( 13 haploid genomes)<br>
Cotton SAH7: primers SEQ ID NO: 66 and SEQ ID NO' 67: Tm = 75,5C: size = 115 bp;<br>
LOD =  0.02 ng target DNA (z 9 haploid genomes)<br>
Potato UGPase: primers SEQ ID NO. 68 and SEQ ID NO: 69; Tm = 81 C; size = 87 bp; LOD<br>
= 3 0.0002 ng target DNA<br>
p35S (longer amplicon): primers SEQ ID NO: 7 and SEQ ID NO: 8; Tm = 80,5C; size = 147<br>
bp: LOD =  0.016 ng target DNA ( 6 haploid genomes)<br>
p35S (shorter amplicon): primers SEQ ID NO: 60 and SEQ ID NO: 61; Tm = 76C; size = 75<br>
bp; LOD : RRS 100%  0.032 ng target DNA ( 25 haploid genomes)' NK603 5%  6.25 ng<br>
target DNA ( 62,5 haploid genomes)<br>
tNOS-L: primers SEQ ID NO: 28 and SEQ ID NO: 29; Tm = 72C; size = 172 bp: LOD = <br>
0.03 ng target DNA ( 25 haploid genomes)<br>
tNOS-D: primers SEQ ID NO: 9 and SEQ ID NO: 10; Tm = 72C; size = 69 bp; LOD =  0.03<br>
ng target DNA ( 25 haploid genomes)<br>
CrylAb: primers SEQ ID NO: 11 and SEQ ID NO: 12; Tm = 78.5C; size = 73 bp; LOD = <br>
0.06 ng (12 haploid genomes)<br>
PAT/Bar: primers SEQ ID NO: 13 and SEQ ID NO: 14; Tm = 80C; size = 69bp: LOD = <br>
0.125ng (50 haploid genomes)<br>
PAT/pat: primers SEQ ID NO' 15 and SEQ ID NO: 16; Tm = 77C; size = 109 bp; LOD = =<br>
0.03ng (=12 haploid genomes)<br>
 <br>
CP4-EPSP: primers SEQ ID NO: 22, SEQ ID NO: 62 and SEQ ID NO: 63; Tm = 80,5 or 84,5C; sizes = 108 bp; LOD : NK603 5% 3,125 ng (31 haploid genomes), RRS 100% 0,032 ng (25 haploid genomes), GT73 100% 0,016 ng (12 haploid genomes) CP4-EPSP: primers SEQ ID NO: 17, SEQ ID NO: 18 and SEQ ID NO: 19; Tm = 85C; sizes = 94 bp or 124 bp; LOD = 0.03 ng (25 haploid genomes)<br>
Example 2: Exemplary simplified method according to the invention<br>
A hypothetical sample was made by adding of Bt176 DNA to unrelated carrier DNA. The sample was screened for the presence or absence of "p35S", "tNOS", "CrylAb", "PAT/Bar", "PAT/paf and "CP4-EPSPS" using the real-time PCR methods of Example 1.<br>
The present simplified method intends to conclude on the potential presence or absence in the sample of material derived from Bt176, Bt11 and NK603.<br>
After testing, the sample is positive for p35S, CrylAb and Pat/Bar but not for the other tested nucleic acids. Hence, the sample can be represented as set GSAM e {p35S: CrylAb; Pat/bar}. In this exemplary assay (where the presence of Zea mays nucleic acids is not examined), the event Bt176 is represented by set Genre e {p35S: CrylAb; Pat/bar}, event Bt11 by set Getn e {p35S; tNOS; CrylAb; Pat/pat} and event NK603 by set GNK6D3 e {p35S; tNOS; CP4-EPSP}.<br>
Accordingly, since GBt176 = GSAM, material from event Bt176 may be present in the sample. On the other hand, because GBt11# GSAM and Getnc: GSAM, material from event Bt11 is not present in the sample. Similarly, because GNK603# GSAM and GNK603   GSAM, material from event NK603 is not present in the sample.<br>
in an alternative representation, "p35S", "tNOS", "CrylAb", "PAT/Bar", "PAT/pat" and "CP4-EPSPS" are assigned respective prime values Vp35S = 3, V(NOs = 5, VCo,rAt) = 7, VPAvBar = 11,<br>
VpAT/pa' = 13 and VcP4-EPSPS =17.<br>
Consequently, the set GSAM is assigned a value VGSAM being a multiple of the respective values assigned to nucleic acids detected therein, i.e., VGSAM = VP35s x McryiAb x VPAVBar = 231. Moreover, the set GBti76 is assigned a value VGBn76 being a multiple of the values assigned to those of the tested nucleic acids that are found in the event Bt176, i.e., VGBti76 = VP35s y- ycryiAb x VPAmar = 231. Analogously, the set GBtn is assigned a value VGBtn = Vp35s x VINOS X VcryiAb x VpAVPat = 1365; and the set GNK603 is assigned a value VGNK603 = Vp35S x V,wos x VCP4-EPSP - 255.<br>
 <br>
Accordingly, because VGSAM / VGBti76 = 1, material from event Bt176 may be present in the sample. On the other hand, because VGSAM / VGBtn = 0.169, i.e.. is not 1 and not an integer greater than 1, material from event Bt11 is not present in the sample. Similarly, because VGSAM / VGNK603 = 0.906, i.e., is not 1 and not an integer greater than 1, material from event NK603 is not present in the sample.<br>
Example 3: Exemplary description of a kit according to the invention (96 well plate setup)<br>
Nucleic acids to be screened: "2m", "Bn", "Gm", "p35S", "tNOS", "CP4-EPSPS", "CrylAb", "PAT/bar", "PAT/pat" as defined above using primer pairs as defined in Table 1 above, as well as generic plant gene amplified using primer pair SEQ ID NO: 26 and 27 (Table 4).<br>
Optionally, such as on request, the kit may also include taxon markers for rice ("Or"), cotton ("Gs"). sugar beet ("Bv"j and/or potato ("St"), using the corresponding primer pairs as defined in Table 1.<br>
Operational conditions of the SYBR Green Q-PCR methods<br>
Reagents: SYBR Green PCR Master Mix including the HS Taq-poiymerase; primers 20 M; nuclease-free water<br>
Equipment: ABI PRISM 7700 Sequence Detection System, 7300 Real-Time PCR System, Laminar Flow, Pipettes 20, 200 et 1000 yd, Sterile, aerosol-resistant pipette tips, Optical 96-well reaction plates, Optical caps<br>
Protocol: The PCR is conducted in an ABI PRISM* 7700 Sequence Detection System or the 7300 Real-Time PCR System following the manufacturer's instructions. One general PCR program is used.<br>
PCR Mix preparation: The PCR Mix contains all components of a PCR reaction except the DNA template. This operation is carried out in a laminar flow, which is only used for this kind of operation.<br>
Table 6: Reaction mixture for Q-PCR screening<br>
<br>
(Table Removed)<br>
DNA preparation: The template DNA is extracted using a standard CTAB DNA extraction method and the DNA concentration is measured by picogreen fluorescence determination.<br>
PCR Setup: The PCR setup is carried out in a laminar flow. The DNA templates and the PCR Mix are combined in different rooms.<br>
PCR Run: The PCR will be conducted following the manufacturer's instructions as described in the User's Manual.<br>
The Thermal Cycler Conditions are iisted below in Table 7<br>
Table 7: Thermocycler running conditions<br>
<br>
(Table Removed)<br>
Subsequently a "melting curve" analysis is performed using a standard program (eg gradual melting fom 50C to 95 C and monitoring the fluorescence decrease)<br>
GMO-screening 96 well Plate Set up for food/feed products<br>
Below is described the setup including all general food/feed markers, the rice marker (detection of unauthorized rice events) and the CAMV Reverse Transcriptase marker (absence of CaMV control).<br>
(Table Removed)<br>
Positive controls (+) would be the corresponding amplicons (plasmids or inserts) as described herein in particular in Table 5 amplicons at a certain copy-number (e.g. 100 copies/well)<br>
It would be technically foreseen that the sample wells can be opened sequentially (e.g. first the positive controls, then the samples, and last the NTC and negative controls).<br>
Ct values and Tm values for each well are recorded and used in the decision analysis, for GM-set determination present in the respective samples.<br>
Note that in principle also each row could be produced separately in an alternative set up (whereby increased flexibility for the customer is achieved).<br>
Example 4: p35S. tNOS multiplex PCR<br>
The present example shows exemplary multiplex PCR amplification of the p35S and tNOS amplicons, wherein detection is made by SYBR Green and the amplification products are distinguished on the basis of a difference in their melting temperatures (Tm).<br>
Primer pairs used (20fjMj:<br>
p35S:SEQ ID NO: 60 and 61<br>
tNOS: SEQ ID NO: 9 and 10<br>
Template (50 ng): Roundup Ready Soybean (RRS) 100% (double positive control)<br>
PCR Program: 95C 10', (95C 15", 60C 1') x40 cycles<br>
Melting program: 50'c to 95C in 20'<br>
 <br>
Multiplexing of PCR methods p35S and tNIOS in SYBR Green is possible since both arnplicons are distinguishable thank to there different Tm (see Figure 2). So it is possible to get information with one PCR on the presence of two different targets that are p35S and tIMOS.<br>
Example 5: Additional multiplexing possibilities when using SYBR Green<br>
Experiments analogous to those of example 4 have further defined the feasibility of the following multiplexing combinations:<br>
1.	Maize adh-1 (SEQ ID NO: 56 and 57) with soybean SLTM (SEQ ID NO: 58 and 59)<br>
2.	Maize adh-1 (SEQ ID NO: 56 and 57) with oilseed rape cruciferin (SEQ ID NO: 24 and 25)<br>
3.	Soybean SLTM (SEQ ID NO: 58 and 59) with cotton sah-7 (SEQ ID NO: 66 and 67)<br>
4.	Cotton sah-7 (SEQ ID NO: 66 and 67) with oilseed rape cruciferin (SEQ ID NO: 24 and 25)<br>
Example 6: Parameters of real-time PCR using the primers of the invention<br>
Reactions using conditions and primers as defined in example 1 have been performed on a defined copy number of reference plasmids containing the respective amplicon inserts, and relative fluorescence (RFU) following 40 cycles (i.e., approaching or having reached the plateau) has been determined. The results in Table 8 (the results using +/- 8300 template copies are also graphically represented in Figure 3) show that the present methods and primers are capable of generating RFU values at plateau of at least 2500, which lie within a 3-fold margin.<br>
Table 8<br>
(Table Removed) <br>
<br>
(<br>
<br>
<br>
<br>
<br>
<br>
<br>
<br>
<br>
<br>
<br>
We claim:<br>
1. A method to examine a sample for the presence or absence of material derived from one or more transgenic plant events comprising the steps of:<br>
(1)	detecting the presence or absence in the sample of nucleic acids comprising:<br>
-	one, more than one or all of nucleic acids chosen from:<br>
a)	"Zm": a nucleic acid derived from and specific for Zea mays taxon, preferably Zea mays ssp. mays,<br>
b)	"Bn": a nucleic acid derived from and specific for Brassica napus taxon,<br>
c)	"Gm": a nucleic acid derived from and specific for Glycine max taxon, o) "Or": a nucleic acid derived from and specific for Oryza sativa taxon, p) "Bv": a nucleic acid derived from and specific for Beta vulgaris taxon, q) "Gs": a nucleic acid derived from and specific for Gossypium taxon, and<br>
t) "St": a nucleic acid derived from and specific for Solarium tuberosum taxon; and<br>
-	one, more than one or all of nucleic acids chosen from:<br>
d)	"p35S": a nucleic acid derived from the 35S promoter of Cauliflower<br>
Mosaic Virus,<br>
e)	"tNOS": a nucleic acid derived from the 3' terminator of the<br>
Agrobacterium tumefaciens nopaline synthetase gene,<br>
f)	"CrylAb": a nucleic acid derived from the crystal protein gene Cry1Ab<br>
of Bacillus thuringiensis,<br>
g)	"PAT/bar": a nucleic acid derived from the phosphinothricin<br>
acetyltransferase (PAT) gene bar of Streptomyces hygroscopicus,<br>
h) "PAT/pat": a nucleic acid derived from the phosphinothricin acetyltransferase (PAT) gene pat of Streptomyces viridochromogenes, and<br>
i) "CP4-EPSPS": a nucleic acid derived from the 5-Enol-pyruvylshikimate-3-phosphate synthase EPSPS gene from Agrobacterium sp. CP4; and<br>
(2)	concluding the presence or absence in the sample of material derived from<br>
one or more transgenic plant events chosen from the group comprising: events<br>
Bt176,  Bt11,  Bt10,  MON810,  MON863, TC1507,  NK603, T25,  GA21,  DAS-<br>
 <br>
59122, MIR604, LY038, MON88017, crosses thereof, and related events thereof; events Topas 19/2, MS1, RF1, RF2, RF3, MS8, GT73, T45, Liberator pHoe6/Ac, GS40/90pHoe6/Ac, OXY235, crosses thereof including MS1/RF1, MS1/RF2, MS8/RF3, and related events thereof; events MON 40-3-2, MON89788, A2704-12, A5547-127, crosses thereof, and related events thereof, events LL62, LL06 and LL601, crosses thereof, and related events thereof; events T120-7, H7-1 and A5-15, crosses thereof, and related events thereof; events LL cotton 25, MON 1445, MON 531, MON15985, crosses thereof, and related events thereof; and event EH92-527-1 and related events thereof,<br>
wherein in step (1) the presence or absence of nucleic acids in a sample is detected using PCR amplification, preferably real-time PCR amplification, using the respective primer pairs from the following table, or variants or derivatives of said primer pairs:<br>
"Zm"	Fwd:  5' TCTCTTCCTCCTTTAGAGCTACCACTA 3'  (SEQ  ID<br>
NO: 56)<br>
Rev: 5' AATCGATCCAAAGCGAGATGA 3' (SEQ ID NO: 57)<br>
"Bn"	Fwd: 5' CAGCTCAACAGTTTCCAAACGA 3' (SEQ ID NO: 24)<br>
Rev: 5' CGACCAGCCTCAGCCTTAAG 3' (SEQ ID NO: 25)<br>
"Gm"	Fwd: 5' AACCGGTAGCGTTGCCAG 3' (SEQ ID NO: 58)<br>
Rev': 5' AGCCCATCTGCAAGCCTTT 3' (SEQ ID NO: 59)<br>
"p35S"	Fwd: 51 AAAGCAAGTGGATTGATGTGATA 31 (SEQ ID NO: 60)<br>
Rev: 5' GGGTCTTGCGAAGGATAGTG 3' (SEQ ID NO: 61)<br>
"tNOS"	Fwd: 5'-GATTAGAGTCCCGCAATTATACATTTAA-3' (SEQ ID<br>
NO: 9)<br>
Rev:  5'-TTATCCTAGKTTGCGCGCTATATTT-3' (SEQ ID NO:<br>
10)<br>
"CrvlAb"	Fwd: 5'-ACCGGTTACACTCCCATCGA-3' (SEQ ID NO: 11)<br>
Rev: 5'-CAGCACCTGGCACGAACTC-3' (SEQ ID NO: 12)<br>
"PAT/bar"	Fwd:   5'-CGTCAACCACTACATCGAGACAA-3'  (SEQ  ID   NO:<br>
13}<br>
Rev: 5'-GTCCACTCCTGCGGTTCCT-3' (SEQ ID NO: 14)<br>
"PAT/pat"	Fwd: 5'-CCGCGGTTTGTGATATCGTT-3' (SEQ ID NO: 15)<br>
Rev: 5'-TCTTGCAACCTCTCTAGATCATCAA-3' (SEQ ID NO:<br>
161<br>
"CP4-	Fwd: 5' GCATGCTTCACGGTGCAA 3' (SEQ ID NO: 22)<br>
EPSPS"	Rev: 5' GGACCTGTGGGAGATAGACTTGTC 3' (SEQ ID NO:<br>
23]<br>
Rev 1: 5' TGAAGGACCGGTGGGAGAT 3' (SEQ ID NO: 62)<br>
Rev 2: 5' TGAAGGACCTGTGGGAGAT 3' (SEQ ID NO: 63)<br>
"Or"	Fwd': 5' GCTTAGGGAACAGGGAAGTAAAGT 3' (SEQ ID NO:<br>
51)<br>
Rev: 5' CTTAGCATAGTCTGTGCCATCCA 3' (SEQ ID NO: 21)<br>
"By"	Fwd: 5' GACCTCCATATTACTGAAAGGAAG 3' (SEQ ID NO:<br>
64)<br>
Rev: 5' GAGTAATTGCTCCATCCTGTTCA 3' (SEQ ID NO: 65)<br>
 <br>
"Gs"	Fwd: 5' AGTTTGTAGGTTTTGATGTTACATTGAG 3' (SEQ ID<br>
NO: 66)<br>
Rev: 5' GCATCTTTGAACCGCCTACTG 3' (SEQ ID NO: 67)<br>
"St"	Fwd: 5' GGACATGTGAAGAGACGGAGC 3' (SEQ ID NO: 68)<br>
Rev: 5' CCTACCTCTACCCCTCCGC 3' (SEQ ID NO: 69)<br>
2.	The method according to claim 1, wherein step (1) involves detecting the presence or absence in the sample of nucleic acids comprising all nucleic acids listed under d) - i).<br>
3.	The method according to claim 1, wherein step (1) involves detecting the presence or absence in the sample of nucleic acids comprising all nucleic acids listed under a) - c) and d) - i).<br>
4.	The method according to any of claims 1 to 3,<br>
<br>
-	wherein step (1) further comprises detecting the presence or absence in the sample of nucleic acid j) "mCry3A": a nucleic acid derived from the modified crystal protein gene Cry3A of Bacillus thuringiensis; and/or<br>
-	wherein step (1) further comprises detecting the presence or absence in the sample of one or both nucleic acids k) "cordapA": a nucleic acid derived from the lysine-insensitive dihydrodipicolinate synthase (cDHDPS) gene cordapA of Corynebacterium glutamicum and/or I) "Glb1": a nucleic acid derived from the Glb1 promoter of maize; and/or<br>
-	wherein step (1) further comprises detecting the presence or absence in the sample of nucleic acid m) "Cry3Bb1": a nucleic acid derived from the crystal protein gene Cry3Bb1 of Bacillus thuringiensis; and/or<br>
-	wherein step (1) further comprises detecting the presence or absence in the sample of nucleic acid n) "Bxn": a nucleic acid derived from the nitrilase gene Bxn of Klebsiella pneumoniae ssp. ozaenae; and/or<br>
-	wherein step (1) further comprises detecting the presence or absence in the sample of one or both of nucleic acids r) "Cry1Ac": a nucleic acid derived from the crystal protein gene CrylAc of Bacillus thuringiensis and/or s) "Cry2Ab2": a nucleic acid derived from the crystal protein gene Cry2Ab2 of Bacillus thuringiensis; and/or<br>
 <br>
- wherein step (1) further comprises detecting the presence or absence in the sample of nucleic acid u) "GBSS": a nucleic acid derived from the granule bound starch synthase gene Gbss of Solanum tuberosum.<br>
5.	The method according to any of claims 1 to 4, wherein step (1) further comprises detecting the presence or absence in the sample of a generic plant-derived nucleic acid, preferably derived from the chloroplastic small subunit of Rubisco gene or from the CHL-tRNA synthetase gene.<br>
6.	The method according to claim 5, wherein the presence or absence of said generic plant-derived nucleic acid is detected using PCR amplification, preferably real-time PCR amplification, using the primer pair 5' AGGTCTAADGGRTAAGCTAC 3' (SEQ ID NO: 26) and 5' AGYCTTGATCGTTACAAAGG 3' (SEQ ID NO: 27), or variant or derivative of said primer pair.<br>
7.	The method according to any of claims 1 to 6, wherein the PCR amplification of the nucleic acids is performed simultaneously at the same temperature cycling conditions.<br>
8.	The method according to any one of claims 1 to 7, wherein the PCR amplification of two or more of the nucleic acids is multiplexed.<br>
9.	The method according to any of claims 1 to 8, wherein the amplification products are detected using a detection method that is substantially sequence non-specific, preferably using a DNA-binding fluorescent dye, more preferably using SYBR Green or PicoGreen.<br>
10.	The method according to any of claims 1 to 9, wherein the specificity of the<br>
amplification products is further verified, preferably using melting curve analysis (Tm)<br>
and/or size determination by gel elecrophoresis.<br>
11.	The method according to any of claims 1 to 8, wherein the amplification products are<br>
detected by means of a fluorophore label present in at least one primer of the primer<br>
pair, such as using Light Upon Extension (LUX) technology.<br>
 <br>
12.	The method according to any of claims 1 to 11, wherein the sample comprises plants or parts thereof, including flowers, tepals, petals, sepals, anthers, pollen, seeds, fruits, pericarp, pods, leaves, petioles, stems, roots, rhizomes, stolons, tubers or shoots, or portions thereof, plant cells, plant protoplasts and/or plant tissues, and/or plant-derived material, preferably food or feed material, including processed food or feed material.<br>
13.	The method according to any of claims 1 to 12, wherein in step (1) the presence or absence of nucleic acids in a sample is detected using PCR amplification, preferably real-time PCR amplification, using the respective primer pairs from the following table, or variants or derivatives of said primer pairs:<br>
"Zm"	Fwd: 5' CgTCgTTTCCCATCTCTTCCTCC 3' (SEQ ID NO: 1)<br>
Rev: 5' CCACTCCgAgACCCTCAqTC 3' (SEQ ID NO: 2)<br>
"Zm"	Fwd: 5' TCTCTTCCTCCTTTAGAGCTACCACTA 3' (SEQ ID<br>
NO: 56)<br>
Rev: 5' AATCGATCCAAAGCGAGATGA 3' (SEQ ID NO: 57)<br>
"Bn"	Fwd1: 5' GAGAATGAGGAGGACCAAGCTC 3' (SEQ ID NO:<br>
4)<br>
Fwd2:  5'GGTGAGCTGTATAATCGAGCGA 3' (SEQ ID NO:<br>
79)<br>
Rev: 5' GGCGCAGCATCGGCT 3' (SEQ ID NO: 3)<br>
"Bn"	Fwd: 5' CAGCTCAACAGTTTCCAAACGA 3' (SEQ ID NO: 24)<br>
Rev: 5' CGACCAGCCTCAGCCTTAAG 3' (SEQ ID NO: 25)<br>
"Gm"	Fwd: 5' CATTACCTATgATgCCTCCACC 3' (SEQ ID NO: 5)<br>
Rev: 51 AAgCACgTCATgCgATTC 3' (SEQ ID NO: 6)<br>
Rev': 5' AAqCACqTCATqCqATTCC 3' (SEQ ID NO: 49)<br>
"Gm"	Fwd: 5' AACCGGTAGCGTTGCCAG 3' (SEQ ID NO: 58)<br>
Rev': 5' AGCCCATCTGCAAGCCTTT 3' (SEQ ID NO: 59)<br>
"P35S"	Fwd: 5'-GACAGTGGTCCCAAAGATGG-3' (SEQ ID NO: 7)<br>
Rev: 5'-GTCTTGCGAAGGATAGTGGG-3' (SEQ ID NO: 8)<br>
"P35S"	Fwd: 5' AAAGCAAGTGGATTGATGTGATA 3' (SEQ ID NO:<br>
60)<br>
Rev: 5' GGGTCTTGCGAAGGATAGTG 3' (SEQ ID NO: 61)<br>
"tNOS"	Fwd: 5'-GATTAGAGTCCCGCAATTATACATTTAA-3' (SEQ ID<br>
NO: 9)<br>
Rev: 5'-TTATCCTAGKTTGCGCGCTATATTT-3' (SEQ ID NO:<br>
10)<br>
"tNOS"	Fwd: 5' CGTTCAAACATTTGGCAATAAAG 3' (SEQ ID NO:<br>
28)<br>
Rev:  5' AAATGTATAATTGCGGGACTCTAATC  3'  (SEQ ID<br>
NO: 29)<br>
"Cry1Ab"	Fwd: 5'-ACCGGTTACACTCCCATCGA-3' (SEQ ID NO: 11)<br>
Rev: 5'-CAGCACCTGGCACGAACTC-3' (SEQ ID NO: 12)<br>
"Cry1Ab"	Fwd: 5' GACATCATCTGGGGYATCTT 3' (SEQ ID NO: 30)<br>
Rev: 5' GCGCTGTTCATGTCGTTGAA 3' (SEQ ID NO: 31)<br>
 <br>
"Cry1Ab"	Fwd: 5' ACGCCTTCCTGGTGCAAA 3' (SEQ ID NO: 47)<br>
Rev: 5' CCTGGTTCCTGGCGAACTC 3' (SEQ ID NO: 48)<br>
"PAT/bar"	Fwd: 5'-CGTCAACCACTACATCGAGACAA-3' (SEQ ID NO:<br>
13)<br>
Rev: 5'-GTCCACTCCTGCGGTTCCT-3' (SEQ ID NO: 14)<br>
"PAT/pat"	Fwd: 5'-CCGCGGTTTGTGATATCGTT-3' (SEQ ID NO: 15)<br>
Rev: 5'-TCTTGCAACCTCTCTAGATCATCAA-3' (SEQ ID NO:<br>
16)<br>
"CP4-	Fwd1:   5'-GGCTCTGAGCTTCGTCCTCCTAAGG-3'   (SEQ   ID<br>
EPSPS"	NO: 17)<br>
Fwd1':   5'-GGCTCTGAGCTTCGTCCTCTTAAGG-3'  (SEQ  ID<br>
NO: 50)<br>
Fwd2: 5'-ATCAGTGGCTACAGCCTGCAT-3' (SEQ ID NO: 18)<br>
Rev:   5'-GAATGCGGACGGTTCCGGAAAG-3'  (SEQ  ID  NO:<br>
19)<br>
"CP4-	Fwd: 5' GCATGCTTCACGGTGCAA 3' (SEQ ID NO: 22)<br>
EPSPS"	Rev: 5' GGACCTGTGGGAGATAGACTTGTC 3' (SEQ ID NO:<br>
23)<br>
Rev 1: 5' TGAAGGACCGGTGGGAGAT 3' (SEQ ID NO: 62)<br>
Rev 2: 5' TGAAGGACCTGTGGGAGAT 3' (SEQ ID NO: 63)<br>
"Or"	Fwd:  5' GCTTAGGGAACAGGGAAGTAAAGTT 3' (SEQ ID<br>
NO: 20)<br>
Fwd': 5' GCTTAGGGAACAGGGAAGTAAAGT 3' (SEQ ID NO:<br>
51}<br>
Rev: 5' CTTAGCATAGTCTGTGCCATCCA 3' (SEQ ID NO: 21)<br>
"Bv"	Fwd: 5' GACCTCCATATTACTGAAAGGAAG 3' (SEQ ID NO:<br>
64)<br>
Rev: 5' GAGTAATTGCTCCATCCTGTTCA 3' (SEQ ID NO: 65)<br>
"Gs"	Fwd: 5' AGTTTGTAGGTTTTGATGTTACATTGAG 3' (SEQ ID<br>
NO: 66)<br>
Rev: 5' GCATCTTTGAACCGCCTACTG 31 (SEQ ID NO: 67)<br>
"St"	Fwd: 5' GGACATGTGAAGAGACGGAGC 3' (SEQ ID NO: 68)<br>
Rev: 5' CCTACCTCTACCCCTCCGC 3' (SEQ ID NO: 69)<br>
generic	Fwd: 5' AGGTCTAADGGRTAAGCTAC 3' (SEQ ID NO: 26)<br>
plant	Rev: 5' AGYCTTGATCGTTACAAAGG 3' (SEQ ID NO: 27)<br>
14. The method according to any of the preceding claims,  wherein the step (2) comprises the steps of:<br>
(i) defining a set "GSAM" consisting of nucleic acids detected in step (1) as being<br>
present in the sample;<br>
(ii) defining one or more set of interest ("Gx") chosen from the group of sets<br>
comprising or consisting of sets "GBt176", "GBt11", "GBt10", "GMON810", "GMON863",<br>
"TC1507 ",	"GNK603",   "GT25",  "GA21",  "DAS-59122"  , "GMIR604 " , GLY038" <br>
"GMON88017" , "GTopas 19/2 ",    "GMS1",  "GRF1",  "GRF2" ,   "GMS1/RF1 "  ,    "GMS1/RF2"  ,    "GMS8"  .   "GRF3",<br>
"GMS8/RF3",    "GT73",    "GT745 ",    "GLiberator pHoe6/Ac ",    "GS40/90pHoe6/Ac ",    "GOXY235 ",    "GMON40-3-2 ",<br>
 <br>
"GMON89788" ,     "GA2704-12"  ,     "GA5547-127"  ,     "GLL62",  .     "GLL06",  "GLL.601"  ,     "GT120-7 ",    "GH7-1 "  , "GA5-<br>
15 .     "GLL cotton 25 ",     "GMON1445",  "GMON531",  "GMON15985" and  "GEH92-527-1",  Corresponding<br>
to the respective one or more transgenic plant events of interest ("X") chosen from the group comprising or consisting of events Bt176, Bt11, Bt10, MON810, MON863, TC1507, NK603, T25, A21, DAS-59122, MIR604, LY038, MON88017,Topas 19/2, MS1, RF1, RF2, MS1/RF1, MS1/RF2, MS8, RF3, MS8/RF3, GT73, T45, Liberator pHoe6/Ac, GS40/90pHoe6/Ac, OXY235; MON40-3-2, MON89788, A2704-12, A5547-127, LL62, LL06, LL601, T120-7, H7-1, A5-15, LL cotton 25, MON1445, MON531, MON15985 and EH92-527-1, wherein:<br>
-	GBt176  {Zm; p35S; Cry1Ab; PAT/bar},<br>
-	GBt11  {Zm; p35S; tNOS; Cry1Ab; PAT/pat},<br>
-	GBt10  {Zm; p35S; tNOS; Cry1Ab; PAT/pat},<br>
-	GMON810  {Zm;p35S; tNOS; Cry1Ab},<br>
-	GMON863  {Zm; p35S; tNOS},<br>
-	GTC1507  {Zm; p35S; PAT/pat},<br>
-	GNK603  {Zm; p35S; tNOS; CP4-EPSPS},<br>
-	GT25  {Zm; p35S; PAT/pat},<br>
-	GGA21  {Zm; tNOS},<br>
-	GDAS-59122  {Zm; p35S; PAT/bar},<br>
-	GMIR604  {Zm; tNOS; mCry3A}, or GMIR604  {Zm; tNOS }<br>
-	GLY038  {Zm; p35S; tNOS; cordapA; Glb1}, or GLyo38  {Zm; p35S; tNOS },<br>
-	GMON88017  {Zm; p35S; tNOS; CP4-EPSPS; Cry3Bb1}, or GMON88017  {Zm; p35S; tNOS; CP4-EPSPS},<br>
-	Gropas 19/2  {Bn; p35S; PAT/pat},<br>
-	GMS1  {Bn; tNOS; PAT/bar},<br>
-	GRF1  {Bn; tNOS; PAT/bar},<br>
-	GRF2  {Bn; tNOS; PAT/bar},<br>
-	GMS1/RF1  {Bn; tNOS; PAT/bar},<br>
-	GMS1/RF2  {Bn; tNOS; PAT/bar},<br>
-	GMS8  {Bn; tNOS; PAT/bar},<br>
-	GRF3  {Bn; tNOS; PAT/bar},<br>
-	GMS8/RF3  {Bn; tNOS; PAT/bar},<br>
 <br>
-	GGT736 {Bn; CP4-EPSPS},<br>
-	GT45  {Bn; p35S; PAT/pat},<br>
-	GLiberatorpHoe6/Ac -{Bn; p35S; PAT/pat},<br>
-	GGS40/90PHoe6/Ac  {Bn; p35S; PAT/pat},<br>
-	GOXY235  {Bn; p35S; Bxn}, or GOXY235  {Bn; p35S};<br>
-	GMON40-3-2  {Gm; p35S; tNOS; CP4-EPSPS},<br>
-	GMON89788  {Gm; CP4-EPSPS}<br>
-	GA2704-12  {Gm; p35S; PAT/pat},<br>
-	GA5547-127  {Gm; p35S; PAT/pat},<br>
-	GLL62  {Or; p35S; PAT/bar},<br>
-	GLL06  {Or; p35S; PAT/bar}<br>
-	GLL601  {Or; p35S; tNOS; PAT/bar},<br>
-	GT120-7  {Bv; p35S; PAT/pat},<br>
-	GH7-1  {BV; p35S; CP4-EPSPS},<br>
-	GA5_15  {Bv; p35S; tNOS; CP4-EPSPS},<br>
-	GLL cotton 25  {Gs; p35S; tNOS; PAT/bar},<br>
-	GMON1445  {Gs; p35S; tNOS; CP4-EPSPS},<br>
-	GMON531  {Gs; p35S; tNOS; cry1Ac}, or GMON531  {GS; p35S; tNOS}<br>
-	GM0N15985  {Gs; p35S; tNOS; crylAc; cry2Ab2} or GMON15985  {Gs; p35S; tNOS}, and<br>
-	GEH92-527-1  {St; tNOS; Gbss} or GEH92-527-1  {St; tNOS};<br>
(iii) performing for each set of interest Gx logical operations:<br>
-	if Gx equals GSAM (GX - GSAM), then material derived from the transgenic plant event X or from a cross thereof, or from an event related thereto, is potentially present in the sample;<br>
-	if Gx is a proper subset of GSAM (GX  GSAM), then material derived from transgenic plant event X or from a cross thereof, or from an event related thereto, is potentially present in the sample;<br>
if Gx does not equal GSAM and Gx is not a proper subset of GSAM, (Gx GSAM and Gx  2 GSAM), then material derived from transgenic plant event X or from a cross thereof, or from an event related thereto, is absent from the sample.<br>
 <br>
15.	The method according to claim 14, wherein if Gx equals GSAM, and if no one set or<br>
the sum of two or more sets chosen from the group comprising or consisting of GBt176,<br>
GBt11, GBt10. GMON810, GMON863, GTC1507, GNK603, GT25, GGA21, GDAS-59122, GMIR604, GLY038, GMON88017,GTopas19/2, GMS1, GRF1, GRF2, GMS1/RF1, GMS1/RF2, GMS8, GRF3, GMS8/RF3, GGT73, GT45, GLiberator pHoe6/Ac, GGs40/90pHoe6/Ac, GOXY235, GMON40-3-2, GMON89788, GA2704-12, GA5547-127, GLL62, GLL06,   GLL601,  GT120-7,   GH7-1,   GAS-15,  GLL cotton 25,   GMON1445,   GMON531,   GMON15985 Other than  GX<br>
equals Gx, then material derived from transgenic plant event X or from a cross thereof, or from an event related thereto, is present in the sample.<br>
16.	The method according to claim 14, wherein nucleic acids chosen from the group<br>
comprising or consisting of "Zm", "Bn", "Gm", "p35S", "tNOS", "Cry1Ab", "PAT/bar",<br>
"PAT/pat", "CP4-EPSPS", "mCry3A", "cordap A", "Glb1", "Cry3Bb1", "Bxn", "Or", "Bv",<br>
"Gs", "Cry1Ac", "Cry2ab2", "St" and "Gbss" are assigned respective unique values "VZm",<br>
"VBn ",     "VGm" ,     "V p35S ",    "VtNOS",    "VCry1Ab" ,     "VPAT/bar" ,     "VPAT/pat" ,     "VCP4-EPSPS" ,    "VmCry3A ", "Vcordap A" ,<br>
"VG/B1", "Vcry3Bb1" "VBxn", "Vor", "VBv", "VGs", "Vcry1Ac", "Vcry2ab2", "Vst" and "VGbss"; wherein each set of interest Gx is assigned a value "VGX" chosen from the group comprising consisting of values "VGBt176", "VGBt176", "VGBt10", "VGMON810",  "VGMON863",  "VGTC1507",<br>
"VNK603"  .  "V"G125 ,     "VG3GA21"  ,      "VGDAS-59122"  ,      "VMIR604"  ,      "VGLY038"  ,      "VGMON88017" ,      "VGTopas"<br>
19/2"  ,      "VGMS1"   ,      "VGRF1"   ,      "VGRF2"  ,      "VGMS1/RF1"   ,      "VGMS1/RF2  ,      VGMS8" ,      "VGRF3"  ,      VGMS8/RF3"  ,<br>
VGGT73" ,       VGT45" ,       "VGLiberator   pHoe6/Ac" ,       "VGGS40/90pHoe6/Ac ",       "VGOXY235" ,	"VMON40-3-2" ,<br>
"VMON89788"  ,     "VGA2704-12"  ,     "VGA5547-127" ,     "VGLL62 " ,     "VGLL06 ",     "VLL601 " ,     "VGT120-7 ,"     "VGH7-1"  , VGA5-15",     "VGLL    cotton    25",     "VGMON1445",        "VGMON531",     "VGMON15985"     and     "VGEH92-527-1"<br>
wherein:<br>
-	VGBt176 - Vzm x VP35S x VCrylAb X VPAT/bar,<br>
-	VGBt11 = VZm x Vp35S x VtNOS X VCrylAb x VPAT/pat,<br>
-	VGBt10 = VZm x VP35S x VtNOS x VCry1Ab x VPAT/pat, - VGMON870 = VZm x Vp35S x VtNTOS X VCry1Ab,<br>
-	VGMON863 = VZm X Vp35S X VtNOS,<br>
-	VGTC7507 =   VZm x Vp35s x VPAT/pat,<br>
-	VGNK603 = V2m x Vp35S x VtNos X VCP4-EPSPS,<br>
-	VGT25 = Vzm x Vp35s x VPAT/pat,<br>
-	VGGA21 = Vzm X VtNOS,<br>
 <br>
-	VGDAS-59122 = VZm x VP35S x VPAT/bar,<br>
- VGMIR604 = VZm x VtNOS X VmCry3A, Or VGMIR604 = Vzm X VtNOS,<br>
-	VGLY038 = Vzm x VP35S x VtNOS x VcordapA x VGIb1, or VGLY038 = VZm x VP35S x VtNOS,<br>
- VGMON88017 = Vzm  x  VP35S x  VtNOS x  VCP4-EPSP x  VCry3Bb1,  or VGMON88077 = VZm  x VP35S X VtNOS X VCP4-EPSP,<br>
-	VGTopas 19/2 = VBn x VP35S x VPAT/pat,<br>
- VGMS1 =   VBn x VtNOS X VPAT/bar,<br>
- VGRF1 = VBn x VtNos X VPAT/bar,<br>
-	VGRF2 = VBn x VtNOS x VPAT/bar, - VGMS8 = VBn x VtNOS x VPAT/bar,<br>
-	VGRF3 = VBn x VtNoS x VPAT/bar,<br>
- VGMS1/RF1 = VBn x VtNOS X VPAT/bar,<br>
-	VGMS1/RF2 = VBn X VtNOS X VPAT/bar,<br>
-	VGMS8/RF3 = VBn x VtNOSX VPAT/bar,<br>
-	VGGT73 = VBn x VCP4-EPSPS,<br>
-	VGT45 = VBn x Vp35S x VPAT/pat<br>
- VGLiberator pHoe6/Ac = VBn X VP35S x VPAT/pat,<br>
-	VGGS40/90pHoe6/Ac = VBn x Vp35s x VPAT/pat,<br>
-	VGoxY235 = VBn X VP35S x VBxn, or VGOXY235 = VBn x Vp35S, - VGMOW40-3-2 = VGm x VP35S x VtNOS X VCP4-EPSPS,<br>
-	VGNON89788 = VGm x VCP4-EPSPS,<br>
-	VGA2704-12 = VGm X VP35S x VPAT/pat,<br>
-	VGA5547-127 = VGm X VP35S x VPAT/pat,<br>
-	VGLL62 = Vor x VP35S x VpAT/bar,<br>
-	VGLL06 = Vor x VP35S x VPAT/bar,<br>
- VGLL601 = Vor x VP35s x VtNOs x VPAT/bar,<br>
-	VGT120-7 = VBv x VP35S x VPAT/pat,<br>
-	VGH7-1 = VBV X VP35S x VCP4-EPSPS,<br>
-	VGA5-15 = VBv x VP35S x VtNOS x VCP4-EPSPS,<br>
cotton 25 = VGs x VP35S x VtNOS X VPAT/bar, - VGMON1445 = VGs x VP35S x VtNOS x VCP4-EPSPS,<br>
-	VGMON531 = VGs x VP35S x VtNOS x VCry1Ac, or VGMON531 = VGs x VP35S x VtNOS.<br>
- VGMON15985 = VGs x VP35S x VtNOS x ^Cry1Ac x VCry2Ab2, or VGMON15985 = VGS X Vp35S x VtNOS,<br>
 <br>
- and VGEH92-527-1 = VSt x VtNOS x VGbss, or VGEH92-527-1 = VSt x VtNOS,<br>
and wherein the set GSAM is assigned a value "VGSAM" which is a multiple of the<br>
unique values assigned to the nucleic acids detected in step (1) as being present in<br>
the sample;<br>
and wherein step (iii) comprises performing for each set of interest Gx logical<br>
operations:<br>
-	if VGSAM /VGX equals 1, then material derived from transgenic plant event X or from a cross thereof, or from an event related thereto, is potentially present in the sample;<br>
-	if VGSAM /VGX equals one or a multiple of two or more values chosen from a group comprising or consisting of values VZm, VBn, VGm, Vp35S, MINOS, Vcry1Ab, VPAT/bar, VPAT/pat,<br>
VCP4-EPSPS, VmCry3A, Vcordap A, VGlb1, Vcry3Bb1, VBxn, Vor, VBv, VGs, Vcry1Ac,  VCry2ab2, VSt and<br>
VGbss, then material derived from transgenic plant event X or from a cross thereof, or from an event related thereto, is potentially present in the sample;<br>
-	if VGSAM / VGX does not equal 1 and does not equal one value or a multiple of two<br>
or more values chosen from a group comprising or consisting of values VZm, VBn,<br>
VGM,  VP35S,  VtNOS,  Vcry1Ab,  VPAT/bar,  VPAT/pat,  VCP4-EPSPS,   VmCry3A,  Vcordap A,   VGIB1,  Vcry3Bb1,<br>
VBxn, Vor VBv, VGs, Vcry1Ac, VCry2ab2, VSf and VGbss, then material derived from transgenic plant event X or from a cross thereof, or from an event related thereto, is absent from the sample.<br>
17. The method according to claim 16, wherein if VGSAM I VGX equals 1, and if no one value or the multiple of two or more values of sets chosen from a group comprising or consisting of: VGBt176, VGBf11, VGBt10, VGMON810, VGMON863, VGTC1507, VGNK603, VGT25,<br>
VGGA21, VGDAS-59122, VGMIR604, VGLY038, MGMON88017, MGTopas 19/2, VGMS1, MGRF1, VGRF2, VGMS1/RF1, VGMS1/RF2, VGMS8. VGRF3, VGMS8/RF3, VGG773, VGT45, VGLiberator pHoe6/Ac, vGGS40/90pHoe6/Ac, VGOXY235, VGMON40-3-2, VGMON89788, VGA2704-12, VGA5547-127, VGLL62, VGLL06, VGLL601,    VGT120-7,    VGH7-1.    MGA5-15,   VGLL.   cotton   25,    MGMON1445,    VGMON531,    MGMON15985   and<br>
VGEH92-527-1 other than VGX equals VGX, then material derived from transgenic plant event X or from a cross thereof, or from an event related thereto, is present in the sample.<br>
 <br>
18.	The method according to any of claims 16 or 17, wherein values VZm, VBn, VGm, Vp35S,<br>
VtNOS.   Vcry1Ab.   VPAT/bar,   VpAT/pat,   VCP4-EPSPS,   VmCry3A,   Vcordap A,   "Glb1,   VCry3Bb1,   VBxn,   Vor,   VBv,<br>
VGS, Vcry1Ac, VCry2ab2, VSt and VGbss are unique prime numbers, and wherein step (iii) comprises performing for each set of interest Gx logical operations,<br>
-	if VGSAM /VGX equals 1, then material derived from transgenic plant event X or from a cross thereof, or from an event related thereto, is potentially present in the sample;<br>
-	if VGSAM / VGX is an integer greater than 1, then material derived from transgenic plant event X or from a cross thereof, or from an event related thereto, is potentially present in the sample;<br>
-	if VGSAM / VGX is not an integer, then material derived from transgenic plant event X or from a cross thereof, or from an event related thereto, is absent from the sample.<br>
19.	A method to determine the presence or absence of two or more events, preferably<br>
transgenic plant events, or material thereof in a sample, wherein each event is<br>
characterised by one or more traits, comprising the steps of:<br>
(1)	detecting the presence or absence in the sample of said one or more traits,<br>
(2)	concluding the presence or absence in the sample of said one or more events, comprising:<br>
<br>
(a)	assigning a unique prime number value to each trait,<br>
(b)	representing each event as a product of the prime number values assigned to traits characteristic of said event,<br>
(c)	representing the sample as a product of the prime number values assigned to traits detected as present in the sample, and<br>
(d)	dividing the product representing the sample as defined in (c) by the product representing an event as defined in (b), whereby:<br>
<br>
-	if the value obtained by said division equals 1, then said event or material thereof is potentially present in the sample;<br>
-	if the value obtained by said division is an integer greater than 1, then said event or material thereof is potentially present in the sample;<br>
- if the value obtained by said division is not an integer, then said event or material thereof is absent from the sample.<br>
 <br>
20.	The method according to any of claims 1 to 19, wherein step (2) is carried out by a computing device.<br>
21.	A computing device programmed to carry out step (2) of the method according to any of claims 1 to 20; or a data carrier comprising instructions for a programmable computing device to carry out the step (2) of the method according to any of claims 1 to 20.<br>
22.	Primers and primer pairs as defined in claims 1, 6 and 11.<br>
23.	Amplification products obtainable by using primer pairs as defined in claim 1, 6 and 11; recombinant vectors and plasmids comprising the said amplification products; recombinant microorganisms transformed with the said vectors and/or plasmids.<br>
24.	Recombinant E. coli as deposited under the Budapest Treaty with the Belgian Coordinated Collections of Microorganisms (BCCM) on January 10, 2007 under LMBP accession numbers LMBP 5452, LMBP 5453, LMBP 5454, LMBP 5455, LMBP 5456, LMBP 5457, LMBP 5458, LMBP 5459 and LMBP 5460, on March 6, 2007 under LMBP accession number LMBP 5451, and on April 19, 2007 under LMBP accession numbers LMBP 5587, LMBP 5588, LMBP 5589 and LMBP 5590.<br>
25.	Isolated recombinant plasmids obtainable from any of recombinant E. coli bacteria according to claim 24, or isolated inserts of the said plasmids, preferably isolated EcoRI-EcoRI inserts thereof.<br>
26.	A recombinant microorganism transformed with a plasmid according to claim 25.<br>
27.	A kit for examining a sample for the potential presence or absence of material derived from one or more transgenic plant events, the kit comprising one, more than one or all primers, and preferably one, more than one or all primer pairs as defined in claim 1, 6 and 11, or variants or derivatives of said primer pairs.<br>
28.	The kit according to claim 27, comprising primer pairs or variants or derivatives thereof for amplification of at least the nucleic acids "Zm", "Bn", "Gm", "p35S", "tNOS", "Cry1Ab", "PAT/bar", "PAT/pat" and "CP4-EPSPS" as defined in claim 1.<br>
 <br>
29.	The kit according to any of claims 27 or 28, further comprising one or more nucleic acids suitable as a positive control for the amplification of nucleic acids according to any of claims 1 to 5.<br>
30.	The kit of any of claims 27 to 29, comprising one or more recombinant E. coli bacteria as defined in claim 24, recombinant microorganisms as defined in claim 26, isolated recombinant plasmids or inserts according to claim 25.<br>
31.	The kit of any of claims 27 to 30 further comprising a data carrier comprising instructions for a programmable computing device to carry out step (2) of the method according to any of claims 1 to 20.<br>
</td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/viewdoc.aspx?id=PlW7kENNO+ZFAj+afcybLQ==&amp;amp;loc=+mN2fYxnTC4l0fUd8W4CAA==" target="_blank" style="word-wrap:break-word;">http://ipindiaonline.gov.in/patentsearch/GrantedSearch/viewdoc.aspx?id=PlW7kENNO+ZFAj+afcybLQ==&amp;amp;loc=+mN2fYxnTC4l0fUd8W4CAA==</a></p>
		<br>
		<div class="pull-left">
			<a href="278034-a-process-for-chiral-resolution-of-2-substituted-4-substituted-1-3-oxathiolanes.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="278036-method-and-apparatus-for-supporting-harq.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>278035</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>3991/DELNP/2009</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>52/2016</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>16-Dec-2016</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>08-Dec-2016</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>17-Jun-2009</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>SCIENTIFIC INSTITUTE OF PUBLIC HEALTH (IPH)</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td>RUE JULIETTE WYTSMANSTRAAT 14, B-1050 BRUXELLES, BELGIUM</td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>VAN DEN BULCKE, MARC HENRI GERMAIN</td>
											<td>SCHOONMEERSSTRAAT 29, B-9000 GENT, BELGIUM</td>
										</tr>
										<tr>
											<td>2</td>
											<td>LIEVENS, ANTOON PIET NELLY RAOUL</td>
											<td>KONING BOUDEWIJNSTRAAT 16, B-9000 GENT, BELGIUM</td>
										</tr>
										<tr>
											<td>3</td>
											<td>LEUNDA, AMAYA</td>
											<td>74 RUE DE LIVOURNE, B-1000, BRUXELLES, BELGIUM</td>
										</tr>
										<tr>
											<td>4</td>
											<td>BARBAU-PIEDNOIR, ELODIE</td>
											<td>20, ROUTE D&#x27;AUNAY, F-45130 MEUNG SUR SOIRE, FRANCE</td>
										</tr>
										<tr>
											<td>5</td>
											<td>MBONGOLO MBELLA, ETONDOH GUILLAUME</td>
											<td>188 ROUTE DE LENNIK, B-1080, BRUXELLES, BELGIUM</td>
										</tr>
										<tr>
											<td>6</td>
											<td>SNEYERS, MYRIAM JACQUELINE SYLVIANE</td>
											<td>16, RUE DE LA GOYETTE, B-5030 GEMBLOUX, BELGIUM</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>C12Q 1/68</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>PCT/EP2008/051059</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td>2008-01-29</td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td>07108343.0</td>
									<td>2007-05-16</td>
								    <td>EPO</td>
								</tr>
								<tr>
									<td>2</td>
									<td>07447008.9</td>
									<td>2007-01-29</td>
								    <td>EPO</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/278035-transgenic-plant-event-detection by HTTrack Website Copier/3.x [XR&CO'2014], Thu, 04 Apr 2024 23:59:39 GMT -->
</html>
