<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/278263-compositions-and-methods-of-detection by HTTrack Website Copier/3.x [XR&CO'2014], Thu, 04 Apr 2024 22:48:38 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 278263:&quot;COMPOSITIONS AND METHODS OF DETECTION&quot;</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">&quot;COMPOSITIONS AND METHODS OF DETECTION&quot;</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>The present invention relates generally to the field of diagnostic and detection assays. More particularly, the present invention provides methods and reagents including biochips for detecting the presence of, or distinguishing between, one or more analytes in a sample.</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td>FIELD OF THE INVENTION<br>
[0002] The present invention relates generally to the field of diagnostic and detection assays. More particularly, the present invention provides methods, and reagents including biochips for detecting the presence of, or distinguishing between, one or more analytes in a sample.<br>
BACKGROUND OF THE INVENTION<br>
[0003] Bibliographical details of references provided in the subject specification are listed<br>
at the end of the specification.<br>
[0004] Reference to any prior art in this specification is not, and should not be taken as, an<br>
acknowledgement or any form of suggestion that this prior art forms part of the common<br>
general knowledge in any country.<br>
[0005] The need for rapid and reliable screening methods for detecting multiple analytes in<br>
a single assay is vital not only in the fields of clinical diagnosis, but also for use in<br>
screening, for example, for environmental toxins and drug screening.<br>
[0006] One such area which is desperately in need of improved screening methods and<br>
reagents is in the field of infectious diseases. For example, a conservative estimate of the<br>
world use of diagnostic tests for sexually transmitted diseases, such as Human Papilloma<br>
Virus (HPV), is approximately 20,000,000 tests per year.<br>
[0007] Many of the existing tests for screening for the causes of infectious diseases are<br>
time consuming, labor intensive, expensive, often specific for only one specific pathogen<br>
and/or cannot differentiate between different strains of pathogens.<br>
[0008] HPV is the main causative pathogen for cervical cancer. However, the HPV taxon<br>
comprises many "strains" of the pathogen, only some of which are associated with the<br>
development of cervical cancer and other carcinomas. Accordingly, the strains of HPV are<br>
typically classified as either "high risk" strains, including the 13 strains which account for<br>
 <br>
roughly 98% of cervical cases, or "low risk" strains which are not typically associated with the development of cervical cancer.<br>
[0009] Currently, cervical cancer is detected by a Pap smear. In this technique, cells are collected from the cervix by scraping or washing. These ceils are then placed on a glass microscope slide to produce the "smear". A pathologist then examines the slide, looking for aberrant cells. The Pap smear, however, is a somewhat unsatisfactory assay for unequivocally determining cervical cancer risk, as the technique has a false negative rate of approximately 20% and the technique cannot distinguish "high risk" and "low risk" taxon.<br>
(0010)	Many of the reagents and/or methods for detection of HPV suffer from high<br>
associated costs and/or consume significant amounts of time to complete. More rapid<br>
and/or simplified analyses having lower overall costs and easier application are needed.<br>
The present invention is directed to these and other important ends.<br>
SUMMARY OF THE INVENTION<br>
(11)	The present invention provides assays which enable the detection of one or more analytes and/or which differentiate between members within a class of analytes. In particular, multiplexing analysis based on the properties of the analytes and of the assay components is employed to identify or distinguish between analytes.<br>
(12)	Accordingly, certain aspects of the present invention provides headsets for detecting one or more analytes and/or for differentiating between two or more members within a class of analytes, wherein the headset comprises a plurality of families or subsets of beads wherein:<br>
<br>
(a)	the beads of each subset are homogeneous with respect to size;<br>
(b)	the beads within each subset are coupled to a reactant that will specifically react with a given analyte of interest in a sample to be tested;<br>
<br>
(c)	the reactant on e3ch bead is labelled with the same label with each subset of beads having a different fluorescent intensity to create a heterogeneous mixture of subsets of beads based on fluorescent intensity; and<br>
(d)	at least two subsets of beads are mixed together to produce a headset, wherein the subset identity and therefore the reactant to which the bead has been coupled is identifiable by flow cytometry based on size, fluorescent intensity and analyte discrimination.<br>
 <br>
(0013) In other aspects, the present invention contemplates methods for detecting and/or differentiating between two or more analytes in a sample, comprising the steps of:<br>
(a)	contacting the sample with a beadset specific for the analytes of interest;<br>
(b)	incubating said beadset with said sample for a time and under conditions sufficient to allow said analyte(s) in said sample to react specifically with a reactant on a bead within said beadset; and<br>
(c)	detecting and/or differentiating analytes in the sample which are bound to a reactant on said bead.<br>
[0014] In some other aspects, the reactants may be labelled with one or more fluorochromes that further allow differentiation between members within a class of analytes. In certain preferred aspects, the present invention provides methods and beadsets which are able to detect and/or distinguish between analytes within a biological sample, wherein the analytes are specific for an infectious pathogen.<br>
(0015) Biological samples contemplated herein include blood, serum, saliva, faeces, urine, tissue fluid, semen, exudate, pus, and respiratory fluid and mucus and swabs from topical sores, cancers and lesions.<br>
[0016] The term "pathogen" refers to a microorganism or virus which infects or colonizes a sample. Exemplary padiogens include viruses, bacteria, fiingi and eukaryotic microorganisms. A virus includes a Lentivirus (e.g. AIDS virus, HIV-I, HIV-II, HTLV-IV), Retrovirus and avian flu virus. In some preferred embodiments, "pathogen" includes a microorganism or virus which infects a multicellular organism such as an animal or plant. Accordingly, in some embodiments, the analyte may be regarded as an animal or plant pathogen. However, the present invention encompasses the detection and/or differentiation of non-pathogenic entities which colonize multicellular organisms such as symbionts, endophytes, gastrointestinal colonists, and the like. The methods of the present invention are also applicable to the detection of analytes in a sample which are indicative of the presence of therapeutic agents or substances of abuse. The methods and reagents of the present invention may also be used in the detection of an analyte in a sample which is not derived from a biological sample isolated from an animal or a plant. As such, the reagents and methods of the present invention also extend to the detection of one or more analjtes and/or differentiation between analytes in environmental samples, including air, water and soil samples, including extraterrestrial soil, dust or like samples, industrial samples and the like, in addition to the biological samples listed above.<br>
 <br>
(00171 In certain embodiments, the present invention provides for diagnostic methods and reagents for HPV in human subjects and is able to detect and differentiate between different strains in order to distinguish "high risk" HPV taxon from "low risk" HPV taxon. Accordingly, in some preferred embodiments, the present invention provides beadsets which are able to distinguish between pluralities of different HPV strains. As such, in one aspect, the analytes are specific for a plurality of HPV taxons and the methods and/or reagents are specific for the detection of nucleic acid or antigens or antibodies which are specific for the plurality of HPV taxons.<br>
(0018] Even more preferred, nucleic acid primers or probes capable of binding to a strain-specific portion of an HPV genome are immobilized onto beads in each bead subset. Primers directed to conserved regions within an HPV genome flanking a strain-specific region are then used to amplify the HPV genome. Families of beads or subsets of beads, each for any one strain of HPV are then used to detect or distinguish the HPV strain. [0019] Other aspects of the present invention are directed to beadsets for detecting one or more strains of HPV and/or for differentiating between two or more strains of HPV, wherein the headset comprises a plurality of families or subsets of beads wherein:<br>
(a)	the beads of each subset are homogeneous with respect to size;<br>
(b)	the beads within each subset are coupled to a nucleic acid capture probe which is capable of binding to a HPV strain-specific region of an HPV genome or, optionally, a control nucleic acid sequence;<br>
(c)	the capture probe on each bead is labelled with the same label with each subset of beads having a different fluorescent intensity to create a heterogeneous mixture of beads based on fluorescent intensity; and<br>
(d) at least two subsets of beads are mixed together to produce a headset, wherein the subset identity and therefore the strain of HPV is identifiable by flow cytometry based on size, fluorescent intensity and sequence discrimination.<br>
[0020] Other aspects of the present invention contemplate methods for detecting and/or differentiating between two or more HPV strains in a sample, comprising the steps of:<br>
(a) contacting the sample with a headset comprising a plurality of families or subsets of beads wherein:<br>
(i)       the beads of each subset are homogeneous with respect to size;<br>
 <br>
(ii)      the beads within each subset are coupled to a nucleic add capture probe which is capable of binding to a HPV strain-specific region of an HPV genome or, optionally, a control nucleic acid sequence;<br>
(iii)     the capture probe on each bead is labelled with the same label with each subset of beads having a different fluorescent intensity to create a heterogeneous mixture of beads based on fluorescent intensity; and<br>
(iv)      at least two subsets of beads are mixed together to produce a headset, wherein the subset identity and therefore the strain of HPV is identifiable by flow cytometry based on size, intensity and sequence discrimination;<br>
(b)	incubating said beadset with said sample for a time and under conditions sufficient to allow said probes to bind to the HPV genome amplified to generate a replicon comprising a strain-specific region;<br>
(c)	detecting and/or differentiating the amplicons generated in the sample which are bound to said beads to thereby identify or distinguish between the two or more HPV strains.<br>
[0021] In some preferred aspects, the beads within the headsets are distinguishable on the basis of size, the level of fluorescent intensity, the type of fluorochrome and the reactant which is capable of reacting with a specific analyte.<br>
[0022] In other aspects, the present invention is directed to headsets for detecting a strain of HPV and/or for differentiating between two or more strains of HPV, wherein the beadset comprises a plurality of families or a plurality of subsets of beads having:<br>
(a)	at least two subsets of beads, wherein each subset of beads is physiochemically distinguishable from any other bead subset based on at least size; or<br>
(b)	at least one subset of beads having at least two families, wherein the beads in a bead subset are homogeneous in size with respect to each other, and are physiochemically distinguishable into two or more families of beads based on the level of labelling each contains, or<br>
(c)	a combination of said families of beads and said subsets of beads; and wherein:<br>
the beads within each bead family of a subset are each coupled to an optionally-labelled nucleic acid capture probe capable of binding to a HPV strain-specific region of an HPV genome or, optionally, to at least one of a control nucleic acid sequence, provided that any nucleic acid capture probe or control nucleic acid sequence is labelled with the<br>
 <br>
same label as other labelled capture probes or control nucleic acid sequence in the bead subset, and that within any single subset of beads, each family of beads thereof has a different fluorescent intensity; and<br>
each family of beads is specific for the detection of a strain of HPV different from any other bead family in the headset; wherein said headset is capable of identifying one or more specific strains of HPV through analysis of bead size, fluorescent intensity, or sequence discrimination of the at least one bead family or bead subset, using flow cytometry.<br>
[0023] The invention is also directed to part to methods for preparing a headset for differentiating between two or more strains of HPV, the method comprising selecting a plurality of families or subsets of beads based on size, fluorescent intensity, or both, comprising providing:<br>
(a)	at least two subsets of beads, wherein each subset of beads is physiochemically distinguishable from any other bead subset based on at least size; or<br>
(b)	at least one subset of beads having at least two families, wherein the beads in a bead subset are homogeneous in size with respect to each other, and are physiochemically distinguishable into two or more families of beads based on the level of labelling each contains, or<br>
(c)	a combination of said families of beads and said subsets of beads; and wherein:<br>
the beads within each bead family of a subset are each coupled to an optionally-labelled nucleic acid capture probe capable of binding to a HPV strain-specific region of an HPV genome or, optionally, to at least one of a control nucleic acid sequence, provided that any nucleic acid capture probe or control nucleic acid sequence is labelled with the same label as other labelled capture probes or control nucleic acid sequence in the bead subset, and that within any single subset of beads, each family of beads thereof has a different fluorescent intensity; and<br>
each family of beads is specific for the detection of a strain of HPV different from any other bead family in the headset; wherein said beadset is capable of identifying one or more specific strains of HPV through analysis of bead size, fluorescent intensity, or sequence discrimination of the at least one bead family or bead subset, using flow cytometry.<br>
 <br>
[0024] The methods of the present invention in relation to HPV detection may be used to distinguish to between from 2 and 16 strains of HPV including between 2, 3,4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15 and 16 strains. In view of the usefulness of a control, the present invention therefore can use from 2-17 different beads or beadsets. In another embodiment, the headset comprises at least 16 subsets of beads for HPV strains 6, 11, 16, 18, 31, 33, 35, 39, 45, 51, 52, 56, 58, 59, 66 and 68, and typically a control headset. Suitable capture probes are those listed in Table 2.<br>
[0025] It may be desirable to test for more than 16 strains of an analyte. Thus, the present invention also includes bead sets with 2-30 different beads or bead sets, including 2, 3,4, 5, 6, 7, 8, 9. 10, 11, 12, 13, 14, 15. 16, 17, 18, 19.20,21, 22. 23, 24, 25, 26, 27. 28,29, and 30 beads or beadsets.<br>
{0026] The present invention is directed in part to certain primer pairs for use in any of the numerous PCR techniques for amplification of nucleic acid sequences, wherein the forward and reverse primers optionally have heels conjugated to the 5' terminus of the primer. In certain embodiments the invention is directed to oligonucleotide amplification primer pairs for targeting oncogenic HPV, the forward primer of the pair, selected from the group consisting of:<br>
TR TTT GTT ACT GTK GTD GAT ACY A;<br>
CAR YTR TTT GTT ACT GTK GTD GAT A;<br>
CAR YTR TTT GTT ACT GTK GTD GA;<br>
AAY CAR YTR TTT GTT ACT GTK GT;<br>
TTT GTT ACT GTK GTD GAT ACY AC HCG;<br>
TTT GTT ACT GTK GTD GAT ACY AC HCG YAG;<br>
GTK GTD GAT ACY AC HCG YAG TAC and<br>
GTD GAT ACY AC HCG YAG TAC HAA; and the reverse primer of the primer pair is selected from the group consisting of:<br>
TGA AAA ATA AAY TGY AAA TCA TAT TCY TCM MCA TG;<br>
CAY ARY TGA AAA ATA AAY TGY AAA TC;<br>
TR CAY ARY TGA AAA ATA AAY TG; and<br>
TR CAY ARY TGA AAA ATA AA;<br>
wherein the forward and reverse primers are each optionally conjugated to a heel at<br>
the 5' terminus of the primer.<br>
 <br>
10027} Determination of whether binding has occurred between an analyte and a reactant<br>
present on a bead may be done using any methodology which allows the differentiation<br>
between different beads within the beadset. In certain particularly preferred embodiments,<br>
the methods of differentiating between different beads within the headsets utilize flow<br>
cytometry.<br>
[0028] The present invention further contemplates diagnostic kits for use in accordance<br>
with the reagents and methods of the present invention. In particular, the present invention<br>
extends to biochips and the miniaturization of the solid phase components of the assay to<br>
generate nanoassay reagents. In one embodiment, the bead set or part thereof or other<br>
reagents are immobilized to a solid phase such as a biochip. The biochip may also be<br>
regarded as a "biolab" on which at least part of the assay is performed and/or results<br>
recorded.<br>
[0029] A list of abbreviations used herein is provided in Table 1.<br>
Table 1-Abbreviations<br>
Abbreviation .	D|;$CriptiQn<br>
FITC	Fluorescein isothiocyanate<br>
HEX	Hexachlorofluorescein<br>
HIV	Human Immunodeficiency Virus<br>
HPV	Human Papilloma Virus<br>
JOE	7'-dimethoxyfluorescein<br>
PCR	Polymerase chain reaction<br>
__	„      ________<br>
PMT	PhotomultipUer tube<br>
QD	Quantum dot<br>
TAMRA	Carboxytetramethylrhodamine<br>
TET	~"	Tetrachlorofluorescein<br>
TMR	^	Tetramethylrhodamine<br>
VRE	Vancomycin resistant enterococci<br>
[0030] A summary of the sequences used herein are shown in Table 2.<br>
 <br>
 <br>
<br>
 <br>
<br>
 <br>
<br>
block56/4:	I CAT AT^ TAC TT^ ACT GT@ CTG TAi CAG TAC              ""<br>
block58b/4:	TA CC^ TCC TT0 GTT ACS TCA GTi C                               ~~<br>
bIock59/4:	OTA TAg ATT AGg AAT AGi AGA AGg AGT AGA<br>
block66/4:	"          ' TTA GTja AAT GT^ CTT Tr0 GCT GCl TTA ATA<br>
block68/4:	ATT TCf TAC AGg TGA TTi AGT AGg AGT AGA<br>
bIock68b/4:	TGG TAg AGT AGffl GTC TGg AGT AG® GGA                       "<br>
blockMLC/4:	TGT CI^ TGG GT^ TCT CTg TGT CTg TGT TTG<br>
♦Wherein the shaded bases correspond to the differences.<br>
HPV signal a:	/5AmMC6mT TTT CAT GKK GAR GAR TAT GA/3Phos/<br>
HPVsignal bt	/5AmMC6/TTT TTT CAT GKK GAR GAR TAT /3Phos/	"<br>
MLC signal a:	/5AmMC6/TTT TTT ACA GAC ACA GAC AAC /3Phos/<br>
"MLCsignal b:	" /5AmMC6/TTT TTT ACA GAC ACA GAC AAC A/3Phos/<br>
MLCsignal c:	/5AinMC6/TTT TTT ACA GAC ACA GAC AAC AC/3Phos/<br>
[0031] The following variables are used herein: "K" is a variable for G or T; "H" is a variable for the nucleotides T, A, or C; "M" is a variable for the nucleotides A or C; "Y" is a variable for the nucleotides T or C; "R" is a variable for the nucleotides A or G; "W" is a variable for the nucleotides A or T; "D" is a variable for the nucleotides A, T, or G; 3Phos is a 3' phosphate group; SPhos is a 5' phosphate group; and, 5AmMC6 is a 5' H2N-(CH2)6- group attached via a phosphate group.<br>
BRIEF DESCRIPTION OF THE FIGURES<br>
(0032] Figure 1 is a graphical representation showing a schematic of an example of a<br>
DNA extraction protocol used in the HPV diagnostic method.<br>
[0033] Figure 2 is a graphical representation showing a PCR protocol that can be used to<br>
amplify HPV and human DNA from a DNA sample. GP5+ and GP6+ refer to universal<br>
HPV primers which bind to conserved sequences (Y) in HPV and generate an amplicon<br>
 <br>
which comprises a region which is variable between HPV strains (Xi-ie). Primers LCJ_F<br>
and LC 1_R amplify a human genomic DN A region (Z) which serves as a control in the<br>
later hybridization steps. Primers GP6+ and LC1_R comprise a fluorescent label which is<br>
incorporated into the amplicon generated.<br>
[0034] Figure 3 is a graphical representation showing the differentiation of microspheres<br>
on the basis of size. Six clusters corresponding to microspheres comprising diameters of<br>
3.0yLm, 3.5^im, 4.1|mi, S.Ofim, 5.6jim and 6,8nm are shown.<br>
[00351 Figure 4 is a graphical representation showing the differentiation of microspheres<br>
of the same size on the basis of fluorescent label intensity. TMR relative intensities of 0%,<br>
4%, 20% and 100% could be clearly distinguished.<br>
[0036] Figure 5 is a schematic diagram showing an example of the binding agents that can<br>
be used in the array. The array comprises microspheres comprising diameters of 3.0^m,<br>
3.5|im, 4.1^m, S.O^m, 5.6|im and 6.$^m and fluorescent label signal intensities of 0%,<br>
4%, 20% and 100%. In the case of the smaller bead sizes, i.e., S.O^m, 3.5^m, and 4.1^m,<br>
TMR intensities of 0% and 100% were used; for the S.Ojun microspheres TMR intensities<br>
of 0%, 20% and 100% were used; and for the largest microspheres, the 3.6^m and 6.8|im<br>
diameter, all signal intensities were used.<br>
[0037] Figure 6 shows an example of a headset of the present invention.<br>
[0038] Figure 7 is a graphical representation showing an example of the binding agent<br>
array to which an amplicon has bound. The results show binding to the viral conserved<br>
sequence, Y, the human control sequence Z, and the viral strain-specific sequence Xie.<br>
DETAILED DESCRIPTION OF THE INVENTION<br>
[0039] The present invention provides assays and reagents including biochips which enable the detection of one or more analytes and/or to differentiate between members within a class of analytes. In particular, analytes arc identified or distinguished by methods of multiplexing analysis based on the properties of the analytes and of the assay components. The diagnostic and detection assays and reagents of the present invention have particular application in the diagnosis of pathogen infections in multicellular eukaryotic subjects. In one particular embodiment, the present invention provides a diagnostic assay for HPV in human subjects and is able to differentiate between HPV taxons in order to distinguish "high risk" HPV infections fiom "low risk" HPV infections. Furthermore, the present invention also provides methods of diagnosing or assessing the<br>
 <br>
risk of development of a disease associated with an infection by an anaiyte in a multicellular eukaryotic subject including, inter alia, cervical cancer in a human subject. [0040] It is to be understood that unless otherwise indicated, the subject invention is not limited to specific diagnostic or assay protocols, as such may vary. It is also to be understood that the terminology used herein is for the purpose of describing particular embodiments only and is not intended to be limiting.<br>
(0041] It must be noted that, as used in the subject specification, the singular forms "a", "an" and "the" include plural aspects unless the context already dictates otherwise. Thus, for example, reference to "an anaiyte " includes a single anaiyte as well as two or more analytes; a "physiochemically distinguishable substrate" includes a single substrate as well as two or more substrates; reference to "a target" includes a single target, as well as two or more targets; reference to "an amplification" includes a single amplification, as well as multiple amplification steps; reference to "the amplicon" includes a single or multiple or complex amplicons; and so forth.<br>
(0042] Accordingly, in one aspect, the present invention provides headsets which are capable of detecting, and/or differentiating between two or more analytes in a sample, said headsets comprising:<br>
(a)	the beads of each subset are homogeneous with respect to size;<br>
(b)	the beads within each subset are coupled to a reactant that will specifically react with a given anaiyte of interest in a sample to be tested;<br>
(c)	the reactant on each bead is labelled with the same label with each subset of beads having a different fluorescent intensity to create a heterogeneous mixture of subsets of beads based on fluorescent intensity; and<br>
(d)	at least two subsets of beads are mixed together to produce a headset, wherein the subset identity and therefore the reactant to which the bead has been coupled is identifiable by flow cytometry based on size, fluorescent intensity and anaiyte discrimination.<br>
[0043] In some embodiments, the reactants may be further differentially labelled to create additional subpopulations of beads based on the incorporation of different fluorochromes. [0044] In other embodiments, the present invention provides methods or headsets for the detection and/or differentiation of an anaiyte.<br>
(0045) In certain preferred aspects, the methods or headsets of the present invention are able to detect and/or differentiate pathogenic analytes.<br>
 <br>
[0046] The present invention also provides methods for detecting and/or differentiating between one or more analytes in a sample comprising the steps of:<br>
(a)	contacting the sample with a headset specific for the analytes of interest;<br>
(b)	incubating said beadset with said sample for a time and under conditions sufficient to allow said analyte(s) in said sample to react specifically with a reactant on a bead within said beadset; and<br>
(c)	detecting and/or differentiating analytes in the sample which are bound to a reactant on said bead.<br>
(0047J As used herein, the term "pathogenic analyte" refers to a microorganism or virus which putatively infects, colonizes or has otherwise contaminated the sample. Exemplary pathogen analytes include viruses, bacteria, fungi and eukaryotic microorganisms. In a preferred embodiment, "analyte" includes a microorganism or virus which infects a multicellular organism such as an animal or plant. Accordingly, in some embodiments the pathogenic analytes may be regarded as an animal or plant pathogen. However, the present invention encompasses the detection and differentiation of non-pathogenic analyte which colonize multicellular organisms such as microbial symbionts of animals (e.g., Lactobacillus spp., ruminant bacteria), microbial symbionts of insects (e.g., Streptomyces. spp., Wolbachia spp.), microbial symbionts of sponges (e.g., green algae, dinoflagellates, cyanobacteria) and the like; endophytes of plants (e.g., Mycorrhiza, Rhizobium spp., Frankia spp., Streptomyces spp.); and the like. Furthermore, the analyte may be an analyte which is not associated with a multicellular organism. Such analytes include bacteria, fiingi, viruses, protists, nematodes and the like which colonize particular environments including "natural" environments such as soil, oceans, fresh water, ice, rock, hydrothermal vents and air; health care environments including hospitals, hospital equipment, surgical equipment, health care staff garments and the like; "industrial" environments including manufacturing facilities, pharmaceutical facilities, breweries, wineries and the like; "laboratory" environments including fermenters, cultures, benches, equipment and the like. [0048] Accordingly, samples contemplated by the present invention include industrial samples such as air, water, and soil, and the like, and biological samples such as blood, serum, saliva, faeces, urine, tissue fluid, semen, exudate, pus, respiratory fluid and mucus and swabs from topical sores, cancers and lesions. In addition, a sample may be an extraterrestrial sample such as from a meteorite or on another planet. In regards to the latter, the assay of the present invention may be adapted for use on an interplanetary<br>
 <br>
remote vehicle for testing of soil or dust or ice samples or for testing core material in a planet.<br>
[0049] In some preferred embodiments, the analyte comprises a bacterium, fungus, virus and/or eukaryotic parasite which infects an animal subject. "Animal subjects" contemplated herein include any animal, preferably a mammal and more preferably a primate including a lower primate and even more preferably, a human. For convenience, an "animal" also specifically includes livestock species such as cattle, horses, sheep, pigs, goats and donkeys as well as laboratory animals including mice, rats, rabbits, guinea pigs and hamsters.<br>
[0050] Exemplary human analytes which may be detected using the reagents and methods of the present invention include viruses such Human papilloma virus (HPV), coronaviruses including the SARS virus, influenza viruses, avian flue virus, HIV including HIV-1, HIV-II or HLTV-IV, Lentiviruses in general, hepatitis viruses and the like, the pathogenic agents of sexually transmitted diseases such as Chlamydia, Gonorrhoea, Mycoplasma spp. and Syphilis; Food-bome pathogens such as Listeria spp., Salmonella spp., E. coli (particularly E. coli HO 567), Shigella spp.. Brucella spp., Staphylococcus aureus; Nosicomial pathogens such as S. aureus including Methicillin-Resistant .SI Aureus (MRSA) and enterococci including Vancomycin Resistant Enterococci (VRE); Environmentally acquired pathogens such as Legionella spp., Giardia spp., Crytospiridium spp.. Bacillus anthacis (anthrax) and the like.<br>
[0051] In other aspects, the sample in which the analyte is detected is preferably a sample derived from a multicellular subject which is putatively infected or colonized by the analyte. Therefore, in one aspect, the sample is preferably a "biological sample". Exemplary biological samples which in no way limit the present invention include tissue or cell samples such as cell scapes, biopsies and the like and body fluid samples including blood, urine, lymph, amniotic fluid, cerebrospinal fluid and the like. [0052] In certain other aspects, the methods of the present invention are also applicable to the detection of an "analyte" in a sample which is not exclusively derived from a multicellular eukaryotic organism. As such, the present invention extends to detecting, and/or differentiating between, one or more particular taxons or strains of analytes in samples such as environmental samples (including air, water and soil samples), industrial samples, laboratory samples and the like. For example, the methods of the present invention may be used to assess the prokaryotic microflora, eukaryotic microflora and or<br>
 <br>
viral load of a soil, water or air sample or a sample derived from a man-made object or surface.<br>
[0053] In some particularly preferred embodiments of the present invention, the analyte to be detected is HPV or a strain thereof and the sample is preferably a biological sample derived from a human subject. In a further preferred embodiment, the biological sample comprises one or more cells of the subject, blood or urine. Most preferably, the biological sample comprises cervical cells collected from the subject. HPV is described in detail by Gearhart e/a/. (www.emedicine.com/MED/topicl037.htm, 2004) which is reproduced in part below. HPVs produce epithelial tumors of the skin and mucous membranes. Over 100 HPV types have been detected, and the genomes of almost 70 have been sequenced completely. The current classification system, which is based on similarities in their genomic sequences, generally correlates with the 3 categories used to describe HPV clinically: anogenital and/or mucosal, nongenital cutaneous, and epidermodysplasia verruciformis (EV). A database of HPV genomic sequences and a phylogenic tree are available via the Internet at HPV Sequence Database.<br>
[0054] The mucosal HPV infections are classified further as latent (asymptomatic), subclinical, or clinical. Clinical lesions are grossly apparent, whereas latent infections are detected only by tests for viral DNA. Subclinical lesions are identified by application of 5% acetic acid and inspection under magnification. Most HPV infections are latent; clinically apparent infections usually result in warts rather than malignancies. (005S] HPV types 6 and 11 are typically labelled as low risk because infection with these types has low oncogenic potential and usually results in the formation of condylomata and low-grade precancerous lesions. HPV types 16 and 18 have emerged as the high-risk types of HPV because they are responsible for most high-grade intraepithelial lesions that may progress to carcinomas, particularly those in the anogenital and/or mucosal category. [0056] HPV infection alone does not cause malignant transformation of infected tissue. Cofactors, such as tobacco use, ultraviolet radiation, pregnancy, folate deficiency, and immune suppression have been implicated in this process. Table 3 lists a variety of diseases and the associated HPV subtypes.<br>
Table 3 - Diseases and Associated HPV Subtypes<br>
Nongenital Cutaneous Disease	HPV T^pe<br>
Common warts (verrucae vulgaris)	1, 2,4,26,27,29,41, 57, 65<br>
Plantar warts (myrmecias)	1, 2, 4, 63<br>
 <br>
 <br>
<br>
Low-grade intraepithelial neoplasia	6, 11,43<br>
Intermediate intraepithelial neoplasia	31,33, 35,42, 44, 45, 51, 52<br>
High-grade intraepithelial neoplasia	16, 18, 56, 58<br>
Carcinoma of vulva	6,11, 16, 18<br>
Carcinoma of vagina	16<br>
Carcinoma of cervix	16,18, 31<br>
Carcinoma of anus	16, 31, 32, 33<br>
Carcinoma in situ of penis (erythroplasia of Queyrat)	16<br>
Carcinoma of penis	16, IS<br>
[0057] Papillomaviruses are highly species specific and do not infect other species, even under laboratory conditions. Humans are the only known reservoir for HPV. [0058] Papillomaviruses are nonenveloped viruses of icosahedral symmetry with 72 capsomeres that surround a genome containing double-stranded circular DNA with approximately 8000 base pairs.<br>
(0059J Papillomaviruses are thought to have two modes of replication, i.e., stable replication of the episomal genome in basal cells and runaway, or vegetative, replication in more differentiated cells to generate progeny virus. Although all cells of a lesion contain the viral genome, the expression of viral genes is tightly linked to the state of cellular differentiation. Most viral genes are not activated until the infected keratinocyte leaves the basal layer. Production of virus particles can occur only in highly differentiated keratinocytes; therefore, virus production only occurs at the epithelial surface where the cells are ultimately sloughed into the environment.<br>
[0060] HPV lesions are thought to arise from the proliferation of infected basal keratinocytes. Infection typically occurs when basal ceils are exposed to infectious virus through a disturbed epithelial barrier as would occur during sexual intercourse or after minor skin abrasions. HPV infections have not been shown to be cytolytic, rather viral particles are released as a result of degeneration of desquamating cells. Furthermore, the HPV virus can survive for many months and at low temperatures without a host. [0061] Virus multiplication is generally confined to the nucleus. Consequently, infected cells usually exhibit a high degree of nuclear atypia. Koilocytosis (from the Greek koilos, meaning empty) describes a combination of perinuclear clearing (halo) with a pyknotic or<br>
 <br>
shrunken (rasinoid) nucleus and is a characteristic feature of productive papillomavirus infection.<br>
[0062] The HPV genome exists as a circular episomal DNA separate from the host cell nucleus in benign or low-risk HPV lesions, such as those typically associated with HPV types 6 and 11. The genomes of high-risk HPV types 16 and 18 are typically integrated into the host cell DNA in malignant lesions. The present invention, however, extends to any strain of HPV including but not limited to strains 6, 11. 16, 18, 31,33, 35, 39, 45, 51, 52, 56, 58, 59, 66 and 68, Integration of the viral genome into the host cell genome is considered a hallmark of malignant transformation. HPV proteins E6 and E7 of high-risk serotypes have been shown to inactivate the host's tumor suppressor proteins p53 and Rb, thereby resulting in unregulated host cell proliferation and malignant transformation. |0063] Therefore, in other aspects, the present invention provides methods for detecting, and/or differentiating between, one or more particular strains of HPV in a biological sample, said methods comprising the steps of: (i)       obtaining a biological sample which putatively comprises HPV from a human<br>
subject; (ii)      isolating nucleic acid from said sample; (iii)     amplifying the nucleic acid from said sample using primers which generate an<br>
amplicon which is distinct for each strain of HPV; (iv)     optionally amplifying a control nucleic acid sequence; (v)      effecting labelling of the amplicon(s) recited at steps (iii) and/or (iv); (vi)      hybridizing the labelled amplicon(s) to a headset coated with reactants wherein each member of the headset comprises a nucleic acid molecule having complementarity to a nucleotide sequence of a particular strain of HPV or a control nucleotide sequence, bound or otherwise associated with a physiochemically distinguishable bead; and (vii)    determining to which of the reactants an amplicon has bound; wherein the association of an amplicon with a particular reactant is indicative of the presence of a particular strain of HPV in the sample.<br>
(0064] In yet other aspects, the hybridizing occurs in the presence of at least one signal oligonucleotide sequence.<br>
[0065] In still other aspects, the hybridizing occurs in the presence of at least one blocking oligonucleotide sequence.<br>
 <br>
[0002]       The present invention enables the detection of amplified HPV DNA or, with the use of a reverse transcriptase, corresponding RNA. Hence, the present invention contemplates beads with RNA or DNA or chemical analogs thereof. [0066] The methods of the present invention are predicated, in part, on detecting and/or differentiating between one or more particular strains of a subject analyte in a sample. Reference herein to "particular strains of a subject analyte" includes any variants of the species or taxon of the analyte. Examples of "strains" of an analyte include sub-species of the analyte, variants of the analyte with differing levels of virulence, variants of the analyte which indicate different prognoses when infecting or colonizing a host, biochemical variants of the analyte and the like.<br>
[0067] In some preferred embodiments, the methods of the present invention may be adapted to detecting and/or differentiating between particular strains of HPV which are associated with higher risk or higher oncogenic potential in humans (high risk strains) and those which are associated with lower carcinoma risk or low oncogenic potential (low risk strains). Accordingly, the term "high risk" strain of HPV includes any strain of HPV which is associated with the development of carcinoma, including cervical cancer, in human subjects. As indicated above, exemplary high risk strains of HPV. which in no way limits the invention, include HPV 16. 18, 31, 33, 35, 39,45, 51, 52, 56, 58, 59. 66 and 68. Suitable capture probes on the beads for these strains of HPV are shown in Table 2. The term "probe" and "primer" may be used interchangeably in this context. "Low risk" strains of HPV include those which are not associated or only weakly associated with increased risk of carcinoma in human subjects. Typically, the low-risk strains of HPV are the wart-forming strains, including HPV 6 and HPV 11.<br>
[0068] The beads of the present invention are coupled to reactants which will specifically interact with a given analyte of interest within a sample. In some aspects, the reactants of the present invention are nucleic acids and the analytes within the sample which specifically react with the reactant are also nucleic acids.<br>
[0069] Hence, this aspect of the subject invention uses primers which are directed to conserved regions of a strain of HPV but which flank strain-specific genomic sequences. The strain-specific sequences are referred to as "variable" sequences since they vary between strains compared to the conserved sequences which are constant between strains. Upon amplification, the amplicons are put into contact with subsets of beads in the beadset wherein each bead of each subset carries a capture nucleic acid primer or probe capable of<br>
 <br>
hybridizing to the strain-specific ampiicons. Multiplexing using bead size, fluorescence<br>
intensity and DNA binding specificity enables identification, sorting and distinguishing of<br>
HPV strains.<br>
[0070J Accordingly, other aspects of the present invention contemplates headsets for<br>
detecting one or more strains of HPV and/or for differentiating between two or more<br>
strains of HPV, wherein the headset comprises a plurality of families or subsets of beads<br>
wherein:<br>
(a)	the beads of each subset are homogeneous with respect to size;<br>
(b)	the beads within each subset are coupled to a nucleic acid capture probe which is capable of binding to a HPV strain-specific region of an HPV genome or, optionally, a control nucleic acid sequence;<br>
(c)	the capture probe on each bead is labelled with the same label with each subset of beads having a different fluorescent intensity to create a heterogeneous mixture of beads based on fluorescent intensity; and<br>
(d)	at least two subsets of beads are mixed together to produce a headset, wherein the subset identity and therefore the strain of HPV is identifiable by flow cytometry based on size, fluorescent Intensity and sequence discrimination,<br>
[0071) Still other aspects of the present invention contemplate a methods for detecting and/or differentiating between two or more HPV strains in a sample, comprising the steps of:<br>
(a)      contacting the sample with a headset comprising a plurality of families or subsets of beads wherein:<br>
(i)       the beads of each subset are homogeneous with respect to size;<br>
(ii) the beads within each subset are coupled to a nucleic acid capture probe which is capable of binding to a HPV strain-specific region of an HPV genome or, optionally, a control nucleic acid sequence;<br>
(ill)     the capture probe on each bead is labelled with the same label with each subset of beads having a different fluorescent intensity to create a heterogeneous mixture of beads based on fluorescent intensity; and<br>
(iv)     at least two subsets of beads are mixed together to produce a headset, wherein the subset identity and therefore the strain of HPV is identifiable by flow cytometry based on size, fluorescent intensity and sequence discrimination;<br>
 <br>
(b)	incubating said beadset with said sample for a time and under conditions sufficient to allow said primers to bind to the HPV genome amplified to generate a replicon comprising a strain-specific region;<br>
(c)	detecting and/or differentiating the amplicons generated in the sample which are bound to said beads to thereby identify or distinguish between the two or more HPV strains.<br>
[0072] The amplicons to which the nucleic acid capture probes are specific may also be labelled with a fluorescent reporter molecule.<br>
(0073) Nucleic acids may be isolated from the subject sample using any method which is convenient with regard to the nature of the sample itself and the analyte. As used herein, the term "nucleic acid" refers to DNA and/or RNA. Typically, DNA is isolated, although under some circumstances which would be evident to one of skill in the art, it may be more preferable to isolate RNA, for example, when the analyte of interest is an RNA virus. If RNA is isolated, the RNA may be amplified, or the RNA may be reverse transcribed into cDNA using standard methods, for subsequent amplification and analysis. [0074] Preferably, the "nucleic acid" is DNA. DNA may be isolated from the sample using any convenient means. For example, in the case of a virus such as HPV in a human ceil sample, guanidine or a functionally equivalent agent may be used to lyse the cells. An exemplary guanidine-based method for cell lysis and DNA extraction is the method of Nelson and Krawetz (Anal. Biochem. 207fJ):97-20\, 1992). Guanidine-based lysis solutions are also commercially available from suppliers such as Qiagen, e.g., QlAamp, PAXgene, and the like. However, methods of lysis may change depending on the nature of the sample and analyte. For example, for the detection of an analyte in an environmental sample such as a soil or sediment sample, a glass-bead based cell lysis system may be more appropriate, such as the method of Kuske et al. {Appi Environ. Microbiol. 64(7):246'i-2A12, 1998). In any event, the appropriate lysis protocol for a given analyte and sample would be readily determined by one of ordinary skill in the art with no undue experimentation.<br>
(0075) After lysis of the cells, the DNA may be purified by any convenient means which would be readily evident to one of skill in the art (e.g., see commercially available kits hereinabove). In certain preferred embodiments of the invention, the DNA is purified using a limiting amount of a DNA binding agent such as, but not limited to, silica. By using a limiting amount of the DNA binding agent, a uniform amount of DNA may be<br>
 <br>
isolated from different samples as the amount of DNA recovered in each case is equal to the maximum amount of DNA that can be bound by the limiting amount of DNA binding agent. The DNA bound to the DNA binding agent may then be recovered or eluted from the DNA binding agent using any convenient means.<br>
[0076] Although DNA is a preferred nucleic acid, RNA may also be isolated from the sample using any standard RNA isolation protocol. RNA isolation typically involves a cell disruption step and an RNA isolation step. Exemplary cell disruption techniques which are suitable for the isolation of RNA include those presented in Ambion Technical Bulletin #183 (www.ambion.com/tCchlib/tb/tb 183.html). Furthermore, a range of exemplary RNA isolation kits which are suitable for a range of sample types are presented at www.ambion.com/techlib/basics/products/maisol compchart.html. However, it should be understood that the present invention is not in any way limited by these specific methods and kits for RNA isolation and purification and the present invention is compatible with any RNA isolation methods which would be evident to one of skill in the art. (0077] The beadset may comprise, in relation to HPV detection as many subsets of beads as strains of HPV. Hence, the assay may employ 2, 3,4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15 or 16 subsets of beads each for HPV strains 6,11,16,18, 31, 33, 35, 39, 45, 51, 52, 56, 58, 59,66 and 68. Additional beads may also be used for controls. Suitable capture probes are disclosed in Table 2.<br>
{0078] Hence, other aspects of the present invention are directed to headsets for detecting one or more strains of HPV and/or for differentiating between two or more strains of HPV, wherein the beadset comprises a plurality of families or subsets of beads wherein:<br>
(a)	the beads of each subset are homogeneous with respect to size;<br>
(b)	the beads within each subset are coupled to a nucleic acid capture probe selected from the list in Table 2 that is capable of binding to a HPV strain-specific region of an HPV genome or, optionally, a control nucleic acid sequence;<br>
(c)	the capture probe on each bead is labelled with the same label with each subset of beads having a different fluorescent intensity to create a heterogeneous mixture of beads based on fluorescent intensity; and<br>
(d)	at least two subsets of beads are mixed together to produce a beadset, wherein the subset identity and therefore the strain of HPV is identifiable by flow cytometry based on size, fluorescent intensity and sequence discrimination.<br>
 <br>
[0079] Still a further aspect of the present invention contemplates methods for detecting and/or differentiating between two or more HPV strains in a sample, comprising the steps of-<br>
(a)	contacting the sample with a headset comprising a piuraiity of families or<br>
subsets of beads wherein:<br>
(i)       the beads of each subset are homogeneous with respect to size;<br>
(ii)      the beads within each subset are coupled to a nucleic acid capture probe is selected from the list in Table 2 that is capable of binding to a HPV strain-specific region of an HPV genome or, optionally, a control nucleic acid sequence;<br>
(iii)     the capture probe on each bead is labelled with the same label with each subset of beads having a different fluorescent intensity to create a heterogeneous mixture of beads based on fluorescent intensity; and<br>
(iv)     at least two subsets of beads are mixed together to produce a headset, wherein the subset identity and therefore the strain of HPV is identifiable by flow cytometry based on size, fluorescent intensity and sequence discrimination;<br>
(b)	incubating said headset with said sample for a time and under conditions sufficient to allow said probes to bind to the HPV genome amplified to generate a replicon comprising a strain-specific region; and<br>
(c)	detecting and/or differentiating the amplicons generated in the sample which are bound to said beads to thereby identify or distinguish between the two or more HPV strains.<br>
I0080J Accordingly, the headsets may comprise 2, 3, 4, 5, 6, 7, 8. 10, 11, 12, 13, 14, 15, 16 subsets of beads each with one of a nucleic acid molecule selected from the listing in Table 2. Reference to these HPV strain-specific sequences in Table 2 includes nucleic acid molecules having at least 90% identity to these sequences or capable of hybridizing thereto or their complementary forms under low stringency conditions. Reference to at least 90% includes 90, 91, 92. 93.94. 95, 96, 97,98, 99 and 100%.<br>
[0081} The methods of the present invention rely, in part, on amplifying a nucleic acid from a sample using primers that bind to conserved sequences among different strains of the subject analyte, but which generate an amplicon which comprises a distinct nucleotide sequence for each strain of the analyte. In effect, the primers used in the present invention bind to sequences which are conserved among strains of the subject analyte which flank<br>
 <br>
regions that are at least partially non-conserved or polymorphic between strains. Schematically, the amplified region in the analyte has the general structure of:<br>
wherein:<br>
C is a nucleotide sequence which is conserved among two or more strains of the analyte and is the binding site of the "forward" primer;<br>
X is a nucleotide sequence, part or all of which comprises variation between different strains of the analyte;<br>
C is a nucleotide sequence which is conserved among two or more strains of the analyte and is the binding site of the "reverse" primer.<br>
[0082] In certain embodiments it is desirable to incorporate heels into one or more of the primers to modify the nucleic acid amplification process. Preferably, these heels are conjugated to the S' terminus of a primer. More preferably, one ore more heels are optionally further conjugated to a S Phos or SAmMC6 moiety at the heel's S' terminus. In certain preferred embodiments, the heels conjugated to forward primers are selected from: caatcagc. acaat, ggaac. cagctt. attacc. and ctgtt. more preferably acaat and ctgtt. In other preferred embodiments, the heels conjugated to reverse primers are selected from: actcactatagg. aatacgactcactatagg. tctaatacgactcactataee. and aattctaatacgactcactatapg more preferably actcactatagg and tctaatacgactcactatagg.<br>
[0083] In certain embodiments related to oligonucleotide amplification primer pairs for targeting oncogenic HPV, the forward primer of a primer pair is selected fh)m the group consisting of:<br>
/5Phos/caatcagc TR TTT GTT ACT GTK GTD GAT ACY A;<br>
/SPhos/acaat CAR YTR TTT GTT ACT GTK GTD GAT A;<br>
/5Phos/acaat CAR YTR TTT GTT ACT GTK GTD GA;<br>
/5Phos/ggaac AAY CAR YTR TTT GTT ACT GTK GT;<br>
/SPhos/caecn TTT GTT ACT GTK GTD GAT ACY AC HCG;<br>
/5Phos/caectt TTT GTT ACT GTK GTD GAT ACY AC HCG YAG;<br>
/SPhos/attacc GTK GTD GAT ACY AC HCG YAG TAC; and<br>
/5Phos/ctgtt GTD GAT ACY AC HCG YAG TAC HAA; and the reverse primer of a primer pair is selected from the group consisting of:<br>
/SAmMC6/actcactatagg TGA AAA ATA AAY TGY AAA TCA TAT TCY TCM MCA TG;<br>
 <br>
/5AmMC6/aatacpactcactatagg CAY ARY TGA AAA ATA AAY TOY AAA TC;<br>
/SAmMC6/tctaatacgactcactatagg TR CAY ARY TGA AAA ATA AAY TG; and<br>
/SAmMC6/aattctaatacgactcactataeg TR CAY ARY TGA AAA ATA AA. As shown, a primer is indicated in capital letters and an optional heel in each instance is shown in underlined lower case letters. Preferably, the forward primer of a primer pair is selected from: /SPhos/acaat CAR YTR TTT GTT ACT GTK GTD GA and /5Phos/ctgtt GTD GAT ACY AC HCG YAG TAC HAA. In other preferable embodiments, the reverse primer of a primer pair is selected from /5AmMC6/actcactatagg TGA AAA ATA AAY TGY AAA TCA TAT TCY TCM MCA TG and /5AmMC6/tctaatacgactcactatagg TR CAY ARY TGA AAA ATA AAY TG.<br>
{0084] In certain aspects of the invention, the primers incorporate a non-primer binding region 5' extraneous nucleotide sequence conjugated to the S' end of a 3' template binding primer region. The S' extraneous nucleotide sequence is conveniently referred to herein as a "heel," "heel clamp," "heel sequence," or "extraneous heel sequence," but this is not in any way intended to be limiting on the subject invention.<br>
[0085] In certain preferred embodiments, at least one of the forward and reverse primers is conjugated to a heel at the 5' terminus of the primer. Preferably, when the forward primer is conjugated to a heel at the 5' terminus of the forward primer, the forward primer heel is selected from the group consisting of:<br>
CAATCAGC, ACAAT, GGAACAAT, GGAAC, CAGCTT. ATTACC, CTGTT, /5Phos/CAATCAGC, /5Phos/ACAAT, /SPhos/GGAACAAT, /5Phos/GGAAC, /5Phos/CAGCTT, SPhos/ATTACC. and /5Phos/CTGTT. Preferably when the reverse primer is conjugated to a heel at the 5' terminus of the reverse primer, the reverse primer heel is selected from the group consisting of:<br>
ACTCACTATAGG, AATACGACTCACTATAGG, TCTAATACGACTCACTATAGG, AATTCTAATACGACTCACTATAGG, /5AmMC6/ACTCACTATAGG,/5AmMC6/AATACGACTCACTATAGG, /5AmMC6/TCTAATACGACTCACTATAGG, and<br>
/5AmMC6/AATTCTAATACGACTCACTATAGG. In some more preferred embodiments of oligonucleotide amplification primer pairs containing an optional heel nucleic acid sequence, the forward primer of the primer pair is selected from the group consisting of:<br>
 <br>
CAR YTR TTT GTT ACT GTK GTD GA, optionally having a ACAAT. /5Phos/ACAAT, GGAACAAT, or a /5Phos/GGAACAAT conjugated to the 5' terminus of the primer; and<br>
GTD GAT ACY AC HCG YAG TAC HAA, optionally having a CTGTT or /5Phos/CTGTT conjugated to the 5' terminus of the primer. In some more preferred embodiments of oligonucleotide amplification primer pairs containing an optional heel nucleic acid sequence, the reverse primer of the primer pair is selected from the group consisting of:<br>
TGA AAA ATA AAY TGY AAA TCA TAT TCY TCM MCA TG, optionally having a ACTCACTATAGG or /5AmMC6/ACTCACTATAGG conjugated to the 5' terminus of the primer, TR CAY ARY TGA AAA ATA AAY TG, optionally having a TCTAATACGACTCACTATAGG or /5AmMC6/rCTAATACGACTCACTATAGG conjugated to the 5' terminus of the primer; and TR CAY ARY TGA AAA ATA AAY TG, optionally having a TCTAATACGACTCACTATAGG or<br>
/5AmMC6/TCTAATACGACTCACTATAGG conjugated to the 5" terminus of the primer. [0086] The 3' template binding primer region refers to a primer whose 3' portion binds to a template (e.g., during the first annealing step to prime polymerization). The above-mentioned primer design results in the incorporation of the heel sequence in amplification products after the initial priming event. Subsequently, the S' heel sequence acts as a clamp to even the amplification efficiency across amplicon homologs. [0087] The extraneous heel sequence may be on either the forward primer or the reverse primer or both.<br>
[0088] Hence, certain aspects of the present invention contemplate methods for amplifying a nucleic acid target molecule said methods comprising subjecting a nucleic acid template of said nucleic acid target to amplification using forward and reverse primers wherein at least one primer contains a non-primer binding region S' extraneous nucleotide sequence conjugated to a 3' template binding primer region wherein said extraneous nucleotide sequence is incorporated into an amplification product after initial priming. [0089] As indicated above either the forward or reverse primer may comprise the extraneous sequence or both may carry the sequence. When both cany the extraneous sequence, the heel may be the same or different.<br>
[0090] The present invention is particularly applicable to primers that are potentially capable of priming a set of nucleic acid target homologs.<br>
 <br>
10091] The present invention further contemplates improved methods of amplifying a nucleic acid molecule included within a population of related nucleic acid molecules by amplification with a forward and reverse primer, comprising: selecting one or both of the forward or reverse primers such that one or both contain a non-primer binding region 5' extraneous nucleotide sequence conjugated to a 3' template binding primer region wherein said extraneous nucleotide sequence is incorporated into the amplification production after initial priming.<br>
(0092] Hence, reference to a "nucleic acid molecule" includes a family of related nucleic acid molecules such as a group of homolog nucleic acid molecules. [0093] Kits for reducing amplification bias also form part of the present invention, {0094] All scientific citations, patents, patent applications and manufacturer's technical specifications cited or described in this document are incorporated herein by reference in their entirety.<br>
[0095] It is understood that unless otherwise indicated, the subject invention is not limited to specific reagents, process steps, or applications or the like, as such may vary. It is also to be understood that the terminology used herein is for the purpose of describing particular embodiments only and is not intended to be limiting. [0096] Terms and symbols of nucleic acid chemistry, biochemistry, genetics, and molecular biology used herein follow those of standard treatises and texts in the field, e.g., Kornberg and Baker, DNA Replication, Second Edition (W.H. Freeman, New York, 1992); Lehninger, Biochemistry, Second Edition (Worth Publishers, New York, 1975); Strachan and Read, Human Molecular Genetics, Second Edition (Wiley-Liss, New York, 1999); Eckstein Ed, Oligonucleotides and Analogs: A Practical Approach (Oxford University Press, New York, 1991); Gait Ed, Oligonucleotide Synthesis: A Practical Approach (IRL Press, Oxford, 1984); and the like.<br>
{0097] "Amplicon" means the product of a polynucleotide amplification reaction. That is, it is a population of polynucleotides, usually but not necessarily double stranded, that are replicated from one or more starting sequences. The one or more starting sequences may be one or more copies of the same sequence, or it may be a mixture of different sequences. Amplicons may be produced by a variety of amplification reactions whose products are multiple replicates of one or more target nucleic acids. Generally, amplification reactions producing amplicons are "template-driven" in that base pairing of reactants, either nucleotides or oligonucleotides, have complements in a template polynucleotide that are<br>
 <br>
required for the creation of reaction products. In one aspect, template-driven reactions are primer extensions with a nucleic acid polymerase or oligonucleotide ligations with a nucleic acid ligase. Such reactions include, but are not limited to PCR linear polymerase reactions, NASBAs, rolling circle amplifications, and the like, disclosed in the following references that are incorporated herein by reference: Mullis et al, U.S. Patent Nos. 4,683,195; 4,965,188; 4,683,202; 4,800,159 (PCR); Gelfand etal. U.S. Patent No. 5,210,015 (real-time PCR with "Taqman" or "Taq" [registered trade marks] probes); Wittwer et al, U.S. Patent No. 6,174,670; Kacian et al. U.S. Patent No. 5,399,491 ("NASBA"); Lizardi, U.S. Patent 5,854,033; Aono et al, Japanese Patent Publ. No. JP 4-262799 (rolling circle amplification); and the like.<br>
[0098] An amplification reaction may be a "real-time" amplification where detection chemistry permits a reaction product to be measured as the amplification reaction progresses.<br>
(0099] The present invention is particularly concerned with reducing amplification bias across a range of related or homolog nucleic acid molecules. Examples of homolog nucleic acid molecules include viral homologs, polymorphisms, related cancer cells, bacterial homologs, stem cell homologs amongst many others. One aspect of the present invention is the selection of primers at potential points of difference between the homologs. The primers are proposed to carry a non-primer binding region 5' extraneous nucleotide sequence conjugated to a 3' template binding primer region which is incorporated into an amplification product after initial priming.<br>
[00100]	The heel or S' extraneous sequence may be fi-om about I to about 400 bases<br>
in length and all combinations and subcombinations thereof Preferably the heel is from about 3 to about 100, more preferably from about 4 to about 80 bases in length, still more preferably from about 5 to about 40, yet more preferably about 5 to about 25, with about 10 to about 20 even more preferred.<br>
[00101]	In effect, the 5' extraneous sequence optionally has a level of<br>
complementarity to the target sequence, and it does not contribute substantially to the initial primer binding steps. It is desirable for the heel to have less than 95% complementarity to the core primer. Additional examples of the levels of complementarity can include less than 90%, 85%, 80%, 75%, 70%, 65%, and 60%. Complementarity is determined using standard algorithms.<br>
 <br>
[00102]	As used herein, the term "amplifying" means performing an amplification<br>
reaction. A "reaction mixture" or "reaction vessel" means a solution or compartment containing all the necessary reactants for performing a reaction, which may include, but not be limited to, buffering agents to maintain pH at a selected level during a reaction, salts, co-factors, scavengers, and the like.<br>
[00103]	"Complementary or substantially complementary" refers to the<br>
hybridization or base pairing or the formation of a duplex between nucleotides or nucleic acids, such as, for instance, between the two strands of a dsDNA molecule or between an oligonucleotide primer and a primer binding site on a single stranded nucleic acid. Complementary nucleotides are, generally, A and T (or A and U), or C and G. Two single stranded RNA or DNA molecules are said to be substantially complementary when the nucleotides of one strand, optimally aligned and compared and with appropriate nucleotide insertions or deletions, pair with at least about 80% of the nucleotides of the other strand, usually at least about 90% to 95%, and more preferably from about 98 to 100%. Alternatively, substantial complementarity exists when, for example, a DNA strand hybridizes under selective hybridization conditions to its complement. Typically, selective hybridization occurs when there is at least about 65% complementary over a stretch of at least 14 to 23 nucleotides, preferably at least about 75%, more preferably at least about 90% complementary. Sec, Kmichisa Nucleic Acids Res. 12:203, 1984, incorporated herein by reference in its entirety.<br>
[00104]	"Duplex" means at least two oligonucleotides and/or polynucleotides that<br>
are fully or partially complementary undergo Watson-Crick type base pairing among all or most of their nucleotides so that a stable complex is formed. The terms "annealing" and "hybridization" are used interchangeably to mean the formation of a stable duplex. "Perfectly matched" in reference to a duplex means that the poly- or oligonucleotide strands making up the duplex form a double stranded structure with one another such that every nucleotide in each strand undergoes Watson-Crick base pairing with a nucleotide in the other strand.<br>
[OOlOS]	"Genetic locus," or "locus" in reference to a genome or target<br>
polynucleotide, means a contiguous subregion or segment of the genome or target polynucleotide. As used herein, genetic locus, or locus, may refer to the position of a gene or portion of a gene in a genome, or it may refer to any contiguous portion of genomic sequence whether or not it is within, or associated with, a gene. Preferably, a genetic locus<br>
 <br>
refers to any portion of genomic sequence from a few tens of nucleotides, e.g., 10-30, or<br>
10-100, in length, to a few hundred nucleotides, e.g., 100-1000 or 100- 500 in length, to a<br>
few thousands of nucleotide in length, e.g., 1000-10,000 or 1000-3000 in length. In some<br>
contexts, genetic loci may refer to the location of a nucleotide within a genome.<br>
{001Q6]	"Kit" refers to any delivery system for delivering materials or reagents for<br>
carrying out a method of the instant invention. In the context of reaction assays, such delivery systems include systems that allow for the storage, transport, or delivery of reaction reagents (e.g., probes, enzymes, etc., in the appropriate containers) and/or supporting materials (e.g., buffers, written instructions for performing the assay etc.) from one location to another. For example, kits include one or more enclosures (e.g., boxes) containing the relevant reaction reagents and/or supporting materials. Such contents may be delivered to the intended recipient together or separately. For example, a first container may contain an enzyme for use in an assay, while a second container contains probes. The kits may also contain compartments adapted to contain the reagents. In one example, a compartment comprises a solid matrix having oligonucleotides or primers or polynucleotides immobilized thereon which participate in the amplification reaction. An example of a solid matrix is a microarray.  A kit, therefore, may be part of an overall amplification system having a reagent component, a nucleic acid component, a hardware component and an instructional component.<br>
(00107]	"Microarray" refers to a solid phase support having a planar surface, which<br>
carries an array of nucleic acids, each member of the array comprising identical copies of an oligonucleotide or polynucleotide immobilized to a spatially deflned region or site, which does not overlap with those of other members of the array; that is, the regions or sites are spatially discrete. Spatially defined hybridization sites may additionally be "addressable" in that its location and the identity of its immobilized oligonucleotide are known or predetermined, for example, prior to its use. TypicaUy, the oligonucleotides or polynucleotides are single stranded and are covalently attached to the solid phase support, usually by a 5'-end or a 3'-end. The density of non-overlapping regions containing nucleic acids in a microarray is typically greater than 100 per cm^, and more preferably, greater than 1000 per cm^. Microarray technology is disclosed in the following references that are incorporated by reference in its entirety: Schena, Ed, Microanays: A Practical Approach (IRL Press, Oxford, 2000); Southern, Current Opin. Chem. Biol. 2:404-410, 1998.<br>
 <br>
(00108]	A "random microarray" refers to a microarray whose spatially discrete<br>
regions of oligonucleotides or polynucleotides are not spatially addressed. That is, the identity of the attached oligonucleotides or polynucleotides is not discemable, at least initially, from its location. In one aspect, random microarrays are planar arrays of microbeads wherein each microbead has attached a single kind of hybridization tag complement, such as from a minimally cross-hybridizing set of oligonucleotides. Likewise, after formation, microbeads, or oligonucleotides thereof, in a random array may be identified in a variety of ways, including by optical labels, e.g., fluorescent dye ratios or quantum dots, shape, sequence analysis, or the like.<br>
[00109]	"Nucleoside" as used herein includes the natural nucleosides, including 2'-<br>
deoxy and T- hydroxyl forms, e.g., as described in Komberg and Baker, DNA Replication, 2nd Ed. (Freeman, San Francisco, 1992).<br>
(00110]	"Polymerase chain reaction," or "PCR," means a reaction for the in vitro<br>
amplification of specific DNA sequences by the simultaneous primer extension of complementary strands of DNA. In other words, PCR is a reaction for making multiple copies or replicates of a target nucleic acid flanked by primer binding sites, such reaction comprising one or more repetitions of the following steps: (i) denaturing the target nucleic acid, (ii) annealing primers to the primer binding sites, and (iii) extending the primers by a nucleic acid polymerase in the presence of nucleoside triphosphates. Usually, the reaction is cycled through difTerent temperatures optimized for each step in a thermal cycler instrument. Particular temperatures, durations at each step, and rates of change between steps depend on many factors well-known to those of ordinary skill in the art, e.g., exemplified by the references: McPherson et al, (Eds), PCR: A Practical Approach and PCR2: A Practical Approach (IRL Press, Oxford, 1991 and 1995, respectively). For example, in a conventional PCR using Taq DNA polymerase, a double stranded target nucleic acid may be denatured at a temperature &gt;90°C, primers annealed at a temperature in the range 35-90'*C. The term "PCR" encompasses derivative forms of the reaction, including but not limited to, RT-PCR, real-time PCR, nested PCR, quantitative PCR, multiplexed PCR, enhanced solid phase PCR, and the like.  For further detail of enhanced solid phase PCR, see, for example. Park, et al.. Analytical Biochemistry, 375 (2008), pp. 391-393, the entirety of which is incorporated herein by reference in its entirety. The use of enhanced solid phase PCR is preferred in certain embodiments of the present invention for the generation and/or amplification of nucleic acid molecules related to, for example<br>
 <br>
HPV strains. Still more preferred is the use of one or more forward/reverse primer pairs noted herein in enhanced solid phase PCR amplification of nucleic acid molecules related to HPV strains.<br>
(00111]	Using the ESP-PCR format, 5-prime phosphates are optional since single<br>
stranded amplicon is not required as a final PCR product. The 5-prime amine functional groups are also optional in this format and may be used as a preferred means of labelling with a fluorophore for signal generation. However, many alternative means of signal generation may also be employed where there is no requirement for any functional group or an alternative functional group. For example. PCR primers may be non-fluor labelled, and separate 'signal oligonucleotides', which are conjugated with a signal, may be employed. In another example, using ESP-PCR format, all 'aqueous' PCR primers may be used without functional groups and the signal could alternatively be introduced by means of labelled nucleotide analogue incorporation. In other embodiments using ESP-PCR format, all 'aqiieous' PCR primers may be used without functional groups, and the signal could be introduced by means of labelled solid support primer such that signal increases in the context of double stranded product.<br>
{00112]	Alternative means for the generation of single stranded amplicon include<br>
conversion of double stranded DNA to single stranded DNA by separating the strands or by removing one strand of the duplex. Strands of the duplex can be separated by thermal or chemical means of disrupting interstrand bonds. Removal of one strand permits recovery of the desired strand and elimination of its complement e.g., NiJciforov et al. (US Patent No. 5,518,900), who described modifying one of two primers used for amplification by incorporation of phosphorothiate nucleotide derivatives in the S' end of the modified primer, rendering it resistant to exonuclease digestion. After amplifying target sequences using the polymerase chain reaction (PCR), the dsDNA is subjected to exonuclease digestion. The unprotected strand is preferentially digested by a 5' to 3' exonuclease, leaving a single-stranded product consisting of the other strand. Similar strategies have used exonuclease-resistant branched primers (Shchepinov et al., Nuc. Acids. Res. 2i:4447-4454 1997) or 5' phosphate-bearing substrate preference of Lambda exonuclease (Higuchi et al.. NucL Acids Res. 25:5685, 1989)<br>
{00113]	Asymmetric PCR (Gyllensten and Erlich, Proc. Natl. Acad. ScL USA<br>
55:7652-7656 1998; US Patent No. 5,066,584) generates ssDNA during thermocycling by<br>
 <br>
employing an imbalanced primer pair concentration such that one primer is at a limiting<br>
concentration. This favours ssDNA product primed by the primer in excess.<br>
[00114]	Competitor primer asymmetric PCR (Gillespie. 1997; US Patent Appl. No.<br>
08/628,417) employs the separate addition of competitor primer following PCR<br>
thermocycling and prior to further thermocycling to generate ssDNA. Kaltenboeck et ai,<br>
Biotechniques /2:164-171, 1992 described a method of producing ssDNA by initially<br>
performing a PCR to generate dsDNA, followed by a separate reaction using the product of<br>
the first PCR as a template for a second linear amplification employing one primer. See<br>
also US Patent 6,887,664 for examples of Asynchronous PCR.<br>
[00115]	Reaction volumes range from a few hundred nanoliters, e.g., 200 nL, to a<br>
few hundred ^L, e.g., 200 HL. "Reverse transcription PCR," or "RT-PCR," means a PCR that is preceded by a reverse transcription reaction that converts a target RNA to a complementary single stranded ONA, which is then amplified, e.g., Tecott et al, U.S. Patent No. 5,168,038, which patent is incorporated herein by reference in its entirety. "Real-time PCR" means a PCR for which the amount of reaction product, i.e., amplicon, is monitored as the reaction proceeds. There are many forms of real-time PCR that differ mainly in the detection chemistries used for monitoring the reaction product, e.g., Gelfand et al, U.S. Patent No. 5,210,015 ("taqman"); Wittwer et al, U.S. Patent Nos. 6,174,670 and 6,569,627 (intercalating dyes); Tyagi et al, U.S. Patent No. 5,925,517 (molecular beacons); which patents are incorporated herein by reference in its entirety. Detection chemistries for real-time PCR are reviewed in Mackay etal. Nucleic Adds Research, iO: 1292-1305, 2002, which is also incorporated herein by reference in its entirety. "Nested PCR" means a two-stage PCR wherein the amplicon of a first PCR becomes the sample for a second PCR using a new set of primers, at least one of which binds to an interior location of the first amplicon.<br>
[00116]	It is proposed that the initial priming events be made with forward and<br>
reverse primers where one or both carry a non-primer binding region 5' extraneous nucleotide sequence conjugated to a 3' template binding primer region wherein the extraneous nucleotide sequence is incorporated into an amplification product af^er initial priming.<br>
[00117J	The use of non-primer binding region 5' heel sequence conjugated to a 3'<br>
template binding primer region as the primer design results in the incorporation of the heel sequence in amplification products after the initial primer binding event. Subsequently,<br>
 <br>
the 3' hee) sequence acts as a clamp to even the amplification efficiency across homologs.<br>
As such, amplification bias is restricted to initial primer binding events. The "heel clamp"<br>
can be placed on either or both forward and reverse primers depending on requirement.<br>
There is great freedom of design regarding what sequence the "heel clamp" sequence was<br>
comprised to suit the particular application. The 3' template binding primer sequence<br>
could take the form of degenerate sequence, consensus sequence, or a hybrid of consensus<br>
and degenerate sequences. A pool of exact matching sequences of interest could also be<br>
employed. In all these cases the 5' "heel clamp" principle can be employed.<br>
[00118]	"Polynucleotide" and "oligonucleotide" are used interchangeably and each<br>
mean a linear polymer of nucleotide monomers. Monomers making up polynucleotides and oligonucleotides are capable of specifically binding to a natural polynucleotide by way of a regular pattern of monomer- to-monomer interactions, such as Watson-Crick type of base pairing, base stacking, Hoogsteen or reverse Hoogsteen types of base pairing, or the like.<br>
(00119]	Whenever a polynucleotide or oligonucleotide is represented by a sequence<br>
of letters (upper or lower case), such as "ATGCCTG," it will be understood that the nucleotides are in 5' — &gt;3' order fi-om left to right and that "A" denotes deoxyadenosine, "C" denotes deoxycytidine, "G" denotes deoxyguanosine, and "T" denotes thymidine, "I" denotes deoxyinosine, "U" denotes uridine, unless otherwise indicated or obvious from context.<br>
[00120]	"Primer" means an oligonucleotide, either natural or synthetic that is<br>
capable, upon forming a duplex with a polynucleotide template, of acting as a point of initiation of nucleic acid synthesis and being extended from its 3' end along the template so that an extended duplex is formed.<br>
[00121]	Extension of a primer is usually carried out with a nucleic acid polymerase,<br>
such as a DNA or RNA polymerase. The sequence of nucleotides added in the extension process is determined by the sequence of the template polynucleotide. Usually primers arc extended by a DNA polymerase. Primers usually have a length in the range of from 14 to 40 nucleotides, or in the range of from 18 to 36 nucleotides. Primers are employed in a variety of nucleic amplification reactions, for example, linear amplification reactions using a single primer, or PCRs, employing two or more primers. Guidance for selecting the lengths and sequences of primers for particular applications is well known to those of ordinary skill in the art, as evidenced by the following references that are incorporated by<br>
 <br>
reference in its entirety: Dieffcnbach (Ed), PCR Primer: A Laboratory Manual. T^ Edition (CoJd Spring Harbor Press, New York, 2003).<br>
(00122]	"Sample" means a quantity of material from a biological, environmental,<br>
medical, or patient source in which detection or measurement of target nucleic acids is sought. On the one hand it is meant to include a specimen or culture (e.g., microbiological cultures). On the other hand, it is meant to include both biological and environmental samples. A sample may include a specimen of synthetic origin. Biological samples may be animal, including human, fluid, solid (e.g., stool) or tissue, as welt as liquid and solid food and feed products and ingredients such as dairy items, vegetables, meat and meat by¬products, and waste. Biological samples may include materials taken from a patient including, but not limited to cultures, blood, saliva, cerebral spinal fluid, pleural fluid, milk, lymph, sputum, semen, needle aspirates, and the like. Biological samples may be obtained from ail of the various families of domestic animals, as well as feral or wild animals, including, but not limited to, such animals as ungulates, bear, fish, rodents, etc.. Environmental samples include environmental material such as surface matter, soil, water and industrial samples, as well as samples obtained from food and dairy processing instruments, apparatus, equipment, utensils, disposable and non-disposable items. These examples are not to be construed as limiting the sample types applicable to the present invention.<br>
[00123]	Hence, some aspects of the present invention provide methods for<br>
amplifying a nucleic acid target molecule, comprising: subjecting a single stranded<br>
template of said nucleic acid target to amplification using forward and reverse primers<br>
wherein at least one primer contains a non-primer binding region S' extraneous nucleotide<br>
sequence conjugated to a 3' template binding primer region wherein said extraneous<br>
nucleotide sequence is incorporated into an amplification product after initial priming.<br>
[00124]	Other aspects of the present invention contemplate improved methods of<br>
amplifying a nucleic acid molecule including within a population of related nucleic acid<br>
molecules by amplification with a forward and reverse primer, comprising-, selecting one<br>
or both of the forward or reverse primers such that one or both contain a non-primer<br>
binding region 5' extraneous nucleotide sequence conjugated to a 3' template binding<br>
primer region wherein said extraneous nucleotide sequence is incorporated into the<br>
amplification production after initial priming.<br>
[00125]	Examples of heel sequences include:<br>
 <br>
T7a promoter sequence 25-mer:<br>
AAT TCT AAT ACG ACT CAC TAT AGG G;<br>
M13/pUC Sequencing Primer (-40) GTT TTC CCA GTC ACG AC;<br>
Ml3/pUC sequencing primer (-20), 17-mer GTA AAA CGA CGG CCA GT;<br>
M13/pUC reverse sequencing primer (-26), 17-mer CAG GAA ACA GCT ATG AC;<br>
M13/pUC sequencing primer (-40), 17-mer GTT TTC CCA GTC ACG AC;<br>
M13/pLIC sequencing primer (-46), 22-mer GCC AGG GTT TTC CCA GTC ACG A;<br>
M13/pUC reverse sequencing primer (-46), 24-mer GAG CGG ATA ACA ATT TCA CAC AGG;<br>
SP6 promoter sequencing primer, I8-mer ATT TAG GTG ACA CTA TAG;<br>
SP6 promoter sequencing primer, 24-mer CAT ACG ATT TAG GTG ACA CTA TAG;<br>
T7 promoter sequencing primer, 20-mer TAA TAC GAC TCA CTA TAG GG;<br>
T3 promoter sequencing primer, 17-mer ATT AAC CCT CAC TAA AG; and.<br>
 <br>
T3 promoter sequencing primer, 24-nier GCG CGA AAT TAA CCC TCA CTA AAG.<br>
[00126]	Amplification from primer sites such as those described above effectively<br>
allows the use of a "universal" primer set, which bind at C and C to amplify X from a<br>
range of strains of the analyte. Furthermore, it should be noted that the amplicon amplified<br>
using the universal primers may comprise both conserved and variable regions.<br>
100127]	In some embodiments, primers are used which amplify HPV sequences.<br>
More preferably these primers are GP5+ is 5' TTT GTT ACT GTG GTA GAT ACT AC 3' and GP6+ is 5' GAA AAA TAA ACT GTA AAT CAT ATT C 3'. These primers generate an amplicon from HPV which comprises both a conserved region (defined as Y in Figure 2), and a region which is variable among different strains of HPV (defined as X in Figure 2).<br>
[00128]	In other embodiments, these primers are GP5d2+ is 5' TTT KTT ACH<br>
GTK GTD GAT ACH AC 3' and T7aGP6d+ is 5' AAT TCT AAT ACG ACT CAC TAT AGG GGA AAH ATA AAY TGY AAD TCA TAY TC 3'.<br>
[00129]	In another embodiment, these primers are GP5d3+ is 5' 5Phos TTT GTT<br>
ACH GTD GTD GAY ACH AC 3' and T7aGP6d+* is 5' y5AmMC6/AAT TCT AAT<br>
ACG ACT CAC TAT AGG GGA AAH ATA AAY TGY ARD TCA WAY TC 3'.<br>
[00130]	In other embodiments, the forward primers of a primer pair are selected<br>
from the group consisting of:<br>
TR TTT GTT ACT GTK GTD GAT ACY A, optionally having a caatcagc or<br>
/SPhos/caatcagc conjugated to the S' terminus of the primer;<br>
CAR YTR TTT GTT ACT GTK GTD GAT A, optionally having a acaat or /SPhos/acaat conjugated to the 5' terminus of the primer;<br>
CAR YTR TTT GTT ACT GTK GTD GA, optionally having a acaat or /SPhos/acaat conjugated to the 5' terminus of the primer;<br>
AAY CAR YTR TFT GTT ACT GTK GT, optionally having a ggaac or /SPhos/ggaac conjugated to the 5' terminus of the primer;<br>
TTT GTT ACT GTK GTD GAT ACY AC HCG, optionally having a cacctt or /SPhos/cagctt conjugated to the 5' terminus of the primer;<br>
TTT GTT ACT GTK GTD GAT ACY AC HCG YAG, optionally having a a caectt /SPhos/cagctt conjugated to the 5' terminus of the primer;<br>
 <br>
GTK GTD GAT ACY AC HCG YAG TAG, optionally having a /SPhos/attacc conjugated to the 5' terminus of the primer; and<br>
GTD GAT ACY AC HCG YAG TAC HAA, optionally having a attacc or /SPhos/ctgtt conjugated to the 5' terminus of the primer; and the reverse primers of a primer pair are selected from the group consisting of:<br>
TGA AAA ATA AAY TGY AAA TCA TAT TCY TCM MCA TG, optionally having a actcactatagg or /SAmMC6/actcactataeg conjugated to the 5' terminus of the primer;<br>
CAY ARY TGA AAA ATA AAY TGY AAA TC, optionally having a aatacgactcactatagu or /5AmMC6/aat^cgactcactatagg conjugated to the 5' terminus of the primer;<br>
TR CAY ARY TGA AAA ATA AAY TG, optionally having a tctaatacgactcactatagg or /SAmMC6/tctaatacgactcactatagg conjugated to the 5' terminus of the primer; and<br>
TR CAY ARY TGA AAA ATA AA, optionally having a aattctaatacgactcactatagg or /5AmMC6/aattctaatacgactcactatag8 conjugated to the 5' terminus of the primer. Alternatively, the forward and reverse primers are selected from a group of primers similar to those described hereinabove, but which may differ from the forward or reverse primers described hereinabove by from 0 to about S nucleotide substitutions, preferably 0 to about 2 substitutions, at sites where nucleotide substitutions occur between different HPV strains, but with the exclusion of substitutions of the last two nucleotides at the 3' end. The invention also contemplates similar substitutions to optional heel sequences as those for forward or reverse primers desdribed hereinabove, said optional heels conjugated to the 5" terminus of any of the above disclosed forward or reverse primers. In certain preferred embodiments, the forward primer is selected from CAR YTR TTT GIT ACT GTK GTD GA, optionally having a acaat or /5Phos/acaat conjugated to the 5' terminus of the primer and GTD GAT ACY AC HCG YAG TAC HAA, optionally having a gtgjt or/5Phos/ctgtt conjugated to the S' terminus of the primer. In certain other preferred embodiments, the reverse primer is selected from TGA AAA ATA AAY TGY AAA TCA TAT TCY TCM MCA TG, optionally having a actcactatagg or /5AmMC6/actcactatagg conjugated to the 5' terminus of the primer and TR CAY ARY TGA AAA ATA AAY TG, optionally having a tctaatacpactcactatagg or /S AmMC6/tctaatacgactcactatagg conjugated to the 5' terminus of the primer.<br>
 <br>
[00131 ]	The above descriptions show examples of primer pairs useful in the present<br>
invention. Additional primer pairs can be formed by pairing different forward (e.g., GP5+,<br>
GP5d2+, etc.) and reverse primers (e.g., GP6+, T7aGP6d+) with one another.<br>
(00132]	Certain embodiments of the invention are directed to fragments of any of<br>
the herein disclosed primers including all combinations and subcombinations of said<br>
primer fragments, to a lower limit of about 20-mer 3-prime core fragments of any of the<br>
disclosed primers. As used herein, a "fragment" refers to a truncated primer or probe<br>
nucleic acid oiigomeric sequence formed by deletion of one or more nucleotides from the<br>
original primer.<br>
[00133]	The present invention also contemplates the amplification of control<br>
sequences. In one embodiment, the control sequence may include a region of the genome<br>
of the subject from which a biological sample is derived. However, the present invention<br>
is not in any way limited to these particular control sequences and other control sequences<br>
which would be evident to one of skill in the art are also contemplated. Furthermore, the<br>
methods of the present invention may also be performed without the amplification of a<br>
control sequence.<br>
[00134]	In some embodiments, the control sequence is amplified from the genome<br>
of a human subject using the primers MLC1_F, which comprises the nucleotide sequence<br>
5' TAG ACA CAG GTG TAG ACA GA 3' and MLC1_R which comprises the sequence<br>
5' ACC AAG TAG TCT ACG TGT TG 3".<br>
[00135]	In other embodiments, the control sequence is amplified from the genome<br>
of a human subject using the primers mlcl_9Sf, which comprises the nucleotide sequence<br>
5' GGC ACC CAG ACA TAC AC 3' and T7amlcl_275r (HeelMLCR), which comprises<br>
the sequence 5' AAT TCT AAT ACG ACT CAC TAT AGG GGA AAH ATA AAY TGY<br>
AAD TCA TAY TC 3'.<br>
[00136]	Isolated DNA may be amplified using any DNA amplification protocol. A<br>
range of exemplary methods for the amplification of DNA which in no way limit the<br>
invention are presented in "DNA Amplification: Current Technologies and Applications"<br>
(Ekmidov and Broude Eds., Horizon Bioscience, 2004).<br>
[00137]	Isolated RNA may be amplified using any RNA methods known in the art<br>
and number of RNA amplification technologies have been developed. Two major<br>
categories of these are: (i) those that utilise thermal cycling such as RT-PCR and (ii)<br>
isothermal assays such as nucleic acid sequence-based amplification (NASB A) (Compton,<br>
 <br>
Nature 350:91-92, 1991; Kievits el al., J. Virol. Methods 35:273-286, 1991) and<br>
transcription-mediated amplification (TMA) (Hill, J. Clin, ligandAssay 79:43-51, 1996).<br>
Isothermal assays may be sub-divided, based on whether; (i) they copy and amplify the<br>
target sequence, such as TMA, NASBA and self-sustained sequence replication (3SR)<br>
(Guatelli et al., Proc. Natl Acad Set. USA 87:1874-1878, 1990; Chadwick etal., J. Virol.<br>
Methods 70:59-70, 1998; for review see Chan and Fox, Rev. Med Microbiol. 70:185-196,<br>
1999), or (ii) they generate a target-dependent signal which can be further amplified, e.g.,<br>
invader assays (Lyamichev et al., Nat. Biotechnol. 77:292-296, 1999; Ryan et al., Mol.<br>
Diagn. */: 135-144, 1999). There are various other amplification technologies that do not<br>
fit readily into these categories, such as Q beta replicase (Lizardi et al., Biotechnology<br>
6:1197-1202, 1988) and branched DNA (Todd etal., J. AIDS Hum. Retroviral. 70:S35-<br>
S44, 1995; Pawlotsky et al., J. Virol. Methods 79:227-235, 1999). However, it should be<br>
understood that the present invention contemplates any method of RNA amplification that<br>
would be evident to one of skill in the art. Furthermore, it should be understood that the<br>
present invention also contemplates the use of reverse transcriptase or a functional<br>
equivalent thereof to convert RNA to DNA which may then be subsequently amplified.<br>
[00138]	In accordance with the present invention, the amplicons recited at steps (iii)<br>
and/or (iv) of the methods described supra are labelled. The amplicons of the present<br>
invention may be labelled using any convenient means. Exemplary methods include both<br>
pre- and post-synthesis methods. Pre-synthesis labelling methods include labelling of a<br>
PCR primer that is subsequently used for amplification of, and thereby incorporated into,<br>
an amplicon via PCR. In this method, the label is typically attached to the 5' end of a<br>
primer suitable for the amplification of the amplicon, although labelling at other positions<br>
within the primer, such as 3' labelling or non-terminal labelling, is also contemplated.<br>
[00I39J	A chemical linker may also be used between the label and the<br>
polynucleotide which is labelled. Appropriate linker sequences will be readily ascertained by those of skill in the art, and are likely to include linkers such as Cs, C? and C)2 amino modifiers and linkers comprising thiol groups. As will be readily ascertained, a primer may comprise the linker and label, or the linker alone, to which the label may be attached at a later stage.<br>
(00140]	Post-amplification labelling methods include nick-labelling systems<br>
wherein a labelled polynucleotide is synthesised from the amplicon using Klenow polymerase, or a functional equivalent thereof, from random primers. Labelled<br>
 <br>
nucleotides, or nucleotides comprising a iinlcer group, may be incorporated into the<br>
Klenow polymerase synthesised polynucleotide during synthesis.<br>
[00141]	In any event, other labelling methods should be readily evident to one of<br>
skill in the art and it should be understood that the present invention is in no Avay deflned or limited by the choice of labelling method.<br>
[00142]	Preferably, the label used is a "fluorescent marker" or "fluorophore". Many<br>
different fluorescent markers will be familiar to those of skill in the art, and the choice of fluorescent marker in no way limits the invention. In other preferred embodiments of the present invention the fluorescent markers of the present invention comprise any fluorescent marker that can be attached to a polynucleotide and which is excitable using a light source selected from the group below:<br>
(i)       Argon ion lasers: comprise a blue, 488 nm line, which is suitable for the excitation of many dyes and fluorochromes that fluoresce in the green to red region. Tunable argon lasers are also available that emit at a range of wavelengths (4S8 nm, 488 nm, 496 nm, 515 nm and others), (ii)      Diode lasers: have an emission wavelength of 635 nm. Other diode lasers which are now available operate at S32 nm. Interestingly, this wavelength excites propidium iodide (PI) optimally. PI staining is widely used for DNA analysis, live/dead counting and ploidy determination. Blue diode lasers emitting light around 476 nm are also available (iii)     HeNe gas lasers: operate with the red 633 nm line, (iv)     HeCd lasers: operate at 325 nm. (v)       100 W mercury arc lamp: the most efficient light source for excitation of UV dyes<br>
like Hoechst and DAPI.<br>
[00143]	In more preferred embodiments of the present invention the fluorescent<br>
markers are selected from: hydroxycoumarin, aminocoumarin, methoxycoumarin, cascade blue, Lucifer yellow, NBD, Phyccerythrin (PE), PerCP, allophycocyanin, Hoechst 33342, DAPI, SYTOX Blue, Hoechst 33258. chromomycin A3, mithramycin, YOYO-1. SYTOX green, SYTOX orange, 7-AAD, acridine orange, TOTO-1, To-PRO-I, thiazolc orange, TOTO-3, TO-PRO-3, LDS 751, Alexa Fluor dyes including Alexa Fluro-3S0, -430, -488, -532, -546, -555. -556, -594, -633, -647, -660, -680, -700 and -750; BoDipy dyes, including BoDipy 630/650 and BoDipy 650/665; CY dyes, partlculary Cy2, Cy3, Cy3.5, Cy5, Cy 5.5 and Cy7; 6-FAM (Fluorescein); PE-Cy5, PE-Cy7, Fluorescein dT; Hexachlorofluorescein<br>
 <br>
(Hex); 6-carboxy-4\ 5'-dichloro-2', 7'- dimethoxyfluorescein (JOE); Oregon green dyes,<br>
including 488-X and S14; Rhodamine dyes, including X-Rhodamine, Lissamine<br>
Rhodamine B, Rhodamine Green, Rhodamine Red and ROX; TRITC,<br>
Tetramethylrhodamine (TMR); Carboxytetramethylrhodamine (TAMRA);<br>
Tetrachlorofluorescein (TET); Red 6B, FluorX, BODIPY-FL, SYBR Green I dye, and<br>
Texas Red. In particularly preferred embodiments of the present invention, the marker is<br>
Cy5 which is particularly convenient for practice of the present invention.<br>
(00144]	However, in alternate embodiments of the invention, radioactive or non-<br>
radioactive labels may be used to label the amplicon. Convenient radioactive labels<br>
include ^^P and ^H. These labels may be incorporated into the amplicon and/or primer<br>
using any convenient means. A range of non-radioactive labelling methods may also be<br>
used. Exempjary non-radioactive labelling methods which in no way limit the present<br>
invention are presented in Speel (Histochem. Cell Biol. 112:99-1 M, 1999).<br>
[00145]	The term "reactant" as used herein should be understood to comprise a<br>
polynucleotide immobilized to a bead. More particularly, each reactant comprises a<br>
polynucleotide comprising a sequence which is complementary to an amplicon generated<br>
according to the methods described herein, which is bound or otherwise associated with a<br>
physiochemically distinguishable bead. A reactant may also comprise a sequence which is<br>
complementary to a control sequence as hereinbefore defined, i.e., a sequence amplified<br>
from the genome of a multicellular organism (Z) or the amplicon of a nucleotide sequence<br>
which is conserved among strains of the analyte (Y).<br>
(00146]	Accordingly, a headset of reactants may comprise:<br>
(0003J        (B|-cX|, B2-CX2. Bj-cX3...B„-cX,H By-cY, Bz-c2] wherein:<br>
B|. ■ .Bn, By, Bx    are each physiochemically distinguishable beads;<br>
cXn	is a polynucleotide immobilized to a bead wherein said polynucleotide<br>
comprises a nucleotide sequence which is complementary to a particular nucleic acid sequence which is specific to an analyte or a particular strain of a subject analyte and wherein n is the number of analytes or particular strains of a subject analyte to be detected using the headset;<br>
cY	is an optional member of the headset and is a polynucleotide<br>
immobilized to a bead wherein said polynucleotide comprises a nucleotide sequence which is complementary to a sequence which is conserved among the analytes or strains of a<br>
 <br>
subject anaiyte;<br>
cZ	is an optional member of the beadset and is a polynucleotide<br>
immobilized to a bead wherein said polynucleotide comprises a nucleotide sequence which<br>
is complementary to a control sequence which is amplified from a multicellular subject.<br>
[00147]	Preferably, the subject anaiyte is HPV and the control DNA sequence is a<br>
human genomic DNA sequence.<br>
[00148]	By "complementary", it is to be understood that the immobilized<br>
polynucleotide of the reactant should hybridize to an amplicon generated according to the methods described herein under low stringency conditions. Preferably the immobilized polynucleotide should bind to the sample and standard under medium stringency conditions, and most preferable the immobilized polynucleotide should bind to the sample and standard under high stringency conditions.<br>
[00149]	Reference herein to low stringency includes and encompasses from at least<br>
about 0 to at least about 15% v/v formamide (including 1%, 2%, 3%, 4%, 5%, 6%, 7%, 8%, 9%, 10% 11%, 12%, 13% and 14% v/v formamide) and from at least about 1 M to at least about 2 M salt for hybridization, and at least about 1 M to at least about 2 M salt for washing conditions. Generally, low stringency is at from about 25-30'C to about 52°C, such as 25,26, 27. 28, 29, 30, 31, 32, 33, 34, 35, 36, 37. 38, 39, 40, 41, 42,43,44, 45,46, 47, 48, 49, 50. 51 and 52'C. The temperature may be altered and higher temperatures used to replace formamide and/or to give alternative stringency conditions. Alternative stringency conditions may be applied where necessary, such as medium stringency, which includes and encompasses from at least about 16% v/v to at least about 30% v/v formamide, including 16%, 17%, 18%, 19%. 20%, 21%. 22%, 23%, 24%, 24%, 26%, 27%, 28%,.29% and 30% v/v formamide, and from at least about 0.5 M to at least about 0.9 M salt for hybridization, and at least about 0.5 M to at least about 0.9 M salt for washing conditions, or high stringency, which includes and encompasses from at least about 31% v/v to at least about 50% v/v formamide and from at least about 0.01 M to at least about 0.15 M salt for hybridization, and at least about 0.01 M to at least about 0.15 M salt for washing conditions. In general, washing is carried out Tm = 69.3 + 0.41 (G-K;)% (Marmur and Doty, J. Mol Biol. 5.109, 1962). However, the Tn,of a duplex DNA decreases by PC with every increase of 1% in the number of mismatch base pairs (Bonner and Laskey, Eur. J. Biochem. ^6.83, 1974). Formamide is optional in these hybridization conditions. Accordingly, particularly preferred levels of stringency are defined as follows:<br>
 <br>
low stringency is 6 x SSC buffer, 0.1% w/v SDS at 25-42'C; a moderate stringency is 2 x<br>
SSC buffer, 0.1% w/v SDS at a temperature in the range 20'C to less than es^C; high<br>
stringency is 0.1 x SSC buffer, 0.1% w/v SDS at a temperature of at least eS'C.<br>
[00150]	The beads, B|.. .Bn, By, Bz, of the reactant beadsets are each<br>
physiochemlcally distinguishable beads. The term "physiochemlcally distinguishable"<br>
refers to any physical or chemical characteristic which allows one bead, e.g., Bi to be<br>
differentiated from another bead e.g., Bj. Accordingly, the physiochemlcally<br>
distinguishable beads allow differentiation of particular reactants.<br>
[00151]	In certain preferred embodiments the bead comprises a "microparticle". As<br>
will be evident to those of skill in the art, almost any material, homogeneous or otherwise may be used for the microparticle. The microparticles contemplated herein may also comprise more than one substance, and as such may comprise shells, alloys or mixtures of organic and/or inorganic substances. Particularly useful materials which may be used in accordance with the present invention and which represent preferred embodiments of the present invention include materials selected from the list consisting of: silica (for example: quartz or glass), latex, titania, tin dioxide, yttria, alumina, and other binary metal oxides (such as ZnO), perovskites and other piezoelectric metal oxides (such as BaTiOs), ZnS, sucrose, agarose and other polymeric beads. In a particularly preferred embodiment, the microparticle comprises silica.<br>
[001S2]	In some preferred embodiments, the term "physiochemlcally<br>
distinguishable" refers to a measurable difference in any of bead size, the presence or absence of a particular optically detectable label and/or the intensity of an optically detectable label.<br>
[00153]	Beads contemplated by the present invention may be produced in any<br>
convenient regular or irregular 3-dimensional shape. However, it is generally practical to synthesize small spheres or spheroidal particles. Such spheres or spheroidal particles are also referred to herein as "beads". Accordingly, in preferred embodiments of the present invention, the "microparticles" of the present invention are substantially spherical or spheroidal or comprise a "microsphere".<br>
[00154]	Although the beads of the present invention may be referred to as<br>
"microspheres" the actual size of the particles depends on a variety of factors and the particles may or may not actually comprise measurements in the micrometer range. In some preferred embodiments, the bead comprises a diameter (or equivalent measurement<br>
 <br>
in a non-spheroidal particle) of about 300 nm to about 30 jim, including 300nm, 350nm, 400nm, 450nm, 500nm, 550nm, 600nm, 650nm, 700nm, 750nm, 800nm, 850nm, 900nm, 950nm, l.Oum, 1.1 i^m, l.2M.m, 1.3nm, 1.4^m. 1.5^m, 1.6nm. 1.7^m, 1.8(im, 1.9&gt;im, 2.0^m, 2.1|am, 2.2^im, 2.3fun, 2.4jim. 2.5^m, 2.6^m. 2.7^^m, 2.8nm, 2.9iim, 3.0nm, 3.1^m, 3.2nm, 3.3Mm, 3.4^ni, 3.5^m, 3.6fiin, 3.7fiin, 3.8^m, 3,9j4m, 4.0^m, 4Ayim, 4.2fim, 4.3^m, 4.4pjn, 4.5fim, 4.6nm. 4J\im, 4.8mTi, 4.9nm, S.Oum, S.ljim, S.2(xm, 5.3nm, 5.4^un. S.Sjim, 5.6nni. 5.7jim, 5.8nm, 5.9fim, 6.0^ni, 6.1^m, 6.2|im, 6.3(im, 6.4}im, 6.5^m, 6.6jim. 6.7j«n, 6.8^m, 6.9nm, 7.0^m, l.\)xm, 7.2nm, 7.3fim, 7.4^m. 7.5nm, 7.6^m, 7.7nm, 7.8nm, 7.9jAm, 8.0^, 8.1^m, 8.2pjn, 8.3pm, 8.4pm, 8.5pm, 8.6pm, 8.7pm, 8.8pm, 8.9pm, 9.0pm, 9.1pm, 9.2pm, 9.3pm, 9.4pm. 9.5pm, 9.6pm. 9.7pm, 9.8pm, 9.9pm, 10pm, 11 pm, 12pm, 13pm, 14pm, 15pm, 16pm, 17pm, 18pm, 19pm. 20pm, 21pm, 22pm, 23pm, 24pm, 25pm, 26pm, 27pm, 28pm, 29pm, 30pm. More preferably, the bead comprises a diameter (or equivalent measurement in a non-spheroidal particle) of between 1pm and 10pm.<br>
[00155]	In one particularly preferred embodiment, the beads are AmpaSand (Trade<br>
mark: Genera Biosystems) beads produced by Genera Biosystems. These beads arc commercially available and are described at<br>
www.gencrabiosystems.com/generabiosystems/technologv/AmpaSandBeads/. However, the present invention should not be considered in any way limited to the use of these beads specifically.<br>
[00156]	The beads may be distinguished on the basis of the presence or absence of<br>
one or more "optically detectable labels". Typically, a particular bead may comprise 0, 1,<br>
2, 3,4, 5 optically detectable labels. As used herein, the term "optically detectable label"<br>
refers to any molecule, atom or ion which emits fluorescence, phosphorescence and/or<br>
incandescence. Convenient optically detectable labels include those which emit in the<br>
ultraviolet (wavelength range of about 350nm to about 3nm), visible (wavelength range of<br>
about 350nm to about 800nm, near infrared (NIR) (wavelength range of about SOOnm to<br>
about 1 SOOnm) and/or infrared (IR) (wavelength range of about 1 SOOnm to about 10pm)<br>
ranges. However, due to the case of detection, in one particularly preferred embodiment,<br>
the optically detectable label is detectable in the visible wavelength range.<br>
[00157]	In further preferred embodiments of the subject invention, the optically<br>
detectable label comprises one or more labels selected from the list consisting of: a<br>
 <br>
fluorophore, a semiconductor particle, phosphor particle, a doped particle, or a nanocrystal or quantum dot<br>
[001S8]	In certain particularly preferred embodiments of the present invention, the<br>
optically detectable label is a fluorophore. As used herein, the term "fluorophore" refers to any molecule which exhibits the property of fluorescence. For the purposes herein, the term "fluorescence" may be defined as the property of a molecule to absorb light of a particular wavelength and re-emit light of a longer wavelength. The wavelength change relates to an energy loss that takes place in the process. The term "fluorophore" may encompass a range of optically detectable labels such as chemical fluorophores and dyes as well as quantum dots.<br>
[00159]	Particularly convenient optically detectable labels which may be used in<br>
accordance with the present invention are embedded fluorescent particles of semiconductors. These optically detectable label particles may be so small that their properties and emission become size dependent. Such small optically detectable label particles are referred to in the art as semiconductor nanoparticles, quantum dots, quantum wires, quantum rods or nanocrystals or Q-particIes. However, as used herein, the term "Quantum Dot" or "QD" is to be understood to encompass all such particles. Furthermore, optically detectable labels comprising QDs may comprise approximately spherical or spheroidal particles, or coated spherical or spheroidal particles. However, the term QD should not be considered in any way to be limited to a spherical, spheroidal, circular, cylindrical or any other morphology of a "dot". For example, as used herein QDs may also comprise other morphologies including, inter alia, rod-like, ellipsoidal, or coated rod-like or ellipsoidal particles.<br>
[00160]	QDs consist of a nanometer-scale crystalline core of semiconductor<br>
material; biologically active versions are typically surrounded by a protective shell and<br>
external coat. For example, QDs may comprise semiconductor crystallites which are about<br>
2 nm to about 30 nm in diameter and may contain approximately 50-500,000 atoms within<br>
the crystal, including luminescent crystals comprising materials such as ZnS, ZnSe, ZnTe,<br>
CdS, CdSe. CdTe, PbS, PbSe, PbTe, HgS, HgSe, HgTe, Si, ZnO.<br>
[00161]	QDs fluoresce with a broad absorption spectrum and a narrow emission<br>
spectrum. Unlike some other fluorophores, which have distinct absorption spectra, QDs absorb light over a wide spectral range, which allows quantum dots to be excited with a range of light sources, such as lasers, arc lamps, or LEDs. Furthermore, a collection of<br>
 <br>
different QDs can be used in multiplex applications using only a single excitation source. However, the emission spectra for each dot is typically very narrow, in the order of about 30 nm, the exact color depending on the particle's diameter and composition. Furthermore, the narrow emission spectrum of QDs permits spectral resolution of adjacent dots. In addition to the benefits above, QDs are also relatively photostable, even during intense excitation, and are brighter than fluorophores.<br>
(00162]	In light of the foregoing, it should also be understood that the present<br>
invention encompasses the use of different sized QDs.<br>
[00163]	Furthermore, the present invention contemplates QDs which are treated<br>
with procedures such as thermal treatment, surface modification, alloying, surface passivation or capping with surface coatings to enable the QD to emit with high quantum yield and to improve the photostability for long periods of time.<br>
[00164]	QDs are also commercially available from companies such as Quantum Dot<br>
Corp. (QDC), which produces QDs such as the Qdot [Trade Mark] 605 streptavidin conjugate, containing a cadmium-selenide core that emits at 605 nm. Qdot conjugates that emit at 525, 565, 585, and 655 nm are also available. However, it should be understood that the present invention is not limited in any way by the particular composition of the QD (or any other optically detectable label) and any QD (commercial or otherwise) may be compatible with the present invention.<br>
[00165]	There are also many fluorescent dyes that are available in the art which may<br>
be used as fluorophores in accordance with the present invention. An important property of a fluorescent dye or other fluorophore, which determines its potential for use is the excitation wavelength of the fluorophore; it must match the available wavelengths of the light source. However, many different fluorescent dyes and other fluorophores will be familiar to those of skill in the art, and the choice of fluorescent marker in no way limits the subject invention. Particularly convenient fluorescent dyes which may be used for the labelling of a substrate include those discussed supra with regard to labelling of the PCR amplicon. However, when choosing fluorescent labels, the emission spectra of the fluorescent label used for the binding agent(s) should be distinct from the emission spectrum of the label used for the amplicon(s).<br>
[00166]	Two dyeing techniques are commonly used to fluorescently label beads and<br>
microspheres - internal dyeing and external dyeing (surface-labelling). The two techniques produce beads with unique properties, each beneficial for different applications.<br>
 <br>
internal dyeing produces extremely stable particles with typically narrow fluorescence emissions. These beads often display a greater resistance to photobleaching. As the fluorophore is inside the beads, surface groups are available for use in conjugating ligands (proteins, antibodies, nucleic acids, etc.) to the surface of the bead. For this reason, internally labelled beads are typically used in analyte-detection and immunoassay applications. Surface-labelling involves conjugation of the fluorophore to the bead surface. Because the fluorophores are on the surface of the bead, they are able to interact with their environment just as the fluorophores on a stained cell. The result is a bead standard that exhibits the same excitation and emission properties as stained cell samples, under a variety of different conditions, such as the presence of contaminants or changes in pH. The "environmentally responsive" nature of surface-labelled beads makes them ideally suited for mimicking biological samples. Externally labelled beads are frequently used as controls and standards in a number of applications utilizing fluorescence detection. However, the present invention contemplates the association of a bead with a fluorescent label via any means.<br>
[00167]	The terms "phosphorescent beads", •'phosphor beads" and "phosphors" are<br>
used interchangeably herein. What constitutes a phosphorescent optically detectable label<br>
would be readily understood by one of skill in the art. However, by way of example,<br>
which in no way limits the invention, suitable phosphors include small particles of ZnS,<br>
ZnS:Cu, Eu oxide and other phosphors used in display devices.<br>
(00168]	A optically detectable label comprising a "doped bead" may include a<br>
particle which comprises occluded amounts of one or more rare earth ions, such as Eu, Y, Yb, Sm and the like.<br>
[00169]	As used herein, the term "optically detectable label" should be understood<br>
to also encompass multiple optically detectable labels, mixtures of optically detectable labels, coated nanocrystals, alloys and other complex mixtures that would be evident to the skilled artisan. The use of all such optically detectable labels is to be considered as being within the scope of the methods and agents described herein.<br>
[00170]	Furthermore, the optically detectable label of the reactant may comprise an<br>
optically detectable label incorporated into the immobilized polynucleotide sequence which is bound or otherwise associated with the bead, rather than being a label directly associated with the htz6perse.<br>
 <br>
(00171)	The beads are generally labelled by the immobilized "tag" or probe<br>
oligonucleotide. This tag carries an internal amine (NH^) which is then modified by conjugation with a succinimidyl ester of a dye. In the current set, the dye used is BODIPY-TMR. By mixing labelled and unlabel led tags and then conjugating this mix to the beads, one can produce classes of beads with different levels of the fluorescent marker. The ratios conveniently used are in a series of 1:5* ; that is, the different classes are produced by using the ratio of unlabeled : labelled tags. This is generically exemplified below in Table 4.<br>
Table 4 - Ratio of unlabelled to labelled tags<br>
Class	Rel amount Unlabeled	Rel Amt Labeled<br>
__	__	__	_^<br>
None	All      ~~	~~     0<br>
1/5	"	5	i<br>
__	_-	J<br>
17125	125	i<br>
[00172]	The optically detectable label may be applied to a bead at a range of<br>
concentrations or intensities, thereby providing another basis on which particular beads<br>
may be "physiochemically distinguishable". For example, if the maximum detectable<br>
intensity of the signal of a particular optically detectable is deemed to be 100%, the label<br>
may be applied to a range of beads to give intensities of 0%, 2%, 4%, 6%, 8%, 10%, 15%,<br>
20%, 25%, 30%. 35%, 40%, 45%, 50%, 60% 70%. 80%, 90%, 100%.<br>
[00173)	In one embodiment, the headset of reactants comprises beads of 3.0|im,<br>
3.5|am, 4.l|im, 5.0^m, 5.6&gt;4jn and 6.8fim, wherein the 3,0fim, 3.5|im and 4.1^m diameter<br>
beads are labelled at 0% and 100%, the 5.0^m diameter beads are labelled at 0%, 100%<br>
and 20% and the 5.6fim and d.S^m diameter beads are labelled at 0%, 100%, 20% and 4%.<br>
[00174]	In another embodiment, the beadset of reactants comprises beads of 3.0^m,<br>
3.5fim, 3.77fim. S.O^m, 5.6pm and 6.8^m. All bead sizes, (except for the 3.0pm which is divided by zero level and high level of TMR), are divided according to their respective zero level, medium level, and high level of TMR.<br>
[00175]	In other embodiments, TMR levels are set such that during bead to oligo<br>
conjugation, oligos with defined total oligo to fluor-labelled ratios are used to determine discrete TMR groupings.   For example, zero levels of TMR have unlabelled oligos<br>
 <br>
included in conjugation; high levels of TMR have a 3:1 total oligo to fluor-labelled ratio of<br>
oligos included in conjugation; and medium levels of TMR have from a 150:1 to an 80:1<br>
total oligo to fluor-labelled ratio of oligos included in conjugation, depending on bead size.<br>
100176]	The immobilized polynucleotide component of the reactant, e.g., cXn, cY<br>
and/or cZ may be bound to a bead using any convenient means.<br>
[00177]	The immobilized polynucleotide may be encapsulated in beads during their<br>
production or may be attached to their surface post-production. The choice method used to associate the polynucleotide with the bead will depend on the material used, as would be readily ascertained by the skilled artisan. In addition, further treatments, including silanization (coating of the substrate with silanes), may be performed on the beads prior to attachment of the polynucleotide in order to increase the binding of said polynucleotide to the bead.<br>
{00178]	Generally, beads may be coated with any compound that will covalently<br>
attach, or otherwise adsorb, to the surface of the bead, and in addition the reactant should<br>
also have a chemical moiety for the attachment of a polynucleotide, such as a thiol, amine<br>
or carboxyl group. Non-limiting examples of compounds with these characteristics<br>
include amino-terminated silanes such as amino-propyltrimethoxysilane and amino-<br>
propyltriethoxysilane, as well as thioi-terminated silanes such as thio-<br>
propyltrimethoxysilane and thio-propyltriethoxysilane. In addition to silanes, compounds<br>
such as poly-L-lysine that non-covalently attach to the glass surface and electrostatically<br>
adsorb the phosphate groups of the polynucleotide are also within the scope of the present<br>
invention. Therefore, other compounds, including other silanes suitable for the attachment<br>
of a polynucleotide to a surface would be readily identified by the skilled artisan, and the<br>
present invention is not limited by the choice of compound.<br>
[00179]	The polynucleotide can be attached to the bead using any convenient<br>
means; typically this is done by physical adsorption or chemical linking. In addition, beads may be further coated with an agent that promotes or increases the adsorption or binding of the polynucleotide to the surface of the bead, such as amino-silanes. However, other agents that perform this fimction will be readily identified by persons of skill in the art. In some other embodiments, the polynucleotide is covalently conjugated to the bead by an oligo acrydite group reacting with a bead thiol group.<br>
[00180]	In another embodiment, the nucleic acid molecule is bound to the bead via<br>
the Universal Anchoring System (UAS) (Trade mark: Genera Biosystems). Briefly, this<br>
 <br>
system involves the use of a "bridge" nucleic acid molecule to ligate a nucleic acid "tag" sequence on the substrate with a target sequence. The "bridge" sequence is partially complementary to the tag sequence and partially complementary to the target sequence, such that the bridge sequence may bind to both the tag and target sequences and hold them in alignment such that the tag and target sequences may be ligated using a tigase. The UAS is also commercially available and is described in detail at www.generabiosvstems.com/generabiosvstems/technologv/UAS/- However, the present invention should not be considered in any way limited to this particular method of linking a nucleic acid molecule to a substrate.<br>
[00181]	Determination of whether binding has occurred between an amplicon and a<br>
reactant may be done using any methodology which allows localization of a bound amplicon label to a particular physiochemically distinguishable reactant. In a particularly preferred embodiment, flow cytometry is used.<br>
{0004]       Flow Cytometry may be defined as a technology to measure properties of particles or cells as they move, or flow, in liquid suspension. An analogy may be made with a more familiar item of laboratory equipment, the microscope, to further describe this technology. Most microscopes have the following components: A light source<br>
[00182]	The typical microscope uses a light bulb to illuminate the object. In the<br>
flow cytometer, the light source is often a laser. Lasers are used because they provide a very concentrated and intense beam of monochromatic light. The monochromatic character of the light is especially important in making fluorescence measurements.<br>
The stage<br>
[00183]	In a microscope, the stage is movable to allow passage of the object to the<br>
viewing field of an objective lens. In the flow cytometer, the cells or particles exist in liquid suspension. The liquid flows in response to air pressure, past an objective lens, thus carrying the cells or particles through the viewing fleld.<br>
The lens<br>
[00184]	In both the microscope and the flow cytometer, the lens collects light from<br>
the object.<br>
The filters<br>
[0018S]	Some microscopes have filters to select those characteristics of the light that<br>
are most important to the observer. This is particularly true of fluorescence microscopes.<br>
 <br>
In fluorescence, dye molecules are excited by light of a characteristic wavelength which then produce emitted light of a longer wavelength. The filters remove the excitation light to allow the emission light to be seen or measured.<br>
The detectors<br>
[00186]	In a microscope, the light detector is the observer. The flow cytometer uses<br>
highly sensitive light detectors called photomultiplier tubes (PMT's). The detectors must be able to measure the brief flashes of emitted light from cells or particles that are moving one at a time through the viewing field of the objective lens at rates of up to several thousand per second.<br>
(00187]	Most flow cytometers can measure both Light Scatter and Fluorescence.<br>
[00188]	In certain preferred embodiments, beads are detected and/or sorted<br>
according to the methods of the present invention using flow cytometry. The present invention, however, is in no way limited to the particular flow cytometry method or apparatus hereinbefore described. This example was provided only for illustrative purposes, and the present invention is not to be limited to an instrument or method according to the example provided.<br>
[00189]	Using flow cytometry, the size of a given bead may be determined by the<br>
light scatter of the object.<br>
[00190]	Light scatter is the interaction of light and matter. All materials, including<br>
beads, will scatter light. It is composed largely of light that is reflected or refracted. The position from which an object is viewed often determines what can be told about it. In the flow cytometer, light scatter detectors are usually located opposite the laser (relative to the cell or particle), and to one side of the laser, in-line with the fluid-flow/laser beam intersection. The measurements made by these detectors are called forward light scatter and side light scatter, respectively.<br>
[00191]	Forward light scatter provides some information on the relative size of<br>
individual cells or particles, whereas side light scatter provides some information on the relative granularity of individual beads. They are often used in combination to distinguish the different major categories of white cells in unseparated mammalian blood, but are useful in a wide variety of other assays as well, such as the determination of the size of a microparticle.<br>
[00192]	The present inventors have determined that flow cytometry is able to<br>
distinguish between beads of about 3.0^m, about 3.5fim, about 4. Ipm, about 5.0^m, about<br>
 <br>
S.6|am and about 6.8^m in diameter.  The present inventors also have determined that flow cytometry is able to distinguish between beads of about 3.0fim, about 3.Stun, about 3.77|im, about 5.0^m, about S.6)im and about 6.8)im in diameter. Still other bead classes can be distinguished, such as between S.O and 5.6 and 5.6 and 6.8. Accordingly, the present inventors have identified that flow cytometry can differentiate up to at least 6 different sizes of beads.<br>
[00193]	In addition to size detection, flow cytometers typically have one or more<br>
lasers and detectors for the detection of fluorescence in a sample. Fluorescence is the property of a molecule to absorb light of a particular wavelength and re-emit light of a longer wavelength. The wavelength change relates to an energy loss that takes place in the process. It is a characteristic that makes fluorescence extremely useful: filters may be used to exclude the excitation light from the light detector or the viewer. Thus, the only light measured or seen originates from the fluorophore. Interference from background or stray light striking the detectors is extremely low.<br>
[00194]	There are many fluorescent dyes that are useful for flow cytometry. They<br>
bind to a variety of cytochemical components, such as nucleic acids; proteins; specific cell-membrane, nuclear, and cytoplasmic receptors; intracellular ion molecules; and many more, A key property of a fluorescent dye which determines it's potential for use in a flow cytometric assay is the excitation wavelength, i.e., it must match the available wavelengths of the light source.<br>
[00195]	In other aspects, the present invention provides methods for diagnosing an<br>
infection by a pathogenic analyte in a subject, said method comprising: (i)       obtaining a biological sample from the subject which putatively comprises said<br>
pathogenic analyte; (ii)      isolating nucleic acid firom said sample;<br>
(iii)     amplifying the nucleic acid from said sample using primers which generate an ampiicon which is distinct for said analyte or a particular strain of said analyte; (iv)     optionally amplifying a control nucleic acid sequence from the subject; (v)      optionally effecting labelling of the amplicon(s) recited at steps (iii) and/or (iv); (vi)     hybridizing the labelled amplicon(s) to a headset of reactants wherein each member of the headset comprises a nucleic acid molecule having complementarity to a nucleotide sequence of the analyte or a particular strain of the analyte or a control nucleotide sequence, bound or otherwise associated with a physiochemically<br>
 <br>
distinguishable bead; and (vii)    determining to wliich of the reactants an amplicon has bound; wherein the association of an amplicon with a particular reactant is indicative of an infection by the analyte in the subject.<br>
[00196|	In other aspects, the hybridizing occurs in the presence of at least one signal<br>
oligonucleotide sequence.<br>
(00197]	In other aspects, the hybridizing occurs in the presence of at least one<br>
blocking oligonucleotide sequence.<br>
[00198]	As used herein the term "subject" refers to any organism may be susceptible<br>
to infection by another analyte. As such, a "subject" includes, but is not limited to animals, plants, fungi and bacteria (which may be infected by bacteriophage). As used herein the term "animal" preferably includes a mammal and more preferably a primate including a lower primate and even more preferably, a human. However, the term "animal" also specifically includes livestock species such as cattle, horses, sheep, pigs, goats and donkeys as well as laboratory animals. Examples of laboratory test animals include mice, rats, rabbits, guinea pigs and hamsters. Rabbits and rodent animals, such as rats and mice, provide a convenient test system or animal model as do primates and lower primates. Non-mammalian animals such as avian species, zebrafish, amphibians (including cane toads) and Drosophila species such as Drosophila melanogaster are also contemplated.<br>
[00199]	The "subject" may also be a non-animal such as a plant. The term "plant"<br>
specifically includes plants of agricultural value such as cereal plants (e.g., wheat, barley,<br>
oats, rye, triticale and maize), rice, fruit trees (e.g., apples, bananas, mangoes and oranges),<br>
sugarcane, horticultural crop plants (e.g., potatoes, carrots and onions) and the like.<br>
[00200]	However, in certain preferred embodiments, the present invention provides<br>
methods for diagnosing HPV infection in a human subject, said method comprising: (i)       obtaining a biological sample from the human subject which putatively comprises<br>
HPV; (ii)      isolating nucleic acid from said sample; (iii)     amplifying the nucleic acid from said sample using primers which generate an<br>
amplicon which is distinct for said analyte or a particular strain of said analyte; (iv)     optionally amplifying a control nucleic acid sequence from the genomic DNA of<br>
the human subject;<br>
 <br>
(v)       optionally effecting labelling of the amplicon(s) recited at steps (iii) and/or (iv);<br>
(vi)      hybridizing the labelled amplicon(s) to a beadset of reactants wherein each member of the beadset comprises a nucleic acid molecule having complementarity to a nucleotide sequence of HPV or a particular strain of HPV or a control nucleotide sequence, bound or otherwise associated with a physiochemically distinguishable substrate; and<br>
(vii)    determining to which of ttie reactants an ampUcon has bound;<br>
wherein the association of an amplicon with a particular reactant is indicative of HPV<br>
infection in the human subject.<br>
[00201]	In other aspects, the hybridizing occurs in the presence of at least one signal<br>
oligonucleotide sequence.<br>
[00202]	In other aspects, the hybridizing occurs in the presence of at least one<br>
blocking oligonucleotide sequence.<br>
[00203]	In some aspects, the present invention also contemplates methods for<br>
determining the risk of a human subject developing a disease associated with one or more<br>
strains of HPV said method comprising:<br>
(i)       obtaining a biological sample from the human subject which putatively comprises HPV;<br>
(ii)       isolating nucleic acid from said sample;<br>
(iii)     amplifying the nucleic acid from said sample using primers which generate an amplicon which is distinct for said analyte or a particular strain of said analyte;<br>
(iv)     optionally amplifying a control nucleic acid sequence from the genomic DNA of the human subject;<br>
(v)       optionally effecting labelling of the amplicon(s) recited at steps (iii) and/or (iv);<br>
(vi)      hybridizing the labelled amplicon(s) to a beadset of reactants wherein each member of the beadset comprises a nucleic acid molecule having complementarity to a nucleotide sequence of HPV or a particular strain of HPV or a control nucleotide sequence, bound or otherwise associated with a physiochemically distinguishable substrate; and<br>
(vii)    determining to which of the reactant an amplicon has bound;<br>
wherein the association of an amplicon with a particular reactant comprising a<br>
polynucleotide which is complementary to a strain of HPV associated with a particular<br>
disease, is indicative of an increased risk said disease in the subject.<br>
 <br>
(00204]	In other aspects, the hybridizing occurs in the presence of at least one signal<br>
oligonucleotide sequence.<br>
[00205]	In other aspects, the hybridizing occurs in the presence of at least one<br>
blocking oligonucleotide sequence.<br>
[00206]	When the labelling of the amplicons as outlined in part (v) occurs, both the<br>
amplicons and the beads are labelled.<br>
[00207]	Exemplary diseases associated with one or more particular strains of HPV<br>
include those presented in Table 3. Accordingly, in some aspects, the present invention<br>
provides methods for diagnosing an increased risk of a subject developing a particular<br>
disease by specifically identifying which strain of HPV is infecting the subject. In certain<br>
particularly preferred embodiments, the methods are adapted to determining the risk of a<br>
human subject developing cervical cancer.<br>
[00208]	The present invention further contemplates diagnostic kits for use according<br>
to the methods described herein, including diagnosing HPV infection in a human subject<br>
and/or assessing the risk of a human subject developing an HPV-associated disease,<br>
including cervical cancer. The kits comprise a headset of reactants which each comprise a<br>
polynucleotide which is complementary to a nucleotide sequence of a particular strain of<br>
HPV, bound or otherwise associated with a physiochemically distinguishable substrate.<br>
Optionally, the kits may also comprise primers that bind to conserved sequences among<br>
different strains of HPV. but generate an amplicon which comprises distinct nucleotide<br>
sequences for each strain of HPV wherein the amplicon generated is putatively<br>
complementary to a polynucleotide bound to or otherwise associated with one or more<br>
physiochemically distinguishable beads of the kit.<br>
[00209]	In some embodiment, the headsets of reactants comprise at least group of<br>
beads, each comprising a diameter of any one of about 3.0jim, about 3.5}im, about 4.\\xm,<br>
about 5.0(im, about 5.6nm and about 6.8|im. In a further preferred embodiment, each size<br>
group of beads comprises one or more sub-groups of microspheres each with a fluorescent<br>
label at a range of different intensities. The fluorescent label is TMR and can be applied at<br>
intensities of about 0%, about 4%, about 20% and about 100%.<br>
[00210]	In other embodiments, the headsets of reactants comprises at least group of<br>
beads, each comprising a diameter of any one of about 3.0)im, about 3.S)im, about 3.77fun,<br>
about S.Ofjm, about 5.6^m and about 6.8|im. TMR levels are set such that during bead to<br>
oligo conjugation, oligos with defmed total oligo to fluor-labelled ratios are used to<br>
 <br>
determine discrete TMR groupings. For example, zero levels of TMR have uniabelled<br>
oligos included in conjugation; high levels of TMR have a 3:1 total oligo to fluor-'labelled<br>
ratio of oligos included in conjugation; and medium levels of TMR have from a 150:1 to<br>
an 80:1 total oligo to fluor-labelled ratio of oligos included in conjugation, depending on<br>
bead size.<br>
(002111	In further embodiments, the kits comprises the primers GP5+ and GP6+ and<br>
optionally primers MLC1_F and MLCIR.<br>
[00212]	In yet other embodiments, the kits comprise the primers GP5d2+ and<br>
T7aGP6d+ and optionally primers mlcl_95f and T7amlcl_275r.<br>
[00213]	In still other embodiments, the kits comprise the primers GP5d3+ and<br>
T7aGP6d+* and optionally primers m!cl_95f and T7amlcl_275r.<br>
[00214]	The present invention includes additional embodiments of primer pairs<br>
wherein the individual forward (e.g., GP5+, GP5d2+, etc.) and reverse primers (e.g.,<br>
GP6+, T7aGP6d+) described herein are combined with one another.<br>
[00215]	The kits may also be in the form of solid phase chips or supports,<br>
commonly referred to as biochips. All or part of the reagents used in the subject assay may<br>
be incorporated into a biochip or miniaturized into a nanoassay. Although flow cytometry<br>
is particularly useful in measuring outputs of the subject assay, the biochips may be used to<br>
measure or automate other signals such as those associated with whispering gallery mode<br>
assays.<br>
[00216]	Notwithstanding fluorescent intensities in a preferred aspect of the<br>
multiplexing method, other forms of identification are encompassed by the present<br>
invention. One such alternative method includes whispering gallery mode (WGM)<br>
detection. In this embodiment, a fluorescent marker is incorporated into beads of a subset<br>
or incorporated or bonded to DNA on the surface of the beads. This fluorescent marker<br>
can excite the WGMs with a laser or unfocussed white light source or with filtered<br>
unfocussed white light source.<br>
[00217]	WGMs allow only certain wavelengths of light to be emitted from the<br>
particle. The result of this phenomenon is that the usual broad emission (lO-IOOnm wide)<br>
bands from, for example, a fluorophore become constrained and appear as a series of sharp<br>
peaks corresponding effectively to standing mode patterns of light within the particle. In<br>
accordance with the present invention, it has been determined that the WGM profile is<br>
extremely sensitive to changes at the surface of the microspheroidal particle and that the<br>
 <br>
WGM profile changes when the tnicrospheroidal particle interacts with analytes or molecules within its environment.<br>
[002181	Accordingly, other aspects of the present invention contemplate methods of<br>
detecting an analyte such as an ampltcon from an HPV strain comprising a strain-specific<br>
sequence, said method comprising contacting at least one set of microspheroidal particles<br>
with a sample putatively comprising said analyte, wherein each particle within a set of<br>
microspheroidal particles comprises an optically detectable label and an immobilized<br>
putative binding partner of said analyte (e.g., a primer or probe capable of binding,<br>
capturing or otherwise immobilizing an amplicon from an HPV strain) wherein each<br>
particle set has a defmed WGM profile, wherein binding of said analyte to said<br>
immobilized binding partner results in a change in said WGM profile of said at least one<br>
set of microspheroidal particles which is indicative of the presence of said analyte.<br>
{00219]	The methods of the present invention may be applied to detect modulation<br>
in the WGM profile of a microspheroidal particle wherein said modulation results from detection of binding or other association of molecules in a sample to potential binding particles immobilized to the surface of the microspheroidal particle. Detection of binding reactions between an analyte and its binding partner based on sensitive changes in WGM profiles enables the identification and isolation of the analytes.<br>
[00220]	A feature of the present invention is that the microspheroidal particles may<br>
be excited with a wide range of light sources, facilitating measurement in many different WGM profiles.<br>
[00221]	An "optically detectable label" may be any molecule, atom or ion which<br>
emits fluorescence, phosphorescence and/or incandescence. In some other preferred embodiments of the present invention, the optically detectable label is a fluorophore, which may encompass a range of optically detectable labels such as chemical fluorophores and dyes as well as quantum dots.<br>
[00222]	In some embodiments, the present invention provides at least one<br>
microspheroidal particle comprising a latex or silica particle which is fpm to IOO)im in diameter, labelled with an optically detectable label, such as a fluorophore or quantum dot, the particle further comprising a putative binding partner of an analyte to be detected. An example is a capture nucleic acid molecule capable of binding to an HPV amplicon generated by the amplification using two primers to a conserved region of the HPV genome which flank a strain-specific region. The optically detectable label is detectable at<br>
 <br>
visible wavelengths and the tnicrospheroidal particle exhibits one or more WGM profiles. One or more of the WGM profiles of the mlcrospheroidal particle detectably modulates when analytes interacts with the immobilized binding partner on the particle. Any such change in WGM profile is indicative of the presence of an analyte which has bound to its binding partner.<br>
[00223]	Block oligos can be included in with the beads during hybridization of<br>
single-stranded amplicon to beads. The block oligos have deliberate points of difference from the corresponding bead-probe binding regions of the single-stranded amplicons. TTiere can be a block oligo for each bead-probe in the mix. The block oligos are intended to act as appropriately pitched filters. In the context of a thermal denaturation followed by slow decrease in temperature, most efficient hybridization events occur first. Type-specific amplicons bind to the matching bead-probe according to type early on. The block oligos can bind to and saturate the bead-probes lower down the thermal profile. This can prevent cross-type hybridization events (where between types there are more points of difference at probe binding regions compared with the block oligos) or other non-specific hybridisations such as other amplicons, including primer-dimer products, from occurring. A block oligo can be present for human control as well as the HPV types. Examples of block oligos are described in Table 2 above. The level of similarity between the block oligos and the complementary sequence to the type-specific bead probe region (for which it is designed to block) can be between 40% and 95%. Another example of the level of similarity is between 75% and 85%.<br>
[00224]	Signal oligos can be used as alternatives to the use of directly fiuor labelled<br>
PCR primers for generating signal. Signal oligos are fluor labelled probes that bind to amplicon products at a site different from the bead-probe region to yield a bead-probe/amplicon/signal oligo complex upon hybridization. The advantages of their use would be: to preclude the requirement for fluor-labelling (and excessively handling) PCR primer during production, use of much less fluor-labelled oligo for equivalent sensitivity (and cost benefits), reduction of fiuor background such that higher sensitivity can achieved without wash steps (handling and risk benefits), increased sensitivity in the absence of block oligos due to extra level of'nesting' in the hybridization complex, and potential use of more of PCR product in the 'detection' steps. All of these benefits would facilitate a 'single tube' approach. Examples of signal oligos are described in Table 2 above.<br>
 <br>
[00225]	Alternatively, a double stranded DNA-spccific dye, such as SYBR Green I<br>
dye may be used as a mode of attaching signal to amplicon on bead alongside or instead of<br>
signal oligos.<br>
[00226]	The present invention is further described by the following non-limiting<br>
examples.<br>
EXAMPLE!<br>
HPV diagnosis - DNA isolation and amplification<br>
[00227]	An overview of the DNA extraction protocol used to isolate DNA for the<br>
HPV diagnostic method is shown in Figure 1.<br>
[00228]	As shown in Figure 2, PCR was used to amplify the DNA sample. The<br>
primers GP5d2+ and T7aGP6d+ were used to generate an ampHcon for any HPV strain which was present in the DNA sample. Primer T7aGP6d+ comprised a fluorescent label, specifically A647 to allow later visualization of the amplicon binding to the binding agents. The viral amplicons generated comprised both a conserved region (Y) which is conserved among all the strains of HPV examined and a region which is variable (i.e., strain-specific) between HPV strains, Xn, wherein n represents a variable region associated with each HPV strain. The immobilized binding partners on beads specifically bind to the HPV strain-specific genome.<br>
|00229|	Also, an amplicon from the human subject genomic DNA was also<br>
generated using the mlcl_95f and T7amlcl_275r primers to serve as a control. In this case, primer T7amlcl_275raIso carried an A647 label. EXAMPLE 2<br>
HPV diagnosis - Multiplex detection<br>
[00230]	The amplicons generated in Example 1 were hybridized to an array of<br>
binding agents, each carrying a polynucleotide which is complementary to the variable region of the putative viral amplicon generated fi-om each of HPV strains 6, 11,16, 18, 31, 33, 35, 39, 45, 51, 52, 56, 58, 59, 67 and 68 (X, through Xie). See Table 2 for the nucleotide sequences of the capture nucleic acids immobilized to the beads. Furthermore, the array can optionally comprise a binding agent which comprises a polynucleotide which is complementary to conserved region of the HPV viral amplicons (Y). Finally, a binding agent comprising a polynucleotide which is complementary to the sequence of the human control amplicon is included. The capture nucleic acid may be DNA or RNA. If RNA is used, a reverse transcriptase may be required to generate RNA from the DNA amplicon.<br>
 <br>
[00231]	Each of the binding agents in the array is comprises a microsphere or bead<br>
with a distinct size and distinct intensity of fluorescent (TMR) label. Beads comprising diameters of 3.0nm, 3.5nm, 3.77 \im, 5.0nm, 5,6nm and 6.8fun may be differentiated from each other using flow cytometry, as shown in Figure 4.<br>
(00232]	For each given size of microsphere, a fluorescent label (TMR) was<br>
incorporated at relative intensities of 0%, approximately 150-80; I (medium), and approximately 3:1 (high). These label intensities could be clearly distinguished using flow cytometry. EXAMPLE 3<br>
Comparison of the Multiplex detection method with traditional HPV diagnosis<br>
[00233]	Table 5, below, provides an overview comparing the multiplex HPV<br>
detection method of the present invention with the current histological method for HPV diagnosis.<br>
 <br>
 <br>
 <br>
Agarose gel electrophoretic analysis ofPCR products following amplification from HPV type-specific insert-containing plasmids using various combinations of primers.<br>
(00234)	Five microliters of each product was loaded alongside lanes containing 5<br>
microliters of HyperJadderlV DNA ladder. Reactions were templated with 40000 copies<br>
of type-specific plasmid (numbers in lanes indicate type) and subjected to 40 amplification<br>
cycles using an annealing temperature of 44°C. An 'x' in lanes indicates reactions not<br>
analysed due to evaporative loss during PCR. A'-' in lanes indicates no template controls.<br>
200 or 100 copies of type-specific plasmids along with lOng of Jurkat human genomic<br>
DNA were used to template PCRs employing GBHPVf3+ and GBHPVrl multiplexed with<br>
MLCl_95f and T7aMLCI_275r. Forty-eight amplification cycles were performed using<br>
an annealing temperature of 44°C. Analysis was performed by flow cytometry following<br>
lambda exonuclease digestion and hybridization to PapType probe beads. No template<br>
controls yielded negative results across types. All types tested: 6, 11, 16, 18,31,45,51,<br>
52, 56, 58, 59, 66 and 68 yielded positive results using 200 copies of type-specific<br>
plasmids. All above types except types 51 and 59 yielded positive results using 100 copies<br>
of type-specific plasmids, indicating a very sensitive and relatively unbiased system.<br>
(00235)	When ranges are used herein for physical properties, such as temperature or<br>
number of nucleotides in a sequence, all combinations and subcombinations of ranges and<br>
specific embodiments therein are intended to be included.<br>
 <br>
[00236]	Those skilled in the art will appreciate that the invention described herein is<br>
susceptible to variations and modifications other than those specifically described. It is to be understood that the invention includes all such variations and modifications. The invention also includes all of the steps, features, compositions and compounds referred to or indicated in this specification, individually or collectively, and any and alt combinations of any to or more of said steps or features.<br>
BIBLIOGRAPHY<br>
Bonner and Laskey, Eur. J. Biochem. 46:83, 1974<br>
Chadwicke/fl/., J. Virol. Methods 70:59-70, 1998<br>
Chan and Fox, Rev. Med. Microbiol. 70:185-196, 1999<br>
Com^iXon, Nature 350m~92, 1991<br>
Demidov and Broude (Eds.), "DNA Amplification: Current Technologies and<br>
Applications'\ Horizon Bioscience, 2004 Gearhart et al., vyvyw.emedicine.com/MED/tQpicl037.htm. 2004 Guatelli et al., Proc. Natl Acad Sci. USA 57:1874-1878,1990 Hill.y. Clin. LigandAssay 19A3-51, 1996 Kievitsetal.,y, Virol. Methods 33:273~2S6, 1991 Kuske etal., AppL Environ. Microbiol. 64(7J:2463-2472, 1998 Lizardi et al., Biotechnology 6:1197-1202, 1988 Lyamichev etal., Nat. Biotechnol. 77:292-296, 1999 Marmur and Doty. J. Mol. Biol. 5.T09, 1962 Nelson and Knwetz, Anal. Biochem. 2Q7(l):91-2Q\, 1992 Pawlotsky e/fl/., 7. Virol. Methods 79:227-235,1999 Ryan et al., Mol. Diagn. 4:\35~\44, 1999 Spte], Histochem. Cell Biol. 7 72; 89-113, 1999 Todd et al., J. AIDS Hum. Retroviral 70:S35-S44, 1995<br>
<br>
<br>
<br>
<br>
<br>
<br>
<br>
<br>
<br>
<br>
<br>
1.	A headset for detecting a strain of HPV and/or for differentiating between two or<br>
more strains of HPV, wherein the headset comprises a plurality of families or a plurality of<br>
suhsets of heads having:<br>
(a)	at least two subsets of beads, wherein each subset of heads is physiochemically distinguishable from any other head subset based on at least size; or<br>
(b)	at least one subset of beads having at least two families, wherein the beads in a bead subset are homogeneous in size with respect to each other, and are physiochemically distinguishable into two or more families of beads based on the level of labelling each contains, or<br>
(c)	a combination of said families of beads and said subsets of beads; and wherein:<br>
the beads within each bead family of a subset are each coupled to an optionally-labelled nucleic acid capture probe capable of binding to a HPV strain-specific region of an HPV genome or, optionally, to at least one of a control nucleic acid sequence, provided that any nucleic acid capture probe or control nucleic acid sequence is labelled with the same label as other labelled capture probes or control nucleic acid sequence in the bead subset, and that within any single subset of beads, each family of beads thereof has a different fluorescent intensity; and<br>
each family of beads is specific for the detection of a strain of HPV different from any other bead family in the headset; wherein said headset is capable of identifying one or more specific strains of HPV through analysis of bead size, fluorescent intensity, or sequence discrimination of the at least one bead family or bead subset, using flow cytometry.<br>
2.	The headset of Claim 1, wherein the bead sizes are selected from the group consisting of: 3.0Um, 3.5|Um,4.1(Um, S.Ojim, 5.6(im and 6.8Um.<br>
3.	The headset of Claim 1, wherein the bead sizes are selected from the group consisting of: 3.0nm, 3.5j4m, 3.77nm, S.Ojun, 5.6nm and 6.S\un.<br>
 <br>
4.	The headset of one of Claims 1-3 wherein the labelled beads are labelled with a fluorochrome selected from the group consisting of: hydroxycoumarin, aminocoumarin, methoxycoumarin, cascade blue, Lucifer yellow, NBD, Phyccerythrin (PE), PerCP, allophycocyanin, Hoechst 33342, DAPI, SYTOX Blue, Hoechst 33258, chromomycin A3, mithramycin, YOYO-1, SYTOX green, SYTOX orange, 7-AAD, acridine orange, TOTO-1, To-PRO-1, thiazole orange, TOTO-3, TO-PRO-3, LDS 751. Alexa Fluro-350, Alexa Fluro-430, Alexa Fluro-488, Alexa Fiuro-532, Alexa Fluro-546, Alexa Fluro-555, Alexa Fluro-556, Alexa Fluro-594, Alexa Fluro-633, Alexa Fluro-647, Alexa Fluro-660, Alexa Fluro-680, Alexa Fluro-700 and Alexa Fluro-750, BoDipy 630/650 and BoDipy 650/665, the CY dyes Cy2, Cy3, Cy3.5, Cy5, Cy 5.5 and Cy7, 6-FAM (Fluorescein), PE-Cy5, PE-Cy7, Fluorescein dT, Hexachlorofluorescein (Hex), 6-carboxy-4', 5*-dichloro-2', 7'-dimethpxyfluorescein (JOE), Oregon green dyes 488-X and 514, Rhodamine dyes X-Rhodamine, Lissamine Rhodamine B, Rhodamine Green, Rhodamine Red, and ROX, TRITC, Tetramethylrhodamine (TMR), Carboxytetramethylrhodamine (TAMRA), Tetrachlorofluorescein (JET), Red 6B, FluorX, BODIPY-FL, SYBR Green 1 dye, and Texas Red.<br>
5.	The beadset of Claim 4 wherein the labelled beads are labelled with TMR.<br>
6.	The beadset of Claim 1 comprising to 2 to 17 families of beads.<br>
7.	The beadset of any one of Claims 1 to 6, wherein the beadset comprises 17 families of beads.<br>
8.	The beadset of Claim 1 wherein the two or more HPV strains are selected from strains 6, 11, 16, 18, 31. 33, 35. 39,45, 51, 52, 56, 58, 59, 66 and 68.<br>
9.	The beadset of Claim 1, wherein the nucleic acid capture probe is selected from the list of sequences of Table 2.<br>
 <br>
10.	A method for preparing a headset for use in detecting a strain of HPV and/or<br>
differentiating between two or more strains of HPV, the method comprising selecting a<br>
plurality of families or subsets of beads based on size, fluorescent intensity, or both,<br>
comprising providing:<br>
(a)	at least two subsets of beads, wherein each subset of beads is physiochemically distinguishable from any other bead subset based on at least size; or<br>
(b)	at least one subset of beads having at least two families, wherein the beads in a bead subset are homogeneous in size with respect to each other, and are physiochemically distinguishable into two or more families of beads based on the level of labelling each contains, or<br>
(c)	a combination of said families of beads and said subsets of beads; and wherein:<br>
the beads within each bead family of a subset are each coupled to an optionally* labelled nucleic acid capture probe capable of binding to a HPV strain-specific region of an HPV genome or, optionally, to at least one of a control nucleic acid sequence, provided that any nucleic acid capture probe or control nucleic acid sequence is labelled with the same label as other labelled capture probes or control nucleic acid sequence in the bead subset, and that within any single subset of beads, each family of beads thereof has a different fluorescent intensity; and<br>
each family of beads is specific for the detection of a strain of HPV different from any other bead family in the headset; and<br>
mixing the at least two families or at least two subsets of beads together to produce a headset that permits identification of a specific strain of HPV through analysis of bead size, fluorescent intensity, or sequence discrimination of the at least one bead family or bead subset, using flow cytometry.<br>
11.	A method for diagnosing HPV infection in a human subject, said method<br>
comprising:<br>
(i)       obtaining a biological sample from the human subject which putatively comprises<br>
HPV; (ii)       isolating nucleic acid from said sample;<br>
 <br>
(iii)     amplifying the nucleic acid from said sample using primers which generate an amplicon which is distinct for said anaiyte or a particular strain of said analyte;<br>
(iv)     optionally amplifying a control nucleic acid sequence from the genomic DNA of the human subject;<br>
(v)      optionally effecting labelling of the amplicon(s) recited at steps (iii) and/or (iv);<br>
(vi)     hybridizing the labelled amplicon(s) to a beadset of reactants wherein each member of the beadset comprises a nucleic acid molecule having complementarity to a nucleotide sequence of HPV or a particular strain of HPV or a control nucleotide sequence, bound or otherwise associated with a physiochemically distinguishable substrate; and<br>
(vii)    determining to which of the reactants an amplicon has bound;<br>
wherein the association of an amplicon with a particular reactant is indicative of HPV<br>
infection in the human subject.<br>
12.      A method for determining the risk of a human subject developing a disease<br>
associated with one or more strains of HPV said method comprising:<br>
(i)       obtaining a biological sample from the human subject which putatively comprises<br>
HPV; (ii)      isolating nucleic acid from said sample; (iii)     amplifying the nucleic acid fi'om said sample using primers which generate an<br>
amplicon which is distinct for said analyte or a particular strain of said analyte; (iv)     optionally amplifying a control nucleic acid sequence from the genomic DNA of<br>
the human subject; (v)      optionally effecting labelling of the amplicon(s) recited at steps (iii) and/or (iv); (vi)     hybridizing the labelled amplicon(s) to a beadset of reactants wherein each member<br>
of the beadset comprises a nucleic acid molecule having complementarity to a<br>
nucleotide sequence of HPV or a particular strain of HPV or a control nucleotide<br>
sequence, bound or otherwise associated with a physiochemically distinguishable<br>
substrate; and (vii)    determining to which of the reactants an amplicon has bound; wherein the association of an amplicon with a particular reactant comprising a<br>
 <br>
polynucleotide which is complementary to a strain of HPV associated with a particular disease, is indicative of an increased risk said disease in the subject.<br>
13.	A method for diagnosing HPV infection in a human subject, said method<br>
comprising:<br>
(i)       obtaining a biological sample from the human subject which putatively comprises HPV;<br>
(ii)      isolating nucleic acid from said sample;<br>
(iii)     amplifying the nucleic acid flrom said sample using primers which generate an amplicon which is distinct for said analyte or a particular strain of said analyte;<br>
(iv)     optionally amplifying a control nucleic acid sequence from the genomic DNA of the human subject;<br>
(v)      hybridizing the labelled amplicon(s) to a headset of reactants wherein each member of the headset comprises a nucleic acid molecule having complementarity to a nucleotide sequence of HPV or a particular strain of HPV or a control nucleotide sequence, bound or otherwise associated with a physiochemically distinguishable substrate and wherein the hybridizing occurs in the presence of at least one signal oligonucleotide sequence; and<br>
(vi)     determining to which of the reactants an amplicon has bound;<br>
wherein the association of an amplicon with a particular reactant is indicative of HPV<br>
infection in the human subject.<br>
14.	A method of claim 13, wherein the headset of reactants comprises a plurality of<br>
families or a plurality of subsets of beads having:<br>
(a)	at least two subsets of beads, wherein each subset of beads is physiochemically distinguishable from any other bead subset based on at least size; or<br>
(b)	at least one subset of beads having at least two families, wherein the beads in a bead subset are homogeneous in size with respect to each other, and are physiochemically distinguishable into two or more families of beads based on the level of labelling each contains, or<br>
(c)	a combination of said families of beads and said subsets of beads; and wherein:<br>
 <br>
the beads within each bead family of a subset are each coupled to an optionally-labelled nucleic acid capture probe capable of binding to a HPV strain-specific region of an HPV genome or, optionally, to at least one of a control nucleic acid sequence, provided that any nucleic acid capture probe or control nucleic acid sequence is labelled with the same label as other labelled capture probes or control nucleic acid sequence in the bead subset, and that within any single subset of beads, each family of beads thereof has a different fluorescent intensity; and<br>
each family of beads is specific for the detection of a strain of HPV different from any other bead family in the headset; wherein said headset is capable of identifying one or more specific strains of HPV through analysis of bead size, fluorescent intensity, or sequence discrimination of the at least one bead family or bead subset, using flow cytometry.<br>
15.      A method for determining the risk of a human subject developing a disease associated with one or more strains of HPV said method comprising: (i)       obtaining a biological sample from the human subject which putatively comprises<br>
HPV; (ii)      isolating nucleic acid from said sample;<br>
(iii)     amplifying the nucleic acid from said sample using primers which generate an amplicon which is distinct for said analyte or a particular strain of said analyte; (iv)     optionally amplifying a control nucleic acid sequence from the genomic DNA of<br>
the human subject; (v)      hybridizing the labelled amplicon(s) to a headset of reactants wherein each member of the headset comprises a nucleic acid molecule having complementarity to a nucleotide sequence of HPV or a particular strain of HPV or a control nucleotide sequence, bound or otherwise associated with a physiochemically distinguishable substrate and wherein the hybridizing occurs in the presence of at least one signal oligonucleotide sequence; and (vi)      determining to which of the reactant an amplicon has bound; wherein the association of an amplicon with a particular reactant comprising a polynucleotide which is complementary to a strain of HPV associated with a particular disease, is indicative of an increased risk said disease in the subject.<br>
 <br>
16.	A method of claim 15, wherein the beadset of reactants comprises a plurality of<br>
families or a plurality of subsets of beads having:<br>
(a)	at least two subsets of beads, wherein each subset of beads is physiochemically distinguishable from any other bead subset based on at least size; or<br>
(b)	at least one subset of beads having at least two families, wherein the beads in a bead subset are homogeneous in size with respect to each other, and are physiochemically distinguishable into two or more families of beads based on the level of labelling each contains, or<br>
(c)	a combination of said families of beads and said subsets of beads; and wherein;<br>
the beads within each bead family of a subset are each coupled to an optionally-labelled nucleic acid capture probe capable of binding to a HPV strain-specific region of an HPV genome or, optionally, to at least one of a control nucleic acid sequence, provided that any nucleic acid capture probe or control nucleic acid sequence is labelled with the same label as other labelled capture probes or control nucleic acid sequence in the bead subset, and that within any single subset of beads, each family of beads thereof has a different fluorescent intensity; and<br>
each family of beads is specific for the detection of a strain of HPV different from any other bead family in the beadset; wherein said beadset is capable of identifying one or more specific strains of HPV through analysis of bead size, fluorescent intensity, or sequence discrimination of the at least one bead family or bead subset, using flow cytometry.<br>
17.	A method for diagnosing HPV infection in a human subject, said method<br>
comprising:<br>
(i)       obtaining a biological sample from the human subject which putatively comprises<br>
HPV; (ii)      isolating nucleic acid from said sample; (iii)     amplifying the nucleic acid from said sample using primers which generate an<br>
amplicon which is distinct for said analyte or a particular strain of said analyte; (iv)     optionally amplifying a control nucleic acid sequence from the genomic DNA of<br>
the human subject;<br>
 <br>
(v)       optionally effecting labelling of the amplicon(s) recited at steps (iii) and/or (iv);<br>
(vi)      hybridizing the labelled amplicon(s) to a beadset of reactants wherein each member of the beadset comprises a nucleic acid molecule having complementarity to a nucleotide sequence of HPV or a particular strain of HPV or a control nucleotide sequence, bound or otherwise associated with a physiochemically distinguishable substrate and wherein the hybridizing occurs in the presence of at least one blocking oligonucleotide sequence; and<br>
(vii)    determining to which of the reactants an amplicon has bound;<br>
wherein the association of an amplicon with a particular reactant is indicative of HPV<br>
infection in the human subject.<br>
18.      A method of claim 17, wherein the beadset of reactants comprises a plurality of<br>
families or a plurality of subsets of beads having:<br>
(a)	at least two subsets of beads, wherein each subset of beads is physiochemically distinguishable from any other bead subset based on at least size; or<br>
(b)	at least one subset of beads having at least two families, wherein the beads in a bead subset are homogeneous in size with respect to each other, and are physiochemically distinguishable into two or more families of beads based on the level of labelling each contains, or<br>
(c)	a combination of said families of beads and said subsets of beads; and wherein:<br>
the beads within each bead family of a subset are each coupled to an optionally-labelled nucleic acid capture probe capable of binding to a HPV strain-specific region of an HPV genome or, optionally, to at least one of a control nucleic acid sequence, provided that any nucleic acid capture probe or control nucleic acid sequence is labelled with the same label as other labelled capture probes or control nucleic acid sequence in the bead subset, and that within any single subset of beads, each family of beads thereof has a different fluorescent intensity; and<br>
each family of beads is specific for the detection of a strain of HPV different from any other bead family in the beadset; wherein said beadset is capable of identifying one or more specific strains of HPV through analysis of bead size, fluorescent intensity, or<br>
 <br>
sequence discrimination of the at least one bead family or bead subset, using flow cytometry.<br>
19.	A method for determining the risk of a human subject developing a disease<br>
associated with one or more strains of HPV said method comprising:<br>
(i&gt;       obtaining a biological sample from the human subject which putativety comprises HPV;<br>
(ii)      isolating nucleic acid from said sample;<br>
(iii)     amplifying the nucleic acid from said sample using primers which generate an amplicon which is distinct for said analyte or a particular strain of said analyte;<br>
(iv)     optionally amplifying a control nucleic acid sequence from the genomic DNA of the human subject;<br>
(v)      optionally effecting labelling of the amplicon(s) recited at steps (iii) and/or (iv);<br>
(vi)     hybridizing the labelled amplicon(s) to a headset of reactants wherein each member of the headset comprises a nucleic acid molecule having complementarity to a nucleotide sequence of HPV or a particular strain of HPV or a control nucleotide sequence, bound or otherwise associated with a physiochemically distinguishable substrate and wherein the hybridizing occurs in the presence of at least one blocking oligonucleotide sequence; and<br>
(vii)    determining to which of the reactant an amplicon has bound;<br>
wherein the association of an amplicon with a particular reactant comprising a<br>
polynucleotide which is complementary to a strain of HPV associated with a particular<br>
disease, is indicative of an increased risk said disease in the subject.<br>
20.	A method of claim 19, wherein the headset of reactants comprises a plurality of<br>
families or a plurality of subsets of beads having:<br>
(a)	at least two subsets of beads, wherein each subset of beads is physiochemically distinguishable from any other bead subset based on at least size; or<br>
(b)	at least one subset of beads having at least two families, wherein the beads in a bead subset are homogeneous in size with respect to each other, and are physiochemically distinguishable into two or more families of beads based on the level of labelling each contains, or<br>
(c)	a combination of said families of beads and said subsets of beads; and<br>
 <br>
wherein:<br>
the beads within each bead family of a subset are each coupled to an optionaity-<br>
labelled nucleic acid capture probe capable of binding to a HPV strain-specific region of<br>
an HPV genome or, optionally, to at least one of a control nucleic acid sequence, provided<br>
that any nucleic acid capture probe or control nucleic acid sequence is labelled with the<br>
same label as other labelled capture probes or control nucleic acid sequence in the bead<br>
subset, and that within any single subset of beads, each family of beads thereof has a<br>
different fluorescent intensity; and<br>
each family of beads is specific for the detection of a strain of HPV different from any<br>
other bead family in the headset; wherein said headset is capable of identifying one or<br>
more specific strains of HPV through analysis of bead size, fluorescent intensity, or<br>
sequence discrimination of the at least one bead family or bead subset, using flow<br>
cytometry.<br>
21.      A method for diagnosing HPV infection in a human subject, said method<br>
comprising:<br>
(i)       obtaining a biological sample from the human subject which putatively comprises HPV;<br>
(ii)      isolating nucleic acid from said sample;<br>
(iii)     amplifying the nucleic acid from said sample using primers which generate an amplicon which is distinct for said analyte or a particular strain of said analyte;<br>
(iv)     optionally amplifying a control nucleic acid sequence from the genomic DNA of the human subject;<br>
(v)       hybridizing the labelled amplicon(s) to a headset of reactants wherein each member of the headset comprises a nucleic acid molecule having complementarity to a nucleotide sequence of HPV or a particular strain of HPV or a control nucleotide sequence, bound or otherwise associated with a physiochemically distinguishable substrate and wherein the hybridizing occurs in the presence of at least one signal oligonucleotide sequence and at least one blocking oligonucleotide sequence; and<br>
(vi)     determining to which of the reactants an amplicon has bound;<br>
wherein the association of an amplicon with a particular reactant is indicative of HPV<br>
infection in the human subject.<br>
 <br>
22.	A method of claim 13, wherein the beadset of reactants a plurality of families or a<br>
plurality of subsets of beads having:<br>
(a)	at least two subsets of beads, wherein each subset of beads is physiochemically distinguishable from any other bead subset based on at least size; or<br>
(b)	at least one subset of beads having at least two families, wherein the beads in a bead subset are homogeneous in size with respect to each other, and are physiochemically distinguishable into two or more families of beads based on the level of labelling each contains, or<br>
(c)	a combination of said families of beads and said subsets of beads; and wherein:<br>
the beads within each bead family of a subset are each coupled to an optionally-labelled nucleic acid capture probe capable of binding to a HPV strain-specific region of an HPV genome or, optionally, to at least one of a control nucleic acid sequence, provided that any nucleic acid capture probe or control nucleic acid sequence is labelled with the same label as other labelled capture probes or control nucleic acid sequence in the bead subset, and that within any single subset of beads, each family of beads thereof has a different fluorescent intensity; and<br>
each family of beads is specific for the detection of a strain of HPV different from any other bead family in the beadset; wherein said beadset is capable of identifying one or more specific strains of HPV through analysis of bead size, fluorescent intensity, or sequence discrimination of the at least one bead family or bead subset, using flow cytometry.<br>
23.	A method for determining the risk of a human subject developing a disease<br>
associated with one or more strains of HPV said method comprising:<br>
(i)       obtaining a biological sample from the human subject which putatively comprises<br>
HPV; (ii)      isolating nucleic acid from said sample; (iii)     amplifying the nucleic acid from said sample using primers which generate an<br>
amplicon which is distinct for said analyte or a particular strain of said analyte; (iv)     optionally amplifying a control nucleic acid sequence from the genomic DNA of<br>
the human subject;<br>
 <br>
(v)       hybridizing the labelled amplicon(s) to a headset of reactants wherein each member of the headset comprises a nucleic acid molecule having complementarity to a nucleotide sequence of HPV or a particular strain of HPV or a control nucleotide sequence, bound or otherwise associated with a physiochemically distinguishable substrate and wherein the hybridizing occurs in the presence of at least one signal oligonucleotide sequence and at least one blocking oligonucleotide sequence; and<br>
(vi)     determining to which of the reactant an amplicon has bound;<br>
wherein the association of an amplicon with a particular reactant comprising a<br>
polynucleotide which is complementary to a strain of HPV associated with a particular<br>
disease, is indicative of an increased risk said disease in the subject.<br>
24.      A method of claim 13, wherein the headset of reactants a plurality of families or a<br>
plurality of subsets of beads having:<br>
(a)	at least two subsets of beads, wherein each subset of beads is physiochemically distinguishable from any other bead subset based on at least size; or<br>
(b)	at least one subset of beads having at least two families, wherein the beads in a bead subset are homogeneous in size with respect to each other, and are physiochemically distinguishable into two or more families of beads based on the level of labelling each contains, or<br>
(c)	a combination of said families of beads and said subsets of beads; and wherein:<br>
the beads within each bead family of a subset are each coupled to an optionally-labelled nucleic acid capture probe capable of binding to a HPV strain-specific region of an HPV genome or, optionally, to at least one of a control nucleic acid sequence, provided that any nucleic acid capture probe or control nucleic acid sequence is labelled with the same label as other labelled capture probes or control nucleic acid sequence in the bead subset, and that within any single subset of beads, each family of beads thereof has a different fluorescent intensity; and<br>
each family of beads is specific for the detection of a strain of HPV different from any other bead family in the headset; wherein said headset is capable of identifying one or more specific strains of HPV thiough analysis of bead size, fluorescent intensity, or<br>
 <br>
sequence discrimination of the at least one bead family or bead subset, using flow cytometry.<br>
25.	An oligonucleotide amplification primer pair for targeting oncogenic HPV, wherein<br>
the forward primer of the primer pair is selected from the group consisting of:<br>
TR TTT GTT ACT GTK GTD GAT ACY A;<br>
CAR YTR TTT GTT ACT GTK GTD GAT A;<br>
CAR YTR TTT GTT ACT GTK GTD GA;<br>
AAY CAR YTR TTT GTT ACT GTK GT;<br>
TTT GTT ACT GTK GTD GAT ACY AC HCG;<br>
TTT GTT ACT GTK GTD GAT ACY AC HCG YAG;<br>
GTK GTD GAT ACY AC HCG YAG TAC and<br>
GTD GAT ACY AC HCG YAG TAC HAA; and the reverse primer of the primer pair is selected from the group consisting of:<br>
TGA AAA ATA AAY TGY AAA TCA TAT TCY TCM MCA TG;<br>
CAY ARY TGA AAA ATA AAY TGY AAA TC;<br>
TR CAY ARY TGA AAA ATA AAY TG; and<br>
TR CAY ARY TGA AAA ATA AA; wherein the forward and reverse primers are each optionally conjugated to a heel at the 5' terminus of the primer.<br>
26.	A primer pair according to claim 25, wherein at least one of the forward and reverse primers is conjugated to a heel at the 5' terminus of the primer.<br>
27.	A primer pair of claim 25, wherein the forward primer heel is selected from the group consisting of:<br>
CAATCAGC, ACAAT, GGAACAAT, GGAAC, CAGCTT, ATTACC, CTGTT. /5Phos/CAATCAGC, /5Phos/ACAAT, /5Phos/GGAACAAT, /5Phos/GGAAC, /5Phos/CAGCTT. 5Phos/ATTACC, and/5Phos/CTGTT.<br>
28.	A primer pair of claim 25, wherein the reverse primer heel is selected from the<br>
group consisting of:<br>
ACTCACTATAGG, AATACGACTCACTATAGG, TCTAATACGACTCACTATAGCAATTCTAATACGACTCACTATAGG, /5AmMC6/ACTCACTATAGG,/5AmMC6/AATACGACTCACTATAGG,<br>
 <br>
/5AmMC6/TCTAATACGACTCACTATAGG, and /5 AmMC6/AATTCTAATACGACTCACTATAGG.<br>
29.	An oligonucleotide amplification primer pair of claim 25, containing an optional<br>
heel nucleic acid sequence, wherein the forward primer of the primer pair is selected from<br>
the group consisting of:<br>
CAR YTR TTT GTT ACT GTK GTD GA, optionally having a ACAAT, /5Phos/ACAAT, GGAACAAT or a /5Phos/GGAACAAT conjugated to the 5' terminus of the primer; and<br>
GTD GAT ACY AC HCG YAG TAC HAA, optionally having a CTGTT or /5Phos/CTGTT conjugated to the 5' terminus of the primer.<br>
30.	An oligonucleotide amplification primer pair of claim 25, containing an optional<br>
hee! nucleic acid sequence, wherein the reverse primer of the primer pair is selected from<br>
the group consisting of:<br>
TGA AAA ATA AAY TOY AAA TCA TAT TCY TCM MCA TG, optionally having a ACTCACTATAGG or /5AmMC6/ACTCACTATAGG conjugated to the 5' terminus of the primer,<br>
TR CAY ARY TGA AAA ATA AAY TG, optionally having a TCTAATACGACTCACTATAGG or /5AmMC6/TCTAATACGACTCACTATAGG conjugated to the 5' terminus of the primer; and TR CAY ARY TGA AAA ATA AAY TG, optionally having a TCTAATACGACTCACTATAGG or /5AmMC6/rCTAATACGACTCACTATAGG conjugated to the 5' terminus of the primer.</td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/viewdoc.aspx?id=J0bTY/m23SugnMfQ/pceSw==&amp;amp;loc=egcICQiyoj82NGgGrC5ChA==" target="_blank" style="word-wrap:break-word;">http://ipindiaonline.gov.in/patentsearch/GrantedSearch/viewdoc.aspx?id=J0bTY/m23SugnMfQ/pceSw==&amp;amp;loc=egcICQiyoj82NGgGrC5ChA==</a></p>
		<br>
		<div class="pull-left">
			<a href="278262-absorbent-article-comprising-lotion-composition-comprising-a-cooling-agent.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="278264-absorbent-article-including-barrier-leg-cuff-structure-and-absorbent-core-with-superabsorbent-material.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>278263</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>5913/CHENP/2009</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>53/2016</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>23-Dec-2016</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>19-Dec-2016</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>07-Oct-2009</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>GENERA BIOSYSTEMS LIMITED</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td>1 DALMORE DRIVE, SCORESBY, VICTORIA, 3179,</td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>PARK, DANIEL, J</td>
											<td>22 RUTLAND STREET, CLIFTON HILL, VICTORIA 3068</td>
										</tr>
										<tr>
											<td>2</td>
											<td>KHAN, ZAHEER</td>
											<td>482 GREAT SOUTH ROAD PAPATOETOE, MANUKAU 2025</td>
										</tr>
										<tr>
											<td>3</td>
											<td>POETTER, KARL, F</td>
											<td>25 URQUHART STREET, NORTHCOTE, VICTORIA 3070</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>C12N 1/00</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>PCT/US2008/004441</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td>2008-04-04</td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td>60/910,373</td>
									<td>2007-04-05</td>
								    <td>U.S.A.</td>
								</tr>
								<tr>
									<td>2</td>
									<td>60/910,381</td>
									<td>2007-04-05</td>
								    <td>U.S.A.</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/278263-compositions-and-methods-of-detection by HTTrack Website Copier/3.x [XR&CO'2014], Thu, 04 Apr 2024 22:48:39 GMT -->
</html>
