<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/278270-asparaginases by HTTrack Website Copier/3.x [XR&CO'2014], Thu, 04 Apr 2024 22:46:23 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 278270:ASPARAGINASES</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">ASPARAGINASES</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>The invention relates to new asparaginases having improved properties, preferably improved thermotolerance, such as improved activity at high temperatures and/or improved thermostability. The invention also relates to DNA sequences encoding such improved asparaginases, their production in a recombinant host cell, as well as methods of using the asparaginases, in particular for reduction of acrylamide in foods. The invention furthermore relates to methods of generating and preparing asparaginase variants having improved properties.</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td>ASPARAGINASES<br>
<br>
<br>
Field of the Invention<br>
<br>
The invention relates to new asparaginases having improved properties, preferably improved thermotolerance, such as improved activity at high temperatures and/or improved thermostability. The invention also relates to DNA sequences encoding such improved asparaginases, their production in a recombinant host cell, as well as methods of using the asparaginases, in particular for reduction of acrylamide in foods. The invention furthermore relates to methods of generating and preparing asparaginase variants having improved properties.<br>
<br>
<br>
Background of the Invention<br>
<br>
It is known that acrylamide is formed in several food materials during heating to high temperatures. The acrylamide formation has been ascribed to a Maillard reaction wherein asparagine is one of the reactants. It is well known that acrylamide formation in heated food products may be reduced by a treatment reducing the amount of asparagine in the food materials, such as by subjecting the food materials to the action of the enzyme asparaginase (see e.g. WO2004/026042 (Frito-Lay North America, Inc.)).<br>
<br>
A number of microbial asparaginases have been identified; see e.g. WO2004/030468 (DSM) disclosing the sequence of an asparaginase derived from Aspergillus niger, or WO2004/032648 (Novozymes A/S) disclosing sequences of asparaginases derived from Aspergillus oryzae and Penicillium citrinum. WO2004/032648 also mentions the amino acid sequences of asparaginases from Aspergillus fumigatus and Aspergillus nidulans. The amino acid sequence of an asparaginase from Aspergillus terreus can be obtained from the UniProt database (accession no. qOcwjl).<br>
<br>
The amino acid sequence and the crystal structure of an L-asparaginase from Erwinia chrysanthemi have been described (Jacek Lubkowski, Miroslawa Dauter, Khosrow Aghaiypour, Alexander Wlodawera and Zbigniew Dauter (2003) Atomic resolution structure of Erwinia chrysanthemi L-asparaginase. Acta Cryst. D, 59, 84-92).<br>
<br>
A method for designing proteins with improved thermal stability using 3-isopropylmalate dehydrogenase from Thermus thermophilus as a model enzyme has been described (Watanabe, K. et al. (2006) J. Mol. Biol. 355, 664-674).<br>
<br>
For some applications, asparaginases having improved properties are desired, such as asparaginases having improved thermotolerance, e.g. improved thermostability or improved  activity at high temperatures.<br>
<br>
It is an object of the present invention to provide alternative asparaginases, in particular novel asparaginases having improved properties. Such improved asparaginases are suitable for use, e.g. in the production of food products.<br>
<br>
Summary of the Invention<br>
<br>
The present inventors have modeled the three-dimensional structure of an asparaginase from Aspergillus oryzae based on the published structure of a homologous enzyme from Erwinia chrysanthemi. Based on the modeled structure, the inventors have identified amino acid residues of relevance for improving the properties of the asparaginase, especially the thermotolerance.<br>
<br>
Further, the present inventors have predicted an inferred ancestral asparaginase sequence and from this sequence identified further amino acid residues of relevance for improving the properties of an asparaginase, especially the thermotolerance.<br>
<br>
Based on such structural and functional considerations, asparaginase variants were constructed having modified amino acid residues at the identified positions and having altered physiochemical properties, especially improved relative activity at high temperatures and/or improved thermostability.<br>
<br>
Accordingly, the present invention relates to a method for preparing a polypeptide comprising:<br>
(a)	providing an amino acid sequence of a parent polypeptide having asparaginase activity;<br>
(b)	selecting at least one amino acid residue at a position in the sequence which corresponds to any of positions 54, 57, 68-74, 82-86, 88, 93-96, 102, 107, 111, 113, 115, 137, 139, 164, 165, 172, 176, 184-186, 194, 196, 201, 206, 209, 212, 214, 215, 219, 220, 223, 224, 226, 228, 231, 235, 246, 249, 255, 260, 262, 264, 266, 271, 275, 278-288, 290, 299, 306, 307, 309-321, 323, 325, 327-342, 349, 351, 353, 356-363, 365, 366 and/or 375 in SEQ ID NO: 1;<br>
(c)	modifying the sequence by substituting or deleting the selected amino acid residue or inserting one or more amino acid residues adjacent to the selected amino acid residue;<br>
(d)	producing a variant polypeptide having the modified sequence;<br>
(e)	testing the variant polypeptide for asparaginase activity and thermotolerance; and<br>
(f)	selecting a variant polypeptide having asparaginase activity and higher thermotolerance compared to the parent polypeptide.<br>
<br>
<br>
The invention also relates to thermotolerant asparaginases, which may be obtained by such method.<br>
<br>
 Accordingly, the present invention relates to a polypeptide which:<br>
(a)	has asparaginase activity;<br>
(b)	has at least 70% identity to amino acids 80 to 378 of SEQ ID NO: 1; and<br>
(c)	shows a residual asparaginase activity after heat treatment of at least 50% of the asparaginase activity without heat treatment, where heat treatment is incubation at pH 6 at a temperature of at least 64°C for 20 minutes.<br>
<br>
The present invention also relates to a polypeptide which:<br>
(a)	has asparaginase activity;<br>
(b)	has at least 70% identity to amino acids 80 to 378 of SEQ ID NO: 1; and<br>
(c)	shows an asparaginase activity at pH 6 which is at least 25% higher at 65°C than at 37°C.<br>
<br>
In other aspects, the invention relates to novel asparaginases comprising differences in the amino acid sequence compared to SEQ ID NO: 1 at positions identified as being of relevance for improving the thermotolerance of the enzyme.<br>
<br>
Accordingly, the present invention in one aspect relates to a polypeptide which:<br>
(a)	has asparaginase activity;<br>
(b)	has at least 80% identity to amino acids 80 to 378 of SEQ ID NO: 1; and<br>
(c)	comprises an amino acid difference compared to SEQ ID NO: 1 at a position<br>
corresponding to any of positions 54, 57, 68-74, 82-86, 88, 93-96, 102, 107, 111, 113, 115,<br>
137, 139, 164, 165, 172, 176, 184-186, 194, 196, 201, 206, 209, 212, 214, 215, 219, 220,<br>
223, 224, 226, 228, 231, 235, 246, 249, 255, 260, 262, 264, 266, 271, 275, 278-288, 290,<br>
299, 306, 307, 309-321, 323, 325, 327-342, 349, 351, 353, 356-363, 365, 366 and/or 375 in<br>
SEQ ID NO: 1.<br>
<br>
In another aspect, the invention relates to a polypeptide which:<br>
(a)	has asparaginase activity;<br>
(b)	has at least 60% identity to amino acids 80 to 378 of SEQ ID NO: 1; and<br>
(c)	comprises an amino acid difference compared to SEQ ID NO: 1 at a position<br>
corresponding to any of positions 71-74, 82, 88, 111, 113, 137, 164, 176, 196, 206, 223, 226,<br>
231, 246, 266, 275, 278-282, 288, 309, 312, 329-332, 341, 357-359 and/or 363 in SEQ ID<br>
NO: 1.<br>
<br>
In another aspect, the invention relates to a polypeptide which:<br>
(a)	has asparaginase activity;<br>
(b)	has at least 60% identity to amino acids 80 to 378 of SEQ ID NO: 1; and<br>
(c) comprises at least one of the following amino acid substitutions compared to SEQ ID NO: 1: 70H/K, 137S, 164D, 1961, 201Q, 278H/Q, 283C, 290V, 307A, 312Y, 334F, 336C/G/L, 337F/I, 366P and/or 375T; wherein each position corresponds to a position of amino acids 1 to 378 of SEQ ID NO: 1.<br>
<br>
In another aspect, the invention relates to a polypeptide which:<br>
(a)	has asparaginase activity;<br>
(b)	has at least 50% identity to amino acids 80 to 378 of SEQ ID NO: 1; and<br>
(c)	comprises an amino acid difference compared to SEQ ID NO: 1 at a position<br>
corresponding to any of positions 71-73, 82, 111, 137, 164, 176, 206, 223, 226, 231, 246,<br>
266, 275, 279, 281, 288, 312, 329-332, 341, 357-359 and/or 363 in SEQ ID NO: 1.<br>
<br>
And in yet another aspect, the invention relates to a polypeptide which:<br>
(a)	has asparaginase activity;<br>
(b)	has at least 50% identity to amino acids 80 to 378 of SEQ ID NO: 1; and<br>
(c)	comprises at least one of the following amino acid substitutions compared to SEQ ID NO: 1: 70H, 137S, 164D, 1961, 201Q, 278H/Q, 283C, 290V, 307A, 312Y, 334F, 336C/G/L, 337F, 366P and/or 375T; wherein each position corresponds to <br>
a position of amino acids 1 to 378 of SEQ ID NO: 1.<br>
<br>
The amino acid differences may be obtained by modifying a parent sequence, e.g., by site-directed mutagenesis. However, they may also be found in naturally occurring polypeptides, which are born with such amino acid differences compared to SEQ ID NO: 1.<br>
<br>
The present invention also relates to isolated nucleic acid sequences encoding the asparaginases and to nucleic acid constructs, expression vectors, and host cells comprising the nucleic acid sequences as well as methods for producing and using the asparaginases.<br>
<br>
In a still further aspect, the present invention provides a method of constructing a variant of a parent asparaginase, wherein the variant has at least one altered property as compared to the parent asparaginase, which method comprises:<br>
(a)	providing a structure of the parent asparaginase;<br>
(b)	analyzing the structure of the parent asparaginase to identify a structural part comprising at least one amino acid residue, which is of relevance for altering said property;<br>
(c)	constructing a variant of the parent asparaginase comprising a modification of at least one amino acid residue identified in (b) so as to alter said property; and<br>
(d)	testing the resulting asparaginase variant for said property.<br>
 <br>
<br>
In a still further aspect, the present invention provides a method of constructing a variant of a parent asparaginase, wherein <br>
the variant has at least one altered property as compared to the parent asparaginase, which method comprises:<br>
(a)	providing a sequence of the parent asparaginase;<br>
(b)	aligning the sequence of the parent asparaginase with an inferred ancestral asparaginase sequence to identify at least one amino acid residue, which is of relevance for altering said property;<br>
(c)	constructing a variant of the parent asparaginase comprising a modification of at least one amino acid residue identified in (b) so as to alter said property; and<br>
(d)	testing the resulting asparaginase variant for said property.<br>
<br>
Brief Description of the Figures<br>
<br>
Figure 1 is a multiple alignment of asparaginases derived from Aspergillus oryzae, Aspergillus fumigatus, Aspergillus nidulans, Penicillium citrinum, Aspergillus terreus and Aspergillus niger.<br>
<br>
Figure 2 shows Molecular Dynamics simulation data for three simulation temperatures (300K, 400K, 500K) and for all four monomers of the modeled tetrameric structure of the asparaginase from A. oryzae. Residue ranges that have high rmsd (root mean square deviation) values (relative to the average value for that temperature) are identified as mobile regions and thus targets of interest for stabilizing mutations.<br>
<br>
Figure 3 shows in a model structure of the Aspergillus oryzae asparaginase that the tetrameric asparaginases have 3 axes of 2-fold symmetry.<br>
<br>
Figure 4 shows a parameter file for the Molecular Dynamics Simulation.<br>
<br>
Figure 5 shows a parameter file for the Molecular Dynamics Simulation.<br>
<br>
Figure 6 shows a phylogenetic tree inferred by the maximum likelihood method applied in Example 6.<br>
<br>
Detailed Description of the Invention <br>
<br>
Novel Asparaginases<br>
<br>
The polypeptides of the present invention have asparaginase activity. Preferably, they have amino acid sequences, which can be aligned with any of SEQ ID NOs: 1-6.<br>
<br>
SEQ ID NO: 1 shows the amino acid sequence of an asparaginase from Aspergillus oryzae<br>
and the numbering system used in the context of the present invention. The amino acid sequence and the DNA sequence encoding it were previously published as SEQ ID NO: 2 and SEQ ID NO: 1 of WO2004/032648.<br>
<br>
SEQ ID NO: 2 shows the amino acid sequence of an asparaginase from Aspergillus niger. The amino acid sequence and the DNA sequence encoding it were previously published as SEQ ID NO: 3 and SEQ ID NO: 2 of WO2004/030468.<br>
<br>
SEQ ID NO: 3 shows the amino acid sequence of an asparaginase from Aspergillus fumigatus (previously published as SEQ ID NO: 6 of WO2004/032648). SEQ ID NO: 4 shows the amino acid sequence of an asparaginase from Aspergillus niduians (previously published as SEQ ID NO: 4 of WO2004/032648). SEQ ID NO: 5 shows the amino acid sequence of an asparaginase from Periicillium citrinum (previously published as SEQ ID NO: 12 of WO2004/'032648). SEQ ID NO: 6 shows the amino acid sequence of an asparaginase from Aspergillus terreus (obtained from the UniProt database, accession no. qOcwjl).<br>
<br>
The polypeptides of the present invention may be partly or completely post-translationally processed. For instance, they may be N-terminally truncated at different positions, so that different N-terminal sequences will be found. The wild type Aspergillus oryzae asparaginase, when overexpressed in Aspergillus oryzae, has been found to be heterogeneously processed such that at least four N-terminal sequences were found in a purified sample, corresponding to polypeptides being truncated to amino acids 27-378, 30-378, 75-378 or 80-378. The polypeptides of the present invention may possibly be truncated at corresponding positions, or they may be truncated at other positions. The polypeptides of the present invention may thus, e.g., be truncated immediately before the position corresponding to any of positions 27, 30, 75 or 80 of SEQ ID NO: 1. In this context, what is meant by 'immediately before' is that truncation takes place at the N-terminal side of the position mentioned. The polypeptides of the present invention may thus have an N-terminal end corresponding, e.g., to any of positions 27, 30, 75 or 80 of SEQ ID NO: 1.<br>
<br>
The polypeptides of the present invention may have an amino acid sequence having more than 50% identity to SEQ ID NO: 1, or a fragment thereof, preferably more than 60%, such as more than 70% or 80%, particularly more than 90%, especially more than 95%, e.g. more than 98%. The fragment of SEQ ID NO: 1 may, e.g., consist of amino acids 27-378, 30-378, 75-378 or 80-378.<br>
<br>
In general, for the purpose of the present invention, when referring to a fragment of a polypeptide, what is meant is a polypeptide or chain of amino acids consisting of at least 100, such as at least 150 or at least 200 or 300, amino acids. A preferred fragment in the context of the present invention comprises or consists of the sequence of amino acids from the position corresponding to position 80 of SEQ ID NO: 1 to the C-terminal, e.g. amino acids 80-378 of SEQ ID NO: 1, amino acids 80-378 of SEQ ID NO: 2, amino acids 80-374 of SEQ ID NO: 3, amino acids 80-378 of SEQ ID NO: 4, amino acids 80-379 of SEQ ID NO: 5, or amino acids 80-375 of SEQ ID NO: 6.<br>
<br>
In a preferred embodiment, the polypeptides of the present invention have an amino acid sequence having 1-50, such as 1-40, 1-30, 1-20 or 1-10 amino acid differences compared to SEQ ID NO: 1, or a fragment thereof.<br>
<br>
In another aspect, the polypeptides of the present invention have an amino acid sequence having more than 50% identity to SEQ ID NO: 2, or a fragment thereof, preferably more than 60%, such as more than 70% or 80%, particularly more than 90%, especially more than 95%, e.g. more than 98%.<br>
<br>
In yet another aspect, the polypeptides of the present invention have an amino acid sequence having more than 50% identity to any of SEQ ID NOs: 3-6, or a fragment thereof, preferably more than 60%, such as more than 70% or 80%, particularly more than 90%, especially more than 95%, e.g. more than 98%.<br>
<br>
In a preferred aspect, the polypeptides of the present invention are eukaryotic, i.e. derived from, obtained from or originating from a eukaryotic organism. In a more preferred aspect, the polypeptides are derived from, obtained from or originating from Aspergillus.<br>
<br>
The polypeptides of the present invention may show a high thermotolerance, e.g. they may have a high thermostability or a high relative asparaginase activity at high temperature.<br>
<br>
In one aspect, the polypeptides of the present invention may be thermostable or have a high thermostability. The thermostability may be determined as the residual asparaginase activity after heat treatment divided by the asparaginase activity without heat treatment. Heat treatment may be incubation at pH 6 or around pH 6 at high temperature for, e.g., 10, 20, 30 or 40 minutes. The asparaginase activity without heat treatment may in this context be determined as the asparaginase activity of a sample which has been incubated at 4'C in the same buffer and for the same time as the sample which is heat treated, or it may be the asparaginase activity before heat treatment.<br>
<br>
The polypeptides of the present invention may show a residual asparaginase activity of at least 90%, such as at least 80%, <br>
at least 70%, at least 60%, at least 50% or at least 40%, after incubation at pH 6 at high temperature for a period of time, e.g. 20 minutes, compared to the asparaginase activity without heat treatment.<br>
<br>
High temperature in the context of the present invention may mean, e.g., 55°C, 58°C, 60°C, 61°C, eZ^C, 63°C, 64°C, 65°C, 66°C, 67°C, 68°C, 69°C, 70°C, 72°C or75°C.<br>
<br>
The asparaginase activity may be determined by any method known in the art. It may be determined by incubating the <br>
enzyme with L-asparagine and hydroxylamine in a potassium phosphate buffer at pH 6 for 20 minutes followed by the coupling reaction with FeCl2 and measuring A490, as described in Example 4. Incubation may be at any suitable temperature, e.g. 55°C.<br>
<br>
In another aspect, the polypeptides of the present invention may have a high relative asparaginase activity at high temperature compared to at a reference temperature, e.g., 37°C, 40°C, 45°C or 50°C. The asparaginase activity at high temperature and, e.g., 37°C may be determined as described above, where the incubation with asparagine is performed at high temperature and 37°C, respectively. The asparaginase activity at high temperature divided by the activity at 37°C may be at least 110%, preferably at least 120%, such as at least 125%, 130%, 140%, 150%, 170% or 200%, more preferably at least 250%, such as at least 300%, and even more preferably at least 500% or at least 700%.<br>
<br>
In some aspects, the polypeptides of the present invention comprise an amino acid difference compared to a reference sequence, e.g. SEQ ID NO: 1, in one or more specified positions. Such positions may be identified by a method of the present invention. An amino acid difference in the context of the present invention may be deletion or substitution of the amino acid residue at the specified position or insertion of one or more additional amino acid residues adjacent to the amino acid residue <br>
at that position. Adjacent to means at either side, i.e. the insertion may be at the N-terminal or the C-terminal side of the amino acid residue at that position, i.e. immediately before or after the specified position. Substitution may mean substitution to another naturally occurring amino acid.<br>
<br>
The amino acid difference may be naturally occurring, i.e. found in a naturally occurring wild-type asparaginase being less than 100% identical to SEQ ID NO: 1, or it may be introduced, e.g. by protein engineering, such as by site-directed mutagenesis, e.g. performed to improve the properties of a parent asparaginase.<br>
<br>
The polypeptides of the present invention may be variants of a parent polypeptide, in which case an amino acid difference <br>
may refer to a difference in the amino acid sequence at a specific position in the variant, compared to the parent sequence. In that case, the amino acid difference may also be referred to as an amino acid modification.<br>
<br>
The polypeptides of the invention, comprising amino acid differences at specified positions compared to SEQ ID NO: 1, or compared to a parent polypeptide, may comprise additional amino acid differences in addition to the amino acid differences described herein.<br>
<br>
Parent Asparaginase<br>
<br>
The parent polypeptide according to the present invention may be an asparaginase classified as EC 3.5.1.1 according to Enzyme Nomenclature (available at http://www.chem.qmul.ac.uk/iubmb/enzyme). It may be a eukaryotic asparaginase, e.g. a fungal asparaginase, such as a filamentous fungal asparaginase, e.g. native to a strain of Aspergillus, in particular Aspergillus oryzae or Aspergillus niger.<br>
<br>
The parent asparaginase may have an amino acid sequence which is at least 50% (particularly at least 70%, at least 80%, at least 85%, at least 90%, at least 95% or at least 98%) homologous to any of SEQ ID NO: 1, SEQ ID NO: 2, SEQ ID NO: 3, SEQ ID NO: 4, SEQ ID NO: 5 or SEQ ID NO: 6. The parent asparaginase may particularly be one that can be aligned with any of these sequences. In a preferred aspect, the parent asparaginase may have the amino acid sequence of any of SEQ ID NO: 1, SEQ ID NO: 2, SEQ ID NO: 3, SEQ ID NO: 4. SEQ ID NO: 5 or SEQ ID NO: 6 or homologues of any of these. In a more preferred aspect, the parent asparaginase may have the amino acid sequence of SEQ ID NO: 1 or SEQ ID NO: 2 or homologues of any of these, preferably SEQ ID NO: 1.<br>
<br>
Nomenclature for Amino Acid Differences<br>
<br>
The specification and claims refer to amino acids by their one-letter codes. A particular amino acid in a sequence is identified by its one-letter code and its position, e.g. M1 indicates Met (methionine) at position 1, i.e. at the N-terminal.<br>
<br>
The nomenclature used herein for defining substitutions is basically as described in WO 92/05249. Thus, G82P indicates substitution of G (Gly) at position 82 with P (Pro). D223N/L indicates substitution of D (Asp) at position 223 with N (Asn) or L (Leu). A plus-sign (+) between substitutions, e.g. 137S+228V means "and", i.e. that these two single substitutions are combined in one and the same asparaginase.<br>
<br>
A substitution embraces a substitution into any one of the other nineteen natural amino acids, or into other amino acids, such <br>
as non-natural amino acids. For example, a substitution of amino acid T in position 71 includes each of the following substitutions: 71A, 71C, 71D, 71E, 71F, 71G, 71H, 711, 71K, 71L, 71M, 71N, 71P, 71Q, 71R, 71S, 71V, 71W, and 71Y. These substitutions can also be designated T71A, T71C, T71D, etc. The same applies by analogy to each and every amino acid difference or amino acid substitution mentioned herein, to specifically include substitution to any other amino acid.<br>
<br>
An insertion at a specific position may mean the insertion of one or more additional amino acid residues adjacent to the amino acid residue at that position. Adjacent to may in this context mean at either side, i.e., at the N-terminal side or at the C-terminal side, i.e., before or after the amino acid residue present at that position in the reference sequence.<br>
<br>
Insertion of an additional amino acid residue, e.g. Lys, at a specific position, e.g. after Gly at position 82, is designated, e.g., G82GK.<br>
<br>
When designating a specific amino acid difference, the amino acid residue in the reference sequence may or may not be specified. For example, a substitution 961 indicates a substitution at the position corresponding to position 96 of e.g. SEQ ID NO: 1 to I (lie) irrespective of which amino acid is present at position 96 in the reference sequence, if the reference sequence is SEQ ID NO: 1, a substitution 961 may also be designated V96I.<br>
<br>
<br>
Homology and Alignment<br>
<br>
For purposes of the present invention, the alignment of two amino acid sequences can be determined by using the Needle program from the EMBOSS package (Rice, P. Longden, I. and Bleasby, A (2000) EMBOSS: The European Molecular Biology Open Software Suite. Trends in Genetics 16, (6) pp276—277; http://emboss.org) version 2.8.0. The Needle program implements the global alignment algorithm described in Needleman, S. B. and Wunsch, C. D. (1970) J. Mol. Biol. 48, 443-453. The substitution matrix used is BLOSUM62, gap opening penalty is 10, and gap extension penalty is 0.5.<br>
<br>
The degree of identity between two amino acid sequences is calculated as the number of exact matches in an alignment of the two sequences, divided by the length of the shortest of the two sequences. The result is expressed in percent identity.<br>
<br>
For the purpose of the present invention, an alignment of two or more sequences can be used to identify in one sequence the position corresponding to (i.e. being equivalent to) a specific position in another sequence, e.g. a reference sequence, e.g. SEQ ID NO: 1. If a position corresponding to a specific position of SEQ ID NO: 1 is to be identified in any of SEQ ID NOs: 2-6, the alignment shown in Figure 1 can be used. 1. If a position corresponding to a specific position of SEQ ID NO: 1 is to be identified in any other homologous sequence, an alignment between SEQ ID NO: 1 and the homologous sequence is to be made as described above, and from this alignment each amino acid residue in the homologous sequence can be assigned a number corresponding to its equivalent amino acid residue in SEQ ID NO: 1. In this context, a homologous sequence is a sequence which can be aligned with SEQ ID NO: 1.<br>
<br>
For the vast majority of amino acid residues in the polypeptides of the invention, and/or for use according to the invention, it is possible to directly and unambiguously assign an amino acid residue in the sequence of amino acids 1 to 378 of SEQ ID NO; 1 to which it corresponds. The only exception is amino acid residues which are additional to amino acids 1 to 378 of SEQ ID NO: 1. For example, it can be seen in Figure 1 that the amino acid sequence from A. niger has additional amino acids S, D, T and A between S338 and 8339 of the A. oryzae sequence, where the numbering is according to SEQ ID NO: 1.<br>
<br>
In the context of the present application, the amino acid residues of the polypeptides of the invention may be numbered according to SEQ ID NO: 1. If the polypeptides of the invention are variants of a parent polypeptide, the amino acid residues of the parent may also be numbered according to SEQ ID NO: 1, so that the position of each amino acid residue may be referred to by the number of its corresponding (equivalent) amino acid in SEQ ID NO: 1.<br>
<br>
<br>
Methods for Designing Novel Asparaginase Variants<br>
<br>
In one aspect, the present invention relates to a method of constructing a variant of a parent asparaginase, wherein the variant has at least one altered property as compared to the parent asparaginase.<br>
<br>
In one embodiment, the invention relates to a method of constructing a variant of a parent asparaginase, wherein the variant has at least one altered property as compared to the parent asparaginase, which method comprises:<br>
(a)	providing a structure of the parent asparaginase;<br>
(b)	analyzing the structure of the parent asparaginase to identify a structural part comprising at least one amino acid residue, which is of relevance for altering said property;<br>
(c)	constructing a variant of the parent asparaginase comprising a modification of at least one amino acid residue identified in (b) so as to alter said property; and<br>
(d)	testing the resulting asparaginase variant for said property.<br>
<br>
In another embodiment, the invention relates to a method of constructing a variant of a parent<br>
asparaginase, wherein the variant has at least one altered property as compared to the<br>
parent asparaginase, which method comprises: <br>
(a)	providing a sequence of the parent asparaginase;<br>
(b)	aligning the sequence of the parent asparaginase with an inferred ancestral asparaginase sequence to identify at least one amino acid residue, which is of relevance for altering said property;<br>
(c)	constructing a variant of the parent asparaginase comprising a modification of at least one amino acid residue identified in (b) so as to alter said property; and<br>
(d)	testing the resulting asparaginase variant for said property.<br>
<br>
The parent asparaginase to be used in a method of the present invention may have an amino acid sequence which is at least 50% identical to SEQ ID NO: 1 or a fragment thereof, preferably at least 60%, 70% or 80% identical, more preferably at least 90% identical, such as at least 95% or at least 98% identical. The fragment of SEQ ID NO; 1 may, e.g., consist of amino acid residues 50-378 or 80-378 of SEQ ID NO: 1.<br>
<br>
In another aspect, the parent asparaginase may have an amino acid sequence which is at least 50% identical to any one of SEQ ID NOs: 2-6 or a fragment of any one of these sequences, preferably at least 60%, 70% or 80% identical, more preferably at least 90% identical, such as at least 95% or at least 98% identical.<br>
<br>
The modification of at least one amino acid residue is typically accomplished by suitable modifications of a DNA sequence encoding the parent polypeptide in question.<br>
<br>
The at least one altered property in the above method of the present invention may be, e.g., stability, such as temperature stability or pH dependent stability; temperature or pH dependent activity; specific activity; substrate specificity; higher or lower optimum temperature; higher or lower temperature of inactivation; or an increased ability to reduce acrylamide formation during the production of foods.<br>
<br>
The at least one altered property may preferably be a higher thermotolerance as compared to the thermotolerance of the parent asparaginase, such as a higher thermostability or a higher relative asparaginase activity at high temperature as compared to the parent asparaginase. More preferably, the at least one altered property may be a higher relative asparaginase activity than the parent enzyme at pH 6 at 65°C compared to 37°C. Or it may be a higher asparaginase activity than the parent enzyme at pH 6 at 65°C. Or it may be a higher asparaginase activity than the parent enzyme after incubation at around pH 6 at a temperature of at least 64°C for 20 minutes.<br>
<br>
Thermostability and relative asparaginase activity at high temperature may be determined as described above in the section 'Novel Asparaginases'. <br>
<br>
<br>
Methods for Designing Novel Asparaginase Variants Based on 3D Structure<br>
<br>
The structure of the parent asparaginase to be applied in a method according to the invention may be provided by any means known in the art. It may be that the structure has already been determined and is known in the art, in which case it may be provided by referring to the literature. Otherwise, the structure may be determined by X-ray diffraction or NMR. Or it may be provided by modeling the parent asparaginase to be used according to the method of the invention on the structure of another asparaginase, the structure of which has been previously determined or modeled.<br>
<br>
The published three-dimensional structure of the L-asparaginase from Erwinia chrysanttiemi (Jacek Lubkowski, Miroslawa Dauter, Khosrow Aghaiypour, Alexander Wlodawera and Zbigniew Dauter (2003) Atomic resolution structure of Erwinia clirysanthemi L-asparaginase. Acta Cryst. D, 59, 84-92) was used to model the three-dimensional structure of the asparaginase from Aspergillus oryzae. The structure of other asparaginases may be modeled in an analogous way.<br>
<br>
The 3D model of Aspergillus oryzae asparaginase was built using the "Nest" homology mod¬elling tool (Petrey, D., Xiang, X., Tang, C. L., Xie, L., Gimpelev, M., Mitors, T., Soto, C. S., Goldsmith-Fischman, S., Kernytsky, A., Schlessinger, A., Koh, I. Y. Y., Alexov, E. and Honig, B. (2003) Using Multiple Structure Alignments, Fast Model Building, and Energetic Analysis in Fold Recognition and Homology Modeling. Proteins: Struc, Func. and Genet. 53:430-435). The A-chain structure of the Envinia clirysanthemi asparaginase was used as a template. The alignment below was used as a basis for the model building. The first 49 amino acid residues of the Aspergillus oryzae asparaginase sequence were not included in the model. The complete tetrameric structure was modeled by aligning a copy of the Asper-gillus oryzae asparaginase model to each of the monomers in the Erwinia chrysanthemi as-paraginase structure.<br>
<br>
&gt;P1 ;Ao_asparaginase sequence:Ao_asparaginase::::::::<br>
TLPNVTIFATG<br>
GTIAGSSADNTATTGYKAGAVGIQTLIDAVPEMLNVANVAGVQVTNVGSPDITSDILLRL SKQINEVVCNDPTMAGAVVTHGTDTLEESAFFLDATVNCRKPVVIVGAMRPSTAISADGP LNLLQSVTVAASPKARDRGALIVMNDRIVSAFYASKTNANTVDTFKAIEMGNLGEVVSNK PYFFYPPVKP-TGKTEVDIRNITSIPRVDILYSYEDMHNDTLYSAIDNGAKGIVIAGSGS GSVSTPFSAAMEDITTKHNIPIVASTRTGNGEVPSSAESSQIASGYLNPAKSRVLLGLLL<br>
 <br>
AQGKSIEEMRAVFERIGVA <br>
&gt;P1;1o7j structure:1o7j::A::A:<br>
KLPNIVILATG<br>
GTIAGSAATGTQTTGYKAGALGVDTLINAVPEVKKLANVKGEQFSNMASENMTGDWLKL<br>
SQRVNELLARD-DVDGVVITHGTDTVEESAYFLHLTVKSDKPVVFVAAMRPATAISADGP<br>
MNLLEAVRVAGDKQSRGRGVMVVINDRIGSARYITKTNASTLDTFRANEEGYLGVIIGNR<br>
lYYQNRIDKLHTTRSVFDVRGLTSLPKVDILYGYQDDPEYLYDAAIQHGVKGIVYAGMGA<br>
GSVSVRGIAGMRKALEKG-VVVMRSTRTGNGIVPPDEELPGLVSDSLNPAHARILLMLAL<br>
TRTSDPKVIQEYFHTY	<br>
<br>
In one embodiment of the method of the present invention, the parent asparaginase is modeled on the known structure of the L-asparaginase from Erwinia chrysanthemi. In other words, the known structure of the Erwinia chrysanthemi asparaginase, such as the A-chain structure of the Erwinia chrysanthemi asparaginase, is used as a template when modeling the structure of the parent asparaginase.<br>
<br>
In another embodiment of the method of the present invention, the parent asparaginase is modeled on the modeled three-dimensional structure of amino acid residues 50-378 of SEQ ID NO: 1, where this structure is modeled as described above.<br>
<br>
The analysis of the structure referred to in step (b) of a method of the invention may involve an analysis of the contemplated impact of the structure or part of the structure on the function of the enzyme. For example change in electrostatic properties, change in surface hydrophobicity or altered local dynamics. Such local changes may lead to altered physical properties of the enzyme, such as thermostability, stability in presence of other chemical components (e.g. surfactants), pH-profile, adsorption properties or solubility.<br>
<br>
The structural part which is identified in step (b) of the method of the invention may be composed of one amino acid residue. However, in some cases the structural part comprises more than one amino acid residue. The structural part to be modified may comprise amino acids located at an interface between different subunits in a multimer, e.g. a dimer or a tetramer, in a loop structure, close to the substrate binding site, or the like. The structural part to be modified is preferably one which in the folded enzyme is believed to contribute to the temperature stability profile of the enzyme, or is otherwise responsible for the properties of the asparaginase.<br>
 <br>
Described in the following are specific concepts which may be applied in the method of the invention and specific suggestions of variants to be designed by use of the different concepts. Specific amino acid modifications are suggested mainly with a view to improving the activity at high temperatures and/or the thermostability; however the modifications may affect other properties as well.<br>
<br>
Concept: Molecular Dynamics (MP) Simulation<br>
Molecular Dynamics (MD) simulations are indicative of the mobility of the amino acids in a protein structure (see McCammon, J A and Harvey, SC, (1987), "Dynamics of proteins and nucleic acids", Cambridge University Press). Such protein dynamics are often compared to the crystallographic B-factors (see Stout, GH and Jensen, LH, (1989), "X-ray structure determination", Wiley). By running the MD simulation at e.g. different temperatures, the temperature related mobility of residues is simulated. Regions having the highest mobility or flexibility (here isotropic fluctuations) may be suggested for random mutagenesis. It is here understood that the high mobility found in certain areas of the protein, may be reduced by changes in these areas by substitution, insertion or deletion of one or more residues.<br>
<br>
The modeled tetrameric structure of the asparaginase from A. oryzae was subjected to 2ns of molecular dynamics simulation, at temperatures 300K, 400K and 500K. The GROMACS 3.3 molecular simulation package was used (D. van der Spoel, E. Lindahl, B. Hess, G. Groenhof, A. E. Mark and H. J. C. Berendsen (2005): GROMACS: Fast, Flexible and Free, J. Comp. Chem. 26 p. 1701-1718). The simulations were set up as follows:<br>
<br>
1)	The modeled tetrameric structure was converted to GROMACS topology, while care was taken to ensure that disulfide bridges were defined correctly. The OPLSAA force field was used.<br>
2)	100 steps of steepest descents minimization were run to remove atom clashes, and correct bond lengths. See Figure 4 for parameter file.<br>
3)	The structure was solvated in an 11x11x11 nm water box.<br>
4)	The solvated structure was minimized by 100 steps of steepest descents minimization. See Figure 4 for parameter file.<br>
5)	2 ns of NVT MD simulation were run at temperatures 300K, 400K and 500K. See Figure 5 for parameter file.<br>
6)	For each run, the first 800 ps were discarded, and the root mean square deviation (rmsd) for each of the C-alpha atom coordinates were calculated for the last 1200 ps, using the "g-rmsf tool of the GROMACS package.<br>
7)	The data were compared between the three simulation temperatures and between the four monomers of the structure (summarized in Figure 2). Residue ranges that have high rmsd values (relative to the average value for that temperature) are identified as mobile regions and thus targets of interest for stabilizing mutations:<br>
<br>
Peak 1	68-74<br>
Peak 2	279 - 288<br>
Peak 3	309-319<br>
Peak 4	329 - 342<br>
Peak 5	356 - 363<br>
<br>
One aspect of the present invention relates to a polypeptide which:<br>
(a)	has asparaginase activity;<br>
(b)	has at least 50%, such as at least 60%, at least 70%, at least 80%, at least 85%, at least 90%, at least 95% or at least 98%, identity to amino acids 80 to 378 of SEQ ID NO: 1; and<br>
(c)	comprises an amino acid difference compared to SEQ ID NO: 1 in at least one of the following regions: position 68-74, position 279-288, position 309-319, position 329-342, and/or position 356-363; wherein each position corresponds to a position of amino acids 1 to 378 of SEQ ID NO: 1.<br>
<br>
Thus, the polypeptide of the present invention may comprise a difference in the amino acid sequence compared to SEQ ID NO: 1 at one or more of the following positions, wherein each position corresponds to a position in SEQ ID NO: 1: 68, 69, 70, 71, 72, 73, 74, 279, 280, 281, 282, 283, 284, 285, 286, 287, 288, 309, 310, 311, 312, 313, 314, 315, 316, 317, 318, 319, 329, 330, 331, 332, 333, 334, 335, 336, 337, 338, 339, 340, 341, 342, 356, 357, 358, 359, 360, 361, 362, and/ or 363. In a preferred embodiment, the amino acid difference is a substitution.<br>
<br>
In a preferred aspect, the polypeptide is a variant of a parent polypeptide. The parent polypeptide may be any asparaginase, <br>
e.g. one of the asparaginases having the sequences set forth in SEQ ID NOs: 1-6 or homologues thereof.<br>
<br>
Polypeptides of particular interest have a combination of one or more of the above amino acid differences with any of the <br>
other amino acid differences disclosed herein.<br>
<br>
Concept: "Electrostatic Happiness"<br>
<br>
The charge network around a specific amino acid residue may be determined. This is particularly relevant for residues located on the surface. The "Electrostatic happiness" concept  aims  to  replace  charged   residues,  which  are  located  in  an  electrostatically unfavorable environment, or to introduce charges in positions where the potential suggest that such a charge would be electrostatically favorable.  Favorable conditions (=happy residues) are negatively charged residues where the electrostatic potential is positive or vice versa; unfavorable conditions (=unhappy residues) are when the residue charge has the same sign as the electrostatic potential. The concept has been described in literature (Jose M. Sanchez-Ruiz and George I. Makhatadze (2001): To charge or not to charge?, TRENDS in Biotechnology 19, pp. 132-135), but is here implemented using GROMACS:<br>
<br>
1)	The structure is converted to GROMACS topology, using the OPLSAA force field. This also applies partial charges to each atom. The partial charges are averaged over the atoms within each residue. This value will be zero for non-titratable residues.<br>
2)	The electrostatic potential at each atom position is computed using the GROMACS tool "genion". The potentials are averaged over the atoms within each residue.<br>
3)	For each residue, the "Electrostatic happiness" is defined as the averaged partial charge multiplied by the averaged electrostatic potential. A positive value indicates an unhappy residue, a negative value suggest a happy residue. The value will be zero for non-titratable residues.<br>
<br>
Charge reversals (e.g. K to E) are in some cases feasible. In other cases, substitution to a structurally similar but neutral residue may be optimal. Specific mutations based on this concept are: Residues which are not at interfaces between monomers: R196E, K194E, D88N, E255Q, K290E, E311K (for those suggesting a charge reversal, a neutral residue may be optimal instead). Residues which are at interfaces between monomers: D111N, D206N, E235Q, R266L, D275N, E331Q.<br>
<br>
One aspect of the present invention relates to a polypeptide which:<br>
<br>
(a)	has asparaginase activity;<br>
(b)	has at least 50%, such as at least 60%, at least 70%, at least 80%, at least 85%, at least 90%, at least 95% or at least 98%, identity to amino acids 80 to 378 of SEQ ID NO: 1; and<br>
(c)	comprises an amino acid difference compared to SEQ ID NO: 1 in at least one of the following positions: D88, D111, K194, R196, D206, E235, E255, R266, D275, K290, E311, E331; wherein each position corresponds to a position of amino acids 1 to 378 of SEQ ID NO: 1.<br>
<br>
Preferably, the amino acid difference is a substitution. Therefore, in a preferred aspect, the polypeptide of the present invention comprises at least one of the following amino acid substitutions compared to SEQ ID NO: 1, wherein each position corresponds to a position in SEQ ID N0:1: D88N, D111N, K194E, R196E, D206N, E235Q, E255Q, R266L, D275N, K290E, E311K, E331Q.<br>
<br>
In a preferred aspect, the polypeptide is a variant of a parent polypeptide. The parent polypeptide may be any asparaginase, <br>
e.g. one of the asparaginases having the sequences set forth in SEQ ID NOs: 1-6 or homologues thereof.<br>
<br>
Polypeptides of particular interest have a combination of one or more of the above amino acid differences with any of the other amino acid differences disclosed herein.<br>
<br>
Concept: Introduction of Proline Residues<br>
<br>
Substitution to proline residues may be suggested at various positions based on phi/psi backbone angles and side chain clashes. Residues that may be substituted for proline was identified using the command SUGPRO from the program WHATIF (G. Vriend (1990): WHAT IF: A molecular modeling and drug design program, J. Mol. Graph. 8, p52-56). An alternative method is described in patent no. EP0585285B1.<br>
<br>
One aspect of the present invention relates to a polypeptide which:<br>
(a)	has asparaginase activity;<br>
(b)	has at least 50%, such as at least 60%, at least 70%, at least 80%, at least 85%, at least 90%, at least 95% or at least 98%, identity to amino acids 80 to 378 of SEQ ID NO: 1; and<br>
(c)	comprises an amino acid difference compared to SEQ ID NO: 1 in at least one of the following positions: G82, T113, 1365, 183, N278, Q84, N70, E366, VI64, A137, F306, D115, T85, T280, L201; wherein each position corresponds to a position of amino acids 1 to 378 of SEQ ID NO: 1.<br>
<br>
Preferably, the amino acid difference is a substitution. Therefore, in a preferred aspect, the polypeptide of the present invention comprises at least one of the following amino acid substitutions compared to SEQ ID NO: 1, wherein each position corresponds to a position in SEQ ID N0:1: G82P, T113P, I365P, I83P, N278P, Q84P, N70P, E366P, V164P, A137P, F306P, D115P, T85P, T280P, L201P.<br>
<br>
In a preferred aspect, the polypeptide is a variant of a parent polypeptide. The parent polypeptide may be any asparaginase, e.g. one of the asparaginases having the sequences set forth in SEQ ID NOs: 1-6 or homologues thereof.<br>
<br>
Polypeptides of particular interest have a combination of one or more of the above amino acid differences with any of the other amino acid differences disclosed herein.<br>
<br>
<br>
Concept: Symmetry Point Interactions<br>
<br>
The tetrameric asparaginases have 3 axes of 2-fold symmetry, as shown in Figure 3. As the enzyme is a homotetramer, a residue located along one of these axes is in close contact with the corresponding residue in one of the other monomers, and the pattern is repeated at the other end of the symmetry axis in concern.<br>
<br>
In this way, two sets of interactions between the monomers may be created by just one mutation. This is particularly true for disulfide bridges, which may be introduced by substituting or inserting just one cysteine residue. Also other types of interactions, such as hydrophobic or electrostatic contacts, may be introduced along the symmetry axes.<br>
<br>
One aspect of the present invention relates to a polypeptide which:<br>
(a)	has asparaginase activity;<br>
(b)	has at least 50%, such as at least 60%, at least 70%, at least 80%, at least 85%, at least 90%, at least 95% or at least 98%, identity to amino acids 80 to 378 of SEQ ID NO: 1; and<br>
(c)	comprises an amino acid difference compared to SEQ ID NO: 1 in at least one of the following positions, wherein each position corresponds to a position of amino acids 1 to 378 of SEQ ID NO: 1: S176, D223, G231, P246, Y271, S283, G328 (substitution to C will potentially result in formation of one or more disulfide bridges); D223, K249, D286 (based on hydrophobic or electrostatic contacts).<br>
<br>
Preferably, the amino acid difference is a substitution. Therefore, in a preferred aspect, the polypeptide of the present invention comprises at least one of the following amino acid substitutions compared to SEQ ID NO: 1, wherein each position corresponds to a position in SEQ ID N0:1: S176C; D223C; G231C; P246C; Y271C; S283C; G328C; D223N/L; K249V/I/L; D286R/N/L.<br>
<br>
In a preferred aspect, the polypeptide is a variant of a parent polypeptide. The parent polypeptide may be any asparaginase, e.g. one of the asparaginases having the sequences set forth in SEQ ID NOs: 1-6 or homologues thereof.<br>
<br>
Polypeptides of particular interest have a combination of one or more of the above amino acid differences with any of the other amino acid differences disclosed herein.<br>
<br>
<br>
Concept: Helix capping<br>
<br>
The first residues in each end of a protein alpha helix, particularly at the N-terminal end, may be involved in backbone-sidechain hydrogen bonds, that are of importance for the stability of the protein (ref: Protein Science (1998), 7, p21-38). Substitutions here may lead to an increase in the thermostability.<br>
<br>
In the regions identified using Molecular Dynamics simulation, the following positions are particularly interesting for mutation, as one or more of the above mentioned concepts may be applied:<br>
<br>
Peak1:	<br>
N70	(Negative potential, reasoi<br>
D69	(Unhappy Asp)<br>
A72	(Negative potential)<br>
<br>
Peak 2:	<br>
D279	(Unhappy Asp)<br>
D286	(Unhappy Asp)<br>
K290	(Unhappy Lys)<br>
T280	(Helix capping)<br>
L281	(Helix capping)<br>
N278	(Negative potential)<br>
S283	(Symmetry point)<br>
<br>
Peak 3:	<br>
S307	(Helix capping)<br>
E311	(Unhappy Glu)<br>
D312	(Unhappy Asp)<br>
H317	(Positive potential)<br>
<br>
Peak 4:	<br>
E337	(Unhappy Glu)<br>
A336	(Reasonable Pro position)<br>
<br>
Peak 5:	<br>
Q361	(Negative potential)<br>
K363	(Unhappy K, reasonable P<br>
<br>
One aspect of the present invention relates to a polypeptide which:<br>
(a)	has asparaginase activity;<br>
(b)	has at least 50%, such as at least 60%, at least 70%, at least 80%, at least 85%, at least 90%, at least 95% or at least 98%, identity to amino acids 80 to 378 of SEQ ID NO: 1; and<br>
(c)	comprises an amino acid difference compared to SEQ ID NO: 1 in at least one of the following positions, wherein each position corresponds to a position of amino acids 1 to 378 of SEQ ID NO: 1: N70; D69; A72; D279; D286; K290; <br>
T280; L281; N278; S283; S307; E311; D312; H317; E337; A336; Q361; and/or K363.<br>
<br>
Preferably, the amino acid difference is a substitution. Therefore, in a preferred aspect, the polypeptide of the present invention comprises at least one of the following amino acid substitutions compared to SEQ ID NO: 1, wherein each position corresponds to a position in SEQ ID N0:1: N70P/R/K; D69R/K; A72R/K; D279NA//R; D286NA//R; K290E/L; T280D/E; L281D/E; N278H/Q/R/K; S307A/D/E; E311Q/I/R; D312Y/NA//R; H317D/E; E337Q/R/K/I; A336P; Q361K/R; and/or K363P/Q/E/L.<br>
<br>
In a preferred aspect, the polypeptide is a variant of a parent polypeptide. The parent polypeptide may be any asparaginase, e.g. one of the asparaginases having the sequences set forth in SEQ ID NOs: 1-6 or homologues thereof.<br>
<br>
Polypeptides of particular interest have a combination of one or more of the above amino acid differences with any of the other amino acid differences disclosed herein.<br>
<br>
<br>
Methods for Designing Novel Asparaginase Variants Based on Ancestral Reconstruction<br>
<br>
An inferred ancestral asparaginase sequence to be used in a method of the present invention can be predicted from the sequences of a set of homologous proteins, as described in the literature (Joseph W. Thornton: Resurrecting Ancient Genes: Experimental Analysis Of Extinct Molecules (2004), Nature Reviews Genetics 5, p 366-375; 0. W. Cunningham, K. E. Omiand &amp; T. H. Oakley: Reconstructing ancestral character states: a critical reappraisal (1998), Trends in Ecology and Evolution 13, p 361-336). Depending on the number of sequences and their homology, the inferred ancestral sequence may be significantly different from each of these sequences, e.g. 40-90%. Hence a significant number of the amino acid residues differ between a protein and its inferred ancestral sequence.<br>
<br>
To obtain a more thermostable protein, a number of amino acid residues of a protein can be substituted with the corresponding residues of the inferred ancestral sequence. Because of the large number of differences, a method for selecting the preferred residue positions for substitution is desired. One way is to identify regions, where most of the residues are the same in the protein and its predicted ancestor. Within this region, residues that do differ between the protein and its ancestor are selected for substitution.<br>
<br>
Based on ancestral reconstruction, the present inventors have identified a number of<br>
asparaginase variants having increased thermostability.<br>
<br>
One aspect of the present invention therefore relates to a polypeptide which:<br>
(a)	has asparaginase activity;<br>
(b)	has at least 50%, such as at least 60%, at least 70%, at least 80%, at least 85%, at least 90%, at least 95% or at least 98%, identity to amino acids 80 to 378 of SEQ ID NO: 1; and<br>
(c)	comprises an amino acid difference compared to SEQ ID NO: 1 in at least one of the following positions: 54, 57, 70, 83, 84, 86, 93-96, 102, 107, 137, 139, 165, 172, 184-186, 209, 212, 214, 215, 219, 220, 224, 260, 262, 264, 266, 299, 318, 320, 321, 323, 325, 327, 349, 351, 353 and/or 356, wherein each position corresponds to a position of amino acids 1 to 378 of <br>
SEQ ID NO: 1.<br>
<br>
Preferably, the amino acid difference is a substitution. Therefore, in a preferred aspect, the polypeptide of the present invention comprises at least one of the following amino acid substitutions compared to SEQ ID NO: 1, wherein each position corresponds to a position in SEQ ID NO:1: V54I, F57L, N70K, 183V, Q84D, L86P, M93L, L94K, N95D, V96L, V102D, V107I, A137I, V139I, I165L, S172A, L184Y, Q185N, S186A, V209G, F212R, A214V, S215T, A219T, N220T, T224A, N260K, T262D, I264L, R266K, S299N, N318G, P320V, 1321V, A323R, T325S, T327V, A349Q, S351A, V353I and/or G356M.<br>
<br>
In a more preferred aspect, a polypeptide of the present invention comprises at least one of the following amino acid substitutions compared to SEQ ID NO: 1, wherein each position corresponds to a position in SEQ ID NO: 1: V54I, F57L, N70K, 183V, Q84D, L86P, V102D, N260K, T262D, A323R, T327V, A349Q, S351A and/or V353I.<br>
<br>
In an even more preferred aspect, a polypeptide of the present invention comprises at least one of the following amino acid substitutions compared to SEQ ID NO: 1, wherein each position corresponds to a position in SEQ ID NO: 1: N70K, A323R, T327V, A349Q, S351A and/or V353I.<br>
<br>
In a preferred aspect, the polypeptide is a variant of a parent polypeptide. The parent polypeptide may be any asparaginase, e.g. one of the asparaginases having the sequences set forth in SEQ ID NOs: 1-6 or homologues thereof.<br>
<br>
Polypeptides of particular interest have a combination of one or more of the above amino acid differences with any of the other amino acid differences disclosed herein.<br>
<br>
<br>
Preferred Asparaginases<br>
<br>
One aspect of the present invention relates to a polypeptide which:<br>
(a)	has asparaginase activity;<br>
(b)	has at least 50%, such as at least 60%, at least 70%, at least 80%, at least 90%, at (east 95%. or at least 98%, identity to amino acids 80 to 378 of SEQ ID NO: 1; and<br>
(c)	comprises an amino acid difference compared to SEQ ID NO: 1 in at least one of the following positions: 54, 57, 68-74, 82-86, 88, 93-96, 102, 107, 111, 113, 115, 137, 139, 164, 165, 172, 176, 184-186, 194, 196, 201, 206, 209, 212, 214, 215, 219, 220, 223, 224, 226, 228, 231, 235, 246, 249, 255, 260, 262, 264, 266, 271, 275, 278-288, 290, 299, 306, 307, 309-321, 323, 325, 327-342, 349, 351, 353, 356-363, 365, 366 and/or 375, wherein each position corresponds to a position in SEQ ID NO: 1.<br>
<br>
Preferably, the polypeptide comprises an amino acid difference compared to SEQ ID NO: 1 in at least one of the following positions: 54, 57, 70, 83, 84, 86, 102, 137, 164, 196, 201, 228, 260, 262, 278, 283, 290, 307, 312, 323, 327, 334, 336, 337, 349, 351, 353, 366 and/or 375, wherein each position corresponds to a position in SEQ ID NO: 1.<br>
<br>
The polypeptide may comprise an amino acid substitution compared to SEQ ID NO: 1. Preferably, the polypeptide comprises at least one of the following substitutions compared to SEQ ID NO: 1: 541, 57L, 69K/R, 70H/K/P/R/S, 72K/R, 82P, 83PA/, 84P/D, 85P, 86P, 88N, 93L, 94K, 95D, 96L, 102D, 1071, 111N, 113P, 115P, 137P/S/I, 1391, 164D/P, 165L, 172A, 1760, 184Y, 185N, 186A, 194E, 196E/I, 201P/Q, 206N, 209G, 212R, 214V, 215T, 219T, 220T, 223C/L/N, 224A, 228V, 2310, 235Q, 2460, 2491/LA/, 255Q, 260K, 262D, 264L, 266L/K, 2710, 275N, 278H/K/P/Q/R, 279N/RA/, 280D/E/P, 281D/E, 2830, 2B6L/N/R/V, 290E/L/V, 299N, 306P, 307A/D/E, 3111/K/Q/R, 312N/R/VA', 317D/E, 318G, 320V, 321V, 323R, 325S, 327V, 3280, 331Q, 334F, 3360/G/L/P, 337F/I/K/Q/R, 349Q, 351A, 3531, 356M, 361 K/R, 363E/L/P/Q, 365P, 366P and/or 375T. More preferably, the polypeptide comprises at least one of the following substitutions compared to SEQ ID NO: 1: 541, 57L, 70H/K/S, 83V, 84D, 86P, 102D, 137S, 164D, 1961, 201Q, 228V, 260K, 262D, 278H/Q, 2830, 290V, 307A, 312Y, 323R, 327V, 334F, 3360/G/L, 337F/I, 349Q, 351A, 3531, 366P and/or 375T. Even more preferably, the polypeptide comprises at least one, such as at least two, at least three, at least four or at least five, of the following substitutions compared to SEQ ID NO: 1: 70K, 323R, 327V, 349Q, 351A and/or 3531. Even more preferably, the polypeptide comprises the following substitutions compared to SEQ ID NO: 1: 70K, 323R, 327V, 349Q, 351A and 3531. Most preferably, the polypeptide has the same sequence as SEQ ID NO: 1, or a homologous sequence, except for the following substitutions: 70K, 323R, 327V, 349Q, 351A and 3531. The polypeptide may be a variant of a parent enzyme having the sequence of SEQ ID NO: 1 or a homologous sequence.<br>
<br>
In one preferred aspect of the present invention, the polypeptide comprises an amino acid difference compared to SEQ ID NO: 1 at a position corresponding to any of positions 70, 137, 164, 196, 201, 228, 278, 290, 366 and/or 375 in SEQ ID NO: 1. Preferably, the polypeptide comprises at least one of the following substitutions: 70H/K/S, 137S, 164D, 1961, 201Q, 228V, 278H/Q, 290V, 366P and/or 375T. The polypeptide may have a high relative asparaginase activity at high temperature, as determined above in the section 'Novel Asparaginases'. If the polypeptide is a variant of a parent polypeptide, the variant polypeptide may have a higher relative asparaginase activity at high temperature as compared to the parent polypeptide.<br>
<br>
Particularly preferred polypeptides according to the present invention comprise the following substitutions or sets of substitutions compared to SEQ ID NO: 1: 70H, 70S, 70K, 70K+278H, 70K+278H+196I, 70K+278H+201Q, 70K+283C, 137S, 137S+228V, 164D, 1961, 201Q, 278H, 278Q, 290V, 366P, and/or 366P+375T.<br>
<br>
In another preferred aspect of the present invention, the polypeptide comprises an amino acid difference compared to SEQ ID NO: 1 in at least one of the following positions: 70, 283, 307, 312, 334, 336 and/or 337. Preferably, the polypeptide comprises at least one of the following substitutions: 70K, 2830, 307A, 312Y, 334F, 336C/G/L and/or 337F/I, wherein each position corresponds to a position of amino acids 1 to 378 of SEQ ID NO: 1. The polypeptide may have a high thermostability, as determined above in the section 'Novel Asparaginases'. If the polypeptide is a variant of a parent polypeptide, the variant polypeptide may have a higher thermostability as compared to the parent polypeptide.<br>
<br>
Particularly preferred polypeptides according to the present invention comprise the following substitutions or sets of substitutions compared to SEQ ID NO: 1: 70K, 70K+307A+312Y, 70K+307A+312Y+336L+337F, 70K+307A+312Y+336L+337F+283C, 70K+307A+312Y+334F, 70K+307A+312Y+336G+337I, 70K+307A+312Y+283C, 70K+283C, 70K+336C+337F, 70K+307A, and/or 70K+312Y.<br>
<br>
In another preferred aspect of the present invention, the polypeptide comprises an amino acid difference compared to SEQ ID NO: 1 at a position corresponding to any of positions 54, 57, 70, 83, 84, 86, 93-96, 102, 107, 137, 139, 165, 172, 184-186, 209, 212, 214, 215, 219, 220, 224, 260, 262, 264, 266, 299, 318, 320, 321, 323, 325, 327, 349, 351, 353 and/or 356 in SEQ ID NO: 1. Preferably, the polypeptide comprises at least one of the following substitutions: V54I, F57L, N70K, 183V, Q84D, L86P, M93L, L94K, N95D, V96L, V102D, V107I, A137I, V139I, I165L, S172A, L184Y, Q185N, S186A, V209G, F212R, A214V, S215T, A219T, N220T, T224A, N260K, T262D, I264L, R266K, S299N, N318G, P320V, 1321V, A323R, T325S, T327V, A349Q, S351A, V353I and/or G356M compared to SEQ ID NO: 1. More preferably, the polypeptide comprises at least one of the following substitutions: V54I, F57L, N70K, 183V, Q84D, L86P, V102D, N260K, T262D, A323R, T327V, A349Q, S351A and/or V353I compared to SEQ ID NO: 1. Most preferably, the polypeptide comprises at least one, such as at least two, at least three, at least four or at least five, of the following substitutions: N70K, A323R, T327V, A349Q, S351A and/or V353I compared to SEQ ID NO: 1. The polypeptide may have a high relative asparaginase activity at high temperature, as determined above in the section 'Novel Asparaginases'. If the polypeptide is a variant of a parent polypeptide, the variant polypeptide may have a higher relative asparaginase activity at high temperature as compared to the parent polypeptide. <br>
Particularly preferred polypeptides according to the present invention comprise the following sets of substitutions compared to SEQ ID NO: 1: N70K+V54I+F57L, N70K+N260K+T262D, N70K+A323R+T327V, N70K+A349Q+S351A+V353I, N70K+I83V+Q84D+A323R+T327V, N70K+L86P+V102D+A323R+T327V, N70K+A323R+T327V+A349Q+S351A+V353I,<br>
N70K+I83V+Q84D+A323R+T327V+A349Q+S351A+V353I, and/or 70K+V102D+A323R+T327V+A349Q+S351A+V353I.<br>
<br>
<br>
Methods for Preparing Asparaginase Variants Based on Sequence<br>
<br>
The specific amino acid positions identified herein as being of relevance for improving the thermotolerance of an asparaginase will be of relevance for improving the thermotolerance of any asparaginase the sequence of which can be aligned with SEQ ID NO: 1.<br>
<br>
Therefore, in one aspect the present invention relates to a method for preparing a polypeptide comprising:<br>
(a)	providing an amino acid sequence of a parent polypeptide having asparaginase activity;<br>
(b)	selecting at least one amino acid residue at a position in the sequence which corresponds to any of positions 54, 57, 68-74, 82-86, 88, 93-96, 102, 107, 111, 113, 115, 137, 139, 164, 165, 172, 176, 184-186, 194, 196, 201, 206, 209, 212, 214, 215, 219, 220, 223, 224, 226, 228, 231, 235, 246, 249, 255, 260, 262, 264, 266, 271, 275, 278-288, 290, 299, 306, 307, 309-321, 323, 325, 327-342, 349, 351, 353, 356-363, 365, 366 and/or 375 in SEQ ID NO: 1;<br>
(c)	modifying the sequence by substituting or deleting the selected amino acid residue or inserting one or more amino acid residues adjacent to the selected amino acid residue;<br>
(d)	producing a variant polypeptide having the modified sequence;<br>
(e)	testing the variant polypeptide for asparaginase activity and thermotolerance; and<br>
(f)	selecting a variant polypeptide having asparaginase activity and higher thermotolerance compared to the parent polypeptide.<br>
<br>
The parent polypeptide preferably has a sequence which has at least 50% identity to SEQ ID<br>
N0:1, SEQ ID N0;2, SEQ ID N0:3, SEQ ID N0:4, SEQ ID N0:5, SEQ ID N0:6, or a fragment of any of these sequences, more preferably at least 60%, at least 70% or at least 80% identity, and even more preferably at least 90%, at least 95% or at least 98% identity. Most preferably, the parent polypeptide has a sequence which comprises or consists of any of SEQ ID NOs: 1-6 or a fragment thereof. A preferred fragment consists of the amino acid sequence from the position corresponding to position 80 of SEQ ID NO: 1 to the C-terminal of the polypeptide.<br>
<br>
A higher thermotolerance in the context of the present invention may mean a higher thermostability or a higher relative asparaginase activity at high temperature.<br>
<br>
Thermostability and relative asparaginase activity at high temperature may be determined as described above in the section 'Novel Asparaginases'.<br>
<br>
In a preferred aspect, the at least one amino acid residue selected has a position in the sequence which corresponds to any of positions 54, 57, 70, 83, 84, 86, 102, 137, 164, 196, 201, 228, 260, 262, 278, 283, 290, 307, 312, 323, 327, 334, 336, 337, 349, 351, 353, 366 and/or 375 in SEQ ID NO: 1.<br>
<br>
In another preferred aspect, the sequence is modified by substituting the at least one selected amino acid residue. Preferably, the sequence is modified by introducing at least one of the following substitutions: 541, 57L, 69K/R, 70H/K/P/R/S, 72K/R, 82P, 83PA/, 84P/D, 85P, 86P, 88N, 93L, 94K, 95D, 96L, 102D, 1071, 111N, 113P, 115P, 137P/S/I, 1391, 164D/P, 165L, 172A, 176C, 184Y, 185N, 186A, 194E, 196E/I, 201P/Q, 206N, 209G, 212R, 214V, 215T, 219T, 220T, 223C/L/N, 224A, 228V, 231C, 235Q, 246C, 2491/LA/, 255Q, 260K, 262D, 264L, 266L/K, 271C, 275N, 278H/K/P/Q/R, 279N/RA/, 280D/E/P, 281D/E, 283C, 286L/N/RA/, 290E/LA/, 299N, 306P, 307A/D/E, 3111/K/Q/R, 312N/RA//Y, 317D/E, 318G, 320V, 321V, 323R, 325S, 327V, 328C, 331Q, 334F, 336C/G/L/P, 337F/I/K/Q/R, 349Q, 351 A, 3531, 356M, 361K/R, 363E/L/P/Q, 365P, 366P and/or 375T; wherein each position corresponds to a position of amino acids 1 to 378 of SEQ ID NO: 1. More preferably, the sequence is modified by introducing at least one of the following substitutions: 541, 57L, 70H/K/S, 83V, 84D, 86P, 102D, 137S, 164D, 1961, 201Q, 228V, 260K, 262D, 278H/Q, 283C, 290V, 307A, 312Y, 323R, 327V, 334F, 336C/G/L, 337F/I, 349Q, 351 A, 3531, 366P and/or 375T; wherein each position corresponds to a position of amino acids 1 to 378 of SEQ ID NO: 1.<br>
<br>
In a more preferred aspect, the sequence is modified by introducing at least one, such as at least two, at least three, at least four or at least five, of the following substitutions: 70K, 323R, 327V, 349Q, 351A and/or 3531; wherein each position corresponds to a position of amino acids 1 to 378 of SEQ ID NO: 1. In a more preferred aspect, the sequence is modified by introducing the following substitutions: 70K, 323R, 327V, 349Q, 351A and/or 3531, possibly in combination with other substitutions; wherein each position corresponds to a position of amino acids 1 to 378 of SEQ ID NO: 1.<br>
<br>
The modification of one or more amino acid residues to obtain a polypeptide of the present invention can be accomplished by any method known in the art, e.g. as described in WO 94/14963 or WO 94/14964 (Unilever). The following describes methods for the cloning of asparaginase-encoding DNA sequences, followed by methods for generating mutations at specific sites within the asparaginase-encoding sequence.<br>
<br>
<br>
Cloning a DNA Sequence Encoding an Asparaginase<br>
<br>
The DNA sequence encoding an asparaginase may be isolated from any cell or microorganism producing the asparaginase in question, using various methods well known in the art. First, a genomic DNA and/or cDNA library could be constructed using chromosomal DNA or messenger RNA from the organism that produces the asparaginase to be cloned. Then, if the amino acid sequence of the asparaginase is known, labeled oligonucleotide probes may be synthesized and used to identify asparaginase-encoding clones from a genomic library prepared from the organism in question. Alternatively, a labeled oligonucleotide probe containing sequences homologous to another known asparaginase gene could be used as a probe to identify asparaginase-encoding clones, using hybridization and washing conditions of lower stringency.<br>
<br>
Yet another method for identifying asparaginase-encoding clones would involve inserting fragments of genomic DNA and/or cDNA into an expression vector, such as a plasmid, transforming an asparaginase-negative bacterial or fungal host with the resulting DNA library, and then plating the transformed cells onto an agar medium that allows clones expressing the asparaginase to be identified. Such a medium may, for example, contain asparagine as a sole nitrogen source, such that only cells with an active asparaginase gene will grow. Another medium may, for example, include asparagine and a pH indicator that shifts color in response to the increase in pH that occurs as ammonium is released from the asparagine by asparaginase.<br>
<br>
Alternatively, if the amino acid sequence is known for the asparaginase or related asparaginases, degenerate DNA primers may be designed that will allow the direct amplification of the asparaginase from genomic DNA, or from DNA libraries.<br>
<br>
Alternatively, the sequences can be determined for the clones in a gene library, or for a selected sub-set of those clones, and these sequences can be compared to known asparaginase sequences in order to identify novel sequences.<br>
<br>
Alternatively, the DNA sequence encoding the enzyme may be prepared synthetically by established standard methods, e.g. the phosphoroamidite method described S.L. Beaucage and M.H. Caruthers, (1981), Tetrahedron Letters 22, p. 1859-1869, or the method described by Matthes et al., (1984), EMBO J. 3, pp. 801-805. In the phosphoroamidite method, oligonucleotides are synthesized, e.g. in an automatic DNA synthesizer, purified, annealed, ligated and cloned in appropriate vectors.<br>
<br>
Finally, the DNA sequence may be of mixed genomic and synthetic origin, mixed synthetic and cDNA origin or mixed genomic and cDNA origin, prepared by ligating fragments of synthetic, genomic or cDNA origin (as appropriate, the fragments corresponding to various parts of the entire DNA sequence), in accordance with standard techniques. The DNA sequence may also be prepared by polymerase chain reaction (PCR) using specific primers, for instance as described in US 4,683,202 or R.K. Saiki et al., (1988), Science 239, pp. 487-491.<br>
<br>
<br>
Construction of Asparaginase Variants<br>
<br>
An asparaginase variant may be obtained by site-directed mutagenesis at selected positions (see below) or by localized random mutagenesis, i.e. by introduction of random amino acid residues in selected positions or regions of the parent polypeptide, e.g. as described in WO 95/22615.<br>
<br>
Random mutagenesis is suitably performed either as localized or region-specific random mutagenesis in at least three parts of the gene translating to the amino acid sequence in question, or within the whole gene. When the mutagenesis is performed by the use of an oligonucleotide, the oligonucleotide may be doped or spiked with the three non-parent nucleotides during the synthesis of the oligonucleotide at the positions which are to be changed. The doping or spiking may be performed so that codons for unwanted amino acids are avoided. The doped or spiked oligonucleotide can be incorporated into the DNA encoding the asparaginase enzyme by any technique, using, e.g., PCR, LCR or any DNA polymerase and ligase as deemed appropriate.<br>
<br>
Preferably, the doping is carried out using "constant random doping", in which the percentage of wild-type and mutation in each position is predefined. Furthermore, the doping may be directed toward a preference for the introduction of certain nucleotides, and thereby a preference for the introduction of one or more specific amino acid residues. The doping may<br>
be made, e.g., so as to allow for the introduction of 90% wM type and 10% mutations in each position. An additional consideration in the choice of a doping scheme is based on genetic as well as protein-structural constraints.<br>
<br>
The random mutagenesis may be advantageously localized to a part of the parent asparaginase in question. This may, e.g., be advantageous when certain regions of the enzyme have been identified to be of particular importance for a given property of the enzyme.<br>
<br>
<br>
Site-directed Mutagenesis<br>
<br>
Once an asparaginase-encoding DNA sequence has been isolated, and desirable sites for mutation identified, mutations may be introduced using synthetic oligonucleotides. These oligonucleotides contain nucleotide sequences flanking the desired mutation sites. In a specific method, a single-stranded gap of DNA, the asparaginase-encoding sequence, is created in a vector carrying the asparaginase gene. Then the synthetic nucleotide, bearing the desired mutation, is annealed to a homologous portion of the single-stranded DNA. The remaining gap is then filled in with DNA polymerase I (Klenow fragment) and the construct is ligated using T4 ligase. A specific example of this method is described in Morinaga et al., (1984), Biotechnology 2, pp. 636-639. US 4,760,025 discloses the introduction of oligonucleotides encoding multiple mutations by performing minor alterations of the cassette. However, an even greater variety of mutations can be introduced at any one time by the Morinaga method, because a multitude of oligonucleotides, of various lengths, can be introduced.<br>
<br>
Another method for introducing mutations into asparaginase-encoding DNA sequences is described in Nelson and Long, (1989), Analytical Biochemistry 180, pp. 147-151. It involves the 3-step generation of a PCR fragment containing the desired mutation introduced by using a chemically synthesized DNA strand as one of the primers in the PCR reactions. From the PCR-generated fragment, a DNA fragment carrying the mutation may be isolated by cleavage with restriction endonucleases and reinserted into an expression plasmid.<br>
<br>
<br>
Expression of Asparaginases<br>
<br>
According to the invention, a DNA sequence encoding a polypeptide of the present invention, including a variant produced by methods described above, or by any alternative methods known in the art, can be expressed, in enzyme form, using an expression vector which typically may include various control sequences, e.g., a promoter, operator, ribosome binding<br>
site, translation initiation signal, and, in some cases, a repressor gene or various activator genes.<br>
 <br>
Expression Vector<br>
The recombinant expression vector carrying the DNA sequence encoding an asparaginase of the invention may be any vector which may conveniently be subjected to recombinant DNA procedures, and the choice of vector will often depend on the host cell into which it is to be introduced. The vector may be one which, when introduced into a host cell, is integrated into the host cell genome and replicated together with the chromosome(s) into which it has been integrated.<br>
<br>
In the vector, the DNA sequence should be operably connected to a suitable promoter sequence. The promoter may be any DNA sequence which shows transcriptional activity in the host cell of choice and may be derived from genes encoding proteins either homologous or heterologous to the host cell.<br>
<br>
The expression vector of the invention may also comprise a suitable transcription terminator and, in eukaryotes, polyadenylation sequences operably connected to the DNA sequence encoding the asparaginase of the invention. Termination and polyadenylation sequences may suitably be derived from the same sources as the promoter.<br>
<br>
The vector may further comprise a DNA sequence enabling the vector to replicate in the host cell in question. Examples of such sequences are the origins of replication of plasmids pUC19, pACYC177, pUB110, pE194, pAMB1 and plJ702.<br>
<br>
The vector may also comprise a selectable marker, e.g. a gene the product of which complements a defect in the host cell, such as the dal genes from 6. subtilis or S. licheniformis, or one which confers antibiotic resistance such as ampicillin, kanamycin, chloramphenicol or tetracyclin resistance. Furthermore, the vector may comprise Aspergillus selection markers such as amdS, argB, niaD and sC, a marker giving rise to hygromycin resistance, or the selection may be accomplished by co-transformation, e.g. as described in WO 91/17243.<br>
<br>
The procedures used to ligate the DNA construct of the invention encoding an asparaginase and optionally a promoter, terminator and/or other elements, and to insert them into suitable vectors containing the information necessary for replication, are well known to persons skilled in the art (cf., for instance, Sambrook et al., Molecular Cloning: A Laboratory Manual, 2nd Ed., Cold Spring Harbor, 1989).<br>
<br>
<br>
Host Cells<br>
The cell of the invention, either comprising a DNA construct or an expression vector of the invention as defined above, is advantageously used as a host cell in the recombinant production of an asparaginase of the invention. The cell may be transformed with the DNA construct of the invention encoding the asparaginase, conveniently by integrating the DNA construct (in one or more copies) in the host chromosome. This integration is generally considered to be an advantage as the DNA sequence is thereby more likely to be stably maintained in the cell. Integration of the DNA constructs into the host chromosome may be performed according to conventional methods, e.g. by homologous or heterologous recombination. Alternatively, the cell may be transformed w/ith an expression vector as described above in connection with the different types of host cells.<br>
<br>
The cell of the invention may be a microbial cell, e.g. a bacterial or a fungal (including yeast) cell.<br>
<br>
Examples of suitable bacteria are Gram positive bacteria such as Bacillus subtilis, Bacillus licheniformis, Bacillus lentus, Bacillus brevis, Bacillus stearothermophilus, Bacillus alkalophilus, Bacillus amyloliquefaciens, Bacillus coagulans, Bacillus circulans, Bacillus lautus, Bacillus megaterium, Bacillus thuringiensis, or Streptomyces lividans or Streptomyces murinus, or Gram negative bacteria such as E. coli. The transformation of the bacteria may, for instance, be effected by protoplast transformation or by using competent cells in a manner known per se.<br>
<br>
The yeast organism may favorably be selected from a species of Saccharomyces or Schizosaccharomyces, e.g. Saccharomyces cerevisiae.<br>
<br>
The host cell may also be a filamentous fungus e.g. a strain belonging to a species of Aspergillus, particularly Aspergillus oryzae or Aspergillus niger, or a strain of Fusarium, such as a strain of Fusarium oxysporum, Fusarium graminearum, Fusarium sulphureum, Fusarium trichothecioides, Fusarium bactridioides, Fusarium sambucinum, Fusarium roseum, Fusarium cerealis, Fusarium crokkwellense, or Fusarium venenatum.<br>
<br>
In a particular embodiment of the invention the host cell is a protease deficient or protease minus strain.<br>
<br>
This may for instance be the protease deficient strain Aspergillus oryzae JaL 125 having the alkaline protease gene named "alp" deleted. This strain is described in WO 97/35956 (Novo Nordisk A/S).<br>
Filamentous fungi cells may be transformed by a process involving protoplast formation and transformation of the protoplasts followed by regeneration of the cell wall in a manner known per se. The use of Aspergillus as a host micro-organism is described in EP 238 023 (Novo Nordisk A/S), the contents of which are hereby incorporated by reference.<br>
<br>
<br>
Production of Asparaginases by Cultivation of Transformant<br>
<br>
The invention relates, inter alia, to a method of producing an asparaginase of the invention, which method comprises cultivating a host cell under conditions conducive to the production of the asparaginase and recovering the enzyme from the cells and/or culture medium.<br>
<br>
The medium used to cultivate the cells may be any conventional medium suitable for growing the host cell in question and obtaining expression of the asparaginase of the invention. Suitable media are available from commercial suppliers or may be prepared according to published recipes (e.g. as described in catalogues of the American Type Culture Collection).<br>
<br>
The asparaginase secreted from the host cells may conveniently be recovered from the culture medium by well-known procedures, including separating the cells from the medium by centrifugation or filtration, and precipitating proteinaceous components of the medium by means of a salt such as ammonium sulphate, followed by the use of chromatographic procedures such as ion exchange chromatography, affinity chromatography, or the like.<br>
<br>
<br>
Screening and Testing<br>
<br>
Asparaginase variants produced by any of the methods described herein may be tested, either prior to or after purification, for asparaginase activity, e.g. in a screening assay which measures the ability of the variant to hydrolyze asparagine, e.g. at high temperature.<br>
<br>
In order to perform such screening, a microorganism being transformed with a DNA sequence encoding a mutated asparaginase of interest, and having the ability to express it, is incubated in a suitable medium and under suitable conditions for secretion of the enzyme.<br>
<br>
Screening for variants having increased thermostability may for instance be done by incubating the culture supernatant at elevated temperature (e.g. in the range from 50°C-80°C, preferably at 60, 62, 64, 66, 68, and/or 70°C) for a time period (e.g. from 5 to 30 minutes, such as for 20 minutes) and measuring the asparagine hydrolyzing activity at suitable temperature, e.g. 55°C.<br>
<br>
Screening for variants having increased relative activity at high temperature as compared to e.g. 37°C may for instance be done by measuring the asparagine hydrolyzing activity of the culture supernatant at both a suitable reference temperature (e.g. 37°C) and at elevated temperature (e.g. in the range from 50°C-80°C, preferably at 60, 62, 64, 66, 68, and/or 70°C). The relative activity is determined as the asparagine hydrolyzing activity at elevated temperature divided by the activity at the reference temperature.<br>
<br>
Measurement of the asparagine hydrolyzing activity may be performed by methods known in the art, e.g. as described in the Examples.<br>
<br>
The clones encoding asparaginase variants having higher thermostability and/or higher relative activity at high temperature may be selected for confirmation of sequence and enzyme purification. Further testing in regard to other altered properties may be performed on purified variants in accordance with methods known in the art. Such altered properties include thermostability, temperature dependent activity, thermoinactivation, pH dependent activity, pH dependent stability, specific activity, or substrate specificity, and any other parameter of interest.<br>
<br>
<br>
Production of Foods<br>
<br>
Asparaginases according to the present invention may be suitable in the production of food products, such as thermally processed carbohydrate-containing foods. Such food products, including coffee, cereals, cookies, potato chips, crackers, french fries, breads and rolls, have been shown in many cases to contain acrylamide, which is considered as probably carcinogenic for animals and humans. Acrylamide may be formed when the amino acid asparagine is heated in the presence of a reducing sugar. Therefore, reduction of acrylamide in thermally processed foods can be achieved by adding an asparaginase of the present invention to a food material prior to heating.<br>
<br>
A food material in the context of the present invention means any unprocessed or partly processed form of the food product that occurs during the production process prior to obtaining the final form of the food product.<br>
<br>
One aspect of the present invention is therefore a method for reduction of acrylamide in a food product, wherein an asparaginase of the invention is added to a food material which is subsequently heated. The enzyme should be added in an amount that is effective in reducing the level of asparagine present in the food material to be heated.<br>
<br>
Examples of food products in which the process according to the invention may be useful include any cereal based products, such as bread, pastry, cake, pretzels, bagels, cookies, gingerbread, gingercake, breakfast cereals and crispbread; any potato based products, such as french fries, pommes frites, potato chips or crisps, fabricated potato snacks and croquettes; and any coffee based products.<br>
<br>
Preferred heating steps are those where at least a part of the food material, e.g. the surface of the food material, is exposed to temperatures at which the formation of acrylamide is promoted, e.g. 110°C or higher, or 120°C or higher. The heating step in the process according to the invention may be carried out in ovens, for instance at a temperature between 180-220°C, such as for the baking of bread and other bakery products, or in oil such as the frying of potato chips, for example at 160-190°C.<br>
<br>
One preferred embodiment is a method for reduction of acrylamide in french fries or other fried vegetable-based food products. The vegetable food material, e.g. potatoes, may be contacted with an asparaginase according to the invention after or during blanching, prior to drying and par-frying, to produce a par-fried vegetable food material. The enzyme may also be added after par-frying. The par-fried vegetable food material is preferably subjected to a final fry to produce a fried vegetable food product, or it is frozen to produce a frozen par-fried vegetable food material, which e.g. after freeze storage may be subjected to a final fry to produce a fried vegetable food product, e.g. french fries. The contacting with the asparaginase of the present invention is preferably accomplished by dipping, soaking or coating the vegetable food material in an aqueous enzyme solution or a mixture containing said enzyme.<br>
<br>
Examples:<br>
<br>
Materials and Methods:<br>
<br>
Strains and Plasmids<br>
E.coli DH12S (available from Gibco BRL) was used for yeast plasmid rescue.<br>
<br>
pJN001N2 is an S. cerevisiae and E.coli shuttle vector under the control of TPI promoter, constructed from pJC039 described in WO 01/92502, in which the Aspergillus oryzae asparaginase gene has been inserted.<br>
<br>
Saccharomyces cerevisiae YNG318: MATa Dpep4[cir+] ura3-52, Ieu2-D2, his 4-539 was used for asparaginase variants expression. It is described in J. Biol. Chem. 272 (15), pp 9720-9727, 1997.<br>
<br>
<br>
Media and Substrates<br>
<br>
10X Basal solution: Yeast nitrogen base w/o amino acids (DIFCO) 66.8 g/l, succinate 100 g/l, NaOH 60 g/l.<br>
SC-glucose: 20% glucose (i.e., a final concentration of 2% = 2 g/100ml)) 100 ml/I, 5% threonine 4 ml/I, 1% tryptophan <br>
10 ml/I, 20% casamino acids 25 ml/I, 10 X basal solution 100 ml/I. The solution is sterilized using a filter of a pore size of 0.20 micrometer. Agar and H2O (approx. 761 ml) is autoclaved together, and the separately sterilized SC-glucose solution added to the agar solution.<br>
<br>
YPD: Bacto peptone 20 g/l, yeast extract 10 g/l, 20% glucose 100 ml/L.<br>
<br>
PEG/LiAc solution: 40% PEG4000 50ml, 5M Lithium Acetate 1ml<br>
<br>
<br>
DNA Manipulations<br>
<br>
Unless othenwise stated, DNA manipulations and transformations were performed using standard methods of molecular biology as described in Sambrook et al. (1989) Molecular cloning: A laboratory manual, Cold Spring Harbor lab. Cold Spring Harbor, NY; Ausubel, F. M. et al. (eds.) "Current protocols in Molecular Biology", John Wiley and Sons, 1995; Hanwood, C. R. and Cutting, S. M. (eds.).<br>
<br>
<br>
Yeast Transformation<br>
<br>
Yeast transformation was carried out by lithium acetate method. Mix 0.5 microL of vector (digested by restriction endnucleases) and 1 microL of PCR fragments. Thaw YNG318 competent cells on ice. Mix 100 microL of the cells, the DNA mixture and 10 microL of carrier DNA (Clontech) in 12ml polypropylene tubes (Falcon 2059). Add 0.6ml PEG/LiAc solution and mix gently. Incubate for 30 min at 30°C, and 200 rpm. Incubate for 30 min at 42°C (heat shock). Transfer to an eppendorf tube and centrifuge for 5sec. Remove the supernatant and resolve in 3ml of YPD. Incubate the cell suspension for 45min at 200rpm at 30°C. Pour the suspension to SC-glucose plates and incubate 30°C for 3 days to make colonies. Yeast total DNA was extracted by the Robzyk and Kassir's method described in Nucleic acids research vol.20, No.14 (1992) 3790.<br>
<br>
<br>
DNA Sequencing<br>
E.coli transformation for DNA sequencing was carried out by electroporation (BIO-RAD Gene Pulser). DNA Plasmids were prepared by alkaline method (Molecular Cloning, Cold Spring Harbor) or with the Qiagen® Plasmid Kit. DNA fragments were recovered from agarose gel by the Qiagen gel extraction Kit. PCR was performed using a PTC-200 DNA Engine. The ABI PRISMTM 310 Genetic Analyzer was used for determination of all DNA sequences.<br>
 <br>
Example 1: Construction of Asparaginase Expression Vector<br>
The Aspergillus oryzae asparaginase gene (SEQ ID NO: 1) was amplified with the primer pair Cutipre-asparaN2 F (SEQ ID NO: 7) and AsparaC R (SEQ ID NO: 8). The resulting PCR fragments were introduced into S. cerevisiae YNG318 together with the pJC039 vector digested with restriction enzymes to remove the mature part of Humicola insolens cutinase gene.<br>
<br>
Cutipre-asparaN2 (41mer)<br>
AGCCTTGTTGCTGCTCTCCCCGCCACAGACTCGAACGTCAC AsparaC R (50mer) AGTCACCCTCTAGATCTCGACTTAATTAATCAAGCAACCCCAATCCGCTC<br>
<br>
Plasmid, which is termed as pJN001N2, was recovered from the yeast transformants on SC-glucose plates and the internal sequence was determined to confirm the asparaginase gene. pJN001N2 thus has the signal sequence from Humicola insolens cutinase (MKFFTTILSTASLVAALP) followed by the sequence of the mature part of Aspergillus oryzae asparaginase (amino acids 27-378 of SEQ ID NO: 1).<br>
<br>
Example 2: Construction of Yeast Libraries and Site-directed Variants<br>
Libraries in yeast and site-directed variants were constructed by SOE PCR method (Splicing by Overlap Extension, see "PCR: A practical approach", p. 207-209, Oxford University press, eds. McPherson, Quirke, Taylor), followed by introduction of the purified PCR fragments mixed with vector digest into Saccharomyces cerevisiae for in vivo recombination.<br>
<br>
Example 3: Library Screening (Relative Activity Selection)<br>
Yeast clones/libraries were prepared as in Example 2, using pJN001N2 as a backbone. Clones on SC-glucose were inoculated to a well of a 96-well micro titre plate containing YPD medium and cultivated at 28°C for 3 days. To determine the relative activity at a certain elevated temperature as compared to a reference temperature (37°C), the asparagine hydrolyzing activity of the culture supernatant was measured at both 37°C and the higher temperature (63°C or 65°C). The relative activity was determined as the asparagine hydrolyzing activity at elevated temperature divided by the activity at 37°C. The clones with higher relative activity were selected and the sequences were confirmed.<br>
<br>
<br>
Reagents:<br>
•    1M Potassium phosphate buffer (pH6.0)<br>
1M KH2PO4(136g/500ml) + 1M K2HPO4(174g/500ml)<br>
Adjust to pH6.0<br>
<br>
100mMPotassium phosphate buffer (pH6.0) + 0.1% tritonX-100 (1L)<br>
100ml 1M Potassium phosphate buffer (pH6.0)<br>
1g    TritonX-100<br>
Adjust to 1000ml<br>
<br>
.2M Hydroxylamine (HA) solution (100ml)<br>
13.9g hydroxylamine<br>
Adjust to 100ml with lOOmM potassium phosphate buffer (pH6)<br>
<br>
.Stop solution (500ml)<br>
23.83ml acetate<br>
13.88g FeClsSHzO<br>
84ml   5N HCI<br>
Adjust to 500ml with H2O<br>
<br>
.Substrate solution (100ml)<br>
10ml 1M Potassium phosphate buffer<br>
0.5g L-asparagine (132.12, final cone. 0.0325M)<br>
5ml2MHAsoln.<br>
Adjust to 100ml with H2O.<br>
<br>
 <br>
Assay:<br>
1    Pipette 20 microL sample into a well.<br>
2   Add 10p microL of substrate solution into the well.<br>
3   Incubate 20 minutes at 37°C and higher.<br>
4   Add lOp microL of stop solution into the well.<br>
5   Measure A490 and determine the relative activity to 37°C.<br>
 <br>
Results:<br>
Table 1. Clone JN001N2 holds pJN001N2 encoding wild type Aspergillus oryzae asparaginase. In the other clones, the encoded asparaginase has the amino acid substitutions indicated.<br>
<br>
<br>
Example 4: Library Screening (Stability Selection)<br>
Yeast clones/libraries were prepared as in Example 2, using pJN001N2 as a backbone. Clones on SC-glucose were inoculated to a well of a 96-well micro titre plate containing YPD medium and cultivated at 28°C for 3 days. To determine the remaining activity after heat treatment, the asparagine hydrolyzing activity of the culture supernatant was measured at 55'C after incubating for 20 minutes at elevated temperature (64°C or 66°C, 4°C as a reference). Then the clones with higher remaining activity were selected and the sequence was confirmed.<br>
<br>
<br>
Reagents:<br>
•	1M Potassium phosphate buffer (pH6.0)<br>
            1M KH2PO4(136g/500ml) + 1M K2HPO4(174g/500ml) Adjust to pH6.0<br>
<br>
•	100mM Potassium phosphate buffer (pH6.0) + 0.1% tritonX-100 (1L)<br>
            100ml 1M Potassium phosphate buffer (pH6.0)<br>
            1g    Triton X-100 Adjust to 1000ml<br>
<br>
•	2M Hydroxylamine (HA) solution (100ml)<br>
            13.9g hydroxylamine<br>
            Adjust to 100ml with 100mM potassium phosphate buffer (pH6)<br>
<br>
•	Stop solution (500ml)<br>
            23.83ml acetate<br>
            13.88g FeCljeHzO<br>
            84ml   5NHCI<br>
           Adjust to 500ml with H2O<br>
<br>
•	Substrate solution (100ml)<br>
            10ml 1M Potassium phosphate buffer<br>
            0.5g L-asparagine (132.12, final cone. 0.0325M)<br>
            5ml 2M HA soln.<br>
           Adjust to 100ml with H2O<br>
<br>
<br>
Assay:<br>
1	Pipette 20 microL sample into a well.<br>
2	Incubate the 96 well plate at appropriate temperature for 20min. (4°C for a control)<br>
3	Add 100 microL of substrate solution into the well.<br>
4	Incubate 20min. at 55°C .<br>
5	Add 1 op microL of stop solution into the well.<br>
6	Measure A490.<br>
<br>
<br>
Results:<br>
<br>
Table 2. Clone JN001N2 holds pJN001N2 encoding wild type Aspergillus oryzae asparaginase. In the other clones, the encoded asparaginase has the amino acid substitutions indicated.<br>
<br>
Example 5: Saturation Libraries and Testing for Relative Activity<br>
<br>
Saturation libraries were prepared as in Example 2. The backbone was JN001, which has an additional K226R mutation compared to JN001N2. Clones on SC-glucose were constructed as described in Example 2. Screening for increased relative activity was performed as described in Example 3. The table shows variants that performed better than JN001.<br>
<br>
Table 3. Clone JN001N2 holds pJN001N2 encoding wild type Aspergillus oryzae asparaginase. In the other clones, the encoded asparaginase has the amino acid substitutions indicated.<br>
<br>
Example 6: Construction of site-directed variants based on ancestral sequence alignment<br>
The 90 retrieved asparaginase sequences of Table 4 below were aligned with CLUSTAL X (Thompson, J. D., Gibson, T. J., Plewniak, F., Jeanmougin, F. &amp; Higgins, D. G. (1997). The CLUSTAL_X windows interface: flexible strategies for multiple sequence alignment aided by quality analysis tools. Nucleic. Acids Res. 25, 4876-82.) using the default settings. The alignment was manually adjusted slightly.<br>
<br>
Asparaginase phylogenetic tree was constructed by the neighbor-joining method. (Saitou, N,&amp; Nei, M. (1987). The neighbor-joining method: a new method for reconstructing phylogenetic trees. Mol. Biol. Evol. 4, 406-425). Eighteen asparaginase sequences (Table 5) were selected for constructing a phylogenetic tree. These sequences were selected by removing the sequences with low homologies and avoiding repetition to cover as broad diversity as possible.<br>
<br>
After selecting well-aligned regions with GBLOCKS (Castresana, J. (2000). Selection of conserved blocks from multiple alignments for their use in phylogenetic analysis. Mol. Biol. Evol. 17, 540-52), the quartet-puzzling tree was inferred with Tree-Puzzle 5.0 (Schmidt, H. A., Strimmer, K., Vingron, M. &amp; von Haeseler, A. (2002), TREE-PUZZLE: maximum likelihood phylogenetic analysis using quartets and parallel computing. Bioinformatics 18, 502-4). The maximum likelihood tree was then searched by the CODEML in PAML3.13 (Yang, Z. (1997). PAML: a program package for phylogenetic analysis by maximum likelihood. Comput. Appl. Biosci. 13, 555-6) program package with the semi-automatic tree search option using the quartet-puzzling tree as the initial tree. The ancestral residues were then inferred with CODEML in PAML with the user-tree option, where the topology of the maximum likelihood tree (Figure 6) was used. Then, node19 and 20 were used as ancestral sequences inferred by the maximum likelihood method. The parsimony method with the software Bogen (Bogenpheil Co. Ltd.) was also used to infer ancestral sequences at the nodes node19 and node20 in the tree. These sequences are shown as V19 and V20 below.<br>
<br>
The inferred ancestral sequences: &gt;node19(SEQIDNO:9)<br>
PNIVILATGGTIAGAAASATVDTLIEAVPELKDLANVKGEQVANIGSEDMNNEILLKLGIVITHGT DTLEETAYFLNLTVKSDKPWLVGAMRPATAISADGPMNLYNAVAVAADKEARGKGVLWM NDRIGSARYVTKTNTTTVDAFKAPGYLGVIVNGKVYFFTRPHTTNSEFDVRKIDSLPKVDILYS YQNAAIDNGAKGIVYAGTGNGSVSKRAKAGLKKAGIVWRSSRVGNGLNPQKARILLMLALT QT<br>
<br>
&gt;node20 (SEQ ID NO: 10)<br>
PNIVILATGGTIAGAGASATVDTLIEAVPELKDLANVKGEQVANIGSEDMNNEILLKLGIVITHGT DTLEETAYFLNLTVKSDKPWLVGAMRPATAISADGPMNLYNAVAVAADKESRGKGVLWM NDRIQSARYVTKTNTTNVDAFKSPGPLGYIVNGKVYFFRSPHTTNSEFDVRKIDSLPKVDILY SYANALIDNGAKGIVHAGTGNGSISKRLKDALKKAGIVWRSSRVGQGLNPQKARILLMLALT QT<br>
<br>
&gt;V19(SEQIDN0: 11)<br>
MLALLSAAPALPNITILATGGTIAGAAASATQTTGYTAGAVGVDTLIAAVPELKDLANVAGEQV ANIDSEDITNAILLKLAKRVNALLADPDVDGIVITHGTDTLEETAYFLNLTLKSAKPNA/LVGAMR PATALSADGPLNLYNAVAVAADKAARGKGVLVAMNDRIGSARFVTKANTTDLDAFKAPEQG NLGAIANGKVYFFTSPAKRHTLDSEFDRAIDSLPKVDILYDYQDADADAYDAAIDNGAKGIVIA GSGNGSVSKRAKAAAKKAAKEGIIWRSSRVGNGWLDAADDAGVAAGSLNPQKARILLMLA LTKTKDPEEIQRYFDQY <br>
<br>
&gt;V20 (SEQ ID NO: 12)<br>
MLLLAAPALPNITILATGGTIAGAGASATDTTGYTAGKVGVDTLIAAVPELKDLANVAGEQVAN IDSEDITNEILLKLAKRVNALLDDPDVDGIVITHGTDTLEETAYFLNLTLKSDKPWLVGAMRPA TALSADGPLNLYNAVAVAADKAARGKGVLVAMNDRIGSARDVAKANTTDLDAFKSPFGPLG<br>
<br>
 <br>
AIANGKVYFFRSPAKRHTLDSEFDVRAIDSLPKVDILYSYANADADAYKALADNGAKGIVHAG SGNGSVSKRAKAALKKAAKEGIIWRSSRVNNGVLDAADDAKLGVAAGDLNPQKARILLMLA LTKTKDPKEIQRYFDEY<br>
<br>
The positions for site-directed mutagenesis were selected by aligning these inferred ancestral sequences with the targeting asparaginase sequence (Aspergillus oryzae asparaginase). Site-directed variants were then constructed on the backbone JN002N2 as described in <br>
Example 3.<br>
<br>
Example 7: Evaluation of constructed ancestral site-directed variants<br>
The site-directed variants of Example 6 were evaluated as described in Example 4 (except that the remaining activity after heat treatment was measured at 37°C instead of 55°C). JN001N2 was included, which holds pJN001N2 encoding wild type Aspergillus oryzae asparaginase. JN002N2 has the N70K substitution.<br>
<br>
<br>
Example 8: Differential Scanning Calorimetry<br>
<br>
Variants tested:<br>
<br>
•	JN002N2Y is wild-type Aspergillus oryzae asparaginase having the following substitution: N70K, the variant being expressed from S. cerevisiae.<br>
•	JN065N2Y is wild-type Aspergillus oryzae asparaginase having the following substitutions: N70K A323R T327V A349Q S351A V353I, the variant being expressed from S. cerevisiae.<br>
<br>
<br>
Sample Preparation for Differential Scanning Calorimetrv (DSC):<br>
<br>
•	Approx. 1.5 ml of sample was transferred to a Slide-A-Lyzer® Dialysis Cassette (Pierce, cat. # 66380, 10 kDa MWCO) and dialyzed against 500 ml of 50 mM MES buffer pH 6.0 for 1-1 Vi hrs with magnetic stirring in a cold room (approx. 5°C).<br>
•	The buffer was changed to a fresh batch of 500 ml of 50 mM MES pH 6.0 and left to dialyze overnight.<br>
•	The dialysis buffer was used for blanking in concentration determination by absorption measurement at 280 nm (A280) and as a reference for DSC.<br>
•	The volume of the sample was adjusted using the dialysis buffer until an A280 of approx. 0.5-0.55 was obtained. Employing an extinction coefficient calculated based on amino acid sequence data (Vector NTI v. 9.0.0), this corresponds to an enzyme concentration of approx. 1.5-1.7 mg/ml.<br>
•	The samples were degassed by vacuum suction and magnetic stirring for approx. 10<br>
minutes prior to loading into the DSC apparatus.<br>
<br>
<br>
DSC Data Recording and Processing:<br>
<br>
•	Apparatus: VP-DSC (MicroCal™)<br>
•	Scan interval: 20-80°C<br>
•	Scanrate: 60°C /h<br>
•	Data processing softw/are; MicroCal Origin© v. 4.10. The thermal denaturation temperature (Td) was determined as the temperature corresponding to the apex of the signal in the thermogram. The data are summarised in Table 8 below.<br>
<br>
Example 9: Asparaginase activity assay<br>
<br>
An asparaginase unit (ASNU) is defined as the amount of enzyme needed to generate 1.0 micromole of ammonia in 1 minute at 37°C and pH 7.0.<br>
Stock solutions<br>
50 mM Tris buffer, pH 7.0<br>
189 mM L-Asparagine solution<br>
1.5 M Trichloroacetic Acid (TCA)<br>
Nessler's reagent, Aldrich Stock No. 34,514-8 (Sigma-Aldrich)<br>
<br>
<br>
Enzvme reaction<br>
<br>
500 microL buffer<br>
100 microL L-asparagine solution<br>
350 microL water<br>
are mixed and equilibrated to 37°C.  100 microL of enzyme solution is added and the<br>
reactions are incubated at 37'C for 30 minutes. The reactions are stopped by placing on ice<br>
and adding 50 microL of 1.5M TCA. The samples are mixed and centrifuged for 2 minutes at<br>
20,000 g.<br>
<br>
<br>
Measurement of free ammonium<br>
50 microL of the enzyme reaction mixture is mixed with 100 microL of water and 50 microL of Nessler's reagent, and absorbance measured at 436 nm after 1 minute.<br>
<br>
<br>
Activity is compared to a known standard.<br>
<br>
Example 10: Application performance of the thermostable variant JN065N2 in treatment of French fries for acrylamide reduction<br>
<br>
Bintje potatoes (Grill-Kartoflen, Interfrugt Catering, Denmark) were manually peeled and cut into French fries using a French fry cutter (Taglia patate), size 0.8x0.8 cm. The potato sticks were divided into portions of 150 g (making sure that each portion consisted of sticks from different potatoes) and held in 400 ml de-ionized water until use. The potato sticks were blanched in two steps by dipping first in 4L de-ionized water at 85°C for 4 min and subsequently in 400 ml de-ionized water preheated to 70°C for 15 minutes. Enzyme treatment was done by dipping the blanched potato sticks for 1 min at 40°C in 300 ml enzyme solution (in deionised water) having 10,000 ASNU/L of the thermostable variant JN065N2 of Example 6 expressed from Aspergillus oryzae. For comparison a control sample with no enzyme treatment (fried right after blanching) was included. Samples were made in duplicate. After enzyme treatment the potato sticks were dried in a ventilated heating cupboard for 10 min at 85°C and parfried in vegetable oil for 1 min at 175°C. The samples were blast frozen and finally second fried 3 min at 175°C.<br>
<br>
The fries were blended and the acrylamide extracted using acetonitrile and an Automated Solvent Extractor (ASE from Dionex). The extract was treated with Carrez solution I and II, left overnight in the fridge and filtered using a 0.22 micrometer syringe filter before HPLC analysis (column: Dionex lonPac ICE-AS1, 9x250 mm, eluent: 5 mM HCI, detection: UV 202 nm). Acrylamide was identified and quantified by comparing with known standards.<br>
Results are given in Table 9 below.<br>
<br>
Acrylamide in the final French fry product has been reduced by 50% showing that the enzyme is active in this application. For the wild-type enzyme typical reductions are around 50-60%, while a dip in water without enzyme results in approximately 25% reduction.<br>
<br>
Example 11: Comparing thermostability of the variant JN065N2 and the wt enzyme in continuous treatment of French fries <br>
for acrylamide reduction<br>
<br>
Bintje potatoes (Grill-Kartoflen, Interfrugt Catering, Denmark) were manually peeled and cut into French fries using a French fry cutter (Taglia patate), size 0.8x0.8 cm. The potato sticks were divided into portions of 150 g (making sure that each portion consisted of sticks from different potatoes) and held in 400 ml de-ionized water until use. The potato sticks were blanched in two steps by dipping first in 4L de-ionized water at 85°C for 4 min and subsequently in 400 ml de-ionized water preheated to 70°C for 15 minutes. Enzyme treatment was done by holding the blanched potato sticks for 5 min at 60°C in 1500 ml enzyme solution (in deionised water) having 10.000 ASNU/L of either the thermostable variant JN065N2 of Example 6 expressed from Aspergillus oryzae or the wt enzyme. Every 5 min a new portion of blanched potato sticks was dipped in the enzyme bath in order to mimic a continuous use of the enzyme bath. Samples from the enzyme bath were taken every 10-15 min for a total period of 2-3 hours and frozen for later activity analysis. Treated potatoes were discarded.<br>
<br>
Upon fitting of the measured enzyme activities to an exponential decay curve, a half-life (TVz)<br>
of the enzyme can be estimated.    For the wt enzyme T!4 was 135 min and for the thermostable variant JN065N2 157 min, corresponding to a 16% increase.<br>
<br>
<br>
The invention described and claimed herein is not to be limited in scope by the specific embodiments herein disclosed, since these embodiments are intended as illustrations of several aspects of the invention. Any equivalent embodiments are intended to be within the scope of this invention. Indeed, various modifications of the invention in addition to those shown and described herein will become apparent to those skilled in the art from the foregoing description. Such modifications are also intended to fall within the scope of the appended claims. In the case of conflict, the present disclosure including definitions will control.<br>
<br>
Various references are cited herein, the disclosures of which are incorporated by reference in their entireties.<br>
<br>
<br>
<br>
WE CLAIM:<br>
1.	A method for preparing a polypeptide comprising:<br>
(a)	providing an amino acid sequence of a parent polypeptide having asparaginase activity;<br>
(b)	selecting at least one amino acid residue at a position in the sequence which corresponds to any of positions 54, 57, 68-74, 82-86, 88, 93-96, 102, 107, 111, 113, 115, 137, 139, 164, 165, 172, 176, 184-186, 194, 196, 201, 206, 209, 212, 214, 215, 219, 220, 223, 224, 226, 228, 231, 235, 246, 249, 255, 260, 262, 264, 266, 271, 275, 278-288, 290, 299, 306, 307, 309-321, 323, 325, 327-342, 349, 351, 353, 356-363, 365, 366 and/or 375 in SEQ ID NO: 1;<br>
(c)	modifying the sequence by substituting or deleting the selected amino acid residue or inserting one or more amino acid residues adjacent to the selected amino acid residue;<br>
(d)	producing a variant polypeptide having the modified sequence;<br>
(e)	testing the variant polypeptide for asparaginase activity and thermotoierance; and<br>
(f)	selecting a variant polypeptide having asparaginase activity and higher thermotoierance compared to the parent polypeptide.<br>
<br>
2.	A method according to claim 1, wherein the sequence of the parent polypeptide has at least 50% identity to any of <br>
SEQ ID NOs: 1-6, or a fragment thereof.<br>
<br>
3.	A method according to any of claims 1-2, wherein the sequence of the parent polypeptide has at least 50% identity to amino acids 80-378 of SEQ ID NO: 1, amino acids 80-378 of SEQ ID NO: 2, amino acids 80-374 of SEQ ID NO: 3, amino acids 80-378 of SEQ ID NO: 4, amino acids 80-379 of SEQ ID NO: 5, or amino acids 80-375 of SEQ ID NO: 6.<br>
<br>
4.	A method according to any of claims 1-2, wherein the sequence of the parent polypeptide comprises any of <br>
SEQ ID NOs: 1-6, or a fragment thereof.<br>
<br>
5.	A method according to any of claims 1-4, wherein the sequence of the parent polypeptide comprises amino acids 80-378 of SEQ ID NO: 1, amino acids 80-378 of SEQ ID NO: 2, amino acids 80-374 of SEQ ID NO: 3, amino acids 80-378 of SEQ ID NO: 4, amino acids 80-379 of SEQ ID NO: 5, or amino acids 80-375 of SEQ ID NO: 6.<br>
<br>
6.	A method according to any of claims 1-5, wherein the higher thermotoierance comprises a higher asparaginase activity after incubation at pH 6 at a temperature of at least 64°C for 20 minutes.<br>
<br>
7.	A method according to any of claims 1-6, wherein the higher thermotoierance comprises a higher relative asparaginase activity at pH 6 at 65°C compared to 37°C.<br>
 <br>
8.	A method according to any of claims 1-7, wherein the at least one amino acid residue selected has a position in the sequence which corresponds to any of positions 54, 57, 70, 83, 84, 86, 102, 137, 164, 196, 201, 228, 260, 262, 278, 283, 290, 307, 312, 323, 327, 334, 336, 337, 349, 351, 353, 366 and/or 375 in SEQ ID NO: 1<br>
<br>
9.	A method according to any of claims 1-8, wherein the sequence is modified by substituting the selected amino acid residue.<br>
<br>
10.	A method according to claim 9, wherein the sequence is modified by at least one of the following substitutions: 541, 57L, 70H/K/S, 83V, 84D, 86P, 102D, 137S, 164D, 1961, 201Q, 228V, 260K, 262D, 278H/Q, 283C, 290V, 307A, 312Y, 323R, 327V, 334F, 336C/G/L, 337F/I, 349Q, 351 A, 3531, 366P and/or 375T; wherein each position corresponds to a position of amino acids 1 to 378 of SEQ ID NO: 1.<br>
<br>
11.	A method according to any of claims 9-10, wherein the sequence is modified by at least one of the following substitutions: 70K, 323R, 327V, 349Q, 351A and/or 3531; wherein each position corresponds to a position of amino acids 1 to 378 of SEQ ID NO: 1.<br>
<br>
12.	A polypeptide which:<br>
<br>
(a)	has asparaginase activity;<br>
(b)	has at least 70% identity to amino acids 80 to 378 of SEQ ID NO: 1; and<br>
(c)	shows a residual asparaginase activity after heat treatment of at least 50% of the asparaginase activity without heat treatment, where heat treatment is incubation at pH 6 at a temperature of at least 64°C for 20 minutes.<br>
<br>
13.	A polypeptide which:<br>
(a)	has asparaginase activity;<br>
(b)	has at least 70% identity to amino acids 80 to 378 of SEQ ID NO: 1; and<br>
(c)	shows an asparaginase activity at pH 6 which is at least 25% higher at 65°C than at 37°C.<br>
<br>
14.	A polypeptide which:<br>
(a)	has asparaginase activity;<br>
(b)	has at least 80% identity to amino acids 80 to 378 of SEQ ID NO: 1; and<br>
(c)	comprises an amino acid difference compared to SEQ ID NO: 1 at a position<br>
corresponding to any of positions 54, 57, 68-74, 82-86, 88, 93-96, 102, 107, 111, 113, 115,<br>
137, 139, 164, 165, 172, 176, 184-186, 194, 196, 201, 206, 209, 212, 214, 215, 219, 220,<br>
223, 224, 226, 228, 231, 235, 246, 249, 255, 260, 262, 264, 266, 271, 275, 278-288, 290,<br>
299, 306, 307, 309-321, 323, 325, 327-342, 349, 351, 353, 356-363, 365, 366 and/or 375 in<br>
SEQIDNO: 1.<br>
15.	The polypeptide of claim 14, wherein the amino acid difference is a substitution.<br>
<br>
16.	The polypeptide of claim 15, which comprises a substitution in at least one of the following positions: 54, 57, 70, 83, 84, 86, 102, 137, 164, 196, 201, 228, 260, 262, 278, 283, 290, 307, 312, 323, 327, 334, 336, 337, 349, 351, 353, 366 and/or 375.<br>
<br>
17.	The polypeptide of any one of claims 15-16, which comprises a substitution at position 70.<br>
<br>
18.	The polypeptide of claim 17, which further comprises a substitution in at least one of the following positions: 54, 57, 83, 84, 86, 102, 137, 164, 196, 201, 228, 260, 262, 278, 283, 290, 307, 312, 323, 327, 334, 336, 337, 349, 351, 353, 366 <br>
and/or 375.<br>
<br>
19.	The polypeptide of claim 15, which comprises at least one of the following substitutions: 541, 57L, 69K/R, 70H/K/P/R/S, 72K/R, 82P, 83PA/, 84P/D, 85P, 86P, 88N, 93L, 94K, 95D, 96L, 102D, 1071, 111N, 113P, 115P, 137P/S/I, 1391, 164D/P, 165L, 172A, 176C, 184Y, 185N, 186A, 194E, 196E/I, 201P/Q, 206N, 209G, 212R, 214V, 215T, 219T, 220T, 223C/L/N, 224A, 228V, 231C, 235Q, 246C, 2491/LA/, 255Q, 260K, 262D, 264L, 266L/K, 271C, 275N, 278H/K/P/Q/R, 279N/RA/, 280D/E/P, 281 D/E, 283C, 286L/N/RA/, 290E/LA/, 299N, 306P, 307A/D/E, 311!/K/Q/R, 312N/RA//Y, 317D/E, 318G, 320V, 321V, 323R, 325S, 327V, 328C, 331Q, 334F, 336C/G/L/P, 337F/I/K/Q/R, 349Q, 351 A, 3531, 356M, 361K/R, 363E/L/P/Q, 365P, 366P and/or 375T.<br>
<br>
20.	The polypeptide of claim 19, which comprises at least one of the following substitutions: 541, 57L, 70H/K/S, 83V, 84D, 86P, 102D, 137S, 164D, 1961, 201Q, 228V, 260K, 262D, 278H/Q, 283C, 290V, 307A, 312Y, 323R, 327V, 334F, 336C/G/L, 337F/I, 349Q, 351A, 3531, 366P and/or 375T.<br>
<br>
21.	The polypeptide of claim 20, which comprises one of the following substitutions: N70H, N70K or N70S.<br>
<br>
22.	The polypeptide of claim 21, which comprises the substitution N70K.<br>
<br>
23.	The polypeptide of claim 22, which further comprises one of the following substitutions: 323R, 327V, 349Q, 351A and/or 3531.<br>
 <br>
24.	An isolated nucleic acid sequence comprising a nucleic acid sequence which encodes the polypeptide of any one of claims 12-23.<br>
<br>
25.	A nucleic acid construct comprising a nucleic acid sequence of claim 24 operably linked to one or more control sequences that direct the production of the polypeptide in a suitable expression host.<br>
<br>
26.	A recombinant expression vector comprising the nucleic acid construct of claim 25.<br>
<br>
27.	A recombinant host cell comprising the nucleic acid construct of claim 25 and/or the<br>
expression vector of claim 26.<br>
<br>
28.	A method for producing the polypeptide of any one of claims 12-23, the method<br>
comprising:<br>
(a)	cultivating the host cell of claim 27 to produce a supernatant comprising the polypeptide; and<br>
(b)	recovering the polypeptide.<br>
29.	A method for producing a food product, comprising:<br>
(a)	providing a food material; and<br>
(b)	treating the food material with the polypeptide of any of claims 12-23.<br>
<br>
30.	The method of claim 29, wherein the food material is heated after step (b).<br>
<br>
31.	The method of any of claims 29-30, wherein the method is for reduction of acrylamide in the food product.<br>
<br>
32.	The method of any of claims 29-31, wherein the food product is a cereal based product, such as bread, pastry, cake, pretzels, bagels, cookies, gingerbread, gingercake, breakfast cereals or crispbread; a vegetable based product, e.g. a potato based product, such as french fries, potato chips or crisps, fabricated potato snacks or croquettes; or a coffee<br>
based product.<br>
<br>
</td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/viewdoc.aspx?id=3X634KoMDdwCJyEP7FLZGA==&amp;amp;loc=egcICQiyoj82NGgGrC5ChA==" target="_blank" style="word-wrap:break-word;">http://ipindiaonline.gov.in/patentsearch/GrantedSearch/viewdoc.aspx?id=3X634KoMDdwCJyEP7FLZGA==&amp;amp;loc=egcICQiyoj82NGgGrC5ChA==</a></p>
		<br>
		<div class="pull-left">
			<a href="278269-method-for-selection-of-mobility-mechanism-for-mobility-management-in-ip-multimedia-service-capable-networks.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="278271-antifouling-composition.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>278270</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>5277/CHENP/2009</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>53/2016</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>23-Dec-2016</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>19-Dec-2016</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>08-Sep-2009</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>NOVOZYMES A/S</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td>KROGSHOEJVEJ 36, DK-2880 BAGSVAERD</td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>FRIIS, ESBEN, PETER</td>
											<td>GAMMELGAARDSVEJ 27A, DK-2730 HERLEV</td>
										</tr>
										<tr>
											<td>2</td>
											<td>YAMAGISHI, AKIHIKO</td>
											<td>1-17, KASUGACHOU 2-CHOME, NERIMA-KU, TOKYO 179-0074</td>
										</tr>
										<tr>
											<td>3</td>
											<td>MATSUI, TOMOKO</td>
											<td>TAKASU 1-24-1-1204, MIHAMA-KU, CHIBA 261-0004</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>C12N9/82</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>PCT/EP08/52759</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td>2008-03-07</td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td>07103862.4</td>
									<td>2007-03-09</td>
								    <td>EPO</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/278270-asparaginases by HTTrack Website Copier/3.x [XR&CO'2014], Thu, 04 Apr 2024 22:46:24 GMT -->
</html>
