<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/278411-polypeptide by HTTrack Website Copier/3.x [XR&CO'2014], Thu, 04 Apr 2024 23:06:44 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 278411:POLYPEPTIDE</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">POLYPEPTIDE</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>The present invention relates to A polypeptide comprising an Fc variant of an IgG Fc polypeptide, wherein said Fc variant comprises a serine at position 434, wherein said Fc variant exhibits increased binding to FcRn as compared to said human IgG Fc polypeptide, and wherein numbering is according to the EU index in Kabat et al.</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td>FORM 2<br>
THE PATENTS ACT 1970<br>
[39 OF 1970]<br>
&amp;<br>
THE PATENTS (AMENDMENT) RULES, 2006<br>
COMPLETE SPECIFICATION<br>
[See Section 10; rule 13]<br>
XENCOR INC., a US company of 111 West Lemon Avenue, Monrovia, California, 91016 United States of America,<br>
The following specification particularly describes and ascertains the nature of this invention and the manner in which it is to be performed:<br>
 <br>
Fc VARIANTS WITH ALTERED BINDING TO FcRn<br>
[1] This application claims benefit under 35 U.S.C. §119(e) to USSN 60/627,763, filed November 12, 2004; USSN, 60/642,886, filed January 11, 2005; USSN 60/649,508, filed February 2, 2005; USSN 60/662,468, filed March 15, 2005; 2005; USSN 60/669,311, filed April 6, 2005; USSN 60/681,607, filed May 16, 2005; USSN 60/690,200, filed June 13, 2005; USSN 60/696,609, filed July 5, 2005; USSN 60/703,018, filed July 27, 2005; and USSN'60/726,453, filed October 12, 2005, all entirely incorporated by reference.<br>
FIELD OF THE INVENTION<br>
[2] The present application relates-to optimized IgG immunoglobulin variants, engineering methods for their generation, and their application, particularly for therapeutic purposes.<br>
BACKGROUND OF THE INVENTION<br>
[3] Antibodies are immunological proteins that bind a specific antigen. In most mammals, including humans and mice, antibodies are constructed from paired heavy and light polypeptide chains. Each chain is made up of individual immunoglobulin (Ig) domains, and thus the generic term immunoglobulin is used for such proteins. Each chain is made up of two distinct regions, referred to as the variable and constant regions, the fight and heavy chain variable regions show significant sequence diversity between antibodies, and are responsible for binding the target antigen. The constant regions show less sequence diversity, and are responsible for binding a number of natural proteins to elicit important biochemical events. In humans there are five different classes of antibodies including IgA (which includes subclasses lgA1 and lgA2), IgD, IgE, IgG (which includes subclasses lgG1, lgG2, lgG3, and lgG4), and IgM. The distinguishing features between these antibody classes are their constant regions, although subtler differences may exist in the V region. Figure 1 shows an lgG1 antibody, used here as an example to describe the general structural features of immunoglobulins. IgG antibodies are tetrameric proteins composed of two heavy chains and two light chains. The IgG heavy chain is composed of four immunoglobulin domains linked from N- to C-terminus in the order VH-CH1-CH2-CH3, referring to the heavy chain variable domain, heavy chain constant domain 1, heavy chain. constant domain 2, and heavy chain constant domain 3 respectively (also referred to as VH-Cy1 Cy2-Cy3, referring to the heavy chain variable domain, constant gamma'1 domain, constant gamma 2 domain, and constant gamma 3 domain respectively). The IgG light chain is composed of two immunoglobulin domains linked from N- to C-terminus in the order VL-CL, referring to the light chain variable domain and the light chain constant domain respectively.<br>
 <br>
[4] The variable region of an antibody contains the antigen binding determinants of the molecule, and thus determines the specificity of an antibody for its target antigen. The variable region is so named because it is the most distinct in sequence from other antibodies within the same class. The majority of sequence variability occurs in the complementarity determining regions (CDRs). There are 6 CDRs total, three each per heavy and light chain, designated VH CDR1, VH CDR2, VH CDR3, VL CDR1, VL CDR2, and VL CDR3.   The variable region outside of the CDRs is referred to as the framework (FR) region.   Although not as diverse as the CDRs, sequence variability does occur in the FR region between different antibodies.   Overall, this characteristic architecture of antibodies provides a stable scaffold (the FR region) upon which substantial antigen binding diversity (the CDRs) can be explored by the immune system to obtain specificity for a broad array of antigens. A number of high-resolution structures are available for a variety of variable region fragments from different organisms, some unbound and some in complex with antigen. The sequence and structural features of antibody variable regions are well characterized (Morea ef al., 1997, Biophys Chem 68:9-16; Morea ef a/., 2000, Methods 20:267-279, entirely incorporated  by  reference),   and  the  conserved  features  of antibodies  have enabled  the development of a wealth of antibody engineering techniques (Maynard ef a/., 2000, Annu Rev Biomed Eng 2:339-376, entirely incorporated by reference).   For example, it is possible to graft the CDRs from one antibody, for example a murine antibody, onto the framework region of another antibody, for example a human antibody.    This process, referred to in the art as "humaru'zaiion", enables generation of less immunogenic antibody therapeutics from nonhuman antibodies.   Fragments including the variable region can exist in the absence of other regions of the antibody, including for example the antigen binding fragment (Fab) including VH-Cy1 and VH-CL, the variable fragment (Fv) including VH and VL, the single chain variable fragment (scFv) including VH and VL linked together in the same chain, as well as a variety of other variable region fragments  (Little ef at., 2000, Immunol Today 21:364-370,  entirely incorporated by reference).<br>
[5] The Fc region of an antibody interacts with a number of Fc receptors and ligands, imparting an array of important functional capabilities referred to as effector functions. For IgG the Fc region, as shown in Figures 1 and 2, comprises Ig domains Cy2 and Cy3 and the N-terminal hinge leading into Cy2. An important family of Fc receptors for the IgG class are the Fc gamma receptors (FcyRs). These receptors mediate communication between antibodies and the cellular arm of the immune system (Raghavan ef al., 1996, Annu Rev Cell Dev Biol 12:181-220; Ravetch ef a/., 2001, Annu Rev Immunol 19:275-290, both entirely incorporated by reference). In humans this protein family includes FcyRI (CD64), including isoforms FcyRIa, FcyRIb, and FcyRIc; FcyRll (CD32), including isoforms FcyRlla (incfuding affotypes H131 and R131), FcyRllb (including -FcyRllb-1 and FcyRllb-2), and FcyRllc; and FcyRll I (CD16), including isoforms FcyRllia (including<br>
 <br>
allotypes V158 and F158) and FcyRlllb (including allotypes FcyRlllb-NA1 and FcyRlllb-NA2) (Jefferis et al., 2002, Immunol Lett 82:57-65, entirely incorporated by reference). These receptors typically have an extracellular domain that mediates binding to Fc, a membrane spanning region, and an intracellular domain that may mediate some signaling event within the cell.    These receptors are expressed in a variety of immune cells including monocytes,  macrophages, neutrophils, dendritic cells, eosinophils, mast cells, platelets, B cells, large granular lymphocytes, Langerhans' cells, natural killer (NK) cells, and yy T cells.   Formation of the Fc/FcyR complex recruits these effector cells to sites of bound antigen, typically resulting in signaling events within the cells  and   important subsequent immune  responses such  as  release of inflammation mediators, B cell activation, endocytosis, phagocytosis, and cytotoxic attack.   The ability to mediate cytotoxic and phagocytic effector functions is a potential mechanism by which antibodies destroy targeted cells.    The cell-mediated reaction wherein nonspecific cytotoxic cells that express FcyRs recognize bound antibody on a target cell and subsequently cause lysis of the target cell is referred to as antibody dependent cell-mediated cytotoxicity (ADCC) (Raghavan ef a/., 1996, Annu Rev Cell Dev Biol 12:181-220; Ghetie et al., 2000, Annu Rev Immunol 18:739-766; Ravetch ef al., 2001, Annu Rev Immunol 19:275-290, all entirely incorporated by reference). The cell-mediated reaction wherein nonspecific cytotoxic cells that express FcyRs recognize bound antibody on a target cell and subsequently cause phagocytosis of the target cell is referred to as antibody dependent cell-mediated phagocytosis (ADCP). A number of structures have been solved of the extracellular domains of human FcyRs, including FcyRlla (pdb accession code 1H9V, entirely incorporated by reference)(Sondermann et al., 2001, J Mol Biol 309:737-749, entirely  incorporated   by   reference)  (pdb  accession  code   1FCG,   entirely  incorporated   by reference)(Maxwell ef al., 1999, Nat Struct Biol 6:437-442, entirely incorporated by reference), FcyRIIb (pdb accession code 2FCB, entirely incorporated by reference)(Sondermann et al, 1999, Embo J 18:1095-1103, entirely incorporated by reference); and FcyRlllb (pdb accession code 1E4J, entirely incorporated by reference)(Sondermann ef al., 2000, Nature 406:267-273, entirely incorporated by reference.).  All FcyRs bind the same region on Fc, at the N-terminal end of the Cy2 domain and the preceding hinge, shown in Figure 1.   This interaction is well characterized structurally  (Sondermann   ef  al.,  2001,   J  Mol Biol 309:737-749,   entirely  incorporated   by reference), and several structures of the human Fc bound to the extracellular domain of human FcyRlllb    have    been    solved    (pdb    accession    code    1E4K,    entirely    incorporated    by reference)(Sondermann ef al., 2000, Nature 406:267-273, entirely incorporated by reference) (pdb accession codes 1IIS and 1IIX, entirely incorporated by reference)(Radaev ef al., 2001, J Biol Chem 276:16469-16477, entirely incorporated by reference), as well as has the structure of the  human  IgE   Fc/FcsRIcc  complex  (pdb accession  code  1F6A,  entirely  incorporated  by reference}(Garman ef al., 2000, Nature 406:259-266, entirely incorporated by reference).<br>
 <br>
[6] The different IgG subclasses have different affinities for the FcyRs, with lgG1 and lgG3 typically binding substantially better to the receptors than lgG2 and lgG4 (Jefferis et ai, 2002, Immunol Lett 82:57-65, entirely incorporated by reference). Ail FcyRs bind the same region on IgG Fc, yet with different affinities: the high affinity binder FcyRI has a Kd for lgG1 of 10~8 M-1, whereas.the low affinity receptors FcyRI I and FcyRIII generally bind at 10"6 and 10"5 respectively. The extracellular domains of FcyRllla and FcyRlllb are 96% identical; however FcyRlllb does not have a intracellular signaling domain. Furthermore, whereas FcyRI, FcyRlla/c, and FcyRllla are positive regulators of immune complex-triggered activation, characterized by having an intracellular domain that has an immunoreceptor tyrosine-based activation motif (ITAM), FcyRIib has an immunoreceptor tyrosine-based inhibition motif (ITIM) and is therefore inhibitory. Thus the former are referred to as activation receptors, and FcyRIib is referred to as an inhibitory receptor. The receptors also differ in expression pattern and levels on different immune cells. Yet another level of complexity is the existence of a number of FcyR polymorphisms in the human proteome. A particularly relevant polymorphism with clinical significance is V158/F158 FcyRllla. Human fgG1 binds with greater affinity to the V158 allotype than to the F158 allotype. This difference in affinity, and presumably its effect on ADCC and/or ADCP, has been shown to be a significant determinant of the efficacy of the anti-CD20 antibody rituximab (Rituxan®, Biogenldec). Patients with the V158 allotype respond favorably to rituximab treatment; however, patients with the lower affinity F158 allotype respond poorly {Cartron. ef a/., 2002, Blood 99:754-758, entirely incorporated by reference). .'Approximately 10-20% of humans are V158A/158 homozygous, 45% are V158/F158 heterozygous, and 35-45% of humans are F158/F158 homozygous (Lehrnbecher era/., 1999, Blood 94:4220-4232; Cartron ef ai, 2002, Blood 99:754-758, all entirely incorporated by reference). Thus 80-90% of humans are poor responders, i.e., they have at least one allele of the F158 FcyRllla.<br>
[7] An overlapping but separate site on Fc, shown in FigunS 1, serves as the interface for the complement protein C1q. In the same way that Fc/FcyR binding mediates ADCC, Fc/C1q binding mediates complement dependent cytotoxicity (CDC). C1q forms a complex with the serine proteases C1r and C1s to form the C1 complex. C1q is capable of binding six antibodies, although binding to two IgGs is sufficient to activate the complement cascade. Similar to Fc interaction with FcyRs, different IgG subclasses have different affinity for C1q, with lgG1 and lgG3 typically binding substantially better to the FcyRs than lgG2 and lgG4 (Jefferis ef a/., 2002, Immunol Lett 82:57-65, entirely incorporated by reference).<br>
[8] In IgG, a site on Fc between the Cg2 and Cg3 domains (Figure 1) mediates interaction with the neonatal receptor FcRn,. the binding of which recycles endocytosed antibody from the endosome back to the bloodstream (Raghavan et al., 1996, Annu Rev Cell Dev Biol 12:181-220;<br>
 <br>
Ghetie et a!., 2000, Annu Rev Immunol 18:739-766, both entirely incorporated by reference). This . process, coupled with preclusion of kidney filtration due to the large size of the full-length molecule, results in favorable antibody serum half-lives ranging from one to three weeks, Binding of Fc to FcRn also plays a key role in antibody transport. The binding site on Fc for FcRn is also the site at which the bacterial proteins A and G bind. The tight binding by these proteins is typically exploited as a means to purify antibodies by employing protein A or protein G affinity chromatography during protein purification. Thus the fidelity of this region on Fc is important for both the clinical properties of antibodies and their purification. Available structures of the rat Fc/FcRn complex (Burmeister et at., 1994, Nature, 372:379-383; Martin et al., 2001, Mol Cell 7:867-877, both entirely incorporated by reference), and of the complexes of Fc with proteins A and G (Deisenhofer, 1981, Biochemistry 20:2361-2370; Sauer-Eriksson et al., 1995, Structure 3:265-278; Tashiro et al., 1995, Curr Opin Struct Biol 5:471-481, all entirely incorporated by reference), provide insight into the interaction of Fc with these proteins. The FcRn receptor is also responsible for the transfer of IgG to the neo-natal gut and to the lumen of the intestinal epithelia in adults (Ghetie and Ward, Annu. Rev. Immunol., 2000, 18:739-766; Yoshida et al., Immunity, 2004, 20(6):769-783, both entirely incorporated by reference).<br>
[9] Studies of rat and human Fcy domains have demonstrated the importance of some Fc residues to the binding of FcRn. The rat and human sequences have about 64% sequence identity in the Fc regions (residues 237-443 in the numbering of Kabat et al.). See figures 3, 4, and 5 for the rat/human alignments of Fc, FcRn heavy chain, and FcRn light chain (beta-2-. microglobulin). A mode) of the human Fc/FcRn complex has been built from the existing structure of the rat Fc/FcRn complex (Martin et al., 2001, Mol Cell 7:867-877, entirely incorporated by reference). The rat and human sequences share some residues that are critical for FcRn binding, such as H310 and H435 (Medesan et al., 1997 J. Immunol. 158(5):221-7; Shields et al., 2001, J. Biol. Chem. 276(9):6591-6604, both entirely incorporated by reference). In many positions, however, the human and rat proteins have different amino acids, giving the residues in the human sequence different environments, and possibly a different identities, than in the rat sequence. This variability limits the ability to transfer characteristics from one homoiog to the other homolog.<br>
[10] In the murine Fey, random mutation and phage display selection at the sites, T252, T254, and T256 lead'to a triple mutant, T252UT254S/T256F, that has a 3.5-fold increase in FcRn affinity and a 1.5-fold increase in serum half-life (Ghetie et al., 1997, Nat. Biotech. 15(7): 637-640, entirely incorporated by reference).<br>
[11] The crystal structures of the rat Fc/FcRn complex identified important Fc residues for FcRn binding (Burmeister et al. Nature. 372:379-383 (1994); Martin et al. Molecular Cell. 7:867-877 (2001), both entirely incorporated by reference). The original Fc/FcRn complex structure was<br>
 <br>
solved in 1994 to a resolution of 6 A (Table 2a, Burmeister et al. Nature. 372:379-383 (1994), entirely incorporated by reference). The higher resolution structure, solved in 2001 by Marin et al, showed a more detailed view of the side chains positions (Martin et ai. Molecular Cell. 7:867-877 (2001), entirely incorporated by reference). This crystal structure1 of rat Fc bound to rat FcRn was solved using an Fc dimer with one monomer containing the mutations T252G/1253G/T254G/H310E/H433E/H435E, which disrupt FcRn binding, and one monomer containing a wild-type Fc monomer.<br>
[12] Mutational studies in human Fey have been done on some of the residues that are important for binding to FcRn and have demonstrated have demonstrated an increased serum half-life. In human Fcy1, Hinton et al. mutated three residues individually to the other 19 common amino acids. Hinton et al., found that some point mutants a double mutant increased the FcRn binding affinity (Hinton et al., 2004, J. Biol. Chem. 279(8): 6213-6216, entirely incorporated by reference). Two mutations had increased half-lives in monkeys. Shields et al. mutated residues, almost exclusively to Ala, and "studied their binding to FcRn and the FcvR's (Shields et al., 2001, J. Bio], Chem., 276{9):6591-6604, entirely incorporated by reference).<br>
[13] Dall'Acqua et al. used phage display to select for Fc mutations that bound FcRn with increased affinity (DalP Acqua et al. 2002, J. Immunol. 169:5171-5180, entirely incorporated by reference). The DNA sequences selected for were primarily double and triple mutants. The reference expressed the proteins encoded by many of their selected sequences and found some that bound to FcRn more tightly than the wild-type Fc.<br>
[14] The administration of antibodies and Fc fusion proteins as therapeutics requires injections with a prescribed frequency relating to the clearance and half-life characteristics of the protein. Longer in vivo half-lives allow more seldom injections or lower dosing, which is clearly advantageous. Although the past mutations in the Fc domain have lead to some proteins with increased FcRn binding affinity and in vivo half-lives, these mutations have not identified the optimal mutations and enhanced in vivo half-life.<br>
[15] One feature of the Fc region is the conserved N-linked glycosylation that occurs at N297, shown in Figure 1. This carbohydrate, or oligosaccharide as it is sometimes referred, plays a critical structural and functional role for the antibody, and is one of the principle reasons that antibodies must be produced using mammalian expression systems. Umana et al, 1999, Nat Biotechnol 17:176-180; Davies et al., 2001, Biotechnol Bioeng 74:288-294; Mimura et a!., 2001, J Biol Chem 276:45539-45547.; Radaev ef al., 2001, J Biol Chem 276:16478-16483; Shields ef al., 2001, J Biol Chem 276:6591-6604; Shields etal., 2002, J Biol Chem 277:26733-26740; Simmons<br>
 <br>
et a/., 2002, J Immunol Methods 263:133-147; Radaev et al., 2001, J Biol Chem 276:16469-16477; and Krapp et a/., 2003, J Mol Biol 325:979-989, all entirely incorporated by reference). [16] Antibodies have been developed for therapeutic use. Representative publications related to such therapies include Chamow et ai, 1996, Trends Biotechnol 14:52-60; Ashkenazi et a/., 1997, CurrOpin Immunol 9:195-200, Cragg et ai, 1999, Curr Opin Immunol 11:541-547; Glennie et at., 2000, Immunol Today 21:403-410, McLaughlin et al, 1998, J Clin Oncol 16:2825-2833, and Cobleigh etal., 1999, J Clin Oncol 17:2639-2648, all entirely incorporated by reference. Currently for anticancer therapy, any small improvement in mortality rate defines success. Certain IgG variants disclosed herein enhance the capacity of antibodies to limit further growth or destroy at least partially, targeted cancer cells.<br>
[17] Anti-tumor potency of antibodies is via enhancement of their ability to mediate cytotoxic effector functions such as ADCC, ADCP, and CDC. Examples include Clynes et a/., 1998, Proc Natl Acad Sci USA 95:652-656; Clynes ef al., 2000, Nat Med 6:443-446 and Cartron et ai, 2002, Blood 99:754-758, both entirely incorporated by reference.<br>
[18] Human lgG1 is the most commonly used antibody for therapeutic purposes, and the majority of engineering studies have been constructed in this context. The different isotypes of the IgG class however, including lgG1, lgG2, lgG3, and lgG4, have unique physical, biological, and clinical properties. There is a need in the art to design improved IgG1, !gG2, JgG3, and IgG4 variants. There is a further need to design such variants to improve binding to FcRn and/or increase in vivo half-life as compared to native IgG polypeptides. The present application meets these and other needs. SUMMARY OF THE INVENTION<br>
[19] The present invention discloses the generation of novel variants of Fc domains, including those found in antibodies, Fc fusions, and immuno-adhesions, which have an increased binding to the FcRn receptor and longer serum retention in vivo. An additional aspect of the invention is the increase in FcRn binding over, wild type specifically at lower pH, about pH 6.0, to facilitate Fc/FcRn binding in the endosome. An additional aspect of the present invention is the preferential binding of the designed variants at about pH 6 compared to their binding at about pH 7.4 to facilitate the re-release of Fc into blood following cellular recycling.<br>
[20] A further aspect of the present invention relates to the design of Fc variants with decreased binding to FcRn and decreased in vivo half-lives. Such proteins comprising mutations to reduce FcRn affinity and/or the in vivo half-lives are useful in many therapies and diagnostics, including the delivery and monitoring of radiotherapeutics wherein, ideally, the half-life of the radiolabel is about equal to the in vivo half-life of its protein conjugate.<br>
 <br>
[213 A further aspect of the invention relates to the alteration of the Fc domain binding to the FcR's, e.g. in humans, FcgRl, FcgRlla, FcgRllb, FcgRllla. These receptors are responsible for inducing the various effector functions of antibodies. Therefore, a further aspect of the invention relates to the alteration of the Fc domain effector functions, such as antibody-dependent cell-mediated toxicity (ADCC), complement-dependent cytotoxicity (CDC), and antibody dependent cell-mediated phagocytosis (ADCP).<br>
[22] A further aspect of the invention relates to Fc variants that contained both altered FcRn binding and altered Fcg binding to affect both the in vivo half life and the efffector functions of the Fc-comprising protein. For example, these variants may have increased half-life in vivo as well as improved ADCC. The variants, for example, may have increased half-life and decreased CDC.<br>
[23] In a further aspect, the invention provides recombinant nucleic acids encoding the variant Fc proteins, expression vectors, and host cells.<br>
[24] In an additional aspect, the invention provides methods of producing a variant Fc-comprising protein comprising cuituring the host cells of the invention under conditions suitable for expression of the protein.<br>
[25] in a further aspect, ihe invention  provides pharmaceutical compositions comprising a variant Fc protein of the invention and a pharmaceutical carrier.<br>
[26] In a further aspect, the invention provides methods for treating disorders comprising administering a protein comprising a variant Fc of the invention to a patient.<br>
[27jln an additional aspect, the invention provides an Fc variant region of a parent Fc polypeptide comprising at least one modification in the Fc region of said- parent polypeptide, wherein said variant protein exhibits altered binding to FcRn as compared to the parent polypeptide, and wherein said Fc variant comprises at least one modification selected from the group consisting of: 246H, 246S, 247D, 247T, 248H, 248P, 248Q, 248R, 24SY, 249T, 249W, 251D, 251.E, 251H, 2511, 251K, 251M, 251N, 251T, 251V, 251Y, 252F, 252L, 253L, 253T, 253V, 254H, 254L, 254N, 254T, 254V, A254N, 255E, 255F, 255H, 255K, 255S, 255V, 256E, 256H, 256V, 257A, 257C, 257D, 257E, 257F, 257G, 257H, 257I, 257K, 257L, 257M, 257N, 257Q, 257R,257S, 257T, 257V, " 257W, 257Y, 258R, 258V, 279A, 279C, 279D, 279F, 279G, 279H, 279I, 279K, 279L, 279M, 279N, 279P, 279Q, 279R, 279S, 279T, 279W, 279Y, 280E, 280H, A281A, A281D, A281S, A281T, 282D, 282F, 282H, 282I, 282T, 283F, 2S3I, 283L, 283Y, 284H, 284K, 284P, 284Q, 284R, 284S, 284Y, 285S, 285V, 286#, 286L, 287H, 287S, 287V, 287Y, 288H, 288Q, 288R, 288S, 305H, 305T,<br>
 <br>
306F, 306H, 306!, 306N, 306T, 306V, 306Y, 307D, 307V, 307Y, 308A, 30SC, 30SD, 308E, 308F, 308G, 308H, 3081, 308K, 308L, 308M, 308N, 308P, 30SQ, 30SR, 308S, 308T.-308W, 308Y, 309F, 309H, 3091, 309N, 309P, 309Q, 309V, 309Y, 310Kr 310N, 310T, 311H, 311L, 311S, 311T, 311V, 311W, 312H, 313Y, 315E, 315G, 315H, 315Q, 315S, 315T, 317H, 317S, 339P, 340P, 341S, 374H, 374S, 376H, 376L, 378H, 37SN, 380T, 380Y, 382H, 383H, 383K, 383Q, 384E, 384G, 384H, 385A, 385C, 385D, 385E, 385F, 385H, 385I, 385K, 385L, 385M, 385N, 385P, 385Q, 385R, 385S, 385T, 385V, 385W, 385Y, 386E, 386H, 386K, 387#, 387A, 387H, 387K, 387Q, 389E, 389H, 426E, 426H, 426L, 426N, 426R, 426V, 426Y, 427I, 429D, 429F, 429K, 429N, 429Q, 429S, 429T, 429Y, 430D, 430H, 430K, 430L, 430Q, 430Y, 431G, 431H, 4311, 431P, 431S, 432F, 432H, 432N, 432S, 432V^ 433E, 433N, 433P, 433R, 433S, 434H, 434Q, 434S, 434Y, 435N, 436E..436F, 436H, 436L, 436Q, 436V, 436W, 437E, 437V, 438E, 438H and 438K, wherein numbering is according to the EU Index and A is an insertion after the identified position and # is a deletion of that identified position.<br>
[28]ln a further aspect, the invention provides Fc variants comprising at least one modification selected from the group consisting of: 246H, 246S, 247D, 247T, 248H, 248P, 248Q, 248R, 248Y, 249T, 249W, 251D, 251E, 251H, 2511, 251K, 251M, 251N, 251T, 251V, 251Y, 252L, 253L, 253T, 253V, 254H, 254L, 254N, 254V, A254N, 255E, 255F, 255H, 255K, 255S, 255V, 256H, 256V, 257A, 257C, 257D, 257E, 257F, 257G, 257H, 257I, 257K, 257L, 257M, 257N, 257Q, 257R, 257S, 257T, 257V, 257W, 257Y, 258R, 258V, 279A, 279C, 279D, 279F, 279G, 279H, 279I, 279K, 279M, 279N, 279P, 279Q, 279R, 279S, 279T, 279W, 279Y, 280H, A281A, A281D, A281S, A281T, 282D, 282F, 282H, 282!, 282T, 283F, 2831, 283L, 283Y, 284H, 284K, 284P, 284Q, 284R, 284S, 284Y, 285S, 285V, 286#, 286L, 287H, 287S, 287V, 287Y, 288H, 288Q, 288S, 305H, 305T, 306F, 306H, 306!, 306N, 306T, 306V, 306Y, 307D, 307V, 307Y, 308C, 308Ef 308F, 308G, 308H, 308I, 308K, 308L, 308M, 308N, 308P, 308Q, 308R, 308S, 308W, 308Y, 309F, 309H, 309N, 309Q, 309V, 309Y, 310K, 310N, 310T, 311L, 311T, 311V, 311W, 312H, 313Y, 315E, 315G, 315H, 315Q, 315S, 315T, 317H, 317S, 339P, 340P, 341S, 374H..374S, 376H, 3/6L, 378H, 378N, 380T, 380Y, 382H, 383H, 383K, 383Q, 384E, 384G, 384H, 385A, 385C, 385F, 385H, 385i, 385K, 385L, 385M, 385N, 385P, 385Q, 385S, 385T, 385V, 385W, 385Y, 386E, 386H, 386K, 387#, 387A, 387H, 387K, 387Q, 389E, 389H, 426E, 426H, 426L, 426N, 426R, 426V, 426Y, 427I, 429D, 429F, 429K, 429N, 429Q, 429S, 429T, 429Y, 430D, 430H, 430K, 430L, 430Q, 430Y, 431G, 431H, 4311, 431P, 431S, 432F, 432H, 432N, 432S, 432V, 433E, 433N, 433P, 433S, 434H, 434Q, 434S, 435N, 436E, 436F, 436L, 436V| 436W, 437E, 437V, 438H, and 438K. [29] In  an  additional aspect,  the invention  provides  Fc variants comprising at least one modification selected from the group consisting of: 246H, 246S, 247D, 247T, 248P, 248Q, 248Y, 249T, 249W, 251D, 251E, 251H, 2511, 251T, 251V, 252L, 253L, 253T, 253V, 254H, 254L, 254N, 254V, A254N, 255E, 255H, 255K, 255V, 256H, 256V, 257A, 257C, 257F, 257G, 257I, 257L,<br>
 <br>
257M, 257N, 257Q, 257S, 257T, 257V, 257W, 257Y, 258V, 279A, 279C, 279F, 2791, 279P, 279S, 279T, 279W, 279Y, A281A, A281D, A281S, A281T, 282F, 282!, 282T, 283F, 283I, 283L, 283Y, 284P, 285V, 286#, 286L, 287V, 288Q, 288S, 305T, 306F. 306H, 306I, 306N..306T, 306V, 306Y, 307V, 308C, 308F, 308G, 308L, 308M, 308N, 308P, 308Q, 308S, 308 W, 308Y, 30SF, 309N, 309Q, 309V, 309Y, 310T, 311L, 311T, 311V, 311W, 313Y, 315G, 315Q, 315S, 315T, 339P, 340P, 341S, 374H, 374S, 376L, 378H, 378N, 380T, 330Y, 382H, 383Q, 384E, 384G, 384H, 385A, 385C, 385F, 385I, 385L, 385M, 385N, 385P, 385Q, 385S, 385T, 385V, 385W, 385Y, 386E, 386H, 386K, 387#, 387A, 387H, 387K, 387Q, 389H, 426L, 426N, 426V, 426Y, 427I, 429D, 429F, 429K, 429N, 429Q, 429S, 429T, 429Y, 430L, 431G, 4311, 431P, 431S, 432F, 432H, 432V, 433E, 433N, 433P, 433S, 434H, 434Q, 434S, 435N, 436F, 436L, 436V, 436W, 437E, and 437V.<br>
BRIEF DESCRIPTION OF THE DRAWINGS<br>
[30] Figure 1. Antibody structure and function. Shown is a model of a full length human lgG1<br>
antibody, modeled using a humanized Fab structure from pdb accession code 1CE1 (James et.<br>
a/., 1999, J Mol Biol 289:293-301, entirely incorporated by reference) and a human lgG1 Fc<br>
structure from pdb accession code 1DN2 (DeLano et al., 2000, Science 287:1279-1283, entirely<br>
incorporated by reference). The flexible hinge that links the Fab and Fc regions is not shown.<br>
lgG1 is a homodimer of heterodimers, made up of two light chains and two heavy chains. The Ig<br>
domains that comprise the antibody are labeled, and include VL and CL for the light chain, and VH,<br>
Cgamma.1 (Cy1), Cgamma2 (Cy2), and Cgamma3 (Cy3) for the heavy chain. The Fc region is<br>
labeled. Binding sites for relevant proteins are labeled, including the antigen binding site in the<br>
variable region, and the binding sites for FcyRs, FcRn, Clq, and proteins A and G in the Fc<br>
region.<br>
[31] Figure 2. Human IgG sequences used in the present invention with the EU numbering as in<br>
Kabat et al.<br>
[32] Figure 3. Example human and rodent IgG sequences used, in the present invention with the<br>
EU numbering as in Kabat.<br>
[33] Figure 4.   Example human and rodent FcRn heavy chain sequences used in the present<br>
invention.<br>
[34] Figure 5.   Example human and rodent beta-2-microglobulin sequences used in the present<br>
invention.<br>
[35] Figure 6.  A human Fc/FcRn complex model created from the rat structures (Burmeister et<br>
al., 1994, Nature, 372:379-383; Martin et al., 2001, Mol Cell 7:867-877, both entirely incorporated<br>
by reference). Some histidine residues are shown in space-fiiling atoms on the FcRn chains (light<br>
grey) and Fc polypeptide (dark grey).<br>
[36] Figure 7.  Illustration of some concepts used in the design of variants comprising insertions<br>
or deletions.<br>
 <br>
[37] Figure 8. Variants of the present invention.<br>
[38] Figure 9. Variants of the present invention.<br>
[39] Figure 10. Variants of the present invention.<br>
[40] Figure 11. Diagram of the vector pcDNA3.1 Zeo+, which may be used in the construct of Fc<br>
variants.<br>
[41] Figure 12. Competition FcRn binding data of wild-type Fc and Fc variants of the present<br>
invention. In each panel, the Fc variants of the present invention are shown as the left (red or<br>
dark grey) curve and the wild-type trastuzumab is shown as the right (blue or light grey) curve.<br>
[42] Figure 13.  Summary of FcRn binding properties of the Fc variants. The columns from right<br>
to left show the FcRn binding modifications, the immunoglobulin used, other modifications, the-<br>
relative FcRn affinity by AlphaScreen™ competition assays compared to wild type (median<br>
value), the number of assays performed, and a reference number of the protein.   Relative FcRn<br>
affinity numbers greater than 1.0. demonstrate Increased binding over wild type.<br>
[43] Figure 14. FcRn binding data of Fc variants of the present invention. The Fc variants are in<br>
alemtuzumab or trastuzumab. The fold-increased binding compared to wild type are shown.<br>
[44] Figure 15.    Summary of surface'plasmon resonance experiments of Fc variants with<br>
improved binding to FcRn. The bar graph shows the fold-increase in FcRn binding affinity of each<br>
variant relative to wild-type Fc domain.<br>
[45] Figure 16.   Surface plasmon resonance experiments of wild-type antibody and variants of<br>
the present invention. The traces shown are the association and dissociation of the Fc variant<br>
antibody to FcRn at pH6.0.   '<br>
[46] Figure 17.   Binding assays of Fc variants of the present invention to FcRn.   Shown are<br>
direct binding assays measured by AlphaScreen™ at pH 6.0 (a and b) and pH 7.0 (c).<br>
[47] Figure 18.   Binding assays of Fc variants of the present invention to FcRn.   Shown are the<br>
surface plasmon resonance units created upon binding of the variant Fc to surface-bound FcRn.<br>
DETAILED DESCRIPTION OF THE INVENTION<br>
[48] The present invention discloses the generation of novel variants of Fc domains, including those found in antibodies, Fc fusions, and immuno-adhesions, which have an increased binding to the FcRn receptor. As noted herein, binding to RcRn results in longer serum retention in.vivo. [49] In order to increase the retention of the Fc proteins in vivo, the increase in binding affinity must be at around pH 6 without a concomitant increase in affinity at around pH 7.4. Although still under examination, Fc regions are believed to have a longer half-lives in vivo, because binding to FcRn at pH 6 in an endosome sequesters the Fc (Ghetie and Ward, 1997 Immunol Today. 18(12): 592-598, entirely incorporated by reference). The endosomal compartment then recycles the Fc to the cell surface. Once the compartment opens to the extracellular space, the higher pH,.  7,4, induces the release of Fc back into the blood. Dall' Acqua et al. showed that Fc mutants<br>
 <br>
with increased FcRn binding at pH 6 and pH 7.4 actually had reduced serum concentrations and<br>
the same half life as wild-type Fc (Dall  Acqua et al. 2002, J. Immunol 169:5171-5180, entirely<br>
incorporated by reference). The increased affinity of Fc for FcRn at pH 7.4 is thought to forbid the<br>
release of the Fc back into the blood. Therefore, the Fc mutations that will increase Fc's half-life<br>
in vivo will ideally increase FcRn binding at the lower pH while still allowing release of Fc at<br>
higher pH.   The amino acid histidine changes its charge state in the pH range of 6.0 to 7.4.<br>
Therefore, it is not surprising to find His residues at important positions in the Fc/FcRn complex<br>
(Figures.)<br>
[50]An additional aspect of the invention is the increase in FcRn binding over wild type<br>
specifically at lower pH, about pH 6.0, to facilitate Fc/FcRn binding in the endosome.   Also<br>
disclosed are Fc variants with altered FcRn binding and altered binding to another class of Fc<br>
receptors, the FcyR's, as differential binding to FcyRs, particularly increased binding to FcyRlllb<br>
and decreased binding to FcyRllb has been shown to result in increased efficacy.<br>
[51]Definitions<br>
[52]In order that the application may be more completely understood, several definitions are set<br>
forth below. Such definitions are meant to encompass grammatical equivalents.<br>
[53]By "ADCC" or "antibody dependent cell-mediated cytotoxicity" as used herein is meant the<br>
cell-mediated reaction wherein nonspecific cytotoxic cells that express FcyRs recognize bound<br>
antibody on a target cell and subsequently cause lysis of the target cell.<br>
[54]By "ADCP" or antibody dependent cell-mediated phagocytosis as used herein is meant the<br>
cell-mediated reaction wherein nonspecific cytotoxic cells that express FcyRs recognize bound .<br>
antibody on a target cell and subsequently cause phagocytosis of the target cell.<br>
[55] By "amino acid modification" herein is meant an amino acid substitution, insertion, and/or<br>
deletion in a polypeptide sequence.<br>
[56]By "amino acid substitution" or "substitution" herein is meant the replacement of an amino<br>
acid at a particular position in a parent polypeptide sequence with another amino acid.   For<br>
example, the substitution E272Y refers to a variant polypeptide, in this' case an Fc variant, in<br>
which the glutamic acid at position 272 is replaced with tyrosine.<br>
[57] By "amino acid insertion" or "insertion" as used herein is meant the addition of an amino acid<br>
at a particular position In a parent polypeptide "sequence.    For example,  -233E' or A233E<br>
designates an .insertion of glutamic acid after position 233 and before position 234. Additionally, -<br>
233ADE or A233ADE designates an insertion of AlaAspGlu after position 233 and before position<br>
234.<br>
[58]By "amino acid deletion" or "deletion" as used herein is meant the removal of an amino acid<br>
at a particular position in a parent polypeptide sequence.    For example,  E233- or E233#<br>
designates a deletion of glutamic acid at position 233.    Additionally, EDA233- or EDA233#<br>
designates a deletion of the sequence GluAspAla that begins at position 233.<br>
 <br>
[59]By "variant protein" or "protein variant", or "variant" as used herein is meant a protein that differs from that of a parent protein by virtue of at least one amino acid modification. Protein variant may refer to the protein itself, a composition comprising the protein, or the amino sequence that encodes it. Preferably, the protein variant has at least one amino acid modification compared to the parent protein, e.g. from about one to about ten amino acid modifications, and preferably from about one to about five amino acid modifications compared to the parent The protein variant sequence herein will preferably possess at least about 80% homology with a parent protein sequence, and most preferably at least about 90% homology, more preferably at least about 95% homology. Variant protein can refer to the variant protein itself, compositions comprising the protein variant, or the amino acid sequence that encodes it. Accordingly, by "antibody variant" or "variant antibody" as used herein is meant an antibody that differs from a parent antibody by virtue of at least one amino acid modification, "IgG variant" or "variant IQG" as used herein is meant an antibody that differs from a parent IgG by virtue of at least one amino acid modification, and "immunoglobulin variant" or "variant immunoglobulin" as used herein is meant an immunoglobulin sequence that differs from that of a parent immunoglobulin sequence by virtue of at least one amino acid modification Variants may comprise non-natural amino acids. Examples include US6586207; WO 98/48032; WO 03/073238; US2004-0214988A1; WO 05/35727A2; WO 05/74524A2; J. W. Chin et al., (2002), Journal of the American Chemical Society 124:9026-9027; J. W. Chin, &amp; P. G. Schultz, (2002), ChemBioChem 11:1135-1137; J. W. Chin, et al, (2002), PICAS United States of America 99:11020-1 (024; and, L Wang, &amp; P. G. Schultz, (2002), Chem. 1-10,' all entirely incorporated by reference.<br>
[60] As used herein, "protein" herein is meant at least two covalently attached amino acids, which includes proteins, polypeptides, oligopeptides and peptides. The peptidyl group may comprise naturally occurring amino acids and peptide bonds, or synthetic peptidomimetic structures, i.e. "analogs", such as peptoids (see Simon et al., PNAS USA 89(20):9367 (1992), entirely incorporated by reference). The amino acids may either be naturally occurring or non-naturally occurring; as will be appreciated by those in the art. 'For example, homo-phenylalamine, citrulline, and noreleucine are considered amino acids for the purposes of the invention, and both D- and L- (R or S) configured amino acids may be utilized. The variants of the present invention may comprise modifications.that include the use of unnatural amino acids incorporated.using, for example, the technologies developed by Schultz and colleagues, including but not limited to methods described by Cropp &amp; Shultz, 2004, Trends Genet. 20(12):625-30, Anderson et al., 2004, Proc Natl Acad Sci USA 101(2):7566-71,-Zhang et al., 2003, 303(5656):371-3, and Chin et al., 2003, Science 301(5635):964-7, all entirely incorporated by reference. In addition, polypeptides may include synthetic derealization of one or more side chains or termini, glycosylation, PEGylation, circular permutation, cyclization, linkers to other molecules, fusion to proteins or protein domains, and addition of peptide tags or labels.<br>
 <br>
[61]By "residue" as used herein is meant a position in a protein and its associated amino acid<br>
identity. For example, Asp'aragine 297 (also referred to as Asn297, also referred to as N297) is a<br>
residue in the human antibody IgGl<br>
[B2]By "Fab" or "Fab region" as used herein is meant the polypeptides that comprises the VH,<br>
CH1, VL, and CL immunoglobulin domains.   Fab may refer to this region in isolation, or this<br>
region in the context of a full length antibody or antibody fragment.<br>
[63] By "IgG subclass modification" as used herein is meant an amino acid modification that<br>
converts one amino acid of one IgG isotype to the corresponding amino acid in a different,<br>
aligned IgG isotype.  For example, because IgGl comprises a tyrosine and igG2 a phenylalanine<br>
at EU position 296, a F296Y substitution in igG2 is considered an IgG subclass modification.<br>
[84]By "non-naturally occurring modification" as used herein is meant an amino acid modification<br>
that is not isotypic.  For example, because none of the IgGs comprise a glutamic acid at position<br>
332, the substitution I332E in lgG1, lgG2, )gG3, or lgG4 is considered a non-naturally occuring<br>
modification.<br>
[65]By "amino acid" and "amino acid identity" as used herein is meant one of the 20 naturally<br>
occurring amino acids or any non-natural analogues that may be present at a specific, defined<br>
position.<br>
[86]By "effector function" as used herein is meant a biochemical event that results from the<br>
interaction of an antibody Fc region with an Fc receptor or iigand.   Effector functions include but<br>
are not limited to ADCC, ADCP, and CDC.<br>
[67]By "effector cell" as used herein is meant a cell of the immune system that expresses one or<br>
more Fc receptors and mediates one or more effector functions.  Effector cells include but are not<br>
limited to monocytes, macrophages, neutrophils, dendritic cells, eosinophils, mast cells, platelets,<br>
B cells, large granular lymphocytes, Langerhans' cells, natural killer (NK) cells, and y5T cells, and<br>
may be from any organism including but not limited to humans, mice, rats, rabbits, and monkeys.<br>
[68]By "IgG Fc Iigand" as used herein is meant a molecule, preferably a polypeptide, from any<br>
organism that binds to the Fc region of an IgG antibody to'form an Fc / Fc Iigand complex.   Fc<br>
ligands include but are. not limited to FcyRs, FcyRs, FcyRs, FcRn, Clq, C3, mannan binding<br>
lectin, mannose receptor, staphylococcal protein A, streptococcal protein G, and viral FcyR.   Fc<br>
ligands also include Fc receptor homologs (FcRH), which are a family of Fc receptors that are<br>
homologous to the FcyRs (Davis et a/., 2002, Immunological Reviews 190:123-136, entirely<br>
incorporated' by reference).    Fc ligands may include undiscovered molecules that bind Fc.<br>
Particular IgG Fc ligands are FcRn and Fc gamma receptors.   By "Fc.Iigand" as used herein is<br>
meant a molecule, preferably a polypeptide, from any organism that binds to the Fc region of an<br>
antibody to form an Fc / Fc Iigand complex.<br>
[69]By "Fc gamma receptor" or "FcyR" as used herein is meant any member of the family of<br>
proteins that bind the IgG antibody Fc region and is encoded by an FcyR gene.   In humans this<br>
 <br>
family includes but is not limited to FcyRI (CD64), including isoforms FcyRia, FcyRib, and FcyRIc; FcyRII (CD32), including isoforms FcyRlla (including allotypes H131 and R131), FcyRllb (including FcyRllb-1 and FcyRMb-2), and FcyRllc; and FcyRIK (CD16), incfuding isoforms FcyRlfla (including allotypes V158 and F158) and FcyRII lb (including allotypes FcyRINb-NA1 and FcyRII I b-NA2) (Jefferis et a/., 2002, Immunol Lett 82:57-65, entirely incorporated by reference), as well as any undiscovered human FcyRs or FcyR isoforms or allotypes. An FcyR may be from any organism, including but not limited to humans, mice, rats, rabbits, and monkeys. Mouse FcyRs include but are not limited to FcyRI (CD64), FcyRII (CD32), FcyPW (CD16), and FcyRIII-2 (CD16-2), as well as any undiscovered mouse FcyRs or FcyR isoforms or allotypes. [70jBy "FcRn" or "neonatal Fc Receptor" as used herein is meant a protein that binds the IgG antibody Fc region and is encoded at least in part by an FcRn gene. The FcRn may be from any organism, including but not limited to humans, mice, rats, rabbits, and monkeys. As is known in the art, the functional FcRn protein comprises two polypeptides, often referred to as the heavy chain and light chain. The light chain is beta-2-microglobu!in and the heavy chain is encoded by the FcRn gene. Unless other wise noted herein, FcRn or an FcRn protein refers to the complex of FcRn heavy chain with beta-2-microglobulin. Sequences of particular interest of FcRn are shown in the Figures, particularly the human species.<br>
[71J6y "parent polypeptide" as used herein is meant an unmodified polypeptide that is subsequently modified to generate a variant. The parent polypeptide may be a naturally occurring polypeptide, or a variant or engineered version of a naturally occurring polypeptide. Parent polypeptide may refer to the polypeptide itself, compositions that comprise the parent polypeptide, or the amino acid sequence that encodes it. Accordingly, by "parent immunoglobulin" as used herein is meant an unmodified immunoglobulin polypeptide that is modified to generate -a variant, and by "parent antibody" as used herein is meant an unmodified antibody that is modified to generate a variant antibody. It should be noted that "parent antibody" includes known commercial, recombinantly produced antibodies as outlined below. [72]By "position" as used herein is meant a location in the sequence of a protein. Positions may be numbered sequentially, or according to an established format, for example the EU index as in Kabat. For example, position 297 is a position in the human antibody IgG1. [73] By "target antigen" as used herein is meant the molecule that is bound specifically by the variable region of a given antibody. A target antigen may be s protein, carbohydrate, lipid, or other chemical compound.<br>
[74]By "target cell" as used herein is meant a cell that expresses a target antigen. [75]By "variable region" as used herein is meant the region of an immunoglobulin that comprises one or more Ig domains substantially encoded by any of the VK, VA, and/or VH genes that make up the kappa, lambda, and heavy chain immunoglobulin genetic loci respectively.<br>
 <br>
[76]By "wild type or WT" herein is meant an amino acid sequence or a nucleotide sequence that is found in nature, including allelic variations. A WT protein has an amino acid sequence or a nucleotide sequence that has not been intentionally modified.<br>
[77].The present invention is directed to antibodies that exhibit moduluated binding to FcRn (modulation including increased as well as decreased binding) por example, in some instances, increased binding results in cellular recycling of the antibody and hence increased half-life, for example for therapeutic antibodies. Alternatively, decrease FcRn binding is desirable, for example for diagnostic antibodies or therapeutic antibodies that contain radiolabels. In addition, antibodies exhibiting increased binding to FcRn and altered binding to other Fc receptors, eg. FcyRs, find use in the present invention. Accordingly, the present invention provides antibodies. Antibodies<br>
[78]The present application is directed to antibodies that incIude amino acid modifications that modulate binding to FcRn. Of particular interest are antibodies that minimally comprise an Fc region, or functional variant thereof, that display increased binding affinity to  FcRn at lowered pH, and do not exhibit substantially altered binding at higher pH.<br>
[79]Traditional antibody structural units typically comprise a tetramer   Each tetramer is typically composed of two identical pairs of polypeptide chains, each, pair having one -light (typically havinq a molecular weight of about 25 kDa) and one "heavy" chain (typically having a molecular weight of about 50-70 kDa). Human light chains are classified as kappa and |ambda light t chains Heavy chains are classified as mu, delta, gamma, alpha, or epsilon, and define the antibody's isotype as IgM, IgD, IgG, IgA, and IgE, respectively.   IgG has several  subclasses, including, but not limited to lgG1, lgG2,lgG3, and lgG4. IgM has subclassesi including, but not limited to, lgM1 . and lgM2.   Thus, "isotype" as used herein is meant any of tr,e subclasses of immunoglobulins defined by the chemical and antigenic characteristics of their cc,nstartt regions. The known human immunoglobulin isotypes are lgG1, lgG2, lgG3, lgG4, lgA1, lgA2_ |gM1, |gM2, IgD, and IgE. [80]The amino-terminal portion of each chain includes a variab|e region of about 100 to 110 or more amino acids primarily responsible for antigen recognition. ,in the variable region, three loops are gathered for each of the V domains of the heavy chain and light chain to form an antigen. binding  sjte.     Each of the  loops  is referred to as  a complementarity-determining region (hereinafter referred to as a "CDR"), in which the variation in the amino acid sequence is most significant.<br>
[81JThe carboxy-terminal portion of each chain defines a constant region primarily responsible for effector function. Kabat et al. collected numerous primary sequences of the  variable regions of heavy chains an.d light chains. Based on the degree of conservation of the sequences, they classified individual primary sequences into the CDR and the framework and made a list thereof (see SEQUENCES OF IMMUNOLOGICAL INTEREST, 5th edition NlH publication, No. 91-3242, E.A. Kabat et al., entirely incorporated by reference).<br>
 <br>
[82]ln the IgG subclass of immunoglobulins, there are several immunoglobulin domains in the heavy chain. By "immunoglobulin (lg) domain'herein is meant a region of an immunoglobulin having a distinct tertiary structure. Of interest in the present invention are the heavy chain domains, including, the constant heavy (CH) domains and the hinge domains. In the context of IgG antibodies, the IgG isotypes each have three CH regions. Accordingly, "CH" domains in the context of IgG are as follows: "CH1" refers to positions 118-220 according to the EU index as in Kabat. "CH2" refers to positions 237-340 according to the EU index as in Kabat, and "CH3" refers to positions 341-447 according to the EU index as in Kabat.<br>
[83]Another type of Ig domain of the heavy chain is the hinge region. By "hinge" or "hinge region" or "antibody  hinge region" or "immunoglobulin  hinge region"  herein  is  meant the flexible polypeptide comprising the amino acids between the first and second constant domains of an antibody.   Structurally, the IgG CH1 domain ends at EU position 220, and the IgG CH2 domain begins at residue EU position 237.  Thus for IgG the antibody hinge is herein defined to include positions 221 (D221 in lgG1) to 236 {G236 in !gG1), wherein the numbering is according to the EU index as in Kabat.   In some embodiments, for example in the context of an Fc region, the lower hinge is included, with the "lower hinge" generally referring to positions 226 or 230. [84]Of particular interest in the present invention are the Fc regions.   By "Fc" or "Fc region", as used herein is meant the polypeptide comprising the constant region of an antibody excluding the first constant region immunoglobulin domain and in some cases, part of the hinge. Thus Fc refers to the last two constant region immunoglobulin domains of IgA, IgD, and IgG, and the last three constant region immunoglobulin domains of IgE and IgM, and the flexible hinge N-terminal to these domains. For IgA and Iglvl, Fc may include the J chain. For IgG, as illustrated in Figure 1, Fc comprises immunoglobulin domains Cgamma2 and Cgamma3 (Cg2 and Cg3) and the lower hinge region between Cgammal (Cg1) and Cgamma2 (Cg2). Although the boundaries of the Fc region may vary, the human IgG heavy chain Fc region is usually defined to include residues C226 or P230 to its carboxyl-terminus, wherein the numbering is according to the EU index as in Kabat. Fc may refer to this region in isolation, or this region in the context of an Fc polypeptide, as described below. By "Fc polypeptide" as used herein is meant a polypeptide that comprises all or part of an Fc region. Fc polypeptides include antibodies, Fc fusions, isolated Fcs, and Fc fragments.<br>
[85]ln some embodiments, the antibodies are full length.  By "full length antibody"herein is meant  the structure that constitutes the natural biological form of an antibody, including variable and constant regions, including one or more modifications as outlined herein.<br>
[86]Alternatively, the antibodies can be a variety of structures, including, but not limited to, antibody fragments, monoclonal antibodies, bispecific antibodies, minibodies, domain antibodies, synthetic antibodies (sometimes referred to herein as "antibody mimetics"), chimeric antibodies,<br>
 <br>
humanized antibodies, antibody fusions (sometimes referred to as "antibody conjugates"), and fragments of each, respectively. Antibody Fragments<br>
[87]in one embodiment, the antibody is an antibody fragment. Of particular interest are antibodies that comprise Fc regions, Fc fusions, and the constant region of the heavy chain (CH1-hinge-CH2-CH3), again also including constant heavy region fusions. [88]Specific antibody fragments include, but are not limited to, (t) the Fab fragment consisting of VL, VH, CL and CH1 domains, (ii) the Fd fragment consisting of the VH and CH1 domains, (iii) the Fv fragment consisting of the VL and VH domains of a single antibody; (iv) the dAb fragment (Ward et aL, 1989, Nature 341:544-546, entirely incorporated by reference) which consists of a single variable, (v) isolated CDR regions, (vi) F(ab')2 fragments, a bivalent fragment comprising two (inked Fab fragments (vii) single chain Fv molecules (scFv), wherein a VH domain and a VL domain are linked by a peptide linker which allows the two domains to associate to form an antigen binding site (Bird et al., 1988, Science 242:423-426, Huston et al., 1988, Proc. Natl. Acad. Sci. U.S.A. 85:5879-5833, entirely incorporated by reference), (viii) bispecific single chain Fv (WO 03/11161, hereby incorporated by reference) and (ix) "diabodies" or "triabodies", multivalent or multispecific fragments constructed by gene fusion (Tomlinson et. al., 2000, Methods Enzymol. 326:461-479; WO94/13804; Holligeret al., 1993, Proc. Natl. Acad. Sci. U.S.A. 90:6444-6448, all entirely incorporated by reference). The antibody fragments may be modified. For example, the molecules may be stabilized by the incorporation of disulphide bridges linking the VH and VL domains (Reiter et al., 1996, Nature Biotech. 14:1239-1245, entirely incorporated by reference).<br>
Chimeric and Humanized Antibodies<br>
[89] In some embodiments, the scaffold components can be a mixture from different species. As such, if the antibody is an antibody, such antibody may be a chimeric antibody and/or a humanized antibody. In general, both "chimeric antibodies" and "humanized antibodies" refer to antibodies that combine regions from more than one species. 'For example, "chimeric antibodies" traditionally comprise variable region(s) from a mouse (or rat, in some cases) and the constant region(s) from a human. "Humanized antibodies" generally refer to non-human antibodies that have' had the variable-domain framework regions swapped for sequences found in human antibodies. Generally, in a humanized antibody, the entire antibody, except the CDRs, is encoded by a polynucleotide of human origin or is identical to such an antibody except within its CDRs. The CDRs, some' or all of which are encoded by nucleic acids originating in a non-human organism, are grafted into the beta-sheet framework of a human antibody variable region to create an antibody, the specificity of which is determined by the engrafted CDRs. The creation of such antibodies is described in, e.g., WO 92/11018, Jones, 1986, Nature 321:522-525, Verhoeyen   et  al.,   1988,   Science  239:1534-1536,   all   entirely   incorporated   by   reference.<br>
 <br>
"Backmutation" of selected acceptor framework residues to the corresponding donor residues is often required to regain affinity that is lost in the initial grafted construct (US 5530101; US-5585089; US 5693761; US 5693762; US 6180370; US 5859205; US 5821337; US 6054297; US 6407213, all entirely incorporated by reference).   The humanized antibody optimally also will comprise at least a portion of an immunoglobulin constant region, typically that of a human immunoglobulin, and thus will typically comprise a human Fc region.   Humanized antibodies can also be generated using mice with a genetically engineered immune system. Roque et al., 2004, BfotechnoL Prog. 20:639-654, entirely incorporated by reference.   A variety of techniques and methods for humanizing and reshaping non-human antibodies are well known in the art (See Tsurushita &amp; Vasquez, 2004, Humanization of Monocional Antibodies, Molecular Biology of 6 Cells, 533-545, Elsevier Science (USA), and references cited therein, all entirely incorporated by reference).   Humanization methods include but are not limited to methods described in Jones ef al., 1986, Nature 321:522-525; Riechmann et al;.,1988; Nature 332:323-329; Verhoeyen et a/., 1988, Science, 239:1534-1536; Queen et al., 1989, Proc Natl Acad Sci, USA 86:10029-33; He et al., 1998, J. Immunol. 160: 1029-1035; Carter et al., 1992, Proc Natl Acad Sci USA 89:4285-9, Presta et al.. 1997, Cancer Res.57(20}:4593-9; Gorman et al., 1991, Proc. Natl. Acad. Sci. USA 88:4181-4185;  O'Connor et al.,   1998,   Protein  Eng  11:321-8,  all  entirely  incorporated  by reference.     Humanization or other methods of reducing the immunogenicity of nonhuman antibody variable regions may include resurfacing methods, as described for example in Roguska et al., 1994, Proc. Natl. Acad. Sci. USA 91:969-973, entirely incorporated by reference.   In one embodiment, the parent antibody has been affinity matured, as is known in the art.   Structure-based methods may be employed for humanization and affinity maturation, for example as described in USSN 11/004,590.  Selection based methods may be employed to humanize and/or affinity mature antibody variable regions, including but not limited to methods described in Wu et al., 1999, J. Mol. Biol. 294:151-162; Baca et al., 1997, J. Biol. Chem. 272(16):10678-10684; Rosoketal., 1996, J, Biol. Chem. 271(37): 22611-22618; Raderetal., 1998, Proc. Natl. Acad. Sci. USA 95: 8910-8915; Krauss et al., 2003, Protein Engineering 16(10):753-759, a!! entirely incorporated by reference. Other humanization methods may involve the grafting of only parts of the CDRs, including but not limited to methods described in USSN 09/810,510; Tan et al., 2002, J. Immunol, 169:1119-1125; De Pascalis et al., 2002, J. Immunol. 169:3076-3084, all entirely incorporated by reference. Bispecific Antibodies<br>
[90]ln one embodiment, the antibodies of the invention multispecific antibody, and notably a bispecific antibody, also sometimes referred to as "diabodtes". These are antibodies that bind to two (or more) different antigens. Diabodies can be manufactured in a variety of ways known in the art (Holliger and Winter, 1993, Current Opinion Biotechnol. 4:446-449, entirely incorporated by reference), e.g., prepared chemically or from hybrid hybridomas.<br>
 <br>
Minibodies<br>
[91]in one embodiment, the antibody is a minibody.   Minibodies are minimized antibody-like proteins comprising a scFv joined to a CH3 domain. Hu et a!., 1996, Cancer Res. 58:3055-3061, entirely incorporated by reference.   In some cases, the scFv can be joined to the Fc region, and may include some or all of the hinge region. Human Antibodies<br>
[92]ln one embodiment, the antibody is a fully human antibody with-at least one modification as outlined herein.    "Fully human antibody" or "comptete human antibody" refers to a human antibody having the gene sequence of an antibody derived from a human chromosome with the modifications outlined herein. [93] Antibody Fusions<br>
[94] In one embodiment, the antibodies of the Invention are antibody fusion proteins (sometimes referred to herein as an "antibody conjugate"). One type of antibody fusions comprises Fc fusions, which join the Fc region with a conjugate partner. By "Fc fusion" as used herein is meant a protein wherein one or more polypeptides is operably linked to an Fc region. Fc fusion is herein meant to be synonymous with the terms "immunoadhesin", "Ig fusion", "Ig chimera", and "receptor globulin" {sometimes with dashes) as used in the prior art (Chamow et a!., 1996, Trends Biotechnol 14:52-60; Ashkenazi et al., 1997, Curr Opin Immunol 9:195-200, both entirely incorporated by reference). An Fc fusion combines the Fc region of an immunoglobulin with a fusion partner, which in general can be any protein" or small molecule. Virtually any protein or small molecule may be linked to Fc to generate an Fc fusion. Protein fusion partners may include, but are not limited to, the variable region of any antibody, the target-binding region of a receptor, an adhesion molecule, a ligand, an enzyme, a cytokine, a chemokine, or some other protein or protein domain. Small molecule fusion partners may include any therapeutic agent that directs the Fc fusion to a therapeutic target. Such targets may be any molecule, preferably an extracellular receptor, which is implicated in disease. Thus, the IgG variants can be linked to one or more fusion partners. In one alternate embodiment, the IgG variant is conjugated or operably linked to another therapeutic compound. The therapeutic compound may be a cytotoxic agent, a chemotherapeutic agent, a toxin, a radioisotope, a cytokine, or other therapeutically active agent-The IgG may be linked to one of a variety of nonproteinaceous polymers, e.g., polyethylene glycol, polypropylene glycol, polyoxyalkylenes, or copolymers of polyethylene glycol and polypropylene glycol.<br>
[95]ln addition to Fc fusions, antibody fusions include the fusion of the constant region of the heavy chain with one or more fusion partners (again including the variable region of any antibody), while other antibody fusions are substantially or completely full length antibodies with fusion partners. In one embodiment, a role of the fusion partner is to mediate target binding, and   . thus it is functionally analogous to the variable regions of an antibody (and in fact can be).<br>
 <br>
Virtually any protein or small molecule may be linked to Fc to generate an Fc fusion (or antibody fusion). Protein fusion partners may include, but are not limited to, the target-binding region of a receptor, an adhesion molecule, a ligand, an enzyme, a cytokine, a chemokine, or some other protein or protein domain. Small molecule fusion partners may Include any therapeutic agent that directs the Fc fusion to a therapeutic target. Such targets may be any molecule, preferably an extracellular receptor, which is implicated in disease.<br>
[96]The conjugate partner can be proteinaceous or non-proteinaceous; the latter generally being generated using functional groups on the antibody and on the conjugate partner. For example linkers are known in the art; for example, homo-or hetero-bifunctional linkers as are well known {see, 1994 Pierce Chemical Company catalog, technical section on cross-linkers, pages 155-200, incorporated herein by reference).<br>
[97]Suitable conjugates include, but are not limited to, labels as described below, drugs and cytotoxic agents including, but not limited to, cytotoxic drugs (e.g., chemotherapeutic agents) or toxins or active fragments of such toxins. Suitable toxins and their corresponding fragments include diptheria A chain, exotoxin A chain, ricin A chain, abrin A chain, curcin, crotin, phenomycin, enomycin and the like. Cytotoxic agents also include radiochemicals made by conjugating radioisotopes to antibodies, or binding of a radionuclide to a chelating agent that has been covalentiy attached to the antibody. Additional embodiments utilize calicheamicin, auristatins, geldanamycin, maytansine, and duocarmycins and analogs; for the latter, see U.S. 2003/0050331A1, hereby incorporated by reference in its entirety. Covalent modifications of Antibodies<br>
[98]Cova!ent modifications of antibodies are included within the scope of this invention, and are generally, but not always, done post-translationally. For example, several types of covalent modifications of the antibody are introduced into the molecule by reacting specific amino acid residues of the antibody with an organic derivatizing agent that is capable of reacting with selected side chains or the N- or C-termina! residues.<br>
[99]Cysteinyl residues most commonly are reacted with α-haloacetates (and corresponding<br>
amines), such as chloroacetic acid or chloroacetamide, to give carboxymethyl or<br>
carboxyamidomethyl derivatives. Cysteinyl residues may also be derivatized by reaction with<br>
bromotrifluoroacetone, α-bromo-β-(5-imfdozoyl)propionic acid, chloroacetyl phosphate, N-<br>
alkylmaleimides, 3-nitro-2-pyridyl disulfide, methyl 2-pyridyl disulfide, p-chloromercuribenzoate, 2-<br>
chloromercuri-4-nitrophenol, or chloro-7-nitrobenzo-2-oxa-1,3-diazole and the like.<br>
[100]	Histidyl residues are derivatized by reaction with diethyipyrocarbonate at pH 5.5-7.0<br>
because this agent is relatively specific for the histidyl side chain.   Para-bromophenacyl bromide<br>
also is useful; the reaction is preferably performed in 0.1 M sodium cacodylate at pH 6.0.<br>
[101]	Lysinyl and amino terminal residues are reacted with succinic or other carboxylic acid<br>
anhydrides.   Derivatization with these agents has the effect of reversing the charge of the lysinyl<br>
 <br>
residues. Other suitable reagents for den'vatizing alpba-amino-coniaJning residues include imidoesters such as methyl picoliriimidate; pyridoxal phosphate; pyridoxal; chloroborohydride; trinitrobenzenesulfonic acid; O-methylisourea; 2,4-pentanedione; and transarninase-cataiyzed reaction with glyoxylate.<br>
[102]	Arginyl residues are modified by reaction with one or several conventional reagents,<br>
among     them     phenylglyoxal,     2,3-butanedione,     1,2-cyclohexanedione,     and     ninhydrin.<br>
Derivatization of arginine residues requires that the reaction be performed in alkaline conditions<br>
because of the high pKa of the guanidine functional group.   Furthermore, these reagents may<br>
react with the groups of lysine as well as the arginine epsilon-amino group.<br>
[103]	The specific modification of tyrosyl residues may be made, with particular interest in<br>
introducing spectral labels into tyrosyl residues by reaction with aromatic diazonium compounds or tetranitromethane. Most commonly, N-acetylimidizole and tetranitromethane are used to form O-acetyl tyrosyl species and 3-nitro derivatives, respectively. Tyrosyl residues are iodinated using 1251 or 1311 to prepare labeled proteins for use in radioimmunoassay, the chloramine T method described above being suitable.<br>
[104]	Carboxyl side groups (aspartyl or glutamyl) are selectively modified by reaction with<br>
carbodiimides (R — N=C=N-R'), where R and R' are optionally different alkyl groups, such as 1-cyclohexyl-3-(2-morpholinyl-4-ethyi) carbodiimide or 1-ethyl-3-(4-azonia-4,4-dimethylpentyl) carbodiimide. Furthermore, aspartyl and glutamyl residues are converted to asparaginyl and glutaminyi residues by reaction with ammonium ions. •<br>
[105]	Derivatization with bifunctional agents is useful for crosslinking antibodies to a water-<br>
insoluble support matrix or surface for use in a variety of methods, in addition to methods described beiow. Commonly used crosslinking agents include, e.g., 1,1-bis(diazoacetyl)-2-phenylethane, glutaraldehyde, N-hydroxysuccihimide esters, for example, esters with 4-azidosalicylic acid, homobifunctional imidoesters, including disuccinimidyl esters such as 3,3'-dithiobis (succinimidylpropionate), and bifunctional maleimides such as bis-N-maleimido-1,8-octane. Den'vatizing agents such as methyl-3-[(p-azidophenyl)dithio]propioimidate yield photoactivatable intermediates that are capable of forming crosslinks in the presence of light. Alternatively, reactive water-insoluble matrices such as cyanogen bromide-activated carbohydrates and the reactive substrates described in U.S. Pat. Nos. 3,969,287; 3,691,016; 4,195,128; 4,247,642; 4,229,537; and 4,330,440, all entirely incorporated by reference, are employed for protein immobilization.<br>
[106]	Glutaminyi and asparaginyl residues are frequently deamidated to the corresponding<br>
glutamyl and aspartyl residues, respectively. Alternatively, these residues are deamidated under mildly acidic conditions. Either form of these residues falls within the scope of this invention. Other modifications include hydroxylation of proline and iysine, phosphorylation of hydroxyl groups of seryl or threonyl residues, methylation of the a-amino groups of lysine, arginine, and<br>
 <br>
histidine side chains (T. E. Creighton, Proteins: Structure and Molecular Properties, W. H. Freeman &amp; Co., San Francisco, pp. 79-86 [1983], entirely incorporated by reference), acetylation of the N-terminal amine, and amidation of any C-terminal carboxyl group. Glycosylation<br>
[107]	Another type of covalent modification is glycosylation.   In another embodiment, the<br>
IgG variants disclosed herein can be modified to include one or more engineered glycoforms. By<br>
"engineered glycoform" as used herein is meant a carbohydrate composition that is covalently<br>
attached to an IgG, wherein said carbohydrate composition differs chemically from that of a<br>
parent IgG. Engineered glycoforms may be useful for a variety of purposes, including but not<br>
limited to enhancing or reducing effector function. Engineered glycoforms may be generated by a<br>
variety of methods known in the art (Umana et al., 1999, Nat Biotechnol 17:176-180; Davies et<br>
al., 2001, Biotechnol Bioeng 74:288-294; Shields et al., 2002, J Biol Chem 277:26733-26740;<br>
Shinkawa et al„ 2003, J Biol Chem 278:3466-3473; US 6,602,684; USSN 10/277,370; USSN<br>
10/113,929; PCT WO 00/61739A1; PCT WO 01/29246A1; PCT WO 02/31140A1; PCT WO<br>
02/30954A1, alt entirely incorporated by reference; (Potelligent™ technology [Biowa, Inc.,<br>
Princeton, NJ]; GlycoMAb® glycosylation engineering technology [Glycart Biotechnology AG,<br>
Zurich, Switzerland]). Many of these techniques are based on controlling the level of fucosylated<br>
and/or bisecting oligosaccharides that are covalently attached to the Fc region, for example by<br>
expressing an IgG in various organisms or ceil lines, engineered or otherwise (for example Lec-<br>
13 CHO cells or rat hybridoma YB2/0 cells), by regulating enzymes involved in the glycosylation<br>
pathway (for example FUT8 [a1,6-fucosyitranserase] and/or β1-4- N-<br>
acetylglucosaminyltransferase III [GnTIII]), or by modifying .carbohydrate(s) after the IgG has<br>
been expressed. Engineered glycoform typically refers to the different carbohydrate or<br>
oligosaccharide; thus an IgG variant, for example an antibody or Fc fusion, can include an<br>
engineered glycoform. Alternatively, engineered glycoform may refer to the IgG variant that<br>
comprises the different carbohydrate or oligosaccharide. As is known in the art, glycosylation<br>
patterns can depend on both the sequence of the protein-(e.g., the presence or absence of<br>
particular glycosylation amino acid residues, discussed below), or the host cell or organism in<br>
which the protein is produced. Particular expression systems are discussed below.<br>
[108]	Glycosylation of polypeptides is typically either N-linked or O-linked.   N-linked refers<br>
to the attachment of the carbohydrate moiety to the side chain of an asparagine residue. The tri-peptide sequences asparagine-X-serine and asparagine-X-threonine, where X is any amino acid except proline, are the recognition sequences for enzymatic attachment of the carbohydrate moiety to the asparagine side chain. Thus, the presence of either of these tri-peptide sequences in a polypeptide creates a potential glycosylation site. O-linked glycosylation refers to the attachment of one of the sugars N-acetylgalactosamine, galactose, or xylose, to a hydroxyamino.<br>
 <br>
acid, most commonly serine or threonine, although 5-hydroxyproline or 5-hydroxylysine may also be used.<br>
[109]	Addition of giycosylation sites to the antibody is conveniently accomplished by<br>
altering the amino acid sequence such that it contains one or more of the above-described tri-peptide sequences (for N-linked giycosylation sites). The alteration may also be made by the addition of, or substitution by, one or more serine or threonine residues to the starting sequence (for O-linked giycosylation sites). For ease, the antibody amino acid sequence is preferably altered through changes at the DNA level, particularly by mutating the DNA encoding the target polypeptide at preselected bases such that codons are generated that will translate into the desired amino acids.<br>
[110].	Another means of increasing the number of carbohydrate moieties on the antibody is<br>
by chemical or enzymatic coupling of glycosides to the protein. These procedures are advantageous In that they do not require production of the protein in a host cell that has giycosylation capabilities for N- and O-linked giycosylation. Depending on the coupling mode used, the sugar(s) may be attached to (a) arginine and histidine, (b) free carboxyl groups, (c) free sulfhydryl groups such as those of cysteine, (d) free hydroxyl groups such as those of serine, threonine, or hydroxyproline, (e) aromatic residues such as those of phenylalanine, tyrosine, or tryptophan, or (f) the amide group of glutamine. These methods are described in WO 87/05330 and in Aplin and Wriston, 1981, CRC Crit. Rev. Biochem., pp. 259-306, both entirely incorporated by reference..<br>
[111]	Removal  of carbohydrate  moieties  present  on   the  starting  antibody  may   be<br>
accomplished chemically or enzymatically. Chemical deglycosylation requires exposure of the protein to the compound trifluoromethanesulfonic acid, or an equivalent compound. This treatment results in the cleavage of most or all sugars except the linking sugar (N-acetylglucosamine or N-acetylgalactosamine), while leaving the polypeptide intact. Chemical deglycosylation is described by Hakimuddin-et al., 1987, Arch. Biochem. Biophys. 259:52 and by Edge et al., 1981, Anal. Biochem. 118:131, both entirely incorporated by reference. Enzymatic cleavage of carbohydrate moieties on polypeptides can be achieved by the use of a variety of endo- and exo-glycosidases as described by Thotakura et al., 1987, Meth. Enzymol. 138:350, entirely incorporated by reference. Giycosylation at potential giycosylation sites, may be prevented by the use of the compound tunicamycin as described by Duskin et al., 1982, J. Biol. Chem. 257:3105, entirely incorporated by reference. Tunicamycin blocks the formation of protein-N-giycoside linkages.<br>
[112]	Another type of covalent modification of the antibody comprises linking the antibody<br>
to various nonproteinaceous polymers, including, but not limited to, various polyols such as polyethylene glycol, polypropylene glycol or polyoxyalkylenes, in the manner set forth in, for . example, 2005-2006 PEG Catalog from Nektar Therapeutics (available at the Nektar website) US<br>
 <br>
Patents 4,640,835;   4,496,689;   4,301,144;   4,670,417;  4,791,192   or 4,179,337,  all  entirely incorporated by reference.   In addition, as is known in the art, amino acid substitutions may be made in various positions within the antibody to facilitate the addition of polymers such as-PEG. See for example, U.S. Publication No. 2005/0114037A1, entirely incorporated by reference. Labeled Antibodies<br>
[113]	In some embodiments, the covalent modification of the antibodies of the invention<br>
comprises the addition of one or more labels. In some cases, these are considered antibody fusions. The term "labelling group" means any detectable label. In some embodiments, the labelling group is coupled to the antibody via spacer arms of various lengths to reduce potential steric hindrance. Various methods for labelling proteins are known in the art and may be used in performing the present invention.<br>
[114]	In general, labels fall into a variety of classes, depending on the assay In which they<br>
are to be detected: a) isotopic labels, which may be radioactive or heavy isotopes; b) magnetic<br>
labels {e.g., magnetic particles); c) redox active moieties; d) optical dyes; enzymatic groups (e.g.<br>
horseradish peroxidase, β-galactosidase, luciferase, alkaline phosphatase); e) biotinylated<br>
groups; and f) predetermined polypeptide epitopes recognized by a secondary reporter (e.g.,<br>
leucine zipper pair sequences, binding sites for secondary antibodies, metal binding domains,<br>
epitope tags, etc.). In some embodiments, the labelling group is coupled to the antibody via<br>
spacer arms of various lengths to reduce potential steric hindrance. Various methods for labelling<br>
proteins are known in the art and may be used in performing the present invention.<br>
[115]	Specific labels include optical dyes, including, but not limited to, chromophores,<br>
phosphors and fluorophores, with the latter being specific in many instances. Fluorophores can be either "small molecule" fluores, or proteinaceous fluores.<br>
[116]	By "fluorescent label" is meant any molecule that may be detected via its inherent<br>
fluorescent properties. Suitable fluorescent labels include, but are not limited to, fluorescein, rhodamine, tetramethylrhodamine, eosin, erythrosin, coumarin, methyl-coumarins, pyrene, Malacite green, stiiberie, Lucifer Yellow, Cascade BlueJ, Texas Red, IAEDANS, EDANS, BODIPY FL, LC Red 640, Cy 5, Cy 5.5, LC Red 705, Oregon green, the Alexa-Fluor dyes (Alexa Fluor 350, Alexa Fluor 430, Alexa Fluor 488, Alexa Fluor 546, Alexa Fluor 568, Alexa Fluor 594, Alexa Fluor 633, Alexa Fluor 660, Alexa Fluor 680), Cascade Blue, Cascade Yellow and R-phycoerythrin (PE) (Molecular Probes, Eugene, OR), FITC, Rhodamine, and Texas Red (Pierce, Rockfo'rd, IL), Cy5, Cy5.5, Cy7 (Amersham Life Science, Pittsburgh, PA). Suitable optical dyes, including fluorophores, are described in Molecular Probes Handbook by Richard P. Haugland, entirely incorporated by reference.<br>
[117]	Suitable proteinaceous fluorescent labels also include, but are not limited to, green<br>
fluorescent protein, including a Renilla, Ptilosarcus, or Aequorea species of GFP (Chalfie et al., 1994, Science 263:802-805), EGFP (Clontech Laboratories, Inc., Genbank Accession Number<br>
 <br>
U55762), blue fluorescent protein (BFP, Quantum Biotechnologies, Inc. 1801-de Maisonneuve Blvd. West, 8th Floor, Montreal, Quebec, Canada H3H 1J9; Stauber, 1998, Biotechniques 24:462-471; Heim et al., 1996, Curr. Biol. 6:178-182), enhanced yellow fluorescent protein (EYFP, Clontech Laboratories, Inc.), luciferase (Ichiki et al., 1993, J. Immunol. 150:5408-5417), 3 galactosidase (Nolan et al., 1988, Proc. Natl. Acad. Sci. U.S.A. 85:2603-2607) and Renilla (W092/15673, WO95/07463, WO98/14605, W098/26277, WO99/49019, U.S. Patent Nos. 5292658, 5418155, 5683888, 5741668, 5777079, 5804387, 5874304, 5876995, 5925558). All of the above-cited references in this paragraph are expressly incorporated herein by reference. IgG Variants<br>
[118]	In one embodiment, the invention provides variant IgG proteins.  At a minimum, IgG<br>
variants comprise an antibody fragment comprising the CH2-CH3 region of the heavy chain.   In<br>
addition, suitable IgG variants comprise Fc domains (e.g. including the lower hinge region), as<br>
well as IgG variants comprising the constant region of the heavy chain (CH1-hinge-CH2-CH3)<br>
also being useful in the present invention, all of which can be fused to fusion partners.<br>
[119]	An IgG variant includes one or more amino acid modifications relative to a parent IgG<br>
polypeptide, in some cases relative to the wild type IgG. The IgG variant can have one or more<br>
optimized properties. An IgG variant differs in amino acid sequence from its parent JgG by virtue<br>
of at least one amino acid modification. Thus IgG variants have at least one amino acid<br>
modification compared to the parent. Alternatively, the IgG variants may have more than one<br>
amino acid modification as compared to the parent, for example from about one to fifty amino<br>
acid modifications,' preferably' from about one to ten amino acid modifications, and most<br>
preferably from about one to about five amino acid modifications compared to the parent<br>
[120]	Thus the sequences of the IgG variants and those of the parent Fc polypeptide are<br>
substantially homologous. For example, the variant IgG variant sequences herein will possess about 80% homology with the parent IgG variant sequence, preferably at least about 90% homology, and most preferably at least about 95% homology.    Modifications may be made genetically using molecular biology, or may be made enzymatically or chemically. Target Antigens for Antibodies<br>
[121].	Virtually any antigen may be targeted by the IgG variants, including but not limited to<br>
proteins, subunits, domains, motifs, and/or epitopes belonging to the following list of target antigens, which includes both soluble factors such as cytokines and membrane-bound factors, including transmembrane receptors: 17-IA, 4-1BB, 4Dc, 6-keto-PGF1a, 8-iso-PGF2a, 8-oxo-dG, A1 Adenosine Receptor, A33, ACE, ACE-2, Activin, Activin A, Activin AB, Activin B, Activin C, Activin RIA, Activin RIA.ALK-2, Activin RIB ALK-4, Activin RIIA, Activin RUB, ADAM, ADAM10, ADAM12, ADAM15, ADAM17/TACE, ADAM8, ADAM9, ADAMTS, ADAMTS4, ADAMTS5, Addressins, aFGF, ALCAM, ALK, ALK-1, ALK-7, alpha-1-antitrypsin, alpha-V/beta-1 antagonist, ANG, Ang,-APAF-1, APE, APJ, APP, APRIL, AR, ARC, ART, Artemin, anti-Id, ASPARTIC, Atrial<br>
 <br>
natriuretic factor, av/b3 integrin, Axl, b2M, B7-1, B7-2, B7-H, B-lymphocyfe Stimulator (BlyS), BACE, BACE-1, Bad, BAFF, BAFF-R, Bag-1, BAK, Bax, BCA-1, BCAM, Bel, BCMA, BDNF, b-ECGF, bFGF, BIO, Bik, BJM, BLC, BL-CAM, BLK, BMP, BMP-2 8MP-2a, BMP-3 Osteogenin, BMP-4 BMP-2b, BMP-5, BMP-6 Vgr-1, BMP-7 (OP-1), BMP-8 (BMP-Sa, OP-2), BMPR, BMPR-fA (ALK-3), BMPR-IB (ALK-6), BRK-2, RPK-1, BMPR-JJ (BRK-3), BMPs, b-NGF, BOK, Bombesin, Bone-derived neurotrophic factor, BPDE, BPDE-DNA, BTC, complement factor 3 (C3), C3a, C4, C5, C5a, C10. CA125, CAD-8, Calcitonin, cAMP, carcinoembryonic antigen (CEA), carcinoma-associated antigen, Cathepsin A, Cathepsin B, Cathepsin C/DPPi, Cathepsin D, Cathepsin E, Cathepsin H, Cathepsin L, Cathepsin 0, Cathepsin S, Cathepsin V, Cathepsin X/Z/P, CBL, CCI, CCK2, CCL, CCL1, CCL11, CCL12, CCL13, CCL14, CCL15, CCL16, CCL17, CCL18, CCL19, CCL2, CCL20, CCL21, CCL22, CCL23, CCL24, CCL25, CCL26, CCL27, CCL28, CCL3, CCL4, CCL5, CCL6,  CCL7,  CCL8, CCL9/10, CCR,  CCR1,  CCR10, CCR1Q,  CCR2,  CCR3,  CCR4, CCR5, CCR6, CCR7, CCR8, CCR9, CD1, CD2, CD3, CD3E, CD4, CD5, CD6, CD7, CD8, CD10, CD11a, CD11b, CD11c, CD13, CD14, CD15, CD16, CD18, CD19, CD20, CD21, CD22, CD23, CD25, CD27L, CD28, CD29, CD30, CD30L, CD32, CD33 (p67 proteins), CD34, CD38, CD40, CD40L, CD44, CD45, CD46, CD49a, CD52, CD54, CD55, CD56, CD61, CD64, CD66e, CD74, CD80 (B7-1), CD89, CD95, CD123, CD137, CD133, CD140a, CD146, CD147, CD148, CD152, CD164, CEACAM5, CFTR, cGMP, CINC, Clostridium botulinum toxin, Clostridium perfringens toxin, CKb8-t, CLC, CMV, CMV UL, CNTF, CNTN-1, COX, C-Ret, CRG-2, CT-1, CTACK, CTGF, CTLA-4, CX3CL1, CX3CR1, CXCL, CXCL1, CXCL2, CXCL3, CXCL4, CXCL5, CXCL6, CXCL7, CXCLS, CXCL9.  CXCL10VCXCL11, CXCL12, CXCL13, CXCL14, CXCL15, CXCL16, CXCR, CXCR1, CXCR2, CXCR3, CXCR4, CXCR5, CXCR6, cytokeratin tumor-associated antigen, DAN, DCC, DcR3, DC-SIGN, Decay accelerating factor, des(1-3)-IGF-I (brain IGF-1), Dhh, digoxin, DNAM-1, Dnase, Dpp, DPP1V/CD26, Dtk, ECAD, EDA, EDA-A1, EDA-A2, EDAR, EGF, EGFR (ErbB-1), EMA, EMMPRIN, ENA, endothelin receptor, Enkephalinase, eNOS, Eot, eotaxinl, EpCAM, Ephrin B2/ EphB4, EPO, ERCC, E-se!ectin, ET-1, Factor lla, Factor VII, Factor VIHe, Factor !X, fibroblast activation protein (FAP), Fas, FcR1. FEN-1, Ferritin, FGF, FGF-19, FGF-2, ' FGF3,  FG.F-8,  FGFR,  FGFR-3,  Fibrin,  FL,  FLIP,  Flt-3,  Flt-4,  Follicle stimulating hormone, Fractaikine, FZ.D1, FZD2, FZD3, FZD4, FZD5, FZD6, FZD7, FZD8, FZD9, FZD10, G250, Gas 6, GCP-2, GCSF. GD2, GD3, GDF, GDF-1, GDF-3 (Vgr-2), GDF-5 (BMP-14, CDMP-1), GDF-6 (BMP-13, CDMP-2), GDF-7 (BMP-12, CDMP-3), GDF-8 (Myostatin), GDF-9, GDF-15 (MIC-1), GDNF, GDNF, GFAP, GFRa-1, GFR-aipha1, GFR-alpha2, GFR-alpha3, GITR, Glucagon, Glut 4, glycoprotein ifb/IIla (GP llb/llla), GM-CSF, gp130, gp72, GRO, Growth hormone releasing factor, Hapten (NP-cap or NiP-cap), HB-EGF, HCC, HCMV gB envelope glycoprotein, HCMV) gH envelope glycoprotein, HCMV UL, Hemopoietic growth factor (HGF), Hep B gp120, heparanase, Her2,  Her2/neu  (ErbB-2),   Her3 (ErbB-3),   Her4 (ErbB-4),  herpes simplex virus (HSV) gB glycoprotein, HSV gD glycoprotein, HGFA, High molecular weight melanoma-associated antigen<br>
 <br>
(HMW-MAA). HIV gp120, HIV 1MB gp 120 V3 loop, HLA, HLA-DR, HM1.24, HMFG PEM, HRG, Hrk, human cardiac myosin, human cytomegalovirus (HCMV), human growth hormone (HGH), HVEM, I-309, iAP, ICAM, ICAM-1, ICAM-3, ICE, ICOS, IFNg, Ig, IgA receptor, IgE, IGF, IGF binding proteins, IGF-1R, IGFBP, IGF-!, IGF-II, IL, IL-1, IL-1R, IL-2, IL-2R, IL-4, IL-4R, IL-5, IL-5R, IL-6, IL-6R, IL-8, IL-9, IL-10, IL-12, IL-13, IL-15, IL-18, IL-18R, IL-23, interferon (INF)-alpha, INF-beta, INF-gamma, Inhtbin, iNOS, Insulin A-chain, Insulin B-chain, Insulin-like growth factor 1, integrin alpha2, integrin alpha3, integrin alpha4, integrin alpha4/beta1, integrin alpha4/beta7, integrin alphao (alphaV), integrin alpha5/beta1, integrin alpha5/beta3, integrin alphas, integrin betal, integrin beta2, interferon gamma, IP-10, l-TAC, JE, Kallikrein 2, Kallikrein 5, Kallikrein 6, , Kallikrein 11, Kallikrein 12, Kallikrein 14, Kallikrein 15, Kallikrein L1, Kallikrein L2, Kallikrein L3, Kallikrein L4; KC, KDR, Keratinocyte Growth Factor (KGF), laminin 5, LAMP, LAP, LAP (TGF-1), Latent TGF-1, Latent TGF-1 bp1, LBP, LDGF, LECT2, Lefty, Lewis-Y antigen, Lewis-Y related antigen, LFA-1,  LFA-3,  Lfo, LIF, LIGHT, lipoproteins, LIX, LKN, Lptn, L-Selectin, LT-a, LT-b, LTB4,  LTBP-1,   Lung surfactant,  Luteinizing  hormone,  Lymphotoxin  Beta Receptor,  Mac-1, MAdCAM,    MAG,    MAP2,    MARC,   MCAM,    MCAM,    MCK-2,    MCP,    M-CSF,    MDC,    Mer, METALLOPROTEASES, MGDF receptor, MGMT, MHC (HLA-DR), MIF, MIG, MIP, MIP-1-alpha, MK, MMAC1, MMP, MMP-1, MMP-10, MMP-11, MMP-12, MMP-13, MMP-14, MMP-15, MMP-2, MMP-24, MMP-3,  MMP-7, MMP-8^ MMP-9, MPIF, Mpo, MSK, MSP, mucin (Mud), MUC18, Muellerian-inhibitin substance, Mug, MuSK, NAIP, NAP, NCAD, N-Cadherin, NCA 90, NCAM, NCAM, Neprilysin, Neurotrophin-3,-4, or -6, Neurturin, Neuronal growth factor (INGFJ, NGFR, NGF-beta. nNOS. NO, NOS, Npn, NRG-3, NT, NTN, OB, OGG1, OPG, OPN, OSM, OX40L, OX40R,   p150,   p95,   PADPr,. Parathyroid  hormone,   PARC,   PARP,   PBR,   PBSF,   PCAD,  P-Cadherin. PCNA.  PDGF, PDGF, PDK-1, PECAM, PEM, PF4, PGE, PGF, PGI2, PGJ2, PIN, PLA2, placental alkaline phosphatase (PLAP), PIGF, PLP, PP14, Prpinsulin, Prorelaxin, Protein C, PS, PSA, PSCA, prostate specific membrane antigen (PSMA), PTEN, PTHrp, Ptk, PTN, R51, RANK, RANKL, RANTES, RANTES, Relaxin A-chain, Relaxin B-chain, renin, respiratory syncytial virus (RSV) F. RSV Fgp, Ret, Rheumatoid factors, RLIP76, RPA2, RSK, S100, SCF/KL, SDF-1, SERINE, Serum albumin, sFRP-3, Shh, SIGIRR, SK-1, SLAM, SLPI, SMAC, SMDF, SMOH, SOD, SPARC, Stat, STEAP, STEAP-II, TACE, TACI, TAG-72 (tumor-associated glycoprotein-72), TARC, TCA-3, T-cell receptors (e.g., T-ceil receptor alpha/beta), TdT, TECK, TEM1, TEM5, TEM7, TEM8, TERT, testicular PLAP-like alkaline phosphatase, TfR, TGF, TGF-alpha, TGF-beta, TGF-beta Pan-'Specific, TGF-beta RI.(ALK-5), TGF-beta Rll, TGF-beta Rllb, TGF-beta Rill, TGF-betal,   TGF-beta2,   TGF-beta3,   TGF-beta4,   TGF-beta5,   Thrombin,   Thymus   Ck-1,   Thyroid stimulating hormone, Tie, TIMP, TIQ, Tissue Factor, TMEFF2, Tmpo, TMPRSS2, TNF, TNF-alpha, TNF-alpha beta, TNF-beta2, TNFc, TNF-RI, TNF-RIl, TNFRSF10A (TRAIL R1 Apo-2, DR4), TNFRSF10B (TRAIL R2 DR5, KILLER, TRICK-2A, TRICK-B), TNFRSF10C (TRAIL R3 . DcR1,  LIT, TRID),  TNFRSF10D (TRAIL R4 DcR2, TRUNDD), TNFRSF11A (RANK ODF R,<br>
 <br>
TRANCE R), TNFRSF11B (OPG OCIF, TR1), TNFRSF12 (TWEAK R FN14), TNFRSF13B (TACI), TNFRSF13C (BAFF R), TNFRSF14 (HVEM ATAR, HveA, LIGHT R, TR2), TNFRSF16 (NGFR p75NTR), TNFRSF17 (3CMA), TNFRSF18 (GITR AITR), TNFRSF19 (TROY TAJ, TRADE). TNFRSF19L (RELT), TNFRSF1A (TNF Rl CD120a, p55-60), TNFRSF1B (TNF Rll CD120b. p75-80), TNFRSF26 (TNFRH3), TNFRSF3 (LTbR TNF Rill, TNFC R), TNFRSF4 (OX40 ACT35, TXGP1 R), TNFRSF5 {CD40 p50), TNFRSF6 {Fas Apo-1, APT1, CD95), TNFRSF6B (DcR3 M68, TR6), TNFRSF7 (CD27), TNFRSF8 (CD30), TNFRSF9 (4-1BB CD137, I LA), TNFRSF21 (DR6), TNFRSF22 (DcTRAIL R2 TNFRH2), TNFRST23 (DcTRAIL R1 TNFRH1), TIMFRSF25 (DR3 Apo-3, LARD, TR-3, TRAMP, WSL-1), TNFSF10 (TRAIL Apo-2 Ligand, TL2), TNFSF11 (TRANCE/RANK Ligand ODF, OPG Ligand), TNFSF12 (TWEAK Apo-3 Ligand, DR3 Ligand): TNFSF13 (APRIL TALL2), TNFSF13B (BAFF BLYS, TALL1, THANK, TNFSF20), TNFSF14 (LIGHT HVEM Ligand, LTg), TNFSF15 (TL1AA/EGI), TNFSF18 (GITR Ligand AITR Ligand, TL6), TNFSF1A (TNF-a Conectm, D\F, TNFSF2), TNFSF1B (TNF-b LTa, TNFSF1), TNFSF3 (LTb TNFC, p33), TNFSF4 (OX40 Ligand gp34, TXGP1), TNFSF5 (CD40 Ligand CD154, gp39, HIGM1, IMD3, TRAP), TNFSF6 (Fas Ligand Apo-1 Ligand, APT1 Ligand), TNFSF7 (CD27 Ligand CD70), TNFSF8 (CD30 Ligand CD153), TNFSF9 (4-1BB Ligand CD137 Ligand), TP-1, t-PA, Tpo, TRAIL, TRAIL R, TRAIL-R1, TRAIL-R2, TRANCE, transferring receptor, TRF, Trk. TROP-2, TSG, TSLP, tumor-associated antigen CA 125, tumor-associated antigen expressing Lewis Y related carbohydrate, TWEAK, TXB2, Ung, uPAR, uPAR-1, Urokinase, VCAM, VCAM-1, VECAD, VE-Cadherin, VE-cadherin-2, VEGR-1 (flt-1), VEGF, VEGFR, VEGFR-3 (flt-4), VEGI, VIM; Viral antigens, VLA, VLA-1, VLA-4, VNR integrin, von Willebrands factor. WIF-1, WNT1, WNT2, WNT2B/13, WNT3, WNT3A, WNT4, WNT5A,. WNT5B, WNT6, WNT7A, WNT73, WNT8A, WNT8B, WNT9A, WNT9A, WNT9B, WNT10A, WNT10B, WNT11, WNT16, XCL1, XCL2, XCR1, XCR1, XEDAR, XIAP, XPD, and receptors for hormones and growth factors.<br>
[122]	One skilled in the art will appreciate that the aforementioned list of targets refers not<br>
only tc specific proteins and biomolecules, but the biochemical pathway or pathways that comprise them. For example, reference to CTLA-4 as a target antigen implies that the ligands and receptors that make up the T cell co-stimulatory pathway, including CTLA-4, B7-1, B7-2, CD28, and any other undiscovered ligands or receptors that bind these proteins, are also targets. Thus target as used herein refers not only to a specific biomolecule, but the set of proteins that interact with said target and the members of the biochemical pathway to which said target belongs. One skilled in the art will further appreciate that any of the aforementioned target antigens, the ligands or receptors that bind them, or other members of their corresponding biochemical pathway, may be operably linked to the Fc variants of the present invention in order to generate an Fc fusion. Thus for example, an Fc fusion that targets EGFR could be constructed. by operabiy linking an Fc variant to EGF, TGF-b, or any other ligand, discovered or undiscovered,<br>
 <br>
that binds EGFR. Accordingly, an Fc variant of the present invention could be operabiy linked to EGFR in order to generate an Fc fusion that binds EGF, TGF-b, or any other ligand, discovered or undiscovered, that binds EGFR. Thus virtually any polypeptide, whether a ligand, receptor, or some other protein or protein domain, including but not limited to the aforementioned targets and the proteins that compose their corresponding biochemical pathways, may be operabiy linked to the Fc variants of the present invention to develop an Fc fusion.<br>
[123]	The choice of suitable antigen depends on the desired application.   For anti-cancer<br>
treatment it is desirable to have a target whose expression is restricted to the cancerous cells. Some targets that have proven especially amenable to antibody therapy are those with signaling functions. Other therapeutic antibodies exert their effects by blocking signaling of the receptor by inhibiting the binding between a receptor and its cognate ligand. Another mechanism of action of therapeutic antibodies is to cause receptor down regulation. Other antibodies do not work by signaling through their target antigen. In some cases, antibodies directed against infectious disease agents are used.<br>
[124]	In one embodiment, the Fc variants of the present invention are incorporated into an<br>
antibody against a cytokine. Alternatively, the Fc variants are fused or conjugated to a cytokine. By "cytokine" as used herein is meant a generic term for proteins released by one cell population that act on another cell as intercellular mediators. For example, as described in Penichet et al., 2001, J Immunol Methods 248:91-101, expressly incorporated by reference, cytokines may be fused to antibody to provide an array of desirable properties. Examples of such cytokines are lymphokines, monokines, and traditional polypeptide hormones. Included among the cytokines are growth hormone such as human growth hormone, N-methionyl human growth hormone, and bovine growth hormone; parathyroid hormone; thyroxine; insulin; proinsulin; relaxin; prorelaxin; glycoprotein hormones such as follicle stimulating hormone (FSH), thyroid stimulating hormone (TSH). and luteinizing hormone (LH); hepatic growth factor; fibroblast growth factor; prolactin; placenta! lactogen; tumor necrosis factor-alpha and -beta; mullerian-inhibiting substance; mouse<br>
<br>
gonadotropin-associated peptide; inhibin; activin; vascular endothelial growth factor; integrin; thrombopoietin (TPO); nerve growth factors such as NGF-beta; platelet-growth factor; transforming growth factors (TGFs) such as TGF-alpha and TGF-beta; insulin-like growth factor-l and -II; erythropoietin (EPO); osteoinductive factors; interferons such as interferon-alpha, beta, and -gamma; colony stimulating factors (CSFs) such as macrophage-CSF (M-CSF); granulocyte-macrophage-CSF (GM-CSF); and granulocyte-CSF (G-CSF); interleukiris (ILs) such as-IL-1, IL-lalpha, IL-2, IL-3, IL-4,.IL-5, IL-6, IL-7, IL-8, 1L-9, IL-10, IL-11, IL-12; IL-15, a tumor necrosis factor such as TNF-alpha or TNF-beta; C5a; and other polypeptide factors including LIF and kit ligand (KL). As used herein, the term cytokine includes proteins from natural sources or from recombinant ceil culture, and biologically active equivalents of the native sequence cytokines.<br>
 <br>
[125]	Cytokines and soluble targets, such as TNF superfamily members, are preferred<br>
targets for use with the variants of the present invention. For example, anti-VEGF, anti-CTLA-4, and anti-TNF antibodies, or fragments thereof, are particularly good antibodies for the use of Fc variants that increase the FcRn binding.- Therapeutics against these targets are frequently involved in the treatment of autoimmune diseases and require multiple injections over long time periods. Therefore, longer serum half-lives and less frequent treatments, brought about from the variants of the present invention, are particularly preferred.<br>
[126]	A number of antibodies and Fc fusions that are approved for use, in clinical trials, or<br>
in development may benefit'from the Fc variants of the present invention. These antibodies and Fc fusions are herein referred to as "clinical products and candidates". Thus in a preferred embodiment, the Fc polypeptides of the present invention may find use in a range of clinical products and candidates. For example, a number of antibodies that target CD20 may benefit from the Fc polypeptides of the present invention. For example the Fc polypeptides of the present invention may find use in an antibody that is substantially similar to rituximab (Rituxan®, IDEC/Genentech/Roche)  (see for example  US 5,736,137),  a chimeric anti-CD20 antibody approved  to   treat  Non-Hodgkirs's  lymphoma;   HuMax-CD20,   an   anti-CD20  currently  being developed by Genmab, an anti-CD20 antibody described in US 5,500,362, AME-133 (Applied Molecular Evaolution),   hA20  (immunomedics,   inc.),   HumaLYM   (Intracel),   and   PRO70769 (PCT/US20C3/Q40426,  entitled "Immunoglobulin Variants and  Uses Thereof). A number of antibodies tna; target members of the family of epidermal growth factor receptors, including EGFR (ErbB-1), Her2/neu (ErbB-2), Her3 (ErbB-3), Her4 (ErbB-4), may benefit from the Fc polypeptides of the present invention. For example the Fc polypeptides of the present invention may find use in an antibody that is substantially similar to trastuzumab (Herceptin®, Genentech) (see for example US 5,677,171), a humanized anti-Her2/neu antibody approved to treat breast cancer; pertuzumab (rhuMab-2C4, Omnitarg™), currently being developed by Genentech; an anti-Her2 antibody described in US 4,753,894; cetuximab (Erbitux®, Imclone) (US 4,943,533; PCT WO 96/40210), a chimeric anti-EGFR antibody in clinical trials for a variety of cancers; ABX-EGF (US 6,235,883), currently being developed by Abgenix-lmmunex-Amgen; HuMax-EGFr (USSN 10/172.317), currently being developed by Genmab; 425, EMD55900, EMD62000, and EMD72000 (Merck   KGaA)   (US  5,558,864;   Murthy  et al.   1987,   Arch   Biochem . Biophys. 252(2):549-60. Rodeck et al., 1987, J Cell Biochem. 35(4):315-20; Kettleborough et al., 1991, Protein Eng. 4(7):773-83); ICR62 (Institute of Cancer Research) (PCT WO 95/20045; Modjtahedi et al., 1993, J. Cell Biophys. 1993, 22(1-3):129-46; Modjtahedi et al., 1993, Br J Cancer. 1993, 67(2):247-53; Modjtahedi et al, 1996, Br J Cancer, 73(2):22S-35; Modjtahedi et al, 2003, Int J Cancer, 105(2):273-80); TheraCIM hR3 (YM Biosciences, Canada and Centro de Immunologia Molecuiar, Cuba (US 5,891,996, US 6, 506,883; Mateo et al, 1997, Immunotechnology, 3(1):71- . 81); mAb-806 (Ludwig Institue'for Cancer Research, Memorial Sloan-Kettering) (Jungbluth et al.<br>
 <br>
2003, Proc Nat! Acad Sci USA. 100(2}:639-44); KSB-102 (KS Biomedix); MR1-1 (I VAX, National Cancer institute) (PCT WO 0162931A2); and SC100 (ScanceiJ) (PCT WO 01/88138). In another preferred embodiment, the Fc polypeptides of the present invention may find use in aiemtuzumab (Campath®, Millenium), a humanized monoclonal antibody currently approved for treatment of B-cell chronic lymphocytic leukemia. The Fc polypeptides of the present invention may find use in a variety of antibodies or Fc fusions that are substantially similar to other clinical products and candidates, including but not limited to muromonab-CD3 (Orthoclone OKT3®), an anti-CD3 antibody developed by Ortho Biotech/Johnson &amp; Johnson, ibritumomab tiuxetan (Zevalin®), an anti-CD20 antibody developed by IDEC/Schering AG, gemtuzumab ozogamicin (Mylotarg®), an anti-CD33 (p67 protein) antibody developed by Celltech/Wyeth, alefacept (Amevive®), an anti-LFA-3 Fc fusion developed by Biogen), abciximab (ReoPro®), developed by Centocorf Lilly, basiliximab   (Simulect®),   developed   by   Novartis,   palivizumab   (Synagis®),   developed   by Medlmmune,   infliximab  (Remicade®),  an  anti-TNFalpha  antibody  developed  by  Centocor, adalimumab (Humira®), an anti-TNFalpha antibody developed by Abbott, Humicade™, an anti-TNFalpha antibody developed by Cefitech, etanercept (Enbrel®), an anti-TNFalpha Fc fusion developed by Immunex/Amgen, ABX-CBL, an anti-CD147 antibody being developed by Abgenix, ABX-IL8, an anti-IL8 antibody being developed by Abgenix, ABX-MA1, an anti-MUC18 antibody being   developed   by  Abgenix,   Pemtumomab  (R1549,   90Y-muHMFG1),   an   anti-MUC1   In development  by  Antisoma,   Therex  (R1550),   an  anti-MUC1   antibody  being   developed  by Antisoma, AngioMab (AS1405), being developed by Antisoma, HuBC-1, being developed by Antisoma, Thioplatin (AS1407) being developed by Antisoma, Antegren® (natalizumab), an anti-alpha-4-beta-1 (VLA-4) and alpha-4-beta-7 antibody being developed by Biogen, VLA-1 mAb, an anti-VLA-1 integrin antibody being developed by Biogen, LTBR mAb, an anti-lymphotoxin beta receptor (LTBR) antibody being developed by Biogen, CAT-152, an anti-TGF-p2 antibody being developed by Cambridge Antibody Technology, J695, an anti-IL-12 antibody being developed by Cambridge Antibody Technology and Abbott, CAT-192, an anti-TGF|31 antibody being developed by Cambridge Antibody Technology and Genzyme, CAT-213, an anti-Eotaxinl antibody being developed by Cambridge Antibody Technology, LymphoStat-B™ an anti-Blys antibody being developed by Cambridge Antibody Technology and Human Genome Sciences Inc., TRAIL-RtmAb, an anti-TRAIL-R1 antibody being developed by Cambridge Antibody Technology and Human   Genome   Sciences,   Inc.,  Avastin™   (bevacizumab,   rhuMAb-VEGF),   an  anti-VEGF antibody being developed by Genentech, an anti-HER receptor family antibody being developed by Genentech, Anti-Tissue Factor (ATF), an anti-Tissue Factor antibody being developed by Genentech,  Xolair™  (Orrializumab),  an  anti-lgE antibody  being  developed  by Genentech, Raptiva™ fEfaiizumab). an anti-CD11a antibody being developed by Genentech and Xoma, MLN-02   Antibody   (formerly   LDP-02),   being   developed   by   Genentech   and   Millenium Pharmaceuticals, HuMax ODA, an anti-CD4 antibody being developed by Genmab, HuMax-JL15,<br>
 <br>
an anti-!L15 antibody being developed by Genmab and Amgen, HuMax-lnflam, being developed<br>
-by Genmab and Medarex, HuMax-Cancer, an anti-Heparanase I antibody being developed by<br>
Genmab and  Medarex  and  Oxford GcoSciences,  HuMax-Lyrnphoma,  being developed  by<br>
Genmab and Amgen, HuMax-TAC, being developed by Genmab, IDEC-131, and anti-CD40L<br>
antibody being developed by IDEC Pharmaceuticals, IDEC-151  (Clenoliximab), an anti-CD4<br>
antibody being developed by IDEC Pharmaceuticals, IDEC-114, an anti-CD80 antibody being<br>
developed  by  IDEC  Pharmaceuticals,  IDEC-152,  an  anti-CD23  being  developed by  IDEC<br>
Pharmaceuticals, anti-macrophage migration factor (MIF) antibodies being developed by IDEC<br>
Pharmaceuticals, BEC2, an anti-idiotypic antibody being developed by Imclone, IMC-1C11, an<br>
anti-KDR antibody being developed by Imclone, DC101, an anti-flk-1 antibody being developed<br>
by   Imclone,    anti-VE    cadherin    antibodies   being   developed    by   imclone,    CEA-Cide™<br>
(labetuzumab),    an   anti-carcinoembryonic   antigen   (CEA)   antibody   being   developed   by<br>
Immunome'dics,  LymphoCide™ (Epratuzumab), an anti-CD22 antibody being developed by<br>
Immunomedics, AFP-Cide, being developed by Immunomedics, MyelomaCide, being developed<br>
by Immunomedics, LkoCide, being developed by Immunomedics, ProstaCide, being developed<br>
by Immunomedics, MDX-010, an anti-CTLA4 antibody being developed by Medarex, MDX-G60,<br>
an anti-CD3Q antibody being developed by Medarex, MDX-070 being developed by Medarex,<br>
MDX-018 being developed by Medarex,  Osidem™  (IDM-1),  and  anti-Her2 antibody being<br>
developed by Medarex and Immuno-Designed Molecules, HuMax™-CD4, an anti-CD4 antibody<br>
being developed by Medarex and Genmab, HuMax-IL15, an anti-IL15 antibody being developed<br>
by Medarex and Genmab, CNTO 148, an anti-TNFa antibody being developed by Medarex and<br>
Centocor/J&amp;J,   CNTO   1275,   an  anti-cytokine  antibody  being  developed  by  Centocor/J&amp;J,<br>
MOR101 and MOR102, anti-intercellular adhesion molecule-1 (ICAM-1) (CD54) antibodies being<br>
developed  by  MorphoSys,  MOR201,  an anti-fibroblast growth  factor receptor 3 (FGFR-3)<br>
antibody being developed by MorphoSys, Nuvion® (visiiizumab), an anti-CD3 antibody being<br>
developed   by   Protein   Design   Labs,   HuZAF™,   an  anti-gamma   interferon   antibody  being<br>
developed by Protein Design Labs, Anti-a5β1 Integrin, being developed by Protein Design Labs,<br>
anti-IL-12, being developed by Protein Design Labs, ING-1, an anti-Ep-CAM antibody being<br>
developed by Xoma, and MLN01, an anti-Beta2 integrin antibody being developed by Xoma, all of<br>
the above-cited references in this paragraph are expressly incorporated herein by reference.<br>
[127]	The   Fc  polypeptides  of the  present  invention   may  be   incorporated  into  the<br>
aforementioned clinical candidates and products, or into antibodies and Fc fusions that are substantially similar to them; The Fc polypeptides of the present invention may be incorporated   . into versions of the aforementioned clinical candidates and products that are humanized, affinity matured, engineered, or modified in some other way.<br>
[128]	In one embodiment, the Fc polypeptides of the present invention are used for the<br>
treatment of autoimmune, inflammatory, or transplant indications. Target antigens and clinical<br>
 <br>
products and candidates that are relevant for such diseases include but are not limited to anti-<br>
a4(37 integrin antibodies such as LDP-02, anti-beta2 integrin antibodies such as LDP-01, anti-<br>
complement (C5) antibodies such as 5G1.1, anti-CD2 antibodies such as BTI-322, MEDi-507,<br>
anti-CD3 antibodies such as OKT3, SMART anti-CD3, anti-CD4 antibodies such as IDEC-151,<br>
MDX-CD4, OKT4A; anti-CD11a antibodies, anti-CD14 antibodies such as IC14, anti-CD18<br>
antibodies, anti-CD23 antibodies such as IDEC 152, anti-CD25 antibodies such as Zenapax, anti-<br>
CD40L antibodies such as 5c8, Antova, IDEC-131, anti-CD64 antibodies such as MDX-33, anti-<br>
CD80 antibodies such as IDEC-114, anti-CD147 antibodies such as ABX-CBL, anti-E-selectin<br>
antibodies such as CDP850, anti-gpllb/llla antibodies such as ReoPro/Abcixima, anti-ICAM-3<br>
antibodies such as ICM3, anti-ICE antibodies such as VX-740, anti-FcR1 antibodies such as<br>
MDX-33, anti-lgE antibodies such as rhuMab-E25, anti-IL-4 antibodies such as SB-240683, anti-<br>
IL-5 antibodies such as SB-240563, SCH55700, anti-lL-8 antibodies such as ABX-IL8, anti-<br>
interferon gamma antibodies, anti-TNF (TNF, TNFa, TNFa, TNF-aipha) antibodies such as<br>
CDP571, CDP870, D2E7, Infliximab, MAK-195F, and anti-VLA-4 antibodies such as Antegren.<br>
[129]	Fc variants of the present invention such as those with increased binding to FcRn<br>
may be utilized in TNF inhibitor molecules to provide enhanced properties. Useful TNF inhibitor molecules include any molecule that inhibits the action of TNF-aipha in a mammal. Suitable examples inciude the Fc fusion Enbrel® (etanercept) and the antibodies Humira® (adalimumab) and Remicade© (infliximab). Monoclonal antibodies (such as Remicade and Humira) engineered using the Fc variants of the present invention to increase FcFn binding, may translate to better efficacy through an increased half-life.<br>
[130] In some embodiments, antibodies against infectious diseases are used. Antibodies against eukaryotic celis include antibodies targeting yeast cells, including but not limited to Saccharomyces cerevisiae, Hansenula polymorphs, Kluyveromyces fragilis and K. iactis, Pichia guiilerimondii and P. pastoris, Schizosaccharomyces pombe, Plasmodium falciparum, and Yarrowia lipolytics.<br>
[131] Antibodies against additional fungal cells are also useful, including target antigens associated with Candida strains including Candida glabrata, Candida albicans, C, krusei, C. lusitaniaear.di C. maltosa, as well as species of Aspergillus, Cryptococcus, Histopiasma, Coccidioides. Blastomyces, and Penicillium, among others<br>
[132] Antibodies directed against target antigens associated with protozoa indude, but are. not limited to, antibodies associated with Trypanosoma , Leishmania species including Leishmania donovanii; , Plasmodium;spp., Pneumocystis carinii, Cryptosporidium parvum, Giardia lamblia, Entamoeba histolytica, and Cyclospora cayetanensis.<br>
[133] Antibodies against prokaryotic antigens are also useful, including antibodies against suitable bacteria such as pathogenic and non-pathogenic prokaryotes including but not limited to Bacillus, including Bacillus anthracis;   Vibrio, e.g. V. cholerae; Escherichia, e.g. Enterotoxigenic<br>
 <br>
£ coli, Shigella, e.g. S. dysenteriae; Salmonella, e.g. S. typhi;. Mycobacterium e.g. M. tuberculosis. M. leprae; Clostridium, e.g. C. botulinum, C. tetani, C. difficile, C.perfringens; Cornyebactenum, e.g. C. diphtheriae; Streptococcus, S. pyogenes, S. pneumoniae; Staphylococcus, e.g. S. aureus; Haemophilus, e.g. H. influenzae; Neisseria, e.g. N. meningitidis, N. gonorrhoeae; Yersinia, e.g. Y. lamblia, Y. pestis, Pseudomonas, e.g. P. aeruginosa, P. putida; Chlamydia, e g. C. trachomatis; Bordetella, e.g. B. pertussis; Treponema, e.g. T. palladium; B. anthracis: Y. pestis, Brucella spp., F. tularensts, B. mallei, B .pseudomallei, B. mallei, B.pseudomallei ; C. botulinum , Salmonella spp., SEB V. cholerae toxin B, E. coli 0157:H7, Listeria spp. Trichosporon beigelii, Rhodotoruta species, Hansenula anomala, Enterobacter sp., Klebsiella sp... Listeria sp., Mycoplasma sp.and the like.<br>
[134] in some aspects, the antibodies are directed against viral infections; these viruses include, but are not limited to, including orthomyxoviruses, (e.g. influenza virus), paramyxoviruses (e.g respiratory syncytial virus, mumps virus, measles virus), adenoviruses, rhinoviruses, coronaviruses. reoviruses, togaviruses (e.g. rubella virus), parvoviruses, poxviruses (e.g. variola virus, vaccinia virus), enteroviruses (e.g. poliovirus, coxsackievirus), hepatitis viruses (including A, B and C), herpesviruses (e.g. Herpes simplex virus, varicella-zoster virus, cytomegalovirus, Epstein-Sarr virus), rotaviruses, Norwalk viruses, hantavirus, arenavirus, rhabdovirus (e.g. rabies virus), retroviruses (including HIV, HTLV-I and -II), papovaviruses (e.g. papillomavirus), polyomaviruses, and picornaviruses, and the like.<br>
Optimized IgG Variant Properties<br>
[135]	The present application also provides IgG variants that are optimized for a variety of<br>
therapeutically relevant properties. An IgG variant that is engineered or predicted to display one or more optimized properties is herein referred to as an "optimized IgG variant". The most preferred properties that may be optimized include but are not limited to enhanced or reduced affinity for a FcRn and increased or decreased in vivo half-life. Suitable embodiments include antibodies that exhibit increased binding affinity to FcRn at lowered pH, such as the pH associated with endosomes, e.g. pH 6.0, while not displaying corresponding increased binding affinity at higher pH, such as 7.4., to allow increased uptake into endosomes but normal release rates. Similarly, these antibodies with modulated FcRn binding may optionally have other desirable properties, such as modulated FcyR binding, such as outlined in U.S.S.N.s U.S.S.N.s 11/174,287, 11/124,640, 10/822,231, 10/672,280, 10/379,392, and the patent application entitled IgG Immunoglobulin variant's with optimized effector function filed on October 21, 2005 having<br>
application no 	.   That is, optimized properties also include' but are not limited to<br>
enhanced or reduced affinity for an FcyR. In one optional embodiment, the IgG variants are optimized to possess enhanced affinity for a human activating FcyR, preferably FcyRllla in addition to the FcRn binding profile. In yet another optional alternate embodiment, the IgG variants are optimized to possess reduced affinity for the human inhibitory receptor FcyRllb. That<br>
 <br>
is, particular embodiments embrace the use of antibodies that show increased binding to FcRn, and increased binding to FcyRllla. Other embodiments utilize use of antibodies that show increased binding to FcRn, and increased binding to FcyRllla. These embodiments are anticipated to provide IgG polypeptides with enhanced therapeutic properties in humans, for example enhanced effector function and greater anti-cancer potency. In an alternate embodiment, the IgG variants are optimized to have increased or reduced affinity for FcRn and increased or decreased affinity for a human FcyR, including but not limited to Fc-fRl, Fc-fRlla, FcyRHb, FcyRlic. FcyRllla. and FcyRlllb including their allelic variations. These embodiments are anticipated to provide IgG polypeptides with enhanced therapeutic properties in humans, lor example increased serum half-life and reduced effector function. In other embodiments, IgG variants provide enhanced affinity for FcRn and enhanced affinity for one or more FcyRs, yet reduced affinity for one or more other FcyRs. For example, an IgG variant may have enhanced binding io FcP.n and FcyRllla, yet reduced binding to FcyRilb. Alternately, an IgG variant may have reduced binding to FcRn and to FcyR's. In another embodiment, an IgG variant may have reduced affinity for FcRn and enhanced affinity for FcyRilb, yet reduced affinity to one or more activating Fcy-;\3. In yet another embodiment, an IgG variant may have increased serum half-life and reduced effector functions.<br>
[136]	Preferred embodiments comprise optimization of binding to a human FcRn and FcyR,<br>
however in alternate embodiments the IgG variants possess enhanced or reduced affinity for FcRn and Fc-R from nonhuman organisms, including but not limited to rodents and non-human primates. IgG variants that are optimized for binding to a nonhuman FcRn may find use in experimentation. For example, mouse models are available for a variety of diseases that enable testing of properties such as efficacy, toxicity, and pharmacokinetics for a given drug candidate. As is known in the art, cancer cells can be grafted or injected into mice to mimic a human cancer, a process referred to as xenografting. Testing of IgG variants that comprise IgG variants that are optimized for FcRn may provide valuable information with regard to the clearance characteristics of the protein, lis mechanism of clearance, and the like. The IgG variants may also be optimized for enhanced functionality and/or solution properties in aglycosylated form. The Fc ligands include but are not limited to FcRn, FcyRs, Clq, and proteins A and G, and may be from any source including but not limited to human, mouse, rat, rabbit, or monkey, preferably human. In an alternately preferred embodiment, the IgG variants are optimized to be more stable and/or more soluble than the aglycosylated form of the parent IgG variant<br>
 <br>
[1373	IgG variants can include modifications that modulate interaction with Fc iigands<br>
other than FcRn and FcyRs, including but not limited to complement proteins, and Fc receptor homologs (FcRHs). FcRHs include but are not limited to FcRHl, FcRH2, FcRH3, FcRH4, FcRHS, and FcRH6 {Davis et ai., 2002, Immunof. Reviews 190:123-135, entirely incorporated by reference).<br>
[138]	Preferably, the Fc ligand specificity of the IgG variant will determine its<br>
therapeutic utility. The utility of a given IgG variant for therapeutic purposes will depend on the<br>
epitope or form of the target antigen and the disease or indication being treated. For most<br>
targets end Indications, enhanced FcRn binding may be preferable as the enhanced FcRn<br>
binding may result in an increase in serum half-life. Longer serum half-lives allow Jess<br>
frequent dosing or lower dosing of the therapeutic. This is particularly preferable when the<br>
therapeutic agant is given in response to an indication that requires repeated administration.<br>
For some targets and indications, decreased FcRn affinity may be preferable. This may be<br>
particularly preferable when a variant Fc with increased clearance or decreased serum half-<br>
life is desired, for example in Fc polypeptides used as imaging agents or radio-therapeutics.<br>
	IgG variants may be used that comprise IgG variants that provide enhanced<br>
affinity for FcRn with enhanced activating FcyRs and/or reduced affinity for inhibitory FcyRs.<br>
For some targets and indications, it may be further beneficial to utilize IgG variants that	<br>
provide differential selectivity for different activating FcyRs; for example, in same cases enhanced binding to FcyRlla and FcyRllla may be desired, but not FcyRI, whereas in other cases, enhanced binding only to FcyRtla maybe preferred. For certain targets and indications, it may be preferable to utiliza IgG variants that alter FcRn binding and enhance both FcyR-mediated and complement-mediated effector functions, whereas for other cases it may be advantageous to utilize IgG variants that enhance FcRn binding, or serum half-life, and either FcyR-medialed or complement-mediated effector functions. For some targets or cancer indications, it may be advantageous to reduce or ablate one or more effector functions, for example by knocking out binding to C1q, one or more FcyR's, FcRn, or one or more other Fc Iigands. For other targets and indications, ft may be preferable to utilize IgG variants that provide enhanced binding to the inhibitory FcyRllb, yet WT level, reduced, or ablated binding to activating FcyRs. This may be particularly useful, for example, when the goal of an IgG variant is to inhibit inflammation or auto-immune disease, or modulate the immune system in some way. Because auto-immune diseases are generally long-lasting and treatment is given in repeated dosing, their treatment with Fc variants with increased half-life from increased FcRn is particularly preferred.<br>
[140]	Modification may be made to improve the IgG stability, solubility, function, or<br>
clinical use. In a preferred embodiment, the IgG variants can include modifications to reduce immunogenidty in humans. In a most preferred embodiment, the immunogeniclty of an IgG variant is reduced Using a method described in USSN 11/004,590, entirely incorporated by<br>
 <br>
reference. In alternate embodiments, the IgG variants are humanized {Clark, 2000, Immunol Today 21:397-402, entirely incorporated by reference).<br>
[141f	the IgG variants can include modifications that reduce immunogenidty.<br>
Modifications to reduce immunogenidty can include modifications that reduce binding of processed peptides derived from the parent sequence to MHC proteins. For example, amino acid modifications would be engineered such that there are no or a minimal number of immune epitopes that are predicted to bind, with high affinity, to any prevalent MHC alleles. Several methods of identifying MHC-binding epitopes in protein sequences are known in the ad and mzy be used to score epitopes in an IgG variant See for example WO 98/52976; WO 02/079232; WO 00/3317; USSN 09/903,378; USSN 10/039,170; USSN 60/222,697;.USSN 10/754,296; FCT WO 01/21823; and PCT WO 02/00165; Mailtos, 1999, Bioinformatics 15; 432-439; Mailios, 2001, Sioinformaiics 17; 942-948; Sturniolo eta!., 1999, Nature Biotech, 17: 555-561; WO 98/59244; WO 02/069232; WO 02/77187; Marshall ei a/., 1995, J. Immunol 154: 5927-5933; and Hammer etai, 1994, J. Exp. Med. 180: 2353-2358, all entirely incorporated by reference. Sequence-based information can be used to determine a binding score for a given peptide- MHC interaction (see far example Maflios, 1999, Bioinformatics 15:432-439; Mailios, 2001, Bioinformatics 17: p942-948; Stumiolo et al, 1999, Nature Biotech. 17: 555-561, all entirely incorporated by reference). Engineering IgG Variants<br>
[142]	Variants of the present invention may be designed by various means. The<br>
variants, as described herein, may be insertions, deletions, substitutions, other modifications,<br>
or combinations of these and other changes. A particularly novel embodiment of the present<br>
invention is the design of insertions and deletions that either Improve or reduce the binding of<br>
an Fc polypeptide to an Fc llgand. As disclosed herein, insertions or deletions may be made<br>
that increase or decrease the affinity of the Fc polypeptide for FcRn. Insertions and deletions<br>
may be designed by rational approaches or by approaches that include the use or random<br>
components, such as random of semi-random library creation or screening. In an alternative<br>
embodiment, substitutions are disclosed that increase or decrease the affinity of the Fc<br>
polypeptide for FcRn.<br>
[143]	insertions and deletions<br>
[144]      •     Variant Fc polypeptides may be created by substituting a variant amino acid in<br>
place of the parent amino acid at a position in the Fc polypeptide. By substituting one or<br>
more amino acids for variant amino acids in the Fc polypeptide, the side chains at those<br>
positions are altered. Most useful substitutions modify the Fc properties by altering the Fc<br>
side chains. The substituted side chains may interact directly or indirectly with an Fc binding<br>
partner that is associated with an Fc function or property. The at least one substitution alters<br>
the covalent structure of one or more side chains of the parent Fc polypeptide.<br>
[145]	Alternatively, variant Fc polypeptides may be created that change the covalent<br>
structure of the backbone of the parent Fc polypeptide. The backbone atoms in proteins are the peptide nitrogen, the alpha carbon, the carbonyi or peptide carbon and the carbonyl<br>
 <br>
oxygen. Changing the covalent structure of the backbone provides additional methods of altering the properties of the Fc polypeptides. The covalent structure of the Fc backbone may be altered by the addition of atoms into the backbone, e.g. by inserting one or more amino acids, or the subtraction of atoms from the backbone, e.g. by deleting one or more amino actds. The covaient structure of the backbone may also be altered by changing individual atoms of the backbone to other atoms (Deechongkit et si,, AM Chem Soc. 2004. 126(51): 16762-71, entirety incorporated by reference). As is known in the art and is illustrated herein, insertions or deletions of amino acids in Fc polypeptides may be done by inserting or deleting the corresponding nucleotides in the DNA encoding the Fc polypeptide. Alternatively, as is known in the art, insertions or deletions of amino acids may be done during synthesis of Fc polypeptides.<br>
[146]	The design of insertions or deletions of amino acids that atter the interaction of<br>
the Fc polypeptide with one or more binding partners (e.g. FcgammaR's, FcRn, C1q) may be<br>
done by considering the structure of the complex of the Fc polypeptide and Its binding<br>
partner, in a less preferred embodiment, the design may be done by considering the<br>
structure of the Fc polypeptide and information about the Fc region involved in binding trie<br>
binding partner. This information may be obtained by mutagenesis experiments, phage<br>
display experiments, homology comparisons, computer modeling or other means.<br>
[147]	Preferred positions in the amino acid sequence for insertions or deletions that<br>
affect the Fc binding interactions, but do not affect the overall structure, stability, expression or use of the Fc polypeptide, are in loops that are involved in the Fc/Fc-binding partner interactions. To alter FcRn binding to the Fc polypeptide, positions 244-257,279-284, 307-317, 383-390, and 428-435 are preferred loop locations for insertions or deletions (numbering from EU index of Kabat et al, Burmeister et a!., 1994, Nature, 372:379-383; Martin et al, 2001, Mol Cell 7:867-877, all entirely incorporated by reference). To alter the Fcgamma receptor binding to the Fc polypeptide, positions 229-239, 266-273, 294-299, and 324-331 are preferred loop locations For insertions or deletions (numbering from EU Index of Kabat et al, PDB code 1E4K.pdb Sondermann et al. Nature. 2000 406:287, all entirely incorporated by reference). Loops are regions of the polypeptide not involved in alpha helical or beta sheet structure. Loops positions are positions that are not In either alpha helical or beta sheet structures (van Holde, Johnson and Ho, Principles of Physical Biochemistry. Prentice Hall, New Jersey 1998, Chapter 1 pp2-67, entirely incorporated by reference). Loop positions are preferred because the backbone atoms are typically more flexible and less likely involved in hydrogen bonds compared to the backbone atoms of alpha helices and beta sheets. Therefore, the lengthening or shortening of a loop due to an insertion or deletion of one or more amino aGids is less ilkeiy to lead to large, disruptive changes to the Fc polypeptide, including stability, expression or other problems.<br>
[148]	insertions and deletions may be used to alter the length of the polypeptide. For<br>
example, in toop regions, altering the loop length results in altered flexibility and conformational entropy of the loop. Insertions in a loop will generally increase the<br>
 <br>
conformational entropy of the loop, which may be defined as Boltzman's constant multiplied by the naturai logarithm of the number of possible conformations (van Holde, Johnson and Ho. Principles of Physical Biochemistry. Prentice Hall, New Jersey 1998, pp78, entirely incorporated by reference), By inserting at least one amino acid into a polypeptide, the total number of conformations available to the polypeptide increases. These additional conformations may be beneficial for forming favorable Fc/Fc-binding partner interactions because the Fc polypeptide may use one of the additional conformations in binding the Fc~ binding protein. !n this case, the insertion may lead to stronger Fc/Fobinding partner interactions, if the additional conformations are not used in the binding Interface, then trie insertion may lead to weaker Fc/Fc-binding partner interactions, because the additional conformations would compete with the binding-competent conformation. Similarly, deletion of a polypeptide segment may also lead to either stronger or Weaker Fc/Fc binding-partner interactions. If deletion of a segment, which reduces the possible number of backbone conformations, removes the binding-competent conformation, then the deletion may lead to weaker Fc/Fc-binding partner Interactions. if the deletion does not remove the binding-competent conformation, then the deletion may lead to stronger Fc/Fc-binding partner interactions because the deletion may remove conformations that compete with the binding-competent conformation.<br>
[149]	Insertions and deletions may be used to alter the positions and orientations of the<br>
amino acids in the Fc polypeptide. Because insertions and deletions cause a change in the<br>
covalent structure of the backbone, they necessarily cause a change in the positions of the<br>
backbone atoms. Figure 7 compares the backbone positions at some loop segments, marked<br>
L1 to L4, in three different backbones. The reference backbone structure contains four loop<br>
segments, whereas the deletion backbone lacks segment L1 and the insertion segment<br>
comprises an additional segment before, ie, N-terminal to, segment L1. Deletions and<br>
insertions cause the largest change In the backbone structure near the site of the insertion or<br>
deletion. By deleting a segment near the N-terminal end of the loop, e.g. segment L1, the<br>
loop shortens and the remaining segments shift their position closer to the loop N-terminus<br>
This has the effect of moving the L2 segment toward the prior location of the L1 segment and<br>
toward the loop N-terminus. This change in position of the L2 segment toward the L1<br>
segment may strengthen the binding of the Fc/Fc-binding partner complex and is preferred<br>
when there is prior information suggesting thai the amino acid or amino acids located in L2<br>
make favorable Interactions with the Fc-binding partner, when located In L1. For example, if<br>
L2 contains alanine and tyrosine and substitution of two L1 amino acids for alanine and<br>
tyrosine previously lead to an Fc variant with increased binding, then deletion of L1 may<br>
create an Fc variant with increased affinity for the Fc-blnding partner.<br>
[150]	Similarly, an insertion of a polypeptide segment into an Fc polypeptide at the N-<br>
terminal side of a loop causes the positions of the hop segments to be shifted toward the C-terminal side of the loop, in Figure 7, an insertion of one or more amino acids before, i.e. N-terminslly to, segment L1 alters the backbone conformation including a shift of the L1<br>
 <br>
segment toward the C-terminal end of the loop. This type of insertion is preferred when the<br>
amino acids' located in segment L1 are known to make favorable interactions when located in<br>
the L2 positions, as the insertion may lead to stronger Fc/Fc-binding partner interactions, if<br>
weaker Fc/Fc-binding partner interactions are desired, then the insertion may be used to shift<br>
unfavorable amino acid into a new position. The inserted, deleted and reference segments<br>
(L1 to L4 in Figure 7) may be one or more than one amino acid in the Fc polypeptide.<br>
[151]	Alternatively, insertions or deletions may be used at the C-terminal end of loops<br>
in a manner analogous to the insertions or deletions at the N-terminal end of loops. Insertions at the loop C-terminus may lead to a movement of the positions N-terminal of the Insertion toward the loop N-terminus. Deletions at the loop C-termtnus may lead to a movement of the positions N-terminal of the deletion toward the loop C-terminus. The choice of using an insertion or deletion at the N-terminal or C-terminal end of the loop is based on the amino acids located in the loop, the desire for increased or decreased Fc/Fc-binding partner affinity, and the positional shift desired.<br>
[152]	Insertions or deletions may be used in any region of an Fc polypeptide, including<br>
the loops, the alpha helical, and the beta sheet regions. Preferred locations for insertions and deletions include loop regions, which are  those that are not alpha helical or beta sheet regions. Loops are preferred because they generally accept alterations in the backbone better than alpha helixes or beta sheets. The particularly preferred locations for insertions or deletions that result in stronger protein/protein interactions are at the N-terminal  or C-terminal edges of a loop. If the loop side chains are involve in the Fc/Fc-binding partner interactions, then insertions or deletion at the edges are less likely to lead to strongly detrimental changes in the binding interactions. Deletions within the exact center of the loop are more likely to remove Important residues in the Fc/Fc-binding partner interface and insertions within the exact center of the loop are more likely to create unfavorable interactions in the Fc/Fc-binding partner interface. The number of residues deleted or inserted may be determined by the size of the backbone change desired with insertions or deletions of 5 or less residues being preferred, insertions or deletions of 10 or less residues being more preferred, and insertions or deletions of 5 or [ess residues being most preferred.<br>
[153]	Ones the position and size of an Fc deletion variant is designed, the entire<br>
polypeptide sequence is completely determined and the polypeptide may be constructed by methods known in the art<br>
[154]	Fc insertion variants, however, have the additional step of designing the<br>
sequence of the at least one amino acid to be inserted. Insertions  of polar residues, including<br>
Ser, Thr, Asn, Gfn, Ala, Giy, His, are preferred at positions expected to be exposed in the FG<br>
polypeptide. The smaller amino acids, including Ser, Thr, and Ala., are particularly preferred<br>
as the small size ls.less likely to sterically interfere with the Fc/Fc-binding partner interactions.<br>
Ser and Thr also have the capability to hydrogen bond with atoms on the Fc-binding partner.<br>
[155]	insertions also have the added flexibility that the inserted polypeptide may be<br>
designed to make favorable Interactions with the Fc-binding partner as would be desire when<br>
 <br>
stronger Fc/Fc-binding partner binding is desired. The length of the backbone insertion may<br>
be determined by modeling the variant backbone with a simple, generic sequence to be<br>
inserted. For example, polyserine, polyglycine or polyalanins insertions of different lengths<br>
may be constructed and modeled. Modeling may be done by a variety of methods, including<br>
homology modeling based on known three-dimensional structures of homologues comprising<br>
the insertion, and by computer modeling Including MODELLER (MA Martt-Renorn et at.<br>
Annu. Rev. Biophys. Biomol. Struct. 29,291-325,2000} and ROSETTA (Kuhlman et 3i.<br>
(2003). Science 302, 1364-8), both entirely incorporated by reference. Typically, various<br>
backbone conformations are initially generated and the final backbone structure may be<br>
determined after the identities of the side chain are established. The side chains may be<br>
designed by PDA® algorithms (US 6,188,965; 6,269,312; 6,403,312; 6,801,861; 6,804,611;<br>
6,792,356,6,950,754, and USSN 09/782,004; 09/927,790; 10/101,499; 10/666,307;<br>
10/666311; 10/218,102, ail entirely incorporated by reference).<br>
[156]	Insertions and deletion may be designed in any polypeptide besides Fc<br>
polypeptides by the methods described herein. For example, insertions or deletions in the<br>
TNF superfamily member, APRIL,may be designed with the aid of its three-dimentional.<br>
structure (POB code 1XU1.pdb, Hymowitz, et al. (2005) J. Biol. Chem. 280:7218, entirely<br>
incorporated by reference). Insertions or deletions may be designed to increase APRIL<br>
binding to Its receptor, TACI." The loop residues preferred as insertion or deletion sites are<br>
residues Ser118-Va124, Asp164-Phe167, Pro192-Ala198, Pro221-Lys226, These loops<br>
interact with TACi in the APRiL/TACI complex and mediate binding.<br>
[157]	Polypeptides incorporating variants<br>
[158]	The IgG variants can be based on human IgG sequences, and thus human igG<br>
sequences are used as the "base" sequences against which other sequences are compared, including but not limited to sequences from other organisms,for example rodent and primate sequences. IgG variants may also comprise sequences from other immunoglobulin classes such as IgA, IgE, IgD, IgM, and the like. It is contemplated that, although the IgG variants are engineered in the context of one parent IgG, the variants may be engineered in or "transferred" to the context of another, second parent IgG. This is done by determining the "equivalent" or "corresponding" residues and substitutions between the first and second IgG, typically based on sequence or structural homology between the sequences of the IgGs. In order to establish homology, the amino acid sequence of a first IgG outlined herein is directly<br>
. compared to the sequence of a second IgG. After aligning the sequences, using one or more of the homology alignment programs known in the art (for example using conserved residues as between species), allowing for necessary insertions and deletions in order to maintain<br>
. alignment (i.e., avoiding the elimination of conserved residues through arbitrary deletion and insertion), the residues equivalent to particular amino acids in the primary sequence of the first IgG variant are defined. Alignment of conserved residues  preferably should conserve 100% of such residues. However, alignment of greater than 75% or as little as 50% of conserved residues is also adequate to define equivalent residues. Equivalent residues may<br>
 <br>
also be defined by determining structural homology between a first snd second IgG that is at<br>
the leve! of tertiary structure for IgGs whose structures have been determined, In this case,<br>
equivalent residues are defined as those for which the atomic coordinates of two or more of<br>
the main chain atoms of a particular amino acid residue of the parent or precursor (N on N,<br>
CA on CA,C on C and O on O) are within 0.13 nrn and preferably 0.1 nm after alignment.<br>
Alignment is achieved after ihe best model has beers oriented and positioned to give the<br>
maximum overlap of atomic coordinates of non-hydrogen protein atoms of the proteins.<br>
Regardless of how equivalent or corresponding residues are determined, and regardless of<br>
the identity of the parent IgG in which the IgGs are made, what Is meant to be conveyed is<br>
that the IgG variants discovered by can be engineered into any second parent IgG that has<br>
significant sequence or structural homology with ihe IgG varianf. Thus for example, li a<br>
variant antibody Is generated wherein the parent antibody is human ]gG1, by using the<br>
methods described above or other methods for determining equivalent residues, the variant<br>
antibody may be engineered in another {gG1 parent antibody that binds a different antigen, a<br>
human lgG2 parent antibody, a human igA parent antibody, a mouse lgG2a or !gG2b parent<br>
antibody, and the like. Again, as described above, the context of the parent IgG variant does<br>
not affect the ability to transfer the lgG variants to other parent IgGs.<br>
[159]	Methods for engineering, producing, and screening IgG variants are provided.<br>
The described methods are not meant to constrain to any particular application or theory of operation. Rather, the provided methods are meant to illustrate generally that one or more IgG variants may be engineered, produced, and screened experimentally to obtain IgG variants with optimized effector function. A variety of methods are described for designing, producing, and testing antibody and protein variants in USSN 10/754,296, and USSN 10/672,280, both entirely Incorporated by reference,<br>
[160]	A variety of protein engineering methods may be used to design IgG variants with<br>
optimized effector function. In one embodiment, a structure-based engineering method may be used, wherein available structural information is used to guide substitutions, insertions or deletions, In a preferred embodiment, a computational screening method may be used, wherein substitutions are designed based on their energetic fitness in computational calculations. 'See for example USSN 10/754,296 and USSN 10/672,280, and references cited therein, all entirely incorporated by reference.<br>
[161]	An alignment of sequences may be used to guide substitutions at the Identified<br>
positions. One skilled In the art will appreciate that the use of sequence Information may curb the Introduction of substitutions that are potentially deleterious to protein structure. The source of the sequences may vary widely, and include one or more of the known databases, including but not limited to ihe Kabat database (Northwestern University); Johnson &amp; Wu, 2001, Nucleic Adds Res. 29:205-206; Johnson &amp; Wu, 2000, Nucleic Acids Res, 28:214-218), the iMGT database (IMGT, the international ImMunoGeneTics information system®;; Lefranc eta/., 1999, Nucleic Adds Res. 27:209-212; Ruiz et aL, 2000 Nucleic Acids Res. 26:219-221; Lefranc et al., 2001, Nucleic Acids Res. 29:207-209; Lefranc ef ai, 2003, Nucleic Acids Res.<br>
 <br>
31 ;307-310), and VBASE, all entirely incorporated by reference. Antibody sequence information can be obtained, compiled, and/or generated from sequence alignments of germline sequences or sequences of naturally occurring antibodies from any organism, including but not limited to mammals. One skilled in the art will  appreciate that the use of sequences that are human or substantially human may further have the advantage of being less immunogenic when administered to a human. Other databases which are more general nucleic acid or protein databases, i.e. not particular to antibodies, include but are not limited to SwissProt, GenBank Entrez. and EMBL Nucleotide Sequence Database. Aligned sequences can include VH, VL, CH, and/or CL sequences. There are numerous sequence-based alignment programs and methods known in the art, and all of these find use in the generation of sequence alignments.<br>
{162]	Alternatively, random or semi-random mutagenesis methods may be used to<br>
make amino acid modifications at the desired positions, in these cases positions are chosen randomly, or amino acid changes are made using simplistic rules. For example all residues may be mutated to alanine, referred to as alanine scanning. Such methods may be coupled with more sophisticated engineering approaches that employ selection methods to screen higher levels of sequence diversity. As is well known in the art, there are a variety of selection technologies that may be used for such approaches, including, for example, display technologies such as phage display, riboeome display, cell surface display, and the like, as described below.<br>
[163]	Methods for production and screening of IgG variants are well known In the art.<br>
General methods for antibody molecular biology, expression, purification, and screening are described In Antibody Engineering, edited by Duebel &amp; Kontermann, Springer-Verlag, Heidelberg, 2001; and Hayhurst &amp; Georgiou, 2001, Curr Opin Chem Biol  5:683-689; Maynard &amp; Georgiou, 2000, Annu Rev Biomed Eng 2:339-76. Also see the methods described in USSN 10/754,296; USSN 10/572,260; and USSN 10/822,231; and 11/124,620, all entirely incorporated by reference.<br>
[164J	Preferred variants of the present fnvention include those found in Figure 8.<br>
Alternatively preferred variants of the present invention include those found in Figure 9. Additionally alternatively preferred variants of the present invention include those found In Figure 10' Particularly preferred variants of the present invention include G385H and N434Y. Most preferred variants of the present Invention include 257C, 257M, 257L, 257N, 257Y, 279Q, 279Y, 308F, and 308Y. Making IgG Variants<br>
[165]	The igG variants can be made by any method known in the art. In one<br>
embodiment, the IgG variant sequences are used to create nucleic acids that encode the member sequences, and that may then be cloned into host cells, expressed and assayed, if desired. These practices are earned out using well-known procedures, and a variety of methods that may find use in are described in Molecular Cloning - A Laboratory Manual, 3^ Ed, (Maniatis, Cold Spring Harbor Laboratory Press, New York, 2001), and Current Protocols<br>
 <br>
in Molecular Biology (John Wiley &amp; Sons), both entirely incorporated by reference. The nucleic acids that encode the IgG variants may be incorporated into an expression vector in order to express the protein. Expression vectors typically include a protein operably linked, that is, placed in a functional relationship, with control or regulatory sequences, selectable markers, any fusion partners, and/or additional elements, The IgG variants may be produced by culturing a host cell  transformed with nucleic acid, preferably an expression vector, containing nucleic acid encoding the IgG variants, under the appropriate conditions to induce or cause expression of the protein. A wide variety of appropriate host cells may be used, including but not limited to mammalian cells, bacteria, insect ceils, and yeast. For example, a variety of ceil lines that may find use are described in the ATCC cell line catalog, available from the American Type Culture Collection, entirely Incorporated by reference. The methods of introducing exogenous nucleic acid into host ceils are well known in the art, and will vary with the host cell used.<br>
In a preferred embodiment, IgG variants are purified or isolated after expression. Antibodies may be isolated or purified in a variety of ways known to those skilled in the art. Standard purification methods include chromatographic techniques, electrophoretic, Immunological, precipitation, dialysis, filtration, concentration, and chromatofocusing techniques. As is well known in the art, a variety of natural proteins bind antibodies, for example bacterial proteins A, G, and L, and these proteins may find use in purification. Often, purification may be enabfed by a particular fusion partner. For example, proteins may be purified using glutathione resin if a GST fusion is employed, Nl2 affinity chromatography if a His-tag is employed, or immobilized anti-flag antibody if a flag-tag is used. For general guidance in suitable purification techniques, see Antibody Purification: Principles and Practice, 3rd Ed., Scopes, Springer-Verlag, NY, 1994, entirely incorporated by reference. Screening IgG Variants<br>
[166]	IgG variants may be screened using a variety of methods, including but not<br>
limited to those that use in vitro assays, in vivo and cell-based assays, and selection technologies. Automation and high-throughput screening technologies may be utilized in the screening procedures. Screening may employ the use of a fusion partner or label, for example an immune label, isotopic label, or small molecule label such as a fluorescent or colorlmetric dye.<br>
[167]	In a preferred embodiment, the functional and/or biophysical properties of IgG<br>
variants are screened in an in vitro assay. In a preferred embodiment, the protein Is screened for functionality, for example its ability to catalyze a reaction or its binding affinity to its target. Binding assays can be carried out using a variety of methods known in the art, including but not limited to FRET (Fluorescence Resonance Energy Transfer) and BRET (Bioluminescence Resonance Energy Transfer) -based assays, AlphaScreen™ (Amplified Luminescent Proximity Homogeneous Assay), Scintillation Proximity Assay, EUSA (Enzyme-Linked immunosorbent Assay), SPR (Surface Piasmon Resonance, aiso known as BIACORE©), isothermal titration calorimetry, differential scanning calorimetry, gel electrophoresis, and<br>
 <br>
chromatography including gel filtration. These and other methods may take advantage of<br>
some fusion partner or label. Assays may employ a variety of detection methods including<br>
but not limited to chromogenic, fluorescent, luminescent, or isolopic labels. The biophysical<br>
properties of proteins, for example stability and solubility, may be screened using a variety of<br>
methods known in the art Protein stability may be determined by measuring the<br>
thermodynamic equilibrium between folded and unfolded states. For example, IgG variants<br>
may be unfolded using chemical denaturant, heat, or pH, and this transition may be monitored<br>
using methods including but not limited to circular dichroism spectroscopy, fluorescence<br>
spectroscopy, absorbance spectroscopy, NMR spectroscopy, calorimetry, and proteolysis.<br>
As will be appreciated by those skilled in the art, the kinetic parameters of the folding and<br>
unfolding transitions may also be monitored using these and other techniques. The solubility<br>
and overall structural integrity of a IgG variant may be quantitatively or qualitatively<br>
determined using a wide range of methods that are known in the art Methods which may find<br>
use in for characterizing the biophysical properties of IgG variants include gel electrophoresis,<br>
chromatography such as size exclusion chromatography and reversed-phase high<br>
performance liquid chromatography, mass spectrometry, ultraviolet absorbance spectroscopy,<br>
fluorescence spectroscopy, circular dichroism spectroscopy, isothermal titration calorimetry,<br>
differential scanning calorimetry, analytical ultra-centrifugation, dynamic light scattering,<br>
proteolysis, and cross-linking, turbidity measurement, filter retardation assays, immunological<br>
assays, fluorescent dye binding assays, protein-staining assays, microscopy, and detection of<br>
aggregates via EL1SA or other binding assay. Structural analysis employing X-ray<br>
crystallographic techniques and NMR spectroscopy may also find use.<br>
[168]	As is known in the art, a subset of screening methods comprises those that select<br>
for favorable members of a library. The methods are herein referred to as "seleclion methods", and these methods find use in screening IgG variants. When protein libraries are screened using a selection method, only those members of a library that are favorable, that is which meet some selection criteria, are propagated, isolated, and/or observed. As will be appreciated, because oniy the most fit variants are observed, such methods enable the screening of libraries that are larger ihan those screenable by methods that assay the fitness of library members individually. Selection is enabled by any method, technique, or fusion partner that links, covalently or noncovalentiy, the phenotype of a protein with its genotype, that is, the function of a protein with the nucleic acid that encodes it For example the use of phage display as a selection method is enabled by the fusion of library members to the gene ill protein, in this way, selection or isolation of IgG variants that meet some criteria, for example binding affinity to the protein's target, also selects for or isolates the nucleic acid that encodes it Once isolated, the gene or genes encoding Fc variants may lhen be amplified. This process of isolation and amplification, referred to as panning, may be repeated, allowing favorable IgG variants in the library to be enriched- Nucleic acid sequencing of the attached nucleic acid ultimately allows for gone identification.<br>
 <br>
[169]	A variety of selection methods are known in the art that may find use in screening<br>
protein libraries. These Include but are not limited to phage display (Phage display of peptides and proteins: a laboratory manual, Kay et al., 1996, Academic Press, San Diego, CA, 1998; Lowman etal., 1991, Biochemistry30:10832-10838; Smith, 1985, Science 228:1315-1317) and its derivatives such as selective phage infection (Malmborg et al., 1997, J Mol Biol 273:544-551), selectively infective phage (Krebber et a/., 1997, J Mo! Bio! 268:619-630), and delayed infectivity panning (3enhar et al, 2000, J Mol Biol 301:893-904), cell surface display (Witrrup, 2001, CurrOpin Biotechnoi, 12:395-399) such as display on bacteria (Georgiou etat., 1997, Nat Biotechnol 15:29-34; Georgiou et al., 1993, Trends Biotechnoi 11:6-10; Lee et al., 2000, Nat Biotechnoi 18:645-648; Jun et al, 1998, Nat Biotechno! 16:576-80), yeast (Boder &amp; Wttfrup, 2000, Methods Enzyme- 323:430-44; Boder &amp; Withrup, 1997, Nat Biotechnoi 15:553-557), and mammalian cells [Whltehom et al,, 1995, Bio/technoiogy 13:1215-1219), as well as in vitro display technologies (Amstutee* a/., 2001, Curr Opin Biotechnoi 12:400-405) such as polysome display (Mattheakis era/, 1994, Proc Nati Acad Sci USA 91:9022-9026), ribosome display (Hanes et a/., 1997, Proc NatiAcad Sci USA 94:4937-4942), mRNA display (Roberts &amp; Szostak, 1997, Proc Natl Acad Sci USA 94:12297-12302; Nemoto et al, 1997, FEBS Lett 414:405-40S}, and ribosome-inactivation display system (Zhou et al, 2002, J Am Chem Sec 124, 538-543). All references entirely incorporated by reference in this paragraph.<br>
	Other selection methods that may find use include methods that do not rely on<br>
display, such as in vivo methods Including but not limited to periplasms expression and cytometric screening (Chen et at, 2001, Nat Biotechnoi 19:537-542, entirely incorporated by reference), the protein fragment complementation assay (Johnsson &amp; Varshavsky, 1994* Proc' NatiAcad Sci USA 91:10340-10344; Pelietier et al., 1998, Proc NatiAcad Sci USA 95:12141-12145, all entirely incorporated by reference), and the yeast two hybrid screen (Fields &amp; Song, 1989, Nature 340:245-246, entirely incorporated by reference) used in selection mode (Visintin eta!., 1999, Proc Nat! Acad Sci USA 96:11723-11728, entirely incorporated by reference)'. In an alternate embodiment, selection is enabled by a fusion partner that binds to a specific sequence on the expression vector, thus linking covalently or noncovalently the fusion partner and associated Fc variant library member with the nucleic acid that encodes them. For example, USSN 09/642,574; USSN 10/080,376; USSN 09/792,630; USSN 10/023,208; USSN 09/792,626; USSN 10/082,871; USSN 09/953,351; USSN 10/097,100; USSN 60/366,658; PCTWO 00/22906; PCT WO 01/49058; PCT WO 02/04852; PCTWO 02/04853; PCT WO 02/08023; PCT WO 01/28702; and PCTWO 02/07466, alt entirely incorporated by reference, describe such a fusion partner and technique that may find use. In an alternative embodiment, in vivo selection can occur if expression of the protein Imparts some growth, reproduction, or survival advantage to the cell.<br>
[ 71]	A subset of selection methods referred to as "directed evolution" methods are<br>
those that include the mating or breading of favorable sequences during selection, sometimes with the Incorporation of new mutations. As will be appreciated by those skilled in the art,<br>
 <br>
directed evolution methods can facilitate Identification of the most favorable sequences in a library, and can increase the diversity of sequences that are screened. A variety of directed evolution methods ere known in the art that may find use in for screening IgG variants, including but not limited to DNA shuffling (PCT WO 00/42561 A3; PCT WO 01/70947 A3), exon shuffling (US 6,365,377; Kolkman &amp; Stemmer, 2001, Nat Biotechnol 19:423-428), family shuffling (Cramer! et a/., 1998, Nature 391:286-291; US 6,376,246), RACHITT™ (Coco et al., 2001, Nat Biotechnol 19:354- 359; PCT WO 02/06469), STEP and random priming of in vitro recombination (Zhao et al., 1998, Nat Bbtechnol 16:258-261; Shao et al 1998, Nucleic Acids Res 25:681 -683), exonuclease mediated gene assembly (US 6,352,842; US 6,361,974), Gene Site Saturation Mutagenesis™ (US 6,358,709), Gene Reassemblytm {US 6,358,709), SCRATCHY (Lutz et al., 2001, Proc Natl Acad Sci USA 98:11248-11253), DNA fragmentation methods (Kikuchi et al, Gene 236:159-167), single-stranded DNA shuffling (Kikuchi et al, 2000, Gene 243:133-137), and AMEsystem™ directed evolution protein engineering technology (Applied Molecular Evolution) {US 5,824,514; US 5,817,483; US 5,814,476; US 5,763,192; US 5,723,323). AH references cited in the paragraph are entirely incorporated by reference.<br>
	in a preferred embodiment, IgG variants are screened using one or more cell-<br>
based or in vivo assays. For such assays, purified or unpurified proteins are typically added exogenously such that cells are exposed to individual variants or pools of variants belonging to a library. These assays are typically, but not always, based on the function of the IgG; that is, the ability of the IgG to bind to its target and mediate some biochemical event, for example effector function, figand/receptor binding inhibition, apoptosis, and the tike. Such assays often involve monitoring the response of cells to the IgG, for example ceil survival, cell death, change in cellular morphology, or transcriptional activation such as cellular expression of a natural gene or reporter gene. For example, such assays may measure the ability of IgG variants to elicit ADCC, ADCP, or CDC. For some assays additional ceils or components, that is in addition to the target  cells, may need to be added, for example example serum complement, or effector cells such as peripheral blood monocytes (PBMCs), NK cells, macrophages, and the like. Such additional cells may be from any organism, preferably humans, mice,' rat, rabbit, and monkey. Antibodies may cause apoptosis of certain cell lines expressing the target, or they may mediate attack on target cells by immune ceils, which have been added to the assay. Methods for monitoring cell death or viability are known in the art, and include the use of dyes, Immunochemical, cytochemical, and radioactive reagents. For example, caspase staining assays may enable apoptosis to be measured, and uptake or release of radioactive substrates or fluorescent dyes such as alamar blue may enable cell growth or activation to be monitored. In a preferred embodiment, the DELF1A© EuTDA-based cytotoxicity assay (Perkin Elmer, MA) is used. Alternatively, dead or damaged target cells may be monitoried by measuring the release of one or more natural intracellular proteins, for example lactate dehydrogenase. Transcriptional activation may also serve as a method for assaying function In cell-based assays. In this case, response may be monitored<br>
 <br>
by assaying for natural genes or proteins which may be upregulated, for example the release of certain interleukins may be measured, or alternatively readout maybe via a reporter construct. Cell-based assays may also involve the measure of morphological changes of cells as a response to the presence of a protein, Celi types for such assays may be prokaryotic or eukaryotic, and a variety of cell lines that are known in the art may be employed- Alternatively, cell-based screens are performed using cells that have been transformed or transfected with nucleic acids encoding the variants. That is, IoG variants are not added exogenously to the cell- For example, in one embodiment, the cell-based screen utilizes cell surface display. A fusion partner can be employed that enables display of JgG variants on the surface of cells (Witrrup, 2001, Curr Optn Biotechnol, 12:395-399, entirely incorporated by reference).<br>
[173]	In a preferred embodiment, the immunogenicity of the IGG variants is determined<br>
experimentally using one or more cell-based assays. Several methods can be used for experimental confirmation of epitopes. In a preferred embodiment, ex vivo T-ceii activation assays are used to experimentally quantitate immunogenicity. In this method, antigen presenting cells and naive T cells from matched donors are challenged with a peptide or whole protein of interest one or more times. Then, T cefi activation can be detected using a number of methods, for example by monitoring production of cytokines or measuring uptake of tritiated thymidine. In the most preferred embodiment, interferon gamma production is monitored using Elispot assays (Schmittel el at, 2000, J. Immunol. Meth. 24: 17-24, entirely incorporated by reference).<br>
[174]	The biological properties of the IgG variants may be characterized in celi, tissue,<br>
and whole organism experiments. As is known in the art, drugs are often tested in animals, including but not limited to mice, rats, rabbits, dogs, cats, pigs, and: monkeys, in order to measure a drug's efficacy for treatment against a disease or disease model, or to measure a drug's pharmacokinetics, toxicity, and other properties. The animals may be referred to as disease models. Therapeutics are often tested In mice, including but not limited to nude mice, SClD mica, xenograft mics, and transgenic m)ce (including knockins and knockouts). Such experimentation may provide meaningful data for determination of the potential of the protein to be used as a therapeutic. Any organism, preferably mammals, may be used for testing. For example because of their genetic similarity to humans, monkeys can be suitable therapeutic models, and thus may be used to test the efficacy, toxicity, pharmacokinetics, or other property of the IgGs. Tests of the in humans are ultimately required for approval as drugs, and thus of course these experiments are contemplated. Thus the IgGs may be tested in humans to determine their therapeutic efficacy, toxicity, immunogenicity, pharmacokinetics, and/or other clinical properties, Methods of Using IgG Variants<br>
	The IgG variants may find use in a wide range of products, in one embodiment<br>
the IgG variant is a therapeutic, a diagnostic, or a research reagent, preferably a therapeutic. The IgG variant may find use In an antibody composition that is monoclonal or polyclonal. In<br>
 <br>
	The concentration of the therapeutically active IgG variant in the formulation may<br>
vary from about 0.1 to 100% by weight. In a preferred embodiment, the concentration of the IgG is in the range of 0.003 to 1.0 molar. In order to treat a patient, a therapeutically effective dose of the IgG variant may be administered. By "therapeutically effective dose" herein is meant a dose that produces the effects for which it is administered. The exact dose will depend on the purpose of the treatment, and will be ascertainable by one skilled in the art using known techniques. Dosages may range from 0.01 to 100 mg/kg of body weight or greater, for example 0.01, 0.1, 1.0, 10, or 50 mg/kg of body weight, with 1 to 10mg/kg being preferred. As is known in the art, adjustments for protein degradation, systemic versus localized delivery, and rate of new protease synthesis, as well as the age, body weight, general health, sex, diet, time of administration, drug interaction and the severity of the condition may be necessary, and will be ascertainable with routine experimentation by those skilled in the art.<br>
	Administration   of  the   pharmaceutical   composition   comprising   an   IgG   variant,<br>
preferably in the form of .a sterile aqueous solution, may be done in a variety of ways, including, but not limited to, orally, subcutaneously, intravenously, parenterafly, intranasally, intraotically, intraocularly, rectally, vaginally, transdermally, topically (e.g., gels, salves, lotions, creams, etc.), intraperitoneally, intramuscularly, intrapulmonary {e.g., AERx® inhalable technology commercially available from Aradigm, or Inhance® pulmonary delivery system commercially available from Nektar Therapeutics, etc.. Therapeutic described herein may be administered with other therapeutics concomitantly, i.e., the therapeutics described herein may be co-administered with other therapies or therapeutics, including for example, small molecules, other bioiogicals, radiation therapy, surgery, etc. EXAMPLES<br>
	Examples are provided below to illustrate the present invention. These examples are<br>
not meant to constrain the present invention to any particular application or theory of operation. For all positions discussed in the present invention, numbering is according to the EU index as in Kabat (Kabat et a/., 1991, Sequences of Proteins of Immunological Interest, 5th Ed., United States Public Health Service. National Institutes of Health, Bethesda. entirely incorporated by reference). Those skilled in the art of antibodies will appreciate that this convention consists of nonsequential numbering in specific regions of an immunoglobulin sequence, enabling a normalized reference to conserved positions in immunoglobulin families. Accordingly, the positions of any given immunoglobulin as defined by the EU index will not necessarily correspond to its sequential sequence.<br>
EXAMPLE 1: DNA construction, expression, and purification of Fc variants<br>
	Fc variants were constructed using the human lgG1 Fc domain and the variable<br>
domain of trastuzumab (Herceptin®, Genentech). The Fc polypeptides were part of Alemtuzumab, Trastuzumab or AC10. Alemtuzumab (Campath®,  a registered trademark of<br>
 <br>
The preferred anti-anglogenic factor herein is an antibody thai binds to Vascular Endothelial Growth Factor (VEGF). In an alternate embodiment, the IgG is administered with a therapeutic agent that induces or enhances adaptive immune response, for example an antibody that targets CTLA-4. In an alternate embodiment, the IgG is administered with a tyrosine kinase inhibitor. By "tyrosine kinase inhibitor" as used herein is meant a molecule that inhibits to some extent tyrosine kinase activity of a tyrosine kinase, in an alternate embodiment, the IgG variants are administered with a cytokine.<br>
	Pharmaceutical compositions are contemplated wherein an IgG variant and one<br>
or more therapeutically active agents are formulated. Formulations of the IgG variants are prepared for storage by mixing the IgG having the desired degree of purity with optional pharmaceutically  acceptable carriers, excipients or stabilizers (Remington's Pharmaceutical Sciences 16th edition, Osol, A. Ed., 1980, entirely incorporated by reference), in the form of lyophilized formulations or aqueous solutions. The formulations to be used for In vivo administration are preferably sterile. This is readily accomplished by filtration through sterile filtration membranes or other methods. The IgG variants and other therapeutically active agents disclosed herein may also be formulated as immunoliposomes, and/or entrapped in microcapsules.<br>
 <br>
a preferred embodiment, the IgG variants are used to kill target cells that bear the target<br>
antigen, for example cancer cells. In an alternate embodiment, the IgG variants are used to<br>
block, antagonize or agonize the target antigen, for example for antagonizing a cytokine or<br>
cytokine receptor, in an alternately preferred embodiment, the IgG variants are used to block,<br>
antagonize or agonize the target antigen and kill the target cells that bear the target antigen.<br>
[176]	The IgG variants may be used for various therapeutic purposes. In a preferred<br>
embodiment, an antibody comprising the IgG variant is administered to a patient to treat an antibody-related disorder. A "patient" for the purposes includes humans and other animals, preferably mammals and most preferably humans. By "antibody related disorder or "antibody responsive disorder" or "condition" or "disease" herein are meant a disorder that may be ameliorated by the administration of a pharmaceutical composition comprising an IgG variant. Antibody related disorders include but are not limited to autoimmune diseases, Immunological diseases, Infectious diseases, inflammatory diseases, neurological diseases, and oncological and neoplastic diseases including cancer. By "cancer" and "cancerous" herein refer to or describe the physiological condition in mammals that is typically characterized by unregulated cell growth. Examples of cancer include but are not limited to carcinoma, lymphoma, biastoma, sarcoma (including liposarcoma), neuroendocrine tumors, mesothelioma, schwanoma, meningioma, adenocarcinoma, melanoma, and leukemia and lymphoid malignancies,<br>
[177J	In one embodiment, an IgG variant is the only therapeutically active agent<br>
administered to a patient Alternatively, the IgG variant is administered in combination with one or more other therapeutic agents, including but not limited to cytotoxic agents, chemoxherapeutic agents, cytokines, growth inhibitory agents, anti-hormonal agents, kinase inhibitors, anti-angiogenic agents, cardioprotectants, or other therapeutic agents. The IgG varariants may be administered concomitantly with one or more other therapeutic regimens. For example, an IgG variant may be administered to the patient along with chemotherapy, radiation therapy, or both chemotherapy and radiation therapy. In one embodiment, the IgG variant may be administered in conjunction with one or more antibodies, which may or may not be an IgG variant. In accordance with another embodiment, the IgG variant and one or more other anti-cancer therapies are employed to treat cancer celis ex vivo. It is contemplated that such ex vivo treatment may be useful in bone marrow transplantation and particularly, autologous bone marrow transplantation. It is of course contemplated that the IgG variants can be employed in combination with still other therapeutic techniques such as surgery,<br>
[178]	A variety of other therapeutic agents may find use for administration with the IgG<br>
variants. In one embodiment, the IgG is administered with an anti-angiogenic agent. By "anti-angiogenic aaent" as used herein is meant a compound that blocks, or interferes to some degree, the development of blood vessels. The anti-angiogenic factor may, for instance, be a small molecule or a protein, for example an antibody, Fc fusion, or cytokine, that binds to a growth factor or growth factor receptor involved in promoting angiogenesis.<br>
 <br>
Millenium) is a humanized monoclonal antibody currently approved for  treatment of B-Ce|| chronic lymphocytic leukemia (Hale et a/., 1990, Tissue Antigens 35:118-127, entirely incorporated by reference). Trastuzumab (Herceptin®, a registered trademark of Genentech) is an anti-HER2/neu antibody for treatment of metastatic breast cancer. AC10 is a anti.CD30 monoclonal antibody. The Herceptin variable region was assembled using recursive pcR Thjs variab|e region was then cloned with human lgG1 into the pcDNA3.1/Zeo(+) vector (invitrogen). shown in Figure 11 Plasmids were propagated in One Shot TOP10 E. coli ceiis (invitrogen) and purified using the Hi_ Speed Plasmid Maxi Kit (Giagen). Plasmids were sequenced to verify the presence of the cloned inserts.<br>
[184]	Site-directed mutagenesis was done using the Quickchange™ method (Stratagene).<br>
Plasmids containing the desired substitutions, insertions, and deletions were propagated in One Shot TOP10 E. coli cells (Invitrogen) and purified using the Hi Speed plasmid Maxi  Kit (Qiagen) DNA was sequenced to confirm the fidelity of the sequences.<br>
[185]	Plasmids containing heavy chain gene (VH-Cy1-Cy2_Cy3) (wild-type or variants) were<br>
co-transfected with plasmid containing light chain gene (VL-CK) into 293T cells   Media were harvested 5 days after transfection. and antibodies were purified from the supernatant using protein  A  affinity   chromatography   (Pierce).   Antibody   contentrations  were   determined   by    biochlonic acid (BCA) assay (pierce) . EXAMPLE 2: Binding affinity measurements<br>
[186]	Binding  of Fc polypeptides- to Fc ligands was assayed with  surface  plasmon<br>
resonance measurements.' Surface plasmon resonance (SPR} measurements were performed using a BIAcore 3000 instrument (BIAcore AB). Wild-type or variant antibody was captured using immobilized protein L (Pierce Biotechnology, Rockford, IL). and binding to receptor analyte was measured. Protein L was covalently coupled to a CM5 sensor chip at a concentration of 1 uM in 10 mM sodium acetate, pH 4.5 on a CM5 sensor chip using N-hydroxysuccinimide/N-ethyl-N'-{-3-dimethylamino-propyl) carbodiimide (NHS/EDC) at a flow rate of 5 ul/min. Flow cell 1 of every sensor chip was mocked with NHS/EDC as a negative control of binding.Running buffer was 0.01 M HEPES pH 7.4, 0.15 M NaCI, 3 mM EDTA, 0.005% v/v surfactant p20 (HBS.EP] Biacore Uppsala, Sweden), and chip regeneration buffer was 10 mM g|ycine_HC( pH 15_ 125 nM Wild-type or variant trastuzumab antibody was bound to the protein t CM 5 chip in HBS-EP- at 1 ul/min for 5 minutes. FcRn-His-GST analyte, a FcRn fused to a His-tag and glutathione s transferase, in serial dilutions between 1 and 250 nM. were injected for 20 mjinutes association 10 minutes dissociation, in HBS-EP at 10 ul/min. Response, measures in resonance units (RU), was acquired at 1200 seconds after receptor injection, reflecting near steady state binding A cycle with antibody and buffer only provided baseline response. RU versus 1/ |og concentration plots were generated and fit to a sigmoidal dose response using noilIinear regression with GraphPad Prism.<br>
 <br>
[187]	Binding  of Fc  polypeptides to Fc  ligands was  also done with  AlphaScreen™<br>
(Amplified Luminescent Proximity Homogeneous Assay). AiphaScreen™ is a bead-based nonradioactive luminescent proximity assay. Laser excitation of a donor bead excites oxygen, which if sufficiently close to the acceptor bead will generate a cascade of chemiluminescent events, ultimately leading to fluorescence emission at 520-620 nm. The principal advantage of the AlphaScreen™ is its sensitivity. Because one donor bead emits up to 60,000 excited oxygen molecules per second, signal amplification is extremely high, allowing detection down to attomolar (10~18) levels. Wild-type antibody was biotinylated by standard methods for attachment to streptavidin donor beads, and tagged Fc ligand, for example FcRn, was bound to glutathione chelate acceptor beads. The AlphaScreen™ was applied as a direct binding assay in which the Fc/Fc ligand interactions bring together the donor and acceptor beads.  Addtionally, the AlphaScreen™ was applied as a competition assay for screening designed Fc polypeptides. In the absence of competing Fc polypeptides, wild-type antibody and FcRn interact and produce a signal at 520-620 nm. Untagged Fc domains compete with wild-type Fc/FcRn interaction, reducing fluorescence quantitatively to enable determination of relative binding affinities. EXAMPLE 3: FcRn-binding properties of Fc variants.<br>
[188]	Binding affinity of lgG1  Fc to FcRn was measured with variant antibodies using<br>
AlphaScreen™. The Fc polypeptides were part of Alemtuzumab or Trastuzumab. Alemtuzumab (Campath®, Hex) is a humanized monoclonal antibody currently approved for treatment of B-celi chronic lymphocytic ieukemia (Hale et a/., 1990; Tissue Antigens 35:118-127, entirely incorporated by reference). Trastuzumab (Herceptin®, Genentech) is an anti-HER2/neu antibody for treatment of metastatic breast cancer.<br>
[189]	Competitive AlphaScreen™ data were collected to measure the relative binding of<br>
the Fc variants compared to the wild-type antibody at pH6.0. Examples of the AlphaScreen™<br>
signal as a function of competitor antibody are shown in Figure 12. The 12 variant curves shown,<br>
those of P257L, P257N, V279E, V279Q, V279Y, A281S. E283F. V284E, L306Y. T307V. V308F,<br>
and Q311V, demonstrate increased affinity as each variant curve is shifted to the left of the wild-<br>
type curve in their box. Competition AphaScreen™ data for Fc variants of the present invention<br>
are summarized in Figures 13 and 14. The relative FcRn binding of the variant compared to wild<br>
type are listed. Values greater than one demonstrate improved binding of the Fc variant to FcRn<br>
compared to the wild type. For example, the variant E283L and V284E have 9.5-fold and 26-fold<br>
stronger binding than the wild type, respectively. Surface plasmon resonance measurements of<br>
many variants also show increased binding to FcRn as shown in Figure 15 and 16.<br>
[190]	At position 257, all variants that remove the imino acid, proline, and substitute an<br>
amino acid without the backbone- N to side chain covalent bond, allow the backbone more flexibility which allows more freedom for the Fc domain to better bind FcRn. In particular, variants  . at position 257 to L and N have strong FcRn binding at pH 6. demonstrating that the four atom<br>
 <br>
side chain and gamma branching pattern of the side chain helps the Fc domain make productive, ie strong, FcRn interactions. Position 308 interacts with position 257. Both of these positions in turn interact with H310, which is directly involved in the Fc/FcRn interactions (Table 2, Burmeister et al (1994) Nature 372:379-383, entirely incorporated by reference). The Fc variants V308F and V08Y have a 2.9-fold and 4.3-fold increase in FcRn affinity over wild type (Figure 13). Positions 279 and 385 interact with FcRn as variants V279E, V279Q and V279Y and G385H and G385N all. have stronger FcRn interactions. These variants all are to amino acids that are capable of hydrogen bonding.<br>
[191]	The Fc variant N434Y has particularly strong binding to FcRn at pH 6.0 as shown in<br>
figure 13. The single variant N434Y has 16-fold increased binding. Combinations of this variant with other modifications led . to even stronger binding. For example, P257L/N434Y, A281S/N434Y, and V308F/N434Y show 830-fold, 180-fold, and 350-fold increases in FcRn binding.<br>
EXAMPLE 4: Variants incorporating insertions and deletions.<br>
[192]	Insertions  and  deletions  that  alter the  strength  of  Fc/FcRn   interactions  were<br>
constructed and their binding properties to various Fc ligands were measured. An Fc variant with<br>
an inserted Ser residue between residues 281 and 282, using the EU numbering of Kabat et al,<br>
was designed to increase the FcRn binding properties of the Fc domain. This variant is referred<br>
to as A281S with "A" meaning an insertion following the position given. The inserted sequence,<br>
which may be more than one residue, is given following the position number. This Fc variant was<br>
constructed in the kappa, lgG1 anitbody trastuzumab (Herceptin®, Genetech) using methods<br>
disclosed herein. An insertion at the site between residues 281 and 282 shifts the Fc loop<br>
residues C-terminal of residue 281 toward the C-terminus of the loop and alters the side chain<br>
positioning. Fc variants comprising substitutions at positions 282, 283, and 284 suggested that<br>
the C-terminal shift of this loop was beneficial (See Figure 14). Another variant, a deletion of<br>
N286, sometimes referred to as N286#, was also constructed to shift the position of this FcRn-<br>
binding loop. Both of these variants show an increased binding affinity for FcRn at pH 6.0.<br>
[193]	The AlphaScreen™ data shows the binding of the A281S variant and other variants to<br>
FcRn, This AlphaScreen™ data was collected as a direct binding assay. Higher levels of chemiluminescent signals demonstrate stronger binding. As the concentrations of the variants are raised in the assay, stronger signals are created. These data at pH 6.0, in Figures 17a and 17b, demonstrate the increased affinity of A281S, P257L, P257L/A281S (a combination substitution/insertion variant) and other variants over the wild-type Fc. Also shown is a double substitution, T250Q/M428L, shown previously to have an increased serum half in monkeys (Hinton et al., 2004, J. Biol. Chem. 279(8): 6213-6216, entirely incorporated by reference). The insertion, A281S, alone increases the Fc/FcRn binding. Additionally, A281S further increases the binding of P257L when the two modifications'are combined in the variant P257L/rt281S as shown<br>
 <br>
in the -40nM data points. The data in figure 17c demonstrate that these variants do not show (increased FcRn binding at pH 7.0. The reduced affinity at pH 7.0 is desired for increased half-life in vivo, because it allows the release of Fc polypeptides from FcRn into the extracellular space, an important step in Fc recycling.<br>
[194}	Surface plasmon resonance experiments also demonstrate the improved binding of<br>
A281S to FcRn. Figure 18 shows the response units created as various Fc variant binding to FcRn on the chip surface. After allowing the variant to fully bind to the chip, the response units are recorded and shown on the ordinate. The insertion, *281S shows binding properties comparable to other variants shown herein to have increased affinity for FcRn over the wild type (See figures 13, 14 and 15, for examples).<br>
[195]	The deletion variant comprising a deletion of N286, N286#, also shows increased<br>
affinity for FcRn over wild type. This variant has a 2.0-fold increase in FcRn affinity as shown in Figure 13. The data therein are also AlphaScreen™ data collected as a competition experiment at pH 6.0. The variants are used to inhibit the binding of wild-type Fc, finked to the donor bead, with FcRn, linked to the acceptor beads. Two-fold less free N236# was needed than free wild-type Fc to inhibit the binding of the donor/acceptor beads through the Fc/FcRn complex. This demonstrates the 2-fold tighter binding of N286# over the wild type.<br>
[196}	Other Fc variants comprising insertions or deletions have decreased affinity for FcRn.<br>
The insertion variant, A254N has greatly decreased FcRn binding as would be expected from the nature and positioning of the variant. This variant places the insertion, an Asn, in the middle of an FcRn binding loop. This insertion has only 1.1% of the binding of the binding affinity of the wild type (Figure 13).<br>
[197] Whereas particular embodiments of the invention have been described above for purposes of illustration, it will be appreciated by those skilled in the art that numerous variations of the details may be made without departing from the invention as described in the appended claims. All references cited herein are incorporated in their entirety.<br>
 <br>
WE CLAIM:<br>
1.	A polypeptide comprising an Fc variant of an IgG Fc polypeptide, wherein said Fc variant comprises a serine at position 434, wherein said Fc variant exhibits increased binding to FcRn as compared to said human IgG Fc polypeptide, and wherein numbering is according to the EU index in Kabat et al.<br>
2.	A polypeptide as claimed in claim 1. wherein said polypeptide has specificity for a target molecule that is selected from the group consisting of a cytokine, a soluble protein factor, and a protein expressed on cancer cells.<br>
3.	A polypeptide as claimed in any one of claims 1-2, wherein said polypeptide is an antibody.<br>
4.	A polypeptide as claimed in any one of claims 1-3. wherein said polypeptide is an Fc fusion.<br>
5.	A method of producing a polypeptide as claimed in claims 1-4, said method comprising providing a cell comprising a nucleic acid encoding said polypeptide, wherein said cell is cultured under conditions suitable for expression of said polypeptide.<br>
6.	A method as claimed in claim 5, wherein said nucleic acid is contained in an expression vector.<br>
7.	A host cell comprising a nucleic acid encoding a polypeptide as claimed in claims 1 to 4.<br>
8.	An expression vector, wherein said expression vector encodes a polypeptide as claimed in<br>
claims 1-4.<br>
</td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/viewdoc.aspx?id=gsb0tbX+h5hwEo3O21UZSQ==&amp;amp;loc=vsnutRQWHdTHa1EUofPtPQ==" target="_blank" style="word-wrap:break-word;">http://ipindiaonline.gov.in/patentsearch/GrantedSearch/viewdoc.aspx?id=gsb0tbX+h5hwEo3O21UZSQ==&amp;amp;loc=vsnutRQWHdTHa1EUofPtPQ==</a></p>
		<br>
		<div class="pull-left">
			<a href="278410-a-process-for-preparation-of-a-formulation-for-hair-shine-and-hair-setting.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="278412-method-and-apparatus-related-to-composite-beacon-and-wideband-synchronization-signaling.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>278411</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>1066/MUMNP/2011</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>53/2016</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>23-Dec-2016</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>22-Dec-2016</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>26-May-2011</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>XENCOR INC.</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td>OF 111 WEST LEMON AVENUE, MONROVIA, CALIFORNIA, 91016 UNITED STATES OF AMERICA</td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>AARON KEITH CHAMBERLAIN</td>
											<td>OF 2445 E. DEL MAR BLVD., #405, PASADENA, CALIFORNIA 91107, UNITED STATES OF AMERICA,</td>
										</tr>
										<tr>
											<td>2</td>
											<td>JOHN DR. DESJARLAIS</td>
											<td>OF 1030 NORTH MICHIGAN AVENUE, PASADENA CALIFORNIA 91104, UNITED STATES OF AMERICA,</td>
										</tr>
										<tr>
											<td>3</td>
											<td>SHER BAHADUR KARKI</td>
											<td>OF 2833 PROVIDENCE WAY, POMONA, CALIFORNIA 91767, UNITED STATES OF AMERICA,</td>
										</tr>
										<tr>
											<td>4</td>
											<td>GREOGORY ALAN LAZAR</td>
											<td>OF 750 ARCADIA AVENUE, #6, ARCADIA, CALIFORNIA 91007, UNITED STATES OF AMERICA,</td>
										</tr>
										<tr>
											<td>5</td>
											<td>JOST VIELMETTER</td>
											<td>OF 495 ALAMEDA STREET , ALTADENA, CALIFORNIA 91001, UNITED STATES OF AMERICA,</td>
										</tr>
										<tr>
											<td>6</td>
											<td>SEAN CHRISTOPHER YODER</td>
											<td>OF 4168-C ORONTES WAY, SIMI VALLEY, CALIFORNIA 93063, UNITED STATES OF AMERICA,</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>C07K 16/00,C07K 16/24,C07K 16/32</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>PCT/US2005/041220</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td>2005-11-14</td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td>60/649,508</td>
									<td>2005-02-02</td>
								    <td>U.S.A.</td>
								</tr>
								<tr>
									<td>2</td>
									<td>60/726,453</td>
									<td>2005-10-12</td>
								    <td>U.S.A.</td>
								</tr>
								<tr>
									<td>3</td>
									<td>60/696,609</td>
									<td>2005-07-05</td>
								    <td>U.S.A.</td>
								</tr>
								<tr>
									<td>4</td>
									<td>60/690,200</td>
									<td>2005-06-13</td>
								    <td>U.S.A.</td>
								</tr>
								<tr>
									<td>5</td>
									<td>60/627,763</td>
									<td>2004-11-12</td>
								    <td>U.S.A.</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/278411-polypeptide by HTTrack Website Copier/3.x [XR&CO'2014], Thu, 04 Apr 2024 23:06:45 GMT -->
</html>
