<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/278739-formulation-of-bacterial-consortium-for-degradation-of-high-density-polyethylene by HTTrack Website Copier/3.x [XR&CO'2014], Thu, 04 Apr 2024 23:29:10 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 278739:&quot;FORMULATION OF BACTERIAL CONSORTIUM FOR DEGRADATION OF HIGH-DENSITY POLYETHYLENE&quot;.</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">&quot;FORMULATION OF BACTERIAL CONSORTIUM FOR DEGRADATION OF HIGH-DENSITY POLYETHYLENE&quot;.</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>This invention relates to a formulation of bacterial consortium for degradation of high density polyethylene, developed by selective adaptability and enrichment under in situ conditions, comprising, Microbacterium sp. strain MK3 (DQ 318884), Pseudomonas putida strain MK 4 (DQ 318885), Bacterium Te68R strain PN12 (DQ 423487) in minimal broth Davis medium without dextrose containing gm per liter, 7.0 K2 HPO4; 2.0 KH2 PO4; 0.5 Na3C6 H5O7; 0.1 MgSO4.7H2O and 1.0 (NH4)2 S04, wherein an aliquot of 200 ml is withdrawn from glycerol stocks and cultures are revived by inoculating into 5 ml Nutrient Broth at their optimum pH 7 ± 0.02 and temperature 37 ± 1°C for overnight (12 hrs) with continuous shaking at 120 rpm.</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td>"Formulation  of bacterial consortium  for degradation  of high-density polyethylene".<br>
FIELD OF INVENTION<br>
This   invention   relates   to   formulation   of   bacterial   consortium   for degradation of high-density polyethylene.<br>
BACKGROUND OF INVENTION<br>
Polyethylene (PE) can be classified into high-density and low-density variants (HDPE and LDPE). LDPE is characterized by good toughness, resistance to chemicals, flexibility, and clarity, whereas HDPE is more rigid, harder, and has a greater tensile strength than the former. Because of their high durability, they accumulate in the environment at the rate of 25 million tons per year (Orhan and Buyukgungor, 2000). It has been observed that a PE sheet incubated in moist soil for 12 years shows no signs of deterioration (Potts, 1978) and only partial degradation could be seen after 32 years (Otake et al, 1995). The resistance of polyethylene to biodegradation stems from its high molecular weight, three-dimensional structure, hydrophobic nature (Hadad et al., 2005) and lack of functional groups recognizable by existing microbial enzyme systems (Chiellini et al, 2003). Major strategies to facilitate PE disintegration and subsequent biodegradation, were focused on the direct incorporation of carbonyl groups within the backbone or on their in-situ generation by pro-oxidant additives like polyunsaturated compounds, transition metal ions and dithiocarbamates. These functional groups act as initiators of thermal and photo-oxidation of the hydrocarbon polymer chains (Chiellini et al., 2003), thereby increasing<br>
the   surface   hydrophilicity   and   facilitating   biodegradation   by  microorganisms.<br>
Various studies (Lee et al., 1991; Glass and Swift, 1989; Imam et al., 1992; Gu, 2003) have illustrated the biodegradability of some of polymer films by measuring changes in physical properties or by observation of microbial growth after exposure to biological or enzymatic environments. Several studies have demonstrated partial biodegradation of polyethylene after U.V. irradiation (Cornell et al, 1984) and thermal treatment (Volke-Sepulveda et al, 2002). PE can be made biodegradable by modifying its crystalline level, molecular weight and mechanical properties that make PE recalcitrant (Albertsson et al., 1994). This can be achieved by improving PE hydrophilic level and/or shortening the polymer chain length by oxidation to be accessible for microbial degradation (Bikiaris et al., 1999). Further, biodegradability of PE can be increased by modifications due to the two additives, starch and pro-oxidants, used in the synthesis of biodegradable polyethylene. Starch blend polyethylene makes the material hydrophilic and is therefore hydro-biodegraded by amylase enzymes. In case of pro-oxidant additive like TDPA or. transition metals, biodegradation occurs following photo and chemical degradation of the polymer (Bonhomme et al., 2003; EI-Shafei et al., 1998; Yamada-Onodera et al., 2001). El-Shafei et al. (1998) investigated the ability of fungi and Streptomyces strains to attack degradable polyethylene bags containing 6% starch. Gilan et al., 2004 isolated a strain of Rhodococcus ruber that could colonize &amp; degrade PE. The ability of this bacterium to form biofilm was attributed to the hydrophobicity of its cell surface. Further, addition of mineral oil (0.05 %) was shown to increase the biofilm formation. Bacteria themselves are<br>
known to release amphiphilic surface active agents called bio surfactants that could adhere to and emulsify hydrophobic surfaces like petroleum oil-spills (Chakrabarty, 1985).<br>
Albertsson and Banhidi (1980) examined the biodegradation of high-density (linear) polyethylene (HDPE) film (Mw 93,000) for 2 years and found that the short-chain oligomeric fraction contained in HDPE film is the main degraded component. Albertsson and Karlsson (1993) indicated that biodegradation of HDPE is limited in the presence of mineral oil, which is preferentially utilized as a carbon source. Orhan et al, (2004) evaluated the degradation of compost bag strips made of supposedly degradable polyethylene and non-degradable LDPE and HDPE in soil mixed with 50% (w/w) mature municipal solid waste compost. The examined films were ranged in order of decreasing susceptibility: degradable PE &gt;&gt;&gt; LDPE &gt; HDPE. It was also found that natural photo-oxidation of HDPE composites could be increased with several inorganic fillers like kaolin, diatomite and mica (Yang et ah, 2005). Koutny et al., 2006 studied the biodegradability of HDPE and LDPE, both containing a balance of antioxidants and pro-oxidants incubated with Rhodococcus rhodochrous and Nocardia asteroides in mineral medium for 200 days which revealed surfacial degradation. Sudhakar et al., (2007) studied biofouling and biodegradation of polyolefins in ocean waters and found maximum weight loss in LDPE (1.5-2.5%), followed by that in HDPE (0.5-0.8%) and finally in propylene (PP) (0.5-0.6%) samples after six month time period. In another study, two marine micro-organisms viz. Bacillus sphericus and Bacillus cereus have also been recently reported for degradation of LDPE and HDPE<br>
(Sudhakar et al., 2008). The weight loss of the thermally treated LDPE and HODPE samples were about 19% and 9% respectively, and unpretreated samples were 10% and 3.5% respectively with B. sphericus in 1 year. Further, a consortium of Bacillus cereus, Bacillus pumilus species and Arthrobacter sp. was reported to degrade both LDPE as well as HDPE to an extent of nearly 22% within a period of two weeks (Satlewal et al., 2008).<br>
•	The techniques used for ascertaining degradation have been limited.<br>
•	Individual cultures have been used as opposed to microbial consortia.<br>
•	The incubation time period has been long ranging from several months up to 2 years.<br>
•	Inventors have used FTIR; along with simultaneous TG-DTG-DTA and SEM which have illustrated substantiate degradation of HDPE by the use of microbial consortia.<br>
•	Simultaneous TG-DTG-DTA has reported multiple-step degradation of the degraded polymer at lower temperatures. This suggests formation of macromolecules of various chain lengths which is further validated by SEM photographs, subsequently.<br>
•	Incubation period with the consortium has been considerably reduced to 10 days.<br>
HDPE is a widely used polymer which is stronger and harder than LDPE. Being recalcitrant, it significantly contributes to the global waste problem<br>
which needs serious attention in terms of its biodegradation. Therefore, this invention has both national and international utility.<br>
To the best of our knowledge, no researcher has yet patented bacterial consortia for HDPE degradation. However, there have been instances of related patents. For example, Yagi et al., 2001 (US 6,313,194 Bl) patented a method for degrading polymers by bringing the same into contact with a solid phase composed of a carrier, microorganisms like Escherichia, Pseudomonas, Bacillus, fungi, etc. and an aqueous solution. Similarly, Abe et al., 2006 (US 2006/0106120 Al) described a ' method of degrading plastics in the presence of a biosurfactant and plastic-degrading enzymes viz. esterase, protease, peptidase, lipase, cutinase and serine hydrase in protein preparation forms or with those as produced by Aspergillus strains. In another instance, Kambe and Shigeno, 2007 (US 2007/0099285 Al) have described Rhodococcus to be able to degrade low-molecular weight urethane compounds. However, there are instances of numerous patents on the formulation and degradation of biodegradable or additive-based polymers.<br>
OBJECTS OF INVENTION<br>
The main object of the invention is to develop formulation of bacterial consortium for degradation of high density polyethylene.<br>
Other object is to explore and characterize indigenous microbial flora which can facilitate polymer biodegradation.<br>
Another object is to do physico-chemical analysis to establish the biodegradation.<br>
Yet another object is to develop formulation by selective adaptability and enrichment under in-situ conditions.<br>
STATEMENT OF INVENTION<br>
This invention relates to a formulation of bacterial consortium for degradation of high density polyethylene, developed by selective adaptability and enrichment under in-situ conditions, comprising, Microbacteriurn sp. strain MK3 (DQ 318884), Pseudomonas putida strain MK 4 (DQ 318885), Bacterium Te68R strain PN12 (DQ 423487), in minimal broth Davis medium without dextrose containing gm per liter, 7.0 K2 HPO4; 2.0 KH2PO4; 0.5 Na3Ce H5O7; 0.1 MgS04.7H20 and 1.0 (NH4)2 SO4, wherein an aliquot of 200 ml is withdrawn from glycerol stocks and cultures are revived by inoculating into 5 ml Nutrient Broth at their optimum pH 7 + 0.02 and temperature 37 + 1°C for overnight (12 hrs) with continuous shaking at 120 rpm.<br>
BRIEF DESCRIPTION OF ACCOMPANYING DRAWINGS<br>
Fig. 1 shows that in the presence of HDPE, the consortium exhibited a decrease in the biomass, but the stationary phase was unaffected and was attained after 7 d in both the cases. Further, the A-max shifted from 201 nm to 205 nm between 7th and 10th days, which suggest that changes in the polymer backbone occurred rapidly within stationary-<br>
phase. However, in case of negative control, the A-max was constant (201 nm) throughout the experimental study. Furthermore, the cfu/ml counts (~112xl06) and generation time (120 h) were nearly unaffected in the presence of HDPE.<br>
Fig. 2 shows that pure HDPE exhibits FT-IR absorptions (KBr, cm-!) corresponding to p CH2 (721.5), 8 CH2 (1467.0), 8 CH3 (symmetrical, 1353.0), &gt;CH2 deformation (1595.9), vs CH2 (2850.1), vas CH2 (2919.9), v CH (3412.3), along with a pair of combination bands due to 8 CH2 and p CH2 at 2340.5 and 2367.9, respectively (2A). Biodegradation with the consortium brought about significant shifts in the fingerprint region of the IR spectrum between 1300 cm4 and 950 cm-1 of HDPE as compared to the control. The degraded sample recorded newer absorptions (cm1) corresponding to 8S CH3 (1383.2) along with a pair of combination bands due to 8 CH2 and p CH2 at 2341.3 and 2368.7, respectively (2B). Further, inclusion of O-atoms into HDPE due to microbial action has introduced v C-O frequencies corresponding to 1086.3 cm"1. In addition to this, deletion of frequencies corresponding to p CH2, 8 CH2, vs CH2 and vas CH2 were also observed.<br>
Fig. 3 shows thermal decomposition of untreated HDPE (control) and its biodegraded samples in the temperature range of 20°C to 550°C. Decomposition of pure HDPE was observed in one-step with a steep weight loss in the temperature ranging 417 to 470°C (3A). However, prior to this temperature, HDPE has shown a DTA endotherm at 123°C with heat of decomposition (H) =153 mJ/mg. The steep weight loss range of HDPE was supported with a DTA endotherm at 455°C with<br>
H = 109 mJ/mg and a DTG peak at 454°C with rate of decomposition 2.0 mg/min.<br>
Conversely, the biodegraded sample recovered from the assay exhibited two-step decomposition (3B). The first-step was observed at 117°C with a weight loss of 7.32%, supported by a combination of DTA endotherms at 119°C, 123°C, 134°C (combined H = 125 mJ/mg) and a combination of DTG peaks at 119°C, 133°C, 147°C with rates of decomposition, 0.16, 0.21, 0.15 mg/min, respectively. The second step was observed at 182°C (17.65% weight loss), supported by a DTA endotherm at 195°C (H = 37.8 mJ/mg) and a DTG peak at 194°C with 0.0 mg/min rate of decomposition. Furthermore, the degree of biodegradation could be confirmed at 400°C, whereby the biodegraded sample showed 24.12% weight loss as compared to the control which exhibited a weight loss of 2.5%.<br>
Fig. 4 shows that in case of undegraded powdered HDPE, the SEM micrographs revealed uneven surface morphology with depressions and elevations throughout (4A). However, as a result of biodegradation, a sharp change in the surface texture was observed as characterized by streaks and patches of whitened areas (4B). Further, the characteristic floral pattern depicted in the control was also suggestively assimilated by the consortium.<br>
DETAILED DESCRIPTION OF INVENTION<br>
The present invention relates to the isolation of a set of bacteria from soil samples and artificial soil beds (containing polyethylene pieces) at Pantnagar, Uttarakhand and their identification by molecular characterization (16S rDNA sequencing). Furthermore, it is aimed at employing these bacteria for the formulation of a microbial consortium capable of degrading high-density polyethylene (HDPE) under in-vitro conditions.<br>
The in-vitro biodegradation assays were carried out in 300 ml minimal broth. The quality checks were kept with suitable positive and negative controls. Herein, we have ascertained the process of biodegradation of HDPE with FTIR spectra, thermal analysis and scanning electron microscopy, respectively.<br>
This invention deals with an indigenous consortium which was developed by selective adaptability and enrichment under in-situ conditions. The individual bacteria used for consortium development were isolated, purified, characterized and conserved individually in five steps. Further, consortium development was also carried out in four different steps. Therefore, the specific combination of these three individual bacteria in defined quantity has tremendous utility for HDPE biodegradation.<br>
MATERIALS AND METHODS USED<br>
Polyethylene<br>
High-density polyethylene (HDPE; purchased from Sigma-Aldrich Chemical Corporation, U.S.A.) beads were powdered in xylene. The<br>
 <br>
powdered HDPE was successively washed with 70% ethanol and used for biodegradation studies as a primary carbon source. The polyethylene beads were powdered so as to increase the surface area and to allow better degradative action of the consortium.<br>
Bacterial isolates<br>
The bacterial cultures were obtained from departmental culture collection, which were originally isolated from different plastic waste disposal sites &amp; artificial soil beds (Table 1 and 2). The bacterial strains were characterized by 16S rDNA sequencing and identified by similarity scores returned by NCBI-BLAST. The sequences were submitted to NCBI GenBank. The cultures were identified as Microbacterium sp. strain MK3 (DQ318884), Pseudomonas putida strain MK4 (DQ318885) and Bacterium Te68R strain PN12 (DQ423487). The medium used for biodegradation study was Minimal Broth Davis without dextrose (minimal broth) containing gm per liter: 7.0 K2HPO4; 2.0 KH2PO4; 0.5 Na3C6H5O7; 0.1 MgSO4.7H2O and 1.0 (NH4)2SO4 (Hi Media, Mumbai, India). An aliquot of 200 ml was withdrawn from glycerol stocks and the cultures were revived by inoculating into 5.0 ml Nutrient Broth (Hi Media, India) test tubes at their optimum pH (7±0.02) and temperature (37±1°C).<br>
Active consortium preparation<br>
Based on preliminary nutritional screening, active consortia were prepared. A single colony from each bacterial strain was inoculated in Nutrient Broth and incubated at optimum pH (7±0.02) and temperature (37±1°C) for overnight (12 h) with continuous shaking (120 rpm) until an<br>
OD of 0.6 was reached at 600 nm [OD600]. Absorbance was recorded by using UV-Vis Spectrophotometer (Perkin Elmer, Lambda 35). The individual strains of each consortium were mixed at equal proportions of the order of 35 x 105 colony forming units and added into minimal broth Davis w/o dextrose. The broth was incubated at 37°C and 120 rpm till the stationary phase was over (10 days).<br>
In-vitro Biodegradation Assay<br>
For the biodegradation assay, 300 ml Minimal broth Davis w/o dextrose (pH 7.0+0.2) was taken in 500 ml Erlenmeyer flasks containing HDPE at a concentration of 5 mg/ml. The flasks were inoculated with 90 µl of active consortium. The assay was performed with respective positive (minimal broth + consortia) and negative (minimal broth + HDPE) controls. The flasks were incubated at 37°C with continuous shaking (120 rpm). Degraded samples were recovered from the broth after the stationary growth phase of the consortium was over, i.e. 10 days.<br>
Recovery of Degraded Products<br>
Degraded compound was recovered from the broth after filtration and subsequent evaporation of the filtrate. The residue left after filtration was collected and centrifugation of the filtrate was done at 5000 rpm for 15 minutes to remove bacterial biomass. Further, supernatant was kept in oven at 60°C for overnight to evaporate water and the residual sample was recovered.<br>
Analysis for Biodegradation<br>
(a)	Determination of A-max Shift<br>
Apart from OD600, A-max was determined for the assay, along with the positive and negative controls, at 3rd, 7th and 10th day after which the consortium attained stationary phase. The change in the A-max values between the control and treatments was calculated.<br>
(b)	Fourier Transform Infra-red (FTIR)<br>
The supernatant and residue samples obtained after the assay were analyzed by FTIR and different peaks relative CH2 deformation, CH2 bending (symmetrical), CH2 bending (asymmetrical), CH2 stretching, CH stretching and C-O bond were compared taking pure HDPE as a reference. The spectra were recorded on Perkin Elmer FTIR Spectrophotometer in KBr.<br>
(c)	Simultaneous TG-DTG-DTA<br>
From the FTIR studies, it was observed that biodegraded HDPE was present in the sample collected from supernatant than in the residue. Therefore, the supernatant was further subjected to thermal analysis. Simultaneous thermogravimetric-derivative thermogravimetry-differential thermal analysis (TG-DTG-DTA) was performed over Perkin Elmer (Pyris Diamond) thermal analyzer under nitrogen atmosphere (200 ml/min) from 20°C to 550°C at 5°C/min on a platinum pan.<br>
(d)	Scanning Electron Microscopy (SEM)<br>
SEM analysis of both biodegraded and undegraded samples was performed. HDPE samples were collected and surface sterilized with 70% ethanol for 10 min, before drying them in a desiccator for 24 h under vacuum. The samples were metallized with gold particles (3 discharges of 40 mA/ 50s in argon atmosphere) in a high vacuum metalizator (Bal-Tec<br>
SCD  005)  and analyzed by scanning electron  microscopy (Leo, 435VF, UK) at 15.00 kV EHT.<br>
Table 1: Isolation profile and 16S rDNA based characterization of the strains<br>
(Table Removed) <br>
Table 2: Morphological characteristics of the bacterial strains<br>
(Table Removed) <br>
The   consortia   will   be   useful   for   the   degradation   of   high-density polyethylene (HDPE) when incubated in 300 ml minimal broth lacking<br>
dextrose (pH 7.0±0.02) as C-source at 37°C &amp; 120 rpm for 10 days. The polymer undergoes progressive degradation of their hydrocarbon backbone followed by introduction of hydroxyl groups, subsequently.<br>
Defined colony forming units (cfu/ml) of individual strains of a consortium are added into 300 ml minimal broth Davis w/o dextrose in 500 ml Erlenmeyer flasks containing 5 mg/ml powdered HDPE. The flasks are incubated till the composite stationary phase of the consortium, i.e. 10 days, is over. With the progression of the assay, A-max shift is determined spectrophotometncally for suggestive changes in polymer structure within the medium. The degraded polymer is then carefully extracted from the broth and analyzed for degradation using FTIR, simultaneous TG-DTG-DTA and SEM analysis. The present invention further relates to the biodegradation of other polymers such as low-density polyethylene (Soni et al., 2009), polycarbonate (Goel et al., 2008), epoxy (Negi et al., 2009), polysulfone, PVC, etc.<br>
<br>
<br>
<br>
<br>
<br>
<br>
<br>
<br>
<br>
WE CLAIM;<br>
1.	A formulation of bacterial consortium for degradation of high<br>
density polyethylene, developed by selective adaptability and<br>
enrichment under in situ conditions, comprising: -<br>
i)	Microbacterium sp. strain MK3 (DQ 318884)<br>
ii)        Pseudomonas putida strain MK 4 (DQ 318885) iii)       Bacterium Te68R strain PN12 (DQ 423487)<br>
in minimal broth Davis medium without dextrose containing gm per liter, 7.0 K2 HPO4; 2.0 KH2 PO4; 0.5 Na3C6 H5O7; 0.1 MgSO4.7H2O and 1.0 (NH4)2 SO4, wherein an aliquot of 200 ml is withdrawn from glycerol stocks and cultures are revived by inoculating into 5 ml Nutrient Broth at their optimum pH 7 + 0.02 and temperature 37 + 1°C for overnight (12 hrs) with continuous shaking at 120 rpm.<br>
2.	A formulation of bacterial consortium as claimed in claim 1,<br>
wherein a single colony from each bacterial strain is inoculated<br>
and the culturing continues until an OD of 0.6 is reached at 600<br>
nm.<br>
3.	A formulation of bacterial consortium as claimed in claim 1, wherein the individual strains of each consortium are mixed at equal proportions of the order of 35 x 105 colony forming units and added into minimal broth Davis medium.<br>
4.	A formulation of bacterial consortium for degradation of high density-polyethylene as described and illustrated herein.<br>
</td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/viewdoc.aspx?id=qA0HC114Zs6LC5xWbZYoWQ==&amp;amp;loc=+mN2fYxnTC4l0fUd8W4CAA==" target="_blank" style="word-wrap:break-word;">http://ipindiaonline.gov.in/patentsearch/GrantedSearch/viewdoc.aspx?id=qA0HC114Zs6LC5xWbZYoWQ==&amp;amp;loc=+mN2fYxnTC4l0fUd8W4CAA==</a></p>
		<br>
		<div class="pull-left">
			<a href="278738-a-reinforcing-bar.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="278740-resuscitation-fluid.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>278739</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>1290/DEL/2010</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>54/2016</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>30-Dec-2016</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>29-Dec-2016</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>02-Jun-2010</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>DEPARTMENT OF BIOTECHNOLOGY,</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td>BLOCK-2, 7TH FLOOR, CGO COMPLEX, LODI ROAD, NEW DELHI-110 003</td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>ANIL KAPRI</td>
											<td>DEPARTEMENT OF MICROBIOLOGY AND DEPARTMENT OF CHEMISTRY, COLLEGE OF BASIC SCIENCES AND HUMANITIES, G.B. PANT UNIVERSITY OF AGRICULTURE AND TECHNOLOGY, PANTNAGAR, UTTARAKHAND-263145.</td>
										</tr>
										<tr>
											<td>2</td>
											<td>ALOK SATLEWAL</td>
											<td>DEPARTEMENT OF MICROBIOLOGY AND DEPARTMENT OF CHEMISTRY, COLLEGE OF BASIC SCIENCES AND HUMANITIES, G.B. PANT UNIVERSITY OF AGRICULTURE AND TECHNOLOGY, PANTNAGAR, UTTARAKHAND-263145.</td>
										</tr>
										<tr>
											<td>3</td>
											<td>MS.HARSHITA NEGI</td>
											<td>DEPARTEMENT OF MICROBIOLOGY AND DEPARTMENT OF CHEMISTRY, COLLEGE OF BASIC SCIENCES AND HUMANITIES, G.B. PANT UNIVERSITY OF AGRICULTURE AND TECHNOLOGY, PANTNAGAR, UTTARAKHAND-263145.</td>
										</tr>
										<tr>
											<td>4</td>
											<td>M.G.H.ZAIDI</td>
											<td>DEPARTEMENT OF MICROBIOLOGY AND DEPARTMENT OF CHEMISTRY, COLLEGE OF BASIC SCIENCES AND HUMANITIES, G.B. PANT UNIVERSITY OF AGRICULTURE AND TECHNOLOGY, PANTNAGAR, UTTARAKHAND-263145.</td>
										</tr>
										<tr>
											<td>5</td>
											<td>REETA GOEL</td>
											<td>DEPARTEMENT OF MICROBIOLOGY AND DEPARTMENT OF CHEMISTRY, COLLEGE OF BASIC SCIENCES AND HUMANITIES, G.B. PANT UNIVERSITY OF AGRICULTURE AND TECHNOLOGY, PANTNAGAR, UTTARAKHAND-263145.</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>C12Q</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>N/A</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td></td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td></td>
									<td></td>
								    <td>NA</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/278739-formulation-of-bacterial-consortium-for-degradation-of-high-density-polyethylene by HTTrack Website Copier/3.x [XR&CO'2014], Thu, 04 Apr 2024 23:29:11 GMT -->
</html>
