<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/279131-a-composition-for-conferring-tolerance-to-viral-disease-in-social-insects by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 00:27:47 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 279131:A COMPOSITION FOR CONFERRING TOLERANCE TO VIRAL DISEASE IN SOCIAL INSECTS</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">A COMPOSITION FOR CONFERRING TOLERANCE TO VIRAL DISEASE IN SOCIAL INSECTS</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>Compositions and methods for reducing susceptibility to infectious disease in bees using RNA interference technology, and more particularly, prevention and treatment of viral infections in honeybees such as Israel acute paralysis virus (IAPV) by feeding of pathogen-specific dsRNA. Further, multiple-pathogen specific dsRNA is disclosed.</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td>FORM 2 <br>
                                      THE PATENTS ACT, 1970<br>
(39 of 1970)<br>
&amp;<br>
 The Patent Rules, 2003<br>
COMPLETE SPECIFICATION<br>
1.	TITLE OF THE INVENTION:<br>
COMPOSITIONS FOR CONFERRING TOLERANCE TO VIRAL DISEASE IN SOCIAL INSECTS, AND THE USE THEREOF<br>
2.	APPLICANT:<br>
Name: BEEOLOGICS, LLC<br>
Nationality: USA<br>
Address:11800 SW 77th Avenue, Miami, Florida 33156, USA.<br>
3.	PREAMBLE TO THE DESCRIPTION:<br>
The following specification particularly describes the invention and the manner in which it is to be performed:<br>
 <br>
FIELD AND BACKGROUND OF THE INVENTION<br>
The invention relates to compositions and methods for reducing susceptibility to infectious disease in bees using RNA interference technology, and more particularly, to the use of dsRNA for prevention and treatment of viral infections in honeybees.<br>
Colony Collapse Disorder<br>
The importance of honeybees and other pollinating insects to the global world economy far surpasses their contribution in terms of honey production. The United States Department of Agriculture (USDA) estimates that every third bite we consume in our diet is dependent on a honeybee to pollinate that food. The total contribution of pollination in terms of added value to fruit crops exceeds $15 billion per annum, with indirect potential consequence of $75 billion dollars.<br>
Viral diseases in Honeybees<br>
The health and vigor of honeybee colonies are threatened by numerous parasites and pathogens, including viruses, bacteria, protozoa, and mites, each with characteristic modes of transmission.<br>
In general, transmission of viruses can occur via two pathways: horizontal and vertical transmission. In horizontal transmission, viruses are transmitted among individuals of the same generation, while vertical transmission occurs from adults to their offspring. Transmission can occur through multiple routes in social organisms (for a detailed review see Chen YP, et al (2006) Appl Environ Microbiol. 72(1):606-11). Recently, horizontal transmission of honeybee viruses has been documented in bee colonies, for example, transmission of deformed wing virus (DWV) and Kashmir Bee Virus (KBV) by the parasitic mite Varroa destructor, as well as some evidence of virus in honeybee eggs and young larvae, life stages not parasitized by Varroa mites. Vertical transmission of multiple viruses from mother queens to their offspring in honeybees has also been recently demonstrated, as well as viruses in feces of queens, suggesting a role for feeding in virus transmission. Moreover, honeybee viruses have been detected in tissues of the gut, suggesting that viruses could be ingested by queens from<br>
 <br>
contaminated foods and passed into the digestive tract, which then acts as a major reservoir for viral replication. Indeed, viruses might penetrate the gut wall and move into the insect hemocoel, spreading infections to other tissues.<br>
In honeybees viruses often persist as latent infections. Thus, group living activities such as trophylaxis and nurse bee brood feeding, can potentially drive high levels of horizontal transmission or amplification of existing infections.<br>
Colony Collapse Disorder<br>
Colony Collapse Disorder (CCD) of honeybees is threatening to annihilate U.S. and world agriculture. Indeed, in the recent outbreak of CCD in the U.S in the winter of 2006-2007, an estimated 25% or more of the 2.4 million honeybee hives were lost because of CCD. An estimated 23% of beekeeping operations in the United States suffered from CCD over the winter of 2006–2007, affecting an average of 45% of the beekeepers operations. In the winter of 2007-2008, the CCD action group of the USDA-ARS estimated that a total of 36% of all hives from commercial operations were destroyed by CCD.<br>
CCD is characterized by the rapid loss from a colony of its adult bee population, with dead adult bees usually found at a distance from the colony. At the final stages of collapse, a queen is attended only by a few newly emerged adult bees. Collapsed colonies often have considerable capped brood and food reserves. The phenomenon of CCD was first reported in 2006; however, beekeepers noted unique colony declines consistent with CCD as early as 2004. Various factors such as mites and infectious agents, weather patterns, electromagnetic (cellular antennas) radiation, pesticides, poor nutrition and stress have been postulated as causes. To date, control of CCD has focused on varroa mite control, sanitation and removal of affected hives, treating for opportunistic infections (such as Nosema) and improved nutrition. No effective preventative measures have been developed to date.<br>
That CCD is due to the introduction of a previously unrecognized infectious agent is supported by preliminary evidence that CCD is transmissible through the reuse of equipment from CCD colonies and that such transmission can be broken by irradiation of the equipment before use.<br>
Recently, Israeli acute paralysis virus of bees (IAPV, SEQ ID NO: 6), was strongly correlated with CCD.   Indeed, Table 1 below shows that although other<br>
 <br>
etiological agents of diseases in honeybees were found in CCD colonies, many were also found in apparently healthy, asymptomatic operations. In contrast, IAPV was not only found in 83% of CCD colonies, but was almost completely absent from apparently healthy colonies.<br>
Table I: Analysis of bees tested for pathological candidates in CCD and non-CCD operations<br>
<br>
Agent	CCD (n=30)	Non-CCD (n=21)	Total (n=51)	Positve predictive value (%)<br>
IAPV	25 (83.3%)	1 (4.8%)	26 (51.0%)	96.1<br>
KBV	30 (100%)	16 (76.2%)	46 (90.2%)	65.2<br>
N.apis	27 (90%)	10 (47.6%)	37 (72.5%)	73.0<br>
N. ceranae	30 (100%)	17 (80.9%)	47 (92.1%)	63.8<br>
All four agents	23 (76.7%)	0 (0%)	23 (45.0%)	100<br>
IAPV- Israel Acute Paralysis Virus; KBV- Kashmir Bee Virus; N. apis- Nosema apis;   N. ceranae- Nosema ceranae.<br>
From: Diana L. Cox-Foster et al. (2007) A Metagenomic Survey of Microbes in Honey Bee Colony Collapse Disorder; Science 318: 283-286.<br>
Moreover, it was recently shown that when injected or fed to the bees, IAPV causes paralysis and death in 98% of bees within days, further confirming IAPV as the infective agent in CCD.<br>
Israeli acute paralysis virus (IAPV) has been characterized as a bee-affecting dicistrovirus. Recently, DNA versions of genomic segments of non-retro RNA viruses have been found in their respective host genomes, and the reciprocal exchange of genome sequences between host and virus has been demonstrated (Maori et al. Virology 2007;362:342). These authors showed that the bees who harbored integrated viral sequences were found to be resistant to subsequent viral infection, and a RNAi mechanism of resistance was postulated. Most recently, as shown in Table 1 above, a metagenomic survey has indicated a close association between CCD and IAPV (Cox-Foster et al., Science, 2007;318:283).<br>
It thus follows that prevention of IAPV infection may prevent development of CCD, significantly improving the state of the beekeeping industry and world agriculture.   The United States Department of Agriculture has developed an urgent<br>
 <br>
action plan intended to cover all aspects of bee management to combat CCD and avoid future threats to honeybee management. They seek to maintain bees with resistance to parasites and pathogens and develop new methods of managing parasites and pathogens (see “CCD_actionplan” at the USDA website). However, no specific measures have been recommended, other than improving general sanitation, nutrition and combating opportunistic infections.<br>
Methods for silencing using siRNAs/dsRNA<br>
RNA interference (dsRNA and siRNA) has been shown effective in silencing gene expression in a broad variety of species, including plants, with wide ranging implications for cancer, inherited disease, infectious disease in plants and animals. It was also shown in a variety of organisms that dsRNA or their siRNA derivatives can be used to arrest, retard or even prevent a variety of pathogens, most notably viral diseases (see, for example, WO/2003/004649 to Tenllado et al).<br>
It has been shown in some species that RNAi mediated interference spreads from the initial site of dsRNA delivery, producing interference phenotypes throughout the injected animal. Recently the same spreading effect of dsRNA has been demonstrated in bee larva, as well as detection of SID transmembrane channels considered responsible for endocytic uptake and spreading effect of dsRNA in humans, mouse and C. elegans (Aronstein et al, J. Apic Res and Bee World, 2006;45:20-24).<br>
Application of interference RNA technology for insects that are plant pests and other plant pests has been suggested. Moderate RNAi-type silencing of insect genes by feeding has been demonstrated (Turner et al., Insect Mol Biol 2006;15:383; and Araujo et al., Insect Mol. Biol 2006;36:683). dsRNA absorbance via honey has also been demonstrated (Aronstein et al., J Apiculture Res Bee World 2006;45:20-24).<br>
U.S. Patent No. 6,326,193 refers to the use of recombinant insect viruses such as baculoviruses expressing dsRNA to silence selected insect genes for pest control. PCT application WO 99/32619 describes generally that dsRNA may be used to reduce crop destruction by other plant pathogens or pests such as arachnids, insects, nematodes, protozoans, bacteria, or fungi. PCT patent application WO 2004/005485 describes the use of vectors comprising sequences designed to control plant-parasitic nematodes by RNA interference, and transgenic plants transformed with such vectors.   US patent<br>
 <br>
application 20030180945 generally describes chimeric genes capable of producing antisense or sense RNA equipped with a prokaryotic promoter suitable for expression of the antisense or sense RNA in a particular prokaryotic host.<br>
US Patent Application 20030154508 describes a method for pest control comprising exposing said pest to a compound (dsRNA) which disrupts, within said pest, a cation-amino acid transporter/channel protein.<br>
PCT patent application WO 02/14472 describes methods for inhibiting target gene expression in a sucking insect, by expressing in a cell a nucleic acid construct comprising an inverted repeat and a sense or antisense region having substantial sequence identity to a target gene, wherein the inverted repeat is unrelated to the target gene. US patent application 20030150017 describes the use of RNA molecules homologous or complementary to a nucleotide sequence of a plant pest such as nematodes and insects.<br>
Raemakers et al (PCT Applications WO 2007/080127 and WO 2007/080126) have disclosed transgenic plants expressing RNAi for controlling pest infestation by insects, nematodes, fungus and other plant pests. Among the sequences taught are sequences targeting essential genes of insects, including the honeybee. Waterhouse et al (US Patent Application 2006 0272049) also disclosed transgenic plants expressing dsRNA, and dsRNA directed to essential genes of plant insect pests, for use as insecticides, particularly against sap-sucking insects such as aphids. Boukharov et al. (US Patent Application 2007 0250947) disclosed constructs for expressing dsRNA in transgenic plants for targeting plant parasitic nematodes, specifically the soybean cyst nematode. While expression and processing of dsRNA were demonstrated, no actual inhibition of infestation with the dsRNA was shown.<br>
SUMMARY OF THE INVENTION<br>
According to some aspects of some embodiments, the present invention provides methods and compositions for preventing the spread of insect epidemics, such as Colony Collapse Disorder through the application of RNA interference technology directed to bee infectious organisms and agents, such as IAPV, Acute Bee Paralysis Virus and Kashmir Bee Paralysis Virus.<br>
 <br>
According to an aspect of some embodiments of the present invention there is provided an isolated nucleic acid agent comprising a nucleic acid sequence downregulating expression of a gene product of a bee pathogen.<br>
According to another aspect of some embodiments of the present invention there is provided a nucleic acid agent comprising a nucleic acid sequence complementary to at least 21 nucleotides of a bee pathogen specific RNA and capable of inducing degradation of the bee pathogen specific RNA.<br>
According to another aspect of some embodiments of the invention, there is provided a nucleic acid construct comprising a nucleic acid sequence encoding the isolated nucleic acid downregulating expression of a gene product of a bee pathogen.<br>
According to some embodiments of the invention, the gene product is a mRNA encoding a polypeptide of the bee pathogen.<br>
According to some embodiments of the invention, the agent is selected from the group consisting of a dsRNA, an hnRNA, an antisense RNA and a ribozyme.<br>
According to some embodiments of the invention, the nucleic acid sequence is greater than 15 base pairs in length.<br>
According to some embodiments of the invention, the nucleic acid sequence is greater than 30 base pairs in length.<br>
According to some embodiments of the invention, the nucleic acid sequence is 19 to 25 base pairs in length.<br>
According to some embodiments of the invention, the bee pathogen is selected from the group consisting of a virus, a bacteria, a parasitic protozoan, a fungus and a nematode.<br>
According to some embodiments of the invention, the bee pathogen is a virus.<br>
According to some embodiments of the invention, the virus is Israel Acute Paralysis Virus.<br>
According to some embodiments of the invention, the virus is Kashmir Paralysis Virus.<br>
According to some embodiments of the invention, the virus is Israel Acute Paralysis Virus and said polypeptide of said virus is selected from the group consisting of IAPV polymerase polyprotein (SEQ ID NO: 51) and IAPV structural polyprotein (SEQ ID NO: 52).<br>
 <br>
According to some embodiments of the invention, the viral pathogen is Israel Acute Paralysis Virus and said nucleic acid sequence is as set forth in SEQ ID NO: 6.<br>
According to some embodiments of the invention, the viral pathogen is Israel Acute Paralysis Virus and said nucleic acid sequence is as set forth in SEQ ID NO: 33 and 34.<br>
According to some embodiments of the invention, the virus is Israel Acute Paralysis Virus, and the nucleic acid sequence is a viral nucleic acid sequence detected in honeybee nucleic acid following Israel Acute Paralysis Virus infection.<br>
According to another aspect of some embodiments of the present invention there is provided a bee- ingestible composition comprising the nucleic acid agent comprising a nucleic acid sequence downregulating expression of a gene product of a bee pathogen or a nucleic acid construct comprising the nucleic acid agent.<br>
According to some embodiments of the invention the bee-ingestible composition is in solid form.<br>
According to some embodiments of the invention, the composition is in liquid form.<br>
According to some embodiments of the invention, the composition comprises protein.<br>
According to some embodiments of the invention, the protein is in the form of pollen and/or soy patties.<br>
According to some embodiments of the invention, the liquid is a sucrose solution.<br>
According to some embodiments of the invention, the liquid is a corn syrup solution.<br>
According to some embodiments of the invention the liquid further comprises a carbohydrate or sugar supplement.<br>
According to an aspect of some embodiments of the present invention there is provided a method for increasing the tolerance of a bee to a disease caused by a pathogen comprising feeding the bee an effective amount of the nucleic acid agent comprising a nucleic acid sequence downregulating expression of a gene product of a bee pathogen or a nucleic acid construct comprising the nucleic acid agent, thereby increasing the tolerance of the bee to the pathogen.<br>
 <br>
According to a further aspect of some embodiments of the present invention there is provided a method for increasing the tolerance of a bee colony to a disease caused by a pathogen comprising feeding bees of the colony an effective amount of the nucleic acid agent comprising a nucleic acid sequence downregulating expression of a gene product of a bee pathogen or a nucleic acid construct comprising the nucleic acid agent, thereby increasing the tolerance of the colony to the pathogen.<br>
According to some embodiments of the invention the bee is a honeybee.<br>
According to some embodiments of the invention the honeybee is a forager.<br>
According to some embodiments of the invention the honeybee is a hive bee.<br>
According to some embodiments of the invention the disease is Colony Collapse Disorder.<br>
According to some embodiments of the invention the bee pathogen is Israel Acute Paralysis Virus.<br>
According to some embodiments of the invention the feeding comprises providing a liquid bee-ingestible composition.<br>
According to some embodiments of the invention the feeding comprises providing a solid bee-ingestible composition.<br>
According to an aspect of some embodiments of the present invention there is provided a method of increasing the tolerance of bees to Colony Collapse Disorder (CCD), the method comprising feeding to the honeybee hive an effective amount of double stranded ribonucleic nucleic acid (RNA), said double stranded RNA being homologous to a contiguous sequence of at least 21 nucleotides of Israel Acute Paralysis Virus.<br>
Unless otherwise defined, all technical and/or scientific terms used herein have the same meaning as commonly understood by one of ordinary skill in the art to which the invention pertains. Although methods and materials similar or equivalent to those described herein can be used in the practice or testing of embodiments of the invention, exemplary methods and/or materials are described below. In case of conflict, the patent specification, including definitions, will control. In addition, the materials, methods, and examples are illustrative only and are not intended to be necessarily limiting.<br>
 <br>
BRIEF DESCRIPTION OF THE DRAWINGS<br>
Some embodiments of the invention are herein described, by way of example only, with reference to the accompanying drawings. With specific reference now to the drawings in detail, it is stressed that the particulars shown are by way of example and for purposes of illustrative discussion of embodiments of the invention. In this regard, the description taken with the drawings makes apparent to those skilled in the art how embodiments of the invention may be practiced.<br>
In the drawings:<br>
FIG. 1 is a histogram showing the effect of IAPV titer on bee mortality. 30 or 50 bees were exposed to increasing concentrations of IAPV in a 50% sucrose solution, in increasing doses (900 μl of 0.0001 to 0.1 microgram/microliter virus particles). Vertical striped bars-100 ng/μl; Cross-hatched bars-10 ng/μl; Stippled bars-1.0 ng/μl; Horizontal striped bars-0.1 ng/μl. Controls (checkered bars) received sucrose without added virus. Dead bees in the containers were counted daily, and the total number of dead bees was calculated as a percentage of the initial number of bees introduced into the container at indicated days;<br>
FIGs. 2A-2F are photographs showing the effect feeding IAPV-specific dsRNA on colonies exposed to IAPV infection. FIGs. 2A and 2B show bees of hives exposed to dsRNA following feeding with IAPV-specific dsRNA (1 µg/bee/feeding), 3 days prior to (FIG. 2A) and 8 days following (FIG. 2B) IAPV exposure (0.01 µgram/µl in sucrose solution). FIGs. 2C and 2D show bees of hives exposed to dsRNA following feeding with non-specific dsRNA (1 µg/bee/feeding), 3 days prior to (FIG. 2C) and 8 days following (FIG. 2D) IAPV exposure (0.01 µgram/µl in sucrose solution). FIGs. 2E and 2F show bees of hives exposed only to IAPV infection, without feeding with IAPV-specific dsRNA (FIG. 2E) three days before and 8 days following (FIG. 2F) IAPV exposure (0.01 µgram/µl in sucrose solution). FIGs. 2B, 2D and 2F show the effects of exposure to IAPV, eight days following exposure to IAPV. “Q” indicates queens, and “CB” indicates capped brood. Note the superior survival of the bees exposed to the IAPV-specific dsRNA, as compared with the decline of the unprotected colonies:<br>
FIG. 3 is a graphic representation of bee survival following IAPV infection in the hives as treated in FIGs 2A to FIGs 2F. Filled squares (■) indicate virus exposure only (IAPV only). Empty circles (o) indicate virus exposure and IAVP-specific dsRNA<br>
 <br>
feeding (IAPV dsRNA+IAPV). Empty squares (□) indicate virus exposure and nonspecific dsRNA feeding (GFP dsRNA+IAPV). Filled circles (•) indicates no virus exposure and no dsRNA exposure (no treatment). Data are mean (+ SE) estimated percent of bees alive. Statistical analyses were performed on arcsin square-root transformed proportions using JMP version 7;<br>
FIGs. 4A-4C are photographs of PAGE analysis of RT-PCR of bees from hives, treated as detailed in FIGs. 2A-2F and 3. FIG. 4A represents PCR products with IAPV-specific primers (SEQ ID NOs. 35 and 36). Arrow indicates migration of 180 bp IAPV product. FIG. 4B represents PCR products with actin-specific primers, as an internal positive control (SEQ ID Nos. 37 and 38). Arrow indicates migration of 500 bp actin product. FIG. 4C represents PCR performed without reverse transcriptase. Absence of products indicates the absence of DNA in the template. Total RNA was extracted (8 days following inoculation with IAPV) from bee samples as follows: Lanes 1 and 2-IAPV-dsRNA treatment followed by IAPV inoculation; Lane 3- GFP-dsRNA treatment followed by IAPV inoculation; Lane 4- no dsRNA followed by IAPV inoculation; Lane 5- no dsRNA, no virus inoculated; Lane 6- negative control without template RNA. Lane M is molecular weight markers;<br>
FIG. 5 is a histogram showing reduction in virus titer following IAVP-dsRNA treatment. Relative titers of virus (as determined by real-time PCT) were determined four days following IAPV inoculation in untreated bees (IAPV- only), sham (GFP) dsRNA treated bees (IAPV+dsRNA-GFP) and bees treated with IAVP-dsRNA (IAVP+dsRNA-IAVP). Relative titers were calibrated against virus titers in non-inoculated bees;<br>
FIG. 6 is a photograph of a PAGE analysis showing the accumulation of dsRNA in bees fed with IAPV-specific dsRNA. Total bee RNA was extracted from dsRNA-fed bees as described herein, treated as indicated with RNase A, RNase III or DNA, separated on PAGE and stained for detection of prominent species. Lane 1: total RNA after digestion with RNase A. Lane 2: total RNA after digestion with DNase I. Lane 3: total RNA after digestion with RNase A + RNase III. Lane 4: untreated extract of total RNA. M is molecular weight markers. Note the presence of dsRNA (RNase A and DNase resistant, RNase III sensitive) band in lanes 1, 2, and 4;<br>
 <br>
FIG. 7 is a photograph of a PAGE analysis showing the accumulation of IAPV-specific siRNA in bees fed with IAPV-specific dsRNA. Total bee RNA was extracted from dsRNA-fed bees as described herein, separated on PAGE and blotted onto a matrix for detection with a probe representing a segment of a IAPV structural protein. Lane 1: IAPV-dsRNA digested with RNaseIII, resulting in 18- to 26-bp fragments. Lane 2: synthetic primers for IAPV as size markers for 20- and 26-bp fragments. Lane 3: electrophoretic pattern of total RNA extracted from bees fed on IAPVdsRNA. Lane 4: blot of total RNA extracted from bees fed on GFP-dsRNA. Lanes 5 &amp; 6: synthetic IAPV-dsRNA and GFP-dsRNA (respectively). The ca. 400-bp dsRNA was excluded from the siRNA gel. Lane 7: total RNA from untreated bees. Note the presence of IAPV-specific siRNA (21, 22, and 25 bp long) in bees fed on IAPV-dsRNA. Arrows indicate size in bp.;<br>
FIG. 8 is a histogram showing increased numbers of returning foragers in IAPV-dsRNA treated hives in large scale field trials. Separate hives received either IAPV alone (white bars), IAPV dsRNA + IAPV (lined bars) or no treatment (controls, black bars). Returning foragers were monitored on several occasions at several times of the day over a period of six weeks. Data is presented from one week following exposure to virus to 5 weeks after exposure. Note the progressively significant increase in numbers of returning foragers among the IAPV dsRNA- treated hives as compared with the IAPV-only hives;<br>
FIG. 9 is a histogram showing the effect of IAPV dsRNA on numbers of bees in the hive following IAPV infection in large scale field trails. Separate hives received either IAPV alone (white bars), IAPV dsRNA + IAPV (diagonal shaded bars) or no treatment (controls, black bars). Numbers of bees per hive was estimated at the midpoint (May 13) and at the end of the experiment (June 10). Note the significantly greater numbers of bees in the IAPV dsRNA- treated hives at the conclusion of the trial;<br>
FIG. 10 is a histogram showing the effect of IAPV dsRNA on honey production in hives following IAPV infection in large scale field trails. Separate hives received either IAPV alone (white bar), IAPV dsRNA + IAPV (diagonal shaded bars) or no treatment (controls, black bar). Amount of honey (in Kg) per hive was weighed with a portable scale at the end of the experiment 6 weeks following IAPV infection. Note the<br>
 <br>
significantly greater honey production in hives treated with IAPV-dsRNA+IAPV, as compared to untreated IAPV-infected and uninfected control hives; FIGs. 11A-11B are photographs of a PAGE analysis and Southern blot showing the accumulation of IAPV-specific siRNA in bees fed with IAPV-specific dsRNA in a large scale field trial. FIG. 11A illustrates the presence of IAPV-specific siRNA at 7 days and the end of the experiment. Lanes 1-6, Day 7: Lane 1- positive control IAPV-specific siRNA; lane 2- blank negative control; lane 3- untreated control bees; lane 4-bees fed with IAPV-specific dsRNA but no virus challenge; lane 5-bees challenged with IAPV, no dsRNA; lane 6- IAPV-specific dsRNA plus IAPV infection. Lanes 7-10, end of Experiment (Day 42): Lane 7- untreated control bees; lane 8- bees fed with IAPV-specific dsRNA but no virus challenge; lane 9- bees challenged with IAPV, no dsRNA; lane 10- IAPV-specific dsRNA plus IAPV infection.<br>
FIG. 11B illustrates the presence of IAPV-specific siRNA at the start (0 days) and the end of the experiment. Lanes 1-4, Day 0: Lane 1- untreated control bees; lane 2- bees fed with IAPV-specific dsRNA but no virus challenge; lane 3-bees challenged with IAPV, no dsRNA; lane 4- IAPV-specific dsRNA plus IAPV infection. Lane 5-blank control. Lanes 6-9, end of Experiment (Day 42): Lane 6- untreated control bees; lane 7- bees fed with IAPV-specific dsRNA but no virus challenge; lane 8- bees challenged with IAPV, no dsRNA; lane 9- IAPV-specific dsRNA plus IAPV infection. Arrow indicates 21 bp RNAi fragment. Note the amplification of IAPV-specific RNAi in treated bees exposed to IAPV infection for a period of time (FIG. 11A, lanes 6 and 10; FIG. 11B, lane 9);<br>
FIG. 12 is a schematic diagram showing the phylogenetic relationship between bee-viruses of the Picornavirus Superfamily;<br>
FIG. 13 is a diagrammatic illustration of the sequences comprising the multiple bee-virus resistance nucleic acid construct SEQ ID NO: 24.<br>
DESCRIPTION OF EMBODIMENTS OF THE INVENTION<br>
The present invention, in some embodiments thereof, relates to methods and compositions for reducing the susceptibility of bees to pathogenic organisms and, more particularly, but not exclusively, to methods for increasing the tolerance to viral diseases, such as Colony Collapse Disorder, by feeding viral-specific dsRNA.<br>
 <br>
Before explaining at least one embodiment of the invention in detail, it is to be understood that the invention is not necessarily limited in its application to the details set forth in the following description or exemplified by the Examples. The invention is capable of other embodiments or of being practiced or carried out in various ways.<br>
While reducing the present invention to practice, the inventors have shown that ingestion by a bee of compositions containing one or more dsRNA molecules, wherein at least one segment of the dsRNA molecule corresponds to a substantially identical segment of RNA produced by a bee pathogen, will result in reduced incidence and severity of infection, and greatly enhanced survival of the bees and the colony overall. These results indicate that a polynucleotide molecule, either DNA or RNA, derived from a bee pathogen sequence can be used to design a nucleic acid agent or nucleic acid construct according to the methods of the present invention to produce one or more RNA sequences that can form into a dsRNA molecule available for ingestion by bees when provided by feeding. While reducing to practice, it was shown that bee colonies exposed to IAPV- specific dsRNA in their feed endured IAPV infection with greater survival (see FIG. 3) and lower incidence of infected bees than untreated colonies (see FIGs. 2E, 2F and 3, 4A-4C and 5). In colonies treated with a non-specific dsRNA mortality and incidence of infection was similar to that in untreated colonies (see FIGs. 2C, 2D, 3-5).<br>
Thus, according to one embodiment of the present invention there is provided a method for increasing the tolerance of a bee to a disease caused by a pathogen comprising feeding the bee an effective amount of an isolated nucleic acid agent comprising a nucleic acid sequence downregulating expression of a polypeptide of a bee pathogen, or a nucleic acid construct comprising the nucleic acid sequence, thereby increasing the tolerance of the bee to the pathogen.<br>
As used herein, the term "bee" is defined as any of several winged, hairy-bodied, usually stinging insects of the superfamily Apoidea in the order Hymenoptera, including both solitary and social species and characterized by sucking and chewing mouthparts for gathering nectar and pollen. Exemplary bee species include, but are not limited to Apis, Bombus, Trigona, Osmia and the like. In one embodiment, bees include, but are not limited to bumblebees (Bombus terrestris) and honeybees (Apis mellifera).<br>
 <br>
As used herein, the term "colony" is defined as a population of dozens to typically several tens of thousand honeybees that cooperate in nest building, food collection, and brood rearing. A colony normally has a single queen, the remainder of the bees being either "workers" (females) or "drones" (males). The social structure of the colony is maintained by the queen and workers and depends on an effective system of communication. Division of labor within the worker caste primarily depends on the age of the bee but varies with the needs of the colony. Reproduction and colony strength depend on the queen, the quantity of food stores, and the size of the worker force. Honeybees can also be subdivided into the categories of "hive bees", usually for the first part of a workers lifetime, during which the "hive bee" performs tasks within the hive, and "forager bee", during the latter part of the bee's lifetime, during which the "forager" locates and collects pollen and nectar from outside the hive, and brings the nectar or pollen into the hive for consumption and storage.<br>
As used herein, the term "tolerance" is defined as the ability of a bee or bee colony to resist infestation by and/or proliferation of a pathogen, including, but not limited to, degree of infection, severity of symptoms, infectivity to other individuals (contagion), and the like. Tolerance can be assessed, for example, by monitoring infectivity, presence of symptoms or time course of a disease in a population following a challenge with the pathogen.<br>
As used herein, the term "pathogen" is defined as a nucleic acid-containing agent capable of proliferation within the bee and/or bee colony, the pathogen causing disease in bees or bee colonies, especially, but not exclusively, a virus, a bacteria and a fungus. A bee or bee colony pathogenic agent can be an intracellular or extra-cellular parasite. According to one embodiment of the invention, the pathogen is a "bee pathogen", causing or facilitating a bee or bee colony disease, such as Colony Collapse Disorder, Sacbrood virus disease, Deformed Wing Disease, Cloudy Wing Disease, Chronic Paralysis, Nosemosis, American Foul Brood and the like.<br>
As used herein, the terms "bee disease" or "bee colony disease" are defined as undesirable changes in the behavior, physiology, morphology, reproductive fitness, economic value, honey production, pollination capability, resistance to infection and/or infestation of a bee, a population of bees and/or a bee colony, directly or indirectly resulting from contact with a bee or bee colony pathogenic agent.<br>
 <br>
A non-limiting list of exemplary disease-causing pathogens, and diseases of bees<br>
and bee colonies associated with the pathogenic agents, suitable for treatment according<br>
to some embodiments of the methods and compositions of the present invention is<br>
found in Table II below. The complete genomes of several known isolates of IAPV and<br>
information on possible phylogenic relationships between strains that can be similarly<br>
targeted with the methods and compositions of the present invention are provided in<br>
Palacios et al. 2008 (published online ahead of print on 23 April 2008, Journal of<br>
Virology)<br>
Table II: Bee and Bee Colony Pathogens<br>
Parasitic Organism	Genes<br>
Acute           bee paralysis virus	Acute bee paralysis virus, complete genome. Accession  NC_002548 (seq id no: 8)<br>
Israel         acute paralysis virus	Accession: NC_009025, israel acute paralysis virus of bees, complete genome (seq id no: 16)<br>
Deformed   wing virus	Deformed wing virus, complete genome. Accession  NC_004830 (seq id no: 10)<br>
Kashmir        bee virus	Accession: AY275710, kashmir bee virus, complete genome (seq id no: 9)<br>
Black queen cell virus	Black queen cell virus strain poland-6 non-structural polyprotein<br>
and structural polyprotein genes, complete cds. Accession : EF517521 (seq id no: 20)<br>
Chronic paralysis virus	Chronic bee paralysis virus rna 2, complete sequence. Accession:  NC_010712 (seq id no: 23)<br>
Cloudy       wing virus	Cloudy wing virus rna polymerase (pol) gene, partial cds. Accession  AF034543 (seq id no: 7)<br>
Paenibacillus larvae (American Foul Brood)	Accession: NZ_AARF01000646, whole genome (shotgun) sequenced. (seq id no: 11)<br>
Melissococcus pluton (European Foul Brood)	Accession: EF666055 Melissococcus plutonius superoxide dismutase (soda) gene (seq id no: 21)<br>
Ascophaera apis (Chalkbrood)	No genomic data<br>
Nosema apis,	1)	Accession DQ996230 (seq id no: 15), Nosema apis RNA polymerase II largest<br>
subunit<br>
2)	Accesions EU545140 (seq id no: 22), EF584425 (seq id no: 19), EF584423 (seq id<br>
no: 18),  EF584418 (seq id no: 17) all are 16S ribosomal RNA gene<br>
Nosema cerana	EF091883 (seq id no: 12), EF091884 (seq id no: 13), and EF091885 (seq id no: 14) are accessions of 5S ribosomal RNA gene, intergenic spacer, and small subunit ribosomal RNA gene.<br>
While reducing the present invention to practice, the inventors have shown that providing a IAPV-specific dsRNA in the feed of bees exposed to IAPV dramatically reduced the incidence and levels of IAPV sequences detected in the bees, after 4 and 8 days (FIGs. 4A – 4C and 5). Thus, in some embodiments of the present invention, the<br>
 <br>
methods and compositions are useful for downregulating expression of a polypeptide of a bee or bee colony pathogenic organism.<br>
As used herein, the term “downregulating expression" is defined as causing, directly or indirectly, reduction in the transcription of a desired gene, reduction in the amount, stability or translatability of transcription products (e.g. RNA) of said gene, reduction in translation of the polypeptide(s) encoded by the desired gene and/or reduction in the amount, stability, or alteration of biochemical function of the polypeptides encoded by the desired gene, so as to reduce the amount or function of the gene products. As used herein, "downregulating expression" also relates to reduction in amount, stability or translatability of bee pathogen RNA molecules in cells of a bee, where the bee pathogen genome is a single stranded RNA molecule, as in case of a single-stranded RNA virus. Downregulating expression of a gene or other bee pathogen RNA can be monitored, for example, by direct detection of gene transcripts (for example, by PCR), by detection of polypeptide(s) encoded by the gene or bee pathogen RNA (for example, by Western blot or immunoprecipitation), by detection of biological activity of polypeptides encode by the gene (for example, catalytic activity, ligand binding, and the like), or by monitoring changes in a cell or organism resulting from reduction in expression of a desired gene or bee pathogen RNA (for example, reduced proliferation of a pathogen, reduced virulence of a pathogen, reduced motility of a cell, reduced response of a cell or organism to stimulus, etc). As used herein, the downregulation can be transient, for example, for the duration of the presence of a downregulating agent, or permanent, resulting in reduction of gene expression or bee pathogen RNA for the lifetime of the organism and/or its future generations.<br>
Downregulation of bee pathogen polypetides can be effected on the genomic and/or the transcript level using a variety of molecules which interfere with transcription and/or translation (e.g., RNA silencing agents, Ribozyme, DNAzyme and antisense). Treatment and prevention of viral infections with dsRNA has been disclosed by WO/2003/004649 to Tenllado et al. Use of dsRNA in insects is disclosed in US Patent Application 2007 0250947, US Patent Application 2006 0272049, PCT Applications WO 2007/080127 and WO 2007/080126, US patent application 20030150017,   PCT   patent   application   WO   02/14472,   US   Patent   Application<br>
 <br>
20030154508, PCT patent application WO 2004/005485, PCT application WO 99/32619 and U.S. Patent No. 6,326,193.<br>
Following is a list of agents capable of downregulating expression level and/or activity of bee pathogen polypeptides.<br>
Downregulation of bee pathogen polypeptides can be achieved by RNA silencing. As used herein, the phrase "RNA silencing" refers to a group of regulatory mechanisms [e.g. RNA interference (RNAi), transcriptional gene silencing (TGS), post-transcriptional gene silencing (PTGS), quelling, co-suppression, and translational repression] mediated by RNA molecules which result in the inhibition or "silencing" of the expression of a corresponding protein-coding gene or bee pathogen RNA sequence. RNA silencing has been observed in many types of organisms, including plants, animals, and fungi.<br>
As used herein, the term "RNA silencing agent" refers to an RNA which is capable of inhibiting or "silencing" the expression of a target gene. In certain embodiments, the RNA silencing agent is capable of preventing complete processing (e.g, the full translation and/or expression) of an mRNA molecule through a post-transcriptional silencing mechanism. RNA silencing agents include noncoding RNA molecules, for example RNA duplexes comprising paired strands, as well as precursor RNAs from which such small non-coding RNAs can be generated. Exemplary RNA silencing agents include dsRNAs such as siRNAs, miRNAs and shRNAs. In one embodiment, the RNA silencing agent is capable of inducing RNA interference. In another embodiment, the RNA silencing agent is capable of mediating translational repression.<br>
RNA interference refers to the process of sequence-specific post-transcriptional gene silencing in animals mediated by short interfering RNAs (siRNAs). The corresponding process in plants is commonly referred to as post-transcriptional gene silencing or RNA silencing and is also referred to as quelling in fungi. The process of post-transcriptional gene silencing is thought to be an evolutionarily-conserved cellular defense mechanism used to prevent the expression of foreign genes and is commonly shared by diverse flora and phyla. Such protection from foreign gene expression may have evolved in response to the production of double-stranded RNAs (dsRNAs) derived from viral infection or from the random integration of transposon elements into a host<br>
 <br>
genome via a cellular response that specifically destroys homologous single-stranded RNA or viral genomic RNA.<br>
The presence of long dsRNAs in cells stimulates the activity of a ribonuclease III enzyme referred to as dicer. Dicer is involved in the processing of the dsRNA into short pieces of dsRNA known as short interfering RNAs (siRNAs). Short interfering RNAs derived from dicer activity are typically about 21 to about 23 nucleotides in length and comprise about 19 base pair duplexes. The RNAi response also features an endonuclease complex, commonly referred to as an RNA-induced silencing complex (RISC), which mediates cleavage of single-stranded RNA having sequence complementary to the antisense strand of the siRNA duplex. Cleavage of the target RNA takes place in the middle of the region complementary to the antisense strand of the siRNA duplex.<br>
Accordingly, the present invention contemplates use of dsRNA to downregulate protein expression from mRNA.<br>
According to one embodiment, the dsRNA is greater than 30 bp. The use of long dsRNAs can provide numerous advantages in that the cell can select the optimal silencing sequence alleviating the need to test numerous siRNAs; long dsRNAs will allow for silencing libraries to have less complexity than would be necessary for siRNAs; and, perhaps most importantly, long dsRNA could prevent viral escape mutations when used as therapeutics.<br>
Various studies demonstrate that long dsRNAs can be used to silence gene expression without inducing the stress response or causing significant off-target effects -see for example [Strat et al., Nucleic Acids Research, 2006, Vol. 34, No. 13 3803–3810; Bhargava A et al. Brain Res. Protoc. 2004;13:115–125; Diallo M., et al., Oligonucleotides. 2003;13:381–392; Paddison P.J., et al., Proc. Natl Acad. Sci. USA. 2002;99:1443–1448; Tran N., et al., FEBS Lett. 2004;573:127–134].<br>
In one embodiment of the present invention, the dsRNA is greater than 30 base-pairs, and is as set forth in SEQ ID NOs: 24, 33 and 34.<br>
Another method of downregulating bee pathogen proteins is by introduction of small inhibitory RNAs (siRNAs).<br>
The term "siRNA" refers to small inhibitory RNA duplexes (generally between 18-30 basepairs, between 19 and 25 basepairs) that induce the RNA interference<br>
 <br>
(RNAi) pathway. Typically, siRNAs are chemically synthesized as 21mers with a central 19 bp duplex region and symmetric 2-base 3'-overhangs on the termini, although it has been recently described that chemically synthesized RNA duplexes of 25-30 base length can have as much as a 100-fold increase in potency compared with 21mers at the same location. The observed increased potency obtained using longer RNAs in triggering RNAi is theorized to result from providing Dicer with a substrate (27mer) instead of a product (21mer) and that this improves the rate or efficiency of entry of the siRNA duplex into RISC.<br>
It has been found that position of the 3'-overhang influences potency of an siRNA and asymmetric duplexes having a 3'-overhang on the antisense strand are generally more potent than those with the 3'-overhang on the sense strand (Rose et al., 2005). This can be attributed to asymmetrical strand loading into RISC, as the opposite efficacy patterns are observed when targeting the antisense transcript.<br>
The strands of a double-stranded interfering RNA (e.g., an siRNA) may be connected to form a hairpin or stem-loop structure (e.g., an shRNA). Thus, as mentioned the RNA silencing agent of the present invention may also be a short hairpin RNA (shRNA).<br>
The term "shRNA", as used herein, refers to an RNA agent having a stem-loop structure, comprising a first and second region of complementary sequence, the degree of complementarity and orientation of the regions being sufficient such that base pairing occurs between the regions, the first and second regions being joined by a loop region, the loop resulting from a lack of base pairing between nucleotides (or nucleotide analogs) within the loop region. The number of nucleotides in the loop is a number between and including 3 to 23, or 5 to 15, or 7 to 13, or 4 to 9, or 9 to 11. Some of the nucleotides in the loop can be involved in base-pair interactions with other nucleotides in the loop. Examples of oligonucleotide sequences that can be used to form the loop include 5'-UUCAAGAGA-3' (Brummelkamp, T. R. et al. (2002) Science 296: 550) and 5'-UUUGUGUAG-3' (Castanotto, D. et al. (2002) RNA 8:1454). It will be recognized by one of skill in the art that the resulting single chain oligonucleotide forms a stem-loop or hairpin structure comprising a double-stranded region capable of interacting with the RNAi machinery.<br>
 <br>
According to another embodiment the RNA silencing agent may be a miRNA. miRNAs are small RNAs made from genes encoding primary transcripts of various sizes. They have been identified in both animals and plants. The primary transcript (termed the "pri-miRNA") is processed through various nucleolytic steps to a shorter precursor miRNA, or "pre-miRNA." The pre-miRNA is present in a folded form so that the final (mature) miRNA is present in a duplex, the two strands being referred to as the miRNA (the strand that will eventually basepair with the target) The pre-miRNA is a substrate for a form of dicer that removes the miRNA duplex from the precursor, after which, similarly to siRNAs, the duplex can be taken into the RISC complex. It has been demonstrated that miRNAs can be transgenically expressed and be effective through expression of a precursor form, rather than the entire primary form (Parizotto et al. (2004) Genes &amp; Development 18:2237-2242 and Guo et al. (2005) Plant Cell 17:1376-1386).<br>
Unlike, siRNAs, miRNAs bind to transcript sequences with only partial complementarity (Zeng et al., 2002, Molec. Cell 9:1327-1333) and repress translation without affecting steady-state RNA levels (Lee et al., 1993, Cell 75:843-854; Wightman et al., 1993, Cell 75:855-862). Both miRNAs and siRNAs are processed by Dicer and associate with components of the RNA-induced silencing complex (Hutvagner et al., 2001, Science 293:834-838; Grishok et al., 2001, Cell 106: 23-34; Ketting et al., 2001, Genes Dev. 15:2654-2659; Williams et al., 2002, Proc. Natl. Acad. Sci. USA 99:6889-6894; Hammond et al., 2001, Science 293:1146-1150; Mourlatos et al., 2002, Genes Dev. 16:720-728). A recent report (Hutvagner et al., 2002, Sciencexpress 297:2056-2060) hypothesizes that gene regulation through the miRNA pathway versus the siRNA pathway is determined solely by the degree of complementarity to the target transcript. It is speculated that siRNAs with only partial identity to the mRNA target will function in translational repression, similar to an miRNA, rather than triggering RNA degradation.<br>
According to one embodiment of the present invention, the nucleic acid agent is capable of causing cleavage and/or degradation of a bee pathogen target polynucleotide sequence. As used herein, the phrases "target" or "target polynucleotide sequence" refer to any sequence present in a bee cell or in a bee, whether naturally occurring sequence or a heterologous sequence present due to an intracellular or extracellular pathogenic<br>
 <br>
infection or a disease, which bee pathogen polynucleotide sequence has a function that is desired to be reduced or inhibited. The bee pathogen target sequence may be a coding sequence, that is, it is translated to express a protein or a functional fragment thereof. Alternatively, the target sequence may be non-coding, but may have a regulatory function. One target polynucleotide sequence is a bee pathogenic virus polynucleotide sequence necessary for replication and/or pathogenesis of the virus in an infected bee cell. Another embodiment of a bee pathogen target polynucleotide sequence is a non-expressed regulatory sequence of a virus-induced disease, which sequence is required for the maintenance of the virus in the bee cell, for example, a polynucleotide sequence of an intracellular or extracellular pathogen necessary for replication and/or pathogenesis of that pathogen in an infected bee. Yet another embodiment of a bee pathogenic target sequence is any sequence to which the nucleic acid agent, or sequences derived therefrom, is capable of binding, which binding results in cleavage and/or degradation ("silencing")of a bee pathogen polynucleotide. The term "gene" is intended to include any target sequence intended to be "silenced", whether or not transcribed and/or translated, including regulatory sequences, such as promoters, enhancers and other non-coding sequences.<br>
In one embodiment of the present invention, synthesis of RNA silencing agents<br>
suitable for use with the present invention can be effected as follows. First, the bee<br>
pathogen polypeptide mRNA or other target sequence is scanned downstream of the<br>
AUG start codon for AA dinucleotide sequences. Occurrence of each AA and the 3’<br>
adjacent 19 nucleotides is recorded as potential siRNA target sites. Preferably, siRNA<br>
target sites are selected from the open reading frame, as untranslated regions (UTRs) are<br>
richer in regulatory protein binding sites. UTR-binding proteins and/or translation<br>
initiation complexes may interfere with binding of the siRNA endonuclease complex<br>
[Tuschl ChemBiochem. 2:239-245]. It will be appreciated though, that siRNAs directed<br>
at untranslated regions may also be effective, as demonstrated for GAPDH wherein<br>
siRNA directed at the 5’ UTR mediated about 90 % decrease in cellular GAPDH<br>
mRNA	and	completely	abolished	protein	level<br>
(www.ambion.com/techlib/tn/91/912.html).<br>
Second, potential target sites are compared to an appropriate genomic database (e.g., human, mouse, rat etc.) using any sequence alignment software, such as the<br>
 <br>
BLAST software available from the NCBI server (www.ncbi.nlm.nih.gov/BLAST/). Putative target sites which exhibit significant homology to other coding sequences are filtered out.<br>
Qualifying target sequences are selected as template for siRNA synthesis. Preferred sequences are those including low G/C content as these have proven to be more effective in mediating gene silencing as compared to those with G/C content higher than 55 %. Several target sites are preferably selected along the length of the target gene or sequence for evaluation. For better evaluation of the selected siRNAs, a negative control is preferably used in conjunction. Negative control siRNA preferably include the same nucleotide composition as the siRNAs but lack significant homology to the genome. Thus, a scrambled nucleotide sequence of the siRNA is preferably used, provided it does not display any significant homology to any other gene or bee pathogen target sequence.<br>
For example, a suitable bee pathogen siRNA can be an IAPV-specific siRNA corresponding to IAPV sequences SEQ ID NOs: 33 and 34. Additional suitable bee pathogen siRNAs can be designed according to sequences from any bee pathogens, for example, the sequences detailed in Table II, including, but not limited to Acute Bee Paralysis Virus (for example, SEQ ID NOs: 32430-41886), Deformed Wing Virus (for example, SEQ ID NOs: 9533-19652), Kashmir Bee Virus (for example, SEQ ID NOs: 42281-51771), Black Queen Cell Virus (for example, SEQ ID NOs: 19653-27934), Chronic Paralysis Virus (for example, SEQ ID NOs: 27935-30219), Cloudy Wing Virus (for example, SEQ ID NOs: 30220-30613), Paenibacillus larvae (for example, SEQ ID NOs: 30614-32007), Melissococcus pluton (for example, SEQ ID NOs: 32008-32429), Nosema apis (for example, SEQ ID NOs: 53774-56822) and Nosema cerana (for example, SEQ ID NOs: 51772-53773). Multiple bee-pathogen sequences can be designed to include sequences suitable for producing siRNAs effective against more than one bee pathogen, such as the multiple bee-virus dsRNA described in detail in Example IV herein (SEQ ID NO: 24). Such multiple bee-pathogen dsRNA can be of the long or short variety, and may include sequences corresponding to homologous sequences within a class of bee pathogens (multiple bee-virus sequences, for example), or sequences corresponding to diverse classes of pathogens (e.g. viral + bacterial +<br>
 <br>
fungal sequences, etc). Further, multiple sequences can be designed to include two or more dsRNA sequences of the same bee-pathogen.<br>
According to yet another embodiment of the present invention, synthesis of RNA silencing agents suitable for use with the present invention can be effected according to bee pathogen target sequences known to integrate into the host genome, target sequences suspected associated with resistance to a bee pathogen infection, target sequences representing intergenic regions of the bee pathogen genome and pathogen-specific sequences shown to be critical for pathogen growth and/or replication. It will be appreciated that, in a further embodiment of the present invention, nucleic acid agents targeted to sequences having a conserved homology between different strains of the bee pathogen, or even between diverse bee pathogens, once such sequences are identified, can be effective against more than one strain of the bee pathogen, or even against different bee pathogens.<br>
It will be appreciated that the RNA silencing agent of the present invention need not be limited to those molecules containing only RNA, but further encompasses chemically-modified nucleotides and non-nucleotides.<br>
In some embodiments, the RNA silencing agent provided herein can be functionally associated with a cell-penetrating peptide. As used herein, a "cell-penetrating peptide" is a peptide that comprises a short (about 12-30 residues) amino acid sequence or functional motif that confers the energy-independent (i.e., non-endocytotic) translocation properties associated with transport of the membrane-permeable complex across the plasma and/or nuclear membranes of a cell. The cell-penetrating peptide used in the membrane-permeable complex of the present invention preferably comprises at least one non-functional cysteine residue, which is either free or derivatized to form a disulfide link with a double-stranded ribonucleic acid that has been modified for such linkage. Representative amino acid motifs conferring such properties are listed in U.S. Pat. No. 6,348,185, the contents of which are expressly incorporated herein by reference. The cell-penetrating peptides of the present invention preferably include, but are not limited to, penetratin, transportan, pIsl, TAT(48-60), pVEC, MTS, and MAP.<br>
Another agent capable of downregulating a bee pathogen polypeptide is a DNAzyme molecule capable of specifically cleaving an mRNA transcript or DNA<br>
 <br>
sequence of the bee pathogen polypeptide. DNAzymes are single-stranded polynucleotides which are capable of cleaving both single and double stranded target sequences (Breaker, R.R. and Joyce, G. Chemistry and Biology 1995;2:655; Santoro, S.W. &amp; Joyce, G.F. Proc. Natl, Acad. Sci. USA 1997;943:4262) A general model (the "10-23" model) for the DNAzyme has been proposed. "10-23” DNAzymes have a catalytic domain of 15 deoxyribonucleotides, flanked by two substrate-recognition domains of seven to nine deoxyribonucleotides each. This type of DNAzyme can effectively cleave its substrate RNA at purine:pyrimidine junctions (Santoro, S.W. &amp; Joyce, G.F. Proc. Natl, Acad. Sci. USA 199; for rev of DNAzymes see Khachigian, LM [Curr Opin Mol Ther 4:119-21 (2002)].<br>
Examples of construction and amplification of synthetic, engineered DNAzymes recognizing single and double-stranded target cleavage sites have been disclosed in U.S. Pat. No. 6,326,174 to Joyce et al. DNAzymes of similar design directed against the human Urokinase receptor were recently observed to inhibit Urokinase receptor expression, and successfully inhibit colon cancer cell metastasis in vivo (Itoh et al, 20002, Abstract 409, Ann Meeting Am Soc Gen Ther www.asgt.org). In another application, DNAzymes complementary to bcr-ab1 oncogenes were successful in inhibiting the oncogenes expression in leukemia cells, and lessening relapse rates in autologous bone marrow transplant in cases of CML and ALL.<br>
Downregulation of bee pathogen polypeptides or cleavage of bee pathogen RNA can also be effected by using an antisense polynucleotide capable of specifically hybridizing with an mRNA transcript encoding the bee pathogen polypeptide or a bee pathogen RNA target sequence.<br>
Design of antisense molecules which can be used to efficiently downregulate a bee pathogen polypeptide must be effected while considering two aspects important to the antisense approach. The first aspect is delivery of the oligonucleotide into the cytoplasm of the appropriate cells, while the second aspect is design of an oligonucleotide which specifically binds the designated mRNA or RNA target sequence within cells in a way which inhibits translation thereof.<br>
The prior art teaches of a number of delivery strategies which can be used to efficiently deliver oligonucleotides into a wide variety of cell types [see, for example, Luft J Mol Med 76: 75-6 (1998); Kronenwett et al. Blood 91: 852-62 (1998); Rajur et<br>
 <br>
al. Bioconjug Chem 8: 935-40 (1997); Lavigne et al. Biochem Biophys Res Commun 237: 566-71 (1997) and Aoki et al. (1997) Biochem Biophys Res Commun 231: 540-5 (1997)].<br>
In addition, algorithms for identifying those sequences with the highest predicted binding affinity for their target mRNA based on a thermodynamic cycle that accounts for the energetics of structural alterations in both the target mRNA and the oligonucleotide are also available [see, for example, Walton et al. Biotechnol Bioeng 65: 1-9 (1999)].<br>
Such algorithms have been successfully used to implement an antisense approach in cells. For example, the algorithm developed by Walton et al. enabled scientists to successfully design antisense oligonucleotides for rabbit beta-globin (RBG) and mouse tumor necrosis factor-alpha (TNF alpha) transcripts. The same research group has more recently reported that the antisense activity of rationally selected oligonucleotides against three model target mRNAs (human lactate dehydrogenase A and B and rat gp130) in cell culture as evaluated by a kinetic PCR technique proved effective in almost all cases, including tests against three different targets in two cell types with phosphodiester and phosphorothioate oligonucleotide chemistries.<br>
In addition, several approaches for designing and predicting efficiency of specific oligonucleotides using an in vitro system were also published (Matveeva et al., Nature Biotechnology 16: 1374 - 1375 (1998)].<br>
For example, a suitable antisense oligonucleotide targeted against the IAPV mRNA would be of the sequences as set forth in SEQ ID NOs: 51 and 52 (IAPV polyproteins).<br>
Several clinical trials have demonstrated safety, feasibility and activity of antisense oligonucleotides. For example, antisense oligonucleotides suitable for the treatment of cancer have been successfully used [Holmund et al., Curr Opin Mol Ther 1:372-85 (1999)], while treatment of hematological malignancies via antisense oligonucleotides targeting c-myb gene, p53 and Bcl-2 had entered clinical trials and had been shown to be tolerated by patients [Gerwitz Curr Opin Mol Ther 1:297-306 (1999)]. Antisense oligonucleotides targeted to nervous system proteins have been used effectively in honeybees (Fiala et al, J. Neuroscience 1999;19:10125-34).<br>
 <br>
Thus, the current consensus is that recent developments in the field of antisense technology which, as described above, have led to the generation of highly accurate antisense design algorithms and a wide variety of oligonucleotide delivery systems, enable an ordinarily skilled artisan to design and implement antisense approaches suitable for downregulating expression of known sequences without having to resort to undue trial and error experimentation.<br>
Another agent capable of downregulating a bee pathogen polypeptide is a ribozyme molecule capable of specifically cleaving an mRNA transcript encoding a bee pathogen polypeptide. Ribozymes are being increasingly used for the sequence-specific inhibition of gene expression by the cleavage of mRNAs encoding proteins of interest [Welch et al., Curr Opin Biotechnol. 9:486-96 (1998)]. The possibility of designing ribozymes to cleave any specific target RNA, including viral RNA, has rendered them valuable tools in both basic research and therapeutic applications. In the area of therapeutics, ribozymes have been exploited to target viral RNAs in infectious diseases, dominant oncogenes in cancers and specific somatic mutations in genetic disorders [Welch et al., Clin Diagn Virol. 10:163-71 (1998)]. Most notably, several ribozyme gene therapy protocols for HIV patients are already in Phase 1 trials. More recently, ribozymes have been used for transgenic animal research, gene target validation and pathway elucidation. Several ribozymes are in various stages of clinical trials. ANGIOZYME was the first chemically synthesized ribozyme to be studied in human clinical trials. ANGIOZYME specifically inhibits formation of the VEGF-r (Vascular Endothelial Growth Factor receptor), a key component in the angiogenesis pathway. Ribozyme Pharmaceuticals, Inc., as well as other firms have demonstrated the importance of anti-angiogenesis therapeutics in animal models. HEPTAZYME, a ribozyme designed to selectively destroy Hepatitis C Virus (HCV) RNA, was found effective in decreasing Hepatitis C viral RNA in cell culture assays (Ribozyme Pharmaceuticals, Incorporated - WEB home page).<br>
An additional method of regulating the expression of a bee pathogen polypeptide gene in cells is via triplex forming oligonuclotides (TFOs). Recent studies have shown that TFOs can be designed which can recognize and bind to polypurine/polypirimidine regions in double-stranded helical DNA in a sequence-specific manner. These recognition rules are outlined by Maher III, L. J., et al., Science,1989;245:725-730;<br>
 <br>
Moser, H. E., et al., Science,1987;238:645-630; Beal, P. A., et al, Science,1992;251:1360-1363; Cooney, M., et al., Science,1988;241:456-459; and Hogan, M. E., et al., EP Publication 375408. Modification of the oligonuclotides, such as the introduction of intercalators and backbone substitutions, and optimization of binding conditions (pH and cation concentration) have aided in overcoming inherent obstacles to TFO activity such as charge repulsion and instability, and it was recently shown that synthetic oligonucleotides can be targeted to specific sequences (for a recent review see Seidman and Glazer, J Clin Invest 2003;112:487-94).<br>
In general, the triplex-forming oligonucleotide has the sequence correspondence:<br>
oligo	3'--A	G	G	T<br>
duplex	5'--A	G	C	T<br>
duplex	3'--T	C	G	A<br>
However, it has been shown that the A-AT and G-GC triplets have the greatest triple helical stability (Reither and Jeltsch, BMC Biochem, 2002, Sept12, Epub). The same authors have demonstrated that TFOs designed according to the A-AT and G-GC rule do not form non-specific triplexes, indicating that the triplex formation is indeed sequence specific.<br>
Thus for any given sequence in the bee pathogen polypeptide regulatory region a triplex forming sequence may be devised. Triplex-forming oligonucleotides preferably are at least 15, more preferably 25, still more preferably 30 or more nucleotides in length, up to 50 or 100 bp.<br>
Transfection of cells (for example, via cationic liposomes) with TFOs, and formation of the triple helical structure with the target DNA induces steric and functional changes, blocking transcription initiation and elongation, allowing the introduction of desired sequence changes in the endogenous DNA and resulting in the specific downregulation of gene expression. Examples of such suppression of gene expression in cells treated with TFOs include knockout of episomal supFG1 and endogenous HPRT genes in mammalian cells (Vasquez et al., Nucl Acids Res. 1999;27:1176-81, and Puri, et al, J Biol Chem, 2001;276:28991-98), and the sequence-and target specific downregulation of expression of the Ets2 transcription factor, important in prostate cancer etiology (Carbone, et al, Nucl Acid Res. 2003;31:833-43), and the pro-inflammatory ICAM-1 gene (Besch et al, J Biol Chem, 2002;277:32473-<br>
 <br>
79). In addition, Vuyisich and Beal have recently shown that sequence specific TFOs can bind to dsRNA, inhibiting activity of dsRNA-dependent enzymes such as RNA-dependent kinases (Vuyisich and Beal, Nuc. Acids Res 2000;28:2369-74).<br>
Additionally, TFOs designed according to the abovementioned principles can induce directed mutagenesis capable of effecting DNA repair, thus providing both downregulation and upregulation of expression of endogenous genes (Seidman and Glazer, J Clin Invest 2003;112:487-94). Detailed description of the design, synthesis and administration of effective TFOs can be found in U.S. Patent Application Nos. 2003 017068 and 2003 0096980 to Froehler et al, and 2002 0128218 and 2002 0123476 to Emanuele et al, and U.S. Pat. No. 5,721,138 to Lawn.<br>
The RNA, dsRNA, siRNA, or miRNA of the present invention may be produced chemically or enzymatically through manual or automated reactions or in vivo in an organism other than the plant for which pest control is intended. RNA may also be produced by partial or total organic synthesis. Any modified ribonucleotide can be introduced by in vitro enzymatic or organic synthesis. The RNA may be synthesized by a cellular RNA polymerase or a bacteriophage RNA polymerase (e.g., T3, T7, SP6). If synthesized chemically or by in vitro enzymatic synthesis, the RNA may be purified prior to feeding or formulated in an acceptable carrier and provided as a liquid, solid or semi-solid to the bees. For example, RNA can be purified from a mixture by extraction with a solvent or resin, precipitation, electrophoresis, chromatography, or a combination thereof. Alternatively, the RNA may be used with no, or a minimum of, purification to avoid losses due to sample processing. The RNA may be dried for storage or dissolved in an aqueous solution. The solution may contain buffers or salts to promote annealing, and/or stabilization of the duplex strands.<br>
For transcription from a transgene in vivo or from an expression cassette, a regulatory region (e.g., promoter, enhancer, silencer, leader, intron and polyadenylation) may be used to modulate the transcription of the RNA strand (or strands). Therefore, in one embodiment, there is provided a nucleic acid construct comprising the nucleic acid agent. The nucleic acid construct can have polynucleotide sequences constructed to facilitate transcription of the RNA molecules of the present invention are operably linked to one or more promoter sequences functional in a host cell. The polynucleotide sequences may be placed under the control of an endogenous promoter normally present<br>
 <br>
in the host genome. The polynucleotide sequences of the present invention, under the control of an operably linked promoter sequence, may further be flanked by additional sequences that advantageously affect its transcription and/or the stability of a resulting transcript. Such sequences are generally located upstream of the promoter and/or downstream of the 3' end of the expression construct. The term "operably linked", as used in reference to a regulatory sequence and a structural nucleotide sequence, means that the regulatory sequence causes regulated expression of the linked structural nucleotide sequence. "Regulatory sequences" or "control elements" refer to nucleotide sequences located upstream, within, or downstream of a structural nucleotide sequence, and which influence the timing and level or amount of transcription, RNA processing or stability, or translation of the associated structural nucleotide sequence. Regulatory sequences may include promoters, translation leader sequences, introns, enhancers, stem-loop structures, repressor binding sequences, termination sequences, pausing sequences, polyadenylation recognition sequences, and the like.<br>
The nucleic acid agent can be delivered to the bees in a great variety of ways. As detailed herein, bee feeding is common practice amongst bee-keepers, for providing both nutritional and other, for example, supplemental needs. Bees typically feed on honey and pollen, but have been known to ingest non-natural feeds as well. Bees can be fed various foodstuffs including, but not limited to Wheast (a dairy yeast grown on cottage cheese), soybean flour, yeast (e.g. brewer's yeast, torula yeast) and yeast products products-fed singly or in combination and soybean flour fed as a dry mix or moist cake inside the hive or as a dry mix in open feeders outside the hive. Also useful is sugar, or a sugar syrup. The addition of 10 to 12 percent pollen to a supplement fed to bees improves palatability. The addition of 25 to 30 percent pollen improves the quality and quantity of essential nutrients that are required by bees for vital activity.<br>
Cane or beet sugar, isomerized corn syrup, and type-50 sugar syrup are satisfactory substitutes for honey in the natural diet of honey bees. The last two can be supplied only as a liquid to bees.<br>
Liquid feed can be supplied to bees inside the hive by, for example, any of the following methods: friction-top pail, combs within the brood chamber, division board feeder, boardman feeder, etc. Dry sugar may be fed by placing a pound or two on the inverted inner cover. A supply of water must be available to bees at all times. In one<br>
 <br>
embodiment, pan or trays in which floating supports-such as wood chips, cork, or plastic sponge-are present are envisaged. Detailed descriptions of supplemental feeds for bees can be found in, for example, USDA publication by Standifer, et al 1977, entitled "Supplemental Feeding of Honey Bee Colonies" (USDA, Agriculture Information Bulletin No. 413).<br>
All the bees in a hive are potentially susceptible to the pathogenic diseases detailed herein. Thus, according to some embodiments, the bees can be honeybees, forager bees, hive bees and the like.<br>
Also provided is a method for reducing the susceptibility of a bee to a disease caused by pathogens, the method effected by feeding the bee on an effective amount of a nucleic acid or nucleic acid construct comprising a nucleic acid agent downregulating expression of a polypeptide of the bee pathogen and/or causing cleavage and/or degradation of a bee pathogen RNA. Methods for reducing the susceptibility of a bee colony or bee-hive to bee pathogens by feeding oligonucleotides and/or polynucleotides are envisaged. Thus, in some embodiments, the present invention can be used to benefit any numbers of bees, from a few in the hive, to the entire bee population within a hive and its surrounding area. It will be appreciated, that in addition to feeding of oligonucleotides and/or polynucleotides for reduction of the bee pathogen infection and infestation, enforcement of proper sanitation (for example, refraining from reuse of infested hives) can augment the effectiveness of treatment and prevention of infections.<br>
It is expected that during the life of a patent maturing from this application many relevant methods for downregulating bee pathogen proteins will be developed and the scope of the term "downregulating bee pathogen protein" or "downregulating bee pathogen polypeptide" is intended to include all such new technologies a priori.<br>
As used herein the term “about” refers to ± 10 %.<br>
The terms "comprises", "comprising", "includes", "including", “having” and their conjugates mean "including but not limited to". This term encompasses the terms "consisting of" and "consisting essentially of".<br>
The phrase "consisting essentially of" means that the composition or method may include additional ingredients and/or steps, but only if the additional ingredients and/or steps do not materially alter the basic and novel characteristics of the claimed composition or method.<br>
 <br>
As used herein, the singular form "a", "an" and "the" include plural references unless the context clearly dictates otherwise. For example, the term "a compound" or "at least one compound" may include a plurality of compounds, including mixtures thereof.<br>
Throughout this application, various embodiments of this invention may be presented in a range format. It should be understood that the description in range format is merely for convenience and brevity and should not be construed as an inflexible limitation on the scope of the invention. Accordingly, the description of a range should be considered to have specifically disclosed all the possible subranges as well as individual numerical values within that range. For example, description of a range such as from 1 to 6 should be considered to have specifically disclosed subranges such as from 1 to 3, from 1 to 4, from 1 to 5, from 2 to 4, from 2 to 6, from 3 to 6 etc., as well as individual numbers within that range, for example, 1, 2, 3, 4, 5, and 6. This applies regardless of the breadth of the range.<br>
Whenever a numerical range is indicated herein, it is meant to include any cited numeral (fractional or integral) within the indicated range. The phrases “ranging/ranges between” a first indicate number and a second indicate number and “ranging/ranges from” a first indicate number “to” a second indicate number are used herein interchangeably and are meant to include the first and second indicated numbers and all the fractional and integral numerals therebetween.<br>
As used herein the term "method" refers to manners, means, techniques and procedures for accomplishing a given task including, but not limited to, those manners, means, techniques and procedures either known to, or readily developed from known manners, means, techniques and procedures by practitioners of the chemical, pharmacological, biological, biochemical and medical arts.<br>
As used herein, the term “treating” includes abrogating, substantially inhibiting, slowing or reversing the progression of a condition, substantially ameliorating clinical or aesthetical symptoms of a condition or substantially preventing the appearance of clinical or aesthetical symptoms of a condition.<br>
It is appreciated that certain features of the invention, which are, for clarity, described in the context of separate embodiments, may also be provided in combination in a single embodiment.  Conversely, various features of the invention, which are, for<br>
 <br>
brevity, described in the context of a single embodiment, may also be provided separately or in any suitable subcombination or as suitable in any other described embodiment of the invention. Certain features described in the context of various embodiments are not to be considered essential features of those embodiments, unless the embodiment is inoperative without those elements.<br>
Various embodiments and aspects of the present invention as delineated hereinabove and as claimed in the claims section below find experimental support in the following examples.<br>
EXAMPLES<br>
Reference is now made to the following examples, which together with the above descriptions illustrate some embodiments of the invention in a non limiting fashion.<br>
Generally, the nomenclature used herein and the laboratory procedures utilized in the present invention include molecular, biochemical, microbiological and recombinant DNA techniques. Such techniques are thoroughly explained in the literature. See, for example, "Molecular Cloning: A laboratory Manual" Sambrook et al., (1989); "Current Protocols in Molecular Biology" Volumes I-III Ausubel, R. M., ed. (1994); Ausubel et al., "Current Protocols in Molecular Biology", John Wiley and Sons, Baltimore, Maryland (1989); Perbal, "A Practical Guide to Molecular Cloning", John Wiley &amp; Sons, New York (1988); Watson et al., "Recombinant DNA", Scientific American Books, New York; Birren et al. (eds) "Genome Analysis: A Laboratory Manual Series", Vols. 1-4, Cold Spring Harbor Laboratory Press, New York (1998); methodologies as set forth in U.S. Pat. Nos. 4,666,828; 4,683,202; 4,801,531; 5,192,659 and 5,272,057; "Cell Biology: A Laboratory Handbook", Volumes I-III Cellis, J. E., ed. (1994); "Culture of Animal Cells - A Manual of Basic Technique" by Freshney, Wiley-Liss, N. Y. (1994), Third Edition; "Current Protocols in Immunology" Volumes I-III Coligan J. E., ed. (1994); Stites et al. (eds), "Basic and Clinical Immunology" (8th Edition), Appleton &amp; Lange, Norwalk, CT (1994); Mishell and Shiigi (eds), "Selected Methods in Cellular Immunology", W. H. Freeman and Co., New York (1980); available immunoassays are extensively described in the patent and scientific literature, see, for example, U.S. Pat. Nos. 3,791,932; 3,839,153; 3,850,752; 3,850,578; 3,853,987; 3,867,517; 3,879,262; 3,901,654; 3,935,074; 3,984,533; 3,996,345; 4,034,074; 4,098,876; 4,879,219; 5,011,771 and 5,281,521; "Oligonucleotide Synthesis" Gait, M.<br>
 <br>
J., ed. (1984); “Nucleic Acid Hybridization" Hames, B. D., and Higgins S. J., eds. (1985); "Transcription and Translation" Hames, B. D., and Higgins S. J., eds. (1984); "Animal Cell Culture" Freshney, R. I., ed. (1986); "Immobilized Cells and Enzymes" IRL Press, (1986); "A Practical Guide to Molecular Cloning" Perbal, B., (1984) and "Methods in Enzymology" Vol. 1-317, Academic Press; "PCR Protocols: A Guide To Methods And Applications", Academic Press, San Diego, CA (1990); Marshak et al., "Strategies for Protein Purification and Characterization - A Laboratory Course Manual" CSHL Press (1996); all of which are incorporated by reference as if fully set forth herein. Other general references are provided throughout this document. The procedures therein are believed to be well known in the art and are provided for the convenience of the reader. All the information contained therein is incorporated herein by reference.<br>
Example I Effect of IAPV virus titer on survival of honeybees<br>
In order to determine whether bees are differentially sensitive to IAPV, the effect of virus titer on bee survival was tested. Bees were introduced into plastic containers and exposed to increasing concentrations of IAPV (in feed solution). Survival of the bees in the hive was monitored over a period of 7 days.<br>
MATERIALS AND METHODS<br>
50 bees were introduced into 0.5 liter plastic containers, or 30 bees into 0.25 liter plastic containers.<br>
All containers were prepared in advance to accommodate air flow and enable feeding with sucrose solution and water. The bees were kept in the dark at a constant temperature of 30° C, and were fed once daily with 2 ml 50% sucrose solution and 1 ml water introduced into reservoirs in the containers.<br>
IAPV was introduced into the sucrose solution, in increasing doses (0.0001 to 0.1 microgram/microliter) in 900 microliters sucrose solution added to the sucrose: Vertical striped bars-100 ng/μl; Cross-hatched bars-10 ng/μl; Stippled bars-1.0 ng/μl; Horizontal striped bars-0.1 ng/μl. Controls (checkered bars) were without added virus. Dead bees in the containers were counted daily, and the total number of dead bees was calculated as a percentage of the initial number of bees introduced into the container.<br>
 <br>
Results<br>
As can be seen in FIG. 1, bee mortality during the days 1 and 2 was negligible for all viral concentrations. Significant mortality, of 35-60% (depending on viral titer) was observed from day 3, steadily increasing to 70-85% by day 6-7. It was noted that lower viral doses resulted in an initially higher mortality, which plateaued by day 6, whereas mortality among bees exposed to higher concentrations continued to increase from day 3.<br>
Example II Feeding viral-specific dsRNA prevents acute disease of honeybees caused by IAPV<br>
In order to determine the effectiveness of ingested IAPV dsRNA on viral infection, honeybees were provided with IAPV-specific and control dsRNA in the feed for 4 days before, and 3 days following infection with IAPV virus. Numbers of dead bees per experimental hive were counted, and sample live and dead bees were collected for molecular analysis.<br>
MATERIALS AND METHODS<br>
Establishment of mini-hive colonies: Young, approximately 2-month-old queens, together with approximately 200 worker bees were collected from hives in a local apiary. The bees were transferred into mini-hives fitted with one mini comb that was previously built by a regular hive. All of the mini-hives were closed and placed in a temperature-controlled room (30°C).<br>
dsRNA preparation: IAPV sequences corresponding to the intergenic region (bases 6168-6594; gi|124494152; 426 b SEQ ID NO: 33) and to a viral sequence known to integrate into the bee genome (bases 8977-9410; gi|124494152; 433 b, SEQ ID NO: 34) were cloned into a plasmid between two opposing T7 promoters. Following propagation of plasmid DNA, the viral fragments, including the T7 promoters, were excised, gel-purified, and served as templates for T7-directed in-vitro transcription (MEGAscriptTM, Ambion, Austin TX). The reaction product was submitted to DNase digestion followed by phenol extraction and ethanol precipitation. The final preparation was dissolved in nuclease-free water.<br>
dsRNA feeding in minihives: 5 gr. pollen supplement patties were placed on top of each comb and 10 ml of 50% sucrose solution was introduced into the hive in a<br>
 <br>
sterile Petri dish nightly. The feeding was continued for 7 days and subsequently only hives in which queens had begun to lay eggs were included in the trial.<br>
Following establishment of active hives (queens laying eggs), some of the mini-hives were supplemented with viral-specific or non-specific control (IAPVds or GFPds) dsRNA, which was added to the 10 ml 50% sugar solution given to the hives, adjusted to approximately 1 microgram dsRNA per feed per bee, assuming all bees consume approximately the same amount of sucrose solution. dsRNA feeding was continued for six days.<br>
IAPV infection in minihives: Three days after feeding in active hives, some of the colonies were fed with 0.01 microgram per microliter of IAPV in the 50% w/v sucrose solution (IAPV). Thereafter dsRNA treatments continued for a further 3 days. Samples of live and dead bees (larvae and adults) were collected daily from each mini-hive post introduction of IAPV for 7 consecutive days. Every bee collected was frozen in liquid nitrogen and preserved at -70°C pending molecular analysis. Vitality of the colonies was monitored by opening the hives (without smoke), withdrawing the mini-comb and photographing the mini-comb from both sides. The hive-combs were photographed daily, and the number of remaining live bees was monitored. The photographs were downloaded onto a computer and the total number of bees was counted for every mini-hive.<br>
To test dsRNA toxicity, another group of hives was provided with IAPV-specific dsRNA, but was not IAPV inoculated. Two sets of hives served as additional controls: hives that were not treated with dsRNA and were not inoculated with IAPV, and hives that were not treated with dsRNA, but were inoculated with IAPV.<br>
RT-PCR analysis:<br>
Extraction of Nucleic Acids: Total RNA was extracted from the preserved bees using the TRIREAGENT method (Sigma, St. Louis MO, USA). Briefly, RNA was extracted by precipitation and separation by centrifugation, then resuspended in RNAsecure solution.<br>
Real-Time RT-PCR: Measured amounts of RNA (100 ng for viral expression analyses and 100 pg for 18S rRNA internal controls) were subjected to one-step RT-PCR using the SYBR Green PCR master mix with Taqman reverse transcriptase (Applied Biosystems, Foster City, CA). Real-time RT-PCR was conducted in GeneAmp<br>
 <br>
PCR System 5700 (Applied Biosystems). Reactions performed without reverse transcriptase or without template did not result in any product. PCR cycles were as follows: 1 cycle of 30 min at 48°C and 10 min at 95°C, followed by 40 cycles each of 15 s at 95°C, 30 s at 60°C, and 45 s at 72°C.<br>
Table III shows the primers for all IAPV-related RT-PCR assays, including realtime RT-PCR:<br>
Table III. Primers used for PCR<br>
 <br>
Northern-Blot Analysis: Total RNA was extracted from treated and control bees. Formaldehyde was added to the RNA to 1.8% and warmed to 65°C. The RNA, 15 µg per lane (in light of the real-time PCR results, only 1.5 µg of RNA was loaded in the case of upper leaves of inoculated plants), was electrophoresed on a 1.2% agarose gel at 70 V, 4°C with stirring. The previously described amplified IAPV-RNA product was digoxigenin labeled and served as a probe for hybridization.  Detection was performed<br>
 <br>
with the DIG luminescent detection kit (Roche Diagnostics GmbH, Mannheim, Germany). RNA sizes were estimated by comparison to electrophoresed RNA Molecular Weight Markers I (Roche). Hybridization was carried out at high stringency (0.1x SSC; 65°C).<br>
Results<br>
As can be seen in FIG. 3, among bees inoculated with IAPV (on day 0) mortality was significantly reduced (25% mortality) in bees treated with IAPV-dsRNA (empty circles) relative to untreated controls (filled squares) (75% mortality) and sham-treated controls (GFP-dsRNA, empty squares)(75% mortality)(LSmeans contrast, F1,82 = 9.74, P=0.002). Mortality of bees treated with IAPV-dsRNA and inoculated with IAPV tended to increase relative to the noninfected bees (filled circles), though the difference did not reach statistical significance (LSmeans contrast, F1,82 = 3.25, NS).<br>
FIG. 3 clearly demonstrates the efficacy of feeding IAPV-dsRNA in protecting bees from subsequent IAPV infection, whereas unrelated dsRNA (sham treated controls- GFP) fails to protect bees from infection. Ingestion of sham dsRNA had no effect on the survival of the bees, relative to untreated controls. Ingestion of dsRNAs (of IAPV and GFP) without subsequent IAPV inoculation did not harm bees in any way, indicating absence of toxicity of the dsRNA.<br>
When detecting RNA using RT-PCR, the effect of feeding dsRNA-IAVP on IAPV infection in the bees is clear. FIG. 4A shows that IAPV-dsRNA-treated bees carry only residual virus transcripts (lanes 1 and 2), whereas considerable amounts of virus transcripts are detected in the untreated (lane 4), and GFP-dsRNA-treated bees (lane 3). As indicated by the identity of the band size and intensity in lanes 1-5 (actin internal controls) of FIG. 4B, all samples contained comparable amounts of template RNA.<br>
Using real-time PCR, FIG. 5 shows the strong decline in IAPV level detected in bee populations treated with IAPV-dsRNA. At the fourth day after inoculation of IAPV, real-time PCR detected approximately two orders of magnitude (2 log units) fewer IAPV in IAPV-dsRNA-treated bees as compared to that detected in bees that had not been treated with IAPV-dsRNA, or treated with unrelated (GFP) dsRNA (FIG.5). With very few exceptions, the queens and a few nursing bees survived IAPV infection, a situation reminiscent of CCD-affected hives. Thus, the reduction in mortality among<br>
 <br>
bees ingesting dsRNA-IAPV is clearly due to extensive reduction in the levels of viral infection and proliferation in the treated bees.<br>
The fate of ingested IAPV-specific dsRNA in honeybees: In order to better understand the mechanism(s) of action by which dsRNA-IAPV protects the bees against IAPV infection and its consequences, total RNA was extracted from dsRNA-IAPV treated, and non-treated control bees, submitted to digestion by a panel of nucleases, and separated on PAGE. As can be seen from FIG. 6 (see lanes 1, 2 and 3, representing digestion with RNase A, digestion with DNase I and digestion with RNase A and RNase II, respectively) the presence of a 500 base pair band representing dsRNA in the treated bees (RNase A and DNase I resistant, and RNase III sensitive) indicates actual, successful ingestion of the dsRNA-IAPV and its persistence in the bee.<br>
When RNA extracted from dsRNA-IAPV treated, and control bees was separated, blotted and probed for IAPV-specific sequences (see FIG. 7) the presence of small IAPV-specific sequences exclusively in the RNA from dsRNA-IAPV bees (see FIG. 7, lane 3 vs FIG. 7, lanes 4-7) was detected. Thus, ingestion of dsRNA-IAPV initiates an RNAi related pathway, leading to the production of small IAPV specific peptides and the silencing of IAVP reproduction in the treated cells.<br>
Taken together, these results show that IAPV can be silenced in bees by feeding with a segment or segments of IAPV-dsRNA and further indicate the activity of an RNAi-related pathway of silencing. The dsRNA-engendered silencing was sufficient to greatly reduce bee mortality resulting from IAPV infection.<br>
Example III Large-scale field trials of viral-specific dsRNA for prevention of IAPV-associated<br>
disease of honeybees<br>
In order to determine the effectiveness of ingested IAPV dsRNA on viral infection under actual field conditions, and to assess effects on important parameters of colony health, bees in sample full size hives were provided with IAPV-specific dsRNA in the feed for 4 days before, and 3 days following infection with IAPV virus.<br>
 <br>
MATERIALS AND METHODS Insect material :<br>
Pools of five bees from the following treatments; Remote control, IAPV-dsRNA only, IAPV only and IAPV-specific dsRNA+IAPV at each time point day 0-(day of virus application), day 7 and end point (day 42). The test was repeated several times.<br>
RNA extraction:<br>
RNA extracted using Tri-Reagent (Sigma,USA) according to protocol provided by the manufacturer. All samples treated with DNaseI and resuspended with loading buffer (90% Formamide, 0.05 Bromophenol Blue, 0.05% Xylene cyanol) prior to loading on gel.<br>
Gel electrophoresis and Blot:<br>
10 ug of freshly prepared RNA was measured using the nanodrop spectrophotometer and loaded on 12% Acrylamide gel (1:19 acrylamide:Bis acrylamide ratio) in danturation environment (gel contains 7M Urea). After electrophoresis samples were transferred to positively charged nylon membrane (Roch,USA) using electrobloting method.<br>
Hybridization and signal detection :<br>
Membrane hybridized with freshly prepared DNA probe of IAPV segment, taken from a region that does not correspond to the dsRNA of the IAPV-specific dsRNA itself. This is made using DIG PCR probe preparation Kit (Roch,USA) o/n 42°C in DIG easyhyb solution (Roch, USA) according to manufacturer protocol. The membrane was washed twice with 2XSSC/0.1%SDS than washed for stringency with 0.1XSSC/0.1%SDS in 65°C. Membranes were further washed using DIG Wash and Block Kit (Roch, USA) according to manufacturer protocol. Detection was preformed using CSPD-star substrate (Roch, USA). Positive control was 21nt DNA primers corresponding to the hybridized sequence.<br>
Signal was detected using membrane exposure for 2-12 hours in chemiluminator manufactured by Kodak<br>
Results<br>
Feeding IAPV-dsRNA to a hive does not affect overall health of the colony:<br>
Basic parameters of bee colony health (numbers of capped brood, numbers of bees in the hive, returning foragers and honey production) were assessed in hives fed IAPV-dsRNA<br>
 <br>
and control hives, in the absence of infection with IAPV. Table IV presents the results of this comparison.<br>
Table IV: Comparison between IAPV-dsRNA-treated and control hives<br>
 <br>
Table IV clearly shows no significant differences between the treated and control hives in any of the measured parameters, indicating that feeding IAPV-dsRNA is benign to the bees and the colony as a whole, in the absence of IAPV infection.<br>
Detection of IAPV-specific siRNAs in treated honeybees under field conditions:<br>
FIGs. 11A and 11B show siRNAs specific to IAPV sequence detected by gel electrophoresis and hybridization of honeybee RNA to a IAPV-specific probe. At day 0 no IAPV-specific siRNAs were detected in treated or untreated bees (FIG 11B). By day 7, IAPV-specific siRNAs were detected exclusively in bees fed IAPV-specific dsRNA and infected with IAPV (FIG. 11A, lane 6). At the end if the experiment, day 42, IAPV-specific siRNA was detected weakly in one sample of RNA from IAPV-infected bees (FIG. 11A, lane 9) and also weakly detected in one sample from bees fed IAPV-specific dsRNA but not infected with IAPV (FIG. 11B, lane 7). In contrast both samples from bees fed IAPV-specific dsRNA and exposed to IAPV (FIG. 11A, lane 10 and FIG. 11B, lane 9) showed a strong signal at 21 bp, indicating greatly increased amounts of IAPV-specific siRNAs. Untreated (remote) control bees (FIG. 11A, lanes 1 and 7, FIG.11B, lanes 1 and 6) showed no signal throughout the experimental period, indicating an absence of IAPV-specific sequences.<br>
These results indicate that IAPV-specific siRNA is present in bees fed IAPV-specific dsRNA and exposed to IAPV infection only. While not wishing to be limited to a single hypothesis, it is postulated that where IAPV infection is severe, the initial IAPV-specific dsRNA silencing signal is amplified (IAPV-specific dsRNA plus IAPV<br>
 <br>
infection, FIG. 11A lane 10 and FIG. 11B lane 9), the strong presence of siRNAs probably restricts the severity of the disease in the bees leading to a longer life-span.<br>
IAPV-dsRNA prevents symptoms of IAPV in IAPV infected colonies<br>
After establishing that IAPV-specific dsRNA alone did not make any difference relative to the untreated control, colonies receiving virus only and IAPV-specific dsRNA + virus were compared to test the efficacy of the treatment in directly preventing the IAPV symptoms.<br>
Colony Collapse Disorder (CCD) is characterized by thinning of the affected colony due to reduced numbers of returning foragers, with dead bees typically found outside the hive. FIG. 8 shows the effect of feeding IAPV-dsRNA on the numbers of returning foragers in virus-infected colonies. At the beginning of the experiment, a small (insignificant) difference in the numbers of returning foragers can be discerned between the treatments. However, with greater time following IAPV infection the IAPV-specific dsRNA+ IAPV treated colonies showed progressively greater numbers of returning foragers, as compared to the other colonies. Without wishing to be limited to a single hypothesis, the initial differences in the numbers of returning foragers observed in the first week can be attributed to the death of foragers in the weeks following infection with IAPV.<br>
Another important parameter characteristic of CCD is a reduction in the total number of bees in the hive. FIG. 9 shows that although mid-point analysis of the field trial hives shows no difference in the estimated number of bees in the hives between treated and non-treated colonies, the advantages of IAPV specific-dsRNA were clearly evident by the end point of the trial. FIG. 9 shows that, at 5 weeks following IAPV infection, the estimated number of bees in the uninfected control and IAPV remained insignificantly different, whereas colonies receiving IAPV-specific dsRNA were significantly more populated (p
Honey production of a hive reflects not only by the numbers of bees in the colony, but their overall health and robustness. Flight activity data was correlated with honey production in the treated and control colonies.<br>
When compared between IAPV-specific dsRNA and control colonies, flight activity data correlated strongly with honey production. FIG. 10 shows that IAPV-specific dsRNA+IAPV  treated hives produced approximately three times more honey<br>
 <br>
than IAPV-infected only hives and nearly twice the amount of honey of the uninfected control hives. Further, the number of hives producing significant honey was much greater in the IAPV-specific dsRNA than those in the untreated virus-infected colonies. Moreover, none (0%) of the IAPV-specific dsRNA treated colonies died during the experiment, compared to four dead out of 20 (20%) of the untreated, virus infected colonies and one dead out of 20 (5%) control colonies.<br>
Taken together, these results show that silencing of IAPV in bees by feeding with a segment or segments of IAPV-dsRNA is effective in preventing symptoms of IAVP in infected colonies, resulting in greater viability of the bee colonies and surprisingly improved honey yields.<br>
Example IV Multiple bee-virus resistance sequence<br>
In order to enhance the efficacy of the use of a nucleic acid agent in reducing susceptibility of the bees and bee colonies to viral pathogens, bee-viral sequences were compared for sequence homology, and a composite nucleic acid agent, comprising multiple bee-virus sequences was designed.<br>
FIG. 12 shows the phylogenetic relationship between several bee viruses whose genomes have been fully sequenced: Acute bee paralysis virus (ABPV) – GenBank AF150629(SEQ ID NO: 3), Kashmir bee virus (KBV) – GenBank AY275710(SEQ ID NO: 9), Sacbrood virus (SBV) – GenBank NC_002066(SEQ ID NO: 2), Black queen cell virus (BQCV) – GenBank AF183905(SEQ ID NO: 1), Kakugo virus (KV) – GenBank AB070959(SEQ ID NO: 4), Deformed wing virus (DWV) – GenBank AJ489744(SEQ ID NO: 53) and Israel acute paralysis virus (IAPV) – GenBank EF219380 (SEQ ID NO: 6). Sequences from IAPV having high homology to ABPV GenBank AF150629 (SEQ ID NO: 3) and KBV AY275710 (SEQ ID NO: 9) were identified by alignment of the viral genomes. Sequences from DWV having high homology to KV GenBank AB070959 (SEQ ID NO: 4) and VDV-1 GenBank AY251269 (SEQ ID NO: 5) were also identified by alignment of the viral genomes. To these, sequences from the BQCV genome (SEQ ID NO: 1) and SBV genome (SEQ ID NO: 2) were added, and a composite nucleic acid construct having high sequence homology to all of the abovementioned bee viruses (with added pT7 viral sequences) was produced (SEQ ID NO: 24, FIG. 13).<br>
 <br>
Table V shows the primers useful for creating multiple bee-virus resistance dsRNA:<br>
Table V. Primers used for multiple bee-virus resistance dsRNA<br>
 <br>
It will be appreciated that feeding bees with a dsRNA comprising this multiple bee-viral homologous sequence will be effective in treating and preventing infection and symptoms of a broad variety of bee-viral infections in affected colonies. Yet further, without being limited to a single hypotheses, it is suggested that the numerous consensus sequences reflecting the high cross-species homology of the multiple bee-virus construct (SEQ ID NO: 24) can be processed (in the cell, by dsRNA processing enzymes) to RNAi effective against many bee viruses, including bee-viruses not yet identified and/or sequenced.<br>
It will be appreciated that multiple bee-pathogen sequences for dsRNA effective in protecting against more than one species or variant can be determined in a similar manner for non-viral bee pathogens, for example, using the sequences of the pathogenic organisms detailed in Table II above. Multiple bee-pathogen sequences can include sequences within a certain class of pathogens (e.g. viruses, bacteria), or even include<br>
 <br>
sequences effective for different and diverse classes of pathogens (e.g, viruses + bacteria + fungi, etc).<br>
Although the invention has been described in conjunction with specific embodiments thereof, it is evident that many alternatives, modifications and variations will be apparent to those skilled in the art. Accordingly, it is intended to embrace all such alternatives, modifications and variations that fall within the spirit and broad scope of the appended claims.<br>
All publications, patents and patent applications mentioned in this specification are herein incorporated in their entirety by reference into the specification, to the same extent as if each individual publication, patent or patent application was specifically and individually indicated to be incorporated herein by reference. In addition, citation or identification of any reference in this application shall not be construed as an admission that such reference is available as prior art to the present invention. To the extent that section headings are used, they should not be construed as necessarily limiting.<br>
 <br>
CD-ROM Content<br>
The following lists the file content of the CD-ROMs which are enclosed herewith and filed with the application. File information is provided as: File name/byte size/date of creation/operating system/machine format.<br>
CD-ROM1 (1 file):<br>
1.        44941 Sequence Listing/ 11.3 MB/ Monday, 03 November, 2008/ Notepad/ PC.<br>
CD-ROM2 (1 file):<br>
1.        44941 Sequence Listing/ 11.3 MB/ Monday, 03 November, 2008/ Notepad/ PC.<br>
CD-ROM3 (1 file):<br>
1.        44941 Sequence Listing/ 11.3 MB/ Monday, 03 November, 2008/ Notepad/ PC.<br>
 <br>
LIST OF REFERENCES<br>
(Additional references are cited in the text)<br>
Araujo et al., Insect Mol. Biol 2006;36:683<br>
Aronstein et al., J Apiculture Res Bee World 2006;45:20-24<br>
Bhargava A et al. Brain Res. Protoc. 2004;13:115–125<br>
Chen YP, et al Appl Environ Microbiol. 2006;72(1):606-11<br>
Cox-Foster et al. Science 2007;318: 283-286<br>
Diallo M., et al., Oligonucleotides. 2003;13:381–392<br>
Maori et al. Virology 2007;362:342<br>
Paddison P.J., et al., Proc. Natl Acad. Sci. USA. 2002;99:1443–1448<br>
Standifer, et al 1977:     "Supplemental Feeding of Honey Bee Colonies" (USDA,<br>
Agriculture Information Bulletin No. 413)<br>
Strat et al., Nucleic Acids Research, 2006, Vol. 34, No. 13 3803–3810<br>
Tran N., et al., FEBS Lett. 2004;573:127–134<br>
Turner et al., Insect Mol Biol 2006;15:383<br>
 <br>
We claim:<br>
1.	A bee-ingestible composition comprising an isolated dsRNA comprising complementary RNA sequences forming an RNA duplex, and wherein one RNA sequence of said duplex is complementary to a nucleic acid sequence of a bee pathogen, wherein said dsRNA downregulates expression of a gene product of said bee pathogen.<br>
2.	The bee-ingestible composition of claim 1, wherein said gene product is an mRNA encoding a bee pathogen specific polypeptide.<br>
3.	The bee-ingestible composition of claim 1, wherein said dsRNA is selected from the group consisting of siRNA, shRNA and miRNA.<br>
4.	A nucleic acid construct comprising a nucleic acid sequence encoding the RNA sequence complementary to said nucleic acid sequence of a bee pathogen of claim 1.<br>
5.	The bee-ingestible composition of claim 1, wherein said RNA duplex comprises an RNA sequence complementary to a nucleic acid sequence of said bee pathogen is greater than 15 bases in length.<br>
6.	The bee-ingestible composition of claim 5, wherein said RNA duplex comprises an RNA sequence complementary to a nucleic acid sequence of said bee pathogen is 19 to 25 bases in length.<br>
7.	The bee-ingestible composition of claim 6, wherein said RNA duplex comprises an RNA sequence complementary to a nucleic acid sequence of said bee pathogen is greater than 30 base pairs in length.<br>
8.	The isolated nucleic acid agent of claim 1, wherein said bee pathogen is selected from the group consisting of a virus, a bacteria, a parasitic protozoan, a fungus and a nematode.<br>
 <br>
9.	The isolated nucleic acid agent of claim 1, wherein said bee pathogen is a virus.<br>
10.	The bee-ingestible composition of claim 9, wherein said virus is Israel Acute Paralysis Virus.<br>
11.	The bee-ingestible composition of claim 9, wherein said virus is Kashmir Paralysis Virus.<br>
12.	The bee-ingestible composition of claim 9, wherein said virus is Israel Acute Paralysis Virus and said polypeptide of said virus is selected from the group consisting of IAPV polymerase polyprotein (SEQ ID NO. 51) and IAPV structural polyprotein (SEQ ID NO: 52).<br>
13.	The bee-ingestible composition of claim 1, wherein said bee pathogen is Israel Acute Paralysis Virus and said nucleic acid sequence is as set forth in any of SEQ ID NOs: 24, 33 or 34.<br>
14.	The bee-ingestible composition of claim 1, wherein said bee pathogen is Israel Acute Paralysis Virus and said nucleic acid sequence is as set forth in any of SEQ ID NOs: 54-9532.<br>
15.	The bee-ingestible composition of claim 1, wherein said bee virus is Israel Acute Paralysis Virus, and said nucleic acid sequence is a viral nucleic acid sequence integrated into a honeybee genome following Israel Acute Paralysis Virus infection.<br>
16.	The bee-ingestible composition of claim 1, wherein said composition is in solid form.<br>
 <br>
17.	The bee-ingestible composition of claim 1, wherein said composition is in liquid form.<br>
18.	The bee-ingestible composition of claim 1, wherein said composition comprises protein.<br>
19.	The bee-ingestible composition of claim 18, wherein said protein is in the form of pollen and/or soy patties.<br>
20.	The bee-ingestible composition of claim 19, wherein said liquid is a sucrose solution.<br>
21.	The bee-ingestible composition of claim 19, wherein said liquid is a corn syrup solution.<br>
22.	The bee-ingestible composition of claim 19, wherein said liquid further comprises a carbohydrate or sugar supplement.<br>
23.	A method for reducing the susceptibility of a bee to a disease caused by a pathogen comprising feeding the bee an effective amount of the bee-ingestible composition of claim 1, thereby reducing the susceptibility of said bee to said pathogen.<br>
24.	The method of claim 23, wherein said bee is a honeybee.<br>
25.	The method of claim 23, wherein said honeybee is a forager.<br>
26.	The method of claim 23, wherein said honeybee is a hive bee.<br>
27.	The method of claim 23, wherein said honeybee is a bee of a colony, wherein said feeding reduces the susceptibility of said bee colony to said disease.<br>
 <br>
28.	The method of claim 23, wherein said disease is Colony Collapse Disorder.<br>
29.	The method of claim 23, wherein said bee pathogen is Israel Acute Paralysis Virus.<br>
30.	The method of claim 29, wherein said dsRNA comprises a nucleic acid sequence selected from the group consisting of SEQ ID NOs: 54-9532.<br>
31.	The method of claim 29, wherein said dsRNA comprises a nucleic acid sequence as set forth in SEQ ID NOs: 24, 33 and 34.<br>
32.	The method of claim 23, wherein said bee-ingestible composition is a liquid bee-ingestible composition.<br>
33.	The methods of claim 23, wherein said bee-ingestible composition is a solid bee-ingestible composition.<br>
34.	A method for reducing the susceptibility of a bee to a disease caused by a bee pathogen comprising feeding the bee an effective amount of the nucleic acid construct of claim 4, thereby reducing the susceptibility of said bee to said bee pathogen.<br>
35.	The method of claim 34, wherein said nucleic acid construct comprises a nucleic acid sequence selected from the group consisting of any of SEQ ID NOs: 54-9532.<br>
36.	The method of claim 34, wherein said nucleic acid construct comprises a nucleic acid sequence as set forth in any of SEQ ID NOs: 24, 33 or 34.<br>
37.	A method of reducing the susceptibility of honeybees to Colony Collapse Disorder (CCD), the method comprising feeding to the honeybee hive an effective amount of double stranded ribonucleic nucleic acid (RNA), said double stranded RNA comprising<br>
 <br>
an RNA duplex comprising a sequence complementary to at least 21 nucleotides of an Israel Acute Paralysis Virus-specific mRNA and capable of inducing degradation of said virus-specific mRNA.<br>
38.	A bee ingestible composition comprising a dsRNA sequence comprising a RNA duplex comprising a complementary to at least 21 nucleotides of a bee pathogen specific RNA and capable of inducing degradation of said bee pathogen specific RNA.<br>
39.	A bee ingestible composition comprising at least two dsRNA sequences downregulating expression of at least one gene product of at least one bee pathogen.<br>
40.	The isolated nucleic acid of claim 39, wherein said at least one bee pathogen is selected from the group consisting of a virus, a bacteria, a parasitic protozoan, a fungus and a nematode.<br>
41.	The bee ingestible composition of claim 39, wherein said at least one bee pathogen comprises at least two bee pathogens.<br>
42.	The bee ingestible composition of claim 39, wherein said dsRNA sequences correspond to conserved sequences in said at least two bee pathogens.<br>
43.	The bee ingestible composition of claim 39, wherein said dsRNA comprises a duplex RNA comprising a nucleic acid sequence as set forth in SEQ ID NO. 24.<br>
</td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/viewdoc.aspx?id=zBPnbkWFc2r1XIEQ3DqcTg==&amp;amp;loc=vsnutRQWHdTHa1EUofPtPQ==" target="_blank" style="word-wrap:break-word;">http://ipindiaonline.gov.in/patentsearch/GrantedSearch/viewdoc.aspx?id=zBPnbkWFc2r1XIEQ3DqcTg==&amp;amp;loc=vsnutRQWHdTHa1EUofPtPQ==</a></p>
		<br>
		<div class="pull-left">
			<a href="279130-dkk-1-antibodies.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="279132-a-process-for-the-synthesis-of-fipronil.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>279131</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>1150/MUMNP/2010</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>02/2017</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>13-Jan-2017</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>11-Jan-2017</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>01-Jun-2010</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>BEEOLOGICS, INC,</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td>11800 SW 77th Avenue, Miami, Florida 33156, USA</td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>PALDI  Nitzan</td>
											<td>Meshek 77  Moshav Bar Giora  99880 Doar-Na HaEla  Israel.</td>
										</tr>
										<tr>
											<td>2</td>
											<td>YARDEN  Gal</td>
											<td>Nir-Moshe 215  85370 Doar-Na Negev  Israel.</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>C12N15/11</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>PCT/IL2008/001440</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td>2008-11-03</td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td>12/222,949</td>
									<td>2008-08-20</td>
								    <td>U.S.A.</td>
								</tr>
								<tr>
									<td>2</td>
									<td>60/996,244</td>
									<td>2007-11-07</td>
								    <td>U.S.A.</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/279131-a-composition-for-conferring-tolerance-to-viral-disease-in-social-insects by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 00:27:48 GMT -->
</html>
