<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/279330-nucleic-acid-encoding-recombinant-antibody by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 01:26:46 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 279330:&quot;NUCLEIC ACID ENCODING RECOMBINANT ANTIBODY&quot;</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">&quot;NUCLEIC ACID ENCODING RECOMBINANT ANTIBODY&quot;</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>The present invention provides a nucleic acid which comprises a non-specific promoter and at least one sequence that encodes a polypeptide that has at least one heterologous T cell epitope therein but does not have any regulatory T cell epitopes. The polypeptide may be one chain of a heterodimer, the heterologous T cell epitope causing disruption of the heterodimer chain such that it cannot bind with the other chain of the heterodimer. The nucleic acid can be used to raise a T cell response against the at least one heterologous T cell epitope.</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td>Nucleic acids<br>
The present invention relates to nucleic acids and to their use as vaccines, the nucleic acids encoding T cell epitopes against which an immune response is to be raised. Such vaccines may be used in the treatment of tumours.<br>
In the field of cancer vaccines and chronic viral infections, it is now becoming clear that factors other than frequency, such as functional avidity of tumour specific T cells and route of priming, are major determinants in maximising vaccine efficacy. A number of groups have shown that high avidity CD8+ T cells demonstrate superior anti-tumour activity (Alexander-Miller, Immunologic Research, 2005;31:13-24, Hodge et al, J Immunol 2005;174: 5994-6004, Valmori et al, J Immunol 2002;168:4231 -40, Zeh et al, J Immunol 1999;162: 989-94). It has been suggested that high avidity T cells play a vital role in tumour regression in patients. This is exemplified in a study where high avidity antigen-specific tumour infiltrating lymphocytes (TIL) were detected in a patient with dramatic tumour regression (Khong &amp; Rosenberg, J Immunol 2002;168: 951 -6). Evidence is also emerging demonstrating that adoptive transfer of in vitro stimulated autologous tumour-specific T cells is successful, possibly as in vitro stimulation enables selection of the high avidity T cells (Vignard et al., J Immunol 2005 ;175: 4797-805, Dudley et al., J Immunother 2001 ;24:363-73, Morgan et al, J Immunol 2003;171: 3287-95, Rosenberg &amp; Dudley, Proceedings of the National Academy of Sciences of the United States of America 2004;101 Suppl 2:14639-45).<br>
Hitherto, a number of groups have attempted to raise a cellular immune response against a pre-determined epitope using an antibody as a carrier for that epitope. For example, WO 96/19584 (Bona et al.) discloses chimeric antibodies in which T cell epitopes are inserted into the complementarity determining regions (CDRs) of an antibody, and alleges that such chimeric antibodies are suitable for raising a cytotoxic T cell (CTL) response. However, this document teaches that the DNA must encode a functional<br>
protein. Thus in the abstract, it is stated that "the functional capabilities of the epitope and the parent immunoglobulin are retained." Also, on page 21, it is stated "that the insertion of the desired epitope should be at a region of the nucleic acid encoding the parent immunoglobulin molecule that is not essential for expression or function of the parent immunoglobulin molecule." Furthermore, all the examples in WO 96/19584 show that intact immunoglobulin is produced following insertion of the T cell epitope.<br>
US Patent No. 7,067,110 discloses a method for eliciting an immune response against an antigen using a fusion protein of antibody which lacks an immunoglobulin variable region domain fused to the antigen by a polypeptide bond. The fusion protein retains the ability to bind to Fc.<br>
EP0759944 discloses a method of incorporating T cell epitopes within an antibody molecule that is secreted as an intact immunoglobulin protein and which can target CTL epitopes to tumours to make them better targets for CTLs.<br>
WO 00/64488 discloses that a CTL response can be raised by nucleic acid encoding a chimeric antibody having heterologous T cell epitopes inserted in the CDRs but not the variable region thereof, provided that the nucleic acid is directed for expression in B cells. As B cells cannot stimulate naive T cell responses, the vaccine described in WO 00/64488 would only be useful in boosting pre-existing T cell responses.<br>
WO 02/092126 discloses that a CTL response can be raised by a polypeptide comprising a heterologous T cell epitope and the part of human Fc which binds to the high affinity CD64 receptor. However, the present inventors have now shown that disruption of the antibody sequence by inserting a T cell epitope, for example within an inappropriate CDR or even within the variable region of an antibody, prevents association of heavy and light chain and no functional antibody is secreted. DNA encoding these mis-folded antibodies<br>
unexpectedly generates strong T cell responses. Furthermore, this is not mediated via CD64 as human lgG2 - which does not bind to mouse or human CD64 - works just as efficiently as human lgG1.<br>
In one aspect of the present invention, there is provided a nucleic acid which comprises a non-specific promoter and at least one sequence that encodes a polypeptide that has at least one heterologous T cell epitope therein but does not have any regulatory T cell epitopes.<br>
This polypeptide is preferably a homologous carrier, e.g. when used to raise a T cell response in humans it may be a human protein, or a foreign protein or human/foreign chimeric protein that has had all T regulatory epitopes identified and removed.<br>
The polypeptide is preferably one chain of a heterodimer, the heterologous T cell epitope causing disruption of the heterodimer chain such that it cannot bind or associate with the other chain of the heterodimer. Many molecules are herodimeric, with one chain being dependent upon the other for folding and then secretion. If the secondary structure is disrupted due to insertion of a heterologous T cell epitope, folding and secretion is inhibited. In certain embodiments, one chain is secreted and includes a heterologous CTL epitope, and the other chain includes a heterologous helper epitope but, due to disruption of the secondary folding, is not secreted. Thus, the nucleic acid may encode both chains of the heterodimer, wherein one chain includes a heterologous cytotoxic T cell (CTL) epitope and is secreted when expressed, and the other chain includes a heterologous helper epitope and is not secreted when expressed. Alternatively, the respective heterodimer chains may be encoded on separate nucleic acid molecules.<br>
The heterodimer may be an immunoglobulin molecule. The heavy chain of the immunoglobulin molecule may include a heterologous cytotoxic T cell (CTL) epitope and be secreted when expressed, and the light chain of the<br>
immunoglobulin molecule may include a heterologous helper epitope and not be secreted when expressed.<br>
The nucleic acid of the first aspect of the present invention encodes an polypeptide that does not include any regulatory T cell (T reg) epitopes. These polypeptides act as inert carriers for the T cell epitope(s) and may be a molecule, or part of a molecule, that can be used by the immune system to stimulate immune responses, as these molecules by definition do not express competing T reg epitopes. Suitable molecules include HLA molecules, T cell receptors, TOL receptors, TOL ligands, cytokines, cytokine receptors, chemokines, chemokine receptors. It is preferred that the molecule is an antibody or part thereof.<br>
Without wishing to be bound by theory, the present invention is based, at least in part, on the concept that a T cell response can be generated against a specific T cell epitope (such as a CTL epitope), by administration of a nucleic acid encoding a polypeptide including the T cell epitope but no regulatory T cell epitopes. It is believed that nucleic acid is either taken up by antigen presenting cells (APCs), migrates to lymph nodes and is directly presented, or is expressed to produce a polypeptide which is secreted and which is then taken up by other APCs. The former nucleic acid is suitable for stimulating helper T cell epitopes and the latter is suitable for stimulating CTL responses. The polypeptide that is encoded by the nucleic acid ideally does not have any natural T cell epitopes. Suitable polypeptides in this regard are immune molecules, such as antibodies. Antibody heavy and light chains which cannot associate so that the light chain remains in the APCs and so that the heavy chain is secreted are suitable for the practice of the present invention, although the present invention is not limited to the use of antibodies as carriers for the T cell epitopes.<br>
A suppressor T cell population was identified approximately 40 years ago, but progress was hampered by the lack of specific techniques to identify the cells<br>
and because of scientific scepticism regarding the existence of suppression. However, Sakaguchi et al resurrected interest in suppressor cells in 1995 by demonstrating that the transfer of lymphocytes depleted of CD4+CD25+T cells into athymic mice caused the development of various autoimmune diseases in the recipient mice and that reconstitution with CD4+CD25+ T cells prevented autoimmune reactions in these mice (Sakaguchi et al J.Immunol 1995;155:1151-1164).   Subsequently, numerous studies in mice and humans have shown that diverse T cell populations with regulatory activity play an important role in the suppression of immune responses (both innate and adaptive) to self (controlling self tolerance) (Sakaguchi et al J Immunol 1995;155:1151-1164) as well as foreign antigens (Shevach, Immunity 2006; 25: 195-201, Coleman etal, J. CellMol. Med. 2007; 11: 1291-1325). Treg-cell depletion in mouse models of cancer has shown to improve endogenous immune-mediated tumour rejection (Shimizu, etal, J. Immunol. 1999; 163: 5211-5218, Onizuka etal, Cancer Research 1999; 59:3128-3133) and antigen-specific anti-tumour immunity (Tanaka, et al, J. Immunother. 2002;25:207-217). In addition, Treg-cell depletion augments tumour immunotherapy including vaccination (Tanaka, etal, J. Immunother. 2002;25:207-217, Dannull etal, J. Clin. Invest. 2005;115:3623-3633) and CTLA-4 blockade (Sutmuller etal, J. Exp. Med. 2001 ;194:823-832). Furthermore, numbers of Treg-cells are increased in the peripheral blood (Woo et al, Cancer Research 2001 ;61:4766-4772, Curiel et al, Nature Medicine 2004;10:942-949, Wolf etal, Clin. Cancer Research 2003;9:606-612, Sasada etal, Cancer2003;98:1089-1099) and populate the tumour microenvironment and draining lymph nodes (Curiel etal, Nature Medicine 2004;10:942-949, Sasada etal, Cancer2003;98:1089-1099, Liyanage etal, J. Immunology 2002;169:2756-2761, Matsuura et al, Cancer 2006,106:1227-1236, Yang etal, Blood2006;107:3639-3646, Alvaro etal, Clin. Cancer Research 2005;11:1467-1473) of patients with different cancers. In patients with gastric carcinoma (Sasada etal, Cancer2003;98:1089-1099, Ichihara et al, Clinical Cancer Research 2003;9:4404-4408) and ovarian cancer (Curiel et al, Nature Medicine 2004;10:942-949), poor prognosis and decreased survival<br>
rates were associated with higher Treg-cell frequencies. Treg-cells have also been shown to suppress/inhibit the proliferation, cytokine-production (IFN, IL-2) and cytolytic activity of tumour-specific CDS" (Liyanage etal, J. Immunology2002;169:2756-2761, Piccirillo etal, J. Immunology 2001;167:1137-1140, Mempel etal, Immunity 206;25A29-141, Annacker et al, J. Immunology2001 ;166:3008-3018, Woo etal, J. Immunology 2002;168:4272-4276) and CD4+(Liyanage etal, J. Immunology 2002;169:2756-2761, Ichihara et al, Clinical Cancer Research 2003;9:4404-4408, Nishikawa et al, Blood 2005;106:1008-1011) T cells. In addition, Treg-cells can suppress the functions of dendritic cells (Romagnani etal, Eur. J. Immunol. 2005;35:2452-2458), NK cells (Ralainirina etal, J. Leukoc. Biol. 2007;81:144-153) and B cells (Lim etal, J. Immunology 2005; 175:4180-4183). Taken together, these studies suggest an important role of Treg-cells in tumour immunopathology and indicate a close correlation between Treg-cell frequencies and tumour growth.<br>
Treg-cells are divided into natural CD4+CD25+T cells and diverse populations of induced/adaptive Treg-cells (Shevach, Immunity 2006; 25:195-201, Bluestone etal, Nat. Immunol. 2005;6:345-352) (Table 1). About 5%-10% of CD4+ T cells in mice and humans are natural Treg-cells (Sakaguchi et al, Nat. Immunology 2005;6:345-352).   Natural Treg-cells develop in the thymus by strong TCR interaction with self peptide (Picca et al, Current Opinion in Immunology 2005;17:131-136, Jordan etal, Nature Immunology 2001 ;2(4):301-306, Picca etal, Immunological Reviews 2006;212:74-85), while induced Treg-cells develop from non-regulatory T cells in the periphery. This extrathymic conversion requires special immunological conditions such as continuous exposure to low dose antigen, exposure to a systemic peripheral antigen or exposure to TGFß (Shevach, Immunity 2006; 25:195-201, Akbar et al, Nat. Rev. Immunol. 2007;7:231-237). Treg-cells may mediate their suppression by one or a combination of the following mechanisms: i) cell-cell contact dependent mechanism, ii) through the secretion of immunosuppressive cytokines like IL-10 or TGFß or iii) direct<br>
killing of the target cells perforin-granzyme pathway (von Boehmer, Nature Immunology 2005;6(4):338-344).<br>
To date, very little is known about the antigen-specificity of human Treg-cells. Wang et al reported the identification of LAGE-1 -specific CD4+CD25+GITR+ functional Treg-cell clones in cancer patients (Wang et al, Immunity 2004;20:107-118). Vence et al demonstrated the presence of tumour antigen-specific CD4+ Treg-cells in the peripheral blood of metastatic melanoma patients (Vence et al, PNAS 2007;104(52):20884-20889). These Treg-cells recognised a broad range of tumour antigens, including TRP1, NY-ESO-1, gp100 and survivin. In addition, Vence et al were the first to demonstrate the presence of NY-ESO-1-specific Treg-cell epitopes within the NY-ESO-1 molecule. Furthermore, vaccination of melanoma patients with dendritic cells either loaded with synthetic peptides or tumour lysates was shown to induce increased frequencies of Treg-cells, concomitant with the expansion of tumour-specific CD8+ T cells (Chakraborty etal, Hum. Immunology 2004;65:794-802). This suggests the possibility that the vaccine contained unidentified Treg-cell epitopes as well as CD8+ T cell epitopes, which lead to the expansion of Treg-cells in vivo by ligand-specific activation through the Treg-cell T cell receptor (TCR). It is widely accepted that Treg-cells require antigen-specific activation through TCR recognition/engagement but mediate antigen-nonspecific bystander suppression (Thorton &amp; Shevach, J. Immunology 2000;164:183190). Furthermore, Li et al suggested the existence of dominant Treg epitopes within the Hepatitis C Virus core protein that stimulated HCV-specific Treg-cells in infected patients (Li et al, Immunol. Cell Biol. 2007;85(3):197-204). Collectively, these studies in addition to the recent finding that immunization of HHD transgenic mice with the anti-endothelial DNA construct C200Fc, failed to stimulate a significant Tie-2M96-specific anti-tumour immune response and the increased frequency of Tie-21. 196-specific IFN secreting cells from splenocytes of HHD mice after the depletion of CD4+CD25+Treg cells (by administration of 400ug PC61 monoclonal antibody) prior to C200Fc DNA immunization (Middleton, PhD<br>
Thesis. University of Nottingham, November 2007) indicates that the Tie-21. 196 within the DNA vaccine contains unidentified Treg-cell epitopes as well as the CD8+ epitope. This would explain the failure of the vaccine to break tolerance to the self antigen Tie-2 and to elicit anti-tumour immunity in HHD mice due to abundant antigen-specific expanded Treg-cells suppressing the cell-mediated anti-tumour immune response. There is therefore an advantage to express T effector epitopes with inert immune carriers which fail to express T reg epitopes to direct the immune response to the effector epitope and prevent stimulation of the dominant T reg response.<br>
Advantageously, the nucleic acid of the present invention includes a sequence encoding a sequence, such as a leader sequence, that allows the expressed polynucleotide to be secreted. This allows the polynucleotide to be transferred to antigen presenting cells (APCs). The sequence could be a leader sequence that is naturally expressed with the polynucleotide or could be a heterologous leader sequence, such as an immunoglobulin leader sequence, which is added. The latter is especially suitable where the polynucleotide encodes a membrane-bound molecule.<br>
According to another aspect of the present invention, there is provided a nucleic acid which comprises a non-specific promoter and at least one sequence that encodes a recombinant heavy chain of an immunoglobulin molecule, wherein the heavy chain has at least one heterologous T cell epitope therein such that the heavy chain cannot take its native conformation when the nucleic acid is expressed.<br>
The nucleic acid of the this aspect of the present invention encodes a recombinant heavy chain of an immunoglobulin molecule. The structure of such a heavy chain is known to those of skill in the art, and generally includes variable and constant regions. The heavy chain may be from an antibody. The antibody may be monoclonal or polyclonal and may be IgA, IgD, IgE, IgG or IgM, although IgG is preferred. The IgG antibody may be any IgG<br>
subclass, such as human lgG1, lgG2, lgG3 or lgG4, or mouse lgG1, lgG2a, lgG2b or lgG3. The IgG antibody may be a human lgG1 antibody having the lgG2 Fc binding domain, or a human lgG2 antibody having the lgG1 Fc binding domain. The heavy chain may have the constant region of a human antibody, and the variable or hypervariable (CDR) region of a mouse monoclonal antibody into which heterologous T cell epitopes have been inserted. The variable region other than the hypervariable region may also be derived from the variable region of a human antibody. When applied to antibodies (i.e. comprising a heavy chain and a light chain), the antibody is said to be humanised. Methods for making humanised antibodies are known in the art. Methods are described, for example, in Winter, U.S. Patent No. 5,225,539. The variable region of the heavy chain outside of the mouse hypervariable region may also be derived from a mouse monoclonal antibody. In such case, the entire variable region is derived from murine monoclonal antibody and, when applied to antibodies, the antibody is said to be chimerised. Methods for making chimerised antibodies are known in the art. Such methods include, for example, those described in U.S. patents by Boss (Celltech) and by Cabilly (Genentech). See also U.S. Patent Nos. 4,816,397 and 4,816,567, respectively.<br>
In certain embodiments, the nucleic acid of the present invention further comprises at least one sequence that encodes a light chain of an immunoglobulin molecule. Alternatively, a separate nucleic acid encoding a light chain of an immunoglobulin molecule may be provided. The light chain may have at least one heterologous T cell epitope therein. The T cell epitope may be such that the light chain cannot take its native conformation when the nucleic acid is expressed. The light chain may have any of the features described herein in respect of the heavy chain-. Accordingly, the invention also provides a nucieic acid encoding a recombinant light chain of an immunoglobulin molecule, wherein the light chain has at least one heterologous T cell epitope therein such that the light chain cannot take its native conformation when the nucleic acid is expressed. The nucleic acid<br>
may include a non-specifc promoter. Such nucleic acid(s) encode an immunoglobulin molecule, such as an antibody.<br>
Thus, according to a further aspect of the present invention, there is provided a nucleic acid which comprises a non-specific promoter and at least one sequence that encodes a recombinant immunoglobulin molecule, wherein the immunoglobulin molecule has at least one heterologous T cell epitope therein such that the immunoglobulin molecule cannot take its native conformation when the nucleic acid is expressed. Preferably, the recombinant immunoglobulin molecule, and heavy and light chains described above do not have any regulatory T cell epitopes.<br>
The invention also provides:<br>
•	a vaccine comprising a nucleic acid of the invention and an adjuvant;<br>
•	a pharmaceutical composition comprising a nucleic acid of the invention and a pharmaceutically acceptable carrier, excipient or diluent;<br>
•	a nucleic acid of the invention for use in medicine;<br>
•	the use of such a nucleic acid of the invention in the manufacture of a medicament for stimulating an immune response against at least one of the T cell epitope(s)<br>
•	a nucleic acid of the invention for stimulating an immune response against at least one of the T cell epitope(s); and<br>
•	a method for stimulating an immune response against a T cell epitope, comprising administering to a subject in need of such immune response a therapeutically effective amount of a nucleic acid of the invention.<br>
Surprisingly, the present inventors have found that antibodies, such as monoclonal antibodies, which may be human or non-human, that have predetermined T cell epitopes cloned within their variable regions, so as to disrupt the primary antibody structure, inhibit folding and/or limit secretion to<br>
either just heavy chain or very low amounts of intact antibody, stimulate strong helper and antigen-specific T cell responses. The inventors have also found that this effect can be achieved using nucleic acid encoding the heavy chain of such an antibody. It is believed that the T cell epitope is processed but not destroyed by the immunoproteosome. In certain embodiments, the invention provides a DNA vaccine presenting pre-defined T cell epitopes within denatured immunoglobulin which enhances the frequency and the avidity of the T cell response. The polypeptides encoded by the nucleic acids of the invention may be referred to herein as "Immunobodies".<br>
The finding that an immune response against a T cell epitope can be stimulated by a nucleic acid encoding at least the heavy chain of an immunoglobulin molecule into which the T cell epitope has been inserted such that the an immunoglobulin molecule cannot take its native conformation runs contrary to the expectations in the art, where it is taught that the antibody must be expressed in a functional form. For example, as discussed above, WO 96/19584 teaches that, where a nucleic acid encodes an antibody in which T cell epitopes are inserted into the CDRs of the antibody, the nucleic acid must encode a functional antibody. Similarly, EP0759944 describes a method of incorporating T cell epitopes within an antibody molecule that is secreted as an intact immunoglobulin protein. Although US patent no. 7,067,110 discloses that an immune response can be raised against an antigen by a fusion protein of antibody and the antigen, the antibody is disclosed as lacking an immunoglobulin variable region. In addition, this fusion protein will have regulatory T cell epitopes in the antigen. Thus, although the protein may stimulate an antibody response, it will not stimulate high avidity T cells responses due to regulatory T cell epitopes s in the antigen.<br>
As discussed above, WO 00/64488 discloses a nucleic acid encoding a chimeric antibody having heterologous T cell epitopes inserted in the CDRs but not the variable region thereof, which nucleic acid is directed for<br>
expression in B cells. The nucleic acid of the present invention is not directed for expression in B cells, and thus will not target B cells specifically either in vitro or in vivo. The nucleic acid of the present invention can be taken up by any antigen presenting cells, including dendritic cells, and can therefore prime naTve CTL and helper T cell responses, whereas the vaccine described in WO 00/64488 would only be useful in boosting pre-existing T cell responses.<br>
Analysis of the functional avidity of responses induced by nucleic acids in accordance with the invention demonstrated that a high avidity response can be generated when compared to immunisation with synthetic peptide. This also correlated with enhanced ability to recognise and kill tumour cells in vitro and in vivo. This observation is comparable to that documented in other studies where better anti-tumour activity is shown by high avidity TRP2 specific CTL (Zeh etal, J Immunol 1999;162:989-94, Harada et al, Immunology 2001 ;104: 67-74).<br>
The nucleic acids of the present invention have a non-specific promoter, i.e. a promoter that will promote expression of the nucleic acid but which has no specificity for cells in which expression is promoted. The promoter preferably causes expression of the nucleic acid in dendritic cells and/or keratinocytes. Examples of suitable promoters include the CMV promoter, the SV40 promoter, and other non-specific promoters known to those of skill in the art. Alternatively, the nucleic acid of the present invention may have one or more promoters that cause specific expression in dendritic cells (e.g. Cd11 b promoter) and in keratinocytes (e.g. MHCII promoter, Chin et al., 2001 J. Immunol. 167, 5549-5557).<br>
The nucleic acid of certain aspects of the invention encodes an immunoglobulin molecule, preferably an antibody that includes all of the major features of an antibody, that is to say heavy and light chains which include variable and constant regions. The antibody may be monoclonal or polyclonal and may be IgA, IgD, IgE, IgG or IgM, although IgG is preferred. The IgG<br>
antibody may be any IgG subclass, such as human lgG1, lgG2, lgG3 or lgG4, or mouse lgG1, lgG2a, lgG2b or lgG3. The IgG antibody may be a human lgG1 antibody having the lgG2 Fc binding domain. The antibody may have the constant region of a human antibody, and the variable or hypervariable region of a mouse monoclonal antibody into which heterologous T cell epitopes have been inserted. The variable region other than the hypervariable region may also be derived from the variable region of a human antibody. Such an antibody is said to be humanised. Methods for making humanised antibodies are known in the art. Methods are described, for example, in Winter, U.S. Patent No. 5,225,539. The variable region of the antibody outside of the mouse hypervariable region may also be derived from a mouse monoclonal antibody. In such case, the entire variable region is derived from murine monoclonal antibody and the antibody is said to be chimerised. Methods for making chimerised antibodies are known in the art. Such methods include, for example, those described in U.S. patents by Boss (Celltech) and by Cabilly (Genentech). See also U.S. Patent Nos. 4,816,397 and 4,816,567, respectively.<br>
The nucleic acid of certain aspects of the invention is such that the heavy chain, light chain or immunoglobulin molecule expressed therefrom has at least one heterologous T cell epitope therein so that the heavy chain, light chain or the immunoglobulin molecule cannot take its native conformation. The T cell epitope may disrupt the expressed protein so that the heavy chain or the immunoglobulin molecule can no longer bind to its antigen, so that the heavy and light chains (where present) can no longer associate, or so that the heavy chain or immunoglobulin molecule cannot be secreted properly, for example. The disruption may be in the tertiary structure of the immunoglobulin molecule which may prevent formation of the disulphide bonds.<br>
As discussed in more detail below, where the immunoglobulin molecule is an antibody, the T cell epitope(s) may be inserted into or substituted for the<br>
CDR1 and CDR2 regions of the antibody. CDR1 and CDR2 form part of the antibody ß sheet conformation and are partially submerged within the folded molecule. Any change to their length, amino acid composition or charge will disrupt this structure and prevent heavy and light chain folding and association. CDRH3 is exposed on the surface of the immunoglobulin molecule and is therefore more permissive of alterations. In the present invention, it is preferred if CDR1 and/ or CDR2 are substituted with T cell epitope(s). Indeed, in certain embodiments, loss of framework regions at the CDRH junctions completely disrupts antibody folding yet insertion of epitopes in these regions gives good T cell responses.   Incorporation of any epitope within the CDRH1 (5 amino acids in length) or CDRH2 (17 amino acids in length) causes sufficient disruption to allows secretion of heavy chain but only very low amounts of intact antibody, even if the light chain has its native sequence. This shows that the secondary structure is important for heavy and light chain pairing. Incorporation of any epitope within CDRL1 of the light chain results in low level secretion of light chain, even if there is only a single epitope incorporated into the CDRH3 of the heavy chain.<br>
"Heterologous T cell epitope" is intended to mean a T cell epitope which is heterologous to the antibody. For example, a heterologous T cell epitope may be one which was not previously present in the antibody. The heterologous T cell epitope may be inserted as a whole, although it may be made up from an inserted amino acid sequence, together with flanking amino acids of the second portion. This is to ensure that the inserted epitope has a similar processing profile in the heterologous nucleic as from the original antigen. One or more CTL/helper epitopes can be inserted within the same variable region.<br>
The T cell epitope(s) can be inserted anywhere in the heavy chain or light chain. It is preferred if the or each epitope is inserted in the variable region of the heavy chain and/or light chain, although nucleic acids encoding heavy chains or antibodies having T cell epitopes inserted in just the constant<br>
region, or in the constant region and the variable region of a heavy chain and/or light chain are included within the invention. In the nucleic acids of the present invention, the sequence(s) encoding the T cell epitopes may be inserted into (i.e. added to) the sequence encoding the heavy chain and/or light chain, or may be substituted into the sequence encoding the heavy chain and/or light chain.<br>
In the variable region, the T cell epitope(s) may be inserted in, or substituted for, any one or more the CDRs of the heavy and/or light chain, i.e. L1, L2, L3, H1, H2, or H3. Of these, L1, H1 and H2 are currently preferred. In certain embodiments, the T cell epitopes are inserted in, or substituted for, CDRL1 and/or H1 and/or H2. Preferably the incorporated T cell epitopes are not of similar size and charge to the amino acids of the original CDR of the antibody so that the antibody does not take its native conformation, e.g. does not fold and is not secreted correctly. Alternatively or additionally, they may be inserted in, or substituted for, the framework region surrounding the CDRs.<br>
The inserted T cell epitopes are preferably cytotoxic T cell (CTL or CD8) epitopes.   Alternatively or additionally, helper T cell (CD4) epitopes may be inserted. T cell epitopes can be predicted using known T cell algorithms or synthesised as peptides and screened using standard T cell assays. The T cell epitopes may have an amino acid length in the range of from 5 to 50, 7 to 40, 8 to 30 or 9 to 20 amino acids, such as 9,10, 11, 12,13,14,15, 16, 17, 18, 19, 20, 21, 22, 23, 24, 25, 26, 27, 28, 29 or 30 amino acids. The epitopes may inserted using complementary oligonucleotides that encode the antigenic epitopes, which are annealed and cloned into specific sites of the antibody framework where CDR's (or other region) have been replaced with unique restriction enzyme sites. The ability of the recombinant antibody to stimulate helper and cytotoxic T cell responses can be screened as exemplified herein.<br>
Various combinations are possible within the present invention. In certain embodiments, one or a plurality of CD8 epitopes is/are inserted in, and/or<br>
substituted for, the CDR H1 and/or H2, or in the non-CDR variable region, of the heavy chain or the antibody. Additionally or alternatively, one or a plurality of CD4 epitopes may be inserted in, and/or substituted for, the CDR L1, or in the non-CDR variable region, of the light chain or the antibody. Where there is a plurality of T cell epitopes, the T cell epitopes may be the same or different. Those of skill in the art will appreciate that numerous combinations are possible, including:<br>
•	a CD8 epitope in CDR H1, and a CD4 epitope in CDR L1;<br>
•	a CD8 epitope in CDR H2, and a CD4 epitope in CDR L1;<br>
•	a CD8 epitope in CDR H1 and CDR H2, and a CD4 epitope in CDR L1;<br>
•	2 CD8 epitopes in CDR H1, and a CD4 epitope in CDR L1;<br>
•	2 CD8 epitopes in CDR H2, and a CD4 epitope in CDR L1; etc<br>
Nucleic acids of the present invention can incorporate multiple T cell epitopes from a single target antigen that can bind to the majority of both class I and class II MHC molecules. This may create a vaccine that can be used in widespread population vaccination. Alternatively nucleic acids useful in the invention can incorporate multiple T cell epitopes from multiple target antigens that can bind to the most common class I and class II phenotypes. This may create a vaccine that may prevent selection of antigen loss variants. Target antigens may be from a single pathogen or tumour type or may be selected to give an immune response against a variety of pathogens or cancers. Nucleic acids useful in the present invention targeting specific common HLA phenotypes may incorporate numerous T cell epitopes from a wide variety of cancers and/or pathogens, providing a single vaccine to prevent disease.<br>
Any T cell epitope can be inserted, provided that it stimulates helper and/or cytotoxic T cell responses. T cell epitopes from pathogens such as HIV, Hepatitis C and other infections that require CTLs to clear latent infections may be used, although it is preferred if the epitope is a "self-epitope", i.e. associated with a condition/disorder associated with cell proliferation such as<br>
 <br>
cancer. Preferably, the T cell epitope is such that the heavy chain or antibody cannot fold correctly and be secreted. It is therefore preferred if the inserted epitopes are of not of similar size and amino acid composition to the original variable region. The nucleic acid may have a plurality of different T cell epitopes so as to generate a wide variety of T cell responses. The nucleic acid may incorporate multiple epitopes from a single antigen, thereby ensuring that the majority of individuals with different HLA types respond to the single vaccine. Alternatively, multiple T cell epitopes from multiple antigens targeting a restricted spectrum of HLA types could be used. The nucleic acid molecules of the invention may include a variety of antigens from a single pathogen or cancer type or they could include disparate antigens targeting a wide range of solid tumours or pathogens. The nucleic acid molecules of the invention may even be designed to target different cell populations within a tumour, such as tumour epithelial and endothelial antigens.<br>
Surprisingly the inventors have found that, when T cell epitopes were inserted into structurally confined CDRs or non-CDR regions of the heavy chain, they gave superior CTL responses. This appears to be due to secretion of large amounts of heavy chain, which can only weakly associate with light chain due to the insertion of bulky epitopes into their variable regions. This is contrary to dogma, which states 'that only proteins synthesised endogenously by antigen presenting cells are presented on MHC class I molecules and recognised by CTLs" - WO 96/19584.   Uptake of exogenous antigen and presentation on MHC class I is a process known as cross presentation and usually requires uptake via specific receptors. This could be the CD64 receptor for human Fc1 antibodies. However, it would be predicted that large amounts of intact antibody or antigen-antibody complexes would be better at targeting this receptor. In contrast, the results presented herein clearly show very low levels of intact antibody and large amounts of free heavy chain, which should not bind to CD64, give superior CTL responses. Indeed, it is shown herein that CTL responses can be stimulated when CTL epitopes are inserted in<br>
antibodies which cannot bind to CD64, such as lgG2 antibodies or lgG1 molecules with their CD64 binding region replaced with the non-CD64 binding region from lgG2.<br>
The nucleic acid encoding the heavy chain preferably includes a leader sequence to allow it to be secreted. The present inventors have found that, if the leader sequence of the heavy chain is removed to prevent secretion and allow more endogenous protein to be produced, this reduces the CTL response. This is completely contrary to expectations. Whilst not wishing to be bound by theory, the inventors believe that this implies that the nucleic acid is expressed in non-antigen presenting cells, which secrete high levels of heavy chain and low amounts of native protein which can then be taken up by antigen presenting cells. Alternatively, the nucleic acid may directly transfect antigen presenting cells which migrate to the draining lymph node where they secrete low amounts of native protein and large amounts of heavy chain that is taken up by the same or adjacent antigen presenting cells and presented on MHC class I to naive CTLs. Therefore, for a nucleic acid vaccine to stimulate efficient CTL responses, it must preferably encode CTL epitopes within a protein that is secreted at very low levels and/or at the same time secretes large amounts of denatured protein. However, a CTL response cannot mature to a high affinity memory response in the absence of helper responses. Therefore, it is preferred if T helper epitopes are inserted into the heavy chain or the immunoglobulin molecule, preferably into the variable region of antibody light chains. Again, surprisingly and in contrast to the dogma which states "only proteins taken up exogenously by the target cells are presented by MHC class II molecules and recognised by helper T cells", light chain was only secreted in very low amounts. Removal of the leader sequence to prevent secretion of the light chain had no effect on the helper responses. Accordingly, the nucleic acid of the present invention may or may not have a leader sequence for the light chain of the antibody. These results imply that the nucleic acid is taken up by the antigen presenting cells which present the T helper epitopes in the context of MHC class II from<br>
endogenously-synthesised protein, possibly by autophagy. For helper T cells to assist CTL responses, both the T cell epitopes they recognise must be presented on the same antigen presenting cells in a process known as linked T cell help. This implies that the antigen presenting cell synthesising the light chain, encoded by the nucleic acid, must either also synthesise, secrete and cross present the CTL epitopes themselves or take up heavy chain from an adjacent APC.<br>
The present invention also provides isolated dendritic cells which present the heterologous helper T cell epitopes on MHC class II from endogenously-produced light chain and heterologous CTL epitopes from cross-presented heavy chain.   Such dendritic cells may be used in the therapies described herein.<br>
Nucleic acids of the present invention can make existing T cell epitopes more immunogenic by encoding a denatured antibody which leads to an increase in both the frequency and avidity of T cell responses.<br>
The nucleic acid of the invention may be DNA, cDNA, or RNA such as mRNA, obtained by cloning or produced wholly or partly by chemical synthesis. For therapeutic use, the nucleic acid is preferably in a form capable of being expressed in the subject to be treated.<br>
The nucleic acid of the present invention may be recombinant or provided as an isolate, in isolated and/or purified form. It may be free or substantially free of nucleic acid flanking the gene in the human genome, except possibly one or more regulatory sequence(s) for expression. Where nucleic acid according to the invention includes RNA, reference to the sequences shown herein should be construed as reference to the RNA equivalent, with U substituted forT.<br>
Nucleic acids of the present invention can be readily prepared by the skilled person, for example using the information and references contained herein and techniques known in the art (for example, see Sambrook, Fritsch and Maniatis, "Molecular Cloning", A Laboratory Manual, Cold Spring Harbor Laboratory Press, 1989, and Ausubel et al, Short Protocols in Molecular Biology, John Wiley and Sons, 1992), given the nucleic acid sequences and clones available. These techniques include (i) the use of the polymerase chain reaction (PCR) to amplify samples of such nucleic acid, e.g. from genomic sources, (ii) chemical synthesis, or (iii) preparing cDNA sequences. DNA encoding the polypeptide may be generated and used in any suitable way known to those of skill in the art, including by taking encoding DNA, identifying suitable restriction enzyme recognition sites either side of the portion to be expressed, and cutting out said portion from the DNA. The portion may then be operably linked to a suitable promoter in a standard commercially available expression system. Another recombinant approach is to amplify the relevant portion of the DNA with suitable PCR primers. Modifications to the sequences can be made, e.g. using site directed mutagenesis, to lead to the expression of modified peptide or to take account of codon preferences in the host cells used to express the nucleic acid.<br>
In order to obtain expression of the nucleic acid sequences, the sequences can be incorporated into a vector having one or more control sequences operably linked to the nucleic acid to control its expression. The vectors may include other sequences such as promoters or enhancers to drive the expression of the inserted nucleic acid, nucleic acid sequences so that the polypeptide is produced as a fusion and/or nucleic acid encoding secretion signals so that the polypeptide produced in the host cell is secreted from the cell. If desired, polypeptide can then be obtained by transforming the vectors into host cells in which the vector is functional, culturing the host cells so that the polypeptide is produced and recovering the polypeptide from the host cells or the surrounding medium. Prokaryotic and eukaryotic cells are used for this purpose in the art, including strains of E. coli, yeast, and eukaryotic cells such<br>
as insect cells, and animal cells, for example, COS, CHO cells, Bowes Melanoma and other suitable human cells. Where the present invention relates to nucleic acid(s) encoding the heavy and light chains of an antibody, the respective nucleic acids may be present in the same expression vector, driven by the same or different promoters, or in separate expression vectors.<br>
The nucleic acids of the present invention may be used to stimulate an immune response against at least one of the T cell epitope(s) in a patient such as a mammal, including human. Helper and/or cytotoxic T cell responses may be stimulated. The T cell response against a particular epitope obtained by the present invention may have a higher avidity than that obtained by immunisation with the same epitope as a simple peptide, or by immunisation with the same epitope encoded within an antigen either as a peptide or a nucleic acid. The nucleic acids of the invention may be administered as a combination therapy, i.e. a nucleic acid encoding the light chain and nucleic acid encoding the heavy chain. The nucleic acid may be administered intravenously, intradermally, intramuscularly, orally or by other routes. Intradermal or intramuscular administration is preferred because these tissues contain dendritic cells<br>
As used herein, the term "treatment" includes any regime that can benefit a human or non-human animal. The treatment may be of an inherited or acquired disease. Preferably, the treatment is of a condition/disorder associated with cell proliferation such as cancer or of infectious disease. Examples of types of cancer that can be treated with the nucleic acid include any solid tumour, colorectal cancer, lung, breast, gastric, ovarian, uterine, liver, kidney, pancreatic, melanoma, bladder, head and neck, brain, oesophageal, pancreatic, and bone tumours, as well as soft tissue cancers, and leukaemias. Examples of infectious diseases that can be treated with the nucleic acid include infection with HIV, Hepatitis C, or any chronic infection that requires T cell immunity for clearance.<br>
The nucleic acid may be employed in combination with a pharmaceutically acceptable carrier or carriers. Such carriers may include, but are not limited to, saline, buffered saline, dextrose, liposomes, water, glycerol, ethanol and combinations thereof.<br>
Adjuvants may be employed to facilitate stimulation of the host's immune response, and may include, aluminium hydroxide, lysolecithin, pluronic, polyols, polyanions, peptides, proteins and oil emulsions.<br>
The nucleic acids useful in the invention can be formulated in pharmaceutical compositions. These compositions may comprise, in addition to one of the above substances, a pharmaceutically acceptable excipient, carrier, buffer, stabiliser or other materials well known to those skilled in the art. Such materials should be non-toxic and should not interfere with the efficacy of the active ingredient. The precise nature of the carrier or other material may depend on the route of administration, e.g. intradermal, oral, intravenous, cutaneous or subcutaneous, nasal, intramuscular, intraperitoneal routes. The formulation is preferably nucleic acid as a stable dry powder precipitated onto the surface of microscopic gold particles and suitable for injection via a gene gun. The formulation may be suitable for intradermal or intramuscular administration using electroporation.<br>
The compositions comprising, or for the delivery of, nucleic acids are preferably administered to an individual in a "therapeutically effective amount", this being sufficient to show benefit to the individual. The actual amount administered, and rate and time-course of administration, will depend on the nature and severity of what is being treated. Prescription of treatment, e.g. decisions on dosage etc, is within the responsibility of general practitioners and other medical doctors, and typically takes account of the disorder to be treated, the condition of the individual patient, the site of delivery, the method of administration and other factors known to practitioners. The nucleic acids of the invention are particularly relevant to the treatment of existing cancer<br>
and in the prevention of the recurrence of cancer after initial treatment or surgery. Examples of the techniques and protocols mentioned above can be found in Remington's Pharmaceutical Sciences, 16th edition, Oslo, A. (ed), 1980.<br>
Preferably, the nucleic acid of the invention stimulate helper and/or cytotoxic T cells that can significantly inhibit the growth of tumour cells when administered to a human in an effective amount. The optimal dose can be determined by physicians based on a number of parameters including, for example, age, sex, weight, severity of the condition being treated, the active ingredient being administered and the route of administration. For example, a dose of 1-1000µg of DNA is sufficient to stimulate both helper and cytotoxic T cell responses.<br>
The nucleic acids of the invention may be administered along with additional pharmaceutically acceptable ingredients. Such ingredients include, for example, immune system stimulators.<br>
A composition may be administered alone or in combination with other treatments, either simultaneously or sequentially dependent upon the condition to be treated. Other cancer treatments include other monoclonal antibodies, other chemotherapeutic agents, other radiotherapy techniques or other immunotherapy known in the art. One particular application of the compositions of the invention are as an adjunct to surgery, i.e. to help to reduce the risk of cancer reoccurring after a tumour is removed.<br>
Injections (id) may be the primary route for therapeutic administration of the nucleic acid of this invention.<br>
The nucleic acids may be administered in a localised manner to a tumour site or other desired site or may be delivered in a manner in which it targets tumour or other cells.<br>
The dose of nucleic acid will be dependent upon the properties of the agent employed, e.g. its binding activity and in vivo plasma half-life, the concentration of the polypeptide in the formulation, the administration route, the site and rate of dosage, the clinical tolerance of the patient involved, the pathological condition afflicting the patient and the like, as is well within the skill of the physician. For example, doses of 100µg of nucleic acid per patient per administration are preferred, although dosages may range from about 10µg to 1 mg per dose. Different dosages are utilised during a series of sequential inoculations; the practitioner may administer an initial inoculation and then boost with relatively smaller doses of nucleic acid.<br>
In certain other embodiments, the present invention relates to a method of engineering T cell epitopes from target antigens into the variable regions of antibodies, and the use of such engineered antibodies as vaccines to stimulate both helper and cytotoxic T cell responses.<br>
A further aspect of the present invention provides a host cell containing a nucleic acid as disclosed herein. The nucleic acid of the invention may be integrated into the genome (e.g. chromosome) of the host cell. Integration may be promoted by inclusion of sequences that promote recombination with the genome in accordance with standard techniques. The nucleic acid may be on an extra-chromosomal vector within the cell, or otherwise identifiably heterologous or foreign to the cell.<br>
A still further aspect provides a method, which comprises introducing the nucleic acid of the invention into a host cell. The introduction, which may (particularly for in vitro introduction) be generally referred to without limitation as "transformation", may employ any available technique. For eukaryotic cells, suitable techniques may include calcium phosphate transfection, DEAE-Dextran, electroporation, liposome-mediated transfection and transduction using retrovirus or other virus, e.g. vaccinia or, for insect cells, baculovirus.<br>
For bacterial cells, suitable techniques may include calcium chloride transformation, electroporation and transfection using bacteriophage. As an alternative, direct injection of the nucleic acid could be employed.<br>
Marker genes such as antibiotic resistance or sensitivity genes may be used in identifying clones containing nucleic acid of interest, as is well known in the art.<br>
The introduction may be followed by causing or allowing expression from the nucleic acid, e.g. by culturing host cells (which may include cells actually transformed although more likely the cells will be descendants of the transformed cells) under conditions for expression of the gene, so that the encoded polypeptide (or peptide) is produced. If the polypeptide is expressed coupled to an appropriate signal leader peptide it may be secreted from the cell into the culture medium. Following production by expression, a polypeptide or peptide may be isolated and/or purified from the host cell and/or culture medium, as the case may be, and subsequently used as desired, e.g. in the formulation of a composition which may include one or more additional components, such as a pharmaceutical composition which includes one or more pharmaceutically acceptable excipients, vehicles or carriers (e.g. see below).<br>
The present inventon also provides a method for identifying T cell epitopes in<br>
a candidate antigen, comprising:<br>
depleting T regulatory cells in a non-human animal; immunising the non-human animal with a candidate antigen; and screening to see whether a T cell response is raised against either<br>
peptides to predicted epitopes in the candidate antigen or all the possible<br>
overlapping peptides within the candidate antigen.<br>
The method may be carried out in a non-human animal, such as a mouse or a rat. T regulatory cells can be depleted in the non-human animal using anti-<br>
CD25 antibodies, which optionally may be conjugated with toxins such as Ontak, or by chemotherapy such as cyclophosphamide which preferentially kills T regulatory cells. Once T regulatory cells have been depleted, the non-human animal may be immunised with DNA encoding the candidate antigen, or by the candidate antigen itself. It is preferred that the candidate antigen is provided as an antigen-Fc fusion protein. In the screening step, the peptide against which any T cell response stimulated in the non-human animal is identified. This can be done in vitro using a technique such as ELISPOT. If a T cell response is elicited to a candidate epitope, this epitope can be used to immunise a non-human animal. If this peptide elicits a T cell response, the avidity and frequency can be enhanced by encoding the epitope within a nucleic acid in accordance with the present invention. This method can allow the identification of T cell epitopes that are processed by the immunoproteosome.<br>
Preferred features of each aspect of the invention are as for each of the other aspects mutatis mutandis. The prior art documents mentioned herein are incorporated to the fullest extent permitted by law.<br>
The invention will now be described further in the following non-limiting examples Reference is made to the following drawings:<br>
Figure 1: Map depicting features of the heavy chain vector pOrigHIB The wild type de-immunised heavy variable region of antibody SC100 was cloned using Hindlll/Afel inframe with the human lgG1 Fc constant region. The Fc region comprises the CH1, CH2, CH3 domains and the hinge region. High-level expression in mammalian cells is driven from the human cytomegalovirus immediate early promoter. BGH polyadenylation signals downstream of the Orig HIB human lgG1 chain to ensure mRNA stability and effective termination. EM7 is a bacterial promoter that controls expression of the zeocin resistance gene allowing antibiotic selection in E.coli while the SV40 early promoter upstream of the resistance gene allows selection in<br>
mammalian cells. SV40 polyadenylation signals downstream of the resistance gene in order to direct proper processing of the 3'end of the zeor mRNA. The vector also contains within its backbone the ColE1 origin of replication for propagation in bacteria. Complimentary determining DNA sequences were effectively removed and exchanged for restriction sites RE1, RE2 and RE3 (Fspl, Mscl and Srf I respectively) singly and in combination.<br>
Figure 2: Map depicting features of the heavy chain vector pOrigLIB The wild type de-immunised light variable region of antibody SC100 was cloned using BamHI/BsiWI inframe with the human kappa constant region. High-level expression in mammalian cells is driven from the human cytomegalovirus immediate early promoter. BGH polyadenylation signals downstream of the Orig LIB chain to ensure mRNA stability and effective termination. The vector also includes the ColE1 origin of replication and the antibiotic resistance gene for ampicillin allowing propagation and selection in bacteria. Complimentary determining regions were effectively removed and exchanged for restriction sites RE4, RE5 and RE6 (EcoRV, Ssp I and Hpa I respectively) singly and in combination.<br>
Figure 3: Sequence of the wild type Immunobody chimeric heavy chain. Nucleotide and on translation amino acid sequence are illustrated for the full length chimeric igG1 heavy chain. Locations of CDR's are within boxes defined by the kabat numbering scheme. The stop codon is depicted by a red astrix. The HindIII/Afe I restriction sites are highlighted utilised in transfer of the variable heavy region.<br>
Figure 4: Sequence of the wild type Immunobody chimeric kappa chain Nucleotide and on translation amino acid sequence are illustrated for the full length chimeric kappa chain. Locations of CDR's are within boxes defined by the kabat numbering scheme. The stop codcn is depicted by an asterisk. The BamHIBs/WI restriction sites utilised in transfer of the variable light region are highlighted.<br>
Figure 5: Overlapping extension PCR<br>
CDR's were removed and replaced with unique restriction sites by overlapping PCR. The forward primers H1, H2. H3, L1, L2 and L3 (Table 2) were designed to replace CDR1, 2 and 3 within the heavy and light chain variable region respectively. Each primer contained, centrally located, the chosen unique enzyme recognition sequence devoid of the CDR sequence to be removed (green section) and flanked by 10-20bp of wild type sequence. The forward primers were used in a first round of PCR in conjunction with a general reverse primer, huHeClonR or huLiClonR (Table 2), that anneals to the human heavy and light constant domains within the wild type constructs pOrigHIB and pOrigLIB respectively. The fragment generated does not contain wild type CDR sequence (red section), but is effectively exchanged for the restriction site. In order to amplify the entire variable heavy and light region, a second round of PCR is required using the PCR product generated from the first round as a reverse primer with the general CMV forward primer that anneals to the CMV promoter within the single plasmids. Second round PCR products were subcloned into pCR2.1 (Invitrogen) and, after sequence confirmation, the heavy/light (VH and VL) variable regions containing H1, H2, H3, L1, L2 and L3 versions singly, in combination and together were inserted back into the single constructs pOrigHIB and pOrigLIB, exchanging the wild type regions using Hindlll/Afel and BamHI/BsiWI respectively.<br>
Figure 6: Sequence of the ImmunoBody heavy chain variable region Nucleotide and amino acid sequence of the heavy variable region where CDR's have been replaced with their corresponding enzyme site H1, H2 and H3, singly in combination and together. The unique restriction enzyme sites are highlighted. CDR1, 2 and 3 were replaced with Fspl, Msc\ and SrfI respectively<br>
Figure 7: Sequence of the ImmunoBody kappa chain variable region Nucleotide and amino acid sequence of the heavy variable region where CDR's have been replaced with their corresponding enzyme site L1, L2 and L3, singly in combination and together. The unique restriction enzyme sites are highlighted. CDR1, 2 and 3 were replaced with EcoRN, SspI and HpaI respectively.<br>
Figure 8: Map depicting features of the double expression vector pDCOrig Once all epitopes have been incorporated into the variable heavy and variable light sites within the single vectors, they are transferred into the double expression vector utilising as highlighted Hindlll/Afel and BamHI/BsiWI in frame with their respective human constant regions. The Fc region of the heavy chain comprises of the CH1, CH2, CH3 domains and the hinge region. High-level expression of both the heavy and light chains in mammalian cells is driven from the human cytomegalovirus immediate early promoter. BGH polyadenylation signals downstream of both chains to ensure mRNA stability and effective termination. EM7 is a bacterial promoter that controls expression of the zeocin resistance gene allowing antibiotic selection in E.coli while the SV40 early promoter upstream of the resistance gene allows selection in mammalian cells. SV40 polyadenylation signals downstream of the resistance gene in order to direct proper processing of the 3'end of the zeor mRNA. The vector also contains within its backbone the ColE1 origin of replication for propagation in bacteria.<br>
Figure 9: Sequence of the immunobody IB15 heavy chain containing a stop codon preventing synthesis of the FC region<br>
Nucleotide and amino acid sequence of the chimeric heavy chain, pDCOrig IB15 CH1 stop. A stop codon was inserted by site directed mutagenesis after the CH1 domain of the human igG1 Fc constant region as depicted by a asterisk. Nucleotides and amino acids in bold represent the CH1 domain. Amino acids within boxes represent the GP100210M epitope in H1 (TIMDQVPFSV) and the TRP2 epitope in H2 (SVYDFFVWL). The HindIII/Afe<br>
I restriction sites are highlighted utilised in transfer of the variable heavy region from the single construct.<br>
Figure 10: Nucleotide and amino acid sequence of the DCIB15 heavy variable region without a leader.<br>
The leader was removed by PCR using the forward primer pOrig heavy no leader with the reverse primer huHeClonR (Table 2) that binds to the human lgG1 CH1 domain effectively re amplifying the heavy variable (VH) region. After sequence confirmation, the VH region minus leader was cloned back into the double expression construct DCIB15 using Hindlll/Afel inframe with the human lgG1 constant region. Amino acids within boxes represent the GP100210M epitope in H1 (TIMDQVPFSV) and the TRP2 epitope in H2 (SVYDFFVWL). The HindIII/Afe I restriction sites utilised in transfer of the variable heavy region are highlighted.<br>
Figure 11: Nucleotide and amino acid sequence of the DCIB15 kappa variable region without a leader<br>
The leader was removed by PCR using the forward primer pOrig light no leader with the reverse primer huLiClonR (Table 2) re amplifying the light variable (VL) region. After sequence confirmation, the VL region minus leader was cloned back into the double expression construct DCIB15 using BamHI/BsiWI in frame with the human kappa constant region. Amino acids within boxes represent the HepB CD4 epitope in L1 (TPPAYRPPNAPIL). The BamHI/BsiW/I restriction sites are highlighted utilised in transfer of the variable light region.<br>
Figure 12: Sequence of human lgG2 constant region Nucleotide and amino acid sequence of the heavy human lgG2 constant region amplified. The Afe\ and Sap\ restriction sites are highlighted utilised in transfer and replacement of the huigGI constant region in the double expression vector DCIB15.<br>
Figure 13: Sequence of human igG3 constant region Nucleotide and amino acid sequence of the heavy human igG2 constant region amplified. The Afe\ and Sapl restriction sites are highlighted utilised in transfer and replacement of the huigGI constant region in the double expression vector DCIB15<br>
Figure 14: Human isotypes of the immunobody double expression vector<br>
A Map of the double expression vector pDCOriglB15 huigG2.<br>
B Map of the double expression vector pDCOriglB15huigG3.<br>
The HindIII/Afe I and BamHI/Bs/WI restriction sites utilised in transfer of the<br>
variable heavy and light region are highlighted.<br>
Figure 15: Sequence of DCIB66 heavy chain containing the G2 motif Nucleotide and amino acid sequence of the chimeric heavy chain. The amino acids E233 L234 L235 within a critical binding motif for interaction with the high affinity FcR1 (CD64) were substituted with P233 V234 A235 from human igG2 highlighted in bold within a box. Other amino acids within boxes represent the GP100210M epitope in H1 (TIMDQVPFSV) and the TRP2 epitope in H2 (SVYDFFVWL).<br>
The Age\IAhdI sites highlighted were used in transfer of the section containing the substitutions into pDCOriglB15 huigGI. The HindIII/Afe I restriction sites utilised in transfer of the variable heavy region are depicted in bold.<br>
Figure 16: Sequence of DCIB67 heavy chain containing the G1 binding motif Nucleotide and amino acid sequence of the chimeric heavy chain. The amino acids P233 V234 A235 within the human lgG2 constant region were substituted with the critical binding motif for interaction with the high affinity FcR1 (CD64) E233 L234 L235 G236 from human lgG1 highlighted in bold within a box. Other amino acids within boxes represent the GP100210M epitope in H1 (TIMDQVPFSV) and the TRP2 epitope in H2 (SVYDFFVWL).<br>
The AgeI/AhdI sites highlighted were used in transfer of the section containing the substitutions into pDCOriglB15 huigG2. The HindIII/Afe I restriction sites utilised in transfer of the variable heavy region are depicted in bold.<br>
Figure 17: Murine lgG2a Immunobody expression vectors Maps of (A) Single chain pMoOrigHIB vector, (B) Double expression vector DCIB53 containing the GP100210M epitope in H1 (TIMDQVPFSV), the TRP2 epitope in H2 (SVYDFFVWL) and the HepB CD4 epitope in L1 (TPPAYRPPNAPIL) and (C) Double expression vector DCIB63 containing the HLA-DR7 restricted gp100 CD4 epitope ( GTGRAMLGTHTMEVTVYH) in H1, the TRP2 epitope (SVYDFFVWL) in H2 and the HLA-DR4 restricted gp100 CD4 epitope in H3 (WNRQLYPEWTEAQRLD). Restriction sites utilised are depicted.<br>
Figure 18: Schematic diagram to depict construction of the regulatory compliant plasmid pVAXDCIB54<br>
The heavy single chain vector pVaxlB54 HIB (A) was linearised using Nrul. The light chain expression cassette from pOrigLIB (B) was excised using Nrul and Hpal and cloned into the linearised plasmid to generate the double expression vector pVaxDCIB54 (C). The HindIIII/Afe I and BamHI/Bs/WI restriction sites utilised in transfer of the variable heavy and light region are highlighted.<br>
Figure 19: Sequence of DCIB15<br>
Nucleotide and amino acid sequence of the heavy and light variable regions cloned in frame with the human lgG1 Fc and kappa constant regions within the expression vector pDCOrig. Amino acids within boxes represent the GP100210M epitope in H1 (TIMDQVPFSV), the TRP2 epitope in H2 (SVYDFFVWL) and the HepB CD4 epitope in L1 (TPPAYRPPNAPIL). The HindIII/Afe I and BamHI//BsiWI restriction sites utilised in transfer of the variable heavy and light region from the single construct are highlighted.<br>
Figure 20: ImmunoBody constructs produce low levels of intact protein.<br>
A,	quantification of the level of ImmunoBody heavy chain by sandwich Elisa<br>
from the supernatant of CHO-S cells transfected with ImmunoBody containing<br>
gp100/H1, TRP2/H2 and HepB CD4/L1 (DCIB15). Supernatant was used<br>
neat and diluted 1 in 3, 1 in 10 and 1 in 30 in media and compared to a<br>
human IgG positive control.<br>
B,	Analysis of purified ImmunoBody containing gp100/H1, TRP2/H2 and<br>
HepB help/L1 (DCIB15) by sandwich Elisa compare to a positive control.<br>
C and D, Determination of expression of heavy chain and intact ImmunoBody from supernatant of CHO-S transfectants by sandwich Elisa. Plates were coated with an anti-human Fc specific antibody. To detect heavy chain an anti-human IgG Fc specific HRP antibody was used and to detect intact ImmunoBody an anti-human kappa chain specific HRP antibody was used. E, Determination of heavy chain, light chain and intact ImmunoBody from supernatant of CHO-S transfectants (DCIB15, DCIB31, DCIB32, DCIB36, DCIB48, DCIB49, DCIB52, DCIB54) by sandwich Elisa. Plates were coated with an anti-human Fc specific antibody or anti-human kappa chain antibody. To detect heavy chain an anti-human IgG Fc specific HRP antibody was used in combination with the anti-human Fc specific coating antibody. To detect intact ImmunoBody an anti-human kappa chain specific HRP antibody was used in combination with anti-human Fc specific coating antibody. To detect light chain anti-human kappa chain specific HRP antibody was used in combination with the anti-human kappa chain specific antibody.<br>
Figure 21: Sequence of DCIB24<br>
Nucleotide and amino acid sequence of the heavy and light variable regions cloned in frame with the human lgG1 Fc and kappa constant regions within the expression vector pDCOrig. Amino acids within boxes represent the ovalbumin epitope in H2 (SIINFEKL) and the HepB CD4 epitope in L1 (TPPAYRPPNAPIL). The HindIII/Afe I and BamHI/BsiWI restriction sites utilised in transfer of the variable heavy and light region from the single construct are highlighted.<br>
Figure 22 Sequence of DCIB25<br>
Nucleotide and amino acid sequence of the heavy and light variable regions cloned in frame with the human IgGI Fc and kappa constant regions within the expression vector pDCOrig. Amino acids within boxes represent the GP100210M epitope in H1 (TIMDQVPFSV), the TRP2 epitope in H2 (SVYDFFVWL) and the HepB CD4 epitope in L3 (TPPAYRPPNAPIL). The HindIII/Afe I and BamHI/Bs/WI restriction sites utilised in transfer of the variable heavy and light region from the single construct are highlighted.<br>
Figure 23: Sequence of DCIB31<br>
Nucleotide and amino acid sequence of the heavy and light variable regions<br>
cloned inframe with the human IgGI Fc and kappa constant regions within the<br>
expression vector pDCOrig. Amino acids within boxes represent the TRP2<br>
epitope (SVYDFFVWL)<br>
in H3. The HindWUAfe I and Ba/nHI/Bs/WI restriction sites utilised in transfer of<br>
the variable heavy and light region from the single construct are highlighted.<br>
Figure 24: Sequence of DCIB32<br>
Nucleotide and amino acid sequence of the heavy and light variable regions cloned inframe with the human IgGI Fc and kappa constant regions within the expression vector pDCOrig. Amino acids within boxes represent the TRP2 epitope (SVYDFFVWL) in H3 and the HepB CD4 epitope in L3 (TPPAYRPPNAPIL). The HindWUAfe I and BamHI/Bs/WI restriction sites utilised in transfer of the variable heavy and light region from the single construct are highlighted.<br>
Figure 25: Sequence of DCIB36<br>
Nucleotide and amino acid sequence of the heavy and light variable regions cloned inframe with the human IgGI Fc and kappa constant regions within the expression vector pDCOrig. Amino acids within boxes represent the TRP2 epitope (SVYDFFVWL) in L3. The HindIII/WIAfe I and BamHI/Bs/WI restriction<br>
sites utilised in transfer of the variable heavy and light region from the single construct are highlighted.<br>
Figure 26: Sequence of DCIB48<br>
Nucleotide and amino acid sequence of the heavy and light variable regions cloned inframe with the human lgG1 Fc and kappa constant regions within the expression vector pDCOrig. Amino acids within boxes represent the TRP2 epitope (SVYDFFVWL) in H2 and the HLA-DR4 restricted gp100 CD4 epitope in H3 (WNRQLYPEWTEAQRLD). The HindWVAfe I and BamHI/Bs/WI restriction sites utilised in transfer of the variable heavy and light region from the single construct are highlighted.<br>
Figure 27: Sequence of DCIB49<br>
Nucleotide and amino acid sequence of the heavy and light variable regions cloned inframe with the human lgG1 Fc and kappa constant regions within the expression vector pDCOrig. Amino acids within boxes represent the HepB CD4 epitope (TPPAYRPPNAPIL) in H3. The HindIII//Afe I and BamHI/BsiWI restriction sites utilised in transfer of the variable heavy and light region from the single construct are highlighted.<br>
Figure 28: Sequence of DCIB52<br>
Nucleotide and amino acid sequence of the heavy and light variable regions cloned inframe with the human lgG1 Fc and kappa constant regions within the expression vector pDCOrig. Amino acids within boxes represent the TRP2 epitope (SVYDFFVWL) in H2 and the HepB CD4 epitope (TPPAYRPPNAPIL) in H3. The HindIII/Afe I and BamHI/BsiWI restriction sites utilised in transfer of the variable heavy and light region from the single construct are highlighted.<br>
Figure 29: Sequence of DCIB54<br>
Nucleotide and amino acid sequence of the heavy and light variable regions<br>
cloned inframe with the human lgG1 Fc and kappa constant regions within the<br>
expression vector pDCOrig.   Amino acids within boxes represent the HLA-DR7 restricted gp100 CD4 epitope ( GTGRAMLGTHTMEVTVYH) in H1, the TRP2 epitope (SVYDFFVWL) in H2 and the HLA-DR4 restricted gp100 CD4 epitope in H3 (WNRQLYPEWTEAQRLD). The HindIII/Afe I and BamHI/BsiWI restriction sites utilised in transfer of the variable heavy and light region from the single construct are highlighted.<br>
Figure 30: Sequence of DCIB18<br>
Nucleotide and amino acid sequence of the heavy and light variable regions cloned inframe with the human lgG1 Fc and kappa constant regions within the expression vector pDCOrig. Amino acids within boxes represent the TRP2 epitope in H2 (SVYDFFVWL) and the HepB CD4 epitope in L1 (TPPAYRPPNAPIL). The HindIII/Afe I and BamHI/BsiWI restriction sites utilised in transfer of the variable heavy and light region from the single construct are highlighted.<br>
Figure 31: CTL epitopes incorporated into ImmunoBody framework are processed and presented to elicit an immune response in vivo.<br>
A,	C57BI/6 mice were immunised on days 0, 7, and 14 with an ImmunoBody<br>
construct containing the TRP2 epitope in CDR H2 and HepB CD4 epitope in<br>
CDR L1 (DCIB18). On day 19 splenocytes were analysed by IFN elispot<br>
assay against TRP2 peptide, HepB helper peptide and a media control.<br>
Responses are measured as spots/million splenocytes.<br>
B,	Splenocytes from immunised mice were assayed for avidity to the TRP2<br>
epitope by measuring responses to increasing peptide concentration in IFN<br>
elispot assay. Responses are measured as spots/million splenocytes and<br>
avidity is assigned as the concentration which gives 50% maximal effector<br>
function.<br>
C,	splenocytes from immunised mice were depleted of CD8 T cells and<br>
analysed against TRP2 peptide, HepB helper peptide and a media control for<br>
the presence epitope specific responses in IFN elispot assay. Responses<br>
are measured as spots/million splenocytes.<br>
D,	cytotoxicity of splenocytes from immunised mice in a 4 hour 51Cr-release<br>
assay against the B16F10, B16F10 iFNα and B16F10 siKb melanoma cell<br>
lines after 6 days in vitro TRP2 peptide stimulation.<br>
E,	C57BI/6 or HLA-DR4 transgenic mice were immunised on days 0, 7, and<br>
14 with ImmunoBody DNA(DCIB15, DCIB31, DCIB32, DCIB36, DCIB48,<br>
DCIB52 and DCIB54). On day 19 splenocytes were analysed by IFN elispot<br>
assay against TRP2 peptide and a media control. Responses are measured<br>
as spots/million splenocytes.<br>
F,	Splenocytes from immunised mice were assayed for avidity to the TRP2<br>
epitope by measuring responses to increasing peptide concentration in IFN<br>
elispot assay. Responses are measured as spots/million splenocytes and<br>
avidity is assigned as the concentration which gives 50% maximal effector<br>
function.<br>
G,	C57BI/6 or HLA-DR4 transgenic mice were immunised on days 0, 7, and<br>
14 with ImmunoBody DNA (DCIB15, DCIB48, DCIB49, DCIB52 and DCIB54).<br>
On day 19 splenocytes were analysed by IFN elispot assay against HepB<br>
helper peptide (DCIB15, DCIB49 and DCIB52) or gp100DR4 helper peptide<br>
(DCIB48 and DCIB54) and a media control. Responses are measured as<br>
spots/million splenocytes.<br>
Figure 32: ImmunoBody DNA immunisation is better than peptide immunisation or immunisation with whole antigen.<br>
A,	ImmunoBody DNA immunisation (DCIB18) was compared to s.c.<br>
immunisation with peptide epitope in Incomplete Freund adjuvant or<br>
immunisation with a DNA expressing the TRP2 antigen. C57BI/6 mice were<br>
immunised on days 0, 7, and 14 and on day 19 splenocytes were analysed by<br>
IFN elispot assay against TRP2 peptide (■), HepB helper peptide (;-) and a<br>
media control (D). Responses are measured as spots/million splenocytes.<br>
B,	Splenocytes from ImmunoBody DNA (0) and peptide (♦) immunised mice<br>
were assayed for avidity to the TRP2 epitope by measuring responses to<br>
increasing peptide concentration in IFNy elispot assay. Responses are<br>
measured as spots/million splenocytes and avidity is assigned as the concentration which gives 50% maximal effector function.<br>
C,	cytotoxicity of splenocytes from immunised mice in a 4 hour 51Cr-release<br>
assay against the B16F10 (■), B16F10 IFNa ('.) and B16F10 siKb (a)<br>
melanoma cell lines after 6 days in vitro TRP2 peptide stimulation.<br>
D,	ImmunoBody DNA immunisation (DCIB18) was compared to immunisation<br>
with TRP2 peptide pulsed DCs. C57BI/6 mice were immunised on days 0, 7,<br>
and 14 and on day 19 splenocytes were analysed by IFN elispot assay against<br>
titrating quantities of TRP2 peptide. Responses are measured as spots/million<br>
splenocytes and avidity is assigned as the concentration which gives 50%<br>
maximal effector function.<br>
E,	ImmunoBody DNA immunisation (DCIB18) was compared to immunisation<br>
with TRP2 peptide pulsed DCs. C57BI/6 mice were immunised on days 0, 7,<br>
and 14 and on day 19 splenocytes were stimulated in vitro with TRP2 peptide<br>
pulsed LPS blasts. Six days post stimulation CTL lines were assessed by<br>
chromium release assay for ability to lyse B16F10 or B16F10 siKb melanoma<br>
lines. Responses are measured as % cytotoxicity.<br>
F,	ImmunoBody DNA immunisation (DCIB24) was compared to immunisation<br>
with SIINFEKL peptide. C57BI/6 mice were immunised on days 0, 7, and 14<br>
and on day 19 splenocytes were analysed by IFN elispot assay against<br>
SIINFEKL peptide and a control peptide. Responses are measured as<br>
spots/million splenocytes.<br>
G,	ImmunoBody DNA immunisation (DCIB15) was compared to immunisation<br>
with gp100 210M peptide. HHDII mice were immunised on days 0, 7, and 14<br>
and on day 19 splenocytes were analysed by IFN elispot assay against<br>
titrating quantities of gp100 210M peptide and a control. Responses are<br>
measured as spots/million splenocytes.<br>
H, ImmunoBody DNA immunisation (DCIB24) was compared to immunisation with SIINFEKL peptide. C57BI/6 mice were immunised on days 0,7, and 14 and on day 19 splenocytes were analysed by IFNy elispot assay against titrating quantities of SIINFEKL peptide. Responses are measured as<br>
spots/million splenocytes and avidity is assigned as the concentration which gives 50% maximal effector function.<br>
I, ImmunoBody DNA immunisation (DCIB15) was compared to immunisation with gp100 210M peptide. HHDII mice were immunised on days 0, 7, and 14 and on day 19 splenocytes were analysed by IFN elispot assay against titrating quantities of gp100 210M peptide. Responses are measured as spots/million splenocytes and avidity is assigned as the concentration which gives 50% maximal effector function.<br>
Figure 33: Sequence of DCIB21<br>
Nucleotide and amino acid sequence of the heavy and light variable regions cloned in frame with the human lgG1 Fc and kappa constant regions within the expression vector pDCOrig. Amino acids within boxes represent the HepB S Ag epitope in H2 (IPQSLDSWWTSL) and the I-Ad restricted Flu HA CD4 epitope in L1 (FERFEIFPKE). The HindIII/Afel and BamHI//BsiWI restriction sites utilised in transfer of the variable heavy and light region from the single construct are highlighted.<br>
Figure 34: Multiple epitopes can be processed from CDR H2 site.<br>
A,	C57BI/6 mice were immunised on days 0, 7 and 14 with ImmunoBody<br>
construct containing SIINFEKL epitope in CDR H2 and HepB CD4 epitope in<br>
CDR L1 (DCIB24). On day 19, splenocytes were analysed in IFN elispot<br>
assay against SIINFEKL peptide, an irrelevant peptide, HepB CD4 peptide and<br>
media control. Responses are measured as spots/million splenocytes.<br>
B,	Balb/c mice were immunised on days 0, 7 and 14 with ImmunoBody<br>
construct containing HepB CD8 epitope in CDR H2 and Flu HA CD4 epitope in<br>
CDR L1 (DCIB21). On day, 19 splenocytes were analysed in IFN elispot<br>
assay against HepB CD8 peptide, an irrelevant peptide, Flu HA CD4 peptide<br>
and media control. Responses are measured as spots/million splenocytes.<br>
Figure 35: Sequence of DCIB17<br>
Nucleotide and amino acid sequence of the heavy and light variable regions cloned in frame with the human lgG1 Fc and kappa constant regions within the expression vector pDCOrig. Amino acids within boxes represent the GP100210M epitope in H1 (TIMDQVPFSV) and the HepB CD4 epitope in L1 (TPPAYRPPNAPIL). The HindIII/Afe I and BamHI/BsiWI restriction sites utilised in transfer of the variable heavy and light region from the single construct are highlighted.<br>
Figure 36: Sequence of DCIB26<br>
Nucleotide and amino acid sequence of the heavy and light variable regions cloned in frame with the human lgG1 Fc and kappa constant regions within the expression vector pDCOrig. Amino acids within boxes represent the Tie-2 Z84 epitope in H1 (FLPATLTMV) and the HepB CD4 epitope in L1 (TPPAYRPPNAPIL). The HindIII/Afe I and BamHI/BsiWI restriction sites utilised in transfer of the variable heavy and light region from the single construct are highlighted.<br>
Figure 37: Multiple CTL epitopes can be processed from the variable region.<br>
A,	HHDII mice were immunised on days 0, 7 and 14 with ImmunoBody<br>
construct containing gp100 IMDQVPFSV epitope in CDR H1 with removal of<br>
part of the framework and HepB CD4 epitope in CDR L1 (DCIB17). On day 19,<br>
splenocytes were analysed in IFNelispot assay against gp100 IMDQVPFSV<br>
peptide, HepB CD4 peptide and media control. Responses are measured as<br>
spots/million splenocytes.<br>
B,	HHDII mice were immunised on days 0, 7 and 14 with ImmunoBody<br>
construct containing Tie2 epitope in CDR H1 with removal of part of the<br>
framework and HepB CD4 epitope in CDR L1 (DCIB26). On day 19,<br>
splenocytes were analysed in IFN elispot assay against Tie2 peptide, HepB<br>
CD4 peptide and media control. Responses are measured as spots/million<br>
splenocytes.<br>
Figure 38: Multiple CTL responses can be generated from different epitopes within the same ImmunoBody construct.<br>
HLA-A2 restricted gp100 epitope IMDQVPFSV was engineered into the CDR H1 site alongside the TRP2 epitope SVYDFFVWL in CDR H2 and the HepB CD4 epitope was present in the CDR L1 site (DCIB15).<br>
A,	HHDII mice were immunised on days 0, 7, and 14 with ImmunoBody DNA.<br>
On day 19 splenocytes were analysed by IFN elispot assay against gp100<br>
peptide, TRP2 peptide, HepB helper peptide and a media control. Responses<br>
are measured as spots/million splenocytes.<br>
B,	Splenocytes from immunised mice were assayed for avidity to the gp100<br>
modified IMDQVPFSV (♦) epitope, gp100 wt ITDQVPFSV epitope (A) and<br>
TRP2 epitope (■) by measuring responses to increasing peptide concentration<br>
in IFN7 elispot assay. Responses are measured as spots/million splenocytes<br>
and avidity is assigned as the concentration which gives 50% maximal effector<br>
function.<br>
C,	cytotoxicity of splenocytes from immunised mice in a 4 hour 51Cr-release<br>
assay against T2 cells pulsed with gp100 IMDQVPFSV peptide, TRP2 peptide<br>
or control and the B16F10 and B16F10 HHD melanoma cell lines.<br>
D,	HHDII mice were immunised on days 0, 7, and 14 with ImmunoBody DNA<br>
containing either i) gp100 epitope in CDR H1, TRP2 epitope in CDR H2 and<br>
HepB CD4 epitope in CDR L1 (DCIB15) or ii) TRP2 epitope in CDR H2 and<br>
HepB CD4 epitope in CDR L1 (DCIB18). On day 19, splenocytes were<br>
analysed by IFN elispot assay against gp100 peptide (■), TRP2 peptide (▒),<br>
HepB helper peptide (▒) and a media control (n). Responses are measured as<br>
spots/million splenocytes.<br>
E,	C57BI/6 mice were immunised i.m. with 10µg DNA solution combined with<br>
electroporation. Immunisations were performed three times at weekly intervals<br>
in the tibialis muscle. Mice were immunised with DCIB24 or DCIB18 alone,<br>
both combined in the same site or with both at the same time but in separate<br>
sites. On day 19 splenocytes were analysed for the presence of TRP2,<br>
SIINFEKL peptide specific immune responses. Responses are measured as<br>
spots/million splenocytes.<br>
Figure 39: Sequence of DCIB37<br>
Nucleotide and amino acid sequence of the heavy and light variable regions cloned in frame with the human lgG1 Fc and kappa constant regions within the expression vector pDCOrig. Amino acids within boxes represent the GP100 F7L epitope in H1 (TITDQVPLSV) and the HepB CD4 epitope in L1 (TPPAYRPPNAPIL). The HindIII/Afe I and BamHI/BsiWI restriction sites utilised in transfer of the variable heavy and light region from the single construct are highlighted.<br>
Figure 40: Sequence of DCIB40<br>
Nucleotide and amino acid sequence of the heavy and light variable regions cloned in frame with the human lgG1 Fc and kappa constant regions within the expression vector pDCOrig. Amino acids within boxes represent the GP100 F7I epitope in H1 (TITDQVPISV) and the HepB CD4 epitope in L1 (TPPAYRPPNAPIL). The HindIII/Afe I and BamHI/BsiWI restriction sites utilised in transfer of the variable heavy and light region from the single construct are highlighted.<br>
Figure 41: Sequence of DCIB41<br>
Nucleotide and amino acid sequence of the heavy and light variable regions cloned in frame with the human lgG1 Fc and kappa constant regions within the expression vector pDCOrig. Amino acids within boxes represent the GP100 wild type epitope in H1 (TITDQVPFSV) and the HepB CD4 epitope in L1 (TPPAYRPPNAPIL). The HindIII/Afe I and BamHI/BsiWI restriction sites utilised in transfer of the variable heavy and light region from the single construct are highlighted.<br>
Figure 42: Sequence of DCIB42<br>
Nucleotide and amino acid sequence of the heavy and light variable regions cloned in frame with the human lgG1 Fc and kappa constant regions within the expression vector pDCOrig. Amino acids within boxes represent the<br>
GP100 F7Y epitope in H1 (TITDQVPYSV) and the HepB CD4 epitope in L1 (TPPAYRPPNAPIL). The HindIII/Afe I and BamHI/Bs/WI restriction sites utilised in transfer of the variable heavy and light region from the single construct are highlighted.<br>
Figure 43: Sequence of DCIB43<br>
Nucleotide and amino acid sequence of the heavy and light variable regions cloned in frame with the human lgG1 Fc and kappa constant regions within the expression vector pDCOrig. Amino acids within boxes represent the GP100 V5L epitope in H1 (TITDQLPFSV) and the HepB CD4 epitope in L1 (TPPAYRPPNAPIL). The HindIIII/Ate I and BamHI/BsiWI restriction sites utilised in transfer of the variable heavy and light region from the single construct are highlighted.<br>
Figure 44: Modification at non-anchor residues can enhance epitope<br>
immunogenicity.<br>
HHDII mice were immunised at days 0, 7 and 14 with ImmunoBody constructs<br>
containing modified gp100 epitopes in the CDR H1 region (DCIB37, DCIB40,<br>
DCIB41, DCIB42 and DCIB43). On day 19, splenocytes were analysed by IFN7<br>
elispot assay against gp100 wild type epitope peptide and a media control.<br>
Responses are measured as spots/million splenocytes.<br>
Figure 45: Sequence of DCIB35<br>
Nucleotide and amino acid sequence of the heavy and light variable regions cloned in frame with the human lgG1 Fc and kappa constant regions within the expression vector pDCOrig. Amino acids within boxes represent the GP100210M epitope in H1 (TIMDQVPFSV), the TRP2 epitope in H2 (SVYDFFVWL) and the HLA-DR4 restricted gp100 CD4 epitope in L1 (WNRQLYPEWTEAQRLD). The HindIII/Afe I and BamHI/BsiWI restriction sites utilised in transfer of the variable heavy and light region from the single construct are highlighted.<br>
Figure 46: Multiple CD4 helper responses can be processed and presented to elicit an immune response in vivo.<br>
A,	HHDII or C57BI/6 mice were immunised at days 0, 7 and 14 with<br>
ImmunoBody constructs containing the l-Ab restricted HepB CD4 epitope in the<br>
CDRL1 region (DCIB15).<br>
B,	Balb/c mice were immunised at days 0, 7 and 14 with ImmunoBody<br>
constructs containing the l-Ad restricted Flu HA CD4 epitope in the CDR L1<br>
region (DCIB21).<br>
C,	HLA-DR4 transgenic mice were immunised at days 0, 7 and 14 with<br>
ImmunoBody constructs containing the HLA-DR4 restricted gp100 CD4 epitope<br>
in the CDR L1 (DCIB35). On day 19, splenocytes were analysed by IFNelispot<br>
assay against corresponding peptide, an irrelevant peptide and a media control.<br>
Responses are measured as spots/million splenocytes.<br>
D,	HLA-DR4 transgenic mice were immunised at days 0, 7 and 14 with<br>
ImmunoBody constructs containing the HLA-DR4 restricted gp100 CD4 epitope<br>
in the CDR L1 (DCIB35), in the CDR H3 (DCIB54) and in the CDR L3<br>
(DCIB50). On day 19, splenocytes were analysed by IFN elispot assay<br>
against corresponding peptide, an irrelevant peptide and a media control.<br>
Responses are measured as spots/million splenocytes.<br>
Figure 47: Sequence of DCIB50<br>
Nucleotide and amino acid sequence of the heavy and light variable regions cloned inframe with the human lgG1 Fc and kappa constant regions within the expression vector pDCOrig. Amino acids within boxes represent the GP100210M epitope (TIMDQVPFSV) in H1, the TRP2 epitope (SVYDFFVWL) in H2 and the HLA-DR4 restricted gp100 CD4 epitope (WNRQLYPEWTEAQRLD) in L3. The HindIII/Afe I and BamHI/BsiWI restriction sites utilised in transfer of the variable heavy and light region from the single construct are highlighted.<br>
Figure 48: CD8 T cell responses are partially dependent upon secreted heavy chain but helper responses do not require secreted light chain.<br>
A,	HHDII mice were immunised at day 0, 7 and 14 with ImmunoBody DNA<br>
constructs containing i) gp100 epitope in CDR H1, TRP2 epitope in CDR H2<br>
and HepB CD4 epitope in CDR L1 (DCIB15), ii) gp100 epitope in CDR H1,<br>
TRP2 epitope in CDR H2 and HepB CD4 epitope in CDR L1 without the leader<br>
sequence on the heavy chain, iii) gp100 epitope in CDR H1, TRP2 epitope in<br>
CDR H2 and HepB CD4 epitope in CDR L1 without the leader sequence on the<br>
light chain. On day 19, splenocytes were analysed by IFN elispot assay<br>
against gp100 (■) and HepB CD4 (▒) peptides and a media control (a).<br>
Responses are measured as spots/million splenocytes.<br>
B,	Determination of heavy chain, light chain and intact ImmunoBody from<br>
supernatant of CHO-S transfectants by sandwich Elisa. Plates were coated<br>
with an anti-human Fc specific antibody or anti-human kappa chain antibody.<br>
To detect heavy chain an anti-human IgG Fc specific HRP antibody was used<br>
in combination with the anti-human Fc specific coating antibody. To detect<br>
intact ImmunoBody an anti-human kappa chain specific HRP antibody was<br>
used in combination with anti-human Fc specific coating antibody. To detect<br>
light chain anti-human kappa chain specific HRP antibody was used in<br>
combination with the anti-human kappa chain specific antibody.<br>
C,	C57BI/6 mice were immunised at day 0, 7 and 14 with ImmunoBody DNA<br>
constructs containing i) gp100 epitope in CDR H1, TRP2 epitope in CDR H2<br>
and HepB CD4 epitope in CDR L1 (DCIB15), ii) gp100 epitope in CDR H1,<br>
TRP2 epitope in CDR H2 and HepB CD4 epitope in CDR L1 without the leader<br>
sequence on the heavy chain. On day 19, splenocytes were analysed by IFN<br>
elispot assay against TRP2 peptide. Responses are measured as spots/million<br>
splenocytes.<br>
D,	C57BI/6 mice were immunised at day 0, 7 and 14 with ImmunoBody DNA<br>
constructs containing i) gp100 epitope in CDR H1, TRP2 epitope in CDR H2<br>
and HepB CD4 epitope in CDR L1 (DCIB15), ii) gp100 epitope in CDR H1,<br>
TRP2 epitope in CDR H2 and HepB CD4 epitope in CDR L1 without the leader<br>
sequence on the heavy chain. On day 19, splenocytes were analysed by IFN<br>
elispot assay against HepB helper peptide. Responses are measured as spots/million splenocytes.<br>
Figure 49: ImmunoBody Fc region is beneficial for establishing an efficient immune response.<br>
A,	C57BI/6 mice were immunised at day 0, 7 and 14 with ImmunoBody DNA<br>
constructs containing i) gp100 epitope in CDR H1, TRP2 epitope in CDR H2<br>
and HepB CD4 epitope in CDR L1 (DCIB15), ii) gp100 epitope in CDR H1,<br>
TRP2 epitope in CDR H2 and HepB CD4 epitope in CDR L1 lacking the Fc<br>
region. On day 19, splenocytes were analysed by IFNelispot assay against<br>
TRP2 (▒) peptide, a media control (p), the B16F10 melanoma line (■) and the<br>
B16F10 siKb negative control cell line (▒). Responses are measured as<br>
spots/million splenocytes.<br>
B,	C57BI/6 mice were immunised at day 0, 7 and 14 with ImmunoBody DNA<br>
constructs containing i) gp100 epitope in CDR H1, TRP2 epitope in CDR H2<br>
and HepB CD4 epitope in CDR L1 (DCIB15), ii) gp100 epitope in CDR H1,<br>
TRP2 epitope in CDR H2 and HepB CD4 epitope in CDR L1 lacking the Fc<br>
region. On day 19, splenocytes were analysed by IFNyelispot assay against<br>
TRP2 peptide. Responses are measured as spots/million splenocytes.<br>
C,	The same mice were analysed for responses specific for the HepB helper<br>
peptide. Responses are measured as spots/million splenocytes.<br>
D,	Splenocytes from mice immunised with DCIB15 or DCIB15 lacking the Fc<br>
region (DCIB15 FcStop) were assayed for avidity to the TRP2 epitope by<br>
measuring responses to increasing peptide concentration in IFNelispot assay.<br>
Responses are measured as spots/million splenocytes and avidity is assigned<br>
as the concentration which gives 50% maximal effector function.<br>
E,	C57BI/6 mice were immunised at day 0, 7 and 14 with ImmunoBody DNA<br>
constructs containing i) gp100 epitope in CDR H1, TRP2 epitope in CDR H2<br>
and HepB CD4 epitope in CDR L1 Human lgG1 (DCIB15), ii) The same<br>
construct with Human lgG2 constant region (DCIB33), iii) The same construct<br>
with Human lgG3 constant region (DCIB65), iv) The same construct with the<br>
Human lgG1 binding motif replaced with the binding motif from Human lgG2<br>
 (DCIB66) and v) DCIB33 with the binding motif replaced by the motif from Human lgG1 (DCIB67). On day 19, splenocytes were analysed by IFNelispot assay against TRP2 peptide (■), a media control (D) and the HepB helper peptide (||). Responses are measured as spots/million splenocytes.<br>
F,	Determination of heavy chain, light chain and intact ImmunoBody from<br>
supernatant of CHO-S transfectants (DCIB15, DCIB33, DCIB65, DCIB66 and<br>
DCIB67) by sandwich Elisa. Plates were coated with an anti-human Fc<br>
specific antibody or anti-human kappa chain antibody. To detect heavy chain<br>
an anti-human IgG Fc specific HRP antibody was used in combination with<br>
the anti-human Fc specific coating antibody. To detect intact ImmunoBody an<br>
anti-human kappa chain specific HRP antibody was used in combination with<br>
anti-human Fc specific coating antibody. To detect light chain anti-human<br>
kappa chain specific HRP antibody was used in combination with the anti-<br>
human kappa chain specific antibody.<br>
G,	Determination of heavy chain ImmunoBody from supernatant of CHO-S<br>
transfected with DCIB53 by sandwich Elisa. Plates were coated with an anti-<br>
mouse Fc specific antibody. To detect heavy chain an anti-mouse lgG2a<br>
specific HRP antibody was used.<br>
Figure 50: ImmunoBody immunisation enhances immune responses and overcomes regulation observed from whole antigen.<br>
A,	HLA-A2 transgenic mice (HHDII) were immunised at day 0, 7 and 14 with<br>
ImmunoBody DNA constructs DCIB15 or whole gp100 antigen in pcDNA3<br>
vector. On day 19, splenocytes were analysed by IFN elispot assay against<br>
gp100 peptide or control. Responses are measured as spots/million<br>
splenocytes.<br>
B,	C57BI/6 mice were depleted of CD25 positive cells by injection of anti-CD25<br>
antibody (PC61) 400ug i.p. Both CD25 depleted mice and undepleted animals<br>
were subsequently immunised at day 4, 11 and 18 with ImmunoBody DNA<br>
constructs DCIB15 or whole TRP2 antigen in pOrig vector. On day 23,<br>
splenocytes were analysed by IFNelispot assay against TRP2 peptide or<br>
control. Responses are measured as spots/million splenocytes.<br>
C and D, HHDII mice were either untreated (c) or treated with 400ug PC61 mAb i.p., (d). 4 days later, all mice were immunized with the Tie2 C200HFc DNA construct. DNA immunizations were repeated at 7 day intervals for a total of 3 immunisations. 6 days after the final immunisation, splenocytes were harvested and restimulated in an ex-vivo IFN7 ELISPOT assay with 1 ug/ml of each of the predicted CTL epitopes from Tie-2. Bars indicate the mean of triplicate values for each individual mouse, normalized to background controls, with error bars representing the standard deviation from the mean.<br>
E and F, HHDII mice were either untreated (e) (n = 3) or treated (f) (n = 2) with 400ug PC61 antibody i.p. After 4 days, all mice were immunised with 100µg Z12 peptide and 100µg Z48 peptide, mixed 1:1 in IFA (s.c). Repeat peptide immunisations were administered 7 days after the first peptide immunisation. Splenocytes were harvested 14 days after the final immunisation and restimulated with 1 µg/ml Z12 peptide (black bars) or media alone (open bars) in an IFN ELIspot assay. Bars indicate the mean of triplicate values with error bars representing the standard deviation from the mean.<br>
G, HHDII mice were immunised with 100µg Z12 peptide mixed 1:1 in IFA (s.c). Repeat peptide immunisations were administered at days 7 and 14 days after the first peptide immunisation. Splenocytes were harvested 7 days after the final immunisation and analysed for the presence of epitope specific responses to increasing peptide concentration in IFNelispot assay. Responses are measured from individual mice as spots/million splenocytes and avidity is assigned as the concentration which gives 50% maximal effector function. H, HHDII mice were immunised with ImmunoBody DNA construe DCIB71 via gene gun at days 0, 7 and 14. Splenocytes were harvested 7 days after the final immunisation and analysed for the presence of epitope specific responses to increasing peptide concentration in IFNelispot assay. Responses are measured from individual mice as spots/million splenocytes and avidity is assigned as the concentration which gives 50% maximal effector function.<br>
Figure 51: Sequence of DCIB71<br>
Nucleotide and amino acid sequence of the heavy and light variable regions cloned inframe with the human lgG1 Fc and kappa constant regions within the expression vector pDCOrig. Amino acids within boxes represent the Tie-2 Z12 epitope (ILINSLPLV) in H1 and the HepB CD4 epitope (TPPAYRPPNAPIL) in L1 .The HindIII/Afe I and BamHI/BsiWI restriction sites utilised in transfer of the variable heavy and light region from the single construct are highlighted.<br>
Figure 52: Sequence of DCIB72<br>
Nucleotide and amino acid sequence of the heavy and light variable regions cloned inframe with the human lgG1 Fc and kappa constant regions within the expression vector pDCOrig. Amino acids within boxes represent the Tie-2 Z12 epitope (ILINSLPLV) in H2 and the HepB CD4 epitope (TPPAYRPPNAPIL) in L1. The HindIIII/Afe I and BamHI/BsiWI restriction sites utilised in transfer of the variable heavy and light region from the single construct are highlighted.<br>
Figure 53: The role of xenogenic Fc in providing T cell help and the requirement for antigen specific T cell help.<br>
A,	C57BI/6 or HHDll mice were immunised at day 0, 7 and 14 with Heavy chain<br>
ImmunoBody DNA constructs containing gp100 epitope in CDR H1 or TRP2<br>
epitope in CDR H2 (IB17 and IB18 respectively). On day 19, spienocytes were<br>
analysed by IFNelispot assay against gp100 peptide or TRP2 peptide and<br>
control. Responses are measured as spots/million spienocytes.<br>
B,	Spienocytes from mice immunised with ImmunoBody heavy chain containing<br>
TRP2 epitope in CDR H2 were assayed for avidity to the TRP2 epitope by<br>
measuring responses to increasing peptide concentration in IFN elispot assay.<br>
Responses are measured as spots/million spienocytes and avidity is assigned<br>
as the concentration which gives 50% maximal effector function.<br>
C,	C57BI/6 mice were immunised at day 0, 7 and 14 with ImmunoBody DNA<br>
constructs containing gp100 epitope in CDR H1, TRP2 epitope in CDR H2 and<br>
HepB CD4 epitope in CDR L1 Human lgG1 (DCIB15) or gp100 epitope in CDR<br>
H1, TRP2 epitope in CDR H2 and HepB CD4 epitope in CDR L1 with murine<br>
lgG2a constant region (DCIB53). On day 19, splenocytes were analysed by IFNelispot assay against TRP2 peptide, HepB helper peptide and control. Responses are measured as spots/million splenocytes.<br>
D,	Splenocytes from mice immunised with DCIB15 or DCIB53 were assayed for<br>
avidity to the TRP2 epitope by measuring responses to increasing peptide<br>
concentration in IFNelispot assay. Responses are measured as spots/million<br>
splenocytes and avidity is assigned as the concentration which gives 50%<br>
maximal effector function.<br>
E,	HLA-DR4 transgenic mice were immunised at day 0, 7 and 14 with<br>
ImmunoBody DNA constructs containing gp100DR4 epitope in CDR H1, TRP2<br>
epitope in CDR H2 and gp100DR7 epitope in CDR H3 Human lgG1 (DCIB54)<br>
or gp100DR4 epitope in CDR H1, TRP2 epitope in CDR H2 and gp100DR7<br>
epitope in CDR H3 with murine lgG2a constant region (DCIB64). On day 19,<br>
splenocytes were analysed by IFNelispot assay against TRP2 peptide,<br>
gp100DR4 helper peptide and control. Responses are measured as<br>
spots/million splenocytes.<br>
F,	Splenocytes from mice immunised with DCIB54 or DCIB64 were assayed for<br>
avidity to the TRP2 epitope by measuring responses to increasing peptide<br>
concentration in IFNelispot assay. Responses are measured as spots/million<br>
splenocytes and avidity is assigned as the concentration which gives 50%<br>
maximal effector function.<br>
Figure 54: Sequence of DCIB53<br>
Nucleotide and amino acid sequence of the murine heavy and light full length chains within the expression vector pDCOrig moigG2a. Amino acids within boxes represent theGP100210M epitope in H1 (TIMDQVPFSV), the TRP2 epitope in H2 (SVYDFFVWL) and the HepB CD4 epitope in L1 (TPPAYRPPNAPIL) in L1. The HindIII/Afe I and BamHI/Hpal restriction sites utilised in transfer of the variable heavy and light region from the single construct are highlighted.<br>
Figure 55: Sequence of DCIB64<br>
Nucleotide and amino acid sequence of the murine heavy and light full length chains within the expression vector pDCOrig moigG2a. The stop codon is depicted by an asterisk. Amino acids within boxes represent the HLA-DR7 restricted gp100 CD4 epitope (GTGRAMLGTHTMEVTVYH) in H1, the TRP2 epitope (SVYDFFVWL) in H2 and the HLA-DR4 restricted gp100 CD4 epitope in H3 (WNRQLYPEWTEAQRLD). The HindIII/Afe I and BamHIHpal restriction sites utilised in transfer of the variable heavy and light region from the single construct are highlighted.<br>
Figure 56: Immunoproteasome processing is important in the generation of responses from epitopes within ImmunoBody constructs. HHDII mice were immunised at day 0, 7 and 14 with ImmunoBody constructs containing the gp100209-217epitope in CDR H1 (DCIB41) or the modified version gp100210M in CDR H1 (DCIB15). On day 19, splenocytes were analysed by IFNelispot assay against gp100209-217 peptide or gp100210M peptide and control. Responses are measured as spots/million splenocytes.<br>
Figure 57: Different immunisation methods are efficient at eliciting immune responses from ImmunoBody vaccine.<br>
A,	C57BI/6 mice were immunised with ImmunoBody DNA (DCIB15) via gene<br>
gun, i.m. +/- electroporation or i.d. +/- electroporation at days 0, 7 and 14. On<br>
day 19, splenocytes were analysed by IFNelispot assay against TRP2<br>
peptide, HepB helper peptide and control. Responses are measured as<br>
spots/million splenocytes.<br>
B,	Splenocytes from mice immunised by different routes were assayed for<br>
avidity to the TRP2 epitope by measuring responses to increasing peptide<br>
concentration in IFNelispot assay. Responses are measured as spots/million<br>
splenocytes and avidity is assigned as the concentration which gives 50%<br>
maximal effector function.<br>
Figure 58: ImmunoBody immunisation induces vitiligo-like depigmentation and protects against tumour challenge.<br>
A,	C57BI/6 mice immunised with ImmunoBody DNA containing the TRP2<br>
epitope in CDR H2 and the HepB CD4 epitope in CDR L1 (DCIB18)<br>
demonstrate depigmentation in hair growth at the site of immunisation.<br>
B,	Immunised C57BI/6 mice were challenged between 3rd and 41h<br>
immunisations with 2x104 B16F10 IFNa cells i.v. Tumour burden in the lungs<br>
was assessed at 49 days post tumour challenge. Tumour burden is expressed<br>
as a mean tumour area as a percentage of total lung area. Immunised mice<br>
were challenged 7 days post final immunisation with 2x104 B16F10 IFNa cells<br>
s.c. Tumour size was measured at 3-4 day intervals and mice euthanized once<br>
tumour growth exceeded limit.<br>
C,	Tumour size assessed at day 46 post tumour injection.<br>
D,	survival.<br>
Figure 59: ImmunoBody immunisation significantly delays tumour growth.<br>
A,	C57BI6 mice were injected with 2x104 B16F10 cells s.c. Four days post<br>
tumour injection mice were immunised with DCIB52 ImmunoBody DNA.<br>
Repeat immunisation were performed at days 11 and 18 post tumour injection.<br>
Tumour burden was analysed at 3-4 day intervals and mice euthanized once<br>
tumour growth exceeded maximum permitted limit. Tumour volume over time<br>
was plotted.<br>
B,	C57BI6 mice were injected with 2x104 B16F10 IFNa cells s.c. Fourteen days<br>
post tumour injection mice were immunised with DCIB52 ImmunoBody DNA.<br>
Repeat immunisations were performed at days 21 and 28 post tumour injection.<br>
Tumour burden was analysed at 3-4 day intervals and mice euthanized once<br>
tumour growth exceeded maximum permitted limit. Tumour volume is shown at<br>
day 47 post tumour implant.<br>
C,	C57BI6 mice were injected with 2x104 B16F10 cells s.c and anti-CD25<br>
antibody i.p. where appropriate. Four days post tumour injection mice were<br>
immunised with DCIB52 ImmunoBody DNA or control ImmunoBody DNA.<br>
Repeat immunisations were performed at days 11 and 18 post tumour injection<br>
Immunisation at day 11 was combined with the injection of anti-CTLA-4 antibody i.p. where appropriate. Tumour burden was analysed at 3-4 day intervals and mice euthanized once tumour growth exceeded maximum permitted limit. Tumour volume over time was plotted.<br>
Figure 60: Sequence of DCIB68<br>
Nucleotide and amino acid sequence of the heavy and light variable regions cloned inframe with the human lgG1 Fc and kappa constant regions within the expression vector pDCOrig. Amino acids within boxes represent the HLA-DR7 restricted gp100 CD4 epitope (GTGRAMLGTHTMEVTVYH) in H1 and L3, the TRP2 epitope (SVYDFFVWL) in H2 and the HLA-DR4 restricted gp100 CD4 epitope in H3 and L1 (WNRQLYPEWTEAQRLD). The HindIII/Afe I and BamHI/BsiWI restriction sites utilised in transfer of the variable heavy and light region from the single construct are highlighted.<br>
Figure 61: Immune responses can be generated from ImmunoBody constructs expressed from different vector backbones.<br>
C57BI/6 mice were immunised at day 0, 7 and 14 with ImmunoBody DNA constructs containing gp100DR4 epitope in CDR H1, TRP2 epitope in CDR H2 and gp100DR7 epitope in CDR H3 Human lgG1 (DCIB54, B1-3) an equivalent construct in the pVax vector (VaxDCIB54, C1-3). On day 19, splenocytes were analysed by IFNelispot assay against TRP2 peptide and control. Responses are measured as spots/million splenocytes.<br>
EXAMPLES<br>
Methods<br>
Generation of DNA vectors<br>
The deimmunised murine heavy and light variable regions of SC100 clone VHd VKb (WO01/88138) within the vectors pSVgptHuigGI and pSVhygHuCk (Biovation Ltd) were amplified by PCR. VH and VL region PCR products were<br>
cloned in frame with the human lgG1 and kappa constant regions using HindIII/AfeI and BamHI//BsiWI sites to produce the single chain constructs pOrigHIB and pOrigLIB (see Figures 1 and 2). The sequence of the full-length chimeric heavy and kappa chain was confirmed by the dideoxy chain termination method (Sanger et al, Proceedings of the National Academy of Sciences of the United States of America 1977;74: 5463-7). DNA and translated protein sequences for the chimeric heavy and light chain are shown in Figures 3 and 4 respectively. Locations of the complementarily determining regions (CDR's) are depicted.<br>
With exception of the heavy CDR2 region that retains six amino acids, the CDRs of the heavy and light chains were completely removed and exchanged for unique restriction enzyme sites. This was achieved by careful examination of the regions either side of the sequence for a removal that will permit a restriction enzyme site to be generated. These unique restriction sites are used to open up the DNA such that an oligonucleotide encoding an antigenic epitope can be inserted. Most framework sequence that is lost on generation of the restriction site is replaced by including in the epitope primers to ensure that, on translation, amino acids are retained and that the sequence remains in frame. Table 1 lists chosen enzyme sites and epitope oligonucleotide sequences for all CDRs.<br>
CDR regions were removed and replaced with unique restriction sites by Overlap Extension PCR as shown in Figure 5. For the heavy variable region, the oligonucleotides H1, H2 and H3 (see Table 2) were designed to replace each of the three CDR's. Each specific primer contains 10-20bp of sequence either side of the enzyme site to be incorporated. Used in conjunction with the general reverse primer huHeClonR (see Table 2) that binds to the human lgG1 constant region first round PCR's were set up consisting of 1µl of the template plasmid pOrigHIB, 2µl dNTPS (2.5mM), 5ul 10 x taq polymerase buffer, 1µl of forward and reverse primer (25pmols), 5units of taq polymerase (New England Biolabs) made up to a final volume of 50µl with sterile distilled<br>
water. Reactions were subjected to an initial denaturation of 5 minutes at 950C followed by 35 cycles of 30s at 950C, 1 minute at 550C (annealing) and 1 minutes at 720C (extension). The final cycle contained a 10 minute extension using a Techne PHC-1 programmable cyclic reactor. Similarly, for the light variable region, the oligonucleotides L1, L2, and L3 were designed to replace each of the three CDR's (see Table 2). First round PCR's were set up as described above but with the reverse primer huLiClonR (see Table 2) that binds to the constant region of the human kappa chain and the template pOrigLIB.<br>
Table 1. List of CDR replacement enzymes and epitope oligonucleotide sequences<br>
(Table Removed)<br>
N represents epitope DNA sequence<br>
The remaining letters represent framework nucleotides that need to be<br>
incorporated<br>
1 µl of the resulting PCR products was then used in a subsequent PCR as a reverse primer in conjunction with the CMV forward primer set up as outlined<br>
above. The 450bp amplified DNA fragment was cloned directly into the TA TOPO vector pCR2.1 (Invitrogen) and clones sequenced to confirm amplification of the VH and VL region devoid of the CDRs and replacement of restriction site.<br>
CDR's within the variable heavy and light have been replaced with their corresponding enzyme site H1, H2, H3, L1, L2 and L3 singly, in combination and altogether (Figure 6 and 7). The different versions were then inserted into pOrig HIB and pOrigLIB using Hindlll/Afel and BamHI/BsiWI with direct replacement of the parental wild type deimmunised SC100 VH and VL regions. This allows generation of molecules containing single or multiple epitopes (from the same or different antigens).<br>
Table 2 - Primers<br>
<br>
 <br>
<br>
<br>
(Table Removed)<br>
Insertion of antigenic epitopes into CDR sites of single chain vectors A number of CD8 CTL and CD4 helper epitopes are listed in Table 3, although any epitope can easily be inserted into any of the sites within the single chain vectors.   For example, insertion of the TRP2 epitope into the H2 site of the pOrigHIB vector was achieved as follows.<br>
Complementary oligonucleotides were designed to encode nucleotide sequence that on translation expresses the epitope. DNA sequence that encodes the epitope was flanked by the corresponding CDR nucleotides to ensure that, on translation, amino acids were retained and that the sequence remained in frame (see Table 1). Primers were sent for synthesis (MWG) and 5' end phosphorylated.<br>
Phosphorylated-T AGT GTT TAT GAT TTT TTT GTG TGG CTC CGA TTC A-3'<br>
3'- A TCA CAA ATA CTA AAA AAA CAC ACC GAG GCT AAG T-Phosphorylated-5'<br>
Complementary oligonucleotides were resuspended to a final concentration of 1 mg/ml in sterile double distilled water and annealed together by setting up a reaction with 10µl of each primer made up to a final volume of 50µl with TE buffer. The reaction was cycled for 95°C- 5mins (0.1oC/sec), 72°C - 20mins 0.1oC/sec, 55°C- 20mins then held at 4°C<br>
For insertion into the H2 site, the vector pOrigHIB H2 and/ or pOrigHIB H1H2 was linearised by setting up a Mscl restriction digest (dependent on CDR to be utilised for insertion of epitope) and incubated overnight at 37°C. The digest was electrophoresed on a 1.5% agarose gel and the cut vector purified<br>
by gel extraction. To prevent self ligation of the linearised vector, phosphate groups from the 5' ends of the vector were removed by treatment and overnight incubation at 37 oC with calf intestinal alkaline phosphatase (CIAP) 5 units, 10|il 10 x NEB buffer 3 made up to a final volume of 100µlwith sterile distilled water. Dephosphorylated vector was purified and ligations set up with neat, 1/100 and 1/200 dilutions of the annealed oligonucleotides to clone directly into the H2 site using standard techniques. Epitope insertions were confirmed by sequencing within the single vectors using the universal primer CMV forward.<br>
Table 3 CTL and helper epitopes<br>
<br>
(Table Removed)<br>
<br>
Transfer into the double expression vector pDCOriq Once all epitopes have been incorporated into the VHand VL sites within the single vectors, they are transferred into the double expression vector pDCOrig using HindIII/Afe1 and BamHI/BsiWI in frame with their respective human constant regions. To generate the ImmunoBody ™ double expression vector pDCOrig, pOrigHIB was linearised using the blunt ended restriction endonuclease Nrul located adjacent to the CMV promoter. pOrigLIB was digested with the blunt ended Nrul and Hpal endonucleases to excise the entire light chain expression cassette consisting of the CMV promoter, deimmunised human kappa chain and the BGH polyA signal. After gel electrophoresis, isolation and gel extraction of the linerised vector pOrigHIB and the light chain expression cassette the vector was dephosphorylated and light chain expression cassette ligated to form the construct pDCOrig (Figure 8). Orientation of the light chain cassette within pDCOrig was confirmed by restriction analysis.<br>
pDCOrig contains both the heavy and light chain gene coding sequences combined within the same construct, eliminating intronic sequences and the two vector system. Expression is driven by the high level CMV Immediate Earty promoters and other DNA control elements, such as Bovine Growth Hormone polyadenylation signal. The selection marker Zeocin has also been<br>
included to maximise expression and efficiency of production. Careful design of this vector has retained the unique restriction enzyme sites at the junctions of the variable and constant regions and provides a quick and easy method to create different combinations of the variable regions (epitope insertions, see Figure 8). Table 4 lists some of the pDcOrig IB constructs generated.<br>
Table 4. pDCOrig constructs<br>
<br>
 <br>
(Table Removed)<br>
Generation of pDcOriq IB15 CH1 stop<br>
A stop codon was incorporated after the CH1 domain of the human lgG1 constant region within the construct pDCOrig IB15 using the Quik change site directed mutagenesis kit (Stratagene) and the complementary oligonucleotides origstophuHeCHI Forward and OrigstophuHeCHI reverse primers (see Table 2) as instructed by the manufacturer. Incorporation of the stop codon was confirmed by DNA sequencing (Figure 9)<br>
Removal of leader sequences from pDCOrig IB15<br>
In order to remove the leader sequence from the heavy and light chain of the vector pDCOrig IB15, PCR's were set up using the template pDCOrig IB15 with the forward primers pOrig light no leader and pOrig heavy no leader in conjunction with the reverse primers huHeClonR and hiLiClonR respectively (Table 2). Amplified fragments were TA TOPO ligated into the vector pCR2.1 (Invitrogen) and clones confirmed by sequencing. Both the IB15 VH and VL regions devoid of leader were cloned back into pDCOrig IB15 using Hindlll/Afel and BamHI/BsiWI sites respectively. DNA sequence and translation for the VH and VL regions are shown in Figures 10 and 11 respectively.<br>
Construction of human lqG2 and lqG3 isotypes of the Immunobodv™ double expression vector pDCOrig<br>
The human lgG3 constant region was amplified by PCR using huigg3 forward and reverse primers (Table 2) incorporating a Afel and EcoRV respectively with the template pOTB7huigG3 (Image clone 4566267 MGC 45809). Similarly the human lgG2 constant region was amplified using igG2For and igG2Rev primers (Table 2) with the template pTOB7 huigG2 (Image clone 6281452 MGC 71314).<br>
Both fragments were TOPO ligated into pCR2.1 and sequence confirmed (Figures 12 and 13). The huigGI constant region within the construct<br>
 <br>
pDC0riglB15 was effectively replaced with both huigG2 and huigG3 cloned inframe with the heavy variable using Afel and Sapl sites to generate pDCOriglB15 huigG2 and pDCOriglB15huigG3 (FigureU). Both the vectors retain the same unique restriction sites at the variable/ constant region junction. This permits easy exchange of variable regions between all human isotype single and double chain Immunobody vectors.<br>
Mutation of human lqG1 Fcv and human lqG2 Receptor binding domain To substitute the amino acids E233 L234 L235 of the huigGI binding motif within the CH2 domain with P233 V234 A235 of huigG2, a short section was reamplified by PCR incorporating the mutation. The reverse primer huigGI PVA Rev containing the substitutions and the constitutive restriction site Ahdl was utilised with the forward primer HIBF (Table 2) and the template pDCOrig IB15. The resulting fragment was ligated into the vector pCR2.1 (Invitrogen). After sequence confirmation, the wild type sequence was effectively replaced with the section containing the mutations by inserting into the single cutter Agel/Ahdl sites of the plasmid pDCOrig IB15 huigGI (Figure 15).<br>
The amino acids P233 V234 A235 within the huigG2 constant domain of the construct pDCOrig IB15 huigG2 was also substituted with the huigGI binding motif ELLG. As before, the reverse primer huigG2ELLGRev (Table 2) containing the substitutions and the constitutive restriction site Ahdl was utilised with the forward primer HIBF and the template pDCOrig IB15 human igG2. The fragment was TATOPO ligated into the vector pCR2.1. After sequence confirmation, the wild type sequence again was replaced with the section containing the huigGI binding motif using Agel/Ahdl sites of the plasmid pDCOrig IB15 huigG2 (Figure 16).<br>
Generation of pDCOrig murine lgG2a plasmids DCIB53 and DCIB63 To construct a murine igG2a version of the double expression vector pDCOrig, cDNA was synthesised from total RNA isolated from the hybridoma<br>
cell line 337. For amplification of the murine igG2a constant region, the forward primer migG2aC1 AfeF2 containing the restriction site Afe1 was used in conjunction with the reverse primer migG2aXbaRA harbouring a Xbal site after the stop codon. PCR fragment was TOPO ligated into the vector pCR2.1. After sequence confirmation, the murine igG2a constant region was excised and cloned inframe with the murine heavy variable region into the Afe1/Xbal sites of the vector pOrigHIB effectively replacing human igG1. A BamHI and Xhol site was removed without altering, on translation, amino acid sequence from the murine igG2a constant region, sequentially by site directed mutagenesis using Quik change site directed mutagenesis kit (Stratagene) and the complimentary primers MoigG2BamHIFOR and REV, MoigG2XholFOR and REV respectively. This generated the single chain ImmunoBody vector pMoOrigHIB (Figure 17A). A section of pMoOrigHIB containing the MoigG2a constant region was transferred from the single construct into the double expression vector pDCOrig IB15 inframe with the murine heavy variable region using Afel and the single cutter Avrll located in the SV40 promoter to generate the intermediate vector pDCOriglB15MoigG2a hukappa still containing a human kappa region.<br>
For amplification of the murine kappa region, the cDNA was used as a template with the primers MoLCIBsiFI containing a BsiWI site and MoLCXhol incorporating a Xhol site after the stop codon. The amplified fragment was TOPO cloned into the vector pCR2.1 as before. The murine kappa region was excised and ligated into the ImmunoBody vector pOrigLIB L1 and pOrigLIB hepB help/L1 replacing the human kappa constant using BsiWI/Xhol generating the intermediate vector pMoLIBU Bsi and pMoLIB HepB help/L1 Bsi. The Immunobody system involves transfer of variable regions using a unique restriction site at the junction of the variable and constant regions while the junction between the murine heavy variable and moigG2a constant can accommodate an Afel site (present within all the human immunobody vectors) and not alter amino acid sequence on translation, the region between the murine variable and kappa is problematic. On analysis of<br>
sequence at this junction no unique restriction site could be incorporated that would not alter amino acid sequence. The BsiWI site at the junction was removed to revert to wild type sequence. This was achieved by amplifying the entire murine full length chain by overlapping PCR. A first PCR was set up using the forward primer MoKappaSDMfor containing wild type sequence at the junction and flanking region effectively removing BsiWI, the BGH reverse primer and the intermediate light chain vectors pMoLIBLI Bsi and pMoLIB hepB help/L1 Bsi as template respectively. Around a 430bp amplified fragment from the first round of PCR was used as a reverse primer with the forward primer ImmunoLikozFor containing a BamHI site. The amplified full length murine kappa chains were TOPO ligated into pCR2.1 and sequence confirmed. The full length murine kappa chain containing hepB help in the L1 site in pCR2.1 was excised and cloned into the BamHI/Xhol sites of the intermediate double expression vector pDCOriglB15MoigG2a hukappa replacing the human kappa chain to generate the murine double expression vector pDCOrigIB GP100210m/H1 TRP2/H2 HepB help/L1 molgG2a (DCIB 53, Figure17Band54).<br>
Similarly, the full length murine kappa chain containing an L1 site was excised and cloned into the BamHI/Xhol sites of the intermediate double expression vector pDCOriglB15MoigG2a hukappa replacing the human kappa chain to generate the intermediate murine double expression vector pDCOriglB15molgG2a with an empty L1 site. To generate the construct with a wild type light variable region, the complimentary 5' phosphorylated primers wtkappavarLlfor and rev (Table2) were annealed and inserted into the L1 site after linearization with EcoRV as described above. Finally the heavy variable region from DCIB 54 containing GP100DR7/H1 TRP2/H2 and GP100DR4/H3 was transferred using Hindlll/Afel to generate pDCOrig GP100DR7/H1 TRP2/H2 GP100DR4/H3 moigG2a wild type kappa (DCIB68 Figure17C and 60).<br>
Removal of the eukaryotic SV40 promoter from the Immunobody double expression vector pDCOriq for regulatory DNA vaccine requirements The EM7 bacterial promoter and zeocin gene was amplified using the forward primer SV40PremFOR incorporating a Nhe1 site and SV40remREV reverse primer (Table 2) with the template pOrigHIB. The resultant 511 bp PCR fragment was pCR2.1 TOPO ligated and confirmed by sequencing. The EM7 promoter and a section of the zeocin gene was excised using Nhel and Fsel from pCR2.1 and cloned directly into pOrigHIB H1 effectively removing the SV40 promoter. The Nhel site resides before the SV40 promoter while the Fsel recognition sequence is a single cutter within the zeocin gene of the vector. After sequence confirmation a larger section was transferred from the single vector into the pDCOrig IB68 vector encoding the tail end of huigGI, BGH polyA, EM7 and part of the zeocin gene digesting with Sapl and Fsel effectively removing the SV40 promoter from the double expression vector.<br>
Alteration of the pDCOriq backbone for the FDA regulatory compliant one of pVaxI (Invitroqen)<br>
The Immunobody full length human igG1 heavy chain was excised from the construct DCIB54 using Hindlll and Xbal and inserted into these sites within the MCS of the vector pVaxI (Figurel 8 A). In order to generate the pVax version of the double chain expression vector, pVaxlB54HIB was linearised using the blunt ended restriction endonuclease Nrul located adjacent to the CMV promoter. pOrigLIB (Figurel 8 B) was digested with the blunt ended Nrul and Hpal endonucleases to excise the entire light chain expression cassette consisting of the CMV promoter, Immunobody human kappa chain and the BGH polyA signal. After gel electrophoresis, isolation and gel extraction of the linerised vector pVaxlB54HIB and the light chain expression cassette the vector was dephosphorylated and light chain expression cassette ligated to form the construct pVaxDCIB54 (Figurel 8 C). Orientation of the light chain cassette within pVaxDCIB54 was confirmed by restriction analysis. pVaxDCIB54 retains the same unique restriction sites at the variable/ constant region junction permitting easy exchange of variable regions between all<br>
human isotype single and double chain Immunobody vectors. For example to generate pVaxDCIB68 (Figure 60) the murine light variable region containing Gp100DR4/L1 and Gp100DR7/L3 was excised from DCIB68 using BamHI/BsiWI and cloned into pVaxDCIB54 effectively replacing the light wild type variable region.<br>
Generation of pOriq murine TRP2 and pCDNA3 GP100 To construct pOrig murine TRP2, cDNA synthesised from 5|ig of total RNA isolated from the cell line B16F10 was used as a template for the amplification of full length murine tyrosinase related protein 2 (TRP2) using the primers murine TRP2 forward and reverse (Table 2) with incorporation of a Hindlll or EcoRV site respectively. Full length TRP2 was ligated into the Hindlll/EcoRV multiple cloning site of the vector pOrigHIB. Full length murine GP100 was also amplified from the cDNA using the designed murine GP100 forward and reverse primers containing EcoRV and Xhol sites respectively (Table 1). The PCR product was cloned into the EcoRV/Xhol sites of the mammalian expression vector pCDNA3 (Invitrogen). Both plasmids were identified by restriction analysis and confirmed by DNA sequencing.<br>
Sandwich Elisas<br>
Falcon 96-well flexible plates were coated, overnight at 4oC, with 50µl of anti-human IgG, Fc specific antibody (Sigma 12136) or anti-human kappa light chain antibody (Dako A0191) at 10µg/ml in PBS. Plates were washed three times with 200ul/well PBS-Tween 20 (0.05%), using a Skan Washer 400 (Molecular Devices), and wells blocked with 1% fish skin gelatin (Sigma) in PBS (1% FSG/PBS). Plates were incubated 1hr at room temperature and washed with 1% FSG/PBS. Tissue culture supernatant containing expressed ImmunoBody or purified ImmunoBody protein (50ul) was added to the wells, in triplicate, and plates were incubated for 1 hr at room temperature. Plates were washed with 1% FSG/PBS and bound ImmunoBody was detected by adding 50ul/well of peroxidase-conjugated anti-human IgG, Fc specific antibody (Sigma A0170) or anti-human kappa light chain antibody (Sigma<br>
A7164), diluted 1/2000 in 1% FSG/PBS, and incubated 1 hr at room temperature. Plates were washed with 1% FSG/PBS and developed by adding TMB substrate(R &amp; D Systems) at 50ul/well. Absorbance was measured at 650nm in a VERSA max microplate reader (Molecular Devices).<br>
Mice and immunisations<br>
Animal work was carried out under a Home Office approved project licence. Male and female C57BI/6 (Harlan) or HLA-A2 transgenic (HHDII) (Pasteur Institute, Paris) were used between 6 and 12 weeks of age. Synthetic peptides (manufactured by John Keyte, Department of Biomedical Sciences, Nottingham University, UK) were emulsified with incomplete Freunds adjuvant and injected via a sub-cutaneous route. Each mouse received 10µg peptide/immunisation. DNA was coated onto 1 .0µm gold particles (BioRad, Hemel Hempstead, UK) using the manufacturer's instructions and administered intradermally by the Helios Gene Gun (BioRad). Each mouse received 1µg DNA/immunisation. Naked DNA solution was also administered i.d. or i.m (10µg/immunisation) combined immediately post injection with a short electric pulse. Mice were immunised at 0,1 and 2 weeks and spleens removed at week 3. Depletion of T cell subsets in vivo was performed by injection of 400ug anti-CD25 antibody (PC61) i.p. four days prior to immunisation or 200ug anti-CTLA-4 antibody i.p. concurrent with secondary immunisation.<br>
Restimulations in vitro<br>
Five days post final immunisation, splenocytes (5x106/ml) were cocultured at 37°C with syngeneic, irradiated (20Gy), peptide pulsed lipopolysaccharide (LPS) blasts (0.5 to 1x106 cells/ml) in 2ml RPMI-1644 with 10% FBS, 2mM glutamine, 20mM HEPES buffer, 100 units/ml penicillin, 100µg/ml-1 streptomycin and 10-5M 2-mercaptoethanol in 24 well plates. LPS blasts were obtained by activating splenocytes (1.5x106 cells/ml) with 25µg/ml LPS (Sigma) and 7µg/ml dextran sulphate (Pharmacia, Milton Keynes, UK) for 3 days at 37'C. Before use, 2x107 LPS blasts were cultured with 100µg/ml synthetic<br>
peptide for 1hr. Cultures were assayed for cytotoxic activity on day 6 in a 51Cr-release assay.<br>
51Cr-release assay<br>
Target cells were labelled for 1hr with 1.85MBq sodium (51Cr) chromate (Amersham, Essex, UK) with or without 100µg/ml peptide. Pcst incubation they were washed 3 times in RPMI and incubated for a further 1 hr with 100µg/ml peptide. 5x103 targets/well of 96-well V-bottomed plates were set up and coincubated with different densities of effector cells in a final volume of 200pl. After 4hrs at 37°C, 50µl of supernatants were removed from each well and transferred to a Lumaplate (Packard, Rigaweg, the Netherlands). Plates were read on a Topcount Microplate Scintillation Counter (Packard). Percentage specific lysis was calculated using the following formula:<br>
specific lysis = 100x[(experimental release-spontaneous release)/(maximum release-spontaneous release)]<br>
Ex vivo Elispot assay<br>
Elispot assays were performed using murine IFN capture and detection reagents according to the manufacturer's instructions (Mabtech, Sweden). In brief, anti-IFN antibodies were coated onto wells of 96-well Immobilin-P plate and replicate wells were seeded with 5x105 splenocytes. Synthetic peptides (at a variety of concentrations) or 5x104 target melanoma cells were added to these wells and incubated for 40hrs at 37°C. After incubation, captured IFNwas detected with by a biotinylated anti-IFN antibody and development with a strepatavidin alkaline phosphatase and chromogenic substrate. Spots were analysed and counted using an automated plate reader (CTL). Functional avidity was calculated as the concentration mediating 50% maximal effector function using a graph of effector function versus peptide concentration. Depletion of CD8 T cells from splenocyte populations was performed using CD8 Dynabeads (Dynal) according to manufacturer's instructions and then added to ex vivo elispot assay.<br>
Tumour studies<br>
C57BI/6 mice were randomised into treatment groups and immunised at weekly intervals for five weeks. Between the third and fourth immunisation they were challenged by i.v. injection into the tail vein with 1x104 B16F10 IFNa melanoma cells. When injected i.v., B16F10 cells migrate to the lungs to form metastases. Mice were monitored for signs of tumour growth and distress. At day 49 post tumour challenge, mice were euthanised and lungs analysed for the presence of metastases. Spleens were analysed for the presence of epitope and tumour specific immune responses in ex vivo elispot assay.<br>
HHDII mice were immunised at weekly intervals for three weeks and 7 days post-final immunisation were challenged s.c. in the right flank with 2x104 B16F10 HHD melanoma cells. Tumour growth was monitored at 3-4 day intervals and size of the tumour was measured using a calliper.<br>
Example 1 - ImmunoBody constructs produce low levels of intact antibody<br>
Stable CHO-S cell transfectants were made with an ImmunoBody construct containing the gp100 epitope IMDQVPFSV and the TRP2 epitope SVYDFFVWL in CDR H1 and CDR H2 respectively with the HepB CD4 epitope TPPAYRPPNAPIL in CDR L1 (DCIB15; Figure 19).<br>
The supernatant from these transfectants was analysed for expression of ImmunoBody protein by sandwich elisa. Plates were coated with anti-human IgG Fc specific antibody and supernatant added. Bound ImmunoBody was detected using an anti-human Fc specific HRP antibody to detect heavy chain. Heavy chain was detected in the supernatant at a concentration of approximately 1 µg/ml compared to the control (Figure 20a). ImmunoBody was purified from the supernatant using a protein A affinity column and analysed for presence of ImmunoBody. Purification of ImmunoBody yielded far lower<br>
quantities of protein than previously expected compared to the control (Figure 20b). Since such low yields of intact protein could be purified, ImmunoBody constructs were analysed for the expression of both heavy chain and intact antibody in the supernatant of transfected cells by sandwich ELISA. Constructs with the HepB CD4 epitope in CDR L1 and the SIINFEKL epitope in CDR H2 (DCIB24; Figure 21) or the gp100 epitope IMDQVPFSV and the TRP2 epitope SVYDFFVWL in CDR H1 and CDR H2 respectively with the HepB CD4 epitope TPPAYRPPNAPIL in CDR L3 (DCIB25; Figure 22) were also tested. Plates were coated with anti-human IgG Fc specific antibody and supernatant added. Bound ImmunoBody was detected using an anti-human Fc specific HRP antibody to detect heavy chain or an anti-human kappa chain specific HRP antibody to detect intact ImmunoBody. ImmunoBody transfectants show high level of heavy chain secretion but very low levels of intact ImmunoBody (Figure 20c and d).<br>
This data indicates that the incorporation of CD8 and CD4 T cell epitopes into the heavy and light chain variable regions has disrupted the overall structure of the ImmunoBody preventing formation of intact antibody.<br>
Additional data on analysis of supernatant from transfected CHO-S cells demonstrates that only constructs with CTL epitopes incorporated into the CDRH3 or CDRL3 are secreted as intact antibody (Figure 20e). In contrast, incorporation of any epitope within the CDRH1 or CDRH2 allowed secretion of heavy but low amounts of intact antibody even if there was nothing incorporated within the light chain and it was secreted. Incorporation of any epitope within CDRL1 any of the light chain resulted in low level secretion of light chain even if there was only an epitopes incorporated into the CDRH3 of the heavy chain.<br>
Example 2 - CTL epitopes incorporated into ImmunoBody framework are processed and presented to elicit an immune response in vivo<br>
The previously-published CTL epitope from TRP2, aa280-288 (Bloom et al, The Journal of Experimental Medicine 1997;185:453-9), was engineered into the CDR H2 region of the ImmunoBody construct alongside a Hepatitis B universal CD4 epitope in CDR L1 (DCIB18; Figure 30). C57BI/6 mice were immunised three times at weekly intervals intradermally with ImmunoBody DNA via the gene gun. Splenocytes were subsequently analysed by IFNelispot for TRP2 specific responses. Mice immunised with ImmunoBody DNA demonstrated considerable TRP2 peptide specific responses compared to control but lower level responses specific for the HepB CD4 peptide (Figure 31a). The avidity of the TRP2 specific responses were also studied by peptide titration in IFN elispot. Over the fifteen mice tested within five different experiments, the avidity of the responses ranges from 10-9 M to 10-11 M peptide. A representative example is showa in Figure 31b.<br>
In order to confirm that this TRP2 specific response was mediated by CD8 T cells, the C57BI/6 mice were immunised three times with ImmunoBody DNA at weekly intervals. Six days after the last immunisation splenocytes were isolated and analysed in vitro for specific responses by IFN elispot. To determine if the TRP-2 specific response was mediated by CD8 T cells, CD8 T cells were depleted prior to analysis in elispot assay. Depletion of CD8 T cells led to abolition of the TRP2-specific response; however CD8 depletion did not affect the HepB CD4 peptide response, suggesting it is most likely mediated by CD4 T cells (Figure 31c).<br>
To determine if the responses generated by ImmunoBody DNA immunisation are capable of killing target cells in vitro, splenocytes were stimulated with TRP2 peptide pulsed LPS blasts in vitro for 6 days and analysed in a chromium release assay against B16F10 melanoma cells. Splenocytes from ImmunoBody DNA immunised mice demonstrated superior lysis of both B16F10 cells, which have low levels of surface MHC class I, and of B16F10 IFNα cells, which have high surface MHC class I expression compared to that of B16F10 line that expresses no H-2Kb molecules (B16F10 siKb). The<br>
abolition of killing against the B16F10 siKb cell line demonstrates that killing is CD8 dependent and restricted through H-2Kb (Figure 31 d).<br>
These results show that TRP2 (SVYDFFVWL) CD8 epitope incorporated into the CDR H2 region of the ImmunoBody framework is processed and presented to elicit high frequency responses mediated via MHC class I. The HepB CD4 epitope is also processed and presented in the context of MHC class II to elicit good CD4 mediated responses from DNA immunisation.<br>
TRP2 epitope specific responses were also analysed from other TRP2 epitope containing constructs using identical methodology. Incorporation of the TRP2 epitope into CDRs within the heavy chain resulted in high frequency peptide specific responses (Figure 31 e). In contrast incorporation of CTL epitopes within the light chain resulted in a significant reduction in CTL frequency (DCIB36). Analysis of the avidity of the TRP2 epitope specific responses reveals that they are of high avidity when generated from epitopes within the heavy chain but this is considerably lower upon expression of epitopes from the light chain (Figure 31 f). High frequency high avidity helper responses where observed for all constructs (Figure 31 g). Suggesting that secretion of heavy chain was an advantage for stimulating CTL responses but not for helper responses.<br>
Example 3 - ImmunoBody DNA immunisation is better than peptide immunisation or immunisation with whole antigen<br>
To analyse the efficiency of ImmunoBody DNA immunisation, it was compared to s.c. immunisation with peptide epitope in Incomplete Freund's adjuvant or immunisation with a DNA expressing the TRP2 antigen.<br>
C57BI/6 mice received three weekly immunisations with DNA or peptide comprising of the TRP2 epitope linked to the universal helper epitope in IFA. TRP2 and helper peptide specific responses generated in ImmunoBody<br>
immunised mice were far superior in magnitude to those elicited by peptide immunisation or immunisation with the whole TRP2 antigen (Figure 32a). Further analysis of the avidity of these peptide specific responses revealed that responses generated by mice immunised with ImmunoBody DNA have greater than a log higher avidity than those from peptide immunised individuals (Figure 32b). The responses generated in C57BI/6 mice were subsequently analysed for cytotoxic ability in vitro against the B16F10 cell line and, as a negative control, the B16F10 siKb cell line. Figure 32c shows that ImmunoBody DNA immunised mice are capable of anti-tumour activity in vitro that is H-2Kb restricted and both peptide immunised mice and whole antigen immunised mice are unsuccessful at killing the same melanoma cell lines.<br>
ImmunoBody immunisation was also compared to immunisation with DC + peptide. C57BI/6 mice received three weekly immunisations with DNA or DC + peptide. TRP2 peptide specific responses were of comparable frequency but ImmunoBody immunised mice generated higher avidity responses compared to those immunised with DC + peptide (Figure 32d). This is also demonstrated when these responses were analysed for ability to kill B16F10 melanoma cells in vitro (Figure 32e). The responses generated by ImmunoBody immunisation showed higher killing of B16F10 melanoma at lower effector to target ratio than responses from DC + peptide immunised mice. They also showed higher specific lysis of the B16F20 siKb melanoma line which has knocked down levels of H-2Kb.<br>
ImmunoBody constructs containing the H-2Kb restricted Ovalbumin epitope, SIINFEKL, and the anchor modified HLA-A2 restricted gp100 epitope, IMDQVPFSV (210M) were compared with the corresponding epitope peptide immunisation in C57BI/6 or HHDII mice respectively. Mice received three weekly immunisations with DNA or peptide in IFA. Analysis of the responses after the final immunisation reveals that ImmunoBody DNA immunised mice generate higher frequency peptide specific responses compared to peptide immunised mice (Figure 32f and g). These responses were also analysed for<br>
 <br>
avidity by peptide titration. ImmunoBody immunisation elicits significantly higher avidity responses than peptide immunisation (Figure 32h and i).<br>
The magnitude of TRP2 specific response generated by the ImmunoBody DNA vaccine is far superior to that generated by either synthetic peptide or whole TRP2 antigen. However, evidence from clinical trials suggests that the presence of a high frequency of tumour specific CD8 T cells does not necessarily lead to tumour regression and generally in vaccine trials the objective clinical response rate is very low (Rosenberg et al, J Immunol 2005;175: 6169-76; Rosenberg etal, Nature Medicine2004;10: 909-15). It is now becoming clear that factors other than frequency such as functional avidity of tumour specific T cells and route of priming are major determinants in maximising vaccine efficacy. A number of groups have shown that high avidity CD8 T cells demonstrate superior anti-tumour activity (Alexander-Miller, Immunologic research, 2005;31:13-24; Hodge etal, J Immunol 2005;174: 5994-6004; Valmori etal, J Immunol 2002;168: 4231-40; Zeh etal, J Immunol 1999;162: 989-94; Alexander-Miller etal, Proceedings of the National Academy of Sciences of the United States of America 1996; 93: 4102-7). In our study, analysis of the functional avidity of ImmunoBody induced TRP2 specific responses demonstrated that a high avidity response can be generated when compared to immunisation with synthetic peptide. This high avidity response also correlated with the enhanced ability to recognise and kill tumour cells in vitro. The signal from the APC or route of priming of the response is also crucial for the induction of high avidity immune responses (Oh etal, J Immunol 2003;170: 2523-30).<br>
Example 4 - Multiple epitopes can be processed from CDR H2 site<br>
To demonstrate that multiple epitopes can be processed and presented from CDR H2 to elicit an immune response, the H-2Kb restricted epitope SIINFEKL (DCIB24; Figure 21) from ovalbumin and the H-2Kd restricted Hepatitis B epitope IPQSLDSWWTSL (DCIB21; Figure 33) were engineered into the H2<br>
site in the heavy variable region. These ImmunoBody constructs also contained a l-Ab restricted (TPPAYRPPNAPIL) epitope Hepatitis B CD4 epitope or l-Ad restricted Influenza haemagluttinin (FERFEIFPKE) epitope in the CDR L1 site in the light variable region.<br>
C57BI/6 or Balb/c mice were immunised three times at weekly intervals intradermally with ImmunoBody DNA via the gene gun. Splenocytes were subsequently analysed by IFN elispot for the presence of epitope specific CD8 and CD4 responses.<br>
C57BI/6 immunised mice demonstrated high frequency SIINFEKL specific responses but lower responses specific for the helper epitope (Figure 34a). Balb/c mice also created high frequency Hepatitis B epitope specific CD8 responses with similar level responses to the helper epitope (Figure 34b).<br>
This data suggests that processing and presentation of CD8 epitopes from the CDR H2 site is not restricted by specific epitope sequence or length.<br>
Example 5 - Multiple CTL epitopes can be processed from the variable region<br>
To demonstrate that epitopes can be processed and presented from the variable region and not solely the CDR regions, epitopes were incorporated into the CDR H1 site with the removal of part of the framework region.<br>
Example epitopes are the modified HLA-A2 restricted epitopes IMDQVPFSV (DCIB17; Figure 35) from gp100 and FLPATLTMV from Tie-2 (DCIB26; Figure 36). ImmunoBody constructs also contained the Hepatitis B CD4 epitope in the CDR L1 site.<br>
HLA-A2 transgenic mice (HHDII) mice were immunised three times at weekly intervals intradermally with ImmunoBody DNA via the gene gun. Splenocytes<br>
were subsequently analysed by IFNy elispot for the presence of epitope specific CD8 and CD4 responses.<br>
HHDII mice elicited high frequency gp100 210M epitope specific responses with reasonable responses to the HepB CD4 epitope (Figure 37a). Responses in HHDII mice immunised with the Tie2 epitope containing construct were not of as high frequency but considerable responses were generated specific for both the Tie2 epitope and the HepB CD4 epitope (Figure 37b).<br>
Data in this example indicates that epitopes inserted within the variable region can be processed and presented to elicit an immune response in vivo. It is also apparent that this is not restricted to one epitope sequence.<br>
Example 6 - Multiple CTL responses can be generated from different epitopes within the same ImmunoBody construct<br>
The previously-mentioned HLA-A2 restricted gp100 epitope IMDQVPFSV was engineered into the CDR H1 site alongside the TRP2 epitope SVYDFFVWL which is also restricted through HLA-A2 in the CDR H2 site of the same construct. The HepB CD4 epitope was present in the CDR L1 site (DCIB15; Figure 19).<br>
HHDII mice were immunised three times at weekly intervals intradermally with ImmunoBody DNA via the gene gun. Splenocytes were subsequently analysed by IFN elispot for the presence of epitope specific CD8 and CD4 responses.<br>
Figure 38a shows that responses are generated specific for both the gp100 and TRP2 epitopes, although the frequency of the TRP2 specific responses are lower. Responses to the HepB CD4 peptide are also generated. The avidity of the TRP2 specific responses were also studied by peptide titration in IFN elispot. The avidity of the responses ranges from 10-10 M to 10-11 M<br>
peptide for the gp100 epitope and 10-9 M to 10-10 M peptide for the TRP2 epitope. Representative examples are shown in Figure 38b. To determine if the responses are capable of killing target cells in vitro, splenocytes were stimulated with TRP2 and gp100 peptide pulsed LPS blasts in vitro for 6 days and analysed in a chromium release assay against peptide labelled T2 cells and B16F10 HHD melanoma cells. Specific killing of B16F10 HHD melanoma line compared to the control B16F10 melanoma line. Responses also demonstrated specific lysis of peptide labelled T2 cells compared to control (Figure 38c).<br>
Combining two CD8 epitopes in a single ImmunoBody construct appears to result in a degree of immunodominance between epitopes. The immunodominant epitope is the epitope with the highest affinity for MHC class I. When mice are immunised with the construct containing both gp100 and TRP2 CD8 epitopes are compared to those immunised with a construct containing only the TRP2 CD8 epitope, the frequency of the TRP2 response decreases (Figure 38d).<br>
This data demonstrates that epitope specific immune responses can be generated from the same DNA construct specific for two different CD8 epitopes. These are also capable of anti-tumour activity in vitro. However, there is a degree of immunodominance that governs the frequency of the response to the subdominant epitope.<br>
A similar study was performed with separate ImmunoBody constructs containing the TRP2 epitope in CDRH2 (DCIB18) or the SIINFEKL epitope in CDRH2 (DCIB24). Mice were immunised with either DCIB18 or DCIB24 alone, DCIB18 and DCIB24 combined in the same site or DCIB18 and DCIB24 at the same time but in separate sites. Immunisations were performed three times at weekly intervals and DNA was injected i.m in the tibialis muscle combined with electroporation. Analysis of the immune responses generated shows that high frequency peptide specific responses can be elicited when mice were<br>
immunised with DCI818 or DCIB24 alone (Figure 38e). Immunising mice with these constructs in the same site results in significant loss of the TRP2 peptide specific response. This suggests that the SIINFEKL epitope is dominant over the TRP2 epitope. The TRP2 specific response can be recovered if mice are immunised with constructs in separate sites (p=0.0026). This data suggests that immunodominance does influence immune responses generated by IB immunisation but this can be resolved by immunisation in spatially separate sites.<br>
Example 7 - Non anchor residue modifications can enhance T cell recognition<br>
The previous example shows that the modified gp100 epitope IMDQVPFSV is immunodominant and has a high affinity for HLA-A2 (predicted using the SYFPEITHI algorithm and demonstrated in T2 stabilisation assay - Table 5). Since the wild type gp100 epitope ITDQVPFSV is not immunogenic, modifications were made at non anchor residues that would have a similar HLA-A2 binding affinity to the wild type epitope but also enhance the immunogenicity. These modified epitopes were engineered into the CDR H1 site of the ImmunoBody construct and tested alongside the wild type epitope (DCIB37, DCIB40, DCIB41, DCIB42, DCIB43; Figures 39-43).<br>
HHDII mice were immunised three times at weekly intervals intradermally with ImmunoBody heavy chain DNA alone via the gene gun. Splenocytes were subsequently analysed by IFNelispot for the presence of epitope specific CD8 responses. Two modifications (F7L and F7I; DCIB37; Figure 39, DCIB40; Figure 40) to the wild type gp100 epitope which retain affinity for HLA-A2 (Table 5) demonstrated superior ability to induce epitope specific immune responses compared to the wild type epitope (Figure 44a).<br>
Table 5<br>
<br>
(Table Removed)<br>
Example 8 - Multiple CD4 helper responses can be processed and presented to elicit an immune response in vivo<br>
To examine if CD4 helper epitopes could be processed and presented to elicit an immune response in vivo, different epitopes were engineered independently into the CDR L1 site of the ImmunoBody construct. These included the l-Ad restricted epitope FERFEIFPKE (DCIB21; Figure 33) from Influenza haemagluttinin, the l-Ab restricted epitope TPPAYRPPNAPIL from HBcAg (DCIB15; Figure 19) and the HLA-DR4 restricted epitope WNRQLYPEWTEAQRLD from gp100 (DCIB35; Figure 45).<br>
Balb/c, C57BI/6 or HHDII and DR4 transgenic mice were immunised three times at weekly intervals intradermally with ImmunoBody DNA via the gene gun. Splenocytes were subsequently analysed by IFN elispot for the presence of epitope specific CD4 responses. Figures 46a, b and c demonstrate that all three CD4 helper epitopes can be processed and presented from the CDR L1 site to elict an epitope specific immune response in vivo.<br>
The gp100 HLA-DR4 restricted epitope was also tested for processing and presentation from different CDRs. Constructs incorporating the epitope into CDRL1 (DCIB35; Figure 45), CDRH3 (DCIB54; Figure 29) or CDRL3 (DCIB50; Figure 47) were used to immunise HLA-DR4 transgenic mice three times at weekly intervals. Figure 46d shows that helper epitope can be efficiently processed from different CDRs to elicit high frequency helper responses.<br>
Example 9 - CTL responses are partially dependent upon secreted heavy chain but helper responses do not require secreted light chain<br>
Classically CD4 T cell epitopes are processed from proteins that are acquired exogenousiy and CD8 T cell epitopes from endogenously produced proteins. There is evidence now for the cross presentation of epitopes from exogenousiy acquired antigen to elicit a CD8 T cell mediated response. This route of priming has also been proposed to be more efficient in the development of CD8 T cell-mediated immune responses. Recently there have been similar findings for CD4-mediated responses. Mounting evidence suggests that CD4 T cell epitopes derived from intracellular proteins can be processed and presented in the context of MHC class II.<br>
In order to determine if secreted ImmunoBody is required for the induction of CD8 and CD4 T cell responses, ImmunoBody constructs containing the HLA-A2 restricted gp100 epitope IMDQVPFSV in the CDR H1 site and the l-Ab restricted HepB helper epitope TPPAYRPPNAPIL in the CDR L1 site were made without leader sequences on the heavy chain or light chain (Figures 10 and 11).<br>
HHDII mice were immunised three times at weekly intervals intradermally with ImmunoBody DNA via the gene gun. Splenocytes were subsequently analysed by IFNelispot for the presence of epitope specific CD8 and CD4 T cell responses. When the responses were analysed for gp100 specific CD8 response, it was observed that removal of the leader sequence from the heavy<br>
chain of the ImmunoBody construct resulted in a decrease in epitope specific responses however the CD4 responses was not affected (Figure 48a). Removal of the leader sequence from the heavy chain affected secretion of heavy chain by transfected CHO-S cells (Figure 48b). Removal of the leader sequence from the light chain, thus preventing light chain secretion, did not appear to affect the epitope specific CD8 or CD4 responses (Figure 48c). CD8 responses were significantly reduced in the absence of a leader sequence on the heavy chain but CD4 responses remained unaffected (Figure 48c and d).<br>
This data implies that the secretion of heavy chain is important for the efficient induction of a CD8 T cell response, suggesting that CD8 epitopes are undergoing cross presentation. Secondly, it implies that CD4 epitopes are derived from intracellular ImmunoBody to elicit an immune response.<br>
Example 10 - Reduced CTL responses without Fc due to lack of protein secretion<br>
This experiment examines whether the presence of the Fc region is beneficial for establishing an efficient immune response. The Fc region has been removed from the ImmunoBody construct, containing the H-2Kb restricted TRP2 epitope SVYDFFVWL in CDR H2 and the l-Ab restricted HepB CD4 epitope TPPAYRPPNAPIL in CDR L1 (DCIB15), by incorporating a stop codon before the Fc to prevent transcription and translation (Figure 9).<br>
C57BI/6 mice were immunised three times at weekly intervals intradermally with ImmunoBody DNA via the gene gun. Splenocytes were subsequently analysed by IFNelispot for the presence of epitope specific CD8 and CD4 T cell responses.<br>
Mice immunised with the ImmunoBody construct lacking the Fc region generated a low level TRP2 peptide specific response that was capable of very low level recognition of the tumour cell line B16F10 compared to a construct<br>
with the Fc region (Figure 49a). Analysis of both the TRP2 and HepB helper peptide specific responses from a number of experiments demonstrates that constructs lacking the Fc region generate significantly lower TRP2 peptide specific responses (Figure 49b). However the HepB helper responses is unaffected by removal of the Fc region (Figure 49c). This is consistent with our previous results showing that help works best in the light chain where it is not secreted and is therefore working by direct presentation. In contrast CTL responses are stimulated by both direct and indirect presentation and the latter may benefit from Fc targeting. Alternatively the Fc stop construct results in lower secretion of the truncated heavy chain which may explain the reduced response. An ImmunoBody encoding TRP-2 was therefore engineered with an lgG2 (DCIB33) and an lgG3 constant region (DCIB65) the former should not bind to CD64 but can bind to CD32 and may also bind to Fc receptor IV in mice. Human lgG3 can bind to both CD32 and CD64. Both ImmunoBodies stimulated strong CTL responses (Figure 49e). This suggests that Fc targeting is not a strong component of the indirect presentation. To further verify this issue, the Fc targeting domain of lgG1 was replaced with the equivalent lgG2 domain and vica versa (DCIB66, 67, Figures 15 and 16). Both constructs stimulated strong CTL responses (Figure 49e). This may be due to the ImmunoBody™ vaccines only secreting heavy chain which may not associate and allow Fc binding (Figure 49f and g).<br>
Example 11- ImmunoBody immunisation enhances immune responses and overcomes regulation observed from whole antigen. It also allows identification of new heterologous T cell epitopes.<br>
This may lead to the second benefit of immunising with a human antibody encoding T cell epitope which is that, in contrast to most self antigens, it is an inert carrier that does not express regulatory epitopes. An ImmunoBody™ expressing either a gp100 epitope or a TRP-2 epitope stimulated a high frequency, high avidity T cell response (frequency 1/103 avidity 10-10M) whereas immunisation with the whole gp100 of TRP-2 antigen stimulated T<br>
cells with low frequency and avidity (frequency 1/105 avidity 10-7M). CD25 depletion partially restored the response to the antigen but ImmunoBody was still 100 fold superior (Figure 50a and b).<br>
Similarly immunisation with DNA encoding the first 200 amino acids of Tie-2 linked to-Fc, failed to stimulate an immune response to the top 10 predicted epitopes. The sequence of the first 196 amino acids of Tie-2 was entered into the EpiJen and NetCTL online prediction algorithms. Both of these methods take into account proteasomal cleavage and TAP transport in addition to predicting HLA-A*0201 binding affinity. The MHCpred and Syfpeithi algorithms were also used as examples of the older prediction algorithms that only take into account predicted MHC binding affinity. The whole Tie-2 molecule could contain additional CTL epitopes that may exert an immunodominant effect over those present in the first 196 amino acids. The whole sequence of Tie-2 was therefore also entered into the same algorithms in order to obtain the ranks of each predicted epitope from the whole molecule. Peptides that were not homologous in mouse and man were discounted. Six of the remaining peptides that were consistently predicted to represent good CTL epitopes by several different prediction algorithms were selected. The relative scores obtained with the different algorithms for each of these peptides, along with the results for Z83 (a previously identified epitope), are summarised in Table 6.<br>
Additional data on analysis of supernatant from transfected CHO-S cells demonstrates that only constructs with CTL epitopes incorporated into the CDRH3 or CDRL3 are secreted as intact antibody (Figure 20e). In contrast, incorporation of any epitope within the CDRH1 or CDRH2 allowed secretion of heavy but low amounts of intact antibody even if there was nothing incorporated within the light chain and it was secreted. Incorporation of any epitope within CDRL1 any of the light chain resulted in low level secretion of light chain even if there was only an epitopes incorporated into the CDRH3 of the heavy chain.<br>
In order to determine whether a T cell repertoire exists in HLA-A*0201 transgenic mice that recognizes any of the predicted CTL epitopes from Tie-2, animals were immunised with the native Tie2 C200hFc DNA construct (Ramage et al, Int. J. Cancer 2004;110:245-250) and splenocytes were screened for peptide specific IFNy responses in an ELISPOT assay. A separate group of mice were immunized with C200hFc following treatment with PC61 mAb, as before, 4 days prior to DNA immunisation.<br>
Mice that were immunised with the native C200HFc DNA construct did not mount an IFNy response that recognised Z83, regardless of whether the animals were depleted of CD25+ regulatory T cells prior to immunisation or not. There were no significant IFN responses to any of the new peptides tested from animals that were not depleted of regulatory T cells prior to immunisation, with the exception of Z284 which appeared to stimulate a response in one animal (M3) with a mean of 69 SFC/million splenocytes (Figure 50c and d). From the animals that were depleted of regulatory T cells prior to DNA immunisation, 2/3 animals (M1 and M3) demonstrated an IFNresponse to restimulation with Z282 peptide, with mean values of 320 and 94 SFC/million splenocytes respectively. M1 also demonstrated a partial response to restimulation with Z285, with a mean of 85 SFC/million splenocytes.<br>
The apparently conflicting results from the in vivo screen of the predicted CD8+ epitopes from Tie-2 could be the result of immunodominance, as the IFN responses from mice that were immunized with the native C200HFc construct in the absence of CD25+ cells appeared to be skewed towards one predominant peptide. In order to further investigate the T cell repertoire that is available to respond to the Z282 epitope, in the absence of competition from other potential CD8+ epitopes, a group of HHD mice were immunizea with the Z282 peptide in I FA in the presence or absence of CD25+ regulatory T cells.<br>
 <br>
All of the mice immunised with Z282 mounted peptide-specific IFNresponses, even when immunised in the presence of CD25+ regulatory T cells. Mouse 3 of the non-depleted animals mounted the highest response, with a mean value of 215 SFC/million cells. The highest response from the depleted animals was observed from mouse 2 with a mean value of 137 SFC/million cells (Figure 50e and f).<br>
Responses induced by peptide immunisation remain of low frequency. To examine if higher frequency responses can be generated if the epitope is removed from any regulatory influence generated by the whole antigen, the z282 (also known as z12) epitope was engineered into the H1 site of an ImmunoBody construct alongside Hep B CD4 in L1 (DCIB71, Figure 51) HLA-A2 transgenic mice were then immunised with z12 peptide or ImmunoBody DNA (via gene gun) three times at weekly intervals and then analysed for the presence of epitope specific immune responses. All mice immunised with z12 peptide exhibit low frequency and avidity epitope specific responses (Figure 50g). However when the z12 epitope is engineered into the ImmunoBody construct higher frequency and avidity responses are induced in all mice (Figure 50h).<br>
To summarize, if CD25 cells were depleted prior to immunisation an immune response was stimulated to 3/10 of the Tie2 epitopes. Similarly if one of these epitopes was presented as a peptide, weak immune responses could be generated. However if this epitope was presented within an ImmunoBody™ construct high frequency and high avidity T cell responses were generated. These results suggest that there are T-reg epitopes within the first 200 amino acids of Tie-2 which inhibit CTL responses. If these T-regs or their epitopes are removed it is possible to uncover a response to self antigens which can be further enhanced by presentation within an ImmunoBody.<br>
<br>
(Table Removed)<br>
Table 6. Predicted HLA-A*0201 restricted CTL epitopes from Tie-2. 1 Name of peptide. 2 Amino acid residue start position within Tie-2 molecule. 3 Peptide sequence   4 Prediction using the EpiJen web server. The score is given in units of IC50nM, with lower scores representing higher affinity peptides  5 Prediction using the NetCTL 1.2 web server. Score represents the weighted sum of three individual prediction methods, with a relative weighting on MHC binding of 1. * indicates a score above the threshold value of 0.75 identified as the cut off point for CTL epitopes from the dataset obtained for known epitopes. 6 Prediction using the SYFPEITHI programme. Maximal score for HLA-A*0201 binding peptides is 36. 7 Prediction using the MHCPred additive method to predict peptide affinity for MHC and TAP. The score is again given in units of IC50nM, with lower scores representing higher affinity peptides. Suggested IC50 values are between 0.01 to 5000 nM. For all prediction methods, the rank values indicate the order in which epitopes are predicted from the 196 amino acid fragment, with values in brackets representing the rank predictions from the whole Tie-2 molecule. Values obtained for the known Z18 CTL epitope derived from the matrix protein of Influenza A virus are included for comparison.<br>
Example 12 - The role of xenogenic Fc in providing T cell help and the requirement for antigen specific T cell help<br>
Stimulation of high avidity T cell responses usually requires T cell help during the priming. It was originally conceived that this would be provided by the Hep B foreign helper epitope encoded within the light chain. Indeed strong helper responses were generated to this epitope. However as the heavy chain was secreted and the light chain was not although the hep B epitope could have provided help for direct presentation when both chains would be produced by the same APC it is unlikely that it could be providing help for the indirectly presented heavy chain as this is unlikely to be taken up by the same antigen presenting cell. Mice were therefore immunised with a DNA vector only encoding heavy chain. High frequency, high avidity CTL responses were still generated (Figure 53 a and b). This implies that either help is not required or that the human Fc which is xenogenic in mice is providing linked foreign help.   A mouse lgG2a construct was therefore assessed for secretion of Heavy and light chains (Figure 49g) and screened for generation of immune responses (DCIB53 figure 54). Although it still gave high frequency high avidity T cell responses these were not as strong as the equivalent human construct suggesting that the xenogenic Fc was providing linked help (Figure 53c and d). An HLA-DR4 gp100 epitope was then incorporated into the mouse lgG2a construct (DCIB64, Figure 55) to provide both linked help for CTL generation but also antigen specific T cell help to stimulate inflammation within the tumour environment. These constructs stimulate high frequency and high avidity CTL and helper responses (Figure 53e and f). A hlgG1 construct expressing the same epitope can be used in human patients.<br>
Example 13 - Immunoproteasome processing is important in the generation of responses from epitopes within ImmunoBody constructs<br>
It has been suggested that the immunoproteasome has the ability to alter the array of epitopes generated from self antigens as it possess a different pattern<br>
of cleavage. In some cases, new epitopes are generated upon upregulation of the immunoproteasome and in others epitopes are destroyed. There is evidence that the immunoproteasome is unable to generate several epitopes derived from melanoma antigens namely MelanA/MART-1, gp100 209-217 and Tyrosinase 369'377 (Chapiro ef a/2006. J Immunol; 176:1053-61). Chapiro and colleagues have suggested that the ability to process and present the gp100 epitope is related to the upregulation of the immunoproteasome. Mature DCs are believed to be responsible for the priming of immune responses and are known to constitutively express the immunoproteasome (Macagno et al. 2001. Eur J Immunol; 31: 3271 -80). The gp100209-217 epitope was therefore engineered into the CDRH1 site of an ImmunoBody construct and tested for its ability to induce peptide specific immune responses in HLA-A2 transgenic mice. No peptide specific responses were observed from this construct (Figure 56). However when the epitope was modified to possess a methionine at position 210 (210M) instead of threonine this prevents its cleavage by the immunoproteasome and epitope specific responses were observed (Figure 56).<br>
A HLA-A2 restricted peptide derived from VEGFR2 (aa 773-781 VIAMFFWLL) and two modified hTERT peptides (aa 572-580 YLFFYRKSV and aa 988-997 YLQVNSLQTV) were also tested for generation of responses from JmmunoBody constructs. These epitopes were initially discovered by in silico epitope prediction and peptide immunisation therefore negating the requirement for proteasomal processing. However they are presented upon the surface of host endothelial/tumour cells which suggests they are processed from whole antigen via the constitutive proteasome. None of these epitopes generated responses when engineered into the ImmunoBody construct suggesting that processing via the immunoproteasome may be required for efficient generation of immune responses.<br>
Example 14 - Different immunisation methods are efficient at eliciting immune responses from ImmunoBody vaccine<br>
ImmunoBody vaccine has been shown to be effective at eliciting high frequency and avidity CD8 and CD4 responses when administered via gene gun. ImmunoBody vaccine was subsequently tested for generation of T cell responses using other methods of immunisation.<br>
C57BI/6 mice were immunised with ImmunoBody DNA containing the TRP2 epitope in CDRH2 via either the i.d. or i.m. route. Immunisations were combined with and without electroporation and performed three times at weekly intervals.<br>
Mice immunised with gene gun show high frequency TRP2 peptide specific responses. These are comparable in mice immunised either via i.m. or i.d. route combined with electroporation. Immunisation via i.m. or i.d. route in absence of electroporation generated lower frequency TRP2 peptide specific responses (Figure 57a). All TRP2 peptide specific responses are of high avidity as measured by peptide titration (Figure 57b).<br>
Example 15 - ImmunoBody immunisation induces vitiligo-like depigmentation and protects against tumour challenge<br>
Since mice immunised with ImmunoBody DNA generate immune responses capable of cytotoxic activity against the highly metastatic and poorly immunogenic tumour cell line B16F10, the vaccine was tested for protective efficacy in vivo.<br>
Mice were immunised with IB DNA (DCIB18; Figure 30) via gene gun into shaved skin of the abdomen at five weekly intervals. Part way through the schedule of immunisations, mice were injected i.v with 1x104 B16F10 cells expressing IFNa which forms metastatic tumours in the lung. When the hair<br>
was permitted to grow back after last immunisation, mice immunised with ImmunoBody DNA were observed to have growth of white hair at the site of immunisation (Figure 58a). Seven weeks post tumour cell injection, mice were sacrificed and the number of internal and external lung metastases analysed. ImmunoBody DNA immunised mice exhibited a significant reduction in the number of lung metastases compared to untreated control mice (Figure 58b).<br>
Mice were also immunised with IB DNA (DCIB18) via gene gun at three weekly intervals. Seven days post final immunisation mice were challenged with 2x104 B16F10 cells expressing IFNα subcutaneously. Mice were monitored for tumour growth and survival. Mice were euthanized once tumours reached the maximum limit according to Home Office regulations. ImmunoBody DNA immunised mice exhibited significantly slower subcutaneous tumour growth and prolonged survival (Figure 58c &amp; d).<br>
The TRP2 specific response is CD8 mediated as depletion of the CD8+ cells abrogates the response. CD8 T cells have been identified as a major player in anti-tumour immunity and our results show that ImmunoBody DNA immunisation elicits in vivo anti-tumour immunity in a mouse model. All immunised mice with no signs of disease exhibited vitiligo-like depigmentation of fur at the site of immunisation. Previously vitiligo is often associated with tumour protection in mice and has been highly correlated with successful IL-2 immunotherapy in patients with metastatic melanoma (Overwijk et al, Proceedings of the National Academy of Sciences of the United States of America 1999; 96:2982-7; Lane et al, Cancer Research 2004;64:1509-14; Steitz etal, Cancer Immunol Immunother2006; 55: 246-53; Rosenberg &amp; White, J Immunother Emphasis Tumor Immunol 1996 ;19: 81-4).<br>
Example 16 - ImmunoBody immunisation significantly delays tumour growth.<br>
ImmunoBody immunisation has previously shown to significantly protect against tumour challenge. The vaccine was subsequently tested for efficacy in a therapeutic setting.<br>
C57BI/6 mice were injected s.c. with 2x104 B16F10 tumour cells. Four days post injection mice were immunised with ImmunoBody DNA containing TRP2 epitope in CDRH2 or control ImmunoBody DNA. Repeat immunisations were performed at days 11 and 18 post tumour injection. Tumour growth was monitored at 3-4 day intervals. ImmunoBody immunised mice demonstrate a significant delay in growth of the aggressive B16F10 melanoma compared to control immunised mice (Figure 59a).<br>
A similar study was also performed using the less aggressive B16F10 IFNalpha tumour line. C57BI/6 mice were injected with 2x104 tumour cells s.c. and immunised at day 14 with ImmunoBody DNA or control DNA. Repeat immunisation were performed at days 21 and 28 post tumour injection. ImmunoBody immunised mice exhibited significantly lower tumour growth than control immunised mice at day 47 post tumour injection (Figure 59b). Previous data has suggested that depletion of T regulatory cells enhances generation of immune responses therefore an anti-tumour study was performed. In this study mice were injected with 2x104 B16F10 tumour cells s.c. and immunised at day 4,11 and 18 with ImmunoBody DNA or control DNA.   On day 0 mice were depleted of T regulatory cells via injection of anti-CD25 antibody (PC61). Concurrent with the second immunisation mice were also injected with anti-CTLA-4 antibody as blockade of CTLA-4 has also shown to be beneficial in the inhibition of regulatory T cells. Tumour growth was monitored and although ImmunoBody immunisation significantly delays the tumour growth (p=0.0188) this was further enhanced by treatment with anti-CD25 and anti-CTLA-4 antibodies (p=0.001) (Figure 59c). The treatment<br>
with anti-CD25 antibody did not appear to significantly delay the tumour growth observed in ImmunoBody immunised mice.<br>
Example 17 - Immune responses can be generated from ImmunoBody constructs expressed from different vector backbones.<br>
Responses were analysed when ImmunoBody constructs were expressed from different vector backbones. ImmunoBody construct containing gp100DR7 epitope in CDRH1, TRP2 epitope in CDRH2 and gp100DR4 epitope in CDRH3 with wildtype light chain was engineered into the double expression vectors DCOrig and DCVax (DCIB54, Figures 18 and 29). HLA-DR4 transgenic mice were immunised via gene gun three times at weekly intervals and responses analysed ex vivo by IFNelispot assay.<br>
Similar experiments were performed using an ImmunoBody construct containing gp100DR7 epitope in CDRH1 and CDRL3, TRP2 epitope in CDRH2, gp100DR4 epitope in CDRH3 and CDRL1 (DCIB68, Figure 60). This construct was engineered into both the DCOrig, DCOrig devoid of the SV40 promoter and DCVax vector backbones.<br>
Mice immunised with the ImmunoBody construct in the Orig vector (B1-3) demonstrate similar frequency epitope responses compared to the Immunobody cionstruct in the pVax vector (C1-3) (Figure 61).<br>
In summary, ImmunoBody technology has superior ability to elicit high frequency and avidity CD8 and CD4 immune responses from a non-immunogenic antibody framework that can efficiently prevent tumour growth in vivo. It has the ability to target up to six different antigens simultaneously and has the capability to avert the problem of regulatory T cells that often occurs when whole antigen immunogens are used. This technology presents a novel approach to vaccination and demonstrates the potential for the ImmunoBody<br>
system to be used as a multivalent vaccine for many other cancer types and micro-organism related diseases.<br>
<br>
<br>
<br>
<br>
<br>
<br>
<br>
<br>
Claims<br>
1.	A nucleic acid which comprises a non-specific promoter and at least one sequence that encodes a polypeptide that has at least one heterologous T cell epitope therein but does not have any regulatory T cell epitopes.<br>
2.	A nucleic acid as claimed in claim 1, wherein the polypeptide is one chain of a heterodimer, the heterologous T cell epitope causing disruption of the heterodimer chain such that it cannot bind with the other chain of the heterodimer.<br>
3.	A nucleic acid as claimed in claim 2, which encodes both chains of the heterodimer, wherein one chain includes a heterologous cytotoxic T cell (CTL) epitope and is secreted when expressed, and the other chain includes a heterologous helper epitope and is not secreted when expressed.<br>
4.	A nucleic acid as claimed in claim 1 or claim 2, wherein the heterodimer is an immunoglobulin molecule.<br>
5.	A nucleic acid as claimed in claim 4, wherein the heavy chain of the immunoglobulin molecule includes a heterologous cytotoxic T cell (CTL) epitope and is secreted when expressed, and the light chain of the immunoglobulin molecule includes a heterologous helper epitope and is not secreted when expressed.<br>
6.	A nucleic acid which comprises a non-specific promoter and at least one sequence that encodes a recombinant heavy chain of an immunoglobulin molecule, wherein the heavy chain has at least one heterologous T cell epitope therein such that the heavy chain cannot take its native conformation when the nucleic acid is expressed.<br>
7.	A nucleic acid as claimed in claim 6, further comprising at least one sequence that encodes a light chain of an immunoglobulin molecule.<br>
8.	A nucleic acid as claimed in claim 6, in combination with a nucleic acid comprising at least one sequence that encodes a light chain of an immunoglobulin molecule.<br>
9.	A nucleic acid as claimed in claim 7 or claim 8, wherein the light chain has at least one heterologous T cell epitope therein.<br>
10.	A nucleic acid as claimed in claim 9, wherein the T cell epitope is such that the light chain cannot take its native conformation when the nucleic acid is expressed.<br>
11.	A nucleic acid as claimed in any one of claims 4 to 10, wherein the at least one T cell epitope is in the variable region of the heavy chain and/or the light chain.<br>
12.	A nucleic acid as claimed in claim 11, wherein the at least one T cell epitope is in at least one of the CDRs of the heavy and/or light chain of the antibody.<br>
13.	A nucleic acid as claimed in claim 12, wherein the CDR is CDRL1, CDRH1 and/or CDRH2.<br>
14.	A nucleic acid as claimed in any one of claims 4 to 13, wherein the sequence encoding the at least one T cell epitope is inserted into the sequence encoding the heavy chain or the light chain.<br>
15.	A nucleic acid as claimed in any one of claims 4 to 13, wherein the sequence encoding the at least one T cell epitope is substituted into the sequence encoding the heavy chain or the light chain.<br>
16.	A nucleic acid as claimed in any preceding claim, which encodes an antibody.<br>
17.	A nucleic acid as claimed in claim 16, wherein the antibody is a monoclonal antibody.<br>
18.	A nucleic acid as claimed in any preceding claim, wherein the at least one T cell epitope is a cytotoxic T cell epitope.<br>
19.	A nucleic acid as claimed in any preceding claim, wherein the at least one T cell epitope is a helper T cell epitope.<br>
20.	A nucleic acid as claimed in any one of claims 4 to 19, wherein the heavy chain and/or light chain has at least one cytotoxic T cell epitope and at least one helper T cell epitope.<br>
21.	A nucleic acid which comprises a non-specific promoter and at least one sequence that encodes a recombinant immunoglobulin molecule, wherein the immunoglobulin molecule has at least one heterologous T cell epitope therein such that the immunoglobulin molecule cannot take its native conformation when the nucleic acid is expressed.<br>
22.	A nucleic acid as claimed in claim 21, modified by the features of any one of claims 6 to 20.<br>
23.	A vaccine comprising a nucleic acid as claimed in any preceding claim and an adjuvant.<br>
24.	A pharmaceutical composition comprising a nucleic acid as claimed in any one claims 1 to 22 and a pharmaceutically acceptable carrier, excipient or diluent.<br>
 <br>
25.	A nucleic acid as claimed in any one of claims 1 to 22 for use in medicine.<br>
26.	A nucleic adic as claimed in claim 25 for use in stimulating an immune response against at least one of the T cell epitope(s).<br>
27.	The use of a nucleic acid as claimed in any one of claims 1 to 22. in the manufacture of a medicament for stimulating an immune response against at least one of the T cell epitope(s).<br>
28.	A method for stimulating an immune response against a T cell epitope, comprising administering to a subject in need of such immune response a therapeutically effective amount of a nucleic acid as claimed in any one of claims 1 to 22.<br>
</td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/viewdoc.aspx?id=xcHS84zZJGwe2/nAUCTaMw==&amp;amp;loc=+mN2fYxnTC4l0fUd8W4CAA==" target="_blank" style="word-wrap:break-word;">http://ipindiaonline.gov.in/patentsearch/GrantedSearch/viewdoc.aspx?id=xcHS84zZJGwe2/nAUCTaMw==&amp;amp;loc=+mN2fYxnTC4l0fUd8W4CAA==</a></p>
		<br>
		<div class="pull-left">
			<a href="279329-nucleotide-and-oligonucleotide-prodrugs.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="279331-communication-terminal.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>279330</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>6178/DELNP/2009</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>03/2017</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>20-Jan-2017</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>18-Jan-2017</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>25-Sep-2009</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>SCANCELL LIMITED</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td>DEPARTMENT OF CLINICAL ONCOLOGY, CITY HOSPITAL, HUCKNALL ROAD, NOTTINGHAM, NG5 1PB, U.K.</td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>PUDNEY, VICTORIA ANNE</td>
											<td>DEPARTMENT OF CLINICAL ONCOLOGY, CITY HOSPITAL, HUCKNALL ROAD, NOTTINGHAM, NOTTINGHAMSHIRE NG5 1PB, U.K.</td>
										</tr>
										<tr>
											<td>2</td>
											<td>DURRANT, LINDA GILLIAN</td>
											<td>DEPARTMENT OF CLINICAL ONCOLOGY, CITY HOSPITAL, HUCKNALL ROAD, NOTTINGHAM, NOTTINGHAMSHIRE NG5 1PB, U.K.</td>
										</tr>
										<tr>
											<td>3</td>
											<td>METHERINGHAM, RACHEL LOUISE</td>
											<td>DEPARTMENT OF CLINICAL ONCOLOGY, CITY HOSPITAL, HUCKNALL ROAD, NOTTINGHAM, NOTTINGHAMSHIRE NG5 1PB, U.K.</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>C12N 15/13</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>PCT/EP2008/053761</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td>2008-03-28</td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td>0706070.0</td>
									<td>2007-03-28</td>
								    <td>U.K.</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/279330-nucleic-acid-encoding-recombinant-antibody by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 01:26:47 GMT -->
</html>
