<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/279416-method-for-modifying-plant-architecture-and-enhancing-plant-biomass-and-or-sucrose-yield by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 01:05:31 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 279416:METHOD FOR MODIFYING PLANT ARCHITECTURE AND ENHANCING PLANT BIOMASS AND/OR SUCROSE YIELD</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">METHOD FOR MODIFYING PLANT ARCHITECTURE AND ENHANCING PLANT BIOMASS AND/OR SUCROSE YIELD</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>The present invention relates to methodology and constructs  for modifying plant architecture and enhancing plant biomass and/or sucrose yield.</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td>METHOD FOR MODIFYING PLANT ARCHITECTURE AND ENHANCING PLANT BIOMASS AND/OR SUCROSE YIELD<br>
<br>
RELATED APPLICATION<br>
<br>
This application claims priority of U. S. Provisional Application No. 60/863,252. filed October 27, 2006, the disclosure of which is incorporated by reference in its entirety.<br>
<br>
FIELD OF THE INVENTION<br>
<br>
The present invention relates to the field of molecular biology and the phenotypic alteration of plant characteristics through the introduction of foreign genes into plant cells. preferably into their genomes. More particularly, the present invention pertains to methods anil compositions for producing transgenic plants having modified plant architecture and enhanced biomass and/or sucrose yield with respect to non-transgenic plants grown in similar conditions.<br>
<br>
BACKGROUND<br>
<br>
Sugarcane is one of the most photosynthetic efficient cultivated crops. Sugarcane, average annual yields in Brazil, world's largest producer, range from 80 to 120 tons/ha. Yet, the productivity potential of 300 tons of biomass per hectare has been suggested. Alexander, THE F.NERGY CANE ALTERNATIVE (Elsevier, 1985).<br>
<br>
The utilization of biomass has been the subject of strong interest recently, particularly with respect to the efforts to develop methods to produce alternate fuels that use non-starch, nonfood-related biomass, such as trees, grasses and waste materials, which would expand the available resource base for sugars and would lower cost sources. In Brazil, sugarcane mills have developed technologies for large-scale sucrose fermentation for producing fuel ethanol. In the 2006/2007 crop, approximately 18 billion liters of sugarcane-derived ethanol were produced. In addition to providing a renewable source of fuel, sugarcane-based fuel provides a means for reducing CO2 emissions.<br>
<br>
Over the centuries, sugarcane was cultivated almost solely as a source of sucrose, which accumulates at high concentrations in the stem internodes. The sucrose then is extracted and purified in large mill factories, and is used as a raw material in food industries.<br>
<br>
While sugarcane is photosynthetically efficient, the average productivity of commercial sugarcane plantations around the world is limited by the highly polyploid nature of the sugarcane genome, which renders sugarcane not amenable to most ot the breeding techniques developed for diploid species. Because traditional breeding programs have provided limited success in producing high-yielding plants, cultivated sugarcane varieties are clones derived from interspecific crosses between Saccharum officinarum with its relatives, most often S. spontaneum but also S sinense or S. barberi, whose progenies were backcrossed to 5. officinaram. On the other hand, the emergence of molecular genetics approaches to manipulating plant genomes has offered researchers a means for developing crops with improved properties or trails, through introduction and expression of recombinant nucleic acid molecules in plants.<br>
<br>
SUMMARY OF THE INVENTION<br>
<br>
In one aspect, the invention provides a transgenic plant belonging to a family selected from the group consisting of Poaceae, Cucurbitaceae, Cruciferaceae, Solanaceae, Legumuio.sae, and Apocynaceae plant family, wherein the plant contains an endogenous Ss2-ODDl DNA sequence the expression of which is reduced compared to a wild-type control plant. In further embodiment, the plant belongs to Poacea family. In even further embodiments, the transgenic plant is sugarcane, sorghum, corn and Miscanthus. In another embodiment, the expression of Ss2-ODDl DNA sequence is reduced by antisense suppression, sense co-suppression, RNA interference, or enzymatic RNA.<br>
<br>
In another aspect, the invention provides a method for producing sucrose, comprising: (a) providing a transgenic plant having suppressed Ss2-ODDl protein levels; and (b) obtaining sucrose from said plant. In one embodiment, the plant is sugarcane or sorghum.<br>
<br>
In another aspect, the invention provides a method for producing biomass, comprising: (a) providing a transgenic plant having suppressed Ss2-ODDl protein levels; and (b) obtaining biomass from said plant. In one embodiment, the plant is sugarcane or sorghum.<br>
<br>
In another aspect, the invention provides a method for enhancing sucrose yield in a plant, comprising: suppressing Ss2-ODDl protein levels in said plant. In one embodiment, the plant is sugarcane or sorghum.<br>
<br>
In another aspect, the invention provides a method for enhancing biomass in a plant yield, comprising: suppressing Ss2-ODDl protein levels in said plant. In one embodiment, the plant is sugarcane or sorghum.<br>
<br>
 <br>
In another aspect., the invention provides a nucleic acid construct comprising an Ss2-ODD1 sequence. In further embodiment, the invention provides a transgenic plant or cell comprising the nucleic acid construct comprising an Ss2-ODDl sequence.<br>
<br>
BRIEF DESCRIPTION OF THE DRAWINGS<br>
<br>
FIGURE 1 shows conservation of functional amino acid residues in Ss2-ODDl, its close homologues from Oryza sativa, Sorghum bicolor and Zea mays and plant ferrous iron-dependent dioxygenases. Numbering of residues was based on the isopenicillin N synthase (1PNS) protein from Streptomyces jumonjinensis. The alignment shows conserved residues (underlined) including the iron binding motif His-X-Asp(53-57)X-His (shaded) that is common to non-heme Fe(II) dependent dioxygenases (Kieisberg-Zakarin el al., Antonie Van Leeuwenhoek 75: 33-39, 1999). These conserved residues are presumed to be necessary for iron binding in the active site (Borovok et al., Biochemistry 35: 1981-1987, 1996). Ss, Saccharum species; Sb, Sorghum bicolor; Zm, Zea mays; ORYSA, Oryza sativa; ARATH, Arabidopsis thaliana; CUCMA, Cucurbita maxima; STRJU, Streptomyces jumonjinensis; PETHY, Petunia hybrida; GARPE, garden petunia; 2-ODD1, 2-oxoacid-dependent dioxygenase; GA2, Gibberel/in 2/1 Hydroxylases; GA20, Gibberellin 20-Oxidases; GA3, GibberellinSB-Hydroxylases; GA7, Oibberellin 7-Oxidases; A2, A2 gene; LDOX, leucoanthocyanidin dioxygenase; FLS, flavonol synthase; FL3H, Flavanone 3|3-Hydroxylase; 2A6, 2A6 gene.<br>
<br>
FIGURE 2 shows phylogenetic analysis of higher plant 2-ODDs. Amino acid sequences of higher plant representatives of major 2-ODDs types were aligned with the Clustal algorithm using the BLOSUM matrix. Phylogenetic reconstructions were obtained m MEGA version 3 1 (Kumar et al., Brief Bioinform. 5: 150-163, 2004) by the Neighbor-Joining method (NJ) with robustness of nodes of the phylogenetic trees assessed by bootstrapping (1,000 resamplings). Bootstrap values larger than 50 are shown. GenBank gi or TIGR TC numbers are provided following sequence names. Ss2-ODDl is boxed. Proteins with corresponding characterized gibberellic acid oxidase mutants of Arabidopsis thaliana (GAS, GA4), Oryza sativa (D18, SEMIDWARF1), and Pisum sativum (SLENDER) are indicated. Species and protein names are abbreviated as follows: At, Arabidopsis thaliana; Cm, Cucurbita maxima; Cr, Capsella rubella; Cro, Catharanthus roseus; Hv, Hordeum vulgare; Hn, Hyoscyamus niger; Le, Lycopersicum esculentum; Mm, Marah macrocarpus; Ms, Medicago sativa; Mt, Medicago truncatula; Nt; Nicotiana tabacum; Os, Oryza sativa; Ps, Pisum sativum; Pt, Petunia x hybrida; St, Solanum tuberosum; Sb, Sorghum bicolor; Ss, Saccharum species; Zm, 'lea mays. A2, protein encoded by the A2 locus that affects anthocyanin biosynthesis; ACCO, 1-aminocyclopropane-l-carboxylate oxidase; ANS, anthocyanidin synthase; CFI, chalcone-flavonone isomerase; DV4IL desacetoxyvindoline 4-hydroxylase; E8, 1-aminocyciopropane-l-carboxylate oxidase homolog (protein E8); F3H, flavonone-3-hydroxylase; FLS, flavonol synthase; GAoxi, gibberellic acid oxidase; H6H, hyoscyamine 6-dioxygenase; 1DS2, iron deficiency specific-2; LDOX, leucoanthocyanidin dioxygenase; P4H, prolyl 4-hydroxylase; SRG1, encoded transcript expressed in growth-arrested Arabidopsis thaliana cells.<br>
<br>
FIGURE 3 shows schematic representation of UBJ-1 ::Ss2-ODDlas::NOS and UBI-l:\Bar. :NOS cassettes. Endonucleases {Earl and HindlU) used for releasing cassettes from original cloning vectors are shown.<br>
FIGURE 4 shows height of selected UB1-1::Ss2-ODDlas::NOS events and non-transgenic control cultivar RB835486. Height of two stems was measured for each sample (n=12) after removal of the stalk apical sections. Double asterisks indicate events significantly different from the cultivar RB835486 non-transgenic control (C) as determined by the Dunnett's method at the 99% confidence level. Error bars indicate standard deviation of the mean.<br>
<br>
FIGURE 5 shows internode number of selected UBI-1: Ss2-ODDlas: N'OS events ano non-transgenic control cultivar RB835486. Internode number of two stalks was determined for each sample (n=12) after removal of the apical section. Double asterisks indicate, events significantly different from the cultivar RB835486 non-transgenic control (C) as determined by the Dunnett's method at the 99% confidence level. Error bars indicate standard deviation of the mean.<br>
<br>
FIGURE 6 shows average internode length of selected UBl-J..Ss2-ODDlas::NOS events and non-transgenic control cultivar RB835486. Length of all internodes of two stalks was measured for each sample (n=12) after removal of the apical part. Double asterisks indicate events significantly different from the cultivar RB835486 non-transgenic control (C) as determined by the Dunnett's method at the 99% confidence level. Error bars indicate standard deviation of the mean.<br>
<br>
FIGURE 7 shows average internode diameter of selected UBl-1 ::Ss2-ODDlay. SOS events and non-transgenic control cultivar RB835486. Internode diameter of two stalks was measured for each sample (n=12) after removal of the apical section. Double asterisks indicate events  significantly  different  from   the   cultivar  RB835486  non-transgenic  control   (C)   as determined by the Dunnett's method at the 99% confidence level. Error bars indicate standard deviation of the mean.<br>
<br>
FIGURE 8 shows stalk fresh weight of selected UB1-1 ::Ss2-ODDlas::NOS events and non-transgenic control cultivar RB835486. Fresh weight of two combined stalks was measured for each sample (n=12) after removal of the apical section. Double asterisks indicate events significantly different from the cultivar RB835486 non-transgenic control (C) as determined by the Dunnett's method at the 99% confidence level. Error bars indicate standard deviation of the mean.<br>
<br>
FIGURE 9 shows juice volume of selected UBl-l::Ss2-ODDlas::NOS events and non-transgenic control cultivar RB835486. Juice volume extracted from two combined stalks was measured for sample (n=12) after removal of the apical section. Double asterisks indicate events significantly different from the cultivar RB835486 non-transgenic control (C) as determined by the Dunnett's method at the 99% confidence level. Error bars indicate standard deviation of the mean.<br>
<br>
FIGURE 10 shows POL of selected UBI-I::Ss2-ODDlas::NOS events and non-transgenic control cultivar RB835486. POL of juice extracted from two combined stalks was measured for each sample (n=12) after removal of the apical section. Double asterisks indicate events significantly different from the cultivar RB835486 non-transgenic control (C) as determined by the Dunnett's method at the 99% confidence level. Error bars indicate standard deviation of the mean.<br>
<br>
FIGURE 11 shows sucrose yield of selected UBI-l::Ss2-ODDlas::NOS events and non-transgenic control cultivar RB835486. Sucrose yield was calculated as a product of total juice volume and its sucrose concentration from two combined stalks for each sample (n-12) after removal of the apical section. Double asterisks indicate events significantly different from the cultivar RB835486 non-transgenic control (C) as determined by the Dunnett's method at the 99% confidence level. Error bars indicate standard deviation of the mean.<br>
<br>
FIGURE 12 shows bagasse fresh weight of selected UBI-l::Ss2-ODDlas::NOb events and non-transgenic control cultivar RB835486. Bagasse fresh weight of two combined stalks was measured for each sample (n=12) after removal of the apical section.    Double asterisks FIGURE 13 shows expression of Ss2-ODDl transcripts in wild-type sugarcane organs measured by quantitative reverse transcription-polymerase chain reaction (qRT-PCR). Quantitative reverse transcription-polymerase chain reaction (qRT-PCR) was performed using RNA isolated from different organs of wild-type sugarcane plants (cultivar RBS35486). Abundance of Ss2-ODDl transcripts relative to those of ACTIN (ACT) are expressed as ACT values. Bars indicate mean values from three independent replicates. Error bars indicate standard deviation of the mean.<br>
<br>
FIGURE H shows Ss2-ODDl protein levels in the leaf +4 of selected UBl-i: S&amp;2-ODDlas::NOS events and wild-type sugarcane cultivar RB835486 controls. Extracts were prepared from blades of leaf +4 harvested from cultivar RB83548-6 wild-type plants (C) and events 09, 14 and 21. Leaf protein extracts (50 p.g) along with 0.5 ng of recombinant Ss2-UDD1 (rSs2-ODDl) were separated by SDS-PAGE. ImnrunobloUing was performed with Ss2-ODD'l affinity-purified antibodies (upper panel). Arrowhead indicates Ss2-ODDl protein. Asterisk indicates a non-specific cross-reacting polypeptide in leaf extracts shown as loading control. Molecular weights are shown at the right (MW) expressed in kDa. The gel stained with Coomassie Blue after transfer is shown as a loading control (bottom panel).<br>
<br>
DETAILED DESCRIPTION OF THE INVENTION<br>
<br>
The present invention relates to methodology and constructs for manipulating a dioxygenase enzyme in planta, thereby modifying plant architecture and enhancing plain biomass and/or sucrose yield. Dioxygenases are iron-containing, non-heme enzymes that are involved in the biosynthesis of many compounds, including penicillin, cephalosporin, cephamycin, clavam, carnitine and collagen biosynthesis (Prescott, and Lloyd, Nat. Prod. Rep., 17: 367-383, 2000). Also, some dioxygenases have long been described to oxidize several amino acids residues in various protein targets to facilitate protein folding. Modifications in amino acid oxidation permit regulatory and structural mechanisms (Ozer and Bruick, Nat. Chem. Biol, 3: 144-153, 2007). In plants particularly, various dioxygenases play crucial roles in the biosynthesis of signaling compounds such as abscisic acid, gibberellins, and ethylene and also of secondary metabolites, notably flavonoids and alkaloids. Surveys of reactions catalyzed by dioxygenases are provided by several articles and references therein (Prescott and John, Amur Rev Plant Physiol Plant Mol. Biol., 47: 245-271,1996; Prescott and Lloyd, Nat. Prod. Rep., 17: 367-383, 2000; Turnbull et al, J Biol. Chem., 279: 1206-1216, 2004; Ozer and Bruick, Nat Chem. Biol., 3: 144-153, 2007).<br>
<br>
Plant dioxygenases fall into two classes: lipoxygenases and 2-oxoglutarate-dependenl dioxygenases (2-ODDs) that catalyze hydroxylation, epoxidation, and desaturation reactions. Members of latter frequently catalyze more than one type of reaction in successive steps in a biosynthetic pathway. Enzymes in 2-ODD group show an absolute requirement for 2-oxoglutarate as a co-substrate. All 2-ODDs are soluble enzymes that require Fe2+ and ascorbate for optimal substrate conversion in vitro. Some enzymes have -din absolute requirement for ascorbate, but many do not. Often the role of ascorbate is undefined and is likely to be indirect and unrelated to the reaction mechanism.<br>
<br>
Sequences of 2-ODDs are characterized by conserved residues mainly clustered in the carboxy-terminal half of the protein. Three-dimensional structures have been published for the microbial 2-ODD, isopenicillin N synthase (IPNS; Kreisberg-Zakarin et al, Antonie van Leeuwenhoek, 75: 33-39, 1999). Although IPNS shows relatively low sequence identity to plant 2-ODDs, it shares a number of conserved residues/motifs and structural elements and has been proposed as a model for the structure of plant 2-ODDs. The structure show that the catalytic core of the INPS consist of a double-stranded (3-helix (DSBH) fold containing a HX[DE] dyad (where X is any amino acid) and a conserved carboxy-terminal histidine which logether chelate a single iron atom (Kreisberg-Zakarin et al, Antonie van Leeuwenhoek, 75: 33-39, 1999). The conserved portion of 2-ODD super-family proteins comprises the core DSBH which is arranged in two sheets in a jelly-roll topology, a comparatively rare structure in enzymes more commonly found in viral capsid proteins.<br>
<br>
Alignments to compare primary amino acid sequences of a large group of non-heme Fe(lf)-dependent dioxygenases, all of which activate molecular oxygen and carry out diverse reactions such as aliphatic hydroxylations, desaturations and cyclizations, were analyze (Kreisberg-Zakarin et al, Antonie van Leeuwenhoek, 75: 33-39, 1999). Sequence analysis of some 52 members of this class of enzymes established that five residues alone are conserved throughout - Gly41, His212, Asp214, His268 and Arg277, numbering according to 5. jumonjinensis IPNS (Borovok et ai, Biochemistry, 35: 1981-1987, 1996). Biochemical analysis revealed that mutant enzymes in His212, Asp214 and His268 have no activity in vitro (Tan and Sim, J. Biol. Chem., 271: 889-894, 1996; Loke et al, FEMS Microbiol. Lett., 157: 137-14U, 1997), implying that the two conserved histidines and aspartic acids are essential for IPNS activity.  A more extensive analysis comprising 139 non-redundant non-heme, Fe(II)-dependent dioxygenases revealed thai both the two histidincs and the aspartic acid and the glycine are fully conserved (Kreisberg-Zakarin et al., Antome van Leeuwenhoek, 75: 33-39, 1999). This analysis was used to define a sequence motif that is common to all the non-heme Fe(II) dependent dioxygenases, His-X-Asp(53-57)X-His, which is presumed to be necessary for binding of the iron in the active site (Borovok et ai, Biochemistry, 35: 1981-1987, 1996). No role so far has been ascribed to the Gly41.<br>
<br>
Against this background, the present invention describes a method for modifying plant architecture and enhancing plant biomass and/or sucrose yield through the down-regulation of a dioxygenase-encoding gene.<br>
<br>
All technical terms employed in this description are in common use in the fields of biochemistry, molecular biology, immunology and agriculture; hence, they are easily understood by a person skilled in the art to which the present invention belongs. These technical terms can be found, for instance, in: MOLECULAR CLONING: A LABORATORY MANUAL, 3rd ed., vol. 1-3, ed. Sambrook and Russel, Cold Spring Harbor laboratory Press, Cold Spring Harbor, N.Y., 2001; CURRENT PROTOCOLS IN MOLECULAR BIOLOGY, ed. Ausubel et ai, Greene Publishing Associates and Wiley-Interscience, New York, 1988 (with periodic updates); SHORT PROTOCOLS IN MOLECULAR BIOLOGY: A COMPENDIUM OF METHODS FROM CURRENT PROTOCOLS IN MOLECULAR BIOLOGY, 5th ed., vol. 1-2, ed. Ausubel et ai, John Wiley &amp; Sons, Inc., 2002; GENOME ANALYSIS: A LABORATORY MANUAL, vol. 1-2, ed. Green et al., Cold Spring Harbor Laboratory Press, Cold Spring Harbor, N.Y, 1997; Harlow and Lane (ANTIBODIES: A LABORATORY MANUAL, Cold Spring Harbor Laboratory Press, Cold Spring Harbor, N.Y., 1988. <br>
<br>
Methodology involving plant biology techniques is described herein and is described m detail in treatises such as METHODS IN PLANT MOLECULAR BIOLOGY: A LABORATORY COURSE MANUAL, ed. Maliga et ai, Cold Spring Harbor Laboratory Press, Cold Spring Harbor. N.Y., 1995. Various techniques using PCR are described, e.g., in Innis el ai, PCR PROTOCOLS. A GUIDE TO METHODS AND APPLICATIONS, Academic Press, San Diego, 1990 and in Dieffenbach and Dveksler, PCR PRIMER: A LABORATORY MANUAL, 2"d ed., Cold Spring Harbor Laboratory Press, Cold Spring Harbor, N.Y., 2003. PCR-primer pairs can be derived from known sequences by known techniques such as using computer programs intended for that purpose, e.g., Primer, Version 0.5, 1991, Whitehead Institute for Biomedical Research, Cambridge, MA. Methods for chemical synthesis of nucleic acids are discussed, for example, in Beaucage &amp; Caruthers, Tetra. Letts. 22: 1859-1862 (1981), and Matteucci &amp; Caruthers, J. Am. Chem. Soc. 103: 3185 (1981).<br>
 <br>
Restriction enzyme digestion, phosphorylation, ligation and transformation were performed as described in Sambrook el al., MOLECULAR CLONING: A LABORATORY MANUAL, 2"d ed. (1989), Cold Spring Harbor Laboratory Press, unless otherwise specified. All reagents and materials used for the growth and maintenance of bacterial cells were obtained from Aldrich Chemicals (Milwaukee, WI), DIFCO Laboratories (Detroit, MI), Invitrogen (Gaithersburg, MD), or Sigma Chemical Company (St. Louis, MO), unless otherwise specified.<br>
<br>
The terms "encoding" and "coding" refer to the process by which a polynucleotide, through the mechanisms of transcription and translation, provides information to a cell from which a series of amino acids can be assembled into a specific polypeptide. Because of the degeneracy of the genetic code, certain base changes in DNA sequence do not change the amino acid sequence of a protein.<br>
The term "expression" denotes the production of the polypeptide encoded by a polynucleotide. Alternatively or additionally, "expression" denotes the combination of intracellular processes, including transcription and translation, undergone by a coding DNA molecule such as a structural gene to produce a polypeptide. 'Down-regulation'" and "suppression" are used synonymously to indicate that the expression of a particular gene sequence in a cell or plant has been reduced relative to a control cell or plant.<br>
<br>
The phrase "altered expression" in reference to a polynucleotide indicates that the pattern of expression in, e.g., a transgenic plant or plant tissue, is different from the expression pattern in a wild-type plant of the same species. Thus, the polynucleotide of interest is expressed in a cell or tissue type other than a cell or tissue type in which the sequence is expressed in the wild type plant, or by expression at a time other than at the time the sequence is expressed in the. wild type plant, or by a response to different inducible agents, such as hormones or environmental signals, or at different expression levels, compared with those found in a wild type plant. The resulting expression pattern can be transient or stable, constitutive or inducible. With reference to a polypeptide, "altered expression" further may relate to altered activity levels resulting either from altered protein levels or from interactions of the polypeptides with exogenous or endogenous modulators, or from interactions with factors or as a result of the chemical modification of the polypeptides.<br>
<br>
The terms "exogenous nucleic acid" and "heterologous nucleic acid" are used interchangeably and refer to a nucleic acid, DNA or RNA, which has been introduced into a cell (or the cell's ancestor) through the efforts of humans. Such exogenous nucleic acid may be a copy of a sequence which is naturally found in the cell into which it was introduced, or fragments thereof.<br>
<br>
In contrast, the term "endogenous nucleic acid" refers to a nucleic acid, gene, polynucleotide, DNA, RNA, mRNA, or cDNA molecule that is present in a plant or organism that is to be genetically engineered. An endogenous sequence is "native" to, i.e., indigenous to, the plant or organism that is to be genetically engineered.<br>
<br>
The phrase "homologous sequences" refers to polynucleotide or polypeptide sequences that are similar due to common ancestry and sequence conservation. Homologous sequences may be "orthologous," if they were separated by a speciation event, or "paralogous," if they were separated by a gene duplication event.<br>
<br>
The phrase "functional homolog" refers to a polynucleotide or polypeptide sequences that are similar due to common ancestry and sequence conservation and have identical or similar function at the catalytic, cellular, or organismal levels.<br>
<br>
Dioxygenase Sequences<br>
<br>
Dioxygenase^encoding genes have been identified in several plant families, exemplified by Poaceae members such as sugarcane and sorghum. In accordance with the present invention provides, 2-oxoglutarate-dependent dioxygenase (2-ODD) sequences can be employed, as described above, to modify plant architecture and to enhance biomass and/or sucrose yield.<br>
<br>
In the context of the present invention, "Ss-20DD1" refers to a plant enzyme for which decreased protein levels and/or activity results in plants with modified architecture and enhanced biomass and/or sucrose yield. In the context of the present invention, "Ss-20DD1" refers to a plant enzyme whose decreased protein levels and/or suppression of activity confers modified plant architecture and increased sucrose and/or biomass, as well as possesses dioxygenase activity having the potential to catalyze at least one of the following reactions:<br>
<br>
(a)	activation of molecular oxygen and catalysis of substrate conversions, including hydroxylation, desaturation, cyclization, and epoxidation, with or without 2-oxoglutarate requirement as a co-substrate;<br>
<br>
(b)	oxidation of biomolecules (metabolites and macromolecules) with or without an ascorbate requeriment as cofactor;<br>
<br>
(c)	catalysis of oxygen incorporation from O? in organic (metabolites and macromolecules) substrates;<br>
 <br>
(d)	biosynthesis or degradation of plant signaling compounds such abscisic acid, gibberellins and ethylene; and<br>
<br>
(e)	biosynthesis or degradation of secondary metabolites, notably flavonoids and alkaloids.<br>
<br>
The polynucleotides of the invention encode polypeptides characterized by a high sequence identity to 2-<br>
oxoglutarate-dependent dioxygenase (2-ODD) polypeptides. While exemplary 2-ODD sequences are derived from Poaceae, such as the Saccharum spp. 2-ODD (Ss2-ODDl), functional homologs from other plant families can be used to produce plants with modified plant architecture and at least one of enhanced biomass and increased sucrose yield. As shown in Figure .2, 2-ODD sequences have been identified in several plant families: Poaceae (e.g., sugarcane and sorghum), Cucwbitaceae (e.g., pumpkin and winter squash), Brassicaceae (e.g., Arabidopsis), Solanaceae (e.g.. tobacco and tomato), Leguminosae (e.g., garden pea), and Apocynaceae (e.g., Madagascar periwinkle), inter alia. It is expected that plant 2-ODD genes from other plant families catalyze the same reactions affected by the Ss2-ODDl enzyme (SEQ ID NO: 2) that illustrative SEQ ID NO: 1 encodes; hence, a reduced expression of such genes should lead to the phenotypes described above and illustrated by plants described in the examples, infra.<br>
<br>
Furthermore, common genetic mechanisms control vegetative architecture within the Poaceae family and even across dicot and monocot species. For example, see Doust, Ann. Bot. 100: 941-50, 2007, and publications cited therein. Accordingly, the identification and isolation from different families of functional homologs that control plant architecture and biomass accumulation should permit the modification of these characteristics in a wide range of plants, pursuant to the present invention.<br>
<br>
Additional 2-ODD sequences can be identified and functionally annotated by sequence comparison. <br>
Thus, the skilled person can readily identify a functionally related Ss2-ODDl sequence in a suitable database, such as GenBank, using publicly available sequence-analysis programs and parameters. Sequences initially identified as 2-ODD may be characterized further, thereby to identify sequences that comprise specified sequence strings corresponding to sequence motifs present in families of known dioxygenases. Alternatively, screening cDNA libraries or genomic libraries, employing suitable hybridization probes and conditions, should lead to the identification of functionally related Ss2-ODDl sequences. It is appreciated in the field as well that sequences with reduced levels of identity also can be isolated with the aid of (degenerate) oligonucleotides and PCR-based methodology.<br>
 <br>
Via the techniques described above, therefore, a sequence can be identified and functionally annotated as belonging to the Ss2-ODDl family. By the same token, "Ss2-ODDl DMA sequence" in this description refers to any nucleic acid molecule with a nucleotide sequence capable of hybridizing under stringent or highly stringent conditions with the sequence set forth in SEQ ID NO: 1, and encodes an Ss2-ODDl enzyme as defined above. The category denoted by the term also encompasses sequences which cross-hybridize with SEO ID NO: 1, preferably having at least 40%, preferably at least 60%, especially preferably at least 80% and particularly preferably at least 90%, 91%, 92%, 93%, 94%, 95%, 96%, 97%, 98%, 99%, or 99.9% identity with the Ss2-0DD1 nucleotide sequence shown in SEO ID NO: 1. The nucleotide sequence of the invention may encode a protein that is homologous to the predicted gene product set forth in SEQ ID NO: 2.<br>
<br>
The phrase "stringent conditions" here connotes parameters with which the art is familiar. Single-stranded polynucleotides hybridize when they associate based on a variety of well-characterized physicochemical forces, such as hydrogen bonding, solvent exclusion, and base stacking. The stringency of a hybridization reflects the degree of sequence identity of the. nucleic acids involved, such that the higher the stringency, the more similar are the two polynucleotide strands. Stringency is influenced by a variety of factors, including temperature, salt concentration and composition, organic and non-organic additives, solvents, etc. present in both the hybridization and wash solutions and incubations (and number).<br>
<br>
For hybridization of complementary nucleic acids, which have more than !()() complementary residues, on a filter in a Southern or Northern blot, "stringent" hybridization conditions are exemplified by a temperature that is about 5°C to 20°C lower than the thermal melting point (Tm) for the specific sequence, at a defined ionic strength and pH. The Tm is the temperature, under defined ionic strength and pH, at which 50% of the target sequence hybridizes to a perfectly matched probe. Nucleic acid molecules that hybridize under stringent conditions typically will hybridize to a probe based on either the entire cDNA or selected portions. More preferably, "stringent conditions" here refers to parameters with which the art is familiar, such as hybridization in 3.5 x SSC, 1 x Denhardt's solution, 25 mM sodium phosphate buffer (pH 7.0), 0.5% SDS, and 2mM EDTA for 18 hours at 65°C, followed by four washes of the filter, at 65°C for 20 minutes, in 2 x SSC and 0.1% SDS, and a final wash for up to 20 minutes in 0.5 x SSC and 0.1% SDS or 0.3 x SSC and 0.1% SDS for greater stringency, and O.lx SSC and 0.1% SDS for even greater stringency. Other conditions may be substituted, as long as the degree of stringency is equal to that provided herein, using a 0.5xSSC final wash.    For identification of less closely related homologues washes can be performed at a lower temperature, e.g., 50°C. In general, stringency is increased by raising the wash temperature and/or decreasing the concentration of SSC.<br>
<br>
The invention provides nucleic acid molecules comprising the nucleotide sequence of SEQ ID NO: 1, encoding an amino acid sequence set forth in SEQ ID NO: 2. It is understood that the protein of the invention encompasses amino acid substitutions, additions, and deletions that do not alter the function of the enzyme. For example, substitutions, deletions and insertions introduced into the inventive sequences are also embraced by the invention. Such sequence modifications can be engineered into a sequence by site-directed mutagenesis or by other methods known in the art. Amino acid substitutions are typically of single residues; insertions usually will be on the order of about from 1 to 10 amino acid residues; and deletions will range about from 1 to 30 residues. In preferred embodiments, deletions or insertions are made in adjacent pairs, e.g., a deletion of two residues or insertion of two residues. Substitutions, deletions, insertions or any combination thereof can be combined to arrive at a sequence. The mutations that are made in the polynucleotide encoding an ODD1 polypeptide, such as the Ss2-ODD1, should not place the sequence out of reading frame and should not create complementary regions that could produce secondary mRNA structure. Preferably, the polypeptide encoded by the DNA performs the desired function.<br>
<br>
Dioxygenases sequences that are homologous to the listed sequences will typically share at least about 40% amino acid sequence identity. More closely related dioxygenases amino acid sequences share at least about 50%, about 60%, about 65%, about 70%, about 75% or about 80% or about 90% or about 95%, 96%, 97%, 98% or even 99.9% amino acid sequence identity with the listed sequences. At the nucleotide level, the sequences will typically share at least about 40% nucleotide sequence identity, preferably at least about 50%, about 60%, about 70% or about 80% sequence identity, and more preferably about 85%, about 90%, about 95% or about 97% or more sequence identity to one or more of the listed sequences. The degeneracy of the genetic code enables major variations in the nucleotide sequence of a polynucleotide while maintaining the amino acid sequence of the encoded protein.<br>
Accordingly, the present invention embraces any nucleic acid, gene, polynucleotide. DNA, RNA, mRNA, or cDNA molecule that is isolated from a plant that encodes a 2-ODD polypeptide and whose suppression modifies plant architecture and increases plant biomass and/or sucrose yield.   The DNA or RNA may be double-stranded or single-stranded.   Single stranded DNA may be the coding strand, also known as the sense strand, or it may be the non-coding strand, also called the anti-sense strand.<br>
<br>
It will readily be appreciated by those of skill in the art, that any of a variety of polynucleotide sequences are capable of encoding 2-ODD polypeptides. Due to the degeneracy of the genetic code, many different polynucleotides can encode identical and/or substantially similar polypeptides in addition to those sequences illustrated in the Sequence Listing.<br>
<br>
The term "variant" is a nucleotide or amino acid sequence that deviates from the standard, or given, nucleotide or amino acid sequence of a particular gene or protein. The variant may have "conservative" changes, wherein a substituted amino acid has similar structural or chemical properties, e.g., replacement of leucine with isoleucine. A variant may have "nonconservative" changes, e.g., replacement of a glycine with a tryptophan. Analogous minor variations may also include amino acid deletions or insertions, or both. Guidance in determining which amino acid residues may be substituted, inserted, or deleted may be found using computer programs well known in the art such as Vector NTI Suite (InforMax, MD) software. "Variant" may also refer to a "shuffled gene," as described, for example, in U.S. patents No. 6,506,603, No. 6,132,970, No. 6,165,793 and No. 6,117,679.<br>
<br>
Also contemplated are fragments and domains, referred herein as oligonucleotides, which hybridize under at least stringent or highly stringent conditions to a polynucleotide sequence described above. The oligonucleotides are useful as primers, probes, and the like. An oligonucleotide suitable for use as probes, primes, sense and antisense agents is at least about 15 nucleotides in length, more often at least about 18 nucleotides, often at least about 21 nucleotides, frequently at least about 30 nucleotides, or about 40 nucleotides, or more in length. A nucleic acid probe is useful in hybridization protocols, e.g., to identify additional polypeptide homologues of the invention, including protocols for microarray experiments. Primers can be annealed to a complementary target DNA strand by nucleic acid hybridization to form a hybrid between the primer and the target DNA strand, and then extended along the target DNA strand by a DNA polymerase enzyme. Primer pairs can be used for amplification of a nucleic acid sequence, e.g., by the polymerase chain reaction (PCR) or other nucleic-acid amplification methods.<br>
<br>
The term "fragment" or "domain," with respect to a polypeptide, refers to a subsequence of the polypeptide. In some cases, the fragment or domain is a subsequence of the polypeptide that performs at least one biological function of the intact polypeptide in substantially the same manner, or to a similar extent, as does the intact polypeptide. For example, a polypeptide fragment can comprise a recognizable structural motif or functional domain such as a DNA binding domain that binds to a DNA promoter region, an activation domain or a domain for protein-protein interactions. Fragments can vary in size from as few as 6 amino acids to the full length of the intact polypeptide, but are preferably at least about 30 amino acids in length and more preferably at least about 60 amino acids in length.<br>
<br>
As used herein, "Ss2-0DD1 DNA sequence" is understood to mean that the Ss2-ODDI gene includes the sequence set forth in SEQ ID NO: 1, as well as nucleic acid molecules comprised of variants of SEQ ID NO: 1, with one or more bases deleted, substituted, inserted, or added, which variant codes for a polypeptide with at least 40% amino acid sequence identity. Accordingly, sequences having "base sequences with one or more bases deleted, substituted, inserted, or added" retain physiological activity even when the encoded amino acid sequence has one or more amino acids substituted, deleted, inserted, or added. Additionally, multiple forms of Ss2-ODDl enzyme may exist, which may be due to post-translational modification of the polypeptide or to multiple forms of the Ss2-ODDl gene. Nucleotide sequences that have such modifications and that code for a dioxygenase enzyme are included within the scope of the present invention.<br>
<br>
For example, the poly A tail or 5'- or 3'-end, nontranslation regions may be deleted, and bases may be deleted to the extent that amino acids are deleted. Bases may also be substituted, as long as no frame shift results. Bases also may be "added" to the extent that amino acids are added. It is essential, however, that any such modification does not result in the loss oi dioxygenase enzyme activity. A modified DNA in this context can be obtained by modifying the DNA base sequences of the invention so that amino acids at specific sites are substituted, deleted, inserted, or added by site-specific mutagenesis, for example, Zoller &amp; Smith, Nucleic Acid Res. 10: 6487-500, 1982.<br>
<br>
Unless otherwise indicated, all nucleotide sequences determined by sequencing a DNA molecule herein were determined using an automated DNA sequencer, such as the Model 373 from Applied Biosystems, Inc. Therefore, as is known in the art for any DNA sequence determined by this automated approach, any nucleotide sequence determined herein may contain some errors. Nucleotide sequences determined by automation are typically at least about 95% identical, more typically at least about 96% to at least about 99.9% identical to the actual nucleotide sequence of the sequenced DNA molecule. The actual sequence can be more precisely determined by other approaches including manual DNA sequencing methods well<br>
known in the art. As is also known in the art, a single insertion or deletion in a determined nucleotide sequence compared to the actual sequence will cause a frame shift in translation of the nucleotide sequence such that the predicted amino acid sequence encoded by a determined nucleotide sequence may be completely different from the amino acid sequence actually encoded by the sequenced DNA molecule, beginning at the point of such an insertion or deletion.<br>
<br>
Nucleic acid constructs<br>
The present invention includes recombinant constructs comprising one or more of the nucleic acid sequences herein. The constructs typically comprise a vector, such as a plasmid, a cosmid, a phage, a virus (e.g., a plant virus), a bacterial artificial chromosome (BAC), a yeast artificial chromosome (YAC), or the like, into which a nucleic acid sequence has been inserted, in a forward or reverse orientation. In a preferred embodiment, the construct further comprises regulatory sequences, including, for example, a promoter operably linked to the sequence. Large numbers of suitable vectors and promoters are known and are commercially available.<br>
<br>
Recombinant nucleic acid constructs may be made using standard techniques. For example, a nucleotide sequence for transcription may be obtained by treating a vector containing said sequence with restriction enzymes to cut out the appropriate segment. The nucleotide sequence for transcription may also be generated by annealing and ligating synthetic oligonucleotides or by using synthetic oligonucleotides in a polymerase chain reaction (PCR) to give suitable restriction sites at each end. The nucleotide sequence then is cloned into a vector containing suitable regulatory elements, such as upstream promoter and downstream terminator sequences.<br>
<br>
Typically, plant transformation vectors include one or more cloned plant coding sequence (genomic or cDNA) under the transcriptional control of 5' and 3' regulatory sequences and a selectable marker. Such plant transformation vectors typically also contain a promoter (e.g., a regulatory region controlling inducible or constitutive, environmentally-or developmentally-regulated, or cell- or tissue-specific expression), a transcription initiation start site, an RNA processing signal (such as intron splice sites), a transcription termination site, and/or a polyadenylation signal.<br>
<br>
Suitable constitutive plant promoters which can be useful for expressing the ODD) sequences of the invention include but are not limited to: the cauliflower mosaic virus (CaMV) 35S promoter and the maize polyubiquitin promoter, which confer constitutive, high-level expression in most plant tissues (see, e.g., U.S. patent No. 5,510,474; Odell et al., Nature 313: 810-812, 1985); the nopaline synthase promoter (An et al, Plant Physiol. 88: 547-552, 1988);<br>
 <br>
and the octopine synthase promoter (Fromm et ai, Plant Cell 1: 977-984, 1989); as well as tissue-specific, tissue-preferred, cell type-specific, and inducible promoters. For example, in sugarcane, sucrose produced in the leaves accumulates in the "internode," which refers to the portion of the shoot axis between two nodes, it may be advantageous to use an internode-specific promoter.<br>
<br>
The vector may also contain termination sequences, which are positioned downstream of the nucleic acid molecules of the invention, such that transcription of mRNA is terminated, and polyA sequences added. Exemplary terminators are the cauliflower mosaic virus (CaMV) 35S terminator and the nopaline synthase gene (NOS) terminator.<br>
<br>
Expression vectors may also contain a selection marker by which transformed cells can be identified in culture. The marker may be associated with the heterologous nucleic acid molecule, i.e., the gene operably linked to a promoter. As used herein, the term "marker"' refers to a gene encoding a trait or a phenotype that permits the selection of, or the screening for, a plant or cell containing the marker. In plants, for example, the marker gene will encode antibiotic or herbicide resistance. This allows for selection of transformed cells from among cells that are not transformed or transfected.<br>
<br>
Examples of suitable selectable markers include adenosine deaminase, dihydrofolale reductase, hygromycin-B-phosphotransferase, thymidine kinase, xanthine-guanine phospho-ribosyltransferase, glyphosate and glufosinate resistance, and amino-glycoside 3'-0-phosphotranserase (kanamycin, neomycin and G418 resistance). These markers may include resistance to G418, hygromycin, bleomycin, kanamycin, and gentamicin. The construct also may contain the selectable marker gene Bar, which confers resistance to herbicidal phosphinothricin analogs like ammonium gluphosinate. Thompson et a!., EMBO ./. 6: 2519-23 (1987). Other suitable selection markers are known as well.<br>
<br>
Visible markers such as green florescent protein (GFP) may be used. Methods for identifying or selecting transformed plants based on the control of cell division have also been described. See John and Van Mellaert, WO 2000/052168, and Fabijansk et ai. WO 2001/059086.<br>
<br>
Replication sequences, of bacterial or viral origin, may also be included to allow the vector to be cloned in a bacterial or phage host. Preferably, a broad host range prokaryotic origin of replication is used.   A selectable marker for bacteria may be included to allow selection of bacterial cells bearing the desired construct. Suitable prokaryotic selectable markers also include resistance to antibiotics such as kanamycin or tetracycline.<br>
<br>
Other nucleic acid sequences encoding additional functions may also be present in the vector, as is known in the art. For instance, when Agrobacterium is the host, T-DNA sequences may be included to facilitate the subsequent transfer to and incorporation into plant chromosomes.<br>
<br>
Methods for altering the expression of polynucleotide and polypeptide sequences<br>
<br>
In an aspect of the invention, plant biomass and/or sucrose yield are enhanced by down-regulating the expression of the polynucleotide sequences of the present invention. Various methods for post-transcriptional gene silencing (PTGS) are well known in the art and may be used in the present invention.<br>
<br>
For example, a reduction or elimination of expression (i.e., a "knock-out") of the sequences of the present invention in a transgenic plant, e.g., to modify a plant trait, can be obtained by introducing an antisense construct corresponding to the polypeptide of interest as a cDNA. For antisense suppression, the polynucleotide sequence or homologue cDNA is arranged in reverse orientation (with respect to the coding sequence) relative to the promoter sequence in the expression vector. The introduced sequence need not be the full-length cDNA or gene, and need not be identical to the cDNA or gene found in the plant type to be transformed. Typically, the antisense sequence may be complementary to any contiguous sequence of the natural messenger RNA, that is, it may be complementary to the endogenous mRNA sequence proximal to the 5'-terminus or capping site, downstream from the capping site, between the capping site and the initiation codon and may cover all or only a portion of the non-coding region, mav bridge the non-coding and coding region, be complementary to all or part of the coding region, complementary to the 3'-terminus of the coding region, or complementary lo the 3'-unstranslated region of the mRNA.<br>
<br>
The particular antisense sequence and the length of the antisense sequence will vary, depending, for example, upon the degree of inhibition desired and the stability of the antisense sequence. Thus, where the introduced sequence is of shorter length, a higher degree of identity to the endogenous Ss2-ODDl sequence will be needed for effective antisense suppression. While antisense sequences of various lengths can be utilized, preferably the introduced antisense sequence in the vector will be from at least 13 to about 15 nucleotides in length, at least 16 to about 21 nucleotides, at least about 20 nucleotides, at least about 30 nucleotides, at least about 50 nucleotides, at least about 75 nucleotides, at least about 100 nucleotides, at least about 125 nucleotides, at least about 150 nucleotides, at least about 200 nucleotides, or even the entire length of the sequence to be down-regulated. In addition, the sequences may be extended or shortened on the 3' or 5' ends thereof. Generally available techniques and the information provided in this specification can guide the selection of appropriate Ss2-ODDl-encoding antisense sequences. With reference to SEQ ID NO: 1 herein, an oligonucleotide of the invention may be a continuous fragment of- an Ss2-0DD1 cDNA sequence in antisense orientation, of any length that is sufficient to achieve the desired effects when transformed into a recipient plant cell.<br>
<br>
The present invention contemplates sense co-suppression of an Ss2-ODDl-encoding sequence. Sense polynucleotides employed in carrying out the present invention are of a length sufficient to suppress, when expressed m a plant cell, the native expression of the plant Ss2-ODD! protein in that plant cell. Such sense polynucleotides may be essentially an entire genomic or complementary nucleic acid encoding the Ss2-ODDl enzyme, or a fragment thereof, with such fragments typically being at least 15 nucleotides in length. Techniques are generally available for ascertaining the length of sense DNA that results in suppression of the expression of a native gene in a cell.<br>
<br>
In an alternate embodiment of the present invention, plant cells are transformed with a nucleic acid construct containing a polynucleotide segment encoding an enzymatic RNA molecule (a "ribozyme"), which enzymatic RNA molecule is directed against (i.e., cleaves) the mRNA transcript of DNA encoding Ss2-ODDl, as described herein. Ribozymes contain substrate binding domains that bind to accessible regions of the target mRNA, and domains thai catalyze the cleavage of RNA, preventing translation and protein production. The binding domains may comprise antisense sequences complementary to the target mRNA sequence; the catalytic motif may be a hammerhead motif or other motifs, such as the hairpin motif.<br>
<br>
Ribozyme cleavage sites within an RNA target may initially be identified by scanning the target molecule for ribozyme cleavage sites (e.g., GUA, GUU or GUC sequences). Once identified, short RNA sequences of 15, 20, 30, or more ribonucleotides corresponding to the region of the target gene containing the cleavage site may be evaluated for predicted structural features.<br>
<br>
The suitability of candidate targets also may be evaluated by testing their accessibility to hybridization with complimentary oligonucleotides, using ribonuclease protection assays as are known in the art. DNA encoding enzymatic RNA molecules may be produced in accordance with known techniques. For example, see U.S. patents No. 4,987,071, No. 5,559,021, No. 5,589,367, No. 5,583,032, No. 5,580,967, No. 5,595,877, No. 5,591,601, and No. 5,622,854.<br>
<br>
Production of such an enzymatic RNA molecule in a plant cell and disruption of Ss2-ODD1 protein production reduces protein activity in plant cells, in essentially the same manner as production of an antisense RNA molecule; that is, by disrupting translation of mRNA in the cell which produces the enzyme. The term "ribozyme" describes an RNA-containing nucleic acid that functions as an enzyme, such as an endoribonuclease, and may be used interchangeably with "enzymatic RNA molecule."<br>
The present invention further includes nucleic acids encoding ribozymes, nucleic acids that encode ribozymes and that have been inserted into an expression vector, host cells containing such vectors, and methodology employing ribozymes to decrease Ss2-ODDl activity in plants.<br>
<br>
In another embodiment, the present invention provides double-stranded nucleic acid molecules of that mediate RNA interference (RNAi) gene silencing. In an embodiment, the siNA molecules of the invention consist of duplex nucleic acid molecules containing about 15 to about 30 base pairs between oligonucleotides comprising about 15 to about 30 (e.g., about 15. 16, 17, 18, 19, 20, 21, 22, 23, 24, 25, 26, 27, 28, 29, or 30) nucleotides. In another embodiment. siNA molecules of the invention comprise duplex nucleic acid molecules with overhanging ends of about 1 to about 32 (e.g., about 1, 2, or 3) nucleotides, for example, about 21-nucleotide duplexes with about 19 base pairs and 3'-terminal mononucleotide, dinucleotide, or trinucleotide overhangs. In yet another embodiment, siNA molecules of the invention comprise duplex nucleic acid molecules with blunt ends, where both ends are blunt, or alternatively, where one of the ends is blunt.<br>
<br>
A siNA molecule of the present invention may comprise modified nucleotides while maintaining the ability to mediate RNAi. The modified nucleotides can be used to improve in vitro or in vivo characteristics such as stability, activity, and/or bioavailability. For example, a siNA molecule of the invention can comprise modified nucleotides as a percentage of the total number of nucleotides present in the siNA molecule. As such, a siNA molecule of the invention can generally comprise about 5% to about 100% modified nucleotides (e.g., about 5%, 10%, 15%, 20%, 25%, 30%, 35%, 40%, 45%, 50%, 55%, 60%, 65%, 70%, 75%, 80%, 85%, 90%, 95% or 100% modified nucleotides). The actual percentage of modified nucleotides present in a given siNA molecule will depend on the total number of nucleotides present in the siNA. If the siNA molecule is single stranded, the percent modification can be based upon the total number of nucleotides present in the single stranded siNA molecules. Likewise, if the siNA molecule is double stranded, the percent modification can be based upon the total number of nucleotides present in the sense strand, antisense strand, or both the sense and antisense strands.<br>
<br>
Plants for genetic engineering<br>
The present invention comprehends the genetic manipulation of plants to modify plant architecture and enhance biomass and/or sucrose yield.<br>
<br>
The term 'plant" denotes any cellulose-containing plant material that can be genetically manipulated, including but not limited to differentiated or undifferentiated plant cells, protoplasts, whole plants, plant tissues, or plant organs, or any component of a plant such as a leaf, stem, root, bud, tuber, fruit, rhizome, or the like. The term "propagule" includes a structure with the capacity to give rise to a new plant, e.g., a seed, a spore, or a part of the vegetative body capable of independent growth if detached from the parent.<br>
<br>
"Transgenic plant" refers to a plant or progeny thereof derived from a genetically engineered plant cell or protoplast, wherein the plant DNA contains an introduced exogenous DNA molecule. Additionally, the "transgenic plant" category includes a transformanl in its lineage, e.g., by way of standard introgression or another breeding procedure, such as conventional breeding. In contrast, a plant that is not genetically manipulated is a control plant and is referred to as a "non-transgenic" plant. Non-transgenic plants can be regenerated from cultured cells or tissues without prior modification by the introduction of a construct comprising polynucleotide sequence. Additionally, "wild-type plant" refers to a non-transgenic plant whose genome is neither modified by the introduction of a construct comprising the polynucleotide sequences or fragment thereof of the present invention nor were regenerated from cultured cells or tissues.<br>
<br>
Plants that can be engineered in accordance with the invention include but are not limited to any higher plants, including gymnosperms, monocotyledonous and dicotyledonous plants. The plants of the present invention could include crops, including but not limited to, soybean, wheat, corn, potato, cotton, rice, oilseed rape (including canola), beans, sunflower, alfalfa, sugarcane, turf, barley, rye, millet, sorghum, beet, sugarbeet, cassava, yam, and sweet potato. The plants also may be woody species, such pine, poplar, aspen, willow, and eucalyptus. Also, the plants may be grasses, including but not being limited to Saccharum, Erianthus, Miscanlhus.<br>
 <br>
Narenga, Sclerostachya, sorghum, maize, teosinte, Tripsacum, . millets, teff, switchgrass, napiergrass, rice, wild rice, oat, barley, rye, Brachypodium, ryegrass, fescue, turf grass, or bamboo species. More specifically, plants that can be engineered in accordance with the invention include but are not limited to woody trees, sugarcane and sorghum.<br>
<br>
"Sugarcane plant" is understood as meaning a plant of the genus Saccharum, preferably the species Saccharum officinarum, and more preferably the interspecific hybrid produced by-crossing Saccharum officinarum with Saccharum spontaneum. "Sorghum"' refers to any plant that is a member of the genus Sorghum, and includes "sweet sorghum" varieties having high sugar content.<br>
<br>
Methods for Genetic Engineering<br>
The polynucleotides of the invention may be used to produce transgenic plants with modified architecture and enhanced plant biomass and/or sucrose yield.<br>
<br>
Transgenic plants (including plant cells, plant explants, or plant tissues) incorporating the polynucleotides and/or expressing the polypeptides of the invention can be produced by a variety of well established techniques as described infra and the selection of the most appropriate transformation technique may be determined by the practitioner. Following construction of a vector, most typically an expression cassette, including a polynucleotide, e.g., a Ss2-ODDl sequence, standard techniques known in the art can be used to introduce and stably integrate the polynucleotide into a plant, a plant cell, a plant explain or a plant tissue of interest. Optionally, the plant cell, explant, or tissue can be regenerated to produce a transgenic plant.<br>
<br>
Both monocotyledonous and dicotyledonous angiosperm or gymnosperm plant cells may be transformed. For example, see Klein and Fitzpatrick-McEUigott, Current Opinion in Biotechnology 4: 583-590, 1993; Bechtold et al, C. R. Acad. Sci. Paris 316: 1194-1199, 1993; Koncz and Schell, Mol. Gen. Genet. 204: 383-396, 1986; Paszkowski et al., EMBO J 3: 2717. 2722, 1984; Sagi et al, Plant Cell Rep. 13: 262-266, 1994. Agrobacterium species such as A tumejaciens and A. rhizogenes can be used, for example, in accordance with Nagel et al, FEMS Microbiol Lett 61: 325-328, 1990. Additionally, plants may be transformed by Rhizobium. Sinorhizobium or Mesorhizobium transformation. Broothaerts et al, Nature 433: 629-633. 2005.<br>
<br>
Additional methods for genetically engineering a plant or cell include but are not limited to electroporation, particle gun bombardment (Klein et al., Nature 327: 70-73, 1987), calcium phosphate precipitation, and polyethylene glycol fusion, transfer into germinating pollen grains, direct transformation (Lorz et al, Mol. Gen. Genet. 199: 179-82, 1985), and other methods known to the art.<br>
<br>
Suitable protocols are available for Poaceae (corn, sugarcane, sorghum, etc.), Leguminosae (alfalfa, soybean, clover, etc.), Umbelliferae (carrot, celery, parsnip), Cruciferae (cabbage, radish, rapeseed, broccoli, etc.), Curcurbitaceae (melons and cucumber), Gramineae (wheat, corn, rice, barley, millet, etc.), Solanaceae (potato, tomato, tobacco, peppers, etc.), and various other crops. See protocols described in Ammirato et al., Handbook of Plant Cell Culture - Crop Species. Macmillan Publ. Co., N.Y., 1984; Shimamoto et al., Nature 338: 274-276, 1989; Fromm et al, Bio/Technology 8: 833-39, 1990; and Vasil et al, Bio/Technology 8: 429-34, 1990. Protocols for sugarcane transformation are of particular interest for the present invention. Sugarcane can be transformed by means of the particle gun bombardment, see Bower &amp; Birch, Plant Journal 2: 409-416, 1992; Franks &amp; Birch, Aust. J. Plant Physiol. 18: 471-80, J 991; Gallo-Meagher &amp; Irvine, Crop Science 36; 1367-1374, 1996; Bower et al., Molecular Breeding 2: 239-249, 1996; Snyman et al, S. Afr. J. Bot. 62: 151-154, 1996, or by Agrobacterium-medinttd gene transfer. See, e.g., Arencibia et al., Transgenic Res. 7: 213-22, 1998.<br>
<br>
Methodology for regenerating a transgenic plant from a transformed cell or culture vary according to the plant species but are based on known methodology. For example, methods for regenerating transgenic sugarcane plants are well-known. See, for instance, Weng et al.. Pest Manag. Sci. 62: 178-87, 2006; Santosa et al., Molecular Biotechnology 28: 113-19, 2004; Bower and Birch, Plant Journal 2: 409-416, 1992; and Falco et al., Plant Cell Reports 19: 1188-194, 2000.<br>
<br>
If a selection marker, such as kanamycin resistance, is employed, it makes it easier to determine which cells have been successfully transformed. Marker genes may be included within pairs of recombination sites recognized by specific recombinases such as ere or flp to facilitate removal of the marker after selection. See U. S. published application No.2004/0143874.<br>
<br>
Transformed plants are selected that have decreased expression of an endogenous 2-ODDl gene. For example, an inventive transgenic sugarcane plant or cell can be distinguished from wild-type sugarcane by the fact that the transgenic plant or cell comprises at least one copy of the nucleic acid molecule set for in SEQ ID NO: 1 stably integrated into their genome in addition to copies of such a molecule which occur naturally in the sugarcane wild-type plants/plant cells. In this case, the transgenic sugarcane plant/cells,  for example, can be distinguished from sugarcane wild-type plants/cells because the additional copy or copies is located at locations in the genome where it does not occur in wild-type plant/cell.<br>
<br>
In the context of the present invention, the transgenic plants produced by the methods described supra can be used as a source of a transgene in a conventional breeding program. In general, pollen from a transgenic plant is used to pollinate a non-transgenic plant. The seeds of the mother plant can be used to produce a new transgenic plant different from the original transgenic plant produced by the method described supra.<br>
<br>
Methods for quantifying enhanced biomass and sucrose yield<br>
Transgenic plants and cells of the invention are characterized by an altered plant architecture and enhanced biomass and/or enhanced sucrose yield. This is achieved by decreasing or suppressing 2-ODD expression.<br>
<br>
The term "trait" refers to a physiological, morphological, biochemical, or physical characteristic of a plant or particular plant material or cell. In some instances, this characteristic is visible to the human eye, such as seed or plant size, or can be measured by available biochemical techniques, such as the protein or sucrose content of plant parts or by the observation of the expression level of genes, e.g., by employing Northern analysis, RT-PCR, microarray gene expression assays or reporter gene expression systems, or by agricultural observations such as yield or modified architecture.<br>
<br>
In this description, the phrase "enhanced biomass" connotes an increase in biomass in one or more parts of a plant, particularly in aboveground (harvestable) parts, increased root biomass or increased biomass of any other harvestable part, relative to the biomass of the corresponding parts of corresponding non-transgenic plants. "Fiber yield" is understood as meaning the total amount of solid, water insoluble material produced by a plant. For the purposes of this invention, fiber is considered a component of biomass and refers to the bagasse remaining after sugarcane stalks are crushed to extract their juice. "Enhanced fiber yield" refers to an increase in total fiber yield in transgenic plants relative to non-transgenic plants.<br>
<br>
"Sucrose yield" is understood as meaning the total amount of sucrose produced by a plant. Sucrose yield is quantified by calculating the total apparent sucrose, which is defined as the product of juice volume that is obtained by crushing the stalk, multiplied by the sucrose concentration in that juice. More preferably, in the context of the present invention, sucrose yield means the total amount of sucrose produced per hectare of a transgenic plantation.   The phrase "enhanced sucrose yield" refers to an increase in sucrose content in a transgenic plant in comparison with a non-transgenic one.<br>
<br>
"Modified architecture" as defined here includes any change in the appearance or number of any one or more of the leaves, shoots, stems, tillers, inflorescence (for monocotyledonous and dicotyledonous plants), panicles, pollen, ovule, seed, embryo, endosperm, seed coat and aleurone. Additionally or alternatively, "modified architecture" is manifested by at least one of: an increase in plant height and/or weight, an increase in the number of stalk internodes, increase of the stalk internode diameter and/or increase of the stalk internode Length, and more homogeneous internode diameter from the bottom to the top of the transgenic plants.<br>
<br>
Quantifying plant biomass<br>
Enhanced biomass can be determined by measuring more readily discernible characteristics, including but not limited to plant weight and size, stem weight and size, leaf weight and size, root weight and size, and seed number and weight.<br>
<br>
A plant having "enhanced biomass" refers to an increase by 2-100%, preferably 5-90%, and more preferably 10-80% of dry and/or fresh weight in a transgenic plant in comparison with a non-transgenic plant.<br>
<br>
For the purposes of the present invention, fiber is included in the biomess. In the case of sugarcane, the fiber content usually ranges from 8 to 14% of the fresh weight. As used herein, '"fiber" refers to the bagasse or biomass remaining after sugarcane stalks are crushed to extract their juice. Fiber yield is therefore quantified by measuring the bagasse fresh weight with a high capacity precision balance. A sugarcane mill produces nearly 30% of bagasse out of its total crushing. Many research efforts have attempted to use bagasse as a renewable feedstock foi power generation and for producing bio-based materials. Additionally, bagasse may be used as a primary fuel source because; when burned in quantity, it produces sufficient heat energy.<br>
<br>
A plant having "enhanced fiber yield" refers to an increase in total fiber yield by 2-70%. preferably by 5-50% and more preferably by 10-40% in comparison with a control plant of the same species or variety.<br>
<br>
Quantifying sucrose yield<br>
"Sucrose yield" is understood as meaning the total amount of sucrose produced by a plant. Sucrose yield is quantified by calculating the total apparent sucrose, which is defined as the product of juice volume that is obtained by crushing the stalk, multiplied by the sucrose concentration in that juice. More preferably, in the context of the present invention, sucrose yield means the total amount of sucrose produced per hectare of a transgenic sugarcane plantation.<br>
<br>
A plant having "enhanced sucrose yield" refers to an increase in total sucrose yield by 2-100%, preferably by 5-90%, and more preferably by 10-80% in comparison with a control plant of the same species or variety.<br>
<br>
Evaluating plant architecture modification<br>
In describing a plant of the invention, a plant having "altered plant architecture" refers to an alteration in, for example, plant height, weight, and diameter, tiller number, internode number, length and homogeneity, leaf number, length, width and angle distribution, root length. length and weight.<br>
<br>
The present invention is further described by reference to the following examples, which are illustrative <br>
only and not limiting on the invention.<br>
<br>
EXAMPLE 1. Identification and characterization of Saccharum spp. 2-oxog)urarate-dependent dioxygenase (Ss2-ODDl).<br>
<br>
Searches with sequences of 2-ODD homologs from rice were performed in the sugarcane expressed sequence tag (EST) data available at GenBank with the BLAST algorithm (Altschul ci al., J. Mol. Biol. 215: 403-410, 19900. Searches with a particular rice 2-ODD homolog (gl 27311281) yielded the identification of two clusters encoding a Saccharum spp putative 2-ODD homologue. One of these clusters is nearly identical to a 2-ODD cDNA clone incomplete at its 3'-end found at GenBank (gi 35984354). These clusters were found to correspond to a single cDNA clone, named hereafter Ss2-ODDl (SEQ ID NO: 1). Ss2-ODDl contains an open reading frame (positions 50 to 982) encoding the probable full-length Ss2-ODDl polypeptide (SEQ ID NO: 2), as determined by sequence alignment with its most similar plant homologues. Ss2 ODD1 encodes a polypeptide of 310 amino acid residues and calculated molecular mass of 34,452. Searches in the National Center for Biotechnology Information Conserved Domain Database collection revealed a 2-oxoglutarate-dependent dioxygenase domain (pfam03171) present in the carboxy-terminal half (residues 170 through 271) of the Ss2-ODDl polypeptide. Figure 1 illustrates the presence of conserved residues in this domain, including the iron binding motif His-X-Asp(53-57)X-His that is common to non-heme Fe(II) dependent dioxygenases (Kreisberg-Zakarin et ai, Antonie Van Leeuwenhoek 75: 33-39, 1999). These conserved residues are presumed to be necessary for iron binding in the active site (Borovok et al., Biochemistry 3:i: 1981-1987, 1996).<br>
<br>
EXAMPLE 2. Phylogenetic analysis<br>
BLAST searches against the National Center for Biotechnology Information GenBank protein databases with the Ss2-ODDl sequence yielded some functionally characterized 2-ODD proteins and several 2-ODD-like proteins of unknown function from various higher plant species. Full-length amino acid sequences of members representative of major 2-ODD types were aligned with the Clustal algorithm (Sugawara- et ai, Nucleic Acids Res. 31: 3497-3500, 2003) using the BLOSUM matrix. Phylogerietic analysis was performed by the Neighbor-Joining (NJ) method using MEGA version 3.1 (Kumar et al., Brief Bio inform. 5: 150-163, 2004) with robustness of nodes of the phylogenetic trees assessed by bootstrapping (1,000 resamplings). In the phylogenetic tree thus generated (Figure 2), major functionally distinct 2-ODD groups arc recognizable. Noticeable are a group represented mostly by 1-aminocyclopropane-l-carboxylatc oxidases, and other group comprising mostly gibberellic acid-2, -3, and -20 oxidases. In the latter are proteins encoded by genes that affect gibberellin metabolism in Arabidopsis thaliana, Oryza sativa and Pisum sativum. Ss2-ODDl belongs to a distinct group of functionally non-characterized 2-ODDs, along with several homologs from Arabidopsis thaliana, Catharanthns roseus, Medicago truncatula, Oryza sativa, Sorghum bicolor, and Zea mays. The identification of Ss2-ODDl-related sequences across this range of species suggests broad conservation of this group in the monocotyledoneous and dicotyledoneous clades.<br>
<br>
EXAMPLE 3. Vector construction<br>
A 619 bp fragment, which spans the nucleotides 58 to 676 of the Ss2-ODDl cDNA (SEQ ID NO: 1), was obtained by RT-PCR using total RNA isolated from immature leaves of the Saccharum hybrid cultivar SP80 1842. Oligonucleotides Ss2-ODDl&amp;s Fwd (SEQ ID NO: 3. CGCGGATCCGAACCTGCACCTCCCCGT) and Ss2-ODDlas Rev (SEQ ID NO: 4, CGGACTAGTCAGGCCAGGAGTGCCATC) were designed based on the sequence of an Ss2-ODD1 EST (GenBank gi 35984354). BamHI and Spel sites (underlined) were designed into primers Ss2-ODDlas Fwd and Ss2-ODDlas Rev, respectively. The amplified fragment was cloned into pGEM-T Easy (Promega), verified by sequencing, and excised by digestion with Spel and BamHI. Digestion products were separated by agarose-gel electrophoresis, purified, and cloned at the Avrll and BamHI sites of pUBI (Christensen and Quail, Transgenic Res. 5: 213-218, 1996), placing the Ss2-ODDl fragment, in antisense orientation, between the polyubiquitin gene UBI-1 promoter from maize and the nopaline synthase gene NOS transcriptional terminator from Agrobacterium tumefaciens. The resulting construct was verified by sequencing. This construct was digested with HindlH to release the UBI-1 ::Ss2-ODDlas::NOS cassete. (Figure 3). The cassete UBI-l::Bar::NOS (Christensen and Quail, Transgenic Res. 5: 213-218, 1996) cloned in pBLUESCRIPT SK- was released by digestion with Earl (Figure 3). Digestion products were separated by agarose-gel electrophoresis, and the fragments corresponding to UBI-1 ::Ss2-ODDlas::NOS and UBI-1 ::Bar::NOS cassettes were purified and used for embryogenic sugarcane calli transformation.<br>
<br>
EXAMPLE 4. Sugarcane transformation<br>
Embryogenic calli cultures were established from apical meristems and primordial leaves of Saccharum hybrid cultivar RB835486. Eight-week old calli were transformed by particle bombardment as described previously (Klein et al, Plant Physiol. 91: 440-444, 1989) with equimolar concentrations of UBI-1::Bar::NOS and UBI-1 ::Ss2-ODDlas::NOS expression cassettes (10 u.g DNA/3 mg particle). After bombardment, calli were transferred to MS medium (Murashige and Skoog, Physiol Plant 15: 473-479, 1962) containing 1 mg/L phosphinothricin (PPT) and 1 mg/L benzylaminopurine (BAP) to inhibit development of non transgenic tissue and promote shoot regeneration. Two weeks later, calli were transferred to MS medium containing I mg/L PPT and 1 mg/L indole-3-butyric acid (IAB) for shoot elongation and to induce root formation. After two weeks, plantlets were placed into magenta boxes for acclimatization, and 2 weeks later shoots (10-15 cm tall) with well-developed roots were transferred to potting soil and placed in a greenhouse. The regenerated transgenic plants were genotyped by PCR analysis for the presence of the selectable marker gene (Bar) and the UB1'::Ss2-ODD 1 as::NOS transgene. Tissue-cultured, non-transgenic controls were obtained by regenerating plants from embryogenic calli of the Saccharum hybrid cultivar RB835486 as described above, except for omitting particle bombardment with expression cassettes and phosphinothricin selection. Plants were propagated via stem cuttings.<br>
<br>
EXAMPLE 5. Alteration of plant architecture in UBl-l::Ss2-ODDlas::NOS plants<br>
 <br>
To assess the effect of reducing Ss2-0DD1 expression in transgenic sugarcane plants, UB1-1::Ss2-ODDlas::NOS independent transgenic events and non-transgenic wild type plants of the sugarcane variety RB835486 were evaluated in a greenhouse. Plants (12 replicates per event) were grown in coconut shell fiber substrate in 50-L pots. Plants were irrigated three times per day (totaling 1.5 L) with a nutrient solution. Up to four tillers per plant were allowed to develop, and the exceeding ones were manually removed. Seven months after seedling emergence, the irrigation regime was switched to three cycles of irrigation per day with tap water (totaling 250 mL) for 14 days to induce maturation. The two most homogeneous tillers from each replicate were selected for analyses, and their stalks were sectioned below their oldest node. The eight internodes below the first visible dewlap were identified, the stalks were sectioned at the nodes between the seventh and eighth internodes, and the apical section of the stalks was discarded along with all leaves. Stalk height and weight, and internode number, width, and length were measured. Analysis of variance (ANOVA) and Dunnett's test for mean comparison for all measurements were performed with M1N1TAB Release 14.<br>
<br>
During the stages of plant growth prior to their complete characterization, various IJB1-l::Ss2-ODDlas::NOS events showed perceptible alterations in their dimensions compared to the non-transgenic control cultivar RB835486. Measurements at seven months after seedling emergence revealed that events 06, 09, 12, 13, 14, 15, IS, and 21 had significantly increased height (Figure 4) compared to the non-transgenic control. In events 09, 13, 14, and 15, this is likely due to a significantly increased internode number (Figure 5), while in events 12, 18, and 21 this is likely a consequence of a significantly increased mean internode length (Figure 6). Measurements also revealed that internode diameter is significantly increased in events Oh and 21 (Figure 7). As a result of their increased overall dimension, events 06, 09 and 21 had significantly enhanced biomass, as illustrated by their stalk fresh weight (Figure 8). Altogether, reduction of Ss2-0DD1 expression appears to cause, at various degrees, changes in plant architecture that lead to increased dimension.<br>
<br>
EXAMPLE 6. Enhanced sucrose yield in UBI-l.:Ss2-ODDlas::NOS plants<br>
To assess the effect of reducing Ss2-0DD1 expression in transgenic sugarcane plants, UBJ-J .Ss2-ODDIas::NOS independent transgenic events and non-transgenic wild type plants of the sugarcane variety RB835486 were evaluated in a greenhouse. Plants (12 replicates per event) were grown in coconut shell fiber substrate in 50-L pots. Plants were irrigated three times per day (totaling 1.5 L) with a nutrient solution. Up to four tillers per plant were allowed to develop, and the exceeding ones were manually removed. Seven months after seedling emergence, the irrigation regime was switched to three cycles of irrigation per day with tap water (totaling 250 mL) for 14 days to induce maturation. The two most homogeneous tillers from each replicate were selected for analyses, and their stalks were sectioned below their oldest node. The eight internodes below the first visible dewlap were identified, the stalks were sectioned at the nodes between the seventh and eighth internodes, and the apical section of the stalks was discarded along with all leaves. For the assessment of appropriate maturation induction, total soluble solids were measured with a handheld Reicherl Brix Scale Retractometer and the ratio between the values obtained for the eight and antepenultimate internodes were calculated. The ratio was found to be larger than 80% in all cases, which is a positive indication of maturation. A three-roller power crusher was used to extract the juice from the stalks. The juice was immediately filtered through a stainless steel sieve. Volume of juice was measured, which was again filtered through a 120-mesh sieve. Estimation of total soluble solids and sucrose concentration in the juice was immediately performed. Juice total soluble solids were measured by refractometry with a Bellingham + Stanley RFM840 Digital Refractometer. Following clarification with a mixture of calcium hydroxide, aluminum chloride hexahydrate and Celite, juice sucrose concentration was estimated by measuring the circular birefringence with a Zeiss Polomat A polarimeter. Sucrose concentration in the juice was expressed as % POL, which indicates the number of grams of sucrose per 100 mL of solution. Total apparent sucrose was calculated as a product of juice volume multiplied by sucrose concentration in it (POL). Analysis of variance (ANOVA) and Dunnett's test for mean comparison were performed with MINITAB Release 14.<br>
<br>
Events 06 and 09 had significantly increased juice volume compared to the non-transgenic control cultivar RB835486 (Figure 9), while events 13, 14, and 21 had significantly increased sucrose concentration in the juice (Figure 10). As an estimate for sucrose yields, we calculated the total apparent sucrose. This analysis revealed that events 09, 13, 14, 20, and 21 had significantly increased sucrose yield (Figure 11). Thus, reduction of Ss2-ODDl expression appears to cause changes in juice volume and its sucrose concentration that lead to an increased sucrose yield.<br>
<br>
EXAMPLE 7. Enhanced biomass yield in UBI-l::Ss2-ODDlas::NOS plants<br>
To assess the effect of reducing Ss2-0DD1 expression in transgenic sugarcane plants, UB1-1: Ss2-ODDlas::NOS independent transgenic events and non-transgenic wild type plants of the sugarcane variety RB835486 were evaluated in a greenhouse. Plants (12 replicates per event) were grown in coconut shell fiber substrate in 50-L pots. Plants were irrigated three times per day (totaling 1.5 L) with a nutrient solution. Up to four tillers per plant were allowed to develop, and the exceeding ones were manually removed. Seven months after seedling emergence, the irrigation regime was switched to three cycles of irrigation per day with tap water (totaling 250 mL) for 14 days to induce maturation. The two most homogeneous tillers from each replicate were selected for analyses, and their stalks were sectioned below their oldest node. The eight internodes below the first visible dewlap were identified, the stalks were sectioned at the nodes between the seventh and eighth internodes, and the apical section of the stalks was discarded along with all leaves. A three-roller power crusher was used to extract the juice from the stalks. Following stalk crushing for juice extraction, the bagasse fresh weight was measured with a high capacity precision balance. Analysis of variance (ANOVA) and Dunnett's test for mean comparison were performed with MINITAB Release 14.<br>
<br>
This analysis revealed that events 9 and 21 had significantly increased amounts of bagasse fresh weight compared to the non-transgenic control (Figure 12), indicating that these events have increased biomass yield.<br>
<br>
EXAMPLE 8. Expression of Ss2-0DD1 transcripts in sugarcane organs<br>
To determine the spatial and temporal abundance of Ss2-ODDJ transcripts, we performed quantitative reverse transcription-polymerase chain reaction (qRT-PCR) using RNA isolated from different organs of wild-type sugarcane plants (cultivar RB835486). Blade of the youngest leaf, shoot apex, blade of leaf +4, sheath of leaf +4, internode +4, blade of the oldest green leaf, sheath of the oldest green leaf, internode connected to the oldest green leaf, antepenultimate internode (from apex to base), and roots were harvested from 8-month old, greenhouse grown plants. Leaf +4 is defined as the fourth youngest leaf with a clearly visible dewlap at the blade joint. Whole leaves and roots were harvested from 30-cm tall seedlings of the same cultivar. Three independent 8 month-old plants and three pools comprising three seedlings each were used in this analysis. Liquid N2-frozen tissue was ground to powder with mortar and pestle, and total RNA was isolated with the Trizol reagent (Invitrogen) according to the manufacturer's instruction.  Total  RNA was treated  with DNasel  (Promega), and  cDNA first strand was synthesized with Superscript II Reverse Transcriptase (Invitrogen) using 2 p.g of total RNA. One tenth of the cDNA was used in combination with gene specific primers at 500 nM concentration and SYBR Green PCR Master Mix (Applied Biosystems). PCR was performed on an ABI Prism 7000 Sequence Detection System (Applied Biosystems). For amplification of Ss2-ODDl transcripts, oligonucleotide primers Ss2-ODDl Fwd (SEQ ID NO: 5, CAGTTGGTAAAGAGCGGTATTCGGTGGC) and Ss2-ODDJ Rev (SEQ ID NO: 6, CTTGATAGGTGGAAACCTTGGTGGACATGC), which anneal at the 3' end of Ss2-ODDl coding sequence (base pair 770 to 880) were used. Actin (SsACT; GenBank gi 5375918S), used as a reference gene, was amplified with oligonucleotide primers SsACT Fwd (SEQ ID NO: 7, <br>
AAGCAGCATGAAGATCAAGGTCGTTGCAC) and SsACT Rev (SEQ ID NO: 8. <br>
<br>
CTGTGAACAATTGCCGGGCCAGACTC), which anneal at the 3' end of SsACT coding sequence (base pair 972 to 1121). Amplification was performed at 50°C for 2 min, 95°C for 10 min, and 45 cycles at 95°C for 15 sec and 60°C for 1 min. The specificity of the amplification reaction was evaluated by the analysis of the dissociation curves. The ratio between the amounts of the Ss2-ODDl and SsACT amplified products was calculated using the 2~AACt method (Livak and Schmittgen, Methods 25: 402-408, 2001). <br>
<br>
The normalized expression level for each replicate was calculated as L = 2"ACt and ACT = CT,Ss2-ODDl -. CT.SsACT. Ss2-ODDl transcript levels relative to those of SsACT were calculated as the average of values obtained from three independent samples used as biological replicates for each organ. Results from this analysis are shown in Figure 13.<br>
<br>
Ss2-ODDl transcripts were detected in all organs analyzed and, in agreement with its low-frequency in ESTs libraries, appeared to be much less abundant than SsACT transcripts, suggesting a low expression level. Ss2-ODDl transcripts were most abundant in leaf blades of 8 month-old plants. The highest expression level was found in the blade of the oldest green leaf, followed by blades of leaf +4 and youngest leaf. Seedling whole leaf showed an intermediate expression level. Lower transcript levels were found in leaf sheath, root, internode, and shoot apex of 8 month-old plants, and the expression was the lowest in seedling root. Although the overall expression pattern indicates that Ss2-ODDl shows a rather ubiquitous expression, it suggests a function for Ss2-ODDl in actively photosynthesizing leaves.<br>
<br>
EXAMPLE 9. Purification of Ss2-ODDl recombinant protein and generation of polyclonal Ss2-ODDl<br>
 <br>
Expression of the UBI-l::Ss2-ODDlas::NOS transgene-derived antisense transcripts were aimed at decreasing Ss2-ODDl protein abundance. We sought to produce polyclonal antibodies against recombinant Ss2-ODDl protein for its immunodetection in UBI-l::Ss2-ODDlas::NOS events and non-transgenic controls to verify whether transgene-derived antisense transcripts decreased Ss2-0DD1 protein abundance.<br>
<br>
The full-length Ss2-ODDl coding sequence was amplified from cDNA prepared from sugarcane (Saccharum hybrid cultivar RB835486) leaves with HiFi Taq DNA polymerase (lnvitrogen) with the Ss2-ODDl CDS Fwd primer (SEQ ID NO: 9, AGCATGGCAGGCAACCTGCACCTCCCCGTG) and the Ss2-ODDl CDS Rev primer (SEQ ID NO: 10, crrATlTGTATGTCGAArrrATTCGCCCAACTACGTAlTCCCCAC). The resulting 936-bp amplified fragment was cloned into pET SUMO (lnvitrogen) according lo the manufacturer's instructions, and its nucleotide sequence was verified by sequencing. The resulting construct, pET SUMO-Ss2-ODDl, encodes a fusion protein consisting of a hexahistidine-tagged, small ubiquitin modifier (SUMO) polypeptide fused at the amino (erminus of Ss2-ODDl (SUMO-Ss2-ODD.l). A serine codon was added, prior the Ss2-ODDJ translation start methionine codon, via the forward oligonucleotide primer to facilitate the removal of SUMO by digestion with SUMO protease (lnvitrogen). pET SUMO-Ss2-ODDl was transformed into Escherichia coli CY(DE3) pLysS strain (Farah and Reinach, Biochemistry 38: J0543-10551, 1999) and expression of SUMO-Ss2-ODDl fusion protein was attained by induction of cell cultures at O.D.&amp;00~0.5 with 1 mM isopropyl-(3-D-thiogalactopyranoside at 37°C for 3 hr. Cells were collected by centrifugation and lysed in PBS pH 7.2 supplemented with 0.2 mg/mL lysozyme for 30 min on ice, followed by sonication. Lysates were cleared by centrifugation at 15,000g for 15 min. Ten mM imidazole was added to the supernatant and the SUMO-Ss2-ODD1 fusion protein was bound to Ni+2 on a 1-mL HiTrap HP column (Amersham). After extensive washing with 10 mM imidazole on phosphate buffer saline (PBS) pH 7.2, the SUMO-Ss2-ODDl fusion protein was eluted with a 10-500 mM imidazole gradient on PBS pH 7.2. Native Ss2-ODD1 protein was obtained by cleavage of affinity-purified SUMO-Ss2-ODDl with SUMO protease (lnvitrogen) and further purified following the manufacturer's instructions, except for carrying out the digestion by SUMO protease at 25°C for 4 h.<br>
<br>
Native Ss2-ODDl (150 ug) was injected in rabbits with Freund's complete adjuvant. Three boost injections were made with 150 u.g of Ss2-ODDl and Freund's incomplete adjuvant. Antibodies were affinity-purified from crude sera as described by Harlow and Lane (ANTIBODIES: A LABORATORY MANUAL, Cold Spring Harbor Laboratory Press, Cold Spring Harbor, N.Y., 1988). An IgG-enriched fraction was obtained from crude antisera by precipitation at 50% ammonium sulfate saturation. The resulting pellet was solubilized in PBS pH 7.2, and dialyzed. This fraction was depleted of non-Ss2-ODDl antibodies by continuous circulation on a 1-mL HiTrap HP column (Amersham) with bound hexahistidine-tagged SUMO-CAT (chloramphenicol acetyl transferase) fusion protein expressed from the construct pET SUMO-CAT (Invitrogen) overnight at 4°C. Subsequently, the flowthrough fraction was continuously circulated on a 1-mL HiTrap NHS-activated HP column (GE Healthcare Bio-Sciences AB) with covalently linked SUMO-Ss2-ODDl fusion protein overnight at 4GC. The antibody-adsorbed column was washed extensively with PBS pH 7.2, and antibodies were eluted with 100 mM NaCl, 100 raM glycine, pH 2.4. Collected fractions were immediately neutralized with 0.1 volumes of 2 M Tris-HCl pH 8.0; those containing the bulk of lgG were pooled and concentrated with Centricon-10 spin cartridges (Millipore), followed by extensive buffer exchange with PBS pH 7.2.<br>
<br>
EXAMPLE 10. Ss2-0DD1 protein levels in UBI-l::Ss2-ODDlas::NOS events and non-transgenic controls<br>
Using affinity-purified polyclonal antibodies raised against recombinant, native Ss2-ODD1, we performed immunoblot analysis of Ss2-ODDl in organs of UBl-l::Ss2-ODDlas::NOS events and non-transgenic control plants to verify whether transgene-derived antisense transcripts decreased Ss2-ODDl protein abundance. For each transgenic event or wild-type control, blades of leaf +4 were harvested from 8-month old, greenhouse grown plants. Leal +4 is defined as the fourth youngest leaf with a clearly visible dewlap at the blade joint.<br>
<br>
Harvested plant material was immediately frozen in liquid N2, stored at -SOX for subsequent processing, was homogenized into a fine powder in liquid N2, and transferred to conical microcentrifuge tubes. Three volumes of ice-cold extraction buffer (100 mM Tris-HCl pH 6.8, 5 mM EDTA, 0.1% SDS, 175 mM p-mercaptoethanol, 10 % glycerol) supplemented with Complete EDTA-free Protease Inhibitor Cocktail (Roche Diagnostics GmbH) was added to the ground tissue and kept under constant agitation for 20 min at 4°C. Homogenates were cleared by centrifugation at 15,000g for 15 min at 4°C. Protein concentration was determined by the Bradford assay (Protein Assay kit, Bio-Rad), using BSA as standard. Soluble protein extracts (50 p.g) were separated by 12.5% SDS-PAGE, and blotted onto polyvinylidene difluoride membranes (Hybond-P PVDF, Amersham) in a tank transfer system (Mini Trans-Blot Cell, Bio-Rad) at 150 Volts X hours in TGM buffer (25 mM Tris, 192 mM glycine, 20% methanol).<br>
 <br>
Membranes were blocked with 9% fat-free milk solids in TBST (20 mM Tris-HCl pH 7.5, 150 mM NaCl, 0.05% Tween-20), and incubated with affinity-purified antibodies at 1:1,000 dilution in TBST overnight at RT. Membranes were washed with TBST for 10 min three times and incubated with a 1:10,000 dilution of a donkey anti-rabbit IgG conjugated to horseradish peroxidase (GE Healthcare Bio-Sciences AB) in TBST for 1.5 h at RT. Membranes were subsequently washed three times with TBST, and developed with ECL Plus (GE Healthcare Bio-Sciences AB) followed by exposure to radiographic film (Hyperfilm ECL; GE Healthcare Bio-Sciences AB) for various time intervals.<br>
<br>
Utilizing affinity-purified antibodies, Ss2-ODDl was detected as a polypeptide of approximately 34 kD in extracts prepared from olades of leaf +4, corresponding in size to recombinant, native Ss2-ODDl protein (Figure 14). Blades of leaf +4 from events 9, 14 and 21 had, at different degrees, decreased amounts of Ss2-ODDl protein compared to wild-type control (Figure 14). Differences observed ranged from partial reduction to nearly complete abolishment of Ss2-ODDl protein expression. Ss2-ODDl protein was hardly detected in event 14, indicating that expression of Ss2-ODDl antisense transcripts caused a drastic reduction in Ss2-ODDl protein expression. Events 9 and 21 had Ss2-ODDl protein levels intermediate to those observed in event 14 and wild-type control. Event 9 apparently had the least reduction of Ss2-ODDl protein levels among the events analyzed.<br>
<br>
Reduction in Ss2-ODDl protein levels in blades of leaf +4 were not highly correlated with the manifested phenotypes observed in the UBI-1 ::Ss2-ODDlas::NOS analyzed. Besides being event-dependent, possibly the extent of reduction in Ss2-ODDl protein levels varies in different organs, and phenotypes are best correlated with changes in Ss2-ODDl abundance in organs other than blades of leaf +4. Also, it is possible that changes in Ss2-ODDl protein levels must be optimally regulated so that most favorable phenotypic effects are attained.<br>
<br>
<br>
<br>
<br>
<br>
<br>
What is claimed is:<br>
<br>
1.	A transgenic plant belonging to a family selected from the group consisting of Poaceae, Cucurbitaceae, Cruciferaceae, Solanaceae, Legwninosae, and Apocynaceae plant family, wherein the plant contains an endogenous Ss2-ODDl DNA sequence the expression of which is reduced compared to a wild-type control plant.<br>
<br>
2.	The transgenic plant of claim 1, wherein said plant belongs to Poaceae family.<br>
<br>
3.	The transgenic plant of claim 2, wherein said plant is sugarcane, sorghum, corn, Miscanthus.<br>
<br>
4.	The transgenic plant of claim 1, wherein said endogenous Ss2-ODDl DNA sequence is reduced by antisense suppression, sense co-suppression, RNA interference, or enzymatic RNA.<br>
<br>
5.	A method for producing sucrose, comprising<br>
<br>
(a)	providing a transgenic plant having suppressed Ss2-ODDl protein levels; and<br>
<br>
(b)	obtaining sucrose from said plant.<br>
<br>
6.	The method of claim 5, wherein said plant is sugarcane or sorghum.<br>
<br>
7.	A method for producing biomass, comprising<br>
<br>
(a)	providing a transgenic plant having suppressed Ss2-ODDl protein levels; and<br>
<br>
(b)	obtaining biomass from said plant.<br>
<br>
8.	The method of claim 8, wherein said plant is sugarcane or sorghum.<br>
<br>
9.	A method for enhancing sucrose yield in a plant, comprising suppressing Ss2-ODD.l protein levels in said plant.<br>
<br>
10.	The method of claim 9, wherein said plant is sugarcane or sorghum.<br>
 <br>
11.	A method for enhancing biomass in a plant, comprising suppressing Ss2-ODDl protein levels in said plant.<br>
<br>
12.	The method of claim 11, wherein said plant is sugarcane or sorghum.<br>
<br>
13.	A nucleic acid construct comprising an Ss2-ODDl sequence.<br>
<br>
14.	A transgenic plant or cell comprising the nucleic acid construct of claim 13.<br>
 <br>
</td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/viewdoc.aspx?id=3NN/DW5/ny6oYPy8qUqwYA==&amp;amp;loc=egcICQiyoj82NGgGrC5ChA==" target="_blank" style="word-wrap:break-word;">http://ipindiaonline.gov.in/patentsearch/GrantedSearch/viewdoc.aspx?id=3NN/DW5/ny6oYPy8qUqwYA==&amp;amp;loc=egcICQiyoj82NGgGrC5ChA==</a></p>
		<br>
		<div class="pull-left">
			<a href="279415-vehicle-body.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="279417-transmission-device.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>279416</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>2608/CHENP/2009</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>04/2017</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>27-Jan-2017</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>20-Jan-2017</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>11-May-2009</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>MONSANTO DO BRASIL LTDA</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td>AVENIDA DAS NACOES UNIDAS N o&amp;nbsp;12.901 NORTH TOWER&amp;nbsp;3RD FLOOR&amp;nbsp;CONJS. N-301 N-302 7TH, 8TH AND 9TH FLOORS, DISTRICT OF BROOKLIN PAULISTA CITY OF SAO PAULO&amp;nbsp;STATE SAO PAULO BRAZIL POSTAL CODE 04578 - 000</td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>FERRO, JESUS APARECIDO</td>
											<td>RUA JAMES CLERK MAXWELL, 320, TECHNO PARK (KM 104 VIA ANHANGUERA), 13069-380 CAMPINAS/SP.</td>
										</tr>
										<tr>
											<td>2</td>
											<td>ALEGRIA, MARCOS</td>
											<td>RUA JAMES CLERK MAXWELL, 320, TECHNO PARK (KM 104 VIA ANHANGUERA), 13069-380 CAMPINAS/SP.</td>
										</tr>
										<tr>
											<td>3</td>
											<td>ARAUJO, PAULA GONCALVES DE</td>
											<td>RUA JAMES CLERK MAXWELL, 320, TECHNO PARK (KM 104 VIA ANHANGUERA), 13069-380 CAMPINAS/SP.</td>
										</tr>
										<tr>
											<td>4</td>
											<td>DANTE, RICARDO AUGUSTO</td>
											<td>RUA JAMES CLERK MAXWELL, 320, TECHNO PARK (KM 104 VIA ANHANGUERA), 13069-380 CAMPINAS/SP.</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>A01H5/00, C12N15/82</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>PCT/BR2007/294</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td>2007-10-26</td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td>60/863,252</td>
									<td>2006-10-27</td>
								    <td>U.S.A.</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/279416-method-for-modifying-plant-architecture-and-enhancing-plant-biomass-and-or-sucrose-yield by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 01:05:32 GMT -->
</html>
