<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/279521-differential-expression-of-subgenome-specific-alleles-in-cotton-and-uses-thereof by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 00:34:04 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 279521:&quot;DIFFERENTIAL EXPRESSION OF SUBGENOME SPECIFIC ALLELES IN COTTON AND USES THEREOF&quot;</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">&quot;DIFFERENTIAL EXPRESSION OF SUBGENOME SPECIFIC ALLELES IN COTTON AND USES THEREOF&quot;</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>Disclosed are methods and means to alter fiber properties such as fiber length in fiber producing plants such as cotton. Additionally plant promoters with a fiber-preferential or fiber-selective expression profile are provided.</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td>Differential expression of subgenome specific alleles in cotton<br>
and uses thereof.<br>
Field of the invention<br>
[1] This invention relates to the field of agriculture, more specifically towards the use of molecular biology techniques to alter fiber producing plants, particularly cotton plants and/or accelerate breeding of such fiber containing plants. Methods and means are provided to alter fiber qualities such as increasing fiber length, particularly lint fiber length or to decrease the length of fuzz fibers. Methods are also provided to identify molecular markers associated with fiber length in a population of cotton varieties and related progenitor plants. Furthermore, plant promoters with a fiber-preferential or fiber-selective expression profile are provided. The promoters are also temporarily regulated.<br>
Background art<br>
[2] Cotton provides much of the high quality fiber for the textile industry. The modification of cotton fibers characteristics to better suit the requirements of the industry is a major effort in breeding by either classical methods or by genetically altering the genome of cotton plants.<br>
[3] About 90% of cotton grown worldwide is Gossypium hirsutum L., whereas Gossypium barbadense accounts for about 8%. As in most flowering plants, cotton genomes are thought to have incurred one or more polyploidization events and to have evolved by the joining of divergent genomes in a common nucleus. The cotton commerce is dominated by improved forms of two "AD" tetraploid species, Gossypium hirsutum L. and Gossypium barbadense L. Tetraploid cottons are thought to have formed about 1-2 million years ago, in the New World, by hybridization between a maternal Old World "A" genome taxon resembling Gossypium h'erbaceum and paternal New World "D" genome taxon resembling Gossypium raimondii or Gossypium gossypioides . Wild A genome diploid and AD tetraploid Gossypium taxa produce spiimable fibers. One A genome diploids species, Gossypium arboreum, remains intensively bred and cultivated<br>
in Asia. Its close relative and possible progenitor, the A genome diploid species G. herbaceum also produces spinnable fiber. Although the seeds of D genome diploids are pubescent, none produce spinnable fibers. No taxa from the other recognized diploid Gossypium genomes (B, C, E, F, and G) have been domesticated. Intense directional selection by humans has consistently produced AD tetraploid cottons that have superior yield and/or quality characteristics compared to the A genome diploid cultivars. Selective breeding of G. hirsutum (AADD) has emphasized maximum yield, whereas G. barbadense (AADD) is prized for its fibers of superior length, strength, and fineness (Jiang et al. 1998 - Proc Natl Acad Sci USA. 1998 April 14; 95(8): 4419-4424).<br>
[4] A cotton fiber is a single cell that initiates from the epidermis of the outer integument of the ovules, at or just prior to anthesis. Thereafter, the fibers elongate rapidly for about 3 weeks before they switch to intensive secondary cell wall cellulose synthesis. Fiber cells intercormect only to the underlying seed coat at their basal ends and influx of solute, water and other molecules occurs through either plasmodesmata or plasma membrane. Ruan et al. 2001 (Plant Cell 13: 47-63) demonstrated a transient closure of plasmodesmata during fiber elongation. Ruan et al. 2004 (Plant Physiology - Vol 136: pp. 4104-4113) compared the duration of plasmodesmata closure among different cotton genotypes differing in fiber length and found a positive correlation between the duration of the plasmodesmata closure and fiber length. Furthermore, microscopic evidence was presented showing callose deposition and degradation at the fiber base, correlating with the timing of plasmodesmata closure and reopening. Furthermore, expression of a 1,3-endoglucanase gene (GhGlucl) in the fibers, allowing to degrade callose, correlated with the reopening of the plasmodesmata at the fiber base.<br>
[5] W02005/017157 desribes methods and means for modulating fiber length in fiber producing plants such as cotton by altering the fiber elongation phase as described in Ruan et al 2001. The fiber elongation phase may be increased or decreased by interfering with callose deposition in plasmodesmata at the base of the fiber cells.<br>
[6] Furthermore, it would be interesting for modification of fibers through genetic engineering, to possess promoters which are preferentially or specifically expressed in fibers cells only, and/or which are expressed only firom a particular fiber development stage on.<br>
[7] WO2004/018620 relates to an isolated nucleic acid molecule encoding an endogenous cotton chitinase and its promoter, which are preferentially expressed in fibers during secondary wall deposition. The polypeptide encoded by the nucleic acid molecule, a DNA construct linking the isolated nucleic acid molecule with a promoter, the DNA construct incorporated in an expression system, a host cell, a plant, or a plant seed are also disclosed. The document also relates to a DNA construct linking the isolated promoter with a second DNA as well as expression systems, host cells, plants, or plant seeds containing the DNA construct. Methods of imparting resistance to insects and fungi, regulating the fiber cellulose content, and methods of expressing a gene preferentially in fibers during secondary wall deposition are also disclosed.<br>
[8] It would be useful to have alternative promoters that would drive gene expression preferentially and/or strongly in fibers throughout secondary wall deposition, i. e. , strongly and continuously (e. g. at &gt; 50% of its maximal activity) e.g. from the initiation of secondary wall deposition to its termination or e.g. fi-om maturation stage on. The initiation of secondary wall deposition is defined as the time when the dry weight/unit length of a cotton fiber begins to increase or when the dry weight/unit surface area of any cell begins to increase via synthesis of new wall material containing more than 40% (w/w) of cellulose. In the case of cotton fiber of G. hirsutum L., this is expected to occur between 14-17 DP A when cotton plants are grown under typical conditions in the greenhouse or the field (day temperature of 26-34°C, night temperature of 20-26°C, light intensity greater than or equal to 1000 einsteins/mVs, with adequate water and mineral nutrition). The end of the secondary cell wall formation and start of the maturation phase is usually around 35DPA in the case of cotton fiber of G. hirsutum L.<br>
[9] Furthermore, it would be usefiil to have alternative promoters that would drive gene expression only or preferentially in fibers while excluding or minimizing expression in other tissues.<br>
[10] The inventions described hereinafter in the different embodiments, examples, figures and claims provide improved methods and means for modulating fiber length by decreasing or increasing callose deposition at the base of the fiber cell at a particular time point. The inventions described hereinafter also provide fiber-specific and/or fiber-preferential promoters and promoter regions.<br>
Summary of the invention<br>
[11] In one embodiment of the invention, a fiber cell preferential promoter is provided comprising a nucleotide sequence selected fbom the following group of nucleotide sequences:<br>
a)	a nucleotide sequence comprising the nucleotide sequence of SEQ ID No 9 firom the nucleotide at position 2149 to the nucleotide at position 2307;<br>
b)	a nucleotide sequence comprising the nucleotide sequence of SEQ ID No 10 firom the nucleotide at position 3109 to the nucleotide at position 3269;<br>
c)	a nucleotide sequence comprising the nucleotide sequence having at least 70% preferably at least 80 %, more preferably at least 90%, particularly at least 95% sequence identity with or, more particularly is identical to any of the nucleotide sequence mentioned under a) or b) or<br>
d)	a nucleotide sequence comprising the nucleotide sequence of an about 150 bp to an about 1000 bp DNA firagment hybridizing under stringent conditions with a DNA firagment having the nucleotide sequence mentioned under a), b) or c).<br>
The fiber cell preferential promoter may comprise the nucleotide sequence of SEQ ID No 9 firom position 465 to position 2307; or the nucleotide sequence of SEQ ED No 9 firom position 1374 to position 2307; or the nucleotide sequence of SEQ ID No 9 from position 1531 to position 2307; or the nucleotide sequence of SEQ ID No 10 fi-om position 1397 to position 3269; or the nucleotide sequence of SEQ ID No 10 fi"om position 2371 to<br>
position 3269; or the nucleotide sequence of SEQ ID No 10 firom position 2718 to position 3269.<br>
[12] In another embodiment, the invention provides a fiber cell preferential promoter region comprising a fiber cell preferential promoter as herein described further comprising the nucleotide sequence of SEQ ID 9 from the nucleotide at position 2308 to the nucleotide at position 2409 or the nucleotide at position 3270 to the nucleotide at position 3372.<br>
[13] In yet another embodiment of the invention, a chimeric gene is provided comprising the following operably linked DNA regions<br>
a)	a fiber cell preferential promoter as herein described;<br>
b)	a heterologous DNA region encoding a biologically active RNA of interest; and<br>
c)	a transcription termination and polyadenylation signal.<br>
[14] The biologically active RNA may encode a protein of interest, such as a N-acetylglucosamine transferase, preferably an N-acetylglucosamine transferase of the Node type, a cellulose synthase, a sucrose synthase, preferably a sucrose synthase of the C-type; a sucrose phosphate synthase or a ß-1,3-endoglucanase. The biologically active RNA may also be an inhibitory RNA or silencing RNA such as a ribozyme, microRNA, double stranded hairpin RNA , particularly targed to downregulate the expression of an endogenous cotton gene, such as ß-1,3-endoglucanase xylan synthase (cslF) xyloglucan synthase (cslD) or 1,3-1,4 glucan synthase (cslC).<br>
[15] Also provided are plant cell and plants, such as cotton plants, and seeds thereof, comprising such chimeric genes. A seed of a plant comprising in its cells a chimeric gene according to any one of claims 11 to 15.<br>
[16] The invention further provides a method for expressing a biologically active RNA preferentially in a fiber cell of a fiber producing plant, such as a cotton plant, the method<br>
comprising the steps of providing the cells of the plants with a chimeric gene according to the invention and growing the plants.<br>
[17] In yet another embodiment, the invention is directed at the use of a fiber-specific and/or fiber-preferential promoter according to the invention for preferential expression of a biologically active KNA in fiber cells of a fiber-producing plant such as a cotton plant.<br>
[18] The invention fiirther provides an isolated DNA molecule comprising a nucleotide sequence encoding a protein comprising an amino acid sequence having at least 95% sequence identity, preferably is identical, to the amino acid sequence of SEQ ID No 11 or SEQ ID No 12 and encoding a j3-l,3-endoglucanase; an isolated DNA molecule comprising a nucleotide sequence selected from the group of SEQ ID No 1; SEQ ID No 2, SEQ ID No 3 and SEQ ID No 4 or an isolated DNA molecule comprising a nucleotide sequence selected from the nucleotide sequence of SEQ ID No 9 fix)m position 2410 to 2443 and the nucleotide sequence of SEQ ID No 9 from position 2556 to 3499 or the nucleotide sequence of SEQ ID No 10 from position 3373 to 3406 and the nucleotide sequence of SEQ ID No 10 from position 3501 to 4444. These isolated DNA molecules may be used for isolation of a fiber cell preferential promoter or promoter region.<br>
[19]     The invention also provides a method for isolating a fiber cell preferential<br>
promoter region, comprising the steps of:<br>
-identifying a genomic fragment encoding an RNA transcript from which a cDNA can be synthesized, the cDNA comprising the nucleotide sequence of SEQ ID 1 , SEQ ID No 2, SEQ ID 3 or SEQ ID No 4 or fiinctional equivalents thereof; - isolating a DNA region upstream of a nucleotide sequence encoding the protein with the amino acid of SEQ ID No 11 or SEQ ID No 12 or functional equivalents thereof<br>
Also provided are fiber selective promoters and promoter regions obtained by this<br>
method.<br>
[20] According to a second aspect of the invention, a method is provided for altering the properties of a fiber in a fiber producing plant, particularly increasing the length of a fiber of a cotton plant, comprising the step of introducing a chimeric gene into a cell of the cotton plant, the chimeric gene comprises the following operably linked DNA elements:<br>
(a)	a plant expressible promoter, preferably a plant expressible promoter which controls transcription preferentially in the fiber cells such as a fiber-specific beta tubulin promoter from cotton, a fiber-specific actin promoter from cotton, a fiber specific promoter Scorn a lipid transfer protein gene from cotton, a promoter from an expansin gene from cotton or a promoter from a chitinase gene in cotton or a promoter as herein described;<br>
(b)	a transcribed DNA region, which when transcribed yields a biologically active RNA molecule capable of reducing the expression of a j3-l,3 endoglucanase encoding gene endogenous to the fiber producing plant, the j8-l,3 endoglucanase being involved in callose removal from the plasmodesmata and<br>
(c)	a 3' end region comprising transcription termination and polyadenylation signals fimctioning in cells of the plant,<br>
characterized in that only the expression of a specific subgenomic allele of the endogenous i9-1,3 endoglucanase encoding gene is downregulated. In a preferred embodiment that j3-l,3 endoglucanase encoding gene comprised within the D subgenome of the cotton plant. The j8-l,3 endoglucanase encoding gene comprised within the D subgenome may be characterized by the presence of an intron sequence having the nucleotide sequence of SEQ ED No 10 from position 3407 to 3500 or may be characterized by the presence of a nucleotide sequence AAGATC about 326 nucleotides downstream the translation initiation codon (and not including that translation initiation codon). Conveniently, the j8-l,3 endoglucanase encoding gene comprised within the D subgenome may be characterized the identification of an about 538 bp fragment and an about 118 bp Augment after PCR amplification with oligonucleotides having the nucleotide sequence of SEQ ID No 5 and SEQ ID no 6 followed by digestion with Alwl restriction enzyme. The ß-l,3 endoglucanase encoding gene may also be encoding a protein comprising the amino acid sequence of SEQ ID No 12 or wherein the gene<br>
comprises the nucleotide sequence of SEQ ID No 2. The cotton plant may be Gossypium hirsutum.<br>
[21] The biologically active RNA may be an antisense RNA comprising at least 19 nucleotides having at least 94% sequence identity to the complement of nucleotide sequence of SEQ ID No 3 or a sense RNA comprising at least 19 nucleotides having at least 94% sequence identity to the nucleotide sequence of SEQ ID No 3 or a double stranded RNA molecule comprising at least 19 nucleotides having at least 94% sequence identity to the complement of the nucleotide sequence of SEQ ID No 3 and a complementary RNA strand essentially similar to the complement of the at least 19 nucleotides. Preferably, the mentioned 19 nucleotides comprises at least one nucleotide specific for the D subgenome Ghglucl gene as indicated in Figure 2. In yet another embodiment according to the invention, double stranded RNA is micro RNA processed firom a pre- microRNA comprising the nucleotide sequence of SEQ ID No 15, SEQ ID No 16, SEQ ID No 20 or SEQ ID No 21.<br>
[22] The invention also provides a method for decreasing the length of a fiber of a fiber producing plant, comprising the step of introducing a chimeric gene into a cell of the fiber producing plant wherein the chimeric gene comprises the following operably linked DNA fragments:<br>
(a)	a plant-expressible promoter according to claims lb, 5, 6 or 7;<br>
(b)	a DNA region encoding a P-1,3 glucanase protein such as a protein having an amino acid sequence having at least 95% sequence identity to the amino acid sequence of SEQ ID No 11 or SEQ ID No 12 ; and<br>
(c)	a 3'end region comprising transcription termination and polyadenylation signals fimctioning in cells of the plant.<br>
[23] Yet another embodiment of the invention concerns a chimeric gene as herein described or a cell of a fiber-producing plant or a fiber producing plant comprising such chimeric gene<br>
[24]     In still another embodiment, the invention provides fibers produced according to the methods herein described.<br>
Brief description of the figures<br>
[25] Figure 1: alignment of promoter regions of the ß-1,3-endoglucanase gene from subgenome A and from subgenome D of Gossypium hirsutum. The nucleotide sequence of promoter of subgenome A (upper sequence) corresponds to the nucleotide sequence of SEQ ID No 9 from position 1460 to position 2408. The nucleotide sequence of promoter of subgenome D (lower sequence) corresponds, to the nucleotide sequence of SEQ ID No 10 from position 2372 to position 3372. Differences in nucleotide sequence are indicated by gray boxes. Nucleotides which do not have a corresponding nucleotide in the other promoter region are indicated by dashes in the nucleotide sequence missing the nucleotides. Overall homology between the tw;o promoter regions is about 71% sequence identity.<br>
[26] Figure 2: alignment of coding regions of the ß-1,3-endoglucanase gene (without intron) from subgenome A and from subgenome D of Gossypium hirsutum. The nucleotide sequence of coding region of subgenome A allele (upper sequence) corresponds to the nucleotide sequence of SEQ ID No 9 from position 2410 to position 2443 and from position 2556 to 3499. The nucleotide sequence of coding region of subgenome D allele (lower sequence) corresponds to the nucleotide sequence of SEQ ID No 10 from position 3373 to position 3406 and from position 3501 to 4444. Differences in nucleotide sequence are indicated by gray boxes. Nucleotides which do not have a corresponding nucleotide in the other promoter region are indicated by dashes in the nucleotide sequence missing the nucleotides. Overall homology between the two promoter regions is about 97% sequence identity.<br>
[27] Figure 3: alignment of the intron sequence from, the |8-l,3-endoglucanase gene from subgenome A and from subgenome D of Gossypium hirsutum. Upper sequence<br>
corresponds to the nucleotide sequence of the subgenome A allele intron, while the lower sequence corresponds to the nucleotide sequence of the subgenome D allele intron.<br>
[28] Figure 4; alignment of the amino acid sequence of the j8-l,3-endoglucanase encoded by the subgenome A allele (upper sequence corresponding to SEQ ED No 11) and subgenome D allele Oower sequence corresponding to SEQ ED No 12).<br>
[29] Figure 5: alignment of the nucleotide sequence of SEQ IDs 1 to 4 fi-om nucleotide position 201 to 211. The Alwl restriction enzyme recognition site present in the nucleotide sequence of the subgenome A allele only of the /S-l,3-endoglucanase gene is underlined and represented in italics.<br>
[30] Figure 6: Correlation between fiber growth curve (panel A) and expression of the j3-l,3-endoglucanase gene (GhGlucl) (panel B). DNA fi-om a cDNA library from (developing) fibers in Gossypium hirsutum was extracted and equalized. PCR fiagments were amplified using oligonucleotide primers SE002 and SE003 (SEQ ID No 5 and 6) and digested with Alwl. A PCR amplified product for the A-genome variant yields 3 fragments (479bp+118bp+59bp) while for the D-genome variant it only yields 2 fragments (538bp +118bp).<br>
[31] Figure 7: Expression analysis of GhGlucl in cotton leaves, roots and stems by RT-PCR. RNA extracted from cotton leaves, roots and stems was subjected to RT-PCR analysis to detect expression of glucanase 1 or protein phosphatase 2A (pp2A). First strand cDNA synthesis was performed using Superscript First-Strand Synthesis System from Invitrogen. Lane 1: 100 bp ladder; lane 2: cotton leaf RNA sample + Glucl specific primers + reverse transcriptase; lane 3: cotton leaf RNA sample + Glucl specific primers - reverse transcriptase; lane 4: cotton root RNA sample + Glucl specific primers + reverse transcriptase; lane 5: cotton root RNA sample + Glucl specific primers - reverse transcriptase; lane 6: cotton stem RNA sample + Glucl specific primers + reverse transcriptase; lane 7: cotton stem RNA sample + Glucl specific primers - reverse transcriptase; lane 8: genomic DNA (Fibermax 400ng) + Glucl specific primers; lane 9:<br>
100 bp ladder; lane 10: cotton leaf RNA sample + pp2A specific primers + reverse transcriptase; lane 11: cotton leaf RNA sample + pp2A specific primers - reverse transcriptase; lane 12: cotton root RNA sample + pp2A specific primers + reverse transcriptase; lane 13: cotton root RNA sample + pp2A specific primers - reverse transcriptase; lane 14: cotton stem RNA sample + pp2A specific primers + reverse transcriptase; lane 15: cotton stem RNA sample + pp2A specific primers - reverse transcriptase; lane 16: 100 bp ladder; lane 17: positive control RNA (supplied with kit).<br>
Detailed embodiments<br>
[32] The current invention is based on the unexpected finding that the timing of expression of the A and D subgenome specific alleles of the fiber specific ß -1,3-endoglucanase gene {GhGlucV) is different at least in Gossypium hirsutum. Whereas the onset of the expression of the D subgenome specific allele correlates with the end of the rapid elongation phase (about 14 to 17 days post-anthesis hereinafter "DPA"), onset of the expression of the A subgenome is delayed until the beginning of the late fiber maturation phase (about 35-40 DP A) depending on growth conditions.<br>
[33] Accordingly, in one aspect, the current invention relates to the identification of novel fiber-specific promoters which either are expressed preferentially in fiber cells fi"om the end of the rapid elongation phase or fi-om the onset of the fiber maturation phase. Such promoters have a utility e.g. in the expression of novel biomolecule producing chimeric genes at late fiber development stages, for example encoding enhanced XET, Cellulose Binding Domain (CBD), Root Swollen (RSW) mutant genes, coloring, plastics, cellulose digestibility, novel cell wall composites/properties. The promoters could also be used to direct gene expression at 20, 30, 40 DPA in fibers; for example production of chitin in late stages of cotton fiber development (20, 30, 40 DPA) or for gene expression at 40DPA in fibers for example production of chitin in very late stages of cotton fiber development (40 DPA). The promoters according to the invention could also be used to direct expression of CesA genes at 40DPA for enhanced cellulose biosynthesis and enhanced fiber properties or to produce biologically active RNA<br>
directed at stage specific silencing of endogenous cotton genes, such as silencing |8-1,3-endoglucanase for longer fiber growth. The promoters could also be applied in the stage specific expression of chimeric genes encoding e.g. callose synthase to alter fiber properties and quality.<br>
[34] In another aspect, the current invention is directed towards the silencing of the expression of one subgenome specific allele of the fiber specific /S-1,3- endoglucanase gene, particularly silencing of the D-subgenome specific allele, to prevent callose degradation and thus increase the rapid fiber elongation phase and the fiber length e.g. in cotton, particularly in Gossypium hirsutum._<br>
[35] According to one embodiment of the first aspect of the invention, the invention provides a fiber-specific and/or fiber-preferential promoter fi:agment which can be isolated fi"om cotton, particularly fi'om Gossypium hirsutum species, and which is located upstream of the nucleic acid sequences encoding a fiber specific i8-l,3- endoglucanase having an amino acid sequence comprising the amino acid sequence of SEQ ID 11,12 , 7 or 8 or variants thereof.<br>
[36] As used herein, the term "promoter" denotes any DNA which is recognized and bound (directly or indirectly) by a DNA-dependent RNA-polymerase during initiation of transcription. A promoter includes the transcription initiation site, and binding sites for transcription initiation factors and RNA polymerase, and can comprise various other sites (e.g., enhancers), at which gene expression regulatory proteins may bind.<br>
[37] The term "regulatory region", as used herein, means any DNA, that is involved in driving transcription and controlling (i.e., regulating) the timing and level of transcription of a given DNA sequence, such as a DNA coding for a protein or polypeptide. For example, a 5' regulatory region (or "promoter region") is a DNA sequence located upstream (i.e., 5') of a coding sequence and which comprises the promoter and the 5'-untranslated leader sequence.  A 3' regulatory region is a DNA sequence located<br>
downstream (i.e., 3') of the coding sequence and which comprises suitable transcription 3' end formation (and/or regulation) signals, including one or more polyadenylation signals.<br>
[38] The term "gene" means any DNA fi-agment comprising a DNA region (the "transcribed DNA region") that is transcribed into a RNA molecule (e.g., a mRNA) in a cell under control of suitable regulatory regions, e.g., a plant expressible promoter region. A gene may thus comprise several operably linked DNA firagments such as a promoter, a 3' untranslated leader sequence, a coding region, and a 3' untranslated region comprising a polyadenylation site. An endogenous plant gene is a gene which is naturally found in a plant species. A chimeric gene is any gene which is not normally found in a plant species or, alternatively, any gene in which the promoter is not associated in nature with part or all of the transcribed DNA region (a "heterologous" DNA region) or with at least one other regulatory regions of the gene.<br>
[39] The term "expression of a gene" refers to the process wherein a DNA region imder control of regulatory regions, particularly the promoter, is transcribed into an RNA which is biologically active i.e., which is either capable of interaction with another nucleic acid or which is capable of being translated into a biologically active polypeptide or protein. A gene is said to encode an RNA when the end product of the expression of the gene is biologically active RNA, such as an antisense RNA or a ribozyme. A gene is said to encode a protein when the end product of the expression of the gene is a functionally or biologically active protein or polypeptide.<br>
[40] The term "fiber-selective" or "fiber cell selective" or "fiber specific" or "fiber cells specific", with respect to the expression of a DNA in accordance with this invention, refers to, for practical purposes, the highly specific, expression of a DNA in fiber cells of plants, such as cotton plants ("fiber cell-selective"). In other words, transcript levels of a DNA in tissues different of fiber cells is either below detection or very low (less than about 0.2 pico grammes per microgram total RNA).<br>
 [41] The term "fiber-preferential" or "fiber-cell preferential" with respect to the expression of a DNA in accordance with this invention, refers to an expression pattern whereby the DNA is expressed predominantly in fiber cells or fibers, but expression can be identified in other tissues of the plant. Preferably, the expression in fiber cells is about 2 to about 10 times higher in the fiber cells than in other tissues.<br>
[42] The nucleotide sequences of SEQ ID No 9 and SEQ ID No 10 represent the nucleotide sequences of genomic DNA molecules encoding respectively a fiber selective P-1,3 endoglucanase of the A subgenome and of the D subgenome. The transcription initation site of SEQ ED No 9 has been determined at position 2308, while the transcription initiation site of SEQ ID No 10 has been determined at position 3270. Thus in one embodiment of the invention a fiber selective promoter is provided having the nucleotide sequence of SEQ ID No 9 firom nucleotide position 1 to nucleotide position 2307 or 2308; in another embodiment of the invention a fiber selective promoter is provided having the nucleotide sequence of SEQ ID No 10 fi-om nucleotide position 1 to nucleotide position 3269 or 3270.<br>
[43] The translation initiation site of the nucleotide sequence of SEQ ID No 9 has been determined as the ATG codon located at positions 2410 to 2412. The 5'untranslated region of the fiber selective promoter of the A subgenomic allele encoding P-1,3 endoglucanase accordingly has a nucleotide sequence of SEQ ID No 9 fi"om nucleotide position 2308 to nucleotide position 2409. A fiber selective promoter region of the A subgenomic allele is accordingly a promoter region comprising nucleotides 1 to 2409 of SEQ ID No 9.<br>
[44] Likewise, the translation initiation site of the nucleotide sequence of SEQ ID No 10 has been determined as the ATG codon located at positions 3373 to 3375. The 5'imtranslated region of the fiber selective promoter of the D subgenomic allele encoding P-1,3 endoglucanase accordingly has a nucleotide sequence of SEQ ED No 10 fi-om nucleotide position 3270 to nucleotide position 3372. A fiber selective promoter region of<br>
the D subgenomic allele is accordingly a promoter region comprising nucleotides 1 to 3372ofSEQIDNolO.<br>
[45] Alignment of the fiber selective promoters regions of SEQ ID No 9 and SEQ ID No 10 (Figure 1) revealed a general sequence homology in the nucleotide sequences about 1000 nucleotides upstream from the ATG translation initiation codon, particularly in the region about 150 nucleotide upstream from the ATG translation initiation codon. Thus, in another embodiment of the invention, a fiber-selective promoter is provided comprising the nucleotide sequence of SEQ ID No 9 from nucleotide position 2149 to nucleotide position 2307 or comprising the nucleotide sequence of SEQ ID No 10 from nucleotide position 3109 to nucleotide position 3269.<br>
[46] It goes without saying that promoter or the promoter region may be contained within larger nucleotide sequences. A fiber cell selective promoter region can thus be determined as the region upstream (i.e., located 5' of) from the codon coding for the first amino acid of the protein encoded by the mRNA indicated in SEQ ID No 9 or 10. Such a promoter region may be at least about 400 to 500 bp, at least about 1000 bp, at least 1200 bp, at least about 1300 bp, or at least about 1500 to 2000 bp, upstream of the start codon. For convenience, it is preferred that such promoter region does not extend more than about 3000 to 5000 bp upstream of the start codon. The size fragment may be partially determined by the presence of convenient restriction sites. E.g. for the fiber selective promoter region of the A subgenome allele encoding P-1,3 endoglucanase, a fragment of about 1945 bp (from nucleotide position 465 to nucleotide position 2409 of SEQ ID No 9) can be conveniently cloned. The presence of suitable restriction enzymes allows to conveniently generate promoter regions of about 1036 bp (from nucleotide 1374 to nucleotide position 2409 of SEQ ID No 9) or of about 879 np (from nucleotide 1531 to nucleotide position 2409 of SEQ ID No 9). For the fiber selective promoter region of the D subgenome allele encoding (3-1,3 endoglucanase, a fragment of about 1976 bp (from nucleotide position 1397 to nucleotide position 3372 of SEQ ID No 10) can be conveniently cloned. The presence of suitable restriction enzymes allows to conveniently generate promoter regions of about 1002 bp (from nucleotide 1397 to nucleotide position<br>
3372 of SEQ ID No 10) or of about 655 np (from nucleotide 2371 to nucleotide position 3372ofSEQIDNolO).<br>
[47] It will further be clear that equivalent fiber selective promoters or promoter regions can be isolated from other cotton plants or cotton progenitor plants. To this end, promoter fragments may be isolated from other cotton plants, such a G. barbadense including the so-called PIMA varieties, or from other varieties using a promoter fragment as herein described as a probe and identifying nucleotide sequences from these other plants which hybridize imder stringent hybridization conditions.<br>
[48] "Stringent hybridization conditions" as used herein means that hybridization will generally occur if there is at least 95% and preferably at least 97% sequence identity between the probe and the target sequence. Examples of stringent hybridization conditions are overnight incubation in a solution comprising 50% formaraide, 5 x SSC (150 roM NaCl, 15 mM trisodium citrate), 50 mM sodium phosphate (pH 7.6), 5x Denhardt's solution, 10% dextran sulfate, and 20 |ig/ml denatured, sheared carrier DNA such as salmon sperm DNA, followed by washing the hybridization support in 0.1 x SSC at approximately 65 °C. Other hybridization and wash conditions are well known and are exemplified in Sambrook et al. Molecular Cloning: A Laboratory Manual, Second Edition, Cold Spring Harbor, NY (1989), particularly chapter 11.<br>
[49] Such equivalent fiber selective promoters isolated from other cotton plants or cotton progenitor plants may represent variant promoters wherein one or more nucleotides have been replaced by substition, deletion or insertion. Variant fiber selective promoter according to the invention may also be generated synthetically. Figure 1 represents the alignment of fiber selective promoters from the A and D subgenome of G. hirsutum and variant positions, where the nucleotide sequence may be apparently changed without a significant effect on the fiber selective transcription initiation capacity, have been indicated by gray boxes. From Figure 1 and its legend, it also apparent that the overall sequence identity between the two exemplified fiber selective promoter fragments maybe as low as about 71% sequence identity.<br>
 [50] Accordingly, in another embodiment of the current invention, fiber selective promoter and promoter regions are provided which comprise a nucleotide sequence having at least 70%, or at least 80 % , or at least 90% or at least 95 % sequence identity to the herein described fiber selective promoters and promoter regions.<br>
[51] For the purpose of this invention, the "sequence identity" of two related nucleotide or amino acid sequences, expressed as a percentage, refers to the number of positions in the two optimally aligned sequences which have identical residues (xlOO) divided by the number of positions compared. A gap, i.e. a position in an alignment where a residue is present in one sequence but not in the other, is regarded as a ]x&gt;sition with non-identical residues. The alignment of the two sequences is performed by the Needleman and Wimsch algorithm (Needleman and Wunsch 1970). The computer-assisted sequence alignment above, can be conveniently performed using standard software program such as GAP which is part of the Wisconsin Package Version 10.1 (Genetics Computer Group, Madision, Wisconsin, USA) using the default scoring matrix with a gap creation penalty of 50 and a gap extension penalty of 3.<br>
[52] It will be clear that whenever nucleotide sequences of RNA molecules are defined by reference to nucleotide sequence of corresponding DNA molecules, the thymine (T) in the nucleotide sequence should be replaced by uracil (U). Whether reference is made to RNA or DNA molecules will be clear fi"om the context of the application.<br>
[53] Fiber selective expression capacity of the identified or generated promoter or promoter regions can be conveniently tested by operably linking such DNA molecules to a coding region encoding an easily scorable marker,e.g. a ^-glucuronidase gene, introducing such a chimeric gene into a fiber producing plant, and analyzing the expression pattern of the marker in fiber cells (preferably during fiber development) in comparison with the expression pattern of the marker in other parts of the plants.<br>
 [54] It goes without saying that promoters and promoter regions of the invention may also comprise additional elements known to improve transcription efficiency such as enhancers, introns, etc.<br>
[55] Further, the exemplified 5' UTR sequences of the exemplified fiber selective promoter regions are rather similar in nucleotide sequence, and it is expected that the 5'UTR sequences are interchangeable.<br>
[56] The invention further includes DNA molecules comprising the fiber selective promoters or promoter regions of the invention operably linked to one or more heterologous regions coding for a biologically active RNA, peptide or protein. The promoters of the invention may be used to express any heterologous coding region desired.<br>
[57] Examples of other protein-encoding DNA molecules that could be expressed using the promoter of the current invention include, but are not limited to, homologous and heterologous cellulose synthases (CesA genes), both in normal and mutated form (Arioli et al. ."Molecular Analysis of Cellulose Biosynthesis in Arabidopsis," Science, 279: 717-720 (1998) ; Holland et al. ,"A Comparative Analysis of the Plant Cellulose Synthase (CesA) Gene Family,"Plant Physio., 123: 1313-1324 (2000)); genes that may modulate carbon partitioning to cellulose (Delmer, "Cellulose Biosynthesis in Developing Cotton Fibers"in: A. S. Basra(ed.), Cotton Fibers: Developmental Biology Quality Improvement, and Textile Processing, The Haworth Press, New York, pp. 85-112 (1999)) such as sucrose synthase (Amor et al. ,"A Membrane-Associated Form of Sucrose Synthase and Its Potential Role Synthesis of Cellulose and Callose in Plants," Proc. Natl. Acad. Sci. USA, 92: 9353-9357 (1995), sucrose phosphate synthase (Haigler et al., "Transgenic Cotton Over-Expressing Sucrose Phosphate Synthase Produces Higher Quality Fibers with Increased Cellulose Content and Has Enhanced Seed Cotton Yield" Abstract 477. In: Proceedings of Plant Biology 2000, July 15-19, San Diego, CA. American Society of Plant Physiologists, Rockville, MD, (2000), UDPG- pyrophosphorylase (Waflfler and Meier, "Enzyme Activities in Developing<br>
Cotton Fibers, "Plant Phvsiol. Biochem. 32: 697-702 (1994), inorganic<br>
pyrophosphosphatase (Geigenberger et al., "Overexpression of Pyrophosphatase Leads to<br>
Increased Sucrose Degradation and Starch Synthesis, Increased Activities of Enzymes for<br>
Sucrose-Starch Interconversions, and Increased Levels of Nucleotides in Growing Potato<br>
Tubers," Planta, 205:428-437 (1998)), hexokinases (Smeekens, "Sugar Regulation of<br>
Gene Expression" Curr. Op. Plant Biol, 1: 230-234 (1998), and invertases (Sturm and<br>
Tang, "The Sucrose-Cleaving Enzymes of Plants are Crucial for Development, Growth,<br>
and Carbon Partitioning,"Trends Plant Sci., 4: 401-407 (1999)); genes that might affect<br>
the molecular and biophysical properties of cellulose including degree of polymerization,<br>
degree of crystallinity, crystallite size, and micro fibril orientation (i. e. genes for<br>
encoding proteins, including co- crystallizing protein polymers or cellulose binding<br>
domains, and polysaccharide- synthesizing and other enzymes) (Delmer, "Cellulose<br>
Biosynthesis: Exciting Times for a Difficult Field of Study, "Ann. Rev. Plant Physio.<br>
Mol. Biol. 50: 245- 276 (1999); Delmer, "Cellulose Biosynthesis in Developing Cotton<br>
Fibers. In: A. S. Basra(ed.), Cotton Fibers: Developmental Biology, Quality<br>
Improvement, and Textile Processing, The Haworth Press, New York, pp.85-112 (1999);<br>
Hsieh, "Structural Development of Cotton Fibers. In: A. S. Basra(ed.), Cotton Fibers:<br>
Developmental Biology, Quality Improvement, and Textile Processing, The Haworth<br>
Press, New York, pp. 137-166 (1999)); transcription factors such as MYB genes that<br>
could prolong elongation growth and/or change the timing or extent of secondary wall<br>
deposition (Wilkins and Jemstedt,"Molecular Genetics of Developing Cotton Fibers. In:<br>
A. S. Basra(ed.), Cotton Fibers: Developmental Biology. Quality Improvement, and<br>
Textile	Processing,	The	Haworth	Press,	New<br>
York, pp. 231-270 (1999)); genes to effect the synthesis of plant hormones and change fiber properties (John, "Genetic Engineering Strategies for Cotton Fiber Modification. In: A. S. Basra(ed.), Cotton Fibers: Developmental Biology, Quality Improvement, and Textile Processing, The Haworth Press, New York, pp. 271-289 (1999)); genes for cytoskeletal elements or cytoskeletal-associated proteins that might affect fiber properties (Seagull, "Cytoskeletal Involvement in Cotton Fiber Growth andDevelopment," Micron, 24: 643-660 (1993)); genes for lipid synthesizing or modifying enzymes that might change membrane properties and thereby improve fiber<br>
quality, including under stressful environmental conditions (Haigler,"The Crystallinity of<br>
Cotton Cellulose in Relation to Cotton Improvement, "Proc. Cotton Fiber Cellulose :<br>
Structure. Function and Utilization Conference, National Cotton Council of America:<br>
Memphis, TN. , p. 211-225 (1992)); enzymes such as xyloglucan endotransferase,<br>
peroxidase, expansin, or vacuolar ATPase that might, through increased or decreased<br>
activity, prolong or increase extension growth during secondary wall deposition (Wilkins<br>
and Jemstedt, "Molecular Genetics of Developing Cotton Fibers. In: A. S. Basra(ed.),<br>
Cotton Fibers: Developmental Biology Quality Improvement, and Textile Processing,<br>
The Haworth Press, New York, pp. 231-270 (1999); genes for protein or plastic polymers<br>
that might be retained in the fiber lumen or integrated into the cell wall to increase fiber<br>
strength or change its textile properties (John, "Genetic Engineering<br>
Strategies for Cotton Fiber Modification, "In : A. S. Basra(ed.), Cotton Fibers:<br>
Developmental Biology. Quality Improvement, and Textile Processing, The Haworth<br>
Press, New York, pp. 271-289 (1999); Guda et al., "Hyperexpression of an<br>
Environmentally Friendly Synthetic PolymerGene, "BiotechnologY Letters,<br>
17: 745-750 (1995)); genes for plant cell wall matrix biosynthetic enzymes or their<br>
regulatory proteins so that other carbohydrates could be integrated into the cell wall and<br>
change fiber properties (Haigler, "The Relationship Between Polymerization and<br>
Crystallization in CelluIoseBiogenesis,"in C. H. Haigler and P. Weimer, eds. ,<br>
Biosynthesis and Biodegradation of Cellulose, New York: Marcel Dekker, pp.<br>
99-124 (1991); Andrawis et al. ,"Cotton Fiber Annexins: A Potential Role in the<br>
Regulation of Callose Synthase, "Plant J., 3: 763-772 (1993); genes for molecules such as<br>
tannins, suberins, or dyes that might confer valuable color to fibers (Ryser, "Cotton Fiber<br>
Initiation and Histodifferentiation, "In : A. S. Basra(ed.), Cotton Fibers:Developmental<br>
Biology. Quality Improvement, and Textile Processing, The<br>
Haworth Press, New York, pp. 1-46 (1999); genes for molecules such as cutin, suberin,<br>
or wax that might change the absorptivity and strength of cotton fibers (May,"Genetic<br>
Variation in Fiber Quality, "In : A. S. Basra (ed. ), Cotton Fibers: Developmental<br>
Biology, Quality Improvement, and Textile Processing, The Haworth Press, New York,<br>
pp. 183-230 (1999); Ryser, "Cotton Fiber Initiation and Histodifferentiation, "In: A. S.<br>
Basra(ed.),	Cotton	Fibers:	Developmental	Biology.<br>
Quality Improvement, and Textile Processing, The Haworth Press, New York, pp. 1-46 (1999); and genes for signal transduction molecules such as Rac that may regulate shifts between fiber developmental stages (Delmer et al. ,"Genes Encoding Small GTP-Binding Proteins Analogous to Mammalian rac are Preferentially Expressed in Developing     Cotton     Fibers,     "Mol.     Gen.     Genet.,     248:     43-51     (1995).<br>
[58] Particularly preferred protein encoding regions are N-acetylglucosamine synthase coding regions as described e.g. in WO2006/136351 or sucrose synthase genes as described in WO2002/45485 or EP06015433.3 (herein incorporated by reference).<br>
[59] Biologically active RNA may also code for so-called antisense RNA, sense RNA, double stranded RNA as described in WO99/53053 or synthetic micro RNA molecules designed, according to rules well known in the art, to downregulate the expression of other genes comprised within the cell of the fiber-producing plants or even of genes comprised within a pathogen or pest feeding upon the fiber producing plant.<br>
[60] Also provided are methods to express a protein or biologically active RNA specifically in fiber cells of a fiber producing plant, comprising the step of introducing a DNA molecule comprising a fiber selective promoter or promoter region as herein described operably Unked to a transcribed DNA region encoding the biologically active RNA molecule, into the fiber cells of a fiber producing plants.<br>
[61] In the second aspect of the current invention, a method is provided for altering the fiber length of a fiber in a fiber producing plant by specifically altering the expression of one of the subgenomic alleles encoding a fiber selective P-l,3-endoglucanase. As indicated above, such alteration of the expression may prevent or enhance degradation of the callose plugging the plasmodesmata at the base of the fiber cell, and thus increase or decrease the rapid fiber elongation phase and the fiber length e.g. in cotton, particularly in Gossypium hirsutum. Modulation of the length of the fiber may also impact strength and other qualities or properties of the fiber. Accordingly, the invention is also directed at altering fiber properties or qualities of the fiber by the herein described methods.<br>
 [62] In one embodiment of the second aspect of the invention, a method is provided for increasing the length of a fiber in a cotton plant, particularly a Gossypium hirsutum plant, by introducing a chimeric gene into cells of said cotton plant whereby the chimeric gene comprises the following operably linked DNA elements:<br>
a.	a plant expressible promoter<br>
b.	a transcribed DNA region which when transcribed yields a biologically<br>
active RNA molecule capable of reducing the expression of reducing the<br>
expression of a fiber selective j3-l,3 endoglucanase comprised within the<br>
D genome and<br>
c.	a 3' end region comprising transcription termination and polyadenylation<br>
signals functioning in cells of said plant,<br>
[63] As has been uncovered by the current inventors, the increased expression of/S-1,3 endoglucanase in the fiber cells of a cotton plant previously reported to be correlated with the reopening of the plasmodesmata and marking the end of the rapid fiber elongation phase, is specifically due to the onset of the expression of the D subgenome specific allele at this time, while the A subgenome specific allele is only expressed at later stages in the fiber development, at least in Gossypium hirsutum. Accordingly, the methods as described in WO 2005/017157 can be further improved, at least for Gossypium hirsutum by specifically altering the expression pattern of a particular subgenomic allele encoding the fiber selective ß-1,3 endoglucanase.<br>
[64] As described herein, a D subgenome specific allele encoding a fiber selective j3-1,3 endoglucanase can be recognized in several manners. Isolation of the (genomic) nucleic acid encoding a fiber selective ß-l,3 endoglucanase from the D subgenome of different cotton plants, as well as from diploid cotton progenitor plants with a D like genome revealed that the intron in the A-subgenomic allele had an about 18 nt long insertion in the intron sequence (see figure 3) which is absent in the intron sequence of the D-subgenomic allele.<br>
 [65] Furthermore, comparison of the nucleotide sequences of the coding region (cDNA; figure 2) and encoded amino acid sequences (figure 4) revealed a number of characteristic differences between A and D-subgenomic alleles and encoded proteins, indicated in the figures by the gray boxes.<br>
[66] Very conveniently, the A and D subgenomic alleles can be distinguished by a polymorphism resulting in the presence of an Alwl restriction enzyme recognition site about 326 nucleotides downstream the translation intitation codon in the genomic clone of the A subgenomic allele, which is absent in D subgenomic alleles encoding the fiber selective j8-l,3 endoglucanase. One possible method to distinguish A and D subgenomic alleles, both on genomic DNA or cDNA level, is PCR amplification using cDNA or genomic DNA as template and the oligonucleotides having the nucleotide sequence of SEQ ID No 5 and 6 as primers, followed by a digestion of the amplified PCR product with Alwl restriction enzyme. The PCR generated product which is about 656 bp for both subgenomic alleles is cleaved into a 538 and 118 bp fitigment when the PCR product is generated firom a D-subgenomic allele template, while the PCR generated product amplified fi-om the A-subgenomic allele template is cleaved into three fitigments of 476 bp, 118bp and 59 bp respectively. The presence of the 118 bp serves as an internal control for proper fimctioning of the restriction enzyme reaction.<br>
[67] Several methods are available in the art to produce a silencing RNA molecule, i.e. an RNA molecule which when expressed reduces the expression of a particular gene or group of genes, including the so-called "sense" or "antisense" RNA technologies.<br>
[68] Thus in one embodiment, the inhibitory RNA molecule encoding chimeric gene is based on the so-called antisense technology. In other words, the coding region of the chimeric gene comprises a nucleotide sequence of at least 19 or 20 consecutive nucleotides of the complement of the nucleotide sequence of the fiber selective ^1,3 endoglucanase. Such a chimeric gene may be constructed by operably linking a DNA fragment comprising at least 19 or 20 nucleotides from fiber selective i3-l,3 endoglucanase encoding gene, isolated or identified as described elsewhere in this<br>
application, in inverse orientation to a plant expressible promoter and 3' end formation region involved in transcription termination and polyadenylation.<br>
[69] In another embodiment, the inhibitory RNA molecule encoding chimeric gene is based on the so-called co-suppression technology. In other words, the coding region of the chimeric gene comprises a nucleotide sequence of at least 19 or 20 consecutive nucleotides of the nucleotide sequence of the fiber selective ß-l,3 endoglucanase gene. Such a chimeric gene may be constructed by operably linking a DNA fragment comprising at least 19 or 20 nucleotides from the fiber selective ß-l,3 endoglucanase gene, in direct orientation to a plant expressible promoter and 3' end formation region involved in transcription termination and polyadenylation.<br>
[70] The efficiency of the above mentioned chimeric genes in reducing the expression of the fiber selective ^-1,3 endoglucanase gene may be fiirther enhanced by the inclusion of DNA element which result in the expression of aberrant, impolyadenylated inhibitory RNA molecules or results in the retention of the inhibitory RNA molecules in the nucleus of the cells. One such DNA element suitable for that purpose is a DNA region encoding a self-splicing ribozyme, as described in WO 00/01133 (incorporated by reference). Another such DNA element suitable for that purpose is a DNA region encoding an RNA nuclear localization or retention signal, as described in PCT/AU03/00292 published as WO03/076619 (incorporated by reference).<br>
[71] A convenient and very efficient way of downregulating the expression of a gene of interest uses so-called double-stranded RNA (dsRNA) or interfering RNA (RNAi), as described e.g. in WO99/53050 (incorporated by reference). In this technology, an RNA molecule is introduced into a plant cell, whereby the RNA molecule is capable of forming a double stranded RNA region over at least about 19 to about 21 nucleotides, and whereby one of the strands of this double stranded RNA region is about identical in nucleotide sequence to the target gene ("sense region"), whereas the other strand is about identical in nucleotide sequence to the complement of the target gene or of the sense region ("antisense region"). It is expected that for silencing of the target gene expression.<br>
the nucleotide sequence of the 19 consecutive nucleotide sequences may have one mismatch, or the sense and antisense region may differ in one nucleotide. To achieve the construction of such RNA molecules or the encoding chimeric genes, use can be made of the vector as described in WO 02/059294.<br>
[72] Thus, in one embodiment of the invention, a method for increasing the length of a fiber of a fiber producing plant, such as cotton, is provided comprising the step of introducing a chimeric gene into a cell of the fiber producing plant, wherein the chimeric gene comprises the following operably linked DNA elements:<br>
(a)	a plant expressible promoter, preferably a plant expressible promoter which controls transcription preferentially in the fiber cells;<br>
(b)	a transcribed DNA region, which when transcribed yields a double-stranded RNA molecule capable of reducing specifically the expression of a fiber selective j8-l,3 endoglucanase subgenomic allele, and the RNA molecule comprising a first and second RNA region wherein<br>
i)   the first RNA region  comprises  a nucleotide sequence of at least  19<br>
consecutive nucleotides having at least about 94% sequence identity to the<br>
nucleotide sequence of the subgenomic allele ; ii) the second RNA region comprises a nucleotide sequence complementary to<br>
the at least 19 consecutive nucleotides of the first RNA region; iii) the first and second RNA region are capable of base-pairing to form a double<br>
stranded RNA molecule between at least the 19 consecutive nucleotides of the<br>
first and second region; and<br>
(c)	a 3' end region comprising transcription tennination and polyadenylation signals<br>
fimctioning in cells of the plant.<br>
[73] The length of the first or second RNA region (sense or antisense region) may vary from about 19 nucleotides (nt) up to a length equaling the length (in nucleotides) of the endogenous gene involved in callose removal. The total length of the sense or antisense nucleotide sequence may thus be at least at least 25 nt, or at least about 50 nt, or at least about 100 nt, or at least about 150 nt, or at least about 200 nt, or at least about 500 nt. It is<br>
expected that there is no upper limit to the total length of the sense or the antisense nucleotide sequence. However for practical reasons (such as e.g. stability of the chimeric genes) it is expected that the length of the sense or antisense nucleotide sequence should not exceed 5000 nt, particularly should not exceed 2500 nt and could be limited to about 1000 nt.<br>
[74] It will be appreciated that the longer the total length of the sense or antisense region, the less stringent the requirements for sequence identity between these regions and the corresponding sequence in the fiber selective ß-l,3 endoglucanase gene or its complement. Preferably, the nucleic acid of interest should have a sequence identity of at least about 75% with the corresponding target sequence, particularly at least about 80 %, more particularly at least about 85%, quite particularly about 90%, especially about 95%, more especially about 100%, quite especially be identical to the corresponding part of the target sequence or its complement. However, it is preferred that the nucleic acid of interest always includes a sequence of about 19 consecutive nucleotides, particularly about 25 nt, more particularly about 50 nt, especially about 100 nt, quite especially about 150 nt with 100% sequence identity to the corresponding part of the target nucleic acid. Preferably, for calculating the sequence identity and designing the corresponding sense or antisense sequence, the number of gaps should be minimized, particularly for the shorter sense sequences.<br>
[75] dsRNA encoding chimeric genes according to the invention may comprise an intron, such as a heterologous intron, located e.g. in the spacer sequence between the sense and antisense RNA regions in accordance with the disclosure of WO 99/53050 (incorporated herein by reference).<br>
[76] It is preferred for the current invention that the target specific gene sequence included in the antisense, sense or double stranded RNA molecule comprises at least one nucleotide, and preferably more which are specific for the subgenomic allele whose expression is to be downregulated. Such specific nucleotides are indicated at least in figure 2 by the gray boxes.<br>
 [77] In a preferred embodiment, the biologically active RNA is specifically adapted to downregulate the D-subgenomic allele of the fiber selective ß-l,3 endoglucanase encoding gene (GhGlucl). In another preferred embodiment, the biologically active RNA is specifically adapted to downregulate the A-subgenomic allele of the fiber selective /?-1,3 endoglucanase encoding gene (GhGlucl).<br>
[78] The use of synthetic micro-RNAs to downregulate expression of a particular gene in a plant cell, provides for very high sequence specificity of the target gene, and thus allows conveniently to discriminate between closely related alleles as target genes the expression of which is to be downregulated.<br>
[79] Thus, in another embodiment of the invention, the biologically active RNA or silencing RNA or inhibitory RNA molecule may be a microRNA molecule, designed, synthesized and/or modulated to target and cause the cleavage of specific subgenomic alleles, preferably the D-subgenomic allele of fiber selective j8-l,3 endoglucanase encoding gene in a fiber producing plants, such as a cotton plant. Various methods have been described to generate and use miRNAs for a specific target gene (including but not limited to Schwab et al. (2006, Plant Cell, 18(5):1121-1133), WO2006/044322, WO2005/047505, EP 06009836, incorportated by reference). Usually, an existing miRNA scaffold is modified in the target gene recognizing portion so that the generated miRNA now guides the RISC complex to cleave the RNA molecules transcribed fi"om the target nucleic acid. miRNA scaffolds could be modified or synthesized such that the miRNA now comprises 21 consecutive nucleotides of one of the subgenomic alleles of the fiber selective j3-l,3 endoglucanase encoding nucleotide sequence, such as the sequences represented in the Sequence listing, and allowing mismatches according to the herein below described rules.<br>
 [80] Thus, in one embodiment, the invention provides a method for downregulating the expression of a or increasing the resistance of plants to adverse growing conditions, comprising the steps of<br>
a.   Introducing a chimeric gene into cells of said plants, said chimeric gene comprising the following operably linked DNA regions: i.   A plant expressible promoter;<br>
ii. A DNA region which upon introduction and transcription in a plant cell is processed into a miRNA, whereby the miRNA is capable of recognizing and guiding the cleavage of the mRNA of one subgenomic allele of a fiber selective )8-l,3 endoglucanase encoding gene of the plant but not the other subgenomic allele; and iii. optionally, a 3' DNA region involved in transcription termination and polyadenylation.<br>
[81] The mentioned DNA region processed into a miRNA may comprise a nucleotide sequence which is essentially complementary to a nucleotide sequence of at least 21 consecutive nucleotides of a fiber selective j8-l,3 endoglucanase encoding gene, provided that one or more of following mismatches are allowed:<br>
a.	A mismatch between the nucleotide at the 5' end of the miRNA and the<br>
corresponding nucleotide sequence in the RNA molecule;<br>
b.	A mismatch between any one of the nucleotides in position 1 to position 9<br>
of the miRNA and the corresponding nucleotide sequence in the RNA<br>
molecule;<br>
c.	Three mismatches between any one of the nucleotides in position 12 to<br>
position 21 of the miRNA and the corresponding nucleotide sequence in<br>
the RNA molecule provided that there are no more than two consecutive<br>
mismatches.<br>
[82] As used herein, a "miRNA" is an RNA molecule of about 20 to 22 nucleotides in length which can be loaded into a RISC complex and direct the cleavage of another RNA molecule, wherein the other RNA molecule comprises a nucleotide sequence essentially<br>
complementary to the nucleotide sequence of the miRNA molecule whereby one or more of the following mismatches may occur:<br>
a.	A mismatch between the nucleotide at the 5' end of said miRNA and the<br>
corresponding nucleotide sequence in the target RNA molecule;<br>
b.	A mismatch between any one of the nucleotides in position 1 to position 9<br>
of said miRNA and the corresponding nucleotide sequence in the target<br>
RNA molecule;<br>
c.	Three mismatches between any one of the nucleotides in position 12 to<br>
position 21 of said miRNA and the corresponding nucleotide sequence in<br>
the target RNA molecule provided that there are no more than two<br>
consecutive mismatches.<br>
d.	No mismatch is allowed at positions 10 and 11 of the miRNA (all miRNA<br>
positions are indicated starting from the 5' end of the miRNA molecule).<br>
[83] A miRNA is processed from a "pre-miRNA" molecule by proteins, such as DCL proteins, present in any plant cell and loaded onto a RISC complex where it can guide the cleavage of the target RNA molecules.<br>
[84] As used herein, a "pre-miRNA" molecule is an RNA molecule of about 100 to about 200 nucleotides, preferably about 100 to about 130 nucleotides which can adopt a secondary structure comprising a double stranded RNA stem and a single stranded RNA loop and further comprising the nucleotide sequence of the miRNA (and its complement sequence) in the double stranded RNA stem. Preferably, the miRNA and its complement are located about 10 to about 20 nucleotides from the free ends of the miRNA double stranded RNA stem. The length and sequence of the single stranded loop re^on are not critical and may vary considerably, e.g. between 30 and 50 nt in length. An example of a synthetic pre-miRNA is represented in figure 1. Preferably, the difference in free energy between unpaired and paired RNA structure is between -20 and -60 kcal/mole, particularly around -40 kcal/mole. The complementarity between the miRNA and the miRNA* need not be perfect and about 1 to 3 bulges of unpaired nucleotides can be tolerated.The secondary structure adopted by an RNA molecule can be predicted by computer algorithms conventional in the art such as mFOLD. The particular strand of the<br>
double stranded RNA stem from the pre-miRNA which is released by DCL activity and loaded onto the RISC complex is determined by the degree of complementarity at the 5' end, whereby the strand which at its 5' end is the least involved in hydrogen bounding between the nucleotides of the different strands of the cleaved dsRNA stem is loaded onto the RISC complex and will determine the sequence specificity of the target RNA molecule degradation. However, if empirically the miRNA molecule from a particular synthetic pre-miRNA molecule is not functional (because the "wrong" strand is loaded on the RISC complex, it will be immediately evident that this problem can be solved by exchanging the position of the miRNA molecule and its complement on the respective strands of the dsRNA stem of the pre-miRNA molecule. As is known in the art, binding between A and U involving two hydrogen bounds, or G and U involving two hydrogen bounds is less strong that between G and C involving three hydrogen bounds.<br>
[85] Naturally occurring miRNA molecules may be comprised within their naturally occurring pre-miRNA molecules but they can also be introduced into existing pre-miRNA molecule scaffolds by exchanging the nucleotide sequence of the miRNA molecule normally processed from such existing pre-miRNA molecule for the nucleotide sequence of another miRNA of interest. The scaffold of the pre-miRNA can also be completely synthetic. Likewise, synthetic miRNA molecules may be comprised within, and processed from, existing pre-miRNA molecule scaffolds or synthetic pre-miRNA scaffolds.<br>
[86] The pre-miRNA molecules (and consequently also the miRNA molecules) can be conveniently introduced into a plant cell by providing the plant cells with a gene comprising a plant-expressible promoter operably linked to a DNA region, which when transcribed yields the pre-miRNA molecule. The plant expressible promoter may be the promoter naturally associated with the pre-miRNA molecule or it may be a heterologous promoter.<br>
 [87] Suitable miRNA and pre microRNA molecules for the specific downregulation of the expression of the D-subgenomic allele of GhGlucl gene are set forth in the sequence listing entries SEQ ID No 15,16,20 and 21.<br>
[88] Suitable miRNA and pre microRNA molecules for the specific downregulation of the expression of the A-subgenomic allele of GhGlucl gene are set forth in the sequence listing entries SEQ ID No 13, 14, 17, 18 and 19.<br>
[89] As used herein, the term "plant-expressible promoter" means a DNA sequence which is capable of controlling (initiating) transcription in a plant cell. This includes any promoter of plant origin, but also any promoter of non-plant origin which is capable of directing transcription in a plant cell, i.e., certain promoters of viral or bacterial origin such as the CaMV35S, the subterranean clover virus promoter No 4 or No 7, or T-DNA gene promoters and the like.<br>
[90] A plant-expressible promoter that controls initiation and maintenance of transcription preferentially in fiber cells is a promoter that drives transcription of the operably linked DNA region to a higher level in fiber cells and the underlying epidermis cells than in other cells or tissues of the plant. Such promoters include the promoter from cotton firom a fiber-specific p- tubulin gene (as described in WO0210377), the promoter fi-om cotton fix&gt;m a fiber-specific actin gene(as described in WO0210413), the promoter fi-om a fiber specific lipid transfer protein gene fi-om cotton (as described in US5792933), a promoter from an expansin gene fi-om cotton (WO9830698) or a promoter fi-om a chitinase gene in cotton (US2003106097) or the promoters of the fiber specific genes described in US6259003 or US6166294, Fiber selective promoters as described herein may also be used.<br>
[91] For some applications, it may be beneficial to decrease the fiber length in a fiber producing plant, such as a cotton plant. E.g. it would be beneficial to decrease the amount of fiizz fiber in a cotton plant, thus leaving more energy and material for incorporation into   the   lint   fiber   in   cotton   plants.   This   may   be   conveniently   achieved   by<br>
 (over)expressing a )3-l,3-endoglucanase in fiber cells at an early stage in the development of a fiber cell e.g. using a fiber selective promoter as described herein which is expressed at the end of the rapid elongation phase.<br>
[92] The invention also encompasses the chimeric genes herein described, as well as plants, seeds, tissues comprising these chimeric genes, and fibers produced fi"om such plants<br>
[93] Fiber length or other fiber properties and fiber quality in cotton may also be increased or altered by isolating D-subgenome allelic variants which no longer encode a fimctional ß-1,3-endoglucanase or which encode a variant protein with weak enzymatic activity.<br>
[94] Methods to transform plants are well known in the art and are of minor relevance for the current invention. Methods to transform cotton plants are also well known in the art. Agrobacterium-mediated transfonnation of cotton has been described e.g. in US patent 5,004,863 or in US patent 6,483,013 and cotton transformation by particle bombardment is reported e.g. in WO 92/15675.<br>
[95] The chimeric genes according to the invention may be introduced into plants in a stable manner or in a transient manner using methods well known in the art. The chimeric genes may be introduced into plants, or may be generated inside the plant cell as described e.g. in EP 1339859.<br>
[96] The chimeric genes may be introduced by transformation in cotton plants fi-om which embryogenic callus can be derived, such as Coker 312, Coker310, Coker 5Acala SJ-5, GSC25110, FIBERMAX 819 , Siokra 1-3, T25. GSA75, Acala SJ2, Acala SJ4, Acala SJ5, Acala SJ-Cl, Acala B1644, Acala B1654-26, Acala B1654-43, Acala B3991, Acala GC356, Acala GC510, Acala GAMl, Acala CI, Acala Royale, Acala Maxxa, Acala Prema, Acala B638, Acala B1810, Acala B2724, Acala B4894, Acala B5002, non Acala "picker" Siokra, "stripper" variety FC2017, Coker 315, STONEVILLE 506,<br>
STONEVILLE 825, DP50, DP61, DP90, DP77, DES119, McN235, HBX87, HBX19I, HBX107, FC 3027, CHEMBRED Al, CHEMBRED A2, CHEMBRED A3, CHEMBRED A4, CHEMBRED Bl, CHEMBRED B2, CHEMBRED B3, CHEMBRED CI, CHEMBRED C2, CHEMBRED C3, CHEMBRED C4, PAYMASTER 145, HS26, HS46, SICALA, PIMA S6 ORO BLANCO PIMA, FIBERMAX FM5013, FIBERMAX FM5015, FIBERMAX FM5017, FIBERMAX FM989, FIBERMAX FM832, FIBERMAX FM966, FIBERMAX FM958, FIBERMAX FM989, FIBERMAX FM958, FBERMAX FM832, FIBERMAX FM991, FIBERMAX FM819, FIBERMAX FM800, FIBERMAX FM960, FIBERMAX FM966, FIBERMAX FM981, FIBERMAX FM5035, FIBERMAX FM5044, FIBERMAX FM5045, FIBERMAX FM5013, FIBERMAX FM5015, FIBERMAX FM5017 or FIBERMAX FM5024 and plants with genotypes derived thereof.<br>
[97] "Cotton" as used herein includes Gossypium hirsutum or Gossypium barbadense. "Cotton progenitor plants" include Gossypium arboretum, Gossypium herbaceum and Gossypium raimondii and Gossypium longicalyx.<br>
[98] The methods and means of the current invention may also be employed for other plant species such as hemp, jute, flax and woody plants, including but not limited to Pinus spp., Populus spp., Picea spp.. Eucalyptus spp. etc.<br>
[99] The obtained transformed plant can be used in a conventional breeding scheme to produce more transformed plants with the same characteristics or to introduce the chimeric gene according to the invention in other varieties of the same or related plant species, or in hybrid plants. Seeds obtained from the transformed plants contain the chimeric genes of the invention as a stable genomic insert and are also encompassed by the invention.<br>
[100] In another embodiment of the invention, antibodies raised against the ß-l,3-endoglucanase are provided, particularly antibodies recognizing the ß-1,3-endoglucanase proteins having the amino acid sequences of SEQ ID No.s 11 and 12.<br>
 [101] As used herein "comprising" is to be interpreted as specifying the presence of the stated features, integers, steps or components as referred to, but does not preclude the presence or addition of one or more features, integers, steps or components, or groups thereof Thus, e.g., a nucleic acid or protein comprising a sequence of nucleotides or amino acids, may comprise more nucleotides or amino acids than the actually cited ones, i.e., be embedded in a larger nucleic acid or protein. A chimeric gene comprising a DNA region, which is functionally or structurally defined, may comprise additional DNA regions etc.<br>
[102] The following non-limiting Examples describe the identification of two i8-l,3-endoglucanases of the A and D subtype in cotton as well as their promoter regions, and analysis of the timing of expression and involvement in secondary plant cell wall synthesis. Also described are chimeric genes for the alteration of fiber characteristics, for fiber-preferential and for fiber-specific expression in fiber-producing plants such as cotton and uses thereof Unless stated otherwise in the Examples, all recombinant DNA techniques are carried out according to standard protocols as described in Sambrook et al. (1989) Molecular Cloning: A Laboratory Manual, Second Edition, Cold Spring Harbor Laboratory Press, NY and in Volumes 1 and 2 of Ausubel et al. (1994) Current Protocols in Molecular Biology, Cvurent Protocols, USA. Standard materials and methods for plant molecular work are described in Plant Molecular Biology Labfax (1993) by R.D.D. Croy, jointly published by BIOS Scientific Publications Ltd (UK) and Blackwell Scientific Publications, UK.<br>
[103] Throughout the description and Examples, reference is made to the following sequences represented in the sequence listing:<br>
SEQ ID No 1: amplified genomic fi-agment of ß-1,3-endoglucanase from Gossypium<br>
hirsutum Fiber Max966, subtype A SEQ ID No 2: amplified genomic fragment of ß-1,3-endoglucanase from Gossypium<br>
hirsutum Fiber Max966, subtype D<br>
SEQ ID No 3: amplified genomic fhigment of ß-I,3-endoglucanase from Gossypium<br>
arboretum SEQ ID No 4: amplified genomic fragment ofß-1,3-endoglucanase firom Gossypium<br>
raimondii SEQ ID No 5: reverse primer (SE002) for amplification ofß-1,3-endoglucanase genomic<br>
fi^gment SEQ ID No 6: forward primer (SE003) for amplification of j8-l,3-endoglucanase genomic<br>
fi'agment SEQ ID No 7: amino acid sequence encoded by the amplified genomic fragment of/5-l,3-<br>
endoglucanase firom Gossypium hirsutum Fiber Max966, subtype A SEQ ID No 8: amino acid sequence encoded by the amplified genomic Augment ofß-1,3-endoglucanase firom Gossypium hirsutum Fiber Max966, subtype D SEQ ID No  9:  nucleotide  sequence  of the complete genomic clone  of -l,3-<br>
endoglucanase fi-om Gossypium hirsutum Fiber Max966, subtype A,<br>
including promoter sequence SEQ ID No  10: nucleotide sequence of the complete genomic clone of -1,3-<br>
endoglucanase from  Gossypium hirsutum Fiber Max966, subtype D,<br>
including promoter sequence SEQ ID No 11: amino acid sequence of complete j3-l,3-endoglucanase firom Gossypium<br>
hirsutum Fiber Max966, subtype A SEQ ID No 12; amino acid sequence of complete |3-l,3-endoglucanase from Gossypium<br>
hirsutum Fiber Max966, subtype D SEQ ID No 13: nucleotide sequence of pre-miRAl SEQ ID No 14: nucleotide sequence of pre-miRAl with cloning sites SEQ ID No 15: nucleotide sequence of pre-miRDl SEQ ID No 16: nucleotide sequence of pre-miRDl with cloning sites SEQ ED No 17: nucleotide sequence of pre-miRA2 SEQ ID No 18: nucleotide sequence of pre-miRA2aa SEQ ID No 19: nucleotide sequence of pre-miRA2aa with cloning sites SEQ ID No 20: nucleotide sequence of pre-miRD2 SEQ E) No 21: nucleotide sequence of pre-miRD2 with cloning sites<br>
Example 1: Identification of A and D-subgenome specific alleles encoding a fiber selective -l,3-glucanase in cotton plants and cotton progenitor plants.<br>
[104] It has been proposed that callose is involved in the process of maintaining turgor in growing fiber cells. The removal of callose at the base of bridging structures between fiber and non-fiber cotton cells, knovwi as plasmadesmata, dissipates the turgor pressure. This results in the termination of rapid fiber elongation. Therefore by delaying the removal of callose fiber length should be enhanced.<br>
[105] The ß-1,3-glucanase enzyme catalyzes the hydrolysis of -l,3-D-glucosidic linkages in ß -1,3-D-glucan (callose). A fiber specific  -1,3-glucanase gene encoded by GhGlucl, was undetectable when callose was deposited at the fiber base but became evident at the time of callose degradation.<br>
[106] In short, plasmodesmata closure appears to play an important role in elongating cotton fibers. Callose deposition and degradation should be involved in the plasmodesmata closure and reopening, respectively. The expression of GhGlucl could play a role in this process by degrading callose, thus opening the plasmodesmata. (Ruan et al., 2004, Plant Physiology - 136: 4104-4113).<br>
[107] Based on the GhGlucl nucleotide sequence (EMBL accession number D88416) described in Ruan et al, 2004, Plant Physiology - 136: 4104-4113, 2 primers (SE002: ggccgaagccgatcttatctagg (reverse primer; SEQ ID No 5) en SE003: cggcaacaatcttccatctccag (forward primer, SEQ ID No 6)) were designed to amplify genomic DNA jBragments for G. hirsutum (AD genome), G. arboreum (A genome) and G. raimondii (D genome). These fi-agments have been sequenced (see SEQ ID No 1-4). For G. hirsutum 2 consensus sequences were derived, for G. arboreum I consensus sequence was derived and for G. raimondii 1 consensus sequence was derived.<br>
(table removed)<br>
[109] Accordingly, one type of the sequence corresponds to the A genome sequence, and the other type corresponds to the D genome.<br>
[110] Using an Alwl (recognition site = GGATC) digest on the amplified PCR fragment, both subgenomic variants within the hirsutum or genome can be distinguished (see figure 5.) Thus the following protocol can be used to distinguish A and D subgenome alleles of GhGlucl. The primers used are:<br>
a.	SE002: GGCCGAAGCCGATCTTATCTAGG (SEQ ID No 5)<br>
b.	SE003: CGGCAACAATCTTCCATCTCCAG (SEQ ID No 6)<br>
The expected length PCR product is 655 bp.<br>
PCR conditions:<br>
(table removed)<br>
 [111]   After PCR ampUfication, the PCR fragment is digested with Alwl digest (3h incubation @ 37°C) using 10 ^1 template; 1 jil Alwl enzyme; 2 jil NEB 4 restriction<br>
buffer; 7 nl MQ water. The resulting fragments are analysed on 1.5% TAE gel stained with EtBr.<br>
[112] The expected band sizes for the A subgenome allele specific PCR fragment are : 479 + 118 + 59 bp. The expected band sizes for the D subgenorae allele specific PCR fragment are: 538 + 118 bp.<br>
Example 2: Differential expression of A and D-subgenome specific alleles encoding a fiber selective ß-13-glucanase and correlation with fiber development stages in Gossypium hirsutum.<br>
[113] Allelic specific expression across fiber growth/developmental profile was analyzed for G. hirsutum. DNA fi-om the cDNA libraries created firom fiber cells and seed at 5DPA and from fiber cells at 10, 15, 20 and 40 DPA was extracted, the concentration was equalized and the above mentioned PCR amplification was performed. Differences in band intensities correspond to relative differences in expression (Figiu-e 6, lanes 2, 4, 6, 8 and 10. A positive control (genomic DNA) was included (Figure 6, lane 12). An Alwl digest was performed as described in Example 1 to distinguish between the 2 subgenomic alleles of the GhGlucl genes in (Figure 6, lanes 3, 5, 7, 9,11 and 13).<br>
[114]   The expression profile can be summarized as follows:<br>
<br>
(table removed)<br>
 [115] During the rapid elongation phase in fiber development (5 and 10 dpa), there is no expression of GhGlucl. Therefore, the callose plug is not degraded; the turgor in the cell is not released and elongation continues. We believe that the faint band at 5 dpa indicates some expression in the seed (not in the fiber). At the transition between the elongation phase and secondary cell wall formation phase (15 DPA), GhGlucl is expressed. This releases the turgor in the cell in rapid elongation stops. Only the D genome like variant of the GhGlucl gene in is expressed. At 20 DPA there is more expression then at 15 DPA (more intense band).   The A genome like variant is not expressed at 15 and 20<br>
DPA. The A genome like variant, as well as the D genome like variant, is expressed at 30 and 40 DPA and might play a role in maturation or other fiber properties.<br>
Example 3: Isolation and identification of promoter region of A and D-subgenome specific alleles encoding a fiber selective /^1,3-glucanase<br>
[116] PCR firagment comprising the nucleotide sequence of the A and D subgenomic specific alleles of GhGlucl were used to screen a BAG library containing genomic DNA clones of Gossypium hirsutum variety. 4 different clones were identified, 2 for each subgenomic variant. The nucleotide sequence of genomic fragments for each of the allelic variants were identified and are represented in SEQ ID no 9 (A genome) and SEQ IDNo 10 (D genome).<br>
[117] For the A-genome variant, a TATA box could be identified at positions 2278 to 2281; a transcription initiation site at position 2308. The 5' untranslated leader extends fi'om nucleotide 2308 to 2409; the translation initation codon is located at positions 2410 to 2412. The coding sequence consist of two exons (nt 2410 to 2443 and nt 2556 to 3499) separated by an intron sequence (2444 to 2555). The translation stop codon is located at position 3497 to 3499 and the polyadenylation site is located at position 3624.<br>
[118] For the D-genome variant, a TATA box could be identified at positions 3242 to 3245; a transcription initiation site at position 3270. The 5' untranslated leader extends from nucleotide 3270 to 3372; the translation initation codon is located at positions 3373 to 3375. The coding sequence consist of two exons (nt 3373 to 3406 and nt 3501 to 4444) separated by an intron sequence ( nt 3407 to 3500). The translation stop codon is located at position 4442 to 4444 and the polyadenylation site is located at position 4566.<br>
[119] The nucleotide sequence of the fused coding regions has been aligned (Figure 2), and differences are indicated; similarly the nucleotide sequences of the introns have been<br>
aligned (Figure 3). Figure 4 shows an alignment of the encoded proteins, while figure 1 shows an alignment of the nucleotide sequences located upstream of the coding region.<br>
Example 4: Chimeric gene constructs comprising different fiber selective promoter regions operably linked to a marker gene.<br>
[120]   The following DNA fragments were operably linked using standard recombinant techniques:<br>
a.	The nucleotide sequence of the A-allele specific promoter comprising the<br>
nucleotide sequence of SEQ ID No 9 from nucleotide 465 to nucleotide<br>
2409.<br>
b.	A ß-glucuronidase coding region (GUS)<br>
c.	A fragment comprising a transcription termination and polyadenylation<br>
signal (CaMV) 3' 35S (indicated hereinafter as Glucl-SGA (A1.9))<br>
and<br>
d.	The nucleotide sequence of the A-allele specific promoter comprising the<br>
nucleotide sequence of SEQ ED No 9 from nucleotide 1374 to nucleotide<br>
2409.<br>
e.	A ß-glucuronidase coding region (GUS)<br>
f.	A fragment comprising a transcription termination and polyadenylation<br>
signal (CaMV) 3'35S<br>
and<br>
g.	The nucleotide sequence of the A-allele specific promoter comprising the<br>
nucleotide sequence of SEQ ED No 9 from nucleotide 1531 to nucleotide<br>
2409.<br>
h.   A /S-glucuronidase coding region (GUS)<br>
i.    A fragment comprising a transcription termination and polyadenylation<br>
signal (CaMV) 3' 35S and<br>
j.   The nucleotide sequence of the D-allele specific promoter comprising the<br>
nucleotide sequence of SEQ ID No 10 fi"om nucleotide 1397 to nucleotide<br>
3372. (indicated hereinafter as Glucl-SGD (D2.0)) k.  A ß-glucuronidase coding region (GUS) 1.   A firagment comprising a transcription termination and polyadenylation<br>
signal (CaMV) 3' 35S And m. The nucleotide sequence of the D-allele specific promoter comprising the<br>
nucleotide sequence of SEQ ID No 10 from nucleotide 2371 to nucleotide<br>
3372. n.  A ß-glucuronidase coding region (GUS)<br>
A fi^gment comprising a transcription termination and polyadenylation signal (CaMV) 3' 35S Or o.   The nucleotide sequence of the D-allele specific promoter comprising the<br>
nucleotide sequence of SEQ ID No 10 from nucleotide 2718 to nucleotide<br>
3372. p.  A j8-glucuronidase coding region (GUS) q.   A fi-agment comprising a transcription termination and polyadenylation<br>
signal (CaMV) 3' 35S<br>
[121] The above chimeric genes were separately cloned into a T-DNA vector, in the presence of a chimeric selectable marker gene, and introduced into Agrobacterium tumefaciens strain comprising a disarmed helper Ti-plasmid. The Agrobacterium strains were used to generate transgenic cotton plants according to the method described in WOOO/71733. Fiber cells and other tissue from these plants are histochemically stained.<br>
[122] Expression of the different chimeric constructs is predominantly observed in fiber cells and fibers.<br>
Example 5: Chimeric gene constructs comprising different fiber selective promoter regions operably linked to N-acetylglucosamine transferase coding region.<br>
[123] Constructs similar to the constructs of Example 4, but wherein the Gus coding region has been exchanged for a NodC coding region (see WO2006/136531, specifically SEQ ID 1 to 9 therein, incorporated by reference) preceded by a Cab22 untranslated leader sequence, were assembled using current recombinant DNA techniques.<br>
[124] These chimeric genes are separately cloned into a T-DNA vector, in the presence of a chimeric selectable marker gene, and introduced into Agrobacterium tumefaciens strain comprising a disarmed helper Ti-plasmid. The Agrobacterium strains are used to generate transgenic cotton plants according to the method described in WOOO/71733. Fiber cells and other tissue firom these plants are analyzed for the presence of positively charged oligosaccharides as described in WO2006/136531).<br>
Example 6: Increasing fiber length in Gossypium hirsutum by downregulating specifically the D-subgenome specific alleles encoding a fiber selective -glucanase<br>
[125] Synthetic pre-raicroRNAs specific for either the A or the D subgenomic allelic variant of GhGlucl have been designed using design rules available in the art.<br>
[126] Suitable miRNA and pre microRNA molecules for the specific doAvnregulation of the expression of the D-subgenomic allele of GhGlucl gene are set forth in the sequence listing entries SEQ m No 15, 16, 20 and 21.<br>
[127] Suitable miRNA and pre microRNA molecules for the specific downregulation of the expression of the A-subgenomic allele of GhGlucl gene are set forth in the sequence listing entries SEQ ID No 13, 14, 17, 18 and 19.<br>
 [128] The premicroRNA encoding nucleotide sequences are cloned under control of a CaMV 35 S promoter or under control of a fiber-specific and/or fiber-preferential promoter as described in Example 4.<br>
[129] These chimeric genes are separately cloned into a T-DNA vector, in the presence of a chimeric selectable marker gene, and introduced into Agrobactenum tumefaciens strain comprising a disarmed helper Ti-plasmid. The Agrobactenum strains are used to generate transgenic cotton plants according to the method described in WOOO/71733. Fibers fi-om such plants or their progeny are analyzed for increased fiber length.<br>
Example 7: Analysis of the specificity of glucl promoters for fibers<br>
[130] RNA was isolated firom cotton leaves, roots and stems and analyzed for the expression of glucl genes and for the protein phosphatase 2A gene (a ubiquitous and conserved Serine/Threonine phosphatase with broad substrate specificity and diverse cellular functions). First strand cDNA synthesis was performed using the Superscript First Strand Synthesis system for RT-PCR (Invitrogen), after which the glucl or pp2A specific primers were added and the RT-PCR reaction performed. The results are visuaUzed in Figure 7. The positive result in lane 17 indicated that first strand synthesis was successfully achieved. Positive results in lanes 10, 12, 14 using pp2A specific primers in leaf, root and stem RNA samples specifically indicated that first strand synthesis in leaf, root and stem sample worked. Lanes 11, 13, 15 are negative controls omitting the reverse transcriptase from the reaction. Lane 8 is a positive control consisting of genomic DNA to verify the correctness of the glucl specific PCR reaction. ]n lanes 2, 4, 6 no signal can be detected indicating the absence of Glucl expression in cotton leaves, roots and stems. Lanes 3, 5 and 7 are negative controls omitting the reverse transcriptase.<br>
[131] In conclusion, no Glucl expression could be detected in cotton leaf, root and stem tissue.<br>
Example 8: Cotton ovule transformation with Agrobacterium strains containing the<br>
[132] Ovules of cotton flowers were isolated at ODPA, IDPA and 2DPA and used for ovule culture experiments (as described e.g by Feng and Brown, 2000, In vitro Cellular &amp; Developmental Biology, Volume 36, Number 4 pages 293-299) after transformation with Agrobacterium tumefaciens carrying the different chimeric glucl promoter::GUS fusion genes indicated in Example 4 as Glucl-SGA(A1.9) and Glucl-SGD (D2.0). The initiating fibers cells/fibers were analyzed histochemically for GUS expression after 1 week, 2 weeks, 3 weeks, 4 weeks or 5 weeks. As a positive control, cultured ovules were included which had been infected with an Agrobacterium strain harboring a chimeric CaMV35 promoter:: Gus fusion gene. Note that the GUS coding region used contained an intron sequence to avoid accidental GUS expression in the infecting Agrobacteria. The results are summarized in Table 1.<br>
[133]    Table 1: Gus histochemical analysis of the cultured and transfected cotton ovules<br>
<br>
(table removed)<br>
These data corroborate the expression profiles deduced for the promoters from the different subgenomic alleles for GhGlucl from the data in Example 2.<br>
<br>
<br>
<br>
<br>
<br>
Claims:<br>
1)	A fiber cell preferential promoter comprising a nucleotide sequence selected from the<br>
following group of nucleotide sequences:<br>
a)	a nucleotide sequence comprising the nucleotide sequence of SEQ ID No 9 from the nucleotide at position 2149 to the nucleotide at position 2307;<br>
b)	a nucleotide sequence comprising the nucleotide sequence of SEQ ID No 10 from the nucleotide at position 3109 to the nucleotide at position 3269;<br>
c)	a nucleotide sequence comprising the nucleotide sequence having at least 70% preferably at least 80 %, more preferably at least 90%, particularly at least 95% sequence identity with or, more particularly is identical to any of said nucleotide sequence mentioned imder a) or b) or<br>
d)	a nucleotide sequence comprising the nucleotide sequence of an about 150 bp to an about 1000 bp DNA fragment hybridizing under stringent conditions with a DNA fragment having said nucleotide sequence mentioned under a), b) or c).<br>
<br>
2)	The fiber cell preferential promoter according to claim 1 comprising the nucleotide sequence of SEQ ID No 9 from position 465 to position 2307.<br>
3)	The fiber cell preferential promoter according to claim 1 comprising the nucleotide sequence of SEQ ID No 9 from position 1374 to position 2307.<br>
4)	The fiber cell preferential promoter according to claim 1 comprising the nucleotide sequence of SEQ ID No 9 from position 1531 to position 2307.<br>
5)	The fiber cell preferential promoter according to claim 1 comprising the nucleotide sequence of SEQ ID No 10 from position 1397 to position 3269.<br>
6)	The fiber cell preferential promoter according to claim 1 comprising the nucleotide sequence of SEQ ID No 10 from position 2371 to position 3269.<br>
7)	The fiber cell preferential promoter according to claim 1 comprising the nucleotide sequence of SEQ ED No 10 from position 2718 to position 3269.<br>
8)	A fiber cell preferential promoter region comprising a fiber cell preferential promoter according to any one of claims 1 to 7.<br>
9)	The fiber cell preferential promoter region according to claim 8, fiirther comprising the nucleotide sequence of SEQ ID 9 from the nucleotide at position 2308 to the nucleotide at position 2409.<br>
10)	The fiber cell preferential promoter region according to claim 8, fiirther comprising the nucleotide sequence of SEQ ID 10 from the nucleotide at position 3270 to the nucleotide at position 3372.<br>
11)	A chimeric gene comprising the following operably linked DNA regions<br>
<br>
a)	a fiber cell specific promoter according to any one of claims 1 to 7 ;<br>
b)	a heterologous DNA region encoding a biologically active RNA of interest; and<br>
c)	a transcription termination and polyadenylation signal.<br>
<br>
12)	The chimeric gene according to claim 11, wherein said biologically active RNA encodes a protein of interest.<br>
13)	The chimeric gene according to claim 12, wherein said heterologous DNA region encodes a molecule selected from a N-acetylglucosamine transferase, preferably an N-acetylglucosamine transferase of the NodC type, a cellulose synthase, a sucrose synthase, preferably a sucrose synthase of the C-type; a sucrose phosphate synthase; a ß-l,3-endoglucanase, xylan synthase (cslF) xyloglucan synthase (cslD) and 1,3-1,4 glucan synthase (cslC).<br>
14)	The chimeric gene according to claim 11, wherein said biologically active RNA is a ribozyme, microRNA, double stranded hairpin RNA .<br>
15)	The chimeric gene according to claim 12, wherein said biologically active RNA<br>
downregulates the expression of an endogenous cotton gene, such as /3-l,3-<br>
endoglucanase.<br>
16)	A plant cell comprising a chimeric gene according to any one of claims 11 to 15.<br>
17)	A plant comprising in its cells a chimeric gene according to any of claims 11 to 15.<br>
18)	The plant according to claim 17, which is a cotton plant.<br>
19)	A seed of a plant comprising in its cells a chimeric gene according to any one of claims 11 to 15.<br>
20)	A method for expressing a biologically active RNA preferentially in a fiber cell of a<br>
fiber producing plant, such as a cotton plant, said method comprising<br>
a)	providing the cells of said plants with a chimeric gene according to any one of claims 11 to 15; and<br>
b)	growing said plants.<br>
21)	The method according to claim 20, wherein said plant is a cotton plant.<br>
 ..22) An isolated DNA molecule comprising a nucleotide sequence encoding a protein comprising an amino acid sequence having at least 95% sequence identity, preferably<br>
is identical, to the amino acid sequence of SEQ ID No 11 or SEQ ID No 12 and encoding a ß-1,3-endoglucanase.<br>
23) An isolated DNA molecule comprising a nucleotide sequence selected from the group of SEQ ID No 1; SEQ ID No 2. SEQ ID No 3 and SEQ ID No 4.<br>
24)An isolated DNA molecule comprising a nucleotide sequence selected from the nucleotide sequence of SEQ ID No 9 from position 2410 to 2443 and the nucleotide sequence of SEQ ID No 9 from position 2556 to3499 or the nucleotide sequence of SEQ ID No 10 from position 3373 to 3406 and the nucleotide sequence of SEQ ID No 10 from position 3501 to 4444.<br>
25) A method for isolating a fiber cell preferential promoter region, comprising the steps<br>
of:<br>
a)	identifying a genomic firagment encoding an RNA transcript from which a cDNA can be synthesized, said cDNA comprising the nucleotide sequence of SEQ ID 1 , SEQ ID No 2, SEQ ID 3 or SEQ ID No 4 or functional equivalents thereof;<br>
b)	isolating a DNA region upstream of a nucleotide sequence encoding the protein with the amino acid of SEQ ID No 11 or SEQ ID No 12 or functional equivalents thereof<br>
26) A fiber cell preferential promoter or promoter region obtained by the method of claim<br>
27) A method for increasing the length of a fiber of a cotton plant, comprising the step of introducing a chimeric gene into a cell of said cotton plant, said chimeric gene comprises the following operably linked DNA elements:<br>
(d)	a plant expressible promoter, preferably a plant expressible promoter which controls transcription preferentially in said fiber cells;<br>
(e)	a transcribed DNA region, which when transcribed yields a biologically active RNA molecule capable of reducing the expression of a ß-1,3 endoglucanase encoding gene endogenous to said fiber producing plant, said ß-l,3 endoglucanase being involved in callose removal from said plasmodesmata and<br>
(f)	a 3' end region comprising transcription termination and polyadenylation signals functioning in cells of said plant,<br>
characterized in that said endogenous ß-l,3 endoglucanase encoding gene is a ß-l,3 endoglucanase encoding gene comprised within the D subgenome of said cotton plant.<br>
28) The method according to claim 27, wherein said promoter is a fiber-specific beta tubulin promoter from cotton, a fiber-specific actin promoter from cotton, a fiber specific promoter from a lipid transfer protein gene from cotton, a promoter fi'om an expansin gene from cotton or a promoter from a chitinase gene in cotton or a promoter according to any one of claims 1 to 7.<br>
29)The method according to claim 28, wherein said ß-l,3 endoglucanase encoding gene comprised within the D subgenome is characterized by the presence of an intron sequence having the nucleotide sequence of SEQ ID No 10 from position 3407 to 3500.<br>
30)The method according to claim 31, wherein said ß-l,3 endoglucanase encoding gene comprised within the D subgenome is characterized by the presence of a nucleotide sequence AAGATC about 326 nucleotides downstream the translation initiation codon (and not including that translation initiation codon).<br>
31) The method according to claim 28, wherein said ß-l,3 endoglucanase encoding gene comprised within the D subgenome is characterized the identification of an about 538 bp   fragment   and   an   about   118   bp   fragment   after  PCR   amplification  with<br>
oligonucleotides having the nucleotide sequence of SEQ ID No 5 and SEQ ID no 6 followed by digestion with Alwl restriction enzyme.<br>
32)The method according to claim 28, wherein said endogenous gene encodes a protein comprising the amino acid sequence of SEQ ID No 12 or wherein said gene comprises the nucleotide sequence of SEQ ID No 2.<br>
33) The method of any one of claims 20 t, wherein said cotton plant is Gossypium hirsutum.<br>
-34) The method of any one of claims 50 to 35T wherein said biologically active RNA is an<br>
antisense RNA comprising at least 19 nucleotides having at least 94% sequence<br>
identity to the complement of nucleotide sequence of SEQ ED No 3.<br>
35) The method of any one of claims 30 to 35, wherein said biologically active RNA is a sense RNA comprising at least 19 nucleotides having at least 94% sequence identity to the nucleotide sequence of SEQ ED No 3.<br>
36) The method of any one of claims 27to 32 , wherein said biologically active RNA is a double stranded RNA molecule comprising at least 19 nucleotides having at least 94% sequence identity to the complement of the nucleotide sequence of SEQ ED No 3 and a complementary RNA strand essentially similar to the complement of said at least 19 nucleotides.<br>
37) The method according to any of claims 57 to 58-wherein said 19 nucleotides comprises at least one nucleotide specific for the D subgenome Ghglucl gene as indicated in Figure 2.<br>
38) The method according to claim wherein said double stranded RNA is micro RNA processed from a pre- microRNA comprising the nucleotide sequence of SEQ ID No 15, SEQ ID No 16, SEQ ID No 20 or SEQ ID No 21.<br>
39) The method according to any one of claims., wherein said fiber is a lint fiber.<br>
40) The method according to any one of claims, wherein said fiber is a fuzz fiber.<br>
41) The method according to any one of claims , further comprising<br>
(a)	growing plants obtained according to said method; and<br>
(b)	isolating fibers  said fiber-producing plants.<br>
42) A method for decreasing the length of a fiber of a fiber producing plant, comprising the step of introducing a chimeric gene into a cell of said fiber producing plant wherein said chimeric gene comprises the following operably linked DNA fragments:<br>
(d)	a plant-expressible promoter according to claims lb, 5, 6 or 7;<br>
(e)	a DNA region encoding a P-1,3 glucanase protein; and<br>
(f)	a 3'end region comprising transcription termination and polyadenylation signals functioning in cells of said plant.<br>
43)The method according to claim 17, wherein said DNA region encoding said p-1,3 glucanase protein encodes an amino acid sequence having at least 95% sequence identity to the aminoacid sequence of SEQ ID No 11 or SEQ ID No 12.<br>
44)A chimeric gene as described in any one of claims <br>
45)A cell of a fiber-producing plant comprising a chimeric gene according to claim <br>
<br>
46) A fiber producing plant comprising a chimeric gene according to claim:<br>
47) ) A fiber producing plant according to claim, wherein fibers of said plant are increased in length compared to untransformed control plants.<br>
48)A fiber producing plant according to any one of claims, wherein said plant is cotton.<br>
49) The cotton plant of claim is Gossypium hirsutum.<br>
50) Seed of the fiber producing plant according to any one of claims 49 to 5, said seed comprising a chimeric gene according to claim<br>
51) Fibers produced according to the methods of any one of claims 30 to 46<br>
<br>
</td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/viewdoc.aspx?id=tkqmAKdV0jiIUm88c2R5uw==&amp;amp;loc=+mN2fYxnTC4l0fUd8W4CAA==" target="_blank" style="word-wrap:break-word;">http://ipindiaonline.gov.in/patentsearch/GrantedSearch/viewdoc.aspx?id=tkqmAKdV0jiIUm88c2R5uw==&amp;amp;loc=+mN2fYxnTC4l0fUd8W4CAA==</a></p>
		<br>
		<div class="pull-left">
			<a href="279520-preamble-based-uplink-power-control-for-lte.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="279522-apparatus-and-methods-of-a-zero-single-point-of-failure-load-balancer.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>279521</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>4287/DELNP/2009</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>04/2017</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>27-Jan-2017</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>24-Jan-2017</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>29-Jun-2009</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>BAYER CROPSCIENCE NV</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td>J.E. MOMMAERTSLAAN 14, 1831 DIEGEM, BELGIUM,</td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>STEVEN ENGELEN</td>
											<td>BASTELARE 62, 9080 ZEVENEKEN, BELGIUM</td>
										</tr>
										<tr>
											<td>2</td>
											<td>ANTONIO ARIOLI</td>
											<td>7307 93RD STREET 79424 LUBBOCK TEXAS, U.S.A</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>C12N 15/82</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>PCT/EP2008/000161</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td>2008-01-07</td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td>07000550.9</td>
									<td>2007-01-11</td>
								    <td>U.S.A.</td>
								</tr>
								<tr>
									<td>2</td>
									<td>60/884564</td>
									<td>2007-01-11</td>
								    <td>U.S.A.</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/279521-differential-expression-of-subgenome-specific-alleles-in-cotton-and-uses-thereof by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 00:34:05 GMT -->
</html>
