<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/279822-a-method-for-exchanging-a-target-dna-sequence-in-the-genome-of-a-eukaryotic-cell by HTTrack Website Copier/3.x [XR&CO'2014], Thu, 04 Apr 2024 23:02:38 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 279822:&quot;A METHOD FOR EXCHANGING A TARGET DNA SEQUENCE IN THE GENOME OF A EUKARYOTIC CELL&quot;</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">&quot;A METHOD FOR EXCHANGING A TARGET DNA SEQUENCE IN THE GENOME OF A EUKARYOTIC CELL&quot;</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>Methods and means are provided for the exact exchange in eukaryotic cells, such as plant cells, of a target DNA sequence for a DNA sequence of interest through homologous recombination, whereby the selectable or screenable marker used during the homologous recombination phase for temporal selection of the gene replacement events can subsequently be removed without leaving a foot-print employing a method for the removal of a selected DNA flanked by two nucleotide sequences in direct repeats.</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td>Methods and means for exact replacement of target DNA in<br>
eukaryotic organisms<br>
Field of the invention<br>
[1.] The current invention relates to improved methods and means for the exact exchange of a target DNA sequence for a DNA sequence of interest through homologous recombination in eukaryotic cells and organism, such as plant cells and plants, whereby the selectable or screenable marker used during the homologous recombination phase for temporal selection of the gene replacement events can subsequently be removed without introducing any other sequence variations.<br>
Background art<br>
[2.] Homologous recombination allows numerous targeted genetic modifications in prokaryotic and selected eukaryotic organisms including selected deletions, insertions or replacements.<br>
[3.] In higher eukaryotic organisms, homologous recombination may be stimulated through the induction of double stranded DNA breaks via rare-cutting endonucleases, such as e.g. I-Scel.<br>
[4.] WO2004/067753 describes the use of meganucleases for inducing homologous recombination ex vivo and in toto in vertebrate somatic tissues and the application thereof for genome engineering and gene therapy.<br>
[5.] WO2000/46386 describes methods of modifying, repairing, attenuating and inactivating a gene or other chromosomal DNA in a cell through I-Scel induced double stranded breaks. Also disclosed are methods of treating or phrophylaxis of a genetic disease in an individual in need thereof.<br>
[6.] In plants, induction of double stranded DNA breaks using I-Scel has been shown to increase the frequency of homologous recombination by at least two orders of magnitude using Agrobacteria to deliver the repair DNA to the plant cells (Puchta et al., 1996, Proc. Natl. Acad. Sci. U.S.A., 93, pp5055-5060). Chilton and Que (2003, Plant Physiol. 133: pp 956-965) and Tzifira et al. (2003, Plant Physiol. 133: pplOl 1-1023) report that T-DNA preferentially integrates in double stranded DNA breaks, artificially induced by the rare-cleaving enzymes I-Scel or I-Ceul. The reports also included donor T-DNA vectors which comprised a recognition site for the respective rare-cleaving enzyme.<br>
endonucleases to alter substrate or sequence-specificity of the enzymes, thus allowing to induce a double stranded break at virtually any locus of interest without being dependent on the presence of a recognition site for any of the natural rare-cleaving endonucleases. Briefly, chimeric restriction enzymes can be prepared using hybrids between a zinc-finger domain designed to recognize a specific nucleotide sequence and the non-specific DNA-cleavage domain from a natural restriction enzyme, such as Fokl. Such methods have been described e.g. in WO 03/080809, W094/18313 or WO95/09233 and in Isalan et al., 2001, Nature Biotechnology 19, 656- 660; Liu et al. 1997, Proc. Natl. Acad. Sci. USA 94, 5525-5530). Another way of producing custom-made meganucleases, by selection from a library of variants, is described in WO2004/067736. Custom made meganucleases with altered sequence specificity and DNA-binding affinity may also be obtained through rational design as described in WO2007/047859.<br>
[8.] WO2007/049095 describes "LADGLIDADG" homing endonuclease variants having mutations in two separate subdomains, each binding a distinct part of a modified DNA target half site,such that the endonuclease variant is able to cleave a chimeric DNA target sequence comprising the nucleotides bound by each subdomain.<br>
[9.] WO2007/049156 and WO 2007/093836 describe I-Crel homing endonuclease variants having novel cleavage specificity and uses thereof.<br>
[10.] WO2007/047859 describes rationally designed meganucleases with altered sequence specificity and DNA binding affinity.<br>
[11.] WO2006/105946 described a method for the exact exchange in plant cells and plants of a target DNA sequence for a DNA sequence of interest through homologous recombination, whereby the selectable or screenable marker used during the homologous recombination phase for temporal selection of the gene replacement events can subsequently be removed without leaving a foot-print and without resorting to in vitro culture during the removal step, employing the therein described method for the removal of a selected DNA by microspore specific expression of a double stranded break inducing rare cleaving endonuclease.<br>
[12.] US provisional patent application 60/828,042 and European patent application 06020370.0, and WO2008/037436 describe variants of the methods and means of WO2006/105946 wherein the removal step of a selected DNA fragment induced by a double stranded break inducing rare cleaving endonuclease is under control of a germline-specific promoter. Other embodiments of the method relied on non-homologous endjoining at one end of the repair DNA and homologous recombination at the other end.<br>
[13.] Some of the embodiments of the above identified methods and means for exact exchange of a target DNA fragment for a DNA fragment of interest require that the introduced repair DNA is introduced in the plant cell in the presence of the double stranded break inducing enzyme. The repair DNA normally also contains the preselected site recognized by a double stranded break inducing rare cleaving endonuclease and therefore the repair DNA is also prone to DNA cleavage. Accordingly, the efficiency of DNA insertion by homologous recombination may be lowered. To avoid this decrease in efficiency, the preselected site in the repair DNA may be altered in such a way that it is no longer recognized by the double stranded break inducing rare cleaving endonuclease.<br>
However, this entails the introduction of an extra change in the repair DNA compared to the the target DNA in addition to the desired change.<br>
[14.] The current invention provides an alternative solution to this problem, which does not require the modification of the preselected site in the repair DNA and consequently allows the exchange of the target DNA with only the desired nucleotide change, without modification of the preselected site. These and other problems are solved as described hereinafter in the different detailed embodiments of the invention, as well as in the claims.<br>
Summary of the invention<br>
[15.] In one embodiment of the invention, a method is provided for exchanging a target DNA sequence in the genome a eukaryotic cell or eukaryotic organism for a DNA sequence of interest comprising the following steps:<br>
a.	Inducing a first double stranded DNA break at a preselected site in the<br>
genome of a cell of the eukaryotic organism, the preselected site being<br>
located within the target DNA sequence or in the vicinity of the target<br>
DNA sequence and the preselected site being recognized by a first double-<br>
stranded break inducing (DSBI) enzyme;<br>
b.	Introducing a repair DNA molecule into the eukaryotic cell, the repair<br>
DNA molecule comprising<br>
i. a DNA sequence of interest located between two flanking DNA regions having at least 80 % sequence homology to a DNA region flanking the target DNA sequence, and preferably flanking the preselected site in the genome of the eukaryotic cell;<br>
ii. A selectable or screenable marker gene located between the flanking DNA regions, the selectable or screenable marker gene further being located between a first repeat sequence consisting of the 5'-terminal part of the preselected site and a second sequence consisting of the 3' terminal part of the preselected site, whereby<br>
the sequences common between the first and second repeat sequences are in direct repeat; and iii. At least one recognition site for a second DSBI enzyme located between the one of the flanking DNA regions and the first and second repeat sequence, preferably between the first and sequence direct repeat sequence;<br>
c.	Selecting a population of cells comprising the selectable or screenable<br>
marker;<br>
d.	Selecting a cell wherein the selectable or screenable marker has been<br>
introduced by homologous recombination through the flanking DNA<br>
regions;<br>
e.	Introducing a double stranded break at the recognition site for the second<br>
DSBI enzyme in the cell;<br>
f.	Selecting a progeny cell wherein the selectable or screenable marker gene<br>
is deleted by homologous recombination between the direct repeats<br>
thereby recreating the preselected site.<br>
[16.] In another embodiment of the invention, a method is provided for exchanging a target DNA sequence in the genome, particularly the nuclear genome, of a plant for a DNA sequence of interest comprising the following steps:<br>
a)	Inducing a first double stranded DNA break at a preselected site in the genome of a cell of the plant, the preselected site being located within the target DNA sequence or in the vicinity of the target DNA sequence and the preselected site being recognized by a first double-stranded break inducing (DSBI) enzyme<br>
b)	Introducing a repair DNA molecule into the eukaryotic cell, the repair DNA molecule comprising<br>
i) The DNA sequence of interest located between two flanking DNA regions having at least 80 % sequence homology to a DNA region flanking the target DNA sequence, and preferably flanking the preselected site in the genome of the eukaryotic cell;<br>
ii) A selectable or screenable marker gene located between the flanking DNA regions, the selectable or screenable marker gene further being located between a first repeat sequence consisting of the 5'-terminal part of the preselected site and a second sequence consisting of the 3' terminal part of the preselected site, whereby the sequences common between the first and second repeat sequences are in direct repeat; and<br>
iii) At least one recognition site for a second DSBI enzyme located between the one of the flanking DNA regions and the first and second repeat sequence;<br>
c)	Selecting a population of plant cells comprising the selectable or screenable marker;<br>
d)	Selecting a plant cell wherein the DNA sequence of interest (and the selectable or<br>
the flanking DNA regions, and regenerating a plant from the plant cell;<br>
e)	Crossing the regenerated plant or a progeny plant thereof comprising the<br>
selectable marker gene with a plant comprising a rare cleaving double stranded<br>
break inducing ("DSBI") enzyme encoding chimeric gene, the chimeric gene<br>
comprising the following operably linked DNA segments:<br>
i.   a germline specific promoter; ii.    a DNA region encoding a double stranded DNA break inducing enzyme<br>
recognizing the recognition site located in the DNA of interest (i.e. the second<br>
double stranded DNA break inducing enzyme); iii.   a transcription termination and polyadenylation region;<br>
f)	Selecting a progeny plant (Fl-plant) comprising the selectable or screenable marker gene and the DSBI enzyme encoding chimeric gene;<br>
g)	Crossing the progeny plant with another plant whereby the progeny plant is used as pollen donor;<br>
h)   Selecting a population of progeny plants (F2-population) which comprises the<br>
DSBI enzyme encoding chimeric gene; and i)    Selecting a progeny plant wherein the selectable or screenable marker gene is<br>
deleted by homologous recombination between the first and second direct repeat<br>
sequence.<br>
The invention relates to the eukaryotic cells and plants obtainable by the above described methods.<br>
Brief description of the drawings<br>
[17.] Figures 1 and 2 represent different embodiments of the method to exchange a target DNA for a DNA of in a plant cell without any additional modifications. These figures are for illustration purposes only and should not be used to construe the claims in a limiting manner.<br>
[18.] Figure 1 is a schematic representation of a method allowing exact replacement of a target DNA sequence with a replacement DNA sequence. DSB1: recognition site for a first double stranded break inducing enzyme; FS1: flanking sequence 1; FS2: flanking sequence 2; DSB2: recognition site for a second double stranded break inducing enzyme; SMG1: selectable marker gene 1 or screenable marker gene 1; SMG2: selectable marker gene 2 or screenable marker gene 2; DSBIE: double stranded break inducing enzyme; drl: direct repeat sequence 1 contained within the 5' part of the preselected site recognized by DSBIE 1; dr2: direct repeat sequence 2 contained within the 5' part of the preselected site recognized by DSBIE1; GSP: germline specific promoter; 3': transcription termination and polyadenylation signal. In Figure 1, the preselected site is located in the vicinity of the target DNA.<br>
[19.] Fig 2 is a schematic representation of a method allowing exact replacement of a target DNA sequence with a replacement DNA sequence similar to the method illustrated in Fig 1. In this embodiment, the preselected site is located within the target DNA.<br>
[20.] Fig 3 is a schematic representation of a hypothetical 20 nucleotide long recognition site (N1-N20) for a double stranded break inducing rare cleaving enzyme. Indicated are a first and second repeat sequence as described elsewhere whereby drl<br>
corresponds to the 5' part of the recognition site (here exemplified as N1-N17) and dr2 corresponds to the 3' part of the recognition site (here exemplified as N4-N20). The direct repeat is between nucleotides N4 -N17.<br>
Detailed embodiments of the invention<br>
[21.] The current invention is based on the realization that the efficiency of several of the methods described e.g. in W02006/105946 (particularly for the embodiments relying homologous recombination on both sides of the repair DNA) can be enhanced by providing a specific repair DNA wherein the direct DNA repeat (used in the removal step of the screenable or selectable marker) consist on the one end of the 5'-terminal part of the preselected site (recognized by the first double stranded br=ak inducing ctizymc) and on the other hand of the 3' terminal part of the preselected site. In this way the repair DNA does not contain a preselected site prone to cleavage by the first double stranded break inducing enzyme. However, upon induction of homologous recombination between the nucleotide sequences common to the 5' terminal and 3' terminal part of the preselected site in direct repeat (which results in the removal of the intermittent selectable or screenable marker gene), the original nucleotide sequence of the preselected site is reconstructed.<br>
[22.] Thus, in a first embodiment, the invention provides a method for exchanging a target DNA sequence in the genome, particularly the nuclear genome, of a plant for a DNA sequence of interest comprising the following steps:<br>
a)	Inducing a first double stranded DNA break at a preselected site in the genome of a cell of the plant, the preselected site being located within the target DNA sequence or in the vicinity of the target DNA sequence and the preselected site being recognized by a first double-stranded break inducing (DSBI) enzyme<br>
b)	Introducing a repair DNA molecule into the plant cell, the repair DNA molecule comprising<br>
i)   The DNA sequence of interest located between two flanking DNA regions having at least 80 % sequence homology to a DNA region flanking the target<br>
DNA sequence, and preferably flanking the preselected site in the genome of the eukaryotic cell;<br>
ii) A selectable or screenable marker gene located between the flanking DNA regions, the selectable or screenable marker gene further being located between a first repeat sequence consisting of the 5'-terminal part of the preselected site and a second sequence consisting of the 3' terminal part of the preselected site, whereby the sequences common between the first and second repeat sequences are in direct repeat; and<br>
iii) At least one recognition site for a second DSBI enzyme located between the one of the flanking DNA regions and the first and second repeat sequence;<br>
c)	Selecting a population of plant cells comprising the selectable or screenable marker;<br>
d)	Selecting a plant cell wherein the DNA sequence of interest (and the selectable or screenable marker) has been introduced by homologous recombination through the flanking DNA regions, and regenerating a plant from the plant cell;<br>
e)	Crossing the regenerated plant or a progeny plant thereof comprising the selectable marker gene with a plant comprising a rare cleaving double stranded break inducing ("DSBI") enzyme encoding chimeric gene, the chimeric gene comprising the following operably linked DNA segments:<br>
iv.   a germline specific promoter, such as a microspore specific promoter; v.    a DNA region encoding a double stranded DNA break inducing enzyme<br>
recognizing the recognition site located in the DNA of interest; vi.   a transcription termination and polyadenylation region;<br>
f)	Selecting a progeny plant (Fl-plant) comprising the selectable or screenable marker gene and the DSBI enzyme encoding chimeric gene;<br>
g)	Crossing the progeny plant with another plant whereby the progeny plant is used as pollen donor if the germline specific promoter is a promoter expressed during microsporogenesis or whereby the progeny plant is used as a pollen acceptor if the germline specific promoter is a promoter expressed during macrosporogenesis;<br>
h)  Selecting a population of progeny plants (F2-population) which comprises the<br>
DSBI enzyme encoding chimeric gene; and i)   Selecting a progeny plant wherein the selectable or screenable marker gene is<br>
deleted by homologous recombination between the one of the flanking DNA<br>
regions and a partial flanking DNA region comprising part of the one of the<br>
flanking DNA regions.<br>
[23.] As used herein, a "double stranded DNA break inducing rare-cleaving endonuclease" is an enzyme capable of inducing a double stranded DNA break at a particular nucleotide sequence, called the "recognition site". Rare-cleaving endonucleases, also sometimes called mega-nucleases have a recognition site of 14 to 40<br>
frequency of cleaving, even in the larger plant genomes. Homing endonucleases constitute a family of such rare-cleaving endonucleases. They may be encoded by introns, independent genes or intervening sequences, and present striking structural and functional properties that distinguish them from the more classical restriction enzymes, usually from bacterial restriction-modification Type II systems. Their recognition sites have a general asymmetry which contrast to the characteristic dyad symmetry of most restriction enzyme recognition sites. Several homing endonucleases encoded by introns or inteins have been shown to promote the homing of their respective genetic elements into allelic intronless or inteinless sites. By making a site-specific double strand break in the intronless or inteinless alleles, these nucleases create recombinogenic ends, which engage in a gene conversion process that duplicates the coding sequence and leads to the insertion of an intron or an intervening sequence at the DNA level.<br>
[24.] A well characterized homing endonuclease is I-Scel. I-Scel is a site-specific endonuclease, responsible for intron mobility in mitochondria in Saccharomyces cerevisea. The enzyme is encoded by the optional intron Sc LSU.l of the 21S rRNA gene and initiates a double stranded DNA break at the intron insertion site generating a 4 bp staggered cut with 3'OH overhangs. The recognition site of I-Scel endonuclease extends over an 18 bp non-symmetrical sequence (Colleaux et al. 1988 Proc. Natl. Acad. Sci.<br>
USA 85: 6022-6026). The amino acid sequence for I-Scel and a universal code equivalent of the mitochondrial I-Scel gene have been provided by e.g. WO 96/14408. WO 96/14408 further discloses a number of variants of I-Scel protein which are still functional.<br>
[25.] PCT application PCT/EP04/013122 (incorporated herein by reference) provides synthetic nucleotide sequence variants of I-Scel which have been optimized for expression in plants. The nucleotide sequence of such synthetic I-Sce I coding regions is set forth in SEQ ID No 1 in UIPAC code. The symbols of the UIPAC code have their usual meaning i.e. N= A or C or G or T; R= A or G; Y= C or T; B= C or G or T (not A); V= A or C or G (not T); D= A or G or T (not C); H=A or C or T (not G); K= G or T; M= A or C; S= G or C; W=A or T.<br>
[26.] A list of other rare cleaving DSB inducing enzymes and their respective recognition sites is provided in Table I of WO 03/004659 (pages 17 to 20) (incorporated herein by reference). These include I-Sce I, I-Chu I, 1-Dmo I, I-Cre I, I-Csm I, PI-Fli I, Pt-Mtu I, I-Ceu I, I-Sce II, I-Sce III, HO, Pi-Civ I, Pl-Ctr I, PI-Aae I, PI-BSU I, PI-Dhal, PI-Dra I, PI-Mav I, PI-Mch I, PI-Mfu I, PI-Mfl I, PI-Mga I, PI-Mgo I, PI-Min I, PI-Mka I, PI-Mle I, PI-Mma I, PI-Msh I, PI-Msm I, PI-Mth I, PI-Mtu I, PI-Mxe I, PI-Npu I, PI-Pfu I, PI-Rma I, Pl-Spb I, PI-Ssp I, PI-Fac I, PI-Mja I, Pl-Pho I, PI-Tag I, PI-Thy I, PI-Tko I or PI-Tsp I.<br>
[27.] Furthermore, methods are available to design custom-tailored rare-cleaving endonucleases that recognize basically any target nucleotide sequence of choice. Briefly, chimeric restriction enzymes can be prepared using hybrids between a zinc-finger domain designed to recognize a specific nucleotide sequence and the non-specific DNA-cleavage domain from a natural restriction enzyme, such as Fokl. Such methods have been described e.g. in WO 03/080809, W094/18313 or WO95/09233 and in Isalan et al., 2001, Nature Biotechnology 19, 656- 660; Liu et al. 1997, Proc. Natl. Acad. Sci. USA 94, 5525-5530). Another way of producing custom-made meganucleases, by selection from a library of variants, is described in WO2004/067736. Custom made meganucleases with<br>
altered sequence specificity and DNA-binding affinity may also be obtained through rational design as described in WO2007/047859<br>
[28.] As used herein "a preselected site" indicates a particular nucleotide sequence in the plant nuclear genome, located in or near the target DNA sequence at which location it is desired to insert the foreign DNA or to exchange the target DNA sequence. A person skilled in the art would be able to either choose a double stranded DNA break inducing ("DSBI") enzyme recognizing the selected target nucleotide sequence or engineer such a DSBI endonuclease. Alternatively, a DSBI endonuclease recognition site may be introduced into the plant genome using any conventional transformation method or by conventional breeding using a plant line having a DSBI endonuclease recognition site in<br>
previously introduced preselected target site.<br>
[29.] The double stranded DNA breaks in the transforming DNA molecule may be induced conveniently by transient introduction of a plant-expressible chimeric gene comprising a plant-expressible promoter region operably linked to a DNA region encoding a double stranded break inducing enzyme. The DNA region encoding a double stranded break inducing enzyme may be a synthetic DNA region with plant-optimized codon usage. The endonuclease itself, as a protein, could also be introduced into the plant cells, e.g. by electroporation. However, the endonuclease can also be provided in a transient manner by introducing into the genome of a plant cell or plant, a chimeric gene comprising the endonuclease coding region operably linked to an inducible plant-expressible promoter, and providing the appropriate inducible compound for a limited time prior to, during or immediately after introduction of the transforming DNA molecule. The endonuclease could also be provided as an RNA precursor encoding the endonuclease.<br>
[30.] The double stranded break inducing enzyme may comprise, but need not comprise, a nuclear localization signal (NLS), such as the NLS of SV40 large T-antigen [Raikhel, Plant Physiol. 100: 1627-1632 (1992) and references therein] [Kalderon et al. Cell 39:<br>
499-509 (1984)]. The nuclear localization signal may be located anywhere in the protein, but is conveniently located at the N-terminal end of the protein. The nuclear localization signal may replace one or more of the amino acids of the double stranded break inducing enzyme.<br>
[31.] As used herein, the "target DNA sequence" is the DNA sequence located in the genome of the plant cell which is modified, by addition, deletion or substitution.<br>
[32.] As used herein "flanking DNA regions" are DNA sequences having homology to the DNA regions respectively upstream or downstream of the target DNA sequence. This allows to better control the insertion of the foreign DNA or the DNA molecule of interest. Indeed, integration by homologous recombination will allow precise joining of the foreign DNA fragment to the plant nuclear genome up to the nucleotide level.<br>
[33.] The flanking DNA regions may vary in length, and should be at least about 10 nucleotides in length. However, the flanking region may be as long as is practically possible (e.g. up to about 100-150 kb such as complete bacterial artificial chromosomes (BACs)). Preferably, the flanking region will be about 50 bp to about 2000 bp. Moreover, the regions flanking the foreign DNA of interest need not be identical to the DNA regions flanking the preselected site and may have between about 80% to about 100% sequence identity, preferably about 95% to about 100% sequence identity with the DNA regions flanking the preselected site. The longer the flanking region, the less stringent the requirement for homology. Furthermore, it is preferred that the sequence identity is as high as practically possible in the vicinity of the location of exact insertion of the foreign DNA. Furthermore, to achieve exchange of the target DNA sequence without changing the DNA sequence of the adjacent DNA sequences, the flanking DNA sequences should preferably be identical to the DNA regions flanking the preselected site.<br>
[34.] Moreover, the regions flanking the foreign DNA of interest need not have homology to the regions immediately flanking the preselected site, but may have homology to a DNA region of the nuclear genome further remote from that preselected<br>
site. Insertion of the foreign DNA will then result in a removal of the target DNA between the preselected insertion site and the DNA .region of homology. In other words, the target DNA located between the homology regions will be substituted for the foreign DNA of interest.<br>
[35.] Preferably, the preselected site and the further mentioned recognition sequence are recognized by different rare cleaving double stranded break inducing endonucleases.<br>
[36.] As used herein flanked by two DNA sequences "arranged in direct repeat" indicates that the sequence to be removed from the introduced DNA molecule is immediately preceded and followed by two DNA regions, one at each end, wherein the<br>
invention, the DNA sequences arranged in direct repeat are a first DNA sequence upstream of the sequence to be removed (i.e. the selectable or screenable marker gene) consist of the 5'terrninal part of the preselected site (i.e. the site selected by the first double stranded break inducing rare cleaving enzyme), whereas the second DNA sequence downstream of the sequence to be removed consists of the 3'terminal part of the preselected site. It will be immediately clear to the person skilled in the art that different preselected sites may differ in length and range e.g. from 15 nucleotides to 50 nucleotides. Accordingly, a sequence corresponding to the 5'terminal part of the preselected sequence is a sequence corresponding to the nucleotide sequence of the preselected sequence (or recognition site) which lacks at the 3' end of that sequence one or more nucleotides, such that the "5' terminal sequence" is no longer recognized and/or cleaved by the double stranded break inducing rare cleaving enzyme. Similarly, a sequence corresponding to the 3'terminal part of the preselected sequence is a sequence corresponding to the nucleotide sequence of the preselected sequence (or recognition site) which lacks at the 5' end of that sequence one or more nucleotides, such that the "3' terminal sequence" is no longer recognized and/or cleaved by the double stranded break inducing rare cleaving enzyme. 5' terminal part and 3'terminal part may lack 1, 2, 3, 4, 5, 6, 7, 8 or more nucleotides. Moreover, there is no need that there is a correspondence in the number of nucleotides lacking in the 5'terminal and 3' terminal part. E.g. while the<br>
5'terminal part may be lacking 2 nucleotides of the recognition site at the 3' end, the 3'terminal part may be lacking 5 nucleotides of the recognition site at the 5' end. The actual sequences in direct repeat, which will allow removal of the nucleotide sequence located in-between, correspond to the nucleotide sequence common between the 5'terminal part and the 3' terminal part. Although the length of the actual direct repeat sequences will depend on the length of the recognition site or preselected site as well as on the amount of nucleotides lacking at respectively the 3' and 5' end, it is preferred that the nucleotide sequence in common comprises at least 5, or 10, or 14 or more nucleotides. Reference is made to Figure 3 for a schematic representation of a 5'terminal part (drl) and 3' terminal part of a hypothetical preselected site or recognition site.<br>
[37.] It will be immediately clear to a person skilled in the art that for the purpose of the current invention that the repair DNA does not contain the preselected site. To this end, the selectable or screenable marker should preferably be immediately flanked by the direct repeat sequences or the 5'terminal part and 3' terminal parts of the preselected site. Preferably, one of the two sequences (5'terminal part or 3' terminal part) should be located in the repair DNA at its corresponding position with regard to the target DNA.<br>
[38.] For avoidance of doubt, if the two DNA regions similar in nucleotide sequence are contained within a double stranded DNA molecule, these DNA sequences are to be located on the same DNA strand, in the same 5'-&gt;3' direction.<br>
[39.] For the purpose of this invention, the "sequence identity" of two related nucleotide or amino acid sequences, expressed as a percentage, refers to the number of positions in the two optimally aligned sequences which have identical residues (xl00) divided by the number of positions compared. A gap, i.e. a position in an alignment where a residue is present in one sequence but not in the other, is regarded as a position with non-identical residues. The alignment of the two sequences is performed by the Needleman and Wunsch algorithm (Needleman and Wunsch 1970) Computer-assisted sequence alignment, can be conveniently performed using standard software program such as GAP which is part of the Wisconsin Package Version  10.1  (Genetics Computer Group,<br>
Madison, Wisconsin, USA) using the default scoring matrix with a gap creation penalty of 50 and a gap extension penalty of 3.<br>
[40.] As used herein "located in the vicinity" refers to the DSBI being located at a distance of between 500 bp, 1 kbp to 10 kbp from the reference DNA sequence.<br>
[41.] According to the current invention, at least one recognition site for a second double stranded break inducing rare cleaving enzyme is located between the direct repeats. In case where two such recognition sites are present, such recognition sites or parts thereof may be present in direct repeats and homologous recombination between two such sites or subparts thereof may remove the selectable or screenable marker. Upon such homologous recombination however an intact recognition site for the second double stranded break inducing rare cleaving enzyme is generated, and the first and second direct repeat sequences which are part of the preselected site are brought in closer contact via the deletion of the intervening sequences. Such a constellation of direct repeat sequences in close vicinity, flanking a recognition site for a double stranded break inducing enzyme is beneficial to induce high efficient recombination between the two direct repeat sequences deleting the remaining intervening sequences.<br>
[42.] The methods described herein for the use in plants can be conveniently carried out, using a chimeric gene encoding a rare-cleaving double stranded break inducing enzyme, whereby the coding region for the endonuclease is under control of a germline specific promoter fragment.<br>
[43.] As used herein, a "germline-specific promoter" is a promoter region, promoter or fragment which can promote transcription selectively, preferably specifically in plant cells that ultimately produce the gametes starting from megaspore-mother cell or the meiocyte. A germline-specific promoters as defined herein thus include gametophyte-specific promoter, gamete-specific promoters, promoters which control expression in microspores and/or megaspores or in their respective immediate precursor cells.<br>
[44.] As used herein, a "promoter specific for gametogenesis" is a promoter region, promoter or fragment which can promote transcription selectively, preferably specifically in plant cells which are the immediate precursor cells of the gametes.<br>
[45.] In angiosperm plants, sexual reproduction requires the production of viable male and female gametophytes. Pollen, as the male gametophyte is formed within the anther and is initiated from sporogenous cells, which develop into meiocytes. The meiocyte undergoes meiosis to form a tetrad of haploid microspores, which are subsequently released into the anther locule. Following expansion and vacuolation, an asymmetrical mitosis of the microspore results in bicellular pollen, containing a vegetative and a generative cell. In the majority of species, pollen is shed in bicellular condition. The female gametophyte, the embryo sac, initiates in the ovary from the megaspore mother cell or megasporocyte through two meiotic divisions, resulting in the formation of a linear tetrad of haploid megaspores. The chalazal megaspore enlarges in the preparation for the first mitotic division in the female gametophyte development, while the other three megaspores degenerate. Mitotic divisions occur in three generations of nuclei so that an eight nucleate embryo sac is formed. During these divisions the former megaspore cell enlarges and becomes much vacuolated. The eight-nucleate cell is organized into the seven-celled embryo sac through the delimitation by cell walls of six of the nuclei and associated cytoplasm. The three cells at the micropylar end constitute the egg apparatus which is composed of the egg and two synergids. At the opposite end of the embryo sac are three antipodal cells. Between the two groups of cells is the large central cell containing two polar nuclei, which may fuse prior to fertilization and form the diploid secondary endosperm nucleus.<br>
[46.] As used herein "a microspore specific promoter region" or "a microspore specific promoter" or a "a microspore specific promoter fragment" is a promoter region or promoter or promoter fragment which can promote transcription selectively, preferably specifically, in the unicellular microspore of a plant.. A suitable microspore specific promoter region is described in WO 97/30166 (incorporated herein by reference) as the<br>
promoter region from NTM19 gene in tobacco and its use in a method for targeted exchange in plants is exemplified in WO2006/105946.<br>
[47.] As used herein "a megaspore specific promoter region" or "a megaspore specific promoter" or a. "a megaspore specific promoter fragment" is a promoter region or promoter or promoter fragment which can promote transcription selectively, preferably specifically, in a unicellular megaspore of a plant, preferably a megaspore which develops into an embryo sac.<br>
[48.] Particular promoters such as the BnSKPlyl may control transcription specifically or selectively both in microspores and megaspores of plants (Drouad et al. 2000 Sex Plant Renrod. 13: 29-35)<br>
*<br>
[49.] Further suitable germline-specific promoters exemplified in US provisional patent application 60/828,042 and European patent application 06020370.0 may be any one of the following (citations below are herein incorporated by reference):<br>
i.   A  promoter  comprising   an   Arabidopsis   egg  apparatus  (EA) specific enhancer, fused to a minimal promoter element such as a minimal 35S promoter, as described by Yang et al., 2005, Plant Physiol. 139(3): 1421-1432 ii.   An Arabidopsis TAG1 promoter as described by Galli et al., 2003 Genetics.   165(4):2093-2105   (expressed   in   male   and   female gametophytes) iii.    An Arabidopsis Duol promoter (male generative cell and sperm cell activity as described by Rotman et al., 2005 Curr Biol. 15(3):244-248 iv.   promoters as could be isolated from the female gametophytic genes described by Yu et al., 2005 Plant Physiology 139(4): 1853-1869 v.    a promoter from LGC1 from Lilium expressed in male generative cell and sperm cells (Xu te al., 1999 Proc Natl Acad Sci USA<br>
96(5):2554-2558; Singh et al. 2003 FEBS Lett. 2003 542(1-3):47-<br>
52. vi.   A promoter from the ERCC1 homolog expressed in male sperm<br>
cells (Xu et al. 1998 Plant J. 13(6):823-829) vii.   A promoter from H2A or H3 histone genes (Xu et al. 1999 Male<br>
gametic cell-specific expression of H2A and H3 histone genes.<br>
Plant  Molecular  Biology 39,   607-614;  Okada  et  al.  (2005)<br>
Transcriptional activity of male gamete-specific histone gcH3<br>
promoter in sperm cell of Lilium longiflorum. Plant and Cell<br>
Physiology 46, 797-802 ) viii.   Promoters from sperm cell genes as identified in rice (Chen,<br>
Schuan University, GenBank entries BE225314 to BE225323,<br>
BF475189 to BF475237) and as identified in corn (Engel et al.,<br>
2003 The Plant Journal 34: 697-707) ix.   The Zmeal promoter (Marton et al. Science. 2005, 307:573-576)<br>
and   Zmes promoters (Cordts et al. Plant J. 2001 25(1):103-114)<br>
specific for egg apparatus and embryosac, respectively x.   Promoters comprising silencer elements recognized by GRSF or<br>
germline  restrictive  silencing  factor  (Haerizadeh  et  al.   2006<br>
Science 28 313: pp. 496 - 499) xi.   BnMl  or BnM3.4 promoter described by Guerche et al.  1999<br>
(Plant Molecular Biology 40:  857-872) and promoters driving<br>
expression of microspore-specific cDNAs M21.<br>
[50.] As used herein "coding region for a rare cleaving double stranded break inducing endonuclease" or "coding region for a rare cleaving double stranded break inducing enzyme" is a nucleotide sequence which encodes a polypeptide that is characterized as a rare cleaving DSBI enzyme such as the homing endonucleases or the chimeric endonucleases described elsewhere in this application. The coding region may thus comprise any nucleotide sequence that encodes any of the amino acid sequences of the<br>
homing endonucleases listed in the following table, which can be found in public databases under the mentioned accession numbers (all herein incorporated by reference):<br>
(TABLE REMOVED)<br>
[51.] It will be clear that for expression of the endonucleases under the control of a microspore specific promoter fragment, the coding region should be adapted so that the universal codon language is used to encode the above mentioned polypeptides. The coding region may further be optimized for expression in plants and the synthetic coding region may have a nucleotide sequence which has been designed to fulfill the following criteria:<br>
a)   the nucleotide sequence encodes a functional rare cleaving double stranded break inducing endonuclease,<br>
b)	the nucleotide sequence has a GC content of about 50% to about 60%<br>
c)	the nucleotide sequence does not comprise a nucleotide sequence selected from the group consisting of GATAAT, TAT AAA, AATATA, AATATT, GATAAA, AATGAA, AATAAG, AATAAA, AATAAT, AACCAA, ATATAA, AATCAA, ATACTA, ATAAAA, ATGAAA, AAGCAT, ATTAAT, ATACAT, AAAATA, ATTAAA, AATTAA, AATACA and CATAAA;<br>
d)	the nucleotide does not comprise a nucleotide sequence selected from the group consisting of CCAAT, ATTGG, GCAAT and ATTGC;<br>
e)	the nucleotide sequence does not comprise a sequence selected from the group consisting of ATTTA, AAGGT, AGGTA, GGTA or GCAGG;<br>
f)	the nucleotide sequence does not comprise a GC stretch consisting of 7 consecutive nucleotides selected from the group of G or C;<br>
g)	the nucleotide sequence does not comprise a GC stretch consisting of 5 consecutive nucleotides selected from the group of A or T; and<br>
h) the nucleotide sequence does not comprise codons coding for Leu, He, Val, Ser, Pro, Thr, Ala that comprise TA or CG duplets in positions 2 and 3 (i.e. the nucleotide sequence does not comprise the codons TTA, CTA, ATA, GTA, TCG, CCG, ACG and GCG).<br>
[52.] The double stranded break inducing enzyme may comprise, but need not comprise, a nuclear localization signal (NLS) [Raikhel, Plant Physiol. 100: 1627-1632 (1992) and references therein], such as the NLS of SV40 large T-antigen [Kalderon et ail. Cell 39: 499-509 (1984)]. The nuclear localization signal may be located anywhere in the protein, but is conveniently located at the N-terminal end of the protein. The nuclear localization signal may replace one or more of the amino acids of the double stranded break inducing enzyme.<br>
[53.] Having understood the underlying principles of the current invention, a person skilled in the art will realize that the method can be used in cells of eukaryotic organisms different from plants.<br>
[54.] Thus, in another embodiment of the invention, a method is provided for exchanging a target DNA sequence in the genome a eukaryotic cell or eukaryotic organism for a DNA sequence of interest comprising the following steps:<br>
g. Inducing a first double stranded DNA break at a preselected site in the genome of a cell of the eukaryotic organism, the preselected site being located within the target DNA sequence or in the vicinity of the target DNA sequence and the preselected site being recognized by a first double-stranded break inducing (DSBI) enzyme; h. Introducing a repair DNA molecule into the eukaryotic cell, the repair DNA molecule comprising<br>
regions having at least 80 % sequence homology to a DNA region flanking the target DNA sequence, and preferably flanking the preselected site in the genome of the eukaryotic cell; ii.   A selectable  or screenable marker gene located between the flanking DNA regions, the selectable or screenable marker gene further being located between a first repeat sequence consisting of the 5'-terminal part of the preselected site and a second sequence consisting of the 3' terminal part of the preselected site, whereby the  sequences  common between  the  first  and  second  repeat sequences are in direct repeat; and iii.   At least one recognition site for a second DSBI enzyme located between the one of the flanking DNA regions and the first and second repeat sequence; i.    Selecting a population of cells comprising the selectable or screenable<br>
marker; j.    Selecting a cell wherein the selectable or screenable marker has been introduced by homologous recombination through the flanking DNA regions;<br>
k.   Introducing a double stranded break at the recognition site for the second<br>
DSBI enzyme in the cell; 1.    Selecting a progeny cell wherein the selectable or screenable marker gene<br>
is deleted by homologous recombination between the direct repeats<br>
thereby recreating the preselected site.<br>
[55.] The terms herein defined with regard to their meaning in plant cells, can be applied mutatis mutandis to apply to eukaryotic cells, particularly higher eukaryotic cells such as vertebrates, animals, mammals in general.<br>
[56.] It will also be clear that the terms used to describe the method such as "introduction of a DNA fragment" as well as "regeneration of a plant from the cell" do not imply that such DNA fragment necessarily needs to be introduced by transformation techniques. Indeed, it will be immediately clear to the person skilled in the art that the DNA molecule of interest may also be introduced by breeding or crossing techniques from one plant to another.<br>
[57.] However, it will be clear that the DNA molecule of interest may be introduced into the plant cells by any method known in the art, including Agrobacterium mediated transformation but also by direct DNA transfer methods. The transforming DNA molecule can be transferred into plant cells using any conventional method, including but not limited to direct DNA transfer method. As used herein "direct DNA transfer" is any method of DNA introduction into plant cells which does not involve the use of natural Agrobacterium spp. and which is capable of introducing DNA into plant cells. This includes methods well known in the art such as introduction of DNA by electroporation into protoplasts, introduction of DNA by electroporation into intact plant cells or partially degraded tissues or plant cells, introduction of DNA through the action of agents such as PEG and the like, into protoplasts, use of silicon whiskers, and bombardment with DNA coated microprojectiles.<br>
[58.] The DNA may be integrated by homologous recombination or non-homologous end-joining methods involving a double stranded break induction at a preselected site as described e.g. in PCT/EP04/013122.<br>
[59.] "Selectable or screenable markers" as used herein have there usual meaning in the art and include, but are not limited to plant expressible phosphinotricin acetyltransferase, neomycine phosphotransferase, glyphosate oxidase, glyphosate tolerant EPSP enzyme, nitrilase gene, mutant acetolactate synthase or acetohydroxyacid synthase gene, fi-glucoronidase (GUS), R-locus genes, green fluorescent protein and the likes.<br>
[60.]    The selection of the plant cell or plant wherein the selectable or screenable marker<br>
recombination through the flanking DNA regions can e.g. be achieved by screening for the absence of sequences present in the transforming DNA but located outside of the flanking DNA regions. Indeed, presence of sequences from the transforming DNA outside the flanking DNA regions would indicate that the transformed plant cells origination by random DNA insertion. To this end, selectable or screenable markers may be included in the transforming DNA molecule outside of the flanking DNA regions, which can then be used to identify those plant cells which do not have the selectable or screenable markers located outside of the transforming DNA and which may have arisen by homologous recombination through the flanking DNA regions. Alternatively, the transforming DNA molecule may contain selectable markers outside the flanking DNA regions that allow selection for the absence of such genes (negative selectable marker genes).<br>
[61.] It will be appreciated that the means and methods of the invention may be used in any plant capable of reproduction through pollen or egg cells, including com, tobacco, cereal plants including wheat, oat, barley, rye, rice, turfgrass, sorghum, millet or sugarcane plants. The methods of the invention can also be applied to any plant (Angiospermae or Gymnospermae) including but not limited to cotton, canola, oilseed<br>
rape, soybean, vegetables, potatoes, Lemna spp., Nicotiana spp., Arabidopsis, alfalfa, barley, bean, com, cotton, flax, pea, rape, rice, rye, safflower, sorghum, soybean, sunflower, tobacco, wheat, asparagus, beet, broccoli, cabbage, carrot, cauliflower, celery, cucumber, eggplant, lettuce, onion, oilseed rape, pepper, potato, pumpkin, radish, spinach, squash, tomato, zucchini, almond, apple, apricot, banana, blackberry, blueberry, cacao, cherry, coconut, cranberry, date, grape, grapefruit, guava, kiwi, lemon, lime, mango, melon, nectarine, orange, papaya, passion fruit, peach, peanut, pear, pineapple, pistachio, plum, raspberry, strawberry, tangerine, walnut and watermelon.<br>
[62.] It is also an object of the invention to provide plant cells and plants generated according to the methods of the invention. Gametes, seeds, embryos, either zygotic or somatic, progeny or hybrids of plants comprising the DNA insertion events, which are produced by traditional breeding methods are also included within the scope of the present invention. Such plants may contain a heterologous DNA sequence instead of a target sequence, and will only be different from their progenitor plants by the presence of this heterologous DNA or DNA sequence post exchange.<br>
[63.] The plants obtained by the methods described herein may be further crossed by traditional breeding techniques with other plants to obtain progeny plants comprising the targeted DNA insertion events obtained according to the present invention.<br>
[64.] The following non-limiting Examples describe the removal of a selected subfragment from an introduced DNA molecule using a double strand DNA break inducing enzyme, such as I-Scel, and direct repeats which are subfragments of an I-Ceul recognition site.<br>
[65.] Unless stated otherwise in the Examples, all recombinant DNA techniques are carried out according to standard protocols as described in Sambrook et al. (1989) Molecular Cloning: A Laboratory Manual, Second Edition, Cold Spring Harbor Laboratory Press, NY and in Volumes 1 and 2 of Ausubel et al. (1994) Current Protocols in Molecular Biology, Current Protocols, USA. Standard materials and methods for plant<br>
molecular work are described in Plant Molecular Biology Labfax (1993) by R.D.D. Croy, jointly published by BIOS Scientific Publications Ltd (UK) and Blackwell Scientific Publications, UK. Other references for standard molecular biology techniques include Sambrook and Russell (2001) Molecular Cloning: A Laboratory Manual, Third Edition, Cold Spring Harbor Laboratory Press, NY, Volumes I and II of Brown (1998) Molecular Biology LabFax, Second Edition, Academic Press (UK). Standard materials and methods for polymerase chain reactions can be found in Dieffenbach and Dveksler (1995) PCR Primer: A Laboratory Manual, Cold Spring Harbor Laboratory Press, and in McPherson at al. (2000) PCR - Basics: From Background to Bench, First Edition, Springer Verlag, Germany.<br>
sequences:<br>
[67.] SEQ ID No 1: nucleotide sequence of synthetic I-Scel coding region (UIPAC code).<br>
[68.]    SEQ ID No 2: nucleotide sequence of synthetic I-Scel coding region.<br>
[69.]    SEQ ID No 3: nucleotide sequence of the I-Scel recognition site<br>
[70.]    SEQ ID No 4: nucleotide sequence of the I-Ceul recognition site<br>
EXAMPLES<br>
[71.] Using conventional recombinant DNA techniques a T-DNA vector is constructed comprising the following operably linked DNA fragments:<br>
•	a CaMV 35S promoter region<br>
•	a DNA region encoding the non-functional N-terminal part of screenable marker (e.g. ß-glucuronidase)<br>
•	a DNA sequence consisting of the 5'terminal region of 22 nucleotides of the I-Ceul recognition site (SEQ ID No 4)<br>
•	a selectable marker gene (such as a plant-expressible phosphinotricin acetyltransferase)<br>
•	a I-Scel recognition site (SEQ ID No 3)<br>
•	a DNA sequence consisting of the 3'terminal region of 22 nucleotides of the I-Ceul recognition site<br>
•	a DNA region encoding the non-functional C-terminal part of the screenable marker (such that upon homologous recombination between the two I-Ceul derived repeat sequences a functional coding region for the screenable marker is generated).<br>
•	a 3'end region involved in transcription termination and polyadenylation.<br>
[72.] The T-DNA is introduced in a tobacco plant cells, and transgenic plants are regenerated as conventional in the art. These transgenic tobacco plant lines can be used conveniently to determine whether homologous recombination occurs between the two subparts of the I-Ceul recognition site, resulting in a removal of the selectable marker gene and generation of an intact screenable marker.<br>
[73.] Transgenic tobacco plants expressing I-Scel under control of the microspore specific promoter of the NTM19 linked to a neomycin resistance gene have been described in WO2006/105946. In a similar manner, transgenic tobacco plants expressing<br>
I-Ceul under control of the microspore specific promoter of the NTM19 gene are generated.<br>
[74.] Transgenic plants containing the test-construct are crossed with transgenic tobacco plants expressing I-Scel under control of the microspore specific promoter of the NTM19 or with transgenic tobacco plants expressing I-Ceul under control of the microspore specific promoter of the NTM19, and progeny plants comprising both the test-construct and the chimeric I-Scel or I-Ceul region are identified. These plants are used as pollen donor to pollinate a non-transgenic plant. In the<br>
[75.] Progeny plants from the initial cross between transgenic plants containing the test-coiisiiuct and I-Ceul expressing pi an is do noi show an increased frequency of homologous recombination, whereas progeny plants from the initial cross between transgenic plants containing the test-construct and I-Scel expressing plants exhibit an increased frequency of homologous recombination.<br>
[76.] To test whether homologous recombination can occur between short DNA sequences of 16 identical nucleotides in length, a T-DNA vector has been constructed which contains right and left T-DNA border sequences with in between the T-DNA borders a selectable chimeric gene comprising a nopaline synthase promoter, operably linked to a nptll coding region and a terminator region from the nopaline synthase gene and a further chimeric construct comprising in order:<br>
a.	a constitutive promoter region<br>
b.	a nucleotide sequence of 16 nucleotides<br>
c.	a recognition site for 1-SceI<br>
d.	a DNA region encoding green fluorescent protein (GFP)<br>
e.	a recognition site for I-Scel in inverted orientation with regard to the I-<br>
Scel site sub c)<br>
f.	the same nucleotide sequence of 16 nucleotides as sub b)<br>
g.	a DNA region encoding (J-glucuronidase (GUS)<br>
h.   a terminator region from the nopaline synthase gene<br>
[77.] Transgenic tobacco plants have been generated by leaf-disk transformation using Agrobacterium tumefaciens (EHA105) comprising the above described T-DNA vector whereby selection was performed for kanamycin resistant plant cells.<br>
[78.] As expected, no GUS expression was observed in leaves of transgenic plants, and four different transgenic TO lines with a clear GFP expression and no GUS expression were selected.<br>
[79.] Leaf disks of the selected transgenic lines where subjected to Agrobacterium mediated transformation using a Agrobacterium strain comprising a T-DNA vector comprising a chimeric I-Scel coding region under control of a plant-expressible promoter and a selectable gene encoding phosphinotricin-resistance between the T-DNA borders. Selection is performed for phosphinotricin resistant plant cells.<br>
[80.] From the selected plant cells, transgenic calli are obtained and plants are regenerated. Plant tissue and/or callus tissue are screened for GFP and GUS expression. In plants where a homologous recombination has occurred between the direct repeat sequence of 16 nucleotides, the coding region for GFP is deleted and the coding region for GUS is placed under control of the constitutive promoter. The plant material is accordingly negative for GFP expression and positive for GUS expression. From GFP negative, GUS positive plant material, the DNA fragment downstream of the constitutive promoter is amplified by PCR and the nucleotide sequence determined for determination whether homologous recombination occurred through the 16 nt direct repeat.<br>
<br>
<br>
<br>
<br>
<br>
<br>
<br>
Claims<br>
1) A method for exchanging a target DNA sequence in the genome a eukaryotic cell or eukaryotic organism for a DNA sequence of interest comprising the following steps:<br>
a.	Inducing a first double stranded DNA break at a preselected site in the<br>
genome of a cell of said eukaryotic organism, said preselected site being<br>
located within said target DNA sequence or in the vicinity of said target<br>
DNA sequence and said preselected site being recognized by a first<br>
double-stranded break inducing (DSBI) enzyme;<br>
b.	introducing a repair DNA moiecuie into said eukaryoiic ceii, said repair<br>
DNA molecule comprising<br>
i. Said DNA sequence of interest located between two flanking DNA regions having at least 80 % sequence homology to a DNA region flanking said target DNA sequence, and preferably flanking said preselected site in the genome of said eukaryotic cell;<br>
ii. A selectable or screenable marker gene located between said flanking DNA regions, said selectable or screenable marker gene further being located between a first repeat sequence consisting of the 5'-terminal part of said preselected site and a second sequence consisting of the 3' terminal part of said preselected site, whereby the sequences common between said first and second repeat sequences are in direct repeat; and<br>
iii. At least one recognition site for a second DSBI enzyme located between said one of the flanking DNA regions and said first and second repeat sequence;<br>
c.	Selecting a population of cells comprising said selectable or screenable<br>
marker;<br>
d.	Selecting a cell wherein said selectable or screenable marker has been<br>
introduced by homologous recombination through said flanking DNA<br>
regions;<br>
e.	Introducing a double stranded break at the recognition site for said second<br>
DSBI enzyme in said cell;<br>
f.	Selecting a progeny cell wherein said selectable or screenable marker gene<br>
is deleted by homologous recombination between said direct repeats<br>
thereby recreating said preselected site.<br>
2)	The method of claim 1, wherein said first double stranded break at said preselected site is induced by introduction of a first DSBI inducing enzyme, said first DSBI inducing enzyme not recognizing said recognition site for said second DSBI inducing enzyme located in said repair DNA.<br>
3)	The method of claim 1, wherein said first DSBI enzyme and said second DSBI enzyme are two different DSBI enzymes selected from the group of I-Sce I, I-Chu I, I-Dmo I, I-Cre I, I-Csm I, PI-Fli I, Pt-Mtu I, I-Ceu I, I-Sce II, I-Sce III, HO, Pi-Civ I, Pl-Ctr I, PI-Aae I, PI-BSU I, PI-Dhal, PI-Dra I, PI-Mav I, PI-Mch I, PI-Mfu I, PI-Mfl I, PI-Mga I, PI-Mgo I, PI-Min I, PI-Mka I, PI-Mle I, PI-Mma I, PI-Msh I, PI-Msm I, PI-Mth I, PI-Mtu I, PI-Mxe I, PI-Npu I, Pl-Pfu I, PI-Rma I, Pl-Spb I, PI-Ssp I, PI-Fac I, PI-Mja I, Pl-Pho I, PI-Tag I, PI-Thy I, PI-Tko I or PI-Tsp I or a chimeric endonuclease comprising a Zn finger DNA binding domain and a DNA cleavage domain or a custom meganuclease.<br>
4)	The method of claim 1, wherein said first DSBI enzyme is a custom meganuclease recognizing said preselected site.<br>
5)	The method of any one of claims 1 to 4, wherein said second DSB inducing enzyme tion site for a DSBI inducing enzyme located in said DNA of interest is I-Scel.<br>
6)	The method of claim 5, wherein said DNA region encoding said double stranded DNA break inducing enzyme comprises the nucleotide sequence of SEQ ID No 1 or SEQ ID No 2.<br>
7)	The method of any one of claims 1 to 6, wherein said eukaryotic cell is a plant cell or said eukaryotic organism is a plant.<br>
g 9) A method for exchanging a target DNA sequence in the genome, particularly the nuclear genome, of a plant for a DNA sequence of interest comprising the following steps:<br>
a)	Inducing a first double stranded DNA break at a preselected site in the genome of a cell of said plant, said preselected site being located within said target DNA sequence or in the vicinity of said target DNA sequence and said preselected site being recognized by a first double-stranded break inducing (DSBI) enzyme<br>
b)	Introducing a repair DNA molecule into said eukaryotic cell, said repair DNA molecule comprising<br>
i) Said DNA sequence of interest located between two flanking DNA regions having at least 80 % sequence homology to a DNA region flanking said target DNA sequence, and preferably flanking said preselected site in the genome of said eukaryotic cell;<br>
ii) A selectable or screenable marker gene located between said flanking DNA regions, said selectable or screenable marker gene further being located between a first repeat sequence consisting of the 5'-terminal part of said preselected site and a second sequence consisting of the 3' terminal part of said preselected site, whereby the sequences common between said first and second repeat sequences are in direct repeat; and<br>
iii) At least one recognition site for a second DSBI enzyme located between said one of the flanking DNA regions and said first and second repeat sequence;<br>
c)	Selecting a population of plant cells comprising the selectable or screenable marker;<br>
d)	Selecting a plant cell wherein the DNA sequence of interest (and the selectable or screenable marker) has been introduced by homologous recombination through the flanking DNA regions, and regenerating a plant from the plant cell;<br>
e)	Crossing the regenerated plant or a progeny plant thereof comprising the selectable marker gene with a plant comprising a rare cleaving double stranded break inducing ("DSBI") enzyme encoding chimeric gene, the chimeric gene comprising the following operably linked DNA segments:<br>
vii.   a germline specific promoter;<br>
viii.    a DNA region encoding a double stranded DNA break inducing enzyme recognizing the recognition site located in the DNA of interest; ix.   a transcription termination and polyadenylation region;<br>
f)	Selecting a progeny plant (Fl-plant) comprising the selectable or screenable marker gene and the DSBI enzyme encoding chimeric gene;<br>
g)	Crossing the progeny plant with another plant whereby the progeny plant is used as pollen donor if said germline specific promoter is a microsporespecific promoter and whereby said progeny plant is used a pollen acceptor or female plant if said germline specific promoter is a megaspore specific promoter;<br>
h)  Selecting a population of progeny plants (F2-population) which comprises the<br>
DSBI enzyme encoding chimeric gene; and i)   Selecting a progeny plant wherein the selectable or screenable marker gene is<br>
deleted by homologous recombination between the first and second direct repeat<br>
sequence.<br>
 9.   The method of claim 9; wherein said first double stranded break at said preselected<br>
site is induced by introduction of a first DSBI inducing enzyme, said first DSBI<br>
inducing enzyme not recognizing said recognition site for said second DSBI inducing<br>
enzyme located in said repair DNA.<br>
10) The method of claim 9, wherein said first DSBI enzyme and said second DSBI enzyme are two different DSBI enzymes selected from the group of I-Sce I, I-Chu I, I-Dmo I, I-Cre I, I-Csm I, PI-Fli I, Pt-Mtu I, I-Ceu I, I-Sce H, I-Sce III, HO, Pi-Civ I, Pl-Ctr I, PI-Aae I, PI-BSU I, PI-Dhal, PI-Dra I, PI-Mav I, PI-Mch I, PI-Mfu I, PI-Mfl I, PI-Mga I, PI-Mgo I, PI-Min I, PI-Mka I, PI-Mle I, PI-Mma I, PI-Msh I, PI-Msm I, PI-Mth I, PI-Mtu I, PI-Mxe I, PI-Npu I, Pl-Pfu I, PI-Rma I, Pl-Spb I, PI-Ssp I, PI-Fac I, PI-Mja I, Pl-Pho I, Pi-Tag I, PI-Thy I, PI-Tko I or PI-Tsp I or a chimeric endonuclease comprising a Zn finger DNA binding domain and a DNA cleavage domain or a custom meganuclease.<br>
11)The method of claim 9% wherein said first DSBI enzyme is a custom meganuclease<br>
12) The method of any one of claims 9 to 12; wherein said second DSB inducing enzyme tion site for a DSBI inducing enzyme located in said DNA of interest is I-Scel.<br>
13) A DNA vector for exchanging a target DNA sequence in the genome of a plant cell for a DNA sequence of interest through the induction of a double stranded break at a preselected site within said target sequence or in the vicinity thereof, said DNA vector comprising<br>
a.	said DNA sequence of interest located between two flanking DNA regions<br>
having at least 80 % sequence homology to a DNA region flanking said<br>
target DNA sequence and flanking said preselected site;<br>
b.	a selectable or screenable marker gene located between said flanking<br>
DNA regions, said selectable or screenable marker gene further being<br>
located between a first repeat sequence consisting of the 5'-terminal part<br>
of said preselected site and a second sequence consisting of the 3' terminal<br>
part of said preselected site, whereby the sequences common between said<br>
first and second repeat sequences are in direct repeat; and<br>
c. a recognition site for a DSBI enzyme located between said one of the flanking DNA regions and said partial flanking DNA region located in direct repeat.-<br>
</td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/viewdoc.aspx?id=63wrxLfFvSn3rmRA+juXCg==&amp;amp;loc=+mN2fYxnTC4l0fUd8W4CAA==" target="_blank" style="word-wrap:break-word;">http://ipindiaonline.gov.in/patentsearch/GrantedSearch/viewdoc.aspx?id=63wrxLfFvSn3rmRA+juXCg==&amp;amp;loc=+mN2fYxnTC4l0fUd8W4CAA==</a></p>
		<br>
		<div class="pull-left">
			<a href="279821-lifting-member-with-load-and-or-stress-measuring-means.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="279823-method-of-manufacturing-carbon-rich-product-and-co-products.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>279822</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>7695/DELNP/2009</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>05/2017</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>03-Feb-2017</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>31-Jan-2017</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>26-Nov-2009</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>BAYER CROPSCIENCE NV</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td>J.E. MOMMAERTSLAAN 14, 1831 DIEGEM, BELGIUM</td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>ANNE ROLLAND</td>
											<td>28, CHEMIN DE LA CREUZETTE, FR-69270 FONTAINE SUR SAONE, FRANCE</td>
										</tr>
										<tr>
											<td>2</td>
											<td>MANUEL DUBALD</td>
											<td>59, CHEMIN DU COLIN, FR-69370 SAINT DIDIER AU MONT D&#x27;OR, FRANCE</td>
										</tr>
										<tr>
											<td>3</td>
											<td>MICHIEL VAN LOOKEREN CAMPAGNE</td>
											<td>78004 STOKES, NC 27517 CHAPEL HILL, USA</td>
										</tr>
										<tr>
											<td>4</td>
											<td>RENE RUITER</td>
											<td>BLAUWE STEENSTRAAT 72, BE-9070 HEUSDEN (DESTELBERGEN), BELGIUM</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>C12N 15/82</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>PCT/EP08/004524</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td>2008-06-03</td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td>60/933,814</td>
									<td>2007-06-08</td>
								    <td>EUROPEAN UNION</td>
								</tr>
								<tr>
									<td>2</td>
									<td>07010998.8</td>
									<td>2007-06-05</td>
								    <td>EUROPEAN UNION</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/279822-a-method-for-exchanging-a-target-dna-sequence-in-the-genome-of-a-eukaryotic-cell by HTTrack Website Copier/3.x [XR&CO'2014], Thu, 04 Apr 2024 23:02:39 GMT -->
</html>
