<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/279826-a-method-for-the-preparation-of-an-evolved-strain-of-micro-organism-for-the-production-of-1-2-propandiol-obtained-by-a-combination-of-evolution-and-rational-design by HTTrack Website Copier/3.x [XR&CO'2014], Thu, 04 Apr 2024 23:01:22 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 279826:A METHOD FOR THE PREPARATION OF AN EVOLVED STRAIN OF MICRO-ORGANISM FOR THE PRODUCTION OF 1, 2-PROPANDIOL OBTAINED BY A COMBINATION OF EVOLUTION AND RATIONAL DESIGN</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">A METHOD FOR THE PREPARATION OF AN EVOLVED STRAIN OF MICRO-ORGANISM FOR THE PRODUCTION OF 1, 2-PROPANDIOL OBTAINED BY A COMBINATION OF EVOLUTION AND RATIONAL DESIGN</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>The present invention concerns a new method combining evolution and rational design for the preparation of a strain of micro-organism for the production of 1,2- propanediol from a carbon source. The said method comprises: growing an initial strain under selection pressure in an appropriate growth medium, said initial bacterial strain comprising an attenuation of the expression of the tpiA gene and an attenuation the expression of at least one gene involved in the conversion of methylglyoxal to lactate, in order to promote evolution in said initial strain,then selecting and isolating the evolved strain having an increased 1,2 propanediol production rate,then reconstructing a functional tpiA gene in the evolved strain; The present invention also concerns the evolved strain such as obtained, that may be furthermore genetically modified in order to optimize the conversion of a carbon source into ,2-propanediol without by-products and with the best possible yield.</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td>The present invention concems a new method combining evolution and rational design for the preparation of a micro-organism to produce 1,2-propanediol, the micro¬organism thereby obtained and its use for the preparation of 1,2-propanediol.<br>
1,2-propanediol or propylene glycol, a C3 dialcohol, is a widely-used chemical. It is a component of unsaturated polyester resins, liquid detergents, coolants, anti-freeze and de-icing fluids for aircraft. Propylene glycol has been increasingly used since 1993-1994 as a replacement for ethylene derivatives, which are recognised as being more toxic than propylene derivatives.<br>
1,2-propanediol is currently produced by chemical means using a propylene oxide hydration process that consumes large amounts of water. Propylene oxide can be produced by either of two processes, one using epichlorhydrin, and the other hydroperoxide. Both routes use highly toxic substances. In addition, the hydroperoxide route generates by¬products such as tert-hutanol and 1-phenyl ethanol. For the production of propylene to be profitable, a use must be found for these by-products. The chemical route generally produces racemic 1,2-propanediol, whereas each of the two stereoisomers (/?) 1,2-propanediol and (5) 1,2-propanediol are of interest for certain applications (e.g. chiral starting materials for specialty chemicals and pharmaceutical products)..<br>
The disadvantages of the chemical processes for the production of 1,2-propanediol make biological synthesis an attractive alternative. Two routes have been characterized for the natural production of 1,2-propanediol fi-om sugars by microorganisms.<br>
In the first route 6-deoxy sugars (e.g. L-rhamnose or L-fiicose) are cleaved into dihydroxyacetone phosphate and (S)-lactaldehyde, which can be fiirther reduced to (S)-1,2-propanediol (Badia et al, 1985). This route is fiinctional in E. coli, but can not yield an economically feasible process due to the elevated cost of the deoxyhexoses.<br>
The second route is the metabolism of conmion sugars (e.g. glucose or xylose) through the glycolysis pathway followed by the methylglyoxal pathway. Dihydroxyacetone phosphate is converted to methylglyoxal that can be reduced either to lactaldehyde or to acetol. These two compounds can then imdergo a second reduction reaction yielding 1,2-propanediol. This route is used by natural producers of (R)-1,2-propanediol, such as Clostridium sphenoides and Thermoanaerobacter thermosaccharolyticum. Clostridium sphenoides has been used to produce 1,2-propanediol at a titer of 1,58 g/1 under phosphate limited conditions (Tran Din and Gottschalk, 1985). Thermoanaerobacter thermosaccharolyticum has also been investigated for the production of 1,2-propanediol (Cameron and Cooney, 1986, Sanchez-Rivera et al, 1987). The best performances obtained<br>
2<br>
 <br>
were a titer of 9 g/1 and a yield from glucose of 0,2 g/g. However, the improvement of the performances obtained with these organisms is likely to be limited due to the shortage of available genetic tools.<br>
PRIOR ART<br>
Cameron et al (1998) have investigated the use ofE. coli as a platform for metabolic engineering for the conversion of sugars to 1,2-propanediol. Their theoretical analysis showed that the upper limit of the realistic product yield (considering mass balances and production of energy for growth) is significantly different depending on the culture conditions. Under anaerobic conditions, acetate will be produced as a by-product in order to recycle the reduced co-factors and the best yield shall be limited to 1 mole of 1,2-propanediol per mole of glucose (0,42 g/g). Under aerobic conditions, recycling of co-factors shall be ensured by the respiratory chain using oxygen as terminal electron acceptor and it could become possible to produce 1,2-propanediol without the production of by¬products. Under these conditions, yield could reach at best 1.42 mol/mol (0,6 g/g). Considering- the maximum titer of 1,2-propanediol, Cameron et al discussed its dependence on product and by-product toxicity. 1,2-propanediol is significantly less toxic than 1,3-propanediol and E. coli exhibits a residual growth rate of 0.5 h' with 100 g/1 1,2-propanediol. The inhibition of growth is more likely to be due to the by-product acetate that is known to be highly growth inhibiting. Development of an anaerobic process for the production of 1,2-propanediol with high titers and yields will have to address the acetate issue. Conversion of acetate into acetone, which is less inhibitory and easily removed in situ has been proposed (WO 2005/073364).<br>
Several investigations for genetic modifications of E. coli in order to obtain a 1,2-propanediol producer using simple carbon sources have been done by the group of Cameron (Cameron et al, 1998, Altaras and Cameron, 1999, Altaras and Cameron, 2000) and the group of Bennett (Huang et al, 1999, Berrios-Rivera et al, 2003). These studies rely on the one hand on the expression of one or several enzymatic activities in the pathway from dihydroxyacetone phosphate to 1,2-propanediol and on the other hand on the removal of NADH and carbon consuming pathways in the host strain. The best results obtained by the group of Cameron are production of 1.4 g/11,2-propanediol in anaerobic flask culture with a yield of 0.2 g/ g of glucose consumed. When extrapolated to an anaerobic fed-batch fermenter, the production was 4.5 g/1 1,2-propanediol with a yield of 0.19 g/g from glucose, far from the theoretical evaluation of Cameron et al.. These performances have been<br>
3<br>
 <br>
obtained with the overexpression of the methylglyoxal synthase gene of E. coli (mgs), the glycerol dehydrogenase gene ofE. coli (gldA) and the 1,2-propanediol oxidoreductase gene of E. coli (fucO) in a strain lacking the gene coding for lactate dehydrogenase (IdhA). Results obtained with the same approach but with lower titers and yields are also described in the patents US 6,087,140, US 6,303,352 and WO 98/37204.<br>
The group of Bennett also used an E. coli host strain lacking IdhA for the overexpression of the mgs gene from Clostridium acetobutylicum and the gldA gene from E. coli. Flask cultures under anaerobic conditions gave a titer of 1.3 g/1 and a yield of 0.12 g/g whereas microaerobic cultures gave a titer of 1.4 g/1 with a yield of 0.13 g/g.<br>
An alternative method to obtain a sfrain producing 1,2-propanediol is to direct the evolution of an "initial sfrain" towards a state where the "evolved strain" produces the desired compound with better characteristics. This method is based on the natural evolution of a microorganism which is first modified by attenuation of two genes, tpiA and one gene involved in the conversion of methylglyoxal into lactate. The purpose for attenuating the tpiA gene coding for triose phosphate isomerase is to separate the two metabolic branches starting at glyceraldehyde-3-phosphate (GA3P) and dihydroxyacetone phosphate (DHAP) that are normally interconverted by this enzyme. The pathway from DHPA to 1,2-propanediol will be the "reducing branch" consuming reduced co-factors (NADH), whereas the metabolism from GA3P to acetate will be the "oxidative branch" producing NADH and energy for the growth of the cell. Without a ftmctional tpiA gene, the metabolism of the cell is "locked" and the growth of the sfrain, the production of 1,2-propanediol and the production of acetate are tightly coupled. Under selection pressure in an appropriate growth medium, this initial sfrain will evolve to a state where the production of 1,2-propanediol by said strain is improved. This procedure to obtain an "evolved strain" of micro-organism for the production of 1,2-propanediol is described in the patent application WO 2005/073364. This evolution process and the following step of fermentation are preferentially performed under anaerobic conditions. This technology is a clear improvement over the prior art. A 1,2-propanediol titer of 1.8 g/1 was obtained, with a yield of 0.35 gram per gram of glucose consumed, close to the theoretical result of Cameron et al.<br>
The object of the present invention is the improvement of an 1,2-propanediol produca: sfrain by evolution and subsequent rational genetic engineering of the evolved strain. A special feature is the reconstruction of a ftinctional tpiA gene in the evolved tpiA minus strain. These modifications lead to an improved production of 1,2-propanediol.<br>
DESCRIPTION OF THE INVENTION<br>
 <br>
The present invention concerns a new method combining evolution and rational design for the preparation of a strain of micro-organism for the production of 1,2-propanediol from a carbon source. The said method comprises:<br>
growing an initial sfrain under selection pressure in an appropriate growth medium, said initial bacterial strain comprising an attenuation of the expression of the tpiA gene and an attenuation the expression of at least one gene involved in the conversion of methylglyoxal to lactate (such as gloA, aldA, aldB), in order to promote evolution in said initial sfrain, then selecting and isolating the evolved sfrain having an increased 1,2 propanediol production rate (increased by at least 20%), then reconstructing a fimctional tpiA gene in the evolved strain;<br>
hi one aspect of the invention, the synthesis of unwanted by-products is attenuated by deleting the genes coding for enzymes involved in synthesis of lactate from pyruvate (IdhA), formate (pflA,pflB), ethanol (adhE). In another aspect of the invention, the Entner-Doudorofif pathway is eliminated by deleting either the edd or eda gene or both. Advantageously, in order to make more NADH available for the reduction steps in the biosynthesis pathway of 1,2-propanediol, at least one gene selected among arc A and ndh is attenuated.<br>
The microorganism used for the preparation of 1,2-propanediol is selected among bacteria, yeasts and ftmgi, but is preferentially from the species Escherichia coli or Clostridium acetobutylicum.<br>
The present invention also concerns the evolved strain such as obtained, that may be furthermore genetically modified in order to optimize the conversion of a carbon source into 1,2-propanediol without by-products and with the best possible yield. In one aspect of the invention, the glyceraldehyde 3 phosphate dehydrogenase activity is reduced in order to redirect a part of the available glyceraldehyde 3 phosphate toward the synthesis of 1,2-propanediol. In another aspect of the invention, the efficiency of the sugar import is increased, either by using a sugar import independent of phosphoenolpyruvate (PEP) like the one encoded by gat?, or by providing more PEP to the sugar-phosphofransferase system. This is obtained by eliminating the pathways consuming PEP like pyruvates kinases (encoded by the pykA and pykV genes) and/or by promoting the synthesis of PEP e. g. by overexpressing the ppsA gene coding for PEP synthase. Additionally, it is valuable for the enzyme converting pyruvate into acetyl-coA to be resistant to high concenfrations of NADH<br>
5<br>
 <br>
found under anaerobic conditions. This can be obtained by a specific mutation in the Ipd gene. Advantageously, the synthesis of the by-product acetate is prevented by attenuating one or several of the genes ackA, pta, poxB.<br>
This invention is also related to a method for the production of 1,2-propanediol at an optimal yield, under aerobic, microaerobic or anaerobic conditions, using said evolved and optionally genetically modified strain of E. coli grown in an appropriate growth medium containing a simple carbon source. Additionally, the invention is related to a method for the production of 1,2-propanediol at an optimal yield, under anaerobic conditions, using said evolved and optionally genetically modified strain of C. acetobutylicum grown in an appropriate growth medium containing a simple or a complex carbon source. The produced 1,2 propanediol according to this method is subsequently recovered and optionally purified. BRIEF DESCRIPTION OF THE DRAWINGS<br>
The accompanying drawing that is incorporated in and constitutes a part of this specification exemplifies the invention and together with the description, serves to explain the principles of this invention.<br>
Figure I depicts the genetic engineering of central metabolism in the development of a 1,2-propanediol production system firom carbohydrates.<br>
DETAILED DESCRIPTION OF THE INVENTION<br>
As used herein the following terms may be used for interpretation of the claims and specification.<br>
The term 'strain' denotes a genetic variant of a species. Thus the term 'strain of microorganism' denotes a genetic variant of a species of a specific microorganism. The characteritics given for any strain apply also for the corresponding microorganism or vice versa.<br>
According to the invention the terms 'culture', 'growth' and 'fermentation' are used interchangeably to denote the growth of bacteria in an appropriate growth medium containing a simple carbon source.<br>
The term 'carbon source' according to the present invention denotes any source of carbon that can be used by those skilled in the art to support the normal growth of a micro¬organism, and which can be hexoses, pentoses, monosaccharides, disaccharaides, oligosaccharides, starch or its derivatives, hemicelluloses, glycerol and combinations thereof<br>
 <br>
The term 'appropriate growth medium' according to the invention denotes a medium of known molecular composition adapted to the growth of the micro-organism and designed in such a way that it promotes the wanted evolution.<br>
The evolution process according to the invention is a process for the preparation of evolved micro-organisms presenting improved production characteristics, and comprises the following steps:<br>
a)	Modification of a micro-organism to obtain an initial strain with a "locked" metabolism where the evolution can only take the desired direction when the cells of the initial strain are grown on an appropriate medium,<br>
b)	Growth of the initial strain obtained above on said appropriate medium in order to cause it to evolve, wherein the initial strain is grown under aerobic, micro-aerobic or anaerobic conditions,<br>
c)	Selection of the "evolved strains" able to grow imder these specific conditions, presenting improved production characteristics for the desired compoimd.<br>
This evolution process has been extensively described in the patent applications WO 2004/076659 filed on 17/02/2004, and WO 2005/073364 filed on 12/01/2005, by the same applicants.<br>
The term 'selection' according to the invention denotes a process wherein the only strains of microorganisms that are retained in the culture medium are those presenting a better fitness imder the selection pressure conditions. Typically, the fittest strains are outgrowing their competitors and are then selected. A simple way to select a specific evolved strain of microorganism in a population consists in growing the population in continuous culture in which slow-growing strains are eventually eluted firom the culture. This is not an exclusive example for selection, and other methods of selection known by the expert in the field may be applied.<br>
The term 'isolation' denotes a process where an individual strain presenting specific genetic modifications is separated fi-om a population of strains presenting different genetic characteristics. This is done by sampling the biomass after the period of evolution and spreading it on Petri dishes to isolate single colonies.<br>
The term "1,2-propanediol production rate" means a production rate expressed in g/l/h, that is calculated as follows:<br>
Concentration of 1.2-propanediol produced in the medium fg/n / time necessary for this production (hour)<br>
 <br>
Additionally, a specific production rate expressed in g/g/h, taking into account the quantity of biomass can be calculated as follows:<br>
Concentration of 1.2-propanediol produced in the medium (e/i) I concentration of biomass produced in the medium (g/1) / time necessary for these productions fh^<br>
The concentration of biomass is determined either by measuring the absorbance of the fermentation broth with a spectrophotometer reading for example at 600 ran or by determining the dry weight of cells after drying a defined volvime of fermentation broth.<br>
The quantity of 1,2-propanediol produced is measured by high performance liquid chromatography (HPLC) with an adapted column according to a state of the art protocol.<br>
In the present invention, evolved strains are selected for the following characteristics : an increased glucose uptake rate and an improved 1,2 propanediol production rate. The strains showing these characteristics are then isolated, and advantageously compared to each other, in the way to identify the best producer.<br>
The glucose uptake rate, expressed in g/l/h is calculated as follow : Concentration of glucose consumed by the culture (g/1) / time necessary for this consumption (h)<br>
A specific glucose uptake rate can be calculated by taking into account the concentration of biomass in the medium, as previously described.<br>
Glucose uptake rate and 1,2 propanediol production rate are intimately linked. If the consumption of glucose is increased, the production of the products from the glucose metabolism is increased in the same proportion.<br>
After selection and isolation, the best evolved strains present a glucose uptake that is about 20% higher than the uptake of the initial strain, preferentially about 30% higher or more, more preferentially 50% higher.<br>
The increased 1,2 propanediol production rate is of about 20% higher than the production rate of the initial strain, preferentially about 30% higher or more, more preferentially about 50% higher.<br>
The tpiA gene encodes the enzyme 'triose phosphate isomerase', which catalyses the interconversion of DHAP and GA3P (see figure 1). The pxupose of the attenuation of this gene is to engineer the metabolism of the cell in such a way that the evolution toward the most efficient 1,2-propanediol production becomes possible.<br>
The term 'attenuation of the expression of a gene' according to the invention denotes the partial or complete suppression of the expression of a gene, which is then said to be 'attenuated'. This suppression of expression can be either an inhibition of the<br>
8<br>
 <br>
expression of the gene, the suppression of an activating mechanism of the gene, a deletion of all or part of the promoter region necessary for the gene expression, or a deletion in the coding region of the gene. Preferentially, the attenuation of a gene is essentially the complete deletion of that gene, which gene can be replaced by a selection marker gene that facilitates the identification, isolation and purification of the strains according to the invention. A gene is preferentially inactivated by the technique of homologous recombination as described in Datsenko, K.A. &amp; Wanner, B.L. (2000) "One-step inactivation of chromosomal genes in Escherichia coli K-12 using PCR products". Proc. Natl. Acad. Sci. USA 97: 6640-6645. Other methods are described below.<br>
The term "expression" refers to the transcription and translation of a gene sequence leading to the generation of the corresponding protein, product of the gene.<br>
The term "reconstructing a fiinctional jfp/A gene in the evolved strain" means that the selected evolved strain is modified after the process of evolution by introducing a fiinctional tpiA gene; this can be accomplished by replacing via homologous recombination the attenuated copy of the gene by a wild-type fiinctional copy, thus restoring a triose phosphate isomerase activity similar to the activity measured in the initial strain, or by the introduction of a fimctional tpiA gene on a different chromosomal locus or by introducing a fiinctional tpih gene on a plasmid. This restoration can allow a yield of 1,2-propanediol production from glucose greater than 1 mole/mole by partly recycling GA3P into DHAP for the production of 1,2-propanediol through the action of triose phosphate isomerase.<br>
The purpose of the attenuation of the expression of at least one gene involved in the conversion of methylglyoxal (2-oxo propanal) into lactate is to inhibit the conversion of methylglyoxal into lactate, so that the methylglyoxal present is used by the cell machinery essentially for the synthesis of 1,2-propanediol. Genes involved in the conversion of methylglyoxal into lactate are in particular:<br>
-    a gene coding for a glyoxalase, for example the gloh gene coding for glyoxalase I, catalysing the synthesis of lactoyl glutathione from methylglyoxal; the aldA and aldB genes coding for a lactaldehyde dehydrogenase (catalysing the synthesis of (5) lactate from (5) lactaldehyde).<br>
The expression of one or more of these genes is advantageously attenuated (or the gene is completely deleted) in the initial strain. Preferentially the gloA gene is deleted.<br>
An additional modification is advantageously made to the initial strain consisting in suppressing the natural glucose fermentation routes, which consume reducing equivalents as NADH and therefore compete with 1,2-propanediol biosynthesis pathway.<br>
9<br>
 <br>
In particular, it is advantageous to attenuate the expression of the gene IdhA coding for lactate dehydrogenase catalysing the synthesis of lactate fi-om pyruvate, and the expression of the gene adhE coding for alcohol-aldehyde dehydrogenase catalysing the synthesis of ethanol from acetyl-CoA.<br>
Similarly, it is possible to force the micro-organism to use the pyruvate dehydrogenase complex to produce acetyl-CoA and NADH from pyruvate. This can be achieved by attenuating the expression of genes pflA and pflB coding for pyruvate formate lyase.<br>
Attenuation of at least one of the genes edd and eda coding for the enzymes involved in the Entner-Doudoroff pathway, is also usefiil to prevent the direct metabolism of glucose into glyceraldehyde-3-phosphate and pyruvate that can bypass the 1,2-propanediol synthesis pathway.<br>
Under anaerobic or microaerobic conditions, availability of NADH for the reduction of the precursors into 1,2-propanediol is advantageously increased. This is obtained by alleviating the repression on the tricarboxylic acid cycle mediated by the global regulator ArcA (encoded by the arcA gene). NADH concentration in the cell can also be increased by inactivating the NADH dehydrogenase II encoded by the gene ndh. Therefore, preferably, at least one gene selected among arc A and ndh has its expression attenuated.<br>
Preferentially, the initial strain is selected from the group consisting of bacteria, yeasts and fimgi.<br>
More preferentially, the initial strain is selected from the group consisting of Enterobacteriaceae, Bacillaceae, Clostridiaceae, Streptomycetaceae and Corynebacteriaceae.<br>
In a preferred embodiment of the invention, the initial strain is either Escherichia coli or Clostridium acetobutylicum.<br>
The evolved sfrain susceptible to be obtained, and the evolved sfrain such as obtained by the process previously described, is also an object of the invention.<br>
In this evolved sfrain, it is advantageous to modify the expression of specific genes, i.e. increasing or attenuating gene expression. These modifications allow to improve the 1,2-propanediol production performance.<br>
To obtain an overexpression of a gene of interest, the man skilled in the art knows different methods, for example :<br>
-    Replacement of the endogenous promoter with a stronger promoter.<br>
10<br>
 <br>
Introduction into the microorganism of an expression vector carrying said gene of interest.<br>
-	Introduction of additional copies of the gene of interest into the chromosome.<br>
The man skilled in the art knows several techniques for introducing DNA into a bacterial strain. A preferred technique is electroporation, which is well known to those skilled in the art.<br>
To obtain the attenuation of the expression of a gene, different methods are known by the man skilled in the art, and are described below.<br>
In a specific embodiment of the invention, the evolved strain is modified by an attenuation of the glyceraldehyde 3 phosphate dehydrogenase (GAPDH) activity, in order to reduce the flux in the lower part of glycolysis and to redirect it toward the synthesis of DHAP and finally 1,2-propanediol (see figure 1). This decreased activity may in particular be obtained by an attenuation of the expression of the gap A. gene.<br>
The term "attenuation of the activity of an enzyme" refers to a decrease of activity of the enzyme of interest, compared to the observed activity in an evolved strain before any modification. The man skilled in the art knows numerous means to obtain this result, and for example:<br>
-	Introduction of a mutation into the gene, decreasing the expression level of this gene, or the level of activity of the encoded protein.<br>
-	Replacement of the natural promoter of the gene by a low strength promoter, resulting in a lower expression.<br>
Use of elements destabilizing the corresponding messenger RNA or the protein.<br>
-	Deletion of the gene if no expression at all is desired.<br>
Advantageously in the evolved strain, the efficiency of sugar import is increased. A strong attenuation of the expression of the gapA gene resulting in a decrease of the carbon flux in the GAPDH reaction by more than 50%, this will result in the synthesis of less than 1 mole of phosphoenolpymvate (PEP) per mole of imported glucose. The sugar-phosphotransferase system (PTS) usually assuring the import of simple sugars into the cell is coupled to a phosphorylation reaction giving glucose-6-phosphate. The phosphate needed for this reaction is provided by the conversion of PEP into pyruvate. Thus deacreasing the amount of PEP produced by reducing the flux through glyceraldehyde-3-phosphate reduces sugar import.<br>
In a specific embodiment of the invention, the sugar might be imported into the microorganism by a sugar import system independent of phosphoenolpymvate availability.<br>
11<br>
 <br>
The galactose-proton symporter encoded by the gene galP that does not involve phosphorylation can be utilized. In this case, the imported glucose has to be phosphorylated by glucose kinase encoded by the glk gene. To promote this pathway, the expression of at least one gene selected among galP and glk is increased. As a result the PTS becomes dispensable and may be eliminated by attenuating the expression of at least one gene selected among/?toH,/?toI or err.<br>
In another specific embodiment of the invention, the efficiency of the PTS is increased by increasing the availability of the metabolite PEP. Due to the attenuation of the gapA activity and of the lower carbon flux toward pyruvate, the amoimt of PEP in the modified strain of the invention could be limited, leading to a lower amount of glucose transported into the cell.<br>
Various means exist that may be used to increase the availability of PEP in a strain of microorganism. In particular, a mean is to attenuate the reaction PEP —» pyruvate. Preferentially, the expression of at least one gene selected among/yfcA eaidpykF, coding for the pyruvate kinase enzyme, is attenuated in said strain to obtain this result. Another way to increase the availability of PEP is to favour the reaction pyruvate —* PEP, catalyzed by phosphoenolpyruvate synthase by increasing the activity of the enzyme. This enzyme is encoded by the ppsA gene. Therefore, preferentially in the microorganism the expression of the ppsA gene is increased. Both modifications can be present in the microorganism simultaneously.<br>
Especially imder anaerobic or microaerobic conditions, it is advantageous that the pyruvate dehydrogenase complex (PDC), converting pyruvate into acetyl-coA has low sensitivity to inhibition by NADH. The term "low sensitivity" is defined with reference to the sensitivity of an unmodified enzyme, as already demonstrated in WO 2005/073364. In particular, such characteristic can be obtained by introducing a specific mutation in the Ipd gene (coding for the sub-unit lipoamide dehydrogenase of the PDC) resulting in the replacement of alanine 55 in the protein sequence of the enzyme by a valine.<br>
In another specific embodiment of the invention, the synthesis of the by-product acetate is prevented. Under fiilly aerobic conditions, the reduced co-factor NADH is preferentially oxidised into NAD+ via the respiratory chain with oxygen as a terminal electron acceptor. Therefore, the synthesis of a co-product (e.g. acetate) is not mandatory. It is preferable to avoid such acetate synthesis to optimize the production of 1,2-propanediol.<br>
To prevent the production of acetate, advantageously the activity of at least one enzyme involved in the synthesis of acetate is attenuated. Preferentially, the expression of at<br>
12<br>
 <br>
least one gene selected among ackh, pta and poxB is attenuated, all these genes coding for enzymes involved in different acetate biosynthesis pathways (see figure 1).<br>
Another object of the invention is a method for preparing 1,2-propanediol wherein an evolved strain such as described previously is grown in an appropriate growth medium containing a carbon source, and then the 1,2-propanediol produced is recovered. The production of 1,2-propanediol is performed under aerobic, microaerobic or anaerobic conditions.<br>
The culture conditions (fermentation) for the micro-organisms according to the invention can be readily defined by those skilled in the art. In particular, bacteria are fermented at temperatures between 20°C and 55°C, preferably between 25°C and 40°C, and preferably at about 35°C for C. acetobutylicum and at about 37°C for E. coli.<br>
This process can be carried out either in a batch process, in a fed-batch process or in a continuous process.<br>
The evolved strain may be used to produce 1,2-propanediol imder aerobic, micro-aerobic or anaerobic conditions.<br>
'Under aerobic conditions' means that oxygen is provided to the culture by dissolving the gas into the liquid phase. This could be obtained by (1) sparging oxygen containing gas (e.g. air) into the liquid phase or (2) shaking the vessel containing the culture medium in order to transfer the oxygen contained in the head space into the liquid phase. Advantages of the fermentation under aerobic conditions instead of anaerobic conditions is that the presence of oxygen as an electron acceptor improves the capacity of the strain to produce more energy in form of ATP for cellular processes. Therefore the strain has its general metabolism improved.<br>
Micro-aerobic conditions are defined as culture conditions wherein low percentages of oxygen (e.g. using a mixture of gas containing between 0.1 and 10% of oxygen, completed to 100% with nitrogen), is dissolved into the liquid phase.<br>
Anaerobic conditions are defined as culture conditions wherein no oxygen is provided to the culture medium. Strictly anaerobic conditions are obtained by sparging an inert gas like nitrogen into the culture medium to remove traces of other gas. Nitrate can be used as an electron acceptor to improve ATP production by the strain and improve its metabolism.<br>
The culture of strains, during the evolution process and the fermentation process for 1,2-propanediol production, is conducted in fermentors with a culture medium of known set composition adapted to the bacteria used, containing at least one carbon source. In<br>
13<br>
 <br>
particular, a mineral growth medium for E. coli can thus be of identical or similar composition to M9 medium (Anderson, 1946, Proc. Natl. Acad. Sci. USA 32:120-128), M63 medium (Miller, 1992; A Short Course in Bacterial Genetics: A Laboratory Manual and Handbook for Escherichia coli and Related Bacteria, Cold Spring Harbor Laboratory Press, Cold Spring Harbor, New York) or a medium such as that defined by Schaefer et al. (1999, Anal. Biochem. 270: 88-96), and in particular the minimimi culture medium named MPG described below :<br>
<br>
K2HP04	1.4 g/1<br>
Nitrilo Triacetic Acid	0.2 g/1<br>
trace element solution*	10 ml/1<br>
(NH4)2S04	IgA<br>
NaCl	0.2 g/1<br>
NaHCOa	0.2 g/1<br>
MgS04	0.2 g/1<br>
glucose	20 to 100 g/1<br>
NaNOs	0.424 g/1<br>
thiamine	lOmg/1<br>
FeS04,7H2O	50mg/l<br>
yeast extract	4gA<br>
The pH of the medium is adjusted to 7.4 with sodium hydroxide.<br>
*trace element solution : Citric acid 4.37 g/L, MnS04 3 g/L, CaCla 1 g/L, C0CI2, 2H2O 0.1 g/L, ZnS04, 7H2O 0.10 g/L, CUSO4, SHaO 10 mg/L, H3BO3 10 mg/L, Na2Mo04 8.31 mg/L.<br>
In a specific embodiment of the invention, the method is performed with an evolved strain of E. coli, in a growth medium containing a simple carbon source that can be : arabinose, fhictose, galactose, glucose, lactose, maltose sucrose or xylose. An especially preferred simple carbon source is glucose.<br>
In another specific embodiment of the invention, the method is performed with an evolved strain of Clostridium acetobutylicum, in a growth medium containing a simple or a complex carbon source.<br>
The growth mediiam for C acetobutylicum can thus be of identical or similar composition to Clostridial Growth Medium (CGM, Wiesenbom et al., Appl. Environm.<br>
14<br>
 <br>
Microbiol., 54 : 2717-2722) or a mineral growth medium as given by Monot et al. (Appl. Environm. Microbiol., 44: 1318-1324) or Vasconcelos et al. (J. Bacteriol., 176 : 1443-1450).<br>
The carbon source used for the culture of C. acetobutylicum is either a simple or a complex carbon. The simple carbon source can be arabinose, fructose, galactose, glucose, lactose, maltose sucrose or xylose. An especially preferred simple carbon source is glucose. The complex carbon source can be starch or hemicellulose. An especially preferred complex carbon source is starch.<br>
Preferentially, the recovered 1,2-propanediol is fiirthermore purified. The man skilled in the art knows methods for recovering and purifying the produced 1,2-propanediol. These methods are usual processes.<br>
The invention is described above, below and in the Examples with respect to E. coli. Thus the genes that can be attenuated, deleted or over-expressed for the initial and evolved strains according to the invention are defined mainly using the denomination of the genes fi-om E. coli. However, this designation has a more general meaning according to the invention, and covers the corresponding genes in other micro-organisms. Using the GenBank references of the genes from E. coli, those skilled in the art can determine equivalent genes in other organisms than E. coli.<br>
The means of identification of the homologous sequences and their percentage homologies are well-known to those skilled in the art, and include in particular the BLAST programmes that can be used on the website http://www.ncbi.nlm.nih.gov/BLAST/ with the default parameters indicated on that website. The sequences obtained can be exploited (aligned) using for example the programmes CLUSTALW (http://www.ebi.ac.uk/clustalw/). with the default parameters indicated on these websites.<br>
The PFAM database (protein families database of alignments and hidden Markov models http://www.sanger.ac.uk/Software/Pfam/) is a large collection of alignments of protein sequences. Each PFAM makes it possible to visualise multiple alignments, view protein domains, evaluate distributions among organisms, gain access to other databases and visualise known protein structures.<br>
COGs	(clusters	of	orthologous	groups	of	proteins<br>
http://www.ncbi.nlm.nih. gov/COG/) are obtained by comparing protein sequences derived from 66 fully sequenced genomes representing 44 major phylogenetic lines. Each COG is defined fi-om at least three lines, making it possible to identify ancient conserved domains.<br>
15<br>
 <br>
REFERENCES in the order of the citation in the text<br>
1.	Badia J, Ros J, Aguilar J (1985), J. Bacterial 161: 435-437.<br>
2.	Tran Din K and Gottschalk G (1985), Arch. Microbiol. 142: 87-92<br>
3.	Cameron DC and Cooney CL (1986), Bio/Technology, 4: 651 -654<br>
4.	Sanchez-Rivera F, Cameron DC, Cooney CL (1987), Biotechnol. Lett. 9 : 449-454<br>
5.	Altaras NE and Cameron DC (1999), Appl. Environ. Microbiol. 65 : 1180-1185<br>
6.	Cameron DC, Altaras NE, Hofl&amp;nan ML, Shaw AJ (1998), Biotechnol. Prog. 14 : 116-125<br>
7.	Altaras NE and Cameron DC (2000), Biotechnol. Prog. 16 : 940-946<br>
8.	Huang K, Rudolph FB, Bennett GN (1999), Appl. Environ. Microbiol. 65: 3244-3247<br>
9.	Berrios-Rivera SJ, San KY, Bennett GN (2003), J. Ind. Microbiol. Biotechnol. 30: 34-40<br>
10.	Datsenko KA and Wanner BL (2000), Proc. Natl. Acad. Sci. USA 97: 6640-6645<br>
11.	Anderson EH (1946), Proc. Natl. Acad Sci. USA 32:120-128<br>
12.	Miller (1992), A Short Course in Bacterial Genetics: A Laboratory Manual and Handbook for Escherichia coli and Related Bacteria, Cold Spring Harbor Laboratory Press, Cold Spring Harbor, New York<br>
13.	Schaefer U, Boos W, Takors R, Weuster-Botz D (1999), Anal. Biochem. 270: 88-96<br>
14.	Wiesenbom DP, Rudolph RB, Papoutsakis ET (1987), Appl. Environ. Microbiol., 54 : 2717-2722<br>
15.	Monot F, Martin JR, Petitdemange H, Gay R (1982), Appl. Environ. Microbiol. 44: 1318-1324<br>
16.	Vasconcelos I, Girbal L, SoucailleP (1994), J. Bacterial. 176: 1443-1450<br>
EXAMPLES<br>
The following examples show:<br>
I- Construction of a modified strain ofE. coli MG1655 Ipd*, OtpiA, OpflAB, OadhE, OldhA, DgloA, DaldA. UaldB. Uedd. UarcA, Qndh<br>
16<br>
 <br>
2-	Evolution of said initial strain<br>
3-	Reconstruction of the tpiA gene in the selected evolved strain<br>
4-	Attenuation   of the   gapA   gene;   Deletion   of the   genes   pykA   and   pykF; Overexpression of the ppsA gene<br>
5-	Deletion of the genes ackA-pta, poxB<br>
6-	Comparison of several obtained strains for 1,2-propanediol production under aerobic conditions<br>
7-	Production of 1,2-propanediol in fed-batch culture with the best strain.<br>
Example 1: construction of a modified strain oiE. coli MG1655 lpd\ UtpiA, UpflAB, UadhE, DldhA, DgloA, UaldA, UaldB, Uedd, UarcA, Ondh able to evolve toward improved 1,2-propanediol production:<br>
a) Construction of a modified strain E. coli MG1655 Ipd*, UtpiA, UpJlAB^ UadhE, ldhA::Km, UgloA, UaldA, UaldB, Uedd<br>
The chloramphenicol resistance cassette was eliminated in the strain E. coli MG1655 Ipd*, UtpiA. UpJlAB. UadhE. IdhAr.km. UgloA, UaldA. UaldB, Uedd::cai (See WO2005073364) according to Protocol 1.<br>
Protocol 1 : Elimination of resistance cassettes<br>
The chloramphenicol and/or kanamycin resistance cassettes were eliminated according to the following technique. The plasmid pCP20 carrying the FLP recombinase acting at the FRT sites of the chloramphenicol and/or kanamycin resistance cassettes was introduced into the strain by electroporation. After serial culture at 42°C, the loss of the antibiotic resistance cassettes was checked by PCR analysis with the oligonucleotides given in Table 1.<br>
The presence of the modifications previously built in the strain was checked using the oligonucleotides given in Table 1.<br>
The strain obtained was named E. coli MG1655 Ipd*, UtpiA. UpflAB, UadhE, ldhA::km. UgloA, UaldA. UaldB. Uedd.<br>
Table 1 : Oligonucleotides used for checking the insertion of a resistance cassette or the loss of a resistance cassette<br>
<br>
Region name	Names of oligos	SEQID	Homology with chromosomal region<br>
tpiA gene	cdh	N°l	See WO2005073364<br>
17<br>
 <br>
(deletion)	YIIQ	N°2	<br>
p/1 AB gene	pflABF pflABR	N°3 N°4	SeeWO2005073364<br>
adhE gene	ychGf adhECr	N°5 N°6	SeeWO2005073364<br>
IdhA gene (cassette insertion)	hsIJC ldhAC2	N°7 N°8	SeeWO2005073364<br>
gloA gene	NemACd RntCr	N°9 N°10	SeeWO2005073364<br>
aid A gene	YdcFCf gapCCr	N°ll N°12	SeeWO2005073364<br>
aldB gene	aldBCf YiaYCr	N°13 N°14	SeeWO2005073364<br>
edd gene	Edad Zwfr	N°15 N°16	SeeWO2005073364<br>
WAA gene (deletion)	IdhAF IdhAR	N°17 N°18	1439724 to 1439743 1441029 to 1441007<br>
arc A gene	arcAF arcAR	N°19 N°20	4638292 to 4638273 4636854 to 4636874<br>
nJA gene	ndhF ndhR	N°21 N°22	1164722tol164742 1167197 to 1167177<br>
/^/A gene (reconstruction)	YIIQ tpiAR	N°2 N°23	4109599 to 4109580 4108953 to 4108973<br>
gap A promoter (Ptrcl6-gapA)	yeaAF gapAR	N°24 N°25	1860259-1860287 1861068-1861040<br>
pykA gene	pykAF pykAR	N°26 N°27	1935338 to 1935360 1937425 to 1937401<br>
pykF gene	pykFF pykFR	N°28 N°29	1753371 to 1753392 1755518 to 1755495<br>
ackA-pta genes	B2295 YfcCR	N°30 N°31	2410900 to 2410919 2415164 to 2415145<br>
poxB gene	poxBF poxBR	N°32 N°33	908475 to 908495 910375 to 910352<br>
18<br>
 <br>
b) Construction of a modified strain E. coli MG1655 Ipd*, UtpiA, OpflAB, UadhE, UldhA::cm, UgloA, UaldA, UaldB, Uedd<br>
In order to eliminate the kanamycin resistance cassette and to inactivate the IdhA gene, the chloramphenicol resistance cassette was inserted into the IdhA gene deleting most of the gene concerned according to Protocol 2.<br>
Protocol 2 : Introduction of a PCR product for recombination and selection of the recombinants<br>
The oligonucleotides chosen and given in Table 2 for replacement of a gene or an intergenic region were used to amplify either the chloramphenicol resistance cassette from the plasmid pKD3 or the kanamycin resistance cassette from the plasmid pKD4 (Datsenko, K.A. &amp; Wanner, B.L. (2000)). The PCR product obtained was then infroduced by electroporation into the recipient strain bearing the plasmid pKD46 in which the system Q Red (QDexo) expressed greatly favours homologous recombination. The antibiotic-resistant fransformants were then selected and the insertion of the resistance cassette was checked by PCR analysis with the appropriate oligonucleotides given in Table 1.<br>
The other modifications of the strain were checked with the oligonucleotides given in Table 1.<br>
The resulting strain was named E. coli MG1655 Ipd*, DldhAiicm, UtpiA, UpflAB, UadhE. UgloA, UaldA. UaldB. Uedd.<br>
Table 2 : Oligonucleotides used for replacement of a chromosomal region by recombination with a PCR product in the strain E. coli MG1655<br>
<br>
Region name	Names of oligos	SEQID	Homology with chromosomal region<br>
IdhA gene	DldhAF DldhAR	N°34 N°35	1440865-1440786 1439878-1439958<br>
arcK gene	DarcAF DarcAR	N°36 N°37	4637868-4637791 4637167-4637245<br>
ndh gene	DndhF DndhR	N°38 N°39	1165071-1165149 1166607-1166528<br>
tpiA gene (reconstruction)	tpiA::kmF tpiA::kmR	N°40 N°41	4109264-4109195 4109109-4109193<br>
gaph promoter	Ptrc-gapAF	N°42	1860478-1860536<br>
19<br>
 <br>
(Ptrcl6-gapA)	Ptrc-gapAR	N°43	1860762-1860800<br>
pykh gene	DpykAF	N°44	1935756-1935836<br>
	DpykAR	N°45	1937055-1937135<br>
pyW gene	DpykFF	N°46	1753689-1753766<br>
	DpykFR	N°47	1755129-1755051<br>
ackA-pta genes	DackAF	N°48	2411494-2411573<br>
	DptaR	N°49	2414906-2414830<br>
/70JcB gene	DpoxBF	N°50	908557-908635<br>
	DpoxBR	N°51	910262-910180<br>
c)	Construction of a modified strain E. coli MG1655 OarcA::km<br>
The gene arcA was inactivated in strain E. coli MG1655 by inserting a kanamycin antibiotic resistance cassette and deleting most of the gene concerned using the technique described in Protocol 2 with the oligonucleotides given in Table 2. The resulting strain was named E coli MG1655 DarcA::km.<br>
d)	Construction of a modified strain of E. coli MG16SS lpd\ UtpiA, UplfAB, UadhE,<br>
UldhA, UgloA, QaldA, UaldB, Dedd, UarcA<br>
The deletion of the gene arcA by replacement of the gene by a kanamycin resistance cassette in the strain E. coli MG1655 Ipd*, UtpiA. DpflAB. UadhE. UldhA::cm. UgloA. UaldA. UaldB. Dedd wasperformedby the technique of transduction with phage PI.<br>
Protocol 3 : Transduction with phage PI for deletion of a gene<br>
The deletion of the chosen gene by replacement of the gene by a resistance cassette (kanamycin or chloramphenicol) in the recipient E. coli strain was performed by the technique of transduction with phage PI. The protocol was in two steps, (i) the preparation of the phage lysate on the strain MG1655 with a single gene deleted and (ii) the transduction of the recipient strain by this phage lysate.<br>
Preparation of the phage lysate<br>
-	Seeding with 100 ^1 of an overnight culture of the strain MG1655 with a single gene<br>
deleted of 10 ml of LB + Cm 30 jig/ml + glucose 0.2% + CaCb 5 mM.<br>
-	Incubation for 30 min at 37°C with shaking.<br>
-	Addition of 100 |il of phage lysate PI prepared on the wild type strain MG1655<br>
(approx. 1 X lO'phage/ml).<br>
-	Shaking at 37°C for 3 hours until all cells were lysed.<br>
-	Addition of 200 nl of chloroform, and vortexing.<br>
20<br>
 <br>
-	Centrifiigation for 10 min at 4500 g to eliminate cell debris.<br>
-	Transfer of supernatant in a sterile tube and addition of 200 ^1 of chloroform.<br>
-	Storage of the lysate at 4°C<br>
Transduction<br>
-	Centrifugation for 10 min at 1500 g of 5 ml of an overnight culture of the E. coli recipient strain in LB medium.<br>
-	Suspension of the cell pellet in 2.5 ml of MgS04 10 mM, CaCh 5 mM.<br>
-	Control tubes:      100 \i\ cells<br>
100 \i\ phages PI of the strain MG1655 with a single gene deleted.<br>
-	Tube test: 100 nl of cells + 100 |il phages PI of strain MG1655 with a single gene<br>
deleted.<br>
Incubation for 30 min at 30°C without shaking.<br>
-	Addition of 100 pl sodixim citrate 1 M in each tube, and vortexing.<br>
-	Addition of 1 ml of LB.<br>
-	Incubation for 1 hour at 37°C with shaking<br>
Plating on dishes LB + Cm 30 ng/ml after centrifugation of tubes for 3 min at<br>
7000 rpm.<br>
Incubation at 37°C ovemight.<br>
The antibiotic-resistant transformants were then selected and the insertion of the deletion was checked by a PCR analysis with the appropriate oligonucleotides given in Table 1.<br>
The resulting strain was named E. coli MG1655 Ipd*, UtpiA, UpflAB, OadhE, UldhA::cm. DgloA. UaldA. UaldB. Uedd, UarcA.Mm.<br>
The chloramphenicol and kanamycin resistance cassettes were then eliminated according to Protocol 1. The strain obtained was named E. coli MG1655 Ipd*, UtpiA, UpflAB. UadhE. UldhA, UgloA. UaldA. UaldB. Uedd. UarcA.<br>
e)	Construction of a modified strain off. coliMG16SS Undh:: km<br>
The gene ndh was inactivated by inserting a kanamycin antibiotic resistance cassette and deleting most of the gene concerned using the technique described in Protocol 2 with the oligonucleotides given in Table 2. The resulting strain was named E. coli MG1655 Undhvkm.<br>
f)	Construction of a strain E. coliMG16SS Ipd*, UtpiA, UplfAB, UadhE, UldhA, UgloA,<br>
UaldA, UaldB, Uedd, UarcA, Undh<br>
21<br>
 <br>
The deletion of the gene ndh by replacement of the gene by a kanamycin resistance cassette in the strain E. coli MG1655 Ipd*, UtpiA, UplfAB. UadhE. UldhA. UgloA, UaldA, UaldB, Uedd, UarcA was performed as previously using the transduction technique with phage PI described in Protocol 3.<br>
The resulting strain was named E. coli MG1655 Ipd*, UtpiA,, DpflAB. OadhE. UldhA. UgloA. UaldA. UaldB. Uedd. UarcA. Undk.-Mm.<br>
The kanamycin resistance cassette was then eliminated according to Protocol 1. The strain obtained was named E. coli MG1655 Ipd*, UtpiA. UpflAB. UadhE. UldhA. UgloA. UaldA. UaldB. Uedd UarcA. Undh.<br>
At each step, the presence of the modifications previously built in the strain was checked using the oligonucleotides given in Table 1.<br>
Example 2: Evolution of the modified strain E. coli MG1655 Ipd* UtpiA, UpflAB, UadhE, UldhA::cm, UgloA, UaldA, UaldB, Uedd in chemostat culture under microaerobic conditions and physiological characterization of evolution :<br>
To evolve it toward improved 1,2 propanediol production, the strain E. coli MG1655 Ipd* UtpiA. UpflAB, UadhE, UldhAr.cm, UgloA, UaldA. UaldB, Uedd was cultivated in continuous culture under anaerobic conditions on one side and under microaerobic conditions (1% oxygen) on the other side in the culture medium MPG such as described previously, with excess glucose (from 20 g/1 initially with addition if the glucose becomes exhausted). The temperature was set at 37°C and the pH was regulated at 6.5 by addition of base. The evolution of the strain in the chemostats was followed by the increase of the biomass concentration coupled with the increase of the concentrations of the product, 1,2-propanediol and the co-product acetate, over several weeks (from 4 weeks up to 6 months). This denoted the improvement of the performances of the strains. When the cultures reached a steady state with no fijrther increase of the concentrations under these conditions, the evolution was done.<br>
The characteristics of the strains before and after evolution were assessed. Single colonies representing individual clones were isolated on Petri dishes. These clones were assessed using the initial strain as confrol in an Erlenmeyer flask assay, using the same medium MPG used in the chemostat culture. Among these clones, several presented better 1,2-propanediol specific production rates as compared to the control. These clones were selected for the following steps. The results obtained on the best clone for each condition of evolution are reported in Table 4 and 5 below.<br>
22<br>
 <br>
Table 4 : Comparison of the best evolved clone obtained after evolution under anaerobic conditions with the initial strain<br>
<br>
Strain E. coli MG1655 Ipd* UtpiA	Initial   strain   before	Best evolved clone<br>
UpflAB UadhE UldhA::cm UgloA	evolution	(performances<br>
Oald, QaldB Oedd	(performances	measured after 2 days<br>
	measured after 2 days of culture)	of culture)<br>
Glucose specific consumption rate	0.12	0.21 (+75%)<br>
(g glucose /g biomass /h)		<br>
1,2-propanediol specific production	0.02	0.07 (+250%)<br>
rate (g 1,2-propanediol /g biomass /h)		<br>
1,2-propanediol + hydroxyacetone	0.04	0.08 (+100%)<br>
specific production rate		<br>
(g 1,2-propanediol + hydroxyacetone		<br>
/g biomass /h)		<br>
Table 5 : Comparison of the best evolved clone obtained after evolution under microaerobic conditions with the initial strain<br>
<br>
Strain E. coli MG1655 Ipd* DtpiA	Initial   strain   before	Best evolved clone<br>
UpflAB UadhE UldhA::cm UgloA	evolution	(performances<br>
Uald, UaldB Uedd	(performances	measured after 2 days<br>
	measured after 2 days of culture)	of culture)<br>
Glucose specific consimiption rate	0.10	0.22 (+120%)<br>
(g glucose /g biomass /h)		<br>
1,2-propanediol specific production	0.01	0.08 (+700%)<br>
rate (g 1,2-propanediol /g biomass /h)		<br>
1,2-propanediol + hydroxyacetone	0.04	0.08 (+100%)<br>
specific production rate		<br>
(g 1,2-propanediol + hydroxyacetone		<br>
/g biomass /h)		<br>
23<br>
 <br>
As these clones have been cultivated over an extended period of time on cultvire medium with yeast extract, they needed to be adapted for the growth in minimal medium. The two best clones whose performances are given in Table 4 and 5 were adapted by serial cultwe on minimal medium in order to increase their growth rates under such conditions and the adaptation was stopped when their growth rates were stable. Clones from the final culture were isolated and checked to be representative of the adapted population.<br>
Example 3: reconstruction of tpiX gene in the selected evolved strain of E. coU MG1655 //»* UtpiA, OpflAB, DadhE, UldhA::cm, UgloA, UaldA, UaUB, Dedd: a) Construction of a modified strain E. coli MG165$ tpiA::km<br>
A kanamycin antibiotic resistance cassette was inserted upstream of the gene tpiA according to the technique described in Protocol 2 with the oligonucleotides given in Table 2. The resulting strain was named E coli MG1655 /p/A::km.<br>
Then the reconstruction of the gene tpiA into the evolved strain E. coli MG1655 Ipd*, UtpiA, UplfAB. DadhE. DldhA::cm. OgloA, UaldA. UaldB, Uedd. UarcA, Qndh was performed using the transduction technique with phage PI described in Protocol 3.<br>
The resulting strain was named evolved E. coli MG1655 Ipd*, tpiArc::km,, UplfAB, UadhE, DldhA::cm, UgloA. UaldA. UaldB, Uedd, UarcA. Undh.<br>
The kanamycin and chloramphenicol resistance cassettes were then eliminated according to Protocol 2. The strain obtained was named 'evolved E. coli tpiAxc"<br>
The presence of the modifications previously built in the strain was checked using the oligonucleotides given in Table 1.<br>
Example 4: Modifications of the 'evolved K coli tpiXrc* : attenuation of the gapA gene; deletion of the genes/&gt;yA:A stadpykF; over-expression of ppsA gene with a vector pJB13 7-PgapA-ppsA a) Rejpbceinentof the natural gapA promoter with the synthetic short Vtrc\6 promoter :<br>
The replacement of the natural gap A promoter with the synthetic short ?trc\6 promoter (SEQ ID NO 52 : gagctgttgacgattaatcatccggctcgaataatgtgtggaa) into the strain 'evolved E. coli tpiArc' was made by replacing 225 pb of upstream gap A sequence with FRT-CmR-FRT and an engineered promoter.<br>
The technique used is described in Protocol 2 with the oligonucleotides given in Table 2. The resulting strain was named 'evolvedE. coli tpiArc' Ptrcl6-gapA::cm.<br>
24<br>
 <br>
The chloramphenicol resistance cassette was then eliminated according to Protocol 1. The strain obtained was named 'evolved E. coli tpiArc' Ptrc\6-gapA.<br>
b)	Deletion oiih&amp;pykX gene<br>
The gene pykA is inactivated by inserting a kanamycin antibiotic resistance cassette and deleting most of the gene concerned using the technique described in Protocol 2 with the oligonucleotides given in Table 2. The resulting strain is named ""evolved E. coli tpiArc' Ptrc\6-gapA DpykA::km.<br>
The kanamycin resistance cassette is then eliminated according to Protocol 1. The strain obtained is named 'evolvedE. coli tpiArc' Ptrc\6-gapA OpykA.<br>
c)	Deletion of the pykF gene<br>
The genepy/cP is inactivated by inserting a kanamycin antibiotic resistance cassette and deleting most of the gene concerned using the technique described in Protocol 2 with the oligonucleotides given in Table 2. The resulting strain is named 'evolved E. coli tpiArc' Ptrc\6-gapA, DpykA^ DpykF::km.<br>
As previously, the kanamycin resistance cassette is then eliminated according to Protocol 1. The strain obtained is named 'evolved E. coli tpiArc' Ptrc\6-gapA. OpykA, DpykF.<br>
d)	Introduction of an expression \eciorpJB137-PgapA-ppsA into the strain<br>
To increase the production of phosphoenolpyruvate the ppsA gene was expressed from the plasmid pJB137 using the gap A. promoter. For the construction of plasmid pJB137-PgapA-ppsA, the %&amp;a&amp; ppsA was PCR amplified from genomic DNA oi E. coli MG1655 using the following oligonucleotides:<br>
1.	gapA-ppsAF, consisting of 65 bases (SEQ ID NO 53)<br>
ccttttattcactaacaaatagctggtggaatatATGTCCAACAATGGCTCGTCACCGCTGGTGC<br>
with:<br>
-	a region (upper-case letters) homologous to the sequence (1785106-1785136) of the gene ppsA {11^5X36 to 1782758), a reference sequence on the website http://genolist.pasteur.fr/ColibriA. and<br>
-	a region (lower letters) homologous to the gap A promoteur (1860794 -1860761).<br>
2.	ppsAR, consisting of 43 bases (SEQ ID NO 54)<br>
aatcgcaagcttGAATCCGGTTATTTCTTCAGTTCAGCCAGGC<br>
with:<br>
25<br>
 <br>
-	a region (upper letters) homologous to the sequence (1782758 -1782780) the region of the gene ppsA (1785136 to 1782758)<br>
-	a restriction site Hindill (underlined letters)<br>
At the same time the gapA promoter region of the E. coli gene gapA was amplified using the following oligonucleotides:<br>
1.	gapA-ppsAR, consisting of 65 bases (SEQ ID NO 55)<br>
GCACCAGCGGTGACGAGCCATTGTTGGACATatattccaccagctatttgttagtgaataaaagg<br>
with:<br>
-	a region (upper-case letters) homologous to the sequence (1785106 -1785136) of the gene/?p5^ (1785136 to 1782758), and<br>
-	a region (lower letters) homologous to the gapA promoteur (1860794 -1860761).<br>
2.	gapAF, consisting of 33 bases (SEQ ID NO 56)<br>
ACGTCCCGGGcaagcccaaaggaagagtgaggc<br>
with:<br>
-	a region (lower letters) homologous to the gap A promoteur (1860639 -1860661).<br>
-	a restriction site Smal (underlined letters)<br>
Both fragments were subsequently fiised using the oligonucleotides ppsAR and gapAF (Horton et al. 1989 Gene 77:61-68). The PCR amplified fragment were cut with the restriction enzymes Hindill and Smal and cloned into the HindllVSmal sites of the vector pJB137 (EMBL Accession number: U75326) giving vector pJB137-PgapA-ppsA. Recombinant plasmids were verified by DNA sequencing.<br>
The plasmid pJB137-PgapA-ppsA is introduced into the strain 'evolved E. coli tpiArc'Ptrc\6-gapA, OpykA, UpykF.<br>
The strain obtained is named 'evolved E. coli tpiArc'. Ptrc\6-gapA. DpykA, OpykF, (pJB13 7-PgapA-ppsA).<br>
At each step, the presence of the modifications previously built in the strain was checked using the oligonucleotides given in Table 1.<br>
Example 5: construction of a strain 'evolved E. coli tpiArc* Ptrcl6-gapA, UpykA, UpykF, UackA-pta,  UpoxB (pJB137-PgapA-ppsA) able to produce 1,2-propanediol without acetate as by-product a) Construction of a modified strain E. coli MG1655 D ackA-pta: :cm<br>
26<br>
 <br>
The genes ackA and pta are inactivated by inserting a chloramphenicol antibiotic resistance cassette and deleting most of the gene concerned using the technique described in Protocol 2 with the oligonucleotides given in Table 2. The resulting strain is named E coli MG1655 UackA-ptav.cm.<br>
b)	Construction of a strain 'evolved E. coli tpiArc' Ptrcl6-gapA,  DpykA,  UpykF,<br>
UackA-pta<br>
The deletion of the genes ackA and pta in the strain 'evolved E. coli tpiArc' Ptrcl-gapA, DpykA, OpykF is performed as previously using the transduction technique with phage PI as described in Protocol 3.<br>
The resulting strain is named 'evolved E. coli tpiArc' Ptrc\6-gapA, DpykA, DpykF, DackA-pta: :cm.<br>
As previously, the chloramphenicol resistance cassette is then eliminated according to Protocol 1. The strain obtained is named 'evolved E. coli tpiArc' Ptrcl6-gapA, DpykA, DpykF, DackA-pta.<br>
c)	Construction of a modified strain 'evolved E. coli piArc' Ptrcl-gapA, DpykA,<br>
DpykF, DackA-pta, DpoxB (pJB137-PgapA-ppsA)<br>
The gene poxB is inactivated by inserting a chloramphenicol antibiotic resistance cassette and deleting most of the gene concerned using the technique described in Protocol 2 with the oligonucleotides given in Table 2.<br>
The resulting strain is named evolved E. coli tpiArc Ptrcl6-gapA, DpykA, DpykF. DackA-pta, DpoxB::cm.<br>
As previously, the chloramphenicol resistance cassette is then eliminated according to protocol 1. The strain obtained is named evolved E. coli tpiArc Ptrc\6-gapA, DpykA, DpykF, DackA-pta, DpoxB.<br>
The plasmid pJB137-PgapA-ppsA is introduced into the strain evolved E. coli tpiArc Ptrcl-gapA, DpykA, DpykF, DackA-pta. DpoxB. The strain obtained is named evolved E. coli tpiArc Ptrc\6-gapA, DpykA, DpykF, DackA-pta, UpoxB (pJB137-PgapA-ppsA).<br>
At each step, the presence of the modifications previously built in the strain is checked using the oligonucleotides given in Table 1.<br>
Example  6   :  Comparison  of the  different  evolved  strains  for  1,2-propanediol production under aerobic conditions<br>
The strains obtained as described in Example 4 and the control strains (control 1 : MG1655 Ipd* DtpiA DpflAB DadhE DldhAr.Cm. DgloA Dald, DaldB Dec/J evolved under<br>
27<br>
 <br>
anaerobic conditions and control 2 : MG1655 Ipd* DtpiA UpJlAB UadhE UldhA::Cm UgloA Oald, UaldB Uedd evolved under microaerobic conditions) were cultivated in an Erlenmeyer flask assay under aerobic conditions in minimal medium supplemented with yeast extract and with glucose as carbon source. The culture was carried out at 34°C and the pH was maintained by buffering the culture medium with MOPS. At the end of the culture, 1,2-propanediol, acetol and residual glucose in the fermentation broth were analysed by HPLC and the yields of 1,2-propanediol over glucose and 1,2-propanediol + acetol over glucose were calculated. The best strain is then selected for a fermenter fed-batch culture.<br>
<br>
Strain	1,2-	Acetol	1,2-	1,2-propanediol<br>
	propanediol	titer	propanediol	+ acetol<br>
	titer	(g/1)	yield	yield<br>
	(g/1)		(g/g glucose)	(g/g glucose)<br>
Control 1	1.88	2.1	0.16	0.34<br>
Control 2	0.7	3.56	0.06	0.37<br>
'evolved E. coli tpiArc',	0.5	2.77	0.06	0.42<br>
Ptrcl6-gapA,   (pJB137-				<br>
PgapA-ppsA)				<br>
(built from control 1)				<br>
'evolved E. coli tpiArc',	3.71	3.85	0.20	0.41<br>
Ptrc\6-gapA.   (pJB137-				<br>
PgapA-ppsA)				<br>
(built from control 2)				<br>
Example 7 : Production of 1,2-propanediol in fed-batch culture with the best strain<br>
The best strain selected in the previous experiment is cultivated in a 21 fermenter using a fed-batch protocol.<br>
The temperature of the culture is maintained constant at 37 °C and the pH is permanently adjusted to values between 6.5 and 8 using an NH4OH solution. The agitation rate is maintained between 200 and 300 rpm during the batch phase and is increased to up to 1000 rpm at the end of the fed-batch phase. The concentration of dissolved oxygen is maintained at values between 30 and 40% saturation by using a gas controller. When the optical density reachs a value between three and five, the fed-batch is started with an initial<br>
28<br>
 <br>
flow rate between 0.3 and 0.5 ml/h, and a progressive increase up to flow rate values between 2.5 and 3.5 ml/h. At this point the flow rate is maintained constant for 24 to 48 hours. The medium of the fed is based on minimal media containing glucose at concentrations between 300 and 500 g/1.<br>
29<br>
<br>
<br>
WE CLAIM:<br>
1.	A method for the preparation of an evolved strain of microorganism for the<br>
production of 1,2-propanediol from a carbon source, said method comprising:<br>
growing an initial strain under selection pressure in an appropriate growth<br>
medium, said initial bacterial strain comprising an attenuation of the<br>
expression of the tpiA gene and an attenuation of the expression of at least<br>
one gene involved in the conversion of methylglyoxal into lactate, in order<br>
to promote evolution in said initial strain,<br>
then selecting and isolating the evolved strain having an increased 1,2<br>
propanediol production rate,<br>
then reconstructing a functional tpiA gene in the evolved strain.<br>
2.	The method of claim 1, wherein the gene involved in the conversion of methylglyoxal into lactate is selected from the group consisting of: gloA, aid A and ald&amp; and combinations thereof.<br>
3.	The method of claims 1 to 2, wherein the initial strain comprises furthermore the attenuation of the expression of at least one of the genes selected among the group consisting oildhA, pflA, pflB, adhE, edd and eda.<br>
4.	The method of claims 1 to 3, wherein the initial strain comprises futhermore the attenuation of at least one gene selected among the group consisting of arcA and ndh.<br>
5.	The method of anyone of claims 1 to 4 wherein the evolved sfrain is selected and isolated on the basis of its 1,2-propanediol production rate, increased by at least 20% compared to the production rate of the initial strain.<br>
6.	The method of anyone of claims 1 to 5, wherein the initial sfrain is selected from the group consisting of bacteria, yeasts and fimgi.<br>
7.	The method of claim 6, wherein the initial sfrain is selected from the group consisting of Enterobacteriaceae, Bacillaceae, Clostridiaceae, Sfreptomycetaceae and Corynebacteriaceae.<br>
8.	The method of claim 7, wherein the initial strain is either Escherichia coli or Clostridium acetobutylicum.<br>
9.	An evolved strain of microorganism susceptible to be obtained by the method according to anyone of claims 1 to 8.<br>
30<br>
 <br>
10.	The evolved strain according to claim 9 wherein the glyceraldehyde 3 phosphate dehydrogenase activity is attenuated.<br>
11.	The evolved strain according to claim 10 wherein the expression of the gapA gene is attenuated.<br>
12.	The evolved strain according to anyone of claims 9 to 11 wherein the efficiency of the sugar import is increased.<br>
13.	The evolved strain according to claim 12 wherein a sugar import system independent of phosphoenolpyruvate is used.<br>
14.	The evolved strain according to claim 13 wherein the expression of at least one gene selected among gal? and glk is increased.<br>
15.	The evolved strain according to anyone of claims 12 wherein the efficiency of the sugar-phosphotransferase system is improved by increasing the availability of the metabolite phosphoenolpyruvate.<br>
16.	The evolved strain of claim 15 wherein the activity of at least one pyruvate kinase is attenuated.<br>
17.	The evolved strain according to claim 16 wherein the expression of at least one gene selected among joy^A aoApykV is attenuated.<br>
18.	The evolved strain according to anyone of claims 15 to 17 wherein the phosphoenolpyruvate synthase activity is increased.<br>
19.	The evolved strain according to anyone of claim 18 wherein the expression of the ppsA gene is increased.<br>
20.	The evolved strain according to anyone of claims 9 to 19 wherein the enzyme that favours the metabolism of pyruvate into acetyl-CoA has a lower sensitivity to the inhibition by NADH than the unmodified enzyme.<br>
21.	The evolved strain according to claim 20 wherein the Ipd gene has a point mutation leading to a replacement of alanine 55 by valine.<br>
22.	The evolved strain according to anyone of claims 9 to 21 wherein the activity of at least one enzyme involved in the synthesis of acetate is attenuated.<br>
23.	The evolved strain according to claim 22 wherein the expression of at least one gene selected among ackA, pta. poxB is attenuated.<br>
24.	A method for preparing 1,2-propanediol wherein an evolved strain according to anyone of claims 9 to 23 is grown in an appropriate growth medium containing a carbon source, and the produced 1,2-propanediol is recovered.<br>
31<br>
 <br>
25.	The method according to claim 24 wherein an evolved strain of Escherichia coll is<br>
grown in an appropriate growth medium containing a simple carbon source, and the<br>
produced 1,2-propanediol is recovered.<br>
26.	The method according to claim 24 wherein an evolved strain of Clostridium<br>
acetobutylicum is grown in an appropriate growth medium containing a simple or a complex carbon source, and the produced 1,2-propanediol is recovered.<br>
27.	The method according to anyone of claims 24 to 26, wherein the recovered 1,2-<br>
propanediol is furthermore purified.<br>
 <br>
32<br>
</td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/viewdoc.aspx?id=dRVnx8lqot4mAx9WNnulHg==&amp;amp;loc=egcICQiyoj82NGgGrC5ChA==" target="_blank" style="word-wrap:break-word;">http://ipindiaonline.gov.in/patentsearch/GrantedSearch/viewdoc.aspx?id=dRVnx8lqot4mAx9WNnulHg==&amp;amp;loc=egcICQiyoj82NGgGrC5ChA==</a></p>
		<br>
		<div class="pull-left">
			<a href="279825-dual-equal-cam-phasing-with-variable-overlap.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="279827-system-and-process-for-high-density-low-energy-plasma-enhanced-vapor-phase-epitaxy.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>279826</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>5338/CHENP/2009</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>05/2017</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>03-Feb-2017</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>31-Jan-2017</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>10-Sep-2009</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>METABOLIC EXPLORER</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td>BIOPOLE CLERMONT-LIMAGNE, F-63360 SAINT BEAUZIRE</td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>VOELKER, FRANCOIS</td>
											<td>16 RUE YVES MONTAND, 42210 MONTROND LES BAINS</td>
										</tr>
										<tr>
											<td>2</td>
											<td>SOUCAILLE, PHILIPPE</td>
											<td>CHANT DU COUCOU. 31450 DEYME</td>
										</tr>
										<tr>
											<td>3</td>
											<td>FIGGE, RAINER</td>
											<td>25 RUE MARIVAUX, 63200 RIOM</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>C12P7/18</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>PCT/EP2008/053445</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td>2008-03-21</td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td>PCT/IB07/001680</td>
									<td>2007-03-23</td>
								    <td>PCT</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/279826-a-method-for-the-preparation-of-an-evolved-strain-of-micro-organism-for-the-production-of-1-2-propandiol-obtained-by-a-combination-of-evolution-and-rational-design by HTTrack Website Copier/3.x [XR&CO'2014], Thu, 04 Apr 2024 23:01:23 GMT -->
</html>
