<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/279963-hafnia-phytase by HTTrack Website Copier/3.x [XR&CO'2014], Thu, 04 Apr 2024 22:34:23 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 279963:&quot;HAFNIA PHYTASE&quot;</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">&quot;HAFNIA PHYTASE&quot;</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>The present invention relates to isolated polypeptides having phytase activity and isolated polynucleotides encoding the polypeptides. The polypeptides are related to a phytase derived from Hafnia alvei, the amino acid sequence of which is shown in the appended sequence listing as SEQ ID NO: 10. The invention also relates to nucleic acid constructs, vectors, and host cells comprising the polynucleotides as well as methods for producing and using the polypeptides, in particular within animal feed.</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td>TITLE: HAFNIA PHYTASE<br>
SEQUENCE LISTING AND DEPOSITED MICROORGANISMS Sequence listing<br>
The present application contains a paper copy and computer readable form of a sequence listing. The contents of the computer readable form are fully incorporated herein by reference.<br>
Deposit of biological material<br>
A phytase producing bacterial strain was isolated from Danish soil. The strain was demonstrated to produce a phytase with acidic pH optimum and high thermostability. The strain was identified as Hafnia alvei and it was deposited on March 21, 2007, under the terms of the Budapest Treaty with Deutsche Sammlung von Mikroorganismen und Zellkulturen (DSMZ) and given the following accession number;<br>
Deposit	Accession Number	Date of Deposit<br>
Hafnia alvei NN020125	DSM 19197	March 21,2007<br>
FIELD OF THE INVENTION<br>
The present invention relates to isolated polypeptides having phytase activity and isolated polynucleotides encoding the polypeptides. The polypeptides are related to a phytase derived from Hafnia alvei, the amino acid sequence of which is shown in the appended sequence listing as SEQ ID NO: 10. The invention also relates to nucleic acid constructs, vectors, and host cells comprising the polynucleotides as well as methods for producing and using the polypeptides, in particular within animal feed.<br>
BACKGROUND OF THE INVENTION<br>
Phytases are well-known enzymes, as are the advantages of adding them to foodstuffs for animals, including humans. Phytases have been isolated from very many sources, including a number of fungal and bacterial strains.<br>
It is an object of the present invention to provide alternative polypeptides having phytase activity and polynucleotides encoding the polypeptides. The polypeptides of the invention are preferably of amended, more preferably improved, properties, for example of a different substrate specificity, of a higher specific activity, of an increased stability (such as acid-stability, heat-stability, and/or protease stability, in particular pepsin stability), of an altered pH optimum (such as a lower, or higher pH optimum) and/or of an improved performance in animal feed (such as an improved release and/or degradation of phytate).<br>
 <br>
SUMMARY OF THE INVENTION<br>
The present invention relates to polypeptides having phytase activity, selected from the group consisting of: (a) a polypeptide having an amino acid sequence which has at least 75% identity with (i) amino acids 1 to 413 of SEQ ID NO: 10, and/or (ii) the mature polypeptide part of SEQ ID NO: 10; (b) a variant comprising a deletion, insertion, and/or conservative substitution of one or more amino acids of (i) amino acids 1 to 413 of SEQ ID NO: 10, and/or (ii) the mature polypeptide part of SEQ ID NO: 10; and/or (c) a fragment of (!) amino acids 1 to 413 of SEQ ID NO: 10, and/or (ii) the mature polypeptide part of SEQ ID NO: 10.<br>
The invention also relates to isolated polynucleotides encoding a polypeptide having phytase activity, selected from the group consisting of: (a) a polynucleotide encoding a polypeptide having an amino acid sequence which has at least 75% identity with amino acids 1 to 413 of SEQ ID NO: 10; and (b) a polynucleotide having at least 75% identity with nucleotides 100 to 1338 of SEQ ID NO: 9.<br>
The invention also relates to nucleic acid constructs, recombinant expression vectors, and recombinant host cells comprising the polynucleotides.<br>
The invention also relates to methods for producing such polypeptides having phytase activity comprising (a) cultivating a rcjcombinant host cell comprising a nucleic acid construct comprising a polynucleotide encoding the polypeptide under conditions conducive for production of the polypeptide; and (b) recovering the polypeptide.<br>
The invention further relates to a nucleic acid construct comprising a gene encoding a protein operably linked to a nucleotide sequence encoding a signal peptide consisting of (i) nucleotides 1 to 99 of SEQ ID NO: 11.<br>
The invention also relates to methods of using the phytases of the invention in animal feed, as well as animal feed and animal feed additive compositions containing the polypeptides.<br>
The invention also relates to methods of using the phytases of the invention in producing a fermentation product, such as, e.g., ethanoi, beer, wine, wherein the fermentation is carried out in the presence of a phytase of the present invention.<br>
The present invention relates to methods for treating proteins, including vegetable proteins, with the phytases of the present invention.<br>
BRIEF DESCRIPTION OF FIGURES<br>
Fig. 1 shows the residual inositol-phosphate bound phosphorous after in vitro incubation in a comparison between the Hafnia alvei phytase and a CItrobacter braakii phytase dosed from 125to500FYT/kgFeed.<br>
 <br>
Fig. 2 shows a comparison of the residual inositol-phosphate bound phosphorous after in vitro incubation between the f-iafnia alvei ph'/tase and a Peniophora lycii phytase phytase dosed at 250 FYT/kg Feed and 500 FYT/kg Feed,<br>
Fig. 3 shows the Appendix of the structural coordinates for the solved crystal three dimensional structure of the Hafnia alvei phytase.<br>
DEFINITIONS<br>
Phytase activity: In the present context a polypeptide having phytase activity (a phytase) is an enzyme which catalyzes the hydrolysis of phytate (myo-inositol hexakis-phosphate) to (1) myo-inositoi and/or (2) mono-, di-, tri-, tetra- and/or penta-phosphates thereof and (3) inorganic phosphate.<br>
The ENZYME site at the internet (www.expasy.ch/enzyme/) is a repository of information relative to the nomenclature of enzymes. It is primarily based on the recommendations of the Nomenclature Committee of the International Union of Biochemistry and Molecular Biology (lUB-MB) and it describes each type of characterized enzyme for which an EC (Enzyme Commission) number has been provided (Bairoch A. The ENZYME database, 2000, Nucleic Acids Res 28:304-305). See also the handbook Enzyme Nomenclature from NC-IUBMB, 1992).<br>
According to the ENZYME site, three different types of phytases are known: A 3-phytase (myo-inositol hexaphosphate 3-phosphohydrolase, EC 3.1.3.8), a 6-phytase (myo¬inositol hexaphosphate 6-phosphohydrolase, EC 3.1.3.26), and a 5-phytase (EC 3.1.3.72). For the purposes of the present invention, all types are included in the definition of phytase.<br>
In a particular embodiment, the phytases of the invention belong to the family of acid histidine phosphatases, which includes the Escherichia coli pH 2.5 acid phosphatase (gene appA) as well as fungal phytases such as Aspergillus awamorii phytases A and B (EC: 3.1.3.8) (gene phyA and phyB). The histidine acid phosphatases share two regions of sequence similarity, each centered around a conserved histidine residue. These two histidines seem to be involved in the enzymes' catalytic mechanism. The first histidine is located in the N-terminal section and forms a phosphor-histidine intermediate while the second is located in the C-terminal section and possibly acts as proton donor.<br>
In a further particular embodiment, the phytases of the invention have a conserved active site motif, viz. R-H-G-X-R-X-P, wherein X designates any amino acid (see amino acids 18 to 24 of the mature phytase shown in SEQ ID NO: 10).<br>
For the purposes of the present invention the phytase activity is determined in the unit of FYT, one FYT being the amount of enzyme that liberates 1 micro-mol inorganic ortho-phosphate per min. under the following conditions: pH 5.5; temperature 37°C; substrate:<br>
 <br>
sodium phytate (Cg H6024P6Nai2) in a concentration of 0.0050 mol/l. Suitable phytase assays are the FYT and FTU assays described in Example 1 of WO 00/20569. FTU is for determining phytase activity in feed and premix. Phytase activity may also be determined using the phytase assays of the examples herein.<br>
pH optimum: The pH-optimum of a polypeptide of the invention is determined by incubating the phytase at various pH-values, using a substrate in a pre-determined concentration and a fixed incubation temperature. The pH-optimum is then determined from a graphical representation of phytase activity versus pH. In a particular embodiment, the FYT assay is used, viz. the substrate is 5mM sodium phytate, the reaction temperature 37°C, and the activity is determined in FYT units at various pH-values, as done in the examples below. In another particular embodiment, the phytase assay of any one of the examples is used. A relatively low pH-optimum means a pH-optimum below pH 5.0, for example below pH 4.5, 4.0, 3.5, 3.0, 2.5, or even below 2.0. A relatively high pH-optimum means a pH-optimum above pH 5.0, for example above pH 5.5, 6.0, 6.5, 7.0, 7.5, 8.0, 8.5, or even above 9.0.<br>
Isolated polypeptide: The term "isolated polypeptide" as used herein refers to a polypeptide which is at least 20% pure, preferably at least 40% pure, more preferably at least 60% pure, even more preferably at least 80% pure, most preferably at least 90% pure, and even most preferably at least 95% pure, as determined by SDS-PAGE.<br>
Substantially pure polypeptide: The term "substantially pure polypeptide" denotes herein a polypeptide preparation which contains at most 10%, preferably at most 8%, more preferably at most 6%, more preferably at most 5%, more preferably at most 4%, at most 3%, even more preferably at most 2%, most preferably at most 1%, and even most preferably at most 0.5% by weight of other polypeptide material with which it is natively associated. It is, therefore, preferred that the substantially pure polypeptide is at least 92% pure, preferably at least 94% pure, more preferably at least 95% pure, more preferably at least 96% pure, more preferably at least 96% pure, more preferably at least 97% pure, more preferably at least 98% pure, even more preferably at least 99%, most preferably at least 99.5% pure, and even most preferably 100% pure by weight of the total polypeptide material present in the preparation.<br>
The polypeptides of the present invention are preferably in a substantially pure form. In particular, it is preferred that the polypeptides are in "essentially pure form", i.e., that the polypeptide preparation is essentially free of other polypeptide material with which it is natively associated. This can be accomplished, for example, by preparing the polypeptide by means of well-known recombinant methods or by classical purification methods.<br>
Herein, the term "substantially pure polypeptide" is synonymous with the terms "isolated polypeptide" and "polypeptide in isolated form."<br>
Identity: The relatedness between two amino acid sequences or between two nucleotide sequences is described by the parameter "identity".<br>
 <br>
For purposes of the present invention, the degree of identity between two amino acid sequences, as well as the degree of identity between two nucleotide sequences, is determined by the program "align" which is a Needleman-Wunsch alignment (i.e. global alignment), useful for both protein and DNA alignments. The default scoring matrix BLOSUM50 and the default identity matrix are used for protein and DNA alignments respectively. The penalty for the first residue in a gap is -12 for proteins and -16 for DNA. While the penalties for additional residues in a gap are -2 for proteins and -4 for DNA.<br>
"Align" is part of the FASTA package version v20u6 (see W. R. Pearson and D. J. Lipman (1988), "Improved Tools for Biological Sequence Analysis", PNAS 85:2444-2448, and W. R. Pearson (1990) "Rapid and Sensitive Sequence Comparison with FASTP and FASTA," Methods in Enzymology 183:63-98). FASTA protein alignments use the Smith-Waterman algorithm with no limitation on gap size (see "Smith-Waterman algorithm", T. F. Smith and M. S. Waterman (1981) J. Mol. Biol. 147:195-197).<br>
The Needleman-Wunsch algorithm is described in Needleman, S.B. and Wunsch, CD., (1970), Journal of Molecular Biology, 48: 443-453, and the align program by Myers and W. Miller in "Optimal Alignments in Linear Space" CABIOS (computer applications in the biosciences) (1988)4:11-17.<br>
The degree of identity between the target (or sample, or test) sequence and a specified sequence (e.g. amino acids 1 to 413 of the mature phytase shown in SEQ ID NO: 10) may also be determined as follovi's: The sequences are aligned using the program "align." The number of perfect matches ("N-perfect-match") in the alignment is determined (a perfect match means same amino acid residue in same position of the alignment, usually designated with a "I" in the alignment). The length of the specified sequence (the number of amino acid residues) is determined ("N-specified", in the example mentioned above = 413). The degree of identity is calculated as the ratio between "N-perfect-match" and "N-specified" (for conversion to percentage identity, multiply by 100).<br>
In an alternative embodiment, the degree of identity between a target (or sample, or test) sequence and the specified sequence (e.g. amino acids 1 to 413 of SEQ ID NO: 10) is determined as follows: The two sequences are aligned using the program "align." The number of perfect matches ("N-perfect-match") in the alignment is determined (a perfect match means same amino acid residue in same position of the alignment, usually designated with a "|" in the alignment). The common length of the two aligned sequences is also determined, viz. the total number of amino acids in the overlapping part of the alignment ("N-overlap"). The degree of identity is calculated as the ratio between "N-perfect-match" and "N-overlap" (for conversion to percentage identity, multiply by 100). In one embodiment, N-overlap includes trailing and leading gaps created by the alignment, if any. In another embodiment, N-overlap excludes trailing and leading gaps created by the alignment, if any.<br>
 <br>
In another alternative embodiment, the degree of identity between a target (or sample, or test) sequence and a specified sequence (e.g. amino acids 1 to 413 of SEQ ID NO: 10) is determined as follows: The sequences are aligned using the program "align." The number of perfect matches ("N-perfect-match") in the alignment is determined (a perfect match means same amino acid residue in same position of the alignment, usually designated with a "I" in the alignment). The length of the target sequence (the number of amino acid residues) is determined ("N-target"). The degree of identity is calculated as the ratio between "N-perfect-match" and "N-target" (for conversion to percentage identity, multiply by 100).<br>
Preferably, the overlap is at least 20% of the specified sequence ("N-overlap" as defined above, divided by the number of the amino acids in the specified sequence ("N-specified"), and multiplied by 100), more preferably at least 25%, 30%, 35%, 40%, 45%, 50%, 55%, 60%, 65%, 70%, 75%, 80%, 85%, 90%, or at least 95%. This means that at least 20% (preferably 25-95%) of the amino acids of the specified sequence end up being included in the overlap, when the sample sequence is aligned to the specified sequence.<br>
In the alternative, the overlap is at least 20% of the target (or sample, or test) sequence ("N-overlap" as defined above, divided by "N-target" as defined above, and multiplied by 100), more preferably at least 25%, 30%, 35%, 40%, 45%, 50%, 55%, 60%, 65%, 70%, 75%, 80%, 85%, 90%, or at least 95%. This means that at least 20% (preferably 25-95%) of the amino acids of the target sequence end up being included in the overlap, when aligned against the specified sequence).<br>
Polypeptide Fragment: The term "polypeptide fragment" is defined herein as a polypeptide having one or more amino acids deleted from the amino and/or carboxyl terminus of the mature peptide part of the specified sequence, e.g. SEQ ID NO: 10, or a homologous sequence thereof, wherein the fragment has phytase activity. In particular embodiments, the fragment contains at least 350, 360, 370, 380, 390, 400, 405, or at least 410 amino acid residues.<br>
Subsequence: The term "subsequence" is defined herein as a nucleotide sequence having one or more nucleotides deleted from the 5' and/or 3' end of the mature peptide encoding part of the specified sequence, e.g. SEQ ID NO: 9, or a homologous sequence thereof, wherein the subsequence encodes a polypeptide fragment having phytase activity. In particular embodiments, the subsequence contains at least 1050, 1080, 1110, 1140, 1170, 1200, 1215, 1230, 1245, 1260, 1275, 1290, 1305, 1320, or at least 1335 nucleotides.<br>
Allelic variant: The term "allelic variant" denotes herein any of two or more alternative forms of a gene occupying the same chromosomal locus. Allelic variation arises naturally through mutation, and may result in polymorphism within populations. Gene mutations can be silent (no change in the encoded polypeptide) or may encode polypeptides having altered amino acid sequences. An allelic variant of a polypeptide is a polypeptide<br>
 <br>
encoded by an allelic variant of a gene.<br>
Substantially pure polynucleotide: The term "substantially pure polynucleotide" as used herein refers to a polynucleotide preparation free of other extraneous or unwanted nucleotides and in a form suitable for use within genetically engineered protein production systems. Thus, a substantially pure polynucleotide contains at most 10%, preferably at most 8%, more preferably at most 6%, more preferably at most 5%, more preferably at most 4%, more preferably at most 3%, even more preferably at most 2%, most preferably at most 1%, and even most preferably at most 0.5% by weight of other polynucleotide material with which it is natively associated. A substantially pure polynucleotide may, however, include naturally occurring 5' and 3' untranslated regions, such as promoters and terminators. It is preferred that the substantially pure polynucleotide is at least 90% pure, preferably at least 92% pure, more preferably at least 94% pure, more preferably at least 95% pure, more preferably at least 96% pure, more preferably at least 97% pure, even more preferably at least 98% pure, most preferably at least 99%, and even most preferably at least 99.5% pure by weight. The polynucleotides of the present invention are preferably in a substantially pure form. In particular, it is preferred that the polynucleotides disclosed herein are in "essentially pure form", i.e., that the polynucleotide preparation is essentially free of other polynucleotide material with which it is natively associated. Herein, the term "substantially pure polynucleotide" is synonymous with the terms "isolated polynucleotide" and "polynucleotide in isolated form." The polynucleotides may be of genomic, cDNA, RNA, semisynthetic, synthetic origin, or any combinations thereof.<br>
Nucleic acid construct: The term "nucleic acid construct" as used herein refers to a nucleic acid molecule, either single- or double-stranded, which is isolated from a naturally occurring gene or which is modified to contain segments of nucleic acids in a manner that would not otherwise exist in nature. The term nucleic acid construct is synonymous with the term "expression cassette" when the nucleic acid construct contains the control sequences required for expression of a coding sequence of the present invention.<br>
Control sequence: The term "control sequences" is defined herein to include all components, which are necessary or advantageous for the expression of a polynucleotide encoding a polypeptide of the present invention. Each control sequence may be native or foreign to the nucleotide sequence encoding the polypeptide. Such control sequences include, but are not limited to, a leader, polyadenylation sequence, propeptide sequence, promoter, signal peptide sequence, and transcription terminator. At a minimum, the control sequences include a promoter, and transcriptional and translational stop signals. The control sequences may be provided with linkers for the purpose of introducing specific restriction sites facilitating ligation of the control sequences with the coding region of the nucleotide sequence encoding a polypeptide.<br>
 <br>
Operably linked: The term "operably linked" denotes herein a configuration in which a control sequence is placed at an appropriate position relative to the coding sequence of the polynucleotide sequence such that the control sequence directs the expression of the coding sequence of a polypeptide.<br>
Coding sequence: When used herein the term "coding sequence" means a nucleotide sequence, which directly specifies the amino acid sequence of its protein product. The boundaries of the coding sequence are generally determined by an open reading frame, which usually begins with the ATG start codon or alternative start codons such as GTG and TTG. The coding sequence may a DNA, cDNA, or recombinant nucleotide sequence.<br>
Mature polypeptide part: When used herein the terms "mature polypeptide part" or "mature peptide part" refer to that part of the polypeptide which is secreted by a cell which contains, as part of its genetic equipment, a polynucleotide encoding the polypeptide. In other words, the mature polypeptide part refers to that part of the polypeptide which remains after the signal peptide part is cleaved off once it has fulfilled its function of directing the encoded polypeptide into the cell's secretory pathway. The predicted signal peptide part of SEQ ID NO: 10 is amino acids -33 to -1 thereof, which means that the predicted mature polypeptide part of SEQ ID NO: 10 corresponds to amino acids 1 to 413 thereof. However, a slight variation may occur from host cell to host cell, and therefore the expression mature polypeptide part is preferred.<br>
Mature polypeptide encoding part: When used herein the term "mature polypeptide encoding part" or "mature polypeptide coding sequence" refers to that part of the polynucleotide encoding the polypeptide which encodes the mature polypeptide part. For example, for SEQ ID NO: 9, the predicted mature polypeptide encoding part corresponds to nucleotides 100 to 1338 (encoding amino acids 1 to 413 of SEQ ID NO: 10).<br>
Expression: The term "expression" includes any step involved in the production of the polypeptide including, but not limited to, transcription, post-transcriptional modification, translation, post-translationai modification, and secretion.<br>
Expression vector: The term "expression vector" is defined herein as a linear or circular DNA molecule that comprises a polynucleotide encoding a polypeptide of the invention, and which is operably linked to additional nucleotides that provide for its expression.<br>
Host cell: The term "host cell", as used herein, includes any cell type which is susceptible to transformation, transfection, transduction, and the like with a nucleic acid construct comprising a polynucleotide of the present invention.<br>
Modification: The term "modification" means herein any chemical modification of the specified polypeptide, e.g. the polypeptide consisting of the amino acids 1 to 413 of SEQ ID NO:  10, as well as genetic manipulation of the DNA encoding that polypeptide. The<br>
 <br>
modification(s) can be substitution(s), deletion(s) and/or insertions(s) of the annino acid(s) as well as rep!acement(s) of amino acid side chain(s).<br>
Artificial variant: When used herein, the term "artificial variant" means a polypeptide having phytase activity produced by an organism expressing a modified nucleotide sequence of mature phy1:ase encoding part of SEQ ID NO: 9. The modified nucleotide sequence is obtained through human intervention by modification of the mature phytase encoding part of the nucleotide sequence disclosed in SEQ ID NO: 9.<br>
DETAILED DESCRIPTION OF THE INVENTION Polypeptides Having Phytase Activity<br>
In a first aspect, the present invention relates to isolated polypeptides having phytase activity and having an amino acid sequence which has a degree of identity to amino acids 1 to 413 of SEQ ID NO: 10 (i.e., the mature polypeptide) of at least 75%.<br>
In particular embodiments, the degree of identity is at least 76%, 77%, 78%, 79%, 80%, 81%, 82%, 83%, 84%, 85%, 8B%, 87%, 88%, 89%, 90%, 91%, 92%, 93%, 94%, 95%, 96%, 96.5%, 97%, 97.5%, 98%, 98.5%, 98.7%, 98.8%, 98.9%, 99%, 99.0%, 99.1%, 99.2%, 99.3%, 99.4%, 99.5%, 99.6%, 99.7%, 99.8%, or at least 99.9%, which have phytase activity (hereinafter "homologous polypeptides").<br>
In other particular embodiments, the homologous polypeptides have an amino acid sequence which differs by 20, 19, 18, 17, 16, 15, 14, 13, 12, 11, 10, 9, 8, 7, 6, 5,4, 3, 2, or 1 amino acid from amino acids 1 to 413 of SEQ ID NO: 10.<br>
In particular embodiments^ the polypeptide of the present invention comprises the mature part of the amino acid sequence of SEQ ID NO: 10, or is an allelic variant thereof; or a fragment thereof that has phytase activity. In still further particular embodiments, the polypeptide comprises amino acids 1 to 413 of SEQ ID NO: 10, or an allelic variant thereof; or a fragment thereof that has phytase activity.<br>
In a second aspect, the present invention relates to isolated polypeptides having phytase activity which are encoded by polynucleotides which hybridize under at least medium, preferably medium, stringency conditions with (i) nucleotides 100 to 1338 of SEQ ID NO: 9, (ii) the mature polypeptide encoding part of SEQ ID NO: 9, and/or (iii) a complementary strand of any one of (i), and (ii), and/or (iv) a subsequence of (i), (ii), or (iii) (J. Sambrook, E.F. Fritsch, and T. Maniatus, 1989, Molecular Cloning, A Laboratory Manual, 2d edition. Cold Spring Harbor, New York), A subsequence of SEQ ID NO: 9 contains at least 100 contiguous nucleotides or preferably at least 200 contiguous nucleotides. Moreover, the subsequence may encode a polypeptide fragment which has phytase activity.<br>
In particular embodiments, the hybridization takes place under at least medium-high, at least high, or at least very high stringency conditions; preferably under medium-high, high,<br>
<br>
or very high stringency conditions.<br>
In alternative embodiments, the hybridization is conducted under very low, or low stringency conditions.<br>
The nucleotide sequence of SEQ ID NO: 9, or a subsequence thereof, as well as the amino acid sequence of SEQ ID NO: 10, or a fragment thereof, may be used to design a nucleic acid probe to identify and clone DNA encoding polypeptides having phytase activity from strains of different genera or species according to methods well known in the art. In particular, such probes can be used for hybridization with the genomic or cDNA of the genus or species of interest, following standard Southern blotting procedures, in order to identify and isolate the corresponding gene therein. Such probes can be considerably shorter than the entire sequence, but should be at least 14, preferably at least 25, more preferably at least 35, and most preferably at least 70 nucleotides in length. It is, however, preferred that the nucleic acid probe is at least 100 nucleotides in length. For example, the nucleic acid probe may be at least 200 nucleotides, preferably at least 300 nucleotides, more preferably at least 400 nucleotides, or most preferably at least 500 nucleotides in length. Even longer probes may be used, e.g., nucleic acid probes which are at least 600 nucleotides, at least preferably at least 700 nucleotides, more preferably at least 800 nucleotides, or most preferably at least 900 nucleotides in length. Both DNA and RNA probes can be used. The probes are typically labeled for detecting the corresponding gene (for example, with ^^P, ^H, ^^S, biotin, or avidin). Such probes are encompassed by the present invention.<br>
A genomic DNA library prepared from such other organisms may, therefore, be screened for DNA which hybridizes with the probes described above and which encodes a polypeptide having phytase activity. Genomic or other DNA from such other organisms may be separated by agarose or polyacrylamide gel electrophoresis, or other separation techniques. DNA from the libraries or the separated DNA may be transferred to and immobilized on nitrocellulose or other suitable carrier material. In order to identify a clone or DNA which Is homologous with SEQ ID NO: 9, or a subsequence thereof, the carrier material is used in a Southern blot.<br>
For purposes of the present invention, hybridization indicates that the nucleotide sequence hybridizes to a labeled nucleic acid probe corresponding to the nucleotide sequence shown in SEQ ID NO: 9, the complementary strand thereof, or a subsequence thereof, under very low to very high stringency conditions. Molecules to which the nucleic acid probe hybridizes under these conditions can be detected using X-ray film.<br>
In a particular embodiment, the nucleic acid probe is any one of SEQ ID NOs: 1-8. In another particular embodiment, the nucleic acid probe is the complementary strand of nucleotides 100 to 450, nucleotides 450 to 900, or nucleotides 900 to 1338 of SEQ ID NO: 9, in a further particular embodiment, the nucleic acid probe is a polynucleotide sequence which<br>
 <br>
encodes the mature part of the polypeptide of SEQ ID NO: 10, or a subsequence thereof. In a still further particular embodiment, the nucleic acid probe is SEQ ID NO: 9, in particular any one of the mature polypeptide coding regions thereof.<br>
For long probes of at least 100 nucleotides in length, very low to very high stringency conditions are defined as prehybridization and hybridization at 42°C in 5X SSPE, 0.3% SDS, 200 microgram/ml sheared and denatured salmon sperm DNA, and either 25% formamide for very low and low stringencies, 35% formamide for medium and medium-high stringencies, or 50% formamide for high and very high stringencies, following standard Southern blotting procedures for 12 to 24 hours optimally.<br>
For long probes of at least 100 nucleotides in length, the carrier material is finally washed three times each for 15 minutes using 2X SSC, 0.2% SDS preferably at least at 45°C (very low stringency), more preferably at least at 50°G (low stringency), more preferably at least at 55°C (medium stringency), more preferably at least at 60°C (medium-high stringency), even more preferably at least at 65°C (high stringency), and most preferably at least at 70°C (very high stringency).<br>
For short probes which are about 15 nucleotides to about 70 nucleotides in length, stnngency conditions are defined as prehybridization, hybridization, and washing post-hybridization at about 5°C to about 10°C below the calculated T^ using the calculation according to Bolton and McCarthy (1962, Proceedings of the National Academy of Sciences USA 48:1390) in 0.9 M NaCI, 0.09 M Tris-HCI pH 7.6, 6 mM EDTA, 0.5% NP-40, 1X Denhardt's solution, 1 mM sodium pyrophosphate, 1 mM sodium monobasic phosphate, 0.1 mM ATP, and 0.2 mg of yeast RNA per ml following standard Southern blotting procedures.<br>
For short probes which are about 15 nucleotides to about 70 nucleotides in length, the carrier material is washed once in 6X SCO plus 0.1% SDS for 15 minutes and twice each for 15 minutes using 6X SSC at 5°C to 10°C below the calculated T^.<br>
Under salt-containing hybridization conditions, the effective T^ is what controls the degree of identity required between the probe and the filter bound DNA for successful hybridization. The effective T^ may be determined using the formula below to determine the degree of identity required for two DNAs to hybridize under various stringency conditions.<br>
Effective T^ = 81.5 + 16.6(log M[Na1) + 0.41 (%G+C) - 0.72(% formamide)<br>
(See www.ndsu.nodak.edu/instruct/mcclean/plsc731/dna/dna6.htm)<br>
"G+C" designates the content of nucleotides G and T in the probe. For medium stringency, for example, the formamide is 35% and the Na"" concentration for 5X SSPE is 0.75 M.<br>
In one aspect, the present invention relates to isolated polypeptides having phytase activity, and the following physicochemical properties (as analyzed on the substantially pure polypeptides):<br>
 <br>
(i) a high specific activity, such as a specific activity on phytate of at least 50% of the specific activity of E. coli appA (SPTREMBL:Q8GN88), the specific activity being preferably measured in the units of FYT per mg phytase enzyme protein;<br>
(ii)     acid-stability; such as<br>
(a)	at least 60%, preferably at least 65%, at least 70%, or at least 75%, residual activity after incubation over night at 37°C in glycine/hydrochloric acid buffer pH 2.2, relative to the residual activity after incubation over night at 37°C in HEPES buffer pH 7.0;<br>
(b)	at least 80%, preferably at least 85%, at least 90%, or at least 95%, residual activity after incubation over night at 37°C in glycine/hydrochloric acid buffer pH 3.0, relative to the residual activity after incubation over night at 37°C in HEPES buffer pH 7.0; and/or<br>
(c)	a residual phytase activity after 2 hours incubation at a temperature of 25, 30, 35, or 37X, preferably 37°C, and a pH of 2.2, 2.4, 2.5, 2.6, 2.8, 3.0, 3.2, 3.4, or 3.5, preferably glycine/hydrochloric acid buffers of pH 2.2, or 3.0, of at least 50%, compared to the residual activity of E. coli appA (SPTF?EMBL:Q8GN88);<br>
(iii) heat-stability, such as a residual phytase activity after 0.5, 1, 1.5, or 2 hours, preferably 0.5 hours, of incubation at a pH of 5.5 and a temperature of 55, 60, 65, 70, 75, 80, 85 or 95°C, preferably 70°C, of at least 50%, compared to the residual activity of E. coli appA (SPTREMBL:Q8GN88);<br>
In the alternative. Differential Scanning Calorimetry (DSC) measurements may be used to determine the denaturation temperature, Td, of the purified phytase protein. The Td is indicative of the heat-stability of the protein: The higher the Td, the higher the heat-stability. DSC measurements may be performed at various pH values, e.g. using the VP-DSC from Micro Cai. Scans are performed at a constant scan rate of 1.5°C/min from 20-90°C. Preferred pH values are 4.0 and 5.5, preferably 4.0. Before running the DSC, the phytases are desalted, e.g. using NAP-5 columns (Pharmacia) equilibrated in appropriate buffers (e.g. 25mM sodium acetate pH 4.0; 0.1 M sodium acetate, pH5.5). Data-handling is performed using the MicroCal Origin software (version 4.10), and the denaturation temperature, Td (also called the melting temperature, Tm) is defined as the temperature at the apex of the peak in the thermogram.<br>
(iv) protease-stability, such as a residual phytase activity after 0,5, 1, 1.5, or 2 hours, preferably 1 hour, incubation at a temperature of 20, 25, 30, 35, or 37°C, preferably 37°C, and a pH of 5.5, in the presence of 0.1 mg/ml pepsin, of at least 50%, compared to the residual activity of E. coli appA (SPTREMBL:Q8GN88); and/or<br>
(v) a pH-optimum belovi/ pH 5.0, for example below pH 4.5, 4.0, 3.5, 3.0, 2.5, or even below 2.0, determined using the FYT assay, and/or using the assay of Example 4, as described hereinbefore.<br>
In particular embodiments of aspect (i) above, the specific activity is at least 60, 70,<br>
 <br>
80, 90, 100, 110, 120, 130, 140, or at least 150% of the specific activity of E. coli appA. In particular embodiments of each of aspects (ii) to (iv) above, the residual activity is at least 60, 70, 80, 90, 100, 110, 120, 130, 140, or at least 150% of the residual activity of E. coli appA.<br>
In a fifth aspect, the activity of the enzyme of the invention, at pH 5.0 and 37°C, measured on the substrate pNP-phosphate is less than 11% of the activity of the enzyme measured on the substrate phytate. F'referably, the ratio is less than 10%, 9%, 8%, 7%, 6%, or less than 5%. The ratio of pNP to phytate hydrolysis is indicative of the true phytase nature of the enzyme. A high ratio of activity on pNP relative to activity on phytate may indicate that the enzyme in question is a phosphatase w/ith relatively low substrate specificity, whereas a low ratio indicates that this is an enzyme more specifically accepting phytate as a substrate.<br>
In a sixth aspect, the phytase of the invention has a higher release of phosphorous (P) in an in vitro model, as compared to the phytase from Peniophora lycii, preferably at least 110 % thereof, more preferably at least 120%, 130%, or at least 140% thereof. In one embodiment, the phytase of the invention, dosed 0.25 FYT/g feed, releases at least 150% phosphorous (P), relative to the phosphorous released by the phytase from Peniophora lycii, also dosed 0.25 FYT/g feed, in the in vitro model. Preferably, the release is at least 155%, 160%, 165%, 170%, 175%, or at least 180%. In another embodiment, the phytase of the invention, dosed 0.75 FYT/g feed, releases at least 150% phosphorous (P), relative to the phosphorous released by the phytase from Peniophora lycii, also dosed 0.75 FYT/g feed, in the in vitro model. Preferably, the release is at least 155%, 160%, 165%, 170%, 175%, 180%, 185%, or at least 190%.<br>
In a seventh aspect, the phytase of the invention has a residual activity following incubation at 37°C and in a 0.1 M Glycine/HCI buffer, pH 2.0, for 4 hours of at least 20%, as compared to the activity at time, t = 0, the activity (and the residual activity) being assayed at 37°C and pH 5.5 on 1% (w/v) Na-phytate, using a 0.25 M Na-acetate buffer pH 5.5, buffer blind subtracted. In preferred embodiments, the residual activity is at least 25%, 30%, 35%, 40%, 45%, 50%, 55%, 60%, 65%, 70%, 75%, or at least 80%. In another embodiment, the phytase of the invention has a residual activity following incubation at 37°C and in a 0.1M Glycine/HCI buffer, pH 2.5, for 1 day (24 hours) of at least 20%, as compared to the activity at time, t = 0, the activity (and the residual activity) being assayed at 37°C and pH 5.5 on 1% (w/v) Na-phytate, using a 0.25 M Na-acetate buffer pH 5.5, buffer blind subtracted. In preferred embodiments, the residual activity is at least 25%, 30%, 35%, 40%, 45%, 50%, 55%, 60%), 65%, 70%, 75%, or at least 80%.<br>
In an eighth aspect, the present invention relates to artificial variants comprising a conservative substitution, deletion, and/or insertion of one or more amino acids of SEQ ID NO: 10, or the mature polypeptide thereof. An insertion can be inside the molecule, and/or at the N- and/or C-terminal end of the molecule in which case it is also designated extension.<br>
 <br>
Preferably, amino acid changes are of a minor nature, that is conservative amino acid substitutions; small deletions, typically of one to about 30 amino acids; small amino- or carboxyl-terminal extensions, such as an amino-terminal methionine residue; a small linker peptide of up to about 20-25 residues; or a small extension that facilitates purification by changing net charge or another function, such as a poly-histidine tract, an antigenic epitope or a binding domain - in other v^/ords: Changes that do not significantly affect the folding and/or activity of the protein.<br>
Examples of conservative substitutions are v\/ithin the group of basic amino acids (arginine, lysine and histidine), acidic amino acids (glutamic acid and aspartic acid), polar amino acids (glutamine and asparagine), hydrophobic amino acids (leucine, isoleucine and valine), aromatic amino acids (phenylalanine, tryptophan and tyrosine), and small amino acids (glycine, alanine, serine, threonine and methionine).<br>
Other examples of conservative substitutions are substitutions of the 20 standard amino acids with non-standard amino acids (such as 4-hydroxyproline, 6-N-methyl lysine, 2-aminoisobutyric acid, isovaline, and alpha-methyl serine). Conservative substitutions may also include a substitution into amino acids that are not encoded by the genetic code, and unnatural amino acids. "Unnatural amino acids" have been modified after protein synthesis, and/or have a chemical structure in their side chain(s) different from that of the standard amino acids. Unnatural amino acids can be chemically synthesized, and preferably, are commercially available, and include pipecolic acid, thiazolidine carboxylic acid, dehydroproline, 3- and 4-methylproline, and 3,3-dimethylproline,<br>
Alternatively, the amino acid changes are of such a nature that the physico-chemical properties of the polypeptides are altered. For example, amino acid changes may improve the thermal stability of the polypeptide, alter the substrate specificity, change the pH optimum, and the like.<br>
Essential amino acids in the parent polypeptide can be identified according to procedures known in the art, such as site-directed mutagenesis or alanine-scanning mutagenesis (Cunningham and Wells, 1989, Science 244: 1081-1085). In the latter technique, single alanine mutations are introduced at every residue in the molecule, and the resultant mutant molecules are tested for biological activity (i.e., phytase activity) to identify amino acid residues that are critical to the activity of the molecule. See also, Hilton et al., 1996, J. Biol. Chem. 271: 4699-4708. The active site of the enzyme or other biological interaction can also be determined by physical analysis of structure, as determined by such techniques as nuclear magnetic resonance, crystallography, electron diffraction, or photoaffinity labeling, in conjunction with mutation of putative contact site amino acids. See, for example, de Vos et al., 1992, Science 255: 306-312; Smith et al., 1992, J. Mol. Biol. 224: 899-904; Wlodaver et al., 1992, FEBS Lett. 309:59-64. The identities of essential amino acids<br>
 <br>
can also be inferred from analysis of identities with polypeptides which are related to a polypeptide according to the invention<br>
Single or multiple amino acid substitutions can be made and tested using known methods of mutagenesis, recombination, and/or shuffling, followed by a relevant screening procedure, such as those disclosed by Reidhaar-Olson and Sauer, 1988, Science 241: 53-57; Bowie and Sauer, 1989, Proc. Natl, Acad. Sci. USA 86: 2152-2156; WO 95/17413; or WO 95/22625, Other methods that can be used include error-prone PCR, phage display (e.g., Lowman et al., 1991, Biochem. 30:10832-10837; U.S. Patent No. 5,223,409; WO 92/06204), and region-directed mutagenesis (Derbyshire et al., 1986, Gene 46:145; Ner et a!., 1988, DNA 7:127),<br>
Mutagenesis/shuffling methods can be combined with high-throughput, automated screening methods to detect activity of cloned, mutagenized polypeptides expressed by host cells. Mutagenized DNA molecules that encode active polypeptides can be recovered from the host cells and rapidly sequenced using standard methods in the art. These methods allow the rapid determination of the importance of individual amino acid residues in a polypeptide of interest, and can be applied to polypeptides of unknown structure.<br>
The total number of amino acid substitutions (preferably conservative substitutions), deletions and/or insertions in the sequence of amino acids 1 to 413 of SEQ ID NO: 10 is at most 10, preferably at most 9, more preferably at most 8, more preferably at most 7, more preferably at most 6, more preferably at most 5, more preferably at most 4, even more preferably at most 3, most preferably at most 2, and even most preferably 1.<br>
The total number of amino acid substitutions, deletions and/or insertions of amino acids 1 to 413 of SEQ ID NO: 10 is 10, preferably 9, more preferably 8, more preferably 7, more preferably at most 6, more preferably at most 5, more preferably 4, even more preferably 3, most preferably 2, and even most preferably 1. In the alternative, the total number of amino acid substitutions (preferably conservative substitutions), deletions and/or insertions in the sequence of amino acids 1 to 413 of SEQ ID NO: 10 is at most 50, 45, 40, 35, 30, 25,20, 19, 18, 17, 16, 15. 14, 13, 12, or at most 11.<br>
In a specific embodiment, the polypeptide of the invention is a low-allergenic variant, designed to invoke a reduced immunological response when exposed to animals, including man. The term immunological response is to be understood as any reaction by the immune system of an animal exposed to the polypeptide. One type of immunological response is an allergic response leading to increased levels of IgE in the exposed animal. Low-allergenic variants may be prepared using techniques known in the art. For example the polypeptide may be conjugated with polymer moieties shielding portions or epitopes of the polypeptide involved in an immunological response. Conjugation with polymers may involve in vitro chemical coupling of polymer to the  polypeptide,  e.g.  as described  in WO  96/17929,<br>
 <br>
VVO98/30682. WO98/35026, and/or WO99/00489. Conjugation may in addition or alternatively thereto involve in vivo coupling of polymers to the polypeptide. Such conjugation may be achieved by genetic engineering of the nucleotide sequence encoding the polypeptide, inserting consensus sequences encoding additional glycosylation sites in the polypeptide and expressing the polypeptide in a host capable of glycosylating the polypeptide, see e.g. WOOO/26354, Another way of providing low-allergenic variants is genetic engineering of the nucleotide sequence encoding the polypeptide so as to cause the polypeptide to self-oligomerize, effecting that polypeptide monomers may shield the epitopes of other polypeptide monomers and thereby lowering the antigenicity of the oligomers. Such products and their preparation is described e.g. in W096/16177. Epitopes involved in an immunological response may be identified by various methods such as the phage display method described in WO 00/26230 and WO 01/83559, or the random approach described in EP 561907. Once an epitope has been identified, its amino acid sequence may be altered to produce altered immunological properties of the polypeptide by known gene manipulation techniques such as site directed mutagenesis (see e.g. WO 00/26230, WO 00/26354 and/or WOOO/22103) and/or conjugation of a polymer may be done in sufficient proximity to the epitope for the polymer to shield the epitope.<br>
Three Dimensional Structure of a Hafnia alvei Phytase<br>
The three-dimensional structure of a Hafnia alvei phytase (amino acids 1 to 413 of SEQ ID NO: 10) is provided in the Appendix. The structure was solved in accordance with the principles for x-ray crystallographic methods, for example, as given in X-Ray Structure Determinations, Stout, G. K. and Jensen, L. H., John Wiley and Sons, Inc. NY 1989. The structural coordinates for the solved crystal structure of Hafnia alvei phytase are given in standard PDB format (Protein Database Bank, Brookhaven National Laboratory, Brookhaven, Conn.) as set forth in the Appendix. It is to be understood that the Appendix forms part of the present application. The Appendix provides the coordinates of the heavy atoms, excluding the hydrogen atoms. The first three residues of the enzyme were not visible in the crystal structure as well as the amino acid residues between amino acids 180 and 189. However, the structure between 180 and 189 was built using modelling combining the homology modelling (see, for example, Marti-Renom et al., 2000) program NEST from the JACKAL package (wiki.c2b2.columbia.edu/honiglab_public/index.php/Software:Jackal) and the simulation software called CHARMm (//accelrys.com/products/scitegic/component-collections/charmm.html).<br>
Sources of Polypeptides Having Phytase Activity<br>
A polypeptide of the present invention may be obtained from microorganisms of any<br>
 <br>
genus. For purposes of the present inveniion, the term "obtained from" as used herein in connection with a given source shall mean that the polypeptide encoded by a nucleotide sequence is produced by the source or by a strain in which the nucleotide sequence from the source has been inserted. In a preferred aspect, the polypeptide obtained from a given source is secreted extracellularly.<br>
A polypeptide of the present invention may be a bacterial polypeptide. For example, the polypeptide may be a gram positive bacterial polypeptide such as a Bacillus polypeptide, or a Streptomyces polypeptide; or a gram negative bacterial polypeptide, e.g., an Escherichia coli. Yersinia, Klebsiella, Citrobacter, or a Pseudomonas polypeptide. In a particular embodiment, the polypeptide is derived from Proteobacteria, such as Gammaproteobacteria, for example Enterobacteriales, such as Enterobacteriaceae.<br>
In a particular aspect, the polypeptide derived from Enterobacteriaceae is a Hafnia polypeptide, such as a Hafnia alvei species polypeptide.<br>
A polypeptide of the present invention may also be a fungal polypeptide, such as a yeast polypeptide or a filamentous fungal polypeptide.<br>
Strains of the above microorganisms are readily accessible to the public in a number of culture collections, such as the American Type Culture Collection (ATCC), Deutsche Sammlung von Mikroorganismen und Zellkulturen GmbH (DSM), Centraalbureau Voor Schimmelcultures (CBS), and Agricultural Research Service Patent Culture Collection, Northern Regional Research Center (NRRL).<br>
Furthermore, such polypeptides may be identified and obtained from other sources including microorganisms isolated from nature (e.g., soil, composts, water, etc.) using the above-mentioned probes. Techniques for isolating microorganisms from natural habitats are w/ell known in the art. The polynucleotide may then be obtained by similarly screening a genomic or cDNA library of another microorganism. Once a polynucleotide sequence encoding a polypeptide has been detected with the probe(s), the polynucleotide can be isolated or cloned by utilizing techniques which are well known to those of ordinary skill in the art (see, e.g., Sambrook et al., 1989, supra).<br>
Polypeptides of the present invention also include fused polypeptides or cleavable fusion polypeptides in which another polypeptide is fused at the N-terminus or the C-terminus of the polypeptide or fragment thereof. A fused polypeptide is produced by fusing a nucleotide sequence (or a portion thereof) encoding another polypeptide to a nucleotide sequence (or a portion thereof) of the present invention. Techniques for producing fusion polypeptides are known in the art, and include ligating the coding sequences encoding the polypeptides so that they are in frame and that expression of the fused polypeptide is under control of the same promoter(s) and terminator.<br>
 <br>
Polynucieotides<br>
The present invention also relates to isolated polynucleotides having a nucleotide sequence which encodes a polypeptide of the present invention. In a preferred aspect, the nucleotide sequence is set forth in SEQ ID NO: 9. In another preferred aspect, the nucleotide sequence is the mature polypeptide coding region of SEQ ID NO: 9. The present invention also encompasses nucleotide sequences which encode a polypeptide having the amino acid sequence of SEQ ID NO: 10, or the mature polypeptides thereof, which differ from SEQ ID NO: 9, by virtue of the degeneracy of the genetic code. The present invention also relates to subsequences of SEQ ID NO: 9, which encode fragments of SEQ ID NO: 10, that have phytase activity.<br>
The present invention also reflates to mutant polunucleotides comprising at least one mutation in the mature polypeptide coding sequence of any one of SEQ ID NO: 9, in which the mutant nucleotide sequence encodes a polypeptide which consists of amino acids 1 to 413 of SEQ ID NO: 10.<br>
The techniques used to isolate or clone a polynucleotide encoding a polypeptide are known in the art and include isolation from genomic DNA, preparation from cDNA, or a combination thereof. The cloning of the polynucleotides of the present invention from such genomic DNA can be effected, e.g., by using the well known polymerase chain reaction (PCR) or antibody screening of expression libraries to detect cloned DNA fragments with shared structural features. See, e.g., Innis et al., 1990, PCR: A Guide to Methods and Application, Academic Press, New York. Other nucleic acid amplification procedures such as ligase chain reaction (LOR), ligated activated transcription (LAT) and nucleotide sequence-based amplification (NASBA) may be used. The polynucleotides may be cloned from a strain of Hafnia, or another or related organism and thus, for example, may be an allelic or species variant of the polypeptide encoding region of the nucleotide sequence.<br>
The present invention also relates to polynucleotides having nucleotide sequences which have a degree of identity to the mature polypeptide coding sequence of SEQ ID NO: 9 (i.e., nucleotides 100 to 1338) of at least 75%, and which encode a polypeptide having phytase activity. In particular embodiments, the degree of identity is at least In particular embodiments, the degree of identity is at least 76%, 77%, 78%, 79%, 80%, 81%, 82%, 83%, 84%, 85%, 86%, 87%, 88%, 89%,, 90%, 91%, 92%, 93%, 94%, 95%, 96%, 96.5%, 97%, 97.5%, 98%, 98.5%, 98.7%, 98.8%, 98.9%, 99%, 99.0%, 99.1%, 99.2%, 99.3%, 99.4%, 99.5%, 99.6%, 99.7%, 99.8%, or at least 99.9%. In alternative embodiments, the degree of identity is at least 75%, 80%, 85%, 90%, 94, 97, 98, 98.0, 98.1, 98.2, or at least 98.3%.<br>
Modification of a nucleotide sequence encoding a polypeptide of the present invention may be necessary for the synthesis of polypeptides substantially similar to the polypeptide.  The term "substantially similar" to the polypeptide  refers to non-naturally<br>
 <br>
occurring forms of the polypeptide. These polypeptides may differ in some engineered way from the polypeptide isolated from its native source, e.g., artificial variants that differ in specific activity, thermostability, pH-optimum, or the like. The variant sequence may be constructed on the basis of the nucleotide sequence presented as the polypeptide encoding region of SEQ ID NO: 9, e.g., a sub.sequence thereof, and/or by introduction of nucleotide substitutions which do not give rise to another amino acid sequence of the polypeptide encoded by the nucleotide sequence, but which correspond to the codon usage of the host organism intended for production of the polypeptide, or by introduction of nucleotide substitutions which may give rise to a different amino acid sequence. For a general description of nucleotide substitution, see, e.g., Ford et al., 1991, Protein Expression and Purification 2: 95-107.<br>
It will be apparent to those skilled in the art that such substitutions can be made outside the regions critical to the function of the molecule and still result in an active polypeptide. Amino acid residues essential to the activity of the polypeptide encoded by an isolated polynucleotide of the invention, and therefore preferably not subject to substitution, may be identified according to procedures known in the art, such as site-directed mutagenesis or alanine-scanning mutagenesis (see, e.g., Cunningham and Wells, 1989, Science 244: 1081-1085). In the latter technique, mutations are introduced at every positively charged residue in the molecule, and the resultant mutant molecules are tested for phytase activity to identify amino acid residues that are critical to the activity of the molecule. Sites of substrate-polypeptide interaction can also be determined by analysis of the three-dimensional structure as determined by such techniques as nuclear magnetic resonance analysis, crystallography or photoaffinity labelling (see, e.g., de Vos et al., 1992, Science 255: 306-312; Smith et al., 1992, Journal of Molecular Biology 224: 899-904; Wlodaver et al., 1992, FEBS Letters 309: 59-64).<br>
The present invention also relates to isolated polynucleotides encoding a polypeptide of the present invention, which hybridize under medium stringency conditions, more preferably medium-high stringency conditions, even more preferably high stringency conditions, and most preferably very high stringency conditions with (i) nucleotides 100 to 1338 of SEQ ID NO: 9, (ii) the mature polypeptide encoding part of SEQ ID NO: 9, and/or (Hi) a complementary strand of any one of (i), and/or (ii); or allelic variants and subsequences thereof (Sambrook et al., 1989, supra), as defined herein. In alternative embodiments the hybridization is conducted under very low, or low, stringency conditions.<br>
The present invention also relates to isolated polynucleotides obtained, or obtainable, by (a) hybridizing a population of DNA under very low, low, medium, medium-high, high, or very high stringency conditions with (i) nucleotides 100 to 1338 of SEQ ID NO: 9, (ii) the mature polypeptide encoding part of SEQ ID NO: 9, and/or (iii) a complementary<br>
 <br>
strand of any one of (i), and/or (ii); and (b) isolatincj the hybridizing polynucleotide, which encodes a polypeptide having phytase activity.<br>
Nucleic Acid Constructs<br>
The present invention also relates to nucleic acid constructs comprising an isolated polynucleotide of the present invention operably linked to one or more control sequences which direct the expression of the coding sequence in a suitable host cell under conditions compatible with the control sequences.<br>
An isolated polynucleotide encoding a polypeptide of the present invention may be manipulated in a variety of ways to provide for expression of the polypeptide. Manipulation of the polynucleotide's sequence prior to its insertion into a vector may be desirable or necessary depending on the expression vector. The techniques for modifying polynucleotide sequences utilizing recombinant DNA methods are well known in the art.<br>
The control sequence may be an appropriate promoter sequence, a nucleotide sequence which is recognized by a host cell for expression of a polynucleotide encoding a polypeptide of the present invention, The promoter sequence contains transcriptional control sequences which mediate the expression of the polypeptide. The promoter may be any nucleotide sequence which shows transcriptional activity in the host cell of choice including mutant, truncated, and hybrid promoters, and may be obtained from genes encoding extracellular or intracellular polypeptides either homologous or heterologous to the host cell.<br>
Examples of suitable promoters for directing the transcription of the nucleic acid constructs of the present invention, especially in a bacterial host cell, are the promoters obtained from the E. coli lac operon, Streptomyces coelicolor agarase gene (dagA), Bacillus subtilis levansucrase gene (sacB), Bacillus licheniformis alpha-amylase gene (amyL), Bacillus stearothermophilus maltogenic amylase gene (amyM), Bacillus amyloliquefaciens alpha-amylase gene (amyQ), Bacillus licheniformis penicillinase gene (penP), Bacillus subtilis xylA and xylB genes, and prokaryotic beta-lactamase gene (Villa-Kamaroff et al., 1978, Proceedings of the National Academy of Sciences USA 75: 3727-3731), as well as the tac promoter (DeBoer et al., 1983, Proceedings of the National Academy of Sciences USA 80: 21-25). Further promoters are described in "Useful proteins from recombinant bacteria" in Scientific American, 1980, 242: 74--94; and in Sambrook et al., 1989, supra.<br>
Examples of suitable promoters for directing the transcription of the nucleic acid constructs of the present invention in a filamentous fungal host cell are promoters obtained from the genes for Aspergillus oryzae TAKA amylase, Rhizomucor miehei aspartic proteinase, Aspergillus niger neutral alpha-amylase, Aspergillus niger acid stable alpha-amylase, Aspergillus niger or Aspergillus awamori glucoamylase (glaA), Rhizomucor miehei lipase, Aspergillus oryzae alkaline protease, Aspergillus oryzae triose phosphate isomerase,<br>
 <br>
Aspergillus nidulans acetamidase, Fusarium venenatum amyloglucosidase (WO 00/56900), Fusarium venenatum Daria (WO 00/56900), Fusarium venenatum Quinn (WO 00/56900), Fusarium oxysporum trypsin-like protease (WO 96/00787), Trichoderma reesei beta-glucosidase, Trichoderma reesei cellobiohydrolase I, Trichoderma reesei endoglucanase I, Trichoderma reesei endoglucanase II, Trichoderma reesei endoglucanase III, Trichoderma reesei endoglucanase IV, Trichoderma reesei endoglucanase V, Trichoderma reesei xylanase I, Trichoderma reesei xylanase II, Trichoderma reesei beta-xylosidase, as well as the NA2-tpi promoter (a hybrid of the promoters from the genes for Aspergillus niger neutral alpha-amylase and Aspergillus oryzae triose phosphate isomerase); and mutant, truncated, and hybrid promoters thereof.<br>
In a yeast host, useful promoters are obtained from the genes for Saccharomyces cerevisiae enolase (ENO-1), vSaccharomyces cerevisiae galactokinase (GAL1), Saccharomyces cerevisiae alcohol dehydrogenase/glyceraldehyde-3-phosphate dehydrogenase (ADH1,ADH2/GAP), Saccharomyces cerevisiae triose phosphate isomerase (TPI), Saccharomyces cerevisiae metallothionine (CUP1), Saccharomyces cerevisiae 3-phosphoglycerate kinase, and Pichia pastoris alcohol oxidase (A0X1). Other useful promoters for yeast host cells are described by Romanes et al., 1992, Yeast 8: 423-488.<br>
The control sequence may also be a suitable transcription terminator sequence, a sequence recognized by a host cell to terminate transcription. The terminator sequence is operably linked to the 3' terminus of the nucleotide sequence encoding the polypeptide. Any terminator which is functional in the host cell of choice may be used in the present invention.<br>
Preferred terminators for filamentous fungal host cells are obtained from the genes for Aspergillus oryzae TAKA amylase, Aspergillus niger glucoamylase, Aspergillus nidulans anthraniiate synthase, Aspergillus niger alpha-glucosidase, and Fusarium oxysporum trypsin-like protease.<br>
Preferred terminators for yeast host ceils are obtained from the genes for Saccharomyces cerevisiae enolase, Saccharomyces cerevisiae cytochrome C (CYC1), and Saccharomyces cerevisiae glyceraldehyde-3-phosphate dehydrogenase. Other useful terminators for yeast host cells are described by Romanes et al., 1992, supra.<br>
The control sequence may also be a suitable leader sequence, a nontranslated region of an mRNA which is important for translation by the host cell. The leader sequence is operably linked to the 5' terminus of the nucleotide sequence encoding the polypeptide. Any leader sequence that is functional in the host cell of choice may be used in the present invention.<br>
The control sequence may also be a suitable leader sequence, a nontranslated region of an mRNA which is important for translation by the host cell. The leader sequence is operably linked to the 5' terminus of the nucleotide sequence encoding the polypeptide. Any<br>
 <br>
leader sequence that is functional in the host cell of ctioice may be used in the present invention.<br>
Preferred leaders for filamentous funga! host cells are obtained from the genes for Aspergillus oryzae TAKA amylase and Aspergillus nidulans triose phosphate isomerase.<br>
Suitable leaders for yeast host cells are obtained from the genes for Saccharomyces cerevisiae enolase (ENO-1), Saccharomyces cerevisiae 3-phosphoglycerate kinase, Saccharomyces cerevisiae alpha-factor, and Saccharomyces cerevisiae alcohol dehydrogenase/giyceraldehyde-3-phosphate dehydrogenase (ADH2/GAP).<br>
The control sequence may also be a polyadenylation sequence, a sequence operably linked to the 3' terminus of the nucleotide sequence and w/hich, w/hen transcribed, is recognized by the host cell as a signal to add polyadenosine residues to transcribed mRNA. Any polyadenylation sequence which is functional in the host cell of choice may be used in the present invention.<br>
Preferred polyadenylation sequences for filamentous fungal host cells are obtained from the genes for Aspergillus oryzae TAKA amylase, Aspergillus niger glucoamylase, Aspergillus nidulans anthraniiate synthase, Fusarium oxysporum trypsin-like protease, and Aspergillus niger alpha-glucosidase.<br>
Useful polyadenylation seo|uences for yeast host cells are described by Guo and Sherman, 1995, Molecular Cellular Biology 15: 5983-5990.<br>
The control sequence may also be a signal peptide coding region that codes for an amino acid sequence linked to the amino terminus of a polypeptide and directs the encoded polypeptide into the cell's secretoi"y pathway. The 5' end of the coding sequence of the nucleotide sequence may inherently contain a signal peptide coding region naturally linked in translation reading frame with the segment of the coding region which encodes the secreted polypeptide. Alternatively, the 5' end of the coding sequence may contain a signal peptide coding region which is foreign to the coding sequence. The foreign signal peptide coding region may be required where the coding sequence does not naturally contain a signal peptide coding region. Alternatively, the foreign signal peptide coding region may simply replace the natural signal peptide coding region in order to enhance secretion of the polypeptide. However, any signal peptide coding region which directs the expressed polypeptide into the secretory pathway of a host cell of choice may be used in the present invention.<br>
Effective signal peptide coding regions for bacterial host cells are the signal peptide coding regions obtained from the genes for Bacillus NCIB 11837 maltogenic amylase. Bacillus stearothermophilus alpha-amylase. Bacillus licheniformis subtilisin. Bacillus licheniformis beta-lactamase, Bacillus stearothermophilus neutral proteases (nprT, nprS, nprM), and Bacillus subtilis prsA. Further signal peptides are described by Simonen and<br>
<br>
Palva, 1993. Microbiological Reviews 57: 109-137.<br>
Effective signal peptide coding regions for filamentous fungal host cells are the signal peptide coding regions obtained from the genes for Aspergillus oryzae TAKA amylase, Aspergillus niger neutral amylase, Aspergillus niger glucoamylase, Rhizomucor miehei aspartic proteinase, Humicola insolens cellulase, and Humicola lanuginosa lipase.<br>
Useful signal peptides for yeast host cells are obtained from the genes for Saccharomyces cerevisiae alpha-factor and Saccharomyces cerevisiae invertase. Other useful signal peptide coding regions are described by Romanos et al., 1992, supra, and by Xiong et al in Journal of Applied Microbiology 2005, 98, 418-428.<br>
In a preferred aspect, the signal peptide coding region is nucleotides 1 to 99 of SEQ ID NO; 9, which encode amino acids 1 to 33 of SEQ ID NO: 10. In another preferred aspect, the signal peptide coding region is nucleotids 1 to 81 of SEQ ID NO: 11, which encode amino acids 1 to 27 of SEQ ID NO: 12.<br>
The control sequence may also be a propeptide coding region that codes for an amino acid sequence positioned at the amino terminus of a polypeptide. The resultant polypeptide is known as a propolypeptide or propolypeptide (or a zymogen in some cases). A propolypeptide is generally inactive and can be converted to a mature active polypeptide by catalytic or autocatalytic cleavage of the propeptide from the propolypeptide. The propeptide coding region may be obtained from the genes for Bacillus subtilis alkaline protease (aprE), Bacillus subtilis neutral protease (nprT), Saccharomyces cerevisiae alpha-factor, Rhizomucor miehei aspartic proteinase, and Myceliophthora thermophila laccase (WO 95/33836).<br>
Where both signal peptide and propeptide regions are present at the amino terminus of a polypeptide, the propeptide region is positioned next to the amino terminus of a polypeptide and the signal peptide region is positioned next to the amino terminus of the propeptide region.<br>
it may also be desirable to add regulatory sequences which allow the regulation of the expression of the polypeptide relative to the growth of the host cell. Examples of regulatory systems are those which cause the expression of the gene to be turned on or off in response to a chemical or physical stimulus, including the presence of a regulatory compound. Regulatory systems in prokaryotic systems include the lac, tec, and trp operator systems. In yeast, the ADH2 system or GAL1 system may be used. In filamentous fungi, the TAKA alpha-amylase promoter, Aspergillus niger glucoamylase promoter, and Aspergillus oryzae glucoamylase promoter may be used as regulatory sequences. Other examples of regulatory sequences are those which allow for gene amplification.<br>
Expression Vectors<br>
The present invention also relates to recombinant expression vectors comprising a<br>
<br>
polynucleotide of the present invention, a promoter, and transcriptional and translational stop signals. The various nucleic acids and control sequences described above may be joined together to produce a recombinant expression vector which may include one or more convenient restriction sites to allow for insertion or substitution of the nucleotide sequence encoding the polypeptide at such sites. Alternatively, a nucleotide sequence of the present invention may be expressed by inserl:ing the nucleotide sequence or a nucleic acid construct comprising the sequence into an appropriate vector for expression. In creating the expression vector, the coding sequence is located in the vector so that the coding sequence is operably linked with the appropriate control sequences for expression.<br>
The recombinant expression vector may be any vector (e.g., a plasmid or virus) which can be conveniently subjected to recombinant DNA procedures and can bring about expression of the nucleotide sequence. The choice of the vector will typically depend on the compatibility of the vector with the host cell into which the vector is to be introduced. The vectors may be linear or closed circular plasmids.<br>
The vector may be an autonomously replicating vector, i.e., a vector which exists as an extrachromosomal entity, the replication of which is independent of chromosomal replication, e.g., a plasmid, an extrachromosomal element, a minichromosome, or an artificial chromosome. The vector may contain any means for assuring self-replication. Alternatively, the vector may be one which, when introduced into the host cell, is integrated into the genome and replicated together with the chromosome(s) into which it has been integrated. Furthermore, a single vector or plasmid or two or more vectors or plasmids which together contain the total DNA to be introduced into the genome of the host cell, or a transposon may be used.<br>
The vectors of the present invention preferably contain one or more selectable markers which permit easy selection of transformed cells. A selectable marker is a gene the product of which provides for biocide or viral resistance, resistance to heavy metals, prototrophy to auxotrophs, and the like.<br>
A conditionally essential gene may function as a non-antibiotic selectable marker. Non-limiting examples of bacterial conditionally essential non-antibiotic selectable markers are the dal genes from Bacillus subtilis. Bacillus licheniformis, or other Bacilli, that are only essential when the bacterium is cultivated in the absence of D-alanine. Also the genes encoding enzymes involved in the turnover of UDP-galactose can function as conditionally essential markers in a cell when the cell is grown in the presence of galactose or grown in a medium which gives rise to the presence of galactose. Non-limiting examples of such genes are those from B. subtilis or B. licheniformis encoding UTP-dependent phosphorylase (EC 2.7.7.10), UDP-glucose-dependent uridylyltransferase (EC 2.7.7.12), or UDP-galactose epimerase (EC 5.1.3.2). Also a xylose isomerase gene such as xylA, of Bacilli can be used as<br>
 <br>
selectable markers in cells grown in minimal medium with xylose as sole carbon source. The genes necessary for utilizing gluconate, gntK, and gntP can also be used as selectable markers in cells grown in minima! medium with gluconate as sole carbon source. Other examples of conditionally essential genes are known in the art. Antibiotic selectable markers confer antibiotic resistance to such antibiotics as ampicillin, kanamycin, chloramphenicol, erythromycin, tetracycline, neomycin, hygromycin or methotrexate.<br>
Suitable markers for yeast host cells are ADE2, HiS3, LEU2, LYS2, MET3, TRP1, and URA3. Selectable markers for use in a filamentous fungal host cell include, but are not limited to, amdS (acetamidase), argB (ornithine carbamoyltransferase), bar (phosphinothricin acetyltransferase), hph (hygromycin phosphotransferase), niaD (nitrate reductase), pyrG (orotidine-5'-phosphate decarboxylase), sC (sulfate adenyltransferase), and trpC (anthranilate synthase), as well as equivalents thereof. Preferred for use in an Aspergillus cell are the amdS and pyrG genes of Aspergillus nidulans or Aspergillus oryzae and the bar gene of Streptomyces hygroscopicus.<br>
The vectors of the present invention preferably contain an element(s) that permits integration of the vector into the host cell's genome or autonomous replication of the vector in the cell independent of the genome.<br>
For integration into the host cell genome, the vector may rely on the polynucleotide's sequence encoding the polypeptide or any other element of the vector for integration into the genome by homologous or nonhomologous recombination. Alternatively, the vector may contain additional nucleotide sequences for directing integration by homologous recombination into the genome of the host cell at a precise location(s) in the chromosome(s). To increase the likelihood of integration at a precise location, the integrational elements should preferably contain a sufficient number of nucleic acids, such as 100 to 10,000 base pairs, preferably 400 to 10,000 base pairs, and most preferably 800 to 10,000 base pairs, which have a high degree of identity with the corresponding target sequence to enhance the probability of homologous recombination. The integrational elements may be any sequence that is homologous with the target sequence in the genome of the host cell. Furthermore, the integrational elements may be non-encoding or encoding nucleotide sequences. On the other hand, the vector may be integrated into the genome of the host cell by non-homologous recombination.<br>
For autonomous replication, the vector may further comprise an origin of replication enabling the vector to replicate autonomously in the host cell in question. The origin of replication may be any plasmid replicator mediating autonomous replication which functions in a cell. The term "origin of replication" or "plasmid replicator" is defined herein as a nucleotide sequence that enables a plasmid or vector to replicate in vivo.<br>
Examples of bacterial origins of replication are the origins of replication of plasmids<br>
 <br>
pBR322, pUC19, pACYC177, and pACYC184 permitting replication in E. coli, and pUBHO, pE194, pTA1060, and pAMU1 permitting replication in Bacillus.<br>
Examples of origins of replication for use in a yeast host cell are the 2 micron origin of replication, ARS1, ARS4, the combination of ARS1 and CEN3, and the combination of ARS4 and CEN6.<br>
Examples of origins of replication useful in a filamentous fungal cell are AMA1 and ANSI (Gems et al., 1991, Gene 98:61-67; Cullen et al., 1987, Nucleic Acids Research 15: 9163-9175; WO 00/24883). Isolation of the AMA1 gene and construction of plasmids or vectors comprising the gene can be accomplished according to the methods disclosed in WO 00/24883.<br>
More than one copy of a polynucleotide of the present invention may be inserted into the host cell to increase production of the gene product. An increase in the copy number of the polynucleotide can be obtained by integrating at least one additional copy of the sequence into the host cell genome or by including an amplifiable selectable marker gene with the polynucleotide where cells containing amplified copies of the selectable marker gene, and thereby additional copies of the polynucleotide, can be selected for by cultivating the cells in the presence of the appropriate selectable agent.<br>
The procedures used to ligate the elements described above to construct the recombinant expression vectors of the present invention are well known to one skilled in the art (see, e.g., Sambrook et al., 1989, supra).<br>
Host Cells<br>
The present invention also relates to recombinant host cells, comprising a polynucleotide of the present invention, which are advantageously used in the recombinant production of the polypeptides. A vector comprising a polynucleotide of the present invention is introduced into a host cell so that the vector is maintained as a chromosomal integrant or as a self-replicating extra-chromosomal vector as described earlier. The term "host cell" encompasses any progeny of a parent cell that is not identical to the parent cell due to mutations that occur during replication. The choice of a host cell will to a large extent depend upon the gene encoding the polypeptide and its source.<br>
The host cell may be a unicellular microorganism, e.g., a prokaryote, or a non-unicellular microorganism, e.g., a eukaryote.<br>
Useful unicellular microorganisms are bacterial cells such as gram positive bacteria including, but not limited to, a Bacillus cell, e.g.. Bacillus alkalophilus. Bacillus amyloliquefaciens. Bacillus brevis. Bacillus circulans. Bacillus clausii, Bacillus coagulans. Bacillus lautus, Bacillus lentus, Bacillus licheniformis. Bacillus megaterium, Bacillus stearothermophilus. Bacillus subtilis, and Bacillus thuringiensis; or a Streptomyces cell, e.g.,<br>
<br>
Streptomyces lividans and Streptomyces murinus, or gram negative bacteria such as E. coli and Pseudomonas sp. In a preferred aspect, the bacterial host cell is a Bacillus lentus, Bacillus licheniformis. Bacillus stearothermophilus, or Bacillus subtilis cell. In another preferred aspect, the Bacillus cell is an alkalophilic Bacillus.<br>
The introduction of a vector into a bacterial host cell may, for instance, be effected by protoplast transformation (see, e.g., Chang and Cohen, 1979, Molecular General Genetics 168: 111-115), using competent cells (see, e.g.. Young and Spizizin, 1961, Journal of Bacteriology 81: 823-829, or Dubnau and Davidoff-Abelson, 1971, Journal of Molecular Biology 56: 209-221), electroporation (see, e.g., Shigekawa and Dower, 1988, Biotechniques 6: 742-751), or conjugation (see, eg., Koehler and Thome, 1987, Journal of Bacteriology 169:5771-5278).<br>
The host cell may also be a eukaryote, such as a mammalian, insect, plant, or fungal cell.<br>
In a preferred aspect, the host cell is a fungal cell. "Fungi" as used herein includes the phyla Ascomycota, Basidiomycota, Chytridiomycota, and Zygomycota (as defined by Hawksworth et al.. In, Ainsworth and Bisby's Dictionary of The Fungi, 8th edition, 1995, CAB International, University Press, Cambridge, UK) as well as the Oomycota (as cited in Hawksworth et al., 1995, supra, page 171) and all mitosporic fungi (Hawksworth et al., 1995, supra).<br>
In a more preferred aspect, the fungal host cell is a yeast cell. "Yeast" as used herein includes ascosporogenous yeast (Endomycetales), basidiosporogenous yeast, and yeast belonging to the Fungi Imperfecti (Blastomycetes). Since the classification of yeast may change in the future, for the purposes of this invention, yeast shall be defined as described in Biology and Activities of Yeast (Skinner, F.A., Passmore, S.M., and Davenport, R.R., eds, Soc. App. Bacteriol. Symposium Series No. 9, 1980).<br>
In an even more preferred aspect, the yeast host cell is a Candida, Hansenula, Kluyveromyces, Pichia, Saccharomyces, Schizosaccharomyces, or Yarrowia cell.<br>
In a most preferred aspect, the yeast host cell is a Pichia pastoris, Pichia methanolica, Saccharomyces carlsbergensis, Saccharomyces cerevisiae, Saccharomyces diastaticus, Saccharomyces dougiasii, Saccharomyces kluyveri, Saccharomyces norbensis or Saccharomyces oviformis cell. In another most preferred aspect, the yeast host cell is a Kluyveromyces lactis cell. In another most preferred aspect, the yeast host cell is a Yarrowia lipolytica cell.<br>
In another more preferred aspect, the fungal host cell is a filamentous fungal cell. "Filamentous fungi" include all filamentous forms of the subdivision Eumycota and Oomycota (as defined by Hawksworth et al., 1995, supra). The filamentous fungi are generally characterized by a mycelial wall composed of chitin, cellulose, glucan, chitosan, mannan, and<br>
28<br>
 <br>
other complex polysaccharides. Vegetative growth is by hyphal elongation and carbon catabolism is obligately aerobic. In contrast, vegetative growth by yeasts such as Saccharomyces cerevisiae is by budding of a unicellular thallus and carbon catabolism may be fermentative.<br>
In an even more preferred aspect, the filamentous fungal host cell is an Acremonium, Aspergillus, Aureobasidium, Bjerkandera, Ceriporiopsis, Coprinus, Coriolus, Cryptococcus, Filobasidium, Fusarium, Humicola, Magnaporthe, Mucor, Myceliophthora, Neocallimastix, Neurospora, Paecilomyces, Penicillium, Phanerochaete, Phlebia, Piromyces, Pleurotus, Schizophyllum, Talaromyces, Thermoascus, Thielavia, Tolypocladium, Trametes, or Trichoderma cell.<br>
In a most preferred aspect, the filamentous fungal host cell is an Aspergillus awamori, Aspergillus fumigatus, Aspergillus foetidus, Aspergillus japonicus, Aspergillus nidulans, Aspergillus niger or Aspergillus oryzae cell. In another most preferred aspect, the filamentous fungal host cell is a Fusarium bactridioides, Fusarium cerealis, Fusarium crookwellense, Fusarium culmorum, Fusarium graminearum, Fusarium graminum, Fusarium heterosporum, Fusarium negundi, Fusarium oxysporum, Fusarium reticulatum, Fusarium roseum, Fusarium sambucinum, Fusarium sarcochroum, Fusarium sporotrichioides, Fusarium sulphureum, Fusarium torulosum, Fusarium trichothecioides, or Fusarium venenatum cell. In another most preferred aspect, the filamentous fungal host cell is a Bjerkandera adusta, Ceriporiopsis aneirina, Ceriporiopsis aneirina, Ceriporiopsis caregiea, Ceriporiopsis gilvescens, Ceriporiopsis pannocinta, Ceriporiopsis rivulosa, Ceriporiopsis subrufa, or Ceriporiopsis subvermispora, Coprinus cinereus, Coriolus hirsutus, Humicola insolens, Humicola lanuginosa, Mucor miehei, Myceliophthora thermophila, Neurospora crassa, Penicillium purpurogenum, Phanerochaete chrysosporium, Phlebia radiata, Pleurotus eryngii, Thielavia terrestris, Trametes villosa, Trametes versicolor, Trichoderma harzianum, Trichoderma koningii, Trichoderma longibrachiatum, Trichoderma reesei, or Trichoderma viride strain cell.<br>
Fungal cells may be transformed by a process involving protoplast formation, transformation of the protoplasts, and regeneration of the cell wall in a manner known per se. Suitable procedures for transformation of Aspergillus and Trichoderma host cells are described in EP 238 023 and Yelton et al., 1984, Proceedings of the National Academy of Sciences USA 81: 1470-1474. Suitable methods for transforming Fusarium species are described by Malardier et al., 1989, Gene 78: 147-156, and WO 96/00787. Yeast may be transformed using the procedures described by Becker and Guarente, In Abelson, J.N. and Simon, M.I., editors, Guide to Yeast Genetics and Molecular Biology, Methods in Enzymology, Volume 194, pp 182-187, Academic Press, Inc., New York; Ito et al., 1983, Journal of Bacteriology 153: 163; and Hinnen et al., 1978, Proceedings of the National<br>
29<br>
 <br>
Academy of Sciences USA 75: 1920.<br>
Methods of Production<br>
The present invention also relates to methods for producing a polypeptide of the present invention, comprising (a) cultivating a cell, which in its wild-type form is capable of producing the polypeptide, under conditions conducive for production of fhe polypeptide; and (b) recovering the polypeptide. Preferably, the cell is of the genus Hafnia, and more preferably Hafnia alvei.<br>
The present invention also relates to methods for producing a polypeptide of the present invention, comprising (a) cultivating a host cell under conditions conducive for production of the polypeptide; and (b) recovering the polypeptide.<br>
The present invention also relates to methods for producing a polypeptide of the present invention, comprising (a) cultivating a host cell under conditions conducive for production of the polypeptide, wherein the host cell comprises a mutant nucleotide sequence having at least one mutation in the mature polypeptide coding region of any one of SEQ ID NO: 9, wherein the mutant nucleotide sequence encodes a polypeptide which consists of amino acids 1 to 413 of SEQ ID NO: 10, and (b) recovering the polypeptide.<br>
In the production methods of the present invention, the cells are cultivated in a nutrient medium suitable for production of the polypeptide using methods well known in the art. For example, the cell may be cultivated by shake flask cultivation, and small-scale or large-scale fermentation (including continuous, batch, fed-batch, or solid state fermentations) in laboratory or industrial fermentors performed in a suitable medium and under conditions allowing the polypeptide to be expressed and/or isolated. The cultivation takes place in a suitable nutrient medium comprising carbon and nitrogen sources and inorganic salts, using procedures known in the art. Suitable media are available from commercial suppliers or may be prepared according to published compositions (e.g., in catalogues of the American Type Culture Collection). If the polypeptide is secreted into the nutrient medium, the polypeptide can be recovered directly from the medium. If the polypeptide is not secreted, it can be recovered from cell lysates.<br>
The polypeptides may be detected using methods known in the art that are specific for the polypeptides. These detection methods may include use of specific antibodies, formation of a polypeptide product, or disappearance of an polypeptide substrate. For example, an polypeptide assay may be used to determine the activity of the polypeptide as described herein.<br>
The resulting polypeptide may be recovered using methods known in the art. For example, the polypeptide may be recovered from the nutrient medium by conventional procedures including, but not limited to, centrifugation, filtration, extraction, spray-drying,<br>
30<br>
 <br>
evaporation, or precipitation.<br>
The polypeptides of the present invention may be purified by a variety of procedures known in the art including, but not limited to, chromatography (e.g., ion exchange, affinity, hydrophobic, chromatofocusing, and size exclusion), electrophoretic procedures (e.g., preparative isoelectric focusing), differential solubility (e.g., ammonium sulfate precipitation), SDS-PAGE, or extraction (see, e.g.. Protein Purification, J.-C. Janson and Lars Ryden, editors, VCH Publishers, New York, 1989).<br>
Transgenic Plants<br>
The present invention also relates to a transgenic plant, plant part, or plant cell which has been transformed with a nucleotide sequence encoding a polypeptide having phytase activity of the present invention so as to express and produce the polypeptide in recoverable quantities. The polypeptide may be recovered from the plant or plant part. Alternatively, the plant or plant part containing the recombinant polypeptide may be used as such for improving the quality of a food or feed, e.g., improving nutritional value, palatability, and rheological properties, or to destroy an antinutritive factor.<br>
In a particular embodiment, the polypeptide is targeted to the endosperm storage vacuoles in seeds. This can be obtained by synthesizing it as a precursor with a suitable signal peptide, see Horvath et al in PNAS, Feb. 15, 2000, vol. 97, no. 4, p. 1914-1919.<br>
The transgenic plant can be dicotyledonous (a dicot) or monocotyledonous (a monocot) or engineered variants thereof. Examples of monocot plants are grasses, such as meadow grass (blue grass, Poa), forage grass such as Festuca, Lolium, temperate grass, such as Agrostis, and cereals, e.g., wheat, oats, rye, barley, rice, sorghum, triticale (stabilized hybrid of wheat (Triticum) and rye (Secale), and maize (corn). Examples of dicot plants are tobacco, legumes, such as sunflower (Helianthus), cotton (Gossypium), lupins, potato, sugar beet, pea, bean and soybean, and cruciferous plants (family Brassicaceae), such as cauliflower, rape seed, and the closely related model organism Arabidopsis thaliana. Low-phytate plants as described e.g. in US patent no. 5,689,054 and US patent no. 6,111,168 are examples of engineered plants.<br>
Examples of plant parts are stem, callus, leaves, root, fruits, seeds, and tubers, as well as the individual tissues comprising these parts, e.g. epidermis, mesophyll, parenchyma, vascular tissues, meristems. Also specific plant cell compartments, such as chloroplast, apoplast, mitochondria, vacuole, peroxisomes, and cytoplasm are considered to be a plant part. Furthermore, any plant cell, whatever the tissue origin, is considered to be a plant part. Likewise, plant parts such as specific tissues and cells isolated to facilitate the utilisation of the invention are also considered plant parts, e.g. embryos, endosperms, aleurone and seed coats.<br>
31<br>
 <br>
Also included within the scope of the present invention are the progeny of such plants, plant parts and plant cells.<br>
The transgenic plant or plant cell expressing a polypeptide of the present invention may be constructed in accordance with methods known in the art. Briefly, the plant or plant cell is constructed by incorporating one or more expression constructs encoding a polypeptide of the present invention into the plant host genome and propagating the resulting modified plant or plant cell into a transgenic plant or plant cell.<br>
Conveniently, the expression construct is a nucleic acid construct which comprises a nucleic acid sequence encoding a polypeptide of the present invention operably linked with appropriate regulatory sequences required for expression of the nucleic acid sequence in the plant or plant part of choice. Furthermore, the expression construct may comprise a selectable marker useful for identifying host cells into which the expression construct has been integrated and DNA sequences necessary for introduction of the construct into the plant in question (the latter depends on the DNA introduction method to be used).<br>
The choice of regulatory sequences, such as promoter and terminator sequences and optionally signal or transit sequences are determined, for example, on the basis of when, where, and how the polypeptide is desired to be expressed. For instance, the expression of the gene encoding a polypeptide of the present invention may be constitutive or inducible, or may be developmental, stage or tissue specific, and the gene product may be targeted to a specific cell compartment, tissue or plant part such as seeds or leaves. Regulatory sequences are, for example, described by Tague et al., 1988, Plant Physiology 86: 506.<br>
For constitutive expression, the following promoters may be used: The 35S-CaMV promoter (Franck et al., 1980, Cell 21: 285-294), the maize ubiquitin 1 (Christensen AH, Sharrock RA and Quail 1992. Maize polyubiquitin genes: structure, thermal perturbation of expression and transcript splicing, and promoter activity following transfer to protoplasts by electroporation), or the rice actin 1 promoter (Plant Mo. Biol. 18, 675-689.; Zhang W, McElroy D. and Wu R 1991, Analysis of rice Acti 5' region activity in transgenic rice plants. Plant Cell 3, 1155-1165). Organ-specific promoters may be, for example, a promoter from storage sink tissues such as seeds, potato tubers, and fruits (Edwards &amp; Coruzzi, 1990, Ann. Rev, Genet. 24: 275-303), or from metabolic sink tissues such as meristems (Ito et al., 1994, Plant Mol. Biol. 24: 863-878), a seed specific promoter such as the glutelin, prolamin, globulin, or albumin promoter from rice (Wu et al., 1998, Plant and Cell Physiology 39: 885-889), a Vicia faba promoter from the legumin 134 and the unknown seed protein gene from Vicia faba (Conrad et al., 1998, Journal of Plant Physiology 152: 708-711), a promoter from a seed oil body protein (Chen et al., 1998, Plant and Cell Physiology 39: 935-941), the storage protein napA promoter from Brassica napus, or any other seed specific promoter known in the art, e.g., as described in WO 91/14772. Furthermore, the promoter may be a leaf specific<br>
32<br>
 <br>
promoter such as the rbcs promoter from rice or tomato (Kyozuka et al., 1993, Plant Physiology 102: 991-1000, the chlorella virus adenine methyltransferase gene promoter (Mitra and Higgins, 1994, Plant Molecular Biology 26: 85-93), or the aldP gene promoter from rice (Kagaya et al., 1995, Molecular and General Genetics 248: 668-674), or a wound inducible promoter such as the potato pin2 promoter (Xu et al., 1993, Plant Molecular Biology 22: 573-588). Likewise, the promoter may be inducible by abiotic treatments such as temperature, drought or alterations in salinity or inducible by exogenously applied substances that activate the promoter, e.g. ethanol, oestrogens, plant hormones like ethylene, abscisic acid, gibberellic acid, and/or heavy metals.<br>
A promoter enhancer element may also be used to achieve higher expression of the polypeptide in the plant. For instance, the promoter enhancer element may be an intron which is placed between the promoter and the nucleotide sequence encoding a polypeptide of the present invention. For instance, Xu et al., 1993, supra disclose the use of the first intron of the rice actin 1 gene to enhance expression.<br>
Still further, the codon usage may be optimized for the plant species in question to improve expression (see Horvath et al referred to above).<br>
The selectable marker gene and any other parts of the expression construct may be chosen from those available in the art.<br>
The nucleic acid construct is incorporated into the plant genome according to conventional techniques known in the art, including Agrobacterium-mediated transformation, virus-mediated transformation, microinjection, particle bombardment, biolistic transformation, and electroporation (Gasser et al., 1990, Science 244: 1293; Potrykus, 1990, Bio/Technology 8: 535; Shimamoto et al., 1989, Nature 338: 274).<br>
Presently, Agrobacterium tumefaciens-mediated gene transfer is the method of choice for generating transgenic dicots (for a review, see Hooykas and Schilperoort, 1992, Plant Molecular Biology 19: 15-38), and it can also be used for transforming monocots, although other transformation methods are more often used for these plants. Presently, the method of choice for generating transgenic monocots, supplementing the Agrobacterium approach, is particle bombardment (microscopic gold or tungsten particles coated with the transforming DNA) of embryonic calli or developing embryos (Christou, 1992, Plant Journal 2: 275-281; Shimamoto, 1994, Current Opinion Biotechnology 5: 158-162; Vasii et al., 1992, Bio/Technology 10: 667-674). An alternative method for transformation of monocots is based on protoplast transformation as described by Omirulleh et al., 1993, Plant Molecular Biology 21:415-428.<br>
Following transformation, the transformants having incorporated therein the expression construct are selected and regenerated into whole plants according to methods well-known in the art. Often the transformation procedure is designed for the selective<br>
33<br>
 <br>
elimination of selection genes either during regeneration or in the following generations by using e.g. co-transformation with two separate T-DNA constructs or site specific excision of the selection gene by a specific recombinase.<br>
The present invention also relates to methods for producing a polypeptide of the present invention comprising (a) cultivating a transgenic plant or a plant cell comprising a nucleic acid sequence encoding a polypeptide having phytase activity of the present invention under conditions conducive for production of the polypeptide; and (b) recovering the polypeptide.<br>
Transgenic Animals<br>
The present invention also relates to a transgenic, non-human animal and products or elements thereof, examples of which are body fluids such as milk and blood, organs, flesh, and animal cells. Techniques for expressing proteins, e.g. in mammalian cells, are known in the art, see e.g. the handbook Protein Expression: A Practical Approach, Higgins and Hames (eds), Oxford University Press (1999), and the three other handbooks in this series relating to Gene Transcription, RNA processing, and Post-translational Processing. Generally speaking, to prepare a transgenic animal, selected cells of a selected animal are transformed with a nucleic acid sequence encoding a polypeptide having phytase activity of the present invention so as to express and produce the polypeptide. The polypeptide may be recovered from the animal, e.g. from the milk of female animals, or the polypeptide may be expressed to the benefit of the animal itself, e.g. to assist the animal's digestion. Examples of animals are mentioned below in the section headed Animal Feed.<br>
To produce a transgenic animal with a view to recovering the polypeptide from the milk of the animal, a gene encoding the polypeptide may be inserted into the fertilized eggs of an animal in question, e.g. by use of a transgene expression vector which comprises a suitable milk protein promoter, and the gene encoding the polypeptide. The transgene expression vector is is microinjected into fertilized eggs, and preferably permanently integrated into the chromosome. Once the egg begins to grow and divide, the potential embryo is implanted into a surrogate mother, and animals carrying the transgene are identified. The resulting animal can then be multiplied by conventional breeding. The polypeptide may be purified from the animal's milk, see e.g. Meade, H.M. et al (1999): Expression of recombinant proteins in the milk of transgenic animals. Gene expression systems: Using nature for the art of expression. J. M. Fernandez and J. P. Hoeffier (eds.). Academic Press.<br>
In the alternative, in order to produce a transgenic non-human animal that carries in the genome of its somatic and/or germ cells a nucleic acid sequence including a heterologous transgene construct including a transgene encoding the polypeptide, the transgene may be<br>
34<br>
 <br>
operably linked to a first regulatory sequence for salivary gland specific expression of the polypeptide, as disclosed in WO 00/0G4247.<br>
Compositions and Uses<br>
In still further aspects, the present invention relates to compositions comprising a polypeptide of the present invention, as well as methods of using these.<br>
The polypeptide compositions may be prepared in accordance with methods known in the art and may be in the form of a liquid or a dry composition. For instance, the polypeptide composition may be in the form of granulates or microgranulates. The polypeptide to be included in the composition may be stabilized in accordance with methods known in the art.<br>
The phytase of the invention can be used for degradation, in any industrial context, of, for example, phytate, phytic acid, and/or the mono-, di-, tri-, tetra- and/or penta-phosphates of myo-inositol. It is well known that the phosphate moieties of these compounds chelates divalent and trivalent cations such as metal ions, i.a. the nutritionally essential ions of calcium, iron, zinc and magnesium as well as the trace minerals manganese, copper and molybdenum. Besides, the phytic acid also to a certain extent binds proteins by electrostatic interaction.<br>
Accordingly, preferred uses of the polypeptides of the invention are in animal feed preparations (including human food) or in additives for such preparations.<br>
In a particular embodiment, the polypeptide of the invention can be used for improving the nutritional value of an animal feed. Non-limiting examples of improving the nutritional value of animal feed (including human food), are: Improving feed digestibility; promoting growth of the animal; improving feed utilization; improving bio-availability of proteins; increasing the level of digestible phosphate; improving the release and/or degradation of phytate; improving bio-availability of trace minerals; improving bio-availability of macro minerals; eliminating the need for adding supplemental phosphate, trace minerals, and/or macro minerals; and/or improving egg shell quality. The nutritional value of the feed is therefore increased, and the growth rate and/or weight gain and/or feed conversion (i.e. the weight of ingested feed relative to weight gain) of the animal may be improved.<br>
Furthermore, the polypeptide of the invention can be used for reducing phytate level of manure.<br>
Animals, Animal Feed, and Animal Feed Additives<br>
The term animal includes all animals, including human beings. Examples of animals are non-ruminants, and ruminants. Ruminant animals include, for example, animals such as sheep, goats, horses, and cattle, e.g. beef cattle, cows, and young calves. In a particular<br>
35<br>
 <br>
embodiment, the animal is a non-ruminant animal. Non-ruminant animals include mono-gastric animals, e.g. pigs or swine (including, but not limited to, piglets, growing pigs, and sows); poultry such as turkeys, ducks and chicken (including but not limited to broiler chicks, layers); young calves; and fish (including but not limited to salmon, trout, tilapia, catfish and carps; and crustaceans (including but not limited to shrimps and prawns).<br>
The term feed or feed composition means any compound, preparation, mixture, or composition suitable for, or intended for intake by an animal.<br>
In the use according to the invention the polypeptide can be fed to the animal before, after, or simultaneously with the diet. The latter is preferred.<br>
In a particular embodiment, the polypeptide, in the form in which it is added to the feed, or when being included in a feed additive, is substantially pure. In a particular embodiment it is well-defined. The term "well-defined" means that the phytase preparation is at least 50% pure as determined by Size-exclusion chromatography (see Example 12 of WO 01/58275). In other particular embodiments the phytase preparation is at least 60, 70, 80, 85, 88, 90, 92, 94, or at least 95% pure as determined by this method.<br>
A substantially pure, and/or well-defined polypeptide preparation is advantageous. For instance, it is much easier to dose correctly to the feed a polypeptide that is essentially free from interfering or contaminating other polypeptides. The term dose correctly refers in particular to the objective of obtaining consistent and constant results, and the capability of optimising dosage based upon the desired effect.<br>
For the use in animal feed, however, the phytase polypeptide of the invention need not be that pure; it may e.g. include other polypeptides, in which case it could be termed a phytase preparation.<br>
The phytase preparation can be (a) added directly to the feed (or used directly in a treatment process of proteins), or (b) it can be used in the production of one or more intermediate compositions such as feed additives or premixes that is subsequently added to the feed (or used in a treatment process). The degree of purity described above refers to the purity of the original polypeptide preparation, whether used according to (a) or (b) above.<br>
Polypeptide preparations with purities of this order of magnitude are in particular obtainable using recombinant methods of production, whereas they are not so easily obtained and also subject to a much higher batch-to-batch variation when the polypeptide is produced by traditional fermentation methods.<br>
Such polypeptide preparation may of course be mixed with other polypeptides.<br>
The polypeptide can be added to the feed in any form, be it as a relatively pure polypeptide, or in admixture with other components intended for addition to animal feed, i.e. in the form of animal feed additives, such as the so-called pre-mixes for animal feed.<br>
In a further aspect the present invention relates to compositions for use in animal<br>
36<br>
 <br>
feed, such as animal feed, and animal feed additives, e.g. premixes.<br>
Apart from the polypeptide of the invention, the animal feed additives of the invention contain at least one fat-soluble vitamin, and/or at least one water soluble vitamin, and/or at least one trace mineral. The feed additive may also contain at least one macro mineral.<br>
Further, optional, feed-additive ingredients are colouring agents, e.g. carotenoids such as beta-carotene, astaxanthin, and lutein; aroma compounds; stabilisers; antimicrobial peptides; polyunsaturated fatty acids; reactive oxygen generating species; and/or at least one other polypeptide selected from amongst phytase (EC 3.1.3,8 or 3.1.3.26); xylanase (EC 3.2.1.8); galactanase (EC 3.2.1.89); alpha-galactosidase (EC 3.2.1.22); protease (EC 3.4.-.-), phospholipase A1 (EC 3.1.1.32); phospholipase A2 (EC 3,1.1.4); lysophospholipase (EC 3.1.1.5); phospholipase C (3.1.4.3); phospholipase D (EC 3.1.4.4); amylase such as, for example, alpha-amylase (EC 3.2.1.1); and/or beta-glucanase (EC 3.2,1.4 or EC 3.2.1.6).<br>
In a particular embodiment these other polypeptides are well-defined (as defined above for phytase preparations).<br>
In a particularly preferred embodiment, the phytase of the invention having a relatively low pH-optimum is combined with at least one phytase having a higher pH-optimum. Preferred examples of phytases of higher pH-optimum are Bacillus phytases, such as the phytases from Bacillus licheniformis and Bacillus subtilis, as well as derivatives, variants, or fragments thereof having phytase activity.<br>
The phytase of the invention may also be combined with other phytases, for example ascomycete phytases such as Aspergillus phytases, for example derived from Aspergillus ficuum, Aspergillus niger, or Aspergillus awamori; or basidiomycete phytases, for example derived from Peniophora lycii, Agrocybe pediades, Trametes pubescens, or Paxillus involutus; or derivatives, fragments or variants thereof which have phytase activity.<br>
Thus, in preferred embodiments of the use in animal feed of the invention, and in preferred embodiments of the animal feed additive and the animal feed of the invention, the phytase of the invention is combined with such phytases.<br>
The above-mentioned ascomycete and basidiomycete phytases, in particular the RONOZYME P phytase derived from Peniophora lycii as well as derivatives, variants, and fragments thereof, may also be combined with Bacillus phytases, in particular the B. licheniformis phytase as well as with a derivative, fragment or variant thereof, in particular for animal feed purposes.<br>
Examples of antimicrobial peptides (AMP's) are CAP18, Leucocin A, Tritrpticin, Protegrin-1, Thanatin, Defensin, Lactoferrin, Lactoferricin, and Ovispirin such as Novispirin (Robert Lehrer, 2000), Plectasins, and Statins, including the compounds and polypeptides disclosed in WO 03/044049 and V\/0 03/048148, as well as variants or fragments of the above that retain antimicrobial activity.<br>
37<br>
 <br>
Examples of antifungal polypeptides (AFP's) are the Aspergillus giganteus, and Aspergillus niger peptides, as well as variants and fragments thereof which retain antifungal activity, as disclosed in WO 94/01459 and WO 02/090384.<br>
Examples of polyunsaturated fatty acids are CI 8, C20 and C22 polyunsaturated fatty acids, such as arachidonic acid, docosohexaenoic acid, eicosapentaenoic acid and gamma-linoleic acid.<br>
Examples of reactive oxygen generating species are chemicals such as perborate, persulphate, or percarbonate; and polypeptides such as an oxidase, an oxygenase or a syntethase.<br>
Usalty fat- and water-soluble vitamins, as well as trace minerals form part of a so-called premix intended for addition to the feed, whereas macro minerals are usually separately added to the feed. Either of these composition types, when enriched with a polypeptide of the invention, is an animal feed additive of the invention.<br>
In a particular embodiment, the animal feed additive of the invention is intended for being included (or prescribed as having to be included) in animal diets or feed at levels of 0.01 to 10.0%; more particularly 0.05 to 5.0%; or 0.2 to 1.0% (% meaning g additive per 100 g feed). This is so in particular for premixes.<br>
The following are non-exclusive lists of examples of these components:<br>
Examples of fat-soluble vitamins are vitamin A, vitamin D3, vitamin E, and vitamin K, e.g. vitamin K3.<br>
Examples of water-soluble vitamins are vitamin B12, biotin and choline, vitamin B1, vitamin 82, vitamin 86, niacin, folic acid and panthothenate, e.g. Ca-D-panthothenate.<br>
Examples of trace minerals are manganese, zinc, iron, copper, iodine, selenium, and cobalt.<br>
Examples of macro minerals are calcium, phosphorus and sodium.<br>
The nutritional requirements of these components (exemplified with poultry and piglets/pigs) are listed in Table A of WO 01/58275. Nutritional requirement means that these components should be provided in the diet in the concentrations indicated.<br>
In the alternative, the animal feed additive of the invention comprises at least one of the individual components specified in Table A of WO 01/58275. At least one means either of, one or more of, one, or two, or three, or four and so forth up to ail thirteen, or up to all fifteen individual components. More specifically, this at least one individual component is included in the additive of the invention in such an amount as to provide an in-feed-concentration within the range indicated in column four, or column five, or column six of Table A.<br>
The present invention also relates to animal feed compositions. Animal feed compositions or diets have a relatively high content of protein. Poultry and pig diets can be characterised as indicated in Table B of WO 01/58275, columns 2-3. Fish diets can be<br>
38<br>
 <br>
characterised as indicated in column 4 of this Table B. Furthermore such fish diets usually have a crude fat content of 200-310 g/kg.<br>
WO 01/58275 corresponds to US 09/779334 which is hereby incorporated by reference.<br>
An animal feed composition according to the invention has a crude protein content of 50-800 g/kg, and furthermore comprises at least one polypeptide as claimed herein.<br>
Furthermore, or in the alternative (to the crude protein content indicated above), the animal feed composition of the invention has a content of metabolisable energy of 10-30 MJ/kg; and/or a content of calcium of 0.1-200 g/kg; and/or a content of available phosphorus of 0.1-200 g/kg; and/or a content of methionine of 0.1-100 g/kg; and/or a content of methionine plus cysteine of 0.1-150 g/kg; and/or a content of lysine of 0.5-50 g/kg.<br>
In particular embodiments, the content of metabolisable energy, crude protein, calcium, phosphorus, methionine, methionine plus cysteine, and/or lysine is within any one of ranges 2, 3, 4 or 5 in Table B of WO 01/58275 (R. 2-5).<br>
Crude protein is calculated as nitrogen (N) multiplied by a factor 6.25, i.e. Crude protein (g/kg)= N (g/kg) x 6.25. The nitrogen content is determined by the Kjeldahl method (A.O.A.C, 1984, Official Methods of Analysis 14th ed., Association of Official Analytical Chemists, Washington DC).<br>
Metabolisable energy can be calculated on the basis of the NRC publication Nutrient requirements in swine, ninth revised edition 1988, subcommittee on swine nutrition, committee on animal nutrition, board of agriculture, national research council. National Academy Press, Washington, D.C., pp. 2-6, and the European Table of Energy Values for Poultry Feed-stuffs, Spelderholt centre for poultry research and extension, 7361 DA Beekbergen, The Netherlands. Grafisch bedrijf Ponsen &amp; looijen bv, Wageningen. ISBN 90-71463-12-5.<br>
The dietary content of calcium, available phosphorus and amino acids in complete animal diets is calculated on the basis of feed tables such as Veevoedertabel 1997, gegevens over chemische samenstelling, verteerbaarheid en voedenrt/aarde van voedermlddelen. Central Veevoederbureau, Runderweg 6, 8219 pk Leiystad. ISBN 90-72839-13-7.<br>
In a particular embodiment, the animal feed composition of the invention contains at least one protein. The protein may be an animal protein, such as meat and bone meal, and/or fish meal; or it may be a vegetable protein. The term vegetable proteins as used herein refers to any compound, composition, preparation or mixture that includes at least one protein derived from or originating from a vegetable, including modified proteins and protein-derivatives. In particular embodiments, the protein content of the vegetable proteins is at least 10, 20, 30, 40, 50, or 60% (w/w).<br>
Vegetable proteins may be derived from vegetable protein sources, such as legumes<br>
39<br>
 <br>
and cereals, for example materials from plants of the families Fabaceae (Leguminosae), Cruciferaceae, Chenopodiaceae, and Poaceae, such as soy bean meal, lupin meal and rapeseed meal.<br>
In a particular embodiment, the vegetable protein source is material from one or more plants of the family Fabaceae, e.g. soybean, lupine, pea, or bean.<br>
In another particular embodiment, the vegetable protein source is material from one or more plants of the family Chenopodiaceae, e.g. beet, sugar beet, spinach or quinoa.<br>
Other examples of vegetable protein sources are rapeseed, sunflower seed, cotton seed, and cabbage.<br>
Soybean is a preferred vege;table protein source.<br>
Other examples of vegetable protein sources are cereals such as barley, wheat, rye, oat, maize (corn), rice, triticaie, and sorghum.<br>
In still further particular embodiments, the animal feed composition of the invention contains 0-80% maize; and/or 0-80% sorghum; and/or 0-70% wheat; and/or 0-70% Barley; and/or 0-30% oats; and/or 0-40% soybean meal; and/or 0-25% fish meal; and/or 0-25% meat and bone meal; and/or 0-20% whey.<br>
Animal diets can e.g. be manufactured as mash feed (non pelleted) or pelleted feed. Typically, the milled feed-stuffs are mixed and sufficient amounts of essential vitamins and minerals are added according to the specifications for the species in question. Polypeptides can be added as solid or liquid polypeptide formulations. For example, a solid polypeptide formulation is typically added before or during the mixing step; and a liquid polypeptide preparation is typically added after the pelleting step. The polypeptide may also be incorporated in a feed additive or premix.<br>
The final polypeptide concentration in the diet is within the range of 0.01-200 mg polypeptide protein per kg diet, for example in the range of 0.1-10 mg/kg animal diet (typical dosage is in the range of 250 to 2000 FYT/kg animal diet).<br>
The phytase of the invention should of course be applied in an effective amount, i.e. in an amount adequate for improving solubilisation and/or improving nutritional value of feed. It is at present contemplated that the polypeptide is administered in one or more of the following amounts (dosage ranges): 0.01-200; 0.01-100; 0.5-100; 1-50; 5-100; 10-100; 0.05-50; or 0.10-10 - all these ranges being in mg phytase polypeptide protein per kg feed (ppm).<br>
For determining mg phytase polypeptide protein per kg feed, the phytase is purified from the feed composition, and the specific activity of the purified phytase is determined using a relevant assay. The phytase activity of the feed composition as such is also determined using the same assay, and on the basis of these two determinations, the dosage in mg phytase protein per kg feed is calculated.<br>
The same principles apply for determining mg phytase polypeptide protein in feed<br>
40<br>
 <br>
additives. Of course, if a sample is available of the phytase used for preparing the feed additive or the feed, the specific activity is determined from this sample (no need to purify the phytase from the feed composition or the additive).<br>
Methods For Producing Fermentation Products<br>
Yet another aspect of the present invention relates to the methods for producing a fermentation product, such as, e.g., ethanol, beer, w/ine, distillers dried grains (DDG), w/herein the fermentation is carried out in the presence of a phytase of the present invention. Examples of fermentation processes include, for example, the processes described in WO 01/62947. Fermentation is carried out using a fermenting microorganism, such as, yeast.<br>
In a particular embodiment^ the present invention provides methods for producing fermentation product, comprising (a) fermenting (using a fermenting microorganism, such as yeast) a carbohydrate containing material (e.g., starch) in the presence of a phytase of the present invention and (b) producing the fermentation product from the fermented carbohydrate containing material.<br>
In a particular embodiment, the present invention provides methods for producing ethanol, comprising fermenting (using a fermenting microorganism, such as yeast) a carbohydrate containing material (e.g., starch) in the presence of a phytase of the present invention and producing or recovering ethanol from the fermented carbohydrate containing material.<br>
In another embodiment, the present invention provides methods for producing ethanol comprising a) hydrolyzing starch, e.g., by a liquefaction and/or saccharification process, a raw starch hydrolysis process, b) fermenting the resulting starch in the presence of a phytase of the present invention, and c) producing ethanol.<br>
The phytase may be added to the fermentation process at any suitable stage and in any suitable composition, including alone or in combination with other enzymes, such as, one or more alpha-amylases, glucoamylases, proteases, and/or cellulases.<br>
In another embodiment, the present invention provides methods for producing ethanol comprising hydrolyzing biomass, and fermenting (using a fermenting microorganism, such as yeast) the resulting biomass in the presence of a phytase of the present invention.<br>
Signal Peptide<br>
The present invention also relates to nucleic acid constructs comprising a gene encoding a protein operably linked to a first nucleotide sequence consisting of nucleotides 1 to 99 of SEQ ID NO: 9, encoding a signal peptide consisting of amino acids 1 to 33 of SEQ ID NO: 10, wherein the gene is foreign to the first nucleotide sequences.<br>
The   present   invention   also   relates   to   recombinant   expression   vectors   and<br>
41<br>
 <br>
recombinant host cells comprising such nucleic acid constructs.<br>
The present invention also relates to methods for producing a protein comprising (a) cultivating such a recombinant host cell under conditions suitable for production of the protein; and (b) recovering the protein.<br>
The first nucleotide sequences may be operably linked to foreign genes individually with other control sequences or in combination with other control sequences. Such other control sequences are described supra.<br>
The protein may be native or heterologous to a host cell. The term "protein" is not meant herein to refer to a specific length of the encoded product and, therefore, encompasses peptides, oligopeptides, and proteins. The term "protein" also encompasses two or more polypeptides combined to form the encoded product. The proteins also include hybrid polypeptides which comprise a combination of partial or complete polypeptide sequences obtained from at least two different proteins wherein one or more may be heterologous or native to the host cell. Proteins further include naturally occurring allelic and engineered variations of the above mentioned proteins and hybrid proteins.<br>
Preferably, the protein is a hormone or variant thereof, polypeptide, e.g., enzyme, receptor or portion thereof, antibody or portion thereof, or reporter. In a more preferred aspect, the protein is an oxidoreductase, transferase, hydrolase, lyase, isomerase, or ligase. In an even more preferred aspect, the protein is an aminopeptidase, amylase, carbohydrase, carboxypeptidase, catalase, cellulase, chitinase, cutinase, cyclodextrin glycosyltransferase, deoxyribonuclease, esterase, alpha-galactosidase, beta-galactosidase, glucoamyiase, alpha-glucosidase, beta-glucosidase, invertase, laccase, lipase, mannosidase, mutanase, oxidase, pectinolytic polypeptide, peroxidase, phytase, polyphenoloxidase, proteolytic polypeptide, ribonuclease, transglutaminase or xylanase.<br>
The gene may be obtained from any prokaryotic, eukaryotic, or other source.<br>
Various embodiments<br>
The following are additional embodiments of the present invention. Also included herein are the corresponding aspects relating to nucleic acid sequences, nucleic acid constructs, recombinant expression vectors, recombinant host cells, methods for production of the polypeptides, transgenic plants and animals, and the various uses, methods of use and feed compositions/additives, all as claimed.<br>
An isolated polypeptide having phytase actvity and a residual activity following incubation at 37°C and in a 0.1 M Giycine/HCI buffer, pH 2.0, for 4 hours of at least 20%, as compared to the activity at time, t == 0, the activity being assayed at 37°C and pH 5.5 on 1% (w/v) Na-phytate, using a 0.25 M Na-acetate buffer pH 5.5, buffer blind subtracted;<br>
preferably with an identity to i) amino acids 1 to 413 of SEQ ID NO: 10, of at least<br>
42<br>
 <br>
75%, preferably at least 76%, 77%, 78%, 79%, 80%, 81%, 82%, 83%, 84%, 85%, 86%, 87%, 88%, 89%, 90%, 91%, 92%, 93%, 94%, 95%, 96%, 96.5%, 97%, 97.5%, 98%, 98.5%, 98.7%, 98.8%, 98.9%, 99%, 99.0%, 99.1%, 99.2%, 99.3%. 99.4%, 99.5%, 99.6%, 99.7%, 99.8%, or at least 99,9%.<br>
An isolated polypeptide having phytase activity and a residual activity follov\/ing incubation at 37°C and in a 0,1 M Glycine/HCI buffer, pH 2.5, for 24 hours of at least 20%, as compared to the activity at time, t = 0, the activity being assayed at 37°C and pH 5.5 on 1% (w/v) Na-phytate, using a 0.25 M Na-acetate buffer pH 5.5, buffer blind subtracted;<br>
preferably with an identity to i) amino acids 1 to 413 of SEQ ID NO: 10, of at least 75%, preferably at least 76%, 77%, 78%, 79%, 80%, 81%, 82%, 83%, 84%&gt;, 85%, 86%, 87%, 88%, 89%, 90%, 91%, 92%, 93%, 94%, 95%, 96%, 96.5%, 97%, 97.5%, 98%, 98.5%, 98.7%, 98.8%, 98.9%, 99%, 99.0%, 99.1%, 99.2%, 99.3%, 99.4%, 99.5%, 99.6%, 99.7%, 99.8%, or at least 99.9%.<br>
An isolated polypeptide having phytase activity, wherein the activity of the polypeptide, at pH 5.0 and 37°C, measured on the substrate pNP-phosphate is less than 11% of the activity of the polypeptide measured on the substrate phytate; preferably with an identity to i) amino acids 1 to 413 of SEQ ID NO: 10, of at least 75%, preferably at least 76%, 77%, 78%, 79%, 80%, 81%, 82%, 83%, 84%, 85%, 86%, 87%, 88%, 89%, 90%, 91%, 92%, 93%, 94%, 95%, 96%, 96.5%, 97%, 97.5%, 98%, 98.5%, 98.7%, 98.8%, 98.9%, 99%, 99.0%, 99.1%, 99.2%, 99.3%, 99.4%, 99.5%, 99.6%, 99.7%, 99.8%, or at least 99.9%.<br>
An isolated polypeptide having phytase activity, wherein the polypeptide has a higher release of phosphorous (P), as compared to the phytase from Peniophora lycii; preferably as measured in the in vitro model; and/or, wherein the polypeptide preferably has an identity to i) amino acids 1 to 413 of SEQ ID NO: 10, of at least 75%, preferably at least 76%, 77%, 78%), 79%, 80%, 81%, 82%, 83%, 84%, 85%, 86%, 87%, 88%, 89%, 90%, 91%, 92%, 93%, 94%, 95%, 96%, 96.5%, 97%, 97.5%, 98%, 98.5%, 98.7%, 98.8%, 98.9%, 99%, 99.0%, 99.1%, 99.2%, 99.3%, 99.4%, 99.5%, 99.6%, 99.7%, 99.8%, or at least 99.9%o.<br>
An isolated polypeptide having phytase activity, wherein the polypeptide, dosed 0.25 FYT/g feed, releases at least 150% phosphorous (P), relative to the phosphorous released by the phytase from Peniophora lycii, also dosed 0.25 FYT/g feed; and/or, wherein the polypeptide preferably has an identity to i) amino acids 1 to 413 of SEQ ID NO; 10, of at least 75%, preferably at least 76%, 77%, 78%, 79%, 80%, 81%, 82%, 83%, 84%, 85%, 86%, 87%, 88%, 89%, 90%, 91%, 92%, 93%, 94%, 95%, 96%, 96.5%, 97%, 97.5%, 98%, 98.5%, 98.7%, 98.8%, 98.9%, 99%, 99.0%, 99.1%, 99.2%, 99.3%, 99.4%, 99.5%, 99.6%, 99.7%, 99.8%, or at least 99.9%.<br>
An isolated polypeptide having phytase activity, wherein the polypeptide, dosed 0.75 FYT/g feed, releases at least 150% phosphorous (P), relative to the phosphorous released by<br>
43<br>
 <br>
the phytase from Peniophora lycii, also dosed 0.75 FYT/g feed; and/or, wherein the polypeptide preferably has an identity to i) amino acids 1 to 413 of SEQ ID NO: 10, of at least 75%, preferably at least 76%, 77%, 78%, 79%o, 80%o, 81%, 82%,, 83%, 84%,, 85%, 86%, 87%, 88%, 89%, 90%, 91%, 92%o, 93%, 94%, 95%o, 96%c 96.5%, 97%, 97.5%, 98%o, 98.5%o, 98.7%&gt;, 98.8%o, 98.9%, 99%o, 99.0%, 99.1%, 99.2%, 99.3%, 99.4%, 99.5%, 99.6%,, 99.7%, 99.8%, or at least 99.9%o.<br>
I.	An isolated polypeptide having phytase activity, selected from the group<br>
consisting of: (a) a polypeptide having an amino acid sequence which has at least 75%,<br>
identity with (i) amino acids 1 to 413 of SEQ ID NO: 10, and/or (ii) the mature polypeptide part<br>
of SEQ ID NO: 10, (b) a polypeptide which is encoded by a polynucleotide which hybridizes<br>
under at least medium stringency conditions with (i) nucleotides 100 to 1338 of SEQ ID NO:<br>
9, (ii) the mature polypeptide encoding part of SEQ ID NO: 9, and/or (iii) a complementary<br>
strand of any one of (i), or (ii); (c) a variant of any one of the polypeptides of (a)(i)-(a)(ii),<br>
comprising a conservative substitution, deletion, and/or insertion of one or more amino acids;<br>
and (d) a fragment of any one of the polypeptides of (a)(i)-(a)(ii).<br>
II.	An isolated polynucleotide comprising a nucleotide sequence which encodes the polypeptide of section I.<br>
III.	An isolated polynucleotide encoding a polypeptide having phytase activity, selected from the group consisting of: (a) a polynucleotide encoding a polypeptide having an amino acid sequence which has at least 75%, identity, preferably at least 76%o, 77%, 78%,, 79%, 80%,, 81%), 82%,, 83%, 84%o, 85%o, 86%, 87%o, 88%, 89%,, 90%o, 91%o, 92%,, 93%, 94%,, 95%, 96%o, 96.5%, 97%o, 97.5%,, 98%, 98.5%, 98.7%, 98.8%, 98.9%, 99%,, 99.0%, 99.1%, 99.2%, 99.3%,, 99.4%o, 99.5%o, 99.6%, 99.7%,, 99.8%o, or at least 99.9%o identity with amino acids 1 to 413 of SEQ ID NO: 10; (b) a polynucleotide having at least 75%o identity, preferably at least 76%, 77%,, 78%,, 79%, 80%, 81%, 82%, 83%, 84%, 85%o, 86%o, 87%o, 88%, 89%, 90%, 91 %o, 92%o, 93%o, 94%, 95%, 96%o, 96.5%,, 97%o, 97.5%o, 98%o, 98.5%,, 98.7%o, 98.8%o, 98.9%o, 99%o, 99.0%o, 99.1%o, 99.2%o, 99.3%o, 99.4%o, 99.5%o, 99.6%o, 99.7%o, 99.8%o, or at least 99.9%, identity with nucleotides 100 to 1338 of SEQ ID NO: 9; and (c) a polynucleotide which hybridizes under at least medium stringency conditions with (i) nucleotides 100 to 1338 of SEQ ID NO: 9, (ii) the mature polypeptide encoding part of SEQ ID NO: 9, (iii) a complementary strand of any one of (i), or (ii).<br>
IV.	The isolated polynucleotide of any one of sections II and III, having at least one mutation in the mature polypeptide coding sequence of SEQ ID NO: 9, in which the mutant nucleotide sequence encodes a polypeptide comprising amino acids 1 to 413 of SEQ ID NO: 10.<br>
V.	A nucleic acid construct comprising the polynucleotide of any one of sections ll-IV operably linked to one or more control sequences that direct the production of<br>
44<br>
 <br>
the polypeptide in an expression host.<br>
VI.	A recombinant expression vector comprising the nucleic acid construct of section V.<br>
VII.	A recombinant host cell comprising the nucleic acid construct of section V.<br>
VIII.	A method for producing the polypeptide of section I comprising (a)<br>
cultivating a cell, which in its wild-type form is capable of producing the polypeptide, under<br>
conditions conducive for production of the polypeptide; and (b) recovering the polypeptide.<br>
IX.	A method for producing the polypeptide of section I comprising (a)<br>
cultivating a recombinant host cell comprising a nucleic acid construct comprising a<br>
nucleotide sequence encoding the polypeptide under conditions conducive for production of<br>
the polypeptide; and (b) recovering the polypeptide.<br>
X.	A transgenic plant, plant part or plant cell, which has been transformed with<br>
a polynucleotide encoding the polypeptide of section I.<br>
XI.	A transgenic, non-human animal, or products, or elements thereof, being capable of expressing the polypeptide of section I.<br>
XII.	Use of at least one polypeptide of section I in animal feed.<br>
<br>
XIII.	Use of at least one polypeptide of section I in the preparation of a composition for use in animal feed.<br>
XIV.	A method for improving the nutritional value of an animal feed, wherein at least one polypeptide of section I is added to the feed.<br>
XV.	An animal feed additive comprising (a) at least one polypeptide of section I; and (b) at least one fat soluble vitamin, (c) at least one water soluble vitamin, and/or (d) at least one trace mineral.<br>
<br>
XVI.	The animal feed additive of section XV, which further comprises at least one amylase, at least one additional phytase, at least one xylanase, at least one galactanase, at least one alpha-galactosidase, at least one protease, at least one phospholipase, and/or at least one beta-glucanase.<br>
XVII.	The animal feed additive of section XVI, wherein the additional phytase has a pH-optimum which is higher than the pH-optimum of the polypeptide having the amino acid sequence of amino acids 1 to 413 of SEQ ID NO: 10.<br>
IIXX. An animal feed composition having a crude protein content of 50 to 800 g/kg and comprising at least one polypeptide of section I.<br>
A polypeptide having phytase activity which comprises, preferably has or consists of, an amino acid sequence which has at least 75% identity, preferably at (east 76%, 77%, 78%, 79%, 80%, 81%, 82%, 83%, 84%, 85%, 86%, 87%, 88%, 89%, 90%, 91%, 92%, 93%, 94%, 95%, 96%, 96.5%, 97%, 97.5%, 98%, 98.5%, 98.7%, 98.8%, 98.9%, 99%, 99.0%, 99.1%, 99.2%, 99.3%, 99.4%, 99.5%, 99.6%, 99.7%, 99.8%, or at least 99.9% identity with<br>
45<br>
 <br>
amino acids 1 to413of SEQ ID NO: 10.<br>
A polypeptide having phytase activity which comprises, preferably has, the<br>
sequence of<br>
(i)      amino acids 1 to 413 of SEQ ID NO: 10, and/or<br>
(ii)     the mature polypeptide part of SEQ ID NO: 10; or which polypeptide<br>
(a)	is a variant of any one of the polypeptides of (i)-(ii), comprising a deletion, insertion, and/or conservative substitution of one or more amino acids; or<br>
(b)	is a fragment of any one of the polypeptides of (i)-(ii).<br>
The present invention is further described by the following examples which should not be construed as limiting the scope of the invention.<br>
EXAMPLES<br>
Example 1. Cloning of a Hafnia alvei phytase gene<br>
A multiple alignment was made of the following histidine acid phosphatases (HAP);<br>
appA Escherichia coil (SPTREI\/IBL:Q8GN88), Citrobacter gillenii DSM 13694 phytase<br>
(geneseqp:aeh04533),	Citrobacter        amalonaticus        ATCC        25407        phytase<br>
(geneseqp:aeh04535), Citrobacter braakii phytase (geneseqp:aeh04827), and ypo1648 Yersinia pestis C092 (SPTREMBL:Q8ZFP6). Two degenerate oligonucleotide primers were designed on the basis of consensus sequences:<br>
2123fw: 5'- CATGGTGTGCGNGCNCCNACNAA -3' (SEQ ID N0:1) 2065rev: 5'- CCCACCAGGNGGNGTRTTRTCNGGYTG -3' (SEQ ID N0:2), wherein Y designates T or C, R designates A or G, and N designates A, C, G or T.<br>
The primers were used for PCR screening of a number of bacterial species at annealing temperatures between 40 and 50°C but typical as touch down program starting with 50°C and then reduced the annealing temperature with 1°C for each cycle over the next 10 cycles before conducting standard PCR.<br>
A partial phytase gene in the form of an approximately 950 bp PCR fragment was identified in Hafnia alvei (DSM 19197).<br>
The PCR fragment was isolated from agarose gel and the fragment was sequenced using the same PCR primers the fragment was generated with. By translation of the nucleotide sequence, it was confirmed that the DNA fragment was part of a HAP phytase gene.<br>
For obtaining the full length nucleotide sequence of the gene, the DNA WALKING SPEEDUP™ Kit (DWSK-V102 from Seegene, Inc., 2nd Fl., Myungji BIdg., 142-21, Samsung-<br>
46<br>
 <br>
dong, Kangnam-gu, Seoul, 135-090, Korea) was used, which is designed to capture unknown target sites. For this purpose, 6 specific oligonucleotides were designed and used with the kit.<br>
2328	TSPIdw: 5'- ACTTGCATCGACGTTGGCTG (SEQ ID NO: 3)<br>
2329	TSP2dw: 5'- ACTGAGCAGCAATGGAACTCTCTG (SEQ ID NO: 4)<br>
2330	TSPSdw: 5- ACTGGGTTCCAATATCACGAGTC (SEQ ID NO: 5)<br>
2331	TSPIup: 5'- ATGGTGGATCGCTAAATCACACTG (SEQ ID NO: 6)<br>
2332	TSP2up: 5- ACGTCTGCCCAAACATACACG (SEQ ID NO: 7)<br>
2333	TSP3up: 5'- ACCGCCCATCAGGCTAATC (SEQ ID NO: 8)<br>
The full length nucleotide sequence encoding the phytase from Hafnia alvei DSM 19197 is shown in the sequence listing as SEQ ID NO: 9, and the corresponding encoded amino acid sequence is shown in SEQ ID NO: 10. The first 33 amino acids of SEQ ID NO: 10 (i.e. amino acids -33 to -1) are a signal peptide, as predicted by the software Signal P V3.0 (seewww.cbs.dtu.dk/services/SignalP/).<br>
Example 2. Expression of the Hafnia alvei phytase gene<br>
A 27 amino-acid signal peptide encoding polynucleotide of a native protease, Savinase™, from Bacillus licheniformis was fused by PCR in frame to the gene encoding the mature phytase from from Hafnia alvei. The signal peptide coding sequence is shown in SEQ ID NO: 11, encoding the signal peptide of SEQ ID NO: 12.<br>
The DNA coding for the fusion polypeptide was integrated by homologous recombination on a Bacillus subtilis host cell genome. The gene construct was expressed under the control of a triple promoter system (as described in WO 99/43835), consisting of the promoters from Bacillus licheniformis alpha-amylase gene (amyL), Bacillus amyloliquefaciens alpha-amylase gene (amyQ), and the Bacillus thuringiensis crylllA promoter including the mRNA stabilizing sequence. The gene coding for Chloramphenicol acetyl-transferase was used as marker, as described in, e.g., Diderichsen et al., A useful cloning vector for Bacillus subtilis. Plasmid, 30, p. 312, 1993.<br>
Chloramphenicol resistant transformants was cultured in PS-1 medium (10% sucrose, 4% soybean flour, 1% Na3P04-12H20, 0.5% CaCOa, and 0.01% pluronic acid) shaken at 250 RPM at SOX. After 2-5 days of incubation the supernatant was removed and the phytase activity was identified by applying 20 microliter of the supernatant into 4 mm diameter holes punched out in 1% LSB-agarose plates containing 0.1M Sodium acetate pH 4.5 and 0.1% Inositol hexaphosphoric acid. The plates were left over night at 37°C and a buffer consisting of 0.25M CaCb and 500mM MES (adjusted to pH 6.5 with 4N NaOH) was poured over the plates. The plates were left at room temperature for 1h and the inositolphosphate phosphatase, or phytase, activity was then identified as a clear zone.<br>
47<br>
 <br>
Several phytase positive transformants were analyzed by DNA sequencing to ensure the correct DNA sequence of the constructs. One correct clone was selected.<br>
Example 3. Fermentation of the Hafnia NN020125 phytase host<br>
The selected clone of Bacillus subtilis, which was harboring the Hafnia alvei phytase construct and was capable of expressing the phytase (mature part) was cultivated at 30°C and with 250rpm for 6 days in SK-1M medium (Sodium Caseinate (Miprodan 30 from Aria) 40g, Maltodextrin 01 (Giucidex 6, catalogue no. 332203 from Roquette), 200g, Soybean Meal 50g, Dowfax 63N10 (a non-ionic surlfactant from Dow) 0.1ml, tap water up to 1000 ml, CaCOs tablet 0.5g/100 ml).<br>
Example 4. Purification of Hafnia alvei phytase<br>
The fermentation supernatant with the phytase was first centrifuged at 7200rpm and 5°C for one hour and filtered through a sandwich of four Whatman glass microfibre filters (2.7, 1.6, 1.2 and 0.7 micrometer). Following this the solution was sterile filtered through a Seitz-EKS depth filter using pressure. Next, the filtered supernatant was pre-treated as follows:<br>
The sample solution was washed with water and concentrated using an ultrafiltration unit (Filtron, from Filtron Technology Corporation) equipped with a lOkOa cut-off ultrafiltration membrane. Then pH was adjusted to 4.5 with 10% (w/v) acetic acid, which caused a minor precipitation. No activity was found in the precipitate and it was removed by filtration through a Fast PES bottle top filter with a 0.22 micrometer cut-off.<br>
After pretreatment the phytase was purified by chromatography on S Sepharose, approximately 50 ml in a XK26 column, using as buffer A 50 mM sodium acetate pH 4.5, and as buffer B 50 mM sodium acetate + 1 M NaCI pH 4.5. The fractions from the column were analyzed for activity using the phosphatase assay (see below) and fractions with activity were pooled.<br>
The solution was added solid ammonium sulfate giving a final concentration of 1.5 M and the pH was adjusted to 6.0 using 6 M HCI. The phytase-containing solution was applied to a butyl-sepharose column, approximately 30 ml in a XK26 column, using as buffer A 25 mM bis-tris (Bis-(2-hydroxyethyl)irnino-tris(hydroxymethyl)methan)) + 1.5 M ammonium sulfate pH 6.0, and as buffer B 25 mM bis-tris pH 6.0. The fractions from the column were analyzed for activity using the phosphatase assay (see below) and fractions with activity were pooled. Finally, the solution containing the purified phytase was buffer-changed into 50 mM sodium acetate + 0.1 M NaCI, pH 4.5 and concentrated using an Amicon ultra-15 filtering device with a 30 kDa cut-off membrane.<br>
The molecular weight, as estimated from SDS-PAGE, was approximately 40 kDa and<br>
48<br>
 <br>
the purity was &gt; 95%.<br>
Example 5. Activity assays<br>
Determination of phosphatase activity<br>
75 microliter phytase-containing enzyme solution is dispensed in a microtiter plate well, e. g. NUNC 269620 and 75 microliter substrate is added (for preparing the substrate, two 5 mg p-nitrophenyl phophate tablets (Sigma, Cat.No. N-9389) are dissolved in 10 ml 0.1 M Na-acetate buffer, pH 5.5). The plate is sealed and incubated 15 min., shaken with 750 rpm at 37°C. After the incubation time 75 microliter stop reagent is added (the stop reagent is 0.1 M di-sodiumtetraborate in water) and the absorbance at 405 nm is measured in a microtiter plate spectrophotometer.<br>
Determination of phvtase activity<br>
75 microliter phytase-containing enzyme solution, appropriately diluted in 0.25M sodium acetate, 0.005% (w/v) Tween-20. pH5.5, is dispensed in a microtiter plate well, e. g. NUNC 269620, and 75 microliter substrate is added (prepared by dissolving lOOmg sodium phytate from rice (Aldrich Cat.No, 274321) in 10ml 0.25M sodium acetate buffer, pH5.5). The plate is sealed and incubated 15min. shaken with 750rpm at 37°C. After incubation, 75 microliter stop reagent is added (the stop reagent being prepared by mixing 10 ml molybdate solution (10% (w/v) ammonium hepta-molybdate in 0.25% (w/v) ammonia solution), 10ml ammonium vanadate (0.24% commercial product from Bie&amp;Berntsen, Cat.No. LABI7650), and 20ml 21.7% (w/v) nitric acid), and the absorbance at 405nm is measured in a microtiter plate spectrophotometer. The phytase activity is expressed in the unit of FYT, one FYT being the amount of enzyme that liberates 1 micromole inorganic ortho-phosphate per minute under the conditions above. An absolute value for the measured phytase activity may be obtained by reference to a standard curve prepared from appropriate dilutions of inorganic phosphate, or by reference to a standard curve made from dilutions of a phytase enzyme preparation with known activity (such standard enzyme preparation with a known activity is available on request from Novozymes A/S, Krogshoejvej 36, DK-2880 Bagsvaerd).<br>
Determination of specific phvtase activity<br>
The specific activity of the phytase was determined in sodium acetate buffer, pH 5.5. The phytase was highly purified as described above, i.e. only one component was identified on an SDS poly acryl amide gel.<br>
The protein concentration was determined by amino acid analysis as follows: An aliquot of the sample was hydrolyzed in 6M HCI, 0.1% phenol for 16h at 110°C in an evacuated   glass  tube.   The   resulting   amino  acids  were   quantified   using   an  Applied<br>
49<br>
 <br>
Biosystems 420A amino acid analysis system operated according to the manufacturer's instructions. From the amounts of the amino acids the total mass - and thus also the concentration - of protein in the hydrolyzed aliquot was calculated.<br>
The phytase activity was determined in the units of FYT as described above and the specific activity was calculated as the phytase activity measured in FYT units per mg phytase enzyme protein.<br>
The resulting specific activity was 980 FYT/mg protein. The specific activity was determined on sodium phytate at pH5.5 and 37°C.<br>
Example 6. Determination of the phytase pH profile<br>
The pH profile was determined at 37°C in the pH range of 2.0 to 7.5 (in 0.5 pH-unit steps) as described above in the section "Determination of phytase activity", except that a buffer cocktail (50mM glycine, SOmM acetic acid and 50mM Bis-Tris was used instead of the 0,25M sodium acetate pH5.5 buffer. The results are summarized in table 1 below. The values given for each pH in the range of 2.0 - 7.5 are the relative activity in % normalized to the value at optimum.<br>
Table 1: pH profile<br>
<br>
Phytase	2.0	2.5	3.0	3.5	4.0	P^<br>
4.5	5.0	5.5	6.0	6.5	7.0	7.5<br>
Hafnia alvei	46	61	83	95	100	100	88	71	43	18	3	0<br>
Example 7. Determination of the phytase isoelectric point<br>
The isoelectric point, pi, for the phytase was determined using isoelectric focusing gels (Novex pH 310 lEF gel from invitrogen, catalog number EC6655A2) run as described by the manufacturer. The pi for the Hafnia alvei phytase is about 7.4.<br>
Example 8. Phytase temperature profile<br>
The temperature profile (phytase activity as a function of temperature) was determined for the Hafnia alvei phytase in the temperature range of 20-90'C essentially as described above ("Determination of phytase activity"). However, the enzymatic reactions (100 microliter phytase-containing enzyme solution + 100 microliter substrate) were performed in PCR tubes instead of microtiter plates. After a 15 minute reaction period at desired temperature the tubes were cooled to 20°C for 20 seconds and 150 microliter of each reaction mixture was transferred to a microtiter plate. 75 microliter stop reagent was added<br>
50<br>
 <br>
and the absorbance at 405 nm was measured in a microtiter plate spectrophotometer. The results are summarized in Table 2 below. The numbers given for each temperature are relative activity (in %) normalized to the value at optimum.<br>
Table 2: Temperature profile<br>
<br>
Temperature °C	20	30	40	50	55	60	65	70	80	90<br>
Relative activity	17	27	45	69	79	85	100	95	7	0<br>
Example 9. Phytase Thermostability<br>
Hafnia alvei phytase expressed in both Aspergillus oryzae and Bacillus subtilis were subjected to thermostability measurements by Differential Scanning Calorimetry (DSC) and compared to the E, coli phytase (commercially available as PHYZYME XP from Danisco A/S).<br>
An aliquot of the protein sample of Hafnia alvei phytase (purified as described in Example 4) was dialysed against 2 x 500 ml 20 mM Na-acetate, pH 4.0 at 4°C in a 2-3h step followed by an overnight step. The sample was 0.45 pm filtered and diluted with buffer to approx. 2 A280 units. The exact absorbance values measured are given in the results table shown below. The dialysis buffer was used as reference in Differential Scanning Calorimetry (DSC). The samples were degassed using vacuum suction and stirring for approx. 10 minutes. An aliquot of the E. coli phytase from the commercial product PHYZYME XP was purified in a similar fashion as described in Example 4.<br>
A DSC scan was performed at a constant scan rate of 1.5 °C/min from 20-80 °C. Filtering period: 16 s. Before running the DSC, the phytases were dialyzed against the appropriate buffers (e.g. 0.1 M glycine-HCI, pH 2.5 or 3.0; 20mM sodium acetate pH 4,0; 0.1 M sodium acetate, pH 5.5; 0.1M Tris-HCI, pH 7.0). Data-handling was performed using the MicroCal Origin software (version 4.10), and the denaturation temperature, Td (also called the melting temperature, Tm) is defined as the temperature at the apex of the peak in the thermogram. To probe the reversibility of the unfolding process, a second scan was performed immediately after a short cooling phase. For the second scan the peak area (the area between the peak and the baseline = enthalpy of unfolding which is compared) is compared to the peak area of the first scan. A large peak (between 75-100 % of the peak area of the first scan) is interpreted as a reversible unfolding/folding process.<br>
The results of DSC for Hafnia alvei phytase expressed in both Aspergillus oryzae and<br>
Bacillus subtilis and the E. coli phytase are summarized in the Table 3 below.<br>
51<br>
 <br>
Table 3. Comparative Thermostability of Hafnia alvei Phytase and E. coli Phytase<br>
<br>
Phytase	Buffer	A280	Td 1st Scan	TD           2nd	Relative<br>
			(°C)	Scan (°C)	peak      size<br>
					(actual area) on 2 scans<br>
Aspergillus	20 nM NaAc	3.2	70.2	70.3	large<br>
expressed	ph4.0				<br>
H. alvei					<br>
Bacillus	20 nM NaAc	1.6	70.1	70.3	large<br>
expressed	pH4.0				<br>
H. alvei					<br>
E. coli	20 nM NaAc pH4.0	2.4	62.6	62.9	medium<br>
As illustrated in the above table, the Hafnia alvei phytase had greater thermostability than the £. coli phytase.   It is also clear that the thermostability of the Hafnia alvei phytase was not affected by the expression host.<br>
Example 10. Gastric Proteolytic Resistance of Hafnia alvei Phytase and E. coli Phytase<br>
Samples of H. alvei phytase and E. coli phytase (PHYZYME XP, available from Danisco) were treated with pepsin (Pepsin 1:60000 from Porcine Stomach Mucosa, Wako 162-18721, 2900 Units/mg, Lot SDK5232) in 250 mM glycine buffer pH 3.0 (approx. 1000 pepsin Units/mg phytase). Incubation for 30 minutes at 40 °C with shaking (750 rpm). Following incubation with pepsin, the phytase activity was determined as described in Example 5 and compared to the activity of a sample treated in the same way, but without addition of pepsin. The results are summarized in Table 4 below.<br>
TABLE 4. Gastric Proteolytic Resistance of Hafnia alvei Phytase and E. coli Phytase<br>
<br>
Phytase	Mean (res. act. %)	<br>
E. coli	96	result of two runs with results of 95% and 97% residual activity.<br>
52<br>
 <br>
		<br>
H. alvei	95.5	result of two runs with results of 97% and 94 % residual activity.<br>
Thus, the E. coli phytase and H. alvei phytase had very similar gastro proteolytic resistance properties.<br>
Example 11: Performance in animal feed in an in vitro model for the Hafnia alvei phytase and Citrobacter braaliii phytase<br>
The performance in animal feed of the i-lafnia alvei phytase was compared, in an in vitro model, to the performance of a Citrobacter braal<ii phytase. the in vitro model simulates gastro-intestinal conditions a monogastric animal and correlates well with results obtained trials vivo. phytase activity sample is determined as described example under of comparison was performed follows-.></ii>
Feed samples composed of 30% soybean meal and 70% maize meal with added CaCI2 to a concentration of 5 g calcium per leg feed are then prepared and pre-incubated at 40°C and pH 3.0 for 30 minutes followed by addition of pepsin (3000 U/g feed) and suitable dosages of the phytases (identical dosages are used for all phytases to be tested to allow comparison), for example between 0 1 to 1.0 phytase units FYT/g feed. A blank with no phytase activity was also included as reference. The samples was then incubated at 40°C and pH 3.0 for 60 minutes followed by pH 4.0 for 30 minutes.<br>
The reactions were stopped and phytic acid and inositol-phosphates extracted by addition of HCI to a final concentration of 0.5 M and incubation at 40°C for 2 hours, followed by one freeze-thaw cycle and 1 hour incubation at 40°C.<br>
Phytic acid and inositol-phosphates were separated by high performance ion chroma-"tography as described by Chen et al in Journal of Chromatography A (2003) vol. 1018, pp. 41-52 and quantified as described by Skog-'lund et al in J. Agric. Food Chem. (1997), vol. 45, pp. 431-436.<br>
53<br>
 <br>
Figure 1 shows a dose-response of the IHafnia alvei phytase compared to a Citrobacter braai<ii phytase at dosing of fyt feed and feed. the effects phytases in vitro are shown as residual inositol-phosphate bound phosphorous remaining a sample after incubation compared to ip-p control without phytase. all numbers given average standard deviation or replica incubations l-iafnia reduced amount about same degree citrobacter></ii>
Accordingly, the IHafnia phytase was able to obtain a very good reduction in the amount of residual inositol-phosphate bound phosphorous.<br>
Example 12: Performance in animal feed in an in vitro model for the Hafnia alvei phytase and Penioptiora lycii phytase phytase<br>
The performance in animal feed of the Hafnia alvei phytase in an in vitro model was also compared to the performance of a Peniophora lycii phytase at dosing of 250 FYT/kg and 500 FYT/kg feed. The results were obtained following the experimental protocol as described in Example 11. As shown in Figure 2, the Hafnia alvei phytase reduced the amount of residual IP-P in the in vitro sample better than the Peniophora lycii phytase.<br>
54<br>
<br>
<br>
WE CLAIM:<br>
1.	An isolated polypeptide having phytase activity, selected from the group consisting of:<br>
(a)	a polypeptide comprising an amino acid sequence which has at least 75% identity with<br>
(i)        amino acids 1 to413of SEQ ID NO: 10, and/or<br>
(ii)       the mature polypeptide part of SEQ ID NO: 10;<br>
(b)	a fragment of any one of the polypeptides of (a)(i) - (a)(ii).<br>
2.	An isolated polynucleotide comprising a nucleotide sequence which encodes the polypeptide of claim 1.<br>
3.	An isolated polynucleotide encoding a polypeptide having phytase activity, selected from the group consisting of:<br>
<br>
(a)	a polynucleotide encoding a polypeptide having an amino acid sequence which has at least 75% identity with amino acids 1 to 413 of SEQ ID NO: 10; and<br>
(b)	a polynucleotide having at least 75% identity with nucleotides 100 to 1338 of SEQ ID NO: 9.<br>
<br>
4.	A nucleic acid construct comprising the polynucleotide of any one of claims 2 or 3 operably linked to one or more control sequences that direct the production of the polypeptide in an expression host.<br>
5.	A recombinant expression vector comprising the nucleic acid construct of claim 4.<br>
6.	A recombinant host cell comprising the nucleic acid construct of claim 5.<br>
7.	A method for producing the polypeptide of claim 1, the method comprising (a) cultivating a cell, which in its wild-type form is capable of producing the polypeptide, under conditions conducive for production of the polypeptide; and (b) recovering the polypeptide.<br>
8.	A method for producing the polypeptide of claim 1, the method comprising (a) cultivating a recombinant host cell comprising a nucleic acid construct comprising a nucleotide sequence encoding the polypeptide under conditions conducive for production of the polypeptide; and (b) recovering the polypeptide.<br>
9.	A transgenic plant, plant part or plant cell, which has been transformed with a polynucleotide encoding the polypeptide of claim 1.<br>
55<br>
 <br>
10.	A transgenic, non-human animal, or products, or elements thereof, being capable of expressing the polypeptide of claim 1.<br>
11.	Use of at least one polypeptide of claim 1 in animal feed.<br>
12.	Use of at least one polypeptide of claim 1 in the preparation of a composition for use in animal feed.<br>
13.	A method for improving the nutritional value of an animal feed, wherein at least one polypeptide of claim 1 is added to the feed.<br>
14.	An animal feed additive comprising<br>
<br>
(a)	at least one polypeptide of claim 1; and<br>
(b)	at least one fat soluble vitamin, at least one water soluble vitamin, at least one trace mineral or combinations thereof.<br>
<br>
15.	The animal feed additive of claim 14, which further comprises at least one amylase, at least one additional phytase, at least one xylanase, at least one galactanase, at least one alpha-galactosidase, at least one protease, at least one phospholipase, and/or at least one beta-glucanase.<br>
16.	An animal feed composition having a crude protein content of 50 to 800 g/kg and comprising at least one polypeptide of claim 1.<br>
17.	A method for producing a fermentation product, comprising (a) fermenting using a fermenting microorganism a carbohydrate containing material in the presence of a phytase of claim 1 and (b) producing the fermentation product or fermentation coproduct from the fermented carbohydrate containing material.<br>
18.	The method of claim 1, wherein the fermentation product is ethanol, beer, wine, or distillers dried grains (DDG).<br>
19.	The polypeptide of claim 1, wherein the polypeptide comprises an amino acid sequence which has at least 80% identity, at least 85% identity, at least 90% identity, at least 95% identity, or at least 99% identity with amino acids 1 to 413 of SEQ ID NO: 10.<br>
56<br>
 <br>
20. The polynucleotide of claim 3, wherein the polynucleotide comprises a polynucleotide which has at least 80% identity, at least 85% identity, at least 90% identity, at least 95% identity, or at least 99% identity with nucleotides 100 to 1338 of SEQ ID NO: 9.<br>
 <br>
57<br>
</td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/viewdoc.aspx?id=MUZXpsUP9KmunjBh14Webg==&amp;amp;loc=egcICQiyoj82NGgGrC5ChA==" target="_blank" style="word-wrap:break-word;">http://ipindiaonline.gov.in/patentsearch/GrantedSearch/viewdoc.aspx?id=MUZXpsUP9KmunjBh14Webg==&amp;amp;loc=egcICQiyoj82NGgGrC5ChA==</a></p>
		<br>
		<div class="pull-left">
			<a href="279962-clamping-stub.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="279964-vapor-phase-decarbonylation-process.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>279963</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>5615/CHENP/2009</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>06/2017</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>10-Feb-2017</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>06-Feb-2017</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>23-Sep-2009</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>NOVOZYMES A/S</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td>KROGSHOEJVEJ 36, DK-2880 BAGSVAERD,</td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>LASSEN, SOREN, FLENSTED</td>
											<td>GYDEBAKKEN 3B, DK-3520 FARUM</td>
										</tr>
										<tr>
											<td>2</td>
											<td>SJOEHOLM, CARSTEN</td>
											<td>ALLEROEDVEJ 17, DK-3450 ALLEROED</td>
										</tr>
										<tr>
											<td>3</td>
											<td>SKOV, LARS, KOBBEROEE</td>
											<td>AAGE BERNTSENS ALLE 63, DK-2750 BALLERUP</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>C12N15/55</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>PCT/EP08/53561</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td>2008-03-26</td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td>07104870.6</td>
									<td>2007-03-26</td>
								    <td>EUROPEAN UNION</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/279963-hafnia-phytase by HTTrack Website Copier/3.x [XR&CO'2014], Thu, 04 Apr 2024 22:34:23 GMT -->
</html>
